"
Page
Discussion
View source
History
teams
Log in
 
Team:SDU-Denmark/Project
From 2009.igem.org
Home
Background S. aureus | Biofilm | Quorum-sensing | Quenrum-quenching | Brainstorm
Project Concept | Methods | Results | Implications | Protocols | Tools | Safety
Parts Submitted parts | Contributions | pSB1A3 characterization
Team Students | Advisors | About SDU | What we did to help others
Concept
Bacto-Bandage
Different kinds of plastic adhesive drapes are already in use today.
The main goal of our project was to assemble biobricks in to a plasmid and in this way make E. coli produce and excrete the quorum-quenching protein RIP.
We propose making a bandage that contains our engineered bacteria behind a semipermeable membrane, allowing only small peptides such as RIP to pass through, into the wound.
We have reason to hope that this will weaken the biofilm formation, thereby making multiresistant bacteria easier to treat with conventional antibiotics. This should also increase the effectiveness of the antibiotics and thereby decrease the amount and range of antibiotics that are needed today, to fight the infections.
Since our system doesn't kill S. aureus but only prevents it from becoming pathogenic, it is less likely to promote the development of resistant strains.
Parts
Our biological machine, composed of a promoter and RBS , our novel RIP part and a double terminator .
Our idea is to compose a plasmid containing a promoter, which should either be constitutive or inducible. After the promoter sequence we have a RBS, and then our RIP sequence, which we based on the amino acid sequence of the protein and made in PCR via two flanking primers. We are working with a clean RIP sequence and a RIP sequence with an export sequence on it, called sRIP. Following the RIP sequence we have a terminator.
All the four parts were incorporated into an ampicillin resistant plasmid backbone.
We designed our system with two different promotors, since we didn't know whether a high concentration of RIP will quorum quench the E. coli itself.  It will therefore be practical for us to control the rate of transcription - hence the inducible promoter.
On the other hand we know that a higher concentration of RIP is more efficient in quorum-quenching S. aureus, and it will therefore be more efficient if the plasmid is transcribed at all times. Furthermore if the export of the protein out of the cell is weak, an overproduction of the protein might puncture the cell and RIP will leak out. This will of course kill the cells, but we don't see this as a big problem as they easily can be replaced. Our goal is to get as much RIP exported from the bandage to the wound one way or another.
Methods
Flowchart detailing our project.
Finding parts
We used the Registry of Standard Biological Parts (Parts Registry in short) to find a suitable backbone , promoter, RBS and terminators . All the parts where checked for compatibility with the standard BioBrick assembly method ( RFC 10 ).
We also wanted to test our system with an inducible promoter , but sadly never had the time.
Ligations
We ligated our parts together using a slightly different protocol then suggested in the Parts Registry help pages , before inserting them into our backbone:
1. Amplification by PCR using VF2 and our modified VR primer ( Protocol ).
2. Digest with fast digest restriction enzymes, upstream part with SpeI and downstream part with XbaI ( Protocol ).
3. Ligate with T4 DNA Ligase, resulting in a mixed site ( Protocol ), of either 6 or 8 bp length.
Inserting parts in backbone
The steps was as follows:
To skip a bacerial growth step, and thereby save some time, our parts where amplified by PCR, as suggested in Ginkgo Bioworks assembly manual .
1. Amplification the ligated products by PCR using VF2 primer and our modified VR2 primer ( Protocol ).
2. Digesting with fast digest restriction enzymes EcoRI-HF™ and PstI ( Protocol ).
We used fast digest enzymes, since normal enzymes where either slow or ineffective in cutting our bricks.
3. Ligated with T4 DNA Ligase ( Protocol ).
4. Transformation into competent E.coli ( Protocol ).
Testing
Flowchart concerning our testing phase.
To test if our 4 part brick was actually inserted into E.coli and functional, we prepared three different test systems: gene transcription by measuring RIP mRNA expression (Northern Blot), MALDI to measure protein expression and a actual test of quorum-quenching.
Colony PCR
Gel picture of Colony PCR.
We started by checking our part lengths by colony PCR ( Protocol ).
Sequencing
We sent our parts to sequencing to be sure that we had exactly the right DNA in our bacteria. We used the Plasmid editor APE , to see our sequencing data.
RNA expression
To measure if RNA was being expressed, we used northern blotting ( Protocol ). We used a radioactive probe (32P) that binds to RIP mRNA and ran it through a membrane, to separate mRNA's into different sizes. The membrane was then put on a film, and scanned so we could visualize if there was any RIP mRNA in our cells.
Biofilm assay
Biofilm assay.
To test whether or not our RIP part was being secreted out of our E. coli biomachine, we designed a biofilm assay.
The assay was made on a 4*6-well plate. S.aureus were grown in the wells, thereby making biofilms.
We then added different E. coli components to the wells: supernatant, crushed pellet and living E. coli. After 24 hours the wells were washed three times with sterile 1x PBS solution to remove planctonic bacteria. Cells remaining adhered to the wells are subsequently stained with violet blue, the solution stands for 15 minutes and was washed twice.
1 ml of 96% ethanol was added to each stained well to loosen the biofilm. The biofilm was tested by measuring the absorbance by transferring the crystal violet/ethanol solution from each well to a cuvette, and measure the optical density (OD) of each of these samples at a wavelength of 595 nm.
MALDI
Matrix-assisted laser desorption/ionization (MALDI) is a technique used in mass spectrometry, allowing the analysis of for example proteins.
We did a MALDI on the supernatant of our 4 part brick, to see whether or not RIP was being exported out of the bacteria.
Results
Sequencing confirmed that we have the right DNA in our bacteria. The red parts are restriction-sites, green is the composite promoter and RBS, blue is RIP and orange is our double terminator.
New parts and primers
We have successfully constructed the RIP and sRIP parts , and used these with older parts, all of which have been added to the registry. We have also added a slightly modified VR primer to the registry.
RIP expression in E. coli
Northern blot of our 4 brick part () and our RIP-Terminator part (). As seen here only the 4 brick part (left) is transcribed into mRNA and hopefully further translated into peptide.
We have successfully assembled a 4 brick part containing a constitutive promotor with RBS, our novel RIP part and a terminator.
Sequencing confirmed the 4 brick part DNA sequence and a northern blot analysis confirmed that our E. coli indeed transcribes the RIP gene. To our knowledge this is the first recombinant organism to harbour and transcribe the RIP gene.
Mass spectrometry of supernatant from our 4 part RIP colony (A and C) compared with a reference colony (B and D). No conclusive results could be gathered from this analysis.
We also did protein MALDI mass spectrometry, which didn't show anything conclusive, perhaps as a consequence of the RIP peptides small size, which makes it hard to distinguish from fragmentation peaks of larger proteins. Another possible explanation could be that our bacteria is not transporting RIP out of the cell.
Quorum-quenching and biofilm inhibition
We are currently testing the quorum-quenching effect on S. aureus biofilm formation and the results are promising. The big question is whether or not, RIP gets exported out of E. coli and past the peptideglycan layer and capsule. The final results will be presented at the Jamboree.
Problems
Throughout most of the summer, we've had problems getting our ligations product sizes to match. What ever we tried nothing seemed to work, since it was always the wrong size. Then a few weeks ago, it was discovered, that the problem was with our backbone (1K from plate 1, 2009 spring distribution), not our work. 1K was suppose to be , but instead it was .
Sadly we didn't discover this till the very last week, and so we didn't get to do all the experiments we wanted, including one with an inducible promoter and one with our signaling peptide coupled with RIP.
Implications
If indeed our E. coli can inhibit biofilm formation by producing RIP, this generates many different opportunities for further development.
Our machine opens up for the possibility to combat infections more intelligently.
As mentioned earlier we considered using a inducible promotor to control the expression of RIP. Optimally the promotor should react to the level of autoinducers in the environment by use of quorum-signaling receptors.  The concept could also be implicated on other strains of bacteria performing quorum sensing with other types of autoinducers.
A logical next step would be to couple several kinds of quorum-quenching molecules to our machine, together with different quorum-signaling receptors. This would allow for inhibition of only the specific bacteria that are currently infecting the wound.
Essentially the Bacto-Bandage would sense if any kind of bacteria was trying to colonize the wound and inhibit this by means of producing the most effective quorum-quenching molecules.
Our project in 1 minute
We want to fight hospital infections, using a biological machine.
For this purpose we have constructed a recombinant E. coli, which produces the RIP peptide. RIP inhibits biofilm formation in the bacteria S. aureus.
This peptide will make it easier to fight infections and kill S. aureus with conventional antibiotics.
Other stuff
Check out all our protocols .
See what tools we used.
Read our safety page.
Content
Contents
1Concept
1.1Bacto-Bandage
1.2Parts
2Methods
2.1Finding parts
2.2Ligations
2.3Inserting parts in backbone
2.4Testing
2.4.1Colony PCR
2.4.2Sequencing
2.4.3RNA expression
2.4.4Biofilm assay
2.4.5MALDI
3Results
3.1New parts and primers
3.2RIP expression in E. coli
3.3Quorum-quenching and biofilm inhibition
3.4Problems
4Implications
4.1Our project in 1 minute
4.2Other stuff
4.3Content
Retrieved from " http://2009.igem.org/Team:SDU-Denmark/Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
