"
Page
Discussion
View source
History
teams
Log in
 
Team:SDU-Denmark/Protocols
From 2009.igem.org
Home
Background S. aureus | Biofilm | Quorum-sensing | Quenrum-quenching | Brainstorm
Project Concept | Methods | Results | Implications | Protocols | Tools | Safety
Parts Submitted parts | Contributions | pSB1A3 characterization
Team Students | Advisors | About SDU | What we did to help others
Protocol for Primer PCR
dNTP mix (10mM)
10ul 100mM dGTP
10ul 100mM dATP
10ul 100mM dTTP
10ul 100mM dCTP
60ul H2O
(Total V=100ul)
PCR reaction
39ul H2O
5.0ul Pfx-buffer (-MgSO4)
1.0ul 50mM MgSO4
1.5ul 10mM dNTP mix
1.5ul forward primer
1.5ul reverse primer
0.5ul Pfx enzyme (add just before PCR run)
PCR program
For PCR of BioBricks:
1.
Start
94˚C
2min
2.
Denaturing
94˚C
1min
3.
Annealing
52˚C
1min
4.
Elongation
68˚C
2min
5.
GOTO 2
rep. 29x
6.
End
68˚C
3min
7.
Hold
4˚C
Protocol for colony PCR
• A little colony is transfered to each tube (afterwards the same tip is used for plating out on a LA+Amp plate)
• The tubes are placed in the microwave at full power for 2 min with an open lid.
• 25 μl Pre-mix (with taq-pol) is added to the PCR tubes. Mix well by pipetting up/down.
Pre-mix (to 1 colony)
2,5  μl   10 x TAQ Buffer + MgCl2
1,25  μl 10 pmol/μl forward primer
1,25 μl  10 pmol/μl reverse primer
0,50  μl dNTP
19,25 μl H2O
0,25 μl Taq. Pol → Pre-mix is made without taq pol. which is added after  the colonies have been zapped in the microwave,
=25 μl   taq . pol is added to the pre-mix → PCR-tube
Make enough premix for your number of colonies + 3
PCR program
1.
Start
94˚C
2min
2.
Denaturing
94˚C
1min
3.
Annealing
52˚C
1min
4.
Elongation
72˚C
2min
5.
GOTO 2
rep. 29x
6.
End
72˚C
3min
7.
Hold
4˚C
Protocol for purification of DNA from TAE and TBE agarose gel bands
Kit from GFX
Sample capture
Weigh a DNase-free 1,5 ml microcentrifuge tube
Excise band of interest from the gel and place in microcentrifuge tube
Weigh microcentrifuge tube plus agarose gel band
Calculate weight of agarose gel slice
Add 10 ul Capture buffer type 2 for each 10 mg agarose gel slice
Mix by inversion
Place at 60 degrees until agarose is completely dissolved
Sample binding
Add up to 600 ul Capture buffer-sample mix to assembled GFX MicroSpin columns and Collection tubes.
Leave at room temperature for 60 sec.
Centrifuge for 30 sec at 16000 g.
Discard the flow through in the Collection tube and place the MicroSpin column in the Collection tube again.
Repeat sample binding step until all sample is loaded onto the MicroSpin column.
Wash & dry
Add 500 ul Wash buffer type 1
Centrifuge for 30 sec at 16000 g.
Discard flow through and keep Collection tube as above.
Centrifuge again for 30 sec at 16000 g. More flow through will appear in the Collection tube. It is important to centrifuge this second time to get the sample completely dry. This step is not provided in the original protocol.
Discard Collection tube and transfer MicroSpin column to a clean 1,5 ml DNase-free microcentrifuge tube.
Elution
Add 10 – 50 ul Elution buffer type 4 or 6. We eluted with 10 ul in order to obtain a small volume and a high concentration of purified DNA. Only very big amounts of sample require higher elution volumes.
Leave at room temperature for 60 sec.
Centrifuge for 60 sec at 16000 g.
Retain flow through and discard MicroSpin columns
Store purified sample DNA at -20 degrees or proceed to cutting DNA or ligation.
Protocol for purification of DNA from PCR mixtures or an enzymatic reaction
Kit from GFX
Sample capture
Add 500 ul Capture buffer type 2 to up to 100 ul sample
Mix thoroughly
Sample binding
Add Capture buffer-sample mix to assembled GFX MicroSpin columns and Collection tubes
Centrifuge for 30 sec at 16000 g.
Discard the flow through in the Collection tube and place the MicroSpin column in the Collection tube again.
Wash & dry
Add 500 ul Wash buffer type 1
Centrifuge for 30 sec at 16000 g.
Discard flow through and keep Collection tube as above.
Centrifuge again for 30 sec at 16000 g. More flow through will appear in the Collection tube. It is important to centrifuge this second time to get the sample completely dry. This step is not provided in the original protocol.
Discard Collection tube and transfer MicroSpin column to a clean 1,5 ml DNase-free microcentrifuge tube.
Elution
Add 10 – 50 ul Elution buffer type 4 or 6. We eluted with 10 ul in order to obtain a small volume and a high concentration of purified DNA. Only very big amounts of sample require higher elution volumes.
Leave at room temperature for 60 sec.
Centrifuge for 60 sec at 16000 g.
Retain flow through and discard MicroSpin columns
Store purified sample DNA at -20 degrees or proceed to cutting DNA or ligation.
Protocol for making cells (E.coli) competent for transformation
(Cells are kept on ice at all times!! If the cells temperature rises above ~5º C they'll lose their competency!)
600 µl overnight (ON) Top10 E. coli culture is added to 60 ml Luria-Bertani (LB) medium.
Grows at 37º C while being shaken until the optic density (OD550) is 0,2.
Cool cells on ice.
Harvest the cells in screwcap tubes (4 × 10 ml).
Pour away the supernatant and keep the pellet on ice.
Wash the cells with 10 ml cold 50mM CaCl2.
Distribute 200 µL cells to each of many eppendorf tubes.
Add 41.7 µl 87% glycerol and mix well.
Store at -80º C.
Protocol for making cells (E. coli) competent for electroporation
(Cells are kept on ice at all times!! If the temperature rises above ~5º C they'll lose their competency!)
2ml over-night cell culture is transferred to 200ml Luria Bertani (LB)-medium.
Incubate at 37˚C on shaker until OD450=0.5-0.7
Keep on ice for 15-30min and harvest by 4000 x g for 15 min at 4˚C.
Remove all supernatant and resuspend carefully in 200ml ice cold dH2O. Harvest as in 3.
Resuspend in 100ml ice cold dH2O. Harvest as in 3.
Resuspend in 20ml ice cold 10% glycerol. Harvest as in 3.
Resuspend in ice cold 10% glycerol until a total volume of 1ml. Cell concentration should be 1-3x1010 cells/ml.
Cells are distributed in tubes with 40ul in each and kept at -80˚C.
Protocol for transforming (E.coli)
Take 2 and 5 µl of the plasmid from the distribution plates and add it to 2 different tubes with 50 µl competent cells in each. As controls use 1 tube with competent cells only (control of the cells) and 1 tube with puc plasmid which is supercoiled and known to work (control of our plasmid).
Store on ice for 40 min.
Keep at 42º C for 2 min.
Keep on ice for 5 min.
Add 1 ml LB to each tube.
The tubes are stored at 37 º C for 2 hours while being shaken.
Centrifuge the tubes at 3500 rpm for 5 min.
Suck up and throw out 850 µl and resuspend the remaining by pipetting up and down.
Spread out 75 µl on 2 LA (LB + agarose) plates with antibiotics (100 µg/ml ampicillin). The competent cells of the control are spread out on 1 LA plate with antibiotics and 1 without, since they are not resistant.
Store at 37 º C over night.
Electroporation
This protocol is very useful compared to standard-transformation, because it uses the power of ZAP! Requires less raw material than a normal transformation and is faster.
Thaw competent cells at room temperature and then keep on ice. 0.2cm cuvettes for electroporation are cooled on ice.
For each transformation, transfer 40ul cells to a 1.5ml Eppendorf tube and add 1.5ul plasmid from distribution plates. Mix well and transfer to the bottom of a cuvette without making air bubbles. Keep on ice for 5min.
Place the cuvette in the electroporator and pulse once. Setup: Gene pulser: 25uF 2.5kV Pulse controller: 200ohm
Immediately add 1ml SOC medium to the cuvette, mix well by pipetting, and transfer to a 1.5ml Eppendorf tube and incubate immediately at 37˚C for 1h on shaker (500 rpm).
Plate the transformed cells onto LA plates containing antibiotics (100 ug/ml ampicillin). Untransformed cells (negative control) should be plated with and without antibiotics.
Miniprep protocol
We used BIO-RAD Quantum Prep, Plasmid Miniprep Kit.
All centrifugation steps are performed at maximum speed (12.000-14.000 g = 14.000 rpm).
Transfer an ON culture (2 mL) of plasmid-containing cells to a microcentifuge tube. Pellet the cells by centrifugation for 30 sec. Remove all of the supernatant by aspirating or pipetting.
Add 200 µl of the Cell Resuspension Solution and vortex or pipet up and down until the cell pellet is completely resuspended.
Add 250 µl of the Cell Lysis Solution and mix by gently inverting the capped tube about 10 times (do not vortex). The solution should become viscous and slightly clear if cell lysis has occurred.
Add 250 µl of the Neutralization Solution and mix by gently inverting the capped tube about 10 times (do not vortex). A visible precipitate should form.
Pellet the cell debris for 5 min in a microcentrifuge. A compact white debris pellet will form along the side or at the bottom of the tube. The supernatant (cleared lysate) at this step contains the plasmid DNA.
While waiting for the centrifugation step at step 5, insert a Spin Filter into one of the 2 ml wash tubes supplied with the kit. Mix the Quantum Prep matrix by repeated shaking and inversion of the bottle to insure that it is completely suspended (no tubes are supplied with the sample kit, however, most 2 and 1,5 ml tubes will accommodate the Spin Filters).
Transfer the cleared lysate (supernatant) from step 5 to a Spin Filter, add 200 µl of thoroughly suspended matrix, then pipet up and down to mix. If you have multiple samples, transfer the lysates first, then add matrix and mix. When matrix has been added to all samples and mixed, centrifuge for 30 sec.
Remove the Spin Filter from the 2ml tube, discard the filtrate at the bottom of the tube and replace the filter in the same tube. Add 500µL of Wash Buffer and wash the matrix by centrifugation for 30 seconds.
Remove the Spin Filter from 2 ml tube, discard the filtrate at the bottom of the tube and replace the filter in the same tube. Add 500 µL of Wash buffer and wash the matrix by centrifugation for a full 2 minutes to remove residual traces of ethanol.
Remove the Spin Filter and discard the microcentrifuge tube. Place the filter in one of the 1.5 mL collection tubes supplied with the kit or any other standard 1.5 mL microcetrifuge tube which will accommodate the Spin Filter. Add 50 µL of deionized H2O. Elute the DNA by centrifugation for 1 minute at top speed.
Discard the Spin Filter and store DNA at -20 degress Celsius.
Restriction digest - Protocol 1
Pool plasmid and dry on vacuum centrifuge down to about 50uL
Mix the following into one tube:
4uL Plasmid and RIP
5uL 10x Buffer
0,5uL BSA
Fill with water to 47uL (37,5uL)
1,5uL Enzyme 1
1,5uL Enzyme 2
Incubate for 2 hours on 37 degress C.
Inactivate the enzymes at 80 degress C for 20 minutes.
Place product on -20 degress C or continue immediately to ligation.
Restriction digest - Protocol 2
Fast digest restriction enzymes. Fast digest restriction enzymes have proved more efficient for cutting DNA, and is less time-consuming to work with. Fast Digest enzymes can be bought at Fermentas.
24 ul water
2 ul enzyme
4 ul Fast Digest buffer
10 ul PCR product
Leave for 15 min at 37 degrees. Afterwards, inactivate the enzyme for 20 min at 80 degrees.
Be aware,  cut only with ONE enzyme at a time.
After cutting with enzyme 1, isolate and purify the DNA fragments on a gel before applying enzyme 2.
The final volume when cutting is 40 ul.
Add 4 ul loading buffer to the eppendorf tube and load directly onto gel.
Run gel and purify from gel.
Eluate with 10 ul when purifying from gel.
Eventually, place your sample in the vacuum centrifuge in order to get a smaller volume and greater concentration before ligation is applied. Final volume prior to ligation should be 5 ul.
Restriction digest - Protocol 3
Fav
Restriction protocol á la Anna.
Check that you have all the enzymes and prepare two gels.
2 ul PCR product
2 ul buffer (Green buffer)
1 ul Enzyme 1
1 ul Enzyme 2
14 ul H20
Mix the products gently together. Quick spin down. Incubate 5 min. at 37 degrees, mildly shaken.
After 5 min. quickly load 15 ul in one gel (purification gel), and the remaining 5 ul in the other gel (test-gel). After 15-20 min. check the test-gel, print a picture, and cut out the correct band of the purification gel and purify using the TAE/TBE protocol .
Ligation - Protocol 1
Fav
Mix the following:
2uL 10x Ligase buffer
1uL T4 DNA ligase
2 or 4 uL cut backbone
5 or 10 uL cut PCR product
Leave at 17 degress C overnight.
Use for electroporation.
Ligation - Protocol 2
Takes place in eppendorf tube.
2 ul 10x T4 ligase buffer
1 ul T4 ligase
5 ul PCR product (cut) of each brick which is to be ligated together - or 1 part plasmid and 5 part bricks
Leave at 17 degrees over-night.
Test ligation using PCR and run a test gel afterwards in order to check the PCR product has the right size.
Ligation - Protocol 3
Fav
Use our plasmid ligation helper to calculate the volume of Plasmid and Insert to be mixed.
Mix cut Plasmid and cut Insert with the following:
2uL 10x Ligase buffer
1uL T4 DNA ligase
Leave at 17 degress C overnight.
Use for electroporation.
Protocol to Hot Phenol RNA Isolation
Solution 1: 0,3M sucrose; 0,01M NaAc, pH 4,5
Solution 2: 2% SDS; 0,01M NaAc, pH 4,5
Phenol saturated in 0,01-0,03M NaAc pH 4,5 containing 0,1% 8-hudroxyquinolin
I. 200 ml phenol + H2O (2cm water on top)
II. Add 0,2g 8-hudroxyquinolin
III. Add 6,66ml 3M NaAc pH4,5
IV. Shake, leave over night and in the end check pH of the water phase.
1. Resuspend frozen cell pellet in 300 µl ice cold solution 1 and transfer into a 1,5 ml microcentrifuge tube containing 300 µl solution 2. Proceed immediately with 2.
2. Add 400 µl Phenol, mix and incubate for more than 3 min. at 65˚C. When handling multiple samples these steps are repeated until all tubes are collected in the heating block. Snap Freeze in liquid nitrogen, 15s and spin at max speed for 5 min.
3. Transfer the water layer to a fresh 1,5 ml microcentifuge tube containing 600 µl hot phenol (it is important to avoid the organic phase in all steps). Mix and incubate for 3 min. at 65˚C. Snap Freeze in liquid nitrogen, 15s and spin at max speed for 5 min.
4. Transfer the water layer to a fresh 1,5 ml micorcentrifuge tube containing 300 µl phenol and 300 µl chloroform. Mix and spin.
5. Transfer the water layer to a fresh 1.5 ml microcentrifuge tube containing 40 µl 3M NaAc pH 4,5 + 900 µl 96% EtOH. Incubate at -20˚C for 15 min (or over night if required).
6. Centrifuge at 20.000g (14681 rpm, Eppendorf 5417R) for 20 min. At 4˚C.
7. Carefully remove the supernatant and wash the pellet with 200 µl ice cold 70% EtOH. Spin at 20.000 g for 5 min. at 4˚C.
8. Carefully remove the supernatant and dry the pellet for max. 10 min. in speed-vac.
9. Resuspend the pellet in 50 µl RNase-free H2O. spin again and transfer to a new tube.
Test of RNA
1. Mix. 2 µl RNA, 2 µl H2O and 2 µl formaid loading buffer.
2. Run on agarose gel.
3. Test concentration on nanodrop.
Protocol for Northern Blotting
Gel mix (6%):
• 50 ml 10×TBE
• 75 ml 40% Acrylamide
• 30ml  2% Bis-acrylamide
• 240g CH(NH2)2
• Ajust with, H2O to 500 ml
Preparation of samples
1. After determining the concentration, delute the samples to a concentration of 2 µg/µl. Take 5 µl of the diluted sample. This gives you 10 µg/lane.
2. Run prepared samples on agarose gel to check for decomposition.
3. Add the calculated amount to an eppendorf tube and adjust with loading buffer to 15 µl. (minimum 10 µl). Make sure the ratio between loading buffer and sample is 3:1
4. Preheat the gel for 30 min.
5. Denature the prepared samples at 95˚C for 3 min. Put samples on ice immediately after to avoid renaturation. The samples are now ready to be loaded on the gel.
6. Run the gel at 300 V for 2-3 hours until the BPB colour (the lowest) is 3-4cm form the bottom of the gel.
7. The gel is now ready for blotting
Semi dry northern electroblotting
1. Cut 6 pieces Whatman 3MM paper and 1 piece of Zeta-probe membrane (14,5 ×16 cm for small gel, 24×17 cm for large gel-or whatever area of the needs to be blotted)
2. Wet 2 pieces of Whatman paper in 1×TBE and place them on the top electrode (the cathode).
3. Place a pieces of paper on the gel and cut away the gel not covered by the paper. Remove the gel and paper from the glass plate and wet it slowly in 1×TBE. Place it on top of the sandwich with the gel facing up.
4. Wet the membrane in 1×TBE and place it on top of the gel.
5. Wet the remaining 3 pieces of paper in 1×TBE and place them on top of the membrane. Roll carefully with a glass or plastic pipette to remove air bubbles.
6. Assemble the blotting device and turn it around. Run at 400 mA for 1 hour.
Protocol for 5S rRNA hybridization
Probe to 5S rRNA:
• 2 µl ss RNA 1pmol/ µl
• 1 µl PNK buffer (10x)
• 4 µl H2O
• 2 µl 32P-ATP
• 1 µl PNK (polynucleotide kinase)
• Incubate for 1 hour at 37˚C
• Add 1 µl  DNAse
• Incubate for 15 min. at 75˚C.
• Store in LED tubes at -20˚C until use.
• Prior to use denature probe for 2 min. at 90˚C.
Hybridization
1. Preincubate the membrane in 7 ml hybridization buffer for 30 min. at 42˚C.
2. Add 10 µl probe
3. Incubate over night at 42˚C.
Wash of membrane
1. Add Low stringency buffer (fill up the tube half-way) at room temp. and wash for 5 min.
2. Add High stringency buffer (fill up the tube half-way) and wash for 10min at 42˚C.
3. Let the membrane air-dry and wrap it in vita-wrap.
Buffer-solutions:
Low stringency buffer (2X SSC, 0,1% SDS) To 500ml milliQ water add 100ml 20X SSC and 10 ml 10% SDS. Adjust with milliQ water to 1l.
High stringency buffer (0,5% SSC, 0,1%SDS) To 500ml milliQ water add 25ml 20X SSC and 10ml 10% SDS. Adjust with milliQ water to 12l.
Protocols
Here you can find all our protocols. We've marked all our favorite protocols with a yellow star.
Content
Contents
1Protocol for Primer PCR
1.1dNTP mix (10mM)
1.2PCR reaction
1.3PCR program
2Protocol for colony PCR
2.1Pre-mix (to 1 colony)
2.2PCR program
3Protocol for purification of DNA from TAE and TBE agarose gel bands
3.1Sample capture
3.2Sample binding
3.3Wash & dry
3.4Elution
4Protocol for purification of DNA from PCR mixtures or an enzymatic reaction
4.1Sample capture
4.2Sample binding
4.3Wash & dry
4.4Elution
5Protocol for making cells (E.coli) competent for transformation
6Protocol for making cells (E. coli) competent for electroporation
7Protocol for transforming (E.coli)
8Electroporation
9Miniprep protocol
10Restriction digest - Protocol 1
11Restriction digest - Protocol 2
12Restriction digest - Protocol 3
13Ligation - Protocol 1
14Ligation - Protocol 2
15Ligation - Protocol 3
16Protocol to Hot Phenol RNA Isolation
17Protocol for Northern Blotting
17.1Preparation of samples
17.2Semi dry northern electroblotting
17.3Protocol for 5S rRNA hybridization
17.4Buffer-solutions:
17.5Protocols
17.6Content
Retrieved from " http://2009.igem.org/Team:SDU-Denmark/Protocols "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
