<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_IONIS_Paris_Project_Optogenetics skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:IONIS Paris/Project/Optogenetics</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Optogenetics</TITLE></P><DIV class="banner row" id="banner" style="width:101%;height: 100vh; "><DIV class="parallax text-center" style="background-image: url(https://static.igem.org/mediawiki/2015/thumb/b/b1/Optogenetics.jpg/800px-Optogenetics.jpg);"><DIV class="parallax-pattern-overlay" style="height: 100vh; "><DIV class="container text-center" style="height:650px;padding-top:200px;"><H1 class="intro wow zoomIn" wow-data-delay="0.4s" wow-data-duration="0.9s" style="font-family:sans-serif; color:#B00000 ">Optogenetics</H1></DIV></DIV></DIV></DIV><DIV id="about" class="container spacer about"><H2 class="text-center wowload fadeInUp" style="color:#ff0000"><BIG>What is Optogenetics?</BIG></H2><DIV class="row"><DIV class="wowload fadeInUp"><H4> Sensing light </H4><P align="justify">Nature has evolved a multiple of photoreceptors able to sense light.
						Those systems have provided <FONT style="color:#66cdaa"><B>synthetic biology tools </B></FONT>for the precise control of biological functions, offering superior spatial and temporal resolution. 
						The research field focusing on the combination of optical methods and genetics has been named «optogenetics» and can lead to the control of gene expression, cell morphology or even signaling pathway thank to light signals.
					</P></DIV><DIV class="col-sm-6 wowload fadeInLeft"><H4> Photoreceptors </H4><H5> What is VVD and how does it work? </H5><P align="justify"> Among other photoreceptors,<FONT style="color:#66cdaa"><B> Vivid </B></FONT>(VVD) is the smallest known <FONT style="color:#66cdaa"><B>Light–oxygen–voltage</B></FONT> (LOV) domain protein and photo-inducible dimer. 
								Isolated from Neurospora crassa, VVD forms a homo-dimer in response to a blue-light stimulus.
								The LOV domain, present in VVD, is a small blue-light sensing domain found in prokaryotes, fungi and plants.
								After blue-light activation, a covalent bond is formed between the co-factor <FONT style="color:#66cdaa"><B>Flavin mononucleotide</B></FONT> (FMN) and one of the cysteine residue. 
								This bond leads to a conformational change inducing functions by dissociating the C-terminal a-helix (Ja) and the LOV-core. 
								In VVD, this undock triggers <FONT style="color:#66cdaa">homodimerization</FONT> (Bilwes, Dunlap, &amp; Crane, 2007; Müller &amp; Weber, 2013).
							</P></DIV></DIV><DIV class="row"><DIV class="col-sm-6 wowload  fadeInLeft"><H4> Why did we choose the photoreceptor VVD ?  </H4><P align="justify" align="right">Contrary to other photoreceptors, VVD is a<FONT style="color:#66cdaa"> small protein </FONT>with 150 amino-acids facilitating accurate molecular design and avoiding steric issues (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1616014">BBa_K1616014)</A>. 
						Moreover, it is a homo-dimer when most of photo-inducible dimers are heterodimers. 
						In addition, the use of  VVD is easy; and doesn’t need any addition of co-factors: VVD works with <FONT style="color:#66cdaa"><B>Flavin adenine dinucleotide</B></FONT> (FAD) which is already abundant in eukaryote and prokaryote cells (Müller &amp; Weber, 2013; Nihongaki, Suzuki, Kawano, &amp; Sato, 2014).   
						</P></DIV></DIV></DIV><DIV id="about" class="container spacer about"><H2 class="text-center wowload fadeInUp" style="color:#ff0000"><BIG>Split-Protein</BIG></H2><DIV class="row"><DIV class="col-sm-6 wowload fadeInLeft"><H4>What is a split-protein and how does it work?</H4><P align="justify"> A split protein is a protein whose sequence has been<FONT style="color:#66cdaa"> divided into two</FONT> (or more) different parts. 
						Often used to study protein-protein interactions, the protein can not perform its function until the parts are put back together. 
						For instance, <FONT style="color:#66cdaa"><B>YFP</B></FONT>, the yellow-fluorescent protein that we are willing to produce in our engineered bacteria, will only express fluorescence when its two parts will be reunited. 
					</P></DIV><DIV class="col-sm-6 wowload fadeInRight"><H4>Why did we choose a split-protein?</H4><P align="justify"> In normal condition, the production of a protein in response to a stimulus can easily reach several hours due to the many steps required for the protein synthesis.
						By using split-proteins, we are taking advantage of the absence of fluorescence when the two parts are apart. 
						Indeed, the two parts of our split-YFP, when remaining separated, can be produced without being effective. 
						Therefore, the overall process is far<FONT style="color:#66cdaa"> less time-consuming</FONT>. 
						However, to implement a light control on the fluorescence activation, a genetic construction gathering the VVD photoreceptor and our split-YFP has to be engineered.  
					</P></DIV></DIV><DIV class="wowload fadeInUp"><H4>Biomolecular fluorescence complementation </H4><P align="justify">The new alternative approach for the visualization of  protein interactiosn has been developed; the<FONT style="color:#66cdaa"><B> biomolecular fluorescence complementation</B></FONT> (BiFC) techniques based on the complementation between fragments of fluorescent proteins;  fragments of the <FONT style="color:#66cdaa"><B>yellow fluorescent protein</B></FONT> (YFP) brought together by the association of two interaction partners fused to the fragments. 
				They noticed that the spectral characteristics of BiFC of YFP were virtually identical to those of intact YFP.(Chang-Deng Hu, 2003)  
			</P></DIV><DIV class="wowload fadeInUp"><H4>VVD - Split YFP in our Bioconsole </H4><P align="justify">The initiation of our Bioconsole consists of making visible our bacteria for the player. 
					In this way, the idea is to induce bacteria fluorescence through light signals.
					For this, we have added to each part of the YFP-split the VVD homodimer, so we have a system triggering by light inducing fluorescence (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1616001">BBa_K1616001</A> and <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1616002">BBa_K1616002</A>).
				</P></DIV></DIV><DIV class="highlight-info"><DIV class="overlay spacer"><DIV class="container"><DIV class="row text-center  wowload fadeInDownBig"><CENTER><DIV class="col-sm-3 col-xs-6"><H4>Light signals</H4></DIV><DIV class="col-sm-3 col-xs-6"><H4>Photoreceptors</H4></DIV><DIV class="col-sm-3 col-xs-6"><H4>Bacteria</H4></DIV><DIV class="col-sm-3 col-xs-6"><H4>Bio-Console</H4></DIV></CENTER></DIV></DIV></DIV></DIV><DIV id="about" class="container spacer about"><H2 class="text-center wowload fadeInUp" style="color:#ff0000"><BIG>Kill switch </BIG></H2><DIV class="row"><H4>What is a kill switch?</H4><P align="justify"> A kill switch is a <FONT style="color:#66cdaa"><B>genetically-encoded suicide trigger</B></FONT>.
						This trigger can be a change in the environment which will make the organism’s life depend on it. 
						In our project, the change will be the presence or not of light.
						That is very interesting as we could create a game over in our Bio Console, once we trigger the bacteria’s death through light.
						Moreover, our team is  really concerned about<FONT style="color:#66cdaa"> BioSafety</FONT>. 
						It is important for us to avoid any possibility of escaping from our microfluidic chip.
						The Kill switch would allow to contain our genetically modified bacteria into the chip through lethality.
					</P></DIV></DIV><DIV id="about" class="container spacer about"><H2 class="text-center wowload fadeInUp" style="color:#ff0000" id="killswitch"><BIG>A photosensible system: the pDawn and pDusk plasmids</BIG></H2><DIV class="row"><H4>How do they work?</H4><P align="justify">  After reading many papers to select an appropriate light-sensing system to use for the kill switch, our team came across the pDawn and pDusk plasmids. 
						Those plasmids employ <FONT style="color:#66cdaa">bluelight photoreceptors</FONT> to confer light-repressed or light-induced gene expression in with up to 460-fold induction upon illumination. 
						Controlled by this system, we thought to add <FONT style="color:#66cdaa"><B>toxins</B></FONT> in it to kill our bacteria whenever we want(Ohlendorf, Vidavski, Eldar, Moffat, &amp; Möglich, 2012). </P><P align="justify"> They are not dependent on <FONT style="color:#66cdaa">nonnative chromophores</FONT> that are often supplied exogenously and do not need any introduction of cofactor-synthesis genes. 
						Moreover, they are practical to use since current optogenetic tools require multiple plasmid components.</P><P align="justify"> Finally, they deal with many limitations in a one-plasmid system which is based on the <FONT style="color:#66cdaa"><B>YF1/FixJ system</B></FONT>.</P><P align="justify">The plasmids pDusk for light repressed and pDawn for light-activated gene expression. 
						In pDusk, the YF1/FixJ drives gene expression from the <FONT style="color:#66cdaa"><B>pFixK2 promoter</B></FONT> in a blue-light repressed manner. 
						When we insert the<FONT style="color:#66cdaa"><B>λ phage repressor cI</B></FONT> and the <FONT style="color:#66cdaa"><B>λ promoter pR</B></FONT> in pDawn, this will invert the signal polarity and lead to the gene expression.</P><P align="justify">The <FONT style="color:#66cdaa"><B>histidine kinase YF1</B></FONT> employs a light oxygen voltage blue light photosensor domain. 
						In the absence of blue light, YF1 phosphorylate the <FONT style="color:#66cdaa"><B>regulator FixJ</B></FONT> which induces gene expression from the FixK2 promoter. 
						The opposite happens with pDawn.
						We obtained the plasmids from the Centre for Biological Signalling Studies and the University of Freiburg. As soon as we received them, we observed that the multiple cloning site as well as the λ promoter pR were located upstream the YF1/FixY system.</P><DIV class="col-sm-6 wowload fadeInLeftt"><H4>pDawn &amp; pDusk - Toxin systems </H4><P align="justify">So we aim to construct an efficient kill-switch triggered by light. In this way we will construct two different parts. The first one will be composed of pDawn system and <FONT style="color:#66cdaa"><B>toxin</B>.</FONT>
						Toxins will be produced after light stimulation and induce death. The second part will be composed of pDusk system and toxin, this will lead to the cell death when the bacteria won’t be exposed to light. 
						This second part will be more difficult to characterized indeed all steps before characterization of the plasmid as plasmid amplification will have to be under light. </P></DIV><DIV class="col-sm-6 wowload fadeInRight"><H4>Toxins</H4><P align="justify"> In order to have the most efficient kill-switch, we decided to test several toxins in our system. We have choosen three toxins; holin/endolysin, ccdB and HokD.
						Holin (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1616007">BBa_K1616007</A>) and endolysin (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1616006">BBa_K1616006</A>) are toxins; after a period of late-gene expression, holin, an inner permeabilizing protein creates holes in the membrane in order to permeabilize it and then endolysin, a muralytic enzyme traverses the holes created by holin and degraded the cell wall.
						This complex induces cell lysis of bacteria (Young &amp; Bläsi, 1995). The HokD protein (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1616008">BBa_K1616008</A>) is a toxin when overexpressed kills the cells from the inside by interfering with a vital function in the cell membrane.
						Then, ccdB (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1616012">BBa_K1616012</A>) comes from the ccd system in  Escherichia coli F plasmid and acts as a gyrase poison.
						</P></DIV><P align="justify">
					The pDawn &amp; pDusk system, have been optimized and constructed in the way that the toxins sequences we introduce in our construction are in fact the reverse sequences.
					In this way; for holin/endolysin we have taken the reverse sequence of the previous iGEM <A href="http://parts.igem.org/Part:BBa_K124003"><FONT style="color:#66cdaa"><B>BioBrick BBa_K124003</B></FONT></A>. 
				</P></DIV></DIV><DIV class="highlight-info"><DIV class="overlay spacer"><DIV class="container"><DIV class="row text-center  wowload fadeInDownBig"><CENTER><DIV class="col-sm-3 col-xs-6"><H4>Light signals</H4></DIV><DIV class="col-sm-3 col-xs-6"><H4>Photoinducible systems</H4></DIV><DIV class="col-sm-3 col-xs-6"><H4>Bacteria</H4></DIV><DIV class="col-sm-3 col-xs-6"><H4>1 Kill switch</H4></DIV></CENTER></DIV></DIV></DIV></DIV></DIV><DIV id="about" class="container spacer about"><H2 class="text-center wowload fadeInUp" style="color:#ff0000"><BIG> Going further: possible applications of our systems  </BIG></H2><DIV class="row"><DIV class="col-sm-6 wowload fadeInLeftt"><H4>VVD constructions</H4><P align="justify">
						This construction involving a photorecepteur dimerization upon light illumination could be of great interest in some proteins interaction researches. 
						Indeed, our team decided to use this construction in a playful and entertaining system: the Bio-Console, but we also thought of  more &quot;serious&quot; applications and uses of this tool.
						The VVD construction could be used in order to induce forced and controlled proteins interaction:
						
						For example, coupled to the knockout of normal protein expression, it would initiate the establishment of a metabolic pathway at a specific time T (in the absence of the original signal).
							
							This process would allow  in particular,the design of some diseases models.
						
						The system, could also be used in order to induce forced protein interactions inhibition using the lack of light signal at a specific time T.
						
						For example, by delaying proteins interaction it is possible to study the development of compensatory mechanisms; including the initiation of stress mechanisms.
							</P></DIV><DIV class="col-sm-6 wowload fadeInRight"><P align="center">
						Example: protein recruitment initiating the activation of a metabolic pathway
						</P></DIV><DIV class="col-sm-6 wowload fadeInLeft"><H4>pDawn-pDusk constructions</H4><P align="justify"> 
						
						The pDawn construction (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1616019">BBa_K1616019)</A>used as a kill switch in our project is a reusable tool for all safety matters. Morevover, regarding the construction and design of both systems pDawn and pDusk, these ones could be used in any way taking into account that any gene of interest can be inserted in reverse in these constructions. 
						</P></DIV></DIV></DIV><DIV id="about" class="container spacer about"><H4> References </H4><P align="justify" style="font-size:14px"> Bilwes, A. M., Dunlap, J. C., &amp; Crane, B. R. (2007). Conformational Switching in the Fungal Light Sensor Vivid, 36(May), 1054–1058. 
			 Chang-Deng Hu, T. K. K. (2003). Simultaneous visualization of multiple protein interactions in living cells using multicolor fluorescence complementation analysis. 
			   Nature Biotechnology, 21(5), 539–545.
			 Ohlendorf, R., Vidavski, R. R., Eldar, A., Moffat, K., &amp; Möglich, A. (2012). From Dusk till Dawn: One-Plasmid Systems for Light-Regulated Gene Expression, 534–542.
			<A href="http://doi.org/10.1016/j.jmb.2012.01.001" style="text-decoration:none; color:#66cdaa">   http://doi.org/10.1016/j.jmb.2012.01.001</A> Müller, K., &amp; Weber, W. (2013). Optogenetic tools for mammalian systems. Molecular bioSystems, 9(4), 596–608.
			<A href="http://doi.org/10.1039/c3mb25590e" style="text-decoration:none; color:#66cdaa">    http://doi.org/10.1039/c3mb25590e </A> Young, R., &amp; Bläsi, U. (1995). Holins: Form and function in bacteriophage lysis. In FEMS Microbiology Reviews (Vol. 17, pp. 191–205).
			<A href="http://doi.org/10.1016/01686445(94)00079-4" style="text-decoration:none; color:#66cdaa">http://doi.org/10.1016/01686445(94)00079-4 </A></P></DIV><DIV id="works" class=" clearfix grid"><FIGURE class="effect-oscar"><A href="https://2015.igem.org/Team:IONIS_Paris/Description"><FIGCAPTION><H2>PROJECT</H2><P>Our reality game using bacteria</P></FIGCAPTION></A></FIGURE><FIGURE class="effect-oscar"><A href="https://2015.igem.org/Team:IONIS_Paris/Biobricks"><FIGCAPTION><H2>BIOBRICKS</H2><P>Our parts of DNA</P></FIGCAPTION></A></FIGURE><FIGURE class="effect-oscar"><A href="https://2015.igem.org/Team:IONIS_Paris/Project/Microfluidics"><FIGCAPTION><H2>MICROFLUIDICS</H2><P>Our challenge in achieving the Bio-console</P></FIGCAPTION></A></FIGURE></DIV><DIV id="blueimp-gallery" class="blueimp-gallery blueimp-gallery-controls"><H3 class="title">Title</H3><A class="prev">‹</A><A class="next">›</A><A class="close">×</A></DIV><DIV class="navbar-wrapper"><DIV class="container"><NAV class="navbar navbar-inverse navbar-fixed-top" role="navigation" id="top-nav"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="navbar-collapse  collapse" id="navbar-collapse"><UL class="nav navbar-nav navbar-right scroll"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2015.igem.org/Team:IONIS_Paris/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:IONIS_Paris/Project/Optogenetics">Optogenetics</A></LI><LI><A href="https://2015.igem.org/Team:IONIS_Paris/Project/BioLuminescence">BioLuminescence</A></LI><LI><A href="https://2015.igem.org/Team:IONIS_Paris/Project/Microfluidics">Microfluidics</A></LI><LI><A href="https://2015.igem.org/Team:IONIS_Paris/Project/Game_Over">Game Over</A></LI><LI><A href="https://2015.igem.org/Team:IONIS_Paris/Experiments">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:IONIS_Paris/Results">Results</A></LI></UL><LI><A href="https://2015.igem.org/Team:IONIS_Paris/Team">Team</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts </A></LI><LI><A href="https://2015.igem.org/Team:IONIS_Paris/Parts">Team parts</A></LI><LI><A href="https://2015.igem.org/Team:IONIS_Paris/Biobricks">Biobricks</A></LI><LI><A href="https://2015.igem.org/Team:IONIS_Paris/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:IONIS_Paris/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:IONIS_Paris/Practices">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:IONIS_Paris/Safety">Safety</A></LI></DIV></DIV></NAV></DIV></DIV><DIV class="footer text-center spacer">
		IONIS Paris team. 2015 Edition </DIV></DIV></DIV></DIV></BODY></HTML>