<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Kobe_RandD skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Kobe/RandD</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><UL class="accordion-menu"><LI class="menu-title"><DIV onclick="obj=document.getElementById('open').style; obj.display=(obj.display=='none')?'block':'none';"><A style="cursor:pointer;">MENU</A></DIV></LI><DIV id="open"><LI class="directlink"><A href="https://2017.igem.org/Team:Kobe">
                    HOME
                </A></LI><DIV class="dropdownlink">
                PROJECT
                </DIV><UL class="submenuItems"><LI><A href="https://2017.igem.org/Team:Kobe/Background">
                        Background
                    </A></LI><LI><A href="https://2017.igem.org/Team:Kobe/Overview">
                        Overview

                    </A></LI><LI><A href="https://2017.igem.org/Team:Kobe/Design">
                        Design
                    </A></LI><LI><A href="https://2017.igem.org/Team:Kobe/Parts">
                        Parts
          </A></LI><LI><A href="https://2017.igem.org/Team:Kobe/Experiments">
                        Experiments
                    </A></LI><LI><A href="https://2017.igem.org/Team:Kobe/RandD">
                        Results &amp; Discussions
                    </A></LI><LI><A href="https://2017.igem.org/Team:Kobe/Notebook">
                        Notebook
                    </A></LI></UL><LI class="directlink"><A href="https://2017.igem.org/Team:Kobe/Safety">
                    SAFETY
                </A></LI><DIV class="dropdownlink">
                HUMAN PRACTICE
                </DIV><UL class="submenuItems"><LI><A href="https://2017.igem.org/Team:Kobe/HP/Gold_Integrated">
                        Integrated Human Practice
                    </A></LI><LI><A href="https://2017.igem.org/Team:Kobe/HP/Silver">
                        Education and Public Engagement
                    </A></LI></UL><LI class="directlink"><A href="https://2017.igem.org/Team:Kobe/InterLab">
                    INTERLAB
                </A></LI><LI><DIV class="dropdownlink">
                    ABOUT US
                    </DIV></LI><LI><A href="https://2017.igem.org/Team:Kobe/Team">
                            Meet the Team
                        </A></LI><LI><A href="https://2017.igem.org/Team:Kobe/Attributions">
                            Attributions
                        </A></LI><LI><A href="https://2017.igem.org/Team:Kobe/Collaborations">
                            Collaborations

                        </A></LI><LI><A href="https://2017.igem.org/Team:Kobe/Acknowledgement">
                            Acknowledgements and Sponsors

                        </A></LI></DIV></UL></P><DIV class="main"><H1>Results &amp; Discussion</H1><H2>Results</H2><H3>
        1. Growth test
    </H3><P>
        Fig.1 shows the result of growth test of each bacteria. In every picture, the plate on the bottom contains L-theanine, the plate on the right contains NH<SUB><SMALL>4</SMALL></SUB>Cl (positive control), and the plate on the left contains nothing as a nitrogen source (negative control).
    </P><P>
        From the results, we learned that <I>B. subtilis</I> 168 and <I>B. subtilis</I> NCIB 3610 could grow much better than the others in the media containing L-theanine as a sole nitrogen source.
    </P><FIGURE><FIGCAPTION>Fig1. Growth test on agar plates with different nitrogen source.</FIGCAPTION><SMALL>
            5 strains of bacteria were cultured on minimum media for 6 days at 37℃. In every picture, the plate on the bottom contains L-theanine, the plate on the right contains NH<SUB><SMALL>4</SMALL></SUB>Cl, and the plate on the left contains nothing as a nitrogen source. These pictures show that <I>B. subtilis</I> strain NCIB 3610 and 168 could grow well using L-theanine as a sole nitrogen source.
        </SMALL></FIGURE><H3>
        2. RNA extraction
    </H3><P>
        Fig.2 shows bacterial growth and time when we added each component and sampled each medium. We extracted RNA from these media.
    </P><SMALL>
        Fig. 2 Bacterial growth
        When the OD reached 0.15, L-theanine (final concentration 18.7mM), glutamate (final concentration 1.1 mM), L-theanine(18.7 mM)+glutamate(1.1 mM) ,and water were added to each flask. When the OD 600 reached 0.3, 1 ml of each medium was transferred to a fresh Eppendorf tube.

    </SMALL><P>
        Fig.3 shows the results of electrophoresis of RNA samples. It indicates that RNA extraction was successful, as there were two clear bands derived from ribosomal RNA.
    </P><FIGURE><SMALL>
            (M: Marker,　G: Glutamate,　T: Theanine,　W: Water)
        </SMALL><FIGCAPTION>
            Fig.3 Electrophoresis to check the RNA samples
        </FIGCAPTION></FIGURE><H3>
        3. RNA-seq
    </H3><P>
        We have selected possible candidate genes based on the RPKM value.


    </P><P>
        Below are the conditions we set to discover the theanine responsive genes (that are induced stronger in the presence of L-theanine than glutamate and water.)
    </P><UL><LI>
            RPKM
            RPKM（theanine）&gt;100
        </LI><LI>
            Gene Expression
            theanine/water&gt;3, glutamate/water≲1

        </LI></UL><P>
        In this analysis, we were able to narrow down the number of candidate genes to 31. Below are the data of these genes.
    </P><P><A href="https://static.igem.org/mediawiki/2017/9/9e/T--Kobe--bfyubyu.png">Fig. 4 Rough selection of candidate genes by RNA-seq</A>
        Moreover, we narrowed down these 31 genes to 8 by picking up the genes that were induced well in the presence of L-theanine but not induced much in the presence of glutamate.
    </P><P><A href="https://static.igem.org/mediawiki/2017/b/b9/T--Kobe--result10.pdf">Fig. 5 More strict selection of candidate genes by RNA-seq.</A>
        Finally, we selected 8 genes (<I>nasD, nasA, iseA, yrbD, pucR, amtB, glnK</I>, and <I>ureA</I>)
    </P><H3>
        4. RT-qPCR
    </H3><P>
        To measure the expression of these 8 genes more accurately, we carried out RT-qPCR. Fig. 6 is the result of qPCR analysis.
    </P><FIGURE><FIGCAPTION> Fig. 6  Relative expression of 8 genes compared with the expression level of <I>rpsJ</I> (reference gene) <I>rpsJ</I> is one of the housekeeping genes, which codes for 30S ribosomal protein S10 in <I>B. subtilis</I></FIGCAPTION></FIGURE><P>
        Finally, we selected three genes (<I>nasA, yrbD</I>, and <I>amtB</I>), and designed our Biobrick parts so that our parts would be inserted into these three genes. (If you want to know how we designed these parts, please visit <A href="https://2017.igem.org/Team:Kobe/Design">our design page</A>.)
    </P><H3>
        5. Transformation
    </H3><P>
        Our Biobricks
        <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2233000">BBa_K2233000(target for nasA)</A>,
        <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2233001"> BBa_K2233001(target for yrbD)</A>, and
        <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2233002">BBa_K2233002(target for amtB)</A>
        were taken up into different <I>B. subtilis</I> NCIB 3610 respectively. After the transformation, DNA sequences of <I>nasA</I>, <I>yrbD</I>, and <I>amtB</I> were amplified by PCR. We verified that our Biobrick parts were successfully inserted into the chromosome of <I>B. subtilis</I> NCIB 3610.
    </P><FIGURE><FIGCAPTION>
            Fig. 7 Transformation check using electrophoresis of PCR products
            (marker, <I>nasA</I> 1-1, <I>nasA</I>, 1-2, <I>nasA</I> 2-1, <I>nasA</I> 2-2, <I>nasA</I> (negative control), <I>amtB</I> 1, <I>amtB</I> 2, <I>amtB</I> 3, <I>amtB</I> (negative control),<I> yrbD</I> 1-1, <I>yrbD</I> 1-2, <I>yrbD</I> 3-1, <I>yrbD</I> (negative control))
            Two of four <I>nasA</I>, three of three <I>amtB</I>, and three of three <I>yrbD</I> had longer sequences (about 5.7 kbp) than negative control (about 1.4 kbp). This indicated that our Biobrick parts were successfully fused to these genes.
        </FIGCAPTION></FIGURE><H3>
        6. Beta-galactosidase assay
    </H3><P>
        Fig. 8 shows the result of beta-galactosidase assay test. It was revealed that both <I>yrbD</I> and <I>amtB</I> were induced much more strongly in the presence of theanine than glutamate and water four hours after adding each component.
    </P><FIGURE><FIGCAPTION>
            Fig. 8 Beta galactosidase assay test
             Enzyme activity is calculated by the following formula.
            Enzyme activity=Changes in o-nitrophenol concentration/ bacterial density (OD<SUB><SMALL>600</SMALL></SUB>)/ reaction volume (ml) / reaction time (min)
        </FIGCAPTION></FIGURE><P>
         We carried out X-gal plate assay in an attemp to create a easier kit for measuring the concentration of theanine.
        Below is the result of X-gal plate assay with <I>B. subtilis</I> NCIB 3610 whose <I>yrbD</I> region was fused to <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2233001">BBa_K2233001</A></P><P>
        It indicates that beta-galactosidase was not expressed well when we added glutamate, and it was expressed well when we added theanine as we anticipated. However, it was also expressed well when water was added.
    </P><FIGURE><FIGCAPTION>
            Fig. 9 X-gal plate assay
            (top: 18.7 mM theanine, middle: water, bottom: 1.1 mM glutamate)
            100 µL of each liquid was added to every penicillin cup.
        </FIGCAPTION></FIGURE><H2>
        Discussion and future work
    </H2><P>
        In our experiments, we proved that our Biobrick parts(BBa_K2233000, BBa_K2233001, and BBa_K2233002) were successfully inserted at the target sites by homologous recombination as we had expected.
    </P><P>
        By using our Biobrick parts, we could analyze the expression of <I>yrbD</I> and <I>amtB</I> in the synthetic media containing L-theanine, glutamate, or water respectively as a sole nitrogen source. We revealed that <I>yrbD</I> fused to our Biobrick parts were expressed much more with L-theanine than glutamate and water. Therefore, it is estimated that our parts BBa_K2233001(target for <I>yrbD</I>) has a potential to change our chassis <I>B. subtilis</I> NCIB 3610 into a biosensor for L-theanine.
    </P><P>
        In order to prove whether our parts are useful for L-theanine sensor, we believe that next step we should take is to check how the expression level of <I>yrbD</I> change according to the concentration of L-theanine.
    </P><P>
        As for BBa_K2233000(target for <I>nasA</I>), we gave up to  use this part due to a lack of time.
    </P><P>
        Although we have successfully verified that <I>yrbD</I> were induced much more by L-theanine, we couldn’t see clear difference of L-theanine and water in X-gal plate. We estimated that it was due to inappropriate setting of the content of X-gal in the plate, the amounts of bacteria we add, the timing of observation, or simple different conditions between liquid culture and solid culture. We need more research !
    </P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>