Team:Kobe/Results&Discussion
MENU
HOME
PROJECT
Background
Overview
Design
Parts
Experiments
Results & Discussions
Notebook
SAFETY
HUMAN PRACTICE
Integrated Human Practice
Education and Public Engagement
INTERLAB
ABOUT US
Meet the Team
Attributions
Collaborations
Acknowledgements and Sponsors
Results
1. Growth test
Fig.1 shows the result of growth test of each strain. In every picture, the plate on the bottom contains L-theanine, the plate on the right contains NH
4
Cl (positive control), and the plate on the left contains nothing as a nitrogen source (negative control).
As a consequence, it turned out that B.subtilis 168 and B.subtilis NCIB 3610 could grow well in the media containing L-theanine as a sole nitrogen source.
B.subtilis 168
B.subtilis NCIB 3610
B.subtilis var. natto
E.coli K-12 C600
E.coli BL21
Fig1. Growth test on agar plates with different nitrogen source.
5 strains of bacteria were cultured on minimum media for 6 days at 37℃. In every picture, the plate on the bottom contains L-theanine, the plate on the right contains NH
4
Cl, and the plate on the left contains nothing as a nitrogen source. These pictures show that B.subtilis strain NCIB 3610 and 168 could grow well using L-theanine as a sole nitrogen source.
2. RNA extraction
Fig.2 shows bacterial growth and time when we added each component and sampled each medium. We extracted RNA from these media we sampled.
Fig. 2 Bacterial growth
When the OD reached 0.15, L-theanine (final concentration 18.7mM), glutamate (final concentraton 1.1 mM), L-theanine(18.7 mM)+glutamate(1.1 mM) ,and water were added to each flask. When the OD 600 reached 0.3, 1 ml of each medium was transferred to a fresh Eppendorf tube.
Fig.3 shows the results of electrophoresis of RNA samples. It indicates that RNA extraction was succeeded, as there were two clear bands derived from ribosomal RNA.
(M: Marker, G: Glutamate, T: Theanine)
(M: Marker, G+T: Glutamate+ Theanine, W: Water)
Fig.3 Electrophoresis to check the RNA samples
3. RNA-seq
We have selected possible candidate genes based on the RPKM value.
RPKM stands for “Reads Per Kilobase of exon per Million mapped fragments”. This value indicates the gene expression from RNA sequencing data by normalizing the total read length and the number of sequence reads.
Below are the conditions we set to discover genes that are induced stronger in the presence of L-theanine than glutamate and water.
RPKM
RPKM（theanine）>100
Gene Expression             theanine/water>3, glutamate/water<3
As a result, we were able to narrow down the number of genes to 31. Below are the data of these genes.
Fig. 4 Rough selection of candidate genes by RNA-seq
Moreover, we narrowed down these 31 genes to 8 by picking up genes that were induced well in the presence of L-theanine and not induced well in the presence of glutamate
Fig. 5 More strict selection of candidate genes by RNA-seq.
Finally, we selected 8 genes (nasD, nasA, iseA, yrbD, pucR, amtB, glnK, and ureA)
4. RT-qPCR
To measure the expression of these 8 genes more accurately, we carried out RT-qPCR. Fig. 6 is the result of qPCR analysis.
Fig.6
Fig. 6  Relative expression of 8 genes compared with the expression level of rpsJ (reference gene). rpsJ is one of the housekeeping genes, which codes for 30S ribosomal protein S10 in B.subtilis .
Finally, we selected three genes (nasA, yrbD, and amtB), and designed our Biobrick parts so that our parts would be inserted into these three genes. (If you want to know how we designed these parts, please visit our design page .)
5. Transformation
Our Biobricks BBa_K2233000(target for nasA) , BBa_K2233001(target for yrbD) , and BBa_K2233002(target for amtB) were taken up into different B.subtilis NCIB 3610 respectively. After the transformation, DNA sequences of nasA, yrbD, and amtB were amplified by PCR. We verified that our Biobrick parts were successfully inserted into the chromosome of B.subtilis NCIB 3610.
Fig. 7 Transformation check using electrophoresis of PCR products
(marker, nasA 1-1, nasA, 1-2, nasA 2-1, nasA 2-2, nasA (negative control), amtB 1, amtB 2, amtB 3, amtB (negative control), yrbD 1-1, yrbD 1-2, yrbD 3-1, yrbD (negative control))         Two of four nasA, three of three amtB, and three of three yrbD had longer sequences (about 5.7 kbp) than negative control (about 1.4 kbp). This indicated that our Biobrick parts were successfully fused to these genes.
