<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Kobe_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Kobe/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><UL class="accordion-menu"><LI class="menu-title"><DIV onclick="obj=document.getElementById('open').style; obj.display=(obj.display=='none')?'block':'none';"><A style="cursor:pointer;">MENU</A></DIV></LI><DIV id="open"><LI class="directlink"><A href="https://2017.igem.org/Team:Kobe">
                    HOME
                </A></LI><DIV class="dropdownlink">
                PROJECT
                </DIV><UL class="submenuItems"><LI><A href="https://2017.igem.org/Team:Kobe/Background">
                        Background
                    </A></LI><LI><A href="https://2017.igem.org/Team:Kobe/Overview">
                        Overview

                    </A></LI><LI><A href="https://2017.igem.org/Team:Kobe/Design">
                        Design
                    </A></LI><LI><A href="https://2017.igem.org/Team:Kobe/Parts">
                        Parts
                    </A></LI><LI><A href="https://2017.igem.org/Team:Kobe/Experiments">
                        Experiments
                    </A></LI><LI><A href="https://2017.igem.org/Team:Kobe/RandD">
                        Results &amp; Discussions
                    </A></LI><LI><A href="https://2017.igem.org/Team:Kobe/Notebook">
                        Notebook
                    </A></LI></UL><LI class="directlink"><A href="https://2017.igem.org/Team:Kobe/Safety">
                    SAFETY
                </A></LI><DIV class="dropdownlink">
                HUMAN PRACTICE
                </DIV><UL class="submenuItems"><LI><A href="https://2017.igem.org/Team:Kobe/HP/Gold_Integrated">
                        Integrated Human Practice
                    </A></LI><LI><A href="https://2017.igem.org/Team:Kobe/HP/Silver">
                        Education and Public Engagement
                    </A></LI></UL><LI class="directlink"><A href="https://2017.igem.org/Team:Kobe/InterLab">
                    INTERLAB
                </A></LI><LI><DIV class="dropdownlink">
                    ABOUT US
                    </DIV></LI><LI><A href="https://2017.igem.org/Team:Kobe/Team">
                            Meet the Team
                        </A></LI><LI><A href="https://2017.igem.org/Team:Kobe/Attributions">
                            Attributions
                        </A></LI><LI><A href="https://2017.igem.org/Team:Kobe/Collaborations">
                            Collaborations

                        </A></LI><LI><A href="https://2017.igem.org/Team:Kobe/Acknowledgement">
                            Acknowledgements and Sponsors

                        </A></LI></DIV></UL></P><DIV class="main"><H1>Experiments</H1><P>In this page, we are going to describe the overview of the research and experiments during the iGEM activity. Please go through our Notebook for the detailed protocols.</P>Our project consists of 6 main steps.

    <OL><LI><A href="#a7">Growth test</A></LI><LI><A href="#a1">RNA extraction</A></LI><LI><A href="#a2">RNA-seq</A></LI><LI><A href="#a3">RT-qPCR</A></LI><LI><A href="#a4">Transformation</A></LI><LI><A href="#a5">Beta-galactosidase assay</A></LI></OL><H3 id="a7">1. Growth test</H3><P>
        As stated in overview, our project depends on the theanine responsive gene the bacteria have. Roughly predicted, if bacteria can assimilate L-theanine, at least one of their genes related to the assimilation of L-theanine must be strongly induced in the presence of L-theanine.
        Based on this assumption, we carried out a simple screening test to see whether the bacteria could use L-theanine as a sole nitrogen source.
    </P><P>
        We arranged 2 strains of <I>E. coli</I> and 3 strains of <I>B. subtilis</I> and cultivated them on the derivative of M9  minimum agar media (supplemented with L-theanine as a sole nitrogen source).
 </P><P>
        As a result, <I><I>B. subtilis</I></I> 168 and <I><I>B. subtilis</I></I> NCIB 3610 could grow using L-theanine as sole nitrogen source. Finally, we decided to analyze the genes expression of <I><I>B. subtilis</I></I> NCIB 3610 and identified the genes which were active only in the presence of L-theanine. (Please visit our
        <A href="https://2017.igem.org/Team:Kobe/RandD">result page</A>
        for more information)
    </P><H3 id="a1">2. RNA extraction</H3><P>
        In order to identify the &quot;theanine responsive genes&quot;, we compared the whole gene expression in <I>B. subtilis</I> NCIB 3610 depending on different nitrogen sources (L-theanine, glutamate, water, L-theanine+glutamate). Before we extracted RNA samples from the cells, we worked at developing the medium for RNA sampling. This medium was synthesized based on M9 minimum media with a little amount of yeast extract as a nitrogen source. After setting the condition, we cultivated the bacteria in each media and extracted RNA from each sample.
    </P><H4>Protocols</H4><OL><LI>
            500ml of following solution was prepared in 500 ml of Erlenmeyer flask.
            (10×M9、0.5％glucose, 0.0001M CaCl
            <SUB><SMALL>2</SMALL></SUB>, 0.002M MgSO
            <SUB><SMALL>4</SMALL></SUB>, Trace element, 0.05g /L NH
            <SUB><SMALL>4</SMALL></SUB>Cl, 0.02% Yeast Extract, water)
            Cells were  inoculated at OD
            <SUB><SMALL>600</SMALL></SUB> of 0.02. and grown at 37℃, 180 rpm.
        </LI><LI>
            When the OD reached 0.15, L-theanine (final concentration 18.7mM), glutamate (final concentration 1.1 mM), L-theanine (18.7 mM)+glutamate (1.1 mM) ,and water were added to each flask.
        </LI><LI>
            Take the medium as a reference. When the OD
            <SUB><SMALL>600</SMALL></SUB> reached 0.3, 1 ml of culture was transferred to a fresh Eppendorf tube.
        </LI><LI>
            The cells were collected by centrifugation for 10 min (4℃, 6000 rpm). mRNA was extracted according to the protocols as described in the Protocol page.
        </LI></OL><H3 id="a2">3. RNA－seq</H3><H4>What is RNA-seq?</H4><P>
        RNA-seq is a powerful technique to analyze the gene expression in different conditions (different tissues or different stage of developments).   RNA-seq uses next-generation sequence technology to read the sequences of transcripts. The reads obtained from this sequencing can then be aligned to reference genome to make a map of whole-genome transcriptome map. The number of mRNA reflects the difference of gene expression, giving us new insight about the regulation of the whole genes in the genome. </P><H4>Why RNA-seq?</H4><P>In our project, RNA-seq was used to compare the gene expression in <I>B. subtilis</I> NCIB 3610 depending on different nitrogen sources (L-theanine, glutamate, water). What we want to find out is the gene which is more induced in the presence of L-theanine than in the presence of glutamate and water.</P><P>In this RNA-seq analysis, we were tremendously helped by our instructor, Dr. Tanaka. Although we students prepared RNA samples, Dr. Tanaka generously carried out RNA-seq for us. Mapping the reads to the genome was also done by our instructors. Based on the expert's advice, we could roughly narrow down the candidate for theanine responsive genes. </P><H3 id="a3">4. Quantitative Real Time PCR (RT-qPCR)</H3><H4> What is RT-qPCR?</H4><P>RT-qPCR is a technique to measure the expression of each gene. </P><P>
        In RT-qPCR analysis, RNA (mRNA) is first transcribed into complementary DNA (cDNA) by reverse transcription. cDNA is then used as a DNA template in conventional PCR. At this point, DNA binding dye is added in the PCR mixture. This dye intercalates with any double stranded DNA, causing the fluorescence, which allows to detect the number of PCR products.
        By tracking the progress of PCR reaction through fluorescent measurement, we can accurately deduce the initial numbers of cDNA, which is proportionate to the levels of gene expression.
    </P><H4> Why RT-qPCR?</H4><P>
        After the previous RNA-seq analysis, we selected some promising candidates for theanine responsive genes. In this RT-qPCR steps, we examined the gene expression of <I>B. subtilis</I> NCIB 3610 more deeply and confirmed the RNA-seq results (RT-qPCR is quantitatively more accurate analysis than RNA-seq)         </P><H4>
            Protocols
        </H4><OL><LI>

                After mRNA samples were treated with DNaseI (sample preparation), concentration of RNA was measured using NanoVue.
                <TABLE><TBODY><TR><TH>ng/µl</TH></TR><TR><TH>sample T</TH><TD>412</TD></TR><TR><TH>sample G</TH><TD>332</TD></TR><TR><TH>sample T+G</TH><TD>232</TD></TR><TR><TH>sample W</TH><TD>610</TD></TR></TBODY></TABLE>
                (T: theanine, G: glutamate, T+G: theanine+glutamate, W: water)



            </LI><LI>
                Reverse transcription was carried out according to ReverTra Ace® qPCR RT Kit from TOYOBO.


            </LI><LI>
                qPCR was carried out according to THUNDERBIRD® SYBR® qPCR Mix.
                ThemalDice (TaKaRa) Real Time SystemⅡ was used for the analysis.
                The expression level of <I>rpsJ</I> (30S ribosomal protein S10 in <I>B. subtilis</I>) was also analyzed and as a reference gene.
                Make the serial dilution of cDNA. Prepare the mixture below.
                <TABLE><TBODY><TR><TH>Forward primer</TH><TD>0.3 µM</TD></TR><TR><TH>Reverse primer</TH><TD>0.3 µM</TD></TR><TR><TH>ThunderBird</TH><TD>10 µｌ</TD></TR><TR><TH>cDNA </TH><TD>5 µl </TD></TR><TR><TH>milliQ</TH><TD>up to 20µl</TD></TR></TBODY><TFOOT><TR><TH>total</TH><TD>20 µl</TD></TR></TFOOT></TABLE></LI></OL><P>In this step, we finally identified three genes (<I>nasA</I>, <I>amtB</I>, <I>yrbD</I>) that are strongly induced in the presence of L-theanine. After identifying those genes,  we designed our BioBrick parts as described in the design page. (BBa_K2233000, BBa_K2233001, BBa_K2233002)</P><H3 id="a4">5. Transformation</H3><P>After we identified the candidates for theanine responsive genes, we designed our BioBrick parts and transformed the <I>B. subtilis</I> strain NCIB 3610 and strain 168. But there was a problem with transformation. The problem is that strain NCIB 3610, which is a wild-type natural isolate of <I>B. subtilis</I>, has low transformation efficiency compared to <I>B. subtilis</I> strain 168, which is a laboratory strain widely used for research. So we transform the <I>B. subtilis</I> strain 168 first and then transformed the <I>B. subtilis</I> strain NCIB 3610 using the extracted recombinant 168 genomic DNA.</P><H4>Protocols</H4><OL>Cells were pre-cultured overnight on LB agar plates at 37℃. The cells were inoculated into 10 ml MDCH liquid media at OD<SUB><SMALL>600</SMALL></SUB> of 0.3. The cells were grown at 37℃,180 rpm until the OD<SUB><SMALL>600</SMALL></SUB> reached 1.5, and then the same volume (10 ml) of MD medium was added. After 1 hour of incubation at 37℃ with shaking at 180 rpm, 1.0 ml of the culture was transferred to a fresh conical tube, where 100-1000 ng of DNA was added. After 2hours of further incubation at 37℃, the cells were spread onto LB agar plates with chloramphenicol (final concentration: 5 µg/ml) and grown overnight at 37℃.</OL><H3 id="a5">6. Beta-galactosidase assay</H3><P>
        In order to measure the gene expression level of <I>nasA</I>, <I>amtB</I>, <I>yrbD</I>, we used the Beta-galactosidase assay test. The recombinant <I>B. subtilis</I> was cultured in 50 ml of the media (the same media we used for RNA-seq) and cells were collected at different time points.
        Cells are exposed to lysozyme, which breaks down cell membrane and extracts the Beta-galactosidase inside the cell. The amount of Beta-galactosidase is proportionate to the levels of target gene expression.
    </P><H4>Protocols</H4><P>
        Day1
    </P><OL><LI>Prepare the 5 ml media in each test tube.  </LI><LI>Make the serial dilution of bacteria (Take 500 µl of media and transfer to the next one) and grow the cells overnight at 37℃, 180 rpm.  </LI></OL><P>Day2 </P><OL><LI>The cells were inoculated into the media at OD<SUB><SMALL>600</SMALL></SUB> of 0.02. Start cultivation (10 ml) with vigorous aeration.</LI><LI>When the OD<SUB><SMALL>600</SMALL></SUB> reached 0.15, nitrogen source (water, L-theanine, Glutamate, theanine+Glutamate) was added to each flask.</LI><LI>The culture was taken at each time point 0h, 1h, 2h, 3h, 4h. The cells were collected by centrifugation for 10 min at 150 rpm, and stored at -20℃.</LI></OL><P>Day3</P><OL><LI>Resuspend the cells in appropriate amount of Z-buffer</LI><LI>Incubate the cells at 37℃ for 30 min</LI><LI>Spin down the cells at 15000 rpm for 5 min and collect the supernatant.</LI><LI>Dilute the supernatant 10 times. Measure the concentration of protein. Pierce™ BCA Protein Assay Kit was used to measure the concentration of protein.</LI><LI>Take another 100 µl of supernatant into another Eppendorf tube.</LI><LI>Incubate at 28℃ for 5 min.</LI><LI>Add 300 µl of 4mg/ml ONPG</LI><LI>Leave for 10 min</LI><LI>Add 600 µl of 1M Na<SUB><SMALL>2</SMALL></SUB>CO<SUB><SMALL>3</SMALL></SUB></LI><LI>Measure the OD<SUB><SMALL>420</SMALL></SUB></LI><LI>Standard curve was generated by diluting 100 mM O-Nitrophenol (in N,N-Dimethylformamide) with water.</LI></OL><P>
        Finally, we did X-gal plates test. First, we prepared the following X-gal agar plates(10×M9 salts、0.5％glucose, 0.0001M CaCl<SUB><SMALL>2</SMALL></SUB>, 0.002M MgSO<SUB><SMALL>4</SMALL></SUB>, Trace element, 0.05g /L NH<SUB><SMALL>4</SMALL></SUB>Cl, 0.02% Yeast Extract, 1.5% agar, 0.016% X-gal, water) and covered the top of the plates with semi-solid agar mixed with pre-culture cell media at the OD<SUB><SMALL>600</SMALL></SUB> of around 0.4. Penicillin cups were put on the top of the agar overlay, and appropriate amount of nitrogen source samples were poured into the cup.
</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>