<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Calgary_Notebook_Calendar_OriT"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Calgary/Notebook/Calendar/OriT">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Calgary/Notebook/Calendar/OriT&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Notebook/Calendar/OriT&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Notebook/Calendar/OriT&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Calgary/Notebook/Calendar/OriT" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Calgary/Notebook/Calendar/OriT</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="headercontainer"><DIV id="headerbar"><UL id="topnav"><LI id="homebutton"><A class="navtext" href="https://2011.igem.org/Team:Calgary">Home</A></LI><LI id="teambutton"><A class="navtext" href="https://2011.igem.org/Team:Calgary/Team">Team</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Team/Biographies">Biographies</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Team/Facilitators">Facilitators</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Team/About_The_University">The University</A></LI><LI><A href="http://ung.igem.org/Team.cgi?id=640">Official Profile</A></LI></UL><LI id="projectbutton"><A class="navtext" href="https://2011.igem.org/Team:Calgary/Project">Project</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Promoter">Promoter</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Reporter">Reporter</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Chassis">Chassis</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Reporter/Prototype">Prototype</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/DataPage">Data Page</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Acomplishments">Accomplishments</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Future_Directions">Future Directions</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/References">References</A></LI><LI id="partsbutton"><A class="navtext" href="https://2011.igem.org/Team:Calgary/Parts">Parts</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Parts">Parts Submitted</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Attributions">Attributions</A></LI><LI id="notebookbutton"><A class="navtext" href="https://2011.igem.org/Team:Calgary/Notebook">Notebook</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/PostRegionals">Post-Regionals</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Calendar">Journal</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Safety">Safety</A></LI><LI id="outreachbutton"><A class="navtext" href="https://2011.igem.org/Team:Calgary/Outreach">Outreach</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Outreach/Human">Human Practices</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Conferences">Conferences</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Outreach/Follow">Follow Us!</A></LI><LI id="sponsorsbutton"><A class="navtext" href="https://2011.igem.org/Team:Calgary/Sponsors">Sponsors</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Sponsors">Sponsors</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Sponsors/Acknowledgements">Acknowledgements</A></LI><LI id="igembutton"><A class="navtext" href="https://2011.igem.org" target="_blank">iGEM</A></LI></DIV></DIV><DIV id="sidebar"><H2 id="sidetitle">Notebook</H2><DIV id="sidetext"><H4><A href="https://2011.igem.org/Team:Calgary/Notebook/PostRegionals">Post-Regionals</A></H4><H4><A href="https://2011.igem.org/Team:Calgary/Notebook/Calendar">Journal</A></H4><UL><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Calendar/Week1">Defining Objectives</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Calendar/Week2">Techniques Workshop</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Calendar/Binf">Bioinformatics &amp; qRT-PCR</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Calendar/ChipSEQ">Sensory Element Fishing</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Calendar/Engineers">Electrochem - CPR Oxidation</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Calendar/LiveCell">Electrochem - Reporter Gene</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Calendar/OriT"><I>Pseudomonas</I> Tools</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Calendar/Microalgae">Microalgae Tools</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Journals/NA_viability">NA Viability Assays</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Journals/OLD_ENTRIES">May - July Group Journal </A></LI></UL><H4><A href="https://2011.igem.org/Team:Calgary/Notebook/Protocols">Protocols</A></H4><H4><A href="https://2011.igem.org/Team:Calgary/Safety">Safety</A></H4></DIV></DIV><H1 id="pagetitle">Genomic Tools for <I>Pseudomonas</I></H1><DIV id="bodycontainer"><DIV id="bodytext"><H2>PROJECT/SUBPROJECT FOCUS </H2><P>Making a genetic tool/plasmid to make a genomic and plasmid library of P. putida.</P><H3> Project Participants</H3><P> Peter Qi, Saeed Qureshi</P><H3>July 7-14th, 2011</H3><H3>Author: Peter Qi</H3><P>Miniprepped the B0015 plasmid (iGEM) from overnight cultures in LB/AK. Plasmid yield is acceptable, with relatively high purity in most samples. The plasmids are cut with EcoRI and blunted ended with T4 polymerase. A streak plate (Gent15) of the pUCP30T backbone was acquired from the Schryvers lab glycerol stocks, and overnight cultures were made to collect the backbone plasmid. The yield was high with good purity. pUCP30T plasmid DNA was digested with SfiI with recommended reaction conditions, and frozen/stored at -20C, to stop but not inactivate the SfiI enzyme. Further cloning steps to be continued in the next week.</P><H3>July 14-21st, 2011</H3><H3>Author: Peter Qi</H3><P>Cleaned up SfiI restriction digest reaction with PCR purification kit, and measured DNA concentration on Nanodrop, the concentration was low, but the purity was relatively high for most samples. The product is the pUC30T plasmid cut and blunt-ended (still 1 linear piece of DNA).</P><P>Blunt ended the products of B0015 (iGEM plasmid) EcoRI digest, and blunt-ended by T4 reaction, the products subsequently purified by gel extraction. Concentration measured as before, but is too low and impure. However, it is acceptable to use these samples to do the ligation step with product from above, introducing the T7 double terminator in B0015 into the pUCP30T backbone. </P><P>Plasmid DNA was prepared from overnight P. putida cultures, two preps were made. A genomic prep was made as well. The plasmid prep samples contained high nucleotide concentration, but is likely contaminated with RNA, confirmed by RNase treatment results. There is also a large amount of undissolved material, which is likely RNA. The genomic prep yielded little DNA. Once all samples are properly solubilized, they will all be treated with RNase, and concentration measured. The DNA prepared here is for the genomic and plasmid promoter libraries, which can start once all DNA are collected. </P><P>More plasmid DNA of the parts to be cloned will be prepped, since low DNA concentration has been an issue. The pUCP30T-ter plasmid should constructed and transformed next week.</P><H3>July 21st-28th</H3><H3>Author: Peter Qi</H3><P> Performed 10 minipreps each for the B0015 and pUCP30T plasmid, and DNA/nucleotide concentration of each prep is measured on Nanodrop. All B0015 preps are digested with EcoRI for 2 hours, while all pUCP30T preps are digested with SfiI for 2 hours. The pUCP30T digest is purified using a PCR purification kit, with modification to the original protocol (see list of protocols). All digests are blunt-ended by Invitrogen T4 DNA polymerase, and purified again using PCR purification. The purified blunt-end fragments are all digested in PstI for 2 hours, and run on a 1% agarose gel for electrophoresis. The proper fragments from pUCP30T (~4.1kb) is isolated using Qiagen gel extraction kit. The fragment from the B0015 plasmid (129kb) was lost due to over elution. The B0015 fragment will be recovered again next week using a 1.5% agarose gel, and a ladder appropriate for its size. Gel extraction protocols will also be refined to give higher and purer yields of cloned parts.</P><H3> July 28th – Aug 5th, 2011</H3><H3>Author: Peter Qi</H3><H4>OBJECTIVES:</H4><OL><LI> To purify the B0015 part (T7 double terminator element, 129bp) from restriction digests.</LI><LI>Ensure that the B0015 terminator part is flanked by one sticky end made by PstI, and a blunt ended with a poly-A tail.</LI><LI>To purify the pUCT30T plasmid (-lacZ a) from restriction digests.</LI></OL><P><B>REFERENCE:</B> July 28th – Aug 5th, 2011</P><H4>SUMMARY OF ACTIONS TAKEN</H4><P>For objectives #1 and #2:</P><OL><LI>Restriction digest of 10µg of B0015 plasmid DNA by EcoRI (incubation at 37C for 2 hrs, inactivate enzyme by incubation at 80C for 20min).</LI><LI>Blunt end the restriction digest products using NEB T4 DNA polymerase.</LI><LI> Use Qiagen PCR purification kit to remove T4 DNA polymerase and nucleotides (dNTPs)</LI><LI>Added a poly-A tail to the blunt ends.</LI></OL><P>For objective #3:</P><OL><LI>Restriction digest of 10µg of pUCP30T plasmid DNA by SfiI (incubation at 50C for 2 hrs, remove enzyme by PCR purification).</LI><LI>Blunt end the restriction digest products using NEB T4 DNA polymerase.</LI><LI>Use Qiagen PCR purification kit to remove T4 DNA polymerase and nucleotides (dNTPs)</LI><LI>Added a poly-T tail to the blunt ends.</LI></OL><H4>RESULTS:</H4><P>Purified results show 50-60% yields from PCR purifications at the end of step 3 (for both sections), or approximately 5 µg of DNA.</P><H4>INTERPRETATION</H4><P>The amount of DNA retrieved after each cloning step may be too little eventually to acquire enough DNA, especially for the B0015. Potentially, PCR amplification of the iGEM part (B0015) can provide enough DNA for cloning.</P><H3>Aug 5th – Aug 12th, 2011</H3><H3> Author: Peter Qi</H3><P><B>OBJECTIVES:</B></P><OL><LI> Finish preparing B0015 and pUCP30T (without lacZa) </LI><LI> Ligate the two fragments in objective 1. </LI></OL><B>PROJECT/SUBPROJECT:</B><P> Making a genetic tool/plasmid to make a genomic and plasmid library of P. putida.</P><P><B>REFERENCE:</B> Aug 5th – Aug 12th, 2011</P><H4> SUMMARY OF ACTIONS TAKEN</H4><P>For objective #1:</P><OL><LI>Antarctic phosphatase treated the vector/pUCP30T (see protocol list for procedure)</LI></OL><P>For objective #2:</P><OL><LI>Set up 2 sets of ligation reactions, each with 3 different insert:backbone ratios (3:1, 1:1, 3:1) for overnight reactions.</LI><LI>Ligations were transformed into DH5a competent cells found in BHSc freezer (-80C) from 2005.</LI><LI>Plated transformations on LB+Gent15 plates with IPTG (100uL at 0.1M) and XGal (50uL at 10mg/mL).</LI></OL><H4>RESULTS:</H4><P>Transformations of the ligations showed a natural logarithm function of growth overnight, only white colony/growth is observed, no </P><H4>INTERPRETATION</H4><P>The transformation of the ligation mixture is likely not successful, because the gentamicin plates may not function properly, or the competent cells are not the right genotype or have lost competency. The gentamicin plates are tested with E. coli strains with amp resistance but not gentamicin resistance, and also Pseudomonas putida LD1 (not resistant to gentamicin), and both showed slight growth overnight, so new gent plates are needed. The competent cells can be tested by performing the appropriate controls (transforming plasmids with LacZ and other resistance markers).</P><H3>Aug 12th – Aug 19th, 2011</H3><H3> Author: Peter Qi</H3><P><B>OBJECTIVES:</B> Prepare the LacZ part for the next step in the cloning procedure (to insert in to the pUCP30T vector without lacZa and B0015/double terminator inserted). </P><P><B>PROJECT/SUBPROJECT:</B> Making a genetic tool/plasmid to make a genomic &amp; plasmid library of P. putida.</P><P><B>REFERENCE:</B> Aug 12th – Aug 19th, 2011</P><H4>SUMMARY OF ACTIONS TAKEN</H4><OL><LI>Plated cultures of miniCTX2-pfad-LacZ from Tung’s group on LB+Tet15 plates.</LI><LI>Made overnight cultures of colonies on the plate in LB+Tet15.</LI><LI>	Plasmid miniprep of the overnight cultures.</LI><LI>	NcoI digest for 2 hours (60uL rxns)</LI><LI>	Gel extraction of the ~7kb fragment using phenol/alcohol/alcohol extraction (see protocol list for procedure)</LI></OL><H4>RESULTS:</H4><P>Little DNA was isolated from the extraction process, shown by Nanodrop, also, the sample was unpure, likely contaminated from the reagents in the extraction in step 5)</P><H4>INTERPRETATION</H4><P>The method of gel extraction is likely not very effective, and a different approach may be needed to provide better results. Better yields are needed for better cloning results.</P><H3>Monday August 15, 2011 – Friday August 19, 2011</H3><H4> Author: Saeed Qureshi </H4><H4>OBJECTIVE</H4><P> Designing a plasmid which contains ORI-T/R, gentamycin resistance and a double terminator flowed by site where genomic DNA from pseudomonas fluorescence and putida can be inserted, followed by a specific LacZ reporter.</P><P><B>REFERENCE:</B> See Peter’s Journal entries from the previous weeks for some of the background on what is being done this week. </P><H4> SUMMARY OF ACTIONS TAKEN </H4><OL><LI>The gentamicin agar plates in storage were no longer potent thus a new stock of gentamicin plates were made, as well as Kan and Amp plates.</LI><LI>I PCR amplified the part B0015 (double terminator). Then ran a gel of some of the product (Figure 1).</LI><LI>The PCR products were then purified (PCR purification).</LI><LI>I restriction digested the amplified parts with EcoR1.</LI><LI>The digested parts were then modified using T4 Polymerase to create “blunt ends”, followed by another purification to isolate the DNA.</LI></OL><H4>Additional Activities Performed</H4><P>Also, I had Emily check up on the parts sent for sequencing and his preliminary analysis indicated that the parts did not actually contain he promoter region, so there was an error in the procedure somewhere. However there is no time to go back and try to repeat this and this portion of the project must be abandoned.</P><H4>RESULTS:</H4><P><B>Figure 1.</B> B0015 (151B) and B0015 (152B) both show up in the 300-400 bp range. The positive control does not show up in the gel.</P><H4>INTERPRETATION</H4><P>Figure 1 shows bands for B0015 which confirm that the part was amplified and showed up in the appropriate size range. The failure of the (+) control to show up on the gel is likely because the during the PCR the annealing time was too short for the 800bp part, this was because times were reduced in order to obtain the PCR products faster. Though there was enough time for the actual desired product to be amplified.</P><H3>Aug 19th – Aug 24th, 2011</H3><H3>Author: Peter Qi</H3><H4>OBJECTIVES</H4><OL><LI> Repeat cloning steps summarized July 28th to Aug 5th for another ligation reaction with Saeed.</LI><LI>Obtain more DNA for the second cloning step referred to in previous entry.</LI></OL><P><B>REFERENCE:</B> Aug 19th – Aug 24th, 2011</P><H4>Summary of Actions Taken</H4><P>For objectives #1: See entry for Aug 5th-Aug 12th.</P><H4>RESULTS:</H4><P>After transformation using competent DH5a cells from Maggie’s lab collection on Gent20 plates (new) with IPTG and XGal (concentrations as before), numerous colonies grew overnight. Most colonies in the middle of the plates are blue, while the colonies on the border of the plate are slight blue or colorless. Repeat transformation using Top10 competent cells on IPTG and XGal gent20 plates give better results (i.e. white colonies in the middle of the plate).</P><H4>INTERPRETATION</H4><P>The competent cells from Maggie’s lab likely do not have the right genotype (i.e. they have a lacZ gene, rather than the M15 genotype, which does not have a lacZ), which explains why most colonies are blue. The colonies on the border are not white because the IPTG and XGal was not spread evenly on the border. White colonies in the repeated transformations using Top10 can be cultured overnight to test for the right genotype (by restriction digest). Saeed checked the colonies, but none seemed to have the right genotype. A new strategy will be developed by Saeed with Dr. Mayi and Maggie to ensure faster and more accurate cloning steps for the next week.</P><H3>August 22nd-26th, 2011</H3><H4> Author: Saeed Qureshi</H4><H4>OBJECTIVE</H4><P> Designing a plasmid which contains ORI-T/R, gentamicin resistance and a double terminator flowed by site where genomic DNA from Pseudomonas Fluorescence and Putida can be inserted, followed by a specific LacZ reporter.</P><P><B>REFERENCE:</B> See Saeed's previous journal for the week of August 15-19.</P><H4> Summary of Actions Taken</H4><OL><LI>A Poly A tail was added to the insert fragment.</LI><LI>This was then cut with Pst1 and run on a gel (Figure 1).</LI><LI>The Vector (plasmid was also prepared for insertion of the plasmid by cutting with SfiI</LI><LI>It was then blunt ended using T4 polymerase followed by a PCR cleanup and running a sample on a gel (Figure 1)</LI><LI>A poly T tail was added followed by a PCR cleanup.</LI><LI>It was then cut with Pst1 and run on a gel (Figure 2).</LI><LI>The vector tubes were then incubated with Antarctic phosphate.</LI><LI>The vector and Insert were ligated together for either an hour or overnight, and then this sample was used to transform  DH5 alpha and TOP10 competent cells.</LI><LI>I tried selecting for white colonies (successfully transformed) and culturing them, as they would lack the LacZ gene necessary for blue color on an IPTG/X-Gal plate.</LI><LI>These cultures were then miniprepped with the intention of performing restriction digests to ascertain the identity of the plasmid used to transform the bacteria.</LI><LI>In preparation for the next step of the process strains carrying the specific LacZ gene (mini CTX 2) were cultured and then miniprepped.</LI><LI>In case the process had to be repeated some of the insert was again amplified by PCR and then purified (Figure 2).</LI><LI>To see whether any of the samples of the plasmid (vector) were impure, samples of all of these were run on a gel (Figure 3)
</LI></OL><H4>Additional Activities Performed</H4><OL><LI>I transformed TOP10 cells using the part I732901 which was an IPTG inducible LacZ reporter.</LI><LI>The transformed bacteria was then cultured and miniprepped.</LI></OL><H4>RESULTS</H4><P><B>Figure 1.</B> Faint bands/smears can be seen in the 100bp range for the 151 and 152 working product (possibly due to the low concentration and small size). There appears to be two distinct bands for the 303 and 304 samples cut with SfiI but these haven’t been able separate sufficiently.</P><P><B>Figure 2.</B>The PCR amplified products 151 and 152 show up in the 400-500 bp range. The 303 and 304 working product clearly displays distinct bands as opposed to smears proving that they have been digested.</P><P><B>Figure 1.</B>No bands can be observed for the PCR reactions. Very faint bands can be observed for the plasmid samples 301-308.</P><H4>INTERPRETATION</H4><P>Observations from Figure 1 suggest that the process is going well but that significant quantity of the insert may have been lost along the way (which is expected) given the number of PCR clean ups that have taken place.</P><P>Figure 2 suggests that the both 151 and 152 were successfully amplified and the single distinct band observed for lanes 6 and 8 suggest that samples 303 and 304 were successfully restriction digested, though not much more can be determined. </P><P>Figure 3 indicates that the PCR was not successful perhaps because the buffer concentration was wrong, also a different Taq was used than normal. This will have to be looked into. And faint bands can be seen for the plasmid DNA tubes but only a single band can be seen suggesting that they are pure.</P><H3>Aug 25th – Sep 1st, 2011</H3><P>Peter was on his week of vacation.</P><H3>Monday August 29, 2011 – Friday September 2, 2011</H3><H4> Author: Saeed</H4><H4>OBJECTIVE</H4><P> Designing a plasmid which contains ORI-T/R, gentamicin resistance and a double terminator flowed by site where genomic DNA from Pseudomonas Fluorescence and Putida can be inserted, followed by a specific LacZ reporter.</P><P><B>REFERENCE:</B> See the previous journal for the week of August 22-26.</P><H4> Summary of Actions Taken</H4><OL><LI>Ran restriction digests of the cultured I732901 transformed bacteria as well as the bacteria transformed with the constructs, and ran them on a gel (Figure 1).</LI><LI>Designed a new approach to the main plasmid construction technique and ordered primers to accomplish this. </LI><LI>Basically this new technique would involve adding new cut sites into amplified fragments, thereby making the procedure more efficient and shorter as previous attempts had failed.</LI><LI>Cleaned the lab.</LI></OL><H4>Additional Activities Performed</H4><OL><LI>I transformed TOP10 cells using the part I732019 which is a LacZ reporter fused to RBS.</LI><LI>The transformed bacteria were then cultured and miniprepped and the identity confirmed by restriction digests (Figure 2).</LI></OL><H4>RESULTS</H4><P><B>Figure 1.</B>  The gel is of poor quality due to breakage, two faint barely separable bands can be observed for lanes 2-11. Two distinct bands observed at lanes 12 and 13.</P><P><B>Figure 2.</B> All lanes show either the digested or undigested I732019 part. *Note the legend should say should say I732019 not I732109.</P><H4>INTERPRETATION</H4><P>Figure 1 illustrates that the identity of the pUC plasmids cannot be ascertained as it is very unclear and some lanes show two bands while others only show one. However since three bands cannot be seen the construction/ligation/transformation was not successful and must be repeated. But the identity of the I732901 samples has been confirmed.</P><P>Figure 2 illustrates that the I732019 was successfully digested and therefore the identity has been confirmed. Both the insert and the vector are of similar lengths so they overlap.</P><H3>Sep 2nd, 8th, 9th, 2011</H3><H3> Author: Peter Qi</H3><H4>OBJECTIVES:</H4><OL><LI>Choose additional candidate genes for RT-PCR experiments (with NA treatment) from Pseudomonas.com searches.</LI><LI>Help Saeed with assays of E. coli strains with IPTG-LacZ inserts with chlorophenol red-ß-D-galactopyranoside.</LI><LI>Design RT-PCR experiments to check validity of primers, and design a new primer for the housekeeping gene (rpoD)</LI></OL><H4>SUMMARY OF ACTIONS TAKEN</H4><P>For objective #1:</P><OL><LI>Went over list of gene hits with Stephen to choose genes that maybe relevant for NA-response genes/promoters, by doing internet searches on gene function etc.</LI></OL><P>For objective #2:</P><OL><LI>Cultures of E. coli (with IPTG+LacZ plasmid) exposed to IPTG and CPRG were incubated at 37C.</LI><LI>Every 30 minutes, a culture was spun down to remove the bacteria, and 200uL of the supernatant was measured in a plate reader to measure absorbance.</LI></OL><P>For objective #3:</P><OL><LI>Literature searches were conducted to search for housekeeping genes in Pseudomonas.</LI></OL><H4>RESULTS:</H4><UL><LI>Objective 1: Found one gene that is relevant, named as a dioxygenase, and showed homology to ring-cleaving proteins</LI><LI>Objective 2: See Saeed’s records for results.</LI><LI>Objective 3: Found 1 housekeeping gene commonly used for Pseudomonas RT-PCR, rpoD, a sigma factor.</LI></UL><H4>INTERPRETATION</H4><P>The primers of the gene found in objective one needs to have primers designed to be included in RT-PCR experiment. The rpoD primer needs to be ordered.</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Calgary/Notebook/Calendar/OriT">http://2011.igem.org/Team:Calgary/Notebook/Calendar/OriT</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Calgary/Notebook/Calendar/OriT" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Calgary/Notebook/Calendar/OriT" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Calgary/Notebook/Calendar/OriT&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Calgary/Notebook/Calendar/OriT&amp;oldid=203770" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>