"
Page
Discussion
View source
History
teams
Log in
 
Team:Calgary/Project/Project/Chassis/Microalgae
From 2011.igem.org
Home
Team
Biographies
Facilitators
The University
Official Profile
Project
Overview
Promoter
Reporter
Chassis
Prototype
Data Page
Accomplishments
Future Directions
References
Parts
Parts Submitted
Attributions
Notebook
Post-Regionals
Journal
Protocols
Safety
Outreach
Human Practices
Conferences
Follow Us!
Sponsors
Sponsors
Acknowledgements
iGEM
The Project
Overview
Promoter Project
Sensory Element Fishing
Bioinformatics and RT-PCR
Reporter Project
Preliminary Testing
Characterization
Optimization
Chassis Project
Microalgae Tools
Pseudomonas Tools
Prototype
Accomplishments
Data Page
Future Directions
References
A Chassis Using Native Tailings Pond Species
Microalgae Tools
In order to be able to work efficiently with microalgae, protocols for various procedures were needed. We wanted to eventually use our microalgae strain to construct a promoter library out of nuclear and chloroplast DNA. As such, transformation as well as nuclear DNA and chloroplast extraction protocols were necessary. Unfortunately, our strain of microalgae, Dunaliella tertiolecta, does not have well established transformation protocols defined in the literature, and as such time was spent optimizing literature protocols. All microalgae protocols we used can be found in our protocol section.
Transformation
An appropriate reporter also needed to be found. A reporter construct containing GFP downstream of a Nopaline Synthase (NOS) promoter was first transformed. Autofluorescence levels were found to be too high to detect fluorescence above the background, making this a poor choice for a reporter. A GUS reporter commonly used in plants, and available in the registry ( BBa_K330002 ) was attempted.  GUS (beta-glucuronidase) is an enzyme from E. coli, that is frequently used as a plant reporter. A construct containing the GUS reporter gene, downstream of the NOS reporter was successfully transformed (figure 1.)
Figure 1. Transformation of Dunaliella tertiolecta with a GUS containing plasmid with a NOS promoter.  Expression was measured at 500 nm and bright field view.  Resulting images were merged showing successful transformation of the microalgae plasmid.
New Microalgae Parts
Looking through the registry, our team recognized that for future teams to use microalgae effectively, there were several parts that would be required.  The registry contains a few appropriate selectable markers (kanamycin and gentamycin), a strong constitutive 35S CaMV promoter (BBa_K509000), as well as a visual GUS reporter (BBa_K330002).  What was needed was an inducible promoter and a more quantitative reporter gene.  To this end, we biobricked the Hsp70A/RbcS2 promoter from Chlamydomonas reinhardtii (BBa_K640001), which is inducible by light/ heat shock.  We also biobricked a gene for a codon optimized algal luciferase.  Both genes were obtained from the Clamydomonas Database, an online registry for Chlamydomonas tools.
Characterization of the Hsp70A/RbcS2 promoter (BBa_K640001)
Preliminary characterization data was collected for a microalgae construct containing both the Hsp70A/RbcS2 promoter attached to the algal luciferase. Chlamydomonas reinhardtii cells were transformed with a plasmid DNA of a construct containing the Hsp70A/RbcS2 promoter upstream of luciferase.  This was done using our optimized glass bead method.  Following transformation, cells were incubated at various temperatures (4°C, 21°C, 30°C, 37°C and 50°C) as well as in the dark for 1 hour.  After this, luminescence and OD was measured (at 595 nm) using a Victor.   When CPS was corrected for OD, luciferase activity was seen for cells incubated at 37°C and 50°C.  Although this was higher than baseline levels, is was not very significant.  This is likely due to the very small number of algal cells.  This indicates that our luciferase seems to be working, and our promoter is also working, however we cannot make any statements about how inducible it may be.  Further characterization will be needed for this part.
After Indianapolis, we worked on building a Biobrick testing construct for our algae codon-optimized Luciferase.  This included our luciferase gene downstream of our Hsp70A/RbcS2 promoter  This part was submitted to the Registry of Standard Biological parts, however, no characterization results were able to be obtained due to slow growth of our algae cultures.
Retrieved from " http://2011.igem.org/Team:Calgary/Project/Project/Chassis/Microalgae "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
