<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Calgary_Project_Promoter_Bioinformatics"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Calgary/Project/Promoter/Bioinformatics">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Calgary/Project/Promoter/Bioinformatics&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Project/Promoter/Bioinformatics&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Project/Promoter/Bioinformatics&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Calgary/Project/Promoter/Bioinformatics" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Calgary/Project/Promoter/Bioinformatics</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="headercontainer"><DIV id="headerbar"><UL id="topnav"><LI id="homebutton"><A class="navtext" href="https://2011.igem.org/Team:Calgary">Home</A></LI><LI id="teambutton"><A class="navtext" href="https://2011.igem.org/Team:Calgary/Team">Team</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Team/Biographies">Biographies</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Team/Facilitators">Facilitators</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Team/About_The_University">The University</A></LI><LI><A href="http://ung.igem.org/Team.cgi?id=640">Official Profile</A></LI></UL><LI id="projectbutton"><A class="navtext" href="https://2011.igem.org/Team:Calgary/Project">Project</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Promoter">Promoter</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Reporter">Reporter</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Chassis">Chassis</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Reporter/Prototype">Prototype</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/DataPage">Data Page</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Acomplishments">Accomplishments</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Future_Directions">Future Directions</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/References">References</A></LI><LI id="partsbutton"><A class="navtext" href="https://2011.igem.org/Team:Calgary/Parts">Parts</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Parts">Parts Submitted</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Attributions">Attributions</A></LI><LI id="notebookbutton"><A class="navtext" href="https://2011.igem.org/Team:Calgary/Notebook">Notebook</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/PostRegionals">Post-Regionals</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Calendar">Journal</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Safety">Safety</A></LI><LI id="outreachbutton"><A class="navtext" href="https://2011.igem.org/Team:Calgary/Outreach">Outreach</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Outreach/Human">Human Practices</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Conferences">Conferences</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Outreach/Follow">Follow Us!</A></LI><LI id="sponsorsbutton"><A class="navtext" href="https://2011.igem.org/Team:Calgary/Sponsors">Sponsors</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Sponsors">Sponsors</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Sponsors/Acknowledgements">Acknowledgements</A></LI><LI id="igembutton"><A class="navtext" href="https://2011.igem.org" target="_blank">iGEM</A></LI></DIV></DIV><DIV id="sidebar"><H2 id="sidetitle">The Project</H2><DIV id="sidetext"><P><UL><LI><A href="https://2011.igem.org/Team:Calgary/Project">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Promoter">Promoter Project</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/ProjectPseudomonas">Sensory Element Fishing</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Promoter/Bioinformatics">Bioinformatics and RT-PCR</A></LI></UL><LI><A href="https://2011.igem.org/Team:Calgary/Project/Reporter">Reporter Project</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Preliminary Data">Preliminary Testing</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Promoter/Final Data">Characterization</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Reporter/Optimization">Optimization</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Chassis">Chassis Project</A></LI><UL><LI><A href="https://2011.igem.org/Team:Calgary/Project/Project/Chassis/Microalgae">Microalgae Tools</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Chassis/Pseudomonas"><I>Pseudomonas</I> Tools</A></LI></UL><LI><A href="https://2011.igem.org/Team:Calgary/Project/Reporter/Prototype">Prototype</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Acomplishments">Accomplishments</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/DataPage">Data Page</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Future_Directions">Future Directions</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/References">References</A></LI></P></DIV></DIV><H1 id="pagetitle">NA-Sensitive Promoters using Bioinformatics and qRT-PCR</H1><DIV id="bodycontainer"><DIV id="bodytext"><H2>Introduction</H2><P>Several species of bacteria have been found to survive in toxic tailing pond environments.  Only recently however, have environmental researchers begun to identify the strains and species of these microorganisms.  In a study directed by Moore <I>et al.</I> 2006, two species of <I> Pseudomonas</I> (<I>putida</I> and <I>fluorescens</I>) were shown to degrade naphthenic acids in co-culture to a much greater degree than culturing each species individually. The percentage of remaining naphthenic acids in a solution after four weeks of bacteria culturing is shown in Figure 1.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:400px;"><DIV class="thumbcaption"><B>Figure 1</B> Naphthenic Acid Degradation Profile after 4 weeks of degradation by <I> Pseudomonas </I> strains LD1 and LD2. Source: Del Rio, Hadwin, Pinto, MacKinnon, &amp; Moore, 2006</DIV></DIV></DIV></DIV><P>This result would suggest that each strain of <I>Pseudomonas</I> provides a unique set of degradation machinery (likely in the form of enzymes) to complete the breakdown of naphthenic acids.  Figure 2 shows how the degradation affected the concentrations of specific naphthenic acids of varying carbon number and saturation, as measured by GC-MS.  The chart on the left of figure 2 shows the initial concentrations of each of the naphthenic acids, and the chart on the right shows the concentration of the naphthenic acids after four weeks of a LD1-LD2 co-culture.  Based on the decreased height and scale, the figure demonstrates that naphthenic acids were more or less indiscriminately degraded by the co-culture.  In terms of the development of a detection system for naphthenic acids, the indescriminate nature of this degradation would be advantageous in detecting a large group of NAs.  


</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 2</B> Naphthenic acid degradation profile after 4 weeks of degradation, shown with respect to saturation and carbon numbers. Source: Del Rio, Hadwin, Pinto, MacKinnon, &amp; Moore, 2006</DIV></DIV></DIV></DIV><P>When both figure 1 and 2 are taken together, the most likely conclusion is that the genes responsible for the degradation are spread across the two species. Furthermore, there exists a collection of essential genes in this degradation pathway which are unique to one species and not present in the other. The degradation pathway has not been experimentally confirmed, but it is believed to be a metabolic pathway which involves beta-oxidation (Moore <I>et al.</I> 2006).  The promoters upstream of these essential genes would be likely places to find a naphthenic acid promoter.  A weakness of the bioinformatics search was that LD1 and LD2 have not yet been sequenced, and therefore we assume similarity between them and corresponding strains that have been sequenced; this weakness complicated primer design but was not prohibitive to the project.</P><H2>Abstract</H2><P>Three online databases (<A href="http://darkhorse.ucsd.edu/">DarkHorse</A>, <A href="http://157.86.176.108/ProteinWorldDB/index.php">ProteinWorld DB</A>, and <A href="http://www.pseudomonas.com/">Pseudomonas.com</A>) were used to find target gene candidates <I>in silico</I>, and a fourth (<A href="http://mummer.sourceforge.net/">MUMmer</A>) was used to confirm experimental hypotheses. Using the data from these sources, we were able to construct a short list of gene candidates which were then processed through a quantitative Real-Time Polymerase Chain Reaction (qRT-PCR) to verify gene expression as a response to naphthenic acids. The qRT-PCR found that a putative Enoyl CoA Hydratase (GeneID=YP_260124), from <I>Pseudomonas fluorescens</I>, was expressed as a response to naphthenic acids. </P><H2>DarkHorse and ProteinWorld DB Results</H2><P><A href="http://darkhorse.ucsd.edu/">DarkHorse</A> is an online database for finding phylogenetically atypical proteins in various microbial species. Generally, phylogenetic atypicality can be useful for predicting the horizontal transfer of genes; for our purposes, the algorithm identified relatively unique genes for various strains of <I>Pseudomonas putida</I> and <I>fluorescens</I>.  We conjectured that genes also found in tailing pond microorganisms, or involved in the degradation of naphthenic acids and similar acids, were the most suitable for further investigation. DarkHorse uses only genomic DNA and does not include any of the super plasmids (up to 100kb) known to be carried by different strains of <I>Pseudomonas</I>.  The exclusion of these superplasmids limit the scope of our screening, but allowed us to focus on investigating the roles of the gene candidates and their source species in relevant degradation pathways.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 3</B> Diagram showing how to search for horizontally transferred genes using the DarkHorse database. Source: <A href="http://darkhorse.ucsd.edu/search.shtml" class="external free" rel="nofollow">http://darkhorse.ucsd.edu/search.shtml</A>&lt;/a&gt;.&lt;/p&gt;</DIV></DIV></DIV></DIV><P>Figure 3 shows a sample of the input used for our DarkHorse Query. The list returned by querying the database for phylogenetically atypical genes in LD1 and LD2 was 258 records long. Each record contained the GI number, strain, and gene annotation for the query species and its best phylogenetic match. To sort through the data, we started by collecting data about the best match species and the gene annotations.  Using the data about the species, we found that some of them had known degradation pathways and were naturally occurred in tailings ponds.  We were especially interested in such species when they donated a predicted transcription factor or degrative enzyme related to fatty acid degradation. Fatty acids are structurally similar to naphthenic acids and therefore may have similar promoters, or enzymes that would interact with this class of compounds. One putative short-listed gene, enoyl CoA hydratase, was selected due to its very low RPI score (0.001) - which suggests a strong likelihood that it was horizontally transferred - and also because of it was a member of a gene family involved in degrading compounds similar to naphthenic acids, such as caffeic and cinnanoic acid (Fiedler <I>et. al.</I> 2004 Journal of Applied Microbiology). A predicted Acyl-CoA-ligase with a LPI score of 0.401 also appeared to be involved in caffeic acid degradation, and was also short-listed.  To view our DarkHorse bioinformatics results as a whole please <A href="https://2011.igem.org/File:UofC2011_DarkHorse.pdf">click here</A>. </P><P>Additionally we used <A href="http://157.86.176.108/ProteinWorldDB/index.php">Protein World DB</A>, a database for finding unique genes across a set of compared species. This program was useful for identifying potential metabolic genes unique to one species, which our hypothesis suggested is the case for the desired promoter.  We used the only available <I>Pseudomonas</I> strains in their registry, <I>putida</I> KT2440 and <I>fluorescens</I> Pf-5, in our search.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 4.</B>Picture showing the layout of the ProteinWorld database website. Source: &lt;a href=&quot;<A href="http://darkhorse.ucsd.edu/search.shtml" class="external free" rel="nofollow">http://darkhorse.ucsd.edu/search.shtml</A>&quot;&gt;<A href="http://darkhorse.ucsd.edu/search.shtml/" class="external free" rel="nofollow">http://darkhorse.ucsd.edu/search.shtml/</A>&lt;/a&gt;</DIV></DIV></DIV></DIV><P>ProteinWorld DB returned 55 unique hits for both species of bacteria.  Visual inspection and classification of these hits revealed a number of transcription factors known to be involved metabolic signaling in a variety of other organisms.  In particular, two LysR based transcription factors, and a theoretical protein seemed to BLAST against other known transcription factors found in various species of bacteria.  What was interesting, was the involvement of these transcription factor hits in metabolic pathways particularly in beta-oxidation.  For this reason, we added these to a list for further examination.  A thioesterase which might also be involved in naphthenic acid degradation was also short-listed onto our list.  Please <A href="https://2011.igem.org/File:UofC2011_ProteinWorld.pdf">click here</A> for a complete list of our Protein World DB hits.</P><P>After examining both the DarkHorse and Protein World DB hits, our team was able to short-list six genes which we thought to be potentially involved in naphthenic acid detection and potentially degradation.  These genes are listed below:</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 5. </B>List of genes short-listed for qRT-PCR verification, as queried from DarkHorse and ProteinWorldDB.</DIV></DIV></DIV></DIV><H2>Quantitative RT-PCR Results</H2><P>In order to validate that our short-listed genes were involved in naphthenic acid sensing, we designed and implemented a qRT-PCR approach to analyzing changes in mRNA expression for these genes in the prescence of naphthenic acids. </P><P>Untreated cells and cells treated with 80mg/L of NAs were subjected to Qiagen RNA bacterial extraction protocol (see protocols section).  Reverse transcription PCR using random primers and Reverse Transcriptase was performed to generate cDNAs from RNA.</P><P>The cDNAs generated were then used for quantitative real time PCR (qRT-PCR).  During the PCR, we used primers which specifically bound to the transcribed region of the each gene, generating small DNA sequences later bound by  SYBR Green, a DNA binding fluorophore.  The binding of SYBR is directly proportional to the amount of DNA generated in the PCR reaction.  A fluorescent scanner in the qRT-PCR machine periodically measures the fluorescence of the reaction as the PCR proceeds. Initial concentrations are regressively and indirectly determined when the fluorescence passes a predefined intensity threshold. The threshold is known as a the concentration threshold (Ct), and the Ct value refers to the cycle at which this threshold is surpassed.  Because the amount of DNA in a PCR doubles every cycle, the Ct value can be used to determine the amount of cDNA, and therefore mRNA, produced as a result of reverse transcription and gene transcription respectively. As a further control, differences in transcription of a tested gene are normalized to the transcription of a control gene expressed at the same rate in both treated and untreated samples.  Control genes are generally transcripts that do not vary significantly with different treatments.  If the Ct value of candidate genes differ from each other more than the control genes, then the evidence suggests that the differences in expression are a result of the treatment given to the cells. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 6</B> RT-PCR melt curve of ECH primer set with LD2 cDNA</DIV></DIV></DIV></DIV><P>Before conclusions can be drawn, it is important to determine if the primers accurately represent the changes in RNA levels of one gene exclusively.  The quality of the primer can be determined using a melt curve.  If the primers work properly, then a melt curve analysis will yield only one melting point.  In an effort to characterize the primers, RT-PCR was used with some of the previously synthesized cDNA.  The primer set ECH (Enoyl-CoA Hydratase) demonstrated that the Ct values proportionately changed with different cDNA levels of the strain LD2, as demonstrated in figure 6. The ECH primers showed a strong dissociation curve with a single peak suggesting that a single target was being synthesized. Primers used for ECH were as follows:</P><UL><LI> Forward: 5'-GGCCTGTTTTGCCGACAT-3'</LI><LI> Reverse: 5'-GCAGGTAACGCCGAACGA-3'.</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 7</B> RT-PCR melt curve of ECH primer set with LD2 cDNA</DIV></DIV></DIV></DIV><P>Primers for the Polymerase Chain Reaction (PCR) were designed with the following guidelines in mind:</P><UL><LI> Melting Temperature: 70-74°C (optimally 72°C)</LI><LI> Primer Length: 22-24 (optimally 24)</LI><LI> Product Length: 150-250 base pairs (optimally 200)</LI><LI> No runs of 4 or more of the same nucleotide letter in either the forward or reverse primer.</LI><LI> The primers have a GC clamp at the end (ie. end with G, C, GC, or CG)</LI><LI> The last 5 nucleotides contain 3 or 4 GC's.</LI><LI> GC Content between 40-60% (optimally 50%)</LI><LI> All unintentional matches calculated using Primer Blast are at least 25% mismatched.</LI></UL><P>We attempted to optimize the other five genes from the short list, but unfortunately we are still in the process of characterizing their primers. Further characterization of newly synthesized ECH cDNA demonstrated the linear relationship between the amount of cDNA and the amount of mRNA.  cDNA was synthesized for all treatments (with and without NA treatment), and qRT-PCR was performed to test whether there was an increase in amplification between NA treated cells and cells not treated with NAs. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 8</B> Relative transcript levels of Enoyl CoA Hydratase normalized to 16S RNA, showing an increase in expression between control cells and those treated with Naphthenic Acids.</DIV></DIV></DIV></DIV><P> To obtain the values for Figure 8, 16S-DNA Ct values were subtracted from ECH Ct values for the same treatment.  The standard Relative Quantitation formula was then applied to the ΔCt values, in which the Ct value is an exponent of POWER2.</P><P>Based on this information, it appears that the putative Enoyl Coa Hydratase is upregulated when cells are incubated with naphthenic acids.  Further validation is required to ensure that this response is specific to a variety of conditions including general stress response, heat stress, and fatty acid degradation by regular beta-oxidation.  These control experiments have been planned however there was insufficient time before the Americas regional wiki freeze to complete this.</P><H2>Further Characterization of ECH/fad-2 Gene Expression</H2><P>In the course of pursuing a naphthenic acid inducible promoter, we decided to continue investigating the putative enoyl-CoA hydratase orthologue, fad-2.  After performing the first experiment, fad-2 appeared to be enriched 1600-fold compared to untreated control when naphthenic acids were exposed to the <I>Pseudomonas fluorescens</I> strain LD2.</P>However, upon further examination of the RT-PCR experiment, we found an error in previous calculations, showing that fad-2 transcript levels decreased upon NA exposure instead of increasing as we originally thought, and the fold difference is not as substantial.  The method currently adopted is based on Pfaffl et al., (2001), using the following website for further guidance (http://pathmicro.med.sc.edu/pcr/realtime-home.htm).  Whether transcription is activated or inactivated, we believed that the change in activity itself was worth investigating in the hopes of finding a regulatory element. <DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:270px;"><DIV class="thumbcaption"><B>Figure 9</B><P>A calculation error revealed that the enoyl-CoA hydratase orthologue fad-2 is downregulated not upregulated in response to ~80mg/L naphthenic acid treatment. (correction for Figure 8).</P></DIV></DIV></DIV></DIV><P>We examined the chromosomal region surrounding fad-2 and observed that it was likely part of an operon, with another putative operon on the opposite strand.  We reasoned that if NA consistently represses fad-2 expression, it may be transitioning expression to genes on the opposite strand.  We adopted the strategy to determine if repression of fad-2 in the presence of naphthenic acids was a consistent result, and if it was, did that mean that the opposite strand operon was being activated, thereby causing the repression of the fad-2 operon?  Due to time constraints, we have not been able to characterize amplification of genes on the opposite strand and went straight to synthesis of the putative promoter to position it before a response element and test if it was inducible.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 10</B><P>Chromosomal map of fad-2 and neighboring genes and intergenic regions. Source: <A href="http://www.pseudomonas.com" class="external free" rel="nofollow">http://www.pseudomonas.com</A></P></DIV></DIV></DIV></DIV><P><B>Is fad-2 consistently repressed by naphthenic acids?</B></P><P>Repeats of these experiments demonstrated that similar to the initial experiment, NA treated samples have a lower enrichment of fad-2 transcript than the control sample, however there was only a 2-fold expression difference instead of the original 6.  The reason for this is possibly because more recent experiments were performed using different conditions from the original.  In both experiments, cells were treated with 80mg/L of NA and other stimuli between 6 and 10 hours and were harvested while in exponential growth phase.  The original experiment used cultures that were treated for a longer time course and may have been in stationary phase, which potentially resulted in a greater amount of fad-2 mRNA transcripts.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:364px;"><DIV class="thumbcaption"><B>Figure 11</B><P>The regulation of Fad-2 by naphthenic acid treatment appears to be consistent, although much less than what was found in the first experiment.  Decanoic acid, a straight chained fatty acid appears to increase the amount of Fad-2 transcript. Expression levels are relative to 16s rDNA levels, and normalized to untreated control (relative quantiation=1).</P></DIV></DIV></DIV></DIV><P>To determine if the regulation we were seeing is specific to naphthenic acids, we performed the experiment with other chemical treatments: decanoic acid, a straight chained fatty acid; cinnamic acid, a naphthenic acid with an aromatic ring followed by an unsaturated fatty acid tail, which is a known substrate for enoyl CoA hydratase, and is thus used as a positive control treatment; and hydrogen peroxide, an oxidizing agent that produces a general cellular stress response. Chemical structures of the chemicals used are shown in the figure below.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 12</B><P><I>Pseudomonas LD2</I> cultures were exposed to cinnamic acid, decanoic acid, naphthenic acid (cyclohexane pentanoic acid) or hydrogen peroxide (not shown) prior to total RNA isolation</P></DIV></DIV></DIV></DIV><P>In this experiment, the decrease in Fad-2 due to naphthenic acids was not substantial.  Cinnamic acid has been previously described as being modified by enoyl-CoA hydratase.  In cells treated with cinnamic acid, a 1.6 fold increase in fad-2 transcript was observed.  Decanoic acid exposure did not appear to increase fad-2 transcript in this experiment. The lack of response to decanoic acid indicates that fad-2 expression probably decreases specifically to NA exposure, and fatty acids, which is a similar class of hydrophobic compounds with a carboxylic acid group, cannot elicit the same change.  Hydrogen peroxide did not alter Fad-2 transcript levels either, suggesting that fad-2 or ECH is likely not a general stress response gene.  There is a continual loss of amplification as we have performed these experiments.  Coincident with this is the appearance of a small second peak in the melt curve that did not appear during primer optimization studies (<A href="https://2011.igem.org/Team:Calgary/Notebook/Calendar/Binf#RTPCR">see figure here</A>).  These experiments need to be repeated when new primers are ordered.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:364px;"><DIV class="thumbcaption"><B>Figure 13</B><P>Relative Quantitation of Fad-2 Transcript upon exposure to naphthenic acids (dissolved in water or ethanol), decanoic acid, cinnamic acid, and H2O2 after normalization to control (relative quantiation=1).</P></DIV></DIV></DIV></DIV><B>Future experiments in characterization of NA response:</B><P>Future characterization experiments would explore enrichment of opposite strand transcripts of the fad-2 operon.  Although they are all putative proteins, BLINK analysis (Source: NCBI) demonstrated that the first 2 predicted ORFs encode a lipoprotein and a choloyl glycine hydratase.  These proteins may be altering the composition of the plasma membrane in response to the naphthenic acid surfactant properties.</P><P>We would also continue to modify the culture conditions for NA treated cells.  In these experiments, we did not optimize the use of minimal media, which is more suited to experiments for RNA extraction, compared to LB. </P><P>Lastly, there were other candidate genes to investigate by bioinformatics screen that we were unsuccessful in developing PCR primers for.  We are interested in sequencing the environmental isolate strains LD1 and LD2 to refine our screen and perhaps be able to design better primers for this line of investigation. </P><H2>Cloning putative naphthenic acid responsive promoter elements</H2><P>According to the chromosomal map (Figure 10), fad-2 (gene coding for ECH) appear to be in a gene cluster or operon, where transcription of the genes in the operon would be theoretically controlled by a single promoter element upstream of the operon. Three intergenic regions in the vicinity of the genes were explored for possible promoter sequences that are responsive to naphthenic acid exposure. First, in the chromosomal map, the fad-2 gene is read from right to left, and thus the intergenic region (Tax ID: 220664) (Source: http://www.pseudomonas.com) on the right of fad-2 in the chromosomal map is directly upstream of fad-2. The sequence (178bp) of this region (in the right direction) is :</P><CENTER><P>GGTGCTGATTCCTGTTTTATTCGGCCGAGACGGCGC</P><P>GGGTCTCTGAACTCATACCAGTCTTTTTCCTTGTCA</P><P>AGAAAGCCCGGAGTGACCTGAAAATCGCCATAGCCT</P><P>CACTTTTAGCACTGCCTATACTGGATTGCGCAGCCG</P><P>TTTCACCGCAGGACACACATAAACAGGAGCGTTT</P></CENTER><P>The second promoter candidate is further upstream of the fad-2 gene, and is the next intergenic region available. Similar to the first candidate, a promoter region may be present in this region, which controls a cluster of related genes or a operon. The sequence (329bp) of this intergenic region is:</P><CENTER><P>TGTTGTTCCTCACGCATAGGGCGAAGTGTTTCGCCGGGGTAATGGCCATG</P><P>ATCGAGAGGTCATGAAAACGGCAGGACGTCCCTTTCCAGCCAAGCTTCAG</P><P>GTCGGTTGCATTTCACGGGAGCGGCCAAGGCGCCAAGGAAAGTCCTCCAC</P><P>CCTGGGCAAAATTCAAAAACAGCAATGAAATATTTTTTTGAATCAAATAG</P><P>CTGGCAATCTCCGATCGGTCAAGGCTCCCCATCGGCCTTCAACCCTGTGG</P><P>CAAGCCTGCCGGGGGGCCTGTTAGGCTAACCCGCCCACCGCGTCGCCGCG</P><P>GTCTCTACAGCTCTTCGCGGAACCGCACC </P></CENTER><P>The third candidate for the NA-responsive promoter is downstream of fad-2 gene, and is more likely to be a promoter that is selectively activated in the presence of naphthenic acids. Since fad-2 is down-regulated with NA-exposure, and other genes in the cluster, including PFL_3017 (hypothetical protein) and PFL_3019 (choloylglycine hydrolase) (Source: http://www.pseudomonas.com), are transcribed in the opposite direction as fad-2, these genes (PFL_3017 and PFL_3017) are likely up-regulated in response to NAs. Thus, this intergenic region can act as a promoter that can be activated in response to NAs. The sequence (350bp) of this intergenic region (Tax ID: 220664) (Source: http://www.pseudomonas.com) is:</P><CENTER><P>TCACTGATTTTGATCAAGGCACAGCTTCAGTTTGCTTTGGATACCCAGGA</P><P>ATCACTGAGAAATACAGTATTGTCAGTCCATCTCCTCAGGATGACCTGGT</P><P>AGTTTTGGAGCTGGCTGGCAAACCGATTTTCAGACTTTGGCAGCGAGATT</P><P>CGGATGATTTGAGCATTGGTGCACCGGTTACGGTGAGCCAAGGCGAATTC</P><P>GACCGGAGTCAGGGCCGTGTTACCCGACGAACCTGGTCATAGATGGAGGG</P><P>CCATGAATGCTGGTCTCTTCGGCTTCCTATGTATTCAGATATCTTATAAC</P><P>TAAACTAAATGAATATTTAGTTTAGTTATATTGGATATCAGGAGTTACCC</P></CENTER><P>We submitted the first candidate promoter to the Parts Registry (BBa_K640008) as a stand-alone part, as well as a composite part (BBa_K640006) with LacZ used for electrochemical assays, as well as oriT and ori1600 for the transfer of the reporter system into <I>Pseudomonas spp.</I>. We are in the process of biobricking the other two promoter elements and performing more electrochemical assays to test the reporter system in the presence of naphthenic acids. </P><H2>Appendix A: Pseudomonas.com Results</H2><P>The genes found from a subsequent bioinformatic search on Pseudomonas.com were not used in the RT-PCR, but have been included as an appendix for the iGEM community.  Pseudomonas.com is a database specializing in the annotation and collection of information related to the <I>Pseudomonas</I> genus, but it also has a comparative genome search tool which claims to allow the location of genes unique to one species and not the other. The list, which has not been completely processed yet, can be found <A href="https://static.igem.org/mediawiki/2011/2/2e/Calgary2011_Unique_Genes_from_Pseudomonas.zip"> in this .zip file</A> in both .xls and .xlsx format (the latter is recommended).  One of the most interesting genes found on the list is annotated as <A href="http://pseudomonas.com/getAnnotation.do?locusID=PP_4705">glyoxalase/bleomycin resistance protein/dioxygenase</A>, but is similar to a ring-cleaving dioxygenase also found in <I>P. putida</I>.</P><H2>Appendix B: MUMmer Findings</H2><P><A href="http://mummer.sourceforge.net/">MUMmer</A> is an open source program which can align genomes within a matter of seconds.  We used MUMmer to verify our assumption that there was a substantial degree of homology between <I>Pseudomonas putida</I> and <I>fluorescens</I>.  This assumption is important, because its corollary is that there is a limited number of genes unique to either species.  The fewer unique genes there are, the easier it is to find the right one.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 6</B><P>Figure 6 shows the alignment between <I>Pseudomonas putida</I> (x-axis) and <I>fluorescens</I> (y-axis).  Linear regions are more conserved and scattered regions are less conserved.</P></DIV></DIV></DIV></DIV><P>The graph above shows the genome alignment between <I>P. putida</I> and <I>P. fluorescens</I>, where the former is on the x-axis and the latter on the y-axis.  Regions conserved between both species form a straight line going from the bottom left corner to the top right corner; the line going from the top left corner to the bottom right corner is also a conserved region between both species, except that it is backwards.  Areas with a more scattered distribution indicate a more random, and therefore less homologous, aligned region between the two species.</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Calgary/Project/Promoter/Bioinformatics">http://2011.igem.org/Team:Calgary/Project/Promoter/Bioinformatics</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Calgary/Project/Promoter/Bioinformatics" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Calgary/Project/Promoter/Bioinformatics" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Calgary/Project/Promoter/Bioinformatics&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Calgary/Project/Promoter/Bioinformatics&amp;oldid=262549" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>