"
Page
Discussion
View source
History
teams
Log in
 
Team:Calgary/Notebook/Calendar/LiveCell
From 2011.igem.org
Home
Team
Biographies
Facilitators
The University
Official Profile
Project
Overview
Promoter
Reporter
Chassis
Prototype
Data Page
Accomplishments
Future Directions
References
Parts
Parts Submitted
Attributions
Notebook
Post-Regionals
Journal
Protocols
Safety
Outreach
Human Practices
Conferences
Follow Us!
Sponsors
Sponsors
Acknowledgements
iGEM
Notebook
Post-Regionals
Journal
Defining Objectives
Techniques Workshop
Bioinformatics & qRT-PCR
Sensory Element Fishing
Electrochem - CPR Oxidation
Electrochem - Reporter Gene
Pseudomonas Tools
Microalgae Tools
NA Viability Assays
May - July Group Journal
Protocols
Safety
Reporter Gene Project
Project Description
The purpose of this project was to identify a suitable reporter gene for the electrochemical detector, to characterize it and to make a construct that included both an easily inducible promoter such as pBAD and the reporter. Ultimately the reporter that was decided on was LacZ and the IPTG inducible promoter was decided on as opposed to pBAD for the purposes of testing the LacZ gene and characterizing it as an electrochemical reporter.
Project Participants
Saeed Qureshi
July 12-15, 2011
Author: Saeed
Tuesday, July 12, 2011
Spent the day going over all previous gels numbering the lanes and making legends. Also consolidated the images/results from the E. coli assay. Overnight cultures were set-up from the E. coli strains carrying the three constructs created the previous week.
Wednesday, July 13, 2011
Mini-prepped all of the cultures from the previous day and restriction digested them. These digests as well as some previous digests (which were used to make the constructs) were run on an agarose gel to confirm their identity.
Figure 1. Restriction digests to confirm identity of parts.
Thursday, July 14, 2011
Discussed and planned out another E. coli viability/survival assay in NAs and made some autoclaved PBS buffer in preparation for it. Also according to the results of the gel from the previous day it was decided that the constructs made previously would have to be remade perhaps using other parts. Some more of the previous gels were also labelled in preparation for being posted on the website.
July 16-22, 2011
Author: Saeed
Saeed was camping from July 15-18, and was ill on the 21st.
Tuesday, July 19, 2011
In Preparation for the E. coli viability assay that was to be performed the next day the strain carrying the I0500_B0034_I732005 insert was cultured. The purpose of the assay was to determine whether or not the E. Coli could survive in the presence of NAs and therefore whether it would be practical to use it in the final biosensor. On this particular occasion the NAs were diluted in autoclaved PBS buffer rather than ddH2O, mainly because it was theorized that the inability of the cells to survive in the previous assays was because of the hypotonicity of the water. On a different note some TOP10 cells were transformed for Emily.
Wednesday, July 20, 2011
Performed the E. coli viability assay, which took most of the day. Also I did some reading on electrochemical sensors
Friday, July 22, 2011
I went over the results of the E. coli assay from earlier in the week and concluded that our strain of E. coli (Invitrogen TOP10) was not a suitable candidate in use for the biosensor, because the E.coli could not survive sufficient levels of naphthenic acids.
July 23-29, 2011
Author: Saeed
Monday, July 25, 2011
Designed an experiment to test whether or not the cells transformed with the I0500_B0034_I732005 construct could convert CPRG into CPR in the presence of arabinose, the experiment was closely based on an experiment found in the Journal of Biotechnology (Shin, Park and Lim, 2005). I prepared a culture for use in the experiment. tried transforming  I0500 from the 2011 and 2010 registries and transformed R0010 from the 2011 registry.
Tuesday, July 26, 2011.
I ran a colony PCR on the cells transformed the previous day and on a streak plate of the cells carrying the contruct I0500_B0034_I732005.I subcultured the culture from the previous day until it reached log phase and then incubated the cells in CPRG and arabinose and monitored the response. The results of the experiment indicated that the cells were converting CPRG into CPR in the presence of arabinose. Details of the experiment are below:
I performed a test on E.coli carrying the I0500_B0034_I732005 (arabinose promoter followed by LacZ reporter) to measure the conversion of CPRG into CPR.  The experiment was not kept  buffered at a constant pH of 7 at which pH the CPR would have have very good absorbance. Near the end though the pH was 7 presumably because the E.coli would have produced their own buffer and increased the pH with it.  At different pH's though, the solution has different optical density because of the indicator (CPR) used in the experiment. So the results may not be indicative of the level of CPR production over time. The results are below
Time(hours)
Optical Density Reading
0.0
0.063
0.5
0.174
1.0
0.336
1.5
0.582
2.0
0.709
3.0
1.381
4.0
1.282
5.0
2.029
5.5
2.447
21.0
2.797
The above table shows the time in hours and the corresponding optical density when measured with light with a wavelength of 535nm.
Figure 1. OD readings measuring the production of PCR by the cleavage of CPRG by β-galactosidase expressed by E. coli over a range of time
Wednesday, July 27, 2011.
I ran a gel to determine the results from the PCR performed the previous day, it turned out that the PCR was a failure. I retried the PCR again, this time using a different recipe for the mastermix.  On a different note I sonicated some Pseudomonas cells which had been lysed to try to get the genome cut into units of 1KB, however a gel ran by Emily determined that the sonication failed to do so.
Figure 1. Nothing useful can be determined from this gel
Thursday, July 28, 2011.
I ran a gel which confirmed the success of the PCR of the previous day. R0010 was successfully transformed into the E. coli cells, however the gel did not indicate any bands in the region of 1200KB indicating that either the 2011 registry plate did actually contain I0500 DNA in the well that was supposed to contain it or that consistently we were not able to transform bacteria with I0500 from the 2011 registry plates. Also I cultured one of the bacterial colonies carrying R0010 (confirmed by PCR). Later in the day I sonicated samples of lysed E. coli and Pseudomonas for Emily.
Figure 1. Reattempt of the Previous Day's PCR.
Extra Journal Entry for August 2nd, 2011
Author: Saeed
DATE CREATED:
August 8, 2011
OBJECTIVE:
Characterize the electrochemical reporter system.
PROJECT/SUBPROJECT:
Project Electro
OBJECTIVE:
Make a standard curve for Chlorophenol red (CPR) concentrations as a function of optical density.  This curve will determine the CPR production in the July 26 experiment.
REFERENCE:
This experiment continues the one performed on July 26, 2011.  The July 26 experiment characterized the reporter.
WHAT DID YOU DO:
Dissolved and diluted samples of CPR and took optical density readings.
RESULTS
CPR Concentration(mM)
Optical Density Reading
0.25
0.021
0.50
0.033
0.75
0.070
1.00
0.129
1.25
0.198
1.50
0.230
1.75
0.304
2.00
0.348
2.25
0.371
2.50
0.444
2.75
0.473
3.00
0.507
Table 1. Optical Density (OD) readings for concentrations of CPR. The plate reader was blanked with water prior to taking readings.
Figure 1. OD readings (595nm) for different concentration of CPR.
Observation:
The color of the solutions were all much lighter and not as red as the color of the CPR containing supernatant in the experiment performed on July 26, 2011.
INTERPRETATION
The standard curve shows a linear relationship between the CPR concentration and the optical density readings of the sample.  Unfortunately, because the pH of the water samples were not measured, the actual standard curve at the pH from the July 26 experiment is likely to be different than the one measured above. The pH matters because chlorophenol red is an acid-base indicator which changes absorbency based on pH; at lower pH, the CPR turns yellow and has lower absorbency.   In future follow-ups, we will maintain a pH of 7. without a constant pH taking OD readings of CPR is not useful.
Friday July 29, 2011 – Friday August 5, 2011
Author: Saeed Qureshi
DATE CREATED August 7, 2011
OBJECTIVE Have a backup reporter system, in case there are complications with the electrochemical reporter system.
PROJECT/SUBPROJECT Fluorescent reporter system
OBJECTIVE Create IPT induced firefly luciferase and GFP reporter systems, since the I0500 constructs were not cooperating.
REFERENCE Thursday July, 28, 2011
SUMMARY OF ACTIONS TAKEN
Miniprepped the R0010 culture (from Thursday) but obtained very low yield of plasmid DNA.
Samples of R0010, E0240, pSB1AC3, K325210 were all restriction digested in preparation for making constructs. Then a gel of these was run to confirm their identity (Figure 1).
I tried restreaking some of the agar saps of I0500 from the previous year but again I was met with no success, so the agar saps were discarded.
The restriction digested samples were ligated together then used to transform TOP10 cells.
A ladder tests was done to see whether or not the ladder that was being used previously had been contaminated, and some other DNA samples were also run in the gel, namely the previously mentioned restriction digests (Figure 2).
It was decided that new TAE buffer would have to be made so I along with Emily, Patrick, and Robert made some more TAE buffer.
Colony PCR was done on the colonies of transformed cells carrying the constructs in order to determine which colonies could be used to make cultures. A gel was then run of this PCR (Figure 3)
RESULTS
Figure 1. Gel that was run in order to confirm the identity of the identity of the restriction digests. Since the ladder does not appear not much can be inferred from this gel.
Figure 2. Lanes 1 & 19 contain the invitrogen ladder, while 2, 16 and 20 contain an NEB ladder. 3-6 contain restriction digests. Nothing can be inferred from this gel.
Figure 3. Gel run using the PCR products. Lanes 2-6 are for the IPTG promoter followed by the firefly luciferase generator, while 7-11 are for the IPTG promoter followed by the GFP gene.
INTERPRETATION
Figure 1 indicated that there was something wrong with either the ladder or the buffer. Figure 2 indicated that the problem was most likely with the TAE buffer (wrong proportion of solutes or contamination), so it had to be replaced. Figure 3 indicates that the colonies chosen were successfully transformed and that plasmid they were transformed with do appear to carry the construct, in other words the IPTG induced reporter systems were successfully assembled.
Monday August 8, 2011 – Friday August 12, 2011
Author: Saeed
DATE CREATED: August 12, 2011
OBJECTIVE: Have a backup reporter system, in case there are complications with the electrochemical reporter system.
OBJECTIVE Assessing the success of the previously Created IPT induced firefly luciferase and GFP reporter systems.
REFERENCE Friday August 5, 2011
Summary of Actions Taken
Ran PCR to amplify the parts E0240 and K325210 and compare these with the constructs made last week, but the PCR failed.
A second attempt was made to run PCR to amplify the parts E0240 and K325210 and compare these with the constructs made last week by running them all on a gel (Figure 1).
The K325210 and R0010_K325210 were run again on a second gel at different settings from the first time (Figure 2).
A further attempt was made to PCR the part E0240 as well as the construct R0010_E0240 using a primer which was compatible with both plasmids. The products were then run on a gel (Figure 3).
The colonies carrying the R0010_K325210 and R0010_E0240 constructs were cultured and miniprepped, samples of these were sent for sequencing.
The colonies carrying R0010_E0240 were also used to make a streak plate.
Additional Activities Performed this Week
I tried transforming TOP10 cells with I723032 from the 2008 registry parts however days after plating on LB Agar, no growth was observed.
The ori-T/R project was described to me.
I with the help of Robert and Emily, discovered that the concentration of NAs that I had been exposing the E. coli in my three previous viability assays was in fact 1000 times higher than I had thought.
RESULTS
Figure 1. Ran the PCR products from the previous week (constructs) and some new PCR amplified parts K325210, E0240 on this gel.
Figure 2. Lanes 2-6 contain the construct R0010_K325210 while lane 7 contains K325210.
Figure 3. lanes 2-6 contain the recently PCR amplified parts, while lanes 8, 9, and 10 contain the previously amplified parts.
INTERPRETATION
Figure 1 shows bands which are not very clear or distinct but are smeared thus the difference in size between the K325210 and the R0010_K325210 are not visible and the gel has to be repeated with less DNA and running at a lower voltage. Also the primers used were actually not compatible with the plasmid carrying the E0240, so a different primers were also needed for the next PCR amplification of E0240 and this also meant that the construct R0010_E0240 also had to be amplified once further using the same primer which would be used for the E0240 PCR.
  Though Figure 2 shows clear and distinct bands no real difference in length or size between the samples designated K325210 and R0010_K325210 can be seen.
  Figure 3 shows that there is a size difference between the R0010_K325210 and K325210 and between R0010_E0240 and E0240. This further indicates that the construct was successfully made.
Retrieved from " http://2011.igem.org/Team:Calgary/Notebook/Calendar/LiveCell "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
