"
Page
Discussion
View source
History
teams
Log in
 
Team:Calgary/Notebook/Calendar/Microalgae
From 2011.igem.org
Home
Team
Biographies
Facilitators
The University
Official Profile
Project
Overview
Promoter
Reporter
Chassis
Prototype
Data Page
Accomplishments
Future Directions
References
Parts
Parts Submitted
Attributions
Notebook
Post-Regionals
Journal
Protocols
Safety
Outreach
Human Practices
Conferences
Follow Us!
Sponsors
Sponsors
Acknowledgements
iGEM
Notebook
Post-Regionals
Journal
Defining Objectives
Techniques Workshop
Bioinformatics & qRT-PCR
Sensory Element Fishing
Electrochem - CPR Oxidation
Electrochem - Reporter Gene
Pseudomonas Tools
Microalgae Tools
NA Viability Assays
May - July Group Journal
Protocols
Safety
Genomic Tools for Microalgae
Project Participants
Ellen Widdup, Chelsea Leichman, Emily Hicks
July 8-14
Author: Chelsea, Ellen, Emily
Ellen, Chelsea, and Emily met with Dr. Alcantara (Joenel) and discussed achievements made thus far on the microalgae project (successful GUS transformation of D.t., outline of potential construct to be made in agrobacterium vector) and delegated specific tasks to members in the group for the next two weeks. Ellen and Chelsea plan to isolate nuclear and chloroplastic genomic DNA of Dt and potentially Chlamydomonas- this involves settling on a protocol and making sure all reagents/equipment are in order. Emily was assigned the task of building a vector for the agro construct.
Emily finalized and ordered primers for a microalgae construct. This construct includes a methyl benzoate-sensitive promoter and a luciferase reporter gene, and will be characterized in algae as a model system. Emily also made a list of important genes in microalgae to investigate for the purposes of biobricking.
July 15-22
Author: Chelsea, Ellen
This week we obtained a culture of D.t. from the Chua lab. We attempted to isolate genomic DNA from the culture. We have been trouble-shooting and adjusting our exisitng protocol (originally created for Arabidopsis) to try to obtain the DNA. So far we have had limited success but we are attempting some more trials. Tomorrow we will know if we have been succesful. A protocol will be provided once we determine the best method. In addition, we have obtained an effective protocol for isolating chloroplast DNA from Dr. Muench. Next week we should be able to isolate the chloroplast DNA from D.t. as well as obtain a Chlamydomonas culture from which to also isolate DNA.
On July 21st, Joenell Alcantara, a professor at the University of Calgary, told us that there was a published protocol for isolating DNA already, since the genome was already sequenced.  Results next week.
July 23-28, 2011
Author: Chelsea, Ellen, Emily
This week we ran a gel of our isolated DNA to determine if our extraction protocol was successful. We got the correct sized band indicating that we do indeed have DNA. Since we have successfully isolated total DNA from the alga D.t. we can soon perform this for Chlamydomas (the model alga). We have been attempting to find a chloroplast DNA isolation protocol that will work. This DNA is needed to make the promoter library. So far we lack the equipment and/or expertise to undertake this kind of experiment. Despite this, we still plan to isolate chloroplast DNA from D.t. next week.
Emily tried amplifying microalgae luciferase for construction of a methyl benzoate reporter construct.  Attempted to amplify other components of the construct, however further optimization is still needed.  Next week, these will be attempted again.
July 29-August 4, 2011
Author: Chelsea
Objective
To isolate chloroplasts from Dunaliella tertiolecta
Summary
Isolated chloroplasts from Dt.
Reference
Holwerda, B.C., S. Jana, W.L. Crosby, Chloroplast and mitochondrial DNA variation in Hordeum vulgare and Hordeum spontaneum, Genetics 114, 1271-1291 (1986).
Protocol (adapted from reference)
Obtained subculture of Dt (~50 mL) and spun down (500 x g, 10 min, 25?C) to pellet cells out of culture media using swinging-bucket rotor.
Resuspended cells in 2 mL of isolation buffer.
Added ~.1 g 100um glass beads. Grinded mixture with mortar and pestle to break open cells on ice.
Pipetted mixture into a 2 mL eppendorf tube on ice. Let sit for 5 min. Pelleted out glass beads by centrifugation (200 rpm, 1 min).
Passed mixture through 2 layers of miracloth.
Centrifuged homogenate twice (1500 x g, 6 min, 4?C) in a swinging-bucket rotor.
Chloroplast pellets were resuspended in 16 uL resuspension buffer (15 mL sterile falcon tube)
Improvements
grind pellet with sand and mortar and pestle before adding isolation buffer
had issues with centrifugation at step 4- many cells appeared to be intermingled with the glass beads, hopefully the previous improvement would help remedy this
miracloth may not be the best for small volumes- at least 1 mL absorbed by miracloth (even after squeezed)
Reagents
Isolation buffer – 50 mM Tris-HCl (pH 8), 25 mM EDTA, 10 mM 2-mercaptoethanol, 1.25 M NaCl, 0.1% BSA, Resuspension buffer – 50 mM Tris-HCl (pH 8), 25 mM EDTA
Interpretation
Although we got a pellet at the end of the isolation, we are unsure of a way to verify the purity of our chloroplasts (i.e. is there genomic DNA contamination?), and are not sure of the success of the isolation at this time. We intend to find this out ASAP. Further, the yield was quite low and we would likely repeat the experiment with the improvements suggested above before we isolate chloroplastic DNA.
August 5-12, 2011
Author: Chelsea and Ellen
Objective:
Grow enough Dt to isolate a significant yield of chloroplasts using the protocol from last week.
Summary:
Subcultured Dt so that we have ~2L of culture by next week to re-attempt chloroplast isolation and get a more significant yield.
August 13-19, 2011
Chelsea and Ellen
Objective:
Grow enough Dunaliella tertiolecta (Dt) to isolate a significant yield of chloroplasts using the protocol from two weeks ago. Our subculture last week did not grow well enough to provide us with enough Dt.
Summary:
We made culture media (3L) for Dt, consisting of:
60 mL Guillard’s (f/2) marine water enrichment solution 50x
1.5 L artificial sea water
1.5 L ddwater
The media was mixed, pH’d to 8.0, aliquoted and autoclaved. Next, we subcultured D.t. again so that we will have ~2L of culture by next week to re-attempt chloroplast isolation and get a more significant yield.
This project ended on August 19th, as Ellen and Chelsea went on vacation. But it was resumed again for the purpose of
October 17-28, 2011
Author: Saeed Qureshi, Emily Hicks
Project Title: Microalgae Tools
Objectives:
Create a construct containing the hsp70 promoter (BBa_K640001) followed by the microalgae optimized luciferase for further characterization of the parts and for submitting it to the registry (BBa_K640000).
Summary of Activities Performed:
We digested the above parts as well as the pSB1AK3 plasmid and ran them on a gel (Figure 1). Then ligated them, and transformed the construct into TOP10 cells.
We ran two colony PCRs of the colonies on the plates carrying the transformed TOP10 cells (Figure 2, Figure 3)
We tried the above process again and ran a gel of the digests (Figure 4), before ligating them and transforming them.
Results:
Figure 1. Restriction digests of the parts that would be used to make the construct
Figure 2. Results of the PCR of colonies 1-15.
Figure 3. Results of the PCR of colonies 14-27.
Figure 4. Second restriction digest of the parts that would be used to make the construct.
Interpretation:
Both restriction digests showed a successful digestion of the parts in preparation for the ligation. However the PCR gels did not show the desired results and thus the required inserts were not present and the construction had to be reattempted.
Retrieved from " http://2011.igem.org/Team:Calgary/Notebook/Calendar/Microalgae "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
