<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Calgary_Project_ProjectPseudomonas"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Calgary/Project/ProjectPseudomonas">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Calgary/Project/ProjectPseudomonas&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Project/ProjectPseudomonas&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Project/ProjectPseudomonas&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Calgary/Project/ProjectPseudomonas" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Calgary/Project/ProjectPseudomonas</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="headercontainer"><DIV id="headerbar"><UL id="topnav"><LI id="homebutton"><A class="navtext" href="https://2011.igem.org/Team:Calgary">Home</A></LI><LI id="teambutton"><A class="navtext" href="https://2011.igem.org/Team:Calgary/Team">Team</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Team/Biographies">Biographies</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Team/Facilitators">Facilitators</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Team/About_The_University">The University</A></LI><LI><A href="http://ung.igem.org/Team.cgi?id=640">Official Profile</A></LI></UL><LI id="projectbutton"><A class="navtext" href="https://2011.igem.org/Team:Calgary/Project">Project</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Promoter">Promoter</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Reporter">Reporter</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Chassis">Chassis</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Reporter/Prototype">Prototype</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/DataPage">Data Page</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Acomplishments">Accomplishments</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Future_Directions">Future Directions</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/References">References</A></LI><LI id="partsbutton"><A class="navtext" href="https://2011.igem.org/Team:Calgary/Parts">Parts</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Parts">Parts Submitted</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Attributions">Attributions</A></LI><LI id="notebookbutton"><A class="navtext" href="https://2011.igem.org/Team:Calgary/Notebook">Notebook</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/PostRegionals">Post-Regionals</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Calendar">Journal</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Safety">Safety</A></LI><LI id="outreachbutton"><A class="navtext" href="https://2011.igem.org/Team:Calgary/Outreach">Outreach</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Outreach/Human">Human Practices</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Conferences">Conferences</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Outreach/Follow">Follow Us!</A></LI><LI id="sponsorsbutton"><A class="navtext" href="https://2011.igem.org/Team:Calgary/Sponsors">Sponsors</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Sponsors">Sponsors</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Sponsors/Acknowledgements">Acknowledgements</A></LI><LI id="igembutton"><A class="navtext" href="https://2011.igem.org" target="_blank">iGEM</A></LI></DIV></DIV><DIV id="sidebar"><H2 id="sidetitle">The Project</H2><DIV id="sidetext"><P><UL><LI><A href="https://2011.igem.org/Team:Calgary/Project">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Promoter">Promoter Project</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/ProjectPseudomonas">Sensory Element Fishing</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Promoter/Bioinformatics">Bioinformatics and RT-PCR</A></LI></UL><LI><A href="https://2011.igem.org/Team:Calgary/Project/Reporter">Reporter Project</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Preliminary Data">Preliminary Testing</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Promoter/Final Data">Characterization</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Reporter/Optimization">Optimization</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Chassis">Chassis Project</A></LI><UL><LI><A href="https://2011.igem.org/Team:Calgary/Project/Project/Chassis/Microalgae">Microalgae Tools</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Chassis/Pseudomonas"><I>Pseudomonas</I> Tools</A></LI></UL><LI><A href="https://2011.igem.org/Team:Calgary/Project/Reporter/Prototype">Prototype</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Acomplishments">Accomplishments</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/DataPage">Data Page</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Future_Directions">Future Directions</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/References">References</A></LI></P></DIV></DIV><H1 id="pagetitle">Novel Application of Biotinylation</H1><DIV id="bodycontainer"><DIV id="bodytext"><P>Our team set out to identify a novel responsive element capable of detecting and quantifying naphthenic acids (NAs) in solution. While numerous studies have begun to identify species of bacteria which can survive and/or degrade NAs, the pathways of degradation have not yet been characterized.  Therefore this goal required our team to design and implement novel methods to isolate the cellular systems of tailing ponds biological organisms which detect and potentially degrade naphthenic acids.  We developed a naphthenic acid biotinylation/immunoprecipitation technique which allows for identification of novel  protein complexes which interact and may potentially detect naphthenic acids.  Our team developed an optimized protocol for the biotinylation of cyclohexanepentanoic acid (CHPA), cyclohexanebutyric acid (CHBA), and cyclohexanecarboxylic acid (CHCA), and a protocol for the immunoprecipitation of <I>Pseudomonas</I> lysates using commercially available streptavidin magnetic beads.</P><P>In addition, our team developed an <I>in silico</I> approach to isolating proteins which may be involved in degrading naphthenic acids using two specific species of <I>Pseudomonas</I> previously reported to degrade NAs in co-culture.  Through use of various bioinformatic databases, a list of potential naphthenic acid degrading proteins was identified.  Of these, a set were tested using reverse transcriptase polymerase chain reaction (RT-PCR) to identify a change in expression when incubated with naphthenic acids.  A novel Enoyl-Coa-hydratase was identified and may be a responsive element to naphthenic acids.</P><P><B><A href="https://2011.igem.org/Team:Calgary/Project/Promoter/Bioinformatics">Click here</A> for more information about our Bioinformatic search.</B></P><H2>Sensory Element Fishing: Designing A Novel Biotinylation-Immunprecipitation System For Isolation of Naphthenic Acid Interactors</H2><H2>What Are We Looking For?</H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 1</B> The model naphthenic acid Cyclohexanepentanoic acid and the potential targets which our team was looking for in our screen for a NA responsive element.  Our biotinylation/immunoprecipitation technique can be optimized for searching for any of these four response elements.  The techniques described further in this document focus on transcription factor and degradation enzyme based optimization.</DIV></DIV></DIV><P>Due to the complexity of biological systems, our team focused our efforts on identifying a system for identification of proteins or complexes which bind DNA.  Transcription factors, which bind to specific elements of a gene known as the promoter, have been well characterized in other metabolic systems to respond to metabolites in the cell and change the level of transcription of particular metabolic response genes.  A common example of one of these systems is the Lac operon which has been harnessed as a selection system already in the registry.  Identifying a system such as this for naphthenic acids, if one exists, would allow for a specific response to be generated to many different kinds of naphthenic acids.  Our hypothesis requires that the organisms we use respond specifically to naphthenic acids and result in specific upregulation of metabolic genes with little background effect in the cell.  Additionally, numerous  protein elements have been shown to bind and modify DNA, resulting in downstream changes to protein expression.  Examples of this are histone modifying enzymes in Eukaryotes which have opened up a new realm of research known as epigenetics.  These systems can be complicated and often do not directly recognize metabolic chemical compounds.  Furthermore, it is possible that naphthenic acids may even interact with specific RNA responsive elements known as riboswitches, which may be involved in the modification of protein expression.  While the identification of any of these systems were of specific interest to us to identify a detection system for naphthenic acids, we wanted our approach to be broad enough to idenitfy any components interacting with naphthenic acids.</P><H2>Naphthenic Acid Degrading Organisms To Be Used</H2><P>Due to the diversity and ambiguity of microbial species in the tailings ponds, we chose to focus our efforts for our promoter search on two species of <I>Pseudomonas</I> and a species of microalgae <I>Dunaliella tertiolecta</I> (<B>please see our <A href="https://2011.igem.org/Team:Calgary/Project/Chassis">chassis section</A> for more information on these species</B>).  All of these organisms have been shown to degrade naphthenic acids, however little work has been done in characterizing the degradation pathways of both species.</P><H2>Method Design</H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 2</B> A typical immunoprecipitation (IP) performed using an antibody raised against a particular protein target.  The antibody is incubated with a cell lysate containing purified protein.  The antibody will bind to its specific antigen and the solution may be incubated with beads that are capable of binding the antibodies.  The beads, coupled to the antibody and protein target can then be isolated from the mixture through the use of a magnet (if the beads are magnetic) or through centrifugation.  The protein can be denatured off of the beads and analyzed using mass spectrometry to identify their sequence.</DIV></DIV></DIV><P>As we plan to screen both bacterial and microalgae species, it was vital that our screening technique be robust and efficient in a variety of cellular systems.  It was also important that it selectively targeted protein elements which were specific for naphthenic acids and did not result in unwanted background.  A common technique used in similar applications including protein biochemistry is immunoprecipitation (IP) for the identification of protein interactors against a particular bait protein.  An antibody is incubated with the lysate of the organism of interest.  The target of this antibody and its interacting proteins can be isolated using beads that bind to a conserved region opposite the antigen binding domain.  Isolation of the target protein along with its interactors allows for their identification through mass spectrometry (MS), a routinely used technique to identify proteins.  This technique has been well established and is used routinely in a variety of applications.</P><P>In a similar manner we designed a protocol to use naphthenic acids in an immunoprecipitation.   This however, required that we develop a system to conjugate our naphthenic acids to a bead platform similar to the commercially available antibodies used in traditional IP.  To accomplish this, we attempted to react the conserved carboxylic acid of the naphthenic acid cyclohexanepentanoic acid (CHPA) (<B>see <A href="https://2011.igem.org/Team:Calgary/Project"> overview section</A></B>) with a commercially available biotin-amine derivative.  By reacting these compounds in the presence of EDC (1-ethyl-3-(3-dimethylaminopropyl)carbodiimide) the carboxylic acid and primary amine can produce an amide linkage under low pH conditions.  Biotinylation is commonly performed with DNA and protein side chains using commercial kits designed for these applications.  However, the biotinylation of small molecules is rare and techniques involving hydrophobic compounds such as CHPA are not found in the literature.  This required that we generate a <B>novel</B> protocol for biotinylating CHPA along with other naphthenic acid compounds.</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 3</B> Chemical structure of Cyclohexanepentanoic acid (CHPA), Biotin-Amine, and the final product with an amide bonding formed between the carboxylic acid of CHPA and the primary amine group of the Biotin linker.</DIV></DIV></DIV><P>Once our naphthenic acid has become biotinylated, the biotin group will strongly interact with a protein known as streptavidin.  The biotin/streptavidin interaction is one of the strongest binding interactions characterized in biology (Kd=1x10<SUP>-15</SUP>M) and is commonly cross-linked to magnetic beads allowing for our biotinylated naphthenic acid to be used in similar protocols as in immunoprecipitation.  In theory, proteins which interact with the cyclic ring structure of cyclohexane pentanoic acid can be isolated using our biotinylated CHPA/streptavidin beads platform. Finally mass spectrometry can be used to determine the identity of these regulatory elements (See Figure 3).</P><H2>Biotinylation</H2><DIV class="thumb tleft"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 4</B> Chemical mechanism for the formation of a amide bond between cylohexanepentanoic acid and a commercially available biotin-amine compound.</DIV></DIV></DIV><P>The biotinylation of molecules containing carboxylic acid groups follows a well characterized reaction mechanism.  In the context of our biotinylation reagents (CHPA and Biotin-PEG-amine) the reaction mechanism can be seen in Figure 4. The cyclohexanepentanoic acid will react with the carbodiimide (EDC) to produce an O-acylisourea intermediate activating the carboxylic acid making it an excellent leaving group.  This requires a low pH environment to deprotonate the carboxylic acid.  From this intermediate the desired product can be produced via direct attack of the negatively charged carboxylic acid with the amine group of our biotinylated compound producing a urea based compound as a by-product.   Further reaction of the O-acylisourea intermediate with cyclohexanepentanoic acid may result in an acid anhydride which then produces the biotinylated compound of interest.  The main byproduct of the reaction involves the rearrangement of the O-acylisourea to a stable N-acylurea.  Common solutions to reduce the reaction towards this product involves the use of low-dielectric constant solvents such as dichloromethane.  Due to the hydrophobic nature of cyclohexanepentanoic acid, aqueous low pH solutions will cause precipitation of this reaction substrate, therefore, the use of other solvents such as dimethylsulfoxide need to be used.  The final reaction solution used in all biotinylation experiments used 9% DCM/ 18% ddH2O / 73% DMSO. Typical reactions run overnight at room temperature. (Please see our protocols page for more information on the biotinylation experiment).</P><H2>Biotinylation Confirmation</H2><P>In order to confirm the biotinylation reaction was occurring, reverse phase High Performance Liquid Chromatography (HPLC) was used.  A C18 analytical column (Dionex Acclaim 120) connected to a Dionex ICS-5000 series HPLC was adapted from a program which was based on  complex naphthenic acid solution.</P><P>The HPLC conditions used for the analysis were as follows.  A 60 min. program was designed with a mobile phase of 40% Acetonitrile / 60% 0.1% TFA Water with a gradient from 3-28 min. changing to 85% Acetonitrile/15% 0.1% TFA Water solution holding until 40 min.  The final 5-20 min. of the program (runs later in the year were modulated such that there was only a 5 minute ramp back to starting conditions) ramped back to original gradient to clean the column.  A 25 μL sample loop was used for all injections except for those used in purifying large amounts of biotinylated compound where a 500 μL sample loop was used.   Samples were loaded onto the column at a flow rate of 0.6 mL/min at a temperature of 30<DEGREE>C.  

</DEGREE></P><P>To establish a set of control conditions for this data analysis, DMSO and DCM were individually injected onto the column and a read out at 210 nm on a UV/VIS spectrophotometer over a 60 min.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:627px;"><DIV class="thumbcaption"><B>Figure 5.a</B> Blank run of 100% dimethylsulfoxide (DMSO) injected onto a C18 column.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:627px;"><DIV class="thumbcaption"><B>Figure 5.b</B> Blank run of 100% dichloromethane (DCM) injected onto a C18 column.</DIV></DIV></DIV></DIV><P>Due to the large peak produced by both of these solvents, it was thought that identification of potential aqueous solutions which elute between 3-7 min. of the program may be masked by the large peak produced during elution of DMSO or DCM.  Therefore to examine the peaks for the reactants of the solution (biotin, and EDAC) were examined in water solutions.  CHPA was diluted in 80% DMSO.  Biotin, EDAC, and CHPA were loaded onto the column at 25 uL of 100 mM, 500 mM, and 2mg/mL stock solutions</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:627px;"><DIV class="thumbcaption"><B>Figure 6.a</B> Biotin control sample injected onto the column.  The biotin peak has a retention time between 3-7 min. Therefore when ran with DMSO, the biotin peak will be masked.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:627px;"><DIV class="thumbcaption"><B>Figure 6.b</B> EDAC catalyst sample injected onto the column. Due to the large amount of EDAC loaded onto the column, the peak is above the absorbance limit of the detector.  Like the biotin peak, EDAC has a retention time similar to that of DMSO making it impossible to see when running the reaction solution onto the HPLC column.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:627px;"><DIV class="thumbcaption"><B>Figure 6.c</B> Control run of cyclohexanepentanoic acid (CHPA) injected onto the column.  This produced a specific peak with a retention time of 28 min.  Other peaks found in the run correspond to non-specific peaks observed in the DMSO control sample as seen in Figure 1.a.</DIV></DIV></DIV></DIV><P>Due to the retention time of biotin and EDAC being within the rentention time of DMSO, there presence was difficult to detect during the biotinylation reaction.  A reaction was set up in 80% DMSO/ 20% ddH2O to test the efficiency of reaction.  A control and reaction was set up as written in the protocols section of the wiki, and incubated O/N at room temperature.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:627px;"><DIV class="thumbcaption"><B>Figure 7.a</B> Control of Biotinylation Reaction.  Biotin was injected into the column in a solution of 80% DMSO/20% Water.  The HPLC graph has been zoomed into the corresponding region identifying the change in peaks between the control and reaction solutions.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:627px;"><DIV class="thumbcaption"><B>Figure 7.b</B> Reaction solution containing CHPA, Biotin, and EDAC incubated at room temperature for 24 hours. Reaction solution was injected onto a C18 column.</DIV></DIV></DIV></DIV><P>Four peaks were observed in the reaction dataset which was not observed in the control with retention times of 15 min. 19 min. 20 min. and 28 min.  The 28 min. peak was found to correspond to the CHPA control.  Therefore the remaining three unique peaks were determined to show that the reaction was progressing. Independent HPLC analysis showed that after 24 hours the reaction had gone to completion and there was no difference in peak sizes observed (data not shown).  
In order to further optimize this reaction, the reaction was repeated with and without the use of 9% DCM, in the same conditions as before.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:627px;"><DIV class="thumbcaption"><B>Figure 7.a</B> Biotinylation reaction in 80% DMSO/20% Water.  Samples were incubated at room temperature overnight and injected onto a C18 column using regular protocol.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:627px;"><DIV class="thumbcaption"><B>Figure 7.b</B> Biotinylation reaction in DCM/DMSO/Water.  Samples were incubated at room temperature overnight and injected onto a C18 column using regular protocol.</DIV></DIV></DIV></DIV><P>Comparison of the two datasets shows an increase in the three reaction peaks with relatively no change in the CHPA peak, suggesting that dichloromethane’s low dielectric constant increases the relative rate of reaction.  Finally, to confirm the identity of our biotinylated product, peak fractions were collected for a potential biotin peak (~4 min.), the peak at 19 min. and the second peak at 20 min. The expected masses of biotin control and the expected products are as follows:

<UL><LI>Biotin = 374.50 Da
</LI><LI>Biotin-CHPA = 540.76 Da
</LI><LI>N-acylurea = 339.52 Da
</LI><LI>EDAC-Urea = 173.26 Da
</LI></UL>

Samples were collected from the HPLC in 30 sec. fractions of 0.3mL and dried down in a vacufuge at 45C. Samples were submitted to the University of Calgary Department of Chemistry MS Facility for electrospray analysis.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:627px;"><DIV class="thumbcaption"><B>Figure 8</B> Mass Spectrometry Electrospray data for the three peaks with retention times (from left to right) of 4 min. (Control Biotin), 19 min. (The Biotinylated Product), and 20 min. (N-acylurea side reaction).  Please click on enlarge to view a better quality version of this image.</DIV></DIV></DIV></DIV><P>MS analysis identified the control Biotin at the correct size as determined by the literature.  The peak at 19 min. corresponds to the size of Biotin-CHPA plus a sodium ion (22.99 Da) which is found in high quanitites in the methanol used in electrospray which may be responsible for the higher peak size identified.  Finally, the peak at 20 min. corresponded to the N-acylurea, the by-product of the reaction.  This was an interesting result as it suggests that the DCM provided in the solution (as shown in Figure 7 reaction DMSO, reaction DCM) did not decrease the relative amount of byproduct to biotinylated-CHPA, but the overall use of biotin in the solution, perhaps through increasing the degree of intermediate formation during the first steps of the reaction.</P><P>From the electrospray data, the peak at a retention time of 19 min. was determined to be the biotinylated product.</P><H2>Biotinylation of Other Naphthenic Acids</H2><P>Once cyclohexanepentanoic acid was identified to be biotinylated other compounds were selected and attempted to be biotinylated using the same procedure as above.  Two other naphthenic acids, cyclohexanebutyric acid, and cyclohexanecarboxylic acid, were selected due to their availability, low cost, and differing side chains from CHPA.  Their unique hydrophobic characters also allowed for our procedure to be tested against compounds which were slightly more hydrophilic than CHPA.  Both of these compounds were incubated using the same reaction mixture as before and the solutions were run on HPLC, to analyze the success of these reactions.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:555px;"><DIV class="thumbcaption"><B>Figure 9</B> Chemical structure of CHBA and CHCA, two other naphthenic acids which were conjugated to our biotinylated linker.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:627px;"><DIV class="thumbcaption"><B>Figure 10</B> HPLC analysis of CHBA, CHCA, and CHPA, comparing their unique peaks.  CHBA and CHCA elute off of the column with a lower retention time than CHPA due to CHPA's hydrophobicity.  All solutions were incubated overnight in the solvent solution described above.</DIV></DIV></DIV></DIV><P>Interestingly, the peaks identified by both compounds were very similar in layout to those identified for CHPA, suggesting that the compounds were being biotinylated and producing the same expected side reaction products.  Both reactions additionally showed a reduction in retention time for all peaks which was expected. CHBA and CHCA were also analyzed alone, to identify their corresponding peaks throughout the time course.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:627px;"><DIV class="thumbcaption"><B>Figure 11</B> HPLC analysis of pure CHCA (top) and CHBA (bottom) loaded at 50 ug.  Each peak is denoted by a red arrow.  Non-specific peaks in the CHCA run were a result of a previous run from a CHPA reaction solution which had been retained on the column.  These peaks were not observed in any follow up runs using CHCA in reactions.</DIV></DIV></DIV></DIV><P>CHCA was found to have a retention time of approximately 12.5 minutes and CHBA, a retention time of <B>24</B> min.  These corresponded to peaks in the reaction solution similar to that of CHPA.  An advantage of using less hydrophobic compounds such as CHCA is that it allows for a decrease in the concentration of DMSO to be used in solution, making it easier to handle these biotinylated compounds for downstream aqueous applications.  <B>Biotinylation of these three compounds, CHPA, CHBA, and CHCA identifies a proof of concept that our biotinylation strategy is effective at conjugating biotin-linkers to a variety of hydrophobic compounds.</B></P><H2>Biotinylated-Hydrophobic Small Molecules and Streptavidin Coated Beads</H2><P>With an optimized protocol for the biotinylation of small hydrophobic molecules, our team turned to identifying a method by which these conjugates could be efficiently taken up by streptavidin coated beads.  Streptavidin and avidin are known to tightly interact with biotin, so much so that this interaction is considered one of the strongest non-covalent interactions known to man.  The interaction is resistant to detergents, heat, alkalinity, reduction/oxidation, as well as a variety of other stresses making it a useful platform for use with hydrophobic small molecules.  However, it was important that our team identify a method by which we could test the binding of our beads to our biotinylated naphthenic acids under non-polar solvent conditions.  Commercially available streptavidin coated magnetic beads from Solulink were used to test this interaction.  100μg of beads were mixed with biotinylated naphthenic acids end-over-end for 30 min. at room temperature in a variety of solvent conditions and HPLC analysis was used to identify a change in peak height suggesting the beads were binding to the biotinylated compound (see Figure 12).  </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:627px;"><DIV class="thumbcaption"><B>Figure 12</B> HPLC analysis of 10 nmol biotin-linker incubated with and without 500 ug Solulink streptavidin coated magnetic beads end-over-end 30 min. room temperature. Samples were dried down using a vacufuge to remove solvent and resuspended in dH2O.  Run's labelled with &quot;Beads&quot; were incubated, while those without this title had water added to the solution.  All solutions were made to a 300 uL volume.  Solvents were as follow: <I>+ve Control</I> - Water, <I>-ve Control</I> - 9% DCM/ 18% ddH2O / 73% DMSO, <I>No DCM</I> - 27% ddH2O / 73% DMSO, <I>1/2DMSO/DCM</I> - 4.5% DCM/ 59% ddH2O / 36.5% DMSO, <I>1/2 DMSO No DCM</I> - 63.5% ddH2O / 36.5% DMSO.  Peaks were analyzed based on their height and width qualitatively.</DIV></DIV></DIV></DIV><P>Unfortunately, no significant shift in peak size was identified in peak height using the Solulink beads, even with the biotinylated linker alone dissolved in water.  A distinct difference was noticed when beads were incubated with the 1/2DMSO No DCM runs, however, due to the lack of binding in water alone, further optimization was required.  With these concerns a new bead platform would be required to increase the efficiency of this binding.  Moving on, we attempted to optimize our biotinylated naphthenic acids using DynaBeads supplied by Invitrogen.  Through use of their sample kit we were able to identify the relative amount of binding to the beads of our biotinylated linker alone in water using HPLC. 

</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:627px;"><DIV class="thumbcaption"><B>Figure 13</B> HPLC analysis of 10 nmol biotin-linker incubated with and without 100 uL Invitrogen streptavidin coated magnetic beads end-over-end 30 min. room temperature. Beads were incubated in 1x PBS Buffer pH 8.3.  High pH was due to downstream application in order to keep hydrophobic compounds such as CHPA into solution allowing for binding to the beads.  Four types of beads were tested and quantitated against a biotin control incubated without beads.  All solutions were dried down and resuspended in water and 1/12th of the solution was injected onto the column.  After quantification relative to the control, the 270 beads were identified to be the strongest binder of the biotinylated linker.</DIV></DIV></DIV></DIV><P>After quantification against four different kinds of Invitrogen DynaBeads, 270, was determined to be the best binder of the biotin-linker alone.  1xPBS pH 8.3 was used to test the beads as it was previously noted that numerous highly hydrophobic molecules (such as CHPA in its biotinylated form) seemed to readily stay in solution at the pH.  This assay confirmed that the high pH of the PBS buffer did not comproomise the binding of biotin to streptavidin.  This quantification identified the 270 streptavidin magnetic beads to be the best at binding this linker piece and the best type of bead system to use for downstream analysis with our buffer system.  DMSO and DCM were not tested in this assay when unlike in the Solulink beads assay because the supplier was able to provide data suggesting that concentrations of DMSO at least up to 8% would not be harmful to the beads (Please contact Invitrogen for this information).    

</P><H2>Immunoprecipitation and Future Directions</H2><P>With the success of our biotinylation reaction, and the ability to determine a successful type of beads to perform our pull downs, we established a protocol by which we could perform a traditional IP for proteins which may bind to the cyclohexane ring of CHPA.  This protocol can be found in the <A href="https://2011.igem.org/Team:Calgary/Notebook/Protocols/Process22">Protocols</A> section of the notebook  in the menu.  This procedure allowed us to successfully pull down protein interactors as seen in Figure 14.

</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:627px;"><DIV class="thumbcaption"><B>Figure 14</B> BioRad 4-20% Bis-Tris gradient gel analysis of potential CHPA interacting proteins.  Stars highlight enriched bands found in the CHPA elution but not identified in the Biotin control.  For a step by step protocol of this procedure please see the protocols section.</DIV></DIV></DIV></DIV>


This pull-down suggests that our biotinylation scheme is capable of selectively pulling down protein elements specific to the conjugated element on the biotin linker.  Our protocol is capable of enriching these interactions and concentrating them onto a gradient gel.  The ability to pull down protein elements was the task which we chose to focus on as an initial proof of concept of this technique, however, one could adapt this protocol for the &quot;fishing&quot; of RNA, DNA (as in the example of chIP-SEQ), or other elements.<P>With isolated proteins, the University of Calgary team plans to determine their identity through mass spectrometry, and thus identify a potential naphthenic acid sensitive promoter.  Additionally, by developing a larger consortium of biotinylated hydrophobic molecules, our team can identify unique responsive elements which may aid in the development of our biosensor.  The biotinylation and immuno-based precipitation of small hydrophobic molecules is a novel approach to identification of protein binding partners in the literature and our team aims to optimize this further in the future</P><H2>Conclusions</H2><P>A novel system for biotinylating cyclohexanepentanoic acid as well as other model naphthenic acids has been developed and verified using HPLC and MS analysis.  While further optimization of the protocols is required before performing IP based experiments, this technique provides the first steps toward a novel system to detect and isolate proteins involved in the naphthenic acid detection pathways of <I>Pseudomonas</I> and <I>Dunaliella</I>.</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Calgary/Project/ProjectPseudomonas">http://2011.igem.org/Team:Calgary/Project/ProjectPseudomonas</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Calgary/Project/ProjectPseudomonas" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Calgary/Project/ProjectPseudomonas" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Calgary/Project/ProjectPseudomonas&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Calgary/Project/ProjectPseudomonas&amp;oldid=262164" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>