<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Calgary_Notebook_Journals_OLD_ENTRIES"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Calgary/Notebook/Journals/OLD_ENTRIES">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Calgary/Notebook/Journals/OLD_ENTRIES&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Notebook/Journals/OLD_ENTRIES&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Notebook/Journals/OLD_ENTRIES&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Calgary/Notebook/Journals/OLD_ENTRIES" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Calgary/Notebook/Journals/OLD ENTRIES</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="headercontainer"><DIV id="headerbar"><UL id="topnav"><LI id="homebutton"><A class="navtext" href="https://2011.igem.org/Team:Calgary">Home</A></LI><LI id="teambutton"><A class="navtext" href="https://2011.igem.org/Team:Calgary/Team">Team</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Team/Biographies">Biographies</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Team/Facilitators">Facilitators</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Team/About_The_University">The University</A></LI><LI><A href="http://ung.igem.org/Team.cgi?id=640">Official Profile</A></LI></UL><LI id="projectbutton"><A class="navtext" href="https://2011.igem.org/Team:Calgary/Project">Project</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Promoter">Promoter</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Reporter">Reporter</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Chassis">Chassis</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Reporter/Prototype">Prototype</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/DataPage">Data Page</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Acomplishments">Accomplishments</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/Future_Directions">Future Directions</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Project/References">References</A></LI><LI id="partsbutton"><A class="navtext" href="https://2011.igem.org/Team:Calgary/Parts">Parts</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Parts">Parts Submitted</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Attributions">Attributions</A></LI><LI id="notebookbutton"><A class="navtext" href="https://2011.igem.org/Team:Calgary/Notebook">Notebook</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/PostRegionals">Post-Regionals</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Calendar">Journal</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Safety">Safety</A></LI><LI id="outreachbutton"><A class="navtext" href="https://2011.igem.org/Team:Calgary/Outreach">Outreach</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Outreach/Human">Human Practices</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Conferences">Conferences</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Outreach/Follow">Follow Us!</A></LI><LI id="sponsorsbutton"><A class="navtext" href="https://2011.igem.org/Team:Calgary/Sponsors">Sponsors</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Sponsors">Sponsors</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Sponsors/Acknowledgements">Acknowledgements</A></LI><LI id="igembutton"><A class="navtext" href="https://2011.igem.org" target="_blank">iGEM</A></LI></DIV></DIV><DIV id="sidebar"><H2 id="sidetitle">Notebook</H2><DIV id="sidetext"><H4><A href="https://2011.igem.org/Team:Calgary/Notebook/PostRegionals">Post-Regionals</A></H4><H4><A href="https://2011.igem.org/Team:Calgary/Notebook/Calendar">Journal</A></H4><UL><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Calendar/Week1">Defining Objectives</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Calendar/Week2">Techniques Workshop</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Calendar/Binf">Bioinformatics &amp; qRT-PCR</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Calendar/ChipSEQ">Sensory Element Fishing</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Calendar/Engineers">Electrochem - CPR Oxidation</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Calendar/LiveCell">Electrochem - Reporter Gene</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Calendar/OriT"><I>Pseudomonas</I> Tools</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Calendar/Microalgae">Microalgae Tools</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Journals/NA_viability">NA Viability Assays</A></LI><LI><A href="https://2011.igem.org/Team:Calgary/Notebook/Journals/OLD_ENTRIES">May - July Group Journal </A></LI></UL><H4><A href="https://2011.igem.org/Team:Calgary/Notebook/Protocols">Protocols</A></H4><H4><A href="https://2011.igem.org/Team:Calgary/Safety">Safety</A></H4></DIV></DIV><H1 id="pagetitle">Group Log Entries from May to Early July</H1><DIV id="bodycontainer"><DIV id="bodytext"><H2> Project Description </H2><P> At this early stage in the project clearly distinct sub-projects had not emerged thus journal entries take the form of group logs on the activities that took place each day.</P><H3>Project Participants</H3><P> Everyone.</P><H3>May 2, 2011 - May 5, 2011</H3><H4>May 2, 2011</H4><P>Brainstorming based on ideas we had in a preparation meeting.
Main ideas were investigating biofilms, biosensor development, and promoting methane production over sulfur production. 
While the discussion leaned towards creating a Biosensor, we decided that we needed more information to make a decision and researched different types of cells in tailings ponds that are able to detect and/or break down naphthenic acids. </P><H4>May 3, 2011</H4><P>Everybody presented the findings of their research about different cells to the group. We determined to create a biosensor and discussed the specific requirements for it. One of the most obvious problems was how the cells would react to the presence of naphthenic acids. The three options we decided to test were via pigment secretion, fluorescence, and an electrochemical response.</P><H4>May 5, 2011</H4><P>Research on the two different project paths was continued. We decided on developing a biosensor for naphthenic acids in tailings ponds.</P><P>We finished the first draft for the OSLI application and began editing it. </P><H4>May 6, 2011</H4><P>We worked on the OSLI application and did more research on the background of naphthenic acid reporters and sensors.</P><P>We also had a full meeting (Undergraduate &amp; Graduate students and professors) to recap the week and decided what we needed to do next week.</P><H3>May 9, 2011 – May 13, 2011</H3><P>See  the section on the <A href="https://2011.igem.org/Team:Calgary/Notebook/Calendar/Week2"> Techniques Workshop</A>.</P><H3>May 16, 2011 – May 20, 2011</H3><H4>May 16, 2011</H4><P>We separated the members into different groups and began the inventory of the supplies that we had. Work also began on the iGEM team wiki. </P><H4>May 17, 2011</H4><P>Work continued on the website. Members began work on their individual biographies, and website design has begun, using the previous years’ website as a starting point. Different websites were consulted for optimal design concepts. The inventory of our current assets started yesterday was completed.</P><H4>May 18, 2011</H4><P>Patrick conducted a tutorial for the group on the parts registry and how to look up specific segments of DNA online. Also parts agar plates were made for culturing transformed <I>E. coli</I>. the registry parts R0040, K325219, K325100, K274002, K274003, K274004, K274100, and K274120 were selected for <I>E. coli</I> transformation.</P><H4>May 19, 2011</H4><P>Continued working on the wiki and added team bios and pictures. Competent TOP10 bacterial cells were transformed with registry parts and plated. Discussed advantages and disadvantages of the output methods (Fluorescence, Pigment, Electrochemical) and sketched first prototype for the kit with electrochemical response. Preparing sponsorship levels as well as for presenting our ideas to companies.</P><H4>May 20, 2011</H4><P>At 9:00AM, no growth/colonies were observed on the plates. However, by the late afternoon, colonies were visible on all of the plates except for the tetracycline plates. We continued working on the wiki and researching biosensors.</P><P>Inventory is still being compiled. All that is left is the fridge inventory and the -80 freezer. Aesthetic designs for team logo, sponsorship package, and wiki are being considered and experimented with.</P><P>Sponsorship package is under construction, slowly taking form. Team meeting at 3pm, large discussion with lots of valuable ideas put forward.</P><H3>May 24, 2011 – May 27, 2011</H3><H4>May 24, 2011</H4><P>Tetracycline plates produced previously were suspected of having the wrong amount of tetracycline. as a result none of the transformed bacteria grown on the tetracycline plates grew. new tetracycline agar plates were made and the successfully incubated plates (i.e. those that displayed colonies) were cultured. Patrick and Robert worked on the sponsorship package to submit to possible donors and sponsor. Also skeleton side bars and skeleton content were made for most of the wiki pages and information was added to the security and research was done in regards to ethics.</P><P>Spoke with a representative from Worley-Parsons.</P><H4>May 25, 2011</H4><P>The registry parts that had not been successfully cloned were again used to transform competent top 10 cells. The registry part I732019 was also used to transform DNA. Most of the transformed bacteria that had been cultured exhibited growth, however four of them did not. these four bacteria samples were cultured again. The sponsorship package was completed pending proofreading by experts.</P><H4>May 26, 2011</H4><P>None of the cultures prepared the previous day exhibited any growth. Only the parts transformed on ampicillin plates, the previous day, resulted in any observable colonies while those on tetracycline plates showed no growth. all of the previously transformed and isolated parts were also used to run an agarose gel confirming the identity of the transformed parts.</P><H4>May 27, 2011</H4><P>Isolated I732019 from transformed cells, measured the concentration and added it to the stock in the freezer. We held a meeting in the afternoon and went over the week’s events before faculty members and the whole team. </P><H3>May 30, 2011 – June 3, 2011</H3><H4>May 30, 2011</H4><P>Attempted to create a biobrick containing the promoter I0500 (then from the previous year’s stock) and the lacZ reporter (I732019), in the plasmid pSB1AC3 (transformed previously). the parts were restriction digested.</P><P>Continued literature searches of riboswitches and other promoter ideas.</P><H4>May 31</H4><P>The parts from the previous day were ligated, then used to transform TOP10 cells, the blue script plasmid was also used to transform TOP10 cells which were then plated.</P><H4>June 1, 2011</H4><P>The transformed cells successfully produced colonies. The colonies formed by the bacteria carrying the construct made previously were used to run a colony PCR and the products were kept for running an agarose gel to confirm them. cells from the colonies (two plates carrying the construct) were used to make an overnight culture.</P><P>Emily, Robert, and Patrick went to the town of Bassano, Alberta as volunteers for Let’s Talk Science. In exchange for helping do some of the labs, they introduced iGEM and microbiology research to students in grades 6, 8, 9, and 12.</P><H4>June 2, 2011</H4><P>We ran an agrose gel to confirm the products of the colony PCR, however the gel was old and had been exposed to light so the results were inconclusive. A gel was also run to test whether the restriction digested parts (used to make the construct) had been properly digested. the cell culture from the previous day was miniprepped, and the isolated plasmid (construct containing I732019, 1AC3, I0500) was restriction digested as was the stock of I0500.</P><H4>June 3, 2011</H4><P>The restriction digested parts from the previous day and the colony PCR parts were used to again run a gel to confirm their identities. This time a freshly made gel was used. Another colony PCR was prepared to be run over the weekend.</P><H3>June 6, 2011 – June 10, 2011</H3><H4>June 6, 2011</H4><P>A gel was run for the colony PCR performed over the weekend. Also all the previous stocks of I0500 were removed from the freezer and restriction digested then run on a gel. The I0500 that displayed complete digestion after 70 min was then selected to remake the construct made the previous week (I0050, I732019, 1AC3). The construct was then used to transform top10 cells which were plated.</P><P>Discussed plans for the week. A poster for the ISMOS-3 conference on Monday is required. Patrick is working on this.</P><H4>June 7, 2011</H4><P>Hardly any colonies could be observed on the plates from the previous day in the morning so the construct was remade again and TOP10 cells were transformed and then plated. Later in the day several colonies were observed on the plates containing the transformed cells from the previous day so the colonies were used to run a colony PCR and cultured overnight. TOP10 competent cells were also cultured overnight with the intention of generating new cells.</P><P>BNH: Got the amplifier needed to test the electrical signal. started familiarization with software. we soon realized that we would need to shield the probe from electric interference to reduce noise. </P><H4>June 8, 2011</H4><P>The PCR products from the previous night were run on a gel however the results of the gel indicated that the PCR was not successful, thus the colony PCR was repeated. the plates from the previous day showed many colonies but since they were not immediately needed the plates were sealed and put in storage. the cultures from the previous day were miniprepped; samples were restriction digested and run on a gel, the results confirmed the identity of the plasmid DNA. the TOP10 cell cultures were used to produce new TOP10 cells.</P><P>BNH: More familiarization with amplifier and software. We decided easiest way of shielding would be to construct a Faraday box from a shoebox and aluminium foil. </P><H4>June 9, 2011</H4><P>The PCR products from the previous day were run on an agarose gel. The results indicated once again that the PCR was unsuccessful. it was suspected that there was a problem in either the primers or the TAQ polymerase used in all the PCRs. Consequently a PCR was run in which a new sample of TAQ was used and the old biobrick primers (using a variety of plasmids) to assess whether or not the problem lay with the primers. Also the DNA which had been extracted the previous day (miniprepped) was sent for sequencing to confirm the identity of the construct before going on to testing it.</P><P>BNH: Constructed Faraday box. realized that the measurements aren’t changing. During error diagnosis we noticed that the software wasn’t properly recognizing the hardware due to driver errors. Spent rest of the day looking for drivers and ended up updating the whole software to the newest edition. </P><H4>June 10, 2011</H4><P>Ran an agarose gel for the products of the PCR reaction. the results indicated that there was no problem with the primers. </P><H3>June 13, 2011 - 17, 2011</H3><P>This week was mainly taken up by the ISMOS-3 conference, as well two of the team members were away for the Syn Bio conference in California. the remainder of the week was spent planning for future projects and reading up on designing growth and viability assays.</P><H3>June 20, 2011 – June 24, 2011</H3><H4>June 20, 2011</H4><P>BNH: Did first test readings with the amplifier. Since it was the first day, most data we got turned out to be a learning experience. Since we were working with PAP, without knowing that it oxidizes in air and water, most of the readings were confusing. As a result we started working with a fixed current and measured the voltage, that setup gave us better readings with PAP. </P><H4>June 21, 2011</H4><P>BNH: Took control measurements with pure H<SUB>2</SUB>O and TPW. Measurements are (relatively) useless since we were still working with constant current, measured voltage.</P><P>After running comprehensive BLAST searches it was decided that our sequenced construct was not what we thought it was, we decided to make a culture (in the hopes of running a gel to confirm the identity) of the plate made from the third transformation of the construct made on June 7, 2011.</P><H4>June 22, 2011</H4><P>BNH: Felix created .4% by volume (0.01M) CPR stock solution. We ran tests on 10 and 100 times diluted solutions to characterize it. In the process we developed a new test protocol; We switched back to a voltage clamp set-up, the Faraday box was shown to be very effective (10 times less noise in box). We noticed that a higher concentration of CPR recovers faster from the voltage stress. In response to that we increased the time to at least 90 seconds to give all samples sufficient time to reach equilibrium while still keeping tests short. </P><P>Started an experiment to measure how well <I>pseudomonas</I> can grow on naphthenic acids by measuring OD over times and concentrations. </P><P>The culture from the previous day was miniprepped and restriction digested and a gel was run. Another culture of transformed <I>E. coli</I> was made for the viability assay planned this week to determine the viability of <I>E. coli</I> cells in different concentrations of NAs and in tailings water.</P><H4>June 23, 2011</H4><P>BNH: Ran tests on TPW based CPR solutions. </P><P>Finished growth experiment, results were as expected, the more naphthenic acid there was the slower the growth. Surprisingly the difference wasn’t large though, will repeat at different concentrations. </P><P>The gel from the previous day did not turn up well, so no conclusions were drawn from it. A growth curve experiment was performed for the <I>E. coli</I> cultred the previous day to assess what conditions (heat, shaking) the cells could grow in and to track the progression from lag to log to death phase. New agar plates were also made.</P><H4>June 24, 2011</H4><P>New project: a bioinformatics analysis of <I>Pseudomonas putida</I> and <I>Pseudomonas fluorescens</I> genomes to search for non-homologous genes/pathways. 
The viability assay was performed and using the results of the previous day’s growth curve we managed to expose the <I>E. coli</I> cells to the different treatments (different concentrations of different types of NAs, and tailings water) when they were rapidly dividing in log phase. A culture was made of bacteria transformed with I732005.</P><H3>June 27, 2011 – July 1, 2011</H3><H4>June 27, 2011</H4><P>BNH: Tested Multiple Voltages (1000, 500, 350, 290, 220 mV) on two different concentrations (0.04% &amp; 0.004% by Volume) of CPR.</P><P>Examined the results of the viability assay and miniprepped the cultures of I732005 and another culture for I0500 + B0034 was also miniprepped. they were both digested and run on a gel and the gel from June 22, 2011 was repeated.</P><P>Created a Foght minimal media with no carbon source so we can test <I>Pseudomonas</I>’ ability to work with naphthenic acids. Made a test overnight culture to make sure it could grow in it (40mg/L naphthenic acids).</P><H4>June 28, 2011</H4><P>Another gel was run for the restriction digests after which it was concluded that they had fully digested. the digests were used to make a construct of I0500+B0034+I732005 which was then used to transform TOP10 cells. XyLR operon was also used to transform TOP10 cells.</P><P>The overnight culture worked really well so we can perform survival and growth assays with this media now. Cells grew in the presence of the acid in the new media.</P><H4>June 29, 2011</H4><P>The transformed parts from the previous day were cultured.</P><P>Transformations of violacein (K274002) consistently seem to fail to grow. Most likely discontinung this particular part.</P><P>Wiki concept art created. Team appears to approve of it. Pseudocode written out for the next day.</P><H4>June 30, 2011</H4><P>The cultures from  the previous day were miniprepped and restriction digested to confirm identity however the gel did not turn out nicely.</P><P>Attempted to write out the pseudocode for the wiki written the day before. Probably not the best idea. Too many untraceable failures in the code appeared. Will rethink the coding strategy for next week.</P><H4>July 1, 2011</H4><P>Canada Day long weekend.</P><H3>July 4, 2011 – July 7, 2011</H3><H4>July 4, 2011</H4><P>The parts from the restriction digest (identity confirmation) were run on a 0.5% gel instead of a 1% gel and the identity of the parts was confirmed. About 70 LB/agar plates were made and the strain carrying the I0500_B0034_I732005 was cultured in order to make a glycerol stock of it the next day. Also the strain used previously in the viability assay was also cultured as a new viability assay was planned for the next day.</P><P>New battle plan set up for the wiki coding. Will be done in modules with constant testing. Established the minimum iGEM header bar (with some help from Brown-Stanford) and created the basic framework of how the menu will be set up.</P><H4>July 5, 2011</H4><P>Glycerol stocks were prepared of the strain cultured the previous day, also the construct was sent for sequencing to confirm the identity. The viability assay (<I>E. coli</I> in NAs and tailings) design was improved and the assay was carried out (details of this can be found under the NA Viability Assays).</P><P>Ran through the general architecture of the wiki with the team. Seems to be approved. Uploaded images for the menu bar. </P><P>Started a viability assay with <I>Pseudomonas</I> and <I>E. coli</I> to see which we will use in our final product. Fun readings at 8am and pm for a week.</P><H4>July 6, 2011</H4><P>The results of the assay showed that the <I>E. coli</I> did not survive in the naphthenic acid solutions but did survive in the tailings water (see NA viability Assays).</P><P>The assay is going well, with the <I>Pseudomonas</I> outperforming the <I>E. coli</I> so far. The race is on!</P><H4>July 7, 2011</H4><P>The results for the DNA sent for sequencing was received and the identity was confirmed. The strain carrying the DNA (I0500_B0034_I732005) was then plated on a plate containing X-gal and IPTG. The parts E0240, K325210, I0500, and R0040 were used to make constructs I0500_E0240, R0040_E0240, and R0040_K32510. These parts were then used to transform TOP10 cells. These fluorescent reporters systems present a possible avenue for an alternative reporter system to the proposed electrochemical LacZ reporter system.</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Calgary/Notebook/Journals/OLD_ENTRIES">http://2011.igem.org/Team:Calgary/Notebook/Journals/OLD_ENTRIES</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Calgary/Notebook/Journals/OLD_ENTRIES" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Calgary/Notebook/Journals/OLD_ENTRIES" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Calgary/Notebook/Journals/OLD_ENTRIES&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Calgary/Notebook/Journals/OLD_ENTRIES&amp;oldid=203673" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>