"
Page
Discussion
View source
History
teams
Log in
 
Team:Goettingen/notebook wetlab
From 2014.igem.org
Home
Project ▾
Background
Our project
Results
Perspectives
BioBricks
Notebook ▾
Safety
Wet lab
Dry lab
Materials
Protocols
Our Team ▾
Team members
Instructors
Acknowledgements
Sponsors
Outreach ▾
Human Practice
Gallery
Video
Presentation
Paris Bettencourt Newsletters
German teams' proposal
Achievements
Notebook
Overview
Safety
Dry lab
Wet lab
PepTag Team
October
Week 23
Week 22
Week 21
September
Week 20
Week 19
Week 18
Week 17
August
Week 16
Week 15
Week 14
Week 13
Week 12
July
Week 11
Week 10
Week 9
Week 8
June
Week 7
Week 6
Week 5
Week 4
May
Week 3
Week 2
Week 1
BioBricks Team
GFP Team
Materials
October 13th, week 23
A. fumigatus & A. nidulans
A. fumigatus
- Results of robot Y2H:
(red color indicates a positive interaction, a blue colony. The grey color means that the result is not clear and it could possibly be a positive colony)
      ⇒ Growth and blue color indicates that there is an interaction between surface protein and the peptag.
      ⇒ to verify the interaction we will select on the peptag-plasmids and isolate them and transfer them into strep-vector (pGP172) for over expression in BL21(DE3)
      ⇒ the isolated protein is then used for in vivo studies
      ⇒ More in vivo experiments with the interacting peptide of Sun1 from A. fumigatus.
A. nidulans
      ⇒ In vivo experiment with the Sun1 interacting peptide from A. fumigatus with A. nidulans to prove that the peptide binds specifically to A. fumigatus.
C. albicans & C. glabrata
- Results of the robot Y2H (see Aspergillus)
- New in vivo experiments with Ssr1
October 6th, week 22
A. fumigatus & A. nidulans
A. fumigatus
- Continue the robot Y2H: replicate colonies from selection medium to selection medium with X-α-Gal
- First in vivo experiment with the interacting peptide of Sun1 from Aspergillus fumigatus StrepMAB Classic conjugated with ChromeoTM 488
A. nidulans
- Continue the robot Y2H (plate on to SC-LWH + 3mM 3-AT plates and X-α-gal plates)
C. albicans & C. glabrata
- Continue the robot Y2H (plate on to SC-LWH + 3mM 3-AT plates and X-α-gal plates)
- In vivo test with Candida glabrata ssr1 – k.o. strain → no fluorescence, due to too low protein concentrations (only worked for Sun1 from Aspergillus fumigatus)
S. cerevisiae
- Continue the robot Y2H (plate on to SC-LWH + 3mM 3-AT plates and X-α-gal plates)
T. gondii
- Continue the robot Y2H (plate on to SC-LWH + 3mM 3-AT plates and X-α-gal plates)
September 29th, week 21
A. fumigatus & A. nidulans
A. fumigatus
-  Robot Y2H to confirm interaction
      ⇒ Tested samples: AIG001, 02, 03, 04, 05, 06, 09, 10, 12, 13, 14, 15, 17, 19
- Grow AIG strains and isolated interaction partners (YIG strains) in selection medium over night at 30 °C
- Pipet grown colonies in 96 well plate (200 µl each)
- Preparation of robo-agar-plates
- Use AIG 96 well to stamp to -trp plates, YIG for - leu plates and incubate for 2 days at 30 °C
- Mating of grown colonies on YPAD plates
- Replicate colonies from YPAD to selection plates
- Replicate colonies from selction to selection medium
A. nidulans
- Robot Y2H to confirm interaction.
C. albicans & C. glabrata
-  Robot Y2H to confirm interaction (see Aspergillus)
S. cerevisiae
- Inoculation of AIG021 + 22 and YIG021 + 22 for robot Y2H
T. gondii
- Inoculation of AIG025and YIG025 for robot Y2H
September 22nd, week 20
C. albicans & C. glabrata
-  Dialysis of purificated proteins (see Aspergillus)
- Preparation for robot Y2H
A. fumigatus & A. nidulans
A. fumigatus
-  Contamination on robot plates → Again
- Prepare 10x PBS buffer and medium for in vivo tests
- Boil dialysis tubes for 10 minutes and add EDTA. Cool them down and fill in proteins for dialysis. Dialyze overnight at 4 °C.
- Streak out AIG04 on SC - Trp, YIG04.1 and YIG04.2 on SC - Leu
- Prepare robot plates the robot Y2H
September 15th, week 19
C. albicans & C. glabrata
- Purification of the peptides 3, 4.1, 4.2, 5 and 13
      • 1 L LB inoculated at an OD600 of 0.2 → grown to 0.8, then expression is induced with 1mM IPTG
      • After 3h: centrifuged at 5000rpm for 15 min at 4°C
      • After solving in Buffer W the cells are disrupted by French press and centrifuged for 1h at 35.000 rpm and 4 °C
      • The proteins are extracted via Strep-Tactin® columns
      • Samples were loaded onto an 12% SDS Gel
SDS Gels (12%)
Strep 3:
⇒ There should be a clear band in the elutions steps for ~ 20 -22 kDa; for strep 3 is no such band
Strep 4.1:
⇒ No band of ~20-22 kDa in the elution steps for strep 4
Strep 4.2:
⇒ The right band can be seen in the elution steps, but it is not as strong as it should be
Strep 5:
⇒ The enlarged band in the elution steps has a size of ~ 15 kDa, which seems to be a little small for our peptide
Strep 13:
⇒ For strep 13 the right bands of ~20-22 kDa can be seen in all lines, especially in the elution steps
A. fumigatus & A. nidulans
A. fumigatus
- Starting Seamless cloning mix, incubate at RT for 30 minutes, 4 minutes on ice and then E. coli transformation into TOP10 strain. Plating 200 µl and rest on cat plates.
- Determination of OD600 of strep cultures and inoculate 1 L LB + Amp to an OD of 0.2 and let them grow to an OD of 0.8 (approx. 1 h). Induction with 1 ml of 1 M IPTG and incubation at 180 rpm and 37 °C for 3 hours. Centrifugation in autoclaved bottles for 15 minutes at 5 krpm at 4 °C. Freeze pellets in -20 °C overnight.
- Preparing robot plates and distribute 200 µl of each cell culture
- Run spotting program on the robot. And incubate at 30 °C
- Pre-cool french press stuff in ice water. Pre-cool Buffer W and tips and columns in cold room at 4 °C. Resuspend each pellet in 15 ml ice cold Buffer W. Run french press as in the protocol. Centrifuge the disrupted cells at 4 °C for 1 h and 35.000 krpm.
- Prepare columns, Buffer E, Buffer R and pack each column with 1 ml streptacin 50 %.
- Run the supernatant from UZ over the columns. Save rest in falcon tubes from not purified, flow through. Save wash step 1- 4 elution step 1- 4 in labeled 1.5 ml Ecups.
- Prepare 4 SDS gels 12.0 % (Gels see Candida)
- Calculation of protein amount in CE and FT with Bradford.
- Y2H with robot to confirm interactions
S. cerevisiae
- Repeat  trafo in Y187
      ⇒ Trafo worked
T. gondii
- Repeat  trafo in Y187
      ⇒ Trafo worked
September 8th, week 18
C. albicans & C. glabrata
- PCR for sequencing on strep-tag plasmids
- Purification and sequencing
      ⇒ Sequencing looks ok
- Test expression for peptides 3, 4.1, 4.2, 5, 13
SDS gels (12%)
Peptide 3:
Peptides 13 and 4.1:
Peptides 4.2 and 5:
- Test expression looks good; nothing in negative control (NK)
- super-competent cells with iGEM protocol → see Aspergillus
A. fumigatus & A. nidulans
A. fumigatus
- PCR with T7 primers for sequencing of 6 plasmid inserts (seqlab) → purification
- NanoDrop:
- Pick colonies from strep plates inoculate over night culture. Inoculation over day culture to OD600 0.3. Prepare test expression
- Ligation of cut clean pGP172 (11ng/µl) with cc5, 9, 10 ,15
- Follow blunt end-ligation protocol: 20 µl sample included 2 µl PEG4000 1 µl T4 ligase
- Preparation of final robot Y2H
- Transformation into E. coli DH5α
-  Sequences of Seqlab looks ok
- Test expression of 3, 4, 13
- Samples from test expression were filled up with 15 µl of SDS loading dye and boiled for 15 min at 95 °C. Loaded to SDS 12% PAGE. (Gels see Candida)
- Prepare comp. cells from TOP10 strain. Overnight culture was grown at 18°C for 19h OD600=0.395 → 1:1 dilution with 250 ml SOB for 1 h incubation at 18 °C. OD600 = 0.32, putted on ice and filled into prechilled 50 ml flasks. Followed the protocol.
      ⇒ Test the competence → worked very well!
A. nidulans
- prepare robot screening for 2st Y2H with the isolated plasmids out of the Y187 transformation
- 2st Y2H screening on plates - failed
S. cerevisiae
- Pick colonies from E. coli plates and isolate plasmids
- Transformation of  yeast (Y187)
      ⇒ Trafo failed
T. gondii
- Pick colonies from E. coli plates and isolate plasmids
- Transformation of  yeast (Y187)
      ⇒ Trafo failed
September 1st, week 17
C. albicans & C. glabrata
- New ligation of 4.1 (ssr1 interacting peptide) (see Aspergillus)
      ⇒ Test via PCR, if insert is in the vector (bigger fragment) → see Aspergillus
- Sequencing: again 10 + 15 (+ BP1)
      ⇒ Results: 10 and 15 are exactly the same interacting peptide
A. fumigatus & A. nidulans
A. fumigatus
- Measure DNA conc. pGP172 → 350ng/µl
- Use 1.000 ng for restriction. 16 µl mixture 4 µl buffer (Tango) 2 µl Sma1 incubation for 120 minutes at 30 °C. Then add 2 µl of BamH1 for more 2 h. dephosphorylation with SAP for additional 30 minutes.
- Measure OD of o/N culture Y187 (5.4) and AH109 (18.4). Inoculation of 50 ml YPAD 0.54 ml and 1.89 ml
- AH109 0.7 OD600 follow long term storage protocol. Freeze at -80°C there was no sorbitol inside
- purification of pGP172 - nanoDrop: 50ng
- Ligation over night at 16 °C
- cut pGP172 + fragments cc_number
- Inoculation of AH109/Y187 for storage cultures
- PCR with PIGY9 and 10 with IG099 and IG1003
- Measure OD600 from o/N cultures; AH109 [12.9] and Y187 [12.1]. →use 0.78 ml and 0.82 ml of the strains to inoculate to an OD600 of 0.2. After 3 hours follow long term storage protocol
- Purification of PCR product from 9 and 10
- Cut pIGY9 and 10 with Hpa1 and BamH1
Agarose Gel (1%)
- Prep. of pGP172cut and clean: Sma1 and BamH1 - SAP 50 µl sample and purification
- Transformation of all ligations → nothing on plates
- Inoculation of BL21blue and pGP172.
- Restriction of pGP172 again for ligation of inserts. 1:1 ratio, Ligation with three inserts 3, 4.1 and 15.
- Transformation into DH5α.
- Growth on plates: Pick 10 colonies from each transformation plates grow over day 37°C with respective antibiotics and start plasmid isolation - stopped at 2-prop step over night [also pGP172 3x purification].
- New Oligos: Check PCR with T7p and T7t (52°) for strep
- PCR (12,5µl samples) with strep-4.1 and strep-NK(15/3?)
Agarose Gels (1%):
      ⇒ Bands in 3, 4, 5, 7, 8, 9 are positive
      ⇒ Bands in 4 and 8 are positive
- Inoculations from overnight culture a 500 ml flask with 50 ml LB and shake it till OD600 0.35 and follow fast comp. cell protocol.  Use 2 µl from each plasmid and incubation over 2 days at 37°C.
S. cerevisiae
- Spot positive clones on SC-LWH plates with 3mM 3-AT
- Wash cells from plates and isolate plasmids
- Transformation into DH5α
T. gondii
- Spot positive clones on SC-LWH plates with 3mM 3-AT
- Wash cells from plates and isolate plasmids
- Transformation into DH5α
August 25th, week 16
C. albicans & C. glabrata
- PCR with Primers IG1003 and IG0099 → product that can be sequenced and cut with enzymes for strep tag plasmid insertion (samples of prey from 3, 13 and 15)
Agarose gel (1%):
      ⇒ Bands in lines 1, 2, 3, 4, 5 6, 9 and 11 have the right size and are first purified and then send for sequencing with Primer IG1003 (rev)
- Prey samples of 3A.1, 3A.2, 3A.3, 3B.1, 3B.2, 13.3, 15 digested with BamHI and HpaI
      ⇒ Clear difference between cut and uncut; also cut bands are very weak
- Results of sequencing (3, 13, 15):
      ⇒ 3A.1, 3A.2, 3A.3 are the same peptide
      ⇒ 3B.1 and 3B.2: sequences are not useful
      ⇒ 13.3 and 13.4: same peptide
      ⇒ 9: one interacting peptide
      ⇒ 10 + 15: same interacting peptide→?
- New test expression (see Aspergillus)
A. fumigatus & A. nidulans
A. fumigatus
- centrifuge 5 min at 13 krpm - freeze at -20 °C
- Preparation of Amp and Cm plates, streak out of used strains on Amp-plates
- SDS-gels prepared and stored in PLP 1x buffer at 4 °C
- staining of sds gel → empty
- PCR with fragments (3, 13, 15) (Agarose gel see Candida)
      ⇒ Purification of: 13.3 15 3.1A 3.2A 3.3A 3.1B 3.2B
- purified PCR products for restriction, ligation and transformation
- Preparation of 10 x PLP buffer and new SDS-gels
- Loading of 5 µl sample onto the gels→ expected size empty ~11.5 kDa
- New test expression at 30 °C and test also supernatant.
- Inoculation over night cultures of desired strains.
- Toxicity cannot be the case, Protein G subunit backbone was shown to hide this
- Streak out DB3.1 strains containing pDONR and pDEST
- Inoculation of over day cultures to OD600 of 0.1. Grow for 90 minutes at 30 °C
- Measure OD600 and take sample (safe also the supernatant)
- Add IPTG and take samples after 1, 2, 3 hours.
- Follow test expression protocol
- PCR with strep-tagged proteins T7 promotor / terminator
Agarose Gel (1%):
- cut again pGP172 (520 ng and 1080 ng) with Sma1 and BamH1 (Tango Buffer for 2 h)
- pick 5 different colonies from strep tagged inserts into DH5α → colony PCR
      ⇒ false inserts ☹
- Transformation of ligation into DH5α
A.nidulans
- Repeat yest transformation in Y187 with the genes rodA and npc2
- isolate plasmids
S. cerevisiae
- Y2H in semi solid media with the two genes
T. gondii
- Y2H in semi solid media
August 18th, week 15
C. albicans & C. glabrata
PCR with different plasmid isolation samples for prey amplification (IG1003 and IG0099; 52,5 °C)
Agarose gel (1%):
      ⇒ Bands in all samples
      ⇒ Purification and restriction with HpaI and BamHI
- Restriction of of 0,1 µl (14500ng/µl) pGP172 with SmaI and BamHI o/n 16°C
- ligation of cut cleaned PCR prey products with pGP172 (10ng/µl)  4µl of each with 1µl T4 Ligase in 20 µl sample
- Transformation into DH5α
- Again: plasmid isolation and PCR with T7 promotor and terminator primer on plasmids
Agarose gel (1%):
      ⇒ In lines 4 to 9 seems to be the right fragment; 2 and 3 are wrong
- Transformation into BL21
- Test-expression of strep-tag strains: 4.1A, 4.1B, 5A, 5B, 14.5A, 14.5B, 14.8A, 14.8B
- Preparation of SDS gels
- PCR with Primers IG1003 and IG0099 → product that can be sequenced and cut with enzymes for strep tag plasmid insertion (samples of prey from 3, 13 and 15)
Agarose gel (1%):
      ⇒ Bands in lines 1, 2, 3, 4, 5 6, 9 and 11 have the right size and are first purified and then send for sequencing with Primer IG1003 (rev)
- Prey samples of 3A.1, 3A.2, 3A.3, 3B.1, 3B.2, 13.3, 15 digested with BamHI and HpaI
      ⇒ Clear difference between cut and uncut; also cut bands are very weak
- Results of sequencing (3, 13, 15):
      ⇒ 3A.1, 3A.2, 3A.3 are the same peptide
      ⇒ 3B.1 and 3B.2: sequences are not useful
      ⇒ 13.3 and 13.4: same peptide
      ⇒ 9: one interacting peptide
      ⇒ 10 + 15: same interacting peptide→?
A. fumigatus & A. nidulans
A. fumigatus
-  PCR from diff. plasmids for prey amplification 52,5 °C (Agarose Gel see Candida)
      ⇒ Worked for all
- Purification of prey pcr products, restriction with HpaI and BamH1
- restriction of 0.1 µl (14500 ng/µl) pGP172 with Sma1 and BamH1 o/n 16 °C
- ligation of cutted cleaned PCR prey products with pGP172 (10 ng/µl);  4 µl of each with 1 µl T4 Ligase in 20 µl sample
- PCR (100 µl vol) with sample 5 and 4.1 to increase amount of product
Agarose gel(1%)
- Transformation of strep-prey plasmids worked. over day cultures in LB at 42 °C for plasmid isolation and colony PCR
- Inoculation of E. coli BL21blue for fresh comp cell to transform strep plasmids
- pGP172 o/N 16 °C restriction compared with 0,1 µl and 1 µl non cut plasmid
- nothing in colony PCR
- pellets from over day cultures
- Plasmid isolation
NanoDrop:
- PCR with T7 promotor and terminator primer on plasmids (Agarose gel see Candida)
- LB medium, BL21 blue competent cells, transformation with plasmids
- Transformation of BL21blue with the plasmids worked
- Inoculation of 10 ml LB medium incubation for 2 h at 37 °C
- Take samples before induction and after 1 - 2 - 3 h [ 100/OD600 ]
- centrifuged 5 min at 13 krpm → freeze at -20°C
- Preparation of Amp and Cm plates, streak out of used strains on Amp-plates
- SDS-gels prepared and stored in 1x PLP buffer at 4 °C
A. nidulans
- Repeat transformation of isolated plasmids out of the 1st Y2H experiment in E. coli DH5α
- Plasmid isolation out of E. coli DH5α
- NanoDrop:
- Repeat transformation in Y187 on SC-Leu medium - failed
August 11th, week 14
C. albicans & C. glabrata
- Yeast Plasmid isolation of preys for: 01, 02, 06, 08, 12, 13, 15, 16 (pIGX)
NanoDrop:
- Preparation of fresh competent cells
- E. coli Transformation: supercompetent cells and fresh competent cells are transformed with different plasmids (see table)
- Results of E. coli transformation:
      ⇒ Plasmid isolation and transformation into Yeast Y187 → but no colonies
A. fumigatus & A. nidulans
A. nidulans
- Repeat transformation in Y187 on SC-Leu medium - failed
- Repeat transformation in Y187 on SC-Leu medium- failed
- running out of isolated plasmids out of E. coli DH5α for Yeast transformation
August 4th, week 13
A. fumigatus & A. nidulans
A. fumigatus
-  O/N cultures from E. coli prey colonies to isolate plasmids again for restriction and ligation into pGP172
A. nidulans
- Transformation in E. coli DH5α
- Plasmid isolation out of E. coli DH5α
- Transformation in Y187 - failed
July 28th, week 12
C. albicans & C. glabrata
- Yeast transformation (Y187) from preys of 01, 02, 02C, 03A+B, 06, 07, 13, 15, 16, 16C
      ⇒ no colonies for 02C, 06 and 16C
- Plasmid multiplication and isolation of prey plasmids from 04, 05, 09, 10, 14
S. cerevisiae
- Yeast transformation (AH109) of pIGB22 and 21
T. gondii
- Yeast transformation (AH109)
July 21st, week 11
C. albicans & C. glabrata
- Evaluation of Sequencing:
- Bad/ not useful sequences: 9, 10, 14.1, 14.3, 14.4, 14.7, 4.2 (4.2→ new PCR necessary, wrong fragment)
- Peptides from: 4.1, 5, 14.5, 14.6, 14.8 (=14.10), 14.9
A. fumigatus & A. nidulans
A. fumigatus
-  pick colonies = o/n cultures for plasmid isolation
- plasmid- prep. of pGP172 acc. to the protocol
A. nidulans
- 1st Y2H experiment with the genes rodA, xlnA, npc2, slno1 in semi solid medium - worked only for the plasmids pIGA17 (rodA) and pIGA19 (npc2)
- after growing the yeast colonies are washed from plates
- plasmid isolation
July 14th, week 10
C. albicans & C. glabrata
- Confirmed interactions from robot Y2H are used for Plasmid isolation (prey): 4.1; 4.2; 5; 9; 10; 14.1; 14.3; 14.4; 14.5; 14.6; 14.7; 14.8; 14.9 and 14.10 + Isolation of interacting preys of AIG003, 07, 08, 12, 13
- NanoDrop
- additional plasmid purification for sequencing of plasmids 4.1, 4.2, 5, 9, 10, 14.1
- all plasmids are used in a PCR with Primers IG1002 (1170_fwd) and IG1003(1170_rev) (56°C)
Agarose gel (1%):
      ⇒ PCR products with IG1003 for sequencing (line 3→ fragment bigger than others, but also sequenced)
- Transformation E. coli DH5α with 03, 07, 08, 12, 13
- Plasmid isolation of E. coli 02, 06, 15, 16, 2C, 16C
NanoDrop:
A. fumigatus & A. nidulans
A. fumigatus
- Test of mated colonies from robot Y2H on X- α-gal plates
- 3rd Y2H: after spotting, wash colonies from plates and plasmid isolation
July 7th, week 9
C. albicans & C. glabrata
- Plasmid isolation of prey interactions of 02, 06, 15, 16 and transformation into E. coli (Amp plates)
- Another Y2H is performed with AIG004, 05, 09, 10, 14 and YIG04, 05, 09, 10, 14 to confirm the interactions we saw in the first Y2H (using a robot ); testing on SC-LWH plates with 3mM 3-AT and X-α-Gal plates
A. fumigatus & A. nidulans
A. fumigatus
- Plasmid isolation of the genes 02, 06, 15, 16
- NanoDrop:
- 3rd Y2H is performed with the strains AIG003, 07, 08, 12 and 13
Titerplates:
- Robot Y2H with 04, 05, 09, 10, 14 (see Candida)
- Transformation of 02, 06, 15, 16 pool into DH5α
      ⇒ Only grown with single colonies → again with different concentrations
June 30th, week 8
C. albicans & C. glabrata
- 1st) Transformation in yeast: some colonies are grown, but not for all → tried transformation again
- Plasmid isolation and saving of prey strains on SC-L plates
- 2nd) Plasmid isolation from yeast for prey plasmids interacting with AIG002, AIG006, AIG015, AIG016 (pIGX plasmids)
- Transformation in DH5α → not successful (next week again)
- Third Y2H screening of AIG003 (sim1), 07 (pir3), 08 (utr2), 12(ecm33), 13 (sun1)
A. fumigatus & A. nidulans
A. fumigatus
- Yeast Trafo into Y187 again
- 2nd Y2H: Plasmid isolation for Transformation into E. coli (pIGZ)
- Safe prey interacting partners on SC-L
- Preparation of new AA mixture
T. gondii
- Repeat LR reaction and transformation into DH5α
      ⇒ Trafo worked
- Cracking of E. coli cells → right plasmid in cells
- Plasmid isolation (2528,7 ng/µl)
June 23rd, week 7
C. albicans & C. glabrata
- Spotting of second Y2H (AIG002 (tos1), 06 (scw4), 15 (als3), 16 (rodA)) (picture)
Figure: Different interacting and matted colonies are spotted onto the plate(Example)
- 1st) Yeast colonies from first spotting are solved in water and centrifugated to pellets
- Isolation of plasmids from pellets (see protocol ) (pIGX plasmids)
- Transformation in DH5α (70 µl of pooled plasmid DNA plated on Amp-plates)
- Thin layer of E. coli cells are washed from plates with 1,5 ml of sterile water; centrifugation (4krpm, 5min), plasmid isolation (pIGY plasmids)
- Transformation into yeast (Y187) cells. Plated onto SC-L plates (YIG strains)
- 2nd) Yeast colonies from second spotting are solved in water and centrifugated to pellets → frozen
A. fumigatus & A. nidulans
A. fumigatus
- Preparation of yeast plasmid prep. Solutions P1, P2, and N3
- 1st Y2H: Yeast colonies are washed from plates; plasmid isolation, Transformation into E. coli; E. coli plasmid isolation and again transformation into yeast (Y187) (YIG strains) (see Candida)
- 2nd Y2H: yeast colonies are washed from plates and centrifuged to pellets → frozen
A. nidulans
- E. coli plasmid isolation of cells with LR product
- PCR on plasmids → no positives
- Repeat LR reactions and transformation
- Again plasmid isolation and PCR
Agarose Gel (1%):
      ⇒ Right bands in: 1, 2, 4, 5, 6, 7, 8, 9, 16
- Plasmid isolation of all positive clones
S. cerevisiae
- PCR on isolated plasmids
      ⇒ Nothing on Gel
- LR reaction was repeated for CWP1, CWP2, SED1, MID2
- Transformation of DH5 α
      ⇒ Worked for all, except CWP1
      ⇒ Cracking failed
- Colony PCR
- NanoDrop:
T. gondii
- LR reaction and transformation into DH5α
      ⇒ Trafo didn’t work
June 16th, week 6
C. albicans & C. glabrata
- Second Y2H screening for (tos1), 06 (scw4), 15 (als3), 16 (rodA) (titerplates didn’t work)
- From first Y2H screening (AIG001, 04,05,09,10,14): 10 µl of each growing colony from semi solid medium is transferred into 100 µl of SC-LWH → 3µl spotted onto SC-LWH + 3mM 3-AT plates (picture) (rest: glycerol added and frozen)
Figure: Colonies are growing inside the semi-solid medium
A. fumigatus & A. nidulans
A. fumigatus
- Preparation for Y2H
- Second Y2H (see Candida)
- Spotting of colonies from first Y2H
A. nidulans
- Repeat LR reaction and Transformation in fresh competent DH5α cells
      ⇒ Trafo worked
- Cracking
      ⇒ No positives → cracking may have failed
      ⇒ Try with colony PCR → failed
S. cerevisiae
- Transformation of DH5α with LR product
      ⇒ Trafo worked for all, except CWP2
- Cracking of the colonies
      ⇒ No positives
- Isolation of plasmids
T. gondii
- Repeat BP reaction and transformation into DH5α
      ⇒ Trafo worked
- E. coli cracking
Agarose Gel (1%):
- Plasmid isolation of ama1 (pIGA25); NanoDrop: 1308,3 ng/µl
June 9th, week 5
C. albicans & C. glabrata
- No growth on auto-activity plates
- Preparation of  media for Y2H
- First Y2H screening (semi solid medium) of AIG001 (mp65); AIG004 (ssr1); AIG005 (pir4), AIG009 (crf2full); AIG010 (crf2act); AIG014 (als1);  - Titerplates on SC-LW medium
Titerplates:
A. fumigatus & A. nidulans
A. fumigatus
- LR reaction and transformation of pIGA11 and 13
- Transformation into DH5α
      ⇒ Trafo worked!
- Transformation into Yeast AH109
- First Y2H of AIG09 and AIG10 (see Candida notebook)
A. nidulans
- Repeat BP reactions and transformation into DH5α
      ⇒ Trafo worked
- Cracking (see Candida)
- o/n cultures of positive clones
Agarose Gel (1%) of cracking:
- Plasmid isolation
- NanoDrop:
- LR reaction with the 4 plasmids
- Transformation into fresh competent DH5α cells
      ⇒ Trafo failed
S. cerevisiae
- Cracking for CWP1, CWP2, SED1, MID2
Agarose gel (1%):
      ⇒ All of the E. coli strains contain the right plasmid (band 4 is negative due to pipetting mistake)
      ⇒ Sample 2 of each gene was chosen for plasmid isolation
- NanoDrop:
- LR reactions
June 2nd, week 4
C. albicans & C. glabrata
- LR reactions with pDEST and pIGA-plasmids and Transformation into DH5α
- Transformation was successful → Cracking + 4 mL LB+ kan o/n cultures
Agarose gels (1%):
- Samples 1 (mp65), 6 (tos1), 7(sim1), 11(ssr1), 13 (pir4) and 15 (scw4) are used for plasmid isolation.
- Samples 2 (pir3), 4 (utr2), 6 (crf2 full), 9 (crf2 act) and 13 (ecm33) were used to isolate the plasmids.
- The samples from 10-12 (bglCno) were cracked again.
- Samples 9 (als3) and 14 (eglC) are used to isolate the plasmids. The samples for sun1, als1 and rodA are cracked again.
- Cracking of each 3 samples of 11(bglEno), 13(sun1), 14(als1), 16(rodA)
Agarose gels (1%):
- Samples 9 (als1) and 12 (rodA) are used to isolate the plasmids. No results for bglEno and sun1 (A. fumigatus)
- Plasmid isolation
- Nano-Drop results:
- Transformation in Yeast (AH109) → plated on SC-W plates
- Transformation worked → single colonies are streaked on SC-W and SC-HW +3mM 3AT (auto-activity test) plates
A. fumigatus & A. nidulans
A. fumigatus
- LR reaction with pDEST and pIGA- plasmids
- Transformation into DH5α
      ⇒ Trafo worked
- Validation of positive clones via E. coli cracking (from each plate 3 colonies) (pictures see Candida)
- 4 ml LB with kanamycin inoculated for positive clones
- Plasmid isolation (pIGB plasmids)
- NanoDrop: (pIGB11 and 13 were empty)
- Yeast (AH109) transformation (AIG strains)
      ⇒ Trafo worked
- Test on autoinduction (His auxotrophy) on SC-HW + 3mM 3-AT
      ⇒ No autoinduction (grid plate see Candida labbook)
A. nidulans
- Amplification of GOI (size)
      ⇒ A. nidulans:
             rodA (467 bp)
             xlnA (749 bp)
             sho1 (934 bp)
             npc2 (627 bp)
- Purification of PCR products
- BP reaction into pDONR vector
- Transformation into DH5α
      ⇒ Trafo failed
T. gondii
- Amplification of the gene of interest via PCR:
      ⇒ Toxoplasma:  ama1 (Annealing temperature) 54°C
Agarose Gel(1%):
      ⇒ Purification of fragment. NanoDrop  → 157,6 ng/µl
- BP reaction and transformation into DH5α
      ⇒ Trafo didn’t work
May 26th, week 3
C. albicans & C. glabrata
- Transformation of BP reaction into fresh competent cells (50 µl+ rest)
- Transformation Efficienty test for supercompetent cells with iGEM kit (control with fresh competent cells)
   ⇒ Did not work → Kit is not useful !
- Transformation of all BP reactions worked, except for als1 and als3
- 3 colonies from each transformation were picked and dissolved in 10 µl H2O. 5µl of this were used for cracking and 5 µl to incubate a 4 ml LB culture over day.
Agarose Gels:
Samples A (from each plasmid):
Samples B (from each plasmid):
Samples C (from each plasmid):
      ⇒ Plasmids can be found in almost all cultures → one positive culture for each plasmid was chosen and centrifuged (5 min, 5000rpm)
- PCR for als1 and als3 were repeated:
      ⇒1+3 purification (eluted in 70 µl H2O)
      ⇒ Nano Drop:
            1        8,8 ng/µl
            2        24,3 ng/µl
→ PCR product from first PCR for als1 and from second PCR for als3 were used for the BP reaction (+Trafo)
    - Plasmid isolation of the samples from C. albicans, C. glabrata and A. Fumigatus (see protocol ) (o/n cultures)
    - NanoDrop results:
A. fumigatus & A. nidulans
- BP reactions of PCR products into pDONR207 and transformation into DH5α
- Transformation into E. coli DH5α cells
      ⇒ Trafo worked
- Cracking of eight different colonies from each transformation (Agarose gel pictures see Candida notebook)
- 4 ml LB + gentamycin inoculated with positive clones
- Plasmid isolation (pIGA plasmids)
- NanoDrop:
S. cerevisiae
- Amplification of the genes of interest via PCR:
      ⇒ S. cerevisiae:
            CWP1 (700 bp)
            CWP (300 bp)
            TIR1 (800 bp)
            SED1 (1000 bp)
            TIR4 (1500bp)
            MID2 (1200 bp)
Agarose Gel (1%):
      ⇒ 1,2,3,4,7,8,11,12 worked; 9 + 10 will be repeated because of the two separate bands
- Purification of right fragments
- NanoDrop:
- BP reaction of CWP1, CWP2, SED1, MID2
- Transformation into DH5α
      ⇒ Trafo worked
- Repeat PCR for TIR1 and TIR4 fragments → no bands in agarose gel
May 19th, week 2
C. albicans & C. glabrata
- Amplification of the genes of interest via PCR
      ⇒ C. albicans: mp65, als1, tos1, sim1, als3
      ⇒ C. glabrata: ssr1, pir4, scw4, pir3, utr2
      ⇒ Annealing temperature: 56 °C
- Agarose Gel (1%):
      ⇒ Band 5 (als3) is a little bit weak → but PCR product is also purified and further used
     ⇒ Purification of the PCR products with the Nucleo Spin® Gel and PCR Clean-up kit from Macherey-Nagel
      ⇒ Nano Drop Values were not useful
- Preparation of supercompetent E. coli DH5α cells (see protocol )
- BP reaction of all genes (see protocol )
A. fumigatus & A. nidulans
- Amplification of the genes of interest via PCR (→ Annealing Temperature)
⇒ A. fumigatus:
       crf2 full (mature peptide AA: 21-416)        (58°C)
       crf2 act (active side AA: 21-190)        (56°C)
       bglE no ( no GlcNAc outside AA: 594-908)        (58°C)
       ecm33 ( active protein AA: 20-372)        (58°C)
       sun1 (sequence between GlcNAc AA: 80-1133)        (58°C)
       rodA (AA: 19 – 159)        (56°C)
       prm1 (extracellular AA: 419-603)        (58°C)
       crf1 (active side before GlcNac AA: 20-310)        (58°C)
Agarose gel (1%):
⇒ Purification of the PCR products with the Nucleo Spin® Gel and PCR Clean-up kit from Macherey-Nagel
May 12th, week 1
C. albicans & C. glabrata
- Test of supercompetent E. coli cells with the transformation efficiency kit (from iGEM)
    ⇒ Did not work! No colonies
- Got cDNA of Candida albicans and Candida glabrata from Oliver Baders Lab :
       Candida albicans strain SCS314
       Candida glabrata strain CBS138
    ⇒ Both high concentrations : dilution (10 µl DNA + 990 µl H2O)
A. fumigatus & A. nidulans
- Preparation of competent cells E. coli DB3.1 (mutated gyrase, resistant against ccdB) with CaCl2 (see Protocol for fresh comp. cells )
- Transformation with pDONR207 and pDEST (Plating of 50µl and rest on LB+ Cm plates, incubation over night at 37°C)
       ⇒ Trafo worked! Single colonies in 4 ml LB + 4µl Cm
       ⇒ Also saving of the strains on plates
- Plasmid isolation with M&N Kit, Elution in 70µl dH2O
- Nano Drop:
Previous Next
Retrieved from " http://2014.igem.org/Team:Goettingen/notebook_wetlab "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
