<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Goettingen_project_overview_project_wetlab"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Goettingen/project_overview/project_wetlab">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Goettingen/project_overview/project_wetlab&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Goettingen/project_overview/project_wetlab&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Goettingen/project_overview/project_wetlab&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Goettingen/project_overview/project_wetlab" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Goettingen/project overview/project wetlab</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="globalWrapper"><DIV id="header"><DIV id="headerNavi"><DIV id="menuBar"><UL><LI><A href="/Team:Goettingen">Home</A></LI><LI><A href="/Team:Goettingen/project_overview_navi">Project ▾ </A></LI><LI><A href="/Team:Goettingen/project_overview">Background</A></LI><LI><A href="/Team:Goettingen/project_overview/project">Our project</A></LI><LI><A href="/Team:Goettingen/project_overview/project_wetlab">Results</A></LI><LI><A href="/Team:Goettingen/project_overview/perspectives">Perspectives</A></LI><LI><A href="/Team:Goettingen/project_overview/project_biobrick">BioBricks</A></LI></UL></DIV><LI><A href="/Team:Goettingen/notebook_overview">Notebook ▾ </A></LI><LI><A href="/Team:Goettingen/notebook_safety">Safety</A></LI><LI><A href="/Team:Goettingen/notebook_wetlab">Wet lab</A></LI><LI><A href="/Team:Goettingen/notebook_drylab">Dry lab</A></LI><LI><A href="/Team:Goettingen/notebook_materials">Materials</A></LI></DIV><LI><A href="/Team:Goettingen/protocol">Protocols</A></LI><LI><A href="/Team:Goettingen/team_overview">Our Team ▾ </A></LI><LI><A href="/Team:Goettingen/team_members">Team members</A></LI><LI><A href="/Team:Goettingen/team_instructors">Instructors</A></LI><LI><A href="/Team:Goettingen/team_acknowledges">Acknowledgements</A></LI><LI><A href="/Team:Goettingen/team_sponsors">Sponsors</A></LI></DIV><LI><A href="/Team:Goettingen/outreach_overview">Outreach ▾ </A></LI><LI><A href="/Team:Goettingen/outreach_human_practices">Human Practice</A></LI><LI><A href="/Team:Goettingen/outreach_gallery">Gallery</A></LI><LI><A href="/Team:Goettingen/outreach_videos">Video</A></LI><LI><A href="/Team:Goettingen/outreach_presentation">Presentation</A></LI><LI><A href="/Team:Goettingen/outreach_newsletter">Paris Bettencourt Newsletters</A></LI><LI><A href="https://2014.igem.org/Team:Goettingen/outreach_germanproposal">German teams' proposal</A></LI></DIV><LI><A href="/Team:Goettingen/achievements">Achievements</A></LI></DIV></DIV></DIV><DIV class="proLP"><H3>Project</H3><UL><LI><A href="https://2014.igem.org/Team:Goettingen/project_overview_navi">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Goettingen/project_overview">Background</A></LI><LI><A href="https://2014.igem.org/Team:Goettingen/project_overview#burden">The global burden of fungal infections</A></LI><LI><A href="https://2014.igem.org/Team:Goettingen/project_overview#tools">Current diagnostic tools</A></LI></UL><LI><A href="https://2014.igem.org/Team:Goettingen/project_overview/project">Our project</A></LI><LI><A href="https://2014.igem.org/Team:Goettingen/project_overview/project_wetlab">Results</A></LI><LI><A href="https://2014.igem.org/Team:Goettingen/project_overview/project_wetlab#wetlab">Wetlab</A></LI><LI><A href="https://2014.igem.org/Team:Goettingen/project_overview/project_wetlab#drylab">Drylab</A></LI><LI><A href="https://2014.igem.org/Team:Goettingen/project_overview/perspectives">Perspectives</A></LI><LI><A href="https://2014.igem.org/Team:Goettingen/project_overview/perspectives#diagnosis">Diagnosis</A></LI><LI><A href="https://2014.igem.org/Team:Goettingen/project_overview/perspectives#therapeutics">Therapeutics</A></LI><LI><A href="https://2014.igem.org/Team:Goettingen/project_overview/project_biobrick">BioBricks</A></LI></DIV><DIV id="subpage"><DIV class="proRP" id="rpart1"><DIV class="trans_zh"><A href="https://2014.igem.org/Team:Goettingen/project_overview/project_wetlab_ZH"><B>中文</B></A></DIV><H1 id="wetlab">Results: Wetlab</H1><H2 id="approach">Approach</H2><P>Our experimental approach can be summarized as follows:</P><OL><LI><P>1. Organisms and surface proteins selection</P></LI><LI><P>2. Surface protein amplification and cloning</P></LI><LI><P>3. Yeast two-hybrid assay with an existing peptide library</P></LI><LI><P>4. Peptide functionalization</P></LI></OL><H2 id="wlresults">Results summary</H2><P>We chose the five fungi <I>Aspergillus fumigatus</I>, <I>A. nidulans</I>, <I>Candida glabrata</I>, <I>C. albicans</I> and <I>Saccharomyces cerevisiae</I> as well as the parasite <I>Toxoplasma gondii</I> for our project. In the first step, we amplified all in all 35 different genes encoding surface proteins of the mentioned organisms in a standard PCR reaction (compare table 1). We were able to transform 20 of our 35 genes into a so called bait vector (contains a DNA-binding domain) and performed the Yeast-Two-Hybrid (Y2H) screening with them. Thereby, we accomplished to find more than 3000 probable interaction partners for our surface proteins. </P><P><B>Table 1| Selected surface proteins and the corresponding organism.</B></P><CENTER><TABLE class="table1" style="width:700px"><TBODY><TR style="font-weight:bold;"><TH>Gene</TH><TH>Organism</TH></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/AJ010064.1" target="1"><I>mp65</I></A></TD><TD><I>C. albicans</I></TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/XM_716672.1" target="1"><I>tos1</I></A></TD><TD><I>C. albicans</I></TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/NW_139623.1" target="1"><I>sim1</I></A></TD><TD><I>C. albicans</I></TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/XM_712917.1" target="1"><I>als1</I></A></TD><TD><I>C. albicans</I></TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/U87956.1" target="1"><I>als3</I></A></TD><TD><I>C. albicans</I></TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/CAGL0H06413g" target="1"><I>ssr1</I></A></TD><TD><I>C. glabrata</I></TD></TR><TR class="con"><TD colspan="3">Click to expand</TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/CAGL0I06160g " target="1"><I>pir4</I></A></TD><TD><I>C. glabrata</I></TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/CAGL0G00308g " target="1"><I>scw4</I></A></TD><TD><I>C. glabrata</I></TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/CAGL0M08492g " target="1"><I>pir3</I></A></TD><TD><I>C. glabrata</I></TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/CAGL0C02211g " target="1"><I>utr2</I></A></TD><TD><I>C. glabrata</I></TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/Q4WI45" target="1"><I>crf2full</I></A></TD><TD><I>A. fumigatus</I></TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/Q4WI46" target="1"><I>crf2act</I></A></TD><TD><I>A. fumigatus</I></TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/Q4WD56" target="1"><I>bgleno</I></A></TD><TD><I>A. fumigatus</I></TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/Q4WNS8" target="1"><I>ecm33</I></A></TD><TD><I>A. fumigatus</I></TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/Q4WGL5" target="1"><I>sun1</I></A></TD><TD><I>A. fumigatus</I></TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/P41746" target="1"><I>roda</I></A></TD><TD><I>A. fumigatus</I></TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/Q4WD56" target="1"><I>bgleex</I></A></TD><TD><I>A. fumigatus</I></TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/Q4WQ14" target="1"><I>prm1 b</I></A></TD><TD><I>A. fumigatus</I></TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/Q8J0P4   " target="1"><I>crf1</I></A></TD><TD><I>A. fumigatus</I></TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/AB243112.1" target="1"><I>roda</I></A></TD><TD><I>A. nidulans</I></TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/Z49892" target="1"><I>xlna</I></A></TD><TD><I>A. nidulans</I></TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/XM_658391.1" target="1"><I>npc2</I></A></TD><TD><I>A. nidulans</I></TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/XM_675875.1" target="1"><I>sho1</I></A></TD><TD><I>A. nidulans</I></TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/50952829" target="1"><I>eglc</I></A></TD><TD><I>A. nidulans</I></TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/Q4WG16" target="1"><I>eglc</I></A></TD><TD><I>A. fumigatus</I></TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/D37975.1" target="1"><I>CWP1</I></A></TD><TD><I>S. cerevisiae</I></TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/NM_001180025.1" target="1"><I>CWP2</I></A></TD><TD><I>S. cerevisiae</I></TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/NM_001183428.1" target="1"><I>TIR4</I></A></TD><TD><I>S. cerevisiae</I></TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/NM_001178902.1" target="1"><I>TIR1</I></A></TD><TD><I>S. cerevisiae</I></TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/NM_001182221.1" target="1"><I>MID2</I></A></TD><TD><I>S. cerevisiae</I></TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/AB530985.1" target="1"><I>SED1</I></A></TD><TD><I>S. cerevisiae</I></TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/FJ593880.1" target="1"><I>ama1</I></A></TD><TD><I>T. gondii</I></TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/AY587208.1" target="1"><I>rom5</I></A></TD><TD><I>T. gondii</I></TD></TR><TR><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/AY704175.1" target="1"><I>rom4</I></A></TD><TD><I>T. gondii</I></TD></TR><TR class="con"><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/DQ459452.2" target="1"><I>gra7</I></A></TD><TD><I>T. gondii</I></TD></TR></TBODY></TABLE></CENTER><P>The results for all the genes we selected are summarized in this <A href="https://static.igem.org/mediawiki/2014/6/60/Goettingen2014-NumberS.pdf" target="1">file.</A></P><P>To exclude “false positive” interaction partners (e.g. caused by a very weak interaction) we picked grown colonies and transferred them onto plates with a selection medium that allows only yeast cells containing peptides with a strong affinity to our surface proteins to grow. After this procedure and several previous selection steps, we still had around 70% of our interaction partners from the first screening. </P><P>To have a manageable amount of samples, we chose 10 of the interaction partners (peptides) which we found in the Y2H screen for each of our 20 surface proteins and used them for further analysis.  The cells containing the plasmids were plated on selection plates and the Y2H screen was repeated to finally confirm the interaction between one of our surface proteins and a specific peptide. This time, we used a robot to fasten up the screen and to be able to analyze a large number of possible interactions at the same time. <B>Interactions could be verified in 70% of all tested interaction partners! </B>
In the last step, we tested the interaction of our surface proteins and the peptides in <I>in vivo</I> experiments.  Previously, we chose some of the peptides and sequenced them to get to know the sequence and to exclude the possibility that we isolated the same peptides in different samples. The sequences encoding the peptides were introduced into an over-expression vector for <I>E. coli</I>; thereby, we were able to get large amounts of our peptides which we called <I>PepTags</I> from then on. In the last step for the time being, we tested the fungi <I>A. fumigatus</I>, <I>A. nidulans</I> and <I>C. glabrata</I> with peptides, which are supposed to interact specific with one of their surface proteins<I> in vivo </I>using fluorescence microscopy. 
With the<I> in vivo </I>experiments we want to proof that the identified <I>PepTags</I> not only bind to their surface proteins in an Y2H screen, but also to living fungi. To visualize the fungal surface itself, we stained structures containing chitin and cellulose in the fungal cell wall with calcofluor (figure 1C and 2C). The first organism with whom the functionality of our system was tested was A. fumigatus and the <I>PepTag</I> No.13.  Binding of this <I>PepTag</I> to the fungi was visualized with an antibody fused to a fluorescent molecule (Chromeo 488, figure 1A and 2A).</P><CENTER><P style="font-size=8px;"><B>Figure 1| Fluorescence microscopy of <I>Aspergillus fumigatus</I> hyphae.</B> This figure shows an A. fumigatus hyphae incubated with <I>PepTag</I>13, antibody and stained with calcofluor. Under light with a wavelength of 488 nm, the complexes containing <I>PepTag</I>13 and antibody show fluorescence. The excitation of calcofluor at a wavelength of 358 nm allows the identification of the hyphal surface (blue light).  The second picture shows the hyphae in a bright field, while the merged picture demonstrates that the green fluorescence is located in the same area as the blue one.</P></CENTER><P>We were able to show that our peptide binds specifically to the surface of <I>A. fumigatus</I>, which is visualized by the green dots along the outer surface of the fungal cells. We also tested <I>PepTag</I> No.13 with <I>A. nidulans</I> (figure 2) but could not see any fluorescent signal. That means that our <I>PepTag</I> No.13 binds not only specific to the fungal surface, but also specific to the surface protein Sun1 of <I>A. fumigatus</I>. Moreover, we could show that the antibody cannot bind unspecific to the fungal surface when the <I>PepTag</I> is missing . For further information <A href="https://2014.igem.org/Team:Goettingen/notebook_wetlab#week23">have a look at our lab-book.</A></P><P style="font-size=8px;">Figure 2| Fluorescence microscope picture of an <I>Aspergillus nidulans</I> hyphae. This figure shows an <I>A. nidulans</I> hyphae incubated with <I>PepTag</I> No.13, antibody and calcofluor. Upon irradiation with light of 488 nm, no fluorescence occurs (first picture). The excitation of calcofluor at a wavelength of 358 nm allows identification of the surface of the hyphae (blue light). The <I>PepTag</I> cannot bind to the surface of <I>A. nidulans</I> and seems to be specifically to the surface of <I>A. fumigatus.</I></P><P>Finally, it was possible to identify a small peptide which binds specific to the surface of fungal hyphae. Moreover this <I>PepTag</I> is specific for the chosen fungus.</P><VIDEO width="640" height="390" controls="" autoplay=""><SOURCE src="https://static.igem.org/mediawiki/2014/b/b2/Goettingen2014-fm-3d.mp4" type="video/mp4">Your browser does not support the video tag.</SOURCE><H1 id="drylab">Results: Dry lab</H1><H2 id="approach">Homology modelling</H2><P>We modeled the 3D structure of our peptide-containing scaffolds (domain B1 of protein G from <I>Staphylococcus aureus</I>) by homology modelling. We used the <A href="http://salilab.org/modeller/" target="1">Modeller library</A> to accomplish that; since the peptide is about 20-25 amino acids long, it can be modeled as a loop inside the B1 domain scaffold for which there are already a number of 3D structures available at the Protein Data Bank.</P><CENTER><P align="justify">This is the general homology modeling procedure we followed. All peptides were cloned inside a scaffold (B1 domain of protein G from <I>Staphylococcus aureus</I>. The peptides were short enough to be modeled as an internal loop by loop refinement; these structures should only be considered as a working model, since we do not have direct crystallographic data. <A href="https://2014.igem.org/Team:Goettingen/notebook_drylab">A detailed procedure can be found in this link.</A></P></CENTER><H2 id="dlresults">Results summary</H2><H3>Protein G scaffold</H3><P>The table 2 is a summary of the models we generated for our peptides inside the protein G scaffold. Their main purpose is to give us an idea if the calculated energy profile matches our assumption that the peptide is being exposed to the exterior of the scaffold. These models are in no way definitive, since we lack direct crystalographic data. As can be seen, all the structures show two distinct domains: one for the protein G scaffold and the other for the peptide.</P><P><B>Table 2| Homology modeling for the peptides in a protein G scaffold.</B></P><CENTER><TABLE class="table1" style="width:700px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Peptide</TH><TH style="font-size:80%;">Video*</TH><TH style="font-size:80%;">PDB</TH><TH style="font-size:80%;">Peptide location</TH><TH style="font-size:80%;">QMEAN score</TH><TH style="font-size:80%;">Overall DOPE score</TH><TH style="font-size:80%;">Energy profile</TH></TR><TR><TD style="font-size:80%;">Prey 3</TD><TD style="font-size:80%;"><VIDEO width="120" poster="" autoplay=""><SOURCE src="https://static.igem.org/mediawiki/2014/a/a1/Goettingen2014-Prey3.ogg" type="video/ogg">Your browser does not support HTML5 video.</SOURCE></VIDEO></TD><TD style="font-size:70%;"><A href="https://static.igem.org/mediawiki/2014/1/1e/Goettingen-Prey3-input.txt" target="1">Download</A></TD><TD style="font-size:80%;">K17-E42</TD><TD style="font-size:70%;">0.512</TD><TD style="font-size:70%;">-7053.966797</TD><TD style="font-size:70%;"><A href="https://static.igem.org/mediawiki/2014/3/37/GoettingenPrey3-profile.txt" target="1">Download</A></TD></TR><TR><TD style="font-size:80%;">Prey 4.1</TD><TD><VIDEO width="120" poster="" autoplay=""><SOURCE src="https://static.igem.org/mediawiki/2014/f/f1/Goettingen2014-Prey4.ogg" type="video/ogg">Your browser does not support HTML5 video.</SOURCE></VIDEO></TD><TD style="font-size:70%;"><A href="https://static.igem.org/mediawiki/2014/1/1b/Goettingen-Prey4-1.txt" target="1">Download</A></TD><TD style="font-size:80%;">K17-E44</TD><TD style="font-size:70%;">0.468</TD><TD style="font-size:70%;">-7246.956055</TD><TD style="font-size:70%;"><A href="https://static.igem.org/mediawiki/2014/0/0b/GoettingenPrey4-1-profile.txt" target="1">Download</A></TD></TR><TR><TD style="font-size:80%;">Prey 5</TD><TD><VIDEO width="120" poster="" autoplay=""><SOURCE src="https://static.igem.org/mediawiki/2014/f/f8/Goettingen2014-Prey5.ogg" type="video/ogg">Your browser does not support HTML5 video.</SOURCE></VIDEO></TD><TD style="font-size:70%;"><A href="https://static.igem.org/mediawiki/2014/8/86/Goettingen-Prey5.txt" target="1">Download</A></TD><TD style="font-size:80%;">K17-E44</TD><TD style="font-size:70%;">0.495</TD><TD style="font-size:70%;">-7501.071777</TD><TD style="font-size:70%;"><A href="https://static.igem.org/mediawiki/2014/1/10/GoettingenPrey5-profile.txt" target="1">Download</A></TD></TR><TR><TD style="font-size:80%;">Prey 13</TD><TD><VIDEO width="120" poster="" autoplay=""><SOURCE src="https://static.igem.org/mediawiki/2014/2/23/Goettingen2014-Prey13.ogg" type="video/ogg">Your browser does not support HTML5 video.</SOURCE></VIDEO></TD><TD style="font-size:70%;"><A href="https://static.igem.org/mediawiki/2014/8/8e/Goettingen-Prey13-input.txt" target="1">Download</A></TD><TD style="font-size:80%;&gt;">K17-E42</TD><TD style="font-size:70%;">0.535</TD><TD style="font-size:70%;">-7729.056152</TD><TD style="font-size:70%;"><A href="https://static.igem.org/mediawiki/2014/e/eb/GoettingenPrey13-profile.txt" target="1">Download</A></TD></TR><TR><TD style="font-size:80%;">Prey 15</TD><TD><VIDEO width="120" poster="" autoplay=""><SOURCE src="https://static.igem.org/mediawiki/2014/1/1d/Goettingen2014-Prey15.ogg" type="video/ogg">Your browser does not support HTML5 video.</SOURCE></VIDEO></TD><TD style="font-size:70%;"><A href="https://static.igem.org/mediawiki/2014/7/77/Goettingen-Prey15-input.txt" target="1">Download</A></TD><TD style="font-size:80%;">K17-E42</TD><TD style="font-size:70%;">0.441</TD><TD style="font-size:70%;">-7685.998535</TD><TD style="font-size:70%;"><A href="https://static.igem.org/mediawiki/2014/2/20/GoettingenPrey15-profile.txt" target="1">Download</A></TD></TR></TBODY></TABLE></CENTER><P style="font-size:10px"><B>*The peptide is shown in red and the scaffold in blue.</B></P><H3>GFP scaffold</H3><P>Table 3 shows the structures we generated for our peptides inside a GFP scaffold. In contrast to the models with the protein G scaffold, the GFP-peptide models show discontinuities.</P><P><B>Table 3| Homology modeling for the peptides in a GFP scaffold.</B></P><CENTER><TABLE class="table1" style="width:700px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Peptide</TH><TH style="font-size:80%;">PNG*</TH><TH style="font-size:80%;">PDB</TH><TH style="font-size:80%;">Peptide location</TH><TH style="font-size:80%;">QMEAN score</TH><TH style="font-size:80%;">Overall DOPE score</TH><TH style="font-size:80%;">Energy profile</TH></TR><TR><TD style="font-size:80%;">Prey 3</TD><TD style="font-size:70%;"><A href="https://static.igem.org/mediawiki/2014/8/8c/Prey3-gfp.pdb.txt" target="1">Download</A></TD><TD style="font-size:80%;">E103-E127</TD><TD style="font-size:70%;">0.43</TD><TD style="font-size:70%;">-26798.28125</TD><TD style="font-size:70%;"><A href="https://static.igem.org/mediawiki/2014/4/4a/Prey3-gfp.pdb.profile.txt" target="1">Download</A></TD></TR><TR><TD style="font-size:80%;">Prey 4.1</TD><TD style="font-size:70%;"><A href="https://static.igem.org/mediawiki/2014/8/89/Prey4-gfp.pdb.txt" target="1">Download</A></TD><TD style="font-size:80%;">E103-E127</TD><TD style="font-size:70%;">0.448</TD><TD style="font-size:70%;">-27891.212891</TD><TD style="font-size:70%;"><A href="https://static.igem.org/mediawiki/2014/d/dd/GFP-aSsr1.pdb.profile.txt" target="1">Download</A></TD></TR><TR><TD style="font-size:80%;">Prey 5</TD><TD style="font-size:70%;"><A href="https://static.igem.org/mediawiki/2014/b/bc/Prey5-gfp.pdb.txt" target="1">Download</A></TD><TD style="font-size:80%;">E103-E127</TD><TD style="font-size:70%;">0.483</TD><TD style="font-size:70%;">-26633.4375</TD><TD style="font-size:70%;"><A href="https://static.igem.org/mediawiki/2014/3/3c/Prey5-gfp.pdb.profile.txt" target="1">Download</A></TD></TR><TR><TD style="font-size:80%;">Prey 13</TD><TD style="font-size:70%;"><A href="https://static.igem.org/mediawiki/2014/0/09/Prey13-gfp.pdb.txt" target="1">Download</A></TD><TD style="font-size:80%;&gt;">E103-E127</TD><TD style="font-size:70%;">0.404</TD><TD style="font-size:70%;">-27075.160156</TD><TD style="font-size:70%;"><A href="https://static.igem.org/mediawiki/2014/a/a6/Prey13-gfp.pdb.profile.txt" target="1">Download</A></TD></TR><TR><TD style="font-size:80%;">Prey 15</TD><TD style="font-size:70%;"><A href="https://static.igem.org/mediawiki/2014/e/ed/Prey15-gfp.pdb.txt" target="1">Download</A></TD><TD style="font-size:80%;">E103-E127</TD><TD style="font-size:70%;">0.413</TD><TD style="font-size:70%;">-26713.355469</TD><TD style="font-size:70%;"><A href="https://static.igem.org/mediawiki/2014/0/00/Prey15-gfp.pdb.profile.txt" target="1">Download</A></TD></TR></TBODY></TABLE></CENTER><P style="font-size:10px"><B>*The peptide is shown in red and the scaffold in green.</B></P><H2>References</H2><OL><LI><P>1. N. Eswar, M. A. Marti-Renom, B. Webb, M. S. Madhusudhan,
D. Eramian, M. Shen, U. Pieper, A. Sali. Comparative Protein Structure
Modeling With MODELLER. Current Protocols in Bioinformatics, John Wiley
&amp; Sons, Inc., Supplement 15, 5.6.1-5.6.30, 2006.</P></LI><LI><P>2. M.A. Marti-Renom, A. Stuart, A. Fiser, R. Sánchez,
F. Melo, A. Sali. Comparative protein structure modeling of genes and genomes.
Annu. Rev. Biophys. Biomol. Struct. 29, 291-325, 2000.</P></LI><LI><P>3. A. Sali &amp; T.L. Blundell.
Comparative protein modelling by satisfaction of spatial restraints. J.
Mol. Biol. 234, 779-815, 1993.</P></LI><LI><P>4. A. Fiser, R.K. Do, &amp; A. Sali.
Modeling of loops in protein structures, Protein Science 9. 1753-1773,
2000.</P></LI></OL><DIV><A href="https://2014.igem.org/Team:Goettingen/project_overview/project" class="button_pre"><B>Previous</B></A><A href="https://2014.igem.org/Team:Goettingen/project_overview/perspectives" class="button_next"><B>Next</B></A></DIV></VIDEO></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Goettingen/project_overview/project_wetlab">http://2014.igem.org/Team:Goettingen/project_overview/project_wetlab</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Goettingen/project_overview/project_wetlab" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Goettingen/project_overview/project_wetlab" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Goettingen/project_overview/project_wetlab&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Goettingen/project_overview/project_wetlab&amp;oldid=378786" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>