<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Goettingen_notebook_wetlab"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Goettingen/notebook_wetlab">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Goettingen/notebook_wetlab&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Goettingen/notebook_wetlab&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Goettingen/notebook_wetlab&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Goettingen/notebook_wetlab" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Goettingen/notebook wetlab</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="globalWrapper"><DIV id="header"><DIV id="headerNavi"><DIV id="menuBar"><UL><LI><A href="/Team:Goettingen">Home</A></LI><LI><A href="/Team:Goettingen/project_overview_navi">Project ▾ </A></LI><LI><A href="/Team:Goettingen/project_overview">Background</A></LI><LI><A href="/Team:Goettingen/project_overview/project">Our project</A></LI><LI><A href="/Team:Goettingen/project_overview/project_wetlab">Results</A></LI><LI><A href="/Team:Goettingen/project_overview/perspectives">Perspectives</A></LI><LI><A href="/Team:Goettingen/project_overview/project_biobrick">BioBricks</A></LI></UL></DIV><LI><A href="/Team:Goettingen/notebook_overview">Notebook ▾ </A></LI><LI><A href="/Team:Goettingen/notebook_safety">Safety</A></LI><LI><A href="/Team:Goettingen/notebook_wetlab">Wet lab</A></LI><LI><A href="/Team:Goettingen/notebook_drylab">Dry lab</A></LI><LI><A href="/Team:Goettingen/notebook_materials">Materials</A></LI></DIV><LI><A href="/Team:Goettingen/protocol">Protocols</A></LI><LI><A href="/Team:Goettingen/team_overview">Our Team ▾ </A></LI><LI><A href="/Team:Goettingen/team_members">Team members</A></LI><LI><A href="/Team:Goettingen/team_instructors">Instructors</A></LI><LI><A href="/Team:Goettingen/team_acknowledges">Acknowledgements</A></LI><LI><A href="/Team:Goettingen/team_sponsors">Sponsors</A></LI></DIV><LI><A href="/Team:Goettingen/outreach_overview">Outreach ▾ </A></LI><LI><A href="/Team:Goettingen/outreach_human_practices">Human Practice</A></LI><LI><A href="/Team:Goettingen/outreach_gallery">Gallery</A></LI><LI><A href="/Team:Goettingen/outreach_videos">Video</A></LI><LI><A href="/Team:Goettingen/outreach_presentation">Presentation</A></LI><LI><A href="/Team:Goettingen/outreach_newsletter">Paris Bettencourt Newsletters</A></LI><LI><A href="https://2014.igem.org/Team:Goettingen/outreach_germanproposal">German teams' proposal</A></LI></DIV><LI><A href="/Team:Goettingen/achievements">Achievements</A></LI></DIV></DIV></DIV><DIV id="protocolOverview"><DIV class="proLP"><H3>Notebook</H3><UL><LI><A href="/Team:Goettingen/notebook_overview">Overview</A></LI><LI><A href="/Team:Goettingen/notebook_safety">Safety</A></LI><LI><A href="/Team:Goettingen/notebook_drylab">Dry lab</A></LI><LI><A href="/Team:Goettingen/notebook_wetlab">Wet lab</A></LI><LI><A href="/Team:Goettingen/notebook_wetlab">PepTag Team</A></LI><UL id="note_nav" style="padding-left:15px;"><LI class="collapsibleListTitle" style="cursor:pointer; padding-left:30px;">October</LI><UL class="collapsibleListBody"><LI><A href="#week23">Week 23</A></LI><LI><A href="#week22">Week 22</A></LI><LI><A href="#week21">Week 21</A></LI></UL><LI class="collapsibleListTitle" style="cursor:pointer;  padding-left:30px;">September</LI><UL class="collapsibleListBody"><LI><A href="#week20">Week 20</A></LI><LI><A href="#week19">Week 19</A></LI><LI><A href="#week18">Week 18</A></LI><LI><A href="#week17">Week 17</A></LI></UL><LI class="collapsibleListTitle" style="cursor:pointer; padding-left:30px;">August</LI><UL class="collapsibleListBody"><LI><A href="#week16">Week 16</A></LI><LI><A href="#week15">Week 15</A></LI><LI><A href="#week14">Week 14</A></LI><LI><A href="#week13">Week 13</A></LI><LI><A href="#week12">Week 12</A></LI></UL><LI class="collapsibleListTitle" style="cursor:pointer; padding-left:30px;">July</LI><UL class="collapsibleListBody"><LI><A href="#week11">Week 11</A></LI><LI><A href="#week10">Week 10</A></LI><LI><A href="#week9">Week 9</A></LI><LI><A href="#week8">Week 8</A></LI></UL><LI class="collapsibleListTitle" style="cursor:pointer; padding-left:30px;">June</LI><UL class="collapsibleListBody"><LI><A href="#week7">Week 7</A></LI><LI><A href="#week6">Week 6</A></LI><LI><A href="#week5">Week 5</A></LI><LI><A href="#week4">Week 4</A></LI></UL><LI class="collapsibleListTitle" style="cursor:pointer; padding-left:30px;">May</LI><UL class="collapsibleListBody"><LI><A href="#week3">Week 3</A></LI><LI><A href="#week2">Week 2</A></LI><LI><A href="#week1">Week 1</A></LI></UL></UL><LI><A href="/Team:Goettingen/notebook_wetlab/biobricks">BioBricks Team</A></LI><LI><A href="/Team:Goettingen/notebook_wetlab/gfp">GFP Team</A></LI></UL><LI><A href="/Team:Goettingen/notebook_materials">Materials</A></LI></DIV><DIV class="proRP" id="notebookWrap"><DIV id="week23"><H2>October 13th, week 23</H2><H4 class="collapsibleListTitle" id="lab_team2"><I>A. fumigatus &amp; A. nidulans</I></H4><P class="collapsibleListBody"><B><I>A. fumigatus</I></B>
-	Results of robot Y2H:<B>(red color indicates a positive interaction, a blue colony. The grey color means that the result is not clear and it could possibly be a positive colony)</B>      ⇒	Growth and blue color indicates that there is an interaction between surface protein and the peptag.      ⇒	to verify the interaction we will select on the peptag-plasmids and isolate them and transfer them into strep-vector (pGP172) for over expression in BL21(DE3)      ⇒	the isolated protein is then used for <I>in vivo</I> studies      ⇒	More <I>in vivo</I> experiments with the interacting peptide of Sun1 from <I>A. fumigatus</I>.<B><I>A. nidulans</I></B>      ⇒	<I>In vivo</I> experiment with the Sun1 interacting peptide from <I>A. fumigatus </I>with <I>A. nidulans </I>to prove that the peptide binds specifically to <I>A. fumigatus</I>.</P><H4 class="collapsibleListTitle" id="lab_team1"><I>C. albicans &amp; C. glabrata</I></H4><P class="collapsibleListBody">
-	Results of the robot Y2H (see <I>Aspergillus</I>)
-	New<I> in vivo </I>experiments with Ssr1</P></DIV><DIV id="week22"><H2>October 6th, week 22</H2><H4 class="collapsibleListTitle" id="lab_team2"><I>A. fumigatus &amp; A. nidulans</I></H4><P class="collapsibleListBody"><B><I>A. fumigatus</I></B>
-	Continue the robot Y2H: replicate colonies from selection medium to selection medium with X-α-Gal
-	First<I> in vivo</I> experiment with the interacting peptide of Sun1 from <I>Aspergillus fumigatus Strep</I>MAB Classic conjugated with Chromeo<SUP>TM</SUP> 488 <B><I>A. nidulans</I></B>
-	Continue the robot Y2H (plate on to SC-LWH + 3mM 3-AT plates and X-α-gal plates)</P><H4 class="collapsibleListTitle" id="lab_team1"><I>C. albicans &amp; C. glabrata</I></H4><P class="collapsibleListBody">
-	Continue the robot Y2H (plate on to SC-LWH + 3mM 3-AT plates and X-α-gal plates)
-	<I>In vivo</I> test with<I> Candida glabrata ssr1</I> – k.o. strain → no fluorescence, due to too low protein concentrations (only worked for Sun1 from <I>Aspergillus fumigatus</I>)</P><H4 class="collapsibleListTitle" id="lab_team3"><I>S. cerevisiae</I></H4><P class="collapsibleListBody">
-	Continue the <A href="https://2014.igem.org/Team:Goettingen/protocol_Colony#Y2H_robot">robot Y2H</A> (plate on to SC-LWH + 3mM 3-AT plates and X-α-gal plates)
</P><H4 class="collapsibleListTitle" id="lab_team4"><I>T. gondii</I></H4><P class="collapsibleListBody">
-	Continue the robot Y2H (plate on to SC-LWH + 3mM 3-AT plates and X-α-gal plates)</P></DIV><DIV id="week21"><H2>September 29th, week 21</H2><H4 class="collapsibleListTitle" id="lab_team2"><I>A. fumigatus &amp; A. nidulans</I></H4><P class="collapsibleListBody"><B><I>A. fumigatus</I></B>
-	 Robot Y2H to confirm interaction      ⇒	Tested samples: AIG001, 02, 03, 04, 05, 06, 09, 10, 12, 13, 14, 15, 17, 19
-	Grow AIG strains and isolated interaction partners (YIG strains) in selection medium over night at 30 °C
-	Pipet grown colonies in 96 well plate (200 µl each)
-	Preparation of robo-agar-plates
-	Use AIG 96 well to stamp to -trp plates, YIG for - leu plates and incubate for 2 days at 30 °C
-	Mating of grown colonies on YPAD plates
-	Replicate colonies from YPAD to selection plates
-	Replicate colonies from selction to selection medium<B><I>A. nidulans</I></B>
-	Robot Y2H to confirm interaction.</P><H4 class="collapsibleListTitle" id="lab_team1"><I>C. albicans &amp; C. glabrata</I></H4><P class="collapsibleListBody">
-	 Robot Y2H to confirm interaction (see <I>Aspergillus</I>)</P><H4 class="collapsibleListTitle" id="lab_team3"><I>S. cerevisiae</I></H4><P class="collapsibleListBody">
-	Inoculation of AIG021 + 22 and YIG021 + 22 for<A href="https://2014.igem.org/Team:Goettingen/protocol_Colony#Y2H_robot"> robot Y2H</A></P><H4 class="collapsibleListTitle" id="lab_team4"><I>T. gondii</I></H4><P class="collapsibleListBody">
-	Inoculation of AIG025and YIG025 for robot Y2H</P></DIV><DIV id="week20"><H2>September 22nd, week 20</H2><H4 class="collapsibleListTitle" id="lab_team1"><I>C. albicans &amp; C. glabrata</I></H4><P class="collapsibleListBody">
-	 Dialysis of purificated proteins (see <I>Aspergillus</I>)
-	Preparation for robot Y2H</P><H4 class="collapsibleListTitle" id="lab_team2"><I>A. fumigatus &amp; A. nidulans</I></H4><P class="collapsibleListBody"><B><I>A. fumigatus</I></B>
-	 Contamination on robot plates → Again
-	Prepare 10x PBS buffer and medium for<I> in vivo</I> tests
-	Boil dialysis tubes for 10 minutes and add EDTA. Cool them down and fill in proteins for dialysis. Dialyze overnight at 4 °C.
-	Streak out AIG04 on SC - Trp, YIG04.1 and YIG04.2 on SC - Leu
-	Prepare robot plates the robot Y2H</P></DIV><DIV id="week19"><H2>September 15th, week 19</H2><H4 class="collapsibleListTitle" id="lab_team1"><I>C. albicans &amp; C. glabrata</I></H4><P class="collapsibleListBody">
-	Purification of the peptides 3, 4.1, 4.2, 5 and 13      •	1 L LB inoculated at an OD600 of 0.2 → grown to 0.8, then expression is induced with 1mM IPTG      •	After 3h: centrifuged at 5000rpm for 15 min at 4°C      •	After solving in Buffer W the cells are disrupted by French press and centrifuged for 1h at 35.000 rpm and 4 °C      •	The proteins are extracted via Strep-Tactin® columns       •	Samples were loaded onto an 12% SDS Gel
SDS Gels (12%)
Strep 3:
⇒	There should be a clear band in the elutions steps for ~ 20 -22 kDa; for strep 3 is no such band
Strep 4.1:
⇒	No band of ~20-22 kDa in the elution steps for strep 4
Strep 4.2:
⇒	The right band can be seen in the elution steps, but it is not as strong as it should be
Strep 5:

⇒	The enlarged band in the elution steps has a size of ~ 15 kDa, which seems to be a little small for our peptide
Strep 13:
⇒	For strep 13 the right bands of ~20-22 kDa can be seen in all lines, especially in the elution steps</P><H4 class="collapsibleListTitle" id="lab_team2"><I>A. fumigatus &amp; A. nidulans</I></H4><P class="collapsibleListBody"><B><I>A. fumigatus</I></B>
-	Starting Seamless cloning mix, incubate at RT for 30 minutes, 4 minutes on ice and then <I>E. coli</I> transformation into TOP10 strain. Plating 200 µl and rest on cat plates.
-	Determination of OD600 of strep cultures and inoculate 1 L LB + Amp to an OD of 0.2 and let them grow to an OD of 0.8 (approx. 1 h). Induction with 1 ml of 1 M IPTG and incubation at 180 rpm and 37 °C for 3 hours. Centrifugation in autoclaved bottles for 15 minutes at 5 krpm at 4 °C. Freeze pellets in -20 °C overnight.
-	Preparing robot plates and distribute 200 µl of each cell culture 
-	Run spotting program on the robot. And incubate at 30 °C
-	Pre-cool french press stuff in ice water. Pre-cool Buffer W and tips and columns in cold room at 4 °C. Resuspend each pellet in 15 ml ice cold Buffer W. Run french press as in the protocol. Centrifuge the disrupted cells at 4 °C for 1 h and 35.000 krpm. 
-	Prepare columns, Buffer E, Buffer R and pack each column with 1 ml streptacin 50 %. 
-	Run the supernatant from UZ over the columns. Save rest in falcon tubes from not purified, flow through. Save wash step 1- 4 elution step 1- 4 in labeled 1.5 ml Ecups.  
-	Prepare 4 SDS gels 12.0 % (Gels see <I>Candida</I>)
-	Calculation of protein amount in CE and FT with Bradford.
-	Y2H with robot to confirm interactions</P><H4 class="collapsibleListTitle" id="lab_team3"><I>S. cerevisiae</I></H4><P class="collapsibleListBody">
-	Repeat  trafo in Y187      ⇒	Trafo worked</P><H4 class="collapsibleListTitle" id="lab_team4"><I>T. gondii</I></H4><P class="collapsibleListBody">
-	Repeat  trafo in Y187      ⇒	Trafo worked</P></DIV><DIV id="week18"><H2>September 8th, week 18</H2><H4 class="collapsibleListTitle" id="lab_team1"><I>C. albicans &amp; C. glabrata</I></H4><P class="collapsibleListBody">
-	PCR for sequencing on strep-tag plasmids
-	Purification and sequencing       ⇒	Sequencing looks ok
-	Test expression for peptides 3, 4.1, 4.2, 5, 13
SDS gels (12%)
Peptide 3:
Peptides 13 and 4.1:
Peptides 4.2 and 5:
-	Test expression looks good; nothing in negative control (NK)
-	super-competent cells with iGEM protocol → see <I>Aspergillus</I></P><H4 class="collapsibleListTitle" id="lab_team2"><I>A. fumigatus &amp; A. nidulans</I></H4><P class="collapsibleListBody"><B><I>A. fumigatus</I></B>
-	PCR with T7 primers for sequencing of 6 plasmid inserts (seqlab) → purification
-	<B>NanoDrop:</B>
-	Pick colonies from strep plates inoculate over night culture. Inoculation over day culture to OD600 0.3. Prepare test expression 
-	Ligation of cut clean pGP172 (11ng/µl) with cc5, 9, 10 ,15
-	Follow blunt end-ligation protocol: 20 µl sample included 2 µl PEG4000 1 µl T4 ligase
-	Preparation of final robot Y2H
-	Transformation into <I>E. coli</I> DH5α
-	 Sequences of Seqlab looks ok
-	Test expression of 3, 4, 13
-	Samples from test expression were filled up with 15 µl of SDS loading dye and boiled for 15 min at 95 °C. Loaded to SDS 12% PAGE. (Gels see<I> Candida</I>)
-	Prepare comp. cells from TOP10 strain. Overnight culture was grown at 18°C for 19h OD600=0.395 → 1:1 dilution with 250 ml SOB for 1 h incubation at 18 °C. OD600 = 0.32, putted on ice and filled into prechilled 50 ml flasks. Followed the protocol.       ⇒	Test the competence → worked very well!<B><I>A. nidulans</I></B>
-	prepare robot screening for 2st Y2H with the isolated plasmids out of the Y187 transformation
-	2st Y2H screening on plates - failed</P><H4 class="collapsibleListTitle" id="lab_team3"><I>S. cerevisiae</I></H4><P class="collapsibleListBody">
-	Pick colonies from<I> E. coli </I>plates and isolate plasmids
-	Transformation of  yeast (Y187)      ⇒	Trafo failed
</P><H4 class="collapsibleListTitle" id="lab_team4"><I>T. gondii</I></H4><P class="collapsibleListBody">
-	Pick colonies from E. coli plates and isolate plasmids
-	Transformation of  yeast (Y187)      ⇒	Trafo failed</P></DIV><DIV id="week17"><H2>September 1st, week 17</H2><H4 class="collapsibleListTitle" id="lab_team1"><I>C. albicans &amp; C. glabrata</I></H4><P class="collapsibleListBody">
-	New ligation of 4.1 (ssr1 interacting peptide) (see <I>Aspergillus</I>)      ⇒	Test via PCR, if insert is in the vector (bigger fragment) → see <I>Aspergillus</I>
-	Sequencing: again 10 + 15 (+ BP1)      ⇒	Results: 10 and 15 are exactly the same interacting peptide</P><H4 class="collapsibleListTitle" id="lab_team2"><I>A. fumigatus &amp; A. nidulans</I></H4><P class="collapsibleListBody"><B><I>A. fumigatus</I></B>
-	Measure DNA conc. pGP172 → 350ng/µl
-	Use 1.000 ng for restriction. 16 µl mixture 4 µl buffer (Tango) 2 µl <I>Sma1</I> incubation for 120 minutes at 30 °C. Then add 2 µl of <I>BamH1</I> for more 2 h. dephosphorylation with SAP for additional 30 minutes. 
-	Measure OD of o/N culture Y187 (5.4) and AH109 (18.4). Inoculation of 50 ml YPAD 0.54 ml and 1.89 ml
-	AH109 0.7 OD600 follow long term storage protocol. Freeze at -80°C there was no sorbitol inside
-	purification of pGP172 - nanoDrop: 50ng
-	Ligation over night at 16 °C
-	cut pGP172 + fragments cc_number
-	Inoculation of AH109/Y187 for storage cultures 
-	PCR with PIGY9 and 10 with IG099 and IG1003 
-	Measure OD600 from o/N cultures; AH109 [12.9] and Y187 [12.1]. →use 0.78 ml and 0.82 ml of the strains to inoculate to an OD600 of 0.2. After 3 hours follow long term storage protocol
-	Purification of PCR product from 9 and 10 
-	Cut pIGY9 and 10 with<I> Hpa1</I> and <I>BamH1</I><B>Agarose Gel (1%)</B>
-	Prep. of pGP172cut and clean: Sma1 and BamH1 - SAP 50 µl sample and purification
-	Transformation of all ligations → nothing on plates	
-	Inoculation of BL21blue and pGP172.
-	Restriction of pGP172 again for ligation of inserts. 1:1 ratio, Ligation with three inserts 3, 4.1 and 15. 
-	Transformation into DH5α.
-	Growth on plates: Pick 10 colonies from each transformation plates grow over day 37°C with respective antibiotics and start plasmid isolation - stopped at 2-prop step over night [also pGP172 3x purification].
-	New Oligos: Check PCR with T7p and T7t (52°) for strep
-	PCR (12,5µl samples) with strep-4.1 and strep-NK(15/3?)<B>Agarose Gels (1%):</B>      ⇒	Bands in 3, 4, 5, 7, 8, 9 are positive      ⇒	Bands in 4 and 8 are positive
-	Inoculations from overnight culture a 500 ml flask with 50 ml LB and shake it till OD600 0.35 and follow fast comp. cell protocol.  Use 2 µl from each plasmid and incubation over 2 days at 37°C.</P><H4 class="collapsibleListTitle" id="lab_team3"><I>S. cerevisiae</I></H4><P class="collapsibleListBody">
-	Spot positive clones on SC-LWH plates with 3mM 3-AT
-	Wash cells from plates and isolate plasmids
-	Transformation into DH5α</P><H4 class="collapsibleListTitle" id="lab_team4"><I>T. gondii</I></H4><P class="collapsibleListBody">
-	Spot positive clones on SC-LWH plates with 3mM 3-AT
-	Wash cells from plates and isolate plasmids
-	Transformation into DH5α </P></DIV><DIV id="week16"><H2>August 25th, week 16</H2><H4 class="collapsibleListTitle" id="lab_team1"><I>C. albicans &amp; C. glabrata</I></H4><P class="collapsibleListBody">
-	PCR with Primers IG1003 and IG0099 → product that can be sequenced and cut with enzymes for strep tag plasmid insertion (samples of prey from 3, 13 and 15)
Agarose gel (1%):      ⇒	Bands in lines 1, 2, 3, 4, 5 6, 9 and 11 have the right size and are first purified and then send for sequencing with Primer IG1003 (rev)
-	Prey samples of 3A.1, 3A.2, 3A.3, 3B.1, 3B.2, 13.3, 15 digested with<I> BamHI </I>and <I>HpaI</I>      ⇒	Clear difference between cut and uncut; also cut bands are very weak
-	Results of sequencing (3, 13, 15):      ⇒	3A.1, 3A.2, 3A.3 are the same peptide      ⇒	3B.1 and 3B.2: sequences are not useful      ⇒	13.3 and 13.4: same peptide      ⇒	9: one interacting peptide      ⇒	10 + 15: same interacting peptide→?
-	New test expression (see <I>Aspergillus</I>)</P><H4 class="collapsibleListTitle" id="lab_team2"><I>A. fumigatus &amp; A. nidulans</I></H4><P class="collapsibleListBody"><B><I>A. fumigatus</I></B>
-	centrifuge 5 min at 13 krpm - freeze at -20 °C
-	Preparation of Amp and Cm plates, streak out of used strains on Amp-plates
-	SDS-gels prepared and stored in PLP 1x buffer at 4 °C
-	staining of sds gel → empty
-	PCR with fragments (3, 13, 15) (Agarose gel see <I>Candida</I>)      ⇒	Purification of: 13.3 15 3.1A 3.2A 3.3A 3.1B 3.2B
-	purified PCR products for restriction, ligation and transformation
-	Preparation of 10 x PLP buffer and new SDS-gels 
-	Loading of 5 µl sample onto the gels→ expected size empty ~11.5 kDa
-	New test expression at 30 °C and test also supernatant. 
-	Inoculation over night cultures of desired strains.
-	Toxicity cannot be the case, Protein G subunit backbone was shown to hide this 
-	Streak out DB3.1 strains containing pDONR and pDEST
-	Inoculation of over day cultures to OD600 of 0.1. Grow for 90 minutes at 30 °C
-	Measure OD600 and take sample (safe also the supernatant)
-	Add IPTG and take samples after 1, 2, 3 hours. 
-	Follow test expression protocol
-	PCR with strep-tagged proteins T7 promotor / terminator<B>Agarose Gel (1%):</B>
-	cut again pGP172 (520 ng and 1080 ng) with <I>Sma1</I> and <I>BamH1 </I>(Tango Buffer for 2 h) 
-	pick 5 different colonies from strep tagged inserts into DH5α → colony PCR      ⇒	false inserts ☹ 
-	Transformation of ligation into DH5α<B><I>A.nidulans</I></B>
-	Repeat yest transformation in Y187 with the genes <I>rodA </I>and <I>npc2</I>
-	isolate plasmids </P><H4 class="collapsibleListTitle" id="lab_team3"><I>S. cerevisiae</I></H4><P class="collapsibleListBody">
-	Y2H in semi solid media with the two genes</P><H4 class="collapsibleListTitle" id="lab_team4"><I>T. gondii</I></H4><P class="collapsibleListBody">
-	Y2H in semi solid media </P></DIV><DIV id="week15"><H2>August 18th, week 15</H2><H4 class="collapsibleListTitle" id="lab_team1"><I>C. albicans &amp; C. glabrata</I></H4><P class="collapsibleListBody">
PCR with different plasmid isolation samples for prey amplification (IG1003 and IG0099; 52,5 °C)<B>Agarose gel (1%):</B>      ⇒	Bands in all samples      ⇒	Purification and restriction with<I> HpaI</I> and <I>BamHI</I>
-	Restriction of of 0,1 µl (14500ng/µl) pGP172 with <I>SmaI </I>and <I>BamHI</I> o/n 16°C
-	ligation of cut cleaned PCR prey products with pGP172 (10ng/µl)  4µl of each with 1µl T4 Ligase in 20 µl sample
-	Transformation into DH5α 
-	Again: plasmid isolation and PCR with T7 promotor and terminator primer on plasmids<B>Agarose gel (1%):</B>      ⇒	In lines 4 to 9 seems to be the right fragment; 2 and 3 are wrong
-	Transformation into BL21
-	Test-expression of strep-tag strains: 4.1A, 4.1B, 5A, 5B, 14.5A, 14.5B, 14.8A, 14.8B
-	Preparation of SDS gels
-	PCR with Primers IG1003 and IG0099 → product that can be sequenced and cut with enzymes for strep tag plasmid insertion (samples of prey from 3, 13 and 15)<B>Agarose gel (1%):</B>      ⇒	Bands in lines 1, 2, 3, 4, 5 6, 9 and 11 have the right size and are first purified and then send for sequencing with Primer IG1003 (rev)
-	Prey samples of 3A.1, 3A.2, 3A.3, 3B.1, 3B.2, 13.3, 15 digested with <I>BamHI</I> and <I>HpaI</I>      ⇒	Clear difference between cut and uncut; also cut bands are very weak
-	Results of sequencing (3, 13, 15):      ⇒	3A.1, 3A.2, 3A.3 are the same peptide      ⇒	3B.1 and 3B.2: sequences are not useful      ⇒	13.3 and 13.4: same peptide      ⇒	9: one interacting peptide      ⇒	10 + 15: same interacting peptide→?</P><H4 class="collapsibleListTitle" id="lab_team2"><I>A. fumigatus &amp; A. nidulans</I></H4><P class="collapsibleListBody"><B><I>A. fumigatus</I></B>
-	 PCR from diff. plasmids for prey amplification 52,5 °C (Agarose Gel see <I>Candida</I>)      ⇒	Worked for all
-	Purification of prey pcr products, restriction with <I>HpaI</I> and <I>BamH1 </I>
-	restriction of 0.1 µl (14500 ng/µl) pGP172 with <I>Sma1</I> and <I>BamH1</I> o/n 16 °C
-	ligation of cutted cleaned PCR prey products with pGP172 (10 ng/µl);  4 µl of each with 1 µl T4 Ligase in 20 µl sample
-	PCR (100 µl vol) with sample 5 and 4.1 to increase amount of product<B>Agarose gel(1%)</B>
-	Transformation of strep-prey plasmids worked. over day cultures in LB at 42 °C for plasmid isolation and colony PCR
-	Inoculation of<I> E. coli</I> BL21blue for fresh comp cell to transform strep plasmids
-	pGP172 o/N 16 °C restriction compared with 0,1 µl and 1 µl non cut plasmid
-	nothing in colony PCR
-	pellets from over day cultures
-	Plasmid isolation<B>NanoDrop:</B>
-	PCR with T7 promotor and terminator primer on plasmids (Agarose gel see <I>Candida</I>)
-	LB medium, BL21 blue competent cells, transformation with plasmids
-	Transformation of BL21blue with the plasmids worked 
-	Inoculation of 10 ml LB medium incubation for 2 h at 37 °C
-	Take samples before induction and after 1 - 2 - 3 h [ 100/OD600 ]
-	centrifuged 5 min at 13 krpm → freeze at -20°C
-	Preparation of Amp and Cm plates, streak out of used strains on Amp-plates
-	SDS-gels prepared and stored in 1x PLP buffer at 4 °C<B><I>A. nidulans</I></B>
-	Repeat transformation of isolated plasmids out of the 1st Y2H experiment in <I>E. coli</I> DH5α 
-	Plasmid isolation out of <I>E. coli</I> DH5α 
-	NanoDrop:
-	Repeat transformation in Y187 on SC-Leu medium - failed</P></DIV><DIV id="week14"><H2>August 11th, week 14</H2><H4 class="collapsibleListTitle" id="lab_team1"><I>C. albicans &amp; C. glabrata</I></H4><P class="collapsibleListBody">
-	Yeast Plasmid isolation of preys for: 01, 02, 06, 08, 12, 13, 15, 16 (pIGX)<B>NanoDrop:</B>
- Preparation of fresh competent cells
- <I>E. coli</I> Transformation: supercompetent cells and fresh competent cells are transformed with different plasmids (see table)		
- Results of <I>E. coli</I> transformation:      ⇒	Plasmid isolation and transformation into Yeast Y187 → but no colonies</P><H4 class="collapsibleListTitle" id="lab_team2"><I>A. fumigatus &amp; A. nidulans</I></H4><P class="collapsibleListBody"><B><I>A. nidulans</I></B>
-	Repeat transformation in Y187 on SC-Leu medium - failed
-	Repeat transformation in Y187 on SC-Leu medium- failed
-	running out of isolated plasmids out of <I>E. coli</I> DH5α for Yeast transformation</P></DIV><DIV id="week13"><H2>August 4th, week 13</H2><H4 class="collapsibleListTitle" id="lab_team2"><I>A. fumigatus &amp; A. nidulans</I></H4><P class="collapsibleListBody"><B><I>A. fumigatus</I></B>
-	 O/N cultures from <I>E. coli</I> prey colonies to isolate plasmids again for restriction and ligation into pGP172<B><I>A. nidulans</I></B>
-	Transformation in <I>E. coli</I> DH5α 
-	Plasmid isolation out of<I> E. coli</I> DH5α 
-	Transformation in Y187 - failed</P></DIV><DIV id="week12"><H2>July 28th, week 12</H2><H4 class="collapsibleListTitle" id="lab_team1"><I>C. albicans &amp; C. glabrata</I></H4><P class="collapsibleListBody">
-	Yeast transformation (Y187) from preys of 01, 02, 02C, 03A+B, 06, 07, 13, 15, 16, 16C      ⇒	no colonies for 02C, 06 and 16C
-	Plasmid multiplication and isolation of prey plasmids from 04, 05, 09, 10, 14</P><H4 class="collapsibleListTitle" id="lab_team3"><I>S. cerevisiae</I></H4><P class="collapsibleListBody">
-	Yeast transformation (AH109) of pIGB22 and 21</P><H4 class="collapsibleListTitle" id="lab_team4"><I>T. gondii</I></H4><P class="collapsibleListBody">
-	Yeast transformation (AH109)</P></DIV><DIV id="week11"><H2>July 21st, week 11</H2><H4 class="collapsibleListTitle" id="lab_team1"><I>C. albicans &amp; C. glabrata</I></H4><P class="collapsibleListBody">
- Evaluation of Sequencing:
- Bad/ not useful sequences: 9, 10, 14.1, 14.3, 14.4, 14.7, 4.2 (4.2→ new PCR necessary, wrong fragment)
- Peptides from: 4.1, 5, 14.5, 14.6, 14.8 (=14.10), 14.9</P><H4 class="collapsibleListTitle" id="lab_team2"><I>A. fumigatus &amp; A. nidulans</I></H4><P class="collapsibleListBody"><B><I>A. fumigatus</I></B>
-	 pick colonies = o/n cultures for plasmid isolation
-	plasmid- prep. of pGP172 acc. to the protocol<B><I>A. nidulans</I></B>
-	1st Y2H experiment with the genes rodA, xlnA, npc2, slno1 in semi solid medium - worked only for the plasmids pIGA17 (<I>rodA</I>) and pIGA19 (<I>npc2</I>)
-	after growing the yeast colonies are washed from plates 
-	plasmid isolation </P></DIV><DIV id="week10"><H2>July 14th, week 10</H2><H4 class="collapsibleListTitle" id="lab_team1"><I>C. albicans &amp; C. glabrata</I></H4><P class="collapsibleListBody">
-	Confirmed interactions from <A href="https://2014.igem.org/Team:Goettingen/protocol_Colony#Y2H_robot">robot Y2H </A>are used for Plasmid isolation (prey): 4.1; 4.2; 5; 9; 10; 14.1; 14.3; 14.4; 14.5; 14.6; 14.7; 14.8; 14.9 and 14.10 + Isolation of interacting preys of AIG003, 07, 08, 12, 13
-	<B>NanoDrop</B>
-	additional plasmid purification for sequencing of plasmids 4.1, 4.2, 5, 9, 10, 14.1
-	all plasmids are used in a PCR with Primers IG1002 (1170_fwd) and IG1003(1170_rev) (56°C)<B>Agarose gel (1%):</B>      ⇒	PCR products with IG1003 for sequencing (line 3→ fragment bigger than others, but also sequenced)
-	Transformation <I>E. coli</I> DH5α with 03, 07, 08, 12, 13
-	Plasmid isolation of E. coli 02, 06, 15, 16, 2C, 16C<B>NanoDrop:</B></P><H4 class="collapsibleListTitle" id="lab_team2"><I>A. fumigatus &amp; A. nidulans</I></H4><P class="collapsibleListBody"><B><I>A. fumigatus</I></B>
-	Test of mated colonies from robot Y2H on X- α-gal plates
-	3rd Y2H: after spotting, wash colonies from plates and plasmid isolation</P></DIV><DIV id="week9"><H2>July 7th, week 9</H2><H4 class="collapsibleListTitle" id="lab_team1"><I>C. albicans &amp; C. glabrata</I></H4><P class="collapsibleListBody">
-	Plasmid isolation of prey interactions of 02, 06, 15, 16 and transformation into<I> E. coli</I> (Amp plates) 
-	Another Y2H is performed with AIG004, 05, 09, 10, 14 and YIG04, 05, 09, 10, 14 to confirm the interactions we saw in the first Y2H (using a <A href="https://2014.igem.org/Team:Goettingen/protocol_Colony#Y2H_robot">robot</A>); testing on SC-LWH plates with 3mM 3-AT and X-α-Gal plates</P><H4 class="collapsibleListTitle" id="lab_team2"><I>A. fumigatus &amp; A. nidulans</I></H4><P class="collapsibleListBody"><B><I>A. fumigatus</I></B>
-	Plasmid isolation of the genes 02, 06, 15, 16
-	<B>NanoDrop:</B>
-	3rd Y2H is performed with the strains AIG003, 07, 08, 12 and 13
Titerplates:
-	Robot Y2H with 04, 05, 09, 10, 14 (see <I>Candida</I>)
-	Transformation of 02, 06, 15, 16 pool into DH5α      ⇒	Only grown with single colonies → again with different concentrations</P></DIV><DIV id="week8"><H2>June 30th, week 8</H2><H4 class="collapsibleListTitle" id="lab_team1"><I>C. albicans &amp; C. glabrata</I></H4><P class="collapsibleListBody">
-	1st) Transformation in yeast: some colonies are grown, but not for all → tried transformation again
-	Plasmid isolation and saving of prey strains on SC-L plates
-	2nd) Plasmid isolation from yeast for prey plasmids interacting with AIG002, AIG006, AIG015, AIG016 (pIGX plasmids)
-	Transformation in DH5α → not successful (next week again)
-	Third Y2H screening of AIG003 (<I>sim1</I>), 07 (<I>pir3</I>), 08 (<I>utr2</I>), 12(<I>ecm33</I>), 13 (<I>sun1</I>)</P><H4 class="collapsibleListTitle" id="lab_team2"><I>A. fumigatus &amp; A. nidulans</I></H4><P class="collapsibleListBody"><B><I>A. fumigatus</I></B>
-	Yeast Trafo into Y187 again
-	2nd Y2H: Plasmid isolation for Transformation into <I>E. coli</I> (pIGZ)
-	Safe prey interacting partners on SC-L
-	Preparation of new AA mixture</P><H4 class="collapsibleListTitle" id="lab_team4"><I>T. gondii</I></H4><P class="collapsibleListBody">
-	Repeat LR reaction and transformation into DH5α      ⇒	Trafo worked
-	Cracking of <I>E. coli</I> cells → right plasmid in cells
-	Plasmid isolation (2528,7 ng/µl)</P></DIV><DIV id="week7"><H2>June 23rd, week 7</H2><H4 class="collapsibleListTitle" id="lab_team1"><I>C. albicans &amp; C. glabrata</I></H4><P class="collapsibleListBody">
-	Spotting of second Y2H (AIG002 (<I>tos1</I>), 06 (<I>scw4</I>), 15 (<I>als3</I>), 16 (<I>rodA</I>)) (picture)<B>Figure: Different interacting and matted colonies are spotted onto the plate(Example)</B>
-	1st) Yeast colonies from first spotting are solved in water and centrifugated to pellets
-	Isolation of plasmids from pellets (see <A href="/Team:Goettingen/protocol_Plasmid_Tran">protocol</A>) (pIGX plasmids)
-	Transformation in DH5α (70 µl of pooled plasmid DNA plated on Amp-plates)
-	Thin layer of  <I>E. coli</I> cells are washed from plates with 1,5 ml of sterile water; centrifugation (4krpm, 5min), plasmid isolation (pIGY plasmids)
-	Transformation into yeast (Y187) cells. Plated onto SC-L plates (YIG strains)
-	2nd) Yeast colonies from second spotting are solved in water and centrifugated to pellets → frozen</P><H4 class="collapsibleListTitle" id="lab_team2"><I>A. fumigatus &amp; A. nidulans</I></H4><P class="collapsibleListBody"><I><B>A. fumigatus</B></I>
-	Preparation of yeast plasmid prep. Solutions P1, P2, and N3
-	1st Y2H: Yeast colonies are washed from plates; plasmid isolation, Transformation into<I> E. coli; E. coli</I> plasmid isolation and again transformation into yeast (Y187) (YIG strains) (see<I> Candida</I>)
-	2nd Y2H: yeast colonies are washed from plates and centrifuged to pellets → frozen<I><B>A. nidulans</B></I>
-	<I>E. coli</I> plasmid isolation of cells with LR product
-	PCR on plasmids → no positives
-	Repeat LR reactions and transformation
-	Again plasmid isolation and PCR<B>Agarose Gel </B>(1%):      ⇒	Right bands in: 1, 2, 4, 5, 6, 7, 8, 9, 16
-	Plasmid isolation of all positive clones</P><H4 class="collapsibleListTitle" id="lab_team3"><I>S. cerevisiae</I></H4><P class="collapsibleListBody">
-	PCR on isolated plasmids      ⇒	Nothing on Gel
-	LR reaction was repeated for <I>CWP1, CWP2, SED1, MID2</I>
-	Transformation of DH5 α       ⇒	Worked for all, except<I> CWP1</I>      ⇒	Cracking failed
-	Colony PCR
-	<B>NanoDrop:</B></P><H4 class="collapsibleListTitle" id="lab_team4"><I>T. gondii</I></H4><P class="collapsibleListBody">
-	LR reaction and transformation into DH5α      ⇒	Trafo didn’t work</P></DIV><DIV id="week6"><H2>June 16th, week 6</H2><H4 class="collapsibleListTitle" id="lab_team1"><I>C. albicans &amp; C. glabrata</I></H4><P class="collapsibleListBody">
-	Second Y2H screening for (<I>tos1</I>), 06 (<I>scw4</I>), 15 (<I>als3</I>), 16 (<I>rodA</I>) (titerplates didn’t work)
-	From first Y2H screening (AIG001, 04,05,09,10,14): 10 µl of each growing colony from semi solid medium is transferred into 100 µl of SC-LWH → 3µl spotted onto SC-LWH + 3mM 3-AT plates (picture) (rest: glycerol added and frozen) <B>Figure: Colonies are growing inside the semi-solid medium</B></P><H4 class="collapsibleListTitle" id="lab_team2"><I>A. fumigatus &amp; A. nidulans</I></H4><P class="collapsibleListBody"><I><B>A. fumigatus</B></I>
-	Preparation for Y2H
-	Second Y2H (see <I>Candida</I>) 
-	Spotting of colonies from first Y2H<I><B>A. nidulans</B></I>
-	Repeat LR reaction and Transformation in fresh competent DH5α cells      ⇒	Trafo worked
-	Cracking      ⇒	No positives → cracking may have failed      ⇒	Try with colony PCR → failed</P><H4 class="collapsibleListTitle" id="lab_team3"><I>S. cerevisiae</I></H4><P class="collapsibleListBody">
-	Transformation of DH5α with LR product      ⇒	Trafo worked for all, except<I> CWP2</I>
-	Cracking of the colonies      ⇒	No positives
-	Isolation of plasmids</P><H4 class="collapsibleListTitle" id="lab_team4"><I>T. gondii</I></H4><P class="collapsibleListBody">
-	Repeat BP reaction and transformation into DH5α      ⇒	Trafo worked
-	<I>E. coli</I> cracking<B>Agarose Gel (1%):</B>
-	Plasmid isolation of <I>ama1 </I>(pIGA25); NanoDrop: 1308,3 ng/µl</P></DIV><DIV id="week5"><H2>June 9th, week 5</H2><H4 class="collapsibleListTitle" id="lab_team1"><I>C. albicans &amp; C. glabrata</I></H4><P class="collapsibleListBody">
- No growth on auto-activity plates
- Preparation of  media for Y2H
- First Y2H screening (semi solid medium) of AIG001 (<I>mp65</I>); AIG004 (<I>ssr1</I>); AIG005 (<I>pir4</I>), AIG009 (<I>crf2full</I>); AIG010 (<I>crf2act</I>); AIG014 (<I>als1</I>); 
- Titerplates on SC-LW medium<B>Titerplates:</B></P><H4 class="collapsibleListTitle" id="lab_team2"><I>A. fumigatus &amp; A. nidulans</I></H4><P class="collapsibleListBody"><I><B>A. fumigatus</B></I>
-	LR reaction and transformation of pIGA11 and 13
-	Transformation into DH5α      ⇒	Trafo worked!
-	Transformation into Yeast AH109
-	First Y2H of AIG09 and AIG10 (see <I>Candida</I> notebook)<I><B>A. nidulans</B></I>
-	Repeat BP reactions and transformation into DH5α      ⇒	Trafo worked
-	Cracking (see <I>Candida</I>)
-	o/n cultures of positive clones<B>Agarose Gel (1%) </B>of cracking:
-	Plasmid isolation
-	<B>NanoDrop</B>:
-	LR reaction with the 4 plasmids
-	Transformation into fresh competent DH5α cells      ⇒	Trafo failed</P><H4 class="collapsibleListTitle" id="lab_team3"><I>S. cerevisiae</I></H4><P class="collapsibleListBody">
-	Cracking for<I> CWP1, CWP2, SED1, MID2</I><B>Agarose gel (1%): </B>      ⇒	All of the <I>E. coli </I>strains contain the right plasmid (band 4 is negative due to pipetting mistake)      ⇒	Sample 2 of each gene was chosen for plasmid isolation
-	<B>NanoDrop</B>:

-	LR reactions</P></DIV><DIV id="week4"><H2>June 2nd, week 4</H2><H4 class="collapsibleListTitle" id="lab_team1"><I>C. albicans &amp; C. glabrata</I></H4><P class="collapsibleListBody">
- LR reactions with pDEST and pIGA-plasmids and Transformation into DH5α
- Transformation was successful → Cracking + 4 mL LB+ kan o/n cultures<B>Agarose gels (1%):</B>
- Samples 1 (mp65), 6 (tos1), 7(sim1), 11(ssr1), 13 (pir4) and 15 (scw4) are used for plasmid isolation.
- Samples 2 (pir3), 4 (utr2), 6 (crf2 full), 9 (crf2 act) and 13 (ecm33) were used to isolate the plasmids. 
- The samples from 10-12 (bglCno) were cracked again.
- Samples 9 (als3) and 14 (eglC) are used to isolate the plasmids. The samples for sun1, als1 and rodA are cracked again.
- Cracking of each 3 samples of 11(bglEno), 13(sun1), 14(als1), 16(rodA)<B>Agarose gels (1%):</B>
- Samples 9 (als1) and 12 (rodA) are used to isolate the plasmids. No results for bglEno and sun1 (<I>A. fumigatus</I>)
- Plasmid isolation
- <B>Nano-Drop results:</B>
- Transformation in Yeast (AH109) → plated on SC-W plates
- Transformation worked → single colonies are streaked on SC-W and SC-HW +3mM 3AT (auto-activity test) plates</P><H4 class="collapsibleListTitle" id="lab_team2"><I>A. fumigatus &amp; A. nidulans</I></H4><P class="collapsibleListBody"><B><I>A. fumigatus </I></B>
-	LR reaction with pDEST and pIGA- plasmids
-	Transformation into DH5α      ⇒	Trafo worked
-	Validation of positive clones via<I> E. coli </I>cracking (from each plate 3 colonies) (pictures see <I>Candida</I>)
-	4 ml LB with kanamycin inoculated for positive clones
-	Plasmid isolation (pIGB plasmids)
-	<B>NanoDrop:</B> (pIGB11 and 13 were empty)
-	Yeast (AH109) transformation (AIG strains)      ⇒	Trafo worked
-	Test on autoinduction (His auxotrophy) on SC-HW + 3mM 3-AT       ⇒	No autoinduction (grid plate see<I> Candida</I> labbook)<B><I>A. nidulans</I></B>
-	Amplification of GOI (size)	      ⇒	<I>A. nidulans</I>: 	            <I>rodA</I> (467 bp)            <I>xlnA</I> (749 bp)            <I>sho1</I> (934 bp)            <I>npc2</I> (627 bp)
-	Purification of PCR products
-	BP reaction into pDONR vector
-	Transformation into DH5α      ⇒	Trafo failed</P><H4 class="collapsibleListTitle" id="lab_team4"><I>T. gondii</I></H4><P class="collapsibleListBody">
-	Amplification of the gene of interest via PCR:      ⇒	<I>Toxoplasma: 	ama1 </I>(Annealing temperature) 54°C<B>Agarose Gel(1%):</B>      ⇒	Purification of fragment. NanoDrop  → 157,6 ng/µl
-	BP reaction and transformation into DH5α      ⇒	Trafo didn’t work</P></DIV><DIV id="week3"><H2>May 26th, week 3</H2><H4 class="collapsibleListTitle" id="lab_team1"><I>C. albicans &amp; C. glabrata</I></H4><P class="collapsibleListBody">
- Transformation of BP reaction into fresh competent cells (50 µl+ rest)
- Transformation Efficienty test for supercompetent cells with iGEM kit (control with fresh competent cells)   ⇒ Did not work → Kit is not useful !
- Transformation of all BP reactions worked, except for als1 and als3
- 3 colonies from each transformation were picked and dissolved in 10 µl H2O. 5µl of this were used for cracking and 5 µl to incubate a 4 ml LB culture over day.<B>Agarose Gels:</B>
Samples A (from each plasmid):
Samples B (from each plasmid):
Samples C (from each plasmid):      ⇒	Plasmids can be found in almost all cultures → one positive culture for each plasmid was chosen and centrifuged (5 min, 5000rpm)
-	PCR for als1 and als3 were repeated:       ⇒1+3 purification (eluted in 70 µl H2O)      ⇒ <B>Nano Drop: </B>            1        8,8 ng/µl            2        24,3 ng/µl
→ PCR product from first PCR for als1 and from second PCR for als3 were used for the BP reaction (+Trafo)    - Plasmid isolation of the samples from <I>C. albicans</I>, <I>C. glabrata</I> and <I>A. Fumigatus</I> (see <A href="">protocol</A>) (o/n cultures)    -<B> NanoDrop results:</B></P><H4 class="collapsibleListTitle" id="lab_team2"><I>A. fumigatus &amp; A. nidulans</I></H4><P class="collapsibleListBody">
-	BP reactions of PCR products into pDONR207 and transformation into DH5α
-	Transformation into<I> E. coli</I> DH5α cells      ⇒	Trafo worked
-	Cracking of eight different colonies from each transformation (Agarose gel pictures see<I> Candida</I> notebook)
-	4 ml LB + gentamycin inoculated with positive clones
-	Plasmid isolation (pIGA plasmids)
-	<B>NanoDrop:</B></P><H4 class="collapsibleListTitle" id="lab_team3"><I>S. cerevisiae</I></H4><P class="collapsibleListBody">
-	Amplification of the genes of interest via PCR:      ⇒	<I>S. cerevisiae:</I>            <I>CWP1</I> (700 bp)            <I>CWP</I>  (300 bp)            <I>TIR1</I> (800 bp)            <I>SED1</I> (1000 bp)            <I>TIR4</I> (1500bp)            <I>MID2</I> (1200 bp)<B>Agarose Gel</B> (1%):      ⇒	1,2,3,4,7,8,11,12 worked; 9 + 10 will be repeated because of the two separate bands
-	Purification of right fragments
-	<B>NanoDrop</B>:
-	BP reaction of <I>CWP1, CWP2, SED1, MID2</I>
-	Transformation into DH5α      ⇒	Trafo worked
-	Repeat PCR for<I> TIR1</I> and<I> TIR4</I> fragments → no bands in agarose gel</P></DIV><DIV id="week2"><H2>May 19th, week 2</H2><H4 class="collapsibleListTitle" id="lab_team1"><I>C. albicans &amp; C. glabrata</I></H4><P class="collapsibleListBody">
- Amplification of the genes of interest via PCR      ⇒ <I>C. albicans: mp65, als1, tos1, sim1, als3</I>      ⇒ <I>C. glabrata: ssr1, pir4, scw4, pir3, utr2</I>      ⇒ Annealing temperature: 56 °C
- <B>Agarose Gel (1%):</B>      ⇒ Band 5 (als3) is a little bit weak → but PCR product is also purified and further used     ⇒ Purification of the PCR products with the Nucleo Spin® Gel and PCR Clean-up kit from Macherey-Nagel      ⇒ Nano Drop Values were not useful
-	Preparation of supercompetent <I> E. coli</I> DH5α cells (see <A href="https://2014.igem.org/Team:Goettingen/protocol_Plasmid_Tran#E_comp">protocol</A>)
-	BP reaction of all genes (see <A href="https://2014.igem.org/Team:Goettingen/protocol_Plasmid_Con#BP">protocol</A>)</P><H4 class="collapsibleListTitle" id="lab_team2"><I>A. fumigatus &amp; A. nidulans</I></H4><P class="collapsibleListBody">
-	Amplification of the genes of interest via PCR	(→ Annealing Temperature)
⇒	<I>A. fumigatus: </I>      <I>crf2</I> full (mature peptide AA: 21-416)		       (58°C)      <I>crf2</I> act (active side AA: 21-190)			       (56°C)      <I>bglE</I> no ( no GlcNAc outside AA: 594-908)	       (58°C)      <I>ecm33</I> ( active protein AA: 20-372)		       (58°C)      <I>sun1</I> (sequence between GlcNAc AA: 80-1133)	       (58°C)      <I>rodA</I> (AA: 19 – 159)				       (56°C)      <I>prm1</I> (extracellular AA: 419-603)		       (58°C)      <I>crf1 </I>(active side before GlcNac AA: 20-310)	       (58°C)<B>Agarose gel (1%):</B>
⇒	Purification of the PCR products with the Nucleo Spin® Gel and PCR Clean-up kit from Macherey-Nagel</P></DIV><DIV id="week1"><H2>May 12th, week 1</H2><H4 class="collapsibleListTitle" id="lab_team1"><I>C. albicans &amp; C. glabrata</I></H4><P class="collapsibleListBody" id="organ1">
- Test of supercompetent <I>E. coli</I> cells with the transformation efficiency kit (from iGEM)     ⇒ Did not work! No colonies
- Got cDNA of <I>Candida albicans</I> and <I>Candida glabrata</I> from<A href="https://2014.igem.org/Team:Goettingen/team_acknowledges#ackn7"> Oliver Baders Lab</A>:      <I>Candida albicans</I> strain SCS314      <I>Candida glabrata</I> strain CBS138    ⇒ Both high concentrations : dilution (10 µl DNA + 990 µl H2O)</P><H4 class="collapsibleListTitle" id="lab_team2"><I>A. fumigatus &amp; A. nidulans</I></H4><P class="collapsibleListBody">
-	Preparation of competent cells <I>E. coli</I> DB3.1 (mutated gyrase, resistant against ccdB) with CaCl2 (see <A href="https://2014.igem.org/Team:Goettingen/protocol_Plasmid_Tran#E_comp">Protocol for fresh comp. cells</A>)
-	Transformation with pDONR207 and pDEST (Plating of 50µl and rest on LB+ Cm plates, incubation over night at 37°C)       ⇒	Trafo worked! Single colonies in 4 ml LB + 4µl Cm       ⇒	Also saving of the strains on plates
-	Plasmid isolation with M&amp;N Kit, Elution in 70µl dH2O
-	<B>Nano Drop:</B></P></DIV><DIV><A href="https://2014.igem.org/Team:Goettingen/notebook_drylab" class="button_pre"><B>Previous</B></A><A href="https://2014.igem.org/Team:Goettingen/notebook_wetlab/biobricks" class="button_next"><B>Next</B></A></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Goettingen/notebook_wetlab">http://2014.igem.org/Team:Goettingen/notebook_wetlab</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Goettingen/notebook_wetlab" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Goettingen/notebook_wetlab" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Goettingen/notebook_wetlab&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Goettingen/notebook_wetlab&amp;oldid=370700" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>