"
Page
Discussion
View source
History
teams
Log in
 
Team:Goettingen/notebook drylab
From 2014.igem.org
Home
Project ▾
Background
Our project
Results
Perspectives
BioBricks
Notebook ▾
Safety
Wet lab
Dry lab
Materials
Protocols
Our Team ▾
Team members
Instructors
Acknowledgements
Sponsors
Outreach ▾
Human Practice
Gallery
Video
Presentation
Paris Bettencourt Newsletters
German teams' proposal
Achievements
Notebook
Overview
Safety
Dry lab
Wet lab
Materials
Homology modelling procedure
To generate our models we followed the procedure from the basic and advanced tutorials of the Modeller web page with some modifications. We first selected ten templates with known structures for our peptide-contaning scaffold. In the case of the protein G scaffold, this was done using the best five results from the Modeller "profile.build()" command and the best five results from an homology search in the Protein Data Bank; the PDB files of the selected templates are: 1PGB, 1FD6, 1FCL, 1QKZ, 1EM7, 2GB1, 1GB1, 2RPV, 2QMT and 3GB1.
In the case of the GFP scaffold, the selected templates were only those from the results of the "profile.build()" command; their PDB files are: 1XSS, 1OXD, 1XA9, 1MOU, 1XMZ, 1UIS and 1GGX.
Before starting with the alignments, we made a secondary structure prediction using the GeneSilico Metaserver . The consensus data was parsed from the results of the metaserver and processed so that the secondary structure information was associated with their corresponding amino acid residues.
The PDB files from the selected templates were aligned using the "salign()" command to generate a template alignment with the "alignment_type" in "tree" mode. The fm01645 family tree was used for the protein G scaffold template alignment and fm02248 in the case of the GFP scaffold. Then the sequence of each of our peptides was aligned separately to the template alignment using the ".append" method. Ten initial models were generated for each peptide using the secondary structure data as restrains; the best model with the best DOPE score was selected for loop refinement. The loop refinement generated ten models for each peptide; this was iterated until no improvements were observed in the DOPE score of the resulting structures.
Scripts
We modified some scripts from the Modeller tutorial and created a few new ones to automate our work. The following table summarizes them and gives a link to download them as .txt files (you have to change the extension to .py manually since the wiki server doesn't allow us to upload the files as such).
Script name
Environment
Description
Notes
File
align_and_model.py
Modeller 9.14
Template alignment, target alignment and secondary structure-restrained homology modeling. It iterates through a given list of initial .ali files.
The initial .ali file names (variable "preys"), the tree alignment and secondary structure restraints need to be manually changed.
Download
evaluate_model.py
Modeller 9.14
Small modification of the original evaluation script from the Modeller tutorial; this modified version iterates through the basic names of the generated .pdb files from "align_and_model.py".
The variable "preys" needs to be manually changed. Works if there are less than 100 models per initial .ali sequence.
Download
DOPE-parsing.py
Python 3.4.1
Reads the log file from "evaluate_model.py" and outputs a .txt file with a list of the structures with the best DOPE score.
The input and output directories have to be manually changed
Download
loop_refinement.py
Modeller 9.14
Small modification of the original loop refinement script from the Modeller tutorial; this modified version iterates through the names of the selected structures of the first homology modelin round.
The names of the structures ("preys" variable) has to be manually changed.
Download
References
1. Kurowski MA, Bujnicki JM. GeneSilico protein structure prediction meta-server. Nucleic Acids Res 2003; 31: 3305-3307
2. N. Eswar, M. A. Marti-Renom, B. Webb, M. S. Madhusudhan, D. Eramian, M. Shen, U. Pieper, A. Sali. Comparative Protein Structure Modeling With MODELLER. Current Protocols in Bioinformatics, John Wiley & Sons, Inc., Supplement 15, 5.6.1-5.6.30, 2006.
3. M.A. Marti-Renom, A. Stuart, A. Fiser, R. Sánchez, F. Melo, A. Sali. Comparative protein structure modeling of genes and genomes. Annu. Rev. Biophys. Biomol. Struct. 29, 291-325, 2000.
4. A. Sali & T.L. Blundell. Comparative protein modelling by satisfaction of spatial restraints. J. Mol. Biol. 234, 779-815, 1993.
5. A. Fiser, R.K. Do, & A. Sali. Modeling of loops in protein structures, Protein Science 9. 1753-1773, 2000.
Previous Next
Retrieved from " http://2014.igem.org/Team:Goettingen/notebook_drylab "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
