"
Page
Discussion
View source
History
teams
Log in
 
Team:Goettingen/project overview/project drylab
From 2014.igem.org
Home
Project ▾
Background
Our project
Results
Perspectives
BioBricks
Notebook ▾
Safety
Wet lab
Dry lab
Materials
Protocols
Our Team ▾
Team members
Instructors
Acknowledgements
Sponsors
Outreach ▾
Human Practice
Gallery
Video
Presentation
Paris Bettencourt Newsletters
German teams' proposal
Achievements
Project
Overview
Background
The global burden of fungal infections
Current diagnostic tools
Our project
Results
Wetlab
Drylab
Perspectives
Diagnosis
Therapeutics
BioBricks
中文
Dry lab
Homology modelling
We modeled the 3D structure of our peptide-containing scaffolds (domain B1 of protein G from Staphylococcus aureus) by homology modelling. We used the Modeller library to accomplish that; since the peptide is about 20-25 amino acids long, it can be modeled as a loop inside the B1 domain scaffold for which there are already a number of 3D structures available at the Protein Data Bank.
This is the general homology modeling procedure we followed. All peptides were cloned inside a scaffold (B1 domain of protein G from Staphylococcus aureus. The peptides were short enough to be modeled as an internal loop by loop refinement; these structures should only be considered as a working model, since we do not have direct crystallographic data. A detailed procedure can be found in this link.
Results summary
Protein G scaffold
The following table is a summary of the models we generated for our peptides inside the protein G scaffold. Their main purpose is to give us an idea if the calculated energy profile matches our assumption that the peptide is being exposed to the exterior of the scaffold. These models are in no way definitive, since we lack direct crystalographic data.
Peptide
PNG*
PDB
Peptide location
QMEAN score
Overall DOPE score
Energy profile
Prey 3
Download
K17-E42
0.512
-7053.966797
Download
Prey 4.1
Download
K17-E44
0.468
-7246.956055
Download
Prey 5
Download
K17-E44
0.495
-7501.071777
Download
Prey 13
Download
K17-E42
0.535
-7729.056152
Download
Prey 15
Download
K17-E42
0.441
-7685.998535
Download
*The peptide is shown in red and the scaffold in blue.
GFP scaffold
These are the structures we generated for our peptides inside a GFP scaffold.
Peptide
PNG*
PDB
Peptide location
QMEAN score
Overall DOPE score
Energy profile
Prey 3
Download
E103-E127
0.43
-26798.28125
Download
Prey 4.1
Download
E103-E127
0.448
-27891.212891
Download
Prey 5
Download
E103-E127
0.483
-26633.4375
Download
Prey 13
Download
E103-E127
0.404
-27075.160156
Download
Prey 15
Download
E103-E127
0.413
-26713.355469
Download
*The peptide is shown in red and the scaffold in green.
References
1. N. Eswar, M. A. Marti-Renom, B. Webb, M. S. Madhusudhan, D. Eramian, M. Shen, U. Pieper, A. Sali. Comparative Protein Structure Modeling With MODELLER. Current Protocols in Bioinformatics, John Wiley & Sons, Inc., Supplement 15, 5.6.1-5.6.30, 2006.
2. M.A. Marti-Renom, A. Stuart, A. Fiser, R. Sánchez, F. Melo, A. Sali. Comparative protein structure modeling of genes and genomes. Annu. Rev. Biophys. Biomol. Struct. 29, 291-325, 2000.
3. A. Sali & T.L. Blundell. Comparative protein modelling by satisfaction of spatial restraints. J. Mol. Biol. 234, 779-815, 1993.
4. A. Fiser, R.K. Do, & A. Sali. Modeling of loops in protein structures, Protein Science 9. 1753-1773, 2000.
Previous Next
Retrieved from " http://2014.igem.org/Team:Goettingen/project_overview/project_drylab "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
