"
Page
Discussion
View source
History
teams
Log in
 
Team:Goettingen/protocol Plasmid Con
From 2014.igem.org
Home
Project ▾
Background
Our project
Results
Perspectives
BioBricks
Notebook ▾
Safety
Wet lab
Dry lab
Materials
Protocols
Our Team ▾
Team members
Instructors
Acknowledgements
Sponsors
Outreach ▾
Human Practice
Gallery
Video
Presentation
Paris Bettencourt Newsletters
German teams' proposal
Achievements
Overview
PCR Methods
PfuS PCR
Fusion PCR
E.coli colony PCR
Yeast colony PCR
Plasmid Construction
Restriction of DNA
Ligation of DNA fragments
BP recombination reaction
LR recombination reaction
SEAMLESS Cloning
Peptide Library construction
Plasmid Transformation
E.coli competent cells
Plasmid isolation (E.coli)
E.coil transformation
Plasmid isolation (Yeast)
Yeast transformation
Colony Scanning
E.coli cracking
Auto-activity Test
Yeast-Two-Hybrid Assay with Robot
Yeast-Two-Hybrid Screening
Protein Assessment
SDS PAGE
Coomassie Staining
Protein Expression
Protein Purification
Protein Dialysis
Bradford Assay
In vivo tests
Production of spore suspension
Cell cultivation
Immunofluorescence Staining
Calcofluor staining
Restriction of DNA
1. Use up to 1 µg of DNA in a 20 µl reaction volume. Add 1 µl enzyme, 2 µl 10x recommended buffer and add up to 20 µl water.
2. Incubate for at least 1 hour at 37°C (or enzyme specific temperature).
3. Following the incubation, add 1.5 µl SAP (shrimp alkaline phosphatase) to plasmid backbones for 5’‑dephosphorylation and incubate again at 37°C for 10-30 minutes.
4. Stop reaction by heat-inactivation at 65°C for 5 minutes.
Ligation of DNA fragments
1. Measure the concentration of fragments which should be used for the ligation reaction.
2. Calculate the ratio between insert and backbone (1:1) using the following formula:
3. Pipette all things together. Use 2 µl of 10x T4 Ligation buffer (stored at -20°C, ATP containing) and at least 1 µl T4 ligase.
4. Incubate your reaction mix for 1 hour at room temperature or over night at 16°C.
5. Use at least 4 µl of the reaction mixture for transformation. (Previous purification of the ligation product can avoid false positive colonies!)
BP recombination reaction
Perform a BP recombination reaction between an attB-flanked DNA fragment and an attP-containing donor vector to generate an entry clone:
1. Add the following components to a 1.5 ml microcentrifuge tube at room temperature and mix:
     attB-PCR product    0.6 μl
     pDONR™ vector (supercoiled, 150 ng/μl)    0.2 μl
2. Vortex BP Clonase™ enzyme mix briefly.Add 0.2 μl to the components above and mix well by vortexing briefly twice.
3. Immediately back into -80°C!
4. Incubate reaction at 25°C for 1 hour.
5. Add 2 μl of 2 μg/μl Proteinase K solution and incubate at 37°C for 10 minutes.
6. Transform competent E. coli and select for the appropriate antibiotic-resistant entry clones.
LR recombination reaction
Perform an LR recombination reaction between an attL-containing entry clone and an attR-containing destination vector to generate an expression clone:
1. Add the following components to a 1.5 ml microcentrifuge tube at room temperature and mix:
     Entry clone (supercoiled, 100-300 ng)    0.6 μl
     Destination vector (supercoiled, 150 ng/μl)    0.2 μl
2. Vortex LR Clonase™ enzyme mix briefly. Add 0.2 μl to the components above and mix well by vortexing briefly twice.
3. Immediately back into -80°C!
4. Incubate reaction at 25°C for 1 hour.
5. Add 2 μl of 2 μg/μl Proteinase K solution and incubate at 37°C for 10 minutes.
6. Transform competent E. coli and select for the appropriate antibiotic-resistant expression clones.
SEAMLESS Cloning
1. Construct primers for the desired construct with 15 bp overhangs for homologous recombination later.
2. Amplify DNA fragments, purify them and measure concentration.
3. Calculate the amount of PCR product amount you need with the formula:
     ng of insert = (2 x bp of insert x 100 (ng vector))  /  bp of vector
4. Pipette 100 ng of vector and the calculated amount of insert together. Scale down the reaction from 20 µl to the smallest volume possible.
5. Add 5 fold buffer and last the 10 fold enzyme mix.
6. Incubate for 30 min at room temperature.
7. Cool the sample on ice for not more than 5 minutes.
8. Use the whole sample for transformation.
Peptide Library Construction
Starting with GFP scaffolds (Denoted pRS 316-GFPM), random loop libraries were created. Randomized  amino acids were inserted at Asp-102/Asp-103 and Glu-172/Asp-173 by using the NNK method in order to create the random loop libraries. NNK sequences in primers number 13 and 16 were used to create regions I and ӀӀӀ. After having randomized DNA sequences, these two regions were assembled with region ӀӀ which remained intact. In each of the amplification and extension steps, Pfus PCR protocol was used. For the assembly of the three regions, a modified fusion PCR was used.
Figure. Peptide Library Construction
References
Pavoor, Tej V., Yong Ku Cho, and Eric V. Shusta. "Development of GFP-based biosensors possessing the binding properties of antibodies." Proceedings of the National Academy of Sciences 106.29 (2009): 11895-11900.
Previous Next
Retrieved from " http://2014.igem.org/Team:Goettingen/protocol_Plasmid_Con "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
