"
Page
Discussion
View source
History
teams
Log in
 
Team:Goettingen/project overview/project wetlab
From 2014.igem.org
Home
Project ▾
Background
Our project
Results
Perspectives
BioBricks
Notebook ▾
Safety
Wet lab
Dry lab
Materials
Protocols
Our Team ▾
Team members
Instructors
Acknowledgements
Sponsors
Outreach ▾
Human Practice
Gallery
Video
Presentation
Paris Bettencourt Newsletters
German teams' proposal
Achievements
Project
Overview
Background
The global burden of fungal infections
Current diagnostic tools
Our project
Results
Wetlab
Drylab
Perspectives
Diagnosis
Therapeutics
BioBricks
中文
Results: Wetlab
Approach
Our experimental approach can be summarized as follows:
1. Organisms and surface proteins selection
2. Surface protein amplification and cloning
3. Yeast two-hybrid assay with an existing peptide library
4. Peptide functionalization
Results summary
We chose the five fungi Aspergillus fumigatus, A. nidulans, Candida glabrata, C. albicans and Saccharomyces cerevisiae as well as the parasite Toxoplasma gondii for our project. In the first step, we amplified all in all 35 different genes encoding surface proteins of the mentioned organisms in a standard PCR reaction (compare table 1). We were able to transform 20 of our 35 genes into a so called bait vector (contains a DNA-binding domain) and performed the Yeast-Two-Hybrid (Y2H) screening with them. Thereby, we accomplished to find more than 3000 probable interaction partners for our surface proteins.
Table 1| Selected surface proteins and the corresponding organism.
Gene
Organism
mp65
C. albicans
tos1
C. albicans
sim1
C. albicans
als1
C. albicans
als3
C. albicans
ssr1
C. glabrata
Click to expand
pir4
C. glabrata
scw4
C. glabrata
pir3
C. glabrata
utr2
C. glabrata
crf2full
A. fumigatus
crf2act
A. fumigatus
bgleno
A. fumigatus
ecm33
A. fumigatus
sun1
A. fumigatus
roda
A. fumigatus
bgleex
A. fumigatus
prm1 b
A. fumigatus
crf1
A. fumigatus
roda
A. nidulans
xlna
A. nidulans
npc2
A. nidulans
sho1
A. nidulans
eglc
A. nidulans
eglc
A. fumigatus
CWP1
S. cerevisiae
CWP2
S. cerevisiae
TIR4
S. cerevisiae
TIR1
S. cerevisiae
MID2
S. cerevisiae
SED1
S. cerevisiae
ama1
T. gondii
rom5
T. gondii
rom4
T. gondii
gra7
T. gondii
The results for all the genes we selected are summarized in this file.
To exclude “false positive” interaction partners (e.g. caused by a very weak interaction) we picked grown colonies and transferred them onto plates with a selection medium that allows only yeast cells containing peptides with a strong affinity to our surface proteins to grow. After this procedure and several previous selection steps, we still had around 70% of our interaction partners from the first screening.
To have a manageable amount of samples, we chose 10 of the interaction partners (peptides) which we found in the Y2H screen for each of our 20 surface proteins and used them for further analysis.  The cells containing the plasmids were plated on selection plates and the Y2H screen was repeated to finally confirm the interaction between one of our surface proteins and a specific peptide. This time, we used a robot to fasten up the screen and to be able to analyze a large number of possible interactions at the same time.
Interactions could be verified in 70% of all tested interaction partners!
In the last step, we tested the interaction of our surface proteins and the peptides in in vivo experiments.  Previously, we chose some of the peptides and sequenced them to get to know the sequence and to exclude the possibility that we isolated the same peptides in different samples. The sequences encoding the peptides were introduced into an over-expression vector for E. coli; thereby, we were able to get large amounts of our peptides which we called PepTags from then on. In the last step for the time being, we tested the fungi A. fumigatus, A. nidulans and C. glabrata with peptides, which are supposed to interact specific with one of their surface proteins in vivo using fluorescence microscopy.
With the in vivo experiments we want to proof that the identified PepTags not only bind to their surface proteins in an Y2H screen, but also to living fungi. To visualize the fungal surface itself, we stained structures containing chitin and cellulose in the fungal cell wall with calcofluor (figure 1C and 2C). The first organism with whom the functionality of our system was tested was A. fumigatus and the PepTag No.13.  Binding of this PepTag to the fungi was visualized with an antibody fused to a fluorescent molecule (Chromeo 488, figure 1A and 2A).
Figure 1| Fluorescence microscopy of Aspergillus fumigatus hyphae. This figure shows an A. fumigatus hyphae incubated with PepTag13, antibody and stained with calcofluor. Under light with a wavelength of 488 nm, the complexes containing PepTag13 and antibody show fluorescence. The excitation of calcofluor at a wavelength of 358 nm allows the identification of the hyphal surface (blue light).  The second picture shows the hyphae in a bright field, while the merged picture demonstrates that the green fluorescence is located in the same area as the blue one.
We were able to show that our peptide binds specifically to the surface of A. fumigatus, which is visualized by the green dots along the outer surface of the fungal cells. We also tested PepTag No.13 with A. nidulans (figure 2) but could not see any fluorescent signal. That means that our PepTag No.13 binds not only specific to the fungal surface, but also specific to the surface protein Sun1 of A. fumigatus. Moreover, we could show that the antibody cannot bind unspecific to the fungal surface when the PepTag is missing . For further information have a look at our lab-book.
Figure 2| Fluorescence microscope picture of an Aspergillus nidulans hyphae. This figure shows an A. nidulans hyphae incubated with PepTag No.13, antibody and calcofluor. Upon irradiation with light of 488 nm, no fluorescence occurs (first picture). The excitation of calcofluor at a wavelength of 358 nm allows identification of the surface of the hyphae (blue light). The PepTag cannot bind to the surface of A. nidulans and seems to be specifically to the surface of A. fumigatus.
Finally, it was possible to identify a small peptide which binds specific to the surface of fungal hyphae. Moreover this PepTag is specific for the chosen fungus.
Your browser does not support the video tag.
Results: Dry lab
Homology modelling
We modeled the 3D structure of our peptide-containing scaffolds (domain B1 of protein G from Staphylococcus aureus) by homology modelling. We used the Modeller library to accomplish that; since the peptide is about 20-25 amino acids long, it can be modeled as a loop inside the B1 domain scaffold for which there are already a number of 3D structures available at the Protein Data Bank.
This is the general homology modeling procedure we followed. All peptides were cloned inside a scaffold (B1 domain of protein G from Staphylococcus aureus. The peptides were short enough to be modeled as an internal loop by loop refinement; these structures should only be considered as a working model, since we do not have direct crystallographic data. A detailed procedure can be found in this link.
Results summary
Protein G scaffold
The table 2 is a summary of the models we generated for our peptides inside the protein G scaffold. Their main purpose is to give us an idea if the calculated energy profile matches our assumption that the peptide is being exposed to the exterior of the scaffold. These models are in no way definitive, since we lack direct crystalographic data. As can be seen, all the structures show two distinct domains: one for the protein G scaffold and the other for the peptide.
Table 2| Homology modeling for the peptides in a protein G scaffold.
Peptide
Video*
PDB
Peptide location
QMEAN score
Overall DOPE score
Energy profile
Prey 3
Your browser does not support HTML5 video.
Download
K17-E42
0.512
-7053.966797
Download
Prey 4.1
Your browser does not support HTML5 video.
Download
K17-E44
0.468
-7246.956055
Download
Prey 5
Your browser does not support HTML5 video.
Download
K17-E44
0.495
-7501.071777
Download
Prey 13
Your browser does not support HTML5 video.
Download
K17-E42
0.535
-7729.056152
Download
Prey 15
Your browser does not support HTML5 video.
Download
K17-E42
0.441
-7685.998535
Download
*The peptide is shown in red and the scaffold in blue.
GFP scaffold
Table 3 shows the structures we generated for our peptides inside a GFP scaffold. In contrast to the models with the protein G scaffold, the GFP-peptide models show discontinuities.
Table 3| Homology modeling for the peptides in a GFP scaffold.
Peptide
PNG*
PDB
Peptide location
QMEAN score
Overall DOPE score
Energy profile
Prey 3
Download
E103-E127
0.43
-26798.28125
Download
Prey 4.1
Download
E103-E127
0.448
-27891.212891
Download
Prey 5
Download
E103-E127
0.483
-26633.4375
Download
Prey 13
Download
E103-E127
0.404
-27075.160156
Download
Prey 15
Download
E103-E127
0.413
-26713.355469
Download
*The peptide is shown in red and the scaffold in green.
References
1. N. Eswar, M. A. Marti-Renom, B. Webb, M. S. Madhusudhan, D. Eramian, M. Shen, U. Pieper, A. Sali. Comparative Protein Structure Modeling With MODELLER. Current Protocols in Bioinformatics, John Wiley & Sons, Inc., Supplement 15, 5.6.1-5.6.30, 2006.
2. M.A. Marti-Renom, A. Stuart, A. Fiser, R. Sánchez, F. Melo, A. Sali. Comparative protein structure modeling of genes and genomes. Annu. Rev. Biophys. Biomol. Struct. 29, 291-325, 2000.
3. A. Sali & T.L. Blundell. Comparative protein modelling by satisfaction of spatial restraints. J. Mol. Biol. 234, 779-815, 1993.
4. A. Fiser, R.K. Do, & A. Sali. Modeling of loops in protein structures, Protein Science 9. 1753-1773, 2000.
Previous Next
Retrieved from " http://2014.igem.org/Team:Goettingen/project_overview/project_wetlab "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
