"
Page
Discussion
View source
History
teams
Log in
 
Team:Goettingen/project overview/project biobrick
From 2014.igem.org
Home
Project ▾
Background
Our project
Results
Perspectives
BioBricks
Notebook ▾
Safety
Wet lab
Dry lab
Materials
Protocols
Our Team ▾
Team members
Instructors
Acknowledgements
Sponsors
Outreach ▾
Human Practice
Gallery
Video
Presentation
Paris Bettencourt Newsletters
German teams' proposal
Achievements
Project
Overview
Background
The global burden of fungal infections
Current diagnostic tools
Our project
Results
Wetlab
Drylab
Perspectives
Diagnosis
Therapeutics
BioBricks
BioBrick Tools
Basic parts - auxotrophic selection marker for transformed yeasts
We started by searching parts and DNA that could be used for working with yeast and other fungi. However, there were only few data related to this kind of organism, actually we could not find any selection marker for yeast in the delivered Biobrick plates. This situation took our efforts to create a Biobrick which allows the selection of positively transformed cells by means of auxotrophic markers. We are working with strains that are deficient of either the LEU2 gene or the TRP1 gene. Both of them are essential during amino acid biosynthesis of leucine and tryptophane respectively . The TRP1 gene encodes for a phosphoribosyl-anthranilate isomerase, an enzyme needed in the third step of tryptophane synthesis. The LEU2 gene codes the β isopropylmalate dehydrogenase, which is necessary for the third step in the biosynthesis of leucine. Therefore, these 2 specific genes were chosen for BioBrick construction. For amplification of plasmids in yeasts, we also created the 2 micron ori (see table 1) for plasmid replication in the yeast cells.
Table  1| Created basic parts for registry.
We used genomic DNA of S. cerevisiae to amplify the genes. We checked the sequences and noticed that most of the yeast genes contain EcoR1 or Xba1 restriction sites; this could be the main reason of the absence of many BioBricks of such type. Hence, we decided to mutagenize our genes with primers that allow the introduction of a point mutation (shown in figure 1). With this mutation, a so called silent mutation, the amino acid sequence will be the same, but the restriction site will be deleted.
Figure 1| Workflow for the construction of selection marker for yeast. A| Selection of the gene coding for the phosphoribosyl-anthranilate isomerase (TRP1) in the pathway of L-tryptophane synthesis and the gene for β isopropylmalate dehydrogenase (LEU2) in the pathway of the amino acid L-leucine. Both contain restriction sites not conform to the RFC10. B| Amplification of the genes in two fragments. Here, the specific sequence for restriction sites has been deleted using mutagenic primers. These primers contain a mismatch that creates a silent mutation at a specific site in the DNA (base pair is changed - amino acid sequence is not). C| In the last step, we used a set of primers that add prefix and suffix sequences and fused the two fragments of each gene. The obtained genes no longer contain restriction sites (*).
We cloned these genes into the BioBrick backbone pSB1C3 so that it can be introduced into the library.
Composite parts - device to tag proteins in yeast with GFP or mRFP
Furthermore, we intend to create a tool for yeast, which is made out of BioBricks, in order to be able to tag proteins in yeast or fungal cells with different fluorescent markers. We chose two promoters (TEF2 and TDH3) and two terminators (ADH1 and HHO1) from the BioBrick library (full list of used BioBricks in table 2).
Table  2| Used BioBricks for amplification of parts via PCR.
The genes for mRFP and GFP were chosen as fluorescent markers and were integrated downstream of and 19 bp long sequence that contain the BamH1/Bgl2 restriction site. The gene of interest (GOI) follows directly behind this adaptor (compare figure 2).
Figure 2|Functional device to tag proteins in yeasts with different fluorescent proteins. The first part of this is the selection marker that allows selection after transformation (in LEU2 or TRP1 deficient strains). The second part is the major part. There the restriction site for two enzymes allows easy introduction of proteins of interest into yeast and tag them with GFP or either mRFP. Color code: red - terminator; green - promoter; orange - adapter; blue - protein.
In order to get a functional and successfully expressed plasmid, we added the mutated 2-micron ori of yeast into the plasmids we got so far. The ori was introduced downstream of the ADH1 terminator (see figure 3 for plasmid maps).
Figure 3| The final plasmids of our composite parts. The TRP-GFP- tool can be used to select for tryptophane auxotrophy and to tag a protein of interest in yeast with GFP. The LEU-mRFP-tool can be used to select for leucine auxotrophy and to tag a protein of interest in yeast with mRFP. These two BioBricks can also be used together, to analyse two different proteins in yeast.
Intermediate parts
In the first step, we incorporated the TDH3 promoter upstream of our selection marker and inserted the terminator HHO1 downstream. This promoter-marker-terminator brick could also be used as a single part for other purposes (intermediate parts, table 3).
Table  3| Intermediate parts that we constructed.
In the second step, we introduced the TEF2 promoter upstream of the BamH1/Bgl2 site, the fluorescent marker and the ADH1 terminator (see figure 2).  In the last step we added in this construction the mutated 2 micron (compare creation of TRP1 and Leu2 genes, figure 1) and finalized our two plasmids as functional device (figure 3).
Previous
Retrieved from " http://2014.igem.org/Team:Goettingen/project_overview/project_biobrick "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
