"
Page
Discussion
View source
History
teams
Log in
 
Team:Kyoto/Hunger
From 2011.igem.org
Home
Project
Hunger
Hunger Modeling
Capture
Digestion
Digestion Modeling
Consideration
Safety
Human Practice
Method
Parts
Protocol
Material
Team
Notebook
Lab Work
Diary
Acknowledgement
Criteria
Contents
1 Project Hunger
1.1 Introduction
1.1.1 Mechanism of Regulation of Transcription Depending on the Concentration of Glutamine
1.2 Method
1.3 Result
1.3.1 New equation for measurement RPU
1.3.2 How long E.coli can keep steady state after transferred to medium lack of nutrition
1.4 Discussion
1.5 Reference
Project Hunger
Carnivorous E.coli attracts insects by emitting light, but it is a burden for the E.coli. To reduce this burden, we use nitrogen regulatory proteins, NtrB and NtrC. They activate σ54 promoter when the supply of nitrogen is not enough. NtrB and NtrC are coded in glnL and glnG, respectively.
† NtrB and NtrC are otherwise called NRII, NRI
Introduction
Mechanism of Regulation of Transcription Depending on the Concentration of Glutamine
Ammonia is an essential nitrogen source for the bactria. When enteric bacteria are deprived of ammonia, they express glnA to produce glutamine synthetase(GS) under the σ54promoter. The transcription from σ54promoter is stimilated by phosphorylated form of NtrC(NtrC-P). The σ54 RNA polymerase binds to the glnA promoter, forming a closed complex, but cannot form an open complex and initiate transcription until it is activated by NtrC-P. NtrC is phosphorylated by NtrB-P, an autokinase which phosphorylates itself with ATPs. Phosphorylation and dephosphorylation of NtrB and C are controlled so that a cell has sufficient NtrC-P when the concentration of ammoniacal source is low.
Fig.1 Regulation of σ54promoter
Fig.2 Cycles of NtrB and NtrC
The concentration of ammoniacal source is detected by the ratio of α-ketoglutarate to glutamine. If glutamine levels are low, less α-ketoglutarate is synthesized by GS and, as a result, Pii retains UMP and so cannot bind to NtrB. NtrB can then phosphorylate itself and transfer this phosphate to NtrC.
Fig.3 Detection of Ammoniacal Source
[GS Reaction Under Low NH3 Concentration] Glutamate + NH3 + ADP → Glutamine + ADP + phosphate                       GS Glutamine + αketoglutarate → 2 glutamate                             GS
In our project, we decided to use NtrB and NtrC to control the expression level of genes for the following reasons:
They were not submitted to the partsregistry.
Many researches have been done on NtrB and NtrC, while there are not as many reports on Pii and GS.
The cell signalling mechanism can be made simple.
σ54promoter was not quantitatively characterized.
We characterized the σ54promoter by Relative Promoter Unit (RPU), because absolute promoter activity depends on test conditions and measuring instruments. RPU can reduce this Coefficient of Variation (CV) from 39.1% to 17.5% [2]. Therefore RPU can make it easier for us to share the data of promoter activity and use BioBricks.
We have used GFP fluorescence to measure RPU previously, but this time, we tried to calculate RPU using another way for the reasons below:
We know that it is a lot of trouble to calculate RPU using GFP fluorescence without platereader.
As the σ54promoter relates to the metabolism of glutamine, the change in the concentration of glutamine results into the change of activity of σ54promoter.
When we calculate RPU using GFP fluorescence, we need to measure GFP fluorescence at two points in an exponential growth phase and on the same glutamine concentration, but we have the following problems.
before E.coli reach exponential growth phase, the concentration of glutamine changes
the concentration of glutamine is different between one point and the other point.
We devised the new way of calculating RPU using the amount of mRNA, according to the following equation.
We can characterize promoters with RPU easily by using this new way.
Method
We created the following two constructions to characterize σ54 promoter with RPU depending on the concentration of glutamine. One construction includes BBa_J23101 as a promoter which is used as the standard and the other includes binding sites and σ54 promoter.
Result
New equation for measurement RPU
We get this new equation for measuring RPU.(in steady state)
(To see the derivation, see Team:Kyoto/Hunger/Modeling .)
How long E.coli can keep steady state after transferred to medium lack of nutrition
To measure RPU by our new method, we need to know how long E.coli can keep steady state after transferred to medium lack of nutrition. We think that cell population and gene expression keep in steady state. So we measured OD600 and expression of stationary phase specific gene, csiE and bolA.
In order to measure the RPU of σ54promoter depending on glutamine concentration, the cellular density of mRNA must be measured when it is constant(this means that the concentration of glutamine is kept invariable). So, E.coli must be in the steady state. However, E.coli consumes glutamine and it is impossible to add glutamine at the first point of cultivation. Cultivation without any glutamine is necessary. Also, From our last year's experience, we knew that E.coli grew really inefficiently in M9 medium without glutamine, which is often suplied from the casamino acids. Thus, first we cultivated E.coli with casamino acids overnight, then exchanged the medium with that is free of casamino acids.
As a preliminary experiment of measurement of the RPU of σ54promoter, we performed an experiment to determine how long it takes to reach a steady state.
We cultivated E.coli in M9 media(+ casamino acids) for about 15 hours and dispenced 2.4ml to each tubes. Then, we centrifuged these tubes (13,000 rpm , 4°C, 1min) and discarded the supernatant. We added 2.4ml media(- casamino acids) and centrifuged at 4°C twice. Again, we centrifuged these tubes and discarded the supernatant and added 2.4ml media(-casamino acids) at 37 °C. We brought up E.coli at 0,5,10,15,20,25,30,60min and mesured OD600 and extracted RNA and synthesized cDNA. Finally, we used real time PCR.
Following table is the result of OD600 in the preliminary experiment.
over night culture
0min
5min
10min
15min
20min
25min
30min
60min
OD600
1.643
1.600
1.605
1.594
1.563
1.550
1.587
1.595
1.653
To use real time PCR, we need to choose internal control to correct data. We use GAPDH, TBP, PGK, and actin as candidate of internal control and research which gene is suitable to internal control.
Following graphs are the results of real time PCR in the preliminary experiment.
Fig.1 :graph1. the relative RNA expression level
This table is the relative RNA expression level used in graph1.
0min
5min
10min
15min
20min
25min
30min
60min
GAPDH
1
1.15796
1.477987
0.669569
0.216789
0.123519
0.158918
0.090074
TBP
1
1.586912
3.103184
36.00222
16.53465
16.81689
26.70678
14.0751
PGK
1
2.764137
2.807991
1.45171
0.473917
0.235602
0.689141
0.309861
actin
1
1.31823
1.743549
0.626175
0.316547
0.152318
0.191665
0.125879
Fig.2 :graph2. the relative RNA expression level except TBP
This table is the relative RNA expression level except TBP used in graph2.
0min
5min
10min
15min
20min
25min
30min
60min
GAPDH
1
1.15796
1.477987
0.669569
0.216789
0.123519
0.158918
0.090074
PGK
1
2.764137
2.807991
1.45171
0.473917
0.235602
0.689141
0.309861
actin
1
1.31823
1.743549
0.626175
0.316547
0.152318
0.191665
0.125879
Fig.3 :graph3. the relative RNA expression level corrected by PGK
This table is the relative RNA expression level corrected by PGK used in graph3.
0min
5min
10min
GAPDH
1
0.445449
0.588774
TBP
1
0.60403
1.191124
actin
1
0.520922
0.680571
Fig.4 :graph4. the relative RNA expression level corrected by TBP
This table is the relative RNA expression level corrected by TBP used in graph4.
0min
5min
10min
GAPDH
1
0.733254
0.478325
PGK
1
1.702282
0.897464
actin
1
0.84523
0.562937
Fig.5 :graph5. the relative RNA expression level corrected by GAPDH
This table is the relative RNA expression level corrected by GAPDH used in graph5.
0min
5min
10min
TBP
1
1.365421
2.127406
PGK
1
2.337666
1.973042
actin
1
1.149192
1.188311
Fig.6 :graph6. the relative RNA expression level corrected by actin
This table is the relative RNA expression level corrected by actin used in graph6.
0min
5min
10min
GAPDH
1
0.877036
0.84605
TBP
1
1.20119
1.782596
PGK
1
2.086329
1.623779
From these result, we chose GAPDH and actin as internal control. Next, we measured csiE and bolA, stationary phase specific gene, expression level to confirm whether 0~10 minutes after transferred is steady state.
Fig.7 :graph7. the csiE, stationary phase specific gene, expression level corrected by GAPDH or actin
This table is the csiE, stationary phase specific gene, expression level corrected by GAPDH or actin used in graph7.
csiE
0min
5min
10min
15min
corrected by GAPDH
1
1.225641019
0.783607523
5.373500165
corrected by actin
1
0.979871284
0.711422654
5.451538605
Fig.8 :graph8. the bolA, stationary phase specific gene, expression level corrected by GAPDH or actin
This table is the bolA, stationary phase specific gene, expression level corrected by GAPDH or actin used in graph8.
bolA
0min
5min
10min
15min
corrected by GAPDH
1
0.938259866
0.516878973
3.324786006
corrected by actin
1
0.750116784
0.469264778
3.373071314
Discussion
Graph1 shows the relative RNA expression level(GAPDH,TBP,PGK,actin)  which is not corrected by internal control gene.  Although the initial RNA expression level is not equal, it can be said that more than twice the difference is not an accidental error of the experiment. Since the expression level of TBP becomes 35 times higher than its initial amount in 15 minutes, we can say that TBP is not suitable for the internal control and that E.coli is not in the steady state after 15 minutes in this experiment.
Graph2 is the same data with the graph1 except TBP. Since the expression level of PGK changes a lot, PGK is not suitable for the internal control either. In 0-10 minutes the expression level of GAPDH and actin doesn’t change so much and behave similarly.
Graph3-6 show the data corrected by each internal control. When GAPDH is used as the internal control, the change of actin is little (x1.0~1.2) When actin is used as the internal control, the change of GAPDH is little (x0.8~1.0) According from these data, we assumed that GAPDH and actin is suitable for the internal control and that steady state is probably 0~10 minutes.
Graph7,8 is the data of stationary phase specific genes corrected by GAPDH or actin as internal control. These graphes show that 10 minutes is not steady state but 0~5 minutes is steady state. So, we know the timing of steady state and can measure RPU with our new way.
Reference
[1] P. Jiang and A. J. Ninfa, “Escherichia coli PII signal transduction protein controlling nitrogen assimilation acts as a sensor of adenylate energy charge in vitro.,” Biochemistry, vol. 46, no. 45, pp. 12979-96, Nov. 2007.
[2] J. R. Kelly et al., “Measuring the activity of BioBrick promoters using an in vivo reference standard.,” Journal of biological engineering, vol. 3, p. 4, Jan. 2009.
[3] M. Al-Azri, H. Al-Azri, F. Al-Hashmi, S. Al-Rasbi, K. El-Shafie, and A. Al-Maniri, “Factors Affecting the Quality of Diabetic Care in Primary Care Settings in Oman: A qualitative study on patients’ perspectives.,” Sultan Qaboos University medical journal, vol. 11, no. 2. pp. 207-13, May-2011.
[4] L. J. Reitzer and B. Magasanik, “Transcription of glnA in E. coli is stimulated by activator bound to sites far from the promoter.,” Cell, vol. 45, no. 6, pp. 785-92, Jun. 1986.
[5] B. Magasanik, “Regulation of transcription of the glnALG operon of Escherichia coli by protein phosphorylation.,” Biochimie, vol. 71, no. 9-10, pp. 1005-12, 1989.
[6] J. Keener and S. Kustu, “Protein kinase and phosphoprotein phosphatase activities of nitrogen regulatory proteins NTRB and NTRC of enteric bacteria: roles of the conserved amino-terminal domain of NTRC.,” Proceedings of the National Academy of Sciences of the United States of America, vol. 85, no. 14, pp. 4976-80, Jul. 1988.
[7] J. R. Kelly et al., “Measuring the activity of BioBrick promoters using an in vivo reference standard.,” Journal of biological engineering, vol. 3, p. 4, Jan. 2009.
Retrieved from " http://2011.igem.org/Team:Kyoto/Hunger "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
