Team:Lethbridge/Results
Project
Description
Design
Demonstrate
Results
Collaborations
Parts
Overview
Part Collection
Composite
Improved Part
Basic
Human Practices
Overview
User Groups - Silver
Integrated HP - Gold
Biosecurity Software
Engagement
Lab
Experiments
Notebook
Safety
InterLab
Team
Team Members
Attributions
Sponsors
top
Overview
Provided simplified protocols and learning tools for the education system in collaboration with the Lethbridge High School Team
Addressed policy issues regarding cell-free systems and the biosecurity implications of our project
Developed and tested software tools for biocontainment and to mitigate dual-use
Improved 5 parts by optimizing coding sequences for optimal expression in E. coli, adding hexahistidine tags for simple purification and improving part characterization
Demonstrated proof of concept for multi-protein purification to simplify the system
17 parts characterized and documented
Worked closely with user groups to inform our design
Assisted Lethbridge High School with wet lab work and received help with education interviews
Collaborated with Florida State University by providing mutual project feedback
Team is registered!
Project is showcased on wiki
Project attributions clearly detailed
Safety forms submitted
Judging form completed
Parts documented on the registry
9 parts submitted
Participated in the InterLab study
Part Collection: We have provided a collection of open-source parts for cell-free protein expression
Integrated Human Practices: The design of Next vivo was informed by the needs of three user groups, resulting in the development of an educational tool and an assessment of the biosecurity implications of cell-free systems
Public Engagement and Education: In celebration of our 10th anniversary as an iGEM team, we attended events in our community to share our work and developed curriculum aligned lesson plans, as well as a safe learning tool for the education system
Software: We addressed the dual-use potential of our system and developed software tools as a solution to mitigate biosecurity threats
Proteins
Overexpressions
We conducted overexpressions of our constructs in BL21-Gold (DE3), an E. coli strain capable of overexpressing T7 RNA polymerase by induction with IPTG (see design page for details). All overexpression characterization is documented on each individual parts page.
In total, we successfully overexpressed:
BioBrick
Protein
BioBrick
Protein
BBa_K2443001
ArgRS
BBa_K2443022
MTF
BBa_K2443003
AspRS
BBa_K2443027
EF-Tu
BBa_K2443007
GlyRSα
BBa_K2443028
EF-Ts
BBa_K2443008
GlyRSβ
BBa_K2443031
RF3
BBa_K2443009
HisRS
BBa_K2443032
RRF
BBa_K2443013
MetRS
BBa_K2443033
MK
BBa_K2443014
PheRSα
BBa_K2443034
CK
BBa_K2443017
SerRS
BBa_K2443037
T7 RNA Polymerase
BBa_K2443019
TrpRS
As an example, Figure 1 showcases the overexpression of four individual TX-TL proteins.
Figure 1 - Overexpressions of key proteins required for TX-TL systems. Overexpressions of proteins  in E. coli BL21-Gold (DE3) visualized on a 12% SDS-PAGE for 80 min at 200 V. Lanes are as follows: 1- Protein ladder; 2- HisRS; 3- HisRS induced with IPTG; 4- TrpRS; 5- TrpRS induced with IPTG; 6- RF3; 7- RF3 induced with IPTG; 8- RRF; 9- RRF induced with IPTG. Protein sizes: HisRS 48.3 kDa; TrpRS 38.8 kDa; RF3 60.9 kDa; RRF 21.9 kDa.
Multi-Protein Purification
One of the key features of Next vivo is the ability to purify all of the components in a single step purification . As a proof of concept we expressed four of the TX-TL components and co-purified them all using a Nickel Sepharose chromatography column (Figure 2). The four proteins used in this initial test were selected based on their molecular weights relative to each other for visualization purposes.
Figure 2 - Representative overexpression and multi-protein purification of TX-TL components. Each TX-TL component was expressed from E. coli cells carrying the plasmid encoding the specified component and samples three hours post induction were collected. The expressing cells of each component were pooled and lysed before applying the lysate to a Nickel Sepharose affinity column for isolation of just the hexahistidine tagged TX-TL components. After washing away the unwanted cellular proteins and debris, the TX-TL components were eluted from the Nickel Sepharose to a high level of purity.  Lanes are as follows: 1- Protein ladder; 2- HisRS overexpression; 3- TrpRS overexpression; 4- RF3 overexpression; 5- RRF overexpression; 6- HisRS, TrpRS, RF3 and RRF elution: HisRS 48.3 kDa; TrpRS 38.8 kDa; RF3 60.9 kDa; RRF 21.9 kDa
tRNA Purification
The biggest issue we initially faced in developing Next vivo was determining how we could purify tRNA quickly and efficiently. The solution we decided upon was an adapted MS2 RNA co-purification combined with a subsequent incubation with RNase H and a DNA oligo that would selectively cleave and release a tRNA of the proper size. For more information on the design,  see the tRNA purification section here.
Both the tRNAphe-MS2 aptamer construct and MS2BP were overexpressed individually in E. coli BL21-Gold (DE3) cells. Upon which time the cells were lysed, the lysate combined and applied to a Nickel Sepharose affinity column. In order to cleave the RNA, 1 µM of DNA oligo was added to the column, as well as varying amounts of RNase H. Incubation times on the column with RNase H and DNA oligo varied from 2 hours (Figure 3) to 12 hours (Figure 4), and the amount of RNase H used varied from 10 units to 100 units (Figure 3 and 4). Based upon the varied conditions, a longer incubation time had the greatest effect on tRNA cleavage efficiency with units of RNase H being optimal between the range of 5-50. With these improvements from our initial attempt at tRNA cleavage, we successfully purified tRNAPhe, as shown in Figure 4.
Figure 3 - Preliminary tRNAPhe Purification. 12% 8M urea PAGE run for 45 mins at 300 V. All concentrated fractions were phenol chloroform extracted, ethanol precipitated and re-suspended in 30 µL of ddH2O. Lanes are as follows: 1- tRNA fraction with 20 units of RNase H added; 2- concentrated tRNA fraction 20 units of RNase H added; 3- concentrated MS2 fraction 1 20 units of RNase H added; 4- concentrated MS2 fraction 2  20 units of RNase H added; 5- tRNA standard (76 nt).
Figure 4 - Successful tRNAPhe Purification. 12% 8M urea PAGE run for 45 mins at 300 V. All fractions were phenol chloroform extracted, ethanol precipitated and re-suspended in 30 µL of ddH2O. Lanes are as follows - 1- MS2 fraction 25 units of RNase H added; 2- tRNA fraction 25 units of RNase H added; 3- MS2 fraction 50 units of RNase H added; 4- tRNA fraction 50 units of RNase H added; 5- MS2 fraction 100 units of RNase H added; 6- tRNA fraction 100 units of RNase H added; 7- MS2 fraction 10 units of RNase H added; 8- tRNA elution 10 units of RNase H added; 9- tRNA standard (76 nt).
Ribosomes
Initial work to produce MS2 tagged ribosomes was carried out by attempting to isolate the 23S and 16S rRNA genes from the E. coli rrnB operon (see design page for details). The 23S rRNA is the major component of the  large ribosomal subunit and the 16S rRNA in the small ribosomal subunit. Amplifying (Figure 6) and inserting the rRNA genes in the BioBrick standard was the first step in inserting the MS2 hairpin into the ribosomal RNA. Ligations into pSB1C3 have thus far been unsuccessful.
Figure 5 - Amplification of the 23S and 16S ribosomal RNA genes from E. coli rrnB operon. PCR amplification of the 23S and 16S rRNA genes run on a 1% Agarose gel for 30 min at 100 V. Lanes are as follows: 1- 1 kb ladder; 2- 16S reaction before amplification; 3- 23S reaction before amplification; 4- 16S reaction; 5- 16S reaction; 6- Empty; 7- 23S reaction; 8- 23S reaction. The 16S rRNA is ~2000 bp and the 23S rRNA is ~2900 bp.
Validation Construct
The validation construct was designed to detect successful transcription or translation, or both simultaneously. In addition to providing the initial characterization of our system, this construct can be used as a measurement tool to test and standardize the performance of Next vivo. To confirm our ability to detect successful transcription, the Spinach aptamer was in vitro transcribed using T7 RNA polymerase (previously purified) and purified by phenol chloroform extraction (Figure 6A). Following addition of the fluorophore, DFHBI, fluorescence was measured using a fluorimeter. The fluorimeter data shows that fluorescence was observed following addition of DFHBI (Figure 6B), indicating that the Spinach RNA can be used as a measure of transcriptional activity. Next, to confirm our ability to detect successful translation, EYFP was expressed using the PURExpress system (Figure 7). Additional fluorescence analysis is still required to confirm EYFP can be used as a measure of translational activity.
Figure 6 - Characterization of the EYFP-Spinach Construct - Transcription. A In vitro transcribed Spinach RNA visualized on 1% agarose for 30 min at 100 V. Lane 1- 1kb ladder and Lane 2- Spinach mRNA. B Fluorimeter data illustrating green fluorescence following addition of DFHBI, with an excitation of 447 nm and emission of 497 nm.
Figure 7 - Characterization of the EYFP-Spinach Construct - Translation. In vitro expressed EYFP visualized on a 12% SDS-PAGE for 80 min at 200 V. Lanes are as follows: 1- EYFP (~30 kDa); 2- Protein Ladder
Statistics
