<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Lethbridge_Software skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Lethbridge/Software</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Project</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Description">Description</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Design">Design</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Demonstrate">Demonstrate</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Results">Results</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Collaborations">Collaborations</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="https://2017.igem.org/Team:Lethbridge/Parts">Parts</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Parts">Overview</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Part_Collection">Part Collection</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Composite_Part">Composite</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Improve">Improved Part</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Basic_Part">Basic</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Human Practices</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Human_Practices">Overview</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/HP/Silver">User Groups - Silver</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/HP/Gold_Integrated">Integrated HP - Gold</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Software">Biosecurity Software</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Engagement">Engagement</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Lab</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Experiments">Experiments</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Notebook">Notebook</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Safety">Safety</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/InterLab">InterLab</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="https://2017.igem.org/Team:Lethbridge/Team">Team</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Team">Team Members</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Attributions">Attributions</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Attributions#Sponsors">Sponsors</A></LI></DIV></DIV><DIV class="circleTop" style="margin-left:90%"><A href="#top">top</A></DIV><DIV class="container"><DIV class="row"><DIV class="col s12"><H1 style="margin-left: 15%; margin-right: 15%;"><SPAN style="font-weight:normal">One of these sequences encodes a toxin.</SPAN></H1><H1>Do you know which?</H1></DIV></DIV><DIV class="row"><DIV class="col s12 m6 l6"><P class="text12" style="word-wrap: break-word; font-family: Monaco">ACAGTTACACGGACAACAAGGTGTTCCAGGCTTCTTTCCTCCCTTCGACGATGTATTTCCAATAGGTGTAATCGTCGGCGAAATCGTGTTCGGTTCTCCCGACGACTGCGAAGGAAACGGATTGTTCGGTGTATTCGGCGTGTACAGCAGAATTTCACCCGACAGCGGTCCTTCTTCACCAAATCCCAGCGGCGGCGG</P></DIV><DIV class="col s12 m6 l6"><P class="text12" style="word-wrap: break-word; font-family: Monaco">CTGCGCGATGCTCGACGAGTCAATTCCGCTGTAGTACAGGAAGTCGTACGAGTCATTGCTATTGTATCAGCTAGAGATAATCGCGTACGCGCTCGAGCTCGAGCTATTTCGTCCTGAGCTGATGTCTCCGTCGATAATGAAAAATCCTCCGCTGATGTCCAGGTACAGACCCAGTCCGTCGTATCCTCCAATCGCTGA</P></DIV><DIV class="col s12 m6 l6"><P class="text12" style="word-wrap: break-word; font-family: Monaco">GGTGCGAAGATTGACGACCCGCTCTATATTCATCATGTGTGGCCGCATGACCCGACAATTACACATTTCATTTTAAAGCTCGCGCATGCGATTGACATTGACATTACACTATATGAAATTAAGCCGTATCCGAAGCTCTGGCGTTATTATATTAAGCCGGTGCATGTGTCTGTGTATCCGCATTATTATCTCGCGGAA</P></DIV><DIV class="col s12 m6 l6"><P class="text12" style="word-wrap: break-word; font-family: Monaco">AGGCACTTCCTACTTCTTAAGAAACGGCTAAGCAGCAGAGTTAAGAGCCTTAAGTCACTATCAAGCCCGCTAGTATTCAAACACAGCCACCTACTTCTACTTCTATCATGGCGGATGCTATTCAAGCGGAAGTTCAAAGTTTGCCGGCGGCTATTCAAGAGAAGCAGACCAAGACGGAAGAGCCGGCGGAAACACATG</P></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col s12"><H2 class="segmentHeader">The N<I>ex</I>t <I>vivo</I> Connection</H2></DIV></DIV><DIV class="row"><DIV class="col s12 m6"><H5>Rapid Cell-free Systems</H5><P class="text12">
          The guiding vision behind our project is to be able to provide an easy-to-purify cell-free system in order to bring the benefits of synthetic biology to the masses. To achieve this, we have provided methods to easily purify all of the necessary transcriptional and translational components. This includes essential proteins and RNAs, with a strong emphasis on transfer RNAs (tRNAs). In addition, because the N<I>ex</I>t <I>vivo</I> system lacks genomic DNA it cannot replicate or regenerate energy. It is essentially a simple protein production machine that translates a transcribed messenger RNA (mRNA). Because of these characteristics, N<I>ex</I>t <I>vivo</I> is highly amenable to genetic recoding.
        </P></DIV><DIV class="col s12 m6"><P class="text12">
          For a more comprehensive look at our system, check out our design page.
          </P><DIV align="center"><A class="waves-effect waves-light btn cyan darken-3" href="https://2017.igem.org/Team:Lethbridge/Design">NEXT VIVO DESIGN</A></DIV></DIV></DIV><DIV class="row"><DIV class="col s12 m6"><H5>Genetic Recoding</H5><P class="text12">
          The conversion from and RNA message to a protein is mediated by a set of evolutionarily conserved tRNAs that make up what we know as the &quot;Universeal Genetic Code.&quot; Genetic recoding then, is a process by which the conventionally understood relationships between codon-anticodon and tRNA-amino acid are altered. For example, the amber stop codon (UAG) can be reassigned to instead incorporate a standard or non-standard amino acid into a growing peptide. [1]</P><BLOCKQUOTE class="grey lighten-2">
            Accordingy, it follows that modifying the relationship between codon and amino acid incorporation is equivalent to the creation of a novel genetic code.
          </BLOCKQUOTE></DIV><DIV class="col s12 m6"><DIV class="card cyan darken-3"><DIV class="card-content white-text"><SPAN class="card-title">Genetic Recoding vs. Codon Reassignment</SPAN><P class="text12">There is some discussion surrounding the use of the term “Genetic Recoding” and “Codon Reassignment.” Becuase our system falls in between two proposed definitions, we have chosen to refer to the practice as “Genetic Recoding” in the context of our project and will refer to it accordingly.</P><P class="text12">
                You can read more about the distinction at the link below:
              </P></DIV><DIV class="card-action"><A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2921827/">The Distinction Between Recoding and Codon Reassignment</A></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col s12 m6"><P class="text12">
          Disrupting this relationship has numerous benefits including the expanding the available codon space to allow for the incorporation of non-standard amino acids into a system and biocontainment by designing orthogonal genes that are fundamentally incompatible with ordinary organisms. Read more about Biocontainment on our Design Page
        </P><DIV align="center"><A class="waves-effect waves-light btn cyan darken-3" href="https://2017.igem.org/Team:Lethbridge/Design#anchor8">BIOCONTAINMENT</A></DIV></DIV><DIV class="col s12 m6"><H5>Methods for Genetic Recoding</H5><P class="text12">
          As with all problems in biology, there is more than one way to achieve a goal. In the case of genetic recoding however, the constant involved is the manipulation of the tRNA.
        </P><P class="text12">
          Recoding can be accomplished via:
        </P><P class="text12">Introducing orthogonal tRNA-aaRS pairs [2]</P><P class="text12">Mutating tRNA-aaRS pairs [3]</P><P class="text12">tRNA misacylation by promiscuous RNA enzymes (Flexizymes) [4]</P></DIV><DIV class="row"><DIV class="col s12 m4"><DIV class="card pink darken-3"><DIV class="card-content white-text"><P class="text12">Other iGEM Teams are also working on codon reassignment and recoding for alternative purposes. Check out the awesome project at Bielefeld where they focus on expanding the genetic code!</P></DIV><DIV class="card-action"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Bielefeld-CeBiTec</A></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col s12"><H2 class="segmentHeader">Encrypted Sequences</H2></DIV></DIV><DIV class="row"><DIV class="col s12"><H5>Novel Genetic Codes</H5><P class="text12">
        Again, though genetic recoding is a developing technology, the field will only grow and develop as our scientific understanding and computational ability to design RNAs and proteins improves. In the same way that we currently have access to large libraries of promoters, ribosomal binding sites, and protein coding sequences, it is not hard to imagine the construction of a library of tRNAs that can be charged with non-canonical amino acids. Whether this is achieved via flexizymes, mutant pairs, or orthogonal introduction, computational selection of internally consistent sets of tRNAs and charging machinery will make it trivially easy to design a novel genetic code, and cell-free options like the N<I>ex</I>t <I>vivo</I> system would make deploying a novel genetic code relatively uncomplicated and achievable for individuals that possess basic technical ability.
      </P><P class="text12">
        Despite the benefits of genetic recoding, we should be careful and consider the unintended consequences of this technology. Undermining the faithful reproduction of a protein by a universally conserved genetic codes strikes down a cornerstone used in many fields within biology- particularly within genomics and bioinformatics.
  </P><BLOCKQUOTE class="grey lighten-2">
        The apparent risk of this technology is that genetic recoding may allow harmful sequences to be “encrypted”, thus masking the information contained within while retaining the ability to faithfully produce the encoded protein.
      </BLOCKQUOTE><P class="text12">
        When the available sample space provided by the genetic code and our understanding of the translational machinery is analyzed, it becomes apparent that recoding allows for a potential to generate numerous genetic codes. A lower-bound estimate of the total set of non-redundant genetic codes can be found ccording to the following formula:</P><P class="text12">Where n is the number of nucleic acid bases, l is the length of the codon, and a is the number of amino acids that need to be assigned a unique codon.
      </P><BLOCKQUOTE class="grey lighten-2">
          When this space is calculated with conventional parameters: n=4, l=3, and a=20, we estimate that there are (64 Choose 20)*20! possible combinations. Or put another way, <B>4.77 x 10^34</B> entirely novel genetic codes within which to encrypt a harmful DNA sequence.
        </BLOCKQUOTE></DIV></DIV><DIV class="row"><DIV class="col s12 m8"><H5>GRecoS (Genetic Recoding Space)</H5><P class="text12">
        That's 47 decillion, or 47 million billion billion billion genetic codes. Even at the lower bound, this is an extremely large sample space to search iteratively. Despite the size of the sample space, it remains to be seen whether or not this relationship is cryptographically strong. The program used to perform this calculation can be found on our GitHub page.
      </P></DIV><DIV class="col s12 m4"><DIV class="card orange darken-3"><DIV class="card-content white-text"><A href="https://github.com/chrisaac/CODONxCHANGE" class="align-center">Check out GRecoS on the <B>CODONxCHANGE</B> GitHub repository!</A></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col s12"><P class="text12">Our estimation of the full genetic code sample space, including functionally redundant genetic codes is as above. However, it should be noted that this is an estimation of the upper limit of the coding space and may overestimate the true diversity of the given coding space.
        </P></DIV></DIV><DIV class="row"><DIV class="col s12"><H5>Preliminary Testing</H5><P class="text12">
          The potential for harm as a result of this technology is not to be underestimated. If recoded systems become as prevalent and easy to obtain as we expect them to be, control over where DNA containing funtional toxin sequences are distributed greatly diminishes. Accordingly, we reached out to gene synthesis companies to determine whether or not current bioinformatic technologies can detect radically recoded toxin sequences. For most of our analysis, we selected alpha conotoxin because it is short, lethal, and previously described as a potential threat in the paper <A href="https://www.omicsonline.org/conotoxins-potential-weapons-from-the-sea-2157-2526.1000120.pdf" id="pageLink">Conotoxins: Potential Weapons from the Sea</A> in the Journal of Bioterrorism and Biodefense.
        </P></DIV></DIV><DIV class="row"><DIV class="col s12"><H5>Detecting Encrypted Sequences</H5><P class="text12">
          A total of five companies from the IGSC (n=5) of the 11 possible agreed to test our sequences. Two control sequences were sent along with the encrypted sequences: unencrypted GFP and unencrypted conotoxin. The remaining 10 sequences consisted of equal numbers of encrypted GFP and conotoxin.
        </P><TABLE class=""><THEAD><TR><TH colspan="2" style="background-color: #377b7e; color: #ffffff; font-weight: bold;">Unencrypted Sequences (n=2)</TH><TH colspan="2" style="background-color: #377b7e; color: #ffffff; font-weight: bold;">Encrypted Sequences (n=10)</TH></TR></THEAD><TBODY><TR><TD class="text12"><B>Sequence Identity</B></TD><TD class="text12">Green Fluorescent Protein (n=1)</TD><TD class="text12">Conotoxin (n=1)</TD><TD class="text12">Green Fluorescent Protein (n=5)</TD><TD class="text12">Conotoxin (n=5)</TD></TR><TR><TD class="text12"><B>Company 1</B></TD><TD class="text12">100% (±0%)</TD><TD class="text12">100% (±0%)</TD><TD class="text12">0% (±0%)</TD><TD class="text12">0% (±0%)</TD></TR><TR><TD class="text12"><B>Company 2</B></TD><TD class="text12">100% (±0%)</TD><TD class="text12">100% (±0%)</TD><TD class="text12">0% (±0%)</TD><TD class="text12">0% (±0%)</TD></TR><TR><TD class="text12"><B>Company 3</B></TD><TD class="text12">100% (±0%)</TD><TD class="text12">100% (±0%)</TD><TD class="text12">0% (±0%)</TD><TD class="text12">0% (±0%)</TD></TR><TR><TD class="text12"><B>Company 4</B></TD><TD class="text12">100% (±0%)</TD><TD class="text12">100% (±0%)</TD><TD class="text12">0% (±0%)</TD><TD class="text12">0% (±0%)</TD></TR><TR><TD class="text12"><B>Company 5</B></TD><TD class="text12">100% (±0%)</TD><TD class="text12">100% (±0%)</TD><TD class="text12">0% (±0%)</TD><TD class="text12">0% (±0%)</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="row"><DIV class="col s12"><P class="text12">
          Emails were sent to all current members of the IGSC asking them to screen twelve sequences for us. Of the five companies that were willing to help us, all of them correctly identified the un-encrypted toxic proteins. </P><BLOCKQUOTE class="grey lighten-2">
            However, no organization could correctly identify the encrypted toxins.
          </BLOCKQUOTE><P class="text12">
            While this result is unsettling, it is not unexpected. All genomic science relies heavily on the assumption that there is a known relationship between DNA and protein. Though the technology for large-scale recoding does not presently exist, it is prudent to be prepared instead of ignoring a potentially dangerous problem.</P></DIV></DIV><DIV class="row"><DIV class="col s12"><P class="text12">
          This experiment was repeated using each variation of the <A href="https://blast.ncbi.nlm.nih.gov/Blast.cgi" id="pageLink">BLAST</A> software hosted on the NCBI website. Again, the software could not identify any of the completely recoded sequences. If you are curious about the results, the sequences that we sent are available for you to try as well.</P><DIV align="center"><A class="waves-effect waves-light orange darken-3 btn" href="https://static.igem.org/mediawiki/2017/e/e5/T--Lethbridge--seqtesting.txt">Get Data</A></DIV><P class="text12">
          Since the initial testing, we have maintained correspondence with individuals at these companies and are looking forward to working closely with them to ensure that DNA synthesis remains a safe and secure practice. We would also like to thank the companies and the wonderful individuals that we had the chance to interact with for their tremendous assistance in identifying and dealing with this problem before it becomes a pressing security issue. Synthetic biologists need DNA, and DNA synthesis needs new bioinformatic screening tools.
        </P></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col s12"><H2 class="segmentHeader">Beating BLAST</H2></DIV></DIV><DIV class="row"><DIV class="col s12 m6"><H5>Basic Local Alignment Search Tool</H5><P class="text12">
        To the best of our knowledge, the only tool maintaining the safety and security of DNA synthesis is BLAST. We have shown earlier that complete genetic recoding completely nullifies the ability of BLAST to detect a sequence, but it remains to be seen how much recoding BLAST can tolerate before a sequence becomes totally unrecognizable. BLAST works by breaking a query sequence into small ‘words’ of a specified length. Words that exactly match a sequence within the database are ‘high-scoring pairs’ and contribute to a positive scoring alignment. In essence, the more exact word matches in a query sequence to a database sequence, the better the alignment score will be.
      </P></DIV></DIV><DIV class="row"><DIV class="col s12"><P class="text12">
        However, it is not intuitively obvious what degree of genetic recoding is required to evade detection by BLAST. To test this, we developed a software tool within the <B>CODONxCHANGE</B> suite, written in <A href="https://www.python.org/#python-network" id="pageLink">Python 2.7</A> to test the integrity of the BLAST platform against sequences that have been partially encrypted via a set number of recoding events. This tool can also be used to prepare genes for implementation in an orthogonal cell-free system for biocontainment purposes.
      </P></DIV></DIV><DIV class="row"><DIV class="col s12 m8"><H5>SeCReT (Sequential Codon Reassignment Tool)</H5><P class="text12">
        The tool is designed to take a nucleic acid coding sequence or protein sequence as an input, and return an ‘encrypted’ version of the sequence based upon translation via a novel genetic code. It achieves this by translating a DNA sequence into a protein sequence, and then sequentially assigning a random codon to each unique amino acid required within the protein. The resultant sequence is returned in a newly encrypted state.
      </P></DIV><DIV class="col s12 m4"><DIV class="card orange darken-3"><DIV class="card-content white-text"><A href="https://github.com/chrisaac/CODONxCHANGE" class="align-center">Check out SeCReT on the <B>CODONxCHANGE</B> GitHub repository!</A></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col s12"><H5>How Robust is BLAST?</H5><P class="text12">
       The sequence of an alpha conotoxin was randomly encrypted 29 times for each of the possible 17 recoding events. The sequences were then analyzed via BLAST using a locally curated database containing only the original sequence. The percent identity was averaged with non-hits counting as a 0 value. The percent identity of each sequence and the number of matches returned per group was then plotted as a function of the number of recoding events. The orange trace represents the percent identity of the query to the subject sequence, and the blue trace represents the proprotion of recoded events that BLAST found a match for.
     </P></DIV></DIV><DIV class="row"><DIV class="col s12"><P class="text12">
      The results of this analysis suggest that the most effective number of switches to BEAT Blast is 4. This level of recording is generally sufficient to bring the similarity of a protein to below 80%, which is the consesus value reported in the HHS guidelines and adhered to by the majority of gene synthesis companies.</P><BLOCKQUOTE class="grey lighten-2">After 9 recoding events, a number of sequences become able to evade BLAST entirely.</BLOCKQUOTE><P class="text12">
        This suggests that rational recoding to disrupt the generation of high scoring words in BLAST may improve the cryptographic ability of genetic recoding even further. Furthermore, having more than 12 recoding events significantly improves the chances that the encrypted sequence is not detected by BLAST at all. Again, the raw data generated from the experiments are available for public analysis and more results can be found on the GitHub page.
    </P><DIV align="center"><A class="waves-effect waves-light btn orange darken-3" href="https://github.com/chrisaac/CODONxCHANGE">Get Data</A></DIV></DIV></DIV><DIV class="row"><DIV class="col s12 m6"><H5>Rational Evasion of BLAST</H5><P class="text12">
          Curiously, the number of recoding events does not necessarily always indicate whether a read is detected by BLAST or not. A small section of data from one SeCReT run is presented to indicate that in some cases, recoding can be especially disruptive to determining the identity of a protein product.
        </P><DIV align="center"><A class="waves-effect waves-light btn orange darken-3" href="https://github.com/chrisaac/CODONxCHANGE">Get Data</A></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col s12"><H2 class="segmentHeader">Building Solutions</H2></DIV></DIV><DIV class="row"><DIV class="col s12 m6"><H5>Changes on the Horizon</H5><P class="text12">
        We have shown that the power of the BLAST program to identify proteins arising from genetically recoded sequences is extremely limited, there are initiatives to develop new biosecurity tools. For example, Intelligence Advanced Research Projects Activity (IARPA), the cousin of DARPA, has a program called Functional Genomic and Computational Assessment of Threats (Fun GCAT) which aims to catalyze the development of tools to improve DNA screening capabilites. Several of the synthesis companies that we spoke to are involved in this program. It is our hope that we see more mobilization on this front to ensure that the benefits that arise from Synthetic Biology continue to greatly outweigh the risks.
      </P></DIV><DIV class="col s6 m3"><DIV class="card blue darken-3"><DIV class="card-content white-text"><A href="https://www.iarpa.gov" class="align-center">Check out <B>IARPA</B>!</A></DIV></DIV></DIV><DIV class="col s6 m3"><DIV class="card yellow darken-2"><DIV class="card-content white-text"><A href="https://www.iarpa.gov/index.php/research-programs/fun-gcat" class="align-center">Check out <B>FunGCAT</B>!</A></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col s12 m6"><H5>DeToxIT (Decryption and Toxin Identification Tool)</H5><P class="text12">
        Here we throw our own hat into the ring with a simple tool to decrypt DNA sequences that have been radically recoded and compare them to a database of known select agents.
      </P><P class="text12">
        Test it out with a trial data set found on our GitHub page!
      </P><DIV align="center"><A class="waves-effect waves-light btn orange darken-3" href="https://github.com/chrisaac/CODONxCHANGE">Get Data</A></DIV></DIV><DIV class="col s12 m6"><DIV class="card orange darken-3"><DIV class="card-content white-text"><A href="https://github.com/chrisaac/CODONxCHANGE" class="align-center">Check out DeToxIT in the <B>CODONxCHANGE</B> GitHub repository!</A></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col s12"><H2 class="segmentHeader">Summary</H2></DIV></DIV><DIV class="row"><DIV class="col s12 m6"><H5>Biosecurity Analysis</H5><P class="text12">We determined that current biosecurity protocols and tools (BLAST) are ineffective at identiying proteins that could arise from recoded sequences.</P><P class="text12">We also determined that the minimum degree of recoding required to evade BLAST is approximately 9 recoding events.</P><P class="text12">We also determined that 4 recoding events are sufficient to reduce the percent identity of a toxin to 80% of its closest reference sequence.</P><P class="text12">Lastly, we have built relationships with members of the IGSC and are excited to continue working to keep DNA synthesis safe.</P></DIV><DIV class="col s12 m6"><H5>CODONxCHANGE Software Suite</H5><P class="text12"><B>GRecoS</B> (Genetic Recoding Space)</P><P class="text12"><B>SeCReT</B> (Sequential Codon Reassignment Tool)</P><P class="text12"><B>DeToxIT</B> (Decryption and Toxin Identification Tool)</P></DIV></DIV><DIV class="row"><DIV class="col s12"><H4>Looking for the Source Code?</H4><P class="text12">While most of our software can be found freely availble on our GitHub repository, you will notice that some of the tools do not have their source distributed. For the interim, we have been advised not to publish the source code for the fully functional encryption software. </P><BLOCKQUOTE class="grey lighten-2">Until we know the ethical and legal standing surrounding the distribution of the software, it will be available via direct contact only.</BLOCKQUOTE><P class="text12">Instead, a feature-reduced version of the software is available as a pre-compiled binary to get a taste of how it works, and a tangible idea for just how effective recoding is at hiding the identity of a sequence. If you would like to get access to the source code, you can get in touch with us by following the link and verifying that you are affiliated with an academic institution or other trusted party. We love how safe and accessible Synthetic Biology is, and we are excited to continue developing tools to keep it that way. Thank you for your understanding.</P></DIV><DIV align="center"><A class="waves-effect waves-light btn orange darken-3" href="https://goo.gl/forms/WFMgcBHVFg8cI13o2">Get Access</A></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col s12"><H2>References</H2></DIV></DIV><DIV class="row"><DIV class="col s12"><P class="text12" id="litRef">[1] Young, T. S. and P. G. Schultz, Beyond the Canonical 20 Amino Acids: Expanding the Genetic Lexicon. Journal of Biological Chemistry, 2010. 285: 11039-11044.</P><P class="text12" id="litRef">[2] Javahishvili, T., A. Manibusan, S. Srinagesh, D. Lee, S. Ensari, M. Shimazu, and P. G. Schultz, Role of tRNA Orthogonality in an Expanded Genetic Code. ACS Chemical Biology, 2014. 9(4): 874-879. </P><P class="text12" id="litRef">[3]Chatterjee, A., H. Xiao, and P. G. Schultz, Evolution of multiple, mutually orthogonal prolyl-tRNA synthetase/tRNA pairs for unnatural amino acid mutagenesis in Escherichia coli. Proceedings of the National Academy of Sciences of the United States of America, 2012. 109(37): 14841-14846.</P><P class="text12" id="litRef">[4] Ohuchi, M., H. Murakami, and H. Suga, The flexizyme system: a highly flexible tRNA aminoacylation tool for the translation apparatus. Current Opinion in Chemical Biology, 2007. 11(5): 537-542.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>