<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Lethbridge_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Lethbridge/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Project</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Description">Description</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Design">Design</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Demonstrate">Demonstrate</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Results">Results</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Collaborations">Collaborations</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="https://2017.igem.org/Team:Lethbridge/Parts">Parts</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Parts">Overview</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Part_Collection">Part Collection</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Composite_Part">Composite</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Improve">Improved Part</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Basic_Part">Basic</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Human Practices</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Human_Practices">Overview</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/HP/Silver">User Groups - Silver</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/HP/Gold_Integrated">Integrated HP - Gold</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Software">Biosecurity Software</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Engagement">Engagement</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Lab</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Experiments">Experiments</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Notebook">Notebook</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Safety">Safety</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/InterLab">InterLab</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="https://2017.igem.org/Team:Lethbridge/Team">Team</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Team">Team Members</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Attributions">Attributions</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Attributions#Sponsors">Sponsors</A></LI></DIV></DIV><DIV class="circleTop" style="margin-left:90%"><A href="#top">top</A></DIV><DIV id="allContent"><DIV style="margin-left:10%; margin-right:10%"><P class="text12" style="font-size: calc(22px + 0.5vw) !important; text-align: center !important"><B> “The International Genetically Engineered Machine (iGEM)  Foundation is an independent, non-profit organization dedicated to <SPAN style="color: #f5af3e">EDUCATION</SPAN> and <SPAN style="color: #703a96">COMPETITION</SPAN>, the <SPAN style="color: #377b7e">ADVANCEMENT</SPAN> of synthetic biology, and the development of an <SPAN style="color: #1fb151">OPEN COMMUNITY</SPAN> and collaboration.” </B></P></DIV><DIV style="margin-left:5%; margin-right:5%"><P class="text12" style="font-size: calc(17px + 0.5vw) !important; text-align: center !important"> For our tenth year as an iGEM team, we wanted to give back to the community and looked to the iGEM mission statement for inspiration. To align our project with the foundation,  we developed a tool to advance synthetic biology and increase its accessibility to novices, hobbyists and experts.</P></DIV><DIV class="segmentDiv"><DIV class="centerContainer"><H2 class="segmentHeader">Cell-free Systems</H2><P class="text12j">  Cell-free systems allow for a reliable and consistent expression of recombinant proteins outside of a living cell, bypassing issues with genetic regulation and cellular noise [1].</P><P class="text12">  Such systems are advantageous over cell-based synthetic biology due to the:</P><UL class="text12" style="list-style: disc"><LI class="text12">Capability of <B>tolerating toxins</B> normally detrimental to the cell</LI><LI class="text12">Ability to direct all energy resources to the application, increasing the <B>freedom of design</B></LI><LI class="text12">Inherent feature of <B>reduced biocontamination</B>, as components do not replicate mutate or evolve</LI><LI class="text12">Easy <B>control of transcription and translation</B> in an open environment</LI><LI class="text12">Easy incorporation of <B>unnatural amino acids</B></LI><LI class="text12">Capacity to <B>modulate the environment</B> for optimal protein expression</LI><LI class="text12">Rapid design-build-test cycle</LI><LI class="text12">Proficiency of using both linear and circular template DNA</LI></UL><DIV class="segmentDiv"><DIV class="centerContainer"><H2 class="segmentHeader">Applications</H2><P class="text12j">  Emerging as a new platform for synthetic biology, cell-free systems have shown potential for use in a variety of applications exemplifying the utility of such systems [2,3].</P></DIV></DIV><DIV class="segmentDiv"><DIV class="centerContainer"><H2 class="segmentHeader">Types of Cell-free Systems</H2><P class="text12j">Cell-free systems include all of the necessary biomachinery for protein production, utilizing cell extracts or a purified reconstituted system.</P></DIV></DIV><H2>Cell Lysates</H2><DIV class="contentDiv"><P class="text12j"> Extracts are isolated from <I>Escherichia coli (E. coli)</I>, rabbit reticulocytes, wheat germ, insect cells or human cells [16-21].<B><I>Advantages</I></B> - cell lysates are easy to obtain.<B><I>Disadvantages</I></B> - lysates are associated with a degree of uncertainty; the composition is unknown and unwanted factors such as DNases and RNases may be present.</P></DIV><H2>Purified Reconstituted</H2><DIV class="contentDiv"><P class="text12j"> Highly purified and reconstituted coupled transcription and translation (TX-TL) systems are commercially available. The PURExpress system from New England Biolabs (NEB) is an example of a protein synthesis system made up of recombinant elements. The PURExpress system derived from <I>E. coli</I> consists of only 10 translation factors, 20 aminoacyl-tRNA synthetases, 7 enzymes (including ribosomes), 4 energy sources, 20 amino acids and a tRNA mix [22].<B><I>Advantages</I></B> - reduced level of contaminating activities, free of nonspecific nuclease activity, recombinant protein factors are histidine tagged, and various kits are available (standard PURExpress, ∆ ribosome, ∆ tRNA, and ∆ release factor kits).<B><I>Disadvantages</I></B> - not open source, not available for everyday consumers, and customization is limited to the kits available.</P></DIV><H2 class="segmentHeader">Our System</H2><DIV class="segmentDiv"><DIV class="centerContainer"><P class="text12j"> It is our goal to make a completely customizable and accessible cell-free system, that is inherently safe and user-friendly. To accomplish this, we developed N<I>ex</I>t <I>vivo</I>, a standardized and modular system that contains all of the necessary biomachinery for protein production.</P><P class="text12j">We designed, in BioBrick standard, an open-source <A href="https://2017.igem.org/Team:Lethbridge/Part_Collection" id="pageLink"> collection of parts </A>for cell-free protein synthesis. N<I>ex</I>t <I>vivo</I> allows individual user groups to select which components they want in their system and leave out any additional factors of their choosing. As such, future modules can be added to the system with ease, creating a wide variety of customizable TX-TL kits.</P><P class="text12j"> Specifically, we aim to over-express all TX-TL components simultaneously and pool the resulting cell lysates for co-purification, providing a simple method for producing the system. Using our approach, the tRNA and ribosomes will bind to an MS2 coat protein/Ni<SUP>2+</SUP> complex, and the TX-TL proteins will bind directly to the Ni<SUP>2+</SUP> affinity chromatography resin. In this way, components can be purified in batch and a tailored cell-free system can be created on demand.</P></DIV></DIV><H2> Biocontainment </H2><DIV class="contentDiv"><P class="text12j"> A modular and accessible cell-free platform enables the development of recoding technologies leading to the creation of a modified codon table. Recoded systems are unable to be horizontally transferred to a living system, providing an intrinsic form of biocontainment and preventing accidental environmental release. We have created a <A href="https://2017.igem.org/Team:Lethbridge/Software" id="pageLink">codon re-assignment tool</A> that will produce recoded sequences to support such developments.</P></DIV><H2> Biosecurity </H2><DIV class="contentDiv"><P class="text12j"> With the development of recoded systems comes additional consideration of the impact on biosecurity and the potential for synthesizing compounds undetected by current screening methods. We propose a solution to this problem by providing <A href="https://2017.igem.org/Team:Lethbridge/Software" id="pageLink">software tools</A> to combat the potential misuse of our system and highlight the significant implications of cell-free synthetic biology.</P></DIV><H2> Education </H2><DIV class="contentDiv"><P class="text12j">N<I>ex</I>t <I>vivo</I> provides a useful learning tool. Taking advantage of the non-proliferating nature, we have developed <A href="https://2017.igem.org/Team:Lethbridge/HP/Gold_Integrated" id="pageLink">simplified protocols tailored to the Alberta curriculum</A> to teach new users the basic concepts of synthetic biology.</P></DIV><DIV class="segmentDiv"><DIV class="centerContainer"><H2 class="segmentHeader">Recent Advancements</H2><P class="text12j">Recent work by Shephard <I>et al.</I> [23] published in August of 2017 developed a 30-cistron translation factor module to provide an affordable and scalable method for obtaining a reconstituted TX-TL system. Focusing only on 30 of the 31 translation factors needed for protein synthesis, other necessary factors such as tRNA and ribosomes were not included in their design. Our work improves on this by <A href="https://2017.igem.org/Team:Lethbridge/Design" id="pageLink">providing a strategy</A> for purifying and obtaining tRNA and ribosomes, in addition to all of the factors required for transcription and translation.</P></DIV></DIV><H2> References</H2><UL id="litRef" style="list-style: none"><LI>[1] Hodgman, C.E. and M.C. Jewett, Cell-Free Synthetic Biology: Thinking Outside the Cell. Metabolic Engineering, 2012. 14(3): p. 261-269.</LI><LI>[2] Chong, S., Overview of cell-free protein synthesis: historic landmarks, commercial systems, and expanding applications. Current Protocols in Molecular Biology, 2014. 108: p. 16.30.1-11.</LI><LI>[3] Smith, M.T., et al., The emerging age of cell-free synthetic biology. FEBS Letters, 2014. 588(17): p. 2755-2761.</LI><LI>[4] Caschera, F. and V. Noireaux, Compartmentalization of an all-E. coli Cell-Free Expression System for the Construction of a Minimal Cell. Artificial Life, 2016. 22(2): p. 185-195.</LI><LI>[5] Jia, H., et al., Cell-free protein synthesis in micro compartments: building a minimal cell from biobricks. New Biotechnology, 2017. 39(Pt B): p. 199-205.</LI><LI>[6] France, S.P., et al., Constructing Biocatalytic Cascades: In Vitro and in Vivo Approaches to de Novo Multi-Enzyme Pathways. ACS Catalysis, 2017. 7(1): p. 710-724.</LI><LI>[7] Jewett, M.C., et al., An integrated cell-free metabolic platform for protein production and synthetic biology. Molecular Systems Biology, 2008. 4: p. 220.</LI><LI>[8] Pardee, K., et al., Paper-Based Synthetic Gene Networks. Cell. 159(4): p. 940-954.</LI><LI>[9] Karig, D.K., Cell-free synthetic biology for environmental sensing and remediation. Current Opinion in Biotechnology, 2017. 45: p. 69-75.</LI><LI>[10] Wang, L., Genetically encoding new bioreactivity. New Biotechnology, 2017. 38(Part A): p. 16-25.</LI><LI>[11] Kawakami, T., et al., Directed Evolution of a Cyclized Peptoid-Peptide Chimera against a Cell-Free Expressed Protein and Proteomic Profiling of the Interacting Proteins to Create a Protein-Protein Interaction Inhibitor. ACS Chemical Biology, 2016. 11(6): p. 1569-77.</LI><LI>[12] Sun, Z.Z., et al., Linear DNA for rapid prototyping of synthetic biological circuits in an Escherichia coli based TX-TL cell-free system. ACS Synthetic Biology, 2014. 3(6): p. 387-97.</LI><LI>[13] Nagumo, Y., et al., PURE mRNA display for in vitro selection of single-chain antibodies. Journal of Biochemistry, 2016. 159(5): p. 519-26.</LI><LI>[14] Elani, Y., R.V. Law, and O. Ces, Vesicle-based artificial cells: recent developments and prospects for drug delivery. Therapeutic Delivery, 2015. 6(5): p. 541-3.</LI><LI>[15] Menezes, A.A., et al., Grand challenges in space synthetic biology. Journal of the Royal Society Interface, 2015. 12(113): p. 20150803.</LI><LI>[16] Carlson, E.D., et al., Cell-Free Protein Synthesis: Applications Come of Age. Biotechnology advances, 2012. 30(5): p. 1185-1194.</LI><LI>[17] Shin, J. and V. Noireaux, An E. coli Cell-Free Expression Toolbox: Application to Synthetic Gene Circuits and Artificial Cells. ACS Synthetic Biology, 2012. 1(1): p. 29-41.</LI><LI>[18] Garamella, J., et al., The All E. coli TX-TL Toolbox 2.0: A Platform for Cell-Free Synthetic Biology. ACS Synthetic Biology, 2016. 5(4): p. 344-355.</LI><LI>[19] Tarui, H., et al., Establishment and characterization of cell-free translation/glycosylation in insect cell (Spodoptera frugiperda 21) extract prepared with high pressure treatment. Applied Microbiology and Biotechnology, 2001. 55(4): p. 446-53.</LI><LI>[20] Katzen, F., G. Chang, and W. Kudlicki, The past, present and future of cell-free protein synthesis. Trends in Biotechnology, 2005. 23(3): p. 150-6.</LI><LI>[21] Mikami, S., et al., A human cell-derived in vitro coupled transcription/translation system optimized for production of recombinant proteins. Protein Expression and Purification, 2008. 62(2): p. 190-8.</LI><LI>[22] Shimizu, Y., et al., Cell-free translation reconstituted with purified components. Nature biotechnology, 2001. 19(8): p. 751-755.</LI><LI>[23] Shepherd, T.R., et al., De novo design and synthesis of a 30-cistron translation-factor module. Nucleic Acids Research, 2017. 45(18): p. 10895-10905.</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>