<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Lethbridge_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Lethbridge/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Project</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Description">Description</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Design">Design</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Demonstrate">Demonstrate</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Results">Results</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Collaborations">Collaborations</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="https://2017.igem.org/Team:Lethbridge/Parts">Parts</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Parts">Overview</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Part_Collection">Part Collection</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Composite_Part">Composite</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Improve">Improved Part</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Basic_Part">Basic</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Human Practices</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Human_Practices">Overview</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/HP/Silver">User Groups - Silver</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/HP/Gold_Integrated">Integrated HP - Gold</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Software">Biosecurity Software</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Engagement">Engagement</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Lab</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Experiments">Experiments</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Notebook">Notebook</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Safety">Safety</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/InterLab">InterLab</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="https://2017.igem.org/Team:Lethbridge/Team">Team</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Team">Team Members</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Attributions">Attributions</A></LI><LI><A class="color-wh" href="https://2017.igem.org/Team:Lethbridge/Attributions#Sponsors">Sponsors</A></LI></DIV></DIV><DIV class="circleTop" style="margin-left:90%"><A href="#top">top</A></DIV><DIV id="allContent"><DIV class="segmentDiv"><DIV class="centerContainer"><H2 class="segmentHeader">Overview</H2></DIV></DIV><CENTER><DIV style="margin-left:5%; margin-right:5%"><P class="text12j" style="font-size: calc(12px + 0.5vw) !important; text-align: center !important"><B>Provided</B><A href="https://2017.igem.org/Team:Lethbridge/HP/Gold_Integrated" id="pageLink">simplified protocols and learning tools</A> for the education system in collaboration with the Lethbridge High School Team <B>Addressed</B><A href="https://2017.igem.org/Team:Lethbridge/HP/Gold_Integrated" id="pageLink">policy issues</A> regarding cell-free systems and the biosecurity implications of our project <B>Developed</B> and tested <A href="https://2017.igem.org/Team:Lethbridge/Software" id="pageLink">software tools</A> for biocontainment and to mitigate dual-use <A href="https://2017.igem.org/Team:Lethbridge/Improve" id="pageLink"><B>Improved</B> 5 parts</A> by optimizing coding sequences for optimal expression in <I>E. coli</I>, adding hexahistidine tags for simple purification and improving part characterization <A href="https://2017.igem.org/Team:Lethbridge/Demonstrate" id="pageLink"><B>Demonstrated</B></A> proof of concept for multi-protein purification to simplify the system</P></DIV><DIV style="margin-left:5%; margin-right:5%"><P class="text12j" style="font-size: calc(12px + 0.5vw) !important; text-align: center !important"><B>17 parts</B> characterized and documented<A href="https://2017.igem.org/Team:Lethbridge/HP/Silver" id="pageLink"><B>Worked</B></A> closely with user groups to inform our design <A href="https://2017.igem.org/Team:Lethbridge/Collaborations" id="pageLink"><B>Assisted</B></A> Lethbridge High School with wet lab work and received help with education interviews <A href="https://2017.igem.org/Team:Lethbridge/Collaborations" id="pageLink"><B>Collaborated</B></A> with Florida State University by providing mutual project feedback </P></DIV><DIV style="margin-left:5%; margin-right:5%"><P class="text12j" style="font-size: calc(12px + 0.5vw) !important; text-align: center !important"><B>Team</B> is registered! <B>Project</B> is showcased on wiki <B>Project</B><A href="https://2017.igem.org/Team:Lethbridge/Attributions" id="pageLink">attributions</A> clearly detailed <B>Safety</B> forms submitted <B>Judging</B> form completed <B>Parts</B> documented on the registry <A href="https://2017.igem.org/Team:Lethbridge/Parts" id="pageLink"><B>9 parts</B></A> submitted <B>Participated</B> in the <A href="https://2017.igem.org/Team:Lethbridge/InterLab" id="pageLink">InterLab study</A></P></DIV><DIV class="segmentDiv"><DIV class="centerContainer"><P class="text12j"><A href="https://2017.igem.org/Team:Lethbridge/Part_Collection" id="pageLink"><B>Part Collection:</B></A> We have provided a <I>collection of open-source parts</I> for cell-free protein expression
<A href="https://2017.igem.org/Team:Lethbridge/HP/Gold_Integrated" id="pageLink"><B>Integrated Human Practices:</B></A>The design of N<I>ex</I>t <I>vivo</I> was <I>informed by the needs of three user groups</I>, resulting in the development of an educational tool and an assessment of the biosecurity implications of cell-free systems

<A href="https://2017.igem.org/Team:Lethbridge/Engagement" id="pageLink"><B>Public Engagement and Education:</B></A> In celebration of our 10th anniversary as an iGEM team, we attended events in our community to share our work and developed <I>curriculum aligned lesson plans</I>, as well as a <I>safe learning tool</I> for the education system
<A href="https://2017.igem.org/Team:Lethbridge/Software" id="pageLink"><B>Software:</B></A> We addressed the dual-use potential of our system and developed <I>software tools</I> as a solution to mitigate biosecurity threats
</P></DIV></DIV><DIV class="segmentDiv"><DIV class="centerContainer"><H2 class="segmentHeader">Proteins</H2><H2 align="center">Overexpressions</H2><P class="text12j"> We conducted overexpressions of our constructs in BL21-Gold (DE3), an <I>E. coli</I> strain capable of overexpressing T7 RNA polymerase by induction with IPTG (see <A href="https://2017.igem.org/Team:Lethbridge/Design#anchor4" id="pageLink">design page</A> for details). All overexpression characterization is documented on each individual parts page. 


In total, we successfully overexpressed:

</P><DIV class="partTable"><TABLE id="partTable"><TBODY><TR><TH id="col1" style="background-color: #377b7e; color: #ffffff;">BioBrick</TH><TH id="col2" style="background-color: #377b7e; color: #ffffff;">Protein</TH><TH id="col1" style="background-color: #377b7e; color: #ffffff;">BioBrick</TH><TH id="col2" style="background-color: #377b7e; color: #ffffff;">Protein</TH></TR><TR><TD class="xl99"><A href="http://parts.igem.org/Part:BBa_K2443001" target="_blank">BBa_K2443001</A></TD><TD class="xl94">ArgRS</TD><TD><A href="http://parts.igem.org/Part:BBa_K2443022">BBa_K2443022</A></TD><TD>MTF</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K2443003" target="_blank">BBa_K2443003</A></TD><TD>AspRS</TD><TD class="xl99"><A href="http://parts.igem.org/Part:BBa_K2443027" target="_blank">BBa_K2443027</A></TD><TD class="xl94">EF-Tu</TD></TR><TR><TD class="xl99"><A href="http://parts.igem.org/Part:BBa_K2443007" target="_blank">BBa_K2443007</A></TD><TD class="xl94">GlyRSα</TD><TD><A href="http://parts.igem.org/Part:BBa_K2443028">BBa_K2443028</A></TD><TD>EF-Ts</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K2443008" target="_blank">BBa_K2443008</A></TD><TD>GlyRSβ</TD><TD class="xl99"><A href="http://parts.igem.org/Part:BBa_K2443031" target="_blank">BBa_K2443031</A></TD><TD class="xl94">RF3</TD></TR><TR><TD class="xl99"><A href="http://parts.igem.org/Part:BBa_K2443009" target="_blank">BBa_K2443009</A></TD><TD class="xl94">HisRS</TD><TD><A href="http://parts.igem.org/Part:BBa_K2443032">BBa_K2443032</A></TD><TD>RRF</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K2443013" target="_blank">BBa_K2443013</A></TD><TD>MetRS</TD><TD class="xl99"><A href="http://parts.igem.org/Part:BBa_K2443033" target="_blank">BBa_K2443033</A></TD><TD class="xl94">MK</TD></TR><TR><TD class="xl99"><A href="http://parts.igem.org/Part:BBa_K2443014" target="_blank">BBa_K2443014</A></TD><TD class="xl94">PheRSα</TD><TD><A href="http://parts.igem.org/Part:BBa_K2443034">BBa_K2443034</A></TD><TD>CK</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K2443017" target="_blank">BBa_K2443017</A></TD><TD>SerRS</TD><TD class="xl99"><A href="http://parts.igem.org/Part:BBa_K2443037" target="_blank">BBa_K2443037</A></TD><TD class="xl94">T7 RNA Polymerase</TD></TR><TR><TD class="xl99"><A href="http://parts.igem.org/Part:BBa_K2443019" target="_blank">BBa_K2443019</A></TD><TD class="xl94">TrpRS</TD></TR></TBODY></TABLE><P class="text12j"> 
As an example, Figure 1 showcases the overexpression of four individual TX-TL proteins. 
</P><P class="figure" text-align:="" left=""><B>Figure 1 - Overexpressions of key proteins required for TX-TL systems. </B> Overexpressions of proteins  in <I>E. coli</I> BL21-Gold (DE3) visualized on a 12% SDS-PAGE for 80 min at 200 V. Lanes are as follows: 1- Protein ladder; 2- HisRS; 3- HisRS induced with IPTG; 4- TrpRS; 5- TrpRS induced with IPTG; 6- RF3; 7- RF3 induced with IPTG; 8- RRF; 9- RRF induced with IPTG. Protein sizes: HisRS 48.3 kDa; TrpRS 38.8 kDa; RF3 60.9 kDa; RRF 21.9 kDa.
 </P><H2 align="center">Multi-Protein Purification</H2><P class="text12j">One of the key features of N<I>ex</I>t <I>vivo</I> is the ability to purify all of the components in a <A href="https://2017.igem.org/Team:Lethbridge/Design#anchor4" id="pageLink">single step purification</A>. As a proof of concept we expressed four of the TX-TL components and co-purified them all using a Nickel Sepharose chromatography column (Figure 2). The four proteins used in this initial test were selected based on their molecular weights relative to each other for visualization purposes. </P><P class="figure"><B>Figure 2 - Representative overexpression and multi-protein purification of TX-TL components. </B> Each TX-TL component was expressed from <I>E. coli</I> cells carrying the plasmid encoding the specified component and samples three hours post induction were collected. The expressing cells of each component were pooled and lysed before applying the lysate to a Nickel Sepharose affinity column for isolation of just the hexahistidine tagged TX-TL components. After washing away the unwanted cellular proteins and debris, the TX-TL components were eluted from the Nickel Sepharose to a high level of purity.  Lanes are as follows: 1- Protein ladder; 2- HisRS overexpression; 3- TrpRS overexpression; 4- RF3 overexpression; 5- RRF overexpression; 6- HisRS, TrpRS, RF3 and RRF elution: HisRS 48.3 kDa; TrpRS 38.8 kDa; RF3 60.9 kDa; RRF 21.9 kDa </P></DIV></DIV><DIV class="segmentDiv"><DIV class="centerContainer"><H2 class="segmentHeader">tRNA Purification</H2><P class="text12j">The biggest issue we initially faced in developing Next <I>vivo</I> was determining how we could purify tRNA quickly and efficiently. The solution we decided upon was an adapted MS2 RNA co-purification combined with a subsequent incubation with RNase H and a DNA oligo that would selectively cleave and release a tRNA of the proper size. For more information on the design,  see the <A href="https://2017.igem.org/Team:Lethbridge/Design#anchor5" id="pageLink"> tRNA purification</A> section here.</P><P class="text12j">Both the tRNA<SUP>phe</SUP>-MS2 aptamer construct and MS2BP were overexpressed individually in <I>E. coli</I> BL21-Gold (DE3) cells. Upon which time the cells were lysed, the lysate combined and applied to a Nickel Sepharose affinity column. In order to cleave the RNA, 1 µM of DNA oligo was added to the column, as well as varying amounts of RNase H. Incubation times on the column with RNase H and DNA oligo varied from 2 hours (Figure 3) to 12 hours (Figure 4), and the amount of RNase H used varied from 10 units to 100 units (Figure 3 and 4). Based upon the varied conditions, a longer incubation time had the greatest effect on tRNA cleavage efficiency with units of RNase H being optimal between the range of 5-50. With these improvements from our initial attempt at tRNA cleavage, we successfully purified tRNA<SUP>Phe</SUP>, as shown in Figure 4.</P><P class="figure"><B>Figure 3 - Preliminary tRNA<SUP>Phe</SUP> Purification. </B>12% 8M urea PAGE run for 45 mins at 300 V. All concentrated fractions were phenol chloroform extracted, ethanol precipitated and re-suspended in 30 µL of ddH<SUB>2</SUB>O. Lanes are as follows: 1- tRNA fraction with 20 units of RNase H added; 2- concentrated tRNA fraction 20 units of RNase H added; 3- concentrated MS2 fraction 1 20 units of RNase H added; 4- concentrated MS2 fraction 2  20 units of RNase H added; 5- tRNA standard (76 nt).
</P><P class="figure"><B>Figure 4 - Successful tRNA<SUP>Phe</SUP> Purification. </B>12% 8M urea PAGE run for 45 mins at 300 V. All fractions were phenol chloroform extracted, ethanol precipitated and re-suspended in 30 µL of ddH<SUB>2</SUB>O. Lanes are as follows - 1- MS2 fraction 25 units of RNase H added; 2- tRNA fraction 25 units of RNase H added; 3- MS2 fraction 50 units of RNase H added; 4- tRNA fraction 50 units of RNase H added; 5- MS2 fraction 100 units of RNase H added; 6- tRNA fraction 100 units of RNase H added; 7- MS2 fraction 10 units of RNase H added; 8- tRNA elution 10 units of RNase H added; 9- tRNA standard (76 nt). 
</P></DIV></DIV><DIV class="segmentDiv"><DIV class="centerContainer"><H2 class="segmentHeader">Ribosomes</H2><P class="text12j">Initial work to produce MS2 tagged ribosomes was carried out by attempting to isolate the 23S and 16S rRNA genes from the <I>E. coli rrnB</I> operon (see <A href="https://2017.igem.org/Team:Lethbridge/Design#anchor6" id="pageLink">design page</A> for details). The 23S rRNA is the major component of the  large ribosomal subunit and the 16S rRNA in the small ribosomal subunit. Amplifying (Figure 6) and inserting the rRNA genes in the BioBrick standard was the first step in inserting the MS2 hairpin into the ribosomal RNA. Ligations into pSB1C3 have thus far been unsuccessful.</P><P class="figure" text-align:="" left=""><B>Figure 5 - Amplification of the 23S and 16S ribosomal RNA genes from <I>E. coli rrnB</I> operon. </B> PCR amplification of the 23S and 16S rRNA genes run on a 1% Agarose gel for 30 min at 100 V. Lanes are as follows: 1- 1 kb ladder; 2- 16S reaction before amplification; 3- 23S reaction before amplification; 4- 16S reaction; 5- 16S reaction; 6- Empty; 7- 23S reaction; 8- 23S reaction. The 16S rRNA is ~2000 bp and the 23S rRNA is ~2900 bp.
 </P></DIV></DIV><DIV class="segmentDiv"><DIV class="centerContainer"><H2 class="segmentHeader">Validation Construct</H2><P class="text12j">  The validation construct was <A href="https://2017.igem.org/Team:Lethbridge/Design#anchor7" id="pageLink">designed</A> to detect successful transcription or translation, or both simultaneously. In addition to providing the initial characterization of our system, this construct can be used as a measurement tool to test and standardize the performance of N<I>ex</I>t <I>vivo</I>. To confirm our ability to detect successful transcription, the Spinach aptamer was  <I>in vitro</I> transcribed using T7 RNA polymerase (previously purified) and purified by phenol chloroform extraction (Figure 6A). Following addition of the fluorophore, DFHBI, fluorescence was measured using a fluorimeter. The fluorimeter data shows that fluorescence was observed following addition of DFHBI (Figure 6B), indicating that the Spinach RNA can be used as a measure of transcriptional activity. Next, to confirm our ability to detect successful translation, EYFP was expressed using the PURExpress system (Figure 7). Additional fluorescence analysis is still required to confirm EYFP can be used as a measure of translational activity. 
 </P><P class="figure"><B>Figure 6 - Characterization of the EYFP-Spinach Construct - Transcription.</B><B>A</B><I>In vitro</I> transcribed Spinach RNA visualized on 1% agarose for 30 min at 100 V. Lane 1- 1kb ladder and Lane 2- Spinach mRNA. <B>B</B> Fluorimeter data illustrating green fluorescence following addition of DFHBI, with an excitation of 447 nm and emission of 497 nm.
</P><P class="figure"><B>Figure 7 - Characterization of the EYFP-Spinach Construct - Translation.</B><I>In vitro</I> expressed EYFP visualized on a 12% SDS-PAGE for 80 min at 200 V. Lanes are as follows: 1- EYFP (~30 kDa); 2- Protein Ladder
</P></DIV></DIV><DIV class="segmentDiv"><DIV class="centerContainer"><H2 class="segmentHeader">Statistics</H2></DIV></DIV></DIV></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>