<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cambridge-JIC_Chromoproteins"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Cambridge-JIC/Chromoproteins">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cambridge-JIC/Chromoproteins&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge-JIC/Chromoproteins&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge-JIC/Chromoproteins&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cambridge-JIC/Chromoproteins" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cambridge-JIC/Chromoproteins</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="wrap"><DIV id="left"><UL id="accmenu"><LI><A class="head" href="#"> Welcome</A></LI><LI><A href="/Team:Cambridge-JIC">Home</A></LI><LI><A href="/Team:Cambridge-JIC/Project">Project Overview</A></LI><LI><A href="/Team:Cambridge-JIC/Team">Meet the Team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=Cambridge-JIC">Official Team Profile</A></LI></UL><LI><A class="head" href="#">Introducing Marchantia</A></LI><LI><A href="/Team:Cambridge-JIC/Marchantia">Why Marchantia?</A></LI><LI><A href="/Team:Cambridge-JIC/Background_on_Byrophytes">Background on Bryophytes</A></LI><LI><A class="head" href="#">Labwork</A></LI><LI><A href="/Team:Cambridge-JIC/Notebook">Notebook</A></LI><LI><A href="/Team:Cambridge-JIC/Chromoproteins">Chromoproteins</A></LI><LI><A href="/Team:Cambridge-JIC/RaspberryKetones">Raspberry Ketones</A></LI><LI><A href="/Team:Cambridge-JIC/Hammerhead_Ribozyme">Hammerhead Ribozyme</A></LI><LI><A href="/Team:Cambridge-JIC/Enhancer_Trap">Enhancer Trap</A></LI><LI><A href="/Team:Cambridge-JIC/Protocol">Protocols</A></LI><LI><A href="/Team:Cambridge-JIC/Dry_Work">Dry Work</A></LI><LI><A href="/Team:Cambridge-JIC/Informatics" class="head">Bioinformatics</A></LI><LI><A href="/Team:Cambridge-JIC/Informatics#Codon-optimisation">Codon Optimisation</A></LI><LI><A href="/Team:Cambridge-JIC/Informatics#Promoter">Promoter Screen</A></LI><LI><A href="/Team:Cambridge-JIC/Technology">Technology</A></LI><LI><A class="head" href="#">Parts and Results</A></LI><LI><A href="/Team:Cambridge-JIC/Parts">Parts</A></LI><LI><A href="/Team:Cambridge-JIC/Results">Results</A></LI><LI><A href="/Team:Cambridge/Partners" class="head">Outside the lab</A></LI><LI><A href="/Team:Cambridge-JIC/TimeLine">Preparing for iGEM</A></LI><LI><A href="/Team:Cambridge-JIC/Getting the word out">Getting the word out</A></LI><LI><A href="/Team:Cambridge-JIC/Outreach and Human Practices">Outreach</A></LI><LI><A href="/Team:Cambridge-JIC/Socials">Socials</A></LI><LI><A href="/Team:Cambridge-JIC/Safety">Safety and Considerations</A></LI><LI><A href="/Team:Cambridge/Attributions" class="head">Attributions</A></LI><LI><A href="/Team:Cambridge-JIC/Attributions#Collaborators">Acknowledgements</A></LI><LI><A href="/Team:Cambridge-JIC/Attributions#Sponsors">Sponsors</A></LI><LI><A href="/Team:Cambridge/Partners" class="head">iGEM Home</A></LI><LI><A href="/Main_Page">iGEM Home</A></LI></DIV><DIV id="right"><DIV align="center"><A href="https://2014.igem.org/wiki/index.php?title=Team:Cambridge-JIC/Chromoproteins&amp;action=edit">Edit this page</A></DIV><H2>Chromoproteins</H2><H3>Gibson</H3><H4>Attempt 1:</H4><UL><LI>Date: 21.07.2014</LI><LI>Comments: Attempted to assemble all 7 constructs. The Gibson assembly was done following the protocol.
Fragments 1 and 2 were used for all constructs since they represent the pGreen backbone. Nuclearly-localised constructs (N7) also required the addition of 3 (i.e. N7 fragment). The other fragments were added to each constuct reaction specifically:
</LI><LI>eforRed (cyto) - 4</LI><LI>tsPurple (cyto) - 7</LI><LI>eforRed (nucl) - 5</LI><LI>amilCP (nucl) - 6</LI><LI>tsPurple (nucl) - 8</LI><LI>asPink (nucl) - 9</LI><LI>aeBlue (nucl) - 10 </LI></UL><H4>Attempt 2:</H4><UL><LI>Date: 22.07.2014</LI><LI>Comments: Attempted to assemble all 7 constructs. The Gibson assembly protocol was slightly altered, i.e. 1ul of each sample was added to the reaction tube instead of 0.5ul. To account for this change, the amount of master mix added was scaled to 3x the final volume.</LI><LI>The rest is identical to Attempt 1.</LI></UL><H4>Attempt 3:</H4><UL><LI>Date: 23.07.2014 </LI><LI>Comments: Attempted to assemble ONLY nuclear-localised constructs. The Gibson assembly was done following the protocol.The higher-concentration backbone fragments from the cpIII aplification series were used
in conjunction with cpII chromoprotein fragments. </LI></UL><H3>E-Coli transformation and Plasmid Purification</H3><H4>Attempt 1: </H4><UL><LI>Transformation Date: 21.07.2014 </LI><LI>Comments: Left to grow on Kan+ LB agar plates O/N at 37 C, with a negative (Gibson) control.</LI><LI>Colony Screening Date: 22.07.2014 </LI><LI>Comments: Only eforRed (cyto) produced 3 colonies after O/N incubation. Later in the day, colonies were spotted on the asPink (nucl) plate, too. The rest of the plates possessed no colonies. </LI><LI>After colony PCR, selected colonies from the positive plates were grown in 15ml LB overnight and the plasmid DNA was extracted the next day using QIAGEN MiniPrep Plasmid Purification kit.</LI><LI>The concentrations of the plasmid DNAs were read using Nanodrop and documented in the DNA library (A10, A11, A12).</LI></UL><H4>Attempt 2: </H4><UL><LI>Transformation Date: 22.07.2014 </LI><LI>Comments: Left to grow on Kan+ LB agar plates O/N at 37 C. A positive control (p35sRLti) was also used.</LI><LI>Colony Screening Date: 24.07.2014</LI><LI>Comments: Out of the transformed constructs from the second round, only tsPurple (cyto), amilCP (nucl) and eforRed (cyto) plates ended up with colonies.</LI><LI>After colony PCR, selected colonies were picked from each, grown O/N and the plasmids were then extracted by miniprep (DNA Library A13, A14, A15, A16).</LI></UL><H4>Attempt 3: </H4><UL><LI>Transformation Date: 25.07.2014 </LI><LI>Comments: Left to grow on Kan+ LB agar plates O/N at 37 C. A positive control (p35sRLti) was also used, in addition to the Gibson negative control. The plates were removed from
the 37 degrees cabinet and placed into a fridge (-4 C) until further analysis on Monday. </LI><LI>Colony Screening Date: 28.07.2014 </LI><LI>Comments: Out of the 5 nuclear constructs, only asPink and tsPurple plates possessed colonies. After colony PCR, selected colonies were grown O/N followed by miniprep (DNA Library A19, A20, A21, A22, A23). </LI></UL><H3>Colony PCR, Digest and Sequencing</H3><H4>Colony PCR</H4><H4>Attempt 1: </H4><UL><LI>Date:24.07.2014 </LI><LI>Comments: The eforRed (cyto) colonies showed a band of approx. 1kb, agreeing with the predicted 1042bp. asPink colony bands resembled the bands of the p35sRLti control. However, the band was close enough to the predicted 1303bp for us to pick one of the colonies for O/N culture.
[PHOTO 1 SCAN HERE] </LI></UL><H4>Attempt 2: </H4><UL><LI>Date:25.07.2014 </LI><LI>Comments: The predicted band size for tsPurple (cyto) is 1045bp, eforRed (cyto) 1042bp and amilCP (nucl) 1270bp. Colonies which fit this prediction most closely were chosen for further analysis.
[PHOTO 2 SCAN HERE]</LI></UL><H4>Attempt 3: </H4><UL><LI>Date:28.07.2014 </LI><LI>Comments: Due to a large degree of smudging present on the gel, all colonies were chosen for future analysis (digest).
[PHOTO 3 SCAN HERE] </LI></UL><H4>Restriction Digest</H4><UL><LI>Date: 30.07.2014 </LI><LI>Comments: All of the selected DNA constructs (A10, A11, A12, A13, A14, A15, A16, A19, A20, A21, A22, A23 and A24) were used in a restriction digest assay using the enzyme DraI. </LI><LI>The protocol used was based on the instructions given by the manufacturer (NEB, DraI, see: https://www.neb.com/protocols/2012/12/07/optimizing-restriction-endonuclease-reactions).
The digested DNA was resolved on an agarose gel (60min, 100V).
[DIGEST PHOTO SCAN HERE] </LI></UL><H4>Sequencing</H4><UL><LI>Date: 31.07.2014 </LI><LI>Comments: The constructs were prepared for sequencing as specified by ______[insert company] and using the nosT_Bseq primer (P28). </LI><LI>Sequences: </LI></UL><H3>Spore preparation</H3><H4>Attempt 1:</H4><UL><LI>Date: 30.07.2014 </LI><LI>Comments: Standard protocol was followed. Used 1 spore head per 2 spore preparations.</LI></UL><H3>Agrobacteria transformation</H3><H4>Attempt 1:</H4><UL><LI>Date: 29.07.2014 </LI><LI>Comments: Electroporation was done on our stock of competent Agrobacteria (protocol from the Plants Dept - http://www.plantsci.cam.ac.uk/research/jillharrison/protocols/cloning/preparation-of-agrobacterium-electrocompetent.pdf/view), following the Abrobacteria Transformation protocol. All 13 constructs were transformed,
including a positive and negative control. The transformed cells were then plated onto quadruple resistence plates (Kan 50mg/ml, Rif 10mg/ml, Tet 5 mg/ml, Carb 50 mg/ml) and left for 3 days. </LI></UL><H4>Agrobacterial induction and Co-cultivation</H4><UL><LI>Induction date: 04.08.2014 </LI><LI>Co-cultivation start: 04.08.2014 </LI><LI>Co-cultivation stop: 07.08.2014 </LI><LI>Comments: The selected colonies of transformed Agrobacteria were induced in a 1/2 GB + 5% sucrose + 100 uM acetosyringone medium for 6 hours
prior to co-cultivation with Marchantia sporelings. The selected colonies were: </LI><LI>Co-cultivation was carried on until Thursday, when the sporelings were washed with ddH2O + 100 uM cefotaximine, and plated onto double-antibiotic plates (Hygromycin and Cefotaximine). </LI></UL><H3>RESULTS:</H3>
11.07.2014 - many sporelings are becoming yellow in appearance, hinting to dying due to not being transformed. Parts of individual plants appear greener, suggesting these areas may have been successfully transformed. No colour is detectable at the current moment. There
also appears to be minor contamination of certain plates with a small, relatively isolated orange colony of either fungus or bacterium.



<DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Cambridge-JIC/Chromoproteins">http://2014.igem.org/Team:Cambridge-JIC/Chromoproteins</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cambridge-JIC/Chromoproteins" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cambridge-JIC/Chromoproteins" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cambridge-JIC/Chromoproteins&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cambridge-JIC/Chromoproteins&amp;oldid=66899" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>