"
Page
Discussion
View source
History
teams
Log in
 
Team:Cambridge-JIC/Marchantia/Assembly
From 2014.igem.org
Cambridge iGEM 2014
Toggle navigation
Team
mösbi
Overview
Components
mösbi&You
Growth Chamber
Vision
Marchantia
Background
Enhancer trap
Promoter hunt
Codon optimisation
Request For Comments
Assembly
Guide
Results
Labwork results
Judging criteria
Parts table
Community
Scientific community
Outreach
Safety
Policy
Workshop
Acknowledgements
Executive Summary
Keep in contact with us...
DNA Assembly
Working with Marchantia, we threw the kitchen sink at building constructs! This page outlines our efforts!
Overview
When dealing with a eukaryotic chassis such as marchantia, it becomes increasingly important to be able to assemble constructs with a range of parts into complex constructs.   When working with a new chassis - where parts are scattered from source to source, it becomes greatly important to have something very flexible
We considered 3 main assembly techniques
Gibson assembly. This was our de facto default, and did the job for most of the constructs we wanted to build
MoClo. This enbled rapid one pot assembly of parts, but required the parts to have been domesticated for use with them. Since all of our parts were new, we decided not to domesticate our parts prior to building constructs, but instead to domesticate them after having verified that they work!
Ligase cycling reaction. This promised rapid and efficient assembly of large constructs - but was new. We tried it out! (thanks to generous sponsorship from CamBio )
Ligase Cycling Reaction (LCR)
When dealing with a eukaryotic chassis such as marchantia, it becomes increasingly important to be able to assemble constructs with a range of parts. For us, in particular, we wanted to assemble a sample processing module consruct: with 35S driving HAP1, with a hammerhead ribozyme in the 3'UTR, and with the HAP1UAS driving Venus YFP with the N7 nuclear localisation tag.
Now, we figured out that for the above construct, we'd need to do a 6 part Gibson assembly for it to work. It was attempted, but no colonies were achieved.
We were drawn in to Ligase Cycling Reaction, described in December 2013 in this paper by its simplicity and promised efficiency over other assembly techniques, for example Gibson. According to the  paper, it is possible to assemble 12 fragments together in one reaction, with >80% efficiency.
The reaction is simple: DNA is denatured. Each fragment, either side of a junction, anneals to a bridging oligo which spans the junction. The ligase seals the gaps. This joint fragment acts then acts as a template for annealing.
The Test Construct
Deciding that it could be extremely profitable to establish a protocol, we decided to design a simple, two part construct using chromoproteins to assay whether it would be feasible to include it.
The construct inserted amilCP into a cassette with a different resistance, containing mRFP1 instead of amilCP. That is, red colonies would be template, blue would be something sensible, and white would be misassembly of some sort.
We contacted CamBio , the UK distributor of the Ampligase thermostable ligase enzyme, to see if we could use it.
We received, generously, from them 2500U of the enzyme.
For LCR, fragments need to be 5' phosphorylated. This can be done in two ways: either by ordering phosphorylated primers [currently costing 3x the amount of non-phosphorylated primers] OR by ordering normal primers, and then kinasing PCR amplified fragments using T4 Polynucleotide Kinase.
The optimized protocol was followed as per the reference above. A Dpn1 digest following PCR was not done, as template would appear red.
The results were the following:
Left: colonies after assembly using nonphosphorylated primers. Right: colonies after assembly using phosphorylated primers [template in top right of right figure]
An explanation of the difference in efficiency was explained by not having added additional NAD to the left-hand reaction. The overall low efficiency was perhaps explained by recircularisation of the backbone. Additionally, 10x the recommended concentration of bridging oligos was used. Time to repeat this experiment was not available.
Building the Marchantia construct
The construct was split into 7 fragments, each around 2000-2500bp, and nonphosphorylated primers were used.
Initially, reactions were done in a 25ul volume, and a Dpn1 digest was performed.
The results yielded no colonies following transformation. We ran out the PCR purified fragments on a gel, and saw that no bands were present.
After having discussed with the authors of the paper, we found also that Ampligase performs NAD-dependent ligation, and that the T4 PNK also has NAD dependent kinase activity.
In the subsequent attempt, we ran out PCR products on a gel, and then performed an extraction. This way, it was clear from the outset that all of the PCR's had worked.  The buffer used in commercial gel extraction kits means that concentrations are very inaccurate on a spectrophotometer. We used the Qubit fluorescent probe based assay to measure the concentration, but unfortunately, it was not high enough to continue with the reaction.
Using the gel extracted fragments as a template, we repeated PCRs, and subsequently performed a PCR-purification. This yielded accurate measurements of concentration, and enabled us to continue with the assembly.
Following the reactions, colonies were observed on all of the hammerhead ribozyme constructs. A colony PCR was done with 4 primers, and all four fragments were observed by gel electrophoresis, on some of the colonies. It was repeated with 2 primers, and the bands were observed on the same colonies.
We miniprepped DNA from the colonies for which the colonyPCR was positive, and performed a restriction digest. Two colonies' DNA appeared to produce the correct fragments upon gel electrophoresis.
These were sent for sequencing. Unfortunately, sequencing did not produce  reads on any of the samples sent (which included samples from other constructs later verified).
We continued with the Marchantia transformation.
A Suggested Protocol
We followed the protocol in the paper above for the PNK treatment. You might find the following useful:
- Have a total reaction volume of 15ul. You only transform up to 5ul, and so it saves on valuable enzyme to use a total volume smaller than 25ul.
Recommended concentrations for reagents:
Betaine: 9M, 0.75ul, final concentration: 0.45M. Make a suspension, or warm the tube if you're struggling to dissolve.
Bridging oligos: make a mix of the oligos, and dilute this with ddH2O until the final volume you add is 1.6ul. Final concentration: 30nM.
Ampligase: 5U/ul, 0.45ul
Ampligase buffer: 10x, 0.5ul [note: the kinase is done in the Ampligase buffer, so you need to add accordingly less]
DMSO: 100%, 1.2ul
NAD: 15mM, 0.5ul
PNK mix: 10ul
Retrieved from " http://2014.igem.org/Team:Cambridge-JIC/Marchantia/Assembly "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
