<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cambridge-JIC_Results_Lab"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Cambridge-JIC/Results/Lab">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cambridge-JIC/Results/Lab&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge-JIC/Results/Lab&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge-JIC/Results/Lab&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cambridge-JIC/Results/Lab" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cambridge-JIC/Results/Lab</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Cambridge iGEM 2014</TITLE></P><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-ex1-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse navbar-right navbar-ex1-collapse"><UL class="nav navbar-nav"><LI><A href="https://2014.igem.org/Team:Cambridge-JIC/Team">Team</A></LI><LI class="dropdown"><A href="https://2014.igem.org/Team:Cambridge-JIC" class="dropdown-toggle" data-toggle="dropdown">mösbi</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Project/Overview">Overview</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Project/Components">Components</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Project/Mosbiandyou">mösbi&amp;You</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Project/MGF">Growth Chamber</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Project/Vision">Vision</A></LI></UL><LI class="dropdown"><A href="https://2014.igem.org/Team:Cambridge-JIC" class="dropdown-toggle" data-toggle="dropdown">Marchantia</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Marchantia/Background">Background</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Marchantia/EnhancerTrap">Enhancer trap</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Marchantia/Promoter">Promoter hunt</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Marchantia/Codon">Codon optimisation</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Marchantia/RFC">Request For Comments</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Marchantia/Assembly">Assembly</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Guide">Guide</A></LI><LI class="dropdown"><A href="https://2014.igem.org/Team:Cambridge-JIC" class="dropdown-toggle" data-toggle="dropdown">Results</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Results">Labwork results</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Results/Judging">Judging criteria</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Results/Parts">Parts table</A></LI><LI class="dropdown"><A href="https://2014.igem.org/Team:Cambridge-JIC" class="dropdown-toggle" data-toggle="dropdown">Community</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Community/Sci">Scientific community</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Community/Outreach">Outreach</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Safety">Safety</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Policy">Policy</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Community/Workshop">Workshop</A></LI><LI><A href="https://2014.igem.org/Team:Cambridge-JIC/Attributions">Acknowledgements</A></LI><LI><A href="https://2014.igem.org/Team:Cambridge-JIC/ExecSummary">Executive Summary</A></LI></DIV></DIV><DIV class="footer"><DIV class="content-section-a"><DIV class="container"><DIV class="row"><DIV class="col-lg-12"><UL class="list-inline"><LI>Keep in contact with us...</LI></UL></DIV></DIV></DIV></DIV></DIV><DIV align="center"><A href="https://2014.igem.org/wiki/index.php?title=Team:Cambridge-JIC/Results&amp;action=edit">Edit this page</A></DIV><H2>Results and further experiments</H2><H3 id="Chromoproteins">Chromoproteins </H3><P>
As an example output plugin we transformed into Marchantia a selection of the chromoproteins brought to iGEM by Uppsala 2011. This was to test whether a simple change in colour would work as an easily visible reporter.
</P><P>
Five chromoproteins were selected for expression: eforred (BBa_K592012), tspurple (BBa_K1033906),  aspink (BBa_K1033927), aeblue (BBa_K1033929) and amilCP(BBa_K1033930, a deep blue colour). The N7 nuclear localisation tag (INSERT BIOBRICK NUMBER HERE) was added to tspurple, aspink and amilCP. Each was put into pGreen
</P><P>
The transformation procedure yielded a large number of transformants, approximately 100 per construct, which were confirmed via PCR using homogenised plants as a template and chromoprotein-specific primers for each tDNA added.
</P>

Left: A PCR of the transformed plants showing successful transformation. Right: A plate of transformants growing happily on hygromycin
<P>For each chromoprotein construct, the transformation yielded approximately 100 transformed sporelings (7-days old plants). The petri dish to the right contains hygromycin as a selective marker. The gel confirms the transformation in three plants each from two other plates.  The forward and reverse primers used in the PCR anneal to 35S and N7 respectively, and the template was homogenised plant material. </P><P> After approximately four weeks of growth, sparsely distributed bright red cells became apparent under the microscope in the plants transformed with the tsPurple gene. Although similar cells were not found in wild type plants, diffuse red regions were found and are probably due to release of anthocyanins by stressed or aging cells. </P><P align="center">
The bright red cells seen in Marchantia transformed with the TsPurple construct
</P><P>It is notable that in the red cells, the vacuole is not visually distinct from the cytoplasm. As GFP, a protein of similar structure, is not accumulated in the vacuole without specific targeting, this implies that either the colour we observe in the bright cells is not due to chromoprotein expression, or perhaps that tsPurple is toxic enough to cause cellular damage and leaky organelles. </P><P>Plants transformed with asPink (which is also known as asFP595 due to its fluorescence), nuclear localised with an N7 tag, while not visibly pigmented, did have areas that fluoresced at the literature wavelengths.</P><P>Part of an asPink:N7 transformant visualised using a fluorescence microscope with a GFP filter (the only one with wavelength windows that intersect those of asPink's absorption and emission peaks)</P><P>In addition, we measured the absorption spectra of the chromoprotein as expressed in E.coli for comparison purposes.</P><H1>trang Anh please insert diag here </H1><H3 id="Raspberry ketone production">Raspberry ketone production </H3><P>As another example output module, the biosynthetic pathway from coumaroyl-coA to raspberry ketone was assembled, to allow the smell of raspberries to be used as an odour signal that the input module has been activated.</P><P> To characterise the enzymes involved, a bacterial constructs was made, and the E. Coli transformed with it we called RaspberrE coli. The construct consisted of an arabinose inducible expression plasmid coding for 4-coumarate coA ligase, benzalacetone synthase, and benzalacetone reductase, and the strain of E. Coli we used had the TnaA indole synthase gene knocked out to somewhat reduce the faecal odour associated with this molecule. (growing in M9 salts with casamino acids and no tryptophan eliminated the faecal odour entirely, as the smell is due entirely to tryptophan derivatives).</P><P>The RaspberrE coli were incubated for two and a half days with 3 or 9mM p-coumaric acid and 50mM arabinose to induce expression. The cultures were then shaken with ethyl acetate which was then run on an LCMS setup (which was only possible due to the kind giving of time and expertise by Ben Pilgrim at the Cambridge Department of Chemistry).</P><P>As a control, one of the tubes incubated contained a strain of E coli resistant to chloramphenicol but with none of the raspberry ketone producing genes. There was one peak that existed on the RaspberrE coli spectra but not on the control, which when run through the mass spectrometer turned out to be benzalacetone, the precursor to raspberry ketone.</P><P> None of the cultures which also had benzalacetone reductase transformed in produced any peaks which gave mass spectra corresponding to that of raspberry ketone. These results suggest that while the 4-coumarate coA ligase and benzalacetone synthase both worked, the benzalacetone reductase, as shown once already by the Dutch research team who first cloned the enzyme’s gene sequence (led by Jules Beekwider in Wageninen), does not actually reduce benzalacetone.</P>


Some other experiments we did:
<H2 id="Overcoming the smell of E-coli by two different techniques and their comparison">Overcoming the Indol smell of E-coli by two different techniques and their comparison </H2><H4>Problem</H4><P>
E-coli cultures have a pungent odour due mostly to the production of indole. We were afraid that the strong smell of E-coli would mask any other synthesized volatiles. We therefore decided to look into ways to reduce their production of indole.
</P><H4>Summary</H4><P>
We overcame the odour issue by advancing a new protocol to make normal E-coli strains produce less indole. We compared the efficiency of our technique in terms of both odour and growth rate with indole-free strain of bacteria.
Our technique produced pleasant smelling E-coli, indistinguishable from indole-free bacteria. However they required 6h more to reach similar population density.
We also produced E-coli with equal growth rate as standard bacteria in standard media which smelled significantly less.
</P><H4>Story</H4><P>
1-	We located an indole free strain of Ecoli from a neighbouring lab. This mutant strain has a non-functional TnA enzyme so can no longer metabolize Tryptophan into Indole.
2-	 We also learnt that normal bacteria grown on a Tryptophan-free medium and minimal salts have been shown to produce significantly less indole and still grow at good rates. No data had been recorded on smell or growth rate.

</P><H4>Experiment</H4><P>
We set up cultures of 
Ecoli +T (wild type) and Ecoli –T (which can’t make indole) and grew them in
</P><P>-minimal medium [salts, Mg, ]
-minimal medium + Casamino acids (which doesn’t contain Tryptophan)
-minimal medium + Casamino acids + Tryptophan</P><H4>Results</H4><UL>
Density
<P>
The cultures were grown overnight. In the morning, the absorption was measured to compare population growth. All tubes were cloudy with cultures apart from the control. 
Both strains in minimal medium grew at the same rate, proving that the indole-free strain was a vigorous.
For both strains the min.medium + casamino Acid contained approximately twice the bacteria concentration as the min.medium alone.  This was expected as the lack of amino acid restricts the growth of bacteria.
There was no significant difference in population concentration between the Tryptophan and Tryptophan free medium for the Ecoli-T strain. This is understandable as these bacteria cannot use this added amino acid either way.
There was a slight increase in population for the Ecoli + T strain. This shows Tryptophan does limit to some degree the growth of bacteria but not very much.
</P>

Odour
<P>
To compare scent given off, all cultures were grown to a same OD (optical density). When this was achieved, the tubes were smelt blind by different people outside our own lab.
The Ecoli-T strain had a pleasant smoky odour, very distinct from the wild Ecoli+T strain.
The Ecoli+T strain smelled only very mildly in the Tryptophan free medium and had the full pungent smell with Tryptophan. 
</P></UL><H4>Conclusion</H4><P>
1- Ecoli-T (indole free strain) grew as efficiently as the standard E-coli strain, thus demonstrating their virulence.
This strain did not produce the pungent smell thus proving that indole is indeed the culprit.
2- E-coli grown on minimal medium produce significantly less indole and therefore have less pungent smell.
The culture took 6h longer to reach the same population density as the E-coli optimum medium.
3-We recommend using Ecoli-T (indole free) strain to have lower smell and prevent over masking other volatiles.
4- As a more accessible alternative, we suggest to other teams who would not have access to Indole free E-coli strains to grow E-coli on minimal medium (protocol described above) with up to 6h more of incubation (however this is not essential to get adequate yield for a miniprep)</P>
___________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________
<DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Cambridge-JIC/Results/Lab">http://2014.igem.org/Team:Cambridge-JIC/Results/Lab</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cambridge-JIC/Results/Lab" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cambridge-JIC/Results/Lab" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cambridge-JIC/Results/Lab&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cambridge-JIC/Results/Lab&amp;oldid=355056" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>