"
Page
Discussion
View source
History
teams
Log in
 
Team:Cambridge-JIC/Marchantia/EnhancerTrap
From 2014.igem.org
Cambridge iGEM 2014
Toggle navigation
Team
mösbi
Overview
Components
mösbi&You
Growth Chamber
Vision
Marchantia
Background
Enhancer trap
Promoter hunt
Codon optimisation
Request For Comments
Assembly
Guide
Results
Labwork results
Judging criteria
Parts table
Community
Scientific community
Outreach
Safety
Policy
Workshop
Acknowledgements
Executive Summary
Keep in contact with us...
Contents
1How does an Enhancer Trap work?
2To sum up the biology of the construct:
3Construct Design
4Example of Uses
5Progress and Results
How does an Enhancer Trap work?
An enhancer trap constructs contains a transcription activator (e.g.: GAL4) fused to a weak or basal promoter (TATA box). The construct also contains a reporter gene (e.g.: YFP) fused to a UAS (upstream activator sequence activated by GLA4). This allows YFP to report the GAL4 expression. On its own the basal promoter is insufficient to drive detectable expression of the reporter gene, but can respond to transcriptional enhancers from the genome it is inserted next to.. Once a transgenic line is generated using such a construct it can be crossed with any other transgenic line containing a UAS::GOI (gene of interest). The offspring will express the gene of interest with the same characteristic seen as the reporter gene (e.g.: YFP).
To sum up the biology of the construct:
A red fluorescent protein expressed all the time, everywhere (to check construct insertion)
A hygromycin selectable marker (to allow only transformant to survive)
A minimal promoter fused to a gal4 sequence (gal4 will be expressed if minimal prom has been inserted next to an enhancer)
A gal4 induced YFP (to be able to detect expression pattern of gal4 by the ‘trapped enhancer’)
Construct Design
The construct was formed from two plasmids A and B that were given to us by Jim Haseloff and Bernado Pollak respectively. The minimal promoter was extracted from plasmid A and inserted in plasmid B. The position of the hygromycin cassette and minimal promotergal4 cassette were exchanged to the position seen below. This was done to place the minimal promoter at the very border of the sequence allowing genomic enhancers to act directly on it.
LB
Starting point of integration of DNA into plant
minimal promoter
TATAbox from 35S (a standard promoter)
GAL4
Transcription activator molecule
VP-16
viral protein 16: nuclear localization tag
MpEF1a promoter + 5’UTR
endogenous early expressing promoter
Hygromycin resistance gene
Hygromycin Terminator
Nos Terminator
UAS for GAL4
Gal4 activated Upstream Activator Sequence
VENus
A yellow fluorescent molecule
Linker
N7
nuclear localization tag
Nos Terminator
35S
CaMV promoter for the 35S Ribosomal
RFP
Red fluorescent protein
Linker
LTI
Plasma membrane localization tag
Terminator
Start of integration of DNA into plant
RB
Example of Uses
Targeted Expression Often we do not want the whole of an organism to express our gene of interest but only specific cells in order to minimize variables and get targeted expression. We find a transformed plant that expresses YFP only in the roots (or rhizoids). We cross the line of plant with a plant that contains a gla4UAS_gene-of-interest. The offspring will express the ‘gene of interest’ only in the rhizoids. The gene of interest could be anything from a chromoprotein, to a growth hormone. In this way researchers can target expression to specific cells.
Early Sexual Differentiation we find a transformed plant that expresses YFP only at the onset of male sexual maturation. This gives us a way of sexing plants before full maturation. It also tells us that the gene downstream of the ‘trapped promoter’ will have some interesting properties concerning development and should be looked into.
Progress and Results
The construct has successfully been constructed and sequenced in E-coli. We electroporated the plasmid into Agrobacterium tumefaciens, which were then grown. Marchantia spores were then co-cultured with the Agrobacteria. We plated the plantlets on hygromycin containing plates. At the moment selection is occurring: only transformed plants with the hygromycin resistance gene are developing and non-transformants dwindle. Our next step is to re-plate the surviving transformants, to spread them out and allow more effective screening. This is to be done in the next coming days when the plantlets will have reached adequate size. After this screening begins! We very happy with our progress and look forward to finding interesting expression patterns. We’ll keep you up to date with the upcoming results! Keep checking the page out for colourful photos!
Retrieved from " http://2014.igem.org/Team:Cambridge-JIC/Marchantia/EnhancerTrap "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
