"
Page
Discussion
View source
History
teams
Log in
 
Team:Cambridge-JIC/Progress
From 2014.igem.org
Cambridge iGEM 2014
Toggle navigation
Team
mösbi
Overview
Components
mösbi&You
Growth Chamber
Vision
Marchantia
Background
Enhancer trap
Promoter hunt
Codon optimisation
Request For Comments
Assembly
Guide
Results
Labwork results
Judging criteria
Parts table
Community
Scientific community
Outreach
Safety
Policy
Workshop
Acknowledgements
Executive Summary
Keep in contact with us...
Labwork
Status, protocols and safety considerations for our everyday wetwork
Chromoprotein Constructs
Design
Aim of the construct
Chromoproteins represent ideal reporter genes. Unlike fluorescent protein, from which chromoproteins were originally derived, they require no specialised optical equipment for detection. Our aim is to use a selection of such proteins, donated by the iGEM 2012 Uppsala team, as our most basic and main output in regards to the Marchantia framework.
We decided to test the following chromoprotein constructs:
35s - eforRed - nosT
35s - eforRed - N7 - nosT
35s - amilCP - N7 - nosT
35s - tsPurple - nosT
35s - tsPurple - N7 - nosT
35s - asPink - N7 - nosT
35s - aeBlue - N7 - nosT
By expressing the chromoproteins in the pGreen vector with the 35s promoter and the nosT terminator, we control for the possiblity of the promoter/terminator not functioning in our chassis.
The N7 fragment allows for the selected constructs to localise chromoprotein to the nucleus. It is believed by Bernardo Pollak that this may end up increasing the likelihood of detecting the chromoproteins with the naked eye.
End goal plasmid
insert image here
Experimentation (up to date)
PCR
Attempt 1:
Date: 18.07.2014
UIDs of primers/plasmids used:
Backbone Fragment 1 (35s - Hyg promoter): B14, P16, P14. Tubes: CpI 1
Backbone Fragment 2(Hyg promoter - nosT): B14, P15, P17. Tubes: CpI 2
N7 fragment: B15, P13, P22. Tubes: CpI 3
eforRed gene(cytoplasmic): B13, P1, P2. Tubes: CpI 4
eforRed gene(nuclear): B13, P1, P3. Tubes: CpI 5
amilCP gene(nuclear): B10, P4, P5. Tubes: CpI 6
tsPurple gene(cytoplasmic): B12, P6, P7. Tubes: CpI 7
tsPurple gene(nuclear): B12, P6, P8. Tubes: CpI 8
asPink gene(nuclear): B9, P9, P10. CpI 9
aeBlue gene(nuclear): B11, P11, P12. Tubes: CpI 10
After running the gel, the concentrations of CpI 1, CpI 2 and CpI 3 were much lower compared to the chromoprotein genes (see gel picture). Amplification of CpI 3 failed. The problem was an alteration to the primers necessary for amplification. Re-ran PCR for CpI 3. Modification: Appropriate primers (P13 & P22) and the extension phase of PCR has been reduced from 2 min to 15 sec since N7 is only ~300bp long. The DNA was then purified using the QIAGEN QiaQuick Gel Purification Kit. The DNA was eluted in 50 ul of EB buffer. Due to the large elution volume and the lower yield of the backbone DNA, the concentrations (measured using Nanodrop) were lower than the suggested 10ng/ul threshold for Gibson.
PCR Excell Protocol
Gel Picture
Attempt 2:
Date: 21.07.2014
UIDs of primers/plasmids used:
Backbone Fragment 1 (35s - Hyg promoter): B14, P16, P14. Tubes: CpII 1
Backbone Fragment 2(Hyg promoter - nosT): B14, P15, P17. Tubes: CpII 2
N7 fragment: B15, P13, P22. Tubes: CpII 3
eforRed gene(cytoplasmic): B13, P1, P2. Tubes: CpII 4
eforRed gene(nuclear): B13, P1, P3. Tubes: CpII 5
amilCP gene(nuclear): B10, P4, P5. Tubes: CpII 6
tsPurple gene(cytoplasmic): B12, P6, P7. Tubes: CpII 7
tsPurple gene(nuclear): B12, P6, P8. Tubes: CpII 8
asPink gene(nuclear): B9, P9, P10. CpII 9
aeBlue gene(nuclear): B11, P11, P12. Tubes: CpII 10
Due to the low yield of the DNA purification fro Attempt 1, the entire PCR was repeated. Upon gel separation we realised the PCRs were nearly identical (see Gel Picture), which is what we were expecting. However, the DNA was euted into 10ul of EB buffer instead of 50ul during the gel purification step, which increased the concentrations of CpII 4-10 significantly. However, CpII 1-3 concentrations were still below the 10 ng DNA/ul range.
PCR Excell Protocol
Gel Picture
Attempt 3:
Date: 22.07.2014
UIDs of primers/plasmids used:
Backbone Fragment 1 (35s - Hyg promoter): B14, P16, P14. Tubes: CpIII 1
Backbone Fragment 2 - Full Size(Hyg promoter - nosT): B14, P15, P17. Tubes: CpIII 2 RBLB
Backbone Fragment 2 - LB(Hyg promoter - KanR): B14, P18, P17. Tubes: CpIII 2 LB
Backbone Fragment 2 - RB(KanR - nosT): B14, P15, P19. Tubes: CpIII 2 RB
N7 fragment: B15, P13, P22. Tubes: CpIII 3
The backbone fragments were re-aplified to allow for future assemblies to be made. The PCR was altered to run for 35 amplification cycles instead of 30, which increase the yield of DNA.
PCR Excell Protocol
Gel Picture
Gibson
Attempt 1:
Date: 21.07.2014
End date/time:
Comments:
E-Coli transformation
Attempt 1:
Start date/time:
End date/time:
Comments:
Agrobacteria transformation
Attempt 1:
Start date/time:
End date/time:
Induce comments:
Growth comments:
Electroporation/selection comments:
Spore preparation
Attempt 1:
Start date/time:
End date/time:
Comments:
Spore transformation
Attempt 1:
Start date/time:
End date/time:
Comments:
Evaluation
Start date/time:
End date/time:
Comments:
Gal4 - Hap1GR - H2BmRFP
Design
Aim of the construct
We'll try out 35s->GAL4->HAP1GR->H2BmRFP.
Since H2BmRFP has not been tried in Mp, we'll also try 35s->H2BmRFP. A positive control for lab technique will be 35s->Venus N7
We'll also use FF's original plasmid to try 35s-Hap1GR-mRFP, as a control for the Hap1GR. I expect low transformation rate as in the original plasmid, we're using 35s as promoter for HygR selection marker.
Not sure whether 35s->Gal4 has been tried (check with BP), so 35s-Gal4-venus n7 is doable with BP's construct as a control
As another control, we'll try 35s-Gal4-H2mRFP
End goal plasmid
Experimentation (up to date)
PCR
Attempt 1:
Start date/time: 24/07, 11:08
End date/time:24/07, 1550
UIDs of primers/plasmids used:
A1, P33, P34
A1, P35, P36
A2, P37, P38
A2, P39, P40
PCR Spreadsheet (on google drive)
Gel 27/07/2014
Gibson
Attempt 1:
Start date/time: 25/07
Comments: seemed to be fine.
E-Coli transformation
Attempt 1:
Start date/time: 25/07
Comments:
Negative control negative, positive control positive.
Selection
Attempt 1:
Start date/time: 28/07
Comments:
Selected initially just 3 colonies. Grew them in LB and miniprepped on 27/07. Performed digest with SalI to check for template carryover and see that assembly was done. DEXX3, pb>RVe each have a unique site. Following digest with Sal1, we should have a 12kb fragment and a 2kb fragment.  After digest, 2kb fragment was visible, but 12kb fragment was not. Very mysterious.
Some error in the digestion protocol is likely to have occurred.
To insert a promoter, a digest with Kpn1 was performed. Gel was illegible, although a faint band at 13kb may have been visible. No extraction was made due to poor quality of the gel.  Attempt 2:
Start date/time: 30/07
Comments:
Grew up in LB 8 more colonies. Attempted colony PCR using inoculation loop in dry PCR tube - will wait to see results from GEL. Expect an 8kb fragment: unlikely that this will show results. Will perform a SalI digest with correct protocol on 31/07 following miniprep from colonies.
Agrobacteria transformation
Attempt 1:
Start date/time:
End date/time:
Induce comments:
Growth comments:
Electroporation/selection comments:
Spore preparation
Attempt 1:
Start date/time:
End date/time:
Comments:
Spore transformation
Attempt 1:
Start date/time:
End date/time:
Comments:
Evaluation
Start date/time:
End date/time:
Comments:
Edit this page
Retrieved from " http://2014.igem.org/Team:Cambridge-JIC/Progress "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
