"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Cambridge-JIC/Chromoproteins
From 2014.igem.org
Welcome
Home
Project Overview
Meet the Team
Official Team Profile
Introducing Marchantia
Why Marchantia?
Background on Bryophytes
Labwork
Notebook
Chromoproteins
Raspberry Ketones
Hammerhead Ribozyme
Enhancer Trap
Protocols
Dry Work
Bioinformatics
Codon Optimisation
Promoter Screen
Technology
Parts and Results
Parts
Results
Outside the lab
Preparing for iGEM
Getting the word out
Outreach
Socials
Safety and Considerations
Attributions
Acknowledgements
Sponsors
iGEM Home
iGEM Home
Edit this page
Chromoproteins
Gibson
Attempt 1:
Date: 21.07.2014
Comments: Attempted to assemble all 7 constructs. The Gibson assembly was done following the protocol. Fragments 1 and 2 were used for all constructs since they represent the pGreen backbone. Nuclearly-localised constructs (N7) also required the addition of 3 (i.e. N7 fragment). The other fragments were added to each constuct reaction specifically:
eforRed (cyto) - 4
tsPurple (cyto) - 7
eforRed (nucl) - 5
amilCP (nucl) - 6
tsPurple (nucl) - 8
asPink (nucl) - 9
aeBlue (nucl) - 10
Attempt 2:
Date: 22.07.2014
Comments: Attempted to assemble all 7 constructs. The Gibson assembly protocol was slightly altered, i.e. 1ul of each sample was added to the reaction tube instead of 0.5ul. To account for this change, the amount of master mix added was scaled to 3x the final volume.
The rest is identical to Attempt 1.
Attempt 3:
Date: 23.07.2014
Comments: Attempted to assemble ONLY nuclear-localised constructs. The Gibson assembly was done following the protocol.The higher-concentration backbone fragments from the cpIII aplification series were used in conjunction with cpII chromoprotein fragments.
E-Coli transformation and Plasmid Purification
Attempt 1:
Transformation Date: 21.07.2014
Comments: Left to grow on Kan+ LB agar plates O/N at 37 C, with a negative (Gibson) control.
Colony Screening Date: 22.07.2014
Comments: Only eforRed (cyto) produced 3 colonies after O/N incubation. Later in the day, colonies were spotted on the asPink (nucl) plate, too. The rest of the plates possessed no colonies.
After colony PCR, selected colonies from the positive plates were grown in 15ml LB overnight and the plasmid DNA was extracted the next day using QIAGEN MiniPrep Plasmid Purification kit.
The concentrations of the plasmid DNAs were read using Nanodrop and documented in the DNA library (A10, A11, A12).
Attempt 2:
Transformation Date: 22.07.2014
Comments: Left to grow on Kan+ LB agar plates O/N at 37 C. A positive control (p35sRLti) was also used.
Colony Screening Date: 24.07.2014
Comments: Out of the transformed constructs from the second round, only tsPurple (cyto), amilCP (nucl) and eforRed (cyto) plates ended up with colonies.
After colony PCR, selected colonies were picked from each, grown O/N and the plasmids were then extracted by miniprep (DNA Library A13, A14, A15, A16).
Attempt 3:
Transformation Date: 25.07.2014
Comments: Left to grow on Kan+ LB agar plates O/N at 37 C. A positive control (p35sRLti) was also used, in addition to the Gibson negative control. The plates were removed from the 37 degrees cabinet and placed into a fridge (-4 C) until further analysis on Monday.
Colony Screening Date: 28.07.2014
Comments: Out of the 5 nuclear constructs, only asPink and tsPurple plates possessed colonies. After colony PCR, selected colonies were grown O/N followed by miniprep (DNA Library A19, A20, A21, A22, A23).
Colony PCR, Digest and Sequencing
Colony PCR
Attempt 1:
Date:24.07.2014
Comments: The eforRed (cyto) colonies showed a band of approx. 1kb, agreeing with the predicted 1042bp. asPink colony bands resembled the bands of the p35sRLti control. However, the band was close enough to the predicted 1303bp for us to pick one of the colonies for O/N culture. [PHOTO 1 SCAN HERE]
Attempt 2:
Date:25.07.2014
Comments: The predicted band size for tsPurple (cyto) is 1045bp, eforRed (cyto) 1042bp and amilCP (nucl) 1270bp. Colonies which fit this prediction most closely were chosen for further analysis. [PHOTO 2 SCAN HERE]
Attempt 3:
Date:28.07.2014
Comments: Due to a large degree of smudging present on the gel, all colonies were chosen for future analysis (digest). [PHOTO 3 SCAN HERE]
Restriction Digest
Date: 30.07.2014
Comments: All of the selected DNA constructs (A10, A11, A12, A13, A14, A15, A16, A19, A20, A21, A22, A23 and A24) were used in a restriction digest assay using the enzyme DraI.
The protocol used was based on the instructions given by the manufacturer (NEB, DraI, see: https://www.neb.com/protocols/2012/12/07/optimizing-restriction-endonuclease-reactions). The digested DNA was resolved on an agarose gel (60min, 100V). [DIGEST PHOTO SCAN HERE]
Sequencing
Date: 31.07.2014
Comments: The constructs were prepared for sequencing as specified by ______[insert company] and using the nosT_Bseq primer (P28).
Sequences:
Spore preparation
Attempt 1:
Date: 30.07.2014
Comments: Standard protocol was followed. Used 1 spore head per 2 spore preparations.
Agrobacteria transformation
Attempt 1:
Date: 29.07.2014
Comments: Electroporation was done on our stock of competent Agrobacteria (protocol from the Plants Dept - http://www.plantsci.cam.ac.uk/research/jillharrison/protocols/cloning/preparation-of-agrobacterium-electrocompetent.pdf/view), following the Abrobacteria Transformation protocol. All 13 constructs were transformed, including a positive and negative control. The transformed cells were then plated onto quadruple resistence plates (Kan 50mg/ml, Rif 10mg/ml, Tet 5 mg/ml, Carb 50 mg/ml) and left for 3 days.
Agrobacterial induction and Co-cultivation
Induction date: 04.08.2014
Co-cultivation start: 04.08.2014
Co-cultivation stop: 07.08.2014
Comments: The selected colonies of transformed Agrobacteria were induced in a 1/2 GB + 5% sucrose + 100 uM acetosyringone medium for 6 hours prior to co-cultivation with Marchantia sporelings. The selected colonies were:
Co-cultivation was carried on until Thursday, when the sporelings were washed with ddH2O + 100 uM cefotaximine, and plated onto double-antibiotic plates (Hygromycin and Cefotaximine).
RESULTS:
11.07.2014 - many sporelings are becoming yellow in appearance, hinting to dying due to not being transformed. Parts of individual plants appear greener, suggesting these areas may have been successfully transformed. No colour is detectable at the current moment. There also appears to be minor contamination of certain plates with a small, relatively isolated orange colony of either fungus or bacterium.
Retrieved from " http://2014.igem.org/Team:Cambridge-JIC/Chromoproteins "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
