"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Cambridge-JIC/Notebook/CL 2014 7 18
From 2014.igem.org
Home
Team
Official Team Profile
TimeLine
Project
Outreach
Marchantia
Informatics
Parts
Modeling
Notebook
Protocols
Safety
Dry Work
Technology
Attributions
Hugh
Sent abstract for Synthetic Biology article to The Triple Helix Science Magazine.
Some work on the wiki
Reading: html and biology.
Will
Extracted the DNA from the purification/check gel.
Continued study of HTML
Continued hoax website. Available here for comment (note childish logos will be removed later). PhytosystemsCambridge
Guy
The E.coli have been happily amplifying the chromoprotein plasmids (although only the aeBlue ones seemed to bother expressing any themselves as they have a bacterial promoter) and are chilling in the fridge over the weekend ready to be overnight cultured and lysed to give us a Summer's worth of chromoprotein gene DNA to play with.
I ran my first PCR with Miha to amplify up fragments for the chromoprotein constructs - we're trying five different colours, all of which we're nuclear localising and two of which we are trying unlocalised as well.
After running on a gel, the PCR had worked its magic perfectly on all but track 3... The template used apparently had several kb of stuff inserted between the two primer sites that I was unaware of and therefore did not work at all, whoops. Upon trying again with a new primer that only copied the N7 localisation sequence to Gibson in later, when I came down at the finish time it emerged that the PCR cycles were only 10% done. Lessons learnt - don't get minutes and seconds mixed up when reprogramming a PCR cycler. I stayed late refixing my mistake today :)
Excitingly, a lab in Tokyo that ran a mutant screen on a benzalacetone reductase from Rheum Palmatum (Chinese rhubarb) and found that the S338V mutant is twice as efficient as the wild type enzyme has agreed to send us DNA for both the wild type and the mutant. I'm excitedly awaiting a package of DNA from Japan.
Trang-Anh
Reading on computational promoter identification in plants, based on studies on Arabidopsis
Some more playing around with the Marchantia genome data with Gos:
what Bernardo et al have done before: predicted open reading frames with geneious (see mRNA fasta file), translated into amino acids, fed the sequences into Blast to get their names (see XML excel file)
from looking at the ORF predictions: there are too many reading ORFs of which half are 100 aa long, so Gos suggested setting 300 aa as the min length for a gene. It seems like all the ORFs start in frame 1.
still according to the mRNA file, there also seems to be, for the same frame and the same ORF: how is ths possible? Seems strange...
Miha
Ran a PCR with all of our chromoprotein genes and the plasmid backbone. The PCR products were separated on a gel. After screening, the N7 localisation sequence, attached to some of the chromoprotein constructs, was not successfully amplified. The PCR with this fragment was then repeated (Guy). In addition, I prepared a large batch of electrocompetent Agrobacteria, hopefully to suffice for the rest of the project (if it worked). The GFPLti and RFPLti AgroB transformant O/N cultures were discarded as they would likely become non-viable until Monday. This step will be repeated on Monday.
Retrieved from " http://2014.igem.org/Team:Cambridge-JIC/Notebook/CL_2014_7_18 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
