<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cambridge-JIC_Protocol"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Cambridge-JIC/Protocol">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cambridge-JIC/Protocol&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge-JIC/Protocol&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge-JIC/Protocol&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cambridge-JIC/Protocol" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cambridge-JIC/Protocol</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Cambridge iGEM 2014</TITLE></P><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-ex1-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse navbar-right navbar-ex1-collapse"><UL class="nav navbar-nav"><LI><A href="https://2014.igem.org/Team:Cambridge-JIC/Team">Team</A></LI><LI class="dropdown"><A href="https://2014.igem.org/Team:Cambridge-JIC" class="dropdown-toggle" data-toggle="dropdown">mösbi</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Project/Overview">Overview</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Project/Components">Components</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Project/Mosbiandyou">mösbi&amp;You</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Project/MGF">Growth Chamber</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Project/Vision">Vision</A></LI></UL><LI class="dropdown"><A href="https://2014.igem.org/Team:Cambridge-JIC" class="dropdown-toggle" data-toggle="dropdown">Marchantia</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Marchantia/Background">Background</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Marchantia/EnhancerTrap">Enhancer trap</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Marchantia/Promoter">Promoter hunt</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Marchantia/Codon">Codon optimisation</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Marchantia/RFC">Request For Comments</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Marchantia/Assembly">Assembly</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Guide">Guide</A></LI><LI class="dropdown"><A href="https://2014.igem.org/Team:Cambridge-JIC" class="dropdown-toggle" data-toggle="dropdown">Results</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Results">Labwork results</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Results/Judging">Judging criteria</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Results/Parts">Parts table</A></LI><LI class="dropdown"><A href="https://2014.igem.org/Team:Cambridge-JIC" class="dropdown-toggle" data-toggle="dropdown">Community</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Community/Sci">Scientific community</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Community/Outreach">Outreach</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Safety">Safety</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Policy">Policy</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Community/Workshop">Workshop</A></LI><LI><A href="https://2014.igem.org/Team:Cambridge-JIC/Attributions">Acknowledgements</A></LI><LI><A href="https://2014.igem.org/Team:Cambridge-JIC/ExecSummary">Executive Summary</A></LI></DIV></DIV><DIV class="footer"><DIV class="content-section-a"><DIV class="container"><DIV class="row"><DIV class="col-lg-12"><UL class="list-inline"><LI>Keep in contact with us...</LI></UL></DIV></DIV></DIV></DIV></DIV><P>

Contents: 
<OL><LI><A href="#Introduction">Introduction</A></LI><LI><A href="#PCR">Polymerase Chain Reaction (PCR) to obtain DNA fragments</A></LI><LI><A href="#Gel">Gel electrophoresis to select the correct fragments</A></LI><LI><A href="#purification">DNA purification for plasmid assembly </A></LI><LI><A href="#Gibson">Gibson Assembly </A></LI><LI><A href="#E._coli_Transformation">E. coli transformation and plating </A></LI><LI><A href="#Colony_PCR">Colony analysis and selection by colony PCR </A></LI><LI><A href="#Miniprep">Miniprep and sequencing </A></LI><LI><A href="#Marchantia_method">The protocol for Marchantia </A></LI></OL></P><H3 id="Introduction">Introduction</H3>
We transform Marchantia using agrobacteria. The agrobacteria inject synthetic plasmids into the Marchantia nuclei. A rough outline of the process:
<UL><LI>Fragments of DNA are produced from a library of existing templates (plasmids)</LI><LI>These fragments are combined to produce the artificial plasmid</LI><LI>The artificial plasmid is amplified in E. coli</LI><LI>The artificial plasmid is incorporated into agrobacterium</LI><LI>The agrobactria are used to infect Marchantia plants which are 5 days old</LI><LI>The plants are grown and analysed</LI></UL><H3 id="PCR">PCR</H3><P>
In this step we will amplify the fragments required for our plasmids with 
Phusion DNA polymerase. PCR will amplify a fragment from a DNA 
template, with the help of short DNA sequences called primers. Primers are oligonucleotides which can bind onto a template by complementary base pairing. Two primers are used to demarcate the two ends of the fragment of interest on the template. The sequence of each primer is designed to bind only to the correct site on the template and nowhere else.
</P><FIGURE><FIGCAPTION>Figure 1: The target synthetic plasmid which contains the gene we want to put into Marchantia. The sequence above the left and right borders is the part which will affect the behaviour of the plant (hopefully).</FIGCAPTION></FIGURE><P>
It is quicker and less error prone to PCR short fragments (&lt;5kb). All the fragments are designed to overlap with each other by 20-40 bp. This allows the fragments to be joined together into a complete plasmid using the Gibson Assembly technique. 
</P><H4>PCR Protocol</H4><OL><LI> For each fragment: Add the template DNA (1µl) and primers (2.5µl at working concentration x10) to a labelled PCR tube </LI><LI> Create the phusion mix: the recipe for four fragments: in a 1.5 ml eppendorf: 
</LI><LI> HPLC H20 162.5 µl </LI><LI> 5x HF buffer 50 µl </LI><LI> 10mM dNTPs 5 µl </LI><LI> Phusion polymerase 2.5 µl </LI><LI> Shake or centrifuge the PCR tubes to ensure all DNA is at the bottom of the tube.</LI><LI> Add 44 µl of the Phusion mix to each PCR tube. </LI><LI> Without delay, place the PCR tubes into a PCR machine and set the Phusion protocol running: 
</LI><LI> 30 sec of 98°C (initial denaturation) </LI><LI> 30 cycles of:
</LI><LI> 10 sec of 98°C (denaturation) </LI><LI> 20 sec of 58°C (annealing) </LI><LI> 2:00 mins of 72°C (extension) - note the length of this step should be 30 seconds for each kb of the longest fragment </LI><LI> 5 mins of 72°C (final extension) </LI><LI> Hold at 4°C </LI></OL>
 
The PCR process takes ~2 hours, and after completion the PCR tubes can be held at 4°C 
without problems for many hours. 

<H3 id="Gel">Gel Electrophoresis</H3><P>The PCR tubes now contain our fragments, as well as template and 
primer DNA which must be removed. The fragments are extracted by running the PCR tube mixture through an agarose gel. 
A voltage applied to the gel causes the negatively 
charged DNA to migrate. Larger fragments migrate more slowly, and thus 
DNA molecules will be separated according to size. The size of the fragments can be measured by comparison with a calibration ladder, which produces a characteristic pattern of bands corresponding to known lengths. </P><P>First, we should make the gel,(100 ml 1% (w/v) agarose TAE gel):
<OL><LI> Weigh out 1g Ultrapure agarose, and add it to a glass flask </LI><LI> Add 100 ml 1xTAE </LI><LI> Microwave for 2:30 mins, swirling the flask after 1:30 mins </LI><LI> Place the glass flask in a 55°C hybridizer to cool for 20-30 mins (the gel must not be poured when it is too hot) </LI><LI> Pour the gel into two 50 ml falcon tubes </LI><LI> In a dedicated gel area, add 5 µl of Sybr Safe DNA dye to each falcon </LI><LI> Seal the open sides of the gel mould with masking tape</LI><LI> Pour the gel from both falcon tubes into the gel mould and insert an 8 tooth comb</LI><LI> Leave the gel to set for 30 mins </LI><LI> Take the PCR tubes which contain the DNA fragments. Add 12.5 µl of 5x loading dye to each PCR 
tube </LI><LI> Remove the comb and walls from the set gel, and place into a gel tank. Make sure there is enough TAE to cover the gel </LI><LI> Load the gel lanes:
</LI><LI> Put Hyperladder 1kb in lane 1</LI><LI> Fill the remaining lanes with the contents of the PCR tubes. One lane per tube. Make a note.</LI><LI> Run at 100V for 40 mins </LI></OL></P><H3 id="purification">DNA purification </H3><P>
We will now recover our fragments from the gel. First, label an eppendorf for each fragment and place a gel cutting tip in each tube. Take the finished gel to a dark room along with your tubes, gel 
cutting tips and a P1000. For each fragment, use the blue light illuminator to locate the 
appropriately sized DNA band and extract it using the cutting pipette, place the 
gel fragment into the appropriate eppendorf. Take a picture of the gel in the imaging box (The imaging box 
uses UV light, which can damage DNA, so do this after you cut out your band). 
</P><P>
Once the gel fragment is in your tube, take it back to the lab and purify 
the DNA using the Qiagen Minelute kit. A protocol is provided. 
</P><H3 id="Gibson">Gibson Assembly</H3><P>
We combine our DNA fragments into the target synthetic plasmid using Gibson Assembly. 
The Gibson Assembly reaction is an isothermal one-pot reaction 
in which three enzymes convert the linear DNA 
fragments from PCR into circular DNA. (For more information see the 
synbio.org guide, or Gibson et al., Nature Methods, 2009) 
</P><OL><LI> Pipette 0.5 µl of each DNA fragment into a PCR tube (make sure all the liquid is in the bottom of the tube) </LI><LI> Start the PCR machine running a Gibson protocol (50°C for 1 hour, 
then hold at 4°C) </LI><LI> Add 3n µl of 1.33x Gibson Master Mix to the tube (where n is the total volume of the fragments </LI><LI> Place immediately into a PCR machine </LI></OL><P>
Note that it is important to place the tube at 50°C very quickly after 
adding the master mix. If you wait too long (more than 20-30 secs), the 
exonuclease enzyme (which does not work well at 50°C) will degrade too 
much of the DNA, lowering the efficiency of the reaction significantly.
</P><P>
Second Note: a PCR tube with water in place of the Master Mix can be used as a negative control. (It should contain 0.5 µl of each DNA fragment and 3n µl of water).

</P><H3 id="E._coli_Transformation">E. coli Transformation </H3><P>
The target synthetic plasmid can be amplified by inserting it into E. coli</P><OL><LI> Get a tube of 50 µl E. coli chemically competent cells from the -80°C 
freezer, and place it on ice </LI><LI> Once the cells have thawed, place the whole Gibson reaction into 
the competent cells tube, and return the mixture to ice for 15-20 
minutes </LI><LI> Place the mixture into a 42°C water bath for 1 minute </LI><LI> Return the tube to the ice for another minute </LI><LI> Add 250 µl of SOC </LI><LI> Place into a 37°C shaking incubator for 1 hour for recovery, giving 
the cells time to express the antibiotic resistance. Also place a 9mm 
petri dish containing LB agar and the appropriate antibiotic 
(kanamycin 50 in our case) in the incubator with the cells to warm 
up. </LI><LI> Plate the entire 300 µl onto LB agar plates, spreading the liquid 
over the plate with the L-shaped spreader </LI><LI> Incubate the plate overnight at 37°C </LI></OL><P>
Note that the competent cells are very fragile, and must be kept on ice 
before the heat shock step. Also, do not to touch the bottom of the 
competent cell tubes with your hands, as this may warm them too 
much. Also, when adding the results of the Gibson reaction, do not 
pipette the cells up and down. 
</P><H3 id="Colony_PCR">Colony PCR </H3><P>
Now that we (hopefully) have colonies containing plasmids, we need 
to choose a colony to grow to obtain the correct plasmid for further 
transformations into Agrobacterium and Marchantia. 
Misassembly is possible with Gibson, particularly if there are repeated 
sequences present or if the plasmid is somewhat toxic to the bacteria 
(which should not be a problem in our case). 
</P><P>
We will now verify that the plasmid is correctly assembled by PCR. We 
will amplify a fragment across two Gibson junctions, to confirm that 
our insert has been correctly incorporated. To do this, we will use the 
35s_seq and nosT_Bseq primers, which bind as shown on the diagram 
below. We can also use these primers for sequencing later. 
</P>

!!Put in picture of the plasmid segment !!

<H4> Colony PCR Protocol: </H4><OL><LI> Identify 8 colonies to screen, and label them (scale the reaction 
down if you have fewer than 8 colonies) </LI><LI> Make a mix with the phusion and primers in an eppendorf tube: 
</LI><LI>HPLC H20 134 µl </LI><LI>5x HF buffer 40 µl </LI><LI>10mM dNTP 4 µl </LI><LI>10µM FWD primer 10 µl </LI><LI>10µM REV primer 10 µl </LI><LI>TAQ polymerase 2 µl </LI><LI> Prepare 9 small PCR tubes, and pipette 20 µl of the phusion 
mixture into each tube </LI><LI> With a small tip, touch a colony, and mix into the appropriate 
PCR tube. Note: do not pick up too much bacteria, as it will 
disrupt the PCR, a very small amount of DNA can act as a 
template  10 </LI><LI> Onto the last PCR tube, place 1 µl of the 35s_RLTI template DNA 
to act as a positive control </LI><LI> Place into a PCR machine and run a Phusion protocol, with an 
additional lysing step: 
</LI><LI> 6 mins of 98°C (lysing and denaturation) </LI><LI> 30 cycles of:
</LI><LI> 10 sec of 98°C (denaturation) </LI><LI> 20 sec of 58°C (annealing) </LI><LI> 1:00 mins of 72°C (extension) </LI><LI> 5 mins of 72°C (final extension) </LI><LI> Hold at 4°C </LI><LI> Prepare a gel whilst you wait (1% agarose (w/v) in TAE = 1g of 
UltraPure agarose in 100 ml of TAE), using the thinner toothed 
combs in the mould </LI><LI> Place the PCR products into the gel lanes (with 5 µl of loading 
dye), as well as Hyperladder 1kb in each row </LI><LI> Run for 40 mins at 100V </LI><LI> Image the gel </LI></OL><P>
The correct band size depends on the construct is being built, but they 
should be around 1-1.2 kb. Select a colony with a correct band, and place 
with an inoculation loop into an unskirted falcon containing 15 ml of LB + 
kanamycin 50µg/ml. Leave overnight in a 37°C shaking incubator. 
</P><H3 id="Miniprep">Miniprep </H3><P>
We will now harvest the plasmids from the E. coli for further use. We will 
do this using the Qiaprep kit, so follow the protocol provided in the kit. 
Colony PCR can only tell us if the fragment is the correct length, which is 
indicative of a correct assembly (and only in the junctions we are PCRing 
over). Once we have our plasmids, we should sequence them to confirm 
that their sequences are correct. To do this, obtain an account with 
Source Biosciences, and send them the appropriate amount of DNA and 
primers for Sanger sequencing. 
</P><P>
After receiving the sequencing data, we will (hopefully) have a newly 
constructed DNA plasmid with the sequence we wanted. 
Congratulations, you have now built a new plasmid! 
</P><H3 id="Marchantia_method"> The protocol for Marchantia:</H3><DL><DT>Day 1 (need primers &amp; template DNA)</DT><DD><UL><LI>PCR (3 hours)</LI><LI>Gel purification (2 hours)</LI><LI>Gibson (1.5 hours)</LI><LI>E. coli transformation (1.5 hours then overnight)</LI></UL></DD><DT>Day 2 &amp; 3</DT><DD><UL><LI>Select E. coli</LI><LI>50ml culture</LI><LI>Plasmid extraction</LI><LI>Marchantia spore sterilisation &amp; plating</LI></UL></DD><DT>Day 4</DT><DD><UL><LI>Agrobacterium transformation</LI></UL></DD><DT>Day 7</DT><DD><UL><LI>Pick Agrobacterium and grow overnight</LI></UL></DD><DT>Day 8</DT><DD><UL><LI>Transform Marchantia</LI></UL></DD><DT>Day 10</DT><DD><UL><LI>Plate Marchantia transformants</LI></UL></DD><DT>Day 13+</DT><DD><UL><LI>Results</LI></UL></DD></DL><DIV align="center"><A href="https://2014.igem.org/wiki/index.php?title=Team:Cambridge-JIC/Protocol&amp;action=edit">Edit this page</A></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Cambridge-JIC/Protocol">http://2014.igem.org/Team:Cambridge-JIC/Protocol</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cambridge-JIC/Protocol" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cambridge-JIC/Protocol" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cambridge-JIC/Protocol&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cambridge-JIC/Protocol&amp;oldid=110358" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>