<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cambridge-JIC_Results"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Cambridge-JIC/Results">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cambridge-JIC/Results&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge-JIC/Results&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge-JIC/Results&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cambridge-JIC/Results" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cambridge-JIC/Results</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Cambridge iGEM 2014</TITLE></P><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-ex1-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse navbar-right navbar-ex1-collapse"><UL class="nav navbar-nav"><LI><A href="https://2014.igem.org/Team:Cambridge-JIC/Team">Team</A></LI><LI class="dropdown"><A href="https://2014.igem.org/Team:Cambridge-JIC" class="dropdown-toggle" data-toggle="dropdown">mösbi</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Project/Overview">Overview</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Project/Components">Components</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Project/Mosbiandyou">mösbi&amp;You</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Project/MGF">Growth Chamber</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Project/Vision">Vision</A></LI></UL><LI class="dropdown"><A href="https://2014.igem.org/Team:Cambridge-JIC" class="dropdown-toggle" data-toggle="dropdown">Marchantia</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Marchantia/Background">Background</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Marchantia/EnhancerTrap">Enhancer trap</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Marchantia/Promoter">Promoter hunt</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Marchantia/Codon">Codon optimisation</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Marchantia/RFC">Request For Comments</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Marchantia/Assembly">Assembly</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Guide">Guide</A></LI><LI class="dropdown"><A href="https://2014.igem.org/Team:Cambridge-JIC" class="dropdown-toggle" data-toggle="dropdown">Results</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Results">Labwork results</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Results/Judging">Judging criteria</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Results/Parts">Parts table</A></LI><LI class="dropdown"><A href="https://2014.igem.org/Team:Cambridge-JIC" class="dropdown-toggle" data-toggle="dropdown">Community</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Community/Sci">Scientific community</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Community/Outreach">Outreach</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Safety">Safety</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Policy">Policy</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Community/Workshop">Workshop</A></LI><LI><A href="https://2014.igem.org/Team:Cambridge-JIC/Attributions">Acknowledgements</A></LI><LI><A href="https://2014.igem.org/Team:Cambridge-JIC/ExecSummary">Executive Summary</A></LI></DIV></DIV><DIV class="footer"><DIV class="content-section-a"><DIV class="container"><DIV class="row"><DIV class="col-lg-12"><UL class="list-inline"><LI>Keep in contact with us...</LI></UL></DIV></DIV></DIV></DIV></DIV><H2>Results and other experiments</H2><H3 id="Chromoproteins">Chromoproteins </H3><P>
As an example output plugin we transformed into <I>Marchantia polymorpha</I> a selection of the chromoproteins brought to iGEM by Uppsala 2011. This was to test whether a simple change in colour would work as an easily visible reporter.
</P><P>
Five chromoproteins were selected for expression: eforred (BBa_K592012), tspurple (BBa_K1033906),  aspink (BBa_K1033927), aeblue (BBa_K1033929) and amilCP(BBa_K1033930, a deep blue colour). The N7 nuclear localisation tag (BBa_K1484104) was added to tspurple, aspink and amilCP. Each was put into a binary vector called pGreen and transformed first into agrobacterium, then transferred to marchantia.
</P><P>
The transformation procedure yielded a large number of transformants, approximately 100 per construct, which were confirmed via PCR using homogenised plants as a template and chromoprotein-specific primers for each tDNA added.
</P><P>Left: A PCR of the transformed plants showing successful transformation. Right: A plate of transformants growing happily on hygromycin</P><P>For each chromoprotein construct, the transformation yielded approximately 100 transformed sporelings (7-days old plants). The petri dish to the right contains hygromycin as a selective marker. The gel confirms the transformation in three plants each from two other plates.  The forward and reverse primers used in the PCR anneal to 35S and N7 respectively, and the template was homogenised plant material. </P><P> After approximately four weeks of growth, sparsely distributed bright red cells became apparent under the microscope in the plants transformed with the tsPurple gene. Although similar cells were not found in wild type plants, diffuse red regions were found and are probably due to release of anthocyanins by stressed or aging cells. </P><P align="center">
The bright red cells seen in Marchantia transformed with the TsPurple construct
</P><P>It is notable that in the red cells, the vacuole is not visually distinct from the cytoplasm. As GFP, a protein of similar structure, is not accumulated in the vacuole without specific targeting, this implies that either the colour we observe in the bright cells is not due to chromoprotein expression, or perhaps that tsPurple is toxic enough to cause cellular damage and leaky organelles. </P><P>Plants transformed with asPink (which is also known as asFP595 due to its fluorescence), nuclear localised with an N7 tag, while not visibly pigmented, did have areas that fluoresced at the literature wavelengths.</P>
Part of an asPink::N7 transformant visualised using a fluorescence microscope with under a GFP filter (this particular filter gives a yellow signal for reddish fluorescent proteins. signal is present in the RFP filter, but difficult to image due to chlorophyll fluorescence)  one with wavelength windows that intersect those of asPink's absorption and emission peaks)

Taking a confocal stack of a small portion of the plants, we observed the following. 
<FIGURE><FIGCAPTION>Left: asPink-N7 transformed plant. Right: GFP-LTI transformed plant. The laser excitation wavelength was 595nm. The red channel corresponds to 597-614nm, and the blue channel corresponds to chlorophyll autofluorescence at 647nm+</FIGCAPTION></FIGURE>

These images suggest that what we observed was indeed expression of asPink.

We also did absorbance assays of the chromoproteins in E. coli for comparison.
<H3 id="Raspberry ketone production">Raspberry ketone production </H3><P>As another example output module, the biosynthetic pathway from coumaroyl-coA to raspberry ketone was assembled, to allow the smell of raspberries to be used as an odour signal that the input module has been activated.</P><P> To characterise the enzymes involved, a bacterial constructs was made, and the E. Coli transformed with it we called RaspberrE coli. The construct consisted of an arabinose inducible expression plasmid coding for 4-coumarate coA ligase, benzalacetone synthase, and benzalacetone reductase, and the strain of E. Coli we used had the TnaA indole synthase gene knocked out to somewhat reduce the faecal odour associated with this molecule. (growing in M9 salts with casamino acids and no tryptophan eliminated the faecal odour entirely, as the smell is due entirely to tryptophan derivatives).</P><P>The RaspberrE coli were incubated for two and a half days with 3 or 9mM p-coumaric acid and 50mM arabinose to induce expression. The cultures were then shaken with ethyl acetate which was then run on an LCMS setup (which was only possible due to the kind giving of time and expertise by Ben Pilgrim at the Cambridge Department of Chemistry).</P><P>As a control, one of the tubes incubated contained a strain of E coli resistant to chloramphenicol but with none of the raspberry ketone producing genes. There was one peak that existed on the RaspberrE coli spectra but not on the control, which when run through the mass spectrometer turned out to be benzalacetone, the precursor to raspberry ketone.</P><P> A comparison of E.coli transformed with the benzalacetone generator (BBa_K1484317) against a control strain with none of the relevant enzymes.</P><P> None of the E. coli which also contained the benzalacetone reductase gene produced any peaks which gave mass spectrum peaks corresponding to that of raspberry ketone. These results suggest that while the 4-coumarate coA ligase and benzalacetone synthase both worked, the benzalacetone reductase, as shown once already by the Dutch research team who first cloned the enzyme’s gene sequence (led by Jules Beekwilder in Wageninen, The Netherlands), does not actually reduce benzalacetone.</P><P>A comparison of bacteria incubated with p-coumaric acid. Top: Transformed with benzalacetone generator (BBa_K1484317) Bottom: Transformed with benzalacetone generator and benzalacetone reductase (BBa_M36705) as a polycistronic operon. No raspberry ketone was detected in either culture.


</P><H3 id="Enhancer Trap">Enhancer Trap </H3><P>  Marchantia spores have successfully been transformed and are growing at the moment. Screening has just begun and we wait to see some interesting expression pattern as development occurs.</P><H3 id="Growth Facility">Growth Facility </H3><P>The Marchantia growth facility family of devices were developed, and a guide to building either one for yourself is <A href="https://2014.igem.org/Team:Cambridge-JIC/Project/MGF">available</A>  on this wiki. </P><FIGURE><P style="clear: both;"><FIGCAPTION> Left: The Marchantia Growth Facility, a device capable of controlling the light and airflow conditions to four plates simultaneously. Right: The Mini Growth Facility, a smaller version of the MGF, capable of controlling the light conditions of one plate. </FIGCAPTION></P></FIGURE><P>A transformation has not yet been attempted in the growth facility.</P><P>The MGF family of growth facilities offers a low-cost, open source and easily modifiable platform for the use of marchantia in a scientific setting, which we hope will be adopted and improved by other labs in future competitions. </P><H3 id="Hammerhead Ribozymes">Hammerhead Ribozymes </H3><P>We received four hammerhead ribozymes as a kind donation from the Smolke lab. These included  2 versions of theophylline deactivated cleavage, 1 version of theophylline activated cleavage, and 1 always cleaving ribozyme.  </P><P> Each of these  were built into two separate constructs.</P><P>The first contained the Cauliflower Mosaic Virus 35S promoter driving expression of eGFP-N7, with the hammerhead ribozymes inserted into the 3' UTR </P><P> The second contained the the Cauliflower Mosaic Virus 35S promoter driving expression of the transcription activator HAP1, with the hammerhead ribozymes inserted into the 3' UTR. This then activated the HAP1-UAS, which was driving Venus-YFP-N7.</P><P> We had trouble assembling these constructs, which we put down to poor quality competent cells (assembly became considerably faster after the creation of a new batch). Eventually, all four 35S constructs were built using Gibson assembly and sequenced. Two HAP1 constructs were built using LCR, and verified by restriction endonuclease digest, but not sequenced </P><P> The Marchantia transformation was hindered at several stages. First, agrobacteria transformation via electroporation was repeatedly unsuccessful. Second, Marchantia transformations were rapidly contaminated, leading to the death of all selected plants (prior to them growing to the stage of being able to assay them). We abandoned transformation knowing that time constraints would prevent us from successfully assaying the results. We leave the constructs, being RFC10 incompatible, for further users of Marchantia/Mösbi to try.</P><H3 id="Heat Shock Promoter">Heat Shock Promoter </H3><P>We received from Jeremy Solly a vector containing a Marchantia heat-shock promoter. 2 constructs were designed, containing this promoter driving Venus-YFP-N7, as well as an input for our system, driving GAL4, which activated the GAL4-UAS driving Venus-YFP-N7. </P><P>We had trouble assembling these constructs, which we put down to poor quality competent cells (assembly became considerably more reliable after the creation of a new batch). </P><P>Finally, these constructs were built and sequenced, and Marchantia transformation attempted. However, this failed at the stage of transformation of Agrobacteria, and was eventually abandoned due to time constraints. We leave them as a legacy for future users of Marchantia/Mösbi.</P>

Some other experiments we did:
<H2 id="Overcoming the smell of E-coli by two different techniques and their comparison">Overcoming the indole smell of E-coli through two different techniques and their comparison. </H2><H4>Problem</H4><P>
E. coli cultures have a pungent odour due to the production of indole and other tryptophan derivatives. We were afraid that the strong smell of E. coli would mask any other synthesized volatiles. We therefore decided to look into ways to reduce their production of indole.
</P><H4>Summary</H4><P>
We overcame the odour issue by trying a new protocol to make normal E. coli strains produce less indole. We compared the efficiency of our technique in terms of both odour and growth rate with an indole synthase free strain of bacteria.
Our technique produced pleasant smelling E. coli, indistinguishable from indole-free bacteria. However they required 6h more to reach similar population density.
We also produced E. coli with equal growth rate as standard bacteria in standard media which smelled significantly less.
</P><H4>Story</H4><P>
1-	We located an indole free strain of E. coli from a nearby lab. This mutant strain has a non-functional version of the TnaA gene for indole synthase so can no longer metabolize tryptophan into indole.
2-	 We also learnt that normal bacteria grown on a tryptophan-free medium and minimal salts have been shown to produce insignificant amounts of indole and still grow at good rates. No data had been recorded on smell or growth rate.

</P><H4>Experiment</H4><P>
We created three flasks of growth medium: </P><P>-minimal medium [M9 salts, Mg]
-minimal medium + Casamino acids (which don't contain tryptophan)
-minimal medium + Casamino acids + tryptophan</P><P> With each medium we grew up two cultures, one with wild type E. coli, the other with the TnaA knockout strain.</P><H4>Results</H4><UL>
Density
<P>
The cultures were grown overnight. In the morning, the absorption was measured to compare population growth. All tubes were cloudy with cultures apart from the control. 
Both strains in minimal medium grew at the same rate, proving that the indole-free strain was still vigorous.
For both strains the minimal medium + casamino acids contained approximately twice the bacteria concentration as the minimal medium alone.  This was expected as the lack of tryptophan and inability to communicate population number via indole production restricts the growth of bacteria.
There was no significant difference in population concentration between the tryptophan and tryptophan free medium for the E. coli TnaA knockout strain. This is understandable as indole is used as a primitive quorum sensing molecule which allows E. coli to control its growth rate with regard to population density. With the ability to produce indole removed, tryptophan availability is no longer as big an advantage.
There was a slight increase in population for the wild type E. coli grown with tryptophan compared to those grown without it. This shows tryptophan does limit to some degree the growth of bacteria but not very much.
</P>

Odour
<P>
To compare scent given off, all cultures were grown to the same optical density. When this was achieved, the tubes were smelt blind by different people outside our own lab.
The E. coli TnaA knockout strain, when grown without tryptophan, had a pleasant odour which has been described as having 'smokey notes'  and 'reminiscent of squished grapes'. The smell is very distinct from the faecal smell produced by wild type E. coli strains grown with tryptophan, and different but not hugely so from the smell of wild type E. coli grown in the minimal medium without tryptophan.</P><P>
The wild type E. coli smelled only very mildly in the tryptophan free minimal medium but of course had the full pungent toilet smell that we expect when incubated with tryptophan. 
</P></UL><H4>Conclusion</H4><P>
1- The TnaA knockout strain grew as quickly as the standard E. coli strain, thus demonstrating its fecundity.
This strain did not produce such a pungent smell showing that indole is in large part the culprit.
2- E. coli grown on minimal medium produces drastically lower amounts of indole, skatole and other foul-smelling tryptophan derivatives, and therefore have a less pungent smell.
The culture in minimal medium took 6h longer to reach the given population density than those grown in LB.
3-We recommend using a TnaA knockout strain to achieve less smelly cultures, for even better results grow not in LB but in M9 minimal salts with casamino acids.</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Cambridge-JIC/Results">http://2014.igem.org/Team:Cambridge-JIC/Results</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cambridge-JIC/Results" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cambridge-JIC/Results" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cambridge-JIC/Results&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cambridge-JIC/Results&amp;oldid=376552" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>