"
Page
Discussion
View source
History
teams
Log in
 
Team:Cambridge-JIC/Project/Overview
From 2014.igem.org
Cambridge iGEM 2014
Toggle navigation
Team
mösbi
Overview
Components
mösbi&You
Growth Chamber
Vision
Marchantia
Background
Enhancer trap
Promoter hunt
Codon optimisation
Request For Comments
Assembly
Guide
Results
Labwork results
Judging criteria
Parts table
Community
Scientific community
Outreach
Safety
Policy
Workshop
Acknowledgements
Executive Summary
Keep in contact with us...
mösbi
We share the belief that synthetic biology will play an important role in the future – from drug development and the creation of smarter biosensors, to agriculture, the food industry and one day space exploration.
However – synthetic biology is at the moment almost exclusively restricted to academic institutions and specialised industries. Our team wants to change this by bringing synthetic biology  to your living room!
mӧsbi stands for marchantia open-source biosensor. The name encapsulates the team's vision of creating a user-friendly, open-source biosensor using the liverwort Marchantia polymorpha.
Overview
THE CHASSIS
Marchantia polymorpha represents the perfect choice in the endeavour of popularising synthetic biology and making it accessible to a larger audience. There are several reasons for this:
Marchantia polymorpha is a plant.
Since plants play a role in every culture, people are comforable around them – unlike bacteria. We realised that if we wanted to push forward the idea of a biosensor accessible to everyone it would have to be intrinsically user-friendly. What better than a small, low-maintenance, thalloid plant with tiny palm tree-like structures?
It has a small haploid genome which can be easily manipulated.
Marchantia's genome size is only 280 Mb, which is little compared to other plants comonly used in biology. A haploid genome allows the user to disregard factors such as gene dominance relationships on genetically modifying the plant. With upcoming advances in mapping the genome, Marchantia represents an ideal candidate for an open-source platform.
Marchantia is dioecious and has a short life cycle.
Being dioecious, Marchantia exists as either male or female plants, which are easily distinguishable at adult stages through the shape of their gametophyte. This in turn simplifies the crossing of two plants to an inexperienced user. Compared to the majority of other plants, the Marchantia polymorpha life cycle is short. It takes less than 3 months for a Marchantia plant to grow from a spore into an adult plant capable of reproduction. It is no match to E. coli but is surely beats other plants used in synthetic biology.
A single plant can produce hundreds of thousands of spores.
The spores are very hardy, allowing them to be kept for over a year in a dry container. In addition, the hardiness allows for spores to be easily shipped without worrying about handling and temperature swings. Upon adding water to the spores, germination begins and the tiny plant begins to develop. The sheer amount of spores produced by a single plant ensures*the presence of the desired offspring in the progeny.
Marchantia biosensors are renewable.
The plants are easy to vegetatively propagate – either by grafting or via gemmae, tiny clones of the plants sporadically generated in gemma cups on the plant's surface. This ensures once a useful biosensor is generated for it to be easily distributed to your friends and family.
THE FRAMEWORK
Our team's efforts over the summer have been focused on developing a plug-and-play style framework which allows the end user to easily design custom mӧsbi biosensors without requiring any specialist scientific knowledge. This framework represents the core concept behind mӧsbi.
The mӧsbi framework is composed of 3 modules – input, processing and output. A combination of any modules from each of these 3 categories enables the formation of a unique mӧsbi biosensor.
To ensure a theoretically unlimited level of mix-and-match is possible we developed a sturdy and universal framework design based around two transcription factors – GAL4 and HAP1.
Figure: Illustration of the idea behind mӧsbi's modularity
GAL4 represents the transcription factor relaying the information between the input and processing modules. GAL4 is a yeast transcription factor commonly used in eukaryote gentic engineering. Gal4 has been previously shown to work in Marchantia polymorpha within the Haseloff lab which convinced us in using it as part of the mӧsbi framework.  HAP1 is also a transcription factor that has been highly speculated to work in our chassis. HAP1 is in charge of transmitting the information between the processing and output modules.
These two transcription factors act as connectors between the input, processing and output modules. Input modules act as the sensing modules in the mӧsbi framework. A specific signal is eventually relayed to the promoter present in the input module. Upon doing so, the transcription of the first linking transcription factor GAL4 is initiated. Upon successful translation, GAL4 acts on the GAL4 UAS promoter present in the processing module thus initiating the transcription of HAP1. HAP1 acts as the second linking transcription factor, relaying information between the processing and the output modules. HAP1 acts on the HAP1 UAS promoter of the output, commencing the transcription of the output gene – which are selected to be easily detected by the user.
The universally adopted mӧsbi framework design allows compatibility between existing and future mӧsbi modules. For the average user this means no worries associated with compatibility between modules since they all operate in the same fashion. On the other hand more adventurous developers can rely on a sturdy and universal framework when designing new modules.
Figure: Diagram of the mӧsbi framework design.
ASSEMBLING THE MODULES
Assembling a mӧsbi biosensor is arguably the simplest stage in creating bespoke biosensors. We have taken advantage of an innate ability of Marchantia polymorpha which allows us to generate new mӧsbis – good old fashioned crossing.
By crossing plant lines with different single modules or module pairs (i.e. input-processor) – the so called seed lines - a proportion of the progeny will inevitably carry both parents' modules – in this way forming a partially or fully-assembled mӧsbi biosensor.
The crossing itself is an easy task which can be performed by anyone – including a child. One needs only to use a brush, dip it into water and brush the archegoniophores of a male and female plants from pre-selected seed line plants. This can be repeated several times and between different lines, giving rise to multiple mӧsbi variations from relatively few seed lines.
Selecting the recombinant progeny is done using pre-supplied (or independently created) antibiotic resistance plates. All mӧsbi modules are to be designed possessing specific antibiotic resistance genes in each module. After the cross is completed and progeny spores are produced, these are incubated on the antibiotic plates. Only mӧsbis with 2 (in case of a 2-step cross) or all 3 modules will be able to grow on a given plate. In this way only the mӧsbi biosensors of interest are selected.
The assembly time varies depending if a 1 or 2-step assembly process is chosen by the user. A 1-step assembly process, starting with growing the seed lines from spores and finishing with the assembled mӧsbi biosensors growing on a selective plate will likely take approximately 2 months. Assembling a mӧsbi biosensor is certainly not an overnight task, however its simplicity and low equipment and knowledge requirements make it an interesting product for a new audience.
Retrieved from " http://2014.igem.org/Team:Cambridge-JIC/Project/Overview "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
