"
Page
Discussion
View source
History
teams
Log in
 
Team:Cambridge-JIC/Informatics
From 2014.igem.org
Cambridge iGEM 2014
Toggle navigation
Team
mösbi
Overview
Components
mösbi&You
Growth Chamber
Vision
Marchantia
Background
Enhancer trap
Promoter hunt
Codon optimisation
Request For Comments
Assembly
Guide
Results
Labwork results
Judging criteria
Parts table
Community
Scientific community
Outreach
Safety
Policy
Workshop
Acknowledgements
Executive Summary
Keep in contact with us...
Edit this page
Informatics
To introduce and facilitate future use of the novel chassis Marchantia polymorpha, we computationally analysed its genome to:
find out the most efficient codon usage in order to optimise our Marchantia specific registry parts and facilitate all future synthetic biology work on Marchantia;
submit a small library of Marchantia promoters to the iGEM registry, in particular looking for those which are strong, inducible, tissue-specific or expressed in an early development stage.
Codon usage optimisation
Our start point was the Marchantia genome and the mRNA transcriptome predicted with Geneious software ( http://www.geneious.com/ ). The data was given to us from Jim's lab by Bernardo. 99 000 ORFs were predicted, which seems too large to be a realistic number of genes expressed by this small liverwort. Half of these were only 100 amino acids long. We set the threshold for candidate genes among these at 300 amino acids, obtaining the expected normal distribution of lengths.
Using BLAST software ( http://blast.ncbi.nlm.nih.gov/Blast.cgi ), we compared the proteins coded by these candidate genes to the proteins present in Arabidopsis thaliana, given on Araport ( www.araport.org ). Some of the sequences showed incomplete matches, indicated that our predicted ORFs should be regarded with some vigilance. The longest sequence showed a 40% match, a small number as expected.
The relevant DNA sequences from the candidate mRNAs using the BLAST output. Then the different codons were counted in these genes, in order to obtain a codon table for Marchantia. While the table is not strikingly similar to Arabidopsis', we can note a similarity in the slight preference for C over other bases at the end of codons and that for G-p-C sites.
A .xls version of this table with relevant filters can be found here .
Promoter identification
To identify potential promoter regions in Marchantia's genome, we looked for gene sequences from other species, mainly Arabidopsis thaliana and Physcomitrella patens. We were particularly interested in genes that were:
Nitrate inducible
Sulphate inducible
Phosphate inducible
Circadian rythm driven
Metabolism related
Development related
Light responsive
Development
Standard templates for proteins
Standard templates for genes
Reading recent research papers in search for these genes and looking for their sequences on GenBank and ThaleMine, we constructed a list of protein sequences to compare to the Marchantia predicted scaffolds. (For genes where only the nucleotide sequence was available, instead of the protein sequence, a C++ code was written to perform the translation). We ran a tblastn search in BLAST for the best matches between our candidate proteins and the Marchantia scaffolds. Then, for the best matches (~60% and above, with some judging by eye), we annotated the 2kb upstream of the start codon as a potential promoter region.
Example of a BLAST hit, matching an inducible nitrate transporter sequence to a Marchantia gene
We identified 30 candidate promoters this way, that we are planning to screen by inserting in a construct driving the yellow fluorescent protein Venus. For each promoter, we will make a construct with and one without amplification by GAL4 and GAL4 UAS, to evaluate the promoter strength and get around any leakages due to the use of GAL4.
Edit this page
Retrieved from " http://2014.igem.org/Team:Cambridge-JIC/Informatics "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
