<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cambridge-JIC_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Cambridge-JIC/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cambridge-JIC/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge-JIC/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge-JIC/Project&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cambridge-JIC/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cambridge-JIC/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Cambridge iGEM 2014</TITLE></P><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-ex1-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse navbar-right navbar-ex1-collapse"><UL class="nav navbar-nav"><LI><A href="https://2014.igem.org/Team:Cambridge-JIC/Team">Team</A></LI><LI class="dropdown"><A href="https://2014.igem.org/Team:Cambridge-JIC" class="dropdown-toggle" data-toggle="dropdown">mösbi</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Project/Overview">Overview</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Project/Components">Components</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Project/Mosbiandyou">mösbi&amp;You</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Project/MGF">Growth Chamber</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Project/Vision">Vision</A></LI></UL><LI class="dropdown"><A href="https://2014.igem.org/Team:Cambridge-JIC" class="dropdown-toggle" data-toggle="dropdown">Marchantia</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Marchantia/Background">Background</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Marchantia/EnhancerTrap">Enhancer trap</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Marchantia/Promoter">Promoter hunt</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Marchantia/Codon">Codon optimisation</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Marchantia/RFC">Request For Comments</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Marchantia/Assembly">Assembly</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Guide">Guide</A></LI><LI class="dropdown"><A href="https://2014.igem.org/Team:Cambridge-JIC" class="dropdown-toggle" data-toggle="dropdown">Results</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Results">Labwork results</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Results/Judging">Judging criteria</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Results/Parts">Parts table</A></LI><LI class="dropdown"><A href="https://2014.igem.org/Team:Cambridge-JIC" class="dropdown-toggle" data-toggle="dropdown">Community</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Community/Sci">Scientific community</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Community/Outreach">Outreach</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Safety">Safety</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Policy">Policy</A></LI><LI role="presentation"><A role="menuitem" href="https://2014.igem.org/Team:Cambridge-JIC/Community/Workshop">Workshop</A></LI><LI><A href="https://2014.igem.org/Team:Cambridge-JIC/Attributions">Acknowledgements</A></LI><LI><A href="https://2014.igem.org/Team:Cambridge-JIC/ExecSummary">Executive Summary</A></LI></DIV></DIV><DIV class="footer"><DIV class="content-section-a"><DIV class="container"><DIV class="row"><DIV class="col-lg-12"><UL class="list-inline"><LI>Keep in contact with us...</LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="intro-header-proj"><DIV class="container"><DIV class="row"><DIV class="col-lg-9"><DIV class="intro-message"><H1>mösbi</H1><FONT color="black" style="BACKGROUND-COLOR: #E6E6E6">We share the belief synthetic biology will play an important role in the future – from drug development and the creation of smarter biosensors, to agriculture, the food industry and inevitably space exploration. 
However – synthetic biology is at the moment almost exclusively restricted to academic institutions and specific industries. Our team wants to change this by bringing synthetic biology  to your living room!<B>mӧsbi</B> stands for <B>m</B>archantia <B>o</B>pen-<B>s</B>ource <B>biosensor. The name encapsulates the team's vision of creating a user-friendly, open-source biosensor using the liverwort Marchantia polymorpha. 
</B></FONT></DIV></DIV></DIV></DIV></DIV><DIV class="content-section-a" id="Overview"><DIV class="container"><DIV class="row"><DIV class="col-lg-9 col-sm-push-0.75  col-sm-6"><H2 class="section-heading">Overview</H2><DIV><H4>THE CHASSIS</H4><P>
Marchantia polymorpha represents the perfect choice in the endeavour of popularising synthetic biology and making it accessible to a larger audience. There are several reasons for this:<OL><LI><I>Marchantia polymorpha is a plant.</I> Since plants play a role in every culture, people are comfortable around them – unlike bacteria. We realised that if we wanted to push forward the idea of a biosensor accessible to everyone it would have to be intrinsically user-friendly. What better than a small, low-maintenance, thalloid plant with tiny palm tree-like structures?</LI><LI><I>It has a small haploid genome which can be easily manipulated.</I>Marchantia's genome size is only 280 Mb, which is little compared to other plants commonly used in biology. A haploid genome allows the user to disregard factors such as gene dominance relationships while genetically modifying the plant. With upcoming advances in mapping the genome, marchantia represents an ideal candidate for an open-source platform.</LI><LI><I>Marchantia is dioecious and has a short life cycle.</I>Being dioecious, marchantia exists as either male or female plants, which are easily distinguishable at adult stages through the shape of their gametophyte. This in turn simplifies the crossing of two plants to an inexperienced user. Compared to the majority of other plants, the <I>Marchantia polymorpha</I> life cycle is short. It takes less than 3 months for a marchantia plant to grow from a spore into an adult plant capable of reproduction. It is no match for E. coli but surely beats other plants used in synthetic biology.</LI><LI><I>A single plant can produce hundreds of thousands of spores.</I>The spores are very hardy, allowing them to be kept for over a year in a dry container. In addition, the hardiness allows for spores to be easily shipped without worrying about handling and temperature swings. Upon adding water to the spores, germination begins and the tiny plant begins to develop. The sheer number of spores produced by a single plant ensures*the presence of the desired offspring in the progeny.</LI><LI><I>Marchantia biosensors are renewable.</I>The plants are easy to vegetatively propagate – either by grafting or via gemmae, tiny clones of the plants sporadically generated in gemma cups on the plant's surface. This ensures that once a useful biosensor is generated it can be easily distributed to your friends and family.</LI></OL></P></DIV></DIV></DIV></DIV></DIV><DIV class="content-section-b"><DIV class="container"><DIV class="row"><DIV class="col-lg-9 col-sm-push-0.75  col-sm-6"><DIV><H4>THE FRAMEWORK</H4><P>
Our team's efforts over the summer have been focussed on developing a plug-and-play style framework which allows the end user to easily design custom mӧsbi biosensors without requiring any specialist scientific knowledge. This framework represents the core concept behind mӧsbi.
The mӧsbi framework is composed of <B>3 modules – input, processing and output</B>. A combination of any modules from each of these 3 categories enables the formation of a unique mӧsbi biosensor. 
To ensure a theoretically unlimited number of mix-and-match combinations is possible we developed a sturdy and universal framework design based around two transcription factors – <B>GAL4</B> and <B>HAP1</B>.</P><FIGURE><FIGCAPTION>Figure 1: Illustration of the idea behind mӧsbi's modularity</FIGCAPTION></FIGURE><P>
GAL4 represents the transcription factor relaying the information between the input and processing modules. GAL4 is a yeast transcription factor commonly used in eukaryote genetic engineering. GAL4 has been previously shown to work in <I>Marchantia polymorpha </I>within the Haseloff lab which made using it as part of the mӧsbi framework look like a good option. 
HAP1 is also a transcription factor which has been greatly expected to work in our chassis. HAP1 is in charge of transmitting the information between the processing and output modules. 
These two <B>transcription factors act as connectors</B> between the input, processing and output modules. Input modules act as the sensing modules in the mӧsbi framework. A specific signal is eventually relayed to the promoter present in the input module. Upon doing so, the transcription of the first linking transcription factor GAL4 is initiated. Upon successful translation, GAL4 acts on the GAL4 Upstream Activation Sequence (UAS) promoter present in the processing module thus initiating the transcription of HAP1. HAP1 acts as the second linking transcription factor, relaying information between the processing and the output modules. HAP1 acts on the HAP1 UAS promoter of the output, commencing the transcription of the output gene – which are selected to be easily detected by the user.
The universally adopted mӧsbi framework design allows compatibility between existing and future mӧsbi modules. For the average user this means no worries associated with compatibility between modules since they all operate in the same fashion. On the other hand more adventurous developers can rely on a sturdy and universal framework when designing new modules.
</P><FIGURE><FIGCAPTION>Figure 1: Illustration of the idea behind mӧsbi's modularity</FIGCAPTION></FIGURE></DIV></DIV></DIV></DIV></DIV><DIV class="content-section-a"><DIV class="container"><DIV class="row"><DIV class="col-lg-9 col-sm-push-0.75  col-sm-6"><DIV><H4>ASSEMBLING THE MODULES</H4><P>
Assembling a mӧsbi biosensor is arguably the simplest stage in creating bespoke biosensors. We have taken advantage of an innate ability of <I>Marchantia polymorpha</I> which allows us to generate new mӧsbis – good old fashioned crossing.
By crossing plant lines with different single modules or module pairs (i.e. input-processor) – the so called <B>seed lines</B> - a proportion of the progeny will inevitably carry both parents' modules – in this way forming a partially or fully-assembled mӧsbi biosensor.
The crossing itself is an easy task which can be performed by anyone – even a child. One needs only to take a brush, dip it into water, and brush first the antheridium of a male plant, then the archegonium of a female, from pre-selected seed lines. This can be repeated several times and between different lines, giving rise to multiple mӧsbi variations from relatively few seed lines.
Selecting the recombinant progeny is done using pre-supplied (or independently created) antibiotic resistance plates for now. All mӧsbi modules are to be designed possessing specific antibiotic resistance genes in each module, or once more have been identified, different auxotrophic mutations. After the cross is completed and progeny spores are produced, these are either germinated on the three-antibiotic plates, or in the future auxotrophic system, a regular nutrient plate which will select for only those plants with complementary mutations. Only mӧsbis with 2 (in case of a 2-step cross) or all 3 modules will be able to grow on a given plate. In this way only the mӧsbi biosensors of interest are selected.

The assembly time varies depending if a 1 or 2-step assembly process is chosen by the user. A <B>1-step assembly</B> process, starting with growing the seed lines from spores and finishing with the assembled mӧsbi biosensors growing on a selective plate will likely take <B>approximately 2 months</B>. Assembling a mӧsbi biosensor is certainly not an overnight task, however its simplicity and low equipment and knowledge requirements make it an interesting product for a new audience.</P></DIV></DIV></DIV></DIV></DIV><DIV class="content-section-b" id="Vision"><DIV class="container"><DIV class="row"><DIV class="col-lg-9 col-sm-push-0.75  col-sm-6"><H2 class="section-heading">Vision</H2><DIV><H4>Our Vision</H4><DIV><P>We envisioned mösbi as a new approach to popularise synthetic biology – both within the scope of the iGEM competition and beyond. However, as most people who have dabbled with biology know, nothing in biology ever seems to work first time. In our case the time limitations imposed prevented us on developing mösbi to its full potential. </P><P>Here we share some thoughts on future steps that would have been taken in order to complete this part of the project and our vision for this new and exciting undertaking. </P><P>As you may have noted so far, mösbi is aimed at the people to give a larger-than-usual audience. The key message about mösbi remains its unorthodox accessibility. Accessibility in terms of requiring little or no scientific knowledge to use the product; accessibility in terms of its honest and open-source design so that everyone wishing to understand its working and willing to contribute may do so; accessibility in terms of creating an affordable and powerful analytic tool for everyone to use. </P><P>Education plays an important role in our vision. We see mösbi as a herald of the current post-genomic era – bringing current technological advances directly to people who may otherwise be unaffected or indifferent towards them. Bringing synthetic biology into the living room would allow us to educate people about this area of science: Its current status, its future and their role in it. It would allow us to communicate honestly about the benefits and risks associated with genetic engineering and synthetic biology.
In addition, mösbi would represent something of a biological version of Arduino – an open-source platform to experiment and learn with, thus likely a popular choice for young science aficionados throughout primary and secondary education. </P><P>Although remaining open-source, some level of supervision over the future development of mösbi would be required. This should be carried out by a not-for-profit organisation which would keep researching new modules, reviewing user created ones and supervising the distribution of seed lines and paraphernalia required for mösbi assembly. This paraphernalia – i.e. antibiotic resistance plates, auxotroph nutrient supplemented soil, growth boxes like the one our team built could provide a partial source of income for further platform development and organisation running costs. </P><P>There is much more that would need to be done before mösbi could reach this level of impact. Firstly, output reporter genes especially chromoproteins, need to be optimised for expression in the new chassis. Development of auxotrophic lines of Marchantia represents another necessity. Tests with seed line crossing and possible framework optimisation would need to be carried out to ensure mösbi works as planned. Additional thought would need to be given about legal implications of such a product and carefully assessed. There is a large scope for improvement. The above listed undertakings will certainly become easier to achieve as the chassis develops through basic research in the upcoming years. We like to think iGEM has made and will keep making contributions on this front over the next decade. </P><P>To conclude, we hope that in a decade or so - when accessible, user-friendly biosensors are a reality, one will be able to look back on this effort and see it as a grain of sand in a larger picture that helped drive synthetic biology forward. </P><FIGURE style="float:left"><FIGCAPTION>Figure 3: Mosbi in the comfort of your own living room - artist's view</FIGCAPTION></FIGURE></DIV></DIV></DIV></DIV></DIV></DIV><DIV align="center"><A href="https://2014.igem.org/wiki/index.php?title=Team:Cambridge-JIC/Project&amp;action=edit">Edit this page</A></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Cambridge-JIC/Project">http://2014.igem.org/Team:Cambridge-JIC/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cambridge-JIC/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cambridge-JIC/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cambridge-JIC/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cambridge-JIC/Project&amp;oldid=291329" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>