"
Page
Discussion
View source
History
teams
Log in
 
Team:Cambridge-JIC/Marchantia/Codon
From 2014.igem.org
Cambridge iGEM 2014
Toggle navigation
Team
mösbi
Overview
Components
mösbi&You
Growth Chamber
Vision
Marchantia
Background
Enhancer trap
Promoter hunt
Codon optimisation
Request For Comments
Assembly
Guide
Results
Labwork results
Judging criteria
Parts table
Community
Scientific community
Outreach
Safety
Policy
Workshop
Acknowledgements
Executive Summary
Keep in contact with us...
Codon Optimisation
A codon table for Marchantia polymorpha
Method
Our dataset was made of large gap read mapping transcripts obtained by mRNA sequencing conducted by the Haseloff Lab on the M. polymorpha Cam strain.  Open Reading Frame (ORF) and Coding Sequence (CDS) predictions were made using the CLC bio Transcript Discovery plugin [1] .
We used ran blastx on our data [2] to verify the validity of the predicted ORFs and to compare the sequences with the proteins present in Arabidopsis. A list of candidate genes was compiled from the results and this list was used to calculate a frequency table of codon usage for M. polymorpha.
Results and Discussion
Figure 1: Our codon table for m. polymorpha
The number of Open Reading Frames (ORFs) resulting from our initial predictions totalled 99 000, which seemed too large to be realistic, and half of these were only 100 amino acids (100 aa) long. By filtering the dataset using a threshold of 300 aa for candidate genes we obtained a distribution of lengths that seemed reasonable.
Figure 2: A histogram of the lengths of predicted genes in m. polymorpha
In the blastx output, the longest complete sequence match was 40%. This is small as expected given the phylogenetic distance between M.polymorpha and A. thaliana [3] [4] .
The Codon Usage table we obtained for M. polymorpha is not strikingly similar to that of A. thaliana, as expected from the 400 million years of evolutionary divergence between them [5] .
However, there is a similarity in the slight preference for C over other bases at the end of codons and that for G-p-C sites, as can be seen in the codon table.
References
1. Qiagen, CLC bio Transcript Discovery ®, http://www.clcbio.com/clc-plugin/transcript-discovery/#description back to top
2. NCBI Blast ®, http://blast.ncbi.nlm.nih.gov/Blast.cgi back to top
3. Turmel M, Otis C and Lemieux C. 2003. The Mitochondrial Genome of Chara vulgaris: Insights into the Mitochondrial DNA Architecture of the Last Common Ancestor of Green Algae and Land Plants. The Plant Cell 15(8), pp. 1888-1903. back to top
4. Cantarel BL, Morrison HG, Pearson W. 2006. Exploring the Relationship between Sequence Similarity and Accurate Phylogenetic Trees. Molecular Biology and Evolution 23(11), pp. 2090-2100. back to top
5. Wellman CH, Osterloff PL, Mohiuddin U. 2003. Fragments of the earliest land plants. Nature 425, pp. 282-285. back to top
Retrieved from " http://2014.igem.org/Team:Cambridge-JIC/Marchantia/Codon "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
