"
Page
Discussion
View source
History
teams
Log in
 
Team:Cambridge-JIC/Protocol
From 2014.igem.org
Cambridge iGEM 2014
Toggle navigation
Team
mösbi
Overview
Components
mösbi&You
Growth Chamber
Vision
Marchantia
Background
Enhancer trap
Promoter hunt
Codon optimisation
Request For Comments
Assembly
Guide
Results
Labwork results
Judging criteria
Parts table
Community
Scientific community
Outreach
Safety
Policy
Workshop
Acknowledgements
Executive Summary
Keep in contact with us...
Contents:
Introduction
Polymerase Chain Reaction (PCR) to obtain DNA fragments
Gel electrophoresis to select the correct fragments
DNA purification for plasmid assembly
Gibson Assembly
E. coli transformation and plating
Colony analysis and selection by colony PCR
Miniprep and sequencing
The protocol for Marchantia
Introduction
We transform Marchantia using agrobacteria. The agrobacteria inject synthetic plasmids into the Marchantia nuclei.
A rough outline of the process:
Fragments of DNA are produced from a library of existing templates (plasmids)
These fragments are combined to produce the artificial plasmid
The artificial plasmid is amplified in E. coli
The artificial plasmid is incorporated into agrobacterium
The agrobactria are used to infect Marchantia plants which are 5 days old
The plants are grown and analysed
PCR
In this step we will amplify the fragments required for our plasmids with  Phusion DNA polymerase. PCR will amplify a fragment from a DNA  template, with the help of short DNA sequences called primers. Primers are oligonucleotides which can bind onto a template by complementary base pairing. Two primers are used to demarcate the two ends of the fragment of interest on the template. The sequence of each primer is designed to bind only to the correct site on the template and nowhere else.
Figure 1: The target synthetic plasmid which contains the gene we want to put into Marchantia. The sequence above the left and right borders is the part which will affect the behaviour of the plant (hopefully).
It is quicker and less error prone to PCR short fragments (<5kb). All the fragments are designed to overlap with each other by 20-40 bp. This allows the fragments to be joined together into a complete plasmid using the Gibson Assembly technique.
PCR Protocol
For each fragment: Add the template DNA (1µl) and primers (2.5µl at working concentration x10) to a labelled PCR tube
Create the phusion mix: the recipe for four fragments: in a 1.5 ml eppendorf:
HPLC H20 162.5 µl
5x HF buffer 50 µl
10mM dNTPs 5 µl
Phusion polymerase 2.5 µl
Shake or centrifuge the PCR tubes to ensure all DNA is at the bottom of the tube.
Add 44 µl of the Phusion mix to each PCR tube.
Without delay, place the PCR tubes into a PCR machine and set the Phusion protocol running:
30 sec of 98°C (initial denaturation)
30 cycles of:
10 sec of 98°C (denaturation)
20 sec of 58°C (annealing)
2:00 mins of 72°C (extension) - note the length of this step should be 30 seconds for each kb of the longest fragment
5 mins of 72°C (final extension)
Hold at 4°C
The PCR process takes ~2 hours, and after completion the PCR tubes can be held at 4°C  without problems for many hours.
Gel Electrophoresis
The PCR tubes now contain our fragments, as well as template and  primer DNA which must be removed. The fragments are extracted by running the PCR tube mixture through an agarose gel.  A voltage applied to the gel causes the negatively  charged DNA to migrate. Larger fragments migrate more slowly, and thus  DNA molecules will be separated according to size. The size of the fragments can be measured by comparison with a calibration ladder, which produces a characteristic pattern of bands corresponding to known lengths.
First, we should make the gel,(100 ml 1% (w/v) agarose TAE gel):
Weigh out 1g Ultrapure agarose, and add it to a glass flask
Add 100 ml 1xTAE
Microwave for 2:30 mins, swirling the flask after 1:30 mins
Place the glass flask in a 55°C hybridizer to cool for 20-30 mins (the gel must not be poured when it is too hot)
Pour the gel into two 50 ml falcon tubes
In a dedicated gel area, add 5 µl of Sybr Safe DNA dye to each falcon
Seal the open sides of the gel mould with masking tape
Pour the gel from both falcon tubes into the gel mould and insert an 8 tooth comb
Leave the gel to set for 30 mins
Take the PCR tubes which contain the DNA fragments. Add 12.5 µl of 5x loading dye to each PCR  tube
Remove the comb and walls from the set gel, and place into a gel tank. Make sure there is enough TAE to cover the gel
Load the gel lanes:
Put Hyperladder 1kb in lane 1
Fill the remaining lanes with the contents of the PCR tubes. One lane per tube. Make a note.
Run at 100V for 40 mins
DNA purification
We will now recover our fragments from the gel. First, label an eppendorf for each fragment and place a gel cutting tip in each tube. Take the finished gel to a dark room along with your tubes, gel  cutting tips and a P1000. For each fragment, use the blue light illuminator to locate the  appropriately sized DNA band and extract it using the cutting pipette, place the  gel fragment into the appropriate eppendorf. Take a picture of the gel in the imaging box (The imaging box  uses UV light, which can damage DNA, so do this after you cut out your band).
Once the gel fragment is in your tube, take it back to the lab and purify  the DNA using the Qiagen Minelute kit. A protocol is provided.
Gibson Assembly
We combine our DNA fragments into the target synthetic plasmid using Gibson Assembly.  The Gibson Assembly reaction is an isothermal one-pot reaction  in which three enzymes convert the linear DNA  fragments from PCR into circular DNA. (For more information see the  synbio.org guide, or Gibson et al., Nature Methods, 2009)
Pipette 0.5 µl of each DNA fragment into a PCR tube (make sure all the liquid is in the bottom of the tube)
Start the PCR machine running a Gibson protocol (50°C for 1 hour,  then hold at 4°C)
Add 3n µl of 1.33x Gibson Master Mix to the tube (where n is the total volume of the fragments
Place immediately into a PCR machine
Note that it is important to place the tube at 50°C very quickly after  adding the master mix. If you wait too long (more than 20-30 secs), the  exonuclease enzyme (which does not work well at 50°C) will degrade too  much of the DNA, lowering the efficiency of the reaction significantly.
Second Note: a PCR tube with water in place of the Master Mix can be used as a negative control. (It should contain 0.5 µl of each DNA fragment and 3n µl of water).
E. coli Transformation
The target synthetic plasmid can be amplified by inserting it into E. coli
Get a tube of 50 µl E. coli chemically competent cells from the -80°C  freezer, and place it on ice
Once the cells have thawed, place the whole Gibson reaction into  the competent cells tube, and return the mixture to ice for 15-20  minutes
Place the mixture into a 42°C water bath for 1 minute
Return the tube to the ice for another minute
Add 250 µl of SOC
Place into a 37°C shaking incubator for 1 hour for recovery, giving  the cells time to express the antibiotic resistance. Also place a 9mm  petri dish containing LB agar and the appropriate antibiotic  (kanamycin 50 in our case) in the incubator with the cells to warm  up.
Plate the entire 300 µl onto LB agar plates, spreading the liquid  over the plate with the L-shaped spreader
Incubate the plate overnight at 37°C
Note that the competent cells are very fragile, and must be kept on ice  before the heat shock step. Also, do not to touch the bottom of the  competent cell tubes with your hands, as this may warm them too  much. Also, when adding the results of the Gibson reaction, do not  pipette the cells up and down.
Colony PCR
Now that we (hopefully) have colonies containing plasmids, we need  to choose a colony to grow to obtain the correct plasmid for further  transformations into Agrobacterium and Marchantia.  Misassembly is possible with Gibson, particularly if there are repeated  sequences present or if the plasmid is somewhat toxic to the bacteria  (which should not be a problem in our case).
We will now verify that the plasmid is correctly assembled by PCR. We  will amplify a fragment across two Gibson junctions, to confirm that  our insert has been correctly incorporated. To do this, we will use the  35s_seq and nosT_Bseq primers, which bind as shown on the diagram  below. We can also use these primers for sequencing later.
!!Put in picture of the plasmid segment !!
Colony PCR Protocol:
Identify 8 colonies to screen, and label them (scale the reaction  down if you have fewer than 8 colonies)
Make a mix with the phusion and primers in an eppendorf tube:
HPLC H20 134 µl
5x HF buffer 40 µl
10mM dNTP 4 µl
10µM FWD primer 10 µl
10µM REV primer 10 µl
TAQ polymerase 2 µl
Prepare 9 small PCR tubes, and pipette 20 µl of the phusion  mixture into each tube
With a small tip, touch a colony, and mix into the appropriate  PCR tube. Note: do not pick up too much bacteria, as it will  disrupt the PCR, a very small amount of DNA can act as a  template  10
Onto the last PCR tube, place 1 µl of the 35s_RLTI template DNA  to act as a positive control
Place into a PCR machine and run a Phusion protocol, with an  additional lysing step:
6 mins of 98°C (lysing and denaturation)
30 cycles of:
10 sec of 98°C (denaturation)
20 sec of 58°C (annealing)
1:00 mins of 72°C (extension)
5 mins of 72°C (final extension)
Hold at 4°C
Prepare a gel whilst you wait (1% agarose (w/v) in TAE = 1g of  UltraPure agarose in 100 ml of TAE), using the thinner toothed  combs in the mould
Place the PCR products into the gel lanes (with 5 µl of loading  dye), as well as Hyperladder 1kb in each row
Run for 40 mins at 100V
Image the gel
The correct band size depends on the construct is being built, but they  should be around 1-1.2 kb. Select a colony with a correct band, and place  with an inoculation loop into an unskirted falcon containing 15 ml of LB +  kanamycin 50µg/ml. Leave overnight in a 37°C shaking incubator.
Miniprep
We will now harvest the plasmids from the E. coli for further use. We will  do this using the Qiaprep kit, so follow the protocol provided in the kit.  Colony PCR can only tell us if the fragment is the correct length, which is  indicative of a correct assembly (and only in the junctions we are PCRing  over). Once we have our plasmids, we should sequence them to confirm  that their sequences are correct. To do this, obtain an account with  Source Biosciences, and send them the appropriate amount of DNA and  primers for Sanger sequencing.
After receiving the sequencing data, we will (hopefully) have a newly  constructed DNA plasmid with the sequence we wanted.  Congratulations, you have now built a new plasmid!
The protocol for Marchantia:
Day 1 (need primers & template DNA)
PCR (3 hours)
Gel purification (2 hours)
Gibson (1.5 hours)
E. coli transformation (1.5 hours then overnight)
Day 2 & 3
Select E. coli
50ml culture
Plasmid extraction
Marchantia spore sterilisation & plating
Day 4
Agrobacterium transformation
Day 7
Pick Agrobacterium and grow overnight
Day 8
Transform Marchantia
Day 10
Plate Marchantia transformants
Day 13+
Results
Edit this page
Retrieved from " http://2014.igem.org/Team:Cambridge-JIC/Protocol "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
