<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tongji_China_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tongji China/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="eth-main-contents" class="eth-main-contents"><HEADER id="header"><NAV><UL class="main-menu col-to-row enlarge-line-height custom_cursor"><LI onclick="return true"><SPAN class="menu-entry"><A href="/Team:Tongji_China/Project">Project</A></SPAN></LI><LI onclick="return true"><A href="/Team:Tongji_China/Background">Background</A></LI><LI onclick="return true"><A href="/Team:Tongji_China/Description">Description</A></LI><LI onclick="return true"><A href="/Team:Tongji_China/Design">Design</A></LI></UL><LI onclick="return true"><SPAN class="menu-entry">Lab</SPAN></LI><LI onclick="return true"><SPAN class="wet"><A href="/Team:Tongji_China/WetLab">Wet Lab</A></SPAN></LI><LI onclick="return true"><A href="/Team:Tongji_China/Demonstrate">Experiment</A></LI><LI onclick="return true"><A href="/Team:Tongji_China/Protocol">Protocol</A></LI><LI onclick="return true"><A href="/Team:Tongji_China/Notebook">Notebook</A></LI><UL class="subsubmenu"><LI onclick="return true"><SPAN class="dry"><A href="/Team:Tongji_China/Model">Dry Lab</A></SPAN></LI><LI onclick="return true"><A href="/Team:Tongji_China/Programme">Programming</A></LI><LI onclick="return true"><A href="/Team:Tongji_China/Modeling">Modeling</A></LI></UL></NAV></HEADER></DIV><LI onclick="return true"><SPAN class="menu-entry"><A href="/Team:Tongji_China/Safety">Safety</A></SPAN></LI><LI onclick="return true"><A href="/Team:Tongji_China/Safety_P.A."><I>P. aeruginosa</I></A></LI><LI onclick="return true"><A href="/Team:Tongji_China/Light_control">Light control</A></LI><LI onclick="return true"><A href="/Team:Tongji_China/Animal_check_in">Animal check in</A></LI><LI onclick="return true"><SPAN class="menu-entry"><A href="/Team:Tongji_China/Human_Practices">HP</A></SPAN></LI><LI onclick="return true"><A href="/Team:Tongji_China/Public_Engagement">Education&amp;Engagement</A></LI><LI onclick="return true"><A href="/Team:Tongji_China/Integrated">Integrated</A></LI><LI onclick="return true"><SPAN class="menu-entry"><A href="/Team:Tongji_China/Team">Team</A></SPAN></LI><LI onclick="return true"><A href="/Team:Tongji_China/Team_members">Team members</A></LI><LI onclick="return true"><A href="/Team:Tongji_China/Collaborations">Collaborations</A></LI><LI onclick="return true"><A href="/Team:Tongji_China/Attributions">Attributions</A></LI><LI onclick="return true"><SPAN class="menu-entry">Achievement</SPAN></LI><LI onclick="return true"><A href="/Team:Tongji_China/Judging_form">Judging form</A></LI><LI onclick="return true"><A href="/Team:Tongji_China/Part">Parts</A></LI><LI onclick="return true"><A href="/Team:Tongji_China/InterLab">Interlab</A></LI><LI onclick="return true"><A href="/Team:Tongji_China/Capsule">Capsule</A></LI><TITLE>Programme</TITLE><DIV class="background"><DIV class="content"><DIV class="title">
			Wet Lab
		</DIV><DIV class="title_2">
			Experiment
		</DIV>
Here you can read detailed descriptions of the experiments we've done. To read about how we structured and planned our work, see the <A href="https://2018.igem.org/Team:Tongji_China/Design">Design</A>.
You can get straight to a section by following these links:<DIV style="text-align:center"><FONT size="5" face="Trebuchet MS"><A href="#1">Plasmid Construction</A><A href="#2">Electroporation of <I>P. aeruginosa</I></A><A href="#3">Functional Analysis of T3SS</A><A href="#4">Future Plans</A></FONT></DIV><H1>Plasmid Construction</H1><DIV class="achievement"><P class="littletitle">Achievements</P><UL><LI>Successfully construct 2 plasmids containing positive control antigen DNA.</LI><LI>Successfully construct 4 plasmids containing antigen DNA according to our filtration.</LI></UL></DIV><P class="littletitle">Introduction</P>
In order to let the <I>P. aeruginosa</I> inject the antigens into the antigen presenting cells (APCs), we first need to add the DNA sequences of antigens into the T3SS plasmid. The plasmid we use is <I>Escherichia-Pseudomonas</I> shuttle expression plasmid pExoS54F (shown in <B>Figure 1</B>), which encodes the promoter of T3SS effector ExoS with N-terminal ExoS1–54 signal sequence, followed by a FLAG-tag and a multiple cloning site (MCS). The pExoS54F plasmid contains two promoter region which can be activated simultaneously by ExsA's binding to their common promoter region. PexoS is the promoter region which originally belongs to the toxin gene ExoS and the wild type <I>P. aeruginosa</I> inject the toxin ExoS into the host cells through the T3SS. The ExoS gene of the <I>P. aeruginosa</I> strain we use has been knocked out so we utilize its promoter and its N-terminal ExoS1–54 signal sequence which act as a T3SS secretion signal to let the T3SS secret proteins of interest. SpcS is a kind of T3SS chaperone and helps the proteins of interest to enter the T3SS secretion channel.<DIV class="instructionOfPicture"><B>Figure 1</B> | Escherichia-Pseudomonas shuttle expression plasmid pExoS54F
			</DIV>
The proteins encoded by the pExoS54F plasmid are actually not all the proteins that function in the process of T3SS protein delivery. There are approximately 40 proteins that regulate the secretion of T3SS effector proteins and many of them are encoded in the <I>P. aeruginosa</I> genome. The protein ExsE, ExsC, ExsD and ExsA are four cytoplasmic proteins (shown in the <B>Figure 2</B>) that control the coupling of transcription and secretion. ExsA is a DNA-binding protein required for transcriptional activation of the entire T3SS. The second regulatory protein, ExsD, functions as an anti-activator by directly binding to ExsA. ExsC functions as an anti-anti-activator by directly binding to and inhibiting ExsD.  ExsE functions as a direct inhibitor of ExsC and provide an initiating signal for the whole process. <B>Figure 2</B> shows the situation when the T3SS secretion is inhibited because the direct activator ExsA is inhibited by the binding ExsD.<DIV class="instructionOfPicture"><B>Figure 2</B> | Four cytoplasmic proteins ExsE, ExsC, ExsD and ExsA control the coupling of transcription and secretion.
			</DIV><P class="littletitle">Overview</P>
1.Sequence Synthesis
2.Plasmid Restricted Digestion
3.Ligation &amp; Transformation<P class="littletitle">Results</P><B>1.Sequence Synthesis</B>
As we successfully filtered many antigens which may active the immune system and guide the T cells to target to the tumor cells, we choose 4 of them and two positive controls: NY-ESO-A and NY-ESO-B. NY-ESO is widely known as a germ cell protein that is often expressed by tumor cells but not normal somatic cells. The frequent finding of humoral and cellular immune responses against this antigen in cancer patients with NY-ESO-expressing tumors makes it one of the most immunogenic human tumor antigens ever known. <B>Table 1</B> shows the sequences of these antigens.<TABLE width="90%" align="center"><TBODY><TR><TH align="center">Part name</TH><TH align="center">Antigen</TH><TH align="center">Sequence</TH></TR><TR><TD align="center">
						BBa_K2730001
					</TD><TD align="center">
						NY-ESO-A
					</TD><TD align="center">
                                        atgtcgttgttgatgctgatcacccagtgcccgttgtga
                                        </TD></TR><TR><TD align="center">
						BBa_K2730002
					</TD><TD align="center">
						NY-ESO-B
					</TD><TD align="center">
                                        atgcagttgtcgttgttgatgctgatcacctga
                                        </TD></TR><TR><TD align="center">
						BBa_K2730003
					</TD><TD align="center">
						0201
					</TD><TD align="center">
                                        atgttgcacttgtagggctcgtagccgccggcgtga
                                        </TD></TR><TR><TD align="center">
						BBa_K2730004
					</TD><TD align="center">
						0301A
					</TD><TD align="center">
                                        atgcacttgtagggctcgtagccgccggcgcggtga
                                        </TD></TR><TR><TD align="center">
						BBa_K2730005
					</TD><TD align="center">
						0301B
					</TD><TD align="center">                        atggcgatctcgacccgggacccgttgtcgaagtga
                                        </TD></TR><TR><TD align="center">
						BBa_K2730006
					</TD><TD align="center">
						0301C
					</TD><TD align="center">
                                        atgaagttgttgaagcggcaggcggaaggcaagtga
                                        </TD></TR></TBODY></TABLE><DIV class="instructionOfPicture"><B>Table1</B> | sequences of antigens
			</DIV> Because the antigen sequences are quite short, we cannot choose the common way of synthesizing double strands. So we synthesize the 5’-3’ single strands and the 3’-5’ single strands with restriction site on both sides, then take the method of annealing (see in the <A href="/Team:Tongji_China/Protocol">protocol</A>) to pair two single strands into a double strand (<B>Figure 3</B>). <B>Table 2</B> shows all the single strands we synthesized.
			<DIV style="text-align: center;"><TABLE border="0" cellspacing="0.3em"><TBODY><TR><TD>NY-ESO-A-F</TD><TD>ctagaATGTCGTTGTTGATGCTGATCACCCAGTGCCCGTTGTGAg</TD></TR><TR><TD>NY-ESO-A-R</TD><TD>tcgacTCATCACAACGGGCACTGGGTGATCAGCATCAACAACGACATt</TD></TR><TR><TD>NY-ESO-B-F</TD><TD>ctagaATGCAGTTGTCGTTGTTGATGCTGATCACCTGAg</TD></TR><TR><TD>NY-ESO-B-R</TD><TD>tcgacTCAGGTGATCAGCATCAACAACGACAACTGCATt </TD></TR><TR><TD>0201-F</TD><TD>ctagaATGTTGCACTTGTAGGGCTCGTAGCCGCCGGCGTGAg</TD></TR><TR><TD>0201-R</TD><TD>tcgacTCACGCCGGCGGCTACGAGCCCTACAAGTGCAACATt </TD></TR><TR><TD>0301A-F</TD><TD>ctagaATGCACTTGTAGGGCTCGTAGCCGCCGGCGCGGTGAg </TD></TR><TR><TD>0301A-R</TD><TD>tcgacTCACCGCGCCGGCGGCTACGAGCCCTACAAGTGCATt</TD></TR><TR><TD>0301B-F</TD><TD>ctagaATGGCGATCTCGACCCGGGACCCGTTGTCGAAGTGAg </TD></TR><TR><TD>0301B-R</TD><TD>tcgacTCACTTCGACAACGGGTCCCGGGTCGAGATCGCCATt </TD></TR><TR><TD>0301C-F</TD><TD>ctagaATGAAGTTGTTGAAGCGGCAGGCGGAAGGCAAGTGAg</TD></TR><TR><TD>0301C-R</TD><TD>tcgacTCACTTGCCTTCCGCCTGCCGCTTCAACAACTTCATt</TD></TR></TBODY></TABLE></DIV><DIV class="instructionOfPicture"><B>Table2</B> | All the single strands
			</DIV><DIV class="instructionOfPicture"><B>Figure 3</B> | Annealing
			</DIV><B>2.Plasmid Restriction Digestion</B>
We use the restriction endonuclease Xbal I and Sal I to digest the pExoS54F plasmid (see in the <A href="/Team:Tongji_China/Protocol">protocol</A>). Also the antigen sequences we synthesized have the restriction site of Xbal I and Sal I. we set the single digestion control and the plasmid control to figure out whether the plasmid is digested completely. The DNA gel electrophoresis results (<B>Figure 4</B>) show that the digestion is complete.
<DIV class="instructionOfPicture"><B>Figure 4</B> | DNA gel electrophoresis results for plasmid restriction digestion
</DIV><B>3. Ligation &amp; Transformation</B>
We link the digestion product and double-stranded fragments using T4 DNA ligase and conduct the chemical transformation (see in the <A href="/Team:Tongji_China/Protocol">protocol</A>). We conduct the colony PCR to test whether the colonies contain the right plasmids (see in the <A href="/Team:Tongji_China/Protocol">protocol</A>). The DNA gel electrophoresis results (<B>Figure 5</B>) show that some of the colonies contain the right plasmids we want.
<DIV class="instructionOfPicture"><B>Figure 5</B> | DNA gel electrophoresis results for colony PCR
</DIV><H1>Electroporation of <I>P. aeruginosa</I></H1><DIV class="achievement"><P class="littletitle">Achievements</P><CENTER>Successfully transfer the plasmids into <I>P. aeruginosa</I>.</CENTER></DIV><DIV class="littletitle">Introduction </DIV>
To deliver antigens into cells T3SS system, plasmids containing antigen sequences should be transfered to <I>Pseudomonas aeruginosa</I>. We choose <I>Pseudomonas aeruginosa</I> strain PAK-JΔ9 which is an attenuated strain, for it has many advantages varying from efficiency to safety.

The transform technique we use is electroporation, the most efficient bacterial transformation method available, which orders of magnitude more efficient and versatile than chemical methods. Electroporation uses accurately pulsed electric currents to induce transient gaps in the phospholipid bilayer of cells, and extracellular genetic material passes through these transient gaps. Genetic material is assimilated by the target cells’ DNA.

The highest transformation efficiencies are obtained when cells are harvested in early mid-log growth, so we need to prepare bacteria before operating and make competent cells.

We transform <I>Pseudomonas aeruginosa</I> (see in the <A href="https://2018.igem.org/Team:Tongji_China/Protocol">Protocol</A>) following the Gene Pulser Xcell<SUP>TM</SUP> Electroporation System Instruction Manual.
<DIV class="littletitle">Results</DIV><B>1. Electroporation</B>

We use Gene Pulser Xcell™ Electroporation Systems (Bio-Rad) to transform bacteria.

Gene Pulser Xcell conditions: C = 25 μF; PC = 200 ohm; V = 2.5 kV.

After pulsing the competent cells, incubate for 1 hour and plate cells onto LB agar plates with carbenicillin. Incubate for 12 hours at 37°C. Next day, observe the growth of the bacteria.
<DIV class="instructionOfPicture"><B>Figure 1</B> | The growth of the <I>P. aeruginosa</I> the next day.
</DIV>
From <B>figure 1</B>, we can see that there are some colonies on the LB agar plates, however, the majority are lawns. From many times of experiments, we verify this conclusion.
<B>2. Colony PCR</B>

Though there are colonies on the carbenicillin-resistant LB plate, we are not sure that the colonies contain the plasmids we want, because some colonies may be satellite colonies. So, we did colony PCR to test the presence of plasmids(Results are shown in <B>Figure 2</B>).
<DIV class="instructionOfPicture"><B>Figure 2</B> | The electrophoresis results of colony PCR.
</DIV>
From the electrophoresis results of colony PCR, we can see that the bands of our target DNA fragments are in the corresponding position (~200 bp). Therefore, it is indicated that our recombinant plasmids have been successfully transferred into <I>P. aeruginosa</I>.
<H1>Functional Analysis of T3SS</H1><DIV class="achievement"><P class="littletitle">Achievements</P><UL><LI>We confirm that <I>P. aeruginosa</I> can synthesize the antigens by the induction of EGTA.</LI><LI>Through infection of HeLa cells <I>in vitro</I>, we prove that bacteria can inject antigens into cells.</LI><LI>We briefly confirm that the bacteria did not secrete proteins outside the cells while attaching cells and injecting.</LI><LI>Using the animal model, the T3SS is proved to work <I>in vivo</I>.</LI></UL></DIV><DIV class="littletitle">Introduction</DIV>
In normal high calcium environment, antigens can’t be synthesized by bacteria.<DIV class="instructionOfPicture"><B>Figure 1 </B>| In normal environment, pore of T3SS is closed, and antigens can’t be synthesized.
</DIV>
Secretion of the antigens can be activated in two ways.
One way is to form the host cell contact. When a contact signal has been sensed by the bacteria, a rapid production and specific insertion into the translocon follows and the antigens can successfully be injected into the host cell cytosol, without wasting them into the culture supernatant. This way is called “polar translocation”.<DIV class="instructionOfPicture"><B>Figure 2 </B>| Polar Translocation 
</DIV>
Another way is triggered by low calcium environment, such as in the presence of calcium chelator EGTA. It can trigger the bacteria to release the antigens into the culture medium without the formation of the T3SS translocon. This way works without the presence of host cells and is defined as “non-polar translocation”.
<DIV class="instructionOfPicture"><B>Figure 3 </B>| Non-polar Translocation 
</DIV>
To analyze the antigen secretion ability of the engineered bacteria we constructed, we use both the “non-polar translocation” way and the “polar translocation” way to conduct the experiment.
The signature is detected by the FLAG-tag which is carried in plasmids: pExoS54F-NY-ESO-A, pExoS54F-NY-ESO-B, pExoS54F-0201, pExoS54F-0301A, pExoS54F-0301B, pExoS54F-0301C and pExoS54F-mCherry. The molecular weight of the antigens is ~30 kDa in theory.
<DIV class="littletitle">Overview</DIV>
1. First, in order to test whether the bacteria can synthesize the antigens we want, EGTA is used to induce <I>Pseudomonas aeruginosa</I> to secrete the antigen polypeptides into the culture solution, and the presence is detected by <A href="#re1">western blot (see result 1)</A>.
2. In order to test whether bacteria can inject antigens into cells, we perform <I>in vitro</I> infection of HeLa cells, and the presence of FLAG-tag in the cells was detected by <A href="#re2">western blot (see result 2)</A> and <A href="#re3">immunofluorescence (see result 3)</A> to confirm the successful injection of antigens.
3. At the same time, in order to prove that the bacteria do not secrete proteins outside the targeted cells while attaching cells and injecting, we detect the culture supernatant of the infected cells by <A href="#re4"> western blot (see result 4)</A>.
4. Finally, using the animal model, the T3SS is further verified <I>in vivo</I> by <A href="#re5">immunohistochemistry (see result 5)</A>.
<DIV class="littletitle">Results</DIV><B>1. Western Blot (EGTA Induction)</B>
All the proteins secreted by the <I>P. aeriginosa</I> strain PAK-J△9 are analyzed by western blot. 
We use the calcium chelator EGTA to trigger the secretion of proteins. (To see the exact process of the induction, you can go to the <A href="https://2018.igem.org/Team:Tongji_China/Protocol">Protocol</A>.) We add 100% TCA to culture medium to reach a final concentration of approximately 10% which can precipitate the proteins form culture medium. But the final concentration of TCA may vary with the molecular weight of the precipitated protein but can't be too high, otherwise other substances will be precipitated. 

First, we test which TCA concentration can better precipitate proteins from culture medium so we set the TCA concentration gradient of 10%, 15% to 20%. We randomly select one of <I>P. aeruginosa</I> which carries the antigen in theory to conduct the experiment. 

The result is analyzed by western blot (<B>Figure 1.1</B>). As is shown in figure, 20% TCA concentration successfully precipitates the proteins. So we decide to use the 20% TCA concentration.
<DIV class="instructionOfPicture"><B>Figure 1.1 </B>| As is shown in the figure, both 10% and 15% TCA concentration cannot precipitate the proteins from culture medium. Only the 20% TCA concentration can successfully precipitate the proteins.
</DIV>

According to the conclusion we made in the first experiment, we conduct EGTA induction experiment to test whether the engineered <I>P. aeruginosa</I> we constructed can produce and secret the antigens.
We set 6 test groups and a control group. Each test group contains engineered <I>P. aeruginosa</I> carrying the plasmids: pExoS54F-NY-ESO-A, pExoS54F-NY-ESO-B, pExoS54F-0201, pExoS54F-0301A, pExoS54F-0301B and pExoS54F-0301C. The culture medium of the control group is without the presence of EGTA and the rest of the conditions are consistent with the test groups. 
The result is analyzed by western blot (<B>Figure 1.2</B>). Five antigens (NY-ESO-A, NY-ESO-B, 0201, 0301A and 0301C) are successfully secreted by bacteria.<DIV class="instructionOfPicture"><B>Figure 1.2 </B>| Five antigens (NY-ESO-A, NY-ESO-B, 0201, 0301A and 0301C) are successfully secreted by bacteria but no extinct signal was detected from 0301B.
</DIV>

There may be something wrong in our operation with pExoS54F-0301B plasmid, but we don’t have enough time to test. If we have more time, we will do this experiment again using pExoS54F-0201 plasmid as positive control to make sure the whole experiment operation is correct. At the same time, we will do colony PCR to check whether the pExoS54F-0301B plasmids are still in our engineered <I>P. aeruginosa</I>. If we can’t get positive result, we will repeat experiments from the pocedure of electroporation of <I>P. aeruginosa</I>. However, from the results below we can conclude that the bacteria contain these plasmids and our design is correct.

From another 5 plasmids, we can draw a conclusion that the bacteria can synthesize the antigens we want by the induction of EGTA.

<B>2. Western Blot (Lysates of Infected Cells)</B>
We conduct an experiment to analyze whether the host cell contact can trigger the production of the antigens and whether the engineered <I>P. aeruginosa</I> can inject these antigens into host cell cytosol in the level of protein. 
We infect the HeLa cells with the engineered <I>P. aeruginosa</I> respectively carrying the plasmids: pExoS54F-NY-ESO-A, pExoS54F-NY-ESO-B, pExoS54F-0201, pExoS54F-0301A, pExoS54F-0301B and pExoS54F-0301C, then collect the lysates of HeLa cells. We analyze the cell lysates by western blot (<B>Figure 2</B>).<DIV class="instructionOfPicture"><B>Figure 2 </B>| Western Blot (Using the cell lysates samples)

</DIV>
The result shows that all of the six antigens can be detected in the western blot. The engineered <I>P. aerugnosa</I> carrying the plasmids: pExoS54F-NY-ESO-A, pExoS54F-NY-ESO-B, pExoS54F-0201, pExoS54F-0301A, pExoS54F-0301B and pExoS54F-0301C can be successfully induced to express and inject the peptides into the host cells.<B>3. Immunofluorescence of Infected Cells</B>
In order to increase the persuasiveness of the experiments, we also use immunofluorescence technique to prove that bacteria injecte antigens into the cells.
Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. This technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample.
We use anti-FLAG antibody to specifically target FLAG-tag, which is supposed to be in the cells. Then we use secondary antibody which carries the fluorophore to recognize the primary antibody and bind to it. And we also use DAPI to label DNA. (You can see more details in <A href="/Team:Tongji_China/Protocol">protocol</A>.)
We use an epifluorescence microscope to confirm the presence of intracellular antigens and observe finer structures by confocal microscope.
s
<DIV class="instructionOfPicture"><B>Figure 3.1 </B>| Results of immunofluorescence pictured by epifluorescence microscope. Antigens are in green. The nuclei are stained blue by DAPI. Scar bar, 50 μm.</DIV><DIV class="instructionOfPicture"><B>Figure 3.2 </B>| Results of immunofluorescence pictured by confocal microscope. Antigens are in green. The nuclei are stained blue by DAPI. Scar bar, 100 μm.</DIV>


We can see antigens several cells are injected into cells by T3SS. But the efficiency is not high, which is related to the relative amount of cells and bacteria. And what catches our attention is that we can find green antigens in the group of pExoS54F-0301B plasmid, which support this plasmid works.
So, we can confirm that, <I>P. aeruginosa</I> can inject antigens into cells <I>in vitro</I>.
<B>4. Western Blot (Culture Supernatant of Infected Cells)</B>
In order to prove the safety of our system, we conduct an experiment to test whether the culture supernatant of bacterially infected cells contains the antigens. 

We collect the culture supernatant of bacterially infected cells and use the TCA-acetone precipitation to precipitate the secreted proteins which may be contained in the culture supernatant. To prove the experimental operation is correct, we set a positive control of cell lysates (These cells are infected by <I>P. aeruginosa</I> carrying pExoS54F-mCherry). Then we analyze the sediment by western blot to figure out the existence of the secreted proteins (<B>Figure 4</B>). As is shown in the figure, all of the culture supernatant of bacterially infected cells does not contain the proteins, which further proves that our system is safe enough.
<DIV class="instructionOfPicture"><B>Figure 4 </B>| Six samples (left) of culture supernatant of bacterially infected cells which was infected respectively by the <I>P. aeruginosa</I> carrying the pExoS54F-NY-ESO-A, pExoS54F-NY-ESO-B, pExoS54F-0201, pExoS54F-0301A do not show extinct signature of existence of proteins. And the signal of positive control (Flag-mCherry) is successfully detected.
</DIV>
We can draw a conclusion that our system is safe enough and only inject the antigens into cell cytosol when attaching to the cell membrane. 
<B>5. Immunohistochemistry</B>
Finally, using the animal model, the T3SS is further verified <I>in vivo</I> by immunohistochemistry. We give mice bacteria by oral gavage. After 4 hours, the mice are dissected, frozen sections of the mice intestine are taken for immunofluorescence experiment. In this way, we check whether the T3SS is effective <I>in vivo</I>. The pictures below are part of our results, and the design of this experiment accords with our animal check in form.
<DIV class="instructionOfPicture"><B>Figure 5 </B>| The pictures are the immunohistochemistry of frozen selections of mice intestine with anti-FLAG antibody. The pictures above are all taken by the ZEISS Axio Imager 2. The time for FITC is 1000 ms while the time for DAPI is 29 ms.
</DIV>
According to the pictures above, compared to the negative control, two test groups with different kind of antigens both have some areas with higher intensity of green fluorescence. In these areas, we can find blue fluorescence inside which shows the nuclei and the peptides are secreted into the cytoplasm. From the pictures, we can conclude that it is not that efficient since there are only several positive cells in one visual field.
According to the articles before and the experiments done by other institutes, the efficiency should be higher. For we do not have enough time to revise the experimental conditions, we come up with several possible reasons for the low efficiency. Firstly, about the bacteria, there used to be bacteria saved in -80℃ which lost the plasmid. It is possible that this happened to us again. Secondly, the concentration of bacteria lavaged to the mice may be a little bit low. Thirdly, we do not add extra termination codon to the antigen so that there are several amino acids behind the antigen which may have influence on the efficiency. Also, for the background of the negative control is high, there may be something wrong with our sections or antibody.
In all, it is proved that this system works, though the efficiency is relatively low according to <I>our</I> results. With more conditions to be optimized and the design of experiments to be more precise, the results should be better. It is after all a promising system used in immunotherapy for cancer. And we will continue to work for it.
<DIV class="reference"><DIV class="littletitle"><FONT face="Trebuchet MS">Reference</FONT></DIV>
https://en.wikipedia.org/wiki/Immunofluorescence
</DIV><H1>Future plan</H1><DIV class="littletitle">1 Optimizing Experiment Condition</DIV>
Although we obtained many positive results during the experiment, there is still a certain distance from the result we expect. We analyzed our results and list many feasible directions to optimize our experiment condition. We are planning to conduct more experiments to figure out better experiment conditions.
<DIV class="littletitle">2 Further Experimental Proof</DIV>
In our lab experiment, we just do the early stage work to prove that our system works efficiently. There are lots of things we can do to further prove the feasibility of our system. 
<B>1. ELISA &amp; ELISpot</B>
Firstly, we plan to conduct ELISA and ELISpot using wild type mice to analyze whether our system can cause immune system response after at least one month engineered <I>P. aeruginosa</I> immunizing.

Enzyme-linked immunosorbent assay (abbreviated ELISA) refers to the qualitative and quantitative detection method of binding soluble antigen or antibody to a solid phase carrier, using antigen-antibody binding specificity for immune reaction. We will immunize the wild type mice by the engineered <I>P. aeruginosa</I> for at least one month and collect their serum to conduct the ELISA to analyze the IFN-γ level, which can reflect the extent of T cell activation.

Enzyme-linked immunoSpot assay (ELISpot) is a modified version of the ELISA immunoassay, allowing for the detection of cells that secrete cytokines or antibodies. Unlike ELISA, ELISpot is able to detect a single cell that secretes a protein of interest.

To conduct the ELISpot, we first need to place cytokine-specific antibodies onto an ELISpot plate, then the CD8+ T cells will be added. The activated T cells produce cytokine which binds to the antibodies. The cells are then removed whilst detection antibodies that can bind with the enzyme being added. Finally the substrate forms a colored spot on the T cells which secrete the antibody and the cells are then counted through a microscope. Through the ELISpot we can detect the percentage of activated T cells to evaluate the antigenic activity.
<B>2. Tumor cell challenge experiment</B>
If some positive signals can be detected, we will then do the tumor cell challenge experiment.

During the protection against tumor cell challenge, we will immunize the mice for at least two weeks and then challenge the immunized mice with tumor to see whether the immune system can protect the immunized mice from tumor.

Then, we will conduct experiment on tumor therapy model to test whether the activated immune system can kill the tumor cells. We can analyze the ability of the antigens by measuring the size of the tumor.
</DIV></DIV><DIV class="footer-all"><DIV class="footer-right"><H5 class="title-h4">Contact</H5><P class="my-content-p3">College of Life Science and Technology</P><P class="my-content-p3">Tongji university </P><P class="my-content-p3">No.1239, Yangpu District, Shanghai, China</P><P class="my-content-p3" style="word-wrap:break-word;">Email: tongjiigem2018@126.com</P></DIV></DIV><DIV id="footer-info" class="inner"><P class="my-content-p3" style="color: #777; text-align: center !important;">Copyright © 2018 Tongji_China</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>