<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tongji_China_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tongji China/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="eth-main-contents" class="eth-main-contents"><HEADER id="header"><NAV><UL class="main-menu col-to-row enlarge-line-height custom_cursor"><LI onclick="return true"><SPAN class="menu-entry"><A href="/Team:Tongji_China/Project">Project</A></SPAN></LI><LI onclick="return true"><A href="/Team:Tongji_China/Background">Background</A></LI><LI onclick="return true"><A href="/Team:Tongji_China/Description">Description</A></LI><LI onclick="return true"><A href="/Team:Tongji_China/Design">Design</A></LI></UL><LI onclick="return true"><SPAN class="menu-entry">Lab</SPAN></LI><LI onclick="return true"><SPAN class="wet"><A href="/Team:Tongji_China/WetLab">Wet Lab</A></SPAN></LI><LI onclick="return true"><A href="/Team:Tongji_China/Demonstrate">Experiment</A></LI><LI onclick="return true"><A href="/Team:Tongji_China/Protocol">Protocol</A></LI><LI onclick="return true"><A href="/Team:Tongji_China/Notebook">Notebook</A></LI><UL class="subsubmenu"><LI onclick="return true"><SPAN class="dry"><A href="/Team:Tongji_China/Model">Dry Lab</A></SPAN></LI><LI onclick="return true"><A href="/Team:Tongji_China/Programme">Programming</A></LI><LI onclick="return true"><A href="/Team:Tongji_China/Modeling">Modeling</A></LI></UL></NAV></HEADER></DIV><LI onclick="return true"><SPAN class="menu-entry"><A href="/Team:Tongji_China/Safety">Safety</A></SPAN></LI><LI onclick="return true"><A href="/Team:Tongji_China/Safety_P.A."><I>P. aeruginosa</I></A></LI><LI onclick="return true"><A href="/Team:Tongji_China/Light_control">Light control</A></LI><LI onclick="return true"><A href="/Team:Tongji_China/Animal_check_in">Animal check in</A></LI><LI onclick="return true"><SPAN class="menu-entry"><A href="/Team:Tongji_China/Human_Practices">HP</A></SPAN></LI><LI onclick="return true"><A href="/Team:Tongji_China/Public_Engagement">Education&amp;Engagement</A></LI><LI onclick="return true"><A href="/Team:Tongji_China/Integrated">Integrated</A></LI><LI onclick="return true"><SPAN class="menu-entry"><A href="/Team:Tongji_China/Team">Team</A></SPAN></LI><LI onclick="return true"><A href="/Team:Tongji_China/Team_members">Team members</A></LI><LI onclick="return true"><A href="/Team:Tongji_China/Collaborations">Collaborations</A></LI><LI onclick="return true"><A href="/Team:Tongji_China/Attributions">Attributions</A></LI><LI onclick="return true"><SPAN class="menu-entry">Achievement</SPAN></LI><LI onclick="return true"><A href="/Team:Tongji_China/Judging_form">Judging form</A></LI><LI onclick="return true"><A href="/Team:Tongji_China/Part">Parts</A></LI><LI onclick="return true"><A href="/Team:Tongji_China/InterLab">Interlab</A></LI><LI onclick="return true"><A href="/Team:Tongji_China/Capsule">Capsule</A></LI><TITLE>Design</TITLE><DIV class="background"><DIV class="content"><DIV class="title">
			Project
		</DIV><DIV class="title_2">
			Design
		</DIV><A name="PHASE1" style="color:black; text-decoration:none;">Here you can read how we establish, organize and execute our project of OCANDY:</A><DIV class="littletitle">PHASE 1. Dry lab filter</DIV>
			We use bioinformatic methods to filter our item antigens from SNVs (single nucleotide variations) which occur duing the development of cancer cells.
			For some SNVs will produce proteins that are not found in normal tissues and normal cells. These proteins are likely to activate and attract immune system to attack the tumor cells.
			According to making peptide windows and testing the MHC-I affinity, we can analyse the immunogenicity of our item antigens which are related to colon cancer, then we remould the plasmid of Pseudomonas aeruginosa, adding the gene of interest--antigen gene behind the signing peptide gene.
                        If you want to know more information about dry lab filter, please go to our <A href="https://2018.igem.org/Team:Tongji_China/Programme">dry lab_programme</A>.<DIV class="littletitle">PHASE 2. Plasmid construction</DIV>
			We want to create a new method to deliver the neoantigens into mammalian immune cells. After research, we choose Type III secretion system (T3SS), which is an amazing protein delivery tool. To make use of T3SS, we need to insert our antigen sequences into T3SS plasmid first. 
			<FONT color="#EEC778" face="charcoal" size="4"><I><B># de novo neoantigen gene synthesis</B></I></FONT>
			Because the antigen sequence is quite short, we cannot choose the common way of synthesizing double strand. So we synthesized the 5’-3’single strand and the 3’-5’ single strand with restriction site on both side, then take the method of annealing to pair two single strands into a double strand.<DIV style="text-align:center"><TABLE><TBODY><TR><TD align="center">
						Part name
					</TD><TD align="center">
						Antigen
					</TD><TD align="center">
						Sequence
					</TD></TR><TR><TD align="center">
						BBa_K2730001
					</TD><TD align="center">
						NY-ESO-A
					</TD><TD align="center">
                                        atgtcgttgttgatgctgatcacccagtgcccgttgtga
                                        </TD></TR><TR><TD align="center">
						BBa_K2730002
					</TD><TD align="center">
						NY-ESO-B
					</TD><TD align="center">
                                        atgcagttgtcgttgttgatgctgatcacctga
                                        </TD></TR><TR><TD align="center">
						BBa_K2730003
					</TD><TD align="center">
						0201
					</TD><TD align="center">
                                        atgttgcacttgtagggctcgtagccgccggcgtga
                                        </TD></TR><TR><TD align="center">
						BBa_K2730004
					</TD><TD align="center">
						0301A
					</TD><TD align="center">
                                        atgcacttgtagggctcgtagccgccggcgcggtga
                                        </TD></TR><TR><TD align="center">
						BBa_K2730005
					</TD><TD align="center">
						0301B
					</TD><TD align="center">
                                        atggcgatctcgacccgggacccgttgtcgaagtga
                                        </TD></TR><TR><TD align="center">
						BBa_K2730006
					</TD><TD align="center">
						0301C
					</TD><TD align="center">
                                        atgaagttgttgaagcggcaggcggaaggcaagtga
                                        </TD></TR></TBODY></TABLE></DIV><DIV class="instructionOfPicture">
				Table1.our antigen sequences
			</DIV><FONT color="#EEC778" face="charcoal" size="4"><B><I># T3SS &amp; neo-antigen plasmid construction</I></B></FONT>
			We use the attenuated P. aeruginosa strain PAK-J△9, in which 7 virulence-related genes (exoS/T/Y, ndk, xcpQ, lasI, rhlI) and one T3S suppressor gene (popN) are knocked out. The μA gene is mutated and it makes this strain a auxotrophic strain that cannot live without D-Glutamate.
			Antigens of interest are cloned and expressed on an Escherichia-Pseudomonas shuttle expression plasmid, which encodes T3S effector ExoS promoter with N-terminal ExoS1–54 signal sequence, followed by a FLAG tag and  multiple cloning site (MCS). Also on the vector, there is an intact spcS gene encoding the chaperone for the protein delivery.
			Antigens of interest can be fused in-frame utilizing the MCS and the fusion proteins can be detected by the FLAG tag. Under the guidance of ExoS1–54 secretion signal and the assistance of SpcS chaperone, the item antigens can be efficiently injected into host cells via the T3SS.
			<DIV class="instructionOfPicture">
				Figure3 T3SS-based protein delivery tool box
			</DIV><DIV class="littletitle">PHASE 3. Testing in vitro and in vivo</DIV>To figure out whether our filtered antigens are expressed and have strong immunogenicity, we conduct a series of experiments and use the Bayesian statistics to build the model, you can see them in <A href="https://2018.igem.org/Team:Tongji_China/WetLab">Wet Lab</A>.We use western blot to analyze whether our antigens can be successfully translated and secreted after inducing by the low Ca2+ environment or the host cell attachment.
We use immunocytochemistry method to figure out whether the antigens can be delivered into HELA cells in vitro.
We use immunohistochemistry method which is conducted on wild type mouse to detect whether the antigens can be delivered into host cells in intestine in vivo.
			<DIV class="littletitle">PHASE 4. Improvement</DIV><FONT color="#EEC778" face="charcoal" size="4"><I><B># Further experiment proof</B></I></FONT>
			However, our wet lab experiment is not completed, we just did the earlier stage work to prove that our system can work efficiently. There are many things we can do to further prove the feasibility of our system. We plan to conduct ELISA and ELISPOLT using wild type mice to analyze whether our system can cause immune system response after at least one month engineered P. aeruginosa immunizing. If some positive signals can be detected, we will then do the tumor cell challenge experiment to test whether the activated immune system can kill the tumor cells.
			<FONT color="#EEC778" face="charcoal" size="4"><B><I># Individual therapy</I></B></FONT>
			Now the method we use to filter item antigens in our project is catching many samples from patients with cancer to find some common SNVs and then get the antigen sequences. This method will have some effects but, as we all know, cancer cells' mutations vary among people. If we want to let a specific person's immune system to target to his tumor cells, the best way which was described in the concept &quot;neoantigen&quot; is to using the antigens that are filtered from this person's tumor cells' SNVs. But due to the limitation of the data resources, we cannot get individual SNVs data. So we need to get information from TCGA database. However, the programs we have constructed are also suitable for the individual SNVs data. We are looking forward to expanding our current results into individual therapy area.
			Also, maybe we can expand and enrich our data from people with cancer and optimize our algorithm to find more effective neo-antigens.
                        Before we get more patients' data, we build a <A href="https://2018.igem.org/Team:Tongji_China/Modeling">modeling</A> to help predict which mutantional site can be the best one for peptide making as a high immunogenic neoantigen for the certain patient. We think our method will contribute to the individual therapy. 
			<FONT color="#EEC778" face="charcoal" size="4"><I><B># Combination therapy</B></I></FONT>
			Neoantigen as an immunotherapy can also have some side effects, if the immunogenicity is very strong, and the targeting of cancer is not very powerful, maybe it will hurt normal tissue. So we need to match our method with other medicine of cancer therapy. On the one hand, it can reduce the new mutants to emerge, on the other hand, it will help us to alleviate the bad influence of our method.
			We need to test which medicine can be a good partner.<A href="https://2018.igem.org/Team:Tongji_China/Project"><FONT size="2">Return to the Project Overview</FONT></A></DIV></DIV><FOOTER class="footer-distributed"><DIV class="footer-all"><DIV class="footer-right"><H5 class="title-h4">Contact</H5><P class="my-content-p3">College of Life Science and Technology</P><P class="my-content-p3">Tongji university </P><P class="my-content-p3">No.1239, Yangpu District, Shanghai, China</P><P class="my-content-p3" style="word-wrap:break-word;">Email: tongjiigem2018@126.com</P></DIV></DIV><DIV id="footer-info" class="inner"><P class="my-content-p3" style="color: #777; text-align: center !important;">Copyright © 2018 Tongji_China</P></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>