<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CSU_Fort_Collins_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CSU Fort Collins/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrapper"><DIV class="collapsable_menu_control"> MENU ▤ </DIV><SPAN style="color:#D7D392; margin-left:45px"><B>CSU iGEM</B></SPAN><UL id="accordion" class="accordion"><LI class="menu_item" style="border-top: 1px solid black"><A href="https://2016.igem.org/Team:CSU_Fort_Collins">HOME </A></LI><LI class="menu_item"> TEAM
			</LI><LI><A href=" https://2016.igem.org/Team:CSU_Fort_Collins/Team"> Team   </A></LI><LI><A href="https://2016.igem.org/Team:CSU_Fort_Collins/Collaborations">  Collaborations </A></LI></UL><LI class="menu_item"> PROJECT  
			</LI><LI><A href="https://2016.igem.org/Team:CSU_Fort_Collins/Description">   Description </A></LI><LI><A href="https://2016.igem.org/Team:CSU_Fort_Collins/InterLab">   Interlab Study</A></LI><LI><A href="https://2016.igem.org/Team:CSU_Fort_Collins/Experiments"> Experiments </A></LI><LI><A href="https://2016.igem.org/Team:CSU_Fort_Collins/Results"> Results </A></LI><LI class="menu_item"> NOTEBOOK
			</LI><LI><A href="https://2016.igem.org/Team:CSU_Fort_Collins/NoteBook/JanThroughApr">Jan Through Apr</A></LI><LI><A href="https://2016.igem.org/Team:CSU_Fort_Collins/NoteBook/May">May</A></LI><LI><A href="https://2016.igem.org/Team:CSU_Fort_Collins/NoteBook/June">June</A></LI><LI><A href="https://2016.igem.org/Team:CSU_Fort_Collins/NoteBook/July"> July</A></LI><LI><A href="https://2016.igem.org/Team:CSU_Fort_Collins/NoteBook/Aug">August</A></LI><LI><A href="https://2016.igem.org/Team:CSU_Fort_Collins/NoteBook/Sep">September</A></LI><LI><A href="https://2016.igem.org/Team:CSU_Fort_Collins/NoteBook/Oct">October</A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:CSU_Fort_Collins/Protocols"> PROTOCOLS </A></LI><LI class="menu_item"> PARTS  
			</LI><LI><A href="https://2016.igem.org/Team:CSU_Fort_Collins/Parts">Parts </A></LI><LI><A href="https://2016.igem.org/Team:CSU_Fort_Collins/Basic_Part"> Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:CSU_Fort_Collins/Composite_Part">Composite Parts </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:CSU_Fort_Collins/Safety"> SAFETY </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:CSU_Fort_Collins/Attributions">ATTRIBUTIONS </A></LI><LI class="menu_item"> HUMAN PRACTICES 
			</LI><LI><A href="https://2016.igem.org/Team:CSU_Fort_Collins/HP/Silver">Silver </A></LI><LI><A href="https://2016.igem.org/Team:CSU_Fort_Collins/HP/Gold">Gold </A></LI></DIV><DIV class="content_wrapper"><H1> Project Description </H1><H2> Background </H2><P>Boolean logic is largely used in computer science but can be applied to various other disciplines. It regulates the occurrence of desired outcomes by stipulating necessary inputs. One operation from this system is AND. In its simplest form, an AND gate requires two simultaneous inputs in order for an output to be produced. An example of this operation is a driver moving a car forward. For this situation, the engine must be on AND the gearbox must be in drive before pushing the gas pedal will allow the car to move forward. </P><P>Adhering to Boolean logic, our goal is to create a modular genetic circuit in the cyanobacterium <I> Synechocystis </I> sp. PCC 6803 to optimize expression of a desired product. 
<A href="https://https://2016.igem.org/Team:CSU_Fort_Collins/HP/Gold" target="_blank">We decided to produce biofuels</A>. Additionally, we want to eliminate the need for expensive induction molecules such as IPTG. To accomplish these goals, we selected two conditions to implement in an AND gate and regulate gene expression: light and a quorum of cells. These conditions make use of natural mechanisms of <I> Synechocystis </I> and the quorum sensing mechanism from <I> Vibrio fischeri</I>. </P><P><I> Synechocystis </I> is a photosynthetic bacterium which has the ability to react diurnally (Labiosa 2006, Beck 2014), suggesting that it is able to change central metabolic flux following a circadian clock. In fact, studies using constant light found that about 2% to 9% of the organism's genes change following a circadian rhythm (Kucho 2005). Using data collected by Beck et al. about the relative abundance of transcripts during light-dark cycles, we identified potential light-induced promoters from <I>Synechocystis</I>. Allison Zimont assisted us in narrowing the options down to two promoters: sll0517 and ssl1633.</P><P> According to Merriam-Webster, a quorum is defined as ”the smallest number of people who must be present at a meeting in order for decisions to be made.” For our project, we are construing a quorum as the lowest density cell culture at which a product can be optimally produced. Quorum sensing is a communication mechanism bacteria may use to assess population density before allowing gene expression to occur.  The bacterium <I> Vibrio fischeri </I> uses quorum sensing to provide bioluminescence for the bobtail squid. Bioluminosity isn’t easily visible until the <I> V. fischeri </I> population reaches a certain density, so to conserve resources, the population will act as a group and drive gene expression only when a quorum is met (Waters and Bassler 2005). Similarly, we want to optimize production in <I> Synechocystis </I>. We chose to implement the <I> luxR-luxI </I> intergenic region from <I> V. fischeri </I> strain MJ1S because its promoters and RBS result in considerably high expression of bioluminescence (Bose et al. 2011). This system allows us to create an autoinduced system.</P><P>In order to create an AND gate, we adopted the split T7 RNA polymerase system (Shis and Bennett, 2013). They found that the gene sequence of polymerase from T7 bacteriophage can be split into two fragments. Upon expression, the N-terminal and C-terminal peptides recombine into a functional RNA polymerase.</P><P> Shis and Bennett, <I>PNAS</I>, 2013.
</P><H2> Choosing a product - BIOFUELS </H2><P>Our intention was for this AND gate to be modular, able to be used to express any product in <I>Synechocystis</I>. However, we needed to choose one product to focus on as our proof of concept. We were able to finalize this decision mainly due to what we learned in our <A href="https://2016.igem.org/Team:CSU_Fort_Collins/HP/Gold" target="_blank">outreach activity</A> this summer. Finding most people were willing to support synthetic biology when it pertained to creating biofuels over any medicinal or food products, we decided to choose biofuels as our target product.</P><P>Biofuel production is an increasingly growing field. According to the Colorado River Commission of Nevada, fossil fuels will run out by 2100. Currently, 80% energy needs are supplied by fossil fuels (Demirbas and Demirbas, 2011). We decided that our organism would be perfect for biofuel production. <I>Synechocystis</I> fits into the 3rd generation of biofuels; it is not a land crop, so it will not take away valuable land for food production such as fuels made from corn or sorghum. In addition, our organism naturally produces heptadecane which can be used directly as a biofuel. The relevant genes are <I>sllr0208</I> and <I>sllr0209</I>. </P><P> Wang et al., <I>Biotechnology for Biofuels</I>, 2013.</P><P>A previous iGEM team, <A href="https://2011.igem.org/Team:Utah_State/Project" target="_blank">Utah State 2011</A>, wanted to make biofuels in cyanobacteria. However, they only looked at different promoters, trying to find a stronger one for production. We decided to approach our project differently aside from building the logic gate. Wang et al. 2013 found that by introducing another copy of the alkane genes into the organism, they were able to obtain 1.1% heptadecane in dry cell weight compared to wild type 0.14%.</P><H2> The CyanoLogic Gate </H2><P> For the first portion of our AND gate, we placed expression of the N-terminal T7 RNA polymerase under the control of the<I> luxR-luxI</I> quorum sensing system. At the second portion, we placed the C-terminal T7 RNA polymerase under the control of a light induced promoter. For all the RBS, we chose SigA, as it is native to our organism. Finally, we placed the alkene genes (<I>sllr0208</I> and <I>sllr0209</I>) behind the T7 promoter.</P><H2> Improving on a Previous BioBrick and Transforming into <I>Synechocystis</I></H2><P>The challenge in working with <I>Synechocystis</I> is that very few plasmids replicate in the organism. Instead, researchers take advantage of the fact that they are naturally competent and have very active recombinases. This allows us to insert new DNA sequences into the genome by homologous recombination.</P><P>A previous team, <A href="https://2012.igem.org/Team:UC_Chile" target="_blank">UC Chile 2012</A>, created a transformation vector following this very concept. A gene of interest can be recombined into a chosen neutral site along with a Kanamycin resistance gene. However, we could not use it because <A href="http://parts.igem.org/Part:BBa_K743006">the part 2012</A> was not released due to an illegal EcoRI restriction site. So, we set out to create our own transformation vector.</P><P>Based on a transformation protocol devised by Cheah et al. 2012, we improved on the previous recombination vector. The neutral region we chose (slr0168), does not have any illegal restriction sites. Also, the transformation method creates a markerless system which allows additional cloning to be much easier, eliminating the need for introducing another selection marker. We first needed to insert <I>mazF</I>, a protein synthesis inhibitor, and the Kanamycin resistance gene under the control of a nickel inducible promoter. Then, we transform the <I>Synechocystis</I> with our CyanoLogic gate housed in the transformation vector. As we plate the cyanobacteria on agar plates with nickel, they are placed under selective pressure and segregate across all copies of their chromosome. The process can take up to three weeks. At the end, CyanoLogic is in every chromosome while mazF and the Kanamycin resistance gene are eliminated. The parts needed for the two step transformation for genome integration are <A href="http://parts.igem.org/Part:BBa_K1945001
">BBa_K1945001</A> and <A href="http://parts.igem.org/Part:BBa_K1945002
">BBa_K1945002</A>. </P><H2> Works Cited </H2><P> Beck C, Hertel S, Rediger A, Lehmann R, Wiegard A, Kölsch A, Heilmann B, Georg J, Hess WR, Axmann IM. 2014. Daily expression pattern of protein-encoding genes and small noncoding RNAs in Synechocystis sp. strain PCC 6803. Appl Environ Microbiol 80:5195–5206. 10.1128/AEM.01086-14. </P><P> Bose, J. L., Wollenberg, M. S., Colton, D. M., Mandel, M. J., Septer, A. N., Dunn, A. K., &amp; Stabb, E. V. (2011). Contribution of Rapid Evolution of the luxR-luxI Intergenic Region to the Diverse Bioluminescence Outputs of Vibrio fischeri Strains Isolated from Different Environments. Applied and Environmental Microbiology, 77(7), 2445-2457. doi:10.1128/aem.02643-10 </P><P>Cheah, Y. E., Albers, S. C., &amp; Peebles, C. A. (2012). A novel counter-selection method for markerless genetic modification inSynechocystissp. PCC 6803. Biotechnology Progress, 29(1), 23-30. doi:10.1002/btpr.1661 </P><P>The Colorado River Commission of Nevada. &quot;World Fossil Fuel Reserves and Projected Depletion.&quot; (2002): n. pag. Web. 28 Apr. 2016.</P><P>Demirbas, A., and Demirbas, M.F. “Importance of algae oil as a source of biodiesel.” “Energy Conversion and Management.” 2011. 52:163-170.</P><P> Kucho K, Okamoto K, Tsuchiya Y, Nomura S, Nango M, Kanehisa M,Ishiura M. 2005. Global analysis of circadian expression in the cyanobacterium Synechocystis sp. strain PCC 6803. J. Bacteriol. 187:2190 –2199. http://dx.doi.org/10.1128/JB.187.6.2190-2199.2005. </P><P> Labiosa, R. G., Arrigo, K. R., Tu, C. J., Bhaya, D., Bay, S., Grossman, A. R. and Shrager, J. (2006), EXAMINATION OF DIEL CHANGES IN GLOBAL TRANSCRIPT ACCUMULATION IN SYNECHOCYSTIS (CYANOBACTERIA). Journal of Phycology, 42: 622–636. doi: 10.1111/j.1529-8817.2006.00217.x </P><P> Shis, D. L., &amp; Bennett, M. R. (2013). Library of synthetic transcriptional AND gates built with split T7 RNA polymerase mutants. Proceedings of the National Academy of Sciences, 110(13), 5028-5033. doi:10.1073/pnas.1220157110 </P><P> Wang et al.: Engineering cyanobacteria to improve
photosynthetic production of alka(e)nes. Biotechnology for Biofuels 2013
6:69. </P><P> Waters, C. M., &amp; Bassler, B. L. (2005). QUORUM SENSING: Cell-to-Cell Communication in Bacteria. Annual Review of Cell and Developmental Biology Annu. Rev. Cell Dev. Biol., 21(1), 319-346. doi:10.1146/annurev.cellbio.21.012704.131001 </P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>