<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Potsdam_Background skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Potsdam/Background</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>No Sidebar - Escape Velocity by HTML5 UP</TITLE></P><DIV id="page-wrapper-bg"><DIV id="page-wrapper"><DIV id="header-wrapper" class="wrapper"><DIV id="header"><DIV id="logo"><H1><A href="https://2017.igem.org/Team:Potsdam">IGEM Potsdam</A></H1><P>The world's first enzyme dating service </P></DIV><NAV id="nav"><UL><LI class="current"><A href="https://2017.igem.org/Team:Potsdam/Homepage">Home</A></LI><LI><A href="https://2017.igem.org/wiki/index.php?title=Team%3APotsdam%2FTeam">Team</A></LI><LI><A href="https://2017.igem.org/wiki/index.php?title=2017.igem.org%2FWeIntroduceOurselves">We introduce ourselves</A></LI><LI><A href="https://2017.igem.org/wiki/index.php?title=Team%3APotsdam%2Fmotivation">Motivation for participation</A></LI><LI><A href="https://2017.igem.org/wiki/index.php?title=Team%3APotsdam%2FCollabos"> Collaborations</A></LI><LI><A href="https://2017.igem.org/wiki/index.php?title=Team%3APotsdam%2FModelling">Eindhoven</A></LI><LI><A href="https://2017.igem.org/wiki/index.php?title=Team%3APotsdam%2FiGem-goes-green">iGem goes green </A></LI><LI><A href="https://2017.igem.org/wiki/index.php?title=Team%3APotsdam%2Fskype">Skype conversations</A></LI><LI><A href="https://2017.igem.org/Team:Potsdam/collabos/berlin">Berlin</A></LI><LI><A href="https://2017.igem.org/wiki/index.php?title=Team%3APotsdam%2Fpostcards">Postcards</A></LI><LI><A href="https://2017.igem.org/wiki/index.php?title=Team%3APotsdam%2Fmeetup">German-Wide Meetup</A></LI></UL><LI><A href="https://2017.igem.org/Team:Potsdam/Description">Our project</A></LI><LI><A href="https://2017.igem.org/wiki/index.php?title=Team%3APotsdam%2FBackground">Background facts and information</A></LI><LI><A href="https://2017.igem.org/Http://2017.igem.org/2017.igem.org/Team:Potsdam/Abstract">Abstract</A></LI><LI><A href="https://2017.igem.org/wiki/index.php?title=Team%3APotsdam%2FDesign">Project Design</A></LI><LI><A href="https://2017.igem.org/Team:Potsdam/Model">Modelling</A></LI><LI><A href="https://2017.igem.org/wiki/index.php?title=Team%3APotsdam%2FProtocols">Research work &amp; protocols</A></LI><LI><A href="https://2017.igem.org/Team:Potsdam/Notebook">lab book</A></LI><LI><A href="https://2017.igem.org/wiki/index.php?title=Team%3APotsdam%2FResults">Results</A></LI><LI><A href="https://2017.igem.org/wiki/index.php?title=Team%3APotsdam%2FParts">Parts</A></LI><LI><A href="https://2017.igem.org/wiki/index.php?title=Team%3APotsdam%2Foutlook">Outlook</A></LI><LI><A href="https://2017.igem.org/wiki/index.php?title=Team%3APotsdam%2FVision">our vision</A></LI><LI><A href="https://2017.igem.org/Team:Potsdam/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/wiki/index.php?title=Team%3APotsdam%2FHumanPractise">Human Practices</A></LI><LI><A href="https://2017.igem.org/wiki/index.php?title=Team%3APotsdam%2Fcrowdfunding">Crowdfunding</A></LI><LI><A href="https://2017.igem.org/Team:Potsdam/Practice/Klas">KLAS</A></LI><LI><A href="https://2017.igem.org/wiki/index.php?title=Team%3APotsdam%2FLabmesse">Laboratory exhibition</A></LI><LI><A href="https://2017.igem.org/wiki/index.php?title=Team%3APotsdam%2FPTDW">Potsdam Day of Science</A></LI><LI><A href="https://2017.igem.org/wiki/index.php?title=Team%3APotsdam%2Fschools">school presentations</A></LI><LI><A href="https://2017.igem.org/Team:Potsdam/MarchForScience">March for Science</A></LI><LI><A href="https://2017.igem.org/wiki/index.php?title=Team%3APotsdam%2Fawards">Judging</A></LI><LI><A href="https://2017.igem.org/Team:Potsdam/Attributions">Attributions - Bronze</A></LI><LI><A href="https://2017.igem.org/Team:Potsdam/Contribution">Contribution - Bronze</A></LI><LI><A href="https://2017.igem.org/Team:Potsdam/Collaborations">Collaborations - Silver</A></LI><LI><A href="https://2017.igem.org/Team:Potsdam/HP/Silver">Human practices - Silver</A></LI><LI><A href="https://2017.igem.org/Team:Potsdam/Model">Modelling - Gold</A></LI><LI><A href="https://2017.igem.org/Team:Potsdam/Demonstrate">Demonstrate - Gold</A></LI><LI><A href="https://2017.igem.org/Team:Potsdam/HP/Gold_Integrated">Human Practices Integrated - Gold</A></LI><LI><A href="https://2017.igem.org/wiki/index.php?title=Team%3APotsdam%2FSpo">Sponsors</A></LI><LI><A href="https://2017.igem.org/wiki/index.php?title=Team%3APotsdam%2FImpressum">Contact</A></LI></NAV></DIV></DIV><DIV class="wrapper style2"><DIV class="title">
            Background facts and information
        </DIV><DIV id="main" class="container"><DIV id="content"><DIV align="justify">
The field of synthetic biology combines different disciplines like molecular biology, engineering, mathematics and physics to improve the knowledge of existing biological systems or to design and manipulate genes, pathways and entire genomes of organisms, for them to fulfill a certain purpose <SUP> [1] </SUP>. Synthetic Biology is applied in various fields and disciplines. For biomedical applications it can be used to design bacteria that specifically attack cancer cells or to create bacteriophages that are able to infect and eliminate multiresistant bacteria, that cannot be any longer be treated by antibiotics <SUP> [2] </SUP>. Another application is the storage of information in microbial genomes, such as images <SUP>[3]</SUP>. Next to these applications, bacteria and other microorganisms are widely used as biological factories of chemical substances of interest. At this point many efforts have been made to enhance the yield of the synthesized products. One approach deals with the principle of metabolic channeling. By spatial organization of enzymes belonging to a certain metabolic pathway the yield of this pathway may be increased. This was for example investigated for the indole-3-acetic acid pathway using a transcription activator-like effector (TALE)-based scaffold system in E.coli <SUP> [4]</SUP>. 

To investigate whether applying metabolic channeling enhances the yield of the indole-3-acetic acid pathway we devised two different approaches next to the one of Zhu et al.<SUP>[3]</SUP> The first one utilizes the DNA-binding property of the dCas9-protein to put the enzymes next to each other on a DNA-scaffold. This will be done in <I>E.coli</I>, whereas our second approach will be implemented in <I> S.cerevisiae </I>. This second approach deals with liquid-liquid phase separation (LLPS). In this process membraneless organelles are formed, induced by variable domains of the Ddx4 protein. We want to investigate whether fusing the enzymes of the pathway with the Ddx4 protein enable droplet formation and thereby induces metabolic channeling.
</DIV><SUP> [1] </SUP> Systems and Synthetic Biology (Hg.): Systems and Synthetic Biology. Springer Netherlands. Online verfügbar unter <A href="https://link.springer.com/journal/11693">ps://link.springer.com/journal/11693 </A>, zuletzt geprüft am 01.10.2017. <SUP> [2] </SUP> Ruder, Warren C.; Lu, Ting; Collins, James J. (2011): Synthetic biology moving into the clinic. In: Science (New York, N.Y.) 333 (6047), S. 1248–1252. DOI: 10.1126/science.1206843. <SUP> [3] </SUP> Shipman, Seth L.; Nivala, Jeff; Macklis, Jeffrey D.; Church, George M. (2017): CRISPR-Cas encoding of a digital movie into the genomes of a population of living bacteria. In: Nature 547 (7663), S. 345–349. DOI: 10.1038/nature23017. <SUP> [4] </SUP> Zhu, Ling-Yun; Qiu, Xin-Yuan; Wu, Xiao-Min; Zhang, Yuan; Zhu, Qian-Hui; Fan, Dong-Yu et al. (2016): Spatial organization of heterologous metabolic system in vivo based on TALE. In: Scientific reports 6, S. 26065. DOI: 10.1038/srep26065. </DIV></DIV></DIV><DIV class="wrapper style3"><DIV class="title">
            Our three main methods
        </DIV><DIV id="main" class="container"><DIV id="content"><DIV align="justify"><B>1. Gibson Assemblay </B>
To design the needed constructs, we are using different cloning techniques. Regarding the first approach the construction of the plasmid containing the needed proteins and guide RNAs for the scaffolding is conducted by <B>Gibson Assembly </B>. Gibson Assembly is an isothermal, single reaction method to assemble multiple overlapping DNA molecules. By this a seamless construction of synthetic genes, pathways and entire genomes is possible. A precondition for the seamless combination of two DNA fragments are identical terminal DNA sequences. The overlapping DNA molecules are added to a reaction mixture containing the needed enzymes and are afterwards incubated at 50 °C. A T5 exonuclease removes the terminal nucleotides at the 5´end and creates sticky ends. This way formed complementary overhangs of the DNA fragments anneal to each other. A DNA Ligase seals the nicks while the gaps are refilled by a DNA Polymerase. During the 50 °C Incubation the T5 exonuclease is heat inactivated. Since there is no competition between the T5 exonuclease and DNA polymerase reaction, both enzymes can be simultaneously active <SUP> [1]</SUP>.
<B>2. 3 A Assembly</B>
Meanwhile <B> 3A Assembly </B> is utilized to enhance the number of target sites on a high copy plasmid. The term 3A Assembly is an abbreviation for 3 antibiotic assembly. It uses three different inputs for the assembly and the antibiotic resistances for the selection of correct assembly products. The needed inputs are the up- and downstream part which should be combined and the vector, in which the combined parts should be inserted. The antibiotic resistances of the three inputs need to be different. Each part/ input consists of a BioBrick pre-and suffix. The BioBrick prefix shows an EcoRI and a XbaI site, whereas the the BioBrick suffix is composed of a SpeI and PstI site.  The three different inputs are in the first step digested by different restriction enzymes. The upstream part is digested using EcoRI and SpeI, the downstream part using XbaI and PstI and the vector using EcoRI and PstI. Because the digestion with XbaI and SpeI causes identical overhangs, the upstream and the downstream parts are ligated and form a XbaI-SpeI gap. By ligation into the opened vector backbone a plasmid is formed <SUP> [2]</SUP>. This Plasmid contains the two parts in a defined order and can be selected by its antibiotic resistance. In a next assembly cycle the product can be used as one of the inputs and another DNA fragment as second input. By conducting several 3A assembly cycles more DNA fragments can be added stepwise to the construct. DNA fragments showing the BioBrick prefix and suffix and no further EcoRI, SpeI, XbaI or PstI site in their sequence are belonging to the BioBrick Standard and can always be used for 3A Assembly <SUP>[2]</SUP>.
<B>3. TAR</B>
The needed constructs of the second approach are created using <B>TAR </B>. The term TAR is an abbreviation for transformation associated recombination occurring in yeast <I>Saccharomyces cerevisiae</I>. The process is similar to the gap repair during homologous recombination. This technique can be used for the selectively recovery of chromosome segments up to a size of 250 kb from complex genomes and is a useful tool for the characterization of genome variations and gene functions. The needed TAR vector consists in opened state of two terminal gene-specific target sequences, so called hooks, a yeast selectable marker and a centromere sequence. A further requirement is that the cloned DNA shows at least one autonomously replication sequence (ARS). The TAR vector and the DNA carrying the DNA sequence of interest are transformed into yeast spheroplasts. By recombination between the hooks and the homologous sequences of the DNA fragment of interest a circular yeast artificial chromosome (YAC) is formed. The yeast selectable marker allows selection while the ARS of the cloned DNA leads to replication of the YAC <SUP> [3]</SUP>.
</DIV><SUP>[1]</SUP> Gibson, Daniel G.; Young, Lei; Chuang, Ray-Yuan; Venter, J. Craig; Hutchison, Clyde A.; Smith, Hamilton O. (2009): Enzymatic assembly of DNA molecules up to several hundred kilobases. In: Nature methods 6 (5), S. 343–345. DOI: 10.1038/nmeth.1318. <SUP>[2]</SUP> Shetty, Reshma; Lizarazo, Meagan; Rettberg, Randy; Knight, Thomas F. (2011): Assembly of BioBrick standard biological parts using three antibiotic assembly. In: Methods in enzymology 498, S. 311–326. DOI: 10.1016/B978-0-12-385120-8.00013-9. <SUP>[3]</SUP>Kouprina, Natalay; Larionov, Vladimir (2006): TAR cloning. Insights into gene function, long-range haplotypes and genome structure and evolution. In: Nature reviews. Genetics 7 (10), S. 805–812. DOI: 10.1038/nrg1943. </DIV></DIV></DIV><DIV class="wrapper style1"><DIV class="title">
            What is Auxin?
        </DIV><DIV id="main" class="container"><DIV id="content"><DIV align="justify">
1881, Charles Darwin and his son Francis examined coleoptiles (the protective sheath covering the emerging shoot in monocotyledons) exposed to unidirectional light. Consequential, they concluded the existence of a signalling molecule directing their bending <SUP>[1]</SUP>. This small molecule named auxin has much more functions in the plant than just regulating organ bending in response to light and gravity: it mediates growth reactions of plants to adapt to current environmental conditions, controls genetically pre-programmed physiological processes such as leaf and other organ formation and many more<SUP> [1] [2]</SUP>.

There exist different auxins but the auxin which is mainly produced is Indole-3-acetic acid (IAA).

IAA is known to be synthesized de novo using tryptophan (Trp) as a precursor or using a Trp-independent pathway <SUP>[3]</SUP>.The Trp-dependent pathway is shown in the figure below<SUP>[4]</SUP>. The catalysing enzymes for the first step is IAAM and for the second step it is IAAH <SUP>[5]</SUP>. These are the two enzymes we were working with to approach metabolic channeling.
</DIV><SUP> [1] </SUP> Zažímalová, Eva; Petrášek, Jan; Benková, Eva (2014): Auxin and Its Role in Plant Development. Vienna: Springer Vienna. <SUP> [2] </SUP> Taiz, Lincoln (Hg.) (2015): Plant physiology and development. 6. ed. Sunderland Mass.: Sinauer. <SUP> [3] </SUP> Zhao, Yunde (2012): Auxin biosynthesis: a simple two-step pathway converts tryptophan to indole-3-acetic acid in plants. In: Molecular plant 5 (2), S. 334–338. DOI: 10.1093/mp/ssr104. <SUP> [4] </SUP> Mashiguchi, Kiyoshi; Tanaka, Keita; Sakai, Tatsuya; Sugawara, Satoko; Kawaide, Hiroshi; Natsume, Masahiro et al. (2011): The main auxin biosynthesis pathway in Arabidopsis. In: Proceedings of the National Academy of Sciences of the United States of America 108 (45), S. 18512–18517. DOI: 10.1073/pnas.1108434108. <SUP> [5] </SUP> Won, Christina; Shen, Xiangling; Mashiguchi, Kiyoshi; Zheng, Zuyu; Dai, Xinhua; Cheng, Youfa et al. (2011): Conversion of tryptophan to indole-3-acetic acid by TRYPTOPHAN AMINOTRANSFERASES OF ARABIDOPSIS and YUCCAs in Arabidopsis. In: Proceedings of the National Academy of Sciences of the United States of America 108 (45), S. 18518–18523. DOI: 10.1073/pnas.1108436108. </DIV></DIV></DIV><DIV class="wrapper style2"><DIV class="title">
           Comparison E.coli and yeast
        </DIV><DIV id="main" class="container"><DIV id="content"><TABLE class="tg"><TBODY><TR><TH class="tg-031e" align="center"><B>E.Coli</B></TH><TH class="tg-031e"><B>Yeast</B></TH></TR><TR><TD class="tg-031e">Domain</TD><TD class="tg-031e" align="center">Bacteria</TD><TD class="tg-031e" align="center">Eucaryote</TD></TR><TR><TD class="tg-031e">Diameter</TD><TD class="tg-031e" align="center">0.25-1.0µm</TD><TD class="tg-031e" align="center">5-10µm</TD></TR><TR><TD class="tg-031e">Cell growth</TD><TD class="tg-031e" align="center">very fast</TD><TD class="tg-031e" align="center">medium</TD></TR><TR><TD class="tg-031e">Complexity of growth medium</TD><TD class="tg-031e" align="center">minimum</TD><TD class="tg-031e" align="center">little more complex</TD></TR><TR><TD class="tg-031e">Cost og growth medium</TD><TD class="tg-031e" align="center">low</TD><TD class="tg-031e" align="center">little higher</TD></TR><TR><TD class="tg-031e">Expression level</TD><TD class="tg-031e" align="center">high</TD><TD class="tg-031e" align="center">low - high</TD></TR><TR><TD class="tg-031e">Optimal growth temperature</TD><TD class="tg-031e" align="center">37°C</TD><TD class="tg-031e" align="center">30 - 35°C</TD></TR><TR><TD class="tg-031e">Handling</TD><TD class="tg-031e" align="center">easy, because of the possibilty to work with antibiotic resistances</TD><TD class="tg-031e" align="center">a little more difficult because everything has to be completely sterile</TD></TR><TR><TD class="tg-031e">Posttranslational modifications</TD><TD class="tg-031e" align="center">none</TD></TR></TBODY></TABLE>
source:  Fernandez, J.M. &amp; Hoeffler, J.P. (1999): Gene Expression Systems. Using nature for the art of expression. In: Academic Press.  Feldmann, Horst (2010): Yeast. Molecular and cell biology. Weinheim: Wiley-Blackwell.
</DIV></DIV></DIV><DIV class="wrapper style3"><DIV class="title">
           Metabolic Channeling – state of the art
        </DIV><DIV id="main" class="container"><DIV id="content"><DIV align="justify">
Metabolic engineering holds promise as an alternative to synthetic chemistry for cheaply and renewably producing molecules of value. Even though the construction of novel pathways out of enzymes from different organisms proves to be a powerful strategy for synthesizing a variety of compounds, achieving commercially viable productivity remains a challenge. Engineered metabolic pathways are constructed from enzymes that are heterologous to the production host often suffer from low enzymatic activities and flux imbalances, unintended and difficult-to-characterize interactions between synthetic pathways as well as the cellular environment of the host organisms. One probable reason for this is, that they typically lack the regulatory mechanisms characteristic of natural metabolism. A potential solution to this could be to organize enzymes of a synthetic pathway on synthetic scaffolds.

The small molecules of the cellular metabolism can, in general, rapidly diffuse throughout the whole cytoplasm. Exceptions from this are the products of an enzyme active site, which are locally processed by a subsequent active site. This is known as direct channeling. Direct channeling relies on the formation of protein tunnels. Those tunnels are connecting the active sites of the reactants. They help to prevent the metabolic intermediates from diffusing away from each other. <SUP> [2] </SUP>.
<I>Figure 1: The direct channeling Mechanism. The intermediate is funneled from enzyme E1 to enzyme E2 by means of a protein tunnel that connects the active sites of E1 and E2, thus preventing the intermediate from diffusing away. The intermediate is then turned into a product. </I>
There are also two diverse types of intermediate channeling in a two-step metabolic pathway next to direct channeling: proximity channeling and enzyme clustering. For proximity channeling two enzymes are needed, that are positioned close to each other. The intermediate product of the first enzyme is processed by a second enzyme before it can escape due to its drift by diffusion. Proximity channeling therefore occurs when enzymes are colocalized on a scaffold. The processing of a diffusing substrate only becomes likely in the case that the substrate approaches the catalytic site of an enzyme within a radius of about 0.1 - 1 nm. This can be problematic, because the intermediate that is produced by the first member of an enzyme pair is unlikely to be processed by the second member even if the active sites of the two enzymes are just as much as about 10 nm apart.  
<I>Figure 2: The mechanism of proximity channeling. E1 and E2 are positioned near enough to each other such that the intermediate produced by E1 is processed by E2 before it can escape by diffusion. No actual channel is needed. Close proximity of the enzymes is crucial for proximity channeling.</I>
An alternative way to achieve metabolic channeling is the assembly of multiple copies of both upstream and downstream enzymes into a functional cocluster (‘agglomerate’). The probability that the intermediate will be processed by one of the many downstream enzymes in the agglomerate can be very high. For example, there could be two consecutive enzymes in a pathway present at high concentration in the same region of space, leading to a high processing probability. This way coclustering of multiple enzymes into compact agglomerates accelerates the processing of intermediates, yielding the same efficiency benefits as direct channeling.
<I>Figure 3: The mechanism of enzyme clustering. Once E1 produces an intermediate molecule, even though the probability of the intermediate being processed by any individual E2 enzyme is low, the probability that the intermediate will be processed by one of the many E2 enzymes in the agglomerate can be high.</I>
Another attempt to increase the effective concentration of the components of a pathway of interest could be the construction of synthetic protein scaffolds which spatially recruit metabolic enzymes in a designable manner. 

On the scaffold the interaction domains from metazoan signaling proteins are localized, recruiting the pathway enzymes specifically. These enzymes are tagged with their related peptide ligands. This increases the effective concentrations of metabolic intermediates, while preventing their accumulation to toxic levels. This strategy was inspired by natural biosynthetic machines (e.g., polyketide synthases, fatty acid synthases and non-ribosomal peptide synthases) that produce small molecules by channeling intermediates iteratively through assembly lines of catalytic activities. Although an artificially created system is much simpler than these elegant natural systems, the modular design of the scaffold strategy should prove to be both programmable and generalizable.

By encapsulating the pathway components within protein shells a more direct way of compartmentalizing would be achieved. Here, the pathway enzymes get physically encapsulated into distinct compartments, thereby limiting cross-talk between engineered pathways and the cellular milieu. The most critical step of reaching an ideal compartmentalizing is the evolution of pores in the protein shells, which specifically limit the diffusion of the intermediate.

A further possibility to reach metabolic channeling is the use of plasmid scaffolds. The 2010 Slovenian iGEM team constructed a plasmid scaffold on which the violacein biosynthetic enzymes (VioA-E) were co-assembled and tethered to zinc finger DNA binding domains. The amount of deoxychromoviridans, the products of the unwanted side reaction, was significantly reduced, while there was an approximately six-fold increase of the yield of the violacein biosynthesis. An advantage of plasmid scaffolds is that they accommodate many interaction motifs and variable length linkers without solubility issues. The disadvantage of this method however is that to be able to achieve this, the enzymes must be significantly modified with multiple zincfinger domains (usually 3–4 domains with a total addition of 90-120 amino acids). A problem is, that the maximal concentration of DNA scaffolds in cell is limited by the maximal plasmid copy number (roughly 500 per cell).

By incorporating a transcription activator-like effector (TALE)-based scaffold system into an Escherichia coli chassis, it is also possible to accelerate a heterologous metabolic system. The TALEs bind to an artificial DNA scaffold. The validation of the effects of such systems was done by the heterologous production of indole-3-acetic acid (IAA) after the integration of TALE-fused IAA biosynthetic enzymes in E. coli. Compared to zinc finger domains, this shows to be a simpler design with a higher specificity and lower toxicity. To form in vivo clusters of multi-enzymes the TALEN technique seems to be more efficient based on TALE-fused enzymes and their corresponding DNA scaffolds. It has been demonstrated that the TALE–DNA scaffold system can be efficiently used to cluster and order TALE-fused proteins, as well as enhance the rates of heterologous metabolic pathways in prokaryotic chassis.  

Another possibility is an in vivo RNA scaffold that is the target of synthetic proteins that are linked to specific RNA binding domains. This way a co-localization of the proteins in living E. coli cells can be achieved. The synthetic RNA strands consist of polymerization domains and aptamers. The presence of the polymerization domains enable the formation of a macromolecular structure out of the single RNA strands. By applying a certain orientation and localization of the enzymes on the scaffold the diffusion of the substrate is limited and the pathway flux is increased.
</DIV>
Michele Castellana et al., “Enzyme clustering accelerates processing of intermediates through metabolic channeling”, Nat Biotechnol. 2014 Oct; 32(10), 1011–1018 
John E Dueber et al., “Synthetic protein scaffolds provide modular control over metabolic flux”, Nature Biotechnology 27 (2009), 753 - 759 
Lee H et al., “Spatial organization of enzymes for metabolic engineering”, Metab Eng. 2012 May;14(3):242-51 
Gairik Sachdeva et al., “In vivo co-localization of enzymes on RNA scaffolds increases metabolic production in a geometrically dependent manner”, Nucleic Acids Research, 2014, Vol. 42, No. 14, 9493–9503 </DIV></DIV></DIV><DIV class="wrapper style1"><DIV class="title">
          Liquid-liquid phase separation – state of the art
        </DIV><DIV id="main" class="container"><DIV id="content"><DIV align="justify">
Phase transitions play an important role in many processes. The most well-known example of phase transitions are those involving water, ranging from vapor, over condensed water up to solid ice phases. Phase transitions however, also play a key role for proteins, lipids and other macromolecules. Usually an in vitro formation of a liquid droplet phase can be detected in concentrated protein solutions. The physics describing this process are similar to the ones describing the condensation of dew droplets from concentrated water vapor. Phase transitions may be represented graphically by a phase diagram, in which the different regions of the diagram indicate the distinct phase(s) which are favored under given conditions, such as temperature and molecular concentration. A schematic phase diagram is shown here, with two regions: The upper region is representing a single phase of soluble molecules and the lower region is representing a two-phase coexistence of droplets and soluble molecules. Although the phase diagram typically shows just two parameters, this is just a slice of a multi-dimensional phase diagram that can include other parameters such as pH, or osmolarity.
<DIV align="center"><I>Figure 4: Schematic phase diagram.</I></DIV>Eukaryotic cells possess non-membrane-bound organelles, a lot of them show an enhanced RNA and protein concentration and play roles in mediating RNA-protein complexes. Such organelles are generically referred to as ribonucleoprotein (RNP) granules, e.g. the nucleolus. Intrinsically disordered regions (IDRs) of RNA-binding proteins can phase separate. Those IDR phase separations are promoted by RNA, crowding agents, and low salts. A stabilization of the phase-separated droplets is achieved by the formation of amyloid-like fibers over time. In this process, IDR elements become less dynamic, which goes along with formation of fibrous structures.  

It has been demonstrated that a single protein constituent can reversibly form membraneless organelles, both in vitro and in cells. The mainly proteinaceaous interior of membraneless organelles is partially excluding the bulk aqueous phase. Such organelles act like liquid droplets, being spherical with an internal mobility. The organelles enable a different solvent environment that causes an enhanced concentration of single-stranded RNA, while excluding double-stranded RNA. Phase separation of disordered proteins is a general mechanism for the regulated formation of membraneless organelles.

The disordered tails of Ddx4 proteins form phase-separated organelles both in live cells and in vitro. The Ddx4 proteins play a vital role for the assembly and maintenance of the related nuage in mammals, P-granules in worms, and pole plasm and polar granules in flies. In addition to a central DEAD-box RNA helicase domain that utilizes ATP to unwind short RNA duplexes, Ddx4 has extended N and C termini that are estimated to be intrinsically disordered. The Ddx4 protein shows conserved sequence properties which are responsible for the droplet formation: 8-10 blocks of changing net charge, while the positively charged regions show an over-expresion of FG, GF, RG, and GR motifs. Temperature, ionic strength, arginine methylation and splicing are influencing the patterned electrostatic interactions which are stabilizing the droplets. As already mentioned membraneless organelles, such as the Ddx4 droplets, are providing a different environment compared to the rest of the aqueous cellular interior, which is resulting in the increased concentration of certain biomolecules.

The liquid-like structures formed by these processes would be distinct from traditional macromolecular assemblies in at least two important ways: In contrast to approved multicomponent complexes (e.g. the ribosome) the stereochemical definition of a granule formed by LLPS would not be possible. The elements would be randomly organized and dynamically rearranging. Because the phase-separated droplets are &gt;90% water by mass (based on protein concentrations of ~300 mM), macromolecules could enter, diffuse within, and exit easily. These two properties allow biomolecules to be concentrated inside of granules but still be able to diffuse freely and rapidly.  
</DIV><DIV align="center"><I>Figure 5: Formation and Maturation of phase-separated liquid droplets by RNA-binding proteins. Intrinsically disordered regions of RNA-binding proteins (e.g. Ddx4) can phase separate. Phase-separated droplets stabilize over time by formation of amyloid-like fibers (red regions in lower droplet). RRM: RNA recognition motif.</I></DIV>
Clifford P. Brangwynne, “Phase transitions and size scaling of membrane-less organelles”, J. Cell Biol. Vol. 203 No. 6 875–881 
Yuan Lin et al., “Formation and Maturation of Phase-Separated Liquid Droplets by RNA-Binding Proteins”, October 15, 2015, Molecular Cell 60, 208–219 
Timothy J. Nott et al., “Phase Transition of a Disordered Nuage Protein Generates Environmentally Responsive Membraneless Organelles”, Molecular Cell 57, March 5, 2015, 936–947 </DIV></DIV></DIV><DIV id="footer-wrapper" class="wrapper"><DIV class="title">
            Did you see this?
        </DIV><DIV class="boxWrapper"><DIV class="leftBox"><DIV class="topBox"><A href="https://2017.igem.org/Http://2017.igem.org/2017.igem.org/Team:Potsdam/Abstract"><P><FONT color="black">Abstract</FONT></P></A></DIV><DIV class="bottomBox"><A href="https://2017.igem.org/wiki/index.php?title=Team%3APotsdam%2FResults"><P><FONT color="black">What we've achieved</FONT></P></A></DIV></DIV><DIV class="centerBox"><DIV class="topBox"><A href="https://2017.igem.org/Team:Potsdam/iGem-goes-green"><P><FONT color="black">iGEM Goes Green</FONT></P></A></DIV><DIV class="bottomBox"><A href="https://2017.igem.org/wiki/index.php?title=Team%3APotsdam%2FTeam"><P><FONT color="black"> Who we are...</FONT></P></A></DIV></DIV><DIV class="rightBox"><DIV class="topBox"><A href="https://2017.igem.org/wiki/index.php?title=Team%3APotsdam%2FBackground"><P><FONT color="black">Home</FONT></P></A></DIV><DIV class="bottomBox"><A href="https://2017.igem.org/wiki/index.php?title=Team%3APotsdam%2Fcrowdfunding"><P><FONT color="black">Crowdfunding</FONT></P></A></DIV></DIV></DIV></DIV></DIV><DIV id="sponsor-wrapper" class="wrapper"><P><FONT color="black"> Main sponsors: </FONT></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>