Team:Potsdam/Design
No Sidebar - Escape Velocity by HTML5 UP
IGEM Potsdam
The world's first enzyme dating service
Home
Team
We introduce ourselves
Motivation for participation
Collaborations
Eindhoven
iGem goes green
Skype conversations
Berlin
Postcards
German-Wide Meetup
Our project
Background facts and information
Abstract
Project Design
Modelling
Research work & protocols
lab book
Results
Parts
Outlook
our vision
Safety
Human Practices
Crowdfunding
KLAS
Laboratory exhibition
Potsdam Day of Science
school presentations
March for Science
Judging
Attributions - Bronze
Contribution - Bronze
Collaborations - Silver
Human practices - Silver
Modelling - Gold
Demonstrate - Gold
Human Practices Integrated - Gold
Sponsors
Contact
Project design
The number of biotechnological applications has skyrocketed and have evolved to be one of the most ubiquitous aspects in our daily life in the past years. From uses in food industry to dietary supplements and science, white biotechnology is everywhere.
That’s why we try to recreate a widespread natural phenomenon called metabolic channelling synthetically to increase the yield of biochemical reactions. Metabolic channelling occurs when the enzymes responsible for a specific reaction are in close proximity to each other and thereby decrease the diffusion time of the substrates.
dCas9 approach:
In our first method, we utilize the DNA-binding function of deadCas9 (dCas9) to bind two enzymes of the Indoleacetic acid (IAA) pathway, IAA tryptophan monooxygenase (IAAM) and indoleacetamide hydrolase (IaaH) on a DNA-scaffold which we introduce into E.coli. By binding the enzymes on DNA, we decrease the distance the substrates have to travel in the cytoplasm and hope to achieve a higher output of Indoleacetic acid.
Scaffolds for inducing metabolic channelling with nucleic acids has been tried in several different ways already. For example, by the iGEM ZJU China team of 2012, where RNA cloverleaf structures were used to bind IAA enzymes to RNA[4]. They fused the enzymes to RNA-binding proteins from viral coat proteins (MS2 and PP7). These bound to the RNA and thus put the IAA enzymes close to each other (see picture below). With this concept, they were able to record a 1.4-fold increase in IAA output.
Project design of ZJU China 2012
NUDT China 2015 mopdified this project and devised a DNA scaffold with the use of TALEs[5]. They fused the IAA enzymes to TALEs and used different lengths between the TALE recognition sites to test is metabolic channelling was dependent on spacer lengths.
Different channeling designs by NUDT_China 2015 and their respective fold increase in IAA production
With these constructs, they could achieve a up to 10-fold increase in IAA output, with more output being registered at a spacer distance of 6 bp[5].
Other methods using crisprRNA as a RNA scaffold have also been reported with some success as well as DNA scaffold with ZFNs[6].
Our first approach focuses on dCas9. dCas9 is a mutated version of Cas9 where the endonuclease domains were modified so that the protein no longer cuts dsDNA[7]. This enables us to bind the IAA enzymes to DNA and thereby reduce the diffusion distance for the substrates.
We achieve this by fusing IAAM and IAAH to MS2 and PP7 like ZJU China 2012 did (we created the part BBa_ 2483000 for this, because ZJU China didn’t) and use sgRNA designs from iGEM Warwick 2016. They used these sgRNAs to bring transcription factors closer to promoter regions[8]. The sgRNAs contain signal sequences for the RNA-binding proteins MS2 and PP7, the DNA recognition sites were slightly modified and are included in the part BBa_2483004.
When dCas9 binds to the DNA scaffold, the IAA-RBP fusions bind to the signal sequence on the sgRNA. By bringing in specific scaffold DNA, we can control the distance between the IAA enzymes and achieve metabolic channelling.
The synthesis of everything is happening on a low-copy plasmid, pSB4A5. The scaffold DNA is on a separate, high-copy plasmid, containing many sgRNA recognition sites. We planned single, so called “target cassettes” for our scaffolds, which we assembled with themselves with 3A assembly 4 times. Three cassettes were designed with 6, 12 and 18 bp spacer between the PAMs.
Target cassette 1 and 3 with 6bp and 18 bp spacer, respectively.
We had big problems regarding the 12 bp cassette and had to stop working with that one. The other two cassettes (BBa_2483005 and BBa_2483006) were successfully assembled 5 times resulting in 25 recognition site pairs per plasmid.  The PAMs and recognition sequences are facing each other to achieve the smallest possible distance between IAAM and IAAH. We put “big spacers” of about 120 bp length in between every recognition site pair to reduce problems of homologous recombination.
We hope to get better channelling results with this 2-plasmid concept than NUDT China 2015 because they only used one big part with less recognition sites.
We also uploaded two “subparts” of the final low-copy part BBa_K2483000 and K2483002. The first one consists of both IAA enzymes fused to RBPs, each with rbs and one Promoter (pVeg2). It was inspired by the part BBa_K515100 from iGEM Imperial College 2011.
K2483002 is a part containing LacI (BBa_C0012), a Lac inducible promoter (BBa_R0010) and dCas9. We made dCas9 inducible to reduce potential growth inhibition by it.
After both plasmids have been introduced into E.coli, IAA concentration was measured via GC-MS. The results were compared to controls with IAA-RBP fusions only and no dCas9, high-copy plasmid or sgRNA.
LLPS approach:
The second method revolves around Liquid-liquid-phase separation, a process naturally occurring in cells[1]. Through this process, proteins aggregate together to form droplets or membraneless bodies in the cytoplasm or nucleoplasm. This can be thought of as lipid droplets forming in water, just with proteins. Examples for this process in nature are stress granules or cajal bodies. The formation of droplets has already been studied by attaching YFP to Ddx4 (Nott et al.)[2], a protein known for aggregation. There are also indicators for this to happen in bacteria[3]. To use this mechanism for improvement of enzyme efficiency, the YFP-coding region will be replaced by the coding region of the IAA enzymes and expressed in yeast.
Formation of fluorescing droplets in HeLa cells, courtesy of Tim Nott
First, we had to confirm the formation of droplets also works in S. cerevisiae, as we neither had the courage nor the equipment to work with human cells. We created a part (BBa_K2483007) with yeast optimized Ddx4-YFP where we took the amino acid sequence from the publication above and with a galactose induced promoter (GAL1) to have more control over protein expression. Droplet formation can be induced by changing temperatures and change in osmolarity. Yeast cells were viewed under the fluorescence microscope.
After confirmation of droplet formation, we fused Ddx4 with IAAM and IAAH respectively and put them both under the same GAL1 promoter (BBa_K2483010). The idea is, to induce metabolic channeling by concentrating the IAA enzymes in the Ddx4 droplets and thereby reducing diffusion time.
Measurements were again done with GC-MS, as a control we used freely diffusing, yeast optimized IAAM and IAAH.
[1] Anthony A. Hyman et al. (2014), Liquid-Liquid Phase Separation in Biology, Max Planck Institute of Molecular Cell Biology and Genetics Dresden and Max Planck Institute for the Physics of Complex Systems, Dresden, 39-58
[2] Nott et al. (2015), Phase Transition of a Disordered Nuage Protein Generates Environmentally Responsive Membraneless Organelles, Mol. Cell, 57, 936-947
[3] Yuan A. H., Hochschild A. (2017).A bacterial global regulator forms a prion. Science355, 198–201
[4] https://2012.igem.org/Team:ZJU-China/project.htm
[5] Zhu L, Qiu X, Zhu L, et al.: Spatial organization of heterologous metabolic system in vivo based on TALE. Scientific Reports. 2016;6:27321. doi:10.1038/srep27321
[6] Conrado RJ et al. (2011): DNA-guided assembly of biosynthetic pathways promotes improves catalytic efficiency, Nucleic Acids Research, 012,Vol.40, No. 4
[7] Qi LS, Larson MH, Gilbert LA, Doudna JA, Weissman JS, Arkin AP, Lim WA. Repurposing CRISPR as an RNA-guided platform for sequence-specific control of gene expression. Cell. 2013;152:1173–1183
[8] https://2016.igem.org/Team:Warwick/Design
Did you see this?
Abstract
What we've achieved
iGEM Goes Green
Who we are...
Background information
Crowdfunding
Main sponsors:
