Team:Potsdam/Protocols
No Sidebar - Escape Velocity by HTML5 UP
IGEM Potsdam
The world's first enzyme dating service
Home
Team
We introduce ourselves
Motivation for participation
Collaborations
Eindhoven
iGem goes green
Skype conversations
Berlin
Postcards
German-Wide Meetup
Our project
Background facts and information
Abstract
Project Design
Modelling
Research work & protocols
lab book
Results
Parts
Outlook
our vision
Safety
Human Practices
Crowdfunding
KLAS
Laboratory exhibition
Potsdam Day of Science
school presentations
March for Science
Judging
Attributions - Bronze
Contribution - Bronze
Collaborations - Silver
Human practices - Silver
Modelling - Gold
Demonstrate - Gold
Human Practices Integrated - Gold
Sponsors
Contact
Our research work
Research work
Finding a suitable topic was very challenging and time consuming. Initially, we looked through projects of prior teams and assembled a list of possible topics.
A big influence was a new method for assembling genes in a manufacturing manner which was being developed by a research group on our university. Based on the quick and easy synthesis of proteins a first idea was the creation of enzymes that could convert blood groups. Also working with cyanobacteria was an option we considered.
After many seminars, we established the idea of metabolic channeling using dCas9 as our main project. One of our advisors also worked with membraneless organelles and suggested this approach for achieving metabolic channeling and therefor our secondary project with LLPS.
We thought about using either violacein or beta carotene as exemplary pathways for our increased production but finally decided for beta carotene. This brought many new challenges in the form of understanding the pathway and implementing it in E. coli.
Also, we very worried that an increased output would end up consuming too much precursor substrate and hinder growth of the transformed cells. Additionally, we found that team Edinburgh/Glasgow had problems with toxicity if the enzymes of the beta carotene pathway were in a specific order.
But all the planning was for nothing when we realized that some of the enzymes of the beta carotene pathway were localized in the membrane and therefore not suitable for our metabolic channeling approach.
After planning the design more precise we eventually arrived at our scaffold design of a low and a high-copy plasmid.
Protocols
1. Preparation of starting plasmids
1. Preparation restriction of components
1. Restriction using biobrick assembly enzymes.
2. This preparation step is needed to create sticky ends on the cassettes.
3. This step is only performed once.
4. Restriction with EcoRI and PstI (see restriction protocol) for all components.
2. Ligation of cassettes into plasmids
1. Ligation using T4-ligase (see ligation protocol)
plasmid
cassettes
psB1C3
1
2
3
psB1A3
1
2
3
psB1K3
1
2
3
3. Final preparation steps
1. Transformation of 9 different combinations into competent cells (see transformation protocol).
2. Selection with corresponding antibiotics.
4. Next day
1. Colony pcr and gel run to check for sizes of cassettes.
2. Miniprep and check concentration via nanodrop.
3. Many aliquots needed (small volume because thawing time) for future reactions!
2. 3A-assembly
1. Good to know
1. The 3-A-assembly will be used to add more and more cassettes in a row.
2. It is important to only combine cassettes with the same number (1, 2 and 3 have varying spacer length).
3. We will add cassettes and test frequently for viability to determine the maximum target-sequence length.
4. We want to combine about five cassettes.
2. Assembly scheme
1. In each assembly cycle, there will be two cassettes (same number/length) added to one linearized plasmid.
2. In the first step, we can either just use the prepared plasmid with cassettes already inserted or use an empty one, because the part in between will be cut out anyway.
3. The be able to select for plasmid with higher cassette content the resistances will cycle.
4. The resistance cycle (for plasmids) is K C A.
5. For the inserts, the resistance signals from which plasmid they will be cut.
6. The plasmids resistance determines the selection antibiotics for that step!
3. Legend
CX – Chloramphenicol (Insert/Plasmid) from step X
AX – Ampicillin (Insert/Plasmid) from step X
KX – Kanamycin (Insert/Plasmid) from step X
E – EcoRI
S – SpeI
X – XbaI
P – PstI
Insert 1 E+S
Insert 2 X+P
Plasmid E+P
C1
A1
K1
A2
K1
C2
K3
C2
A3
C4
A3
K4
4. Starting a cycle
1. Insert 1 is cut out of plasmid C1/Insert 2 is cut out of plasmid A1 and plamsid K1 is linearized .
2. Insert 1 and 2 are ligated into plasmid K1.
3. Insert 1 is cut out of plasmid A2/Insert 2 (here the fusion from Insert 1+2 from the first cycle) is cut out of K1 and plasmid C2 is linearized.
4. Insert 1 and 2 are ligated into plasmid C2.
5. With insert 1 coming from K3 and insert 2 from C2 (fusion) the steps will repeated until maximum number of inserts is reached.
6. The assembly needs to be done for all 3 cassettes simultaneously after each step.
7. Transformation of ligation into competent cells (see transformation protocol).
8. Selection with corresponding antibiotics.
5. Next day
1. Colony pcr and gel run to check for sizes of cassettes.
2. Miniprep and check concentration via nanodrop.
[1] http://parts.igem.org/Help:Protocols/3A_Assembly (accessed 31 October 2017)
1. Aim
1. Is the insert DNA in the plasmid present or absent?
2. Much easier than to isolate and purify the vector.
2. Good to know before the start
1. Take typical measures to prevent PCR cross over contamination, keep your bench clean, wear gloves, use sterile tubes and filter pipet tips.
2. Include a no-template control and positive control in parallel.
3. Thaw and keep reagents on ice.
4. Mix well before use.
5. The longer the amplicon, the longer the extension time: Use 15 sec/kb extension.
6. Use 90 sec extension for multiplexing.
7. Run an annealing temperature gradient from 55 °C to 65 °C to choose the best specificity conditions. Do not use fast cycling for multiplexing.
8. ALLin™ Red Taq Mastermix, 2X is premixed with red dye and density reagents for direct loading on the gels after the PCR. In a 2 % agarose TAE gel the dye migrates with ~350 bp  DNA, in 1 % agarose TAE gel with ~600 bp DNA fragments.
3. Are you working with A. E.coli or B. yeast?
A.step by step for E.coli:
1. Resuspend colonies.
2. Prepare masterplate.
3. Prepare a PCR master mix  (always prepare at least 10 % more).
4. Mix gently, avoid bubbles.
5. Aliquote the 22.0 μl of PCR master mix into each PCR tube.
6. Pick colony and put the toothpick or tip one into the Masterplate and then in the PCR tube/mix.
7. Do not forget the negative control!
8. Close tube.
9. Perform the PCR using Thermocycler as follow:
Steps
Cycles
Temperature
Time
Initial denaturation
1 cycle
95°C
60s
Denaturation
30-40 cycles
95°C
15s
Annealing
30-40 cycles
55-65°C
15s
Extension
30-40 cycles
72°C
15-90s (15 sec per 1 kb)
Final extension
1 cycle
72°C
5 min
10. Store probes for short time on ice, for long at -20 °C.
11. Load probes on the agarose gel e.g. 10 μl (so in case you have enough left for another round).
B. Step by step  for yeast:
1. If resuspended colonies are to be used: pipette 50 μl of a 0.02 M NaOH  solution into each of a set of  appropriately labelled PCR tubes or wells of a PCR plate. Using sterile pipette tips or toothpicks, transfer transformants to individual tubes/wells. The amount of cells  resuspended must just be visible. Resuspend cells by pipetting or vortexing and incubate for  ≥ 5 min at 37 °C.
2. If overnight cultures are to be used: pipette 40 μl of a 0.01 M NaOH solution into each of a set of appropriately labelled PCR tubes or wells of a PCR plate. Transfer 10 μl of each overnight culture to be tested to the appropriate tube/well and mix by pipetting up and down. Incubate for ≥ 5 min at 37 °C.
3. Prepare a PCR master mix (always prepare at least 10 % more).
4. Aliquot 22.5 μl of PCR master mix into each PCR tube.
5. Add 2.5 μl of the resuspended colony or overnight culture mixed with NaOH to the  appropriate PCR tube.
6. Close the tubes
7. Perform the PCR using the following cycling profile:
Steps
Cycles
Temperature
Time
Initial denaturation
1 cycle
95°C
60s
Denaturation
30-40 cycles
95°C
15s
Annealing
30-40 cycles
55-65°C
15s
Extension
30-40 cycles
72°C
15-90s
Final extension
1 cycle
72°C
5 min
8. Load probes on the agarose gel.
9. Store probes for short time on ice, for long at -20°C.
[1]https://www.highqu.com/media/wysiwyg/ressources/manuals/PCM02_ALLin_Red_Taq_Mastermix_PI.pdf (accessed 31 October 2017)
1. Aim
1. Purification of DNA  after a gel electrophoresis
2. Steps
1. Depending on the PCR product
If there is more than the wanted DNA band:
A. Dissolving the Gel Slice
If there iss only one DNA band:
B. Processing PCR Amplifications
1.  Following electrophoresis, excise DNA band from gel (with scalpel) and place gel slice in a 1.5 ml microcentrifuge tube.
2.  Add 10 μl Membrane Binding Solution per 10 mg of gel slice. Vortex and incubate at 50 – 65 °C until gel slice is  completely dissolved.
You can just work with the rest of your PCR aliquote (when you did not use all of it for the gel electrophoresis).
1. Add an equal volume (so you have to know how much aliquote is still in your  eppi!) of  Membrane  Binding Solution to the PCR amplification.
2. Binding of DNA
1.  Insert SV Minicolumn into Collection Tube.
2.  Transfer dissolved gel mixture or prepared PCR product to the Minicolumn assembly. Incubate at room temperature for 1 minute.
3.  Centrifuge at 16,000 ×g for 1 minute. Discard flowthrough and reinsert Minicolumn into Collection Tube.
4. Heat NE-buffer to 70 °C.
3. Washing
1.  Add 700 μl Membrane Wash Solution (ethanol added). Centrifuge at 16,000 × g for 1 minute. Discard flowthrough and reinsert Minicolumn into Collection Tube.
2.  Repeat Step 4 with 500 μl Membrane Wash Solution. Centrifuge at 16,000 × g for 5 minutes.
3.   Empty the Collection Tube and recentrifuge the column assembly for 1 minute with the microcentrifuge lid open(or off to allow evaporation of any residual ethanol.
4. Elution
1.  Carefully transfer Minicolumn to a clean 1.5 ml microcentrifuge tube.
2.  Add 50 μl of Nuclease-Free Water to the Minicolumn. Incubate at room temperature for 1 minute. Centrifuge at 16,000 × g for 1 minute.
3.  Discard Minicolumn and measure the concentration.
4. Store DNA at 4°C or –20°C.
[1]https://www.promega.de/-/media/files/resources/protcards/wizard-sv-gel-and-pcr-clean-up-system-quick-protocol.pdf?la=de-de (accessed 31 October 2017)
1. What is it ?
1. Standard lab procedure for separating DNA by size (e.g., length in base pairs) for visualization and purification.
2. Uses an electrical field to move the negatively charged DNA through an agarose gel matrix  toward a positive electrode.
3. Shorter DNA fragments migrate through the gel more quickly than longer ones.
2. Why are we doing it ?
1. to determine the approximate length of a DNA fragment by running it on an agarose gel alongside a DNA ladder (a collection of DNA fragments of known lengths).
3. Protocol
1. Pouring a Standard 1% Agarose Gel:
1. Measure 1g agarose and and mix it with 100 ml of TBE in a microwaveable flask.
Note:  Agarose gels are commonly used in concentrations of 0.7 % to 2 % depending on the size of bands needed to be separated -  Simply adjust the mass of agarose in a given volume to make gels of other agarose concentrations (e.g., 2 g of agarose in 100 mL of TAE will make a 2 % gel).
2. Microwave for 1-3 min until the agarose is completely dissolved (but do not overboil the solution, as some of the buffer will evaporate and thus alter the final percentage of agarose in the gel. Many people prefer to microwave in pulses, swirling the flask occasionally as the solution heats up.).
Note: gloves and glasses ! Caution HOT! Be careful stirring, eruptive boiling can occur.
It is a good idea to microwave for 30-45 sec, stop and swirl, and then continue towards a boil. Keep an eye on it as the initial boil has a tendency to boil over. Placing saran wrap over the top of the flask can help with this, but is not necessary if you pay close attention.
4. Pouring of the gel
1. Let agarose solution cool down to about 50 °C (about when you can comfortably keep your hand on the flask), about 5 mins.
Note:  Or cool down in water bath about 30 min.
2. Add ethidium bromide (EtBr) to a final concentration of approximately 0.2-0.5 μg/mL (usually about 2-3 μl of lab stock solution per 100 mL gel). EtBr binds to the DNA and allows you to visualize the DNA under ultraviolet (UV) light.
Note:  Caution EtBr is a known mutagen. Wear a lab coat, eye protection and gloves when working  with this chemical. If you add EtBr to your gel, you will also want to add it to the running buffer when you run  the gel.
3. Pour the agarose into a gel tray with the well comb in place.
Note: Think about witch gel tray size you need (a small one or a big one).
Pour slowly to avoid bubbles which will disrupt the gel. Any bubbles can be pushed away from the well comb or towards the sides/edges of the gel with a pipette trip.
4. Let the newly poured gel sit at room temperature for 20-30 mins, until it has completely solidified.
if  you are in a hurry the gel can also be set more quickly if you place the gel tray at 4 °C  earlier so that it is already cold when the gel is poured into it.
5. Loading Samples and Running an Agarose Gel
1. Add loading buffer to each of your digest samples.
Note: Loading buffer serves two purposes: 1) it provides a visible dye that helps with gel  loading and will also allows you to gauge how far the gel has run while you are running  your gel; and 2) it contains a high percentage of glycerol, so it increases the density of your  DNA sample causing it settle to the bottom of the gel well, instead of diffusing in the buffer.
2. Once solidified, place the agarose gel into the gel box (electrophoresis unit).
3. Fill gel box with 1xTAE (or TBE) until the gel is covered.
4.  Carefully load a molecular weight ladder into the first lane of the gel.
Note: When loading the sample in the well, maintain positive pressure on the sample to prevent  bubbles or buffer from entering the tip. Place the very top of the tip of the pipette into the buffer just above the well. Very slowly and steadily, push the sample out and watch as the sample fills the well. After all of the sample is unloaded, push the pipettor to the second stop and carefully raising the  pipette straight out of the buffer.
5.  Carefully load your samples into the additional wells of the gel.
6.  Run the gel at 80-150 V until the dye line is approximately 75-80 % of the way down the gel.
Note:  Black is negative, red is positive. (The DNA is negatively charged and will run towards the  positive electrode.)  Always Run to Red.
Note:  A typical run time is about 1-1.5 hours, depending on the gel concentration and voltage.
7.   Turn OFF power, disconnect the electrodes from the power source, and then carefully   remove  the gel from the gel box.
8. Using any device that has UV light, visualize your DNA fragments.
Note: When using UV light, protect your skin by wearing safety goggles or a face shield, gloves  and a lab coat.
Note: If you will be purifying the DNA for later use, use long-wavelength UV and expose for as  little time as possible to minimize damage to the DNA.
Note: The fragments of DNA are usually referred to as ‘bands’ due to their appearance on the gel.
6. Analyzing Your Gel
Using the DNA ladder in the first lane as a guide (the manufacturer's instruction will tell you the  size of each band), you can interpret the bands that you get in your sample lanes to determine if the  resulting DNA bands that you see are as expected or not. For more details on doing diagnostic  digests and how to interpret them please see the  Diagnostic Digest  page.
7. Purifying DNA from Your Gel
If you are conducting certain procedures, such as molecular cloning, you will need to purify the  DNA away from the agarose gel. For instructions on how to do this, visit the  Gel Purification  page.
[1] http://www.addgene.org/protocols/gel-electrophoresis/ (accessed 31 October 2017)
1. Aim
1. Enzymatically linkage of two DNA/RNA segments.
2. Steps
1. Set up the following reaction in a microcentrifuge tube on ice.
component
volume
T4 DNA Ligase Buffer (10 x)
2 µl
10x buffer
1 µl
T4 DNA Ligase
1 µl
Vector DNA
Insert DNA
Nuclease-free water
to 20 µl
1. Calculation of the DNA
kb (smaller DNA)/ kb (larger DNA)⋅mass (Vector DNA)⋅relation (Insert DNA)
Example calculation
1:3 vector to insert
mass Vector DNA: 100 ng
Vector DNA: 10 kb
Insert DNA: 3 kb
3 kb/ 10 kb⋅100 ng⋅3 = 90 ng
2. T4 DNA Ligase should be added last.
3. Use nebiocalculator.neb.com/#!/ to calculate molar ratios.
4. The T4 DNA Ligase Buffer should be thawed and resuspended at room temperature.
2. Gently mix the reaction by pipetting up and down and microfuge briefly.
3. Incubation
1. Cohesive (sticky) ends.
1. 16 °C overnight or room temperature for 10 minutes.
2. Blunt ends or single base overhangs.
1. 16°C overnight or room temperature for 2 hours (alternatively, high concentration T4 DNA Ligase can be used in a 10 minute ligation).
4.Heat inactivate at 65°C for 10 minutes.
5. Chill on ice and transform 1-5 μl of the reaction into 50 μl competent cells.
[1] https://www.neb.com/protocols/1/01/01/dna-ligation-with-t4-dna-ligase-m0202 (accessed 31 October 2017)
1. Aim
Isolation of DNA as a plasmid
2.Production of cleared lysate
1. Isolation of the bacteria
1.  Harvest 1–5 ml (high-copy-number plasmid) or 10 ml (low-copy-number plasmid)  of bacterial culture .
2. Centrifugation for 5 minutes at 10,000 xg in a tabletop centrifuge.
3. Pour off the supernatant.
4. Reinsert again bacterial culture to the pellet and repeat step 2 and 3.
5. Blot the inverted tube on a paper towel to remove excess media.
2. Resuspension of the cells
1. Add 250 μl of Cell Resuspension Solution.
2. Completely resuspend the cell pellet by vortexing or pipetting.
3. It is essential to thoroughly resuspend the cells.
3. Lysing
1. Add 250 μl of Cell Lysis Solution.
2. Mix by inverting the tube 4 times - do not vortex.
3. Incubate until the cell suspension clears (clear ≠ colorlessly) (approximately 1–5 minutes).
4. Splitting proteins
1. Add 10 μl of Alkaline Protease Solution.
2. Mix by inverting the tube 4 times - do not vortex.
3. Incubate for 5 minutes at room temperature.
5. Neutralization
1. Add 350 μl of Neutralization Solution.
2. Immediately mix by inverting the tube 4 times - do not vortex.
6. Isolation of the plasmids
1. Centrifuge the bacterial lysate at maximum speed (around 14,000 ×g) in a microcentrifuge for 10 minutes at room temperature.
3. Isolation of the plasmid DNA
1. Transfer the cleared lysate (approximately 850 μl, Section 3.B, Step 6) to the prepared Spin Column by decanting. Avoid disturbing or transferring any of the white precipitate with the supernatant.
1. If the white precipitate is accidentally transferred to the Spin Column, pour the Spin Column contents back into a sterile 1.5ml microcentrifuge tube and centrifuge for another 5–10 minutes at maximum speed. Transfer the resulting supernatant into the same Spin Column that was used initially for this sample. The Spin Column can be reused but only for this sample.
2. Centrifuge the supernatant at maximum speed in a microcentrifuge for 1 minute at room temperature. Remove the Spin Column from the tube and discard the flowthrough from the Collection Tube. Reinsert the Spin Column into the Collection Tube.
3. Wash the plasmid DNA.
1. Add 750 μl of Column Wash Solution.
2. Centrifuge at maximum speed in a microcentrifuge for 1 minute at room temperature.
3. Remove the Spin Column from the tube and discard the flowthrough.
4. Reinsert the Spin Column into the Collection Tube.
4. Wash again the plasmid DNA.
1. Add 250 μl of Column Wash Solution.
2. Centrifuge at maximum speed in a microcentrifuge for 2 minutes at room temperature.
3. If the Spin Column has Column Wash Solution associated with it, centrifuge again for 1 minute at maximum speed.
4. Transfer the Spin Column to a new, sterile 1.5ml microcentrifuge tube, being careful not to transfer any of the Column Wash Solution with the Spin Column.
5. Elute the plasmid DNA
1. Add 50 μl of Nuclease-Free Water to the Spin Column, wait 5 minutes
2. Centrifuge at maximum speed for 1 minute at room temperature in a microcentrifuge.
6. After eluting the DNA, remove the assembly from the 1.5ml microcentrifuge tube and discard the Spin Column.
[1]https://www.promega.de/-/media/files/resources/protocols/technical-bulletins/0/wizard-plus-sv-minipreps-dna-purification-system-protocol.pdf (accessed 31 October 2017)
1. What is the PCR ?
polymerase chain reaction
Method to make multiple copies of a  the specific  DNA-sequence
2. Reaction Setup
1. Assemble all reaction components on ice, work on ice while assembling.
2. Preheat the thermocycler to the denaturation temperature (98 °C).
3. Prior to use all components should be mixed.
4. Work quickly when transferring the reactions to a thermocycler.
3. Steps
1. Assemble all components on ice for the reaction :
Component
25 μl Reaction
50 μl Reaction
Final Concentration
Q5 High-Fidelity 2X Master Mix
12.5 μl
25 μl
1X
10 μM Forward Primer
1.25 μl
2.5 μl
0.5 μM
10 μM Reverse Primer
1.25 μl
2.5 μl
0.5 μM
Template DNA
variable
variable
< 1,000 ng
Nuclease-Free Water
to 25 μl
to 50 μl
Notes: Two Primers have to be diluted 1:10 !
Notes: Gently mix the reaction. Collect all liquid to the bottom of the tube by a quick spin if  necessary. Overlay the sample with mineral oil if using a PCR machine without a heated lid.
2. Transfer PCR tubes to a PCR machine and begin thermocycling.
4. Steps  of  PCR
1.Denaturation : double- stranded template DNA is heated to separate it into two single stands.
2. Annealing    :  temperature is lowered to enable the DNA primers to attach to the template DNA.
3. Extending    : temperature is raised and the new strand of DNA is made by the  polymerases.
Thermocycling Conditions for a Routine PCR:
Step
Temperature
Time
Initial Denaturation
98°C
30 seconds
98°C
5–10 seconds
25–35 Cycles
*50–72°C
10–30 seconds
72°C
20–30 seconds/kb
Final Extension
72°C
2 minutes
Hold
4-10°C
Please note that protocols with Q5 High-Fidelity DNA Polymerase may differ from protocols  with other polymerases. Conditions recommended below should be used for optimal  performance.
[1] https://www.neb.com/protocols/2012/12/07/protocol-for-q5-high-fidelity-2x-master-mix-m0492 (accessed 31 October 2017)
1. Aim
A restriction digest is the division of DNA in a  specific area by the help of restriction enzymes. The aim afterwards could be to analyze and characterize (restriction maps) the DNA, to compare it to others or to clone it into e.g. a vector. Previous to the restriction, the DNA has to be isolated (see protocol miniprep).
2. Test digest - What for?
In a test digest the cleavage products are analyzed to verify that the used DNA has e.g. taken in a specific fragment. Only few of the DNA is digested, because the uncut DNA will be used in further steps. The whole DNA strand is often too long to analyze, therefore:
targeted cuts between specific base sequences
cleavage fragments small enough to run on gel
segregation and analysis by gel electrophoresis
→ always with a control: uncut plasmid (see protocol gel electrophoresis) In a preparative digest normally 1 U enzyme digests 1 µg DNA in one hour.
3. Preparative digest - What for?
In a preparative digest the entire available DNA is digested, because the cleavage products are used in further steps. The cut DNA can be extracted from a gel.  It is important that as much DNA as possible is digested. Therefore 0,2 - 0,4 µl enzyme per µg DNA are applied and the reaction should run for ~2h.
4. Procedure
exemplary pipetting scheme:
component
volume
DNA (typically 200 - 500 ng)
1 µl
10x buffer
1 µl
H2O
7,6 µl
enzyme 1
0,1 µl
enzyme 2
0,1 µl
final volume
10 µl
If more than one reaction is done, it is convenient to prepare a master mix containing everything except for the DNA (!). This saves time and tips and keeps you from having to pipet very small volumes (e.g. 0,1 µl).
It is also possible to use more than two or just one restriction enzyme per reaction. They just need to have the same buffer preferences. Volumes have to be adjusted to the number of enzymes.
[1] http://www.log2embl.de/sites/default/files/Labor-Protokoll-Restriktionsverdau.pdf (accessed 30 October 2017)
[2] https://www.uni-hohenheim.de/fileadmin/einrichtungen/pflanzenphysiologie/Protokolle/V.Klonierung/restriktionsverdau_de.pdf (accessed 30 October 2017)
[3] http://www.biochemie.uni-jena.de/files/Praktikum/plasmid%20dna%20+%20restrictionsverdau.pdf (accessed 30 October 2017)
1. Aim
1. Large qualitative screening of IAA-producing colonies at the same time to see if our constructs are still functional in our E.coli/yeasts.
2. Helps us to pick the right colonies for colony-PCR and GC-MS measurements.
2. Safty
1. Reagent: 2 % 0.5M FeCl3 in 35 % perchloric acid
1. Perchloric acid is highly corrosive and dangerous!! Always uses protective gear and work under a fume hood!
2. Reagent will always be mixed together on the spot, FeCl3 stock solution is finished, acid will be taken from the chemicals sheld from the AG plant physiology (has been negotiated).
3. What happens?
1. Reagent reacts to IAA (and other indolic compounds) to make several colored products.
2. IAA will be seen as bright red (other compounds brown or yellowish).
4. Assay conditions
1. Plates were inoculated in a grid pattern and overlaid with an 82 mm-diameter disk of Nitrocellulose membranes.
2. Plates are overlaid with Nitrocellulose immediately after inoculation with toothpicks. After normal incubation (i.e. overnight) time, the membrane was removed and soaked in reagent (or reagent-saturated [2.5 mL] filter paper, here “Whatman grade 2” had best results), in glass chamber ( danger symbol and written information).
3. After 30 - 60 minutes, coloring reaction is finished and fading began.
4. Best results with colony sizes between 0.5 to 2 mm.
5. Addition of Tryptophan greatly enhances color reaction but does not interfere with distinguishing IAA positive and negative colonies (yellow background and strong red to pink positives).
6. Other indolic compounds (i.e. indolepyruvic acid) are distinguishable by a more yellow-brownish color.
5. Afterwards
1. Neutralize the acid with NaOH and use a pH-test strip.
2. Throw away liquid and solid waste separately.
[1]Bric JM, Bostock RM, Silverstone SE. Rapid In Situ Assay for Indoleacetic Acid Production by Bacteria Immobilized on a Nitrocellulose Membrane. Applied and Environmental Microbiology. 1991;57(2):535-538. (accessed 31 October 2017)
1. Aim
1. Cloning method that uses bacterial cell extracts.
2. Assembly  of  multiple  DNA  fragments  into  a  recombinant  DNA  molecule → in  a single  in  vitroreaction.
3. General principle: recombining short end homologies (15-52 bp.)
4. Homologous  ends  can  be  flanked  by  heterologous  sequences  (e.g.  for  inducing  a  linker sequence).
2. Steps
No steps have to be done at the clean bench, working at room temperature.
1. Prepare 10X SLiCE Buffer in a 1.5 mL tube :
500 μL 1 M Tris-HCl pH 7.5
+     50 μL 2 M MgCl2
+    100 μL 100 mM ATP (Thermo #R0441)
+     10 μL 1 M DTT
+     ddH2O to 1 mL
Store at -20 °C in 40-60 μl aliquots.
2. Add the following ingredients into a 0.2 mL tube in this orde rand vortex:
linearized vector backbone (50 - 200 ng)
+   each additional assembly piece (1:1 - 1:10 molar ratio of vector:insert)
+   1 μL 10X SLiCE buffer
+   1 μL PPY SLiCE extract
+   ddH2O to 10 μL
-> We use ratio vector:insert of 1:10
-> Good to know: PPY in the strain whose cell extract is used for SLiCE reaction
3. Incubate the SLiCE reaction mix as above at 37 °C for 1 hour using a PCR machine or water bath, and then place on ice.
4. Transform 1 - 10 μL of the assembly reaction into 50 μL of competent E. coli and/or run a diagnostic agarose gel to check for successful assembly.
Transformation of E. coli safer, but takes more time.
5. For electroporation, transform 1 μL into 50 μL electrocompetent cells. For large recombinant DNA, electroporation is required. In complex cloning, electroporation is recommended, as it is 10-100 times as efficient as chemical transformation.
Electrocompetent cells have to be made, or we use heat shock (see protocol“transformationof E. coli”), protocol for electrocompetent cells can be taken from NEB
[1] https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4672941/ (accessed 01 November 2017)
1. What is it?
Transmission of genetic information into competent cells (or plants, algas, mushrooms) (the target organismn)
2. Steps
1. Prechil 1.5 ml tube on ice.
2. Thaw a tube competent E. coli  cells on ice for 10 minutes.
1.  Mix gently.
2. Pipette 50 µl of the cells  into the 1.5 ml tube.
3. Add 1-5 µl (containing 1 pg-100 ng of plasmid) DNA to the cell mixture.
(as soon as as the last bit of ice in the tube is disappeared!)
4. Flick the tube 4-5 times to mix cells and DNA.  (No vortexing!)
5. Place the mixture on ice for 30 minutes. (without mixing!)
(2-fold loss in transformation efficiency for every 10 minutes this step is shortened!) 
6. Heat shock at exactly 42°C for exactly 30 seconds.  (without mixing!)
(temperature and timing specific to transformation volume and vessel)
7. Place on ice for 5 minutes.
(without mixing!)
8. Pipette 950 µl of room temperature SOC into the mixture.
9. Place at 37°C for 60 minutes and shake vigorously (800 rpm in thermo mix block).
(2-fold loss in transformation efficiency for every 15 minutes this step is shortened)
(SOC gives 2-fold higher transformation efficiency than LB medium)
(incubation with shaking or rotating the tube gives 2-fold higher transformation efficiency than without)
10. Warm selection plates to 37 °C.
11. Mix the cells thoroughly by flicking the tube and inverting.
12. Spread 200 µl onto a selection plate and incubate overnight at 37 °C.
13.For low efficiency cloning reactions: spin down the whole transformation mixture and remove the nearly complete supernatant (approx. 900 µl). Resuspend cells in remaining liquid and plate completely.
[1]https://www.neb.com/protocols/1/01/01/high-efficiency-transformation-protocol-c2987 (accessed 01 November 2017)
1.Preparation of Competent Cells
1.Grow  yeast cells  at 30 °C in 10 ml YPD broth until mid-log phase
(~5 x 106 - 2 x 107 cells/ml  or OD 600 of  0.8-1.0).
The following steps are accomplished at room temperature.
2.    Pellet the cells at 500 x g for 4 minutes and discard the supernatant.
3.   Add 10 ml EZ 1 solution to wash the pellet. Repellet the cells and discard the supernatant.
4.    Add 1 ml EZ 2 solution to resuspend the pellet. At this point, the competent cells can be used for transformations directly or stored frozen at or below -70°C for future use.  It is important to freeze the cells slowly.  To accomplish this, either wrap the aliquotted cells in 2-6 layers of paper towels or place in a Styrofoam box before placing in the freezer.DO NOT use liquid nitrogen to snap-freeze the cells.
2. Transformation
This  part  of  the  procedure  is  the  same  for  both  frozen  stored  (thawed  at  room  temperature)  and  freshly prepared competent yeast cells.
1.  Mix  50 μl  of  competent  cells  with  0.2-1 μg  DNA  (in less  than  5 μl  volume);  add  500 μl EZ 3 solution and mix thoroughly.
2. Incubate  at  30 °C  for  45  minutes.  Mix  vigorously  by  flicking  with  finger  or  vortexing  (if appropriate  for your DNA) 2-3 times during this incubation.
3.    Spread 50-150  μl of the above transformation mixture on an appropriate plate.  It is unnecessary to pellet and wash the cells before spreading. Incubate the plates at 30°C for 2-4 days to allow for growth of transformants.
[1] http://www.zymoresearch.com/downloads/dl/file/id/165/t2001i.pdf (accessed 01 November 2017)
Did you see this?
Abstract
What we've achieved
iGEM Goes Green
Who we are...
Background information
Crowdfunding
Main sponsors:
