Team:Potsdam/notebook/lowcopy
No Sidebar - Escape Velocity by HTML5 UP
IGEM Potsdam
The world's first enzyme dating service
Home
Team
We introduce ourselves
Motivation for participation
Collaborations
Eindhoven
iGem goes green
Skype conversations
Berlin
Postcards
German-Wide Meetup
Our project
Background facts and information
Abstract
Project Design
Modelling
Research work & protocols
lab book
Results
Parts
Outlook
our vision
Safety
Human Practices
Crowdfunding
KLAS
Laboratory exhibition
Potsdam Day of Science
school presentations
March for Science
Judging
Attributions - Bronze
Contribution - Bronze
Collaborations - Silver
Human practices - Silver
Modelling - Gold
Demonstrate - Gold
Human Practices Integrated - Gold
Sponsors
Contact
Lab book
We separated our lab book into the following parts:
Friday, 07/21/17
-  took pSB4A5 backbone out of DNA distribution Kit and transformed into E.coli
(rest from Kit was put in -20 °C)
Tuesday, 07/25/17
- Inoculated LB media with psB4A5
Wednesday, 07/26/17
- Miniprep with Promega “Plus SV Miniprep DNA Purification System”
- 2 approaches with psB4A5
- final concentration:
- approach 1: 30 ng/µl
- approach 2: 40 ng/µl
Thursday, 08/10/17
- glycerol stock (30%) of IG_C_P1_G (pSB4A5)
- prepared 10 mL preculture with pdCas9-Bacteria E.coli from plate from Fabian for Miniprep
Monday, 08/14/17
- primer resuspended to 100 uM (except Q8):
primer
volume in µl
Q5
354
Q6
957
Q7
749
Q8
657 (83,1 uM)
Q15
956
Q16
1455
Q17
1693
Q18
1440
- diluted all primers to 10 uM in separate aliquotes
- prepareed 3 separate PCRs with primer pairs:
Q5/Q8 (Eppi A), Q6/Q7 (Eppi B), Q15/Q16 (Eppi c)
- pipetting scheme:
Q5 2X MasterMix: 5 µL
10 uM Primer: each 0,5 µL
Template: 1 µL of 1:100 diluted Miniprep (about 0,5 ng)
ddH2O: 3 µL
total: 10 µL
- cycling conditions (prog01 in Epepndorf Mastercycler nexus gradient (Torsten):
temperature
time in sec
98 °C
30
98 °C
10
64 °C
20
72 °C
110
72 °C
120
- after PCR: gel electrophoresis as follows:
- 2.5 µL DNA + 2µL Loading Dye + 7.5 µL (total 12 µL)
- ran gel at 120 V for 40 minutes
gel picture:
- order of pockets: ABC, A and B are correct size, C did not work well, there is a band at the desired result and one at the unopened plasmid (and several other stuff) so PCR has to be done again
- dCas9 fragments look good and can be worked with tomorrow
Tuesday, 08/15/17
1. DpnI digestion of dCas9 fragments A + B
component
volume in µl
DpnI
2,6
DNA
6,5
cutsmart-Buffer
5,0
H2O
35,9
final volume
50
- Duration: 1 hour
IMPORTANT: Fabian told me you can just add about 1 µL DpnI to the reaction-mix
2. Gel run of dCas9 A + B
- let it run on gel for purification
- 140 V a 40 min
- cut out was done by Natalie and Pauline
3. Linearization PCR of psB4A5 (second atempt) via gradient PCR
- made new PCR using Q15/Q16 primers and psB4A5 plasmid fpr linearization
- used 3 different DNA concentrations and 3 different annealing temperatures (9 reactions total)
component
1 reaction:
volume in µl
10 reaction MM:
volume in µl
HiQ
5
50
primer
1,25
12,5
DNA
variable (1 µl)
variable
H2O
17,5
175
final volume
25
250
- annealing temperatures: A = 60°C/B = 64,5°C/C = 68°C
- DNA concentrations: 0,1 = 1:1000 / 1 = 1:100 / 10 = 1:10
Scheme:
0,1 A
0,1 B
0,1 C
1 A
1 B
1 C
10 A
10 B
10 C
Wednesday, 08/16/17
1. Gel run of psB4A5 gradient PCR (A,B,C,0.1,1,10)
- run at 120 V a 40 min -> only 2 bands at well 1A
- run at 120 V another 30 min -> lost DNA from gel
- 1A (60°C/1:100 DNA) might have worked so I did a PCR purification for this one
2. Gel purification of dCAs9 A + B fragments and 1A-psB4A5 linearized plasmid
- used Promega PCR/Gel purification kit: final concentration pSB4A5 18 ng/µL
https://www.promega.de/resources/protocols/technical-bulletins/101/wizard-sv-gel-and-pcr-cleanup-system-protocol/
IMPORTANT: We use AE-buffer for elution in the last step (50 µL) instead of water!
3. Fusion PCR of dCAs9 fragments A + B
- Primer used: Q5 + Q6
component
volume in µl
Q5-MM
12,5
DNA
2(1 pro fragment)
H2O
10,5
final volume
25
IMPORTANT: Fusion PCR done like team munich!
https://static.igem.org/mediawiki/2014/c/c9/LMU_Munich14_FusionPCR.pdf
- first made a PCR mix of 25 µL without primers
- then run mix in thermocycler with programm (fusion pcr)
- in the middle there is a hold step in the programm
- you then add the primers mixed with buffer (5 µL total volume) for an endvolume of 30 µL
- IG_C_Q19/20 and Q24 resuspended auf 100 µM and diluted to 10 µM
Thursday, 08/17/17
1. dCas9:
- DpnI- digested Fusion-PCR product (1 µL in PCR-tube for 1h at 37 °C)
- ran 0,7 % agarose gel --> negative result
- gel picture:
- possible reasons: not enough DNA in PCRPCR cleanup was done and not gel extraction, wrong PCR protocol
- PCR with Q5-Q8 was done again (see above for protocol)
- no test gel but preperative gel and gel extraction with Promega protocol
- dCas9-A fragment: 2,8 ng/µL, dCas9-B fragment: 3,3 ng/µL
- not enough DNA again for PCR, tomorrow run PCR with 50 µL reaction tomorrow!
2. pSB4A5:
DpnI-digestion:
- 1 µl added to the PCR mix and incubated 1 h at 37 °C
- SLiCE reaction of pSB4A5 with IAAM-MS2 + IAAH-PP7:
because of to low DNA concentration we needed to increase the concentration by drying the DNA
- for this reaction we take:
2,8 µl backbone (50 ng, 3395 bp, 18 ng/µl)
+ 32,2 µl IAAM-MS2 (1:10 ratio, 2188 bp, 10 ng/µl)
+ 26,4 µl IAAH-PP7 (1:10 ratio, 1790 bp, 10 ng/µl)
all in one eppi + 38,6 µl ddwater to get 100 µl
to the 100 µl we added:
+ 1/10 of original volume (100µl) 3 M Sodium acetate, pH 5,2 --> 10 µl
+ 2 x original volume 100% ethanol (needs to be cooled down for 20 min at -20°C) --> 200 µl
mix thorougly
- put 30 min to -20 °C (in this time cool down centrifuge to 4 °C)
- centrifuge 10 min with maiximal speed at 4 °C
- discard supernatant with a pipette, BE CAREFUL!!!, pellet is maybe (in most cases, because we don't have much DNA) nit visible, pay attenteion on the orinetation of the Eppis in the centrifuge (lids to the outside) and go with the pipette onto this side of the Eppi, where the pellet DON'T is, DON'T TOUCH THE PELLET!!!
- add 300 µl 70 % ethanol (needs to be cooled down for at least 20 min at -20 °C)
- centrifuge 10 min with maiximal speed at 4 °C
- again CAREFULLY discard supernatant with a pipette
- make a short spin 10-30 sec (Short-Button) and take carefully additional supernatant with a pipette
- put Eppi open lid for 10 min to 37 °C
- add SLiCE reaction directly to this Eppi (NO RESUSPENSION BEFORE)
SLiCE reaction:
Eppi with dried backbone, IAAM-MS2 and IAAH-PP7
+ 1 µl 10 x SLiCE buffer
+ 1 µl PPY SLiCE extract
+ 8 µl ddwater
--> 10 µl SLiCE reaction
- incubate 1 h at 37°C, no shaking (flip a few times to the Eppi to resuspend the DNA)
- Transformation in E.coli clls with all the 10 µl SLiCE reaction in the same Eppi like the transformation protocol, no changes from protocol
- 200 µl to Ampicillin-LB-agar plates
- and centrifuge rest of the culture, discard most of the supernatant, resuspend in the rest supernatant and put this resuspenden rest also to an Ampicillin-LB-agar plate
- plates over night to 37 °C
4. for tomorrow:
Colony-PCR with the SLiCE plate and at the same time preparation of the Slakowski-Assay plate
Friday, 08/18/17
- no colonies on pSB4A5 plate
- did q5-Q8 PCr again with 50 uL reaction following protocol (5 uL of 1:100 diluted miniprep as template)
used Q5678 protocol in Torsten Thermocycler
- gel extraction:
fragment A: 3,7 ng/uL
fragment b: 31,4 ng/uL
- PCR Q5/Q8 again for fragment A with gradient PCR
- for Monday: program in Torsten: "Q58":
- use 4 different wells/temperatures:
- well 1: 60°C; 5: 62,5°C; 7:64,5 °C; 9: 66,5°C
- elongation temperature reduced to 30 sec
- make 50 uL reaction again,
afterwards: gel extraction and fusion PCR of dCas-A and dCas9-B
ask Fabian for better Fusion-PCR cycler program (the one from last time was unnecessarily long according to Fabian)
Monday, 08/21/17
- gradient PCR for dCas9-A fragment
- Mastermix for 10 reactions (10 µl/ reaction):
component
volume in µl
Q5 2xMastermix
50
primer Q5 (10µM)
5
primer Q8 (10µM)
5
water
20
template (~1ng)
20
total volume
100
- template: pdCas9 from miniprep (48,3 ng/µl) -> dilution: 1:100 ~ 0,5 ng/µl
- PCR program: dcas9a_gradient (in Torsten) (also changed elongation time to 30 s)
temperatures of gradient:
60°C 60,2°C 61,5°C 62,5°C 64,5°C 65,5°C 67,8°C 68°C
- 1 % agarose gel -> 25 min, 140 V
- gel picture:
- used annealing temperature of 60,2°C (B) for preparative PCR (50 µl / reaction):
- two different reactions with different template amounts:
- reaction 1: 0,5 ng
- reaction 2: 5 ng
component
volume in µl
Q5 2xMastermix
25
primer Q5 (10µM)
2,5
primer Q8 (10µM)
2,5
water reaction 1/2
19/10
template reaction 1/2)
1/10
total volume
50
- 1 % agarose gel -> 30 min, 120 V
- excision of gel bands from reaction 1 and 2 (1,3 kb) -> put in one tube
- stored at 4 °C over night
Tuesday, 08/22/17
- Gel clean-up of dCas9-A (Elution with 50 µl NE-Buffer) -> c = 17,6 ng/µl
- Fusion PCR
- changes in the PCR Program (in Torsten):
- elongation time in the second cycle: 3 min 18 s (30 s/kb * 6,6 kb (size of the fusion product) = 198 s)
- finale elongation time after the second cycle: 10 min
- for the first part of the fusion PCR:
component
volume in µl
Q5 2xMastermix
12.5
dCas9-A (50 ng/µl)
2.8
dCas9-B (50 ng/µl)
1.6
water
8.1
template reaction 1/2)
1/10
total volume
25
at the "hold-step" in PCR program added the primer mix (5 µl) and continued the PCR:
component
volume in µl
Q5 2xMastermix
2.5
dCas9-A (50 ng/µl)
1.25
dCas9-B (50 ng/µl)
1.25
total volume
5
- DpnI digestion after the fusion PCR:
- added 1 µl DpnI to the  PCR product
- incubated 30 min at 37°C
stored digested PCR product at -20 °C
Wednesday, 08/23/17
- gel extraction of fusion-PCR product (1 % agarose, next time use 0, 7%; 100V. 55 min)
- elution in 45 uL NE-Buffer (25 + 20), concentration: 3,3 ng/µL
- not enough --> PCR amplification with Q5/Q6 with standard 50 µL reaction (0,5 µL Plasmid template)
- program "dCas9 after fusion in PeqSAR mixer, Eppi: "PCR fusion", 35 cycles
temperature
time
98
30
98
10
64
30
72
3:21 min
72
2 min
- into -20, tomorrow test gel run
- Primer for amplification (Q25-Q30) resuspended to 100µM and separately diluted to 10 µM
- IDT-fragments were diluted 1:10 separately
- PCR for amplification of PCR fragments:
- pipetted normal 50 µL reaction (1 µL of diluted fragment)
- PCR reaction ("idt" in Torsten): 35 cycles
temperature
time
98
30
98
10
gradient 58-71
20
72
60
72
2 min
- PP7 and MS2 at 71 °C, LacI at 58 °C
- put into -20 and tomorrow test gel run
Thursday, 08/24/17
- Fusion PCR: purifaction gel electrophoresis
- gel electrophoresis:
- 50 µl DNA
- 10 µl Loading Dye
- 0,7 % Agarose gel
- 6,6kb fragment
- 1h with 80 V
- purifaction with the Promega Purification Kit
- final concentration: 45,7 ng/µL
- test gel of PCR with IDT fragments of yesterday:
IAAH-PP7 - IAAM-MS2 - LacI - TaC1 - TaC2 - TaC3 - Ddx4-YFP
- only IAAH-PP7 and TaC3 bands are okay
- gel purification: PP7: 40,7 ng/µL
- TaC3: 27,3 ng/µL
- second PCR with other fragments, used three different temperatures for annealling for every fragment:
fragment
1st TM (well)
2nd TM
3rd TM
MS2
68,3 (9)
69,7 (10)
71 (12)
LacI
56 (1)
57,3 (3)
60,7 (5)
TaC1
66,3 (8)
68,3 (9)
69,7 (10)
TaC2
68,3 (9)
69,7 (10)
71 (12)
YFP
62,6 (6)
64,4 (7)
66,3 (8)
- Tip from Fabian: next time more gradient Temperature distance!
- fragments from left to right (Temperatures are alaso from left to right) : MS2, LacI, TaC1, TaC2,  YFP
2017-08-24 15hr 44min-1.jpg
- MS2 / 9 band correct
- YFP / 6 and 7 bands are correct
- LacI no bands, TaC1 almost no bands, TaC2 bands incorrect
- gel run of MS2/9 and YFP/ 6,7 and put cut gel slices in the fridge for extraction tomorrow
- test gel of 4 uL (10 ng/uL) of IDT-fragments: LacI, TaC1, TaC2 with 6 uL H2O und 2 uL Dye to see if there's something wrong with the IDT fragments
- TaC1 and YFP show correct bands
- TaC2 correct and wrong band (maybe low concentration?) --> Nanodrop result: 11,4 ng/uL; problem that DNA concentration of desired
DNA is too low for restriction?
Friday, 08/25/17
- Gel and PCR clean up (Promega kit)
- MS2: 240mg gel; final concentration: 16,6 ng/µL
- YEP 6: 250mg gel; final concentration: 19,6 ng/µL
- YEP 7: 200mg gel; final concentration: 10,9 ng/µL
Gibbson Assembly "NEBuilder® HiFi DNA Assembly Master Mix"
https://www.neb.com/protocols/2014/11/26/nebuilder-hifi-dna-assembly-reaction-protocol
component
volume in µl
size in bp
concentration
Vector: psB4A5 (IG_C_P1_linearized)
2,75
3395
Insert 1: IAAM-MS2
6,1
2200
10,6 ng/µl
Insert 2: IAAH-PP7
1,3
1800
40,7 ng/µl
Master Mix
10,15ul
Trafo
https://www.neb.com/protocols/2015/02/17/nebuilder-hifi-dna-assembly-chemical-transformation-protocol-e2621
- Gibbson Assembly (5 µl) + competen cells (C2987)
- plate cells: 100µl and 200µl
Monday, 08/28/17
- Gibson Assembly worked, we have colonies but they are useless (see below)
- colony PCR following the protocol (beware that the protocol has been updated in the lab),
annealing Temperature: 52°C wiht Q24 and Q18 and 10 colonies
- negative control: IG_C_P1 miniprep
- negative/Gibson 1-10 and TaC3
- negative results for all reactions except the last onw which is just the gel purified TaC3
- excpept it isnt because the band cleearlyc shwows a sifferent size and is more likely to be IAAAH-PP7 (a suspicion of Sarah who wasn't sure if the tubes were swapped) which explains the negative results in the Gibson assembly
- Gibson assembly was done again withcorrect fragments:
- pSB4A5 miniprep: 2,75 uL (18 ng/µL)
- IAAM-MS2 6,1 uL (10,6 ng/µL)
- IAAH-PP7 1,9 uL (27,3 ng/µL)
- MasterMix 10,77 µL (fragments to Mastermix : 1:1)
- two Trafos were done with already used competent cells and new competent cells (always use the cells we got from - the Gibson Kit for Gibson assembly!) and plated with 200 uL and centrifuged
- repeated PCR to amplify IDT-fragments with TaC1, TaC2 and LacI (always as 50 uL reactions):
- 3 different concentrations and temperatures for each fragment:
temperature in °C/ concentration
0,1 ng
1 ng
10 ng
60
1
4
7
68
3
6
9
from left to right: Marker- TaC1/ 1-9 - one empty lane - TaC2/1-4 - Marker Marker - TaC2/5-9 - LacI/ 1-9 (but not 5!) - Marker - 6LacI/5
low concentrations worked best, results at Tac1/1, TaC2/3 and LacI/1 were best and gel purified
- Tac1: 5,8 ng/µL
- Tac2: 9,2 ng/µL
- LacI: 12,9 ng/µL
Tuesday, 08/29/17
Colony-PCR of P4 with Q24 and Q18 (see protocol and note the changes in the folder!)
- 10 colonies
- masterplate
- did a test gel (1 % Agarose) (actualy did two beacuse of missing marker in the first one, see lab book)
- gel picture shows: all fragments have expected length
- no miniprep culturs inoculated because colonies were too small -> To do tomorrow
Gibbson assembly of P5 (IAA-RBP in pSB4A5) (see online protocol from NEB) Transformation of P5 into Jm109
- for tomorrow: inoculate miniprep cultures of positive clones on masterplate for P4, Colony-PCR and test gel for P5
Wednesday, 08/30/17
Colony PCR of P5 (low copy plasmid, see protocol) with Q19 and Q20
- gel picture: all clones okay
- inoculated three precultures for minipreps tomorrow (10 ml, 37 °C shaker)
- also inoculated three precultures of P4 for miniprep tomorrow (10 ml, 37 °C shaker)
Thursday, 08/31/17
- Miniprep of IG_C_P4 and P5 (three colonies each)
component
concentration in ng/µl
P4-2
370,8
P4-4
346,1
P4-8
287,5
P5-1
153,7
P5-3
137,9
P5-8
124,7
- Eppis named like this: "IG_C_P4_2_Mini"
- test digests:
P4:
colony
2
4
8
200 ng Plasmid
0,5 µL
0,6 µl
0,7 µl
NotI-HF
1 µl
1 µl
1 µl
10 x cutSmart buffer
1 µl
1 µl
1 µl
ddH2O
7,5 µl
7,4 µl
7,3 µl
total volume
10 µl
10 µl
10 µl
P5:
digest to see if mutation worked
colony
1
3
8
200 ng Plasmid
1,3 µL
1,5 µl
1,6 µl
EcoRI-HF
1 µl
1 µl
1 µl
10 x cutSmart buffer
1 µl
1 µl
1 µl
ddH2O
6,7 µl
6,5 µl
6,4 µl
total volume
10 µl
10 µl
10 µl
test digest in general:
colony
1
3
8
200 ng Plasmid
1,3 µL
1,5 µl
1,6 µl
SpeI
1 µl
1 µl
1 µl
HindIII-HF
1 µl
1 µl
1 µl
10 x cutSmart buffer
1 µl
1 µl
1 µl
ddH2O
5,7 µl
5,5 µl
5,4 µl
total volume
10 µl
10 µl
10 µl
we used HindIII-HF and NotI-HF from the group but bought one NotI-HF again for them ran digests on 0,7 % gel:
- P5 looks good on both digests,
- P4 only one band but probably just two bands which semm ike one so it should be fine
- glycerol stock was put in -80 °C of P4 2,4 and 8 and P5 1 and P5 3
Friday, 09/01/17
- diluted P4 / 2,4,8 and P5/ 1,3 to 100 ng/µl and Q21/q22 to 5 µM
- sent fo rsequencing as follows:
component
sequenze
P4-2 forward
33FE16
P4-2 reverse
33FE17
P4-4 forward
33FE18
P4-4 reverse
33FE19
P4-8 forward
33FE20
33FE21
P5-1 forward
33FE22
P5-1 reverse
33FE23
P5-3 forward
33FE24
P5-3 reverse
33FE25
Monday, 09/04/17
- prepared 2 ml over night culture for glycerol stocks
- IG_C_P4 clone 2, 4 and 8
- IG_C_P5 clone 1 and 3
Tuesday, 09/05/17
- Preparation for Gibson assembly: PCR with Q5 polymerase for P5 (to amplify LacI-dCas9 with overhangs to P4) and P4 (to open the plasmid)
- prepared dilutions of 1 ng/µl for each template:
template
starting concentration in ng/µ
water in µl
plasmid in µl
P5 clone 1
153.7
153
1
P5 clone 3
137.9
137
1
P4 clone 4
346.1
345
1
P4 clone 8
287.3
286
1
P4-clone 2
1
P4-clone 2
370.8
370
1
- Master Mix:
component
primer for P5
volume for P5 in µl
primer for P4
volume for P4 in µl
Q5 Master Mix
62.5
87.5
Primer 1
Q9
6.25
Q11
8.75
Primer 2
Q10
6.25
Q12
8.75
Water
47.5
66.5
sum
122.5
171.5
-> 49 µl Master Mix + 1 µl diluted Template (1 ng/µl -> 1 ng)
- PCR programs (name in Torsten: P4_P5_PCR-vor-Gibson)
- PCR programs how they should be: - amplified fragment size
- P4: 7309 bp - P5: 5680 bp
P5
P4
cycles
98°C 30s
62.5
98°C  10s
30
6.25
30
57°C 30s
30
62°C 30s
30
72°C 170s
30
72°C 220s
30
72°C 2 min
72°C 2 min
-> made a gradient PCR to performe both PCRs simultaneously!
-> P5 in well 5 (58.1 °C)
-> P4 in well 8 (61.9 °C)
- test gel (0.7 % agarose): used 2 µl of PCR product + 8 µl water + 2 µl LD (loading dye)
- 130 V, 30 min
Wednesday, 09/06/17
- remove master plate (Gibson Assembly for P4) from waste
- repeat colony PCR, because sequencing results are negative (all clones)
- expected size: 700bp
- colony PCR: difficult to analyze (too many bands)
- repeat PCR for IAAM_MS2
- 50 µl Reaction
- 25 µl Q5 MasterMix
- 2,5 µl Primer Q27
- 2,5 µl Primer Q28
- 1 µl Template
- 19 µl ddH2O
- IDT PCR Programm
- Anealing temperature = 68,3°C
- Extension time = 66s
- Trafo of Gibson Assembly (28.08.17; P4 in C2987) https://www.neb.com/protocols/2015/02/17/nebuilder-hifi-dna-assembly-chemical-transformation-protocol-e2621
- Gibson Assembly (5 µl) + competen cells of NEB (C2987)
- plate cells: 100 µl and 200 µl
- Gel and PCR clean up (Promega kit)
- LacI-dCas9 from P4: final concentration: 114,1 ng/µL
- tried Salkowski mixture with Nitrocellulose membrane (lab 1.17 above EtBr place, orange box)
- membrane did not dissolve, so we can do this now
Thursday, 09/07/17
- PCR for IAAH-PP:
-charging stock:
component
volume in µl
Mastermix (Q5 2x)
25
Q25
2.5
Q26
2.5
IAAH-PP7
1
ddH2O
19
- annealing temp. = 71°C
- extension time   = 55s
- Gel run: PP7 has expected size
- c of PP7 =409,1 ng/uL
- Colony PCR of Gibson Assembly:
- charging stock:
component
volume in µl
ddH2O
92,3
Q18
11.7
Q24
11.7
Taq Polymerase
144.3
- per aliquot: 20 µl
- picked 10 clones, just numer 2 was useful -> pre culture for miniprep with this one
- MS2:
- testgel with 2 µL of PCR product, 8 µL ddwater, 2 µL LD
- preparative gel
- gel clean up
- c = 10,8 ng/µL
Friday, 09/08/17
1. Redo of Gibson Assembly of IAAM-MS2 + IAAH-PP7 into psB4A5
- Gibson assembly (done with same ratios like before)
- pSB4A5 miniprep: 2,75 µL (18 ng/µL)
- IAAM-MS2 6,1 µL (10,6 ng/µL)
- IAAH-PP7 0,1 µL (409,9 ng/µL) <- only with different volume due to higher concentration (amplified DNA from 7.9)
- MasterMix 10,77 µL (fragments to Mastermix : 1:1)
- Trafo of Gibson Assembly (P4 in C2987)
- https://www.neb.com/protocols/2015/02/17/nebuilder-hifi-dna-assembly-chemical-transformation-protocol-e2621
- Gibson Assembly (5 µl) + competen cells of NEB (C2987)
- plate cells (Carb): 100 µl and 200 µl
2. Miniprep of clone 2 (see page 50 in labbook) with psB4A5: IAAM-MS2+IAAH-PP7
- used Promega miniprep kit + protocol
- concentration: 380,8 ng/µl
- made glycerol stock using 700 µl cells + 700 µl Glycerol 80 %
- Test digest of P4 (miniprep) with
- 0,53 µl DNA (miniprep)
- 0,5 µl KpnI
- 1 µl CutSmart
- 7,7 µl ddH2O
- Gel picture see lab book
Monday, 09/11/17
- pictures fare from the same gel, left looks a bit promising but with an extra band (maybe uncut plasmid?), right is with more lighting and only one band
- made colony PCR of Gibson plates from yesterday -> all 16 clones negative,
- Salkwoski was still prepared and has to be done tomorrow
- made restriction digest like yesterday to check if the correct insert is there and digest Friday was just bad but it was also negative
- Gibson assembly was done again:
component
volume in µl
IG_c_P1 linearized (18 ng/µL)
2,75
IAAM-MS2 (10,6 ng/µL)
6,1
IAAH-PP7 (19,8 ng/µL)
2,7
11,55
- IaaM from the 7.9. was empty!
- IAAH-PP7 was purified (it was not purified and the unpurified was used for last Gibson so there is probably the mistake)
- incubated gibson for 40 minutes at 50°C and plated on Amp
- plated with 4 g/L TrP which was done today, too
Tuesday, 09/12/17
- Colony PCR of the Gibson Assembly (P4)
- 11 clones, all negative
- Gel Electrophoresis: 0,7 %, 120V, 25 min
temperature in °C
time in sec
95
60
95
15
52
15
72
11
- 30 cycles
- Gibson Assembly
- 50 min at 50 °C
- approach
- 6,1µl IAAM-MS2 (10,6 ng/µl)
- 2,7µl IAAH-PP7 (19,8 ng/µl)
- 1,2µl ddH2O
- 10µl MasterMix
Wednesday, 09/13/17
- PCR
- approaches
1) 1 µl Gibson Assembly as template
2) 1 µl (1:10 diluted)Gibson Assembly as template
3) 1 µl (1:100 diluted)Gibson Assembly as template
- Master Mix
- 75 µl Q5 Master Mix
- 7,5 µl Primer Q26 (10µM)
- 7,5 µl Primer Q27 (10µM)
- 57 µl ddH2O
- PCR
- 49 µl Master Mix + 1 µl Template
temperature in °C
time in sec
98
30
98
10
71
30
72
120
72
120
10
- 35 cycles
- Test Gel
- 2 µl PCR Product + 8µl ddH2O + 2µl loading Dye
- all negative
- second PCR
- approaches
- 9 * 1:10 diluted (tube 1-9)
- 9 * 1:100 diluted (tube 10-18)
- PCR with gradient
temperature in °C
tube
60,1
1,10
60,9
2,11
61,8
3,12
63,1
4,13
64,4
5,14
65,6
6,15
66,9
7,16
68,2
8,17
69,1
9,18
Friday, 09/15/17
- prepared P5 for shipping to iGEM HQ
- used Q37/Q38
- PCR program: dCas9 iGEM
temperature in °C
time in sec
98
30
98
10
65
15
72
1:50 min
72
2 min
- standard pcr protocol with 50 µL reaction and 1 ng template
- ran gel for 50 min at 90 V
- expected size 5700 bp
- gel extraction (forgot test gel so I just cut it out even if it was just one band)
- labelled epp: "IG_CP5" for iGEM HQ
- final concentration: 95,7 ng/µl
- for monday: restriction with EcoRI and PstI, ligation into pSB1C3 and Trafo
Monday, 09/18/17
- PCR for IaaH-PP7 to get rid of erroneous overhang (overhang to IaaM-MS2)
- reaction mix
component
volume in µl
Q5 Master mix
25
IG_C_Q26 (10µM)
2,5
IG_C_Q39 (10µM)
2,5
IaaH-PP7 IDT (1:10)
1
water
19
sum
50
- annealing temperature: 71°C
- elongation time: 54 s
- expected fragment size: 1,8 kb
- Gel clean-up of PCR product
- 0.7 % agarose gel -> run 1 h at 80 V
- cut the band at ~ 1,8 kb
- purification of fragment with promega kit
- concentration of IaaH-PP7: 7,8 ng/µl
- Gibson assembly for P4
component
volume in µl
Assembly Master mix
15.65
IaaM-MS2 (10.6 ng/µl)
IG_C_Q39 (10µM)IaaH-PP7 (7.8 ng/µl)
6.8
pSB4A5 (IG_C_P1 linearized, 18 ng/µl)
2.75
sum
31.3
- incubation: 63 min at 50 °C
- transformation of 5 µl assembly mix into 50 µl competent DH5a (C2987)
- plated on amp + trp plates (200 µl and rest)
- digestion of PCR product IG_C_P5 for iGEM HQ (from 15.09., LacI-dCas9) with EcoRI and PstI
component
volume in µl
Cutsmart
1
EcoRI
1.5
PstI
1.5
LacI-dCas9 (400 ng)
4.18
water
1.82
sum
10
- incubation: 2h 10 min at 37 °C
- Gel clean-up of digestion product
- 0.7 % agarose gel -> run 1 h at 80 V
- cut the band at ~ 5700 kb
- purification of fragment with promega kit
- concentration of digested LacI-dCas9: 10.2 ng/µl
- Ligation of purified and digested LacI-dCas9 into pSB1C3 (digested with E + P, 28.08.)
component
volume in µl
T4 DNA Ligation Buffer
2.5
1
LacI-dCas9 (10.2 ng/µl)
10.7
pSB1C3 (12.5 ng/µl)
8
water
2.8
sum
25
- incubation: 1 h 8 min at room temperature
- transformation of 5µl ligation mix into 50 µl competent DH5a (C2987)
- plated on cap plates (200 µl and rest)
Tuesday, 09/19/17
- Colony PCR of dCas9-LacI in pSB1C3
- mastermix for 10 clones + negative control
component
volume in µl
water
100,8
Q17
9,6
-> annealing Tm 52°C, binds to backbone
Q20
9,6
-> annealing Tm 52,6°C, binds to dCas9
Red Taq
120
- fragment length: 1,6 kb -> elongation time 23 sek
- gel picture: except for clone 3 and 9 all clones positive
- inoculated miniprep cultures (5 ml LB + 5 µl CAP) of clone 4,7,8 and 10
- Colony PCR of P4
- mastermix for 10 clones + negative control
- Colony PCR of dCas9-LacI in pSB1C3
- mastermix for 10 clones + negative control
component
volume in µl
water
100,8
Q18
9,6
Q24
9,6
Red Taq
120
- annealing Tm 52°C
- fragment length: 700 bp -> elongation time: 10,5 sek
- gel picture: no bands!
- possible reasons: not enough DNA (IAAH-PP7 with corrected overhangs) applied due to low concentration (see 18.09.)  or colony PCR did not work properly. In case of that miniprep cultures were inoculated with clone 1,6,8 and 10 (10 ml LB + 10 µl Amp)
- Tomorrow test digest of those!
- master plate with cellulose membrane in 37 °C incubator
- Amplification of IAAH-PP7 with corrected overhangs
- gradient PCR: tried 1:10 and 1:100 dilutions
- master mix for 11 x 10 µl aliquots
component
volume in µl
water
48
Q39
8
Q26
8
Q5 master mix
80
- per tube: 9 µl master mix + 1 µl template (1:10, 1:100 or 1:1000)
- same gradient, same conditions
- gel: 1% agarose; 150 V, 30 min
- clear bands visible, all clones positive!
- annealing temperatures between 65-67 °C (all three equaly good) and concentration of 1:1000
( 0,01 ng/µl) worked best
- extracted all bands at 1,8 kb from gel -> three yellow labeled tubes in the fridge
Wednesday, 09/20/17
- PCR of iAAM-MS2 IDT fragment, (50 µL reaction, 1 ng template)
- 68,3 °C annealling Temperature, 30 cycles
- gel exrraction, final concentration: 5 ng/µL
- gel extraction of IAAH-PP7 from yesterday: 80 ng/µl
- Minipreps of Dcas9 in pSB1C3 and P4
Cas-4
47
Cas-7
66,2
Cas-8
48,9
Cas-10
64,4
Cas-10
64,4
P4-1
129,5
P4-6
158,2
P4-8
140,2
P4-10
173,7
- test digest of minipreps
- for P4 see to do list (KpnI)
- for dCas9: 200 ng Plasmid, CutSmart Buffer, EcoRI-HF and SpeI
- ran on gel with IAAM-MS2
- expected bands:
- P4: 5393 +  1916 bp
- for dCas9: 5701 bp + 2051 bp
- for MS2: 2200 bp
- MS2 was cut out
- P4 is negative, bands at 3 kb --> backbone without insert
- dCas9 was not cut
- checked Geneious again, primer were designed wrong, ordered new Q37/Q38
- Salkowski assay for P4 was done
- seems negative, need to improve protocol:
1. incubate 2 master plates (1 for assay, 1 for cloning)
2. mark nitrocellulose mebrane with cuts to indicate direction
3. negative control on master plate (amp resistacne!)
4. for example a clone from a Tac-4 Fra plate
- Gibson assembly for P4 done again
- used new IAAH-PP7: only 0.7 µL, 0.45 µL H2o and 10 µL Gibson master Mix
- incubation at 50 °C for 15 minutes
- plated on Amp + Trp plates
- 3 x 300 uL plated
Thursday, 09/21/17
- PCR MS2
- 50 µl approach
- 25 µl Q5 MasterMix
- 2,5µl IG_C_Q21
- 2,5µl IG_C_Q28 - 1 µl IAAM_MS2 (1:100)
- 19µl ddH2O
- 30 cycle, 44 s extension time
tube
annealing temp.
1
60,0°C
2
63,7°C
3
65,8°C
4
68,1°C
- Gel extraction from clone 1 and 2 (clone 3 with ladder and clone 4 negative)
- final conzentrations
- clone 1 = 19 ng/µl
- clone 2 = 10,3 ng/µl
- Colony PCR
- approach
- 100 µl AllIn RedTaq
- 44 µl ddH20
- 8 µl IG_C_Q18
- 8 µl IG_C_Q24
- program:
temperature in °C
time in sec
95
60
95
15
52
15
72
108
75
5 min
- 30 cycles
- gel: all clones negative
Friday, 09/22/17
- Gibson assembly of IAAH-PP7 (corr.) with IAAM-MS2
component
volume in µl
IAAH-PP7
0,7
IAAM-MS2
3,4
H2O
5,9
Hifi DNA Assembly Master Mix
10
- incubated at 50 °C for 40 minutes
- PCR to amplify Gibson product:
- Primer: Q26/27
- Product size: 4 kb
- 4 different dilutions as template: (1, 1:10, 1:100, 1:1000)
- standard 50 µL reaction
- ran test gel:
- only 1 and 1:10 dilutions show correct band (very thin gel extraction for 1 and 1:10 dilution) pooled in 1 eppi:
-> concentration of IAAM-MS2-IAAH-PP7: 17 ng/µl
- Gibson Assembly of IAAM-MS2-IAAH-PP7 with pSB4A5
component
volume in µl
IAAM-MS2-IAAH-PP7 (17 ng\µl)*
6.86
IG_C_P1 lin. (18 ng/µl)
2.75
H2O
0.39
Hifi DNA Assembly Master Mix
10
*(calculation for IAAM-MS2-IAAH-PP7: (4000 bp/ 3395 bp) * 50 ng * 2 = 116.6 ng Insert)
- Incubation 15 min at 50 °C
- Transformation of 5 µl assembly mix into DH5a from NEB
- inoculation of 6 liquid cultures (10 ml LB + 10 µl Amp) for miniprep of P4
-> used master plates from 21.09.
- resuspension of MS2 sgRNA and PP7 sg RNA from IDT in 50 µl TE-Buffer (in the lab accidentally written in the high copy lab book)
-> final concentration: 10 ng/µl
Saturday, 09/23/17
- colony PCR with P4 of yesterday
- only 9 colonies oon plate with centrifugated cells
- MasterMix:
- 60,5 µL H2O
- 11 µL Q18/Q24 primer
- 137,5 µL Taq Mastermix
- each 20 µL Mastermix (10 tubes, 9 colonies, 1 negative control)
- PCR program: "colony PCRp4"
temperature in °C
time in sec
95
60
95
15
52
15
72
1
75
30
- 40 cycles
- program was taken from Highqu website
- PCR product on 1 % Agarose, 100 V für 40 min
- all clones negative
- Miniprepped P4 clones from 20.09.
P4-1
152,1 ng/ul
P4-2
117,5
P4-3
130,2
P4-4
118,8
P4-5
91,6
P4-6
99,9
- test digest with KpnI-HF
- expected bands: 5393 + 1916 bp
- picture very weird
- 4 sometimes up to four bands:
- did not run through (over "10kb")
- about 7 kb, uncut plasmid ?
- about 5,5 kb, big insert
- 2,5 kb, too big for small insert
- talk with fabian to clarify
- clonme 3 was sent to sequencing
- IDs: FE34/35 (I forgot to label which one was forward and reverse becuase IAA-fusion has Biobrick prefix and suffix, we can use restriction and ligation fr P4 restricion digest):
component
<psB4A5 (66 ng/µl)
IAA-Fusion (17 ng/µl)
template
3 µl
7,5 µl
EcoRI-HF
1 µl
1,5 µ
PstI
1 µl
1,5 µl
NEB 2.1 Buffer
1 µl
1,5 µl
H2O
4 µl
3 µl
- incubated 2:40 h at 37 °C
- monday ligation and trafo
- amplification of IAA-Fsions:
- standard 50 µL PCR
- 4 different tempereatures ("ampli gibson")
52 °C
54,8 °C
58,2 °C
61 °C
- monday gel extraction
Monday, 09/25/17
- Gel  of IAA fusion (see Sept. 24th) at 120 V for 45 min
- expected lenght: 4000 bp
- aliquots 3 and 4  positive (annealing temp 58,2 °C and 61 °C)
- gel clean up with kit c= 11,1 ng/µl
- Did a test gel for Fabian with all parts that have been used during the work on P4 (25 ng each, 0,7%, 40 min at 120V)
track
name concentration
exp. lenght
1
P1 linearized (18 ng/µl)
3395 bp
2
P1_D_E+P (19,8 ng/µl)
3395 bp
3
IAAM-MS2 (10,6 ng/µl)
2200 bp
4
IAAM-MS2 (19 ng/µl)
2200 bp
5
IAAH-PP7 corr (80,1 ng/µl)
6
IAA fusion (11,1 ng/µl)
7
IAA fusion E+P (8,5 ng/µl)
- They're all okay!
- Only P1 linearized seems to have a lower concentration than written on tube
- Sent empty backbone (pSB4A5 miniprep) for sequencing with Q17 and Q18
- ID 33FE36 - pSB4A5 + Q17
- ID 33FE37 - pSB4A5 + Q18
- Ligation and transformation of iAA fusions from yesterday
- Ligation: incubation at room temperature for 6 h
component
volume in µl
P1
1,5 µl
IAA fusion
12,5 µl
water
3 µl
T4 buffer
2 µl
T4 ligase
1 µl
- Transformation of IAA fusions in pSB4A5 into competent DH5α (see protocol)
- plated on AMP + Trp (200 µl and rest)
Tuesday, 09/26/17
- Transformation of ligation of IAA-fusion from monday 25.09. (200 µl -> no colonies, pellet -> 2 colonies)
- Lena: Is this because of the Trp?
- Transformation of ligation of IAA-fusion in pSB4A5 in competent DH5α-cells (followed protocol)
- plated on Amp (200µl and rest) [notice from 27.09. : no colonies!!!]
- Colony-PCR of ligation of IAA-fusion in pSB4A5 (P4)
- Mastermix: 3x: 16,5µl ddH2O + 3µl IG_CQ18 + 3µl IG_C_Q24 + 37,5µl Taq
- 2µl Mastermix each
- 2 colonies + 1 neagtive control
- PCR-program "colony pcrp4": 95°C 1min, (95°C 15sec, 52°C 15sec, 72°C 1sec) x 40, 72°C 30sec
- PCR product on 1% agarose gel at 120 V for 40 min
- expected bands at 700 bp -> all negative
- P5 for sending to iGEM HQ preparation:
new Q37/Q38
- Standard PCR protocol with 50µl and 1ng template (IG_C_P5-1 1ng/µl from 5.9.)
- product on gel for 30 min at 120V
- gel extraction, one clear band
PCR-clean-up would have been better
- band at 5700bp
- c=10,1 ng/µl
Wednesday, 09/27/17
- 1) Miniprep of liquid culture P4 IAA-fusion from 26.09.17
Miniprep followed Promega protocol
elution in 75µl H2O (3x25µl)
Nandrop: 252,9ng/µl
Test-digestion in 10µl final volume with 0,5µl KpnI (2µl Miniprep)
- 2) PCR of IG_C_P5-1 (1ng/µl; 5.9.) with Q37 and Q38 repeated, because yesterday there was a really low concentration after purification
25µl Q5 MM, 2,5µl 10µM Q37, 2,5µl 10µM Q38, 1µl Template (1ng/µl), H2O 19µl
for the test gel: 2µl PCR product + 8µl H2O + 2µl LD
0,7% agarose gel, 50 min at 120V
- "gel picture"
- Band 1: Test digestion of P4 with KpnI (see point 1) -> negative
- Band 2: Test gel for PCR of P5 (more than 1 band, therefore Gel extraction/clean-up)
- Band 3: Digestion pYES (see LLPS)
- Digestion pYES (see LLPS)
- 3) LacI-dCas9 (26.09.17, 10,1 ng/µl)  despite of really low concentration digestion with EcoRI+PstI
15µl LacI-dCas9 (202ng) + 2µl Cutsmart + 1µl water + 1µl EcoRI-HF + 1µl PstI
final concentration of LacI-dCas9 was 7,6ng/µl
incubation for 1,5h at 37°C
heat inactivation for 20 min at 80°C
- 4) Beacause pSB1C3 is nearly empty: amplification with Q15 and Q16
25µl Q5 mastermix + 2,5µl IG_C_Q15 + 2,5µl IG_C_Q16 + 1µl Template (1ng) + 19µl H2O
Template: linearised backbone pSB1C3 from iGEM (25ng/µl) -> diluted to 1ng/µl
PCR program: "pSB1C3_Q5"
annealing temperature 67°C
elongation time: 1 min (2000bp)
Test gel: 2 µl PCR product + 8µl H2O + 2µl loading dye
0,7% agarose gel, 40 min at 120V
"agarose gel picture"
Band 1: Testgel of pSB1C3 (only 1 band at 2000bp -> PCR clean-up)
Band 2: Gel cleam-up of LacI-dCas9 from P5 (5700bp)
- 5) Gel clean up: pSB1C3 (see point 4): 57,1ng/µl; LacI-dCas9 iGEMHQ (see point 2): 8 ng/µl
- 6) again very low concentration of LacI-dCas9 -> gradient PCR with different concentrations to find optimal condition
tested concentrations: 1ng/µl; 0,1ng/µl; 0,001ng/µl
Master mix: 80µl Q5 MM + 8µl 10µM IG_C_Q37 + 8µl 10µM IG_C_Q38 + 48µl H2O
9µl master mix per tube + 1µl template (1ng; 0,1ng; 0,01ng)
gradient:
well
temperature
1
58
3
59
4
60,2
6
63,3
8
66,3
elongation time: 1min 50sec
PCR product stored at -20°C
tomorrow: Gel + Gel clean up, if several samples show clear bands
- 7) Digestion of new amplified pSB1C3 (see point 4) with EcoRI, PstI and DpnI
2µl NEBuffer 2.1 + 0,5µl EcoRI-HF + 0,5µl PstI + 0,5µl DpnI + 12,1µl water + 4,4µl pSB1C3 (57,1 ng/µl; 250ng; final concentration: 12,5ng/µl)
incubation: 1h at 37°C
heat inactivation: 20 min at 80°C
- 8) ligation of LacI-dCas9 (for iGEM-HQ)_d. E+P (see point 3) with pSB1C3_d_E+P+D (see point 7)
Insert: LacI-dCas9_d -> 5700bp
Vector: pSB1C3_d -> 2000bp
because of low concentration of LacI-dCas9 after PCR: only 25ng vector and 1:1 ratio of Insert to vector -> 71,25ng Insert
2µl T4 DNA Ligase Buffer (10x) + 1µl T4 DNA Ligase + 9,4µl Insert + 2µl vector + 5,6µl water
Incubation over night at 16°C
tomorrow: heat inactivation at 65°C for 20min
Thursday, 09/28/17
- gel electrophoresis of P5
-gel cleanup of 4 bands: final concentration 7,1ng/µl
Friday, 09/29/17
-colony PCR for LacI-dCas9 in pSB1C3
-4 clones, all negative
-amplification of P1 and IAA-fusion for assembly of P4:
Q40/Q41:
- amplification of P1 creating new overhangs for Gibson assembly --> 3,4 kb
Q42/Q43:
- amplificaton of IAA-fusion  creating new overhangs for Gibson assembly --> 4 kb
Q44/Q45:
- amplification of IAA-fusion creating overhangs for restriction enzymes to cut more efficiently
-made 50uL reactions and ran test gel (left 4 bands are colony PCR then 40/42/44)
-only 42 has correct band
-gel extraction: 33,7 ng/ul
-named Eppi "IAA-fusion for Gibson"
-gradient PCR for 40 and 44:
well
temperature in °C
1
52
4
55
5
57
6
59
7
61
8
63
9
65
12
68
-10 µL reaction and gel run
-40/1,4,5,6 and 44/1 were positive --> a PCR-clean-up has to be done
-repeat PCR from 27.07.2017:
A2: 1µl Template 59°C
A4: 1µl Template 63,3°C
A5: 1µl Template 66,5°C
B4 0,1µl Template 63,3°C
(A2, A4, A5 and B5 are the names of the original well names of the gradient PCR)
-just A 5 was positive -> a gel clean up has to be done the next day
Monday, 10/02/17
-gel-clean-up LacI-dCas9 (A5) and PCR-clean-up 40/1 and 44/1 (40/4,5,6 were found in the waste, 40/4 was empty, because open cap, but 40/5,6 were also clean-up by PCR clean-up and purifien over one column but separated from 40/1 to not possibly destroy the one safe sample)
-concentration: 40/1: 19,7 ng/µl
-40/5,6: 31 ng/µl (but probably waste/just denatured DNA or something else)
-44/1: 30,6 ng/µl
-LacI-dCas9: 25,9 ng/µl
-Eppi-names: "IAA-Fusion for Restriction" name of 44/1 (is from PCR with Q44 and Q45)
-"P1 + overhangs 1" name of 40/1 and "P1 + overhangs 5,6" name of 40/5,6 (is from PCR with Q40 and Q41)
-Restriction of IAA-Fusion (is from PCR with Q44 and Q45):
10,2µl IAA-Fusion (19,7ng/µl) --> 200ng
1,5µl Eco-RI-HF
1,5µl PstI
1,5µl NEB2.1
0,3µl water
-total volume: 15µl
-final concentration of IAA-Fusion: 13,3ng/µl
-was incubated 1h 35min at 37°C
-Ligation of digested IAA-Fusion in pSB4A5 = P1
-176,7ng Insert = 13,25µl Iaa-Fusion digested (13,3 ng/µl)
-50ng backbone = 2,5µl pSB4A5 digested (with EcoRI and PstI) (19.8ng/µl)
-1,25µl water
-2µl Ligase-buffer
-1µl Ligase
-total volume: 20µl
-90 min at room temperature
-for ligation an insert-backbone-ratio of 3:1 was choosen
-IAA-Fusion (Insert): 4000bp
-pSB4A5 (backbone): 3395bp
-(4000bp/3395bp) * 50ng * 3 = 176,7ng = m(Insert)
-Restriction of LacI-dCas9 for iGEM-HQ of Part P5:
-digestion of 400ng LacI-dCas9 = 15,4µl LacI-dCas9
-15,4µl LacI-dCas9 (25,9ng/µl)
-2µl NEB2.1
-0,5µl EcoRI
-0,5µl PstI
-1,6µl H2O
-total volume: 20µl
-final concentration LacI-dCas9: 20ng/µl
-was incubated for 1h15min at 37°C
-Ligation of digested LacI-dCas9 in pSB1C3 for iGEM-HQ:
-Ligation was sheduled in a 1:1 Insert:backbone-ratio, because the insert is much bigger than the backbone and a very high concentration of insert would be needed to get this ratio in a 20µl reaction. Such a high concentration we don't have. But the concentration was this time at least high enough for a 1:1-ratio with 50ng of backbone.
-size Insert: 5700bp
-size backbone: 2000bp
-(5700bp/2000bp)*50ng*1 = 142,5ng
-142,5ng LacI-dCas9 (20ng/µl) = 7,1µl LacI-dCas9
-7,1µl LacI-dCas9 digested (20ng/µl; 142,5ng)
-4µl pSB1C3 digested (12,5ng/µl; 50ng)
-2µl Ligase-buffer
-1µl Ligase
-5,9µl H2O
-total volume: 20µl
-incubated 90 min at room temperature
-Gibbson-Assembly of IAA-Fusion with P1=pSB4A5:
-2,5µl pSB4A5 linearised (19,7ng/µl; 50ng; P1 with new overhangs from today PCR-clean-up, Eppi-name: P1 +overhangs 1") -> we used 40/1, because even if 5/6 has a higher concentration, 5/6 was the whole weekend not cooled in the waste and we can not be sure, that what we measured as concentration is really our DNA-fragment of interest, if this Gibbson-Assembly should not work we can try with 5/6 or better we check 5/6 on a gel before
-3,5µl IAA-Fusion for Gibbson (33,7ng/µl; 116,6ng; from Friday 29.09.)
-4µl water
-10µl Gibbson-Master-Mix
-total volume: 20µl
-IAA-Fusion + pSB4A5 reaction for 19 min at 50°C incubated
-Transformation of
-IAA-Fusion in pSB4A5 (Restriction-Ligation)
-IAA-Fusion in pSB4A5 (Gibbson-Assembly)
-LacI-dCas9 in pSB1C3 (Restriction-Ligation)
-in chemo-competent E.coli-cells (followed transformation protocol)
-Inactivation of ligase of the both ligase reactions after transformation for 10min at 65°C
-eppis with names "Ligation IAA-Fusion in pSB4A5" and "Ligation LacI-dCas9 in pSB1C3" stored at -20°C
-Gibbson-Assembly with eppi name: "Gibbson-Assembly IAA-Fusion in pSB4A5" stored at -20°C
-plating of the cells after 1h incubation at 37°C: 200µl and pellet, respectively
Tuesday, 10/03/17
-colony-PCR of IAA-fusion in psB4A5 (Gibson-Assembly) (primer: IG_C_Q17 and IG_C_Q23)
-no colony-PCR of restriction-ligation, because no colonies
-colony-PCR of LacI-dCas9 in psb1C3 (restriction-ligation) (primer: IG_C_Q17 and IG_C_Q20)
-for all colony-PCR following PCR-program:
-initial denaturation: 1 min 95 °C
-denaturation: 15 sec 95 °C
-annealing: 15 sec 52 °C
-extension: 30 sec 72 °C
-final extension: 5 min 72 °C
-40 cycles
-run 10 µl PCR-reaction (each) on 0,7 % agarose gel
-result: - LacI-dCas9: all negative
- IAA-fusion in psB4A5: 1 and 4 possible postiv
=> ideas: dephosphorylate psb1C3 before ligation to prevent selfligation, because PstI in NEB 2.1- Buffer  75 % activity, so possible plasmid only cut by EcoRI
-minpreps of colonie 1 and 4 (IAA-fusion in pSB4A5)
Wednesday, 10/04/17
-Minipreps of IAA fusions in pSB4A5 of the colonies 1 and 4 (Promega kit)
-test digest with KpnI
component
volume
 plasmid
200 ng
 KpnI-HF
0,5 µl
CutSmart
1 µl
water
to 10 µl
-colony 1: concentration of plasmid : 65,4 ng/µl
-> 3,1 µl plasmid + 5,4 µl water for test digest
-colony 4: concentration of plasmid : 135,4 ng/µl
-> 1,5 µl plasmid + 7 µl water for test digest
-gel picture: expected bands at ~3400 bp and 1900 bp
-result negative: fragments of colony 1 much too big, bigger fragment of colony 2 okay but the smaller one is much too small
-Amplification of LacI-dCas9
component
volume
Q5 Master mix
25 µl
C_Q17
2,5 µl
C_Q18
2,5 µl
template (5..9.: "IG_C_P5", 1 ng/µl)
1 µl
water
19 µl
-PCR program:
steps
temp (°C)
time
cycles
Initial Denaturation
98°
30s
Denaturation
98°
10s
35x
Annealing
66,3°
30s
Elongation
72°
171s
Final elongation
75°
5 min
-Amplification of pSB1C3
component
volume
Q5 Master mix
25 µl
C_Q15
2,5 µl
C_Q16
2,5 µl
template (pSB1C3 linearized, 1 ng/µl)
1 µl
water
19 µl
-PCR program:
steps
temp (°C)
time
cycles
Initial Denaturation
98°
30s
Denaturation
98°
10s
40x
Annealing
65°, 66°, 67°
30s
Elongation
72°
90s
Final elongation
75°
5 min
-Test gel with LacI-dCas9 and pSB1C3 taken from the previous PCRs
-->each 2 µl sample + 8 µl water + 2 µl LD on 0,7% agarose gel
-->also on this gel (just to have a look at them):
IAA fusion (30,6 ng/µl) digested with EcoRI and PstI from October 2nd
pSB4A5 (19,8 ng/µl) digested with EcoRI and PstI
pSB4A5 with overhangs 1 (19,7 ng/µl)
pSB4A5 with overhangs 5/6 (31 ng/µl)
IAA fusion for Gibson (33,7 ng/µl)
--> digested 50 ng each
-gel picture:
-1-4) LaCI-dCas9 and pSB1C3 correct at 65° and 67° annealing temperature
--> PCR clean up + restriction and ligation
-6) IAA fusion digest: expected 4 kb -> looks good
-7-9) pSB4A5s should all have 3,4 kb -> also look good
-10) IAA fusion Gibson: no (!) bands at all: either the stated concentration is much too high or there is no DNA at all
--> did new amplification of IAA fusions with new overhangs for Gibson
-Restriction and ligation of IAA in pSB4A5
1) Restriction of new IAA fusions (4.10.)
component
volume
IAA fusion (30,6 ng/µl -> 200 ng)
6,5 µl
EcoRI-HF
1,5 µl
PstI
1,5 µl
NEB 2.1
1,5 µl
water
4 µl
-->final concentration of IAA: 13,3 ng/µl
-incubation for 1h at 37°
2)  Ligation of old IAA restriction digest (2.10.)
component
volume
IAA fusion (20,8 ng/µl -> 114 ng (everything))
5,5 µl
pSB4A5 digested
2,5 µl
water
9 µl
ligase buffer
2 µl
ligase
1 µl
3) Ligation of new IAA restriction digest (4.10.)
component
volume
IAA fusion digested (13,3 ng/µl ->176,7 ng)
13,25 µl
pSB4A5 digested
2,5 µl
water
1,25
ligase buffer
2 µl
ligase
1 µl
-incubation for 1h at 33room temperature
-Afterwards: Transformation into DH5α cells (see protocol), plated 200 µl and rest on AMP plates
-Amplification of IAA fusion for Gibson
component
volume
Q5 Master mix
25 µl
C_Q42
2,5 µl
C_Q43
2,5 µl
template (IAA fusion, 1 ng/µl)
1 µl
water
19 µl
-PCR program:
steps
temp (°C)
time
cycles
Initial Denaturation
98°
30s
Denaturation
98°
10s
35x
Annealing
63°
30s
Elongation
72°
120s
Final elongation
75°
5 min
-PCR clean up of LacI-dCas9 and pSB1C3 + following restriction
-clean up concentrations: LacI-dCas9: 161,8 ng/µl and pSB1C3: 106,0 ng/µl
-restriction of LacI-dCas9:
component
volume
LacI-dCas9 (161,8 ng/µl -> 250 ng)
1,5 µl
EcoRI-HF
1 µl
PstI
1 µl
NEB 2.1
2 µl
water
14,5 µl
-restriction of pSB1C3
component
volume
pSB1C3 (106,0 ng/µl -> 250 ng)
2,4 µl
EcoRI-HF
1 µl
PstI
1 µl
NEB 2.1
2 µl
water
14,1 µl
-total: 20 µl, final concentratio: 12,5 ng/µl
-heat inactivation for 20 min at 80°
-Ligation of LacI-dCas9 in pSB1C3
-->5700bp/2000bp * 50ng *1 = 142,5 ng
component
volume
LacI-dCas9 digested (12,5 ng/µl ->142,5 ng)
11,4 µl
pSB1C3 digested
4 µl
water
1,6
ligase buffer
2 µl
ligase
1 µl
-ligation over night -Preparation of
LB medium and Agar
1% and 0,7% agarose
SD-Ura medium (+Trp)
SD-Ura medium for plates (add 100 ml of 20% Galactose and 10% Raffinose -> already prepared)
40% Glucose
Thursday, 10/05/17
-Minipreps of clone 5 and 6 of IAA-fusion in pSB4A5 (from masterplate from 3.10.17)
-clone 5: 325,4ng/µl
-clone 6: 159,6 ng/µl
-test digestion with KpnI:
-clone 5: 1µl plasmid (325,4ng); 0,5µl KpnI-HF; 1µl Cut-Smart; 7,5µl water
-clone 6: 1,3µl plasmid (200ng); 0,5µl KpnI-HF; 1µl Cut-Smart; 7,2µl water
-incubated for 1h at 37°C
-put on 0,7% agarose gel
-Colony-PCR of IAA-fusion in pSB4A5 of the transformation of 4.10.17 (transformation of the restriction-ligation-approach) with the primers IG_C_Q24 and IG_C_Q18, at annealing temperature 52°C, else followed colony-pcr-protocol
-put 10µl on 0,7% agarose gel
-restriction of the at hte 4.10.17 amplified LacI-dCas9: 16µl LacI-dCas9 (161,8 ng/µl; 2588,5ng); 2µl NEB2.1; 1µl EcoRI/HF, 1µl PstI -> total volume 20µl, final concentration of LacI-dCas9: 129,44ng/µl
-incubated for 1 hour at 37°C
-Ligation of LacI-dCas9 in pSB1C3:
-(5700/2000)*50ng*2 = 285ng
-(5700/2000)*50ng*3 = 427,5ng
-Ligation was set up one time in an insert-backbone ratio of 1:2 and one time of 1:3:
-Ligation 1:2: 2,2µl LacI-dCas9 digested (digestion from 05.10.17); 4µl pSB1C3 digested (digestion from 04.10.17, 50ng); 13,8µl water, 2µl ligae-buffer, 1µl ligase
-Ligation 1:3: 3,3µl LacI-dCas9 digested (digestion from 05.10.17); 4µl pSB1C3 digested (digestion from 04.10.17, 50ng); 12,7µl water, 2µl ligae-buffer, 1µl ligase
-incubated for 6,5h at room temperature
-0,7% agarose gel with Colony-PVR results, results of test digestion and also of the amplification-PCR from 04.10.17
-Colony-PCR: no bands (could be expected, because the colonies looked very strange, didn't even know if it was really E.coli growing there)
-test digestion: clone 5: 2 bands, but wrong size, clone 6: just one band
-IAA-fusion aplification-PCR looks pretty good from size, but smear and more than one band -> therefore gel-clean-up (we load then the 48µl again on a gel and cleaned it up from there) --> 51,3ng/µl
-with the purified IAA-fusion there was made a Gibbson-Assembly:
-Gibbson-Assembly of IAA-fusion in pSB4A5:
-2,5µl pSB4A5 (50ng; 19,7ng/µl)
-2,27µl IAA for Gibbson (51,3ng/µl: 116,6ng)
-5,23 water
-10µl Gibbson-Assembly mix
-15 min at 50°C
-Transformation of the LacI-dCas9 Restriction-Ligation reactions in the ratios 1:1 (from 04.10.17 ligation over night at 16°C), 1:2 and 1:3 and of IAA-fusion in pSB4A5 (from Gibbson-Assembly from 05.10.17)
-followed transformation protocol, but using 25µl NEB 10 beta cells, because ther were no DH5alpha cells anymore
-plating 200µl and pellet
-overnight culture LacI-dCas9 in pSB1C3 10ml LB-CAM-media from transformation from 2.10.17 (was negative, but to maybe send to sequencing to check was was going inside of the backbone)
Friday, 10/06/17
-Miniprep of LacI-dCas9 in pSB1C3 -> 297,2ng/µl
-LacI-dCas9 in pSB1C3 from transformation from 5.10.17: every cloning ratio (insert:backbone 1:1, 1:2 and 1:3) show colonies on 200µl and pellet plate
-from each plates 5 clones were taken (1-5 from each ratio were from the pellet plate and 6-10 from each ratio from the 200µl plates)
-colony-PCR followed protocol but using 5µl reaction (because there was not enough Taq mastermix)
-PCR mastermix for 30 reactions: 27µl H2O + 24µl IG_C_Q17 + 24µl IG_C_Q20 + 75µl Tag-Mastermix
-PCR-program: initial denaturation: 1min 95°C, (denaturation: 15s 95°C, annealing: 15s 52°C, extension: 30s 72°C)x35, final extension: 5min 72°C
-put to 0,7% agarose gel (140V, 60min)
-band between 1,5kb and 2 kb could be the right one (but is really low, on the picture hardly visible but was identifiable on the picture in the computer)
-band size should be 1839bp
-clone 6 to 10 of the 1:1 ratio might be positive
-also on this gel picture is the IAA-Gibbson-Assembly of 4.10.17 (band size shpuld be 4kb, is at 4-5kb, but less concentration that shoudl be when comparing with the strange of the 1kb band that also should have 50ng like our IAA-fusion-fragment)
-therefore we repeat th Gibbson-Assembly with of IAA-fusion in pSB4A5 with the two-folde amount of IAA-fusion:
-2,5µl pSB4A5 + 4,54µl IAA for Gibbson + 2,96µl H2O + 10µl Gibbson-Assembly-Mix
- --> 15min at 50°C
-might be that there are more than the IAA-fusion-fragment inside of the sample, beacuse on the gel picture there were 2 bands directly stacked, that would be an explanation why the concentration of our IAA-fusion-fragment is lower than the measurement shows
Saturday, 10/07/17
-Miniprep of LacI-dCas9 in pSB1C3 (no labeling left on the flasks -> we gave them numbers from 1 to 5):
-1: 579.8 ng/µl
-2: 226.6 ng/µl
-3: 124.0 ng/µl
-4: 205.5 ng/µl
-5: 133.2 ng/µl
-stored in low copy box II (-20°C)
-accidentally used Lysis Buffer first instead of Resuspension Buffer!
-test digestion of minipreps
-Master mix:
-EcoRI-HF 3ul
-SpeI 3ul
-CutSmart 6 ul
-Water 42 ul
-9 µl Master mix per tube + 1 µl Miniprep
-9 µl Master mix per tube + 1 µl Miniprep
-incubation: 1 h, 37°C
-add 2 µl LD
-run a 0.7% agarose gel: 30 min, 130 V
-expected bands: 5701 bp + 2051 bp
-no positive clones
-Transformation of 5 µl gibson assembly product (P4, 06.10.17) in DH5a (NEB)
-plated on amp plates
-incubation: at 28°C until monday
Monday, 10/09/17
-colony-PCR of P4-Gibson of 7.10.:
-used Q5 polymerase because Taq is empty
-10 uL reaction for 20 colonies
-used Q17/Q23 primers which are wrong (Q18 and 24 are correct!)
-one reaction: Q5 5 uL
-Primer each 0,5 uL
-H2O 4 uL
-PCR protcol: (35 cycles)
-98°C 2min
-98°C 10sec
-52°C 20sec
-72°C 17sec
-72°C 2min
-ran gel, all negative (of course...)
-picked 31 clones again and remade colony PCR
-ran gel again, expected ands 700bp
-clones positive: 22,25,27,32,35,37,38,43,44,45,49,50,52,54,55,56 (after 6 1/2 weeks!!)
-minipreps angeimpft von: 22,25,27,32,35,37,43,44,55
-New way of assembling P4 (just in case that the colony PCRs are false positivs)
-new amplification IAAH-PP7 / IAAM-MS2 / psb4A5 with overhangs for Gibson assembly with different DNA polymerase
-2x25 uL reaction per amplification:
-5x PrimeSTAR CXL Buffer 10ul
-dNTP Mixture (2.5mM) 4ul
-Primer each 1ul
-Prime STAR GXL DNA Polymerase 42ul
-Water 31ul
--> 24 uL Mastermix + 1 uL Template
-IAAH-PP7
-Primer: Q43, Q39
-c(template)=0,01ng/ul
-IAAM-MS2
-Primer: Q42, Q28
-c(template)=0,1ng/ul
-pSB4A5
-Primer: Q40, Q41
-c(template)=1ng/ul
use of two different PCR programs:
-Denaturation: 98°C 10sec
-Annealing: 60°C 15sec
-Elongation: 68°C 2:30min
-Denaturation: 98°C 10sec
-Annealing/Elongation: 68°C 2:45min
pSB4A5 was digested again
-template (66ng/ul) 15.2ul
-NEB 2.1 3ul
-EcoRI-HF 1ul
-PstI 1ul
-H2O 9.8ul
final c(pSB4A5_d_E+P)=33.44 ng/µL
digested over night
The LacIdCAs9 in psB1C3 was send for sequencing:
IG_C_Q17 and LacIdCas9 2: 33FE40
Tuesday, 10/10/17
Miniprep of cultures from 9.10 (P4: pSB4A5 with IAA Fusion)
-Elution with 50 µL H2O
-Determination of the concentration
-clone 22: 371.5ng/ul
-clone 25: 358.8ng/ul
-clone 27: 351.1ng/ul
-clone 32: 372.8ng/ul
-clone 35: 414ng/ul
-clone 37: 413.5ng/ul
-clone 43: 370.9ng/ul
-clone 44: 393ng/ul
-clone 55: 425.6ng/ul
Test digestion using KpnI:
-Mastermix for 10 samples:
-10 µL 10x Cutsmart + 3 µL KpnI-HF + 77 µL H2O
--> 9 µL Mastermix + 1µL P4 plasmid (200 ng/µL)
-->digest for 1h at 37°C
---> add 2 µL 6x Loading Dye
---> Load on a 0.7 % agarose gel
---> run for 45 min at 120 V
-all plasmid samples (except clone 22 show 2 bands that are around the estimated fragment sizes (5393 and 1916 bp)
--> but they are slightly higher than expected
-->moreover the gell looks like it did not run proberly--> slightly curve of bands
--> the plasmids of clone 44 and 55 were send for sequencing using the Q17 and Q18 Primers
ID 33FE43 Clone 44+Q17
ID 33FE44 Clone 44+Q18
ID 33FE45 Clone 55+Q17
ID 33FE46 Clone 55+Q18
new amplification of IAAH-PP7, IAAM-MS2 and pSB4A5
Testgel of amplifications from 9.10:
2.5 µL PCR sample + 7.5 µL H2O + 2 µL 6x Loading Dye
-DpnI digestion of the PCR samples 1-6 (Adding 1 µL to the sample)
-Gel purification of the samples 1-6
pSB4A5 (sample1 and 2 combined) 10.6ng/ul
IAAH-PP7 (sample 3 and 4 combined) 17ng/ul
IAAM-MS2 (sample 5 and 6 combined) 12.2ng/ul
Check concentration with testgell
--> load 25 ng
-->concentration of samples seems to be correct
1)Gibson Assembly of IAAH-PP7 and IAAM-MS2 to form IAA-Fusion:
IAAH-PP7 (--> 50 ng) 2.94 µL
IAAM-MS2 (--> 61.11 ng) 5 µL
Gibson Mastermix 10 µL
H2O 2.06 µL
--> incubation for 40 min at 50 °C
2)PCRs using Gibson Assembly Product "IAA-Fusion)
Mastermix
20 µL 5x GXL buffer
8 µL dNTPs
2 µL Primer 1
2 µL Primer 2
2 µL GXL DNA Polymerase
32 µL H2O
3x 24 µL + 1 µL template (IAA-Fusion 1:10, 1:100, 1:1000)
2a) Primer: IG_C_Q42,  IG_C_Q43
--> Overhang for Gibson Assembly with pSB4A5
2b)Primer: IG_C_Q43, IG_C_Q45
--> Overhang for restrichtion and ligation in digested pSB4A5
PCR program:
Denaturation 98 °C 10 s
Annealing 60 °C 15 s
Elongation 68 °C 4 min
3)digested pSB4A5 from 9.10
Dephosphorylation of the digested pSB4A5:
27.5 µL pSB4A5_d_E+P
6 µL 10x Cutsmart buffer
3 µL Quick CIP
3 µL H2O
-calculated the volumes of the other components with a wrong volume of the digested pSB4A5, I thought it would be 48 µL
Incubation for 10 min at 37 °C
Heatinactivation for 2 min at 80 °C
c(dephosphorylated)=(27.5 µL*33.44 ng/µL)/39,5 µL=23.28 ng/µL
4)Salkowski Assay
-AMP+Trp LB-agar plate was inoculated with positive P4 clones from 9.10
-Additionally 2 negativ controls were used:
negative clone 24
clone 12 of TAC1_4Fra
-Incubation until 18.00 at 37 °C
--> over night at room temperature
-Test gel of IAAM-MS2, pSB4A5 and IAAH-PP7
-Result: stated concentrations seem to be correct!
Wednesday, 10/11/17
-Salkowski from yesterday did not grow at all (no(!) colonies)
New plate (Amp+Trp) was inoculated with the same clones as yesterday. Plate was placed in 37° incubator for ~3h, now on our bench over night. New membrane will be put on top tomorrow.
Test gel of IAA fusions (2,5 µl PCR product; 7,5 µl water; 2 µl LD)
PCR done with Q44/45 -> overhangs for Gibson
-PCR done with Q42/43 -> overhangs for restriction&ligation
-each applied in the dilutions 1:10, 1:100, 1:1000
-Gel picture: expected lengths: 4 kb for both
result: IAA fusions from PCR with Q44/45 show correct band (but also others)
-IAA fusions from PCR with Q42/43 show no bands at all -> apparently PCR did not work
-Did a gel clean up of remaining IAA fusions (Q44/45)
final concentration: 42,6 ng/µl
-digested the clean up of IAA fusions (Q44/45) with DpnI (1µl added to the whole aliquot)
-incubation for 30 min at 37°
-Test gel: clean up is clean, no other bands visible. Concenteration is probably lower than stated (<42,6 ng/µl)
-Repeated PCR of IAA fusion with Q42/43 from october 10th
three aliquots with the dilutions 1:10, 1:100, 1:1000
see volumes p.115
PCR program: PCR/GXL
Thursday, 10/12/17
-Transformation of ligation from 11.10.17 (IAA-fusion in pSB4A5) in competent DH5alpha (plated 200µl and rest on Amp-plates)
-Evaluation of sequencong results:
LacI-dCas9 in pSB1C3 clone 2 from 09.10.17: like from the gel (s:109) just the backbone is sequenced -> no LacI-dCas9 insert
P4 IAA-fusion in pSB4A5: sequencing results are positiv -> with clone 44 and 55 will now be continued, step 3 of High-Copy to do
--> 3) Insertion of LacI-dCas9 from pEX-A2
PCR extracting LacI-dCas9 was made at 6.9.17
PCR opening pSB4A5 with IAA-fusion
mastermix for 2 50µl reaction:
2x Q5 mastermix 50µl + 5µl Primer IG_C_Q11 + 5µl Primer IG_C_Q12 + 38µl H2O
-> 49µl + 1µl template (1ng/µl)
-> a) P4_clon44 (1ng)
->b) P4_clon 55 (1ng)
PCR program: initial denaturation 98°C 30s, (denaturation 98°C 10s, annealing 58°C 30s, elongation 72°C 3min39s)x35,final elongation 72°C 2min)
DpnI digestion: addition of 1µl DpnI
Incubation of 30min 37°C
load on 0,7% agarose gel
3µl (-> test gel -> to estimate the sizes of the bands)
rest (for gel clean up)
gel for 45min at 140V
Gel clean up of P4 linearised with Q11+Q12: 4,7ng/µl
bands from line 18 + 19 were cleaned up together, because bands were really really small, even if we can not be sure that there are really the same or one of them has a punct mutation
Gibson Assembly)
P4 linearised
c=4.7ng/ul
7309bp
50ng
10.6ul
LacI-dCas9 (with overhangs to P4)
c=114.1ng/ul
5690bp
77,7ng
0,7ul
Mastermix
c=2x
11.3ul
-transformation of 5ul Gibson Assembly into 25ul competent DH5alpha cells, plated on AMP plates (200ul+rest)
Friday, 10/13/17
Colony-PCR transformation 12.10.
mastermix for 12 reactions
120µl 2xTaq-mastermix
4,8µl IG_C_Q17
4,8µl IG_C_Q23
110,4µl H2O
20µl per tube (10tubes) pick clone from plates transfer to masterplate and then resuspend cells in mastermix
PCR program:
initial denaturation: 95°C 30s
denaturation: 95°C 20s
annealing: 52°C 30s
elongation: 68°C 77s
final elongation: 68°C 5min
-load to 1% agarose gel
60min 80V 30min 140V
no positive clones
troubleshooting:
-It was taking a look to the sequences and the problem was found: As we changed from SLiCE to Gibson NOT ALL OF OUR PRIMERS WERE CHANGED
-therefore following problems are found:
1.) P4 vector can not be opened with Q11 and Q12, because Q12 is bidning to the BioBrick Prefix, that P4 does not have anymore, because of the change to Gibson
2.) The LacI-dCas9 fragment has a fase overhang on one end, same reason
-> new primers were ordered to solve the problem unfortunately they will be most likely not be there before thursday
Saturday, 10/14/17
-Test gel with sgRNAs of IDT (PP7 and MS2)
-for both:
-2,5 µl fragment (10 ng/µl-> 1:10 of 100 ng/µl) + 7,5 µl water + 2 µl LD test gel (1%, 30 min)
-No bands at all (wrong dilution -> fragments were already at 10 ng/µl and not at 100 ng/µl )
-repetition, with same dilution but also the undiluted fragments (we wanted to see if there was any DNA at all)
-undiluted fragments turned out to have 10 ng/µl
-result: both positive with equal concentrations
Thursday, 10/19/17
sgRNAs:
MS2 sgRNA
PP7 sgRNA
Primer 1
IG_C_Q63
IG_C_Q64
Primer 2
IG_C_Q65
IG_C_Q66
Annealing temperature
70 °C
72 °C
size
292 bp
263 bp
Master mix for 6 samples
component
volume
Q5 Reaction buffer
30 µL
10 mM
3 µL
Q5 Polymerase
1.5 µL
GC Enhancer
30 µL
ddH2O
64 µL
-21.5 µL Mastermix + 1.25 µL Primer 1 + 1.25 µL Primer 2 + 1 µL template
-(templates: 0.1 ng/µL and 1 ng/µL of the sgRNAs)
-testgel: 2 µL PCR sample+ 8 µL H2O + 2 µ Loading Dye
--> samples were nearly run out of the gel--> but it looks like there is only one band
--> PCR Clean up
--> determination of the concentration:
sgRNA-Ms2: 169.1 ng/µL
sgRNA-PP7: 155.2 ng/µL
--> testgel with 25 ng sgRNA:
-sgRNA bands were at the expected height and no additional bands were visible
-the bands show also an expected intensity
-Opening of the P4 plasmid
-->the GXL Polymerase was used
component
volume
5x GXL buffer
10 µL
10 mM dNTPs
4 µL
IG_C_Q59
1 µL
IG_C_Q60
1 µL
GXL DNA Polymerase
1 µL
H2O
31 µL
--> 24 µL Mastermix + 1 µL template (P4 clone 44/55, 1 ng/µL)
PCR Program:
temp (°C)
time
98 °C
10 s
68 °C
7 min 18 s
--> was running over night
Friday, 10/20/17
-Opening of the P4 plasmid
-2 µL PCR sample + 2 µL Loading Dye + 8 µL H2O
-> Load on a 0.7 % agarose --> run for 30 min at 120 V
--> one clear band is visible
--> we will do a PCR clean up, but before we digest the samples with DpnI:
--> add 1 c DpnI to the sample--> incubate at 37°C for 80 min--> heat inactivation at 80 °C for 20 min
--> PCR Clean-up and determination of the concentration:
P4 clone 44: 70 ng/µL
P4 clone 55: 83.2 ng/µL
-Preparation of parts for iGEM HQ:
-IAA Fusion from P4:
componente
volume
Primer 1
IG_C_Q68
Primer 2
IG_C_Q67
Annealing temperature
63 °C
size
4000 bp
-LacI-dCas9 from P5:
componente
volume
Primer 1
IG_C_Q37
Primer 2
IG_C_Q38
Annealing temperature
65 °C
size
5728 bp
-PCR Mastermix for 3 samples:
component
volume
5x Q5 reaction buffer
15 µL
10 mM dNTPs
1.5 µL
Primer 1
3.75 µL
Primer 2
3.75 µL
GC Enhancer
15 µL
Q5 DNA Polymerase
32.25 µL
-24 µL Mastermix + 1 µL template
template 1: P4 clone 44
template 2: P4 clone 55
template 3: P5 (1 ng/µL solution from 5.9.17)
--> start PCR programm:
component
temp (°C)
time
Initial Denaturation
98 °C
Denaturation
98 °C
35x
Annealing
63 °C/ 65 °C
Elongation
72 °C
final Elongation
72 °C
--> 2 µL PCR sample + 2 µL Loading Dye + 8 µL water
--> Load on 0.7 % Agarose gel
--> run for 30min  at 120 V
--> Purification: PCR clean up for LacI-dCas9, Gel clean up for IAA-Fusion
--> DNA concentration:
-LacIdCas9: 59.1 ng/µL
-IAA-Fusion (P4 clone 44): 39.8 ng/µL
-IAA-Fusion (P4 clone 55): 51.1 ng/µL
--> Digestion:
component
volume
template (LacIdCas9/IAA-Fusion (P4 clone 44)/IAA-Fusion (P4 clone 55))
16 µl
EcoRI-HF
1 µl
PstI
1 µl
NEB buffer 2.1
2 µl
--> incubate at 37°C for 1h
--> Heat inactivation for 20 min at 80 °C
--> Ligation:
-Composition of ligation sample:
component
volume
template (digested LacIdCas9/IAA-Fusion (P4 clone 44)/IAA-Fusion (P4 clone 55))
x µl
T4 Ligase
1 µl
T4 Ligase buffer
2 µl
pSB1C3
4,75 µl
water
to 20 µl
-Needed volumes of template "x":
LacIdCas9:  7.2 µL
IAA-Fusion (P4 clone 44): 7.5 µL
IAA-Fusion (P4 clone 55): 5.9 µL
--> Incubation at 16 °C over night
Monday, 10/21/17
-Transformation of ligation (20.10) LacI dCas9, IAA-Fusion (clone 55) and IAA-Fusion (clone 44) in psB1C3
-5 µl template each in DH5α (25 µl)
-> plated on CAP
-> incubation over night at 37 °C
label: IAA-Fusion clone 55
1: with7,4 µl template
2: with 5,9 µl template
Sunday,10/22/17
-Colony PCR of the Transformations from saturday
-LacI-dCAS9 in pSB1C3:
Primer 1
IG_C_Q17
Primer 2
IG_C_Q20
Annealingtemp
53°c
Elongation time
1min 39s
size
1642 bp
-IAA Fusion in pSB1C3
Primer 1
IG_C_Q18
Primer 2
IG_C_Q24
Annealingtemp
52°c
Elongation time
44s
size
725 bp
-Mastermix for 12 samples:
component
volume
Quick Load Taq Mastermix
120 µL
Primer 1
4.8 µL
Primer 2
4.8 µL
H2O
110.4 µL
-aliquote 20 µL Mastermix in the tubes and a resuspended clone
10x IAA Fusion (out of P4 clone 44) in pSB1C3 + control
10x IAA Fusion (out of P4 clone 55) in pSB1C3 + control
10x LacIdCAs9 in pSB1C3 + control
--> Start PCR Program
steps
temp (°C)
time
Initial Denaturation
95 °C
30s
Denaturation
95°C
20s
Annealing
(52°C/53°C)
30s
Elongation
68°C
(44s/1min 39s)
Final Elongation
68°C
5min
--> Load on a 0.7 % Agarose gel
--> Colony PCR for the IAA Fusion was positive --> Preparation of 5 mL liquid culture (CAM) using the clones indicated with red dot
-->Colony PCR of LacIdCas9 was negative, Althought we prepared 5 mL liquid culture (CAM) using 2 clones (indicated with red dot)
(check if negative result is due to PCR or transformation in yeast--> an monday a testdigestion should be conducted)
Monday, 10/23/17
-Minipreps of IAA fusion in pSB1C3 and LacI-dCas9 in pSB1C3 (22.10.) according to protocol
concentrations (IAA fusion P4 (44))
concentrations (IAA fusion P4 (55))
c(1) = 662,5 ng/µl
c(4) = 466,4 ng/µl
c(2) = 630,9ng/µl
c(5) = 518,5 ng/µl
c(3) = 397,5 ng/µl
c(7) = 535,3 ng/µl
c(4) = 536,3 ng/µl
c(8) = 501,6 ng/µl
c(6) = 604,9 ng/µl
c(10) = 410,6 ng/µl
c(7) = 593,4 ng/µl
c(8) = 617,4 ng/µl
c(9) = 520,8ng/µl
c(10) = 537,0 ng/µl
-> Now: decision which of those will besent to iGEM HQ -> DNA has to be dried
-Gycerol stocks of clone 1 and 7 (1:1)
-Test digest of those
component
volume
template (miniprep)
2 µl
NEB 2.1
1 µl
EcoRI-HF
0,2 µl
PstI
0,2 µl
H2O
6,6 µl
-incubation for 1h 15 min at 37°
-Test gel (0,7% agarose)
-expected lengths:
IAA fusion: 4300 bp
pSB1C3: 2070 bp
LacI-dCAs9: 5760 bp
-gel run unusual: curved bands, ladder unclear
-Bands not exactly were they should be
-> IAA fusion P4 (44) clones all positiv
-> IAA fusion P4(55) clones 1+2 -> 3 bands (but others positive)
-LacI-dCas9 definetely negative
-PCR for P6
component
volume
5x Q5 reaction buffer
5 µl
10 mM dNTPs
0,5 µl
10 µM IG_C_Q69
1,25 µl
10 µM IG_C_Q62
1,25 µl
Q5 Polymerase
0,25 µl
GC Enhancer
5 µl
ddH2O
10,75 µl
Program:
steps
temp (°C)
time
cycles
Initial Denaturation
98 °C
80 s
Denaturation
98 °C
10 s
40x
Annealing
72°
30 s
Elongation
72 °C
2:51 min
final Elongation
72 °C
5 min
-Test gel: negative (gel again weird!)
-Repeated gel: negative again (but also ladder looked strange: made new Agarose and discarded the old)
-Repeated PCR (over night)
-Please put on gel tomorrow
Tuesday, 10/24/17
-TestGel of PCR (120V, 0,7%, 30min)
-->all clones negativ => gradient PCR
-gradient PCR of IG_C_P5_I (1ng/µl)
Master Mix:
component
volume
5x Q5 reaction buffer
35 µlm
10 mM dNTPs
3,5 µl
10 µM IG_C_Q69
8,75 µl
10 µM IG_C_Q61
8,75 µl
Q5 Polymerase
1,75 µl
GC Enhancer
35 µl
ddH2O
75,25 µl
Program:
steps
temp (°C)
time
cycles
Initial Denaturation
98 °C
30 s
Denaturation
98 °C
10 s
40x
Annealing
tube 1 = 71,6°C
30s
tube 2 = 69,4°C
30s
tube 3 = 73,8°C
30s
tube 4 = 68,5°C
30s
tube 5 = 74,8°C
30s
Elongation
72 °C
2 min 15s
Final elongation
72 °C
5 min
-Media for GC-MS-Cultures:
a)
component
volume
M9 Minimal salts
500 ml
20% Glucose
20 ml
MgSO4
2 ml
CaCl2
1 ml
Trace element mix
1 ml
10g/L Tryptophan
400 ml
water
76 ml
b)
component
volume
M9 Minimal salts
500 ml
20% Glucose
20 ml
MgSO4
2 ml
CaCl2
1 ml
Trace element mix
1 ml
water
476 ml
*trace element mix:
component
mass
H3BO3
28,6mg
MnCl2*4H2O
18,1mg
ZnSO4*7H2O
2,22mg
Na2MoO4*2H2O
3,9mg
CuSO4*5H2O
0,8mg
Co(NO3)2*6H2O
0,5mg
water
to 10 ml
Wednesday, 10/25/17
-gel clean up of LacI-dCas9 (gradient PCR from yesterday)
final concentration: 10,1 ng/µl
-new test digest of the IAA fusions (because  gel from october 23rd looked so strange)
-batch and expected sizes see low copy lab book p.132
-Result: all four clones (44 (1), 44 (8), 55 (3) & 55 (7) - they were picked randomly) are positive!
-Help for Berlin: digest of Cap TaC3 3 Fra to gain pSB1C3
component
volume
template (Cap TaC3 3 fra) (143,6 ng/µl)
14 µl
NotI-HF
2 µl
CutSmart
2 µl
water
2 µl
-Transfered some µl of this to in new tube, just in case we still need this
Thursday, 10/26/17
-gibson assembly of LacI-dCas9Q62/Q69 with P4 Q59,Q60
-did it for both P4 clones: 44 and 55
component
volume
HiFi DNA Assembly Mix
10 ul
P4 (70 / 83,2 ng/ul)
44: 0,36 ul/ 55: 0,3 uL
LacI-dCas9 (10,1 ng/ul)
5,86 ul
ddH2O
3,78 uL/3,84 uL
-probably won't work because LacI-dCas9 was left on the bench ver night
-incubated at 50°C for 20 min
-transformed into JM109 competent cells
-platend on Amp
-prepared parts for shipping to iEM HQ:
IAA-RBP (K2483000)
TaC1 (K2483005)
TaC3 (K2483006)
Ddx4-YFP (K2483007)
-preparing overnight cultures of E-coli for GC-MS measurements:
-all cultures in minimal media with or without tryptophan
8 x JM109 cells without plasmid (4x with and 4x without tryptophan)
8 x empty pSB4A5 backbone (4x with and 4x without tryptophan)
8 x media without anythin else (4x with and 4x without tryptophan)
4 x P4 clone 44 (2x with and 2x without tryptophan)
4 x P4 clone 55 (2x with and 2x without tryptophan)
Friday, 10/27/17
-shipping of Parts (prepared on 26.10.17)
-centrifuged and extracted cultures for GC-MS-calculation (inoculated on 26.10.17)
-amplification of LacI-dCas9 with homologous overlabs to P4 with IG_C_Q26 and Q69
-PCR Program of 25.10.17 with 4 diffrent Temperatures ( 66°C-69°C)
-->gel clean up: 137,7 ng/ul
Saturday, 10/28/17
-Gibson assembly of LacI-dCas9 and P4 (clone 44 and 55)
P4 (44)
P4 (55)
component
volume
component
volume
HiFi DNA Assembly Mix
10 ul
HiFi DNA Assembly Mix
10 ul
P4, 70 ng/µl (44)
0,7 µl
P4, 83,7 ng/µl (55)
0,6 µl
LacI-dCas9
0,6 µl
LacI-dCas9
0,6 µl
ddH2O
8,7 µl
ddH2O
8,8 µl
-incubation for 20 min at 50°
-Transformation of both Gibson assemblies into competent JM109
-plated 110 µl of both clones on one Amp plate (plate divided) -> only one Amp plate left!
-Plated the rest on Amp+Trp plates
-> in 37° incubator over night
Sunday, 10/29/17
-Transformation from yesterday (28.10): no colonies
Did you see this?
Abstract
What we've achieved
iGEM Goes Green
Who we are...
Background information
Crowdfunding
Main sponsors:
