Team:Ionis Paris/25 07 16
Project
Project description
Biology
Engineering
Applied Design
Entrepreneurship
In the lab
Working at La Paillasse
Biobrick design
Cloning Strategy
Protocols
Notebook
Side-projects
CelloCad
Collaborations
Mobile application
Education
Results
Parts and Characterization
Proof of concept
Demonstration
Human Practices
Overview
Project Genesis
Air Pollution
Opinion Survey
Ethics and Biosafety
Our Actions
Safety
Lab safety
Project safety
Team
The team
Events
Attributions
Acknowledgement (Sponsors and advisors)
Prizes
Project
Home Page 1
Home Page 2
Home Page 3
Home Page 4
Home Page 5
Home Page 6
Home Page 7
Home Page 8
Home Page 9
Services
About
Portfolio
Protfolio 1
Protfolio 2
Protfolio 3
Case Study
Blog
Blog Grid
Blog Left Sidebar
Blog Right Sidebar
Single Blog Full
Single Blog Left Sidebar
Single Blog Right Sidebar
Contact US
July 25th 2016
PCR colony: on colonies transformed by BB1 and BB2
NB: BB1 and BB2 are the ligation products of P1 + pSB1C3 and P2 + pSB1C3.
Objectives
The overall purpose is to check if the bacteria obtain from the transformation with BB1 and BB2 are containing the good genetic construction.
Materials
Bacteria tansformed with BB1 and BB2 (made on 21/07/16)
Primers: A12 (forward) and A13 (reverse).
Protocol
PCR
1.  1 Mix for 18 samples of BB1 (Total volume Mix : 900 µL) in a Eppendorf tube :
747 µL H2O
90 µL Buffer Taq (1 X final, NEB #B9014S)
18 µL Primer A12  (1µM final)
18 µL Primer A13 (1µM final)
18 µL dNTP (200µM final, NEB #N0447S)
9 µL Taq polymerase (2.5units/50 µL PCR final, NEB #M0273S
2.  1 Mix for 7 samples of BB2 (Total volume Mix :  350 µL) in a Eppendorf tube :
290.5 µL H2O
35 µL Buffer Taq (1 X final, NEB #B9014S)
7 µL Primer VF (1 µM final)
7 µL Primer VR  (1 µM final)
7 µL dNTP (200µM final, NEB #N0447S)
3.5 µL Taq polymerase (2.5 units / 50 µL PCR final, NEB #M0273S)
3.  Add in 21 PCR tubes, in the respected order:
50 µL Mix
One colony from the different petri dish (pick one colony, put some on a new, divided into squares  petri dish and put the remaining bacteria into the PCR mix)
Gently mix the reaction
4.  Short spin centrifugation
5.  Set the following parameters for the PCR reaction :
Part 1 (532 bp)
Lid température 95°C
Initial denaturation : 95°C, 5 min
30 cycles of : 95°C, 30 s
58°C, 1 min
68°C, 30 s
Final extension : 68°C, 5 min
Hold : 4°C
Part 2 (1,785 bp)
Lid température 95°C
Initial denaturation : 95°C, 5 min
30 cycles of : 95°C, 30 s
58°C, 1 min
68°C, 1 min 42 s
Final extension : 68°C, 5 min
Hold : 4°C
Electrophoresis for screening the PCR results
1% Agarose gel:
Put 1g of agarose low melting point + 100 mL of TAE 1 X in a bottle of 500 mL
Mix and heat it 2 min 30 s in the microwaves. Wait the cooling of the bottle until it is tepid.
Add 5 µL of Gel Red 10,000X (0.5 X final)
Flow the gel and place the combs
Wait until it is solidified. Remove slowly the combs.
Drop-off:
Speed centrifugation of samples.
Addition of 2 µL of Purple loading dye 6 X in 10 µL of sample.
Drop-off 10 µL of Purple ladder and 12 µL of each samples.
Run at 80V
Mini-culture: bacteria transformed with BB1
11 Mini-cultures of bacteria transformed with BB1 (colonies 1,2,3,7,8,9,10,11,12,14,15):
Put the colony with satisfying PCR results from the plates divided into squares to a 50 mL Falcon tube containing 5 mL LB+CHL.
Results
Electrophoresis for screening the PCR results
1st electrophoresis: Expected results / Obtained results:
2nd electrophoresis: Expected results / Obtained results:
Interpretation
We obtain the desired strip for BB1, as shown on the gel above the strips are close to 0.5 kb, which is the size of P1. A sequencing is necessary to be sure of the obtained biobrick.
However, concerning P2, some strip can be seen at 100 pb, meaning that P2 has not been ligated into pSB1C3, our bacteria have integrated only the plasmid pSB1C3 that has been closed up on itself.
Transformation efficiency test : Competent DH5⍺ cells with pSB1C3-RFP plasmid
Objectives
The objective is to test the competency of the DH5⍺ cells done on the 23/07/16 with the stock of pSB1C3-RFP plasmid.
Materials
Plasmid DNA : « pSB1C3-RFP 1 » at 85.151 ng/µL
DH5⍺ competent cells (23/07/16)
Petri dish LB+Cm: Cm concentration = 25 µg/mL
Protocol
Experimental conditions:
We need 5 LB+Cm plates + 2 LB plates
Thaw 1 tube of DH5α competent cells on ice for 10 min. Mix gently and carefully pipette 50 µL of cells into 2 transformation tubes on ice.
Add 1 µL (85.151 ng) of plasmid DNA to the cell mixture. Carefully flick the tube 4-5 times to mix cells and DNA. Do not vortex.
Place the mixture on ice for 30 min. Do not mix.
Heat shock at exactly 42°C for 30 s. Do not mix.
Place on ice for 5 min. Do not mix.
Pipette 950 µL of room temperature SOC into the mixture.
Place at 37°C for 60 min at 250 rpm.
Warm selection plates to 25°C.
Mix the cells thoroughly by flicking the tube and inverting.
Spread 100 µL of each dilution (100%,10%,1%) onto a selection plate.
Pellet the remaining cells (except for the control), discard mostly all the supernatant and resuspend the remaining cells in 100 µL of LB. Spread 100 µL onto a selection plate.
Incubate all the plates O/N at 37°C.
Results
Expected results :
Some colonies on the petri dishes LB+Cm plated with 1% of transformed bacteria, more with 10%, a lot with 100% and a bacterial lawn with the pellet.
A bacterial lawn is expected on the LB petri dishes (+ control).
No colony is expected on the LB+Cm petri dishes plated with no plasmid (- control).
All obtained colonies are expected to be red because of the RFP gene that codes for a red fluorescent protein.
Obtained results:
We obtain the expected results.
Interpretation
The transformation worked, we obtained satisfying results. Colonies contain a plasmid with the Chloramphenicol resistance gene, present in pSB1C3. The DH5α cells are competent.
June 18,2015
2
Creative and impressive building
June 18,2015
2
How you can be a super creative
June 18,2015
2
World best photographer and photography
Facebook
Twitter
youtube
LinkedIn
iGEM IONIS
We're a group of six different schools from the IONIS Education Group. For this                                     competition we wanted to take advantages of the multiple schools and activity field                                     given by the IONIS education group to create a solid project.
Read More
Stay in touch
Location: 66 Rue Guy Môquet, 94800 Villejuif, France
email: igem.ionis@gmail.com
Download the app
Home
in the Lab
Side Projects
Results
Human                                             Practice
Team
©IONIS_IGEM_2016.
