<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Ionis_Paris_Proof skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Ionis Paris/Proof</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="row top_menu_area"><DIV class="col-xs-12 col-sm-9"><NAV class="navbar-collapse collapse in"><UL class="nav navbar-nav navbar-right menu"><LI class="smoth-scroll"><A data-hover="Project">Project</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Description">Project description</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Biology">Biology</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Hardware">Engineering</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Design">Applied Design</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Entrepreneurship">Entrepreneurship</A></LI></UL><LI class="smoth-scroll"><A data-hover="In the lab">In the lab</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Working_at_La_Paillasse">Working at La Paillasse</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Biobrick_Design">Biobrick design</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Cloning_Strategy">Cloning Strategy</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Protocols">Protocols</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Notebook">Notebook</A></LI><LI class="smoth-scroll"><A data-hover="Side-projects">Side-projects</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Measurement">CelloCad</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Collaborations">Collaborations</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Mobile_application">Mobile application</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Engagement">Education</A></LI><LI class="smoth-scroll"><A data-hover="results">Results</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Parts">Parts and Characterization</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Proof">Proof of concept</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Demonstrate">Demonstration</A></LI><LI class="smoth-scroll"><A data-hover="Human Practices">Human Practices</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/HPintro">Overview</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Quantifly">Project Genesis</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Air_pollution">Air Pollution</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Survey">Opinion Survey</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Ethics">Ethics and Biosafety</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/IntegratedHP">Our Actions</A></LI><LI class="smoth-scroll"><A data-hover="Safety">Safety</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Labsafety">Lab safety</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Projectsafety">Project safety</A></LI><LI class="smoth-scroll"><A data-hover="Team">Team</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Team">The team</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Events">Events</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Attributions">Attributions</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Acknowledgement">Acknowledgement (Sponsors and advisors)
</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Prices">Prizes</A></LI></NAV><NAV class="mobile_menu hidden"><UL><LI class="smoth-scroll"><A href="#body">Project</A></LI><LI><A href="index.html">Home Page 1</A></LI><LI><A href="index-2.html">Home Page 2</A></LI><LI><A href="index-3.html">Home Page 3</A></LI><LI><A href="index-4.html">Home Page 4</A></LI><LI><A href="index-5.html">Home Page 5</A></LI><LI><A href="index-6.html">Home Page 6</A></LI><LI><A href="index-7.html">Home Page 7</A></LI><LI><A href="index-8.html">Home Page 8</A></LI><LI><A href="index-9.html">Home Page 9</A></LI></UL><LI class="smoth-scroll"><A href="#services">Services</A></LI><LI class="smoth-scroll"><A href="#about">About</A></LI><LI class="smoth-scroll"><A href="#Portfolio">Portfolio</A></LI><LI><A href="protfolio-1.html">Protfolio 1</A></LI><LI><A href="protfolio-2.html">Protfolio 2</A></LI><LI><A href="protfolio-3.html">Protfolio 3</A></LI><LI><A href="case_study.html">Case Study</A></LI><LI class="smoth-scroll"><A href="#blog">Blog</A></LI><LI><A href="blog_grid.html">Blog Grid</A></LI><LI><A href="blog-left_sidebar.html">Blog Left Sidebar</A></LI><LI><A href="blog-right_sidebar.html">Blog Right Sidebar</A></LI><LI><A href="single_blog-full.html">Single Blog Full</A></LI><LI><A href="single_blog-left_sidebar.html">
                                                Single Blog Left Sidebar
                                            </A></LI><LI><A href="single_blog-right_sidebar.html">
                                                Single Blog Right Sidebar
                                            </A></LI><LI class="smoth-scroll"><A href="#contact">Contact US</A></LI></NAV></DIV></DIV></DIV><SECTION class="blog_banner_proof"><DIV class="container"><DIV class="row"><DIV class="col-sm-12"><DIV class="banner_title"><H1 id="back_to_the_top">Proof of Concept</H1></DIV></DIV></DIV></DIV></SECTION><SECTION class="blog_area section-padding"><DIV class="container"><DIV class="row"><DIV class="blog_grid_area"><DIV class="col-xs-12 col-sm-10"><DIV class="bloggrid_left"><DIV class="blog_top"><H4 class="blog_topHd"> Toluene evaporation test</H4></DIV><DIV class="blog_content"><H3><FONT color="A2233C">Protocol </FONT></H3><P> The toluene is known to be a volatile compound. This is why we decided to determine its rate of evaporation before proceeding to any subsequent manipulation with it. This will determine the care taken when handling it as well as the time the stock solution will be kept. 
1.2mL of pure toluene (244511, Sigma-Aldrich) was added in a 1.5mL Eppendorf tube. The tube was left open under a chemical hood and its weight was determine every 30 minutes for 6 hours. </P><H3><FONT color="A2233C">Results</FONT></H3><TABLE><TBODY><TR><TD><P>Time</P></TD><TD><P> t = 0 min</P></TD><TD><P> t = 30 min</P></TD><TD><P> t = 60 min</P></TD><TD><P> t = 90 min</P></TD><TD><P> t = 120 min</P></TD><TD><P> t = 150 min</P></TD><TD><P> t = 180 min</P></TD><TD><P> t = 210 min</P></TD><TD><P> t = 240 min</P></TD><TD><P> t = 270 min</P></TD><TD><P> t = 300 min</P></TD><TD><P> t = 330 min</P></TD><TD><P> t = 360 min</P></TD></TR><TR><TD><P> Weight</P></TD><TD><P> 1051 mg </P></TD><TD><P> 979.6 mg </P></TD><TD><P> 937.7 mg </P></TD><TD><P> 889.4 mg </P></TD><TD><P> 836.5 mg </P></TD><TD><P> 805.7 mg </P></TD><TD><P> 778.2 mg </P></TD><TD><P> 766.1 mg </P></TD><TD><P> 732.4 mg </P></TD><TD><P> 717.2 mg </P></TD><TD><P> 683.7 mg </P></TD><TD><P> 656 mg </P></TD><TD><P> 632.5 mg </P></TD></TR></TBODY></TABLE><P>Toluene is definitely a volatile compound, left open under a chemical hood 419 mg of pure toluene have evaporate from the tube within 6 hours. Those results show that toluene evaporate with a rate of 72 mg per hours. However, we supposed that this rate could have been increased by the chemical hood suction.

Therefore care have to be taken when manipulating it. Each tube containing toluene have to be open under a chemical hood and any toluene tubes must be left open. Care have to be taken when adding toluene to bacteria culture medium. 
The stock solution of toluene will be prepare again for each replicates.
With these results, we can then test the lethality of <I>E.Coli</I> DH5α when exposed to several toluene concentrations. 
</P><DIV class="blog_top"><H4 class="blog_topHd"> Survival test to toluene </H4></DIV><DIV class="blog_content"><H3><FONT color="A2233C">Protocol and tested concentrations </FONT></H3><P> The protocol 12 (available <A href="https://2016.igem.org/Team:Ionis_Paris/Protocol_12"><FONT color="DeepPink"> here </FONT></A>) was used to asses cell survival in presence of toluene. We tested a large range of concentrations (from 5ng/L to 5mg/L) to evaluate the overall impact of toluene on the <I> E.coli </I> DH5α cell growth. 5ng/L is a concentration closed to environmental toluene concentration. 
An overnight culture of bacteria transformed with our biosensor was realized. 300µL of this overnight culture was inoculated into 150mL of LB containing chloramphenicol at 25ng/mL. This culture was homogenized and separated into several falcon tubes. Toluene was then added to the culture medium to reach desired concentrations and the samples OD600 was checked every hours during 8 hours.<I><B> Three replicates of each concentration were done to ensure statistical results.</B></I></P><H3><FONT color="A2233C"> Results </FONT></H3><P>
The graph 1 shows the evolution of sample optic density (OD600) according to the time and the toluene concentration added in cell culture medium. The curve shows normal growth profile with a lag phase of three hours and then an exponential phase for all samples whether the toluene concentration added in the cell culture medium. 
Toluene addition at a concentration of 5ng/L and 500ng/L did not significantly affect cell development. The growth curve shows the same profile than cells grew without toluene. At a concentration of 5mg/L of toluene in the medium, an effect on cells development can be noticed. Four hours after toluene addition, the OD600 of cell grown in 5mg/L of toluene increased more slowly than the OD600 of cell grown without toluene.
</P><CENTER><FIGCAPTION><P><I> Figure 1: Evolution of <I>E.coli DH5α</I> growth depending on the toluene concentration added in the cell culture medium </I></P></FIGCAPTION></CENTER><P>In compliance with those results, bioluminescence test were realized with a toluene concentration in the culture medium comprised between 0 and 10 mg/L.</P><DIV class="blog_top"><H4 class="blog_topHd"> Biosensor construction </H4></DIV><DIV class="blog_content"><P> Three parts were synthesized by IDT in order to construct the whole biosensor : <A href="http://parts.igem.org/Part:BBa_K2023004"><FONT color="DeepPink">BBa_K2023004</FONT></A> composed of Pr-RBS, <A href="http://parts.igem.org/Part:BBa_K2023005"><FONT color="DeepPink">BBa_K2023005</FONT></A> coding for the XylR protein and <A href="http://parts.igem.org/Part:BBa_K2023006"><FONT color="DeepPink">BBa_K2023006</FONT></A> made of Terminator-Pu-RBS-Gaussia Luciferase-Terminator. Those three BioBricks are described in the <A href="https://2016.igem.org/Team:Ionis_Paris/Parts"><FONT color="DeepPink">Parts and characterization </FONT></A> section. 

The biosensor was then assembled using digestion and ligation as described in the <A href="https://2016.igem.org/Team:Ionis_Paris/Notebook"><FONT color="DeepPink"> notebook Biosensor Construction </FONT></A>. The 2nd of August, <I> E.coli </I> bacteria were transformed into the whole biosensor genetic construction. Colony PCR were realized to select the recombinant bacteria. The results (Figure 2) indicated that the colony 6 and 7 have integrated the whole biosensor as strips can be seen at 3400bp. Those colonies were miniprep and the collected DNA was sent to sequencing.

</P><P><I> Figure 2: Colony PCR results of bacteria transformed with our whole biosensor. Strips are expected at 3400 bp </I></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></SECTION><P>Sequencing results were not what we expected: a mutation on the start codon of the XylR protein was present in the plasmid of all our clones. Therefore, we performed a <A href="https://2016.igem.org/Team:Ionis_Paris/05_09_16"><FONT color="DeepPink"> &quot;site  directed mutagenesis </FONT></A> to overcome this problem.</P><DIV class="blog_top"><H4 class="blog_topHd"> Biosensor characterization </H4></DIV><DIV class="blog_content"><H3><FONT color="A2233C"><B> XylR production by modified <I> E.coli</I> bacteria </B></FONT></H3><P>
We aimed to asses the XylR production by <I> E.coli </I> cells modified with the BioBrick BBa_K2023013 composed of Pr-RBS-XylR-RBS-mRFP-Terminator. This BioBrick was designed to monitor XylR synthesis by our cells. 
To do so, we thaw a tubes of transformed cells on ice for 10 minutes, we added different volumes of SOC medium to obtain several dilutions (1/2, 1/4 and 1/8), we plated different concentrations of bacteria cells on petri dish with LB Agar + Chloramphenicol at 25µg/L final and incubate overnight. 

We obtained the expected results : red colonies were shown on the petri dish (Figure 3) due to the synthesis of the red protein mRFP meaning that the Pr promoter trigger XylR and mRFP synthesis.
</P><CENTER><FIGCAPTION><P><I> Figure 3: Monitoring of XylR production though control of the activity of the Pr promoter. Modified bacteria containing XylR coding device with mRFP grown on a petri dish. The red color of the bacteria indicate that mRFP protein is produce and therefore that XylR is produce too. </I></P></FIGCAPTION></CENTER><P> This construction will not allow XylR quantification but those results show that XylR protein was constitutively produce by our genetically modified bacteria containing XYlR coding device with mRFP. 
We wanted to realize a XylR-mRFP fusion but this part was too big for IDT synthesis. 
We created the BioBrick BBa_K2023010 (XylR-His) to give to future iGEM the opportunity to purify and quantify XylR protein. 
</P><H3><FONT color="A2233C"><B> Bioluminescence assays </B></FONT></H3><H4><FONT color="A2233C"> Protocol and tested concentrations </FONT></H4><P>
We used the protocol 13 (available <A href="https://2016.igem.org/Team:Ionis_Paris/Protocol_13"><FONT color="DeepPink"> here </FONT></A>)and decided to test several concentration of toluene (0ng/L, 10ng/L, 100ng/L, 10µg/L and 10mg/L).Bioluminescence assays were realized several times after toluene addition (1h, 3h, 4h30 and 5h30). 
Bacteria transformed with our biosensor used in each assays come from the same culture (OD = 0.1). This culture was well mixed and divided in 50 mL falcon. Toluene was then added in each Falcon. 

Bioluminescence assays were realized quickly after substrate addition and the bioluminescence intensity was measured using <A href="https://www.berthold.com/en/bio/monochromator_multilabel_microplate_reader_Mithras2_LB943"><FONT color="DeepPink"> Mithras² LB 943 Monochromator Multimode Reader.</FONT></A> This machine was kindly lend by Berthold company for 3 days in order to let us realized the bioluminescence measurement. In addition to bioluminesence intensity, sample's OD was also measured. <I>All tests were realized in triplicates.</I>

Several negative controls were realized :  </P><LI><P>Measurement of LB bioluminescence intensity to determine the background noise</P></LI><LI><P>Measurement of LB + toluene bioluminescence intensity to determine toluene effect on the bioluminescent intensity</P></LI><LI><P>Measurement of the bioluminescence intensity of bacteria transformed with a genetic construction that do not contain the gaussia luciferase gene (Pr-XylR).</P></LI><P>
A positive control using Gluc codig device with Pr (BBa_K2023007) could have been made during the three days of measurement to test substrate efficiency and luminometer parameters. However, this BioBrick was created afterwards. We hoped that this BioBrick will serve as a future control for all bioluminescence assays in iGEM project to ensure the substrate efficiency.
</P><H4><FONT color="A2233C"> Preliminary considerations </FONT></H4><B><FONT color="D32C50"> Toluene does not have auto-bioluminescence </FONT></B><P>
We determine the effect of adding toluene to cell culture medium on bioluminescence intensity. As shown on the graph below the bioluminescence intensity of the LB + toluene at 10 mg/L (47.71 RLU) do not significantly differ from the bioluminescence intensity of the LB (48.93 RLU). The little bioluminescence found was due to the substrate added in our sample. 
Therefore, toluene addition in our samples will not impact their bioluminescence intensity.</P><CENTER><TABLE><TBODY><TR><TD><P>LB without toluene</P></TD><TD><P>LB with toluene at 10mg/L</P></TD></TR><TR><TD><P>Mean</P></TD><TD><P>48.93 RLU</P></TD><TD><P>47.71 RLU</P></TD></TR><TR><TD><P>Standard deviation (SD)</P></TD><TD><P>6.96</P></TD><TD><P>9.55</P></TD></TR><TR><TD><P>Std. error of mean(SEM)</P></TD><TD><P>1.87</P></TD><TD><P>2.55</P></TD></TR></TBODY></TABLE></CENTER><CENTER><FIGCAPTION><P><I> Figure 4: Bioluminescence intensity expressed in RLU of LB and LB with toluene at 10mg/L. Mann whitney statistical test was performed to asses the effect of toluene on LB auto-bioluminescence (ns: non significant). </I></P></FIGCAPTION></CENTER><B><FONT color="D32C50"> Bacteria E.coli have little auto-bioluminescence </FONT></B><P> Bacteria transformed with a genetic construction that does not contain Gaussia luciferase gene were used in this study. The auto-bioluminescence of E.coli cells was assessed by comparate <I>E.coli</I> bacteria bioluminescence intensity to LB bioluminescence intensity. As shown on the figure 3, the bioluminescence intensity of <I>E.coli </I> cells (53.14 RLU) was a little superior to LB bioluminescence intensity (48.93 RLU) but not significantly superior. <I>E.coli</I> cells have little auto-bioluminescence. However, this bioluminescence in greatly inferior to the one produce by our biosensor in presence of pollutant (results presented after) </P><P>LB without toluene</P><P>BB12 without toluene</P><P>Mean</P><P>48.93 RLU</P><P>53.14 RLU</P><P>Standard deviation (SD)</P><P>6.96</P><P>11.34</P><P>Std. error of mean(SEM)</P><P>1.86</P><P>3.03</P><CENTER><FIGCAPTION><P><I> Figure 5: Bioluminescence intensity expressed in RLU of LB and LB + <I>E.coli </I> cells. Mann whitney statistical test was performed (ns: non significant). </I></P></FIGCAPTION></CENTER><B><FONT color="D32C50"> Little leakage in Gaussia luciferase expression</FONT></B><P> We investigate the inductility of the Pu promoter to ensure that the gaussia luciferase is not constitutively produce in our biosensor. To do so we compared the bioluminescence intensity of bacteria containing our biosensor to the bioluminescence intensity of bacteria transformed with a genetic construction that do not contain the Gaussia luciferase gene (BB12 : Pr-RBS-XylR). 

The obtained results (figure 6) indicate that the gaussia luciferase gene is not produce in a constitutive manner in the cells. The bioluminescence intensity of our biosensor without toluene injection (158.86 RLU) is a little higher than the bioluminescence intensity of cells transformed with BB12 (53.14 RLU). However this bioluminescence intensity is significantly lower than the bioluminescence intensity of the cells transformed with our biosensor in presence of toluene (results presented in the next section)
</P><P>BB12 without toluene</P><P>BB123 without toluene</P><P>Mean</P><P>53.14 RLU</P><P>158.86 RLU</P><P>Standard deviation (SD)</P><P>11.34</P><P>20.08</P><P>Std. error of mean(SEM)</P><P>3.03</P><P>3.10</P><CENTER><FIGCAPTION><P><I> Figure 6: Bioluminescence intensity expressed in RLU of <I>E.coli </I> cells transformed with a construction genetic that contain (BB123) or do not contain (BB12) the gaussia gene. A little leakage of the gaussia gene can be seen as the bioluminescence intensity of the sample that contain bacteria transformed with our biosensor is significantly superior to the one that contain bacteria transformed with Pr-RBS-XylR. Mann whitney statistical test was performed (*** : extremely significant). </I></P></FIGCAPTION></CENTER><H4><FONT color="A2233C"> Data processing </FONT></H4><P> Due to the little LB background noise and leakage in gaussia luciferase synthesis, obtain bioluminescence intensity results were treated as followed. The background noise (LB+toluene) was subtracted to each sample’s bioluminescence intensity. Those latest were then expressed depending on the negative control (bacteria transformed with our biosensor in a medium without toluene, C0).</P><H4><FONT color="A2233C"> Results </FONT></H4><B><FONT color="D32C50"> Investigation of the best measurement time</FONT></B><P>
It was first necessary to determine when, after toluene injection, bioluminescence results were the most relevant. To do so, we determine the bioluminescence intensity of our sample (bacteria transformed with our biosensor within LB medium containing different toluene concentration) several times after toluene addition. 
The bioluminescence intensity of the negative control was also determine at each time after toluene addition and bioluminescent results for each assay were normalized on the corresponding negative control bioluminescence intensity. This normalization was necessary as we are dealing with several incubation time. This enable us to handle the bacteria OD increase within our samples and ensure that an increase in sample’s bioluminescence intensity is not correlated with an increase in cells concentration.

The graphs presented below represent the bioluminescence intensity of the sample depending on the time after a toluene addition for a fix concentration of toluene injected. 
</P><CENTER><FIGCAPTION><P><I> Figure 7: Evolution of bioluminescence intensity at different times after toluene addition (10ng/L). The bioluminescence intensity of each sample is normalized on the bioluminescence intensity of the corresponding negative control. Mann whitney statistical test was performed (ns : non significant, * : significant, ** : very significant, *** : extremely significant) </I></P></FIGCAPTION></CENTER><CENTER><FIGCAPTION><P><I> Figure 8: Evolution of bioluminescence intensity at different times after toluene addition (100ng/L). The bioluminescence intensity of each sample is normalized on the bioluminescence intensity of the corresponding negative control. Mann whitney statistical test was performed (ns : non significant, * : significant, ** : very significant, *** : extremely significant) </I></P></FIGCAPTION></CENTER><CENTER><FIGCAPTION><P><I> Figure 9: Evolution of bioluminescence intensity at different times after toluene addition (10µg/L). The bioluminescence intensity of each sample is normalized on the bioluminescence intensity of the corresponding negative control. Mann whitney statistical test was performed (ns : non significant, * : significant, ** : very significant, *** : extremely significant) </I></P></FIGCAPTION></CENTER><CENTER><FIGCAPTION><P><I> Figure 10: Evolution of bioluminescence intensity at different times after toluene addition (10mg/L). The bioluminescence intensity of each sample is normalized on the bioluminescence intensity of the corresponding negative control. Mann whitney statistical test was performed (ns : non significant, * : significant, ** : very significant, *** : extremely significant) </I></P></FIGCAPTION></CENTER><P>
The bioluminescence intensity of all our samples was compared to the bioluminescence intensity of the negative control for each time after toluene injection. Mann Whitney statistics test were realized to determine whether the bioluminescence intensity of the sample was significantly higher than the bioluminescence intensity of the negative control.
On the graphics we can notice that the bioluminescence intensity increase as the time after toluene injection increase. However, for the highest concentration (10 mg/L), a decrease in the bioluminescence intensity can be notice 5h30 after toluene injection. It is due to the accumulation of gaussia luciferase and the impact of toluene on the cell metabolism and especially protein synthesis (for more information click <A href="https://2016.igem.org/Team:Ionis_Paris/E.coli"><FONT color="DeepPink"> here </FONT></A>). 
Therefore, it is essential to realize our bioluminescence test at a precise time after sampling. As we want to detect environmental toluene concentration (less than 20ng/L), it is to be suitable to wait 5h30 before proceeding to the bioluminescence test. We did not incubate our bacteria with toluene for a longer time because of the effect of toluene on <I>E.coli </I> cells. We though that this effect will decrease our biosensor sensitivity and precision.
</P><B><FONT color="D32C50"> Investigation of detectable concentration 5h30 after toluene addition to the cells medium</FONT></B><P>
As we determined that it was preferable to wait 5h30 before realizing the bioluminescence assay, we decided to focus on the results obtain for this time after toluene addition in the cells medium. We studied the evolution of the bioluminescence intensity according to the toluene concentration added in the cells medium 5h30 before. Those tests were realize in triplicates. 
 
First of all, we wanted to ensure that the OD of the different samples is the same 5h30 after bacteria inoculation in presence of toluene. As shown on the Figure 11, the sample’s ODs are quite similar going from 1.464 to 1.61. 
</P><CENTER><FIGCAPTION><P><I> Figure 11: Sample's ODs depending on the toluene concentration added 5h30 before. </I></P></FIGCAPTION></CENTER><P>
The bioluminescence intensity of each samples was then determined and shown depending on the toluene concentration. As we are dealing with extremely different toluene concentration, it was preferable to give our results in function of the logarithm of the toluene concentration. The bioluminescence intensity data in RLU for each replicates (R1, R2, R3) are given in the table below and represented on the Figure 12. 
</P><CENTER><TABLE><TBODY><TR><TD><P>Toluene concentration</P></TD><TD><P>Logarithm of toluene concentration</P></TD><TD><P>Mean - R1 (RLU)</P></TD><TD><P>SD - R1</P></TD><TD><P>Mean - R2 (RLU)</P></TD><TD><P>SD - R2</P></TD><TD><P>Mean - R3 (RLU)</P></TD><TD><P>SD - R3</P></TD></TR><TR><TD><P>0 ng/L</P></TD><TD><P>0</P></TD><TD><P>180.2</P></TD><TD><P>16.0</P></TD><TD><P>146.5</P></TD><TD><P>23.4</P></TD><TD><P>167.9</P></TD><TD><P>11.8</P></TD></TR><TR><TD><P>10 ng/L</P></TD><TD><P>1</P></TD><TD><P>1007.3</P></TD><TD><P>210.2</P></TD><TD><P>337.7</P></TD><TD><P>39.3</P></TD><TD><P>1041.9</P></TD><TD><P>139.4</P></TD></TR><TR><TD><P>100 ng/L</P></TD><TD><P>2</P></TD><TD><P>754.4</P></TD><TD><P>150.4</P></TD><TD><P>762.4</P></TD><TD><P>77.9</P></TD><TD><P>1823.1</P></TD><TD><P>243.1</P></TD></TR><TR><TD><P>10 µg/L</P></TD><TD><P>4</P></TD><TD><P>1110.4</P></TD><TD><P>150.0</P></TD><TD><P>1373.8</P></TD><TD><P>162.4</P></TD><TD><P>2073.7</P></TD><TD><P>202.9</P></TD></TR><TR><TD><P>10 mg/L</P></TD><TD><P>7</P></TD><TD><P>882.8</P></TD><TD><P>131.6</P></TD><TD><P>2485.4</P></TD><TD><P>259.5</P></TD><TD><P>1247.2</P></TD><TD><P>151.4</P></TD></TR></TBODY></TABLE></CENTER><CENTER><FIGCAPTION><P><I> Figure 12: Evolution of bioluminescence intensity for each replicate depending on the logarithm of the toluene concentration injected 5h30 before. </I></P></FIGCAPTION></CENTER><P>
As shown on the graph, the obtained data differ from one replicates to another due to difference in the metabolism which is normal when working with living organism. However the curve profile stay is the same : the bioluminescence intensity of the sample increases until a toluene concentration of 10 µg/,  then it reaches a plateau for two replicates and continues to increase for the other replicate.
Those results were pooled in order to draw a standard curve and being able to predict toluene concentration in a sample according to the bioluminescence intensity.
</P><B><FONT color="D32C50"> Quantification of environmentally relevant toluene concentration </FONT></B><P>The Figure 13 shows the evolution of the bioluminescence intensity depending on the logarithm of toluene concentration. For a toluene concentration inferior to 10 µg/L (10,000 ng/L : log(4)) the curve increases steadily, then the curve reach a plateau. This can be due to gaussia accumulation in the cells. 
We were able to detect environmentally relevant toluene concentration and as the standard curve is linear for a toluene concentration inferior to 10µg/L, we are able to quantify the toluene concentration of a given sample.
</P><CENTER><TABLE><TBODY><TR><TD><P>0 ng/L</P></TD><TD><P>10 ng/L</P></TD><TD><P>100 ng/L</P></TD><TD><P>10 µg/L</P></TD><TD><P>10 mg/L</P></TD></TR><TR><TD><P>Mean (RLU)</P></TD><TD><P>164,84</P></TD><TD><P>795,63</P></TD><TD><P>1139,88</P></TD><TD><P>1519,28</P></TD><TD><P>1538,45</P></TD></TR><TR><TD><P>Standard deviation</P></TD><TD><P>17,62</P></TD><TD><P>129,73</P></TD><TD><P>146,68</P></TD><TD><P>171,75</P></TD><TD><P>180,84</P></TD></TR><TR><TD><P>Induction percentage</P></TD><TD><P>383%</P></TD><TD><P>592%</P></TD><TD><P>822%</P></TD><TD><P>833%</P></TD></TR></TBODY></TABLE></CENTER><CENTER><FIGCAPTION><P><I> Figure 13: Evolution of bioluminescence intensity for pooled of the replicate depending on the logarithm of the toluene concentration injected 5h30 before. Correlation between toluene concentration in the cell culture medium and the bioluminescence intensity.</I></P></FIGCAPTION></CENTER><H3><FONT color="A2233C"><B> Comparison with existing methods </B></FONT></H3><P> 5h30 after toluene injection, our biosensor is able to detect environmentally relevant toluene concentration (10ng/L). This way of pollutant detection is environmentally friendly as the bacteria only required the appropriate molecules in its culture medium to grow and quantify pollutant. In addition, the use of a biosensor, even if it requires an expensive machine (a luminometer) for results analysis, is cheapper compared to others physical chemical existings methods.

Find out more informations about existing methods of pollutant detection <A href="https://2016.igem.org/Team:Ionis_Paris/Design"><FONT color="DeepPink"> here </FONT></A>


However, our biosensor is only a prototype and we present below the different improvement that should be made. </P><H3><FONT color="A2233C"><B> Perspectives and improvement </B></FONT></H3><P> Based on the results present above, we though about improvement and future experiments that should be realize. First of all, the standard curve drawn is based on few toluene concentration. We can improve this standard curve by focusing on low toluene concentration and realize bioluminescence assay for more low toluene concentration. 
Then we have to determine the lowest concentration that our biosensor can detect while ensuring precise and reproductible results.Moreover, <A href="https://2016.igem.org/Team:Ionis_Paris/Measurement"><FONT color="DeepPink"> CelloCad software </FONT></A> can be use to optimize our genetic circuit. 

We have pointed out the variation between our different replicates for bioluminescence assay. Those variations are due to metabolism variations of living organisms. Several measurements have to be realize for each toluene concentration and the
reproductibility of the results have to be assess on more experiments.
Also, we did not have the time to optimize our bioluminescence assay protocol and to test the different parameters of the luminometer. 

We did not performed tests with others pollutants such as the xylene. Therefore, the sensibility of our biosensor to those pollutants should be assesses in future experiments. 
For a convenient reason and as we could not evaporate toluene within our laboratory, we only performed bioluminescence assay using liquid toluene. Future work could consist in the creation of an isolated and hermetic room in which gazeous toluene could be add ad our drone flight to perform air sampling.

You will find out more informations about our investigation on the relationship between our bacteria and drone <A href="https://2016.igem.org/Team:Ionis_Paris/Demonstrate"><FONT color="DeepPink"> here </FONT></A></P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="footer_social"><DIV class="container-fluid"><DIV class="row"><UL class="ft_top clearfix"><LI class="waves-effect waves-light"><A href="https://www.facebook.com/ionisigem/?fref=ts" target="_blank">Facebook
                        </A></LI><LI class="waves-effect waves-light"><A href="https://twitter.com/igem_ionis" target="_blank">Twitter
                        </A></LI><LI class="waves-effect waves-light"><A href="https://www.youtube.com/channel/UC0RyQsB5YpweSRBYQlAdZDA" target="_blank">youtube
                        </A></LI><LI class="waves-effect waves-light"><A href="https://www.linkedin.com/company/igem-ionis" target="_blank">LinkedIn
                        </A></LI></UL></DIV></DIV></DIV><FOOTER class="footer_area"><DIV class="footer_middle"><DIV class="container"><DIV class="row"><DIV class="row"><DIV class="col-lg-5 col-md-5 col-sm-12"><DIV class="middle_content"><H4>iGEM IONIS</H4><P> We're a group of six different schools from the IONIS Education Group. For this
                                    competition we wanted to take advantages of the multiple schools and activity field
                                    given by the IONIS education group to create a solid project.</P><A href="https://2016.igem.org/Team:Ionis_Paris/Team">Read More</A></DIV></DIV><DIV class="col-lg-3 col-lg-offset-1 col-md-4 col-sm-7"><DIV class="footer_Widgets"><H4>Stay in touch</H4><DIV class="ft_contact"><UL><LI><SPAN>Location: 66 Rue Guy Môquet, 94800 Villejuif, France</SPAN></LI><LI><A href="mailto:igem.ionis@gmail.com">email: igem.ionis@gmail.com</A></LI></UL></DIV></DIV></DIV><DIV class="col-lg-3 col-md-3 col-sm-5"><DIV class="footer_Widgets"><H4>Download the app</H4></DIV></DIV></DIV></DIV></DIV><DIV class="footer_bottom"><DIV class="container"><DIV class="ft_area"><DIV class="row"><DIV class="col-md-7 fix_p_l"><NAV><UL class="ft_bottom"><LI><A href="https://2016.igem.org/Team:Ionis_Paris">Home</A></LI><LI><A href="https://2016.igem.org/Team:Ionis_Paris/Working_at_La_Paillasse">in the Lab</A></LI><LI><A href="https://2016.igem.org/Team:Ionis_Paris/Measurement"> Side Projects</A></LI><LI><A href="https://2016.igem.org/Team:Ionis_Paris/Parts">Results</A></LI><LI><A href="https://2016.igem.org/Team:Ionis_Paris/HPintro">Human
                                            Practice</A></LI><LI><A href="https://2016.igem.org/Team:Ionis_Paris/Team">Team</A></LI></UL></NAV></DIV><DIV class="col-md-5 fix_p"><DIV class="ft_paragraph"><P>©IONIS_IGEM_2016.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></FOOTER></BODY></HTML>