Team:Ionis Paris/05 09 16
Project
Project description
Biology
Engineering
Applied Design
Entrepreneurship
In the lab
Working at La Paillasse
Biobrick design
Cloning Strategy
Protocols
Notebook
Side-projects
CelloCad
Collaborations
Mobile application
Education
Results
Parts and Characterization
Proof of concept
Demonstration
Human Practices
Overview
Project Genesis
Air Pollution
Opinion Survey
Ethics and Biosafety
Our Actions
Safety
Lab safety
Project safety
Team
The team
Events
Attributions
Acknowledgement (Sponsors and advisors)
Prizes
Project
Home Page 1
Home Page 2
Home Page 3
Home Page 4
Home Page 5
Home Page 6
Home Page 7
Home Page 8
Home Page 9
Services
About
Portfolio
Protfolio 1
Protfolio 2
Protfolio 3
Case Study
Blog
Blog Grid
Blog Left Sidebar
Blog Right Sidebar
Single Blog Full
Single Blog Left Sidebar
Single Blog Right Sidebar
Contact US
September 5th 2016
Site Directed Mutagenesis: on BB12 and BB13
Objectives
Our goal is to remove a mutation of one base which was revealed by sequencing of BB12 and BB123 (the biosensor). This mutation is situated in the start codon of the XylR gene. We need to change a G for a T: AGG —> ATG. The principle of site-directed-mutagenesis (SDM) is to remove a mutation with primers that contain the good base in the middle of its DNA sequence. Those primers are called “back-to-back primers”.
At the end of the SDM we will get the correct BB12 and BB123 (biosensor) biobricks with the ATG in the XylR gene. It will be named BB12mut and BB123mut.
Preview of the BB123 sequencing that revealed the start codon mutation in the XylR gene:
Materials
BB12-4 (from miniprep 29/07)
BB123-6 (from miniprep 04/08)
Primers: SDM-F (forward) and SDM-R (reverse) —> Ta = 57°C
Protocol
PCR
1.  Mix for 2 samples (Total volume Mix : 48 µL) in an Eppendorf tube :
18 µL H2O
2.5 µL Primer SDM-F ( 0.5 µM final)
2.5 µL Primer SDM-R (0.5 µM final)
25 µL Q5 Hot start high-fidelity 2X Master Mix (1 X final, NEB #E0552S)
2.  Add in PCR tube in the respected order:
24 µL Mix
1 µL of DNA fragment (BB12 & BB123)
Gently mix the reaction and spin down microcentrifuge
3.  Set the following parameters for the PCR reaction :
BB12 (4272 bp):
Lid temperature: 98°C
Initial denaturation: 98°C, 30 s
30 cycles of: 98°C, 10 s
57°C, 30 s
72°C, 2 min 8 s
Final extension: 72°C, 2 min
Hold: 4°C
BB123 (5377 pb):
Lid temperature: 98°C
Initial denaturation: 98°C, 30 s
30 cycles of: 98°C, 10 s
57°C, 30 s
72°C, 2 min 41 s
Final extension: 72°C, 2 min
Hold: 4°C
Kinase, Ligase & DpnI (KLD) Treatment:
1.  In PCR tube, adding in the respected order (bigger volume first and enzyme last):
5 µL 2X KLD Reaction Buffer (1X final, NEB #E0552S)
3 µL Nuclease-free H2O
1 µL PCR product (BB12 / BB123)
1 µL 10X KLD Enzyme Mix (1X final, NEB #E0552S)
2.  Incubation 5 min at room temperature
Transformation: competent DH5⍺ cells with SDM products BB12mut and BB123mut
Objectives
The objective is to transforme competent DH5⍺ cells with the SDM products BB12mut and BB123mut.
Materials
2 aliquots of DH5⍺ Competent cells (from the 23/07/16)
Plasmid DNA : KLD mix with SDM products BB12mut and BB123mut
Petri dish LB+Cm: Cm concentration = 25 µg/mL
Protocol
Experimental conditions realized :
We need 6 LB+Cm plates + 4 LB plates
Transformations protocol:
Thaw tubes of DH5⍺ competent cells on ice for 10 min. Mix gently and carefully pipette 50 µL of cells into the 4 transformation tubes on ice.
Add 5 µL of the KLD mix to the cell mixture./p>
Carefully flick the tubes 4-5 times to mix cells and DNA. Do not vortex./p>
Place on ice for 30 min. Do not mix./p>
Heat shock at exactly 42°C for 45 s. Do not mix./p>
Place on ice for 5 min. Do not mix./p>
Pipette 250 µL of room temperature SOC into the mixture./p>
Place at 37°C for 1h at 250 rpm./p>
Warm selection plates to 25°C./p>
Mix the cells thoroughly by flicking the tubes and inverting./p>
Spread the corresponding volume onto each plate./p>
Incubate all the plates O/N at 37°C./p>
Results (obtained the 06/09)
Expected results:
Some colonies on the petri dishes LB+Cm plated with 50 µL of bacteria transformed with the different ligation products and more on the petri dishes LB+Cm plated with 200 µL of bacteria.
A bacterial lawn on the LB petri dishes without antibiotic.
No colonies on the LB+Cm petri dish plated with bacteria transformed with no plasmid (- control).
Obtained results:
We obtain the expected results.
Interpretation
The transformation worked. Colonies contain a plasmid with the Chloramphenicol resistance gene, present in pSB1C3. However, it is possible that the PCR of the SDM do not work properly, that it only amplified a section containing the resistance chloramphenicol coding device. A PCR colonie is necessary to check the size of the plasmid present in colonies, and therefore to ensure that bacteria incorpore the correct plasmid BB12mut / BB123mut.
June 18,2015
2
Creative and impressive building
June 18,2015
2
How you can be a super creative
June 18,2015
2
World best photographer and photography
Facebook
Twitter
youtube
LinkedIn
iGEM IONIS
We're a group of six different schools from the IONIS Education Group. For this                                     competition we wanted to take advantages of the multiple schools and activity field                                     given by the IONIS education group to create a solid project.
Read More
Stay in touch
Location: 66 Rue Guy Môquet, 94800 Villejuif, France
email: igem.ionis@gmail.com
Download the app
Home
in the Lab
Side Projects
Results
Human                                             Practice
Team
©IONIS_IGEM_2016.
