Team:Ionis Paris/Notebook/17 10 2016
Project
Project description
Biology
Engineering
Applied Design
Entrepreneurship
In the lab
Working at La Paillasse
Biobrick design
Cloning Strategy
Protocols
Notebook
Side-projects
CelloCad
Collaborations
Mobile application
Education
Results
Parts and Characterization
Proof of concept
Demonstration
Human Practices
Overview
Project Genesis
Air Pollution
Opinion Survey
Ethics and Biosafety
Our Actions
Safety
Lab safety
Project safety
Team
The team
Events
Attributions
Acknowledgement (Sponsors and advisors)
Prizes
Project
Home Page 1
Home Page 2
Home Page 3
Home Page 4
Home Page 5
Home Page 6
Home Page 7
Home Page 8
Home Page 9
Services
About
Portfolio
Protfolio 1
Protfolio 2
Protfolio 3
Case Study
Blog
Blog Grid
Blog Left Sidebar
Blog Right Sidebar
Single Blog Full
Single Blog Left Sidebar
Single Blog Right Sidebar
Contact US
October 17th 2016
PCR : X6 and P2mut from BB2
Objectives
The overall purpose is to amplify P2 from BB2 by mutate the deficient codon start with the SDM-F primer to obtain BB2mut and by adding a 6X-His tag with the primer AHis-tag to obtain BBX2 (XylR-6His).
Materials
Plasmid template: BB2-39 (from Miniprep 29/07/2016).
Primers: SDM-F (forward), AHis-tag (reverse) and A13 (reverse).
Protocol
PCR
1.  2 Mix for 2 samples (Total volume of Mix : 98 µL), in an Eppendorf tube :
96 µL H2O
1 µL Primer SDM-F  (0.5 µM final)
1 µL Primer A13 in Mix 1 / AHis-tag in Mix 2 (0.5 µM final)
50 µL Q5 High-Fidelity 2X Master Mix (NEB #M0492S)
2.  Add in 4 PCR tubes, in the respected order:
49 µL Mix 1 (P2mut and Control1) / 49 µL Mix 2 (X6 and Control 2)
1 µL DNA (P2mut and X6) / 1 µL H20 (Control 1 and 2)
—> Gently mix the reaction and short spin centrifugation
3.  Set the following parameters for the PCR reaction :
X2 (1794 bp) and P2mut (1747 bp)
Lid température: 98°C
Initial denaturation: 98°C, 30 s
30 cycles of: 98°C, 10 s
59 °C, 30 s
72°C, 52 s
Final extension: 72°C, 2 min
Hold: 4°C
Electrophoresis: for screening the PCR results
1% Agarose gel:
Put 1 g agarose + 100 mL TAE 1X in a bottle of 500 mL
Mix and heat it 2 min 30 s in the microwaves. Wait the cooling of the bottle until it is tepid.
Add 5 µL of Gel Red 10,000 X (0.5 X final)
Flow the gel and place the combs
Wait until it is solidified. Remove slowly the combs.
Drop-off:
Short Speed centrifugation of samples.
Addition of 1 µL of Purple loading dye 6 X in 5 µL of sample.
Drop-off 10 µL of Purple ladder and 6 µL of each samples.
Run at 90 V.
PCR purification :
QIAquick PCR purification kit (qiagen, 28106), according to the protocol given by the supplier (available here )
Add 5 volumes Buffer PB (225 µL) to 1 volume of the PCR reaction (45 µL) and mix. The color of the mixture is yellow.
Load the sample to the QIAquick column. Centrifuge for 1 min at 13,000 rpm and discard flow-through.
Add 750 µL Buffer PE. Centrifuge for 1 min at 13,000 rpm and discard flow-through.
Centrifuge once more for 1 min at 13,000 rpm.
Place each QIAquick column in a clean 1.5 mL microcentrifuge tube.
Add 30 µL Buffer EB to the center of the QIAquick membrane, let stand for 1 min, and centrifuge for 1 min at 13,000 rpm.
Calculate the quantity of DNA with the Nanodrop.
Store the purified DNA at -20°C.
Results
Expected results / Obtained results:
Interpretation
We obtain the desired strip for P2mut and X2, as shown on the gel above the strips are closed to 1,747 bp and 1,794 bp.
It seems that P2mut and X2 have been properly amplified.
Digestion: pSB1C3, P2mut, X2, C4 and G2
Objective
Double digestion of pSB1C3-RFP, P2mut, X2, C4 and G2 by EcoRI and PstI for the subsequent ligation of P2mut, X6, C4 and G2 in pSB1C3.
Materials
Stock concentrations:
pSB1C3-RFP 3 : 104.14 ng/µL (from miniprep 19/07)
P2mut: ~ 50 ng/µL (from PCR 17/10)
X2: ~ 150 ng/µL (from PCR 17/10)
C4: ~ 150 ng/µL (from PCR 08/10)
G2: ~ 150 ng/µL (from PCR 08/10)
Quantity of DNA required for the ligations:
P2mut: Digestion of 75 ng (ratio 1:1 = 21.10 ng needed, ratio 2:1 = 42.20 ng needed —> 64 ng needed)
X2: Digestion of 75 ng (ratio 1:1 = 21.67 ng needed, ratio 2:1 = 43.33 ng needed —> 65 ng needed)
C4: Digestion of 50 ng (ratio 3:1 = 14.57 ng needed, ratio 5:1 = 24.28 ng needed —> 40 ng needed)
G2: Digestion of 50 ng (ratio 2:1 = 17.08 ng needed, ratio 3:1 = 25.62 ng needed —> 43 ng needed)
pSB1C3-RFP: Digestion of 312 ng (25 ng needed per ratio —> 200 ng of digested pSB1C3 needed —> 304 ng of pSB1C3-RFP needed)
Protocol
Digestion:
In a 1.5 mL Eppendorf tube, adding in the respected order (bigger volume first and enzyme last) :
NB: The digestion were done in 20 µL.
Short Spin Centrifugation
ncubation 1h at 37°C
Store at 4°C before gel electrophoresis and purification
Electrophoresis for digested pSB1C3-RFP 1, 2 and 3:
1% Agarose gel:
Put 1 g of agarose low melting point + 100 mL of TAE 1X in a bottle of 500 mL.
Mix and heat it 2 min 30 s in the microwaves. Wait the cooling of the bottle until it is tepid.
Add 3 µL of Gel Red 10,000 X (0.3 X final).
Flow the gel and place the combs.
Wait until it is solidified. Remove slowly the combs.
Drop-off:
Short Speed centrifugation of samples.
Addition of 4 µL of Purple loading dye 6X in the 20 µL of sample.
Drop-off 10 µL of Purple ladder and 25 µL of sample.
Run at 90 V.
Gel purification for digested pSB1C3 1, 2 and 3:
QIAquick Gel purification kit (Qiagen, 28704), according to the protocol given by the supplier (available here )
Excise the DNA fragments from the agarose gel. Gel slice Weigh = 0.211 g (1) / 0.225 g (2) / 0.231 g (3)
Add 3 volumes Buffer QG (633 µL (1) / 675 µL (2) / 693 µL (3) ) to 1 volume of gel.
Incubate at 50°C for 10 min until the gel slice has completely dissolved. Vortex the tube every 2–3 min to help dissolve gel. The color of the mixture is yellow.
Add 1 gel volume isopropanol to the sample and mix.
Load 800 µL of each samples to the QIAquick column. Centrifuge for 1 min at 13,000 rpm and discard flow-through. Load the rest and spin again.
Add 500 µL Buffer QG. Centrifuge for 1 min at 13,000 rpm and discard flow-through.
Add 750 µL Buffer PE. Centrifuge for 1 min at 13,000 rpm and discard flow-through.
Centrifuge once more for 1 min at 13,000 rpm.
Place QIAquick column into a clean 1.5 mL microcentrifuge tube.
Add 30 µL Buffer EB to the center of the QIAquick membrane, let stand for 1 min, and centrifuge for 1 min at 13,000 rpm.
Store the purified DNA at 4°C before the ligation.
PCR purification for digested P2mut, X2, C4, G2:
QIAquick PCR purification kit (qiagen, 28106), according to the protocol given by the supplier (available here )
Add 5 volumes Buffer PB (100 µL) to 1 volume of the sample (20 µL) and mix. The color of the mixture is yellow.
Load the sample to the QIAquick column. Centrifuge for 1 min at 13,000 rpm and discard flow-through.
Add 750 µL Buffer PE. Centrifuge for 1 min at 13,000 rpm and discard flow-through.
Centrifuge once more for 1 min at 13,000 rpm.
Place each QIAquick column in a clean 1.5 mL microcentrifuge tube.
Add 30 µL Buffer EB to the center of the QIAquick membrane, let stand for 1 min, and centrifuge for 1 min at 13,000 rpm.
Store the purified DNA at 4°C before the ligation.
Results
Electrophoresis:
Expected results / Obtained results:
Interpretation
The digestion of pSB1C3-RFP was efficient, we get 2 strips at the end of the electrophoresis. The strip at 2070 pb was the digested pSB1C3 that we purified for the subsequent ligation.
Ligation of P2mut, X2, C4 and G2 into pSB1C3:
Objective
Ligation of P2mut, X2, C4 and G2 in pSB1C3 in order to obtain BBP2mut, BBX2, BBC4 and BBG2, for subsequent transformation and creation of a stock of bacteria. The molar ratios for the ligation were calculated using NEB BioCalculator (available here )
Materials
Concentrations of the different components after digestion and PCR purification :
pSB1C3-RFP 3 : 2.22 ng/µL (200 ng/ 90 µL)
P2mut: 2.5 ng/µL (75 ng / 30 µL)
X2: 2.5 ng/µL (75 ng / 30 µL)
C4: 1.67 ng/µL (50 ng / 30 µL)
G2: 1.67 ng/µL (50 ng / 30 µL)
In the following order, add :
NB: The ligations were done in 40 µL
Mix by pipetting
Incubate for 1h at room temperature
Transformation: competent DH5⍺ cells with ligation product BBP2mut, BBX2, BBC4, BBG2
Objective
The objective is to transforme competent DH5⍺ cells with the ligation products BBP2mut, BBX2, BBC4, BBG2.
Materials
5 aliquots of 100 µL DH5⍺ competent cells (from the 20/09/16).
Plasmid DNA : Ligation product BBP2mut, BBX2, BBC4, BBG2.
Petri dish LB+Cm: Cm concentration = 25 µg/mL.
Protocol
Experimental conditions realized :
We need 18 LB+Cm plates + 10 LB plates
Transformations protocol:
Thaw tubes of DH5⍺ competent cells on ice for 10 min. Mix gently and carefully pipette 50 µL of cells into the 4 transformation tubes on ice.
Add the 40 µL plasmid DNA to the cell mixture.
Carefully flick the tubes 4-5 times to mix cells and DNA. Do not vortex.
Place on ice for 30 min. Do not mix.
Heat shock at exactly 42°C for 45 s. Do not mix.
Place on ice for 5 min. Do not mix.
Pipette 250 µL of room temperature SOC into the mixture.
Place at 37°C for 1h at 250 rpm.
Warm selection plates to 25°C.
Mix the cells thoroughly by flicking the tubes and inverting.
Spread the corresponding volume onto each plate.
Incubate all the plates O/N at 37°C.
Results (obtain the 18/10)
Expected results :
Some colonies on the petri dishes LB+Cm plated with 50 µL of bacteria transformed with the different ligation products and more on the petri dishes LB+Cm plated with 200 µL of bacteria.
A bacterial lawn on the LB petri dishes without antibiotic.
No colonies on the LB+Cm petri dish plated with bacteria transformed with no plasmid (- control).
Obtained results:
We obtained expected results on petri-dishes plated with controls and cells transformed with BBP2mut, BBC4 and BBG2.  We obtained some colonies on petri-dishes plated with BBX2.
Interpretation
The transformation worked for BBP2mut, BBC4, BBX2 and BBG2. Colonies contain a plasmid with the Chloramphenicol resistance gene, present in pSB1C3. A PCR colonie is necessary to check the size of the plasmid present in colonies, and therefore in order to know if bacteria incorporated the correct plasmids BBP2mut, BBX2, BBC4, BBG2.
June 18,2015
2
Creative and impressive building
June 18,2015
2
How you can be a super creative
June 18,2015
2
World best photographer and photography
Facebook
Twitter
youtube
LinkedIn
iGEM IONIS
We're a group of six different schools from the IONIS Education Group. For this                                     competition we wanted to take advantages of the multiple schools and activity field                                     given by the IONIS education group to create a solid project.
Read More
Stay in touch
Location: 66 Rue Guy Môquet, 94800 Villejuif, France
email: igem.ionis@gmail.com
Download the app
Home
in the Lab
Side Projects
Results
Human                                             Practice
Team
©IONIS_IGEM_2016.
