<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Ionis_Paris_Notebook_17_10_2016 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Ionis Paris/Notebook/17 10 2016</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="row top_menu_area"><DIV class="col-xs-12 col-sm-9"><NAV class="navbar-collapse collapse in"><UL class="nav navbar-nav navbar-right menu"><LI class="smoth-scroll"><A data-hover="Project">Project</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Description">Project description</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Biology">Biology</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Hardware">Engineering</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Design">Applied Design</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Entrepreneurship">Entrepreneurship</A></LI></UL><LI class="smoth-scroll"><A data-hover="In the lab">In the lab</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Working_at_La_Paillasse">Working at La Paillasse</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Biobrick_Design">Biobrick design</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Cloning_Strategy">Cloning Strategy</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Protocols">Protocols</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Notebook">Notebook</A></LI><LI class="smoth-scroll"><A data-hover="Side-projects">Side-projects</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Measurement">CelloCad</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Collaborations">Collaborations</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Mobile_application">Mobile application</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Engagement">Education</A></LI><LI class="smoth-scroll"><A data-hover="results">Results</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Parts">Parts and Characterization</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Proof">Proof of concept</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Demonstrate">Demonstration</A></LI><LI class="smoth-scroll"><A data-hover="Human Practices">Human Practices</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/HPintro">Overview</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Quantifly">Project Genesis</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Air_pollution">Air Pollution</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Survey">Opinion Survey</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Ethics">Ethics and Biosafety</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/IntegratedHP">Our Actions</A></LI><LI class="smoth-scroll"><A data-hover="Safety">Safety</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Labsafety">Lab safety</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Projectsafety">Project safety</A></LI><LI class="smoth-scroll"><A data-hover="Team">Team</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Team">The team</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Events">Events</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Attributions">Attributions</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Acknowledgement">Acknowledgement (Sponsors and advisors)
</A></LI><LI class="waves-effect waves"><A href="https://2016.igem.org/Team:Ionis_Paris/Prices">Prizes</A></LI></NAV><NAV class="mobile_menu hidden"><UL><LI class="smoth-scroll"><A href="#body">Project</A></LI><LI><A href="index.html">Home Page 1</A></LI><LI><A href="index-2.html">Home Page 2</A></LI><LI><A href="index-3.html">Home Page 3</A></LI><LI><A href="index-4.html">Home Page 4</A></LI><LI><A href="index-5.html">Home Page 5</A></LI><LI><A href="index-6.html">Home Page 6</A></LI><LI><A href="index-7.html">Home Page 7</A></LI><LI><A href="index-8.html">Home Page 8</A></LI><LI><A href="index-9.html">Home Page 9</A></LI></UL><LI class="smoth-scroll"><A href="#services">Services</A></LI><LI class="smoth-scroll"><A href="#about">About</A></LI><LI class="smoth-scroll"><A href="#Portfolio">Portfolio</A></LI><LI><A href="protfolio-1.html">Protfolio 1</A></LI><LI><A href="protfolio-2.html">Protfolio 2</A></LI><LI><A href="protfolio-3.html">Protfolio 3</A></LI><LI><A href="case_study.html">Case Study</A></LI><LI class="smoth-scroll"><A href="#blog">Blog</A></LI><LI><A href="blog_grid.html">Blog Grid</A></LI><LI><A href="blog-left_sidebar.html">Blog Left Sidebar</A></LI><LI><A href="blog-right_sidebar.html">Blog Right Sidebar</A></LI><LI><A href="single_blog-full.html">Single Blog Full</A></LI><LI><A href="single_blog-left_sidebar.html">
                                                Single Blog Left Sidebar
                                            </A></LI><LI><A href="single_blog-right_sidebar.html">
                                                Single Blog Right Sidebar
                                            </A></LI><LI class="smoth-scroll"><A href="#contact">Contact US</A></LI></NAV></DIV></DIV></DIV><SECTION class="blog_banner_cloning"><DIV class="container"><DIV class="row"><DIV class="col-sm-12"><DIV class="banner_title"><H1 id="back_to_the_top">October 17th 2016</H1></DIV></DIV></DIV></DIV></SECTION><SECTION class="blog_area section-padding"><DIV class="container"><DIV class="row"><DIV class="blog_grid_area"><DIV class="col-xs-12 col-sm-9"><DIV class="bloggrid_right"><DIV class="blog_top"><H2 class="blog_topHd"><FONT color="”#279AD3”">PCR : X6 and P2mut from BB2</FONT></H2><H3><FONT color="”94FAF1”"> Objectives </FONT></H3><P>The overall purpose is to amplify P2 from BB2 by mutate the deficient codon start with the SDM-F primer to obtain BB2mut and by adding a 6X-His tag with the primer AHis-tag to obtain BBX2 (XylR-6His).</P><H3><FONT color="”94FAF1”"> Materials </FONT></H3><P>Plasmid template: BB2-39 (from Miniprep 29/07/2016).
Primers: SDM-F (forward), AHis-tag (reverse) and A13 (reverse).</P><H3><FONT color="”94FAF1”"> Protocol </FONT></H3><H5><FONT color="”#3CB5E1”">PCR</FONT></H5><P>1.  2 Mix for 2 samples (Total volume of Mix : 98 µL), in an Eppendorf tube :</P><LI><P>96 µL H2O</P></LI><LI><P>1 µL Primer SDM-F  (0.5 µM final)</P></LI><LI><P>1 µL Primer A13 in Mix 1 / AHis-tag in Mix 2 (0.5 µM final)</P></LI><LI><P>50 µL Q5 High-Fidelity 2X Master Mix (NEB #M0492S)</P></LI><P>2.  Add in 4 PCR tubes, in the respected order:</P><LI><P>49 µL Mix 1 (P2mut and Control1) / 49 µL Mix 2 (X6 and Control 2)</P></LI><LI><P>1 µL DNA (P2mut and X6) / 1 µL H20 (Control 1 and 2)</P></LI><P>—&gt; Gently mix the reaction and short spin centrifugation</P><P>3.  Set the following parameters for the PCR reaction :</P><LI><P>X2 (1794 bp) and P2mut (1747 bp)
               Lid température: 98°C
               Initial denaturation: 98°C, 30 s
               30 cycles of: 98°C, 10 s
                                  59 °C, 30 s
                                  72°C, 52 s
               Final extension: 72°C, 2 min
               Hold: 4°C</P></LI><H5><FONT color="”#3CB5E1”">Electrophoresis: for screening the PCR results</FONT></H5><P>1% Agarose gel:</P><OL><LI><P>Put 1 g agarose + 100 mL TAE 1X in a bottle of 500 mL</P></LI><LI><P>Mix and heat it 2 min 30 s in the microwaves. Wait the cooling of the bottle until it is tepid.</P></LI><LI><P>Add 5 µL of Gel Red 10,000 X (0.5 X final)</P></LI><LI><P>Flow the gel and place the combs</P></LI><LI><P>Wait until it is solidified. Remove slowly the combs.</P></LI></OL><P>Drop-off:</P><OL><LI><P>Short Speed centrifugation of samples.</P></LI><LI><P>Addition of 1 µL of Purple loading dye 6 X in 5 µL of sample.</P></LI><LI><P>Drop-off 10 µL of Purple ladder and 6 µL of each samples.</P></LI><LI><P>Run at 90 V.</P></LI></OL><H5><FONT color="”#3CB5E1”">PCR purification :</FONT></H5><P>QIAquick PCR purification kit (qiagen, 28106), according to the protocol given by the supplier (available <A href="https://www.qiagen.com/fi/resources/resourcedetail?id=390a728a-e6fc-43f7-bf59-b12091cc4380&amp;lang=en">here</A>)</P><OL><LI><P>Add 5 volumes Buffer PB (225 µL) to 1 volume of the PCR reaction (45 µL) and mix. The color of the mixture is yellow.</P></LI><LI><P>Load the sample to the QIAquick column. Centrifuge for 1 min at 13,000 rpm and discard flow-through.</P></LI><LI><P>Add 750 µL Buffer PE. Centrifuge for 1 min at 13,000 rpm and discard flow-through.</P></LI><LI><P>Centrifuge once more for 1 min at 13,000 rpm.</P></LI><LI><P>Place each QIAquick column in a clean 1.5 mL microcentrifuge tube.</P></LI><LI><P>Add 30 µL Buffer EB to the center of the QIAquick membrane, let stand for 1 min, and centrifuge for 1 min at 13,000 rpm.</P></LI><LI><P>Calculate the quantity of DNA with the Nanodrop.</P></LI><LI><P>Store the purified DNA at -20°C.</P></LI></OL><H3><FONT color="”94FAF1”"> Results  </FONT></H3><P><FONT color="”46BB0A”"> Expected results / Obtained results:</FONT></P><H3><FONT color="”94FAF1”"> Interpretation  </FONT></H3><P>We obtain the desired strip for P2mut and X2, as shown on the gel above the strips are closed to 1,747 bp and 1,794 bp.
It seems that P2mut and X2 have been properly amplified.</P><H2 class="blog_topHd"><FONT color="”#279AD3”">Digestion: pSB1C3, P2mut, X2, C4 and G2</FONT></H2><H3><FONT color="”94FAF1”"> Objective  </FONT></H3><P>Double digestion of pSB1C3-RFP, P2mut, X2, C4 and G2 by EcoRI and PstI for the subsequent ligation of P2mut, X6, C4 and G2 in pSB1C3.</P><H3><FONT color="”94FAF1”"> Materials </FONT></H3><H5><FONT color="”#3CB5E1”">Stock concentrations:</FONT></H5><P>pSB1C3-RFP 3 : 104.14 ng/µL (from miniprep 19/07)
P2mut: ~ 50 ng/µL (from PCR 17/10)
X2: ~ 150 ng/µL (from PCR 17/10)
C4: ~ 150 ng/µL (from PCR 08/10)
G2: ~ 150 ng/µL (from PCR 08/10)</P><H5><FONT color="”#3CB5E1”">Quantity of DNA required for the ligations:</FONT></H5><P>P2mut: Digestion of 75 ng (ratio 1:1 = 21.10 ng needed, ratio 2:1 = 42.20 ng needed —&gt; 64 ng needed)
X2: Digestion of 75 ng (ratio 1:1 = 21.67 ng needed, ratio 2:1 = 43.33 ng needed —&gt; 65 ng needed)
C4: Digestion of 50 ng (ratio 3:1 = 14.57 ng needed, ratio 5:1 = 24.28 ng needed —&gt; 40 ng needed)
G2: Digestion of 50 ng (ratio 2:1 = 17.08 ng needed, ratio 3:1 = 25.62 ng needed —&gt; 43 ng needed)
pSB1C3-RFP: Digestion of 312 ng (25 ng needed per ratio —&gt; 200 ng of digested pSB1C3 needed —&gt; 304 ng of pSB1C3-RFP needed)</P><H3><FONT color="”94FAF1”"> Protocol </FONT></H3><H5><FONT color="”#3CB5E1”">Digestion:</FONT></H5><OL><LI><P>In a 1.5 mL Eppendorf tube, adding in the respected order (bigger volume first and enzyme last) :</P></LI><P>NB: The digestion were done in 20 µL.</P><LI><P>Short Spin Centrifugation</P></LI><LI><P>ncubation 1h at 37°C</P></LI><LI><P>Store at 4°C before gel electrophoresis and purification</P></LI></OL><H5><FONT color="”#3CB5E1”">Electrophoresis for digested pSB1C3-RFP 1, 2 and 3:</FONT></H5><P>1% Agarose gel:</P><OL><LI><P>Put 1 g of agarose low melting point + 100 mL of TAE 1X in a bottle of 500 mL.</P></LI><LI><P>Mix and heat it 2 min 30 s in the microwaves. Wait the cooling of the bottle until it is tepid.</P></LI><LI><P>Add 3 µL of Gel Red 10,000 X (0.3 X final).</P></LI><LI><P>Flow the gel and place the combs.</P></LI><LI><P>Wait until it is solidified. Remove slowly the combs.</P></LI></OL><P>Drop-off:</P><OL><LI><P>Short Speed centrifugation of samples.</P></LI><LI><P>Addition of 4 µL of Purple loading dye 6X in the 20 µL of sample.</P></LI><LI><P>Drop-off 10 µL of Purple ladder and 25 µL of sample.</P></LI><LI><P>Run at 90 V.</P></LI></OL><H5><FONT color="”#3CB5E1”">Gel purification for digested pSB1C3 1, 2 and 3:</FONT></H5><P>QIAquick Gel purification kit (Qiagen, 28704), according to the protocol given by the supplier (available <A href="https://www.qiagen.com/us/resources/resourcedetail?id=f4ba2d24-8218-452c-ad6f-1b6f43194425&amp;lang=en">here</A>)</P><OL><LI><P>Excise the DNA fragments from the agarose gel. Gel slice Weigh = 0.211 g (1) / 0.225 g (2) / 0.231 g (3)</P></LI><LI><P>Add 3 volumes Buffer QG (633 µL (1) / 675 µL (2) / 693 µL (3) ) to 1 volume of gel.</P></LI><LI><P>Incubate at 50°C for 10 min until the gel slice has completely dissolved. Vortex the tube every 2–3 min to help dissolve gel. The color of the mixture is yellow.</P></LI><LI><P>Add 1 gel volume isopropanol to the sample and mix.</P></LI><LI><P>Load 800 µL of each samples to the QIAquick column. Centrifuge for 1 min at 13,000 rpm and discard flow-through. Load the rest and spin again.</P></LI><LI><P>Add 500 µL Buffer QG. Centrifuge for 1 min at 13,000 rpm and discard flow-through.</P></LI><LI><P>Add 750 µL Buffer PE. Centrifuge for 1 min at 13,000 rpm and discard flow-through.</P></LI><LI><P>Centrifuge once more for 1 min at 13,000 rpm.</P></LI><LI><P>Place QIAquick column into a clean 1.5 mL microcentrifuge tube.</P></LI><LI><P>Add 30 µL Buffer EB to the center of the QIAquick membrane, let stand for 1 min, and centrifuge for 1 min at 13,000 rpm.</P></LI><LI><P>Store the purified DNA at 4°C before the ligation.</P></LI></OL><H5><FONT color="”#3CB5E1”">PCR purification for digested P2mut, X2, C4, G2:</FONT></H5><P>QIAquick PCR purification kit (qiagen, 28106), according to the protocol given by the supplier (available <A href="https://www.qiagen.com/fi/resources/resourcedetail?id=390a728a-e6fc-43f7-bf59-b12091cc4380&amp;lang=en">here</A>)</P><OL><LI><P>Add 5 volumes Buffer PB (100 µL) to 1 volume of the sample (20 µL) and mix. The color of the mixture is yellow.</P></LI><LI><P>Load the sample to the QIAquick column. Centrifuge for 1 min at 13,000 rpm and discard flow-through.</P></LI><LI><P>Add 750 µL Buffer PE. Centrifuge for 1 min at 13,000 rpm and discard flow-through.</P></LI><LI><P>Centrifuge once more for 1 min at 13,000 rpm.</P></LI><LI><P>Place each QIAquick column in a clean 1.5 mL microcentrifuge tube.</P></LI><LI><P>Add 30 µL Buffer EB to the center of the QIAquick membrane, let stand for 1 min, and centrifuge for 1 min at 13,000 rpm.</P></LI><LI><P>Store the purified DNA at 4°C before the ligation.</P></LI></OL><H3><FONT color="”94FAF1”"> Results </FONT></H3><H5><FONT color="”#3CB5E1”">Electrophoresis:</FONT></H5><P><FONT color="”46BB0A”"> Expected results / Obtained results:</FONT></P><H3><FONT color="”94FAF1”"> Interpretation </FONT></H3><P>The digestion of pSB1C3-RFP was efficient, we get 2 strips at the end of the electrophoresis. The strip at 2070 pb was the digested pSB1C3 that we purified for the subsequent ligation.</P><H2 class="blog_topHd"><FONT color="”#279AD3”">Ligation of P2mut, X2, C4 and G2 into pSB1C3:</FONT></H2><H3><FONT color="”94FAF1”"> Objective </FONT></H3><P>Ligation of P2mut, X2, C4 and G2 in pSB1C3 in order to obtain BBP2mut, BBX2, BBC4 and BBG2, for subsequent transformation and creation of a stock of bacteria.
The molar ratios for the ligation were calculated using NEB BioCalculator (available <A href="http://nebiocalculator.neb.com/#!/ligation">here</A>)</P><H3><FONT color="”94FAF1”"> Materials </FONT></H3><H5><FONT color="”#3CB5E1”">Concentrations of the different components after digestion and PCR purification :</FONT></H5><LI><P>pSB1C3-RFP 3 : 2.22 ng/µL (200 ng/ 90 µL)</P></LI><LI><P>P2mut: 2.5 ng/µL (75 ng / 30 µL)</P></LI><LI><P>X2: 2.5 ng/µL (75 ng / 30 µL)</P></LI><LI><P>C4: 1.67 ng/µL (50 ng / 30 µL)</P></LI><LI><P>G2: 1.67 ng/µL (50 ng / 30 µL)</P></LI><OL><LI><P>In the following order, add :</P></LI><P>NB: The ligations were done in 40 µL</P><LI><P>Mix by pipetting</P></LI><LI><P>Incubate for 1h at room temperature</P></LI></OL><H2 class="blog_topHd"><FONT color="”#279AD3”">Transformation: competent DH5⍺ cells with ligation product BBP2mut, BBX2, BBC4, BBG2</FONT></H2><H3><FONT color="”94FAF1”"> Objective </FONT></H3><P>The objective is to transforme competent DH5⍺ cells with the ligation products BBP2mut, BBX2, BBC4, BBG2.</P><H3><FONT color="”94FAF1”">Materials</FONT></H3><LI><P>5 aliquots of 100 µL DH5⍺ competent cells (from the 20/09/16).</P></LI><LI><P>Plasmid DNA : Ligation product BBP2mut, BBX2, BBC4, BBG2.</P></LI><LI><P>Petri dish LB+Cm: Cm concentration = 25 µg/mL.</P></LI><H3><FONT color="”94FAF1”"> Protocol </FONT></H3><H5><FONT color="”#3CB5E1”">Experimental conditions realized : </FONT></H5><P>We need 18 LB+Cm plates + 10 LB plates</P><H5><FONT color="”#3CB5E1”">Transformations protocol:</FONT></H5><OL><LI><P>Thaw tubes of DH5⍺ competent cells on ice for 10 min. Mix gently and carefully pipette 50 µL of cells into the 4 transformation tubes on ice. </P></LI><LI><P>Add the 40 µL plasmid DNA to the cell mixture. </P></LI><LI><P>Carefully flick the tubes 4-5 times to mix cells and DNA. Do not vortex.</P></LI><LI><P>Place on ice for 30 min. Do not mix.</P></LI><LI><P>Heat shock at exactly 42°C for 45 s. Do not mix.</P></LI><LI><P>Place on ice for 5 min. Do not mix.</P></LI><LI><P>Pipette 250 µL of room temperature SOC into the mixture.</P></LI><LI><P>Place at 37°C for 1h at 250 rpm.</P></LI><LI><P>Warm selection plates to 25°C.</P></LI><LI><P>Mix the cells thoroughly by flicking the tubes and inverting.</P></LI><LI><P>Spread the corresponding volume onto each plate.</P></LI><LI><P>Incubate all the plates O/N at 37°C.</P></LI></OL><H3><FONT color="”94FAF1”"> Results (obtain the 18/10) </FONT></H3><P><FONT color="”46BB0A”"> Expected results :</FONT></P><P>Some colonies on the petri dishes LB+Cm plated with 50 µL of bacteria transformed with the different ligation products and more on the petri dishes LB+Cm plated with 200 µL of bacteria.
A bacterial lawn on the LB petri dishes without antibiotic.
No colonies on the LB+Cm petri dish plated with bacteria transformed with no plasmid (- control).</P><P><FONT color="”46BB0A”">Obtained results:</FONT></P><P>We obtained expected results on petri-dishes plated with controls and cells transformed with BBP2mut, BBC4 and BBG2. 
We obtained some colonies on petri-dishes plated with BBX2.</P><H3><FONT color="”94FAF1”"> Interpretation </FONT></H3><P>The transformation worked for BBP2mut, BBC4, BBX2 and BBG2. Colonies contain a plasmid with the Chloramphenicol resistance gene, present in pSB1C3. A PCR colonie is necessary to check the size of the plasmid present in colonies, and therefore in order to know if bacteria incorporated the correct plasmids BBP2mut, BBX2, BBC4, BBG2.</P><DIV role="tabpanel" class="tab-pane fade" id="popular_post"><DIV class="post_list"><DIV class="post_text"><UL class="blog_s"><LI><A href="#">June 18,2015</A></LI><LI><A href="#">2
                                                        </A></LI></UL><H4><A href="#">Creative and impressive building</A></H4></DIV></DIV><DIV class="post_list"><DIV class="post_text"><UL class="blog_s"><LI><A href="#">June 18,2015</A></LI><LI><A href="#">2
                                                        </A></LI></UL><H4><A href="#">How you can be a super creative</A></H4></DIV></DIV><UL class="blog_s"><LI><A href="#">June 18,2015
                                                        </A></LI><LI><A href="#">2
                                                        </A></LI></UL><H4><A href="#">World best photographer and photography</A></H4></DIV></DIV></DIV></DIV></DIV></DIV></DIV></SECTION></DIV></DIV></DIV><DIV class="footer_social"><DIV class="container-fluid"><DIV class="row"><UL class="ft_top clearfix"><LI class="waves-effect waves-light"><A href="https://www.facebook.com/ionisigem/?fref=ts" target="_blank">Facebook
                        </A></LI><LI class="waves-effect waves-light"><A href="https://twitter.com/igem_ionis" target="_blank">Twitter
                        </A></LI><LI class="waves-effect waves-light"><A href="https://www.youtube.com/channel/UC0RyQsB5YpweSRBYQlAdZDA" target="_blank">youtube
                        </A></LI><LI class="waves-effect waves-light"><A href="https://www.linkedin.com/company/igem-ionis" target="_blank">LinkedIn
                        </A></LI></UL></DIV></DIV></DIV><FOOTER class="footer_area"><DIV class="footer_middle"><DIV class="container"><DIV class="row"><DIV class="row"><DIV class="col-lg-5 col-md-5 col-sm-12"><DIV class="middle_content"><H4>iGEM IONIS</H4><P> We're a group of six different schools from the IONIS Education Group. For this
                                    competition we wanted to take advantages of the multiple schools and activity field
                                    given by the IONIS education group to create a solid project.</P><A href="https://2016.igem.org/Team:Ionis_Paris/Team">Read More</A></DIV></DIV><DIV class="col-lg-3 col-lg-offset-1 col-md-4 col-sm-7"><DIV class="footer_Widgets"><H4>Stay in touch</H4><DIV class="ft_contact"><UL><LI><SPAN>Location: 66 Rue Guy Môquet, 94800 Villejuif, France</SPAN></LI><LI><A href="mailto:igem.ionis@gmail.com">email: igem.ionis@gmail.com</A></LI></UL></DIV></DIV></DIV><DIV class="col-lg-3 col-md-3 col-sm-5"><DIV class="footer_Widgets"><H4>Download the app</H4></DIV></DIV></DIV></DIV></DIV><DIV class="footer_bottom"><DIV class="container"><DIV class="ft_area"><DIV class="row"><DIV class="col-md-7 fix_p_l"><NAV><UL class="ft_bottom"><LI><A href="https://2016.igem.org/Team:Ionis_Paris">Home</A></LI><LI><A href="https://2016.igem.org/Team:Ionis_Paris/Working_at_La_Paillasse">in the Lab</A></LI><LI><A href="https://2016.igem.org/Team:Ionis_Paris/Measurement"> Side Projects</A></LI><LI><A href="https://2016.igem.org/Team:Ionis_Paris/Parts">Results</A></LI><LI><A href="https://2016.igem.org/Team:Ionis_Paris/HPintro">Human
                                            Practice</A></LI><LI><A href="https://2016.igem.org/Team:Ionis_Paris/Team">Team</A></LI></UL></NAV></DIV><DIV class="col-md-5 fix_p"><DIV class="ft_paragraph"><P>©IONIS_IGEM_2016.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></BODY></HTML>