"
Page
Discussion
View source
History
teams
Log in
 
Team:Uppsala/p-coumaric-acid
From 2013.igem.org
Chassi
Probiotics
Promoters
Reporter genes
Toxin-antitoxin system
Vectors
Signal peptide
Metabolic engineering
p-Coumaric acid
Resveratrol
Lycopene
β-carotene
Saffron
Astaxanthin
Zeaxanthin
Miraculin
Chromoproteins
Safety experiment
Results
Kinetic model
Modeling tutorial
Toxicity model
Resveratrol group
Carotenoid group
Chassi group
Advisors
Yoghurt +
SynBioDay
Biosafety and ethics
Public opinion
High school & media
BioArt
A LactoWorld
Killswitches
Patent
Safety form
Protocols
p-Coumaric Acid
Opening up the phenylpropanoid pathway to iGEM
The phenylpropanoid pathway metabolites, which starts with the aminoacids tyrosine or phenylalanine as precursors has many important health aspects. Tyrosine ammonia lyase produces p-coumaric acid from tyrosine. We have searched the parts registry of a  working and characterized tyrosine ammonia lyase, but all earlier iGEM attempts showed no characterization or availability whatsoever. That is why we decided to make a fully working and characterized biobrick of tyrosine ammonia lyase. By using a new to iGEM bacterial version of tyrosine ammonia lyase, we will open up the diverse metabolic pathway of flavonoids, anthocyanins, and stilbenes, to name a few. [1] In this project, we will try to produce the antioxidants p-coumaric acid and resveratrol.
p-Coumaric acid
p-Coumaric acid is an antioxidant with many beneficial health aspects.It has been shown that it reduces the risk of one of our greatest modern western diseases,  atherosclerosis. [2] It has also been shown to reduce the risk of stomach cancer. [3] That is why we think this is a very important ingredient in our probiotic bacteria.
Methods
Tyrosine ammonia lyase (TAL) from rhodobacter sphaeroides was obtained from J.Conrado et al. [4] The previous uncharacterized versions used in iGEM have all been of eukaryotic origin. We biobricked TAL with the ribosome binding site B0034 and overhangs in a single PCR with primer overhangs. We also made a version with 6-HIS-tag for enzyme expression analysis. TAL was also mutagenized to remove an illegal Not1 site. We verified all of our genetic constructs with sequencing.
We have expressed TAL in E. coli DH5alpha and E. coli nissle, a probiotic E. coli obtained from Trieste iGEM 2012. TAL will also be characterized in lactobacillus, by transforming the construct with our shuttle vector.   This construct can also be used to produce the precursor for our resveratrol producing bacteria.
Results
Summary
We managed to clone out and biobrick tyrosine ammonia lyase and verify the biobrick by sequencing. We have also done successful characterization of this part, showing that it works as expected. We managed to express our enzyme and detect it in a western blot, and also detected our metabolite in both spectrophotometry and HPLC. The biobrick was characterized in E. coli d5alpha and E. coli nissle. For detailed information about the characterization experiments, see the protocol section.
Biobrick
We succeded in the cloning and sequencing of our biobrick, Tyrosine ammonia lyase from  rhodobacter sphaeroides with the RBS B0034. This construct was designed to work in various organisms, such as lactobacillus and E. coli. Sequencing was done at GATC biotech and Uppsala Genome center using Sanger  sequencing.
Tyrosine ammonia lyase with rbs: BBa_K1033000
Western blot
We also succeeded in expressing the enzyme tyrosine ammonia-lyase (TAL) in E. coli.
To enable the detection of this protein by anti-his antibodies, 6-histidine tags were incorporated in the  sequence. By doing this we would be able detect our enzyme with anti-his antibodies.
We expressed our protein with a promoter working in both lactobacillus and E. coli. We did this to later be able to transfer this TAL construct to lactobacilli strains.
The size of our protein was calculated using ProtParam [5] to 54.9 kDA.
1. Positive control
2. TAL with CP8 promoter
3. TAL with J23110 promoter
4. Negative control
Figure 1: SDS-page and western blot. Expression of Tyrosine ammonia lyase with constitutive promoters in E. coli. The negative control has no bands, showing that there is no natural protein in wildtype E. coli with the same attributes. The positive control shows that the staining has worked. The two sample lanes 2 and 3 have clear bands around 55 kDA which suggests that these two recombinant E. coli strains produce the TAL protein.
Spectrophotometry
As the next step, we have made a spectrophotometric assay of our metabolite p-coumaric acid produced by E. coli. By using n-octanol and a two-phase extraction, we were able to extract our metabolite from the LB medium incubated with TAl producing bacteria. After this purification step, we were able to characterize our biobrick on a spectrophotometer. The positive control we used had a final p-coumaric concentration of 500 µl. The spectrophotometry experiments show that there is a somewhat lower concentration in the sample extracts after 48 hours of stationary phase incubation compared to the positive control.
The absorbance spectra of extracts from bacterial cultures grown at stationary phase for 48 hours were collected and the concentration of p-coumaric acid was calculated from the absorbance spectra in an ordinary least squares manner with a MATLAB script . The concentrations of p-coumaric acid in the cultures were: 228 µM for pSB3K3-J23101-TAL, 158 µM for pSB3K3-J23110-TAL and 311 µM for pSB3K3-CP8-TAL (spectra shown in figure 2 below).
Figure 2: Absorbance spectra of n-octanol extracts collected from bacterial cultures. Samples were collected 21 h and 48 h after 30 °C incubation. The negative control is an extract from a strain with no TAL gene on the transformed plasmid. The positive control is an extract of a culture of the same strain as the negative control but with added p-coumaric acid to a concentration of 500 µM before extraction. P-coumaric acid absorbance spectra has two peaks. The one around 305 nm is preferable to detect because of background noise from components found in LB that do transfer to the octanol phase in the extraction. (a) Spectra from the strain with TAL CDS with promoter J23110. (b) Spectra from the strain with TAL CDS with promoter CP8. (c) Spectra from the strain with TAL CDS with promoter J23101.(d) Standards of pure p-coumaric acid solved in n-octanol. The standards show that p-coumaric acid has a broad peak in absorbance around 305 nm.
High pressure liquid chromatography
HPLC characterization further proved that we have succefull production of p-coumaric acid in E. coli. The next step will be to transfer the genes to our shuttle vector to express them in lactobacillus.
Figure 3. Graph showing the HPLC result of an octanol extract prepared from E. coli expressing tyrosine ammonia lyase. Reverse phase HPLC with a C18 matrix was used. A mix of acetate buffer and acetonitrile were used as mobile phase in this gradient elution experiment. The absorbance of the eluate was detected at 303 nm. The peak for p-coumaric acid can be seen ~9 min: the same retention time as the standard sample below.
Figure 4. Graph showing the HPLC result of a standard sample with pure p-coumaric acid (50 µM) in n-octanol.
Figure 5. HPLC chromatogram of a n-octanol extract from a E. coli culture without our biobrick tyrosine ammonia lyase. No peak at 9 minutes can be detected in this negative control.
References
[1] Zhixion Xue et al, Identification and functional expression of tyrosine ammonia-lyase and its mutants from the photosynthetic bacterium rhodobacter sphaeroides, J Ind Microbiol Biotechnol (2007) 34:599-604
[2] Lun-Yi Zang et al, Effect of antioxidant protection by p-coumaric acid on low-density lipoprotein cholesterol oxidation, Am J Physiol Cell Physiol. 2000 Oct;279(4):C954-60.
[3] Lynnette R. Ferguson, Shuo-tun Zhu, Philip J. Harris, Antioxidant and antigenotoxic effects of plant cell wall hydroxycinnamic acids in cultured HT-29 cells, Issue, Molecular Nutrition & Food Research, Volume 49, Issue 6, pages 585–593, June 2005
[4] Robert J. Conrado et al, DNA guided assembly of biosynthetic pathways promotes improved catalytic effiency. Nucleic Acids Research , 2012, Vol 40 NO 4, 1879-1889
[5] http://web.expasy.org/protparam/
Retrieved from " http://2013.igem.org/Team:Uppsala/p-coumaric-acid "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
