"
Page
Discussion
View source
History
teams
Log in
 
Team:Uppsala/miraculin
From 2013.igem.org
Chassi
Probiotics
Promoters
Reporter genes
Toxin-antitoxin system
Vectors
Signal peptide
Metabolic engineering
p-Coumaric acid
Resveratrol
Lycopene
β-carotene
Saffron
Astaxanthin
Zeaxanthin
Miraculin
Chromoproteins
Safety experiment
Results
Kinetic model
Modeling tutorial
Toxicity model
Resveratrol group
Carotenoid group
Chassi group
Advisors
Yoghurt +
SynBioDay
Biosafety and ethics
Public opinion
High school & media
BioArt
A LactoWorld
Killswitches
Patent
Safety form
Protocols
Miraculin
Sweet polypeptide of goodness
Synsepalum Dulcificum is a plant found along the coast of West Africa. It grows red berries about the size of coffee beans that have a quite peculiar property: they make sour things taste sweet. This effect was first documented in the late 1700's, and has caused the berries to be referred to as "miracle berries". The "miraculous" property is actually caused by a glycoprotein dubbed miraculin, that binds to the taste receptors on the  tongue and modifies their response. The exact mechanism of the interaction is not entirely understood, but it has been used  as a natural sweetener in different circumstances throughout the centuries. [1]
The gene coding for the protein miraculin has never been used in iGEM before. Due to its unusual properties we decided to implement miraculin in our probiotic yoghurt where it would act as a form of sweetner
Structure
In its glycoprotein form, miraculin appears as a homotetramer, composed of two dimer subunits. The dimers, in turn, are composed of two monomers liked by disulfide bridge. The exact mechansm of interaction between miraculin and the taste-receptors is not completely explored, but it is known that miraculin changes it's shape at low pH-level, and that two histidine residues play an important role in binding to the receptor. This means that while miraculin can assocate to the receptors at normal circuimstances, it is not until the environment turns acidic that the conformation change that presumably causes the sweet sensation is initiated. When the pH returns to normal, so does the miraculin structure. [2]
Figure 3. the structure of the miraculin homodimer subunit. The 191 residues long polypeptide is glycosylated to a 13.9% carbohydrate content.
Methods
We synthesized the miraculin gene along with a B0034 ribosome binding site for expression, the export tag USP45 for exportation of the complete glycopeptide out of the cell, and a 6xhis-tag for characterization purposes. Additionally, the gene was isolated with primers using biobrick overhangs for increased modularity in further use. We verify all our genetic constructs by sequencing
Results
We were successful in both subcloning the synthetic construct to a desired plasmid  and isolating the Miraculin gene by itself. The sequencencing was done at GATC biotch and Uppsala Genome center. We were not successful in either assembling or mutagenizing a promoter to the construct, and as such could not characterize it further.
Biobricks
BBa_K1033120 - B0034-ExpTag-Linker-Miraculin-Linker-6xHis
BBa_K1033122 - B0034-Miraculin
BBa_K1033121 - Miraculin
References
[1] Slater, Joanna (2007-03-30). "To Make Lemons Into Lemonade, Try 'Miracle Fruit'" . Wall Street Journal. [2013-10-01].
[2] Theerasilp S, Hitotsuya H, Nakajo S, Nakaya K, Nakamura Y, Kurihara Y (April 1989). "Complete amino acid sequence and structure characterization of the taste-modifying protein, miraculin". J. Biol. Chem. 264 (12): 6655–9. < br>
All pictures are taken from wikimedia commons
Retrieved from " http://2013.igem.org/Team:Uppsala/miraculin "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
