"
Page
Discussion
View source
History
teams
Log in
 
Team:Uppsala/saffron
From 2013.igem.org
Chassi
Probiotics
Promoters
Reporter genes
Toxin-antitoxin system
Vectors
Signal peptide
Metabolic engineering
p-Coumaric acid
Resveratrol
Lycopene
β-carotene
Saffron
Astaxanthin
Zeaxanthin
Miraculin
Chromoproteins
Safety experiment
Results
Kinetic model
Modeling tutorial
Toxicity model
Resveratrol group
Carotenoid group
Chassi group
Advisors
Yoghurt +
SynBioDay
Biosafety and ethics
Public opinion
High school & media
BioArt
A LactoWorld
Killswitches
Patent
Safety form
Protocols
Saffron
Saffron made accessible
The carotenoid pathway provides an incredible range of metabolites, just waiting to be utilized. Despite this, the availability is today quite limited, at least within the scope of an iGEM project. For our project, we decided to focus on the pathway that leads from zeaxanthin (a derivative of beta-Carotene ) to the main providers of taste and color in saffron: picrocrocin, crocin and safranal [1] . This is a pathway that had previously been admitted to the registry, but never made available as a part. We decided to characterize each separate gene and further make them available for use.
Figure 1 Crocus Sativus, the flower from which saffron can be extracted. Figure 2 Saffron samples in different dilutions.
The compounds
Picrorocin, crocin and safranal are all derivatives of zeaxanthin. Picrocrocin, along with safranal are the main providers in the characteristic taste of saffron, while crocin is responsible for the color of said spice. Apart from antioxidative properties, shown to have some positive health benefits, these compounds provide an intriguing flavoring and coloring option for a probiotic product.
Methods
The genes for zeaxanthin 7,8-cleavage dioxygenase (ZCD) and Uracil gycosyltransferase (UGTCs2) were obtained in the a synthetic construct provided to us by the Washinton University iGEM-team WashU 2012. The genes were individually isolated with PCR using primers with overhangs containing the ribosome binding site B0034 for expression in E.coli (only for UGTCs2), and 6x-his tag for characterization purposes. Additionally, each part was isolated with only biobrick overhangs for increased modularity in further use. We also designed a primer with an additional restriction site for fusing a synthetic histidine patch thioredoxin to the zeaxanthin 7,8-cleavage dioxygenase to solve solubility issues reported by previous users.  We attempted assembly with strong and weak constitutive promoters from the J23-series for expression in E.coli, as well as our new  CP-series for use in both E.coli and lactobacillus. E.coli and lactobacillus on all our constructs.
Results
Summary
We managed to clone both UGTCs2 and ZCD with and without RBS and His-tag, as well as subcloning the synthetic histidine patch thioredoxin, and verified the result by sequencing. We experienced issues assembling our constructs with promoters, as well as the histidine patch thioredoxin with ZCD. We managed to mutagenize a UGTCs2 construct with a promoter, and characterization was attempted using western blot with a negative result.
Characterization
From previous user experiences, we know that zeaxanthin 7,8-cleavage dioxygenase in its native form forms inclusion bodies in the cytoplasm of probiotic cells. These insoluble inclusion bodies form aggregates, which inhibits growth of the transformed bacteria. We decided to solve this problem by fusing the enzyme with a synthetic histidine patch thioredoxin domain that increases solubility of the translated product. This was done by including a NgoMIV restriction site found right at the end of the thioredoxin sequence in the overhang of forward cloning primer of the ZCD gene, and use this to ligate the two. While all clonings of ZCD were successful, neither the ligation of ZCD and histidine patch thioredoxin, nor the assembly of promoters to the histidine patch thioredoxin were successful.
All clonings of UGTCs2 were successful. We did not manage to assemble any promoters to the gene by means of biobrick asssembly, but we did manage to perform a mutagenesis with primers containing a J23-series promoter. This construct was analyzed with a western blot experiment that showed a negative result. We can only speculate on this lack of expression. Previous experiences do not mention any difficulties expressing Uracil gycosyltransferase, but the descriptions on this area are generally quite lacking.
Biobricks
We succeeded in the cloning and sequencing of several biobricks isolated from the synthetic construct we received from WashU 2012. Sequencing was done at GATC biotech and Uppsala Genome center.
BBa_K1033110 - Zeaxanthin 7,8-cleavage dioxygenase (ZCD)
BBa_K1033111 - Zeaxanthin 7,8-cleavage dioxygenase (ZCDHis) with His tag
BBa_K1033112 - "Uracil glucosyltransferase (UGTCs2)"
BBa_K1033113 - Uracil glucosyltransferase (B0034UGTCs2-6xHis) with RBS and Histag
BBa_K1033115 - Histidine Patch Thioredoxin
References
[1] http://www.leffingwell.com/download/saffron.pdf
[2] http://www.tandfonline.com/doi/abs/10.1081/FRI-100100281#.UhUO4tK-2So
[3] http://pubs.acs.org/doi/abs/10.1021/jf062638l
[4] http://www.sciencedirect.com/science/article/pii/S0944711310003028
[5] http://online.liebertpub.com/doi/abs/10.1089/dna.2007.0631?journalCode=dna
Figure 1 is taken from wikimedia commons
Retrieved from " http://2013.igem.org/Team:Uppsala/saffron "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
