<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_IIT_Delhi_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:IIT Delhi/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM IIT Delhi</TITLE></P><DIV class="navbar "><BUTTON class="dropbtn2"><A href="https://2017.igem.org/Team:IIT_Delhi">iGEM IIT Delhi</A></BUTTON><DIV class="right_menu"><DIV class="dropdown"><BUTTON class="dropbtn">Project 
     							</BUTTON><DIV class="dropdown-content"><A href="/Team:IIT_Delhi/Description">Overview</A><A href="/Team:IIT_Delhi/Design">Squarewave Generator</A><A href="/Team:IIT_Delhi/InterLab">Interlab</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Results 
     							</BUTTON><DIV class="dropdown-content"><A href="/Team:IIT_Delhi/Circuit_Design">Circuit design and construction</A><A href="/Team:IIT_Delhi/Microfluidics">Microfluidics and Fluorescence</A><A href="/Team:IIT_Delhi/Photobleaching">Photobleaching</A><A href="/Team:IIT_Delhi/Promoter">Promoter strength</A><A href="/Team:IIT_Delhi/Oscillations">Oscillations</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Parts 
     							</BUTTON><DIV class="dropdown-content"><A href="/Team:IIT_Delhi/Basic_Part">Basic Parts</A><A href="/Team:IIT_Delhi/Composite_Part">Composite Parts</A><A href="/Team:IIT_Delhi/Improved_Part">Improved Parts</A><A href="/Team:IIT_Delhi/Part_Collection">Part Collection</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Modeling
                                                        </BUTTON><DIV class="dropdown-content-big"><A href="/Team:IIT_Delhi/Model">Overview</A><A href="/Team:IIT_Delhi/Write_Model">Writing a Model</A><A href="/Team:IIT_Delhi/Deterministic_Model">Deterministic Model </A><A href="/Team:IIT_Delhi/Stochastic_Model">Stochastic Model</A><A href="/Team:IIT_Delhi/Bifurcation">Bifurcation and Squareness</A><A href="/Team:IIT_Delhi/Resources">Resource sharing</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Human Practices 
     							</BUTTON><DIV class="dropdown-content-big"><A href="/Team:IIT_Delhi/HP/Silver">Human Practices</A><A href="/Team:IIT_Delhi/HP/Gold_Integrated">Integrated Practices</A><A href="/Team:IIT_Delhi/Engagement">Public Engagement</A><A href="/Team:IIT_Delhi/Collaborations">Collaborations</A><A href="/Team:IIT_Delhi/Safety">Safety</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Collaborations 
     							</BUTTON><DIV class="dropdown-content"><A href="/Team:IIT_Delhi/Collaborations">Overview</A><A href="/Team:IIT_Delhi/GMM_legislation">GMM Legislation</A><A href="/Team:IIT_Delhi/berlin">iGEM Berlin</A><A href="/Team:IIT_Delhi/mohali">Mentoring IISER Mohali</A><A href="/Team:IIT_Delhi/glasgow">iGEM Glasgow</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Notebook 
     							</BUTTON><DIV class="dropdown-content"><A href="/Team:IIT_Delhi/Notebook">Lab Records</A><A href="/Team:IIT_Delhi/protocol">Protocols</A><A href="/Team:IIT_Delhi/recipe">Recipes</A><A href="/Team:IIT_Delhi/medal">Medal Criteria</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Team 
     							</BUTTON><DIV class="dropdown-content"><A href="/Team:IIT_Delhi/Team">The Team</A><A href="/Team:IIT_Delhi/Attributions">Attributions</A></DIV></DIV></DIV></DIV><CENTER><DIV class=" panel_back bggg"><HEADER class="major"><H2 class="h2font">Introduction</H2><P>                                                                                                                                                                                                            </P><H2 id="pfont">Several circuits have been proposed, constructed, and implemented, leading to landmark discoveries in synthetic biology. These include systems such as the bi-stable toggle switch, and the repressilator, which brought about a paradigm shift in the field. Since then, several systems have been constructed to employ memory modules, create counters, adders, digital biosensors, and a whole wide range of other products. 

</H2><H6>Figure – A brief timeline of major notable events in the creation and development of synthetic biology (Source: Del Vecchio, Domitilla et al, Journal of The Royal Society Interface 13.120 (2016): 20160380.)</H6><H2 id="pfont">However, there are several limitations that still need to be overcome, as the field continues to make strides in every area. These involve the fact that biological systems have a lot of noises that cannot be modeled accurately to date, and the fact that metabolic burden is a major issue. Along these lines, one of the central issues is the distinct lack of digital responses in synthetic biology. 

	Thus, as described in the project overview, we wished to use the high cooperativity TetR homologs in such a manner so as to generate a square wave oscillator circuit. Such a system could have a whole multitude of applications; some of which were mentioned briefly in the overview, and the same are also discussed below,
 
</H2></HEADER><HEADER class="major"><H2 class="h2font">Applications</H2><P>                                                                                                                                        </P></HEADER><BUTTON class="accordion back1" style="font-weight: bold;">Generating a clock input to time biological events</BUTTON><DIV class="panel "><DIV align="left" style="font-size: 84%;" class="padding1"><LEFT><H2 id="pfont1">The potential of genetic clock lies in its role to triggering logic reaction for sequential biological circuits. A square wave generator could be used as a genetic clock, since square waves lie at the heart of clocks. Further, these clocks could be used in any cellular system to time particular events. 
</H2></LEFT></DIV></DIV><BUTTON class="accordion back2" style="font-weight: bold;">Characterising gene regulatory network nodes through impulses</BUTTON><DIV class="panel "><DIV align="left" style="font-size: 84%;" class="padding1"><LEFT><H2 id="pfont1">This could be used to study correlation between two genes, by coupling one of the genes to the oscillator, then observe the dynamics of the second gene. In this manner, the effect of one gene on the others could be studied.
</H2></LEFT></DIV></DIV><BUTTON class="accordion back3" style="font-weight: bold;">Periodic Drug delivery</BUTTON><DIV class="panel "><DIV align="left" style="font-size: 84%;" class="padding1"><LEFT><H2 id="pfont1">Like diabetic patients, insulin needs to be provided externally through injections or other means. Our oscillator system could be used as a patch containing bacteria that are oscillating to produce square levels of insulin to the patient once. This then would deliver insulin automatically at intervals, guided by our system .Further when the requirement for insulin would be high, the amount of insulin being delivered to them could be changed through some source which could dive a change in frequency of the oscillations (future applications could be focused on engineering frequency modulation).
</H2></LEFT></DIV></DIV><BUTTON class="accordion back4" style="font-weight: bold;">Metabolic Switching</BUTTON><DIV class="panel "><DIV align="left" style="font-size: 84%;" class="padding1"><LEFT><H2 id="pfont1">A bacterial species could be engineered to produce levels of the permease for a particular sugar in the form of a square wave. Thus, at varying intervals of time, the permease (say lac permease) would be expressed, which would cause the bacteria to start metabolizing lactose. When it goes off, the bacteria would not express the lac permease and consume glucose. In this manner, we could tune the frequency of the oscillations to ensure metabolic switching and activation of pathway shunts in the manner that we want.
</H2></LEFT></DIV></DIV><BUTTON class="accordion back1" style="font-weight: bold;">Temporal Barcodes</BUTTON><DIV class="panel "><DIV align="left" style="font-size: 84%;" class="padding1"><LEFT><H2 id="pfont1">If we can control the frequency of the oscillations and modulate the time spent by the wave in the ON and OFF state (which is basically a function of the frequency itself), we could generate a combination of 0’s and 1’s, in order to generate a bar code. This bar code would be read in time, and therefore would be a temporal bar code. A typical example of this would be to encode the word “iGEM” by 11001001, where 1 represents an ON state for say, 20 minutes. Thus, an 11 response, which represents the letter i, would then be read if the fluorescence stays on for 40 minutes. In this fashion, our device could be used for encryption of data. 
</H2></LEFT></DIV></DIV><BUTTON class="accordion back2" style="font-weight: bold;">For memory storage</BUTTON><DIV class="panel "><DIV align="left" style="font-size: 84%;" class="padding1"><LEFT><H2 id="pfont1">The system could be used as the bacterial analogue of memory storage. The base of the oscillations could be called the 0 bit, and the high point could be called the 1 bit, and these combinations of 0 and 1 bits could be used to store short term memory in biological cultures, performing the functions that the RAM (random access memory) does in computers.
</H2></LEFT></DIV></DIV><HEADER class="major"><H2 class="h2font">From Sinusoids to Square Waves</H2><P>                                                                                                                                                                                                            </P><H2 id="pfont">Sinusoidal waves have been generated and characterized in several circuits employing various topologies so far, including single cell oscillators, small colony synchronized quorum sensing based oscillators, wide range synchronized oscillators (that synchronize over different channels inside a microfluidic chamber), and even mammalian oscillators. Thus, we used these as a starting point in our project, and thought of ways to transform this sinusoid, so as to generate a square wave. 
	After brainstorming, basic modeling and simulations, and discussing several possible topologies (which have been linked below), we finally found the 5 node ring topology with our design modifications to be the best solution to the problem.
 


The basic topology for the system was inspired by the 3 node repressilator, which basically had repressors arranged in a cyclic fashion, where one represses the second, the second represses the third, and the third represses the first. The same schematic, with an even number addition to the number of nodes can generate oscillations, as has been shown by several groups in the literature. Thus, we employed a novel 5 node system in the same fashion, with our design modifications, in order to get the square waves to work.
	For this transformation from “y = sin (t)” to “y = square(t)” (as MATLAB calls it), the major strategies for conversion, and how our 5 node oscillator employs these to generate square waves have been outlined below.
 </H2></HEADER><HEADER class="major"><H2 class="h2font">Graded to Digital Response – High Cooperativity</H2><P>                                                                                                                                                                                                            </P><H2 id="pfont">	The first issue that we identified was that sine waves go up and come down in a slow fashion; slower than what we would want in our square waves. Thus, the first thing to do was to increase the rate of the response, by employing a mechanism wherein the repressors act quickly, and only after a certain point. This would make rise and fall times faster.
As mentioned in the earlier section as well, cooperativity is the property of a repressor that represents the number of molecules of the repressor that need to combine and polymerize before they can repress the promoter of their corresponding gene. Repressor cooperativities bring down the rate of production of the protein in a multiplicative fashion via the hill function, which can be represented as (P represents protein concentration, n represents cooperativity) – 


And again, as the cooperativity of the repressor changes, the level of the protein changes as follows – 
	Therefore, the high cooperativity promotes faster degradation. This fact was exploited by us for this purpose, employing repressors with the highest cooperativity from the 73 TetR homologs that were reported in Voigt’s paper (see references below). Thus, we were able to take care of the rise and fall times using these high cooperativity TetR homologs. The cooperativities and names of the repressors used in our system are as follows – 
 <CENTER><TABLE style="width:50%" class="table1"><TBODY><TR><TH>Node</TH><TH>Repressor Used</TH><TH>Cooperativity Reported</TH></TR><TR><TD>A</TD><TD>Orf2</TD><TD>6.3</TD></TR><TR><TD>B</TD><TD>TetR</TD><TD>3</TD></TR><TR><TD>C</TD><TD>SrpR</TD><TD>3.2</TD></TR><TR><TD>D</TD><TD>PhlF</TD><TD>4.5</TD></TR><TR><TD>E</TD><TD>Bm3RI</TD><TD>4.5</TD></TR></TBODY></TABLE></CENTER></H2></HEADER><HEADER class="major"><H2 class="h2font">Holding the Constant ON State – Time Delay</H2><P>                                                                                                                                                                                                            </P><H2 id="pfont">	Once we were ensured that the rates would be faster, we next turned to holding the constant ON and OFF states, a property that square wave oscillators exhibit, and sinusoidal ones don’t. 
	In order to maintain its state once it is high, we required a time delay between when the level of a node rises and when it falls. This is taken care of by our square wave oscillator by having the two extra repressor nodes. Basically, the two nodes do not qualitatively change the state of the oscillations, since if A has to repress D, then now in our circuit, A represses B, which in turn now cannot repress C, and therefore C is produced, which represses D. Therefore, the effect remains the same, but there is a delay introduced. This is shown diagrammatically below – 
</H2></HEADER><HEADER class="major"><H2 class="h2font">Noise Reduction – Low Copy Reporter and Removal of Degradation Tags</H2><P>                                                                                                                                                                                                            </P><H2 id="pfont">	Having catered to the issues of rise time and fall time, as well as holding the ON and OFF states for a long time similar to how square waves behave, we then also wanted to have the oscillator be noise free. Noise in biological systems can hamper the oscillations by driving the cells out of phase due to copy number variations of the plasmid, leaky expression, and other factors. 
	How our square wave oscillator caters to the noise is as follows – 

<OL><LI>High cooperativity repressors – High cooperativity ensures a near digital response, and therefore lower concentrations of the repressor would not be able to repress their corresponding node well enough, due to their requirement to dimerize in high numbers.

</LI><LI>Low copy reporter – Demonstrated by Paulsson et al, one of the main sources of noise in the repressilator was that the reporter was placed in a high copy number plasmid. Such high copy numbers (containing pUC19 or pMB1 origins of replication) can have large cell to cell variations in copy number, thereby reporting in a faulty, noisy manner. Therefore, as was done by their group, our square wave oscillator contains the reporter on a low copy backbone (p15A ori). 

</LI><LI>Removing degradation Tags – Paulsson et al, in the same paper, reported that the ssrA degradation tags that were used at the end of the repressor genes in the repressilator also employed machinery that was noisy. Removing these tags from his system brought down the variance significantly. Our square wave oscillator employs the same strategy for noise reduction. 
</LI></OL></H2></HEADER><HEADER class="major"><H2 class="h2font">References</H2><P>                                                                                                                                                                                                            </P><H2 id="pfont"><OL><LI>Gardner, Timothy S., Charles R. Cantor, and James J. Collins. &quot;Construction of a genetic toggle switch in Escherichia coli.&quot; Nature 403.6767 (2000): 339-342.
</LI><LI>	Elowitz, Michael B., and Stanislas Leibler. &quot;A synthetic oscillatory network of transcriptional regulators.&quot; Nature 403.6767 (2000): 335-338.
</LI><LI>Niederholtmeyer, Henrike, et al. &quot;Rapid cell-free forward engineering of novel genetic ring oscillators.&quot; Elife 4 (2015): e09771.
</LI><LI>Stanton, Brynne C., et al. &quot;Genomic mining of prokaryotic repressors for orthogonal logic gates.&quot; Nature chemical biology 10.2 (2014): 99-105.
</LI><LI>Potvin-Trottier, Laurent, et al. &quot;Synchronous long-term oscillations in a synthetic gene circuit.&quot; Nature538.7626 (2016): 514-517.
</LI><LI>Danino, Tal, et al. &quot;A synchronized quorum of genetic clocks.&quot; Nature463.7279 (2010): 326.
</LI><LI>Slomovic, Shimyn, Keith Pardee, and James J. Collins. &quot;Synthetic biology devices for in vitro and in vivo diagnostics.&quot; Proceedings of the National Academy of Sciences 112.47 (2015): 14429-14435.
</LI><LI>Callura, Jarred M., Charles R. Cantor, and James J. Collins. &quot;Genetic switchboard for synthetic biology applications.&quot; Proceedings of the National Academy of Sciences 109.15 (2012): 5850-5855.
</LI><LI>Weber, Wilfried, and Martin Fussenegger. &quot;Emerging biomedical applications of synthetic biology.&quot; Nature reviews. Genetics 13.1 (2012): 21.
</LI><LI>Xie, Zhen, et al. &quot;Multi-input RNAi-based logic circuit for identification of specific cancer cells.&quot; Science 333.6047 (2011): 1307-1311.
</LI></OL></H2></HEADER></DIV></CENTER></DIV><FOOTER id="footer"><TABLE style="font-family:'Lato'; border: 0px solid black; "><TBODY><TR align="center"><TD align="center" width="50%" colspan="2" class="tr2"><B>Sponsored By</B></TD></TR><TR align="right"><TD align="center" width="50%"><B>Contact Us</B></TD><TD align="center"><B>Address</B></TD></TR><TR align="center"><TD align="center" style="padding-top: 0px;" class="tr2"><CENTER><P style="text-align: center; color:white; font-size: 80%;"><B>E-mail:</B> iitd.igem@gmail.comUndergraduate LaboratoryDepartment of Biotechnology and Biochemical Engineering, IIT Delhi</P></CENTER></TD></TR></TBODY></TABLE></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>