"
Page
Discussion
View source
History
teams
Log in
 
Team:BrownTwo/Notebook
From 2008.igem.org
Home
Team
About Us The Duo Advisors Acknowledgements
The Limiter
Introduction Utility Our Network Alternative Design Modeling
Implementation
Synthetic Transcription Factors Advantages of Yeast Construction Testing
Protocols
Moving Biobrick Parts onto Yeast Chromosomes Making Ultracompetent Cells
Parts
Lab Notebook
6/9/09
Made 500 mL LB liquid media as well as 1 L LB + agar for plates.  Put approximately 20g each of LB Broth and Bacto Agar into the agar mixture, but it came out cloudy so we had to get rid of it.  For the autoclave we use, seeing as it is so old, we should now heat the agar mixture briefly to a boil on a hotplate first and then leave it in the autoclave for 25 mins.  The LB media came out just fine.
6/10/08
Re-made LB-agar plates according to method described above.  They came out rather successfully.  Good consistency.
With these plates, we then streaked out glycerol stocks of J63001 (EYFP) and J63002 (terminator) onto plates for growth overnight.
However, after realizing that J63006 had never been made into a glycerol stock, we attempted to transform cells with J63006 from the 2007 parts kit.  We decided against using the same part from the new 2008 kit because we would have then had to purchase Tris-HCl and make TE, which would have meant more money and more time.  Positive control for the transformation was a 3x flag provided by Diana.
We plated this transformation as well and put all of the plates into the incubator at around 11:30 pm.  Since we were unsure about how successful we would be, we made duplicate plates for each part. Our streaking technique was a little unique, but I had found it to work in the past.  You take out a dab of glycerol and add it to 1 mL of LB in a microentrifuge tube.  After mixing it uniformly, you then take a loop and sterilize it by getting it red hot.  After first cooling the loop in the agar of the plate, you then dip the loop into the liquid and get just a drop.  You apply this drop to the surface of the agar and streak as if you would streak a solid, moving back and forth horizontally while gradually bringing the loop down.  I've found this method to work especially well when you believe that the cells have a fairly high concentration, such as in a glycerol stock or in an overnight culture.
Note with regard to the transformation that after the 2 hour incubation we were forced to put it in the fridge for about 4.5 hours since other obligations prevented us from finishing the protocol.  Normally, we would have plated the transformations immediately after they came out of the incubator.  Since we had refridgerated them, we stuck the transformed cells into the incubator for about half an hour, but it is possible that all of the DNA had been ejected due to the cooling.  We also plated these cells as described instead, rather than adding all of the liquid to the plates.  We saved the remaining liquid in case we needed it the following day.
6/11/08
Came in to discover wonderful colonies on all plates except J63006.  After putting J63006 in the incubator for several more hours, exactly one colony was discovered later on and used for an overnight culture.
Attempted another transformation with the remaining plasmid DNA from the 2007 parts kit (that we had placed in the fridge yesterday).  Put the plates into the incubator at roughly 7:15 pm, including one positive control (3x flag), one negative control, one transformation from today, and another containing cells from yesterday's transformation that had been in the incubator for the majority of the day.
6/12/08
Came in around 11 am to take tubes out of incubator.  Total time in incubator was 15hr and 48 min.  Found substantial growth in all three test tubes.  Also, took out plates around 11:17 to discover good colonies on both the positive control and the transformation that I had done yesterday.  There were only about a couple dozen colonies on the J63006 plate, so it is apparent that one should use all of the liquid from the transformation steps when plating.  However, the positive control had No growth on the transformation from 6/10, but this was to be expected since the cells had been refridgerated for an entire night and were likely dead or had lost their plasmids.
Made two glycerol stocks of J63006, since it was not glycerol stocked.  Note that both are from the same colony.
Conducted miniprep using the overnight cultures.  Spun down 1.9 mL of overall culture in two rounds of 1 mL and 0.9 mL, each 10 min.  In the future we should try to use more volume, like 5 mL.  Used 60 uL of EB.
End result of the miniprep:
J63001 - 37.1 ng/uL in ~58uL
J63002 - 29.5 ng/uL in ~58uL
J63006 - 36.5 ng/uL in ~58uL
Ran 50uL restriction digest.  In each tube we put 5uL BSA, 5uL multicore buffer, 10 uL H2O, and 27uL DNA and 1.5uL each of two enzymes.  For J63001, this turned out to be 1001.7ng of DNA and we used EcoRI and SpeI.  For J63002, this turned out to be 796.5ng of DNA and we used EcoRI and XbaI.
Stuck in incubator for 2 hours.  After incubation, put in heating block at 70 degrees Celsius for 10 minutes to kill the enzymes.  Then stuck them in the freezer.
6/13/08
Discovered that our parts from Caroline were on the way.  As such, we decided to make a lot of plates in preparation for their arrival.
First batch:
BactoAgar 10.0012g + 10.0021g = 20.0033g
LB Broth 10.0345g +  10.0144g = 20.1785g
Added 0.1008 g of AMP to mixture
Put into plates (make sure mixture is cool)
Note:  The agar was slightly burned when stirring on the hot plate.  This was primarily due to using too much heat to boil the agar and also due to stopping the stirring within the flask.  Will have to be more careful next time.
Second batch:
BactoAgar 10.0190g
LB Broth  10.0113g
Added 0.0502 g of AMP to mixture
Put into plates (make sure mixture is cool)
We also ran a gel purification of the restriction digests of J63001 and J63002 in preparation for the ligation.
Gel Mixture Protocol for 1% Agarose gel:
Agarose:  0.5012 g
0.5xTBE with SYBR Safe:  50 mL
Microwaved for 1 minute on high.
Ran gel in previously used TBE 0.5x buffer.  Used all 50uL of each restriction digest by splitting up this amount among 3 wells apiece with roughly 18uL per well.  Also, ran with both a 100 bp ladder and a 1 kb ladder.  This is because one hopes to extract the excised part J63001, which is 744 bp long, as well as the cloning vector contaning J63002, which should be around 2292 bp.  When the blue dye had gone halfway down the length of the gel, removed it and placed it on the transilluminator.  Visualization at this time showed that the 1 kb ladder did not appear to work, as nothing appeared up in that lane.  The 100 bp ladder lane had visible dye, but it seemed too clumped together to get a good measurement.  In the lanes with J63001, there were two bands, as one might expect. The higher band would be the smaller piece, meaning the actual part J63001, while the lower band would be the plasmid from which this part was removed.  In the lanes with J63002, there was only one visible band, since only a small portion was removed during the restriction digest (no more than 10 bp).
Unfortunately, made the bad decision of putting the gel back into electrophoresis to help separate the bands.  This only appeared to wash all of the DNA from the gel, making it basically impossible to extract any at all.  Looks like the restriction digest will have to be conducted again using some more of the miniprep DNA.
The mixtures for the restriction digest were the same as those listed for yesterday, noting, however that there was slightly less than 27uL left of each plasmid.
Incubated for , then placed on heating block at 70 degrees Celsius for 10 minutes before putting them in fridge.
Parts from Caroline actually came in today, so we decided to attempt transformations.  Note that the parts were estimated to be at 200ng/uL in 10mM Tris-Cl, pH 8.5.  At first we were going to do transformations of all of them, but we quickly realized that, although the sheet indicated that there was 10uL of plamid in each tube, there actually only appeared to be about 2uL.  Our guess is that some of each plasmid dried up during shipment.  Therefore, we decided instead to attempt a trial transformation with a part that we already had in glycerol stock, P0501 or J63006, to use Caroline's naming system and the registry's, respectively.  Placed only about 2 uL in with 50uL competent cells, but put the whole 4 uL in for the positive control.  Only realized after the two hours of incubation that the volume for the positive control should have matched that for the part P0501.  Will have to redo this experiment should transformation fail.  Incubated for 2 hr and 2 min, then plated all of the liquid onto two separate plates.  For each tube, used 20% of the liquid on one plate and 80% on the other.  Placed in incubator at 10:14 pm.  Will come in tomorrow around 12 pm to check on them.
Note: For the transformation, used the plates that had been slightly burnt.
6/14/08
Came in around 1:30 pm, after the plates had been in the incubator for 15 hr and 23 min.  Discovered good colonies on P0501 as well as the positive control, with no colonies on the negative control.  For some reason, it appears that the incubator had been set to 25 degrees Celsisus, so the colonies were still quite small.  Set the incubator to 37 degrees and put them back in for several hours.  Took them out again after the timer had said 19hr and 34 min.  Will use two colonies from P0501 to make glycerol stocks and store the plates in the fridge.
Re-ran gel for the new restriction digests.
Agarose: 0.5007g
TBE + SYBR Safe: ~50uL
Note that agarose gel mixture should be clear before pouring into the mold.
Used 100bp and 1kbp ladders from NEB, called "Quick Load," which appeared to already have loading dye.  Applied 15uL of each in the wells.  Also added 5.6uL of 10x loading dye to each tube, which contained around 50uL of restriction digest.  This gave a final volume of 55.6uL, of which approximately 10% was loading dye.  Decided to only use 2/3 of each tube, so only applied two wells of 18uL apiece for each, leaving about 18uL left in each.  Voltage setting was 104 volts.
Gel extraction:
Note that one must add 300uL of QG for every 100mg of gel.
J63001 tube #1:
Starting weight = 1.0304g
End weight = 1.1014g
=> Weight of gel = 0.071g
=> used 213uL of QG
J63001 tube #2:
Starting weight = 1.0060g
End weight = 1.0518g
=> Weight of gel = 0.0458g
=> used 137.4uL of QG
Combined volume: 350.4uL
J63002 tube #1:
Starting weight = 1.0047g
End weight = 1.1069g
=> Weight of gel = 0.1022g
=> used 306.6uL of QG
J63002 tube #2:
Starting weight = 1.0257g
End weight = 1.1080g
=> Weight of gel = 0.0823g
=> used 246.9uL of QG
Combined volume: 553.5uL
After dissolving the gel in QG, the two tubes for each part were combined into a single column to be spun down.  Note that 50uL of EB was used to elute each column.  The results of the extraction were placed in the freezer at -20 degrees Celsius.
In addition to the extraction for these two parts, also started two liquid cultures, each from a separate colony on the P0501 plate.  Placed these cultures inthe incubator around 8:45 pm.  Should take them out around 10:45 am tomorrow for making glycerol stocks.
6/16/08
No one took the cultures out of the incubator yesterday.  Glycerol stocks were not made.
A conversation with Caroline served to be quite informative.  It appears that it will be better to simply test the activity of the PGAL1 and PMET25 promoters while they are driving the production of their respective activators or repressors.  Promoter activity can change depending upon what the coding sequence is.  However, will still continue on with the other ligation process (between J63001 and J63002) for practice.  To this end, the next step is application of Calf Intestinal Alkaline Phosphatase (CIAP) to the vector only, preventing ligation to itself.  After this, one does an ethanol precipitation before doing the ligation itself. The protocol that I have in my lab notebook for CIAP indicates adding 4uL of CIAP 10x Reaction Buffer and 1 uL of CIAP enzyme to 34 uL of the gel extraction.  Promega's protocol, however, requires diluting the CIAP to 0.01u/uL.  The CIAP we have in the fridge is at a concentration of 1u/uL.  They then recommend using up to 5uL of this diluted CIAP in 50uL of reaction, to yield a final concentration of 0.001u/uL.  Also in Promega's protocol is the use of 40 uL of DNA (up to 10 pmol of 5' ends, where 1μg of 1,000bp DNA = 1.52pmol DNA = 3.03pmol of ends) and 5 uL of 10x Reaction Buffer.
Concentration of J63001 gel extract: 8.4 ng/uL => 40 uL contains 336 ng  (note: don't need to do CIAP on this sample)
Concentration of J63002 gel extract: 13.7 ng/uL => 40 uL contains 548 ng
0.05u CIAP/10 pmol 5' ends => 0.005u CIAP per pmol 5' ends
So we only need about 0.025u CIAP for 40 uL of J63002.
After some consideration (see above), decided to follow a modification of this protocol.  To the reaction mixture I will add 5 uL 10x buffer, 44.5 uL gel extract and 0.5 uL CIAP.  Note that final concentration of CIAP was 0.01u/uL.
Incubated this reaction for 31 mins at 37 degrees C.  Added another 0.5uL of CIAP and then put into incubator again for ~32 mins.  Put on heating block at 70 degrees C for 10 minutes to denature the enzyme activity.
After this was completed, went straight into the ethanol precipitation.  It was quite evident that this was necessary due to the low concentration of DNA in both tubes (J63001 and J63002).
Need to make 3 M sodium acetate for the ethanol precipitation.
MW of sodium acetate trihydrate = 136.08 g/mol
For 10mL, this translates into 3 M * 0.010 L = 0.030 mol
0.030 mol * 136.08 g/ mol =  4.0824 g sodium acetate trihydrate dissolved in 10 mL water
Weighed out 4.0826g sodium acetate trihydrate and dissolved in 10 mL of solution.
To each tube, which contained roughly 50 uL, added 3 volumes of 100% ethanol (150uL) as well as 20uL of 3M sodium acetate (which was 1/10 the volume of the DNA and the ethanol combined).  Also added 1 uL of Glyco Blue.  Incubated in -80 degree C freezer for 1 hour, then spun in centrifuge that had been placed in the fridge (since a cold centrifuge was specified).  Spun first tube at 13000 rpm, which was possibly too much.  Spun second tube at 8000 rpm.  After centrifuging, removed the supernatant, allowed the tubes to air dry, and then resuspended in 20 uL of distilled water.
After the ethanol precipitation, it was discovered that the new concentrations were as follows:
J63001: 17.6 ng/uL
J63002: 15.5 ng/uL
Clearly, there was not much improvement for either.  Also note that now there is only 18 uL left, since 2 uL was used to do the ligation.
Also decided to transform some of Caroline's parts tonight, seeing as P0501 had been so successful.  It turns out that some of the plasmid solution had stuck to the caps inside the tubes containing them, so that by pounding the tube against the table it was possible to accumulate at least 2 uL of liquid in most cases.  At first, began transformation of all 38 parts, but realized that this was impossible given the amount of plates available in the fridge (50 plates).  Since it was necessary to use two plates per transformation as a safeguard, this would allowfor 23 parts, including two for a positive control and two for a negative control.  Thus, decided to transform the first 20 parts on the list sent by Caroline, which correspond to parts that we hope to use for our tests.  These parts are:
pRS304*: TRP shuttle vector
pRS305: LEU shuttle vector
pRS306: URA3 shuttle vector
pRS305_Z0267: Zif268-HIV activator
pRS306_Z0637: Zif268-HIV activable reporter
pRS305_Z0277: Gli1 activator
pRS306_Z0739: Gli1 activable reporter
pRS305_Z0279: LexA activator
V0002_Z0842: LexA activable reporter
pRS304*_Z0411: null repressor
pRS304*_Z0415: Gli1 repressor
pRS306X_Z0510: Gli1 repressor reporter
pRS304*_Z0421: Zif268-HIV repressor
V0002_Z0507: Zif268-HIV repressor reporter
pRS304*_Z0412: LexA repressor
pRS306_Z0418: LexA repressible reporter
pRS304*_Z0416: YY1 repressor
pRS306_Z0511: YY1 repressible reporter
V0002_P0501: pGAL1 + Kozak
V0120_P0560: pMET25
Unfortunately, a slew of competent cells were put into tubes for the remaining 19 parts, but went unused tonight.  After being in an ice bath for about an hour and a half they were put back into the -80 in their respective tubes.  Hopefully they are still viable.  A test of their viability using a simply positive control (3x flag plasmid) should have occurred tonight, but can be done tomorrow instead.  The rest of the cells were all in the ice bath for 30-40 minutes.  They were heat shocked in pairs, starting with the first to have their plasmids added and progressing towards those whose plasmids had been added last.  The heat shock was placement in a water bath at roughly 45 degrees C for 40-50 seconds.  After heating, each pair was placed in ice bath.  After three heat shockings, the first ones to be heat shocked had been in the ice bath for an estimated 2-5 minutes, so LB was added to them and they were placed in a tube holder at room temperature.  After LB had been added to all tubes, they were placed in the incubator for 2 hours.  In this fashion, it was possible to adhere rather closely to the transformation protocol despite the larger numbers of transformations.  The cells were plated as described previously and placed in the incubator around 11:50 pm.  Should check up on them around 1:50 pm tomorrow.
6/17/08
Met with Suzanne today.  Spent some time catching her up on the system and began to discuss ways to observe individual protein concentrations and to further model and analyze the system.
Took plates out of incubator around 2:35 pm.  They had been in the incubator for 14 hr and 42 min.  There were good colonies on all plates except Z0279 and Z0842, which were placed back into the incubator along with the controls to make sure that colonies could grow if they were present.  Note that these two parts are the LexA activator and the LexA activator reporter, respectively.
Began ligation between J63001 and J63002.  For ligation, use 1:3 ratio of vector to insert.  This should be in relative amounts, which corresponds to the weight of DNA present in the volume (in ng) normalized by the length of each DNA strand.  Prefer to have more insert than vector.  Thus, try using 8 uL water, 1 uL vector, 3 uL insert, 1.5 uL 10x ligase buffer, and 1.5 uL T4 DNA ligase.  Decrease water if you use higher volumes of insert and vector.  Maintain total volume of 15 uL.
1 uL of J63001 contains 15.5 ng while J63002 contains 17.6 ng.  Note that J63001 has a length of 744 bp, while J63002 has a length of around 2292 bp.  Thus, the normalized amounts of J63001 and J63002 per uL are 17.6 ng/744 = 0.023656 and 15.5 ng/2292 = 0.006763.  Thus, a ratio of 0.023656/0.006763 =3.498 natrually exists when one uses the same volume of insert and vector.  If one uses a volume ratio of 3:1 between insert and vector, the ratio of DNA amounts will end up being 10.494, which is even better.
Used the volumes listed above for two separate yet identical ligation reactions, which began around 4:25 pm at room temperature.  Will try to let them run for a full 24 hours.
Inoculated liquid cultures with two from each part colonies except those that did not have colonies on them (Z0842 and Z0279).  Also made cultures for P0501 since the glycerol stocks were not made for this over the weekend.  Placed them in the incubator around 11:50 pm.  Will check on them again at 1:50 pm tomorrow.
6/18/08
Decided to make some more LB agar + AMP plates for the rest of the transformations.
First batch:
LB Broth (Lennox): 20.0163g
Bacto agar: 20.0038g
Amp: 0.1009g
Second batch:
LB Broth (Lennox): 20.0041g
Bacto agar: 20.0030g
Amp: 0.1053g
Note: New technique for making agar plates involves placing flasks that have just come out of the autoclave into a water bath at 55 degrees Celsius without a stir bar.  This will help to cool down the flasks as well as to maintain them at a temperature that is good for pouring and also for adding antibiotic.
The second batch was left in the water bath at around 60 degrees for about eight hours and had turned significantly darker.  Poured plates with this anyway, but they may be unusable.  Another reason why these plates may be unusable is that the agar was around 60 degrees when antibiotic was added, meaning that they may not actually have active AMP within them.
Also, almost all plates from both batches had bubbles in them, requiring application of flame.  However, lacking a good flame, we were forced to heat up a loop and attempt to remove bubbles using this.  The result was that some bubbles remained on the plates and that the agar was burnt in some places.
Took cultures out of incubator and made glycerol stocks of the following:
Z0421 C#1,2
Z0507 C#1,2
Z0416 C#1
Z0510 C#1,2
Z0415 C#1,2
Z0411 C#1,2
Z0739 C#1,2
Z0277 C#1,2
pRS306 C#1,2
Z0267 C#2
Z0637 C#1,2
pRS304* C#1,2
pRS305 C#1,2
Ran out of cryotubes, so reordered them and remade cultures of the following parts to stock tomorrow:
Z0412 C#1,2
P0501 C#1,2
Z0267 C#1
Z0418 C#1,2
Z0511 C#1,2
Z0416 C#2
P0560 C#1,2
Note that 'C' above stands for 'colony'.
Innoculated 2mL of LB + Amp in each tube.  Put them in the incubator at 11:35 PM and restarted the timer on the incubator.  Put the plates back in the fridge.
Placed ligation between J63001 and J63002 on heat block at 70 degrees for about 10 minutes around 12:50 am on 6/19/08 then placed in fridge for transformation tomorrow.
6/19/08
Picked up cryotubes and dropout media from the stockroom.
transformed ligations, all with 4uL DNA into 50uL competent cells:
Ligation #1 into new competent cells
Ligation #2 into new competent cells
+ control of 3x flag vector into new cells
+ control of 3x flag vector into a relabeled tube of competent cells from the upper shelf in the -80 freezer
- control of nothing into a relabeled tube
The reason for doing two positive controls is that the second one is determining the effectiveness of the competent cells that had been thawed on 6/16/08 and placed back into the -80 freezer after it was determined an hour or so later that they were not going to be used.  If these competant cells are still viable, they will be used to transform the remainder of the limiter parts.
Put cells and DNA on ice at 2:05 PM
Heat shocked for 60 sec at 2:35 PM, then put cells on ice for 2 min.
200 uL of LB added to each culture, and cultures placed in incubator at 2:45 PM
Should plate cells at 4:45 PM.  Indeed, took them out at 4:45 pm.  Plated using the 20-80 volume technique.
All transformations plated onto older plates except ligation #2 at 80%, which went on one of new plates with bubbles
All controls were plated with 100% of volume rather than splitting into 80% and 20%
Plates went into incubator around 10:45 pm.  Should come out again around 12:45 pm tomorrow.
Also re-attempted transformations with the LexA DNA, which corresponds to parts Z0279 and Z0842.  These parts were the only two to not have successful transformations during the first round of limiter part transformations on 6/16/08.  Spun down tubes with remaining plasmid DNA in centrifuge in an attempt to get enough plasmid for the transformation.  Took only 1 uL for each part and mixed with 50 uL competent cells.  Also included negative control and two positve controls.  Both positive controls were 3x flag, but one was subjected to 1 min of mircowave at the lowest setting immediately following the 40-60 second heat shock.  Note that the positive control that went through the microwave started with less than 50 uL of competent cells, though the actual starting volume was not determined.  Transformations were in the incubator for 2 hr and 1 min.  The cells for each part were plated on two separate plates according to 20-80 volume technique while all controls were simply plated on a single plate with 100% of the transformation volume.  The plates were placed in the incubator at 2:46 am on 6/20/08.  Should be removed at 4:46 pm tomorrow.
Have also decided to miniprep the following parts tomorrow in preparation for initial construction steps:
pRS305_Z0267
pRS306_Z0637
pRS305_Z0277
pRS306_Z0739
pRS304*_Z0411
pRS304*_Z0415
pRS306X_Z0510
pRS304*_Z0421
V0002_Z0507
pRS304*_Z0412
pRS306_Z0418
pRS304*_Z0416
pRS306_Z0511
Picked colonies from plates that were in fridge (as opposed to using glycerol stocks) and inoculated 7 mL of culture (LB + amp) for each.  Will try placing them in shaking incubator at an angle to enhance the amount of aeration that occurs.  Placed in incubator around 3:30 am on 6/20/08.  Will check on the cultures again at 5:30 pm.
Will streak out some yeast cells onto plates tomorrow in preparation for transformations into yeast next week.  This will require making yeast media plates tomorrow.
6/20/08
Need to make media/plates today.
When thinking about making simple tests of all of the full parts that we received, one should consider which dropouts are necessary for construction.  Note that all activators are on pRS305, which is a LEU2 shuttle vector.  All repressors are on pRS304*, which is a TRP1 shuttle vector.  Finally, all reporters are found on the pRS306 plasmid, which is a URA3 shuttle vector, with the exception of the LexA activable reporter and the Zif268-HIV repressor reporter, both of which are on the V0002 vector for some reason.  This means that to make simple tests of our parts, we'll have to primarily make plates and media with the following pairs of dropouts: LEU and URA, TRP and URA.  This also means that it should be relatively simple to construct Test#2 by using three different plasmids.  However, we will need to construct Test #2 so that it conforms with our design for the final device.  This means that we might have to attempt putting multiple parts on the same plasmid.
For YPD plates:  made 250 mL of YPD Agar for growing yeast strains
YPD mix: 12.5003g
Bacto Agar: 5.0021g
Took transformations of ligation between J63001 and J63002 out of incubator roughly 14hr and 15 min after it had been put in.   No colonies visible.  It is more than likely that the UV light from the two transilluminators used with the gel mutated the DNA such as to inhibit ligation.  This UV light was UV-B.  One should note that of the two positive controls, the one incorporating cells that had been thawed for about one hour still had colonies, although at a significantly lower density (about a dozen colonies as compared to more than one hundred in the other positive control).  Should take this into consideration when transforming the remaining parts tonight, which means being frugal with the plasmid DNA (using 1.5 uL, for example).  This might also simply mean sticking to newer competent cells.
Took transformations of Z0842 and Z0279 out of incubator after 14 hr and 10 min.  Found that there were colonies on these plates.  Also note a significant amount of colonies on the plate containing the 3x flag that was microwaved.  However, there were no colonies on the positive control that did not go through the microwave, indicating that something went wrong (e.g. perhaps the two were switched).  Further investigation seems to indicate that both positive controls were accidentally spread onto the same plate.  Evidence for this is that there are two distinct circules on the agar that indicate where the transformation liquid was applied onto the plate before it was spread.   In any case, will inoculate cultures with the colonies on the plates containing the LexA parts later tonight.  Caution concerning this mistake:  DON'T WORK LATE AT NIGHT.
Miniprep
Took cultures out of incubator after 14 hr and 45 min.  Put in large centrifuge for 10 mins on high at speed setting of 8, which corresponds to roughly 2100 rpm.
Note: Used 60 uL of EB to elute.
Z0267: ~58 uL of 135.8 ng/uL; 260/280 = 1.92; 260/230 = 2.17;
Z0637: ~56 uL of 123.4 ng/uL; 260/280 = 1.89; 260/230 = 1.96;
Z0277: ~58 uL of 123.2 ng/uL; 260/280 = 1.91; 260/230 = 2.17;
Z0739: ~58 uL of 108.4 ng/uL; 260/280 = 1.92; 260/230 = 1.88;
Z0415: ~58 uL of 139.4 ng/uL; 260/280 = 1.93; 260/230 = 2.18;
Z0411: ~58 uL of 162.8 ng/uL; 260/280 = 1.90; 260/230 = 2.13;
Z0510: ~56 uL of 67.1 ng/uL; 260/280 = 1.96; 260/230 = 2.03;
Z0421: ~58 uL of 171.7 ng/uL; 260/280 = 1.93; 260/230 = 2.17;
Z0507: ~58 uL of 102.7 ng/uL; 260/280 = 1.91; 260/230 = 2.12;
Z0412: ~58 uL of 199.8 ng/uL; 260/280 = 1.92; 260/230 = 2.15;
Z0418: ~58 uL of 53.8 ng/uL; 260/280 = 1.93; 260/230 = 1.94;
Z0416: ~58 uL of 165.0 ng/uL; 260/280 = 1.92; 260/230 = 2.23;
Z0511: ~58 uL of 89.2 ng/uL; 260/280 = 1.91; 260/230 = 2.07;
For Z0842 and Z0279, inoculated liquid cultures to grow overnight.  Used 8 mL LB + AMP so that both glycerol stocks and minipreps could be done.  Placed in incubator at 9:05 pm.  Should check on them tomorrow at 11:05 am.  Note that the tubes were held in a styrofoam container that was leaned against another styrofoam container such that the tubes were not exactly vertical but rather pointing at an angle from the vertical line.  This ensured a greater amount of aeration while shaking in the incubator.
Streaked out yeast plasmids on freshly made YPD plates.  Used streaking technique described above for glycerol stocks (shake n' plate).  Note that there were what appeared to be little dots on the plates before any medium was applied.  Thus, decided to do two negative controls: one is a plate with just YPD streaked on it while the other is a plain PD plate.  Streaked out MHY500 colony #1, MHY501, and MHY606 colony #1.  Placed in 30 degree incubator at 9:45 pm.
6/21/08
Took parts Z0279 and Z0842 out of incubator around 2:15 pm after being in the incubator for 17 hr and 8 min.
Z0279: 0.179 600 nm absorbance
Z0842: 0.193 600 nm absorbance
Glycerol stock and miniprep of Z0279 and Z0842
Made 2 glycerol stocks from each of the 2 cultures.  (Glycerol stocks for each part are from the same colony.)
Miniprepped the remaining culture.
Used 60 uL of EB for elution.  Final results:
Z0279: ~58 uL of 109.2 ng/uL; 260/280 = 1.91; 260/230 = 2.05;
Z0842: ~56 uL of 34.4 ng/uL; 260/280 = 1.97; 260/230 = 1.60;
Note that the 260/230 ratio is usually between 1.8 and 2.2 for pure nucleic acid.  According to the Nanodrop manual, "If the ratio is appreciably lower, this may indicate the presence of co-purified contaminants. " It seems that Z0842 is entirely suspect, so we should try re-doing the miniprep tomorrow.  Should innoculate a culture tonight in order to do that.
Also, realized that it would be beneficial to miniprep some of the yeast plasmids in preparation for the necessary insertions of parts into shuttle vectors.  Thus, in addition to Z0842, also innoculated three cultures with pRS304*, pRS305, and pRS306.  Put these cultures in around 5 pm.  Will try to check up on them around 10 am tomorrow, giving them roughly 17 hours in the incubator (again, put tubes at a slant).
Transformations of remaining limiter parts will be postponed until their significance can be demonstrated in the construction scheme.  For now, it is more important to construct the simple test devices to show that the functional parts we received are, indeed, functional in our testing conditions.
These simple test devices consist of two constructs.  The first construct is either a repressor or an activator, while the second construct is a reporter that has specific paired activity with the first.  We will be testing their function not only in a different yeast strain than that used in the original testing of these parts (MHY500 or MHY501 as opposed to PSY580A) but also in different media (i.e. a mixture of methionine and galactose at set concentrations rather than galactose and raffinose?).
6/22/08
--------------------------------------------------
To Do:
Miniprep Z0842, pRS304*, pRS305, and pRS306
--------------------------------------------------
Took cultures out of incubator at 11:00 AM after 18 hours.
ODs:
Z0842: .059, .New sample: 151  pRS304*: .125, pRS305: .140, and pRS306: .133
Miniprepped using 60 uL of EB to elute.
Results from miniprep:
Z0842: ~56 uL of 43.6 ng/uL; 260/280 = 1.97; 260/230 = 1.88;
pRS304*: ~58 uL of 117.6 ng/uL; 260/280 = 1.95; 260/230 = 2.18;
pRS305: ~58 uL of 87.1 ng/uL; 260/280 = 1.94; 260/230 = 1.71;
pRS306: ~58 uL of 134.3 ng/uL; 260/280 = 1.93; 260/230 = 2.25;
Took yeast plates out of incubator at 1:45 pm.  Found one colony on MHY606, three colonies on MHY500, and more than a dozen on MHY501, while there were no colonies on either negative control.  Wrapped them in parafilm and kept in fridge.
Plan cloning strategy
6/23/08
--------------------------------------------------
To Do:
Meeting at 9:30 am.
Make dropout plates and LB media
Buy more microcentrifuge tubes AND sterilize them
Purchase ingredients for yeast transformation if not done already
Acquire access to a roller drum
--------------------------------------------------
Need to make dropout plates as well as dropout media.  The plates will consist of dropout for LEU and URA and also URA and TRP.  This will allow us to select for yeast that have successfully integrated both a transcription factor construct (activator or repressor) and a reporter construct specific to each transcription factor.  We can also do a dropout for TRP, LEU, and URA in preparation for conducting Test #2, if we choose.  Media will have to have the same dropouts as the plates, with the addition of a methionine dropout.  This is so that different amounts of methionine can be added.
The scheme for making all of this is as follows:  Mix 27.26 g dropout media and 2 g histidine in 1 L of distilled water.  Then split this into half, adding 10 g bacto agar and 1 g methionine to one of the halves.  This will become the plates.  Each of these 0.5 L volumes will then be split into halves once more.  To one half of each, we will add 0.5 g tryptophan.  This will serve as the plates (A1) and media (A2) for the activator tests because it makes LEU and URA dropouts.  To the other halves, we will add 1 g leucine.  This will create the plates (B1) and media (B2) for the repressor tests because it makes TRP and URA dropouts.  Note the decision to include methionine within the plates but not within the media.  The rationale is that the plates will be used simply to allow for successfully transformed yeast to grow, while the media will be used to test different concentrations of methionine as needed.
Initial mixture
Drop out media mix: 17.2605g + 10.0030 g = 27.2635 g
Histidine: 2.0022 g
Dissolved in 1 L
Plate mixture
Bacto Agar: 10.0050 g
Methionine: 1.0010 g
Added to 0.5 L of initial mixture
A1: activator plates
Tryptophan: 0.5006 g
Added to 0.25 L of plate mixture
A2: activator media
Tryptophan: 0.50006 g
Added to 0.25 L of initial mixture
B1: repressor plates
Leucine: 1.0005 g
Added to 0.25 L of plate mixture
B2: repressor media
Leucine: 1.0032 g
Added to 0.25 L of initial mixture
 
Restriction Digests:
Insert V0002_Z0842 and V0002_Z0507 each into pRS_306 shuttle vector
Digest both insert and vector with EcoRI and SpeI.  For the parts and plasmids in question, we have ~56 uL of Z0842 at a concentration of 43.6 ng/uL, ~58 uL of Z0279 at a concentration of 109.2 ng/uL, and ~58 uL of pRS306 at a concentration of 134.3 ng/uL. For a 50 uL reaction, one adds 5 uL 10x BSA, 5 uL 10x buffer (multicore), as much as 10 ug of DNA, the necessary enzymes required to cut this DNA, and distilled water.
For future reference, one unit of enzyme from Promega is roughly defined as, "the amount of enzyme required to completely digest 1 ug of DNA in one hour at 37 degrees C in 50 uL of assay buffer containing Acetylated BSA added to a final concentration of 0.1 mg/uL."
EcoRI has 12 u/uL while SpeI has 10 u/uL.  For future reference, XbaI has 12 u/uL, PstI has 10 u/uL, and NotI has 10 u/uL.
For 10 ug of DNA, one will need 1 uL of SpeI and 0.83 uL of EcoRI.
For Z0842, one can use 38 uL of plasmid, to give 1.657 ug of DNA.  This would require 1.657 units of each enzyme which translates into 0.127 uL of EcoRI and 0.1526 uL SpeI.  However, one can probably use 1 uL of each just to be safe.    In such a case, one will add no water.  This will yield a 50 uL reaction.
For Z0279, one can use 38 uL of plasmid, to give 4.150 ug of DNA.  This would require 4.150 units of each enzyme which translates into 0.346 uL of EcoRI and 0.415 uL SpeI.  However, one can probably use 1 uL of each just to be safe.    In such a case, one will add no water.  This will yield a 50 uL reaction.
For pRS306, one can use 38 uL of plasmid, to give 5.103 ug of DNA.  This would require 5.103 units of each enzyme which translates into 0.425 uL of EcoRI and 0.510 uL SpeI.  However, one can probably use 1 uL of each just to be safe.    In such a case, one will add no water.  This will yield a 50 uL reaction.
It should be noted that EcoRI has star activity in the presence of multicore buffer so the restriction digest should not proceed for too long.  Placed in incubator for 1 hour then denatured enzymes on heat block at 70 degrees for 10 min.  Put in freezer in preparation for next step tomorrow.
Decided to postpone the linearization reactions until tomorrow since we were lacking BstEII.
6/24/08
--------------------------------------------------
To Do:
Meet with Adrian at 10 am in SFH.
Linearize parts
autoclave pipette tips and microcentrifuge tubes
--------------------------------------------------
Eletrophoresis gel: We messed up and digested the wrong part.  We did Z0279 instead on Z0507.  Z0279 is already in a yeast plasmid, while Z0507 is not.  This digest will yield nothing useful to us.
Thus, we decided to begin a restriction digest on Z0507 in order to get it into pRS306.
For Z0507, one can use 38 uL of plasmid, to yield 3.903 ug of DNA.  This would require 3.903 units of each enzyme which translates into 0.326 uL of EcoRI and 0.390 uL SpeI.  However, one can probably use 1 uL of each just to be safe.    In such a case, one will add no water.  Together with 5 uL BSA and 5 uL 10x buffer, this will yield a 50 uL reaction.  Ran the restriction digest for slightly over an hour in the incubator and then placed on heating block to denature enzymes.  Placed in fridge in preparation for work with it tomorrow or later in the week.
Running the gel: As mentioned before, since Z0279 is useless when digested with the enzymes we used, there was no point to extract it from the gel.  Also note that Z0842, when cut with EcoRI and SpeI, yields two pieces of DNA that are only about 2100 bp apiece and only 40 bp from each other.  This is impossible to separate on a gel.  Due to this oversight, we must run another restriction digest for Z0842, except this would be a triple digest that cuts a unique site only in the unwanted strand and not in the insert and also not on the desired vector for the part (pRS306).
Pre-weighed tubes for gel extraction:
{
Z0842 tube #1:
Starting weight = 1.0113g
End weight = g
=> Weight of gel = g
=> used uL of QG
Z0842 tube #2:
Starting weight = 1.0150g
End weight = g
=> Weight of gel = g
=> used uL of QG
}-
Note: The above tubes were NOT used because we did not separate the insert from the vector.
pRS306 tube #1:
Starting weight = 1.0140 g
End weight = 1.1035 g
=> Weight of gel = 0.0895 g   89.5 mg
=> used 268.5 uL of QG
pRS306 tube #2:
Starting weight = 0.9904 g
End weight = 1.1727 g
=> Weight of gel = 0.1823g    182.3 mg
=> used 546.9 uL of QG
Combined volume: 815.4 uL
Combined the 2 tubes in one gel extraction, eluted with 50 microliters of EB, put the results in the freezer.
Nanodrop measures a DNA concentration of 37.6 ng/uL    260/280: 2.18   260/280: 0.12
Did transformations of remaining parts today.  Tried microwaving half of the positive control after heat shock and plating it separately to get another sense of the effect of microwaving on transformation efficiency.  Placed in incubator for 2 hr and 6 min.
Plated almost all parts on 2 plates, with 80% of volume on one and 20% of volume on another.  Ran out of plates, so divided one plate apiece for parts K0001 and L0101 that we believe to already have from another source (i.e. iGEM registry).  In each half of each plate for these parts, we streaked the same difference in volumes (20% AND 80%).  Note that the plates used were those made on 6/18/08 that were suspected of either having too many bubbles or that had the antibiotic poured at too great a temperature.
Put in incubator at around 10:25 pm.  Should check on them again at 12:25 pm tomorrow.
This is a complete list of the parts that were transformed today:
V0120_P0520: pGAL1 + w/ XhoI sites
V0120_P0521: pGAL1 + w/ lexA op sites
V0120_K0001: Kozak sequence
V0120_C0061: Sin3p repression domain
V0002_E0110: mCherry
V0002_E0130: YFP
V0120_C0100: Zif268-HIV DBD
V0002_C0132: Gli-1 DBD
V0120_C0141: YY1 DBD
V0120_C0151: LexA DBD
V0120_L0101: terminator
V0120_P0450: Zif268-HIV binding sites
V0120_P0453: Gli1 binding sites
V0120_P0454: YY1 binding sites
V0120_Z0375: Kozak+LexA DBD+Sin3p RD+mCherryx2+SV40NLS+tADH1
V0120_Z0378: Kozak+Sin3p RD+Gli1 DBD+mCherryx2+SV40NLS+tADH1
V0120_Z0379: Kozak+Sin3p RD+YY1 DBD+mCherryx2+SV40NLS+tADH1
V0120_Z0402: Kozak+Sin3p RD+Zif268-HIV DBD+mCherryx2+SV40NLS+tADH1
V0002_Z0038: SV40NLS+tADH1
Also decided to miniprep more of the plasmid that was used for the ligations.  This includes pRS306, Z0279 (which had been chosen by mistake), Z0842 (which requires another scheme for ligation into pRS306), and Z0507 (which we might need should the current ligation into pRS306 fail).
However, decided against doing this tonight because the only available LB + Amp media appeared to have chunks of some white cloudiness.  Does not appear to be contamination, but will leave it out on the table overnight to make sure.  (Also, it did not make sense to add antibiotic to the remaining 50 mL of plain and clear LB because of the small mass of antibiotic required (0.5 mg).
6/25/08
--------------------------------------------------
To Do:
Ask Adrian if we can use a bit of any restriction enzymes that will work to cut the plasmid fragment from ZO842
Do that restriction digest.
Gel purify ZO507 & ZO842
Ligate these into yeast vector
Receive shipment at stockroom.
Mix up solutions for yeast transformation
Linearize parts for yeast.
--------------------------------------------------
Took out culture plates at 2:35 PM, after about 16 hours.
All plates have good colonies but the (-) control!  Hard to distinguish between the positive control that had been microwaved and the one that had not, but both seemed to yield a high density of cells.  Decided to put them into fridge until later tonight, when we will innoculate colonies from these plates for use in glycerol stocks and, if we have enough media, minipreps.
Making LB:
10.0082g + 10.0074g  Lennox Base
1 L water
Making ampicillin stock solution:
Collected information about ampicillin - Typical concentrations of ampicillin are 50 ug/mL for low copy plasmids and 100 ug/mL for high copy plasmids.  Stock solutions are usually in 100 mg/mL.  Ampicillin will last about 4-6 months or 1 year (according to different sources) when stored in dH2O at -20 degrees C.  It will only last about 3 days at 37 degrees C.   We will begin to use 100 mg of ampicillin per liter of LB, as opposed to what we used in the past, which was only 10 mg per L.  Our concentration for plates was already 100 mg/L, so this will remain unchanged.
1.0001g of ampicillin dissolved in 10 mL.  This yields a concentration of approximately 100 mg/mL.  Solution was extremely frothy and difficult to manage, so we attempted to continue dissolving into 100 mL instead.  However, this also resulted in a fair amount of undissolved ampicillin.  Read online that sometimes with ampicillin trihydrate powder it is easier to dissolve if one adds 10 M NaOH to the solution.  We had 5 N NaOH, so we added 20 uL.  This appeared to work, so we added an additional 80 uL.  The mixture was a little cloudy still, so we decided to simply add powdered ampicillin to some of the freshly autoclaved LB.  This should work for a while until we get our stock solution.
Note that the transformation protocol from the Silver Lab indicates a minimum DNA concentration of 250 ng/uL, requiring 4 uL of this.  This means that we may very well have to purify the DNA even further before linearizing it.  We shall commence with the linearization and then purify it afterwards.  We need to put more than 1 ug of DNA into the linearization reaction.  2 ug or more seems appropriate.  Will shoot for 2.5 ug.
Z0267: ~58 uL of 135.8 ng/uL; => 39 uL yields 5296.2 ng of DNA.
Z0637: ~56 uL of 123.4 ng/uL; => 39 uL yields 4812.6 ng of DNA.
Z0277: ~58 uL of 123.2 ng/uL; => 39 uL yields 4804.8 ng of DNA.
Z0739: ~58 uL of 108.4 ng/uL; => 39 uL yields 4227.6 ng of DNA.
Z0415: ~58 uL of 139.4 ng/uL; => 39 uL yields 5436.6 ng of DNA.
Z0411: ~58 uL of 162.8 ng/uL; => 39 uL yields 6349.2 ng of DNA.
Z0510: ~56 uL of 67.1 ng/uL; => 39 uL yields 2616.9 ng of DNA.
(Z0421: ~58 uL of 171.7 ng/uL; => 39 uL yields 6696.3 ng of DNA).
Z0412: ~58 uL of 199.8 ng/uL; => 39 uL yields 7792.2 ng of DNA.
Z0418: ~58 uL of 53.8 ng/uL; => 39 uL yields 2098.2 ng of DNA.
Z0416: ~58 uL of 165.0 ng/uL; => 39 uL yields 6435 ng of DNA.
Z0511: ~58 uL of 89.2 ng/uL; => 39 uL yields 3478.8 ng of DNA.
Note that we did not linearize Z0421 as we are currently in the process of inserting its reporter (Z0507) into a yeast shuttle vector.  We will linearize this part along with its promoter and the LexA activator parts (Z0279 and Z0842).
Both BstEII and PstI have concentrations of 10 u/uL. Thus, 1 uL of each should be able to completely cut 10 ug of DNA in 1 hour at 37 degrees.  As before, use 5 uL BSA and 5 uL 10x buffer, which would be H for PstI and either D or Multicore for BstEII.
During linearization, use PstI for pRS304*, BstEII for pRS305, and PstI for pRS306.  BstEII is listed as having best efficiency at 60 degrees Celsius, so we placed it on a heating block at that temperature.  PstI, on the other hand, is most efficient at 37 degrees C.
Both spent roughly 1 hour and 9 min at their respective temperatures.  Placed parts with PstI on heating block at 65 degrees for 25 min. to help inactivate the enzyme.  Note:  due to some misordering of the tubes, it appears that the digest of Z0277, which contains BstEII, was accidentally placed on the heating block for an additional 25 mins, meaning that it received almost a full additional half hour of enzyme activity.  Hopefully this will not affect the outcome too greatly.  Placed all linearized DNA in the freezer in preparation for the yeast transformation.
We also decided to start cultures of parts that we were going to culture last night (pRS306, Z0842, Z0507, and Z0279).  In addition to these parts, we decided to miniprep the 19 remaining parts that were successfully transformed last night.  As such, we made cultures for these parts as well.  Note that all cultures are grown in 16 mL culture tubes with 10 mL of LB except for 8 parts, namely P0450, P0453, P0454, Z0375, Z0378, Z0379, Z0402, and Z0038.  These parts correspond to the binding sites for Zif286-HIV, Gli1, and YY1 (P0450-P0454), the entire repressor proteins minus the MET25 promoter (Z0375-Z0402), and the NLS plus the terminator (Z0038).  For these specific parts, we grew them in 50 mL centrifuge tubes with the caps slightly loosened in 30 mL of LB.  The reasoning for this is that we are going to attempt a massive miniprep of these cultures to accrue a lot of DNA, thus ensuring that we will not have to do another miniprep for these parts.  These larger tubes were also affixed at a slant in the shaking incubator as we usually do for the 16 mL culture tubes in order to increase the amount of aeration.  Shaking at 210 rpm.
Important note:  We observed that there were (two) colonies on a YPD plate that we had left on the table at room temperature and also (three) colonies on a YPD plate that we had streaked with YPD media and left in the 30 degree C incubator as a negative control when we had streaked out the yeast strains from glycerol stocks.  Both plates were made on 6/20.  This is bad.  It means that our YPD agar mix was probably contaminated or that our plates were subject to contamination when we made them.  It may also implicate our YPD media as contaminated.  Indeed, a check right now confirmed that there were small pieces of cloudy substance within the YPD media, indicating its contamination.  Our yeast plates are thus no good and we should make more of them tomorrow.  We should also make some more YPD media.  Then we will need to restreak our yeast strains from glycerol stocks.  One will also note at this point that the MHY501 glycerol stock may be contaminated from before.  Thus, one should primarily focus with using MHY500 or MHY606.  It would be idea if we could simply acquire fresh strains of all three seeing as their purity is somewhat questionable.
For future reference:
1M Tris, pH 8.0 (1 L):
Mix 121.1 g of Tris base with 800 mL dH2O. Add 42 mL cHCl. pH is temperature dependent: ~0.03 pH units per 1oC increase. Make sure it is at RT before making final pH adjustments. Add dH2O to 1 L (Autoclave)
6/26/08
--------------------------------------------------
To Do:
Make YPD & plates
Ampcillin stock solution?
Salmon Sperm? Anyone?
Continue preparations for yeast transformation (make solutions)
Meet at 1 pm with Richard Bennett, who will be providing us access to a roller drum at 30 degrees C
Miniprep all parts from last night
--------------------------------------------------
Met with Richard Bennett.  He was quite informative.  He convinced us that it might be easier to find two more shuttle vectors than trying to put multiple parts on the same vector.  He put us in touch with the Laney lab to help acquire a yeast strain (W303?) and also to maybe get two more yeast plasmids.  He suggested that we find a vector with a HIS selection marker, and one more with a positive selection marker such as G418.  Another alternative would be to clone separate constructs into different types of haploid strains (e.g. three constructs in 'a', two in 'alpha') and then mate them to form a diploid strain.  This way, we could use the same shuttle vectors that we already have to build our construct.  Unfortunately, however, this process has the effect of making the ratio between all other endogenous genes in the yeast cell and those on our own device 2:1.  We should still consider this possiblity.  He also indicated that it might be possible to visualize how much protein we have using a Western blot, with antibodies for the mCherry region (which four of our constructs will share).  Then, if possible we separate them out on a gel and determine how much there is from that.
Note that for the yeast transformation we will be following the protocol outlined in this paper: http://www.nature.com/nprot/journal/v2/n1/pdf/nprot.2007.13.pdf
Will require the use of salmon sperm.
Lithium acetate: 10.2g lithium acetate dihydrate in 100 mL
Actual weight:  10.2007 g
TE: 10 mM Tris-HCl, 1 mM Na2EDTA, pH 8.0
EDTA:  To make 1 M solution (which we should do eventually).  This will require 1 M = x (mol) / 100 (mL) => x = 0.1 (mol) EDTA
M.W. EDTA = 372.23 g/mol
0.1 (mol) = y (g) / 372.23 (g/mol) => 37.223 g EDTA disodium salt dihydrate
Scale this down into 50 mL, so use 18.6115 g.
Weighed out 18.6122 g of EDTA.
We have 1M Tris-HCl, pH 8.0, so we need to dilute this by 100. to get 10 mM Tris-HCl.  Thus, one should add 1 mL for every 100 mL.
To get 10 mM Tris-HCl and 1 mM EDTA, add 1 mL of 1 M Tris-HCl and 0.1 mL of 1 M EDTA to 100 mL of water.
Will autoclave all of these tomorrow, in addition to YPD media.
Took cultures out after 15 hr and 55 min.
Miniprepped.  Results:
V0120_P0520: pGAL1 + w/ XhoI sites
V0120_P0521: pGAL1 + w/ lexA op sites
V0120_K0001: Kozak sequence
V0120_C0061: Sin3p repression domain
V0002_E0110: mCherry
V0002_E0130: YFP
V0120_C0100: Zif268-HIV DBD
V0002_C0132: Gli-1 DBD
V0120_C0141: YY1 DBD
V0120_C0151: LexA DBD
V0120_L0101: terminator
V0120_P0450: Zif268-HIV binding sites
V0120_P0453: Gli1 binding sites
V0120_P0454: YY1 binding sites
V0120_Z0375: Kozak+LexA DBD+Sin3p RD+mCherryx2+SV40NLS+tADH1
V0120_Z0378: Kozak+Sin3p RD+Gli1 DBD+mCherryx2+SV40NLS+tADH1
V0120_Z0379: Kozak+Sin3p RD+YY1 DBD+mCherryx2+SV40NLS+tADH1
V0120_Z0402: Kozak+Sin3p RD+Zif268-HIV DBD+mCherryx2+SV40NLS+tADH1
V0002_Z0038: SV40NLS+tADH1
Sample ID    User ID    Date     Time     ng/ul     A260     A280     260/280     260/230     Constant     Cursor Pos.    Cursor abs.    340 raw
Default    6/26/2008    6:34 PM    68.92    1.378    0.760    1.81    2.07    50.00    230    0.667    0.008
PO520    Default    6/26/2008    6:36 PM    65.88    1.318    0.705    1.87    2.00    50.00    230    0.659    0.029
PO521    Default    6/26/2008    6:37 PM    60.90    1.218    0.636    1.92    2.15    50.00    230    0.566    0.030
K0001    Default    6/26/2008    6:38 PM    68.79    1.376    0.727    1.89    2.09    50.00    230    0.657    0.011
C0061    Default    6/26/2008    6:40 PM   115.92    2.318    1.218    1.90    2.27    50.00    230    1.022    0.015
E0110    Default    6/26/2008    6:41 PM    20.29    0.406    0.219    1.85    1.81    50.00    230    0.225    0.025
E0130    Default    6/26/2008    6:43 PM    24.95    0.499    0.277    1.80    1.81    50.00    230    0.276    0.009
E0130    Default    6/26/2008    6:43 PM    25.68    0.514    0.264    1.94    1.79    50.00    230    0.287    0.029
C0100    Default    6/26/2008    6:44 PM    68.33    1.367    0.719    1.90    2.11    50.00    230    0.647    0.014
C0132    Default    6/26/2008    6:46 PM    23.88    0.478    0.243    1.97    1.98    50.00    230    0.241    0.016
C0141    Default    6/26/2008    6:48 PM    70.27    1.405    0.740    1.90    2.14    50.00    230    0.657    0.019
C0151    Default    6/26/2008    6:49 PM    68.62    1.372    0.739    1.86    2.16    50.00    230    0.634    0.009
-L0101    Default 6/26/2008    6:50 PM    80.34    1.607    0.848    1.89    2.06    50.00    230    0.780    0.014
P0450$  Default    6/26/2008    6:52 PM    164.05    3.281    1.679    1.95    2.19    50.00    230    1.501    0.033
P0453$  Default    6/26/2008    6:53 PM    203.13    4.063    2.094    1.94    2.25    50.00    230    1.807    0.030
P0454$  Default    6/26/2008    6:54 PM    131.36    2.627    1.355    1.94    2.20    50.00    230    1.193    0.040
Z0375$  Default    6/26/2008    6:55 PM    266.25    5.325    2.762    1.93    2.34    50.00    230    2.277    0.013
Z0378$  Default    6/26/2008    6:57 PM    240.94    4.819    2.514    1.92    2.25    50.00    230    2.145    0.081
Z0379$  Default    6/26/2008    6:58 PM    235.52    4.710    2.438    1.93    2.30    50.00    230    2.049    0.046
Z0402$  Default    6/26/2008    7:00 PM    237.26    4.745    2.492    1.90    2.02    50.00    230    2.344    0.214
Z0038$  Default    6/26/2008    7:01 PM    124.87    2.497    1.264    1.98    2.24    50.00    230    1.116    0.028
pRS306  Default    6/26/2008    7:02 PM    75.55    1.511    0.779    1.94    2.15    50.00    230    0.703    0.024
P0501    Default    6/26/2008    7:03 PM    31.32    0.626    0.315    1.99    2.02    50.00    230    0.311    0.028
P0560    Default    6/26/2008    7:03 PM    61.55    1.231    0.639    1.93    2.13    50.00    230    0.578    0.009
Z0842    Default    6/26/2008    7:05 PM    22.04    0.441    0.231    1.91    1.68    50.00    230    0.262    0.033
Z0279    Default    6/26/2008    7:06 PM    148.81    2.976    1.526    1.95    2.27    50.00    230    1.310    0.034
Z0507    Default    6/26/2008    7:07 PM    98.75    1.975    1.035    1.91    2.23    50.00    230    0.886    0.030
$ indicates that a 30ml culture was used.
also saved at C:\nanodrop data\Reports
Used 75 uL EB to elute the DNA from the larger cultures and 60 uL for all others.
Forgot to glycerol stock, so will have to innoculate more colonies.
Must make glycerol stocks of all parts except for Z0279, pRS306, Z0507, Z0842, P0501, and P0560.
Will also redo the minipreps for the parts that seemed to have a considerably low yield.  This includes E0110, E0130, C0132, P0501, Z0842, P0520, P0521, K0001, C0100, C0151, P0560, and even C0141, pRS306 and L0101.  One can make 20 mL cultures of each.
0.0252 g of ampicillin in 250 mL LB
Note: Will start cultures for both glycerol stocks and minipreps tomorrow night.  Just too tired tonight.  Worried about making a mistake.
6/27/08
--------------------------------------------------
Salmon Sperm? Anyone?
Make YPD & plates
run triple digest for oddball part
run a gel of both oddball digests
ligate oddball parts into vector
Get yeast strain and/or vectors from Jeff Laney's lab?
Ampcillin stock solution?
Continue preparations for yeast transformation (make solutions)
--------------------------------------------------
Prioritized autoclaving YPD media, YPD plates, 1M LiOAc solution, 0.1 M EDTA solution, and 10mM Tris-HCl 1mM EDTA solution.  Note that there were some issues while making the 0.1 M EDTA (i.e. spilling some of the volume before the EDTA had been completely dissolved), such that the estimated molarity is less than 0.1M.  Thus, since this EDTA solution was used to make the 10mM Tris-HCl 1 mM EDTA solution, the concentration of EDTA in the latter solution is also lower than specified.
Also note contamination in the LB left over from making LB + amp last night.  One must thus be wary about using the LB + amp produced from what ended up being contaminated media, especially if it is to be used for a glycerol stock.
Added 20.5 mg of ampicillin to 200 mL LB from a different batch of LB from 6/25/08 that had been sitting out and was most certainly not contaminated.
For parts that were listed yesterday as requiring a miniprep, they were innoculated in 7-10 mL LB + amp (almost all parts were in around 8 mL of culture except P0520, which was in 10 mL).  For parts that had sufficient amount of DNA from yesterday's miniprep, they were cultured in 2 mL liquid in preparation for making glycerol stocks tomorrow.
Placed in incubator set to 250 rpm, with tubes slanted at an angle, at 1:12 am on 6/28/08.  Will check on them tomorrow at 3:12 pm.
Triple or quadruple digest is necessary for inserting Z0842 into pRS306.  If a triple digest, we should do a single digest in compatible buffer.  If a quadruple digest, we can switch buffers using Qiagen spin columns, noting that each time we do this we will lose some DNA.
Looks like triple digest will be unlikely.  Will thus consider quadruple digest:  EcoRI has 100% activity in buffer H.  SpeI has 100% activity in buffer B.  Thus, will try to locate two enzymes that have 100% activity in either buffer, that are unique to the unwanted portion of Z0842, that will cut in a location to allow distinguishing between the strands on a gel, and that are also not present at all on pRS306.  Buffer B candidates: HaeII, NciI (cuts in 3 places on Z0842 according to constraints), XmnI.  Buffer H candidates:  none.  Best option would be NciI.  Unfortunately, all three restriction sites can be found in pRS306.  One must wonder if it is possible to run the Buffer B reaction first with EcoRI and NciI, then run it through a column to change the buffer to Buffer H and run a second restriction digest using just SpeI.  One could then run the results on a gel and ligate to pRS306.  Would this work?
Ordered salmon sperm DNA.
6/28/08
--------------------------------------------------
By some fluke, came in at 7:30 pm instead of 3:12 pm.  Cultures were in incubator for 18 hr and 16 min.
Took some time to make glycerol stocks of all of them.  They finally went into the -80 freezer around 9:45 pm.
This is the list:
V0120_P0520: pGAL1 + w/ XhoI sites
V0120_P0521: pGAL1 + w/ lexA op sites
V0120_K0001: Kozak sequence
V0120_C0061: Sin3p repression domain
V0002_E0110: mCherry
V0002_E0130: YFP
V0120_C0100: Zif268-HIV DBD
V0002_C0132: Gli-1 DBD
V0120_C0141: YY1 DBD
V0120_C0151: LexA DBD
V0120_L0101: terminator
V0120_P0450: Zif268-HIV binding sites
V0120_P0453: Gli1 binding sites
V0120_P0454: YY1 binding sites
V0120_Z0375: Kozak+LexA DBD+Sin3p RD+mCherryx2+SV40NLS+tADH1
V0120_Z0378: Kozak+Sin3p RD+Gli1 DBD+mCherryx2+SV40NLS+tADH1
V0120_Z0379: Kozak+Sin3p RD+YY1 DBD+mCherryx2+SV40NLS+tADH1
V0120_Z0402: Kozak+Sin3p RD+Zif268-HIV DBD+mCherryx2+SV40NLS+tADH1
V0002_Z0038: SV40NLS+tADH1
Of these parts, did minipreps of P0520, P0521, K0001, E0110, E0130, C0100, C0132, C0141, C0151, and L0101.  Also miniprepped Z0842 and pRS306.
Eluted all in 65 uL EB, though some of P0520 fell out of its column.  What had fallen out was pipetted up from the table and placed in a second column, which was eluted with 50 uL EB.
Results from miniprep:
P0520: ~61 uL of 42.5 ng/uL; 260/280 = 1.98; 260/230 = 2.07;
P0521: ~61 uL of 58.6 ng/uL; 260/280 = 1.91; 260/230 = 1.86;
K0001: ~61 uL of 57.0 ng/uL; 260/280 = 1.93; 260/230 = 1.71;
E0110: ~61 uL of 24.7 ng/uL; 260/280 = 2.04; 260/230 = 1.91;
E0130: ~61 uL of 31.1 ng/uL; 260/280 = 1.93; 260/230 = 1.74;
C0100: ~61 uL of 68.6 ng/uL; 260/280 = 1.91; 260/230 = 1.88;
C0132: ~61 uL of 33.8 ng/uL; 260/280 = 1.94; 260/230 = 1.87;
C0141: ~61 uL of 67.2 ng/uL; 260/280 = 1.92; 260/230 = 2.17;
C0151: ~61 uL of 59.5 ng/uL; 260/280 = 1.95; 260/230 = 2.09;
L0101: ~61 uL of 63.4 ng/uL; 260/280 = 1.93; 260/230 = 2.05;
Z0842: ~61 uL of 40.4 ng/uL; 260/280 = 1.89; 260/230 = 1.89;
pRS306: ~61 uL of 90.1 ng/uL; 260/280 = 1.93; 260/230 = 1.83;
P0520#2: ~46 uL of 20.1 ng/uL; 260/280 = 1.97; 260/230 = 1.79;
These results only signify improvements from the last set of minipreps for parts E0110, E0130, C0132, pRS306, and Z0842.  Clearly overestimated the amount of DNA and used too much liquid for elution.  This overestimation could have stemmed from the fact that 1-1.5 mL of each culture was used for making glycerol stocks, thus reducing the number of cells going into the miniprep.  However, one should note that there were rather sizable pellets after centrifuging the overnight cultures.  Thus, it is possible that, due to the 18 hours in the incubator, many of the cells may have been dead when the miniprep began, which could have played a role in lower-than-anticipated yields.
Realized that P0501 and P0560 had been forgotten in this second round of minipreps, as no cultures of these two parts were made.
6/30/08
--------------------------------------------------
Remember to ask Adrian about making the stock solution for ampicillin
--------------------------------------------------
Decided to remake EDTA solution as well as Tris-Hcl EDTA solution.
Around 3.72 g EDTA in 100 mL to make a 0.1 M solution.  Then took 1 mL of this and 1 mL of 1M Tris-HCl and added water until a final volume of 100 mL to create 10mM Tris-HCl 1mM EDTA solution.  Autoclaved both.
Also decided to remake YPD plates and media.  250 mL of liquid for each one
25.0024 g YPD broth
5.0015 g Bactoagar
Note that we used a much newer autoclave than the one we usually use.  This still resulted in darkening the mixture, which we discovered is typical of YPD media after it has been autoclaved.
Met with Jeff Laney today.  He provided us with some excellent advice concerning our construction strategy.  In particular, he helped to convince us that the easiest approach for inserting all five constructs into single yeast cells is to utilize two or three separate shuttle vectors to insert different constructs into different haploid yeast strains and then mate the two strains together.  For example, one can insert constructs #1-3 into the first yeast strain using pRS304, pRS305, and pRS306 (select using triple dropout of ura, leu, and trp) and then one can insert constructs #4 and 5 into the second yeast strain with pRS303 and pRS304 (select using double dropout of his and trp).  One would then mate the two strains and plate the resulting cells onto dropout for his and also for either leu or ura, using one selection marker that is unique to each strain.  This would ensure that only cells that had incorporated all inserts would survive, but also only necessitate the usage of four different auxotrophic markers rather than five.  He also generously provided us with two haploid yeast strains, W303_1a (3-9-1) and W303_1alpha (3-9-2), which can be mated together to produce a diploid W303.  W303 is auxotrophic for leu2, trp1, his2, ura3, and (possibly) ade.  He also gave us pRS303 (1-7-3), a yeast shuttle vector that complements the auxotrophy for the his2 gene.
Another key issue that he helped us resolve was with regard to Test #2, which would test competative binding of an activator and a repressor for the same binding site.  He indicated that the PGAL1 promoter that drives expression of our synthetic activators does not have a very tunable response.  Thus, he suggested that we switch this promoter with PMET25 and replace the promoter in front of each repressor with a constitutive promoter.  This actually is closer to what we would have in our final device, so it makes sense to simply incorporate this redesign of Test#2 into the cloning strategy for the final device.   For this problem of ours, he provided a set of constitutive promoters called 'Funk promoters', in reference to the researcher who tested them (I think).  The promoters provided to us are p413_PCYC (V1-60), p414_PADH (V1-55), p413_PTEF (V1-45), and p413_PGPD (V1-37).  The reference for these promoters is Mumberg et al.  Gene (1995) 156:119.
7/01/08
--------------------------------------------------
11 am meeting with IP lawyers
Plan out construction strategy with four yeast shuttle vectors
purchase crystalline ampicillin
purchase NciI, XmnI, or HaeII???
email Laney
email Sindi
make LB agar plates
--------------------------------------------------
Need LB Agar plates for constitutive promoter parts.  Will make these plates today.
Bacto Agar: 20.0012 g
LB Broth (Lennox): 20.0015 g
ampicillin: 0.1010 g
Also made LB media.
LB Broth (Lennox): 20.0053 g
At this juncture, we realized the necessity of planning out which primers we will need to correctly assemble
These are the parts that we will need to PCR out of their respective plasmids to make biobrick/biofusion parts:
-Z0267, Zif268-HIV activator (no promoter)
-Z0277, Gli1 activator (no promoter)
-Z0279, LexA activator (no promoter)
Note that before we can consider using Z0279, we should find a way to insert its reporter, Z0842, into a shuttle vector
-CYC promoter
-TET promoter
-GPD promoter
Note that ADH1 is already in the registry as a Biofusion part.  It is located in Plate 3, Well 8M of the 2007 parts kit, listed as E2020.  It has Kanamycin resistance.  Note that we still have this DNA and should transform it soon.  Also important is the fact that GPD has a PstI site located in the middle of its sequence.  This promoter will thus require more work in order to make it a Biobrick.  Should consider using this part only if the others do not give the desired behavior.
-ECFP, (E2020 from registry), might want to convert into biofusion
-mCYC promoter from our own parts , both by itself and also combined with multiple different bindings sites
-LexA binding sites
PCR Primers:
Here, we insert the biobrick prefix and suffix (with extra BPs in caps to permit cutting at the end sites) when we PCR these 3 promoters out of their plasmids.
Q's:  Are the melt temps OK?  Is it OK to add ~30bp onto a 20-25bp complimentary primer
pCYC
forward:
GTTTCTT + gaattcgcggccgcttctagag+ (binding sites?) + agcgttggttggtggatcaag
22bp 63.6C
GTTTCTTgaattcgcggccgcttctagagagcgttggttggtggatcaag
74.2C Total
reverse seq:
cacaaacacaaatacacacactaaattaata + tactagtagcggccgctgcag + AAGAAAC
30bp 53.9C
cacaaacacaaatacacacactaaattaata tactagtagcggccgctgcag AAGAAAC
rev primer:
GTTTCTTctgcagcggccgctactagtatattaatttagtgtgtgtatttgtgtttgtg
67.5C Total
 
pTEF
forward:
GTTTCTT + attacccataaggttgtttgtgacgg + agcgttggttggtggatcaag
26bp 60.4C
GTTTCTTattacccataaggttgtttgtgacggagcgttggttggtggatcaag
71.4C total
rev. seq:
cggtcaacgaactataattaacta + tactagtagcggccgctgcag + AAGAAAC
52.2C
cggtcaacgaactataattaactatactagtagcggccgctgcagAAGAAAC
rev primer:
GTTTCTTctgcagcggccgctactagtatagttaattatagttcgttgaccg
68.7C total
 
pGPD (TDH3)
fwd:
GTTTCTT + attacccataaggttgtttgtgacgg + TCAGTTCGAGTTTATCATTATCAAT
25bp 51.8C
GTTTCTT attacccataaggttgtttgtgacgg TCAGTTCGAGTTTATCATTATCAAT
rev. seq:
cgaataaacacacataaacaaacaaa + tactagtagcggccgctgcag + AAGAAAC
26bp 52.7C
cgaataaacacacataaacaaacaaa tactagtagcggccgctgcag AAGAAAC
rev primer:
GTTTCTTctgcagcggccgctactagtatttgtttgtttatgtgtgtttattcg
Streaked out plasmids pRS303, p413_PCYC, p414_PADH, p413_PTEF, and p413_PGPD onto new plates.  Placed in incubator at 11:45 pm.  Should check on them again at 1:45 pm tomorrow.
Promoters:
pCYC: -287,-1 bp
CYC1- Chr. X 526328-526657
7/2/08
--------------------------------------------------
Plan out construction strategy with four yeast shuttle vectors
purchase NciI, XmnI, or HaeII???
email Sindi
purchase more culture plates
--------------------------------------------------
Good colonies on yeast plates.  Will be ready to innoculate some liquid cultures tonight around 5 pm or 6 pm and grow them up overnight for making glycerol stocks.
Unfortunately, the plates for our new shuttle vector and promoters were spread far too thick to achieve colonies.  Will try to restreak them tonight with hopefully better results.
Primer design:
Due to the palindromic nature of the enzymes that flank a Biobrick part, all primers that contain these sequences will have some degree of self-complmentarity.  This may or may not present a problem.
We are discussing the issue of where to put the binding sequences on each promoter.  It appears that for all activator reporters that we received, the binding sites come right before the promoter, which is mCyc.  However, for all repressor reporters, the binding sites appear to come in the middle of the promoter.  Since our system incorporates a combination of both, we will have to consider where the optimal placement of the binding sites should be. This will likely require a literature search on our part.
We also devised another test (Test #1.5) to conduct after we have done Test #2.  This test will involve two mutually-repressive constructs, only one of which is under control of an inducible promoter (PMET25, perhaps?).  This test will occur only after Test #2 because Test #2 will tell us which repressor/activator pair we require in order to get good competitive binding.  Then we can choose a different repressor to be the second construct in Test #1.5, which will require a different fluorescence.  This Test #1.5 will be a closer approximation of constructs Z and Y from our final design, which mutually repress each other.
Amp stock solution: 1.0010 g  in 10 mL to create 10 mL of 100mg/mL ampicillin stock solution.  Put them in 1 mL aliquots and froze them at -20 degrees C.  Add 1 mL for 1 L of media, or 1 uL for each mL of media.
Also, restreaked the promoters and shuttle vector.  Note the attempt at a new streaking technique:  Use a loop to collect a small sample of cells.  Then rub the loop holding cells back and forth (along an imaginary horizontal line) at the top of an agar plate.  After doing this, using the loop, draw a short vertical line down from the newly deposited cells. * Turn the loop over so that you can use a side of the loop that was containing no cells. *  Then drag the loop across the agar in a zigzag pattern downward, making sure that the loop crosses through the vertical line of cells ONCE before continuing downward. If there is room at the sides of the plate after this initial zigzag pattern has been made, one can make even more smaller zigzags that borrow cells from the original zigzag pattern. Placed plates in the incubator at 11:28 pm.  Will check on them tomorow at 1:28 pm.  Put the plates from last night into the fridge.
7/3/08
--------------------------------------------------
Plan out construction strategy with four yeast shuttle vectors
continue to design primers for PCR
email Sindi
make glycerol stocks of W303 'alpha' and 'a'
--------------------------------------------------
Jamie indicated that hairpins and self-complimentarities should not be a problem in our primers, especially since they can be expected for primers with Biobrick restriction sites.  He suggested that we run a control PCR reaction without any plamid DNA to see what kinds of PCR products will be made just with the primers we use.  Granted, we expect these unwanted PCR products to be small.
Took yeast cells out of incubator at 1:45 pm.  Made two glycerol stocks of each strain (W303_1a and W303_1alpha), each from the same culture/colony.
Also, took plates out of incubator around 1:40 pm.  With the exception of PCYC, they demonstrated few if any isolated colonies. However, it looks like it will be possible to take at least one isolated colony from all plates but p413_GPD. Replated p413_GPD in order to isolate more colonies.  Placed in incubator around 11:40 pm.
7/4/08
Transformations:
 
For 10 mL of culture, one requires 10 uL of ampicillin stock solution in each tube to supply 100 ug of antibiotic per mL.  Innoculated cultures with pRS303, p413_PCYC, p414_PADH, and p413_PTEF.  At around 11:40 pm, placed them in incubator at a slant at 240 rpm.    Will do a glycerol stock and miniprep of these tomorrow.
Began yeast tranformation protocol by innoculating yeast cultures and putting them in incubator overnight.  Will do dilution tomorrow morning and follow the procedure from there.
7/4/08 (BOOM!!)
--------------------------------------------------
yeast transformation of linearized activators, repressors, and reporters
Plan out construction strategy with four yeast shuttle vectors
email Sindi
make glycerol stocks and miniprep promoters (CYC, ADH, TEF) and pRS303
--------------------------------------------------
Note: The transformation protocol is according to Gietz and Schiestl, Nature Protocols (2007) Vol 2 No. 1.
Continued with yeast transformations.  Had optical density readings of 0.259 for W303_1alpha and 0.188 for W303_1a from the overnight cultures.  This corresponds roughly to cell concentrations on the order of 10^7 cells per mL (Note that our Nanodrop seems to report OD values one order of magnitude lower than they actually are, so that ).  One possible reason for this low density was the lack of adenine in the media, which requires the cells to produce their own.  Decided to put all 5 mL of W303_1a into the pre-warmed 50 mL of YPD.  This addition made for roughly 1.3 x 10^8 cells by our estimates, which should account for about half the required cells in the protocol.  This should not matter too greatly seeing as we will be growing them up again during the 4 hour incubation period.  However, low cell count may indicate low transformation efficiency.  The flask used was an Erlenmeyer flask covered with tin foil.  The incubator was set to 210 rpm.
The OD of the
Resuspended in 0.4 mL at the final step.  Decided to limit the transformations to four sets of parts, each being a double transformation:
Gli1 activator + Gli1 activable reporter (Z0277 + Z0739),
Gli1 repressor + Gli1 repressor reporter (Z0415 + Z0510),
null repressor + Gli1 repressor reporter (Z0411 + Z0510)
Zif268-HIV activator + Zif268-HIV activable reporter (Z0267 + Z0637).
The reason for choosing the first three parts is because they serve as a good point of comparison between parts with the same binding site.  We decided to transform the Zif activator + reporter simply because we had it available, because we had extra cells after the final centrifugation step, and because this activator had been characterized beforehand in the paper by Caroline Ajo-Franklin.
Resuspended in 0.75 mL before plating.  Then plated 200 uL on one half of each plate and 50 uL on the other half.  Placed in incubator at 6:00 pm.
Also note that because we needed a shaking incubator set to 30 degrees C, we were ultimately forced to turn down the temperature on the incubator that already contained the E. coli cultures from last night (promoters and shuttle vector).  This was done around 10 hr and 30 min into their incubation time.  The result will be a slowing of cell growth for these cultures, possibly also bringing them out of log phase.  As a result, it will be necessary to make glycerol stocks and to do minipreps a lot later tonight.  Note that the incubator was switched back to 37 degrees around the 14th hour.  The cells were then taken out of the incubator around 16 hr and 35 min after they had been put in.  Note that pRS303 showed no signs of growth.  Hopefully this is just due to the fact that the colony on the plate was not properly collected, instead of a sign of contamination on the plates.  In any case, successfully glycerol stocked p413_PCYC, p414_PADH, and p413_PTEF, making two stocks for each part from the same culture.
Minipreps of promoters:
Sample ID User ID Date  Time  ng/ul  A260  A280  260/280  260/230  Constant  Cursor Pos. Cursor abs. 340 raw
TEF Default 7/4/2008 5:47 PM 271.31 5.426 2.852 1.90 2.26 50.00 230 2.396 0.021
CYC Default 7/4/2008 5:49 PM 271.71 5.434 2.850 1.91 2.27 50.00 230 2.389 0.011
ADH Default 7/4/2008 5:49 PM 249.90 4.998 2.614 1.91 2.26 50.00 230 2.213 0.028
New record for [DNA].
7/7/08
--------------------------------------------------
x-prepare for more yeast transformations (grow yeast strains tonight)
x-if possible, make more dropout plates and media
x-order adenine, 96 well plates
Plan out construction strategy with four yeast shuttle vectors
email Sindi
x-email Pam Silver
x-order primers
x-contact Richard Freiman
look up properties of W303 yeast strain in comparison to the one Ajo-Franklin used (email Laney?)
make sure that CFP will have comparable stability to
look up composition information about our yeast media (to see if it could be causing leakiness)
innoculate liquid cultures of pRS303, GPD, (and pRS425, John's part) tonight in order to glycerol stock and miniprep tomorrow
-big culture of pRS303 because we will potentially need a lot of it
add parts to registry
--------------------------------------------------
Came in today to find that all transformations were successful!  All four plates had yeast colonies, which appeared red to the naked eye.  This might be good, except that one cannot determine at the present time whether the red appearance stems from the production of mCherry or from another biochemical process (Richard Bennett mentioned that the W303 strain natrually turns red in the absence of adenine).  If the former, this would be bad, as it would indicate an issue with either our media, our strain, or leakiness in our promoters.  Looked at plates under fluorescent microscope.  Note that there is a distinction between the number of visible colonies in brightfield and those seen in either yellow or green filter (for RFP and YFP, respectively).  RFP and YFP fluorescence was observed for the null repressor (Z0411).  For this part, one would expect to see YFP because it is under control of a constitutively active promoter.  The YFP fluorescence was significantly less pronounced for the activators, which is also according to expectations.  Should remember to take pictures of the fluorescence later in the week.  However, note that there was significant red fluorescence in all colonies, which is against our expectations seeing as the plates had a fair amount of methionine in them (meaning that the promoter driving expression of the repressors, PMET25, would be inhibited) and no galactose (meaning that there is no inducer for the promoter driving expression of the activators, PGAL1).  Thus, one might expect lower levels of RFP for all plates, low levels of YFP for the activator networks and high levels of YFP for the repressor networks.  These discrepancies could stem from the fact that there is leakiness of the PGAL1 promoters (explaining too much RFP) or the mCYC promoters (explaining too much YFP in the activator plates), that the yeast strain used could have different pathways that affect the activity of the promoters in a different way than originally observed by Ajo-Franklin, or that the media itself contains something that could affect the activity of the promoters.  Also, is the Bacto Agar completely free of both amino acids and galactose/methionine?  If not, it could contribute to changing what we observe on the plates, but would have no effect on the liquid cultures.  The only way it would have an effect is if the Agar somehow complemented the auxotrophies itself, thus allowing cells to grow without being properly transformed.  At the very least, this would explain the difference between the brightfield and fluorescent views. Another possibility to consider is autofluorescence.  What is interesting is the difference in the number of observable colonies between brightfield and fluorescent modes.  One might expect that all colonies on a plate would have the same fluorescence because, in order to grow, they all need to complement the correct auxotrophies as those contained in the shuttle vectors.  That there is a discrepancy here as well might even point to how we linearized the DNA.  It might be worthwhile to run a gel on the linearization digests in case we suspect that they did not cut as we wanted them to.
Description of Bacto Agar:
"BactoAgar is optimized for beneficial calcium and magne-
sium content. Detrimental ions such as iron and copper are
reduced.Bacto Agar is recommended for clinical applications,
auxotrophic studies, bacterial and yeast transformation
studies and bacterial molecular genetics applications."
It seems we don't have a problem using BactoAgar.  This quote came from http://www.bd.com/ds/technicalCenter/inserts/Agars.pdf
Information about the media formulation:
"Complete Synthetic Defined Media contains Yeast Nitrogen Base (1.7g/L), ammonium sulfate (5g/L), dextrose (20g/L) and a Complete Supplement Mixture (CSM) of amino acids minus those indicated in the product name." This is the only information that can be gleaned from their website.
This quote came from http://sunrisescience.com/pages/ystmedia_sc_sd.html
Plan for Test #1:  To test the activity of our activators and repressors, we would grow out an overnight culture of yeast cells that have successfully  been transformed with both a synthetic transcription factor and its respective reporter.  Thee cultures would have an amount of galactose and methionine that will remain the same for all subsequent tests.  We would then count the cells in these overnight cultures in the morning, dilute them, split them up into well plates with an equal number of cells in each plate in media with differing amounts of galactose or methionine (water would be added as a control and to maintain the same volume in each well).  We could also centrifuge the cells before putting them into these wells to remove traces of the previous media.  Perhaps two washes would be sufficiently rigorous to ensure that the concentration of input in the media is what we would want it to be with high confidence.  Then we would grow the cells in these wells for 4 hours or so to get them back into log phase in their respective media.  After this, we would take their fluorescence measurement using a plate reader, perhaps taking out samples at different time points during the log phase.
Realized that regarding further yeast transformations for Test #1 that we will have to wait until we insert Z0842 (LexA activable reporter) and Z0507 (Zif268-HIV repressor reporter) into pRS306 in order to actually get them into yeast cells.  Thus, we cannot run basic tests of the LexA activator or the Zif repressor until this is done.  We already have two of the activators transformed into yeast.  This leaves four more possible transformations that we can do right away:
LexA repressor + LexA repressor reporter (Z0412 + Z0418)
null repressor + LexA repressor reporter (Z0411 + Z0418)
YY1 repressor + YY1 repressor reporter (Z0416 + Z0511)
null repressor + YY1 repressor reporter (Z0411 + Z0511)
Again, the null repressor has the repression domain but lacks any binding sites, so we expect it to have no activity on the reporter.  We will attempt to include it in every test where we are looking at activity of repressors as a control.  It might be nice to make an activator version of this, too.
Thus, we decided to make some Ura, Trp dropout plates today.  In 250 mL water, dissolved 6.815 g of SD complete dropout mix, 5 g BactoAgar, 0.5 g histidine, 0.5 g methionine, and 1 g leucine.
BactoAgar:
histidine:
methionine:
leucine:
Yeast cultures went into incubator at 3:45 pm. Will take them out in the morning to use for transformations of the remaining parts.
In desigining Test #2, should compare PMET25 activity to that of our constitutive promoters.  Is there literature outlining this?  Yes, the two papers are both by the same group.
"Regulatable Promoters of S. cerevisiae: comparison of transcriptional activity and their use for heterologous activity", Mumberg et al, Nucleic Acids Research (1994)
"Yeast vectors for the controlled expression of heterologous proteins in different genetic backgrounds", Mumberg et al., Gene (1995).
Note that based upon the graphs it seems that the PTEF promoter is the best choice for PMET25.  It appears that for two different levels of activity (low vs. high copy number plasmids), TEF has constitutive activity that is higher than the highest recorded activity of PMET25.  Ideally, we would want a promoter that has constitutive level in the middle of PMET25's operating range, so that we could alter PMET25's activity above and below this value.  We were told that PGAL1 has a poor degree of tunability in comparison with PMET25, which is why we decided to select PMET25 to drive expression of our activator. Mumberg's results were not recorded in our particular strain of yeast (W303) so it might be beneficial to continue searching for promoter comparisons done in this type of yeast.
We also still need to look into location of binding sites with respect to affecting transcriptional activity.  We have contaced Richard Freiman with hopes that he might provide some insight.
PCR Primers:
As pADH is already in the registry, we're not PCRing it.
pCYC fw: GTTTCTT CG AAT TCG CGG CCG CTT CTA GAG agcgttggttggtggatcaagcc
pCYC rv: GTTTCTT CC TGC AGC GGC CGC TAC TAG TA tattaatttagtgtgtgtatttgtgtttgtgtgtc
pTEF fw: GTTTCTT CG AAT TCG CGG CCG CTT CTA GAG attacccataaggttgtttgtgacgg
pTEF rv: GTTTCTT CC TGC AGC GGC CGC TAC TAG TA gtttagttaattatagttcgttgaccg
pGPD fw: GTTTCTT CG AAT TCG CGG CCG CTT CTA GAG tcagttcgagtttatcattatcaatactgcc
pGPD rv: GTTTCTT CC TGC AGC GGC CGC TAC TAG TA tttgtttgtttatgtgtgtttattcgaaac
Also ordered the following primers to biobrick the activators and
Activator fw: GTTTCTT C GAATTC GCGGCCGC  T  TCTAGA  G ctagacccgccgccacc
Activator rv: GTT TCT TC + c tgc agc ggc cgc tac tag
mCYC fw: GTTTCTT C GAATTC GCGGCCGC  T  TCTAGA  G gcagatccgccaggcgtg
mCYC rv: GTTTCTTCCTGCAGCGGCCGCTACTAGTAgttattaatttagtgtgtgtatttg
mCYC + binding sites fw: GTTTCTTcgaattcgcggccgcatc
Note that the mCYC rv can be used for PCR involving the mCYC + binding sites since the binding sites all occur before the mCYC promoter.
Note: We decided that because it will be so much work to run the initial test of the LexA activator that we will simply not bother using it.  We will only be using Z0842 to PCR out the LexA binding sites attached to the mCYC promoter.
Innoculated cultures with pRS303, pRS425, and GPD in preparation for making glycerol stocks and miniprepping tomorrow.
7/8/08
--------------------------------------------------
Plan out construction strategy with four yeast shuttle vectors
email Sindi about choosing the right parts for our design
do yeast transformations of remaining repressors
--------------------------------------------------
Yeast transformations round 2:
5mL W3031a culture OD: .300
added to 50mL YPD in the baffler flask
In Richard Bennet's shaking incubator at 11 AM
Out at 3.
LexA repressor + LexA repressor reporter (Z0412 + Z0418)
null repressor + LexA repressor reporter (Z0411 + Z0418)
YY1 repressor + YY1 repressor reporter (Z0416 + Z0511)
null repressor + YY1 repressor reporter (Z0411 + Z0511)
Final resuspension step was in 0.5 mL of water, as opposed to 1 mL.  This was due to our cell count after the 4 hr incubation period.  Transformation mixtures in 42C heat shock bath at 4:17.  Took them out around .
Plated them out and put in the incubator around.  Split each plate into halves.  On one half, used 10 uL of transformation liquid.  On the other, used 100 uL of transformation liquid.
Also, since there was contamination in the LB + AMP control from last night, we decided to throw out the overnight cultures for pRS303, pRS425, and GPD.  We innoculated fresh cultures  (30 mL of each) and put them in the shaking incubator around .
Received a response email from Dr. David Drubin at Harvard, who was co-author with Dr. Caroline Ajo-Franklin on the paper detailing the memory device.  He indicated that they had more intermediates that we could consider using.  This would include a biofusion version of CFP.  Will ask him if he has a list of parts that we could look at.
7/9/08
--------------------------------------------------
x-Buy cultcha tubes!
x-Miniprepz
--------------------------------------------------
Took cultures out of incubator around noon.  They had been in the incubator for 18 hr and 7 min at that point.  Made two glycerol stocks apiece of pRS303, pRS425, and GPD.  Also did minipreps of the three parts.  There appeared to be a sizable amount of DNA, seeing as the resultant measurments were as follows:
Sample ID User ID    Date     Time       ng/ul   A260   A280   260/280  260/230 Constant Cursor Pos. Cursor abs.
p413_GPD Default 7/9/2008 2:24 PM   357.77  7.155  3.815    1.88           2.31       50.00        230          3.095
pRS425      Default 7/9/2008 2:26 PM  328.17  6.563  3.489    1.88           2.30       50.00        230          2.853
pRS303      Default 7/9/2008 2:27 PM  311.76  6.235  3.312    1.88           2.28       50.00        230          2.731
Note that the DNA was eluted in 75 uL of EB.
Sent an email to David Drubin.  We asked for an opportunity to meet with him next week to discuss our project, to learn more about what work has been going on at Harvard, and to possibly obtain some more parts.  This is the request for parts that we sent:
1) repressing transcription factors with CFPx2- Ideally HIVzif, Gli1, and/or Lex A on a integrating shuttle vector
2) CFP and CFPx2
3) repressor & activator intermediates that lack fluorescent tags
4) null activator consisting of the fusion transcription factor with fluorescence but without a DNA binding domain to act as a negative control
5) any constitutive yeast promoters- ideally one with activity somewhere in the range of the inducible MET25 promoter
6) constitutive promoters with binding sites for the transcription factors (as seen in the repressible reporter, for example)
7) mCYC or other minimal promoter
8) mCYC with binding sites for the transcription factors
Note that we recently discovered that Harvard might have some yeast strains that already contain the parts we choose to utilize.
7/10/08
--------------------------------------------------
before 5pm: innoculate yeast cultures of successful transformations from 7/4/08 and later in order to make glycerol stocks and also to run tests tomorrow
--------------------------------------------------
Checked on the plated transformations from the second round- all were successful, and there was no growth of the negative control.
Devised a plan for setting up the plate test:
Make SD-Trp-Ura media with a 2 mM (2000 microM) concentration of methionine.  This will be easily diluted to the various concentrations required (25 uM, 50 uM, 100 uM, 200 uM, 350 uM, 500 uM, and 1000 uM).
M.W. (Met)= 149.2 g/mol
thus, 2 mM solution has .2984 g/L
250 mL Recipe for SD-Trp-Ura with 2mM Met
6.815g SD Dropout media powder
.5g Histidine
1g Leucine
.0746 g Methionine
Test #1:  We have decided to use a 96 well plate reader to measure the fluorescence emitted by our various parts.  For the time being, it appears that this will be sufficient.
The protocol is as follows:
Apparently the doubling time of W303_1a, the strain used for all of the transformations, is 2.8 h.  This was gleaned from the following source: http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1140095.  Other sources appear to range between 95 min and about 3 h, depending upon the media.
We have decided to sweep across different levels of methionine.  For the activators, we will need to invetigate as to whether the presence of raffinose is necessary, especially since this sugar was used in the yeast memory paper by Caroline Ajo-Franklin.
Made cultures of the following parts:
W303_1a, to be used as one of the controls tomorrow, grown in 5 mL YPD
Gli1 activator + Gli1 activable reporter (Z0277 + Z0739), grown in 5 mL trp- ura- SD dropout media
Gli1 repressor + Gli1 repressor reporter (Z0415 + Z0510), grown in 5 mL trp- ura- SD dropout media
null repressor + Gli1 repressor reporter (Z0411 + Z0510), grown in 2 mL leu- ura- SD dropout media
Zif268-HIV activator + Zif268-HIV activable reporter (Z0267 + Z0637), grown in 2 mL leu- ura- SD dropout media
Also made controls for YPD, trp- ura- dropout media, and leu- ura- dropout media.  The first three will be involved in the test tomorrow.  Will be making glycerol stocks of all of these strains (except W303_1a) tomorrow.  Note that the leu- ura- dropout media used for the activator + reporter parts appeared to have a slight amount of cloudiness, which indicates possible contamination.  However, decided to use it anyway just in case to see whether there will be noticeable growth in the control in the incubator.  Put all cultures into the incubator at 4:52 pm tomorrow, except for the W303 strain which went in at 5:05 pm.
Stock solutions of Uracil, Histidine, Tryptophan, and Leucine:
for 100x batches: 2 g of Uracil in 10 mL, 2 g of Histidine in 10 mL, 2 g Tryptophan in 10 mL, and 4 g Leucine in 10 mL
Good source for fluorometry: http://openwetware.org/wiki/Endy:Victor3_plate_reader#Programming_a_basic_protocol
Note that the Leu-,Ura- media for activators looked slightly contaminated, with some cloudiness at the bottom.  Decided to use it anyway just in case.  Will have to pay close attention to the control.
Approximate maximum excitation and emission for the types of fluorescence we are using: mCherry - 587 ex, 610 em and EYFP - 514 ex, 527 em (http://www.microscopyu.com/articles/livecellimaging/fpintro.html)
7/11/08
--------------------------------------------------
Glycerol stocks of yeast parts
Innoculate cultures of transformed yeasties.
Test #1 on parts Z0267, Z0277, and Z0415
--------------------------------------------------
Unfortunately it appears that two of the yeast cultures did not grow.  These are the ZifA + ZifA reporter strain and also the null + Gli1R reporter.  The second one is more sever, given that it was intended for use today.  It is likely that the colony was simply missed during innoculation due to its small size.  Decided to use Gli1A + Gli1A reporter instead.  This is simply a test run of our protocol.
Took out of incubator around 9:45 am.  Diluted by adding 150 uL of each culture to 5 mL of fresh media.  Note that this is simply to get them back into log phase, so that it may not be important to make sure that the cell concentrations are identical for all three.
Put into incbuator again at 10:18 am.  Took them out around 1:31 pm.  Observed some apparent growth, but not a significant amount.
OD measurements: GliR had a range between 0.004 and 0.011.  For five measurements (repeated measurements from two samples), it had an average of 0.0074.  Put 100 uL of these cells directly into each well devoted towards testing the Gli1 repressor.
For W3031_a, the range was between 0.012 and 0.030.  For five measurements (repeated measurements from two samples), it had an average of 0.0202.  Diluted these cells to match the concentration of the Gli1R cells.  Added 2.5 mL of this culture to 4.3 mL fresh culture.  Then added 100 uL of this mixture to each well devoted to testing W303.  For wells that contain media alone, we added an additional 100 uL of the respective media in each well.
First plate reading done at 3:15 pm.  Second reading done at 4:18 pm.  Final reading taken around 5:07 pm.  Discovered some interesting patterns in fluorescence.  Saved all plate reader results in the Plate Reader folder on the iGEM computer.  However, it was quite apparent that we did not use enough cells in each well.  Our reasoning for this is that the graphs for plain media as compared to are almost identical.  To correct this problem, we propose adding more cells to the   We also discovered that the YPD we use in our lab has a significant amount of fluorescence in approximately the same range as EYFP.  Of course, we do not plan to use this media from now on, but it is quite good to know this.
Another important point concerning media is that the dropout media we have been using apparently contains glucose as its carbon source.  This presents a problem for all constructs that are under the control of the pGAL1 promoter (including all of our activators) because glucose very strongly represses the activity of this promoter.  It may also explain some of the results we saw today, seeing as all of the reporters specific to repressors are actually under control of the pGAL1 promoter.  We had originally been under the impression that all reporters were under control of the mCYC promoter.  However, while this would explain why we do not see any difference between the YFP of wells with transformed yeast and the YFP of wells with just media, it does not explain the same observation for mCherry.  Thus, we still maintain that we did not use enough yeast cells per well.
Growing e.coli cultures of the blank vectors for miniprep tomorrow:
pRS304
pRS305
pRS306
Approximately 30 mL cultures.  Innoculated directly from glycerol stocks, colony #1 in all cases.  Placed in incubator at a slant at 230 rpm at roughly 7:25 pm.  Should check on them tomorrow around 9:25 am.
7/12/08
--------------------------------------------------
Miniprep plasmids pRS304*, pRS305, and pRS306
Linearize DNA (what DNA?)
Discuss plans for next round of transformations to prepare for Test #1, Test #2, etc.
--------------------------------------------------
Came in around 12:45 pm.  Took cultures out of incbuator after 17 hr and 41 min.  Realized that we had neglected to include a negative control for last night's cultures.  Should be ok, considering that the bottle of LB from which the cultures were made is still free of contamination, and due to the fact that ampcillin stock solution was also included in the cultures.  Still, will continue on with the minipreps with full knowledge of possible contamination.  Also, placed a control tube of LB into the incubator just to ensure that there is no contamination.
Eluted in 75 uL of EB.  The results from the miniprep are as follows:
Sample ID  User ID  Date  Time  ng/ul  A260  A280  260/280  260/230  Constant  Cursor Pos.  Cursor abs.  340 raw
pRS304*   Default    7/12/2008    2:55 PM   105.12  2.102  1.042  2.02  2.54  50  230  0.829  0.061
pRS304*   Default    7/12/2008    2:56 PM   109.26  2.185  1.09  2.01  2.52  50  230  0.868  0.075
pRS305   Default    7/12/2008    2:57 PM   112.01  2.24  1.114  2.01  2.93  50  230  0.765  0.009
pRS305   Default    7/12/2008    2:57 PM   116.9  2.338  1.166  2.01  2.88  50  230  0.812  0.022
pRS306   Default    7/12/2008    2:58 PM   135.26  2.705  1.357  1.99  2.73  50  230  0.99  0.018
pRS306   Default    7/12/2008    2:59 PM   139.88  2.798  1.39  2.01  2.71  50  230  1.033  0.017
For next week, this is the list of things we need to accomplish:
1. Order new dropout media that has raffinose as its carbon source rather than glucose
2. Once the new media has arrived, we need to prepare the proper double dropout mix for media and for plates
3. Glycerol stock successful transformations from 7/4 and from 7/8
4. Additional double transformations of parts
5. (Most importantly) we need to run Test #1 again using what we learned yesterday.  Of course, 1-4 simply prepare us for re-doing Test #1
Concerning our next round of transformations, we have decided to focus on preparing for the test of our three available repressors.  This include GliR (Z0415), LexR (Z0412), and YY1R (Z0416).  One of the main reasons for focusing on these parts is that we currently lack a null activator part, so we cannot complete rigorous tests of our parts individually.  For testing these three parts, we will need to complete the following transformations:
GliR
1. W303 transformed with pRS304* and pRS306
2. null repressor (Z0411) + pRS306
3. pRS304* + GliR reporter (Z0510)
4. GliR (Z0415) + pRS306
5. nullR + GliR reporter**
6. GliR + GliR reporter**
LexR
1. W303 transformed with pRS304* and pRS306
2. null repressor (Z0411) + pRS306
7. pRS304* + LexR reporter (Z0418)
8. LexR (Z0412) + pRS306
9. nullR + LexR reporter**
10. LexR + LexR reporter**
YY1
1. W303 transformed with pRS304* and pRS306
2. null repressor (Z0411) + pRS306
11. pRS304* + YY1R reporter (Z0511)
12. YY1R (Z0416) + pRS306
13. nullR + YY1R reporter**
14. YY1R + YY1R reporter**
** Indicates transformations that have already been accomplished.
Note the double transformation scheme, which allows us to use the same trp- ura- dropout media for testing all transformations, even W303 yeast cells that lack synthetic constructs.  Also note that 1 and 2 have been included for all three.  They will be needed to show the
So, we will be doing the following transformations this week:
1. W303 transformed with pRS304* and pRS306
2. null repressor (Z0411) + pRS306
3. pRS304* + GliR reporter (Z0510)
4. GliR (Z0415) + pRS306
7. pRS304* + LexR reporter (Z0418)
8. LexR (Z0412) + pRS306
11. pRS304* + YY1R reporter (Z0511)
12. YY1R (Z0416) + pRS306
13. YY1R (Z0416) into [YY1A & YY1A Report]
14. GliR (Z0415) into [YY1A & YY1A Report]
15. ZifR (Z0421) into [YY1A & YY1A Report]
16. GliR (Z0415) into [GliA & GliA Report]
17. YY1R (Z0416) into [GliA & GliA Report]
18. ZifR (Z0421) into [GliA & GliA Report]
19. ZifR (Z0421) into [ZifA & ZifA Report]
20. YY1R (Z0416) into [ZifA & ZifA Report]
21. GliR (Z0415) into [ZifA & ZifA Report]
This will require 8 trp- ura- plates (not including an additional 2 plates for controls), as well as 9 triple dropout plates. This will also probably require growing two separate batches of W303 yeast cells for the transformations.  Since in the past we have managed to get enough cells for four transformations, we will likely need to double the recipe.
Plasmid DNA required:
pRS304* - x4 = 68 uL - do 100 uL
pRS306 - x5 = 85 uL - do 100 uL
Z0411 (304*)- x1 = 17 uL- do 25 uL
Z0510 (306) - x1 = 17 uL - do 25 uL
Z0415 (304*)- x4 = 68 uL - have 1, do 50 uL * Didn't have enough miniprep to use 20 ul dna- used 15 ul and 5 ul water
Z0418 (306)- x1 = 17uL - do 25 uL
Z0412 (304*)- x1 = 17uL - done
Z0511 (306)- x1 = 17 uL - do 25 uL
Z0416 (304*)- x4 = 68 uL have 1, do 50 uL  * Didn't have enough miniprep to use 20 ul dna- used 10 ul and 10 ul water
Z0421 (304*)- x3 = 51 uL - do 50 uL
Total= 450 uL, 9 50 uL reactions make 500 uL mix
All use PstI
Buffer H
50uL Buffer H
5 uL BSA
235 uL H20
10uL ( or less?) enzyme
10 uL DNA for each 25 uL reaction
Since a restriction digest is 50 uL, and we will be using 17 uL per transformation, we will need to do two linearization reactions for pRS304* and pRS306 and one linearization reaction for all of the rest of the parts.
In the fridge we appear to have enough linearized DNA for Z0415, Z0412, and Z0416, seeing as we have at least 17 uL of each.  We need to do minipreps for Z0411, Z0418, Z0510, and Z0511.  These should be grown starting tomorrow night.
We also need to figure out the arrangement on the 96-well plates.  It will require either reducing the number of strains of yeast tested against each other on a single plate or reducing the number of methionine concentrations.  It appears to be a good idea to do the former first so that we can observe a good sweep of activity across several concentrations.
7/13/08
--------------------------------------------------
Innoculated overnight cultures of Z0411, Z0418, Z0510, and Z0511 in preparation for minipreps.
Cultures in the incubator at 8:15 PM
7/14/08
--------------------------------------------------
x-Miniprep of cultures from last night
x-Organize glycerol stocks
Order or obtain dropout media for raffinose
Put together a list of q's for David
x-Innoculate overnight cultures of transformed yeast parts from 7/4/04 for glycerol.
--------------------------------------------------
Questions to consider:
Cloning strategy using a Gal2 knockout strain:  is there a knockout vector that we can use to make W303_a with this knockout as well?
Population fluorescence vs. individual fluorescence: A population will demonstrate a graded fluorescent output even if the promoter has an all-or-none response.  Thus, won't we want to show that we get suitable activity on an individal cell level?
Removed from incubator at 12:30PM after 16hr and 7 min of growth and miniprepped
Results from the miniprep.  Note that duplicate measurements were taken off of the same sample for each part.
Sample ID  User ID  Date  Time  ng/ul  A260  A280  260/280  260/230  Constant  Cursor Pos.  Cursor abs.  340 raw
Z0511   Default    7/14/2008    3:34 PM   171.97  3.439  1.755  1.96  2.53  50  230  1.359  0.001
Z0511   Default    7/14/2008    3:34 PM   179.13  3.583  1.834  1.95  2.51  50  230  1.43  0.011
Z0510   Default    7/14/2008    3:35 PM   55.89  1.118  0.561  1.99  2.58  50  230  0.434  -0.005
Z0510   Default    7/14/2008    3:35 PM   58.55  1.171  0.581  2.02  2.56  50  230  0.458  0.007
Z0418   Default    7/14/2008    3:37 PM   137.27  2.745  1.388  1.98  2.59  50  230  1.058  -0.006
Z0418   Default    7/14/2008    3:37 PM   142.41  2.848  1.417  2.01  2.59  50  230  1.099  0.015
Z0411   Default    7/14/2008    3:38 PM   145.26  2.905  1.465  1.98  2.59  50  230  1.122  0.002
Z0411   Default    7/14/2008    3:38 PM   151.26  3.025  1.534  1.97  2.57  50  230  1.175  0.018
Innoculated cultures for the following parts:
LexA repressor + LexA repressor reporter (Z0412 + Z0418)
null repressor + LexA repressor reporter (Z0411 + Z0418)
YY1 repressor + YY1 repressor reporter (Z0416 + Z0511)
null repressor + YY1 repressor reporter (Z0411 + Z0511)
null repressor + Gli1 repressor reporter (Z0411 + Z0510)
Zif268-HIV activator + Zif268-HIV activable reporter (Z0267 + Z0637)
Put them in the incubator at 5:03 pm.  Will make glycerol stocks of these parts tomorrow.
7/15/08
--------------------------------------------------
Make glycerol stocks
Meeting with David Drubin
--------------------------------------------------
Meeting with David went well, though he did not have too much additional information that Caroline had not already provided to us.  It appears that no one really knows where the placement of the binding sites for each promoter should be.  He did convince us that Cell-ID is a necessary approach for our project.
7/16/08
--------------------------------------------------
Transformation of the following parts:
1) CFPx2 [CFP-CFP]
TFs w/o Fluor tags
2) Activator Lex [266] (w/ NLS-term)
3) activator HIV  [259] (w/ NLS-term)
4) repressor Lex [323]
5) repressor Gli [329]
6) null activator [256]
7) pAdh1 [54]
8) mCYC1 [P0661]
9) LexA op8 mCYC kozak [DD21]
10) LexA op8 mCYC [1A]
11) LexA activatable reporter in pRS306 [DD25]
Things we need for test constructs:
reverse primer for TEF w/o TATA box
2xYFP+Terminator  for reporting repressable const. promoter activity
--------------------------------------------------
PCR of the following parts:
1. pCYC promoter
2. pTEF promoter
3. pGPD promoter
4. ZifA minus GAL1 promoter (from Z0267 )
5. Gli1A minus GAL1 promoter (from Z0277)
6. LexA minus GAL1 promoter (from Z0279)
7. mCYC + ZifA binding sites (from Z0637)
8. mCYC + Gli1A binding sites (from Z0739)
9. mCYC + LexA binding sites (from Z0842)  Note that there was an issue with the binding sites for the part
Plus the following cotrols:
10. pCYC forward primer + reverse primer (no DNA)
11. pTEF forward primer + reverse primer (no DNA)
12. pGPD forward primer + reverse primer (no DNA)
13. Activator forward primer + reverse primer (no DNA)
14. mCYC +BS forward primer + reverse primer (no DNA)
15. Positive control from Taq kit
The recipe for a PCR reaction is as follows:
36.8 uL H2O
5 uL 10x buffer
3 uL 25 mM MgCl2
1.0 uL 10mM dNTP
1.0 uL forward primer
1.0 uL reverse primer
0.2 uL Taq
2.0 uL template DNA @ .5 ng/ul
First, need to dilute primers and also DNA so that they are in the same concentrations.
pCYC forward primer: 24.51 nmol / 0.5 nmol/uL = 49.02 uL; Added 49 uL => 500.2 uM
pCYC reverse primer: 20.09 nmol / 0.5 nmol/uL = 40.18 uL; Added 40.2 uL => 499.8 uM
pTEF forward primer: 19.89 nmol / 0.5 nmol/uL = 39.78 uL; Added 39.8 uL => 499.7 uM
pTEF reverse primer: 20.22 nmol / 0.5 nmol/uL = 40.44 uL; Added 40.4 uL => 500.5 uM
pGPD forward primer: 29.25 nmol / 0.5 nmol/uL = 58.5 uL; Added 58.5 uL => 500 uM
pGPD reverse primer: 25.16 nmol / 0.5 nmol/uL = 50.32 uL; Added 50.3 uL => 500.2 uM
mCYC + BS forward primer: 44.53 nmol / 0.5 nmol/uL = 89.06 uL; Added 89 uL => 500.3 uM
mCYC + BS reverse primer: 20.48 nmol / 0.5 nmol/uL = 40.96 uL;  ACCIDENTALLY ADDED 89 uL!!! => 230.1 uM
Activator forward primer: 26.85 nmol / 0.5 nmol/uL = 53.7 uL; Added 53.7 uL => 500 uM
Activator reverse primer: 28.93 nmol / 0.5 nmol/uL = 57.86 uL; Added 57.9 uL => 499.7 uM
If we add the calculated amount of water to each tube, we will get a final concentration of 500uM of primer.
We can then put 5 uL of each into 120 uL of 10mM Tris-HCl to make a concentration of 20 uM for use for PCR.  For mCYC + BS reverse primer, one will add 5 uL to 52.525 uL Tris H-Cl instead. Added 52.5 uL.
 
plasmid DNA concentration
pCYC: 271.71 ng/uL => 271.71 ng/uL * 1 uL = 271.71 ng => 271.71 ng / 0.025 ng/uL = 10868.4 uL
pTEF: 271.31 ng/uL => 271.31 ng/uL * 1 uL = 271.31 ng => 271.31 ng / 0.025 ng/uL = 10852.4 uL
pGPD: 357.77 ng/uL => 357.77 ng/uL * 1 uL = 357.77 ng => 357.77 ng / 0.025 ng/uL = 14310.8 uL
Z0267: (from 6/20) 135.8 ng/uL => 135.8 ng/uL * 1 uL = 135.8 ng => 135.8 ng / 0.025 ng/uL = 5432 uL
Z0277: (from 6/20) 123.2 ng/uL => 123.2 ng/uL * 1 uL = 123.2 ng => 123.2 ng / 0.025 ng/uL = 4928 uL
Z0279: (from 6/21) 109.2 ng/uL => 109.2 ng/uL * 1 uL = 109.2 ng => 109.2 ng / 0.025 ng/uL = 4368 uL
Z0637: (from 6/20) 123.4 ng/uL => 123.4 ng/uL * 1 uL = 123.4 ng => 123.4 ng / 0.025 ng/uL = 4936 uL
Z0739: (from 6/20) 108.4 ng/uL => 108.4 ng/uL * 1 uL = 108.4 ng => 108.4 ng / 0.025 ng/uL = 4336 uL
Z0842: (from 6/28) 40.4 ng/uL => 40.4 ng/uL * 1 uL = 40.4 ng => 40.4 ng / 0.025 ng/uL = 1616 uL
For plasmid DNA, the NEB Taq PCR instruction manual indicates that we need 0.1-1 ng/mL of DNA in the final 50 uL reaction.  This means 0.05 ng of DNA for 1 ng/mL in 50 uL.  If 2.0 uL of plasmid DNA is added to each PCR reaction, this means that the desired concentration is 0.05 ng / 2 uL = 0.025 ng /uL.
Because we read online that one should use 1 ng of template for the entire reaction, we ended up diluting the plasmid in each case to 0.5 ng/uL and using 2 uL of each per reaction.
7/17/08
--------------------------------------------------
PCR of promoters and activators
overnight culture of parts from Silver Lab that were transformed on 7/16
 
--------------------------------------------------
Looked at plates to discover that the transformations had barely worked, if at all.  Observed colonies on the following plates: positive control 256 (1 colony), 54, P0661, CFPx2 (1 colony), and 259 (1 colony).  Approximately 16 hr 20 min in the incubator.  Since the transformation mix from yesterday had been saved in the fridge overnight, decided to add some more LB to this and then to replate it after an incubation perdiod.  Added 200 uL of LB to each and then placed in incubator with shaking for 4 hr and 8 min. Will prioritize replating the following parts: 323, 266, 329, DD21, DD25, and 1A.  On a fresh plate for each part, added 50 uL to half the plate and 200 uL to the other. Put in incubator at 4:57 pm.
Also decided to replate the parts that only had a single colony per plate.  Did a streak from that one colony for half of the new plate and use 200 uL of transformation liquid from yesterday on the other half.  Put the in the incubator around 5:18 pm, 21 min. after the first plates went in.
Also, streaked out pSB1AK3 from glycerol stock onto a plate.  This vector will be used to carry the results from our PCR.
Mixed up PCR concentrations.  Some sources indicated that it was necessary for there to be 1 ng of DNA in the entire mix, so we diluted the plasmid DNA to 0.5 ng/uL in all cases and added 2 uL of each to its respective reaction to serve as the template.  These were also suspended in 10 mM tris HCl.
Ran PCR according to the following program:
1. 95°C for 40 sec
2. 95°C for 15 sec
3. 50°C for 30 sec
4. 72°C for 2:30 min
5. repeat steps 2-4 29 times
6. 72°C for 5 min
7. hold at 4°C
Also, in addition to replating all parts received from the Silver lab, made 30 mL cultures of the parts that had at least one colony.  This is for glycerol stock and miniprep tomorrow.  The parts innoculated include 54, 256, 259, CFPx2, and P0661.  Placed in incubator at 12:19 am on 7/18/08 at a slant at 250 rpm.
7/18/08
--------------------------------------------------
Glycerol stock and miniprep of 54, 256, 259, CFPx2, and P0661
x-innoculate cultures of 323, 266, 32, DD21, DD25, and 1A for glycerol stock and minipreps tomorrow
x-replate pSB1AK3, 266
x-Run gel of PCR products from yesterday
x-Make LB Agar + AMP plates
Document lab notebook
--------------------------------------------------
Took plates out of incubator after 18 hr and 34 min.  All parts that had been replated last night had at least one colony on them.  This includes 323, 266, 329, DD21, DD25, 1A, 256, CFPx2, and 259.  The exception is pSB1AK3, whose cells were far too dense to get a single colony.  Decided to replate 266 and also pSB1AK3.  Put plates into incubator at 6:05 pm.
Also, innoculated overnight cultures in preparation for more glycerol stocks and minipreps tomorrow.  This is for the same parts listed above, with the exception of pSB1AK3, which had no distinct colonies.
Ran a gel of all PCR products from yesterday.  Ran about 5 uL of each, added to 4 uL of water and 1 uL of loading dye. The gel was 1% gel, a large one with 26 combs of 9 mm spacing.  For this, mixed 100 mL 0.5x TBE, 10 uL SYBR Safe, and 1 g Agarose and poured it into the large electrophoresis device.  The gel was running for about 4 hours.
Imaging indicated that none of the PCR reactions had worked, apparently not even the positive control.  Could only see bands for the three ladders included, which were the 100 bp, the 1 kbp, and the 2-log ladders.  Given this finding, one might suggest that something went wrong either with the PCR itself or with the gel.  It seems unlikely that something was wrong with the gel given that a restriction digest for a different project run on the same gel appeared to have a distinct band (for a piece of DNA expected to be 1-2 kbp).
Made more LB + agar plates.  Some of this spilled out in the autoclave.  Estimated that there are still 800 mL remaining, so added 800 uL of ampicillin stock solution.
Made 20 plates.
A talk with Jamie convinced us that we should run the initial plate reader tests we had considered of the activator + reporter and repressor + repoter constructs.  These should be done in parallel to the construction of our real parts.
In terms of our ligations, we need to determine which promoters we will be using, which sets of binding sites, etc.  We need to determine a good inducible promoter for the entire device.  If we choose pGAL1, we will need to make it more tunable, which will require the yeast strain from Smolke that has a knockout.  If we choose pMET25, we might need to use two constructs in order to generate enough product to have decent activity of our device.  However, one will note that we plan to use the activator without fluorescence, which will likely be made in greater quantities than activator that is paired with fluorescence.
Took overnight cultures of  54, 256, 259, CFPx2, and P0661 out of incubator at 5:50 pm.  Thus, they had been in the incubator for about 17 hr 30 min.  Made two glycerol stocks for each part.  Note that both are from the same colony.  Proceeded to miniprep the remaining culture.
**Need to take concentrations.**
Note the following that needs to be done:
glycerol stock of null + Gli1 repressor
transformations of the following yeast parts:
the following ligations:
7/19/08
--------------------------------------------------
Came in and spun down the overnight cultures.
Realized that we only have 2 columns left.
Will innoculate new cultures tomorrow afternoon for glycerol stock & miniprep.
I think perhaps we should switch back to 8 ml cultures to avoid problems with our miniprepped plasmid DNA.
Construction of new parts:
binding sites + mCYC
digest all 3 binding site plasmids as vector & mCYC as insert
Digest with S & P
V0120_P0450 (HIV)
V0120_P0453 (Gli)
V0120_P0454 (YY1)
*Note- we already have LexA op8 mCYC as part 1A from D. Drubin
Do this double digest in Buffer B?  It has 100% efficiency for SpeI and 50-75% efficiency for PstI.
Digest with X & P
P0661 from D. Drubin
Do this double digest in buffer H. (100% PstI and 100-125% XbaI)
Run a gel and ligate all 3.
Another round adding 2nd binding sites would exclude YY1 (as it has no activator), totalling 4*3=12 combinations.  Lets test competitive binding on all the sites 1st to see if it works, and on which sites, then ligate these to the beginning of the BS+mCYC parts.
CFP repressors (& activators?)
1) digest CFPx2 as insert and NLS+term as vector
Digest with E & S
CFP-CFP
Do this double digest in buffer E. (75-100% EcoRI and 100% SpeI)
Digest with E & X
V0002_Z0038 (NLS+term)
and
V0120_L0101 (term)
Do this double digest in buffer H. (100% EcoRI and 100-125% XbaI)
to yield: BluNucEnd (CFP+CFP+NLS+TERM)
and BluEnd
digest kozak as vector and untagged repressors as insert
Digest with S & P
Z0120_K0001 (kozak)
Do this double digest in Buffer B?  It has 100% efficiency for SpeI and 50-75% efficiency for PstI.
Digest with X & P
323 (LexR-tag)
329 (GliR-tag)
Do this double digest in buffer H. (100% PstI and 100-125% XbaI)
gel all + ligate the right ones.
2) digest untagged repressors as insert and CFPx2+NLS+term as vector
Digest with E & S
323 (LexR-tag)
329 (GliR-tag)
(or these + Kozak if necessary above)
Do this double digest in buffer E. (75-100% EcoRI and 100% SpeI)
Digest with E & X
BLUEND (CFPx2+NLS+term)
Do this double digest in buffer H. (100% EcoRI and 100-125% XbaI)
gel + ligate
Final ligation: add desired promoter and insert into shuttle vector
Switching Repressor shuttle vector
Cut repressor vector and target vector with E & S.
Purify & ligate
*THIS WORKS FOR pRS303 and pRS306, but not pRS305
7/20/08
--------------------------------------------------
Came in and innoculated 8 mL cultures for gly stock and miniprep tomorrow.  In incubator at 9:40 PM, started timer.
Restreaked a part that had only 1 colony.
Wrapped the drying-out gel in plastic wrap and put it in the fridge.
Media for Testing Parts
We don't need to buy raffinose if we do tests of pGAL1 as either ON (galactose carbon source) or OFF (glucose).  Because the response is basically all or nothing anyway, this would make a lot of sense.  It will also simplify our testing greatly.
Galactose media is about $175 less expensive than raffinose
DOB w/ galactose: http://www.shop.sunrisescience.com/product.sc?productId=161
YNB Powder http://www.shop.sunrisescience.com/product.sc?productId=814
galactose (20 g/L)
ammonium sulfate (5 g/L)
SC-His-Leu-Met-Trp-Ura does not appear on the sunrise website.  I sent an email to see if it is available.
7/21/08
--------------------------------------------------
order galactose media
Gly stock & miniprep parts
check PCR on g. wessel's UV box
double digests for ligation run on gel, extract, ligate, transform
innoculate yeast cells for part transformations tomorrow
linearize DNA for part transformations
Transformations for tests
single transformations (controls) for activators & repressors
for competitive binding test- triple transformations
-can simply add repressors to the strains with [activator & activator reporters]
-see if pMET is strong enough to compete with activator from pGAL (though protein expression will certainly be lower, perhaps the effect of the factor will not)
-if pMET25 is not strong enough, we will need to either
A) put in 2 copies of the repressor constructs by either
1)putting 1 in W303a and one in W303alpha and mating (Selection may require putting repressors on another vector [cut out of pRS304* with Eco and Spe, cut pRS303 with the same, gel purify & ligate] or
2) putting 2 copies of the repressor construct on one vector
B) put the activator under constituative promoter and the repressor under pGAL (too hard & lacks utility)
Took cultures from last night out of incubator after 15 hr and 59 min.  Observed no growth in 266, but all others seemed fine.  Made two glycerol stocks for each part (both being from the same colony).  Miniprepped the remaining culture.  Regrew 266 overnight.
Note about the fluorescence that were are using: Supplementary information from the paper by Ajo-Franklin indicates that the rate constant for RFP maturation is kRFP = 3.1 ± 1.3 x 10-4 s-1 and for YFP is 1.03 ± 0.17 x 10-3 s-1.  These numbers are corrected for photobleaching.
Redoing the PCR
Our gel results indicated that we simply did not run the PCR correctly because we did not even see the positive control on the gel.  Thus, we decided to retry the PCR reactions.  It was noted that we probably used too much template DNA and that 0.1-1 ng/mL is much more realistic.  Thus, we need stock solutions of our plasmid DNA of 0.025 ng/uL.  Since the stock solutions that we currently have are in a concentration of 0.5 ng/uL, we will simply add 5 uL of the latter to 95 uL of Tris HCl.
Also, it should be noted that we did not vortex the primer DNA after suspending it in water.  Thus, we will try to re-make the diluted primer mixes using the remaining water in the "500uM" tubes, but only after wortexing thoroughly.  Finally, we will check the program on the thermal cycler to ensure that it is indeed accurate for our purposes.
1x  16x
H2O  36.5  584
10x buffer  5  80
25mM MgCl2  3  48
10mM dNTP  1  16
Taq  0.5
8
Add 46 uL of this master mix to each PCR reaction.  This is after adding 2 uL of template DNA and 1 uL of each primer.  For the negative controls, added 2 uL of water instead.  Also, added 4 uL of positive control provided with the Taq kit.
The following PCR cycle was used:
1. 95°C  3:30 min
2. 95°C  30 sec
3. 50°C  30 sec
4. 72°C  2:30 min
6. repeat steps 2-4 29 times
7. 72°C  5:00 min
8. hold at 4°C
This was saved on the PCR machine as 'IGEM72'.
One possibility we must consider should this PCR attempt fail again is that the DNA we are using is not what we think it is.  Even if the identity is accurate, it could be impure.
7/22/08
--------------------------------------------------
Took 266 out of incubator after 18 hours.  Made two glycerol stocks of 266 and miniprepped.  Eluted in 70 uL EB.
Concentration:
Discovered contamination in some bottles of media that we had, including one bottle of YPD that was used to make the yeast cultures from last night.  Indeed, the negative control (plain YPD with no innoculated cells) demonstrated growth.  As such, we were forced to throw away our overnight cultures and postpone doing the transformations we had planned.  Will try to do the transformations tomorrow.  We also found contamination in the double drop out media used to grow activator + reporter parts, which has dropouts for Met, Leu, and Ura. There was also a need to make plates for use to grow the successfully transformed yeast.  In particular, we need about 12 trp-, ura- plates and 12 leu-, ura- plates.
We began with 1L, to which we added 27.2614 g SD dropout mix, 2.0006 g histidine, and 2.0028 g methionine.  At this point we then realized that we had been following an incorrect recipe for adding amino acids; we were adding far too much.  For example, instead of 2 g histidine per liter, we only need 20 mg per liter.  We thus decided to finish up the mixture for plates by putting in the correct amount of amino acids from stock solutions (leucine for the trp-, ura- plates and tryptophan for the leu-, ura- plates).
We also decided to make 250 mL of leu-, ura- dropout mix with the correct amount of amino acids.  We ended up leaving methionine out of this mixture as well.
For future reference, we have the following stock solutions:
Leucine: 20 mg/mL, add 5 mL per liter
Histidine: 100 mg/mL, add 200 uL per liter
Tryptophan: 10 mg/mL, add 2 mL per liter
Methionine: 10 mg/mL, add 2 mL per liter
Of these, leucine, tryptophan, and methionine were made today.  To make leucine stock sol'n, dissolved 2.0025 g in 100 mL ddH2O.  To make tryptophan stock sol'n, dissolved 1.001 g in 100 mL ddH2O.  To make methionine stock sol'n, dissolved 0.500 g in 50 mL ddH2O.
Ran a gel of our PCR products from yesterday.  Visualization indicated that there were some successful PCR reactions.  From the right of the gel, there were bands in wells 7, 9, 21-24, and possibly also 16.  This appears to be slightly different than seen in the pictures taken earlier.  This corresponds to all three activators and two of the mCYC + BS parts (GliA and LexA).  Note that the actual sizes were very distinct for the activators but there was some confusion concerning the lengths and thus the identities of the mCYC + BS parts.
Ordered new dropout media.  This media has no carbon source, so we will be able to supplement it with galactose or glucose as needed.
Innoculated fresh colonies in YPD in preparation for transformations tomorrow.  Three 5 mL cultures of W3031a and one 5 mL culture apiece of GliA + GliArpt and ZifA + ZifArpt.  Took all cells directly from glycerol stocks, glycerol stock #1 in all cases (e.g. W3031a #1).  Placed in incubator ar 5:03 pm.
Linerization digests:
Plasmid DNA required:
pRS304* - x4 = 68 uL - do 100 uL
pRS306 - x5 = 85 uL - do 100 uL
Z0411 (304*)- x1 = 17 uL- do 25 uL
Z0510 (306) - x1 = 17 uL - do 25 uL
Z0415 (304*)- x4 = 68 uL - have 1, do 50 uL * Didn't have enough miniprep to use 20 ul dna- used 15 ul and 5 ul water
Z0418 (306)- x1 = 17uL - do 25 uL
Z0412 (304*)- x1 = 17uL - done
Z0511 (306)- x1 = 17 uL - do 25 uL
Z0416 (304*)- x4 = 68 uL have 1, do 50 uL  * Didn't have enough miniprep to use 20 ul dna- used 10 ul and 10 ul water
Z0421 (304*)- x3 = 51 uL - do 50 uL
Total= 450 uL, 9 50 uL reactions make 500 uL mix
All use PstI
Buffer H
50uL Buffer H
5 uL BSA
235 uL H20
10uL ( or less?) enzyme
Minipreps to replenish:
Z0510
Z0411
Z0418
Z0511
Z0416
Z0415
10 uL DNA for each 25 uL reaction with the exceptions noted above
Digests began at 6:50 PM
7/23/08
--------------------------------------------------
Took yeast cells out of incubator around 11:42 am.  Discovered that there was essentially no growth in any of the W3031a tubes, and some growth for each of the activator cells.  Thankfully there was no growth in either of the controls.
The OD measurements are as follows:
Sample ID  User ID  Date  Time  Baseline  600 nm Abs.
Default    7/23/2008    11:53 AM   0  -0.001
W3031a #1   Default    7/23/2008    11:55 AM   0  0.039
W3031a #1   Default    7/23/2008    11:56 AM   0  0.018
W3031a #2   Default    7/23/2008    11:58 AM   0  0.004
W3031a #2   Default    7/23/2008    11:58 AM   0  -0.003
W3031a #3   Default    7/23/2008    12:00 PM   0  0.033
W3031a #3   Default    7/23/2008    12:00 PM   0  0.006
ZifA+ZifArpt   Default    7/23/2008    12:02 PM   0  0.061
GliA+GliArpt   Default    7/23/2008    12:04 PM   0  0.096
GliA+GliArpt   Default    7/23/2008    12:04 PM   0  0.135
ZifA+ZifArpt   Default    7/23/2008    12:07 PM   0  0.077
ZifA+ZifArpt   Default    7/23/2008    12:07 PM   0  0.029
In any case, put W3031a cells from all three tubes into 90 mL of fresh YPD media (newly made today) and each of the activator strains into 50 mL of the same media. These mixtures all went into the shaking incubator around 1:15 pm.
Decided to run another gel for the remaining PCR products to determine whether they actually worked.
Saw two bands on the PCR, both around 400 bp in length.  These were from wells 3 and 10 (from the right of the gel), which correspond to mCYC + ZifA BS and pCYC promoter, respectively.
Also, will do PCR purifications of the parts that were successful, namely all activators and apparently all mCYC + BS parts as well.
So far, we have positive PCR results for all 3 activators and also for all three mCYC + binding sites, as well as what appears to be the pCYC promoter.  However, while the band seemed to be at 400 bp, whereas the actual length of this part is 288 bp.  Thus, we were unsure about the true identity of that band of DNA.  Incidentally, since we only plan to use the pCYC promoter as a back-up in our design (should we need a different constitutive promoter strength), it should not be a problem if we cannot get it as a viable part right now.
Took cultures out of incubator after approximately 4 hr and 30 min.  The ODs are as follows:
Sample ID  User ID  Date   Time   Baseline   600 nm Abs.  Cursor Pos.  Cursor Abs.
W3031a #1  Default  7/23/2008  6:18 PM  0  0.016  281  -0.049
W3031a #1  Default  7/23/2008  6:18 PM  0  0.028  281  0.02
W3031a #2  Default  7/23/2008  6:20 PM  0  0.01  281  -0.017
W3031a #2  Default  7/23/2008  6:21 PM  0  0.01  281  -0.027
W3031a #3  Default  7/23/2008  6:23 PM  0  0.034  281  -2.701
W3031a #3  Default  7/23/2008  6:23 PM  0  0.237  281  -2.53
ZifA + ZifArpt  Default  7/23/2008  6:25 PM  0  0.026  281  -2.104
ZifA + ZifArpt  Default  7/23/2008  6:26 PM  0  0.058  281  0.14
GliA + GliArpt  Default  7/23/2008  6:29 PM  0  0.038  281  -0.028
GliA + GliArpt  Default  7/23/2008  6:29 PM  0  0.074  281  0.043
Transformations completed.  The following transformations were done:
Into W3031a (trp-, ura- double dropout):
1. pRS304* + pRS306
2. nullR (Z0411) + pRS306
3. ZifR (Z0421) + pRS306
4. pRS304* + GliRrpt (Z0510)
5. GliR (Z0415) + pRS306
6. pRS304* + LexRrpt (Z0418)
7. LexR (Z0412) + pRS306
8. pRS304* + YY1Rrpt (Z0511)
9. YY1R (Z0416) + pRS306
Into GliA + GliArpt strain (leu-, trp-, ura- triple dropout):
10. ZifR (Z0421)
11. GliR (Z0415)
12. YY1R (Z0416)
Into ZifA + ZifArpt strain (leu-, trp-, ura- triple dropout):
13. ZifR (Z0421)
14. GliR (Z0415)
15. YY1R (Z0416)
Note that 10-15 are for Test #2.  11 and 13 test competitive binding while the remaining transformations test to ensure that there is no nonspecific binding between the repressors that do not share the same binding domain as the activator and the target site for this binding domain.  This is important seeing as we will have multiple repressors in our system and we would like to characterize the effects of their presence on the functionality of the system.  Thus, we expect the transformations 10, 12, 14, and 15 to emit the same YFP response as a control which contains pRS304* + activator + activable reporter.  This control will be constructed later.
Half of each plate for these transformations had 20 uL while the other half had 100 uL.
Put plates into the incubator at 30 degrees C at 11:30 pm.
7/24/08
--------------------------------------------------
x-Innoculate null + GliRrpt in preparation for glycerol stock tomorrow
x-Make LB media, YPD media, and YPDx2 media
x-Autoclave more microcentrifuge tubes and more ddH2O
x-Innoculate yeast cultures in preparation for activator transformations tomorrow
Linearizations in preparation for activator transformations
Nanodrop minipreps with unknown concentrations (Silver lab parts)
x-Miniprep cultures from last night
Will do the following transformations tomorrow, prioritizing those that are for Test #1:
Into W3031a:
1. pRS305 + pRS306
2. pRS305 + ZifArpt (Z0637)
3. ZifA (Z0267) + pRS306
4. GliA (Z0277) + ZifArpt (Z0637) (Note that this is supposed to serve as a replacement for our null activator)
5. pRS305 + LexArpt (DD25)
6. LexA (Z0279) + pRS306
7. ZifA (Z0267) + LexArpt (DD25)
8. LexA (Z0279) + LexArpt (DD25)
9. pRS305 + GliArpt (Z0739)
10. GliA (Z0277) + pRS306
11. LexA (Z0279) + GliArpt (Z0739)
A couple of notes about our linearizations:
We need to clean up our restriction digests, especially the linearizations, using a PCR purification kit.  The problem is not so much the enzymes themselves as it is the components that go into the restriction digest (such as BSA).
Adrian suggested that we use a specific, well-characterized concentration of linearized DNA for our transformations.  He also explained why we should actually run a serial dilution on one of our transformations in order to determine the transformation efficiency.  This is how we can calculate it:
1) we know the amount of linearized DNA (in ng) that we add to a known number of cells (estimated from an OD measurement right after the 4 hr incubation)
2) we know the volume of cells that we distribute onto each plate
3) if we make serial dilutions (say, 100x dilution for each dilution) and plate the results on separate plates, we can observe how many
We should also run another gel on our PCr products just to make sure that we did not get pTEF.  We should only run the gel for 20-30 mins this time (as opposed to several hours).  Also, when imaging, we should flip the gel upside down for easier comparison to the ladders.
Competent cell protocol will be provided by Adrian.
7/25/08
--------------------------------------------------
This is an example of the calculations involved:
1 ug of plasmid used per SINGLE transformation.  (we could use 0.5 ug of each plasmid for a double transformation).  OD after the incubation period indicates approximately 1x10^8 cells.  We resuspend this in 1 mL and then add 100 uL to each transformation.  Assuming that the cells themselves take up a negligible volume, this means that 1x10^7 cells goes into each transformation.  This assumption is slightly off, seeing as the cells usually take up an additional 100-200uL.  Thus, let's say 8.33x10^6 cells goes into each transformation.  These cells are then diluted into a volume of 460 uL (adding 360 to the 100 already there), heat shocked for 40 mins, and then spun down and resuspended in 1 mL.  Of this 1 mL, let's say that you directly plate out 20 uL or 100 uL (just as we did on 7/23/08).  This ends up being 1.66x10^5 cells or 8.33x10^5 cells, respectively or about 16,000 or 83,000 cells.  Even if the transformation efficiency is 10%, one would still have to count about 1600-8300 colonies on the plates.  This would be impossible and would yield no isolated colonies.
Thus, after resuspending in 1 mL, one can then do two dilutions by 100x.  The first dilution will be 10 uL in 1 mL of water, which will yield 8.33x10^4 cells in 1 mL.  If you plate out 20 uL or 100 uL of this first dilution, you put 1.66x10^3 cells or 8.33x10^3 cells on the plates.  Then, if the transformation efficiency is 10%, you will be able to count 166-833 colonies, which is a lot, but more manageable.  The second dilution would put 10 uL of the 8.33x10^4 cells / mL tube into 1 mL.  The number of cells in this second dilution would thus be 8.33x10^2.  Plating out 20 uL or 100 uL of this final dilution will give you either 1.66x10^1 cells or 8.33x10^1 cells onto the plate.  This last situation is the most accurate for determining plating efficiency, because even with 100% transformation efficiency, one would only have to count 166-833 colonies.
We made an excel spreadsheet to help us calculate this in the future.
PCR results
Well
2 - 300-400 (pCYC)
3 - 100 or less (PTEF)
4 - none (pGPD)
5 - ~400 (mCYC + ZifA BS)
6 - 400-500 (mCYC + GliA BS)
7 - ~500 (mCYC + LexA BS)
8 - 100 or less (pTEF primers, control)
9 - 100-200 (pGPD primers, control)
10 -100 or less (mCYC + BS primers, control)
The ones that seem most promising are mCYC + ZifA BS and mCYC + GliA BS, seeing as they are within the correct lengths.  The mCYC + LexA BS should be 244 bp, but the band appeared around 500 bp.  pCYC should be 288 bp, but its band was at 300-400.
Note that wells 2, 3, 4, 5, and 6 all came from the first round of PCR.  As such, we did not get an accurate assessment of the PCR that really did work, which was the second round.  We need to run another gel tomorrow using the correct PCR products.
Did 30 uL restriction digests of the purified PCR products, with the exception of mCYC + LexA BS (mLexA).  Thus, the parts that were digested with EcoRI and PstI in buffer H were ZifA (no promoter), GliA (no promoter), LexA (no promoter), mCYC + ZifA BS (mZifA), mCYC + GliA BS (mGliA), and also the vector into which these parts will be inserted, pSB1AK3.
These were all digested at 37 degrees C for 10 hours, heatshocked at 80 degress C for 10 mins, and then held at 4 degrees C using the thermal cycler.  The amounts of DNA included in each reaction were calculated precisely based upon concentration of the miniprep and the desired amount of DNA to give a molar ratio of 3:1 for insert to vector (thus, the mass of each part was normalized to its length).  These calculations are included on a spreadsheet.
7/26/08
--------------------------------------------------
Need to run a gel of restriction digest products from last night, with the hope of extracting the results.  Took a picture of the gel.  Saw clear bands around 2000-3000 for the activator parts, which is expected.  Cut these out of the gel.  Also saw a distinct band for the PSB1AK3 restriction digest slightly larger than 3000.  This corresponds to the known length of this plasmid of 3189 bp.  Did not see bands for either mZifA or mGliA, though this could simply be due to the fact that the gel was not run long enough to show resolution for pieces of DNA as small as these (318 bp and 422 bp, respectively).  Attempted to re-run the remaining gel after extracting the other parts, but this did not show any results.  Will thus have to redo these restriction digests, as all 30 uL from last night's reaction were run on the gel.
Gel extraction:
Part  Initial weight of tube (g)  Weight of tube with extract (g)  Mass of extract from gel (g)  Volume of QC required (uL)  Total volume in tube (uL)
LexA, no promoter  1.0256  1.189  0.1634  490.2  653.6
GliA, no promoter  0.9926  1.2124  0.2198  659.4  879.2
ZifA, no promoter  1.0161  1.1612  0.1451  435.3  580.4
pSB1AK3  1.0309  1.2157  0.1848  554.4  739.2
Decided not to use isopropanol seeing as this step only serves to increase the yield for fragments that are not 0.5 - 4 kbp in length.
Eluted in 30 uL EB.  Nanodrop results:
Also, will run a large gel of the PCR purification products, as well as of the promoters, 10 uL in each well.  Decided to run these on a 1.5% agarose gel, since this percentage will give resolution between 0.2 kbp and 3 kbp, and all of the parts in question range in size from 244 bp to 681 bp.  Will run the following:
Well #
1 - 100 bp ladder
2 - pCYC promoter (1)
3 - control, pCYC primers (10)
4 - pTEF promoter (2)
5 - control, pTEF primers (11)
6 - pGPD promoter (3)
7 - control, pGPD primers (12)
8 - purified PCR product, mZifA (7 pu)
9 - purified PCR product, mGliA (8 pu)
10 - purified PCR product, mLexA (9 pu)
Note that for this particular gel, well #1 is the leftmost well when the gel is oriented with the wells at the top.
The results of this gel were as follows:
Sample ID  User ID  Date  Time  ng/ul  A260  A280  260/280  260/230
pSB1AK3   Default    7/26/2008 11:24 PM   11.15  0.223  0.11  2.03  0.12
pSB1AK3   Default    7/26/2008    11:24 PM   12.04  0.241  0.134  1.8  0.13
LexA -prom   Default    7/26/2008    11:25 PM   13.11  0.262  0.145  1.81  0.08
LexA -prom   Default    7/26/2008    11:25 PM   13.21  0.264  0.15  1.76  0.08
GliA -prom   Default    7/26/2008    11:26 PM   7.4  0.148  0.075  1.96  0.03
GliA -prom   Default    7/26/2008    11:27 PM   6.74  0.135  0.064  2.11  0.03
ZifA -prom   Default    7/26/2008    11:28 PM   7.16  0.143  0.078  1.84  0.04
ZifA -prom   Default    7/26/2008    11:28 PM   8.02  0.16  0.096  1.66  0.05
Terrible concentrations, but potentially still usable.
2 - band around 350 bp, 288
3 - no visible band
4 - band around 100 bp
5 - band around 100 bp
6 - two separate bands between 100 and 200 bp
7 - two separate bands between 100 and 200 bp
8 - two bands, one at 300-400 bp and another  at 100-200 bp
9 - two bands, one at 400-500 bp and another  at 100-200 bp
10 - two bands, one at 400-500 bp (longer than the band in well 9) and another  at 100-200 bp
This confirms that we do not have any of our constitutive promoters.  The only promising promoter is pCYC, which has a band in the well with the promoter but not in the control well.  However, the length is not as expected (350 bp on the gel, 288 bp expected).
This gel also confirms that we definitely do have our mZifA and mGliA within expected ranges.  mLexA, however, should only be 244 bp according to the vector map.  That it is about 200 bp different from expected indicates either an unexpected PCR product or that the vector map we are using to predict its length is incorrect.
Took all yeast transformations out of incubator.  Although the cells are rather dense, it is possible to find at least one or two decently isolated colonies on each one.  Noticed significant contamination on the transformation for pRS304* + pRS306.  Decided to take a colony from the opposite side of the plate and streak it out onto a fresh plate, just in case this might work.  However, it will probably be necessary to re-do this transformation.  For the others, we'll make glycerol stocks next week, possibly in conjunction with testing.
7/27/08
------------------------------------
Innoculated a 5ml culture of HB101 in SOB from the frozen cometent cells to innoculate a big culture for competent cells tomorrow.
Decided to double the amount of dna in restriction digests to compensate for loss in gel extraction
Could not find miniprep for part 259.  Innoculated 20 mL culture of 259 in SOB + Amp (no LB)
Will miniprep and do all digests tomorrow morning.
7/28/08
------------------------------------
Remake LB, YPAD, and YPADx2 (all three were contaminated)
Make stock solutions of CaCl2 and adenine
If media has come in, make leu-, ura- plates
50 mM concentration of CaCl2:
100ml?
Calcium chloride dihydrate, 147.02 g/mol
Adenine:
1.0002g
In the media:
A: 25.0054 g YPD broth
B: 25.0030g YPD broth
C: 25.0065g YPD
D: 20.0020g LB
One guy- make competent cells
Another guy- Run a gel of restriction digests, extract the parts we want, and ligate them.  Then transform them into competent cells made by aforementioned guy.
4PM- put in cultures for transformations
7/29/08
------------------------------------
replate W303a and alpha from plates & gly stocks
gel extracted digests- didn't run enough dna.  will do again tomorrow
innoculated HB101 culture in SOB for comp. cells tomorrow *Contaminated?* check -CTRL good.
7/30/08
------------------------------------
Streak out another plate of HB101 cells
Another round of PCR in hopes of attaining higher yields than last time
Run restriction digests (for constructs) on gel, extract them
order another gel extraction or PCR cleanup kit (for the columns)
Discovered that the negative control for HB101 overnight culture was contaminated.  Also, the plate that had been made yesterday had no distinct colonies on it.  As such, decided to replate HB101.  Put this into incubator around 8:45 pm.
Also, innoculated a culture of HB101 from a plate that we had in the fridge from 7/28/08.  Placed this in the incubator at 9:55 pm.
Weighing out for making more media:
Need 0.7351 g CaCl2 dihydrate in 100 mL to make 50 mM solution
A - 25.0030 g YPD broth
B - 25.0040 g YPD broth
C - 25.0028 g YPD broth
D - 20.0025g Lennox LB broth
E - 0.7354g CaCl2
PCR:
Double checked the primers for the promoters, seeing as those did not work for either previous PCR reaction.  Discovered that the primers were, in fact, correct.
Note that one guide for PCR (http://www.fermentas.com/profiles/modifyingenzymes/pdf/protocols/protocolhotstart.pdf) recommends a concentration of 2 mM MgCl2 if one uses 0.2 mM dNTP to maximize PCR output.  Since 1 uL of 10 mM dNTP solution in 50 uL gives 0.2 mM dNTP, one should thus consider using 4 uL of 25 mM MgCl2 to yield a 2 mM concentration.
Will do PCR tomorrow using this information.
1x  16x
H2O  35.5  572.8
10x buffer  5  80
25mM MgCl2  4   64
10mM dNTP  1  16
Taq  0.2
3.2
Also decided to remake the 20 mM solutions for the six primers that correspond to the promoters.  This is just in case the primers were somehow not included in the mixes for the other two PCR attempts.
Add 46 uL of this master mix to each PCR reaction.  This is after adding 2 uL of template DNA and 1 uL of each primer.  For the negative controls, added 2 uL of water instead.  Also, added 4 uL of positive control provided with the Taq kit.
The following PCR cycle was used:
1. 95°C  3:30 min
2. 95°C  30 sec
3. 50°C  30 sec
4. 72°C  2:30 min
6. repeat steps 2-4 29 times
7. 72°C  5:00 min
8. hold at 4°C
7/31/08
----------------------------------
Discovered that the HB101 plate that had been streaked out yesterday had sufficiently isolated colonies.  The same was true of the W3031a plate that was streaked from a glycerol stock two days ago.  Will use this plate to innoculate cultures tonight so that transformations might be accomplished tomorrow.
Innoculated three colonies of W3031a and placed them in the incubator around 5:17 pm.
However, observed contamination of the YPAD that was used for these.
Re-ran the PCR according to the description from yesterday.
PCR #3
I entered the following program into the old schoo' thermocycler (and put some mineral oil in the temp. sensor tube- check the volume.)
1. 95°C  3:30 min
2. 95°C  30 sec
3. 50°C  30 sec
4. 72°C  2:30 min
6. GOTO step 2-4 33 times
7. 72°C  5:00 min
8. 4°C    24 hrs
This was saved on the ERICOMP DeltaCyclerI as 'IGEM'.
Also, innoculated a culture of HB101 in preparation for making competent cells tomorrow.
8/1/08
--------------------------------
Poster presentation today.  We signed the invention disclosure form before sharing our project with people who were not part of the Brown community.  From casual observation it appeared that by far most of the people (if not all) attending the symposium were from Brown anyway.
Because of the poster presentation, and the fact that Adrian was not around to help us with the competent cell protocol, we decided to hold off on this.
Need to run gel on PCR products.
Also, heard back from Christina Smolke.  She is sending us the W303_alpha strain that has a knockout for the gal2 gene.  She is also sending us the construct that will enable us to do the same knockout in W303_1a.  Apparently, the construct can generate a knockout via homologous recombination for anything as long as we attach the correct ends during PCR.  The ends should target the construct to a specific site (locus) in the yeast genome, which will then cause a knockout of that gene.  Thus, once we receive the gene, we only need to PCR it with the primers that will target it to the GAL2 gene.  Note that this construct operates using G418 resistance this gene replaces the GAL2 gene via homologous recombination, thus simultaneously providing the knockout and inserting a selection marker.  G418 is expensive!! ($100 per gram).
Are there other possible uses for this powerful construct as we move into tests?
One other consideration to keep in mind is that if we decide to use the gal2 knockout strains we need to change our plan for subsequent transformations.  Hopefully we will not have to redo the transformations we have already completed, though we should keep that option open, especially if we do not see a good enough response from the GAL1 promoter in the normal W303 strains.
8/2/08
Streak out HB101 plate
8/3/08
Innoculate HB101 culture
Ligations of everything
Came in to discover that both new bottles of LB media were contaminated.  The same was true of the 2xYPAD.  This means that it will be impossible to make an overnight culture of HB101 tonight.  However, streaked out a new plate and put into the incubator at 11:20 pm.
Will attempt ligations in ratios of 3:1 and 6:1 for insert to vector and see which one gives better results.
8/4/08
Came in to discover that the HB101 plate was too thickly streaked.
Made lots of media.  Decided to use a different autoclave because the one that we had been using seemed to be inadequate for sterilization.
Run gel of PCR products:
Ran them in two separate gels; for the activators, whose length was around 2200 bp, ran in a 1% gel.  For the other parts, whose lengths are all less than 1 kb, ran them on a 1.5% gel, which has a separation down to 0.2 kbp.
Took a picture of the first gel
well #
4 - pCYC
5 - pCYC control
6 - pTEF
7 - pTEF control
8 - pGPD
9 - pGPD control
4 - band around 300-400 bp
5 - band less than 100 bp?
6 - two bands, around 100 bp
7 - two bands, around 100 bp
8 - two bands, around 100 bp
9 - one band around 100 bp
Clearly the results indicate that no activator was made.  However, because it took a long time to set up the apparatus before imaging, it is quite possible that some bands were missed.  Will re-run this gel once more.
**Mistakenly ran a gel of the promoters in place of the activators.  How foolish!!!  Will definitely have to run another gel for the activators separately.  Note that the results we saw for this gel are basically identical to those observed for the promoters previously.
Gel for activators:
well #
4 - ZifA minus promoter
5 - GliA minus promoter
6 - LexA minus promoter
7 - activator primers
well #
4 - one band around 2000-3000 and another two bands around 100 bp
5 - only two bands around 100 bp
6 - one band around 2000-3000, another at 1000-2000, and then two bands around 100 bp
7 - two bands around 100 bp
Resuls indicate that PCR definitely worked for ZifA minus promoter and LexA minus promoter.  The presence of the second band on the LexA is somewhat concerning, however.  Could this be due to adding the incorrect products to this tube?
Second gel (gel for binding sites and promoters): 1.500 g agarose in 100 mL 0.5x TBE, with 10 uL of SYBR Safe.
Well #
2 - pCYC
3 - pCYC control
4 - pTEF
5 - pTEF control
6 - pGPD
7 - pGPD control
8 - mZifA
9 - mGliA
10 - mLexA
11 - mCYC + binding site control
12 - PCR positive control (500 bp amplicon)
Well #
2 - strong band at 200-300 bp, weak band at 700-800 bp
3 - small and diffuse band larger than 100 bp
4 - small and diffuse band larger than 100 bp
5 - small and diffuse band larger than 100 bp
6 - small and diffuse band larger than 100 bp
7 - large diffuse band stretching from 100 to 200 bp
8 - sharp band around 300 bp
9 - band around 400-500 bp
10 - band around 400-500 bp, longer than band in well 9
11 - barely visible band around 100 bp
12 - band at 500, longer than bands in wells 9 and 10
Results from this reaction are about the same as in previous PCR reactions.  mZifA and mGliA are definitely present, being 318 bp and 422 bp in length, respectively.  mLexA may be present, though while it should be only 244 bp in length, the band for this PCR reaction was longer than 422 bp.  pCYC might also be present.  This is corroborated by the fact that there is a strong band between 200 and 300 bp, which corresponds nicely to the length of pCYC (288 bp).  However, it is important to note that there was also a small band around 700-800 bp as well.
Innoculated cultures of W3031a in YPAD for transformations tomorrow.  Placed into incubator at 5:17 pm.  Have Ricahrd's permission to use his shaking incubator tomorrow should we need it.
Innoculated two cultures of HB101 in LB in preparation for making competent cells tomorrow.  Also restreaked another plate of HB101 for use should we fail tomorrow.  Both the cultures and the plate went into the incubator at 8:42 pm.
Ligations will occur today instead.
8/5/08
---------------------------------------
Yeast transformation protocol:
Did transformations of activator parts as described on 7/24/08.
Took initial OD's.  These were as follows:
Sample ID  User ID  Date  Time  Baseline  600 nm Abs.
W3031a #1   Default    8/5/2008    10:48 AM   0  0.371
W3031a #1   Default    8/5/2008    10:49 AM   0  0.323
W3031a #2   Default    8/5/2008    10:50 AM   0  0.387
W3031a #2   Default    8/5/2008    10:50 AM   0  0.374
W3031a #3   Default    8/5/2008    10:51 AM   0  0.363
W3031a #3   Default    8/5/2008    10:52 AM   0  0.364
Note that two measurements were taken for each sample, which accounts for the two values seen for each colony.
Combined all three overnight cultures with 100 mL of fresh media.  This yielded a titer of approximately 4.74E6 cells per mL, which is just under the recommended titer of 5E6 cells per mL.  Note that 115 mL should make more than 20 transformations.
Put into shaking incubator at 11:20 am.  Took out around 3:50 pm, about 4.5 hours of incubation time.
Second set of ODs, from the large culture, were as follows:
0.137
0.208
Spun down two tubes of 45 mL of the culture.  Resuspended each one in 1 mL.  The approximate volume after resuspending was 1.2 mL.  Applied 100 uL to each of the transformations.  This corresponds to about 6.47E7 cells per transformation.
There were only 11 plates available.  As such, it was necessary to hold off on plating the pRS305 + pRS306 part.  This was placed in the fridge for plating tomorrow.  However, for the plates that were made, half the plate contained 50 uL of the original transformation mixture while the other half contained 50 uL of a 1:100 dilution of the same mixture.  This was done in an attempt to glean some information about the transformation efficiency, as well as to hopefully attain sufficiently isolated colonies.  However, given the calculations, it is likely that there will still be too many cells even on the 1:100 dilution to get the deisred paucity of colonies.  Only used a single negative control consisting of the yeast cells plated onto a dropout plate.  Did not use an empty plate as a control.  Thus, if we find contamination on the control plate, we will not know whether it was due to the transformation mixture or the actual plate.
Is should also be noted that, due to timing issues, the cells were left sitting for considerable amounts of time at various steps during the protocol.  For example, after being centrifuged the final time, they were left sitting in the centrifuge for about 20 minutes before being resuspended in fresh water.  If anything, this should have a significant effect on the transformation effeciency.
Also linearized DNA:
Tried to make reactions that were multiples of 17 uL for use in the transformation protocol.  Put in about 0.5 ug of DNA per 17 uL.  Heat inactivated the PstI, but this would have been ineffective for BstEII.  Thus, cleaned up all reactions using a PCR purification kit.  Eluted in volumes that were identical to the original volumes of each reaction (either 36 uL, 53 uL, or 70 uL, accounting for the loss of 2 uL in the column).  However, concentrations were not measured, so it is possible that the DNA was at different concentrations than we assumed.  Should have taken concentration measurements beforehand, or else there will be no basis for standardization.  At this point, will merely have to observe whether the transformation efficiencies on each plate are comparable to each other.
Placed all plates into incubator around 11:30 pm.
Competent E. coli cells:
Used overnight culture of HB101 to inoculate a large culture of SOB media (500 mL). Monitored growth until the OD was around 0.0741 according to the nanodrop, which we have reason to believe corresponds to 0.741.
Will test the competence of these cells tomorrow.
8/6/08
-----------------------------
Completed the following ligations:
CFP-CFP+Z0038
CFP-CFP+L0101
K0001+266
K0001+259
K0001+323
K0001+329
P0501+256 on pRS305
P0501+266 on pRS305
P0501+259 on pRS305
Plated out pRS305 + pRS306 on a triple dropout plate that had been supplemented with 100 uL of 10 mg/mL tryptophan.  Used the same plating scheme as for the other transformations.  (transformation mix and its dilution from yesterday).  These will definitely have lower efficiency than the other plates, and will be ready a day later, but will hopefully yield at least a few colonies.  Put into incubator around 6:30 pm along with a control plate that was triple dropout supplemented with tryptophan in the same manner.
Planning to run a test of activator parts on Friday.  Will build the test of our repressors into Test #2 instead. Thus, we have decided to make the following transformations for Test #2:
1. pRS304* added to the pRS305 + pRS306 strain in W3031a
2.
Transformations:
Note that we have received the specified parts from Smolke.  This includes a plate of the W3031_alpha strain that has GAL2 knockout, as well as the plasmid that contains the G418 gene.  Now it will be necessary to order primers for the GAL2 gene (the sequences for the primers were also provided by Smolke) so that we can make a PCR product using the G418 as template to create the proper knockout DNA.  We can then apply this.  We will also need to order G418 for growing all of our new strain of GAL2 knockouts
-Put transformation plates in the 37C incubator at 2:15AM after finishing all my laundry.
 
8/7/08
Not critical:
Make activator test media:
SD-Leu-Ura with glucose
SD-Leu-Ura with galactose
(autoclave w/o sugar @ 20mL below final vol, then add filter sterilized sugar)
Order primers for Gal2ko
Order oligos & SacI (or borrow enzyme from wessel)to make sticky ends to get constitutive TEF promoter
and/or
Redo PCR
Plan further ligations with any successful ligations in mind
 
Grow overnight miniprep cultures of parts for all yeast transformations
and/or if there is enough (prolly not)
Grow overnight culture to redo activator test transformations (they didn't work- maybe we let the cells grow or sit around too long, or didn't use enough DNA)
Grow overnight cultures for competitive binding test transformations (or save 'til we have the double to transform into)
Grow overnight miniprep cultures of any successful ligations
--------------------------------
Media:
For tests, decided to make 250 mL of activator test media and 500 mL of competitive binding test media.
8/8/08
--------------------------------
Make glycerol stocks of yeast cells in test media before making measurements.
Discovered as of 5 pm no growth on any of the yeast transformation plates. Will most likely have to do these transformations again.  This will require preparing the necessary DNA for linearizations.
The following minipreps are needed:
Z0421
DD25
pRS304*
pRS305
pRS306
Innoculated 20 mL cultures with the #1 glycerol stock in all cases.
For the E. coli ligations, observed at least one colony on each plate!  Success! Also observed the beginning of contamination on several of the plates.  Chose colonies that were removed from this contamination for use in innoculating cultures.  Also decided to replate all of them due to this contamination.
The following successful ligations were grown in culture (20 mL) and on plates:
A3 -CNTerm
C3 - K266
D3 - K259
E3 - K323
F3 - K329
G3 - LexA, no fluor tag (266) on pRS305
G6 - ?
H3 - ZifA, no fluor tag (259) on pRS305
I3 - null A on pRS305
pCS33
Before miniprepping them, should make glycerol stocks, which we will also snap freeze using ethanol and dry ice (better technique).  We will also have to remember to allow the other group to take about 1 mL of one of our cultures for their own test.
Cultures to innoculate:
gly stocks of those above
15 mL minis for pRS306 & CNTerm
Placed plates and cultures into inbucator around 8:30PM
8/9/08
gly stocked the ligations and pCS33.
miniprepped pRS306 and CNTerm
Noticed that some colonies popped up on a few of the yeast transformation plates, and none on the -CTRL just cells. -yay.  Some still have to be redone.
1 had contamination, so I threw it out.
The following transformations had at least one definitive YEAST colony:
LexA (Z0279) + LexA rpt (DD25) (note some contamination, requires a replate)
ZifA (Z0267) + LexA rpt (DD25)
GliA (Z0277) + ZifA rpt (Z0637)
LexA (Z0279) + GliA rpt (Z0739)
possibly also pRS305 + ZifA rpt (Z0637)
The following transformations had some colonies but they were small and were not the characteristic red of this yeast strain:
GliA (Z0277) + pRS306
ZiFA (Z0267) + pRS306
LexA (Z0279) + pRS306
Thus, it seems that it will be necessary to redo the transformations for these three, as well as for
pRS305 + pRS306
pRS305 + LexA rpt (DD25)
pRS305 + GliA rpt (Z0739)
and even for pRS305 + ZifA rpt (Z0637).
Ligation Recap
Inserted CFP-CFP as a prefix onto a vector with Z0038 (NLS+Term) as the suffix.
Inserted 266, 259, 323, and 329 onto a vector with K0001 (kozak) as the prefix.
Did double insertions of P0501 (pGAL+Kozak) and 266, 259, 256 onto pRS305.
treated all vectors with antarctic phosphatase in their gel extraction tube and heat-killed at 65C for 10min.
Ligated in a volume of 15 ul using the newest promega t4 tigase & alliquots of t4 buffer.
Did ligations with 3:1 and 6:1 molar excesses of insert:vector.
Used the following amount of vector in each reaction to keep total amount of DNA/rxn under 100ng:
ng of vector in rxn
Vector  Insert  3:1  6:1
A  NLS+Term  CFP-CFP  30  20
B  Term  CFP-CFP  30  20
C  K0001  266  50  40
D  K0001  259  50  40
E  K0001  323  20  14
F  K0001  329  20  14
G  pRS305 P0501+266  50  40
H  pRS305  P0501+259  50  40
I  pRS305  P0501+256  40  25
 
8/10/08
---------------------------------
Stock solution of 5000uM Met: 0.0750 g Met in 100 mL
Activator test media:
1.826 g SD -met -dex
5 mL of 5000 uM Met solution
0.5 mL of 10 mg/mL His solution
0.5 mL of 10 mg/mL Trp solution
Filled to 225 mL with milli q water
Activator plates:
3.6518 g SD - dex
1 mL of 10 mg/mL His solution
1 mL of 10 mg/mL Trp solution
10.0016 g Bacto Agar
Filled to 450 mL with milli q water
Added 50 mL of filter-sterilized 20% dextrose
Competitive test media:
3.6520 g SD - met -dex
1 mL of 10 mg/mL His solution
Filled to 400 mL with milli q water
Competitive test plates:
7.302 g SD -dex
2 mL of 10 mg/mL His solution
20.001 g Bacto Agar
Filled to 900 mL with distilled water (ran out of milli q)
Added 100 mL of filter-sterilized 20% dextrose
Filter sterilized 45.0012 g of dextrose dissolved in 250 mL of milli-q water to yield a 20% solution.
Rethinking our PCR:
Since we definitely need to Biobrick the TEF promoter, we have decided to rethink our PCR approach.  As mentioned before, one possibility is that our template DNA is bad.  Another is that the 20 uM solution for our primers could not contain any primer after all, remembering the issue about forgetting to vortex the 500 uM solution after adding water initially.  However, a nanodrop of the forward and reverse primers for TEF showed nucleic acid concentrations of 490 and 509 ng/uL, respectively.
8/12/08
-------------------------------
Did the following transformations today:
Single:
pRS304* in ZifA +ZifA rpt (ZaZar)
pRS304* in GliA + GliA rpt (GaGar)
pRS304* in LexA + LexA rpt (LaLar)
nullR (Z0411) in GaGar
nullR (Z0411) in ZaZar
nullR (Z0411) in LaLar
LexR (Z0412) in GaGar
LexR (Z0412) in ZaZar
LexR (Z0412) in LaLar
GliR (Z0415) in LaLar
YY1R (Z0416) in LaLar
ZifR (Z0421) in LaLar
Included a control for GaGar, ZaZar, and LaLar.  Note that several of the triple dropout plates made on 8/11 appeared to be contaminated, seeing as there were splotches of white within the agar.  Used some 8/11 tirple dropout plates and also some from 7/23.
For each one, plates 50 uL of 1:1 and 50 uL of 1:100 dilution.  The diultion was made by adding 0.5 uL of 1:1 to a puddle of 49.5 uL of sterile water on the plate.
made 1:10000 dilutions of Z0411 on ZaZar, GaGar, and LaLar by putting 0.5 uL of initial transformation mix into 49.5 uL sterile water.  Then, plated out 2 uL of this dilution into a puddle of 198 uL of water that had been applied to the plate.
Double (all in W3031a):
pRS305 + pRS306
pRS305 + LexA rpt (DD25)
pRS305 + ZifA rpt (Z0637)
pRS305 + GliA rpt (Z0739)
nullA + pRS306
nullA + ZifA rpt (Z0637)
nullA + GliA rpt (Z0739)
*(are we missing nullA + LexA rpt?)
ZifA (Z0267) + pRS306
ZifA noflo + ZifA rpt (Z0637)
LexA (Z0279) + pRS306
LexA noflo + LexA rpt (DD25)
GliA (Z0277) + pRS306
Triple (all in W3031a):
pRS304* + pRS305 + pRS306
ZifR (Z0421) + pRS305 + ZifA rpt (Z0637)
GliR (Z0415) + pRS305 + GliA rpt (Z0739)
LexR (Z0412) + pRS305 + LexA rpt (DD25)
LexR (Z0412) + LexA no flo + LexA rpt (DD25)
For double and triple transformations, simply plated 200 uL onto each plate.  The reason for this is because these transformations were delayed while the single transformations were being done.  As such, we expect a lower efficiency from them.
Placed all plates into incubator around 2:40 am on 8/13/08.
8/13/08
---------------------------------
8/14/08
---------------------------------
There were some minipreps that needed to be done (C0141, CFPx2, E0130, E0110, C0110, and CNTerm).  These were for continuing the construction process.  Note that two elutions were done for each column, both with 55 uL EB.  The idea behind this is to get one tube with higher concentration DNA and another with lower concentration DNA, since different concentrations are useful for different protocols (for some it is better to have a higher concentration, for some it is better to have a low concentration).
Sample ID  User ID  Date  Time  ng/ul  A260  A280  260/280  260/230
C0141, first elu.   Default    8/14/2008    5:23 PM   127.85  2.557  1.407  1.82  1.41
C0141, first elu.   Default    8/14/2008    5:23 PM   123.81  2.476  1.346  1.84  1.4
C0141 #2 (second elu)   Default    8/14/2008    5:24 PM   24.12  0.482  0.282  1.71  0.91
C0141 #2 (second elu)   Default    8/14/2008    5:25 PM   21.22  0.424  0.234  1.81  0.95
CFPx2, first elu   Default    8/14/2008    5:25 PM   102.36 2.047  1.166  1.76  1.03
CFPx2, first elu   Default    8/14/2008    5:26 PM   72.02  1.44  0.773  1.86  1.52
CFPx2, first elu   Default    8/14/2008    5:26 PM   72.53  1.451  0.771  1.88  1.51
CFPx2 #2 (second elu)   Default    8/14/2008    5:28 PM   18.48  0.37  0.221  1.68  0.85
CFPx2 #2 (second elu)   Default    8/14/2008    5:28 PM   14.02  0.28  0.157  1.79  0.98
E0130, first elu   Default    8/14/2008    5:29 PM   41.73  0.835  0.435  1.92  1.93
E0130, first elu   Default    8/14/2008    5:29 PM   40.8  0.816  0.417  1.95  1.92
E0130 #2, (second elu)   Default    8/14/2008    5:30 PM   9.15  0.183  0.11  1.66  1.7
E0130 #2, (second elu)   Default    8/14/2008    5:30 PM   9.04  0.181  0.092  1.97  1.64
E0110, first elu   Default    8/14/2008    5:31 PM   34.33  0.687  0.356  1.93  2.05
E0110, first elu   Default    8/14/2008    5:32 PM   34.14  0.683  0.356  1.92  2.74
E0110 #2 (second elu)   Default    8/14/2008    5:33 PM   25.3  0.506  0.302  1.68  0.73
E0110 #2 (second elu)   Default    8/14/2008    5:33 PM   28.9  0.578  0.341  1.69  0.69
C0100, first elu   Default    8/14/2008    5:34 PM   101.97  2.039  1.081  1.89  1.69
C0100, first elu   Default    8/14/2008    5:34 PM   97.74  1.955  1.039  1.88  1.77
C0100 #2 (second elu)   Default    8/14/2008    5:36 PM   27.78  0.556  0.317  1.75  0.89
C0100 #2 (second elu)   Default    8/14/2008    5:37 PM   24.16  0.483  0.274  1.77  0.93
CNTerm, first elu   Default    8/14/2008    5:38 PM   41.15  0.823  0.435  1.89  1.99
CNTerm, first elu   Default    8/14/2008    5:38 PM   41.22  0.824  0.417  1.98  1.86
CNTerm #2 (second elu)   Default    8/14/2008    5:39 PM   13.44  0.269  0.155  1.74  0.95
CNTerm #2 (second elu)   Default    8/14/2008    5:40 PM   21.72  0.434  0.261  1.67 0.78
Also, did the gel extraction protocol for CNTerm, K323, C0061, K329, P0454, GPD, P0453, and P0450.  Since there was no isopropanol in the lab, did not do this step although it is possible that several of the parts were larger than 4 kb.  Also did not do the additional wash with QG. Eluted in 50 uL.
Sample ID  User ID  Date  Time  ng/ul  A260  A280  260/280  260/230
CNTerm   Default    8/15/2008    3:56 PM   38.97  0.779  0.411  1.89  0.08
CNTerm   Default    8/15/2008    3:56 PM   42.98  0.86  0.455  1.89  0.08
K323   Default    8/15/2008    3:57 PM   96.69  1.934  1.017  1.9  0.56
K323   Default    8/15/2008    3:58 PM   98.26  1.965  1.048  1.88  0.53
C0061   Default    8/15/2008    3:59 PM   34.67  0.693  0.37  1.88  0.19
C0061   Default    8/15/2008    3:59 PM   36.61  0.732  0.389  1.88  0.19
K329   Default    8/15/2008    4:02 PM   107.47  2.149  1.149  1.87  0.54
K329   Default    8/15/2008    4:02 PM   108.82  2.176  1.17  1.86  0.53
P0454   Default    8/15/2008    4:04 PM   45.03  0.901  0.49  1.84  0.23
P0454   Default    8/15/2008    4:04 PM   41.62  0.832  0.449  1.85  0.21
GPD   Default    8/15/2008    4:06 PM   93.07  1.861  0.994  1.87  0.66
GPD   Default    8/15/2008    4:07 PM   90.62  1.812  0.953  1.9  0.64
P0453   Default    8/15/2008    4:08 PM   63.59  1.272  0.653  1.95  0.17
P0453   Default    8/15/2008    4:08 PM   63.93  1.279  0.665  1.92  0.16
P0450   Default    8/15/2008    4:09 PM   52.81  1.056  0.558  1.89  0.23
P0450   Default    8/15/2008    4:09 PM   56.92  1.138  0.594  1.92  0.23
Where did the GPD come from?  What will these extractions be use for?
What seems concerning for both sets of values is the low numbers for 260/230.  This could signify a poor purity.
8/15/08
----------------------------
Checked on the yeast transformations.  '**' signifies a successful transformation.
Single:
pRS304* in ZifA +ZifA rpt (ZaZar)**
pRS304* in GliA + GliA rpt (GaGar)**
pRS304* in LexA + LexA rpt (LaLar)** (contaminated)
nullR (Z0411) in GaGar**
nullR (Z0411) in ZaZar** (contaminated)
nullR (Z0411) in LaLar**
LexR (Z0412) in GaGar**
LexR (Z0412) in ZaZar** (contaminated)
LexR (Z0412) in LaLar**
GliR (Z0415) in LaLar**
YY1R (Z0416) in LaLar**
ZifR (Z0421) in LaLar**
Note that there was a single colony in the negative control for ZaZar.  This appeared to be a yeast colony because of its characteristic reddish coloring. Could this signify contamination of all ZaZar cells that were used in the transformation, such that some of the cells already had the ability to grow in the triple dropout environment without successful transformation?  This more than anything points to the necessity of verifying that we have what we expected.  We need to run colony PCR to do this.
Double (all in W3031a):
pRS305 + pRS306**
pRS305 + LexA rpt (DD25)**
pRS305 + ZifA rpt (Z0637)**
pRS305 + GliA rpt (Z0739)**
nullA + pRS306**
nullA + ZifA rpt (Z0637)**
nullA + GliA rpt (Z0739)**
ZifA (Z0267) + pRS306**
ZifA noflo + ZifA rpt (Z0637)**
LexA (Z0279) + pRS306**
LexA noflo + LexA rpt (DD25)**
GliA (Z0277) + pRS306**
Note: (are we missing nullA + LexA rpt?)  Other than this one potential part, we seem to have enough transformations to run several activator tests next week.  Hooray!
Triple (all in W3031a):
pRS304* + pRS305 + pRS306
ZifR (Z0421) + pRS305 + ZifA rpt (Z0637)** (a couple small colonies)
GliR (Z0415) + pRS305 + GliA rpt (Z0739)
LexR (Z0412) + pRS305 + LexA rpt (DD25)** (several colonies)
LexR (Z0412) + LexA no flo + LexA rpt (DD25)** (one colony, somewhat small)
Note: will have to redo a few of these, but this will now be a single transformation seeing as all of our activator transformations were successful.
Next week:
1) Calculate estimate of transformation efficiency
2) Calibration of nanodrop respective to yeast (and E. Coli) OD measurements <== important for activator tests
3) Activator and competitive binding plates
4) Replate transformations if possible, especially those that were contaminated (for those without contamination, attempt to reuse plates)
5) Verification of transformations (massive amount of colony PCRs and subsequent gels)
-must verify the expected lengths of each band
6) Activator tests
8/17/08
-------------------------
Found one colony on the LaLar control plate in addition to the one already discovered for ZaZar.  Will also have to run colony PCR on these controls to see what they are.
8/19/08
----------------------------
Triple dropout plates:
7.3058g SD -dex
20.0061g Bacto Agar
2 mL His stock solution
(after autoclaving) 100 mL 20% dextrose solution
22.5023g + 22.5022g dextrose  in 250 mL milli-Q water to yield 20% dextrose solution
20.0012g galactose in 100 mL milli-Q water to yield 20% galactose solution
Stirred at room temperature for about 15 minutes.
Note: One protocol indicates that it can take as long as an hour with stirring for galactose to dissolve at room temp.
Filter sterilized both solutions.
Will need to make 20% raffinose solution tomorrow after autoclaving more glassware.
Notably, none of the double transformations, which signified, required replating.  All of them had a handful of distinct isolated colonies.  These are the transformations that will be tested:
pRS305 + pRS306**
pRS305 + LexA rpt (DD25)**
pRS305 + ZifA rpt (Z0637)**
pRS305 + GliA rpt (Z0739)**
nullA + pRS306**
-missing: null A + LexA rpt (DD25)
nullA + ZifA rpt (Z0637)**
nullA + GliA rpt (Z0739)**
ZifA (Z0267) + pRS306**
ZifA noflo + ZifA rpt (Z0637)**
LexA (Z0279) + pRS306**
-missing: LexA (Z0279) + LexA rpt (DD25) (needs to be replated)
LexA noflo + LexA rpt (DD25)**
GliA (Z0277) + pRS306**
Note: Don't forget ZifA + ZifA rpt and GliA + GliA rpt, which were some of the first transformations.
However upon second observation, it was noted that three of the activator plates had potential contamination on them.  They were replated as well.
Z0277 + pRS306
nullA + Z0637
nullA + Z0739.
Replating of single and triple transformations:
Single:
Z0411 on ZaZar
Z0412 on ZaZar
pRS304* on LaLar
Z0421 on LaLar
Triple:
Z0412 + LexA - tag + DD25
Z0421 + pRS305 + Z0637
Put replated plates into incubator at 2:35 am on 8/20/08.
8/20/08
----------------------------
^ update list of replated parts ^
wash and autoclave glassware
prepare 20% raffinose stock solution, filter sterilize
calculate transformation efficiency from last week's transformations (need OD measurements from that day)
figure out primers to verify yeast transformations (pRS primers don't seem like they'll work!)
recheck sizes of all parts to ensure that they do not overlap (making sure that colony PCR will work)
innoculate cultures by 5 pm and put into incubator for TEST #1
----------------------------
Due to time constraints, both today and also predicted for tomorrow, decided to only test all parts related to ZifA and its reporter.  Thus, the following parts were innoculated today:
pRS305 + pRS306
nullA + pRS306
ZifA (Z0267) + pRS306
pRS305 + ZifA rpt (Z0637)
nullA + ZifA rpt (Z0637)
ZifA (Z0267) + ZifA rpt (Z0637) from colony (7/4)
ZifA (Z0267) + ZifA rpt (Z0637) from glycerol stock #1
ZifA noflo + ZifA rpt (Z0637)
This represents the largest amount of parts that we have for any activator.  Also innoculated two controls, one with dex and one with gal.  Placed all tubes into incubator just before 6 pm.
It was discovered going through the plates that there is a strong need to consolidate the glycerol stocks.  It appears that there are several plates containing transformations that have never been made into glycerol stocks, but because the shared list is incomplete, it is hard to say.
Number of colonies on the 1:1 plate:
8/21/08
------------------------------
innoculate cultures in order to make glycerol stocks
------------------------------
mCherry - 587 ex, 610 em and EYFP - 514 ex, 527 em
Took cultures out of incubator around 2:25 pm, 20 hr and 25 min.
Discovered lack of growth in several of the cultures from last night, especially those whose carbon source was galactose.  This could simply be due to the fact that the tubes with galactose were innoculated after each respective tube containing dextrose, such that the dextrose tubes had first contact with each colony.  All dex tubes experienced growth except for
1) the negative control and
2) the ZifA + ZifA rpt that had been innoculated from the glycerol stock.
Gal tubes that lacked growth included:
pRS305 + pRS306
nullA + ZifA rpt
ZifA + ZifA rpt (both glycerol and colony)
ZifA -tag + ZifA rpt
Note: nullA + ZifA reporter may have been switched.  It is labeled as a dex tube but it might have actually contained galactose.  Be wary for results that would corroborate such a possibility.
The following glycerol stocks were made in duplicate (all duplicates from a single colony):
pRS305 + pRS306
nullA + pRS306
ZifA (Z0267) + pRS306
pRS305 + ZifA rpt (Z0637)
nullA + ZifA rpt (Z0637)
ZifA noflo + ZifA rpt (Z0637)
For lack of a better place, these were put for the time being in the box labelled, "Mike's RNA" in the -80.
Took 1 mL of overnight culture from each and put into a microcentrifuge tube.  Centrifuged for 4 mins at 13300 rpm (16300 g).  Resuspended in 1 mL PBS.  Applied 250 uL to each well in a 96 well plate, ensuring that there were triplicates for each transformation under each test condition.
Took optical density readings using one plate reader and fluorescence measurements using another.  Also compared the optical densities with those taken using the Nanodrop.  However, the file of all Nanodrop ODs was closed before saving it.  What a stupid mistake!
Some notes about planning the colony PCR to verify these transformations:
First, note that the general procedure is to order primers that will give a PCR product of a known size if and only if the desired insert on the shuttle vector has been integrated into the yeast genome, indicative of a successful transformation.  Of course, the entire shuttle vector itself is integrated, so the primers will be designed using the provided sequences for the shuttle vectors.  The PCR product sizes will be predicted using the sequences of the parts in conjunction with those of the shuttle vectors.  Once colony PCR has been run, one then runs the products on a gel to determine if bands appear at the predicted lengths, indicating whether 1) PCR was successful and whether 2) the transformation was successful.  If the bands match up with expectations, one then sends both the PCR product and the original primers to a sequencing company to sequence the part.
Have to check to see whether the URA, LEU, and TRP (and HIS) genes are in close proximity to each other in the yeast genome.
We found some primers that, while not listed on all of the vector maps, are indeed present on pRS304*, pRS305, and pRS306 flanking the multiple cloning site, and thus the inserted part.  Originally we considered simply using these .  Instead, we consider it.  However, this will only work if the sequence of the vectors are different from each other.  Another consideration to keep in mind is that for extremely large expected PCR products (greater than 5 kbp?)
Checked the replates from two days ago.  All had wonderfully isolated colonies.  Gorgeous streaks.  These transformations were:
Double:
Z0277 + pRS306
nullA + Z0637
nullA + Z0739.
Single:
Z0411 on ZaZar
Z0412 on ZaZar
pRS304* on LaLar
Z0421 on LaLar
Triple:
Z0412 + LexA - tag + DD25
Z0421 + pRS305 + Z0637
Browsing through the fridge, it was discovered that the following parts will require replating as well:
nullR (Z0411) + pRS306 (contaminated, requires trp-, ura-)
ZifA (Z0267) + LexA rpt (DD25), not enough isolated colonies, could be useful in the place of nullA + LexA rpt
LexA (Z0279) + LexA rpt (DD25) (contaminated, requires leu-, ura-)
GliA (Z0277) + GliA rpt (Z0739), an old plate that will be used in the future for testing (requires leu-, ura-)
ZifA (Z0267) + ZifA rpt (Z0637), an old plate that will be used in the future for testing (requires leu-, ura-)
Can use triple dropout plates for all of them, making sure to supplement for the amino acid that is needed for each one.
Also, need to figure out which parts have been glycerol stocked.  The list on Google Docs is not comprehensive as of today.
Known parts that have not yet been glycerol stocked:
From 7/04/08:
none
From 7/08/08:
nullR + GliR rpt
From 7/23/08:
1. pRS304* + pRS306
2. nullR (Z0411) + pRS306
3. ZifR (Z0421) + pRS306
4. pRS304* + GliRrpt (Z0510)
5. GliR (Z0415) + pRS306
6. pRS304* + LexRrpt (Z0418)
7. LexR (Z0412) + pRS306
8. pRS304* + YY1Rrpt (Z0511)
9. YY1R (Z0416) + pRS306
Into GliA + GliArpt strain (leu-, trp-, ura- triple dropout):
10. ZifR (Z0421)
11. GliR (Z0415)
12. YY1R (Z0416)
Into ZifA + ZifArpt strain (leu-, trp-, ura- triple dropout):
13. ZifR (Z0421)
14. GliR (Z0415)
15. YY1R (Z0416)
From 8/5/08:
ZifA (Z0267) + LexA rpt (DD25)
GliA (Z0277) + ZifA rpt (Z0637)
LexA (Z0279) + GliA rpt (Z0739)
From 8/12/08:
Single:
pRS304* in ZifA +ZifA rpt (ZaZar)
pRS304* in GliA + GliA rpt (GaGar)
pRS304* in LexA + LexA rpt (LaLar)
nullR (Z0411) in GaGar
nullR (Z0411) in ZaZar
nullR (Z0411) in LaLar
LexR (Z0412) in GaGar
LexR (Z0412) in ZaZar
LexR (Z0412) in LaLar
GliR (Z0415) in LaLar
YY1R (Z0416) in LaLar
ZifR (Z0421) in LaLar
Double:
pRS305 + LexA rpt (DD25)
pRS305 + GliA rpt (Z0739)
nullA + GliA rpt (Z0739)
LexA (Z0279) + pRS306
LexA noflo + LexA rpt (DD25)
GliA (Z0277) + pRS306
Triple:
ZifR (Z0421) + pRS305 + ZifA rpt (Z0637)
LexR (Z0412) + pRS305 + LexA rpt (DD25)
LexR (Z0412) + LexA no flo + LexA rpt (DD25)
**Also note that the following transformations still need to be done:
-missing: null A + LexA rpt (DD25)
-missing: pRS304* + pRS305 + pRS306
-missing: GliR (Z0415) + pRS305 + GliA rpt (Z0739)
8/22/08
------------------------------
autoclave pipet tips, glassware
innoculate cultures in order to make glycerol stocks
------------------------------
9/14/08
Began to troubleshoot PCR
Nanodropped samples of the TEF primers, and realized that their actual concentration was about 3 orders of magnitude less than the concentration they were labled with.  Perhaps in diluting them the first time, Aaron mistook the number of micrograms of each primer in the tube for a number of micromoles.  In fact, there were only ~20 nanomoles of each primer synthesized.  Based on the nanodrop readings of DNA concentration and the number of micrograms/micromole printed on the tubes, I rediluted them to 20 micromolar.  I checked these via nanodrop.  The 20 uM dilutions of both the TEF primers and the GalKO primers were put into microcentrifuge tubes lables with a $, as I believe that they are on the money.
I made new dilutions of the templates as well (TEF and pCS33), bringing them to ~.5ng/ul
Using the new sample of GoTaq Hot Start taq polymerase and green buffer, I performed the following reactions:
1) + control using the template/primers mix from the NEB pcr kit
2) GalKO using the GalKO primers and the pCS33 template
3) TEF with no adjuvants
4) TEF with BSA
5) TEF with DMSO
6) TEF with Betaine
7) TEF with Betaine & DMSO
8) - CTRL GalKO (no template)
9) - CTRl TEF (no template
I used the following program:
1  95  2:30
2  95  30
3  50  30
4  72  1:30
5  goto2  rep 33
9/15
Ran a gel of the PCR reactions.  1% agarose.  Loaded 10 uL of each reaction and 2-log NEB  ladder.
500bp product for the + control reaction, as expected
Got a 1.5 kB product from the GalKO reaction- this looks to be correct.  There is also a faint band at ~4kb.  This is likely template- perhaps I'll use less next time.
No products were visible in the - CTRL or TEF reactions.
Thinking about trobleshooting the TEF reaction- maybe the template is wrong?  I'd like to try using our primers in a whole cell reaction with W303 yeast.  It should pull the TEF promoter out of the genome if the primers are correct.  Also, I found a paper detailing their generation of variants of the TEF1 promoter using error-prone PCR.  They list their primers- perhaps we could use those instead.
(http://aem.asm.org/cgi/content/full/72/8/5266)
Streaked out plates of W303a and W303alpha from plate & gly stock to try out the GalKO and to use as template for the pTEF PCR reaction.
9/23
Made a liter of Activator plates and a liter of Triple dropout plates to have plenty.
9/24
Did whole-cell PCR with the TEF primers on W303a and W303alpha- got a band at the expected 500bp!
Unfortunately, also got extraneous product at ~100bp.  This must be from primer-primer interactions.
Tried varying the annealing temp and adding adjuvants, but nothing helped.
Ran the remainder of the a and alpha reactions on a gel and extracted the 500bp band.  Bad concentration- around 7 ng/ul.  This might be enough for ligation.
PCR purified
Cataloged all of the yeast transformations in a file called "Yeast Catalogue"
They are now organized in the fridge according to which system of factors they belong to, and whether they are double or triple transformations.  Ones with the repressable reporters were sent to the back of the fridge without desert and were not catalogued.
9/25
order another miniprep kit (i think)
State of construction
(look @ gel images and figure out what we actually have)
Did PCR to get the 3 mCYC+BS from the pRS activable reporter plasmids
Got expected bands on gel, plus an extraneous band of slightly larger size, necessitating gel extraction
9/26
gel extracted the mCYC+BS PCR products
We now have pcr products with each of the following binding sites+ mCYC:
gli.mCYC (mGli)
zif.mCYC (mZif)
lex.mCYC (mLex)
As there is no YY1 activable reporter, we don't yet have that part.  Perhaps I will build it.
 
9/28 (sunday)
plated out yeast parts for Lex test
 
10/2 (thursday)
Ran a test on Lex Parts in inducing media (galactose) and repressing media(glucose), with varying concentrations of methionine to control the repressor TF when present.
trouble growing on raffinose- slow.  Some parts grew.  Took fluorescenece readings on the plate reader and OD600s on a separate plate reader to correct for different states of growth.
10/7 (tuesday)
Restriction digests on MANY construction parts:
As fwd. insert:
mZif
mGli
mLex
TEFa
TEFalpha
As rev. insert:
C0100
C0141
CFPx2
E0110
E0130
P0661
C0061
As vector:
pSB1AK3
Z0375
Z0378
Z0379
Z0402
10/8 wed
Ran inserts on a gel and cut them out
10/9 thurs
PCR purified vectors and digested pcr products
10/10 friday
gel extracted vectors
nanodropped all purified construction parts
set up ligations and let them sit on the bench overnight (not on purpose, but should work)
10/11 saturday
ran a gel of the purified vectors to make sure they are the right size.  Used the 26-well comb on the big gel, with 2 ul vector per well, 4ul volume total.  (50ml gel volume in the big box) This worked out OK, but the ladder was dim.  Next time use twice the thickness of gel, and half the length.  The vectors were all the right size. Also ran the WC-pcr reactions, but nothing showed up.  gotta redo it in bigger wells.
transformed most of the ligations.  We're now out of competent cells- make more on wednesday.
NOT TRANSFORMED:
1,2,3,5,6
Set up 100ul PCR reactions of all 3 mCYC+BS, and of TEF, using the TEFa purified product as template so I dont have to gel extract again.
10/12 sunday
purified last night's PCR reactions
quarry
after 22 hours, only some of the transformations have sprouted.
Colonies on:
4
10
12
16
19
23
25
replate:
10
12
16
colony PCR
restriction digests
10/11 monday
purify digests
run gel (extract 54 & P0560)
gel of colony PCRs
Ligations
ON culture of yeasties for imaging
10/12 tues
TV
10/17 fri.
made competent DH5alpha cells following inuoe method.
100ul and 150ul alliquots in the -80C
did some restriction digests for construction:
with X+P:
CNTerm
mYY1
Z0375
Z0378
Z0379
Z0402
E+S:
mYY1
S+P:
TEFbb
10/18:
gel extractions.  ligations. transformations.
10/19
innoculated miniprep cultures from transformations.  decided to forgo YY1 parts to simplify things.
10/20
glycerol stocked ligations
miniprepped ligations
learned how to quantify fluorescence from microscope images
did 2 more ligations zif.MET, zif.ADH, transformed them, grew plates, and innoculated miniprep/glystock cultures at about 5:30PM
4 ligations began at 2:15 AM
}
Retrieved from " http://2008.igem.org/Team:BrownTwo/Notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
