<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Baltimore_BioCrew_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Baltimore BioCrew/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><UL class="navbar cf" id="teamnav"><LI><A href="https://2018.igem.org/Team:Baltimore_BioCrew">HOME</A></LI><LI><A href="#">TEAM</A></LI><LI><A href="https://2018.igem.org/Team:Baltimore_BioCrew/Team">THE CREW</A></LI><LI><A href="https://2018.igem.org/Team:Baltimore_BioCrew/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2018.igem.org/Team:Baltimore_BioCrew/Part_Collection">Thank you Ed</A></LI></UL><LI><A href="#">PROJECT</A></LI><LI><A href="https://2018.igem.org/Team:Baltimore_BioCrew/Description">DESCRIPTION</A></LI><LI><A href="https://2018.igem.org/Team:Baltimore_BioCrew/Design">DESIGN</A></LI><LI><A href="https://2018.igem.org/Team:Baltimore_BioCrew/Experiments">EXPERIMENTS</A></LI><LI><A href="https://2018.igem.org/Team:Baltimore_BioCrew/Notebook">NOTEBOOK</A></LI><LI><A href="https://2018.igem.org/Team:Baltimore_BioCrew/InterLab">INTERLAB</A></LI><LI><A href="https://2018.igem.org/Team:Baltimore_BioCrew/Model">MODEL</A></LI><LI><A href="https://2018.igem.org/Team:Baltimore_BioCrew/Results">RESULTS</A></LI><LI><A href="https://2018.igem.org/Team:Baltimore_BioCrew/Demonstrate">DEMONSTRATE</A></LI><LI><A href="https://2018.igem.org/Team:Baltimore_BioCrew/Attributions">ATTRIBUTIONS</A></LI><LI><A href="#">PARTS</A></LI><LI><A href="https://2018.igem.org/Team:Baltimore_BioCrew/Parts">PARTS OVERVIEW</A></LI><LI><A href="https://2018.igem.org/Team:Baltimore_BioCrew/Basic_Part">BASIC PARTS</A></LI><LI><A href="https://2018.igem.org/Team:Baltimore_BioCrew/Composite_Part">COMPOSITE PARTS</A></LI><LI><A href="https://2018.igem.org/Team:Baltimore_BioCrew/Improve">IMPROVED PARTS</A></LI><LI><A href="https://2018.igem.org/Team:Baltimore_BioCrew/Safety">SAFETY</A></LI><LI><A href="#">HUMAN PRACTICES</A></LI><LI><A href="https://2018.igem.org/Team:Baltimore_BioCrew/Human_Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2018.igem.org/Team:Baltimore_BioCrew/Public_Engagement">EDUCATION &amp; ENGAGEMENT</A></LI><LI><A href="#">AWARDS</A></LI><LI><A href="https://2018.igem.org/Team:Baltimore_BioCrew/Model">MODEL</A></LI><LI><A href="https://2018.igem.org/Team:Baltimore_BioCrew/Entrepreneurship">SUMMARY</A></LI><TITLE>Accordion Menu</TITLE></P><DIV id="teamcontent"><H1 style="color:#ffffff; background-color:#CE1620; -moz-border-radius: 15px; -webkit-border-radius: 15px; padding:15px; text-align: center"> Notebook </H1><NAV class="menu"><NAV class="drop-down-menu"><LABEL for="accordion-1" class="menu-title">June 23</LABEL><DIV class="drop-down"><H3>Clot Makers</H3><P style="margin-left: 40px">We found that we may not need to order any BioBricks, as Lethbridge only used the sequences of the parts, and then proceeded synthesized them. We decided that we want</P><P> some more time to determine what we need to send for synthesis, as we want to be exact with what we need. We also decided some different ways to continue the second construct, which Lethbridge never got to test. We want to add purification tags to both construct one (clotting blood) and construct two (localizing the clot). We could use Histags, and in construct two we need Factor 13, an antimicrobial peptide. </P><H3>Clot Breakers</H3><UL><LI>Talked about specific direction for project</LI><LI>Requested TPA part from IGEM</LI><LI>Talked about goal to reduce the side effects of TPA by making it more targeted (make TPA only function in low oxygen tension next to blood clot)</LI></UL><H3>Interlab</H3><UL><LI>Signed up for interlab (Baltimore Bio-Crew)</LI><LI>Creating contact list to purchase absorbance filter D for plate reader </LI><LI>Interviewed clot breakers and clot makers for potential assays</LI></UL><H3>Wiki</H3><UL><LI>Wrote a first draft of the project description</LI><LI>Filled out as much as possible of safety form</LI><LI>Discussed wiki design</LI></UL><H3>Human Practices</H3><P>Came up with ideas and people to contact</P></DIV></NAV><NAV class="drop-down-menu"><LABEL for="accordion-2" class="menu-title">June 30</LABEL><DIV class="drop-down"><H3>Clot Makers</H3><P style="margin-left: 40px">We began looking at places/people we could contact who we can ask for more insight into potential applications for our project. We sent an email to Dr. Lavik, who is</P><P> knowledgeable about using nanoparticles to increase the rate of coagulation and lessen the possibility of infection and inflammation. We also talked a little bit about a potential way to test the effectiveness of the enzymes we produce with our constructs. We also introduced the idea of potentially adding a numbing agent into the construct.</P><H3>Clot Breakers</H3><UL><LI>Designed and ordered primers</LI><LI>Researched more about redox switches</LI></UL><H3>Interlab</H3><UL><LI>We looked at the interlab protocol and verified that our machine has all of the settings and requirements to participate EXCEPT for the Filter D for absorbance </LI><LI>We successfully poured 20 plates with Chlor antibiotic for next Saturday transformations</LI><LI>Understanding the interlab this year and why its important</LI></UL><H3>Wiki</H3><UL><LI>Came up with some logo designs</LI><LI>Discussed colors for wiki</LI></UL></DIV></NAV><NAV class="drop-down-menu"><LABEL for="accordion-3" class="menu-title">July 7</LABEL><DIV class="drop-down"><H3>Clot Makers</H3><P style="margin-left: 40px">Did research to figure out how to best design constructs</P><H3>Clot Breakers</H3><UL><LI>Primers arrived</LI><LI>Transformed tPA biobrick into E.coli</LI></UL><H3>Interlab</H3><UL><LI>Transformed our cells w/ different parts </LI><LI>Ensured that we can use a different filter to complete interlab for plate reading protocol </LI><LI>Part of the team got to understand the plate reader and the way that we are going to obtain data </LI><LI>Made a couple Clor plates for transformations</LI></UL><H3>Wiki</H3><UL><LI>Short wiki meeting today</LI><LI>Looked at other websites for inspiration</LI></UL><H3>Human Practices</H3><UL><LI>Open house at BUGSS today</LI><LI>Sent follow-up emails</LI></UL></DIV></NAV><NAV class="drop-down-menu"><LABEL for="accordion-4" class="menu-title">July 14</LABEL><DIV class="drop-down"><H3>Clot Makers</H3><UL><LI>Designed a potential assay to test the effectiveness of the clotting agent in a simulated injury scenario</LI><LI>Discussed other assays that could potentially be used through all stages of testing</LI><LI>Ordered our constructs</LI></UL><H3>Clot Breakers</H3><P><UL><LI>Discussed gene design and looked for bacterial promoters to use</LI><LI>Need to find bacterial promoters to replace the mammalian promoter on the the TPA gene</LI><LI>Miniprepped TPA plasmids from last time using Monarch Plasmid Miniprep Kit</LI><LI>Measured concentration of DNA with nanodrop</LI> 
Nanodrop Results:
E: 57.0 ng/uL		Purity: 1.74
J: 39.8 ng/uL		Purity: 1.78
Z 75.8 ng/uL		Purity: 1.71

<LI>Digested DNA in 1 spot in order to open the plasmids so that we would be able to see the length of them on a gel later</LI><LI> Used Typical RD Protocol from NEB</LI><LI>Ran a gel electrophoresis on a 1% gel of the three digests and one lane of uncut plasmid</LI></UL></P><H3>Interlab</H3><P>Finished calibration protocol for absorbance and fluorescence to obtain needed measurement parameters for our experiment</P><P> 
Absorbance settings: 
<LI>Room temperature</LI><LI>ABS 550 on plate reader (excitation 535)</LI></P><P>
Fluorescence settings: </P><LI>Room temperature</LI><LI>Reading from top</LI><LI>Optimal reading</LI><P>
Complete protocol for absorbance/fluorescence for parts 1-6 (not controls) and recorded data</P><LI>(Incubated ½ samples for 6 hours in 37* incubator, and on ice for as 0 hour time point) </LI><LI>Made ~40 Chlor plates for next week interlab and CFU protocol</LI></DIV></NAV><NAV class="drop-down-menu"><LABEL for="accordion-5" class="menu-title">July 21</LABEL><DIV class="drop-down"><H3>Clot Makers</H3><UL><LI>Completed task list for weeks coming up</LI><LI>Edited clot making ppt for presentation on August 11</LI></UL><H3>Clot Breakers</H3><UL><LI>Tissue Plasminogen Activator (TPA) genes were designed</LI></UL><H3>Interlab</H3><P>We finished all calculations for LB+Chlor media</P><P>Finished taking absorbance and fluorescence of time point 0 hours data</P><P>We made 70% alcohol</P><P>Labeled and prepped plates and tubes for CFU protocol </P><P><LI>Example of label -1a/ - = Negative control 1= (1st set), a= first replicate</LI></P><P>Completing dilutions for protocol and then needs to plate</P><P>.1 dilution was not precise, here is the absorbance data from that (measuring 550ABS)</P></DIV></NAV><NAV class="drop-down-menu"><LABEL for="accordion-6" class="menu-title">July 28</LABEL><DIV class="drop-down"><H3>Clot Breakers</H3><UL><P>LWe checked the sequence of the TPA we amplified by aligning it to the biobrick parts sequence.
</P><P>We used Clustal Omega to do this.
</P><P>
We checked from around 580 bp to the end and everything was great (almost perfect matches).</P><P>

We also checked the quality of the sequence using a chromatography viewer.
(Quality was very good)
</P><P>
Next Step: Will request that VF2 sample be resequenced
</P><H4>Elena’s guest lecture</H4><P>
Areas to look more into:
</P><P>
Learned that tPA is very potent and only used as a treatment in high-risk/severe stroke cases. What makes it so potent? Is there a way to modify that aspect?</P><P>
Methods of delivery for tPA?  Literature shows some use for targeted treatment of blood clots. (Reference: https://www.ncbi.nlm.nih.gov/pubmed/28796540)
</P></UL></DIV></NAV><NAV class="drop-down-menu"><LABEL for="accordion-7" class="menu-title">August 4</LABEL><DIV class="drop-down"><H3>Clot Breakers</H3><P>Plasmid prep with bacterial biobrick plasmid backbone BBa-I746909 (906 bp) using Monarch Miniprep kit. We obtained plasmid concentration and purity using the Nanodrop. </P><P><LI>1: 105.7 ng/microliter, 260/280: 1.67, 260/230: .91; trial 2; 59.7, 1.79, 1.54</LI><LI>2: 72.2 ng/microliter 260/280: 1.56, 260/230: .67; trial 2: 19.8, 1.66, 1.15</LI><LI>3: 74.2 ng/microliter 260/280: 1.75, 260/230: 1.36</LI><LI>4: 73.2 ng/microliter 260/280: 1.73, 260/230: 1.15</LI></P><P>
Ran it through gel
Gene designing (shorter TPA: reteplase) for optimized expression in E. Coli 
Changing codons from mammalian preference to codons based on E.Coli codon preference. 
Lisa ordered three constructs for us: 
<LI>1) reteplase with T7 promoter</LI><LI>2) TPA with inducible T7 promoter</LI><LI>3) TPA with weak constitutively active promoter</LI></P></DIV></NAV><NAV class="drop-down-menu"><LABEL for="accordion-8" class="menu-title">August 11</LABEL><DIV class="drop-down"><H3>Clot Makers</H3><UL><LI>Recut, ligated and purified gblocks and vectors</LI><LI>Ran gel electrophoresis to verify</LI><H4>Wednesday August 15</H4><LI>Transformed the ligated genes and plated them</LI></UL><H3>Clot Breakers</H3><UL><LI>PCR amplified plasmid backbone samples</LI><LI>Ran a gel of amplified samples</LI></UL></DIV></NAV><NAV class="drop-down-menu"><LABEL for="accordion-9" class="menu-title">August 18</LABEL><DIV class="drop-down"><H2>Interlab</H2><P>Started to work with assays we created one that had the following materials DPBS,Trypsin and Acetate.
With these materials we did the following  
<LI>1.Add 27 ug fluorescein to 1.5 ml  acetate.</LI><LI>2.5 ml DPBS and 1.5 ml acetate and trypsin mix</LI><LI>3. Did 3 of 10 señal dilution
100 ml trypsin (not in well 12)
100 ml DPBS, acetate and fluorescein </LI> 
When it comes to setting the plates :
<LI>Put in plate reader</LI><LI>shake for 5 seconds</LI><LI>Room temperature</LI><LI>Read from top</LI> 
Recorded 5 times every five minutes
Result: 
<LI>Dilution 3 is most           (5/9mg/ml)</LI></P><H2>Clot Breakers</H2><P><LI>ran digestions</LI><LI>Performed PCR screening</LI><LI>purification(The final elutions were not displayed in tubes so purification will be repeated)</LI></P></DIV></NAV><NAV class="drop-down-menu"><LABEL for="accordion-10" class="menu-title">August 25</LABEL><DIV class="drop-down"><H3>Clot Makers</H3><UL><P>Performed colony PCR screening, with a couple potential positives.
</P></UL></DIV></NAV><NAV class="drop-down-menu"><LABEL for="accordion-11" class="menu-title">September 1</LABEL><DIV class="drop-down"><H2>Clot Makers</H2><P>Miniprepped genes and sent them to Johns Hopkins for screening</P><H2>Clot Breakers</H2><P>set up four ligations:
<LI>TPA-T7 (from gblock)</LI><LI>TPA weak (from gblock)</LI><LI>Reteplase (from gblock)</LI><LI>Mammalian TPA (from PCR of biobrick)</LI></P></DIV></NAV><NAV class="drop-down-menu"><LABEL for="accordion-12" class="menu-title">September 8</LABEL><DIV class="drop-down"><H2>Clot Makers</H2><P>Redigested and ligated g-blocks</P><H2>Clot Breakers</H2><P>Observations of plates:
<LI>about 3 colonies on</LI><LI>about 5 colonies on </LI><LI>no colonies on Reteplase</LI><LI>dozens of colonies on mammalian TPA plates</LI></P><P>Colony PCR screen to verify that insert is present
</P></DIV></NAV><NAV class="drop-down-menu"><LABEL for="accordion-13" class="menu-title">September 15</LABEL><DIV class="drop-down"><H2>Clot Makers</H2><P>Redigested and ligated g-blocks</P><H2>Clot Breakers</H2><P>Colony PCR screen
https://www.neb.com/protocols/2012/09/06/protocol-for-onetaq-2x-master-mix-with-standard-buffer-m0482

There were many colonies on Reteplase today.

Observations:
<LI>2 colonies on TPA weak</LI><LI>4 colonies on T7</LI><LI>Dozens of colonies on Reteplase 1:2</LI><LI>Around a dozen from Reteplase 1:1</LI></P><P>
Colony PCR screen to check for insert
</P></DIV></NAV><NAV class="drop-down-menu"><LABEL for="accordion-14" class="menu-title">September 22</LABEL><DIV class="drop-down"><H2>Clot Makers</H2><P>Screened colonies with colony PCR</P><P>
Various positive samples that need to be sequenced to determine whether they’re our samples or not
</P><H2>Clot Breakers</H2><P>looked at sequencing results
<LI>The Clot Makers also had issues when using the linearized vector.  Following their lead, we’ll try ligating our 3 inserts into the BBxxxx vector (contains RFP)</LI></P><P>3 reactions -- T7-TPA, weak-TPA, reteplase</P></DIV></NAV><NAV class="drop-down-menu"><LABEL for="accordion-15" class="menu-title">September 29</LABEL><DIV class="drop-down"><H2>Clot Makers</H2><P>Conducted PCR Screening on our next set of potential positives and sent the rear third of construct to be sequenced</P><H2>Clot Breakers</H2><P> No colonies found on reteplase 
16 colonies on weakpromoter-TPA
10 colonies on T7-TPA

check for insert using colony PCR screen
<LI>select 5 colonies from each of two plates (10 colonies total)</LI><LI>primers VF2 and VR; NEB oneTaq</LI></P></DIV></NAV><NAV class="drop-down-menu"><LABEL for="accordion-16" class="menu-title">October 6</LABEL><DIV class="drop-down"><H3>Clot Makers</H3><P>Prepped for a His Tag Screening. Decided to use samples 4-3 and 1-5 (and some of 1-4). Didn’t finish.
</P><H2>Clot Breakers</H2><P>Performed PCR cleanup of 9 colonies</P><P>
Concentrated samples using NanoSep
</P></DIV></NAV><NAV class="drop-down-menu"><LABEL for="accordion-17" class="menu-title">October 10</LABEL><DIV class="drop-down"><H3>Clot Makers</H3><P>Finished His Tag Screening and ran a protein gel.
</P></DIV></NAV></NAV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>