<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Chalmers-Gothenburg_Project skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Chalmers-Gothenburg/Project</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Interlab - iGEM Chalmers-Gothenburg 2018</TITLE></P><NAV id="navbar" class="mynav navbar-expand-md bg-dark navbar-dark sticky-top"><DIV class="collapse navbar-collapse justify-content-around" id="collapsibleNavbar"><UL class="navbar-nav align-items-center"><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" id="navbardrop-project" role="button" data-toggle="dropdown">
        PROJECT
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Description" style="color:#16181b">Project Overview</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Project" style="color:#16181b">Project Summary</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Achievements" style="color:#16181b">Achievements</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" id="navbardrop-wetlab" data-toggle="dropdown">
        LABWORK
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Attachment" style="color:#16181b">Binding</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Detection" style="color:#16181b">Detection</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Treatment" style="color:#16181b">Treatment</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/InterLab" style="color:#16181b">InterLab</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Parts" style="color:#16181b">BioBricks</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Protocols" style="color:#16181b">Protocols</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Safety" style="color:#16181b">Safety</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" id="navbardrop-hp" data-toggle="dropdown">
        HUMAN PRACTICES
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Human_Practices" style="color:#16181b">Human Practices</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Collaborations" style="color:#16181b">Collaborations</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Outreach" style="color:#16181b">Outreach</A></DIV></LI></UL><UL class="navbar-nav align-items-center"><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" id="navbardrop-model" data-toggle="dropdown">
        MODELING
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Model" style="color:#16181b">Model Overview</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/ModelConstruction" style="color:#16181b">Model Design</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" id="navbardrop-team" data-toggle="dropdown">
        PEOPLE
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Team" style="color:#16181b">Team</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Supervisors" style="color:#16181b">Supervisors</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Sponsors" style="color:#16181b">Sponsors</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Attributions" style="color:#16181b">Attributions</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" id="navbardrop-social" data-toggle="dropdown">
        SOCIAL MEDIA
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://www.facebook.com/chalmersigem" style="color:#16181b">Facebook</A><A class="dropdown-item" href="https://www.instagram.com/chalmersigem" style="color:#16181b">Instagram</A><A class="dropdown-item" href="https://twitter.com/igemchalmers" style="color:#16181b">Twitter</A></DIV></LI></UL></DIV></NAV><DIV class="container-fluid" style="margin-top:30px;"><DIV class="row"><DIV class="col-lg-2"><DIV class="d-none d-lg-block" style="position:sticky; top:130px;"><H4>Navigation</H4><NAV id="mySidebar"><UL class="nav nav-pills flex-column"><LI class="nav-item"><A class="nav-link" href="#Anchor">Attachment</A></LI><LI class="nav-item"><A class="nav-link" href="#TargetM">Treatment</A></LI><LI class="nav-item"><A class="nav-link" href="#GasV">Monitoring</A></LI><LI class="nav-item"><A class="nav-link" href="#org">Organism</A></LI><LI class="nav-item"><A class="nav-link" href="#OurProduct">Product</A></LI></UL></NAV></DIV></DIV><DIV class="col-lg-8" style="text-align:justify"><DIV class="row"><DIV class="col-12"><H1 style="text-align: center;"> Project Summary</H1><P style="text-align: justify;">Our product is a pill containing a genetically engineered organism that can be used to detect and treat colon cancer. The organism should be able to localize cancer cells in the colon, locally kill the cells and report the presence of cancer. For this purpose, we have chosen to work with the yeast <I>Saccharomyces boulardii</I> and divided our project into three parts. The first part of the project concerns cancer cell localization, which is achieved with yeast to cancer cell anchoring. By making the yeast express Histone-like protein A on its surface it can bind to cancer cells specifically. In the second part of the project, we make the yeast secrete a target molecule with the purpose to treat cancer. Here, we exploit the native mating system of yeast in order to express the target molecule only when the yeast has accumulated around a tumor. Lastly, in order for the yeast to be able to report the presence of cancer cells, we make it express gas vesicles that can be detected with the help of ultrasound. </P><H1 style="text-align: center;">Yeast Cell to Cancer Cell Attachment</H1><P style="text-align: justify;">In order to make it possible to treat the cancer locally, as well as to detect the tumor location, the yeast needs to bind specifically to the colon cancer cells. This is made possible through the use of Histone like protein A (HlpA), which is originally from the colon cancer associated (Boleji et al. 2009; Ho et al., 2018) bacteria <I>Streptococcus gallolyticus</I>. HlpA binds to heparan sulfate proteoglycans (HSPG), more specifically Syndecan 1 and possibly also Syndecan 2, on the surface of the colon cancer cell (Ho et al., 2018). In this way <I>S. gallolyticus</I> has been found to adhere to colon cancer cells and then infiltrate colon cancer tumors (Boleji et al.2009).</P><H2>Yeast Surface Anchor Protein</H2><P style="text-align: justify;">To express HlpA at the <I>S. boulardii</I> cell surface, an anchoring protein is needed since HlpA does not naturally adhere to the yeast cell surface. For this purpose, Alpha-Agglutin is used, since it is the most commonly used C-terminal anchor for expression of recombinant proteins at the yeast surface (Tanaka and Kondo, 2015). Alpha-Agglutin consists of two subunits, Aga1, and Aga2 (Tanaka and Kondo, 2015). Aga1 is expressed in the cell wall of the yeast, while Aga2 is secreted and then binds to the to Aga1 with a double disulfide bond (Tanaka and Kondo, 2015). To anchor HlpA at the yeast surface, the N-terminal of the protein is fused to the C-terminal of Aga2. A flexible linker is added in between the proteins, which exposes HlpA on the yeast surface without altering the active site (Chen et at. 2013; Ho et al., 2018), as can be seen in figure 1.1.</P><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 1.1:</B> Illustration of how HlpA is anchored at the yeast surface.</I></FIGCAPTION></FIGURE><H2>Cancer Cell Surface Target Proteins</H2><P style="text-align: justify;">As previously mentioned, HlpA binds to HSPGs present at the the cancer cell's surface. HSPGs are in fact present at the surface of all epithelial cells in the gut, but usually, they are not exposed in a way that allows for the HlpA to bind effectively (Ho et al., 2018). However, in some cancer cell lines, there is a loss of apicobasal polarity, as well as changes in HSPG expression (Boleji et al. 2009; Ho et al., 2018). This results in Syndecan 1 and 2 being exposed at the cell surface, and thereby it allows for HlpA to bind (Ho et al., 2018). Since the HSPGs in healthy cells are less exposed, this results in higher adherence to colorectal cancer cells compared to healthy cells.</P><H1 style="text-align: center;">Target Molecule</H1><P style="text-align: justify;">In this project, the yeast interacts with the cancer cells by producing one of two cancer toxins: Myrosinase or p28. Both of these proteins inhibit the cancer's cell cycle and thus lead to a decrease in cell proliferation (Lulla et al., 2016a; Li et al., 2010). The final effect of these proteins is fairly similar but the mechanism they utilize is completely different: p28 directly interacts with the cancer whereas myrosinase is an enzyme that catalyzes a reaction, which produces a cancer toxin. Both affect cancer cells while not having adverse effects on healthy cells.</P><H2>p28</H2><P style="text-align: justify;">The peptide p28 is hydrophobic and composed of only 28 amino acids, hence the name. These characteristics allow it to penetrate cell membranes through diffusion (NCBI, n.d.). This means that no active transport is needed for the protein to get into the target cells. It is often referred to as a cell penetrating peptide (CPP). When the peptide has made its way into the cell, it interacts with p53, a protein that plays a major part in regulating cell division and apoptosis (Vogelstein, 2000). Indeed, p28 inhibits the ubiquitination of p53, by for example COP1, which leads to a higher intracellular concentration of p53, which in turn has a cell cycle regulating function(Dornan et al., 2018). A schematic of how p28 interacts with p53 and cellular mechanisms can be seen in figure 2.1. It is very common for cancer cells to have a mutation in this protein; approximately 50% of cancers stem from a mutation in the p53 gene (Ozaki, 2011). This makes it an excellent target for treatment as it covers a majority of all human cancer cases. During the duration of the project, it was decided to drop using p28 as a toxin. This was due to a strict time limit for the laboratory work and based on our modeling results. These results indicated that p28 is less effective than our other chosen cancer toxin, Myrosinase.</P><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 2.1:</B> p28 effect on tumor cell cycle and apoptosis</I></FIGCAPTION></FIGURE><H2>Myrosinase</H2><P style="text-align: justify;"> Although Myrosinase does not have a direct impact on the cancer cells, it is an enzyme that catalyses the reaction of glucosinolate to sulforaphane, which is shown in figure 2.2. The product, sulforaphane, has been shown to possess anticancer properties. Myrosinase is coupled to a secretion peptide for it to be present in the media and to interact with the glucosinolates (Clarke, 2008). This implies that this treatment needs to be accompanied by addition of glucosinolate to have an effect. Glucosinolate is naturally occuring in cruciferous vegetable, such as broccoli and cauliflower. A diet rich in these vegetables would thus be enough to provide the myrosinase with enough substrate to be effective.</P><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 2.2:</B> Reaction schematic of glucusinolate to sulforaphane as catalysed by myrsoinase</I></FIGCAPTION></FIGURE><P style="text-align: justify;"> The mechanism through which sulforaphane interacts with cancer cells is more complex than for p28. However, as mentioned earlier the end results are similar; it interacts with proteins that regulate either the cell cycle or apoptosis. See figure 2.3 for a general schematic of the metabolic interactions of sulforaphane.</P><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 2.3:</B> Sulforaphane's effect on tumor apoptosis through intermediates such as Bax, p21, and cytochrome C (CytC)</I></FIGCAPTION></FIGURE><H2>Pheromone sensing</H2><P style="text-align: justify;"> One of the major limitation of drug development is the possibility of off-target effects; drugs with too severe off-target effects cannot be used safely. Whilst the toxins used in this project should not have a negative effect on healthy cells, it is still desirable to keep the chance of adverse effects to a minimum. Therefore, a pheromone sensing system is implemented into the yeast genome via a CRISPR-Cas9 system. There are actually two mating types of yeast, the a-type and the alpha-type. It is possible for these two types to interact with each other via a pheromone sensing system,  the a-type yeast produces a-pheromone whereas the alpha-type produce alpha-pheromone and the mating types are sensitive to the opposite pheromone type (Williams, 2016). The gene <A href="https://www.yeastgenome.org/locus/S000001277">BAR1,</A>which codes for a protein that inactivates pheromones, is deleted by this CRISPR-Cas9 system and replaced via homologous recombination by <A href="https://www.yeastgenome.org/locus/S000003057">MF(ALPHA)2,</A>which is the gene coding for the alpha-pheromone. A full schematic for how the pheromone sensing system looks like in our project can be seen in figure 2.4.</P><P style="text-align: justify;">In nature, the yeast uses this system to find a partner to mate with. However, in this project the aim is to couple protein production to this system and to have the production be dependant on the concentration of pheromone. This is done through the implementation of the naturally-occuring FUS1 promoter (pFUS1) which responds to yeast pheromones (Williams, 2013). A high pheromone concentration only occurs with a high yeast concentration. When ingested the yeast will not reach such high concentrations in a healthy patient. However, keeping the yeasts ability to attach to cancer cells in mind, the yeast will amass around tumors and thus activate the production of our protein of interest.</P><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 2.4:</B> Simplified Schematic of the engineered pheromone sensing system</I></FIGCAPTION></FIGURE><H1 style="text-align: center;">Detection</H1><H2>Natural Buoyancy</H2><P style="text-align: justify;">Gas vesicles are hollow proteins complexes that take cylindrical shapes of 45 to 200 nm diameter (Walsby, 1994). In nature, gas vesicles are produced by a variety of aquatic microorganisms to control their buoyancy in water. A well-conserved operon of more than 10 genes is responsible for the production of these vesicles. The two most important genes are the genes Gas Vesicle Protein A or its homolog B and Gas Vesicle Protein C (GvpA/B and GvpC). They form the rigid structure of the gas vesicles (Englert &amp; Pfeiffer, 1993), which can be seen in figure 3.1. The gas vesicles are permeable to gasses but not liquids (Sivertse et al, 2010). Thus, from the cytoplasm, only gas will diffuse into the vesicles. The other genes present in the operon are less known but are thought to be mainly involved in regulation, folding and gas vesicle assembly. These proteins do not appear in the final structure (Sivertse et al, 2010).</P><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 3.1:</B> “Electron micrograph of a gas vesicle of Anabaena flos-aquae” from Daviso, Belenky, Griffin, &amp; Herzfeld (2013)</I></FIGCAPTION></FIGURE><H2>Acoustic Reporter Gene</H2><P style="text-align: justify;">An interesting property of these gas vesicles is that, additionally to their original buoyancy function, they also refract ultrasound waves (Bourdeau et al., 2018). This property has been found to be useful in synthetic biology; Bourdeau et al. (2018) have shown that organisms can be engineered to produce gas vesicles as acoustic reporter genes. Gas vesicles can be observed non-invasively, even through non-transparent tissue, with ultrasound imaging instruments present in most modern hospitals. This could be done as represented in figure 3.2.</P><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 3.2:</B> &quot;Diagram of gastrointestinal (GI) imaging experiment&quot; from Bourdeau et al. (2018)</I></FIGCAPTION></FIGURE><P style="text-align: justify;">In their study, Bourdeau et al. (2018) have combined gas vesicle genes from <I>Bacillus megaterium</I> and <I>Anabaena flos-aquae</I> to produce the acoustic reporter gene in <I>Escherichia coli</I> as represented in figure 3.3.</P><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 3.3:</B> &quot;Organization of acoustic reporter gene clusters&quot; from Bourdeau et al. (2018)</I></FIGCAPTION></FIGURE><H2>Operon Expression in Yeast</H2><P style="text-align: justify;">Yeast, as a eukaryote, is unable to transcribe gene operons. To express these genes, a multicistronic gene expression system based on 2A viral peptides is used. The 2A peptides cleaves itself after transcription and separates the genes (Souza-Moreira et al. 2018), as shown in figure 3.4. It is placed in between each gene of the operon. Without this system it would also be possible to express each gene separately, however, each gene should then be introduced under its own promoter and terminator while here, the gene set can be combined under one promoter and one terminator.</P><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 3.4:</B> &quot;&quot;Self-cleavage mechanism of 2A self-cleaving peptide&quot; from Wang et al. (2015&gt;</I></FIGCAPTION></FIGURE><H1 style="text-align: center;">Organism: <I>Saccharomyces boulardii</I></H1><P style="text-align: justify;">For our product to work, we need to implement it in an organism that can survive in the gut of the patient, but that does not have pathogenic properties. Since the yeast <I>Saccharomyces cerevisiae</I> is commonly used in our lab, the organism of choice would be this yeast. However, <I>S. cerevisiae</I> is not adapted to the pH variations and higher temperatures in the human gut environment (Palma et al., 2015). It is therefore not the best candidate organism for our product. Alternatively, the probiotic yeast <I>Saccharomyces boulardii</I>, a subspecies of <I> S.cerevisiae</I>, is better adapted to the gut pH variations and has an optimal growth temperature of 37°C (Czerucka et al., 2007; Edwards-Ingram et al., 2007; Palma et al. 2015; Liu et al. 2016). <I>S. boulardii</I> already has a GRAS (Generally Regarded As Safe) status for several different applications, and the genetic makeup is very similar to that of <I>S.cerevisiae</I>, apart from differences in gene copy numbers (Edwards-Ingram et al., 2007; Liu et al., 2016). Since we are testing our system in <I>S. cerevisiae</I> in the lab, <I> S.boulardii</I> is a fitting target organism for our system. Right now, several alternative probiotic <I>S. cerevisiae</I> strains are also under development, but none of these have a GRAS status at the moment (Palma et al.,2015). In the future, our system could also be implemented in these yeast strains. </P><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 4.1:</B> Brewer's yeast Saccharomyces boulardii Probiotic Candidiasis Fungus - yeast illustration @kisspng</I></FIGCAPTION></FIGURE><H1 style="text-align: center;">Our Product</H1><P style="text-align: justify;">Our genetically modified probiotic yeast will target colon cancerous cells. Thus, it has to be delivered to the colon. Our idea is to put our yeast in a capsule that, passing unaffected through the gastrointestinal tract, will dissolve inside the colon releasing the microorganism. This delivery system has to be designed taking into consideration the gastrointestinal physiology (pH, microflora, enzymes, different fluid volumes and transit times) and also the increase of complexity induced by food. Furthermore, the inner part of the capsule should also provide an optimal environment for the yeast to survive and activate when it reaches the colon.
The first idea would be to deliver the probiotic via rectal route because it is the shortest route. However, it is difficult to reach the colon and patients could find it uncomfortable (Philip &amp; Philip, 2010). Since the aim is to create a medicine as practical as possible, an oral delivery route is preferable.</P><H2>Pulsincap System</H2><P style="text-align: justify;">For colon drug delivery, it can be useful to combine the effect of a PH sensitive and a time-released system. This is what Abraham &amp; Srinath (2007) did in their Pulsincap system. This capsule is designed by three different parts and a coating:</P><UL class="mylist"><LI style="text-align: justify;">Water insoluble body: hard gelatin body treated with formaldehyde </LI><LI style="text-align: justify;">Hydrogel plug made of polymers such as: guar gum, hydroxypropylmethylcellulose 10K, carboxymethylcellulose sodium and sodium alginate (Abraham &amp; Srinath, 2007)
</LI><LI style="text-align: justify;">Water soluble cap: hydroxypropyl methylcellulose (HPMC)(‘Gastrointestinal-specific multiple drug release system’, 2005)
</LI><LI style="text-align: justify;">Acid insoluble coating: 5% cellulose acetate phthalate
</LI></UL><P><FIGURE style="text-align:center;"><FIGCAPTION><B>Figure 5.1:</B> Summary of the PULSINCAP system ideated by Abraham &amp; Srinath (2007) that inspired our product design</FIGCAPTION></FIGURE></P><P style="text-align: justify;">Combined with the capsule design, the probiotic yeast has to be prepared with a specific method and then placed inside of the capsule. We propose the same method that Hébrard G et.al used with S. boulardii: yeast is mixed 2:1 with whey protein and alginate, microparticles are formed through extrusion/cold gelation and coated with WP or ALG by immersion (Hébrard et al., 2010).</P><H1 style="text-align:center;">References</H1></DIV></DIV><DIV class="row" style="font-size:70%;"><DIV class="col-6"><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Abraham, S., &amp; Srinath, M. (2007). Development of modified pulsincap drug delivery system of metronidazole for drug targeting. <EM>Indian Journal of Pharmaceutical Sciences</EM>, <EM>69</EM>(1), 24. https://doi.org/10.4103/0250-474X.32102</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Boleij, A., Schaeps, R. M. J., de Kleijn, S., Hermans, P. W., Glaser, P., Pancholi, V., … Tjalsma, H. (2009). Surface-Exposed Histone-Like Protein A Modulates Adherence of Streptococcus gallolyticus to Colon Adenocarcinoma Cells. <EM>Infection and Immunity</EM>, <EM>77</EM>(12), 5519–5527. https://doi.org/10.1128/IAI.00384-09</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Bourdeau, R. W., Lee-Gosselin, A., Lakshmanan, A., Farhadi, A., Kumar, S. R., Nety, S. P., &amp; Shapiro, M. G. (2018). Acoustic reporter genes for noninvasive imaging of microorganisms in mammalian hosts. <EM>Nature</EM>. https://doi.org/10.1038/nature25021</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Chen, X., Zaro, J. L., &amp; Shen, W.-C. (2013). Fusion protein linkers: Property, design and functionality. <EM>Advanced Drug Delivery Reviews</EM>, <EM>65</EM>(10), 1357–1369. https://doi.org/10.1016/j.addr.2012.09.039</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Cheng, B., Montmasson, M., Terradot, L., &amp; Rousselle, P. (2016). Syndecans as Cell Surface Receptors in Cancer Biology. A Focus on their Interaction with PDZ Domain Proteins. <EM>Frontiers in Pharmacology</EM>, <EM>7</EM>. https://doi.org/10.3389/fphar.2016.00010</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Clarke, J. D., Dashwood, R. H., &amp; Ho, E. (2008). Multi-targeted prevention of cancer by sulforaphane. <EM>Cancer Letters</EM>, <EM>269</EM>(2), 291–304. https://doi.org/10.1016/j.canlet.2008.04.018</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">CZERUCKA, D., PICHE, T., &amp; RAMPAL, P. (2007). Review article: yeast as probiotics -Saccharomyces boulardii. <EM>Alimentary Pharmacology &amp; Therapeutics</EM>, <EM>26</EM>(6), 767–778. https://doi.org/10.1111/j.1365-2036.2007.03442.x</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Daviso, E., Belenky, M., Griffin, R. G., &amp; Herzfeld, J. (2013). Gas Vesicles across Kingdoms: A Comparative Solid-State Nuclear Magnetic Resonance Study. <EM>Journal of Molecular Microbiology and Biotechnology</EM>, <EM>23</EM>(4–5), 281–289. https://doi.org/10.1159/000351340</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Dornan, D., Wertz, I., Shimizu, H., Arnott, D., Frantz, G. D., Dowd, P., … Dixit, V. M. (2004). The ubiquitin ligase COP1 is a critical negative regulator of p53. <EM>Nature</EM>, <EM>429</EM>(6987), 86–92. https://doi.org/10.1038/nature02514</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Dornan, D., Wertz, I., Shimizu, H., Arnott, D., Frantz, G. D., Dowd, P., … Dixit, V. M. (2004). The ubiquitin ligase COP1 is a critical negative regulator of p53.<EM>Nature</EM>, <EM>429</EM>(6987), 86–92. https://doi.org/10.1038/nature02514</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Edwards-Ingram, L., Gitsham, P., Burton, N., Warhurst, G., Clarke, I., Hoyle, D., … Stateva, L. (2007). Genotypic and Physiological Characterization of Saccharomyces boulardii, the Probiotic Strain of Saccharomyces cerevisiae. <EM>Applied and Environmental Microbiology</EM>, <EM>73</EM>(8), 2458–2467. https://doi.org/10.1128/AEM.02201-06</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Elmer, McFarland, Surawicz, Danko, &amp; Greenberg. (1999). Behaviour of Saccharomyces boulardii in recurrent Clostridium difficile disease patients. <EM>Alimentary Pharmacology and Therapeutics</EM>, <EM>13</EM>(12), 1663–1668. https://doi.org/10.1046/j.1365-2036.1999.00666.x</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Englert, C., &amp; Pfeiffer, F. (1993). Analysis of gas vesicle gene expression in Haloferax mediterranei reveals that GvpA and GvpC are both gas vesicle structural proteins. <EM>The Journal of Biological Chemistry</EM>, <EM>268</EM>(13), 9329–9336. Retrieved from http://www.jbc.org/content/268/13/9329.short</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Hébrard, G., Hoffart, V., Beyssac, E., Cardot, J.-M., Alric, M., &amp; Subirade, M. (2010). Coated whey protein/alginate microparticles as oral controlled delivery systems for probiotic yeast. <EM>Journal of Microencapsulation</EM>, <EM>27</EM>(4), 292–302. https://doi.org/10.3109/02652040903134529</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Ho, C. L., Tan, H. Q., Chua, K. J., Kang, A., Lim, K. H., Ling, K. L., … Chang, M. W. (2018). Engineered commensal microbes for diet-mediated colorectal-cancer chemoprevention. <EM>Nature Biomedical Engineering</EM>, <EM>2</EM>(1), 27–37. https://doi.org/10.1038/s41551-017-0181-y</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Li, Y., Zhang, T., Korkaya, H., Liu, S., Lee, H.-F., Newman, B., … Sun, D. (2010). Sulforaphane, a dietary component of broccoli/broccoli sprouts, inhibits breast cancer stem cells. <EM>Clinical Cancer Research : An Official Journal of the American Association for Cancer Research</EM>, <EM>16</EM>(9), 2580–2590. https://doi.org/10.1158/1078-0432.CCR-09-2937</P></DIV><DIV class="col-6"><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Liu, J.-J., Kong, I. I., Zhang, G.-C., Jayakody, L. N., Kim, H., Xia, P.-F., … Jin, Y.-S. (2016). Metabolic Engineering of Probiotic Saccharomyces boulardii. <EM>Applied and Environmental Microbiology</EM>, <EM>82</EM>(8), 2280–2287. https://doi.org/10.1128/AEM.00057-16</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Lulla, R. R., Goldman, S., Yamada, T., Beattie, C. W., Bressler, L., Pacini, M., … Fouladi, M. (2016). Phase I trial of p28 (NSC745104), a non-HDM2-mediated peptide inhibitor of p53 ubiquitination in pediatric patients with recurrent or progressive central nervous system tumors: A Pediatric Brain Tumor Consortium Study. <EM>Neuro-Oncology</EM>, <EM>18</EM>(9), 1319–1325. https://doi.org/10.1093/neuonc/now047</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Lulla, R. R., Goldman, S., Yamada, T., Beattie, C. W., Bressler, L., Pacini, M., … Fouladi, M. (2016). Phase I trial of p28 (NSC745104), a non-HDM2-mediated peptide inhibitor of p53 ubiquitination in pediatric patients with recurrent or progressive central nervous system tumors: A Pediatric Brain Tumor Consortium Study. <EM>Neuro-Oncology</EM>, <EM>18</EM>(9), 1319–1325. https://doi.org/10.1093/neuonc/now047</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">NCBI. (n.d.). NCBI - Azurin:50-77 Cell Penetrating Peptide p28. Retrieved from https://www.ncbi.nlm.nih.gov/medgen/443551</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Ozaki, T., &amp; Nakagawara, A. (2011). Role of p53 in Cell Death and Human Cancers. <EM>Cancers</EM>, <EM>3</EM>(1), 994–1013. https://doi.org/10.3390/cancers3010994</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Palma, M. L., Zamith-Miranda, D., Martins, F. S., Bozza, F. A., Nimrichter, L., Montero-Lomeli, M., … Douradinha, B. (2015). Probiotic Saccharomyces cerevisiae strains as biotherapeutic tools: is there room for improvement? <EM>Applied Microbiology and Biotechnology</EM>, <EM>99</EM>(16), 6563–6570. https://doi.org/10.1007/s00253-015-6776-x</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Philip, A., &amp; Philip, B. (2010). Colon Targeted Drug Delivery Systems: A Review on Primary and Novel Approaches. <EM>Oman Medical Journal</EM>, <EM>25</EM>(2), 70–78. https://doi.org/10.5001/omj.2010.24</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Sivertsen, A. C., Bayro, M. J., Belenky, M., Griffin, R. G., &amp; Herzfeld, J. (2010). Solid-State NMR Characterization of Gas Vesicle Structure. <EM>Biophysical Journal</EM>, <EM>99</EM>(6), 1932–1939. https://doi.org/10.1016/J.BPJ.2010.06.041</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Souza-Moreira, T. M., Navarrete, C., Chen, X., Zanelli, C. F., Valentini, S. R., Furlan, M., … Krivoruchko, A. (2018). Screening of 2A peptides for polycistronic gene expression in yeast. <EM>FEMS Yeast Research</EM>, <EM>18</EM>(5). https://doi.org/10.1093/femsyr/foy036</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Souza-Moreira, T. M., Navarrete, C., Chen, X., Zanelli, C. F., Valentini, S. R., Furlan, M., … De Montreal User, U. (2018). Screening of 2a Peptides for Polycistronic Gene Expression in Yeast, <EM>46</EM>(May 2018). https://doi.org/10.1093/femsyr/foy036/4956763</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Tanaka, T., &amp; Kondo, A. (2014). Cell-surface display of enzymes by the yeast <EM>Saccharomyces cerevisiae</EM> for synthetic biology. <EM>FEMS Yeast Research</EM>, n/a-n/a. https://doi.org/10.1111/1567-1364.12212</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Vogelstein, B., Lane, D., &amp; Levine, A. J. (2000). Surfing the p53 network. <EM>Nature</EM>, <EM>408</EM>(6810), 307–310. https://doi.org/10.1038/35042675</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Walsby, A. E. (1994). Gas vesicles. <EM>Microbiological Reviews</EM>, <EM>58</EM>(1), 94–144. Retrieved from http://www.ncbi.nlm.nih.gov/pubmed/8177173</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Wang, Y., Wang, F., Wang, R., Zhao, P., &amp; Xia, Q. (2015). 2A self-cleaving peptide-based multi-gene expression system in the silkworm Bombyx mori. <EM>Scientific Reports</EM>, <EM>5</EM>(1), 16273. https://doi.org/10.1038/srep16273</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Williams, T. C., Nielsen, L. K., &amp; Vickers, C. E. (2013). Engineered Quorum Sensing Using Pheromone-Mediated Cell-to-Cell Communication in <EM>Saccharomyces cerevisiae</EM>. <EM>ACS Synthetic Biology</EM>, <EM>2</EM>(3), 136–149. https://doi.org/10.1021/sb300110b</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Williams, T. C., Peng, B., Vickers, C. E., &amp; Nielsen, L. K. (2016). The Saccharomyces cerevisiae pheromone-response is a metabolically active stationary phase for bio-production. <EM>Metabolic Engineering Communications</EM>, <EM>3</EM>, 142–152. https://doi.org/10.1016/J.METENO.2016.05.001</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;"> </P></DIV></DIV></DIV></DIV></DIV><DIV id="footer"><FOOTER><H4 style="font-size:12pt;">
    CONTACT INFORMATION: 
  </H4><DIV><P style="font-size:10pt;">
      CHALMERS University of Technology, Gothenburg, Sweden, igemgothenburg@gmail.com   
</P><SPAN><A style="font-size:10pt;" href="https://www.facebook.com/chalmersigem">Facebook</A><A style="font-size:10pt;" href="https://www.instagram.com/chalmersigem">Instagram</A><A style="font-size:10pt;" href="https://twitter.com/igemchalmers">Twitter</A></SPAN></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>