<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Chalmers-Gothenburg_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Chalmers-Gothenburg/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Model- iGEM Chalmers-Gothenburg 2018</TITLE></P><NAV id="navbar" class="mynav navbar-expand-md bg-dark navbar-dark sticky-top"><DIV class="collapse navbar-collapse justify-content-around" id="collapsibleNavbar"><UL class="navbar-nav align-items-center"><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" id="navbardrop-project" role="button" data-toggle="dropdown">
        PROJECT
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Description" style="color:#16181b">Project Overview</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Project" style="color:#16181b">Project Summary</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Achievements" style="color:#16181b">Achievements</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" id="navbardrop-wetlab" data-toggle="dropdown">
        LABWORK
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Attachment" style="color:#16181b">Binding</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Detection" style="color:#16181b">Detection</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Treatment" style="color:#16181b">Treatment</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/InterLab" style="color:#16181b">InterLab</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Parts" style="color:#16181b">BioBricks</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Protocols" style="color:#16181b">Protocols</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Safety" style="color:#16181b">Safety</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" id="navbardrop-hp" data-toggle="dropdown">
        HUMAN PRACTICES
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Human_Practices" style="color:#16181b">Human Practices</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Collaborations" style="color:#16181b">Collaborations</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Outreach" style="color:#16181b">Outreach</A></DIV></LI></UL><UL class="navbar-nav align-items-center"><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" id="navbardrop-model" data-toggle="dropdown">
        MODELING
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Model" style="color:#16181b">Model Overview</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/ModelConstruction" style="color:#16181b">Model Design</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" id="navbardrop-team" data-toggle="dropdown">
        PEOPLE
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Team" style="color:#16181b">Team</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Supervisors" style="color:#16181b">Supervisors</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Sponsors" style="color:#16181b">Sponsors</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Attributions" style="color:#16181b">Attributions</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" id="navbardrop-social" data-toggle="dropdown">
        SOCIAL MEDIA
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://www.facebook.com/chalmersigem" style="color:#16181b">Facebook</A><A class="dropdown-item" href="https://www.instagram.com/chalmersigem" style="color:#16181b">Instagram</A><A class="dropdown-item" href="https://twitter.com/igemchalmers" style="color:#16181b">Twitter</A></DIV></LI></UL></DIV></NAV><DIV class="container-fluid" style="margin-top:30px;"><DIV class="row"><DIV class="col-lg-2"><DIV class="d-none d-lg-block" style="position:sticky; top:130px;"><H4>Navigation</H4><NAV id="mySidebar"><UL class="nav nav-pills flex-column"><LI class="nav-item"><A class="nav-link" href="#Objective">Modeling Objective</A></LI><LI class="nav-item"><A class="nav-link" href="#Elements">Modeled Elements</A></LI><LI class="nav-item"><A class="nav-link" href="#KineticCC">Cell Cycle Model</A></LI><LI class="nav-item"><A class="nav-link" href="#KineticAP">α Pheromone Model</A></LI><LI class="nav-item"><A class="nav-link" href="#COMdFBA">Community dFBA</A></LI><LI class="nav-item"><A class="nav-link" href="#Integrated">Integrated Modeling</A></LI><LI class="nav-item"><A class="nav-link" href="#Future">Future Work</A></LI></UL></NAV></DIV></DIV><DIV class="col-lg-8" style="text-align:justify"><DIV class="row"><DIV class="col-12"><H1 style="text-align: center;">Modeling Objective</H1><P>Before elaborating on the modeling part of this iGEM project, a good question to ask is why would we model our project? In general, creating a mathematical model for synthetic biology can give precious insight into a complex system, which ideally leads to predictive power over the outcome of this system and allows the improvement of the experimental design. </P><P>As described by Ledley (1960), modeling is a cyclical and dynamic process where the formulation of the model is repetitively changed to account for new literature data, experimental data and previous modeling outcomes. This new formulation yields a new outcome, which in turn can serve as a basis for a new modeling cycle or an improvement of the experimental design (see figure 1). </P><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 1:</B> “the pentagon of research” (Ledley, 1960)</I></FIGCAPTION></FIGURE><H1 style="text-align: center;">Modeled Elements</H1><P>In this project, several aspects can be modeled. Because the yeast is modified to produce anti-cancer proteins as a response of an engineered pheromone sensing feedback loop, one model of interest is the pheromone production and signaling cascade and its corresponding anti-cancer protein production (see blue dotted line in figure 2). Another model consists of the anti-cancer protein effect on the cancer cells, specifically the effect on the cell cycle of the cancer cells (see green dotted line in figure 2). While these models are based purely on reaction kinetics, the third element to model is more complex: the growth of, and interaction between all the types of involved cells in the gut (see orange dotted line in figure 2). Thus including the probiotic <I>S. boulardii</I> and the cancer cells but also the key microbiome species and the healthy gut cells. All these types of cells interact by competing for metabolites and through the anti-cancer protein. The community interaction model thus includes metabolic models of all cells and the two first models. To read all about the construction of these models, please consult the <A href="https://2018.igem.org/Team:Chalmers-Gothenburg/ModelConstruction">model design page.</A></P><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 2:</B> Model overview; different models used in the simulation of the cancer treatment. Blue: α pheromone kinetic model, green: cell-cycle kinetic model, purple: community interaction model.</I></FIGCAPTION></FIGURE><P>To give an overview of the modeling done in this project, the core consists a community interaction model supported by two kinetic models. The community interaction model is a dynamic Flux Balance Analysis (dFBA) framework, consisting of three blocks: an FBA block with genome scale metabolic models (GEMs), a dynamic block consisting of a system of differential equations based on exchange metabolite mass balances, and a kinetic block with kinetic uptake expressions for the exchange reactions of the metabolic models. The dFBA iteratively computes and adjusts the boundaries of each exchange metabolite of every metabolic model, based on Michaelis-Menten kinetics. The kinetic models additionally provide kinetic input to the dFBA for the production of pheromones and anti-cancer proteins and the efficacy of the anti-cancer protein. Both the kinetic and interaction models are formulated using values and parameters from literature. The modeling output enables us to improve our experimental design, which can yield different experimental results that can in turn be used to improve the model. This cyclic flow of information is illustrated as a flowchart in figure 3.  </P><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 3:</B> Flow chart representing the exchange of information between and within the models and how they are integrated with literature and experiments</I></FIGCAPTION></FIGURE><H1 style="text-align: center;">Kinetic Model - α Pheromone Feedback Loop</H1><P>In order to simulate the α pheromone system and the production of anti-cancer agent, a model by Kofahl &amp; Klipp (2004) was implemented. The model describes the α pheromone pathway from receptor activation to the production of the transcription factor Ste12. The α pheromone feedback loop described by the model is illustrated in figure 4.</P><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 4:</B> Schematic of the engineered α pheromone sensing system</I></FIGCAPTION></FIGURE><P>In this project, Bar1 is removed from the genome of the yeast. Because of this, the concentrations and rate of changes of Bar1 and all of its deriviatives included in the model by Kofahl &amp; Klipp (2004) was set to zero. To get the anti-cancer agent production as a function of the α pheromone concentration in the environment, additional reactions where added to the model. The additional reactions include production of anti-cancer agent mRNA and protein as well as production of α pheromone. For more details about the construction of this model, please consult the <A href="https://2018.igem.org/Team:Chalmers-Gothenburg/ModelConstruction">model design page.</A></P><P>To investigate how changes in the α pheromone concentration affect the anti-cancer agent production, the model was run for initial concentrations of α pheromone between 0 and 1000 nM. Figure 5 shows the maximum concentrations of p28 and myrosinase obtained as a function of initial α pheromone concentration. </P><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 5:</B> Maximum concentrations of p28 and myrosinase as functions of initial α pheromone concentration</I></FIGCAPTION></FIGURE><P>Figure 5 suggests that the anti-cancer agent production peaks at an initial concentration of approximately 20 nM of α pheromone. This would mean that the highest concentration of anti-cancer agent is obtained when the concentration of α pheromone in the environment is around this value.  A value of 20 nM can seem low considering that we do not want production of anti-cancer agent unless several yeast cells have accumulated. However, it should be taken into account that as the yeast cells produce α pheromone, it will disperse in the colon making the effective α pheromone concentration lower than the one produced.</P><H1 style="text-align: center;">Kinetic Model - Cell Cycle</H1><P>In order to determine and compare the efficiency of p28 and myrosinase on cancer cell survival, a kinetic model of the cell cycle was implemented. The cell cycle model is based on the work of Hamada et al. (2009) and consists of three parts; p53 dynamics, cell cycle arrest and apoptosis. The peptide p53 is a main regulator of the induction of cell cycle arrest and apoptosis. The p53 dynamics was implemented in a separate ODE system that is described in the <A href="https://2018.igem.org/Team:Chalmers-Gothenburg/ModelConstruction">model design page</A>. The cell cycle arrest and apoptosis parts of the model was implemented according to Hamada et al. (2009) based on the models created by Bagci, Vodovotz, Billiar, Ermentrout, &amp; Bahar (2000) and Aguda (1999). Based on the cell cycle model, two ODE models were built, one for each anti-cancer agent. The first model includes p28 and its effect on the p53 levels in the cell. The ubiquitin-protein ligase COP1 is added to the model to illustrate how p28 inhibits ubiquination of p53 via COP1. The second model includes the enzymatic activity of myrosinase and the effect of sulphoraphane on cell survival. Upregulation of Bax, Cytochrome C and p21, elements involved in apoptosis, via sulphoraphane was added to this model to show an effect of sulphoraphane. A simplified illustration of the system is shown in figure 6. The construction of these two models is described in detail on the <A href="https://2018.igem.org/Team:Chalmers-Gothenburg/ModelConstruction">model design page</A>.</P><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 6:</B> Simplified schematic of the effect of p28 and sulphoraphane on the cancer cell. The small peptide p28 inhibits ubiquination of p53 via the protein-ligase COP1 while sulphoraphane upregulates elements involved in apoptosis</I></FIGCAPTION></FIGURE><P>To test the effect of p28 and myrosinase on cell survival, the cell cycle models were run for anti-cancer agent concentrations between 0 and 200 μM.  For simplification, the concentration of sulforaphane was set directly and the conversion of glucosinolate by myrosinase was left out from the simulations. For each concentration of anti-cancer agent, cell survival was calculated based on 1000 realizations evaluated for 86400 time steps, corresponding to 24 hours. The results are shown in the figure below.</P><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 7:</B> Cell survival as a function of anti-cancer agent concentration obtained with simulation during a time period corresponding to 24 hours</I></FIGCAPTION></FIGURE><P>Observing figure 7, it can be seen that the effect of sulforaphane seems to be stronger than that of p28. It should be noted, however, that the effect of myrosinase might be less strong than that of sulforaphane since it will be limited by the concentration of glucosinolate in the colon. Moreover, it should also be considered that the results from the α pheromone model suggest that the cells will produce more p28 than myrosinase at a given cell concentration.</P><H1 style="text-align: center;">Community Dynamic Flux Balance Analysis</H1><P>The first step in designing a community dFBA framework is to determine which cell types are to be included. First and foremost, our community dFBA framework must contain the GEMs for colorectal cancer cells, human gut cells, and <I>S. boulardii</I>, since they are key aspects of our project. The question of how to model the complex system that is the gut microbiome is a more complicated problem. We decided to use GEMs of three representative species, in terms of composition and biosynthetic capabilities, which were taken from a gut microbiome study by Shoaie et al. (2013). In this paper, the authors generated GEMs for three key gut microbiome member species: <I>Bacteroidetes thetaiotaomicron</I>, <I>Eubacterium rectale</I> and <I>Methanobrevibacter smithii</I>, which represent the main phyla Bacteroidetes, Firmicutes, and Euryarchaeota, respectively. Below is a schematic depicting the system modeled by our community dynamic Flux Balance Analysis framework. Note that the schematic below corresponds to a scenario of a healthy person without colorectal cancer.</P><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 8:</B> Exchange of metabolites between the key microbiotic species in the colon and the engineered probiotic yeast S. boulardii</I></FIGCAPTION></FIGURE><P>However, we are also interested in modeling the specific metabolic interactions between colorectal cancer and our engineered yeast. Below is a schematic showing this interaction.</P><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 9:</B> Interaction between the engineered probiotic yeast S. boulardii and the cancer cells</I></FIGCAPTION></FIGURE><P>Once we established the system to be modeled, we move onto the implementation of our dFBA scheme. We developed COM-dFBA, short for <INS>COM</INS>munity <INS>d</INS>ynamic <INS>F</INS>lux <INS>B</INS>alance <INS>A</INS>nalysis, a simple framework for carrying out dFBA simulations for complex communities, such as the one shown above. COM-dFBA makes use of the RAVEN toolbox functionalities, and consists of six original functions and one script, which can be found in our GitHub repository under the folders COM-dFBA/Scripts. To understand the workings of COM-dFBA it is important to know that the chore of this simulation is to perform an iterative cycle consisting of three parts: the kinetic block, the FBA block, and the dynamic block. This framework is summarized in the figure below. Based on user-defined initial metabolite concentrations, the kinetic block first determines the upper uptake rate of each metabolite <I>j</I> by each organism <I>i</I>, based on Michaelis-Menten kinetics. These rates are then used to update the allowed uptake bound of each exchange metabolite <I>j</I> by each organism <I>i</I>. Next, an FBA simulation is performed for each model in the FBA block. In the dynamic block, the fluxes for each exchange metabolite <I>j</I> is identified in the FBA solution of each organism <I>i</I>, and mass balances are generated for each exchange metabolite. An ODE solver then integrates these mass balances to determine the concentrations of each exchange metabolite <I>j</I> at a given time point t. These new metabolite concentrations are then used by the kinetic block to recalculate the allowed uptake bounds of metabolite <I>j</I> by organism <I>i</I>. This continues in a cyclic fashion until a steady state is reached, and thus dynamic simulation is obtained. Note that in the dynamic block, X<SUB>i</SUB> corresponds to the biomass of organism <I>i</I>, and D is the dilution rate, defined as flow rate divided by volume.</P><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 10:</B> Detailed extract of the modeling overview as presented in figure [...]</I></FIGCAPTION></FIGURE><P> To investigate the effect of our engineered organism, we set up two distinct <I>in silico</I> experiments: </P><OL type="“1”"><LI><I>S. boulardii</I> and Cancer</LI><LI> Gut Community Interaction</LI><LI>Gut Microbiome and Colon before <I>S. boulardii</I> Inflow</LI><LI>Gut Microbiome and Colon with <I>S. boulardii</I> Inflow</LI><LI>Gut Microbiome and Colon after interrupting <I>S. boulardii</I> Inflow</LI></OL><P>In the first <I>in silico</I> experiment we can see the interaction dynamics between <I>S. boulardii</I> and colorectal cancer. More specifically, we aim to investigate: if our engineered yeast can compete with the cancer for substrates, if our engineered yeast can produce enough α pheromone to trigger the production of our anti-cancer agent, and if the resulting anti-cancer agent concentration is sufficient to disrupt the growth of the cancer. This simulation was run for 500 hours.

</P><P>In experiment number two, which is actually three consecutive simulations, we aim to investigate the effect of adding <I>S. boulardii</I> to the gut microbiome. In the first sub-simulation, we determine the steady state composition of the gut microbiome without <I>S. boulardii</I>. In the second sub-simulation, we then introduce an inflow of <I>S. boulardii</I> to the steady state obtained in the first sub-simulation. Finally, in the third sub-simulation, we stop the inflow of <I>S. boulardii</I>. The rationale behind this is to see the extent to which <I>S. boulardii</I> disrupts the gut microbiome composition, to see if <I>S. boulardii</I> will become out competed by the gut microbiome once the dosage/inflow is halted, and to see if the gut microbiome can return to its pre-<I>S. boulardii</I> composition once the dosage/inflow is halted. </P><H2>Results Simulation 1: <I>S. boulardii</I>&amp; Cancer Cells</H2><P> In this simulation we aim to investigate the interaction between a growing colorectal cancer and our engineered yeast. As can be seen on the top subplot, both the colon cancer and <I>S. boulardii </I> biomass are near zero at the starting time point. The cancer begins to grow in its characteristic exponential fashion, while the yeast accumulates and begins producing α pheromone. Just before the 50 hour mark, enough α pheromone accumulates to trigger the production of the anti cancer protein Myrosinase. This causes the growth rate of the cancer to slow down, resulting in a global maximum of cancer biomass between 200 and 250 hours. After this phase there is enough myrosinase in the environment to “overtake” the growth rate of cancer, and we see cancer biomass begin to drop until it reaches zero just after 400 hours. <I>S. boulardii</I> approaches a steady state of around 12 g/L at the end of the simulation. The observed behavior of the α pheromone concentration, specifically after the peak around hour 50, is due to the fact that less α pheromone can be synthesized once myrosinase production begins. This seems reasonable considering there are amino acid and protein pool constraints. The production level of myrosinase relative to α pheromone concentration is validated by the myrosinase kinetic model. The system of ODEs defined by the exchange metabolite mass balances was solved for 1342 time steps, and the simulation took approximately 4.92 hours to complete. This simulation was run using MATLAB 2017b on the Hebbe computer cluster, which is part of the Chalmers Center for Computational Science and Engineering. The Hebbe cluster is built on Intel 2650v3 CPU's, and the system consists of a total 315 compute nodes (total of 6300 cores) with 26 TiB of RAM and 6 GPUs.</P><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 11:</B> Results Simulation 1: <I>S. boulardii</I>&amp; Cancer Cells</I></FIGCAPTION></FIGURE><H2>Results Simulation 2: Gut Community Interaction</H2><P>As discussed previously, this simulation is actually composed of three successive simulations steps with the final concentrations of each step determining the initial concentrations of the successive one. The first simulation step serves to initialize the community model. The initial biomass and metabolite concentrations found in literature, which are summarized in the <A href="https://2018.igem.org/Team:Chalmers-Gothenburg/ModelConstruction">model design page</A>, are used to find the a steady-state for the system. In the next step, <I>S. boulardii</I> is added to the inflow at a concentration of 1.00 g/L, which corresponds to a daily intake of in total 1.2 g of yeast. This is done to observe how our engineered yeast will establish itself, and how this will influence the rest of the gut microbiome. Finally, in the third  step, the inflow of <I>S. boulardii</I> is interrupted, i.e. set to 0 g/L, to simulate the end of the treatment, in order to check whether <I>S. boulardii</I> will remain in the gut or if it will be washed-out.</P><P>The simulation result can be seen in the figure below, where the biomass concentrations are plotted over time. Vertical lines were plotted to delimit each of the three simulation step.</P><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 12:</B> Results Simulation 2: Community dFBA of Colon Cells and gut microbiome</I></FIGCAPTION></FIGURE><TABLE class="table table-striped table-bordered table-responsive-sm"><CAPTION><I><B>Table 1: </B>Final steady-state biomass concentrations</I></CAPTION><THEAD class="thead-light"><TR><TH scope="col">Sim. Step 1</TH><TH scope="col">Sim. Step 2</TH><TH scope="col">Sim. Step 3</TH></TR></THEAD><TBODY><TR><TH scope="row"><I>S. boulardii</I></TH><TD>0.00</TD><TD>1.05</TD><TD>0.01</TD></TR><TR><TH scope="row">Bacteroidetes</TH><TD>25.94</TD><TD>21.64</TD><TD>26.31</TD></TR><TR><TH scope="row">Firmicute</TH><TD>33.95</TD><TD>34.98</TD><TD>33.69</TD></TR><TR><TH scope="row">Euryarcheota</TH><TD>5.23</TD><TD>4.04</TD><TD>5.32</TD></TR></TBODY></TABLE><P>First, it is noteworthy that the final <I>S. boulardii</I> concentration is 1.05 g/L while the inflow is 1.00 g/L. While this is is not an enormous difference, it does mean that the <I>S. boulardii</I> is not only reaching the concentration in the inflow but is also able to grow.</P><P>As expected, the introduction of <I>S. boulardii</I> induced a change in the gut microbiome composition. We quantified this change by dividing the final concentration of each organism at the end of a simulation step by the steady state concentration of the corresponding organism obtained at the end of the first simulation step. These results are presented in Table 2 below.</P><TABLE class="table table-striped table-bordered table-responsive-sm"><CAPTION><I><B>Table 2: </B>Factor biomass concentrations change</I></CAPTION><THEAD class="thead-light"><TR><TH scope="col">Sim. Step 2</TH><TH scope="col">Sim. Step 3</TH></TR></THEAD><TBODY><TR><TH scope="row">Bacteroidetes</TH><TD>0.83</TD><TD>1.01</TD></TR><TR><TH scope="row">Firmicute</TH><TD>1.03</TD><TD>0.99</TD></TR><TR><TH scope="row">Euryarcheota</TH><TD>0.77</TD><TD>1.02</TD></TR></TBODY></TABLE><P>The change in Firmicute biomass concentration after the introduction and interruption of <I>S. boulardii</I> is negligible. However, the Bacteroidetes and Euryarcheota growth is significantly influenced by the introduction of <I>S. boulardii</I>. However, it is important to note that we know from literature that the exact composition of the gut microbiome is naturally variable (Karlsson et al., 2012). Especially variable is the proportion of Euryarcheota. Karlsson et al. (2012) measured it to be 2.0%±4.3, which means that our results are reasonable within these boundaries. Because of this, we can conclude that the introduction of <I>S. boulardii</I> as simulated does not disrupt the healthy gut microbiome in a significant way.</P><P>Additionally, we can conclude that, because the <I>S. boulardii</I> concentration returns to values very close to 0 g/L in around 50 hours after stopping the inflow of <I>S. boulardii</I>, it is unable to survive on its own, without continuous dietary intake. Also, the other gut microbiome species return to concentrations very close to those before the treatment. This is an encouraging outcome, because patients would want the engineered yeast to leave their gastro-intestinal tract shortly after the treatment, without having their gut microbiome altered in a detrimental or permanent way.</P><H1 style="text-align: center;">Integrated Modeling</H1><P> The purpose of the mathematical models created in this project is to make an impact on the course of the project and to be able to make inferences about the biological systems that we introduce into yeast. Although the models all have areas for improvement, the modeling made an impact on the overall project and the work in the wet lab. In this section we explain in what ways it did so. </P><H2>Proof of Concept</H2><P>First and foremost, the kinetic models and GEMs implemented in this project could be used as proof of concept and for illustrations of how our product will work in practice.</P><P>When it comes to the kinetic models, the kinetic model of the α pheromone system illustrates how the feedback loop of α pheromone works.  It shows how the concentration of α pheromone in the environment induces the MAPK cascade that in turn leads to the production of anti-cancer agent. The cell cycle models with p28 and myrosinase tell us how the anti-cancer agents potentially affect the cell cycle. They also illustrate how p28 and sulforaphane can lead to the induction of apoptosis and thereby effectively kill cancer cells.</P><P>For the GEMs, the results from the simulations of the gut microbiota with and without <I>S. boulardii</I> indicate that the yeast has no dramatic effects on the composition of the gut microbiota. It demonstrates how <I>S. boulardii</I> can survive alongside gut microbial species while also not harming the patient. Moreover, simulations with <I>S. boulardii</I> , suggest that <I>S. boulardii</I> can produce myrosinase without depleting its amino acid resources. This means that the yeast can kill cancer cells while also growing.</P><H2>α Pheromone Threshold</H2><P>The α pheromone model gave additional insight into our anti-cancer agent production system. The results from the simulations indicate that the initiating threshold of α pheromone for the production of anti-cancer agent is lower than expected. Based on these findings, we should look into the system more closely and consider to use a weaker version of the FUS1 promoter. This is of importance since we do not want our <I>S. boulardii</I> to produce anti-cancer agent unless yeast cells has accumulated as a consequence of the presence of cancer cells.</P><H2>Choice of Anti-cancer Agent</H2><P>Based on the p28 and myrosinase models, we decided to move on with only myrosinase as the potential anti-cancer agent. The reason for this was that the models indicated that the efficiency of sulforaphane would be better than that of p28.  The α pheromone model showed that cells would produce more p28 than myrosinase, which would perhaps favor choosing p28. However, myrosinase has the additional benefit that it can keep producing sulforaphane as long as there is any enzyme and substrate (glucosinolate) present, whereas the anti-cancer potential of p28 is limited by its own concentration. Had the results from the cell cycle models been added to the model of <I>S. boulardii</I> and cancer cells, it is expected that even further conclusions could have been drawn regarding anti-cancer agent efficiency.</P><H2><I>S. boulardii</I> Content in Pill</H2><P>The simulation of <S. boulardii=""> together with the gut microbiota gave an indication of how much yeast to put inside of the final product, i.e. the pill. In the simulations, the inflow of <I>S. boulardii</I> was set to 1 g/L with a total liquid inflow of 0.05 L/h. This gives a daily intake of 1.2 g of yeast. Therefore, a patient should ingest 1.2 g of yeast every day during treatment. The patient could either take one pill per day or the content could be split into several pills to get an even flow of yeast throughout the day. Since part of the yeast cells will die both in the manufacturing process and on the journey from the patient’s mouth to the gut, this should be considered a lower bound.</S.></P><H1 style="text-align: center;">Future Work</H1><P>The models created in this project gave us valuable insights into project design and helped us improve our visualised product. While this is true, the modeling could still be subject to improvement. Below, we bring up model improvements that did not fit the time scope of the project.</P><P>For all of the kinetic models, optimization was used to find parameter values. However, in the process of optimization, an identifiability analysis should optimally be performed to determine the validity of the outcome. If the work on the α pheromone model and the cell cycle models would continue, this would be the next step. Based on the identifiability analysis it can be decided if more data is needed in order to fit the models to reality and if there are parameter values that cannot be found with optimization. </P><P>In the implementation of the cell cycle model, parameter values and initial concentrations were obtained from Hamada et al. (2009). However, these values are based on the dynamics of a normal, healthy cell. For future work, the parameters in the model could be modified in order for the dynamics to better resemble those of a cancer cell. To develop the cell cycle models further, the models could be made more time responsive in the sense that cell survival is not only a matter of anti-cancer agent concentration but also something that is affected by exposure time. If the models had been fit to data available at several time points, they could have been rendered more realistic based on exposure time. </P><P>Regarding our dFBA framework implementation, it would be interesting to compare the accuracy and speed of COM-dFBA to other high quality dFBA tools, such as DFBAlab. To do this, we would simply run the same simulations using DFBAlab instead of COM-dFBA, and compare the results. Another aspect for future improvement would be to add a flux variability analysis (FVA) step to the COM-dFBA framework. This would help ensure that the FBA solutions obtained are unique, and that the models are sufficiently constrained. Additionally, improvements towards computational efficiency and speed would need to be made in order to feasibly run simulations with the all the GEMs implemented in this project, including the gut cell, the cancer cell, the gut microbiota and <I>S. boulardii</I>, in order to get an abstraction of the whole gut system. In addition to this, glucosinolate should be added to the gut inflow to get a better understanding of how the diet can affect the efficiency of myrosinase. Finally, it would be interesting to run a parameter sensitivity analysis, in order to determine which specific parameters have a significant influence on the interaction dynamics of interest. This would also provide us with potential engineering targets in the future.</P><P>To further analyze the required dosage of <I>S. boulardii</I>, we should model the yeast to cancer cell binding in the colon. A simulation with such a model would help us link the dosage required to get a stable growth of <I>S. Boulardii</I> to the dosage required to effectively kill the cancer cells.</P><P>The optimal last step of this modeling project would be more closely integrate the results of the α pheromone and cell cycle models to the system of dFBA framework. In this way,  we would obtain a large-scale model that can be used to evaluate the effects of our yeast on cancer cell survival and to further improve our product.</P><H1 style="text-align: center;">References</H1></DIV></DIV><DIV class="row" style="font-size:70%;"><DIV class="col-6"><P>Aguda, B. D. (1999). A quantitative analysis of the kinetics of the G2 DNA damage checkpoint system. <EM>Proceedings of the National Academy of Sciences</EM>, <EM>96</EM>(20), 11352–11357. https://doi.org/10.1073/pnas.96.20.11352</P><P>Azzouz, L. L., &amp; Sharma, S. (2018). <EM>Physiology, Large Intestine</EM>. <EM>StatPearls</EM>. StatPearls Publishing. Retrieved from http://www.ncbi.nlm.nih.gov/pubmed/29939634</P><P>Bagci, E. Z., Vodovotz, Y., Billiar, T. R., Ermentrout, G. B., &amp; Bahar, I. (2006). Bistability in Apoptosis: Roles of Bax, Bcl-2, and Mitochondrial Permeability Transition Pores. <EM>Biophysical Journal</EM>, <EM>90</EM>(5), 1546–1559. https://doi.org/10.1529/biophysj.105.068122</P><P>Bianconi, E., Piovesan, A., Facchin, F., Beraudi, A., Casadei, R., Frabetti, F., … Canaider, S. (2013). An estimation of the number of cells in the human body. <EM>Annals of Human Biology</EM>, <EM>40</EM>(6), 463–471. https://doi.org/10.3109/03014460.2013.807878</P><P>Bianconi, E., Piovesan, A., Facchin, F., Beraudi, A., Casadei, R., Frabetti, F., … Canaider, S. (2013). An estimation of the number of cells in the human body. <EM>Annals of Human Biology</EM>, <EM>40</EM>(6), 463–471. https://doi.org/10.3109/03014460.2013.807878</P><P>Chaddock, G., Lam, C., Hoad, C. L., Costigan, C., Cox, E. F., Placidi, E., … Spiller, R. C. (2014). Novel MRI tests of orocecal transit time and whole gut transit time: studies in normal subjects. <EM>Neurogastroenterology and Motility : The Official Journal of the European Gastrointestinal Motility Society</EM>, <EM>26</EM>(2), 205–214. https://doi.org/10.1111/nmo.12249</P><P>Crane, R. K. (1975). The Physiology of the Intestinal Absorption of Sugars This review has to do with the physiology of the intestinal absorption of sugars and should properly begin with a brief discussion of the components of the physiological system which carries out this i, 2–19. https://doi.org/10.1021/bk-1975-0015.ch001</P><P>EFSA NDA Panel. (2015). Scientific Opinion on Dietary Reference Values for phosphorus. <EM>EFSA Journal</EM>, <EM>13</EM>(7), 4185. https://doi.org/10.2903/j.efsa.2015.4185</P><P>Frydoonfar, H. R., McGrath, D. R., &amp; Spigelman, A. D. (2004). Sulforaphane inhibits growth of a colon cancer cell line. <EM>Colorectal Disease</EM>, <EM>6</EM>(1), 28–31. https://doi.org/10.1111/j.1463-1318.2004.00488.x</P><P>Hamada, H., Tashima, Y., Kisaka, Y., Iwamoto, K., Hanai, T., Eguchi, Y., &amp; Okamoto, M. (2009). Sophisticated Framework between Cell Cycle Arrest and Apoptosis Induction Based on p53 Dynamics. <EM>PLoS ONE</EM>, <EM>4</EM>(3), e4795. https://doi.org/10.1371/journal.pone.0004795</P><P>Hochkoeppler, A., &amp; Palmieri, S. (1992). Kinetic properties of myrosinase in hydrated reverse micelles. <EM>Biotechnology Progress</EM>, <EM>8</EM>(2), 91–96. https://doi.org/10.1021/bp00014a001</P><P>Hughes, R., Magee, E. A., &amp; Bingham, S. (2000). Protein degradation in the large intestine: relevance to colorectal cancer. <EM>Current Issues in Intestinal Microbiology</EM>, <EM>1</EM>(2), 51–58. Retrieved from http://www.ncbi.nlm.nih.gov/pubmed/11709869</P><P>Karlsson, F. H., Fåk, F., Nookaew, I., Tremaroli, V., Fagerberg, B., Petranovic, D., … Nielsen, J. (2012). Symptomatic atherosclerosis is associated with an altered gut metagenome. <EM>Nature Communications</EM>, <EM>3</EM>(1), 1245. https://doi.org/10.1038/ncomms2266</P><P>Kofahl, B., &amp; Klipp, E. (2004). Modelling the dynamics of the yeast pheromone pathway. <EM>Yeast</EM>, <EM>21</EM>(10), 831–850. https://doi.org/10.1002/yea.1122</P><P>Kretsinger, R. H., Uversky, V. N., &amp; Permi︠a︡kov, E. A. (Evgeniĭ A. (2013). <EM>Encyclopedia of metalloproteins</EM>.</P><P>Ledley, R. S. (1960). <EM>Report on the Use of Computers in Biology and Medicine - Robert Steven Ledley - Google Boeken</EM>. Retrieved from https://books.google.se/books?id=J5grAAAAYAAJ&amp;printsec=frontcover&amp;hl=nl&amp;source=gbs_ge_summary_r&amp;cad=0#v=onepage&amp;q&amp;f=false</P><P>Lev Bar-Or, R., Maya, R., Segel, L. A., Alon, U., Levine, A. J., &amp; Oren, M. (2000). Generation of oscillations by the p53-Mdm2 feedback loop: A theoretical and experimental study. <EM>Proceedings of the National Academy of Sciences</EM>, <EM>97</EM>(21), 11250–11255. https://doi.org/10.1073/pnas.210171597</P></DIV><DIV class="col-6"><P>Lewis, J., &amp; Fenwick, G. R. (1987). Glucosinolate content of brassica vegetables: Analysis of twenty-four cultivars of calabrese (green sprouting broccoli, Brassica oleracea L. var. botrytis subvar. cymosa Lam.). <EM>Food Chemistry</EM>, <EM>25</EM>(4), 259–268. https://doi.org/10.1016/0308-8146(87)90012-4</P><P>Moscetti, I., Bizzarri, A. R., &amp; Cannistraro, S. (2018). Imaging and kinetics of the bimolecular complex formed by the tumor suppressor p53 with ubiquitin ligase COP1 as studied by atomic force microscopy and surface plasmon resonance. <EM>International Journal of Nanomedicine</EM>, <EM>Volume 13</EM>, 251–259. https://doi.org/10.2147/IJN.S152214</P><P>Moscetti, I., Cannistraro, S., &amp; Bizzarri, A. R. (2017, November 20). Surface plasmon resonance sensing of biorecognition interactions within the tumor suppressor P53 network. <EM>Sensors (Switzerland)</EM>. https://doi.org/10.3390/s17112680</P><P>Pertoft, H., &amp; Laurent, T. C. (1977). Isopycnic Separation of Cells and Cell Organelles by Centrifugation in Modified Colloidal Silica Gradients. In <EM>Methods of Cell Separation</EM> (pp. 25–65). Boston, MA: Springer US. https://doi.org/10.1007/978-1-4684-0820-1_2</P><P>Pubchem. (n.d.). Compound Summary for CID 23682211 - Sinigrin. Retrieved October 7, 2018, from https://pubchem.ncbi.nlm.nih.gov/compound/Sinigrin#section=Top</P><P>Shoaie, S., Karlsson, F., Mardinoglu, A., Nookaew, I., Bordel, S., &amp; Nielsen, J. (2013). Understanding the interactions between bacteria in the human gut through metabolic modeling. <EM>Scientific Reports</EM>, <EM>3</EM>(1), 2532. https://doi.org/10.1038/srep02532</P><P>Stephen, A. M., &amp; Cummings, J. H. (1980). THE MICROBIAL CONTRIBUTION TO HUMAN FAECAL MASS. <EM>Journal of Medical Microbiology</EM>, <EM>13</EM>(1), 45–56. https://doi.org/10.1099/00222615-13-1-45</P><P>Stephen, A. M., &amp; Cummings, J. H. (1980). THE MICROBIAL CONTRIBUTION TO HUMAN FAECAL MASS. <EM>Journal of Medical Microbiology</EM>, <EM>13</EM>(1), 45–56. https://doi.org/10.1099/00222615-13-1-45</P><P>Tortorella, S. M., Royce, S. G., Licciardi, P. V., &amp; Karagiannis, T. C. (2015). Dietary Sulforaphane in Cancer Chemoprevention: The Role of Epigenetic Regulation and HDAC Inhibition. <EM>Antioxidants &amp; Redox Signaling</EM>, <EM>22</EM>(16), 1382–1424. https://doi.org/10.1089/ars.2014.6097</P><P>Wang, L., He, G., Zhang, P., Wang, X., Jiang, M., &amp; Yu, L. (2011). Interplay between MDM2, MDMX, Pirh2 and COP1: the negative regulators of p53. <EM>Molecular Biology Reports</EM>, <EM>38</EM>(1), 229–236. https://doi.org/10.1007/s11033-010-0099-x</P><P>Warso, M. A., Richards, J. M., Mehta, D., Christov, K., Schaeffer, C., Rae Bressler, L., … Das Gupta, T. K. (2013). A first-in-class, first-in-human, phase I trial of p28, a non-HDM2-mediated peptide inhibitor of p53 ubiquitination in patients with advanced solid tumours. <EM>British Journal of Cancer</EM>, <EM>108</EM>(5), 1061–1070. https://doi.org/10.1038/bjc.2013.74</P><P>WHO. (2003). <EM>Ammonia in Drinking-water - Background document for development of WHO Guidelines for Drinking-water Quality</EM>. <EM>Guidelines for drinking-water quality</EM> (Vol. 2). World Health Organization. Retrieved from http://www.who.int/water_sanitation_health/dwq/ammonia.pdf</P><P>Wikibooks. (n.d.). Medical Physiology/Gastrointestinal Physiology/Secretions - Wikibooks, open books for an open world. Retrieved October 7, 2018, from https://en.wikibooks.org/wiki/Medical_Physiology/Gastrointestinal_Physiology/Secretions</P><P>Wiśniewski, J. R., Ostasiewicz, P., Duś, K., Zielińska, D. F., Gnad, F., &amp; Mann, M. (2012). Extensive quantitative remodeling of the proteome between normal colon tissue and adenocarcinoma. <EM>Molecular Systems Biology</EM>, <EM>8</EM>, 611. https://doi.org/10.1038/msb.2012.44</P><P>Yamada, T., Mehta, R. R., Lekmine, F., Christov, K., King, M. L., Majumdar, D., … Das Gupta, T. K. (2009). A peptide fragment of azurin induces a p53-mediated cell cycle arrest in human breast cancer cells. <EM>Molecular Cancer Therapeutics</EM>, <EM>8</EM>(10), 2947–2958. https://doi.org/10.1158/1535-7163.MCT-09-0444</P></DIV></DIV></DIV></DIV></DIV><DIV id="footer"><FOOTER><H4 style="font-size:12pt;">
    CONTACT INFORMATION: 
  </H4><DIV><P style="font-size:10pt;">
      CHALMERS University of Technology, Gothenburg, Sweden, igemgothenburg@gmail.com   
</P><SPAN><A style="font-size:10pt;" href="https://www.facebook.com/chalmersigem">Facebook</A><A style="font-size:10pt;" href="https://www.instagram.com/chalmersigem">Instagram</A><A style="font-size:10pt;" href="https://twitter.com/igemchalmers">Twitter</A></SPAN></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>