<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Chalmers-Gothenburg_Detection skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Chalmers-Gothenburg/Detection</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Interlab - iGEM Chalmers-Gothenburg 2018</TITLE></P><NAV id="navbar" class="mynav navbar-expand-md bg-dark navbar-dark sticky-top"><DIV class="collapse navbar-collapse justify-content-around" id="collapsibleNavbar"><UL class="navbar-nav align-items-center"><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" id="navbardrop-project" role="button" data-toggle="dropdown">
        PROJECT
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Description" style="color:#16181b">Project Overview</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Project" style="color:#16181b">Project Summary</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Achievements" style="color:#16181b">Achievements</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" id="navbardrop-wetlab" data-toggle="dropdown">
        LABWORK
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Attachment" style="color:#16181b">Binding</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Detection" style="color:#16181b">Detection</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Treatment" style="color:#16181b">Treatment</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/InterLab" style="color:#16181b">InterLab</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Parts" style="color:#16181b">BioBricks</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Protocols" style="color:#16181b">Protocols</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Safety" style="color:#16181b">Safety</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" id="navbardrop-hp" data-toggle="dropdown">
        HUMAN PRACTICES
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Human_Practices" style="color:#16181b">Human Practices</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Collaborations" style="color:#16181b">Collaborations</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Outreach" style="color:#16181b">Outreach</A></DIV></LI></UL><UL class="navbar-nav align-items-center"><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" id="navbardrop-model" data-toggle="dropdown">
        MODELING
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Model" style="color:#16181b">Model Overview</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/ModelConstruction" style="color:#16181b">Model Design</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" id="navbardrop-team" data-toggle="dropdown">
        PEOPLE
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Team" style="color:#16181b">Team</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Supervisors" style="color:#16181b">Supervisors</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Sponsors" style="color:#16181b">Sponsors</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Attributions" style="color:#16181b">Attributions</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" id="navbardrop-social" data-toggle="dropdown">
        SOCIAL MEDIA
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://www.facebook.com/chalmersigem" style="color:#16181b">Facebook</A><A class="dropdown-item" href="https://www.instagram.com/chalmersigem" style="color:#16181b">Instagram</A><A class="dropdown-item" href="https://twitter.com/igemchalmers" style="color:#16181b">Twitter</A></DIV></LI></UL></DIV></NAV><DIV class="container-fluid" style="margin-top:30px;"><DIV class="row"><DIV class="col-lg-2"><DIV class="d-none d-lg-block" style="position:sticky; top:130px;"><H4>Navigation</H4><NAV id="mySidebar"><UL class="nav nav-pills flex-column"><LI class="nav-item"><A class="nav-link" href="#Labwork">Labwork and Results</A></LI><LI class="nav-item"><A class="nav-link" href="#Attachments">Attachments</A></LI><LI class="nav-item"><A class="nav-link" href="#Protocols">Protocols</A></LI></UL></NAV></DIV></DIV><DIV class="col-lg-8" style="text-align:justify"><DIV class="row"><DIV class="col-12"><H1 style="text-align:center">Labwork and Results</H1><P>Bourdeau et al. (2018) have shown that organisms can be engineered to produce gas vesicles as acoustic reporter genes. They have combined gas vesicle genes from <I>Bacillus megaterium</I> and <I>Anabaena flos-aquae</I> to produce the acoustic reporter gene in <I>Escherichia coli</I>. To further improve their acoustic reporter gene they have modified the gene GvpC by deleting several copies of a repetitive region. Their final Acoustic reporter gene that serves as a starting point for this part of our iGEM project is represented in figure 1.</P><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 1:</B> Acoustic Reporter Gene 2 (ARG2) as designed by Bourdeau et al. (2018)</I></FIGCAPTION></FIGURE><H2>Design of Eukaryotic Acoustic Reporter Gene</H2><P>In order to express this prokaryotic operon in the eukaryotic yeast, which is not able to transcribe gene operons, a multicistronic gene expression system is used. Namely, 2A viral peptides. This sequence, which we place in between each gene of the operon, codes for a peptide sequence that cleaves itself and separates the genes (Souza-Moreira et al. 2018). Without this system it would also be possible to express each gene separately, however, each gene should then be introduced under its own promoter and terminator while here, the gene set can be combined under one promoter and one terminator.</P><P>Because the set of genes has to be expressed at once, all stop codons before 2a peptides have to be removed</P><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 2:</B> 2a peptide cleaving efficiency (top) and sequences of the three most efficiently cleaving 2a peptides (bottom) from Souza-Moreira et al. (2018)</I></FIGCAPTION></FIGURE><P>From figure 2. we conclude that ERBV-1 is the most efficiently cleaving 2a peptide. Thus, it is this sequence that is introduced in between the genes. Our synthesis company limits the size of DNA constructs to around 2000 nucleotides. It is thus unfortunately not possible to make one continuous sequence containing all genes. Instead, three gene sets of around 2000bp are designed. To distribute the genes more or less evenly over these 3 different constructs, we assumed that the gene order is not important, this allowed us to distribute them as presented in figure 3</P><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 3:</B> Constructs designed for acoustic reporter gene expression in yeast</I></FIGCAPTION></FIGURE><P>These three constructs are inserted in 3 different plasmid vectors, all with an ampicillin resistance gene for selection in <I>E. coli</I> and with a different yeast amino acid auxotrophic marker for each plasmid:</P><UL><LI>p413, with a Histidine (His) marker</LI><LI>p414, with a Tryptophan (Trp) marker</LI><LI>p416, with a Uracil (Ura) marker</LI></UL><P>The plasmids will be transformed into a triple auxotrophic <I>S. cerevisiae</I> strain and only the cells with all three plasmids inserted, thus with all three constructs, are selected.</P><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 4:</B> Plasmid vectors used to insert the acoustic reporter gene into yeast </I></FIGCAPTION></FIGURE><P>Unfortunately, our supplier was not able to synthesize the DNA as we designed the constructs (figure 2). Only Gas Vesicle construct 2 was produced and delivered as ordered. As advised by them, we redesigned the constructs by re-codon-optimizing the coding sequences, limiting the amount of repetitive sequences and splitting up the sequences in shorter fragments. After a few cycles of ordering, failure to produce and redesigning, the final constructs as they could be produced and were delivered to us are presented in figure 5. The main changes are:</P><UL><LI>In order to have less repeats in the sequences, we decided to use a combination of two different 2a peptides in construct 1, the middle ERBV-1 2a peptide is replaced by a OpbuCPV18. The PTV 2a peptide has slightly higher cleavage efficiency but its sequence is almost identical to ERBV-1 (see figure 2.).</LI><LI>The third construct caused more problems and had to be split in three fragments. Specifically the gene GvpT was difficult to synthesize due to its highly repetitive sequence.</LI></UL><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 5:</B> Constructs delivered by supplier</I></FIGCAPTION></FIGURE><P>In order to fit these five fragments in the same three plasmid vectors (figure 4), we have opted for a double promoter strategy, as used in the pSP-G1 and pSP-G2 plasmids (Partow, Siewers, Bjørn, Nielsen, &amp; Maury, 2010). In this paper, the authors present two plasmids that have two sets of promoters and terminators, which allow the introduction of two genes. In our project we do not use the whole plasmid, we only amplify the double promoter and terminator on this plasmid (figure 6), in order to insert it in the previously chosen plasmids (figure 4).

<FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 6:</B> Amplification of double promoter and ADH1 terminator from pSP plasmid</I></FIGCAPTION></FIGURE></P><H2>Acoustic Reporter Gene Transformation Strategy</H2><P>To combine all these fragments and insert them into the plasmid backbone, Gibson assembly is used for each plasmid separately. This is done with a general approach as described on the labwork protocol page, and specifically for these constructs as represented in the figure below. Construct 2 is inserted into the p413TEFplasmid. GvpU and the construct containing GvpK and GvpJ are inserted into the p414TEF plasmid with both the double promoter and the terminator amplified from pSP-GM1. Similarly, Construct 1 and GvpT are inserted with the double promoter and terminator into p416TEF</P><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 7:</B> Transformation strategy to insert the acoustic reporter gene into yeast</I></FIGCAPTION></FIGURE><H2>Methods for testing gas vesicle production</H2><P>Because the whole point of expressing gas vesicles in our yeast is to be able to detect them, the straightforward testing method is to image the cells using an ultrasound transducer. We have been granted access to two ultrasound machines from friendly people at the Laboratory for Experimental Biomedicine of the Sahlgrenska Academy, afiliated to Gothenburg University. They have one portable machine: Acuson Cypresse and one machine with higher resolution but bulkier: ATL 5000</P><P>Around the start of our full-time work on the project, we have tested these machines on petri dishes with and without (untransformed) yeast. With some practice, it was possible to visualise the cells on the ultrasound image. This experiment is documented in the lab Journal - Ultrasound Imaging linked in the attachments below.</P><P>Before the yeast is transformed with all three plasmids there are many milestones along the way that have to be verified, such as each PCR amplification or Gibson assembly. These steps are tested by gel electrophoresis of the DNA that is cut with chosen restriction enzymes that give different band patterns in electrophoresis. These are described in the Lab Journal - <I>S. cerevisiae</I> Transformation linked in the attachments below.</P><H2>Results</H2><P>Unfortunately, this project did not yield noteworthy results, all constructs were assembled through gibson assembly and gel electrophoresis bands were faint thus not decisive but seemed to suggest the successful assembly of the plasmids. However, the transformation of E.coli was not successful for any of the three transformations. Because this is an ambitious part of our project and because we are only a small team, we decided to drop it and prioritise other parts of our iGEM project</P><H1 style="text-align:center">Attachments</H1><H3>Lab Journals</H3><UL><LI><A href="https://static.igem.org/mediawiki/2018/9/97/T--Chalmers-Gothenburg--Gas_Vesicles_all_lab_work_Benchling.pdf">Lab Journal - <I>S. cerevisiae</I> Transformation</A></LI><LI><A href="https://static.igem.org/mediawiki/2018/6/65/T--Chalmers-Gothenburg--Gas_Vesicles_Ultrasound_Testing_Benchling.pdf">Lab Journal - Ultrasound Imaging</A></LI></UL><H3>Primer, Construct, and Plasmid Sequences</H3><UL><LI><A href="https://static.igem.org/mediawiki/2018/7/79/T--Chalmers-Gothenburg--Detection_Constructs_and_plasmids.zip">Construct and Plasmid Sequences</A></LI><LI><A href="https://static.igem.org/mediawiki/2018/0/08/T--Chalmers-Gothenburg--PCRPrimersGasVesicles.xlsx">Primer Sequences</A></LI></UL><H1 style="text-align:center">Protocols</H1><P> Find all protocols here, <A href="https://2018.igem.org/Team:Chalmers-Gothenburg/Protocols">Labwork-Protocols</A>.</P><H1 style="text-align:center">References</H1></DIV></DIV><DIV class="row" style="font-size:70%;"><DIV class="col-6"><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Bourdeau, R. W., Lee-Gosselin, A., Lakshmanan, A., Farhadi, A., Kumar, S. R., Nety, S. P., &amp; Shapiro, M. G. (2018). Acoustic reporter genes for noninvasive imaging of microorganisms in mammalian hosts. <EM>Nature</EM>. https://doi.org/10.1038/nature25021</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Chen, X., Zaro, J. L., &amp; Shen, W.-C. (2013). Fusion protein linkers: Property, design and functionality. <EM>Advanced Drug Delivery Reviews</EM>, <EM>65</EM>(10), 1357–1369. https://doi.org/10.1016/j.addr.2012.09.039</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Englert, C., &amp; Pfeiffer, F. (1993). Analysis of gas vesicle gene expression in Haloferax mediterranei reveals that GvpA and GvpC are both gas vesicle structural proteins. <EM>The Journal of Biological Chemistry</EM>, <EM>268</EM>(13), 9329–9336. Retrieved from http://www.jbc.org/content/268/13/9329.short</P></DIV><DIV class="col-6"><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Liu, Z., Tyo, K. E. J., Martínez, J. L., Petranovic, D., &amp; Nielsen, J. (2012). Different expression systems for production of recombinant proteins in Saccharomyces cerevisiae. <EM>Biotechnology and Bioengineering</EM>, <EM>109</EM>(5), 1259–1268. https://doi.org/10.1002/bit.24409</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Partow, S., Siewers, V., Bjørn, S., Nielsen, J., &amp; Maury, J. (2010). Characterization of different promoters for designing a new expression vector in Saccharomyces cerevisiae. <EM>Yeast</EM>, <EM>27</EM>(11), 955–964. https://doi.org/10.1002/yea.1806</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Souza-Moreira, T. M., Navarrete, C., Chen, X., Zanelli, C. F., Valentini, S. R., Furlan, M., … Krivoruchko, A. (2018). Screening of 2A peptides for polycistronic gene expression in yeast. <EM>FEMS Yeast Research</EM>, <EM>18</EM>(5). https://doi.org/10.1093/femsyr/foy036</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="footer"><FOOTER><H4 style="font-size:12pt;">
    CONTACT INFORMATION: 
  </H4><DIV><P style="font-size:10pt;">
      CHALMERS University of Technology, Gothenburg, Sweden, igemgothenburg@gmail.com   
</P><SPAN><A style="font-size:10pt;" href="https://www.facebook.com/chalmersigem">Facebook</A><A style="font-size:10pt;" href="https://www.instagram.com/chalmersigem">Instagram</A><A style="font-size:10pt;" href="https://twitter.com/igemchalmers">Twitter</A></SPAN></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>