<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Chalmers-Gothenburg_Treatment skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Chalmers-Gothenburg/Treatment</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Interlab - iGEM Chalmers-Gothenburg 2018</TITLE></P><NAV id="navbar" class="mynav navbar-expand-md bg-dark navbar-dark sticky-top"><DIV class="collapse navbar-collapse justify-content-around" id="collapsibleNavbar"><UL class="navbar-nav align-items-center"><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" id="navbardrop-project" role="button" data-toggle="dropdown">
        PROJECT
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Description" style="color:#16181b">Project Overview</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Project" style="color:#16181b">Project Summary</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Achievements" style="color:#16181b">Achievements</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" id="navbardrop-wetlab" data-toggle="dropdown">
        LABWORK
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Attachment" style="color:#16181b">Binding</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Detection" style="color:#16181b">Detection</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Treatment" style="color:#16181b">Treatment</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/InterLab" style="color:#16181b">InterLab</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Parts" style="color:#16181b">BioBricks</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Protocols" style="color:#16181b">Protocols</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Safety" style="color:#16181b">Safety</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" id="navbardrop-hp" data-toggle="dropdown">
        HUMAN PRACTICES
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Human_Practices" style="color:#16181b">Human Practices</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Collaborations" style="color:#16181b">Collaborations</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Outreach" style="color:#16181b">Outreach</A></DIV></LI></UL><UL class="navbar-nav align-items-center"><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" id="navbardrop-model" data-toggle="dropdown">
        MODELING
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Model" style="color:#16181b">Model Overview</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/ModelConstruction" style="color:#16181b">Model Design</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" id="navbardrop-team" data-toggle="dropdown">
        PEOPLE
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Team" style="color:#16181b">Team</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Supervisors" style="color:#16181b">Supervisors</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Sponsors" style="color:#16181b">Sponsors</A><A class="dropdown-item" href="https://2018.igem.org/Team:Chalmers-Gothenburg/Attributions" style="color:#16181b">Attributions</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" id="navbardrop-social" data-toggle="dropdown">
        SOCIAL MEDIA
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://www.facebook.com/chalmersigem" style="color:#16181b">Facebook</A><A class="dropdown-item" href="https://www.instagram.com/chalmersigem" style="color:#16181b">Instagram</A><A class="dropdown-item" href="https://twitter.com/igemchalmers" style="color:#16181b">Twitter</A></DIV></LI></UL></DIV></NAV><DIV class="container-fluid" style="margin-top:30px;"><DIV class="row"><DIV class="col-lg-2"><DIV class="d-none d-lg-block" style="position:sticky; top:130px;"><H4>Navigation</H4><NAV id="mySidebar"><UL class="nav nav-pills flex-column"><LI class="nav-item"><A class="nav-link" href="#Labwork">Labwork and Results</A></LI><LI class="nav-item"><A class="nav-link" href="#Attachments">Attachments</A></LI><LI class="nav-item"><A class="nav-link" href="#Protocols">Protocols</A></LI></UL></NAV></DIV></DIV><DIV class="col-lg-8" style="text-align:justify"><DIV class="row"><DIV class="col-12"><H1 style="text-align:center">
              Labwork and Results
            </H1><P>There are a few different parts that constitutes this project, to give you a quick overview:</P><UL><LI>Use CRISPR- Cas9 on the yeast in order to allow it to produce and accumulate alpha pheromone </LI><LI>Use Myrosinase as a cancer toxin
</LI><LI>Use p28 as a cancer toxin</LI></UL><P>The goal of the CRISPR-Cas9 system was to delete BAR1, which is a part of the mating system of yeast and leads to the degradation of alpha pheromone inside of the yeast, and replace it with MF(ALPHA)2, the gene which codes for the alpha  pheromone. This Cas9 construct was transformed into yeast and the resulting modified strain was used for all further yeast transformations related to the cancer toxins. The reason for this is that the cancer toxins were put under a FUS1 promoter which is only responsive to the alpha pheromone.</P><P>Moving into this project both Myrosinase and p28 were planned to be utilized at the same time in the same yeast. However as the project progressed it was decided that p28 should be dropped and that myrosinase would be the sole focus. This was due to poor modeling results for the p28 and a strict time limit to this project.</P><P>This means that the construct containing p28 was never transformed into yeast and thus never tested <I>in vitro</I>. The Myrosinase however was transformed into our edited yeast strain and its efficiency was tested on human colon carcinoma cells (RKO cells) <I>in vitro</I>.</P><P>There were few different plasmid used for these constructs. For integration and deletion of genes a CRISPR-Cas9 plasmid carrying a kanMX marker was used. It has a ampicillin resistance for the <I>E. coli</I> and a g418 resistance for the yeast selection. The remaining constructs were put into the p413TEF plasmid which contains the gene HIS3 as its marker.  The p413TEF plasmid was cut with with the enzymes SacI and XhoI to get rid of the TEF1 promoter and was switched out for the FUS1 promoter. This was done since we wanted the yeast to express the proteins of interest only in the presence of alpha pheromones. </P><H2>kanMX-Cas9 and gRNA</H2><P>The plasmid kanMX-Cas9 was used as the source for the CRISPR system that was utilized in order to edit the genome of a CEN.PK113-11c <I>S. cerevisiae</I> strain. The gRNA sequence (GCTGACGGAACATTTGCTGA) was developed via a tool built into benchling. The sequence was integrated into a kanMX-Cas9 plasmid to be transformed into yeast and selected for on g418 plates, see figure 1 and table 1. 
</P><TABLE class="table table-striped table-bordered table-responsive-sm"><CAPTION><I><B>Table 1: </B>kanMX-Cas9 and gRNA proteins</I></CAPTION><TBODY><TR><TH>Protein</TH><TH>Full name</TH><TH>Native organism</TH><TH>Sequence source</TH></TR><TR><TD>Cas9</TD><TD>CRISPR associated protein 9</TD><TD>Optimized for <I>Saccharomyces cerevisiae</I></TD><TD>Sysbio lab at Chalmers</TD></TR><TR><TD>Mating factor alpha 2</TD><TD>MF(ALPHA)2</TD><TD>Optimized for <I>Saccharomyces cerevisiae</I></TD><TD>Saccharomyces Genome Database</TD></TR></TBODY></TABLE><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 1:</B> The finalized cas9 plasmid containing the gRNA transformed into yeast in conjuncture with the MF(ALPHA)2 fragments with homologous ends to the up/downstream DNA of BAR1</I></FIGCAPTION></FIGURE><P>Since we wanted to delete BAR1 and introduce MF(ALPHA)2 into the genome a repair fragment was constructed. The MF(ALPHA)2 was put under the FUS1 promoter as we want a positive feedback loop for the production of alpha pheromone, see figure 2.   </P><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 2:</B> The gBlock for MF(ALPHA)2 that was transformed into the yeast as a repair fragment for the double stranded break made in BAR1</I></FIGCAPTION></FIGURE><H3>Checking the Deletion of BAR1 and Insertion of MF(ALPHA)2</H3><P>In order to evaluate if the deletion and insertion was successful a genomic PCR was carried out. Using primers which are placed up/downstream of the BAR1 site. Since the size difference between the BAR1 and the FUS1 promoter plus the MF(ALPHA)2 gene is about 700bp, 2.3kb for BAR1 and 1.6 kb for FUS1+MF(ALPHA)2, the differenc between the two should be easy to check via PCR.</P><P>Unfortunately the deletion seems to not have worked. The PCR was run on several different colonies from the plates that selected for the Cas9 plasmid. The PCR was also ran with several different temperatures and polymerases. There were however always no bands or bands at the 2.3 kb mark. So it was assumed that the gRNA or the gibson had not worked as intended, considering that the plasmid obviously had been transformed into the yeast as it was able to grow on g418 agar plates.</P><H2>pFUS1-GFP</H2><P>Even thought the incorporation of MF(ALPHA)2 into the genome of our yeast strain seemed to have failed we still wanted to check how expression levels were related to the prescence of alpha pheromone. They way we went about this was by making a construct with GFP regulated by the FUS1 promotor, see figure 3 and table 2. The idea is that an assay with different concentrations of alpha pheromone could be used to improve our modelling with our own results from the lab. 
</P><TABLE class="table table-striped table-bordered table-responsive-sm"><CAPTION><I><B>Table 2: </B>pFUS1-GFP proteins</I></CAPTION><TBODY><TR><TH>Protein</TH><TH>Full name</TH><TH>Native organism</TH><TH>Sequence source</TH></TR><TR><TD>pFUS1</TD><TD>FUS1 promoter</TD><TD>Optimized for <I>Saccharomyces cerevisiae</I></TD><TD>iGEM parts registry: BBa_K1072023</TD></TR><TR><TD>GFP</TD><TD>Green Fluorescent Protein</TD><TD>Optimized for <I>Saccharomyces cerevisiae</I></TD><TD>SysBio lab at Chalmers</TD></TR></TBODY></TABLE><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 3:</B> The finalized FUS1-GFP plasmid transformed into yeast</I></FIGCAPTION></FIGURE><H3>Testing the Construct</H3><P>The constructs were tested in a very simple manner. A overnight culture of the yeast was prepared in delft+HIS media and left to grow at 30 degrees. The next day the culture was split into 16 different PCR tubes each containing 100 microlitres of culture. Alpha pheromone, final concentration of 20 micromolar was added to half of the tubes and they were incubated in 30 degrees for an additional two hours.</P><P>However it turned out that both the negative and positive cultures all had yeast expressing GFP. The conclusion that was drawn was that the gibson had failed and that the TEF1 promoter was somehow still present. As the results for the genomic PCR for verification of the CRISPR insert did not yield any results this seemed to be the only explanation.</P><H2>p413-FUS1-Alphafactor-Myrosinase</H2><P> The final construct for the expression of myrosinase in the prescens of alpha pheromone, see figure 4 and table 3 for a overview of the plasmid and proteins </P><TABLE class="table table-striped table-bordered table-responsive-sm"><CAPTION><I><B>Table 3: </B>p413-FUS1-Alphafactor-Myrosinase</I></CAPTION><TBODY><TR><TH>Protein</TH><TH>Full name</TH><TH>Native organism</TH><TH>Sequence source</TH></TR><TR><TD>pFUS1</TD><TD>FUS1 promoter</TD><TD>Optimized for <I>Saccharomyces cerevisiae</I></TD><TD>iGEM parts registry: BBa_K1072023</TD></TR><TR><TD>Kozak sequence</TD><TD>Kozak sequence</TD><TD>Optimized for <I>Saccharomyces cerevisiae</I></TD><TD>SysBio lab at Chalmers</TD></TR><TR><TD>Alpha factor</TD><TD>Alpha factor leader</TD><TD>Optimized for <I>Saccharomyces cerevisiae</I></TD><TD>SysBio lab at Chalmers</TD></TR><TR><TD>Kex2</TD><TD>Kex2</TD><TD>Synthetic sequence</TD><TD>SysBio lab at Chalmers</TD></TR><TR><TD>Spacer</TD><TD>gaagaaggtgaaccaaaa</TD><TD>Synthetic sequence </TD><TD>Article (Liu et al, 2011)</TD></TR><TR><TD>Myr1</TD><TD>Myrosinase</TD><TD>Optimized for <I>Saccharomyces cerevisiae</I></TD><TD>Uniprot</TD></TR></TBODY></TABLE><P>As the experiments with pFUS-GFP seemed to indicate that the TEF1 promoter had not been deleted properly we were worried about how our larger constructs would be integrated into the plasmids. But sequencing of this construct later revealed that the pFUS promoter had been efficiently inserted into the plasmid and that the construct was perfectly fine.</P><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 4:</B> The finalized Myrosinase plasmid transformed into yeast</I></FIGCAPTION></FIGURE><H3>Testing the Construct</H3><P>To test the activity and secretion of the myrosinase in the <I> S.cerevisiae </I> the following procedure was performed:</P><P>RKO cells were grown in Eagle's Minimum Essential Media + 10% FBS, by one of our supervisor in the university's cancer cell lab. The cells were grown in 24-well plates until 80% confluence. Whilst the cancer cells were growing an 50 ml Delft+HIS media overnight culture of the <I> S.cerevisiae </I> transformed with the myrosinase construct was prepared and incubated at 30 degrees. The next day the culture was pelleted through centrifugation and split into two tubes. The cells were resuspended in 4 ml DELFT+HIS media and 66 microlitres of alpha pheromone (1mg/ml) to each tube to start production of the myrosinase. The yeast was incubated at 30 degrees for 2 hours and then placed in the fridge for use the next day.</P><P>The  cultures with added alpha pheromone were filtered through a 0.45 micrometer sterile cellulose filter, to yield a cell free myrosinase solution. The cell free myrosinase solution was split into two tubes containing 600 microlitres each, to one of the tubes 100mg of sinigrin was added. 100 microlitres of the cell free solutions was mixed into 400 microlitres of fresh media. The solution with sinigrin was added to six wells and the solution without sinigrin was added to another six wells. The cancer cell + myrosinase solution was incubated for 48 hours and after that the wells were examined under a microscope, see figure 5. The results was that all the cells in the wells containing sinigrin had died and there were still a considerable amount of cells in the control wells.</P><FIGURE style="text-align:center;"><FIGCAPTION><I><B>Figure 5:</B> Microscope comparison of cancer cells in different media. A: RKO cancer cell growing in EMEM + 10% FBS. B: RKO cancer cell growing in EMEM + 10% FBS + Myrosinase solution after 48h incubation. C: RKO cancer cell growing in EMEM + 10% FBS + Myrosinase and Sinigrin solution after 48h incubation </I></FIGCAPTION></FIGURE><H1 style="text-align:center">Attachments</H1><H3>Lab Journals</H3><UL><LI><A href="https://static.igem.org/mediawiki/2018/f/f6/T--Chalmers-Gothenburg--Myr_GFP_Benchling.pdf">Lab Journal - Treatment</A></LI></UL><H3>Primer, Construct, and Plasmid Sequences</H3><UL><LI><A href="https://static.igem.org/mediawiki/2018/b/b9/T--Chalmers-Gothenburg--plasmids-for-treatment.zip">Construct and Plasmid Sequences</A></LI><LI><A href="https://static.igem.org/mediawiki/2018/a/a4/T--Chalmers-Gothenburg--treatment-primers.xlsx">Primer Sequences</A></LI></UL><H1 style="text-align:center">Protocols</H1><P> Find all protocols here, <A href="https://2018.igem.org/Team:Chalmers-Gothenburg/Protocols">Labwork-Protocols</A>.</P><H1 style="text-align:center">References</H1></DIV></DIV><DIV class="row" style="font-size:70%;"><DIV class="col-6"><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Bourdeau, R. W., Lee-Gosselin, A., Lakshmanan, A., Farhadi, A., Kumar, S. R., Nety, S. P., &amp; Shapiro, M. G. (2018). Acoustic reporter genes for noninvasive imaging of microorganisms in mammalian hosts. <EM>Nature</EM>. https://doi.org/10.1038/nature25021</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Chen, X., Zaro, J. L., &amp; Shen, W.-C. (2013). Fusion protein linkers: Property, design and functionality. <EM>Advanced Drug Delivery Reviews</EM>, <EM>65</EM>(10), 1357–1369. https://doi.org/10.1016/j.addr.2012.09.039</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Englert, C., &amp; Pfeiffer, F. (1993). Analysis of gas vesicle gene expression in Haloferax mediterranei reveals that GvpA and GvpC are both gas vesicle structural proteins. <EM>The Journal of Biological Chemistry</EM>, <EM>268</EM>(13), 9329–9336. Retrieved from http://www.jbc.org/content/268/13/9329.short</P></DIV><DIV class="col-6"><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Liu, Z., Tyo, K. E. J., Martínez, J. L., Petranovic, D., &amp; Nielsen, J. (2012). Different expression systems for production of recombinant proteins in Saccharomyces cerevisiae. <EM>Biotechnology and Bioengineering</EM>, <EM>109</EM>(5), 1259–1268. https://doi.org/10.1002/bit.24409</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Partow, S., Siewers, V., Bjørn, S., Nielsen, J., &amp; Maury, J. (2010). Characterization of different promoters for designing a new expression vector in Saccharomyces cerevisiae. <EM>Yeast</EM>, <EM>27</EM>(11), 955–964. https://doi.org/10.1002/yea.1806</P><P style="margin-left: 24.0pt; text-indent: -24.0pt;">Souza-Moreira, T. M., Navarrete, C., Chen, X., Zanelli, C. F., Valentini, S. R., Furlan, M., … Krivoruchko, A. (2018). Screening of 2A peptides for polycistronic gene expression in yeast. <EM>FEMS Yeast Research</EM>, <EM>18</EM>(5). https://doi.org/10.1093/femsyr/foy036</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="footer"><FOOTER><H4 style="font-size:12pt;">
    CONTACT INFORMATION: 
  </H4><DIV><P style="font-size:10pt;">
      CHALMERS University of Technology, Gothenburg, Sweden, igemgothenburg@gmail.com   
</P><SPAN><A style="font-size:10pt;" href="https://www.facebook.com/chalmersigem">Facebook</A><A style="font-size:10pt;" href="https://www.instagram.com/chalmersigem">Instagram</A><A style="font-size:10pt;" href="https://twitter.com/igemchalmers">Twitter</A></SPAN></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>