Team:Chalmers-Gothenburg/Attachment
Interlab - iGEM Chalmers-Gothenburg 2018
PROJECT
Project Overview Project Summary Achievements
LABWORK
Binding Detection Treatment
InterLab BioBricks Protocols Safety
HUMAN PRACTICES
Human Practices Collaborations Outreach
MODELING
Model Overview Model Design
PEOPLE
Team Supervisors Sponsors Attributions
SOCIAL MEDIA
Facebook Instagram Twitter
Navigation
Labwork and Results
Attachments
Protocols
Yeast Cell to Cancer Cell Binding
For this part of the labwork, three different plasmids are assembled. The first plasmid is the “final product” for the cancer cell attachment, a plasmid containing the yeast anchoring protein Aga2 as well as the colorectal cancer cell surface attaching protein HlpA. To control the function of this fusion protein, yeast to cancer cell binding is tested in vitro.
The second plasmid is assembled to confirm that the final product is expressed correctly, and in the desired location. To confirm this, GFP is fused to the assembled Aga2-HlpA protein and expressed in yeast. The GFP expression and location of the expressed protein is then examined under microscope.
The third plasmid serves as confirmation of the cell surface display system, namely the function of Aga2. This is done through fusing only GFP to Aga2, after which the GFP expression and location is examined under a microscope.
General Workflow
An overview of the general workflow, from amplification of genes to cloning of plasmids into yeast, is presented in Labwork-Protocols .
The only plasmid backbone used for expression of proteins in this part of the lab-work is p416TEF, digested with XbaI and XhoI. The plasmids consecutive promoter, TEF1, as well as terminator, CYC1, are both kept and used for the expression of our protein. The plasmid also contains the bacterial ampR gene, as well as a the yeast URA3 marker gene. The yeast strain used was PK113-11C, which carries the auxotrophic his3 and ura3 markers. Since only the plasmid p416TEF is used, all minimal media used are supplemented with Histidine.
In the following sections, the construction of the parts mentioned above, the methods used for checking the protein function as well as the results are be presented. Note that the exact mechanisms of function for the proteins are not presented here, please go to Project Summary page for detailed description. The week by week lab journal is attached in the end of this chapter.
p416TEF-Kozak-Aga2-Linker-HlpA
In the section below, the gBlock and primer design as well as the methods used for testing the protein function are discussed. The assembly itself is not described here, for information about this, please consult the Protocols section.
gBlock Construction
Because Aga2 attaches to the yeast cell with the N-terminal side of the protein, HlpA is fused to the other side, namely the C-terminal. In terms of gBlock construction, this means that HlpA is fused to the 5’-end of Aga2. The stop codon of Aga2 is removed, as well as the start codon of HlpA to allow continuous expression of both. A flexible linker is placed in between the genes to avoid sterical hindrance of both proteins. To increase protein expression, a Kozak sequence is added before Aga2. Non-native yeast sequences is optimized for S. cerevisiae. The constructed gBlock was synthesized by IDT. The full names of the proteins as well as the source of the gene sequence are listed in the table below.
Table 1: p416TEF-Kozak-Aga2-Linker-HlpA proteins
Protein
Full name
Native organism
Sequence source
Aga2
Alpha-Agglutin 2
Saccharomyces cerevisiae
Uniprot
HlpA
Histone like protein A
Streptococcus gallolyticus
Homology BLAST in genome of S. gallolyticus, with HlpA gene from Streptococcus pyrogenes
(GGGS)x4
Flexible linker
Synthetic sequence
SysBio lab Chalmers
Kozak sequence
Kozak sequence
Synthetic sequence
SysBio lab Chalmers
Figure 1: Plasmid p416TEF with inserted Aga2-HlpA construct for expression of HlpA at the yeast cell surface.
Primer Design
Primers are designed with about 20 bp of overlap to the gBlock, and a tail that overlaps by about 30 bp to the ends of the XhoI/XbaI linearized p416TEF backbone. This allows for efficient Gibson assembly. Primers are listed in the file "HlpAPrimerList" below.
Methods for Testing Protein Function
The following method is followed for testing the colon cancer cells binding with S. cerevisiae engineered with Aga2-HlpA: RKO cells are grown in Eagle's Minimum Essential Medium (EMEM) + FBS 10%, by one of our supervisors in the university's cancer cell lab. Cells are grown both in a 42 wells plate and subsequently, after trypsin treatment, are placed in falcon tubes in PBS buffer solution. S. cerevisiae is grown in Delft media + His overnight both at 30°C and 22°C. A negative control is also made with S. cerevisiae containing only the empty p416Tef plasmid, the negative control is grown in the same conditions.  Yeast is centrifuged, then the media removed and the cells resuspended in PBS buffer. Resuspended cancer cells and cells in the wells plate attached to the surface are incubated with yeast (both negative control and test sample) in a 30°C room.  After 30 minutes, the wells plate is washed with PBS couple of times, then the surface is scratched with a pipette tip and PBS is added, half the amount is checked under the microscope and the other half is plated in a YPD plate. The liquid culture is also checked under the microscope.  For detailed method please check the Lab Notebook.
Results of Cancer Cell Binding
The result from plating was not as expected, in fact, in both negative control and test sample, yeast cells grew. This can be because the PBS washing was not effective or because the yeast did not attach to the cancer cells.
The microscope results are shown in the figures below, we saw that cancer cells died. However, we could not observe that the yeast was considerably binding to the cancer cells. The only binding we could see was to cells that were still alive. However, the binding frequency was so small to be considered relevant.
Figure 2:Dying cancer cell surrounded by S. cerevisiae cells on the left; Round, live cancer cell with attached S. cerevisiae cells on the right.
p416TEF-Kozak-Aga2-Linker-HlpA-Linker-GFP
Instead of synthesizing a new gBlock with the new fusion protein, the previously synthesized Kozak-Aga2-HlpA was used as a template. GFP was amplified from the plasmid p413TEF-GFP available in our lab. Below, the primer design and origins of new parts are described, as well as the methods used to test the protein function. Previously mentioned genes come from the same origin as discussed previously.
Origin of New Genes
New parts in this plasmid are both the second linker and the GFP. The full names of the proteins, as well as the source of the gene sequence are listed in the table below.
Table 2: p416TEF-Kozak-Aga2-Linker-HlpA-Linker-GFP proteins
Protein
Full name
Native organism
Sequence source
(GGGS)x2
Flexible Linker
Synthetic sequence
Article (Chen et al., 2013)
GFP
Green Fluorescent Protein
Optimized for Saccharomyces cerevisiae
Plasmid in SysBio lab at Chalmers
Figure 2: Plasmid p416TEF with inserted Aga2-HlpA-GFP construct for expression of GFP-tagged HlpA at the yeast cell surface.
Primer Design
Primers were designed for Gibson assembly of Kozak-Aga2-HlpA and GFP into the XhoI/XbaI linearized p416TEF. In addition to this, the primers were designed to add a linker between the HlpA and GFP to allow for better protein function, and to remove the stop codon of HlpA as well as start codon of GFP. Since The GFP was amplified from a plasmid, the plasmid template had to be digested with DnpI before purification of PCR-product. Primers are listed in file "HlpAPrimerList" below.
Methods for Testing Protein Function
The purpose of GFP-tagging Kozak-Aga2-HlpA is to see if the protein was expressed properly, and if it reaches the correct place; the cell surface. In order to check this, 3 colonies of 11C yeast containing p416TEF-Kozak-Aga2-HlpA-Linker-GFP were inockuIated in Delft+His media and were grown in at 30°C overnight. A negative control of 11C yeast was grown in Delft+His+Ura media in the same way. The results of the protein expression were then checked under microscope. The same experiment was also repeated several times at room temperature to improve protein folding.
Results Recombinant Protein Expression
Even with several replicates of the experiment, and with yeast grown at room temperature, no GFP showed under the microscope. There are three possible explanations for this, since the protein sequences are correct based on the sequencing. The first explanation would be that the protein could be expressed, but misfolded. In this case HlpA is the most likely the cause of the misfolding, since it has never been expressed in yeast before while the other proteins have. The second explanation would be that the error occurred on an transcriptional level, however we did not have time or resources to look further into this. Lastly, the protein could be secreted, but Aga2 failed to adhere to Aga1 on the yeast surface. This is also something we lacked the resources to confirm. Moving on with what we had, checking the function of our anchor is the next course of action.
p416TEF-Kozak-Aga2-Linker-GFP
Primer Design
Primers were designed for Gibson assembly of Kozak-Aga2 and GFP into the XhoI/XbaI linearized p416TEF. Just like in the previous case, the primers were designed to add a linker in between the Aga2 and GFP to allow for better protein function. The Aga2 sequence already lacked a stop codon, but the start codon of the GFP still had to be removed. Since the GFP was amplified from a plasmid, the plasmid template had to be digested with DnpI before purification of PCR product. Primers are listed in the file "HlpAPrimerList" below.
Figure 3: Plasmid p416TEF with inserted Aga2-GFP construct for expression of GFP at the yeast cell surface.
Methods for Testing Cell Surface Binding
In order to see if Aga2 and GFP were properly expressed and functioning, the cell surface binding was once again tested under microscope. The plasmid p416TEF-Kozak-Aga2-Linker-GFP was transformed into yeast 11C, after which 3 colonies were picked and inoculated in Delft-His minimal media. As a negative control a 11C colony without the plasmid was inoculated overnight in Delft-His-Ura minimal media, and as a positive control 11C containing p413TEF-GFP was inoculated overnight in Delft-Ura minimal media. All inoculated cultures were grown overnight in 30 degrees celsius, after which the GFP expression and location was checked under the microscope. This experiment was also repeated with incubation at room temperature.
Results Recombinant Protein Expression
The results were not as expected. Since the anchoring sequence has been shown to work with GFP before, and since the GFP expression of the GFP template plasmid works well by itself, the expected results were GFP expression at the cell surface. However, in our cells expressing Aga2-GFP, the fluorescence was weak compared to when the GFP alone was expressed. It also seemed like the Aga2 did not attach to the surface of the cell, but was located close to the nucleus of the cell. One theory is that the Aga2-GFP is expressed and secreted, but does not attach to the cell surface. With more time, a follow-up experiment would have been to overexpress Aga1, the yeast cell-wall protein that Aga2 attaches to, to see if this would make a difference.
Attachments
Below, links to the lab journal and the GenBank files are presented.
Lab Journal
Lab Journal
Note: In the lab journal the Aga2-Linker-HlpA gBlock is often referred to as “HlpA” only.
Plasmids, Primers and Constructs
Primers
Constructs and plasmids
Protocols
Find all protocols Labwork-Protocols .
References
Bourdeau, R. W., Lee-Gosselin, A., Lakshmanan, A., Farhadi, A., Kumar, S. R., Nety, S. P., & Shapiro, M. G. (2018). Acoustic reporter genes for noninvasive imaging of microorganisms in mammalian hosts. Nature. https://doi.org/10.1038/nature25021
Chen, X., Zaro, J. L., & Shen, W.-C. (2013). Fusion protein linkers: Property, design and functionality. Advanced Drug Delivery Reviews, 65(10), 1357–1369. https://doi.org/10.1016/j.addr.2012.09.039
Englert, C., & Pfeiffer, F. (1993). Analysis of gas vesicle gene expression in Haloferax mediterranei reveals that GvpA and GvpC are both gas vesicle structural proteins. The Journal of Biological Chemistry, 268(13), 9329–9336. Retrieved from http://www.jbc.org/content/268/13/9329.short
Liu, Z., Tyo, K. E. J., Martínez, J. L., Petranovic, D., & Nielsen, J. (2012). Different expression systems for production of recombinant proteins in Saccharomyces cerevisiae. Biotechnology and Bioengineering, 109(5), 1259–1268. https://doi.org/10.1002/bit.24409
Partow, S., Siewers, V., Bjørn, S., Nielsen, J., & Maury, J. (2010). Characterization of different promoters for designing a new expression vector in Saccharomyces cerevisiae. Yeast, 27(11), 955–964. https://doi.org/10.1002/yea.1806
Souza-Moreira, T. M., Navarrete, C., Chen, X., Zanelli, C. F., Valentini, S. R., Furlan, M., … Krivoruchko, A. (2018). Screening of 2A peptides for polycistronic gene expression in yeast. FEMS Yeast Research, 18(5). https://doi.org/10.1093/femsyr/foy036
CONTACT INFORMATION:
CHALMERS University of Technology, Gothenburg, Sweden, igemgothenburg@gmail.com
Facebook Instagram Twitter
