Team:Chalmers-Gothenburg/Labwork
Interlab - iGEM Chalmers-Gothenburg 2018
PROJECT OVERVIEW
Project Summary Project Achievements
LABWORK
Attachment Treatment Detection
Interlab Biobricks Protocols Safety
MODELING
Model Overview Model Design
HUMAN PRACTICES
Human Practices Collaborations Outreach
PEOPLE
Team Supervisors Sponsors Attributions
SOCIAL MEDIA
Facebook Instagram Twitter
Navigation
Labwork and Results
Lab Journal
Protocols
Safety
Labwork and Results
Yeast to cancer cell attachment
For this part of the labwork, three different plasmids were assembled. The first plasmid presented here was the “final product” for the cancer cell attachment, a plasmid containing the yeast anchoring protein Aga2 as well as the colorectal cancer cell surface attaching protein HlpA. To control the function of this fusion protein, yeast to cancer cell binding was tested in vitro.
The second plasmid presented here was assembled to confirm that the final product was expressed correctly, and in the desired location. To confirm this, GFP was fused to the assembled Aga2-HlpA protein and expressed in yeast. The GFP expression and location of the expressed protein was then examined under microscope.
The third plasmid presented was assembled to confirm the function of the cell surface display system, namely the function of Aga2. This was done through fusing only GFP to Aga2, after which the GFP expression and location was examined under a microscope.
General workflow
An overview of the general workflow, from amplification of genes to cloning of plasmids into yeast, is presented below.
The only plasmid backbone used for expression of proteins in this part of the lab-work was p416TEF, digested with XbaI and XhoI. The plasmids consecutive promoter, TEF1, as well as terminator, CYC1, were both kept and used for the expression of our protein. The plasmid also contains the bacterial ampR gene, as well as a the yeast URA3 marker gene. The yeast strain used was PK113-11C, which carries the auxotrophic his3 and ura3 markers. Since only the plasmid p416TEF was used, all minimal media used had to be supplemented with Histidine.
In the following sections, the construction of the parts mentioned above, the methods used for checking the protein function as well as the results will be presented. Note that the exact mechanisms of function for the proteins will not be presented here, please go to Project Overview - Anchoring for detailed description. The week by week lab journal is attached in the end of this chapter.
p416TEF-Kozak-Aga2-Linker-HlpA
Below the gBlock and primer design is described, as well as the methods used to testing the protein function. The assembly itself is not desctibed here, since the description can be found in the Protocols section.
gBlock construction
Due to that Aga2 attaches to the yeast cell with the N-terminal of the protein, HlpA was fused to the C-terminal of Aga2. In terms of gBlock construction, this means that HlpA was fused to the 5’-end of Aga2. The stop codon of Aga2 was removed as well as the start codon of HlpA. To avoid sterical hindrance of the function of both proteins, a flexible linker was placed in between the genes. To get a better protein expression a Kozak sequence was added before Aga2. Non native yeast sequences were optimized for S. cerevisiae. The constructed gBlock was synthesized by IDT. The full names of the proteins as well as the source of the gene sequence are listed in the table below.
Protein
Full name
Native organism
Sequence source
Aga2
Alpha-Agglutin 2
Saccharomyces cerevisiae
Uniprot
HlpA
Histone like protein A
Streptococcus gallolyticus
Homology BLAST in genome of S. gallolyticus, with HlpA gene from Streptococcus pyrogenes
(GGGS)x4
Flexible linker
Synthetic sequence
SysBio lab Chalmers
Kozak sequence
Kozak sequence
Synthetic sequence
SysBio lab Chalmers
Figure 1: Plasmid p416TEF with inserted Aga2-HlpA construct for expression of HlpA at the yeast cell surface.
Primer design
Primers were designed with about 20 bp overlap to gBlock, and an about 30 bp long tail that overlaps with the ends of the XhoI/XbaI linearized p416TEF backbone. This allows for efficient Gibson assembly. Primers are listed in file "HlpAPrimerList" below.
Methods for testing protein function
For testing the colon cancer cells binding with S.cerevisiae engineered with Aga2-HlpA the following method was followed:  RKO cells were grown in Eagle's Minimum Essential Medium (EMEM) + FBS 10%, by one of our supervisors in the university's cancer cell lab. Cells were grown both in a 42 wells plate and subsequently, after trypsin treatment, were also placed in falcon tubes in PBS buffer solution. S.cerevisiae was grown in Delft media + His overnight both at 30°C and 22°C. A negative control was also made with S.cerevisiae containing only the empty p416Tef plasmid, the negative control was grown in the same conditions.  Yeast was centrifuged, then the media was removed and the cells were resuspended in PBS buffer. Resuspended cancer cells and cells in the wells plate attached to the surface were incubated with yeast (both negative control and test) in a 30°C room.  After 30 minutes, the wells plate was washed with PBS couple of times, than the surface was scratched with a pipette tip and PBS was added, half the amount was checked under the microscope and the other half was plated in a YPD plate. The liquid culture was also checked under microscope.  For detailed method please check the Lab Notebook.
Results of cancer cell attachment
The result from plating was not as expected, in fact, in both negative control and test, yeast cells grew. This can be because the PBS washing was not effective or because the yeast did not attached.
Figure n°x: Negative control: 11C S.cerevisiae + p416tef plasmid after cancer cells binding The results from the microscope are showed in the figures below, we saw that cancer cells were dying and we could not conclude that the yeast was considerably binding to the cancer cells. However, the only binding we could see was to cells that were still alive but still the quantity was so small that it is not considered interesting. FIGURESSSSSSSSS
Important for lab journal
In lab journal the Aga2-Linker-HlpA gBlock is often referred to as “HlpA” only.
p416TEF-Kozak-Aga2-Linker-HlpA-Linker-GFP
Instead of synthesizing a new gBlock with the new fusion protein, the previously synthesized Kozak-Aga2-HlpA was used as a template. GFP was amplified from the plasmid p413TEF-GFP available in our lab. Below the primer design and origin of new parts are described, as well as the methods used to test the protein function. Please see previous chapter for origin of previously mentioned genes.
Origin of new genes
The new parts in this plasmid are the second linker and the GFP. The full names of the proteins as well as the source of the gene sequence are listed in the table below.
Protein
Full name
Native organism
Sequence source
(GGGS)x2
Flexible Linker
Synthetic sequence
Article (Chen et al., 2013)
GFP
Green Fluorescent Protein
Optimized for Saccharomyces cerevisiae
Plasmid in SysBio lab at Chalmers
Figure 2: Plasmid p416TEF with inserted Aga2-HlpA-GFP construct for expression of GFP-tagged HlpA at the yeast cell surface.
Primer design
Primers were designed for Gibson assembly of Kozak-Aga2-HlpA and GFP into the XhoI/XbaI linearized p416TEF. In addition to this, the primers were designed to add a linker in between the HlpA and GFP to allow for better protein function, and to remove the stop codon of HlpA as well as start codon of GFP. Since The GFP was amplified from a plasmid, the plasmid template had to be digested with DnpI before purification of PCR-product. Primers are listed in file "HlpAPrimerList" below.
Methods for testing protein function
The purpose of GFP-tagging Kozak-Aga2-HlpA was to see that the protein was expressed properly, and at the correct place. In order to check this, 3 colonies of 11C yeast containing p416TEF-Kozak-Aga2-HlpA-Linker-GFP were inockuIated in Delft+His media and were grown in a 30 degree C overnight. A negative control of 11C yeast was grown in Delft+His+Ura media in the same way. The the results of the protein expression were then checked under microscope. The same experiment was also repeated in room temperature to help with protein folding and repeated several times.
Results recombinant protein expression
Even with several replicates of the experiment, and with yeast grown in room temperature, no GFP showed under the microscope. There are three possible explanations for this, since the protein sequences are correct based on the sequencing. The protein could be expressed, but misfolded. In this case HlpA is the most likely the cause of the misfolding, since it has never been expressed in yeast before while the other proteins have. It is also possible that the error occurred on an transcriptional level, however we did not have time or resources to look further into this.  The third possibility is that the protein is secreted, but that Aga2 failed to adhere to Aga1 on the yeast surface. This is also something that we lacked the resources to confirm. Moving on with what we had, checking the function of our anchor was the next course of action.
p416TEF-Kozak-Aga2-Linker-GFP
Primer design
Primers were designed for Gibson assembly of Kozak-Aga2 and GFP into the XhoI/XbaI linearized p416TEF. Just like in the previous case, the primers were designed to add a linker in between the Aga2 and GFP to allow for better protein function.the Aga2 sequence already lacked a stop codon, but the start codon of the GFP still had to be removed. Since the GFP was amplified from a plasmid, the plasmid template had to be digested with DnpI before purification of PCR-product. Primers are listed in file "HlpAPrimerList" below.
Figure 3: Plasmid p416TEF with inserted Aga2-GFP construct for expression of GFP at the yeast cell surface.
Methods for testing cell surface binding
In order to see if Aga2 and GFP were properly expressed and functioning, the cell surface binding was once again tested under microscope. The plasmid p416TEF-Kozak-Aga2-Linker-GFP was transformed into yeast 11C, after which 3 colonies were picked and inoculated in Delft-His minimal media. As a negative control a 11C colony without the plasmid was inoculated overnight in Delft-His-Ura minimal media, and as a positive control 11C containing p413TEF-GFP was inoculated overnight in Delft-Ura minimal media. All inoculated cultures were grown overnight in 30 degrees celsius, after which the GFP expression and location was checked under microscope. This experiment was also repeated with incubation in room temperature.
Results recombinant protein expression
The results were not what we expected. Since the anchoring sequence has been shown to work with GFP before, and since the GFP expression foom the GFP template plasmid works well by itself the results should have been GFP expressed at the surface at the cell. However, in our cells expressing Aga2-GFP the fluorescence was weak compared to when the GFP alone was expressed. It also seemed like the Aga2 did not attach to the surface of the cell, but was located close to the nucleus of the cell. One theory is that the Aga2-GFP is expressed and secreted, but does not attach to the cell surface. If time had allowed a follow-up experiment would have been to overexpress Aga1, the yeast cell-wall protein that Aga2 attaches to, to see if this would make a difference.
Labwork Journals
Protocols
General workflow
In the figure below, a schematic overview of the workflow in the lab is presented.
Safety
Treatment
There are a few different parts that constitutes this project, to give you a quick overview: CRISPR- cas9 the yeast in order to allow it to produce alpha-pheromone  Myrosinase as a cancer toxin p28 as a cancer toxin
The goal of the CRISPR-Cas9 system was to delete BAR1, which is a part of the mating system of yeast and leads to the degradation of alpha-pheromone inside of the yeast, and replace it with mf-alpha, the protein which codes for the alpha  pheromone https://www.yeastgenome.org/locus/S000001277 . This construct was transformed into yeast and the resulting strain was used for all further transformations related to the cancer toxins. The reason for this is that the cancer toxins were put under a FUS1 promoter which is only responsive to the alpha pheromone.
Moving into this project both Myrosinase and p28 were planned to be utilized at the same time in the same yeast. However during the beginning of august it was decided that p28 should be dropped and that myrosinase would be the sole focus. This was due to poor modeling results for the p28 and a strict time limit to this project.
This means that the construct containing p28 was never transformed into yeast and thus never tested in vitro. The Myrosinase however was transformed into our edited yeast strain and its efficiency was tested on RKO cells in vitro.
There was a few different plasmid used for these constructs. The CRISPR-cas9 plasmid used was a KanMX Cas9 Plasmid. It has a kanamycin resistance for the e.coli and a g418 resistance for the yeast selection process. BOIIIIIIIII ASKE ALEXE. The remaining constructs were put into the p413TEF plasmid which contains the gene HIS3 as its marker. The p413TEF plasmids were however edited and the TEF promoter was switched out for the FUS1 promoter. This was done since we wanted the yeast to express the proteins of interest only in the presence of pheromones.
kanMX-cas9 and gRNA
The plasmid kanMX-cas9 was used as the source for the CRISPR system that was utilized in order to edit the genome of a 11C S. cerevisiae strain. The gRNA sequence (GCTGACGGAACATTTGCTGA)  was developed via a tool built into benchling.
Checking the deletion of BAR1 and insertion of mf-alpha
In order to evaluate if the deletion and insertion was successful a genomic PCR was carried out. Using the fw_mfAlpha_Part1 and rv_mfAlpha_Part2 primers, which are placed up/downstream of the BAR. Since the size difference between the BAR1 and the FUS1 promoter and mf-alpha gene is about 700bp, 2.3kb for BAR1 and 1.6 bp for FUS1+mf-alpha.
Unfortunately the deletion had not worked. The PCR was run on several different colonies from the plates that selected for the cas9 plasmid. The PCR was also ran with several different temperatures and polymerases. There were however always no bands or bands at the 2.3 kb mark. So it was assumed that the gRNA or the gibson had not worked as intended, considering that the plasmid obviously had been transformed into the yeast as it was able to grow on g418 agar plates.
pFUS1-GFP
fw_GFP(pFUS1) and rv_GFP(pFUS1), rv_pFUS1(GFP) and fw_pFUS1(Myr1)(works for all pFUS1 constructs not just Myrosinase ones)
Figure XX: The finalized FUS1-GFP plasmid transformed into yeast
Testing the construct
The constructs were tested in a very simple manner. A overnight culture of the yeast was prepared in delft+HIS media and left to grow at 30 degrees. The next day the culture was split into 16 different pcr tubes each containing 100 microlitres. Alpha-pheromone was added to half of the tubes and they were incubated in 30 degrees for an additional two hours.
However it turned out that both the negative and positive cultures all had yeast expressing GFP. The conclusion that was drawn was that the gibson had failed and that the TEF promoter was somehow still present. As the results for the genomic PCR for verification of the CRISPR insert did not yield any results.
p413-FUS1-Alphapro-Myrosinase
The p413TEF plasmid was cut with with the enzymes Sac1 and Xho1 to get rid of the TEF-promoter and in order to allow the protein of interest to expressed exclusively in the presence of alpha-pheromone.
Protein
Full name
Native organism
Sequence source
Kozak sequence
Kozak sequence
Optimized for Saccharomyces cerevisiae
SysBio lab at Chalmers
Alpha factor
Alpha factor leader
Optimized for Saccharomyces cerevisiae
SysBio lab at Chalmers
Kex2
Kex2
Synthetic sequence
SysBio lab at Chalmers
Spacer
gaagaaggtgaaccaaaa
Synthetic sequence
https://onlinelibrary.wiley.com/doi/full/10.1002/bit.24409
Myr1
Myrosinase
Optimized for Saccharomyces cerevisiae
Uniprot
As the experiments with pFUS-GFP seemed to indicate that the TEF promoter had not been deleted properly we were worried about how our larger constructs would be integrated into the plasmids. But sequencing of this construct later revealed that the pFUS promoter had been efficiently inserted into the plasmid and that the construct was perfectly fine.
Figure XX: The finalized Myrosinase plasmid transformed into yeast
Testing the construct
To test the activity and secretion of the myrosinase in the S.cerevisiae the following procedure was performed:
RKO cells were grown in Eagle's Minimum Essential Media + 10% FBS, by one of our supervisor in the university's cancer cell lab. The cells were grown in 24-well plates until 80% confluence. Whilst the cancer cells were growing an 50 ml Delft+HIS media overnight culture of the S.cerevisiae transformed with the myrosinase construct was prepared and incubated at 30 degrees. The next day the culture was pelleted through centrifugation and split into two tubes. The cells were resuspended in 4 ml DELFT+HIS media and 66 microlitres of alpha-pheromone (1mg/ml) to each tube to start production of the myrosinase. The yeast was incubated at 30 degrees for 2 hours and then placed in the fridge for use the next day.
The  cultures with added alpha-pheromone were filtered through a SIZE OF THE FILTER GOTTA CHECK THE LAB, to yield a cell free myrosinase solution. The cell free myrosinase solution was split into two tubes containing 600 microlitres each, to one of the tubes 100mg of sinigrin was added. 100 microlitres of the cell free solutions was mixed into 400 microlitres of fresh media. The solution with sinigrin was added to six wells and the solution without sinigrin was added to another six wells. The cancer cell + myrosinase solution was incubated for 48 hours and after that the wells were examined under a microscope. The results was that all the cells in the wells containing sinigrin had died and there were still a considerable amount of cells in the control wells.
Figure XX: Microscope comparison of cancer cells in different media. A: RKO cancer cell growing in EMEM + 10% FBS. B: RKO cancer cell growing in EMEM + 10% FBS + Myrosinase solution after 48h incubation. C: RKO cancer cell growing in EMEM + 10% FBS + Myrosinase and Sinigrin solution after 48h incubation
Detection
Bourdeau et al. (2018) have shown that organisms can be engineered to produce gas vesicles as acoustic reporter genes. They have combined gas vesicle genes from Bacillus megaterium and Anabaena flos-aquae to produce the acoustic reporter gene in Escherichia coli. To further improve their acoustic reporter gene they have modified the gene GvpC by deleting several copies of a repetitive region. Their final Acoustic reporter gene that serves as a starting point for this part of our iGEM project is represented in figure 3.1.
Figure 3.1: Acoustic Reporter Gene 2 (ARG2) as designed by Bourdeau et al. (2018)
Design of Eukaryotic Acoustic Reporter Gene
HAVE TO EDIT Operon from Prokaryote to Eukaryote polycistronic gene expression BLABLA  What are 2a peptides? Removal of stop codons before 2a peptides (because continuous expression) Also codon optimization for yeast and ease of synthesis In order to express this prokaryotic operon in the eukaryotic yeast, which is not able to transcribe gene operons, a multicistronic gene expression system is used. Namely, 2A viral peptides. This sequence, which we place in between each gene of the operon, codes for a peptide sequence that cleaves itself and separates the genes (Souza-Moreira et al. 2018). Without this system it would also be possible to express each gene separately, however, each gene should then be introduced under its own promoter and terminator while here, the gene set can be combined under one promoter and one terminator.
Figure 3.2: 2a peptide cleaving efficiency (top) and sequences of the three most efficiently cleaving 2a peptides (bottom) from Souza-Moreira et al. (2018)
From figure 3.2. we conclude that ERBV-1 is the most efficiently cleaving 2a peptide. Thus, it is this sequence that is introduced in between the genes. Our synthesis company limits the size of DNA constructs to around 2000 nucleotides. It is thus unfortunately not possible to make one continuous sequence containing all genes. Instead, three gene sets of around 2000bp are designed. To distribute the genes more or less evenly over these 3 different constructs, we assumed that the gene order is not important, this allowed us to distribute them as presented in figure 3.3
Figure 3.3: Constructs designed for acoustic reporter gene expression in yeast
These three constructs are inserted in 3 different plasmid vectors, all with an ampicillin resistance gene for selection in E. coli and with a different yeast amino acid auxotrophic marker for each plasmid:
p413, with a Histidine (His) marker
p414, with a Tryptophan (Trp) marker
p416, with a Uracil (Ura) marker
The plasmids will be transformed into a triple auxotrophic S. cerevisiae strain and only the cells with all three plasmids inserted, thus with all three constructs, are selected.
Figure 3.4: Plasmid vectors used to insert the acoustic reporter gene into yeast
Unfortunately, our supplier was not able to synthesize the DNA as we designed the constructs (figure 3.2). Only Gas Vesicle construct 2 was produced and delivered as ordered. As advised by them, we redesigned the constructs by re-codon-optimizing the coding sequences, limiting the amount of repetitive sequences and splitting up the sequences in shorter fragments. After a few cycles of ordering, failure to produce and redesigning, the final constructs as they could be produced and were delivered to us are presented in figure 3.5. The main changes are:
In order to have less repeats in the sequences, we decided to use a combination of two different 2a peptides in construct 1, the middle ERBV-1 2a peptide is replaced by a OpbuCPV18. The PTV 2a peptide has slightly higher cleavage efficiency but its sequence is almost identical to ERBV-1 (see figure 3.2.).
The third construct caused more problems and had to be split in three fragments. Specifically the gene GvpT was difficult to synthesize due to its highly repetitive sequence.
Figure 3.5: Constructs delivered by supplier
In order to fit these five fragments in the same three plasmid vectors (figure 3.4), we have opted for a double promoter strategy, as used in the pSP-G1 and pSP-G2 plasmids (Partow, Siewers, Bjørn, Nielsen, & Maury, 2010). In this paper, the authors present two plasmids that have two sets of promoters and terminators, which allow the introduction of two genes. In our project we do not use the whole plasmid, we only amplify the double promoter and terminator on this plasmid (figure 3.6), in order to insert it in the previously chosen plasmids (figure 3.4).
Figure 3.6: Amplification of double promoter and ADH1 terminator from pSP plasmid
Acoustic Reporter Gene Transformation Strategy
HAVE TO EDIT
Figure 3.7: Transformation strategy to insert the acoustic reporter gene into yeast
Results
HAVE TO EDIT
Methods for testing gas vesicle production
HAVE TO EDIT
Supplementary Data
Lab Journals
HAVE TO EDIT
Construct Sequences
HAVE TO EDIT
Primer Sequences
HAVE TO EDIT
Safety
Safety Introduction of Department
Before being granted access to the laboratory, every team member received a day long laboratory introduction including a lecture, a tour of the lab and an introduction to the safety rules and guidelines established in our laboratory. These are determined by both European (European Directive 2009/41/EC) and Swedish (Swedish regulation AFS 2005:1, in Swedish) regulation. Additional safety guidelines covering general safety instructions, accident risk identification and appropriate responses in case of an accident are set by the department of Biology and Biotechnology of Chalmers. Three laboratory engineers responsible for the safety in our department, as well as our supervisors have certified our competences allowing us to work in the lab.
Risk Assessment
Additionally to this safety training, we have carefully identified the risks related to our iGEM project and summarised these in a risk assessment document. This document presents an overview of all the equipment and chemicals necessary for the project, the risks that are involved in their use and how to minimise these. In order to involve every team-member and to ensure that everyone is aware of its content, we have split up our team and written two distinct risk assessments independently. Only after both of these risk assessments were approved we could start working in the laboratory.
Herewith both written risk assessments:
Contact information
igemgothenburg@gmail.com Facebook Instagram Twitter
