<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Pasteur_Paris_Science skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Pasteur Paris/Science</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><NAV class="menu"><UL><LI><FONT size="2.25pt" color="#00000"><SPAN class="lastword">Team </SPAN></FONT></LI><LI><FONT size="2.25pt" color="#00000"><SPAN class="lastword">Project </SPAN></FONT></LI><LI><FONT size="2.25pt" color="#00000"><SPAN class="lastword">Design </SPAN></FONT></LI><LI><FONT size="2.25pt" color="#00000"><SPAN class="lastword">Lab work </SPAN></FONT></LI><LI><FONT size="2.25pt" color="#00000"><SPAN class="lastword">Human Practice </SPAN></FONT></LI><LI><FONT size="2.25pt" color="#00000"><SPAN class="lastword">Achievements </SPAN></FONT></LI></UL></NAV></P><DIV id="Team" style="display:none"><DIV class="outer"><DIV class="inner"><DIV class="list rotate"><A href="https://2017.igem.org/Team:Pasteur_Paris/Team">Our Team </A></DIV></DIV></DIV><DIV class="outer1"><DIV class="inner"><DIV class="list rotate"><A href="https://2017.igem.org/Team:Pasteur_Paris/Team#pagesponsors">Sponsors </A></DIV></DIV></DIV><DIV class="outer1b"><DIV class="inner"><DIV class="list rotate"><A href="https://2017.igem.org/Team:Pasteur_Paris/Attributions">Attributions </A></DIV></DIV></DIV></DIV><DIV id="Project" style="display:none"><DIV class="outer2"><DIV class="inner"><DIV class="list rotate"><A href="https://2017.igem.org/Team:Pasteur_Paris/Project">Context</A></DIV></DIV></DIV><DIV class="outer3"><DIV class="inner"><DIV class="list rotate"><A href="https://2017.igem.org/Team:Pasteur_Paris/Entrepreneurship">Entrepreneurship</A></DIV></DIV></DIV><DIV class="outer3a"><DIV class="inner"><DIV class="list rotate"><A href="https://2017.igem.org/Team:Pasteur_Paris/Engagement">Engagement</A></DIV></DIV></DIV><DIV class="outer3b"><DIV class="inner"><DIV class="list rotate"><A href="https://2017.igem.org/Team:Pasteur_Paris/Model">Model</A></DIV></DIV></DIV></DIV><DIV id="Design" style="display:none"><DIV class="outer5"><DIV class="inner"><DIV class="list rotate"><A href="https://2017.igem.org/Team:Pasteur_Paris/Design">Design Methodology</A></DIV></DIV></DIV><DIV class="outer6"><DIV class="inner"><DIV class="list rotate"><A href="https://2017.igem.org/Team:Pasteur_Paris/Applied_Design">AppliedDesign</A></DIV></DIV></DIV><DIV class="outer7"><DIV class="inner"><DIV class="list rotate"><A href="https://2017.igem.org/Team:Pasteur_Paris/Science">Science</A></DIV></DIV></DIV></DIV><DIV id="Labwork" style="display:none"><DIV class="outer8"><DIV class="inner"><DIV class="list rotate"><A href="https://2017.igem.org/Team:Pasteur_Paris/Notebook">Notebook</A></DIV></DIV></DIV><DIV class="outer8b"><DIV class="inner"><DIV class="list rotate"><A href="https://2017.igem.org/Team:Pasteur_Paris/Demonstrate">Demonstrate</A></DIV></DIV></DIV><DIV class="outer9"><DIV class="inner"><DIV class="list rotate"><A href="https://2017.igem.org/Team:Pasteur_Paris/InterLab">InterLab</A></DIV></DIV></DIV><DIV class="outer10"><DIV class="inner"><DIV class="list rotate"><A href="https://2017.igem.org/Team:Pasteur_Paris/Protocol">Protocol</A></DIV></DIV></DIV><DIV class="outer11"><DIV class="inner"><DIV class="list rotate"><A href="https://2017.igem.org/Team:Pasteur_Paris/Safety">SafetySecurity</A></DIV></DIV></DIV></DIV><DIV id="HumanPractice" style="display:none"><DIV class="outer12"><DIV class="inner"><DIV class="list rotate"><A href="https://2017.igem.org/Team:Pasteur_Paris/HP/Law">Law</A></DIV></DIV></DIV><DIV class="outer13"><DIV class="inner"><DIV class="list rotate"><A href="https://2017.igem.org/Team:Pasteur_Paris/Collaborations">Collaborations</A></DIV></DIV></DIV><DIV class="outer13b"><DIV class="inner"><DIV class="list rotate"><A href="https://2017.igem.org/Team:Pasteur_Paris/HP/Silver">Human Practices</A></DIV></DIV></DIV><DIV class="outer14"><DIV class="inner"><DIV class="list rotate"><A href="https://2017.igem.org/Team:Pasteur_Paris/HP/Gold_Integrated">Integrated Practices</A></DIV></DIV></DIV></DIV><DIV id="Achievements" style="display:none"><DIV class="outer15"><DIV class="inner"><DIV class="list rotate"><A href="https://2017.igem.org/Team:Pasteur_Paris/Parts">Parts</A></DIV></DIV></DIV><DIV class="outer16"><DIV class="inner"><DIV class="list rotate"><A href="https://2017.igem.org/Team:Pasteur_Paris/GiantJamboree">GiantJamboree</A></DIV></DIV></DIV></DIV><DIV style="height:75vh"><P id="title" style="font-size:50pt; font-family: MinionPro, serif;"> Science</P><P id="subtitles" style=" font-family: MinionPro, serif; font-size:18pt">Design of our PAHs-degrading Bio-composite material</P></DIV><DIV class="text"><P class="maintext" style="font-family: Helvetica; text-align:center"><I>Figure 1:  Assembly of the basic structure</I></P><DIV class="subtitle"><P style="font-family: MinionPro, serif;font-size:18pt">Process</P></DIV><P class="maintext" style="line-height:16pt;"><B><SPAN style="font-size:15pt; font-family: MinionPro">Our </SPAN> approach to deal with the air pollution issue was inspired by nature. Atmospheric water condensation is able to naturally absorb and concentrate atmospheric particles. Therefore, we imagined a framework that can condense water droplets. The mechanical aspect of our bio-filter device is based on the spider web structure.
However, the main component is the enzymatic system capable of attacking and degrade PAHs.</B></P><DIV class="subtitle"><P style="font-family: MinionPro, serif; font-size:18pt">A. The PAHs degrading enzyme cocktail</P></DIV><P class="maintext" style=" font-size:9pt; line-height:16pt;">Our device must be able to degrade air pollutants. We chose to focus on the degradation of the organic components namely the Polycyclic Aromatic Hydrocarbons (PAHs) and more precisely the anthracene molecule, an endocrine disrupting and carcinogenic pollutant. We implemented a 4-step degradation pathway of this molecule, requiring 4 different enzymes.  The anthracene pollutant which is originally insoluble in water may, at the end of the degradation process, be recovered as sub moieties in an aqueous solution.

The PAHs are stable molecules that can be degraded through complex and very aggressive chemical processes. Due to the specific resonance properties of anthracene, it can be chemically modified on the central ring by oxidation through hydrogenation, oxidation with anhydrous chromic acid Cr03, yielding anthroquinone.  However, as we plan to use a biological process to proceed to the degradation of PAHs, we had to devise an alternative strategy for attacking the aromatic systems present in them. The first enzyme we chose is naphthalene 1,2-dioxygenase from <I>Pseudomonas putida</I>. This enzyme is a homodimer (E1-1 (<B>BBa_K2198000</B>), E1-2 (<B>BBa_K2198001</B>)) that is able to attack the lateral ring of anthracene to introduce the same modification as the harsh chromic acid chemical step, i.e. an oxydative hydrogenation (Figure 2.a). This oxidation with dioxygen forms a vicinal diol and adds two hydrogens on the ring. This step needs energy to break the aromaticity. This complex enzyme needs NADH,H+. The Second step in the reaction uses the dihydro diol dehydrogenase from <I>Ralstonia sp.</I> (E2, <B>BBa_K2198002</B>). This step allows the restoration of the aromaticity with a dehydrogenation of the vicinal diol. As a result, the next enzyme in the process will be able to open the ring with an oxidative cleavage. The reaction requires the action of the catechol 2,3- dioxygenase from <I>P. putida</I> (E3, <B>BBa_K2198003</B>).
</P><DIV style="width:50% ;  margin-left:10%"><DIV style="width:100%"><P style=" font-family: Helvetica; text-align: justify;font-size: 9pt; line-height:16pt; text-align:center"><I>Figure 2: Enzymatic cocktail degradation of Poly Aromatic hydrocarbons: The case of Anthracene. 
Enzymes involved in the multi-step catalysis of PAHs degradation. E1:naphthalene 1,2-dioxygenase is a homodimer made of E1-1 and E1-2, E2:dihydrodiol dehydrogenase, E3:catechol 2,3-dioxygenase, E4:trans-2-carboxybenzalpyruvate hydratase-aldolase. The green arrow shows a possible cyclic effect in the reaction scheme between E3 and E4. The dotted line shows a possible feed forward step by removing pyruvate from the reaction.</I></P></DIV></DIV><P class="maintext" style=" font-size:9pt; line-height:16pt;"><B>The E3 is able to do an extradiol ring cleavage. Thanks to this step, the PAHs are now less toxic and available for an alkyl removal by E4, trans-2-carboxybenzalpyruvate hydratase-aldolase from <I>Nocardioides sp</I> (<B>BBa_K2198004</B>). If we were to stop at this step of the process, all the products can be handled by chemistry thanks to this alkyl chain. Furthermore, E3 can begin an iterative degradation with E4, if the E4 main product enters in the range of substrate specificity of E3.
The last step of our degradation pathway allows us to remove a ring from the PAHs by producing a pyruvic acid molecule. This product is a common harmless cell intermediate molecule. Moreover, this step produces a smaller PAH if there are more than 2 rings at the beginning. 
This second-generation PAH can reintegrate the pathway at the previous step thanks to a low substrate specificity of E3. 
</B></P><DIV class="subtitle"><P style="font-family: MinionPro, serif;font-size:16pt">B. The bio-filter substructure for the PAHs degrading enzymes</P></DIV><P class="maintext" style=" font-size:9pt; line-height:16pt;"><B>The physical support of our bio-filter structure will be composed of spider silk. The proteins forming spider silk are MaSp1 and MaSp2 from the major spindroin structure of <I>Euprosthenops australis</I>. Among these proteins only one has been gene synthesized and expressed in <I>E. coli</I>: S1, a recombinant form of MaSp1 (<B>BBa_K2198005</B>). The fibers will be fastened to any kind of physical support (plastics, resin, metal, glass, wood, concrete…) through mussel proteins (Mfp2, Mfp3, Mfp4, Mfp5, Mfp6) from <I>Mytilus californianus</I>.</B></P><P class="maintext" style=" font-size:9pt; line-height:16pt">The first part of the structure will be a fusion protein between Mfp5 and S1. Subsequent members of the chain will be assembled as a mix of this fusion and polymers with free S1. S1 is also designed in a form with streptavidin called S2f. Termination of the polymerization process will be ensured by this S2f protein fusion in a random way, yielding a statistical mix of chain lengths. When all components are available by gene synthesis, they will be assembled in polymer forms using the appropriate buffer. A final capping with individually biotinylated-enzymes (E1, E2, E3, E4) will provide the link between the scaffold and our purifying components: the enzymes themselves (See Figure 3).</P><P class="maintext" style="font-size:9pt; line-height:16pt;"><B>The bio-material we have designed is composed of several proteins with distinct roles (Figure 3). On the surface is present the enzymes allowing the degradation of pollutants. Then we have a structural support for the enzymes, the spider silk protein. The spider silk polymer is linked to the enzymes by a streptavidin-biotin complex. Finally, our material can bind to many surfaces thanks to a mussel protein (S1fM1) connected to the silk (S1) by a flexible linker.
</B></P><DIV style="width:50% ;  margin-left:10%"><DIV style="width:100%"><P style=" font-family: Helvetica; text-align: justify;font-size: 9pt; text-align:center"><I>Figure 3: Schematic representation of the Protein assembly of the device æther</I></P></DIV></DIV><DIV class="subtitle"><P style="font-family: MinionPro, serif;font-size:18pt">The genetic constructs for our device</P></DIV><P class="maintext" style=" font-size:9pt; line-height:16pt;">To produce our proteins, we asked Eurofins Genomics to chemically synthesize our DNA sequences. We optimized the coding sequences for expression in <I>Escherichia coli</I> with Geneious software (Ver. 11.0) and Eurofins Genomics, to avoid biobrick prefix/suffix sequences, and phospho-amidite synthesis loopholes. The repetitive nature of some of our sequences made them difficult to produce by synthesis.  Thus, we couldn’t produce S1fM1 and S2f.</P><DIV style="width:50% ;  margin-left:10%"><DIV style="width:100%"><TABLE><TBODY><TR><TD style="font-family: CrimsonTextRoman, serif; font-size: 11pt">Code name</TD><TD style="font-family: CrimsonTextRoman, serif; font-size: 11pt">Protein name</TD><TD style="font-family: CrimsonTextRoman, serif; font-size: 11pt">Function</TD></TR><TR><TD style="font-family: CrimsonTextRoman, serif; font-size: 11pt">E1_1</TD><TD style="font-family: CrimsonTextRoman, serif; font-size: 11pt">naphthalene 1,2-dioxygenase (part 1)</TD><TD style="font-family: CrimsonTextRoman, serif; font-size: 11pt">Enzymatic degradation</TD></TR><TR><TD style="font-family: CrimsonTextRoman, serif; font-size: 11pt">E1_2</TD><TD style="font-family: CrimsonTextRoman, serif; font-size: 11pt">naphthalene 1,2-dioxygenase (part 2)</TD><TD style="font-family: CrimsonTextRoman, serif; font-size: 11pt">Enzymatic degradation</TD></TR><TR><TD style="font-family: CrimsonTextRoman, serif; font-size: 11pt">E2</TD><TD style="font-family: CrimsonTextRoman, serif; font-size: 11pt">dihydrodiol dehydrogenase</TD><TD style="font-family: CrimsonTextRoman, serif; font-size: 11pt">Enzymatic degradation</TD></TR><TR><TD style="font-family: CrimsonTextRoman, serif; font-size: 11pt">E3</TD><TD style="font-family: CrimsonTextRoman, serif; font-size: 11pt">catechol 2,3-dioxygenase </TD><TD style="font-family: CrimsonTextRoman, serif; font-size: 11pt">Enzymatic degradation</TD></TR><TR><TD style="font-family: CrimsonTextRoman, serif; font-size: 11pt">E4</TD><TD style="font-family: CrimsonTextRoman, serif; font-size: 11pt">trans-2-carboxybenzalpyruvate hydratase-aldolase</TD><TD style="font-family: CrimsonTextRoman, serif; font-size: 11pt">Enzymatic degradation</TD></TR><TR><TD style="font-family: CrimsonTextRoman, serif; font-size: 11pt">S1</TD><TD style="font-family: CrimsonTextRoman, serif; font-size: 11pt">Major ampullate Spidroin 1 (MaSp1)</TD><TD style="font-family: CrimsonTextRoman, serif; font-size: 11pt">Silk polymerization</TD></TR><TR><TD style="font-family: CrimsonTextRoman, serif; font-size: 11pt">S1fM1</TD><TD style="font-family: CrimsonTextRoman, serif; font-size: 11pt">Major ampullate Spidroin 1 + Mussel adhesive proteins Foot protein-5 (fp-5)</TD><TD style="font-family: CrimsonTextRoman, serif; font-size: 11pt">Surface fixation</TD></TR><TR><TD style="font-family: CrimsonTextRoman, serif; font-size: 11pt">S2f</TD><TD style="font-family: CrimsonTextRoman, serif; font-size: 11pt">Major ampullate Spidroin 1 + Streptavidin </TD><TD style="font-family: CrimsonTextRoman, serif; font-size: 11pt">Enzymes fixation</TD></TR></TBODY></TABLE><P style=" font-family: Helvetica; text-align: justify;font-size: 9pt; text-align:center"><I>Table 1: List of genetic constructions</I></P></DIV></DIV><DIV style="width:50% ;  margin-left:10%"><DIV style="width:100%"><P style=" font-family: Helvetica; text-align: justify;font-size: 9pt"><I>Figure 4 : Schematic representation of DNA sequence constructs</I></P></DIV></DIV><DIV class="subtitle"><P style="font-family: MinionPro, serif;font-size:18pt">The protein production for our device</P></DIV><P class="maintext" style=" font-size:9pt; line-height:16pt;">For the production of proteins we inserted in our sequences the iGEM suffixes and prefixes, the ribosome binding sites (RBS) and the T7 terminator, and the codons START and STOP. 
The vector plasmid pET-32.a(+) used, contains a strong T7 promoter for protein production. In addition, we added a His-Tag followed by a TEV site in the sequence. This His-Tag will facilitate the purification of our proteins in a Ni-NTA chromatographic column. After purification, the His-Tag will be removed at the TEV site by a protease. (see Figure 4)
</P><DIV style="width:50% ;  margin-left:10%"><DIV style="width:100%"><P style=" font-family: Helvetica; text-align: center;font-size: 9pt; text-align:center"><I>Figure 5: Pet32a protein expression construct</I></P></DIV></DIV><P class="maintext" style=" font-size:9pt; line-height:16pt;">As soon as we received our sequences in a Eurofins plasmid, we performed a transformation in DH5α competent cells. We made liquid cultures of different colonies then extracted the plasmids with an Qiagen Midiprep Kit. We also performed midipreps with the plasmid vector pET-32a(+) and the iGEM plasmid vector pSB1C3. We carried out the digestion of the inserts and the vectors, and after migration on agarose gel and gel extraction, we isolated the inserts and both vectors: pET-32A(+) and pSB1C3. After a step of dephosphorylation of the vectors we carried out the ligation of the inserts with both vector plasmids and then directly transformed these new plasmids in competent cells DH5α (for storage) or BL21DE3 (for protein expression). (see Figure 6)</P><DIV style="width:50% ;  margin-left:10%"><DIV style="width:100%"><P style=" font-family: Helvetica; text-align: justify;font-size: 9pt; text-align:center"><I>Figure 6: Scheme for the cloning and expression strategy for the protein components of our device.</I></P></DIV></DIV><P class="maintext" style=" font-size:9pt; line-height:16pt;">The plasmids pSB1C3 containing the inserts were then placed in liquid cultures, extracted, sequenced and finally sent to iGEM. The plasmids pET-32a(+) containing the inserts were also placed in liquid cultures and then protein expression was induced by adding IPTG. Once the production of the proteins was completed, the cells were lysed by sonication.</P><DIV class="subtitle"><P style="font-family: MinionPro, serif;font-size:18pt">Degradation of Anthracene by the Aether enzyme cocktail</P></DIV><P class="maintext" style=" font-size:9pt line-height:16pt;"><B>To demonstrate the activity of the enzyme cocktail mix in its ability to degrade PAHs, we used anthracene as substrate.  We devised a spectrophotometric assay based on the absorbance at 254 nm of the substrate. 
Cells were induced to produce the enzymes E1 (E1-1, E1-2), E2, E3, and E4. After lysis, cells extracts were used as the source of each enzyme component. A control was set with induced cell extract transformed with the empty pET32a expression vector. The degradation of anthracene over time was effective only with the enzyme mix. Therefore, we have shown that our Aether cocktail mix is able to degrade PAHs. This assay can be potentially extended to other compounds belonging to the same family of compounds, such as benzo[a]pyrene.
</B></P><B><A href="https://static.igem.org/mediawiki/2017/d/de/T--Pasteur_Paris--Annexe_1.pdf" target="_blank" style="color:black">Annexe 1</A></B><B><A href="https://static.igem.org/mediawiki/2017/b/b6/T--Pasteur_Paris--Annexe_2.pdf" target="_blank" style="color:black">Annexe 2</A></B><P class="maintextsuite">REF :
</P><P class="maintextsuite">[1] Journal of Hazardous Materials 169 (2009) 1–15; Biodegradation aspects of Polycyclic Aromatic Hydrocarbons (PAHs): A.K. Haritash∗, C.P. Kaushik; Department of Environmental Science &amp; Engineering, Guru Jambheshwar University of Science &amp; Technology, Hisar, Haryana, India. 
</P><P class="maintextsuite">[2] J Bacteriol. 2000 Mar;182(6):1641-9. Substrate specificity of naphthalene dioxygenase: effect of specific amino acids at the active site of the enzyme. Parales RE1, Lee K, Resnick SM, Jiang H, Lessner DJ, Gibson DT.
</P><P class="maintextsuite">[3] J Bacteriol. 1998 May; 180(9): 2522–2530. A Gene Cluster Encoding Steps in Conversion of Naphthalene to Gentisate in Pseudomonas sp. Strain U2 Sergio L. Fuenmayor,1 Mark Wild,2 Alastair L. Boyes,2 and Peter A. Williams1.
</P><P class="maintextsuite">[4] Int J Biol Macromol. 2017 Aug 25. pii: S0141-8130(17)32824-6. doi: 10.1016/j.ijbiomac.2017.08.113. [Epub ahead of print] Isolation and characterization of three novel catechol 2,3-dioxygenase from three novel haloalkaliphilic BTEX-degrading Pseudomonas strains. Hassan HA1, Aly AA2.
</P><P class="maintextsuite">[5] J Bacteriol. 1998 Feb; 180(4): 945–949. PMCID: PMC106976 - Biochemical and Genetic Characterization of trans-2′-Carboxybenzalpyruvate Hydratase-Aldolase from a Phenanthrene-Degrading Nocardioides Strain - Tokuro Iwabuchi† and Shigeaki Harayama.
</P><P class="maintextsuite">[6] “Geneious version (the version you are using) (https://www.geneious.com, Kearse et al., 2012)” Kearse, M., Moir, R., Wilson, A., Stones-Havas, S., Cheung, M., Sturrock, S., Buxton, S., Cooper, A., Markowitz, S., Duran, C., Thierer, T., Ashton, B., Mentjies, P., &amp; Drummond, A. (2012). Geneious Basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data.Bioinformatics, 28(12), 1647-1649.
</P><P class="maintextsuite">[7] Biomacromolecules 2007, 8, 1695-1701 // Macroscopic Fibers Self-Assembled from Recombinant Miniature Spider Silk Proteins; Margareta Stark, Stefan Grip, Anna Rising, My Hedhammar, Wilhelm Engström, Go ̈ran Hjälm, and Jan Johansson*, Department of Anatomy, Physiology, and Biochemistry, The Biomedical Centre, Swedish University of Agricultural Sciences, SE-751 23 Uppsala, Sweden, and Department of Biomedical Sciences and Veterinary Public Health, Swedish University of Agricultural Sciences, SE-750 07 Uppsala, Sweden Received January 16, 2007; Revised Manuscript Received February 20, 2007.
</P></DIV><UL class="com"><LI><A href="https://www.instagram.com/igem.pasteur2017/" target="_blank"><I id="logo" class="material-icons">photo_camera</I></A></LI><LI><A href="mailto:igem.pasteur@gmail.com " target="_blank"><I id="logo" class="material-icons">email</I></A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>