<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Linkoping_Sweden_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Linkoping Sweden/Protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="content-container"><DIV class="closed-menu"><DIV id="mobile-menu">
            LiU iGEM
            </DIV><DIV class="menu-category"><A href="/Team:Linkoping_Sweden/Team">Team</A><DIV class="menu-items"><A href="/Team:Linkoping_Sweden/Collaborations">Collaborations</A><A href="/Team:Linkoping_Sweden/Attributions">Attributions</A></DIV></DIV><DIV class="menu-category"><SPAN>Project</SPAN><DIV class="menu-items"><A href="/Team:Linkoping_Sweden/Design">Design</A><A href="/Team:Linkoping_Sweden/Experiments">Experiments</A><A href="/Team:Linkoping_Sweden/Protocols">Protocols</A><A href="/Team:Linkoping_Sweden/Results">Results</A><A href="/Team:Linkoping_Sweden/Safety">Safety</A><A href="/Team:Linkoping_Sweden/Notebook">Notebook</A><A href="/Team:Linkoping_Sweden/Economic_Viability">Economic Viability</A></DIV></DIV><DIV class="menu-category"><A href="/Team:Linkoping_Sweden/Description">Project Description</A><DIV class="menu-items"><A href="/Team:Linkoping_Sweden/Description/CRISPR/Cas9-system">CRISPR/Cas9 System</A><A href="/Team:Linkoping_Sweden/Description/Gibson_assembly">Gibson Assembly</A><A href="/Team:Linkoping_Sweden/Description/Chlamydomonas_reinhardtii">Chlamydomonas Reinhardtii</A></DIV></DIV><DIV class="menu-category"><A href="/Team:Linkoping_Sweden/Parts">Parts</A></DIV><DIV class="menu-category"><A href="/Team:Linkoping_Sweden/Human_Practices">Human Practices</A><DIV class="menu-items"><A href="/Team:Linkoping_Sweden/HP/Silver">Silver</A><A href="/Team:Linkoping_Sweden/HP/Gold">Gold</A></DIV></DIV></DIV><DIV class="content"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Protocols"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Protocols</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Medium_and_solutions"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Medium and solutions</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#Hutner.E2.80.99s_Trace_Elements"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">Hutner’s Trace Elements</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#LB_medium_.281_L.29"><SPAN class="tocnumber">1.1.2</SPAN><SPAN class="toctext">LB medium (1 L)</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#LB_agar-plates"><SPAN class="tocnumber">1.1.3</SPAN><SPAN class="toctext">LB agar-plates</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Soc_medium_.281_L.29"><SPAN class="tocnumber">1.1.4</SPAN><SPAN class="toctext">Soc medium (1 L)</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#TAP.2FTris_medium_.281_L.29"><SPAN class="tocnumber">1.1.5</SPAN><SPAN class="toctext">TAP/Tris medium (1 L)</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-8"><A href="#Protocols_2"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Protocols</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Agarose_gelelectrophoresis"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Agarose gelelectrophoresis</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Cultivation_of_algae"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Cultivation of algae</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Cultivation_of_algae_in_the_dark"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">Cultivation of algae in the dark</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Digestion_and_Ligation"><SPAN class="tocnumber">1.2.4</SPAN><SPAN class="toctext">Digestion and Ligation</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#E._Coli_calcium_chloride_competent_cells"><SPAN class="tocnumber">1.2.5</SPAN><SPAN class="toctext">E. Coli calcium chloride competent cells</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Gibson_Assembly"><SPAN class="tocnumber">1.2.6</SPAN><SPAN class="toctext">Gibson Assembly</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Glycerol_stock"><SPAN class="tocnumber">1.2.7</SPAN><SPAN class="toctext">Glycerol stock</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#PCR"><SPAN class="tocnumber">1.2.8</SPAN><SPAN class="toctext">PCR</SPAN></A></LI><LI class="toclevel-3 tocsection-17"><A href="#Plasmid_preparation"><SPAN class="tocnumber">1.2.9</SPAN><SPAN class="toctext">Plasmid preparation</SPAN></A></LI><LI class="toclevel-3 tocsection-18"><A href="#Transformation"><SPAN class="tocnumber">1.2.10</SPAN><SPAN class="toctext">Transformation</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#Transformation_of_Chlamydomonas_Reinhardtii_by_electroporation"><SPAN class="tocnumber">1.2.11</SPAN><SPAN class="toctext">Transformation of Chlamydomonas Reinhardtii by electroporation</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#References"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">References</SPAN></A></LI></DIV><H1><SPAN class="mw-headline" id="Protocols">Protocols</SPAN></H1><H2><SPAN class="mw-headline" id="Medium_and_solutions">Medium and solutions</SPAN></H2><H3><SPAN class="mw-headline" id="Hutner.E2.80.99s_Trace_Elements">Hutner’s Trace Elements</SPAN></H3><P>For 1 liter final mix, dissolve each compound in the volume of water indicated. The EDTA should be dissolved in boiling water, and the FeSO4 should be prepared last to avoid oxidation.
</P><P><I><B>Material</B></I></P><UL><LI> 50 g EDTA disodium salt dissolved in 250 ml water</LI><LI> 22 g ZnSO4 ∙ 7 H2O dissolved in 100 ml water</LI><LI> 11.4 g H3BO3	dissolved in 200 ml water</LI><LI> 5.06 g MnCl2 ∙ 4 H2O dissolved in 50 ml water</LI><LI> 1.61 g CoCl2 ∙ 6 H2O dissolved in 50 ml water</LI><LI> 1.57 g CuSO4 ∙ 5 H2O	dissolved in 50 ml water</LI><LI> 1.10 g (NH4)6Mo7O24 ∙ 4 H2O dissolved in 50 ml water</LI><LI> 4.99 g FeSO4 ∙ 7 H2O dissolved in 50 ml water</LI></UL><P><I><B>Procedure</B></I></P><OL><LI> Mix all solutions except EDTA. Bring the solution to boil and then add EDTA solution. The mixture should turn green. </LI><LI> When everything is dissolved, cool to 70 °C. </LI><LI> Keep the temperature at 70 °C, add 85 ml hot 20% KOH solution (20 grams / 100 ml final volume). </LI><LI> Bring the final solution to 1 liter total volume. It should be clear green initially. </LI><LI> Stopper the flask with a cotton plug and let it stand for 1-2 weeks, shaking it once a day. <I><B>Note:</B></I> If no precipitate forms, the solution is still usable. However, you might want to check the pH in this case and adjust it to around 7.0 using either KOH or HCl as needed, do NOT use NaOH to adjust the pH. .</LI><LI> The solution should eventually turn purple and leave a rust-brown precipitate, which can be removed by filtering through two layers of Whatman#1 filter paper, repeating the filtration if necessary until the solution is clear. </LI><LI> Store refrigerated or frozen convenient aliquots (1).</LI></OL><H3><SPAN class="mw-headline" id="LB_medium_.281_L.29">LB medium (1 L)</SPAN></H3><P><I><B>Material</B></I></P><UL><LI> 10 g peptone</LI><LI> 5 g yeast extract</LI><LI> 5 g NaCl</LI><LI> 1 M NaOH</LI><LI> Antibiotic if needed</LI><LI> dH2O</LI></UL><P><I><B>Procedure</B></I></P><OL><LI> Dissolve 10 g peptone, 5 g yeast extract, and 5 g NaCl in 950 mL deionized water.</LI><LI> Adjust the pH of the medium to 7.0 using 1M NaOH and bring volume up to 1 liter.</LI><LI> Autoclave on liquid cycle for 20 min at 15 psi. Allow solution to cool to 55°C, and add antibiotic if needed (34µg/ ml of Amp or Kan). </LI><LI> Store at room temperature or +4°C (2).</LI></OL><H3><SPAN class="mw-headline" id="LB_agar-plates">LB agar-plates</SPAN></H3><P><I><B>Material</B></I></P><UL><LI> LB medium </LI><LI> 17 g/l Agar</LI></UL><P><I><B>Procedure</B></I></P><OL><LI> Prepare LB medium as above, but add 17 g/l agar before autoclaving.</LI><LI> After autoclaving, cool to approx. 55°C, add antibiotic (if needed, the concentration of antibiotic to LB should be 1:1000), and pour into petridishes.</LI><LI> Let it cool, then invert and store at +4°C in the dark (2).</LI></OL><H3><SPAN class="mw-headline" id="Soc_medium_.281_L.29">Soc medium (1 L)</SPAN></H3><P><I><B>Material</B></I></P><UL><LI> 5 g yeast extract</LI><LI> 20 g peptone</LI><LI> 0.584 g NaCl</LI><LI> 0.186 g KCl</LI><LI> 2.4 g MgSO4</LI></UL><P><I><B>Procedure</B></I></P><OL><LI> Bring the final solution to 1 l using distilled water. </LI><LI> Adjust to pH 7.5 prior to use. This requires approximately 25 ml of 1M NaOH per liter. </LI><LI> Make SOB in to SOC: Cool medium (1 l of SOB) to less than 50°C, then add 20 ml filter sterilized 20% glucose solution</LI></OL><P><I><B>Glucose (20 %)</B></I></P><OL><LI> Add 20 g Glucose</LI><LI> Bring the final solution to 100 ml using dH2O (3).</LI></OL><H3><SPAN class="mw-headline" id="TAP.2FTris_medium_.281_L.29">TAP/Tris medium (1 L)</SPAN></H3><P><I><B>Material</B></I></P><UL><LI> 20 ml 1 M Tris base</LI><LI> 1.0 ml Phosphate Buffer II
</LI><LI> Phosphate Buffer II (for 100 ml)
</LI><LI> 10.8 g K2HPO4</LI><LI> 5.6 g KH2PO4</LI></UL><LI> 10.0 ml Solution A
</LI><LI> Solution A (for 500 ml) 
</LI><LI> 20 g NH4Cl	</LI><LI> 5 g MgSO4 ∙ 7 H2O</LI><LI> 2.5 g CaCl2 ∙ 2 H2O</LI><LI> 1.0 ml Hutner’s trace elements</LI><LI> 1.0 ml Glacial acetic acid</LI><P><I><B>Procedure</B></I></P><OL><LI> Mix all the solutions together</LI><LI> Adjust the final pH to 7.0 <I><B>Note:</B></I> For Tris-minimal medium omit the acetic acid and titrate the final solution to pH 7.0 with HCl (4).</LI></OL><H2><SPAN class="mw-headline" id="Protocols_2">Protocols</SPAN></H2><H3><SPAN class="mw-headline" id="Agarose_gelelectrophoresis">Agarose gelelectrophoresis</SPAN></H3><P><I><B>Material</B></I></P><UL><LI> Agarose</LI><LI> 10x TBE buffer</LI><LI> Loading dye </LI><LI> Molecular weight marker</LI></UL><P><I><B>Procedure</B></I></P><P><B>Agarose gel</B></P><OL><LI> Dilute 10 ml 10x TBE buffer with water to an final volume of 100 ml.</LI><LI> Add 0,5-2,0 g agarose depending on the size of your fragment.</LI><LI> Heat the agarose in microwave until completely liquid. Let mixture cool until approximately 60℃ and pour agarose gel. Let cool for at least 30 minutes.</LI></OL><P><B>DNA separation</B></P><OL><LI> Mix 4 μl loading dye with 5 μl DNA and 11 μl H2O (and conntrol without DNA).</LI><LI> Add agarose gel in electrophoresis apparature and fill with 1x TBE buffer to cover the gel.</LI><LI> Add 5 μl premixed molecular weight marker and 20 μl of DNA mixture or control to separate wells. </LI><LI> Separate the DNA fragments by applying 150 Volts </LI><LI> Stain DNA fragments with ethidium bromide (15 ul in approximately 400 1x TBE) for 20 minutes.</LI><LI> Wash in dH2O for 5 minutes</LI><LI> Illuminate with 300-360 nm, photograph gel and estimate size or amount of DNA (5).</LI></OL><H3><SPAN class="mw-headline" id="Cultivation_of_algae">Cultivation of algae</SPAN></H3><P><I><B>Material</B></I></P><UL><LI> 100 ml TAP – medium </LI><LI> 1ml Alg suspension </LI></UL><P><I><B>Procedure</B></I></P><OL><LI> Mix 100 ml TAP – medium with 1 ml alg suspension</LI><LI> Incubate the culture on a shaker (or pump if available) (100 - 150 rpm, LED light 14 hours/day, temperature constant 22 °C)</LI><LI> Measure the growth using a spectrometer (750 nm) every day about the same time. Incubate until the growth curve reaches 0.5 (6).</LI></OL><H3><SPAN class="mw-headline" id="Cultivation_of_algae_in_the_dark">Cultivation of algae in the dark</SPAN></H3><P><I><B>Material</B></I></P><UL><LI> 100 ml TAP – medium </LI><LI> 1 ml Alg suspension </LI></UL><P><I><B>Procedure</B></I></P><OL><LI> Make a new TAP - medium with 3.4 g/l acetate instead of 3.2 ml/l acetic acid</LI><LI> Mix 100 ml of the new TAP – medium with 1 ml alg suspension</LI><LI> Incubate the culture on a shaker (or pump if available) (100 - 150 rpm, completely in the dark, temperature constant 22 °C)</LI><LI> Measure the growth using a spectrometer (750 nm) every day about the same time. Incubate until the growth curve reaches 0.5 (6).</LI></OL><H3><SPAN class="mw-headline" id="Digestion_and_Ligation">Digestion and Ligation</SPAN></H3><P><B>Digestion</B></P><P><I><B>Material</B></I></P><UL><LI> 1 μl Restriction enzyme         		</LI><LI> DNA </LI><LI> 1 μl 10 X CutSmart buffer    </LI><LI> Autoclaved H2O To a final volume of 10 μl </LI></UL><P><I><B>Procedure</B></I></P><OL><LI> Mix restriction enzyme, DNA, Cutsmart buffer and autoclaved H2O.</LI><LI> Incubate the restriction digest at 37 °C for 1 h</LI><LI> Incubate at 80 °C for 20 min for heat inactivation (7)</LI></OL><P><B>Ligation</B></P><P><I><B>Material</B></I></P><UL><LI> 50 ng Linearized Vector                                                               	</LI><LI> 3X molar excess Gene Fragment                                     	</LI><LI> 10 μl 2X Quick Ligase Buffer 	                  	</LI><LI> 1 μl Quick DNA Ligase                              	</LI><LI> Autoclaved H2O To a final volume of 21 μl</LI></UL><P><I><B>Procedure</B></I></P><OL><LI> Combine 50 ng of vector with a 3-fold molar excess of insert. Use NeBioCalculator to calculate molar ratios. Adjust volume to 10 μl with autoclaved H2O.</LI><LI> Add 10 μl of 2X Quick Ligation Buffer and mix.</LI><LI> Add 1 μl of Quick T4 DNA Ligase and mix thoroughly.</LI><LI> Centrifuge briefly and incubate at room temperature (25 °C) for 5 min.</LI><LI> Chill on ice, then transform or store at -20 °C. </LI></OL><P><I><B>Note: </B></I>Do not heat inactivate the ligase. Heat inactivation dramatically reduces the transformation efficiency (8).
</P><H3><SPAN class="mw-headline" id="E._Coli_calcium_chloride_competent_cells">E. Coli calcium chloride competent cells</SPAN></H3><P><I><B>Procedure</B></I></P><OL><LI> Inoculate a single colony into 5ml LB in a 50ml Falcon tube. Grow overnight at 37°C.</LI><LI> Use 1ml to inoculate 100ml of LB in 250ml bottle the next morning.</LI><LI> Shake at 37°C for 1.5-3hrs. OR Inoculate a single colony into 25ml LB in a 250 ml bottle in the morning. and then shake at 37°C for 4-6 hrs.</LI><LI> Put the cells on ice for 10 mins (keep cold form now on).</LI><LI> Collect the cells by centrifugation for 3 mins at 6000 rpm. </LI><LI> Decant the supernatant and gently resuspend on 10 ml cold 0.1M CaCl (cells are susceptible to mechanical disruption, so treat them nicely).</LI><LI> Incubate on ice for 20 mins</LI><LI> Centrifuge for 3 minutes at 6000 rpm. </LI><LI> Discard supernatant and gently resuspend on 5ml cold 0.1 M CaCl2/15% Glycerol</LI><LI> Dispense in microtubes (300μl/tube). Freeze in - 80°C. </LI></OL><P><I><B>Transformation of Ca2+ competent cells</B></I></P><OL><LI> Put 1μl of circular plasmid or all of a ligation reaction of plasmid DNA in a microtube. Gently add ~100μl of competent cells. Do NO DNA control tube with cells and no DNA.</LI><LI> Incubate for 30 mins on ice.</LI><LI> Heat shock for 2 mins at 42°C. Put back on ice.</LI><LI> Add 900 μl of LB to tubes. Incubate at 37°C for 30 mins.</LI><LI> Plate 100-1000 μL of the cells in LB-Amp or LB-Carb (100μg/ml) plates. Plate 100 μl of the NO DNA control in a blood plate (to check for quality of cells). Grow overnight. You can save the rest in -80°C with 15% of Gly in case there would not be any colonies.</LI><LI> If you need a lot of colonies or the ligation is of low efficiency, centrifuge the transformation for 1 min at 8000 rpm, discard 900 μL of supernatant, resuspend on the 100 μL left and plate the whole lot. </LI></OL><P><I><B>CaCl2/15% Glycerol – solutions for competent cells</B></I><I><B>Material</B></I></P><UL><LI> 0.1 M CaCl2</LI><LI> 15 % glycerol solution</LI></UL><P><I><B>Procedure</B></I></P><OL><LI> Mix 1 ml 1 M CaCl2 solution with 1.5 ml 100 % glycerol and 7.5 ml dH2O (9).</LI></OL><H3><SPAN class="mw-headline" id="Gibson_Assembly">Gibson Assembly</SPAN></H3><P>New England Biolab’s NEBuilder HiFi DNA Assembly Reaction Protocol was used for the assembly. Vector:insert ratio was calculated with the NEBio Calculator.
</P><P><I><B>Material</B></I></P><UL><LI><B>4-6 Fragment assembly</B></LI><LI> DNA Ratio: Vector:insert = 1:1</LI><LI> DNA Fragments: 0.2–0.5 pmols** (X μl)            </LI><LI> NEBuilder HiFi DNA Assembly Master Mix: 10 μl  </LI><LI> dH2O: 10-X μl</LI><LI> Total Volume: 20 μl</LI></UL><UL><LI><B>Positive control</B></LI><LI> DNA Fragments: 10 μl</LI><LI> NEBuilder HiFi DNA Assembly Master Mix: 10 μl</LI><LI> dH2O: 0 μl</LI><LI> Total Volume: 20 μl</LI></UL><P><I><B>Procedure</B></I></P><OL><LI> Add all components to an eppendorf tube</LI><LI> Incubate samples at 50°C for 60 minutes </LI><LI> Store samples on ice and continue with transformation (or store at –20°C)</LI></OL><P><I><B>Transformation</B></I></P><OL><LI> Add 2 μl of assembled product to 50 ul NEB competent cells in an eppendorf tube </LI><LI> Mix gently by pipetting up and down. Do not vortex. </LI><LI> Incubate on ice for 30 minutes. </LI><LI> Heat shock at 42°C for 30 seconds.* Do not mix.</LI><LI> Thaw  on ice for 2 minutes.</LI><LI> Add 950 μl of room temperature SOC media</LI><LI> Incubate  at 37°C for 60 minutes. Shake vigorously (250 rpm) or rotate.</LI><LI> Spread 100 μl of the cells onto preheated (37°C.) LB-plates with CHL.. Use Amp plates for positive control sample.</LI><LI> Incubate plates overnight at 37°C (10).</LI></OL><H3><SPAN class="mw-headline" id="Glycerol_stock">Glycerol stock</SPAN></H3><P><I><B>Procedure</B></I></P><OL><LI> After you have bacterial growth, add 500 μL of the overnight culture to 500μL of 50% glycerol in a 2 mL snap top tube and gently mix. <I><B>Note:</B></I> Make the 50% glycerol solution by diluting 100% glycerol in dH20.</LI><LI> Freeze the glycerol stock tube at -80°C. The stock is now stable for years, as long as it is kept at -80°C. Subsequent freeze and thaw cycles reduce shelf life (11). </LI></OL><H3><SPAN class="mw-headline" id="PCR">PCR</SPAN></H3><P><I><B>Material</B></I></P><UL><LI> Standard Taq Reaction Buffer (10X)	5 μl</LI><LI> Deoxynucleotide Solution Mix		1 μl</LI><LI> Forward primer (10 μM stock solution)	1 μl</LI><LI> Reverse primer (10 μM stock solution)	1 μl</LI><LI> DNA, plasmid				1 pg- 1ng</LI><LI> Taq DNA polymeras (diluted)*		1 μl</LI><LI> Nuclease free water			Bring reaction voulme to 50 μl</LI></UL><P><I><B>Procedure</B></I></P><OL><LI> *Dilute stock Taq DNA polymeras 1:4 in 1X Standard Taq Reaction Buffer</LI><LI> Prepare reaction mixture. Add Taq DNA polymeras just before starting PCR-reaction.</LI><LI> PCR-reaction:</LI></OL><DL><DD> Initial denaturation at 98 °C for 30 seconds</DD><DD> 25 cycles		
</DD><DD> Denaturation at 95 °C	for 15-30 seconds</DD><DD> Annealing at 45-68 °C	for 15-60 seconds</DD><DD>  Extension at 68 °C 1 minute/kb</DD></DL><DL><DD> Final extension at 72 °C for 2 minutes</DD><DD> Hold at 4 °C</DD></DL><P>New England Biolab’s TM calculator can be used to estimate annealing temperature: <A rel="nofollow" class="external free" href="http://tmcalculator.neb.com/#!/(12)">http://tmcalculator.neb.com/#!/(12)</A></P><H3><SPAN class="mw-headline" id="Plasmid_preparation">Plasmid preparation</SPAN></H3><P><I><B>Procedure</B></I></P><OL><LI> Pellet 1–5 ml of an overnight recombinant E. coli culture by centrifugation. Transfer the appropriate volume of the recombinant E. coli culture to a microcentrifuge tube and pellet cells at ≥12,000 3 g for 1 minute. Discard the supernatant.</LI><LI> Resuspend cells: Completely resuspend the bacterial pellet with 200 µl of the Resuspension Solution. Vortex or pipette up and down to thoroughly resuspend the cells until homogeneous.</LI><LI> Lyse cells: Lyse the resuspended cells by adding 200 µl of the Lysis Solution. Immediately mix the contents by gentle inversion (6–8 times) until the mixture becomes clear and viscous. Do not vortex. Harsh mixing will shear genomic DNA, resulting in chromosomal DNA contamination in the final recovered plasmid DNA. Do not allow the lysis reaction to exceed 5 minutes. </LI><LI> Neutralize: Precipitate the cell debris by adding 350 µl of the Neutralization/Binding Solution. Gently invert the tube 4–6 times. Pellet the cell debris by centrifuging at ≥12,000 x g or maximum speed for 10 minutes. If the supernatant contains a large amount of floating particulates after centrifugation, recentrifuge the supernatant before proceeding to step 6. </LI><LI> Prepare Column: Insert a GenElute Miniprep Binding Column into a provided microcentrifuge tube, if not already assembled. Add 500 µl of the Column Preparation Solution to each miniprep column and centrifuge at ≥12,000 x g for 30 seconds to 1 minute. Discard the flow-through liquid.</LI><LI> Load cleared lysate: Transfer the cleared lysate from step 3 to the column prepared in step 4 and centrifuge at ≥12,000 x g for 30 seconds to 1 minute. Discard the flow-through liquid. </LI><LI> Wash column: Add 750 µl of the diluted Wash Solution to the column. Centrifuge at ≥12,000 x g for 30 seconds to 1 minute. Discard the flow-through liquid and centrifuge again at maximum speed for 1 to 2 minutes without any additional Wash Solution. </LI><LI> Elute DNA: Transfer the column to a fresh collection tube. Add 50 µl water to the column. Centrifuge at ≥12,000 x g for 1 minute. The DNA is now present in the eluate and is ready for immediate use or storage at –20 °C (13). </LI></OL><H3><SPAN class="mw-headline" id="Transformation">Transformation</SPAN></H3><P><I><B>Material</B></I></P><UL><LI> E.coli XL1-blue supercompetent cells	20 μl</LI><LI> DNA					1 μl (5 ng)</LI><LI> Soc medium				100 μl</LI><LI> LB-agarplates with Chloramphenicol (CHL) resistence</LI></UL><P><I><B>Procedure</B></I></P><OL><LI> Thaw the E.coli XL1-blue supercompetent cells on ice</LI><LI> Portion in 20 μl and add approximately 1 μl (5 ng) DNA </LI><LI> Mix gently with pipette tip and incubate on ice for 30 minutes</LI><LI> Heat shock cells in heating block 45 seconds at 42℃, then place on ice for 2 minutes</LI><LI> Add 100 μl preheated (42℃) Soc medium and incubate the transformation mixture for 2 h at 37℃</LI><LI> Spread all of the transformation mixture (121 μl) on preheated agarplates (37℃) with Chloramphenicol resistance</LI><LI> Incubated over night (at least 16 hours). Count the number of colonies </LI><LI> Store in cold room, approximately 4℃ (14)</LI></OL><P><I><B>Note:</B></I> Control plate is done without DNA in order to control the antibiotic resistance works.
</P><H3><SPAN class="mw-headline" id="Transformation_of_Chlamydomonas_Reinhardtii_by_electroporation">Transformation of Chlamydomonas Reinhardtii by electroporation</SPAN></H3><P>All steps, except the growing of Chlamydomonas, will be proceeded at room temperature.
</P><UL><LI> Electroporator: Bio Rad Micropulser</LI><LI> Cuvettes: 4 mm gap</LI><LI> OD750nm: 0.3 – 0.5 </LI><LI> Applied pulses: 3</LI><LI> Sample volume: 250 μl</LI><LI> Voltage: 0.80 kV</LI></UL><P><I><B>Procedure</B></I></P><P><B>Day -x:</B></P><OL><LI> Start 6 ml Chlamydomonas r. culture in TAP media</LI><LI> Grow under light and agitation of 120 rpm until OD750nm: ≈ 0.6</LI></OL><P><B>Day 0:</B></P><OL><LI> Measure OD750nm</LI><LI> Dilute culture to OD750nm: 0.09 using TAP media (45 ml is needed for the next day)</LI><LI> Grow under light and agitation for 22-24 h (OD750nm should reach 0.3-0.5)</LI></OL><P><B>Day 1:</B></P><OL><LI> Measure OD750nm and note it above</LI><LI> Centrifuge 45 ml culture at 2000 x g for 10 min in 50 ml conical tubes</LI><LI> Discard supernatant completely </LI><LI> Dissolve pellet gently in 750 μl TAP -40mM sucrose media </LI><LI> Transfer aliquot of 250 μl into 4 mm cuvette </LI><LI> Add 2 ug of linearized vector </LI><LI> Incubate for 2-4 min </LI><LI> Pulse 3 times at 0.80 kV </LI><LI> Add aliquots of 125 μl into 5 ml TAP-40 mM sucrose media (6 well)</LI><LI> Rinse cuvettes with media from the plates</LI><LI> Grow under light and agitation for max. 24 h</LI></OL><P><B>Day 2: </B></P><OL><LI> Centrifuge 10 ml culture at 2000 x g for 10 min in 25 ml conical tubes</LI><LI> Resuspend cells in 300 μl TAP-40 mM sucrose media</LI><LI> Split in half and add each 150 μl culture to 5 ml TAP with 10 ug/ μl Hygromycin</LI><LI> Grow for 5-8 days until culture turns green</LI><LI> Plate 50 μl on 1 % TAP agar plates, seal with Parafilm and grow under light until colonies are clearly visible</LI><LI> Otherwise add 50 μl culture to fluid TAP media with increasing Hygromycin concentrations to screen for high expressing lines due to positional effects.</LI></OL><P><I><B>Note:</B></I> Too much salts will decrease the electrical resistance of the sample. This can lead to a decrease of the applied voltage during the Electroporation. The Biorad Micropulser is able to show the actual applied Voltage. Digest at least 2 ug Plasmid-DNA per reaction. After inactivation of RE the digest can directly be used for the transformation. After the electroporation be very gently with the cells (esp. pipetting). When plating on agar plates use plating spatulas and try to spread the cultures on the middle of plate for best results. Otherwise most colonies will grow at the edge of the plates (15-17).
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P>1. 	Hutner et.al. Hutner’s trace Elements. Proc Am Philos Soc. 1950;94, 152–70. 
</P><P>2. 	Medina Lab. LB medium and LB agar-plates [Internet]. Medina Lab. 2016 [cited 2016 Jun 10]. Available from: <A rel="nofollow" class="external free" href="http://www.medinalab.org/portal/protocols/48-recipes/81-lb-medium-and-lb-agar-plates">http://www.medinalab.org/portal/protocols/48-recipes/81-lb-medium-and-lb-agar-plates</A></P><P>3. 	SOC [Internet]. <A rel="nofollow" class="external free" href="http://parts.igem.org/SOC">http://parts.igem.org/SOC</A>. [cited 2016 Jun 10]. Available from: <A rel="nofollow" class="external free" href="http://parts.igem.org/SOC">http://parts.igem.org/SOC</A></P><P>4. 	Gorman DS, Levine RP. Cytochrome f and plastocyanin: their sequence in the photosynthetic electron transport chain of Chlamydomonas reinhardi. Proc Natl Acad Sci U S A 
[Internet]. National Academy of Sciences; 1965 Dec [cited 2016 Oct 16];54(6):1665–9. Available from: <A rel="nofollow" class="external free" href="http://www.ncbi.nlm.nih.gov/pubmed/4379719">http://www.ncbi.nlm.nih.gov/pubmed/4379719</A></P><P>5. 	Addgene. Addgene: Protocol - How to Run an Agarose Gel [Internet]. Addgene. [cited 2016 Jun 30]. Available from: <A rel="nofollow" class="external free" href="https://www.addgene.org/plasmid-protocols/gel-electrophoresis/">https://www.addgene.org/plasmid-protocols/gel-electrophoresis/</A></P><P>6. 	Edqvist J (Linköping U. Cultivation of Algae. Unpubl Mater. Linköping; 2016; 
</P><P>7. 	New England Biolabs. Single-temperature Double Digest [Internet]. New England Biolabs (NEB). 2014 [cited 2016 Jun 30]. Available from: <A rel="nofollow" class="external free" href="https://www.protocols.io/view/Single-temperature-Double-Digest-imsuj5">https://www.protocols.io/view/Single-temperature-Double-Digest-imsuj5</A></P><P>8. 	New England Biolabs. Quick Ligation Protocol (M2200) | NEB [Internet]. [cited 2016 Jun 30]. Available from: <A rel="nofollow" class="external free" href="https://www.neb.com/protocols/1/01/01/quick-ligation-protocol">https://www.neb.com/protocols/1/01/01/quick-ligation-protocol</A></P><P>9. 	Blaser Lab Group. E. coli Calcium Chloride competent cell protocol and Transformation of Ca++ competent cells [Internet]. NYU School of Medicine. 2011 [cited 2016 Jun 30]. Available from: <A rel="nofollow" class="external free" href="http://www.med.nyu.edu/medicine/labs/blaserlab/v1-protocols.html">http://www.med.nyu.edu/medicine/labs/blaserlab/v1-protocols.html</A></P><P>10. 	New England Biolabs. NEBuilder HiFi DNA Assembly Reaction Protocol | NEB [Internet]. [cited 2016 Aug 20]. Available from: <A rel="nofollow" class="external free" href="https://www.neb.com/protocols/2014/11/26/nebuilder-hifi-dna-assembly-reaction-protocol">https://www.neb.com/protocols/2014/11/26/nebuilder-hifi-dna-assembly-reaction-protocol</A></P><P>11. 	Addgene. Addgene: Protocol - How to Create a Bacterial Glycerol Stock [Internet]. Addgene.org. [cited 2016 Jun 30]. Available from: <A rel="nofollow" class="external free" href="https://www.addgene.org/plasmid-protocols/create-glycerol-stock/">https://www.addgene.org/plasmid-protocols/create-glycerol-stock/</A></P><P>12. 	New England Biolabs. PCR Protocol for Taq DNA Polymerase with Standard Taq Buffer (M0273) | NEB [Internet]. New England biolabs (NEB). [cited 2016 Jul 10]. Available from: <A rel="nofollow" class="external free" href="https://www.neb.com/protocols/1/01/01/taq-dna-polymerase-with-standard-taq-buffer-m0273">https://www.neb.com/protocols/1/01/01/taq-dna-polymerase-with-standard-taq-buffer-m0273</A></P><P>13. 	Thermo Fisher Scientific. GeneJET Plasmid Miniprep Kit [Internet]. Thermo Fisher Scientific. [cited 2016 Jul 30]. Available from: <A rel="nofollow" class="external free" href="https://www.thermofisher.com/order/catalog/product/K0502">https://www.thermofisher.com/order/catalog/product/K0502</A></P><P>14.     Unpublished protocol provided by L.G Mårtensson, Linköpings University.
</P><P>15. 	Ozgenc A. GeneArt® Chlamydomonas TOPO® Engineering Kits. 
</P><P>16. 	MicroPulser TM Electroporation Apparatus Operating Instructions and Applications Guide Catalog Number 165-2100. :800–424. 
</P><P>17. 	Shimogawara K, Fujiwara S, Grossman A, Usuda H. High-efficiency transformation of Chlamydomonas reinhardtii by electroporation. Genetics [Internet]. Genetics Society of America; 1998 Apr [cited 2016 Oct 16];148(4):1821–8. Available from: <A rel="nofollow" class="external free" href="http://www.ncbi.nlm.nih.gov/pubmed/9560396">http://www.ncbi.nlm.nih.gov/pubmed/9560396</A></P></DIV><DIV id="sponsors"><P>
            LiU iGEM is proudly sponsored by
        </P></DIV><DIV class="footer"><DIV><DIV>
                LiU iGEM
                
                Kårallen, Linköpings Universitet
                
                581 83 Linköping
                <A href="mailto:liuigemgroup@gmail.com" target="_blank">liuigemgroup@gmail.com</A></DIV><DIV><A href="http://www.liu.se/om-liu?l=en" target="_blank">About LiU</A><A href="http://www.liu.se/utbildning?l=en" target="_blank">Study in Linköping</A><A href="http://www.liu.se/forskning?l=en" target="_blank">Research</A><A href="http://www.liu.se/?l=en" target="_blank">www.liu.se</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>