<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Linkoping_Sweden_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Linkoping Sweden/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="content-container"><DIV class="closed-menu"><DIV id="mobile-menu">
            LiU iGEM
            </DIV><DIV class="menu-category"><A href="/Team:Linkoping_Sweden/Team">Team</A><DIV class="menu-items"><A href="/Team:Linkoping_Sweden/Collaborations">Collaborations</A><A href="/Team:Linkoping_Sweden/Attributions">Attributions</A></DIV></DIV><DIV class="menu-category"><SPAN>Project</SPAN><DIV class="menu-items"><A href="/Team:Linkoping_Sweden/Design">Design</A><A href="/Team:Linkoping_Sweden/Experiments">Experiments</A><A href="/Team:Linkoping_Sweden/Protocols">Protocols</A><A href="/Team:Linkoping_Sweden/Results">Results</A><A href="/Team:Linkoping_Sweden/Safety">Safety</A><A href="/Team:Linkoping_Sweden/Notebook">Notebook</A><A href="/Team:Linkoping_Sweden/Economic_Viability">Economic Viability</A></DIV></DIV><DIV class="menu-category"><A href="/Team:Linkoping_Sweden/Description">Project Description</A><DIV class="menu-items"><A href="/Team:Linkoping_Sweden/Description/CRISPR/Cas9-system">CRISPR/Cas9 System</A><A href="/Team:Linkoping_Sweden/Description/Gibson_assembly">Gibson Assembly</A><A href="/Team:Linkoping_Sweden/Description/Chlamydomonas_reinhardtii">Chlamydomonas Reinhardtii</A></DIV></DIV><DIV class="menu-category"><A href="/Team:Linkoping_Sweden/Parts">Parts</A></DIV><DIV class="menu-category"><A href="/Team:Linkoping_Sweden/Human_Practices">Human Practices</A><DIV class="menu-items"><A href="/Team:Linkoping_Sweden/HP/Silver">Silver</A><A href="/Team:Linkoping_Sweden/HP/Gold">Gold</A></DIV></DIV></DIV><DIV class="content"><H2><SPAN class="mw-headline" id="Project_Design"><CENTER>Project Design</CENTER></SPAN></H2><P>
The problem we have addressed is the unprofitability of algal biofuels due to insufficient lipid content, and indirectly the greenhouse effect by promoting renewable energy. The way we have tackled this problem is by designing a construct that might enable us to control the metabolism of <I>Chlamydomonas reinhardtii</I>. The construct is thought to convey inducible CRISPR/Cas9 knockout of a specific metabolic target gene, directing the cells flux of carbon sources towards triacylglycerol (TAG) synthesis.
</P><P><SPAN id="ankare" class="number-link">Knockout</SPAN> of metabolic target genes in <I>C. reinhardtii</I> has been successfully performed many times before. But so far, the outcome of these knockouts has not been algal strains producing enough biomass and lipids to be profitable for biodiesel production (1). For example, knocking out the sta6 gene in <I>C. reinhardtii</I>, encoding the starch synthesis enzyme ADP-glucose pyrophosphorylase, results in accumulation of 20 % less biomass during nutrient-replete conditions (2). However, upon nitrogen starvation, the cells turn obese. Their cytoplasm and chloroplasts fill up with TAG-bodies as a result of their increased carbon flux towards lipid synthesis (3). Even though, this knockout strain is not more productive than other strains from an overall perspective due to its reduced growth rate.
</P><P>
We believe the ultimate <I>C. reinhardtii strain</I> for biofuel production would have the growth rate of a wild type and the fatty acid accumulation of the sta6 null strain, and our hypothesis is that this could be accomplished with an inducible knockout system. By inducing the expression of Cas9 in <I>C. reinhardtii</I> at the biomass peak, before nitrogen starvation, more TAG-bodies would accumulate without affecting the growth rate of the algae.
</P><P>
Our construct is designed to demonstrate the functionality of an inducible CRISPR/Cas9 system in <I>C. reinhardtii.</I> The <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:pSB1C3">pSB1C3 expression vector</A>, and its chloramphenicol resistance gene, serves as base point and selection marker for cloning in <I>E. coli.</I> Cas9 is positively regulated by the light-inducible <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2095000">LIP promoter</A>. The sgRNA, containing target sequence and scaffold sequence, is constitutively expressed under a U6-promoter. Lastly, we have a hygromycin resistance gene (aph7), serving as selection marker for successful transformation of <I>C. reinhardtii.</I></P><P>The construct is constituted by six DNA fragments, assembled by Gibson assembly and designed accordingly. These are the components and their functions:
</P><P><B id="num1" class="number-link">1.</B> The high copy number expression vector <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:pSB1C3">pSB1C3</A>, enabling cloning and selection in <I>E. coli</I>. It also provides iGEM compatibility.
</P><P><B id="num2" class="number-link">2.</B> The light inducible promotor, <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2095000">LIP</A>, enables light inducible expression of Cas9 and target gene knockout. The truncated SORLIPx2 promoter sequence was retrieved from published work by Park S et al. and Baek K et al (4,5). 
</P><P>Turning the switch from proliferation to fatty acid accumulation on an industrial scale requires an effective stimulus. There are several variants of inducible promoters that could be used in <I>C. reinhardtii</I> to express a gene such as Cas9 at given stimuli, and many of these promoters include addition and removal of chemicals. Due to the potential complexity of using chemicals for this purpose, we decided to use high-intensity light as promotor stimulus. Despite that the use of a light-inducible knockout promotor in a photosynthetic organism might seem contradictory, it has its advantages due to its manageability. We also believe that a light-inducible promoter could be particularly useful for algae growing on carbon sources other than carbon dioxide, such as sugar fed or green waste algae, which do not require intensive solar energy.
</P><P><B id="num3" class="number-link">3.</B> Cas9 serves as the knockout mediator in CRISPR/Cas9 systems. The sequence used to create the fragment was retrieved from <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K12180111">BBa_K121801</A> and adopted for Gibson assembly through PCR. The theoretical functionality of this part is questionable due to its lack of codon optimization for <I>C. reinhardtii</I>. We yet decided to use it, mainly because of the lack of other alternatives and the great effort of optimizing it ourselves, and because only a very low Cas9 expression should be required to enable target gene knockout. 
</P><P><B id="num4-5" class="number-link">4-5.</B> sgRNA under a U6 promoter were synthesized as two fragments, based on sequences from IDT protocols (6). These fragments convey constitutive expression of sgRNA in <I>C. reinhardtii</I>.  Since the purpose of this DNA construct was to prove the functionality of CRISPR/Cas9 in <I>C. reinhardtii</I>, the sgRNA does not target a metabolic gene. Instead, the sgRNA was designed to guide Cas9 to the 12-kD FK506-binding protein gene in <I>C. reinhardtii</I>, allowing rapamycin resistance as a selection marker for successful gene knockout.
</P><P>This gene encodes the rapamycin binding protein FKBP12. A protein sensitive to rapamycin, naturally occurring in plants, including the algae of interest, <I>C. reinhardtii</I> (7). Rapamycin is an antibiotic which inhibits cell growth. It acts by binding to FKBP12, and the rapamycin-FKBP12 complex then inhibits the target of rapamycin (TOR). The normally occurring, active form of TOR frames the cell growth of the algae, and its inhibition induces cell death. Consequently, knock out of this gene mediates rapamycin resistance – allowing colonies to survive and grow on rapamycin plates (7).
</P><P><B id="num6" class="number-link">6.</B> The function of the last fragment is to enable selection of successfully transformed colonies of <I>C. reinhardtii</I>. It includes of a codon optimized hygromycin resistance gene, Aph7, which can be used as a selection marker (8). The fragment we designed is equivalent to <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1857001">BBa_K1857001</A>. Unfortunately, the BioBrick was unavailable but the cassette could be retrieved through PCR from the Popt_mVenus_Hyg expression plasmid, provided by <A rel="nofollow" class="external text" href="http://www.chlamycollection.org/">Chlamydomonas Resource Center</A>. This could successfully be achieved but one problem in the design was that the region of interest in <A rel="nofollow" class="external text" href="http://www.chlamycollection.org/content/uploads/2015/06/pOpt_mVenus_Hyg.ape">Popt_mVenus_Hyg</A>, was not unique. Our primers also isolated the mVenus part of the plasmid, a yellow fluorescent protein (YFP) in almost exactly same size as the hygromycin-casette fragment. The only way to separate these fragments was by sequencing. Theoretically, the YFP reporter gene could also serve as a selection marker, therefore we continued with the primer design.
</P><P>
Despite multiple attempts, the assembly has not been fully successful, yet. An insert that appeared to be correct was transformed into <I>C. reinhardtii</I> but selection failed, and the insert later proved to be wrong (<A rel="nofollow" class="external text" href="https://2016.igem.org/Team:Linkoping_Sweden/Results">see results</A>).
</P><H4><SPAN class="mw-headline" id="References">References</SPAN></H4><P>1. 	Goncalves EC, Wilkie AC, Kirst M, Rathinasabapathi B. Metabolic regulation of triacylglycerol accumulation in the green algae: Identification of potential targets for engineering to improve oil yield. Plant Biotechnology Journal. 2016;
</P><P>2. 	Krishnan A, Kumaraswamy GK, Vinyard DJ, Gu H, Ananyev G, Posewitz MC, et al. Metabolic and photosynthetic consequences of blocking starch biosynthesis in the green alga Chlamydomonas reinhardtii sta6 mutant. Plant J. 2015;81(6):947–60. 
</P><P>3. 	Goodenough U, Blaby I, Casero D, Gallaher SD, Goodson C, Johnson S, et al. The path to triacylglyceride obesity in the sta6 strain of Chlamydomonas reinhardtii. Eukaryot Cell [Internet]. 2014 May [cited 2016 Sep 18];13(5):591–613. Available from: <A rel="nofollow" class="external free" href="http://www.ncbi.nlm.nih.gov/pubmed/24585881">http://www.ncbi.nlm.nih.gov/pubmed/24585881</A></P><P>4. 	Park S, Lee Y, Lee J-H, Jin E. Expression of the high light-inducible Dunaliella LIP promoter in Chlamydomonas reinhardtii. Planta [Internet]. 2013 Dec [cited 2016 Sep 4];238(6):1147–56. Available from: <A rel="nofollow" class="external free" href="http://www.ncbi.nlm.nih.gov/pubmed/24043576">http://www.ncbi.nlm.nih.gov/pubmed/24043576</A></P><P>5. 	Baek K, Lee Y, Nam O, Park S, Sim SJ, Jin E. Introducing Dunaliella LIP promoter containing light-inducible motifs improves transgenic expression in Chlamydomonas reinhardtii. Biotechnol J. 2016;11(3):384–92. 
</P><P>6. 	IDT. CRISPR/Cas9 editing: transfection of gBlocks® Gene Fragments that encode sgRNAs. 2015 [cited 2016 Oct 7];4. Available from: <A rel="nofollow" class="external free" href="http://eu.idtdna.com/pages/docs/default-source/user-guides-and-protocols/gblocks-gene-fragment-transfection.pdf?sfvrsn=6">http://eu.idtdna.com/pages/docs/default-source/user-guides-and-protocols/gblocks-gene-fragment-transfection.pdf?sfvrsn=6</A></P><P>7. 	Crespo JL, Díaz-Troya S, Florencio FJ. Inhibition of target of rapamycin signaling by rapamycin in the unicellular green alga Chlamydomonas reinhardtii. Plant Physiol [Internet]. 2005 Dec [cited 2016 Sep 4];139(4):1736–49. Available from: <A rel="nofollow" class="external free" href="http://www.ncbi.nlm.nih.gov/pubmed/16299168">http://www.ncbi.nlm.nih.gov/pubmed/16299168</A></P><P>8. 	Lauersen KJ, Kruse O, Mussgnug JH. Targeted expression of nuclear transgenes in Chlamydomonas reinhardtii with a versatile, modular vector toolkit. Appl Microbiol Biotechnol [Internet]. 2015 Apr [cited 2016 Sep 4];99(8):3491–503. Available from: <A rel="nofollow" class="external free" href="http://www.ncbi.nlm.nih.gov/pubmed/25586579">http://www.ncbi.nlm.nih.gov/pubmed/25586579</A></P></DIV><DIV id="sponsors"><P>
            LiU iGEM is proudly sponsored by
        </P></DIV><DIV class="footer"><DIV><DIV>
                LiU iGEM
                
                Kårallen, Linköpings Universitet
                
                581 83 Linköping
                <A href="mailto:liuigemgroup@gmail.com" target="_blank">liuigemgroup@gmail.com</A></DIV><DIV><A href="http://www.liu.se/om-liu?l=en" target="_blank">About LiU</A><A href="http://www.liu.se/utbildning?l=en" target="_blank">Study in Linköping</A><A href="http://www.liu.se/forskning?l=en" target="_blank">Research</A><A href="http://www.liu.se/?l=en" target="_blank">www.liu.se</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>