<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Cornell_Foundations skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Cornell/Foundations</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team:Cornell/Foundations - 2018.igem.org</TITLE></P><DIV class="standard-page-wrapper"><NAV><DIV class="nav-bar-wrapper"><DIV><DIV class="nav-bar-main-menu"><DIV class="dropdown"><BUTTON class="dropbtn">TEAM</BUTTON><DIV class="dropdown-content team-dropdown-content"><A href="https://2018.igem.org/Team:Cornell/Team">BIOS</A><A href="https://2018.igem.org/Team:Cornell/Sponsors">SPONSORS</A><A href="https://2018.igem.org/Team:Cornell/Attributions">ATTRIBUTIONS</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">OUTREACH</BUTTON><DIV class="dropdown-content outreach-dropdown-content"><A href="https://2018.igem.org/Team:Cornell/Collaborations">COLLABORATIONS</A><A href="https://2018.igem.org/Team:Cornell/Public_Engagement">PUBLIC ENGAGEMENT</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">HUMAN-CENTERED DESIGN</BUTTON><DIV class="dropdown-content human-dropdown-content"><A href="https://2018.igem.org/Team:Cornell/Human_Practices">PRACTICES</A><A href="https://2018.igem.org/Team:Cornell/Policies">POLICIES</A><A href="https://2018.igem.org/Team:Cornell/Entrepreneurship">ENTREPRENEURSHIP</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">MODELING</BUTTON><DIV class="dropdown-content modeling-dropdown-content"><A href="https://2018.igem.org/Team:Cornell/Model">MODELING</A><A href="https://2018.igem.org/Team:Cornell/Testing">TESTING</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn active-page">TOOLKIT</BUTTON><DIV class="dropdown-content toolkit-dropdown-content"><UL><DIV class="nav-first-col"><LI class="wet-lab-list-title"><B>WET LAB</B></LI><LI><A href="https://2018.igem.org/Team:Cornell/Foundations">FOUNDATIONS</A></LI><LI><A href="https://2018.igem.org/Team:Cornell/Demonstrate">DEMONSTRATE</A></LI><LI><A href="https://2018.igem.org/Team:Cornell/InterLab">INTERLAB</A></LI><LI><A href="https://2018.igem.org/Team:Cornell/Parts">PARTS</A></LI><LI><A href="https://2018.igem.org/Team:Cornell/Basic_Part">BASIC PARTS</A></LI><LI><A href="https://2018.igem.org/Team:Cornell/Composite_Part">COMPOSITE PARTS</A></LI></DIV><DIV class="nav-second-col"><LI class="doc-list-title"><B>DOCUMENTATION</B></LI><LI><A href="https://2018.igem.org/Team:Cornell/Notebook">NOTEBOOK</A></LI><LI><A href="https://2018.igem.org/Team:Cornell/Safety">SAFETY</A></LI></DIV></UL></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn" onclick="location.href='https://2018.igem.org/Team:Cornell/About'">ABOUT</BUTTON></DIV><DIV class="dropdown"><BUTTON class="dropbtn" onclick="location.href='https://2018.igem.org/Team:Cornell'">HOME</BUTTON></DIV></DIV></DIV></DIV></NAV><HEADER class="standard-page-banner foundations-page-banner"><SVG viewbox="0 0 100 100" width="100%" height="100%"><TEXT class="standard-page-banner-title" text-anchor="middle" alignment-baseline="middle" x="50%" y="50%">Foundations</TEXT></SVG></HEADER><DIV class="standard-page-side-bar-content-wrapper"><DIV class="standard-page-side-bar-wrapper"><UL class="standard-page-side-bar"><LI><A href="#problemstatement">PROBLEM STATEMENT</A></LI><LI><A href="#systemoverview">SYSTEM OVERVIEW</A></LI><LI><A href="#components">COMPONENTS</A></LI></UL></DIV><DIV class="standard-page-content-wrapper"><DIV class="standard-page-content-section"><DIV class="standard-page-content-title"><A id="problemstatement">Problem Statement</A></DIV><P class="standard-page-content-text">
                        Presently, it is difficult to tune genetic circuits to respond to environmental stimuli in a precise manner due to the amount of signal noise that accompanies many environmental cues. Traditional promoters operate via an intensity-dependent input-output system that accepts input from both the desired stimulus and excess noise. To combat this, most inducible systems are designed to respond to a chemical such as IPTG or arabinose to function as transcriptional switches.  These chemicals are expensive and irreversibly turn expression on in induced cells.
                        We have designed a genetic circuit that responds to frequency as opposed to intensity. This allows us to finely control expression by having the system only respond to specific and intentional variation in stimulus frequency. Thus, our system serves as a more robust response system to minimize noise by only responding to signals that fall within a set frequency range.
                    </P><P class="standard-page-content-text">
                        The specific circuit we designed employs a genetic AND-gate to combine discrete high-pass and low-pass filters for temperature fluctuations into a band-pass filter. The high-pass circuit only allows downstream transcription when the frequency of temperature variation is above a certain threshold. Conversely, the low-pass circuit allows downstream transcription when the frequency of temperature variation is below a different threshold. Combining these two circuits leads to a tunable frequency range for expression. In addition to being less sensitive to noise, this system also avoids the need for expensive or toxic chemical inducers. 
                    </P><P class="standard-page-content-text">
                        Additionally, many natural phenomena are cyclical, so cells that respond to a frequency of stimulus may be more well-suited to respond to such stimuli than traditional amplitude-based expression platforms.
                    </P></DIV><DIV class="standard-page-content-section"><DIV class="standard-page-content-title"><A id="systemoverview">System Overview</A></DIV><P class="standard-page-content-text">We designed a modular, tunable system that could in the future be customized to respond to different kinds of oscillatory input signals (light, heat, small molecule, etc.) and different frequencies. For this project, as a proof-of-concept, we focused on a response to variation in temperature. We sought to not only show functionality and specificity of such a system but also to characterize baseline rates of our specific parts as part of our study.
                    </P><P class="standard-page-content-text">The goal of our project was to build a band pass filter. Our system was thus broken into two components, a “high-pass” and a “low-pass” filter, with both parts operating with similar principles. The low-pass filter places an orthogonal transcription factor, σ<SUP>F</SUP>, under the control of a RNA thermometer, a regulatory RNA sequence that only allows translation of σ<SUP>F</SUP> when the bacteria are above a certain temperature. Downstream of σ<SUP>F</SUP>, we placed a hrpS gene whose transcription was dependent on the presence of the σ<SUP>F</SUP>.</P><P class="standard-page-content-text">The high-pass system was constructed in a similar manner, but instead of having a transcriptional activator regulated by the RNA thermometer, the Tet repressor was placed under the control of the same RNA thermometer. Downstream, a hrpR gene was placed under the control of P<SUB>Tet</SUB>, ensuring hrpR would only be produced in the absence of TetR, i.e. when the bacteria are above the melting point of the RNA thermometer, they will stop producing hrpR. 
                    </P><P class="standard-page-content-text">Our end goal was to combine the high-pass and low-pass circuits in one multi-plasmid system <I>in vivo</I> so that expression of a downstream product only occurs within a specific frequency range of signal. This would be possible because the hrpS and hrpR genes act as an AND gate, as both of them together are necessary to drive transcription of a reporter under the control of the hrpL promoter. Transcription of the final product relies on a balance between production of σ<SUP>F</SUP> and TetR and their respective degradation. They will only exist at the correct balance when the temperature is fluctuated at the appropriate rate for the system. 
                    </P><P class="standard-page-content-text">To test, we placed a reporter gene (sfGFP) downstream of an P<SUB>hrpL</SUB> promoter sequence.
                    </P><P class="standard-page-content-text">In short, our system serves as a biological AND-type logic gate that requires that frequency of signal change not be too high (low pass) AND not too low (high pass), creating a functional range of frequency inputs to trigger reporter expression. When combined, they form a band-pass analogous to an electronic band pass.
                    </P><DIV class="foundations-image-caption">Figure 1. Circuit overview. In practice, the reporter would be on the same plasmid as one of the filters, and mf-Lon would be on the other plasmid.</DIV><DIV class="foundations-image-caption">Figure 2. Electronic band pass. The low pass filter attenuates signals with high frequencies, and the high pass cuts off signals with low frequencies. Together, they form a band-pass filter.</DIV></DIV><DIV class="standard-page-content-section"><DIV class="standard-page-content-title"><A id="components">Components</A></DIV><DIV class="standard-page-content-subheading">RNA Thermometer:</DIV><P class="standard-page-content-text">A critical component of our system is a regulatory element that can respond to oscillatory stimuli. Our system relies on an RNA thermometer (RNAT), that is found in the 5’ UTR of the downstream gene transcript. The RNAT is a rudimentary riboswitch that responds to temperature instead of a small molecule, and forms a hairpin than blocks the Shine-Dalgarno sequence, an RBS. At high enough temperatures, the hairpin melts and allows for translation of the mRNA [1,2]. In addition to providing comparably tight control over gene expression, RNA thermometers have relatively fast kinetics since they rely simply on the melting of a few hydrogen bonds to unwind.
                    </P><P class="standard-page-content-text">We sought an RNA thermometer with near binary-behavior to allow for tighter control in our system. We began with a RNAT designed by Neupert, Karcher, and Bock [3], and mutated it to make it RFC 10 compatible. The RNAT designed by Neupert and colleagues was based on the <I>Listeria monocytogenes</I> RNAT. We performed an analysis of the free energy of the mutated RNAT, and found that it had a similar free energy of ΔG = -9.4 kcal/mol compared to -6.7 kcal/mol to the original structure [4]. The single most effective parameter to predict the behavior of the thermometer is the free energy, and thus we determined the performances of the two RNAT should be similar.
                    </P><DIV class="foundations-image-caption">Figure 3. Predicted folding of RNA thermometer. Image generated in RNAStructure.</DIV><DIV class="foundations-image-caption">Figure 4. Temperature-dependent functionality of the RNA thermometer and state table.</DIV><DIV class="standard-page-content-subheading">σ<SUP>F</SUP> and P<SUB>F2</SUB>:</DIV><P class="standard-page-content-text">σ<SUP>F</SUP> is a transcription factor endogenous to <I>Bacillus subtilis</I> and orthogonal to the <I>E. coli</I> transcriptional machinery [5]. While sigma factors tend to be diverse and dissimilar between different species, the bacterial RNA polymerase is highly conserved and the <I>B. subtilis</I> sigma factors can recruit polymerases even in <I>E. Coli</I>. In our system, σ<SUP>F</SUP> acts as an activator.
                    </P><P class="standard-page-content-text">P<SUB>F2</SUB> is a promoter that contains the consensus binding sequence from <I>B. subtilis</I> for σ<SUP>F</SUP>. Bervoets and colleagues created and characterized a library of promoters for use in E. Coli with alternative sigma factor networks. The σ<SUP>F</SUP>/P<SUB>F2</SUB> system allowed us to have tight transcriptional control of genes in allowing transcription to only occur in the presence of σF whose translation was regulated directly by the external input signal. Thus, this part of the system is responsible for direct response to the oscillatory input signal.
                    </P><DIV class="standard-page-content-subheading">hrpS, hrpR, hrpL:</DIV><P class="standard-page-content-text">hrpS and hrpR are positive transcriptional regulatory proteins originally discovered in <I>Pseudomonas syringae</I> that interact together to drive transcription downstream of a P<SUB>hrpL</SUB> promoter sequence [6]. The cooperativity of hrpS and hrpR was key in designing a system that acted as a logical AND gate [7]. We leveraged the fact that both proteins are necessary to see downstream transcription to design our system to require two simultaneous inputs (in our case these inputs were temperature variation above a specific frequency threshold AND below a different threshold). The hrp regulatory sequences worked as the signal processing component of the circuit.
                    </P><P class="standard-page-content-text">When both hrpR and hrpS are present, they dimerize to form a complex (we will refer to this complex as hrpR/S). Only when hrpR and hrpS are BOTH present in the system can they form the hrpR/S dimer, which functionally acts as a transcription factor. The complex binds to the P<SUB>hrpL</SUB> promoter sequence, and induces production of the downstream gene. In our system, sfGFP is downstream of the promoter, and is only produced when both hrpR and hrpS are present in the system.
                    </P><DIV class="foundations-image-caption">Figure 5. AND gate dependent on hrpR/S complex, as described by Wang, Kitney, Joly, and Buck (2011).</DIV><DIV class="standard-page-content-subheading">sfGFP:</DIV><P class="standard-page-content-text">“Superfolder” green fluorescent protein (sfGFP) is a GFP variant with the primary added advantage of strong folding capability even when fused to poorly folded protein product. Additionally, sfGFP folds 3.5 times faster than normal folder GFP, and is reported to be one of the fastest folding GFPs studied thus far [8,9]. Furthermore, its relatively long half-life makes it easily measurable over long periods of time. These factors made it a prime candidate for incorporation into our system as a reporter because its quick maturation makes it ideal for studying expression dynamics, and its long half-life allows readings to be made on an hourly time scale. 
                    </P><P class="standard-page-content-text">sfGFP was chosen as the reporter because one of the goals of the project was to quantify and subsequently model the amounts of protein expression when the system is exposed to different rates of variation of temperature. Utilizing a fluorescent protein was obviously very convenient for this goal as it allowed accurate, quick quantification of protein expression. However, in the future, this part of the system could be swapped out for any functional protein so that the system could respond in the desired manner to the input signal fluctuation.
                    </P><DIV class="standard-page-content-subheading">mf-Lon:</DIV><P class="standard-page-content-text"><I>mf</I>Lon is a Lon protease originally found in <I>Mesoplasma florum</I> that degrades proteins tagged with a corresponding protein degradation tag [10], described by Cameron and Collins in 2014 and characterized by the 2017 William and Mary iGEM Team [11,12]. It was selected to be incorporated into our system because its protein target sequence is orthogonal to native <I>E. coli</I> degradation tags. It only degrades the specially marked TetR and σ<SUP>F</SUP> protein products so as to prevent false fluorescence readings resulting from protein accumulation. 
                    </P><P class="standard-page-content-text">Furthermore, because mf-Lon is foreign to <I>E. coli</I>, in future studies we could tune the rate of its expression by modifying the promoter or ribosome binding sequences that precede the mfLon gene. This would give us greater control over our system. The rate of degradation of the hrpR/S proteins would become an additional tunable parameter to determine the range of frequency of input signal for which the reporter would be expressed.
                    </P><OL><LI>Chowdhury, S., Maris, C., Allain, F. H. T., &amp; Narberhaus, F. (2006). Molecular basis for temperature sensing by an RNA thermometer. <I>The EMBO journal, 25</I>(11), 2487-2497.</LI><LI>Sen, S., Apurva, D., Satija, R., Siegal, D., &amp; Murray, R. M. (2017). Design of a Toolbox of RNA Thermometers. <I>ACS synthetic biology, 6</I>(8), 1461-1470.</LI><LI>Neupert, J., Karcher, D., &amp; Bock, R. (2008). Design of simple synthetic RNA thermometers for temperature-controlled gene expression in Escherichia coli. <I>Nucleic acids research, 36</I>(19), e124-e124.</LI><LI>Reuter, J. S., &amp; Mathews, D. H. (2010). RNAstructure: software for RNA secondary structure prediction and analysis. BMC Bioinformatics. 11,129.</LI><LI>Bervoets, I., Van Brempt, M., Van Nerom, K., Van Hove, B., Maertens, J., De Mey, M., &amp; Charlier, D. (2018). A sigma factor toolbox for orthogonal gene expression in Escherichia coli. <I>Nucleic acids research, 46</I>(4), 2133-2144.</LI><LI>Hutcheson, S. W., Bretz, J., Sussan, T., Jin, S., &amp; Pak, K. (2001). Enhancer-Binding Proteins HrpR and HrpS Interact To Regulate hrp-Encoded Type III Protein Secretion in Pseudomonas syringae Strains. <I>Journal of bacteriology, 183</I>(19), 5589-5598.</LI><LI>Wang, B., Kitney, R. I., Joly, N., &amp; Buck, M. (2011). Engineering modular and orthogonal genetic logic gates for robust digital-like synthetic biology. <I>Nature communications, 2</I>, 508.</LI><LI>Kiss, C., Temirov, J., Chasteen, L., Waldo, G. S., &amp; Bradbury, A. R. (2009). Directed evolution of an extremely stable fluorescent protein. <I>Protein Engineering, Design &amp; Selection</I>, 22(5), 313-323.</LI><LI>Pédelacq, J. D., Cabantous, S., Tran, T., Terwilliger, T. C., &amp; Waldo, G. S. (2006). Engineering and characterization of a superfolder green fluorescent protein. <I>Nature biotechnology</I>, 24(1), 79.</LI><LI>Huang, D., Holtz, W. J., &amp; Maharbiz, M. M. (2012). A genetic bistable switch utilizing nonlinear protein degradation. <I>Journal of biological engineering</I>, 6(1), 9.</LI><LI>Cameron, D. E., &amp; Collins, J. J. (2014). Tunable protein degradation in bacteria. <I>Nature biotechnology, 32</I>(12), 1276.</LI><LI>2017.igem.org/Team:William_and_Mary</LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>