Team:LN-Shiyan-China/Design
MENU ▤
HOME
TEAM
Team
★  Collaborations
PROJECT
★  Description
★ Design
Experiments
★ Proof of Concept
★ Demonstrate
Results
Notebook
PARTS
Parts
★ Basic Parts
★ Composite Parts
★ Part Collection
SAFETY
★  ATTRIBUTIONS
HUMAN PRACTICES
Human Practices
★ Silver
★ Gold
★ Integrated Practices
★ Engagement
AWARDS
★ Entrepreneurship
★ Hardware
★ Software
★ Measurement
★ Model
Faced with the stress of human pollution such as pesticides, nature itself has evolved many methods to deal with these problems. For example, many natural micro-organisms contain enzymes to degrade organophosphorus pesticides. Currently the micro-organisms which are capable to degrade organophosphorus pesticides include bacteria, fungus, actinomycete and alga. As the research goes further, people find that these degrading effects come from secreting an enzyme, which can hydrolyze phosphoester bonds, organophosphorus degradation enzyme. Because each organophosphorus pesticide has similar structure and protein sequence, one kind of organophosphorus degradation enzyme is capable todegrade multiple kinds of organophosphorus pesticides. Organophosphorus-degradation enzyme has been mostly recognized as the best method to eliminate pesticide residues currently. At present, many enzymes have been identified to be used to degrade organophosphate pesticides. Among these enzymes, the organophosphorus-degradation enzyme (opdA) which comes from Agrobacterium radiobacter P230 has wider targets and higher enzyme-catalyst efficiency. In recent years, the research on the structure and function of organophosphorus-degradation enzyme has gained promising progress,. Thus, it is possible to improve the properties of organophosphorus-degradation enzyme through genetic engineering and protein engineering method, which meet requirements of different applications.
OPDA
London
Paris
Tokyo
London
The organophosphorus-degradation enzyme (opdA) gene opdA （NCBI genbank：Accession: AY043245.2） programmed by Agrobacterium radiobacter contains 1,155 nucleic acids, programming 384 amino acid residues. The N-terminal of protein sequence is the signal peptide while the C-terminal is the degradation-enzyme sequence.
edit by ZTJ
