<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SCUT-China_B_Propoptosis skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SCUT-China B/Propoptosis</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>IGEM-China_B</TITLE></P><DIV class="nav"><LI><A href="https://2016.igem.org/Team:SCUT-China_B">Home</A></LI><LI><A href="https://2016.igem.org/Team:SCUT-China_B/Overview">Project</A></LI><LI><A href="https://2016.igem.org/Team:SCUT-China_B/Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:SCUT-China_B/Propoptosis">Apoptosis Induction</A></LI><LI><A href="https://2016.igem.org/Team:SCUT-China_B/Protumor">Tumor specific promoter</A></LI><LI><A href="https://2016.igem.org/Team:SCUT-China_B/Results">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:SCUT-China_B/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:SCUT-China_B/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:SCUT-China_B/Judging">Judging</A></LI><LI><A href="https://2016.igem.org/Team:SCUT-China_B/Modeling">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:SCUT-China_B/Apoptosis">Apoptosis</A></LI><LI><A href="https://2016.igem.org/Team:SCUT-China_B/Crispri">CRISPRi/a</A></LI><LI><A href="https://2016.igem.org/Team:SCUT-China_B/Protocol">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:SCUT-China_B/Protocol">Protocol</A></LI><LI><A href="https://2016.igem.org/Team:SCUT-China_B/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:SCUT-China_B/HP/Gold">Human Practice</A></LI><LI><A href="https://2016.igem.org/Team:SCUT-China_B/HP/Gold">Overview</A></LI><LI><A href="https://2016.igem.org/Team:SCUT-China_B/HP/HP1">Project Design</A></LI><LI><A href="https://2016.igem.org/Team:SCUT-China_B/HP/HP2">Public Outreach</A></LI><LI><A href="https://2016.igem.org/Team:SCUT-China_B/HP/HP3">Hospice Care</A></LI><LI><A href="https://2016.igem.org/Team:SCUT-China_B/Team">Lab</A></LI><LI><A href="https://2016.igem.org/Team:SCUT-China_B/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:SCUT-China_B/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:SCUT-China_B/Attributions">Attributions</A></LI></DIV><DIV class="bar"><DIV class="Ltitle"><A href="#Mitochondria">Apoptosis</A></DIV><DIV class="Ltitle"><DIV class="Mtitle"><A href="#">Bcl-2/Bax Regulation</A></DIV><DIV class="Stitle"><A href="#CRISPRi">CRISPRi/a</A></DIV><DIV class="Stitle"><A href="#Methodology">Methodology of CRISPRi/a</A></DIV><DIV class="Stitle"><A href="#Target">Target</A></DIV><DIV class="Stitle"><A href="#Construction">sgRNA construction</A></DIV><DIV class="Stitle"><A href="#Transfection">Transfection</A></DIV><DIV class="Stitle"><A href="#Results">Results</A></DIV></DIV><DIV class="Ltitle"><DIV class="Mtitle"><A href="#Apoptosis">Apoptosis Detection</A></DIV><DIV class="Stitle"><A href="#Apoptosis">Annexin-FIFC/PI double staining</A></DIV></DIV><DIV class="Ltitle"><A href="#Results2">Results</A></DIV></DIV><DIV class="article"><DIV id="Mitochondria" class="cont"><P class="tit">2.1 Mitochondria-dependent Apoptosis</P><P class="detail">Figure 1.Mitochondria-dependent Apoptosis, caspase-3 activation </P><P class="detail">After knowing about the current situation of the lung cancer, we considered about developing a new therapy that was targeted, efficient and safe.</P><P class="detail">Apoptosis, a normal mechanism for deleting unwanted or moderately damaged cells, has important implications for cancer[3]. </P><P class="detail">According to the literature, caspase-3 plays a dominant role of apoptosis, however, it normally exists as an inactive proenzyme[4].</P><P class="detail">There are two major pathways that activate the caspase-3, extrinsic pathway and intrinsic pathway[5].</P><P class="detail">The intrinsic signaling, also known as mitochondria-dependent apoptosis, is activated when Bax is translocated into outer mitochondria and changes the permeability of outer mitochondria membrane , ending up releasing cytochrome c. Cytochrome c then recruits apoptosis protease-activating factor 1 to generate apoptosome that activates caspase 9, leading to processing of caspase-3.</P><P class="detail">Bax is the key of protein both in the intrinsic pathway and extrinsic pathway.On theotherhand,high expression rate of Bcl-2 can counteract the effects of Bax. As is also observed , overexpression of antiapoptotic Bcl-2 exists in a large number of human cancers[6], and inactivates mutations of proapoptotic proteins occur in numerous cancers leading to uncontrolled growth of tumors. Moreover, overexpression of antiapoptotic Bcl-2 and its close relatives is a major component of chemoresistance, which means the ratio of bax to bcl-2 is critical to the apoptosis of cancer cells.</P><P class="detail">Taken together, we plan to upregulate bax expression and downregulate bcl-2 expression.</P></DIV><DIV id="CRISPRi" class="cont"><P class="tit">2.2 Upregulation of Bax and Downregulation of Bcl-2</P><P class="tit">2.2.1 Model For Regulation </P><P class="detail">Figure 2.</P><P class="detail">A: Using caspase-3 as the indicator of apoptosis, we found that caspase-3 steady state concentration increased above nanomolar concentration under 3 fold activation of bax transcription.</P><P class="detail">B: Using caspase-3 as the indicator of apoptosis, we found that caspase-3 steady state concentration increased significantly under 4 fold repression of bcl-2.</P><P class="detail">We built a model for mitochondria-dependent apoptosis, it showed that caspase-3 activation can be trigger by the regulation of bax and bcl-2 transcription.</P><P class="detail">The results inspired us to apply CRISPRi/a to regulate our gene transcription.</P></DIV><DIV id="Methodology" class="cont"><P class="tit">2.2.2 Methodology of CRISPRi/a</P><P class="detail">The CRISPR/Cas(Clustered Regularly Interspaced Short Palindromic Repeats/CRISPR associated nucleases) system is a immune system in most archaea and bacteria. When foreign genetic elements invaded into the archaea or bacteria, CRISPR associated proteins (Cas) that have endonuclease activity would cut these exogenous genetic elements.</P><P class="detail">And CRISPR interference is generated from CRISPR/Cas. The only obvious difference between them is that the proteins called dead Cas9 or dCas9 in CRISPR interference lack endonuclease activity.</P><P class="detail">CRISPR interference is a genetic perturbation technique. It consists two elements-- sgRNA and dcas9. sgRNA is a chimeric noncoding RNA that can be subdivided into three regions: a 20 nt base-pairing sequence, a 42 nt dCas9-binding hairpin and a 40 nt terminator. dcas9 is a point mutated protein.  </P><P class="detail">It is from the gene encoding Cas9 that experienced point mutations in the two catalytic residues (D10A and H840A). Besides, CRISPR interference usually allows for sequence-specific repression (CRISPRi) or activation of gene expression (CRISPR a) in prokaryotic and eukaryotic cells.</P><P class="detail">CRISPRi can specifically repress transcription in two ways – by blocking transcriptional initiation or elongation. It works by designing sgRNA complementary to the promoter or exonic sequences. Dcas9 will combine with sgRNA to form a complex to target on genomic loci under the guidance of sgRNA.</P><P class="detail">CRISPRi can also repress transcription via an effector domain, such as Krüppel associated box (KRAB) domain. Fusing KRAB to dCas9 allows transcription to be further repressed by inducing heterochromatinization.</P><P class="detail">Also, dcas9 can fuse with a transcriptional activator like VP16(4 vp16 comform vp64). Then it turns as CRISPRa. </P><P class="detail">Figure 3.</P><P class="detail">Left: CRISPRa : dCas9-VP64, a CRISPR-mediate activator, is to help the formation of initial transcription complex. </P><P class="detail">Right: CRISPRi : dCas9-KRAB, a CRISPR-mediate repressor, is to block the initialization or the elongation of transcription[7].</P></DIV><DIV id="Target" class="cont"><P class="tit">2.2.3 Target</P><P class="detail">Figure 4. Up: sgRNA target sites of bax promoter </P><P class="detail">Down: sgRNA target sites of bcl-2 promoter. </P><P class="detail">According to the literature, sequence preference in CRISPRi/a is significantly different from that in CRISPR/Cas9 knowout, and regulation of CRISPR-mediated effectors tend to have higher activity when they are close to the transcription start site. </P><P class="detail">In our project，We designed 10 sgRNAs for dCas9-VP64 targeting Bax promoter from 928bp to 37bp upstream the transcription start site, and 4 sgRNAs for dCas9-KRAB targeting Bcl-2 promoter from -21bp to 106bp around the transcription start site.</P></DIV><DIV id="Construction" class="cont"><P class="tit">2.2.4 sgRNA Construction</P><P class="detail">Figure 5. We constructed all the sgRNAs by inserting 20bp oligo at Bbs1 site.</P></DIV><DIV id="Transfection" class="cont"><P class="tit">2.2.5 Transfection </P><P class="detail">For A549 cells, we explored the best transfection ratio of transfection reagent GenjetTM (Sigma) to DNA in 24 well is 1.3µL:0.5µg, and 6 well is 7.8μL：3μg.</P></DIV><DIV id="Results" class="cont"><P class="tit">2.2.6 Results</P><P class="detail">Transcription level of bcl-2 and bax</P><P class="detail">CRISPRi:</P><P class="detail">The transcription level of bcl-2 is confirmed by real-time PCR, bcl-2 is nearly 2 to 3 fold repressed.</P><P class="detail">Figure 5.dCas9-KRAB co-transfected with Bcl-2-sgRNA-1 to Bcl-2-sgRNA4 (BBa_K2080004, BBa_K2080005, BBa_K2080006, BBa_K2080007) respectively, and we detected the mRNA level after 48 hour. </P><P class="detail">CRISPRa:</P><P class="detail">The transcription level of bax is confirmed by real-time PCR, bax is nearly 2 fold to 4 fold activation:</P><P class="detail">Figure 6.dCas9-VP64 co-transfected with Bax-sgRNA-4(BBa_K2080002) and Bax-sgRNA-10 (BBa_K2080003)respectively, and we detected the mRNA level after 48 hour.</P></DIV><DIV id="Apoptosis" class="cont"><P class="tit">2.3 Apoptosis Detection</P><P class="detail">Based on the observation that soon after initiating apoptosis, cells translocate the membrane phosphatidylserine (PS) from the inner face of the plasma membrane to the cell surface, the apoptosis index is measured by Annexin V-FITC/PI double staining, which can differentiate apoptosis vs necrosis. </P><P class="detail">Apoptosis was mostly observed after 48 h. Early apoptosis (lower right quadrant) andlate apoptosis/necrosis (upper right quadrant) were clearly evident in dot plots of Figures below.</P></DIV><DIV id="Results2" class="cont"><P class="tit">2.4 Results and Disscussion</P><P class="detail">CRISPRi Apoptosis Analysis:</P><P class="detail"> Four sgRNAs that we design is validated for dCas9-KRAB to downregulate the bcl-2 mRNA level, and we chose bcl-2-sgRNA1 and bcl-2-sgRNA2, which both show strong repression of bcl-2. </P><P class="detail">About 10^5 A549 cells were harvested after 48h for apoptosis analysis. Resuspended cells were incubated with Annexin V-FITC for 15 min in the dark. Propidium iodide was used as a counterstain to discriminate necrotic/ dead cells from apoptotic cells.</P><P class="detail">Figure 7. Apoptosis analysis after Bcl-2-sgRNA1 and dCas9-KRAB were co-trasfected into A549 cells after 48h</P><P class="detail">Figure 8. Apoptosis analysis after Bcl-2-sgRNA2 and dCas9-KRAB were co-trasfected into A549 cells after 48h.</P><P class="detail">Figure 9.  Apoptosis analysis after negative sgRNA and dCas9-KRAB were co-transfected into A549 cells after 48h.</P><P class="detail">Figure 10. Apoptosis analysis for no treatment group after 48h.</P><P class="detail">Apoptosis detection after applying CRISPRi:</P><P class="detail">Figure 11. Apoptosis index of Bcl2-sgRNA1 and Bcl2-sgRNA2.</P><P class="detail">We further upregulated the bcl-2 expression by co-transfecting all the bcl2-sgRNAs into the A549 cells. The apoptosis analysis and the apoptotic index results are listed below.</P><P class="detail">CRISPRa Apoptosis Analysis:</P><P class="detail">Three sgRNAs that we design is validated for dCas9-VP64 to upregulate the bax mRNA level, and we chose bax-sgRNA-4 and bax-sgRNA-10, which both show 4 fold of activation of bax.</P><P>10^5 A549 cells were used for analysis. Resuspended cells were incubated with Annexin V-FITC for 15 min in the dark. Propidium iodide was used as a counterstain to discriminate necrotic/ dead cells from apoptotic cells. </P><P class="detail">Figure 12. Apoptosis analysis after Bax-sgRNA4 and dCas9-VP64 were co-trasfected into A549 cells after 48h. </P><P class="detail">Figure 13. Apoptosis analysis after Bax-sgRNA10 and dCas9-VP64 were co-trasfected into A549 cells after 48h.</P><P class="detail">Figure 14.Apoptosis analysis after negative sgRNA and dCas9-VP64 were co-transfected into A549 cells after 48h</P><P class="detail">Apoptosis detection after applying CRISPRa:</P><P class="detail">Figure 15. Apoptosis analysis of Bax-sgRNA-4 and Bax-sgRNA-10.</P><P class="detail">We further upregulated the bax expression by co-transfecting all the bax-sgRNAs into the A549 cells. The apoptosis analysis and the apoptotic index results are listed below.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>