<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Caltech_Notebook_Degradation"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Caltech/Notebook/Degradation">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Caltech/Notebook/Degradation&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Caltech/Notebook/Degradation&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Caltech/Notebook/Degradation&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Caltech/Notebook/Degradation" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Caltech/Notebook/Degradation</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><UL id="css3menu1" class="topmenu"><LI class="topfirst"><A href="https://2012.igem.org/Team:Caltech" style="height:14px;line-height:14px;">
					Home
				</A></LI><LI class="topmenu"><A href="https://2012.igem.org/Team:Caltech/Team" style="height:14px;line-height:14px;"><SPAN>
						Team
					</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Team#Mentors">
							Mentors
						</A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Team#Undergraduates">
							Undergraduates
						</A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Team#CalArts">
							CalArts
						</A></LI></UL><LI class="topmenu"><A href="https://2012.igem.org/Team:Caltech/Project" style="height:14px;line-height:14px;"><SPAN>
						Project
					</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Project">
							Overall Project
						</A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Notebook/material">
							Materials &amp; Methods
						</A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Project#Degradation_Project">
							Degradation
						</A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Project#Proteorhodopsin_Project">
							Proteorhodopsin
						</A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Project#Coliroid_Project">
							Bacterial Animation
						</A></LI><LI class="topmenu"><A href="https://2012.igem.org/Team:Caltech/Parts" style="height:14px;line-height:14px;"><SPAN>
						Parts
					</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Parts">
							Submitted Parts
						</A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Parts/Characterization">
							Characterization
						</A></LI><LI class="topmenu"><A href="https://2012.igem.org/Team:Caltech/Results" style="height:14px;line-height:14px;"><SPAN>
						Results
					</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Results#Degradation">
							Degradation
						</A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Results#Proteorhodopsin">
							Proteorhodopsin
						</A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Results#Coliroid">
							Bacterial Animation
						</A></LI><LI class="topmenu"><A href="https://2012.igem.org/Team:Caltech/Project" style="height:14px;line-height:14px;"><SPAN>
						Notebook
					</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Notebook/Degradation">
							Degradation
						</A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Notebook/Proteorhodopsin">
							Proteorhodopsin
						</A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Notebook/Coliroid">
							Bacterial Animation
						</A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Notebook/material">
							Materials &amp; Methods
						</A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Notebook/Safety">
							Safety
						</A></LI><LI class="topmenu"><A href="https://2012.igem.org/Team:Caltech/Sponsors" style="height:14px;line-height:14px;">
					Sponsors
				</A></LI><LI class="toplast"><A href="https://2012.igem.org/Team:Caltech/Human_Practice" style="height:14px;line-height:14px;">
					Human Practice
				</A></LI></P><H1>Degradation Notebook</H1><H5> All of the following has been conducted and written up by Huey-Ru (Debra) Tsai </H5><FONT size="+2"><A name="6_22_12">June 22, 2012</A></FONT> Created a spreadsheet detailing various possible bacterial strains and plasmids to be used for complex polymer degradation
<A a="" style="float:right;" href="https://2012.igem.org/Team:Caltech/Notebook/Degradation#Calendar">Back to the top</A><FONT size="+2"><A name="6_25_29_12">June 25 - 29, 2012</A></FONT> Sent out requests for bacterial strains and plasmids based on spreadsheet
 Emailed Dr. Jared Leadbetter asking if he has strains + protocol for lignocellulose plates
 Made carb. R plates
 Made minimal media agar plates, bottom layer of overlay plates
 Met with Professor Leadbetter to discuss difficulties with creating enrichment plates and finding bacterial samples
 Will be following up on some of his suggestions
 Notes from Meeting with Professor Jared Leadbetter from June 28th
 Overlay Plate Options
 - Top layer with minimal media, although either minimal media or just water + agarose should work
 - Embed bacteria into top layer mix
 - Use filter paper made of chosen carbon sources
 Bacterial Strains
 - Ponds around campus
 - Beach
 - Teredinibacter turnerae gen: helps degrade wood in shipworms (http://ijs.sgmjournals.org/content/52/6/2261.abstract)
 May also choose a different host organism for transformation
 - Instead of using E. coli can try using yeast (simplifies problem to only of degradation, and not also ethanol synthesis)
 - Can also try bacteria found by Aztecs to make alcohol (Zymomonas mobilis)
 Emailed for samples of nitrocellulose, latex, and teflon membranes.
 Researched Z. mobilis.
<A a="" style="float:right;" href="https://2012.igem.org/Team:Caltech/Notebook/Degradation#Calendar">Back to the top</A><FONT size="+2"><A name="7_2_6_12">July 2-6, 2012</A></FONT> Researched zymomonas mobilis, narrowed down focus to strain ZM4
 Gathered more information on protocols specific to that strain.
 Looked up additional information about potential vectors to use to transform z. mobilis
 Requested p42-0119 plasmid for expression z. mobilis from the Oak Ridge National Laboratory
 Made 20% glucose solution stock and RM plates for Z. mobilis
<A a="" style="float:right;" href="https://2012.igem.org/Team:Caltech/Notebook/Degradation#Calendar">Back to the top</A><FONT size="+2"><A name="7_9_13_12">July 9-13, 2012</A></FONT> Ordered strains from the ATCC.
 Discussed Biolog plates idea
 Research z mobilis degradation 5 and 6 carbon sugars
 Researched for genetic sequences for sugar degradation enzymes.
 Sent in request for oligos sequences for alpha-gluctosidase gene (can degrade maltose)
 Made tetracycline plates
<A a="" style="float:right;" href="https://2012.igem.org/Team:Caltech/Notebook/Degradation#Calendar">Back to the top</A><FONT size="+2"><A name="7_16_20_12">July 16-20, 2012</A></FONT> The pMQ30 is a deletion plasmid. We are using this plasmid to knock out expression of NADH dehydrogenase. According to the paper ''Respiratory chain analysis of high ethanol producing Zymomonas mobilis mutants,'' a mutant strain of Z. mobilis lacking NADH dehydrogenase had increased ethanol production. However, pMQ30 has gentamicin resistance, and Zymomonas is already resistant to gentamicin, so we are replacing it with tetracycline resistance from vector backbone pSB1T3.
 The pLAFR5 plasmid is being used for conjugation from E. coli into Z. mobilis.
 DKN79 (DH5alpha + pLAFR5) has been plated to grow more copies of pLAFR5.
 DKN1005 (BW20427 [WM3064] + pLAFR5)  has been plated to be used for testing conjugation into Z. mobilis ZM4.
 The pMQ95 and pMQ97 plasmids are to be expressed in Zymomonas; however they contain antibiotic resistance genes that Z. mobilis is already naturally resistant to. So, we will need to switch out the resistances on the plasmids to tetracycline.
 Electroporated pSB1T3 into competent E coli cells, incubated in LB for an hour, and streaked transformants onto tetra plates
 Streaked DH5alpha + pMQ30 onto a gentamicin 20 ug/mL plate
 Streaked Z. mobilis ZM4 from ATCC onto a RM plate and liquid culture.
 Streaked out DKN79 (DH5alpha + pLAFR5) onto a tetracycline plate
 Streaked out DKN1005 (BW20427 [WM3064] + pLAFR5) onto a tetracycline plate
 Made LB media
 No growth with pSB1T3 transformants, so retransformed and plated E coli + pSB1T3
 Kept a liquid culture of E coli + pSB1T3
 Ordered primers as part of switching out the pSB1T3 plasmid's resistance to tetracycline
 Z. mobilis plate has no growth
 Sent DNAWorks requests for als genes (alsBACE)
 Re-streaked out DH5alpha + pMQ30 because first plate's growth was poor
 Streaked out pMQ95 and pMQ97
 There was no growth on the pSB1T3 plates. I kept a liquid culture of the electroporated cells that were left over after plating the on the tetra plates, and the tube looked very cloudy. The liquid culture did not have any antibiotics, but the plates had tetracycline. So it is possible that the gene is broken or its the wrong part or something. We decided to get the tetra resistance gene from a different source on the registry plates. So, I transformed BBa_K274002 and BBa_K274003 into E. coli from 2012 Kit Plate 3 12B, 2012 Kit Plate 4 20D, and 2012 Kit Plate 3 22D. I was originally trying to use BBa_J04450 from Plate 1, well 7A.
 I was able to liquid cultures of pMQ30, pMQ95, pMQ97, DKN79 (DH5alpha + pLAFR5), and DKN1005 (BW20427 [WM3064] + pLAFR5). There has been no growth on the Z. mobilis plates yet. There seems to be growth in the Z. mobilis liquid culture, but not enough to make a glycerol stock. I designed primers for switching out resistance on pMQ95 or pMQ97 and for getting the tet resistance gene from the plated electroporated cells.
 Growth has appeared on the Z. mobilis plates!!
 Cleaned out the lab space
 Made glycerol stock of Z. mobilis (one without gentamicin, one with gent)
 Made liquid cultures of DKN1005 (BW20427 [WM3064] + pLAFR5), part from 2012 registry plate 1 7A, and 2012 registry part from plate 4 20A
 Made glycerol stocks from liquid cultures of DKN1005 (BW20427 [WM3064] + pLAFR5) and part from plate 1 7A
 Made more LB + tetracycline plates
 Z. mobilis liquid cultures currently growing:
 1. original (only Rich Media)
 2. RM + amp (taken from original)
 3. RM + amp (taken from original)
 4. RM + gent (taken from plate)
 Ordered primers for amplifying out the desired products from the plasmids (tetracycline gene from pSB1T3 and everything but the antibiotics gene from pMQ30, pMQ95, and pMQ97)
<A a="" style="float:right;" href="https://2012.igem.org/Team:Caltech/Notebook/Degradation#Calendar">Back to the top</A><FONT size="+2"><A name="7_23_27_12">July 23-27, 2012</A></FONT> Made liquid cultures of strains carrying pMQ30, pMQ95, pMQ97, and pSB1T3
 Made Rich Media + tetracycline (25ug/mL) plates
 Mini prep'ed the plasmids from the liquid cultures
 Ran PCR on the mini preps
 Ran a gel of the PCR products -&gt; indicated that the plasmid backbones did not PCR correctly
 Had another PCR run -&gt; gel indicated that the correct plasmid backbones were produced
<B> Constructing plasmids to be expressed in Z. mobilis ZM4</B><I><FONT size="-1"> There are three plasmids that we worked on synthesizing this week. We would like to use pMQ30, pMQ97, and pMQ95 in Z. mobilis. These plasmids carry gene resistances that Z. mobilis is already resistant. Therefore, we need to switch out the resistances to tetracycline. For the first two plasmids, we used Gibson assembly. Plasmid pMQ95 was altered by using yeast to assemble the fragments together. We could not use Gibson assembly for pMQ95 because the primers used for PCR amplification of the various DNA fragments would interfere with the promoter region for the new gene for tetracycline resistance. </FONT></I> Purified the PCR products with QIAquick PCR purification kit of the tet gene DNA insert from pSB1T3 into the plasmid pMQ30 (i30), the tet gene DNA insert from pSB1T3 into the plasmid pMQ95 (i95), the backbone of pMQ30 (bb30), the backbone of pMQ97 (bb97), and the backbone of pMQ95 (bb95)
 Digested i30 and bb95 with restriction enzyme DpnI and purified these products again
 Ran gibson to build two plasmids: one made of bb30 and i30, and the other made of bb97 and i30
 These plasmids were electroporated into E. coli and plated on tetracycline plates
 Synthesized a plasmid made from bb95 and i95 in yeast
 Made a liquid culture of DKN1005, which is the donor conjugation strain carrying pLAFR5
 Tested conjugation of pLAFR5 into Z. mobilis ZM4 from DKN1005, set to incubate at 30C, on the bench for a day, and on the bench overnight, and plate 
 Electroporated E. coli cells with pMQ30 and pMQ97 did not grow -&gt; Plated a new set of electroporated cells with pMQ30 and pMQ97
 Saturday:
 Check up on electroporated cells -&gt; success!
 Plated the overnight-incubation conjugation cells
<A a="" style="float:right;" href="https://2012.igem.org/Team:Caltech/Notebook/Degradation#Calendar">Back to the top</A><FONT size="+2"><A name="7_30_3_12">July 30 - August 3, 2012</A></FONT> Ran colony PCR on e. coli and yeast colonies carrying the new plasmids made from last week
 Obtained mixed results from the gel of the PCR products
 Inoculated overnight cultures of samples of the e coli and yeast colonies
 Made liquid cultures with tet from pMQ30 and pMQ97 from e coli
 Mini prep'ed two samples each from pMQ30 and pMQ97 from e coli
 Made RM and LB
 Made Z. mobilis liquid cultures
 Made glycerol stocks of the liquid cultures of the pMQ30 and pMQ97 e coli samples
 Designed primers for integrating the allose operon into pMQ97, a three step gibson to create a biobrick plasmid backbone that can be expressed in many organisms, and a primer with for integrating RFP into pMQ97 to check the promoter for the plasmid with the allose operon
 Prep'ed an overnight culture of WM3064 (This strain will be used as the donor strain for conjugation with Z. mobilis)
 Electroporated pMQ97-tet into the prepared WM3064
 Plated transformed WM3064 on LB + tet + DAP plates
 Made a glycerol stock of WM3064
 Started another liquid culture of Z. mobilis
 Made LB + tet plates
 Made new RM and sterile 20% glucose solution
 Started 3 new ZM4 liquid cultures + 1 negative control
 Restreaked WM3064 on LB + tet + DAP plates because growth was too high on original plates
 Plated contaminants from old ZM4 liquid cultures and old RM
 Looked at contaminated cultures under a microscope (very cool!!)
 Restreaked growth on conjugation plates -&gt; may be contaminated
<A a="" style="float:right;" href="https://2012.igem.org/Team:Caltech/Notebook/Degradation#Calendar">Back to the top</A><FONT size="+2"><A name="8_6_3_10">August 6 - 10, 2012</A></FONT> Sunday: Start liquid cultures of pMQ95 + tet transformants
 Liquid cultures from Sunday did not grow
 Made ZM4, Top10 e. coli pMQ95 + tet, and DH5alpha + pLAFR5 liquid cultures in preparation to test conjugation
 Finished designing primers
 Made glycerol stocks of ZM4
 Top10 e. coli pMQ95 + tet liquid cultures did not grow
 Started biobricks culture with Z. mobilis
 Tested new conjugation protocol using DH5alpha + pLAFR5 as donor strain and ZM4 as receiver
 Re-electroporated pMQ95 + tet into Top10 e. coli
 Re-streaked Top10 e. coli + pMQ97 + tet from glycerol stock on LB + tet plate
 Made a liquid culture of Top10 e. coli + (pMQ97 + tet)
 Made a liquid culture of WM3064 + pLAFR5
 Made a liquid culture of ZM4, ZM4 + amp (50 ug/mL), and ZM4 + chlor (17 ug/mL)
 Minipreped plasmid pMQ97 from the e. coli liquid culture started the day before to electroporate the plasmid into ZM4 at 1 5kv/cm
 Conjugated ZM4 with WB3064 + pLAFR5
 Made amp 50 tet 25 plates
 Made liquid culture Top10 - pMQ97+ tet and ZM4
 Streaked a plate from the glycerol stock of ZM4 made the day before
 Friday: conjugate pMQ97 + tet from Top10 e. coli carrying plasmid into ZM4
 Run pcr reactions to build pMQ97 + tet + alsBACEK (alsBACEK will be amplified from e. coli genome)
<A a="" style="float:right;" href="https://2012.igem.org/Team:Caltech/Notebook/Degradation#Calendar">Back to the top</A><H5> The following has been conducted and written up by Daisy Lin </H5><FONT size="+2"><A name="8_21_12">8/21/12</A></FONT> ligated pSB1C3 w/ alsBACEK using EcoRI (high fidelity) and SpeI, making positive and negative controls (+/- alsBACEK)
 plated the transformed E. coli cells
 performed a DpnI digest of pMQ97-tet to cut up genomic DNA
<A a="" style="float:right;" href="https://2012.igem.org/Team:Caltech/Notebook/Degradation#Calendar">Back to the top</A><FONT size="+2"><A name="8_24_12">8/24/12</A></FONT> performed Gibson Assembly of pMQ97-tet, P_adhA (promoter), and alsBACEK
 sent pSB1C3+alsBACEK in for sequencing
<A a="" style="float:right;" href="https://2012.igem.org/Team:Caltech/Notebook/Degradation#Calendar">Back to the top</A><FONT size="+2"><A name="8_27_12">8/27/12</A></FONT> sequencing results appear to be 'unpossible' (not found in BLAST); will attempt to redo the ligation
 started overnight cultures of other Gibson Assembly colonies
<A a="" style="float:right;" href="https://2012.igem.org/Team:Caltech/Notebook/Degradation#Calendar">Back to the top</A><FONT size="+2"><A name="8_28_12">8/28/12</A></FONT> redid pSB1C3+alsBACEK ligation
 miniprepped Gibson Assembly for sequencing
<A a="" style="float:right;" href="https://2012.igem.org/Team:Caltech/Notebook/Degradation#Calendar">Back to the top</A><FONT size="+2"><A name="8_30_12">8/30/12</A></FONT> positive and negative alsBACEK controls did not grow at all
<A a="" style="float:right;" href="https://2012.igem.org/Team:Caltech/Notebook/Degradation#Calendar">Back to the top</A><FONT size="+2"><A name="8_31_12">8/31/12</A></FONT> sent in 3 of the colonies from the original pSB1C3+alsBACEK ligation for sequencing
<A a="" style="float:right;" href="https://2012.igem.org/Team:Caltech/Notebook/Degradation#Calendar">Back to the top</A><FONT size="+2"><A name="9_3_12">9/3/12</A></FONT> sequencing results came back; all of the inserts in each colony are incorrect
<A a="" style="float:right;" href="https://2012.igem.org/Team:Caltech/Notebook/Degradation#Calendar">Back to the top</A><FONT size="+2"><A name="9_7_12">9/7/12</A></FONT> redid pSB1C3+alsBACEK ligation using twice the amount of insert
 performed PCR cleanup and started DpnI digest on pMQ97 and pMQ97-tet mini-vector fragments
<A a="" style="float:right;" href="https://2012.igem.org/Team:Caltech/Notebook/Degradation#Calendar">Back to the top</A><FONT size="+2"><A name="9_10_12">9/10/12</A></FONT> streaked Debbie's glycerol stocks of pMQ97-tet E1, E2, and E3 for individual colonies
 started Z. mobilis liquid cultures
 started liquid cultures of E. coli with pMQ97-tet
 performed PCR cleanup on the digested mini-vector fragments
 performed Gibson Assembly on the fragments
<A a="" style="float:right;" href="https://2012.igem.org/Team:Caltech/Notebook/Degradation#Calendar">Back to the top</A><FONT size="+2"><A name="9_11_12">9/11/12</A></FONT> started 3 Z. mobilis liquid cultures
 started liquid cultures of E1, E2, and E3
 made Rich Media
 Gibson Assembly plates have very small colonies, so I left them in the incubator for another day
<A a="" style="float:right;" href="https://2012.igem.org/Team:Caltech/Notebook/Degradation#Calendar">Back to the top</A><FONT size="+2"><A name="9_12_12">9/12/12</A></FONT> started 3 Z. mobilis liquid cultures (the ones from yesterday were accidentally thrown out)
 miniprepped E1, E2, and E3 and electroporated them into electrocompetent E. coli cells
 poured RM+amp+carb plates
 Gibson Assembly colonies probably do not have the plasmid as we want it, since no cells grew on the positive control of pMQ97, although cells grew on the negative control; regarding pMQ97-tet, more cells grew on the positive control than the negative one; I plan to redo the Gibson Assembly starting from the PCR
<A a="" style="float:right;" href="https://2012.igem.org/Team:Caltech/Notebook/Degradation#Calendar">Back to the top</A><FONT size="+2"><A name="9_13_12">9/13/12</A></FONT> performed biparental conjugation and pLAFR5 conjugation to transfer E1, E2, and E3 from E. coli to Z. mobilis
<A a="" style="float:right;" href="https://2012.igem.org/Team:Caltech/Notebook/Degradation#Calendar">Back to the top</A><FONT size="+2"><A name="9_26_12">9/26/12</A></FONT> made LB+carb plates and LB media
 redid PCR of Gibson Assembly fragments 1-G, 1-T, and 3
<A a="" style="float:right;" href="https://2012.igem.org/Team:Caltech/Notebook/Degradation#Calendar">Back to the top</A><FONT size="+2"><A name="9_27_12">9/27/12</A></FONT> performed PCR cleanup on fragments 1-G, 1-T, and 3 and measured their concentrations (1-G and 1-T ~300 ng/uL, while 3 ~100 ng/uL)
 started DpnI digest of fragments 1-G, 1-T, and 3
<A a="" style="float:right;" href="https://2012.igem.org/Team:Caltech/Notebook/Degradation#Calendar">Back to the top</A><FONT size="+2"><A name="9_28_12">9/28/12</A></FONT> performed PCR cleanup on the digested fragments and measured their concentrations (~15 ng/uL)
 there is too little DNA for Gibson Assembly; possibly something wrong with our Qiagen PCR Purification kit (buffers expired?)
<A a="" style="float:right;" href="https://2012.igem.org/Team:Caltech/Notebook/Degradation#Calendar">Back to the top</A><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Caltech/Notebook/Degradation">http://2012.igem.org/Team:Caltech/Notebook/Degradation</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Caltech/Notebook/Degradation" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Caltech/Notebook/Degradation" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Caltech/Notebook/Degradation&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Caltech/Notebook/Degradation&amp;oldid=266215" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>