"
Page
Discussion
View source
History
teams
Log in
 
Team:Caltech/Notebook/material/General Protocols
From 2012.igem.org
Home
Team
Mentors
Undergraduates
CalArts
Project
Overall Project
Materials & Methods
Degradation
Proteorhodopsin
Bacterial Animation
Parts
Submitted Parts
Characterization
Results
Degradation
Proteorhodopsin
Bacterial Animation
Notebook
Degradation
Proteorhodopsin
Bacterial Animation
Materials & Methods
Safety
Sponsors
Human Practice
General Protocols
Making electrocompetent cells
PREPARATION:
Sterile 1 liter SOB medium (20 g Tryptone, 5 g Yeast Extract, 0.584 g NaCl, 0.186 g KCl. pH = 7.0)
2 Sterile 500 ml centrifuge bottles
Sterile 10% Glycerol (~ 600 ml)
Sterile eppendorf
2 Sterile Falcon tubes
PROCEDURE
Maintain sterile conditions throughout.  Grow culture at 30ºC shaking until OD600 = 0.5-0.7 (takes 4-5 hours). Chill cells at 4ºC for 30 minutes. Also chill centrifuge bottles, 10% glycerol, Falcon tubes and eppendorf tubes. From this step onwards, everything must be kept on ice.     Transfer 250 ml culture into each of the two centrifuge bottles. Spin at 5000 g for 10 minutes at 4ºC. Discard supernatant and repeat this step with rest of the 500 ml culture.     Resuspend pellet in each of the two centrifuge bottles in 50 ml 10% glycerol. Transfer to 2 sterile 50 ml Falcon tubes. Vortexing helps in resuspension.     Centrifuge at 5000 g for 10 minutes at 4ºC. Discard supernatant. Resuspend pellet in each of the two Falcon tubes in 50 ml 10% glycerol by vortexing – WASH 1.     Repeat step 5 for 4 more times for a total of 5 washes.     Discard supernatant and resuspend in the residual 10% glycerol. Aliquot 50 microliter into each of the eppendorf tubes. Store at -80ºC.
Transformation by Electroporation
Add 1 ul of DNA to 50 ul of competent cells on ice.
Pipette up and down and chill on ice.
Transfer to a pre-chilled cuvette.
Chill on ice for 2 minutes.
Put cuvette into electroporater machine and slide it in.
Press on the pulse button once and wait until you hear a beep sound.
Take cuvette out and quickly dilute with 1 mL of LB or SOC.
Transfer DNA into falcon tube and shake in 37C for 1 hour.
Plate 100 ul of culture onto appropriate antibiotic plate.
Leave plate(s) in 37C overnight.
Transformation from DNA distribution plate
PREPARATION:
10 ul of sterile water
50 ul of competent cells
500 ul of LB
PROCEDURE:
Find which well your DNA is in.
Poke with a pipette tip.
Put 10 ul of sterile water into the well, pipetting up and down until the DNA turns red/orange.
Wait 5 minutes to incubate.
Take 10 ul of DNA out of the well and place into a sterile eppendorf tube, chill on ice.
Add 50 ul of competent cells to empty sterile eppendorf tubes with your DNA in them, chill on ice with competent cells on ice at all times.
Put 1 ul of red DNA into the eppendorf tubes with the competent cell.
Chill on ice for 15 minutes.
Heat shock at 42C for 45 seconds.
Immediately chill on ice for 2 minutes.
Add 500 ul of LB into falcon tubes.
Put DNA into falcon tubes with LB.
Shake in 37C for 1 hour.
Plate on antibiotic appropriate plates.
Leave plates upside down in plates in 37 overnight
Cell Transformation
1 uL of DNA from plasmid
2 uL for the gibson
to 50 uL cells
ice for 15-30 mins
42C for 45 s
ice for 2 mins
add 500 uL of LB
37 for an hour
plate 200
Gene Assembly
Get the Amino Acid sequence of the gene you want to assemble
Submit a job for oligonucleotides to assemble the gene at the DNAworks website (http://helixweb.nih.gov/dnaworks/) settings I use are as follows:
Standard Organism: E. coli
Annealing Temperature: 58
Oligo Length: 60 nt (check Random)
Number of Solutions: 10
Check no gaps in assembly! (important)
Restriction Site Screen: select the Biobrick Enzymes
Sequences (protein), paste amino acid sequence
Overlay Plates
Overlay plates are made out of two layers. The bottom layer is made of minimal media agar with no carbon source. The top layer is made of minimal media mixed with a chosen carbon source. Bacterial samples are grown in between the two layers.
Bottom Minimal Media Agar Layer
Makes 100 mL of agar mix, approximately 5 plates.
PREPARATION:
70 mL sterilized water
10 mL phosphate buffer
10 mL salt solution 1
10 mL salt solution 2
100 uL trace minerals
1.5 g of bacto agar
PROCEDURE:
Mix all solutions and agar together
Autoclave, making sure to place mixture into a water bath and cracking the lid of the container
Let cool in 50ºC water bath for about 30 minutes
Pour into plates
Let plates set at room temperature overnight
Gently apply bacterial sample to the plates
Top Layer
Makes a 0.2% carbon source overlay layer, approximately 50 mL. If the bacterial sample was in an aqueous solution, be sure to allow the plate to dry out before applying the top layer. Alternatively, the top layer could be made from a solution mix of just the water, agarose, and the chosen carbon source.
PREPARATION:
35 mL sterilized water
5 mL phosphate buffer
5 mL salt solution 1
5 mL salt solution 2
50 uL trace minerals
0.75 g of low melting point temperature bacto agarose
0.10 mL of chosen carbon source
PROCEDURE:
Mix everything but the carbon source together
Autoclave, making sure to place mixture into a water bath and cracking the lid of the container
Let cool in 37ºC water bath for 30 minutes
Mix carbon source into cooled mixture
Pour about 5 mL into each plate
Enrichment cultures
Minimal media recipe for 100 mL:
70 mL water
10 mL 10x phosphate buffer
10 mL each salt soln (also 10x)
100 uL trace mineral mix
Preparation
5 mL minimal media in each tube
Add carbon source (25 mg alginate, vanillin, lignin; 10 mg lignin; 20 uL polystyrene, teflon)
1 mL from pond sample or previous culture (for later dilutions, 100 uL from previous culture)
Place in room-temp shaker
Redilute every 2-3 days
Rich Media (RM)
Makes 400 mL of agar mix, approximately 1 sleeve of plates.
PREPARATION:
400 mL sterilized water
8 g glucose (40 mL from 20% stock solution)
4 g yeast extract
0.8 g KH2PO4
8 g bacto-agar (for plates; leave out agar for liquid rich media)
PROCEDURE:
Mix everything except the glucose together
Autoclave, making sure to place mixture into a water bath and cracking the lid of the container
Let cool in 50ºC water bath for about 30 minutes
Stir in the glucose
Pour into plates
Let plates set at room temperature overnight
Gently apply bacterial sample to the plates
LB plates
Makes 1 L of agar mix, approximately 2 sleeves of plates.
PREPARATION:
10g tryptone
10g NaCl
5g yeast extract
15g agar (not needed if making only media)
1 L water
PROCEDURE:
Mix everything together, add water to volume
Autoclave, making sure to place mixture into a water bath and cracking the lid of the container
Let cool in 50ºC water bath for about 30 minutes
Stir in the antibiotics as needed
Pour into plates
Let plates set at room temperature overnight
Gently apply bacterial sample to the plates
Colony PCR
PREPARATION
Nanopure water 20uL per colony
PCR tube(s) - one per colony
GoTaq  - 12.5uL per tube
Primers - 1.25uL of forward, 1.25uL of reverse
PROCEDURE:
Take a colony from a plate and put in 20uL dH2O in PCR tube (at least one colony from each plate)
Restreak on new plate using the same pipette tip (or wooden stick)
Boil tube content in PCR at 98C for 10min (optional)
Use that as template for colony PCR (25uL reaction)
Mix in PCR tube:
dH2O
8.5 μL
template
1 μL
forward primer
10 μM 1.25 μL
reverse primer
10 μM 1.25 μL
GoTaq
12.5 μL
PCR and verify on gel
PCR program: 98°C 30s → 35*[98°C 10s → 53°C 20s → 72°C 35s] → 72°C 5min → 4°C forever
Primers come in little envelope (get paper spreadsheet due to stickers)
Primers come in little blue tubes in a package
Take one; see DNA dried up on the side of the tube
Put little blue tubes in centrifuge (all of them) to spin DNA to bottom, and click double arrow for about 2 seconds to spin lightly
Now, dissolve them to 100 uM concentration in nanopure water (ideally sterile)
Amount of DNA on tube (ex 22.6 nm (nanomoles)) -> add 226 uL of water to get it to 100uM concentration
2 at a time, vortex them for 5 seconds to mix DNA and water
Phusion DNA polymerase (not tac): phusion is expensive because it’s a high fidelity polymerase (few mistakes) so it’s better if you’re gonna use the DNA later
Phusion is from the freezer, 2x master mix
Add water, primer, and template to phusion
Template: from mini prep set
PSB1T3: vector which has tetracycline resistance gene
pMQ30: plasmid for gene deletions
pMQ95: expression plasmid
pMQ97: expression plasmid
amplify pMQ97, 30, 95 to amplify the backbone (will amplify everything except tetracycline resistance)
2 rxns of PSB1T3, because overhangs for 30&97 are the same so they share
Ending in r=reverse, f=forward; starting with bb=backbone
Mix primers that go together to concentration of 10 uM
Put primers in freezer in pMQ vector box
get 5 PCR tubes, in cupboard, with racks on the counter
look online at phusion site (neb.com) and look up 50 ul reaction protocol
2.5 ul primer mix, .5 ul of template, 22 ul H20 = 25 total, then 25 ul phusion mastermix (from 2x phusion)
put in pSB1T3 into the i’s, and the respective pMQ’s into each tube for the template
for the primer mix, the bb97 and bb30 share the bb30 mix, all others get paired with their respective numbers
25 ul phusion into each tube
put leftover phusion in buffers box in freezer
largest backbone: 10 kb -> 2.5 minute extension time
Electrophoresis with 0.1% agarose gel
PREPARATION:
0.5 g agarose
0.5 ul cybersafe stain
50 mL TAE 1x buffer
PROCEDURE OF MAKING GEL:
Mix the agarose and 50 mL of TAE buffer together
Microwave mixture until mixture does not have floating particles (roughly a minute)
Mix in the cybersafe stain until solution is homogeneous
Pour into gel box mold (make sure the comb/teeth are in the gel box for the wells)
Let solidify (about 30 minutes) at room temperature
PROCEDURE OF OF RUNNING GEL:
Place gel in electrophoresis chamber where the wells are closer to the side that hooks up to the negative electrode
Add enough TAE buffer such that the gel is just barely submerged in buffer
Load samples (with dye)
Hook up electrodes
Run (~30 minutes)
PROCEDURE OF OF IMAGING GEL:
Remove gel from electrophoresis box
Put gel into UV-box (in the Murray Lab), make sure UV light is off when you open the box
Turn on the UV light, make sure the bax is closed
Open up the imaging program
Change the settings until gel is in focus
Save gel image in iGEM folder (file should be named dateYOUNAME, ex: 7_12Debbie)
Print gel
Yeast transformation
PROCEDURE:
Thaw competent yeast cells at RT (RT ok for all steps)
Aliquot 50 uL into new tube
Add 100 ng of plasmid to 50 uL cells
For homologous recombination, add 100 ng of plasmid + 300-500 ng insert
Vortex vigorously for 2 minutes
Heat shock 42C water bath for 40 min (up to 180 min ok)
Add 250 uL of sterile water, vortex vigorously for 2 minutes
Plate 100 uL on selective plate
Plasmid Transfer by Conjugation
Plasmid with a transfer origin (oriT) can be shuttled between strains using conjugation, provided that the donor strain (or a helper strain) contains the code for the necessary transfer machinery. The standard donor strain for conjugation in the Newman lab is E. coli BW29427 (DKN164, formerly known as WM3064). BW29427 is a diaminopimelic acid (dap) auxotroph, i.e. dap (300 µM) has to be added to any medium before it can grow. Plates can be prepared by spreading 200 µl of a 60 mM dap stock.
Biparental conjugation:
Mix 800 µl of an overnight culture (LB + dap + suitable antibiotic) of E. coli BW29427 harboring the plasmid to be transferred and 200 µl of the recipient strain (e.g. PA14, grown overnight in LB).
Centrifuge the cells and wash the pellet in LB to remove the antibiotic.
Centrifuge and resuspend the pellet in 30 µl LB and, spot onto the center of an LB+dap plate and incubate at 30°C overnight.
Pipette ~2 ml of LB onto the mating plate and resuspend the cells by scraping, e.g. with a pipette tip.
Collect the cell suspension with a pipette after tilting the plate, centrifuge, and resuspend the cells in a volume suitable for plating.
Plate onto one or more plates without dap (so BW29427 cannot grow) and with a suitable antibiotic to select for the plasmid (this prevents growth of plasmidless recipient cells). Only recipients carrying the plasmid will form colonies.
Presence of the plasmid (replicable, transposon carrier, or integrated suicide vector) in colonies can be verified by PCR.
PREPRARATION:
Liquid cultures of donor strain (DKN1005) and receptor strain (Z. mobilis ZM4)
DAP
0.9% salt solution
PROTOCOL FOR pLAFR5 CONJUGATION:
Treat RM plates made for Z. mobilis with 100 uL of DAP
Incubate membranes on plates for 30 minutes
Mix 750 uL of the donor strain (DKN1005) and 250 uL of the receptor strain (Z. mobilis ZM4). If the receptor strain has low density, then mix the strains 50:50.
Centrifuge for 1 minute and pour out the supernatant (This step removes the antibiotics)
Resuspend in 1 mL of RM
Repeat steps 4 and 5
Then plate 50 uL of the mixture onto the filter membranes
Incubate at 30 C
Place the filter in a falcon tube containing the 0.9% salt solution (1 mL each)
Vortex vigorously until solution is cloudy
Plate 100 uL onto RM + tet plates (no DAP on these plates causes the DKN1005 donor to die)
Ethanol Assay
Ethanol Color Assay Reaction
Materials
water
1.0M bicine (pH 8.0)
40 mM EDTA
10 mM NAD
4.2 mM thiazolyl blue
16.6 mM phenazine ethosulfate
1 mg/mL alcohol dehydrogenase enzyme
ethanol
Method
Make a master mix consisting of:
3.6 mL water
1.2 mL bicine
1.2 mL EDTA
1.2 mL NAD
1.2 ml thiazolyl blue
2.4 mL phenazine ethosulfate
Before running the experiment, add 1.2 mL of alcohol dehydrogenase enzyme.
Pipette 5 uL of sample into the 96-well microtiter plate.  Add 95 uL of master mix.   Measure absorbance at 570 nm.  We found that the maximum absorbance of each well occurred at a different time relative to the concentration, so it is best to run a series of measurements over a period of approximately 4 hours.
MOPS Buffer
We used the recipe from Teknova listed here but substituted 10 ml of 20% glucose for 8 ml of 50% glucose.  Mix all the following ingredients together to get MOPS buffer:
100 ml 10X MOPS Mixture
10 ml 0.132 M K2HP04
100 ml 10X ACGU
200 ml 5X Supplement EZ
582 ml Sterile H20*
8 ml 20% Glucose
Back to Materials and Methods
Retrieved from " http://2012.igem.org/Team:Caltech/Notebook/material/General_Protocols "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
