"
Page
Discussion
View source
History
teams
Log in
 
Team:Caltech/Results
From 2012.igem.org
Home
Team
Mentors
Undergraduates
CalArts
Project
Overall Project
Materials & Methods
Degradation
Proteorhodopsin
Bacterial Animation
Parts
Submitted Parts
Characterization
Results
Degradation
Proteorhodopsin
Bacterial Animation
Notebook
Degradation
Proteorhodopsin
Bacterial Animation
Materials & Methods
Safety
Sponsors
Human Practice
Results Page
Degradation
First, vectors that can stably replicate in Z. mobilis were constructed. Using the techniques Gibson assembly and homologous recombination in yeast, new vectors called pMQ95+tet and pMQ97+tet were constructed by replacing the original gene resistances on pMQ95 and pMQ97 with tetracycline and transformed into E. coli colonies. The next step was to identify and obtain the genes that code for the degradation enzymes of allose. It was found that one of the standard laboratory strains of E. coli had the genes for allose degradation enzymes, which are collectively called alsBACEK. This collection of genes was amplified out of the E. coli genome, and we attempted to ligate alsBACEK with pSB1C3, a high-expression BioBrick plasmid; however, the sequence we received from Retrogen did not match our desired sequence nor any other in BLAST. We also attempted to construct new expression plasmids using Gibson Assembly to join the necessary components of the pMQ97 (with gentamicin resistance) and pMQ97-tet plasmids with the standard BioBrick RFP insert and the forward (VF2) and reverse (VR) primer binding sites; due to low DNA yields of the fragments from PCR purification (using the Qiagen kit), Gibson Assembly did not seem to work. Finally, we attempted to transfer the plasmid into Z. mobilis using 4 parent conjugation; however, the resulting colonies were an abnormal color of white and displayed abnormal growth patterns, so conjugation was unsuccessful.
The liquid minimal media cultures were plated on LB after two weeks, and significant growth was observed, indicating that growth had taken place in the minimal media cultures. To further isolate these organisms, each culture was then plated on solid minimal media with the carbon source added, and growth was again observed. As a control, the cultures were also plated on solid minimal media without the carbon source and failed to grow, indicating that they are not using agar as a carbon source.
The 16S sequences of these organisms were amplified via PCR and sequenced, and compared with NCBI’s nucleotide database. The lignin-degrading cultures were identified to be Sphingobacterium multivorum, and the polystyrene-degrading cultures were identified to be Raoultella terrigena.
Proteorhodopsin
We successfully constructed the part for proteorhodopsin, as can be verified by our sequencing data in the parts registry.  We then conducted an ATP assay on our proteorhodopsin strain to determine if it effectively produced a proton gradient to generate more ATP.  Unfortunately our results were inconclusive.  In this experiment, we expected to see increased ATP generation when the promoter was induced and light was available for the proteorhodopsin pump.  We used cyanide (CN) to inhibit the electron transport chain, and covered tubes in foil to imitate darkness.  A 60 Watt incandescent light bulb was used for excitation.  aTc was used to bind to the R0040's tetR repressor.  Luminescence correlates to ATP concentration.
In the J123106 strain, we saw an unexpected slight increase in luminescence when there was no light source, meaning the proteorhodopsin did not yield increased ATP.  The cells treated with CN did not produce markedly higher levels of ATP, suggesting that a 1 mM concentration of cyanide may have been insufficient in shutting down the electron transport chain.
Similarly in the R0040 strain, when aTc was bound to the promoter repressor, we saw only a slight increase in luminescence corresponding to ATP yield.  When the repressor was not bound (no aTc was present) the levels of ATP were about the same as when the repressor was bound, signifying the promoter may be a little "leaky".
When we sequenced the proteorhodopsin strain, we found that we had the sequence we expected.  However, there are many reasons that our proteorhodopsin did not behave as we expected in the characterization assay.  Missing from our part is the ribosome binding site; this could cause lack of expression of the gene.  Another issue may be our protocol for reading luminescence.  We plan to add a ribosome binding site and research our plate reader protocol for further characterization of the part.
We also adapted a successful ethanol assay from the common NADH-detecting MTT formazan assay using ATP dehydrogenase.  The description of the assay is in the Materials and Methods section.
Bacterial Animation
For the results of the bacterial animation project, we have successfully created two constructs that we have submitted to the Parts Registry as BioBricks: mCherry-LVA in pSB1C3 and mCherry-AAV in pSB1C3.  In parallel we inserted mCherry into R0040.  The sequencing data showed all constructs have been built successfully.  We ran a RFP assay on our parts in R0040 (with a tetR promoter) and found that our constructs behaved exactly as expected.  We ran a characterization assay measuring the fluorescence of R0040, mCherry-LVA, mCherry-AAV, and mCherry (no degradation tag) both with and without aTc, which binds to the tetR repressor.  As we hypothesized, the untagged cells produced over 50 times more fluorescent protein than those with degradation tags.  The mCherry-LVA/AAV still had more fluorescence than R0040, showing that the protein was produced successfully and the degradation tags worked.  Also as we predicted, the LVA had less fluorescence than the AAV, demonstrating that LVA degrades more quickly than AAV.
The picture below shows the same data from the previous graph without the untagged.
We are still in the progress of making the bacterial animation system. We also had an exhibit at Cal Arts on bacterial plate art and had students draw on and make their own plate art.  More about our collaboration with CalArts can be found on our Human Practice page.
Retrieved from " http://2012.igem.org/Team:Caltech/Results "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
