<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Caltech_Notebook_material_General_Protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Caltech/Notebook/material/General_Protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Caltech/Notebook/material/General_Protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Caltech/Notebook/material/General_Protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Caltech/Notebook/material/General_Protocols&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Caltech/Notebook/material/General_Protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Caltech/Notebook/material/General Protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><UL id="css3menu1" class="topmenu"><LI class="topfirst"><A href="https://2012.igem.org/Team:Caltech" style="height:14px;line-height:14px;">
					Home
				</A></LI><LI class="topmenu"><A href="https://2012.igem.org/Team:Caltech/Team" style="height:14px;line-height:14px;"><SPAN>
						Team
					</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Team#Mentors">
							Mentors
						</A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Team#Undergraduates">
							Undergraduates
						</A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Team#CalArts">
							CalArts
						</A></LI></UL><LI class="topmenu"><A href="https://2012.igem.org/Team:Caltech/Project" style="height:14px;line-height:14px;"><SPAN>
						Project
					</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Project">
							Overall Project
						</A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Notebook/material">
							Materials &amp; Methods
						</A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Project#Degradation_Project">
							Degradation
						</A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Project#Proteorhodopsin_Project">
							Proteorhodopsin
						</A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Project#Coliroid_Project">
							Bacterial Animation
						</A></LI><LI class="topmenu"><A href="https://2012.igem.org/Team:Caltech/Parts" style="height:14px;line-height:14px;"><SPAN>
						Parts
					</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Parts">
							Submitted Parts
						</A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Parts/Characterization">				Characterization
						</A></LI><LI class="topmenu"><A href="https://2012.igem.org/Team:Caltech/Results" style="height:14px;line-height:14px;"><SPAN>
						Results
					</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Results#Degradation">
							Degradation
						</A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Results#Proteorhodopsin">
							Proteorhodopsin
						</A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Results#Coliroid">
							Bacterial Animation
						</A></LI><LI class="topmenu"><A href="https://2012.igem.org/Team:Caltech/Project" style="height:14px;line-height:14px;"><SPAN>
						Notebook
					</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Notebook/Degradation">
							Degradation
						</A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Notebook/Proteorhodopsin">
							Proteorhodopsin
						</A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Notebook/Coliroid">
							Bacterial Animation
						</A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Notebook/material">
							Materials &amp; Methods
						</A></LI><LI><A href="https://2012.igem.org/Team:Caltech/Notebook/Safety">
							Safety
						</A></LI><LI class="topmenu"><A href="https://2012.igem.org/Team:Caltech/Sponsors" style="height:14px;line-height:14px;">
					Sponsors
				</A></LI><LI class="toplast"><A href="https://2012.igem.org/Team:Caltech/Human_Practice" style="height:14px;line-height:14px;">
					Human Practice
				</A></LI></P><H1><A name="General_Protocols">General Protocols</A></H1><H2><A name="electrocompetent_cells">Making electrocompetent cells</A></H2><H3>PREPARATION:</H3><UL><LI>Sterile 1 liter SOB medium (20 g Tryptone, 5 g Yeast Extract, 0.584 g NaCl, 0.186 g KCl. pH = 7.0)</LI><LI>2 Sterile 500 ml centrifuge bottles</LI><LI>Sterile 10% Glycerol (~ 600 ml)</LI><LI>Sterile eppendorf </LI><LI>2 Sterile Falcon tubes</LI></UL><H3>PROCEDURE</H3><P class="tab">Maintain sterile conditions throughout.

Grow culture at 30ºC shaking until OD600 = 0.5-0.7 (takes 4-5 hours).
Chill cells at 4ºC for 30 minutes. Also chill centrifuge bottles, 10% glycerol, Falcon tubes and eppendorf tubes. From this step onwards, everything must be kept on ice.
    Transfer 250 ml culture into each of the two centrifuge bottles. Spin at 5000 g for 10 minutes at 4ºC. Discard supernatant and repeat this step with rest of the 500 ml culture.
    Resuspend pellet in each of the two centrifuge bottles in 50 ml 10% glycerol. Transfer to 2 sterile 50 ml Falcon tubes. Vortexing helps in resuspension.
    Centrifuge at 5000 g for 10 minutes at 4ºC. Discard supernatant. Resuspend pellet in each of the two Falcon tubes in 50 ml 10% glycerol by vortexing – WASH 1.
    Repeat step 5 for 4 more times for a total of 5 washes.
    Discard supernatant and resuspend in the residual 10% glycerol. Aliquot 50 microliter into each of the eppendorf tubes. Store at -80ºC.
</P><H2><A name="Transformation_by_Electroporation">Transformation by Electroporation</A></H2><UL><LI>Add 1 ul of DNA to 50 ul of competent cells on ice.
</LI><LI>Pipette up and down and chill on ice.
</LI><LI>Transfer to a pre-chilled cuvette.
</LI><LI>Chill on ice for 2 minutes.
</LI><LI>Put cuvette into electroporater machine and slide it in.
</LI><LI>Press on the pulse button once and wait until you hear a beep sound.
</LI><LI>Take cuvette out and quickly dilute with 1 mL of LB or SOC.
</LI><LI>Transfer DNA into falcon tube and shake in 37C for 1 hour.
</LI><LI>Plate 100 ul of culture onto appropriate antibiotic plate.
</LI><LI>Leave plate(s) in 37C overnight.  
</LI></UL><H2><A name="Transformation_from_distribution">Transformation from DNA distribution plate</A></H2><H3>PREPARATION:</H3><UL><LI>10 ul of sterile water
</LI><LI>50 ul of competent cells
</LI><LI>500 ul of LB
</LI></UL><H3>PROCEDURE:</H3><UL><LI>Find which well your DNA is in.
</LI><LI>Poke with a pipette tip.
</LI><LI>Put 10 ul of sterile water into the well, pipetting up and down until the DNA turns red/orange.
</LI><LI>Wait 5 minutes to incubate.
</LI><LI>Take 10 ul of DNA out of the well and place into a sterile eppendorf tube, chill on ice.
</LI><LI>Add 50 ul of competent cells to empty sterile eppendorf tubes with your DNA in them, chill on ice with competent cells on ice at all times.
</LI><LI>Put 1 ul of red DNA into the eppendorf tubes with the competent cell.
</LI><LI>Chill on ice for 15 minutes.
</LI><LI>Heat shock at 42C for 45 seconds.
</LI><LI>Immediately chill on ice for 2 minutes.
</LI><LI>Add 500 ul of LB into falcon tubes.
</LI><LI>Put DNA into falcon tubes with LB.
</LI><LI>Shake in 37C for 1 hour.
</LI><LI>Plate on antibiotic appropriate plates.
</LI><LI>Leave plates upside down in plates in 37 overnight
</LI></UL><H2><A name="Cell_Transformation">Cell Transformation</A></H2><UL><LI>1 uL of DNA from plasmid
</LI><LI>2 uL for the gibson
</LI><LI>to 50 uL cells
</LI><LI>ice for 15-30 mins
</LI><LI>42C for 45 s
</LI><LI>ice for 2 mins
</LI><LI>add 500 uL of LB
</LI><LI>37 for an hour
</LI><LI>plate 200
</LI></UL><H2><A name="Gene_Assembly">Gene Assembly</A></H2><UL><LI>Get the Amino Acid sequence of the gene you want to assemble
</LI><LI>Submit a job for oligonucleotides to assemble the gene at the DNAworks website (http://helixweb.nih.gov/dnaworks/) settings I use are as follows:
</LI><LI>Standard Organism: E. coli
</LI><LI>Annealing Temperature: 58
</LI><LI>Oligo Length: 60 nt (check Random)
</LI><LI>Number of Solutions: 10
</LI><LI>Check no gaps in assembly! (important)
</LI><LI>Restriction Site Screen: select the Biobrick Enzymes
</LI><LI>Sequences (protein), paste amino acid sequence
</LI></UL><H2><A name="Overlay_plates">Overlay Plates</A></H2><P class="tab">
Overlay plates are made out of two layers. The bottom layer is made of minimal media agar with no carbon source. The top layer is made of minimal media mixed with a chosen carbon source. Bacterial samples are grown in between the two layers.
</P><H3>Bottom Minimal Media Agar Layer</H3><P>Makes 100 mL of agar mix, approximately 5 plates.</P><H3>PREPARATION:</H3><UL><LI>   70 mL sterilized water
 </LI><LI>   10 mL phosphate buffer
 </LI><LI>   10 mL salt solution 1
  </LI><LI>   10 mL salt solution 2
 </LI><LI>    100 uL trace minerals
 </LI><LI>    1.5 g of bacto agar
</LI></UL><H3>PROCEDURE:</H3><UL><LI>    Mix all solutions and agar together
  </LI><LI>   Autoclave, making sure to place mixture into a water bath and cracking the lid of the container
  </LI><LI>   Let cool in 50ºC water bath for about 30 minutes
 </LI><LI>    Pour into plates
  </LI><LI>   Let plates set at room temperature overnight
 </LI><LI>    Gently apply bacterial sample to the plates
</LI></UL><H3>Top Layer</H3><P class="tab">Makes a 0.2% carbon source overlay layer, approximately 50 mL. If the bacterial sample was in an aqueous solution, be sure to allow the plate to dry out before applying the top layer. Alternatively, the top layer could be made from a solution mix of just the water, agarose, and the chosen carbon source.
</P><H3>PREPARATION:</H3><UL><LI>    35 mL sterilized water
 </LI><LI>    5 mL phosphate buffer
 </LI><LI>    5 mL salt solution 1
  </LI><LI>   5 mL salt solution 2
 </LI><LI>    50 uL trace minerals
 </LI><LI>    0.75 g of low melting point temperature bacto agarose
 </LI><LI>    0.10 mL of chosen carbon source
</LI></UL><H3>PROCEDURE:</H3><UL><LI>    Mix everything but the carbon source together
</LI><LI>     Autoclave, making sure to place mixture into a water bath and cracking the lid of the container
</LI><LI>     Let cool in 37ºC water bath for 30 minutes
</LI><LI>     Mix carbon source into cooled mixture
 </LI><LI>    Pour about 5 mL into each plate
</LI></UL><H2><A name="Enrichment_cultures">Enrichment cultures</A></H2><H3>Minimal media recipe for 100 mL:</H3><UL><LI>    70 mL water
   </LI><LI>       10 mL 10x phosphate buffer
  </LI><LI>       10 mL each salt soln (also 10x)
  </LI><LI>       100 uL trace mineral mix
</LI></UL><H3>Preparation</H3><UL><LI>   5 mL minimal media in each tube
</LI><LI>   Add carbon source (25 mg alginate, vanillin, lignin; 10 mg lignin; 20 uL polystyrene, teflon)
</LI><LI>   1 mL from pond sample or previous culture (for later dilutions, 100 uL from previous culture)
</LI><LI>  Place in room-temp shaker
</LI><LI>   Redilute every 2-3 days
</LI></UL><H2><A name="Rich_Media">Rich Media (RM)</A></H2><P>
Makes 400 mL of agar mix, approximately 1 sleeve of plates.
</P><H3>
PREPARATION:
</H3><UL><LI>    400 mL sterilized water
 </LI><LI>    8 g glucose (40 mL from 20% stock solution)
  </LI><LI>   4 g yeast extract
  </LI><LI>   0.8 g KH2PO4
  </LI><LI>   8 g bacto-agar (for plates; leave out agar for liquid rich media)
</LI></UL><H3>
PROCEDURE:
</H3><UL><LI>   Mix everything except the glucose together
 </LI><LI>    Autoclave, making sure to place mixture into a water bath and cracking the lid of the container
 </LI><LI>    Let cool in 50ºC water bath for about 30 minutes
 </LI><LI>    Stir in the glucose
 </LI><LI>    Pour into plates
 </LI><LI>    Let plates set at room temperature overnight
 </LI><LI>    Gently apply bacterial sample to the plates
</LI></UL><H2><A name="LB_plates">LB plates</A></H2><P>
Makes 1 L of agar mix, approximately 2 sleeves of plates.
</P><H3>
PREPARATION:
</H3><UL><LI>     10g tryptone
</LI><LI>     10g NaCl
</LI><LI>     5g yeast extract
 </LI><LI>    15g agar (not needed if making only media)
 </LI><LI>    1 L water
</LI></UL><H3>
PROCEDURE:
</H3><UL><LI>    Mix everything together, add water to volume
</LI><LI>     Autoclave, making sure to place mixture into a water bath and cracking the lid of the container
 </LI><LI>    Let cool in 50ºC water bath for about 30 minutes
 </LI><LI>    Stir in the antibiotics as needed
 </LI><LI>    Pour into plates
 </LI><LI>    Let plates set at room temperature overnight
 </LI><LI>    Gently apply bacterial sample to the plates
</LI></UL><H2><A name="colony_PCR">Colony PCR</A></H2><H3>PREPARATION</H3><LI>  Nanopure water 20uL per colony
</LI><LI>  PCR tube(s) - one per colony
</LI><LI>  GoTaq  - 12.5uL per tube
</LI><LI>  Primers - 1.25uL of forward, 1.25uL of reverse

<H3>PROCEDURE:</H3></LI><LI>  Take a colony from a plate and put in 20uL dH2O in PCR tube (at least one colony from each plate)
</LI><LI>  Restreak on new plate using the same pipette tip (or wooden stick)
</LI><LI>  Boil tube content in PCR at 98C for 10min (optional)
</LI><LI>  Use that as template for colony PCR (25uL reaction)
</LI><LI>  Mix in PCR tube:
<TABLE border="1"><TBODY><TR><TD> dH2O	</TD><TD>	8.5 μL</TD></TR><TR><TD> template</TD><TD>	1 μL</TD></TR><TR><TD> forward primer	</TD><TD> 10 μM	1.25 μL </TD></TR><TR><TD> reverse primer	</TD><TD> 10 μM	1.25 μL </TD></TR><TR><TD> GoTaq		</TD><TD>12.5 μL</TD></TR></TBODY></TABLE></LI><LI>PCR and verify on gel

</LI><LI>PCR program: 98°C 30s → 35*[98°C 10s → 53°C 20s → 72°C 35s] → 72°C 5min → 4°C forever

</LI><LI> Primers come in little envelope (get paper spreadsheet due to stickers)

</LI><LI>      Primers come in little blue tubes in a package

</LI><LI>       Take one; see DNA dried up on the side of the tube

</LI><LI>       Put little blue tubes in centrifuge (all of them) to spin DNA to bottom, and click double arrow for about 2 seconds to spin lightly
</LI><LI>      Now, dissolve them to 100 uM concentration in nanopure water (ideally sterile)
</LI><LI>       Amount of DNA on tube (ex 22.6 nm (nanomoles)) -&gt; add 226 uL of water to get it to 100uM concentration

</LI><LI>       2 at a time, vortex them for 5 seconds to mix DNA and water

</LI><LI>         Phusion DNA polymerase (not tac): phusion is expensive because it’s a high fidelity polymerase (few mistakes) so it’s better if you’re gonna use the DNA later

</LI><LI>       Phusion is from the freezer, 2x master mix
</LI><LI>    Add water, primer, and template to phusion

</LI><LI>       Template: from mini prep set
</LI><LI>  PSB1T3: vector which has tetracycline resistance gene

</LI><LI>    pMQ30: plasmid for gene deletions

</LI><LI>  pMQ95: expression plasmid

</LI><LI>   pMQ97: expression plasmid
</LI><LI>       amplify pMQ97, 30, 95 to amplify the backbone (will amplify everything except tetracycline resistance)

</LI><LI>        2 rxns of PSB1T3, because overhangs for 30&amp;97 are the same so they share
</LI><LI>        Ending in r=reverse, f=forward; starting with bb=backbone

</LI><LI>         Mix primers that go together to concentration of 10 uM

</LI><LI>       Put primers in freezer in pMQ vector box

  </LI><LI>   get 5 PCR tubes, in cupboard, with racks on the counter
 </LI><LI>    look online at phusion site (neb.com) and look up 50 ul reaction protocol
 </LI><LI>    2.5 ul primer mix, .5 ul of template, 22 ul H20 = 25 total, then 25 ul phusion mastermix (from 2x phusion)
 </LI><LI>    put in pSB1T3 into the i’s, and the respective pMQ’s into each tube for the template
  </LI><LI>   for the primer mix, the bb97 and bb30 share the bb30 mix, all others get paired with their respective numbers
  </LI><LI>   25 ul phusion into each tube
 </LI><LI>    put leftover phusion in buffers box in freezer
</LI><LI> 
    largest backbone: 10 kb -&gt; 2.5 minute extension time
<H2><A name="Electrophoresis">Electrophoresis with 0.1% agarose gel</A></H2><H3>PREPARATION:</H3></LI><LI>    0.5 g agarose
</LI><LI>     0.5 ul cybersafe stain
</LI><LI>     50 mL TAE 1x buffer
<H3>PROCEDURE OF MAKING GEL:</H3></LI><LI>    Mix the agarose and 50 mL of TAE buffer together
</LI><LI>     Microwave mixture until mixture does not have floating particles (roughly a minute)
</LI><LI>     Mix in the cybersafe stain until solution is homogeneous
</LI><LI>     Pour into gel box mold (make sure the comb/teeth are in the gel box for the wells)
</LI><LI>     Let solidify (about 30 minutes) at room temperature
<H3>PROCEDURE OF OF RUNNING GEL:</H3></LI><LI>     Place gel in electrophoresis chamber where the wells are closer to the side that hooks up to the negative electrode
 </LI><LI>    Add enough TAE buffer such that the gel is just barely submerged in buffer
</LI><LI>     Load samples (with dye)
</LI><LI>     Hook up electrodes
 </LI><LI>    Run (~30 minutes)
<H3>
PROCEDURE OF OF IMAGING GEL:</H3></LI><LI>    Remove gel from electrophoresis box
</LI><LI>     Put gel into UV-box (in the Murray Lab), make sure UV light is off when you open the box
</LI><LI>     Turn on the UV light, make sure the bax is closed
</LI><LI>     Open up the imaging program
</LI><LI>     Change the settings until gel is in focus
</LI><LI>     Save gel image in iGEM folder (file should be named dateYOUNAME, ex: 7_12Debbie)
</LI><LI>     Print gel

<H2><A name="yeast_transformation">Yeast transformation</A></H2><H3>
PROCEDURE:</H3></LI><LI>  Thaw competent yeast cells at RT (RT ok for all steps)
</LI><LI>  Aliquot 50 uL into new tube
</LI><LI>  Add 100 ng of plasmid to 50 uL cells
</LI><LI>  For homologous recombination, add 100 ng of plasmid + 300-500 ng insert
</LI><LI> Vortex vigorously for 2 minutes
</LI><LI> Heat shock 42C water bath for 40 min (up to 180 min ok)
</LI><LI> Add 250 uL of sterile water, vortex vigorously for 2 minutes
</LI><LI> Plate 100 uL on selective plate
<H2><A name="Conjugation">Plasmid Transfer by Conjugation</A></H2><P class="tab">Plasmid with a transfer origin (oriT) can be shuttled between strains using conjugation, provided that the donor strain (or a helper strain) contains the code for the necessary transfer machinery. The standard donor strain for conjugation in the Newman lab is E. coli BW29427 (DKN164, formerly known as WM3064). BW29427 is a diaminopimelic acid (dap) auxotroph, i.e. dap (300 µM) has to be added to any medium before it can grow. Plates can be prepared by spreading 200 µl of a 60 mM dap stock.
</P><H3>Biparental conjugation:</H3></LI><LI>      Mix 800 µl of an overnight culture (LB + dap + suitable antibiotic) of E. coli BW29427 harboring the plasmid to be transferred and 200 µl of the recipient strain (e.g. PA14, grown overnight in LB).
</LI><LI>      Centrifuge the cells and wash the pellet in LB to remove the antibiotic.
</LI><LI>      Centrifuge and resuspend the pellet in 30 µl LB and, spot onto the center of an LB+dap plate and incubate at 30°C overnight.
</LI><LI>    Pipette ~2 ml of LB onto the mating plate and resuspend the cells by scraping, e.g. with a pipette tip.
</LI><LI>      Collect the cell suspension with a pipette after tilting the plate, centrifuge, and resuspend the cells in a volume suitable for plating.
</LI><LI>      Plate onto one or more plates without dap (so BW29427 cannot grow) and with a suitable antibiotic to select for the plasmid (this prevents growth of plasmidless recipient cells). Only recipients carrying the plasmid will form colonies.
</LI><LI>       Presence of the plasmid (replicable, transposon carrier, or integrated suicide vector) in colonies can be verified by PCR.
<H3>PREPRARATION:</H3></LI><LI>     Liquid cultures of donor strain (DKN1005) and receptor strain (Z. mobilis ZM4)
 </LI><LI>     DAP
 </LI><LI>     0.9% salt solution
<H3>PROTOCOL FOR pLAFR5 CONJUGATION:</H3></LI><LI>     Treat RM plates made for Z. mobilis with 100 uL of DAP
 </LI><LI>     Incubate membranes on plates for 30 minutes
 </LI><LI>     Mix 750 uL of the donor strain (DKN1005) and 250 uL of the receptor strain (Z. mobilis ZM4). If the receptor strain has low density, then mix the strains 50:50.
</LI><LI>      Centrifuge for 1 minute and pour out the supernatant (This step removes the antibiotics)
</LI><LI>      Resuspend in 1 mL of RM
</LI><LI>      Repeat steps 4 and 5
</LI><LI>      Then plate 50 uL of the mixture onto the filter membranes
</LI><LI>      Incubate at 30 C
</LI><LI>      Place the filter in a falcon tube containing the 0.9% salt solution (1 mL each)
</LI><LI>      Vortex vigorously until solution is cloudy
</LI><LI>      Plate 100 uL onto RM + tet plates (no DAP on these plates causes the DKN1005 donor to die)

<H2><A name="Ethanol_Assay">Ethanol Assay</A></H2><H5>Ethanol Color Assay Reaction</H5><H4>Materials</H4></LI><LI> water
</LI><LI> 1.0M bicine (pH 8.0)
</LI><LI> 40 mM EDTA
</LI><LI> 10 mM NAD
</LI><LI> 4.2 mM thiazolyl blue
</LI><LI> 16.6 mM phenazine ethosulfate
</LI><LI> 1 mg/mL alcohol dehydrogenase enzyme
</LI><LI> ethanol
<H4>Method</H4><P> Make a master mix consisting of:</P></LI><LI> 3.6 mL water
</LI><LI> 1.2 mL bicine
</LI><LI> 1.2 mL EDTA
</LI><LI> 1.2 mL NAD
</LI><LI> 1.2 ml thiazolyl blue
</LI><LI> 2.4 mL phenazine ethosulfate
<P> Before running the experiment, add 1.2 mL of alcohol dehydrogenase enzyme. </P><P> Pipette 5 uL of sample into the 96-well microtiter plate.  Add 95 uL of master mix.  
Measure absorbance at 570 nm.  We found that the maximum absorbance of each well occurred
at a different time relative to the concentration, so it is best to run a series of measurements
over a period of approximately 4 hours. </P><H2><A name="MOPS">MOPS Buffer</A></H2><P>
We used the recipe from Teknova listed <A href="http://www.teknova.com/EZ-RICH-DEFINED-MEDIUM-KIT-p/m2105.htm"> here </A> but substituted 10 ml of 20% glucose for 8 ml of 50% glucose.  Mix all the following ingredients together to get MOPS buffer:
</P></LI><LI> 100 ml 10X MOPS Mixture </LI><LI> 10 ml 0.132 M K2HP04 </LI><LI> 100 ml 10X ACGU </LI><LI> 200 ml 5X Supplement EZ </LI><LI> 582 ml Sterile H20* </LI><LI> 8 ml 20% Glucose </LI><A style="float:right;" href="https://2012.igem.org/Team:Caltech/Notebook/material#Materials_and_Methods">Back to Materials and Methods</A><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Caltech/Notebook/material/General_Protocols">http://2012.igem.org/Team:Caltech/Notebook/material/General_Protocols</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Caltech/Notebook/material/General_Protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Caltech/Notebook/material/General_Protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Caltech/Notebook/material/General_Protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Caltech/Notebook/material/General_Protocols&amp;oldid=249741" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>