"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Caltech/Notebook/Proteorhodopsin
From 2012.igem.org
Home
Team
Mentors
Undergraduates
CalArts
Project
Overall Project
Materials & Methods
Degradation
Proteorhodopsin
Bacterial Animation
Parts
Submitted Parts
Characterization
Results
Degradation
Proteorhodopsin
Bacterial Animation
Notebook
Degradation
Proteorhodopsin
Bacterial Animation
Materials & Methods
Safety
Sponsors
Human Practice
Proteorhodopsin Notebook
6/22/12
Julia - transformed K572005 (proteorhodopsin gene from the Parts Registry)
Daisy - ordered primers for Polymerase Cycling Assembly (PCA) reaction to create a proteorhodopsin optimized for E. coli
Back to the top
6/25/12
Chenxi - Transformed pKD46 into wild type MG1655 E.coli strain using electroporation; designed primers to PCR pKD4 Kan resistance gene for lambda red knockout of nuo and ndh genes (NADH dehydrogenases)
Julia - miniprepped K572005 and sent it in for sequencing
Daisy - made 50 uL aliquots of electrocompetent E. coli cells
Edward -Ran PCR on AMP resistant back bone with annealing temperatures from 61-65 degrees Celcius
Back to the top
6/26/12
Chenxi - PCR pKD4 kan for nuo and ndh; ran gel
Julia - K572005 sequence is completely different from the one listed in the Parts Registry, but the sequence matches with a different part (contamination)
Edward- Ran low melting temp gel with the AMP resistant backbone
Samples should be 2.2kb long
BB should be shorter than insert (if they are not they need to be relabled) ---- was correct
Sample (25 microL DNA + 6 microL Dye)
5 microL ladder
Purified (2 microL DNA + .5 microL Dye)
61C 62C 63C 64C 65C Ladder BB insert
1% low melting temp gel
followed by a gel extraction
Resulted in 2ng/microL and two 5ng/microL samples ---- not good enough to do the Gibson reaction
Next will order new primers to try and have more efficient copying of backbone with AMP resistance
Gibson-bb-f Gibson suffix region 5' - TACTAGTAAACAGGGTTCTCGAGC - 3'              Melting temp 55.7
Gibson-bb-r RC from Gibson prefix region 5' - CTCTAGAAAGATCTCCGCAGCA - 3'     Melting temp 54.8
Back to the top
6/27/12
Daisy - began PCA protocol: ran PCR to make the preassembly mix, IPIPE, and VPIPE constructs; ran gel electrophoresis on the PCR products to check that the PCRs worked
Edward - Ran PCR with new primers to make backbone
Back to the top
6/28/12
Daisy - continued PCA protocol: ran PCR to make the CPEC and IPIPE (from existing IPIPE, not preassembly mix) constructs; transformed electrocompetent cells with CPEC+VPIPE (positive control; no change the protocol), CPEC-VPIPE (negative control; replaced VPIPE with sterile water), IPIPE+VPIPE (alternate positive control), and VPIPE (alternate negative control) and plated them
Katie - We ran a test of the Ethanol Assay to determine its range of effectiveness.  We measured the absorbance over time to determine how long we should run the assay to detect maximum absorbance in which ethanol was the limiting reactant.  The data is shown graphically below.
Back to the top
6/29/12
Daisy - plate results: VPIPE and CPEC-VPIPE negative controls had no colonies, IPIPE+VPIPE had only one colony, CPEC+VPIPE had many colonies; started overnight cultures of IPIPE+VPIPE and CPEC+VPIPE constructs of proteorhodopsin
Back to the top
6/30/12
Daisy - miniprepped the overnight cultures and measured their concentrations
Back to the top
7/2/12
Edward- Gel extraction for the bb on both Products from extraction mixed
Daisy - sent miniprep samples in for sequencing
Back to the top
7/3/12
Edward - transformation with product from 7/2 ligated to pSB1A3
Daisy - sequencing results returned: IPIPE+VPIPE construct is incorrect; accidentally mixed the CPEC+VPIPE sequences (will redo)
Back to the top
Katie - We ordered materials for the ethanol assay and set up experiments to grow cells anaerobically to see if ethanol could be detected.
7/4/12
Edward-PCR with Phusion on colonies
Daisy - regrew 3 CPEC+VPIPE overnight cultures and made glycerol stocks
Back to the top
7/5/12
Daisy - miniprepped the CPEC+VPIPE cultures, measured their concentrations, and sent them in for sequencing
Back to the top
7/6/12
Edward - sequencing unsuccessful, ordered new primers
Back to the top
7/10/12
Katie - We started an aerobic and an anaerobic liquid culture of E. coli cells so that we could measure ethanol produced the next day.
Back to the top
7/10/12
Edward - ran pcr with new primers 53C annealing temp (unsuccessful)
Katie - We used a plate reader to measure the ethanol concentrations but found both anaerobic and aerobic cultures produced elevated ethanol levels.  This could have been because the cells grew too quickly and many were forced into anaerobic conditions in the aerobic culture.  The OD (optical density) level of the aerobic culture was elevated. Back to the top
7/13/12
Katie - We decided to try a few new ways to decrease aerobic ethanol yields and decided to try the experiment in flasks in a shaking incubator.
Back to Biofuel Notebook Calendar
7/15/12
Edward - reran pcr with primers at 60C - successful
Back to the top
7/17/12
Katie - We made additional MOPS buffer and then set our aerobic culture experiment to incubate in a shaker overnight.
Back to Biofuel Notebook Calendar
7/18/12
Katie - We ran an ethanol assay on our aerobic cultures.  We found that the cells had overgrown in the aerobic culture tubes and were likely producing ethanol due to forced anaerobic conditions.  We also noticed our ethanol standard curve was becoming less linear.
Back to Biofuel Notebook Calendar
7/19/12
Katie - We decided to do the ethanol assay again, this time in aerated flasks, in order to decrease the chance of anaerobic growth.  The cultures were put in MOPS media again.
Back to Biofuel Notebook Calendar
7/20/12
Edward - ligated and transformed using proteorhodopsin without retinal pathway
Katie - Since the cultures once again showed high ethanol levels after the ethanol assay, we prepared a timepoint assay for the next day.  We put e. coli in flasks with 20 ml MOPS and then took OD measurements at 3 hour timepoints to determine when ethanol production was low.  We took timepoints at 1.30, 3.15, 6.30, and 22.15 from the start of the culture, and used two different colonies in two different flasks.  The results are below. Back to the top
7/21/12
Edward - colony pcr on cells
Back to the top
7/25/12
Edward - received successful results from sequencing
Back to the top
7/30/12
Daisy - started Z. mobilis and E. coli liquid cultures in Rich Media or Mops Media in aerobic or anaerobic conditions
Back to the top
8/4/12
Daisy - measured optical density (OD) of all Rich Media cultures, took and froze stocks of their supernatants
Back to the top
8/7/12
Daisy - measured optical density (OD) of all Mops Media cultures, took and froze stocks of their supernatants; the colonies grew much more slowly in Mops than in Rich Media
Back to the top
8/17/12
Katie - We ran a standard curve of the ethanol assay to see where it became nonlinear and found that instead of modeling it linearly, it would be better to model with a logarithmic curve.
Back to the top
9/21/12
Katie - We unsuccessfully tried to transform the proteorhodopsin with a ribosome binding site.
Back to the top
9/25/12
Katie - We ran a characterization assay of the proteorhodopsin strain.  The results are on the characterization page.
Back to the top
Retrieved from " http://2012.igem.org/Team:Caltech/Notebook/Proteorhodopsin "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
