<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_East_Chapel_Hill_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:East Chapel Hill/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="nav-container"><DIV class="brand"><A rel="nofollow" rel="noreferrer" href="https://2018.igem.org/Team:East_Chapel_Hill">East Chapel Hill</A></DIV><NAV><UL class="nav-list"><LI><A rel="nofollow" rel="noreferrer" href="#!">Team</A></LI><LI><A href="https://2018.igem.org/Team:East_Chapel_Hill/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:East_Chapel_Hill/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:East_Chapel_Hill/Attributions">Attributions</A></LI></UL><LI><A rel="nofollow" rel="noreferrer" href="#!">Project</A></LI><LI><A href="https://2018.igem.org/Team:East_Chapel_Hill/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:East_Chapel_Hill/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:East_Chapel_Hill/Measurement">Measurement</A></LI><LI><A rel="nofollow" rel="noreferrer" href="#!">Lab</A></LI><LI><A href="https://2018.igem.org/Team:East_Chapel_Hill/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:East_Chapel_Hill/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:East_Chapel_Hill/Notebook">Notebook</A></LI><LI><A rel="nofollow" rel="noreferrer" href="https://2018.igem.org/Team:East_Chapel_Hill/Parts">Parts</A></LI><LI><A rel="nofollow" rel="noreferrer" href="#!">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:East_Chapel_Hill/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:East_Chapel_Hill/Public_Engagement">Public Engagement</A></LI></NAV></DIV><DIV class="navbar"><DIV class="navbar-inner"><A href="#Introduction">Introduction</A><A href="#Solution">Solution</A><A href="#Our Design">Our Design</A></DIV></DIV><H1>Description</H1><H1>Introduction</H1><P2 style="font-size:18px;"> Granite and volcanic rocks are extremely high in fluoride due to large amounts of fluoride-rich minerals including biotite, fluorite, amphibole, and apatite. These high-fluoride rock deposits rise through faults and hot springs into groundwater. Prolonged exposure to high levels of fluoride correlates to diseases such as dental and skeletal fluorosis. These diseases can severely impact young children, whose enamel is still developing. Please see our interview with <A href="https://2018.igem.org/Team:East_Chapel_Hill/MaikoSuzuki">Maiko Suzuki</A> to learn specifically how fluorosis manifests in the teeth. 

Unfortunately, mitigating fluoride problems has proven to be very expensive and challenging. One of the issues we are attempting to address with our project is diligently tracking fluoride concentrations after treatment attempts. In rural communities, even once there has been treatment to high-fluoride water, it is difficult to monitor fluoride concentrations after the treatment. 
 
We hope that the operon we have developed may assist the monitoring of fluoride concentrations in small, low-technology villages after treatment of the water has been administered. 
</P2><H1>Solution</H1><P2 style="font-size:18px;">
	The previously developed Chloramphenicol Acetyltransferase Operon (CHOP) by the 2017 East Chapel Hill iGem team was our first attempt in creating an accessible device that may serve as a visual indicator of fluoride in water. This year, we tested a series of promoters and riboswitch constructs to determine which are conducive to an operon with highest binding ability to fluoride. We were successful in being able to alter the previous CHOP operon so that it could detect concentrations of fluoride as low as 100uM. 

		</P2><H2 style="text-align: left;"> About the Riboswitch</H2><P2 style="font-size:18px;">A riboswitch is a segment of messenger RNA that is able to control gene expression by selectively binding to certain ligands. Riboswitches have 2 main domains: the aptamer and expression. The aptamer primarily serves as a receptor for specific ligands to bind to. Meanwhile, the expression may switch between 2 secondary structures, controlling gene expression.  

Riboswitches may be translational or transcriptional. A transcriptional riboswitch has a “switching sequence” in the aptamer that directs the formation of a transcriptional terminator, which signals to RNA polymerase to stop transcription. One may think of this process as an “on” or “off” switch, with “on” allowing for transcription of a gene. 
When the aptamer (ligand-binding) region of the fluoride riboswitch interacts with fluoride, the terminator is not formed allowing the RNA polymerase to proceed and transcribe the downstream gene. 
		</P2><CENTER><FIGURE><FIGCAPTION>Figure 2: Schematic of a transcriptional riboswitch<FONT size="2">2015 Exeter iGEM Team, RNA Riboswitches</FONT></FIGCAPTION></FIGURE></CENTER><P2 style="font-size:18px;">
	The riboswitch we used is from the previously characterized <I>B. cereus</I>. In Figure 3 we have displayed the crystal structure of the aptamer of this riboswitch. We tested two variations of this riboswitch, which we labeled FRS1 and FRS2. Figure 4a shows the predicted folding structure of FRS1, and Figure 4b shows the predicted folding structure of FRS2. We were interested in determining how the predicted folding structure may influence the binding ability of this riboswitch to fluoride. 

In nature, this riboswitch regulates the expression of genes that are able to pump high levels of fluoride out of the cell. The <I>crcB</I> gene in <I>E.coli</I> bacteria encodes the fluoride efflux channel, which is capable of pumping fluoride out of the cell so that it is no longer toxic. In our experiments, we used a modified <I>crcB</I><I>E.coli</I> strain so that fluoride may accumulate in the cell. 
	</P2><CENTER><FIGURE><FIGCAPTION><I> Figure 4a: Predicted folding structure of fluoride riboswitch &quot;FRS1&quot; </I></FIGCAPTION></FIGURE></CENTER><CENTER><FIGURE><FIGCAPTION><I>Figure 4b: Predicted folding structure of fluoride riboswitch &quot;FRS2&quot;  </I></FIGCAPTION></FIGURE><CENTER><FIGURE style="width:50%;"><FIGCAPTION>Figure 3: Crystal structure of a fluoride riboswitch <FONT size="2">Aiming Ren, Kanagalaghatta R. Rajashankar, Dinshaw J. Patel “Fluoride ion encapsulation by Mg2+ ions and phosphates in a fluoride riboswitch” 2012 Nature 486, 85–89 

		</FONT></FIGCAPTION></FIGURE></CENTER><H1> Our Design </H1><P2 style="font-size:18px;"><P2 style="text-align:left;" font-size:18px=""> 
We modified the previously developed chloramphenicol acetyltransferase operon (CHOP) by the 2017 East Chapel Hill iGem. We used Gibson overhangs with homology to pSB1A3 so we could clone the operon into the pSB1A3 vector. This operon was designed so that future users may easily test a library of promoters and riboswitches simply by cutting with restriction enzyme HindIII. One may even test the expression of a new gene by using the XhoI restriction enzyme. 
</P2><CENTER><FIGURE><FIGCAPTION><I>Schematic of operon BBa_K2843000</I></FIGCAPTION></FIGURE></CENTER><P2 style="text-align:left;" font-size:18px=""> 
The fluoride binding mutant has two point mutations that prevent the antiterminator loop from forming. Therefore, fluoride can’t bond and there should be no growth. This acts as a control to verify that bacterial growth is directly a result of fluoride concentrations. 
<FIGCAPTION><I>The figure above shows the conserved fluoride riboswitch</I><FIGCAPTION><I>The figure above shows the 2 point mutation of the fluoride riboswitch, creating the fluoride binding mutant</I></FIGCAPTION></FIGCAPTION></P2></P2></CENTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>