<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Florida_Atlantic_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Florida Atlantic/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2017_menu_wrapper"><DIV class="menu_button" id="display_menu_control">
		MENU  
	</DIV><DIV id="menu_content"><A href="https://2017.igem.org/Team:Florida_Atlantic"><DIV class="menu_button direct_to_page">
				HOME
			</DIV></A><DIV class="menu_button"> TEAM 
		</DIV><DIV class="submenu_wrapper" id="team_submenu"><A href="https://2017.igem.org/Team:Florida_Atlantic/Team"><DIV class="submenu_button" id="Team_page">
					Team 
				</DIV></A><A href="https://2017.igem.org/Team:Florida_Atlantic/Acknowledgements"><DIV class="submenu_button" id="Acknowledgements_page">
					Acknowledgements
				</DIV></A></DIV><DIV class="menu_button"> PROJECT
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Florida_Atlantic/Description"><DIV class="submenu_button" id="Description_page">
					Description
				</DIV></A><A href="https://2017.igem.org/Team:Florida_Atlantic/Design"><DIV class="submenu_button" id="Design_page">
					Design
				</DIV></A><A href="https://2017.igem.org/Team:Florida_Atlantic/Experiments"><DIV class="submenu_button" id="Experiments_page">
					Experiments
				</DIV></A><A href="https://2017.igem.org/Team:Florida_Atlantic/InterLab"><DIV class="submenu_button" id="InterLab_page">
					InterLab
				</DIV></A><A href="https://2017.igem.org/Team:Florida_Atlantic/Model"><DIV class="submenu_button" id="Model_page">
					Model
				</DIV></A><A href="https://2017.igem.org/Team:Florida_Atlantic/Results"><DIV class="submenu_button" id="Results_page">
					Results
				</DIV></A><A href="https://2017.igem.org/Team:Florida_Atlantic/Attributions"><DIV class="submenu_button" id="Attributions_page">
					Attributions 
				</DIV></A><A href="https://2017.igem.org/Team:Florida_Atlantic/Pictures"><DIV class="submenu_button" id="Pictures_page">
					Pictures
				</DIV></A></DIV><DIV class="menu_button"> HUMAN PRACTICES 
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Florida_Atlantic/HP/Silver"><DIV class="submenu_button" id="Silver_page">
					Silver HP
				</DIV></A><A href="https://2017.igem.org/Team:Florida_Atlantic/HP/Gold"><DIV class="submenu_button" id="Gold_page">
					Gold HP
				</DIV></A></DIV><DIV class="menu_button"> AWARDS
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Florida_Atlantic/Model"><DIV class="submenu_button" id="Model_page">
					Model
				</DIV></A><A href="https://2017.igem.org/Team:Florida_Atlantic/Software"><DIV class="submenu_button" id="Software_page">
					Software
				</DIV></A></DIV><A href="https://2017.igem.org/Team:Florida_Atlantic/Safety"><DIV class="menu_button direct_to_page">
				SAFETY
			</DIV></A><A href="https://igem.org/2017_Judging_Form?team=Florida_Atlantic"><DIV class="menu_button direct_to_page">
				JUDGING FORM
			</DIV></A></DIV></DIV><DIV class="igem_2017_content_wrapper"><H1>Florida_Atlantic</H1><FORM><FIELDSET><DIV class="column full_size"><DIV class="column full_size"><CENTER><H1 style="font-size: 26px">Experiments</H1></CENTER><H2>Wet Lab Protocols</H2><P style="font-size: 18px">
LB Broth
25g LB Broth, Miller
1L Water
-Mix LB powder into water, autoclave for 15 minutes. Supplement with antibiotics
(Chloramphenicol, 35μL/mL) as needed.
LB Agar
25g LB Broth, Miller
15g Granulated Agar
1L Water
-Mix LB powder and Agar into water. Heat until agar is dissolved and solution is clear, avoid
boiling over. Autoclave for 15 minutes. Supplement with antibiotics (Chloramphenicol,
35μL/mL) as needed. Pour 20mL into sterile culture plates and let cool.</P><H3>Transformation of Competent Cells </H3><P style="font-size: 18px">
50μL Competent Cells (DH5α E. coli)
10ng Plasmid DNA
450μL LB
LB/Chloramphenicol Plate
-Thaw Competent cells on ice and add plasmid DNA. Let sit for 20 minutes on ice. Heat shock
cells at 45 o C for 30 seconds and then return to ice for 2 minutes. Add LB and incubate at 37 o C
for 2-3 hours. Plate 100μL of the cells on a LB/Chloramphenicol plate and incubate at 37 o C
overnight.</P><H3>Dry Lab Protocols</H3><P style="font-size: 18px">
Protein Reverse Translation
-Isolate the protein sequence of interest and reverse translate using the E. coli preferred codon
library in SnapGene. After reverse translation, look for out-of- frame coding regions and alter the
codons so that no transcription is likely to occur. Finally, run a BLASTX protocol to ensure that
the nucleotide sequence still encodes the protein of interest.</P><H2>Machine Learning Protocols</H2><P style="font-size: 18px">
LSTM model was coded using Tensorflow library and imported to Jupyter notebooks for 3 main experiments.</P><H4>Artemisinin Binding</H4><P style="font-size: 18px">
- Created LSTM model. 
- Collected and imported positive dataset for proteins that bind to Artemisinin and negative dataset for proteins that do not bind to Artemisinin by hand from NCBI and literature.
- Train model to learn binding vs. not binding.
- Established consistent parameters for learning process (number of proteins per dataset, learning rate, network size, and sub-sequence length viewed). 
- Ran test and repeated to establish consistency.</P><H4>Artemisinin Consensus Sequence</H4><P style="font-size: 18px">
- Created LSTM. 
- Imported dataset (from previous model^). 
- Trained model to look at specified sequence length for binding vs. not binding. 
- Established consistent parameters for learning process (number of proteins per dataset, learning rate, network size, and sub-sequence length viewed). 
- Set variables consistent (batch size, epochs, and sequence length to run in range loop).
- Ran test and repeated to establish consistency.</P><H4>Homeobox Consensus Sequence</H4><P style="font-size: 18px">
- Created LSTM. 
- Imported dataset from Uniprot data website for proteins containing homeo-domain sequence and proteins and proteins not containing homeodomain sequence. 
- Trained model to look at specified sequence length for binding vs. not binding. 
- Established consistent parameters for learning process (number of proteins per dataset, learning rate, network size, and sub-sequence length viewed). 
- Set variables consistent (batch size, epochs, and sequence length to run in range loop). 
- Ran test and repeated to establish consistency.</P></DIV></DIV></FIELDSET></FORM></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>