"
Page
Discussion
View source
History
teams
Log in
 
Team:uOttawa CA/Project
From 2012.igem.org
Assembly
Constructs
Networks
Comprehensive Approach to Universal Network Design
The goal of the 2012 uOttawa iGEM team was to continue to work off of the strengths of our 2011 project, which was the construction of gene networks. This year we showcase the benefits of using yeast as a model organism for network construction, characterization and expression. Saccharomyces cerevisiae is a eukaryotic organism that has many physiological properties which makes it superior to E. coli for synthetic biology. The ability to exist in both haploid and diploid states can be used to build networks through mating protocols and its repair mechanisms can be used for a robust assembly of gene networks in vivo.
Network Construction
The first task of building a gene network is to synthesize the sequences that are required for the network. Last year, the uOttawa project showed how PCR could be used to put together many gene dimers (ex. A-B, B-C, C-D) as long as there were homologous regions that could prime the reaction. This method worked very well but we continued to improve our protocols to increase efficiency and reduce the amount of time required to construct networks. We adopted the DNA assembler method [1], and optimized it for our purposes. This protocol takes advantage of yeast repair mechanisms and uses homologous recombination to put together networks. By building gene fragments with homologous regions and co-transforming them in yeast we reduced the amount of PCR required to build gene networks. Dimers of gene fragments can be built using traditional BioBrick assembly methods or fusion PCR and co-transformed as per the protocol supplied on our results page.
Once the gene constructs were assembled and transformed into yeast we were able to take advantage of yeast mating protocols to build larger networks. This allows for the rapid integration of constructs to build large networks without the time commitment of multiple transformation procedures. For example, simultaneously making haploid strains of two constructs each, can build a gene network composed of four constructs once mated together. Furthermore, if one gene module is required in multiple networks it can be built into one mating type haploid (a or α) and mated with many different gene modules that have been integrated into the opposite mating type.
Promoter Design
A current problem with characterizing synthetic networks is the lack of ability to externally control systems. In S. cerevisiae, a few options exist such as the copper-inducible CUP1 promoter and the galactose-inducible GAL1 promoter. The GAL1 promoter has been extensively quantified and researched, and thus makes for an ideal part in many synthetic networks. However due to some major limitations, it cannot be used efficiently in vivo. The GAL1 promoter contains an upstream activation sequence (UAS) with GAL4 binding sites that are activated in the presence of galactose. The problem is that since galactose is also a metabolite, yeast will preferentially use it as a source of energy.
The biggest challenge with the GAL1 promoter is that 3 of the 4 gal4 sites are in tandem, and one site is 100 bp downstream. This makes it extremely difficult to incorporate repeats that are so close together due to hairpins when synthesizing genes. Also assembly by PCR would be difficult due to multiple overlapping possibilities of the primers.
To this end, we have designed a protocol modified from Gibson 2009 which, in conjunction with our co-transformation method, will allow incorporation of novel operator sites into the GAL1 promoter. We designed a system of primers comprising the new GAL1 promoter with tet operator sites that had homology to a linearized PRS416 vector, a plasmid that contains the ura3 gene for uracil production and the ampicillin resistance cassette. Using this method, we co-transformed the plasmid and the primers into yeast and plated on 2x YPD lacking uracil to take advantage of the organism's repair mechanism. The circularized plasmid would allow successful transformants to produce uracil and screen out any unsuccessful ones.  The yeast was then incubated for 2 days followed by overnight growth of multiple yeast colonies in one tube. We followed a DNA isolation protocol to extract the plasmid DNA. This DNA was then transformed into competent E. coli cells and plated on ampicillin in order to obtain significant copies of our desired product. After a 1-day incubation, E. coli colonies were picked, grown overnight, and mini-prepped the next day.
Reference Strains
A large roadblock on the construction of large gene networks has been the predictability of the components. In order to construct a network with desired function, the parts need to first be characterized and compared to a standard reference strain. The reference strain acts to set a basal expression rate for testing strains and other inducible systems to compare their expression levels to.
Characterization of Network Components
Instead of creating new BioBricks of different transcription factors, we decided to continue with the characterization of the Tet-BFP (BBa_K642000) that we submitted last year to improve upon the documentation of this part. By improving the characterization data we hope to provide more information to the synthetic biology community so the part can be used in a very predictable fashion.
To characterize transcription factors and their responsive promoters a feed forward cascade as shown in Figure 2 was built. The network was controlled via an inducible activator (GEV), which was driven by a constitutive promoter (mrp7). This activator consists of the Gal4 DNA Binding Domain, Human Estrogen Receptor and VP16 activation domain. The activator is inducible with the addition of β estradiol, which causes the human estrogen receptor to dimerize, and enables the proper binding of the activator on the DNA [2]. The amount of Tet repressor produced is directly proportional to the BFP produced, which is detectable. The effect of the Tet repressor can be correlated to the GFP expression from the Tet responsive promoter. The expression of the reporters is detectable with a standard Flow cytometer. The entire network was built in both haploid and diploid forms of yeast, which allows us to determine whether the state of the cell has an effect on repression.
The same network as described above was also built in a diploid cell without the repressor being tagged. Due to increasing evidence that tagging transcription factors effects their expression due to the increase in protein size, we separated the reporter from the repressor. Both the reporter and repressor are approximately the same size, are expressed by the same promoter and can be found in the same locus. By correlating the BFP expression of the tagged Tet repressor from Figure 2 with the BFP expression of the untagged Tet repressor from Figure 4 we can determine whether the presence of reporter tags effects transcription factor expression levels.
Expression of Gene Networks in E. coli
By building an E.coli/S.cerevisiae shuttle vector we can take advantage of the high reproductive rate  of E.coli and the gene synthesis capabilities of S.cerevisiae. Networks will be built via homologous recombination in yeast and replicated in bacteria. Traditional drug selection will be supplemented with colour selection to increase the accuracy of the transformations.
References
[1] Shao, Z., Zhao, H., & Zhao, H. (2009). DNA assembler, an in vivo genetic method for rapid construction of biochemical pathways. Nucleic acids research, 37(2), e16. doi:10.1093/nar/gkn991
[2] Louvion, J. F., Havaux-Copf, B., & Picard, D. (1993). Fusion of GAL4-VP16 to a steroid-binding domain provides a tool for gratuitous induction of galactose-responsive genes in yeast. Gene, 131(1), 129-34. Retrieved from http://www.ncbi.nlm.nih.gov/pubmed/8370533
[3] Gibson, D. G. (2009). Synthesis of DNA fragments in yeast by one-step assembly of overlapping oligonucleotides. Nucleic acids research, 37(20), 6984-90. doi:10.1093/nar/gkp687
Retrieved from " http://2012.igem.org/Team:uOttawa_CA/Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
