<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU_Eindhoven_Parts skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU Eindhoven/Parts</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="menuwrapper"><DIV id="menuTotal"><DIV id="menu"><A id="menu1" href="https://2015.igem.org/Team:TU_Eindhoven">
        Home</A><A id="menu2" href="https://2015.igem.org/Team:TU_Eindhoven/Team"> 
        Team</A><A id="menu3" href="https://2015.igem.org/Team:TU_Eindhoven/Project"> 
        Project</A><A id="menu4" href="https://2015.igem.org/Team:TU_Eindhoven/ResultsHome">
        Results</A><A id="menu5" href="https://2015.igem.org/Team:TU_Eindhoven/Modeling">
        Modeling</A><A id="menu6" href="https://2015.igem.org/Team:TU_Eindhoven/Practices">
        Policy Practices</A><A id="menu7" href="https://2015.igem.org/Team:TU_Eindhoven/Collaborations">
        Collaborations</A><A id="menu8" href="https://2015.igem.org/Team:TU_Eindhoven/Team/Sponsors">
        Sponsors</A><A id="menu9" href="https://2015.igem.org/Team:TU_Eindhoven/Safety">
        Safety</A></DIV></DIV><DIV id="submenuwrapper"><DIV id="submenu"><DIV id="sub2"><A id="sub2a" href="https://2015.igem.org/Team:TU_Eindhoven/Team/Members">Team members</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Team/Supervisors">Supervisors</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Attributions">Attributions</A></DIV><DIV id="sub3"><A id="sub3a" href="https://2015.igem.org/Team:TU_Eindhoven/Description">Description</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Project/Design">Design</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Project/Expression_System">Expression System</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Project/Experimental_Approach">Experimental Approach</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Project/Protocols">Protocols</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Notebook">Timeline</A></DIV><DIV id="sub4"><A id="sub4a" href="https://2015.igem.org/Team:TU_Eindhoven/Parts">Parts</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Results">Results</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Achievements">Medal Motivation</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Measurement">Interlab Study</A></DIV><DIV id="sub5"><A id="sub5a" href="https://2015.igem.org/Team:TU_Eindhoven/Modeling/Approach">Approach</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Modeling/Script">Script</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Modeling/Results">Results</A></DIV><DIV id="sub6"><A id="sub6a" href="https://2015.igem.org/Team:TU_Eindhoven/Policy_Practices/Human_Outreach">Outreach</A><A style="width:12%;" href="https://2015.igem.org/Team:TU_Eindhoven/Policy_Practices/Application_scenarios">Application Scenarios</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Policy_Practices/Stakeholders">Stakeholders</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Policy_Practices/Regulations">Regulations</A></DIV><DIV id="sub7"><A id="sub7a" href="https://2015.igem.org/Team:TU_Eindhoven/Collaborations/CloningGuide">Cloning Guide</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Collaborations/Collaboration_Attributions">Collaboration Attributions</A></DIV><DIV id="sub9"><A id="sub9a" href="https://2015.igem.org/Team:TU_Eindhoven/Safety/Lab">Lab Safety</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Safety/Project">Project Safety</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Safety/LabEquipment">Lab Equipment</A></DIV></DIV></DIV></DIV><DIV id="body"><DIV id="container"><DIV id="wikiTour"><A style="text-decoration: none" href="https://2015.igem.org/Team:TU_Eindhoven/Results"><DIV id="nextChapter"><SPAN class="tekst1BI">Next Chapter</SPAN><SPAN class="tekst1">
Find out what results we obtained with our COMB Prototypes over the summer.</SPAN></DIV></A></DIV><DIV id="containercontent"><DIV id="landingPage"><H1>
Parts
</H1><SPAN class="tekst1"><UL class="activiteitlijst"><LI class="activiteit"><H6> 
Outer Membrane Protein X (OmpX)
</H6><H7> - <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1761000" target="_blank">BBa_K1761000</A></H7><DIV class="spoiler" id="spoiler1"><H7>Quick facts:</H7>
Length: 513 bp 
Weight: 20kDa 
Diameter of the beta barrel: 2 nm <H7>Usage and biology</H7><DIV id="imageText"><DIV class="left5">
OmpX, or Outer Membrane Protein X, belongs to a family of highly conserved bacterial proteins that promote bacterial adhesion to and entry into mammalian cells. It presents both C- and N-termini in the intracellular domain (see Figure 1). OmpX consists of an eight-stranded antiparallel all-next-neighbor β barrel. The core of the protein consists of an extended hydrogen-bonding network of highly conserved residues (see Figure 1) <A name="reft1" href="#ref1" class="textanchor">[1]</A>.
</DIV><DIV class="right5"><SPAN class="caption">
Figure 1: A) The OmpX protein structure has been elucidated through NMR and X-ray crystallography. B) The square residues are important for the secondary structure of OmpX. To keep the structure intact, we introduce an amber stop codon in one of the protruding loops. Figure 1B has been adapted from <A name="reft1" href="#ref1" class="textanchor">[1]</A>.
</SPAN></DIV></DIV><H7>Gene Design</H7>
OmpX was identified and characterized by Mecsas et al and is 513 bp long. It contains a putative signal sequence, which makes sure that OmpX will be in the membrane <A name="reft2" href="#ref2" class="textanchor">[2]</A>. To be able to use OmpX as a scaffold, you need to integrate the unnatural amino acid. To do so, some modifications must be made to this part. Within the original OmpX part, a codon was mutated into the amber stop codon TAG. This codon will be incorporated in the protruding loops of OmpX, loop 2 and loop 3 (see Figure 1). These loops were chosen because they are easily accessible and are not part of the beta-barrel of OmpX <A name="reft3" href="#ref3" class="textanchor">[3]</A>. We believe that mutations in these loops will not disturb the secondary structure of OmpX. With a specific tRNA it is possible to implement a unnatural amino acid at the place of the TAG codon. For more information on which specific mutations we made, see our Project Design page.

To be able to use OmpX in a sensor system, we attached several intracellular domains, namely NanoLuc, mNeonGreen, mTurquoise, LargeBiT and SmallBiT. These domains can interact with each other in different ways. NanoLuc and mNeonGreen form a BRET pair, mTurquoise and mNeonGreen a FRET pair and LargeBiT and SmallBiT a luciferase. These constructs were attached to OmpX with a linker and were inserted in a pETDuet-1 vector that has 2 multiple cloning sites (MCS-1 and MCS-2).

<H7>Sequence</H7>

The sequence of our OmpX part has been verified by sequencing at StarSeq.
<DIV class="spoiler" id="spoiler6"><SPAN class="dnaText">
5'-ATG AAA AAA ATT GCA TGT CTT TCA GCA CTG GCC GCA GTT CTG GCT TTC ACC GCA GGT ACT TCC GTA GCT GCG ACT TCT ACT GTA ACT GGC GGT TAC GCA CAG AGC GAC GCT CAG GGC CAA ATG AAC AAA ATG GGC GGT TTC AAC CTG AAA TAC CGC TAT GAA GAA GAC AAC AGC CCG CTG GGT GTG ATC GGT TCT TTC ACT TAC ACC GAG AAA AGC CGT ACT GCA AGC TCT GGT GAC TAC AAC AAA AAC CAG TAC TAC GGC ATC ACT GCT GGT CCG GCT TAC CGC ATT AAC GAC TGG GCA AGC ATC TAC GGT GTA GTG GGT GTG GGT TAT GGT AAA TTC CAG ACC ACT GAA TAC CCG ACC TAC AAA CAC GAC ACC AGC GAC TAC GGT TTC TCC TAC GGT GCG GGT TTG CAA TTC AAC CCG ATG GAA AAC GTT GCT CTG GAC TTC TCT TAC GAG CAG AGC CGT ATT CGT AGC GTT GAC GTA GGC ACC TGG ATT GCC GGT GTT GGT TAC CGC TTC -3' 
</SPAN></DIV><DIV id="imageText"><DIV class="left5">
This BioBrick Part looks like Figure 2. It contains the prefix and suffix with the correct restriction sites (EcoRI, XbaI, SpeI and PstI). OmpX itself is 513 bp long.
<H7>Characterization</H7>
When mutated with the amber stop codon TAG, a unnatural amino acid with an azide-functionalized group can be expressed. After the expression of this amino acid, OmpX can covalently bind almost anything, as long as it contains a DBCO-functionalized group. The binding finds place by using a bio-orthogonal “click” reaction (SPAAC click chemistry). 

</DIV><DIV class="right5"><SPAN class="caption">
Figure 2: SnapGene figure of the OmpX Part.
</SPAN></DIV></DIV>

To test the functionality of this “click” reaction, some experiments were done by clicking DBCO-PEG4-TAMRA at the surface. 
For all the experiments we used the following vectors: pETDuet-1 with one or two construct(s) inserted (OmpX + intracellular protein) and pEVOL-pAzF (tRNA + tRNA synthetase). Both vectors were transformed into BL21(DE3). The expression was introduced by adding arabinose, IPTG and the unnatural amino acid.

<H7> DBCO-PEG4-TAMRA Confirmation</H7>
To confirm whether OmpX is in the membrane and whether or not the unnatural amino acid is being incorporated into OmpX, DBCO-PEG4-TAMRA was used. TAMRA is a fluorescent dye that can be used to verify the “click” reaction. If the unnatural amino acid is present, DBCO-PEG4-TAMRA should “click” to the transmembrane protein OmpX and stay there. This can be analyzed with FACS. For more information about how to perform FACS experiments, see our <A href="https://2015.igem.org/Team:TU_Eindhoven/Project/Protocols" target="_blank">Protocol Page</A>.

To verify that OmpX is in the membrane, we used the OmpX – NanoLuc and OmpX – mNeonGreen constructs. These gave the following results after clicking with DBCO-PEG4-TAMRA (see Figure 3 and 4). From this it can be concluded that OmpX is in the membrane and that the “click” reaction works. 
<DIV id="imageText"><DIV id="imageText"><DIV class="columnleft"><SPAN class="caption">
Figure 3: FACS results of OmpX – NanoLuc.
</SPAN></DIV><DIV class="columnright"><SPAN class="caption">
Figure 4: FACS results of OmpX – mNeonGreen.
</SPAN></DIV></DIV><DIV class="references"><SPAN class="caption"><A href="#reft1" name="ref1">[1]</A>	Vogt J. and Schulz G.E., “The structure of the outer membrane protein OmpX from Escherichia coli reveals possible mechanisms of virulence”, Structure, vol. 7, no. 10, pp. 1301–9, Oct. 1999.<A href="#reft2" name="ref2">[2]</A>	Mescas J., Welch R., Erickson J.W. and Gross C.A., “Identification and characterization of an Outer Membrane Protein, OmpX, in Escherichia coli that is homologous to a family of outer membrane proteins including ail of Yersinia enterocolitica”, Journal of Bacteriology, vol. 177, no. 3, pp. 799-804, Nov. 1994. <A href="#reft3" name="ref3">[3]</A>	Rice J.J., Schohn A., Bessette P.H., Boulware K.T. and Daugherty P.S., “Bacterial display using circularly permuted outer membrane protein OmpX yields high affinity peptide ligands”, Protein Sci., vol. 15, no. 4, pp. 825–36, Apr. 2006.</SPAN></DIV></DIV></DIV></LI><LI class="activiteit"><H6> Outer Membrane Protein X (OmpX) with BamHI-linker and mNeonGreen
</H6><H7> - <A href="http://parts.igem.org/Part:BBa_K1761001" target="_blank">BBa_K1761001</A></H7><DIV class="spoiler" id="spoiler2"><H7>Quick facts:</H7>
Total length: 1439 bp 
BamHI-linker length: 213 bp 
mNeongreen length: 711 bp 
Weight: 53kDa <H7>Usage and biology</H7><DIV id="imageText"><DIV class="left5">
OmpX, or Outer Membrane Protein X, belongs to a family of highly conserved bacterial proteins that promote bacterial adhesion to and entry into mammalian cells. It presents both C- and N-termini in the intracellular domain (see Figure 5). OmpX consists of an eight-stranded antiparallel all-next-neighbor β barrel. The core of the protein consists of an extended hydrogen-bonding network of highly conserved residues (see Figure 5) <A name="reft4" href="#ref1" class="textanchor">[4]</A>.
</DIV><DIV class="right5"><SPAN class="caption">
Figure 5: A) The OmpX protein structure has been elucidated through NMR and X-ray crystallography. B) The square residues are important for the secondary structure of OmpX. To keep the structure intact, we introduce an amber stop codon in one of the protruding loops. Figure 5B is adapted from <A name="reft1" href="#ref1" class="textanchor">[4]</A>.
</SPAN></DIV></DIV><DIV id="imageText"><DIV class="right6">
mNeonGreen is a yellow-green fluorescent protein and is derived from a tetrameric fluorescent protein from cephalochordate Branchiostoma lanceolatum. mNeonGreen is the brightest monomeric green or yellow fluorescent protein yet described and is an excellent fluorescence resonance energy transfer (FRET) acceptor for the newest cyan fluorescent proteins <A name="reft5" href="#ref5" class="textanchor">[5]</A>.
<H7>Gene Design</H7>
This construct consists of three parts, namely OmpX, a BamHI-linker, and mNeonGreen. OmpX is already described as a single part, and that same part, with the amber stop codon TAG incorporated for the integration of the unnatural amino acid, is used for this construct. For more information, see our <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1761000" target="_blank">Part OmpX</A>. 
mNeonGreen is connected to OmpX with a BamHI-linker. This linker is a 213 bp long flexible GGSGGS linker and by using the restriction enzyme BamHI, the linker can become 45 bp shorter.
mNeonGreen was characterized by Shaner et al <A name="reft5" href="#ref5" class="textanchor">[5]</A>. The BamHI-linker is inspired by the article of T.H. Evers et al <A name="reft6" href="#ref6" class="textanchor">[6]</A>.
</DIV><DIV class="left6"><SPAN class="caption">
Figure 6: Schematical overview of the OmpX – mNeonGreen Part.
</SPAN></DIV></DIV>
These three parts together give a construct with OmpX as transmembrane protein  and mNeonGreen as intracellular domain   (see Figure 6)
<DIV id="imageText"><DIV class="columnleft">
For all our experiments we used pETDuet-1 as a vector. This vector has two multiple cloning sites, namely MCS-1 and MCS-2. To be able to use this part as a sensor, we inserted both OmpX – NanoLuc and OmpX – mNeonGreen into this vector, both in different multiple cloning sites. Both multiple cloning sites were expressed. In this way, NanoLuc and mNeonGreen can function as a BRET pair (see Figure 7).
</DIV></DIV><DIV id="imageText"><DIV class="columnright"><SPAN class="caption">
Figure 7: Schematical overview of the BRET pair.
</SPAN></DIV></DIV><H7>Sequence</H7>

The sequence of our OmpX - mNeonGreen part has been verified by sequencing at StarSeq.
<DIV class="spoiler" id="spoiler7"><SPAN class="dnaText">
5'- ATG AAA AAA ATT GCA TGT CTT TCA GCA CTG GCC GCA GTT CTT GCT TTC ACC GCA GGT ACT TCC GTA GCA GCG ACT TCT ACT GTA ACT GGC GGT TAC GCA CAG AGC GAC GCT CAG GGC CAA ATG AAC AAA ATG GGC GGT TTC AAC CTG AAA TAC CGC TAT GAA GAA GAC AAC AGC CCG CTG GGT GTG ATC GGT TCT TTC ACT TAC ACC GAG AAA AGC CGT ACT GCA AGC TCT GGT GAC TAC AAC AAA AAC CAG TAC TAC GGC ATC ACT GCT GGT CCG GCT TAC CGC ATT AAC GAC TGG GCA AGC ATC TAC GGT GTA GTG GGT GTG GGT TAT GGT AAA TTC CAG ACC ACT GAA TAC CCG ACC TAC AAA CAC GAC ACC AGC GAC TAC GGT TTC TCC TAC GGT GCG GGT TTG CAA TTC AAC CCG ATG GAA AAC GTT GCT CTG GAC TTC TCT TAT GAG CAG AGC CGT ATT CGT AGC GTT GAC GTA GGT ACG TGG ATT GCT GGT GTT GGT TAT CGC TTC ACT CTC GGC ATG GAT GAG CTG TAC AAA AGC GGC ATt CGT GGG GGC TCT GGA GGC TCA GGC GGA TCC GGT GGT AGT GGA GGT TCG GGC GGT AGC GGT GGC TCT GGC GGA TCC GGT GGA TCT GGG GGG TCA GGC GGC TCT GGC GGA AGT GGC GGC AGC GGC GGC AGT GGC GGC AGC GGG GGA TCA GGT GGA AGC GGT GGC TCC ACC ATG GTG AGC ATG GTG AGC AAG GGC GAG GAG GAT AAC ATG GCC TCT CTC CCA GCG ACA CAT GAG TTA CAC ATC TTT GGC TCC ATC AAC GGT GTG GAC TTT GAC ATG GTG GGT CAG GGC ACC GGC AAT CCA AAT GAT GGT TAT GAG GAG TTA AAC CTG AAG TCC ACC AAG GGT GAC CTC CAG TTC TCC CCC TGG ATT CTG GTC CCT CAT ATC GGG TAT GGC TTC CAT CAG TAC CTG CCC TAC CCT GAC GGG ATG TCG CCT TTC CAG GCC GCG ATG GTA GAT GGC TCC GGA TAC CAA GTC CAT CGC ACA ATG CAG TTT GAA GAT GGT GCC TCC CTT ACT GTT AAC TAC CGC TAC ACC TAC GAG GGA AGC CAC ATC AAA GGA GAG GCC CAG GTG AAG GGG ACT GGT TTC CCT GCT GAC GGT CCT GTG ATG ACC AAC TCG CTG ACC GCT GCG GAC TGG TGC AGG TCG AAG AAG ACT TAC CCC AAC GAC AAA ACC ATC ATC AGT ACC TTT AAG TGG AGT TAC ACC ACT GGA AAT GGC AAG CGC TAC CGG AGC ACT GCG CGG ACC ACC TAC ACC TTT GCC AAG CCA ATG GCG GCT AAC TAT CTG AAG AAC CAG CCG ATG TAC GTG TTC CGT AAG ACG GAG CTC AAG CAC TCC AAG ACC GAG CTC AAC TTC AAG GAG TGG CAA AAG GCG TTT ACC GAT GTT ATG GGC ATG GAC GAG TTG TAC AAG TAA AA -3'
</SPAN></DIV><DIV id="imageText"><DIV class="left5">
This BioBrick Part looks like Figure 8. It contains the prefix and suffix with the correct restriction sites (EcoRI, XbaI, SpeI and PstI). OmpX is 513 bp long, the BamHI-Linker is 213 bp long and mNeonGreen 711 bp.
<H7>Characterization</H7>
When mutated with the amber stop codon TAG, OmpX can covalently bind anything by using a bio-orthogonal “click” reaction. The presence of mNeonGreen in this construct is tested with a fluorescence assay.

</DIV><DIV class="right5"><SPAN class="caption">
Figure 8: SnapGene figure of the OmpX – mNeonGreen Part.
</SPAN></DIV></DIV>

For all the experiments we used the following vectors: pETDuet-1 with a construct inserted (OmpX + intracellular protein) and pEVOL-pAzF (tRNA + tRNA synthetase). Both vectors were transformed into BL21(DE3). The expression was introduced by adding arabinose, IPTG and the unnatural amino acid.

<H7> DBCO-PEG4-TAMRA Confirmation</H7>
To confirm whether OmpX is in the membrane and whether or not the unnatural amino acid is being incorporated into OmpX, DBCO-PEG4-TAMRA was used. TAMRA is a fluorescent dye that can be used to verify the “click” reaction.
 
<DIV id="imageText"><DIV class="right1">
If the unnatural amino acid is present, DBCO-PEG4-TAMRA should “click” to the transmembrane protein OmpX and stay there. This can be analyzed with FACS. OmpX – mNeonGreen gave the following results after clicking with DBCO-PEG4-TAMRA (see Figure 9). From this it can be concluded that OmpX is in the membrane and that the “click” reaction works.
</DIV><DIV class="left1"><SPAN class="caption">
Figure 9: FACS results of OmpX – mNeonGreen.
</SPAN></DIV></DIV>
For more information about how to perform FACS experiments, see our <A href="https://2015.igem.org/Team:TU_Eindhoven/Project/Protocols" target="_blank">Protocol Page</A>.

<H7>Fluorescence Confirmation</H7>
To confirm whether mNeonGreen is present in the bacteria, a fluorescence assay was performed. Excitation took place at a wavelength of 480 nm with a laser. Emission was read out at 517 nm. From this experiment, it can be concluded that mNeonGreen is present and works (see Figure 10). 
<DIV id="imageText"><DIV id="imageText"><DIV class="columnleft"><SPAN class="caption">
Figure 10: Fluorescence results of OmpX – NanoLuc.
</SPAN></DIV><DIV class="references"><SPAN class="caption"><A href="#reft4" name="ref4">[4]</A>	Vogt J. and Schulz G.E., “The structure of the outer membrane protein OmpX from Escherichia coli reveals possible mechanisms of virulence”, Structure, vol. 7, no. 10, pp. 1301–9, Oct. 1999.<A href="#reft5" name="ref5">[5]</A>	Shaner N.C. et al, “A bright monomeric green fluorescent protein derived from Branchiostoma lanceolatum”, Nature Methods, vol. 10, no. 5, pp. 407-9, Mar. 2013. <A href="#reft6" name="ref6">[6]</A>	Evers T.H. et al, “Quantitative Understanding of the Energy Transfer between Fluorescent Proteins Connected via Flexible Peptide Linkers”, Biochemistry, vol. 45, no. 44, pp. 13183-92, Nov. 2006.</SPAN></DIV></DIV></DIV></DIV></LI><LI class="activiteit"><H6> Outer Membrane Protein X (OmpX) with BsoBI-linker and NanoLuc
</H6><H7> - <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1761002" target="_blank">BBa_K1761002</A></H7><DIV class="spoiler" id="spoiler3"><H7>Quick facts:</H7>
Total length: 1241 bp 
BsoBI-linker length: 213 bp 
NanoLuc length: 513 bp 
Weight: 46 kDa <H7>Usage and biology</H7><DIV id="imageText"><DIV class="left5">
OmpX, or Outer Membrane Protein X, belongs to a family of highly conserved bacterial proteins that promote bacterial adhesion to and entry into mammalian cells. It presents both C- and N-termini in the intracellular domain (see Figure 11). OmpX consists of an eight-stranded antiparallel all-next-neighbor β barrel. The core of the protein consists of an extended hydrogen-bonding network of highly conserved residues (see Figure 11) <A name="reft7" href="#ref7" class="textanchor">[7]</A>.
</DIV><DIV class="right5"><SPAN class="caption">
Figure 11: A) The OmpX protein structure has been elucidated through NMR and X-ray crystallography, B) The square residues are important for the secondary structure of OmpX. To keep the structure intact, we introduce an amber stop codon in one of the protruding loops. Figure 11B is adapted from <A name="reft7" href="#ref7" class="textanchor">[7]</A>.
</SPAN></DIV></DIV>
NanoLuc is a small sized and bright luciferase protein engineered for optimal performance as a bioluminescent reporter. The small size and bright luminescence bring exquisite sensitivity to reporter assays and other bioluminescence applications. The luciferase naturally occurs in the organism Oplophorus gracilirostis.

<H7>Gene Design</H7>
This construct consists of three parts, namely OmpX, a BsoBI-linker, and NanoLuc. OmpX is already described as a single part, and that same part, with the amber stop codon TAG incorporated for the integration of the unnatural amino acid, is used for this construct. For more information, see our <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1761000" target="_blank">Part OmpX</A>. 
<DIV id="imageText"><DIV class="right6">
Between OmpX and mNeonGreen we inserted a BsoBI-linker. This is a 213 bp long flexible GGSGGS-linker with two BsoBI-restriction sites. These sites give the possibility to shorten the linker with 45 bp. The BsoBI-linker is inspired by the article &quot;Quantitative Understanding of the Energy Transfer between Fluorescent Proteins Connected via Flexible Peptide Liners&quot; by T.H. Evers et al from 29 August 2006 <A name="reft8" href="#ref8" class="textanchor">[8]</A>.
These three parts together give a construct with OmpX as transmembrane protein  and NanoLuc as intracellular domain  (see Figure 12).
</DIV></DIV><DIV id="imageText"><DIV class="left6"><SPAN class="caption">
Figure 12: Schematical overview of the OmpX – NanoLuc Part.
</SPAN></DIV></DIV><DIV id="imageText"><DIV class="columnleft">
For all our experiments we used pETDuet-1 as a vector. This vector has two multiple cloning sites, namely MCS-1 and MCS-2. To be able to use this part as a sensor, we inserted both OmpX – NanoLuc and OmpX – mNeonGreen into this vector, both in different multiple cloning sites. Both multiple cloning sites were expressed. In this way, NanoLuc and mNeonGreen can function as a BRET pair (see Figure 13).
</DIV></DIV><DIV id="imageText"><DIV class="columnright"><SPAN class="caption">
Figure 13: Schematical overview of the BRET pair.</SPAN></DIV></DIV><H7>Sequence</H7>

The sequence of our OmpX - NanoLuc part has been verified by sequencing at StarSeq.
<DIV class="spoiler" id="spoiler8"><SPAN class="dnaText">
5'- ATG AAA AAA ATT GCA TGT CTT TCA GCA CTG GCC GCA GTT CTT GCT TTC ACC GCA GGT ACT TCC GTA GCA GC  GAC TTC TAC TGT AAC TGG CGG TTA CGC ACA GAG CGA CGC TCA GGG CCA AAT GAA CAA AAT GGG CGG TTT CAA CCT GAA ATA CCG CTA TGA AGA AGA CAA CAG CCC GCT GGG TGT GAT CGG TTC TTT CAC TTA CAC CGA GAA AAG CCG TAC TGC AAG CTC TGG TGA CTA CAA CAA AAA CCA GTA CTA CGG CAT CAC TGC TGG TCC GGC TTA CCG CAT TAA CGA CTG GGC AAG CAT CTA CGG TGT AGT GGG TGT GGG TTA TGG TAA ATT CCA GAC CAC TGA ATA CCC GAC CTA CAA ACA CGA CAC CAG CGA CTA CGG TTT CTC CTA CGG TGC GGG TTT GCA ATT CAA CCC GAT GGA AAA CGT TGC TCT GGA CTT CTC TTA TGA GCA GAG CCG TAT TCG TAG CGT TGA CGT AGG TAC GTG GAT TGC TGG TGT TGG TTA TCG CTT CAC TCT CGG CAT GGA TGA GCT GTA CAA AAG CGG CAT tCG TGG CGG CTC GGG TGG AAG CGG TGG GAG TGG TGG AAG CGG TGG GAG TGG CGG CTC GGG TGG GTC GGG CGG ATC GGG GGG TTC TGG TGG CAG TGG CGG CTC AGG CGG CTC CGG TGG ATC AGG TGG TTC GGG TGG TTC TGG GGG AAG CGG CGG GTC AGG TGG CTC TAC TAT GGT TAG CGT ATT TAC TCT TGA AGA TTT TGT CGG TGA TTG GCG CCA GAC CGC CGG CTA TAA CCT GGA CCA AGT GCT TGA ACA GGG CGG GGT TAG CAG CCT GTT TCA AAA CCT GGG GGT GAG TGT CAC GCC AAT TCA GCG CAT CGT TCT GTC GGG AGA GAA TGG TCT GAA AAT CGA TAT CCA CGT CAT TAT CCC GTA CGA AGG TCT TTC TGG TGA TCA GAT GGG GCA GAT AGA AAA AAT ATT CAA AGT GGT GTA CCC AGT AGA CGA TCA TCA CTT CAA GGT TAT ACT GCA CTA TGG CAC CCT CGT TAT CGA TGG CGT TAC TCC GAA TAT GAT CGA TTA CTT TGG GCG TCC TTA TGA AGG TAT TGC GGT GTT CGA CGG TAA AAA AAT TAC GGT TAC CGG GAC GCT CTG GAA TGG TAA TAA AAT CAT TGA TGA GCG CTT GAT AAA CCC AGA TGG CAG CCT TCT GTT CAG AGT TAC GAT AAA CGG GGT TAC GGG TTG GCG ACT GTG CGA AAG AAT ATT AGC TTA AAA -3' 
</SPAN></DIV><DIV id="imageText"><DIV class="left5">
This BioBrick Part looks like Figure 14. It contains the prefix and suffix with the correct restriction sites (EcoRI, XbaI, SpeI and PstI). OmpX is 513 bp long, the BsoBI-Linker is 213 bp long and NanoLuc 513 bp.
<H7>Characterization</H7>
When mutated with the amber stop codon TAG, OmpX can covalently bind anything by using a bio-orthogonal “click” reaction. The presence of NanoLuc in this construct can be tested by adding furimazine and read out its emission spectrum. If it is present, there should be a maximum around 460 nm. 

</DIV><DIV class="right5"><SPAN class="caption">
Figure 14: SnapGene overview of the OmpX – NanoLuc Part.
</SPAN></DIV></DIV>

For all the experiments we used the following vectors: pETDuet-1 with a construct inserted (OmpX + intracellular protein) and pEVOL-pAzF (tRNA + tRNA synthetase). Both vectors were transformed into BL21(DE3). The expression was introduced by adding arabinose, IPTG and the unnatural amino acid.

<H7> DBCO-PEG4-TAMRA Confirmation</H7>
To confirm whether OmpX is in the membrane and whether or not the unnatural amino acid is being incorporated into OmpX, DBCO-PEG4-TAMRA was used. TAMRA is a fluorescent dye that can be used to verify the “click” reaction. 
OmpX – NanoLuc gave the following results after clicking with DBCO-PEG4-TAMRA (see Figure 15). From this it can be concluded that OmpX is in the membrane and that the “click” reaction works. 
<DIV id="imageText"><DIV class="right1">
If the unnatural amino acid is present, DBCO-PEG4-TAMRA should “click” to the transmembrane protein OmpX and stay there. This can be analyzed with FACS. For more information about how to perform FACS experiments, see our <A href="https://2015.igem.org/Team:TU_Eindhoven/Project/Protocols" target="_blank">Protocol Page</A>.

</DIV><DIV class="left1"><SPAN class="caption">
Figure 15: FACS results of OmpX – NanoLuc.
</SPAN></DIV></DIV><DIV id="imageText"><DIV class="left5"><H7>Bioluminiscence Confirmation</H7>
To confirm whether NanoLuc is present in the bacteria, a bioluminescence measurement was performed. This gave the following results (see Figure 16). From this it can be concluded that OmpX - mNeonGreen is present.
For more information about how to perform a bioluminescence measurement, see our <A href="https://2015.igem.org/Team:TU_Eindhoven/Project/Protocols" target="_blank">Protocol Page</A>. 
</DIV><DIV class="right5"><SPAN class="caption">
Figure 16: Bioluminescence results of OmpX - mNeonGreen. 
</SPAN></DIV></DIV><DIV class="references"><SPAN class="caption"><A href="#reft7" name="ref7">[7]</A>	Vogt J. and Schulz G.E., “The structure of the outer membrane protein OmpX from Escherichia coli reveals possible mechanisms of virulence”, Structure, vol. 7, no. 10, pp. 1301–9, Oct. 1999.<A href="#reft8" name="ref8">[8]</A>	Evers T.H. et al, “Quantitative Understanding of the Energy Transfer between Fluorescent Proteins Connected via Flexible Peptide Linkers”, Biochemistry, vol. 45, no. 44, pp. 13183-92, Nov. 2006. </SPAN></DIV></DIV></LI><LI class="activiteit"><H6> mNeongreen
</H6><H7> - <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1761003" target="_blank">BBa_K1761003</A></H7><DIV class="spoiler" id="spoiler4"><H7>Quick facts:</H7>
Length: 711 bp 
Weight: 28kDa <DIV id="imageText"><DIV class="left6"><SPAN class="caption">
Figure 17: mNeonGreen.
</SPAN></DIV><DIV class="right6"><H7>Usage and biology</H7>
mNeonGreen is a yellow-green fluorescent protein and is derived from a tetrameric fluorescent protein from cephalochordate Branchiostoma lanceolatum (see Figure X). It is the brightest monomeric green of yellow fluorescent protein yet described and is an excellent fluorescence resonance energy transfer (FRET) acceptor for the newest cyan fluorescent proteins <A name="reft9" href="#ref9" class="textanchor">[9]</A>.
</DIV></DIV><H7>Gene Design</H7>
mNeonGreen was characterized by Shaner et al <A name="reft9" href="#ref9" class="textanchor">[9]</A>. We inserted mNeonGreen in the pETDuet-1 vector together with OmpX and a BamHI-linker (see Figure 18). This linker is a 213 bp long flexible GGSGGS linker and by using the restriction enzyme BamHI, the linker can become 45 bp shorter.
The BamHI-linker is inspired by the article &quot;Quantitative Understanding of the Energy Transfer between Fluorescent Proteins Connected via Flexible Peptide Liners&quot; by T.H. Evers et al from 29 August 2006 <A name="reft10" href="#ref10" class="textanchor">[10]</A>.
<H7>Sequence</H7>

The sequence of our mNeonGreen part has been verified by sequencing at StarSeq. 
<DIV class="spoiler" id="spoiler13"><SPAN class="dnaText">
ATG GTG AGC AAG GGC GAG GAG GAT AAC ATG GCC TCT CTC CCA GCG ACA CAT GAG TTA CAC ATC TTT GGC TCC ATC AAC GGT GTG GAC TTT GAC ATG GTG GGT CAG GGC ACC GGC AAT CCA AAT GAT GGT TAT GAG GAG TTA AAC CTG AAG TCC ACC AAG GGT GAC CTC CAG TTC TCC CCC TGG ATT CTG GTC CCT CAT ATC GGG TAT GGC TTC CAT CAG TAC CTG CCC TAC CCT GAC GGG ATG TCG CCT TTC CAG GCC GCc ATG GTA GAT GGC TCC GGA TAC CAA GTC CAT CGC ACA ATG CAG TTT GAA GAT GGT GCC TCC CTT ACT GTT AAC TAC CGC TAC ACC TAC GAG GGA AGC CAC ATC AAA GGA GAG GCC CAG GTG AAG GGG ACT GGT TTC CCT GCT GAC GGT CCT GTG ATG ACC AAC TCG CTG ACC GCT GCG GAC TGG TGC AGG TCG AAG AAG ACT TAC CCC AAC GAC AAA ACC ATC ATC AGT ACC TTT AAG TGG AGT TAC ACC ACT GGA AAT GGC AAG CGC TAC CGG AGC ACT GCG CGG ACC ACC TAC ACC TTT GCC AAG CCA ATG GCG GCT AAC TAT CTG AAG AAC CAG CCG ATG TAC GTG TTC CGT AAG ACG GAG CTC AAG CAC TCC AAG ACC GAG CTC AAC TTC AAG GAG TGG CAA AAG GCG TTT ACC GAT GTT ATG GGC ATG GAC GAG TTG TAC AAG TAA 
</SPAN></DIV><DIV id="imageText"><DIV class="left5">
This BioBrick Part looks like Figure 3. It contains the prefix and suffix with the correct restriction sites (EcoRI, XbaI, SpeI and PstI). mNeonGreen is 711 bp long.
<H7>Characterization</H7>
mNeonGreen was characterized mNeonGreen by fusing it to an outer membrane protein (see below) as well as by expressing it on itself in the cytosol. For the expression of mNeonGreen in the cytosol, mNeonGreen was inserted after the constitutive promotor J23101. Even though no RBS was introduced into the composite part, the cells still showed fluorescence which could be measured with the Cary Eclipse spectrofluorometer for both an in vivo excitation scan and a in vivo emission scan.

</DIV><DIV class="right5"><SPAN class="caption">
Figure 14: SnapGene overview of mNeongreen
</SPAN></DIV></DIV>

If you want to read all about the in vivo excitation and emission spectrum using different promoters. Check it out at the biobrick page <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1761003" target="_blank">BBa_K1761003</A></DIV></LI><LI class="activiteit"><H6> 	
Split Luciferase 
</H6><DIV class="spoiler" id="spoiler5">
Another intracellular domain we used is a split luciferase. This split luciferase consists of two parts, namely LargeBit and SmallBit. We inserted these two parts both in a pETDuet-1 vector together with OmpX. LargeBit and SmallBit have an affinity towards each other, so when in close proximity they will come together and will give a luminescence signal. Followed is a short description of each part and of the NanoBit construct.
<H7>OmpX - LargeBit <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1761005" target="_blank">BBa_K1761005</A></H7><DIV id="imageText"><DIV class="right6">
LargeBit is the big part of this Split Luciferase. Its molecular weight is 18 kDa. OmpX (1) (with a correct mutation for the amber stop codon TAG), a BamHI-linker and LargeBit (2) together will look like Figure 1. This construct on its own has no function since LargeBit on itself has no luminescence activity.
*Sequence will be published later
</DIV></DIV><DIV class="left6"><SPAN class="caption">
Figure 1: Schematical overview of the OmpX - LargeBit construct..
</SPAN></DIV><H7>OmpX-SmallBit <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1761006" target="_blank">BBa_K1761006</A></H7>
SmallBit is the small part of this Split Luciferase. Its molecular weight is 1 kDa, it is only 11 amino acids long. OmpX (1) (with a correct mutation for the amber stop codon TAG), a BsoBI-linker and SmallBit (2) together will look like Figure 2. This construct on its own has no function since SmallBit on itself has no luminescence activity.
* Sequence will be published later.

<H7>NanoBit construct</H7><DIV id="imageText"><DIV class="left5">
NanoBit utilizes a structural complementation-based approach to monitor protein interactions within living cells. Protein interaction promotes structural complementation and generation of a bright, luminescent enzyme. Protein dynamics can be followed in real-time in living cells following addition of the Nano-Glo Live Cell Reagent, a non-lytic detection reagent containing the cell-permeable furamizine substrate. See Figure 3 for the whole construct.
</DIV><DIV class="right5"><SPAN class="caption">
Figure 2: Schematical overview of the NanoBit construct.
</SPAN></DIV></DIV><H7>Characterization</H7>
When mutated with the amber stop codon TAG, a non-natural amino acid with an azide-functionalized group can be expressed. After the expression of this amino acid, OmpX can covalently bind almost anything, as long as it contains a DBCO-functionalized group. The binding finds place by using a bio-orthogonal “click” reaction (SPAAC chemistry). To test the functionality of this “click” reaction, some experiments were done by clicking DBCO-PEG4-TAMRA at the surface. 
For all the experiments we used the following vectors: pETDuet-1 with one or two construct(s) inserted (OmpX + intracellular protein) and pEVOL-pAzF (tRNA + tRNA synthetase). Both vectors were transformed into BL21(DE3). The expression was introduced by adding arabinose, IPTG and the non-natural amino acid.


<H7> DBCO-PEG4-TAMRA click &amp; bioluminiscence confirmation</H7>
To confirm whether OmpX is in the membrane and whether or not the non-natural amino acid is being incorporated into OmpX, DBCO-PEG4-TAMRA was used. TAMRA is a fluorescent dye that can be used to verify the “click” reaction. If the non-natural amino acid is present, DBCO-PEG4-TAMRA should “click” to the transmembrane protein OmpX and stay there. This can be analyzed with FACS. For more information about how to perform FACS experiments, see our <A href="https://2015.igem.org/Team:TU_Eindhoven/Project/Protocols" target="_blank">Protocol Page</A>.

To verify that OmpX is in the membrane, we used the OmpX – LgBiT and OmpX – SmBiT constructs. These gave the following results after clicking with DBCO-PEG4-TAMRA (see Figure 3 and 4). From this it can be concluded that OmpX is in the membrane and that the “click” reaction works. 
<DIV id="imageText"><DIV id="imageText"><DIV class="columnleft"><SPAN class="caption">
Figure 3: FACS results of OmpX – LargeBit.
</SPAN></DIV><DIV class="columnright"><SPAN class="caption">
Figure 4: FACS results of OmpX – SmallBit.
</SPAN></DIV></DIV><DIV id="imageText"><DIV id="imageText"><DIV class="columnleft"><SPAN class="caption">
Figure 5: FACS results of OmpX – LargeBit &amp; OmpX-Smallbit.
</SPAN></DIV><DIV class="columnright"><SPAN class="caption">
Figure 6: Bioluminiscence of NanoBit.
</SPAN></DIV></DIV></DIV></DIV></DIV></LI></UL></SPAN>&lt;groupparts&gt;iGEM015 TU_Eindhoven&lt;/groupparts&gt;</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>