<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU_Eindhoven_Project_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU Eindhoven/Project/Protocols</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="menuwrapper"><DIV id="menuTotal"><DIV id="menu"><A id="menu1" href="https://2015.igem.org/Team:TU_Eindhoven">
        Home</A><A id="menu2" href="https://2015.igem.org/Team:TU_Eindhoven/Team"> 
        Team</A><A id="menu3" href="https://2015.igem.org/Team:TU_Eindhoven/Project"> 
        Project</A><A id="menu4" href="https://2015.igem.org/Team:TU_Eindhoven/ResultsHome">
        Results</A><A id="menu5" href="https://2015.igem.org/Team:TU_Eindhoven/Modeling">
        Modeling</A><A id="menu6" href="https://2015.igem.org/Team:TU_Eindhoven/Practices">
        Policy Practices</A><A id="menu7" href="https://2015.igem.org/Team:TU_Eindhoven/Collaborations">
        Collaborations</A><A id="menu8" href="https://2015.igem.org/Team:TU_Eindhoven/Team/Sponsors">
        Sponsors</A><A id="menu9" href="https://2015.igem.org/Team:TU_Eindhoven/Safety">
        Safety</A></DIV></DIV><DIV id="submenuwrapper"><DIV id="submenu"><DIV id="sub2"><A id="sub2a" href="https://2015.igem.org/Team:TU_Eindhoven/Team/Members">Team members</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Team/Supervisors">Supervisors</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Attributions">Attributions</A></DIV><DIV id="sub3"><A id="sub3a" href="https://2015.igem.org/Team:TU_Eindhoven/Description">Description</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Project/Design">Design</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Project/Expression_System">Expression System</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Project/Experimental_Approach">Experimental Approach</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Project/Protocols">Protocols</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Notebook">Timeline</A></DIV><DIV id="sub4"><A id="sub4a" href="https://2015.igem.org/Team:TU_Eindhoven/Parts">Parts</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Results">Results</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Achievements">Medal Motivation</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Measurement">Interlab Study</A></DIV><DIV id="sub5"><A id="sub5a" href="https://2015.igem.org/Team:TU_Eindhoven/Modeling/Approach">Approach</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Modeling/Script">Script</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Modeling/Results">Results</A></DIV><DIV id="sub6"><A id="sub6a" href="https://2015.igem.org/Team:TU_Eindhoven/Policy_Practices/Human_Outreach">Outreach</A><A style="width:12%;" href="https://2015.igem.org/Team:TU_Eindhoven/Policy_Practices/Application_scenarios">Application Scenarios</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Policy_Practices/Stakeholders">Stakeholders</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Policy_Practices/Regulations">Regulations</A></DIV><DIV id="sub7"><A id="sub7a" href="https://2015.igem.org/Team:TU_Eindhoven/Collaborations/CloningGuide">Cloning Guide</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Collaborations/Collaboration_Attributions">Collaboration Attributions</A></DIV><DIV id="sub9"><A id="sub9a" href="https://2015.igem.org/Team:TU_Eindhoven/Safety/Lab">Lab Safety</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Safety/Project">Project Safety</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Safety/LabEquipment">Lab Equipment</A></DIV></DIV></DIV></DIV><DIV id="body"><DIV id="container"><DIV id="wikiTour"><DIV id="moreDepth"><SPAN class="tekst1BI">Dig Even Deeper</SPAN><SPAN class="tekst1"><UL id="wikiTourList"><LI>See our <A href="https://2015.igem.org/Team:TU_Eindhoven/Notebook">Notebook</A></LI></UL></SPAN></DIV><A style="text-decoration: none" href="https://2015.igem.org/Team:TU_Eindhoven/Modeling"><DIV id="nextChapter"><SPAN class="tekst1BI">Next Chapter</SPAN><SPAN class="tekst1">
See how modelling helped us understand our device</SPAN></DIV></A><A style="text-decoration: none" href="https://2015.igem.org/Team:TU_Eindhoven/Project"><DIV id="previousChapter"><SPAN class="tekst1BI">Previous Chapter</SPAN><SPAN class="tekst1">
Missed something? Jump back to our Project page</SPAN></DIV></A></DIV><DIV id="containercontent"><H1> 
Protocols
</H1><SPAN class="tekst1">
For the labwork various protocols were created. These are applied during the work in the Biolab.

<H2> 
General Protocols
</H2><UL class="activiteitlijst"><LI><DIV id="imageText"><SPAN class="activiteit"><A href="https://static.igem.org/mediawiki/2015/a/a8/TU_Eindhoven_Protocols_Preparation_of_culture_media%2C_agar_plates%2C_antibiotics_and_general_necessities.pdf" target="_blank"> Preparation of general necessities </A></SPAN><SPAN class="tekstProtocols">Various stocks of antibiotics, growth media, stock solutions and other basic necessities have to be available when working in the lab. This protocol describes the way we prepared them.</SPAN></DIV></LI><LI><DIV id="imageText"><SPAN class="activiteit"><A href="https://static.igem.org/mediawiki/2015/9/96/TU_Eindhoven_Protocols_PCR_Amplification.pdf" target="_blank">PCR Amplification</A></SPAN><SPAN class="tekstProtocols">Amplification of DNA can be done using PCR in a very easy way. With the use of a thermal cycler, template DNA and designed primers, the template can be amplified in only a couple of hours. This saves a lot of time and the product DNA can be used for transformations right away.</SPAN></DIV></LI><LI><DIV id="imageText"><SPAN class="activiteit"><A href="https://static.igem.org/mediawiki/2015/4/47/TU_Eindhoven_Protocols_Small_Culturing.pdf" target="_blank"> Small Culturing</A></SPAN><SPAN class="tekstProtocols">Cultures of competent cells need to be prepared before protein expression is possible.</SPAN></DIV></LI><LI><DIV id="imageText"><SPAN class="activiteit"><A href="https://static.igem.org/mediawiki/2015/8/8b/TU_Eindhoven_Protocols_Colony_Picking_Colony_PCR.pdf" target="_blank"> Colony picking </A></SPAN><SPAN class="tekstProtocols">To evaluate if the inserted fragments are of the correct length, colony picking and colony PCR has to be done.</SPAN></DIV></LI><LI><DIV id="imageText"><SPAN class="activiteit"><A href="https://static.igem.org/mediawiki/2015/f/f8/TU_Eindhoven_Protocols_Colony_PCR.pdf" target="_blank"> Colony PCR </A></SPAN><SPAN class="tekstProtocols">To evaluate if the inserted fragments are of the correct length, colony picking and colony PCR has to be done.</SPAN></DIV></LI><LI><DIV id="imageText"><SPAN class="activiteit"><A href="https://static.igem.org/mediawiki/2015/f/f5/TU_Eindhoven_Protocols_Miniprep.pdf" target="_blank"> Miniprepping </A></SPAN><SPAN class="tekstProtocols">Miniprepping of the bacteria is needed to obtain the plasmid DNA.</SPAN></DIV></LI><LI><DIV id="imageText"><SPAN class="activiteit"><A href="https://static.igem.org/mediawiki/2015/a/a0/TU_Eindhoven_Protocols_NanoDrop.pdf" target="_blank"> NanoDrop </A></SPAN><SPAN class="tekstProtocols">To determine the concentration of DNA samples, nanodropping is done.</SPAN></DIV></LI><LI><DIV id="imageText"><SPAN class="activiteit"><A href="
https://static.igem.org/mediawiki/2015/3/3e/TU_Eindhoven_Protocols_Agarose_Gel_Electrophoresis.pdf" target="_blank"> Agarose Gel Electrophoresis </A></SPAN><SPAN class="tekstProtocols">To be able to evaluate the length of DNA samples, an agarose gel is prepared.</SPAN></DIV></LI><LI><DIV id="imageText"><SPAN class="activiteit"><A href="https://static.igem.org/mediawiki/2015/b/ba/TU_Eindhoven_Protocols_Glycerol_Stock.pdf" target="_blank"> Preparing Glycerol Stocks </A></SPAN><SPAN class="tekstProtocols">In order to save the competent cells with the right vectors for later usage, they can be stored in glycerol stocks. These stocks are easy to use when cells are needed at a later moment in your project.</SPAN></DIV></LI><LI><DIV id="imageText"><SPAN class="activiteit"><A href="https://static.igem.org/mediawiki/2015/f/fe/TU_Eindhoven_Protocols_Plating.pdf" target="_blank"> Plating </A></SPAN><SPAN class="tekstProtocols">To amplify bacteria, plating is required.</SPAN></DIV></LI><LI><DIV id="imageText"><SPAN class="activiteit"><A href="https://static.igem.org/mediawiki/2015/2/20/TU_Eindhoven_Protocols_Preparing_StarSeq_Sequencing_Samples.pdf" target="_blank"> Sequencing </A></SPAN><SPAN class="tekstProtocols">When DNA is amplified and tranformed into competent cells, you want to know wheter the DNA is correct. Using template DNA and specific designed primers, the sequence of vectors can be analyzed.</SPAN></DIV></LI><LI><DIV id="imageText"><SPAN class="activiteit"><A href="https://static.igem.org/mediawiki/2015/c/cb/TU_Eindhoven_Protocols_PCR_Purification.pdf" target="_blank"> PCR Purification</A></SPAN><SPAN class="tekstProtocols">PCR purifaction is for the purification of the product obtained from a PCR reaction. With the QIAquick PCR Purification Kit, products up to 10 ng can be purified. This means that oligos varying from 100 bp to 10 kb can be purified with the simple bind-wash-elute procedure.</SPAN></DIV></LI><LI><DIV id="imageText"><SPAN class="activiteit"><A href="https://static.igem.org/mediawiki/2015/e/e8/TU_Eindhoven_Protocols_Gel_purification.pdf" target="_blank"> PCR Gel Purification</A></SPAN><SPAN class="tekstProtocols">PCR gel purifaction is for the purification of the product obtained from a PCR reaction. With the QIAquick PCR Gel Purification Kit, oligos varying from 70 bp to 10 kb can be purified with the simply running a gel and a bind-wash-elute procedure.</SPAN></DIV></LI><LI><DIV id="imageText"><SPAN class="activiteit"><A href="https://static.igem.org/mediawiki/2015/a/a1/TU_Eindhoven_Protocols_Streaking_Glycerol_Stock.pdf" target="_blank"> Streaking Glycerol Stock</A></SPAN><SPAN class="tekstProtocols">After storing competent cells in glycerol stocks, they can be prepared for usage by means of streaking them on agar plates.  </SPAN></DIV></LI></UL><H2>
Traditional Cloning &amp; BioBricking
</H2><SPAN class="tekst1"><DIV class="spoiler" id="spoiler1"><SPAN class="tekst1">Traditional cloning remains the workhorse of DNA recombinant technology as it is cheap and effective. It has been characterized by the use of restriction enzymes which yield sticky ends. These sticky ends can be ligated to each other by a ligase. The ligated plasmid can subsequently be transformed. We used traditional cloning on and off as well as as a back-up plan if our Gibson Assemblies failed.</SPAN><SPAN class="caption"> Figure 1: Overview of the workflow of Traditional Cloning. Traditional Cloning uses restriction enzymes (the scissors) to cut DNA at specific places. Cutting the DNA yields distinct sticky ends which can be ligated together. By cutting both the to be inserted fragment as well as the vector, one can insert the fragment into a vector. As a result, one can obtain a new plasmid. 
</SPAN></DIV></SPAN><UL class="activiteitlijst"><LI><DIV id="imageText"><SPAN class="activiteit"><A href="https://static.igem.org/mediawiki/2015/9/97/TU_Eindhoven_Protocols_Digestion.pdf" target="_blank"> Digestion </A></SPAN></DIV></LI><LI><DIV id="imageText"><SPAN class="activiteit"><A href="https://static.igem.org/mediawiki/2015/d/da/TU_Eindhoven_Protocols_Ligation.pdf" target="_blank"> Ligation </A></SPAN></DIV></LI></UL><H2>
Transformations
</H2><SPAN class="tekst1"><DIV class="spoiler" id="spoiler2"><SPAN class="tekst1">Transformation refers to the act of introducing new plasmid DNA into bacteria. The plasmids can be introduced in bacteria in numerous ways. Perhaps the most well-known methods are electroporation and heat shocking of competent cells. In our project, we used competent cells exclusively. These cells have been chemically modified to transform efficiently. Competent cells can be directly ordered from a wide range of life sciences companies. We used BL21(DE3), NovaBlue, NEB 5-alpha and XL10-Gold (ultra)competent cells exclusively.</SPAN><SPAN class="caption"> Figure 2: Transformation is a term used for the introduction of new plasmids into bacteria. Transformation can have place in numerous ways. Often, the term is used intechangeably with transfection. This latter term is, however, reserved for the introduction of plasmid DNA into eukaryotic cells. 
</SPAN></DIV></SPAN><UL class="activiteitlijst"><LI><DIV id="imageText"><SPAN class="activiteit"><A href="https://static.igem.org/mediawiki/2015/f/f9/TU_Eindhoven_Protocols_Transformation_into_NovaBlue.pdf" target="_blank"> Transformation into NovaBlue</A></SPAN><SPAN class="tekstProtocols">NovaBlue cells can be used for plasmid amplification. </SPAN></DIV></LI><LI><DIV id="imageText"><SPAN class="activiteit"><A href="https://static.igem.org/mediawiki/2015/2/2b/TU_Eindhoven_Protocols_Double_Transformation.pdf" target="_blank"> Double transformation </A><SPAN class="tekstProtocols">A double transformation can be used to introduce multiple plasmids into competent cells within a single heatshock. In comparison to a singular transformation, the amount of DNA used in a double transformation is far greater. This protocol has been adapted from iGEM TU Eindhoven 2014. To obtain more colonies, the amount of DNA used in a double transformation was increased.</SPAN></SPAN></DIV></LI></UL><H2> 
Gibson Assembly
</H2><SPAN class="tekst1"><DIV class="spoiler" id="spoiler3"><SPAN class="tekst1">Gibson Assembly is a one-pot assembly method. It requires a linearized vector as well as dsDNA fragments. Linearization of the vector can be realized by the use of restriction enzymes or through PCR. The dsDNA fragments can be obtained through PCR or they can be ordered directly from a manufacturer. In our project, we linearized the vector through PCR and ordered dsDNA fragments directly from IDT. </SPAN><SPAN class="caption">
Figure 3: General workflow of Gibson Assembly. The first step consists of linearizing the vector using either PCR or digestion by restriction enzymes. Next, the linearized vector and dsDNA fragments are introduced in a tube with the Gibson Assembly Master Mix, and incubated at 50°. The resulting mixture is transformed into competent cells and analyzed using colony PCR to select the correctly assembled vectors.</SPAN></DIV></SPAN><UL class="activiteitlijst"><LI><DIV id="imageText"><SPAN class="activiteit"><A href="https://static.igem.org/mediawiki/2015/5/5c/TU_Eindhoven_Protocols_Vector_Linearization.pdf" target="_blank"> Vector Linearization</A></SPAN><SPAN class="tekstProtocols">A linear vector is a prerequisite for Gibson Assembly. Linearization can be realized through restriction or through PCR. In our protocol, we use PCR as this yields scarless constructs. This protocol consists of a PCR step, an optional DpnI digestion step, an optional PCR purification step, a NanoDrop step and an optional gel electrophoresis step. </SPAN></DIV></LI><LI><DIV id="imageText"><SPAN class="activiteit"><A href="https://static.igem.org/mediawiki/2015/8/8e/TU_Eindhoven_Protocols_Gibson_assembly.pdf" target="_blank"> NEBuilder HiFi Assembly </A><SPAN class="tekstProtocols">During our iGEM summer, we used the NEBuilder HiFi Assembly Kits. These kits contain a high-fidelity polymerase rather than a normal polymerase, limiting the occurence of errors during the Gibson Assembly. This protocol contains the one-pot assembly method as well as transformation of the product into NEB 5-alpha cells. </SPAN></SPAN></DIV></LI></UL><H2> 
Unnatural protein expression &amp; FACS
</H2><DIV class="spoiler" id="spoiler4"><SPAN class="tekstProtocols">The Fluorescence-Assisted Cell Sorter will be used to determine whether the click reaction occurs. To enable the click reaction, proteins have to be expressed with the unnatural amino acid. The proteins expressing this unnatural amino acid can subsequently be incubated with DBCO-functionalized TAMRA dye to enable quantitative measurement of the click reaction. </SPAN><SPAN class="caption">
Figure 4: To verify whether the click reaction has occured, we incubate the cells with DBCO-functionalized TAMRA. If the outer membrane protein is functionalized with the unnatural amino acid, this TAMRA dye binds to the membrane proteins covalently. In that case, the cells will remain fluorescent after a few washing steps. </SPAN></DIV><UL class="activiteitlijst"><LI><DIV id="imageText"><SPAN class="activiteit"><A href="https://static.igem.org/mediawiki/2015/6/60/TU_Eindhoven_Protocols_Preparing_FACS_Samples.pdf" target="_blank"> Preparation of FACS samples</A></SPAN></DIV></LI><LI><DIV id="imageText"><SPAN class="activiteit"><A href="https://static.igem.org/mediawiki/2015/a/a9/TU_Eindhoven_Protocols_ProteinExpression.pdf" target="_blank"> Protein expression with the unnatural amino acid</A></SPAN></DIV></LI></UL><H2> 
Alginate Beads
</H2><DIV class="spoiler" id="spoiler5"><SPAN class="tekst1">
Korte uitleg!
</SPAN></DIV><UL class="activiteitlijst"><LI><DIV id="imageText"><SPAN class="activiteit"><A href="https://static.igem.org/mediawiki/2015/4/4c/TU_Eindhoven_Protocols_Alginate_beads.pdf" target="_blank"> Making alginate beads</A></SPAN><SPAN class="tekstProtocols">Bacteria can be placed in an controlled environment using alginate beads </SPAN></DIV></LI></UL><H2> 
Testing the COMBs
</H2><DIV class="spoiler" id="spoiler6"><SPAN class="tekst1">
Korte uitleg!
</SPAN></DIV><UL class="activiteitlijst"><LI><DIV id="imageText"><SPAN class="activiteit"><A href="https://static.igem.org/mediawiki/2015/c/cc/TU_Eindhoven_Protocols_SDS-Page_Analysis.pdf" target="_blank">SDS-Page Analysis</A></SPAN><SPAN class="tekstProtocols">A SDS-page gel is a useful to check if your proteins are expressed.  </SPAN></DIV></LI><LI><DIV id="imageText"><SPAN class="activiteit"><A href="https://static.igem.org/mediawiki/2015/1/11/TU_Eindhoven_Protocols_Complementary_DNA_assay.pdf" target="_blank">Complementary DNA assay</A></SPAN><SPAN class="tekstProtocols">Bringing them together with complementary DNA.</SPAN></DIV></LI><LI><DIV id="imageText"><SPAN class="activiteit"><A href="https://static.igem.org/mediawiki/2015/b/b1/TU_Eindhoven_Protocols_Measuring_Bioluminescense_and_fluorescence.pdf" target="_blank">Bioluminescence &amp; Fluorescence</A></SPAN><SPAN class="tekstProtocols">One can measure bioluminiscence of the luciferases (NanoLuc &amp; NanoBit) for verifying its functionality. One can measure the fluorescence of the fluorophore (mNeongreen). When both NanoLuc &amp; mNeongreen are present in the cell BRET can occur &amp; be measured as well.</SPAN></DIV></LI></UL></SPAN></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>