<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU_Eindhoven_Project_Expression_System skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU Eindhoven/Project/Expression System</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="infoTab"><DIV id="infoTabTitle">
Project - Expression System
</DIV><DIV id="infoTabContent"><DIV id="contentKop1" class="contentKop"><A href="#h1-1" class="contentLink">Overview - the expression system</A></DIV><UL id="contentList1" class="contentList"><LI><A href="#h2-2" class="contentLink">Unnatural amino acid incorporation</A></LI><LI><A href="#h2-1" class="contentLink">The vectors</A></LI></UL><DIV id="contentKop2" class="contentKop"><A href="#h1-2" class="contentLink">Unnatural amino acid incorporation</A></DIV><UL id="contentList2" class="contentList"><LI><A href="#h2-3" class="contentLink">Unnatural Amino Acid Expression</A></LI><LI><A href="#h2-4" class="contentLink">p-Azido-phenylalanine</A></LI></UL><DIV id="contentKop3" class="contentKop"><A href="#h1-3" class="contentLink">The Vectors</A></DIV><UL id="contentList3" class="contentList"><LI><A href="#h2-5" class="contentLink">pEVOL-pAzF</A></LI><LI><A href="#h2-6" class="contentLink">pETDuet-1</A></LI></UL></DIV></DIV><DIV id="menuwrapper"><DIV id="menuTotal"><DIV id="menu"><A id="menu1" href="https://2015.igem.org/Team:TU_Eindhoven">
        Home</A><A id="menu2" href="https://2015.igem.org/Team:TU_Eindhoven/Team"> 
        Team</A><A id="menu3" href="https://2015.igem.org/Team:TU_Eindhoven/Project"> 
        Project</A><A id="menu4" href="https://2015.igem.org/Team:TU_Eindhoven/ResultsHome">
        Results</A><A id="menu5" href="https://2015.igem.org/Team:TU_Eindhoven/Modeling">
        Modeling</A><A id="menu6" href="https://2015.igem.org/Team:TU_Eindhoven/Practices">
        Policy Practices</A><A id="menu7" href="https://2015.igem.org/Team:TU_Eindhoven/Collaborations">
        Collaborations</A><A id="menu8" href="https://2015.igem.org/Team:TU_Eindhoven/Team/Sponsors">
        Sponsors</A><A id="menu9" href="https://2015.igem.org/Team:TU_Eindhoven/Safety">
        Safety</A></DIV></DIV><DIV id="submenuwrapper"><DIV id="submenu"><DIV id="sub2"><A id="sub2a" href="https://2015.igem.org/Team:TU_Eindhoven/Team/Members">Team members</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Team/Supervisors">Supervisors</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Attributions">Attributions</A></DIV><DIV id="sub3"><A id="sub3a" href="https://2015.igem.org/Team:TU_Eindhoven/Description">Description</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Project/Design">Design</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Project/Expression_System">Expression System</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Project/Experimental_Approach">Experimental Approach</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Project/Protocols">Protocols</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Notebook">Timeline</A></DIV><DIV id="sub4"><A id="sub4a" href="https://2015.igem.org/Team:TU_Eindhoven/Parts">Parts</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Results">Results</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Achievements">Medal Motivation</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Measurement">Interlab Study</A></DIV><DIV id="sub5"><A id="sub5a" href="https://2015.igem.org/Team:TU_Eindhoven/Modeling/Approach">Approach</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Modeling/Script">Script</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Modeling/Results">Results</A></DIV><DIV id="sub6"><A id="sub6a" href="https://2015.igem.org/Team:TU_Eindhoven/Policy_Practices/Human_Outreach">Outreach</A><A style="width:12%;" href="https://2015.igem.org/Team:TU_Eindhoven/Policy_Practices/Application_scenarios">Application Scenarios</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Policy_Practices/Stakeholders">Stakeholders</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Policy_Practices/Regulations">Regulations</A></DIV><DIV id="sub7"><A id="sub7a" href="https://2015.igem.org/Team:TU_Eindhoven/Collaborations/CloningGuide">Cloning Guide</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Collaborations/Collaboration_Attributions">Collaboration Attributions</A></DIV><DIV id="sub9"><A id="sub9a" href="https://2015.igem.org/Team:TU_Eindhoven/Safety/Lab">Lab Safety</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Safety/Project">Project Safety</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Safety/LabEquipment">Lab Equipment</A></DIV></DIV></DIV></DIV><DIV id="body"><DIV id="container"><DIV id="wikiTour"><A style="text-decoration: none" href="https://2015.igem.org/Team:TU_Eindhoven/Project/Experimental_Approach"><DIV id="nextChapter"><SPAN class="tekst1BI">Dig Deeper</SPAN><SPAN class="tekst1">
Explore how we tested our COMBs experimentally</SPAN></DIV></A></DIV><DIV id="containercontent"><DIV id="landingPage"><H1 id="h1-1">The Expression System</H1><SPAN class="tekst1">
The system we have designed relies on the presence of two Clickable Outer Membrane Proteins within the cells. To obtain cells which are functionalized with these proteins, an expression system is needed capable of producing two proteins which carry unnatural amino acids. In addition to co-expression of two proteins, we thus need to gear up our bacteria with the tools to incorporate unnatural amino acids within their proteins. An overview of the expression system we used to obtain these bacteria is presented below.
<DIV class="imageText"><DIV class="left2"><SPAN class="caption">		
Figure 1: As a proof of concept, we will construct and express our device within E. coli BL21(DE3). The device we test features the fast cellular response signaling components. To obtain this device, the BL21(DE3) cells will be cotransformed with two plasmids. The first plasmid, pETDuet-1 (blue), carries the genes for the outer membrane proteins and expression is triggered by the addition of IPTG. The second plasmid, pEVOL-pAzF (red), is necessary for the incorporation of the unnatural amino acid within the outer membrane proteins.
</SPAN></DIV></DIV><DIV class="imageText"><DIV class="rightColumn"><A name="name=" jumpback=""><H2 id="h2-1">The Vectors</H2></A>
Our cells will be co-transformed with two plasmids to enable the expression of our COMBs. The first plasmid is the pETDuet-1 vector, a vector which has been optimized for expression of two proteins. The second plasmid is the pEVOL-pAzF vector which contains pAzF aminoacyl-tRNA synthetase, which is necessary for the incorporation of the unnatural amino acid. Read more about the vectors here: </DIV></DIV></SPAN></DIV><DIV class="leftColumn"><H2 id="h2-2">Unnatural amino acid incorporation</H2>
To enable the post-translational click reaction of aptamers to proteins, the outer membrane proteins have to be functionalized with azide groups. These azide groups can be incorporated into the outer membrane proteins by incorporating the unnatural amino acid p-Azido-phenylalanine.  This incorporation is enabled through amber codon suppression, expression of the pAzF aminoacyl-tRNA synthetase and addition of pAzF to the medium. Read more on unnatural amino acid expression here: </DIV></DIV></DIV><H1 id="h1-2">Unnatural amino acid incorporation</H1><H2 id="h2-3">Unnatural Amino Acid Expression </H2><SPAN class="tekst1">
DNA is best known for its function as carrier of hereditary information: DNA carries the biological information that is carried over to the next generation each time cells divide into daughter cells. This biological information is merely composed of four base pairs, being guanine, cytosine, adenine and tyrosine. The sequence of the base pairs is translated into an amino acid code in the form of triplets. These triplets are recognized by tRNA's, which carry the correct to be incorporated amino acid, which is added onto the growing chain (see Figure 2).
<DIV class="imageText"><DIV class="right1"><SPAN class="caption">		
Figure 2: tRNAs play a key role in translation of the genetic instructions to amino acids. These tRNAs present the ribosomes with the correct amino acid for the correct codon. After the tRNAs recognize their corresponding codon, they enter the ribosomes where their amino acid will be added onto the growing peptide chain.
</SPAN></DIV></DIV>
In addition to encoding an amino acid sequence, some of the triplets encode stop codons. In total, three of the 64 possible triplets encode these stop codons. Instead of a tRNA, the triplet is recognized by release factors which bind to the triplet and enter the ribosome. Since these release factors do not carry any amino acid, the amino acid chain will be released from the ribosome into the cytosol, allowing it to fold into intricate structures (see Figure 3).

<DIV class="imageText"><DIV class="left2"><SPAN class="caption">		
Figure 3: There are no tRNAs which have affinity towards stop codons. Instead of tRNAs, there are release factors which bind to any ribosome encountering the stop codon. These release factors closely mimic the shape and charge distribution of tRNAs such that they can bind to the ribosome, but they lack an amino acid. Therefore, the ribosome can no longer extend the growing chain of amino acids, causing the peptide chain to be released.
</SPAN></DIV></DIV>

All of the naturally occuring proteins are encoded by the twenty primary amino acids normally present in proteins. Since this severely limited protein research, researchers have explored the possibility of incorporating unnatural amino acids into proteins. A way to do this is to hijack the cells stop codon by introducing an unnatural tRNA to the cells. This tRNA will compete with the release factor for the stop codon and will incorporate the unnatural amino acid from time to time.
<DIV class="imageText"><DIV class="right1"><SPAN class="caption">		
Figure 4: tRNAs have been developed which have affinity towards stop codons. These tRNAs can carry an unnatural amino acid and have a high affinity towards the targeted stop codon. This results in competition for the stop codon between release factors and the unnatural tRNA. If the tRNA binds to the ribosome, an unnatural amino acid will be incorporated into the growing peptide chain. This results in a protein which includes an unnatural amino acid. Not shown is the unnatural aminoacyl tRNA synthetase, which is necessary to couple the unnatural amino acid to the unnatural tRNA.
</SPAN></DIV></DIV>
The genetic code includes three distinct stop codons. One of these stop codons is known as the amber stop codon. The amber stop codon (TAG) is frequently targeted for the incorporation of unnatural amino acids as this is the least abundant stop codon in most organisms, including E. coli. <DIV class="spoiler" id="spoiler7"><SPAN class="spoilertext">
The <SPAN class="tekst1i">Escherichia coli</SPAN> genome has 4,290 open reading frames of which only 326 end with the amber stop codon <A name="reft1" href="#ref1" class="textanchor">[1]</A>. Therefore, amber tRNA suppressors have been used preferentially: hijacking the amber stop codon is expected to cause the least harm to E. coli cells. This does, however, not mean that amber stop codon suppression does not interfere with E. coli's natural function. Research has shown that hijacking the genetic code of E. coli does result in phenotype changes: Herring &amp; Blattner found that even though amber stop codon suppression does not lead to a stress response in E. coli, it does lead to transcriptional changes.
</SPAN></DIV></SPAN></DIV>
In our project, we make use of this amber stop codon to enable click chemistry on E. coli's outer membrane.
<H2 id="h2-4">p-Azido-phenylalanine</H2>

To click the aptamers to our membrane proteins post-translationally, we use the SPAAC click chemistry which was introduced to iGEM by Eindhoven 2014 (see Figure 5). Highlights of this click chemistry reaction are that it is bio-orthogonal, non-toxic and occurs very rapidly, both <SPAN class="tekst1i">in vivo</SPAN> as well as <SPAN class="tekst1i">in vitro</SPAN>. 
<DIV class="imageText"><DIV class="right1"><SPAN class="caption">
Figure 5: Strain-promoted Azide-Alkyne Cycloaddition (SPAAC) is a well known click chemistry reaction.
The reaction features two molecules, one functionalized with an azide group (A) and one functionalized
with a cyclooctyne group (B). SPAAC is very efficient and selective, no further reagents have to
be added to the reaction method and the reaction is completely bio-orthogonal. This image was adapted
from last year’s iGEM team’s wiki.
</SPAN></DIV></DIV>
In 2002, Chin et al. succeeded in mutating an existing tRNA synthetase such that it could incorporate p-Azido-L-Phenylalanine, an amino acid which is functionalized with an azide group <A name="reft2" href="#ref2" class="textanchor">[2]</A>. This enabled researchers to use cells to synthesize proteins to which any DBCO-modified compound can be attached post-translationally. We use this mechanism to be able to click our aptamers post-translationally to our membrane proteins. 
<H1 id="h1-3">
    The Vectors   
</H1><SPAN class="tekst1">
    As shown in Figure 1, our system relies on the presence of two vectors within the host cell. The first of these vectors is pEVOL-pAzF. The second one is the pETDuet-1 expression vector.
</SPAN><H2 id="h2-5">
    pEVOL-pAzF
</H2><SPAN class="tekst1">
    pEVOL-pAzF is a small vector which has been designed and optimized for the incorporation of unnatural amino acids into proteins in E. coli. The coding sequence of pEVOL-pAzF encodes tRNA synthetases, which can translate the amber stop codon sequence into the incorporation of the unnatural amino acid. Optimization of the vector has enabled higher yields of mutant proteins in comparison to previous vectors: pEVOL-pAzF showed roughly 250% greater yields in comparison with vectors previously used for the incorporation of unnatural amino acids <A href="#ref3" name="reft3">[3]</A>. One of the first amino acids which has been incorporated into proteins using the relatively novel pEVOL vector was pAzF and we will use this exact vector to construct our mutant protein in vivo.
</SPAN><DIV class="imageText"><DIV class="left1"><SPAN class="tekst1">
			Features of the pEVOL-pAzF vector:
			<UL><LI>The pEVOL-pAzF expression vector features the p15A origin of replication which makes the pEVOL-pAzF plasmid compatible 
					<DIV class="spoiler" id="spoiler5">
						Plasmid compatibility is generally defined as the failure of two coresident plasmids to be stably inherited in the absence of external selection <A href="#ref4" name="reft4">[4]</A>. The cause of the failure to be stably co-inherited lies in the fact that the origins of replication are too analogous. In that case, the bacteria cannot distinguish between the plasmids and can eventually lose either one of the plasmids as the amount of both plasmids is limited by a single copy number.
					</DIV>
					with many other frequently used plasmids.
				</LI><LI>
					The chloramphenicol resistance gene
				</LI><LI>
					The tRNA synthetase is under the control of the arabinose-inducible AraBAD promotor
				</LI></UL></SPAN></DIV><DIV class="right1"><SPAN class="caption">
			Figure 6: Simplified vectormap of the pAzF.
		</SPAN></DIV></DIV><H2 id="h2-6">
	pETDuet-1
</H2><SPAN class="tekst1">
	The designed system relies on the two membrane proteins which come into close proximity as a result of ligand binding. Often, when such protein assemblies are to be obtained, one can isolate endogenous complexes and reconstitute those components in vitro to analyze whether the assembly takes place <A href="#ref5" name="reft5">[5]</A>. As we have designed the system to be used in vivo, however, we rely on the co-expression of all components within the same host cell.
</SPAN><SPAN class="tekst1">
	Generally, this heterologous expression can be reached into two different ways, firstly by transforming multiple constructs and secondly by introducing a single plasmid carrying multiple genes in E. coli. As our device already featured two plasmids, we devised to use a plasmid which could co-express multiple genes, preferably in an equimolar ratio. Since the pETDuet-1<SUP>TM</SUP> Expression System from Novagen has been developed for this particular purpose, we have chosen to use this system as our vector of choice.
</SPAN><DIV class="imageText"><DIV class="columnleft"><SPAN class="tekst1">
			Features of the pETDuet-1 expression vector:
			<UL><LI>
					The pET-Duet1 vector features two multiple cloning sites, each carrying a dozen cloning sites. This enables insertion of multiple fragments.
				</LI><LI>
					Each of the multiple cloning sites contains its own T7 lac promotor and a ribosome binding site. A single terminator is located after MCS2, such that transcription yields two different mRNAs.
				</LI><LI>
					The pET-Duet1 vector not only carries multiple genes but is bicistronic. Therefore, it allows simultaneous expression of two proteins separately from the same RNA transcript. Hence, both the MCS1 insert and MCS2 insert are expressed from the longer mRNA strand. Only the MCS2 insert is expressed from the shorter mRNA strand.
					<DIV class="spoiler" id="spoiler6"><SPAN class="caption">
								Figure 8: Transcription of pETDuet-1 yields two different messenger RNA strands. The longer strain is bicistronic and encodes both the MCS1 as well as the MCS2 insert. The smaller strain only encodes MCS2. The smaller strain compensates for the reduced expression of MCS2 on the longer strand.
							</SPAN></DIV></LI></UL></SPAN></DIV><DIV class="columnright"><SPAN class="caption">
			Figure 7: General overview of the pETDuet-1 expression vector. The vector carries the ampicillin resistance gene and carries the pBR332 origin of replication
		</SPAN></DIV></DIV><SPAN class="tekst1">
	Usually, bicistronic vectors containing two target genes under the control of a single promotor preceding the two genes, show strongly reduced expression of the gene located more distant from the promotor site <A href="#ref6" name="reft6">[6]</A>. The second promotor which initiates the translation of the second mRNA aims to correct for this reduced expression.  
<DIV class="references"><SPAN class="caption"><A href="#reft1" name="ref1">[1]</A> Herring C.D. and Blattner F.R., “Global transcriptional effects of a suppressor tRNA and the inactivation of the regulator frmR”, J. Bacteriol., vol. 186, no. 20, pp. 6714–20, Oct. 2004. <A href="#reft2" name="ref2">[2]</A> Chin J.W., Santoro S.W., Martin A.B., King D.S., Wang L. and Schultz P.G., “Addition of p-Azido-L-phenylalanine to the Genetic Code of Escherichia c oli”, J. Am. Chem. Soc., vol. 124, no. 31, pp. 9026–7, Aug. 2002. <A href="#reft3" name="ref3">[3]</A> Young T.S., Ahmad I., Yin J.A. and Schultz P.G., “An enhanced system for unnatural amino acid mutagenesis in E. coli”, J. Mol. Biol., vol. 395, no. 2, pp. 361–74, Jan. 2010. <A href="#reft4" name="ref4">[4]</A> Novick R.P., “Plasmid incompatibility”, Microbiol. Rev., vol. 51, no. 4, pp. 381–95, Dec. 1987.<A href="#reft5" name="ref5">[5]</A> Busso D. et al., “Expression of protein complexes using multiple Escherichia coli protein co-expression systems: a benchmarking study”, J. Struct. Biol., vol. 175, no. 2, pp. 159–70, Aug. 2011. <A href="#reft6" name="ref6">[6]</A> Glück J.M., Hoffmann S., Koenig B.W. and Willbold D., “Single vector system for efficient N-myristoylation of recombinant proteins in E. coli”, PLoS One, vol. 5, no. 4, p. e10081, Jan. 2010. </SPAN></DIV></SPAN></DIV></DIV></DIV></DIV></BODY></HTML>