<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU_Eindhoven_Project_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU Eindhoven/Project/Design</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="infoTab"><DIV id="infoTabTitle">
Project - Design
</DIV><DIV id="infoTabContent"><DIV id="contentKop1" class="contentKop"><A href="#h1-1" class="contentLink">Overview</A></DIV><UL id="contentList1" class="contentList"><LI><A href="#h2-10" class="contentLink">The recognition elements - Aptamers</A></LI><LI><A href="#h2-11" class="contentLink">The scaffold - Outer Membrane Proteins</A></LI><LI><A href="#h2-12" class="contentLink">The signaling components</A></LI></UL><DIV id="contentKop2" class="contentKop"><A href="#h1-2" class="contentLink">The Recognition Element – Aptamers</A></DIV><UL id="contentList2" class="contentList"><LI><A href="#h2-1" class="contentLink">The Rise of Aptamers</A></LI><LI><A href="#h2-2" class="contentLink">Aptamers in Biotechnology</A></LI><LI><A href="#h2-3" class="contentLink">Aptamers in our Biosensor</A></LI><LI><A href="#h2-4" class="contentLink">Dual Aptamers</A></LI><LI><A href="#h2-5" class="contentLink">Split Aptamers</A></LI></UL><DIV id="contentKop3" class="contentKop"><A href="#h1-3" class="contentLink">The Scaffold - OmpX </A></DIV><UL id="contentList3" class="contentList"><LI><A href="#h2-6" class="contentLink">CPX &amp; OmpX</A></LI><LI><A href="#h2-7" class="contentLink">The Loops</A></LI></UL><DIV id="contentKop4" class="contentKop"><A href="#h1-4" class="contentLink">The Signaling Components</A></DIV><UL id="contentList4" class="contentList"><LI><A href="#h2-8" class="contentLink">Fast Signaling Components - Exploiting Bioluminiscence</A></LI><LI><A href="#h2-9" class="contentLink">Split Luciferases </A></LI></UL></DIV></DIV><DIV id="menuwrapper"><DIV id="menuTotal"><DIV id="menu"><A id="menu1" href="https://2015.igem.org/Team:TU_Eindhoven">
        Home</A><A id="menu2" href="https://2015.igem.org/Team:TU_Eindhoven/Team"> 
        Team</A><A id="menu3" href="https://2015.igem.org/Team:TU_Eindhoven/Project"> 
        Project</A><A id="menu4" href="https://2015.igem.org/Team:TU_Eindhoven/ResultsHome">
        Results</A><A id="menu5" href="https://2015.igem.org/Team:TU_Eindhoven/Modeling">
        Modeling</A><A id="menu6" href="https://2015.igem.org/Team:TU_Eindhoven/Practices">
        Policy Practices</A><A id="menu7" href="https://2015.igem.org/Team:TU_Eindhoven/Collaborations">
        Collaborations</A><A id="menu8" href="https://2015.igem.org/Team:TU_Eindhoven/Team/Sponsors">
        Sponsors</A><A id="menu9" href="https://2015.igem.org/Team:TU_Eindhoven/Safety">
        Safety</A></DIV></DIV><DIV id="submenuwrapper"><DIV id="submenu"><DIV id="sub2"><A id="sub2a" href="https://2015.igem.org/Team:TU_Eindhoven/Team/Members">Team members</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Team/Supervisors">Supervisors</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Attributions">Attributions</A></DIV><DIV id="sub3"><A id="sub3a" href="https://2015.igem.org/Team:TU_Eindhoven/Description">Description</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Project/Design">Design</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Project/Expression_System">Expression System</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Project/Experimental_Approach">Experimental Approach</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Project/Protocols">Protocols</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Notebook">Timeline</A></DIV><DIV id="sub4"><A id="sub4a" href="https://2015.igem.org/Team:TU_Eindhoven/Parts">Parts</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Results">Results</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Achievements">Medal Motivation</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Measurement">Interlab Study</A></DIV><DIV id="sub5"><A id="sub5a" href="https://2015.igem.org/Team:TU_Eindhoven/Modeling/Approach">Approach</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Modeling/Script">Script</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Modeling/Results">Results</A></DIV><DIV id="sub6"><A id="sub6a" href="https://2015.igem.org/Team:TU_Eindhoven/Policy_Practices/Human_Outreach">Outreach</A><A style="width:12%;" href="https://2015.igem.org/Team:TU_Eindhoven/Policy_Practices/Application_scenarios">Application Scenarios</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Policy_Practices/Stakeholders">Stakeholders</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Policy_Practices/Regulations">Regulations</A></DIV><DIV id="sub7"><A id="sub7a" href="https://2015.igem.org/Team:TU_Eindhoven/Collaborations/CloningGuide">Cloning Guide</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Collaborations/Collaboration_Attributions">Collaboration Attributions</A></DIV><DIV id="sub9"><A id="sub9a" href="https://2015.igem.org/Team:TU_Eindhoven/Safety/Lab">Lab Safety</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Safety/Project">Project Safety</A><A href="https://2015.igem.org/Team:TU_Eindhoven/Safety/LabEquipment">Lab Equipment</A></DIV></DIV></DIV></DIV><DIV id="body"><DIV id="container"><DIV id="wikiTour"><A style="text-decoration: none" href="https://2015.igem.org/Team:TU_Eindhoven/Project/Expression_System"><DIV id="moreDepth"><SPAN class="tekst1BI">Dig Deeper</SPAN><SPAN class="tekst1">
Read how unnatural amino acids enable click chemistry</SPAN></DIV></A><A style="text-decoration: none" href="https://2015.igem.org/Team:TU_Eindhoven/Project/Experimental_Approach"><DIV id="nextChapter"><SPAN class="tekst1BI">Next Chapter</SPAN><SPAN class="tekst1">
Explore how we tested our COMBs experimentally</SPAN></DIV></A></DIV><DIV id="containercontent"><DIV id="landingPage"><H1 id="h1-1">
Clickable Outer Membrane Biosensors - Design
</H1><SPAN class="tekst1"> 
We have developed Clickable Outer Membrane Biosensors as a universal membrane sensor platform for biosensors. Our dream is that <SPAN class="tekst1B">any chassis</SPAN> may be functionalized with COMBs. This chassis should then be able to <SPAN class="tekst1B">detect virtually any biomarker</SPAN> and translate detection of this biomarker into <SPAN class="tekst1B">any desired response</SPAN>. To obtain such a system, it is of the utmost importance that the system is inherently modular. The modularity of our system is reached through rational choices for the recognition element, the scaffold and the signaling components for our COMBs.
<DIV class="textImage"><DIV class="left2"><SPAN class="tekst1"><H3 id="h2-10">The recognition elements - Aptamers</H3>
Molecular recognition is central to biosensing <A name="reft1" href="#ref1" class="textanchor">[1]</A>. The moieties responsible for molecular recognition are better known as recognition elements. Recognition elements frequently used in biosensors are <SPAN class="tekst1B">aptamers</SPAN>. These aptamers are small oligonucleotides which fold spontaneously into intricate <SPAN class="tekst1B">three-dimensional structures</SPAN>. These structures <SPAN class="tekst1B">fit perfectly</SPAN> into a wide range of disease markers, with affinities reaching into the low nanomolar range. Through modifications, these aptamers can be attached post-translationally to our proteins, resulting in a truly modular membrane sensor platform. Read more on aptamers: </SPAN></DIV><DIV class="left2imglink"><DIV class="linkTxt"><SPAN class="caption">
Aptamers are folded oligo- nucleotides (orange) which perfectly fit into biomarkers (purple).
Image credit: Flanders' FOOD
</SPAN></DIV></DIV></DIV><DIV class="textImage"><DIV class="right2i"><H3 id="h2-11">The scaffold - Outer Membrane Proteins</H3><SPAN class="tekst1">
Outer membrane proteins form an essential component for Gram-negative bacteria, providing the bacteria with protection against a harsh environment <A name="reft2" href="#ref2" class="textanchor">[2]</A>. <SPAN class="tekst1B">More than six outer membrane protein families</SPAN> have been discovered, which all share a beta barrel secondary structure. Thanks to this structure, the proteins feature <SPAN class="tekst1B">loops protruding from the bacterial outer membrane</SPAN>, and <SPAN class="tekst1B">intracellular C-termini</SPAN>. This enables clicking of aptamers to the membrane proteins and fusing signaling components to the outer membrane proteins intracellularly, making outer membrane proteins the perfect scaffold for COMBs. Read more on outer membrane proteins: </SPAN></DIV><DIV class="right2imglink"><DIV class="linkTxt"><SPAN class="caption">
OMPs can feature as a scaffold as aptamers can be clicked on the protruding loops &amp; signaling components fused intracellularly.
</SPAN></DIV></DIV></DIV></SPAN></DIV><DIV class="textImage"><DIV class="left2"><H3 id="h2-12">The signaling components</H3><SPAN class="tekst1">
The signaling components used for COMBs translate a close proximity into a response. Signaling components which do exactly this are <SPAN class="tekst1B">split luciferases &amp; fluorescent and bioluminescent proteins</SPAN>, through Resonance Energy Transfer. In the future, TEV Proteases may be used to translate a proximity into the release of the transcription factor, coupling COMBs recognition to the vast amount of cellular responses bacteria have available. Read more our signaling components: </SPAN></DIV><DIV class="left2imglink"><DIV class="linkTxt"><SPAN class="caption">
COMBs signaling components translate a close proximity into a measurable signal. Here shown is BRET.</SPAN></DIV></DIV></DIV></DIV><H1 id="h1-2"> 
The Recognition Element – Aptamers    </H1><SPAN class="tekst1">
Since the discovery of the nucleic acid structure, nucleic acids were long thought to have a single function – storage of the heredity information as genetic instructions. Our perception on the function of DNA and RNA changed radically, however, as it was discovered that small RNA molecules could fold into a three-dimensional structure, exposing a surface onto which other small molecules could perfectly fit. 
Soon after the discovery that nucleic acids could have interaction with other molecules, Craig Tuerk and Larry Gold described a procedure, called SELEX, to isolate high-affinity nucleic acid ligands for proteins through a Darwinian-like evolution process carried out in vitro <A name="reft3" href="#ref3" class="textanchor">[3]</A>.
<DIV class="spoiler" id="spoiler2"> 
SELEX is an acronym for Systematic Evolution of Ligands by EXponential enrichment. The process is carried out in vitro and requires a very large oligonucleotide library (~10<SUP>15</SUP> members), which can be generated by combinatorial DNA Synthesis. SELEX starts with incubation of the library with the target molecule. This incubation is followed by a washing step which ensures that only oligonucleotides with higher affinities are selected. Next, the oligonucleotides which are still present in the mixture after the washing step are recovered and amplified. During this amplification step, mutations are introduced into the oligonucleotides, yielding a new library. This library may again be incubated with the target molecule. Typically, these steps are repeated 8-15 times. In the last round of SELEX the oligonucleotides still present in the mixture are identified by sequencing.
</DIV>
This procedure enabled researchers to discover oligonucleotides with high affinities and a perfect fit for arbitrary proteins, cells, small molecules and even viruses. After this perfect fit, these synthetic oligonucleotides were called ‘aptamers’ – stemming from the Latin aptus which can be translated into ‘fit’.
</SPAN><H2 id="h2-1">
The Rise of Aptamers
</H2><SPAN class="tekst1">
25 Years after the discovery of Tuerk’s and Gold’s invention of SELEX, aptamers have become available for hundreds of ligands, including proteins, viruses, other small molecules and even whole cells. 
<DIV class="spoiler" id="spoiler3">Aptamer sequences have been published for over 500 ligands. These ligands include small molecules, viruses, proteins and whole cells. Among these are some very notable ligands, such as HIV, cancerous cells and ricin, the extremely toxic compound which plays a prominent role in the critically acclaimed Breaking Bad series. These figures were obtained through Aptamer Base, a collaboratively created knowledge base about aptamers and the experiments that produced them, curated and maintained by a team of aptamer researchers. For more information, visit <A href="http://aptamerbase.semanticscience.org/" target="_blank"> Aptamer Base's website</A></DIV>
As such, aptamers have become a viable alternative for biology’s traditional recognition elements, antibodies.
Aptamers also provide numerous advantages over antibodies. As mentioned, aptamers are oligonucleotides whereas antibodies are proteins. As such, aptamers have a remarkable stability in a wide range of pH and temperatures, have higher shelf lifes, are non-toxic and lack immunogenicity <A name="reft4" href="#ref4" class="textanchor">[4]</A><A name="reft5" href="#ref5" class="textanchor">[5]</A>. Other advantages stem from the fact that aptamers are generated in vitro, whereas antibodies are generated through in vivo enrichment followed by purification through monoclonal cell lines <A name="reft6" href="#ref6" class="textanchor">[6]</A>. As a result, generation of aptamers is less laborious, production costs are lower and reproducibility is higher. Finally, in contrast to antibodies, aptamers can be generated against virtually any molecule, including toxins and poor immunogenic targets <A name="reft7" href="#ref7" class="textanchor">[7]</A>. This places aptamers amongst the most powerful tools in biotechnology <A name="reft8" href="#ref8" class="textanchor">[8]</A>. 
A major limitation of aptamers in comparison with antibodies is their stability in vivo, where nucleic acids are rapidly degraded. Aptamers which have been used as therapeutic agents, for example, suffered from a half-life of 2 minutes <A name="reft9" href="#ref9" class="textanchor">[9]</A>. This problem has, however, partially been overcome by using chemical modifications to the aptamers. An example of these modifications are Spiegelmers <A name="reft8" href="#ref8" class="textanchor">[8]</A>. These oligonucleotides’ backbones contain L-Ribose instead of R-Ribose – “spiegel” means mirror. These mirrored aptamers are more stable in vivo, because they suffer less from degradation.
</SPAN><H2 id="h2-2">
Aptamers in Biotechnology
</H2><SPAN class="tekst1">
Since aptamers can be easily modified chemically, they can be attached to numerous surfaces, enabling their way into a wide array of biosensors and drug delivery systems. Generally, the biosensors couple binding of the aptamer to a change in structure in the latter, which generates either a fluorescent or electrical signal. Aptamers have also found their way into drug delivery systems. A particular example of such a system is developed by Wu et al., featuring aptamers targeted at cancer cells with lipid tails. These aptamers could be used as building blocks of micelles, enabling efficient delivery of the micelles to cancerous cells <A name="reft10" href="#ref10" class="textanchor">[10]</A> .
</SPAN><DIV id="imageText"><H2 id="h2-3">
Aptamers in our Biosensor
</H2><DIV class="left1"><SPAN class="tekst1">
Aptamers constitute the ideal sensor domain for our universal membrane sensor: they can be targeted at virtually any molecule, can easily be attached to the membrane using SPAAC chemistry and offer affinities similar to antibodies. Taking into account their remarkable stability and small size, they are unmatched by the traditional recognition elements of biotechnology, antibodies. An essential requirement for our system which limits the choice of aptamers is bifunctionality: the system relies on bringing two membrane proteins in close proximity by binding the ligand with two different moieties. Conceptually, this bifunctionality can be reached either through the use of dual aptamers or through the use of split aptamers.
</SPAN></DIV><DIV class="right1"><SPAN class="caption">
Figure 1: Bifunctionality is required to bring the membrane proteins into close proximity. 
</SPAN></DIV></DIV><DIV id="imageText"><DIV class="columnleft"><H2 id="h2-4">
Dual Aptamers
</H2><SPAN class="tekst1">
The first way in which bifunctionality can be reached is through the use of dual aptamers acting on different binding sites, called apitopes. A well-known example of a ligand for which dual aptamers are available is human thrombin. A drawback of the use of dual aptamers is that these are in general not available for small molecules and viruses, significantly reducing the available targets for the aptamers. The main target for dual aptamers are proteins. </SPAN><SPAN class="caption">
Figure 2: A) Some ligands have multiple apitopes, such that a dual approach can be taken. B) Thrombin (yellow) is one of the protein ligands which has two apitopes, allowing different aptamers to bind at different spots.
</SPAN></DIV><DIV class="columnright"><H2 id="h2-5">
Split Aptamers
</H2><SPAN class="tekst1">
Alternatively, aptamers can be split into two parts. These parts can form heterodimers and thereby self-assemble into the three-dimensional structure required for binding the apitope. A crucial choice in engineering split aptamers is the choice of splitting site <A name="reft11" href="#ref11" class="textanchor">[11]</A>. This choice is often not straightforward and the aptamer often suffers from a significantly decreased affinity. Obtaining split aptamers is, however, still in its infancy: of the approximately 100 small-molecule-binding aptamers, only six have been successfully engineered into split aptamers <A name="reft11" href="#ref11" class="textanchor">[11]</A>. </SPAN><SPAN class="caption">
Figure 3: A) Small molecules generally have only a single apitope. An alternative to the dual approach is the construction
of split aptamers. B) Engineering of split aptamers is complicated, but split-aptamers for three-way junction aptamers could be obtained. The image under B is adapted from Kent et al. <A name="reft11" href="#ref11" class="textanchor">[11]</A>.
</SPAN></DIV></DIV><H1 id="h1-3"> 
The Scaffold - OmpX     </H1><SPAN class="tekst1">
To realize conduction of a signal over the cell membrane, the aptamers have to be attached to a membrane protein. A procedure to attach oligonucleotides to membrane proteins has succesfully been tested by iGEM TU Eindhoven 2014. They presented this procedure as a part of their Click Coli project to the iGEM community. The procedure exploits the SPAAC reaction, which is one of the most well-described click chemistry reactions available (see Figure 4). </SPAN><DIV id="imageText"><DIV class="right1"><SPAN class="caption">
Figure 4: Strain-promoted Azide-Alkyne Cycloaddition (SPAAC) is a well known click chemistry reaction.
The reaction features two molecules, one functionalized with an azide group (A) and one functionalized
with a cyclooctyne group (B). SPAAC is very efficient and selective, no further reagents have to
be added to the reaction method and the reaction is completely bio-orthogonal. This image was adapted
from last year’s iGEM team’s wiki.
</SPAN></DIV></DIV><SPAN class="tekst1">
As a proof of concept, iGEM TU Eindhoven employed two membrane proteins as anchors, to
which they anchored DNA. These membrane proteins were CPX and INPNC. CPX in particular
showed promising results and is still being explored today.
</SPAN><H2 id="h2-6">
CPX &amp; OmpX
</H2><SPAN class="tekst1">
CPX is a membrane display protein developed by Rice et al. as an alternative to traditional display
methodologies, such as yeast display and phage display <A name="reft12" href="#ref12" class="textanchor">[12]</A>. The protein itself is derived from the
naturally occuring Outer membrane protein X (OmpX) through circular permutation (see Figure
5). Circular permutation was a necessary step to obtain the termini on the exterior of the bacterial
cells.</SPAN><DIV id="imageText"><DIV class="right1"><SPAN class="caption">
Figure 5: CPX (right) is a membrane protein derived from OmpX (left) through circular permutation.
OmpX and CPX feature signal sequences (yellow) which ensure that the membrane protein is localized
to E.coli’s outer membrane. These signal sequences are cleaved from the peptides after localization. In
CPX, an amber stop codon was introduced to introduce a unnatural azide-functionalized amino acid. 
</SPAN></DIV></DIV><SPAN class="tekst1">
iGEM Eindhoven 2014 inserted an amber stop codon (TAG) at the N-terminus to enable expression
of an azide-functionalized amino acid. This amino acid could be used to attach DBCO-functionalized
molecules to the membrane protein, allowing synthetic biologists to attach virtually
any molecule to the cell membrane (see Figure 6). The now azide-functionalized CPX-protein was
dubbed Clickable Outer Membrane Protein x (COMPx) by iGEM TU Eindhoven 2014. </SPAN><DIV id="imageText"><DIV class="right2text"><SPAN class="caption">
Figure 6: The azide-functionalized amino acid enables clicking any DBCO-functionalized molecule to
be attached to the cell membrane through SPAAC click chemistry. iGEM TU Eindhoven 2014 called this
azide-functionalized CPX COMPx.
</SPAN></DIV></DIV><SPAN class="tekst1">
To enable signaling over the outer membrane, our system requires intracellular signaling domains.
Unfortunately, the COMPx developed and characterized by last year’s iGEM team does
not allow for fusing protein domains intracellularly. To obtain fusion domains with additional
intracellular domains, we devised to revert to go back to the basis and use OmpX rather than CPX
as the platform for our clickable outer membrane proteins. The azide-functionalized amino acid will be
introduced in the loops of OmpX, to which the aptamers can be attached (see Figure 7). </SPAN><DIV id="imageText"><DIV class="right1"><SPAN class="caption">
Figure 7: For our signaling proteins, we revert to the basis: OmpX. We functionalize OmpX by introducing
the amber stop codon into its loops (red). The signaling domains are fused to OmpX C-terminally,
since the N-terminus is occupied by the signaling sequence (yellow). The signaling sequence is cleaved
from OmpX after localization.
</SPAN></DIV></DIV><H2 id="h2-7">
The Loops
</H2><SPAN class="tekst1">
The amber stop codon will be incorporated in the protruding loops of OmpX. We have chosen
to introduce the mutations into the loops since they are easily accessible and are not a part of the
beta-barrel of OmpX [<A name="reft12" href="#ref12" class="textanchor">12]</A>. Hence, we believe that mutations in the loops will not disturb the secondary
structure of OmpX (see Figure 8B). </SPAN><DIV id="imageText"><DIV class="right1"><SPAN class="caption">
Figure 8: A) The OmpX protein structure has been elucidated through NMR and X-ray crystallography.
B) The square residues are important for the secondary structure of OmpX. To keep the structure
intact, we introduce an amber stop codon in one of the protruding loops. Figure 8B is adapted from <A name="reft13" href="#ref13" class="textanchor">[13]</A>.
</SPAN></DIV></DIV><DIV id="imageText"><DIV class="left1"><SPAN class="tekst1">
The to be substituted residues in the loops were chosen in such a way that the clicked aptamer would protrude from
OmpX. Moreover, residues were selected such that the signaling protein's structure closely resembles OmpX natural’s
structure. The serine residue in loop 2 was chosen since it was the only available residue with side-groups
pointing outward from OmpX. The tyrosine residue in loop 3 was chosen both because tyrosine closely resembles
the unnatural azide-functionalized amino acid (see Figure 9) and because the tyrosine points outward from
OmpX. </SPAN></DIV><DIV class="right1"><SPAN class="caption">
Figure 9: The unnatural azide-functionalized amino acid closely resembles tyrosine. 
</SPAN></DIV></DIV><H1 id="h1-4"> 
The Signaling Components      </H1><SPAN class="tekst1">
Quite paradoxically, the number of signaling proteins pales in comparison with the vast amount of
cellular responses. This signaling paradox can be explained by the fact that cells employ amazingly
complex signaling pathways, which feature signal amplification, signal integration and cross communication
<A name="reft14" href="#ref14" class="textanchor">[14]</A>. In addition to the vast amount of effects signaling pathways can have, these pathways
operate over a staggering range of time scales; some signaling pathways act within less than
a second, whereas others affect cell behavior only after hours or days.
An essential aspect of a universal membrane sensor is that it allows for these wide ranges of time
scales and responses. We believe that the modularity of our membrane sensor and the complexity
of intracellular cell signaling allows for a wide range of responses. To enable the sensors to act
upon a wide range of time scales we consider multiple intracellular signaling components. These
components all respond to a decrease in mutual distance of the membrane proteins as a result of
ligand binding.
</SPAN><H2 id="h2-8">
Fast Signaling Components - Exploiting Bioluminiscence
</H2><SPAN class="tekst1">
Two well-known principles to translate a close proximity into a measurable signal are Resonance
Energy Transfer and the use of split luciferases. Even though these principles are very different,
both response elements emit light, yielding a measurable signal. After ligand binding, these signaling
components yield a virtually immediate response.
</SPAN><DIV id="imageText"><DIV class="right1"><SPAN class="caption">
Figure 10: A) Resonance Energy Transfer and B) split luciferases both translate close proximity into a
measurable signal in the form of light.
</SPAN></DIV></DIV><SPAN class="tekst1">
Resonance Energy Transfer is a physical process which can take place between fluorophores light
emission when they are in close proximity (1-10nm) <A name="reft15" href="#ref15" class="textanchor">[15]</A>. An electron which has been transferred to
its ‘excited state’ falls back to its ‘ground state’. The energy which is released by the electron falling
back to its ground state is normally released in the form of light. In the case of Resonance Energy
Transfer, however, the energy of the electron falling back to its ground state is coupled to a transition of an electron in the RET Acceptor from its ground state
to the excited state (see Figure 11). 
</SPAN><DIV id="imageText"><DIV class="right1"><SPAN class="caption">
Figure 11: Simplified energy-level diagram of RET. A) Normally, an excited electron falls back to
its ground state under the emission of light (a radiative transition). B) In the case of Resonance
Electron Transfer, an excited electron in the donor falls back to its ground state. This transition is coupled
to the excitation of an electron in the acceptor. This excited electron falls back normally under the
emission of light.
</SPAN></DIV></DIV><SPAN class="tekst1">
The efficiency of Resonance Energy Transfer is known to be dependent on a few factors, most
importantly the relative orientation of the chromophores as well as the mutual distance of the
chromophores <A name="reft15" href="#ref15" class="textanchor">[15]</A>. A decreased mutual distance between the donor and acceptor increases the efficiency very significantly. This feature of Resonance Energy Transfer is exploited in the design
of our membrane sensor. As a result of the reduced distance between the donor and acceptor, Resonance Energy
Transfer is thus more efficient. Relatively more light will be emitted by the acceptor
when the ligand is bound. The resulting signal is thus, per definition, ratiometric. Intensity of the light emitted by the donor will decrease and intensity of the light emitted by the acceptor will increase as a consequence of ligand binding (see Figure 12).
</SPAN><DIV id="imageText"><DIV class="right1"><SPAN class="caption">
Figure 12: A) Efficient energy transfer is not possible since the donor and fluorophore are to remote.
B) Efficient energy transfer is possible due to the proximity of the donor and acceptor. As a
result, light intensity of the donor decreases whereas light intensity of the acceptor increases.
</SPAN></DIV></DIV><H2 id="h2-9">
Split Luciferases
</H2><SPAN class="tekst1">
Luciferases are naturally occuring proteins which convert a substrate to a product and light.
Many organisms feature luciferases to produce light, such as the sea spansy, fireflies and shrimps.
These luciferases have been isolated from these organisms and can be succesfully used in vitro to
produce light.
Some luciferases have also been succesfully split into two complementary parts. These complementary
parts can self-assemble into the functional luciferase, restoring its functionality to emit
light by converting the substrate. As such, these split luciferases have become a major tool to
study protein-protein interactions: these interactions are coupled to a close proximity of the proteins,
allowing the parts to assemble into the functional luciferase and emit light (see Figure 13).
We use these split luciferases in addition to BRET as a rapid signaling component.
</SPAN><DIV id="imageText"><DIV class="left1"><SPAN class="caption">
Figure 13: A) The split parts of the luciferase are too remote to assemble into the functional luciferase. B) Upon ligand binding, the two luciferases can self-assemble into the functional luciferase.
restoring its capability to produce light from the substrate, yielding a measurable signal.
</SPAN></DIV></DIV><DIV class="references"><SPAN class="caption"><A href="#reft1" name="ref1">[1]</A>	Chambers J.P., Arulanandam B.P., Matta L.L., Weis A. and Valdes J.J., “Biosensor recognition elements”, Curr. Issues Mol. Biol., vol. 10, no. 1, pp. 1–12, 2008. <A href="#reft2" name="ref2">[2]</A>    Koebnik R., Locher K.P. and Van Gelder P., “Structure and function of bacterial outer membrane proteins: barrels in a nutshell”, Mol. Microbiol., vol. 37, no. 2, pp. 239–53, Jul. 2000. <A href="#reft3" name="ref3">[3]</A>	Tuerk C. and Gold L., “Systematic evolution of ligands by exponential enrichment: RNA ligands to bacteriophage T4 DNA polymerase”, Science, vol. 249, no. 4968, pp. 505–10, Aug. 1990.<A href="#reft4" name="ref4">[4]</A>	Musumeci D. and Montesarchio D., “Polyvalent nucleic acid aptamers and modulation of their activity: A focus on the thrombin binding aptamer”, Pharmacol. Ther., vol. 136, no. 2, pp. 202–15, Jul. 2012.<A href="#reft5" name="ref5">[5]</A>	Cibiel A., Pestourie C. and Ducongé F., “In vivo uses of aptamers selected against cell surface biomarkers for therapy and molecular imaging”, Biochimie, vol. 94, no. 7, pp. 1595–1606, Feb. 2012.<A href="#reft6" name="ref6">[6]</A>	Lauridsen L.H. and Veedu R.N., “Nucleic acid aptamers against biotoxins: a new paradigm toward the treatment and diagnostic approach”, Nucleic Acid Ther., vol. 22, no. 6, pp. 371–79, Oct. 2012.<A href="#reft7" name="ref7">[7]</A>	Rhouati A., Yang C., Hayat A. and Marty J.L., “Aptamers: A promosing tool for ochratoxin a detection in food analysis”, Toxins (Basel)., vol. 5, no. 11, pp. 1988–2008, Nov. 2013.<A href="#reft8" name="ref8">[8]</A>	Radom F., Jurek P.M., Mazurek M.P., Otlewski J. and Jeleń F., “Aptamers: Molecules of great potential”, Biotechnol. Adv., vol. 31, no. 8, pp. 1260–74, Apr. 2013.<A href="#reft9" name="ref9">[9]</A>	Hicke B.J. et al., “Tumor Targeting by an Aptamer”, J. Nucl. Med., vol. 47, no. 4, pp. 668–78, Apr. 2006.<A href="#reft10" name="ref10">[10]</A>	Wu Y., Sefah K., Liu H., Wang R. and Tan W., “DNA aptamer-micelle as an efficient detection/delivery vehicle toward cancer cells”, Proc. Natl. Acad. Sci. U. S. A., vol. 107, no. 1, pp. 5–10, Nov. 2009.<A href="#reft11" name="ref11">[11]</A>	Kent A.D., Spiropulos N.G. and Heemstra J.M., “General approach for engineering small-molecule-binding DNA split aptamers”, Anal. Chem., vol. 85, no. 20, pp. 9916–23, Oct. 2013.<A href="#reft12" name="ref12">[12]</A>	Rice J.J., Schohn A., Bessette P.H., Boulware K.T. and Daugherty P.S., “Bacterial display using circularly permuted outer membrane protein OmpX yields high affinity peptide ligands”, Protein Sci., vol. 15, no. 4, pp. 825–36, Apr. 2006.<A href="#reft13" name="ref13">[13]</A>	Vogt J. and Schulz G.E., “The structure of the outer membrane protein OmpX from Escherichia coli reveals possible mechanisms of virulence”, Structure, vol. 7, no. 10, pp. 1301–9, Oct. 1999.<A href="#reft14" name="ref14">[14]</A>	Alberts, Johnson, Lewis, Raff, Roberts, Molecular Biology of The Cell. Pearson, 2005.<A href="#reft15" name="ref15">[15]</A>	Medintz I. and Hildebrandt N., Eds., FRET - Förster Resonance Energy Transfer. Weinheim, Germany: Wiley-VCH Verlag GmbH &amp; Co. KGaA, 2013. </SPAN></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>