Team:TU Eindhoven/Project/Experimental Approach
Project - Experimental
Verifying the click reaction
Measuring bioluminescence & fluorescence
Verifying whether oligonucleotides click
DNA Strand Displacement
Home
Team
Project
Results
Modeling
Policy Practices
Collaborations
Sponsors
Safety
Team members Supervisors Attributions
Description Design Expression System Experimental Approach Protocols Timeline
Parts Results Medal Motivation Interlab Study
Approach Script Results
Outreach Application Scenarios Stakeholders Regulations
Cloning Guide Collaboration Attributions
Lab Safety Project Safety Lab Equipment
Dig Even Deeper
See our Notebook
See our Protocols
Next Chapter
See how modelling helped us understand our device
Previous Chapter
Missed something? Jump back to our Project page
Experimental approach
We have designed a universal membrane platform which is inherently modular and can elicit both slow as well as fast responses. To verify the viability of the platform, we devised characterizing the fast cellular responses. In the construction of our device, we planned on conducting a wide range of experiments to verify whether the individual elements of our system worked. These elements include whether the click reaction occurs, verifying whether the signaling components work, whether proximity invokes a response and whether the DBCO-modified aptamers work. An overview of the experiments is given below.
Verifying the click reaction
A vital aspect of our device is clicking the aptamers to the membrane proteins. For this click, we made use of the exact same click chemistry used by iGEM TU Eindhoven 2014. The click reaction was used N-terminally by iGEM TU Eindhoven 2014, in order to minimize sterical strain. To analyze whether the localization of the azide-functionalized amino acid within the loops of OmpX impedes the click reaction, we clicked a DBCO-functionalized fluorophore (TAMRA) to the outer membrane proteins (see Figure 1). After some washing steps and spinning down, we expected the cells to remain fluorescent.
Figure 1: To verify whether the click reaction has occured, we incubate the cells with DBCO-functionalized TAMRA. If the outer membrane protein is functionalized with the unnatural amino acid, this TAMRA dye binds to the membrane proteins covalently. In that case, the cells will remain fluorescent after a few washing steps.
Since the fluorescence of the TAMRA-dye falls within the visible spectrum, we expected to see whether the dye clicked on the outer membrane proteins with the naked eye. To analyze the fluorescence at the single-cell level, we measured cells using the Fluorescence-Activated Cell Sorter (FACS).
A Fluorescence-Activated Cell Sorter (FACS) is a specialized flow cytometer (see Figure B). The FACS can provide information about cell size, complexity and fluorescence.    The relative cell complexity is measured using size scatter (SSC).   The relative cell size is measured using forward scatter (FSC).   The fluorescence can be measured by using a wide range of filters.    These cell characteristics can be combined to sort cells.
Measuring bioluminiscence & fluorescence
A crucial element in the sensor system is the reporter system. For our COMBs, we have considered several options for reporter systems. As a proof of concept, we will use the split luciferase (NanoBiT) and BRET reporter systems. Both these reporter systems can easily be followed through the measurement of bioluminescence and fluorescence.
The NanoBiT reporter system is a split luciferase system. The system can only generate a response if both parts of the split luciferase system, LgBiT and SmBiT, are available and in close proximity. Moreover, the substrate furimazine has to be present to obtain a signal.
The BRET system relies on the presence of a bioluminescent and fluorescent protein. In our system, we have used NanoLuc in combination with mNeonGreen. To verify the presence of NanoLuc, a bioluminescent assay will be conducted. To verify the presence of mNeonGreen, fluorescence will be measured.
Verifying whether oligonucleotides click
A third verification step is to see whether or not the click reaction takes place and if this click reaction takes place equimolarly. Therefore, oligos will be clicked to the outer membrane proteins and incubated with complementary strands that are fluorescently labeled (see Figure 2). After some washing steps and spinning down, it is expected that the fluorescent strands will remain if and only if the oligos succesfully click. Thereby, fluorescence indicates whether oligonucleotides can be succesfully clicked to the COMBs.
Figure 2: To verify whether the click reaction has occured with DBCO-modified DNA, we incubate the cells with DBCO-functionalized complementary strands. If the DBCO-modified DNA clicks, the fluorescently labeled DBCO-functionalized complementary strands will anneal with the clicked strands. In that case, the cells will remain fluorescent after a few washing steps.
DNA Strand Displacement
As we have already touched upon lightly, it has long been thought that nucleic acids had only a single role: carrying hereditary information. As discussed, the discovery that DNA could fold into higher-order structures gave way to SELEX, an evolutionary method of discovering aptamers. The construction of DNA nanostructures, however, was not only carried out through a combinatorial approach; the specificity and predictability of Watson-Crick basepairing enabled rational design for engineering at the nanoscale [1] . Initially, the designed DNA nanostructures were mostly static, but dynamic nanostructures have become available over the years. Most of these dynamic nanostructures share a common feature: they exploit a biophysical phenomenon known as DNA strand displacement.
DNA strand displacement is the workhorse of dynamic DNA technology, the field which uses DNA’s non-covalent interactions to assemble higher-order structures. It is a process where two strands with partial complementarity are hybridized. These pre-assembled DNA strands have only partial complementary, leaving room for a toehold region. When a DNA sequence with full complementary binds to this region (the input), branch migration takes place (see the figure below). In the end, the sequence with full complementary binds to the sequence, yielding the output.
Figure C: Schematic overview of DNA strand displacement. DNA Strand displacement is initiated when the input binds to the toehold region of a partially hybridized DNA strand. After the initiation, branch migration takes place and the input fully hybridizes with its perfect match, yielding the output.
DNA strand-displacement is a very robust technology; the sequences of the used strands often go unreported as they play a minor role. The robustness of the technology enabled rational design of numerous nanostructures. These nanostructures include DNA walkers, strand displacement cascades and self-assembling dendrimers [1] .
In our project, we want to bring two membrane proteins in close proximity through aptamers. To test whether a close proximity indeed triggers an intracellular signal, we used DNA to bring the membrane proteins in close proximity. DNA is the ideal probe for this purpose, as its high specificity and predictability allowed us to bring the membrane proteins in vicinity. To make the system with DNA reversible, we designed a simple system exploiting the strand displacement (see Figure 3).
Figure 3: The membrane proteins can be brought in close proximity by clicking oligonucleotides on the loops and adding a long strand complementary to both oligonucleotides. It is expected that this results in a measurable signal. The longer strand is functionalized with a toehold region. Upon addition of a strand perfectly complementary to the longer strand, the system disassembles and the signal will fade out.
[1] Zhang D.Y. and Seelig G., “Dynamic DNA nanotechnology using strand-displacement reactions.,” Nat. Chem., vol. 3, no. 2, pp. 103–13, Mar. 2011.
