Team:IIT Kharagpur/Demo Protocol
IGEM-IIT Kharagpur- Protocol
Toggle navigation
IITKharagpur
Home
Project
Team
Parts
Practices
Interlab
Blog
PROTOCOL
SILKOTRON
PROTOCOL - TRANSFORMATION
Approximation time:
120 mins
Material required:
70% ethanol
Tissue paper
Ice
Container for ice
Timer
DH5alpha cells
LB agar plates – Chloramphenicol, (kanamycin+ ampicillin), Kanamycin
Protocol:
1. Keep all materials on ice unless otherwise specified! This will help make the cells more competent and easier to transform.
2. Label a 2.0ml microcentrifuge tube as Transformation Control, another as Ligation: New Part, and one more as Ligation Control.           (Add 20ul of the New Part ligation product into the Ligation: New Part tube.)
3. Place the tubes on ice to pre-chill them.
4.   Thaw one competent cell aliquot tube on ice (this takes about 5-8 minutes).
5.   Gently flick the tube of competent cells, then pipet 50ul of competent cells into each 2.0ml microcentrifuge tube.   (Try to keep the cells as cold as possible by holding just the top of the tube, not the bottom where the cells are.)
6.    Incubate the DNA and cell mixtures on ice for 30 minutes. During this incubation, pre-heat the waterbath to 40°C.
7.    Place the tubes into the waterbath for 90 seconds. Immediately place the tubes back on ice for 10 minutes.
8.    Add 1000ul of LB media to each tube. Gently tap the tubes with your finger to mix.
9.    Incubate the tubes at 37°C for 1 hours.
10.  Pipet 200ul of the Transformation Control onto the appropriate plate. Spread evenly over the surface of the agar by gently shaking the plate back and forth. The beads will do the work for you!
11.  Repeat step 1 to 10 for the other two transformations.
12.  Place the agar plates into the incubator with the agar side facing up, lid facing down. Incubate the agar plates at 37°C for 12 - 14 hours. Alternately, incubate at room temperature for 24 hours.
Safety:
● Gloves must be worn all the time.
● Every step must be done in the hood to avoid contamination and 4 degrees needs to be maintained during the procedure.
PROTOCOL - RESTRICTION DIGEST
Approxmation time:
6hrs
Material required:
70% ethanol
Tissue paper
Ice
Container for ice
DNA sample
RFP Control
NEB buffer 2
NEB enzymes: EcoRI/SpeI/XbaI/PstI
pSB1C3/pSB1K3/pSB1A3/pSB1AK8( linearized plasmid backbone)
BSA
Protocol:
1.  Clean the lab bench by wiping down with 70% ethanol and paper towels.
2.  Keep all enzymes and buffers used in this section on ice.
3.  Thaw NEB Buffer 2 and BSA in room temperature water. Re-homogenize both by inverting the tubes, and flick/spin them to collect the liquid at the bottom of the tube.
4.  Label three 0.6 tubes: DNA sample, (pSB1C3/pSB1K3/pSB1A3/pSB1AK8) (linearized plasmid backbone), and RFP Control
5.     Add 500ng of DNA to the appropriate tube. Add distilled water to the tubes for a total volume of 42.5ul in each tube.
.  Pipet 5ul of Buffer 2 to each tube.
2.  Pipet 0.5ul of BSA to each tube.
3.  In the DNA sample: Add 1ul of EcoRI/SpeI/XbaI/PstI enzyme.
4.  In the pSB1C3/pSB1K3/pSB1A3/pSB1AK8  tube: Add 1ul of restricted enzyme(EcoRI/SpeI/XbaI/PstI).
5.  In the RFP Control tube: Add 1ul of restricted enzyme(EcoRI/SpeI/XbaI/PstI).
6.  The total volume in each tube should be approximately 50ul. Mix well by pipetting slowly up and down 5x. Be gentle, and do not vortex. Spin the samples for 5 seconds in a microcentrifuge, or flick them to collect all of the mixture to the bottom of the tube.
7.  Incubate the restriction digests at 37°C for 30 minutes, then 80°C for 20 minutes. We use a thermocycler, but a waterbath and an accurate thermometer works well also!
8.     The digested DNA can be stored at 4°C for a few days. For longer storage, keep at -20°C.
PROTOCOL - LIGATION
Approximation time:
12hrs
Materials required:
70% ethanol
Paper towels
Distilled water
Ice
Container for ice
T4 DNA Ligase Reaction Buffer
T4 DNA Ligase
Thermocycler, or waterbath
Restriction Digest: Part A and Part B
Restriction Digest: (pSB1K3/pSB1C3/pSB1A3/pSB1AK8)linearized plasmid backbone
Protocol:
1. Clean the lab bench by wiping down with 70% ethanol and paper towels.
2. Thaw T4 DNA Ligase Reaction Buffer at room temperature. Keep the T4 DNA Ligase in the freezer until you're ready to use it.
3.     Label one 0.6ml tube as New Part.
1.  Add 2ul from the (pSB1K3/pSB1C3/pSB1A3/pSB1AK8)linearized plasmid backbone digest.
2.  Add 3.3ul from the Part A digest.
3.  Add 3.9ul from the Part B digest.
4.  Add 1ul of T4 DNA Ligase Reaction Buffer.
5.  Add 0.5ul of T4 DNA Ligase (keep this at -20°C until use!).
6.  Mix by gently pipetting up and down 3x. Do not vortex; this inactivates the enzymes. Place tube in microcentrifuge for a quick 5 second spin or flick the tube to collect the mixture at the bottom.
4.     Label one 0.6 tube as Ligation Control.
1.  Add 2ul from the RFP Control digest.
2.  Add 6.5ul of distilled water.
3.  Add 1ul of T4 DNA Ligase Reaction Buffer.
4.  Add 0.5ul of T4 DNA Ligase.
5.  Mix by gently pipetting up and down 3x. Do not vortex; this inactivates the enzymes. Place tube in microcentrifuge for a quick 5 second spin or flick the tube to collect the mixture at the bottom.
5. Incubate at 16°C for 30 minutes, then at 80°C for 20 minutes. We use a thermocycler, but a waterbath and thermometer combination works great too! The ligated products can be stored at -20°C.
Ligation: New part
Ligation: control
Digest 1
2µl backbone
2µl RFP control
Digest 2
3.3µl Part A
-
Digest 3
3.9µl Part B
-
Distill water
0µl
6.5µl
T4 DNA Ligase
0.5µl
0.5µl
T4 DNA Ligase buffer
1µl
1µl
PROTOCOL - PLASMID ISOLATION
Approximation time:
3hrs
Material required:
Qiagen Miniprep Kit
Cell Culture
1.6 ml Microcentrifuge tubes
TE (1:10)
Protocol
1. Spin the cell culture in a centrifuge to pellet the cells, empty the supernatant (media) into a waste collection container.
2. Resuspend pelleted bacterial cells in 250 µl Buffer P1 (kept at 4 °C) and transfer to a          microcentrifuge tube. No cell clumps should be visible after resuspension of the pellet.
3. Add 250 μl Buffer P2 and gently invert the tube 4–6 times to mix. Do not vortex, as this will result in shearing of genomic DNA. If necessary, continue inverting the tube until the solution becomes viscous and slightly clear.
4. Add 350 μl Buffer N3 and invert the tube immediately and gently 4–6 times. To avoid localized precipitation, mix the solution gently but thoroughly, immediately after addition of Buffer N3. Keep in ice for 10 minutes.
5. Centrifuge for 25 min at 12,000 rpm in a  microcentrifuge. A White pellet will form.
6. Load the supernant in QIAprep column and wait for 10 minutes. Discard the flow-through
7. Wash QIAprep spin column by adding 0.75 ml Buffer PE and centrifuging for 1 minute.
8. Discard the flow-through, and centrifuge for an additional 1 min at 10,000 rpm to remove residual wash buffer.
9. Discard the flow-through, and centrifuge for an additional 2 min at 11,000 rpm. Discard the flow-through and transfer the column to fresh eppendorf tube.
10. Add 20 – 50µL of Nuclease free water in the column. Centrifuge at 11000 rpm for 2 minutes for eluting the DNA bound to the Column.
PROTOCOL - PREPARATION OF COMPETENT CELLS (DH5α)
Approximation time:
Material required:
100 mM CaCl2
200mL LB Media
sterile centrifuge tubes
Overnight DH5α culture
Protocol:
1. Add 200μl of overnight culture to 10ml of fresh LB medium for preparation of overday (O/D) culture.
2. Keep the O/D culture in incubator(37°C) for 2.5 hours.
3. Keep the O/D culture in refrigerator for some time and then transfer it to 15ml tubes.
4. Centrifuge at 300 rpm and 4°C for 10 minutes.
5. Decant the supernatant and gently resuspend each pellet in about 5mL of  100mM ice cold CaCl2.
6. Again centrifuge the CaCl2 suspended cells at 4000rpm for 4 minutes at 4 degrees       And discard the supernatant. Add 1ml of CaCl2 and resuspend the cells and aliquote 100uL into sterile micro centrifuge tubes of 1.5ml tubes and store at 4 degrees for overnight. (Which  can be used to transform the ligated products)
PROTOCOL - AGAROSE GEL PERPARATION
Material required:
1X TAE buffer
Ethidium bromide
Protocol:
1. Mix 0.4 grams of agarose with 40ml of TAE buffer in a conical flask.
2. Weigh the solution and then heat in the oven for 2 mins.
3. Add 4 µl (10mg/ml) of EtBr in the solution.
4. Weigh it again and add the TAE buffer to make it 250ml add pour in the tray.
PROTOCOL - PLASMID ISOLATION FOR SCREENING
Approximation time:
Material required:
Protocol:
1. Centrifuge cells at 1000 rpm for 6 minutes.
2. Discard suspension and resuspend pellet in 200 µl of P1 and vortex it.
3. Add 200 µl of P2 and invert mix it 4 to 6 minutes.
4. Add 200 µl of P3 and invert mix it .Keep on ice for 10 minutes.
5. Centrifuge at 12000 rpm for 25 minutes at 4 °C.
6. Transfer supernatant into a fresh tube and add equal volume of chloroform. Invert mix it. Centrifuge at 1000 rpm for 10 minutes at 4 °C.
7. Aspirate supernatant and add ethanol to it.Mix and keep at -20°C for 1 hour.
8. Centrifuge at 1200 rpm for 15 minute at 4°C.
9. Resuspend pellet in 70% alcohol and centrifuge at 1200 rpm for 10 min at 4°C.
10. Discard supernatant and allow the pellet to dry. Add milli Q water.
PROTOCOL- CULTURING, STREAKING AND PLATING
Approxmiation time:
Material required:
Mixed culture of bacteria
Sterile petri dish with appropriate bacterial media
Inoculating loop
Bunsen burner
Protocol
1. Label the petri dishes including organism ,type of agar ,date and name on the bottom
2. Sterilise the transfer loop by flaming it in the bunsen burner and let the loop cool down.
3. Flame the neck of the test tube.
4. Insert the loop into the culture broth and withdraw. At all times hold the loop as still as possible.
5. Flame the neck of the test tube again.
6. Replace the cap/ cotton wool plug of the test tube using the little finger of your right hand. Place the test tube in a rack. For a liquid culture, dip the loop into the broth, or for solid media, lightly touch a colony with the loop.
7. Partially lift the lid of the Petri dish containing the solid medium.
8. Place a loopful of the culture on the agar surface on a area and then drag it rapidly several times across the surface of area.
9. Turn the dish 90°C anticlockwise and streak the area hitting last area several times. Repeat the step three more time.
10. Remove the loop and close the Petri dish.
11. Tape the plate closed and incubates the plate in an inverted position in an incubator for 24-48 hours.
12. Flame the loop before putting it aside.
PROTOCOL - POLYMERASE CHAIN REACTION
Approximation time:
2-4hrs
Material required:
10x Amplification buffer
dNTP solution containing all four dNTPs
Thermostable DNA polymerase
Nucleic Acids and Oligonucleotides
Template DNA
PCR tubes
Primers(Foward & Reverse)
Protocol:
1. In the amplification tube,add and mix in the order
10x PCR buffer
5µl
20mM solution of four dNTPs
1µl
10 pmol/µl forward primer
3µl
10 pmol/µl reverse primer
3µl
Thermostable DNA polymerase mix
0.5 - 1 µl
Template DNA
6µl
water
31µl
2. If the thermal cycler is not fitted with a heated lid, overlay the reaction mixtures with 1 drop (approx. 50 μl) of light mineral oil. Alternatively, place a bead of wax into the tube if using a hot start protocol. Place the tubes or the micro titer plate in the thermal cycler.
3. Amplify the nucleic acids using the denaturation, annealing, and polymerization times and temperatures listed below.
Cycle Number
Denaturation
Annealing
Polymerisation
35
30 sec at 94 °C
1 min at 58 °C
2 mins at 68°C
4. Withdraw a sample (5-10 μl) from the test reaction mixture and the four control reactions, analyze them by electrophoresis through an agarose gel, and stain the gel with ethidium bromide or SYBR Gold to visualize the DNA. A successful amplification reaction should yield a readily visible DNA fragment of the expected size.
5. If mineral oil was used to overlay the reaction (Step 2), remove the oil from the sample by extraction with 150 μl of chloroform. The aqueous phase, which contains the amplified DNA, will form a micelle near the meniscus. The micelle can be transferred to a fresh tube with an automatic micropipette.
Indian Insitiute
of Technology
Kharagpur
Department of
Biotechnology
Goverment of India
Connect
