Team:Gifu/result-page
PROJECT
RESULT
EFFICIENCY
RNase processing
   The result of "RNase processing" was shown below.
Fig.1  As a result of qualitative experiment of Circular mRNA
1:normal(iGEM Gifu 2014) 2:outside complementarity 3:inside complementarityⅠ 4:inside complementarityⅡ
a…To detect the sequence of the circular mRNA in the cDNA derived from the RNA after RNaseR processing.
b…To detect the sequence of the linear mRNA in the cDNA derived from the RNA after RNaseR processing.
c…To detect the sequence of the circular mRNA in the cDNA derived from the non-treated RNA.
d…To detect the sequence of the linear mRNA in the cDNA derived from the non-treated RNA.
e…To detect the sequence of the circular mRNA in the non-treated RNA
f…To detect the sequence of the linear mRNA in the non-treated RNA
M…marker
See the line e,f → RNA is not detected by the electrophoresis, namely, DNA does not contaminate.
See the line b,d → Linear mRNA is cleaved by exoribonuclease.
See the line a,c → mRNA is not cleaved by exoribonuclease.
  From the above, the circular mRNA exists in all samples.
   According to the line e and f, RNA was not detected by the electrophoresis, namely, DNA did not contaminate.
   According to the line b and d, linear mRNA was cleaved by exoribonuclease.
   According to the line a and c, mRNA was not cleaved by exoribonuclease.
   From the above, the circular mRNA existed in all samples.
Semi-quantitative PCR
   The result of Semi-quantitative PCR was shown below. And, we made the graphs by using the raw data.
Fig.2 As a result of quantitative experiment of Circular mRNA
Table.1 The mean of the fluorescence (tripartite)
Fig.3 Relationship of fluorescence and cycle number
From the left, normal, outside, insideⅠ and insideⅡ
Table.2 Ct value at fluorescence 1.90
  According to this result, we calculated Ct value which indicates 1.90 fluorescence intensity. These value is summarized right. Ct is a value that the more the gene template in PCR increases, the more Ct value decreases. If there is a difference between “C” and “D”, there is a difference in the quantity of template geneThus, the difference between “C” and “D” is small; the cyclization may be efficiency.As a result, Ct in " normal [BBa_1332011] " was 2.31, but “ outside [BBa_1859026] ”, “ inside Ⅰ [BBa_1859024] ”, “ inside Ⅱ [BBa_1859025] ” were 1.28, 2.02, 1.89 collectively. From these things, it can be said that the efficiency of cyclization rose with all devices which we designed in this time. Especially, in the case of “outside”, there was a difference with Ct more than 1 point than “normal”. When we think that a quantity of the gene doubles for 1 cycle in PCR simply, it can be said that the cyclic efficiency of “outside” was twice as high as that of “normal”.
  After all, it is thought that it fitted this splice site because it is a complementarity chain derived from the creature.
Table 3. Efficiency of circularization (relativity value)
normal
outside
insideⅠ
insideⅡ
Efficiency of circularization (relativity value)
1.00
2.05
1.22
1.34
FUNCTION
  We made 7 kinds of linker in this experiment.
  We made parts which have these sequence of linker in the downstream of the 3’ side of the intron [BBa_K1332005] or the upstream of the 5’ side of the intron without stop codon [BBa_K1332003] .
  We constructed plasmids like following it.
  In case of inserting these plasmid into E. coli, the following circular mRNA is expressed and the long chain protein is synthesized.
  We inserted these plasmid into an E. coli and made it synthesize proteins and did SDS-PAGE using this proteins. If the protein is not boiled, we can do SDS-PAGE keeping it fluorescence because RFP’s structure is strong. Therefore we applied samples that were not boiled.
Fig.4 Fluorescence of protein before dying by CBB                                                          Fig.5 after dying by CBB
Fig.6 Overlapping Fig.4 and Fig.5
  There was not fluorescence at the place of long chain protein. We found that long chain protein didn’t have a function.
Table 4. The generators used to research the qualitative of a protein
No.
generator
A
R0010 + B0034 + K1332001 + B0015
B
BBa_K1332011
C
BBa_K1859027
D
BBa_K1859028
E
BBa_K1859020
F
BBa_K1859021
G
BBa_K1859022
H
BBa_K1859023
I
BBa_K1859023
J
BBa_K1859025
K
BBa_K1859024
Reason of why [BBa_K1859020] and [BBa_K1859022] don’t synthesize the long chain protein
  In this experiment, [BBa_K1859020] and [BBa_K1859022] didn’t generate the long chain protein. The reason of this phenomenon can be deduced in terms of the sequence of the circular mRNA.
  We made prediction of synthesized mRNA’s secondary structure.
Fig.7  secondary structure of mRNA in [BBa_K1332011] Fig.8 secondary structure of mRNA in [BBa_K1859021]
Fig.9  secondary structure of mRNA in [BBa_K1859020]
Fig.10  secondary structure of mRNA in [BBa_K1859027] Fig.11  secondary structure of mRNA in [BBa_K1859023]
Fig.12  secondary structure of mRNA in [BBa_K1859028] Fig.13  secondary structure of mRNA in [BBa_K1859025]
Fig.14  secondary structure of mRNA in [BBa_K1859022]
Fig.15  secondary structure of mRNA in [BBa_K1859029] Fig.16  secondary structure of mRNA in [BBa_K1859024]
We can see the mRNA which is not to synthesize a long chain protein (BBa_K1859020 and BBa_K1859022) forms hydrogen bonds in downstream of the ribosome binding site. It is assumed that these binds prevent mRNA from starting to be translated.
   MENU
DESCRIPTION  
Self-Splicing  
PIE Method  
Last Year  
EXPERIMENTS  
RESULT  
MODELING  
FUTURE WORKS  
CONCLUSIONS  
REFERENCES  
