<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CIEI-China_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CIEI-China/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="backtop-wrapper"><A href="#" class="back-link"><DIV class="back-txt">Backtop</DIV></A></DIV><HEADER class="header-navigation" id="header"><DIV class="navi_body"><UL class="navi_ul"><LI><A href="https://2017.igem.org/Team:CIEI-China">Home</A></LI><LI><A href="#">Team</A></LI><LI><A href="https://2017.igem.org/Team:CIEI-China/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:CIEI-China/Attributions">Attributions</A></LI></UL><LI><A href="#">Project</A></LI><LI><A href="https://2017.igem.org/Team:CIEI-China/Background">Background</A></LI><LI><A href="https://2017.igem.org/Team:CIEI-China/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:CIEI-China/Design">Our Design</A></LI><LI><A href="https://2017.igem.org/Team:CIEI-China/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:CIEI-China/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:CIEI-China/Future">Future Direction</A></LI><LI><A href="https://2017.igem.org/Team:CIEI-China/Achievements">Achievement</A></LI><LI><A href="#">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:CIEI-China/Notebook">Lab_Book</A></LI><LI><A href="https://2017.igem.org/Team:CIEI-China/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:CIEI-China/Safety">Safety</A></LI><LI><A href="#" style="font-size: 14px;width: 120px;">Human Practice</A></LI><LI><A href="https://2017.igem.org/Team:CIEI-China/HP/Our Story">Our Story</A></LI><LI><A href="https://2017.igem.org/Team:CIEI-China/HP/Silver">Silver</A></LI><LI><A href="https://2017.igem.org/Team:CIEI-China/HP/Gold_Integrated">Gold</A></LI></DIV></HEADER><DIV id="my-container"><DIV id="my-sidebar"><DIV class="sidebar__inner"><DIV class="site-title">Results</DIV><UL class="page-anchors"><LI><A href="#a1"><I>SpTPS1</I></A></LI><LI><A href="#a2">Optimization</A></LI><LI><A href="#a3">Amplification of <I>ScTPS1</I></A></LI><LI><A href="#a4">Construction of the device</A></LI><LI><A href="#a5">SDS-Page</A></LI><LI><A href="#a6">Salinity of the food waste</A></LI><LI><A href="#a7">The survivability results</A></LI></UL><LI><A href="#a8"><I>SpTPS1</I></A></LI><LI><A href="#a9">Optimization of <I>SpTPS1</I></A></LI><LI><A href="#a10">Amplification</A></LI><LI><A href="#a11">Construction of the device</A></LI><LI><A href="#a12">Sequence</A></LI><LI><A href="#a13">SDS-Page</A></LI><LI><A href="#a14"><I> gltB</I></A></LI><LI><A href="#a15">Optimization</A></LI><LI><A href="#a16">Amplification of <I>gltB</I></A></LI><LI><A href="#a17">Construction of the device on PIC9K</A></LI><LI><A href="#a18">Sequence</A></LI><LI><A href="#a19">SDS-Page</A></LI></DIV></DIV><DIV id="my-adjust-content"><DIV class="page-title">Results</DIV><DIV class="first-level" id="a1"><I>SpTPS1</I></DIV><P class="my-content"><I>Saccharomyces cerevisiae trehalose-6-phosphate synthase</I> is commonly used by the cells to enhance their tolerance in hypertonic environments. <I>SpTPS1</I> is the essential gene in synthesis of trehalose. As a small molecular substance, <I>SpTPS1</I> can increase the cell’s ability of regulating the osmotic pressure. Changes in expression of <I>SpTPS1</I> can significantly affect the amount of trehalose production in the cell. This gene is originated from the yeast, which  have a significant effect on production of trehalose in the yeast.</P><DIV class="second-level" id="a2">Optimization</DIV><DIV class="third-level">Sequence before Optimization</DIV><P class="my-content">Saccharomyces cerevisiae S288c chromosome II</P><P class="my-content">ATGACTACGGATAACGCTAAGGCGCAACTGACCTCGTCTTCAGGGGGTAACATTATTGTGGTGTCCAACAGGCTTCCCGTGACAATCACTAAAAACAGCAGTACGGGACAGTACGAGTACGCAATGTCGTCCGGAGGGCTGGTCACGGCGTTGGAAGGGTTGAAGAAGACGTACACTTTCAAGTGGTTCGGATGGCCTGGGCTAGAGATTCCTGACGATGAGAAGGATCAGGTGAGGAAGGACTTGCTGGAAAAGTTTAATGCCGTACCCATCTTCCTGAGCGATGAAATCGCAGACTTACACTACAACGGGTTCAGTAATTCTATTCTATGGCCGTTATTCCATTACCATCCTGGTGAGATCAATTTCGACGAGAATGCGTGGTTGGCATACAACGAGGCAAACCAGACGTTCACCAACGAGATTGCTAAGACTATGAACCATAACGATTTAATCTGGGTGCATGATTACCATTTGATGTTGGTTCCGGAAATGTTGAGAGTCAAGATTCACGAGAAGCAACTGCAAAACGTTAAGGTCGGGTGGTTCTGCACACACCATTCCCTTCGAGTGAAATTTACAGAATCTTACCTGTCAGACAAGAGATTTTGAAGGGTGTTTTGAGTTGTGATTTAGTCGGGTTCCACACATACGATTATGCAAGACATTTCTTGTCTTCCGTGCAAAGAGTGCTTAACGTGAACACATTGCCTAATGGGGTGGAATACCAGGGCAGATTCGTTAACGTAGGGGCCTTCCTATCGGTATCGACGTGGACAAGTTCACCGATGGGTTGAAAAAGGAATCCGTACAAAAGA GAATCCAACAATTGAAGGAAACTTTCAAGGGCTGCAAGATCATAGTTGGTGTCGACAGGCTGGATTACATCAAAGGTGTGCCTCAGAAGTTGCACGCCATGGAAGTGTTTCTGAACGAGCATCCAGAATGGAGGGGCAAGGTTGTTCTGGTACAGGTTGCAGTGCCAAGTCGTGGAGATGTGGAAGAGTACCAATATTTAAGATCTGTGGTCAATGAGTTGGTCGGTAGAATCAACGGTCAGTTCGGTACTGTGGAATTCGTCCCCATCCATTTCATGCACAAGTCTATACCATTTGAAGAGCTGATTTCGTTATATGCTGTGAGCGATGTTTGTTTGGTCTCGTCCACCGTGATGGTATGAACTTGGTTCCTACGAATATATTGCTTGCCAAGAAGAAAAGAAAGGTTCCTTAATCCTGAGTGAGTTCACAGGTGCCGCACAATCCTTGAATGGTGCTATTATTGTAAATCCTTGGAACACCGATGATCTTTCTGATGCCATCAACGAGGCCTTGACTTTGCCCGATGTAAAGAAAGAAGTTAACTGGGAAAAACTTTACAAATACTCTCTAAATA CACTTCTGCCTTCTGGGGTGAAAATTTCGTCCATGAATTATACAGTACATCATCAAGCTCAACAAGCTCCTCTGCCACCAAAAACTGA</P><DIV class="third-level">Optimization</DIV><P class="my-content">1. Avoid the  restriction enzyme cutting sites <I>Eco</I>RⅠ, <I>Xba</I>Ⅰ,<I> Spe</I>Ⅰ, <I>Pst</I>Ⅰ to make sure the following operations on the standard carrier, pSB1C3, can work effectively.</P><P class="my-content">2. Avoid other common restriction enzyme cutting sites: <I>Bam</I>HⅠ, <I>Hin</I>dⅢ, <I> Kpn </I>Ⅰ, <I> Noco </I>Ⅰ, <I> Nde </I>Ⅰ, <I> Not </I>Ⅰ, <I> Sal </I>Ⅰ, <I> Xho </I>Ⅰ.</P><P class="my-content">3. Add the restriction enzyme cutting sites, <I> Xba </I>Ⅰ and <I> Spe </I>Ⅰ at the beginning and the end of  the gene to ligate it with the standard carrier conveniently.</P><P class="my-content">4. We are aimed to enhance the tolerance of <I> saccharomyces cerevisiae </I>’s in hypertonic environment, so we optimized the sequence to get the highest efficiency by using the <I> saccharomyces cerevisiae </I> favored codons. In this way, we made sure that the basic sequence will express the same amino acid sequence as in the previous gene before optimization.</P><P class="my-content">5. The <I>SpTPS1</I> gene is successfully optimized.</P><P class="my-content"><I>SpTPS1</I></P><P class="my-content fig-txt">Figure 1-1: the relative position of codons of the optimization</P><P class="my-content">The distribution of codon usage frequency along the length of the <I>SpTPS1</I> gene. The perfect CAI in our expression organism is 1.0, and a CAI of &gt; 0.8 is considered to be acceptable, in terms of high gene expression level.</P><DIV class="third-level">Sequence after Optimizaiton</DIV><P class="my-content">Optimized Sequence</P><P class="my-content">ATGACTACAGATAATGCTAAAGCACAATTGACATCTTCATCTGGTGGTAACATCATCGTTGTTTCTAACAGATTACCAGTTACTATTACTAAAAATTCTTCTACTGGTCAATATGAATACGCTATGTCATCTGGTGGTTTGGTTACTGCATTGGAAGGTTTGAAGAAAACTTACACTTTTAAATGGTTTGGTTGGCCAGGTTTAGAAATCCCAGATGATGAAAAGGATCAAGTTAGAAAAGATTTGTTAGAAAAGTTTAATGCTGTTCCAATTTTCTTGTCTGATGAAATCGCAGATTTGCATTACAACGGTTTTTCTAATTCAATCTTGTGGCCATTGTTCCATTACCATCCAGGTGAAATTAATTTCGATGAAAACGCTTGGTTGGCATACAACGAAGCTAACCAAACTTTTACAAATGAAATTGCAAAAACAATGAACCATAACGATTTGATTTGGGTTCATGATTACCATTTGATGTTAGTTCCAGAAATGTTGAGAGTTAAGATCCATGAAAAACAATTGCAAAACGTTAAGGTTGGTTGGTTCTTGCATACTCCATTCCCATCATCTGAAATATATAGAATTTTGCCAGTTAGACAAGAAATTTTGAAGGGTGTTTTGTCTTGTGATTTGGTTGGTTTCCATACTTATGATTACGCTAGACATTTCTTGTCTTCTGTTCAAAGAGTTTTGAACGTTAACACTTTGCCAAATGGTGTTGAATACCAAGGTAGATTTGTTAATGTTGGTGCTTTCCCAATCGGTATCGATGTTGATAAGTTTACTGATGGTTTGAAGAAAGAATCTGTTCAAAAGAGAATACAACAATTGAAGGAAACTTTTAAAGGTTGTAAGATCATCGTTGGTGTTGATAGATTGGATTACATCAAGGGTGTTCCACAAAAATTGCATGCTATGGAAGTTTTCTTGAATGAACATCCAGAATGGCGTGGTAAAGTTGTTTTAGTTCAAGTTGCTGTTCCATCAAGAGGTGACGTTGAAGAATACCAATACTTAAGATCAGTTGTTAACGAATTGGTTGGTAGAATTAATGGTCAATTCGGTACAGTTGAATTTGTTCCAATCCATTTCATGCATAAGTCTATTCCATTCGAAGAATTGATTTCTTTATACGCTGTTTCAGATGTTTGTTTGGTTTCATCTACAAGAGATGGTATGAATTTGGTTTCTTATGAATACATCGCATGTCAAGAAGAAAAGAAAGGTTCTTTGATCTTGTCAGAGTTTACTGGTGCTGCACAATCTTTGAACGGTGCTATCATCGTTAACCCTTGGAACACAGATGATTTGTCAGATGCTATTAATGAAGCATTGACTTTGCCAGATGTTAAGAAAGAAGTTAACTGGGAAAAATTGTACAAGTACATCTCTAAGTACACTTCAGCTTTCTGGGGTGAAAACTTCGTTCATGAATTGTACTCTACTTCATCTTCATCTACATCATCTTCAGCAACTAAAAATTAA</P><DIV class="second-level" id="a3">Amplification of <I>ScTPS1</I></DIV><DIV class="third-level">The device before PCR in carrier PUC57</DIV><P class="my-content">pUC57 is the carrier of the optimized <I>SpTPS1</I> gene synthesized by a bio-company (GeneScript).</P><P class="my-content">pUC57 is the common carrier among the <I>E.coli</I>plasma cloning. It can be used in <I>DH5α</I>. The best condition is 37̊C with sufficient pUC57 contain the ampicillin to make sure the bacteria less likely to be polluted. The Promoter in pUC57 is lac. The plasmid in pUC57 is 2710 bp.</P><P class="my-content">The 5' sequencing primer is M13R: CAGGAAACAGCTATGACC.</P><P class="my-content">The 3'sequencing primer is M13F: TGTAAAACGACGGCCAGT.</P><P class="my-content fig-txt">Figure 1-2 Plasmid pUC57 of <I>E.coli</I></P><DIV class="third-level">Amplification</DIV><P class="my-content">To get the desired segments of our target gene <I>SpTPS1</I>:</P><P class="my-content">Amplify the gene through PCR;</P><P class="my-content">The forward primer is designed to be ATGACTACAGATAATGCTAAAGC;</P><P class="my-content">The reverse primer is designed to be TTAATTTTTAGTTGCTGAAG;</P><P class="my-content">As shown in figure 2-2, we successfully cloned segments of our gene <I>SpTPS1</I>.</P><P class="my-content fig-txt">Figure 1-3 Gel electrophoresis of <I>SpTPS1</I>  cloned from <I>PUC57</I></P><P class="my-content">The left column is the marker, and the two strips on the right are PCR products of <I>SpTPS1</I>. The sizes of the two strips are about 2000 bp.</P><DIV class="second-level" id="a4">Construction of the device</DIV><DIV class="third-level">Construction of the device on the pPIC9K</DIV><P class="my-content">We will ligate the <I>SpTPS1</I> gene into pPIC9K at <I>Sna</I>BⅠ and <I>Not</I>Ⅰ restriction enzyme cutting sites. The promoter, AOX1, will promote the expression of the <I>SpTPS1</I> gene. Then the completed pPIC9K will be transformed into  <I>GS115</I>  yeast.</P><P class="my-content">Cut off the secretion signals in the extracellular device to get the intracellular expression device, SpTPS1-W, in the pPIC9K carrier. Then transform the intracellular expression vector into <I>GS115</I> yeast.</P><DIV class="forth-level">pPIC9K</DIV><DIV class="content-wrapper"><DIV class="content-right"><P class="my-content">Figure 1-4: pPIC9K is a carrier for the protein expression in Pichia Pastoris. The carrier is 9276 bp. The promoter is AOX1. The carrier contains the Kanamycin that can keep the bacteria from pollution. HIS tag is a histidine tag in pPIC9K related to SDS-Page, the protein expression test.</P></DIV></DIV><P class="my-content">The 5’end sequence is the 5´ AOX1: 5´-GACTGGTTCCAATTGACAAGC-3´.</P><P class="my-content">The 3’end sequence is the 3´ AOX1: 5´-GCAAATGGCATTCTGACATCC-3´.</P><DIV class="forth-level">Verification of <I>SpTPS1</I> PCR product</DIV><P class="my-content">1.To test whether <I>SpTPS1</I> had been successfully ligated to the extracellular expression vector <I>pPIC9K</I>, the carrier pPIC9K containing gene <I>SpTPS1</I> for the extracellular expression was amplified and the results proved it had been digested and ligated successfully.</P><DIV class="content-wrapper"><DIV class="content-left"><P class="my-content fig-txt">Figure 1-5 The electrophoresis result of the <I>SpTPS1</I> in the extracellular expression device</P></DIV><DIV class="content-right"><P class="my-content">The electrophoresis result showed that the <I>SpTPS1</I> in the extracellular expression device is ligated since we got a single band slightly larger than 1000 bp.</P></DIV></DIV><P class="my-content">2.To test whether <I>SpTPS1</I>had been successfully ligated to to intracellular expression vector pPIC9K , the carrier pPIC9K containing gene <I>SpTPS1</I> for the intracellular expression was amplified and the results proved that it had been digested and ligated successfully</P><DIV class="content-wrapper"><DIV class="content-left"><P class="my-content fig-txt">Figure 1-6 The electrophoresis result of the <I>SpTPS1</I> in the extracellular expression device</P></DIV><DIV class="content-right"><P class="my-content">The electrophoresis result showed that the <I>SpTPS1</I> in the intracellular expression device is ligated since we got a single band slightly larger than 1000 bp.</P></DIV></DIV><DIV class="third-level">Identify Sequence</DIV><P class="my-content">To verify that the sequence of <I>SpTPS1</I> gene after ligation is as same as it was before ligation, we chose three sample of the bacteria solution for sequencing. After sequencing, we chose the sample with the sequence 100% identical to the sequence before optimization toto carry on the following experiments. The sequencing result is 100% identical, so the sequence of SpTPS1 gene didn’t change in this process.</P><P class="my-content fig-txt">Figure 1-7: the comparison of the sequencing</P><P class="my-content">The figure shows the sequence of the original sequence along with the sequence of the gene after  ligation. The first line shows the original sequence, and the second line shows the sequence of the gene on the vector.</P><DIV class="second-level" id="a5">SDS-Page</DIV><P class="my-content">Because expression of the yeast vector pPIC9Kwhich contains the foreign gene functioned with the whole genome of yeast, to make sure that <I>SpTPS1</I> has been recombined in the yeast genome, we did PCR of the transformed yeast, and the following figure indicated that both the intracellular and extracellular expression vector which contains <I>SpTPS1</I> have been successfully recombined to the yeast genome.</P><P class="my-content fig-txt">Figure 1-8 Gel electrophoresis of ScTPS1-contain yeast genome</P><P class="my-content">The left column is the marker, and 1 stands for the control which is empty vector; 2,3 stands for ScTPS1 in the intracellular expression device 4,5 stands for ScTPS1 in the extracellular expression, The sizes of the two strips are about 2000 bp.</P><P class="my-content">We cultured the device that ligated vector pPIC9K（containing gene <I>SpTPS1</I>）in YPD. We used 1% methanol to induce the protein that will produce, After ultrasonication, we broke the membrane of the yeast and then used SDS-PAGE to test expression of <I>SpTPS1</I>.</P><P class="my-content fig-txt">Fig 1-9. The SDS-PAGE electrophoresis gel of the protein with <I>SpTPS1</I> ligation.</P><P class="my-content">Line 1, protein with <I>SpTPS1</I> without induced; Line 2, protein with SctTPS1 induced for 24 hrs; Line 3, protein with <I>SpTPS1</I> w induced for 48 hrs. white arrows shows the target recombined protein.</P><DIV class="second-level" id="a6">Salinity of the food waste</DIV><TABLE><TBODY><TR><TD>Restaurant Sample</TD><TD>1</TD><TD>2</TD><TD>3</TD><TD>4</TD><TD>5</TD><TD>6</TD><TD>average</TD></TR><TR><TD>Salinity (%)</TD><TD>0.64</TD><TD>0.32</TD><TD>0.51</TD><TD>1.43</TD><TD>0.32</TD><TD>1.68</TD><TD>0.82</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD>Cateen Sample</TD><TD>1</TD><TD>2</TD><TD>3</TD><TD>4</TD><TD>5</TD><TD>average</TD></TR><TR><TD>Salinity (%)</TD><TD>0.58</TD><TD>0.72</TD><TD>0.51</TD><TD>0.72</TD><TD>0.84</TD><TD>0.67</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD>Family Sample</TD><TD>1</TD><TD>2</TD><TD>3</TD><TD>4</TD><TD>5</TD><TD>6</TD><TD>7</TD><TD>8</TD><TD>9</TD><TD>10</TD><TD>average</TD></TR><TR><TD>Salinity (%)</TD><TD>1.66</TD><TD>1.30</TD><TD>1.79</TD><TD>0.13</TD><TD>2.05</TD><TD>2.46</TD><TD>0.72</TD><TD>1.03</TD><TD>0.65</TD><TD>1.51</TD><TD>1.39</TD></TR></TBODY></TABLE><P class="my-content">We collected 5 canteen samples, 8 family samples, and 6 restaurant samples for this experiment.</P><P class="my-content">We analyzed salinity of the samples based on testing of their conductivities and   the principle that the salinity of the samples are positively correlated with  conductivity of the dialysate. At 25℃, with the mass/volume ratio of 1: 5 between the sample and the boiling water, the salt concentration was calculated as the conductivity multiplied by 0.064. Then we could calculate salinity of the samples based on their conductivities.</P><P class="my-content">The range of salinity of our restaurant samples was from 0.32% to 1.68%, with an average salinity of 0.82%. Salinity of the restaurant samples was lower than what we expected because the restaurants which allowed us to collect samples mainly produces Sichuan Cuisine which contains less salt but more oil and pepper to achieve the desired flavor. The range of our canteen samples was from 0.58% to 0.84; with an average of   0.67%. The schools’ canteens tend to use less salt, because they have a special standard for the levels of each nutrition for the students. The range of our family samples was from 0.13% to 2.46%, and the average is 1.39%. We found that the standard derivation of salinity in the family samples was very high in the family samples, because each family have their own preferred flavor.</P><DIV class="second-level" id="a7">The survivability results</DIV><DIV class="content-wrapper"><DIV class="content-right"><P class="my-content">Figure 1-10: The figure shows the survivability  results of the <I>GS115</I> along with the <I>SpTPS1</I> contained <I>GS115</I>. 1 is <I>GS115</I>, 2 is <I>SpTPS1</I> for extracellular expression, and 3 is <I>SpTPS1</I> for intracellular expression.</P></DIV></DIV><P class="my-content">According to the results of the salinity test of the food waste, we decided to use the treatments with a salinity from 0% to 3%.</P><P class="my-content">Comparing the <I>GS115</I> and the <I>GS115</I> containing <I>SpTPS1</I>, the tolerance of the modified yeast to hypertonicity is clearly greater than that of <I>GS115</I>. We still need to verify the growth and reaction of the yeast in higher salt rate environment (lower osmotic pressure environment).</P><P class="my-content">When the salinity of the environment is lower than 2%, the difference of survivability between the original <I>GS115</I> and the <I>GS115</I> containing <I>SpTPS1</I> is not apparent. When the salinity of the environment is higher than 2%, the <I>GS115</I>containing <I>SpTPS1</I> has a remarkably higher survivability. Although most of the salinity testing results of the food waste are lower than 2.5%, this experiment result is still reasonable, because we noticed that the percentage of sugar, oil, and some other substances can also affect the osmotic pressure of the food waste.</P><DIV class="first-level" id="a8"><I>SpTPS1</I></DIV><P class="my-content"><I>Schizosaccharomycespombe trehalose-6-phosphate synthase</I> is with a gene length of 1554 bp. It is proved that SpTPS1 can enhance the tolerance of yeast in hypotonic environment, because it is an essential gene in the synthesis of the trehalose. Trehalose, as a small molecular substance, can increase the cell’s ability of regulating the osmotic pressure. Change in the expression of <I>SpTPS1</I> can significantly affect the amount of trehalose production in the cell.</P><DIV class="second-level" id="a9">Optimization of <I>SpTPS1</I></DIV><DIV class="third-level">Sequence before Optimization</DIV><P class="my-content">Schizosaccharomycespombe 972h- chromosome I</P><P class="my-content">TACCTCATAGCATAATGGGGTTAGTTTTCTCTCTAGCATACATAACCAACTATCCCTCTGCATATCACATTCTACAATACATTTGCAAGCAAAACCTTTGCAAATTCTAAAAGATTCAATCTTAGAATACTTTATCAAGTTTAGACAGTTATTCCTTGTGCTTGTACTTGTCAAGAATCTTTGTTTTGCTGAAAAAAAAAATCCAGAAATCTCAATATGTCGGATGCTCATGATACCATAAAATCACTCACGGGTGATGCTTCTAACTCTCGGCGTTTGATCGTCGTCTCCAATCGTTTACCAATTACAATTAAGCGAAAGGATAATGGCACATATGACTTTAGTATGTCTTCGGGTGGTCTGGTCAGTGCTTTGAGCGGTCTCAAGAAGCTCATGACCTTTCAATGGTTGGGCTGGTGCGGTCAAGAGATTCCTGAGGATGAAAAACCCATGATTATCCAGCGTTTGCAAGATGAGTGTAGCGCTATTCCCGTCTTTTTGGATGATGAGACTGCCGACCGCCATTACAACGGATTTAGTAACAGCATTCTTTGGCCCTTGTTCCACTACCATCCTGGTGAAATTAATTTTGACGAGGAAAATTGGGAGGCCTATCGTGCGGCTAACTACGCTTTTGCCGAGGCCATTGTCAAAAATCTGCAGGATGGTGATTTAATTTGGGTGCAGGATTATCATTTGATGGTTCTTCCTCAAATGTTGCGTGAATTAATCGGTGATAAGTTTAAGGATATCAAAATTGGCTTCTTCTTGCACACTCCTTTCCCAAGTAGCGAAATCTATCGTGTTTTACCCGTTAGAAACGAAATCCTTGAAGGTGTACTCAACTGTGATCTCGTTGGCTTCCATACCTACGACTATGCCCGTCACTTTTTGTCTGCATGCTCTCGTATCCTTAATCTTAGCACACTACCTAACGGTGTGGAATACAATGGTCAAATGGTCAGCGTCGGCACCTTCCCCATCGGTATTGATCCCGAAAAGTTCTCTGATGCTCTGAAGTCTGACGTGGTAAAGGATCGCATTGCAAGCATCGAACGTAGACTACAAGGCGTTAAGGTGATTGTGGGTGTCGATCGTTTGGACTACATTAAGGGTGTTCCCCAAAAATTCCATGCCTTTGAAGTGTTCTTAGAACAATACCCTGAATGGGTTGGAAAGGTCGTGTTGGTTCAAGTTGCCGTTCCTTCTCGTCAAGATGTCGAAGAGTATCAGAATCTTAGAGCCGTTGTCAATGAGCTTGTTGGCCGTATTAACGGTCGTTTTGGTACTGTTGAATATACACCTATTCATTTCTTACATAAAAGTGTTCGCTTTGAAGAGCTGGTTGCTCTGTATAACGTTTCAGACGTTTGTTTAATTACCTCAACTCGTGATGGTATGAATCTTGTTTCATACGAGTACATTTGCACTCAACAAGAGAGACATGGTGCCCTAATTCTTAGTGAATTTGCCGGTGCTGCCCAGTCACTCAATGGTAGTATTGTAATTAATCCATGGAACACGGAGGAATTAGCAAACTCCATTCATGATGCCCTCACTATGCCGGAGAAACAACGTGAGGCTAATGAGAATAAATTATTCCGATATGTTAATAAGTATACCAGTCAATTCTGGGGTCAAAGCTTTGTCGGTGAGTTGCAACGCATTCAACACTACAGCCACCCTCACCCCAGAAGAACGAATCCGATTTTACGAACCAAGTCTGCTCAAGTGCTGTCGATGAATTCTAGCTCGTAAGAGTGCATTAGCAACACTATTGGTTTGTTTTCCCATTAATCAAGTTGTTGCACTTTTACACTTTTACTTCTATAATTTCATTCTCAAAGCAATTAAATCATTTGGTCATATATATTCGATGAGTTAGCAGTTTACGATACAAAGGATGTTGTTTATACTGGCACTTTTTTTATTGAGATGAATTCACAATACATGATATAATGCAACGTTCTTTAATGATTTTTTTATATGCAGAAATTCCAATCAGCTTATGGTGGTTTGAGTATTTGCTGATGATGGAAATTTGTTAGACAAGTATTCGACAATTTTTTTATGAAGTGTTGTTCTTTGTAATTTCCTTTGGTTTTATATTGAACATGATGGCAAAAAGAATAAATGTTTTGTTATTTAACCTTAAAAATCCTGTAACATTCAATTAATATTATTTTTATACATTTTTTTTGGGATACA</P><DIV class="third-level">Optimization</DIV><P class="my-content">1. Avoid the following restriction enzyme cutting sites <I>Eco</I>RⅠ, <I> Xba </I>Ⅰ, <I> Spe </I>Ⅰ, <I> Pst </I>Ⅰ to make sure the following operations on the standard carrier, pSB1C3, can work effectively.</P><P class="my-content">2. Avoid other common restriction enzyme cutting sites: <I>Bam</I>HⅠ, <I>Hin</I>dⅢ, <I> Kpn </I>Ⅰ, <I> Noco </I>Ⅰ, <I> Nde </I>Ⅰ, <I> Not </I>Ⅰ, <I> Sal </I>Ⅰ, <I>Xh o </I>Ⅰ.</P><P class="my-content">3. Add the restriction enzyme cutting sites, <I> Xba </I>Ⅰ and <I> Spe </I>Ⅰ, at the beginning of the gene and the end of the gene to ligate it with the standard carrier conveniently.</P><P class="my-content">4. We were aimed to enhance the saccharomyces cerevisiae’s tolerance in hypertonic environment, so we optimized the sequence to get the highest efficiency by using the <I> saccharomyces cerevisiae </I> favored codons. Make sure that the basic sequence will express the same amino acid sequence as in the previous gene before optimization.</P><P class="my-content">SpTPS1</P><P class="my-content">Firgure 2-1 : Relative position of codons in the optimization</P><P class="my-content">The distribution of codon usage frequency along the length of the SpTPS1 gene. The perfect CAI in our expression organism is 1.0, and a CAI of &gt; 0.8 is also considered to be acceptable, in terms of high gene expression level.</P><DIV class="third-level">Sequence after optimization</DIV><P class="my-content">Optimized Sequence</P><P class="my-content">ATGTCAGATGCACATGATACTATTAAATCTTTGACAGGTGACGCTTCTAATTCAAGAAGATTGATTGTTGTTTCAAACAGATTGCCAATCACTATTAAAAGAAAAGATAATGGTACTTATGATTTTTCTATGTCTTCAGGTGGTTTGGTTTCAGCTTTGTCTGGTTTGAAGAAATTGATGACTTTCCAATGGTTAGGTTGGTGTGGTCAAGAAATCCCAGAAGATGAAAAGCCAATGATCATCCAAAGATTGCAAGATGAATGTTCTGCAATTCCAGTTTTCTTGGATGATGAAACAGCTGATAGACATTACAACGGTTTTTCAAATTCTATCTTGTGGCCATTGTTCCATTACCATCCAGGTGAAATTAATTTCGATGAAGAAAACTGGGAAGCATATAGAGCTGCAAATTACGCTTTCGCAGAAGCTATCGTTAAAAATTTGCAAGATGGTGACTTGATTTGGGTTCAAGATTATCATTTGATGGTTTTGCCACAAATGTTGAGAGAATTGATCGGTGACAAGTTTAAAGATATCAAGATCGGTTTCTTTTTGCATACTCCATTCCCATCTTCAGAAATATATAGAGTTTTGCCAGTTAGAAACGAAATTTTAGAAGGTGTTTTGAACTGTGATTTGGTTGGTTTCCATACTTATGATTACGCAAGACATTTCTTGTCAGCTTGTTCAAGAATTTTGAATTTGTCTACTTTGCCAAACGGTGTTGAATACAACGGTCAAATGGTTTCTGTTGGTACATTCCCAATCGGTATCGATCCAGAAAAATTTTCAGATGCATTGAAGTCTGATGTTGTTAAGGATAGAATCGCTTCTATCGAAAGAAGATTGCAAGGTGTTAAAGTTATTGTTGGTGTTGATAGATTAGATTACATTAAAGGTGTTCCACAAAAGTTCCATGCATTCGAAGTTTTCTTGGAACAATACCCAGAATGGGTTGGTAAAGTTGTTTTAGTTCAAGTTGCAGTTCCATCAAGACAAGATGTTGAAGAATACCAAAATTTGAGAGCTGTTGTTAACGAATTAGTTGGTAGAATTAATGGTAGATTCGGTACTGTTGAATACACACCAATCCATTTCTTGCATAAGTCTGTTAGATTCGAAGAATTAGTTGCTTTGTACAACGTTTCAGATGTTTGTTTGATCACTTCTACAAGAGATGGTATGAATTTGGTTTCTTATGAATACATTTGTACTCAACAAGAAAGACATGGTGCATTGATTTTATCTGAATTTGCTGGTGCTGCACAATCATTAAACGGTTCTATCGTTATTAATCCTTGGAACACTGAAGAATTGGCAAATTCAATCCATGATGCTTTGACAATGCCAGAAAAACAAAGAGAAGCAAACGAAAATAAGTTGTTTAGATACGTTAATAAGTACACATCACAATTTTGGGGTCAATCTTTTGTTGGTGAATTACAAAGAATACAACATTACTCTCATCCACATCCAAGAAGAACTAACCCAATCTTGAGAACAAAGTCAGCTCAAGTTTTGTCTATGAACTCTTCTTCTTAA</P><DIV class="second-level" id="a10">Amplification</DIV><P class="my-content">To get the desired segments of our target gene SpTPS1:</P><P class="my-content">Amplify the gene through PCR;</P><P class="my-content">The forward primer is designed to be ATGTCAGATGCACATGATAC;</P><P class="my-content">The reverse primer is designed to be TTAAGAAGAAGAGTTCATAG;</P><P class="my-content">As shown in the figure 1-2, we successfully cloned segments of our target gene <I>SpTPS1 </I>(1554 bp).</P><P class="my-content fig-txt">Figure 2-2 Gel electrophoresis of <I>SpTPS1 </I> cloned from PUC57</P><P class="my-content">The left column is the marker, and the two strips on the right are PCR products of <I>SpTPS1 </I>. The sizes of the two strips are above 1500 bp.</P><DIV class="second-level" id="a11">Construction of the device</DIV><DIV class="third-level">The device on pPIC9K</DIV><DIV class="third-level">Construction of the device in pPIC9K</DIV><P class="my-content">We ligated the <I>SpTPS</I>1 gene into pPIC9K at <I>SnaB</I>Ⅰ and <I>Not</I>Ⅰ restriction enzyme cutting sites. The promoter is AOX1, which is to promote the expression of the <I>SpTPS</I> gene. Then the completed pPIC9K is transformed into <I>GS115</I> yeast.</P><P class="my-content">Cut off the secretion signals in the extracellular device to get the intracellular expression device, <I>SpTPS</I>, in the pPIC9K carrier. Then transform the intracellular into <I>GS115</I> yeast.</P><P class="my-content">2.3.2 Verification of SpTPS1 PCR product in pPIC9K</P><P class="my-content">1. To test whether <I>SpTPS</I> had been successfully ligated to the extracellular expression vector pPIC9K, <I>SpTPS</I> is amplified and the results proved that it had been digested and ligated successfully in the pPIC9K for extracellular expression.</P><DIV class="content-wrapper"><DIV class="content-left"><P class="my-content fig-txt">Figure 2-3 The electrophoresis result of the <I>SpTPS</I> in the extracellular expression device</P></DIV><DIV class="content-right"><P class="my-content">The electrophoresis result showed that the <I>SpTPS</I> in the extracellular expression device is ligated since we got a single band above 2000 bp.</P></DIV></DIV><P class="my-content">2. We attempted to ligate  <I>SpTPS1</I> to the intracellular expression vector pPIC9K. after three trails, we failed to ligate the intracellular device in pPIC9K. One possible reason is that the condition of ligation is not appropriate.</P><DIV class="second-level" id="a12">Sequence</DIV><DIV class="third-level">Verification of <I>SpTPS1</I> after ligation</DIV><P class="my-content">To verify that the sequence of  <I>SpTPS1</I> gene after ligation in the carrier pPIC9K is still as same as it was before the ligation, we chose three samples of the bacteria solution and did sequencing.</P><P class="my-content">After sequencing, we chose the sample with the sequence 100% identical to the sequence before ligation to carry on the following experiments. The sequencing result is 100% identical, so the sequence of  <I>SpTPS1</I> gene didn’t change in this process.</P><P class="my-content">Sequence Comparison</P><P class="my-content">Figure 2-4. Comparison of the sequence</P><P class="my-content">The sequencing results of the gene after ligation along with the original sequencing. The first line is the original sequence and the second line is the sequence after ligation.</P><DIV class="second-level" id="a13">SDS-Page</DIV><P class="my-content">Because expression of the yeast vector pPIC9K which contains the foreign gene functioned in the yeast genome, to verify that SpTPS1 had been successfully recombined to the yeast genome, we did PRC of the transformed yeast. The following picture showed that the extracellular vector which contain SpTPS had been successfully recombined into the genome of yeast.</P><DIV class="content-wrapper"><DIV class="content-left"><P class="my-content fig-txt">Figure 2-5 Gel electrophoresis of SpTPS1-contain yeast genome</P></DIV><DIV class="content-right"><P class="my-content">The left column is the marker, and 1 stands for the control which is empty vector; 2 stands for ScTPS1 in the intracellular expression device;3 stands for SpTPS1 in the extracellular expression, The sizes of the two strips are above 2000 bp.</P></DIV></DIV><P class="my-content">We culture the device that ligated vector PIC9K and gene <I>SpTPS1</I> with YPD medium. We use 1% methanol to induce the protein that will produce under our gene <I>SpTPS1</I> controlled. After ultrasonication, we break the membrane of the yeast then put the protein into SDS-Page. However, we didn’t got positive result. The possible explanations are that the expression rate of the gene is too low or the condition of the secretion is not suitable.</P><DIV class="first-level" id="a14"><I> gltB</I></DIV><P class="my-content"><I>E.coli</I> glutamate synthase (<I>gltB</I>) gene has a gene length of 924 bp. <I>gltB</I> is found in <I>E.coli</I> and used to enhance the its tolerance to hypertonic environment. <I>gltB</I> is the key enzyme in the synthesis pathway of glutamic acid. Glutamic acid is also a small molecular substance similar to trahalose, and it can also help the cell to regulate osmotic pressure. Therefore, overexpression of <I>gltB</I> can increase <I>E.coli</I>’s ability to regulate the osmotic pressure. We transformed this gene into the yeast and expected that it will have the same regulating effect as in the yeast.</P><DIV class="second-level" id="a15">Optimization</DIV><DIV class="third-level">Sequence before optimization</DIV><P class="my-content">AACACACCTTATGACAGTCAGGAATTGACTGTTTCTCTAACGACTTCCCTTTTAGCCTTAAAGATAAAATCCATTTTAATTTCAGTCATTTAATAAAGAATTTTGCGCTAAAGCACATTTCTGTACCAATAAGCTTGCCATTTGACCTGTATCAGCTTTCCCGATAAGTTGGAAATCCGCTGGAAGCTTTCTGGATGAGCAGCCTGCTCATCATATTTATGCAGTAATTGAGATCCCCTCTTCACCGTATTAACCGATGCGAAAAGGACAACAAGGGGGCGAATCGGAGGCGCGCGTATGACACGCAAACCCCGTCGCCACGCTCTTTCTGTGCCCGTGCGCAGCGGTTCGGAAGTGGGGTTCCCGCAGAGCCTGGGGGAGGTTCACGATATGTTGTACGATAAATCCCTTGAGAGGGATAACTGTGGTTTCGGCCTGATCGCCCACATAGAAGGCGAACCTAGCCACAAGGTAGTGCGTACTGCAATACACGCACTGGCCCGCATGCAGCACCGTGGCGCGATTCTCGCCGATGGTAAAACCGGCGACGGTTGCGGCTTGCTGTTACAAAAACCGGATCGCTTTTTTCGCATCGTTGCGCAGGAGCGCGGCTGGCGTTTAGCAAAAAACTACGCTGTCGGGATGCTCTTCCTGAATAAAGATCCTGAACTCGCCGCTGCCGCACGCCGCATCGTTGAAGAAGAACTGCAACGCGAAACCTTGTCGATTGTGGGCTGGCGTGATGTCCCCACTAACGAAGGCGTGCTGGGTGAAATCGCCCTCTCCTCTCTGCCACGCATTGAGCAAATTTTTGTGAACGCCCCGGCAGGCTGGCGTCCACGCGATATGGAGCGCCGTCTGTTTATCGCCCGCCGCCGCATTGAAAAGCGTCTCGAAGCCGACAAAGACTTCTACGTCTGTAGCCTGTCGAATCTGGTGAACATCTATAAAGGTCTGTGTATGCCGACGGATCTGCCGCGCTTTTATCTGGATCTTGCGGACCTGCGTCTGGAATCGGCCATTTGCCTGTTCCACCAGCGCTTCTCCACTAACACCGTACCGCGCTGGCCGTGGCGCAACCGTTCCGCTATCTGGCGCATAACGGTGAAATCAACACCATCACCGGTAACCGCCAATGGGCGTGCGCGTACTTATAAATTCCAGACACCGCTTATCCCTGACCTGCACGACGCCGCACCGTTCGTC</P><DIV class="third-level">Optimization</DIV><P class="my-content">The requirement of the optimization is as same as the first two genes in order to get better expression of them in the yeast.</P><P class="my-content">     gltB</P><P class="my-content">Figure3-1: The relative position of codons after optimization</P><P class="my-content">The distribution of codons usage frequency along the length of the gene sequence. A CAI of 1.0 is considered to be perfect in the desired expression organism, and a CAI of &gt; 0.8 is regarded as good, in terms of high gene expression level.</P><DIV class="third-level">Sequence after optimization</DIV><P class="my-content">ATGACTAGAAAACCAAGAAGACATGCTTTATCTGTTCCAGTTAGATCAGGTTCAGAAGTTGGTTTTCCACAATCTTTGGGTGAAGTTCATGATATGTTGTACGATAAATCATTGGAAAGAGATAACTGTGGTTTCGGTTTGATCGCTCATATTGAAGGTGAACCATCACATAAAGTTGTTAGAACTGCAATTCATGCTTTAGCAAGAATGCAACATAGAGGTGCTATTTTGGCAGATGGTAAAACAGGTGACGGTTGTGGTTTGTTGTTGCAAAAGCCAGATAGATTTTTCAGAATCGTTGCTCAAGAAAGAGGTTGGAGATTAGCTAAAAATTACGCAGTTGGCATGTTATTTTTGAACAAAGATCCAGAATTGGCTGCTGCTGCTAGAAGAATCGTTGAAGAAGAATTACAAAGAGAAACTTTGTCTATTGTTGGTTGGAGAGATGTTCCAACAAATGAAGGTGTTTTGGGTGAAATCGCATTATCTTCATTGCCAAGAATCGAACAAATTTTTGTTAACGCTCCAGCAGGTTGGAGACCAAGAGATATGGAAAGAAGATTGTTTATTGCTAGAAGAAGAATCGAAAAGAGATTGGAAGCAGATAAGGATTTCTACGTTTGTTCTTTGTCAAATTTGGTTAACATCTATAAGGGTTTGTGTATGCCAACTGATTTGCCAAGATTCTATTTGGATTTGGCTGATTTGAGATTGGAATCTGCAATTTGTTTGTTCCATCAAAGATTTTCAACTAACACAGTTCCAAGATGGCCTTGGAGAAATAGATCAGCTATTTGGAGAATTACTGTTAAATCTACACCATCACCAGTTACAGCTAATGGTAGAGCAAGAACTTACAAGTTCCAAACACCATTAATCCCAGATTTGCATGATGCTGCACCATTTGTTTAA</P><DIV class="second-level" id="a16">Amplification of <I>gltB</I></DIV><P class="my-content">To get the desired segments of our target gene <I>gltB</I>:</P><P class="my-content">Amplify the gene through PCR;</P><P class="my-content">The forward primer is designed to be ATGACTAGAAAACCAAGAAG;</P><P class="my-content">The reverse primer is designed to be TTAAACAAATGGTGCAGC;</P><P class="my-content">As shown in the figure 1-2, we successfully cloned segments of our gene <I>gltB</I>.</P><P class="my-content fig-txt">Figure 3-2 Gel electrophoresis of <I>gltB</I> cloned from PUC57</P><P class="my-content">The left column is the marker, and the two strips on the right are PCR products of <I>gltB</I>. The sizes of the two strips are about 900 bp.</P><DIV class="second-level" id="a17">Construction of the device on PIC9K</DIV><P class="my-content">We ligated the <I>gltB</I> gene into pPIC9K at <I>SnaB</I>Ⅰ and <I>No</I>tⅠ restriction enzyme cutting sites. The promoter is AOX1, which will promote the expression of the <I>gltB</I> gene. Then the completed pPIC9K will be transformed into <I>GS115</I> yeast.</P><DIV class="third-level">Verification of <I>gltB</I> PCR product</DIV><P class="my-content">1.To test whether <I>gltB</I> had been successfully ligated to the extracellular expression vector pPIC9K, the device containing our gene <I>gltB</I>for the extracellular expression is amplified and the results proved that it had been digested and ligated successfully.</P><DIV class="content-wrapper"><DIV class="content-left"><P class="my-content fig-txt">Figure 3-3 The electrophoresis result of the <I>gltB</I> in the extracellular expression device</P></DIV><DIV class="content-right"><P class="my-content">The electrophoresis result showed that the <I>gltB</I> is successfully ligated in the extracellular expression device since we got a single band above 2000 bp.</P></DIV></DIV><P class="my-content">2. To test whether <I>gltB</I> had been successfully ligated to the intracellular expression vector pPIC9K，the device containing our gene <I>gltB</I> for the intracellular expression is amplified and the results proved that it had been digested and ligated successfully</P><DIV class="content-wrapper"><DIV class="content-left"><P class="my-content fig-txt">Figure 3-4 The electrophoresis result of the <I>gltB</I> in the intracellular expression device</P></DIV><DIV class="content-right"><P class="my-content">The electrophoresis result showed that the <I>gltB</I> was successfully ligated in the intracellular expression device since we got a single band about 750 bp large.</P></DIV></DIV><DIV class="second-level" id="a18">Sequence</DIV><DIV class="third-level">Verification of <I>gltB</I> after ligation</DIV><P class="my-content">To verify that the sequence of <I>gltB</I> gene after the ligation is as same as it was before the ligation, we chose three sample of the bacteria solution to do the sequencing. After the sequencing, we chose the sample with the sequence 100% identical to the sequence before optimization to carry on the following experiments. The sequencing result is 100% identical, so the sequence of <I>gltB</I> gene didn’t change in this process.</P><P class="my-content">Figure 3-5. the comparison of the sequencing</P><P class="my-content">The figure shows the sequence of the original sequence along with the sequence of the gene after the ligation. The first line shows the original sequence, and the second line shows the sequence of the ligated gene.</P><DIV class="second-level" id="a19">SDS-Page</DIV><P class="my-content">Because expression of the yeast vector pPIC9K which contains the foreign gene functioned in the yeast genome, to verify that <I>gltB</I> had been successfully recombined to the yeast genome, we did PRC of the transformed yeast. The following picture showed that the extracellular vector which contain <I>gltB</I> had been successfully recombined into the genome of yeast.</P><P class="my-content fig-txt">Figure 3-6 Gel electrophoresis of gltB-contain yeast genome</P><P class="my-content">The left column is the marker, and 1,2 stands for <I>gltB</I> in the extracellular expression device.</P><P class="my-content fig-txt">Figure 3-7. Gel electrophoresis of gltB-contain yeast genome</P><P class="my-content">The left column is the marker, and 1,2 stands for <I>gltB</I> in the intracellular expression device.</P><P class="my-content">We cultured the device with the ligated vector PIC9K and gene <I>gltB</I> in YPD. We used 1% methanol to induce the protein that will produce. After ultrasonication, we broke the membrane of the yeast and then put the protein into SDS-Page. However, we didn’t get positive results. The possible reason is that the expression rate of this gene is too low or the condition of the secretion is not suitable.</P></DIV></DIV><FOOTER><DIV class="sponsor"><DIV class="sponsor-txt">Sponsors</DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>