Team:CIEI-China/Results
Backtop
Home
Team
Team
Attributions
Project
Background
Parts
Our Design
Results
Model
Future Direction
Achievement
Notebook
Lab_Book
Protocols
Safety
Human Practice
Our Story
Silver
Gold
Results
SpTPS1
Optimization
Amplification of ScTPS1
Construction of the device
SDS-Page
Salinity of the food waste
The survivability results
SpTPS1
Optimization of SpTPS1
Amplification
Construction of the device
Sequence
SDS-Page
gltB
Optimization
Amplification of gltB
Construction of the device on PIC9K
Sequence
SDS-Page
Results
SpTPS1
Saccharomyces cerevisiae trehalose-6-phosphate synthase is commonly used by the cells to enhance their tolerance in hypertonic environments. SpTPS1 is the essential gene in synthesis of trehalose. As a small molecular substance, SpTPS1 can increase the cell’s ability of regulating the osmotic pressure. Changes in expression of SpTPS1 can significantly affect the amount of trehalose production in the cell. This gene is originated from the yeast, which  have a significant effect on production of trehalose in the yeast.
Optimization
Sequence before Optimization
Saccharomyces cerevisiae S288c chromosome II
ATGACTACGGATAACGCTAAGGCGCAACTGACCTCGTCTTCAGGGGGTAACATTATTGTGGTGTCCAACAGGCTTCCCGTGACAATCACTAAAAACAGCAGTACGGGACAGTACGAGTACGCAATGTCGTCCGGAGGGCTGGTCACGGCGTTGGAAGGGTTGAAGAAGACGTACACTTTCAAGTGGTTCGGATGGCCTGGGCTAGAGATTCCTGACGATGAGAAGGATCAGGTGAGGAAGGACTTGCTGGAAAAGTTTAATGCCGTACCCATCTTCCTGAGCGATGAAATCGCAGACTTACACTACAACGGGTTCAGTAATTCTATTCTATGGCCGTTATTCCATTACCATCCTGGTGAGATCAATTTCGACGAGAATGCGTGGTTGGCATACAACGAGGCAAACCAGACGTTCACCAACGAGATTGCTAAGACTATGAACCATAACGATTTAATCTGGGTGCATGATTACCATTTGATGTTGGTTCCGGAAATGTTGAGAGTCAAGATTCACGAGAAGCAACTGCAAAACGTTAAGGTCGGGTGGTTCTGCACACACCATTCCCTTCGAGTGAAATTTACAGAATCTTACCTGTCAGACAAGAGATTTTGAAGGGTGTTTTGAGTTGTGATTTAGTCGGGTTCCACACATACGATTATGCAAGACATTTCTTGTCTTCCGTGCAAAGAGTGCTTAACGTGAACACATTGCCTAATGGGGTGGAATACCAGGGCAGATTCGTTAACGTAGGGGCCTTCCTATCGGTATCGACGTGGACAAGTTCACCGATGGGTTGAAAAAGGAATCCGTACAAAAGA GAATCCAACAATTGAAGGAAACTTTCAAGGGCTGCAAGATCATAGTTGGTGTCGACAGGCTGGATTACATCAAAGGTGTGCCTCAGAAGTTGCACGCCATGGAAGTGTTTCTGAACGAGCATCCAGAATGGAGGGGCAAGGTTGTTCTGGTACAGGTTGCAGTGCCAAGTCGTGGAGATGTGGAAGAGTACCAATATTTAAGATCTGTGGTCAATGAGTTGGTCGGTAGAATCAACGGTCAGTTCGGTACTGTGGAATTCGTCCCCATCCATTTCATGCACAAGTCTATACCATTTGAAGAGCTGATTTCGTTATATGCTGTGAGCGATGTTTGTTTGGTCTCGTCCACCGTGATGGTATGAACTTGGTTCCTACGAATATATTGCTTGCCAAGAAGAAAAGAAAGGTTCCTTAATCCTGAGTGAGTTCACAGGTGCCGCACAATCCTTGAATGGTGCTATTATTGTAAATCCTTGGAACACCGATGATCTTTCTGATGCCATCAACGAGGCCTTGACTTTGCCCGATGTAAAGAAAGAAGTTAACTGGGAAAAACTTTACAAATACTCTCTAAATA CACTTCTGCCTTCTGGGGTGAAAATTTCGTCCATGAATTATACAGTACATCATCAAGCTCAACAAGCTCCTCTGCCACCAAAAACTGA
Optimization
1. Avoid the  restriction enzyme cutting sites EcoRⅠ, XbaⅠ, SpeⅠ, PstⅠ to make sure the following operations on the standard carrier, pSB1C3, can work effectively.
2. Avoid other common restriction enzyme cutting sites: BamHⅠ, HindⅢ, Kpn Ⅰ, Noco Ⅰ, Nde Ⅰ, Not Ⅰ, Sal Ⅰ, Xho Ⅰ.
3. Add the restriction enzyme cutting sites, Xba Ⅰ and Spe Ⅰ at the beginning and the end of  the gene to ligate it with the standard carrier conveniently.
4. We are aimed to enhance the tolerance of saccharomyces cerevisiae ’s in hypertonic environment, so we optimized the sequence to get the highest efficiency by using the saccharomyces cerevisiae favored codons. In this way, we made sure that the basic sequence will express the same amino acid sequence as in the previous gene before optimization.
5. The SpTPS1 gene is successfully optimized.
SpTPS1
Figure 1-1: the relative position of codons of the optimization
The distribution of codon usage frequency along the length of the SpTPS1 gene. The perfect CAI in our expression organism is 1.0, and a CAI of > 0.8 is considered to be acceptable, in terms of high gene expression level.
Sequence after Optimizaiton
Optimized Sequence
ATGACTACAGATAATGCTAAAGCACAATTGACATCTTCATCTGGTGGTAACATCATCGTTGTTTCTAACAGATTACCAGTTACTATTACTAAAAATTCTTCTACTGGTCAATATGAATACGCTATGTCATCTGGTGGTTTGGTTACTGCATTGGAAGGTTTGAAGAAAACTTACACTTTTAAATGGTTTGGTTGGCCAGGTTTAGAAATCCCAGATGATGAAAAGGATCAAGTTAGAAAAGATTTGTTAGAAAAGTTTAATGCTGTTCCAATTTTCTTGTCTGATGAAATCGCAGATTTGCATTACAACGGTTTTTCTAATTCAATCTTGTGGCCATTGTTCCATTACCATCCAGGTGAAATTAATTTCGATGAAAACGCTTGGTTGGCATACAACGAAGCTAACCAAACTTTTACAAATGAAATTGCAAAAACAATGAACCATAACGATTTGATTTGGGTTCATGATTACCATTTGATGTTAGTTCCAGAAATGTTGAGAGTTAAGATCCATGAAAAACAATTGCAAAACGTTAAGGTTGGTTGGTTCTTGCATACTCCATTCCCATCATCTGAAATATATAGAATTTTGCCAGTTAGACAAGAAATTTTGAAGGGTGTTTTGTCTTGTGATTTGGTTGGTTTCCATACTTATGATTACGCTAGACATTTCTTGTCTTCTGTTCAAAGAGTTTTGAACGTTAACACTTTGCCAAATGGTGTTGAATACCAAGGTAGATTTGTTAATGTTGGTGCTTTCCCAATCGGTATCGATGTTGATAAGTTTACTGATGGTTTGAAGAAAGAATCTGTTCAAAAGAGAATACAACAATTGAAGGAAACTTTTAAAGGTTGTAAGATCATCGTTGGTGTTGATAGATTGGATTACATCAAGGGTGTTCCACAAAAATTGCATGCTATGGAAGTTTTCTTGAATGAACATCCAGAATGGCGTGGTAAAGTTGTTTTAGTTCAAGTTGCTGTTCCATCAAGAGGTGACGTTGAAGAATACCAATACTTAAGATCAGTTGTTAACGAATTGGTTGGTAGAATTAATGGTCAATTCGGTACAGTTGAATTTGTTCCAATCCATTTCATGCATAAGTCTATTCCATTCGAAGAATTGATTTCTTTATACGCTGTTTCAGATGTTTGTTTGGTTTCATCTACAAGAGATGGTATGAATTTGGTTTCTTATGAATACATCGCATGTCAAGAAGAAAAGAAAGGTTCTTTGATCTTGTCAGAGTTTACTGGTGCTGCACAATCTTTGAACGGTGCTATCATCGTTAACCCTTGGAACACAGATGATTTGTCAGATGCTATTAATGAAGCATTGACTTTGCCAGATGTTAAGAAAGAAGTTAACTGGGAAAAATTGTACAAGTACATCTCTAAGTACACTTCAGCTTTCTGGGGTGAAAACTTCGTTCATGAATTGTACTCTACTTCATCTTCATCTACATCATCTTCAGCAACTAAAAATTAA
Amplification of ScTPS1
The device before PCR in carrier PUC57
pUC57 is the carrier of the optimized SpTPS1 gene synthesized by a bio-company (GeneScript).
pUC57 is the common carrier among the E.coliplasma cloning. It can be used in DH5α. The best condition is 37̊C with sufficient pUC57 contain the ampicillin to make sure the bacteria less likely to be polluted. The Promoter in pUC57 is lac. The plasmid in pUC57 is 2710 bp.
The 5' sequencing primer is M13R: CAGGAAACAGCTATGACC.
The 3'sequencing primer is M13F: TGTAAAACGACGGCCAGT.
Figure 1-2 Plasmid pUC57 of E.coli
Amplification
To get the desired segments of our target gene SpTPS1:
Amplify the gene through PCR;
The forward primer is designed to be ATGACTACAGATAATGCTAAAGC;
The reverse primer is designed to be TTAATTTTTAGTTGCTGAAG;
As shown in figure 2-2, we successfully cloned segments of our gene SpTPS1.
Figure 1-3 Gel electrophoresis of SpTPS1 cloned from PUC57
The left column is the marker, and the two strips on the right are PCR products of SpTPS1. The sizes of the two strips are about 2000 bp.
Construction of the device
Construction of the device on the pPIC9K
We will ligate the SpTPS1 gene into pPIC9K at SnaBⅠ and NotⅠ restriction enzyme cutting sites. The promoter, AOX1, will promote the expression of the SpTPS1 gene. Then the completed pPIC9K will be transformed into GS115 yeast.
Cut off the secretion signals in the extracellular device to get the intracellular expression device, SpTPS1-W, in the pPIC9K carrier. Then transform the intracellular expression vector into GS115 yeast.
pPIC9K
Figure 1-4: pPIC9K is a carrier for the protein expression in Pichia Pastoris. The carrier is 9276 bp. The promoter is AOX1. The carrier contains the Kanamycin that can keep the bacteria from pollution. HIS tag is a histidine tag in pPIC9K related to SDS-Page, the protein expression test.
The 5’end sequence is the 5´ AOX1: 5´-GACTGGTTCCAATTGACAAGC-3´.
The 3’end sequence is the 3´ AOX1: 5´-GCAAATGGCATTCTGACATCC-3´.
Verification of SpTPS1 PCR product
1.To test whether SpTPS1 had been successfully ligated to the extracellular expression vector pPIC9K, the carrier pPIC9K containing gene SpTPS1 for the extracellular expression was amplified and the results proved it had been digested and ligated successfully.
Figure 1-5 The electrophoresis result of the SpTPS1 in the extracellular expression device
The electrophoresis result showed that the SpTPS1 in the extracellular expression device is ligated since we got a single band slightly larger than 1000 bp.
2.To test whether SpTPS1had been successfully ligated to to intracellular expression vector pPIC9K , the carrier pPIC9K containing gene SpTPS1 for the intracellular expression was amplified and the results proved that it had been digested and ligated successfully
Figure 1-6 The electrophoresis result of the SpTPS1 in the extracellular expression device
The electrophoresis result showed that the SpTPS1 in the intracellular expression device is ligated since we got a single band slightly larger than 1000 bp.
Identify Sequence
To verify that the sequence of SpTPS1 gene after ligation is as same as it was before ligation, we chose three sample of the bacteria solution for sequencing. After sequencing, we chose the sample with the sequence 100% identical to the sequence before optimization toto carry on the following experiments. The sequencing result is 100% identical, so the sequence of SpTPS1 gene didn’t change in this process.
Figure 1-7: the comparison of the sequencing
The figure shows the sequence of the original sequence along with the sequence of the gene after  ligation. The first line shows the original sequence, and the second line shows the sequence of the gene on the vector.
SDS-Page
Because expression of the yeast vector pPIC9Kwhich contains the foreign gene functioned with the whole genome of yeast, to make sure that SpTPS1 has been recombined in the yeast genome, we did PCR of the transformed yeast, and the following figure indicated that both the intracellular and extracellular expression vector which contains SpTPS1 have been successfully recombined to the yeast genome.
Figure 1-8 Gel electrophoresis of ScTPS1-contain yeast genome
The left column is the marker, and 1 stands for the control which is empty vector; 2,3 stands for ScTPS1 in the intracellular expression device 4,5 stands for ScTPS1 in the extracellular expression, The sizes of the two strips are about 2000 bp.
We cultured the device that ligated vector pPIC9K（containing gene SpTPS1）in YPD. We used 1% methanol to induce the protein that will produce, After ultrasonication, we broke the membrane of the yeast and then used SDS-PAGE to test expression of SpTPS1.
Fig 1-9. The SDS-PAGE electrophoresis gel of the protein with SpTPS1 ligation.
Line 1, protein with SpTPS1 without induced; Line 2, protein with SctTPS1 induced for 24 hrs; Line 3, protein with SpTPS1 w induced for 48 hrs. white arrows shows the target recombined protein.
Salinity of the food waste
Restaurant Sample
1
2
3
4
5
6
average
Salinity (%)
0.64
0.32
0.51
1.43
0.32
1.68
0.82
Cateen Sample
1
2
3
4
5
average
Salinity (%)
0.58
0.72
0.51
0.72
0.84
0.67
Family Sample
1
2
3
4
5
6
7
8
9
10
average
Salinity (%)
1.66
1.30
1.79
0.13
2.05
2.46
0.72
1.03
0.65
1.51
1.39
We collected 5 canteen samples, 8 family samples, and 6 restaurant samples for this experiment.
We analyzed salinity of the samples based on testing of their conductivities and   the principle that the salinity of the samples are positively correlated with  conductivity of the dialysate. At 25℃, with the mass/volume ratio of 1: 5 between the sample and the boiling water, the salt concentration was calculated as the conductivity multiplied by 0.064. Then we could calculate salinity of the samples based on their conductivities.
The range of salinity of our restaurant samples was from 0.32% to 1.68%, with an average salinity of 0.82%. Salinity of the restaurant samples was lower than what we expected because the restaurants which allowed us to collect samples mainly produces Sichuan Cuisine which contains less salt but more oil and pepper to achieve the desired flavor. The range of our canteen samples was from 0.58% to 0.84; with an average of   0.67%. The schools’ canteens tend to use less salt, because they have a special standard for the levels of each nutrition for the students. The range of our family samples was from 0.13% to 2.46%, and the average is 1.39%. We found that the standard derivation of salinity in the family samples was very high in the family samples, because each family have their own preferred flavor.
The survivability results
Figure 1-10: The figure shows the survivability  results of the GS115 along with the SpTPS1 contained GS115. 1 is GS115, 2 is SpTPS1 for extracellular expression, and 3 is SpTPS1 for intracellular expression.
According to the results of the salinity test of the food waste, we decided to use the treatments with a salinity from 0% to 3%.
Comparing the GS115 and the GS115 containing SpTPS1, the tolerance of the modified yeast to hypertonicity is clearly greater than that of GS115. We still need to verify the growth and reaction of the yeast in higher salt rate environment (lower osmotic pressure environment).
When the salinity of the environment is lower than 2%, the difference of survivability between the original GS115 and the GS115 containing SpTPS1 is not apparent. When the salinity of the environment is higher than 2%, the GS115containing SpTPS1 has a remarkably higher survivability. Although most of the salinity testing results of the food waste are lower than 2.5%, this experiment result is still reasonable, because we noticed that the percentage of sugar, oil, and some other substances can also affect the osmotic pressure of the food waste.
SpTPS1
Schizosaccharomycespombe trehalose-6-phosphate synthase is with a gene length of 1554 bp. It is proved that SpTPS1 can enhance the tolerance of yeast in hypotonic environment, because it is an essential gene in the synthesis of the trehalose. Trehalose, as a small molecular substance, can increase the cell’s ability of regulating the osmotic pressure. Change in the expression of SpTPS1 can significantly affect the amount of trehalose production in the cell.
Optimization of SpTPS1
Sequence before Optimization
Schizosaccharomycespombe 972h- chromosome I
TACCTCATAGCATAATGGGGTTAGTTTTCTCTCTAGCATACATAACCAACTATCCCTCTGCATATCACATTCTACAATACATTTGCAAGCAAAACCTTTGCAAATTCTAAAAGATTCAATCTTAGAATACTTTATCAAGTTTAGACAGTTATTCCTTGTGCTTGTACTTGTCAAGAATCTTTGTTTTGCTGAAAAAAAAAATCCAGAAATCTCAATATGTCGGATGCTCATGATACCATAAAATCACTCACGGGTGATGCTTCTAACTCTCGGCGTTTGATCGTCGTCTCCAATCGTTTACCAATTACAATTAAGCGAAAGGATAATGGCACATATGACTTTAGTATGTCTTCGGGTGGTCTGGTCAGTGCTTTGAGCGGTCTCAAGAAGCTCATGACCTTTCAATGGTTGGGCTGGTGCGGTCAAGAGATTCCTGAGGATGAAAAACCCATGATTATCCAGCGTTTGCAAGATGAGTGTAGCGCTATTCCCGTCTTTTTGGATGATGAGACTGCCGACCGCCATTACAACGGATTTAGTAACAGCATTCTTTGGCCCTTGTTCCACTACCATCCTGGTGAAATTAATTTTGACGAGGAAAATTGGGAGGCCTATCGTGCGGCTAACTACGCTTTTGCCGAGGCCATTGTCAAAAATCTGCAGGATGGTGATTTAATTTGGGTGCAGGATTATCATTTGATGGTTCTTCCTCAAATGTTGCGTGAATTAATCGGTGATAAGTTTAAGGATATCAAAATTGGCTTCTTCTTGCACACTCCTTTCCCAAGTAGCGAAATCTATCGTGTTTTACCCGTTAGAAACGAAATCCTTGAAGGTGTACTCAACTGTGATCTCGTTGGCTTCCATACCTACGACTATGCCCGTCACTTTTTGTCTGCATGCTCTCGTATCCTTAATCTTAGCACACTACCTAACGGTGTGGAATACAATGGTCAAATGGTCAGCGTCGGCACCTTCCCCATCGGTATTGATCCCGAAAAGTTCTCTGATGCTCTGAAGTCTGACGTGGTAAAGGATCGCATTGCAAGCATCGAACGTAGACTACAAGGCGTTAAGGTGATTGTGGGTGTCGATCGTTTGGACTACATTAAGGGTGTTCCCCAAAAATTCCATGCCTTTGAAGTGTTCTTAGAACAATACCCTGAATGGGTTGGAAAGGTCGTGTTGGTTCAAGTTGCCGTTCCTTCTCGTCAAGATGTCGAAGAGTATCAGAATCTTAGAGCCGTTGTCAATGAGCTTGTTGGCCGTATTAACGGTCGTTTTGGTACTGTTGAATATACACCTATTCATTTCTTACATAAAAGTGTTCGCTTTGAAGAGCTGGTTGCTCTGTATAACGTTTCAGACGTTTGTTTAATTACCTCAACTCGTGATGGTATGAATCTTGTTTCATACGAGTACATTTGCACTCAACAAGAGAGACATGGTGCCCTAATTCTTAGTGAATTTGCCGGTGCTGCCCAGTCACTCAATGGTAGTATTGTAATTAATCCATGGAACACGGAGGAATTAGCAAACTCCATTCATGATGCCCTCACTATGCCGGAGAAACAACGTGAGGCTAATGAGAATAAATTATTCCGATATGTTAATAAGTATACCAGTCAATTCTGGGGTCAAAGCTTTGTCGGTGAGTTGCAACGCATTCAACACTACAGCCACCCTCACCCCAGAAGAACGAATCCGATTTTACGAACCAAGTCTGCTCAAGTGCTGTCGATGAATTCTAGCTCGTAAGAGTGCATTAGCAACACTATTGGTTTGTTTTCCCATTAATCAAGTTGTTGCACTTTTACACTTTTACTTCTATAATTTCATTCTCAAAGCAATTAAATCATTTGGTCATATATATTCGATGAGTTAGCAGTTTACGATACAAAGGATGTTGTTTATACTGGCACTTTTTTTATTGAGATGAATTCACAATACATGATATAATGCAACGTTCTTTAATGATTTTTTTATATGCAGAAATTCCAATCAGCTTATGGTGGTTTGAGTATTTGCTGATGATGGAAATTTGTTAGACAAGTATTCGACAATTTTTTTATGAAGTGTTGTTCTTTGTAATTTCCTTTGGTTTTATATTGAACATGATGGCAAAAAGAATAAATGTTTTGTTATTTAACCTTAAAAATCCTGTAACATTCAATTAATATTATTTTTATACATTTTTTTTGGGATACA
Optimization
1. Avoid the following restriction enzyme cutting sites EcoRⅠ, Xba Ⅰ, Spe Ⅰ, Pst Ⅰ to make sure the following operations on the standard carrier, pSB1C3, can work effectively.
2. Avoid other common restriction enzyme cutting sites: BamHⅠ, HindⅢ, Kpn Ⅰ, Noco Ⅰ, Nde Ⅰ, Not Ⅰ, Sal Ⅰ, Xh o Ⅰ.
3. Add the restriction enzyme cutting sites, Xba Ⅰ and Spe Ⅰ, at the beginning of the gene and the end of the gene to ligate it with the standard carrier conveniently.
4. We were aimed to enhance the saccharomyces cerevisiae’s tolerance in hypertonic environment, so we optimized the sequence to get the highest efficiency by using the saccharomyces cerevisiae favored codons. Make sure that the basic sequence will express the same amino acid sequence as in the previous gene before optimization.
SpTPS1
Firgure 2-1 : Relative position of codons in the optimization
The distribution of codon usage frequency along the length of the SpTPS1 gene. The perfect CAI in our expression organism is 1.0, and a CAI of > 0.8 is also considered to be acceptable, in terms of high gene expression level.
Sequence after optimization
Optimized Sequence
ATGTCAGATGCACATGATACTATTAAATCTTTGACAGGTGACGCTTCTAATTCAAGAAGATTGATTGTTGTTTCAAACAGATTGCCAATCACTATTAAAAGAAAAGATAATGGTACTTATGATTTTTCTATGTCTTCAGGTGGTTTGGTTTCAGCTTTGTCTGGTTTGAAGAAATTGATGACTTTCCAATGGTTAGGTTGGTGTGGTCAAGAAATCCCAGAAGATGAAAAGCCAATGATCATCCAAAGATTGCAAGATGAATGTTCTGCAATTCCAGTTTTCTTGGATGATGAAACAGCTGATAGACATTACAACGGTTTTTCAAATTCTATCTTGTGGCCATTGTTCCATTACCATCCAGGTGAAATTAATTTCGATGAAGAAAACTGGGAAGCATATAGAGCTGCAAATTACGCTTTCGCAGAAGCTATCGTTAAAAATTTGCAAGATGGTGACTTGATTTGGGTTCAAGATTATCATTTGATGGTTTTGCCACAAATGTTGAGAGAATTGATCGGTGACAAGTTTAAAGATATCAAGATCGGTTTCTTTTTGCATACTCCATTCCCATCTTCAGAAATATATAGAGTTTTGCCAGTTAGAAACGAAATTTTAGAAGGTGTTTTGAACTGTGATTTGGTTGGTTTCCATACTTATGATTACGCAAGACATTTCTTGTCAGCTTGTTCAAGAATTTTGAATTTGTCTACTTTGCCAAACGGTGTTGAATACAACGGTCAAATGGTTTCTGTTGGTACATTCCCAATCGGTATCGATCCAGAAAAATTTTCAGATGCATTGAAGTCTGATGTTGTTAAGGATAGAATCGCTTCTATCGAAAGAAGATTGCAAGGTGTTAAAGTTATTGTTGGTGTTGATAGATTAGATTACATTAAAGGTGTTCCACAAAAGTTCCATGCATTCGAAGTTTTCTTGGAACAATACCCAGAATGGGTTGGTAAAGTTGTTTTAGTTCAAGTTGCAGTTCCATCAAGACAAGATGTTGAAGAATACCAAAATTTGAGAGCTGTTGTTAACGAATTAGTTGGTAGAATTAATGGTAGATTCGGTACTGTTGAATACACACCAATCCATTTCTTGCATAAGTCTGTTAGATTCGAAGAATTAGTTGCTTTGTACAACGTTTCAGATGTTTGTTTGATCACTTCTACAAGAGATGGTATGAATTTGGTTTCTTATGAATACATTTGTACTCAACAAGAAAGACATGGTGCATTGATTTTATCTGAATTTGCTGGTGCTGCACAATCATTAAACGGTTCTATCGTTATTAATCCTTGGAACACTGAAGAATTGGCAAATTCAATCCATGATGCTTTGACAATGCCAGAAAAACAAAGAGAAGCAAACGAAAATAAGTTGTTTAGATACGTTAATAAGTACACATCACAATTTTGGGGTCAATCTTTTGTTGGTGAATTACAAAGAATACAACATTACTCTCATCCACATCCAAGAAGAACTAACCCAATCTTGAGAACAAAGTCAGCTCAAGTTTTGTCTATGAACTCTTCTTCTTAA
Amplification
To get the desired segments of our target gene SpTPS1:
Amplify the gene through PCR;
The forward primer is designed to be ATGTCAGATGCACATGATAC;
The reverse primer is designed to be TTAAGAAGAAGAGTTCATAG;
As shown in the figure 1-2, we successfully cloned segments of our target gene SpTPS1 (1554 bp).
Figure 2-2 Gel electrophoresis of SpTPS1 cloned from PUC57
The left column is the marker, and the two strips on the right are PCR products of SpTPS1 . The sizes of the two strips are above 1500 bp.
Construction of the device
The device on pPIC9K
Construction of the device in pPIC9K
We ligated the SpTPS1 gene into pPIC9K at SnaBⅠ and NotⅠ restriction enzyme cutting sites. The promoter is AOX1, which is to promote the expression of the SpTPS gene. Then the completed pPIC9K is transformed into GS115 yeast.
Cut off the secretion signals in the extracellular device to get the intracellular expression device, SpTPS, in the pPIC9K carrier. Then transform the intracellular into GS115 yeast.
2.3.2 Verification of SpTPS1 PCR product in pPIC9K
1. To test whether SpTPS had been successfully ligated to the extracellular expression vector pPIC9K, SpTPS is amplified and the results proved that it had been digested and ligated successfully in the pPIC9K for extracellular expression.
Figure 2-3 The electrophoresis result of the SpTPS in the extracellular expression device
The electrophoresis result showed that the SpTPS in the extracellular expression device is ligated since we got a single band above 2000 bp.
2. We attempted to ligate SpTPS1 to the intracellular expression vector pPIC9K. after three trails, we failed to ligate the intracellular device in pPIC9K. One possible reason is that the condition of ligation is not appropriate.
Sequence
Verification of SpTPS1 after ligation
To verify that the sequence of SpTPS1 gene after ligation in the carrier pPIC9K is still as same as it was before the ligation, we chose three samples of the bacteria solution and did sequencing.
After sequencing, we chose the sample with the sequence 100% identical to the sequence before ligation to carry on the following experiments. The sequencing result is 100% identical, so the sequence of SpTPS1 gene didn’t change in this process.
Sequence Comparison
Figure 2-4. Comparison of the sequence
The sequencing results of the gene after ligation along with the original sequencing. The first line is the original sequence and the second line is the sequence after ligation.
SDS-Page
Because expression of the yeast vector pPIC9K which contains the foreign gene functioned in the yeast genome, to verify that SpTPS1 had been successfully recombined to the yeast genome, we did PRC of the transformed yeast. The following picture showed that the extracellular vector which contain SpTPS had been successfully recombined into the genome of yeast.
Figure 2-5 Gel electrophoresis of SpTPS1-contain yeast genome
The left column is the marker, and 1 stands for the control which is empty vector; 2 stands for ScTPS1 in the intracellular expression device;3 stands for SpTPS1 in the extracellular expression, The sizes of the two strips are above 2000 bp.
We culture the device that ligated vector PIC9K and gene SpTPS1 with YPD medium. We use 1% methanol to induce the protein that will produce under our gene SpTPS1 controlled. After ultrasonication, we break the membrane of the yeast then put the protein into SDS-Page. However, we didn’t got positive result. The possible explanations are that the expression rate of the gene is too low or the condition of the secretion is not suitable.
gltB
E.coli glutamate synthase (gltB) gene has a gene length of 924 bp. gltB is found in E.coli and used to enhance the its tolerance to hypertonic environment. gltB is the key enzyme in the synthesis pathway of glutamic acid. Glutamic acid is also a small molecular substance similar to trahalose, and it can also help the cell to regulate osmotic pressure. Therefore, overexpression of gltB can increase E.coli’s ability to regulate the osmotic pressure. We transformed this gene into the yeast and expected that it will have the same regulating effect as in the yeast.
Optimization
Sequence before optimization
AACACACCTTATGACAGTCAGGAATTGACTGTTTCTCTAACGACTTCCCTTTTAGCCTTAAAGATAAAATCCATTTTAATTTCAGTCATTTAATAAAGAATTTTGCGCTAAAGCACATTTCTGTACCAATAAGCTTGCCATTTGACCTGTATCAGCTTTCCCGATAAGTTGGAAATCCGCTGGAAGCTTTCTGGATGAGCAGCCTGCTCATCATATTTATGCAGTAATTGAGATCCCCTCTTCACCGTATTAACCGATGCGAAAAGGACAACAAGGGGGCGAATCGGAGGCGCGCGTATGACACGCAAACCCCGTCGCCACGCTCTTTCTGTGCCCGTGCGCAGCGGTTCGGAAGTGGGGTTCCCGCAGAGCCTGGGGGAGGTTCACGATATGTTGTACGATAAATCCCTTGAGAGGGATAACTGTGGTTTCGGCCTGATCGCCCACATAGAAGGCGAACCTAGCCACAAGGTAGTGCGTACTGCAATACACGCACTGGCCCGCATGCAGCACCGTGGCGCGATTCTCGCCGATGGTAAAACCGGCGACGGTTGCGGCTTGCTGTTACAAAAACCGGATCGCTTTTTTCGCATCGTTGCGCAGGAGCGCGGCTGGCGTTTAGCAAAAAACTACGCTGTCGGGATGCTCTTCCTGAATAAAGATCCTGAACTCGCCGCTGCCGCACGCCGCATCGTTGAAGAAGAACTGCAACGCGAAACCTTGTCGATTGTGGGCTGGCGTGATGTCCCCACTAACGAAGGCGTGCTGGGTGAAATCGCCCTCTCCTCTCTGCCACGCATTGAGCAAATTTTTGTGAACGCCCCGGCAGGCTGGCGTCCACGCGATATGGAGCGCCGTCTGTTTATCGCCCGCCGCCGCATTGAAAAGCGTCTCGAAGCCGACAAAGACTTCTACGTCTGTAGCCTGTCGAATCTGGTGAACATCTATAAAGGTCTGTGTATGCCGACGGATCTGCCGCGCTTTTATCTGGATCTTGCGGACCTGCGTCTGGAATCGGCCATTTGCCTGTTCCACCAGCGCTTCTCCACTAACACCGTACCGCGCTGGCCGTGGCGCAACCGTTCCGCTATCTGGCGCATAACGGTGAAATCAACACCATCACCGGTAACCGCCAATGGGCGTGCGCGTACTTATAAATTCCAGACACCGCTTATCCCTGACCTGCACGACGCCGCACCGTTCGTC
Optimization
The requirement of the optimization is as same as the first two genes in order to get better expression of them in the yeast.
gltB
Figure3-1: The relative position of codons after optimization
The distribution of codons usage frequency along the length of the gene sequence. A CAI of 1.0 is considered to be perfect in the desired expression organism, and a CAI of > 0.8 is regarded as good, in terms of high gene expression level.
Sequence after optimization
ATGACTAGAAAACCAAGAAGACATGCTTTATCTGTTCCAGTTAGATCAGGTTCAGAAGTTGGTTTTCCACAATCTTTGGGTGAAGTTCATGATATGTTGTACGATAAATCATTGGAAAGAGATAACTGTGGTTTCGGTTTGATCGCTCATATTGAAGGTGAACCATCACATAAAGTTGTTAGAACTGCAATTCATGCTTTAGCAAGAATGCAACATAGAGGTGCTATTTTGGCAGATGGTAAAACAGGTGACGGTTGTGGTTTGTTGTTGCAAAAGCCAGATAGATTTTTCAGAATCGTTGCTCAAGAAAGAGGTTGGAGATTAGCTAAAAATTACGCAGTTGGCATGTTATTTTTGAACAAAGATCCAGAATTGGCTGCTGCTGCTAGAAGAATCGTTGAAGAAGAATTACAAAGAGAAACTTTGTCTATTGTTGGTTGGAGAGATGTTCCAACAAATGAAGGTGTTTTGGGTGAAATCGCATTATCTTCATTGCCAAGAATCGAACAAATTTTTGTTAACGCTCCAGCAGGTTGGAGACCAAGAGATATGGAAAGAAGATTGTTTATTGCTAGAAGAAGAATCGAAAAGAGATTGGAAGCAGATAAGGATTTCTACGTTTGTTCTTTGTCAAATTTGGTTAACATCTATAAGGGTTTGTGTATGCCAACTGATTTGCCAAGATTCTATTTGGATTTGGCTGATTTGAGATTGGAATCTGCAATTTGTTTGTTCCATCAAAGATTTTCAACTAACACAGTTCCAAGATGGCCTTGGAGAAATAGATCAGCTATTTGGAGAATTACTGTTAAATCTACACCATCACCAGTTACAGCTAATGGTAGAGCAAGAACTTACAAGTTCCAAACACCATTAATCCCAGATTTGCATGATGCTGCACCATTTGTTTAA
Amplification of gltB
To get the desired segments of our target gene gltB:
Amplify the gene through PCR;
The forward primer is designed to be ATGACTAGAAAACCAAGAAG;
The reverse primer is designed to be TTAAACAAATGGTGCAGC;
As shown in the figure 1-2, we successfully cloned segments of our gene gltB.
Figure 3-2 Gel electrophoresis of gltB cloned from PUC57
The left column is the marker, and the two strips on the right are PCR products of gltB. The sizes of the two strips are about 900 bp.
Construction of the device on PIC9K
We ligated the gltB gene into pPIC9K at SnaBⅠ and NotⅠ restriction enzyme cutting sites. The promoter is AOX1, which will promote the expression of the gltB gene. Then the completed pPIC9K will be transformed into GS115 yeast.
Verification of gltB PCR product
1.To test whether gltB had been successfully ligated to the extracellular expression vector pPIC9K, the device containing our gene gltBfor the extracellular expression is amplified and the results proved that it had been digested and ligated successfully.
Figure 3-3 The electrophoresis result of the gltB in the extracellular expression device
The electrophoresis result showed that the gltB is successfully ligated in the extracellular expression device since we got a single band above 2000 bp.
2. To test whether gltB had been successfully ligated to the intracellular expression vector pPIC9K，the device containing our gene gltB for the intracellular expression is amplified and the results proved that it had been digested and ligated successfully
Figure 3-4 The electrophoresis result of the gltB in the intracellular expression device
The electrophoresis result showed that the gltB was successfully ligated in the intracellular expression device since we got a single band about 750 bp large.
Sequence
Verification of gltB after ligation
To verify that the sequence of gltB gene after the ligation is as same as it was before the ligation, we chose three sample of the bacteria solution to do the sequencing. After the sequencing, we chose the sample with the sequence 100% identical to the sequence before optimization to carry on the following experiments. The sequencing result is 100% identical, so the sequence of gltB gene didn’t change in this process.
Figure 3-5. the comparison of the sequencing
The figure shows the sequence of the original sequence along with the sequence of the gene after the ligation. The first line shows the original sequence, and the second line shows the sequence of the ligated gene.
SDS-Page
Because expression of the yeast vector pPIC9K which contains the foreign gene functioned in the yeast genome, to verify that gltB had been successfully recombined to the yeast genome, we did PRC of the transformed yeast. The following picture showed that the extracellular vector which contain gltB had been successfully recombined into the genome of yeast.
Figure 3-6 Gel electrophoresis of gltB-contain yeast genome
The left column is the marker, and 1,2 stands for gltB in the extracellular expression device.
Figure 3-7. Gel electrophoresis of gltB-contain yeast genome
The left column is the marker, and 1,2 stands for gltB in the intracellular expression device.
We cultured the device with the ligated vector PIC9K and gene gltB in YPD. We used 1% methanol to induce the protein that will produce. After ultrasonication, we broke the membrane of the yeast and then put the protein into SDS-Page. However, we didn’t get positive results. The possible reason is that the expression rate of this gene is too low or the condition of the secretion is not suitable.
Sponsors
