<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Oxford_Safety skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Oxford/Safety</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Project
          </A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Description">Background</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Our_solution">Our Design</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Improve">Part Improvement</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/InitialIdeas">Initial Ideas</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Wet Lab
          </A></LI><LI><A href="https://2018.igem.org/Team:Oxford/wetlab">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/InterLab">Interlab</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/wet_lab_safety">Wet Lab Safety</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/future_experiments">Future Experiments</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Dry Lab
          </A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Model">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Model/Time-Domain-Analysis">Time Domain Analysis</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Model/Frequency-Domain-Analysis">Frequency Domain Analysis</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Model/Killswitch">Kill Switch</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Human Centric Design
          </A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Applied_Design">Product Design</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/ethics">Ethics</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">People
          </A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Awards">Awards</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Team">Meet the Team</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Sponsors">Sponsors</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Medals">Medals</A></LI></DIV></DIV><TITLE>Full Width Pics - Start Bootstrap Template</TITLE><HEADER class="py-5 background-img parent" style="height: 400px;"><A href="#n1" style="text-decoration : none; color:white;"><DIV class="child background-img" style="background-image: url('https://static.igem.org/mediawiki/2018/9/92/T--Oxford--hildas.jpg'); background-size: cover;"><DIV class="translucent-back" style="height: 400px;"><H1 class="header-text2">Safety</H1></DIV></DIV></A></HEADER><SECTION id="n1"><DIV class="container"><DIV class="row"><DIV class=".col-md-8 .offset-md-2"><H2>Introduction</H2><P>Genetically modified probiotics have a huge potential to treat a wide range of health problems such as autoimmune diseases, metabolic disorders and gut infections. However, one of the largest barriers to their acceptance is concerns about safety. People are always suspicious of genetic engineering technology, and the idea of placing GM bacteria into your body would be very daunting for most people. When we received the results of our survey we got quotes such as these.<I>“[I] would want treatment to be fully tested and certified as safe before trying it.”</I><I>“[You] need to ensure that your system is completely safe for the patient taking the medicine and the environment.” </I>

We decided to take inspiration from the 2012 Bettencourt team and design multiple safety features to try and make the failure rate as low as possible. Our design would be a template for other researchers to use when creating GM probiotic systems. We hope that this will improve public acceptance of GM probiotics and lead to their wider use. We also drew up a policy proposal for governments who would be considering the use of this technology. 

Our safety features fall into three main categories. <UL><LI><B>-Environmental containment:</B> preventing the bacteria from escaping into the environment from a laboratory or a patient. It is important that other people or animals do not accidentally ingest the bacteria from the local environment. Autonomy is an important principle in medical ethics so it is essential that only consenting patients have the bacteria in their system. </LI><LI><B>-Preventing gene transfer:</B> we want to avoid other bacteria gaining our artificial plasmids. They will not have the same safety features as our bacteria and our genes may interact with other species in an unintended manner. People could also ingest the bacteria with our genes and this could have unforeseen effects on their bodies. </LI><LI><B>-Removing bacteria during treatment:</B> it is possible that a patient may be experiencing side effects during the treatment or they may want to discontinue it. In this case, we would want to be able to remove the bacteria without modifying the rest of the microbiome. </LI></UL></P></DIV></DIV></DIV></SECTION><SECTION id="n2"><DIV class="container"><DIV class="row"><DIV class=".col-md-8 .offset-md-2"><H2>Environmental containment</H2><H3>NUS Singapore probiotic kill switch</H3><P>In 2017 NUS Singapore designed a kill switch which would kill any bacteria which left the gut. 
The kill switch detects both temperature and phosphate concentrations. It then uses an OR logic gate to decide if it will kill the bacteria using the E2 toxin. If the bacteria detect high levels of phosphate or a temperature above 36 degrees then they will survive due to the production of the IM2 antitoxin. However, if neither of these signals are present, the E2 toxin will kill the bacteria.  

TlpA is a promoter which is active above 36 degrees, upregulating IM2 production. High phosphate concentration upregulates IM2 as well.  

When using toxins, there is always a possibility of the bacteria becoming immune, or the toxin suffering from a loss of function mutation. The other possible issue we could foresee is the repressor proteins losing sensitivity and constantly producing the IM2 antitoxin. </P><H3>Auxotrophies </H3><P>Auxotrophy is when an organism has a mutation compared to the wild-type which means that it requires additional nutrients. There are many different auxotrophies such as amino acids and vitamins. Auxotrophies are often used in synthetic biology as a means of containing organisms within a certain area.  

We propose using uracil and thymidine auxotrophies. Both of these nucleotides are found in large enough concentrations in the gut for the bacteria to survive. When the bacteria leave the gut, they will not be able to replicate their DNA due to a lack of nucleotides. They will eventually die due to starvation and the accumulation of mutations in the DNA. This technique has already been used in trials of humans ingesting live bacteria.  

The auxotrophy can fail if horizontal gene transfer gives the bacteria the correct biosynthesis pathways. </P></DIV></DIV></DIV></SECTION><SECTION id="n3"><DIV class="container"><DIV class="row"><DIV class=".col-md-8 .offset-md-2"><H2>Preventing gene transfer</H2><H3>Toxin-antitoxin systems </H3><P>Toxin-antitoxin systems are used in synthetic biology to prevent plasmids from being lost in the bacteria. However, these systems could also be used to drastically reduce the probability of genes being transferred to other organisms.  

Our bacteria would have three artificial plasmids, two for the therapeutic effect and one for the probiotic kill switch. Each gene would have a different toxin and antitoxin. These genes would then work as a network of toxin-antitoxin systems. All of the plasmids would need to be present together to prevent the toxins from working. For the system to fail all plasmids would have to be transferred within a very short space of time. Or alternatively, all three toxins would have to suffer from a loss of function mutation.  

The toxins would be fitted with a repressor which only works in the presence of an artificial chemical. This would mean that the plasmids could be transferred under laboratory conditions. </P></DIV></DIV></DIV></SECTION><SECTION id="n4"><DIV class="container"><DIV class="row"><DIV class=".col-md-8 .offset-md-2"><H2>Removing bacteria during treatment </H2><P>As mentioned before, autonomy is an essential principle in medical ethics. It is therefore important that patients can stop treatment at any time. They may be experiencing side effects or they may just not feel comfortable using the treatment anymore. In that case, they would want to kill the probiotic bacteria in their stomach.  

It is possible to kill the bacteria by just using an antibiotic at a high enough dosage to kill all bacteria in the microbiome. However, this is not an ideal solution as it can have a very adverse effect on the microbiome. The diversity of the microbiome will drastically drop and it can take months for it to recover. It can also dramatically increase the incidence of antibiotic resistance genes (in one study using tetracycline it was shown to increase the number from 307.3 to 1492.7 ppm). We, therefore, think that this should only be used as a last resort, in the case of severe side effects.  

We propose that there should be a kill switch which only works on The design of a kill switch appears very simple at first. There are many examples of kill switches which have been designed for a variety of reasons. There are two main types: a deadman kill switch which requires the constant exposure of the bacteria to a molecule to prevent death, and an inducible kill switch, which uses a molecule to induce cell death. We were told by the numerous health care professionals who we met that an inducible kill switch would be preferable. Deadman switches require a daily dose of supplement, something which will be more time consuming and costly.  

A probiotic kill switch has a number of limitations. </P><UL><LI>-The inducer molecule must be non-toxic to humans at the doses we are using. </LI><LI>-It cannot be found naturally in the human body beyond a certain concentration otherwise the natural molecules will affect the bacteria. This means that most biomolecules found in plants and animals can't be used. </LI><LI>-It must be able to have a biological effect. This means that a lot of artificial molecules cannot be used. </LI><LI>-It must be cheap and easy to produce. Some signalling molecules are too expensive to use.</LI><LI>-It must be stable at a high pH and not be broken down at earlier points in the digestive system. </LI><LI>-It must be able to easily enter the cell.</LI></UL><P>We researched many different possibilities, however, we eventually decided on a tetR based kill switch. TetR is a very well characterised repressor which is often used in synthetic biology. We linked TetR to an artilysin toxin which the 2015 Oxford iGEM team worked with. TetR can be induced by a number of molecules. Obviously, we want to avoid using tetracycline because of its antibiotic properties. Doxycycline is a similar molecule which binds much more strongly (it works at concentrations as low as 1-2ng/ml. However, it can also be used as an antibiotic so it will be toxic to the microbiome.  

We would, therefore, use computer-based drug design to find a suitable inducer molecule. With this technique, we can search through databases for molecules similar to tetracycline. We would then screen them for their toxicity and binding properties and choose the molecule which showed the strongest tetR binding and the least antibiotic properties. We would then perform clinical trials to ensure that the inducer is safe for human consumption.  

The kill switch passes all the criteria we set earlier. </P><UL><LI><B>-The inducer molecule must be non-toxic to humans at the doses we are using:</B> tetracycline can already safely be used in humans. We will be using a less toxic molecule at a much lower concentration (due to its higher affinity for tetR). </LI><LI><B>-It cannot be found naturally in the human body beyond a certain concentration otherwise the natural molecules will affect the bacteria: </B>tetracycline analogues are artificial molecules which will not normally be ingested by humans. </LI><LI><B>-It must be able to have a biological effect: </B>tetracycline has well characterised and documented biological effects. We would screen our derivative for biological effects.</LI><LI><B>-It must be cheap and easy to produce:</B> chemicals such as tetracycline are cheap to produce and widely available. </LI><LI><B>-It must be stable at a high pH and not be broken down at earlier points in the digestive system: </B>tetracycline is widely used as an antibiotic so obviously, it is stable in the digestive system. We would ensure that any derivatives we use would be the same. </LI><LI><B>-It must be able to easily enter the cell:</B> tetracycline and its analogues have already been widely used in synthetic biology experiments and they can easily enter the cell. </LI></UL></DIV></DIV></DIV></SECTION><SECTION id="n5"><DIV class="container"><DIV class="row"><DIV class=".col-md-8 .offset-md-2"><H2>Policy proposal</H2><P>We created a policy proposal for the implementation of new laws and regulations regarding genetically modified probiotics. At the moment there are no specific regulations for GM probiotics. We believe that the safety features we have outlined would be sufficient protection to prevent environmental contamination or harm to the patient.</P><DIV class="panel-group"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#safetyproposal">Safety Policy Proposal</A></H4></DIV></DIV></DIV><H2> Reference </H2><TABLE class="tftable" border="1"><CENTER><TBODY><TR><TH>References</TH></TR><TR><TD>Yin, J., Zhang, X. X., &amp; Wu, B. (2015). Metagenomic insights into tetracycline effects on microbial community and antibiotic resistance of mouse gut. Ecotoxicology, 2125-2132.</TD></TR><TR><TD>Tet systems. (2008). Tet systems. Retrieved October 12, 2018, from Principles and Components Description: http://www.tetsystems.com/science-technology/principles-components/</TD></TR><TR><TD>iGEM Registry of Standard Biological Parts. (2003, January 31). Retrieved October 11, 2018, from Part: BBa_R0040: http://parts.igem.org/Part:BBa_R0040</TD></TR><TR><TD>NUS iGEM 2017. (2017). Project descirption. Retrieved October 12, 2018, from NUS iGEM 2017: https://2017.igem.org/Team:NUS_Singapore/Description</TD></TR></TBODY></CENTER></TABLE></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></BODY></HTML>