<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Oxford_results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Oxford/results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Project
          </A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Description">Background</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Our_solution">Our Design</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Improve">Part Improvement</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/InitialIdeas">Initial Ideas</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Wet Lab
          </A></LI><LI><A href="https://2018.igem.org/Team:Oxford/wetlab">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/InterLab">Interlab</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/wet_lab_safety">Wet Lab Safety</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/future_experiments">Future Experiments</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Dry Lab
          </A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Model">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Model/Time-Domain-Analysis">Time Domain Analysis</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Model/Frequency-Domain-Analysis">Frequency Domain Analysis</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Model/Killswitch">Kill Switch</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Human Centric Design
          </A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Applied_Design">Product Design</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/ethics">Ethics</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">People
          </A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Awards">Awards</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Team">Meet the Team</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Sponsors">Sponsors</A></LI><LI><A href="https://2018.igem.org/Team:Oxford/Medals">Medals</A></LI></DIV></DIV><TITLE>Full Width Pics - Start Bootstrap Template</TITLE><HEADER class="py-5 background-img parent" style="height: 400px;"><A href="#n1" style="text-decoration : none; color:white;"><DIV class="child background-img" style="background-image: url('https://static.igem.org/mediawiki/2018/8/8f/T--Oxford--Results.png'); background-size: cover;"><DIV class="translucent-back" style="height: 400px;"><H1 class="header-text2">Results</H1></DIV></DIV></A></HEADER><SECTION id="n1"><DIV class="container"><DIV class="row"><DIV class=".col-md-8 .offset-md-2 .bg-light"><H2>Riboswitch Fluorescence Assay</H2><P>The riboswitch parts were characterised by inserting a unidirectional Tet promoter before the upstream folding region and after it. The cells transformed with these constructs were then grown in LB at 37<SUP>o</SUP>C for 3 hours. This would allow us to assess this importance of this region in the sensitivity to the binding of adenine. We decided to use a plate reader assay which would measure the fluorescence of eCFP at 480nm. Adenine has a solubility in water at pH 7 of 7.4uM which is insufficient to accurately measure the mass of solute for a stock solution.</P><P>In preliminary experiments, adenine was dissolved in 100% DMSO to a stock solution of 15uM. The DMSO concentration was shown to be high enough to inhibit cell growth. We believe that this will have altered the sensitivity of the riboswitch and invalidated our results.</P><P>To overcome this issue, we changed our inducer from adenine to adenine sulfate dihydrate which is soluble in water to 15uM. We repeated the assay using this new stock. This assay, shown below, shows that in LB, noise exceeds the response to the adenine. </P><P>Searches in the literature of riboswitch aptamer studies show that minimal media results in an increased sensitivity. Repeating of a fluorescence assay with K12 cells grown in M9 minimal media at 37<SUP>o</SUP>C shows, in the graph below, significant increases in eCFP production in response to adenine showing the riboswitch is functional under these conditions. This shows proof of concept that the adenine signal can be integrated into a bacterial system.</P></DIV></DIV></DIV></SECTION><SECTION id="n2"><DIV class="container"><DIV class="row"><DIV class=".col-md-8 .offset-md-2 .bg-light"><H2>Ribozyme Fluorescence Assay</H2><P>Although we could not assemble a functional ribozyme in the shipping vector, we had expression vectors which used sRNA to repress sfGFP, controlled by a shared bidirectional pTet. Using these plasmids, we could use to validate our decision to include it in the design. To confirm the necessity of the ribozyme in activating the sRNA, we performed a plate reader assay for sfGFP by measuring the fluorescence at 520nm after 3 hours in response to 10nM aTc in a range of constructs with fluorescent proteins under the control of a bidirectional pTet.</P><P>The sRNA shows ineffective inhibition of sfGFP production unless a functional ribozyme is present. The Human Hammerhead Ribozyme allowed a 75% knockdown of translation using a 25 base pair binding sequence. These values are an indicator of the extent of IL-10 knockdown that our system can achieve.</P><DIV class="panel-group"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#profile">Expression Vectors</A></H4></DIV><DIV id="profile" class="panel-collapse collapse"><DIV class="panel-body"><FIGURE><FIGCAPTION><CENTER> Left : Human Hammerhead Ribozyme Vector - Right : Vector with No Ribozyme </CENTER></FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><CENTER> Left : Vector with RFP - Right : Vector with no sRNA</CENTER></FIGCAPTION></FIGURE></DIV></DIV></DIV></DIV><P>As shown in the results of the assay, the ribozyme is critical for effective sRNA mediated knockdown. The ribozyme construct enables the sRNA to repress translation of sfGFP mRNA and high sensitivity of the system to adenine.</P><DIV class="container"><DIV class="row"><DIV class=".col-md-8 .offset-md-2"><H2>Bidirectional Promoter Fluorescence Assay</H2><P class="lead">In addition to characterising the kill switch under the control of the bidirectional Tet promoter, we decided that characterising the strength of the promoter by looking at the production of sfGFP would provide critical information for future experiments. The obtained parameters would allow future models to accurately predict expression of a new part in response to known levels of aTc.
</P><P>To accurately measure the activity of the promoter, two distinct constructs were used. The negative control construct contains an RFP gene under the control of the bidirectional promoter in one direction and TetR in the other. The expression vector we tested in the assay was identical to the negative control however RFP was replaced with sfGFP, the fluorescence of which was measured at 520nm. 
</P><P>By finding the concentration at which the fluorescence is equal to in the absence of inducer, we can determine that the promoter is insensitive to concentrations below \(1 \, nM\).
</P><P>Our data also was used to determine the leakiness of the promoter. Basal gene expression was obtained by measuring fluorescence in the absence of aTc relative to the negative control. This comparison was critical for isolating that the only contributing factor was the promoter inhibited by TetR.    
</P><P> The steady-state plot for the Bidirectional Promoter was used for data fitting which yielded a value of 1nMmin<SUP>-1</SUP> for \(\beta\) for the Rep101 origin of replication. This results in a maximal transcription rate of 200pMmin<SUP>-1</SUP>plasmid copy<SUP>-1</SUP>. More on how experimental data was integrated into the modelling can be found <A href="https://2018.igem.org/Team:Oxford/Model/Killswitch"> here.</A></P><DIV class="panel-group"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#map">Expression Vectors</A></H4></DIV><DIV id="map" class="panel-collapse collapse"><DIV class="panel-body"><FIGURE><FIGCAPTION><CENTER>Left : Vector with RFP - Right : Vector with sfGFP</CENTER></FIGCAPTION></FIGURE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION id="n3"><DIV class="container"><DIV class="row"><DIV class=".col-md-8 .offset-md-2 .bg-light"><H2>Bidirectional pTeT Controlled Kill Switch Growth Assay</H2><P class="lead">The kill switch was assayed by measuring the OD of cultures subjected to varying levels of the inducer. While not intended to be used in final probiotic, the tetracycline analogue, anhydrotetracycline (aTc), has been shown to have a lower antibiotic activity and affinity for TetR and was thus chosen for use in our experiments. By comparing the growth of cells transformed with a plasmid lacking the promoter and cells with a plasmid lacking the killswitch and the promoter, we showed that cell growth was not affected by the kill switch. 
</P><P>Cell growth was unaffected by leaking of the promoter in cells with the bidirectional kill switch construct, showing that our device will not restrict the ability for the chassis to colonise the target region of the gut.
</P><P>The bidirectional promoter characterisation induces cell lysis in response to concentrations of aTc equal to or above 1nM. Higher concentrations increase the rate of expression up to concentrations of 20nM. Above this concentration, the promoter is at maximum activity. At concentrations above 20nM, the rate of cell growth slows in the kill switch assay. This can be partially attributed to aTc toxicity, however, further experiments are required to ascertain if our construct increases sensitivity to aTc or if it exhibits further effects when the promoter is saturated. 
</P><P>By plotting the cell density after 3 hours against aTc concentration in both wild-type and with the killswitch construct, the efficacy of the part can be demonstrated. Cell growth for negative controls, while higher than cells with the kill switch construct, was reduced by high aTc concentrations. This confirms the literature which describes aTc as having less antibiotic activity than tetracycline. An analogue, however, must be found with lower antibiotic activity or increased binding affinity for TetR for a sufficiently selective and effective inducible kill switch in patients.</P></DIV></DIV></DIV></SECTION><SECTION id="n4"><DIV class="container"><DIV class="row"><DIV class=".col-md-8 .offset-md-2 .bg-light"><H2>Unidirectional pTet Controlled Kill Switch Growth Assay</H2><P>The assay for the unidirectional kill switch was performed using the same protocol as the bidirectional kill switch. Although sequencing showed the SDM occurred, it is believed that base deletions during the amplification step resulted in a loss of function mutation in the promoter, preventing Art-175 production. Growth assays showed that there is no significant lysis of cells as a function of aTc with the submitted part.</P></DIV></DIV></DIV></SECTION></DIV></DIV></DIV><SECTION id="n6"><DIV class="container"><DIV class="row"><DIV class=".col-md-8 .offset-md-2"><TABLE class="tftable" border="1"><CENTER><TBODY><TR><TH>Index</TH><TH>Reference</TH></TR><TR><TD>A</TD><TD>https://www.sigmaaldrich.com/content/dam/sigma-aldrich/docs/Sigma/Product_Information_Sheet/a8626pis.pdf</TD></TR></TBODY></CENTER></TABLE></DIV></DIV></DIV></SECTION></DIV></DIV></BODY></HTML>