Team:Oxford/experiments
Project
Background
Our Design
Parts
Part Improvement
Initial Ideas
Wet Lab
Overview
Experiments
Results
Protocols
Notebook
Interlab
Wet Lab Safety
Future Experiments
Dry Lab
Overview
Time Domain Analysis
Frequency Domain Analysis
Kill Switch
Human Centric Design
Human Practices
Entrepreneurship
Public Engagement
Product Design
Safety
Ethics
People
Awards
Meet the Team
Collaborations
Attributions
Sponsors
Medals
Full Width Pics - Start Bootstrap Template
Experiments
Cloning
We designed our gBlocks using Snapgene software. They were then synthesised by IDT alongside the primers we designed. Some of our constructs proved difficult for IDT to make so we decided to break them down into smaller pieces and reconnect them in the lab. These were resuspended in MilliQ upon arrival and stored at -20oC.
We began by amplifying our DNA by PCR. For each of our 12 gBlocks we had designed individual primers which extend out of the Biobrick prefixes and suffixes to try and ensure there would be no non-specific binding/contaminants would not be amplified. We also used overlap extension PCR to reconstruct some of our gBlocks that IDT could not synthesize. Unfortunately, we struggled to find conditions which led to successful amplifications for some of the parts. This led to us resorting to using smaller primers which only anneal to the BioBrick sequences. After running some gradient PCRs probing both temperatures and DMSO content we found that for the majority of our gBlocks, 5% DMSO, 60oC were the optimal conditions.
After PCR, the correctly amplified fragments were separated by running on 1% agarose gels. Achieving a high concentration of DNA from our gels proved difficult but these were resolved once when we began using a different extraction kit.
Digest & Ligate
Our initial method to insert our DNA into the pSB1C3 backbone was to digest both the backbone and our DNA using restriction enzymes and then ligate the fragments together. We chose to use a stock of pSB1C3 containing the insert NH75 left over the linear form provided by iGEM. The reasoning for this was so that we could immediately transform and culture to create a stock. During the first attempts to digest and ligate our constructs, we found that, upon transforming, many false positives from linearized plasmid being re-ligated without our inserts. This led to us treating our vectors with antarctic phosphatase prior to ligating. This eliminated our false positives.
After preventing re-ligation, no viable colonies grew after transformation and antibiotic selection. Nanodropping our digestion products and digested vectors showed low DNA concentrations which we believed were causing the problems. Improving concentration of products was performed, successfully, by cleaning PCR mixes directly without running it on a gel first. Ligated vector concentrations were further improved using an isopropanol/ethanol DNA precipitation procedure followed by resuspension in nuclease-free water. Following these approaches, colonies grew following transformation.
Despite correctly sized bands from colony PCRs of viable colonies after transformations with high DNA concentration, sequencing showed we did not succeed in constructing the intended vectors. This led to us considering alternative methods of inserting our gBlocks into vectors.
Gibson Assemblies
After failing with traditional restriction enzymes we decided to design new primers for our constructs that would allow us to attempt the assemble them via a Gibson Assembly. Gibson Assemblies involve having homologous sections at the ends of DNA fragments which are then exposed by an exonuclease, making complementary strands which may anneal together and be ligated. To introduce the homologous we designed primers which, when used to amplify our gBlocks, added up to 30 base pairs from the terminus of the part to which it is to be ligated. Despite initially showing promise, this method also failed to prove fruitful for us, despite high concentrations of vector and insert and a range of insert excess tried from 1:1 to 1:10, causing us to move on.
Site-Directed-Mutagenesis (SDM)
This versatile method allows you to insert, delete or modify fragments of circular DNA and over the course of the project, we made use of all 3 avenues. To rid our stocks of the initial insert, NH75, we first carried out a deletion on the stock of pSB1c3, leaving no bases between the BioBrick prefix and suffix. Later in the project when we began working on kill switches we designed primers that would insert uni and bidirectional promoters upstream of the artilysin we were improving from Oxford’s 2015 iGEM team. A mistake in the design of these primers resulted in us creating an invalid bidirectional promoter due to an EcoRI site being present. This was corrected by a single base mutation using SDM.
Another way we used SDM was to attempt to test a novel method of introducing large sections of DNA into vectors without having to use the above techniques. The length of DNA that can be inserted by SDM is generally limited to around 100 bases due to the inability to make effective primers long enough and that non-specific binding becomes a larger issue with primer length. Currently, to repeatedly carry out insertions in this manner would be slow due to the need to transform, culture and miniprep before another SDM could be attempted. Our method to bypass this would be to design primers in such a way that the first treatment would introduce a restriction site into the vector which would be removed by the second round of SDM. This would allow you to take the vector product of the first SDM and immediately SDM again. Treating this product with the restriction enzyme would linearize any remaining product from the initial SDM thus ensuring that only the intended vector will be transformed, consequently doubling the rate at which insertions can be conducted.
Labelling System
It is important in the lab to know what you’re working with but also be efficient with your time. Hence we created a few cheat sheets to speed up preparing PCRs as well as labelling all our constructs. We printed these nifty tables out and hung them up in the lab!
