<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Stockholm_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Stockholm/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="post-template nav-closed"><DIV class="navigation"><DIV class="nav-content"><DIV class="nav-items"><DIV class="nav-project" role="presentation"><A href="/Team:Stockholm/Description">Project</A><DIV class="nav-sublevel"><A href="/Team:Stockholm/Description#inshort">In Short</A><A href="/Team:Stockholm/Description#ourpath">Our Path</A><A href="/Team:Stockholm/Description#biofilm">Background</A><A href="/Team:Stockholm/Description#ourproject">Our Project</A><A href="/Team:Stockholm/Description#combatproteins">Combat Proteins</A><A href="/Team:Stockholm/Description#sortasea">Sortase A</A><A href="/Team:Stockholm/Future">Future</A><A href="/Team:Stockholm/Medals">Medal achievements</A></DIV></DIV><DIV class="nav-parts" role="presentation"><A href="/Team:Stockholm/Parts">Parts</A><DIV class="nav-sublevel"><A href="/Team:Stockholm/Parts">All parts</A><A href="/Team:Stockholm/Basic_Parts">Basic Parts</A><A href="/Team:Stockholm/Part_Collection">Part Collection</A></DIV></DIV><DIV class="nav-wet-lab" role="presentation"><A href="/Team:Stockholm/Experiments">Lab</A><DIV class="nav-sublevel"><A href="/Team:Stockholm/Experiments">Experiments</A><A href="/Team:Stockholm/Labbook">Lab book</A><A href="/Team:Stockholm/Protocols">Protocols</A></DIV></DIV><DIV class="nav-results" role="presentation"><A href="/Team:Stockholm/Proof">Results</A><DIV class="nav-sublevel"><A href="/Team:Stockholm/Proof">Proof of concept</A><A href="/Team:Stockholm/Results-nuc">Nuc</A><A href="/Team:Stockholm/Results-lys">Lys</A><A href="/Team:Stockholm/Results-def">Def</A><A href="/Team:Stockholm/Results-esp">Esp</A><A href="/Team:Stockholm/Results-sortase-a">Sort A</A></DIV></DIV><DIV class="nav-team" role="presentation"><A href="/Team:Stockholm/Team">Team</A><DIV class="nav-sublevel"><A href="/Team:Stockholm/Team">Members</A><A href="/Team:Stockholm/Collaborations">Collaborations</A><A href="/Team:Stockholm/Attributions">Attributions</A><A href="/Team:Stockholm/Sponsors">Sponsors</A><A href="/Team:Stockholm/Diary">Diary</A></DIV></DIV><DIV class="nav-practises" role="presentation"><A href="/Team:Stockholm/HP/Silver">Human Practices</A><DIV class="nav-sublevel"><A href="/Team:Stockholm/HP/Silver">CARD</A><A href="/Team:Stockholm/HP/Gold">TECH</A><A href="/Team:Stockholm/Integrated_practices">Integrated</A><A href="/Team:Stockholm/Engagement">Engagement</A></DIV></DIV><DIV class="nav-safety" role="presentation"><A href="/Team:Stockholm/Safety">Safety</A></DIV><DIV class="nav-nic" role="presentation"><A href="/Team:Stockholm/NiC">NiC</A></DIV></DIV></DIV></DIV></DIV><DIV class="site-wrapper post-template"><MAIN class="content"><ARTICLE class="post"><HEADER class="post-header"><H1 class="post-title">SMITe - The Concept</H1></HEADER><SECTION class="post-content"><H2 id="inshort">In short</H2><H4 id="theissue">The Issue</H4><UL><LI>Chronic wounds and antibiotic resistance are issues increasing in importance and often go hand-in-hand</LI><LI>Current treatment strategies for chronic wounds rely on mechanical removal of dead tissue, or systemic antibiotic therapy, which are largely ineffective or come with considerable risks for the patient</LI><LI>A major reason for ineffective treatments are biofilms; structures secreted by bacteria to facilitate establishment of a chronic infection</LI></UL><H4 id="ourgoalsforsmitespidersilkmediatedinfectiontreatment">Our goals for SMITe - Spider Silk Mediated Infection Treatment:</H4><P>We aim to construct a functional dressing for chronic wounds, using spider silk as a scaffold to attach biofilm-degrading enzymes. This includes:</P><UL><LI>Expressing biofilm-degrading 'combat proteins’ and quantifying their potency</LI><LI>Assembling combat proteins with a linker and tag needed for purification and attachment to spider silk</LI><LI>Attaching the enzymes to spider silk using Sortase A, a transpeptidase</LI><LI>Isolating Sortase A from the <EM>S. aureus</EM> genome and creating a ‘Sortase Package’ including basic parts needed for other iGEM teams to use Sortase in their own projects</LI><LI>Testing the efficacy of the fully assembled wound dressing with attached enzymes on biofilms</LI></UL><H2 id="ourpath">Our Path</H2><H4 id="aprojectisborn">A Project is Born</H4><P>We are delighted to say that from its inception, our project has benefitted from being a truly shared endeavor; one which we cannot wait to show you. </P><P>Following initial selection of the team members, we undertook a series of small group exercises which merged team building with the generation of new ideas. Once these were narrowed down to several per group, we tested the ease with which each group could present an unknown idea to the rest of the team. This was a very revealing experiment in terms of the generalisability of an idea, as well as how marketable each one could be.</P><P>It became clear rapidly that the team were stuck between a focus on biomaterials and health, and it was alongside the opportunity to work with recombinant spider silk that our final project plan evolved, joining these two fields perfectly.</P><H4 id="chronicwounds">Chronic Wounds</H4><P>Chronic wounds are a major source of disability and high cost for healthcare systems worldwide, not to mention pain, misery and the continual threat of infection for the affected individual. Referred to as a “snowballing threat to public health”, chronic wound sufferers have been aptly described as “carriers of a silent epidemic”, who are often already diagnosed with “highly branded” diseases including diabetes and obesity <A class="ref" href="#wounds">[1]</A>. In the current context of these conditions twinned with an aging population, the trend in chronic wounds is set only to rise. We decided to approach this clinical problem using <EM>Staphylococcus aureus</EM> biofilm as a model. Since this organism is part of our skin flora, it benefits from easy access to broken skin and easily becomes established in non-healing wounds; it is suggested that <EM>S. aureus</EM> is present in more than 90% of all chronic wounds <A class="ref" href="#aureus">[2]</A>.</P><H4 id="alookbeyond">A Look Beyond</H4><P>In addition, questions on a pressing global public health threat are raised by our project: antimicrobial resistance.  It is well established that bacterial biofilms, fostered by the chronic wound environment, protect bacteria from both antibiotics and an individual’s immune system, generating the perfect backdrop for increasing drug resistance <A class="ref" href="#aureus">[1]</A>. We could not fail to acknowledge the intrinsic link between this world issue and our project plan and as such made it the focus of our final Human Practices and Safety theme.</P><H2 id="biofilm">Biofilm</H2><P>With both chronic wounds and the variety of bacterial resistance mechanisms in our minds, we were concerned about using traditional bactericidal approaches to aid chronic wound healing, which often stumble into one of two pitfalls:</P><UL><LI>a largely unspecific effect which risks to further aggravate the wound <A class="ref" href="#dangers">[3]</A></LI><LI>a limited efficacy on chronic wounds from the very start and / or decreasing efficacy during the course of the treatment <A class="ref" href="#inefficient">[4]</A></LI></UL><P>So instead, iGEM Stockholm decided to focus on a prominent feature of many chronic wounds – the biofilm. Biofilms are complex extracellular structures generated by a variety of bacteria as a self-protection mechanism and are often increased in response to stress such as antibiotic treatment <A class="ref" href="#biofilm">[5]</A>.  They are essential in supporting bacteria when they adopt a ‘sessile’ lifestyle in the form of chronic colonization; they interfere with inflammation meant to rid the wound of bacteria, help the bacteria attach and detach from the wound surface as the infection progresses, and have been found to contribute to quicker development of antibiotic resistance <A class="ref" href="#res1">[6]</A><A class="ref" href="#res2">[7]</A>. In particular the biofilms' contribution to antimicrobial resistance is becoming increasingly problematic and inspired a large proportion of our work in <A href="https://2016.igem.org/Team:Stockholm/HP/Silver">Human Practices</A>.</P><P>The components of biofilms differ among bacterial species, but many of them have been found to serve a specific purpose. Three structures observed commonly in different biofilms are: exopolysaccharides, extracellular DNA, and a variety of biofilm-associated proteins <A class="ref" href="#Flemming">[8]</A>.</P><P>The vital role that biofilm plays in persistent infections provides a solid rationale for making them our primary target. </P><FIGURE><FIGCAPTION>Figure 1: Overview of the typical components of <EM> S. aureus </EM> biofilms</FIGCAPTION></FIGURE><H2 id="ourproject">Our Project</H2><P>Our vision is a functional wound dressing which will be able to disperse early-stage <EM>S. aureus</EM> biofilms in chronic wounds. The scaffold for this dressing is formed by spider silk – a material not only known for its high tensile strength, but also for being biocompatible and largely non-immunogenic. Those features make it an ideal candidate material for potential wound dressings <A class="ref" href="#Meinel">[9]</A>. Since the synthesis of spider silk itself is extremely complex and would constitute a project of its own, we were very lucky to receive ready-made, recombinant spider silk from our adivsors.</P><P>To actively fight the biofilm, several biofilm-degrading 'combat proteins' will be attached to the silk via Sortase A, a transpeptidase isolated from <EM>S. aureus</EM>. This constitutes a broadly-based approach which will hopefully forestall any bacterial escape or resistance mechanisms.</P><FIGURE><FIGCAPTION>Figure 2: SMITe targets several key components of biofilms with spider-silk-attached combat proteins.</FIGCAPTION></FIGURE><P>However, our mission is far from being limited to chronic wound treatment – products similar to our wound dressing could find applications far beyond the medical field.</P><H2 id="combatproteins">Combat Proteins</H2><H4 id="ednaandnuclease">eDNA and Nuclease</H4><BLOCKQUOTE><H6 id="keyachievements">Key Achievements</H6><P>Assembly and expression </P><P>Demonstrating dispersal of <EM>P. aeruginosa</EM> and <EM>S. aureus</EM> biofilm</P></BLOCKQUOTE><P>Extracellular DNA (eDNA) is released by bacteria through autolysis (self-degradation) and serves as a matrix component of biofilm. eDNA is involved in various biofilm processes including adhesion, aggregation of bacterial cells and as a nutrient source. Enzymatic degradation of eDNA can weaken the biofilm and make the bacteria more susceptible to antibiotics and biocides <A class="ref" href="#eDNA1">[10]</A>. Numerous studies have shown that biofilm-degradation can be achieved by a variety of DNases, enzymes that cleave single-stranded and double-stranded DNA <A class="ref" href="#eDNA2">[11]</A>. In our project we aimed to degrade biofilm formed by <EM>S. aureus</EM> which has previously been shown to be susceptible to DNase treatment <A class="ref" href="#eDNA3">[12]</A>. The DNase we used in the scope of this project was a nuclease (Nuc).</P><FIGURE><FIGCAPTION>Figure 3: Mechanism of Nuc</FIGCAPTION></FIGURE><P>In addition to eDNA, nuclease may also cleave neutrophil extracellular traps (NETs) which  are secreted by white blood cells in wounds as a response to bacterial infection <A class="ref" href="#eDNA1">[10]</A>. Thus nucleases may interact negatively with the body’s own immune system and it would be of great advantage to create a nuclease that targets bacterial, but not human DNA. Eukaryotic and prokaryotic DNA typically exhibit different methylation pattern and some studies have taken advantage of this difference to specifically target bacterial DNA using methylation-dependent nucleases <A class="ref" href="#eDNA4">[13]</A>.</P><P>To target biofilm, we ordered a nuclease BioBrick <A href="http://parts.igem.org/Part:BBa_K729004">BBa<EM>_</EM>K729004</A>. Together with a T7 promoter <A href="http://parts.igem.org/Part:BBa_I719005">BBa_I719005</A>, we aimed to express Nuc, characterise its ability to disperse and inhibit biofilms formed by different bacteria and find out whether it displayed any kind of specificity for procaryotic or eucaryotic DNA. Whereas no clear inhibition of biofilm formation was shown, indications for successful dispersal were found.</P><H4 id="exopolysaccharidesandlysostaphin">Exopolysaccharides and Lysostaphin</H4><BLOCKQUOTE><H6 id="keyachievements">Key Achievements</H6><P>Assembly and expression</P><P>Testing of bactericidal activity on <EM>E. coli</EM></P><P>Fulfilling a silver requirement: Demonstrating that lysostpahin works as expected by demonstrating its bacteriolytic activity and its ability to inhibit and disperse <EM>S. aureus</EM> biofilms</P></BLOCKQUOTE><P>After initial formation, when the biofilm enters the maturation phase of development, intercellular aggregation initiates the formation of the 3D biofilm structure, characterized by cell ‘towers’ equipped with fluid-filled channels. In staphylococci, these structures are predominantly supported by an intricate network of beta-1,6-linked N-acetylglucosamine (PNAG) and other polymers including teichoic acids. These exopolymers are partially de-acetylated which generates a positive charge on the network. Since cell surfaces are naturally negative in charge, the electrostatic interaction between the cells and the exopolymers reinforces the build-up of the biofilm structure <A class="ref" href="#Lys1">[14]</A>.</P><P>A strategy to disrupt the formation of biofilm would be to eradicate the production of <EM>S. aureus</EM> mediated PNAG exopolymers. Hence, to perform this particular action we chose lysostaphin, a glycyl-glycine endopeptidase, as one of our combat proteins. </P><FIGURE><FIGCAPTION>Figure 4: Cell wall structure of Gram positive bacterium</FIGCAPTION></FIGURE><P>As a Gram positive bacterium, <EM>Staphylococcus aureus</EM> is encapsulated by a thick layer of peptidoglycan. Lysostaphin works by degrading this peptidoglycan layer which in turn results in cell lysis and death <A class="ref" href="#Lys2">[15]</A>. According to Sabala et al <A class="ref" href="#Lys3">[16]</A>, lysostaphin achieves this by cleaving the pentaglycine bridge cross-linking the parallel stems of the peptidoglycan polymers.</P><FIGURE><FIGCAPTION>Figure 5: Cleavage site of lysostaphin. The enzyme cleaves at the pentaglycine bridge  <A class="ref" href="#Lys3">[16]</A></FIGCAPTION>.  
</FIGURE><P>In order to manipulate and express lysostaphin in this project, we used an existing BioBrick, <A href="http://parts.igem.org/Part:BBa_K748002">BBa_K748002</A>. This truncated and mature form of lysostaphin was first developed by Team HIT-Harbin in 2012. With its specific-target mechanism as well as its relatively small size, lysostaphin can potentially be conjugated to spider silk as a combat protein to treat <EM>S. aureus</EM> infected wounds. </P><P>During the course of our project, we have achieved several of our goals with lysostaphin; after demonstrating expression, we started investigations into its solubility and bacteriolytic action and most importantly were able to show that lysostaphin strongly and specifically inhibited and degraded <EM>S. aureus</EM> biofilm.</P><H4 id="proteinsandesp">Proteins and Esp</H4><BLOCKQUOTE><H6 id="keyachievements">Key Achievements</H6><P>Assembly and expression </P><P>Investigation into bactericidal activity on <EM>E. coli</EM></P><P>Confirming Esp's ability to disperse biofilms</P><P>Fulfilling a gold requirement: Improving the characterisation of a previous team's <A href="http://parts.igem.org/Part:BBa_K531003">Esp BioBrick</A> in a new expression system and expand on their finding regarding biofilm dispersal and inhibition.</P></BLOCKQUOTE><P>Proteins in the biofilm matrix have various functions and play an important role in all stages of the biofilm development. Cell wall anchored (CWA) proteins are vital in <EM>Staphylococcus aureus</EM> biofilms to mediate the primary attachment of cells to the surface of its host and also promote the intercellular adhesion, biofilm accumulation and maturation <A class="ref" href="#Esp1">[17]</A>.</P><P><EM>Staphylococcus epidermidis</EM> is a commensal species with <EM>S. aureus</EM>, and those two species competes with one another for nutrients. Epidemiology studies show that <EM>S. epidermidis</EM> inhibit the growth of <EM>S. aureus</EM> in the human nasal cavity and further studies indicated that this was due to the degradation of a variety of <EM>S. aureus</EM> CWA proteins. This degradation resulted in decreased biofilm formation and forced <EM>S. aureus</EM> back into a planktonic form, presumably resulting in decreased survival chances. <A class="ref" href="#Esp2">[18]</A><A class="ref" href="#Esp3">[19]</A> Heavily involved in this process was extracellular serine protease (Esp); secreted by <EM>S. epidermidis</EM>, this protease degrades the proteins that maintain structural and chemical integrity of the biofilm. Purified Esp inhibits biofilm formation at the preliminary stage and is able to detach a pre-existing one. Additionally, Esp reduces the virulence of <EM>S. aureus</EM> through degrading other important proteins, e.g. quorum sensing molecule receptors, enzymatic proteins for biofilm dispersal. <A class="ref" href="#Esp4">[20]</A></P><FIGURE><FIGCAPTION>Figure 6: Esp cleaves off CWA proteins </FIGCAPTION></FIGURE><P>Esp could be applicable for the prevention or treatment of infectious diseases caused by <EM>S. aureus</EM>.  It has also been proven to be effective for the biofilms formed by MRSA and vancomycin-intermediate <EM>S. aureus</EM>. <A class="ref" href="#Esp3">[19]</A> We used the Biobricks <A href="http://parts.igem.org/Part:BBa_K531003">BBa_K531003</A> and <A href="http://parts.igem.org/Part:BBa_K531006">BBa_K531006</A>, which were developed by team Grinnell 2011.</P><P>Esp is produced as an inactive form of protein which is then secreted into the environment through Sec-mediated translocation. The propeptide at the N-terminal would be cleaved by signal peptidases during or right after the protein secretion. When expressed recombinantly, the protein therefore requires enzymatic activation - an issue we chose to address by working to create a truncated, mature version of the protein. <A class="ref" href="#Esp5">[21]</A> A successfully expressed Esp could be a good choice to tackle <EM>S. aureus</EM> biofilm through multiple pathways.</P><P>We aimed to improve the characterization of the Esp BioBrick by (1) testing expression in a new chassis, <EM>E. coli</EM>, (2) validating the findings of the creators indicating that Esp was able to disperse <EM>S. aureus</EM> biofilm and expand on this by testing other bacterial biofilms. To our knowledge, we were also the first iGEM team to characterize this BioBrick’s ability to (3) inhibit biofilm formation and (4) test a potential bacteriolytic activity.</P><P>We found that expression in <EM>E. coli</EM> was not very strong and Esp failed to show bacteriolytic activity or clear inhibition of biofilm formation by <EM>S. aureus</EM> and <EM>P. aeruginosa</EM> in our setup. However, Esp was able to significantly detach established biofilms of both species. You can read more about our findings on the <A href="https://2016.igem.org/Team:Stockholm/Results#esp">results page</A> on the <A href="http://parts.igem.org/Part:BBa_K531003">registry</A>.</P><H4 id="cellmembranesanddefensin">Cell membranes and Defensin</H4><BLOCKQUOTE><H6 id="keyachievements">Key Achievements</H6><P>Assembly and expression</P><P>Testing activity on <EM>S. aureus</EM> and <EM>P. aeruginosa</EM> biofilm</P><P>Addition of LT</P><P>Expression and purification using IMAC</P><P>Conjugation to Spider Silk</P></BLOCKQUOTE><P>Defensin is an antimicrobial peptide (AMP) that contributes to the innate immune response and is present in both vertebrates and invertebrates. It is highly abundant in tissues involved in the host defense system and its amphipathic character allows it to penetrate and disrupt the integrity of pathogenic microbial membranes for destruction. Honey-bee (<EM>Apis mellifera</EM>) derived defensin-1 was shown to successfully reduce the viability of biofilm-encapsulated wound pathogens, including <EM>P. aeruginosa</EM> and our main target, <EM>S. aureus</EM><A class="ref" href="#Def1">[22]</A>.</P><P>The function of defensin is to lyse the pathogenic cell by permeabilizing the plasma membrane. The surface of defensin has both hydrophobic and hydrophilic regions. Upon binding to the microbial membrane, defensin causes the formation of small channels, disrupting the integrity and increasing permeability of the membrane. This results in efflux (outward movement from cell) of vital components in the cell, thus killing it <A class="ref" href="#Def2">[23]</A>.</P><FIGURE><FIGCAPTION> Figure 7: Defensin interferes with the integrity of bacterial cell membranes.</FIGCAPTION></FIGURE><P>As our primary target in chronic wounds is the infectious bacteria <EM>S. aureus</EM> and considering that defensin has successfully been shown to have a negative effect on membranes of pathogenic organisms, <A class="ref" href="#Def3">[24]</A> defensin was considered a great combat protein candidate. The well-studied, honeybee-derived defensin-1 part <A href="http://parts.igem.org/Part:BBa_K1104301">BBa_K1104301</A>, from the Bee.coli project designed by 2013 NYMU iGEM Team-Taipei, was ordered for usage in SMITe and expressed. A successfully produced and functional defensin would optimize the disintegration of several bacterial components of the chronic wound model by killing <EM>S. aureus</EM>, amongst other pathogens in the biofilm structure. </P><P>Defensin proved to be one of our most successful proteins in the lab - we were able to demonstrate expression of defensin as well as a strong biofilm-inhibiting activity. Additionally, we assembled a defensin construct which allowed us to purify and finally conjugate the protein to spider silk - a big step for the development of SMITe.</P><H2 id="sortasea">Sortase A</H2><H4 id="theenzyme">The Enzyme</H4><P>When designing the assembly of the essential parts SMITE constitutes of, i.e. the silk and the combat proteins, we were aiming to create an easily-applicable system for attaching proteins to the silk. The main concern was to be able to attach proteins to the silk without altering their main structure, in fear of disturbing the integrity of the protein's structure and its functionality.</P><P>Additionally, the spider silk used in our project was produced by Spiber Technologies using a trademarked recombinant protein which meant we were not able to use a classical genetic engineering approach to create fusion proteins due to confidentiality. When looking for options to circumvent this issue, we came across Sortase A, a bacterial transpeptidase commonly expressed by Gram positive bacteria. It has the ability to break and form new peptide bonds through a nucleophilic attack and is used by <EM>Staphylococcus aureus</EM> to anchor virulence factors to the cell wall <A class="ref" href="#sort1">[25]</A>.</P><P>The key feature of the enzyme is the specific conjugation reaction it carries out, where the enzyme recognizes a specific amino acid sequence, a so called sorting motif (LPXTG motif in the case of <EM>S. aureus</EM>) <A class="ref" href="#sort1">[25]</A><A class="ref" href="#sort2">[26]</A>.</P><FIGURE><FIGCAPTION> Figure 8: Sortase A mediated conjugation </FIGCAPTION></FIGURE><P>In our project, we chose to isolate the enzyme from <EM>S. aureus</EM> and make it into a standard BioBrick with an added GB1 sequence to improve solubility <A class="ref" href="#sort2">[26]</A>.</P><H4 id="oursortaseamediatedimmobilizationstrategy">Our Sortase-A-mediated Immobilization Strategy</H4><BLOCKQUOTE><H6 id="keyachievements">Key Achievements</H6><P>Isolation of Sortase A from <EM>S. aureus</EM> genome</P><P>Assembly with solubility tag GB1</P><P>Successful conjugation of peptides to spider silk using commercial enzyme</P></BLOCKQUOTE><P>We aimed to use the specific interaction of Sortase with the sorting motif to conjugate the “combat proteins” to the spider silk. A general strategy for all combat proteins was designed, in which a specific linker-tag sequence (LT) including the sorting motif and a His-tag for purification purposes, are fused with the proteins (see Figure 8), resulting in a finalized gene construct. Each protein when expressed will be equipped with the LT, enabling Sortase mediated conjugation/immobilization of the proteins to the spider silk naturally occurring C-terminal glycine.</P><P>During our iGEM project, we successfully isolated Sortase A from the <EM>S. aureus</EM> genome and employed two cloning strategies to achieve the fusion with GB1. Unfortunately, the problematic assembly meant that we were unable to assemble the full expression construct or conduct functional testing.</P><H4 id="thesortasepackage">The Sortase Package</H4><P>Sortase has been identified as an enzyme with a broad range of applications that can be extrapolated to the usage of the enzyme in future iGEM project serving as a potential standard BioBrick used within the iGEM Community.</P><P>We thus aimed to generalize our Sortase strategy further, to develop a system in which future iGEM teams can easily utilize Sortase to conjugate/immobilize any type of fusion partners of interest. For more information please see the <A href="/part-collection">Part Collection</A> page. </P><H2 id="references">References</H2><P>1: Sen, C.K., Gordillo, G.M., Roy, S. et al. Human Skin Wounds: A Major and Snowballing Threat to Public Health and the Economy. Wound Repair Regen. 17(6):763-771 (2009) 2: Gjodsbol, K., Christensen, J.J., Karlsmark, T. et al. Multiple bacterial species reside in chronic wounds: a longitudinal study. Int Wound J. 3(3):225-231 (2006) 3: Atiyeh, B. S., Dibo, S. A. &amp;     Hayek, S. N. Wound cleansing, topical antiseptics and wound healing.     Int. Wound J. 6, 420–30 (2009) 4: Zhao, G. et al. Biofilms     and Inflammation in Chronic Wounds. Adv. wound care 2,     389–399 (2013). 5: O’Toole, G. A. &amp; Stewart, P.     S. Biofilms strike back. Nat. Biotechnol. 23,     1378–1379 (2005). 6: Spiliopoulou, A. I. et al. Bacterial adhesion, intracellular survival and cytokine induction upon stimulation of mononuclear cells with planktonic or biofilm phase Staphylococcus epidermidis. FEMS Microbiol. Lett. 330,     56–65 (2012), 7: Stewart, P. S. Mechanisms of     antibiotic resistance in bacterial biofilms. Int. J. Med. Microbiol. 292, 107–13 (2002) 8: Flemming, H.-C. &amp; Wingender, J. The biofilm matrix. Nat. Rev. Microbiol. 8, 623–33 (2010) 9: Meinel, L. et al. The inflammatory responses to silk films in vitro and in vivo. Biomaterials 26, 147–55 (2005). 10: Okshevsky M, Regina VR, Meyer RL. Extracellular DNA as a target for biofilm control. Curr Opin Biotechnol. 2015 Jun;33:73-80. 11: Chen WJ1, Liao TH. Structure and function of bovine pancreatic deoxyribonuclease I. Protein Pept Lett. 2006;13(5):447-53. 12: Izano EA1, Amarante MA, Kher WB, Kaplan JB. Differential roles of poly-N-acetylglucosamine surface polysaccharide and extracellular DNA in Staphylococcus aureus and Staphylococcus epidermidis biofilms. Appl Environ Microbiol. 2008 Jan;74(2):470-6. 13: Siwek W1, Czapinska H, Bochtler M, Bujnicki JM, Skowronek K. Crystal structure and mechanism of action of the N6-methyladenine-dependent type IIM restrictionendonuclease R.DpnI. Nucleic Acids Res. 2012 Aug;40(15):7563-72. 14: Otto, M. Staphylococcal Biofilms. Curr Top Microbiol Immunol. 322, 207-228 (2008) 15: Kumar, J. K. Lysostaphin: an antistaphylococcal agent. Appl Microbiol Biotechnol. 80, 555-561 (2008) 16: Sabala, I., Jagielska, E., Bardelang, P. T., Czapinska, H., Dahms, S. O., Sharpe, J. A., James, R., Than, M. E., Thomas, N. R. &amp; M. Bochtler. Crystal structure of antimicrobial peptidase lysostaphin from Staphylococcus simulans FEBS Journal. 281, 4112-4122 (2014). 17: Pietro Speziale, Giampiero Pietrocola, Timothy J. Foster and Joan A. Geoghegan, Protein-based biofilm matrices in Staphylococci. Front Cell Infect Microbiol. 2014; 4: 171. 18: Iwase T, Uehara Y, Shinji H, Tajima A, Seo H, Takada K, Agata T, Mizunoe Y. 2010. Staphylococcus epidermidis Esp inhibits Staphylococcus aureus biofilm formation and nasal colonization. Nature 465:346 –349. 19: Sugimoto S, Iwamoto T, Takada K, Okuda K, Tajima A, Iwase T, Mizunoe Y. Staphylococcus epidermidis Esp degrades specific proteins associated with Staphylococcus aureus biofilm formation and host-pathogen interaction. J Bacteriol. 2013 Apr;195(8):1645-55. 20: Chen Chen, Vengadesan Krishnan, Kevin Macon, Kartik Manne. Secreted Proteases Control Autolysin-mediated Biofilm Growth of Staphylococcus aureus. J Biol Chem. 2013 Oct 11; 288(41): 29440–29452. 21: S. Sugimoto, T. Iwase, F. Sato, A. Tajima, H. Shinji and Y. Mizunoe. Cloning, expression and purification of extracellular serine protease Esp, a biofilm-degrading enzyme, from Staphylococcus epidermidis. J Appl Microbiol. 2011 Dec;111(6):1406-15. 22: Sojka M, Valachova I, Bucekova M, Majtan J. Antibiofilm efficacy of honey and bee-derived defensin-1 on multi-species wound biofilm. J Med Microbiol.  April 2016 65: 337-344 23: Tomas Ganz. Defensins: antimicrobial peptides of innate immunity. Nature Reviews Immunology 3, 710-720 (September 2003) 24: M Shimoda, K Ohki, Y Shimamoto and O Kohashi. Morphology of defensin-treated Staphylococcus aureus. Infect Immun. 1995 Aug; 63(8): 2886–2891. 25: Popp, M. W.-L. and Ploegh, H. L. (2011), Making and Breaking Peptide Bonds: Protein Engineering Using Sortase. Angew. Chem. Int. Ed., 50: 5024–5032 26: Westerlund, K. Karlstrom, A., Honarvar, H., Tolmachev, V. Design, Preparation, and Characterization of PNA-Based Hybridization Probes for Affibody-Molecule-Mediated Pretargeting. Bioconjugate Chem., 2015, 26 (8), pp 1724–1736</P></SECTION></ARTICLE></MAIN></DIV><FOOTER class="site-footer clearfix"><SECTION class="sponsors">
    iGEM Stockholm is sponsored by
    </SECTION></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>