<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Stockholm_Combat-proteins skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Stockholm/Combat-proteins</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="post-template nav-closed"><DIV class="navigation"><DIV class="nav-content"><DIV class="nav-items"><DIV class="nav-project" role="presentation"><A href="/Team:Stockholm/Description">Project</A><DIV class="nav-sublevel"><A href="/Team:Stockholm/Description#inshort">In Short</A><A href="/Team:Stockholm/Description#ourpath">Our Path</A><A href="/Team:Stockholm/Description#biofilm">Background</A><A href="/Team:Stockholm/Description#ourproject">Our Project</A><A href="/Team:Stockholm/Description#combatproteins">Combat Proteins</A><A href="/Team:Stockholm/Description#sortasea">Sortase A</A><A href="/Team:Stockholm/Future">Future</A><A href="/Team:Stockholm/Medals">Medal achievements</A></DIV></DIV><DIV class="nav-parts" role="presentation"><A href="/Team:Stockholm/Parts">Parts</A><DIV class="nav-sublevel"><A href="/Team:Stockholm/Parts">All parts</A><A href="/Team:Stockholm/Basic_Parts">Basic Parts</A><A href="/Team:Stockholm/Part_Collection">Part Collection</A></DIV></DIV><DIV class="nav-wet-lab" role="presentation"><A href="/Team:Stockholm/Experiments">Lab</A><DIV class="nav-sublevel"><A href="/Team:Stockholm/Experiments">Experiments</A><A href="/Team:Stockholm/Labbook">Lab book</A><A href="/Team:Stockholm/Protocols">Protocols</A></DIV></DIV><DIV class="nav-results" role="presentation"><A href="/Team:Stockholm/Proof">Results</A><DIV class="nav-sublevel"><A href="/Team:Stockholm/Proof">Proof of concept</A><A href="/Team:Stockholm/Results-nuc">Nuc</A><A href="/Team:Stockholm/Results-lys">Lys</A><A href="/Team:Stockholm/Results-def">Def</A><A href="/Team:Stockholm/Results-esp">Esp</A><A href="/Team:Stockholm/Results-sortase-a">Sort A</A></DIV></DIV><DIV class="nav-team" role="presentation"><A href="/Team:Stockholm/Team">Team</A><DIV class="nav-sublevel"><A href="/Team:Stockholm/Team">Members</A><A href="/Team:Stockholm/Collaborations">Collaborations</A><A href="/Team:Stockholm/Attributions">Attributions</A><A href="/Team:Stockholm/Sponsors">Sponsors</A><A href="/Team:Stockholm/Diary">Diary</A></DIV></DIV><DIV class="nav-practises" role="presentation"><A href="/Team:Stockholm/HP/Silver">Human Practices</A><DIV class="nav-sublevel"><A href="/Team:Stockholm/HP/Silver">CARD</A><A href="/Team:Stockholm/HP/Gold">TECH</A><A href="/Team:Stockholm/Integrated_practices">Integrated</A><A href="/Team:Stockholm/Engagement">Engagement</A></DIV></DIV><DIV class="nav-safety" role="presentation"><A href="/Team:Stockholm/Safety">Safety</A></DIV><DIV class="nav-nic" role="presentation"><A href="/Team:Stockholm/NiC">NiC</A></DIV></DIV></DIV></DIV></DIV><DIV class="site-wrapper post-template"><MAIN class="content"><ARTICLE class="post"><HEADER class="post-header"><H1 class="post-title">SMITE - The Concept: Combat Proteins</H1></HEADER><SECTION class="post-content"><H5 id="ednaandnuclease">eDNA and Nuclease</H5><BLOCKQUOTE><P>Key Achievements</P><UL><LI>Assembly and expression </LI><LI>Testing of nuc on isolated DNA</LI><LI>Assessment of nuc activity on <EM>S.aureus</EM> biofilm</LI></UL></BLOCKQUOTE><P>Extracellular DNA (eDNA) is released by bacteria through autolysis (self-degradation) and serves as a matrix component of biofilm. eDNA is involved in various biofilm processes including adhesion, aggregation of bacterial cells and as a nutrient source. Enzymatic degradation of eDNA can weaken the biofilm and make the bacteria more susceptible to antibiotics and biocides. <A class="ref" href="#eDNA1">[10]</A> Numerous studies have shown that biofilm-degradation can be achieved by a variety of DNases, enzymes that cleave single-stranded and double-stranded DNA. <A class="ref" href="#eDNA2">[11]</A> In our project we aimed to degrade biofilm formed by <EM>S. aureus</EM> which has previously been shown to be susceptible to DNase treatment. <A class="ref" href="#eDNA3">[12]</A> The DNase we used in the scope of this project was a nuclease (nuc).</P><P>
Figure 1: Mechanism of Nuc</P><P>In addition to eDNA, nuclease may also cleave neutrophil extracellular traps (NETs) which  are secreted by white blood cells in wounds as a response to bacterial infection. <A class="ref" href="#eDNA1">[10]</A> Thus nucleases may interact negatively with the body’s own immune system and it would be of great advantage to create a nuclease that targets bacterial, but not human DNA. Eucaryotic and procaryotic DNA typically exhibit different methylation pattern and some studies have taken advantage of this difference to specifically target bacerial DNA using methylation-dependent nucleases. <A class="ref" href="#eDNA4">[13]</A>
The nuc used in our project is no methlyation specific , to limit degradation of NETs and other bodily functions and the plan is to avoid the wound edges when applying the nuclease to wounds. </P><P>To target biofilm, we ordered a nuclease BioBrick (BBa<EM>K729004). Together with a T7 promoter (BBa</EM>I719005), a previous iGEM team demonstrated nuclease’s capability to digest DNA. </P><P>Additionally to testing nuc’s ability to disperse biofilm we planned to evaluate whether it displayed any specificity for procaryotic or eucaryotic DNA with a simple gel electrophoresis of nuc treated DNA. </P><H4 id="exopolysaccharidesandlysostaphin">Exopolysaccharides and Lysostaphin</H4><BLOCKQUOTE><P>Key Achievements</P><UL><LI>Assembly and expression</LI><LI>Testing of bactericidal activity on <EM>E.coli</EM> and <EM>S.aureus</EM></LI><LI>Testing of biofilm-dispersing activity</LI></UL></BLOCKQUOTE><P>After initial formation, when the biofilm enters the maturation phase of development, intercellular aggregation initiates the formation of the 3D biofilm structure, characterized by cell ‘towers’ equipped with fluid-filled channels. 
In staphylococci, these structures are predominantly supported by an intricate network of beta-1,6-linked N-acetylglucosamine (PNAG) and other polymers including teichoic acids. These exopolymers are partially de-acetylated which generates a positive charge on the network. Since cell surfaces are naturally negative in charge, the electrostatic interaction between the cells and the exopolymers reinforces the build-up of the biofilm structure. <A class="ref" href="#Lys1">[14]</A></P><P>A strategy to disrupt the formation of biofilm would be to eradicate the production of <EM>S. aureus</EM> mediated PNAG exopolymers. Hence, to perform this particular action we chose lysostaphin, a glycyl-glycine endopeptidase, as one of our combat proteins. </P><P>As a Gram positive bacterium, <EM>Staphylococcus aureus</EM> is encapsulated by a thick layer of peptidoglycan. Lysostaphin works by degrading this peptidoglycan layer which in turn results in cell lysis and death. <A class="ref" href="#Lys2">[15]</A> According to Sabala et al <A class="ref" href="#Lys3">[16]</A>, lysostaphin achieves this by cleaving the pentaglycine bridge cross-linking the parallel stems of the peptidoglycan polymers.</P><P>
Figure 2: Cleavage site of lysostaphin, modified from Sabala et al <A class="ref" href="#Lys3">[16]</A></P><P>In order to manipulate and express lysostaphin in this project, we used an existing Biobrick, BBa_K748002. This truncated and mature form of lysostaphin was first developed by Team HIT-Harbin in 2012. With its specific-target mechanism as well as its relatively small size, lysostaphin can potentially be conjugated to spider silk as a combat protein to treat <EM>S. aureus</EM> infected wounds. </P><H4 id="proteinsandesp">Proteins and Esp</H4><BLOCKQUOTE><P>Key Achievements</P><UL><LI>Assembly and expression </LI><LI>Investigation into bactericidal activity on <EM>E.coli</EM></LI></UL></BLOCKQUOTE><P>Proteins in the biofilm matrix have various functions and play an important role in all stages of the biofilm development. Cell wall anchored (CWA) proteins are vital in <EM>Staphylococcus aureus</EM> biofilms to mediate the primary attachment of cells to the surface of its host and also promote the intercellular adhesion, biofilm accumulation and maturation <A class="ref" href="#Esp1">[17]</A><EM>Staphylococcus epidermidis</EM> is a commensal species with <EM>S. aureus</EM>, and those two species competes with one another for nutrients. Epidemiology studies show that <EM>S. epidermidis</EM> inhibit the growth of <EM>S. aureus</EM> in the human nasal cavity and further studies indicated that this was due to the degradation of a variety of <EM>S.aureus</EM> CWA proteins. This degradation resulted in decreased biofilm formation and forced <EM>S.aureus</EM> back into a planktonyic form, presumably resulting in decreased survival chances. <A class="ref" href="#Esp2">[18]</A><A class="ref" href="#Esp3">[19]</A>
Heavily involved in this process was extracellular serine protease (esp); secreted by <EM>S. epidermidis</EM>, this protease degrades the proteins that maintain structural and chemical integrity of the biofilm. Purified esp inhibits biofilm formation at the preliminary stage and is able to detach a pre-existing one. Additionally, esp reduces the virulence of <EM>S. aureus</EM> through degrading other important proteins, e.g. quorum sensing molecule receptors, enzymatic proteins for biofilm dispersal. <A class="ref" href="#Esp4">[20]</A></P><P>
Figure 3: Esp cleaves off CWA proteins </P><P>Esp could be applicable for the prevention or treatment of infectious diseases caused by <EM>S. aureus</EM>.  It has also been proven to be effective for the biofilms formed by MRSA and vancomycin -intermediate S. aureus. <A class="ref" href="#Esp3">[19]</A>
We used the Biobricks BBa<EM>K531003 and BBa</EM>K531006, which were developed by team Grinnell 2011, as the basis for synthesising esp in E. coli. 
It is a wild type gene with 833 bp in size, amplified from the genome of <EM>S. epidermidis</EM>. 
Esp is produced as an inactive form of protein which is then secreted into the environment through Sec-mediated translocation. The propeptide at the N-terminal would be cleaved by signal peptidases during or right after the protein secretion. When expressed recombinantly, the protein therefore requires enzymatic activation - an issue we chose to tackle by working to create a truncated, mature version of the protein. <A class="ref" href="#Esp5">[21]</A> A successfully expressed esp could be a good choice to tackle <EM>S. aureus</EM> biofilm through multiple pathways.</P><H6 id="defensin">Defensin</H6><BLOCKQUOTE><P>Key Achievements</P><UL><LI>Assembly </LI></UL></BLOCKQUOTE><P>Defensin is an antimicrobial peptide (AMP) that contributes to the innate immune response and is present in both vertebrates and invertebrates. It is highly abundant in tissues involved in the host defence system and its amphipathic character allows it to penetrate and disrupt the integrity of pathogenic microbial membranes for destruction. Honey-bee derived defensin-1 was shown to successfully reduce the viability of biofilm-encapsulated wound pathogens, including <EM>P.aeruginosa</EM> and our main target, <EM>S.aureus</EM>. <A class="ref" href="#Def1">[22]</A>
The function of defensin is to lyse the pathogenic cell by permeabilizing the plasma membrane. The surface of defensin has both hydrophobic and hydrophilic regions. Upon binding to the microbial membrane, defensin causes the formation of small channels, disrupting the integrity and increasing permeability of the membrane. This results in efflux (outward movement from cell) of vital components in the cell, thus killing it. <A class="ref" href="#Def2">[23]</A></P><P>
Figure 4: Defensin interferes with the integrity of bacterial cell membranes</P><P>As our primary target in chronic wounds is the infectious bacteria <EM>S.aureus</EM> and considering that Defensin has successfully been shown to have a negative effect on the pathogenic membrane <A class="ref" href="#Def3">[24]</A>, Defensin was considered a great combat protein candidate. The well-studied, honeybee-derived defensin1 part BBa_K1104301, from the Bee.coli project designed by 2013 NYMU iGEM Team-Taipei, was ordered for usage in SMITe and expressed. We planned to purify defensin and test its functionality on <EM>S.aureus</EM> biofilm both before and after it's conjugation to spider silk. A successfully produced and functional defensin would optimize the disintegration of several bacterial components of the chronic wound model by killing <EM>S. aureus</EM>, amongst other pathogens in the biofilm structure. 
This could potentially be a natural antimicrobial treatment as an alternative to extensive use of antibiotics to reduce the bacterial infections. </P><H4 id="references">References</H4></SECTION></ARTICLE></MAIN></DIV><FOOTER class="site-footer clearfix"><SECTION class="sponsors">
    iGEM Stockholm is sponsored by
    </SECTION></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>