<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Stockholm_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Stockholm/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="post-template nav-closed"><DIV class="navigation"><DIV class="nav-content"><DIV class="nav-items"><DIV class="nav-project" role="presentation"><A href="/Team:Stockholm/Description">Project</A><DIV class="nav-sublevel"><A href="/Team:Stockholm/Description#inshort">In Short</A><A href="/Team:Stockholm/Description#ourpath">Our Path</A><A href="/Team:Stockholm/Description#biofilm">Background</A><A href="/Team:Stockholm/Description#ourproject">Our Project</A><A href="/Team:Stockholm/Description#combatproteins">Combat Proteins</A><A href="/Team:Stockholm/Description#sortasea">Sortase A</A><A href="/Team:Stockholm/Future">Future</A><A href="/Team:Stockholm/Medals">Medal achievements</A></DIV></DIV><DIV class="nav-parts" role="presentation"><A href="/Team:Stockholm/Parts">Parts</A><DIV class="nav-sublevel"><A href="/Team:Stockholm/Parts">All parts</A><A href="/Team:Stockholm/Basic_Parts">Basic Parts</A><A href="/Team:Stockholm/Part_Collection">Part Collection</A></DIV></DIV><DIV class="nav-wet-lab" role="presentation"><A href="/Team:Stockholm/Experiments">Lab</A><DIV class="nav-sublevel"><A href="/Team:Stockholm/Experiments">Experiments</A><A href="/Team:Stockholm/Labbook">Lab book</A><A href="/Team:Stockholm/Protocols">Protocols</A></DIV></DIV><DIV class="nav-results" role="presentation"><A href="/Team:Stockholm/Proof">Results</A><DIV class="nav-sublevel"><A href="/Team:Stockholm/Proof">Proof of concept</A><A href="/Team:Stockholm/Results-nuc">Nuc</A><A href="/Team:Stockholm/Results-lys">Lys</A><A href="/Team:Stockholm/Results-def">Def</A><A href="/Team:Stockholm/Results-esp">Esp</A><A href="/Team:Stockholm/Results-sortase-a">Sort A</A></DIV></DIV><DIV class="nav-team" role="presentation"><A href="/Team:Stockholm/Team">Team</A><DIV class="nav-sublevel"><A href="/Team:Stockholm/Team">Members</A><A href="/Team:Stockholm/Collaborations">Collaborations</A><A href="/Team:Stockholm/Attributions">Attributions</A><A href="/Team:Stockholm/Sponsors">Sponsors</A><A href="/Team:Stockholm/Diary">Diary</A></DIV></DIV><DIV class="nav-practises" role="presentation"><A href="/Team:Stockholm/HP/Silver">Human Practices</A><DIV class="nav-sublevel"><A href="/Team:Stockholm/HP/Silver">CARD</A><A href="/Team:Stockholm/HP/Gold">TECH</A><A href="/Team:Stockholm/Integrated_practices">Integrated</A><A href="/Team:Stockholm/Engagement">Engagement</A></DIV></DIV><DIV class="nav-safety" role="presentation"><A href="/Team:Stockholm/Safety">Safety</A></DIV><DIV class="nav-nic" role="presentation"><A href="/Team:Stockholm/NiC">NiC</A></DIV></DIV></DIV></DIV></DIV><DIV class="site-wrapper post-template"><MAIN class="content"><ARTICLE class="post"><HEADER class="post-header"><H1 class="post-title">Experiments</H1></HEADER><SECTION class="post-content"><P>When it comes down to it, SMITe consists of three main components: a spider silk scaffold provided by our sponsors, biofilm-degrading combat proteins and the Sortase A enzyme used to conjugate the one to the other. Our main goals were the expression of functional Sortase A and combat proteins as well as their conjugation to the silk scaffold. Below, you can find an overview of the paths we chose - you can also have a look at our <A href="https://2016.igem.org/Team:Stockholm/Protocols">protocols</A>, <A href="https://2016.igem.org/Team:Stockholm/Proof">results</A> or <A href="https://2016.igem.org/Team:Stockholm/Labbook">lab books</A> if you’re interested in more details.</P><H2 id="combatproteins">Combat Proteins</H2><P>All of our biofilm-degrading combat proteins were assembled in a similar fashion, allowing for the addition of a linker tag sequence (LT) that would include sequences for both purification of the proteins via Immobilized Metal Affinity Chromatography (IMAC) and their conjugation to the spider silk using Sortase A. </P><H5 id="assemblingourconstructs">Assembling our Constructs</H5><P>Assembly was conducted in two separate steps. First, we used the classic 3A method to assemble our Combat Proteins - defensin, lysostaphin, Nuc and esp - with a T7 promoter. Subsequently, we tested two methods to add the linker tag sequence that was needed for purification and conjugation to the spider silk.</P><FIGURE><FIGCAPTION>Figure 1: Assembly of combat protein expression constructs</FIGCAPTION></FIGURE><P>The custom designed LT sequence consists of three elements: a GSSSS linker, a Hisx6 tag for IMAC purification and LPETGG, the C-terminal sequence motif recognized by Sortase A.</P><FIGURE><FIGCAPTION>Figure 2: Structure of the linker and tags (LT)</FIGCAPTION></FIGURE><P>Both options rely on a PCR reaction to add a BamHI restriction site to the N-terminus of the combat proteins so that BamHI can be used to ligate the T7-Combat Protein to the LT oligonucleotide. A modified 3A Assembly was used to ligate the T7-Combat Protein fragment with the LT and the pSB1C3 backbone. </P><H5 id="testingcombatproteins">Testing Combat proteins</H5><P>Once our combat proteins were assembled with the promoters, we needed to test whether they were expressed correctly and decided to use a simple SDS-PAGE for that purpose. Subsequently, we tested the combat proteins' activity, purified them and attached them to spider silk.</P><FIGURE><FIGCAPTION>Figure 3: Expression, purification and testing of combat proteins</FIGCAPTION></FIGURE><P>To test the functionality of the combat proteins, we decided to work primarily with biofilm assays using <EM>S. aureus</EM> and <EM>P. aeruginosa</EM> since they provide a quantifiable, clinically relevant readout. Simply put, the bacteria are incubated in conditions allowing them to form a biofilm and then the biofilm-degrading proteins are added either straight away (providing data regarding the inhibition of biofilm formation) or after biofilm had been established (measuring the ability to disperse biofilm). At different timepoints, the biofilm was stained, liquified and the absorption measured in a spectrophotometer.</P><P>Apart from the biofilm assay, we also used modified Kirby-Bauer tests where filter papers soaked in cell lysates were placed onto agar plates with bacterial cultures streaked on them. If the enzymes within the cell lysates were bacteriolytic or bacteriostatic, they inhibited the colony formation in their environment, leading to halo formation around the filter papers.</P><H5 id="conjugatingcombatproteinstospidersilk">Conjugating Combat Proteins to Spider Silk</H5><P>Once the combat proteins were assembled with an appropriate LT sequence and purified, the conjugation was going to be fairly straightforward. Sortase A is able to catalyze the formation of a peptide bond in between two glycines - one found on the LPETGG sequence of the combat proteins' tag and the other one naturally present on the spider silk.</P><P>Once the wound dressing components are fully assembled, more biofilm tests could be performed, alongside different stainings to visualize how the components of the biofilm are broken down in response to treatment.</P><H2 id="sortasea">Sortase A</H2><P>Sortase A is our matchmaker enzyme and is used to conjugate combat proteins to the spider silk. Its sequence was easily isolated from <EM>S. aureus</EM> genome and subsequently, two modifications from the wild type Sortase A were planned to improve solubility:</P><UL><LI>Truncation of the enzymes so that the transmembrane domain (amino acids 1-59) would not be included in our BioBrick</LI><LI>N-terminal addition of GB1, a Protein G domain well known to increase solubility of recombinant proteins</LI></UL><P>Additionally, an endogenous XbaI site was aimed to be removed via site-directed mutagenesis in order to comply with BioBrick standards.</P><H5 id="assemblingtheconstruct">Assembling the Construct</H5><P>Sortase A was isolated from <EM>S. aureus</EM> genomic DNA using a simple PCR reaction. In order to add the GB1 domain, two methods were used in the lab - a Gibson assembly and an Overlap Extension PCR. Both of those methods require homologous sequences on the fragments that will be ligated. Those overlaps were added to Sortase A with another PCR reaction, which also allowed the introduction of a C-terminal Histidine tag for IMAC purification of Sortase A.</P><FIGURE><FIGCAPTION>Figure 4: Assembly of Sortase A construct</FIGCAPTION></FIGURE><P>Gibson Assembly relies on the activity of a 5’ exonuclease which creates 3’ overhangs on the complementary sequences, leading to annealing of the two fragments. 
Overlap Extension PCR on the other hand requires no exonuclease and instead utilizes a slightly modified PCR reaction. Two different fragments (in our case GB1 and Sortase A) are added into a PCR reaction and will be denatured. In the following step, the DNA strands will be allowed to anneal again and due to the overlapping sequences, a part of the GB1 strands will anneal to Sortase A strand. By carefully selecting the primer combination for this reaction, those recombinant DNA molecules will preferably be amplified and can be used for downstream applications. </P><H5 id="functionaltestingsortasea">Functional Testing Sortase A</H5><P>After successful assembly with Gibson assembly or overlap extension PCR, we planned to add a T7 promoter to the GB1-SortA hybrid for subsequent expression, purification and functional resting.  </P><FIGURE><FIGCAPTION>Figure 5: Expression, purification and functional testing of combat proteins  
</FIGCAPTION></FIGURE><P>To test the activity of Sortase A, simple methods can be applied: attaching two peptides with the correct tags to each other and performing an SDS-Page to observe the shift in molecular weight. Additionally, fluorescent proteins such as RFP or UnaG can be attached and visualized using a fluorescence microscope to get a better idea of the distribution and density of attached proteins. A final test would of course be the assembly of our finished product, SMITe. </P><H2 id="references">References</H2><P>‘Firework’ icon by Giuditta Valentina Gentile, arrow icons by Molly Bramlet and Eugen Belyakoff from The Noun Project. </P></SECTION></ARTICLE></MAIN></DIV><FOOTER class="site-footer clearfix"><SECTION class="sponsors">
    iGEM Stockholm is sponsored by
    </SECTION></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>