<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Stockholm_Results-esp skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Stockholm/Results-esp</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="post-template nav-closed"><DIV class="navigation"><DIV class="nav-content"><DIV class="nav-items"><DIV class="nav-project" role="presentation"><A href="/Team:Stockholm/Description">Project</A><DIV class="nav-sublevel"><A href="/Team:Stockholm/Description#inshort">In Short</A><A href="/Team:Stockholm/Description#ourpath">Our Path</A><A href="/Team:Stockholm/Description#biofilm">Background</A><A href="/Team:Stockholm/Description#ourproject">Our Project</A><A href="/Team:Stockholm/Description#combatproteins">Combat Proteins</A><A href="/Team:Stockholm/Description#sortasea">Sortase A</A><A href="/Team:Stockholm/Future">Future</A><A href="/Team:Stockholm/Medals">Medal achievements</A></DIV></DIV><DIV class="nav-parts" role="presentation"><A href="/Team:Stockholm/Parts">Parts</A><DIV class="nav-sublevel"><A href="/Team:Stockholm/Parts">All parts</A><A href="/Team:Stockholm/Basic_Parts">Basic Parts</A><A href="/Team:Stockholm/Part_Collection">Part Collection</A></DIV></DIV><DIV class="nav-wet-lab" role="presentation"><A href="/Team:Stockholm/Experiments">Lab</A><DIV class="nav-sublevel"><A href="/Team:Stockholm/Experiments">Experiments</A><A href="/Team:Stockholm/Labbook">Lab book</A><A href="/Team:Stockholm/Protocols">Protocols</A></DIV></DIV><DIV class="nav-results" role="presentation"><A href="/Team:Stockholm/Proof">Results</A><DIV class="nav-sublevel"><A href="/Team:Stockholm/Proof">Proof of concept</A><A href="/Team:Stockholm/Results-nuc">Nuc</A><A href="/Team:Stockholm/Results-lys">Lys</A><A href="/Team:Stockholm/Results-def">Def</A><A href="/Team:Stockholm/Results-esp">Esp</A><A href="/Team:Stockholm/Results-sortase-a">Sort A</A></DIV></DIV><DIV class="nav-team" role="presentation"><A href="/Team:Stockholm/Team">Team</A><DIV class="nav-sublevel"><A href="/Team:Stockholm/Team">Members</A><A href="/Team:Stockholm/Collaborations">Collaborations</A><A href="/Team:Stockholm/Attributions">Attributions</A><A href="/Team:Stockholm/Sponsors">Sponsors</A><A href="/Team:Stockholm/Diary">Diary</A></DIV></DIV><DIV class="nav-practises" role="presentation"><A href="/Team:Stockholm/HP/Silver">Human Practices</A><DIV class="nav-sublevel"><A href="/Team:Stockholm/HP/Silver">CARD</A><A href="/Team:Stockholm/HP/Gold">TECH</A><A href="/Team:Stockholm/Integrated_practices">Integrated</A><A href="/Team:Stockholm/Engagement">Engagement</A></DIV></DIV><DIV class="nav-safety" role="presentation"><A href="/Team:Stockholm/Safety">Safety</A></DIV><DIV class="nav-nic" role="presentation"><A href="/Team:Stockholm/NiC">NiC</A></DIV></DIV></DIV></DIV></DIV><DIV class="site-wrapper post-template"><MAIN class="content"><ARTICLE class="post"><HEADER class="post-header"><H1 class="post-title">Esp</H1></HEADER><SECTION class="post-content"><H2 id="summary">Summary</H2><P>EB (Esp as extracted from the <EM>S. epidermidids</EM> genome) is one of our combat proteins targeting biofilm formation and prevents eDNA exudation. By expressing and attaching it to our spider silk, in combination with the other combat proteins, we hope to combat both biofilm and virulence in chronic wounds.  </P><BLOCKQUOTE><P>Assembling a construct for recombinant expression of EB [✔️]</P><P>Demonstrating expression of EB in <EM>E. coli</EM> [❓] </P><P>Adding Sortase and His tags to the expression construct [❌] </P><P>Investigating possible bactericidal activity [❓]</P><P>Assemble truncated EB [❌]</P><P>Demonstrating efficient Biofilm inhibition by EB [❓]</P><P>Demonstrating efficient Biofilm dispersal by EB [✔️]</P></BLOCKQUOTE><H2 id="results">Results</H2><H4 id="assemblingaconstructforrecombinantexpressionofeb">Assembling a construct for recombinant expression of EB</H4><P>By using 3A Assembly, the construct to produce recombinant Esp was assembled. The components used for this experiment were the T7-promoter (BBa<EM>_</EM>K525998), the coding sequence for EB (BBa_K531003) and a backbone, in this case pSB1K3. To obtain the final ligation product a classic 3A assembly was performed. </P><H6 id="digestion">Digestion [✔️]</H6><P>In this experiment double digestion was performed on the BioBricks T7 and Esp to prepare these samples for ligation. Also single digestions were performed to investigate the restriction enzymes functionality.</P><P>Both single and double digestions were considered successful except for the empty wells with T7 PCR products and an unspecific band, in well 9, for the T7 plasmid. However, it is hard to evaluate the results for the double digestion of the T7 plasmid since the expected size of 55 bp is too small to be visualised on a common gel. Since the longer fragments that this digestion generates were clearly visible, double digestion of the T7 plasmid was assumed to be successful. The next step was planned to ligate the digested fragments together into an appropriate backbone.</P><FIGURE><FIGCAPTION>Figure 1. Gel electrophoresis of digested samples.</FIGCAPTION></FIGURE><TABLE class="tg"><CAPTION>Table 1: Samples shown in Figure 1.</CAPTION><TBODY><TR><TH class="tg-9hbo">Well</TH><TH class="tg-9hbo">Sample</TH></TR><TR><TD class="tg-yw4l">1</TD><TD class="tg-yw4l">DNA Ladder</TD></TR><TR><TD class="tg-yw4l">2 &amp; 3</TD><TD class="tg-yw4l">EB Plasmid sd EcorI</TD></TR><TR><TD class="tg-yw4l">4 &amp; 5</TD><TD class="tg-yw4l">EB Plasmid sd SpeI</TD></TR><TR><TD class="tg-yw4l">6 &amp; 7</TD><TD class="tg-yw4l">EB Plasmid dd</TD></TR><TR><TD class="tg-yw4l">8 &amp; 9</TD><TD class="tg-yw4l">T7 Plasmid sd EcorI</TD></TR><TR><TD class="tg-yw4l">10 &amp; 11</TD><TD class="tg-yw4l">T7 Plasmid sd SpeI</TD></TR><TR><TD class="tg-yw4l">12 &amp; 13</TD><TD class="tg-yw4l">T7 Plasmid dd</TD></TR><TR><TD class="tg-yw4l">14 &amp; 15</TD><TD class="tg-yw4l">T7 PCR Product sd EcorI</TD></TR><TR><TD class="tg-yw4l">16 &amp; 17</TD><TD class="tg-yw4l">T7 PCR Product sd SpeI</TD></TR><TR><TD class="tg-yw4l">18 &amp; 19</TD><TD class="tg-yw4l">T7 PCR product dd</TD></TR></TBODY></TABLE><H6 id="ligation">Ligation [✔️]</H6><P>At this point we worked with two versions of Esp referred as EB and EC (Esp optimised for <EM>C. crescentus</EM>, BBa_K531006) </P><P><FIGURE><FIGCAPTION>Figure 2. Gel of ligation products using non-ligated samples as reference. Wells containing: (2) pSB1K3-T7-EB, (3) pSB1K3-T7-EC, (4) double digested EB, (5) double digested EC.</FIGCAPTION></FIGURE></P><P>After the ligation had been performed, a gel electrophoresis was run to analyse the results. However, the results were not satisfying since too little product was present as could be seen in figure 2. Therefore, we were not sure if the ligation had been successful. However, the transformation was performed despite having too little DNA to see it on the gel.</P><H6 id="colonypcrofligationproducts">Colony PCR of Ligation Products [✔️]</H6><P>The  planned transformation mentioned in Result 2 was successful, 1 colony was observed for the assembled plasmid containing T7 and EC (pSB1K3-T7-EC) and two colonies were observed for the plasmid with T7 and EB (pSB1K3-T7-EB) after 18 h incubation.</P><FIGURE><FIGCAPTION>Figure 3: Colony PCR of T7-EB-pSB1K3 and T7-EC-pSB1K3. (2) pSB1K3-T7-EB Colony 1, (3) pSB1K3-T7-EB Colony 2, (4) pSB1K3-T7-EC.</FIGCAPTION></FIGURE><P>The 3A assembly of pSB1K3-T7-EB was considered a success since the colony PCR appeared successful and the sequencing results confirmed a correct assembly. pSB1K3-T7-EC  however, was not correctly assembled, as indicated by the sequencing results and the work with it was not continued.</P><P>Generally, a very low number of colonies was observed when transforming ligation products whereas the positive control worked fine. This likely indicates that the ligation product itself is not of a very high quality or quantity. For any following experiments, both digestion and ligation should be analysed carefully to make sure that high-quality ligation product can be transformed. Additionally, to optimize the ligation a couple of different insert:vector ratios could have been tested.</P><H4 id="demonstratingexpressionofebinecoli">Demonstrating expression of EB in <EM>E. coli</EM></H4><P>In order to express recombinant Esp, BL21(DE3) cells containing the construct pSB1K3-T7-EB were induced with IPTG, triggering the protein production. The cells were lysed by sonication, the samples were spun down and the pellet was resuspended in PBS. Thereafter SDS-PAGE was run to confirm the expression of the protein of interest. Supernatant and the resuspended pellet were run separately on the SDS-PAGE since the solubility of Esp is unknown to us and an insoluble protein might require re-folding in order to be functional. In following results the samples from the supernatant is referred as “soluble fraction” and the samples from the pellet as “insoluble fraction”. </P><H6 id="sdspagefirstattempt">SDS-PAGE, First Attempt [❓]</H6><FIGURE><FIGCAPTION>Figure 4. Results of 12 % SDS-PAGE.</FIGCAPTION></FIGURE><TABLE class="tg"><CAPTION>Table 2: Samples displayed in Figure 3.</CAPTION><TBODY><TR><TH class="tg-9hbo">Well</TH><TH class="tg-9hbo">Sample</TH></TR><TR><TD class="tg-yw4l">1</TD><TD class="tg-yw4l">Protein Ladder</TD></TR><TR><TD class="tg-yw4l">2</TD><TD class="tg-yw4l">Soluble fraction EB1, 1 mM IPTG induction</TD></TR><TR><TD class="tg-yw4l">3</TD><TD class="tg-yw4l">Soluble fraction EB2, 1 mM IPTG induction</TD></TR><TR><TD class="tg-yw4l">4</TD><TD class="tg-yw4l">Soluble fraction EB2, uninduced</TD></TR><TR><TD class="tg-yw4l">5</TD><TD class="tg-yw4l">Protein Ladder</TD></TR><TR><TD class="tg-yw4l">6</TD><TD class="tg-yw4l">Insoluble Fraction EB1, 1 mM IPTG induction</TD></TR><TR><TD class="tg-yw4l">7</TD><TD class="tg-yw4l">Insoluble Fraction EB1, uninduced</TD></TR><TR><TD class="tg-yw4l">8</TD><TD class="tg-yw4l">Insoluble Fraction EB2, 1 mM IPTG induction</TD></TR><TR><TD class="tg-yw4l">9</TD><TD class="tg-yw4l">Insoluble Fraction EB2, uninduced</TD></TR></TBODY></TABLE><P>An intense band at around 40 kDa exists in all the protein samples. Insoluble protein at around 29 kDa showed a significant increase after IPTG induction.</P><P>The expression of pSB1K3-T7-EB seems to be successful. EB is calculated to be 30 kDa and we observed an intense band around 30 kDa after IPTG induction. However, Western blot might be needed to further confirm the result as Coomassie Blue stains total protein and recombinant proteins can be hard to identify among the multitudes of endogenous proteins. Besides, the protein ladder used in this SDS page does not have a band at 30 kDa and the observed band might not be exactly at 30 kDa. </P><H6 id="sdspagesecondattempt">SDS-PAGE, Second Attempt [❓]</H6><P>In this experiment two batches of SDS-PAGE were done. In the second SDS-page, we added the untransformed BL21 cells to see whether there is a difference between the transformed and non-transformed cells since it was hard to interpret the results in last SDS-page without such a control. This is especially important since the T7 expression system can be leaky and recombinant protein may be expressed even in uninduced cells, further complicating the interpretation.</P><FIGURE><FIGCAPTION>Figure 5. Results of 12 % SDS-PAGE.</FIGCAPTION></FIGURE><TABLE class="tg"><CAPTION>Table 3: Samples displayed in Figure 5.</CAPTION><TBODY><TR><TH class="tg-9hbo">Well</TH><TH class="tg-9hbo">Sample</TH></TR><TR><TD class="tg-yw4l">1</TD><TD class="tg-yw4l">Protein Ladder</TD></TR><TR><TD class="tg-yw4l">2</TD><TD class="tg-yw4l">Soluble fraction, untransformed cells</TD></TR><TR><TD class="tg-yw4l">3</TD><TD class="tg-yw4l">Soluble fraction, uninduced</TD></TR><TR><TD class="tg-yw4l">4</TD><TD class="tg-yw4l">Soluble fraction, induced at 37 °C, 4h</TD></TR><TR><TD class="tg-yw4l">5</TD><TD class="tg-yw4l">Soluble fractino, induced at 25 °C overnight</TD></TR><TR><TD class="tg-yw4l">6</TD><TD class="tg-yw4l">Insoluble Fraction, untransformed cells</TD></TR><TR><TD class="tg-yw4l">7</TD><TD class="tg-yw4l">Insoluble Fraction, uninduced</TD></TR><TR><TD class="tg-yw4l">8</TD><TD class="tg-yw4l">Insoluble Fraction, induced at 37 °C, 4h</TD></TR><TR><TD class="tg-yw4l">9</TD><TD class="tg-yw4l">Insoluble Fraction EB2, induced at 25 °C overnight</TD></TR></TBODY></TABLE><FIGURE><FIGCAPTION>Figure 6. Results of 12 % SDS-PAGE.</FIGCAPTION></FIGURE><TABLE class="tg"><CAPTION>Table 4: Samples displayed in Figure 6.</CAPTION><TBODY><TR><TH class="tg-9hbo">Well</TH><TH class="tg-9hbo">Sample</TH></TR><TR><TD class="tg-yw4l">1</TD><TD class="tg-yw4l">Protein Ladder</TD></TR><TR><TD class="tg-yw4l">2</TD><TD class="tg-yw4l">Soluble fraction, untransformed cells</TD></TR><TR><TD class="tg-yw4l">3</TD><TD class="tg-yw4l">Soluble fraction, uninduced</TD></TR><TR><TD class="tg-yw4l">4</TD><TD class="tg-yw4l">Soluble fraction, induced at 37 °C, 4h</TD></TR><TR><TD class="tg-yw4l">5</TD><TD class="tg-yw4l">Soluble fractino, induced at 25 °C overnight</TD></TR><TR><TD class="tg-yw4l">6</TD><TD class="tg-yw4l">Protein Ladder</TD></TR><TR><TD class="tg-yw4l">7</TD><TD class="tg-yw4l">Insoluble Fraction, untransformed cells</TD></TR><TR><TD class="tg-yw4l">8</TD><TD class="tg-yw4l">Insoluble Fraction, uninduced</TD></TR><TR><TD class="tg-yw4l">9</TD><TD class="tg-yw4l">Insoluble Fraction, induced at 37 °C, 4h</TD></TR><TR><TD class="tg-yw4l">10</TD><TD class="tg-yw4l">Insoluble Fraction EB2, induced at 25 °C overnight</TD></TR></TBODY></TABLE><P>The expression of pSB1K3-T7-EB seems to be successful for the first batch but not for the second one. EB is calculated to be 30 kDa and we observed an intense band around 30 kDa after IPTG induction. </P><P>As mentioned above, we added the untransformed BL21 cells to see whether there is a difference in expression between the transformed and non-transformed cells. The result turns out the bands at 40 kDa and 35 kDa are universally expressed in all samples. </P><P>Since the last experiment shows an universal protein expression at 35-40 kDa no conclusion can be drawn whether we have successfully expressed Esp. Furthermore, we have seen protein expression in this range both in the soluble and the insoluble samples, therefore we have not succeed in determining the solubility of Esp. If we had more time we could have used mass spectrometry for more accurate results. Furthermore, if we had succeed to insert the His-tag in the gene encoding for Esp before this experiment was performed we could have purified our samples before running the SDS-PAGE’s or performed a Western Blot with anti-His-tag antibodies. That could have given us easier results to interpret with no surrounding noise due to the impurities. </P><P>Furthermore, during our project we have prepared our own buffers and gels. If we would have had the possibility to use standardized material for the SDS-PAGE maybe we could in a greater extent avoid bad separation and smeary gels.</P><H4 id="addingsortaseandhistagstotheexpressionconstruct">Adding Sortase and His tags to the expression construct</H4><P>In order to purify and attach Esp to spider silk, the addition of linkers and tags (LT) was necessary, which was going to be achieved by a modified 3A assembly using a BamHI restriction site. This site first has to be introduced to EB using PCR. </P><H6 id="pcraddingbamhioverhangfirstattempt">PCR adding BamHI overhang, First Attempt [❌]</H6><P>In the first trial adding BamHI overhang to EB using PCR, the setup reaction was similar as regular PCR with a reverse primer designed to add a BamHI site. It had the same annealing temperature as the pre-existing forward primer (VF2). Positive control was performed with VF2 and VR, which proved to be functioning in our previous experiments. We tested two annealing temperature with 52 °C and 57 °C.</P><P>As shown in figure, in both of the gels, there were intensive bands for positive control. However, the overhang PCR seemed to have been unsuccessful with PCR products that were either unspecific (at 52 °C annealing temperature) or simply not present (57 °C). The failure could due to low compatibility of the new primer or the conditions for overhang PCR not being optimal. For the next trial, various component in the setup were optimised.</P><FIGURE><FIGCAPTION>Figure 7. Overhang PCR with 52 °C annealing temperature. (1) DNA ladder (7) Overhang EB1 (8) Overhang EB2 (9) Positive control (10) EB plasmid (11) Negative control.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 8. Overhang PCR with 57 °C degree annealing temperature.  (1) DNA ladder (6) Overhang EB1 (7) Overhang EB2 (8) Positive control (9) Negative control.</FIGCAPTION></FIGURE><H6 id="pcraddingbamhioverhangsecondtofourthattempt">PCR adding BamHI Overhang, Second to Fourth Attempt [❌]</H6><P>In the following three attempts, we switched to Taq polymerase, increased amount of DNA templates and primers to optimise the PCR. However, the modifications to the protocol were unsuccessful. The bands were either faint or unspecific. We examined all the reagents and found out that we had mixed up the reverse primer.</P><H6 id="pcraddingbamhioverhangfifthattempt">PCR adding BamHI overhang, Fifth Attempt [✔️]</H6><P>In the new trial with correct primers, clear bands were observed at correct size in the gel. As shown in the picture, the well 2, 3 and 4 were EB with 1, 10 and 20 ng of DNA template. The bands occurred at a size slightly bigger than 1000 bp, which was corresponding to EB with 1021 bp. After performing this successful overhang PCR, we were able to continue with digestion and ligation of the product.</P><FIGURE><FIGCAPTION>Figure 9. Overhang PCR of pSB1K3-T7-EB with correct primers.  (1) Overhang EB1 (2) Overhang EB2 (3) Overhang EB3 (4) Overhang EB4 (5) Negative control (6) DNA ladder.  
</FIGCAPTION></FIGURE><H6 id="digestionandligationfirstattempt">Digestion and Ligation, First Attempt [❌]</H6><P>We performed the gel purification with the QIAquick gel extraction kit. After gel purification, the samples had concentrations of 12 and 15 ng/ml, which was adequate for the following digestion and ligation.</P><P>BamHI restriction enzyme was used to digest the purified PCR product under the standard protocol with 60 min 37 °C incubation and 20 min of heat kill. T4 ligase was used to ligate digested EB, LT and the pSB1C3 backbone. 
Subsequently a gel for both the digestion and ligation product was run. Unfortunately, no bands for the ligation and digestion product appeared in the gel, indicating unsuccessful reactions. Perhaps the gel purification might have diluted the digestion product or the conditions for digestion and ligation might not be optimal. </P><H6 id="digestionandligationsecondattempt">Digestion and Ligation, Second Attempt [❌]</H6><P>We had another three attempts for digestion and ligation. Various component and time for incubation and heat kill were optimized. None of them succeeded. If the digestions were successful or not was hard to judge as the digested DNA sequence had similar size as the undigested one. We observed faint bands at over 3000 bp which might correspond to the assembled plasmid. Then we moved on with transformation the new construct into both TOP10 cells and BL21(DE3) cells. Unfortunately, the only colony observed was shown to not contain EB. It is unclear whether the fault lies with low efficacy digestion and ligation or with the transformation process. </P><FIGURE><FIGCAPTION>Figure 10. Result from gel electrophoresis in 1 % agarose. (1) DNA ladder (2) EB1 ligation product (3) EB2 ligation product (4) double digestion of T7-EB-BamHI 1 (5) double digestion of T7-EB-BamHI 2.</FIGCAPTION></FIGURE><H6 id="overhangpcrwithnewstrategy">Overhang PCR with New Strategy [❌]</H6><P>After several failure in digestion and ligation, we decided to use a new strategy: amplifying the whole pSB1C3-T7-EB construct with the addition of a BamHI restriction sequence after the EB coding sequence. The PCR was performed with Phusion High Fidelity Polymerase. Due to time limitation, we only tried three annealing temperatures. Unfortunately, none of the PCR products showed bands in the gel.     </P><FIGURE><FIGCAPTION>Figure 11: Overhang PCR with new strategy. (1) DNA ladder (2) Overhang EB1 (3) Overhang EB2 (4) Negative control.</FIGCAPTION></FIGURE><P>In conclusion, we managed to add BamHI overhang to EB after several failures. However, the digestion and ligation were unsuccessful. The digested product was hard to tell from the gel because the undigested sample had similar size as its. In all four attempts for ligation and transformation, only one colony developed in the third attempt, while it didn’t have the correct sequence.</P><P>A new strategy was tried to add BamHI to the whole pSB1C3-T7-EB construct. However, due to limited time, we didn’t find the optimal conditions and didn’t get any amplified product. For troubleshooting, positive control should be used for the digestion reaction to exclude a systematic error.</P><H4 id="investigatingpossiblebacteriolyticactivity">Investigating possible bacteriolytic activity</H4><P>Esp serves a variety of functions but many of its possible activities are undocumented, which was the reason we wanted to investigate whether it would in fact also be bacteriolytic.</P><P>We chose to perform a modified Kirby-Bauer test with TOB1 <EM>E.coli</EM> for which filter papers soaked with samples were placed on an agar plate to assess whether they inhibit bacterial growth. </P><H6 id="kirbybauerfirstattempt">Kirby-Bauer, First Attempt [❓]</H6><P>In a first attempt at performing a Kirby-Bauer, salt-free agar plates were used as an optimal growth medium for <EM>E.coli</EM> TOB1. Autoclaved cellulose nitrate filter papers were soaked in cell lysates, where the soluble and insoluble fractions were separated in order to also assess solubility of our protein. The plates were incubated overnight and halo formation was assessed.</P><P>Unfortunately, this proved to be rather difficult since the agar and the colonies were very similar in colour. Nevertheless, the positive control (Kanamycin at 35 μg/ml) did display an inhibition of bacterial growth, indicating that our set-up seemed to work. We could, however, not see such a halo for our EB samples samples which might simply be because EB not being bacteriolytic / bacteriostatic or might also be caused freezing our protein in lysis buffer and repeatedly thawing it.</P><P>For a repeated experiment, we planned to use freshly produced protein and selective agar which might help making halos visible.</P><H6 id="kirbybauersecondattempt">Kirby-Bauer, Second Attempt [❓]</H6><P>In this experiment, two different EB solutions were used to assess whether freezing of the samples had a deleterious effect on EB activity. After incubation, especially on Endo Agar the bacterial colonies were quite easily visible and those plates were used to assess halo formation.</P><FIGURE><FIGCAPTION>Figure 12. Kirby-Bauer of (a)'old' (meaning frozen) EB and (b) 'new' (meaning never frozen) EB. Green asterisk marks positive control, black asterisks any sample exhibiting halo formation and red asterisk the corresponding negative control.</FIGCAPTION></FIGURE><P>The samples with considerable halo formation in this case were:</P><UL><LI>EB ‘old’, 1mM IPTG induction, soluble fraction</LI><LI>EB ‘old’, 0.5mM IPTG induction, soluble fraction</LI><LI>EB ‘new’, 1mM IPTG induction, insoluble fraction</LI></UL><P>Unfortunately, those results don’t appear very consistent and even negative controls (cell lysate from untransformed BL21(DE3) cells) showed halo formation in some cases, especially on the plate with the ‘new’ EB. This could  indicate that the mere fact that the disks were soaked in a liquid could inhibit bacterial growth. In a subsequent experiment where triplicates were performed and less liquid was used, unfortunately no colonies formed, likely because the inoculum used to streak the plates had been too diluted.</P><P>After those three experiments, no clear conclusion can be drawn about whether EB has a bacteriolytic activity on <EM>E.coli.</EM> Since its main effects were documented to concern the biofilm and virulence factors, EB might in fact not be bacteriolytic or at least not exhibit that effect on gram negative bacteria.</P><P>For replication of the experiments, EB should be cleaved beforehand in order to remove the pro-peptide, which is reported to greatly improve its efficacy (Sugimoto 2011) This cleavage was not performed in our experiments and would likely be needed to test EB efficiently. Alternatively, Esp could perhaps be assembled in a mature form, as we attempted by truncating the sequence (see below).</P><H4 id="assembletruncatedeb">Assemble truncated EB</H4><P>Esp is initially transcribed as a pre-pro-peptide and subsequently cleaved into an active, mature form. (Sugimoto 2011) The uncleaved pre-pro-peptide has shown little to no activity in biofilm-degradation and is usually treated with thermolysin before functional assays (Sugimoto 2011, Ohara-Nemoto 2002) In order to avoid the additional step of thermolysin-mediated cleavage, which can also represent an additional confounding factor, we decided to create a truncated version of Esp (trEB) which would be missing the first 66 amino acids.</P><P>The set up for this experiment was considered to be fairly simple: we constructed a primer that was going to anneal at the valine in the 67th position and add the BioBrick prefix to its N-terminal. That primer was then used in combination with the EB_reverse primer to allow for subsequent ligation of the LT sequence to the C-terminus to assemble a protein that could be purified and conjugated to the spider silk.</P><H6 id="overhangpcrdigestionandligationfirstattempt">Overhang PCR, Digestion and Ligation, First Attempt [❓]</H6><P>PCR was performed with standard set-up and with the two primers EB<EM>_</EM>trunc and EB_reverse. This should result in a truncated EB that also has the BamHI site required for adding LT. The PCR appeared successful in the first attempt, with a strong band that approximately corresponds to the expected 673 bp. (See Figure 12a) 
Subsequently, the PCR product was digested and ligated to perform a 3A assembly with the LT sequence and the pSB1C3 backbone. The success of the digestion can not be assessed on the gel since only a few bases are removed. The ligation was performed with a 3:1 inserts:vector ratio and appeared partly successful with bands that could correspond to a circular plasmid visible (see arrows). After transformation, unfortunately no colonies could be observed for TOP10 or BL21(DE3) cells.</P><FIGURE><FIGCAPTION>Figure 13. PCR product and subsequent digestion and ligation product for trEB.

a) (2) trEB, (3) EB, (4) negative control of a PCR performed with trEB primers on Nuc template.

b) (2) digestion product (3) ligation product. Arrows indicate bands in third well that might be correctly assembled plasmid.</FIGCAPTION></FIGURE><P>Since the ligation appeared to be somewhat successful and all ligation product had been used for the transformation, we decided to simply re-do the ligation with the already digested fragments.</P><H6 id="digestionandligationsecondattempt">Digestion and Ligation, Second Attempt [❓]</H6><P>In this second attempt, the inserts:vector ratio was set to 10:1  in the hopes of achieving a more efficient ligation. Additionally, the ligation product was not heat-killed, which could inhibit transformation efficacy.</P><FIGURE><FIGCAPTION>Figure 14. Weak bands corresponding to possible correctly assembled plasmid are visible, indicated by arrows. A stronger band around 600bp likely represents unligated PCR product.</FIGCAPTION></FIGURE><P>As can be seen on the gel, the ligation appeared partly successful but after transformation, no colonies were visible. Even after retransformation of the remaining ligation product, no colonies were visible and the project was abandoned due to time constraints.</P><P>Overall, the attempt to assemble a truncated version of EB was unsuccessful since it suffered from the same pitfalls as many other parts of the project. If there had been more time to focus on this part of our project, another strategy for the assembly could have been attempted, such as a Gibson assembly. Alternatively, the PCR could have been performed with primers amplifying the BioBrick suffix, which would have allowed to simply digest the PCR product and ligate it into a backbone without another fragment, which might have improved ligation efficacy.</P><H4 id="demonstratingefficientbiofilminhibitionbyeb">Demonstrating efficient Biofilm inhibition by EB</H4><P>To demonstrate inhibition by EB, a crystal violet assay was performed with <EM>P. aeruginosa</EM> and <EM>S. aureus</EM> biofilms. Since we did not know whether EB is present in the pellet or the supernatant both fractions were saved and tested in this experiment.</P><P>With those assays it has to be kept in mind that crystal violet also stains bacterial cells so that part of the effect may be due to a bacteriolytic function. Given that EB however did not show a clear halo in Kirby-Bauer test, we believe that the decrease of absorbance int he biofilm assays is in fact due to biofilm inhibition or degradation.</P><H6 id="biofilmassayfirstattempt">Biofilm Assay, First Attempt [❓]</H6><P>In this first attempt of demonstrating biofilm inhibition, the bacteria were incubated for 48 hours with different volumes of cell lysate before their absorbance was measured. Protein obtained on several different days was tested but for simplicity, only part of the data is displayed.</P><P>In this assay, EB did not show strong activity. For <EM>S. aureus</EM>, the results were erratic, including the negative controls (cell lysates of untransformed BL21(DE3) cells) and are not displayed here. The tests performed on <EM>P. aeruginosa</EM>, however, indicate a possible biofilm-inhibitingactivity by the soluble fraction of our cell lysates, with an absorbance consistently below the negative controls. </P><FIGURE><FIGCAPTION>Figure 15. Absorbance at 595nm after incubation of <EM>P. aeruginosa</EM> with various cell lysates. Here shown a possible biofilm-inhibiting activity by the soluble fraction.</FIGCAPTION></FIGURE><P>Based on those results, we were not able to draw any kind of conclusion given that the samples seemed inconsistent and the assay had only been performed using triplicates. Biofilm assays often need to be performed with considerably more replicates since they are hard to standardise and washing steps add the danger of unintentionally detaching the biofilm.</P><P>A next assay was planned in which more replicates would be performed and the outer wells of the plate were going to be left empty since they tend to dry out after longer incubation periods. Additionally, the time course was adapted: 48 hours are usually recommended for biofilm growth but we were uncertain as to whether that might be too long; the biofilm growth could possibly overwhelm the activity of Esp or Esp could simply have lost activity after such a long incubation time.</P><H6 id="biofilmassaysecondattempt">Biofilm Assay, Second Attempt [✔️]</H6><P>When repeating this assay, the samples were now not split anymore in supernatant and pellet, reducing the amount of work. Furthermore, 12 replicates of each sample were performed and the plates were quantified after only 27 hours of incubation.</P><P>Again, the <EM>S. aureus</EM> results were unfortunately not usable since both positive and negative controls showed highly unusual behaviour. The tests on <EM>P. aeruginosa</EM>, however, displayed some measure of inhibition though the effect does not seem to correlate with the amount of cell lysate that was added. However, considering that incubation with 50 μl of cell lysate reduced the absorbance by almost 50 % compared to the negative control, some measure of biofilm-inhibiting activity seems to be present.</P><FIGURE><FIGCAPTION>Figure 16. Absorbance at 595 nm after incubation of <I>P. aeruginosa</I> with cell lysates for 27 hours.</FIGCAPTION></FIGURE><P>After those two assays, no clear conclusion can be drawn but there are indications that EB might work as predicted. The results obtained using <EM>P. aeruginosa</EM> imply that EB might in fact inhibit the formation of biofilms though the variance of the samples can be considerable, as explained above. Due to time constraints, we were unfortunately not able to repeat the assays, even though it would have been very beneficial to test either a truncated or cleaved form of EB which has been shown to have a much stronger biofilm-dispersing activity. For any future project involving EB and its activity on biofilm, more replicates and various time points should also be considered. </P><H4 id="demonstratingefficientbiofilmdispersalbyeb">Demonstrating efficient Biofilm dispersal by EB</H4><P>The approach was very similar to the one outlined above for the biofilm inhibition. The main differences lies in the timing: <EM>P. aeruginosa</EM> and <EM>S. aureus</EM> were incubated for 48 hours allowing them to form a biofilm before the combat proteins were added. All statistical analysis was conducted using Student's T-test.</P><H6 id="biofilmdispersalfirstattempt">Biofilm Dispersal, First Attempt [❌]</H6><P>In this test, we encountered a well-known issue with the negative control - even the cell lysates of untransformed cells (or the lysis buffer itself) seem to be disrupting the biofilm to a certain degree, an effect that correlated strongly with the volume of lysate added. The reduction in absorbance after adding EB cell lysates corresponds to the reduction observed by adding the lysates of untransformed cells, indicating little contribution by EB. For <EM>P. aeruginosa</EM>, however, the biofilm dispersal seemed to occur even for smaller volumes of EB, hinting that EB might be more effective on Pseudomonas rather than Staphylococci. This conclusion, however, can of course not be drawn after a single experiment. </P><FIGURE><FIGCAPTION>Figure 17. Absorbance at 595 nm after incubation of established <EM>S. aureus</EM> biofilm with cell lysates.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 18. Absorbance at 595 nm after incubation of established <EM>P. aeruginosa</EM> biofilm with cell lysates.</FIGCAPTION></FIGURE><P>When repeating the experiment, we were aiming to introduce the same changes mentioned in the “Biofilm Inhibition” section; avoid using the outer wells which tend to try out, test whole cell lysates instead of soluble and insoluble fractions and also reduce the incubation time.</P><H6 id="biofilmdispersalsecondattempt">Biofilm Dispersal, Second Attempt [✔️]</H6><P>After adapting our protocol to reflect the changes mentioned above, we were able to gather clearer data for the dispersal of both <EM>S. aureus</EM> and <EM>P. aeruginosa</EM> biofilms. Both experiments showed very similar results with a clear decrease in absorbance after treatment with the EB samples, correlating well with the volume of sample added. Additionally, the differences were highly significant for both <EM>S. aureus</EM> and <EM>P. aeruginosa</EM> when 50 μl of sample were added. </P><FIGURE><FIGCAPTION>Figure 19. Absorbance at 595 nm after incubation of established <EM>S. aureus</EM> biofilm with cell lysates. A significant difference could be observed between the 50 μl EB samples and the 50μl negative control with a p-value of 5.37225E-08, indicated by the asterisks.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 20. Absorbance at 595 nm after incubation of established <EM>P. aeruginosa</EM> biofilm with cell lysates. A significant difference could be observed between the 50 μl EB samples and the 5 μl negative control with a p-value of 9.7229E-08, indicated by the asterisks.</FIGCAPTION></FIGURE><P>In conclusion, after some optimization, valuable data has been obtained regarding the ability of EB to disperse biofilm.  Whereas the data was not always consistent, the final experiments indicates that EB possesses a strong ability to disperse biofilm, especially considering that crude cell lysates were used to perform the tests. 
For the future, the testing of both activated and purified EB would be an obvious goal, as well as further improving the set-up of the assay in order to obtain more reliable data.        </P></SECTION></ARTICLE></MAIN></DIV><FOOTER class="site-footer clearfix"><SECTION class="sponsors">
    iGEM Stockholm is sponsored by
    </SECTION></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>