<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Stockholm_Results-nuc skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Stockholm/Results-nuc</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="post-template nav-closed"><DIV class="navigation"><DIV class="nav-content"><DIV class="nav-items"><DIV class="nav-project" role="presentation"><A href="/Team:Stockholm/Description">Project</A><DIV class="nav-sublevel"><A href="/Team:Stockholm/Description#inshort">In Short</A><A href="/Team:Stockholm/Description#ourpath">Our Path</A><A href="/Team:Stockholm/Description#biofilm">Background</A><A href="/Team:Stockholm/Description#ourproject">Our Project</A><A href="/Team:Stockholm/Description#combatproteins">Combat Proteins</A><A href="/Team:Stockholm/Description#sortasea">Sortase A</A><A href="/Team:Stockholm/Future">Future</A><A href="/Team:Stockholm/Medals">Medal achievements</A></DIV></DIV><DIV class="nav-parts" role="presentation"><A href="/Team:Stockholm/Parts">Parts</A><DIV class="nav-sublevel"><A href="/Team:Stockholm/Parts">All parts</A><A href="/Team:Stockholm/Basic_Parts">Basic Parts</A><A href="/Team:Stockholm/Part_Collection">Part Collection</A></DIV></DIV><DIV class="nav-wet-lab" role="presentation"><A href="/Team:Stockholm/Experiments">Lab</A><DIV class="nav-sublevel"><A href="/Team:Stockholm/Experiments">Experiments</A><A href="/Team:Stockholm/Labbook">Lab book</A><A href="/Team:Stockholm/Protocols">Protocols</A></DIV></DIV><DIV class="nav-results" role="presentation"><A href="/Team:Stockholm/Proof">Results</A><DIV class="nav-sublevel"><A href="/Team:Stockholm/Proof">Proof of concept</A><A href="/Team:Stockholm/Results-nuc">Nuc</A><A href="/Team:Stockholm/Results-lys">Lys</A><A href="/Team:Stockholm/Results-def">Def</A><A href="/Team:Stockholm/Results-esp">Esp</A><A href="/Team:Stockholm/Results-sortase-a">Sort A</A></DIV></DIV><DIV class="nav-team" role="presentation"><A href="/Team:Stockholm/Team">Team</A><DIV class="nav-sublevel"><A href="/Team:Stockholm/Team">Members</A><A href="/Team:Stockholm/Collaborations">Collaborations</A><A href="/Team:Stockholm/Attributions">Attributions</A><A href="/Team:Stockholm/Sponsors">Sponsors</A><A href="/Team:Stockholm/Diary">Diary</A></DIV></DIV><DIV class="nav-practises" role="presentation"><A href="/Team:Stockholm/HP/Silver">Human Practices</A><DIV class="nav-sublevel"><A href="/Team:Stockholm/HP/Silver">CARD</A><A href="/Team:Stockholm/HP/Gold">TECH</A><A href="/Team:Stockholm/Integrated_practices">Integrated</A><A href="/Team:Stockholm/Engagement">Engagement</A></DIV></DIV><DIV class="nav-safety" role="presentation"><A href="/Team:Stockholm/Safety">Safety</A></DIV><DIV class="nav-nic" role="presentation"><A href="/Team:Stockholm/NiC">NiC</A></DIV></DIV></DIV></DIV></DIV><DIV class="site-wrapper post-template"><MAIN class="content"><ARTICLE class="post"><HEADER class="post-header"><H1 class="post-title">Nuclease</H1></HEADER><SECTION class="post-content"><H2 id="summary">Summary</H2><P>Nuclease (Nuc) is one of our combat proteins which targets the extracellular DNA (eDNA) of biofilms. In our potential wound dressing, we hope that Nuc would be able to disperse biofilm which relies on the negative charge of eDNA to maintain its structural integrity.</P><BLOCKQUOTE><P>Assembling a construct for recombinant expression of Nuc [✔️]</P><P>Demonstrating expression and investigating solubility of Nuc in <EM>E. coli</EM> [✔️]</P><P>Demonstrating efficient biofilm inhibition by Nuc [❌]</P><P>Demonstrating efficient biofilm dispersal by Nuc [✔️]</P><P>Demonstrating bacteriolytic activity of Nuc [❌]</P><P>Adding Sortase and His tags to the expression construct [❓]</P></BLOCKQUOTE><H2 id="results">Results</H2><H4 id="assemblingaconstructforrecombinantexpressionofnuc">Assembling a construct for recombinant expression of Nuc</H4><P>The 3A assembly method is used to assemble a pSB1C3-T7-Nuc construct, where the nuclease BioBrick created by the UCL London iGEM team 2012 <A href="http://parts.igem.org/Part:BBa_K729004">BBa_K729004</A> and the T7-promoter created by the Bielefeld Germany iGEM team 2011 <A href="http://parts.igem.org/Part:BBa_K525998">BBa_K525998</A> are cut out from their original constructs and the backbone cut open, using appropriate restriction enzymes.</P><P>The ligation products are then first transformed into TOP10 for maximum transformation efficiency. The plasmid are extracted and transformed into BL21(DE3) for expression.</P><H6 id="digestionandligationfirsttothirdattempt">Digestion and Ligation, First to Third Attempt [❌]</H6><P>Successful digestion of Nuc and backbone (pSB1K3) in the second attempt. Ligation with T7 from first assembly attempt on the other hand failed. The PCR analysis of the ligation showed contamination of bands corresponding to Esp, indicating a simple mistake in probably marking the tubes.</P><P>The third attempt showed successful digestion of backbone (pSB1K3). However, unsuccessful digestion of Nuc. Thus, ligation could not be performed. Since no obvious errors could be identified in either the first or the second attempt, the procedure was simply repeated without any changes.</P><H6 id="digestionandligationfourthattempt">Digestion and Ligation, Fourth Attempt [✔️]</H6><P>pSB1C3-Nuc was successfully digested and the digested product was then also successfully ligated with previously digested backbone (pSB1K3) and T7. However, the analysis of the results were not convincing. A clear band (see arrow in Figure 1) at the expected size of approximately 800 bp but also a lot of other unspecific bands. The subsequent transformation in TOP10 was unsuccessful since there were no growth of colonies.</P><FIGURE><FIGCAPTION>Figure 1. Expected size: ~800 bp. Result of the gel electrophoresis of PCR amplification product after ligation, with 1 % agarose at 100 V. (1) DNA Ladder (11) DNA Ladder (12) pSB1K3-T7-Nuc  (13) pSB1K3-T7-Nuc (14) pSB1K3-T7-Nuc (15) pSB1K3-T7-Nuc (16) Positive control (17) Negative control (18) DNA Ladder. Unrelated samples on the gel were covered for clarity.</FIGCAPTION></FIGURE><P>The decision was made to re-do the ligation, based on the ligation gel results which showed a lot of unspecific bands. Since the transformations in the previous attempts had a very low success rate compared to the expected results, the ligation product would be transformed directly into BL21(DE3) as well as in TOP10 to investigate whether the type of cell was a contributing factor. This is however not believed to be the primary reason for the unsuccessful transformations since the TOP10 strain takes up plasmids more easily than BL21(DE3) which is an expression strain.</P><H6 id="digestionandligationfifthtoseventhattempt">Digestion and Ligation, Fifth to Seventh Attempt [❓]</H6><P>Nuc and backbone (pSB1K3-T7-Esp as source) was successfully digested but no analysis was performed for the ligation product. Conclusions had to be drawn from further transformation in TOP10 and BL21(DE3) with subsequent inoculation in liquid media. Analysis of extracted plasmid from TOP10 cells showed indications of presence of Esp instead of nuclease, as in the second attempt. PCR analysis of plasmids from BL21(DE3) cells were negative. The contamination of Esp had now occurred for the second time and probably derived from the usage of pSB1K3-T7-Esp as backbone source, due to depletion of pure pSB1K3. The pSB1K3-T7-Esp plasmid was digested with the same restriction enzymes as the linear pSB1K3 backbone from iGEM HQ. The backbone fragment from this was then obtained through gel purification for the use in subsequent ligation. This gel purification may have been imprecise and the cause of the Esp contamination. Therefore, ligation was re-done with another backbone source from iGEM HQ containing only backbone, pSB1A3. </P><P>The subsequent attempt was unsuccessful and for the 7th attempt, no analysis on the ligation product was made. The transformation of the ligation product was unsuccessful since there were no colonies on the plate. Since the transformation protocol were thought to be reliable, the conclusion was drawn that the ligation had failed. The backbone used was again changed to pSB1C3 instead, due to depletion of the pSB1A3 sample from iGEM HQ. Also, the ligation conditions were changed - instead of 10 minutes at room temperature, the samples were incubated at 16 °C overnight, a common protocol for ligations which we hoped would be more successful.</P><H6 id="digestionanligationeighttotenthattempt">Digestion an Ligation, Eight to Tenth Attempt [✔️]</H6><P>Successful digestion of Nuc in both the 8th and the 9th attempt (see Figure 2, 9th attempt). There results from the analysis of the ligation product indicated a successful ligation and after transformations, colonies were formed. However for both attempts, the subsequent plasmid analysis of the transformed cells showed negative results.</P><P>The vector:insert ratio of 1:3 was suspected to be insufficient and a 1:10 ratio was set for the subsequent ligations. A mistake of using the wrong restriction enzymes for the digestion of the backbone resulted in an obvious unsuccessful ligation. The following, 10th, attempt where the same samples were used (except for the backbone, which was now correctly digested), was successful. But the results were not confirmed until a single digestion analysis had been performed (successfully, see Figure 3) since the analysis of the ligation product were inconclusive. The sample in well 5 (see Figure 3) had the expected size and was further used. Presence of the correct construct was further proved after overhang PCR amplification with the Nuc_R reverse primer, designed with a BamHI restriction site overhang (see Figure 4). This part (<A href="http://parts.igem.org/Part:BBa_K2144000">BBa_K2144000</A>) was submitted to the iGEM Part Registry.</P><FIGURE><FIGCAPTION>Figure 2. Expected size: 561. Result of the gel electrophoresis after digestion with 1 % agarose at 100 V. (1) DNA Ladder (2) Nuc 2.1.1 GS (3) Nuc 2.1.2 GS (4) Nuc 1.1 HQ (5) Nuc 1.2 HQ (6) Negative control (7) DNA Ladder. Unrelated samples on the gel were covered for clarity.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 3. Expected size: ~2700. Result of the gel electrophoresis after single digestion of extracted plasmid, with 1 % agarose at 100 V. (1) DNA Ladder (3) Nuc 2.1.2 GS BL21(DE3) 1 (4) Nuc 2.1.2 GS BL21(DE3) 2 (5) Nuc 2.1.2 GS BL21(DE3) 3 (6) Nuc 1.2 HQ TOP10 1 (7) Nuc 1.2 HQ TOP10 2 (8) Nuc 2.1.2 GS TOP10 2 (9) DNA Ladder. Unrelated samples on the gel were covered for clarity.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 4. Expected size: ~700 bp. Result of the gel electrophoresis after overhang PCR with BamHI primer, with 1 % agarose at 100 V. Arrow shows band at expected size of 700 bp. (1) DNA Ladder (2) Nuc 2.1.2 GS BL21(DE3) 3 (Nuc_R) (3) Nuc 2.1.2 GS BL21(DE3) 3 (VF2_R) (4) DNA Ladder. Unrelated samples on the gel were covered for clarity.</FIGCAPTION></FIGURE><H4 id="demonstratingexpressionandinvestigatingsolubilityofnucinecoli">Demonstrating expression and investigating solubility of Nuc in <EM>E. coli</EM></H4><P>The assembled expression construct is transformed into BL21(DE3) for optimal expression conditions. Liquid cultures of transformed cells are then induced with 0.5 mM or 1 mM IPTG to initiate expression of nuclease. A non-induced culture serves as a reference. After lysing the cells through sonication and collecting the cell debris with centrifugation, both the supernatant and the resuspended pellet are analyzed using SDS-PAGE.</P><H6 id="expressionextractionsdspage">Expression, extraction &amp; SDS-PAGE [✔️]</H6><FIGURE><FIGCAPTION>Figure 5. Expected size: 20.7 kDa. Result of the SDS-PAGE run at 200 V, 3.00 A. (1) Protein ladder (2) Nuc 1.0 mM supernatant (3) Nuc 0.5 mM supernatant (4) Nuc 0.5 mM supernatant (5) Nuc 1.0 mM pellet (6) Nuc 0.5 mM pellet (7) Nuc 0 mM pellet.</FIGCAPTION></FIGURE><P>The visible band at the expected size, corresponding to nuclease, in well 2 is not present in the wells (3 and 4) with the samples with lower IPTG induction concentrations. This indicates that the plasmid of interest is functional and that nuclease is expressed when inducing with IPTG. Since the same band is also visible in well 5 (pellet resuspension sample), and increasingly faint in well 6 and 7 as the IPTG concentration decreases, it is difficult to draw any conclusions whether nuclease is soluble or not.</P><H4 id="demonstratingefficientbiofilminhibitionbynuc">Demonstrating efficient biofilm inhibition by Nuc</H4><P>A crystal assay is conducted to evaluate how Nuc inhibits biofilm growth by <EM>P. aeruginosa</EM> and <EM>S. aureus</EM>, respectively. Induced BL21(DE3) cell lysate samples (both debris pellet resuspension and supernatant) are used for the experiment.</P><P><EM>S. aureus</EM> was cultured in BHI media and <EM>P. aeruginosa</EM> PAO in LB media. After an overnight incubation, the cell density is measured at OD 600. Also, 200 µl of the liquid culture is added into each well and treated with Nuc of four different volumes; 1, 5, 10 and 20 µl. The biofilm is allowed to develop for 48 hours with subsequent staining. Absorbance is then measured at 595 nm. Cell lysates of BL21(DE3) cells are used as negative control. Differences between the negative control and Nuc is evaluated using the Student’s t-test. A p-value &lt;0.05 is considered statistically significant.</P><H6 id="biofilmassayfirstattempt">Biofilm assay: First Attempt [❌]</H6><P>The samples from supernatants and pellet are labelled &quot;Nuc S&quot; and &quot;Nuc P&quot;, respectively.</P><FIGURE><FIGCAPTION>Figure 6. Absorbance measured for different treatments of Nuc using <I>P. aeruginosa</I> as biofilm-producer.</FIGCAPTION></FIGURE><P>The graph illustrates how the combination of supernatant and resuspended pellet of Nuc cell lysate inhibits the formation of biofilm by <EM>P. aeruginosa</EM>. As the treatment increases, the absorbance tends to increase as well. There is a significant difference (p=0.02) between the negative control and Nuc S for the 1 µl treatment.</P><FIGURE><FIGCAPTION>Figure 7. Absorbance measured for different treatments of Nuc using <I>S. aureus</I> as biofilm-producer.</FIGCAPTION></FIGURE><P>The graph illustrates how the combination of supernatant and resuspended pellet of Nuc cell lysate inhibits the formation of biofilm by <EM>S. aureus</EM>. The negative control seems inconsistent; it may be that it is dose dependent in <EM>S. aureus</EM> or that it has been contaminated. This experiment showed a significant difference between 1 µl of negative control and Nuc S, as well as between 20 µl of negative control and Nuc P. However, these results may not be reliable.</P><H6 id="biofilmassaysecondattempt">Biofilm assay: Second Attempt [❌]</H6><P>The second attempt of inhibiting biofilm growth was similar to the first. However, this time a combination of pellet and supernatant was used as treatment with the ratio 1:1, e.g. for 5 µl treatment, 2.5 µl of each was used. For this attempt the biofilm was treated with 5 µl, 20 µl and 50 µl of Nuc. The samples were labelled &quot;Nuc&quot; and the negative control was labelled &quot;Neg&quot;. Only one volume of negative control was used.</P><FIGURE><FIGCAPTION>Figure 8. Absorbance measured for different treatments of Nuc using <I>P. aeruginosa</I> as biofilm-producer.</FIGCAPTION></FIGURE><P>The figure illustrates how Nuc (supernatant and pellet in a 1:1 ratio) inhibits biofilm formation by <EM>P aeruginosa</EM>. These results show that absorbance decreases as the volume of treatment increases. There is a significant difference between the negative control and 20 µl and 50 µl of Nuc treatment, respectively.</P><FIGURE><FIGCAPTION>Figure 9. Absorbance measured for different treatments of Nuc using <I>S. aureus</I> as biofilm-producer.</FIGCAPTION></FIGURE><P>The figure illustrates how Nuc (supernatant and pellet in a 1:1 ratio) inhibits biofilm formation by <EM>S. aureus</EM>. The poor readout of the negative control implicates that the test is invalid. However, the absorbance of biofilm increases as volume of Nuc increases, and the negative control shows low values. There is a significant difference between the negative control and 50 ul of Nuc.</P><H4 id="demonstratingefficientbiofilmdispersalbynuc">Demonstrating efficient biofilm dispersal by Nuc</H4><P>Biofilm dispersal by Nuc is demonstrated using crystal violet assays; one with <EM>P. aeruginosa</EM> as the biofilm-producer, and the other with <EM>S. aureus</EM>. Induced BL21(DE3) cell lysate samples (both debris pellet resuspension and supernatant) are used for the experiment. The biofilm is treated with 1 µl, 5 µl, 10 µl, or 20 µl of Nuc sample. Differences between the negative control and Nuc are evaluated using the student’s t-test. A p-value &lt;0.05 is considered statistically significant. </P><H6 id="biofilmassayfirstattempt">Biofilm assay: First attempt [❌]</H6><P>The samples from supernatants are labelled &quot;Nuc S&quot;. The samples from the pellets were contaminated and are not shown in the results. </P><FIGURE><FIGCAPTION>Figure 10. Absorbance measured for different treatments of Nuc using <I>P. aeruginosa</I> as biofilm-producer.</FIGCAPTION></FIGURE><P>The graph illustrates how different volumes of Nuc dispersed biofilm formed by <EM>P. aeruginosa</EM>. There is a significant difference (p=0.023) between the negative control and 1 µl of Nuc treatment. Notably, the absorbance, associated with the amount of biofilm present, decreases as the amount of negative control increases. This may be due to contamination or elution or dispersal of biofilm because of the increased amount of fluid <EM>per se</EM> in the treatment.</P><FIGURE><FIGCAPTION>Figure 11. Absorbance measured for different treatments of Nuc using <I>S. aureus</I> as biofilm-producer.</FIGCAPTION></FIGURE><P>The graph illustrates how different volumes of Nuc dispersed biofilm formed by <EM>S. aureus</EM>. There is a significant difference between 5 µl of negative control and 5 µl of Nuc S.</P><H6 id="biofilmassaysecondattempt">Biofilm assay: Second attempt [✔️]</H6><P>The second attempt of dispersing biofilm was similar to the first. However, this time a combination of pellet and supernatant was used as treatment with the ratio 1:1, e.g. for 5 µl treatment, 2.5 µl of each was used. For this attempt the biofilm was treated with 5 µl, 20 µl and 50 µl of Nuc. The samples were labelled &quot;Nuc&quot; and the negative control was labelled &quot;Neg&quot;. Only one volume of negative control was used.</P><FIGURE><FIGCAPTION>Figure 12. Absorbance  measured for different treatments of Nuc using<I>P. aeruginosa</I> as biofilm-producer.</FIGCAPTION></FIGURE><P>The figure illustrates how Nuc dispersed biofilm produced by <EM>P. aeruginosa</EM>. There is a significant difference between the negative control and each of the three volumes of Nuc. As the volume of Nuc increases, the absorbance at 595 nm tends to decrease. This implicates that Nuc significantly dispersed biofilm compared to the negative control. </P><FIGURE><FIGCAPTION>Figure 13. Absorbance measured for different treatments of Nuc using <I>S. aureus</I> as biofilm-producer.</FIGCAPTION></FIGURE><P>The figure illustrates how Nuc dispersed biofilm produced by <EM>S. aureus</EM>. There is a significant difference between the negative control and each of the three volumes of Nuc. This implicates that Nuc significantly dispersed biofilm compared to the negative control.</P><H4 id="demonstratingbacteriolyticactivityofnuc">Demonstrating bacteriolytic activity of Nuc</H4><P>A Kirby-Bauer test is suitable to perform to test the bacteriolytic activity. For this particular test, the <EM>E. coli</EM> TOB1 strain is used. The bacteria are streaked on antibiotic-free agar plates onto which small circular filter papers impregnated with the bactericide are placed. Additionally to the cell lysate samples containing the combat protein, a positive control in the form of an antibiotic are applied as a reference. The bactericidal activity can then be measured by measuring the “halo” around the impregnated Whatman paper. The larger halo, the higher bactericidal activity.</P><H6 id="kirbybauerfirstattempt">Kirby-Bauer: First attempt [❌]</H6><P>The growth of the bacteria was unevenly distributed on the agar plates which made it difficult to observe any clear results from most of the impregnated filter papers. The filter papers that were actually surrounded with cells did not show any signs of impeding growth.</P><P><STRONG>Troubleshooting</STRONG>
A higher concentration of the initial liquid culture of TOB1 cells will be used for the test to make sure that the entire area of the agar plate will be covered with sufficient amount of cells. Also, the filter paper used were not optimal for the experiment, which could have affected the dispersion of the nuclease.</P><H6 id="kirbybauersecondattempt">Kirby-Bauer: Second attempt [❌]</H6><P>The same cell lysates were used for this attempt. However, it was unsuccessful. There were no signs of impeded growth around the nuclease impregnated Whatman papers. There are three possible explanations to this result; i) the nuclease did not have a bactericidal effect on the TOB1 <EM>E. coli</EM> strain, ii) the concentration of nuclease in the cell lysate was too low or iii) the plasmid construct present in the BL21(DE3) cells used for expressing was not correct or had been unintentionally altered.</P><P><STRONG>Troubleshooting</STRONG>
The IPTG induction time when expressing nuclease in BL21(DE3) could have been too short. This time will be increased in future expressions.</P><H6 id="kirbybauertestwithtob1thirdattempt">Kirby-Bauer test with TOB1: Third attempt [❓]</H6><P>The results from the third attempt were inconclusive since it was difficult to determine whether the growth of TOB1 around the impregnated Whatman papers was actually impeded. More clear results of the activity of nuclease were obtained from the biofilm assay of the same cell lysate samples. </P><H4 id="addingsortaseandhistagstotheexpressionconstruct">Adding Sortase and His tags to the expression construct</H4><P>In order to purify and attach nuclease to the spider silk using Sortase A, a DNA sequence coding for a His-tag and a cognition motif (LPETGG) are added downstream to Nuc in the assembled expression construct. This sequence is called linker-tags (LT) in our documentation. BamHI is employed for the ligation of Nuc and tag sequence. BamHI restriction site is introduced downstream to Nuc through overhang PCR of the T7-Nuc sequence, as the first step. 3A assembly is then applied where the PCR amplified part is then digested with EcoRI and BamHI, the backbone with EcoRI and PstI, and the tag sequence with BamHI and either SpeI or PstI. This new final construct is transformed into TOP10 cells for optimum transformation efficiency.</P><H6 id="assemblyfirstattempt">Assembly: First attempt [❓]</H6><P>Gel electrophoresis showed no band at expected sizes and several contaminations at smaller sizes. Result of analysis of the extracted plasmid after transformation were also negative, showing several bands but none of them corresponding to the expected size of approximately 900 bp. However, since there actually were colonies on the plates after transformation, the possibility of a successful ligation can not be rejected.</P><P><STRONG>Troubleshooting</STRONG>
An variant to the 3A assembly process will be performed with only one insert where the entire pSB1C3-T7-Nuc plasmid will be PCR amplified with an BamHI overhang primer, and used as a linearized backbone in the ligation. This way, the T7-Nuc sequence will already be fused with the backbone. The linker-tag (LT) sequence will then be the only insert to be ligated, hopefully increasing the probability if a successful ligation.</P><H6 id="assemblysecondtotenthattempt">Assembly: Second to Tenth Attempt [❌]</H6><P>The assembly method with Phusion overhang PCR amplified pSB1C3-T7-Nuc plasmid discussed in the troubleshooting prior to these attempts was further discussed and modified to only involve on ligation step of both ends (BamHI and PstI). The first step in the modified assembly method was to PCR amplify the whole pSB1C3-T7-Nuc plasmid with the Suffix<EM>F and Nuc</EM>R overhang primer (with BamHI sequence). This linearized sequence were then to be digested at the ends (BamHI and PstI) and PCR purified prior to ligation with the tag sequence digested with at the same restriction sites, resulting in a 1 to 1 fusion (one backbone and one insert). The amplification of the pSB1C3-T7-Nuc construct were performed with a Phusion polymerase and the corresponding protocol. However, the original protocol (from NEB) was very general and a majority of the parameters had to be adjusted according to the specific experiment. This turned out to be very difficult and complicated. Adjustments of the reaction buffer, annealing temperature, extension time, and concentration of DMSO were found to be the major factors affecting the outcome of the PCR. These were set in different combinations where the annealing temperature and the extension time was increased if there were many small sized contaminations (relative the expected size of 2672 bp), and lowered with added DMSO if the results showed no bands. These attempts were unsuccessful since none of the PCR program setups resulted in a product of the correct size.</P><P><STRONG>Troubleshooting</STRONG>
According to the result in the last attempt (see figure) there are no contamination. However, the one visible band is not of the expected size, it is smaller. No contaminations indicate a specific binding of the primers, hence a suitable annealing temperature was set. Parameters that could be adjusted further is then the extension time but also the denaturation time. If the reason for the too short sequence is the limit of in extension time or not enough denaturation, increasing these two factors would then probably solve the problem.</P></SECTION></ARTICLE></MAIN></DIV><FOOTER class="site-footer clearfix"><SECTION class="sponsors">
    iGEM Stockholm is sponsored by
    </SECTION></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>