"
Page
Discussion
View source
History
teams
Log in
 
Team:uOttawa/Project
From 2009.igem.org
uOttawa iGEM2009
Home
Team
Project
Biobricks
Modeling
Notebook
Acknowledgments
uOttawa iGEM2009
Overall project
With more and easier access to high-calorie foods the worldwide prevalence of obesity has been on the rise in the past quarter century. In Canada alone, the average rate of obesity has doubled from 1979 to 2004, and approximately 23% of Canadian adults are obese (As estimated by Statistics Canada). Obesity is typically associated with many adverse health conditions and puts an enormous strain on the public healthcare system. Our goal is to engineer a strain of Lacobacillus to express the enzymes required for the synthesis of cellulose from glucose in an attempt to reduce the caloric intake of obese individuals. Cellulose is a polymer of linked D-glucose units that cannot be digested by humans. Lactobacillus is a strain of bacteria, which is commonly used to make yogurt, cheese, beer and other fermented foods, and is in fact part of the natural human gut fl ora. The idea would be to have an obese individual ingest yogurt containing our engineered strain of bacteria as a probiotic. That way after they have consumed their meal, a portion of the glucose from the meal would be converted into cellulose in the intestines, effectively reducing their caloric intake.
Project Details
This year in the lab we worked mainly with two bacterium: Acetobacter xylinum, which contains the cellulose synthase operon and Lactobacillus plantarum. A. xylinum is a gram-negative bacteria which grows at 26°C in a specialized media and forms a cellulose pellicle around itself and L. plantarum is a gram-positive lactic acid bacteria that grows at 30°C in MRS media.
Currently, we are in the process of transforming a number of different constructs that will characterize the expression of the synthase as well as the expression of the P45 promoter. One construct expressing the synthase will use a gram-positive origin of replication found in many lactic acid bacteria called RepA to propagate the plasmid while the other will use a genome integrating enzyme known as an integrase from a bacteriophage mv4 that specifically targets lactic acid bacteria to integrate the synthase directly into the L. plantarum genome.
The cellulose synthase is a 4-gene operon in Acetobacter xylinum, the gene product of which catalyzes the formation of β-1,4 glycosidic linkages between UDP-glucose monomers, creating a strand of cellulose. In A. xylinum this cellulose is excreted through pores in the membrane. To extract the cellulose synthase, we employed a Long-Range DNA Polymerase with designed primers to PCR amplify the 4-gene cellulose synthase operon from miniprepped A. xylinum genomic DNA.
Constructs
Our finished plasmid-expression of the cellulose synthase construct will include the cellulose synthase operon, a nisin resistance cassette for screening purposes and the gram-positive bacterial origin of replication RepA, all under the expression of the strong constitutive promoter P45.
The PLEB590 genes will also be PCR amplified with restriction sites, digested, and ligated to the cellulose synthase operon to create the plasmid expression synthase construct.
References for Constructs/Methods:
Auvray et al. (1997): Plasmid Integration in a Wide Range of Bacteria Mediated by the Integrase of Lactobacillus delbrueckii Bacteriophage mv4. Journal of Bacteriology, 179:1837–1845.
Takala, T., Saris, P., (2002): A food-grade cloning vector for lactic acid bacteria based on the nisin immunity gene nisI. Applied Microbiology and Biotechnology, 59:467-468.
Our finished genomic-expression construct replaces the gram-positive origin of replication with the mv4 bacteriophage integrase enzyme to integrate the cellulose synthase operon directly into the L. plantarum genome for increased stability and expression levels.
The integrase coding int gene and bacteriophage attP site were PCR amplified from the bacteriophage integrative vector PMC1 with added restriction sites, digested and ligated to the P45 promoter/Nisin resistance.  This will soon be ligated to the cellulose synthase to create the genomic expression synthase construct. These constructs will then be transformed through an electroporation procedure into Lactobacillus plantarum.
The last construct will characterize the expression of the P45 promoter by fusing it to the reporter RFP. This construct contains the P45 promoter, nisin antibiotic resistance and RepA, the origin of replication, all of which are ligated to the RFP/terminator biobrick found in the iGEM registry of parts.
The constitutive P45 promoter, Nisin resistance and RepA were all PCR amplified out of the food grade cloning vector PLEB590 with added restriction sites, digested, and ligated to the RFP/terminator biobrick to create the plasmid expression RFP construct.
We plan to transform the constructs into E. coli. in parallel with transformation into L.plantarum. We will do this to ensure expression of our construct, since L. plantarum and A. xylinum have different gram stain. This variation in membrane structure may affect cellulose synthase expression, since the sugar polymer must be extruded through membrane pores. E. coli and A. xylinum are both gram negative, so we increase the chance of cellulose synthase expression by transforming into E. coli in addition to Lactobacillus.
We are working on conducting some preliminary testing to confirm that our constructs work effectively with L. plantarum. We have finished insertion of RFP into the plasmid construct expression and are in the process of transforming this into L. plantarum.
Methods
h ttp://openwetware.org/wiki/Acetobacter_Xylinum_Culture
http://openwetware.org/wiki/Genomic_miniprep/Sigma_kit
http://openwetware.org/wiki/Electrotransformation_of_Lactobacillus_plantarum
http://openwetware.org/wiki/Preparation_of_Lactobacillus_Competent_Cells
http://openwetware.org/wiki/In_vitro_modification_of_DNA_for_L._plantarum
Results
The team was successful in developing several protocols related to growing and working with both A. xylinum and L. plantarum. Both strains were novel to the Kaern Lab, so development of these protocols required extensive background research and trial and error. The protocols include an A.xylinum media recipe and growth protocol, an optimized miniprep protocol for A. xylinum and some protocols for transformation into Lactobacillus plantarum. All of these protocols have been posted on openwetware.org for future iGEM teams or other interested individuals. The links to these protocols can all be found on the methods section of our Wiki, which provides links to openwetware.org  In addition, iGEM uOttawa successfully managed to amplify the cellulose synthase codon (~10 kb) seen confirmed on a gel below with a lambda hind III digest marker. This required optimization of various parameters of the FinnZymes DyNAzyme Long-Range PCR kit.
The expression-testing RFP construct was also completed and confirmed, although transformation into the target organism was unsuccessful (see Future Directions).
| October 16th
uOttawa iGEM2009
ETHICS
SECURITY
Project Safety
uOttawa iGEM integrating business and science
Health
Glucose and cellulose digestion
Probiotics
Obesity
Acetobacter xylinum
Lactobacillus
Sponsors
uottawa
Faculty of medicine
Faculty of engineering
VP research
Epocal
CREM-CRME
GE healthcare
Latest Events
team bonding at the sushi restaurant
world congress on industrial biotechnology, Montreal
the waterloo get together with other ontario teams
tech museum on the 5th of May
Day one, where do we start
© 2009 uOttawa iGEM, all rights reserved.
Powered by uottawa igem2009 , XHTML , CSS , Feed .
Retrieved from " http://2009.igem.org/Team:uOttawa/Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
