<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_uOttawa_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:uOttawa/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:uOttawa/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:uOttawa/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:uOttawa/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:uOttawa/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:uOttawa/Project</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><P><TITLE>
      uOttawa iGEM2009
    </TITLE></P><DIV id="navcontainer"><UL id="navlist"><LI><A href="https://2009.igem.org/Team:uOttawa/Home" id="current">Home</A></LI><LI><A href="https://2009.igem.org/Team:uOttawa/Team">Team</A></LI><LI><A href="https://2009.igem.org/Team:uOttawa/Project">Project</A></LI><LI><A href="https://2009.igem.org/Team:uOttawa/Biobricks">Biobricks</A></LI><LI><A href="https://2009.igem.org/Team:uOttawa/Modeling">Modeling</A></LI><LI><A href="https://2009.igem.org/Team:uOttawa/Notebook">Notebook</A></LI><LI><A href="https://2009.igem.org/Team:uOttawa/Acknowlegments">Acknowledgments</A></LI></UL></DIV><H1 id="header">uOttawa iGEM2009</H1><DIV id="wrap"><DIV id="page"><H2>Overall project</H2><P>With more and easier access to high-calorie
foods the worldwide prevalence of obesity has
been on the rise in the past quarter century. In
Canada alone, the average rate of obesity has
doubled from 1979 to 2004, and approximately
23% of Canadian adults are obese (As estimated
by Statistics Canada). Obesity is typically
associated with many adverse health conditions
and puts an enormous strain on the public
healthcare system. Our goal is to engineer a
strain of Lacobacillus to express the enzymes
required for the synthesis of cellulose from glucose
in an attempt to reduce the caloric intake
of obese individuals. Cellulose is a polymer of
linked D-glucose units that cannot be digested
by humans. Lactobacillus is a strain of bacteria,
which is commonly used to make yogurt,
cheese, beer and other fermented foods, and
is in fact part of the natural human gut fl ora.
The idea would be to have an obese individual
ingest yogurt containing our engineered strain
of bacteria as a probiotic. That way after they
have consumed their meal, a portion of the
glucose from the meal would be converted into
cellulose in the intestines, effectively reducing
their caloric intake.
        </P><H2>Project Details</H2><P>This year in the lab we worked mainly with two bacterium: Acetobacter xylinum, which contains the cellulose synthase operon and Lactobacillus plantarum. A. xylinum is a gram-negative bacteria which grows at 26°C in a specialized media and forms a cellulose pellicle around itself and L. plantarum is a gram-positive lactic acid bacteria that grows at 30°C in MRS media. </P><P>Currently, we are in the process of transforming a number of different constructs that will characterize the expression of the synthase as well as the expression of the P45 promoter. One construct expressing the synthase will use a gram-positive origin of replication found in many lactic acid bacteria called RepA to propagate the plasmid while the other will use a genome integrating enzyme known as an integrase from a bacteriophage mv4 that specifically targets lactic acid bacteria to integrate the synthase directly into the L. plantarum genome. </P><P>The cellulose synthase is a 4-gene operon in Acetobacter xylinum, the gene product of which catalyzes the formation of β-1,4 glycosidic linkages between UDP-glucose monomers, creating a strand of cellulose. In A. xylinum this cellulose is excreted through pores in the membrane. To extract the cellulose synthase, we employed a Long-Range DNA Polymerase with designed primers to PCR amplify the 4-gene cellulose synthase operon from miniprepped A. xylinum genomic DNA. 
 </P><H2> Constructs </H2><P>Our finished plasmid-expression of the cellulose synthase construct will include the cellulose synthase operon, a nisin resistance cassette for screening purposes and the gram-positive bacterial origin of replication RepA, all under the expression of the strong constitutive promoter P45.</P><P>The PLEB590 genes will also be PCR amplified with restriction sites, digested, and ligated to the cellulose synthase operon to create the plasmid expression synthase construct. 
 </P><EM><P>References for Constructs/Methods:</P><P>Auvray et al. (1997): Plasmid Integration in a Wide Range of Bacteria Mediated by the Integrase of Lactobacillus delbrueckii Bacteriophage mv4. Journal of Bacteriology, 179:1837–1845.</P><P>Takala, T., Saris, P., (2002): A food-grade cloning vector for lactic acid bacteria based on the nisin immunity gene nisI. Applied Microbiology and Biotechnology, 59:467-468.</P></EM><P>Our finished genomic-expression construct replaces the gram-positive origin of replication with the mv4 bacteriophage integrase enzyme to integrate the cellulose synthase operon directly into the L. plantarum genome for increased stability and expression levels.</P><P>The integrase coding int gene and bacteriophage attP site were PCR amplified from the bacteriophage integrative vector PMC1 with added restriction sites, digested and ligated to the P45 promoter/Nisin resistance.  This will soon be ligated to the cellulose synthase to create the genomic expression synthase construct. These constructs will then be transformed through an electroporation procedure into Lactobacillus plantarum.</P><P>The last construct will characterize the expression of the P45 promoter by fusing it to the reporter RFP. This construct contains the P45 promoter, nisin antibiotic resistance and RepA, the origin of replication, all of which are ligated to the RFP/terminator biobrick found in the iGEM registry of parts.</P><P>The constitutive P45 promoter, Nisin resistance and RepA were all PCR amplified out of the food grade cloning vector PLEB590 with added restriction sites, digested, and ligated to the RFP/terminator biobrick to create the plasmid expression RFP construct.</P><P>We plan to transform the constructs into E. coli. in parallel with transformation into L.plantarum. We will do this to ensure expression of our construct, since L. plantarum and A. xylinum have different gram stain. This variation in membrane structure may affect cellulose synthase expression, since the sugar polymer must be extruded through membrane pores. E. coli and A. xylinum are both gram negative, so we increase the chance of cellulose synthase expression by transforming into E. coli in addition to Lactobacillus.</P><P>We are working on conducting some preliminary testing to confirm that our constructs work effectively with L. plantarum. We have finished insertion of RFP into the plasmid construct expression and are in the process of transforming this into L. plantarum.</P><H2>Methods</H2><OL><LI>h<A href="http://openwetware.org/wiki/Acetobacter_Xylinum_Culture">ttp://openwetware.org/wiki/Acetobacter_Xylinum_Culture</A></LI><LI><A href="http://openwetware.org/wiki/Genomic_miniprep/Sigma_kit">http://openwetware.org/wiki/Genomic_miniprep/Sigma_kit</A></LI><LI><A href="http://openwetware.org/wiki/Electrotransformation_of_Lactobacillus_plantarum">http://openwetware.org/wiki/Electrotransformation_of_Lactobacillus_plantarum</A></LI><LI><A href="http://openwetware.org/wiki/Preparation_of_Lactobacillus_Competent_Cells">http://openwetware.org/wiki/Preparation_of_Lactobacillus_Competent_Cells</A></LI><LI><A href="http://openwetware.org/wiki/In_vitro_modification_of_DNA_for_L._plantarum">http://openwetware.org/wiki/In_vitro_modification_of_DNA_for_L._plantarum</A></LI></OL><H2>Results</H2><P>The team was successful in developing several protocols related to growing and working with both A. xylinum and L. plantarum. Both strains were novel to the Kaern Lab, so development of these protocols required extensive background research and trial and error. The protocols include an A.xylinum media recipe and growth protocol, an optimized miniprep protocol for A. xylinum and some protocols for transformation into Lactobacillus plantarum. All of these protocols have been posted on openwetware.org for future iGEM teams or other interested individuals. The links to these protocols can all be found on the methods section of our Wiki, which provides links to openwetware.org 
In addition, iGEM uOttawa successfully managed to amplify the cellulose synthase codon (~10 kb) seen confirmed on a gel below with a lambda hind III digest marker. This required optimization of various parameters of the FinnZymes DyNAzyme Long-Range PCR kit.  </P><P>The expression-testing RFP construct was also completed and confirmed, although transformation into the target organism was unsuccessful (see Future Directions). 
</P><P class="info">|<A href="#">October 16th</A></P></DIV><DIV id="sidebar"><H3>uOttawa iGEM2009</H3><UL><LI><A href="http://www.ipm-int.org/boxmode/pdf/Ethics.pdf" target="_parent">ETHICS</A></LI><LI><A href="http://www.ipm-int.org/boxmode/pdf/Security.pdf">SECURITY</A></LI><LI><A href="http://www.ipm-int.org/boxmode/pdf/Safety.pdf">Project Safety</A></LI><LI><A href="http://www.ipm-int.org/boxmode/pdf/business.pdf">uOttawa iGEM integrating business and
science</A></LI></UL><H3>Health</H3><UL><LI><A href="http://www.ipm-int.org/boxmode/pdf/Glucose.pdf">Glucose and cellulose digestion</A></LI><LI><A href="http://www.ipm-int.org/boxmode/pdf/Probiotics in the Food Industry.pdf" target="_parent">Probiotics</A></LI><LI><A href="http://www.ipm-int.org/boxmode/pdf/Obesity.pdf">Obesity</A></LI><LI><A href="http://www.ipm-int.org/boxmode/pdf/Acetobacter_xylinum.pdf">Acetobacter xylinum</A></LI><LI><A href="http://www.ipm-int.org/boxmode/pdf/lactobacillus_plantarum.pdf">Lactobacillus</A></LI></UL><H3>Sponsors</H3><UL><LI><A href="http://www.uottawa.ca">uottawa</A></LI><LI><A href="http://www.medicine.uottawa.ca/">Faculty of medicine</A></LI><LI><A href="http://www.engineering.uottawa.ca/">Faculty of engineering</A></LI><LI><A href="http://www.uottawa.ca/research/">VP research</A></LI><LI><A href="http://www.epocal.com/">Epocal</A></LI><LI><A href="http://www.medicine.uottawa.ca/crem/eng/">CREM-CRME</A></LI><LI><A href="http://www.gehealthcare.com/caen/">GE healthcare</A></LI></UL><H3>Latest Events</H3><UL id="bits"><LI>team bonding at the sushi restaurant</LI><LI>world congress on industrial biotechnology, Montreal</LI><LI>the waterloo get together with other ontario teams</LI><LI>tech museum on the 5th of May</LI><LI>Day one, where do we start</LI></UL></DIV></DIV><DIV id="footer"><P>© 2009 uOttawa iGEM, all rights reserved.
        Powered by 
        <A href="http://www.example.com/" title="Powered by .."> uottawa igem2009</A>, 
        <A href="http://validator.w3.org/check/referer" title="Valid XHTML">XHTML</A>, 
        <A href="http://jigsaw.w3.org/css-validator/check/referer" title="Valid CSS">CSS</A>, 
        <A href="atom.xml" title="XML feed">Feed</A>.
      </P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:uOttawa/Project">http://2009.igem.org/Team:uOttawa/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:uOttawa/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:uOttawa/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:uOttawa/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:uOttawa/Project&amp;oldid=166756" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>