<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCLA_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCLA/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="header-bar"><DIV id="header-menu-container"><UL id="header-menu"><LI class="dropdown"><A><H2>Projects</H2></A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:UCLA/Protein_Cages">Protein Cages</A><A href="https://2016.igem.org/Team:UCLA/Super_Soldier_Bacteria">Super Soldier Systems</A><A href="https://2016.igem.org/Team:UCLA/InterLab">Interlab Study</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:UCLA/Parts"><H2>Parts</H2></A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:UCLA/Basic_Part">Basic Parts</A><A href="https://2016.igem.org/Team:UCLA/Composite_Part">Composite Parts</A><A href="https://2016.igem.org/Team:UCLA/Part_Collection">Part Collection</A></DIV></LI><LI class="dropdown"><A><H2>Outreach</H2></A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:UCLA/HP/Silver">Human Practices</A><A href="https://2016.igem.org/Team:UCLA/Collaborations">Collaborations</A></DIV></LI><LI class="dropdown"><A><H2>Judging</H2></A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:UCLA/Medal_Requirements">Medal Requirements</A><A href="https://2016.igem.org/Team:UCLA/Safety">Safety</A></DIV></LI><LI class="dropdown"><A><H2>Team</H2></A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:UCLA/Team">Meet the Team</A><A href="https://2016.igem.org/Team:UCLA/Attributions">Attributions</A></DIV></LI></UL></DIV></DIV><DIV class="row"><DIV class="row"><H2>Protein Cages</H2><P> Higher order self assembling protein assemblies are a commonplace in nature, one common example being ferritin, which carries iron in many single-celled organisms. Due to their encapsulating function and box-like structure, such assemblies are often called protein “cages.” Inspired by these natural sources, much research has been done into creating synthetic cages with new properties such as stability, size, and subunit types. Furthermore, in many cases applications of cages have been suggested, but many have not been tested experimentally. </P><P>One such application is targeted drug delivery. By specifying the region to which drugs should be delivered, effect can be limited to the areas desired and avoid damaging or affecting other parts of the body. Moreover it ensures the drugs actually affect their desired targets with ideally high efficiency. Since these drugs would likely be inserted into the bloodstream, a function directly connected to that method would be a first step into drug delivery.</P><P>Thrombosis, when a clot forms inside the blood stream, is associated with widespread diseases including stroke and many heart conditions. Anticoagulants prevent clot formation by interfering with clotting factors and allow a smoother blood flow, restoring normal circulation. However these drugs can have side effects, a main one being problems stopping bleeding from wounds. By localizing and minimizing the amount of drug used, issues such as these would become less serious. A protein cage containing an anticoagulant is well-suited to this. To specify the clot as a target, an associated enzyme or byproduct of clotting could be used. Thrombin is one such enzyme, promoting clotting in most cases. It does, however, cleave a certain amino acid sequence. This sequence, if properly inserted into a protein cage containing the anticoagulant, would result in disassembly of the cage, release of the drug molecule to the targeted area,and destruction of the clot.</P><P>The goal of our project is to modify a protein cage that can be disassembled using a thrombin protease and to use this protein cage to deliver a drug molecule. We are using two previously created synthetic cages, one composed of 12 subunits forming a tetrahedron cage while the other composed of  24 subunits forming an octahedron cage. For each of these protein cages, we have designed mutants with inserted thrombin cleavage sites and hope to induce disassembly of the cages by treatment with thrombin protease. We will then load anticoagulant drug molecules into our mutant cages and assay for the release the drug molecules when the cage disassembles in the presence of thrombin. </P></DIV><DIV class="row"><H2>Super Soldier Bacteria: Manipulation of Contact Dependent Growth Inhibition Systems </H2><P>Antibiotic resistance is a well-known and pressing global health concern. In more recent times, there has been a stagnation in discovering new classes of antibiotic, particularly for gram-negative bacteria. Throughout the mid-1900s, scientists were rapidly discovering the current major classes of antibiotics using the Waksman platform, which entailed systematically screening soil and fungal microbes for growth inhibition. However, the recurring problem of rediscovery and failure of subsequent high-throughput biochemical assays led many pharmaceutical companies to abandon their antibiotic development programs. Current antibiotics are generally broad-spectrum, acting against symbiotic gut flora in addition to pathogenic invaders. Therefore, specificity has recently become a target in modern antibiotic development in order to bypass challenges of both rediscovery and indiscriminate killing.</P><P>The core of antibiotic development lies in targeting a molecule or pathway essential for bacterial survival. Bacteria themselves have already developed several mechanisms to both communicate and compete with other bacterial strains occupying similar niches, which can be exploited for the creation of new antibiotic drugs. While some systems rely on secreted signaling molecules, direct cell-to-cell contact is also used to mediate intercellular interactions. Contact-dependent growth inhibition (CDI) is one such system that allows CDI+ bacterial strains to outcompete closely related CDI− siblings by synthesizing and translocating a toxin through the membrane of strains expressing the appropriate receptor. The CDI system is common in many strains of pathogenic gram-negative bacteria, such as Yersinia pseudotuberculosis and uropathogenic strains of Escherichia coli, making it a potentially suitable target for future antibiotics. </P><P>This type of growth inhibition was first seen in E. coli strain EC93, in which toxin delivery is mediated by the CdiA/CdiB two-partner secretion system and the outer membrane protein BamA, which is a conserved receptor among all strains of E. coli. EC93 is protected from its own toxin, located at the C-terminal region of CdiA (CdiA-CT), by the expression of a CdiI immunity protein that interacts directly with the toxin to neutralize its pore-forming capabilities. The CdiBAI cluster is common throughout most CDI+ strains, significantly varying only in CdiA-CT and CdiI sequences. </P><P>Our project focuses on the translocation of three different CDI systems from EC93 and Enterobacter aerogenes ATCC 13048 (containing two CDI loci) into one lab strain of E. coli to create a super-soldier organism that is able to selectively inhibit the growth of these strains. A lab strain containing CDI EC93, for instance, would be able to target all bacteria containing the corresponding receptor for CDI EC93, which would in this case be BamA Ecoli. In order to target EC93 itself, however, we aim to switch CdiA-CT EC93 with a toxin from ATCC 13048 so that the immunity protein of EC93 does not prevent growth inhibition. It has previously been shown that CdiA-CT regions are modular, and systematically modularizing the toxin region using appropriate restriction sites is another goal of our project. While the CDI system in EC93 targets the BamAEcoli, the relevant receptors in other species of pathogens and even in other strains of E. coli is unknown. An additional facet to our project would be to find the receptors that each CDI system of ATCC 13048 targets by creating a transposon insertion library. Mutants that display a whole or partial CDI immunity phenotype will be sequenced to determine which genes have undergone transposon insertion, providing insight into which receptors may be targeted by ATCC 13048 CDI systems.</P></DIV><DIV class="row"><H3> References </H3><OL><LI> Aoki, S. K., Pamma, R. Hernday, A. D., Bickham, J. E., Braaten, B. A., Low, D. A. (2005) Contact-dependent inhibition of growth in Escherichia coli. Science. 309, 1245-1248. </LI><LI> Gallagher, L. A., Ramage, E., Patrapuvich, R., Weiss, E., Brittnacher, M., and Manoil, C. (2013) Sequence-defined transposon mutant library of Burkholderia thailandensis. mBio, 4(6), e00604-13.</LI><LI> Lewis, Kim. (2013) Platforms for antibiotic discovery. Nature Reviews Drug Discovery. 12, 371–387. </LI><LI> Ruhe, Z. C., Low, D. A., and Hayes, C. S. (2013) Bacterial contact-dependent growth inhibition. Trends Microbiol. 21, 230–237. </LI><LI> Ruhe, Z. C., Wallace, A. B., Low, D. A., &amp; Hayes, C. S. (2013). Receptor Polymorphism Restricts Contact-Dependent Growth Inhibition to Members of the Same Species. mBio, 4(4), e00480–13. </LI><LI> Willett, J. L., Ruhe, Z. C., Goulding, C. W., Low, D. A., and Hayes, C. S. (2015) Contact-dependent growth inhibition (CDI) and CdiB/CdiA two-partner secretion proteins. J. Mol. Biol. 427, 3754–3765. </LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>