<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCLouvain_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCLouvain/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>&quot;</P><DIV class="container-fluid"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2016.igem.org/Team:UCLouvain">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-hashpopup="true" aria-expanded="false">About Us  </A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/Sponsors">Sponsors</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-hashpopup="true" aria-expanded="false">Project  </A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/Context">Context</A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/Background">Background</A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/Experimental">Experimental Approach</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-hashpopup="true" aria-expanded="false">Lab Work </A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/Homemade">Homemade product</A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/Perspectives">Perspectives</A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/Parts">Parts</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-hashpopup="true" aria-expanded="false">Human practices </A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/Story">Our story</A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/Media">In the media</A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/EE16">EE16</A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/Medals">Medals</A></LI></DIV><ARTICLE><DIV class="container"><DIV class="row"><H2>Results</H2><P>As we used antibiotics and transition metal salts, we had to determine their minimum inhibitory concentrations (MIC) before effectively using them. Fortunately, we worked with experienced researchers who are familiar with antibiotics concentrations commonly used in labs. Then, we were taught that we should use these concentrations :</P><UL><LI>Ampiciline : 250µg/mL</LI><LI>Kanamycine : 50µg/mL</LI><LI>Vancomycine : 3µg/mL</LI></UL><P>Concerning our transition metal salts, we conducted experiments to assess the effect of different concentrations of NiCl2 or ZnCl2 in our bacteria growth medium. We tested the following concentrations for both salts : 2mM, 1mM, 0.5mM, 0.25mM and 0.1mM</P><P>We selected the highest concentration at which no toxicity could be observed through the experiment, 0.25mM for both salts. We used this concentration for all manipulations onwards.</P><H3>Control of the basic material :</H3><H4>Plasmids (pIV* and pIV WT)</H4><P>The pPro30 vector is used for the expression of the wild-type and the mutated pIV porin. The insertion of the gene is performed in the multiclonal site downstream of the PprpB promoter. Selection pressure is provided by the ampicillin resistance gene (AmpR). The propionate operon promoter, PprpB, is regulated by the positive regulator, prpR. This system induces the expression of the porin in the presence of propionate (0,5 mM) and suppresses the expression when adding glucose (1%).</P><P>Figure 1: Representation of the pPro30 vector used for the expression of the pIV porin</P><H4>Analysis of the pIV WT and PIV* constructs</H4><P>To check the quality of these constructs, we performed a restriction reaction with HindIII. This restriction enzyme is expected to give 2 fragments when cutting the plasmids, one at 3917bp and the other at 2789bp (see figure 1).</P><P>As expected, we obtained these two bands for each construct after restriction (see figure 2).</P><P>Figure 2: Plasmid restriction by HindIII. “NR” stands for non-restricted plasmid and “R” for restricted plasmid. (Molecular weight : Smart Ladder)</P><H3>Sequencing</H3><P>We then sent these plasmids for sequencing analysis. This step provides us further information on the quality of our constructs.</P><P>The sequencing data confirmed the punctuated mutation (pIV*S324G) located in the mutated porin (see figure 3). Moreover, no other differences have been detected between the wild-type and the mutated porin.</P><P>Figure 3: Sequencing data for the pIV WT and mutated</P><H3>Library construction by QuickLib method</H3><H4>PCR reaction</H4><P>In order to generate linear fragments with a targeted randomized region and overlapping ends, we performed PCR reactions using different set of primers :</P><H4>Cycling conditions:</H4><H4>Electrophoresis</H4><P>To verify the length of our PCR products, we used a linear version of the PIV* construct obtained via restriction with BamHI or XhoI (see figure 4).</P><P>Figure 4: Amplification of the pIV gene using degenerated primers (B1-B4). (Molecular weight : Smart Ladder)</P><H4>Gibson assembly</H4><P>After amplification, those linear fragments are circularized back to a plasmid form thanks to the hybridization of the overlapping region and the Gibson assembly method.</P><DIV class="block-center"><P>Figure 5: Plasmid recircularization using Gibson Assembly (B1-B4).(Molecular weight : Smart Ladder)</P></DIV><H3>Selection</H3><H4>First step: Selection of the open phenotype</H4><P>During the first selection step on selective maltodextrin media, we were able to get leaky mutants (see figure 6).</P><DIV class="block-center"><P>Figure 6: Growth of leaky mutant on selective maltodextrin medium.</P></DIV><P>As a positive control, we used the pIV*S324G porin that grows on selective maltodextrin medium. On the other hand, for the negative control, we used the pIV WT that needs much more time to grow on this selective medium (see figure 7).</P><DIV class="block-center"><P>Figure 7: pIV*/pIV WT growth on selective maltodextrin media (controls)</P></DIV><H4>Second step: Selection of the closed phenotype</H4><P>To select an inducible opening or closing of the porin, we compared bacterial viability on selective maltodextrin media in presence and absence of our transition metal ions (Zn<SUP>2+</SUP> and Ni<SUP>2+</SUP>). If, on the same medium, a mutant survives in presence of the ions and dies in its absence or vice-versa, it possibly contains a porin for which the gate opening or closing is regulated by Zn<SUP>2+</SUP> or Ni<SUP>2+</SUP>. The mutants selected through this step should be further investigated and characterized.</P><P>In the picture below (figure 8), 83 mutants obtained after the first selection have been spotted on 4 selective media to check their potential regulable behavior.</P><DIV class="block-center"><P>Figure 8: Second selection on selective maltodextrin media in presence and absence of our transition metal ions (Zn<SUP>2+</SUP> and Ni<SUP>2+</SUP>).</P></DIV><P>Here, we can already see some growth differences in several mutants that suggest an influence of transition metal ions present in the medium. However, we didn’t have enough time to replicate the data and further characterize these mutants.</P><H3>Mutant screening</H3><P>To analyze the potential regulable leaky mutants, we performed colony PCR reactions to amplify the pIV gene and then check the mutations introduced within the gates.</P><P>For the amplification, we used two primers that amplify most of the pIV sequence. This corresponds to 987bp and covers all the mutated regions.
                </P><H4>Cycling conditions</H4><H4>Electrophoresis<DIV class="block-center"><P>Figure 9: Amplification of the pIV porin.</P></DIV></H4><H4>Sequencing<P>Due to a lack of time, we were only able to sequence over 16 leaky mutants that went through the first selection step, each presenting mutations within one of the two gates (see figure 10). For some mutants, we can observe a strong histidine enrichment within the 15 amino acids mutated region, which demonstrates the efficiency of the QuickLib method used for mutagenesis. We observed up to 7 histidines in one of the mutants and the actual porin is made of 12 identical subunits, which means a total of 84 histidines added right on the hinge of this door. Interestingly, some amino acids present in the mutated regions remain quite stable through mutagenesis, suggesting that they may be important for the integrity of the porin. Also, it’s quite surprising to see that the mutated regions are rather resistant to mutations, with some porins presenting more than 10 mutations in the gate.</P><DIV class="block-center"><P>Figure 10: Sequencing results of the mutated porins. Histidine mutations are shown in red and other mutations are shown in yellow.</P></DIV><P>These first results are very promising since a lot of colonies were obtained after the first selection step and the sequencing analysis shows mutants with many different amino acid combinations. With additional sequencing results, it will be interesting to do some statistical analysis of the mutations introduced in the gates. </P></H4></DIV></DIV></ARTICLE>

&quot;
    </DIV></DIV></DIV></DIV></DIV></BODY></HTML>