Team:UCL/Model/Larginine
<!DOCTYPE html>
UCL iGEM 2016 | BioSynthAge
Search
Menu
Project
Menu
Overview
How are we approaching healthy ageing?
Lungs
Gene therapy approach
Gut
Lycopene probiotic
Teeth
Redesigning the oral microbiome
Heart
Healthy heart, healthy ageing
Lubricant
Detecting and killing UTI causing bacteria
Pest Control
Using oxidative stress to accelerate ageing
Design
Engineering therapies of tomorrow
Notebook
iGEM-ing all day, everyday
Achievements
See what Biosynthage are proud of!
Parts
Lab
Menu
Overview
Transferring concepts to the lab
Proof of Concept
Building systems that solve healthy ageing
Demonstrate
The first step towards reality
Lab-book
Find out what we did in the labs!
Safety
Integrating safety into human based therapies
Protocol
Methodology behind the data
Results
See our data and outputs of our project
Modelling
Menu
Overview
Simulating and optimising our devices
L-arginine
Modelling inhibitory feedback loops
Xylitol
Oscillatory behaviour
Human Practices
Menu
Our Story
Biosynthage's journey
Silver
Building a bridge between Biosynthage and society
Gold
Integrating design with the world
Integrated Practices
Real world science
Engagement
Bringing Biosynthage to people
Entrepreneurship
Menu
Overview
Bringing synthetic biology to the business world
AlphaBrick
Making biobrick design easier
Xweet
Engineering novel bioprocess for Xylitol production
People
Menu
Team
Meet the innovators of tomorrow
Collaborations
Our iGEM community
Sponsors
Without whom, would not be possible
Attributions
To everyone involved with our journey
Why?
How?
Development
Results
L-Arginine
Why we chose to develop a model of the arginine pathway
From a biological point of view, arginine constitutes about 5% of proteins in E. coli and  is an important element of many processes (1)(for more details on biology of arginine please visit our project website . We chose to model the arginine part of the project because the dynamics of the inhibitory feedback loop and the effects of its perturbation are hard to intuitively predict based just on the knowledge of the pathway, but at the same time are not hard to model. This model was intended to help us to formalise analysis of the pathway during our experiments, to help us to describe the involvement of individual regulatory mechanisms in the pathway and to predict the quantity of arginine produced by E. coli strains with various modifications in the arginine production pathway.
We found several published models of the arginine pathway. The focus of the first model (2) was gene expression - in particular promoter activity of the elements of the arginine pathway - as a function of bacterial growth. The authors argue that the level of gene expression is a result of two mechanisms: specific (the pathway) and global. Two principles have been proposed:
1) During metabolic steady state, repression of transcription dominates. Under stable conditions the promoter activity never reaches its full activity.
2) Maximum promoter activity depends on global regulation of growth and metabolism: promoter activity peaks only during transient adaptation in response to dynamic changes – this is linked to the global regulation of the promoter activity.
What was important to keep in mind when developing our model is that the repressive activity varied at most three-fold amongst the seven targets of argR present in the pathway, and that self-repression of argR was ten-fold weaker. Additionally, the observed enzyme concentrations followed the just-in-time rule, according to which concentrations of enzymes decrease along the pathway, and thus the observed concentrations were:  argA > argCBH > argD > argE > argF > argG (with argI being the exception).
The drawback was that the published code of this model contained only the argA and argR parameters, whereas we were interested in understanding the whole arginine pathway. However, in the future, we could use the model to include global regulatory mechanisms in our model: the experimental modifications we planned are likely to affect the growth rate of the bacteria and thus our model may deviate from the experimental data because of not taking the effects of the global machinery on promoter activity into account.
The second model (3) is a mathematical description of the arginine pathway and its cross-section with the pyrimidine pathway. The model is divided into three segments where each segment contains only one regulatory step, whereas other steps are considered non-rate limiting and not linked to other pathways.
In comparison to the second model, we wanted our model to include more steps of the arginine pathway and exclude those elements of the pyrimidine pathway which do not converge with the arginine pathway. However, the publication was a useful guide when it comes to estimating parameters of our model.
Because of the reasons mentioned above and because we wanted to have a simplified model which focuses on the part of the arginine pathway we were going to modify in the lab, which models the pathway at a protein rather than gene level and which is easy to use by scientists without an expertise in modelling, we decided to develop our own model. We chose to build our model using Simbiology because of its user-friendly interface with a number of build-in options for testing and further use of the model.
References
Charlier, D., and Glansdorff, N. (2004) in Biosynthesis of Arginine and Polyamines (Bock, A., Curtiss, R., Kaper, J. B., Neidhardt, F. C., Nystrom, T., Rudd, K. E., and Squires, C. L., eds) module 3.6.1.10, American Society for Microbiology, Washington, DC.
Gerosa, L., Kochanowski, K., Heinemann, M. and Sauer, U. (2013) ‘Dissecting specific and global transcriptional regulation of bacterial gene expression.’, Molecular systems biology. European Molecular Biology Organization, 9, p. 658. doi: 10.1038/msb.2013.14.
Caldara, M., Dupont, G., Leroy, F., Goldbeter, A., De Vuyst, L. and Cunin, R. (2008) ‘Arginine Biosynthesis in Escherichia coli: experimental perturbation and methematical modeling’, Journal of Biological Chemistry. American Society for Biochemistry and Molecular Biology, 283(10), pp. 6347–6358. doi: 10.1074/jbc.M705884200.
Overview of the model
As you can see in the diagram below (Fig. 1), our model describes the metabolic pathway in which arginine is obtained from glutamine and glutamate acetyl-CoA, and includes the inhibitory feedback loops. The green species are the metabolites of the pathway, the yellow circles the reactions producing them. The blue species are the enzymes and enzyme repressor complexes at each stage of the reaction, the red circles represent the repression. The orange species is argR, which is responsible for most of the repression.
Dashed lines were used to indicate species which do not get used up in the reactions, e.g. enzymes, and are both reactants and products of the reactions. Reverse arrows indicate reversible reactions and closed-circle arrows indicate that species are replenished and kept at a constant level.
The reactions in the pathway are all modelled with Michaelis-Menten kinetics, while the repression reactions are modelled using reversible mass action.
Figure 1. Overview of the model.
Detailed description of our model
Parameter adjustment:
Without experimental data from the lab to fit our model to, we adjusted the model until sensible and consistent with the published observations results were reached. The scale of the model would need to be adjusted when our data was available, but we can still make predictions about the relative change of arginine levels under certain conditions. With certain parameters the model produces unphysical results, for example if the initial concentration of arginine is set to 0 there will be an integration error where the gradient of arginine tends to negative infinity.
Table 1. Behaviour of all species.
Table 2. Equations in the model.
Experiments
In our model, the concentration of arginine reached a steady state. The steady level reached can be finely controlled by adjusting the rate of degradation of ArgR. This shows that argR could be a good target for tightly controlling the amount of arginine produced by E. coli. ArgR degradation rate constant kf varied between 0 and 1.
At kf above 0.05, changes in the kf have a minimal effect on the level of arginine reached  (Fig. 2A, a.u. stands for arbitrary units). With kf = 1 the argR level reached equilibrium at a very low concentration (Fig. 2B), and with kf = 0 (Fig. 2C), the argR levels reached a maximum.
 
Figure 2. Effects of argR degradation rate kf on the behaviour of the model. A) kf = 0.05, B) kf = 0.01 C) kf = 0
 
The level of arginine does not scale linearly with the level of degradation, but is more sensitive when argR is at the lower end of the scale. This would have to be taken into account when planning future experiments and we decided to explore it further.
We tested the behaviour of the model with six different values of argR degradation rate kf (Fig. 3A). Then, we analysed the maximal value of arginine reached in each of the simulations with a different kf. This confirmed our earlier observations that the relation is not linear and that the derivative d arginine/d kf is high for low values of kf (0 - 0.01) and decreases towards zero for higher values of kf.
 
Figure 3. Max arginine amount as a function of argR degradation rate.
This model has shown us that fine control over the arginine concentration in e.coli is theoretically possible. Although the functional relationship between the concentration of arginine at steady state and the rate of argR degradation needs to be established, the results obtained using our model suggest that the arginine pathway may be a viable target for any future work exploring the link between the arginine pathway and ageing.
BIOSYNTHAGE
BioSynthAge, Evgeny Saveliev, 2016
University College London, Gower StLondon, United KingdomWC1E 6BT
Project
Description
Modelling
Parts
Notebook
Human Practices
Our Story
Gold
Silver
Integrated
Team
Team
Attributions
Achievements
Sponsors
