<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Unesp_Brazil_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Unesp Brazil/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><H1 class="logo">iGEM Unesp Brazil</H1><NAV class="menuCristo"><LABEL for="toggle">☰</LABEL><DIV class="menuCaramba"><A href="https://2018.igem.org/Team:Unesp_Brazil/Description">PROJECT</A><A href="https://2018.igem.org/Team:Unesp_Brazil/Model">MODEL</A><A href="https://2018.igem.org/Team:Unesp_Brazil/Parts">PARTS</A><A href="https://2018.igem.org/Team:Unesp_Brazil/Hardware">HARDWARE</A><A href="https://2018.igem.org/Team:Unesp_Brazil/Human_Practices">HUMAN PRACTICES</A><A href="https://2018.igem.org/Team:Unesp_Brazil/Team">TEAM</A><A href="https://2018.igem.org/Team:Unesp_Brazil/Collaborations">COLLABORATIONS</A><A href="https://2018.igem.org/Team:Unesp_Brazil/Safety">SAFETY</A><A href="https://2018.igem.org/Team:Unesp_Brazil/InterLab">INTERLAB</A><A href="https://igem.org/2018_Judging_Form?id=2660" target="_blank">FOR JUDGES</A></DIV></NAV><DIV id="bg-c"><DIV id="faixa"><H1>Model</H1></DIV></DIV><DIV id="conteudo"><DIV id="texto"><H2>General model</H2><P>
We decided to describe our system with deterministic modeling of ordinary differential equations that describe the molecular biology’s central dogma of gene expression inside the cell.  
</P><H3>Reactions</H3><P>
Here we considered transcription, translation and mRNA and protein degradation rates.
</P><H3>Equations</H3><P>
We used the law of mass action to describe the reactants equilibrium. Here we also considered that DNA concentration doesn’t change with time.  
</P><H3>Reaction rate description</H3><H3>Translation regulation by sRNA - mRNA interactions</H3><P>
Here, we take the same assumptions as Kelly et al., 2018 (Supplementary Data), considering that bound sRNA-mRNA species have the shorter lifetime than unbound sRNA or mRNA (meaning that the complex degrades before having any chance to unbound), representing then, a first-order reaction.
</P><H2>Circuit modeling</H2><P>
Carbon catabolite repression (CCR) is a regulatory mechanism by which bacteria regulate the expression of functions for the use of secondary carbon sources in the presence of a preferred carbon source. In gram-positive bacteria, the CCR is mediated by the carbon catabolite control protein A (CcpA) that regulates genes of carbon metabolism when glucose is present in the system.
Our system is based on an anti-leakage orthogonal double repression framework, in which the insulin production is controlled by the expression of an orthogonal T7 polymerase, which is under Ptet control, a promoter repressed by TetR repressor protein. Our designed sRNA prevents leaked Insulin mRNA translation from happening. Finally, both TetR and sRNA’s gene expression are under glucose activated CcpA control, possessing CcpA binding sequence ‘cre’.
</P><H3>1. Induction of carbon catabolite system</H3><P>
We simplified CcpA activation by glucose presence by a simple Hill’s equation, in order to avoid large quantities of complex equations involving metabolic processes. ‘Glu’ represents glucose concentration in the cell, n is Hill constant, ‘Kglu’ is glucose-CcpA affinity constant and CCpA total amount is determined by the sum of initial unbound fraction (CcpA0) and glucose activated fraction (CcpA*).
</P><H3>2. Repression module</H3><P>
This module is composed of two genetic circuits that ensure that our system won’t produce any insulin when glucose is absent. TetR is a repressor protein responsible for blocking T7 polymerase expression, which is the only enzyme responsible for insulin gene transcription, under PT7 promoter control. Our designed sRNA aims Insulin’s mRNA, that can be produced by ptet leakage, aiming for complete repression control against insulin production in absence of glucose. Here ptet, pveg and  pT7 denotes promoter activity, ptet0, pveg0 and  pT70 describe promoter leakage; DNA represents DNA copy; Φmrnatet, Φsrna and Φmrnat7 RNA degradation rates; αTet and αT7pol proteins degradation and dilution rate and KCcpA* and KTet are affinity DNA-Repressor protein affinity constant. 
</P><H3>3. Insulin production</H3><P>
When glucose is present, CcpA is activated and represses TetR and sRNA expression, unblocking T7 RNA polymerase gene expression finally leading to Insulin production. Here, we denote T7-pT7  interactions by an activator protein Hill equation without leakage, considering that T7 RNA polymerase executes orthogonal transcription. The sRNA action is described by the second expression in the first equation, in which KdsRNA represents Insulin mRNA - sRNA binding affinity. 
</P><H3>Parameters</H3><H2>Parameter estimations and assumptions:</H2><P>
When creating a simulation, we don’t always find the exact parameter value that we are expecting (we even have different sources providing completely different values for the same thing). That’s why sometimes it’s necessary to make some assumptions and estimations in order to reach the most realistic results.
</P><H3>DNA copy number:</H3><P>We’ve designed this circuit to be integrated into the chromosome as a single copy set of genes.</P><H3>Translation rate:</H3><P>
We’ve calculated the translation rate of each protein by using the equation shown above, fixing ribosome translation rate in 15 aa/s according to Bionumbers (key numbers) and its size according to the amino acid residue number.
</P><H3>Promoters:</H3><P>
Promoter activity and leakage have a strong gene and biochemical context dependency, which makes parameter setting extremely difficult to elaborate. In our simulations, we assumed that ptet and pveg have the same activity, while pT7 possess double of its promoter strength (assuming its known high promoter activity). We also determined that promoter leakage would represent 20% of the total promoter activity.
</P><H3>CCpA activation</H3><P>
Here, we reimagined CcpA activation by glucose, by following similar kinetics and parameters to the CAP (Gram-negative carbon catabolite repression protein) and cAMP reaction. Furthermore, we considered a total amount of 350 CCpA molecules per cell(1 molecule per cell  = 1nM/cell).
</P><H3>Simulation</H3><P>
We used MATLAB software to solve the ODE system and simulate the system dynamics, using ode15s solver. ETH Zurich Team helped us a lot with the coding and also with some modeling advice. After taking a look at our first code and model, they sent us some tips to improve our model and a more elegant version of the code, making it easy to manipulate parameters and variables. They also shared materials about stochastic modeling, that will help future attempts on modeling our circuit or future projects.
<A href="https://static.igem.org/mediawiki/2018/2/26/T--Unesp_Brazil--simulation.zip">Click here to download our simulation files</A></P><H2>Results</H2><H3>sRNA influence in the circuit:</H3><P>
After building our model and starting the parameter setting, we concluded that the sRNA-mRNA affinity constant (Kdsrna) has a high influence in the circuit response.  Then, we decided to analyze how different values would influence the Insulin expression when the system is on and off, by looking at the Blocking efficiency (%):
</P><P>Blocking efficiency (BE) denotes for the percentage of insulin blocked by the circuit in the glucose absence in different  Kdsrna values.</P><P>The first graphic shows that the sRNA Blocking efficiency decreases as Kdsrna increases and the high BEs happens when the sRNA affinity is lower than 0.1. </P><P>
After plotting graphics in lower Kdsrna values, we were able to reach the best affinity constant in order to reach the highest insulin difference between active (glucose ON) and inactive (glucose OFF) circuit and run our simulation.
</P><H2>Simulations</H2><H3>Glucose ON and OFF</H3><P>
We first ran our circuit response in glucose presence and absence</P><P>
This results provided the circuit time response around 5.8 minutes reaching steady-state after 66 minutes.</P><H3>Different concentrations</H3><P>
We tested our circuit response to different glucose concentrations, as shown in the graphic below:</P><P>
Different concentrations of glucose don’t make a relevant difference in the insulin amount according to our model. This is an undesirable characteristic in our circuit since we want our system to respond proportionally to the glucose amount. This results can be explained due to our oversimplification of the CcpA activation by glucose that, in reality, involves active transport and different enzymatic conversion thought the chassis metabolism before the activation of CcpA itself. Avoiding this steps turns glucose into a simple inductor of the circuit and ignores the fact that it participates actively in the chassis metabolism and growth, which may cause the model to exhibit variations when compared to experimental data.
For future improvements of this mathematical modeling, we could add the glucose uptake and glucose participation in metabolism in order to fix this problem.
</P><H3>References</H3><P style="font-size:1em">
Ciarán L Kelly, Andreas W K Harris, Harrison Steel, Edward J Hancock, John T Heap, Antonis Papachristodoulou; Synthetic negative feedback circuits using engineered small RNAs, Nucleic Acids Research, Volume 46, Issue 18, 12 October 2018, Pages 9875–9889, https://doi.org/10.1093/nar/gky828

S Legewie, H Herzel, H Westerhoff, and N Bluthgen. Recurrent design patterns in the feedback regulation of the mammalian signaling network. Molecular Systems Biology. 2008. 4:190 supplementary material section II pp.6-7 table

Bintu L, Buchler NE, Garcia HG, Gerland U, Hwa T, Kondev J, Phillips R. Transcriptional regulation by the numbers: models. Curr Opin Genet Dev. 2005 Apr15(2):116-24. p.118 caption of fig.1c

Emmer, M., deCrombrugghe, B., Pastan, I., and Perlman, R. (1970). Cyclic AMP receptor protein of E. coli: its role in the synthesis of inducible enzymes. Proc. Natl. Acad. Sci. U. S. A. 66, 480–487. p.486 table 4

Stekel, D. J., &amp; Jenkins, D. J. (2008). Strong negative self regulation of prokaryotic transcription factors increases the intrinsic noise of protein expression. BMC systems biology, 2(1), 6. 

Kamionka, A., Sehnal, M., Scholz, O., &amp; Hillen, W. (2004). Independent regulation of two genes in Escherichia coli by tetracyclines and Tet repressor variants. Journal of bacteriology, 186(13), 4399-4401.

I.D.J.Burdett et. al. Growth kinetics of individual Bacillus subtilis cells and correlation with nucleoid extension. J Bacteriol. 1986 Jul 167(1): 219–230. abstract &amp; p.221 right column 2nd paragraph

HUA, Qing-xin et al. Design of an Active Ultrastable Single-chain Insulin Analog SYNTHESIS, STRUCTURE, AND THERAPEUTIC IMPLICATIONS. Journal of Biological Chemistry, v. 283, n. 21, p. 14703-14716, 2008.

Duckworth WC, Bennett RG, Hamel FG. Insulin degradation: progress and potential. Endocr Rev. 1998 Oct19(5):608-24

Marciniak, B. C., Pabijaniak, M., de Jong, A., Dűhring, R., Seidel, G., Hillen, W., &amp; Kuipers, O. P. (2012). High-and low-affinity cre boxes for CcpA binding in Bacillus subtilis revealed by genome-wide analysis. BMC genomics, 13(1), 401.</P></DIV><DIV id="submenus"><DIV id="sublinks2"><UL><LI><A href="https://2018.igem.org/Team:Unesp_Brazil/Model/Bioinformatics">Bioinformatics</A></LI></UL></DIV></DIV></DIV><DIV id="conteudoft"><DIV id="sponsors"><H2>SPONSORS</H2><P style="text-align:center; color:white; font-family: 'Raleway', sans-serif; font-size:1.3em"><B>School of Pharmaceutical Sciences | Chemistry Institute</B></P></DIV><DIV id="find-us" style="text-align: center"><H2>CONTACT US</H2><P style="color: white; text-align:center; font-family: 'Raleway', sans-serif; font-size:1.3em">
School of Pharmaceutical Sciences
São Paulo State University (UNESP)
Rodovia Araraquara Jaú, Km 01 - s/n
Campos Ville
14800-903
Araraquara, São Paulo, Brazil</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>