<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Unesp_Brazil_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Unesp Brazil/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><H1 class="logo">iGEM Unesp Brazil</H1><NAV class="menuCristo"><LABEL for="toggle">☰</LABEL><DIV class="menuCaramba"><A href="https://2018.igem.org/Team:Unesp_Brazil/Description">PROJECT</A><A href="https://2018.igem.org/Team:Unesp_Brazil/Model">MODEL</A><A href="https://2018.igem.org/Team:Unesp_Brazil/Parts">PARTS</A><A href="https://2018.igem.org/Team:Unesp_Brazil/Hardware">HARDWARE</A><A href="https://2018.igem.org/Team:Unesp_Brazil/Human_Practices">HUMAN PRACTICES</A><A href="https://2018.igem.org/Team:Unesp_Brazil/Team">TEAM</A><A href="https://2018.igem.org/Team:Unesp_Brazil/Collaborations">COLLABORATIONS</A><A href="https://2018.igem.org/Team:Unesp_Brazil/Safety">SAFETY</A><A href="https://2018.igem.org/Team:Unesp_Brazil/InterLab">INTERLAB</A><A href="https://igem.org/2018_Judging_Form?id=2660" target="_blank">FOR JUDGES</A></DIV></NAV><DIV id="bg-c"><DIV id="faixa"><H1>Design</H1></DIV></DIV><DIV id="designproject"><DIV id="designproject-r"><H2 style="font-size:2em">Plasmids and strains</H2><P style="font-family: 'PT Serif', sans-serif; font-size: 1.3em; line-height: 1.9em; text-align: left; letter-spacing: 0.3px;"><I>Bacillus subtilis</I> is one of the most used chassis in synthetic biology and biotechnology, with a great toolbox available for its manipulation. Besides, it has been show that <I>B. subtilis</I> can be a probiotic and live in human microbiome. Thus, <I>B. subtilis</I> is the chassis we chose to host our framework, Hope.
Another important question that we thought carefully was what plasmids to use. We chose to work with the integrative plasmid <B>pBs1C</B>, a BioBrick standardized vector for <I>B. subtilis</I> that integrates at the <I>amyE</I> locus. Integrative plasmids are a good choice in our project because they offer more genetic stability when compared to replicative plasmids. However, in some situations that we needed to highly express a gene, we used the replicative plasmid pLike.
</P></DIV><DIV id="submenus"><DIV id="sublinks2"><UL><LI><A href="https://2018.igem.org/Team:Unesp_Brazil/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:Unesp_Brazil/Demonstrate">Results</A></LI><LI><A href="https://2018.igem.org/Team:Unesp_Brazil/Notebook">Notebook</A></LI></UL></DIV></DIV></DIV><DIV id="conteudo"><DIV id="texto" style="width:90%; margin: 0 auto; padding: 1%;"><H2 id="sense" style="color:#F2545B">Sense and control</H2><P>
In order to create a robust, modular and orthogonal regulatory genetic circuit, we designed, built and tested an improved version of our previous circuit. The original circuit was based on a small RNA and well-characterized and standardized parts. The re-designed version regulates gene expression in both transcriptional and translational level in response to the presence or absence of glucose. We used some parts designed for <I>Lactococcus lactis</I> last year. Therefore, we tested whether they are modular enough to work in <I>Bacillus subtilis</I>. 

First, to detect glucose, we used the gal promoter (<A href="”http://parts.igem.org/Part:BBa_K2270005”">BBa_K2270005</A>) from <I>Lactococcus lactis</I> that is regulated by carbon catabolite repression (CCR). In gram-positive bacteria, the process is mediated by CcpA (catabolite control protein A) and HPr (histidine protein). When concentrations of fructose-1,6-bisphosphate and ATP are high enough, reflecting the presence of glucose or other preferred carbon sources, HPr is phosphorylated and binds to CcpA. The complex CcpA-HPr(Ser-P) binds to the cre (catabolite repression element) site at the DNA and represses the transcription of the downstream genes. In other words, the absence of glucose activates the gal promoter.

Next, we used a regulatory small RNA designed last year for <I>Lactococcus lactis</I>.  The sRNA transcription was placed under the control of Pgal (<A href="http://parts.igem.org/Part:BBa_K2270008”">BBa_K2270008</A>) and the transcriptional regulator TetR (<A href="”http://parts.igem.org/Part:BBa_C0040”">BBa_C0040</A>) under the control of the same promoter.

In our circuit, TetR repressor activates the production of T7 RNA polymerase and, which in turn, transcribes our protein of interest. Therefore, we built a composite for biosynthesis of the T7 RNA polymerase regulated by TetR (<A href="”http://parts.igem.org/Part:BBa_K2660005”">BBa_K2660005</A>). Another composite places the protein reporter mCherry under control of the T7 promoter (<A href="”http://parts.igem.org/Part:BBa_K2660011”">BBa_K26600011</A>). Placing all parts together, we constructed a NIMPLY logic gate that displays a TRUE output only when there is T7 RNA polymerase but no sRNA. 
</P><P>
In the absence of glucose, there is TetR production and sRNA expression. TetR represses the synthesis of the T7 RNA polymerase and thus the production of mCherry. To cope with T7 RNA polymerase leakage that would lead to undesired mCherry synthesis, we used the sRNA to block mCherry translation through hybridization with the mRNA. When there is glucose, Pgal will be repressed and turn OFF the expression of TetR and sRNA, that way, T7 RNA polymerase will be produced and, therefore, mCherry. 

Our circuit is modular and can be adapted to respond to other molecules by just exchanging the promoter regulating TetR and the small RNA. 
</P><H2 id="production" style="color:#00272B">Production</H2><P>
The single-chain insulin analog SCI-57 designed by HUA et al (2008) was chosen because it can be produced at its final conformation. The peptide does not require post-translational modifications, allowing for its production in prokaryotic microorganisms such as <I>Bacillus subtilis. </I>
               The expected function of our probiotic machine includes the ability to harmonically colonize the natural gut microbiome. Last year, <A href="https://2017.igem.org/Team:AQA_Unesp/Results" target="_blank">we proved</A> that engineered <I>B. subtilis</I> could survive and grow in a simulated gut environment. After that point, the treatment process should follow two steps: (I) insulin transport by <I>B. subtilis</I> to the extracellular environment; (II) insulin absorption by the epithelial tissue of the intestine. 
 
          Exportation to the extracellular environment will be driven by the signal peptide yncM fused to our modified insulin. Passage of our modified insulin through the intestinal epithelial tissue to reach the bloodstream will be driven by penetratin. The latter is a cell penetration peptide (CPP) capable of carrying other peptides or proteins through cell membranes. Penetratin was chosen because of the results of Kamei et al (2013), which achieved a 35% increase in the permeability of insulin when it was co-administered to the CPP. Our modified insulin is fused to penetratin at the N-terminus.
</P><P>
               We also improved the basic part <A href="http://parts.igem.org/Part:BBa_I746907" target="_blank">BBa_I746907</A>, developed by the Cambridge team (iGEM 2007), by adding the penetratin sequence (BBa_K2660000) to the GFP N-terminus. Our final composite carries the T7 promoter to drive the expression of the penetratin-GFP gene.
 
               Penetratin_GFP (BBa_K2660001) was used in permeation assay through a monolayer of Caco-2 cells to mimic the absorption in the small intestine of a cargo protein driven by penetratin.Both Penetratin-Insulin and Penetratin-GFP proteins carry a histidine tag (histag) to for purification and western-blot assays.

			</P><H2 id="kill" style="color:#0195E1">Biocontainment</H2><H3 style="color:#0195E1"><I>Light-Activated split-Cas9 Kill switch</I></H3><P>
		Currently, physical containment methods such as bioreactors are considered the first safety level. However, this approach is not suitable for engineering probiotics. Alternative methods include auxotrophy and kill switches. Auxotrophy can be easily bypassed by external supply or by horizontal gene transfer. Moreover, it affects cell growth and may jeopardize the designed function (e.g. insulin production and delivery). Therefore, we combined a highly effective approach (nuclease-based kill switch) with photo-switches for optogenetic control of the kinetic states ON/OFF. By doing that, we hope to build a photoactivatable kill switch with low-frequency scape that can be adapted to other living therapeutics changing the Cas9 targets (gRNAs). 

Circuit construction was performed in 2 steps: (I) choose the photo-switch proteins; (II) construct the split-Cas9 nuclease device. 
<B>(I)  Photo-switches</B>

Photo-switches are natural receptors from plants, fungi, and bacteria. Last year, based on <A href="”https://2016.igem.org/Team:NEU-China/Design&quot;">NEU-China 2016 team idea </A>, we propose a system using the light sensors CRY2 and CIB1. This year, we wanted to use the blue light photoreceptor VVD from <I>Neurospora crassa.</I> VVVD proteins are one of the smallest photoreceptors already tested, and the engineered variants named Magnets (pMag and nMag) can be engineered to a large range of kinetics response (seconds to hours). Positive (<A href="”http://parts.igem.org/wiki/index.php?title=Part:BBa_K2660009”">BBa_K2660009</A>) and negative (<A href="”http://parts.igem.org/wiki/index.php?title=Part:BBa_K2660008”">BBa_K2660008</A>) magnets, or pMag and nMag, dimerize in response to blue light through electrostatic interactions. These parts have already been used for Cas9 genome editing in biomedical applications, such as gene therapy.
<B>(II) Split Cas9</B>
The Cas9 provides a way to kill cells by targeting its nuclease activity to essential genes. However, any basal expression of Cas9 can be deathful to a cell [4]. Therefore, we designed a split version of Cas9 (<A href="”http://parts.igem.org/Part:BBa_K2457001”">BBa_K2457001</A>) to construct the split-cas9. In order to create 2 inactive parts, we split Cas9 into the N-terminal (<A href="”http://parts.igem.org/Part:BBa_K2660006”">BBa_K2660006</A>)  and C-terminal (<A href="”http://parts.igem.org/Part:BBa_K2660007”"> BBa_K2660007</A>) domains. Then, we added linkers to join these domains to the photo-switch proteins pMag or nMag. 
			</P><P>
We designed gRNAs against multiple targets to eliminate recombinant DNA and kill the bacterium. As proof of concept, we first targeted GFP and mCherry, and essential genes like DNA ligase, RNA polymerase subunit beta (rpoC) and DNA polymerase III beta subunit (dnaN) as shown below: 
</P><H3>References</H3><P style="font-size:1em">
GÖRKE, B.; STÜLKE, J. Carbon catabolite repression in bacteria: Many ways to make the most out of nutrients. Nature Reviews Microbiology, v. 6, n. 8, p. 613–624, 2008. 
HUA, Qing-xin et al. Design of an Active Ultrastable Single-chain Insulin Analog SYNTHESIS, STRUCTURE, AND THERAPEUTIC IMPLICATIONS. Journal of Biological Chemistry, v. 283, n. 21, p. 14703-14716, 2008.
GUAN, Chengran et al. Construction of a highly active secretory expression system via an engineered dual promoter and a highly efficient signal peptide in Bacillus subtilis. New biotechnology, v. 33, n. 3, p. 372-379, 2016.
KAMEI, Noriyasu et al. Noninvasive insulin delivery: the great potential of cell-penetrating peptides. Therapeutic delivery, v. 4, n. 3, p. 315-326, 2013.
KRISTENSEN, Mie et al. Penetratin-mediated transepithelial insulin permeation: Importance of cationic residues and pH for complexation and permeation. The AAPS journal, v. 17, n. 5, p. 1200-1209, 2015.
CRISTOPHER, M. et al. Auxotrophy to Xeno-DNA: an exploration of combinatorial mechanisms for a high-fidelity biosafety system for synthetic biology applications. J Biol Eng. 2018; v.12 n.13, 2018. 
KAWANO, F. et al. Engineered pairs of distinct photoswitches for optogenetic control of cellular proteins. Nat. Commun. v.6 n.6256, 2015.  
NIHONGAKI, Y. et al.Photoactivatable CRISPR-Cas9 for optogenetic genome editing. Nature Biotechnology v.33 p.755, 2015. 
VOIGT, C;CALIANDO, B. Targeted DNA degradation using a CRISPR device stably carried in the host genome. Nature Communications v.6  p.6989, 2015.</P><P style="text-align: center;"><A href="#texto">back to top ▲</A></P></DIV></DIV><DIV id="conteudoft"><DIV id="sponsors"><H2>SPONSORS</H2><P style="text-align:center; color:white; font-family: 'Raleway', sans-serif; font-size:1.3em"><B>School of Pharmaceutical Sciences | Chemistry Institute</B></P></DIV><DIV id="find-us" style="text-align: center"><H2>CONTACT US</H2><P style="color: white; text-align:center; font-family: 'Raleway', sans-serif; font-size:1.3em">
School of Pharmaceutical Sciences
São Paulo State University (UNESP)
Rodovia Araraquara Jaú, Km 01 - s/n
Campos Ville
14800-903
Araraquara, São Paulo, Brazil</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>