"
Page
Discussion
View source
History
teams
Log in
 
Team:UTK-Knoxville/Project
From 2012.igem.org
Home
Team
Project
Safety
Notebook
Parts
Attributions
Project Overview
Abstract
Project Details
The Experiments
Results
IRESistible: Novel Parts for Use in S. cerevisiae
Synthetic biology has become a staple of chemical engineering, used to produce protein products, biofuels and many other important compounds. Bacteria such as Escherichia coli can be easily manipulated to produce a desired product and to do so efficiently. However it is often advantageous to use a eukaryotic host, such as Saccharomyces cerevisiae. While many techniques have been developed for the genetic manipulation of S. cerevisiae, there are still many challenges with this work, including the difficulty of creating multicistronic constructs so that multiple genes can be under the control of a single promoter.
Cap Dependent Translation Initiation
In eukaryotes, translation initiation is typically dependent on a 7 methyl guanisine cap, located at the 5’ end of the mRNA, upstream of an open reading frame. This cap interacts with the poly-adenine tail of the mRNA and with several eukaryotic initiation factors (eIFs) to recruit the ribosome to the mRNA, allowing translation to begin. This mechanism is known as cap-dependent translation initiation. At the end of the open reading frame, secondary structure in the mRNA causes the ribosome to disassociate, meaning that a second open reading frame in the same piece of mRNA will not be translated. As a result, every gene must have its own promoter. This presents many problems to synthetic biologists. Promoters are large and can greatly increase plasmid size. In addition, using the same promoter for several genes in a pathway can result in recombination events that can delete pieces of the plasmid. Moreover, using different promoters for each gene can cause flux balance issues, as every promoter is transcribed differently.  Luckily, nature has come up with its own remedy for this problem, known as an internal ribosomal entry site (IRES). IRESs are highly structured regions of mRNA located in the untranslated region between two open reading frames. This secondary structure works with primary structure motifs to recruit eIFs to the mRNA, resulting in the binding of the ribosome. This allows for translation of the second gene in a multicistronic operon and is known as IRES mediated translation initiation.
IRES Mediated Translation Initiation
Although IRESs are an important part of eukaryotic cloning, they are still poorly understood and poorly characterized. The UTK-Knoxville iGEM team is attempting to remedy that problem. We have selected seven IRESs, including two commercial IRESs and five from S. cerevisiae genomic DNA. Based on the methodology put forth by Kelly et al in the Journal of Biological Engineering, we will characterize these IRESs to determine a the relative strengths (Kelly et al., 2001). This will be accomplished using a constitutive promoter, two fluorescent proteins, an IRES, and a terminator built into a single construct as shown below.  By comparing the fluorescence produced under the control of each IRES, we can create a relative strength profile.
Retrieved from " http://2012.igem.org/Team:UTK-Knoxville/Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
