<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SZU-China_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SZU-China/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Procedure</TITLE></P><DIV class="navbar" style="top:5px;position:fixed;z-index:5"><DIV class="navbar-nav"><UL id="nav-list-ul" class="nav-list"><LI><A href="#">TEAM</A></LI><LI><A href="https://2017.igem.org/Team:SZU-China/Team">MEMBERS</A></LI><LI><A href="https://2017.igem.org/Team:SZU-China/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2017.igem.org/Team:SZU-China/Attributions">ATTRIBUTION</A></LI></UL><LI><A href="#">PRACTICE</A></LI><LI><A href="https://2017.igem.org/Team:SZU-China/HP/Gold_Integrated">INTEGRATED HP</A></LI><LI><A href="https://2017.igem.org/Team:SZU-China/HP/Silver">SILVER HP</A></LI><LI><A href="https://2017.igem.org/Team:SZU-China/Engagement">ENGAGEMENT</A></LI><LI><A href="#">EXPERIMENT</A></LI><LI><A href="https://2017.igem.org/Team:SZU-China/Procedure">PROCEDURE</A></LI><LI><A href="https://2017.igem.org/Team:SZU-China/Results">RESULTS</A></LI><LI><A href="https://2017.igem.org/Team:SZU-China/Demonstrate">DEMONSTRATE</A></LI><LI><A href="https://2017.igem.org/Team:SZU-China/Protocol">PROTOCOL</A></LI><LI><A href="https://2017.igem.org/Team:SZU-China/Notebook">NOTEBOOK</A></LI><LI><A href="#">PROJECT</A></LI><LI><A href="https://2017.igem.org/Team:SZU-China/Description">DESCRIPTION</A></LI><LI><A href="https://2017.igem.org/Team:SZU-China/Design">DESIGN</A></LI><LI><A href="https://2017.igem.org/Team:SZU-China/Model">MODEL</A></LI><LI><A href="https://2017.igem.org/Team:SZU-China/Safety">SAFTY</A></LI><LI><A href="#">ACHIEVEMENT</A></LI><LI><A href="https://2017.igem.org/Team:SZU-China/Achievement">MEDAL</A></LI><LI><A href="https://2017.igem.org/Team:SZU-China/Parts">PARTS</A></LI></DIV></DIV><DIV id="page" style="position:relative;top:100px;width:100%"><SECTION id="overview" style="padding:96px 0;background-color:white;"><DIV class="container"><DIV class="row"><DIV class="col-sm-12"><H3 class="uppercase color-primary mb40 " style="margin-bottom: 40px;font-size:50px"><CENTER>Results</CENTER></H3><P class="lead mb40" style="width:60%;margin:0 auto">In our project this year, we constructed different expression vectors containing our parts submitted and transformed into<I> Bacillus subtilis</I> , to increase the alkali-tolerance, germination rate and mineralization capacity of <I> B.subtilis</I> . Therefore, we conducted experiments on three aspects and the experimental characterizations of our parts are shown as follow.</P></DIV></DIV></DIV></SECTION><SECTION class="cd-section" style="padding:96px 0;background-color:rgba(245,245,245,0.45)"><DIV class="container"><DIV class="row"><DIV class="col-sm-12"><H3 class="color-primary mb40 mb-xs-24" style="margin-bottom: 40px;"><CENTER>THE RESPONDER: PsspB-gerAa</CENTER></H3><DIV class="row text-center" style="width:85%;margin:0 auto;text-align:justify"><P class="lead" style="color:black;">This part，PsspB-gerAa (<A href="http://parts.igem.org/Part:BBa_K2232020" style="font-size:14px;color:black;">BBa_K2232020</A>) mainly of the PsspB promoter followed by the first 303bp of the gerA gene, plays the role as Responder in the germination of spores.</P><P class="lead" style="color:black;">
                                The gerA from <I> Baillus subtilis</I>  encodes the spore’s germinant receptor for L-alanine. Overexpression of this gene will increase markedly the rate of germination of <I> B. subtilis</I>  spores even at low L-alanine concentration.
                                The promoter for the <I>B.subtilis </I>sspB gene (PsspB) is stronger than the endogenous gerA promoter, and only switch on the transcription after the initiation of sporulation.
                                Since <I>B.subtilis</I> exhibits accurate and efficient homologous recombination, a single cross-over event between the gerAa region in the BioBrick and the endogenous gerA sequence inserts the PsspB promoter upstream of the gerA cds, which increases the germination rate of <I>Bacillus subtilis</I> endospores eventually.
                            </P><P class="lead" style="color:black;">
                                The part was synthesized and insert into the expression vector by restriction sites BamHI and HindIII（Fig.1）, and the correct construction of this recombinant plasmid was confirmed by PCR identification and sequencing of the PCR products.
                            </P><CENTER style="padding-top: 5px; color: #2f2f2f; font-size: 14px; width: 60%; margin: 0 auto; text-align: justify"><STRONG>Fig.1 </STRONG> Construction of the expression vector_PsspB-gerAa. The Rep_B and Kan represents replicon of <I> B.subtilis</I>  and kanamycin resistance marker. The part PsspB-gerAa was inserted by the restriction site BamHI at 4151 bp and HindIII at 4652 bp</CENTER><P class="lead" style="color:black">We transformed the expression vectors into <I> Bacillus subtilis WB800</I>  by optimization of Spizizen method, and the real positive clones was confirmed by kanamycin screening and nucleic acid electrophoresis (Fig.2).</P><CENTER style="padding-top:5px;color:#2f2f2f;font-size:14px;width:60%;margin:0 auto;text-align:justify"><STRONG> Fig.2</STRONG> 1% Agarose Gel Electrophoresis of DNA extracted from the positive clones and its identification by restriction digestion. The product of plasmid digested showed two signal bands at 501 bp and 6393 bp respectively, which correspond to the length of PsspB-gerAa and the blank plasmid.
                                Lane 1: Complete plasmid; Lane 2: Plasmid digested by BamHI and HindIII; Lane M: DL marker.
                            </CENTER><P class="lead" style="color:black">In order to test the germination rate of the <I>B.subtilis endospores</I>, we used the whole yeast solid plate (Fig.3) and the 2xSG liquid medium (Fig.4) to harvest the mature spores, The yield of spores can be calculate by phase contrast microscope using the Hemocytometer，which always reach 90% in 72h after cultivating（Fig.5)</P><DIV style="width:80%;margin:0 auto"><DIV style="text-align:center;width:50%;float:left;"><CENTER style="padding-top:5px;color:#2f2f2f;font-size:14px;width:90%;margin:0 auto"><STRONG>Fig.3 </STRONG>The whole yeast solid plate.</CENTER></DIV><DIV style="text-align: center; width: 50%; float: left;"><CENTER style="padding-top:5px;color:#2f2f2f;font-size:14px;width:90%;margin:0 auto"><STRONG>Fig.4 </STRONG>2xSG liquid medium</CENTER></DIV></DIV><CENTER style="padding-top: 5px; color: #2f2f2f; font-size: 14px; width: 60%; margin: 0 auto; text-align: justify"><STRONG>Fig.5</STRONG> The endospores of <I> B.subtilis</I>  observed under the phase contrast microscope. Sporation rate always reach 90% above in 72h after cultivating at the whole yeast solid plate and 2xSG liquid medium.</CENTER><P class="lead" style="color:black">
                                To verify the increases in the germination rates with L-alanine of spores transformed this part PsspB-gerAa (<A href="http://parts.igem.org/Part:BBa_K2232020" style="font-size:14px;color:black;"> BBa_K2232020</A>), both spores of original strain (control group) and transformed strain (gerAa group) were collected and detected the germination rates  induced by L-alanine(2mmol/L). Germination was monitored by reading the drop in absorbance (A600) in a 96-well microplate reader (Molecular Devices FlexStation 3).Theoretically, the earliest easily measured event in spore germination is the release of various ions, and Ca2+-DPA release is accompanied by the loss of 70% of the total amount of the OD600 that is lost upon spore germination. Consequently, measurement of the OD600 of germinating spores, in particular, the maximum rate of the fall of the OD600 at a given germinant concentration, is a simple and reliable method for quantitating and comparing rates of spore germination. As shown in Fig.6, the gerAa group showed a higher proportion in germination and quicker rate in germination than the control group, which indicated the gerA play a good <STRONG>Responder</STRONG> in germination to L-analine in particular at low concentration. Therefore, gene gerAa is verified to be in good condition and can work efficiently as repected.
                            </P><DIV style="width:80%;margin:0 auto"><CENTER style="padding-top:5px;color:#2f2f2f;font-size:14px;width:90%;margin:0 auto;text-align:justify"><STRONG>Fig.6</STRONG> Variation of values and rates of loss in OD600 during germination of spores with or without transformation of PsspB-gerAa. The results represent the average (SD) of three independent spore batches The drop rate of OD600 is proportional to the germination rate and the value of gerAa group is more than twice of control group with significant difference in statistics.</CENTER></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION class="cd-section" style="padding:96px 0;background-color:white"><DIV class="container"><DIV class="row"><DIV class="col-sm-12"><H3 class="color-primary mb40 mb-xs-24" style="margin-bottom: 40px;"><CENTER>THE SHELTER: TupA &amp; nhaC</CENTER></H3><DIV class="row text-center" style="width:85%;margin:0 auto;text-align:justify"><P class="lead" style="color:black;">In order to strengthen the alkali tolerance of B.subtilis, which is essential for our chassis to live in the concrete, we constructed two expression vectors containing parts C125-TupA(<A href="http://parts.igem.org/Part:BBa_K2232020">BBa_K2232011</A>) and OF4-nhaC(<A href="http://parts.igem.org/Part:BBa_K2232020">BBa_K2232012</A>)(Fig.8), both of them play a role as Shelter protecting<I>  B.subtilis</I>  from Alkaline environment.</P><CENTER style="padding-top: 5px; color: #2f2f2f; font-size: 14px; width: 60%; margin: 0 auto; text-align: justify"><STRONG>Fig.7</STRONG> Construction of the expression vector_P43-tupA-T1. The P43 and T1 represent strong promoter P43 from <I> B.subtilis</I>  and terminator T1 from E. coli rrnB. The part C125-tupA was inserted by the restriction site KpanI at 4430 bp and HindIII at 6001 bp.</CENTER><CENTER style="padding-top: 5px; color: #2f2f2f; font-size: 14px; width: 60%; margin: 0 auto; text-align: justify"><STRONG>Fig.8</STRONG> Construction of the expression vector_P43-nhaC-T1. The P43 and T1 represent strong promoter P43 from <I> B.subtilis</I>  and terminator T1 from E. coli rrnB. The part OF4-nhaC was inserted by the restriction site KpanI at 4430 bp and HindIII at 5194 bp.</CENTER><P class="lead" style="color:black;">
                                We transformed the expression vectors into <I> Bacillus subtilis WB800</I>  by optimization of Spizizen method, and the real positive clones was confirmed by kanamycin screening and nucleic acid electrophoresis (Fig.9, Fig.10).
                            </P><DIV style="width:80%;margin:0 auto;"><DIV style="text-align: center; width:50%; float: left"><CENTER style="padding-top: 5px; color: #2f2f2f; font-size: 14px; width: 90%; margin: 0 auto; text-align: justify"><STRONG>Fig.9</STRONG> 1% Agarose Gel Electrophoresis of DNA extracted from the positive clones and its identification by restriction digestion. The product of plasmid digested showed two signal bands at 733 bp and 6703 bp respectively, which correspond to the length of OF4-nhaC and the blank plasmid.                                    Lane 1: Complete plasmid; Lane 2: Plasmid digested by KpnI and HindIII; Lane M: DL marker.
                                    </CENTER></DIV><DIV style="text-align: center; width: 50%;  float: left"><CENTER style="padding-top: 5px; color: #2f2f2f; font-size: 14px; width: 90%; margin: 0 auto; text-align: justify"><STRONG>Fig.10</STRONG> 1% Agarose Gel Electrophoresis of DNA extracted from the positive clones and its identification by restriction digestion. The product of plasmid digested showed two signal bands at 1540 bp and 6724 bp respectively, which correspond to the length of C125-tupA and the blank plasmid.
                                        Lane 1: Complete plasmid; Lane 2: Plasmid digested by KpnI and HindIII; Lane M: DL marker.
                                    </CENTER></DIV></DIV><P class="lead" style="color:black;">
                                The first part OF4-nhaC(<A href="http://parts.igem.org/Part:BBa_K2232012" style="font-size:14px;color:black;">BBa_K2232012</A>) is the Na+/H+ antiporter coding sequence (CDS) from the <I>Bacillus pseudofirmus OF4 </I>(GenBank Acc.No. CP001878), which is a kind of <I> alkaliphilic Bacillus</I>  that grows in a pH range from 7.5 to above 11.4. The Na+/H+ antiporter NhaC, encoded by gene nhaC are sub-membrane transport protein distributed in the cell membrane, which play a key role in regulating cytoplasmic pH value by coupling net H+ uptake with Na+ extrusion.
                            </P><P class="lead" style="color:black;">
                                After transformation of this part, The recombinant <I>B.subtilis WB800</I> nhaC was grown in LB culture for 24h and the cells were disrupted by sonication in 20 mM Tris-HCl(PH 8.0) buffer. The lysate was then centrifuged and the precipitate were electrophoresed on a sodium dodecyl sulfate(SDS)-12% (wt/vol) polyacrylamide gel, followed by Coomassie blue staining(Fig.11).
                            </P><CENTER style="padding-top: 5px; color: #2f2f2f; font-size: 14px; width: 50%; margin: 0 auto; text-align: justify"><STRONG>Fig.11</STRONG> SDS-PAGE analysis of membrane protein of original <I> B.subtilis</I>  and the transformant of OF4-nhaC. Lane M: Marker ladder; Lane 1 &amp; 2: The recombinant strain <I> WB800</I> _nhaC; Lane 3: Original strain WB800. Lane 1 &amp; 2 showed the same band(in red box) corresponded with the molecular weight of NhaC(36kDa).</CENTER><P class="lead" style="color:black">The second part C125-tupA(<A href="http://parts.igem.org/Part:BBa_K2232011" style="font-size:14px;color:black;">BBa_K2232011</A>) is involved in the synthesis of TUP, a copolymer of polyglutamic acid (PGlu) and polyglucuronic acid (PGlcU),which is one of major structural components in the cell wall of the <I> Bacillus </I> lentus C-125 and can neutralize the extracellular hydroxyl.</P><P class="lead" style="color:black">We used the alkaline solid medium containing Na2CO3 to plant the transformed strain <I>WB800</I>_tupA and the original strain <I>WB800</I>. After 48hours, the plates of same pH value were dyed with crystal violet followed by washing with ddH₂O and compared the size of bacteria ring(Fig.12), which indicates the function of tupA to increase the alkali tolerance of <I>B.subtilis</I>.</P><CENTER style="padding-top:5px;color:#2f2f2f;font-size:14px;width:90%;margin:0 auto;text-align:justify"><STRONG>Fig.12</STRONG>  Growth situation of original and tramsformed strains at different pH. The bacteria rings of <I> WB800</I> _tupA are always bigger than original<I>  WB800</I> , especially at pH 9, which indicated the function of gene tupA as respected.
                            </CENTER><P class="lead" style="color:black">Meanwhile, Strain transformed and original WB800 were plated at 37°C in the alkaline liquid medium (pH = 9.0) simultaneously, then we recorded the OD600 of them at the same time to compare their growth rates (Fig.13).</P><DIV style="text-align:center;width:60%;margin:0 auto"><CENTER style="padding-top:5px;color:#2f2f2f;font-size:14px;width:100%;margin:0 auto;text-align:justify"><STRONG>Fig.13</STRONG>  OD600 of various strains in the alkaline liquid medium (pH = 9.0)
                                    At the same time, the OD600 of recombinant <I> B.subtilis WB800</I> _nhaC and <I> WB800</I> _tupA always exceed the original strain WB800, which proves the effect of two genes to some extent.
                                </CENTER></DIV></DIV></DIV>-
                </DIV></DIV></SECTION><SECTION class="cd-section" style="padding:96px 0;background-color:rgba(245,245,245,0.45)"><DIV class="container"><DIV class="row"><DIV class="col-sm-12"><H3 class=" color-primary mb40 mb-xs-24" style="margin-bottom: 40px;"><CENTER>THE HEALER: Carbonic anhydrase</CENTER></H3><DIV class="row text-center" style="width:85%;margin:0 auto;text-align:justify"><P class="lead" style="color:black;">To realize the self-healing of cracks in concrete, we need to increase the mineralization capacity of <I> B.subtilis</I> . The Healer in our project is Carbonic anhydrase(CA) , which catalyzes the hydration of CO2 to produce HCO3- and captures free Ca2+ with OH- in the environment to form Calcium carbonate precipitation.</P><P class="lead" style="color:black;">We have two parts encoding CA, including the TSLV1-CA (<A href="http://parts.igem.org/Part:BBa_K2232014" style="font-size:14px;color:black;">BBa_K2232014</A>) and the OF4-CA(<A href="http://parts.igem.org/Part:BBa_K2232015" style="font-size:14px;color:black;">BBa_K2232015</A>) both express and function intracellularly. We constructed  two shuttle vector to transform these two parts and the positive clones was confirmed by nucleic acid electrophoresis (Fig.14,15).</P><P class="lead" style="color:black;">The crude enzyme solution was obtained by cell disruption using ultrasonic, followed by SDS-PAGE protein electrophoresis and Coomassie blue staining (Fig.16).</P><DIV style="width:80%;margin:0 auto;"><DIV style="text-align: center; width:50%; float: left"><CENTER style="padding-top: 5px; color: #2f2f2f; font-size: 14px; width: 90%; margin: 0 auto; text-align: justify"><STRONG>Fig.14</STRONG> 1% Agarose Gel Electrophoresis of Vector_ TSLV1-CA and its identification by restriction digestion. Lane 1: Complete plasmid; Lane 2: Plasmid digested by KpnI and HindIII; Lane M: DL marker.The length of part TSLV1-CA was 949 bp and the blank vector was 6785 bp.
                                    </CENTER></DIV><DIV style="text-align: center; width: 50%;  float: left"><CENTER style="padding-top: 5px; color: #2f2f2f; font-size: 14px; width: 90%; margin: 0 auto; text-align: justify"><STRONG>Fig.15</STRONG> 1% Agarose Gel Electrophoresis of Vector_ OF4-CA and its identification by restriction digestion. Lane 1: Complete plasmid; Lane 2: Plasmid digested by KpnI and HindIII; Lane M: DL marker.The length of part OF4-CA was 952 bp and the blank vector was 6785 bp.
                                    </CENTER></DIV></DIV><CENTER style="padding-top: 5px; color: #2f2f2f; font-size: 14px; width: 60%; margin: 0 auto; text-align: justify"><STRONG>Fig.16</STRONG>  SDS-PAGE analysis of endocellular protein of original<I> B.subtilis</I>  and the transformant of CA.
                                Lane M: Marker ladder; Lane 1: Genetic Modified strain<I>  WB800</I> _ TSLV1-CA; Lane 2: Modified strain WB800_ OF4-CA; Lane 3: Original strain<I>  WB800</I> . Lane 1 and lane 2 have a band of 35~37kd respectively (in red box), which correspond with molecular weight of TSLV1-CA (35kDa) and OF4-CA (34.5kDa).
                            </CENTER><P class="lead" style="color:black;">
                                For determining the activity of CA, hydration of CO2 was measured using electrometric Wilbur–Anderson assay according to Khalifah et al. (1991) with certain modifications.
                                The assay was performed at 4 °C by adding 0.5 mL of the crude enzyme solution (0.5 ml distilled water in blank group) to 10 mL of 30mM PBS (pH 8.0). The reaction was initiated by adding 5.0 mL of ice-cold CO2 saturated water. The time interval for the pH to drop by 1.5 unit (from 8.0 to 6.5) due to protons released during hydration of CO2 was measured. The reactions were performed in triplicates and average of three replicates was used in calculations.
                            </P><P class="lead" style="color:black;">
                                We calculated the activity according to the formula U= (T0 –T1)/ T0, where T0 and T1 represent time for pH change of blank group and samples group respectively. The CA activity was shown in Fig.17.
                            </P><CENTER style="padding-top: 5px; color: #2f2f2f; font-size: 14px; width: 60%; margin: 0 auto; text-align: justify">      <STRONG>Fig.17</STRONG> CA activity of crude enzyme solution from measured by Brownell’s method</CENTER></DIV></DIV>-
                </DIV></DIV></SECTION><SECTION class="cd-section" style="padding:80px 0;background-color:white"><DIV class="container"><DIV class="row"><DIV class="col-sm-12"><H3 class="uppercase color-primary mb40 mb-xs-24" style="margin-bottom: 40px;"><CENTER>Improvement</CENTER></H3><DIV class="row text-center" style="width:85%;margin:0 auto;text-align:justify"><CENTER><H4>1.Characterization of existing BioBrick part: Sucrose-limitation induced kill switch（<A href="http://parts.igem.org/Part:BBa_K302035" style="font-size:14px;color:black;">BBa_K302035</A>)</H4></CENTER><P class="lead" style="color:black;">Considering the biosafety in our project, we used the original part <A href="http://parts.igem.org/Part:BBa_K2232035" style="font-size:14px;color:black;">BBa_K302035</A> from iGEM10_Newcastle team, which is a suicide switch induced by sucrose. This part encodes a stable non-specific ribonuclease toxin (mazF) and its inhibitory antitoxin (mazE) in<I>  Bacillus subtilis</I> . Contains Sucrose sensitive inducer that will only allow coding sequence translation in the presence of sucrose. When translation of both genes is turned off by sucrose limitation mazE will be degraded faster than mazF. There is then no inhibiton of mazF, killing the cell.</P><P class="lead" style="color:black;">We made characterization of this part in our chassis <I> B.subtilis WB800</I> .The sucrose was added in the culture medium when the value of OD600 reach stationary phase initially, and the final concentration of sucrose was 20mmol/L. Then we recorded the variation of OD600 by interval of 4 hours (Fig.18) and the result was shown as follow.</P><DIV style="text-align: center; width:70%; margin:0 auto"><CENTER style="padding-top: 5px; color: #2f2f2f; font-size: 14px; width: 90%; margin: 0 auto; text-align: justify"><STRONG>Fig.18</STRONG>  The OD600 of original strain <I> WB800</I>  and recombinant <I> B.subtilis WB800</I> _MazEF recorded by interval of 4 hours.The red arrow represent the adjunction of Sucrose (20mmol/L).
                                </CENTER></DIV><P class="lead" style="color:black;">As can be seen from the figure above, the Sucrose-limitation induced kill switch in <I> B.subtilis</I>  didn’t show obvious function on our chassis. It might because the concentration of sucrose wasn’t suitable or the degradation of ribonuclease toxin (mazF) is too fast.</P><CENTER><H4>2. Improve the function of an existing BioBrick Part : Carbonic anhydrase (csoS3) of the carboxysome of Halothiobacillus neapolitanus (<A href="http://parts.igem.org/Part:BBa_K1465205" style="font-size:14px;color:black;">BBa_K1465205</A>)</H4></CENTER><P class="lead" style="color:black;">In this year, we change the sequence of original part(<A href="http://parts.igem.org/Part:BBa_K1465205" style="font-size:14px;color:black;">BBa_K1465205</A>) submitted by iGEM-Team Bielefeld 2014. We add the RBS of<I>  B.subtilis</I>  and a segment of signal peptide at the upstream of cds of Carboxysomal Carbonic Anhydrase(csoS3)(Fig.19) from Halothiobacillus neapolitanus, in order to achieve the expression and secretion of this part in the<I>  B.subtilis</I> .</P><CENTER style="padding-top: 5px; color: #2f2f2f; font-size: 14px; width: 60%; margin: 0 auto; text-align: justify"><STRONG>Fig.19</STRONG>  Construction of the shuttle expression-secretion vector_CsoS3.
                            </CENTER><P class="lead" style="color:black;">
                                The new part (<A href="http://parts.igem.org/Part:BBa_K2232022" style="font-size:14px;color:black;">BBa_K2232022</A>) was constructed into shuttle expression-secretion vector pP43NMK, and 1% Agarose Gel Electrophoresis of the plasmid(Fig.20) verified the correct construction.
                            </P><CENTER style="padding-top: 5px; color: #2f2f2f; font-size: 14px; width: 60%; margin: 0 auto; text-align: justify"><STRONG>Fig.20</STRONG> 1% Agarose Gel Electrophoresis of Vector CsoS3 and its identification by restriction digestion. Lane 1: Complete plasmid; Lane 2: Plasmid digested by KpnI and HindIII; Lane M: DL marker.The length of part CsoS3 was 1774 bp and the blank vector was 6785 bp.
                            </CENTER><P class="lead" style="color:black;">
                                The crude enzyme solution was obtained from the supernatant of fermentation broth of the <I> Bacillus subtilis</I>  including the original strain <I> WB800</I>  and the recombinant WB800_ CsoS3. Since the fact that CA can also catalyze the hydration reaction of ester, which can be used to assay the activity of CA, we conducted measurement on CsoS3 according to a method from Verpoorte JA(1967), and the result(Fig.21) certified that the CsoS3 has high esterase activity.
                            </P><CENTER style="padding-top: 5px; color: #2f2f2f; font-size: 14px; width: 60%; margin: 0 auto; text-align: justify"><STRONG>Fig.21</STRONG>  The esterase activity of crude enzyme solution from the supernatant of fermentation broth of the <I> Bacillus subtilis</I>  including the original strain <I> WB800 </I> and the recombinant WB800_ CsoS3.</CENTER></DIV></DIV>-
                </DIV></DIV></SECTION></DIV><DIV class="footer" style="position:absolute;margin-bottom:0px;padding-bottom:0"><DIV class="footer-info"><ADDRESS style="color:#888">3688 Nanhai Ave, ShenZhen, GuangDong Province, China</ADDRESS><SPAN class="copyright">Copyright © 2017.<A href="https://2017.igem.org/Team:SZU-China" class="mainpage">SZU-China</A> iGEM</SPAN></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>