"
Page
Discussion
View source
History
teams
Log in
 
Team:Peking/Project/BioSensors/DmpR
From 2013.igem.org
Home
Team
Members
Notebook
Attributions
Project
Biosensor Mining
Biosensors
Adaptors
Band-pass Filter
Devices
Model
Band-pass Filter
Biosensor Fine-tuning
Data page
Parts
Judging Criteria
Safety
Human Practice
Questionnaire Survey
Visit and Interview
Practical Analysis
Team Communication
Biosensors
Biosensors
Individual Biosensors
XylS
XylR
HbpR
HcaR
HpaR
PaaX
DmpR
NahR
Multi-component Analysis
DmpR
Mechanism
Build Our Own Sensor!
DmpR bioinformatically mined from Pseudomonas sp.CF600 [1-6] is a σ54-dependent transcriptional factor that tightly controls the expression of the dmp operon (dmpKLMNOPQBCDEFGHI) (Fig. 1). This operon carries genes encoding enzymes for the degradation of (methyl) phenols into pyruvate and acetyl-CoA [7] (Fig. 2).
The cognate promoter of DmpR is Po promoter. The DmpR protein binds to Po promoter as hexamer on two distinct UAS (Upstream Activating Sequence). The transcription initiation of dmp operon also requires IHF (Integration Host Factor), which has two binding sites in Po promoter and enhances the transcription efficiency (Fig. 3).
DmpR protein consists of four domains (Fig. 4): Domain A is the effector-sensing domain, which undergoes conformational change when exposed to proper inducers, including phenol, 2-chlorophenol, 2,4-dichlorophenol, methyl-phenols and other substituted phenols [3][8] . Domain B is a linker domain where mutations would modulate the interactions between Domain A and Domain C. Domain C is the transcriptional activation domain. Domain D contains a helix-turn-helix motif, which is responsible for the DNA binding at Po promoter [1] .
The mechanism of Po promoter activation consists of four steps, DmpR dimerization, DmpR hexamer formation, DNA bending and RNAP recruitment (Fig. 5). Ater the 4 steps, with the help of IHF, transcription from Po promoter initiates thereby.
We collected and analyzed all of the information about DmpR. See Table 1 for the comprehensive summary of DmpR mutants and accompanied novel aromatics-sensing characteristics, which provides a rich repertoire for the synthetic biologists to customize the aromatics-sensing characteristics of DmpR protein.
Peking iGEM has adopted DmpR to build a biosensor circuit (Fig. 6). Plasmid carrying Pr-DmpR was co-transformed with the plasmid containing the inducible promoter Po and reporter gene sfGFP (Fig. 6). Similar to other biosensors, plasmid with RBS BBa_B0032 preceding sfGFP was chosen due to its relatively higher induction ratio during primary test for the RBS library.
We evaluated the performance of DmpR using  our own protocols and almost every protocol mentioned in the previous studies (for more details about these three protocols, Test Protocol 1-3, Click Here ). Results showed that the Test Protocol 3 works the best, with which we tested the ON-OFF ratios of all the 78 aromatic compounds. 7 compounds out of the 78 showed significant induction ratios (>2) (Fig. 7): They are Phl, 2-MePhl, 2-ClPhl, 3-ClPhl, Cat, 4-NtPhl and 2-APhl ( Click Here for the full names of the aromatic compounds). The presence of phenol is consistent with previous studies. The other 6 compounds, however, have not been reported in previous works.
To provide more detailed information about the aromatics-sensing profile, we carefully examined the individual dose-response curves of the 7 compounds via Test Protocol 3 (Fig. 8).
In summary, we have successfully constructed the DmpR biosensing circuit and fine-tuned it guided by our experience obtained from the building of other biosensors. The aromatics-sensing profile of DmpR biosensor is considerably narrow (Fig. 7), making it a robust and convenient biosensor for the presence of phenol and its derivatives.
Figure. 1. The schematic structure of dmp operon. Dmp operon carries genes encoding enzymes for the degradation of (methyl-)phenols to pyruvate and acetyl-CoA, the intermediates of TCA Cycle. The operon is positively controlled by the dmpR gene product, resulting in expression of catabolic enzymes when inducers like phenol are present.
Figure. 2. The catabolic pathway of phenol controlled by the dmp operon. Metabolic enzymes along the pathway are (from Step 1 to Step 8): 1, phenol hydroxylase (PH) ; 2, catechol 2, 3-dioxygenase (C23O); 3, 2-hydroxymuconic semialdehyde hydrolase (2HMSH); 4, 2-hydroxymuconic semialdehyde dehydrogenase (2HMSD) ;5, 4-oxalocrotonate isomerase (4OI); 6, 4-oxalocrotonate decarboxylase (4OD) ;7, 2-oxopent-4-cnoate hydeatase (OEH); 8, 4-hydroxy-2-2oxovalerate aldolase (HOA).
Figure. 3. The schematic structure of Po promoter. The UASs of this promoter shaded in green are dyad sequences to which the DmpR protein binds. The yellow shaded boxes denote IHF binding sites. The pink shaded box represents σ54 binding site with -24 region and -12 motifs highlighted in red. The highlighted nucleotide G represents the transcription start (+1) site.
Figure. 4. The schematic structure of DmpR protein. From N-terminal to C-terminal are Domain A, Domain B, Domain C, Domain D.
Figure. 5. The mechanism of transcription activation by DmpR. (a) The inactive dimer binds to its inducer, which results in a protein conformational change. (b) Binding of ATP triggers multimerization of the dimers to hexamers (or haptamer). (c) ATP hydrolysis coupled with RNA polymerase recruitment triggers transcription activation. (d) Dissociation of the hexamers into dimers after ATP hydrolysis [6] .
Figure. 6.The schematic diagram for the DmpR biosensor circuit. DmpR is constitutively expressed and functions to regulate the transcription of sfGFP gene via promoter Po. As for the RBS of sfGFP, BBa_B0032 was decided due to its better performance compared to RBS of other transcriptional strengths.
Figure. 7. ON/OFF test to evaluate the induction ratios of all aromatic compounds in the aromatics spectrum (For the full names of the compounds, Click Here ). (a) The induction ratios of all 78 aromatic species for the DmpR biosensor. The DmpR biosensor could respond to 7 out of the 78 aromatics with the induction ratio higher than 2. (b) The aromatics-sensing profile of DmpR biosensor. The aromatic species that can elicit strong responses of DmpR biosensor are highlighted in cyan in the aromatics spectrum. The structure formula of typical inducer is also presented around the spectrum. The induction ratio was calculated by dividing the fluorescence intensity of biosensor exposed to object inducers by the basal fluorescence intensity of the biosensor itself.
Figure. 8. Dose-response curves of DmpR biosensor induced by 7 strong inducers (phenol, its homologs and derivatives). The induction ratio was calculated by dividing the fluorescence intensity of biosensor exposed to inducers by the basal fluorescence intensity of the biosensor. For the full names of the compounds, Click Here .
REFERENCE:
[1]. SHINGLER, V.; PAVEL, H. Direct regulation of the ATPase activity of the transcriptional activator DmpR by aromatic compounds. Molecular microbiology, (1995), 17.3: 505-513.
[2]. SHINGLER, Victoria; MOORE, Terry. Sensing of aromatic compounds by the DmpR transcriptional activator of phenol-catabolizing Pseudomonas sp. strain CF600. Journal of bacteriology, (1994), 176.6: 1555-1560.
[3]. SZE, Chun Chau; LAURIE, Andrew D.; SHINGLER, Victoria. In Vivo and In Vitro Effects of Integration Host Factor at the DmpR-Regulated σ54-Dependent Po Promoter. Journal of bacteriology, (2001), 183.9: 2842-2851.
[4]. SARAND, Inga, et al. Role of the DmpR-mediated regulatory circuit in bacterial biodegradation properties in methylphenol-amended soils. Applied and environmental microbiology, (2001), 67.1: 162-171.
[5]．WISE, Arlene A.; KUSKE, Cheryl R. Generation of novel bacterial regulatory proteins that detect priority pollutant phenols. Applied and environmental microbiology, (2000), 66.1: 163-169.
[6]. TROPEL, David; VAN DER MEER, Jan Roelof. Bacterial transcriptional regulators for degradation pathways of aromatic compounds. Microbiology and Molecular Biology Reviews, (2004), 68.3: 474-500. [7]. SHINGLER, V.; POWLOWSKI, J.; MARKLUND, U. Nucleotide sequence and functional analysis of the complete phenol/3, 4-dimethylphenol catabolic pathway of Pseudomonas sp. strain CF600. Journal of bacteriology, (1992), 174.3: 711-724.
[8]. GUPTA, Saurabh, et al. An Effective Strategy for a Whole-Cell Biosensor Based on Putative Effector Interaction Site of the Regulatory DmpR Protein. PloS one, (2012), 7.8: e43527.
Retrieved from " http://2013.igem.org/Team:Peking/Project/BioSensors/DmpR "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
