"
Page
Discussion
View source
History
teams
Log in
 
Team:Peking/projecttest
From 2013.igem.org
home
Item1
Item2
Item3
team
Item1
Item2
Item3
project
Existing Biosensors
Plug-ins & Expansion
Band-pass Circuit
Application
model
Item1
Item2
Item3
safety
Item1
Item2
Item3
human practice
Item1
Item2
Item3
Biosensors
A FAST, EASY AND ACCURATE METHOD TO DETECT TOXIC AROMATIC COMPOUNDS
Biosensors
CapR
DmpR
HbpR
HcaR
HpaR
NahR
PaaX
XylR
XylS
NahR
Overview
Build Our Own Sensor!
The nahR gene originated from the 83 kb naphthalene degradation plasmid NAH7 of Pseudomonas putida encodes a 34 kDa protein which binds to nah and sal promoters to activate transcription of the degradation genes in response to the inducer salicylate. This plasmid encodes enzymes for the metabolism of naphthalene or salicylate as the sole carbon source (Fig. 1a) [1]. The 14 genes encoding the enzymes for this metabolism are organized in two operons: nah (nahA-F), encoding six enzymes required for metabolism for naphthalene to salicylate and pyruvate, and sal (nahG-M), encoding eight enzymes which metabolize salicylate to intermediates of TCA cycle (Fig. 1b) [2].
Previous work have confirmed that the cloned nah, sal, and nahR genes can be expressed and normally regulated in heterologous host Escherichia coli resembling the situation in Pseudomonas putida [3]. NahR is a member of LysR-type transcriptional factors, which have a conserved N-terminal segment that contains the helix-turn-helix DNA-binding motif. It is sigma70 dependent and functions via contacting the α-unit of RNAP [4]. Mutagenesis experiments also largely facilitated the localization of functional domains in the NahR protein [5,6]. N terminal portion (residues 23-45) accounts for binding DNA. Interestingly, the discovery of C terminal (residues 239-291) mutants unable to bind DNA suggested that the DNA binding requires multimerization through a different protein domain [6]. Gel filtration analysis done by Mark A. Schell showed that the active NahR transcription factor may be a tetramer [5]. Additionally, mutations among residues 140-200 and 207-266 largely affected specificity of inducers, indicating that those residues might serve as a ligand-binding crevice (Fig. 2) [6].
As for promoters it regulates, the -82 to -47 region nal and sal promoters is highly homologous, which suggests a consensus NahR-binding site (Fig. 3) [7].
Several experiments all conformed that NahR tightly binds to DNA in vivo in the presence or absence of salicylate. Either the amount or the affinity of NahR binding to DNA will be affected by salicylate in engineered E. coli and its native host Pseudomonas putida [7]. This fact, along with the evidence from methylation protection experiments, suggested a conformational change in the NahR•DNA complex which results in transcriptional activation (Fig. 4)[8].
Wide Type of NahR responds to its authentic inducer salicylate with the induction ratio over 20 [7]. In the attempt of building different whole-cell biosensors, NahR has been artificially evolved or somehow reshaped by mutagenesis to respond to new signals such as substituted salicylates and substituted benzoates [6,9]. New inducers obtained from mutagenesis are summarized in Table 1.
We ligated BBa_J61051 which contains the constitutively expressed NahR and sal promoter with the reporter gene sfGFP (Fig. 5) via standard assembly. The plasmid verified by Beijing Genomics Institute was transformed into E. coli (TOP10, TransGen Biotech). Single clone of bacteria was picked and grown in rich LB medium added chloromycetin (170 μg/ml) overnight and stored at -80℃ in 20% glycerol, waiting for induction test.
On-off test were first carried out for sensor strain NahR following test protocol 1 (hyperlink is needed). NahR strain with no inducer showed low basal expression of sfGFP and 18 compounds showed apparent activation effect with the induction ratios over 20 (Fig. 6). They are listed as follows: SaA, 2-ABzO, 3-MeSaA, 4-MeSaA, 4-ClSaA, 5-ClSaA, AsPR, 2,4,6-TClPhl, 3-IBzO, 2-MeBzO, 3-MeBzO, 4-FBzO, 3-ClBzO, 3-MeOBzO, 3-HSaA, 4-HSaA, 5-ClSaD, 4-ClBzO(For the full name, CHICK HERE).  Besides salicylate derivatives, our sensor strain specially responded to 2,4,6-TClPhl (a kind of polychlorinated phenol (short for PCP)), which is of significant hazard to water environment and human health.
One step further, NahR strain was subjected to induction experiments with concentration of inducer ranging from 0.03 μM to 1 or 3 mM. Dose-response curves of inducers listed above are obtained according to test protocol 1 (Fig. 7). Hundreds fold of induction can be reached at micro-molar concentration for SaA and its derivatives. Substituted benzoate also functions to activate NahR but with slightly lower induction ratio.
In summary, NahR strain works as a highly-sensitive and robust biosensor for salicylates, benzoate derivatives and water-hazard 2,4,6-TClPhl.
Fig. 1. Degradation pathway of naphthalene in Pseudomonas putida and the gene cluster encoding this function. (a) Gene cluster on the NAH7 plasmid that degrades naphthalene: Naphthalene is transformed into salicylate under the enzymes encoded by the upper operon; salicylate is further degraded to enter TCA cycle via the gene products of the lower operon. Both of the operons are regulated by the transcription factor NahR in response to salicylate, the metabolic intermediate in the pathway. (b) Metabolism of naphthalene encoded by the NAH7 plasmid: Naphthalene is degraded by a series of 13 enzymatic reactions, each catalyzed by a specific nah gene product represented by a capital letter. A through M: A, Naphthalene dioxygenase; B, cis-dihydroxy-naphthalene dioxygenase; D, 2-hydroxychromene-2-carboxylate isomerase; E, 2-hydroxybenzalpyruvate aldolase; F, salicylaldehyde dehydrogenase; G, salicylate hydroxylase; H, catechol 2,3-dioxygenase; I, 2-hydroxymuconate semialdehyde dehydrogenase; J, 2-hydroxymuconate tautomerase; K, 4-oxalcrotonate decarboxylase; L, 2-oxo-4-pentenoate hydratase; M, 2-oxo-4-hydroxypentanoate aldolase.
Fig. 2. Location of the functional domains of NahR transcriptional factor. Domain marked by green near the N terminal accounts for DNA binding, which contains a typical helix-turn-helix motif; red domain functions to bind inducer, while the orange domain is putatively involved in multimerization of NahR in the activation.
Fig. 3. Schematic diagram of the consensus structure of the nahR-regulated promoter nah and sal. Alignment of sal and nah promoter is shown and the consensus forward sequences are marked in color. NahR binding sequence and RNAP binding sequence are shown in green and yellow respectively.
Fig. 4. Schematic diagram of the activation of sal (or nah) promoter via NahR in presence of inducer salicylate: 1. The DNA structure of sal promoter: A,B,C and D represent the binding sites for the putative tetramer of NahR; the yellow arrow shows the direction of sal promoter. 2. RNAP and σ70 bind to the sal promoter by recognizing -35 and -10 region; 3. Transcription factor NahR tightly binds to sal promoter and forms a tetramer no matter whether there is salicylate or not; 4. When salicylate is present, NahR•DNA complex undergoes a conformational change. After the hydrolysis of ATP, DNA is opened and transcription is activated.
Fig. 5. Schematic diagram of the plasmid built for sensor strain NahR. iGEM part BBa_J61051 was ligate with reporter sfGFP in the backbone pSB1C3. Promoters are in orange, RBS in light green, CDS in dark blue and terminators in red.
Fig. 6. Response of sensor strain NahR to various aromatics. (For the full name of the compounds, CLICK HERE(hyperlink is needed here)). (a) The induction ratio column in the On-Off test. NahR could respond to 18 out of 78 aromatics with the induction ratio over 20. (b) The detection range of sensor strain NahR is profiled in green at the aromatics spectrum. The structure formula of typical inducer is listed around the cycle spectrum, near its chemical formula.
Fig. 7. Dose response curves of inducers of NahR. (a) Salicylate and its homologs and derivatives; (b) Benzoate, its derivatives and special inducers like 5-ClSaD and 2,4,6-TClPhl. For the full name of the compounds, CLICK HERE(hyperlink is needed here)).
Reference: [1] Dunn, N. W., and I. C. Gunsalus.(1973) Transmissible plasmid encoding early enzymes of naphthalene oxidation in Pseudomonas putida. J. Bacteriol. 114:974-979 [2] M. A. Schell.(1983) Cloning and expression in Escherichia coli of the naphthalene degradation genes from plasmid NAH7. J. Bacteriol. 153(2):822 [3] M. A. Schell, and P. E. Wender.(1986) Identification of the nahR gene product and nucleotide sequences required for its activation of the sal operon. J. Bacteriol. 116(1):9 [4] Woojun Park, Che Ok Jeon, Eugene L. Madsen.(2002) Interaction of NahR, a LysR-type transcriptional regulator, with the K subunit of RNA polymerase in the naphthalene degrading bacterium, Pseudomonas putida NCIB 9816-4. FEMS Microbiology Letters. 213:159-165 [5] Mark A. Schell, Pamela H. Brown, and Satanaryana Raju.(1990) Use of Saturation  Mutagenesis to Localize Probable Functional domains in the NahR protein, a LysR-type Transcription Activator. The Journal of Biological Chemistry. 265(7): 3384-3850. [6] Angel Cebolla, Carolina Sousa, and Vı´ctor de Lorenzo.(1997) Effector Specificity Mutants of the Transcriptional Activator NahR of Naphthalene Degrading Pseudomonas Define Protein Sites Involved in Binding of Aromatic Inducers. The Journal of Biological Chemistry. 272(7):3986-3992 [7] M. A. Schell, and E. F. Poser.(1989) Demonstration, characterization, and mutational analysis of NahR protein binding to nah and sal promoters. J. Bacteriol. 171(2):837 [8] Jianzhong Huang and Mark A. Schell.(1991) In vivo interaction of the NahR Transcriptional Activator with its target sequences. The Journal of Biological Chemistry. 266(17):10830-10838 [9] Hoo Hwi Park, Hae Yong Lee, Woon Ki Lim, Hae Ja Shin. (2005) NahR: Effects of replacements at Asn 169 and Arg 248 on promoter binding and inducer recognition. Archives of Biochemistry and Biophysics. 434:67-74
Retrieved from " http://2013.igem.org/Team:Peking/projecttest "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
