"
Page
Discussion
View source
History
teams
Log in
 
Team:Peking/Project/BioSensors/XylR
From 2013.igem.org
Home
Team
Members
Notebook
Attributions
Project
Biosensor Mining
Biosensors
Adaptors
Band-pass Filter
Devices
Model
Band-pass Filter
Biosensor Fine-tuning
Data page
Parts
Judging Criteria
Safety
Human Practice
Questionnaire Survey
Visit and Interview
Practical Analysis
Team Communication
Biosensors
Biosensors
Individual Biosensors
XylS
XylR
HbpR
HcaR
HpaR
PaaX
DmpR
NahR
Multi-component Analysis
XylR
Mechanism
Previous Engineering Efforts
Build Our Own Sensor!
XylR is an intensively studied regulatory protein mined from Pseudomonas putida [1] . It responds strongly to toluene, xylene and 4-chlro-toluene, while weakly to 3-methyl benzyl alcohol [1] .
XylR activates the Pu promoter to express the "upper pathway" (xylMABC) when exposed to m-xylene (Fig. 1). It also activates the Ps1 promoter, thus to produce another transcriptional activator, called XylS, to turn on the expression of the downstream pathway (xylXYZLTEGFJGKIH, the meta-cleavage operon) [1][2] (Fig. 1, Fig. 2). Notably, the entire regulatory network is also controlled by several global regulatory elements, such as IHF. This provides an explanation for the genetic-context-dependent performance of XylR-Pu pair; namely, when expressed in different bacterial species, the regulatory performance of XylR/Pu pair often fails [3] . Therefore,  fine-tuning is probably necessary.
The XylR protein consists of four domains (Fig. 3): Domain A is the sensing domain, which will conduct a conformational change upon effector binding. Previous studies showed that Domain A inhibits the DNA binding affinity before conformational change [4][5] . Domain B is a linker domain; mutations in this domain will disrupt the functional coupling and spatial interactions between Domain A and C [6] . Domain C is the activation domain with ATPase activity that is crucial for XylR dimerization. A subdomain in domain C is assumed to account for the dimerization. Domain D is the DNA binding domain featured by helix-turn-helix motif whose DNA binding is sequence-specific.
XylR is capable of forming tetramer when Domain C binds with ATP [7] . Without ATP, the XylR dimers could only bind to two sequence-specific DNA sites, but won't initiate transcription even exposed to inducers. As a typical σ54-dependent transcriptional activator, with the binding of ATP, two dimers of XylR tend to further cooperatively tetramerize, thus to bend the promoter region DNA with the help of integrated host factor (IHF). As a result, the transcription will be launched by the interaction between the XylR tetramer and RNA Polymerase (RNAP). ATP hydrolysis provides energy for this process [8] .
Since one of the criteria of our Sensor Mining for aromatics-sensing transcriptional regulators is "well-studied", it can be expected that some mutants of XylR with novel aromatics-sensing characteristics have been identified. Therefore, we set out to collect the information of XylR. As expected, random mutagenesis on XylR Domain B has been performed in previous studies [9] . One XylR mutant, referred to as XylR28 in ref. [9], carries 4 point mutations in Domain A and Domain B. These point mutations endow XylR with a remarkably improved response to 2.4-DNT and TNT and a reduced response to its natural inducer, m-xylene, indicating the directed evolution may provide possibility to engineer XylR to respond to compounds that it doesn't naturally sense [6] .
As discussed above, the XylR/Pu pair needs fine-tuning. Promoter engineering is considered to be a method. A XylR-controlled Pu promoter shows a high basal level. But the cognate promoter of XylR homolog, DmpR, has a fairly low basal level. We found that XylR could activate the Po promoter of DmpR. A hybrid promoter has been accordingly designed using the binding site of XylR from the Pu promoter. This design has shown that the basal level of the hybrid promoter is low and the XylR binding affinity is high [12] .
We obtained wild type XylR from Biobrick BBa_I723021 designed by iGEM07_Glasgow. To find the optimal performance of biosensor XylR/Pu, we combined Pr-XylR coding sequence with 8 reporter circuits adopting different inducible promoters with Ribosome Binding Sites at different intensity. They are Pu promoter naturally activated by XylR with B0031, B0032 and B0034; Po promoter which is originally regulated by DmpR with B0031, B0032 and B0034; Po' promoter specially designed for XylR with B0031 and B0032. Results showed that the XylR biosensor adopting Pu promoter with RBS B0034 obtained the optimal performance. Then this XylR biosensor was subject to ON-OFF test to determine its detection profile (Fig. 5).
As shown in Fig. 5, the performance of XylR is quite different from the previous studies. The conventional inducers for XylR, for instance, TOL, m-Xyl and 3-CITOL, could not elicit any significant responses. It was probably because that these hydrophobic aromatic compounds can vaporize and permeate the sealing film used in our experiment. Regarding this problem, we performed the ON/OFF Test using centrifuge tubes, rather than the conventional 96-well microplate.  Results indicated that the XylR biosensor indeed give response to conventional hydrophobic compounds such as TOL and m-Xyl if the vaporization could be prevented (Fig. 6).
In summary, we have successfully constructed the XylR biosensing circuit. The aromatics-sensing profile of XylR biosensor is considerably narrow (Fig. 5b), making it a convenient biosensor for the presence of 4-chloro-benzyl-aldehyde and 3-methyl-aniline. Notably, we are the first iGEM team that demonstrates the ability of XylR to really work as a biosensor; this is probably due to our proper fine-tuning and the method to avoid the vaporization of hydrophobic aromatic compounds.
Figure 1. The regulatory network of TOL pathway, including the xyl gene cluster, XylS and XylR. XylR is the master regulator that regulates Pu promoter (controls "upper pathway", xylMABC) and Ps1 promoter (controls the expression of XylS, thus to indirectly activate the expression of "downstream pathway“, xylXYZLTEGFJGKIH). The xylene or its derivatives are supposed to be the typical inducers of XylR.
Figure 2. The TOL degradation pathway. The supposed inducers of XylR, toluene and its derivatives, are highlighted in blue.
Figure 3. Schematic organization of XylR protein domains. From N-terminal to C-terminal: the sensor domain A, the linker domain B, the dimerization domain C and the DNA binding domain D.
Figure 4. The mechanism of σ54-dependent transcription activation by XylR. Step1, RNAP recruitment by σ54; XylR has formed dimers when binding to DNA. Step2, formation of XylR tetramer, coupled with ATP hydrolysis. Step3, RNAP ready to initiate transcription. Step4, transcription start with σ54 released. See the main text for more detailed explanation of transcription activation at the Pu promoter.
Figure 5. The induction ratios of all 78 aromatic compounds obtained from the ON/OFF Test using Test Protocol 1 . (a) XylR biosensor could respond to several aromatics whose induction ratios are not sufficiently high. This result was inconsistent with previous studies [5] , which is probably due to the vaporization of hydrophobic aromatic compounds, as we discussed in the main text. (b) The aromatics-sensing profile of XylR biosensor. The aromatic species that can elicit strong responses of XylR biosensor are highlighted in orange in the aromatics spectrum (For the convenience and clearance of data demonstration, 4-chloro-benzoate, 4-bromo-benzoate and salicylic acid are not included here for they can already be well sensed by other biosensors). The structure formula of the typical inducer(s) is also presented around the spectrum. The induction ratio was calculated by dividing the fluorescence intensity of biosensor exposed to object inducers by the basal fluorescence intensity of the biosensor itself. Click Here for the full names of aromatic compounds.
Figure 6. The induction ratios of hydrophobic aromatics compounds obtained from the ON/OFF Test using the new protocol (the same with Test Protocol 1 except that the experiments were performed in centrifuge tubes to avoid the vaporization).
Reference: [1] Abril, M. A., Michan, C., Timmis, K. N., & Ramos, J. (1989). Regulator and enzyme specificities of the TOL plasmid-encoded upper pathway for degradation of aromatic hydrocarbons and expansion   of the substrate range of the pathway.Journal of bacteriology, 171(12), 6782-6790. [2] Gerischer, U. (2002). Specific and global regulation of genes associated with the degradation of aromatic compounds in bacteria. Journal of molecular microbiology and biotechnology, 4(2), 111-  122. [3] Valls, M., Silva‐Rocha, R., Cases, I., Muñoz, A., & de Lorenzo, V. (2011). Functional analysis of the integration host factor site of the σ54Pu promoter of Pseudomonas putida by in vivo UV   imprinting. Molecular microbiology, 82(3), 591-601. [4] Devos, D., Garmendia, J., Lorenzo, V. D., & Valencia, A. (2002). Deciphering the action of aromatic effectors on the prokaryotic enhancer‐binding protein XylR: a structural model of its N‐  terminal domain. Environmental microbiology, 4(1), 29-41. [5] Salto, R., Delgado, A., Michán, C., Marqués, S., & Ramos, J. L. (1998). Modulation of the function of the signal receptor domain of XylR, a member of a family of prokaryotic enhancer-like   positive regulators. Journal of bacteriology,180(3), 600-604. [6] Garmendia, J., & De Lorenzo, V. (2000). The role of the interdomain B linker in the activation of the XylR protein of Pseudomonas putida. Molecular microbiology, 38(2), 401-410. [7] Tropel, D., & Van Der Meer, J. R. (2004). Bacterial transcriptional regulators for degradation pathways of aromatic compounds. Microbiology and Molecular Biology Reviews, 68(3), 474-500. [8] Pérez-Martín, J., & de Lorenzo, V. (1996). ATP Binding to the σ< sup> 54-Dependent Activator XylRTriggers a Protein Multimerization Cycle Catalyzed by UAS DNA. Cell, 86(2), 331-339. [9] de las Heras, A., Chavarría, M., & de Lorenzo, V. (2011). Association of dnt genes of Burkholderia sp. DNT with the substrate‐blind regulator DntR draws the evolutionary itinerary of 2, 4‐  dinitrotoluene biodegradation. Molecular microbiology, 82(2), 287-299. [10] de las Heras, A., & de Lorenzo, V. (2011). Cooperative amino acid changes shift the response of the σ54‐dependent regulator XylR from natural m‐xylene towards xenobiotic 2, 4‐  dinitrotoluene. Molecular microbiology, 79(5), 1248-1259. [11] Garmendia, J., De Las Heras, A., Galvão, T. C., & De Lorenzo, V. (2008). Tracing explosives in soil with transcriptional regulators of Pseudomonas putida evolved for responding to   nitrotoluenes. Microbial Biotechnology, 1(3), 236-246. [12] Kim, M. N., Park, H. H., Lim, W. K., & Shin, H. J. (2005). Construction and comparison of< i> Escherichia coli whole-cell biosensors capable of detecting aromatic compounds. Journal of   microbiological methods, 60(2), 235-245. [13] Garmendia, J., Devos, D., Valencia, A., & De Lorenzo, V. (2001). À la carte transcriptional regulators: unlocking responses of the prokaryotic enhancer‐binding protein XylR to non‐natural   effectors. Molecular microbiology, 42(1), 47-59.
Retrieved from " http://2013.igem.org/Team:Peking/Project/BioSensors/XylR "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
