<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Peking_Project_Plugins"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Peking/Project/Plugins">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Peking/Project/Plugins&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Peking/Project/Plugins&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Peking/Project/Plugins&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Peking/Project/Plugins" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Peking/Project/Plugins</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="navigationblock"><UL id="navigationbar"><LI id="PKU_navbar_Home" class="Navbar_Item"><A href="https://2013.igem.org/Team:Peking">Home</A></LI><LI id="PKU_navbar_Team" class="Navbar_Item"><A href="">Team</A></LI><LI><A href="https://2013.igem.org/Team:Peking/Team/Members">Members</A></LI><LI><A href="https://2013.igem.org/Team:Peking/Team/Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Peking/Team/Attributions">Attributions</A></LI></UL><LI id="PKU_navbar_Project" class="Navbar_Item"><A href="https://2013.igem.org/Team:Peking/Project">Project</A></LI><LI><A href="https://2013.igem.org/Team:Peking/Project/SensorMining">Biosensor Mining</A></LI><LI><A href="https://2013.igem.org/Team:Peking/Project/BioSensors">Biosensors</A></LI><LI><A href="https://2013.igem.org/Team:Peking/Project/Plugins">Adaptors</A></LI><LI><A href="https://2013.igem.org/Team:Peking/Project/BandpassFilter">Band-pass Filter</A></LI><LI><A href="https://2013.igem.org/Team:Peking/Project/Devices">Devices</A></LI><LI id="PKU_navbar_Model" class="Navbar_Item"><A href="">Model</A></LI><LI><A href="https://2013.igem.org/Team:Peking/Model">Band-pass Filter</A></LI><LI><A href="https://2013.igem.org/Team:Peking/ModelforFinetuning">Biosensor Fine-tuning</A></LI><LI id="PKU_navbar_HumanPractice" class="Navbar_Item" style="width:90px"><A href="">Data page</A></LI><LI><A href="https://2013.igem.org/Team:Peking/DataPage/Parts">Parts</A></LI><LI><A href="https://2013.igem.org/Team:Peking/DataPage/JudgingCriteria">Judging Criteria</A></LI><LI id="PKU_navbar_Safety" class="Navbar_Item"><A href="https://2013.igem.org/Team:Peking/Safety">Safety</A></LI><LI id="PKU_navbar_HumanPractice" class="Navbar_Item" style="width:120px"><A href="https://2013.igem.org/Team:Peking/HumanPractice">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:Peking/HumanPractice/Questionnaire">Questionnaire Survey</A></LI><LI><A href="https://2013.igem.org/Team:Peking/HumanPractice/FactoryVisit">Visit and Interview</A></LI><LI><A href="https://2013.igem.org/Team:Peking/HumanPractice/ModeliGEM">Practical Analysis</A></LI><LI><A href="https://2013.igem.org/Team:Peking/HumanPractice/iGEMWorkshop">Team Communication</A></LI></DIV><DIV id="MajorBody"><DIV id="ProjectTitle"><H1 id="ProjectName">Adaptors for Biosensors</H1><H1 id="ProjectSubname">Expanding Detection Profile</H1></DIV><DIV id="LeftNavigation"><H1 id="SensorsListTitle">Adaptors</H1><UL id="SensorsList"><LI class="SensorsListItem"><A href="#FigureHbpR3">XylB and XylC</A></LI><LI class="SensorsListItem"><A href="#MileStone2">NahF</A></LI></UL></DIV><DIV id="SensorEditingArea"><DIV><H id="PageTitle1">XylB and XylC</H><H id="PageSubtitle1"> Inspiration and intention </H><H id="PageSubtitle2"> Build the Adaptors! </H><P id="ContentHbpR1">
Based on the transcriptional factors bioinformatically mined from the database, we have built a large collection of well-characterized biosensors with low basal level and high induction ratio, including XylR for BTEX (short for benzene, toluene, ethyl-benzene and xylene), XylS for benzoates, NahR for salicylates, DmpR for phenols, HbpR for biphenyls and HcaR for phenylpropionate. However, we noticed that the detection profiles of these biosensors can be further expanded. For example, aromatics containing functional groups like hydroxyl and aldehyde groups often can not be readily detected by our biosensors, probably due to their poor performance in the host <I>E.coli</I><A href="#ReferenceHbpR"><SUP>[1]</SUP></A> or the unknown activation mechanism <A href="#ReferenceHbpR"><SUP>[2]</SUP></A>.
 
To broaden the detection profile of our toolkit, we set out to seek for possible solutions. 
 
Previous studies that aimed to expand the detection profiles of biosensors were mainly protein engineering based on the sequence homology. For example, XylS protein with a R45T mutation could be activated by benzoate with substitutions at 2, 3 and 4 position of the aromatic ring, such as 4-methoxybenzoate (<B>Fig. 1a</B>) <A href="#ReferenceHbpR"><SUP>[3]</SUP></A>. Victor de Lorenzo et al. generated a library of engineered transcription factors by shuffling homologous domains between DmpR and XylR. The product can sense not only xylene but also phenols (<B>Fig. 1b</B>) <A href="#ReferenceHbpR"><SUP> [4]</SUP></A>. 


</P><P id="ContentHbpR2">
Therefore, new methods need to be developed for our goal. Because aromatic compounds can be inter-converted by the enzymes in the natural metabolic network, we reasoned that such enzymes may help to expand the detection profiles of existing biosensors. We analyzed the data of metabolic pathways provided by (<A href="http://www.genome.jp/kegg/">KEGG</A>: Kyoto Encyclopedia of Genes and Genomes), focusing on the potency of enzymes to that catalyze specific reaction to convert undetectable compounds into detectable compounds. 

Excitingly, we found degradation enzymes can could serve our purpose well (<B>Fig. 2</B>). <B>Analogous to the electronic adaptors that functionally connect two incompatible devices, these enzymes could be used as “Adaptors for biosensors circuits” to transform originally undetectable compounds into the aromatics that can be detected</B> (<B>Fig. 3</B>).


</P><P id="ContentHbpR3">

To function properly as “Adaptors for biosensors”, the degradation enzymes need to be relatively biochemically well-studied. XylB (benzaldehyde dehydrogenase) and XylC (benzyl alcohol dehydrogenase) from the toluene degradation pathway and NahF (salicylaldehyde dehydrogenase) from the naphthalene catabolic pathway originated in <I>Pseudomonas putida</I> meet these requirements (<B>Fig. 3</B>). Using them as Adaptors, various benzyl alcohols, benzaldehydes and salicylaldehydes are expected to be detected by the downstream regulator proteins, XylS and NahR (<B>Fig. 3</B>).

</P><P id="ContentHbpR4">
The <I>xyl</I> gene cluster that specify the catabolism of toluene are organized in two units: the upper-operon <I>xylUWCAMBN</I> for the conversion of toluene into benzoate and the meta-operon <I>xylXYZLTEGFJQKIH</I> encoding enzymes for further degradation into TCA cycle intermediates <A href="#ReferenceHbpR"><SUP>[5]</SUP></A>. The transcription factor XylR, which detects toluene, initiates transcription of genes in the upper-operon <A href="#ReferenceHbpR"><SUP>[9-10]</SUP></A>, including <I>xylB</I> and <I>xylC</I> (<B>Fig. 4</B>).
</P><P id="ContentHbpR5">
  XylB is a benzyl alcohol dehydrogenase (BADH) that degrades benzyl alcohol to benzaldehyde <A href="#ReferenceHbpR"><SUP>[7]</SUP></A>. Previous studies showed the xylB gene can be expressed in <I>E.coli</I>, but the enzyme activity of the protein is much lower, compared with that when expressed in <I>Pseudomonas putida</I>. It is speculated that the reason are the inefficient transcription of foreign gene in <I>E.coli</I><A href="#ReferenceHbpR"><SUP>[7]</SUP></A> and different genetic regulation system altered the expression intensity of <I>xylB</I> gene as well as the enzyme's activity. So in order to optimize XylB performance, it is important to tune the expression level.

  XylC is benzaldehyde dehydrogenase (BZDH) degrading benzaldehyde to benzoate in the upper operon <A href="#ReferenceHbpR"><SUP>[7]</SUP></A>. Apart from benzaldehyde, XylC also displays activity towards methyl, nitro, methoxy, and chloro substituents of benzaldehyde <A href="#ReferenceHbpR"><SUP>[4]</SUP></A>. This enzyme has been successfully expressed in <I>E.coli</I> and its catalysis ability in vitro has been verified. However, the substrates of XylC will inhibit enzyme activity when present in milli-molar concentrations <A href="#ReferenceHbpR"><SUP>[4]</SUP></A>. 
  
  In our previous work on XylS, we found that benzaldehyde can indeed induce its response, but the induction ratio is apparent low comparing to its native inducer: benzoate. We speculated that the combination of XylS with enzyme XylC would increase the biosensor’s response to benzaldehyde, and further combination with XylB would enable XylS to detect an originally undetectable compound benzyl alcohol. 

After obtaining correct sequence of XylB and XylC via PCR from TOL plasmid pWW0 in Pseudomonas putida, we put XylC and XylB under the control of Pc promoters and expressed them on different plasmids (<B>Fig. 5</B>).

</P><P id="ContentHbpR6">
The performance of XylC Adaptor was tested in following protocol. Bacteria expressing XylC protein was cultured overnight in LB medium at 37 °C  and then diluted 100 fold into Minimal M9 medium, growed for 12 hours at 30 °C  to transform benzyladehydes into benzoates. The culture was then centrifuged, and supernatant was taken out to further culture XylS biosensor. Induction ratio was then measured by Flow Cytometry.

The Flow Cytometry test confirmed our hypothesis: the induction effect of 3-methyl-benzaldehyde and 3-chlorobenzaldehyde on XylS combined with adaptor XylC improved significantly comparing to biosensor XylS only (<B>Fig. 6a</B>). Furthermore, the dose-response curves for 3-methyl-benzaldehyde of adaptor-equipped XylS and unequipped XylC (<B>Fig. 6b</B>) showed that the Adaptor XylC could reduce the detection limit (the concentration of inducer at which an output three times the basal single is generated) of 3-methyl-benzaldehyde from 100 μM to 1 μM. Similarly, the detection limit of 3-chloro-benzaldehyde was decreased more than 1 order of magnitude (<B>Fig. 6c</B>). In addition, dose-response curves for 3-methyl-benzoate and 3-chlorobenzoate were also collected (<B>Fig. 6d</B>). The almost identical lines for 3-methyl-benzoate and 3-chloro-benzoate showed that adding Adaptor XylC does not significantly influence the original characteristics of the biosensor.

</P><P id="ContentHbpR7">
When further equipped with XylB, the biosensor was expected to obtain the ability to detect benzyl alcohol and its derivatives which cannot be detected directly by either biosensor XylS or XylS equipped with XylC. 
</P><DIV class="XylBBigBlock"><P>Different expression levels of Adaptor XylB and a set of culturing modes were tried respectively to optimize the performance of Adaptor XylB. They were listed in <B>Table 1</B>:</P><P>Then biosensor XylS was added to the supernatant of the processed media. 12h later, fluorescence intensity of biosensor XylS was collected by Flow Cytometer.
The first three methods failed because biosensor XylS had problem growing in the processed M9 minimal media for some unidentified reasons. However, the last three methods worked, manifested by the apparent response of biosensor XylS equipped with two Adaptors XylB and XylC to benzyl alcohol (BAL) and 2-methylbenzyl alcohol (2-MeBAL), which were not sensed by biosensor XylS (<B>Fig. 7</B>).</P><P style="line-height:20px; text-align:justify;left:100px; width:800px; border-bottom:0px; color:#1b1b1b; font-size:14px;font-family:arial,calibri,helvetica,sans-serif;"><B>Fig. 7</B> Three effective testing methods for Adaptor XylB and their results. With Adaptors XylB and XylC, biosensor XylS could response to aromatic alcohols that otherwise could not be responded to, especially benzyl alcohol (BAL). Final concentrations of inducer were 300 uM. Symbols used in this figure: BAL: benzyl alcohol; 2-MeBAL: 2-methylbenzyl alcohol; Coculture: M9 minimal media containing the aromatics mentioned above was processed by E. coli transformed with Adaptor XylB and <I>E. coli</I> transformed with XylC simultaneously; Double Trans.: double transformation; Number before protein: the last three numbers of the Part Number for constitutive promoters in &quot;Registry of Standard Biological Parts&quot;.

</P><P>Benzyl alcohol, 2-methylbenzyl alcohol, benzoate, and 2-methylbenzoate of concentration gradients from 0.3 uM to 3 uM were prepared to characterize the performance of Adaptor XylB and identify the best testing methods therein (<B>Fig. 8</B>). Responses of Adaptor XylB in the three testing methods were compared with that of biosensor XylS without Adaptor.

Testing modes that employing E. coli double-transformed with two Adaptors turned out better than the co-culture one, as shown in the dose-response curves for benzyl alcohol (BAL, <B>Fig. 8a</B>) and 2-methylbenzyl alcohol (2-MeBAL, <B>Fig. 8b</B>). To test the Influence of Adaptor XylB to the responses of biosensor XylS to benzoates, M9 minimal media containing benzoates were processed with the same procedure and then tested by biosensor XylS. Induction ratio was then compared to that without Adaptors’ treatments. It showed that E.coli double-transformed with the Adaptor 119-XylB and 114-XylC showed least interference to biosensor XylS’s responses to both benzoate (BzO, <B>Fig. 8c</B>) and its analog 2-methylbenzoate (2-MeBzO, <B>Fig. 8d</B>).
</P><P style="line-height:20px; text-align:justify;left:100px; width:800px; border-bottom:0px; color:#1b1b1b; font-size:14px;font-family:arial,calibri,helvetica,sans-serif;"><B>Fig. 8</B> Characterization and comparison of biosensor XylS with or without Adaptor XylB and XylC. Final concentrations of inducers were 0, 0.3, 1, 3, 10, 30, 100, 300 uM in the characterization assay. <B>(a)</B> Dose-response curve for benzyl alcohol (BAL); <B>(b)</B> Dose-response curve for 2-methylbenzyl alcohol (2-MeBAL); <B>(c)</B> Dose-response curve for benzoate (BzO); <B>(d)</B> Dose-response curve for benzoate (2-MeBzO). Symbols used in this figure are of the same meaning with <B>Fig. 7</B>.
</P><P>Therefore, using the fifth testing methods, biosensor XylS equipped with two Adaptors XylB and XylC could successfully respond to benzyl alcohols without perturbing its response to benzoates. The detection limit of benzyl alcohol was around 3 uM. Through Adaptor XylB, typical <B>aromatic alcohols</B> are also included in the detection profile of our biosensor toolkit.
</P></DIV><DIV id="NahFBigBlock"><H id="PageTitle2">NahF</H><P id="ContentHbpR8">
NahF is a 50.8 KDa protein functioning as salicylaldehyde dehydrogenase to transform salicylaldehyde into salicylic acid (salicylate) using NAD+ (<B>Fig. 9</B>). It is encoded in the naphthalene degradation plasmid from <I>Pseudomonas putida</I>, in which the oxidation of naphthalene has been extensively investigated. Plasmid pDTG1, NAH7 and pND6-1 identified in different <I>P. putida</I> strains all act to degrade naphthalene and share high identity in amino acid sequences <A href="#ReferenceHbpR"><SUP>[11]</SUP></A>. 
</P><P id="ContentHbpR9">
NahF from plasmid NAH7 is most widely studied. It has a wide range of substrates (including salicylaldehyde, 5-chloro-salicylaldehyde, 3-nitro-benzaldehyde, 2-methoxy-benzaldehyde etc.) and its activity can be further enhanced 40.3% in the presence of Fe<SUP>2+</SUP><A href="#ReferenceHbpR"><SUP>[12]</SUP></A>. The wide range of substrates makes it an appropriate candidate to be an Adaptor since many aldehydes can be transformed to the corresponding acids that can be detected by NahR (for salicylates) or XylS (for benzoates).

NahF has been expressed in <I>E.coli</I> and its ability to catalyze the reaction <I>in vitro</I> and <I>in vivo</I> has been verified <A href="#ReferenceHbpR"><SUP>[13-14]</SUP></A>. However, its reaction efficiency when expressed in <I>E.coli</I> was only about 3% of that when expressd in <I>P. putida</I>, possibly due to the difference of regulation in these two bacteria <A href="#ReferenceHbpR"><SUP>[13]</SUP></A>. Therefore, it is necessary to fine-tune the expression level of NahF in <I>E.coli</I>. We built a library of constitutive promoters for tuning the expression of NahF, and NahR biosensor was used to detect the possible salicylates transformed from salicylaldehydes (<B>Fig. 10</B>).
</P><P id="ContentHbpR10">
The performance of NahF Adaptor was tested following protocol similar to that of XylC. Bacteria carrying NahF enzyme was cultured overnight in LB medium at 37 °C  and then diluted 100 fold into Minimal M9 medium, growed for 12 hours at 30 °C  to transform salicylaldehyde into salicylate. Then the culture was centrifuged, supernatant was taken out to further culture NahR biosensor. Induction ratio was then measured by Flow Cytometry. 

The comparison of induction ratio (calculated as previously mentioned) of adaptor-equipped and unequipped NahR, showed that NahF highly improved the induction performance of biosensor NahR induced by salicylaldehyde and 5-chloro-salicylaldehyde. NahF expressed under constitutive promoter BBa_J23106 performed best for both the aldehydes(<B>Fig. 11a</B>). Therefore, BBa_J23106-NahF Adaptor was used for further experiment. Dose response curves for salicylaldehyde and 5-chloro-salicylaldehyde of NahR equipped with Adaptor NahF were obtained (<B>Fig. 11b,c</B>). Comparing them with the dose-response curves for salicylaldehyde and 5-chloro-salicylaldehyde of NahR following the same test protocol (<B>Fig. 11b,c</B>), we demonstrated that Adaptor NahF also functioned to lower the detection limits (the concentration of inducer at which an output three times the basal single is generated) of salicylaldehyde (from 300 μM to 1 μM) (<B>Fig. 11b</B>) and 5-chloro-salicylaldehyde (from 300 μM to 3 μM) (<B>Fig. 11c</B>). NahR's dose response curves for corresponding acids processed by NahF were obtained as well, showing that adding Adaptor NahF does not significantly influence the original characteristics of the NahR biosensor. (<B>Fig. 11d</B>). 
</P><P id="ContentHbpR11">
In summary, Adaptors XylC and NahF functioned not only to optimize existing biosensors' response to several originally detectable compounds, but also expand detection profile in a new way besides modification on coding sequence such as mutagenesis or DNA shuffling. The advantages of the new concept that using adaptor to expand detection profile lie in that adding enzymes does not influence the original characteristics of the biosensors they adapt to, and this methodology does not require labor-intensive mutant screening work.
</P><P id="FigureHbpR7"><B>Figure 9. </B>Biochemical reaction catalyzed by enzyme NahF: Salicylaldehyde is transformed into salicylic acid (salicylate) accompanied by the reduction of NAD+ to NADH.
</P><P id="FigureHbpR8"><B>Figure 10. </B>Schematic diagrams for the plasmid circuits used as Adaptor: NahF and the Sensor: NahR. A constitutive promoter library for the expression of NahF was constructed to obtain the most appropriate expression level of NahF enzyme in <I>E.coli</I>. The number of the Standard Biological constitutive promoter Parts used in this study and its initiation strength is listed in the left portion of the figure. Promoters are presented in orange, RBS in light green, coding sequence in dark cyan, and terminators in dark red.
</P><P id="FigureHbpR9"><B>Figure 11. </B>Data plots that demonstrate the performance of Apator NahF. <B>(a)</B> Induction ratio of biosensor NahR and NahR equipped with Adaptor NahF elicited by salicylaldehyde at the concentration of 1 mM and 5-chloro-salicylaldehyde at the concentration of 0.1 mM. Biosensor NahR with Adaptor NahF showed higher induction ratio than NahR. <B>(b)</B> Dose-response curves of Biosensor NahR and NahR equipped with NahF for salicylaldehyde. Use of Adaptor NahF significantly reduced the detection limit by more than 100 folds. <B>(c)</B> Dose-response curves of Biosensor NahR and NahR equipped with NahF for 5-chloro-salicylaldehyde. <B>(d)</B> Dose-response curves of Biosensor NahR and NahR equipped with NahF for salicylate and 5-chloro-salicylate. The almost overlapping curves for the two compounds showed that the Adaptor NahF did not interfere the performance of NahR biosensor.
</P><P id="ReferenceHbpR"><B>Reference:</B>
[1] Gabriella Fiorentino, Raffaele Ronca and Simonetta Bartolucci, (2009) A novel E.coli biosensor for detecting aromatic aldehydes based on a responsive inducible archaeal promoter fused to the green fluorescent protein. <I>Appl Microbiol Biotechnol</I>. 82:67–77.

[2] Simonetta Bartolucci et. al, (2007) Sulfolobus solfataricus Compounds in Involved in Detoxification of Aromatic MarR-Like Transcriptional Regulator. <I>JOURNAL OF BACTERIOLOGY</I> , 189:7351–7360.

[3] LUMINGZHOU, KENNETHN.TIMMIS, ANDJUANL. RAMOS, (1990), Mutations Leading to Constitutive Expression from the TOL Plasmid meta-Cleavage Pathway Operon Are Located at the C-Terminal End of the Positive Regulator Protein XylS. <I>JOURNAL OF BACTERIOLOGY</I>, 172(7): 3707-3710.

[4] Junkal Garmendia, Damien Devos, Alfonso Valenciaand Vı´ctor de Lorenzo, (2001), A`la carte transcriptional regulators: unlocking responses of the prokaryotic enhancer-binding protein XylR to non-natural effectors.<I> Molecular Microbiology</I>. 42(1), 47–59

[5] RAMOS, Juan L.; MARQUÉS, Silvia; TIMMIS, Kenneth N. (1997) Transcriptional control of the Pseudomonas TOL plasmid catabolic operons is achieved through an interplay of host factors and plasmid-encoded regulators. <I>Annual Reviews in Microbiology</I>, 51.1: 341-373.

[6] INOUYE, SACHIYE; NAKAZAWA, ATSUSHI; NAKAZAWA, Teruko. (1981) Molecular cloning of TOL genes xylB and xylE in Escherichia coli. <I>Journal of bacteriology</I>, 145.3: 1137-1143.

[7] HARAYAMA, S., et al. (1989) Characterization of five genes in the upper-pathway operon of TOL plasmid pWW0 from Pseudomonas putida and identification of the gene products. <I>Journal of bacteriology</I>, 171.9: 5048-5055.

[8] INOUE, Jun, et al. (1995) Overlapping substrate specificities of benzaldehyde dehydrogenase (the xylC gene product) and 2-hydroxymuconic semialdehyde dehydrogenase (the xylG gene product) encoded by TOL plasmid pWW0 of Pseudomonas putida. <I>Journal of bacteriology</I>, 177.5: 1196-1201.

[9] INOUYE, S.; NAKAZAWA, A.; NAKAZAWA, T. (1983) Molecular cloning of regulatory gene xylR and operator-promoter regions of the xylABC and xylDEGF operons of the TOL plasmid.<I>Journal of bacteriology</I>, 155.3: 1192-1199.

[10] FRANKLIN, F. C., et al. (1981) Molecular and functional analysis of the TOL plasmid pWWO from Pseudomonas putida and cloning of genes for the entire regulated aromatic ring meta cleavage pathway. <I>Proceedings of the National Academy of Sciences</I>, 78.12: 7458-7462.

[11] Baoli Cai et. al, (2004) Complete nucleotide sequence and organization of the naphthalene catabolic plasmid pND6-1 from Pseudomonas sp. strain ND6, <I>GENE</I>, 336:231–240.

[12] Zhao H, Li Y, Chen W, Cai B. (2007) A novel salicylaldehyde dehydrogenase-NahV involved in catabolism of naphthalene from Pseudomonas putida ND6. <I>Chin Sci Bull</I>. 52(14):1942—8.

[13] M. A. Schell, (1983) Cloning and expression in Escherichia coli of the naphthalene degradation genes from plasmid NAH7. <I>J. Bacteriol</I>. 153(2):822-829.

[14] R. W. Eaton and P. J. Chapman, (1992) Bacterial metabolism of naphthalene_construction and use of recombinant bacteria to study ring cleavage of 1,2-dihydroxynaphthalene and subsequent reactions.<I> J. Bacteriol</I>. 174(23):7542-7554.


</P></DIV><P id="FigureHbpR1"><B>Figure 1. </B> Previous efforts to broaden the detection profiles of biosensors. <B>(a)</B>Mutagenesis: The XylS mutant (thr45), aromatics-responsive transcription regulator, showed much higher induction ratios than that of the wild-type. The inducers are not the natural for the wild-type XylS. <B>(b)</B> DNA shuffling: Shuffling N-terminal domains of the homologous regulators DmpR and XylR produced a transcriptional regulator that responds to both benzyl hydrocarbons and phenols.
</P><P id="FigureHbpR2"><B>Figure 2. </B> The schematic diagram for the metabolic pathways concerning XylS and NahR. Aromatics boxed in green can be detected by our biosensor toolkit. Compounds boxed in the orange are those lacking accompanied aromatics-response proteins. All these compounds can be bridged by metabolite enzymes boxed in red. This inspired us to employ metabolic enzymes as the “Adaptors for biosensors” that transform originally undetectable upstream intermediates into detectable downstream aromatic compounds.
</P><P id="FigureHbpR3"><B>Figure 3. </B> Schematic diagram for Adaptors in the context of metabolic pathways. Transcriptional factors (short for TF) XylS and NahR function to sense different aromatic acids. With Adaptors like XylC and NahF transforming aldehydes to acids, aldehydes can be detected. Likewise, with Adaptor XylB transforming alcohols to aldehydes, alcohols can also be detected.
</P><P id="FigureHbpR4"><B>Figure 4. </B> Xylene degradation metabolite operon in TOL plasmid and the catabolic pathway from benzyl alcohol to benzoic acid. <B>(a)</B> Relationship of gene cluster and regulatory protein in the TOL plasmid: XylR controls <I>Pu</I> promoter and <I>Ps2</I> promoter. <I>Ps1</I> is a weak constitutive promoter, which controls the level of XylS. When xylene or its homologous compounds exist, the upper pathway and another promoter called XylS was activated. <B>(b)</B> Biochemical reaction catalyzed by enzyme XylB and XylC: Benzyl alcohol is transformed into benzaldehyde by XylB and then benzaldehyde is converted to benzoic acid via the catalysis of XylC.
</P><P id="FigureHbpR5"><B>Figure 5. </B>Schematic diagrams for the biosensor circuit and the Adaptor: XylB, XylC and XylS. A library of constitutive promoters was used to control XylB. XylC is expressed under constitutive promoter BBa_J23114.  XylS biosensor circuit is shown in the bottom. Promoters are presented in orange, RBS in light green, coding sequence in dark cyan, and terminators in dark red.
</P><P id="FigureHbpR6"><B>Figure 6. </B>Data plots that demonstrate the performance of Apator XylC. <B>(a)</B> Fluorescence intensity of XylS and XylS equipped with Adaptor XylC elicited by 3-methyl-benzaldehyde and 3-chloro-benzaldehyde at the concentration of 1 mM. XylS with Adaptor XylC showed higher fluorescence intensity and induction ratio than XylS. <B>(b)</B> Dose-response curves of XylS and XylS equipped with XylC for 3-methyl-benzaldehyde. Use of Adaptor XylC significantly reduced the detection limit by 100 folds. <B>(c)</B> Dose-response curves of XylS and XylS equipped with XylC for 3-chloro-benzaldehyde. <B>(d)</B> Dose-response curves of XylS and XylS equipped with XylC for 3-methyl-benzoate and 3-chloro-benzoate. The almost overlapping curves for the two compounds showed that the Adaptor XylC did not interfere the performance of XylS biosensor.
</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Peking/Project/Plugins">http://2013.igem.org/Team:Peking/Project/Plugins</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Peking/Project/Plugins" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Peking/Project/Plugins" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Peking/Project/Plugins&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Peking/Project/Plugins&amp;oldid=349434" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>