"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Peking/Project/BioSensors
From 2013.igem.org
Home
Team
Members
Notebook
Attributions
Project
Biosensor Mining
Biosensors
Adaptors
Band-pass Filter
Devices
Model
Band-pass Filter
Biosensor Fine-tuning
Data page
Parts
Judging Criteria
Safety
Human Practice
Questionnaire Survey
Visit and Interview
Practical Analysis
Team Communication
Biosensors
Individual Biosensors
XylS
XylR
HbpR
HcaR
HpaR
PaaX
DmpR
NahR
Multi-component Analysis
Biosensor Introduction
One advantage of biosensors is that the detection profile of a particular biosensor is usually limited to a few specific signals, thus making biosensor's output highly informative. Another advantage is the biosensor response method: it usually involves no more than transcription and translation, both of which are automatically operated by the living cells themselves; so it is more convenient to use, compared with conventional chemical methods that rely heavily on complicated measuring devices. Additionally, originating from functional elements of natural biological systems, biosensors are subject to various tuning methods such as directed evolution, point mutagenesis or other genetic manipulations. A biosensor typically consists of a detector and a reporter; the input signal activates the detector, through which the reporter is stimulated to emit the output signal. In our project, the biosensor input signals are aromatic compounds and the output is the expression of a reporter gene; the aromatics are supposed to be sensed by transcriptional regulators from the bacteria living in aromatics-rich environment. Through the bioinformatic mining, we obtained 17 aromatics-sensing transcriptional regulators (see Sensor Mining ). Noting the fact that their expected aromatics-sensing profiles overlap a lot, we finally determined 8 transcriptional regulators whose profiles could be combined to cover the overall spectrum of aromatic compound species (Table 1).
Sensor
Expected Aromatics-sensing Profile
Source
XylS
BzO; 2-MeBzO; 3-MeBzO; 2,3-DMeBzO; 3,4-DMeBzO
Pseudomonas putida
XylR
TOL; m-Xyl; 3-ClTOL
Pseudomonas putida
HbpR
2-HBP; 2,2'-DHBP
Pseudomonas azelaica
HcaR
PPA; 3-HPPA; 3,4-DHPPA
Escherichia coli
HpaR
3-HPAA; 4-HPAA; 3,4-DHPAA
Escherichia coli
PaaX
PAASCoA
Escherichia coli
DmpR
Phl; 2-MePhl; 3-MePhl; 4-MePhl; 2-ClPhl
Pseudomonas putida
NahR
SaA; ASPR; 3-ClSaA; 4-ClSaA; 5-ClSaA;
Pseudomonas putida
Next we focused on constructing biosensors with low basal level, high induction ratio, and robust detection profiles. The coding sequences of these 8 transcriptional regulators were obtained by either chemical synthesis or PCR amplification. They were then incorporated into our biosensor circuit design (Fig. 1):
It can be expected that the primary construction of a biosensor circuit might not work. For the fine-tuning, a library of constitutive promoters and a library of RBS sequences, both of different strengths,  were utilized to genetically tailor the expression of transcriptional regulators and sfGFP, respectively (Fig. 2).
After sparing no efforts to fine-tune the circuits, we have successfully constructed a comprehensive collection of high-performance aromatics-sening biosensors, including XylS , XylR , HbpR , HcaR , DmpR . and NahR . Results showed that a large variety of aromatic compounds have been taken into the detection profile of our toolkit (Fig. 3). See the detailed information about the performance of individual biosensors.
To allow the combination of these biosensors to analyze the aromatics profiles of practical samples, the orthogonality/crosstalk between inducers of different biosensors should be carefully evaluated. Within a general range of inducer concentration, no significant synergistic and antagonistic effects were observed, making our biosensors effective at profiling practical samples (Fig. 4).
Figure 1. Schematic diagram for the design frame of biosensor circuits A constitutive promoter (Pc) constitutively expresses the transcriptional regulator protein on the low-copy backbone pSB4K5; the cognate promoter of the transcriptional regulator (TF) controls the expression of the reporter gene, super-fold green fluorescent protein (sfGFP, a novel and robust GFP variant designed for in vivo measurement of protein expression levels); its backbone is high-copy pUC57. Symbols used in this figure: Left orange arrow, Pc promoter; right orange arrow, the promoter regulated by the aromatics-sensing transcriptional regulator; green ovals, Ribosome Binding Sites (RBS); red hexagons, transcriptional terminators; dark cyan squares, gene coding sequences.
Figure 2. Libraries of promoters and RBS sequences used for the fine-tuning of biosensor circuits. Pc promoter library was exploited to fine-tune the expression level of transcriptional regulator: J23106, J23105, J23114, J23117, J23109 and J23113.  The RBS library includes B0031, B0032, B0033 and B0034 to tailor the expression level of reporter gene sfGFP. Left orange arrow, Pc promoter; right orange arrow, the promoter regulated by the aromatics-sensing transcriptional regulator; green ovals, Ribosome Binding Sites (RBS); red hexagons, transcriptional terminators; dark cyan squares, gene coding sequences.
Figure 3. The aromatics spectrum showing the aromatics-sensing profiles of our individual biosensors. Each color segment in the central spectrum represents the detection profile of a biosensor. Structural formula highlighted in color stand for the aromatic compounds that can be detected by our biosensors. The "plug" icon stands for Adaptors , enzymes that convert the undetectable compounds into the detectable compounds, thus to reinforce the detection capacity of some biosensors. Click Here for the summary of the aromatics spectrum and aromatics-sensing profiles of individual biosensors.
Figure 4. Summary of the multi-component analysis to evaluate the synergistic/antagonistic effects between the inducers of 4 representative biosensors. No synergistic/antagonistic effects between the sensing-profiles of 4 biosensors (XylS, NahR, HbpR, and DmpR) were observed. For instance, although the sensing profiles of NahR and XylS overlap to some extent, the NahR-specific and XylS-specific inducers proved to be really orthogonal, which is consistent with our expectation. Click Here to see the detailed information about the  multi-component analysis.
Retrieved from " http://2013.igem.org/Team:Peking/Project/BioSensors "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
