"
Page
Discussion
View source
History
teams
Log in
 
Team:Peking/Project/BioSensors/HpaR
From 2013.igem.org
Home
Team
Members
Notebook
Attributions
Project
Biosensor Mining
Biosensors
Adaptors
Band-pass Filter
Devices
Model
Band-pass Filter
Biosensor Fine-tuning
Data page
Parts
Judging Criteria
Safety
Human Practice
Questionnaire Survey
Visit and Interview
Practical Analysis
Team Communication
Biosensors
Biosensors
Individual Biosensors
XylS
XylR
HbpR
HcaR
HpaR
PaaX
DmpR
NahR
Multi-component Analysis
HpaR
PaaX
Mechanism
Build Our Own Sensor!
Mechanism
Build Our Own Sensor!
HpaR is of 17,235 Da (149 amino acid) that belongs to MarR family [1] . It functions as a repressor of the hpa cluster consisting of hpaGEDFHI genes (Fig. 2), which participate in the catabolic pathway of 4-hydroxyphenylacetic acid (4HPAA) (Fig. 1). HpaR de-represses downstream genes when exposed to ligands such as 4HPAA, 3-hydroxyphenylacetic acid (3HPAA) and 3, 4-dihydroxyphenylacetic acid (3,4-DHPAA).
hpa cluster consists of three operons. The regulator gene, hpaR, is driven by Pr promoter in the direction reverse to other genes. The adjacent promoter, Pg, initiates the transcription of the functional hpaGEDFHI operon. Pr and Pg are both regulated by HpaR and located in the intergenic region between hpaR and hpaG (Fig. 2). There are two HpaR binding sites, OPR1 and OPR2, belonging to Pr and Pg respectively. HpaR dimer contacts with a palindrome sequence within the binding site in the absence of ligands and inhibit the transcription initiation.
OPR1 is centered in the +2 site of Pg while OPR2 is centered in the +40 site downstream of Pr. It is hypothesized that HpaR's binding to OPR1 inhibits the formation of open complex whereas binding to OPR2 blocks transcription elongation(Fig. 3).
Interestingly, based on the gel retardation assays, most of the HpaR dimer still contacts with the OPR1 in the presence of the ligands, which recruits RNAP and forms open-complex. In this way, HpaR can also be regarded as an activator.
The two binding sites, OPR1 and OPR2, exhibit obvious cooperative effect, i.e., binding with Pg significantly improves the affinity of HpaR to Pr. It is hypothesized that the two HpaR dimers, when binding to two OPR sites respectively, will dimerize again and generate repression loops, similar with the mechanism of AraC . Binding of ligands disrupts the dimer and consequently initiates transcription of the hpaGEDFHI cluster [1] .
We obtained hpaR coding sequence via PCR and constructed Pg/HpaR expression system. Pc promoter J23106 is selected to drive the transcription of HpaR. However, we didn't obtain a satisfactory induction ratio. We speculated that this is because binding sites of global regulators such as CRP are located in the promoter. The main function of the pathway is to use the alternative carbon sources in the environment, so bacteria will control strictly the expression of such related genes in rich medium we used to culture the cells. We hope by changing mediums we use to characterize the biosensor, we may achieve a better performance in the near future.
PaaX is a repressor with 316-amino acid. As a member of GntR family, it contains 25 consecutive residues similar to the helix-turn-helix motif responsible for DNA recognition and binding [3] . PaaX binds to a palindrome sequence located in its cognate promoter, Pa, inhibiting the promoter in the absence of ligand. Unlike other sensors in E. coli, PaaX detects phenylacetic acid-CoA (PA-CoA), which is the first intermediate in the PA degradation pathway. This step is catalyzed by PaaK [3][4] .
There are three operons in paa clusters, namely paaZ, paaABCDEFGHIJK and paaXY. (Fig. 4) The promoters regulated by PaaX, PaaZ and PaaA are located at the intergenic region between gene paaZ and paaA. They each possesses a palindromic sequence for binding of the repressor. (Fig. 5)
The PaaX binding site on Pa overlaps the transcriptional initiation site. Thus PaaX's binding will interrupt the recruitment of RNAP and the formation of open complex. The cluster is also controlled by binding of global regulators such as IHF and CRP. Like hpaR cluster, it is hypothesized the two dimers may form a repression loop of DNA to strictly inhibit the leakage transcription in the absence of ligand [4] .
We standardized the paaX genes and create Pa/PaaX expression system. We tuned the expression intensity of the repressor via selecting appropriate Pc promoter from a
Pc
library. Similar with HpaR, the expression of Pa promoter is inhibited by the global regulators and we didn't obtain an obvious induction effect. We would like to try more culture conditions in order to improve the performance of the sensors.
Figure. 1.The degradation pathway of 3HPAA. The letters on the arrows are the names of genes in the clusters. eg. HpaB,C means the step 1 is catalyzed by the product of hpaB and hpaC.
Figure. 2. Structure of the hpa cluster.  The arrows inside the squares indicate the transcription direction of the genes.Pr, Pg, Px, Pa and Pbc represent the promoter controlling genes in the cluster. Addition of 3-hydroxyphenylacetic acid can de-repress the promoters inhibited by HpaR (for Pr and Pg activation) or HpaA (for Pbc activation). The enzymes in these clusters will degrade the 3-HPAA to intermediates in TCA cycle.
Figure. 3. Structure of the intergenic region between hpaR and hpaG (Galán, B. et al, 2003). Elements within promoters including two OPRs are marked in squares with different color. The transcription direction of HpaR and HpaG are indicated by arrows.
Figure. 4. Structure of the paa cluster. The arrows indicate the transcription direction of each gene. Pz, Pa, Px are the promoters controlling genes in the cluster. PaaX binding with 3-hydroxyphenylacetic de-repress the downstream promoters, namely Pz and Pa. Enzymes coded by the operons catalyze the degradation of PAA to intermediates in TCA cycle.
Figure. 5. The intergenic region that contains Pz and Pa. (Ferrández, A. et al, 2000) The elements within the promoters are enclosed by square. The transcription direction of paaZ and paaA are indicated by arrows. The IHF and CRP binding sites are also marked.
Reference: [1] Galán, B., Kolb, A., Sanz, J. M., García, J. L., & Prieto, M. A. (2003). Molecular determinants of the hpa regulatory system of Escherichia coli: the HpaR repressor. Nucleic acids research, 31(22), 6598-6609. [2] Prieto, M. A., Diaz, E., & García, J. L. (1996). Molecular characterization of the 4-hydroxyphenylacetate catabolic pathway of Escherichia coli W: engineering a mobile aromatic degradative cluster. Journal of bacteriology, 178(1), 111-120. [3] Ferrández, A., Miñambres, B., Garcı́a, B., Olivera, E. R., Luengo, J. M., Garcı́a, J. L., & Dı́az, E. (1998). Catabolism of phenylacetic acid in Escherichia coli characterization of a new aerobic hybrid pathway. Journal of Biological Chemistry, 273(40), 25974-25986. [4] Ferrández, A., Garcı́a, J. L., & Dı́az, E. (2000). Transcriptional Regulation of the Divergent paaCatabolic Operons for Phenylacetic Acid Degradation inEscherichia coli. Journal of Biological Chemistry, 275(16), 12214-12222.
Retrieved from " http://2013.igem.org/Team:Peking/Project/BioSensors/HpaR "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
