<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Peking_Project_SensorMining"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Peking/Project/SensorMining">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Peking/Project/SensorMining&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Peking/Project/SensorMining&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Peking/Project/SensorMining&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Peking/Project/SensorMining" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Peking/Project/SensorMining</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="navigationblock"><UL id="navigationbar"><LI id="PKU_navbar_Home" class="Navbar_Item"><A href="https://2013.igem.org/Team:Peking">Home</A></LI><LI id="PKU_navbar_Team" class="Navbar_Item"><A href="">Team</A></LI><LI><A href="https://2013.igem.org/Team:Peking/Team/Members">Members</A></LI><LI><A href="https://2013.igem.org/Team:Peking/Team/Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Peking/Team/Attributions">Attributions</A></LI></UL><LI id="PKU_navbar_Project" class="Navbar_Item"><A href="https://2013.igem.org/Team:Peking/Project">Project</A></LI><LI><A href="https://2013.igem.org/Team:Peking/Project/SensorMining">Biosensor Mining</A></LI><LI><A href="https://2013.igem.org/Team:Peking/Project/BioSensors">Biosensors</A></LI><LI><A href="https://2013.igem.org/Team:Peking/Project/Plugins">Adaptors</A></LI><LI><A href="https://2013.igem.org/Team:Peking/Project/BandpassFilter">Band-pass Filter</A></LI><LI><A href="https://2013.igem.org/Team:Peking/Project/Devices">Devices</A></LI><LI id="PKU_navbar_Model" class="Navbar_Item"><A href="">Model</A></LI><LI><A href="https://2013.igem.org/Team:Peking/Model">Band-pass Filter</A></LI><LI><A href="https://2013.igem.org/Team:Peking/ModelforFinetuning">Biosensor Fine-tuning</A></LI><LI id="PKU_navbar_HumanPractice" class="Navbar_Item" style="width:90px"><A href="">Data page</A></LI><LI><A href="https://2013.igem.org/Team:Peking/DataPage/Parts">Parts</A></LI><LI><A href="https://2013.igem.org/Team:Peking/DataPage/JudgingCriteria">Judging Criteria</A></LI><LI id="PKU_navbar_Safety" class="Navbar_Item"><A href="https://2013.igem.org/Team:Peking/Safety">Safety</A></LI><LI id="PKU_navbar_HumanPractice" class="Navbar_Item" style="width:120px"><A href="https://2013.igem.org/Team:Peking/HumanPractice">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:Peking/HumanPractice/Questionnaire">Questionnaire Survey</A></LI><LI><A href="https://2013.igem.org/Team:Peking/HumanPractice/FactoryVisit">Visit and Interview</A></LI><LI><A href="https://2013.igem.org/Team:Peking/HumanPractice/ModeliGEM">Practical Analysis</A></LI><LI><A href="https://2013.igem.org/Team:Peking/HumanPractice/iGEMWorkshop">Team Communication</A></LI></DIV><DIV id="MajorBody"><DIV id="LeftNavigation"><H1 id="SensorsListTitle">Biosensor Mining</H1><UL id="SensorsList"><LI class="SensorsListItem"><A href="#MileStone1">Method</A></LI><LI class="SensorsListItem"><A href="#MileStone2">Result</A></LI><LI class="SensorsListItem"><A href="#MileStone3">Source Code</A></LI></UL></DIV><DIV id="BiosensorOverviewContainer"><H1 id="BiosensorOverviewTitle">Biosensor Mining</H1></DIV><DIV id="BiosensorMiningEditingArea"><P id="Mining_Para1">In order to comprehensively profile aromatics in environment, our toolkit should be equipped with a collection of biosensors that senses diverse aromatic components. However, there is no such a comprehensive collection of biosensors available currently. Noting the abundant genomic and proteomic data in databases today, we speculated that large protein databases, like Uniprot, are ideal gold mines finding new Biobricks. This year, Peking iGEM team has developed a four-step bioinformatic mining method to screen out feasible and well-characterized aromatics-sensing transcriptional regulators from the protein database. This method consists of several computer programs to process massive data and a manual adjustment step to further guarantee the reliability of the mining results: </P><P id="Mining_Legend1"><B>Figure 1.</B> The flow chart of mining aromatic-sensing transcriptional regulators from the database Uniprot. <B>Step 1</B>, narrowing down the scope of proteins into transcription factors (TFs) in specific bacteria species. <B>Step 2</B>, screening out aromatics-related transcription factors. <B>Step 3</B>, the aromatics-related transcription factors with most detailed studies are selected. <B>Step 4</B>, manual adjustment to further evaluate the reliability of the selected transcription factors. <B>Move the mouse cursor to see the detailed explanations of individual steps. </B></P><P id="Mining_Legend2"><B>Figure 2.</B> Summary of data mining process and screening criteria. Numbers of remained  candidates after each step are shown on the left surface of the pyramid. The screening criteria are shown on the right.</P><P id="Mining_Para2">First (Step 1 in <B>Fig. 1</B>), we narrowed down the scope of proteins into transcription factors of specific bacteria species. We chose <I>Pseudomonas putida</I>, <I>pseudomonas sp</I> and <I>pseudomonas nitroreducens</I> as our source organisms because they live in aromatics-rich environments and chose <I>E.coli</I> and <I>bacillus subtilis</I> due to their clear genetic contexts. We downloaded all <B>21,096</B> entries of transcription-regulation-related proteins of these five bacteria species from the protein data base uniprot.

Second (Step 2 in <B>Fig. 1</B>), we screened out aromatics-related transcription factors by analyzing the downloaded entries with a computer program. The computer program searched all the entries with a list of keywords (aromatic, benzene, phenol, phenyl, naphthalene, benzoic, benzaldehyde, tolyl, toluene, xylene, styrene) and scored the proteins. Once a keyword appeared in a protein’s entry, the program added one point to its score. <B>912</B> proteins with scores higher than 0 remained after this step.

Third (Step 3 in <B>Fig. 1</B>), we used another computer program to examine whether the transcription factors remaining after step two were well studied. The computer program excluded unnamed proteins that have open reading frame numbers only. Because proteins that have been characterized in <I>E.coli</I> are more likely to work well in our expected biosensor circuits (that works in <I>E.coli</I>), the computer program then searched the names of the remaining  proteins together with the keyword “E. coli” in google scholar and added <I>k</I>/10 point to its score (<I>k</I> is the number of citations). <B>60</B> proteins scored higher than 10 points remained after this step.

Finally (Step 4 in <B>Fig. 1</B>), we carried out a manual adjustment on the 60 proteins to confirm their reliability. Proteins that has no actual ability to sense aromatic compounds and those other possible false positive cases, such as bacterial two-component systems (their performance is highly genetic-context-dependent across different bacterial species), were excluded. Finally, <B>17</B> proteins were manually determined at last (<B>Table 1</B>). The entire mining process has been summarized in <B>Fig. 2</B>.
</P><H1 id="SensorMiningTableTitle"><B>Table 1. Aromatics-sensing transcriptional regulators mined from the Uniprot</B></H1><TABLE border="1" id="SensorMiningTable"><TBODY><TR><TH>Protein names</TH><TH>Sources</TH><TH>Reported Typical Inducers (<B><A href="https://static.igem.org/mediawiki/igem.org/2/24/Peking2013_Chemicals_V3%2B.pdf">Click Here</A> for the chemical formula of aromatic compounds</B>)</TH><TH>Scores</TH></TR><TR><TD><A href="https://2013.igem.org/Team:Peking/Project/BioSensors/XylS">XylS</A></TD><TD><I>Pseudomonas putida</I> (Arthrobacter siderocapsulatus)</TD><TD>Benzoic acid</TD><TD>259</TD></TR><TR><TD><A href="https://2013.igem.org/Team:Peking/Project/BioSensors/XylR">XylR</A></TD><TD><I>Pseudomonas putida</I> (Arthrobacter siderocapsulatus)</TD><TD>m-Xylene</TD><TD>219</TD></TR><TR><TD>tyrR</TD><TD><I>Escherichia coli</I> (strain K12)</TD><TD>tyrosine</TD><TD>160</TD></TR><TR><TD><A href="https://2013.igem.org/Team:Peking/Project/BioSensors/NahR">nahR</A></TD><TD><I>Pseudomonas putida </I>(Arthrobacter siderocapsulatus)</TD><TD>Salicylic acid</TD><TD>106</TD></TR><TR><TD>CapR</TD><TD><I>Pseudomonas putida</I> (Arthrobacter siderocapsulatus)</TD><TD>phenol</TD><TD>80</TD></TR><TR><TD><A href="https://2013.igem.org/Team:Peking/Project/BioSensors/HcaR">hcaR</A></TD><TD><I>Escherichia coli</I> (strain K12)</TD><TD>3-Phenyl-propionic acid</TD><TD>56</TD></TR><TR><TD><A href="https://2013.igem.org/Team:Peking/Project/BioSensors/DmpR">dmpR</A></TD><TD><I>Pseudomonas sp. </I>(strain CF600).</TD><TD>phenol</TD><TD>43</TD></TR><TR><TD>pobR</TD><TD><I>Pseudomonas putida</I>(Arthrobacter siderocapsulatus)</TD><TD>p-Hydroxybenzoic acid</TD><TD>29</TD></TR><TR><TD>CymR</TD><TD><I>Pseudomonas putida</I> (Arthrobacter siderocapsulatus)</TD><TD>4-Isopropyl benzoate</TD><TD>23</TD></TR><TR><TD><A href="https://2013.igem.org/Team:Peking/Project/BioSensors/HpaR#ContentHpaR4">Paax</A></TD><TD><I>Escherichia coli </I>(strain K12)</TD><TD>phenylacedtyl-CoA</TD><TD>20</TD></TR><TR><TD><A href="https://2013.igem.org/Team:Peking/Project/BioSensors/HpaR">hpaR</A></TD><TD><I>Pseudomonas putida </I>(Arthrobacter siderocapsulatus)</TD><TD>(3-Hydroxy-phenyl)-acetic acid</TD><TD>18</TD></TR><TR><TD>mhpR</TD><TD><I>Escherichia coli</I> (strain K12)</TD><TD>(3-Hydroxy-phenyl)-propionic acid</TD><TD>18</TD></TR><TR><TD>phhR</TD><TD><I>Pseudomonas putida</I> (Arthrobacter siderocapsulatus)</TD><TD>phenylalanine</TD><TD>16</TD></TR><TR><TD>bphS</TD><TD><I>Pseudomonas sp.</I> (strain CF600).</TD><TD>2-hydroxy-6-oxo-6-phenylhexa-2,4-dienoic acid</TD><TD>16</TD></TR><TR><TD><A href="https://2013.igem.org/Team:Peking/Project/BioSensors/HbpR">HbpR</A></TD><TD><I>Pseudomonas nitroreducens</I></TD><TD>2-Hydroxybiphenyl</TD><TD>12</TD></TR><TR><TD>phcR</TD><TD><I>Pseudomonas putida </I>(Arthrobacter siderocapsulatus)</TD><TD>phenol</TD><TD>11</TD></TR><TR><TD>yodB</TD><TD><I>Bacillus subtilis </I>(strain 168)</TD><TD>2-methyl hydroquinone</TD><TD>11</TD></TR></TBODY></TABLE><P id="Mining_Para3">
In summary, using the four-step bioinformatic data mining method. we have successfully screened out a set of aromatics-sensing transcriptional regulators (<B>Fig. 2</B>). These 17 aromatics-sensing regulators are supposed to be reliable and well studied. 
<B>We believe that this method may also be applied to mine other types of Biobricks. Moreover, although our data mining method is conventional in bioinformatics field, we deem such a bioinformatics approach to be highly instructive to routine synthetic biology research, for it will greatly reinforce our ability to mine rich collections of high-quality Biobricks from increasingly massive data in an automated manner</B>. 

In the following study, we will take these regulators as core components to build <A href="https://2013.igem.org/Team:Peking/Project/BioSensors">a comprehensive set of biosensor circuits</A> for aromatics detection.</P><H1 id="SourceCodeTitle">Source Code</H1><P id="SourceCodeProA">Source code for protein sorting: <A href="https://static.igem.org/mediawiki/igem.org/b/b5/Peking2013_Mining_ProteinAnalysis.cpp.txt">Protein analysis.cpp,</A><A href="https://static.igem.org/mediawiki/igem.org/1/1f/Peking2013_Mining_Protein_heap.cpp.txt">Protein heap.cpp,</A><A href="https://static.igem.org/mediawiki/igem.org/b/bd/Peking2013_Mining_Proteinheap.h.txt">Proteinheap.h.</A></P><P id="SourceCodeCrawl">Source code for internet crawler: <A href="https://static.igem.org/mediawiki/igem.org/d/df/Peking2013_Mining_NewCrawler.cpp.txt">New crawler.cpp,</A><A href="https://static.igem.org/mediawiki/igem.org/1/1f/Peking2013_Mining_Protein_heap.cpp.txt">Protein heap.cpp,</A><A href="https://static.igem.org/mediawiki/igem.org/b/bd/Peking2013_Mining_Proteinheap.h.txt">Proteinheap.h,</A><A href="https://static.igem.org/mediawiki/igem.org/7/7e/Peking2013_Mining_Heap_oper.cpp.txt">heap_oper.cpp,</A><A href="https://static.igem.org/mediawiki/igem.org/d/d3/Peking2013_Mining_Heap.h.txt">heap.h.</A></P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Peking/Project/SensorMining">http://2013.igem.org/Team:Peking/Project/SensorMining</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Peking/Project/SensorMining" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Peking/Project/SensorMining" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Peking/Project/SensorMining&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Peking/Project/SensorMining&amp;oldid=349368" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>