"
Page
Discussion
View source
History
teams
Log in
 
Team:Peking/Project/BioSensors/NahR
From 2013.igem.org
Home
Team
Members
Notebook
Attributions
Project
Biosensor Mining
Biosensors
Adaptors
Band-pass Filter
Devices
Model
Band-pass Filter
Biosensor Fine-tuning
Data page
Parts
Judging Criteria
Safety
Human Practice
Questionnaire Survey
Visit and Interview
Practical Analysis
Team Communication
Biosensors
Biosensors
Individual Biosensors
XylS
XylR
HbpR
HcaR
HpaR
PaaX
DmpR
NahR
Multi-component Analysis
NahR
Mechanism
Build Our Own Sensor!
The nahR gene was mined from the 83 kb naphthalene degradation plasmid NAH7 of Pseudomonas putida, encoding a 34 kDa protein which binds to nah and sal promoters to activate transcription in response to the inducer salicylate. This plasmid encodes enzymes for the metabolism of naphthalene or salicylate as the sole carbon and energy source (Fig. 1a) [1] . The 14 genes encoding the enzymes for this metabolism are organized in two operons: nah (nahA-F), encoding six enzymes required for metabolizing naphthalene into salicylate and pyruvate, and sal (nahG-M), encoding eight enzymes which metabolize salicylate into intermediates of TCA cycle (Fig. 1b) [2] .
Previous work have confirmed that the cloned nah, sal, and nahR genes can be expressed and normally function in the heterologous host, Escherichia coli [3] . NahR is a member of LysR-family transcriptional factors, featured by a conserved N-terminal domain that contains a helix-turn-helix DNA-binding motif. It is σ70-dependent and functions via contacting the α-unit of RNAP [4] . Mutagenesis experiments revealed the organization of functional domains of NahR protein [5,6] . N terminal fragment (residues 23-45) accounts for binding DNA. Interestingly, the discovery of C terminal (residues 239-291) mutants unable to bind DNA suggested that the DNA binding requires the multimerization by C-terminal domain [6] . Additionally, mutations at residues 140-200 and 207-266 largely affected the binding specificity of inducers, indicating that those residues might serve as a ligand-binding pocket (Fig. 2) [6] .
As for the promoters NahR regulates, the -82 to -47 regions of nal and sal promoters are highly conserved, which suggests a consensus NahR-binding site (Fig. 3) [7] .
Several experiments conformed that NahR tightly binds to DNA in vivo in the presence or absence of salicylate. Either the amount or the affinity of NahR binding to DNA will be affected by salicylate in both E. coli and its native host Pseudomonas putida [7] . This along with the evidence from methylation protection experiments suggested a conformational change in the NahR•DNA complex before transcription activation (Fig. 4) [8] .
Wild-type NahR responds to its authentic inducer salicylate with the induction ratio higher than 20 [7] . NahR has been artificially evolved by mutagenesis to respond to new aromatic inducers such as substituted salicylates and substituted benzoates [6,9] The novel aromatics-sensing characteristics obtained from mutagenesis have been summarized in Table 1. This provides a rich repertoire for us to customize the aromatics-sensing characteristics of NahR protein.
Peking iGEM team has adopted a Biobrick BBa_J61051 to build a biosensor due to the high induction ratio of wild-type NahR. As described in Biosensor Introduction , we constructed a Psal/NahR biosensor circuit using sfGFP as reporter gene (Fig. 5). NahR is constitutively expressed by a constitutive promoter (Pc), and the expression of sfGFP is positively regulated by NahR in the presence of inducers.
Reassuringly, the NahR biosensor works without fine-tuning. ON/OFF test was then carried out using Test Protocol 1 . NahR biosensor exposed to no inducer showed negligible basal expression of sfGFP and 18 compounds showed significant activation effects with induction ratios higher than 20 (Fig. 6). They are listed as follows: SaA, 2-ABzO, 3-MeSaA, 4-MeSaA, 4-ClSaA, 5-ClSaA, AsPR, 2,4,6-TClPhl, 3-IBzO, 2-MeBzO, 3-MeBzO, 4-FBzO, 3-ClBzO, 3-MeOBzO, 3-HSaA, 4-HSaA, 5-ClSaD, 4-ClBzO ( Click Here for the full names of aromatic compounds).
Notably, in addition to salicylate derivatives, NahR biosensor was found to be able to sense 2,4,6-TClPhl, a kind of polychlorinated phenol that is significantly hazard to water environment and human health. This biosensing activity is a newly discovered characteristics of NahR.
Furthermore, the dose-response curves of NahR biosensor was experimentally measured using gradient concentrations of inducers ranging from 0.03 μM to 1 or 3 mM following Test Protocol 1 (Fig. 7). Hundreds-fold induction can be obtained at micro-molar concentration for SaA and its derivatives. Substituted benzoate also functions to activate NahR but with slightly lower induction ratio. As for 2,4,6-TClPhl, significant response could be elicited by the concentration of only 10 μM.
In summary, NahR biosensor works as a highly-sensitive and robust sensing device for salicylate homologs, benzoate derivatives and water-hazardous 2,4,6-TClPhl. We have systematically characterized its aromatics-sensing profile and accompanied dose responses, which has never been achieved in previous studies.
Figure. 1. Degradation pathway of naphthalene and the corresponding gene cluster in Pseudomonas putida. (a) Gene cluster on the NAH7 plasmid that degrades naphthalene: Naphthalene is degraded into salicylate by enzymes encoded by the "upper operon"; salicylate is further degraded to enter TCA cycle via the gene products of the "lower operon". Both of the operons are regulated by the transcription factor NahR in response to salicylate, the metabolite intermediate in the pathway. (b) Degradation pathway of naphthalene: Naphthalene is degraded by a series of enzymatic reactions, each catalyzed by a specific nah gene product represented by a capital letter. A through M: A, Naphthalene dioxygenase; B, cis-dihydroxy-naphthalene dioxygenase; D, 2-hydroxychromene-2-carboxylate isomerase; E, 2-hydroxybenzalpyruvate aldolase; F, salicylaldehyde dehydrogenase; G, salicylate hydroxylase; H, catechol 2,3-dioxygenase; I, 2-hydroxymuconate semialdehyde dehydrogenase; J, 2-hydroxymuconate tautomerase; K, 4-oxalcrotonate decarboxylase; L, 2-oxo-4-pentenoate hydratase; M, 2-oxo-4-hydroxypentanoate aldolase.
Figure. 2. The organization of NahR protein domains. The domain marked by green near the N terminal accounts for DNA binding, which contains a typical helix-turn-helix motif; red domains function to bind inducer, while the orange domain is putatively involved in multimerization of NahR during the transcription activation.
Figure. 3. Schematic diagram for the NahR-regulated promoters, nah and sal. Alignment of sal and nah promoter is shown and the consensus sequence motifs are highlighted in color. NahR binding sequence and RNAP binding sequence are boxed in green and yellow, respectively.
Figure. 4. Schematic diagram for the transcription activation at sal (or nah) promoter by NahR in the presence of inducer salicylate. 1. The DNA structure of sal promoter: A, B, C and D represent the binding sites for the tetramer of NahR; the yellow arrow shows the direction of sal promoter. 2. RNAP and σ70 bind to the sal promoter by recognizing -35 and -10 boxes; 3. Transcription factor NahR tightly binds to sal promoter and forms a tetramer no matter whether there is salicylate or not; 4. When salicylate is present, NahR•DNA complex undergoes a conformational change. After the hydrolysis of ATP, DNA is opened and transcription is activated.
Figure. 5. Schematic diagram for the NahR biosensor circuit. The Biobrick BBa_J61051 was cloned preceding reporter sfGFP in the backbone pSB1C3. Promoters are presented in orange, RBS in light green, coding sequence in dark blue and terminators in red.
Figure. 6. ON/OFF test to evaluate the induction ratios of all aromatic compounds in the aromatics spectrum.  (For the full names of the compounds, Click Here ). (a) The induction ratios of various aromatic species. NahR could respond to 18 out of 78 aromatics with the induction ratio over 20. (b) The aromatics-sensing profile of NahR biosensor.The aromatic species that can elicit strong responses of NahR biosensor are highlighted  in green in the aromatics spectrum. The structure formula of typical inducer is also listed around the spectrum. The induction ratio was calculated by dividing the fluorescence intensity of biosensor exposed to object inducers by the basal fluorescence intensity of the biosensor itself.
Figure. 7. Dose response curves of NahR biosensor. (a) Dose response curves for salicylate, its homologs and derivatives; (b) Dose response curves for benzoate, its derivatives and special inducers like 5-ClSaD and 2,4,6-TClPhl. The induction ratio was calculated by dividing the fluorescence intensity of biosensor exposed to inducers by the basal fluorescence intensity of the biosensor. For the full names of the compounds, Click Here .
References: [1] Dunn, N. W., and I. C. Gunsalus.(1973) Transmissible plasmid encoding early enzymes of naphthalene oxidation in Pseudomonas putida. J. Bacteriol. 114:974-979 [2] M. A. Schell.(1983) Cloning and expression in Escherichia coli of the naphthalene degradation genes from plasmid NAH7. J. Bacteriol. 153(2):822 [3] M. A. Schell, and P. E. Wender.(1986) Identification of the nahR gene product and nucleotide sequences required for its activation of the sal operon. J. Bacteriol. 116(1):9 [4] Woojun Park, Che Ok Jeon, Eugene L. Madsen.(2002) Interaction of NahR, a LysR-type transcriptional regulator, with the K subunit of RNA polymerase in the naphthalene degrading bacterium, Pseudomonas putida NCIB 9816-4. FEMS Microbiology Letters. 213:159-165 [5] Mark A. Schell, Pamela H. Brown, and Satanaryana Raju.(1990) Use of Saturation  Mutagenesis to Localize Probable Functional domains in the NahR protein, a LysR-type Transcription Activator. The Journal of Biological Chemistry. 265(7): 3384-3850. [6] Angel Cebolla, Carolina Sousa, and Vı´ctor de Lorenzo.(1997) Effector Specificity Mutants of the Transcriptional Activator NahR of Naphthalene Degrading Pseudomonas Define Protein Sites Involved in Binding of Aromatic Inducers. The Journal of Biological Chemistry. 272(7):3986-3992 [7] M. A. Schell, and E. F. Poser.(1989) Demonstration, characterization, and mutational analysis of NahR protein binding to nah and sal promoters. J. Bacteriol. 171(2):837 [8] Jianzhong Huang and Mark A. Schell.(1991) In vivo interaction of the NahR Transcriptional Activator with its target sequences. The Journal of Biological Chemistry. 266(17):10830-10838 [9] Hoo Hwi Park, Hae Yong Lee, Woon Ki Lim, Hae Ja Shin. (2005) NahR: Effects of replacements at Asn 169 and Arg 248 on promoter binding and inducer recognition. Archives of Biochemistry and Biophysics. 434:67-74
Table 1.  Comprehensive summary of NahR mutants and accompanied aromatics-sensing characteristics
Mutations
Expected Aromatics-sensing Profiles
References
Arg248→Cys
SaA, salicylamide, BzO, 2-ClBzO, 3-ClBzO
Lorenzo et al [6]
Asn169→Asp
SaA, BzO, 2-ClBzO, 3-ClBzO
Lorenzo et al [6]
Arg132→Cys
SaA, salicylamide, BzO, 3-ClBzO
Lorenzo et al [6]
Asn169→Asp / Arg248→Cys
SaA, BzO
Hae Ja Shin et al [9]
Asn169→Asp / Arg248→Lys
SaA, BzO
Hae Ja Shin et al [9]
Retrieved from " http://2013.igem.org/Team:Peking/Project/BioSensors/NahR "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
