<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BIT_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BIT/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>&lt;!DOCTYPE html&gt;<TITLE>Biology</TITLE></P><DIV class="navmenu navmenu-default navmenu-fixed-left offcanvas"><UL class="nav navmenu-nav"><LI><A href="https://2016.igem.org/Team:BIT">Home</A></LI><LI><A href="https://2016.igem.org/Team:BIT/Biology"><SPAN class="style1">Project</SPAN></A></LI><LI><A href="https://2016.igem.org/Team:BIT/Human_Practices">Practice</A></LI><LI><A href="https://2016.igem.org/Team:BIT/Judging_Form">Judging</A></LI><LI><A href="https://2016.igem.org/Team:BIT/Parts_Submission">Parts</A></LI><LI><A href="https://2016.igem.org/Team:BIT/HP/Silver">Achievement</A></LI><LI><A href="https://2016.igem.org/Team:BIT/Introduction">Team</A></LI></UL></DIV><DIV id="home"><DIV class="container text-center"><DIV class="content"><H4>PROJECT</H4><DIV class="header-text btn"><H1><FONT color="#c91f77">M</FONT>ODEL</H1></DIV></DIV></DIV></DIV><DIV id="subNav"><UL><LI><A href="https://2016.igem.org/Team:BIT/Biology"><H2>Biology</H2></A></LI><LI class="color1"><A href="https://2016.igem.org/Team:BIT/Model"><H2>Model</H2></A></LI><LI><A href="https://2016.igem.org/Team:BIT/Hardware"><H2>Hardware</H2></A></LI><LI><A href="https://2016.igem.org/Team:BIT/Software"><H2>Software</H2></A></LI><LI><A href="https://2016.igem.org/Team:BIT/Game"><H2>Game</H2></A></LI></UL></DIV><DIV id="contain"><DIV class="container"><DIV id="words" style="float:right;width:70%"><DIV class="row text-center"><H2>Background</H2></DIV><DIV class="indent"><P>
It became the hotspot of research of Life Science since miRNA has been found. It has been shown by Life Informatics that miRNA might be able to regulate more than a third of human gene, including wildly participating in many biological process of growth, differentiation, propagation, apoptosis, cycle regulation, shift and the formation of tumors. There are always changes of expression profile of miRNAs in many different varieties of human tumors, which is a king of latent powerful biological norm of assessing the tumor genesis, development, diagnosis, cure and prognosis. Specific changes of miRNAs’ expression profile of specific cancers are related to tumor genesis, metastatic, clinic pathological characteristics and prognosis, etc. </P><P>
As for the relationship of miRNAs and mammary cancers, Matron adopted chip technology to detect mammary cancers and the aberrant expression of miRNAs of normal mammary tissues matched them. <SUP>[2]</SUP> There were 29 diverse levels of expressions of miRNAs detected between normal mammary tissues and mammary cancer tissues. In addition, Volinia has found there were up-regulations of 15 miRNAs of mammary cancers.<SUP>[3]</SUP> Both two scholars’ researches have shown that the expression of the miRNA-155 was most significant different, and the expression of specific miRNA can be consorted to clinic pathological characteristics and biological and physical characteristics such as ductal type, mesophyll type, the type of metastatic of lymph gland, vascular invasion, proliferation index, the expression of estrogen and progesterone receptor and CerbB-2. The accuracy of experimental data is fairly essential since the experiment of detecting the expression levels of miRNAs is based on gene circuit, where lays the purport of mathematical modeling. </P><DIV class="row text-center"><H2>Introduction</H2></DIV><P>Gene lines in experienmental part as shown below: </P><CENTER><FONT size="1px">Figure1: genetic circuit schematic diagram</FONT></CENTER><P>
The main work of establishing the model research gene regulation network is through the analysis of gene expression data, combined with bioinformatics methods and techniques to build the appropriate topology and to simulate the regulatory mechanism of the cell system, in order to understand the system under the framework of life phenomena. The mathematical model not only integrates gene expression data, but also provides a powerful tool for understanding the dynamic behavior or static expression of gene regulatory networks. At present, the mainstream methods of modeling control network include Petri net, Bayesian network and differential equation model.</P><P>
In many methods, the differential equation can predict the behavior of the system quantitatively and accurately, and the stochastic model can fit the network accurately. The model needs to be simplified based on certain assumptions. In general, the computational cost is often increased as the description of the control network becomes deeper and more detailed. Differential equations have a wide range of applications in biomedicine, such as pharmacokinetics and enzymology, and are increasingly being used in the construction of gene regulatory networks.</P><DIV class="row text-center"><H2>Model</H2></DIV><P><B>
1 Integrase Model</B></P><P>
We establish the Bxb1 model to get the integrase conversion efficiency with the K (a set interval) to detect the trend of experimental data measured by the correctness. In the modeling process there are some parameters of biological simple process can not be obtained from the literature and the actual experimental operation, so the estimated value of these parameters.</P><P>Definition</P><CENTER><FONT size="1px">Chart 1: Vatiable defination</FONT></CENTER><P>Parameters</P><CENTER><FONT size="1px">Chart 2: Paraments Defination</FONT></CENTER><P>Chemical Kinetic Equations</P><P>Differential Equations</P><P>The fimE model as same as Bxb1 model, as blow:
Definition</P><CENTER><FONT size="1px">Chart 3: Variable definition</FONT></CENTER><P>Parameters</P><CENTER><FONT size="1px">Chart 4: Permanents Definition</FONT></CENTER><P>Chemical Kinetic Equations</P><P>Differential Equations</P><P>When the two gene circuits mediate the downstream genetic expression synergistically, they actually change the states of downstream promoters independently. So the simplest way to construct model for this synergetic system is to first separate the two upstream circuits and model them respectively. Then we combine them when calculating their effect on the downstream promoters. That is to say, we add another set of reactions, which is very similar to the one we discussed before, with Lacl replaced by tetR and Bxb1 replaced by Fime. To calculate the regulation of Fime and Bxb1 on the promoter of GFP, we use 4 variables to describe the states of the gene circuit. These 4 variables and their meanings are listed below.</P><P><B>
2 Lock-Key Model
</B></P><P>
In this experiment, two ribosome switches, lock1, key1 and lock3, key3 were used to regulate the background expression of recombinase FimE and BxB1, respectively, even though TetR and LacI proteins inhibited the promoters ptet and plac, but there was still a small amount MRNA is transcribed, it will affect the accuracy of detection. Using the ribosomal switch, the lock1 and lock3 genes, when they are not present or in small quantities, bind to the recombinase FimE and RBS in front of the BxB1 to form a circular structure, thereby preventing the ribosome from binding to it, thus preventing small amounts of mRNA When the amount of miRNA is high, the key1 and key3 genes can be translated, the expression product can unlock the ring structure formed by lock and RBS, and start the expression of the downstream gene, resulting in the expression of the key gene. GFP.
</P><P>Definition</P><CENTER><FONT size="1px">Chart 5: Variable?Definition</FONT></CENTER><P>Parameters</P><CENTER><FONT size="1px">Chart6: Parameters</FONT></CENTER><P>Chemical Kinetic Equations</P><P>Differential Equations</P><DIV class="row text-center"><H2>Result</H2></DIV><P><B>
1 Integrase model
</B></P><CENTER><FONT size="1px">figure 2: GFP level trend as KT (rate for Bxb1 to switch from ‘on’ state to ‘off’ state ) increases.</FONT></CENTER><P>From figure 2, we can see it saturates when KT reaches the value of  1e5/(M*M*min).</P><CENTER><FONT size="1px">Figure 3: The relationship between time and GFP 
（ KT=1e5/(M*M*min，km(lac)=5nM/min.)</FONT></CENTER><P>In figure 3, we can concluded that GFP level reaches its final stable value at about 600 minutes(10h). And this time is not sensitive to KT and km(lac).</P><CENTER><FONT size="1px">Figure 4: The relationship between time and GFP 
（ KT=1e5/(M*M*min，km(lac)=5nM/min.)</FONT></CENTER><CENTER><FONT size="1px">Figure 5: The relationship between time and GFP （KT=1/(M*M*min，km(lac)=5nM/min.)</FONT></CENTER><CENTER><FONT size="1px">Figure 6: The relationship between time and GFP
（KT=1e7/(M*M*min，km(lac)=100nM/min.)</FONT></CENTER><CENTER><FONT size="1px">Figure 7: The relationship between P(T(Bxb1)off) and KT</FONT></CENTER><CENTER><FONT size="1px">Figure 8: The relationship between lg[P(T(Bxb1)off)] and KT</FONT></CENTER><P>The relationship of P(T(Bxb1)off) with KT is almost identical to that of [GFP] with KT. So we can conclude that [GFP] is proportional to P(T(Bxb1)off).</P><CENTER><FONT size="1px">Figure 9: The relationship between GFP and km(lac)</FONT></CENTER><P>It is clearly that GFP level decreases as km(lac) increases, but saturates when km(lac) is too large or too small. But the reference value lies in the unsaturated scope. This indicates that the inhibition of transcription of lacl activates the expression of GFP.</P><CENTER><FONT size="1px">Figure 10: The relationship between lg(km(lac)) and P(T(Bxb1)off)</FONT></CENTER><P>So the figure 10 verified that GFP is proportional to P(T(Bxb1)off) .</P><CENTER><FONT size="1px">Figure 11: The relationship between GFP and km(lac)</FONT></CENTER><P>After we get experimental data(abscissa), we should corresponding to the ordinata to find out whether the trend in range, if within the range, it means our experiments are correct, otherwise it is necessary to continue the experiment to get correct data.</P><CENTER><FONT size="1px">fugure12:The relationship between lg(kt) and lg(GFP)</FONT></CENTER><P>We keep kt1(for Bxb1)and kt1(for FimE) the same, and vary them in the scope we used before. We find that the GFP output changes with kt1 and kt2 in a similar way compared to the independent gene circuits.</P><CENTER><FONT size="1px">Figure13:The relationship between GFP and t. </FONT></CENTER><P>Kt1=Kt2=1/(M*M*min) km(lac)=km(tet)=5e-5nM/min</P><CENTER><FONT size="1px">Figure14: The relationship between GFP and t.  </FONT></CENTER><P>Kt1=Kt2=1e7/(M*M*min) km(lac)=km(tet)=100nM/min
The morphology of the reaction curve is not obviously affected by these two parameters.</P><CENTER><FONT size="1px">Figure15:The relationship between lg(kt)and p(T(off)) </FONT></CENTER><CENTER><FONT size="1px">Figure16:The relationship between lg(kt)and lg(P(T(off))) </FONT></CENTER><P>The relationship of P(Toff) with kt is almost identical to that of GFP with kt. So we can conclude that GFP is proportional to P(Toff).(Here two kt and two P(Toff))s are both the same.</P><CENTER><FONT size="1px">Figure17: The relationship between GFP and lg(km) </FONT></CENTER><P>GFP level decerases as km increases, but saturates when km is too large or too small.But the reference value lies in the unsaturated scope. This indicates that the inhibition of transcription of Lacl and tetR activates the expression of GFP.</P><P>2 Lock-Key Model</P><CENTER><FONT size="1px">Figure18:The noise floor of bacterial fluid </FONT></CENTER><CENTER><FONT size="1px">Figure19: The GFP expression of FimE was caused by the reverse action of the enzyme.</FONT></CENTER><CENTER><FONT size="1px">Figure 20: The GFP expression of between FimE and lock-key systerm were caused by the reverse action of the enzyme.</FONT></CENTER><CENTER><FONT size="1px">Figure 21: The comparison between figure19 and figure20</FONT></CENTER><CENTER><FONT size="1px">Figure 22: The GFP level of different mir155 concentration inducing</FONT></CENTER><DIV class="row text-center"><H2>Discussion</H2></DIV><P>The results of the model are simulated with MATLAB software, then discussed the results. By setting up a k-value interval, calculated the relationship between KT and GFP level, we concluded GFP level are increasing as value of k increasing, when KT = 1e5, GFP level in steady stage. In figure 3, we can concluded that GFP level reaches its final stable value at about 600 minutes(10h). And this time is not sensitive to KT and km(lac). When changed kt and km(lac), the relationship between GFP level and time in common. GFP level decreases as km(lac) increases, but saturates when km(lac) is too large or too small. But the reference value lies in the unsaturated scope. This indicates that the inhibition of transcription of Lacl activates the expression of GFP. Figure 10 verified that GFP is proportional to P(T(Bxb1)off). Not only can we concluded the FimE has influenced on GFP level as same as Bxb1 from figure, but also </P><P>The results of the model are simulated with MATLAB software, then discussed the results. The value of GFP induced by mir155 is regarded as signal and the one without inducing is regarded as noise floor. From the figure 21, we can conclude that the system equipped with Lock and Key part has the higher signal to noise ratio (SNR) than the one without Lock and Key part. And Lock and Key part significantly reduce the background noise of detection. From the figure 22, we can conclude the Lock and Key part will not influence the linearity of detection and the correlation between GFP and concentration of mir155 is correct according to the figure.</P><P>
In this paper, lots of parameters found in the literature, which is not accurate enough. In order to accurate model to check out experimental data, the relevant parameters can be added and modified, and the differential equation model can be modified and improved. Using differential equation modeling is only one of solutions, we also use colored Petri nets modeling and Bias theory modeling. Petri is a graphical and mathematical modeling tool which can be used in many kinds of systems. It provides a powerful means to describe and study the information processing system with parallel, asynchronous, distributed and stochastic characteristics. The main characteristics of Petri network include parallelism, uncertainty, asynchronous and description ability and analysis ability of the distributed system, so it can be used in many practical systems and fields. Petri network is a dynamic graphical tool, it has a similar visualization flow chart, network diagram and description of function, but also through the token (Token) flow to simulate the dynamic behavior of the actual system. Bias introduced the network directed graph model, reveals the causal relationship between the gene expression level of statistical hypothesis based on the linear model, nonlinear model and hidden Markov model as a special case, including covers, the introduction of random element and hidden variables, and can well handle the hidden variable, can clearly model for data acquisition the process, can handle the problem of missing data and noise data, but also the confidence of different characteristics of the network.</P><P>
When the experiment has been done and the data obtained, this model can be applied to examine the accuracy of the data obtained by experiment. We advance that the expression level of miRNA should be obtained by experimental process, basing on the current data document to erect the database and build up the relationship between it and the possibility of getting mammary cancers or clinic pathological characteristics, which lead to our primary predictive result. This result is not assured to be 100 percent correct, but it is not hard to imagine that in the near future, we can make full use of the miRNA as a tumor’s signature, and this model will be labeled with foreknowledge and significant directive. However, since we could not extract basic data from documents, we had to take deep cooperation with a hospital in order to collect a large number of samples, which was far beyond our ability. Regretfully, we had to give up this idea. But by communicating with doctors of oncology department thoroughly, doctors found our idea was very significant meaningful and very suitable for popularizing, and the only thing we need was just more time.   </P><P>
We have improved the problem that reaction rate of the very reversible reaction is the same. The reaction rate is the same, which is the result of the reaction coming to chemical equilibrium, and one of the key points of this model is the beginning of the 20 minutes of each reaction, If we design the model directly from the equilibrium rate, it will have a huge impact on the follow-up results. Based on the estimates of these biological responses, we set a large difference between the positive and negative reaction rate. Considering the consistency of the FimE loci and the terminator in terms of position and biochemical reactions, we have simplified the variables and parameter settings of USTC Team, only need to use TOFF concentration to undertake FimE and mRNA GFP, without the middle of the SABxb1, SIBxb1, SFBxb1, etc. This change is not to simplify the operation, but in order to better fit the actual, the fit should be increased. A control group without mir155 was not set up so the modeling effect was not apparent (FImE conversion effect is reflected in the concentration of GFP which is decided by the presence or absence of mir155). In addition to improving the above problems, this model also reflects the following characteristics: The model is basically consistent with the general experimental fluorescence / OD curve, performs well in terms of time fit. The model is basically consistent with the general experimental fluorescence / OD curve, performs well in terms of time fit. The model can be used to determine the concentration of mir155 by signal intensity under certain conditions and to achieve semi-quantitative measurement in the presence of high concentration of mir155 (the effect of miRNA on the system decreases with increasing concentration), It can be obtained with the confidence of the results within 200min (signal to noise ratio 3.9), the signal to noise ratio will rise to 8 with the rise of time.
</P><DIV class="row text-center"><H2>Lab Note</H2></DIV><DIV class="row text-center"><H2>Reference</H2></DIV><P><I>[1] Wang F, Zheng Z, Guo J, et al. Correlation and quantitation of microRNA aberrant expression in tissues and sera from patients with breast tumor[J]. Gynecologic Oncology, 2010, 119(3):586-93.
[2] Liu J, Mao Q, Yan L, et al. Analysis of miR-205 and miR-155 expression in the blood of breast cancer patients[J]. Chinese Journal of Cancer Research, 2013, 25(1):46-54.
[3]Basu S., Gerchman Y., Collins C.H., Arnold F.H., Weiss R., A synthetic multicellular system for programmed pattern formation, Nature, 2005
[4] Liang S. T., Ehrenberg M., Dennis P., Bremer H.,Decay of rplN and lacZ mRNA in Escherichia coli, Journal of molecular biology, 1999</I></P></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="testimonials"><DIV class="overlay"><DIV class="container"><DIV class="section-title"><H2>Message Board</H2></DIV><DIV id="testimonial"><DIV class="item"><H3>iGEM is a Jamboree! </H3><H6>Wang Xu</H6></DIV><DIV class="item"><H3>Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum. Duis mollis, est non commodo luctus, nisi erat </H3><H6>Kai, Web Geekster</H6></DIV><DIV class="item"><H3>Cras justo odio, dapibus ac facilisis in, egestas eget quam. Donec id elit non mi porta gravida at eget metus.</H3><H6>Jenn, Coders' Playground</H6></DIV></DIV></DIV></DIV></DIV><DIV id="contact"><DIV class="container"><DIV class="section-title text-center"><H2>Contact Us</H2></DIV><DIV class="row"><DIV class="col-md-3"><ADDRESS><STRONG>Address</STRONG>
					 Beijing Institute of Technology,
					No. 5 South Zhong Guan Cun Street,
					Haidian Beijing 100081, P. R. China
					Twitter : @igem_BIT

Sina Weibo : @igem_BIT

Website : http://www.bit.edu.cn
					</ADDRESS></DIV><DIV class="col-md-9"><FORM autocomplete="off"><DIV class="text-right"><A href="#" class="btn send-btn">Send</A></DIV></FORM></DIV></DIV></DIV></DIV><NAV id="footer"><DIV class="container"><DIV class="pull-right"><A href="#home" class="page-scroll">Back to Top </A></DIV></DIV></NAV></DIV></DIV></DIV></BODY></HTML>