<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BIT_Biology skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BIT/Biology</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>&lt;!DOCTYPE html&gt;<TITLE>Biology</TITLE></P><DIV class="navmenu navmenu-default navmenu-fixed-left offcanvas"><UL class="nav navmenu-nav"><LI><A href="https://2016.igem.org/Team:BIT">Home</A></LI><LI><A href="https://2016.igem.org/Team:BIT/Biology"><SPAN class="style1">Project</SPAN></A></LI><LI><A href="https://2016.igem.org/Team:BIT/Human_Practices">Practice</A></LI><LI><A href="https://2016.igem.org/Team:BIT/Judging_Form">Judging</A></LI><LI><A href="https://2016.igem.org/Team:BIT/Parts_Submission">Parts</A></LI><LI><A href="https://2016.igem.org/Team:BIT/HP/Silver">Achievement</A></LI><LI><A href="https://2016.igem.org/Team:BIT/Introduction">Team</A></LI></UL></DIV><DIV id="home"><DIV class="container text-center"><DIV class="content"><H4>PROJECT</H4><DIV class="header-text btn"><H1><FONT color="#c91f77">B</FONT>IOLOGY</H1></DIV></DIV></DIV></DIV><DIV id="subNav"><UL><LI class="color1"><A href="https://2016.igem.org/Team:BIT/Biology"><H2>Biology</H2></A></LI><LI><A href="https://2016.igem.org/Team:BIT/Model"><H2>Model</H2></A></LI><LI><A href="https://2016.igem.org/Team:BIT/Hardware"><H2>Hardware</H2></A></LI><LI><A href="https://2016.igem.org/Team:BIT/Software"><H2>Software</H2></A></LI><LI><A href="https://2016.igem.org/Team:BIT/Game"><H2>Game</H2></A></LI></UL></DIV><DIV id="contain"><DIV class="container"><DIV id="words" style="float:right;width:70%"><DIV class="row text-center"><H2>Background</H2></DIV><DIV class="indent"><P>Breast cancer is one of the most common malignant tumors among women. About one million and two hundred thousand women suffer from breast cancer and five hundred thousand women die of breast cancer every year around the world. The morbidity of breast cancer is always the top of the list. However, the cure rate of breast cancer is higher than that of others. In Western Europe, North America and other developed countries, the morbidity of breast cancer is the highest among women who suffer from cancer. In America, per one hundred thousand people, there are 129.9 breast cancer patients. This number is 48.16 in Japan, 43.71 in Hong Kong, 17.09 in China and 13.94 in Korea.</P><P>Biomarkers usually refer to indicators that are used to measure physiological state or condition. Biomarkers have many applications in measuring and evaluating physiological processes, pathogenic process or in the process of drug treatment. At present, it has important significance not only in the medical field, but also in the field of molecular epidemiology, molecular toxicology and environmental medicine.</P><CENTER><FONT size="1px">biomarkers associated with breast cancer<SUP>[1-16]</SUP></FONT></CENTER><P>MicroRNAs are a class of small, non-coding RNA molecules, containing about 22 nucleotides. MicroRNAs fold itself in a way similar to RNA interference (RNAi), forming a short hairpin structure. It is the precursor substance which Single-stranded RNA forms by folding itself. Then enzyme DICER cleaves it into miRNA (about 22 nt in length) <SUP>[17-18]</SUP>. MicroRNAs mainly have three ways to regulate the expression of gene. The first way is to cleave mRNA of the target gene. miRNA completely complementary combine with target genes and cleave mRNA finally. The mechanism and function is very similar with siRNA. The second way is to inhibit the effect of the translation of target genes through incomplete combination with target genes. In this way translation is suppressed without affecting the stability of the mRNA. The third way Is the combination of the above two methods. When miRNA completely complementary combine with target genes, the target mRNA will be cleaved. When miRNA incompletely complementary combine with target genes, miRNA play a role of regulating the expression of gene. Research shows that the levels of many kinds of microRNA in the sick body are different. Besides, for different diseases, the levels and types of miRNA are different. In other words, miRNAs can be disease markers. Among the vast majority of cancer patients, in their body the levels of miRNAs change compared with the normal level. For example, In the human body suffering from breast cancer, the levels of miR-21、miR-155、miR-244、miR-139 increase, while the levels of miR-497、miR-31 decrease.</P><CENTER><FONT size="1px"> miRNAs associated with breast cancer<SUP>[19-27]</SUP></FONT></CENTER><DIV class="row text-center"><H2>Principle</H2></DIV><DIV style="text-indent: 0em;"><P><B>1. Live cell Logic gate genetic components</B></P></DIV><P>Bonnet and Siuti, who use recombinant enzymes to build a logic gate which is different from the traditional logic gate routes. They insert phage DNA into the host genome integrase site. The integrase recognizes a pair of distinct sequences, called attP (on phage) and attB (in the host), which are different from each other. After recombination and insertion of the phage, attP and attB are flipped and converted into a pair of sites called attL and attR, which are no longer substrates for the recombinase. Using attP and attB sites, deliberately construct a modified DNA sequence, you can &quot;cheat&quot; recombinase, so that the DNA sequence occurs to an irreversible reversal.</P><P>In the figure, the GFP protein is expressed in the &quot;0&quot; state. However, when the recombinase site was specifically inverted, it became &quot;1&quot; and the left RFP protein was expressed</P><DIV style="text-indent: 0em;"><P><B>2. RNA-based gene silencing expression</B></P></DIV><P>RNA interference and related gene silencing led to changes in the metabolic pathway has completely changed people's understanding of gene regulation. Gene silencing technology has been used as a research tool to control the expression of certain cell genes. In eukaryotes, RNA interference manipulates many cellular functions as another way of gene silencing. Especially in mammals, it can regulate gene expression, analyse signal transduction gene interaction, block the pathogenesis of the disease gene. Zheng Xie<SUP>[30]</SUP> et al. designed a transcriptional (post-transcriptional) regulatory gene pathway, which can sense the expression of endogenous microRNAs and cause intracellular reactions only by sensing the level of microRNA expression level. The schematic diagram below designs a sequence that specifically binds to the corresponding microRNA downstream of the gene line and gives a final output signal by sensing the level of microRNA expression.</P><CENTER><FONT size="1px">Figure | Sensing microRNA expression diagram</FONT></CENTER><DIV style="text-indent: 0em;"><P><B>3. Riboswitch</B></P></DIV><P>Riboswitch plays an important role in gene regulation in the prokaryotic system. Riboswitch achieve the &quot;switch&quot; function mainly by changing ribonucleic acid (RNA) conformation, prevent or open the target protein production. Most Riboswitches have only one binding site or aptamer that recognizes a targeting ligand. Aptamers are usually located near the gene expression region. When combined with metabolites, it will change its structure and exercise the function of gene regulation at the level of transcription or translation. Riboswitch can be &quot;turned on&quot; and re-activated for transcription or translation by a regulatory sRNA (here referred to as a small fragment of regulatory RNA, a non-coding RNA that is important for eukaryotic and prokaryotic regulatory action) <SUP>[31]</SUP>. F.J. Isaacs et al. in J. J. Collins task group constructed a modular ribosome switch. They first inserted a short nucleotide sequence complementary to the RBS upstream of the DNA, named the cis-suppression sequence, located between the 5'-UTR and RBS sequences downstream of the promoter. The insertion of the cis sequence does not alter the coding sequence of the target gene, nor does it affect the rate of transcription. The cis sequence consists of two parts: a hairpin sequence consisting of 19 nucleotide residues complementary to the RBS and a loop domain consisting of 6 nucleotide residues. The 5'-UTR of the crRNA (cis-repressed MRNA) produced by transcription is capable of folding and masking RBS, thereby preventing the translation of functional proteins <SUP>[32]</SUP>. The principle is as follows.</P><CENTER><FONT size="1px">Figure | An artificial Riboswitch system for controlling post-transcriptional gene regulation</FONT></CENTER><DIV class="row text-center"><H2>Design</H2></DIV><P>In the system we designed, we introduced three unique and innovative parts including gene silence part based on microRNA, Lock &amp; Key part and double integrases part. We use them to achieve joint detection of breast cancer-related markers microRNA-21 and microRNA-155. The system principle as follows:</P><P>When microRNA-155 and microRNA-21 are absent or low expression, the tetR and LacI proteins are expressed under the promoter of the constitutive promoter, thereby inhibiting the initiation of the downstream of the ptet promoter and the plac promoter, so that the expression of FimE and BxBl recombinase is not expressed or low expression. At this time, downstream integrase site does not occur specific inversion. </P><P>When microRNA-155 and microRNA-21 were present, the specific binding of microRNA and mRNA (transcription of binding sites) prevented the expression of tetR and LacI proteins. Thus, the inhibition of ptet promoter and plac promoter was released and recombinase was expressed, recognition site-specific flipped, site-controlled terminator was flipped, downstream gene expression was opened. As a result, fluorescent protein was expressed by reporter gene.</P><P>In addition, we add ribosome switches before the expression of FimE and BxB1, which aims to reduce the background noise expression of FimE and BxB1 recombinase and enhance the stability of the system.</P><P>The advantages of the system are as followed: First of all, the features of gene silence make the detection of microRNA possible and effective in the engineering bacteria. Secondly, employing the Lock &amp; Key part in our system reduce the background noise in order to achieve the lower limit of detection. Ultimately, the joint determination of microRNA-21 and microRNA-155 is feasible by the design of double integrases part. All in all, the system realizes the highly sensitive simultaneous determination of microRNA-21 and microRNA-155, which are related to the breast cancer.</P><DIV style="text-indent: 0em;"><P><B>1. Live cell Logic gate genetic components</B></P></DIV><P>Bonnet and Siuti, who use recombinant enzymes to build a logic gate which is different from the traditional logic gate routes. They insert phage DNA into the host genome integrase site. The integrase recognizes a pair of distinct sequences, called attP (on phage) and attB (in the host), which are different from each other. After recombination and insertion of the phage, attP and attB are flipped and converted into a pair of sites called attL and attR, which are no longer substrates for the recombinase. Using attP and attB sites, deliberately construct a modified DNA sequence, you can &quot;cheat&quot; recombinase, so that the DNA sequence occurs to an irreversible reversal.</P><P>In this experiment, the recombination enzymes, FimE and BxB1, form an AND gate. When FinE and BxB1 coexist and interact with IRR, IRL, attP and attB, the two termini and the intermediate terminator will be flipped 180 °. So that the original positive terminator flip for the reverse of the terminator and the terminator lost the function of termination of transcription convert into a common base sequence. So GFP can be expressed. When only one of the two recombinases is present or nonexistent, the gene line is blocked and GFP cannot be produced.</P><DIV style="text-indent: 0em;"><P><B>2. RNA-based gene silencing expression</B></P></DIV><P>RNA interference and related gene silencing led to changes in the metabolic pathway which completely change people's understanding of gene regulation. Gene silencing technology has been used as a research tool to control the expression of certain cell genes. In eukaryotes, RNA interference (RNAi) acts as another pathway for gene silencing to manipulate many cellular functions. Especially in mammals, it can regulate the expression of genes, analysis of signal transduction of gene interaction, and block the pathogenesis of the lesion genes.</P><P>In our experiment, miRNA-21 and miRNA-155 are used as the two detection materials. We identified the recognition sites of miRNA-21 and miRNA-155 by reviewing the literature, which is essentially a base sequence. When no miRNAs were present or the concentration of miRNAs were low, the combined promoters located in front of TetR and LacI will initiate the expression of these two proteins. The expressed products TetR and LacI inhibit the activity of the ptet and plac promoters in the downstream gene lines. So the promoters cannot start the recombinase FimE and BxB1 expression, so their respective recombinase binding sites cannot be reversed, the result is unable to express GFP.When the expression of miRNA is hiagher, it will be matched with the mRNA base pairing which is transcribed from its recognition site, forming a small amount of mRNA double-stranded, and the ribosome cannot be combined with it, so the translation is blocked, the synthesis of TetR and LacI protein is blocked, and the promoter ptet and plac cannot be repressed, the downstream gene lines can be activated, resulting in recombinase FimE and BxB1, so that the terminator between the binding sites of the recombination flip with losing the termination of transcription, the result is GFP can be expressed.</P><DIV style="text-indent: 0em;"><P><B>3. ribosomal switch (riboswitch)</B></P></DIV><P>Riboswitch plays an important role in gene regulation in the prokaryotic system. Riboswitch mainly functions as &quot;switch&quot; by changing the conformation of ribonucleic acid (RNA), which prevents or opens the production of target protein. Most Riboswitches have only one binding site or aptamer that recognizes a targeting ligand. Aptamers are usually located near the gene expression region, when it is combined with metabolites, it will change its structure to regulate gene express at the level of transcription or translation. Riboswitch can “open” by a regulatory sRNA (herein referred to as a small fragment of regulatory RNA, a non-coding RNA that is critical for eukaryotic and prokaryotic regulatory action), re-activated for transcription or translation. J. J. Collins, F.J. Isaacs et al. Constructed a modular ribosome switch. They first introduced a short, ribonucleotide sequence complementary to RBS upstream of the DNA, designated the cis-suppression sequence, between the 5'-UTR and RBS sequences downstream of the promoter. The introduction of the cis sequence does not alter the coding sequence of the target gene, nor does it affect the rate of transcription. The cis sequence consists of two parts: a hairpin sequence consisting of 19 nucleotide residues complementary to the RBS and a loop domain consisting of 6 nucleotide residues. The 5'-UTR of the crRNA (cis-repressed RNA) produced by transcription is capable of folding to mask RBS, thereby preventing the translation of functional proteins.</P><P>In this experiment, two riboswitches called lock1, key1 and lock3, key3 were used to regulate the background expression of recombinase FimE and BxB1 respectively. Because even though TetR and LacI proteins inhibited the promoters ptet and plac, there was still a small amount mRNA is transcribed. It will decrease the accuracy of detection. When they are not present or in small quantities, the riboswitches will bind to the RBS which is located in front of recombinase FimE and BxB1. They will form a circular structure and prevent the riboswitch feom binding to it. So it can prevent the translation of the small amounts of mRNA. When the level of miRNA is higher, the key1 and key3 genes can be translated, the expression product can unlock the ring structure formed by lock and RBS. And the expression of the downstream gene will open and GFP can be expressed.</P><DIV class="row text-center"><H2>Materials and methods</H2></DIV><DIV style="text-indent: 0em;"><P><B>1. Agarose gel electrophoresis preparation</B></P></DIV><P>Take a glass bottle, add 25mL 1 x TAE buffer to the bottle. Then weigh 0.25g agarose, add in the bottle, and use the microwave oven to heat about 2min, so that the liquid can reach the boiling state. Adding 2.5μL genecolor to the bottle and shake the mixture. Take the glue box out, insert comb, pour the mixture to the glue box. Then stand above 20min at room temperature.</P><DIV style="text-indent: 0em;"><P><B>2. Gel extraction</B></P></DIV><P>Cut agarose gel containing the target DNA below the ultraviolet lamp. Use a paper towel to absorb the liquid from the gel surface. Then take it to a 1.5mL centrifugation tube, and calculate the gel weight. Transferred the agarose to 1.5mL centrifugation tube with 3 Buffer DE-A of the volume of the gel, mixing in 75℃ water. After about 3min, the gel is completely melted. Add DE-B Buffer of 1.5 gel volume. Mix evenly. When the isolated DNA fragment is less than 400bp, 1 gel volume of isopropyl alcohol should be added. Mix and absorb the solution, and transfer to DNA preparation tube. The next step will be to prepare the DNA tube placed in the 2mL centrifugation tube to centrifuge for 1min in 12000rpm. Discard the waste liquid in the centrifugation tube. Then, DNA preparation tube is prepared to return the 2mL centrifugation tube, and add 500L Buffer W1 to DNA preparation tube for 1min in 12000rpm. Discarded waste liquid. Return 2mL centrifugation tube with DNA preparation tube. After adding 500L Buffer W1 to centrifuge for 30s in 12000rpm. discard the waste liquid in the centrifugation tube and put the DNA preparation tube into the 2mL centrifugation tube. like the previous step, add 500μL Buffer W2 and centrifuge for 1min in 12000rpm. discard the waste liquid. DNA preparation tube will be prepared to put back 2mL centrifugation tube. then centrifuge for 1min in 12000rpm. Also abandon the waste liquid in the centrifugation tube. The DNA preparation tube is arranged in the 1.5mL centrifugation tube, and the center of DNA preparation tube is prepared by adding 30μL sterile water. Stand in room temperature for 1min, and then centrifuge for 1min in 12000rpm, in order to elute DNA.</P><DIV style="text-indent: 0em;"><P><B>3. Cleavage</B></P></DIV><P>Take out the PCR tubule, add 20μL enzyme digestion system: 10μL plasmid solution, 7μL sterile water, 2μL Buffer 2.1, 0.5μL endonuclease A and 0.5μL endonuclease B. Shake the PCR tubule of the enzyme digestion system slightly and make sure the system is mixed evenly. Then put the tubule in 37 ℃ thermostat. It should last at least 2h.</P><DIV style="text-indent: 0em;"><P><B>4. Transformation</B></P></DIV><P>Take the feeling state cells 100μL to the ice box. Take the alcohol lamp, tweezers and sterile centrifugation tubes and sterile gun head out. light the alcohol lamp to make sure the biological experiment is sterile as far as possible. Take out the 1.5mL sterile centrifugation tube, put 50μL cells in it. Then there will be two tube. The two centrifugation tubes which are provided with the feeling state cells are placed in the ice box. Use sterile gun head to take out 5μL plasmid in plasmid plate, add into the two centrifugation tube. Put the centrifugation tube in the ice box and set 30min. Then put centrifugal tube into water bath cauldron in 42℃ for 45s. Take back in the ice box for 2min. Take out no anti-liquid culture medium, Use sterile gun head to remove 500μL liquid added to the centrifugation tube. The centrifugal tube is taken into 37 ℃s Celsius table for an hour.</P><DIV style="text-indent: 0em;"><P><B>5. Plasmid DNA extraction</B></P></DIV><P>Place the adsorption column CP3 into the collecting tube. The adsorption column CP3 is added to the 500μL equilibrium solution BL, and centrifuge for 1min in 12000rpm. Take out the adsorption column CP3, random the equilibrium solution, and put the adsorption column in the collection tube. Take 4~5mL bacteria liquid and add to the centrifugation tube. centrifuge for 1min in 12000rpm. add 250μL solution P1 (check RNaseA has been added) to the centrifugation tub. using tubette repeated tubetting, thoroughly resuspended bacterial precipitation. 250μL solution P2 is added to the centrifugation tube, and the tube is turned up and down for 6~8 times to make the cell fully cracked. Adding 350μL solution P3, immediately mild flip up and down for 6~8 times. white floc appears. The centrifugation tube is centrifuged for 10min in 12000rpm, and the supernatant solution is removed and transferred to the adsorption column CP3. The collecting tube of the adsorption column CP3 is centrifuged for 1min in 12000rpm. the supernatant solution is poured out.  the adsorption column CP3 is put in the recycling collecting tube. Add 600μL PW to the adsorption column CP3(check whether the rinsing liquid PW is added to ethanol). centrifuge for 1min in 12000rpm. the supernatant solution is poured out, and do the last step again (to add PW and centrifuge). The collecting tube of the adsorption column CP3 is provided with a 12000rpm centrifugal for 2min. and the supernatant is poured out. put the adsorption column CP3 into a clean 1.5mL centrifugation tube. the center of CP3 is prepared by adding 30μL sterile water. place at room temperature for 2min. centrifuge for 2min in 12000rpm, and the plasmid solution can be collected in the centrifugation tube.</P><DIV style="text-indent: 0em;"><P><B>6. Primer</B></P></DIV><DIV class="row text-center"><H2>Results</H2></DIV><DIV style="text-indent: 0em;"><P><B>1. Gel electrophoresis</B></P></DIV><P>According to this picture, the NO.1 and No. 2 are J23104+B0034+Mir155 Binding site+Gfp (about 1200bp)</P><P>J23104(35bp)+B0034(12bp)+Mir155(97bp)+Gfp(878bp)+B0015(129bp)=1151(bp)</P><P>J23104(35bp) +B0034(12bp)+Mir21(90)+Rfp(706bp)+B0015(129bp)=972(bp)</P><P>So, the picture proves that that the length of our device is right. No.3 gene located on 30000bp is target gene (972bp) and backbone(2000bp).</P><P>Ptet(54bp)+B0034(12bp)+tetR(685bp)+B0034(129bp)+FimE(558bp)+B0015(129bp)+pLac(35bp)+B0034(129bp)+LacI(1125bp)+B0034(12bp)+BXB1(1503bp)+B0015(129bp)+IRL+B0010+IRR+attB+B0010+attP(300bp)+B0034(12bp)+Gfp(878bp)+B0015(129bp)=5809(bp)</P><P>So, the picture prove that that the length of our device is right.(No.2,No.4,No.6)</P><P>Pconst(55bp)+B0034(12bp)+Mir21(90bp)+LacI(1125bp)+B0015(129)+Plac+key(plac and key 90bp)+Plac+Lock(Plac and Lock 126bp)+Gfp(878bp)+B0015(129bp)=1756(bp)</P><P>So, the picture prove that that the length of our device is right.(No.2,No.4,No.6)</P><P>Pconst(55bp)+B0034(12bp)+Mir21(90bp)+LacI(1250bp)+B0015(129)+Plac+key(plac and key 90)+Plac+Lock(Plac and Lock 126bp)+BXB1(1503) +B0015(129bp)=3384(bp)</P><P>So, the picture prove that that the length of our device is right.(No.2,NO.4)</P><DIV style="text-indent: 0em;"><P><B>2. Sequencing</B></P></DIV><P>We sequencing and alignment J23104-B0034-mi155-4-E0040-B0010-B0012</P><P>The identities are 99% right because of gene sequencing error and gene mutation. So this picture prove our device is right.</P><DIV style="text-indent: 0em;"><P><B>3. Experiment test</B></P></DIV><P>This is the growth curve of bacteria and from this figure, we can see that as time goes on, the bacteria reproduce which results in the rising concentration and OD value of bacterium. As for different experimental groups, all the groups with different concentration of mir155 have almost the same OD value, which indicate the growth of bacteria are synchronous during the detection of florescence. The factor of concentration of mir155 does not have strong impact on the growth of bacteria which accords with expectations of experimental results.</P><P>This figure show the fluorescence value generated by per unit bacteria volume which is also the experimental result of Part BBa_K2041021. When the mir155 presents in the system, mir155 will combine with the binding site. Meanwhile, the combination will influence the transcription and translation of downriver gene. Therefore, the expression of fluorescence will decrease and with the higher concentration of mir155, the expression of GFP will bring down gradually. After culturing for a while, fluorescence value generated by per unit bacteria volume will tend to a stable value.</P><P>This is the growth curve of bacteria and from this figure, we can see that as time goes on, the bacteria reproduce which results in the rising concentration and OD value of bacterium. As for different experimental groups, all the groups with different concentration of tetR have almost the same OD value, which indicate the growth of bacteria are synchronous during the detection of florescence. The factor of concentration of tetR does not have strong impact on the growth of bacteria which accords with expectations of experimental results.</P><DIV class="row text-center"><H2>Discussion</H2></DIV><P>We have successfully built a detection system based on logic gates. We can find promoters associated with the disease markers and simultaneously detect two markers, for example, we can simultaneously detect diabetes and diseases with NO as a marker in the blood.</P><P>At the same time, we test the effect of miRNA-based inhibition of the expression in the design. If we label the corresponding aptamers by treating the macromolecular markers and then let the small molecule aptamers enter the cells by replacing of the aptamers with a certain number of fragments, we can detect macromolecules, which provides a good research idea for many markers whose detection means are complex.</P><P>We put the biological experiment and biological modeling together, introduce implicit functions and random variables into linear models or nonlinear models, which lead to a good simulation to our results, analyze biology problems more effective, and support our experiments more scientific.</P><DIV class="row text-center"><H2>Lab Note</H2></DIV><DIV style="text-indent: 0em;"><P><B>January 1st,2016-March 1st,2016</B></P></DIV><P>iGEM learning, literature researching, previous projects analyzing and preliminary decisions making.</P><DIV style="text-indent: 0em;"><P><B>March 1st,2016-April 1st,2016</B></P></DIV><P>Original project planning and reagents purchasing.</P><P>Finishing the following circuits.</P><P>Circuit 1:</P><P>Circuit 2：</P><DIV style="text-indent: 0em;"><P><B>April 1st,2016-May 1st,2016</B></P></DIV><P>Finishing the following circuits and synthesizing the following gene sequence.</P><P>Circuit 3：</P><P>GGAATTCCTCTAGAGTTTCTCCTCTTTAAGCTTTCGGCCGGCTTGTCGACGACGGCGGTCTCCGTCGTCAGGATCATCCGGGCCCAGGCATCAAATAAAACGAAAGGCTCAGTCGAAAGACTGGGCCTTTCGTTTTATCTGTTGTTTGTCGGTGAACGCTCTCGGGTTTGTACCGTACACCACTGAGACCGCGGTGGTTGACCAGACAAACCACGACTCGAGAAAGAGGAGAAATCTATGAGTCAAAATGGCCCCAATTGTCTTGTATTTCACACTGGCTCACCTTCGGGTGGGCCTTTCTGCGTTTATAAAGATGAAACATTTGGGGCCAAACTGTCCATATTAGACTAGTCGCTGCAGG</P><DIV style="text-indent: 0em;"><P><B>May 1st,2016-June 1st,2016</B></P></DIV><P>Finishing the following circuits.</P><P>Circuit 4：</P><P>Circuit 5：</P><DIV style="text-indent: 0em;"><P><B>June 1st,2016-July 1st,2016</B></P></DIV><P>Finishing Circuit 5 circuit’s test, and designing the following experiments and materials.</P><DIV style="text-indent: 0em;"><P><B>July 1st,2016-August 1st,2016</B></P></DIV><P>Finishing the following circuits.</P><P>Circuit 6：</P><P>Circuit 7：</P><DIV style="text-indent: 0em;"><P><B>August 1st,2016-September 1st,2016</B></P></DIV><P>Finishing the following circuits and testifying their effects.</P><P>Circuit 8：</P><P>Circuit 9：</P><DIV style="text-indent: 0em;"><P><B>September 1st,2016-October 16th,2016</B></P></DIV><P>Finishing the following circuits and testifying their effects. </P><P>Circuit 10：</P><P>Circuit 11：</P><DIV class="row text-center"><H2>Reference</H2></DIV><DIV style="text-indent: 0em;"><P>[1] Masayuki Nagahashi, MD, PhD,Junko Tsuchida. High levels of sphingolipids in human breast cancer [J].Elsevier.2016 Jun 16
</P><P>[2] Junko Tsuchida, MD, Masayuki Nagahashi. Breast cancer sphingosine-1-phosphate is associated with phospho-sphingosine kinase 1 and lymphatic metastasis [J].Elsevier.2016 Jun 16
</P><P>[3] Wen Feng , Hong-chang Li，Ke Xu, Ya-feng Chen, Li-yun Pan, Yi Mei, Han Cai, Yi-ming Jiang. SHCBP1 is over-expressed in breast cancer and is important in the proliferation and apoptosis of the human malignant breast cancer cell line [J].Elsevier.2016 Jun 16
</P><P>[4] Xiao-Qing Li, Jun-Tao Lu, Cong-Cong Tan, Qing-Shan Wang, Yu-Mei Feng. RUNX2 promotes breast cancer bone metastasis by increasing integrin α5-mediated colonization [J]. Elsevier.2016 Jun 7
</P><P>[5] Y. Okumura, R. Nishimura, K. Nakatsukasa. Change in estrogen receptor, HER2, and Ki-67 status between primary breast cancer and ipsilateral breast cancer tumor recurrence [J]. 2015 Feb 7
</P><P>[6] Wei-hao Lina, Jie Lia, Bing Zhangb, Liang-shuai Liub, Ying Zouc, Jin-fu Tand, He-ping Li. Change in estrogen receptor, HER2, and Ki-67 status between primary breast cancer and ipsilateral breast cancer tumor recurrence [J]. Elsevier.2016 Jun 15
</P><P>[7] Hiba N. Mohammed, Mark R. Pickard, Mirna Mourtada-Maarabouni. The protein phosphatase 4 - PEA15 axis regulates the survival of breast cancer cells [J]. Elsevier.2016 Jun 15
</P><P>[8] Ali MA, Mondal K, Jiao Y, Oren S, Xu Z, Sharma A, Dong L.Microfluidic Immuno-Biochip for Detection of Breast Cancer Biomarkers Using Hierarchical Composite of Porous Graphene and Titanium Dioxide Nanofibers [J].ACS Appl Mater Interfaces.2016.
</P><P>[9] Rossetti S, Bshara W, Reiners JA, Corlazzoli F, Miller A, Sacchi N.Harnessing 3D models of mammary epithelial morphogenesis: An off the beaten path approach to identify candidate biomarkers of early stage breast cancer [J].Pharmacol Res. 2016 Jul 13. 
</P><P>[10] Zamanian M, Qader Hamadneh LA, Veerakumarasivam A, Abdul Rahman S, Shohaimi S, Rosli R.Calreticulin mediates an invasive breast cancer phenotype through the transcriptional dysregulation of p53 and MAPK pathways [J].Cancer Cell Int. 2016 Jul 13;16:56.
</P><P>[11] Wende Hao, Xuhui Zhang, Bingshui Xiu, Xiqin Yang.Vitronectin: a promising breast cancer serum biomarker for early diagnosis of breast cancer in patients [J]. Tumor Biol. 2015 Dec 29. 
</P><P>[12] Dai X, Xiang L, Li T, Bai Z.27.Cancer Hallmarks, Biomarkers and Breast Cancer Molecular Subtypes. J Cancer. 2016 Jun 23;7(10):1281-94. 
</P><P>[13] Ozguven S, Inanir S, Turoglu HT, Erdil TY, Ugurlu MU, Gulluoglu B.29.2-(fluorine-18)-fluoro-2-deoxy-D-glucose positron emission tomography/computed tomography after breast conserving surgery: Correlation with molecular markers of breast cancer. Indian J Nucl Med. 2016 Jul-Sep;31(3):166-71.
</P><P>[14] Mahmoud Aghaei, Fatemeh Karami-Tehrani,Siamak Salami, and Morteza Atri. Diagnostic Value of Adenosine Deaminase Activity in Benign and Malignant Breast Tumors. Elsevier. 2009 Jul;14-18
</P><P>[15] LiYang， XingeCui，NingningZhang， MinLi.Comprehensive lipid profiling of plasma in patients with benign breast tumor and breast cancer reveals novel biomarkers. Anal Bioanal Chem. 2014 Oct
</P><P>[16] Parsa Y, Mirmalek SA, Kani FE, Aidun A, Salimi-Tabatabaee SA, Yadollah-Damavandi S, Jangholi E, Parsa T, Shahverdi E. A Review of the Clinical Implications of Breast Cancer Biology. Electron Physician. 2016 May 25;8(5):2416-24.
</P><P>[17] Ambros, V (Sep 16, 2004). The functions of animal microRNAs. Nature 431 (7006): 350–5.
</P><P>[18] Bartel, DP (Jan 23, 2004). MicroRNAs: genomics, biogenesis, mechanism, and function. Cell 116 (2): 281–97.
</P><P>[19] Zhongxing Liang,Xuehai Bian&amp;Hyunsuk Shim.Downregulation of microRNA-206 promotes invasion and angiogenesis of triple negtive breast cancer[J].Biochemical and Biophysical Research Communications,2016, 461-466.
</P><P>[20] Lin Huang,Ting Dai.MicroRNA-224 targets RKIP to control cell invasion and expression of metastasis genes in human breast cancer[J].Biochemical and Biophysical Research Communications.2012,127–133.
</P><P>[21] Wei Hua,Ke-Di Sa.MicroRNA-139 suppresses proliferation in luminal type breast cancer cells by targeting Topoisomeras alpha[J].Biochemical and Biophysical Research Communications.2015, 1077-1083.
</P><P>[22] Zhongxing Liang,Xuehai Bian&amp;Hyunsuk Shim. Downregulation of microRNA-206 promotes invasion and angiogenesis of triple negtive breast cancer[J].Biochemical and Biophysical Research Communications,2016, 461-466.
</P><P>[23] Lin Huang,Ting Dai.MicroRNA-224 targets RKIP to control cell invasion and expression of metastasis genes in human breast cancer[J].Biochemical and Biophysical Research Communications.2012,127–133.
</P><P>[24] Minako Sakurai.Lin28:A regulator of tumor-suppressing activity of let-7miRNA in human breast cancer [J].Journal of Steroid Biochemistry &amp; Molecular Biology 2012 101– 106
</P><P>[25] Tarek Mohamed.Sdudy of microRNAs-21/221 as potential breast cancer biomarker in Egyptian woman[J]. Gene.2016. 
</P><P>[26] Olfat Shaker. Role of microRNAs-29b-2,-155,-197,-205 as diagnostic biomarkers in serum of breast cancer females[J].Gene.2015, 77-82
</P><P>[27] Bailong Li,Ying Lu.miR-221/222 enhance the tumorigeneticity of human breast cancer stem cells via modulation of PTEN/AKT pathways[J]. Biomedicine &amp; Pharmacotherapy.2016,93-101
</P><P>[28] Jerome Bonnet, Peter Yin, Monica E. Ortiz, Pakpoom Subsoontorn, Drew Endy. Amplifying Genetic Logic Gates. Science, 3 MAY 2013 VOL 340.
</P><P>[29] Piro Siuti, John Yazbek, Timothy K Lu. Synthetic circuits integrating logic and memory in living cells. nature biotechnology, 10 February 2013.
</P><P>[30] Zhen Xie, Liliana Wroblewska, Laura Prochazka, Ron Weiss,Yaakov Benenson1. Multi-Input RNAi-Based Logic Circuit for Identification of Specific Cancer Cells. Science ,VOL 333 2 SEPTEMBER 2011.
</P><P>[31] Mandal M,breaker R R.Gene regulation by riboswitch.Nat.Rev.Mol.Cell boil.,2004,5;451-463.
</P><P>[32] Isaacs F J,Dwyer D J,et al. Engineered riboregulators enable post-transcriptional control of gene expression. Nat.Biotechnol, 2004, 22:841-847.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="contact"><DIV class="container"><DIV class="section-title text-center"><H2>Contact Us</H2></DIV><DIV class="row"><DIV class="col-md-3"><ADDRESS><STRONG>Address</STRONG>
					 Beijing Institute of Technology,
					No. 5 South Zhong Guan Cun Street,
					Haidian Beijing 100081, P. R. China
					Twitter : @igem_BIT

Sina Weibo : @igem_BIT

Website : http://www.bit.edu.cn
					</ADDRESS></DIV><DIV class="col-md-9"><FORM autocomplete="off"><DIV class="text-right"><A href="#" class="btn send-btn">Send</A></DIV></FORM></DIV></DIV></DIV></DIV><NAV id="footer"><DIV class="container"><DIV class="pull-right"><A href="#home" class="page-scroll">Back to Top </A></DIV></DIV></NAV></DIV></DIV></DIV></BODY></HTML>