<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BIT-China_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BIT-China/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="col-sm-12" id="content-wrapper" style="padding: 0;margin: 0;"><DIV class="header clearfix" id="header" style="width: 100%;position: relative;padding-bottom: 20px;background-size:cover;
                                background-image: url(https://static.igem.org/mediawiki/2016/9/9a/T--BIT-China--img--top_bg.png);"><DIV class="col-sm-12 clearfix " style="padding: 0;height: 106px;"><DIV class="header-container clearfix home-img" id="header-container" style="margin-top: 20px;z-index: 10"><DIV class="header-navigation col-sm-11"><UL class="pull-right"><LI><A href="https://2016.igem.org/Team:BIT-China/Medals">Medal fullfillments</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Parts">Parts Submission</A></LI></UL><LI class="tabdown-submenu"><A href="https://2016.igem.org/Team:BIT-China/Description">Description</A></LI><LI class="tabdown-submenu"><A href="https://2016.igem.org/Team:BIT-China/Design">Design</A></LI><LI class="tabdown-submenu"><A href="https://2016.igem.org/Team:BIT-China/Results">Results</A></LI><LI class="tabdown-submenu"><A href="https://2016.igem.org/Team:BIT-China/Proof">Proof</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Notebook">Lab Book</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Protocol">Protocol</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/E_M">Equipments &amp; Materials</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Team">Members</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Team/Gallery">Gallery</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Team/Gaint_Jamboree">Gaint Jamboree</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/HP/Gold">HP_Gold</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/HP/Silver">HP_Silver</A></LI></DIV></DIV></DIV></DIV></DIV><DIV class="col-sm-12 clearfix" style="padding: 0;padding-top:30px;
background: url(https://static.igem.org/mediawiki/2016/4/45/T--BIT-China--content_bg.jpg)"><DIV class="content-right" style="float: left;width:100%;padding: 10px;"><DIV class="block-border-outer" id="content-right-border"><DIV class="overview-content clearfix"><DIV class="problem-txt block-content col-sm-12"><DIV class="problem-title">What is the problem?</DIV><DIV><SPAN>Plasmid segragational instability has been the limit step for large-scale protein production and bioremediation. Antibiotics for plasmid retention is not practical in this situation.</SPAN></DIV><DIV><SPAN>High copy number (HCN) plasmids are lost from population at a high rate due to the large metabolic burden and lack of nature factors for plasmid maintenance.</SPAN></DIV></DIV><DIV id="our_p-slackiller" class="block-title">Our P-SLACKiller</DIV><DIV class="block-content"><DIV class="block-content-item"><DIV class="block-content-item-block"><DIV><SPAN class="block-content-header">IDEA</SPAN></DIV><DIV>
                                Equip the bacteria with a plasmid-sensing logically adjustable cell killer (P-SLACKiller), and we need to select a signal which can indicate the intracellular plasmid numbers. Here we give a definition, when the bacteria containing few plasmids consume lots of nutrition but don’t work efficiently, we called them slackers. There is a basic rule: when the plasmid numbers are above a threshold, we regard the bacterium as normally-working and the P-SLACKiller won't start; however, when the plasmid numbers are below the threshold, we judge it as a slacker and the P-SLACKiller will kill these slackers, so that we can achieve the goal of increasing the plasmid maintenance. In the end, in order to kill the slackers and thus control the plasmid numbers, we selected the inhibitor protein as the signal molecular and the in-promoter repressed by inhibitor as a switch to start the killer gene.
                            </DIV></DIV><DIV class="block-content-item-block"><DIV><SPAN class="block-content-header">PART DESIGN</SPAN></DIV><DIV>
                                The basic circuit is shown in Fig.1.
                            </DIV><DIV><DIV class="center-block" style="font-size:0.9em;text-align:center"><B>Fig.1</B> The basic circuit of P-SLACKiller.</DIV></DIV><DIV class="block-paragraph">We use the constitutive promoter to express the inhibitor protein, thus the intracellular inhibitor concentration is positively correlated with the plasmid numbers. So the inhibitor can be used as a signal indicating the plasmid numbers. The inhibitor can repress the in-promoter and control the expression of downstream killer gene. When the inhibitor concentration reduces to a threshold, the downstream killer gene will express. To avoid the unexpected leak out of killer gene caused by the constitutive promoter, we designed the gene circuit with the constitutive promoter and the in-promoter transcribing along different directions like Fig.1.
                            </DIV><DIV class="block-paragraph">
                                The genetic locus of functional gene can be substituted on the basis of production demand in the future.
                            </DIV></DIV><DIV class="block-content-item-block"><DIV><SPAN class="block-content-header">INSIDE MECHANISM</SPAN></DIV><DIV>
                                Under normal circumstance, the plasmid numbers are above the threshold, the inhibitor concentration is high enough to completely repress the activity of in-promoter. Considering the effect of plasmids segregational instability, the concentration of plasmids in some cells will decline with time going on. With no intervention, these cells will gradually become slackers as well as the predominant strain among the population. In our project, when the plasmid numbers are below the threshold, the decrease of inhibitor proteins will lead to the increase of in-promoter activity. At this time, the killer gene will express and kill the cell. We define that, when the killer expressed is just able to kill the cell, the plasmid numbers is the threshold we've talked about. Based on this, we concluded that when the plasmid numbers are above the threshold, the bacteria can survive and have high working efficiency. This range is called survival range. To prevent the slackers consume a lot of nutrition, we kill it when the range removes to the dead one.

                            </DIV><DIV class="block-paragraph">
                                In this way, we can optimize the general population structure through plasmid-numbers-maintenance.
                            </DIV><DIV><DIV class="center-block" style="font-size:0.9em;text-align:center"><B>Fig.2</B> With the decrease of plasmid numbers, the concentration of repressor will reduce to a threshold. According, in-promoter activity will rise up and star the transcription of killer gene. </DIV></DIV></DIV></DIV></DIV><DIV id="threshold_device" class="block-title">Threshold device</DIV><DIV class="block-content"><DIV class="block-content-brief"><DIV>
                            The most critical factor in our system is the inhibitor concentration since it controls the switch of our killer system. Thus we need to prove that the in-promoter will have different responses corresponding to different concentrations of inhibitor.
                        </DIV><DIV class="block-paragraph">
                            We combined the wet experiment results with the mathematical model to measure the final result.
                            <A href="https://2016.igem.org/Team:BIT-China/Model" style="color: blue;"> [know more about our model]</A></DIV><DIV class="step-style block-header-margin1">
                            First Step
                        </DIV><DIV>
                            Since it's hard to control the plasmid numbers, we employed the arabinose induced P<SUB>BAD</SUB> promoter. The plasmid numbers is stable in short time. By adding different concentration of arabinose, we can simulate the production of inhibitors with different concentration. In this way, we can see the downstream in-promoter stay in the corresponding statues. <SPAN class="italic">RFP</SPAN> is used to replace the killer gene for simplifying our wet experiment. To expand the application of our system, we choose two combinations of inhibitor and in-promoter.
                        </DIV></DIV><DIV><SPAN class="block-content-header">Goal</SPAN></DIV><DIV class="content-block-item" style="margin: 0px auto;padding: 0 1em;"><DIV>
                            (1)	Build two parallel circuits employing different inhibitor proteins
                        </DIV><DIV>
                            (2)	Find the relationship between the inhibitor and in-promoter
                        </DIV><DIV>
                            (3)	Find out the initial thresholds by measuring RFP intensity
                        </DIV><DIV>
                            (4)	Provide experimental data for modeling part
                        </DIV></DIV><DIV><SPAN class="block-content-header">Part Design</SPAN></DIV><DIV class="content-block-item" style="margin: 0px auto;padding: 0 1em;"><DIV>
                            The gene circuits are shown in Fig.3.
                        </DIV><DIV><DIV class="center-block" style="font-size:0.9em;text-align:center"><B>Fig.3</B> The gene circuits of threshold device. </DIV></DIV><DIV>(1)	<B>Part:BBa_K808000 <SPAN class="italic">AraC</SPAN>-P<SUB>BAD</SUB></B> is used to control the expression of inhibitor proteins, The concentration of inducer arabinose is regarded as an input signal, and the red fluorescence intensity controlled by in-promoter is the output.
                        </DIV><DIV>
                            (2)	Two combinations of inhibitor and in-promoter are selected to prove the concept of plasmid quorum sensing. Due to the limit of time, we measured 1-2 circuits.
                            <A href="https://2016.igem.org/Team:BIT-China/Inhibitor" target="_blank" style="color: blue;">[see inhibitor mechanisms]</A></DIV><DIV>
                            (3)	<SPAN class="italic">RFP</SPAN> is used as an indicator since it's easier to quantify compared with the killer gene.
                        </DIV><DIV>
                            (4)	The inducible promoter P<SUB>BAD</SUB> and the in-promoter express towards different directions, a terminator B0015 is added between the two promoters.
                        </DIV></DIV><DIV class="block-content-item"><DIV><SPAN class="block-content-header">We assume that:</SPAN></DIV><DIV>
                            Different concentrations of arabinose will induce inhibitor with different concentration of inhibitors.
                        </DIV><DIV>
                            The high concentration of inducers can simulate high concentration of plasmids due to the same effect of producing more inhibitor proteins.
                        </DIV><DIV>
                            By adding different concentration of arabinose, we can measure the fluorescence intensity of RFP and find out the relationship between the arabinose and RFP intensity and the initial thresholds under the control of certain inhibitor proteins.
                        </DIV></DIV><DIV class="block-content-brief"><DIV class="step-style block-header-margin1">
                            Second step
                        </DIV><DIV>
                            In order to find out the concentrations of inhibitor under different concentrations of arabinose and use the results to derive the threshold of plasmids number in the constitutive circuit. Inhibitor is replaced by RFP and induced based on the same condition. The concentration of inhibitor can be represented by measuring the fluorescence intensity of RFP.
                        </DIV></DIV><DIV><SPAN class="block-content-header">Goal</SPAN></DIV><DIV class="content-block-item" style="margin: 0px auto;padding: 0 1em;"><DIV>
                            (1)	Represent the concentration of inhibitor by measuring the fluorescence intensity of RFP.
                        </DIV><DIV>
                            (2) Derive the threshold of plasmids number in the constitutive circuit
                        </DIV></DIV><DIV class="content-block-item"><DIV><SPAN class="block-content-header">Part Design</SPAN></DIV><DIV>
                            The gene circuits are shown in Fig.4.
                        </DIV><DIV><DIV class="center-block" style="font-size:0.9em;text-align:center"><B>Fig.4</B> The gene circuit constructed to find out the concentrations of inhibitor induced by different concentrations of arabinose. </DIV></DIV></DIV><DIV class="block-content-item"><DIV><SPAN class="block-content-header">We assume that:</SPAN></DIV><DIV>
                            More arabinose added, more RFP expressed which can stand for the concentration of inhibitor, and vice versa.
                        </DIV><DIV><DIV class="center-block" style="font-size:0.9em;text-align:center"><B>Fig.5</B> The relationship between the concentrations of inhibitor and plasmid number. </DIV></DIV><DIV>Thus we could find out the relationship between the concentration of arabinose and inhibitor.
                        </DIV></DIV><DIV class="block-content-brief"><DIV class="step-style block-header-margin1">
                            Third step
                        </DIV><DIV>
                            We use plasmids with different copy numbers to simulate the concentration of plasmids losing on different levels. Three kinds of plasmids, <SPAN class="italic">pSB1k3</SPAN> (100~200 copies), <SPAN class="italic">pSB3k3</SPAN> (20~30 copies), <SPAN class="italic">pSB4k5</SPAN> (5~10 copies), are chosen and constructed with target fragment. By observing our system's working conditions, we can finally prove its function of controlling plasmids number.
                        </DIV></DIV><DIV><SPAN class="block-content-header">Goal</SPAN></DIV><DIV class="content-block-item" style="margin: 0px auto;padding: 0 1em;"><DIV>
                            (1) Simulate the plasmids losing on different levels
                        </DIV><DIV>
                            (2) Prove that our system is functional
                        </DIV></DIV><DIV class="content-block-item"><DIV><SPAN class="block-content-header">Part Design</SPAN></DIV><DIV>
                            The gene circuits are shown in Fig.6.
                        </DIV><DIV><DIV class="center-block" style="font-size:0.9em;text-align:center"><B>Fig.6</B> The gene circuits of threshold device with constitutive promoter. </DIV></DIV><DIV style="margin: 10px auto;padding: 0 1em;">
                            (1) We constructed the circuit on different plasmids with different copy numbers.
                            (2) The locus of constitutive promoter will be replaced by another constitutive promoter which has the similar strength as P<SUB>BAD</SUB> promoter.
                        </DIV></DIV><DIV class="block-content-item"><DIV><SPAN class="block-content-header">We assume that:</SPAN></DIV><DIV>
                            When the number of plasmids is less than N, inhibitor is not strong enough to inhibit the in-promoter totally, so the killer gene will express triggering the death of bacteria. On the contrary, if the number of plasmids is higher than N, the bacteria won't die.
                        </DIV><DIV>
                            We expect that in the high copy number plasmid circuit, fluorescence gene will express, and in the low copy number plasmid circuit, there will no fluorescence.
                        </DIV></DIV><DIV class="block-content-brief"><DIV class="step-style block-header-margin1">
                            Forth step
                        </DIV><DIV>
                            The threshold of plasmid number is decided by the relationship between inhibitor and in-promoter. To achievement the goal of build a controllable system, we decide to use different RBS (B0034 B0032) and promoters (J23119 J23116 J23109) with different strengths to change their concentrations. Thus we can finally change the threshold of plasmid as we hope.
                        </DIV></DIV><DIV><SPAN class="block-content-header">Goal</SPAN></DIV><DIV class="content-block-item" style="margin: 0px auto;padding: 0 1em;"><DIV>
                            (1) Adjust the threshold of plasmids by substituting different biobricks.
                        </DIV></DIV><DIV class="content-block-item"><DIV><SPAN class="block-content-header">Part Design</SPAN></DIV><DIV>
                            The gene circuits are shown in Fig.7.
                        </DIV><DIV class="block-paragraph"><DIV class="center-block" style="font-size:0.9em;text-align:center"><B>Fig.7</B> The gene circuits of threshold device constructed on a different plasmids with different copy numbers. </DIV></DIV><DIV style="margin: 10px auto;padding: 0 1em;">
                            (1) Simulate the plasmids losing on different levels.
                            (2) The locus of B0034 and J23119 will be replaced by B0032 and J23109 J23116.
                        </DIV></DIV></DIV><DIV id="killer_device" class="block-title">Killer device</DIV><DIV class="block-content"><DIV class="block-content-brief"><DIV>
                            As an executor for killing the cell labeled as slackers, killer device should include the detecting part like in-promoter interacting with inhibitor protein as well as the killing part like toxin gene <SPAN class="italic">mazF</SPAN> and <SPAN class="italic">hokD</SPAN>.
                        </DIV></DIV><DIV class="block-content-item"><DIV><SPAN class="block-content-header">Ways of self-killing:</SPAN></DIV><DIV class="block-content-item-block" style="margin: 0px auto;padding: 0 1em;"><DIV>
                                1) toxic protein 
                            </DIV><DIV>
                                2) sgRNA targeting genome of strains with no NHEJ repair system
                            </DIV><DIV>
                                3) sgRNA targeting essential genes
                            </DIV></DIV></DIV><DIV class="block-content-item"><DIV class="block-content-item-block"><DIV><SPAN class="block-content-header">PART DESIGN</SPAN></DIV><DIV>
                                At the first stage, we need to verify the function of two toxin proteins and make sure the bacteria can be killed after the expression of toxin proteins. To control the expression of killer gene, instead of J23119, arabinose inducible regulatory promoter/repressor unit is employed in our circuit.
                            </DIV><DIV><DIV class="center-block" style="font-size:0.9em;text-align:center"><B>Fig.8</B> Circuits for toxin gene function testing through induction by arabinose. </DIV></DIV><DIV class="block-paragraph">After measuring the growth curve, we observed the obvious difference between the testing group and control group.
                                <A href="https://2016.igem.org/Team:BIT-China/Results" target="_blank" style="color: blue;">[see results page for killer]</A></DIV><DIV class="block-paragraph">
                                To facilitate the measurement of plasmid threshold, we combined the inhibitor circuit with the killer circuit and replace the <SPAN class="italic">RFP</SPAN> for killer gene.
                                <A href="#threshold_device" style="color: blue;">[see design for threshold]</A></DIV><DIV><DIV class="pic_info"><B>Fig.9</B> Circuits for testing for killing capacity under different concentration of arabinose, which can be regard as under different plasmid concentrations. </DIV></DIV><DIV class="block-paragraph">For the toxic protein, MazF and HokD were chosen as candidates.
                                <A href="https://2016.igem.org/Team:BIT-China/Killing_Mechanisms" target="_blank" style="color: blue;">[see more about the killing mechanisms]</A></DIV></DIV></DIV></DIV><DIV id="upgraded_p-slackiller" class=" block-title">Upgraded P-SLACKiller</DIV><DIV class="problem-txt block-content"><DIV class="problem-title" style="text-align:left">Why do we need to optimize our project?</DIV><DIV><SPAN>
                            It's possible to produce generation of plasmid-free cells, even if all plasmids were losen. Thus, we decided to integrate the killer device into genome to solve this problem.
                            <A href="https://2016.igem.org/Team:BIT-China/Recombination" target="_blank" style="color: blue;">[see reombination]</A></SPAN></DIV><DIV><SPAN>
                            To control the plasmid number above different thresholds, we need to adjust the initial threshold through replacement of RBS as well as in-promoter mutation.
<A href="https://2016.igem.org/Team:BIT-China/Promoter" target="_blank" style="color: blue;">[see mutation]</A></SPAN></DIV></DIV><DIV id="recombination" class="block-title">Recombination</DIV><DIV class="block-content"><DIV class="block-content-brief"><DIV style="text-align:left">
                            Recombineering (
                            <B>recombi</B>nation-mediated genetic engi<B>neering</B>) is an efficient molecular
                            engineering technique used for gene replacement, deletion and insertion.
                        </DIV></DIV><DIV class="content-block-item"><DIV><SPAN class="block-content-header">Ways: a linear plus circular reaction</SPAN></DIV><DIV style="margin: 0px auto;padding: 0 1em;">
                            A：Employment of traditional λ-Red recombination 
                        </DIV></DIV><DIV class="content-block-item"><DIV><SPAN class="block-content-header">PART DESIGN</SPAN></DIV><DIV>              The basic circuit of upgraded P-SLACKiller is shown in Fig.10.
                        </DIV><DIV><DIV class="pic_info"><B>Fig.10</B> The basic circuit of upgraded P-SLACKiller. </DIV></DIV><DIV class="block-paragraph">Compared with the original one, two homologous arms (50bp) are added. With the help of lambda Red recombination system, the killer part can be integrated into genome, preventing the whole system from being invalid. The final gene circuit is shown in Fig.11.
                        </DIV><DIV><DIV class="pic_info"><B>Fig.11</B> The basic circuit of upgraded P-SLACKiller. </DIV></DIV></DIV><DIV class="content-block-item"><DIV>B: CRISPR-Cas9 coupling λ-Red recombineering
                        </DIV></DIV><DIV class="content-block-item"><DIV><SPAN class="block-content-header">PART DESIGN</SPAN></DIV><DIV><DIV class="pic_info"><B>Fig.12</B> Double plasmid to meet different requirements for each element. Equip the high copy number plasmid PUC19 with sgRNA targeting site on genome and donor fragment with homologous arms. The low copy number plasmid are equipped with expression cassette of cas9 protein and λ-Red recombinase. </DIV></DIV><DIV class="block-paragraph">Difference between the two methods above: Traditional way use the resistance gene as a selection marker, while the CRISPR coupled way introduce a double-strand break (DSB) into the strain with no recombination occurred.
                        </DIV></DIV><DIV class="content-block-item"><DIV><SPAN class="block-content-header">Application:</SPAN></DIV><DIV>
                            In our project, we tried both the two ways to test the integration efficiency. However, the CRISPR coupled way are deleted from our plan list due to the failure of cas9 and λ-Red plasmid construction.
                        </DIV><DIV class="block-paragraph">
                            We tried the traditional way with the <SPAN class="italic">pKD46</SPAN> plasmid provided by Chun Li's lab and got the positive results.
                            <A href="https://2016.igem.org/Team:BIT-China/Results" style="color: blue;">[results for recombination]</A></DIV><DIV class="block-paragraph">
                            Two main factors will influence the recombination efficiency: the target site on genome and the length of homologous arms between donor fragment.
                            <A href="https://2016.igem.org/Team:BIT-China/Protocol" style="color: blue;">[recombination protocol]</A></DIV></DIV></DIV><DIV id="promoter_mutation" class="block-title">Promoter mutation</DIV><DIV class="block-content"><DIV class="block-content-brief">
                        For inductive promoters, two kinds of region are significant to its transcription strength. One is common -10 and -35 region which can firmly bind to RNA polymerase, the other is operon region binding with regulatory factors like inhibitor proteins.
                        <A href="https://2016.igem.org/Team:BIT-China/Promoter" target="_blank" style="color: blue;">[see mutation mechanisms]</A></DIV><DIV><SPAN class="block-content-header"> Aim:</SPAN></DIV><DIV style="margin: 0px auto;padding: 0 1em;">
                        (1)	To adjust threshold through changing in-promoter's strength
                        (2)	Tied to specific promoter like P<SUB>Tet</SUB>, build a mutants library and improve the existing part by measuring the strength through RFP
                    </DIV><DIV><SPAN class="block-content-header">Part Design</SPAN></DIV><DIV><DIV class="pic_info"><B>Fig.13</B> The basic circuit of promoter mutation. </DIV></DIV><DIV class="block-paragraph">After analyzing the promoter region of P<SUB>Tet</SUB>, we designed random primers to mutate its -35 region. RFP intensity is measured to indicate the strength of mutated promoters.
                        <A href="" style="color: blue;">[see mutation results]</A></DIV></DIV></DIV></DIV></DIV><DIV class="slow-transition nav-left" id="nav_left_wrapper"><DIV class="nav-left-title" style="position:relative;width: 200px;padding: 0;"><SPAN style="color: #FFFFFF;font-size: 20px;font-weight: bold;position: absolute;
        right: 40px;bottom: 20px;">Design</SPAN></DIV><UL class="nav_left_ul" style="padding-left: 50px;"><LI><A href="#our_p-slackiller" class="nav_point"><SPAN class="nav_left_txt">Our P-SLACKiller</SPAN></A></LI><LI><A href="#threshold_device" class="nav_point"><SPAN class="nav_left_txt">Threshold device</SPAN></A></LI><LI><A href="#killer_device" class="nav_point"><SPAN class="nav_left_txt">Killer device</SPAN></A></LI><LI><A href="#upgraded_p-slackiller" class="nav_point"><SPAN class="nav_left_txt" style="font-size: 12px">Upgraded P-SLACKiller</SPAN></A></LI><LI><A href="#recombination" class="nav_point"><SPAN class="nav_left_txt_plus">Recombination</SPAN></A></LI><LI><A href="#promoter_mutation" class="nav_point"><SPAN class="nav_left_txt_plus">Promoter mutation</SPAN></A></LI></UL></DIV><DIV id="site-change-btn" style="position: absolute;left: 20px;bottom: 20px;z-index:9;display: none;"><A href="https://2016.igem.org/Team:BIT-China/Background"><SPAN style="color: #654D6F;font-weight: bold;font-size: 12px">Background</SPAN></A><A href="https://2016.igem.org/Team:BIT-China/Results"><SPAN style="color: #654D6F;font-weight: bold;font-size: 12px">
            Results</SPAN></A></DIV></DIV><DIV class="col-sm-12" style="padding: 0;"><FOOTER class="footer dark"><DIV class="footer-widgets md-padding-top"><DIV class="container"><DIV class="row"><DIV class="col-sm-3"><DIV class="widget widget-about"><H4 class="widget-title" style="margin-bottom: 43px">P-SLACKiller</H4><P><B>· </B>Work or Die<B>· </B>Monitor Plasmid Numbers<B>· </B>Kill the Slacker<B>· </B>Maintain the Better Biosystem</P></DIV></DIV><DIV class="col-sm-3"><DIV class="widget widget-categories"><H4 class="widget-title" style="margin-bottom: 43px">Useful Links</H4><UL><LI><A href="https://2016.igem.org/Team:BIT-China/Medals" target="_blank">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Results" target="_blank">Results</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Design" target="_blank">Design</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Proof" target="_blank">Proof</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/HP/Gold" target="_blank">HP</A></LI></UL></DIV></DIV><DIV class="col-sm-3"><DIV class="widget"><H4 class="widget-title" style="margin-bottom: 43px">Sponsors</H4></DIV></DIV><DIV class="col-sm-3"><DIV class="widget"><H4 class="widget-title" style="margin-bottom: 43px">Get in touch</H4><P><STRONG>Email:</STRONG> iGEM_BIT_China@163.com <STRONG>Location:</STRONG> No.5 Zhongguancun South Street </P></DIV></DIV></DIV></DIV></DIV><DIV class="footer-meta"><DIV class="container"><DIV class="row"><DIV class="col-sm-12"><DIV class="copyright">BIT-China © Copyright 2016  Powered by Bootstrap.</DIV></DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>