<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BIT-China_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BIT-China/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="col-sm-12" id="content-wrapper" style="padding: 0;margin: 0;"><DIV class="header clearfix" id="header" style="width: 100%;position: relative;padding-bottom: 20px;background-size:cover;
                                background-image: url(https://static.igem.org/mediawiki/2016/9/9a/T--BIT-China--img--top_bg.png);"><DIV class="col-sm-12 clearfix " style="padding: 0;height: 106px;"><DIV class="header-container clearfix home-img" id="header-container" style="margin-top: 20px;z-index: 10"><DIV class="header-navigation col-sm-11"><UL class="pull-right"><LI><A href="https://2016.igem.org/Team:BIT-China/Medals">Medal fullfillments</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Parts">Parts Submission</A></LI></UL><LI class="tabdown-submenu"><A href="https://2016.igem.org/Team:BIT-China/Description">Description</A></LI><LI class="tabdown-submenu"><A href="https://2016.igem.org/Team:BIT-China/Design">Design</A></LI><LI class="tabdown-submenu"><A href="https://2016.igem.org/Team:BIT-China/Results">Results</A></LI><LI class="tabdown-submenu"><A href="https://2016.igem.org/Team:BIT-China/Proof">Proof</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Notebook">Lab Book</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Protocol">Protocol</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/E_M">Equipments &amp; Materials</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Team">Members</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Team/Gallery">Gallery</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Team/Gaint_Jamboree">Gaint Jamboree</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/HP/Gold">HP_Gold</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/HP/Silver">HP_Silver</A></LI></DIV></DIV></DIV></DIV></DIV><DIV class="col-sm-12 clearfix" style="padding: 0;padding-top:30px;
background: url(https://static.igem.org/mediawiki/2016/4/45/T--BIT-China--content_bg.jpg)"><DIV class="content-right" style="float: left;width:100%;padding: 10px;"><DIV class="block-border-outer" id="content-right-border"><DIV class="overview-content clearfix"><DIV id="threshold" class="block-title col-sm-12">Threshold device</DIV><DIV class="block-content"><DIV class="block-content-item"><DIV class="block-content-item-block"><DIV>
                                In the threshold device, we need to prove that the in-promoter will have different responses corresponding to different plasmid numbers. In our project, we use the inhibitor protein as the signal. We combined the wet experiment results and the mathematical model to prove our system can work in order. We divided it into two main goals:
                    </DIV><DIV class="block-paragraph-padding">
                                (1)	To prove that inhibitor’s concentration can regulate the expression of the killer gene by affecting its in-promoter.
                                (2)	To prove that plasmid numbers will influence the inhibitor concentration
                            </DIV><DIV class="block-content-header">
                                Results:
                            </DIV><DIV class="block-paragraph-padding">
                                1.	Simulated the production of inhibitors with different concentrations by adding different concentrations of arabinose
                                2.	Found out the concentrations of inhibitor under different concentrations of arabinose and used the results to derive the threshold of plasmids number in the constitutive circuit.
                                3.	Employed plasmids with different copy numbers and simulated the concentrations of plasmids losing on different levels
                                4.	Employed different RBS and promoters with different strengths to change the inhibitor concentration
                            </DIV><DIV class="block-paragraph" style="margin-top: 30px;margin-bottom: 10px"><A data-toggle="collapse" href="#collapse0" aria-expanded="true" aria-controls="collapse0" style="color: blue;text-decoration: underline">
                                    Experimental details <SPAN>[Expand]</SPAN></A></DIV><DIV id="collapse0" class="panel-collapse collapse " role="tabpanel" aria-labelledby="heading0"><DIV class="block-title" style="font-size: 1.6em">
                                    Build result
                                </DIV><DIV class="build_result"><DIV class="block-content-header">
                                        1. Constructing the circuits.
                                    </DIV><DIV>
                                        We used P<SUB>BAD</SUB> promoters in all three circuits. All the circuits contain a B0034 and a B0015. <A href="http://parts.igem.org/Part:BBa_K2120303" style="color:blue;">[BBa_K2120303]</A><A href="http://parts.igem.org/Part:BBa_K2120304" style="color:blue;">[BBa_K2120304]</A><A href="http://parts.igem.org/Part:BBa_K2120304" style="color:blue;">[BBa_K2120304]</A></DIV><DIV class="block-paragraph"><DIV class="pic_info"><B>Fig. 1</B> Arabinose induced expression cassette of three kinds of inhibitors
                                        </DIV></DIV><DIV class="block-paragraph">
                                        We used the red fluorescent protein to replace the killer gene and constructed three circuits <A href="http://parts.igem.org/Part:BBa_K2120301" style="color:blue;">[BBa_K2120301]</A> with different in-promoters (controlled by inhibitors). Two of them <A href="http://parts.igem.org/Part:BBa_I13521" style="color:blue;">[BBa_I13521]</A><A href="http://parts.igem.org/Part:BBa_I6031" style="color:blue;">[BBa_I6031]</A> comes from DNA Distribution kit.
                                    </DIV><DIV class="block-paragraph"><DIV class="pic_info"><B>Fig. 2</B> In-promoter controlled expression cassette of RFP. RFP is used to replace killer gene.
                                        </DIV></DIV><DIV class="block-paragraph">
                                        And we separately assembled the two devices.
                                    </DIV><DIV class="block-paragraph"><DIV class="pic_info"><B>Fig. 3</B> Assembly of inhibitor and in-promoter. They can be used to test the minimum arabinose concentration which can totally repress the expression of RFP.
                                        </DIV></DIV><DIV class="block-paragraph">
                                        But when we added the arabinose, the RFP intensity increased, and it contradicted with expected results. We thought the terminator cannot completely isolate the two devices.
                                        <B>So we change the promoter direction and add another B0015 to optimize the circuits.</B><A href="http://parts.igem.org/Part:BBa_K2120310" style="color:blue;">[BBa_K2120310]</A><A href="http://parts.igem.org/Part:BBa_K2120311" style="color:blue;">[BBa_K2120311]</A></DIV><DIV class="block-paragraph"><DIV class="pic_info"><B>Fig. 4</B> Optimized circuits after changing the direction of promoter pBAD and adding another terminator B0015.
                                        </DIV></DIV><DIV class="block-paragraph">
                                        Also in order to reflect the inhibitor concentration, we constructed the following circuit. <A href="http://parts.igem.org/Part:BBa_K2120302" style="color:blue;">[BBa_K2120302]</A></DIV><DIV class="block-paragraph"><DIV class="pic_info"><B>Fig. 5</B> RFP is used to replace the inhibitor gene.
      </DIV></DIV><DIV class="block-paragraph">
                                        We constructed 16 (4*2*2) devices <A href="http://parts.igem.org/Part:BBa_K2120312" style="color:blue;">[BBa_K2120312]</A><A href="http://parts.igem.org/Part:BBa_K2120313" style="color:blue;">[BBa_K2120313]</A><A href="http://parts.igem.org/Part:BBa_K2120314" style="color:blue;">[BBa_K2120314]</A><A href="http://parts.igem.org/Part:BBa_K2120315" style="color:blue;">[BBa_K2120315]</A><A href="http://parts.igem.org/Part:BBa_K2120317" style="color:blue;">[BBa_K2120317]</A><A href="http://parts.igem.org/Part:BBa_K2120318" style="color:blue;">[BBa_K2120318]</A><A href="http://parts.igem.org/Part:BBa_K2120319" style="color:blue;">[BBa_K2120319]</A><A href="http://parts.igem.org/Part:BBa_K2120321" style="color:blue;">[BBa_K2120321]</A><A href="http://parts.igem.org/Part:BBa_K2120323" style="color:blue;">[BBa_K2120323]</A><A href="http://parts.igem.org/Part:BBa_K2120325" style="color:blue;">[BBa_K2120325]</A><A href="http://parts.igem.org/Part:BBa_K2120326" style="color:blue;">[BBa_K2120326]</A><A href="http://parts.igem.org/Part:BBa_K2120327" style="color:blue;">[BBa_K2120327]</A><A href="http://parts.igem.org/Part:BBa_K1480003" style="color:blue;">[BBa_K1480003]</A><A href="http://parts.igem.org/Part:BBa_K145201" style="color:blue;">[BBa_K145201]</A><A href="http://parts.igem.org/Part:BBa_K145113" style="color:blue;">[BBa_K145113]</A><A href="http://parts.igem.org/Part:BBa_K1480004" style="color:blue;">[BBa_K1480004]</A> with the alteration of constitutive promoters (4 kinds), RBS (2 kinds) and inhibitors (2 kinds).
                                    </DIV><DIV class="block-paragraph"><DIV class="pic_info"><B>Table 1.</B> The strengths and efficiencies of different promoters and RBS
                            </DIV><DIV class="pic_info"><B>Fig. 6</B> Constitutive circuits to reappear the threshold concentration of inhibitors. Only few will be selected and used to prove the concept of plasmid sensing.
                                        </DIV></DIV><DIV class="block-content-header">
                                        2. We construct the constitutive promoter with the corresponding in-promoter in the one plasmids.
                                    </DIV><DIV>
                                        After constructing the circuits with different constitutive promoters to express the inhibitor, we change the constitutive promoter direction and add corresponding in-promoter. <A href="http://parts.igem.org/Part:BBa_K2120416" style="color:blue;">[BBa_K2120416]</A><A href="http://parts.igem.org/Part:BBa_K2120418" style="color:blue;">[BBa_K2120418]</A><A href="http://parts.igem.org/Part:BBa_K2120419" style="color:blue;">[BBa_K2120419]</A><A href="http://parts.igem.org/Part:BBa_K2120420" style="color:blue;">[BBa_K2120420]</A><A href="http://parts.igem.org/Part:BBa_K2120421" style="color:blue;">[BBa_K2120421]</A><A href="http://parts.igem.org/Part:BBa_K2120423" style="color:blue;">[BBa_K2120423]</A><A href="http://parts.igem.org/Part:BBa_K2120424" style="color:blue;">[BBa_K2120424]</A><A href="http://parts.igem.org/Part:BBa_K2120425" style="color:blue;">[BBa_K2120425]</A></DIV><DIV class="block-paragraph"><DIV class="pic_info"><B>Fig.7</B> The constitutive promoter express inhibitor protein and repress the downstream in-promoter. The constitutive promoter include J23119, J23106, J23116, J23109; inhibitor include CI and tetR and in-promoter include ptet and pR.
                                        </DIV></DIV><DIV class="block-content-header">
                                        3. We choose plasmids with different copy numbers to simulate the situation of different plasmid losing rates.
                                    </DIV><DIV>
                                        Since it’s hard to control the intracellular plasmid numbers, we choose plasmids with different copy numbers to prove our system. Three kinds of plasmids are listed below.
                                    </DIV><DIV class="block-paragraph"><DIV class="pic_info">
                                            Table 2. The copy numbers of different plasmids.
                                        </DIV></DIV><DIV class="block-paragraph">
                                        The fragments containing the constitutive promoter (Fig.8) are linked with and cloned into three types of plasmids individually.
                                    </DIV><DIV class="block-paragraph"><DIV class="pic_info"><B>Fig.8</B> The circuit constructed with constitutive promoter to simulate the situation of different plasmid losing rates.
                                        </DIV></DIV></DIV><DIV class="block-title" style="font-size: 1.6em">
                                    Test result
                                </DIV><DIV class="test_result"><DIV class="block-content-header">
                                        1.	Simulated the production of inhibitors with different concentrations by adding different concentration of arabinose
                                    </DIV><DIV>
                                        In order to construct the environment with different concentrations of inhibitor and find the threshold, we use the PBAD promoter to achieve this. In the wet lab, we chose the CI-PR circuit like the picture (Fig. 1) to do this experiments.
                                    </DIV><DIV class="block-paragraph"><DIV class="pic_info"><B>Fig.1</B> the circuit constructed to measure
                                        </DIV></DIV><DIV class="block-paragraph">
                                        When we added enough arabinose, the CI inhibitor could repress the pR promoter fully, so the RFP can’t be tested. Also when the inhibitor was not enough, the promoter will express RFP which stands the killer gene’s expression..
                                    </DIV><DIV class="block-paragraph">
                                        Firstly, we use the model to simulate the relationship between the arabinose added and RFP intensity. We got the curve below (Fig. 2):
                                    </DIV><DIV class="block-paragraph"><DIV class="pic_info"><B>Fig. 2</B> this curve is about the corresponding RFP intensity under the different concentration of the arabinose.
                                        </DIV></DIV><DIV class="block-paragraph">
                                        This curve describes the in-promoter activity have a relationship with the different concentrations of the arabinose. That means when we add different concentrations of arabinose, the in-promoter’s activity will be different. We regarded the concentration of arabinose 0.003% as a threshold, the corresponding in-promoter is thought just be repressed regarded as the threshold. Thus we can say the in-promoter can have different responses to the inhibitor.
                                    </DIV><DIV class="block-paragraph">
                                        After the simulation, we added different concentrations of arabinose to construct different concentrations of intracellular inhibitor in the wet lab.
                                    </DIV><DIV class="block-paragraph">
                                        After the pre-experiment, the arabinose concentrations we chose are listed in table.1. The negative control is the strain containing the empty pSB1K3 and the positive control is the strain containing the device in Fig.1 on pSB1K3 without adding the arabinose.
                                    </DIV><DIV class="block-paragraph"><DIV class="pic_info">
                                            Table. 1 the arabinose concentration used in pre-experiment
                                        </DIV></DIV><DIV class="block-paragraph">
                                        The strain we used is E.coli Top10.
                                    </DIV><DIV class="block-paragraph">
                                        Firstly, the two strains containing empty pSB1K3 and containing the device in Fig.1 on pSB1K3 separately overnight (12 h) growth in the LB medium. Then the overnight culture was diluted 1:100 into new LB medium while add the certain concentration of the arabinose.
                                    </DIV><DIV class="block-paragraph">
                                        At first, the overnight culture’s OD600 and red fluorescence intensity are measured. And then we measure the OD600 and red fluorescence intensity every 1-2 hours.
                                    </DIV><DIV class="block-paragraph">
                                        And then we draw the diagram about RFP intensity/OD600 and time under different concentrations of arabinose (Fig. 3).
                                    </DIV><DIV class="block-paragraph"><DIV class="pic_info"><B>Fig. 3</B> the diagram of the about RFP intensity/OD600 and time under different concentrations of arabinose
                                        </DIV></DIV><DIV class="block-paragraph">
                                        From this diagram, we can clearly know more arabinose added, less RFP expressed. When the arabinose concentration is 0.003％, the RFP intensity have been very low. So we think when the arabinose concentration is above 0.003％, the in-promoter will be fully repressed that means the killer will not express to kill the bacteria.
                                    </DIV><DIV class="block-paragraph">
                                        Then we collect the data after passing 17.5 hours from Fig. 3 to draw the diagram about the relationship between RFP intensity/OD600 and arabinose concentration (Fig. 4).
                                    </DIV><DIV class="block-paragraph"><DIV class="pic_info"><B>Fig. 4</B> the diagram of the about RFP intensity/OD600 and arabinose concentration
                                        </DIV></DIV><DIV class="block-paragraph">
                                        From this curve, we can directly to see the relationship between RFP intensity/OD600 and arabinose’s concentration: The arabinose’s concentration have a negative correlation with RFP intensity/OD600. We regard that the in-promoter have been repressed fully when the concentration of arabinose between 0.003％ and 0.004％. So we think when the concentration of arabinose is 0.003％, the in-promoter’s activity is very low, and the killer gene wouldn’t kill the bacteria. So we regard that 0.003％ is threshold of arabinose concentration which also is fit in with our modeling part.
                                    </DIV><DIV class="block-paragraph">
                                        After we use the P<SUB>BAD</SUB> promoter to construct the environment with different concentrations of inhibitor, we have successfully proved that the in-promoter will respond to the different concentrations of inhibitor differently and found the threshold of inhibitor’s concentration that the P<SUB>BAD</SUB> promoter expressed under the 0.003％ arabinose concentration.
                                    </DIV><DIV class="block-content-header">
                                        2.	Found out the concentrations of inhibitor under different concentrations of arabinose and used the results to derive the threshold of plasmids number in the constitutive circuit.
                                    </DIV><DIV>
                                        In order to know the exact concentration of inhibitor under the different concentrations of arabinose. In E.coli Top10, we used the RFP to replace the inhibitor gene (Fig. 5), so we can use RFP to reflect the inhibitor when we added the same concentration of arabinose.
                                    </DIV><DIV class="block-paragraph"><DIV class="pic_info"><B>Fig.5</B> The circuit constructed to measure the inhibitor concentration, inhibitor is replaced by RFP.
                                        </DIV></DIV><DIV class="block-paragraph">
                                        At first, we also use the modeling to simulate this circuit get the curve about relationship between the arabinose added and protein intensity. We got the curve below (Fig. 6):
                                    </DIV><DIV class="block-paragraph"><DIV class="pic_info"><B>Fig. 6</B> The concentration of inhibitor expressed by PBAD changes with different concentrations of inducer.
                                        </DIV></DIV><DIV class="block-paragraph">
                                        From this curve, we can clearly see the concentration of arabinose have positive relationship with protein intensity in a proper range. Before we have achieve the threshold of the inhibitor concentration under the 0.003％concentration of arabinose. The point of inducer’s concentration (0.003%) can cast on protein intensity coordinate axis, and the corresponding intensity is about 24.42 units.
                                    </DIV><DIV class="block-paragraph">

                                        In the wet lab, after the pre-experiment, the arabinose concentrations we chose are listed in table.2. The negative control is the strain containing the empty pSB1C3 and the positive control is the strain containing the device in Fig.6 on pSB1C3 without adding the arabinose.
                                    </DIV><DIV class="block-paragraph"><DIV class="pic_info">
                                            Table. 2 the arabinose concentration used in pre-experiment
                                        </DIV></DIV><DIV class="block-paragraph">
                                        Firstly, the two strains containing empty pSB1C3 and the device in Fig.6 separately overnight (12 h) growth in the LB medium. Then the overnight culture was diluted 1:100 into new LB medium while add the certain concentration of the arabinose.
                                    </DIV><DIV class="block-paragraph">
                                        At first, the overnight culture’s OD600 and red fluorescence intensity were measured. And then we measured the OD600 and red fluorescence intensity every 1-2 hours.
                                    </DIV><DIV class="block-paragraph">
                                        We draw the diagram about RFP intensity/OD600 and time under different concentrations of arabinose (Fig. 7).
                                    </DIV><DIV class="block-paragraph"><DIV class="pic_info"><B>Fig. 7</B> The relationship between the fluorescence/OD600 and time
                                        </DIV></DIV><DIV class="block-paragraph">
                                        From this diagram we can see that the concentration of inducer will affect the expression of downstream RFP, more inducer added, more RFP will be expressed.
                                    </DIV><DIV class="block-paragraph">
                                        Then we collect the data after passing 11.3 hours from Fig. 8 to draw the diagram about the relationship between RFP intensity/OD600 and arabinose concentration (Fig. 8).
                                    </DIV><DIV class="block-paragraph"><DIV class="pic_info"><B>Fig. 8</B> The relationship between the fluorescence/OD600 and the concentration of arabinose
                                        </DIV></DIV><DIV class="block-paragraph">
                                        From this diagram, we can get the conclusion that the concentration of arabinose have a positive correlation with RFP intensity/OD600.
                                    </DIV><DIV class="block-content-header">
                                        3. Employed plasmids with different copy numbers and simulated the concentration of plasmids losing on different levels.
                                    </DIV><DIV>
                                        In order to know the concentration of inhibitor changing with number of plasmids, we chose three types of plasmids with different copy numbers to achieve this.
                                    </DIV><DIV class="block-paragraph">
                                        Three kinds of plasmids are listed below.
                                    </DIV><DIV class="block-paragraph"><DIV class="pic_info"><B>Table 3.</B> The copy numbers of different plasmids.
                                        </DIV></DIV><DIV class="block-paragraph">
                                        The fragments below (Fig. 9) containing constitutive promoter to express inhibitor.
                                    </DIV><DIV class="block-paragraph"><DIV class="pic_info"><B>Fig.9</B> The circuit constructed to detect the inhibitor protein and the strengths of different promoters.
                                        </DIV></DIV><DIV class="block-paragraph">
                                        In order to detect the inhibitor protein more easily, we replace the inhibitor with RFP like circuits below (Fig. 10)
                                    </DIV><DIV class="block-paragraph"><DIV class="pic_info"><B>Fig. 10</B> The fragments constructed on different plasmids
                                        </DIV></DIV><DIV class="block-paragraph">
                                        Then we construct two fragments in the Fig. 11 on different plasmids with different copy numbers.
                                    </DIV><DIV class="block-paragraph">
                                        At first, because it’s hard to control the number of plasmids, we use the model to simulate the relationship between the plasmids number and inhibitor protein (Fig. 11).
                                    </DIV><DIV class="block-paragraph"><DIV class="pic_info"><B>Fig. 11</B> The relationship between the protein intensity and the number of plasmids.
                                        </DIV></DIV><DIV class="block-paragraph">
                                        From this curve, we can clearly know that the relationship between number of plasmids and protein intensity. The concentration of arabinose have a positive relationship with protein intensity in a proper range. In the previous section, we think when protein intensity above the 24.42 units, the in-promoter will be repressed fully, so the threshold of the number of plasmids is calculated to be 210 copies.
                                    </DIV><DIV class="block-paragraph">
                                        In the wet lab, the strains containing different plasmids copy numbers separately overnight (16 h) growth in the LB medium.
                                    </DIV><DIV class="block-paragraph"><DIV class="pic_info"><B>Table. 4</B> The form about experimental set up.
                                        </DIV></DIV><DIV class="block-paragraph">
                                        After 16h growth in the LB medium, we measure the RFP intensity of seven groups. Then we calculate the relative RFP intensity by subtracting the negative control
                                    </DIV><DIV class="block-paragraph"><DIV class="pic_info"><B>Table. 5</B>.The RFP intensity of different circuits on different plasmids
                                        </DIV></DIV><DIV class="block-paragraph">
                                        From the table. 5, we can know that different plasmids copy numbers will lead to different RFP intensity. More plasmids copies, more RFP expressed.
                                    </DIV><DIV class="block-paragraph">
                                        At the same time, we constructed the constitutive promoter with in-promoter (Fig. 12) on the plasmids with different copy numbers to see the express of the RFP started by in-promoter.
                                    </DIV><DIV class="block-paragraph"><DIV class="pic_info"><B>Fig. 12</B> the circuit constructed on different plasmids.</DIV></DIV><DIV class="block-content-header">
                                        4. Employed different RBS and promoters with different strengths to change the inhibitor
                                    </DIV><DIV>
                                        In order to achieve the goal of regulating the threshold of plasmid number and finally control its number to meet different needs, we decided to set up different groups consist of different RBS and promoters with different strength
                                    </DIV><DIV class="block-paragraph"><DIV class="pic_info"><B>Table 6.</B> The strengths and efficiencies of different promoters and RBS
                                        </DIV><DIV class="pic_info"><B>Fig. 13</B> Constitutive circuits to reappear the threshold concentration of inhibitors. Only few will be selected and used to prove the concept of plasmid sensing.
                                        </DIV></DIV><DIV class="block-paragraph"><B>
                                            By now, we can give you a summary. By combining with our modeling part, we have successfully proved that:
                                        </B></DIV><DIV class="block-paragraph-padding">
                                        (1) The plasmid copies can affect the concentration of inhibitor in the constitutive circuit.
                                        (2) The concentration of inhibitor can affect the in-promoter’s activity and affect the expression of killer gene in the end.
                                        (3) Our system can help the cell sense the number of plasmids.
                                        (4) The number of plasmids can affect the expression of killer gene.
                                    </DIV><DIV class="block-paragraph">
                                        When the plasmid number is above its threshold, the concentration of inhibitor being high enough, the killer gene can be repressed completely.
                                    </DIV><DIV class="block-paragraph">
                                        When the plasmid number is below the threshold, the inhibitor won’t work anymore, the killer can get the chance.
                                    </DIV></DIV></DIV></DIV></DIV></DIV><DIV id="killer" class="block-title col-sm-12">Killer device</DIV><DIV class="block-content"><DIV class="block-content-item"><DIV class="block-content-item-block"><DIV>
                                Since the lethal efficiency of killer genes will decide the capacity of general circuit, so we have to:
                            </DIV><DIV style="margin: 10px auto;padding: 0 1em;">
                                1.	Prove that the toxin protein we selected can successfully express and the
                                <B>lethal effect</B> is obvious
                                2.	Adjust the translation efficiency of toxin proteins through replace ribosome binding
                                site (RBS), thus to <B>adjust the threshold</B>3.	Construct gene circuits connecting killer device and inhibitor device
                                4.	Find another possible way to produce the lethal effect
                            </DIV><DIV class="block-content-header">
                                Results:
                            </DIV><DIV style="margin: 0px auto;padding: 0 1em;">
                                1.	Successfully constructed the testing circuits of toxin proteins [Part:
                                <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2120002" style="color: blue;text-decoration: underline">BBa_K2120002</A> and Part:
                                <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2120003" style="color: blue;text-decoration: underline">BBa_K2120003</A>]
                                2.	After transformation, we measured the OD<SUB>600</SUB> to draw the growth curve and observed the function of toxin protein.
                                3.	In addition to B0032, we selected a stronger RBS B0034 and a weaker one B0031. Through measuring OD<SUB>600</SUB>, we demonstrated that we can adjust the lethal efficiency through replacing RBS.
                                4.	Successfully constructed the testing circuit of toxin proteins according to two kinds of inhibitors.
                                [For example, Part:
                                <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2120400" style="color: blue;text-decoration: underline">BBa_K2120400</A>]
                                5.	Tested the lethal effect of producing DSB through coupling sgRNA and Cas9
                            </DIV><DIV class="block-paragraph" style="margin-top: 30px;margin-bottom: 10px"><A data-toggle="collapse" href="#collapse1" style="color: blue;text-decoration: underline" aria-expanded="true" aria-controls="collapseOne">
                                    Experimental details <SPAN>[Expand]</SPAN></A></DIV><DIV id="collapse1" class="panel-collapse collapse " role="tabpanel" aria-labelledby="heading2"><DIV class="block-content-header"><B>1.	Constructing the testing circuits</B></DIV><DIV>
                                    The DNA Agarose Gel electrophoresis showed that the whole length of the first circuit is about 1500bp, the second circuit is about 1400bp.
                                </DIV><DIV><DIV class="pic_info"><B>Fig.1</B> DNA Agarose Gel electrophoresis results of <SPAN class="italic">mazF</SPAN></DIV><DIV class="pic_info"><B>Fig.2</B> DNA Agarose Gel electrophoresis results of <SPAN class="italic">hokD</SPAN></DIV></DIV><DIV class="block-content-header"><B>2.	Testing the lethal effect of toxin proteins MazF and HokD by measuring the growth curve.</B></DIV><DIV>
                                    The testing groups are the two circuits above (<SPAN class="italic">mazF</SPAN> and <SPAN class="italic">hokD</SPAN>), while the control is the empty vector. Add arabinose or not, there is another comparison. We used 0.01% arabinose to induce the P<SUB>BAD</SUB> promoter when the OD<SUB>600</SUB> is 0.6. We measured the OD<SUB>600</SUB> every hour until the bacteria reached the flat stage.
                                </DIV><DIV><DIV class="pic_info"><B>Fig.3</B> Compared with the negative control (the empty <SPAN class="italic">pSB1C3</SPAN> vector), OD<SUB>600</SUB> of the two circuits containing toxin proteins are obviously lower, and the difference is evident as time going on. No obvious difference observed among the three groups with no induction. It showed that toxin proteins didn’t leak out.
                                    </DIV>Through the growth curve, we concluded that toxin protein HokD and MazF have different lethal efficiency. Depending on different situations, we can choose either of them.
                                </DIV><DIV class="block-content-header"><B>3.	Replacement of the RBS to adjust the lethal effect as well as the threshold</B></DIV><DIV>
                                    Through one-step mutation, we have separately replaced the B0032 (33.96%) with B0031 (12.64%) and B0034 (100%). Here are the sequencing results.
                                </DIV><DIV><DIV class="pic_info"><B>Fig.4</B> The successful sequencing results of 4 mutations: B0031+<SPAN class="italic">mazF</SPAN>; B0034+<SPAN class="italic">mazF</SPAN>; B0031+<SPAN class="italic">hokD</SPAN>; B0034+<SPAN class="italic">hokD</SPAN></DIV></DIV><DIV>We used 0.01% arabinose to induce the P<SUB>BAD</SUB> promoter when the OD<SUB>600</SUB> is 0.6. We measured the OD<SUB>600</SUB> every hour until the bacteria reached the flat stage.
                                </DIV><DIV><DIV class="pic_info"><B>Fig.5</B> For the toxin protein MazF, we concluded that the influence of strong RBS is more evident.
                                    </DIV><DIV class="pic_info"><B>Fig.6</B> For the toxin protein HokD, the replacement of RBS will influence the expression of <SPAN class="italic">hokD</SPAN>. The weaker RBS B0031 leads to lower expression of toxin protein HokD.
                                    </DIV></DIV><DIV>We tested the lethal efficiency of HokD and MazF under different RBS. Compared with the positive control (<SPAN class="italic">pSB1C3</SPAN> empty vector), the three experimental groups showed obvious difference. For MazF, the strong RBS B0034 was proved to have highest lethal efficiency. For HokD, the weak RBS B0031 was proved to have the lowest lethal efficiency.
                                </DIV><DIV class="block-content-header"><B>4.	sgRNA targeting genome of strains with no NHEJ repair system</B></DIV><DIV>
                                    To test the effects of Cas9 and targeting sgRNA, we separately constructed two plasmids containing sgRNA and Cas9 protein. We co-transformed the two plasmids into <SPAN class="italic">E.coli TOP10</SPAN>, and added 10mM arabinose to induce the expression of Cas9. In order to validate the effect of Cas9 and sgRNA, we did quantitative analysis of colony formation after induction to confirm CRISPR/Cas9-Mediated DSB in <SPAN class="italic">E.coli TOP10</SPAN>.
                                </DIV><DIV><DIV class="pic_info"><B>Fig.7</B> experimental procedure
                                    </DIV></DIV><DIV>The cells were diluted 10<SUP>2</SUP>–10<SUP>4</SUP>  fold and plated onto agar plates containing <SPAN class="italic">AmpR</SPAN> and <SPAN class="italic">CmR</SPAN>. The results showed that the control group with no induction was growing better than the testing group when both of them were diluted 10<SUP>3</SUP>  fold, which means the DSB did have the lethal effect and can be used as a candidate for killer device.
                                </DIV><DIV><DIV class="pic_info"><B>Fig.8</B> Conﬁrmation of CRISPR/Cas9-Mediated DSB in <SPAN class="italic">E.coli TOP10</SPAN></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="recombination" class="block-title col-sm-12">Recombination</DIV><DIV class="block-content"><DIV class="block-content-item"><DIV class="block-content-item-block"><DIV class="block-content-header">
                                Aim
                            </DIV><DIV>
                                Considering the factor of the potential deficiency of executor killer due to the completely lost of plasmids and the number of &quot;in-promoters&quot; will affect our system, we decide to integrate the killer device into the genome. This group mainly provided a tool for genome integration. The insertion fragment is our testing device, including the expression cassette of <SPAN class="italic">rfp</SPAN> and killer gene. With this recombination system, we can get the upgraded type of our P-SLACKiller.
                            </DIV><DIV class="block-content-header">
                                Results
                            </DIV><DIV style="margin: 10px auto;padding: 0 1em;">
                                1.	Coupled CRISPR/Cas9 system with λ-Red recombineering to integrate the donor fragments, but we haven’t finished the second plasmid construction
                                2.	Tried the traditional lambda Red recombination and successfully inserted four testing devices into the genome, locus of <SPAN class="italic">LacI</SPAN>.
                            </DIV><DIV class="block-paragraph" style="margin-top: 30px;margin-bottom: 10px"><A data-toggle="collapse" href="#collapse2" style="color: blue;text-decoration: underline" aria-expanded="true" aria-controls="collapseTwo">
                                    Experimental details <SPAN>[Expand]</SPAN></A></DIV><DIV id="collapse2" class="panel-collapse collapse " role="tabpanel" aria-labelledby="heading2"><DIV class="block-title" style="padding: 5px;font-size: 1.2em">
                                    Coupling CRISPR/Cas9 system with λ-Red recombineering
                                </DIV><DIV class="block-content-header"><B>1.	Plasmid construction</B></DIV><DIV>
                                    We designed a two-plasmid system applied for it. There was a low-copy plasmid <SPAN class="italic">pSTV29</SPAN> for Cas9 and λ-Red expression from the arabinose-inducible promoter P<SUB>BAD</SUB> respectively and a high-copy plasmid <SPAN class="italic">pUC19</SPAN> for sgRNA expression from a constitutive promoter J23119.
                                </DIV><DIV><DIV class="pic_info"><B>Fig.1</B> Double plasmids to meet different requirements for each element. Equip the high copy number plasmid <SPAN class="italic">pUC19</SPAN> with sgRNA targeting site on genome and donor fragment with homologous arms. The low copy number plasmid are equipped with expression cassette of Cas9 protein and λ-Red recombinase
                                    </DIV></DIV><DIV>We mainly used <SPAN style="color: blue;">
                                     Gibson assembly
                                </SPAN> to construct these two plasmids. The <SPAN class="italic">pUC19</SPAN> plasmid containing sgRNA and donor fragment was successfully constructed. Separately, we successfully constructed the lambda Red protein and Cas9 protein on <SPAN class="italic">pSTV29</SPAN>.
                                </DIV><DIV>
                                    However, we didn’t finish the second plasmid construction due to the limit of time. Instead, we decided to employ the traditional lambda Red recombination system to carry out the integration process since we received the plasmid <SPAN class="italic">pKD46</SPAN> from Dr. Bo Lv.
                                </DIV><DIV class="block-title" style="padding: 5px;font-size: 1.2em">
                                    Lambda red recombination
                                </DIV><DIV class="block-paragraph">
                                    For facile blue and white colony screening, the target site we chose is the 5 ’end of <SPAN class="italic">LacZ</SPAN> gene encoding the active site of β-galactosidase. At first, we used strain <SPAN class="italic">TOP10</SPAN> as our host. To test the experiment conditions which is optimal for transformation, we did the pre-experiment and used the resistance Kana gene as the donor fragment to target the gene locus of <SPAN class="italic">LacZ</SPAN>. We have tried several times, but failed.
                                </DIV><DIV class="block-paragraph">
                                    Considering that the complementary sequence between donor fragment and target site is one of the most important factors deciding the successful rate of recombination, we re-designed our experiment. The whole sequence of <SPAN class="italic">TOP10</SPAN> is not open access and the specific sequence for <SPAN class="italic">LacZ</SPAN> has been modified and we couldn’t confirm the modified one. So we decided to replace the host to <SPAN class="italic">DH5α</SPAN>.
                                </DIV><DIV class="block-content-header"><B>1.	Transformed the plasmid <SPAN class="italic">pKD46</SPAN> into host and made it competent cell after induced by L-arabinose</B></DIV><DIV>
                                    We transformed the <SPAN class="italic">pKD46</SPAN> into competent cell <SPAN class="italic">DH5α</SPAN>, and added arabinose to induce the expression of recombinase. When the OD<SUB>600</SUB> is about 0.6 which means the bacteria is at the log phase, we made it competent for electroporation transformation.
                                </DIV><DIV class="block-content-header"><B>2.	Preparation of donor fragment</B></DIV><DIV>
                                    We designed homologous arms with the length of 50bp and 500 bp. The short one was designed on the primer, while the long arm was constructed through over-lap extension PCR
                                    (<SPAN style="color: blue;">OE-PCR</SPAN>).
                                </DIV><DIV class="block-paragraph">
                                    After several times of try-error, we confirmed the comparatively optimal condition for lambda Red recombination. The target site is <SPAN class="italic">LacI</SPAN> and the host is <SPAN class="italic">DH5α</SPAN>. The homologous arm of 50 bp is enough for recombination.
                                </DIV><DIV class="block-paragraph">
                                    Using the <SPAN class="italic">pKD3</SPAN> as template, we added the cat expression cassette behind the testing parts, the whole donor fragment was sandwiched by 50bp left and right homologous arm.
                                </DIV><DIV class="block-paragraph"><DIV class="pic_info"><B>Fig.2</B>Four testing circuits used as our donor fragment
                                    </DIV><DIV class="pic_info">
                                        Fig.3 DNA Agarose Gel electrophoresis of donor fragments
                                    </DIV></DIV><DIV class="block-content-header"><B>3.	Transformed the donor fragment through electroporation into the competent cell we prepared before</B></DIV><DIV>
                                    After transformation, we observed clones on the chloramphenicol plates. We selected the positives clones through colony PCR.
                                </DIV><DIV class="block-paragraph"><DIV class="pic_info">
                                        Fig.4 Recombination model and the sites of verifying primers
                                    </DIV></DIV><DIV class="block-paragraph">
                                    According to our design, the donor fragment is about 2000 bp, while the substituted fragment on genome is 1083bp, which means the successful integration will result in over 1000 bp increasing of length on the locus of <SPAN class="italic">LacI</SPAN>. We used the verifying primers to test whether we have successfully integrated these donor fragments.
                                </DIV><DIV><DIV class="pic_info"><B>Fig.5</B> The positive clones are 1~7, separately indicating the testing circuits above, and 8 is the negative control whose template is the original genome without any integration.
                                    </DIV></DIV><DIV class="block-paragraph">Compared with the negative control with no integration occurred, the colony PCR showed that we have successfully integrated the four circuits into the <SPAN class="italic">LacI</SPAN> locus of genome.
                                </DIV></DIV></DIV></DIV></DIV><DIV id="promoter" class="block-title col-sm-12">Site-directed promoter mutation</DIV><DIV class="block-content"><DIV class="block-content-item"><DIV class="block-content-item-block"><DIV>
                                We have two combinations of inhibitor and &quot;in-promoter&quot;: <SPAN class="italic">TetR</SPAN>-P<SUB>Tet</SUB>, <SPAN class="italic">CI</SPAN>-P<SUB>R</SUB>. The strength of &quot;in-promoter&quot; will decide the expression level of killer gene. Directly applied these two combinations, we got the initial threshold. In order to adjust the threshold, we planned to mutate the &quot;in-promoter&quot; and got mutants with various strengths. We chose P<SUB>Tet</SUB> as our target promoter. Through literature research, we chose the -35 region as our mutation region. We mainly used RFP to indicate the promoter strength in our wet experiment.
                            </DIV><DIV class="block-content-header">
                                Results
                            </DIV><DIV style="margin: 10px auto;padding: 0 1em;">
                                1.	Tried to use one-step mutation, but because of the secondary structure of promoters, we didn’t get positive results
                                2.	Built a library of mutated P<SUB>Tet</SUB> promoters and characterized the promoter activity through measuring the red fluorescence intensity
                                3.	Selected 4 mutants and got the successful sequencing results
                            </DIV><DIV class="block-paragraph" style="margin-top: 30px;margin-bottom: 10px"><A data-toggle="collapse" href="#collapse3" style="color: blue;text-decoration: underline" aria-expanded="true" aria-controls="collapseTwo">
                                    Experimental details <SPAN>[Expand]</SPAN></A></DIV><DIV id="collapse3" class="panel-collapse collapse " role="tabpanel" aria-labelledby="heading2"><DIV class="block-content-header"><B>1.	One step mutation--plan A</B></DIV><DIV>
                                    At first, we used “Quickchange”, using the methylated plasmid as a template, which was extracted from <SPAN class="italic">E.coli</SPAN>, and we designed a pair of 25pb primers containing the desired mutation to amplify the DNA through PCR, and we got the demethylated plasmids. Finally, we treated the products with Dpn1, which was specific for methylated DNA and was used to digest the parental DNA template and selected for mutation-containing synthesized DNA. And we used the method of thermal stimulation to transform it into <SPAN class="italic">E.coli</SPAN>.

                                </DIV><DIV class="block-paragraph">
                                    But we didn’t we get the positive clone of the plasmid. And we thought as a result of the characteristics of the promoter structure, viewing it from the secondary structure, there appears a cervical-loop structure. Also a mutation site just appears in the loop, which results in primers having a comparatively long homologous region. So the complementary pairing occurs between upstream and downstream primer, forming primer dimer, which lead PCR amplification to failure.
                   </DIV><DIV class="block-content-header"><B>
                                        2.	Digestion and ligation—plan B
                                    </B></DIV><DIV>
                                    To avoid forming the cervical-loop structure, we gave up the one-step mutation approach. Instead, we designed the mutation on the primers and planned to construct the testing circuit through the traditional digestion and ligation.
                                </DIV><DIV class="block-paragraph">
                                    We designed random primers containing NNNNNN in -35 region.

                                </DIV><DIV class="block-paragraph">
                                    Through PCR amplification, we purified the PCR product. We used DpnI to digest the template. Constructed the fragment on the vector <SPAN class="italic">pSB1C3</SPAN>, we transformed the ligation product into <SPAN class="italic">E.coli TOP10</SPAN>. After incubation at 37℃for 16h, we observed the formation of single clone.
                                </DIV><DIV class="block-paragraph">
                                    Through colony PCR, we selected the clone on the plate and had successfully got the positive clones. To facilitate the characterization of promoter strength, we constructed circuits containing <SPAN class="italic">rfp</SPAN>. The whole length is 923bp, counting the region between VF and VR, the positive length should be over 1000. Results shown in Fig.1.
                                </DIV><DIV class="block-paragraph"><DIV class="pic_info">
                                        Fig.1 positive clones of mutated promoters containing RFP
                                    </DIV></DIV><DIV class="block-paragraph">
                                    Then we selected the positive clones, incubated it at the tube, and measured the RFP intensity. Under the same condition, the fluorescence intensity of original promoter was 5000, and the P<SUB>Tet</SUB>-1, P<SUB>Tet</SUB>-2 and P<SUB>Tet</SUB>-3 were about 200, while the selected No.25 was about 500.
                                </DIV><DIV><DIV class="pic_info"><B>Fig.2</B> RFP intensity measurement results. The original P<SUB>Tet</SUB> has highest activity, while the strength of mutant P<SUB>Tet</SUB>-25 is about 1/10 of the original one. Other mutants has considerably low activity.
                                    </DIV></DIV><DIV class="block-paragraph">Compared with the original promoter, the promoter activity after mutation was sharply decreased while the difference was obvious between the testing group (data over 200) and the negative control (data below 100). Since we employed P<SUB>Tet</SUB> to control the expression of toxin proteins, it’s better that we can reduce the background expression as much as possible. The sequencing results confirmed the successful mutation of four mutants.
                                </DIV><DIV><DIV class="pic_info"><B>Fig.3</B> The sequencing results. Mutation occurred only in -35 region.
                             </DIV></DIV><DIV class="block-paragraph">Through measuring the RFP intensity, we selected four mutants. They can be used to control the expression of toxin proteins.
                                </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="slow-transition nav-left" id="nav_left_wrapper"><DIV class="nav-left-title" style="position:relative;width: 200px;padding: 0;"><SPAN style="color: #FFFFFF;font-size: 18px;font-weight: bold;position: absolute;
        right: 40px;bottom: 20px;">Results</SPAN></DIV><UL class="nav_left_ul" style="padding-left: 50px;"><LI><A href="#threshold" class="nav_point"><SPAN class="nav_left_txt">Threshold</SPAN></A></LI><LI><A href="#killer" class="nav_point"><SPAN class="nav_left_txt">Killer</SPAN></A></LI><LI><A href="#recombination" class="nav_point"><SPAN class="nav_left_txt" style="font-size: 14px;">Recombination</SPAN></A></LI><LI><A href="#promoter" class="nav_point"><SPAN class="nav_left_txt">Promoter</SPAN></A></LI></UL></DIV><DIV id="site-change-btn" style="position: absolute;left: 20px;bottom: 20px;z-index:9;display: none;"><A href="https://2016.igem.org/Team:BIT-China/Design"><SPAN style="color: #654D6F;font-weight: bold;font-size: 12px">Design</SPAN></A><A href="https://2016.igem.org/Team:BIT-China/Proof"><SPAN style="color: #654D6F;font-weight: bold;font-size: 12px">
            Proof</SPAN></A></DIV></DIV><DIV class="col-sm-12" style="padding: 0;"><FOOTER class="footer dark"><DIV class="footer-widgets md-padding-top"><DIV class="container"><DIV class="row"><DIV class="col-sm-3"><DIV class="widget widget-about"><H4 class="widget-title" style="margin-bottom: 43px">P-SLACKiller</H4><P><B>· </B>Work or Die<B>· </B>Monitor Plasmid Numbers<B>· </B>Kill the Slacker<B>· </B>Maintain the Better Biosystem</P></DIV></DIV><DIV class="col-sm-3"><DIV class="widget widget-categories"><H4 class="widget-title" style="margin-bottom: 43px">Useful Links</H4><UL><LI><A href="https://2016.igem.org/Team:BIT-China/Medals" target="_blank">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Results" target="_blank">Results</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Design" target="_blank">Design</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Proof" target="_blank">Proof</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/HP/Gold" target="_blank">HP</A></LI></UL></DIV></DIV><DIV class="col-sm-3"><DIV class="widget"><H4 class="widget-title" style="margin-bottom: 43px">Sponsors</H4></DIV></DIV><DIV class="col-sm-3"><DIV class="widget"><H4 class="widget-title" style="margin-bottom: 43px">Get in touch</H4><P><STRONG>Email:</STRONG> iGEM_BIT_China@163.com <STRONG>Location:</STRONG> No.5 Zhongguancun South Street </P></DIV></DIV></DIV></DIV></DIV><DIV class="footer-meta"><DIV class="container"><DIV class="row"><DIV class="col-sm-12"><DIV class="copyright">BIT-China © Copyright 2016  Powered by Bootstrap.</DIV></DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>