<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_DTU-Denmark_Software skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:DTU-Denmark/Software</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>New HTML template for the wiki</TITLE></P><DIV id="headerWrapper"><DIV id="Topnav" class="container"><DIV id="logo"><BUTTON id="hamburger" class="mobile-menue">☰</BUTTON><BUTTON id="arrow" class="mobile-menue">⇦</BUTTON><BUTTON id="a-down" class="mobile-menue mobile-scrole">Jump to</BUTTON></DIV><DIV id="navigation"><UL id="nav-items"><LI><A href="https://2016.igem.org/Team:DTU-Denmark/project" class="project nav-item" id="project">Project</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/wetlab" class="wetlab nav-item" id="wetlab">Wetlab</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/compute" class="compute nav-item" id="compute">Compute</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/team" class="team nav-item" id="team">Team</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/outreach" class="outreach nav-item" id="outreach">Outreach</A></LI></UL></DIV></DIV><DIV id="sub-navigation-wrapper"><DIV id="sub-navigation" class="container"><UL id="project-sub"><LI><A href="https://2016.igem.org/Team:DTU-Denmark/description" class="sub-nav-item">Description</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Proof" class="sub-nav-item">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/achievements" class="sub-nav-item">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Notebook" class="sub-nav-item">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Safety" class="sub-nav-item">Safety</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Description" class="sub-nav-item">Parts</A></LI></UL><UL id="wetlab-sub"><LI><A href="https://2016.igem.org/Team:DTU-Denmark/substrate" class="sub-nav-item">Substrate</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/products" class="sub-nav-item">Products</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/molecular_toolbox" class="sub-nav-item">Molecular Toolbox</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/molecular_interlab" class="sub-nav-item">Interlab</A></LI></UL><UL id="compute-sub"><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Software" class="sub-nav-item">Codon Optimization</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Hardware" class="sub-nav-item">Microfermenter</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Model" class="sub-nav-item">Modeling</A></LI></UL><UL id="team-sub"><LI><A href="https://2016.igem.org/Team:DTU-Denmark/team_members" class="sub-nav-item">Team Members</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/team_building" class="sub-nav-item">Team Building</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Attributions" class="sub-nav-item">Attributions</A></LI></UL><UL id="outreach-sub"><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Collaborations" class="sub-nav-item">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/HP/Silver" class="sub-nav-item">Practices</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/HP/Gold" class="sub-nav-item">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Engagement" class="sub-nav-item">Biosensor</A></LI></UL></DIV></DIV></DIV><P><TITLE>Bootstrap Example</TITLE></P><DIV class="MYbody"><DIV class="masthead"><DIV class="container-fluid"><DIV class="row"><DIV class="col-md-12 thumbnail" style="background-image:url(https://static.igem.org/mediawiki/2016/d/df/T--DTU-Denmark--compute-header.jpg)"><DIV class="caption"><DIV class="col-md-5 col-sm-5 col-xs-12 title"><H1>CODON OPTIMIZATION SOFTWARE<P class="lead">The time for implementation of sophisticated computational approaches into biology has finally come. The DTU Biobuilders team was well aware of this fact and therefore took a leap forward by developing TaiCO, a unique specialized yet user-friendly codon optimization tool based on species specific tAI calculation.</P></H1></DIV><DIV class="col-md-5 col-sm-5 hidden-xs intro"><BLOCKQUOTE class="blockquote-reverse"><P>&quot;There are two ways of constructing a software design. One way is to make it so simple that there are obviously no deficiencies. And the other way is to make it so complicated that there are no obvious deficiencies.&quot;</P><SMALL><I>C.A.R. Hoare</I></SMALL></BLOCKQUOTE></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-md-9 col-sm-10 colLeft"><DIV><H2 class="h2">Overview</H2><BLOCKQUOTE class="visible-xs"><P>&quot;There are two ways of constructing a software design. One way is to make it so simple that there are obviously no deficiencies. And the other way is to make it so complicated that there are no obvious deficiencies.&quot;</P><SMALL><I>C.A.R. Hoare</I></SMALL></BLOCKQUOTE><P>
                The increased use of non-conventional organisms for conventional purposes increases the need for codon optimization of coding sequences that are used for heterologous protein production. Codon optimization is typically performed by replacing each codon with the most frequently used synonymous from the host genome. The assumption that the most frequent synonymous codon is also the most efficiently translated codon is not necessarily true when it is considered that a typical or average transcript is not one that is likely to have a higher than average translation efficiency. Highly translated transcripts often contain “reserved” codons that are not the most common. Instead, these transcripts contain codons that best match the tRNA pools in the cell. These tRNA pools can be estimated by the number of tRNA genes that have an anticodon  which corresponds to a given codon. This approach is known as the tRNA Adaptation Index (tAI)<SUP><A href="#references">1</A></SUP> when used to assess the translation efficiency of a coding sequence.
            </P><H3 class="h3">From Blackboard Calculations to Software</H3><P>
                TaiCO (tAI Codon Optimaztion tool) constitutes a unique computational tool for answering  the common biological question: What coding DNA sequence will result in maximum protein expression? The DTU Biobuilders' proposal for solving this task is a stand-alone application that is based completely on species specific tAI calculation and is bundled with a simplistic Graphic User Interface (GUI) compatible with many platforms. We hope that this software will contribute to faster and easier to production of biotechnological results and become a high-end optimizing method with its unique theory implementation.
            </P><DIV class="col-md-10"><FIGURE class="figure"><FIGCAPTION class="figure-caption">The <B>TaiCO</B> Interface</FIGCAPTION></FIGURE><DIV id="img1Modal" class="modal"><SPAN class="close img1">×</SPAN><DIV class="caption">The <B>TaiCO</B> Interface</DIV></DIV></DIV></DIV><DIV><H2 class="h2">Theory</H2><P>
				As mentioned, the central issue in codon optimization is to determine which codons are most efficiently translated for each amino acid. The quantity needed for this task is called 'translatability' and is denoted \(W_i\) for the \(i\)'th codon.</P><P>
				To accomplish this, we have chosen to use a tRNA Adaptation Index-based method (tAI). The fundamental assumption behind this method is that highly expressed proteins have their genes encoded with a set of codons that is overall more susceptible to tRNA-binding and translation compared to proteins that are not highly expressed. Hence, this optimization method estimates the codon preferences in such a way that the correlation between protein level and tAI is maximized.</P><P>
				The formulas for calculating individual \(W_i\)'s were stated by dosReis<SUP><A href="#references">1</A></SUP>. All 64 \(W_i\)'s can be calculated in one matrix multiplication, by letting \(G\) be the 4\(\times\)16 matrix consisting of the tGCN's (in TaiCO referred to as 'gcn') and letting \(S\) be the 4\( \times\)4 matrix containing the (1 \(-s_{ij}\)) values. Hence,
			</P><P>
				$$W = SG$$
			</P><P>
				The computed \(W_i\)'s are then normalized by putting \(w_i = W_i/W_{\text{max}}\), and those normalized translatabilities, \(w_i\) do then form the basis for codon selection. Higher \(w_i\)-values are simply selected over lower values. 
			</P><H3 class="h3">The \(G\) Matrix</H3><P>
						\(G\) consists of 64 tGCN values, which are the gene copy number of tRNA's recognizing specific codons. Normally, available gcn-files list the tGCN's in terms of the reversed anticodon corresponding to the recognized codon, hence, the tricodons in the raw gcn-files are reversed and have their bases replaced by the complementary ones. For instance, in <I>S. cerevisiae</I> the gcn of tRNA's recognizing TTC (encoding glutamic acid) is 10, so in the raw file, this information is presented as the reversed anticodon, GAA, being equal to 10 instead. When converted into their encoding form, the tGCN's are put into the \(G\) matrix such that each column has the first two position fixed and each row has a fixed third position:
	                </P><TABLE><TBODY><TR><TD>AAA</TD><TD>ACA</TD><TD>AGA</TD><TD>ATA</TD><TD>CAA</TD><TD>CCA</TD><TD>CGA</TD><TD>CTA</TD><TD>GAA</TD><TD>GCA</TD><TD>GGA</TD><TD>GTA</TD><TD>TAA</TD><TD>TCA</TD><TD>TGA</TD><TD>TTA
						</TD></TR><TR><TD>AAC</TD><TD>ACC</TD><TD>AGC</TD><TD>ATC</TD><TD>CAC</TD><TD>CCC</TD><TD>CGC</TD><TD>CTC</TD><TD>GAC</TD><TD>GCC</TD><TD>GGC</TD><TD>GTC</TD><TD>TAC</TD><TD>TCC</TD><TD>TGC</TD><TD>TTC
						</TD></TR><TR><TD>AAG</TD><TD>ACG</TD><TD>AGG</TD><TD>ATG</TD><TD>CAG</TD><TD>CCG</TD><TD>CGG</TD><TD>CTG</TD><TD>GAG</TD><TD>GCG</TD><TD>GGG</TD><TD>GTG</TD><TD>TAG</TD><TD>TCG</TD><TD>TGG</TD><TD>TTG
						</TD></TR><TR><TD>AAT</TD><TD>ACT</TD><TD>AGT</TD><TD>ATT</TD><TD>CAT</TD><TD>CCT</TD><TD>CGT</TD><TD>CTT</TD><TD>GAT</TD><TD>GCT</TD><TD>GGT</TD><TD>GTT</TD><TD>TAT</TD><TD>TCT</TD><TD>TGT</TD><TD>TTT
						</TD></TR></TBODY></TABLE><H3 class="h3">The \(S\) Matrix</H3><P>
						While \(G\) is precisely known, \(S\) needs to be optimized. In dosReis 2004, the optimized \(s_{ij}\)-values for <I>S. cerevisiae</I> are published, yielding the \(S\)-matrix,
						$$
						    S =
						    \begin{pmatrix}
						        1 &amp; 0 &amp; 0 &amp; 0.0001 \\
						        0 &amp; 1 &amp; 0 &amp; 0.72 \\
						        0.32 &amp; 0 &amp; 1 &amp; 0 \\
						        0 &amp; 0.59 &amp; 0 &amp; 1
						    \end{pmatrix}
						$$
						where both rows and columns are ordered as A,C,G,T. Thus, the \(W_i\)'s computed from the \(SG\) multiplication are each influenced by two tGCN's. As an example, calculating the translatability of CCG will be equal to the dot product of the third row of \(S\) (because the third position is a G), and the sixth row of \(G\) (because the first two positions are CC):
						$$
						    W_{CCG} = 0.32 \cdot \text{tGCN}_{CCA} + 1 \cdot \text{tGCN}_{CCG}
						$$
						clearly taking the wobbling potential of G to A in the third position into account.
	                </P></DIV><DIV><H2 class="h2">TaiCO Features</H2><P>
            Our proposal for reliable and fast computational production of optimized DNA sequences comes under the name TaiCO. The need for a specialized software tool for optimization of <I>Y. lipolytica</I> DNA sequences became evident when the product subgroup of DTU Biobuilders started to design constructs for protein expression. TaiCO allowed the rapid analysis of the coding sequences of interest and due to the final simplistic architecture and low resources demands it was decided to extend its capabilities for every organism with tGCN files available.  
        </P><H3 class="h3">Software Overview</H3><P>
             TaiCO is implemented in <A href="https://www.python.org/download/releases/3.0/">Python3</A>, a high-level programming languages with a built-in library for GUI development called tkinter. The algorithm was structured in an easily modifiable layout, due to its static philosophy with the exclusive usage of only built-in libraries and modules in addition to the already known and commonly used &quot;Pythonic&quot; data structures (e.g dictionaries,lists). This software comes with the Open Software license <A href="https://www.gnu.org/licenses/gpl-3.0.txt">GPL v3.</A></P><P>
             For a more descriptive view on how the algorithm was implemented, links for the available versions are provided in a further section.
        </P><H3 class="h3">Input Files and Result</H3><P>
            The first input file requested from TaiCO is a GCN table in simple text format. Although the software comes bundled with 7 GCN files from model organisms, other GCN tables can be uploaded. The second input file that the user has to provide is a list with a single or even multiple protein sequences in FASTA format. The final input file that the user can provide is the powerful capability of parsing a simple text file including the sequences of the restriction sites that have to be absent from the optimized DNA resulting sequences. The output of the analysis is a file saved in a FASTA format that contains all the optimized DNA sequences.
        </P><DIV class="panel-group" id="accordion1" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><A data-toggle="collapse" href="#collapse1" aria-expanded="false" aria-controls="collapse1"><DIV class="panel-heading" role="tab" id="heading1"><H4 class="panel-title">
                    Optimization - 3 Steps away from your lab (tutorial)   
                </H4></DIV></A><DIV id="collapse1" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading1"><DIV class="panel-body"><FIGURE class="figure"><FIGCAPTION class="figure-caption">Step 1: Double click on executable named &quot;TaiCO&quot;</FIGCAPTION></FIGURE><DIV id="...Modal" class="modal"><SPAN class="close ...">×</SPAN><DIV class="caption">Step 1: Double click on executable named &quot;TaiCO&quot; </DIV></DIV><FIGURE class="figure"><FIGCAPTION class="figure-caption">Step 2: Click on search file and select the file from your system (repeat for all &quot;Search file&quot; options)</FIGCAPTION></FIGURE><DIV id="...Modal" class="modal"><SPAN class="close ...">×</SPAN><DIV class="caption">Step 2: Click on search file and select the file from your system (repeat for all &quot;Search file&quot; options)</DIV></DIV><FIGURE class="figure"><FIGCAPTION class="figure-caption">Step 3: Click &quot;Start analysis&quot; button and wait until the successful message is poped-up
              </FIGCAPTION></FIGURE><DIV id="...Modal" class="modal"><SPAN class="close ...">×</SPAN><DIV class="caption">Step 3: Click &quot;Start analysis&quot; button and wait until the successful message is poped-up</DIV></DIV></DIV></DIV></DIV></DIV><H3 class="h3">Core Algorithmic Approach, Runtime and Distribution</H3><P>
            The completed script was “converted” into an executable file along with all included modules using the PyInstaller<SUP><A href="#references">2</A></SUP> software. This allowed us to make TaiCO available for almost all Unix-based and Windows platforms. Due to the nature of the supporting PyInstaller software the user has only one mandatory computational task in order to be able to run the software, which is to download the preferred zipped version. The application runtime remains under 5 sec (system dependent) due to the computationally efficient production of optimized sequences codon by codon according to the corresponding maximum \(w_i\) value. In the case of restriction site elimination the selection of codons is performed along with the sequence production and less optimized codons are therefore incorporated. After the downloading procedure is done, the careful reading of the two README files in the relevant folder is strongly recommended. The system compatible version of TaiCO can be downloaded by clicking one of the following links:</P><P><B>Windows</B>: Click <A href="https://static.igem.org/mediawiki/2016/5/5a/TaiCO_windows.zip">here</A> to download Windows version
        </P><P><B>Unix</B>: Click <A href="https://static.igem.org/mediawiki/2016/0/0a/TaiCO_Ubuntu.zip">here </A> to download Unix version
        </P><P><B>Mac OS X</B>: There is no specific bundle for MacOS, both links from above can be used and after python3 installation the original source code can be ran with the following command : <B>python3 TaiCO.py</B></P><P>
            For further information regarding terms of use and how to use TaiCO properly you are strongly advised to inspect the README.txt file or contact the author by email: vrantos@hotmail.gr
        </P></DIV><DIV id="ref_sec"><H2 class="h2">References</H2><OL><LI>dos Reis, Mario, Renos Savva, and Lorenz Wernisch. &quot;Solving the riddle of codon usage preferences: a test for translational selection.&quot; Nucleic acids research 32.17 (2004): 5036-5044.</LI><LI><A href="http://www.pyinstaller.org/">PyInstaller Official Page</A></LI></OL></DIV></DIV><DIV class="col-md-3 col-sm-2 colRight" id="scrollspy"><UL class="nav" id="sidebar"><LI><A href="#section-1">Overview</A></LI><LI><A href="#section-2">Theory</A></LI><LI><A href="#section-3">TaiCO features</A></LI><LI><A href="#references">References</A></LI></UL></DIV></DIV></DIV></DIV><FOOTER id="footer"><DIV class="container"><DIV align="center" class="col-md-3"><UL><LI>FIND US AT:</LI></UL><UL><LI><U>DTU BIOBUILDERS</U></LI><LI>DENMARK</LI><LI>DTU - SØLTOFTS PLADS, BYGN. 221/002</LI><LI>2800 KGS. LYNGBY</LI><LI><U>E-mail:</U></LI><LI>dtu-biobuilders-2016@googlegroups.com</LI></UL></DIV><DIV align="center" class="col-md-6"><UL><LI>MAIN SPONSORS:</LI></UL></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>