<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_DTU-Denmark_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:DTU-Denmark/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>New HTML template for the wiki</TITLE></P><DIV id="headerWrapper"><DIV id="Topnav" class="container"><DIV id="logo"><BUTTON id="hamburger" class="mobile-menue">☰</BUTTON><BUTTON id="arrow" class="mobile-menue">⇦</BUTTON><BUTTON id="a-down" class="mobile-menue mobile-scrole">Jump to</BUTTON></DIV><DIV id="navigation"><UL id="nav-items"><LI><A href="https://2016.igem.org/Team:DTU-Denmark/project" class="project nav-item" id="project">Project</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/wetlab" class="wetlab nav-item" id="wetlab">Wetlab</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/compute" class="compute nav-item" id="compute">Compute</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/team" class="team nav-item" id="team">Team</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/outreach" class="outreach nav-item" id="outreach">Outreach</A></LI></UL></DIV></DIV><DIV id="sub-navigation-wrapper"><DIV id="sub-navigation" class="container"><UL id="project-sub"><LI><A href="https://2016.igem.org/Team:DTU-Denmark/description" class="sub-nav-item">Description</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Proof" class="sub-nav-item">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/achievements" class="sub-nav-item">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Notebook" class="sub-nav-item">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Safety" class="sub-nav-item">Safety</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Description" class="sub-nav-item">Parts</A></LI></UL><UL id="wetlab-sub"><LI><A href="https://2016.igem.org/Team:DTU-Denmark/substrate" class="sub-nav-item">Substrate</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/products" class="sub-nav-item">Products</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/molecular_toolbox" class="sub-nav-item">Molecular Toolbox</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/molecular_interlab" class="sub-nav-item">Interlab</A></LI></UL><UL id="compute-sub"><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Software" class="sub-nav-item">Codon Optimization</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Hardware" class="sub-nav-item">Microfermenter</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Model" class="sub-nav-item">Modeling</A></LI></UL><UL id="team-sub"><LI><A href="https://2016.igem.org/Team:DTU-Denmark/team_members" class="sub-nav-item">Team Members</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/team_building" class="sub-nav-item">Team Building</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Attributions" class="sub-nav-item">Attributions</A></LI></UL><UL id="outreach-sub"><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Collaborations" class="sub-nav-item">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/HP/Silver" class="sub-nav-item">Practices</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/HP/Gold" class="sub-nav-item">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Engagement" class="sub-nav-item">Biosensor</A></LI></UL></DIV></DIV></DIV><P><TITLE>Bootstrap Example</TITLE></P><DIV class="MYbody"><DIV class="masthead"><DIV class="container-fluid"><DIV class="row"><DIV class="col-md-12 thumbnail" style="background-image:url(https://static.igem.org/mediawiki/2016/d/df/T--DTU-Denmark--compute-header.jpg)"><DIV class="caption"><DIV class="col-md-5 col-sm-5 col-xs-12 title"><H1>Genome Scale Modeling<P class="lead">What medium should I use to grow my <I>Yarrowia</I>? Which genes should I mutate to make it produce more beta-carotene? Let genome scale modeling(GSM) help you design your dream organism. Using a GSM of <I>Yarrowia Lipolytica</I> (<I>Y.lipolytica</I>), we explored how <I>Y.lipolytica</I> grows on different substrates and found out what genes need to be manipulated to optimize beta-carotene production. 
</P></H1></DIV><DIV class="col-md-5 col-sm-5 hidden-xs intro"><BLOCKQUOTE class="blockquote-reverse"><P> &quot;I've always seen modeling as a stepping stone.&quot; </P><SMALL>Tyra Banks </SMALL></BLOCKQUOTE></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-md-9 col-sm-10 colLeft"><DIV><H2 class="h2">Overview</H2><BLOCKQUOTE class="visible-xs"><P> &quot;I've always seen modeling as a stepping stone.&quot; </P><SMALL>Tyra Banks </SMALL></BLOCKQUOTE><P><I>Yarrowia lipolytica</I> is a versatile organism. At the start of our project, there are two main production projects in the wetlab. We aimed to produce proinsulin as well as beta-carotene. The beta-carotene project was later shelved, due to problems with the biobricks and cloning. The modeling team, however, saw this as a challenge and adopted genome scale modeling to investigate <I>Y. lipolytica.</I></P><P> A genome-scale-model (GSM) is a representation of the metabolic network in a form that computers can understand and use to calculate the fluxes (rates of reactions) of the system. </P><P> The GSM effort is motivated by future wetlab engineering promises. So if our team, or another, reinitiates the beta-carotene project, the computational support is already in place to guide the wetlab teams towards more optimal growth environment and strain design. </P><P><STRONG>We used GSM for two purposes: </STRONG></P><OL><LI>Explore how <I>Y. lipolytica</I> grows in different environments and identify optimal conditions for growth.</LI><LI>Predict what genes can be amplified to enhance beta-carotene yield.</LI></OL><H3 class="h3">Results</H3><P>
			We used the most recently published GSM for <I>Y. lipolytica</I><SUP><A href="#references"> 5 </A></SUP>, solved using Cameo<SUP><A href="#references"> 9 </A></SUP> and Cobra Matlab. </P><P>
We  fulfilled purpose 1: using Phenotype Phase Planes (PHPP). We investigated different Carbon sources as substrates and identified lines of optimality (the optimal relationship between substrate and oxygen uptake rates). The simulation results are supported by the substrate screening experiments, adding the value of predicting the effect of substrate uptake rates on growth behaviour.
</P><P>
We fulfilled purpose 2: using Flux Scanning based on Enforced Objective Flux (FSEOF) simulations. We identified the short list of genes that could be amplified to optimize beta-carotene production. 
</P><P>
The results will serve as excellent starting point and guidance for future genetic engineering efforts in the lab.
</P></DIV><DIV><H2 class="h2">Theory</H2><P>
As annotated whole genome sequences become common, It is possible to reconstruct a genome scale representations of the metabolic network as a computational model. The GSM has information on what metabolites are in the cell, which reactions each metabolite takes part in, the stoichiometry of the reactions, reactions’ rate limits, reversabilities, charge balances, genes associations etc. Each organism has its own GSM, containing its specific metabolites, reactions and genes. 
</P><P>
The living organisms must abide by certain constraints or limitations, such as mass balance, limitation of nutrients in medium etc. The different types of constraints are explained in Price et al.'s paper from 2004<SUP><A href="#references"> 1 </A></SUP>. These constraints can be modeled in GSM by setting balanced equations and limiting reaction rates (fluxes). 
</P><P>
Setting different constraints on the fluxes allow us to define different cultivation environments. We can set some fluxes to certain levels to simulate a specific environment, or gene manipulation, allowing us to simulate the Wild-Type’s (WT) and/or Mutant’s growth behaviour in that specific environment.
</P><P>
We have algorithms from literatures which help us to perform these simulations. The algorithms are implemented in Cobrapy extension Cameo and Matlab’ Cobra toolbox, some functions are modified by our team to better suit our needs. The algorithms include FBA (flux balance analysis) based PHPP(PHenotype Phase Plane), and FSEOF (Flux scanning based on enforced objective flux). In the next sections, we will tell our story on how we used these algorithms to achieve our goals of exploring <I>Y.lipolytica</I> growth and predicting gene manipulations to achieve better beta-carotene yield.
</P><H3 class="h3">Using FBA to Calculate Steady-State Flux Distribution</H3><P>
Most GSM problems are solved using the flux balance analysis (FBA).  It formulates a constraint based optimization problem from the constraints we defined for the GSM. If solved, FBA reveals the flux distribution within the metabolic network. FBA assumes the network is in steady-state, which means the concentration of metabolites are not changing and in balance. The other constraints are lower and upper bounds on feasible fluxes of various reactions. FBA is the groundstone for all the algorithms we used to conduct the GSM.
</P><H3 class="h3">Genome-Scale Modeling</H3><P>
					The following explains the mathematical basis for GSM. In short, a metabolic network can be modeled as linear equations and optimized by solving the corresponding linear program. The output is the set of reaction rates for all reactions in the network along with the set of shadow prices associated with the metabolites. Both play important roles in the subsequent phenotype phase plane analysis.
				</P><H4 class="h4">A Simple Metabolic Network</H4><P>
					Consider a very simple metabolic network with two metabolites, \(A\) and \(B\), where \(A\) flows into the cell with rate \(r_1\), is converted into \(B\) with rate \(r_2\), which is then excreted from the cell with rate \(r_3\):</P>
					$$
					    \begin{split}
					        &amp; \xrightarrow{r_1} A \\
					        A &amp; \xrightarrow{r_2} B \\
					        B &amp; \xrightarrow{r_3} \\
					    \end{split}
					$$
	 <P>Under a steady-state assumption, i.e. where all metabolite levels are constant and no metabolite can be accumulated in the system, the formation and degradation rates for each metabolite must cancel each other. For the network above, this is equivalent to the following set of constraints:</P>
					$$
					    \begin{split}
					        A:&amp; \quad r_1 - r_2 = 0 \\
					        B:&amp; \quad r_2 - r_3 = 0
					    \end{split}
					$$

					<H4 class="h4">The Linear Program</H4><P>
							Now, consider the full genome-scale model of <I>Yarrowia lipolytica</I> with \(M\) = 1683 metabolites and \(R\) = 1985 reactions.
						</P><P>
							Let \(r = (r_1, ... , r_R)\) be the vector of the \(R\) reaction rates, and similarly let \(m = (m_1, ... ,m_M)\) be the vector of the \(M\) metabolites. The reaction of which the rate is to be maximized, for example the biomass function, is then denoted the <I>objective function</I> and can be expressed as, 
							
						$$
						    f(r) = r_{\text{obj}}
						$$	
						
							Now, for \(m_i\), the <B>equality constraint</B> \(c_i\) can be expressed as a sum of all \(r_j\) weighted by the stoichiometric coefficients \(a_{ij}\), where \(a_{ij}\) = 0 for reactions that do not affect the concentration of the considered metabolite. Thus,
								
						$$
						    c_i(r) = \sum\limits_{j=1}^R a_{ij}r_j
						$$
						
							Which is just a generalization of the constraints of the two-metabolite network presented in the previous section. In addition to the steady-state constraints on the metabolites, for each reaction rate, \(r_j\), there is a lower and an upper bound (\(l_j\) and \(u_j\)). These <B>inequality constraints</B> can be expressed as,
					
					$$
					    \begin{split}
					        c_{j,l} &amp;= r_j \geq l_j  \iff r_j - l_j \geq 0 \\
					        c_{j,u} &amp;= r_j \leq u_j \iff -r_j + u_j \geq 0
					    \end{split}
					$$
					
					Having the objective function and the constraints well-stated, the <B>linear program</B><SUP><A href="#references">6</A></SUP> can be formulated in standard form:
					
					$$
					    \begin{split}
					        \max \quad &amp; f(x) \\
					        s.t. \quad c_i &amp;= 0 \quad \forall i \\
					        c_j &amp;\geq 0 \quad \forall j
					    \end{split}
					$$
					
					Solving this program is equal to optimizing the genome-scale model. In order to accomplish that, it is necessary to define the so-called Lagrangian Function.

					</P><H4 class="h4">Solution to FBA and Shadow Prices</H4><P>
						For each constraint \(c_i\) on the metabolite \(m_i\), let \(\lambda_i\) be the associated Lagrange multiplier<SUP><A href="#references">7</A></SUP> or <B>shadow price</B>, let \(\lambda = (\lambda_i,...,\lambda_M)\) and let \(c = (c_1,...,c_M, c_{M+1}...,c_{M+R})\). Now, define the Lagrangian function:
						
						$$
						    \mathcal{L}(\lambda,r) = f(r) + \lambda^Tc(r)
						$$
						
						The solution to the linear program is then found by maximizing the gradient of the Lagrangian, \(\bigtriangledown \mathcal{L}(\lambda,c,r)\) , while satisfying all constraints, i.e. by solving the system,
						
						$$
						    \begin{pmatrix}
						        \bigtriangledown \mathcal{L}(\lambda,r) \\
						        c(r)
						    \end{pmatrix}
						    =
						    \begin{pmatrix}
						        0 \\ 0
						    \end{pmatrix}
						$$
						
						Then, the output will consist of the set \((f(r),r,\lambda)\), i.e. the reaction rates (including the one corresponding to the objective function) and the shadow prices. In our case, the linear program was solved in COBRA, which uses a simplex algorithm by default.<SUP><A href="#references">8</A></SUP> The objective function and constraints were based on the most recent SBML model of <I>Y. lipolytica</I>.<SUP><A href="#references">5</A></SUP> REFERENCE Further mathematical details are omitted.
					</P><H3 class="h3">Using Phenotype Phase Planes to Explore Growth and Production in Different Environments
</H3><P>
                    The Phenotype Phase Plane is a great way to explore target flux (e.g. growth rate or product secretion rate), as a function of two fluxes, in our case, carbon source uptake rate and oxygen uptake rate. This allows us to predict how <I>Y. lipolytica</I> grows using different Carbon(C) sources and different availability of the C source and oxygen. This kind of dual-variable (C and O<SUB>2</SUB>) flux investigation  is called Phenotype Phase Plane(PHPP) analysis, it predicts the growth rate (or product fluxes, depending on your goal) in different cultivation environments <SUP><A href="#references">2</A></SUP>. We can then find clues on how environment affects fluxes, and identify what conditions are favoured by <I>Y. lipolytica</I>. If we change the substrates in the medium to wastes, we can see how much growth behaviour changes. 

                </P><P>
                    There are multiple feasible solutions to a FBA problem, Not all of these solution have a biological counterpart. However, there are regions in the solution space over which the rate of shadow prices are constant. These regions or phases have been shown to resemble real phenotypes <SUP><A href="#references"> 2 </A></SUP> . For some uptake rate combinations, we have infeasible phenotype, this means this combination of uptake rates cannot satisfy the constraints, this maybe due to inability to satisfy ATP maintenance energy. 
                </P><P>
We produced phenotype phase planes for <I>Y. lipolytica</I> using different carbon sources. This allow us to see what substrates paired with what oxygen level makes better growth. </P><H3 class="h3">Adding Beta-Carotene Pathway into the GSM</H3><P> To model the insertion of the beta-carotene pathway, we introduced 2 more genes - crtYB and crtI, and they catalyze 4 new reactions. These reactions are introduced into the GSM as below: <SUP><A href="#references"> 3 </A></SUP></P><P><UL><LI> phytoeneSynthase (crtYB): 2.0 geranylgeranyl diphosphate &lt; --&gt; 2.0 diphosphate + 2.0 H+ + phytoene </LI><LI>caroteneDesaturaseStep1 (crtI):  phytoene &lt;--&gt; 6.0 H+ + neurosporene </LI><LI>caroteneDesturaseStep2 (crtI): neurosporene &lt;--&gt; lycopene + 2.0 H+ </LI><LI>lycopeneCyslase (crtYB): lycopene --&gt; beta-carotene + 2.0 H+ </LI></UL></P><P> To be able to optimize beta-carotene product flux, we must define a sink, or demand reaction in the GSM, this is done by creating a boundary exchange reaction where beta-carotene is, in silico, secreted into extracellular space, this reaction is defined as:</P><UL><LI> DM_bcarotene: beta-carotene →  </LI></UL><P> DM_bcarotene is then set consequently as objective function when optimizing for production flux. </P><H3 class="h3">Using FSEOF to Identify Gene Amplification and Inhibition Targets to Optimize Yield
</H3><P>
                    Any organism will, by its evolutionary nature, maximize growth rate. As bioengineers however, we want a maximum of desired products while keeping a reasonable growth rate. Therefore we want an organism that is optimized for both growth rate and product yield. We do this by manipulating its genes, forcing product yield and growth rate to go hand-in-hand. In other words, we want an engineered strain that, in order to maximize its growth rate, it <I>must</I> also produce the desired product. This is called coupling of product yield to growth rate. In our study, we couple beta-carotene yield to growth rate, and find out what genes should be amplified, inhibited, or knocked out, to achieve this coupling. This can be done by manipulating the genes, changing the flux bounds of certain reactions, to force the overall flux distribution towards the optimal solution.
      </P><P>
                    We can also use constraints to reflect the effect of gene manipulation. As genes are defined and associated to reactions and metabolites in the GSM, we can simulate gene amplification, inhibition, and KOs, by giving the flux associated with that gene a value larger, smaller, or zero value, respectively. </P><P>
We used FSEOF to identify gene amplification targets.  
</P><P>
FSEOF is a technique to identify what fluxes increase, when beta-carotene flux is enforced to be a certain value. This is done, by incrementally increasing the beta-carotene production flux from zero towards its theoretical maximum, then under the constraints, optimize growth rate. As we increase the enforced production flux, and solve for the other fluxes, some of these fluxes will increase, while other will decrease. The assumption is, if we amplify the genes associated to the reactions whose fluxes increase, we can force the product and growth rate to couple. The FSEOF is introduced in Choi et al.'s paper from 2010<SUP><A href="#references"> 4 </A></SUP>. 
</P></DIV><DIV><H2 class="h2">Results and Discussion</H2><H3 class="h3"> Phenotype Phase Plane </H3><H4 class="h4"> Optimizing growth Rate on Different C sources </H4><P>
               This section’s phenotype phase planes (PHPPs) are produced by setting the growth rate as the objective function.  You can click on the individual graphs to enlarge them. We will compare the PHPPs for four carbon substrates, glucose, fructose, glycerol and oleic acid. The PHPPs are arranged so each column is a C substrate, and each row is a feature studied. The features, from top row to bottom, are growth rate, shadow prices regards to C substrate and shadow prices with regards to oxygen.</P><P><STRONG>Growth Rates</STRONG></P><DIV class="row"><DIV class="col-md-3"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 1 </STRONG>Growth rate on Glucose</FIGCAPTION></FIGURE></DIV><DIV class="col-md-3"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 2 </STRONG>Growth rate on Fructose</FIGCAPTION></FIGURE></DIV><DIV class="col-md-3"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 3 </STRONG>Growth rate on Glycerol</FIGCAPTION></FIGURE></DIV><DIV class="col-md-3"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 4 </STRONG>Growth rate on Oleic acid</FIGCAPTION></FIGURE></DIV><DIV id="glc_biomass_grModal" class="modal"><SPAN class="close glc_biomass_gr">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV><DIV id="fruc_biomass_grModal" class="modal"><SPAN class="close fruc_biomass_gr">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV><DIV id="glyc_biomass_grModal" class="modal"><SPAN class="close glyc_biomass_gr">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV><DIV id="ole_biomass_grModal" class="modal"><SPAN class="close ole_biomass_gr">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV></DIV><P><STRONG> Shadow Prices Regards to C Substrate</STRONG></P><DIV class="row"><DIV class="col-md-3"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 5 </STRONG>shadow price on Glucose</FIGCAPTION></FIGURE></DIV><DIV class="col-md-3"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 6 </STRONG>shadow price on Fructose</FIGCAPTION></FIGURE></DIV><DIV class="col-md-3"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 7 </STRONG>shadow price on Glycerol</FIGCAPTION></FIGURE></DIV><DIV class="col-md-3"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 8 </STRONG>shadow price on Oleic acid</FIGCAPTION></FIGURE></DIV><DIV id="sp_glc_biomassModal" class="modal"><SPAN class="close sp_glc_biomass">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV><DIV id="sp_fruc_biomassModal" class="modal"><SPAN class="close sp_fruc_biomass">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV><DIV id="sp_glyc_biomassModal" class="modal"><SPAN class="close sp_glyc_biomass">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV><DIV id="sp_ole_biomassModal" class="modal"><SPAN class="close sp_ole_biomass">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV></DIV><P><STRONG> Shadow Prices Regards to Oxygen</STRONG></P><DIV class="row"><DIV class="col-md-3"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 9 </STRONG>shadow price on O<SUB>2</SUB> - Glucose</FIGCAPTION></FIGURE></DIV><DIV class="col-md-3"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 10 </STRONG>shadow price on O<SUB>2</SUB> - Fructose</FIGCAPTION></FIGURE></DIV><DIV class="col-md-3"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 11 </STRONG>shadow price on O<SUB>2</SUB> - Glycerol</FIGCAPTION></FIGURE></DIV><DIV class="col-md-3"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 12 </STRONG>shadow price on O<SUB>2</SUB> - Oleic acid</FIGCAPTION></FIGURE></DIV><DIV id="sp_o2_glc_biomassModal" class="modal"><SPAN class="close sp_o2_glc_biomass">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV><DIV id="sp_o2_fruc_biomassModal" class="modal"><SPAN class="close sp_o2_fruc_biomass">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV><DIV id="sp_o2_glyc_biomassModal" class="modal"><SPAN class="close sp_o2_glyc_biomass">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV><DIV id="sp_o2_ole_biomassModal" class="modal"><SPAN class="close sp_o2_ole_biomass">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV></DIV><P>
The PHPP is produced for the following carbon sources: glucose, glycerol, fructose and oleic acid. Growth of <I>Y. lipolytica</I> is feasible on all. The results support the substrates screening experiments. 
</P><P>
From PHPP investigation, we propose the “line of optimal”, the best combination of C source and O<SUB>2</SUB> uptake rate to support optimal growth for each substrate. This line is the brightest line through the plot, it represents the optimal oxygen  uptake rate for the given substrate uptake rate, and vice versa.
</P><P>
It can be seen that <I>Y. lipolytica</I> grows best on simple sugars, like glucose and fructose, (Figure 1 and 2), indeed, the PHPPs for glucose and fructose are almost the same. <I>Y. lipolytica</I> can grow almost.anaerobically on simple sugars. Sugar substrates provide a stable PHPP, where there is a very large phase area, resisting substrate induced phenotypic change (Figure 5 and 6). We see large area on PHPP where O<SUB>2</SUB> and sugar uptake rates affect the growth rate almost equally. A relatively small area where sugar level is low shows an decreased growth rate with increased O<SUB>2</SUB> uptake. So if we want to keep the growth rate high, then we must not cross the line of optimality, beyond which we falls into the small region where O<SUB>2</SUB> uptake rate is must be constrained by sugar uptake rate to keep growth high.
</P><P>
Glycerol shows higher requirement for O<SUB>2</SUB> and glycerol balance, and lessened ability to grow anaerobically (Figure 3). We see a narrower area where growth rate is affected equally by O<SUB>2</SUB> and glycerol. We have more phases than simple sugar PHPP, showing phenotypic changes are more easily induced by substrate changes, the phases near the axes shows if we have either glycerol or O<SUB>2</SUB> to be too quickly uptaken, we lose growth rate (Figure7 and 11). Balance of glycerol and O<SUB>2</SUB> is very important for optimal growth. Indeed, the line of optimality has a higher gradient than that for simple sugar. 
</P><P>
Oleic acid shows very demanding media condition, where oleic acid uptake rate must be low, and O<SUB>2</SUB> uptake rate must be high (Figure 4). We see a large infeasible region, and maximum growth rate in the total simulated conditions are about a half of the other C sources. However, this does means that for the PHPP regions where Carbon source is sparse, Oleic acid cna deliver a lot more growth than the other C sources, given we also have high O<SUB>2</SUB> uptake rate. This property is promising and fits our vision of Yeastilizaiton, turning waste to value, conserving carbon sources (Figures 8 and 12).
            </P><H4 class="h4">Theoretical Maximum Beta-Carotene Flux</H4><P>
If we optimize beta-carotene solely, we will see no growth rate, this is infeasible in the lab, but this theoretical maximum production flux can help us decide if an organism is worth the effort to engineer it, if the theoretical maximum is too low, then we would not attempt using <I>Y. lipolytica</I> to produce beta-carotene at all. Luckily, it is not the case. <I>Y. lipolytica</I> shows excellent beta-carotene yield capacity.  
These PHPPs are produced by setting the beta-carotene sink flux as the objective function. </P><P><STRONG>Beta-carotene production flux</STRONG></P><DIV class="row"><DIV class="col-md-3"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 13 </STRONG>beta-carotene flux on Glucose</FIGCAPTION></FIGURE></DIV><DIV class="col-md-3"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 14 </STRONG>beta-carotene flux on Fructose</FIGCAPTION></FIGURE></DIV><DIV class="col-md-3"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 15 </STRONG>beta-carotene flux on Glycerol</FIGCAPTION></FIGURE></DIV><DIV class="col-md-3"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 16 </STRONG>beta-carotene on Oleic acid</FIGCAPTION></FIGURE></DIV><DIV id="b_caro_glcModal" class="modal"><SPAN class="close b_caro_glc">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV><DIV id="b_caro_frucModal" class="modal"><SPAN class="close b_caro_fruc">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV><DIV id="b_caro_glycModal" class="modal"><SPAN class="close b_caro_glyc">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV><DIV id="b_caro_oleModal" class="modal"><SPAN class="close b_caro_ole">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV></DIV><P><STRONG> Shadow prices regards to C substrate</STRONG></P><DIV class="row"><DIV class="col-md-3"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 17 </STRONG>shadow price on Glucose</FIGCAPTION></FIGURE></DIV><DIV class="col-md-3"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 18 </STRONG>shadow price on Fructose</FIGCAPTION></FIGURE></DIV><DIV class="col-md-3"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 19 </STRONG>shadow price on Glycerol</FIGCAPTION></FIGURE></DIV><DIV class="col-md-3"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 20 </STRONG>shadow price on Oleic acid</FIGCAPTION></FIGURE></DIV><DIV id="sp_glc_b_caroModal" class="modal"><SPAN class="close sp_glc_b_caro">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV><DIV id="sp_fruc_b_caroModal" class="modal"><SPAN class="close sp_fruc_b_caro">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV><DIV id="sp_glyc_b_caroModal" class="modal"><SPAN class="close sp_glyc_b_caro">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV><DIV id="sp_ole_b_caroModal" class="modal"><SPAN class="close sp_ole_b_caro">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV></DIV><P><STRONG> Shadow prices regards to Oxygen</STRONG></P><DIV class="row"><DIV class="col-md-3"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 21 </STRONG>shadow price on O<SUB>2</SUB> - Glucose</FIGCAPTION></FIGURE></DIV><DIV class="col-md-3"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 22 </STRONG>shadow price on O<SUB>2</SUB> - Fructose</FIGCAPTION></FIGURE></DIV><DIV class="col-md-3"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 23 </STRONG>shadow price on O<SUB>2</SUB> - Glycerol</FIGCAPTION></FIGURE></DIV><DIV class="col-md-3"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 24 </STRONG>shadow price on O<SUB>2</SUB> - Oleic acid</FIGCAPTION></FIGURE></DIV><DIV id="sp_o2_glc_b_caroModal" class="modal"><SPAN class="close sp_o2_glc_b_caro">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV><DIV id="sp_o2_fruc_b_caroModal" class="modal"><SPAN class="close sp_o2_fruc_b_caro">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV><DIV id="sp_o2_glyc_b_caroModal" class="modal"><SPAN class="close sp_o2_glyc_bi_caro">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV><DIV id="sp_o2_ole_b_caroModal" class="modal"><SPAN class="close sp_o2_ole_b_caro">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV></DIV><P>
Compared to growth rate optimized PHPPs, the general changes are these: smaller optimal regions (smaller bright yellow area in Figures 13-15); more phase planes (Figures 17, 18, 21, 22) or steeper gradient of lines of optimality, leading to more equally divided areas for phase planes of opposite gradients (Figures 19, 23). The net change from growth rate optimized to production optimized PHPPs show less stability, meaning the production flux is more easily influenced by substrate uptake rates, and we have stricter requirements for oxygen and C substrate uptake rates to match up to stay on the line of optimality. We can take advantage of these trends, to design optimal chemostats or fed-batch strategies by forcing the substrate and oxygen uptake(fed rate) to stay on the line of optimality. </P><P>
The only substrate whose beta-carotene flux optimized PHPPs and growth rate optimized PHPPs remain very similar, without showing the mentioned trends, is oleic acid (Figures 16, 20, 24). Oleic acid’s stability in PHPP topology, independent of these two objective functions, implies it is a substrate that can make control of optimality simpler for both growth and beta-carotene production.
</P><H3 class="h3"> Flux scanning based on enforced objective flux </H3><P> Using FSEOF, we simulated and found out what genes need to be amplified, using glycerol as sole C substrate. These genes are
</P><P><UL><LI>YALI0B16038g</LI><LI>YALI0E06193g</LI><LI>YALI0F05632g</LI><LI>YALI0E23859g or YALI0E31064g or YALI0D08382g or YALI0A15125g or YALI0A21307g</LI><LI>YALI0B08965g</LI><LI>YALI0A12045g</LI><LI>YALI0D17050g</LI><LI>YALI0E02288g</LI><LI>YALI0D25564g</LI><LI>YALI0B01826g</LI><LI>YALI0D06501g</LI><LI>YALI0F12199g</LI><LI>YALI0C01111g</LI><LI>YALI0F04015g</LI><LI>YALI0E05753g</LI><LI>YALI0E23991g or YALI0E27874g</LI><LI>YALI0A20108g</LI><LI>YALI0D09867g</LI><LI>YALI0B02852g</LI><LI>YALI0E25652g</LI><LI>YALI0E07073g</LI><LI>crtI</LI><LI>YALI0B22902g</LI><LI>crtYB</LI><LI>YALI0E14751g and YALI0D11110g</LI><LI>YALI0D17930g</LI><LI>YALI0B20020g or YALI0C06952g</LI><LI>YALI0F12639g</LI></UL></P><P> Amplifying anyone of these genes will potentially provide a significant rise in beta-carotene flux, these genes take part in the following reactions: </P><P><UL><LI>phosphomevalonate kinase</LI><LI>Demand beta_carotene</LI><LI>isopentenyl-diphosphate D-isomerase</LI><LI>lycopene cyclase</LI><LI>geranyltranstransferase</LI><LI>carotene desaturase step 1</LI><LI>CO<SUB>2</SUB> transport</LI><LI>dimethylallyltranstransferase</LI><LI>carotene desaturase step 2</LI><LI>mevalonate pyrophoshate decarboxylase</LI><LI>mevalonate kinase (UTP)</LI><LI>phytoene synthase</LI></UL></P><P> The next steps would be to bring the results to the wetlab, and validate their impacts. </P><P>The code for this project can be found <A href="https://static.igem.org/mediawiki/2016/0/0a/T--DTU-Denmark--gsm_dtu_biobuilders.zip">here</A>.</P></DIV><DIV><H2 class="h2">Attributions</H2><P>
               We would like to thank our supervisor Chris, and many other faculty members, researchers and PhD students for advice, trouble-shooting and tutorials. Every time we needed help, you are always there. A very special thank you goes out to Nikolaus for tirelessly tutoring us online and during Thursday's help desk sessions; Also to Julian for resolving  many puzzles of ours by fast emails; to Patrice, whose constant enthusiasm inspires us all, to you and Anders, who advised us and provided invaluable validation data. To Kristian, who helped James with GSM theories and Cameo setup; To Anne Sophie who gave us very helpful literature reviews; to Mikael who evaluated and supported our modeling strategy; to Vilhelm for introducing us to the support of BioBuilders alumni; and to the authors of our most influencing papers, Martin and Eduard, for detailedly explaining to us by emails about their work; and to all the helping people from DTU Biosustain's Slack forum, thank you very much. The modeling team would take this opportunity to also acknowledge the rest of the DTU BioBuilders, thank you for the many inspiring discussions, supporting words and actions. Thank you for making the iGEM experience much, much stronger than we could have ever modeled.
            </P></DIV><DIV id="ref_sec"><H2 class="h2">References</H2><OL><LI>Price, N. D., Reed, J. L., &amp; Palsson, B. O. (2004). Genome-scale models of microbial cells: Evaluating the consequences of constraints. Nature Reviews Microbiology, 2(11), 886–897. doi:10.1038/nrmicro1023</LI><LI>Edwards, J. S., Ramakrishna, R., &amp; Palsson, B. O. (2002). Characterizing the metabolic phenotype: A phenotype phase plane analysis. Biotechnology and Bioengineering, 77(1), 27–36. doi:10.1002/bit.10047</LI><LI>JHU iGEM (2011). vitamin A. Retrieved from https://2011.igem.org/Team:Johns_Hopkins/Project/VitA</LI><LI> Choi, H. S., Lee, S. Y., Kim, T. Y., &amp; Woo, H. M. (2010). In silico identification of gene amplification targets for improvement of lycopene production. Applied and environmental microbiology, 76(10), 3097-3105.</LI><LI>Kerkhoven, E. J., Pomraning, K. R., Baker, S. E., &amp; Nielsen, J. (2016). Regulation of amino-acid metabolism controls flux to lipid accumulation in Yarrowia lipolytica. npj Systems Biology and Applications, 2, 16005.</LI><LI>Murty, Katta G. &quot;Linear programming.&quot; (1983).</LI><LI>Mécanique Analytique sect. IV, 2 vols. Paris, 1811</LI><LI>Dantzig, George (May 1987). &quot;Origins of the simplex method&quot;</LI><LI>http://cameo.bio/</LI></OL></DIV></DIV><DIV class="col-md-3 col-sm-2 colRight" id="scrollspy"><UL class="nav" id="sidebar"><LI><A href="#overview">Overview</A></LI><LI><A href="#theory">Theory</A></LI><LI><A href="#results">Results and Discussion</A></LI><LI><A href="#attributions">Attributions</A></LI></UL></DIV></DIV></DIV></DIV><FOOTER id="footer"><DIV class="container"><DIV align="center" class="col-md-3"><UL><LI>FIND US AT:</LI></UL><UL><LI><U>DTU BIOBUILDERS</U></LI><LI>DENMARK</LI><LI>DTU - SØLTOFTS PLADS, BYGN. 221/002</LI><LI>2800 KGS. LYNGBY</LI><LI><U>E-mail:</U></LI><LI>dtu-biobuilders-2016@googlegroups.com</LI></UL></DIV><DIV align="center" class="col-md-6"><UL><LI>MAIN SPONSORS:</LI></UL></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>