<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_DTU-Denmark_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:DTU-Denmark/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>New HTML template for the wiki</TITLE></P><DIV id="headerWrapper"><DIV id="Topnav" class="container"><DIV id="logo"><BUTTON id="hamburger" class="mobile-menue">☰</BUTTON><BUTTON id="arrow" class="mobile-menue">⇦</BUTTON><BUTTON id="a-down" class="mobile-menue mobile-scrole">Jump to</BUTTON></DIV><DIV id="navigation"><UL id="nav-items"><LI><A href="https://2016.igem.org/Team:DTU-Denmark/project" class="project nav-item" id="project">Project</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/wetlab" class="wetlab nav-item" id="wetlab">Wetlab</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/compute" class="compute nav-item" id="compute">Compute</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/team" class="team nav-item" id="team">Team</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/outreach" class="outreach nav-item" id="outreach">Outreach</A></LI></UL></DIV></DIV><DIV id="sub-navigation-wrapper"><DIV id="sub-navigation" class="container"><UL id="project-sub"><LI><A href="https://2016.igem.org/Team:DTU-Denmark/description" class="sub-nav-item">Description</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Proof" class="sub-nav-item">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/achievements" class="sub-nav-item">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Notebook" class="sub-nav-item">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Safety" class="sub-nav-item">Safety</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Description" class="sub-nav-item">Parts</A></LI></UL><UL id="wetlab-sub"><LI><A href="https://2016.igem.org/Team:DTU-Denmark/substrate" class="sub-nav-item">Substrate</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/products" class="sub-nav-item">Products</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/molecular_toolbox" class="sub-nav-item">Molecular Toolbox</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/molecular_interlab" class="sub-nav-item">Interlab</A></LI></UL><UL id="compute-sub"><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Software" class="sub-nav-item">Codon Optimization</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Hardware" class="sub-nav-item">Microfermenter</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Model" class="sub-nav-item">Modeling</A></LI></UL><UL id="team-sub"><LI><A href="https://2016.igem.org/Team:DTU-Denmark/team_members" class="sub-nav-item">Team Members</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/team_building" class="sub-nav-item">Team Building</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Attributions" class="sub-nav-item">Attributions</A></LI></UL><UL id="outreach-sub"><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Collaborations" class="sub-nav-item">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/HP/Silver" class="sub-nav-item">Practices</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/HP/Gold" class="sub-nav-item">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Engagement" class="sub-nav-item">Biosensor</A></LI></UL></DIV></DIV></DIV><P><TITLE>Bootstrap Example</TITLE></P><DIV class="MYbody"><DIV class="masthead"><DIV class="container-fluid"><DIV class="row"><DIV class="col-md-12 thumbnail" style="background-image:url(https://static.igem.org/mediawiki/2016/5/56/T--DTU-Denmark--project-header.jpg)"><DIV class="caption"><DIV class="col-md-5 col-sm-5 col-xs-12 title"><H1>Notebook<P class="lead">In this section you can follow our progress in the different subgroups week for week. Some weeks are quite, while some weeks are very hectic - especially right up to the wiki freeze.</P></H1></DIV><DIV class="col-md-5 col-sm-5 hidden-xs intro"><BLOCKQUOTE class="blockquote-reverse"><P>&quot;Every great love starts with a great story...&quot;</P><SMALL>Nicholas Sparks,<CITE title="Source Title"> The Notebook</CITE></SMALL></BLOCKQUOTE></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container notebook"><DIV class="row"><DIV class="col-md-9 col-sm-10 colLeft"><DIV><H2 class="h2">June</H2><BLOCKQUOTE class="visible-xs"><P>&quot;Every great love starts with a great story...&quot;</P><SMALL>Nicholas Sparks,<CITE title="Source Title"> The Notebook</CITE></SMALL></BLOCKQUOTE><DIV class="panel panel-default"><A data-toggle="collapse" href="#collapse1" aria-expanded="false" aria-controls="collapse1"><DIV class=" panel-heading" role="tab" id="heading1"><H4 class="panel-title">
	                    Week 1 (May 30 - June 5) </H4></DIV></A><DIV id="collapse1" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading1"><DIV class="panel-body"><H3>Wetlab</H3><P><EM>Yarowia lipolytica</EM> PO1f Δ<EM>ku70</EM> was obtained from Cory M. Schwartz, cultivated and freeze stocked for future use. </P><H3 style="clear:both;">Compute</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Hardware</H4><P>Our Arduino starter kits arrived! Make an LED blink. That’s how it begins.</P></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" href="#collapse2" aria-expanded="false" aria-controls="collapse2"><DIV class=" panel-heading" role="tab" id="heading2"><H4 class="panel-title">
		                    Week 2 (June 6 - June 12) </H4></DIV></A><DIV id="collapse2" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading2"><DIV class="panel-body"><H3>Wetlab</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Molecular Toolbox</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>PEX10</EM> knockout</H5><P><EM>Y. lipolytica</EM> PO1f genome sequence was annotated and protospacer for targeting <EM>PEX10</EM> was designed and ordered from IDT.  Protospacer for Gibson Assembly with CRISPRyl plasmid (Addgene plasmid #70007) SCR1'- tRNAGly (<STRONG>bold</STRONG>), Protospacer (<U>underlined</U>), sgRNA (<EM>italic</EM>)</P><P>  5'-<STRONG>GGGTCGGCGCAGGTTGACGT</STRONG><U>GTACAAGGAGGAGCTGGAGA</U><EM>GTTTTAGAGCTAGAAATAGC</EM>-3' </P>
                                
		                                Oligos designed to amplify a 1kb region upstream and downstream <EM>PEX10</EM> and anneal together by fusion PCR were also ordered from IDT.</DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>URA3</EM> insertion</H5><P>The <EM>Y. lipolytica</EM> PO1f genome sequence was annotated and uploaded to Benchling for sgRNA design. sgRNAs targeting the <EM>SUC2</EM> gene were designed. Primers were designed that amplify the functional <EM>URA3</EM> gene including 1 kb upstream and downstream flanking regions.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>pSB1A8YL</H5><P>Ran a bunch of PCRs to amplify the pUC19 part of our plasmid, but it’s not working - nothing but smear. Tried to transform the pUC19 plasmid into <EM>Escherichia coli</EM>. </P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Substrates</H4><P>We did an initial experiment determining the full growth cycle of <EM>Y. lipolytica</EM> W29. This will be used to plan and time the following growth experiments.
		                            Waste glycerol from the industrial biodiesel producer DAKA is acquired for late screening.
		</P></DIV></DIV><H3 style="clear:both;">Compute</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Hardware</H4><P>We started building light sensors using photoresistors. Shortlisting ideas for our final project: 
		                            - A microtiter plate reader
		                            - Hack a printer to build a membrane homogenizer
		                            - Chemostat bioreactor
		</P></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" href="#collapse3" aria-expanded="false" aria-controls="collapse3"><DIV class=" panel-heading" role="tab" id="heading3"><H4 class="panel-title">
	                     Week 3 (June 13 - June 19) </H4></DIV></A><DIV id="collapse3" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading3"><DIV class="panel-body"><H3>Wetlab</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Molecular Toolbox</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>PEX10</EM> knockout</H5><P>Genomic DNA from <EM>Y. lipolytica</EM> PO1f Δ<EM>ku70</EM> and <EM>Y. lipolytica</EM> W29 was purified. <EM>PEX10</EM> flanking regions were successfully amplified from <EM>Y. lipolytica</EM> PO1f Δ<EM>ku70</EM>.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>URA3</EM> insertion</H5><P>Genomic DNA from <EM>Y. lipolytica</EM> PO1f Δ<EM>ku70</EM> and <EM>Y. lipolytica</EM> W29 was purified. PCR attempts to amplify <EM>URA3</EM> and flanks failed. sgRNAs targeting the <EM>SUC2</EM> gene were hybridized.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>pSB1A8YL</H5><P>Purified the plasmid from the transformants and use this as template for PCR, although it’s still not giving any bands.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Substrates</H4><P>We did initial growth experiments on minimal media with an array of different carbon sources. This experiment was discarded due to lack of repeats and wrong vitamin solution for minimal media. Waste from canola oil production by Grønninggaard is acquired. Molasses from Dansukker sugar production is acquired.</P></DIV></DIV><H3 style="clear:both;">Compute</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Hardware</H4><P>Exploring the Arduino IDE and all the electronic components we ordered. There is so much to learn.</P></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" href="#collapse4" aria-expanded="false" aria-controls="collapse4"><DIV class=" panel-heading" role="tab" id="heading4"><H4 class="panel-title">
		                    Week 4 (June 20 - June 26) </H4></DIV></A><DIV id="collapse4" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading4"><DIV class="panel-body"><H3>Wetlab</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Molecular Toolbox</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>URA3</EM> insertion</H5><P>PCR attempts to amplify <EM>URA3</EM> + flanks from <EM>Y. lipolytica</EM> W29 genomic DNA failed. New primers were ordered.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>pSB1A8YL</H5><P>We realized that the name of the primer had been mixed up! Now that the right primers are used, we get excellent bands on our gel… Guess you have to make the stupid mistakes in the beginning?
		                                The gBlock containing the other part of the plasmid also arrived. This also gives excellent bands on the gel when amplifying it by PCR.
		                                Ran the first USER and transformed <EM>E. coli</EM> cells. The transformants were left on the bench over the weekend</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Substrates</H4><P>We have data from the first successful growth experiment. Starch, Xylose, Arabinose, Maltose and Lactose are not suitable for <EM>Y. lipolytica</EM> fermentation. This will be repeated next week to make sure.
		                            Waste glycerol from the industrial biodiesel producer Perstop is acquired.
		</P></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" href="#collapse5" aria-expanded="false" aria-controls="collapse5"><DIV class=" panel-heading" role="tab" id="heading5"><H4 class="panel-title">
		                    Week 5 (June 27 - July 3) </H4></DIV></A><DIV id="collapse5" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading5"><DIV class="panel-body"><H3>Wetlab</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Molecular Toolbox</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>PEX10</EM> knockout</H5><P>Received CRISPRyl plasmid (Addgene plasmid #70007). The procedure from Addgene was followed. </P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>URA3</EM> insertion</H5><P>Successful amplification of <EM>URA3</EM> + flanks from <EM>Y. lipolytica</EM> W29 genomic DNA. Because of low quality of the genomic DNA, the initial PCR product was taken for further amplification. 
		                                Received CRISPRyl plasmid (Addgene plasmid #70007). The procedure from Addgene was followed. 
		</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>pSB1A8YL</H5><P>YES, colonies! Colonies were picked and used for colony PCR, but it was not successful. We’ll just have to crank on!
		                                - Colonies from the same plates were re streaked and plasmids were purified from the resulting colonies.
		                                - Restriction analysis yielded weird bands.
		                                - This week passes restreaking colonies to yield pure colonies and trying to find the correct transformants through by purifying the plasmids and subjecting it to analytical digestion. So far no luck!
		</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Substrates</H4><P>Repeated positive results with growth on glucose, glycerol, fructose, sucrose and oil. <EM>Y. lipolytica</EM> should not be able to grow on sucrose. The experiments on starch, xylose, arabinose, Maltose and Lactose are still negative for <EM>Y. lipolytica</EM>.
		                            Waste glycerol from the industrial biodiesel producer Emmelev is acquired.
		</P></DIV></DIV><H3 style="clear:both;">Compute</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Genome Scale Modeling</H4><P>Planning of Genome-scale modelling strategies began, decided to attempt media optimization using phenotype phase plane, team starts to research and learn FBA for GSM. </P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Software</H4><P>Initiation of task by designing the workflow needed to achieve the final purpose of the software. Tasks agreed upon discussion : script in python , number and format proxy of the input files needed , restriction site implementation , development GUI, gui library for python (tkinter) </P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Hardware</H4><P>Still playing.</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV><H2 class="h2">July</H2><DIV class="panel panel-default"><A data-toggle="collapse" href="#collapse6" aria-expanded="false" aria-controls="collapse6"><DIV class=" panel-heading" role="tab" id="heading6"><H4 class="panel-title">
		                    Week 6 (July 4 - July 10) </H4></DIV></A><DIV id="collapse6" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading6"><DIV class="panel-body"><H3>Wetlab</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Molecular Toolbox</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>PEX10</EM> knockout</H5><P>Purification of pCRISPRyl plasmid from O/N cultures.  Successful digestion of CRISPRyl plasmid with restriction enzymes AatII and NdeI to verify the plasmid. 
		                                Gibson Assembly of digested CRISPRyl plasmid and protospacers. 
		</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>URA3</EM> insertion</H5><P>Purification of pCRISPRyl.  
		                                Successful restriction analysis of pCRISPRyl with AatII and NdeI to verify the plasmid.
		                                Gibson assembly of pCRISPRyl and the hybridized sgRNAs targeting the <EM>SUC2</EM> gene. </P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>pSB1A8YL</H5><P>YES! Finally a construct that seems to have the correct length! Both the analytical digestion and PCRs seems to confirm our construct.
		                                The construct was also sent for sequencing.
		                                We spent some time trying to figure out how to test the plasmid. We ended up retrieving the BBa_K592009, BBa_K592010, BBa_E1010 and BBa_J23110 parts from the distribution kit, and pair them. The idea is that if we are able to make a construct in our backbone, we should see a visual output.
		                                Unfortunately we were not able to retrieve the BBa_K592010 from the distribution kit, and it was decided to leave this for now.
		</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Substrates</H4><P>Contamination of experiments was determined with microscopy. Might come from the minimal media being contaminated.
		                            First test on complex glycerol based media. <EM>Y. lipolytica</EM> seems to grow better than <EM>Saccharomyces cerevisiae</EM>.
		                            The growth form on different C-sources is analysed with microscopi. There seem to be different amounts of planktonic and filamentous growth depending on energy source.
		</P></DIV></DIV><H3 style="clear:both;">Compute</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Software</H4><P>Implementation of algorithm already started. The script is being written in Python3 with the intention to be easily modifiable so no external packages are needed although Anaconda is being used. </P></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" href="#collapse7" aria-expanded="false" aria-controls="collapse7"><DIV class=" panel-heading" role="tab" id="heading1"><H4 class="panel-title">
		                    Week 7 (July 11 - July 17) </H4></DIV></A><DIV id="collapse7" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading7"><DIV class="panel-body"><H3>Wetlab</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Molecular Toolbox</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>PEX10</EM> knockout</H5><P>The assembled CRISPRyl plasmid with <EM>PEX10</EM> protospacers was transformed in competent <EM>E. coli</EM> DH5α cells.
		                                Purification of assembled CRISPRyl plasmid with Pex10 protospacers.
		                                The assembled CRISPRyl plasmid with <EM>PEX10</EM> protospacers was digested with AvrII to test if the <EM>PEX10</EM> protospacer was inserted. The digestion was successful and showed no cutting by AvrII indicating correct insertion of <EM>PEX10</EM> protospacers.
		                                First attempt of transformation with CRISPRyl plasmid with <EM>PEX10</EM> protospacers in <EM>Yarowia lipolytica</EM> PO1f Δ<EM>ku70</EM>. The transformation was unsuccessful. There was growth all over the plates. </P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>URA3</EM> insertion</H5><P>Confirmation of successful Gibson assembly through restriction analysis. 
		                                First (unsuccessful) transformation of pCRISPRyl and pCRISPRyl+sgRNAs into <EM>Yarowia lipolytica</EM> PO1f Δ<EM>ku70</EM>.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Products</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Beta-Carotene</H5><P>BioBricks BBa_K152005 (<EM>crtE</EM>,<EM>crtI</EM>,<EM>crtYB</EM>,<EM>gfp</EM>) and BBa_K530002 (<EM>crtI</EM>) were received from the registry. E. coli with BBa_K152005 could only grow without Ampicilin, so we suspect a problem with the AmpR backbone. We decided not to use this biobrick and instead only work with BBa_K530002, BBa_K530000 and BBa_K530001.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Proinsulin</H5><P>Designing and ordering synthetic gBlock from IDT encoding TEF1, proinsulin and GFP.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Substrates</H4><P>Test shows no growth on samples of wastewater from areas contaminated with leftovers of organic molecules from fossil oil.
		                            Experiments with glycerol byproduct from Emmelev biodiesel production showed to be contaminated.
		                            We investigated where the contaminations could come from and a contaminated pipet seemed to be the source. 
		</P></DIV></DIV><H3 style="clear:both;">Compute</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Hardware</H4><P>The library for our LCD Display is commented in Chinese. Shoutout to James for translating. Chris, our supervisor, brought us in contact with Martin and Erik, two professors from DTU Nano and Elektro. Our university is running a new project course in January where 1st semester students ferment their own cultures and build devices to monitor growth. They built a measuring chamber and a simple circuit, reading the voltage over the photodiode with a multimeter. Isn’t there a way to do all this using an Arduino? We set a meeting for next week.  Reading about digital to analog conversion(DAC) and the reverse(ADC). Dimming LEDs in the office.</P></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" href="#collapse8" aria-expanded="false" aria-controls="collapse8"><DIV class=" panel-heading" role="tab" id="heading8"><H4 class="panel-title">
		                    Week 8 (July 18 - July 24) </H4></DIV></A><DIV id="collapse8" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading8"><DIV class="panel-body"><H3>Wetlab</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Molecular Toolbox</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>PEX10</EM> knockout</H5><P>Transformation in <EM>Y. lipolytica</EM> was repeated using the same transformation protocol from last week. However, this time less cells were used for the transformation. This did not work.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>URA3</EM> insertion</H5><P>Gibson assembly of sgRNAs targeting <EM>SUC2</EM> into pCRISPRyl.
		                                More (unsuccessful) transformations of pCRISPRyl and pCRISPRyl+sgRNAs into <EM>Yarowia lipolytica</EM> PO1f Δ<EM>ku70</EM>.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>pSB1A8YL</H5><P>The promoter was paired with the chromoproteins using 3A assembly. Unfortunately, no color was observed even though PCR and analytical digestion showed that the length of the construct are correct.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Promoters</H5><P>TEF1 was amplified from gBlock by PCR.
		                                SCR1’-tRNA promoter was amplified from the pCRISPRyl plasmid by PCR with primers introducing a base substitution to remove illegal restriction site.
		</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Products</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Beta-Carotene</H5><P>Biobricks BBa_K530000 (<EM>crtYB</EM>) and BBa_K530001 (<EM>crtE</EM>) from the distribution kit was successfully obtained.
		                                We designed Gibson primers including 5'-CACA-3' upstream of each start codon for assembly of all three genes.
		</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Proinsulin</H5><P>3A assembly of the proinsulin gBlock and pSB1C3. Electrophoresis did not confirm presence of desire plasmid.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Substrates</H4><P><EM>S. cerevisiae</EM> does not grow as well as <EM>Y. lipolytica</EM> on glycerol based waste, but has an advantage on sucrose based ones. </P></DIV></DIV><H3 style="clear:both;">Compute</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Genome Scale Modeling</H4><P>Ben has kickstarted the modeling, by introducing the phenotype phase plane concept to the team, which is an extension to flux-balance analysis. The hope is that it will be possible to find optimal in-flow of various substrates in order to maximize product, in this case beta-carotene. Downloaded first <EM>Y. lipolytica</EM> model “MODEL1510060001”. Tutorials are studied in Matlab.
		</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Software</H4><P>Script writing is finished. Testing of functionality was initiated in order to find the most “sustainable” solution in terms of resources and the optimization step (reverse-translation) from the desired sequences to protein sequences. Approach decided for main body of script is building and checking the sequences “on the fly” while using as initial files the y_lip.txt (gcn), desired.fsa(protein sequences), ressites.txt(restriction sites).</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Hardware</H4><P>Successful tests at DTU Nano. We can measure growth of S. cerevisiae. Erik found a great photodetector that includes an amplifying circuit.</P></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" href="#collapse9" aria-expanded="false" aria-controls="collapse9"><DIV class=" panel-heading" role="tab" id="heading9"><H4 class="panel-title">
		                    Week 9 (July 25 - July 31) </H4></DIV></A><DIV id="collapse9" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading9"><DIV class="panel-body"><H3>Wetlab</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Molecular Toolbox</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>PEX10</EM> knockout</H5><P>Transformation in <EM>Y. lipolytica</EM> was repeated using transformation protocol from Cory M. Schwartz . The transformation was unsuccessful.  </P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>URA3</EM> insertion</H5><P>More (unsuccessful) transformations of pCRISPRyl and pCRISPRyl+sgRNAs into <EM>Yarowia lipolytica</EM> PO1f Δ<EM>ku70</EM>.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>pSB1A8YL</H5><P>Still trying to figure out why we do not see any color output... 
		                                K592010 was retrieved from the distribution kit.
		</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Promoters</H5><P>3A assembly with the TEF1 fragment and pSB1C3 backbone. Results were inconclusive, so they were repeated in week 10.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Products</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Beta-Carotene</H5><P>PCR with Gibson primers on all three biobricks and pSB1C3 backbone. Only BBa_K53000 (<EM>crtYB</EM>) and backbone was successful.
		                                <EM>crtI</EM> and <EM>crtE</EM> was after several attemps successfully amplified with prefix10 and suffix10 primers, which then worked at template for PCR with Gibson primers.
		</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Proinsulin</H5><P> 3A assembly was repeated. Once again the desired plasmid was not obtained (confirmed with electrophoresis and restriction analysis).
		                                Gibson assembly of proinsulin gBlock and backbone pSB1C3. Transformation gave numerous colonies that were used as a template for colony PCR. One colony was expected to carry a desired construct, however further investigation disproved it. 
		</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Substrates</H4><P>The <EM>Y. lipolytica</EM> growth on sucrose was found to be a false positive. The autoclavaton of sucrose breaks the molecule turning it into glucose and fructose. <EM>S. cerevisiae</EM> grow faster due to the ability to metabolize sucrose, while <EM>Y. lipolytica</EM> can only utilize the glucose and fructose.</P></DIV></DIV><H3 style="clear:both;">Compute</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Hardware</H4><P>Running more fermentations together with Martin and Erik. They changed their design while we got inspired for our final project. Couldn’t we turn this into a cheap alternative for the Hamilton robot? Reading on CAD drawings, 3D printing and laser cutting</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV><H2 class="h2">August</H2><DIV class="panel panel-default"><A data-toggle="collapse" href="#collapse10" aria-expanded="false" aria-controls="collapse10"><DIV class=" panel-heading" role="tab" id="heading10"><H4 class="panel-title">
		                    Week 10 ( August 1 - August 7) </H4></DIV></A><DIV id="collapse10" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading10"><DIV class="panel-body"><H3>Wetlab</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Molecular Toolbox</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>PEX10</EM> knockout</H5><P>Transformation in <EM>Y. lipolytica</EM> was repeated using the same transformation protocol from last week. Transformations were performed with the CRISPRyl plasmid with inserted <EM>PEX10</EM> protospacers and with and without <EM>PEX10</EM> flanking regions, respectively. The transformations were plated on selective media after resuspension in water. The transformations were unsuccessful. </P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>URA3</EM> insertion</H5><P>Unsuccessful transformations of pCRISPRyl and pCRISPRyl+sgRNAs into <EM>Yarowia lipolytica</EM> PO1f Δ<EM>ku70</EM>. 
		                                Unsuccessful co-transformation of pCRISPRyl+sgRNAs and the linear <EM>URA3</EM> +flanks fragment into <EM>Yarowia lipolytica</EM> PO1f Δ<EM>ku70</EM> (protocol link). The concentration of the HR repair fragment should be increased.
		                                Design of new sgRNAs targeting the <EM>SUC2</EM> gene in case the already assembled ones are not working as expected in <EM>Y.lipolytica</EM>.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>pSB1A8YL</H5><P>After having talked with Joel from SDU, we realized that J23110 does not give any output, as it does not contain a RBS. He recommended that we use the BBa_K880005 instead.
		                                The BBa_K880005 promoter was paired with BBa_K592009 and BBa_E1010 BioBricks in our backbone using 3A assmbly.
		                                This time we had colored colonies!
		                                Just to be sure we also confirmed the identity of the construct using PCR and analytical digestion, and they were correct!
		                                The plasmid was also sent to Uppsala, who agreed to test the plasmid for us.
		</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Promoters</H5><P>New 3A assembly of TEF1 and SCR1’-tRNA and the pSB1C3 backbone. </P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Products</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Beta-Carotene</H5><P>Gibson assembly of all three <EM>crt</EM> genes and pSB1C3 backbone. Transformation resulted in colonies, but turned out not to with the correct constructs (only self-ligated backbone).
		                                Gibson assembly with pSB1A3 was attempted to avoid false positives.
		</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Proinsulin</H5><P>3A assembly for the proinsulin gBlock and pSB1A8yl, pSB1C3. No success. </P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Substrates</H4><P><EM>Y. lipolytica</EM> is shown to utilize canola oil production waste a better than <EM>S. cerevisiae</EM>. There is a lot of noise in the OD measurements.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Interlab</H4><P>This week, we started the Interlab Study. We successfully transformed the three devices: Test Device 1, 2, and 3, consisting of the promoters J23101, J23106 and J23117, sharing the RBS (B0034), the terminator (B0015) and the GFP (E0040), as well as the negative control consisting of the pTetR promoter (RR040) with no sequence downstream, and the positive control consisting of the constitutively expressed GFP device (I20270). The plates were put in the fridge for later use. </P></DIV></DIV><H3 style="clear:both;">Compute</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Software</H4><P>Implementation of GUI with tkinter library initiated. </P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Hardware</H4><P>Rough sketch of the requirements finished. Mathias joined the team to program the Arduino. Surprisingly, he managed to fry the first board half an hour later. So glad to have him!
		                            Testing whether air bubbles could be used for stirring and 3D printing a simple Arduino controlled valve to control the airflow.
		</P></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" href="#collapse11" aria-expanded="false" aria-controls="collapse11"><DIV class=" panel-heading" role="tab" id="heading1"><H4 class="panel-title">
		                    Week 11 (August 8 - August 14) </H4></DIV></A><DIV id="collapse11" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading11"><DIV class="panel-body"><H3>Wetlab</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Molecular Toolbox</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>PEX10</EM> knockout</H5><P>Transformations in <EM>Y. lipolytica</EM> were repeated using the same protocol from last week. However, this time the transformations were inoculated in liquid selective media for two days and then plated on YPD media after outgrowth. </P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>URA3</EM> insertion</H5><P>Successful transformations of pCRISPRyl and pCRISPRyl+sgRNAs into <EM>Yarowia lipolytica</EM> PO1f Δ<EM>ku70</EM>. Verification that the constructed pCRISPRyl+sgRNA is working as expected.
		                                Unsuccessful co-transformation of pCRISPRyl+sgRNAs and the linear <EM>URA3</EM>+flanks fragment into <EM>Yarowia lipolytica</EM> PO1f Δ<EM>ku70</EM>. The concentration of the HR repair fragment should be increased.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>pSB1A8YL</H5><P>3A assembly was also performed for K880005 and K592010 in pSB1A8YL. Yellow-colored colonies were observed.
		                                All colored constructs were grown and plated. Nice pictures were taken!
		</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Promoters</H5><P>Assembly of SCR1’-tRNA and pSB1C3 was confirmed by restriction analysis and sequencing.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Products</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Beta-Carotene</H5><P>Several attempts of Gibson assembly was made without correct results. We tried different strategies: different ratio of fragments, assembly two fragments at the time, new PCR of fragments.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Proinsulin</H5><P>3A assembly for the proinsulin gBlock and pSB1A8yl, pSB1A3/C3/K3. No colonies showed the expected length with colony PCR.
		                                Amplification of proinsulin gBlock using gradient PCR
		</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Substrates</H4><P>Results from growth on complex media in the form of different waste streams and byproducts were recreated. </P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Interlab</H4><P>The plate with the negative control was lost during a cleanup.</P></DIV></DIV><H3 style="clear:both;">Compute</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Software</H4><P>After watching and reading extensively about the power and capabilities of python based gui with tkinter, all the main functions have been set and the half-finished software is able to produce good output sequences while the option for taking out the unwanted restriction sites is not implemented yet !</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Hardware</H4><P>Improving Martins measuring cell. Printing a few prototypes. Try and Error. Looking for smart ways to control the light intensity as the photodetector has a saturation level. Experimenting with low-pass filters.</P></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" href="#collapse12" aria-expanded="false" aria-controls="collapse12"><DIV class=" panel-heading" role="tab" id="heading12"><H4 class="panel-title">
		                    Week 12 (August 15 - August 21) </H4></DIV></A><DIV id="collapse12" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading12"><DIV class="panel-body"><H3>Wetlab</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Molecular Toolbox</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>PEX10</EM> knockout</H5><P>Transformants on YPD plates from last week were re-streaked on selective media. The transformations was not successful. </P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>URA3</EM> insertion</H5><P>Unsuccessful co-transformation of pCRISPRyl+sgRNAs and the linear <EM>URA3</EM>+flanks fragment into <EM>Yarowia lipolytica</EM> PO1f Δ<EM>ku70</EM>.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Products</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Beta-Carotene</H5><P>Successful Gibson assembly of <EM>crtE:crtI</EM> and <EM>crtI:crtYB</EM>. These were used as templates for fusion PCR without success.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Proinsulin</H5><P>New 3A assembly (proinsulin + pSB1A8yl, pSB1C3). No transformants were present.
		                                Gibson assembly proinsulin gBlock and pSB1A8yl, pSB1C3. As PCR templates pSB1C3:J04450 and pSB1A8yl:K880005+K592010 were used. Colony PCR showed the expected amplicon length for pSB1A8yl:proinsulin construct.
		</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Substrates</H4><P>We experience problems with the OD measurements from the canola oil waste because of leftover plant material.</P></DIV></DIV><H3 style="clear:both;">Compute</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Genome Scale Modeling</H4><P>Meeting with faculties about genome scale modelling strategies, they think the strategy is doable.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Software</H4><P>The software now works with the full options that were supposed to be provided to the user. Optimization of  some main script parts was initiated.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Hardware</H4><P>Calibration routine for has been invented and implemented in R, ready to be ported to Arduino when needed.</P></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" href="#collapse13" aria-expanded="false" aria-controls="collapse13"><DIV class=" panel-heading" role="tab" id="heading13"><H4 class="panel-title">
		                    Week 13 (August 22 - August 28) </H4></DIV></A><DIV id="collapse13" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading13"><DIV class="panel-body"><H3>Wetlab</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>PEX10</EM> knockout</H5><P>Transformations in <EM>Y. lipolytica</EM> was repeated using the same approach as last week. The cells were inoculated in liquid selective media for two days. The transformations were unsuccessful. </P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>URA3</EM> insertion</H5><P>Successful amplification of URA3+flanks from <EM>Y. lipolytica</EM> W29 genomic DNA.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Products</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Beta-Carotene</H5><P>Designing USER primers.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Proinsulin</H5><P> pSB1A8yl:proinsulin was transformed into <EM>Y. lipolytica</EM>.
		                                pTEF, proinsulin, GFP, proinsulin tagged with GFP and the entire gBlock were extracted from the PCR amplified gBlock by PCR and in further step inserted into the pSB1C3 backbone by 3A assembly. Transformants were observed on most plates, although very few on the pTEF, GFP and Proinsulin thus these insertions was repeated using Gibson. No bands exhibited the correct lengths.
		</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Substrates</H4><P>Neither filtration nor emulgators seem to solve the problem the OD measurements on canola oil waste.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Interlab</H4><P>New transformations in competent DH5-Α cells with Negative Control Device has been attempted

twice with no growth. Positive control grew so the transformation was fine. Probably, the negative

control device was not present in the tube.</P></DIV></DIV><H3 style="clear:both;">Compute</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Genome Scale Modeling</H4><P>Phenotypic phase plane (PHPP) starts to be implemented, initial attempts in Matlab failed at reproducing phpp from literature, wrong signs and values.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Software</H4><P>The final &quot;touches&quot; on GUI layout has been implemented although upon agreement the design may be modified while keeping in mind that it has to be compatible with all &quot;mainstream&quot; OSs (cross-platform). </P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Hardware</H4><P>Diving into laser cutting. 
		                            Sketching out all the forms needed. Getting a headache from thinking in 3D.
		                            The work on assembling all the modules needed has begun. LCD, SD card reader, keypad and RTC are all implemented and working properly.
		</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV><H2 class="h2">September</H2><DIV class="panel panel-default"><A data-toggle="collapse" href="#collapse14" aria-expanded="false" aria-controls="collapse14"><DIV class=" panel-heading" role="tab" id="heading14"><H4 class="panel-title">
		                    Week 14 (August 29 - September 4) </H4></DIV></A><DIV id="collapse14" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading14"><DIV class="panel-body"><H3>Wetlab</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Molecular Toolbox</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>PEX10</EM> knockout</H5><P>Ran out of <EM>PEX10</EM> flanking regions. Therefore, amplification of <EM>PEX10</EM> flanking regions from <EM>Yarowia lipolytica</EM> PO1f Δ<EM>ku70</EM> were repeated in order to have enough for transformation. 
		                                The PCR products were purified and ready to use. 
		</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Products</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Beta-Carotene</H5><P>PCR of genes with USER primers and USER cloning of the three genes and two different backbones (pSB1A3, pSB1K3). No success.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Proinsulin</H5><P>The <EM>Yarowia lipolytica</EM> PO1f Δ<EM>ku70</EM> pSB1A8YL:Proinsulin transformants and controls were observed under a YFP filter. No difference between the pSB1A8YL:Proinsulin and the control was observed. Although, it was noted that the pSB1A8YL:Proinsulin strain grew significantly slower both on the plates and in liquid media suggesting that there are some difference between the pSB1A8YL:Proinsulin and control strain.
		                                3A assembly of pTEF, proinsulin, GFP, proinsulin:GFP and the entire gBlock into pSB1C3. Colony PCR for pTEF and the proinsulin transformations showed the expected fragment lengths. 
		</P></DIV></DIV><H3 style="clear:both;">Compute</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Hardware</H4><P>The calibration routine has been ported to Arduino, and is working perfectly.</P></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" href="#collapse15" aria-expanded="false" aria-controls="collapse15"><DIV class=" panel-heading" role="tab" id="heading15"><H4 class="panel-title">
	                    Week 15 (September 5 - September 11) </H4></DIV></A><DIV id="collapse15" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading15"><DIV class="panel-body"><H3>Wetlab</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Molecular Toolbox</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>PEX10</EM> knockout</H5><P>Since transformations have been unsuccessful so far, we tried a new approach. 
	                                Annealing of <EM>PEX10</EM> upstream and downstream flanking regions. Protocol from Cory M. Schwartz was used. However, this did not work. 
	                                New strategy for fusing the upstream and downstream Pex10 flanking regions: Fusion PCR. Wrong band sizes appeared on the gel after the PCR. </P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>URA3</EM> insertion</H5><P>New strategy for the <EM>URA3</EM>+flanks HR template: ligation onto pCRISPRyl+sgRNA. Primers were ordered for <EM>URA3</EM>+flanks amplification with a 5’-tail (tgc<STRONG>atgcat</STRONG>, NsiI restriction site in bold). 
	                                Unsuccessful amplification of <EM>URA3</EM>+flanks using the new primers.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Promoters</H5><P>The TEF1 promoter was successfully assembled with the pSB1C3 backbone by 3A assembly.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Products</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Beta-Carotene</H5><P>New USER cloning with new PCR’s, still no success.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Proinsulin</H5><P>3A assembly for GFP+pSB1C3, GFP+proinsulin+pSB1C3, pTEF+proinsulin in pSB1C3 and PSB1A8yl</P></DIV></DIV><H3 style="clear:both;">Compute</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Genome Scale Modeling</H4><P>PHPP is reproduced successfully using Cobrapy, and a simple GSM is built from scratch to fully understand FBA. Meeting with DTU Biosustain to get introduction to Cameo, the GSM extension to cobrapy. Modelling strategy is refined to test multiple carbon sources. </P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Software</H4><P>The production of the rest of the codon usage files was started for the, available gcn files by using an R script for matrix products,inspired by formulas provided by : dos Reis, M., Savva, R. and Wernisch, L., 2004. Solving the riddle of codon usage preferences: a test for translational selection. Nucleic acids research, 32(17), pp.5036-5044.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Hardware</H4><P>Learned how to make PCBs over the weekend. Ordering our very own Shield for the Arduino now. Thank you KiCad community for such a nice wiki.</P></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" href="#collapse16" aria-expanded="false" aria-controls="collapse16"><DIV class=" panel-heading" role="tab" id="heading16"><H4 class="panel-title">
		                    Week 16 (September 12 - September 18) </H4></DIV></A><DIV id="collapse16" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading16"><DIV class="panel-body"><H3>Wetlab</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Molecular Toolbox</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>PEX10</EM> knockout</H5><P>Fusion PCR from last week was repeated using a different PCR program. Unsuccessful fusion PCR. </P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>URA3</EM> insertion</H5><P>Unsuccessful amplification of <EM>URA3</EM>+flanks using the new primers. Since a linear template was being used, nested primers were designed (with the same 5’-tail as mentioned above).</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Products</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Beta-Carotene</H5><P>New USER cloning with new protocol: different ratio of fragments, different incubation program. Still no correct constructs.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Proinsulin</H5><P>Assembling pTEF with proinsulin, ECFP or EYFP biobricks into pSB1A8yl by 3A.</P></DIV></DIV><H3 style="clear:both;">Compute</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Genome Scale Modeling</H4><P>Phpp success, reproduced published results in both Matlab and Cobrapy. Matlab code  is modified to produce PHPP analysis and output desired product fluxes. Errors was probably due to wrong solver definition and constraints on the uptake fluxes.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Hardware</H4><P>Bug fixes everywhere. Trying to squeeze everything in the housing.
		                            The low pass filter needed for stabilization of the signal is added to the DIY and working.
		                            First inoculations are carried out! The technical results are promising, although the lack of stirring causes the biomass to fall to the bottom of the cuvettes.
		</P></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" href="#collapse17" aria-expanded="false" aria-controls="collapse17"><DIV class=" panel-heading" role="tab" id="heading17"><H4 class="panel-title">
	                    Week 17 (September 19 - September 25) </H4></DIV></A><DIV id="collapse17" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading17"><DIV class="panel-body"><H3>Wetlab</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Molecular Toolbox</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>PEX10</EM> knockout</H5><P>Fusion PCR from last week was repeated using a different PCR program. However, the fusion PCR was unsuccessful again. </P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>URA3</EM> insertion</H5><P>Successful amplification of <EM>URA3</EM>+flanks using the nested primers.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Products</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Beta-Carotene</H5><P>New USER with gradient incubation, still no success.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Proinsulin</H5><P>3A assembly in order to obtain pTEF+proinsulin, pTEF+CFP and pTEF+YFP in the pSB1A8YL backbone. 
	                                Repetition of 3A assemblies for pTEF+proinsulin, pTEF+CFP and pTEF+YFP using pSB1K3. All of them gave expected bands.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Substrates</H4><P>With high pressure filtration we were able to remove the plant material from the canola oil waste and perform successful growth experiments. </P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Interlab</H4><P>
										Transformation with negative control device in competent DH5 cells was repeated. 1 ul from thedistribution kit (plate 2, well 6F) was used for transformation. Calibration measurements with LUDOX and H<SUB>2</SUB>O were finished. FITC fluorescence standard curve was completed following the provided Plate Reader Protocol. Cell measurement was completed and all data was imported to the provided Excel sheet. The data was sent to measurement@igem.org and the online Plate Reader Form was submitted.
	                            </P></DIV></DIV><H3 style="clear:both;">Compute</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Genome Scale Modeling</H4><P>The newest <EM>Y. lipolytica</EM> model is downloaded, “MODEL1508190002”. Modelling different C-sources on PHPP succeeded with results matching substrate screening experiments. Tried glycerol, glucose, sucrose, xylose, maltose, fructose and oleic acid. </P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Hardware</H4><P>More testing and more bugs. Learning how to solder.</P></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" href="#collapse18" aria-expanded="false" aria-controls="collapse18"><DIV class=" panel-heading" role="tab" id="heading18"><H4 class="panel-title">
		                    Week 18 (September 26 - October 2) </H4></DIV></A><DIV id="collapse18" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading18"><DIV class="panel-body"><H3>Wetlab</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Molecular Toolbox</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>PEX10</EM> knockout</H5><P>Contacted Cory M. Schwartz and explained issues about the different approaches we had tried so far. Cory M. Schwartz have tried the same approaches and it did not work for him either. Cory M. Schwartz sent us <EM>Y. lipolytica</EM> PO1f strain and three plasmids we could use for transformation instead with the same design we attempted with no or limited luck. 
		                                - pIW357: A version of the CRISPRyl plasmid for knocking out PEX10. 
		                                - pIW381: A version of the CRISPRyl plasmid for knocking out MFE1. 
		                                -pIW501: This plasmid enables knockout of PEX10 by integration of a URA3 gene. </P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>URA3</EM> insertion</H5><P>NsiI restriction overnight of pCRISPRyl+sgRNAs, alkaline phosphatase treatment for 5 hours and column purification. Successful restriction (as confirmed on gel).
		                                NsiI restriction of <EM>URA3</EM>+flanks PCR product for 2 hours, followed by column purification.
		                                Ligation of pCRISPRyl+sgRNAs and URA3+flanks and transformation into <EM>E. coli</EM>.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>pSB1A8YL</H5><P>We realized that the GFP sequence we had fused to our proinsulin sequence would not work. Talking to faculty we found a sequence that would work, and ordered that as a gBlock.
		                                The gBlock arrived, and now it’s going to be a mad scramble to finish this in time for the wiki freeze!
		                                The gBlock was amplified and used for 3A assembly together with our TEF promoter.
		</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Promoters</H5><P>Both TEF1 and SCR1’-tRNA in pSB1C3 backbone was confirmed by sequencing.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Products</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Beta-Carotene</H5><P>USER cloning attempted in pSB1A8YL backbone, no luck.
		                                Sequencing of the original biobricks from the distribution kit/registry showed inconsistent prefix/suffix in BBa_K530001 and BBa_K530002, which might explain our troubles assembling the parts.
		</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Proinsulin</H5><P>Assembling pTEF+proinsulin, pTEF+CFP and pTEF+YFP pSB1K3 constructs into linearized pSB1C3 and pSB1A8YL with 3A. Growth was observed on the pSB1A8YL:pTEF+Proinsulin, pSB1C3:pTEF+Proinsulin, pTEF+CFP and pTEF+YFP constructs. pSB1A8yl:pTEF+Proinsulin was transformed into <EM>Yarowia lipolytica</EM> PO1f Δ<EM>ku70</EM> strain. After 2 days of incubation colonies were tested for fluorencence, however no signal was observed.
		                                3A assembly for: a) pSB1A8YL:TEF:hrGFP, b) pSB1A8YL:TEF:eCFP c) pSB1C3:TEF:hrGFP, d) pSB1C3:hrGFP (using hrGFP cut with EcoRI and PstI). Restriction analysis of pSB1A8yl:pTEF+eYFP confirm obtainig a proper construct. pSB1A8yl:pTEF+eYFP and pSB1A8yl:pTEF+proinsulin were transformed to <EM>Y. lipolytica</EM>. 
		                                pSB1A8yl:pTEF+eYFP was tested in both fluorescent microscope and in fluorometer, but no clear signal was observed.
		                                3A assembly for: a) pTEF:hrGFP, b) pTEF:CFP and c) pTEF:YFP in pSB1A8yl. Success! Constructs were transformed to <EM>Y. lipolytica</EM>. Colonies did not give fluorescence.
		</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Substrates</H4><P><EM>Y. lipolytica</EM> seem to grow well on the canola oil waste while <EM>S. cerevisiae</EM> does not grow very well.</P></DIV></DIV><H3 style="clear:both;">Compute</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Genome Scale Modeling</H4><P>Modelling FVA is done in Matlab, no useful information are interpreted. Explored the differences in <EM>Y. lipolytica </EM> and <EM>S. cerevisiae</EM>. with PHPP on the different carbon sources.  Beta-carotene pathway is added to the model.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Software</H4><P>Currently the library of codon usage files consists from 5 organisms with the intention for more to be added soon as .txt files. All of them are commonly used organisms. Research with regards to distribution and repository of software and compatibility regarding different OS versions. PyInstaller software will be used due to its capability of producing executables for all the known OS, so the user doesn't have to do anything else than just downloading the software.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Hardware</H4><P>Did I mention bug fixing already?
		                            The pump has been added to the DIY, apparently causing the program to crash.
		</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV><H2 class="h2">October</H2><DIV class="panel panel-default"><A data-toggle="collapse" href="#collapse19" aria-expanded="false" aria-controls="collapse19"><DIV class=" panel-heading" role="tab" id="heading19"><H4 class="panel-title">
		                    Week 19 (October 3 - October 9) </H4></DIV></A><DIV id="collapse19" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading19"><DIV class="panel-body"><H3>Wetlab</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Molecular Toolbox</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>PEX10</EM> knockout</H5><P>Received <EM>Y. lipolytica</EM> PO1f strain and pIW357, pIW381 and pIW501 from Cory M. Schwartz. 
		                                pIW501 was digested, and transformations into <EM>Y. lipolytica</EM> PO1f and PO1f Δ<EM>ku70</EM> was initiated with pIW357, liniar pIW501, pIW357 + linearized pIW501, pIW357 + circular pIW501 and circular pIW501. </P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>URA3</EM> insertion</H5><P>Plasmid purification and restriction analysis of pCRISPRyl+sgRNAs and <EM>URA3</EM>+flanks ligation/transformation. No successful transformants were obtained. This project was discontinued, and another strategy of inserting <EM>URA3</EM> in the place of <EM>PEX10</EM> is initiated.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>pSB1A8YL</H5><P>Unfortunately the 3A assembly of last week did not work out. We tried again.
		                                This time the colony PCR seemed right.
		</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Products</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Beta-Carotene</H5><P>We removed two illegal restriction sites (AgeI and BglII) from BBa_K530001 and BBa_K530002 by site directed mutagenesis. The improved BioBricks were digested to confirm correct removal. </P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Proinsulin</H5><P> 3A assembly for a) hrGFP+pSB1C3, b) pTEF:hrGFP+pSB1C3, c) pTEF:hrGFP+pSB1A8yl, d) pTEF:CFP+pSB1A8yl. Expected band was observed only for hrGFP+pSB1C3. This construct was used in a new 3A assembly together with pTEF and both pSB1A8yl and pSB1C3, which was successful. </P></DIV></DIV><H3 style="clear:both;">Compute</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Genome Scale Modeling</H4><P>Modelling getting more help from DTU Biosustain to set up Cameo on our computers, succeeded, Cameo can now be run without solvers and library errors. Got review literatures and questions and answers sessions from DTU Biosustain. We are added to DTU Biosustain’s online help platform in slack. And got many answers from DTU Biosustain experts, both online and in person at helpdesk sessions. Model is validated using Cameo with published and unpublished data from DTU Bioengineering, model is considered good enough to simulate general trends from substrate constraints. Tried to move on to strain design investigation using gene KO. Attempted but failed at making OptGene work. Returned no results. Attempted but failed at plotting fluxes on Escher due to reaction and metabolite IDs mismatch and the lack of <I>Y. lipolytica</I> metabolism maps. Same problem with KEGG maps, we do not have the right IDs to plot fluxes. It would be too time consuming to find the IDs or build a map on Escher ourselves, solution pending.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Hardware</H4><P>The shield arrived. After switching from breadboard to PCB, nothing works anymore.  Another weekend full of bug fixes.</P></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" href="#collapse20" aria-expanded="false" aria-controls="collapse20"><DIV class=" panel-heading" role="tab" id="heading20"><H4 class="panel-title">
	                    Week 20 (October 10 - October 16) </H4></DIV></A><DIV id="collapse20" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading20"><DIV class="panel-body"><H3>Wetlab</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Molecular Toolbox</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>PEX10</EM> knockout</H5><P>Transformations continues. There is growth in liquid selective media, but colonies are still growing on SC-oleic acid, which they should not be able to if <EM>PEX10</EM> has been knocked out. On a successful note, we achieved to insert <EM>URA3</EM> by homologous recombination in <EM>Yarowia lipolytica</EM> PO1f Δ<EM>ku70</EM>!</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>pSB1A8YL</H5><P>OH NO! The sequencing tells us that the TEF promoter was not inserted! This is going to be a last minute results, if any! 3A assembly was repeated.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Products</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Beta-Carotene</H5><P>We verified the removal of the illegal restriction sites by sequencing. The sequencing results showed a nucleotide substitution in each restriction site corresponding to the ones introduced with the primers.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>Proinsulin</H5><P>Multiple attempts to detect production of proinsulin in <I>Y. lipolytica</I> were preformed with rtPCR, SDS-PAGE and Western Blotting. No protein or mRNA was detected. we suspect that yields are too low, or protein is not folding correctly. Project discontinued.  </P></DIV></DIV><H3 style="clear:both;">Compute</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Genome Scale Modeling</H4><P>Modelling attempts focused on to OptGene KO simulation and differential FVA, FSEOF. Got help from Biosustain’s online and help desk sessions. Results is generated from FSEOF and FVA, but no result from OptGene.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Software</H4><P>The implementation of tAI based functions in the GUI application is finished and the codon usage are being produced in the fly according to user selection. Also the executables for Windows and Unix based systems have been created with PyInstaller.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Hardware</H4><P>Light at the end of the tunnel. Testing the complete working prototype and adjusting the menu to make it more user friendly.</P></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" href="#collapse21" aria-expanded="false" aria-controls="collapse21"><DIV class=" panel-heading" role="tab" id="heading21"><H4 class="panel-title">
		                    Week 21 (October 17 - October 23) </H4></DIV></A><DIV id="collapse21" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading21"><DIV class="panel-body"><H3>Wetlab</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Molecular Toolbox</H4></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>CRISPR-Cas9 induced <EM>PEX10</EM> knockout</H5><P>We have now observed that the colonies from pIW357 transformation into <EM>Y. lipolytica</EM> PO1f after SC-leu selection and outgrowth on YPD have different morphologies, a rose like shape, and a pearly shape. In consultation with Cory M. Schwartz we found out, that the pearly colonies are probably colonies with the <EM>PEX10</EM> knockouts. We immediately restreaked, and made colony PCR and sent for sequencing. Unfortunately we will not get the final results before after the wiki freeze, but who cares, it finally works!!! </P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H5>pSB1A8YL</H5><P>This time the construct seems right! Now only the fluorescence microscopy will tell us if our construct works.
		                                Aaaaaaaaand - it does!!!! Little autofluorescence on the negative, and clear, strong GFP signal from <EM>Y. lipolytica</EM> cells with the construct integrated! Luckily this data came in on the day of the wiki freeze! 
		</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Substrates</H4><P>Both <EM>S. cerevisiae</EM> and <EM>Y. lipolytica</EM> show growth on oleic acid, one of the main components in canola oil. When <EM>S. cerevisiae</EM> can grow on this there must be some other reason it will not grow on canola oil waste. This shows <EM>Y. lipolytica</EM> to be at growth on complex waste streams.</P></DIV></DIV><H3 style="clear:both;">Compute</H3><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Genome Scale Modeling</H4><P>Final simulations to produce PHPP graphs with fixed axis and simulated gene targets using FSEOF only, instead of KOs, we will use gene amplifications. Success at plotting the gradients of fluxes from FSEOF on to metabolic map created in Cytoscape, directly using the model sbml file with help from supervisor.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Software</H4><P>All the executable and library bundles have been produced and the after zipped they were deposited to IGEM software repository under the GPL v3 license.</P></DIV></DIV><DIV class="grid-row"><DIV class="col-md-10 col-sm-10 col-xs-12"><H4>Hardware</H4><P>Testing the final prototype together with Tobias and Keenan from biologigaragen. Getting a lot of nice feedback and suggestions for “after iGEM”.</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV><H2 class="h2">Protocols</H2><UL><LI><A href="https://static.igem.org/mediawiki/2016/e/e8/T--DTU-Denmark--Genomic_DNA_purification.pdf">Genomic DNA purification</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/7/74/T--DTU-Denmark--OneTaq_PCR_v1.0.pdf">One Taq PCR v1.0</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/7/74/T--DTU-Denmark--Phusion_U_PCR_v1.0.pdf">Phusion U PCR v1.0</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/b/be/T--DTU-Denmark--Chemically_competent_E._coli_cells_v1.0.pdf">Chemically competent E.coli cells</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/3/3f/T--DTU-Denmark--Oligo_annealing.pdf">Oligo annealing</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/6/6e/T--DTU-Denmark--Gibson_Assembly.pdf">Gibson assembly</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/4/48/T--DTU-Denmark--USER_cloning_v1.0.pdf">User cloning</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/2/2c/T--DTU-Denmark--E._coli_transformation_v1.0.pdf">E.coli transformation v1.0</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/8/8f/T--DTU-Denmark--E.coli_colony_pcr.pdf">E. coli colony PCR</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/b/b5/T--DTU-Denmark--Y._lipolytica_transformation_protocol_v1.0.pdf">Y. lipolytica transformation v1.0</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/c/c1/T--DTU-Denmark--Y._lipolytica_transformation_protocol_v2.0.pdf">Y. lipolytica transformation v2.0</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/6/66/T--DTU-Denmark--Y._lipolytica_colony_PCR.pdf">Y. lipolytica colony PCR</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/0/04/T--DTU-Denmark--Yeast_cell_lysis.pdf">Yeast cell lysis</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/1/19/T--DTU-Denmark--SC-oleic_acid_plates.pdf">SD-oleic acid media</A></LI></UL></DIV></DIV><DIV class="col-md-3 col-sm-2 colRight" id="scrollspy"><UL class="nav" id="sidebar"><LI><A href="#section-1">June</A></LI><LI><A href="#section-2">July</A></LI><LI><A href="#section-3">August</A></LI><LI><A href="#section-4">September</A></LI><LI><A href="#section-5">October</A></LI><LI><A href="#section-6">Protocols</A></LI></UL></DIV></DIV></DIV></DIV><FOOTER id="footer"><DIV class="container"><DIV align="center" class="col-md-3"><UL><LI>FIND US AT:</LI></UL><UL><LI><U>DTU BIOBUILDERS</U></LI><LI>DENMARK</LI><LI>DTU - SØLTOFTS PLADS, BYGN. 221/002</LI><LI>2800 KGS. LYNGBY</LI><LI><U>E-mail:</U></LI><LI>dtu-biobuilders-2016@googlegroups.com</LI></UL></DIV><DIV align="center" class="col-md-6"><UL><LI>MAIN SPONSORS:</LI></UL></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>