Team:DTU-Denmark/products
New HTML template for the wiki
☰
⇦
Jump to
Project
Wetlab
Compute
Team
Outreach
Description
Proof of Concept
Achievements
Notebook
Safety
Parts
Substrate
Products
Molecular Toolbox
Interlab
Codon Optimization
Microfermenter
Modeling
Team Members
Team Building
Attributions
Collaborations
Practices
Integrated Practices
Biosensor
Bootstrap Example
Products
In the Molecular tools section we developed several molecular tools, potentially allowing us to genetically engineer Yarrowia lipolytica. In this section we apply these tools and attempt to use these to achieve expression in Y. lipoltyca.
“Productivity is never an accident. It is always the result of a commitment to excellence, intelligent planning, and focus efforts”
Paul J. Meyer,
It's your life. Live big.
Introduction
“Productivity is never an accident. It is always the result of a commitment to excellence, intelligent planning, and focus efforts” 3
Paul J. Meyer,
It's your life. Live big.
<
By developing tools to genetically engineer Yarrowia lipolytica we aim to create a versatile cell factory, which in the future will be able to produce almost any desired product ranging from complex therapeutic proteins to high value chemical compounds. Besides diversity of products cell factory based on Y. lipolytica offers a wide range of substrate tolerance.
We aim to demonstrate this versatility of our Y. lipolytica strain as a cell factory by producing heterologous proteins. Our first attempt is the production of humanized, Renilla reniformis GFP (hrGFP), followed by proinsulin. To achieve both of mentioned ideas we apply the pSB1A8YL plasmid, cloning workflow and codon optimization algorithm developed by our team.
hrGFP
Design
In order to assess if we were able to follow our cloning flow proposed in the molecular tools section which would be expressed in Y. lipolytica, we chose to develop our own BioBricks. Namely a constitutive transcription factor alpha (TEF1) promoter native to Y. lipolytica ( BBa_K2117000 ) and the hrGFP gene ( BBa_K2117003 ) previously used successfully in Y. lipolytica 1 . The hrGFP sequence was codon optimized by the algorithm developed by our team. By combining these two parts in a device ( BBa_K2117005 ) the expression should in theory be easily detected due to the fluorescence signal produced. The cloning flow can be seen in Figure 1.
Figure 1: Cloning flow of the expression test of pSB1A8YL in Y. lipolytica. The expression of the GFP should yield a flourescent ouput detectable by fluorescence microscopy or fluorometer measurements.
×
DESCRIPTION
Results
The parts were ordered as gBlocks and assembled in E. coli using A3 assembly. The assembly was confirmed using restriction analysis (See Figure 2), PCR and sequencing (data not shown).
Afterwards the construct was transformed into Y. lipolytica PO1f and grown on plates containing selective media. Single colonies were picked and grown in liquid selective media, and subjected to fluorescence microscopy. Figure 3 shows the Y. lipolytica PO1f cells under a confocal laser microscope with 100x magnification. The high fluorescent output from with the BBa_K2117005 construct, proves that the cells are producing hrGFP, and ultimately that our expression system consisting of pSB1A8YL and the TEF1 promoter ( BBa_K2117000 ) can be used for heterologous protein expression in Y. lipolytica.
Figure 2: Analytical digestion of the pSB1A8YL containing the BBa_K2117005 device. The fragment lengths can be seen on the ladder, and the restriction enzyme and predicted fragment lengths is stated above the fragments.
×
DESCRIPTION
×
DESCRIPTION
Figure 3: Fluorescence microscopy conducted by a confocal laser microscope with 100x magnification. A and D are taken using standard brightfield, B and E are taken using the GFP filter and with the excitation laser on and C and F are overlays of the two photos where the black bagground has been removed (C is an overlay of A and B, and F is an overlay of D and E). A, B and C are Y. lipolytica PO1f cells with our GFP expressing device ( BBa_K2117005 ) shuttled by our plasmid pSB1A8YL. D, E and F are Y. lipolytica PO1f cells with the empty pSB1A8YL plasmid, which serves as a control for the GFP signal. Notice that even though the empty vector control shows trace amounts of auto-fluoresence the strain with the GFP expressing device clearly exhibits higher levels of fluorescence, which proves that our expression system works as intended.
×
DESCRIPTION
.
As seen in Figure 3 our construct produced a strong GFP signal, which ultimately proves that our expression system works as intended.
Insulin
Our experiments told us that our plasmid was a powerful tool, allowing us to conveniently assembling constructs in E. coli, which would be expressed in Y. lipolytica. Now we wanted to use it to take on the ultimate challange: Could we use this tool to produce proinsulin, and potentially increase the sustainability of insulin production?
Theory
Insulin is a peptrode hormone synthesised in the pacreas, and plays a crucial role in glucose homeostasis and prevents harmful levels of sugar in the blood. Beta-cells in the islets of the pancreas release insulin to the blood stream, which stimulates muscle fat and liver cells to absorb and store glucose from the bloodstream by interacting with insulin-specific receptors 2 . Diabetes is caused by either immune-mediated destruction of pacreatic islet cells (type 1 diabetes), or pancreatic exhaustion causing insulin resistance (type 2 diabetes). Both types of diabetes is cured by artificially controlling the insulin concentration in the patient by injection 3 . The treatment of diabetes with insulin was discovered in 1921, and its ability to restore health is so dramatic, that it was initially described in terms such as "the rising of the dead". Considering this it perhaps does not come as a surprise that insulin is often described as one of the most important biomedical events of the last century 4 , and as the number of adults with diabetes is expected to rise from 415 million today to 642 million in 2040, the relevance of diabetes is higher than ever 6 . To read more about the insulin market, see our descriptions page .
Insulin biosynthesis
At the first stage insulin is synthesized as a chain of 101 amino acids, called preproinsulin with a molecular weight of 12 Kilo daltons (kDa), which comprises of signal peptide (pre-peptide) and three short chains: A, B and C. The pre-peptide, responsible for directing a nascent polypeptide, is removed from preproinsulin giving the proinsulin consisting of 86 amino acids with a molecular weight of 9.4kDa. Subsequently, the proinsulin is folded and two disulphide bonds are created between chain A and B and one links chain A. In the last step, the C chain is digested from the proinsulin by an exoprotease - carboxypeptidase E. The mature insulin contains chains A and B linked by 3 disulphide bonds and in total comprises of 51 amino acids with a molecular weight of 5.8kDa 5 . The maturation of preproinsulin into proinsulin and finally mature insulin is shown in Figure 4.
Figure 4: Schematic representation of the maturation of preproinsulin into proinsulin and finally mature insulin. The colors represents the following: Pink: Signal peptide, yellow: B chain, green: C chain, Blue: A chain, red: Disulphide bonds. The proteolytic events is carried out by the carboxypeptidase E. Figure from WatCut .
×
DESCRIPTION
The first insulin was extracted from animals, and E. coli was emplyoed in the production of the A and B chain separately as recombinant DNA technologies progessed. Today insulin is produced in Saccharomyces cerevisiae by a single chain nonnative "miniproinsulin" precursor, which is matured by trypsin and carboxypeptidase B 4 .
Goal
In order to demonstrate Y. lipolytica as a versatile cell factory we aim to produce proinsulin as an answer to increasing global demand for insulin.
Results
The human proinsulin sequence was obtained from Sures et al. (1980). The sequence was codon optimized for the codon bias of Y. lipolytica using the codon optimization tool developed by our team, and ordered as a gBlock. The proinsulin gene with the RFC10 prefix and suffix was amplified by PCR into our BioBrick BBa_K2117003 and assembled with our TEF1 promoter to form our device BBa_K2117002 . This was done in pSB1A8YL using standard A3 assembly. The construct was transformed into E. coli, purified and confirmed using analytical digenstion, PCR and sequencing (data not shown). After confirmation of the construct, it was transformed into Y. lipolytica. The transformants was subjected to colony PCR (see Figure 5). The cloning flow can be seen in Figure 6.
Figure 5: Analytical digestion of the pSB1A8YL containing the BBa_K2117002 device. The fragment lengths can be seen on the ladder, and the primers predicted fragment lengths is stated above the fragments.
×
DESCRIPTION
Figure 6: The cloning flow of the proinsulin producing strain.
×
DESCRIPTION
We knew that detecting the proinsulin would be the biggest challange yet, as all previous detections was based on chromoproteins and fluorescence, while this was a protein without any intrinsic detectable signal. We had several considerations for detecting the proinsulin production including rtPCR and ELISA, altough ended up chosing SDS page and Western blotting, as these methods provide excellent sensitivity while some of the experiences running the SDS page experiments could be transfered to the western blotting experiments. The results of these experiments are show in Figure 7 and 8.
Figure 7: SDS page of Y. lipoltyica cell lysate stained with Coomassie blue. The construct the cells were transformed with and the dilution factor of the crude protein extract can be seen above the bands. The weight of the proteins can be seen on the ladders. Notice that no distinct band is visible at the approximate size of proinsulin (9.4kDa).
×
DESCRIPTION
Figure 8: Western Blotting of Y. lipoltyica cell lysate. The lysates were loaded the same way as the SDS page gel above. Notice that only noise is present on the picture.
×
DESCRIPTION
Unfortunately, non of these results gave conclusive evidence for the presence of proinsulin in our samples. The lack of proinsulin on the SDS could be caused by the concentration of proinsulin being lower than the detection limit. The lack of signal in the western blotting results could be caused by our antibody not recognising the proinsulin produced due to incomplete folding, or sub-optimal treatment of the membrane. Of course there is also the possibility that our strain is not producing proinsulin, although due to the positive results using hrGFP we are confident that our expression system works. Other effects such as mRNA folding or rapid protein degradation may also be factors leading to us not being able to detect proinsulin in the samples. Future efforts could include a fusion of the proinsulin, a reporter (such as GFP) or a purification tag (such as a His-tag) aiding the detection of proinsulin.
Conclusion
In this section we apply the tools made in the molecular toolbox section, for heterologous protein production in Y. lipolytica. This is done by combining two of our own BioBricks, a constitutive transcription factor alpha (TEF1) promoter native to Y. lipolytica ( BBa_K2117000 ) and the hrGFP gene ( BBa_K2117003 ) codon optimized by the algorithm developed by our team. By combining these two parts using A3 assembly and the cloning flow suggested in the molecular tools section, we created the device BBa_K2117005 . By confirming the production of hrGFP, we were able to prove that heterologous protein production is possible when our tool pSB1A8YL. Even though efforts to translate this into proinsulin production failed, the modular nature of pSB1A8YL means that it has potential to become a powerful tool when using Y. lipolytica for research or indsutrial purposes.
References
Blazeck, J., Liu, L., Redden, H., & Alper, H. (2011). Tuning Gene Expression in Yarrowia lipolytica by a Hybrid Promoter Approach. Applied and Environmental Microbiology, 77(22), 7905–7914. doi:10.1128/AEM.05763-11
NIDDK
Zaykov, A. N., Mayer, J. P., & DiMarchi, R. D. (2016). Pursuit of a perfect insulin. Nat Rev Drug Discov, 15(6), 425–439. JOUR. Retrieved from http://dx.doi.org/10.1038/nrd.2015.36
Mayer, J. P., Zhang, F., & DiMarchi, R. D. (2007). Insulin structure and function. Biopolymers, 88(5), 687–713. doi:10.1002/bip.20734
Baeshen, N. A., Baeshen, M. N., Sheikh, A., Bora, R. S., Ahmed, M. M. M., Ramadan, H. A. I., … Redwan, E. M. (2014). Cell factories for insulin production. Microbial Cell Factories, 13(1), 141. doi:10.1186/s12934-014-0141-0
IDF annual report 2015
Introduction
hrGFP
Insulin
Conclusion
FIND US AT:
DTU BIOBUILDERS
DENMARK
DTU - SØLTOFTS PLADS, BYGN. 221/002
2800 KGS. LYNGBY
E-mail:
dtu-biobuilders-2016@googlegroups.com
MAIN SPONSORS:
