<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_DTU-Denmark_substrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:DTU-Denmark/substrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>New HTML template for the wiki</TITLE></P><DIV id="headerWrapper"><DIV id="Topnav" class="container"><DIV id="logo"><BUTTON id="hamburger" class="mobile-menue">☰</BUTTON><BUTTON id="arrow" class="mobile-menue">⇦</BUTTON><BUTTON id="a-down" class="mobile-menue mobile-scrole">Jump to</BUTTON></DIV><DIV id="navigation"><UL id="nav-items"><LI><A href="https://2016.igem.org/Team:DTU-Denmark/project" class="project nav-item" id="project">Project</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/wetlab" class="wetlab nav-item" id="wetlab">Wetlab</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/compute" class="compute nav-item" id="compute">Compute</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/team" class="team nav-item" id="team">Team</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/outreach" class="outreach nav-item" id="outreach">Outreach</A></LI></UL></DIV></DIV><DIV id="sub-navigation-wrapper"><DIV id="sub-navigation" class="container"><UL id="project-sub"><LI><A href="https://2016.igem.org/Team:DTU-Denmark/description" class="sub-nav-item">Description</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Proof" class="sub-nav-item">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/achievements" class="sub-nav-item">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Notebook" class="sub-nav-item">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Safety" class="sub-nav-item">Safety</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Description" class="sub-nav-item">Parts</A></LI></UL><UL id="wetlab-sub"><LI><A href="https://2016.igem.org/Team:DTU-Denmark/substrate" class="sub-nav-item">Substrate</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/products" class="sub-nav-item">Products</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/molecular_toolbox" class="sub-nav-item">Molecular Toolbox</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/molecular_interlab" class="sub-nav-item">Interlab</A></LI></UL><UL id="compute-sub"><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Software" class="sub-nav-item">Codon Optimization</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Hardware" class="sub-nav-item">Microfermenter</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Model" class="sub-nav-item">Modeling</A></LI></UL><UL id="team-sub"><LI><A href="https://2016.igem.org/Team:DTU-Denmark/team_members" class="sub-nav-item">Team Members</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/team_building" class="sub-nav-item">Team Building</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Attributions" class="sub-nav-item">Attributions</A></LI></UL><UL id="outreach-sub"><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Collaborations" class="sub-nav-item">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/HP/Silver" class="sub-nav-item">Practices</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/HP/Gold" class="sub-nav-item">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Engagement" class="sub-nav-item">Biosensor</A></LI></UL></DIV></DIV></DIV><P><TITLE>Bootstrap Example</TITLE></P><DIV class="MYbody"><DIV class="masthead"><DIV class="container-fluid"><DIV class="row"><DIV class="col-md-12 thumbnail" style="background-image:url(https://static.igem.org/mediawiki/2016/c/c2/T--DTU-Denmark--Lab-header.jpg)"><DIV class="caption"><DIV class="col-md-5 col-sm-5 col-xs-12 title"><H1>Substrate<P class="lead">As the substrate utilization of <I>Yarrowia lipolytica</I> is central to our project we have performed an array of growth experiments. We have tested the growth on simple media to determine strengths and weaknesses in the catabolism of <I>Y. lipolytica</I>. We further expanded our research by acquiring real waste streams and byproducts form organic industrial productions in the Nordic countries and screened <I>Y. lipolytica</I> growth for these substrates.</P></H1></DIV><DIV class="col-md-5 col-sm-5 hidden-xs intro"><BLOCKQUOTE class="blockquote-reverse"><P>&quot;Growth can be the result of many trials&quot;</P><SMALL>Mohamad El Lakany, <CITE title="Source Title">Mohamad's Mantra</CITE></SMALL></BLOCKQUOTE></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-md-9 col-sm-10 colLeft"><DIV><H2 class="h2">Introduction</H2><BLOCKQUOTE class="visible-xs"><P>&quot;Growth can be the result of many trials&quot;</P><SMALL>Mohamad El Lakany, <CITE title="Source Title">Mohamad's Mantra</CITE></SMALL></BLOCKQUOTE><P>
                The dimorphic, non-conventional yeast <I>Yarrowia lipolytica</I>, belonging to the Ascomycota phylum, was first isolated in the 1960s from lipid-rich materials, hence the name “lipolytica”. The organism was classified and reclassified a number of times, first as <I>Candida lipolytica</I>, then <I>Endomycopsis lipolytica</I>, <I>Saccharomycopsis lipolytica</I> and finally <I>Yarrowia lipolytica</I><SUP><A href="#references">1</A></SUP>. Figure 1 shows <I>Y. lipolytica</I> cells under a microscope. 
              </P><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 1:</B><I>Y.lipolytica</I> in plactonic growth with 100x magnification.</FIGCAPTION></FIGURE><DIV id="Y.lMicroModal" class="modal"><SPAN class="close Y.lMicro">×</SPAN></DIV><P>
                  In recent years, <I> Y. Lipolytica</I> has received increased attention from researchers, as studies have found it to possess great potential for producing industrial enzymes and pharmaceutical proteins. This potential is a result of several advantages that <I> Y. Lipolytica</I> has over the conventional yeast <I> S. cerevisiae</I>. <I> Y. Lipolytica</I> prefers secreting proteins through the co-transcription pathway and does so very efficiently<SUP><A href="#references">2</A></SUP> in addition, it does not exhibit hyperglycosylation as <I> S. cerevisiae</I> does<SUP><A href="#references">3</A></SUP>. <I> Y. Lipolytica</I> has also been shown to exhibit excellent characteristics for the production of value-added chemicals such as a long range of organic acids and polyols. The recent introduction of several genome-scale models for <I> Y. Lipolytica</I> will most likely lead to more processes utilizing the chassis for production. Perhaps, the most important advantage for using <I> Y. Lipolytica</I> over <I> S. cerevisiae</I>, to our project at least, is the broad substrate utilization range of <I> Y. Lipolytica</I>. <I> Y. Lipolytica</I> is known to naturally utilize alcohols (especially glycerol), acetate and hydrophobic substrates (eg. alkanes, fatty acids and oils) as carbon source <SUP><A href="#references">4</A></SUP>. This has naturally led to <I> Y. Lipolytica</I> becoming a model organism for several metabolic pathways, especially fatty acid transport, -metabolism, and single cell oil (SCO) accumulation. <I> Y. Lipolytica</I> has even been shown to exhibit enhanced growth on mixed substrates. <I>Yarrowia lipolytica</I> is an oleaginous cell factory platform for production of fatty acid-based biofuels and bioproducts. This renders it ideal for utilization of industrial waste streams due to their complex and variable content. These findings have us believe that we had found an excellent candidate chassis for our project. The table below shows a comparison of the substrate range of <I> Y. Lipolytica</I> W29 and <I> S. cerevisiae</I> CEN.PK113-7D.
              </P><TABLE class="table table-bordered"><THEAD><TR><TH><I>Y. Lipolytica</I></TH><TH><I>S. cerevisiae</I></TH></TR></THEAD><TBODY><TR><TD>Sediment from canola oil production</TD><TD>µ = 0.31</TD><TD>None</TD></TR><TR><TD>Glycerol from Perstop</TD><TD>µ = 0.27</TD><TD>None</TD></TR><TR><TD>Glycerol from Emmelev</TD><TD>µ = 0.45</TD><TD>None</TD></TR><TR><TD>Glycerol from Daka</TD><TD>µ = 0.31</TD><TD>None</TD></TR><TR><TD>Molasses from Dansukker</TD><TD>µ = 0.42*</TD><TD>µ = 0.47</TD></TR></TBODY></TABLE><P>
                * it should be noted that the molasses was autoclaved thus degrading some of the sucrose content. This growth might not be possible to replicate with untreated molasses.
              </P><P>
                As seen in the table <I> Y. Lipolytica </I> is able to grow on all the waste sources we tested, while <I>S. cerevisiae</I> is only able to grow on molasses. 
              </P></DIV><DIV><H2 class="h2">Methods</H2><P>
                Each growth experiment (for <I> Y. Lipolytica </I> and <I>S. cerevisiae</I>) is conducted according to the following setup: 
              </P><P>
                Minimal medium is produced as directed by Mhairi Workman<SUP><A href="#references">5</A></SUP> using 20g/L of a given carbon source for all the growth experiments.

              </P><P>
                The cells were grown overnight in YPD medium, and prepared by spinning down and washed twice. The preculture was then used as inoculum for minimal medium (substituents) to a final concentration of 0,001 (OD<SUB>600</SUB>) measured by Spectrophotometry (Shimadzu UV-1800). The cultivations were carried out in a cytomat (Thermo Scientific) shaking 900 rpm at 30 degree celsius. Cultures were grown, shaked and measured in a 48 well suspension culture plates (Cellstar, Greiner-bio-one). The measurements were carried out using a Hamilton Microlab Robot, (Hamilton Life science Robotics) connected to a plate spectrophotometer (BioTek Synergy 2).OD<SUB>600</SUB> measurements were taken every 2 hours until the cultures reached stationary phase. Data was then analysed and visualized using excel and R-studio Figure 2.
              </P><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 2:</B><B>A.</B> Overnight culture: strains of <I>Y. lipolytica</I> and <I>S. cerevisiae</I> are grown in YPD media overnight at 30℃ (86℉)  to ensure balanced growth and comparable data.
                  <B>B.</B> Washing- and inoculation steps: Cells are spinned down and washed to ensure removal of carbon-sources and other metabolites from the overnight-culturing. Washing and spinning step is repeated. 
                  Simple and complex substrates are inoculated with cells in 48 well suspension culture plates. The cells reaches final OD<SUB>600</SUB> 0.001
                  <B>C.</B> Growth-experiment: Plates are incubated and shaken at 900 rpm in a cytometer and before measurement of OD in a spectrophotometer. Data are recorded and compiled in an excel sheet with two hours intervals. This process is assisted by using the Hamilton Microlab robot.
                  <B>D.</B> Data analysis and -visualization step: The data excel sheet (in step <B>C.</B>) are analyzed and visualized by plots using R-studio.</FIGCAPTION></FIGURE><P>During the growth experiments we kept to strains that were wild type or closely related. This makes the results more general for the organism.</P><DIV id="ABCModal" class="modal"><SPAN class="close ABC">×</SPAN></DIV><TABLE class="table table-bordered"><THEAD><TR><TH>Strains</TH><TH>Genotype</TH><TH>Comment/source</TH></TR></THEAD><TBODY><TR><TD><I>Y. Lipolytica</I></TD><TD>Wildtype</TD><TD>Parent strain to our laboratory bug, PO1f</TD></TR><TR><TD><I>S. cerevisiae</I></TD><TD>CEN.PK113-7D</TD><TD>Derived from parental strains ENY.WA-1A and MC996A,and is popular for use in systems biology studies</TD></TR></TBODY></TABLE></DIV><DIV><H2 class="h2">Outline of Process</H2><DIV class="col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 3:</B> Picture of the waste products we received.</FIGCAPTION></FIGURE><DIV id="CMpModal" class="modal"><SPAN class="close CMp">×</SPAN></DIV></DIV><DIV class="col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 4:</B> Picture of the autoclaved C-source solutions.</FIGCAPTION></FIGURE><DIV id="MMpModal" class="modal"><SPAN class="close MMp">×</SPAN></DIV></DIV><P>
                We performed growth experiments on an array of pure C-sources (seen in figure 3-4) to get a baseline of <I> Y. Lipolytica </I> growth patterns emerged indicating the substrate range. In these experiments we observed the following growth rates or lack of growth.  
              </P><TABLE class="table table-bordered"><THEAD><TR><TH><I>Y. Lipolytica</I></TH><TH><I>S. cerevisiae</I></TH></TR></THEAD><TBODY><TR><TD>Glucose</TD><TD>µ = 0.24</TD><TD>µ = 0.19</TD></TR><TR><TD>Fructose</TD><TD>µ = 0.23</TD><TD>µ = 0.42<SUP><A href="#references">6</A></SUP></TD></TR><TR><TD>Glycerol</TD><TD>µ = 0.27</TD><TD>None</TD></TR><TR><TD>Canola oil</TD><TD>µ = 0.08</TD><TD>None</TD></TR><TR><TD>Sucrose</TD><TD>None</TD><TD>µ = 0.39<SUP><A href="#references">6</A></SUP></TD></TR><TR><TD>Maltose</TD><TD>None</TD><TD>Growth<SUP><A href="#references">7</A></SUP></TD></TR><TR><TD>Xylose</TD><TD>None</TD><TD>None<SUP><A href="#references">8</A></SUP></TD></TR><TR><TD>Arabinose</TD><TD>None</TD><TD>None<SUP><A href="#references">8</A></SUP></TD></TR><TR><TD>Starch</TD><TD>None</TD><TD>None</TD></TR></TBODY></TABLE><P> The graphs representing these results can be seen in the figures 5-9:</P><DIV class="col-md-4"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 5:</B><I>Y. Lipolytica</I> growth on fructose.</FIGCAPTION></FIGURE><DIV id="frucModal" class="modal"><SPAN class="close fruc">×</SPAN></DIV></DIV><DIV class="col-md-4"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 6:</B><I>Y. Lipolytica</I> does not grow on sucrose.</FIGCAPTION></FIGURE><DIV id="sucModal" class="modal"><SPAN class="close suc">×</SPAN></DIV></DIV><DIV class="col-md-4"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 7:</B><I>Y. Lipolytica</I> growth on Canola oil.</FIGCAPTION></FIGURE><DIV id="oleModal" class="modal"><SPAN class="close ole">×</SPAN></DIV></DIV><DIV class="col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 8:</B><I>Y. Lipolytica</I> growth on glucose.</FIGCAPTION></FIGURE><DIV id="gluYModal" class="modal"><SPAN class="close gluY">×</SPAN></DIV></DIV><DIV class="col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 9:</B><I>S. cerevisiae</I> growth on glucose.</FIGCAPTION></FIGURE><DIV id="glusModal" class="modal"><SPAN class="close glus">×</SPAN></DIV></DIV><P>
                Even though the pure carbon sources suggests that <I>Y. Lipolytica</I> exhibits excellent substrate utilization, we did not know if this translated into utilization of industrial waste streams. To investigate this, we had to get our hands on a few waste streams we could test. We contacted local industry that we knew had waste streams containing either sugars, glycerol or oily constituents. After many phone calls and long meetings, we received the following byproducts of organically based productions:
              </P><UL><LI>Canola oil sediment</LI><LI>Glycerol Perstorp Tech</LI><LI>Glycerol Emmelev</LI><LI>Glycerol Daka</LI><LI>Molasses Dansukker</LI></UL></DIV><DIV><H2 class="h2">Industrial Byproduct Screenings</H2><H3 class="h3">Canola Oil Sediment - Grønningaard</H3><P>
                Grønningaard is a canola oil production facility situated on Zealand, Denmark. They produce 100 - 120 tons canola oil annually, and sell the remaining protein rich press cake for animal feed. The oil is derived by cold pressing organic rapeseeds. As cold pressing does not allow for filtering of the oil, small fibres remain in the oil. These fibres are removed by allowing the oil to sediment for 1 month before extracting the sediment. Besides the fibres from the plants and residual oil, the sediment contains polyaromatic hydrocarbons in high concentrations, making the sediment unsuitable to be recycled in the process or used for animal feed, rendering it a “true waste” in the sense that it is only useful for generating heat through incineration. Figure 10 shows an overview of the process. The sediment constitutes 1-1.6% of the biomass of the product, amounting to 1 - 1.92 tons annually. These figures are based on the 4th. biggest producer in Denmark Grønninggård. The largest with an estimated 80% market share is not willing to provide production numbers (Personal communication).  
              </P><CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 10:</B> Flow chart for the production of cold pressed canola oil.</FIGCAPTION></FIGURE></CENTER><DIV id="GGprodModal" class="modal"><SPAN class="close GGprod">×</SPAN></DIV><P>
                During the experiments using this substrate we experienced a lot of problems with the OD measurements because of the high content of plant fibers. Through pressure filtering, a transparent sample was extracted. We demonstrated that <I>Y. Lipolytica</I> grows very well on this waste stream. <I>S. cerevisiae</I> on the other hand is not able to utilize this carbon source as seen in figures 11-12.
              </P><DIV class="col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 11:</B><I>Y. Lipolytica</I> growth on sediment from canola oil production.</FIGCAPTION></FIGURE><DIV id="GGyModal" class="modal"><SPAN class="close GGy">×</SPAN></DIV></DIV><DIV class="col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 12:</B><I>S. cerevisiae</I> does not grow on sediment from canola oil production.</FIGCAPTION></FIGURE><DIV id="GGsModal" class="modal"><SPAN class="close GGs">×</SPAN></DIV></DIV><P>This shows promising growth of <I>Y. Lipolytica</I> while it is clear that <I>S. cerevisiae</I> is unsuitable for fermentation based on canola oil sediments</P></DIV><DIV><H3 class="h3">Glycerol Byproduct</H3><P>
                The push to find an alternative to fossil fuel has increased demand and production of biodiesel tremendously in the last two decades. Biodiesel is produced by a base-catalyzed transesterification by a short chained alcohol and triacylglycerols derived from natural sources as seen in figure 13. This reaction produces 0.102 kg glycerol pr. liter biodiesel<SUP><A href="#references">9</A></SUP>. The increased production has resulted in a plummeting of glycerol prices making it a promising substrate for industrial fermentation.
              </P><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 13:</B> Flow chart for production of biodiesel and glycerol waste.</FIGCAPTION></FIGURE><DIV id="GlyceroModal" class="modal"><SPAN class="close Glycero">×</SPAN></DIV><H3 class="h3">Second Generation Biodiesel Facility (DAKA)</H3><P>
                The Danish branch of DAKA refines waste streams from the feedstock industry (such as meat and agricultural industry), turning it into products such as fertilizers, animal feed and biodiesel. The biodiesel production is based on animal tallow and fats from the Danish meat industry. The glycerol derived from this production has a high salt content and a particularly low pH and therefore requires several purification steps before it can be used in the chemical industry. By adding NaOH and raising the pH to 6 we were able to make <I> Y. Lipolytica </I> grow fairly well in spite of the relatively high salt levels as seen in Figure 14 (Personal communication). 
              </P><DIV class="col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 14:</B><I>Y. Lipolytica</I> growth on glycerol from second generation biodiesel.</FIGCAPTION></FIGURE><DIV id="DakaYModal" class="modal"><SPAN class="close DakaY">×</SPAN></DIV></DIV><DIV class="col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 15:</B><I>S. cerevisiae</I> does not grow on glycerol.</FIGCAPTION></FIGURE><DIV id="DakaSModal" class="modal"><SPAN class="close DakaS">×</SPAN></DIV></DIV><H3 class="h3">First Generation Glycerol (Perstorp and Emmelev) </H3><P>
                Perstop has two biodiesel production facilities, one located in Sweden and one in Norway. They produce high quality glycerin, that is sold as a component for chemical production. This has a purity of 95-100% the remainder of mostly water (Personal communication). This byproduct is also a great substrate for <I> Y. Lipolytica </I> as seen in Figure 16.
              </P><DIV class="col-md-12"><DIV class="col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 16:</B><I>Y. Lipolytica</I> growth on glycerol from first generation biodiesel.</FIGCAPTION></FIGURE><DIV id="PTYModal" class="modal"><SPAN class="close PTY">×</SPAN></DIV></DIV><DIV class="col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 17:</B><I>S. cerevisiae</I> does not grow on glycerol.</FIGCAPTION></FIGURE><DIV id="PTSModal" class="modal"><SPAN class="close PTS">×</SPAN></DIV></DIV></DIV><P>
                Emmelev A/S is a local oil mill, first generation biodiesel plant and a glycerin destillor located on the second biggest island in Denmark, Fyn. The glycerin is distilled to 80% purity and sold to the chemical industry (Personal communication). This is fairly high quality and there are no components that inhibit the growth of <I> Y. Lipolytica </I> as seen in Figure 19.
              </P><DIV class="col-md-12"><DIV class="col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 19:</B><I>Y. Lipolytica</I> growth on glycerol from first generation biodiesel.</FIGCAPTION></FIGURE><DIV id="EmYModal" class="modal"><SPAN class="close EmY">×</SPAN></DIV></DIV><DIV class="col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 20:</B><I>S. cerevisiae</I> does not grow on glycerol.</FIGCAPTION></FIGURE><DIV id="EmSModal" class="modal"><SPAN class="close EmS">×</SPAN></DIV></DIV></DIV><P> In conclusion, <I>Y. Lipolytica</I> is the only suitable candidate for fermentation based on glycerol.</P></DIV><DIV><H3 class="h3">Molasses</H3><P>
                The process of creating refined sugar results in the waste product molasses. Molasses is a byproduct of the refining of sugarcane or sugar beets into sugar. It is brown in color and has a sweet flavor due to the high sucrose, glucose and fructose content. Therefore it is often used for prepacked meals and animal feed. Molasses is created when the juice from sugar canes is heated to boiling point. Sugars are extracted over two times as seen in Figure 21. Leaving molasses as a byproduct of this process. We think that it will be a quite useful substrate for fermentation. 
              </P><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 21:</B> Flow chart for production of refined sugar and molasses.</FIGCAPTION></FIGURE><DIV id="molassesModal" class="modal"><SPAN class="close molasses">×</SPAN></DIV><P>
                As seen in the figure both <I> Y. Lipolytica </I> and <I> S. cerevisiae </I> grows on molasses. Normally <I> Y. Lipolytica </I> does not grow well on sucrose, but there is also a high content of glucose and fructose in molasses. On top of that we realised that sucrose degrades to fructose and glucose when autoclaved (Personal communication) that made fermentation more attractive. 
              </P><DIV class="col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 22:</B><I>Y. Lipolytica</I> growth on molasses from sugar production.</FIGCAPTION></FIGURE><DIV id="molYModal" class="modal"><SPAN class="close molY">×</SPAN></DIV></DIV><DIV class="col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 23:</B><I>S. cerevisiae</I> growth on molasses from sugar production.</FIGCAPTION></FIGURE><DIV id="molSModal" class="modal"><SPAN class="close molS">×</SPAN></DIV></DIV><P> Both organisms might be suitable for growth on molasses as seen in figures 22-23, but because <I>S. cerevisiae</I> is able to utilize sucrose it might be the better choice for this byproduct.</P></DIV><DIV><H2 class="h2">Discussion</H2><P>
                From our experiments it is clear that <I>S. cerevisiae</I> and <I>Y. lipolytica</I> have very different substrate utilization ranges. <I>S. cerevisiae</I> is better at catabolizing simple sugars while <I>Y. lipolytica</I> is better at degrading lipids, its derivatives and other complex substrates. A lot of the sugar-based byproducts like molasses, are suitable for human consumption. A biobased production approach with these substrates competes with the increasing food demand. The lipid-based wastestreams like glycerol and oil sediments are not suitable for neither human nor animal consumption. As <I>Y. lipolytica</I> displays good growth on these wastestreams, it is well-suited for biobased production systems. However, it should be noted that these wastestreams cannot be utilized for all possible compounds of interest. More specifically, a lot of problems need to be overcome in order to produce compounds such as pharmaceuticals that have to adhere to specific regulations. Based on our interviews with industry representatives, bulk compounds might be a more viable option. Still, new challenges arise when switching to productions based on oily carbon sources. Amongst others the cleaning procedure for the fermentation tanks must be adapted. This was pointed out at our meeting with <A href="“https://2016.igem.org/Team:DTU-Denmark/HP/Gold#section-3”">Novozymes</A> and can be read in the <A href="”https://static.igem.org/mediawiki/2016/a/a6/T--DTU-Denmark--InterviewwithGernotNovozymes.pdf”">resume</A>.              
   
              </P></DIV><DIV id="ref_sec"><H2 class="h2">References</H2><OL><LI> Barth, G. and Gaillardin, C. (1997), Physiology and genetics of the dimorphic fungus Yarrowia lipolytica. FEMS Microbiology Reviews, 19: 219–237. doi:10.1111/j.1574-6976.1997.tb00299.x</LI><LI> María Domínguez, Jonathan D. Wasserman, Matthew Freeman, Multiple functions of the EGF receptor in Drosophila eye development, Current Biology, Volume 8, Issue 19, 24 September 1998, Pages 1039-1048, ISSN 0960-9822, http://dx.doi.org/10.1016/S0960-9822(98)70441-5.</LI><LI>E. V. Shusta, R. T. Raines(1998). ncreasing the secretory capacity of Saccharomyces cerevisiae for production of single-chain antibody fragments. http://www.nature.com/nbt/journal/v16/n8/pdf/nbt0898-773.pdf</LI><LI>Barth, G. (2013). Yarrowia lipolytica Genetics, Genomics, and Physiology. http://www.springer.com/us/book/9783642383199</LI><LI>Mhairi Workman, Philippe Holt (2013). Comparing cellular performance of Yarrowia lipolytica during growth on glucose and glycerol in submerged cultivations</LI><LI>H. Shafaghat, G.D. Najafpour. Growth Kinetics and Ethanol Productivity of Saccharomyces cerevisiae PTCC 24860 on Varius Carbon Sources. ISSN 1818-4952</LI><LI>Jansen, M. L. A., Daran-Lapujade, P., de Winde, J. H., Piper, M. D. W., &amp; Pronk, J. T. (2004). Prolonged Maltose-Limited Cultivation of Saccharomyces cerevisiae Selects for Cells with Improved Maltose Affinity and Hypersensitivity. Applied and Environmental Microbiology, 70(4), 1956–1963. http://doi.org/10.1128/AEM.70.4.1956-1963.2004</LI><LI>Wisselink, H. W., Toirkens, M. J., del Rosario Franco Berriel, M., Winkler, A. A., van Dijken, J. P., Pronk, J. T., &amp; van Maris, A. J. A. (2007). Engineering of Saccharomyces cerevisiae for Efficient Anaerobic Alcoholic Fermentation of l-Arabinose . Applied and Environmental Microbiology, 73(15), 4881–4891. http://doi.org/10.1128/AEM.00177-07</LI><LI>Syed Shams Yazdani, Ramon Gonzalez, Anaerobic fermentation of glycerol: a path to economic viability for the biofuels industry, Current Opinion in Biotechnology, Volume 18, Issue 3, June 2007, Pages 213-219, ISSN 0958-1669, http://dx.doi.org/10.1016/j.copbio.2007.05.002.</LI></OL></DIV></DIV><DIV class="col-md-3 col-sm-2 colRight" id="scrollspy"><UL class="nav" id="sidebar"><LI><A href="#section-1">Introduction</A></LI><LI><A href="#section-2">Methods</A></LI><LI><A href="#section-3">Outline of process</A></LI><LI><A href="#section-4">Industrial Byproduct Screenings</A></LI><LI><A href="#section-5">Discussion</A></LI></UL></DIV></DIV></DIV></DIV><FOOTER id="footer"><DIV class="container"><DIV align="center" class="col-md-3"><UL><LI>FIND US AT:</LI></UL><UL><LI><U>DTU BIOBUILDERS</U></LI><LI>DENMARK</LI><LI>DTU - SØLTOFTS PLADS, BYGN. 221/002</LI><LI>2800 KGS. LYNGBY</LI><LI><U>E-mail:</U></LI><LI>dtu-biobuilders-2016@googlegroups.com</LI></UL></DIV><DIV align="center" class="col-md-6"><UL><LI>MAIN SPONSORS:</LI></UL></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>