Team:DTU-Denmark/Description
New HTML template for the wiki
☰
⇦
Jump to
Project
Wetlab
Compute
Team
Outreach
Description
Proof of Concept
Achievements
Notebook
Safety
Parts
Substrate
Products
Molecular Toolbox
Interlab
Codon Optimization
Microfermenter
Modeling
Team Members
Team Building
Attributions
Collaborations
Practices
Integrated Practices
Biosensor
Bootstrap Example
Parts
We have constructed four new basic parts, 4 composite parts, a plasmid backbone and improved two parts made by a previous team. The parts are all a part of our molecular toolbox for protein expression in Yarrowia lipolytica.
"A great building will never stand if you neglect the small bricks"
Ifeanyi Enoch Onuoha
Parts list
"A great building will never stand if you neglect the small bricks"
Ifeanyi Enoch Onuoha
BBa_K2117000
Constitutive TEF1 promoter
BBa_K2117001
hrGFP codon-optimized for Yarrowia lipolytica
BBa_K2117002
Composite part consisting of the TEF1 promoter and the human proinsulin gene codon-optimized for Y. lipolytica
BBa_K2117003
Human proinsulin codon-optimized for Y. lipolytica
BBa_K2117004
SCR1'-tRNA promoter expressing sgRNA
BBa_K2117005
hrGFP expressed by the constitutive TEF1 promoter
BBa_K2117009
Semi-synthetic shuttle vector pSB1A8YL
BBa_K2117010
Composite part of TEF1 and enhanced YFP
BBa_K2117011
Composite part of TEF1 and enhanced CFP
BBa_K2117012
Improved BBa_K530001 (crtE)
BBa_K2117013
Improved BBa_K530002 (crtI)
Improvement and Characterization of Two Existing Parts
In our laboratory work, we wanted to work with the two BioBricks; BBa_K530001 (crtE gene) and BBa_K530002 (crtI gene) created by the John Hopkins iGEM team 2011. The genes encode the two enzymes Geranylgeranyl Diphosphate Synthase and Phytoene Desaturase, respectively,  both from the wildtype strain of xanthophyllomyces dendrorhous used in the biosynthesis of beta-Carotene.
When looking into the specifications of these BioBricks, we realized that both genes contained illegal restriction sites.
BBa_K530001 contained an AgeI restriction site making the part incompatible with the RFC25 Freiburg Standard. The RFC25 standard allows for in-frame assembly of protein domains. BBa_K530002 contained the illegal restriction site BglII making the part incompatible with the RFC21 Berkely Standard, which enables in-frame assembly of proteins.
We wanted to overcome these obstacle by removing the illegal restriction sites from the above described Biobricks.
How did we do it?
The illegal restriction sites were removed using site directed mutagenesis with primers containing nucleotide substitutions in the two restriction sites (see Figure 1).
Figure 1: Primers overlapping the restriction sites are designed with a single nucleotide change to disrupt the restriction site. The primers anneal to the template plasmid and replicate while introducing the point mutation. The elongated plasmids are digested with the enzyme DpnI, which cleaves at the methylated sites breaking down the circular template, resulting in a higher transformation efficiency of the linear PCR product.
×
DESCRIPTION
Table 1: Primers designed for removal of illegal restriction sites. Bold marks the nucleotide  substitution.
×
DESCRIPTION
The new plasmid, BBa_K2117012 , with the removed AgeI restriction site was double digested with the enzyme and SpeI to test if we had successfully removed the AgeI restriction site.
Figure 2: AgeI + SpeI digestion. Electrophoresis on a 1% agarose gel showing the digestion of BBa_K2117012 (lane 1-3). BBa_K530001 digested with AgeI + SpeI and undigested BBa_K530001 were used as controls (lane 4+5).
×
DESCRIPTION
h
The digestion showed the removal of the AgeI restriction site, shown on the gel picture by only one band on the BBa_K2117012 compared to two bands on the control BBa_K530001 .
BBa_K2117012 and BBa_K2117013 were sent for sequencing with the verification primers VR and VF2 primers to further verify the removal of the restriction sites.
Figure 3: Sequencing alignment of BBa_530001 (top sequence) and BBa_K2117012 (bottom sequence). The alignment shows a nucleotide substitution in the AgeI restriction site.
×
DESCRIPTION
Figure 4: Sequencing alignment of BBa_530002 (top sequence) and BBa_K2117013 (bottom sequence). The alignment shows a nucleotide substitution in the BglII restriction site.
×
DESCRIPTION
The sequencing results show a substitution in the restriction sites corresponding to the nucleotide substitution designed in the primers (Table 1).
By deleting the two restriction sites in the BBa_K2117012 and BBa_K2117013 we have made the two BioBricks compatible with RFC25 and RFC21 standards, respectively.
Improvement of Characterization
While working with BBa_K530001 and BBa_K530002 we were struggling with assembly of the genes. Therefore, we sequenced the plasmids received in the distribution kit.
Sequencing results of BBa_K530002 showed nucleotide substitutions in the prefix and a large deletion in the suffix.
Figure 5: Alignment of BBa_K530002 sequence received from the parts page (top) and sequencing results of BBa_K530002 from the distribution kit. Alignment shows many nucleotide substitutions in the prefix seuqence.
×
DESCRIPTION
Figure 6: Alignment of BBa_K530002 sequence received from the parts page (top) and sequencing results of BBa_K530002 from the distribution kit. Alignment shows a deletion in the suffix sequence.
×
DESCRIPTION
The alterations in prefix and suffix make the BBa_K530002 incompatible with the BioBrick standard
Parts list
Improvement and Characterization
FIND US AT:
DTU BIOBUILDERS
DENMARK
DTU - SØLTOFTS PLADS, BYGN. 221/002
2800 KGS. LYNGBY
E-mail:
dtu-biobuilders-2016@googlegroups.com
MAIN SPONSORS:
