<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_DTU-Denmark_molecular_toolbox skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:DTU-Denmark/molecular toolbox</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>New HTML template for the wiki</TITLE></P><DIV id="headerWrapper"><DIV id="Topnav" class="container"><DIV id="logo"><BUTTON id="hamburger" class="mobile-menue">☰</BUTTON><BUTTON id="arrow" class="mobile-menue">⇦</BUTTON><BUTTON id="a-down" class="mobile-menue mobile-scrole">Jump to</BUTTON></DIV><DIV id="navigation"><UL id="nav-items"><LI><A href="https://2016.igem.org/Team:DTU-Denmark/project" class="project nav-item" id="project">Project</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/wetlab" class="wetlab nav-item" id="wetlab">Wetlab</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/compute" class="compute nav-item" id="compute">Compute</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/team" class="team nav-item" id="team">Team</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/outreach" class="outreach nav-item" id="outreach">Outreach</A></LI></UL></DIV></DIV><DIV id="sub-navigation-wrapper"><DIV id="sub-navigation" class="container"><UL id="project-sub"><LI><A href="https://2016.igem.org/Team:DTU-Denmark/description" class="sub-nav-item">Description</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Proof" class="sub-nav-item">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/achievements" class="sub-nav-item">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Notebook" class="sub-nav-item">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Safety" class="sub-nav-item">Safety</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Description" class="sub-nav-item">Parts</A></LI></UL><UL id="wetlab-sub"><LI><A href="https://2016.igem.org/Team:DTU-Denmark/substrate" class="sub-nav-item">Substrate</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/products" class="sub-nav-item">Products</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/molecular_toolbox" class="sub-nav-item">Molecular Toolbox</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/molecular_interlab" class="sub-nav-item">Interlab</A></LI></UL><UL id="compute-sub"><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Software" class="sub-nav-item">Codon Optimization</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Hardware" class="sub-nav-item">Microfermenter</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Model" class="sub-nav-item">Modeling</A></LI></UL><UL id="team-sub"><LI><A href="https://2016.igem.org/Team:DTU-Denmark/team_members" class="sub-nav-item">Team Members</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/team_building" class="sub-nav-item">Team Building</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Attributions" class="sub-nav-item">Attributions</A></LI></UL><UL id="outreach-sub"><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Collaborations" class="sub-nav-item">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/HP/Silver" class="sub-nav-item">Practices</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/HP/Gold" class="sub-nav-item">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:DTU-Denmark/Engagement" class="sub-nav-item">Biosensor</A></LI></UL></DIV></DIV></DIV><P><TITLE>Bootstrap Example</TITLE></P><DIV class="MYbody"><DIV class="masthead"><DIV class="container-fluid"><DIV class="row"><DIV class="col-md-12 thumbnail" style="background-image:url(https://static.igem.org/mediawiki/2016/c/c2/T--DTU-Denmark--Lab-header.jpg)"><DIV class="caption"><DIV class="col-md-5 col-sm-5 col-xs-12 title"><H1>Molecular toolbox<P class="lead">In the substrate section we established that <I>Yarrowia lipolytica</I> constitutes a great platform for utilizing waste streams. In order to unlock the potential of <I>Y. lipolytica</I>, we developed a molecular toolbox which allows for efficient genetic engineering of <I>Y. lipolytica</I>. In this section we present the theory and results of the development of a BioBrick backbone and CRISPR tools for <I>Y. lipolytica</I></P></H1></DIV><DIV class="col-md-5 col-sm-5 hidden-xs intro"><BLOCKQUOTE class="blockquote-reverse"><P>&quot;Man is a tool-using animal. Without tools he is nothing, with tools he is all&quot;</P><SMALL>Thomas Carlyle <CITE title="Source Title">Signs of the Times (1829)</CITE></SMALL></BLOCKQUOTE></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-md-9 col-sm-10 colLeft"><DIV><H2 class="h2">pSB1A8YL - The BioBrick Plasmid</H2><H3 class="h3">Introduction</H3><P>
                A key part of synthetic biology is to streamline the process of engineering biological systems, by standardizing parts and methods<SUP><A href="#references">1</A></SUP>. Perhaps the most versatile standards available is the BioBrick standard, in part due to the contributions made during the annual iGEM competition. The BioBrick registry currently has over 20,000<SUP><A href="#references">2</A></SUP> BioBricks, and by creating BioBrick plasmid backbones compatible with a new organism, one is effectively unlocking the entire BioBrick registry available for that specific organism. Realizing this, it was decided to develop a plasmid that supports the BioBrick standard and replicates in <I>Yarrowia lipolytica</I>. Due to the convenience of manipulating <I>Escherichia coli</I> it was determined to develop a shuttle vector that allows for cloning and confirmation of the construct in <I>E. coli</I>, before the construct is transformed in <I>Y. lipolytica</I>. Additionally, as the only replicative plasmids currently available for <I>Y. lipolytica</I> are low copy yeast chromosomal plasmids (YCp)<SUP><A href="#references">3</A></SUP> our plasmid allows for high amounts of deoxyribonucleic acid (DNA) to easily be propagated in <I>E. coli</I>, before the the plasmid is purified and transformed into <I>Y. lipolytica</I>. Figure 1 shows a suggested workflow for the proposed BioBrick plasmid.
              </P><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 1:</STRONG> The suggested workflow using pSB1A8YL described above. The workflow is broken down to three distinct steps: cloning, confirmation and expression. Cloning is the assembly of the construct, which is conducted in <I>E. coli</I> using typical cloning methods. Confirmation is where the identity of the construct is confirmed, using analytical methods such as polymerase chain reaction (PCR), analytical restriction enzyme digestion and/or sequencing. Expression includes the work done in the final host organism, <I>Y. lipolytica</I>, and includes inserting the construct and optimization of the expression and cultivation conditions.</FIGCAPTION></FIGURE><DIV id="img1Modal" class="modal"><SPAN class="close img1">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV><H3 class="h3">Design</H3><DIV class="col-md-6"><P>
                  For the design of the plasmid, we decided to incorporate a high copy <I>E. coli</I> part for cloning and propagation of DNA. The design was based on the pUC19 vector as it fulfills the criteria of being high copy<SUP><A href="#references">4</A></SUP>, while perhaps being one of the most widely used cloning vectors for <I>E. coli</I>. To support the BioBrick standard, we only used the ampicillin resistance and replication origin elements of the plasmid. It was found that the sequence in and between these elements did not contain any restriction sites of any current BioBrick assembly standard, thus no further modification of the sequence was needed.
                </P><P>
                  For the <I>Y. lipolytica</I> part of the plasmid we decided to base the design on the pSL16-CEN1-1(227), as it has been found to exhibit high transformation efficiency compared to similar plasmids<SUP><A href="#references">5</A></SUP>, and perhaps for this reason this plasmid and its derivatives are utilized in many recent studies.<SUP><A href="#references">6,7,8</A></SUP> Again, only the sequences of the replicative and selective elements were chosen. Although, the original sequence was not BioBrick compatible, and thus it was decided to order the sequence as a gBlock. This also introduced the added benefit of being able to incorporate the BioBrick prefix, suffix and a 5’ terminator in the gBlock and exchange the original leucine autotrophy marker with a uracil autotrophy marker allowing for negative selection of the plasmid with 5-Fluoroorotic Acid (5-FOA)<SUP><A href="#references">9</A></SUP>. In order to comply with the iGEM plasmid nomenclature<SUP><A href="#references">10</A></SUP>, the plasmid was dubbed “pSB1A8YL”, YL was added in the end to emphasize that the plasmid is used for <I>Y. lipolytica</I>. Figure 2 shows a graphical representation of the sequence map of pSB1A8YL.
                </P></DIV><DIV class="col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 2:</STRONG> Sequence map of pSB1A8YL. The colored blocks represents the following: Orange: pUC19 part, Blue modified pSL16-CEN1-1(227) part, pink: BioBrick prefix, purple: BioBrick suffix, red: terminator, green selection markers, grey: origin of replication.  The full annotated sequence can be found <A href="http://parts.igem.org/Part:BBa_K2117009">HERE</A>. </FIGCAPTION></FIGURE><DIV id="img2Modal" class="modal"><SPAN class="close img2">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV></DIV><H3 class="h3">Cloning</H3><P>
                The pUC19 part and the gBlock fragments was amplified using primers with USER tails, and fused using USER cloning (See Figure 3).
              </P><P>
                The USER reactions was transformed into chemically competent <I>E. coli</I> DH5α cells, and purified. The identity of the product was checked using PCR, restriction analysis (see Figure 4) and sequencing (data not shown).
              </P><CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption" style="width:50%;"><STRONG>Figure 3:</STRONG> PCR fragments with Uracil-Specific Excision Reagent (USER) tails, which were fused to construct pSB1A8YL. The fragment lengths can be seen on the ladder. The primers used and the theoretical fragment length can be seen under the bands.</FIGCAPTION></FIGURE></CENTER><DIV id="img3Modal" class="modal"><SPAN class="close img3">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 4:</STRONG> pSB1A8YL undigested and linearized. Analytical PCRs are also included in the right of the gel picture. The fragment lengths can be seen on the ladder. The restriction enzymes and primers used and the theoretical fragment length can be seen under the bands</FIGCAPTION></FIGURE><DIV id="img4Modal" class="modal"><SPAN class="close img4">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV><H3 class="h3">Testing</H3><DIV class="row"><DIV class="col-md-6"><P>
                    After having confirmed the identity of the plasmid, we set out to test its functionality. This was done by: 1. Testing the plasmids replicability and selectivity in <I>Y. lipolytica</I> and 2. testing the plasmids cloning capabilities in <I>E. coli</I>. 
                  </P><H5 class="h5">1. Replicability and selectability in <I>Yarrowia lipolytica</I></H5><P>
                    The pSB1A8YL plasmid was purified from <I>E. coli</I> DH5α, and transformed in <I>Y. lipolytica</I> PO1f cells. The transformants was selected on selective dropout media not containing uracil, thus only yielding uracil autotroph transformants. A negative control was included substituting the plasmid for Milli-Q (MQ) water.
                    The transformations only yielded colonies on the plated containing the cells which were transformed with the plasmid. To ensure that these results indeed meant that our plasmid was stably replicating in the <I>Y. lipolytica</I> cells, a few colonies were subjected to colony PCR (see Figure 5).
                  </P><P>
                    These results confirm that pSB1A8YL replicates in <I>Y. lipolytica</I>, and the chosen uracil selection marker allows for selection of transformants.
                    To further assess the functionality of pSB1A8YL, the possibility of counter selection was investigated. This was done by growing colonies containing pSB1A8YL on plates containing 5-FOA. Colonies appearing on these plates were then transferred to selective dropout media not containing uracil. As no growth was observed on the latter plate, this proved that pSB1A8YL supports counter selection.
                  </P></DIV><DIV class="col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 5:</STRONG> Colony PCR of pSB1A8YL in <I>Y. lipolytica</I>. The fragment lengths can be seen on the ladder. The primers used and the theoretical fragment length can be seen under the bands</FIGCAPTION></FIGURE><DIV id="img5Modal" class="modal"><SPAN class="close img5">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV></DIV></DIV><H5 class="h5">2. Cloning capabilities in <I>E. coli</I></H5><P>
                In order to test this, we decided to produce a device using BioBricks from the distribution kit which would allow us to easily assess whether the clonings were successful. When deciding BioBricks that would allow this, we received inputs from the SDU iGEM team. We ended up choosing the strong Andersson promoter/strong RBS BioBrick (<A href="http://parts.igem.org/Part:BBa_K880005">BBa_K880005</A>) and pair this with three chromoproteins: amilCP (<A href="http://parts.igem.org/Part:BBa_K592009">BBa_K592009</A>), amilGFP (<A href="http://parts.igem.org/Part:BBa_K592010">BBa_K592010</A>) and mRFP (<A href="http://parts.igem.org/Part:E1010">BBa_E1010</A>), which would allow us to easily pick transformants and visually inspect if the cloning was successful. The cloning flow is shown in Figure 6.
              </P><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 6:</STRONG> Cloning flow of the test of pSB1A8YL in <I>E. coli</I>. The expression of the chromoproteins should yield a color ouput detectable by visual inspection.</FIGCAPTION></FIGURE><DIV id="img6Modal" class="modal"><SPAN class="close img6">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV><P>
                The BioBricks were retrieved from the distribution kit, and assembled with our plasmid as carrier backbone using standard 3A assembly and transformed into chemically competent <I>E. coli</I> DH5α cells. The transformants yielded colored colonies (See Figure 7), and the identity of the constructs were confirmed using restriction analysis and PCR (data not shown). These results confirm that pSB1A8YL can be used for cloning in <I>E. coli</I> and supports the BioBrick standard.
              </P><DIV id="img7Modal" class="modal"><SPAN class="close img7">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 7:</STRONG> Actual results from the test of pSB1A8YL in <I>E. coli</I>. Top: A color output was visible both in liquid culutres, pellets and on plates. Bottom: Restriction analysis of the constructs. The fragment lengths can be seen on the ladder. The restriction enzymes used and the theoretical fragment length can be seen under the bands</FIGCAPTION></FIGURE><DIV id="img8Modal" class="modal"><SPAN class="close img8">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV></DIV><DIV><H2 class="h2">CRISPR-Cas9 System</H2><H3 class="h3">Introduction</H3><P>
                By creating a modular plasmid that is compatible with the BioBrick standards and that is able to replicate in both <I>E. coli</I> and <I>Y. lipolytica</I>, the entire biobrick registry has been made available for use in <I>Y. lipolytica</I>. However, one important aspect of industrial-scale protein expression is the stable and homogenous expression of relevant pathway genes. In production organisms with multi-enzyme pathways there is often a need for accurate control of the expression levels of the associated genes, both in terms of promoter strength and copy numbers<SUP><A href="#references">11</A></SUP>. Plasmid based systems for the expression of heterologous genes are widely used, but offer limited control of copy number and in many cases show notable segregational instability even during growth in selective media. Therefore, it is often preferred to integrate the genetic pathway of interest into the genome of the production organism of choice to ensure long-term strain stability and homogenous expression of pathway genes<SUP><A href="#references">11</A></SUP>.
              </P><P>
                The CRISPR-Cas9 system has revolutionized the way in which model organisms can be engineered. In addition, it has made genetic engineering applicable to a wide range of organisms that were previously considered to be uninteresting for cell factory purposes, largely because the genetic engineering tools for these organisms were under characterized. The CRISPR-Cas9 system is ideally suited for the integration of heterologous genes in production organisms, as it relies on Double Strand Break (DSB) induction and its subsequent repair by homologous recombination (HR). In addition, almost any sequence of interest can be targeted with the most commonly used Cas9 from <I>Streptococcus pyogenes</I><SUP><A href="#references">12</A></SUP>. The basic mechanism of CRISPR-Cas9 is illustrated in Figure 8, for a more detailed overview the reader is referred to Nødvig et al.<SUP><A href="#references">12</A></SUP></P><CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 8:</STRONG> Overview of the CRISPR-Cas9 mechanism of DSB induction. The systems consists of two components, the endonuclease Cas9 and a single guide ribonucleic acid (sgRNA). Cas9 is targeted for site-specific DSB induction by the sgRNA. More specifically, a 20 bp region of the sgRNA, known as the protospacer, recognizes the target site through base-pairing. This 20 bp protospacer is easily substituted, allowing for a very modular system. The only restriction for the CRISPR-Cas9 system is the requirement for a Protospacer Adjacent Motif (PAM) in the target region. For the most commonly used Cas9 (from <I>Streptococcus pyogenes</I>) this is the frequently occurring NGG sequence. Reproduced from Nødvig et al.<SUP><A href="#references">12</A></SUP></FIGCAPTION></FIGURE></CENTER><DIV id="img10Modal" class="modal"><SPAN class="close img10">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV><P>
                We adapt the CRISPR-Cas9 system for use in <I>Y. lipolytica</I> in order to obtain a proof-of-concept of the construction of a stable production strain. We do so by a proof-of-concept integration of a gene in <I>Y. lipolytica</I>. In addition, we obtain a proof-of-concept of the knock-out of a native <I>Y. lipolytica</I> gene. With these proof-of-concepts, we envision that the metabolic flux towards a possible compound of interest could be increased, leading to tailored <I>Y. lipolytica</I> strains for cell factory engineering purposes.
              </P><H3 class="h3">Experimental design</H3><P>
                The CRISPR-Cas9 system has previously been successfully applied to <I>Y. lipolytica</I> by Schwartz et al.<SUP><A href="#references">8</A></SUP> As this was a proof-of-concept study, the native <I>Y. lipolytica</I><I>PEX10</I> gene was targeted for disruption, which allowed for an easily selectable phenotype. This is because <I>PEX10</I> codes for Peroxisome biogenesis factor 10, a protein that is involved in peroxisome biogenesis. Consequently, <I>PEX10</I> disruption results in an inability to catabolize long-chain fatty acids. We set out to confirm the observed <I>PEX10</I> knock-out. In addition, we set out to integrate <I>URA3</I>, a commonly used yeast auxotrophic marker gene which codes for Orotidine 5’-phosphate decarboxylase, an enzyme involved in pyrimidine biosynthesis.
              </P><P>
                We had obtained both a commonly used <I>Y. lipolytica</I> laboratory strain (PO1f, <I>MATA</I><I>URA3</I>-­302 <I>LEU2</I>-­270 <I>XPR2</I>-­322 <I>AXP2</I> ­Δ<I>NU49</I><I>XPR2::SUC2</I>; obtained from Schwartz et al.<SUP><A href="#references">8</A></SUP>) and also this same strain with a knockout of the Non-Homologous End-Joining (NHEJ) DNA repair pathway gene <I>Ku70</I> (PO1fΔ<I>Ku70</I>, <I>MATA</I><I>URA3</I>-­302 <I>LEU2</I>-­270 <I>XPR2</I>-­322 <I>AXP2</I> ­Δ<I>NU49</I> Δ<I>Ku70</I><I>XPR2::SUC2</I>; obtained from Schwartz et al.<SUP><A href="#references">8</A></SUP>). The NHEJ pathway for DNA repair is commonly disrupted in laboratory strains to improve the efficiency of CRISPR-Cas9 induced HR integration of heterologous DNA. However, an intact NHEJ pathway can be utilized to disrupt genes without the need for an DNA repair template, as this DNA repair pathway is intrinsically more error-prone than HR. Thus, in order to disrupt the native <I>Y. lipolytica</I><I>PEX10</I> gene, we used a CRISPR-Cas9 plasmid (pCRISPRyl) containing a protospacer that targets <EM>PEX10</EM> in the PO1f strain. For the <I>URA3</I> insertion, we co-transformed the same CRISPR-Cas9 plasmid and a linearized HR donor plasmid (pIW501) in the PO1fΔ<I>Ku70</I> strain. An overview of our workflow is provided in Figure 9. Overviews of pCRISPRyl and pIW501 are provided in Figures 10 and 11, respectively.
              </P><CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 9:</STRONG> Overview of the experimental design for the proof-of-concept of gene disruption and gene integration in <I>Y. lipolytica</I> using CRISPR-Cas9. Left: transformation of pCRISPRyl should result in gene disruption through error-prone repair by NHEJ (bases shown in yellow). Right: co-transformation of pCRISPRyl and a HR donor plasmid (pIW501) should result in integration of the desired donor gene (bases shown in orange). For plasmid map details, see Figures 10 and 11. Adapted from Nødvig et al.<SUP><A href="#references">12</A></SUP></FIGCAPTION></FIGURE></CENTER><DIV id="img11Modal" class="modal"><SPAN class="close img11">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV><DIV class="col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 10:</STRONG> Simplified map of pCRISPRyl. Cas9 is under control of a strong constitutive TEF-derived hybrid promoter and the strong CYC1 terminator. Replication origins for use in <I>E. coli</I> (<EM>ori</EM>) and <I>Y. lipolytica</I> (Cen1-1) &amp; ORI1001) are shown in yellow. Selection markers for use in <I>E. coli</I> (<I>AmpR</I>) and <I>Y. lipolytica</I> (<I>LEU2</I>) are shown in cyan. The synthetic tRNA promoter for sgRNA expression is shown in purple. A novel protospacer can be inserted by linearization of the plasmid with AvrII and subsequent Gibson assembly of a 60 bp fragment containing the desired protospacer into the plasmid. in our case, a protospacer that targets <EM>PEX10</EM> was inserted<SUP><A href="#references">8</A></SUP>. Obtained from Addgene (#70007).</FIGCAPTION></FIGURE><DIV id="img12Modal" class="modal"><SPAN class="close img12">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV></DIV><DIV class="col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 11:</STRONG> Simplified map of pIW501. The HR donor fragment consists of two flanking regions of 1 kb (shown in grey) and the <I>URA3</I> promoter+CDS+terminator (only the CDS is depicted in purple). The flanking regions target the <I>PEX10</I> locus. The replication origin for use in <I>E. coli</I> (<EM>ori</EM>) is shown in yellow, the selection marker for use in <I>E. coli</I> (<I>AmpR</I>) is depicted in cyan. Prior to transformation in <I>Y. lipolytica</I>, the plasmid is linearized with BamHI in order to increase HR efficiency. Obtained from Schwartz et al.<SUP><A href="#references">8</A></SUP></FIGCAPTION></FIGURE><DIV id="img13Modal" class="modal"><SPAN class="close img13">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV></DIV><H3 class="h3">Results &amp; Discussion</H3><H5 class="h5">1. <I>URA3</I> insertion</H5><P>
                In order to integrate the <I>URA3</I> gene into the genome of the <I>Y. lipolytica</I> PO1fΔ<I>Ku70</I> strain, we first constructed a pCRISPRyl-derived plasmid which contained a protospacer that targets the <I>PEX10</I> locus. This protospacer was used previously for successful disruption of <I>PEX10</I><SUP><A href="#references">8</A></SUP>. The pCRISPRyl-derived plasmid (pIW357) was constructed through the Gibson assembly of a 60 bp fragment containing the protospacer of interest into the AvrII linearized pCRISPRyl. Subsequently, pIW357 was purified from selected <I>E. coli</I> DH5α transformants and a double restriction analysis was performed with PstI and AvrII (Figure 12). As the purified pIW357 was no longer digested with AvrII, the protospacer had been correctly inserted. In addition, the protospacer insertion was confirmed through Sanger sequencing (data not shown).  
              </P><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 12:</STRONG> Restriction analysis of pCRISPRyl (lanes 2-3) and pIW357 (lanes 5-8) (left to right). Lane 2: pCRISPRyl digested with PstI. Lane 3: pCRISPRyl digested with PstI and AvrII. Lane 5-8: pIW357 digested with PstI and AvrII (4 unique transformants/plasmids). As pIW357 was digested solely by PstI and not by AvrII, it was concluded that the Gibson assembly had been successful and the protospacer had been correctly inserted.</FIGCAPTION></FIGURE><DIV id="img14Modal" class="modal"><SPAN class="close img14">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV><P>
                Next, we tested pIW357 in the <I>Y. lipolytica</I> PO1fΔ<I>Ku70</I> strain. In order to account for Cas9 activity of pIW357, we also transformed the pCRISPRyl as a positive control. The results of these transformations are depicted in Figure 13. With an intact NHEJ repair pathway and/or a template for HR, the induced DSB should easily be repaired. However, as this is the PO1fΔ<I>Ku70</I> strain and as there is no HR donor template available, the induced DSB should be lethal. Based on the difference in transformant counts between the pIW357 and pCRISPRyl transformations, pIW357 is functioning properly. The observed residual growth in the pIW357 transformation can be attributed to either mutations in pIW357 (for example in Cas9 or in the the sgRNA/tRNA promoter) or to possible repair by the Microhomology-Mediated End-Joining (MMEJ) pathway<SUP><A href="#references">14</A></SUP>.
              </P><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 13:</STRONG> Transformation of pIW357 and pCRISPRyl into the <I>Y. lipolytica</I> PO1fΔ<I>Ku70</I> strain. pIW357 (A), pCRISPRyl (B), control (C), plated on SC-leu media.</FIGCAPTION></FIGURE><DIV id="img15Modal" class="modal"><SPAN class="close img15">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV><P>
                With a functioning pIW357, we then co-transformed pIW357 and pIW501 (linearized) into the <I>Y. lipolytica</I> PO1fΔ<I>Ku70</I> strain (Figure 14). The observation of growth on SC-ura media indicates that the <I>URA3</I> gene had been successfully integrated. Growth was also observed when solely the linearized pIW501 was transformed into <I>Y. lipolytica</I> PO1fΔ<I>Ku70</I>. In addition, transformants from the co-transformation of pIW357 and pIW501 were screened for <I>URA3</I> integration through gap-check colony PCR (Figure 15). The observed shift in band size between non-pIW501 transformants and pIW357/pIW501 co-transformants corresponds to the 1700 bp size of the <I>URA3</I> promoter+CDS+terminator sequence that is present on pIW501. 
              </P><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 14:</STRONG> Co-transformation of pIW357 and linearized pIW501 into the Y. lipolytica PO1fΔKu70 strain. pIW357+linearized pIW501 (A), linearized pIW501 (B), control (C), plated on SC-ura selective media.</FIGCAPTION></FIGURE><DIV id="img16Modal" class="modal"><SPAN class="close img16">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 15:</STRONG> Gap-check colony PCR of <I>URA3</I> integration in the <I>PEX10</I> locus of the <I>Y. lipolytica</I> PO1fΔ<I>Ku70</I> strain. The used gap-check primers anneal slightly upstream and downstream of the 1 kb <I>PEX10</I> upstream and downstream flanking regions, respectively. This allows for a direct comparison of the amplicon size between a control transformant and a transformant which should contain the insert in the same PCR reaction. Although only 3 screened co-transformation transformants are depicted in this Figure, an efficiency of 100% was achieved (12/12). Forward primer: TGAGCGAAACGTCATCTACG. Reverse primer: AGCAGTGAAAAGTCGGGCTA.</FIGCAPTION></FIGURE><DIV id="img17Modal" class="modal"><SPAN class="close img17">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV><P>
                The pIW501 is linearized prior to transformation in order to increase HR efficiency and in order to reduce the amount of false positive transformants (a gene cannot be expressed from a linear DNA fragment). However, growth was also observed when solely the linearized pIW501 is transformed into <I>Y. lipolytica</I> PO1fΔ<I>Ku70</I>, indicating that CRISPR-Cas9 DSB induction is not necessarily required for the integration of an auxotrophic marker gene. We did not have time to compare the efficiencies of co-transformation of pIW357 and pIW501 as compared to solely pIW501 transformation. However, it has been shown previously that co-transformation with a CRISPR-Cas9 plasmid increases the HR efficiency<SUP><A href="#references">12</A></SUP>. We thus hypothesize that the integration of the <I>URA3</I> gene was facilitated by pIW357. We also did not have time to test the integration of a non-marker gene (which would still allow for an easily selectable phenotype such as <I>GFP</I> or <I>lacZ</I>). This might have been a better option, as the efficiency of non-CRISPR-Cas9 induced HR is lower for a non-marker gene. This would have allowed us to more easily distinguish between CRISPR-Cas9-mediated and non-CRISPR-Cas9-mediated gene integration.
              </P><H5 class="h5">2. <I>PEX10</I> knock-out</H5><P>
                In order to disrupt <I>PEX10</I>, we used the same pCRISPRyl-derived plasmid (pIW357) as described above. However, instead of using the <I>Y. lipolytica</I> PO1fΔ<I>Ku70</I> strain, we used the <I>Y. lipolytica</I> PO1f strain, as we were aiming for <I>PEX10</I> disruption through error-prone NHEJ (Figure 9). We thus transformed pIW357 into our selected strain and screened transformants for the inability to grow on a medium that contained solely long-chain fatty acids as a carbon source (SC-oleic acid). In turn, the same transformants were screened on a rich YPD medium which would allow ample growth. However, although we have confirmed that pIW357 functions appropriately (Figure 13), we could not yet identify transformants that were unable to grow on SC-oleic acid (data not shown). Nonetheless, following several rounds of restreaking, transformants were observed that had a distinct morphology (Figure 16). As <I>PEX10</I> disruption can also be the cause of this morphology change (C.M. Schwartz, personal communication, October 18, 2016), transformants with an extraordinary morphology were selected for further analysis by colony PCR. Subsequent Sanger sequencing of the PCR product confirmed that <I>PEX10</I> had been succesfully disrupted (Figure 17). 
              </P><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 16:</STRONG> Restreaked pIW357 transformants in the <I>Y. lipolytica</I> PO1f strain. Overview (A) and detailed view (B), plated on YPD. The irregular, rough morphology is typical for normal <I>Y. lipolytica</I> transformants. In contrast, the smooth transformants are associated with a knock-out of <I>PEX10</I>.</FIGCAPTION></FIGURE><DIV id="img18Modal" class="modal"><SPAN class="close img18">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 17:</STRONG> Sanger sequencing of the colony PCR product from <I>Y. lipolytica</I> PO1f colonies with a distinct morphology. In all cases (3/3) an 8 bp deletion in <I>PEX10</I> was observed.</FIGCAPTION></FIGURE><DIV id="img19Modal" class="modal"><SPAN class="close img19">×</SPAN><DIV class="caption">DESCRIPTION</DIV></DIV></DIV><DIV><H2 class="h2">Conclusion</H2><P>
                We successfully developed a shuttle vector that allows the user to harvest the efficiency and accessibility of cloning in <I>E. coli</I>, while still allowing for replication in <I>Y. lipolytica</I>. We extensively tested the plasmid, and showed that it allows for replication, selection in <I>E. coli</I> and replication, selection and even counterselection in <I>Y. lipolytica</I>. The plasmid was shown to be compatible with BioBricks which, to our knowledge, makes it the first tool that allows for the use BioBricks in <I>Y. lipolytica</I>. It is our hope that other actors in the iGEM community will use this tool in order harvest the great potentials of <I>Y. lipolytica</I> as a chassis for biorefineries of the future.
              </P><P>
                As a proof-of-concept, we have shown that we can integrate a gene of interest into <I>Y. lipolytica</I> and also disrupt a native gene of interest. These proof-of-concept experiments validate the use of <I>Y. lipolytica</I> for future cell factory engineering purposes. 
              </P><P>
                Here, we have focused on the integration of an auxotrophic marker gene. However, the CRISPR-Cas9 system is ideally suited for markerless gene integrations<SUP><A href="#references">13</A></SUP>. Markerless genetic modifications are advantageous as they do not affect host cell physiology and allow for iterative genetic modification cycles of the production organism of interest. In addition, this iterative process can be sped up as the modular nature of the CRISPR-Cas9 system makes it ideally suited for simultaneous multiplex genome editing<SUP><A href="#references">13,15</A></SUP>.
              </P></DIV><DIV id="ref_sec"><H2 class="h2">References</H2><OL><LI>Shetty, R. P., Endy, D., &amp; Knight, T. F. (2008). Engineering BioBrick vectors from BioBrick parts. Journal of Biological Engineering, 2(1), 5. article. http://doi.org/10.1186/1754-1611-2-5</LI><LI><A href="http://parts.igem.org/Collections">iGEM collections</A></LI><LI>Liu, L., Otoupal, P., Pan, A., &amp; Alper, H. S. (2014). Increasing expression level and copy number of a Yarrowia lipolytica plasmid through regulated centromere function. Fems Yeast Research, 14(7), 1124–1127. doi:10.1111/1567-1364.12201</LI><LI>Yanisch-Perron, C., Vieira, J., &amp; Messing, J. (1984). Improved M13 phage cloning vectors and host strains: nucleotide sequences of the M13mpl8 and pUC19 vectors. Gene, 33(33), 103–119. doi:10.1016/0378-1119(85)90120-9</LI><LI>Yamane, T., Sakai, H., Nagahama, K., Ogawa, T., &amp; Matsuoka, M. (2008). Dissection of centromeric DNA from yeast Yarrowia lipolytica and identification of protein-binding site required for plasmid transmission. Journal of Bioscience and Bioengineering, 105(6), 571–578. doi:10.1263/jbb.105.571</LI><LI>Liu, L., Otoupal, P., Pan, A., &amp; Alper, H. S. (2014). Increasing expression level and copy number of a Yarrowia lipolytica plasmid through regulated centromere function. Fems Yeast Research, 14(7), 1124–1127. doi:10.1111/1567-1364.12201</LI><LI>Blazeck, J., Liu, L., Redden, H., &amp; Alper, H. (2011). Tuning Gene Expression in Yarrowia lipolytica by a Hybrid Promoter Approach. Applied and Environmental Microbiology, 77(22), 7905–7914. doi:10.1128/AEM.05763-11</LI><LI>Schwartz, C. M., Hussain, M. S., Blenner, M., &amp; Wheeldon, I. (2016). Synthetic RNA Polymerase III Promoters Facilitate High-Efficiency CRISPR-Cas9-Mediated Genome Editing in Yarrowia lipolytica. Acs Synthetic Biology, 5(4), 356–359. doi:10.1021/acssynbio.5b00162</LI><LI>Sakaguchi, T., Nakajima, K., &amp; Matsuda, Y. (2011). Identification of the UMP Synthase Gene by Establishment of Uracil Auxotrophic Mutants and the Phenotypic Complementation System in the Marine Diatom Phaeodactylum tricornutum. Plant Physiol, 156(1), 78–89.</LI><LI><A href="http://parts.igem.org/Help:Plasmid_backbones/Nomenclature">iGEM nomenclature</A></LI><LI>Ronda, C., Maury, J., Jakočiu̅nas, T., Baallal Jacobsen, S. A., Germann, S. M., Harrison, S. J., Borodina, I., Keasling, J. D., Jensen, M. K., &amp; Nielsen, A. T. (2015). CrEdit: CRISPR mediated multi-loci gene integration in Saccharomyces cerevisiae. Microbial Cell Factories, 14(1), 97. http://doi.org/10.1186/s12934-015-0288-3</LI><LI>Nødvig, C. S., Nielsen, J. B., Kogle, M. E., &amp; Mortensen, U. H. (2015). A CRISPR-Cas9 system for genetic engineering of filamentous fungi. PLoS ONE, 10(7), 1–18. http://doi.org/10.1371/journal.pone.0133085</LI><LI>Jessop-Fabre, M. M., Jakočiūnas, T., Stovicek, V., Dai, Z., Jensen, M. K., Keasling, J. D. &amp; Borodina, I. (2016), EasyClone-MarkerFree: A vector toolkit for marker-less integration of genes into Saccharomyces cerevisiae via CRISPR-Cas9. Biotechnology Journal, 11: 1110–1117. doi:10.1002/biot.201600147</LI><LI>Sakuma, T., Nakade, S., Sakane, Y., Suzuki, K.-I. T., &amp; Yamamoto, T. (2015). MMEJ-assisted gene knock-in using TALENs and CRISPR-Cas9 with the PITCh systems. Nature Protocols, 11(1), 118–133. http://doi.org/10.1038/nprot.2015.140</LI><LI>Gao, S., Tong, Y., Wen, Z., Zhu, L., Ge, M., Chen, D., Jiang, S., &amp; Yang, S. (2016). Multiplex gene editing of the Yarrowia lipolytica genome using the CRISPR-Cas9 system. Journal of Industrial Microbiology &amp; Biotechnology, 43(8), 1085–1093. http://doi.org/10.1007/s10295-016-1789-8</LI></OL></DIV></DIV><DIV class="col-md-3 col-sm-2 colRight" id="scrollspy"><UL class="nav" id="sidebar"><LI><A href="#section-1">pSB1A8YL - The BioBrick Plasmid</A></LI><LI><A href="#section-2">CRISPR-Cas9 System</A></LI><LI><A href="#section-3">Conclusion</A></LI></UL></DIV></DIV></DIV></DIV><FOOTER id="footer"><DIV class="container"><DIV align="center" class="col-md-3"><UL><LI>FIND US AT:</LI></UL><UL><LI><U>DTU BIOBUILDERS</U></LI><LI>DENMARK</LI><LI>DTU - SØLTOFTS PLADS, BYGN. 221/002</LI><LI>2800 KGS. LYNGBY</LI><LI><U>E-mail:</U></LI><LI>dtu-biobuilders-2016@googlegroups.com</LI></UL></DIV><DIV align="center" class="col-md-6"><UL><LI>MAIN SPONSORS:</LI></UL></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>