Team:DTU-Denmark/Notebook
New HTML template for the wiki
☰
⇦
Jump to
Project
Wetlab
Compute
Team
Outreach
Description
Proof of Concept
Achievements
Notebook
Safety
Parts
Substrate
Products
Molecular Toolbox
Interlab
Codon Optimization
Microfermenter
Modeling
Team Members
Team Building
Attributions
Collaborations
Practices
Integrated Practices
Biosensor
Bootstrap Example
Notebook
In this section you can follow our progress in the different subgroups week for week. Some weeks are quite, while some weeks are very hectic - especially right up to the wiki freeze.
"Every great love starts with a great story..."
Nicholas Sparks,
The Notebook
June
"Every great love starts with a great story..."
Nicholas Sparks,
The Notebook
Week 1 (May 30 - June 5)
Wetlab
Yarowia lipolytica PO1f Δku70 was obtained from Cory M. Schwartz, cultivated and freeze stocked for future use.
Compute
Hardware
Our Arduino starter kits arrived! Make an LED blink. That’s how it begins.
Week 2 (June 6 - June 12)
Wetlab
Molecular Toolbox
CRISPR-Cas9 induced PEX10 knockout
Y. lipolytica PO1f genome sequence was annotated and protospacer for targeting PEX10 was designed and ordered from IDT.  Protospacer for Gibson Assembly with CRISPRyl plasmid (Addgene plasmid #70007) SCR1'- tRNAGly (bold), Protospacer (underlined), sgRNA (italic)
5'-GGGTCGGCGCAGGTTGACGTGTACAAGGAGGAGCTGGAGAGTTTTAGAGCTAGAAATAGC-3'
Oligos designed to amplify a 1kb region upstream and downstream PEX10 and anneal together by fusion PCR were also ordered from IDT.
CRISPR-Cas9 induced URA3 insertion
The Y. lipolytica PO1f genome sequence was annotated and uploaded to Benchling for sgRNA design. sgRNAs targeting the SUC2 gene were designed. Primers were designed that amplify the functional URA3 gene including 1 kb upstream and downstream flanking regions.
pSB1A8YL
Ran a bunch of PCRs to amplify the pUC19 part of our plasmid, but it’s not working - nothing but smear. Tried to transform the pUC19 plasmid into Escherichia coli.
Substrates
We did an initial experiment determining the full growth cycle of Y. lipolytica W29. This will be used to plan and time the following growth experiments.                               Waste glycerol from the industrial biodiesel producer DAKA is acquired for late screening.
Compute
Hardware
We started building light sensors using photoresistors. Shortlisting ideas for our final project:                                - A microtiter plate reader                               - Hack a printer to build a membrane homogenizer                               - Chemostat bioreactor
Week 3 (June 13 - June 19)
Wetlab
Molecular Toolbox
CRISPR-Cas9 induced PEX10 knockout
Genomic DNA from Y. lipolytica PO1f Δku70 and Y. lipolytica W29 was purified. PEX10 flanking regions were successfully amplified from Y. lipolytica PO1f Δku70.
CRISPR-Cas9 induced URA3 insertion
Genomic DNA from Y. lipolytica PO1f Δku70 and Y. lipolytica W29 was purified. PCR attempts to amplify URA3 and flanks failed. sgRNAs targeting the SUC2 gene were hybridized.
pSB1A8YL
Purified the plasmid from the transformants and use this as template for PCR, although it’s still not giving any bands.
Substrates
We did initial growth experiments on minimal media with an array of different carbon sources. This experiment was discarded due to lack of repeats and wrong vitamin solution for minimal media. Waste from canola oil production by Grønninggaard is acquired. Molasses from Dansukker sugar production is acquired.
Compute
Hardware
Exploring the Arduino IDE and all the electronic components we ordered. There is so much to learn.
Week 4 (June 20 - June 26)
Wetlab
Molecular Toolbox
CRISPR-Cas9 induced URA3 insertion
PCR attempts to amplify URA3 + flanks from Y. lipolytica W29 genomic DNA failed. New primers were ordered.
pSB1A8YL
We realized that the name of the primer had been mixed up! Now that the right primers are used, we get excellent bands on our gel… Guess you have to make the stupid mistakes in the beginning?                                   The gBlock containing the other part of the plasmid also arrived. This also gives excellent bands on the gel when amplifying it by PCR.                                   Ran the first USER and transformed E. coli cells. The transformants were left on the bench over the weekend
Substrates
We have data from the first successful growth experiment. Starch, Xylose, Arabinose, Maltose and Lactose are not suitable for Y. lipolytica fermentation. This will be repeated next week to make sure.                               Waste glycerol from the industrial biodiesel producer Perstop is acquired.
Week 5 (June 27 - July 3)
Wetlab
Molecular Toolbox
CRISPR-Cas9 induced PEX10 knockout
Received CRISPRyl plasmid (Addgene plasmid #70007). The procedure from Addgene was followed.
CRISPR-Cas9 induced URA3 insertion
Successful amplification of URA3 + flanks from Y. lipolytica W29 genomic DNA. Because of low quality of the genomic DNA, the initial PCR product was taken for further amplification.                                    Received CRISPRyl plasmid (Addgene plasmid #70007). The procedure from Addgene was followed.
pSB1A8YL
YES, colonies! Colonies were picked and used for colony PCR, but it was not successful. We’ll just have to crank on!                                   - Colonies from the same plates were re streaked and plasmids were purified from the resulting colonies.                                   - Restriction analysis yielded weird bands.                                   - This week passes restreaking colonies to yield pure colonies and trying to find the correct transformants through by purifying the plasmids and subjecting it to analytical digestion. So far no luck!
Substrates
Repeated positive results with growth on glucose, glycerol, fructose, sucrose and oil. Y. lipolytica should not be able to grow on sucrose. The experiments on starch, xylose, arabinose, Maltose and Lactose are still negative for Y. lipolytica.                               Waste glycerol from the industrial biodiesel producer Emmelev is acquired.
Compute
Genome Scale Modeling
Planning of Genome-scale modelling strategies began, decided to attempt media optimization using phenotype phase plane, team starts to research and learn FBA for GSM.
Software
Initiation of task by designing the workflow needed to achieve the final purpose of the software. Tasks agreed upon discussion : script in python , number and format proxy of the input files needed , restriction site implementation , development GUI, gui library for python (tkinter)
Hardware
Still playing.
July
Week 6 (July 4 - July 10)
Wetlab
Molecular Toolbox
CRISPR-Cas9 induced PEX10 knockout
Purification of pCRISPRyl plasmid from O/N cultures.  Successful digestion of CRISPRyl plasmid with restriction enzymes AatII and NdeI to verify the plasmid.                                    Gibson Assembly of digested CRISPRyl plasmid and protospacers.
CRISPR-Cas9 induced URA3 insertion
Purification of pCRISPRyl.                                     Successful restriction analysis of pCRISPRyl with AatII and NdeI to verify the plasmid.                                   Gibson assembly of pCRISPRyl and the hybridized sgRNAs targeting the SUC2 gene.
pSB1A8YL
YES! Finally a construct that seems to have the correct length! Both the analytical digestion and PCRs seems to confirm our construct.                                   The construct was also sent for sequencing.                                   We spent some time trying to figure out how to test the plasmid. We ended up retrieving the BBa_K592009, BBa_K592010, BBa_E1010 and BBa_J23110 parts from the distribution kit, and pair them. The idea is that if we are able to make a construct in our backbone, we should see a visual output.                                   Unfortunately we were not able to retrieve the BBa_K592010 from the distribution kit, and it was decided to leave this for now.
Substrates
Contamination of experiments was determined with microscopy. Might come from the minimal media being contaminated.                               First test on complex glycerol based media. Y. lipolytica seems to grow better than Saccharomyces cerevisiae.                               The growth form on different C-sources is analysed with microscopi. There seem to be different amounts of planktonic and filamentous growth depending on energy source.
Compute
Software
Implementation of algorithm already started. The script is being written in Python3 with the intention to be easily modifiable so no external packages are needed although Anaconda is being used.
Week 7 (July 11 - July 17)
Wetlab
Molecular Toolbox
CRISPR-Cas9 induced PEX10 knockout
The assembled CRISPRyl plasmid with PEX10 protospacers was transformed in competent E. coli DH5α cells.                                   Purification of assembled CRISPRyl plasmid with Pex10 protospacers.                                   The assembled CRISPRyl plasmid with PEX10 protospacers was digested with AvrII to test if the PEX10 protospacer was inserted. The digestion was successful and showed no cutting by AvrII indicating correct insertion of PEX10 protospacers.                                   First attempt of transformation with CRISPRyl plasmid with PEX10 protospacers in Yarowia lipolytica PO1f Δku70. The transformation was unsuccessful. There was growth all over the plates.
CRISPR-Cas9 induced URA3 insertion
Confirmation of successful Gibson assembly through restriction analysis.                                    First (unsuccessful) transformation of pCRISPRyl and pCRISPRyl+sgRNAs into Yarowia lipolytica PO1f Δku70.
Products
Beta-Carotene
BioBricks BBa_K152005 (crtE,crtI,crtYB,gfp) and BBa_K530002 (crtI) were received from the registry. E. coli with BBa_K152005 could only grow without Ampicilin, so we suspect a problem with the AmpR backbone. We decided not to use this biobrick and instead only work with BBa_K530002, BBa_K530000 and BBa_K530001.
Proinsulin
Designing and ordering synthetic gBlock from IDT encoding TEF1, proinsulin and GFP.
Substrates
Test shows no growth on samples of wastewater from areas contaminated with leftovers of organic molecules from fossil oil. Experiments with glycerol byproduct from Emmelev biodiesel production showed to be contaminated.                               We investigated where the contaminations could come from and a contaminated pipet seemed to be the source.
Compute
Hardware
The library for our LCD Display is commented in Chinese. Shoutout to James for translating. Chris, our supervisor, brought us in contact with Martin and Erik, two professors from DTU Nano and Elektro. Our university is running a new project course in January where 1st semester students ferment their own cultures and build devices to monitor growth. They built a measuring chamber and a simple circuit, reading the voltage over the photodiode with a multimeter. Isn’t there a way to do all this using an Arduino? We set a meeting for next week.  Reading about digital to analog conversion(DAC) and the reverse(ADC). Dimming LEDs in the office.
Week 8 (July 18 - July 24)
Wetlab
Molecular Toolbox
CRISPR-Cas9 induced PEX10 knockout
Transformation in Y. lipolytica was repeated using the same transformation protocol from last week. However, this time less cells were used for the transformation. This did not work.
CRISPR-Cas9 induced URA3 insertion
Gibson assembly of sgRNAs targeting SUC2 into pCRISPRyl.                                   More (unsuccessful) transformations of pCRISPRyl and pCRISPRyl+sgRNAs into Yarowia lipolytica PO1f Δku70.
pSB1A8YL
The promoter was paired with the chromoproteins using 3A assembly. Unfortunately, no color was observed even though PCR and analytical digestion showed that the length of the construct are correct.
Promoters
TEF1 was amplified from gBlock by PCR.                                   SCR1’-tRNA promoter was amplified from the pCRISPRyl plasmid by PCR with primers introducing a base substitution to remove illegal restriction site.
Products
Beta-Carotene
Biobricks BBa_K530000 (crtYB) and BBa_K530001 (crtE) from the distribution kit was successfully obtained.                                   We designed Gibson primers including 5'-CACA-3' upstream of each start codon for assembly of all three genes.
Proinsulin
3A assembly of the proinsulin gBlock and pSB1C3. Electrophoresis did not confirm presence of desire plasmid.
Substrates
S. cerevisiae does not grow as well as Y. lipolytica on glycerol based waste, but has an advantage on sucrose based ones.
Compute
Genome Scale Modeling
Ben has kickstarted the modeling, by introducing the phenotype phase plane concept to the team, which is an extension to flux-balance analysis. The hope is that it will be possible to find optimal in-flow of various substrates in order to maximize product, in this case beta-carotene. Downloaded first Y. lipolytica model “MODEL1510060001”. Tutorials are studied in Matlab.
Software
Script writing is finished. Testing of functionality was initiated in order to find the most “sustainable” solution in terms of resources and the optimization step (reverse-translation) from the desired sequences to protein sequences. Approach decided for main body of script is building and checking the sequences “on the fly” while using as initial files the y_lip.txt (gcn), desired.fsa(protein sequences), ressites.txt(restriction sites).
Hardware
Successful tests at DTU Nano. We can measure growth of S. cerevisiae. Erik found a great photodetector that includes an amplifying circuit.
Week 9 (July 25 - July 31)
Wetlab
Molecular Toolbox
CRISPR-Cas9 induced PEX10 knockout
Transformation in Y. lipolytica was repeated using transformation protocol from Cory M. Schwartz . The transformation was unsuccessful.
CRISPR-Cas9 induced URA3 insertion
More (unsuccessful) transformations of pCRISPRyl and pCRISPRyl+sgRNAs into Yarowia lipolytica PO1f Δku70.
pSB1A8YL
Still trying to figure out why we do not see any color output...                                    K592010 was retrieved from the distribution kit.
Promoters
3A assembly with the TEF1 fragment and pSB1C3 backbone. Results were inconclusive, so they were repeated in week 10.
Products
Beta-Carotene
PCR with Gibson primers on all three biobricks and pSB1C3 backbone. Only BBa_K53000 (crtYB) and backbone was successful. crtI and crtE was after several attemps successfully amplified with prefix10 and suffix10 primers, which then worked at template for PCR with Gibson primers.
Proinsulin
3A assembly was repeated. Once again the desired plasmid was not obtained (confirmed with electrophoresis and restriction analysis).                                   Gibson assembly of proinsulin gBlock and backbone pSB1C3. Transformation gave numerous colonies that were used as a template for colony PCR. One colony was expected to carry a desired construct, however further investigation disproved it.
Substrates
The Y. lipolytica growth on sucrose was found to be a false positive. The autoclavaton of sucrose breaks the molecule turning it into glucose and fructose. S. cerevisiae grow faster due to the ability to metabolize sucrose, while Y. lipolytica can only utilize the glucose and fructose.
Compute
Hardware
Running more fermentations together with Martin and Erik. They changed their design while we got inspired for our final project. Couldn’t we turn this into a cheap alternative for the Hamilton robot? Reading on CAD drawings, 3D printing and laser cutting
August
Week 10 ( August 1 - August 7)
Wetlab
Molecular Toolbox
CRISPR-Cas9 induced PEX10 knockout
Transformation in Y. lipolytica was repeated using the same transformation protocol from last week. Transformations were performed with the CRISPRyl plasmid with inserted PEX10 protospacers and with and without PEX10 flanking regions, respectively. The transformations were plated on selective media after resuspension in water. The transformations were unsuccessful.
CRISPR-Cas9 induced URA3 insertion
Unsuccessful transformations of pCRISPRyl and pCRISPRyl+sgRNAs into Yarowia lipolytica PO1f Δku70.                                    Unsuccessful co-transformation of pCRISPRyl+sgRNAs and the linear URA3 +flanks fragment into Yarowia lipolytica PO1f Δku70 (protocol link). The concentration of the HR repair fragment should be increased.                                   Design of new sgRNAs targeting the SUC2 gene in case the already assembled ones are not working as expected in Y.lipolytica.
pSB1A8YL
After having talked with Joel from SDU, we realized that J23110 does not give any output, as it does not contain a RBS. He recommended that we use the BBa_K880005 instead.                                   The BBa_K880005 promoter was paired with BBa_K592009 and BBa_E1010 BioBricks in our backbone using 3A assmbly.                                   This time we had colored colonies!                                   Just to be sure we also confirmed the identity of the construct using PCR and analytical digestion, and they were correct!                                   The plasmid was also sent to Uppsala, who agreed to test the plasmid for us.
Promoters
New 3A assembly of TEF1 and SCR1’-tRNA and the pSB1C3 backbone.
Products
Beta-Carotene
Gibson assembly of all three crt genes and pSB1C3 backbone. Transformation resulted in colonies, but turned out not to with the correct constructs (only self-ligated backbone). Gibson assembly with pSB1A3 was attempted to avoid false positives.
Proinsulin
3A assembly for the proinsulin gBlock and pSB1A8yl, pSB1C3. No success.
Substrates
Y. lipolytica is shown to utilize canola oil production waste a better than S. cerevisiae. There is a lot of noise in the OD measurements.
Interlab
This week, we started the Interlab Study. We successfully transformed the three devices: Test Device 1, 2, and 3, consisting of the promoters J23101, J23106 and J23117, sharing the RBS (B0034), the terminator (B0015) and the GFP (E0040), as well as the negative control consisting of the pTetR promoter (RR040) with no sequence downstream, and the positive control consisting of the constitutively expressed GFP device (I20270). The plates were put in the fridge for later use.
Compute
Software
Implementation of GUI with tkinter library initiated.
Hardware
Rough sketch of the requirements finished. Mathias joined the team to program the Arduino. Surprisingly, he managed to fry the first board half an hour later. So glad to have him!                               Testing whether air bubbles could be used for stirring and 3D printing a simple Arduino controlled valve to control the airflow.
Week 11 (August 8 - August 14)
Wetlab
Molecular Toolbox
CRISPR-Cas9 induced PEX10 knockout
Transformations in Y. lipolytica were repeated using the same protocol from last week. However, this time the transformations were inoculated in liquid selective media for two days and then plated on YPD media after outgrowth.
CRISPR-Cas9 induced URA3 insertion
Successful transformations of pCRISPRyl and pCRISPRyl+sgRNAs into Yarowia lipolytica PO1f Δku70. Verification that the constructed pCRISPRyl+sgRNA is working as expected.                                   Unsuccessful co-transformation of pCRISPRyl+sgRNAs and the linear URA3+flanks fragment into Yarowia lipolytica PO1f Δku70. The concentration of the HR repair fragment should be increased.
pSB1A8YL
3A assembly was also performed for K880005 and K592010 in pSB1A8YL. Yellow-colored colonies were observed.                                   All colored constructs were grown and plated. Nice pictures were taken!
Promoters
Assembly of SCR1’-tRNA and pSB1C3 was confirmed by restriction analysis and sequencing.
Products
Beta-Carotene
Several attempts of Gibson assembly was made without correct results. We tried different strategies: different ratio of fragments, assembly two fragments at the time, new PCR of fragments.
Proinsulin
3A assembly for the proinsulin gBlock and pSB1A8yl, pSB1A3/C3/K3. No colonies showed the expected length with colony PCR.                                   Amplification of proinsulin gBlock using gradient PCR
Substrates
Results from growth on complex media in the form of different waste streams and byproducts were recreated.
Interlab
The plate with the negative control was lost during a cleanup.
Compute
Software
After watching and reading extensively about the power and capabilities of python based gui with tkinter, all the main functions have been set and the half-finished software is able to produce good output sequences while the option for taking out the unwanted restriction sites is not implemented yet !
Hardware
Improving Martins measuring cell. Printing a few prototypes. Try and Error. Looking for smart ways to control the light intensity as the photodetector has a saturation level. Experimenting with low-pass filters.
Week 12 (August 15 - August 21)
Wetlab
Molecular Toolbox
CRISPR-Cas9 induced PEX10 knockout
Transformants on YPD plates from last week were re-streaked on selective media. The transformations was not successful.
CRISPR-Cas9 induced URA3 insertion
Unsuccessful co-transformation of pCRISPRyl+sgRNAs and the linear URA3+flanks fragment into Yarowia lipolytica PO1f Δku70.
Products
Beta-Carotene
Successful Gibson assembly of crtE:crtI and crtI:crtYB. These were used as templates for fusion PCR without success.
Proinsulin
New 3A assembly (proinsulin + pSB1A8yl, pSB1C3). No transformants were present.                                   Gibson assembly proinsulin gBlock and pSB1A8yl, pSB1C3. As PCR templates pSB1C3:J04450 and pSB1A8yl:K880005+K592010 were used. Colony PCR showed the expected amplicon length for pSB1A8yl:proinsulin construct.
Substrates
We experience problems with the OD measurements from the canola oil waste because of leftover plant material.
Compute
Genome Scale Modeling
Meeting with faculties about genome scale modelling strategies, they think the strategy is doable.
Software
The software now works with the full options that were supposed to be provided to the user. Optimization of  some main script parts was initiated.
Hardware
Calibration routine for has been invented and implemented in R, ready to be ported to Arduino when needed.
Week 13 (August 22 - August 28)
Wetlab
CRISPR-Cas9 induced PEX10 knockout
Transformations in Y. lipolytica was repeated using the same approach as last week. The cells were inoculated in liquid selective media for two days. The transformations were unsuccessful.
CRISPR-Cas9 induced URA3 insertion
Successful amplification of URA3+flanks from Y. lipolytica W29 genomic DNA.
Products
Beta-Carotene
Designing USER primers.
Proinsulin
pSB1A8yl:proinsulin was transformed into Y. lipolytica.                                   pTEF, proinsulin, GFP, proinsulin tagged with GFP and the entire gBlock were extracted from the PCR amplified gBlock by PCR and in further step inserted into the pSB1C3 backbone by 3A assembly. Transformants were observed on most plates, although very few on the pTEF, GFP and Proinsulin thus these insertions was repeated using Gibson. No bands exhibited the correct lengths.
Substrates
Neither filtration nor emulgators seem to solve the problem the OD measurements on canola oil waste.
Interlab
New transformations in competent DH5-Α cells with Negative Control Device has been attempted  twice with no growth. Positive control grew so the transformation was fine. Probably, the negative  control device was not present in the tube.
Compute
Genome Scale Modeling
Phenotypic phase plane (PHPP) starts to be implemented, initial attempts in Matlab failed at reproducing phpp from literature, wrong signs and values.
Software
The final "touches" on GUI layout has been implemented although upon agreement the design may be modified while keeping in mind that it has to be compatible with all "mainstream" OSs (cross-platform).
Hardware
Diving into laser cutting.                                Sketching out all the forms needed. Getting a headache from thinking in 3D.                               The work on assembling all the modules needed has begun. LCD, SD card reader, keypad and RTC are all implemented and working properly.
September
Week 14 (August 29 - September 4)
Wetlab
Molecular Toolbox
CRISPR-Cas9 induced PEX10 knockout
Ran out of PEX10 flanking regions. Therefore, amplification of PEX10 flanking regions from Yarowia lipolytica PO1f Δku70 were repeated in order to have enough for transformation.                                    The PCR products were purified and ready to use.
Products
Beta-Carotene
PCR of genes with USER primers and USER cloning of the three genes and two different backbones (pSB1A3, pSB1K3). No success.
Proinsulin
The Yarowia lipolytica PO1f Δku70 pSB1A8YL:Proinsulin transformants and controls were observed under a YFP filter. No difference between the pSB1A8YL:Proinsulin and the control was observed. Although, it was noted that the pSB1A8YL:Proinsulin strain grew significantly slower both on the plates and in liquid media suggesting that there are some difference between the pSB1A8YL:Proinsulin and control strain.                                   3A assembly of pTEF, proinsulin, GFP, proinsulin:GFP and the entire gBlock into pSB1C3. Colony PCR for pTEF and the proinsulin transformations showed the expected fragment lengths.
Compute
Hardware
The calibration routine has been ported to Arduino, and is working perfectly.
Week 15 (September 5 - September 11)
Wetlab
Molecular Toolbox
CRISPR-Cas9 induced PEX10 knockout
Since transformations have been unsuccessful so far, we tried a new approach.                                   Annealing of PEX10 upstream and downstream flanking regions. Protocol from Cory M. Schwartz was used. However, this did not work.                                   New strategy for fusing the upstream and downstream Pex10 flanking regions: Fusion PCR. Wrong band sizes appeared on the gel after the PCR.
CRISPR-Cas9 induced URA3 insertion
New strategy for the URA3+flanks HR template: ligation onto pCRISPRyl+sgRNA. Primers were ordered for URA3+flanks amplification with a 5’-tail (tgcatgcat, NsiI restriction site in bold).                                   Unsuccessful amplification of URA3+flanks using the new primers.
Promoters
The TEF1 promoter was successfully assembled with the pSB1C3 backbone by 3A assembly.
Products
Beta-Carotene
New USER cloning with new PCR’s, still no success.
Proinsulin
3A assembly for GFP+pSB1C3, GFP+proinsulin+pSB1C3, pTEF+proinsulin in pSB1C3 and PSB1A8yl
Compute
Genome Scale Modeling
PHPP is reproduced successfully using Cobrapy, and a simple GSM is built from scratch to fully understand FBA. Meeting with DTU Biosustain to get introduction to Cameo, the GSM extension to cobrapy. Modelling strategy is refined to test multiple carbon sources.
Software
The production of the rest of the codon usage files was started for the, available gcn files by using an R script for matrix products,inspired by formulas provided by : dos Reis, M., Savva, R. and Wernisch, L., 2004. Solving the riddle of codon usage preferences: a test for translational selection. Nucleic acids research, 32(17), pp.5036-5044.
Hardware
Learned how to make PCBs over the weekend. Ordering our very own Shield for the Arduino now. Thank you KiCad community for such a nice wiki.
Week 16 (September 12 - September 18)
Wetlab
Molecular Toolbox
CRISPR-Cas9 induced PEX10 knockout
Fusion PCR from last week was repeated using a different PCR program. Unsuccessful fusion PCR.
CRISPR-Cas9 induced URA3 insertion
Unsuccessful amplification of URA3+flanks using the new primers. Since a linear template was being used, nested primers were designed (with the same 5’-tail as mentioned above).
Products
Beta-Carotene
New USER cloning with new protocol: different ratio of fragments, different incubation program. Still no correct constructs.
Proinsulin
Assembling pTEF with proinsulin, ECFP or EYFP biobricks into pSB1A8yl by 3A.
Compute
Genome Scale Modeling
Phpp success, reproduced published results in both Matlab and Cobrapy. Matlab code  is modified to produce PHPP analysis and output desired product fluxes. Errors was probably due to wrong solver definition and constraints on the uptake fluxes.
Hardware
Bug fixes everywhere. Trying to squeeze everything in the housing.                               The low pass filter needed for stabilization of the signal is added to the DIY and working.                               First inoculations are carried out! The technical results are promising, although the lack of stirring causes the biomass to fall to the bottom of the cuvettes.
Week 17 (September 19 - September 25)
Wetlab
Molecular Toolbox
CRISPR-Cas9 induced PEX10 knockout
Fusion PCR from last week was repeated using a different PCR program. However, the fusion PCR was unsuccessful again.
CRISPR-Cas9 induced URA3 insertion
Successful amplification of URA3+flanks using the nested primers.
Products
Beta-Carotene
New USER with gradient incubation, still no success.
Proinsulin
3A assembly in order to obtain pTEF+proinsulin, pTEF+CFP and pTEF+YFP in the pSB1A8YL backbone.                                   Repetition of 3A assemblies for pTEF+proinsulin, pTEF+CFP and pTEF+YFP using pSB1K3. All of them gave expected bands.
Substrates
With high pressure filtration we were able to remove the plant material from the canola oil waste and perform successful growth experiments.
Interlab
Transformation with negative control device in competent DH5 cells was repeated. 1 ul from thedistribution kit (plate 2, well 6F) was used for transformation. Calibration measurements with LUDOX and H2O were finished. FITC fluorescence standard curve was completed following the provided Plate Reader Protocol. Cell measurement was completed and all data was imported to the provided Excel sheet. The data was sent to measurement@igem.org and the online Plate Reader Form was submitted.
Compute
Genome Scale Modeling
The newest Y. lipolytica model is downloaded, “MODEL1508190002”. Modelling different C-sources on PHPP succeeded with results matching substrate screening experiments. Tried glycerol, glucose, sucrose, xylose, maltose, fructose and oleic acid.
Hardware
More testing and more bugs. Learning how to solder.
Week 18 (September 26 - October 2)
Wetlab
Molecular Toolbox
CRISPR-Cas9 induced PEX10 knockout
Contacted Cory M. Schwartz and explained issues about the different approaches we had tried so far. Cory M. Schwartz have tried the same approaches and it did not work for him either. Cory M. Schwartz sent us Y. lipolytica PO1f strain and three plasmids we could use for transformation instead with the same design we attempted with no or limited luck.                                    - pIW357: A version of the CRISPRyl plasmid for knocking out PEX10.                                    - pIW381: A version of the CRISPRyl plasmid for knocking out MFE1.                                    -pIW501: This plasmid enables knockout of PEX10 by integration of a URA3 gene.
CRISPR-Cas9 induced URA3 insertion
NsiI restriction overnight of pCRISPRyl+sgRNAs, alkaline phosphatase treatment for 5 hours and column purification. Successful restriction (as confirmed on gel).                                   NsiI restriction of URA3+flanks PCR product for 2 hours, followed by column purification.                                   Ligation of pCRISPRyl+sgRNAs and URA3+flanks and transformation into E. coli.
pSB1A8YL
We realized that the GFP sequence we had fused to our proinsulin sequence would not work. Talking to faculty we found a sequence that would work, and ordered that as a gBlock.                                   The gBlock arrived, and now it’s going to be a mad scramble to finish this in time for the wiki freeze!                                   The gBlock was amplified and used for 3A assembly together with our TEF promoter.
Promoters
Both TEF1 and SCR1’-tRNA in pSB1C3 backbone was confirmed by sequencing.
Products
Beta-Carotene
USER cloning attempted in pSB1A8YL backbone, no luck.                                   Sequencing of the original biobricks from the distribution kit/registry showed inconsistent prefix/suffix in BBa_K530001 and BBa_K530002, which might explain our troubles assembling the parts.
Proinsulin
Assembling pTEF+proinsulin, pTEF+CFP and pTEF+YFP pSB1K3 constructs into linearized pSB1C3 and pSB1A8YL with 3A. Growth was observed on the pSB1A8YL:pTEF+Proinsulin, pSB1C3:pTEF+Proinsulin, pTEF+CFP and pTEF+YFP constructs. pSB1A8yl:pTEF+Proinsulin was transformed into Yarowia lipolytica PO1f Δku70 strain. After 2 days of incubation colonies were tested for fluorencence, however no signal was observed.                                   3A assembly for: a) pSB1A8YL:TEF:hrGFP, b) pSB1A8YL:TEF:eCFP c) pSB1C3:TEF:hrGFP, d) pSB1C3:hrGFP (using hrGFP cut with EcoRI and PstI). Restriction analysis of pSB1A8yl:pTEF+eYFP confirm obtainig a proper construct. pSB1A8yl:pTEF+eYFP and pSB1A8yl:pTEF+proinsulin were transformed to Y. lipolytica.                                    pSB1A8yl:pTEF+eYFP was tested in both fluorescent microscope and in fluorometer, but no clear signal was observed.                                   3A assembly for: a) pTEF:hrGFP, b) pTEF:CFP and c) pTEF:YFP in pSB1A8yl. Success! Constructs were transformed to Y. lipolytica. Colonies did not give fluorescence.
Substrates
Y. lipolytica seem to grow well on the canola oil waste while S. cerevisiae does not grow very well.
Compute
Genome Scale Modeling
Modelling FVA is done in Matlab, no useful information are interpreted. Explored the differences in Y. lipolytica and S. cerevisiae. with PHPP on the different carbon sources.  Beta-carotene pathway is added to the model.
Software
Currently the library of codon usage files consists from 5 organisms with the intention for more to be added soon as .txt files. All of them are commonly used organisms. Research with regards to distribution and repository of software and compatibility regarding different OS versions. PyInstaller software will be used due to its capability of producing executables for all the known OS, so the user doesn't have to do anything else than just downloading the software.
Hardware
Did I mention bug fixing already?                               The pump has been added to the DIY, apparently causing the program to crash.
October
Week 19 (October 3 - October 9)
Wetlab
Molecular Toolbox
CRISPR-Cas9 induced PEX10 knockout
Received Y. lipolytica PO1f strain and pIW357, pIW381 and pIW501 from Cory M. Schwartz.                                    pIW501 was digested, and transformations into Y. lipolytica PO1f and PO1f Δku70 was initiated with pIW357, liniar pIW501, pIW357 + linearized pIW501, pIW357 + circular pIW501 and circular pIW501.
CRISPR-Cas9 induced URA3 insertion
Plasmid purification and restriction analysis of pCRISPRyl+sgRNAs and URA3+flanks ligation/transformation. No successful transformants were obtained. This project was discontinued, and another strategy of inserting URA3 in the place of PEX10 is initiated.
pSB1A8YL
Unfortunately the 3A assembly of last week did not work out. We tried again.                                   This time the colony PCR seemed right.
Products
Beta-Carotene
We removed two illegal restriction sites (AgeI and BglII) from BBa_K530001 and BBa_K530002 by site directed mutagenesis. The improved BioBricks were digested to confirm correct removal.
Proinsulin
3A assembly for a) hrGFP+pSB1C3, b) pTEF:hrGFP+pSB1C3, c) pTEF:hrGFP+pSB1A8yl, d) pTEF:CFP+pSB1A8yl. Expected band was observed only for hrGFP+pSB1C3. This construct was used in a new 3A assembly together with pTEF and both pSB1A8yl and pSB1C3, which was successful.
Compute
Genome Scale Modeling
Modelling getting more help from DTU Biosustain to set up Cameo on our computers, succeeded, Cameo can now be run without solvers and library errors. Got review literatures and questions and answers sessions from DTU Biosustain. We are added to DTU Biosustain’s online help platform in slack. And got many answers from DTU Biosustain experts, both online and in person at helpdesk sessions. Model is validated using Cameo with published and unpublished data from DTU Bioengineering, model is considered good enough to simulate general trends from substrate constraints. Tried to move on to strain design investigation using gene KO. Attempted but failed at making OptGene work. Returned no results. Attempted but failed at plotting fluxes on Escher due to reaction and metabolite IDs mismatch and the lack of Y. lipolytica metabolism maps. Same problem with KEGG maps, we do not have the right IDs to plot fluxes. It would be too time consuming to find the IDs or build a map on Escher ourselves, solution pending.
Hardware
The shield arrived. After switching from breadboard to PCB, nothing works anymore.  Another weekend full of bug fixes.
Week 20 (October 10 - October 16)
Wetlab
Molecular Toolbox
CRISPR-Cas9 induced PEX10 knockout
Transformations continues. There is growth in liquid selective media, but colonies are still growing on SC-oleic acid, which they should not be able to if PEX10 has been knocked out. On a successful note, we achieved to insert URA3 by homologous recombination in Yarowia lipolytica PO1f Δku70!
pSB1A8YL
OH NO! The sequencing tells us that the TEF promoter was not inserted! This is going to be a last minute results, if any! 3A assembly was repeated.
Products
Beta-Carotene
We verified the removal of the illegal restriction sites by sequencing. The sequencing results showed a nucleotide substitution in each restriction site corresponding to the ones introduced with the primers.
Proinsulin
Multiple attempts to detect production of proinsulin in Y. lipolytica were preformed with rtPCR, SDS-PAGE and Western Blotting. No protein or mRNA was detected. we suspect that yields are too low, or protein is not folding correctly. Project discontinued.
Compute
Genome Scale Modeling
Modelling attempts focused on to OptGene KO simulation and differential FVA, FSEOF. Got help from Biosustain’s online and help desk sessions. Results is generated from FSEOF and FVA, but no result from OptGene.
Software
The implementation of tAI based functions in the GUI application is finished and the codon usage are being produced in the fly according to user selection. Also the executables for Windows and Unix based systems have been created with PyInstaller.
Hardware
Light at the end of the tunnel. Testing the complete working prototype and adjusting the menu to make it more user friendly.
Week 21 (October 17 - October 23)
Wetlab
Molecular Toolbox
CRISPR-Cas9 induced PEX10 knockout
We have now observed that the colonies from pIW357 transformation into Y. lipolytica PO1f after SC-leu selection and outgrowth on YPD have different morphologies, a rose like shape, and a pearly shape. In consultation with Cory M. Schwartz we found out, that the pearly colonies are probably colonies with the PEX10 knockouts. We immediately restreaked, and made colony PCR and sent for sequencing. Unfortunately we will not get the final results before after the wiki freeze, but who cares, it finally works!!!
pSB1A8YL
This time the construct seems right! Now only the fluorescence microscopy will tell us if our construct works.                                   Aaaaaaaaand - it does!!!! Little autofluorescence on the negative, and clear, strong GFP signal from Y. lipolytica cells with the construct integrated! Luckily this data came in on the day of the wiki freeze!
Substrates
Both S. cerevisiae and Y. lipolytica show growth on oleic acid, one of the main components in canola oil. When S. cerevisiae can grow on this there must be some other reason it will not grow on canola oil waste. This shows Y. lipolytica to be at growth on complex waste streams.
Compute
Genome Scale Modeling
Final simulations to produce PHPP graphs with fixed axis and simulated gene targets using FSEOF only, instead of KOs, we will use gene amplifications. Success at plotting the gradients of fluxes from FSEOF on to metabolic map created in Cytoscape, directly using the model sbml file with help from supervisor.
Software
All the executable and library bundles have been produced and the after zipped they were deposited to IGEM software repository under the GPL v3 license.
Hardware
Testing the final prototype together with Tobias and Keenan from biologigaragen. Getting a lot of nice feedback and suggestions for “after iGEM”.
Protocols
Genomic DNA purification
One Taq PCR v1.0
Phusion U PCR v1.0
Chemically competent E.coli cells
Oligo annealing
Gibson assembly
User cloning
E.coli transformation v1.0
E. coli colony PCR
Y. lipolytica transformation v1.0
Y. lipolytica transformation v2.0
Y. lipolytica colony PCR
Yeast cell lysis
SD-oleic acid media
June
July
August
September
October
Protocols
FIND US AT:
DTU BIOBUILDERS
DENMARK
DTU - SØLTOFTS PLADS, BYGN. 221/002
2800 KGS. LYNGBY
E-mail:
dtu-biobuilders-2016@googlegroups.com
MAIN SPONSORS:
