<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UT-Tokyo_Project_Results"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UT-Tokyo/Project/Results">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UT-Tokyo/Project/Results&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UT-Tokyo/Project/Results&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UT-Tokyo/Project/Results&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UT-Tokyo/Project/Results" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UT-Tokyo/Project/Results</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="header"><H1 id="pageTitle">

Results

</H1><DIV id="headerLogo"><H1>iGEM UT-Tokyo</H1></DIV></DIV><DIV id="navi"><DIV class="slider-wrapper theme-default"><DIV id="menu"><DIV id="hauptMenu"><UL><LI><A href="https://2011.igem.org/Team:UT-Tokyo">Top</A></LI><LI><A href="https://2011.igem.org/Team:UT-Tokyo/Team">Team</A></LI><LI><A href="https://2011.igem.org/Team:UT-Tokyo/Project">Project</A></LI><LI><A href="https://2011.igem.org/Team:UT-Tokyo/Data">Data</A></LI><LI><A href="https://2011.igem.org/Team:UT-Tokyo/Society">Society</A></LI></UL></DIV><DIV id="teamMenu" class="subMenu"><UL><LI><A href="https://2011.igem.org/Team:UT-Tokyo/Team">Members</A></LI><LI><A href="https://2011.igem.org/Team:UT-Tokyo/Acknowledgment">Acknowledgment</A></LI></UL></DIV><DIV id="projectMenu" class="subMenu"><UL><LI><A href="https://2011.igem.org/Team:UT-Tokyo/Project/Background">Background</A></LI><LI><A href="https://2011.igem.org/Team:UT-Tokyo/Project/System">System</A></LI><LI><A href="https://2011.igem.org/Team:UT-Tokyo/Project/Results">Results</A></LI><LI><A href="https://2011.igem.org/Team:UT-Tokyo/Project/Discussion">Discussion</A></LI><LI><A href="https://2011.igem.org/Team:UT-Tokyo/Project/Achievement">Achievement</A></LI></UL></DIV><DIV id="dataMenu" class="subMenu"><UL><LI><A href="https://2011.igem.org/Team:UT-Tokyo/Data">Data</A></LI><LI><A href="https://2011.igem.org/Team:UT-Tokyo/Data/Method">Method</A></LI><LI><A href="https://2011.igem.org/Team:UT-Tokyo/Data/Modeling">Modeling</A></LI><LI><A href="https://2011.igem.org/Team:UT-Tokyo/Parts">Parts submitted</A></LI><LI><A href="https://2011.igem.org/Team:UT-Tokyo/LabNote">Lab Note</A></LI></UL></DIV><DIV id="societyMenu" class="subMenu"><UL><LI><A href="https://2011.igem.org/Team:UT-Tokyo/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:UT-Tokyo/Outreach">Outreach</A></LI></UL></DIV></DIV></DIV></DIV><DIV id="pageContent"><H1><SPAN class="mw-headline" id="Results">Results</SPAN></H1><DL><DD>This page is composed of the results of our experiments.
</DD></DL><DL><DD>The experiments are to evaluate the possibility and potential of our idea and can be divided into two sections: chemoattraction and production of L-Asp, and motility regulation by CheZ. Each page of the two sections includes the goal and methodological outline of experiments carried out as well as the results.
</DD></DL><DL><DD>In addition, we deviced “dual luciferase assay kit” in order to establish better measurement of gene expression. This kit is designed so that luciferase assay can easily be conducted for iGEM-standardized promoters. Explanation and analysis of this kit is given here.
</DD></DL><DL><DD>Details of methods and experimental conditions for each experiment are provided in <A href="/Team:UT-Tokyo/Data/Method" title="Team:UT-Tokyo/Data/Method">Method page</A></DD></DL><H1><SPAN class="mw-headline" id="Section_1:_Cell_Assembling_System">Section 1: Cell Assembling System</SPAN></H1><H2><SPAN class="mw-headline" id="1._Summary">1. Summary</SPAN></H2><DL><DD>To accomplish our system, SMART <I>E. coli</I>, it is required that we make worker cells assemble to guider cells. For this we utilize a property known as chemotaxis. Chemotaxis occurs in bacteria including <I>Escherichia coli.</I> They are believed to be attracted toward certain substances, including L-aspartate (L-Asp). We have tried to make workers attract guiders toward itself by a substrate-stimulated production of L-Asp. When guider cells synthesize L-Asp, worker cells assemble around the guider cells and workers are able to perform their assigned task effectively. Also, we characterized the chemoattraction of <I>E. coli</I> toward L-Asp, and compared the results to a computational simulation. We obtained supportive evidence for the agreement of wet and dry experiments. We propose that, if we get an <I>E. coli</I> secreting enough level of L-Asp, we can devise an inducible cell mustering system.
</DD></DL><H2><SPAN class="mw-headline" id="1.1._The_production_of_L-Asp">1.1. The production of L-Asp</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><FONT color="black">AspA synthesizes L-Asp from fumaric acid and ammonia</FONT></DIV></DIV></DIV><DL><DD>In previous studies accomplished L-Asp over-production, the amount of L-Asp was measured by HPLC <SUP class="ref">[1]</SUP><SUP class="ref">[2]</SUP>. However, having no available HPLC-apparaturs, we were unable to use this method, so we tried to detect it in alternative ways.
</DD><DD>We first tried to make <I><I>E. coli</I></I> producing L-Asp using the lac promoter, BBa_R0011. WT cells transformed with lacP-RBS-AspA-d.Ter (<A href="/Team:UT-Tokyo/Parts" title="Team:UT-Tokyo/Parts">BBa_K518004</A>) were pre-cultured with 1mM IPTG. The culture was soaked in fumaric acid solution containing annmonia. Note that AspA synthesizes L-Asp from fumaric acid and ammonia. The reaction mix was incubated at 37 degrees celcius for 1 hour. After the incubation, the concentration of L-Asp was measured by ninhydrin staining and ultraviolet-visible spectroscopy. Unfortunately, we were not able to obtain data that was obvious. We had ninhydrin react with L-Asp produced by AspA. However, Ninhydrin probably reacted not only with L-Asp but also with the remaining ammonia.
</DD></DL><DL><DD>The second attempt was TLC. Two microliters of the supernatant of the incubated reaction solution was spotted onto a TLC sillica plate, and extracted with 70% ethanol. We had expected L-Asp and ammonia to be separated. However, it was impractical to separate them using ethanol. We then tried various concentrations of acetone as a developing solvent, only to observe  lanes with smears.
</DD><DD>One of the main reasons of the failure is that our methods relied on the ninhydrin reaction. Ninhydrin certainly reacts with L-Asp. However, ninhydrin also reacts with an indispensable substrate of the AspA reaction. We should have selected a way in which only one of the reactants and products can be detected. Now then, the sequencing result shows that Assembling <A href="/Team:UT-Tokyo/Parts" title="Team:UT-Tokyo/Parts">BBa_K518004</A> was successful. So, it may be possible confirm the result if the right method is selected. For example, L-Asp can be detected by HPLC and the AspA protein can be detected instead of L-Asp.
</DD></DL><H2><SPAN class="mw-headline" id="1.2._The_characterization_of_L-Asp_chemotaxis">1.2. The characterization of L-Asp chemotaxis</SPAN></H2><DL><DD>Next, to show that <I><I>E. coli</I></I> moves in the direction of a higher L-Asp concentration, we carried out swarming assays. A WT colony was innoculated into a 0.25% agar LB plate and L-Asp solution was instilled. Plates were left at room temperature. After 20 hours, these plates were photographed. The representative photograph of a swarming colony is shown in <I>Fig. 1A</I>. 
</DD><DD>To determine the movement of colonies toward the location of L-Asp instillation, we then performed an image analysis. Obtained images were subjected to computational processing to find edges of the colonies. The processed image is shown in <I>Fig. 1B</I>.
</DD><DD><DIV style="margin-left: 10px;padding-bottom: 10px;"><P><I>Fig. 1A/1B</I>. 　1A: A swarming colony after 20-hour incubation. A cross &quot;×&quot; in the plate center was the position that L-ASp was instilled. 1B: A colony image processed to find the edge. Blue line represents the border of the colony(red).</P></DIV></DD></DL><DL><DD>In this assay, we defined as a vector the distance a colony moves from the center of a colony to the initial tip position. The distances are represented in <I>Fig. 1C</I>. The migration distance of colonies were measured to be 38(±9) mm for a plate with L-Asp solution, while this was 13(±6) mm for a plate without L-Asp. The results clearly show that colonies exposed to the L-Asp solution swarmed significantly, compared to colonies of the control group. Data was averaged from more than 6 experiments.
</DD></DL><DL><DD><DIV style="margin-left: 10px;padding-bottom: 10px;"><P><I>Fig. 1C</I>: the movemant of colonies with L-Asp sol.(Asp+) and without L-Asp sol.(Asp-).</P></DIV></DD></DL><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><DIV id="references"><UL><LI id="ref_1">[1] Chao, Y. P., Lai, Z. J., Chen, P., &amp; Chern, J. T. (1999). Enhanced conversion rate of L-phenylalanine by coupling reactions of aminotransferases and phosphoenolpyruvate carboxykinase in Escherichia coli K-12. Biotechnol Prog, 15(3), 453-458.</LI><LI id="ref_2">[2] . Chao, Y., Lo, T., &amp; Luo, N. (2000). Selective production of L-aspartic acid and L-phenylalanine by coupling reactions of aspartase and aminotransferase in Escherichia coli. Enzyme Microb Technol, 27(1-2), 19-25.</LI></UL></DIV><H1><SPAN class="mw-headline" id="Section_2:_Cell_Arrest_System">Section 2: Cell Arrest System</SPAN></H1><H2><SPAN class="mw-headline" id="2._Summary">2. Summary</SPAN></H2><DL><DD>To maintain the once elevated cell density, we then design a substrate-induced Cell Arrest System. To realize a cell arrest, we controlled the expression of cheZ, one of the flagellum-regulating genes in <I>Escherichia coli</I>, in a cheZ-defective strain (JW1970). We first constructed an inducible-cheZ cassette (<A href="/Team:UT-Tokyo/Parts" title="Team:UT-Tokyo/Parts">BBa_K518007</A>) and successfully demonstrated its capability of rescuing the motility of cheZ-defective cells with proper induction. We then constructed a cheZ expression device (<A href="/Team:UT-Tokyo/Parts" title="Team:UT-Tokyo/Parts">BBa_K518008</A>) which is repressed by the lambda cI repressor (BBa_C0051) in response to IPTG. Our data suggested that our device enables <I>E. coli</I> to respond to the induction ―this time IPTG― and lose motility. In conclusion, we were able to show that our inducible Cell Arrest System is feasible.
</DD></DL><H2><SPAN class="mw-headline" id="2.1._cheZ_transformation_rescues_cheZ-_strain.27s_motility">2.1. cheZ transformation rescues cheZ- strain's motility</SPAN></H2><DL><DD>To show that our <I>cheZ</I> expression cassette (<A href="/Team:UT-Tokyo/Parts" title="Team:UT-Tokyo/Parts">BBa_K518007</A>) rescues the defective motility of cheZ- cells, we first transformed cheZ- cells with an IPTG-induced cheZ expression device (<A href="/Team:UT-Tokyo/Parts" title="Team:UT-Tokyo/Parts">BBa_K518006</A>) (we call these cheZres) , and analyzed their colony diffusion patterns. As a cheZ- cell, a cheZ-defective strain JW1970 was used. After inoculation of respective colonies, 0.25% agar LB plates were left at room temperature. These plates were imaged 3 times each with an interval of approximately 24 hours. The photographs of representative colonies are shown in Fig. 1A. Obtained images were thresholded to identify the edge of the colonies. Processed images corresponding to images in Fig. 1A are shown in Fig. 1B. 
</DD><DD>To determine the motility of these cells, we then performed an image analysis. In this project, we defined two parameters for analyzing cellular motility; one was a colony size, which was defined as the total pixel count of the thresholded colony, and the other was the relative &quot;whiteness&quot; of each colony. We defined an arbitrary whiteness index (WI) as the mean of the pixel intensity of the thresholded colony normalized by its size. This WI value can be interpreted as a parameter of how thin, or diffusible, the colony is.
</DD><DD>The colony sizes are represented in Fig. 2A. This result indicates that cheZ-strains have significantly lower motility compared to cheZres cells after enough IPTG-induction, which suggests that our cheZ cassette (<A href="/Team:UT-Tokyo/Parts" title="Team:UT-Tokyo/Parts">BBa_K518007</A>) and IPTG-induced cheZ expression device (<A href="/Team:UT-Tokyo/Parts" title="Team:UT-Tokyo/Parts">BBa_K518006</A>) works as expected. 
</DD><DD>The WI values of those colonies are also presented in Fig. 2B. This data also supports the hypothesis that cheZ- has a higher tendency to aggregate than the cheZ-rescued strain. For both charts, data obtained from triplicate experiments is presented as mean ± S.D.. 
</DD></DL><DL><DD><DIV style="margin-left: 10px;padding-bottom: 10px;"><P><I>Fig.2A</I>. 　</P></DIV></DD><DD><DIV style="margin-left: 10px;padding-bottom: 10px;"><P><I>Fig.2B</I>. 　</P></DIV></DD></DL><H2><SPAN class="mw-headline" id="2.2._Inducible_repression_of_cheZ_deprives_rescued_cheZ-_of_motility">2.2. Inducible repression of cheZ deprives rescued cheZ- of motility</SPAN></H2><DL><DD>As our cheZ expression cassette (<A href="/Team:UT-Tokyo/Parts" title="Team:UT-Tokyo/Parts">BBa_K518007</A>) was proven to work properly, we next aimed at creating an inducible Cell Arrest System. To achieve this, we must devise an induction-based repression of cheZ. We utilized lambda cI repressor (BBa_C0051) and an inducible promoter (BBa_R0051). As with the previous section, gel swarming assays were conducted, with an induction of IPTG. This time, cheZ- cells transformed with R0051-K518007 only were used as a control, and the motility of cells possessing a construct allowing IPTG-induced cI repression of cheZ (<A href="/Team:UT-Tokyo/Parts" title="Team:UT-Tokyo/Parts">BBa_K518008</A>) in addition to R0051-K518007 were analyzed.
</DD><DD>Colony sizes and WI values are represented in Fig. 3C and D. Our data suggest that, by inducing the c1 repressor with IPTG, R0051-K518007 was strongly repressed and the motility of cheZ- strains decreased, control showed significant diffusion across the gel media. The data shows that induction by even IPTG at 1uM is adequate for repressing cheZ. As with section 1, data obtained from triplicate experiments is presented as mean ± S.D.. 
</DD></DL><DL><DD><DIV style="margin-left: 10px;padding-bottom: 10px;"><P><I>Fig.2C</I>.</P></DIV></DD><DD><DIV style="margin-left: 10px;padding-bottom: 10px;"><P><I>Fig.2D</I>.</P></DIV></DD></DL><H2><SPAN class="mw-headline" id="2.3_Conclusion">2.3 Conclusion</SPAN></H2><DL><DD>We designed and characterized an inducible cheZ expression device and a cheZ repression device. We were able to show that both IPTG-induction of cheZ expression and its repression altered the motility of the cheZ- strain as expected. We found even 1uM IPTG may over-induce the downstream cIp, so future works include finding the proper condition of inductions. With enough induction, our Cell Assembling System can be achieved.
</DD></DL><H2><SPAN class="mw-headline" id="Reference">Reference</SPAN></H2><DIV id="references"><UL>
Shana Topp and Justin P. Gallivan, Supporting Material Guiding Bacteria with Small Molecules and RNA. Department of Chemistry and Center for Fundamental and Applied Molecular Evolution, Emory University, 1515 Dickey Drive, Atlanta, GA 30322, S2-S3.

</UL></DIV><H1><SPAN class="mw-headline" id="Section_3:_UV_Switch">Section 3: UV Switch</SPAN></H1><H2><SPAN class="mw-headline" id="3._Summary">3. Summary</SPAN></H2><DL><DD>We characterized ultraviolet (UV)-induced promoters to devise a &quot;UV Switch&quot;. Our SMART System requires a promoter induction at the location of substrate, so is applicable only to those substrates for which responsive promoters are already known. To address this issue, we tried to provide a well-characterized easy-inducible promoter as a genetic switch. We chose UV as an example of induction, and characterized some UV-induced promoters recAp (BBa_J22106), recNp (<A href="/Team:UT-Tokyo/Parts" title="Team:UT-Tokyo/Parts">BBa_K518011</A>), and sulAp (<A href="/Team:UT-Tokyo/Parts" title="Team:UT-Tokyo/Parts">BBa_K518010</A>). To quantitatively and accurately evaluate the promoter activity, dual luciferase assay method was employed. Our results show that sulAp works as an acute switch. Combining the UV Switch with our SMART System, it becomes possible to &quot;switch on&quot; the self-assembling and localization at the location of our interest. 
</DD></DL><H2><SPAN class="mw-headline" id="3.1._Dual_Luciferase_Assay_System">3.1. Dual Luciferase Assay System</SPAN></H2><DL><DD>It is very important to be able to regulate gene expression as you like in biotechnology.  To apply a regulator, such as a promoter, of the relevant strength to a gene, it is first essential to take an accurate measurement of the activity of regulators. However, it is difficult to measure the activity of regulators because the activity often changes depending on the conditions. In order to deal with this problem, Kelly et. al. (2009) proposed a BioBrick promoter assay system based on relative promoter strengths.<SUP class="ref">[3]</SUP> Their system measures the fluorescence intensity of GFP placed downstream of the promoter of interest, which is used to compute the absolute promoter strength. This value is normalized to the strength of a “standard” promoter which is simultaneously measured so that differences between experiments could be minimized.
</DD><DD>In this project, we suggest an alternative promoter assay system. In the dual luciferase assay(Hannah et al 1998), the luminescent intensity caused by firefly luciferase, used as a reporter of the promoter of choice, is compared to that caused by renilla luciferase, used as a reporter for a control promoter, both placed within the same plasmid.<SUP class="ref">[4]</SUP> Importantly, because the promoter to be measured and its comparison are placed within the same plasmid, differences in expression between cells are minimized. In addition, the need to normalize cell densities between sample and control is abolished, which can often be a difficult and inaccurate task. Also, the use of luciferase as a reporter allows a quick and accurate measurement and in addition is not fluorescence-based, nullifying the need to worry about fluorescence quenching.
</DD></DL><DL><DD>We assembled the Dual luciferase assay system from existing BioBrick parts, and used BBa_J23118 as the control promoter. We measure the relative intensity of this promoter to BBa_J23119, which has been already measured by the aforementioned assay, and so the intensities of promoters from the two systems can be compared to each other using BBa_J23119 as a standard. 
</DD><DD>In order to confirm  that our system is workable, we assayed using our system how the activity of the lactose promoter (BBa_R0010) , an IPTG-inducible promoter, changes according to the strength of induction (<I>Fig. 3A</I>)
</DD><DD><DIV style="margin-left: 10px;padding-bottom: 10px;"><P><I>Fig. 3A</I>: The results of a Dual luciferase assay assayed for the lactose promoter (BBa_R0010). The transformed E.coli were cultured in medium containing the IPTG concentration shown. Cells were lysed and assayed when their OD600 reached 0.4-0.5.</P></DIV></DD></DL><DL><DD>As <I>Fig. 3A</I> shows, it was observed that the relative activity of lactose promoter increased with increasing IPTG concentrations. The result that concentration-dependent activity was observed with little variation between trials confirms that our “Dual luciferase assay” system can be used to measure quantitatively the intensities of promoters. 
</DD></DL><H2><SPAN class="mw-headline" id="3.2._Evaluation_of_UV-induced_Promoters">3.2. Evaluation of UV-induced Promoters</SPAN></H2><DL><DD>As our dual luciferase assay kit (K518002) has been confirmed to work, we then assayed sulAp (<A href="/Team:UT-Tokyo/Parts" title="Team:UT-Tokyo/Parts">BBa_K518010</A>), which is known to belong to the family of SOS promoters, activity. (<I>Fig. 3B</I>)
</DD><DD><DIV style="margin-left: 10px;padding-bottom: 10px;"><P><I>Fig. 3B</I>: The result of a Dual luciferase assay assayed for the sulA promoter. Strains possessing recA and those lacking the gene were transformed. The transformed <I>E. coli</I> were split into dishes after they were cultured until the OD600 is 0.4-0.6. They were then exposed to UV (15W / 254nm) for the shown durations while agitating gently. After UV irradiation finished, they were cultured for recovery for 30 min. at 37° C. Then, the cells were lysed and assayed using our system.</P></DIV></DD></DL><DL><DD><I>Fig. 3B</I> shows the sulAp strength after exposure to UV (15W / 254nm) for a minute. recA is required for expression from the sulAp and therefore a strain lacking the gene is used as a negative control. The relative activity of sulAp decreased after 10 minutes exposure of the same strength compared with that after 1 minutes exposure. This suggests that many E.coli died as a result of the 10 minute exposure. In support of this hypothesis, the OD of <I>E. coli</I> exposed to UV for 10 minutes was less than that of <I>E.coli</I> exposed for 1 minute (data not shown) after a 30 minute recovery culture.
</DD><DD>A detailed explanation of our system can be shown in Data section, and full method can be shown in Method section.
</DD><DD>We assayed promoters this time, but the “Dual luciferase assay” system makes it possible to measure not only transcriptional but also translational activity with appropriate control.
</DD></DL><H2><SPAN class="mw-headline" id="References_2">References</SPAN></H2><DIV id="references"><UL><LI id="ref_3">[3]Jason R Kelly, Adam J Rubin et.al “Measuring the activity of BioBrick promoters using an in vivo reference standard” J Biol Eng. 2009; 3: 4.</LI><LI id="ref_4">[4] Rita R. Hannah, Martha L et.al “Rapid Luciferase Reporter Assay Systems for High
Throughput Studies” Promega Notes Number 65, 1998
</LI></UL></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:UT-Tokyo/Project/Results">http://2011.igem.org/Team:UT-Tokyo/Project/Results</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UT-Tokyo/Project/Results" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UT-Tokyo/Project/Results" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UT-Tokyo/Project/Results&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UT-Tokyo/Project/Results&amp;oldid=234368" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>