<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_IIT_Delhi_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:IIT_Delhi/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:IIT_Delhi/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:IIT_Delhi/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:IIT_Delhi/Notebook&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:IIT_Delhi/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:IIT Delhi/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>Navigation-Top</TITLE></P><DIV id="wrap"><UL class="navbar"><LI><A href="https://2013.igem.org/Team:IIT_Delhi" id="home">Home</A></LI><LI><A href="#" id="Project">Our Project</A></LI><LI><A href="https://2013.igem.org/Team:IIT_Delhi/Project" id="overview">Overview</A></LI><LI><A href="https://2013.igem.org/Team:IIT_Delhi/Idea" id="motivation">Motivation</A></LI><LI><A href="https://2013.igem.org/Team:IIT_Delhi/Model" id="model">The Model</A></LI><LI><A href="https://2013.igem.org/Team:IIT_Delhi/Biobrick" id="biobricks">Biobricks Submitted</A></LI></UL><LI><A href="https://2013.igem.org/Team:IIT_Delhi/Modelling" id="modelling">Modelling and Simulation</A></LI><LI><A href="#" id="methods">Methods and Results</A></LI><LI><A href="https://2013.igem.org/Team:IIT_Delhi/Protocol" id="protocol">Protocols</A></LI><LI><A href="https://2013.igem.org/Team:IIT_Delhi/Notebook" id="notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:IIT_Delhi/Brainstorming" id="brainstorming">Brainstorming</A></LI><LI><A href="https://2013.igem.org/Team:IIT_Delhi/Safety" id="safety">Safety</A></LI><LI><A href="#" id="outreach">Outreach</A></LI><LI><A href="https://2013.igem.org/Team:IIT_Delhi/Outreach" id="orientation">Orientation for Freshmen</A></LI><LI><A href="https://2013.igem.org/Team:IIT_Delhi/BRAI" id="bill">BRAI Bill</A></LI><LI><A href="https://2013.igem.org/Team:IIT_Delhi/Biotoilet" id="biotoilets">Biotoilets</A></LI><LI><A href="https://2013.igem.org/Team:IIT_Delhi/Team" id="team">Team</A></LI><LI><A href="https://2013.igem.org/Team:IIT_Delhi/Student" id="student">Student Members</A></LI><LI><A href="https://2013.igem.org/Team:IIT_Delhi/Advisor" id="faculty">Faculty Advisors</A></LI><LI><A href="https://2013.igem.org/Team:IIT_Delhi/Attributions" id="attributions">Attributions</A></LI></DIV><TABLE style="text-align: left; width: 100%;" border="0" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width: 645px;" align="center" valign="top"><FONT size="+2">Lab Notebook</FONT><DIV style="text-align: left;">
The wet lab experiments throughout the summers were a roller coaster ride. Most of us had very little prior experience in biotechnology laboratories and, initially, were very apprehensive in working in the lab. Gradually, we gained experience in the methods and became more confident about the work. Each new day brought up a set of new problems and every day we had to find new ways for troubleshooting. Following is a concise notebook of our work through the summers:<B>Week 1</B>1. Our plan of work included cloning the asr promoter in the high copy number plasmid, pUC19 (Ampicillin resistance). For this reason, we designed the primers for asr amplification with EcoRI and BamHI as the restriction enzyme sites respectively. We thought of linearizing the plasmid till the primers arrived.2. The pUC19 plasmid was isolated using the Qiagen miniprep plasmid isolation Kit from an overnight culture, inoculated from an old plate of transformed DH5α cells. Running 1uL on 1% Agarose Gel indicated a good concentration of plasmid isolated.3. To check if the restriction enzymes present in the lab are usable, we digested the isolated plasmid with the two enzymes separately and ran them on the gel. Following was the result:Clearly the plasmid was not digested properly by either of the two enzymes, which is why more enzyme (0.5uL each) was added to each microcentrifuge tube and kept for further incubation.4. The plasmid was found to be completely digested by EcoRI. However, the sample containing BamHI was still found to contain uncut plasmid DNA. This plasmid was discarded and fresh colonies inoculated for plasmid isolation.<B>Week 2</B>1. Midiprep plasmid isolation was carried out to obtain high yield of pUC19 plasmid.2. This time the digestions yielded completely cut plasmid and, hence, the single digested product was further digested with the opposite enzyme and run on the gel:3. This was then Gel Eluted and put on self-ligation overnight. The ligation product was stored at -20⁰C freezer.4. Escherichia coli DH5α Competent Cells were prepared and stored in the -80 freezer. Also, LA-Amp plates.<B>Week 3</B>1. The self-ligated product was transformed in the competent cells prepared earlier along with the undigested pUC19 (as Control) and plated on LA-Amp plates.2. The two plates had almost equal number of colonies, indicating that the plasmid self-ligated efficiently, leading us to conclude that the digestion did not take place properly as we had earlier thought. So we were required to repeat the whole procedure.3. By this time, our primers for asr promoter had arrived and we had made a stock of 100uM concentration. We also made a working solution of 5uM primer concentration.4. The asr promoter was PCR amplified using the primers and E. coli K12 genome as the template (asr is an acid shock response gene present inherently in E. coli). Since the amplicon is 178bp in length, the extension time was kept low. A gradient PCR with a number of annealing temperatures was used to find out the optimal annealing temperature for amplification.5. After PCR, the product was run on 1.5% Agarose Gel along with a 100bp ladder.6. Clearly, all Tm gave satisfactory results, so all these bands were cut and eluted.<B>Week 4</B>1. pUC19 vector preparation continued. This time different sources of restriction enzymes were used, resulting in a completely digested plasmid:2. After successful single digestion, EcoRI was added to the sample already digested with BamHI, while BamHI was added to the sample already digested with EcoRI.3. Simultaneously, the eluted P-asr ¬fragment was digested with EcoRI and BamHI.4. All the digested products were run on agarose gel, extracted and stored in the -20⁰C freezer.5. Ligation of P-asr and pUC19 was accompanied by a negative control, i.e. self-ligation of pUC19 plasmid.6. Transformation of the ligated products were accompanied by a positive control of uncut pUC19 plasmid. The transformants were plated on LA-Amp plates and incubated at 37⁰C overnight.7. All the plates contained colonies, including the negative control plate, indicating that there was some impurity of undigested plasmid still remaining in the digested product. Since the multiple cloning site in pUC19 is in the middle of lacZ inducing fragment, we used the method of blue-white screening to identify the clones. Following is the plate with the cloned colonies:<FONT size="1">Figure: The blue colonies indicate the self-ligated plasmid.</FONT>8. The plasmids of these colonies were isolated and P-asr amplified by using them as template to confirm the clone.<B>Week 5</B>1. Since iGEM requires us to submit our parts to the registry in the pSB1C3 backbone, we designed primers for all our parts, i.e. P-asr, SFGFP, lacZ and P-atp2 containing the standard prefix and suffix.2. The isolated plasmids of cloned colonies were digested with BamHI and PstI. Colonies no 1 &amp; 2 were clones, while no 6 is a self-ligated plasmid.3. It was observed that BamHI was again causing problems.<B>Week 6</B>1. The primers arrived. The fragments of P-asr, SFGFP and lacZ were PCR amplified and gel eluted.
2. The concentration of linearized pSB1C3 backbone sent to us by iGEM was checked by running on 1% agarose gel and found to be satisfactory.
3. BamHI from yet another source was taken and used to digest the cloned plasmid. Along with this, pSB1C3 was digested with EcoRI and PstI. Also, P-asr and SFGFP were PCR amplified. All of these were run on 1% Agarose.
4. pUC19 was digested with PstI and cut and stored at -20⁰.
5. P-asr and SFGFP were digested with EcoRI and PstI. The two fragments and pSB1C3 were incubated at 80⁰C for 20mins to heat inactivate the restriction enzymes.
6. P-asr+pSB1C3 and SFGFP+pSB1C3 ligations were carried out and resultant stored at -20⁰C.<B>Week 7</B>1. The ligation products were transformed in DH5α competent cells along with BBa_J04450, that was given to us by iGEM in the Transformation Efficiency kit. The transformants were plated onto LA-Cm plates.
2. There were just too many colonies on the plates. Even the control plate that should’ve had only RFP expressing colonies had lots of white colonies. Screening of SFGFP clones through colony PCR turned out to be unsuccessful even after taking around 19 colonies. For achieving definite double digestion of vector, a unique method was thought out.
3. BBa_J04450 was digested with EcoRI and PstI. The vector, pSB1C3, can be clearly distinguished after running on the gel. It was eluted and then used for ligation with the two inserts, P-asr and SFGFP. These was accompanied by the negative control of no insert.
4. On transformation, however, no colonies were observed in any of the plates. This procedure was repeated once more but yielded no positive result.<B>Week 8</B>1. The digested pUC19+P-asr plasmid was gel eluted and ligated with the SFGFP insert, transformed and plated onto LA-Amp plates but again yielded too many colonies to isolate each colony’s plasmid and screen (colony PCR could not have been carried out as asr is present in the genome and can lead to false positive results).
2. Another try on digesting pUC19+P-asr plasmid yielded degradation of plasmid every time.
3. Since we had colonies of BBa_J04450 biobrick, we thought of utilizing this to find out if there are any effects of pH change on the fluorescence from RFP:
4. The experiment involved growth of these cells, inoculated from a single red colony on the above plate, in LB media set at six different pH.
Following were the results of our experiment:
From this graph it can be concluded that even within the considerable range of pH 5.09 to 7.36, the fluorescence of RFP remains<I> fairly constant </I>and is mainly a function of the growth-related activities within the cell. The fluorescence is nil at pH 4.0, as this is more of an emergency situation, highly unfavorable for cell growth and hence the cell machinery is involved in damage recovery as compared to other transcriptional activities.
      </DIV></TD></TR></TBODY></TABLE><TABLE style="text-align: left; background-color: rgb(175, 175, 175); width: 100%;" border="0" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="vertical-align: top; text-align: center; width: 302px; background-color: rgb(175, 175, 175);">
Feel Free to contact us at igemiitdelhi2013 at gmail dot com if you
have queries; requests; suggestions et cetera.
Thanks to iGEM and IIT Delhi,
we had an awesome summer!</TD><TD style="vertical-align: top; text-align: center; width: 351px; background-color: rgb(175, 175, 175);">Our
Project was supported by and done by the students of IIT Delhi, India.</TD><TD style="vertical-align: top; text-align: center; width: 339px; background-color: rgb(175, 175, 175);">This
project was done as a part of iGEM:<A href="https://igem.org/Main_Page">iGEM Main Website</A></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:IIT_Delhi/Notebook">http://2013.igem.org/Team:IIT_Delhi/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:IIT_Delhi/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:IIT_Delhi/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:IIT_Delhi/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:IIT_Delhi/Notebook&amp;oldid=239704" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>