"
Page
Discussion
View source
History
teams
Log in
 
Team:Korea U Seoul/Project/Description
From 2014.igem.org
Home
Team
Our members
Official Team Profile
Project
Description
Design & Parts
Experiment & Protocol
Judging Critiera
Application & Improvement
Notebook
Human Practice
Outreach
Safety Questions
iGEM
Background
Sortases, enzymes that recognize and cleave the specific sorting signal of secreted proteins to form isopeptide bonds between the secreted proteins and polypeptides, function as protein ligase to form the cell-wall surface of gram-positive bacteria. In case of C.diphtheriae, which belongs to the same genus of our experimental bacteria Corynebacterium glutamicum, has total of 6 sortases, 5 pilus specific sortases (Srt A,B,C,D,E) and 1 housekeeping sortase (Srt F) involved in the formation of all types of pili.
There are 9 types of pillins that comprise pilus; Spa A, B, C, D, E, F, G, H, I, J. SrtA build up SpaA-type pili, which is composed of Spa A, B and C. Srt B/C and Srt d/E each forms SpaD-type pili and SpaH-type pili, which is made up of Spa D, E, F and Spa H, I, J respectively. Like this, the names of each pili types are generally from the names of many Spa proteins.
The picture below shows the process of SpaA type pili formation, using SrtA. Sortase class A enzymes recognize the sequence LPXTG at the carboxyl terminus of surface protein precursors. Cystein of SrtA recognizes LPXTG motif of SpaC, cleaves between T and G, forming SpaC-SrtA intermediate via nucleophilic attack. This intermediate is again attacked by lysine of SrtA bounded SpaA, and the process is continued to form SpaC–SpaAn–SrtA intermediates.
Nucleophilic attack by Lysine of SrtF bounded SpaB form SpaC-SpaA(n)-SpaB-SrtF intermediate. The product of this SrtA reaction is covalently linked to lipidⅡ and is then incorporated into the cell wall envelope, terminating the formation of SpaA-type pilus.
(Architects at the bacterial surface — sortases and the assembly of pili with isopeptide bonds Antoni P. A. Hendrickx, Jonathan M. Budzik, So-Young Oh and Olaf Schneewind)
Description
The main objective of our project is to construct a novel “protein whip” platform, with which we can make Corynebacterium glutamicum to express other corynebacterium’s pili structure comprised of chains of a protein of our choice. As our first try, we decided to make pili made out of green fluorescence proteins (GFP); in order to do so, we substituted SpaA protein, one of the surface proteins in the Pilin A gene cluster, into green fluorescence protein, and transformed a vector containing the modified Pilin A gene cluster into a C. glutamicum strain.
Our “protein whip” platform is expected to have many practical applications. For example, pili made out of an enzyme, enzyme whip will enable the reaction to take place with high efficiency, for a great number of the enzyme included in the pili will be able to “attack” the reactants simultaneously. Biofilms made of strains of bacteria that express pili comprised of chains of specific amino acids such as histidine or cysteine that readily bind to heavy metals may be utilized to purify water contaminated with heavy metals.
Having a number of potential applications is not the sole merit of our project; by using C. glutamicum instead of widely exploited Escherichia coli, our project also contributes to expanding model organisms used in synthetic biology beyond E. coli.
Retrieved from " http://2014.igem.org/Team:Korea_U_Seoul/Project/Description "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
