<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SYSU-Software_model"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:SYSU-Software/model">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SYSU-Software/model&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SYSU-Software/model&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SYSU-Software/model&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SYSU-Software/model" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SYSU-Software/model</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>Team:SYSU-Software</TITLE><TITLE>new</TITLE></P><DIV class="wrapper"><DIV id="model-container"><DIV class="span3 bs-docs-sidebar" id="model-list"><UL class="nav nav-list bs-docs-sidenav ml"><LI><A href="#tab0">Overviews</A></LI><LI><A href="#tab1">Parameters</A></LI><LI><A href="#tab2">Deterministic models</A></LI><LI><A href="#tab3">Stochastic and Time delay models</A></LI><LI><A href="#tab4">Cascade analysis</A></LI><LI><A href="#tab5">References</A></LI></UL></DIV><DIV id="main"><DIV id="model-header"><H1>Models and Algorithms</H1></DIV><DIV id="part0"><DIV class="section" id="tab0"><H2>Overview</H2><P>   Mathematical modeling is the soul of our magical software &quot;Computer Aided Synbio Tools&quot;, CAST. In our models, we take into account the intrinsic dynamics of different circuits or systems hierarchically by deterministic, stochastic and time delay models. Here are the highlights of our models and algorithms:</P><DIV class="mintext"><P>  * We place great emphasis on various kinds of promoters, operons, coding parts, RBSs, terminators and combine regulatory as well as metabolic ODEs and Hill equations in our multi-level modeling;</P><P>  * We create a new iteration and optimization algorithm to calculate the fold-change in different cascades and cyclic circuits to evaluate the strength of various regulators;</P><P>  * Leakage rate diversity and copy number of different plasmids are considered for better reflection of complex biological systems and experimental results in vivo;</P><P>  * What our modeling tries to achieve is to creatively build a bridge between parts and modules in the Registry and widespread experimental data in the papers so that researchers can have a brand new perspective of how these parts, sequences and kinetic parameters are related to each other;</P><P>  * Standardized output PoPS, RIPS is generated from our data processing and testing procedures, which represents our goal of establishing sets of standardized data;</P><P>  * As the current focus of synthetic biology is readily combining modules into complex synthetic pathways<SUP>[1]</SUP>, statistical and probability distribution models are also applied to estimate extrinsic variability among synthetic systems.</P></DIV></DIV></DIV><DIV id="part1"><DIV class="section" id="tab1"><H2>Intrinsic dynamics</H2><H3>Parameters</H3></DIV></DIV><DIV id="part2"><DIV class="section" id="tab2"><H2>Deterministic models</H2><P>   The model mainly emphasizes on leakage rate of different promoters, the mRNA degradation term, and the strength of transcription, or called the rate of production. Also, transcription factors, such as repressors and activators, and coregulators are introduced to describe the dynamics of operons. In addition, the target DNA sequence is placed on Copynumber (CN) plasmids and some parameters are defined, for example, TS means the transcription strength of promoters while DeRNA represent the degradation rate of mRNA. Besides, we apply different levels of Hill input functions to model the behaviors of these regulators.<SUP>[13]</SUP></P><H3>1. Constitutive Promoter</H3><DIV class="mintext"><P>   Transcription:</P><P>   Translation:</P><P>   (</P><P>   According to the iGEM Registry, the standardized input and output PoPS and RIPS can be given by the equations below based on our models:</P><P>   At steady level, the concentration of mRNA and Protein do not change.</P><P>   Therefore, we have:</P><P>Thus,</P><P>When there is a standardized output, we have:</P></DIV><H3>2. Inducible Promoter</H3><H4><B>2.1 one operon, positive or negative inducible</B></H4><P>   The first level: no transcription factor is added.</P><P>  Here, we would introduce the leakage rate in order to estimate the production rate of the inducible promoters when no repressor or activator binds to the operon. Please note that, in negative inducible promoter, the leakage rate (LR) is almost as same as the transcription strength(TS) of the promoter in this situation.</P><P>   Negative:</P><P>   Positive:</P><P>   The second level : Promoter+n1 repressors or activators</P><H5><B>(1) Negative inducible promoter</B></H5><P>   In repression of a promoter, n repressor proteins, and nR, bind together to the operator, a DNA site of the promoter, and thus there is a decrease in the rate of transcription of the promoter, so we have the basic kinetic equations</P><P>   In steady state,</P><P>   where</P><P>   Suppose that the whole promoter regions,<B>[P<SUB>T</SUB>]</B>, consist of active regions and inactive regions while there are two repressors, both active and inactive, can blind with regions, transcription can be only occurred in active regions.</P><P>   Thus, we could conclude that the promoter activity can be defined as (exclude the leakage part)</P><P>   And we can find the ODE</P><H5><B>(2) Positive inducible promoter</B>&gt;</H5><P>   When the specific region is bound to n1 activators,</P><P>   Similarly, in steady state,</P><P>   In our model, the difference between positive and negative regulation is that positive regulated circuits is for the regions that blind with active activators and can perform transcription.</P><P>   The third level: Promoter+n1 repressors or activators+n2 inducers or corepressors.</P><P>   There are four cases at this level:</P><H5><B>(1) Repressors + Inducers: every repressor is inactivated by n inducers;</B></H5><P>   Similarly,</P><P>   Because [n1RP] &lt;&lt; [n2IR]+[R], we can apply</P><P>   In steady state,</P><P>   And</P><P>   At last we can deduce that</P><H5><B>(2) Repressors + Corepressors: every repressor is activated by n corepressors;</B></H5><P>   Similarly,</P><P>   We assume that [n2CR] takes part in the repression of the promoter. Therefore, we have</P><P>   To simplify it,</P><H5><B>(3) Activators + Corepressors: every activator is inactivated by n corepressors;</B></H5><P>Like (1), we can deduce that</P><H5><B>(4) Activators + Inducers: every activator is activated by n inducers;</B></H5><P>   We assume that [n2IR] also takes part in the activation of the promoter, so</P><P>   To simplify it,</P><P>   In conclusion, the output PoPS in these four cases can be defined as:</P><P>   Note that when there is no inducer or corepressor, [C] and [I] would be 0. However, the standardized output RIPS always remains the same. Thus, we have</P><H4><B>2.2 two operons O1 and O2 (double promoter)</B></H4><P>   It is common for prokaryotic cells to have two operons in a specific DNA site and there are also many useful double promoters in the iGEM Registry database. For example, BBa_I739104 is a double promoter (LuxR/HSL, positive / P22 cII, negative). In order to model these circuits, several rules and situations have to be considered:</P><P>   <B>(1)</B>If both O1 and O2 host repressors, they can bind either cooperatively or not.<SUP>[14]</SUP> Cooperativity between repressors implies an increase of the repressor affinity towards O2 when O1 is occupied;</P><P>   <B>(2)</B>If both O1 and O2 host activators, they can bind either cooperatively or give rise to synergistic activation, which increases the promoter strength with a consequent increment of the transcription initiation frequency;</P><P>   <B>(3)</B>O1 hosts an activator, while O2 hosts a repressor, which has proved to be easier for oscillator-building;</P><P>   <B>(4)</B>Just to be simple, we do not consider the complete competition and cooperativity among the inducers or corepressors.</P><P>   Like the example BBa_I739104 mentioned above, we can deduce the following equation for the transcription process:</P><P>   When α is the fold-change of activated transcription, n1a and n1r means the hill coefficient of the activator and repressor, respectively.</P><H3>3.Metabolic network</H3><P>   There are also cases where metabolic networks like enzymatic reactions and degradation process are involved. And metabolic reactions are also suitable for the transition from monomeric molecules to polymers like the mature of repressor protein. In these cases, we apply refined Michaelis-Menten equations to describe them.</P><P>   Case I:</P><P>   CaseⅡ:</P><P>   Case Ⅲ:</P></DIV></DIV><DIV id="part3"><DIV class="section" id="tab3"><H2>Stochastic and Time delay models</H2><H3>1.Stochastic models</H3><P>   Stochasticity in synthetic networks arises from fluctuationsin transcription and translation.First, the binding among different parts are the result of random encounters betweenmolecules, some of which are present in small numbers<SUP>[15]</SUP>; What’s more, the time evolution of a biological reacting system is not a continuous process because molecular population levels can only change by discrete integer amounts. As a result, stochastic models are introduced to describe the dynamics of single molecules.</P><P>   The most famous and common model is Gillespie Algorithm<SUP>[16]</SUP>, but the obvious disadvantages of this algorithm is that a lot of calculations has to be performed. Due to the efficiency of our software, we adopt a <B>Poisson tau-leaping algorithm</B><SUP>[17]</SUP>as following:</P><DIV class="mintext"><P>   1.Choose a time step,τ, we recommend0.1s as a time step.</P><P>   2.Multiply each term of each differential equation( the right-hand values) by τ.</P><P>   3. Apply Poisson distribution function to each result from step 2 and generate a random number using the Poisson distribution.</P><P>   4. Calculate the new number of molecules by adding or subtracting each random number from step 3, from the initial number of molecules.</P><P>   5. Repeat steps 1-4 for a specified number of reactions, N.</P><P>   To make it more stable for calculation, we divided the right side of our deterministic equations into synthetic parts and degradation parts, for example,</P><P>   CN∙TS is the synthetic part,DeRNA∙ [mRNA] is the degradation part. For each part, we generate Poisson random numbers and the other procedures remain the same. We find it more aligned with the real biological situations.</P><P>   2. time delay models</P><P>   Knowledge of molecular noise filtering and biochemical process delay compensation in synthetic circuits is crucial to understand the design of noise-tolerantand delay-robust gene circuits for synthetic biology<SUP>[18]</SUP>.A linear stochastic time-delayed model is applied in our software to mimic the realistic dynamic behavior of anetwork<SUP>[19]</SUP>.</P><P>   Where mRNA(t)=[mRNA1(t),…,mRNAn(t)]T, α0and αk denote the real-time and delay-time interactionamong these circuits, respectively, while the delay time τk rests on the specific circuits.</P></DIV></DIV></DIV><DIV id="part4"><DIV class="section" id="tab4"><H2>Cascade analysis</H2><P>   Cascades including tandem and cyclic ones are widely used in synthetic biology for different purposes. In order to get a better understanding of these circuits, we create a new iteration and optimization algorithm to calculate the fold-change in different cascades. Here is our story:  </P><P>   To begin with,a set of differential equations that represents the character of regulatory network generate an equilibrium solution, which means that the concentration of mRNA and the concentration of protein in the system wouldtend tohave fixed values after periods of time. Therefore, we conclude that the equilibrium solution and the initial conditions in this model are independent to each other and thus the equilibrium solution could be applied to represent the expression level of related circuits.</P><P>   The equilibrium solution is also applied to describe the function of activators, repressors, corepressors and inducers at a regulation rate, which is also known as fold-change, between before regulation and after regulation.</P><P>         Synthetic Cascades<SUP>[20]</SUP></P><H3>The First Level</H3><P>   If any activator or repressor does not bind to the operon, we have the set of differential equations below:</P><P>   As the initial concentration of mRNA and the initial concentration of protein have no effect on the equilibrium state, we define the initial values of both concentrations as follow:</P><P>   Then, we have</P><P>   Therefore, the equilibrium concentration of the targeted portion can be given by the following formula:</P><P>   (1)</P><P>   In the equilibrium state, there is no rate of change in any variable so the above equilibrium concentration can also be given by the following set of equations:</P><P>   Similarly, if the promoter is a positive induced promoter and there is no activator, the equilibrium concentration of the targeted protein is given by:</P><P>   (2)</P><P>   As same as the above, if the promoter is a negative induced promoter and no repressor is presented, the equilibrium concentration of the targeted protein is given by:</P><P>   (3)</P><H3>The Regulation Rate of Activator and Repressor</H3><P>   At first, we consider the computation of the regulation rate of activator.If the promoter is a positive regulated promoter and the promoter is regulated by an activator, we have the following set of differential equations:</P><P>   The concentration of activator is a parameter, which would change its value with time. However, with respect to equilibrium state, it is meaningless to consider the change in the concentration of activator. The more important we concerned is the equilibrium concentration. Therefore, we would let the concentration of activator be aconstant,or, but the set of differential equations is still complicated for solving. As what has been done above, we assume there is no rate of change in any concentration when the system is in equilibrium so we can get the following set of equation:</P><P>   Thus, the equilibrium concentration of targeted protein is:</P><P>   (4)</P><P>   Also, we understand that when the promoter is not regulated by activator, its equilibrium concentration of targeted protein should be:</P><P>   Therefore, the regulation rate of activator is:</P><P>   (5)</P><P>   Similarly, if the promoter is controlled by a negative induced promoter and regulated by repressor, the equilibrium concentration is given by:
      </P><P>   (6)</P><P>   and the regulation rate of repressor is given by:</P><P>   (7)</P><P>   where means the equilibrium concentration of repressor for the regulation of negative expressive induced promoter in such gene.</P><P>   The Regulation Rate of Corepressor and Inducer</P><P>   Applying the same method mention above, we can still find out the situation of targeted protein in the existence of activator or repressor with corepressor or inducer.</P><P>   If the promoter is controlled by an activator, which regulated by a inducer and a co-repressor, the equilibrium concentration of targeted protein can be given by:</P><P>   (8)</P><P>   While, same situation applies but, at this time, the promoter is regulated by an inducer instead of a corepressor, the equilibrium concentration of targeted protein can be given by:</P><P>   (9)</P><P>   If the promoter is controlled by a repressor and regulated by a repressor and an inducer, the equilibrium concentration of targeted protein can be given by:</P><P>   (10)</P><P>   Similarly, for corepressor regulation instead of inducer regulation, the equilibrium concentration of targeted protein can be given by:</P><P>   (11)</P><P>   Moreover, based on the computation of equilibrium concentration of targeted protein with no regulation mentioned above, we can further have the following four equations regarding the regulatory rate in the presence of corepressor and inducer:</P><P>   (12)，where</P><P>   (13)，where</P><P>   (14)，where</P><P>   (15)，where</P><H3>Computation of Regulation Rate in synthetic cascades</H3><P>   The limitation of the above equation derivation is that we assume the equilibrium concentration of activator and repressor are known values but, in a real regulatory network, there are some factors would affect those equilibrium concentrations due to the fact that activator and repressor are parts of the targeted proteins in such network. Therefore, we cannot just simply apply (1)(2)(3)to compute its equilibrium concentration and find out the related regulation rates by applying (5)(7)(12)(13)(14)(15). In the following passage, we would explore how to compute different regulation rates in a gene more effectively with respect to a general regulatory network.  And we would use  to represent the regulation rate. </P><P>   Given the network would arrange some circuits, <I>Circuit<SUB>1</SUB>,Circuit<SUB>2</SUB>,...,Circuit<SUB>N</SUB></I>, in an order. The expression of the first gene  is not regulated while the genes behind are regulated by formers regulatory proteins, i.e.  acts as an activator or a repressor on. The algorithm of regulation rate is as follow.</P><H4>Algorithm 1(tandem circuits)</H4><P>   Step 1: To compute the equilibrium concentrations of targeted proteins,[Protein<SUB>1</SUB>]<SUB>s</SUB>,[Protein<SUB>2</SUB>]<SUB>s</SUB>,...,[Protein<SUB>N</SUB>]<SUB>s</SUB>, in existence of no regulation based on equation (1)(2)(3)with respect to the type of promoter.
      </P><P>   Step2: As the expression of is not regulated, the equilibrium concentration of its target protein is [Protein<SUB>1</SUB>]<SUB>s</SUB><SUP>'</SUP>=[Protein<SUB>1</SUB>]<SUB>s</SUB></P><P>   Step 3: There is a cycle for the remaining <I>Circuit<SUB>k</SUB>(k=2,...,N)</I>. Based on the type of actual regulation, we substitute <I>Circuit<SUB>k-1</SUB></I>’s equilibrium concentration of targeted protein obtained from the former cycle into one of the following equations, (4)(6)(8)(9)(10)(11). Then, we can get ’s equilibrium concentration of targeted protein,[Protein<SUB>k</SUB>]<SUB>s</SUB><SUP>'</SUP> .</P><P>   Step 4: To compute the regulation rate  according to the values of [Protein<SUB>1</SUB>]<SUB>s</SUB>,[Protein<SUB>2</SUB>]<SUB>s</SUB>,...,[Protein<SUB>N</SUB>]<SUB>s</SUB> obtained in step 1 and the values of [Protein<SUB>1</SUB>]<SUB>s</SUB><SUP>'</SUP>,[Protein<SUB>2</SUB>]<SUB>s</SUB><SUP>'</SUP>,...,[Protein<SUB>N</SUB>]<SUB>s</SUB><SUP>'</SUP> obtained in step 2.</P><P>   Algorithm 1 can only be applied effectivelyto chain regulatory network but, in a real situation, we may need to deal with cyclic regulatory network.</P><P>   In cyclic regulatory network, every targeted protein shown on the cyclic path is responsible for one expression of the genes so it does not have such entry in algorithm 2.</P><P>   There are three factors contributing a “negative feedback” system by the regulatory network itself, which are an equilibrium solution would exist in any set of differential equation that represents its related regulatory network; the degradation of each component; and the regulatory relationships between genes. The “negative feedback” system is a system that to maintain the expression level of various genes on an equilibrium solution by regulating the network when the concentration of targeted protein in one of the genes is higher or lower than the equilibrium solution. Based on this concept, we propose another algorithm for computing the regulation rates in a general network. </P><H4>Algorithm 2(cyclic circuits)</H4><P>   Step 1: To compute the equilibrium concentrations of targeted proteins,[Protein<SUB>1</SUB>]<SUB>s</SUB>,[Protein<SUB>2</SUB>]<SUB>s</SUB>,...,[Protein<SUB>N</SUB>]<SUB>s</SUB> , in existence of no regulation based on equation (1)(2)(3)with respect to the type of promoter.</P><P>   Step 2: To assume the initial concentrations of targeted proteins in <I>Circuit<SUB>1</SUB>,Circuit<SUB>2</SUB>,...,Circuit<SUB>N</SUB></I> are zero, </P><P>   Step 3: To make a cycle for each gene,<I>Circuit<SUB>n</SUB>(n=1,...,N)</I> , in the network. If <I>Circuit<SUB>n</SUB></I>  is regulated by <I>Circuit<SUB>m</SUB></I>, we substitute the <I>Circuit<SUB>m</SUB></I>’s concentration of targeted protein obtained from the former cycle into one of the following equations, (4)(6)(8)(9)(10)(11), in order to compute the regulated <I>Circuit<SUB>n</SUB></I>’s concentration of targeted protein.</P><P>   Step 4: If the relative error of ([Protein<SUB>1</SUB>]<SUB>k</SUB>,[Protein<SUB>2</SUB>]<SUB>k</SUB>,...,[Protein<SUB>N</SUB>]<SUB>k</SUB>) and ([Protein<SUB>1</SUB>]<SUB>k-1</SUB>,[Protein<SUB>2</SUB>]<SUB>k-1</SUB>,...,[Protein<SUB>N</SUB>]<SUB>k-1</SUB>) is less than a fixed threshold, or the number of iterations k is greater than a set limit, then please go back to step 3. Otherwise, please continue to step 5.</P><P>   Step 5: To consider each genes’ concentration of targeted protein found in the last iteration as the equilibrium concentration. Therefore, we have  [Protein<SUB>1</SUB>]<SUB>s</SUB><SUP>'</SUP>,[Protein<SUB>2</SUB>]<SUB>s</SUB><SUP>'</SUP>,...,[Protein<SUB>N</SUB>]<SUB>s</SUB><SUP>'</SUP>. </P><P>   Moreover, with [Protein<SUB>1</SUB>]<SUB>s</SUB>,[Protein<SUB>2</SUB>]<SUB>s</SUB>,...,[Protein<SUB>N</SUB>]<SUB>s</SUB> obtained from step 1, we can compute the regulation rate,.</P><P>   Although this algorithm is just an approximation of regulation rate, the result obtained is highly consistent with simulation by numerical method and the accuracy would be increased after several times of iteration. Therefore, we may conclude this method is more effective and reliable.</P></DIV></DIV><DIV id="part5"><DIV class="section" id="tab5"><H2>References</H2><DIV class="mintext"><P>[1]  Priscilla E. M. Purnick and Ron Weiss ,The second wave of synthetic biology: from modules to systems,Nature, 2009 ,10,410-422</P><P>[2]  <A herf="https://2009.igem.org/Team:Aberdeen_Scotland/parameters" target="blank">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></P><P>[3]  Barry Canton1, Anna Labno, Drew Endy, Measuring the activity of BioBrick promoters using an in vivo reference standard,Nature Biotechnolgy,2008,26(7),787-793</P><P>[4]  Kelly JR.el, Measuring the activity of BioBrick promoters using an in vivo reference standard, J Biol Eng. 2009,3</P><P>[5]  <A href="https://2009.igem.org/Team:Groningen/Promoters" target="blank">https://2009.igem.org/Team:Groningen/Promoters</A></P><P>[6]  Iadevaia S., and Mantzaris N.V., Genetic network driven control of PHBV copolymer composition, Journal of Biotechnology, 2006, 122(1), 99-121</P><P>[7]  Braun D., Basu S., and Weiss R., Parameter Estimation for Two Synthetic Gene Networks: A Case Study, IEEE Int Conf. Acoustics, Speech, and Signal Processing 2005, 5</P><P>[8]  J. De Anda, A. Poteete, and R. Sauer, P22 c2 repressor:Domain structure and function,  J. Biol. Chem. 1983,258(17)</P><P>[9]  <A href="http://parts.igem.org/Help:Terminators/Measurement" target="blank">http://parts.igem.org/Help:Terminators/Measurement</A></P><P>[10] <A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters" target="blank">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></P><P>[11] <A href="https://2009.igem.org/Team:Groningen/Promoters" target="blank">https://2009.igem.org/Team:Groningen/Promoters</A></P><P>[12] <A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters" target="blank">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></P><P>[13] Alon, Uri. “An Introduction to Systems Biology Design Principles of Biological Circiuts.” London: Chapman &amp; Hall/CRC, 2007</P><P>[14] Mario Andrea MarchisioA guide to composable parts and pools, ETHZ,2008</P><P>[15] MadsKaern, Timothy C. Elston, William J. Blakeand James J. Collins,STOCHASTICITY IN GENEEXPRESSION: FROM THEORIESTO PHENOTYPES, NATURE REVIEWS GENETICS，2005，451-464</P><P>[16] Gillespie, Approximate accelerated stochastic simulation of chemically reacting systems. Journal of Chemical Physics, 115:1716-1733</P><P>[17] <A href="https://2010.igem.org/Team:Aberdeen_Scotland/Stochastic_Model" target="blank">https://2010.igem.org/Team:Aberdeen_Scotland/Stochastic_Model</A></P><P>[18] Bor-Sen Chen and Yu-Te ChangA systematic molecular circuit design method for gene networksunder biochemical time delays and molecular noises, BMC Systems Biology 2008, 2:103</P><P>[19] T. Tian, K.Burrage, P. M. Burrage and M. Carletti,Stochastic Delay Differential Equations for Genetic Regulatory Networks, Special Issue of J. Comp and Applied Maths,2006</P><P>[20] Gregory Batt, BoyanYordanov, Ron Weissand CalinBeltaRobustness analysis and tuning of synthetic gene networks,BIOINFORMATICS, 2007,23 ,2415–2422</P></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:SYSU-Software/model">http://2013.igem.org/Team:SYSU-Software/model</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SYSU-Software/model" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SYSU-Software/model" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SYSU-Software/model&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SYSU-Software/model&amp;oldid=346604" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>