<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Vilnius-Lithuania_Modeling skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Vilnius-Lithuania/Modeling</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="content_wrapper"><DIV class="navMenu fix-Menu"><DIV class="menuCentered"><UL><LI class="dropdown"><A href="https://2016.igem.org/Team:Vilnius-Lithuania/People" class="dropbtn">PEOPLE</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Members">Members</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Attributions">Attributions</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Collaborations">Collaborations</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Project" class="dropbtn">PROJECT</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Description">Description</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Design">Design</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Modeling">Modeling</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Results">Results</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Proof">Proof of concept</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Notebook">Notebook</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Methods">Methods</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Safety">Safety</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Achievements">Achievements</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Parts" class="dropbtn">PARTS</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Basic_Part">Basic parts</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Composite_Part">Composite parts</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Human_Practices" class="dropbtn">HUMAN PRACTICES</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Vilnius-Lithuania/HP/Gold">Gold Criterion</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/HP/Silver">Silver Criterion</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Integrated_Practices">Integrated Practice</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Engagement">Engagement</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/HP/iGEM/">Successful iGEM team</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Human_Practices/all">All</A></DIV></LI><LI><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Interlab">INTERLAB STUDY</A></LI><LI class="dropdown"><A href="#ACKNOWLEDGEMENTS" class="dropbtn">ACKNOWLEDGEMENTS</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Sponsors">Sponsors</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Crowdfunding">Crowdfunding</A></DIV></LI></UL></DIV></DIV><DIV class="mainBody"><DIV class="column full_size"><H1 class="Raleway" style="text-align:center;">Modelling</H1><DIV style="margin:2em;"><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Description">Description</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Design">Design</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Modeling" class="active">Modelling</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Results">Results</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Proof">Proof</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Notebook">Notebook</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Safety">Safety</A></DIV><P class="Raleway">Mathematical modelling of the natural systems is often a necessity in understanding the probabilities of certain outcomes or dependencies of certain factors on the whole system in experimental science. Such models have a vast application, as they can provide the information based on the interaction of multiple systems, which cannot be achieved in the laboratory conditions. It is essential, that mathematical modelling is of crucial importance in the field of synthetis biology, as it helps to predict all the possible outcomes of a particular novel approach. Mathematical models are useful not only in describing natural systems, but also in predicting the response of a system at various conditions. Here we present the mathematical and kinetic model of a part of our engineered system: a bacteria, which expresses a recombinant enzyme PAL (phenylalanine ammonia lyase) and phenylalanine channels. The purpose of the model is to define the efficiency of the system and the limits of its performance as it is crucial to ensure that the approach is prospective and liable. </P><P class="Raleway">We were very happy to have the opportunity to collaborate with Oxford iGEM team on the modelling part and would like to thank iGEM for providing the possibilities of such interraction.</P><H2 class="Raleway">Systems</H2><P class="Raleway">This year‘s project focuses on <I>Escherichia coli</I>, which acts as a probiotic that counteracts the presence of phenylalanine in the intestine of the patient with PKU. We have developed two systems for this purpose. </P><P class="Raleway">The first system incorporates phenylalanine ammonia lyase (PAL) for the conversion of phenylalanine to trans-cinnamic acid (TCA). As this enzyme is overexpressed inside the bacteria, constantly expressed PheP transporters are facilitating the diffusion, since the concentration of phenylalanine in the bacteria is always deficient. The development of this model was a part of collaboration with Oxford iGEM team. 
The second system, in addition, incorporates the reliable mechanisms of the central dogma to produce a phenylalanine-rich protein that acts as a phenylalanine „sponge“. This system is more reliable than the enzymatic approach and could act in the background. However, for the bacteria to be able to express large amounts of synthetic protein saturated with phenylalanine, the protein itself has to be stable enough and the bacterial machinery has to be suited for the high frequency of phenylalanine codons. We used mathematical approaches (hyperlink) to acquire stable phenylalanine mutants of natural proteins and simulated the efficiency of this system under elevated expression of tRNA-Phe. </P><H2 class="Raleway">Kinetic model</H2><P class="Raleway">We designed two different in vivo systems to decrease the amount of phenylalanine in the intestine. The first system is based on phenylalanine ammonia lyase (PAL), which breaks down phenylalanine to trans-cinnamic acid and ammonia. The second system incorporates phenylalanine into specific mutant proteins that possess high amounts of phenylalanine, thus lowering the concentration of the free amino acid. </P><H2 class="Raleway">Uptake</H2><P class="Raleway">As both of the systems involve the uptake of phenylalanine into the bacteria, the kinetic equations can be applied to either of the systems. 
There are two ways for phenylalanine to enter the cell: 
1.	Passive diffusion; 
2.	Facilitated diffusion by phenylalanine transporter. 
The kinetics of these pathways can be described by classical equations: 
1.	For passive diffusion:</P><P class="Raleway">
2.	For facilitated diffusion: </P><P class="Raleway">The PheP coding genes are expressed constantly, reaching a steady state of the transporter in the membrane, which can be described by the following equation: 
</P><P class="Raleway">where α is the translation rate, β is the degradation and dilution rate of PheP, K is the transcription rate and δ is the degradation and dilution rate of mRNA.  
Together, PheP expression rate and the activity of the transporter dictates the rate of facilitated transport into the cells. Later, by combining both passive and facilitated diffusion rate equations we can determine the rate of phenylalanine uptake.</P><H2 class="Raleway">PAL</H2><P class="Raleway">Bacteria with the first system expresses phenylalanine ammonia lyase (PAL), an enzyme, which breaks down phenylalanine into trans-cynnamic acid and ammonia. This enzyme is either expressed at an induced or constant rate inside the bacteria, so in the latter conditions, a steady state concentration of PAL should be reached.  
The rate of convertion of Phenylalanine to TCA for different concentrations of PAL can be calculated from the following equation:</P><P class="Raleway">We have carried out experiments to acquire some of the constants regarding our model (see wet lab page). If the expression rate and activity of PAL and phenylalanine diffusion rate is known, the process of phenylalanine hydrolysis inside the cell can be discribed by the following equations: </P><H2 class="Raleway">Simulation of phenylalanine hydrolysis <I>in vivo</I></H2><P class="Raleway">We first simulated the break down of phenylalanine by PAL being expressed within the cell. The steady state concentration of PAL inside the cell is assumed to be in the range between 10-4 and 10-7 M. Here the four versions of graphs for each magnitude of PheP steady state concentration, ranging from 10-3 to 10-6 M are shown. The amounts of phenylalanine inside the cell, outside the cell, and the amount of trans-cynnamic acid (in or out?) are presented in the graphs. </P><P class="Raleway">Comparison of transport rate of phenylalanine and production rate of TCA under different concentration conditions of PheP and PAL steady state.</P><H2 class="Raleway">Discussion</H2><P class="Raleway">It is clear that the concentration of PheP has the biggest influence on the efficiency of probiotic. If the cell expressed sufficient amounts of the transporter, it could efficiently reduce the concentration of phenylalanine under laboratory conditions.  
With this in mind, phep gene under a constitutive promoter was cloned to enhance the uptake of phenylalanine. However, it is not likely that such amounts of PheP will be reached; therefore, alternative measures that could increase the membrane permeability should be applied. 
As for PAL, it can be seen in graph 2 that the concentration of PAL needs to be in the range between 10-5 and 10-4 M for an observable phenylalanine concentration decrease. As PheP assists the diffusion of phenylalanine into the cell, it then depends on the rate of phenylalanine breakdown.</P><H2 class="Raleway">Readjustment of the model by PAL expression experiment</H2><P class="Raleway">The model having shown that we will need a certain amount of PAL expression to have a decent conversion of TCA, we proceeded to measuring expression of PAL. The wet lab showed that there is increase in expression of PAL that flattens out after approximately 4 hours; this curvature can be approximated by an exponential function:</P><P class="Raleway">which is the solution of:</P><P class="Raleway">For OD600 = 1, the corresponding cell DW is ~ 0.39g/L. Therefore, the maximum expression of PAL can be approximated as:</P><P class="Raleway">and δ to be approximately:</P><P class="Raleway">Now that we know the approximate PAL production over time, we can make the model reflect that in order to make it more accurate. We created a new model incorporating the PAL production equation to produce the following graphs for possible concentrations for PheP within cell.</P><P class="Raleway">A more comprehencable graph is shown below, taking out the TCA concentration corresponding to each different concentrations of PheP.</P><P class="Raleway">From experimental data (see below), we showed that TCA concentration after 4 hours is about 20 umol/g DW, which corresponds to:</P><P class="Raleway">Experiment results:</P><P class="Raleway">Reaction took place in 20 ml of substrate medium with 25mM of L-phenylalanine for 20 min at 30⁰C. AvPAL was expressed for 4 hours before the experiment to reach a steady state concentration within the cells.</P><H2 class="Raleway">Discussion</H2><P class="Raleway">Comparing the model and experimental data indicates that the concentration of PheP in cell is in the order of 10^1 uM. This value is promissing in that it is a realistic concentration for a transporter protein on the cell membrane. However, we have shown in the previous model that we require over 10^2 uM PheP  expression in order to have a significant decrease in Phenylalanine concentration outside the cell.</P><P class="Raleway">There are two indications:</P><P class="Raleway">1.	The current design may be insufficient to mop phenylalanine so that it may have to be readjusted to suit practical application.</P><P class="Raleway">2.	The method we used to model passive diffusion may have been underestimating the intake of phenylalanine into the cell, in which case convertion into TCA will happen in a shorter period than is indicated in the model.</P></DIV></DIV><FOOTER class="footerholder "><DIV class="footer"><DIV class="centered"><A href="https://2016.igem.org/Special:RecentChanges">Recent changes</A><A href="https://2016.igem.org/Special:WhatLinksHere/Team:Vilnius-Lithuania">What links here</A><A href="https://2016.igem.org/Special:SpecialPages">Special pages</A></DIV><DIV class="centered"><A href="https://2016.igem.org/Special:Preferences">My preferences</A><A href="https://2016.igem.org/2016.igem.org:Privacy_policy">Privacy policy</A><A href="https://igem.org/General_disclaimer">Disclaimer</A></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>