<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Vilnius-Lithuania_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Vilnius-Lithuania/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="content_wrapper"><DIV class="navMenu fix-Menu"><DIV class="menuCentered"><UL><LI class="dropdown"><A href="https://2016.igem.org/Team:Vilnius-Lithuania/People" class="dropbtn">PEOPLE</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Members">Members</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Attributions">Attributions</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Collaborations">Collaborations</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Project" class="dropbtn">PROJECT</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Description">Description</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Design">Design</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Modeling">Modeling</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Results">Results</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Proof">Proof of concept</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Notebook">Notebook</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Methods">Methods</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Safety">Safety</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Achievements">Achievements</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Parts" class="dropbtn">PARTS</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Basic_Part">Basic parts</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Composite_Part">Composite parts</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Human_Practices" class="dropbtn">HUMAN PRACTICES</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Vilnius-Lithuania/HP/Gold">Gold Criterion</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/HP/Silver">Silver Criterion</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Integrated_Practices">Integrated Practice</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Engagement">Engagement</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/HP/iGEM/">Successful iGEM team</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Human_Practices/all">All</A></DIV></LI><LI><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Interlab">INTERLAB STUDY</A></LI><LI class="dropdown"><A href="#ACKNOWLEDGEMENTS" class="dropbtn">ACKNOWLEDGEMENTS</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Sponsors">Sponsors</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Crowdfunding">Crowdfunding</A></DIV></LI></UL></DIV></DIV><DIV class="mainBody"><DIV class="column full_size"><H1 class="Raleway" style="text-align:center;">Results</H1><DIV style="margin:2em;"><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Description">Description</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Design">Design</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Modeling">Modelling</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Results" class="active">Results</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Proof">Proof</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Notebook">Notebook</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Safety">Safety</A></DIV><P class="Raleway">The goal of experiments was to decrease phenylalanine by using few different approaches. The first approach is boosting cells consumption of target amino acid by transforming a vector with a gene of PAL (phenylalanine ammonia-lyase) enzyme, which converts L-phenylalanine into trans-cinnamic acid (tCA). The other approach has the goal to increase cells needs of target amino acid by introducing it with a novel synthetic gene enriched with phenylalanine codons.</P><H2 class="Raleway">PAL (Phenylalanine ammonia-lyase)</H2><P class="Raleway">After obtaining our synthesized <I>Anabaena variabilis</I> phenylalanine ammonia-lyase (PAL) gene, we cloned it into two protein expression vectors: pET (with T7 promoter) and pBAD with (araBAD promoter). By transforming pET recombinant plasmid into Arctic Express (DE3), NovaBlue (DE3), Rosetta (DE3), BL21 (DE3), BL21-AI, C41 (DE3) <I>E. coli</I> strains, which are suitable for T7 promoter, and recombinant pBAD vector into DH10B and TOP10 <I>E. coli</I> strains, recommended for araBAD promoter, we expressed our recombinant protein and ran the SDS-PAGE and Western Blot in order to evaluate the best expression conditions. The result of PAL expression is shown in Fig. 1 and 2. By evaluating the appearance of protein, it is observed that in pBAD vector both C-terminal 6xHis tag and C&amp;N-terminal 6xHis tag proteins were expressed. Further in the experiments only C-his tag PAL protein in pET vector were tested. Even though PAL expression under araBAD promoter was hardly noticeable in SDS-PAGE, Western Blot results proved production of the protein. We made an assumption that the best expression of PAL protein is in <I>E.coli</I> TOP10 strain. After result evaluation of expression with pET-PAL plasmids and comparing it with pBAD- PAL plasmids a significantly higher expression of PAL protein appeared in most of recombinant <I>E.coli</I> strains with pET-PAL-C-terminal 6xHis tag plasmids, particularly in NovaBlue (DE3) strain. Further PAL enzyme activity experiments were performed with pBAD-PAL-C- terminal 6xHis tag in <I>E.coli</I> TOP10 strain and pET-PAL-C- terminal 6xHis tag in <I>E.coli</I> NovaBlue (DE3) strain.</P><DIV class="inlBlock80"><P class="Raleway">Figure 1. SDS-PAGE and Northern Blot results of PAL in pBAD expression system. Red star indicates the place of target protein. A 12 % SDS-PAGE analysis of PAL expression levels under araBAD promoter in different <I>E. coli</I> strains induced by L-arabinose (0.2 %, w/v). Odd numbers represent soluble protein fraction, even numbers represent insoluble fraction. B Western Blot analysis of PAL expression levels under araBAD promoter in different <I>E. coli</I> strains induced by L-arabinose (0.2 %, w/v). Odd numbers represent soluble protein fraction, even numbers represent insoluble fraction.</P></DIV><DIV class="inlBlock80"><P class="Raleway">Figure 2. SDS-PAGE and Northern Blot results of PAL-Chis in pET expression system. Red star indicates the place on of target proteins. C – control (any purified 35 kDa protein with His tag). A 12 % SDS-PAGE analysis of PAL expression levels under T7 promoter in different <I>E. coli</I> Arctic Express (DE3), NovaBlue(DE3), Rosetta (DE3) strains induced by IPTG (2 μM final concentration). Odd numbers represent soluble protein fraction, even numbers represent insoluble fraction. B 12 % SDS-PAGE analysis of PAL expression levels under T7 promoter in different <I>E. coli</I> BL21 (DE3), BL21-AI, C41 (DE3) strains induced by IPTG (2 μM concentration). Odd numbers represent soluble protein fraction, even numbers represent insoluble fraction. C Western Blot analysis of PAL expression levels under T7 promoter in Arctic Express (DE3), NovaBlue(DE3), Rosetta (DE3) <I>E. coli</I> strains induced by IPTG (2 μM final concentration). Odd numbers represent soluble protein fraction, even numbers represent insoluble fraction. D Western Blot analysis of PAL expression levels under T7 promoter in BL21 (DE3), BL21-AI, C41 (DE3) E. coli strains induced by IPTG (2 μM final concentration). Odd numbers represent soluble protein fraction, even numbers represent insoluble fraction.</P></DIV><P class="Raleway">After identification of the best strains for different PAL expression systems, we performed an experiment to identify, which version of recombinant <I>E. coli</I> best fulfills our intention to create recombinant probiotic to quickly decrease the concentration of L-phenylalanine in human intestine by transforming L-phenylalanine into tCA. The PAL activity in recombinant <I>E. coli</I> cells was tested by measuring tCA. Results in Fig. 3 show that cells with PAL enzyme reduce the concentration of L-phenylalanine more rapidly than control cells (the same strain of <I>E. coli</I> without PAL enzyme). In Fig. 3A can be found information how TOP10 <I>E. coli</I> cells, unaffected with any detergents, break down L-phenylalanine into trans-cinnamic acid. In addition to this experiment, we assumed that a bottleneck effect might occur because of insufficient L- phenylalanine transportation from extracellular environment into cell. For this reason, we measured the effect of detergent (Tween-20) and solvent (ethanol) to increase the permeability of the cell membrane. The results showed that the treatment strategy with 60 % ethanol was more effective to enhance the activity of PAL than ethanol or Tween-20 solutions with different concentrations. As for Tween-20, the best result was achieved in 0.3 g/l of detergent. Enzyme activity was increased by 7-folds with cells unaffected by any detergent or solvent, almost by 13- fold when culture was enhanced with 60 % (v/v) ethanol in comparison with that of the control. Fig. 3B, showed the effects of different concentration of surfactant and solvent on PAL activity in recombinant <I>E. coli</I> NovaBlue (DE3). The influence of Tween-20 and ethanol is the same as in the <I>E. coli</I> TOP10 with PAL, but the overall activity of PAL in pET vector and NovaBlue (DE3) strain is almost 2 times weaker than in pBAD-PAL TOP10 systems. The reason of this result could be that big proportion of protein in pET expression system appeared in insoluble fraction where protein is ineffective. The enhanced activity with surfactant and solvent proved the hypothesis about bottleneck effect was right, for this reason further plan was to incorporate both PAL and pheP (natural <I>E. coli</I> membrane permease which transports phenylalanine) biobricks together to achieve the best result with natural biosystem and because overall PAL activity with pBAD protein expression system in TOP10 E. coli strain was better than in pET vector we decided to do not spend more time on experiments with system which showed weaker performance.</P><P class="Raleway">After the best system was chosen, the PAL and PAL combined with pheP (natural <I>E. coli</I> permease, which transports phenylalanine) activity in <I>E. coli</I> was tested in reaction mixture which mimics the total amount of adult’s L-Phenylalanine average daily intake (around 1,100 mg/day) and the pH of small intestine, a place where our probiotics will perform in a final version. Fig 4. demonstrates collected results after measurements of our system in previously mentioned conditions. The activity of PAL was better by around 6 percent when cell was enhanced with pheP biobrick, in comparison with <I>E. coli</I> without permease, by reaching the reduction of more than 25 percent of daily phenylalanine intake. This result is better by 2-fold than activity in control. This result shows that our incorporated membrane transporter for phenylalanine was valuable for our whole system. The overall result indicates that a patient suffering from PKU could increase his daily L-Phenylalanine consumption by using probiotics with the same system.</P><DIV class="inlBlock80"><P class="Raleway">Figure 3. Activity of PAL in different expression systems and different environments during 20 minute period in pH 8.8. A The effects of different concentrations of ethanol and Tween-20 on PAL activity in <I>E. coli</I> with pBAD-PAL-C-his. C – TOP10 <I>E. coli</I>. B The effects of different concentrations of ethanol and Tween-20 on PAL activity in <I>E. coli</I> with pET- PAL-C-his. C – NovaBlue (DE3) <I>E. coli</I>.</P></DIV><DIV class="inlBlock80"><P class="Raleway">Figure 4. Activity of PAL and PAL with pheP in <I>E. coli</I> cell over a period of 20 minutes in pH 7.4. The activity is evaluated by production of tCA. The initial amount of L-phenylalanine in reaction mixture was 1.1g. Every system which was tested during this experiment was transformed into <I>E. coli</I> TOP10. Control - <I>E. coli</I> TOP10 strain without PAL and pheP biobrick. The total mass of recombinant cells in reaction mixture was 5g.</P></DIV><DIV class="inlBlock80"><P class="Raleway">Figure 5. A list of created synthetic proteins derived from GP45 (phage-associated sliding clamp DNA polymerase accessory protein) and Csm4 (CRISPR type III-associated RAMP protein Csm4) with the information about total percentage of phenylalanine incorporated into each different protein.</P></DIV><H2 class="Raleway">Polyphe proteins</H2><P class="Raleway">In order to create synthetic protein, we ordered optimized DNA sequences, which were generated by using our created modelling systems. Those synthetic genes were divided in two groups: those which were derived from a highly expressible protein named Gp45 (phage- associated sliding clamp DNA polymerase accessory protein) and from Csm4 (CRISPR type III- associated RAMP protein Csm4). All in all, we created 5 different synthetic proteins by using Gp45 sequence and 3 new proteins derived from Csm4 and we named them Polyphe, because they are highly enriched in phenylalanine codons. Fig. 5 shows the information about every type of protein. Firstly, we tested the level of expression of every synthetic protein by SDS-PAGE in order to select only those sequences which have the highest percentage of phenylalanine codon and the best expression. Fig. 6 represents the results obtained for evaluation of production of different synthetic proteins in a few <I>E. coli</I> strains. Every SDS-PAGE assay was made by loading equal volumes of samples. By knowing the most suitable ways of expression we tested BL21 (DE3) and C41 (DE3) <I>E. coli</I> strains to validate expression levels of modified Gp45 proteins and the same BL21 (DE3) with ER2566 <I>E. coli</I> strains to test production of modified Csm4 proteins. Only in BL21 (DE3) <I>E. coli</I> strain we managed to get significant amount Gp45 (37) and this system will be used in future measurements. As for other modified protein, the best results were achieved in ER2566 <I>E. coli</I> strain. Csm4(11) and Csm4 (42) proteins were expressed in reasonable amount and in further experiment those two types of systems will be used. After the evaluation of the best expression with the most amount of phenylalanine incorporated into synthetic protein three representatives were chosen GP45 (37), Csm4 (11) and Csm4 (42).</P><DIV class="inlBlock80"><P class="Raleway">Figure 6. SDS-PAGE results of Polyphe in pET expression system. Red star indicates the place on of target proteins. A 12 % SDS-PAGE analysis of modified Gp45 protein expression levels in <I>E. coli</I> BL21 (DE3) induced by IPTG (2 μM final concentration) Odd numbers represent soluble protein fraction, even numbers represent insoluble fraction. B 12 % SDS-PAGE analysis of modified Gp45 protein expression levels in <I>E. coli</I> C41 (DE3) induced by IPTG (2 μM final concentration). Odd numbers represent soluble protein fraction, even numbers represent insoluble fraction. C 12 % SDS-PAGE analysis of modified Csm4 protein expression levels in <I>E. coli</I> BL21 (DE3) induced by IPTG (2 μM final concentration). Odd numbers represent soluble protein fraction, even numbers represent insoluble fraction. D 12 % SDS-PAGE analysis of modified Csm4 protein expression levels in <I>E. coli</I> ER2566 induced by IPTG (2 μM final concentration. Odd numbers represent soluble protein fraction, even numbers represent insoluble fraction.</P></DIV><P class="Raleway">For further improvement of our system a hypothesis had been constructed that the production of our synthetic proteins could be increased by overexpressing tRNA-Phe, because our synthetic genes are filled with phenylalanine codons and there could be a state where the insufficient amount of particular tRNA-Phe could lead to lower speed of translation of our created protein. We created a tRNA-Phe biobricks with two different constitutive promoters (BBa_K1983011 and BBa_K1983011). First of all, to identify the transcription of our target tRNA Norther Blot was performed Fig. 7. By using two different probes, one for mature tRNA and other for immature tRNA. We managed to detect the increased expression of our target mature tRNA in cells.</P><P class="Raleway">Furthermore, to confirm our idea of insufficient amount of tRNA-Phe in cell when synthesizing Polyphe protein we chose to test difference in expression of those types proteins which have the biggest amount of incorporated phenylalanine: Gp45 (37) and Csm4 (42). We transformed a device containing a Polyphe and tRNA-Phe into <I>E. coli</I> BL21 (DE3) strain and tested difference in the production of proteins by SDS-PAGE. For our great disappointment there could not be seen any significant increase in production of target proteins over 3-hour period when compared with controls Fig. 8. This result concludes that the overexpression of tRNA-Phe is not the reason why Polyphe proteins are expressed in lower quantities.</P><DIV class="inlBlock80"><P class="Raleway">Figure 7. Northern Blot of tRNA-Phe by comparing two different promoters. Red star indicates place of target tRNA. A A detection of tRNA-Phe with selective hybridization probe for mature tRNA-Phe in two different <I>E. coli</I> strains. B A detection of tRNA-Phe with selective hybridization probe for immature tRNA-Phe in two different <I>E. coli</I> strains.</P></DIV><DIV class="inlBlock80"><P class="Raleway">Figure 8. Effect of tRNA-Phe on Gp45 (37) and Csm4 (42) expression level. Red star indicates the size of target protein. A 12 % SDS-PAGE analysis of modified Csm4 (42) protein expression levels in <I>E. coli</I> ER2566 (DE3) induced by IPTG (2 μM final concentration). Lane 1 Csm4 (42) after 1h, Lane 2 Csm4 (42) + tRNA-PheC after 1h, Lane 3 Csm4 (42) + tRNA-PheD after 1h, Lane 4 Csm4 (42) after 3h, Lane 5 2 Csm4 (42) + tRNA-PheC after 3h, Lane 6 Csm4 (42) + tRNA-PheD after 3h. B 12 % SDS-PAGE analysis of modified Gp45 (37) protein expression levels in <I>E. coli</I> BL21 (DE3) induced by IPTG (2 μM final concentration). Lane 1 Gp45 (37) after 1h, Lane 2 Gp45 (37) + tRNA-PheC after 1h, Lane 3 Gp45 (37) + tRNA-PheD after 1h, Lane 4 Gp45 (37) after 3h , Lane 5 Gp45 (37) + tRNA-PheC after 3h, Lane 6 Gp45 (37) + tRNA-PheD after 3h.</P></DIV><P class="Raleway">Finally, the overall result of highly enriched proteins was evaluated by measuring the reduction of L-Phenylalanine concentration in <I>E. coli</I> growth medium at pH 7.4, 37ºC over a 30-minute period (Fig. 9). The expression of every synthetic protein increases the consumption of target amino acid in E. coli more than 2-fold in comparison with control. The results show that the activity in reduction of L-Phe of Gp45 (37) is the best (more than 19% of daily phenylalanine intake (1,100 mg)) by overcoming Csm4 (11) and Csm4 (42) just by a few percents.</P><DIV class="inlBlock80"><P class="Raleway">Figure 9. Cell effectiveness in reduction of phenylalanine in growth medium while expressing Polyphe over 30-minute period. Gp45 (37) were expressed in <I>E. coli</I> BL21, Csm4 (11) and Csm4 (42) in ER2566. The whole expression of proteins was controlled by T7 promoter and cells were induced with IPTG (final concentration 2 μM). Control is <I>E.coli</I> cells without any biobrick.</P></DIV><H2 class="Raleway">Conclusions</H2><P class="Raleway">The overall results achieved by trying to create a system which demonstrates higher activity in reduction of L-phenylalanine show positive results in both PAL and Polyphe approaches. By showing that pheP transporter works as a great natural enhancer we demonstrated that today’s result will be improved by additional cellular mechanism. Even though every approach has its own different benefits and disadvantages, in the future they will be combined into one functional device which will have a potential to reach new peaks of its activity by performing in life’s of PKU victims.</P></DIV></DIV><FOOTER class="footerholder "><DIV class="footer"><DIV class="centered"><A href="https://2016.igem.org/Special:RecentChanges">Recent changes</A><A href="https://2016.igem.org/Special:WhatLinksHere/Team:Vilnius-Lithuania">What links here</A><A href="https://2016.igem.org/Special:SpecialPages">Special pages</A></DIV><DIV class="centered"><A href="https://2016.igem.org/Special:Preferences">My preferences</A><A href="https://2016.igem.org/2016.igem.org:Privacy_policy">Privacy policy</A><A href="https://igem.org/General_disclaimer">Disclaimer</A></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>