<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Vilnius-Lithuania_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Vilnius-Lithuania/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="content_wrapper"><DIV class="navMenu fix-Menu"><DIV class="menuCentered"><UL><LI class="dropdown"><A href="https://2016.igem.org/Team:Vilnius-Lithuania/People" class="dropbtn">PEOPLE</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Members">Members</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Attributions">Attributions</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Collaborations">Collaborations</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Project" class="dropbtn">PROJECT</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Description">Description</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Design">Design</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Modeling">Modeling</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Results">Results</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Proof">Proof of concept</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Notebook">Notebook</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Methods">Methods</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Safety">Safety</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Achievements">Achievements</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Parts" class="dropbtn">PARTS</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Basic_Part">Basic parts</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Composite_Part">Composite parts</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Human_Practices" class="dropbtn">HUMAN PRACTICES</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Vilnius-Lithuania/HP/Gold">Gold Criterion</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/HP/Silver">Silver Criterion</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Integrated_Practices">Integrated Practice</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Engagement">Engagement</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/HP/iGEM/">Successful iGEM team</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Human_Practices/all">All</A></DIV></LI><LI><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Interlab">INTERLAB STUDY</A></LI><LI class="dropdown"><A href="#ACKNOWLEDGEMENTS" class="dropbtn">ACKNOWLEDGEMENTS</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Sponsors">Sponsors</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Crowdfunding">Crowdfunding</A></DIV></LI></UL></DIV></DIV><DIV class="mainBody"><DIV class="column full_size"><H1 class="Raleway" style="text-align:center;">Design</H1><DIV style="margin:2em;"><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Description">Description</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Design" class="active">Design</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Modeling">Modelling</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Results">Results</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Proof">Proof</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Notebook">Notebook</A><A href="https://2016.igem.org/Team:Vilnius-Lithuania/Safety">Safety</A></DIV><P class="Raleway">Our project has three main approaches which enable more efficient and liable performance of the designed probiotic.</P><H2 class="Raleway">PAL</H2><P class="Raleway">The first approach is described by the expression of phenylalanine metabolizing enzyme â€“ phenylalanine ammonia lyase (PAL). Originally, human body uses another enzyme called phenylalanine hydroxylase (PAH) to break down this amino acid; however, this enzyme requires oxygen as a co-factor and acts in the bloodstream. Normally, oxygen is not abundant in the intestinal tract, so PAL happens to be more applicable. When breaking down phenylalanine, cinammic acid and ammonia are produced, both of which are considered to be harmless to human body in the occuring concentrations.</P><P class="Raleway">For this approach, PAL sequence from already existing biobrick in iGEM parts registry was used <A href="http://parts.igem.org/Part:BBa_K1692004">K1692004</A>. We have slightly changed the sequence by adding His-Tags to C-terminal, N-terminal and to both C- and N-termini instead of the Flag-Tags which originally existed in the biobrick. His-Tags were added with the help of oligonucleotide primers.</P><H2 class="Raleway">Polyphe proteins</H2><P class="Raleway">This approach can be viewed as a distinctive creation of synthetic biology. We have designed a unique tool that minimizes the concentrations of phenylalanine in the medium. This tool is comprised of a synthetic gene, which is rich in phenylalanine codons. Therefore, the expressed gene will use the majority of phenylalanine in the surroundings during the process of translation.</P><P class="Raleway">In order to find the most stable folds having multiple phenylalanine residues, we have developed a script based on R programming language. This script uses FoldX protein modeling program as a tool for mutagenesis and protein stability calculation. The script and its manual can be accessed at <A href=" https://github.com/algisk/common"> GitHub </A>.</P><P class="Raleway">We have chosen five aromatic and hydrophobic amino acids for mutation (Tyr, Trp, Leu, Ile, Met), as they will most likely be hidden in hydrophobic pockets of the protein and not affect the tertiary structure. For the mutant candidates, we chose the proteins that were reported to have a high inducible expression rates in laboratory conditions: T4 virus clamp protein gp45, <I>Streptococcus thermophilus</I> Csm4. Also, we found that the transmembrane protein lactose permease (LacY) also has a good percentage of phenylalanine and amino acid candidates mentioned above. For this protein, only the transmembrane segments (TMS), excluding TMS1, were selected for mutagenesis.</P><P class="Raleway">As gp45 homotrimer crystal structure is available for examination [1], we have omitted the most probable amino acid residues accounting for the interactions necessary to form the homotrimer complex [2]. Next, we loaded our developed script to acquire the most stable desired mutants. According to FoldX suite, the free energy of mutants was below the energy of the wild type protein up to 15th and 35th mutation for gp45 and Csm4 proteins respectively. Later, the saturation of phenylalanine in the proteins had dramatic effects to the stability.</P><P class="Raleway">It has to be taken into account that the goal is not only to have the highest number of mutations, but also that these mutations destabilize the protein as little as possible. To optimize, we have introduced a coefficient, which depends on the impact of the mutation to the free energy and the number of mutations. Each mutation energy difference is divided by the appropriate number of mutations, meaning that the less is the protein destabilized and the more mutations it has, the lower is the result. We have chosen the mutations that were the minimal extremities on the rising energy of the mutant, as seen from the plots below.</P><P class="Raleway">The crystal structure of gp45 homotrimer is accessible, thus an additional step of refinement for this protein was carried out. We loaded the sequences of these mutants on Phyre2 servers for tertiary structure prediction according to the template. The simulated structures were aligned using TM-Align and the mutants with the least identical sequence (IS) and the most similar structure (TM score) were chosen as candidates for our experiments. </P><P class="Raleway">Additionally, mutants with lower numbers of phenylalanine were adjusted by hand to avoid repetitive phenylalanine codons in the genes, replacing some mutations. These adjusted proteins were also refined and the resulting TM scores (2nd refinement) were compared to that of the 1st refinement results. Four gp45 mutants (m6; m11; m25; m37) with the highest scores and adjusted codons were selected for laboratory experiments. Csm4 with 11, 27 and 40 mutations were also selected to cover the range of possible mutants. For Csm4, the DNA sequences fully saturated with phenylalanine codons did not meet the requirements for the AT/GC content and repeated regions, thus were not ordered for synthesis.</P><H2 class="Raleway">Additional mechanisms</H2><P class="Raleway">To increase the effectiveness of our system, we came up with two advances which will enable better intake of phenylalanine. These include the overexpression of phenylalanine tRNA and phenylalanine transporters. The plasmids containing tRNAphe and pheP were introduced into <I>E. coli</I> cells.</P><P class="Raleway">Generally speaking, the probiotic will have two major advances which will let it to efficiently take up and breakdown phenylalanine. These include the expression of phenylalanine ammonia lyase to breakdown phenylalanine, and Polyphe protein to collect excess phenylalanine. These elements are accompanied by the riboswitch and the enhanced tRNA-Phe and pheP expression. The proof that the designed system performs at the expected capacity is presented in the following sections.</P><H2 class="Raleway">References</H2><P class="Raleway"><UL><LI> 1. Moarefi, I., Jeruzalmi, D., Turner, J., O'Donnell, M., Kuriyan, J. Crystal structure of the DNA polymerase processivity factor of T4 bacteriophage. J.Mol.Biol. 2000; 1215-1223 (296)</LI><LI> 2. M.I. Singh, et al., On the domains of T4 phage sliding clamp gp45: An intermolecular crosstalk governs structural stability and biological activity, Biochim. Biophys. Acta (2016)</LI></UL></P></DIV></DIV><FOOTER class="footerholder "><DIV class="footer"><DIV class="centered"><A href="https://2016.igem.org/Special:RecentChanges">Recent changes</A><A href="https://2016.igem.org/Special:WhatLinksHere/Team:Vilnius-Lithuania">What links here</A><A href="https://2016.igem.org/Special:SpecialPages">Special pages</A></DIV><DIV class="centered"><A href="https://2016.igem.org/Special:Preferences">My preferences</A><A href="https://2016.igem.org/2016.igem.org:Privacy_policy">Privacy policy</A><A href="https://igem.org/General_disclaimer">Disclaimer</A></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>