<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Davidson-MissouriW_OptimizingCodons"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Davidson-MissouriW/OptimizingCodons">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Davidson-MissouriW/OptimizingCodons&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Davidson-MissouriW/OptimizingCodons&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Davidson-MissouriW/OptimizingCodons&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Davidson-MissouriW/OptimizingCodons" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Davidson-MissouriW/OptimizingCodons</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="main_wrapper"><DIV id="header" position:absolute=""><DIV id="navigation"><DIV id="menu" style="position:static"><UL><LI id="mainHome"><A class="aMain" href="https://2010.igem.org/Team:Davidson-MissouriW">Home</A></LI><LI id="mainTeam"><A class="aTeam" href="https://2010.igem.org/Team:Davidson-MissouriW/Team">Team</A></LI><LI><A href="https://2010.igem.org/Team:Davidson-MissouriW/TeamBio">Overview</A></LI><LI><A href="https://2010.igem.org/Team:Davidson-MissouriW/Team#Advisors">Advisors</A></LI><LI><A href="https://2010.igem.org/Team:Davidson-MissouriW/Team#Students">Students</A></LI></UL><LI id="mainProject"><A class="aProject" href="https://2010.igem.org/Team:Davidson-MissouriW/Project">Project</A></LI><LI><A href="https://2010.igem.org/Team:Davidson-MissouriW/Project#Abstract">Abstract</A></LI><LI><A href="https://2010.igem.org/Team:Davidson-MissouriW/Project#Introduction">Introduction</A></LI><LI><A href="https://2010.igem.org/Team:Davidson-MissouriW/OptimizingCodons">Optimizing Codons</A></LI><LI><A href="https://2010.igem.org/Team:Davidson-MissouriW/CreLox">Characterizing Cre/Lox</A></LI><LI><A href="https://2010.igem.org/Team:Davidson-MissouriW/MeasuringExpression">Measuring Gene Expression</A></LI><LI><A href="https://2010.igem.org/Team:Davidson-MissouriW/Project#Summary and Outlook">Summary &amp; Outlook</A></LI><LI><A href="https://2010.igem.org/Team:Davidson-MissouriW/Project#References">References</A></LI><LI id="mainParts"><A class="aParts" href="https://2010.igem.org/Team:Davidson-MissouriW/Parts">Parts</A></LI><LI id="mainModeling"><A class="aModeling" href="https://2010.igem.org/Team:Davidson-MissouriW/Tools">Tools</A></LI><LI><A href="https://2010.igem.org/Team:Davidson-MissouriW/Tools#Veripart">Veripart</A></LI><LI><A href="https://2010.igem.org/Team:Davidson-MissouriW/Tools#Oligator">Oligator</A></LI><LI><A href="https://2010.igem.org/Team:Davidson-MissouriW/Tools#Optimoose">Optimus</A></LI><LI><A href="https://2010.igem.org/Team:Davidson-MissouriW/Tools#Simulator">SimuLox</A></LI><LI id="mainHumanPractices"><A class="aHumanPractices" href="https://2010.igem.org/Team:Davidson-MissouriW/HumanPractices">Human Practices</A></LI><LI id="mainNotebook"><A class="aNotebook" href="https://2010.igem.org/Team:Davidson-MissouriW/Notebook">Notebook</A></LI><LI id="mainSafety"><A class="aSafety" href="https://2010.igem.org/Team:Davidson-MissouriW/Safety">Safety</A></LI><LI id="mainSponsors"><A class="aSponsors" href="https://2010.igem.org/Team:Davidson-MissouriW/Sponsors">Acknowledgements</A></LI></DIV></DIV></DIV></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Optimizing_Codons"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Optimizing Codons</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Overview"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Background"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Background</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Building_Optimized_and_Deoptimized_TetA"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Building Optimized and Deoptimized TetA</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#RFLP_.28Restriction_Fragment_Length_Polymorphism.29"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">RFLP (Restriction Fragment Length Polymorphism)</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#TetA_Toxicity"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">TetA Toxicity</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Tetracycline_Experiments"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Tetracycline Experiments</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Optimizing_Codons">Optimizing Codons</SPAN></H2><H3><SPAN class="mw-headline" id="Overview">Overview</SPAN></H3><DL><DD>Using a variety of programs available online, including the <A href="http://gcat.davidson.edu/igem10/index.html" class="external text" rel="nofollow">Oligator</A> and <A href="http://gcat.davidson.edu/igem10/opt/opt_index.html" class="external text" rel="nofollow">Optimoose</A>,  we were able to formulate a plan that would enable us to synthesize various optimized and deoptimized versions of the TetA gene. We relied on codon bias, the differences in frequency of occurrence of synonymous codons in coding DNA, to allow for varying expression levels of TetA in the cell. By using natural enzyme sites within TetA, we were able to conduct restriction digests on TetA that allowed us to alter roughly 150 base pairs within each segment using codon bias with a total of four segments available. The TetA vector that was used to synthesize segment 1 clones was digested with EcoRI and NheI. The TetA vector that was used to synthesize segment 2 clones was digested with NheI and BamHI. This gave us the ability to insert roughly 144 bp for each segment that were optimized or deoptimized using codon bias. These inserts were fairly large and thus were synthesized by annealing oligos. These annealed oligos were then treated with a polynucleotide kinase to add a phosphate backbone to the oligo DNA. Initial results yielded a lot of background given that it was difficult to distinguish between singly and doubly cut vectors. Therefore, to reduce background noise, we added an extra step in our preparation of the vector using Antarctic Phosphatase. By using this phosphatase on the vector, we were able to remove the 5’ phosphate groups off the backbone of the DNA while still leaving the 3’ end phosphates. This made it difficult for the singly cut vector to go back on itself because ligase requires a phosphate group to be present to seal the nick in the phosphate backbone. With this reduced background, we were able to ligate the annealed oligos to our vectors and screen the candidate clones using RFLP.
</DD></DL><H3><SPAN class="mw-headline" id="Background">Background</SPAN></H3><DL><DD>Resource:  <A href="http://en.wikipedia.org/wiki/File:TetSystem.png" class="external free" rel="nofollow">http://en.wikipedia.org/wiki/File:TetSystem.png</A></DD></DL><DL><DD>Through the understanding of the bacterial tetracycline resistance mechanism, we hypothesized that the optimization of the TetA gene was directly related to the translational efficiency of TetA and therefore a more optimized gene would offer a higher level of tetracycline resistance.
</DD></DL><DL><DD>The diagrams below demonstrate our hypothesis:
</DD></DL><H3><SPAN class="mw-headline" id="Building_Optimized_and_Deoptimized_TetA">Building Optimized and Deoptimized TetA</SPAN></H3><DL><DD>To build the optimized and deoptimized segments of the Tet A gene we used the sequence that the <A href="http://gcat.davidson.edu/igem10/opt/opt_index.html" class="external text" rel="nofollow">Optimoose</A> program gave us and put it into the <A href="http://gcat.davidson.edu/igem10/index.html" class="external text" rel="nofollow">Oligator</A>, a program created by the Davidson-Missouri Western iGEM team.  After getting the recommended Oligo sequences, we added necessary &quot;sticky ends&quot; that were compatible with the naturally occurring restriction enzyme sites that we discovered (see Restriction Enzymes below).  The Oligos that were ordered can be seen below. 
</DD></DL><DL><DD>This diagram demonstrates how the TetA gene was broken into different segments (1 and 2) and created on a pSB1A2 vector. 
</DD></DL><DL><DD>Note:  All above segments have been optimized and deoptimized using the <A href="http://gcat.davidson.edu/igem10/opt/opt_index.html" class="external text" rel="nofollow">Optimoose</A> online program. 
</DD></DL><DL><DD><B>Segment 1 Oligos</B></DD></DL><DL><DD><B>Optimized</B></DD></DL><DL><DD>top 1 5'  AATTCGCGGCCGCTTCTAGATGAAATCTAACAACGCGCTGATCGTTATCCTGGGTACCG 3’
</DD></DL><DL><DD>top 2 5'  TTACCCTGGACGCGGTTGGTATCGGTCTGGTTATGCCGGTTCTGCCGGGTCTGCTGCGTGAC 3’
</DD></DL><DL><DD>top 3 5'  ATCGTTCACTCTGACTCTATCGCGTCTCACTACGGTGTTCTG 3’
</DD></DL><DL><DD>bottom 2  5' CATAACCAGACCGATACCAACCGCGTCCAGGGTAACGGTAC
</DD><DD>CCAGGATAACGATCAGCGCGTTGTTAGATTTCATCTAGAAGCGGCCGCG 3’
</DD></DL><DL><DD>bottom 1 5'  CTAGCAGAACACCGTAGTGAGACGCGATAGAGTCAGAGTGAACGATGTCACGCAGCAGACCCGGCAGAACCGG 3’
</DD></DL><DL><DD><B>Deoptimized</B></DD></DL><DL><DD>top 1 5' AATTCGCGGCCGCTTCTAGATGAAGAGTAATAATGCCCTAATAGTCATACTAGGAACAG 3’
</DD></DL><DL><DD>top 2 5' TCACACTAGATGCCGTCGGAATAGGACTCGTCATGCCCGTCCTACCCGGACTACTAAGGGA 3’
</DD></DL><DL><DD>top 3 5' TATAGTCCATAGTGATAGTATAGCCAGTCATTATGGAGTCCTG 3’
</DD></DL><DL><DD>bottom 2 5' GAGTCCTATTCCGACGGCATCTAGTGTGACTGTTCCTAGTATG
</DD><DD>ACTATTAGGGCATTATTACTCTTCATCTAGAAGCGGCCGCG 3’
</DD></DL><DL><DD>bottom 1 5' CTAGCAGGACTCCATAATGACTGGCTATACTATCACTATGGACTATATCCCTTAGTAGTCCGGGTAGGACGGGCATGAC 3’
</DD></DL><DL><DD><B>Segment 2 Oligos</B></DD></DL><DL><DD><B>Optimized</B></DD></DL><DL><DD>top 1 5' CTAGCCTGTACGCGCTGATGCAGTTCCTGTGCGCGCCGGTTCTGGGTGCGCTGTCTGAC 3’
</DD></DL><DL><DD>top 2 5' CGTTTCGGTCGTCGTCCGGTTCTGCTGGCGTCTCTGCTGGGTGCGACCATCGACTAC 3’
</DD></DL><DL><DD>top 3 5' GCGATCATGGCGACCACCCCGGTTCTGTG 3’
</DD></DL><DL><DD>bottom 1 5' GCGCACAGGAACTGCATCAGCGCGTACAGG 3’
</DD></DL><DL><DD>bottom 2 5' CCAGCAGAACCGGACGACGACCGAAACGGTCAGACAGCGCACCCAGAACCGGC 3’
</DD></DL><DL><DD>bottom 3 5' GATCCACAGAACCGGGGTGGTCGCCATGATCGCGTAGTCGATGGTCGCACCCAGCAGAGACG 3’
</DD></DL><DL><DD><B>Deoptimized</B></DD></DL><DL><DD>top 1 5'     CTAGCGCTATATGCCCTAATGCAATTTCTATGTGCCCCCGTCCTAGGAGCCCTAAGTG 3'
</DD></DL><DL><DD>top 2  5'     ATAGGTTTGGAAGGAGGCCCGTCCTACTAGCCAGTCTACTAGGAGCCACAATAGAT 3'
</DD></DL><DL><DD>top 3  5'     TATGCCATAATGGCCACAACACCCGTCCTATG 3'
</DD></DL><DL><DD>bottom 1  5'     GGCACATAGAAATTGCATTAGGGCATATAGCG 3' 
</DD></DL><DL><DD>bottom 2  5'     TAGGACGGGCCTCCTTCCAAACCTATCACTTAGGGCTCCTAGGACGGG 3' 
</DD></DL><DL><DD>bottom 3  5'     GATCCATAGGACGGGTGTTGTGGCCATTATGGCATAATCTATTGTGGCTCCTAGTAGACTGGCTAG 3'
</DD></DL><DL><DD><B>Restriction Enzymes</B></DD></DL><DL><DD>To insert the annealed Oligo segments that were created we had to digest the Tet A gene using naturally occurring enzyme restriction sites that did not exist in the rest of the gene or in the plasmid (pSBIA2).  
</DD></DL><DL><DD>To build the fully optimized (segments 1 and 2 optimized) and fully deopitmized (segments 1 and 2 are deoptimized) versions of the TetA gene, we also used these same restriction enzymes. For the fully optimized version, we cut the optimized segment 1 TetA gene and cut it with NheI and then ligated in the optimized segment 2 insert (this insert was cut with NheI and BamHI to allow for us to isolate just the optimized segment 2 fragment, without the rest of the TetA gene).  The fully deoptimized version of the TetA gene was constructed using the same procedure with the deoptimzed segment 1 TetA gene and a deoptimized segment 2 insert. 
</DD></DL><DL><DD><TABLE class="wikitable" style="text-align:center" border="1" cellpadding="2"><TBODY><TR><TH>Segment
</TH><TD><B>First Restriction Enzyme</B></TD><TD><B>Second Restriction Enzyme</B></TD></TR><TR><TD> 1  </TD><TD>  EcoRI   </TD><TD> NheI
</TD></TR><TR><TD> 2 </TD><TD> NheI </TD><TD> BamHI
</TD></TR></TBODY></TABLE></DD></DL><P><CENTER><A href="#main_wrapper">Top</A></CENTER></P><H3><SPAN class="mw-headline" id="RFLP_.28Restriction_Fragment_Length_Polymorphism.29">RFLP (Restriction Fragment Length Polymorphism)</SPAN></H3><P>We used restriction fragment length polymorphism (RFLP) as a way to screen our candidate clones for our optimized and deoptimized versions of the TetA gene. RFLP is a laboratory technique whereby DNA is digested by restriction enzymes and the resulting DNA fragments are separated by gel electrophoresis. RFLP relies on a difference between two or more samples of homologous DNA molecules arising from differing locations of restriction sites. Using the program NEBcutter V2 from New England Biolabs, we were able to determine which restriction enzymes to use to screen for our various TetA candidate clones. Based on availability of restriction enzymes and the greatest variability of bands created, we choose four different restriction enzymes for the custom digestion of our candidate clones:
</P><BLOCKQUOTE>Optimized segment 1→ HinFI </BLOCKQUOTE><BLOCKQUOTE>Deoptimized segment 2→ DdeI </BLOCKQUOTE><BLOCKQUOTE>Optimized segment 2→ BsaWI </BLOCKQUOTE><BLOCKQUOTE>Deoptimized segment 2→ AccI </BLOCKQUOTE><P>Using RFLP, we screened our clones and sent strong candidate clones, the ones that had diagnostic bands indicative of the desired clone, off for sequencing. After analyzing the sequencing data, we were able to conclude that RFLP is an excellent method for screening for clones, given that almost all of the clones sent off were correct.
</P><P><CENTER><A href="#main_wrapper">Top</A></CENTER></P><H3><SPAN class="mw-headline" id="TetA_Toxicity">TetA Toxicity</SPAN></H3><DL><DD>Through our understanding of the way the Tetracycline resistance protein (TetA) functions, we hypothesized that too much TetA gene expression would cause the cells to die. To test this hypothesis we conducted an experiment in which we induced the production of TetA with varying levels of IPTG.  The results did not support our hypothesis. (see graph below) 
</DD></DL><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>TetA Toxicity Experiments</CAPTION></TABLE><P><CENTER><A href="#main_wrapper">Top</A></CENTER></P><H3><SPAN class="mw-headline" id="Tetracycline_Experiments">Tetracycline Experiments</SPAN></H3><DL><DD>The graph below illustrates that any amount of Tetracycline is toxic to the cell. With increasing concentrations of Tet, cell viability decreases. 
</DD></DL><P><CENTER><A href="#main_wrapper">Top</A></CENTER></P><DL><DD>The graphs below illustrate that any amount of Tetracycline is toxic to the cell. With increasing concentrations of Tet, cell viability decreases. pSB1A2 was a control in this experiment which has no tetracycline resistant. The psB3T5 vector also served as another control as having Tetracycline resistance that is not controlled by a promoter.  
</DD></DL><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>Tetracycline Titration Experiments</CAPTION></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Davidson-MissouriW/OptimizingCodons">http://2010.igem.org/Team:Davidson-MissouriW/OptimizingCodons</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Davidson-MissouriW/OptimizingCodons" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Davidson-MissouriW/OptimizingCodons" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Davidson-MissouriW/OptimizingCodons&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Davidson-MissouriW/OptimizingCodons&amp;oldid=193887" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>