"
Page
Discussion
View source
History
teams
Log in
 
Team:Davidson-MissouriW/Tools
From 2010.igem.org
Home
Team
Overview
Advisors
Students
Project
Abstract
Introduction
Optimizing Codons
Characterizing Cre/Lox
Measuring Gene Expression
Summary & Outlook
References
Parts
Tools
Veripart
Oligator
Optimus
SimuLox
Human Practices
Notebook
Safety
Acknowledgements
iGEM Davidson – Missouri Western 2010: Tools
The team has designed several tools to guide and support our iGEM project.
VeriPart
Verifying BioBrick parts can be difficult and tedious by hand.  To solve this problem, we have developed a tool to identify BioBrick parts within a given DNA sequence.  When a sequence is entered, the program checks the last archived version of the BioBrick Parts Registry and returns the best matched part.  If your desired part is not found, you can attempt to align the sequence you entered with the part you desire.  When a device is found in your sequence, you can click to view the sequences of individual parts.  An alignment of the sequence with the part is presented to the user allowing for easier sequence analysis.
The Oligator
Designing and building DNA sequences is a fundamental tool for anyone who works in synthetic biology.  The Oligator is designed to choose oligos to build any desired sequence between 20 and 20,000 base pairs.  The program allows the user to choose oligo lengths between 20 and 160 bp and overlap lengths of between 20 and 80 bp.   The program will optionally check for and remove BioBrick restriction sites. Choosing the first checkbox will tell the user if BioBrick restriction sites exist anywhere in the entered DNA sequence and highlight where they occur. The second checkbox will remove the sites, leaving the amino acid sequence intact, based on the user-input reading frame. Since keeping the amino acid sequence the same involves knowing the reading frame, a radio button must be filled in indicating where the reading frame begins. The final option is to add BioBrick ends to your sequence. This is helpful when cloning your sequence into a BioBrick plasmid. Simply choose which site you want to add to the 5' end. The options are a prefix digested with XbaI, a prefix digested with EcoRI or the entire prefix. You then choose which site you want to add to the 3' end. The options are a suffix digested with SpeI, a suffix digested with PstI or the entire suffix. However, if you choose to add a prefix, you must choose a suffix, and vice-versa. Furthermore, you also have the ability to add a custom prefix and suffix.
Optimus
Optimus is a tool designed to allow the user to evaluate the expression level of a gene sequence in E. coli using either RCBS-PC or the CAI formula.  Additionally, the user has the option to optimize or deoptimize the sequence by using one of those two formulas. If the optimized or deoptimized option is selected, the user will be given a new sequence whose codons have been changed to reflect the best, or worst, codons as determined by the selected formula. Assuming the entered sequence is in frame, the amino acid sequence is preserved. Optimus serves as a mechanism by which we can engineer optimized and deoptimized sequences which represent a given "weight" to the items in a knapsack based on their protein production.
SimuLox
In order to better understand our fluorescent protein - TetA constructs, we designed a simulation of the cre-lox system.  This simulation allows the user to test our pre-determined constructs as well as custom constructs from any number of promoters, lox sites, fluorescent proteins, essential genes, and/or terminators.  The program will then show either a single, animated simulation that will allow the user to see how the lox sites interact, or it will run many simulations and then generate a histogram that shows the distribution of what fluorescent proteins were expressed.  In addition, if the user chooses to include weights for the "items" and a capacity for the knapsack the program will tell you whether or not your construct has exceeded the capacity.  In order to run the program, download the jar file .  To see the actual percentages behind the histogram, the program needs to be run through a command line tool, but other than that simply double clicking the jar file to run it should be sufficient.
Top
Retrieved from " http://2010.igem.org/Team:Davidson-MissouriW/Tools "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
