"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Davidson-MissouriW/CreLox
From 2010.igem.org
Home
Team
Overview
Advisors
Students
Project
Abstract
Introduction
Optimizing Codons
Characterizing Cre/Lox
Measuring Gene Expression
Summary & Outlook
References
Parts
Tools
Veripart
Oligator
Optimus
SimuLox
Human Practices
Notebook
Safety
Acknowledgements
Characterizing Cre/lox Recombination Method
Mechanism behind Cre/lox Recombination
The Cre-lox tool is a site-specific recombination system that is widely used in biological research to manipulate DNA. It was discovered in the early 90's through characterization of coliphage P1 recombination system. The Cre recombinase enzyme, a 38kDa protein, catalyzes the recombination of DNA between two lox sites. These lox sites, each 34 bp long, consist of two inverted repeat arms flanking a spacer region of 8bp that is unique to the lox site.
Excision
Inversion
Designing Lox Sites
In order to randomly "select objects" for the knapsack problem, we used the Cre-lox recombination method of excision and inversion. We needed a set of variant lox sites in order to create constructs that would yield different subsets of survival and fluorescence to optimally fill the knapsack. These variant lox sites have mutations specific to the 8bp spacer region and do not recombine. We created 10 new lox sites with mutations in the 8 bp region: loxN Forward and Reverse, loxm2 Forward and Reverse, lox2272 Forward and Reverse, lox5171 Forward and Reverse, and loxBri Forward and Reverse. In addition, we added the wildtype loxP Reverse to the registry.
We floxed red fluorescent protein with these variant lox sites using 16 out of the 21 combinations of the 5 lox forward variants to see immediately if recombination occurred in the presence of Cre. These constructs below follow a moderate promoter, pTet. The key for the constructs is above and colors refer to specific lox sites.
Would you like a bagel with all that lox?
Knapsack Construct Models
We designed five different construct models that could potentially lead us to a biological solution to the knapsack problem. These constructs include various lox sites, a reporter fluorescent protein, and a TetA gene that has an upper and lower threshold of expression in order to survive, thus providing a means to find "objects" (subset of modules) with the correct "weight" (expression of TetA).
To determine which model would be best to solve the knapsack problems, we analyzed each version and its theoretical probabilities for different results that could be left after any number cre interactions. Furthermore, we created the construct simulator that can calculate for us the theoretical probabilities of the possible results after any number of cre steps for our construct and any custom construct. From this tool and an analysis of the possible results provided by the tool, we have decided that version D is the model to solve the knapsack problem.
Version A: Allows inversion and excision within and over multiple modules.
Version B: Allows only excision within and over multiple modules.
Version C: Only allows flipping within modules. Requires only 3 different Lox sites.
Version D: Only allows inversion within modules. Requires "n" different Lox sites.
Version E: Initially only allows flipping over multiple modules, then allows cutting and flipping over multiple modules.
Troubleshooting with Cre
Clearly, characterizing the activity of Cre recombinase is integral to our project. Initially, we chose to use pBad-RBS-Cre because of its moderate promoter to test the floxed RBS-RFP constructs and analyze their level of recombination. While transforming pBad-RBS-Cre ( I718008 ) from the 2010 iGEM kit plate, we observed unexpected results when size verifying the band. We digested the part with PvuI (an internal restriction site in the Cre gene sequence) and PstI and expected to see a 600 bp band along with two additional bands. Instead, we saw bands of unexplained size from digestion of the experimental colonies. In addition, the negative control (which contained only pBad-RBS-Cre) yielded only one band at around 1600bp when there should have been at least two bands (because of the internal PvuI site in Cre) and the band was at a size that pertained to none of the parts that were supposed to be involved in the ligation (see below, left). Furthermore, we religated pBAd-RBS-Cre from our own Davidson GCAT-alog from earlier years and digested with the same restriction enzymes. The presence of the 600 bp band  confirmed the success of the ligation and this was used in furthur ligations (see below, right). We noted the incorrect part on the pBad-RBS-Cre wiki under experience.
top
Retrieved from " http://2010.igem.org/Team:Davidson-MissouriW/CreLox "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
