<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Davidson-MissouriW_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Davidson-MissouriW/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Davidson-MissouriW/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Davidson-MissouriW/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Davidson-MissouriW/Notebook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Davidson-MissouriW/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Davidson-MissouriW/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="main_wrapper"><DIV id="header" position:absolute=""><DIV id="navigation"><DIV id="menu" style="position:static"><UL><LI id="mainHome"><A class="aMain" href="https://2010.igem.org/Team:Davidson-MissouriW">Home</A></LI><LI id="mainTeam"><A class="aTeam" href="https://2010.igem.org/Team:Davidson-MissouriW/Team">Team</A></LI><LI><A href="https://2010.igem.org/Team:Davidson-MissouriW/TeamBio">Overview</A></LI><LI><A href="https://2010.igem.org/Team:Davidson-MissouriW/Team#Advisors">Advisors</A></LI><LI><A href="https://2010.igem.org/Team:Davidson-MissouriW/Team#Students">Students</A></LI></UL><LI id="mainProject"><A class="aProject" href="https://2010.igem.org/Team:Davidson-MissouriW/Project">Project</A></LI><LI><A href="https://2010.igem.org/Team:Davidson-MissouriW/Project#Abstract">Abstract</A></LI><LI><A href="https://2010.igem.org/Team:Davidson-MissouriW/Project#Introduction">Introduction</A></LI><LI><A href="https://2010.igem.org/Team:Davidson-MissouriW/OptimizingCodons">Optimizing Codons</A></LI><LI><A href="https://2010.igem.org/Team:Davidson-MissouriW/CreLox">Characterizing Cre/Lox</A></LI><LI><A href="https://2010.igem.org/Team:Davidson-MissouriW/MeasuringExpression">Measuring Gene Expression</A></LI><LI><A href="https://2010.igem.org/Team:Davidson-MissouriW/Project#Summary and Outlook">Summary &amp; Outlook</A></LI><LI><A href="https://2010.igem.org/Team:Davidson-MissouriW/Project#References">References</A></LI><LI id="mainParts"><A class="aParts" href="https://2010.igem.org/Team:Davidson-MissouriW/Parts">Parts</A></LI><LI id="mainModeling"><A class="aModeling" href="https://2010.igem.org/Team:Davidson-MissouriW/Tools">Tools</A></LI><LI><A href="https://2010.igem.org/Team:Davidson-MissouriW/Tools#Veripart">Veripart</A></LI><LI><A href="https://2010.igem.org/Team:Davidson-MissouriW/Tools#Oligator">Oligator</A></LI><LI><A href="https://2010.igem.org/Team:Davidson-MissouriW/Tools#Optimoose">Optimus</A></LI><LI><A href="https://2010.igem.org/Team:Davidson-MissouriW/Tools#Simulator">SimuLox</A></LI><LI id="mainHumanPractices"><A class="aHumanPractices" href="https://2010.igem.org/Team:Davidson-MissouriW/HumanPractices">Human Practices</A></LI><LI id="mainNotebook"><A class="aNotebook" href="https://2010.igem.org/Team:Davidson-MissouriW/Notebook">Notebook</A></LI><LI id="mainSafety"><A class="aSafety" href="https://2010.igem.org/Team:Davidson-MissouriW/Safety">Safety</A></LI><LI id="mainSponsors"><A class="aSponsors" href="https://2010.igem.org/Team:Davidson-MissouriW/Sponsors">Acknowledgements</A></LI></DIV></DIV></DIV></DIV><DIV style="padding:10px"><H2><SPAN class="mw-headline" id="Notebook">Notebook</SPAN></H2><P><B>May 25, 2010 to May 29, 2010</B></P><UL><LI>Davidson traveled to Missouri Western and further discussed project ideas</LI></UL><P><B>June 1, 2010</B></P><UL><LI>Tested to find lethal concentration of Tet in plated and liquid mediums</LI><LI>Verified RCBS with our sequences and researched codon optimization methods</LI><LI>Analyzed probabilities for constructs A and B</LI><LI>Learned lab protocols and organized lab 
</LI><LI>Created list of needed parts and parts we need
</LI></UL><P><B>June 2, 2010</B></P><UL><LI>Mini prepped, diagnostic RP digest, preparative digest, ligation, pBAD+RBS-RFP </LI><LI>Researched different ways to optimize a gene</LI><LI>Began simulation for Constructs A and B</LI><LI>Learned miniprep and digestion protocols</LI><LI>Learned how to document gels
</LI></UL><P><B>June 3, 2010</B></P><UL><LI>Mini prepped, diagnostic RP digest, preparative digest, ligation, pBAD+RBS-RFP </LI><LI>Researched restriction sites and created proposal for gene optimization</LI><LI>Continued work on simulation in Java and Matlab</LI><LI>Learned about biobrick restriction sites and sticky ends
</LI><LI>Ordered lox oligos
</LI></UL><P><B>June 4, 2010</B></P><UL><LI>Mini prepped RBS-TetA-TT
</LI><LI>RX Fragment digestion for RBS-TetA-TT
</LI><LI>Analyzed the optimization of tetA sequence and researched properties of Lox sites
</LI><LI>Debugged Simulations
</LI><LI>Sent off parts for sequencing
</LI><LI>Learned the transformation protocol 
 </LI></UL><P><B>June 7, 2010</B></P><UL><LI>Ligate and plasmid check for pSB1A2 or pSB1AK3
</LI><LI>Began working to improve the Lancelator and created histograms for simulations
 </LI><LI>Gel purified vector for lox ligations
</LI><LI>Divided up constructs to be ligated
</LI></UL><P><B>June 8, 2010</B></P><UL><LI>Double digest on pBAD-RBS-RFP to make an RS fragment insert </LI><LI>Worked on a Perl program that check for Bio Brick Restriction Sites
</LI><LI>Analyzed sequencing results using ApE program
</LI><LI>BOUGHT HAWAIIAN SHIRTS!!!!!
</LI></UL><P><B>June 9, 2010</B></P><UL><LI>EcoRI and PstI digest on pLac and pBad colonies 
</LI><LI>Arabinose and IPTG concentration experiments started </LI><LI>Continued work on the Lancelator program and began working on a different simulation for Construct C and D
</LI><LI>Annealed lox oligos together
</LI><LI>Gel purified pBad+RBS+Cre (“Cre construct” from 2010 igem kit plate)
</LI></UL><P><B>June 10, 2010</B></P><UL><LI>Picked 14 colonies from pLac-RBS-RFP and pLac-RBS-RFP-RBS plates
</LI><LI> Started inducer curve experiments</LI><LI>Linked Lancelator code to the website and continued simulations
</LI><LI>Performed ligations involving lox sites
</LI></UL><P><B>June 11, 2010</B></P><UL><LI>Ligated S03736 into pSB1A7 vector
</LI><LI>EcoRI/PstI digest on pLac-RBS-RFP-RBS colonies</LI><LI>Added new features to the Lancelator and worked on numerically representing Construct C for the Matlab simulation
</LI><LI>Performed colony PCR screening for the first ligations
</LI></UL><P><B>June 14, 2010</B></P><UL><LI>EcoRI/PstI diagnostic digest performed on pBad-RBS-RFP colonies  
</LI><LI>Inducer curve experiment started on pBad and pLac constructs </LI><LI>Debugged Lancelator and finished numerical representation for the simulation
</LI><LI>Size verified lox forward and lox reverse sites
</LI><LI>Ligated “cre construct” into low, medium, and high copy kanamycin resistant vectors
</LI></UL><P><B>June 15, 2010</B></P><UL><LI>Ligated pLac-RBS-RFP + RBS-TetA
</LI><LI>IPTG experiment started on pLacI-RBS-RFP </LI><LI>Worked on the color schemes for the new Oligator
</LI><LI>Created a new construct that might be helpful for the biologists
</LI><LI>Planning for Amp+Tet experiments
</LI><LI>First “cre construct” ligations failed
</LI></UL><P><B>June 16, 2010</B></P><UL><LI>EcoRI/PstI diagnostic digest performed on pLac-RBS-RFP in pSB1A7 vector
</LI><LI>IPTG experiment started on plates with pLac and pLacI constructs </LI><LI>Began working on a Java code that will run simulations for all the constructs
</LI><LI>Researched on NP-Complete problems
</LI><LI>Religated “cre construct”
</LI><LI>Lox sites sent off for sequencing 
</LI></UL><P><B>June 17, 2010</B></P><UL><LI>Flourimeter data collected from IPTG experiment
</LI><LI>Ligated pLacI-RBS-RFP + RBS-TetA </LI><LI>Worked on graphics of the Java program
</LI><LI>Studied specific NP-Complete problems and brainstormed ways to incorporate them in our project
</LI><LI>Lox sites were frozen down and entered in GCAT-alog
</LI><LI>Agar plates were prepared for Amp+Tet experiments
</LI></UL><P><B>June 18, 2010</B></P><UL><LI>Picked 4 colonies from pLacI-RBS-RFP-RBS-TetA plates 
</LI><LI>IPTG experiment started using newly created IPTG  </LI><LI>Debugged the Java program 
</LI><LI>Created a new page for the GCAT server
</LI><LI>Planned kanamycin experiments 
</LI></UL><P><B>June 21, 2010</B></P><UL><LI>Digested TetA with EcoRI/NheI
</LI><LI>Annealed oligos for optimized and deoptimized TetA segment 1  </LI><LI>Improved the Java program
</LI><LI>Began categorizing the knapsack problem into subparts
</LI><LI>Successfully ligated ptet+different lox sites 
</LI><LI>Planned different combinations of variant lox sites
</LI></UL><P><B>June 22, 2010</B></P><UL><LI>Ligated annealed TetA oligos into digested TetA
</LI><LI>Inducer curve experiment started on pLac and pLacI constructs  </LI><LI>Added buttons to the java program
</LI><LI>Brainstormed biological implications in different NP-Complete problems
</LI><LI>Confirmed successful ligations of lox sites from sequencing results
</LI><LI>First pilot test for Amp+Tet experiments
</LI></UL><P><B>June 23, 2010</B></P><UL><LI>Preparative NheI/BamHI digest performed on TetA
</LI><LI>Preparative EcoRI/NheI digest performed on TetA </LI><LI>Restructuring and reorganizing the Java program
</LI><LI>Reviewed numerical algorithms and its relation to NP-Complete problems
</LI><LI>Researched data on possible transcription terminator in Tet A 
</LI><LI>Failure of cre experiments 
</LI></UL><P><B>June 24, 2010</B></P><UL><LI>Annealed optimized and deoptimized segment 1 TetA ligated into vector 
</LI><LI>Annealed optimized and deoptimized segment 2 TetA ligatied into vector</LI><LI>Added weight characteristic to the fluorescent proteins in the Java code
</LI><LI>Continued exploration of NP-Complete problems
</LI><LI>Ligations to construct tet constructs 
</LI></UL><P><B>June 27, 2010</B></P><UL><LI>Inducer curve experiment started on pLac and pLacI constructs 
</LI><LI>Picked colonies from TetA segment 1 &amp; 2 optimized &amp; deoptimized plates </LI><LI>Added histograms to the Java Code
</LI><LI>Researched and analyzed the effectiveness of Cre
</LI><LI>Size verification of ptet+variant lox sites 
</LI></UL><P><B>June 28, 2010</B></P><UL><LI>Added the option for the user to input the weights of the various modules and began work on custom construct builder
</LI><LI>Picked colonies from TetA segment 1 &amp; 2 optimized &amp; deoptimized plates </LI><LI>Planned to ligate pBad+RBS+Cre in ampicillin resistant vectors 
</LI><LI>Researched plasmid partitioning 
</LI></UL><P><B>June 29, 2010</B></P><UL><LI>Screened segment 1 optimized and deoptimized candidate clones using RFLP
</LI><LI>Screened segment 2 optimized and deoptimized candidate clones using RFLP </LI><LI>Finished the simulation program and exported it
</LI><LI>Explored the set covering problem
</LI><LI>Researched applications of the knapsack problem in cryptography
</LI><LI>Ligated ptet+LoxP+RBS+RFP
</LI></UL><P><B>June 30, 2010</B></P><UL><LI>Tet titration experiment performed on pLac and pLacI +RBS-RFP-RBS-TetA
</LI><LI>Ligated TetA onto pLac and pLacI +RBS-RFP-RBS-TetA with no terminator  </LI><LI>Some final work on the simulation program
</LI><LI>Discussed new ideas such as integer programming
</LI><LI>PCR screening of “RFP construct” 
</LI><LI>Ptet+variant lox sites were frozen down
</LI></UL><P><B>July 1, 2010</B></P><UL><LI>Preparative BamHI/NheI digest performed on TetA 
</LI><LI>New experiment protocol developed for reduced vector background noise   </LI><LI>Added terminators to the program
</LI><LI>Ran test trials of the program and analyzed its results
</LI><LI>Planned experiments for using Cre as “front” and “back” vector
</LI><LI>Ligated first “floxed” constructs 
</LI></UL><P><B>July 2, 2010</B></P><UL><LI>Mini prepped, diagnostic RP digest, preparative digest, ligation, pBAD+RBS-RFP </LI><LI>Continued work on the categorization of the knapsack problem
</LI><LI>Researched Instant Insanity problem
</LI><LI>PCR screening for “floxed constructs”
</LI></UL><P><B>July 6, 2010</B></P><UL><LI>Ligated S04446 and S04447 to J31007
</LI><LI>Tet titration experiment performed on pLac and pLacI +RBS-RFP-RBS-TetA </LI><LI>Added direction of terminators in the simulation
</LI><LI>Researched cryptography application
</LI><LI>Size verified “floxed constructs”
</LI><LI>Digested “Cre construct” as front and back vector 
</LI></UL><P><B>July 7, 2010</B></P><UL><LI>Ligated S04448 to RBS-TetA
</LI><LI>Ligated S04449 to RBS-TetA</LI><LI>Fixed memory leak in the simulation program
</LI><LI>Researched integer programming
</LI><LI>Sent off “floxed constructs&quot; for sequencing
</LI><LI>Ligated “floxed constructs” into pBad+RBS+Cre as “front” and “back” vector  
</LI></UL><P><B>July 8, 2010</B></P><UL><LI>IPTG experiment started using pre-induced method  
</LI><LI>IPTG experiment started using varying levels of tetracycline  </LI><LI>Worked on animation for the simulation and continued analysis of the results of the simulation
</LI><LI>Discussed integer programming as a group
</LI><LI>Data suggests something wrong with “cre construct” 
</LI></UL><P><B>July 9, 2010</B></P><UL><LI>Picked optimized and deoptimized TetA colonies
</LI><LI>Flourimeter data collected from IPTG experiment  </LI><LI>Revised some features on the program
</LI><LI>Began working on theoretical probability models of the different constructs
</LI><LI>Further experiments with “cre construct” 
</LI></UL><P><B>July 10, 2010</B></P><UL><LI>Used RFLP to screen for candidate TetA clones 
</LI><LI>Transformed various pLac and pLacI constructs into MG1655 cells </LI><LI>Experiments suggest pBad+RBS+Cre from iGem 2010 kit plate is incorrect 
</LI></UL><P><B>July 12, 2010</B></P><UL><LI>Used RFLP to screen for candidate TetA clones 
</LI><LI>Transformed various pLac and pLacI constructs into MG1655 cells </LI><LI>Added slider feature to the simulation program
</LI><LI>Continued work on the theoretical probability model for Construct A of size 3
</LI><LI>Tested “cre construct” from 2009 iGem kit plate
</LI></UL><P><B>July 13, 2010</B></P><UL><LI>Used RFLP to screen for candidate TetA clones 
</LI><LI>EcoRI/PstI diagnostic digest performed on K199150 </LI><LI>Added custom prefixes and suffixes to the Oligator
</LI><LI>Found a general equation for finding theoretical probabilities for Construct A of size 3
</LI><LI>Sent off “floxed sites for sequencing” 
</LI></UL><P><B>July 14, 2010</B></P><UL><LI>Ligated segment 2 optimized and deoptimzed TetA behind segment 1
</LI><LI>IPTG experiment started on I715039-1 and I715039-2 </LI><LI>Began analyzing MW optimization code
</LI><LI>Finished theoretical probabilities for Construct A of size 3
</LI><LI>Cre construct from 2009 plate was incorrect as well 
</LI></UL><P><B>July 15, 2010</B></P><UL><LI>Cell count data collected from IPTG experiments
</LI><LI>Flourimeter data collected from IPTG experiments </LI><LI>Worked on a separate program using the CAI equation
</LI><LI>Began working on theoretical probabilities for Construct A of size 4
</LI><LI>Ligated more “floxed constructs” 
</LI></UL><P><B>July 16, 2010</B></P><UL><LI>Tet titration experiment started using pSB3T5 vector
</LI><LI>Tet titration experiment started using optimized and deoptimized TetA </LI><LI>Added changes to the optimizer code
</LI><LI>Researched how to solve a system of recursive relations into closed form
</LI><LI>Decided to ligate cre into pBad+RBS and pLac+RBS 
</LI></UL><P><B>July 19, 2010</B></P><UL><LI>Cell count data collected from Tet titrations
</LI><LI>Tet titration experiment started using optimized and deoptimized TetA </LI><LI>Fixed the error in the CAI optimization program
</LI><LI>Continued analyzing systems of recursive relations
</LI><LI>Ligated pBad+RBS+Cre and pLac+RBS+Cre 
</LI></UL><P><B>July 20, 2010</B></P><UL><LI>Flourimeter data collected on IPTG experiment </LI><LI>Adapted the Oligator page for the Optimization program, now called the Optimus
</LI><LI>Made progress on solving the system of recursive relations that we have
</LI><LI>Performed IPTG and tet experiments 
</LI><LI>PCR screened both “cre constructs”
</LI><LI>pBad+RBS+Cre ligation unsuccesful
</LI></UL><P><B>July 21, 2010</B></P><UL><LI>IPTG experiment started on optimized and deoptmized TetA constructs
</LI><LI>Passage I715039-1 and I715039-2 experiments started  </LI><LI>Started to link the ruby program on the webpage
</LI><LI>Finished theoretical probabilities for Construct A of size 4
</LI><LI>&quot;Floxed constructs&quot; ligated into pLac+RBS+Cre 
</LI></UL><P><B>July 22, 2010</B></P><UL><LI>IPTG experiment started using varying tetracycline concentrations
</LI><LI>Cell count data collected on IPTG TetA experiment </LI><LI>Linked the Optimus on the webpage
</LI><LI>Started theoretical probabilities for Construct A of size 5
</LI><LI>Cre was religated into pBad+RBS
</LI></UL><P><B>July 23, 2010</B></P><UL><LI>Cell count data collected from IPTG experiment
</LI><LI>Flourimeter data collected from IPTG experiment </LI><LI>Worked on the tutorial for the Optimus and added various features
</LI><LI>Worked on a Matlab tool based on the pancake problem that can find the theoretical probabilities of different modules in different positions
</LI><LI>Last of the “floxed constructs” were ligated 
</LI></UL><P><B>July 26, 2010</B></P><UL><LI>Missouri traveled to Davidson to work on wiki and presentation materials
</LI><LI>The two campuses reunite and share recent findings </LI></UL><P><B>July 27, 2010</B></P><UL><LI>Began work on the team wiki </LI></UL><P><B>October 25, 2010</B></P><UL><LI>Finalizing team wiki </LI></UL><P><CENTER><A href="#main_wrapper">top</A></CENTER></P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Davidson-MissouriW/Notebook">http://2010.igem.org/Team:Davidson-MissouriW/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Davidson-MissouriW/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Davidson-MissouriW/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Davidson-MissouriW/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Davidson-MissouriW/Notebook&amp;oldid=154227" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>