"
Page
Discussion
View source
History
teams
Log in
 
Team:Davidson-MissouriW
From 2010.igem.org
Home
Team
Overview
Advisors
Students
Project
Abstract
Introduction
Optimizing Codons
Characterizing Cre/Lox
Measuring Gene Expression
Summary & Outlook
References
Parts
Tools
Veripart
Oligator
Optimus
SimuLox
Human Practices
Notebook
Safety
Acknowledgements
Optimizing Codons
Building weighted items for the Knapsack through codon variation of the TetA gene led to Foundational Advances        
Details
Characterizing Cre/Lox
Foundational Advances were made as 11 novel lox sites for Cre recombination were built for randomly choosing Knapsack objects
Details
Measuring Gene Expression
Design and construction of a Knapsack biological computer required Foundational Advances in the measurement of gene expression
Details
iGEM Davidson – Missouri Western 2010:
Foundational Advances in Biology and the Knapsack Problem
The Davidson/Missouri Western multidisciplinary team is using synthetic biology to address a mathematical problem in Escherichia coli.  Specifically, we are addressing the Knapsack Problem, an NP-complete problem that asks, “Given a finite number of weighted items, can one find a subset of these items that completely fills a knapsack of fixed capacity?”
In our design, weighted items are represented by versions of TetA genes that confer measurably distinct levels of tetracycline resistance.  We have altered the codons of the wild type TetA gene, optimizing and de-optimizing several segments of the coding sequence.  Each TetA variant is coupled with a distinctive fluorescent gene, and each pair of genes is flanked by lox sites.  In the presence of Cre protein, the lox mechanism either inverts or excises the coding sequence, yielding different combinations of expressed TetA variants.  An expressed variant corresponds to an item being placed in the knapsack.  Over-expression of TetA results in cell death, which represents exceeding the capacity of the knapsack.  Under-expression of TetA causes the cells to stop growing due to tetracycline in the growth medium, which represents not completely filling the knapsack.  Surviving cells correspond to cells within a certain range of TetA production and the fluorescence tag allows for comparative measurement within this range.
The team is also working to develop software tools relevant to the specific project and applicable to projects in the wider synthetic biology community.
Team
The 2010 iGEM team from Davidson College and Missouri Western State University is composed of approximately 15 multidisciplinary undergraduate students and 4 professors – 2 biologists and 2 mathematicians. The team includes math, biology, computer science, and chemistry majors. The team has traveled back and forth across the country and research was conducted on both campuses. View the Davidson- Missouri Western team page.
Project
In an attempt to solve the knapsack problem, we explored a variety of different topics. We optimized the codons for a portion of the TetA gene in order to produce variant genes that confer differing amounts of tetracycline resistance. We also created 11 variant lox sites that have differing recombination frequencies. Finally, we explored gene expression of RFP and the TetA gene. View the work done by Davidson and Missouri Western undergrads.
Notebook
Lab notebooks are an integral part of conducting scientific research because the results of a scientific experiment must be reproducible.  In an effort to properly document our efforts, each team member kept a detailed record of their daily activities.  We have condensed the information from all of these sources so that each entry in this virtual notebook contains the highlights of each day’s work.  View the daily progress of our project via the lab Notebook .
Parts
BioBricks are the foundation of iGEM.  We have created more than 40 basic and composite parts that are now available for the entire synthetic biology community to use.  Among these parts are 11 new variant lox sites in both forward and reverse versions. Using these variants, we have constructed “modules” consisting of RFP floxed by multiple different combinations. Furthermore, we have assembled new cre recombinase expression cassettes and added them to the RFP modules. View the parts built by our team.
Tools
We have designed many programs that will be useful to the public.  VeriPart will identify the BioBrick part associated with any DNA sequence thus eliminating the tedious process of manually confirming sequences.  The Oligator suggests which oligos are needed to assemble the submitted sequence.  The Optimus allows users to choose different equations to optimize a given segment of DNA.  The Construct Simulator models how floxed modules behave when exposed to cre.  The Knapsack Game is an educational tool intended to explain the problem.  View our Tools page.
Acknowledgements
This project and our participation in iGEM 2010 would not have been possible without help from numerous sources.  We have received invaluable assistance from numerous people both at Davidson College and at Missouri Western State University.  Furthermore, many organizations have contributed generously to our efforts, and without their help, we could not have come this far.  This section is a thank you to our sponsors and all of those who have helped us in any way.
Top
Retrieved from " http://2010.igem.org/Team:Davidson-MissouriW "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
