"
Page
Discussion
View source
History
teams
Log in
 
Team:Washington/Notebook/NheI
From 2009.igem.org
Home
Project Description
Target
Secretion
Display
Future Directions
CDS
Foldit
Notebook
Safety
References
The Team
Accomplishments & Submitted BioBricks
 
 
 
Gene Assembly using the NheI site
Start with first 23 coding nucleotides of gene, eg. 5'-atgcgtaaaggagaagaacttt...-3'
Replace the atg start codon with the XbaI site: 5'-TCTAGA-3', eg. 5'-TCTAGAcgtaaaggagaagaacttt-3'
Add 6-8 random nucleotides to the 5' end of the primer, eg. 5'-cgggcTCTAGAcgtaaaggagaagaacttt-3'
Tweak the 3' end of the primer (add /remove nucleotides) so that the annealing temperature is close to 58C
Next start with the last 23 coding nucleotides eg.5'-tattttcagggtgctagctaa-3'
Remove the stop codon(s), in this case taa, and replace with the SpeI cut site ACTAGT, eg. 5'-tattttcagggtgctagcACTAGT-3'
Add 6-8 random nucleotides to the 3' end of the primer, eg. 5'-tattttcagggtgctagcACTAGTctgggtc-3'
REVERSE COMPLEMENT PRIMER, eg. 5'-tattttcagggtgctagcACTAGTctgggtc-3'   --> 5'-gacccagACTAGTgctagcaccctgaaaata-3'
Tweak the 3' end of the primer (add /remove nucleotides) so that the annealing temperature is close to 58C
Amplify your gene using the designed forward and reverse primers
PCR purify the PCR product
To ensure that the proper size fragment was amplified 5uL of PCR reaction can be run on an agarose gel
Digest PCR product with XbaI and SpeI
PCR Purify
Digest Vector with NheI and CIP
PCR purify
Mix insert and vector in 3:1 ratio and ligate
Transform into competent cells
SCREEN CELLS FOR CORRECT INSERT ORIENTATION by colony PCR using VF2 and custom reverse oligo
Retrieved from " http://2009.igem.org/Team:Washington/Notebook/NheI "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
