<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Washington_Project_Display"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Washington/Project/Display">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Washington/Project/Display&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Washington/Project/Display&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Washington/Project/Display&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Washington/Project/Display" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Washington/Project/Display</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="uw_leftBar"><DIV id="uw_ss"><UL id="uw_navBar"><LI><A href="/Team:Washington" title="Team:Washington">Home</A></LI><LI><A href="/Team:Washington/Project" title="Team:Washington/Project">Project Description</A></LI><LI><A href="/Team:Washington/Project/Target" title="Team:Washington/Project/Target">Target</A></LI><LI><A href="/Team:Washington/Project/Secretion" title="Team:Washington/Project/Secretion">Secretion</A></LI><LI><STRONG class="selflink">Display</STRONG></LI></UL><LI><A href="/Team:Washington/Future" title="Team:Washington/Future">Future Directions</A></LI><LI><A href="/Team:Washington/Project/CDS" title="Team:Washington/Project/CDS">CDS</A></LI><LI><A href="/Team:Washington/Project/FoldIt" title="Team:Washington/Project/FoldIt">Foldit</A></LI><LI><A href="/Team:Washington/Notebook" title="Team:Washington/Notebook">Notebook</A></LI><LI><A href="/Team:Washington/Safety" title="Team:Washington/Safety">Safety</A></LI><LI><A href="/Team:Washington/Bibliography" title="Team:Washington/Bibliography">References</A></LI><LI><A href="/Team:Washington/Team" title="Team:Washington/Team">The Team</A></LI><LI><A href="/Team:Washington/Accomplishments" title="Team:Washington/Accomplishments">Accomplishments &amp; Submitted BioBricks</A></LI><P>   </P></DIV></DIV><DIV id="uw_content"><DIV><DIV style="float:left;"><B><A href="https://2009.igem.org/Team:Washington/Project" class="external text" rel="nofollow">&lt; Project Description</A></B></DIV><DIV style="float:right;"><B><A href="https://2009.igem.org/Team:Washington/Project/Release" class="external text" rel="nofollow">Release &gt;</A></B></DIV></DIV><H1><SPAN class="mw-headline" id="Background"><B>Background</B></SPAN></H1><P>Our system hinged on finding a protein which could bind other proteins to the outside of the cell, but whose interaction with these proteins was weak enough to be disrupted by another small molecule. streptavidin presented itself as a logical choice.  Several protein tags (such as the Nano-Tag<SUP>1</SUP>) have been developed to bind streptavidin at the biotin binding site, but can be released when biotin is added to the system and binds to streptavidin.  The ability for peptides to bind streptavidin and be released upon the addition of biotin is a technology currently used on a daily basis world-wide by labs purifying proteins.  The major difference between the traditional system and our system is that streptavidin is attached to beads which are used in a column format in the traditional protein purification method.  Our system will have streptavidin attached to the surface of the cell.  Combining the display system with our target and secretion vector our vision is complete.  A single cell can then produce, secrete, bind, and release any protein of interest.
</P><TABLE cellpadding="1" cellspacing="8"><TBODY><TR><TD>The Nano-Tag binds streptavidin with 4nM binding affinity and has been used to purify multiple proteins. The ability of biotin to compete off the Nano-Tag in biding assays indicates that the binding occurs at the same site of streptavidin. Of great importance for incorporating streptavidin into our display system is that streptavidin is a common protein used in bio-chemical assays. Its ability to bind biotin is one of the strongest non-covalent interactions known. This allows for the utilization of biotin and streptavidin to act as molecular connectors in protein systems. Furthermore the availability of biotinylated and streptavidin linked fluorophores allows for an easy assay of their presence, using fluorescence microscopy<SUP>2</SUP>.
</TD></TR></TBODY></TABLE><TABLE cellpadding="1" cellspacing="8"><TBODY><TR><TD>Next we needed a way to anchor the streptavidin to the outer membrane. The obvious choice was to use the LPP signal peptide and Outer Membrane Protein A (OmpA) because this system has been extensively characterized as an expression system to display proteins on the cell surface. This LPP-OmpA system has been used to display a diverse group of proteins, including Green Fluorescent Protein (GFP)<SUP>3</SUP>, Organophosphorus Hydrolase (OPH)<SUP>4</SUP>, Cyclodextrin Glucanotransferase (CGTase)<SUP>5</SUP>,  Methyl Parathion Hydrolase (MPH)<SUP>6</SUP>, and Enhanced Green Fluorescent Protein (EGFP)<SUP>6</SUP>.  Thus, an appropriate display construct for our system would be a Lpp-OmpA-streptavidin fusion protein, which would be anchored in the outer membrane and display streptavidin on the surface of the cell. Streptavidin would bind the nanotag on our target protein in an interaction that could be disrupted by adding biotin.
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Harvard_2006_Surface_Display_System:_Legacy_Parts"> Harvard 2006 Surface Display System: Legacy Parts </SPAN></H4><P>When we checked the Parts Registry to see if someone had already built this or sometime similar that we could use as a starting part for our Idealized Protein Purification Display System, we found the Lpp-OmpA-streptavidin construct submitted by the 2006 Harvard iGEM team<SUP>7</SUP>.  In fact, Harvard had submitted four biobrick parts which were variations of the same function.  All four are fusion proteins which have a LPP signal peptide, either one or five trans-membrane ompA, and either monomeric or a single-chain dimeric streptavidin. All the variations and the resulting biobricks are shown below.
</P><TABLE cellpadding="3" cellspacing="1" border="2" width="75%" align="center"><TBODY><TR><TH> Bio Brick
</TH><TH> OmpA trans-membrane domains
</TH><TH> Type of Streptavidin
</TH></TR><TR align="center"><TD>J36848
</TD><TD>1
</TD><TD>monomeric
</TD></TR><TR align="center"><TD>J36849
</TD><TD>1
</TD><TD>dimeric
</TD></TR><TR align="center"><TD>J36850
</TD><TD>5
</TD><TD>monomeric
</TD></TR><TR align="center"><TD>J36851
</TD><TD>5
</TD><TD>dimeric
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Harvard_2006_Surface_Display_System:_Documented_Data"> Harvard 2006 Surface Display System: Documented Data </SPAN></H4><P>Data regarding the activity of the Harvard iGEM 2006 Lpp-OmpA-Streptavidin construct has been published in IET Synthetic Biology in 2007<SUP>7</SUP>.  The author showed, by Western blot against the His tag fused to this construct, that their protein was being expressed within the cell.  In order to demonstrate binding of their cell-surface-anchored streptavidin to biotin, the author incubated cells expressing Lpp-OmpA-streptavidin with an aptamer that linked biotin to a fluorophore using an oligonucleotide linker (see figure below).  Any cells that express streptavidin on the surface of the cell should fluoresce in their assay - concordantly, when cells were incubated with this biotin-oligo-fluorescein aptamer, the author saw fluorescence on cells expressing Lpp-OmpA-Streptavidin, but not on Lpp-OmpA negative control.  However, cells that expressed streptavidin on the surface also bound to the negative aptamer control (oligo-fluorescein) that lacked biotin, indicating that the oligo aptamer itself (and not biotin) may have been responsible for the association of the fluorophore to the cell surface.  Thus, the Harvard 2006 iGEM team was never able to definitively show specific affinity of their Lpp-OmpA-streptavidin construct.
</P><TABLE><TBODY><TR><TD><P>L01W: Lpp-OmpA-Streptavidin monomer, no His Tag.
L01H: Lpp-OmpA-Streptavidin monomer with His tag.
L01S: Lpp-OmpA-Streptavidin dimer.
L01: Lpp-OmpA alone).
A: no aptamer added.
B: With only ‘F’ 50 -ﬂuorescently-tagged oligonucleotide.
C: With ﬂuorescently tagged streptavidin aptamer.
D: With ‘B-F’ hybrid of 50 -biotinylated oligonucleotide annealed with 50-ﬂuorescently-tagged oligonucleotide<SUP>7</SUP>.
</P></TD></TR></TBODY></TABLE><P>
Based on these experiments, we decided to try using this display system for our project.  But since binding of the system to streptavidin had not be definitively shown, we wanted to verify binding of the Lpp-OmpA-streptavidin construct to biotin for ourselves, and characterize this interaction.
</P><H1><SPAN class="mw-headline" id="Experiments"><B>Experiments</B></SPAN></H1><H2><SPAN class="mw-headline" id="Harvard.27s_Legacy_2006_Cell_Surface_Display_Parts:_Do_They_Work_As_Predicted.3F"> Harvard's Legacy 2006 Cell Surface Display Parts: Do They Work As Predicted? </SPAN></H2><P>In order to determine whether the Harvard 2006 iGEM streptavidin cell surface display parts were, first, expressing properly, and, second, binding biotin on the surface of the cell, we decided to do the following tests: 
</P><OL><LI> Western blot to verify expression of Harvard 2006 surface display parts 
</LI><LI> Assess biotin binding to cell surface by fluorescence microscopy 
</LI><LI> Assess biotin binding to cell surface by flow cytometry
</LI></OL><H3><SPAN class="mw-headline" id="Western_Blot_Demonstrates_that_Harvard_2006_iGEM_Parts_Are_Expressed"> Western Blot Demonstrates that Harvard 2006 iGEM Parts Are Expressed </SPAN></H3><P>The goal of this experiment was to make sure that the proteins were being expressed in our cell strains, and also to make sure that they were the correct length. This was crucial to ascertain before we moved on and began to test the parts. Even though we had the individual parts sequenced confirmed we needed to make sure that the proteins were being expressed correctly in the cell. Since all the Harvard parts are His tagged, we used a Western blot reagent (horseradish peroxidase) that was conjugated to nickel-NTA which binds His tags.<SUP>8</SUP>.
</P><H4><SPAN class="mw-headline" id="Data"> Data </SPAN></H4><P>The expected sizes of each of these proteins are shown in the table below:
</P><TABLE cellpadding="3" cellspacing="1" border="2" width="50%"><TBODY><TR><TH> Bio Brick
</TH><TH> Length (in Da)
</TH></TR><TR align="center"><TD>J36848
</TD><TD>21478.7
</TD></TR><TR align="center"><TD>J36849
</TD><TD>34600.9
</TD></TR><TR align="center"><TD>J36850
</TD><TD>31215.4
</TD></TR><TR align="center"><TD>J36851
</TD><TD>44972.3
</TD></TR></TBODY></TABLE><P>
Based on the table above, we were able to verify expression of the Harvard 2006 iGEM surface display parts.  The varying intensity of the bands does not indicate the strength of expression because the protein amount was not normalized before it was inserted into the gel.  Next we wanted to determine whether functional, biotin-binding streptavidin was displayed on the cell surface in cells expressing these parts. To test this, we used two different methods: visualization by fluorescence microscopy and by flow cytometer. </P><H3><SPAN class="mw-headline" id="Streptavidin-Biotin_Binding_Is_Not_Visualized_By_Fluorescence_Microscopy"> Streptavidin-Biotin Binding Is Not Visualized By Fluorescence Microscopy </SPAN></H3><P>The goal of this experiment was to confirm the display of streptavidin on the surface of the cell. To do this, we incubated cells expressing the Harvard 2006 iGEM streptavidin surface display parts with a biotinylated fluorophore<SUP>9</SUP>.  If cells are expressing functional streptavidin on the surface of the cell, they should bind the biotinylated fluorophore, and this binding should be detectable in our florescence microscope as a halo of fluorescence surrounding each individual cell.  As a positive control for streptavidin-biotin binding we incubated the biotinylated fluorophore with streptavidin-coated beads that were roughly the same size (with respect to volume) as our cells (except spherical) (see <A href="https://2009.igem.org/Team:Washington/Notebook/Microscope" class="external text" rel="nofollow">Notebook</A> page for protocols).
</P><H4><SPAN class="mw-headline" id="Data_2"> Data </SPAN></H4><TABLE><TBODY><TR><TD><P style="font-size:18px"> Positive Control </P></TD><TD><P style="font-size:18px"> Negative Control </P></TD><TD rowspan="4">These images were analyzed using imageJ<SUP>10</SUP>. The image on the left shows an image of the beads with fluorophore added to them. These beads were diluted and spun diluted and spun down until the background level of fluorescence was low enough to get an accurate reading. We then used imageJ to analyze the intensity of a line going through the bead, which is demonstrated by the above schematic. On the positive control the edges of the streptavidin-coated beads show spikes in fluorescence, indicating binding of the biotinylated fluorophore to the beads. The negative control (beads without fluorophore) showed no such increase. This meant that our biotinylated fluorophore binds to streptavidin in a detectable manner.  When our cells were examined in the same manner, no difference could be seen in biotinylated fluorophore binding between cells that had induced expression of surface streptavidin (left) and uninduced cells which should not express the surface display protein (right).  This was evidence that the streptavidin surface display part binding to biotin was very low / nonexistent.  In order to verify this, we measured the binding of the biotinylated fluorophore to entire populations of streptavidin-expressing cells by flow cytometry.</TD></TR><TR><TD><P style="font-size:18px"> Induced Cells </P></TD><TD><P style="font-size:18px"> Uninduced Cells </P></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Streptavidin-Biotin_Binding_Is_Not_Visualized_By_Flow_Cytometry"> Streptavidin-Biotin Binding Is Not Visualized By Flow Cytometry </SPAN></H3><P>The goal of this experiment was to visualize biotin binding to streptavidin over a whole population of cells.  Whereas the Microscope experiment looked at a few localized cells, we set the cytometer to look 50,000 streptavidin-expressing cells after incubation (and washing) with the biotinylated fluorophore and read the resulting fluorescence (<A href="https://2009.igem.org/Team:Washington/Notebook/Flow_Cytometry" class="external text" rel="nofollow">see Notebook page for protocols</A>). Like in the microscopy assay, we used streptavidin-coated beads as a positive control. The cytometer reads the fluorescence of each individual particle (cell or bead) passing through it, which allows for accurate readings, and monitoring of a much larger sample size than we visualized under the microscope. The results of this experiment are described below.
</P><H4><SPAN class="mw-headline" id="Data_3"> Data </SPAN></H4><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 385px;"><DIV class="gallerytext"><P><B>BBa_J36848</B><I>This image shows both the induced and uninduced cells for part 48 in varying levels of  fluorophore (0nM to 100nM). The cells were induced at 1mM IPTG. This data shows that there is no appreciable difference between the induced and uninduced cells at any given level of fluorophore. All curves appear to have the same amount of fluorescence. We found similar results using a microscopy assay.</I></P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 385px;"><DIV class="gallerytext"><P><B>+ Control</B><I>We used the streptavidin coated beads<SUP>11</SUP> to show us what the magnitude of fluorescence increase we should see with increased fluorophore levels. We were able to see that as the level of fluorophore was increased we could see increased retention between the beads and the fluorophore. The black is beads with no fluorophore, the red is with 10 nM, and the purple is 100 nM. These showed a clear difference between the beads without fluorophore, and the beads with fluorophore. The cells shown at right, matched the readings of the beads when they had no fluorophore added.</I></P></DIV></DIV></TD></TR></TBODY></TABLE><P>In the flow cytometer, our positive control, streptavidin-coated beads, showed a clear distinction between beads pre-incubated with biotinylated fluorophore and the beads without fluorophore, indicating that this assay is capable of visualizing streptavidin binding to the biotinylated fluorophore.  Like in the microscopy assay, we did not observe  appreciable binding occurring with any of the ompA-streptavidin parts to biotin at the population level. This data only shows part number 48. However this data is applicable to all of the parts we observed. All streptavidin-expressing cells demonstrated low levels of fluorescence comparable to those of beads not pre-incubated with fluorophore.
</P><H3><SPAN class="mw-headline" id="Conclusion"> Conclusion </SPAN></H3><P>In conclusion, we demonstrated expression of the Harvard 2006 streptavidin surface display parts via Western Blot. We were unable, however, to visualize binding between a biotinylated fluorophore and the cells expressing these proteins. This indicates that the streptavidin display system is likely not binding biotin correctly. We hypothesize that streptavidin might not having enough room on the cell surface to form tetramers (which is its native state), and thus may not be binding to biotin with high efficiency. In order to generate a construct that demonstrates tight binding to to biotin, we propose to design a generalized cell surface display system, and to computationally design a biotin-binding streptavidin monomer using software developed at the University of Washington.</P><P><B>These solutions are described in the <A href="https://2009.igem.org/wiki/index.php?title=Team:Washington/Future" class="external text" rel="nofollow">future directions section</A>.</B></P><H3><SPAN class="mw-headline" id="Citations"> Citations </SPAN></H3><OL><LI>The Nano-tag, a streptavidin-binding peptide for the purification and detection of recombinant proteins <A href="http://www.ncbi.nlm.nih.gov/pubmed/14680960" class="external text" rel="nofollow">doi:10.1016/j.physletb.2003.10.071</A></LI><LI>Structural origins of high-affinity biotin binding to streptavidin. PC Weber, DH Ohlendorf, JJ Wendoloski, and FR Salemme (6 January 1989). Science 243 (4887), 85. <A href="http://www.ncbi.nlm.nih.gov/pubmed/2911722" class="external text" rel="nofollow">DOI: 10.1126/science.2911722</A></LI><LI>Display of green fluorescent protein on <I>Escherichia coli</I> cell surface. Shi H, Wen Su W. Department of Biological &amp; Agricultural Engineering, University of Missouri-Columbia, 65211, Columbia, MO, USA <A href="http://www.ncbi.nlm.nih.gov/pubmed/11118595" class="external text" rel="nofollow">Enzyme Microb Technol. 2001 Jan 2;28(1):25-34</A></LI><LI>Cell surface display of organophosphorus hydrolase using ice nucleation protein. Shimazu M, Mulchandani A, Chen W.Department of Chemical and Environmental Engineering, and Environmental Toxicology Program, University of California, Riverside, CA 92521, USA. <A href="http://www.ncbi.nlm.nih.gov/pubmed/11170483" class="external text" rel="nofollow">Biotechnol Prog. 2001 Jan-Feb;17(1):76-80</A></LI><LI>Anchorage of cyclodextrin glucanotransferase on the outer membrane of <I>Escherichia coli</I>. Wan HM, Chang BY, Lin SC. Department of Chemical Engineering, National Chung Hsing University, Taichung, 402, Taiwan. <A href="http://www.ncbi.nlm.nih.gov/pubmed/12115409" class="external text" rel="nofollow">Biotechnol Bioeng. 2002 Aug 20;79(4):457-64</A></LI><LI>Cell surface display of functional macromolecule fusions on <I>Escherichia coli</I> for development of an autofluorescent whole-cell biocatalyst. Yang C, Zhao Q, Liu Z, Li Q, Qiao C, Mulchandani A, Chen W. State Key Laboratory of Integrated Management of Pest Insects and Rodents, Institute of Zoology, Chinese Academy of Sciences, Beijing 100101, China. <A href="http://www.ncbi.nlm.nih.gov/pubmed/18767673" class="external text" rel="nofollow">Environ Sci Technol. 2008 Aug 15;42(16):6105-10</A></LI><LI>Cell surface streptavidin. Tsai P. <A href="http://dx.doi.org/10.1049/iet-stb:20070002" class="external text" rel="nofollow">IET Synth Biol. 2007;1(1.2):32</A></LI><LI><A href="http://www.kpl.com/catalog/productdetail.cfm?catalog_ID=17&amp;Category_ID=448&amp;Product_ID=1085" class="external text" rel="nofollow">HisDetector Nickel-HRP</A></LI><LI>Biotin-5-fluorescein, <A href="http://www.sigmaaldrich.com/catalog/ProductDetail.do?lang=en&amp;N4=53608|SIGMA&amp;N5=SEARCH_CONCAT_PNO|BRAND_KEY&amp;F=SPEC" class="external text" rel="nofollow">Sigma Aldrich</A></LI><LI><A href="http://rsbweb.nih.gov/ij/docs/index.html" class="external text" rel="nofollow">ImageJ</A></LI><LI>SVP-15-5 streptavidin coated polstyerene spheres (1.5-1.9 µm), <A href="http://www.spherotech.com/coa_pol_par.htm" class="external text" rel="nofollow">Spherotech</A></LI></OL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Washington/Project/Display">http://2009.igem.org/Team:Washington/Project/Display</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Washington/Project/Display" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Washington/Project/Display" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Washington/Project/Display&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Washington/Project/Display&amp;oldid=166359" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>