Team:Paris Bettencourt/Notebook/Differentiationyeast
Ferment It Yourself
iGEM Paris-Bettencourt 2O15
Background
Design
Project
Micronutrients
Continuity
Modeling
Manufacturing
Achievements
Results
Parts
Medal Fulfillments
Practices
Overview
SynBio Challenge
DIλ
Acceptance
Events
Rhizi
Collaborations
Safety
Safety Overview
Safety Form
Review Board
Notebook
Daily notebook
Protocols
Bibliography
Team
Members
Attributions
Acknowledgments
Press
Official Team Profile
Notebook - Differentiation in Yeast
July
August
Wednesday 07/15
Started to work on the packaging experiments for yeast.
Innoculation of S. cerevisiae BY4743 mCherry
Medium: 100 ml YPD with 400 ul Gentamycine (four 25 ml tubes)
Culture @ 30°C + Shaking
Friday 07/17
Checked the yeast megaculture: growth is okay, but let’s wait for a third day to have more biomass.
Designed and ordered a lot of oligos for Yeastbow and Colibow projects.
Saturday 07/18
Manufacturing experiment, part 1.
A strain of Yeast expressing mCherry was grown during three days at 30°C in 100 ml YPD.
In order to remove as much medium as possible, it was centrifuged at 4000 G during 5 min, twice, while discarding the supernatant each time.
Different conditions of drying were attempted:
In the tube, at the sun during 30 min (Tube1). This one was not completely dry when packaged.
Spread on tinfoil (Grattée)
Put on tinfoil without spreading (Air libre)
Put on tinfoil and covered with a plastic plate (Boîte)
Leftover in the tube (Tube2)
After drying these samples were packaged in paper bags and preserved in the fridge.
Monday 07/20
Received oligos o15.74 and o15.75. They’re meant to amplify the Yeastbow source plasmids.
Prepared 9 YPD agar plates.
Manufacturing experiment, part 2.
For each drying protocol, one sample was taken out of the fridge.
They were weighted, transferred in eppendorf tubes and ~1ml of LB was added (YPD not available at that time).
The « tube 1 » sample was not dry enough so a lot of it sticked to the package.
ID
Boîte 2
Tube 1
Air libre 1
Grattée 1
Weight
12 mg
4.5 mg
9.2 mg
1.3 mg
This was then moved to YPD-agar plates and spread with sterile beads.
The plates were cultured at 30°C overnight for colony counting.
Thursday 07/16
Ordered two new oligos for Yeastbow cloning: o15.74 and o15.75
Yeast megaculture for wednesday: A little cell pellet is visible at the bottom.
-> Titled the flasks so they’re shaked better.
Obtained two yeast strains that express CRE recombinase from Zoran, overnight at 30.
Designed and ordered new primers for yeastbow PCRs, based on Matt’s advice.
It was very difficult to find a good priming region since this seems to be in a terminator.
Monday 07/20
Reception of pBrainbow 1.0 and 3.0 propogators as glycerol stocks
Templates for PCR assembly of Yeastbow. Still need primers.
-> Overnight culture for miniprep for PCR
pThy Brainbow 1.0
Reception of oligos o15.74 and o 15.75
For yeastbow first round of PCRs.
Reconstitution in dH2O, in the bottom drawer of the -20.
Reception of pBrainbow URA3 for Yeastbow
From Zack at Hersen lab.
It carries an AmpR.
Use 1 ul as a template for the PCR, and if it doesn’t work transform the rest into E. coli for propagation.
Stored in Barth’s styrofoam box.
Tuesday 07/21
Results for the preliminary manufacturing experiment
We get a lot of background. It’s almost sure that it is yeast, but no mCherry could be detected. The plasmid was likely lost due to the absence of antibiotics in the plating media.
It’s important to dilute before plating to know the number of colonies per mg.
All the plates are covered in yeast, even the less successful ones.
Reception of oligos o15.76 to o15.96
Yeastbow SOE
R6K linearization
Colibow gBlock Amp
Colibow Sequencing
The ultramers (o15.78, o15.79) for Leu2 flanking are not ready yet.
-> Oligo reconstitution and dilution
-> Miniprep of R6K vector
-> PCR of R6K vector
-> Miniprep of pBrainbow 1.0
-> PCR of pBrainbow 1.0
-> PCR of pThy Ura3
Reconstituted all primers to 100 ug/ml.
Miniprep of pR6K and pBrainbow 1.0
For PCR, using Promega kit w/ double wash. Elution in 30 ul
Final concentration: 93 ng/ul
PCR pURA3
1 ul template
1 ul of each primer
25 ul of Phusion master mix
22 ul of dH2O
Template: pKT174 plasmid (AmpR).
We have ~5 ul * 2, for a total amount of 1 ug.
Use 1 ul for 1 PCR, then if needed transform the rest in E. coli for propagation.
Primers:
URA F+
o15.74
56°
URA R
o15.77
58°
Polymerase: Phusion in the form of 2x mix.
Program: (saved as “Phusion”)
98 (30s)
98 (10s) 50 (25s) 72 (1 min) x 25
72 (8 min)
4° hold
Lasts 1h30, product size = 1684 bp
Miniprep of the pThy1.0 plasmid
With promega kit.
Recovery in 30 ul.
Preparation of Agarose 1% TAE
Kept at 55°C in the broken incubator.
PCR of pThy1.0 cassette
1 ul template
1 ul of each primer
25 ul of Phusion master mix
22 ul of dH2O
Template: pThy1.0 (1 ul = 93 ng)
Primers:
B1 Brain1 F
o15.76
55°
B1 Brain1 R+
o15.75
58°
Program: (modified from Phusion)
98 (30s)
98 (10s) 50 (25s) 72 (2’45’’) x 25
72 (8 min)
4° hold
Expected size: 3965 bp
Wednesday 07/22
Gel for product size checking, in that order:
Generuler 1 kb+
PCR Thy1.0 -> 3965 bp
PCR Ura3 -> 1684 bp
Results:
PCR Thy1.0 is ~ 4000
PCR Ura3 is ~ 1700
It worked!
PCR purification
Using the kit.
Titration:
PCR Thy1.0: 32 ng/ul
PCR Ura3: 47 ng/ul
Overlap PCR
Equimolar amounts of purified fragments
Required volumic ratio Thy/Ura = (47 / 1684) / (32 / 3965) = 3.46
PCR Thy1.0
17 ul
PCR Ura3
5 ul
Water
3 ul
Phusion master mix
25 ul
Do not add any primers; the templates will prime each-other.
Run 5 PCR cycles without primers. Predicted annealing temperature ~ 61°C. Use 57° as an annealing temperature and 2’45’’ of elongation time.
Add primers directly to the mix.
B1 Brain1 F (o15.76) 55°C
Ura R (o15.77) 58°C
1 ul each, directly into the PCR product.
Expected product size: 5610 bp
Wednesday 07/23
Gel for SOE yeastbow assembly
1% agar TAE, with 1 kb+ generuler.
30 ul of product + 6 ul loading dye were loaded for gel extraction.
Smear at very large sizes
Band at ~1600, probably from Ura3 PCR
No visible band at ~6000 bp
The SOE didn’t work, so nothing was extracted.
New attempt for SOE yeastbow assembly
The concentration of template was way too high.
PCR Thy1.0         6 ul
PCR Ura3         2 ul
Water                14 ul
Phusion mix        25 ul
o15.76                2 ul
o15.77                2 ul
« S+ » tube: Primers added from the beginning,
« S » tube: Primers added only after 6 cycles.
Program
98° 30s
98° 15s, 50° 25s, 72° 4’20’’ [35 times] -> Shorter elongation time
72° 8 min
4° hold -> 12 next time
Lid at 103°. It takes about four hours to complete.
Friday, 07/24
Reception of oligos o15.78 and o15.79
For flanking yeastbow with homology regions
Reconstitution:
4 nmol oligos + 40 ul eau = 0.1 nmol/ul = 100 mM
Gel for previous SOE (2nd attempt)
Yeastbow SOE gel extraction Enhanced.jpg
SOE        SOE        SOE+        SOE+        1kb+ marker
2 barely visible bands around 5600 bp in SOE+.
It is not possible to know which one is the right one.
Gel extraction of 2 SOE+ bands
“Grande”
“Petite”
Recovery in 30 ul EB
Grande : 22.5 ng/ul
Petite: 6.2 ng/ul
PCR of extracted product
Aim: recover the tiny amount of DNA in the gel.
Template: SOE G or P after purification (29 ul)
Primers: 76 and 77 (2 ul each)
Phusion: 50 ul
Water: 17 ul
Program:
98 (30)
98 (10), 54 (25), 72 (3’20’’) -> 36x
72 (8’00)
12 (hold)
The increased annealing temperature (54 instead of 50) should lower the non-specific hybridation.
This PCR takes ~ 3h30.
Monday, 07/27
Gel for rescue PCR of extracted product
Agar 1% + TAE + SYBRsafe
Laid 5 ul + 1 ul loading dye
Order: « Petite » - Marker 1kb+ - « Grande »
Results
Did. Not. Work.
Thirld attempt at SOEing
Redo the PCRs with gel extraction this time -> No smear in pThy1.0 product
Try to Gibson assemble the two parts
Try again the SOE with a higher annealing temperature
New SOE:
PCR Thy1.0         3 ul
PCR Ura3         1 ul
Water                7 ul
Phusion mix        12.5 ul
o15.76                1 ul
o15.77                1 ul
Program
98° 30s
98° 20s, gradient 30s, 72° 3’10’’ [35 times] (NEB calculator recommends 60°C…)
72° 8 min
12° hold
Gradient: 56 to 62°C (bottom to top).
New PCRs
Objective: Better quality of the SOE reagents, gel-extracted to get rid of the smear.
Larger number of cycles.
Ura3 PCR
pKT174        1ul
o15.74                1ul
o15.77                1ul
Water                22ul
Phusion mix        25ul
Program:
98 (30)
98 (10), 56 (25), 72 (1’00) x35
72 (8)
12 (hold) -> 1684 bp
Thy PCR
pThy1.0        1ul
o15.75                1ul
o15.76                1ul
Water                22ul
Phusion mix        25ul
Program:
98 (30)
98 (10), 56 (25), 72 (1’00) x35
72 (8)
12 (hold) -> 3965 bp
Tuesday 07/28
Gel extraction for PCR Ura, PCR Thy and SOE gradient PCR
Agar 1%, SYBRsafe, TAE
1
1
U
U
M
A
B
C
D
M
1: pThy pcr
U: pUra3 pcr
M: 1kb dna ladder
A, B, C, D: gradient PCR for SOE
It was at first run at 75 kV and then at 100 kV during 25 in total. The buffer in the tank was TBE, and as a result the gel was completely awful, with no possibility to read the size of the bands.
The products were also lost.
New PCRs for Yeastbow
PCR Thy1.0
pThy1.0        1 ul
75                2.5 ul
76                2.5 ul
Phusion mix        25 ul
Water                19 ul
Program:
98 (60)
98 (10) 56 (30) 72 (120) x35
72 (600)
4 (hold)
Expected size: 3965. Should be over at about 18h30.
PCR Ura3
pKT174        1 ul
74                2.5 ul
77                2.5 ul
Phusion        25 ul
Water                19 ul
Program:
98 (60)
98 (10) 56 (30) 72 (60) x35
72 (600)
12 (hold)
Expected size: 1684 bp.
Wednesday 07/29
Gel for Yeastbow 1 and U and extraction
Agar 1%, SYBRsafe, TAE
1kb ladder
1
1
U
U
1kb ladder
3965
3965
1684
1684
The PCR for U worked really well, but the PCR for 1 yielded an extremely low amount of DNA and a smear.
-> Decrease annealing temp
-> Increase extension time
pcr Ura3: 41 ng/ul * 30 ul of EB
pcr Thy1: 13 ng/ul * 30 ul of EB
New attempt at PCR Thy1, 29 july
Mix
Phusion        25
Thy                1
o15.75                2.5
o15.76                2.5
Water                19
= Two tubes of 50 ul each -> 3965 bp
Program
98 (30)
98 (20), 60 (20), 72 (3’00’)
72 (5’00)
12 (hold)
New attempt at SOE, 29 july, with PCR products from 28 july
Mix
Phusion        25
PCR 1                1
PCR U                1
o15.76                2.5
o15.77                2.5
Water                11
= Two tubes of 50 ul each -> 5610 bp
Program
98 (30)
98 (20), 60 (20), 72 (3’00’)
72 (5’00)
12 (hold)
(Along with PCR 1)
Thursday, 07/30
Gel for SOE 29/07
1 kb ladder
SOE
SOE
SOE
1 kb ladder
Expected size: 5600 for all of them.
SOE+29.07.jpg
It didn’t work at all. Try to Gibson-assemble them.
Friday 07/31
Culture of pKT174 for freezing + miniprep
In 4 ml LB + 4 ul ampicilline, @37.
Saturday, 8/1
The culture of pKT174 grew:
Miniprep (not titrated for now)
Glycerol stock @ -20
New strategy based on Gibson assembly
With minimal PCR steps. Low success rate, low mutation rate.
yeastbow next assembly.png
Amplify pThy1.0 with o15.75 (58) and o15.78 (58) -> 4006 bp
Amplify pUra3 with o15.74 (56) and o15.79 (56) -> 1725 bp
Cleanup or extract (or try both)
Gibson assembly (39 bp overlap, Tm = 66°C) -> 5692 bp. Works on Snapgene (see Yeastbow Vector.dna)
Ready to go
Method for PCR: Jake’s magic recipe.
Reagent
Volume
 
1x
10x
Nuclease-free water
71 ul
710 ul
5x Phusion HF Buffer
20 ul
 200 ul
10 mM dNTPs
2 ul
20 ul
Forward Primer (10 uM)
1 ul
10 ul
Reverse Primer (10 uM)
1 ul
10 ul
Template
1 ul
10 ul
DMSO
3 ul
30 ul
Phusion DNA Polymerase
1 ul
 10 ul
Total Volume
100 ul
1000 ul
Thermocycler Protocol: NEB Phusion 
98 (30)
98 (10) 52 (30) 72 (30/kb) *35
72 (5’)
10 (forever)
Gel
L        M        L        R        M        R
2.5 ul sample + 0.5 LD
Marker: 1 kb
Both of the PCR worked quite well, the products are of the right size and there is no significant alien band.
-> PCR purification with the QIAGEN kit
Recovery in 40 ul of water.
Titration
Yeastbow R: 300 ng/ul (1725 bp)
Yeastbow L: 155 ng/ul (4006 bp)
Monday 08/03
Gibson assembly of yeastbow
On ice: 15 ul master mix + 5 ul DNA, incubate at 50° for 1 hour.
Number of ng = pmols * bp * 0.65
Fragment
ul
pmol
Yeastbow R
0.9
0.24
Yeastbow R
4.1
0.24
Gibson master mix
15 ul
Tuesday 8/4
Transformation of yeastbow in the CRE strains
Culture in 50 ml 2x YPD.
Problem: According to Matt these strains already have Ura3. It is not possible to transform yeastbow in them.
-> Transform the yeastbow in Ura- S. cerevisiae strains and add the CRE later (maybe by taking it from the yPH150/151)
Thanks to the fluorescence of the transformant, we attempted to transform the yeastbow cassette and select cells with mere fluorescence.
However, while in the process of doing the transformation, the centrifuge got stuck with my samples inside. I gave up at that point.
Freezer stocks of yPH150 and yPH151
500 ul glycerol 50% and 500 ul overnight culture.
Numbers 15.54 and 15.55 resp.
Yeastbow strikes back
CRE yeast strains we have:
yPH150: CRE under the Ura3 promoter (const)
yPH151: CRE under the Stl1 promoter (inducible)
These strains are based on BY4743. Both of them are auxotrophic for His, Leu and Lys, but no longer for Ura (as it was used for CRE transformation!)
All the work previously done on yeastbow is therefore useless, and it’s necessary to start over with His3 as a selection marker.
The Leu2 integration tails are correct, even if the Leu2 ORF has been destroyed.
Available plasmids with resistance:
pYMC32, pFA6, pPH75 from euroscarf toolbox. The names might not be correct.
For new primers were designed and ordered:
Amplify pThy1.0 with these primers:
o15.160 GCAATATATATATATATATTTCAAGGATATACCATTCTAACTAGCGAGCTCATAACTTCG (Tm=58, specific)
o15.161 GCTTATTTAGAAGTGGCGCGgacctctgcagaggaaggac (Tm=58, may hairpin but not likely, may dimerize, specific)
Product size: 4066 bp
Amplify pFA6-His3 with these primers:
o15.162 gtccttcctctgcagaggtcCGCGCCACTTCTAAATAAGC (may hairpin, may dimerize, specific) (Tm=56)
o15.163 AAGTTTATGTACAAATATCATAAAAAAAGAGAATCTTTCCTGGATGGCGGCGTTAGTATC (Tm=59, specific)
Product size: 1614 bp
Gibson assembly
Overlap: 40 bp, Tm = 70°C
Product size: 5640 bp
Monday, 08/10
Received plasmids and primers for Yeastbow
Plan with SOE:
Forward
Reverse
Template
PCR type
Tm
Extension Time
Product size
BB1F
BB1R
pThy1-Brainbow1.0
Phusion
60
2’
3965
His3F
His3R
pPH21 or PH75
Phusion
60
1’
1532
Pho85F
Pho85R
PCR1 and 2
SOE
60
3’
5457
Alternative setup
What do we have?
One brainbow plasmid: pThy1.0
One His3 resistance plasmid: pPH21
Two border primers without tails (not used here)
Two sets of central Gibson assembly primers: 161+162 or BB1R+His3F
Two sets of integration tails: Pho85F+Pho85R or 160+163
Every four combinations of central primers and integration tails were used separately.
Master mix for both PCRs
Name
1x
5x
Water
40
200
Phusion Master Mix
50
250
DMSO
3
15
Template
1
5
1.0 PCR
Mix:
2 ul of each primers. In that order:
PB: PhoF + BB1 R
1B: 160 + BB1 R
P1: PhoF + 161
11: 160 + 161
Expected result:
~4066 bp
Program (as seen in previous working Yeastbow assembly):
98 (30)
98 (10) 52 (30) 72 (3’00) x35
75 (5)
pPH21 PCR
Mix:
2 ul of each primers. In that order:
HP: His3F + PhoR
1P: 162 + PhoR
H1: His3F + 163
11: 162 + 163
Expected result:
~1614 bp
Program (as seen in previous working Yeastbow assembly):
98 (30)
98 (10) 52 (30) 72 (2’00) x35
75 (5)
The tubes for this one are marked with a blue spot on the hinge of the PCR tube.
Gel for both of these PCR
1kb
PB
1B
P1
11
1kb
HP
1P
H1
11
1kb
The reactions containing the 161 primer for 1.0 generated small fragments, they can’t be used after mere PCR cleanup and would require gel extraction. Don’t use 161-162 as a central junction, use PB or 1B. They might require gel extraction anyways, or re-try with less template.
1P and H1 worked best for His3 PCR. Use H1.
Conclusion: Use 1B : 160 (Leu2 tail F) + BB1 Use H1: His3F + 163 (Leu2 tail R) Try PCR cleanup them and follow with Gibson assembly.
Wednesday, 08/12
Yeastbow assembly
The following assembly was chosen according to the PCR screening:
Amplify pThy1.0 with 160 and BB1R (try to use less template)
Amplify pPH21 with His3F and 163.
Both of these products were purified.
1B was purified by JB, there is "1B Caca" written on the cap. Thanks, JB. It was eluted in 30 ul of water and the concentration is 119 ng/ul -> Rh
H1 was purified, eluted in 30 ul of water and the concentration is 231.8 ng/ul -> Lh
Thursday, 08/13
Yeast transformation
The pre-culture for yPH150/151 grew after two days. At 11 a.m. they were diluted in 50 ml of YPD 2x and put at 30°C for pre-transformation culture.
Gel for 1B PCR
1B and R6K pcrs were ran on the same gel:
1B | 1B | 1 kb ladder | R6K | R6K
2 ul water + 1 ul LD + 3 ul sample.
Result: There is one clean band at around >4000 bp. There is no significant smear above it, hence a better quality for Gibson assembly.
The PCR product was cleaned up and eluted in 30 ul of water. Concentration: 63 ng/ul.
Assembly of Yeastbow His3
Part
Conc. (ng/ul)
size (bp)
volume (ul)
amount (pmol)
L his3
63
4066
4.5
0.11
R his3
231
1614
0.5
0.11
+ 15 ul of MMII Gibson Assembly master mix.
Incubation 30 min at 50°C, then 5 min at 98 and 5 min at 72. The goal is to inactivate the exonuclease without inactivating the polymerase, to generate blunt-ends at the end.
yPH150/151 transformation with yeastbow
Denature 1ml DNA carrier for 5 min at 99° and chill on ice
Centrifuge 5 min at 4000 G, recover in 25 ml sterile water
Centrifuge 5 min at 4000 G, recover in 25 ml sterile water
Centrifuge 5 min at 4000 G, recover in 1 ml sterile water
Transformation mix: PEG 3350 (50%): 580 ul (attention, should have been 480) LiAc 1.0M: 72 ul Carrier: 100 ul Plasmid DNA: 16 ul Sterile water: 52 ul
360 ul + 150 ul cells -> vortex
Heatshock 50 min (instead of 40) at 42°C.
In the absence of appopriate medium, the result was centrifuged, the supernatant discarded and the pellet cultured in YPD overnight.
Friday 08/14
PCR on Yeastbow His3 assembly
Phusion master mix C
50
160
2
163
2
Assembly product
2
DMSO
3
Water
Program: "DMSO" with 55°C annealing temperature, and 4 minutes of extension. Expected size: 5457 bp.
Preparation of selective media without histidine
Base: 16.8g Glucose: 5g Supplement without histidine: 1.92g Agar: 15g Water: qsp 1000 ml This was then split between two 1L bottles and autoclaved.
Preparation of 2x YPD medium
Medium powder: 50g Water: qsp 500 ml
Preparation of LB agar medium
Water: qsp 500 ml Medium powder: 25 g Agar: 7.5 g
Plating of Yeastow transformation products
The YPD culture were centrifugated and recovered in 1 ml sterile water. Each strain was plated twice on Histidine- medium + agar, once with 100 ul and once with 250 ul of transformation product. There were then grown at 30°C.
-> Result: nothing grew. At all.
15/08
Splicing by Overlap Extension with His3
Phusion master mix C
50
160
2
163
2
Lh
4.5 ul
Rh
0.5 ul
DMSO
3
Water
At first the primers were not added to the mix.
98 (1'00) 98 (15) 52 (30) 72 (4'00) 5*
It took ~40 min.
Then the primers were added and the program for "DMSO PCR" was performed, with: - 54°C of annealing - 4' of extension - 37 cycles
Gel: 3 ul sample + 2 ul water + 1 ul LD [Successful]
It worked! There is a band at ~5500
16/08
Gel-extraction of the SOE product
60 ul of SOE product were deposed on a gel. Expected size: 5457 bp.
The migration failed miserably due to incorrect concentration of the TAE buffer. The deposited sample was lost. Fortunately, only a part of the SOE product was deposited.
Culture of yPH150 and ypH151
For transformation on Monday. These strains were put to culture in 3 ml YDP, with a YDP control for contamination.
PCR cleanup of the remaining SOE product
Elution in 30 ul of water. Titration: 40 ng/ul, very clean.
18/08
Yeastbow transformation
Following the "High-efficiency transformation of yeast protocol".
yPH150 did not grow, but it doesn't matter as it's not the most important strain (yPH151).
0D after overnight: 0.7 (=7.10^6 cells/ml)
2.5 ml of culture were diluted in 50 ml YPD at 11h30.
Transformation mix (in ul) PEG, 1080 LiAc, 162 ssDNAcarrier, 225 plasmid, 22.5 -> 200 ng per transformation sterile water, 130.5
Then 360 ul of mix per tube of cells. 1 control tube was made with 360 ul of sterile water.
Heatshock from 20h45 to 21h20.
19/08
Reproducing the successful SOE
Due to the small amount left of Lh product, only three tubes could be made. The tubes 1 and 2 contained exactly the same mix as the SOE of 15/08. The tube 3 was exactly the same except the primers 160 and 163 were replaced by Pho85 F and Pho85 R. These primers use a different homology region for yeast integration.
The protocol and program was exactly similar to what had worked before.
Results: None of the worked (gel not shown). It's weird that this previously-working reaction doesn't work anymore.
Yeastbow transformation results
Nothing visible (wait 3 days). Make new SOE as a plan B.
20/08
PCR on the working SOE product
Maybe we can get something good from what's left in the tube. A full annealing gradient was made with 8 samples.
Name
1x
8x
2x Phusion Master Mix
10
80
DMSO
0.6
4.8
160
1
8
163
1
8
SOE product
1
8
Water
6.4
51.2
Program: "DMSO" with 5 minutes of extension. 37 cycles. It takes like 6h.
Gradient: 61, 60.2, 58.8, 56.7, 54.2, 52.1, 50.8, 50
Results: It did not work. We cannot even repeat previously working experiment. There is definitely a problem with some materials or devices.
