<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Notebook_Phytase skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Paris Bettencourt/Notebook/Phytase</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="menuContainer"><DIV id="smallMenuTitle"><H1>Ferment It Yourself</H1><H3>iGEM Paris-Bettencourt 2O15</H3></DIV><UL><LI class="left"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Background" title="Background">Background</A></LI><LI class="left"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Design" title="Design">Design</A></LI><LI class="left"><BUTTON class="menuButton">Project</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Micronutrients">Micronutrients</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Continuity">Continuity</A></LI><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Manufacturing">Manufacturing</A></LI></DIV></UL><LI class="left"><BUTTON class="menuButton">Achievements</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Parts">Parts</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Medals">Medal Fulfillments</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Practices</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/SynBio_Challenge">SynBio Challenge</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/DILambda">DIλ</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Acceptance">Acceptance</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/Events">Events</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/Rhizi">Rhizi</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Collaborations">Collaborations</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Safety</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Safety">Safety Overview</A></LI><LI><A href="https://2015.igem.org/Safety/Final_Safety_Form?team_id=1678">Safety Form</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/ReviewBoard">Review Board</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Notebook</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Notebook">Daily notebook</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols">Protocols</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Bibliography">Bibliography</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Team</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Team">Members</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Acknowledgments">Acknowledgments</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Press">Press</A></LI><LI><A href="https://igem.org/Team.cgi?id=1678">Official Team Profile</A></LI></DIV></DIV><DIV id="notebook"><DIV class="contentContainer"><DIV id="banner"><H3>Notebook - Phytase
</H3></DIV></DIV><DIV class="textWrapper"><DIV class="textContent {{{page_option}}}" id="caffeine"><DIV id="notebookMenu"><UL><LI><A href="#august">August</A></LI><LI><A href="#september">September</A></LI></UL></DIV><DIV id="notebookContent"><H1 class="date one">August 10th</H1><H2>Design primers</H2><H3>Gene PHO85</H3>

5’Primer  of Kanamycin resistance gene with tails using to transformation with the PHO85 gene of the yeast.
5’-<SPAN style="color:#9A1717">TATCATTATATATACATGGCTACGGTTTTTCGCTGACGGGCTGCG</SPAN><SPAN style="color:#179A89">ATAATCATTTGCA

TCCATACATTTTGATGGC </SPAN>-3’
3’Primer  of Kanamycin resistance gene with tails using to transformation with the PHO85 gene of the yeast.
3’-<SPAN style="color:#9A1717">AAGGGATATATAGCGCGGCAAACTGGGCAAACTTGAGCAATACCA</SPAN><SPAN style="color:#179A89">CAGCAGTATAG

CGACCAGCATTC</SPAN>-5’
<SPAN style="color:#9A1717"> - Homology tail on gene PHO85</SPAN><SPAN style="color:#179A89"> - Kanamycin resistance binding</SPAN><H3>Gene PHO80</H3>

5’Primer  of Kanamycin resistance gene with tails using to transformation with the PHO80 gene of the yeast.
5’-<SPAN style="color:#9A1717">ATCATAAGACGAGGATATCCTTTGGAGACTCATAGAAATC</SPAN><SPAN style="color:#179A89">ATAATCATTTGCATCCAT

ACATTTTGATGGC</SPAN>-3’
3’Primer  of Kanamycin resistance gene with tails using to transformation with the PHO80 gene of the yeast.
3’-<SPAN style="color:#9A1717">CTCAATCATGATTGCTTTCATAATACCCCACGAAAAATCA</SPAN><SPAN style="color:#179A89">CAGCAGTATAGCGACCA

GCATTC</SPAN>-5’


<SPAN style="color:#9A1717"> - Homology tail on gene PHO80</SPAN><SPAN style="color:#179A89"> - Kanamycin resistance binding</SPAN><H3>Gene FRT + PHO85</H3>

5’Primer  of Kanamycin resistance gene with tails using to transformation with the PHO85 gene of the yeast, including FRT sequence to delete both of PHO80 and PHO85.
5’-<SPAN style="color:#9A1717">TATCATTATATATACATGGCTACGGTTTTTCGCTGACGGGCTGCG</SPAN><SPAN style="color:#36C40F">GAAGTTCCTATTC

TCTAGAAAGTATAGGAACTTC</SPAN><SPAN style="color:#179A89">ATAATCATTTGCATCCATACATTTTGATGGC</SPAN>-3’


3’Primer  of Kanamycin resistance gene with tails using to transformation with the PHO85 gene of the yeast, including FRT sequence to delete both of PHO80 and PHO85.
3’-<SPAN style="color:#9A1717">AAGGGATATATAGCGCGGCAAACTGGGCAAACTTGAGCAATACCA</SPAN><SPAN style="color:#36C40F">CTTCAAGGATAT


GAAAGATCTCTTATCCTTGAAG</SPAN><SPAN style="color:#179A89">CAGCAGTATAGCGACCAGCATTC</SPAN>-5’
<SPAN style="color:#9A1717"> - Homology tail on gene PHO85</SPAN><SPAN style="color:#36C40F"> - FRT sequence</SPAN><SPAN style="color:#179A89"> - Kanamycin resistance binding</SPAN><H1 class="date one">August 12nd</H1><H2>Culture</H2><P>Inoculate 100µL of <I>Saccharomyces cerevisiae SK1</I> on YPD medium overnight (at 30°C). 
This yeast will be transformed.</P><H2>PCR</H2>
3 PCR were realized on HO-Poly-KanMX4-HO plasmid to create a Kanamycin resistance marker, thanks to 3 pairs of primers wich have tails we’ll be use to knock out genes PHO80, PHO85 and both in the yeast.<B>Protocol:</B><DIV class="column-left"><TABLE borercolor="black"><TBODY><TR><TD><B>PHO80</B></TD><TD><B>PHO85</B></TD><TD><B>FRT+ PHO85</B></TD></TR><TR><TD><B>Master mix (µL)</B></TD><TD align="center">50</TD><TD align="center">50</TD><TD align="center">50</TD></TR><TR><TD><B>H2O DNAse Free (µL)</B></TD><TD align="center">45</TD><TD align="center">45</TD><TD align="center">45</TD></TR><TR><TD><B>Resistance plasmid (µL)</B></TD><TD align="center">1</TD><TD align="center">1</TD><TD align="center">1</TD></TR><TR><TD><B>PHO80 5'Primer (µL)</B></TD><TD align="center">2</TD></TR><TR><TD><B>PHO80 3'Primer (µL)</B></TD><TD align="center">2</TD></TR><TR><TD><B>PHO85 5'Primer (µL)</B></TD><TD align="center">2</TD></TR><TR><TD><B>PHO85 3'Primer (µL)</B></TD><TD align="center">2</TD></TR><TR><TD><B>PHO85 + FRT 5'Primer (µL)</B></TD><TD align="center">2</TD></TR><TR><TD><B>PHO85 + FRT 3'Primer (µL)</B></TD><TD align="center">2</TD></TR></TBODY></TABLE></DIV><DIV class="column-right"><P class="legend"><B>Figure 1 :</B> PCR cycle</P></DIV><H1 class="date one">August 13th</H1><H2>PCR Purification</H2><B>Protocol</B> :
<A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR_purification"> PCR purification </A><H2>Result of PCR (August 12th)</H2><DIV class="column-right"><P>The PCR product are then revealed with a<A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/Gel_Electrophoresis"> gel electrophoresis.</A>
We expected bands around 1.300bp.
The band corresponding to marker with FRT is bigger than the two others strips because these have just the Kanamycin resistance with tails, and no FRT sequences.</P></DIV><DIV class="column-left"><P class="legend"><B>Figure 2 :</B>Result of PCR</P></DIV><H2>Pre-culture</H2>
Plated one colony of <I>Saccharomyces cerevisiae SK1</I> in 5mL liquid YPD medium and let's grow overnight.<H1 class="date one">August 14th</H1><H2>Transformation of yeast</H2><B>Protocol</B>:
<A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/Heat_shock_transformation_for_yeast"> Heat shock transformation for yeast </A><H1 class="date one">August 17th</H1><H2>Result of plates:</H2>
There is a culture in plates.<DIV class="column-right"><P>
The negative control is not well. The no change yeast grow in the YPD medium with the antibiotic. We will repeat this control on an agar plate and not in a liquid medium.
We analyze anyway down results, the results of the new control will allow us to validate the result of our experiment or search which are our error and try again.
The positive control is well, yeast multiply on YPD medium plate without antibiotic. Yeasts are not dead, so the culture on other agar mediums are not contamination.
We see more colonies on the plates with yeast transforming PHO85 and FRT+PHO85.
We look only few colonies in the plates with yeast transforming PHO80.
The result is well, transformation works.</P></DIV><DIV class="column-left"><P class="legend"><B>Figure 3 : </B>Negative control</P><P class="legend"><B>Figure 4 : </B>Positive control and result of transformation</P></DIV><H2>Design primers</H2>
We design primers to verifie our results on the plates.
Thanks to the colony PCR, we might determinate if the resistance is integrated into the yeast DNA.

Primer 5'-3' PHO80<SPAN style="color:#179A89">ATCATAAGACGAGGATATCCTTTGGAG</SPAN>
Primer 3'-5' PHO80<SPAN style="color:#179A89">CTCAATCATGATTGCTTTCATAATACCCC</SPAN>
Primer 5'-3' PHO85<SPAN style="color:#179A89">TATCATTATATATACATGGCTACGGTTTTTCG</SPAN>
Primer 3'-5' PHO85<SPAN style="color:#179A89">AAGGGATATATAGCGCGGCAAACTG</SPAN>
Primer 5'-3' FRT+PHO85<SPAN style="color:#179A89">TATCATTATATATACATGGCTACGGTTTTTCG</SPAN>
Primer 3'-5' FRT+PHO85<SPAN style="color:#179A89">AAGGGATATATAGCGCGGCAAACTG</SPAN><H1 class="date one">August 18th</H1><H2>Verification of the new negative control</H2><DIV class="column-right"><P>
The verification of the negative control is good, any colony is watching. We can continue our experiments, it will be validated.
</P></DIV><DIV class="column-left"><P class="legend"><B>Figure 5 : </B>Result of the new negative control
</P></DIV><H2>FRT problems</H2><P>
The transformation with the FRT may be run well, but the plasmid with the gene coding for flippase works only for <I>E. coli</I>. We can't use this plasmid because it will be rejected by the yeast.
Other transformation with CreLox system is possible.
CreLox is a gene which have the same fonction than the FRT, it not cut by the flippase but by the Cre recombinase.
</P>

We design two primers for the new transformation with CreLox system.
 
Primer 5'-3' CreLox + PHO855’-<SPAN style="color:#9A1717">TATCATTATATATACATGGCTACGGTTTTTCGCTGACGGGCTGCG</SPAN><SPAN style="color:#36C40F">ATAACTTCGTATAGCATACATTATACGAAGTTAT</SPAN><SPAN style="color:#179A89">ATAATCATTTGCATCCATACATTTTGATGGC</SPAN>-3’


Primer 3'-5' CreLox + PHO853’-<SPAN style="color:#9A1717">AAGGGATATATAGCGCGGCAAACTGGGCAAACTTGAGCAATACCA</SPAN><SPAN style="color:#36C40F">ATAACTTCGTATAGCATACATTATACGAAGTTAT</SPAN><SPAN style="color:#179A89">CAGCAGTATAGCGACCAGCATTC</SPAN>-5’<SPAN style="color:#9A1717"> - Homology tail on gene PHO85</SPAN><SPAN style="color:#36C40F"> - CreLox sequence</SPAN><SPAN style="color:#179A89"> - Kanamycin resistance binding</SPAN><H1 class="date one">August 19th</H1><H2>Colony PCR</H2><B>Protocol:</B><DIV class="column-left"><TABLE borercolor="black"><TBODY><TR><TD><B>PHO80</B></TD><TD><B>PHO85</B></TD><TD><B>FRT+ PHO85</B></TD></TR><TR><TD><B>dreamTaq 2X (µL)</B></TD><TD align="center">3</TD><TD align="center">3</TD><TD align="center">3</TD></TR><TR><TD><B>H2O DNAse Free (µL)</B></TD><TD align="center">9</TD><TD align="center">9</TD><TD align="center">9</TD></TR><TR><TD><B> Colony </B></TD><TD align="center">1</TD><TD align="center">1</TD><TD align="center">1</TD></TR><TR><TD><B>PHO80 5'Primer (µL)</B></TD><TD align="center">0.5</TD></TR><TR><TD><B>PHO80 3'Primer (µL)</B></TD><TD align="center">0.5</TD></TR><TR><TD><B>PHO85 5'Primer (µL)</B></TD><TD align="center">0.5</TD><TD align="center">0.5</TD></TR><TR><TD><B>PHO85 3'Primer (µL)</B></TD><TD align="center">0.5</TD><TD align="center">0.5</TD></TR></TBODY></TABLE></DIV><DIV class="column-right"><P class="legend"><B>Figure 6 :</B> Colony PCR cycle</P></DIV><H1 class="date one">August 20th</H1><H2>Result of colony PCR (August 19th)</H2><DIV class="column-left"><P>
We see bands smaller than 500bp, but not the fragments we expected : this is probably contaminations. We start again the same PCR colony. There is DNA in holes too, which didn't migrate, probably the genomic DNA.
</P></DIV><DIV class="column-right"><P class="legend"><B>Figure 7 :</B> Electrophoresis PCR colony</P></DIV><H2>Colony PCR</H2>
Same to August 19th.

<H2>Result of second colony PCR</H2><DIV class="column-left"><P class="legend"><B>Figure 8 :</B> second electrophoresis colony PCR</P></DIV><DIV class="column-right"><P>
The DNA Ladder is good but DNA of PCR did not migrate. The ADN is in the holes. We think that the yeasts walls being thicker simple thermic shock does not break it. We will be carry a lysis whith NaOH in yeast, to push out DNA, to the primer can be fixed on it.
</P></DIV><H1 class="date one">August 24th</H1><H2>Yeast lysis with NaOH</H2><B>Protocol:</B><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/Yeast_lysis_with_NaOH"> Yeast lysis with NaOH</A>
After the lysis of yeast we realize the new PCR in normal condition, the same as August 12nd. <H1 class="date one">August 25th</H1><H2>Result of PCR (August 24th)</H2><DIV class="column-left"><P class="legend"><B>Figure 9 :</B> Third electrophoresis colony PCR with NaOH lysis</P></DIV><DIV class="column-right"><P>
The DNA ladder migrate, but there was any amplification of the both genes.
We tested two PCR mix : the first did not work. The positive control worked with the second PCR mix. It is composed of HO plasmid and the oligos using for PHO85 gene.
</P></DIV><H1 class="date one">August 26th</H1><H2>Colony PCR</H2><P>To make the colony PCR, we need to lysis yeasts' wall. We realized the lysis with NaOH, but it did not work.
So we realize a new lysis using the   &quot;<A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/yeast-dna-extraction-dneasy-blood-and-tissue-kit"> DNeasy Blood and Tissue Kit</A>&quot; with the zymolyase enzyme on the non-transformed yeasts to verifie the melting temperature of primers (try between 55°C and 65°C) and if the amplification of the genes work.</P><H2>Phytic acid dosage</H2> 
We dose the phytic acid in the fermented rice with the kit &quot;<A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/titration-acid-phytic"> Phytic Acid (Total Phosphorus) Assay Kit </A>&quot;.





<H1 class="date one">August 27th</H1><H2>Result of PCR (August 26th)</H2><DIV class="column-right"><P class="legend"><B>Figure 10 :</B> Electrophoresis colony PCR with temperature gradient on non-transformed yeasts lysed by the zymolyase</P></DIV><DIV class="column-left"><P>We made a PCR gradient on non-transformed yeast to know exactly wich temperature is better to a good fixation of the primers on the DNA. 
The amplification failed, we supposed it is because our MasterMix did not work. We will try this PCR again.
</P></DIV><H2>Phytic acid dosage</H2><DIV class="column-right"><P>
We dose the phytic acid in fermented rice.
After analysis, the fermented rice have very little quantity of phosphorus and of phytic acid.<B>[Phosphorus] :</B> 0.0758 mg/100g of rice.<B>[Phytic acid] :</B> 0.269 mg/100g of rice.</P></DIV><DIV class="column-left"><P class="legend"><B>Figure 11 :</B> Phosphorus calibration curve for titration of phytic acid in fermented rice</P></DIV><H2>Result of second PCR</H2><DIV class="column-right"><P class="legend"><B>Figure 12 :</B> Second electrophoresis colony PCR with temperature gradient (non-transformed yeast)</P></DIV><DIV class="column-left"><P>We watch bands for the gene PHO80, at the good size : 882bp. But the gene PHO85, there was no amplifiction, and the positive control is negative : we only see aband bigger than 10,000bp and it is not what we expected.We try again this PCR to see if the no amplification of the gene PHO85 it is a manipulation error or not.</P></DIV><H1 class="date one">August 28th</H1><H2>Phytic acid dosage in different strains</H2><P class="legend"><B>Figure 13 :</B> Phosphorus calibration curve + results of OD in some strains</P><P class="legend"><B>Figure 14 :</B> Results of acid phytic dosage on fermented rice</P><H2>Results of PCR</H2><DIV class="column-left"><P class="legend"><B>Figure 15 :</B> Third electrophoresis gradient PCR (non-transformed yeast)</P></DIV><DIV class="column-right"><P>We watch bands for the both of genes, but not at the same temperatures. Thanks to the gradient, we can suppose that oligos have not exactly the same melting temperature, and it may be the reason why the previous PCR failed.
The controle postive with the HO plasmid worked, the band matches with the PHO85 gene size (1020bp).
</P></DIV><DIV class="column-left"><P class="legend"><B>Figure 16 :</B> Electrophoresis PCR transformed yeasts</P></DIV><DIV class="column-right"><P>The positive control is negative, PCR does not therefore our work.
</P></DIV><H2>Transformation</H2>
We transform the yeast <I>Saccharomyces cerevisiae SK1</I> with the resistance to geneticin and the <I>Cre</I> gene. We delete the PHO 85 genes and we replace it by the resistance gene.<DIV class="column-right"><P class="legend"><B>Figure 17 :</B>Principle of Cre recombinase</P></DIV><DIV class="column-left"><P>The <I>Cre</I> gene is the sequence that little be cut thanks Cre recombinase and recombined the gene without the sequence between the Cre sequences.</P></DIV><H1 class="date one">August 30th</H1><H2>Transformation on yeasts with <I>Cre</I> gene</H2>
First step, we realize the PCR of <I>Cre</I> and <I>RFP</I> gene with primers (created by Amaury) and plasmid containing RFP. We amplified the gene, its promoter and its terminater.<B>Protocol</B> :<A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR"> PCR</A>

Second step, we make PCR purification, and check the result with electrophoresis.<B>Protocol</B> :<A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR_purification"> PCR purification</A>
Third step, we transformed yeast that is already integrated the resistance instead of PHO85 gene. We introduce the <I>Cre</I> and <I>RFP</I> genes instead of PHO80 gene<B>Protocol</B> :<A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/Heat_shock_transformation_for_yeast"> Transformation of yeast</A>
Fourth step, we cultived the transforming yeasts in YPD agar medium with geneticin overnight.
<H2>PCR and result</H2><DIV class="column-left">We did electrophoresis of transformation product. It's revealed bands around 10,000bp, while we expected 1,000bp. We think the DNA extraction in yeast doesn't work, and these bands are contamination.</DIV><DIV class="column-right"><P class="legend"><B>Figure 18 :</B> Colony PCR on transformed yeast with RFP</P></DIV><H1 class="date two">September 1st</H1><H2>Result of transformation</H2>
We watching any culture on plates YPD + Geneticin. The yeast are not resistant to geneticin, they had not introduced the resistance gene. Transformation does not work.


<H1 class="date two">September 2nd</H1><H2> Transformation</H2>
We start again the transformation of august 28th, with the same protocol.

<H2>Titration of phytic acid</H2>
We test the quantity of phytic acid different strains, at dilution 1/5, 1/50 and 1/500.
Strains : g15,37 / g15,55 / g15,56 / g15,57 / g15,58 / Sook dall + HCl / Sook rice + HCl.



<H1 class="date two">September 4th</H1>

After the transformation of September 2nd, any culture is observable on plates. The tansformation are not function. We restart transformation, but we transform with <I>Cre</I> and <I>RFP</I> genes. It is more interesting because we not forced to simply remove the resistance gene, we can make a double identification. It helps to identify yeast that is already integrated in place of the resistance gene PHO85 (they cultivate on agar with geneticin) and we also integrate the RFP in place of PHO80 gene (they will be red).





<H1 class="date two">September 8th</H1><H2>PCR to product cassette Cre/RFP</H2>

We restart PCR with Cre sequence and RFP gene.<DIV class="column-left"><P class="legend"><B>Figure 19 : </B>Electrophoresis of Cre and RFP amplification</P></DIV><DIV class="column-right"><P>We see only one band. We can continue experiments with this sample and realize the transformation.</P></DIV><H2> Analysis of results of titration phytic acid of September 2nd</H2>  
We test the quantity of phytic acid different strains, at dilution 1/5, 1/50 and 1/500.
Strains : g15,37 / g15,55 / g15,56 / g15,57 / g15,58 / Sook dall + HCl / Sook rice + HCl.

We start to do the phosphorus scale.<P class="legend"><B>Figure 20 :</B> Phosphorus calibration curve</P>
With calculations of the protocol, we can determine the concentration in our samples.<B>Protocol :</B><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols#FSTitrationacidphytic"> Titration of phytic acid</A><P class="legend"><B>Figure 21 :</B> Table of results</P>
Concentrations are very low. There are less to 1mg/100g d’idli. 




<H1 class="date two">September 11th</H1><H2>Transformation</H2> 
We did the transformation on strain <I>Saccharomyces cerevisiae SK1</I>, using the protocol of<A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols#HeatShockYeast"> heat shock yeast transformation</A>.





<H1 class="date two">September 14th</H1><H2>Genomic DNA extraction of transformed yeast</H2>
We try to extract genomic DNA of transformed yeast with the <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols#YeastDNAextractiondneasybloodandtissuekit"> &quot;Blood &amp; Tissue extraction Kit&quot;</A>
We make PCR on it.

<H2>Result of PCR</H2>
The gel electrophoresis revealed any bands. We might suppose that the transformation did not work, but it's probably that we don't succeed to extract the yeast's DNA which prevent us to verify our results.





<H1 class="date two">September 15th</H1><H2>Plate of yeast transformed with RFP</H2><DIV class="column-left"><P class="legend"><B>Figure 22 :</B> Transformed <I>S. cerevisiae SK1</I> on YPD agar + geneticin</P></DIV><DIV class="column-right">Colonies are not isolated, we cannot distinguish all of them. The culture that was inoculate was too concentrate.
We make suspensions low concentrate and inoculate again on YPD medium with geneticin at 200µg/mL.
</DIV><H1 class="date two">September 16th</H1><H2>Plate of yeast transformed with RFP</H2><DIV class="column-left"><P class="legend"><B>Figure 23 :</B> Transformed <I>S. cerevisiae SK1</I> on YPD agar + geneticin</P></DIV><DIV class="column-right">All colonies are the same, but there is no red coloration.
These result can be explicated by the fact that when this gene is introduced in the genomic DNA, it's not revealed, whereas it is in a plasmid.
</DIV><H2>Test new transformed yeast</H2>
We test our new transformed yeast which deletion of PHO80 and PHO85 genes and compare with yeast transformed previously (first with the geneticin resistance gene in place PHO85 gene and second with the  geneticin resistance gene in place PHO80). We put this yeast in YPD + geneticin medium (YPD + geneticinn for always keep the selection). We have grown the yeast with six concentration of phytic acid (0.005, 0.01, 0.02, 0.04, 0.1, 0.2 g/mL). We also realize a scale with phytic acid and YPD medium + geneticin. 
Scale: C(phytic acid;YPD) (g.mL-1)	0,005	0,01	0,02	0,04	0,1 <P class="legend"><B>Figure 24 :</B>Table manipulation</P>

We don’t have enough reagents in our kit to do all manipulations we want, and it is too late to buy it. We choose to delete three concentrations by strain and in the scale. <P class="legend"><B>Figure 25 :</B>New table manipulation</P><P class="legend"><B>Figure 26 :</B>Result of test</P>

Result analysis: We have only optic density result below zero. Normally is no possible. We suppose the reagents of the kit are old and it not work. Would have had to do again the tests but there is not reagents and time anymore. </DIV></DIV></DIV></DIV></BODY></HTML>