<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Notebook_Differentiation skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Paris Bettencourt/Notebook/Differentiation</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="menuContainer"><DIV id="smallMenuTitle"><H1>Ferment It Yourself</H1><H3>iGEM Paris-Bettencourt 2O15</H3></DIV><UL><LI class="left"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Background" title="Background">Background</A></LI><LI class="left"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Design" title="Design">Design</A></LI><LI class="left"><BUTTON class="menuButton">Project</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Micronutrients">Micronutrients</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Continuity">Continuity</A></LI><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Manufacturing">Manufacturing</A></LI></DIV></UL><LI class="left"><BUTTON class="menuButton">Achievements</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Parts">Parts</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Medals">Medal Fulfillments</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Practices</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/SynBio_Challenge">SynBio Challenge</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/DILambda">DIλ</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Acceptance">Acceptance</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/Events">Events</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/Rhizi">Rhizi</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Collaborations">Collaborations</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Safety</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Safety">Safety Overview</A></LI><LI><A href="https://2015.igem.org/Safety/Final_Safety_Form?team_id=1678">Safety Form</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/ReviewBoard">Review Board</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Notebook</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Notebook">Daily notebook</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols">Protocols</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Bibliography">Bibliography</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Team</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Team">Members</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Acknowledgments">Acknowledgments</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Press">Press</A></LI><LI><A href="https://igem.org/Team.cgi?id=1678">Official Team Profile</A></LI></DIV></DIV><DIV id="notebook"><DIV class="contentContainer"><DIV id="banner"><H3>Notebook - Differentiation
</H3></DIV></DIV><DIV class="textWrapper"><DIV class="textContent {{{page_option}}}" id="caffeine"><DIV id="notebookMenu"><UL><LI><A href="#july">July</A></LI><LI><A href="#august">August</A></LI><LI><A href="#september">September</A></LI></UL></DIV><DIV id="notebookContent"><H2 class="date" id="advancement-on-e.-coli">Advancement on <EM>E. coli</EM></H2><P>The whole brainbow system will be integrated chromosomally at the GalK site. It will be synthesized as three gBlocks.</P><P>It should allow expression of mCherry first, then upon expression of the CRE recombinase, differentiation in two states (YFP and CFP).</P><P>It also contains a landing pad that allow insertion of a third state, which would subsequently lower the probability of states 1 and 2 so the three outcomes are possible.</P><H3 id="writing-the-artificial-gene">Writing the artificial gene</H3><P>Promoter J23199 from the Biobrick collection</P><P>4 LoxP sites used together in mammalian brainbow plasmids</P><P>RBS from Ihab</P><P>ORF for mCerulean and mVenus from Ihab</P><P>ORF for mCherry from Antoine</P><P>rrnBT1 terminator used on the pOSIP plasmids</P><P>Landing Pad (PhiC31 attB TT site)</P><P>Check for secondary structure in the RBS Done</P><P>Check for RBS in the LoxP array Done</P><P>Split to have two ~1500 bp gblocks Done</P><P>Design overlaps for Gibson in R6K vector Done</P><P>Check for Biobrick restriction sites Done</P><P>Fix gBlocks problems</P><UL><LI>Palindroms between LoxP sites Done</LI><LI>Repeats in terminator: replaced with Lambda T0 terminator Done</LI><LI>Repeat in RBS Done</LI><LI>More GC in the LoxP array Done</LI><LI>More GC at the end of fragment 1Done</LI><LI>Repeat at the end of mVenus and mCerulean Done</LI><LI>More GC at the beginning of fragment 2 Done</LI></UL><P>It is very difficult to solve these -&gt; make 3 fragments Done</P><P>Fragment A: 0 - 1143</P><P>Fragment B: 1105 - 2006</P><P>Fragment C: 1981 - 2946</P><P>Change the RBS for Fragment A -&gt; “New RBS” Done</P><P>Order gBlocks Done 07/13</P><H3 id="design-order-oligos-for-gblocks-amplification-done">Design + order oligos for gBlocks amplification Done</H3><P>Overlaps melting points are 62, 67, 54, 74.</P><P>The overlap between fragments B and C should be increased to &gt;62.</P><TABLE><TBODY><TR class="odd"><TD align="left"><P>R6K LinR</P></TD><TD align="left"><P>tagcattatacctaggactgagctagctgtcaaggcaaatttgcggccgcaag</P></TD><TD align="left"><P>o15.80</P></TD></TR><TR class="even"><TD align="left"><P>R6K LinF</P></TD><TD align="left"><P>CGGGCGCGTACTCCAgaagggcatcgatggc</P></TD><TD align="left"><P>o15.81</P></TD></TR><TR class="odd"><TD align="left"><P>Colibow A F</P></TD><TD align="left"><P>ttgacagctagctcagtcctag</P></TD><TD align="left"><P>o15.82</P></TD></TR><TR class="even"><TD align="left"><P>Colibow A R</P></TD><TD align="left"><P>GGCCATTCACATCACCATC</P></TD><TD align="left"><P>o15.83</P></TD></TR><TR class="odd"><TD align="left"><P>Colibow B F</P></TD><TD align="left"><P>GCCGATTCTTGTTGAACTTG</P></TD><TD align="left"><P>o15.84</P></TD></TR><TR class="even"><TD align="left"><P>Colibow B R</P></TD><TD align="left"><P>CCATGGTACCTCCTCCTTACTTCTATAACTTC</P></TD><TD align="left"><P>o15.85</P></TD></TR><TR class="odd"><TD align="left"><P>Colibow C F</P></TD><TD align="left"><P>GATACTTTATACGAAGTTATAGAAGTAAGGAGGAG</P></TD><TD align="left"><P>o15.86</P></TD></TR><TR class="even"><TD align="left"><P>Colibow C R</P></TD><TD align="left"><P>gccatcgatgcccttcTGGAGTACGCGCCCG</P></TD><TD align="left"><P>o15.87</P></TD></TR></TBODY></TABLE><P>Overlap melting points: 62, 67, 62, 72.</P><H3 id="design-order-oligos-for-cassette-sequencing-done">Design + order oligos for cassette sequencing Done</H3><P>&gt;50 bp before the interest region</P><P>800 bp max contigs</P><P>Obtain Pir+ strain Done</P><P>Overnight of Pir+ strain Done</P><P>Glycerol of Pir+ strain Done</P><H2 class="date" id="cloning-inside-the-replication-vector">Cloning inside the replication vector</H2><P>Order oligos linR and linF Done</P><P>Obtain R6K vector from Ihab Done</P><P>Overnight culture of the R6K vector propagation strain</P><P>        Failed New attempt with less harsh growth conditions. Done</P><UL><LI>Only 20 ug/ml of Kanamycine</LI><LI>6 ul of Thyamine in 3 ul of LB</LI><LI>Culture at 30°C</LI></UL><P>Miniprep of the R6K vector Done</P><P>Glycerol of R6K strain Done</P><P>Linearization of the R6K vector by PCR Done</P><P>Primer linF:</P><P>CCCTTGGGCTCCCCGGGCGCGTACTCCAgaagggcatcgatggc</P><P>(28 bases: longest possible overlap without having a hairpin at 50°)</P><P>Tm = 55°</P><P>Primer linR:</P><P>tagcattatacctaggactgagctagctgtcaaggcaaatttgcggccgcaag</P><P>(33 bases overlap)</P><P>Tm = 62°</P><P>Product length: 2276 bp</P><P>Synthetize, reconstitute and dilute primers Done</P><P>Run gel for size checking Done</P><P>PCR purification Done</P><P>Obtain gBlocks Done</P><P>PCR of colibow fragments A, B and C Done</P><P>PCR purification of amplicon Done</P><P>Obtain Gibson mix Done</P><P>Make overnight culture of Pir+ strain Done</P><P>Make electrocompetent cells out of the Pir+ strain Done</P><P>Gibson assembly of 3 gblocks with the R6K backbone. Done</P><P>The vector can only replicate in Pir+ strain. Transform into Pir+ strain Got colonies</P><P>Check transformant: colony PCR before culture Failed</P><P>Liquid culture + miniprep</P><P>Analytical digestion or sequencing (find enzymes)</P><TABLE><TBODY><TR class="odd"><TD align="left"><P>SeqF1</P></TD><TD align="left"><P>o15.88</P></TD><TD align="left"><P>cttagtacgttagccatgagg</P></TD></TR><TR class="even"><TD align="left"><P>SeqF2</P></TD><TD align="left"><P>o15.89</P></TD><TD align="left"><P>CTAATTTTCCATCTGATGGCC</P></TD></TR><TR class="odd"><TD align="left"><P>SeqF3</P></TD><TD align="left"><P>o15.90</P></TD><TD align="left"><P>CAAGCTCACGCTCAAATTC</P></TD></TR><TR class="even"><TD align="left"><P>SeqF4</P></TD><TD align="left"><P>o15.91</P></TD><TD align="left"><P>ACAACCATTACCTGTCGACG</P></TD></TR><TR class="odd"><TD align="left"><P>SeqF5</P></TD><TD align="left"><P>o15.92</P></TD><TD align="left"><P>CGACATTAGGGTATGGGCTG</P></TD></TR><TR class="even"><TD align="left"><P>SeqR1</P></TD><TD align="left"><P>o15.93</P></TD><TD align="left"><P>TATAAACATTATGGCTATTATAG</P></TD></TR><TR class="odd"><TD align="left"><P>SeqR2</P></TD><TD align="left"><P>o15.94</P></TD><TD align="left"><P>TTTAGAGAGTTTTGACTGCG</P></TD></TR><TR class="even"><TD align="left"><P>SeqR3</P></TD><TD align="left"><P>o15.95</P></TD><TD align="left"><P>TTTGCCAGTCGTACAGATGAA</P></TD></TR><TR class="odd"><TD align="left"><P>SeqR4</P></TD><TD align="left"><P>o15.96</P></TD><TD align="left"><P>TCTTCTTCTGCATCACCGGGC</P></TD></TR></TBODY></TABLE><H2 class="date" id="chromosomal-integration-with-galk">Chromosomal integration with GalK</H2><P>PCR of the purified R6K plasmid. Done</P><P>PCR-linerization of the R6K backbone</P><UL><LI>Miniprep du backbone R6K (20 ng/ul) -&gt; 5 ul in PCR</LI><LI>Primers: 15.80 (LinR) et 15.81(LinF)  -&gt; 1 ul each</LI><LI>Eau qsp 25 ul</LI><LI>Master mix 25 ul</LI></UL><P>1’30’’ extension, 50°C annealing.</P><P>Find oligos (already there) Done</P><P>IntR:</P><P>GTTTGCGCGCAGTCAGCGATATCCATTTTCGCGAATCCGGAGTGTAAGAACCATATGAATATCCTCCTTAGTTCCTATTCCG</P><P>Alternatively, with only one binding site:</P><P>GTTTGCGCGCAGTCAGCGATATCCATTTTCGCGAATCCGGAGTGTAAGAAttagccatggtccatatgaatatcctccttag</P><P>IntF:</P><P>TTCATATTGTTCAGCGACAGCTTGCTGTACGGCAGGCACCAGCTCTTCCGGGTTGAACTGCGGATCTTGCGGC</P><P>Italique: Homology region with E. coli GalK site.</P><P>Length: 4908 bp</P><P>Attention, autre site potentiel d’amorçage, produit de 3515 bp</P><P>Purification</P><P>Overnight culture de E. coli Lambda Red</P><P>Competent cells out of Lambda Red strain</P><P>Transformation of E. coli with pKD46 that carries Lambda Red recombinase</P><P>Transformation of the Colibow PCR product</P><H2 class="date" id="function-testing">Function testing</H2><P>Tecan + Flux cytometry</P><H2 class="date" id="cre-recombinase-expression">CRE recombinase expression</H2><P>pFHC2938 should work.</P><P>It expresses CRE and has a temperature sensitive ORI (30°C)</P><H3 id="monday-0706">Monday 07/06</H3><H4 id="research-about-the-synthetic-integron">Research about the synthetic integron</H4><P>It might make the landing pads better than with brainbow because the recombination site is always the same.</P><H4 id="plate-culture-of-e.-coli-pit5-kl-and-pit5-kh">Plate culture of E. coli pIT5-KL and pIT5-KH</H4><P>Very important: grow @30</P><P>-&gt; Make a liquid culture of each and freeze at -80</P><P>-&gt; Miniprep from pIT5-KL for first construct</P><P>These two strains produce the vector plasmids for clonetegration. They have to be linearized with EcoR1 and Pst1, and then Gibson assembled to get the self-integrating plasmids.</P><P>The integrase they carry is expressed only at 37 degrees.</P><P>They are resistant to Kanamycine.</P><P>KH integrates into the HK022 phage’s site, while KL integrates into the Lambda phage site. We will probably only need one of them but I took both just in case. If needed there is a whole collection of such vectors.</P><H4 id="plate-culture-of-e.-coli-pe-flp">Plate culture of E. coli pE-FLP</H4><P>Grow @ 30</P><P>This strain produces the pE-FLP plasmid, which expresses the flippase. It will be useful to remove the backbone after the clonetegration. It’s resistant to ampicillin.</P><P>It has a temperature-sensitive ORI and will disappear if grown at 37.</P><H4 id="new-primers">New primers</H4><P>LC81, LC85, LC82, LC86: used in the PCR to check integration with pIT5 KH.</P><P>LC81, LC87, LC83, LC84: used in the PCR to check integration with pIT5 KL.</P><H2 class="date" id="thursday-0709">Thursday 07/09</H2><UL><LI>Prepared 10 tubes of 1 ml Kanamycine, 50 mg/ml</LI><LI>Inoculated pIT5-KL and pIT5-KH E. coli in 2 ml LB Kan+50</LI><LI>Inoculated pE-FLP E. coli in 2ml LB Amp+100</LI><LI>Cultured these three tubes @ 30</LI></UL><H2 class="date" id="monday-0720">Monday 07/20</H2><H3 id="reception-of-two-plates-from-ihab">Reception of two plates from Ihab</H3><UL><LI>Shortened R6K vector -&gt; Kan+ and thyamine auxotrophy. Thyamine 1000x is available in an eppendorf tube.</LI><LI>Pir 116 strain for propagation of the vector.</LI></UL><P>-&gt; Overnight culture for miniprep for PCR and gleezing</P><UL><LI>R6K in LB Kan Thyamine (in antibiotics box)</LI><LI>Pir116 in LB with a control tube</LI></UL><H3 id="started-cultures-for-colibow">Started cultures for Colibow</H3><P>pThy 1.0 aka pBrainbow 1.0 in 3 ml LB-Amp -&gt; Miniprepcr</P><P>pR6K in LB-Kan-Thyamine -&gt; Miniprepcr and Glycerol freezing</P><P>Pir116 for replication of pR6K-vectors -&gt; LB</P><P>Control without innoculation -&gt; LB</P><P>All of them, culture @ 37° overnight.</P><P>No need to culture pKT174 because we already have the purified version. It may be a good idea to transform DH5a with it in case we run out of plasmid.</P><H3 id="reception-of-oligoso15.76-to-o15.96">Reception of oligos o15.76 to o15.96</H3><UL><LI>Yeastbow SOE</LI><LI>R6K linearization</LI><LI>Colibow gBlock Amp</LI><LI>Colibow Sequencing</LI></UL><P>The ultramers (o15.78, o15.79) for Leu2 flanking are not ready yet.</P><P>-&gt; Oligo reconstitution and dilution</P><P>-&gt; Miniprep of R6K vector</P><P>-&gt; PCR of R6K vector</P><P>-&gt; Miniprep of pBrainbow 1.0</P><P>-&gt; PCR of pBrainbow 1.0</P><P>-&gt; PCR of pThy Ura3</P><P>Reconstituted all primers to 100 ug/ml.</P><H3 id="miniprep-of-pr6k-and-pbrainbow-1.0">Miniprep of pR6K and pBrainbow 1.0</H3><P>For PCR, using Promega kit w/ double wash. Elution in 30 ul</P><P>Final concentration: 93 ng/ul</P><H3 id="re-start-of-the-r6k-culture">Re-start of the R6K culture</H3><P>The first culture failed: let’s try again with less harsh conditions.</P><UL><LI>Only 20 ug/ml of Kanamycine</LI><LI>6 ul of Thyamine in 3 ul of LB</LI><LI>Culture at 30°C</LI></UL><H2 class="date" id="wednesday-0722">Wednesday 07/22</H2><H3 id="pcr-linerization-of-the-r6k-backbone">PCR-linerization of the R6K backbone</H3><UL><LI>Miniprep du backbone R6K (20 ng/ul) -&gt; 5 ul in PCR</LI><LI>Primers: 15.80 (LinR) et 15.81(LinF)  -&gt; 1 ul each</LI><LI>Eau qsp 25 ul</LI><LI>Master mix 25 ul</LI></UL><P>1’30’’ extension, 50°C annealing -&gt; 2276 bp</P><H3 id="bad-news-from-idt-about-the-gene-synthesis">Bad news from IDT about the gene synthesis</H3><P>« I would like to notify you about a best effort for gBlock ‘Colibow C’. Even after multiple attempts at trying to correct this issue, the Final QC data for the best prep is not meeting our quality standards. The final prep (mfg# 188635671) has desired peak.  A secondary peak is also present. The target mass was found in the chromatogram, but not as the most abundant product in any of the chromatographic peaks for mass spectrometry, and the sequence has been verified.</P><P>If you are using this sequence for cloning and/or qPCR you should not have any problems with this sequence. It may require you to screen additional colonies. However, if your intentions with using this gBlock was for other purposes, we might need to cancel the gBlock and redesign. I have provided the trace below. »</P><H2 class="date" id="thursday-0723">Thursday, 07/23</H2><H3 id="glycerol-stock-for-pr6k">Glycerol stock for pR6K</H3><P>1 ml of overnight culture of E. coli pR6K.</P><P>1 ml of glycerol.</P><P>-&gt; -20°C</P><H3 id="gel-for-linearized-r6k-pcr">Gel for linearized R6K pcr</H3><P>1% agar TAE, with 1 kb+ generuler.</P><P>5 ul PCR product + 1 ul LB.</P><P>-&gt; band at the right size (2276 bp) , the PCR worked.</P><H3 id="linearized-r6k-pcr-cleanup">linearized R6K pcr cleanup</H3><P>Using the Qiagen kit, taken back in 50 ul water</P><P>Titration: about 80 ng/ul, but very bad 260/230 ratio (presence of organic molecules).</P><H3 id="pir116-electrocompetent-for-bowcoli-transformation">Pir116 electrocompetent for bowcoli transformation</H3><P>2 ml overnight culture in 100 ml LB, at 37°C w/ shaking.</P><P>When OD reaches 0.6, they were put in ice for half an hour.</P><P>Electrocompetent cells were made by Mukit along with DH5a competent cells.</P><H2 class="date" id="tuesday-0728">Tuesday, 07/28</H2><H3 id="reception-of-gblocks-colibow-a-b-c">Reception of gBlocks Colibow A, B, C</H3><P>Reconstitution to the concentration of 10 ng/ul (from spec sheet):</P><UL><LI>Centrifuge @ 11kG</LI></UL><UL><LI>Add 100 ul of RNase-free water</LI><LI>Vortex</LI><LI>Incubate at 50°C for 20 minutes</LI><LI>Vortex/Centrifuge</LI></UL><H3 id="pcr-of-colibow-gblocks">PCR of Colibow gBlocks</H3><P>Colibow A</P><P>Primers: 82 (56°) + 83 (54°) -&gt; 1172 bp</P><P>Colibow B</P><P>Primers: 84 (53°) + 85 (54°) -&gt; 909 bp</P><P>Colibow C</P><P>Primers: 86 (54°) + 87 (57°) -&gt; 992 bp</P><P>Reaction in 50 ul:</P><TABLE><TBODY><TR class="odd"><TD align="left"><P>Compound</P></TD><TD align="left"><P>Volume (ul)</P></TD></TR><TR class="even"><TD align="left"><P>Water</P></TD><TD align="left"><P>19</P></TD></TR><TR class="odd"><TD align="left"><P>Phusion 2x</P></TD><TD align="left"><P>25</P></TD></TR><TR class="even"><TD align="left"><P>Primer 1</P></TD><TD align="left"><P>2.5</P></TD></TR><TR class="odd"><TD align="left"><P>Primer 2</P></TD><TD align="left"><P>2.5</P></TD></TR><TR class="even"><TD align="left"><P>gBlock diluted 10 times</P></TD><TD align="left"><P>1 ul</P></TD></TR></TBODY></TABLE><P>Program:</P><P>98 (30)</P><P>98 (10) 58 (25) 72 (45) x35</P><P>72 (600)</P><P>12 (hold)</P><P>In parallel from running the gel, the PCR products were purified with the QIAGEN kit and eluted in 40 ul of water.</P><P>Titration</P><P>Fragment A: 90 ng/ul</P><P>Fragment B: 88 ng/ul</P><P>Fragment C: 85 ng/ul</P><P>Gel plan</P><P>1% agar, SYBRsafe, 5 ul of PCR product + 1 ul of LD.</P><P>The 100 bp+ marker was used.</P><P>The apparatus didn’t work. It stopped two times, the gel stayed to diffuse in the tank for 30 minutes. At the end it was imaged even though it was far from finished.</P><TABLE><TBODY><TR class="odd"><TD align="left"><P>Ladder 100 bp+</P></TD><TD align="left"><P>Colibow A</P></TD><TD align="left"><P>Colibow B</P></TD><TD align="left"><P>Colibow C</P></TD></TR><TR class="even"><TD align="left"><P>Expected size</P></TD><TD align="left"><P>1172</P></TD><TD align="left"><P>909</P></TD><TD align="left"><P>992</P></TD></TR></TBODY></TABLE><P>Conclusion</P><P>There are a lot of non-specific binding.</P><P>Due to the awful quality of the gel, it’s not possible to determine their size or which bands are the good ones. The top ones seem more consistent regarding their relative sizes.</P><P>To do:</P><OL><LI>Run the whole purified PCR product in a gel and perform a gel extraction. Hopefully the right band can be determined this time.</LI><LI>Try the PCR again with a higher annealing temperature.</LI><LI>Gibson assemble directly the gBlocks fragments.</LI></OL><H2 class="date" id="wednesday-0729">Wednesday, 07/29</H2><H3 id="gel-for-colibow-a-b-c-and-extraction">Gel for Colibow A, B, C and extraction</H3><P>Agar 1%, SYBRsafe, TAE</P><TABLE><TBODY><TR class="odd"><TD align="left"><P>Ladder 100 bp+</P></TD><TD align="left"><P>Colibow A</P></TD><TD align="left"><P>Colibow B</P></TD><TD align="left"><P>Colibow C</P></TD></TR><TR class="even"><TD align="left"><P>Expected size</P></TD><TD align="left"><P>1172</P></TD><TD align="left"><P>909</P></TD><TD align="left"><P>992</P></TD></TR></TBODY></TABLE><DIV class="figure"><P class="caption">Colibow Amp Extracted.jpg</P></DIV><P>For each sample, the top band was extracted. The Colibow B sample was extracted twice: Bg (grande, top band) and Bp (petite, bottom band).</P><P>The bottom band is the right one.</P><P>-&gt; add DMSO 3% next time</P><P>Titration (in 30 ul EB)</P><TABLE><TBODY><TR class="odd"><TD align="left"><P>Name</P></TD><TD align="left"><P>1</P></TD><TD align="left"><P>U</P></TD><TD align="left"><P>A</P></TD><TD align="left"><P>Bp</P></TD><TD align="left"><P>Bg</P></TD><TD align="left"><P>C</P></TD></TR><TR class="even"><TD align="left"><P>C (ng/ul)</P></TD><TD align="left"><P>13</P></TD><TD align="left"><P>41</P></TD><TD align="left"><P>6</P></TD><TD align="left"><P>29</P></TD><TD align="left"><P>11</P></TD><TD align="left"><P>12.1</P></TD></TR></TBODY></TABLE><P>The concentrations of Colibow A and C are critically low. It is necessary to fix the PCR and reduce the non-specific priming before performing the Gibson.</P><H3 id="new-attempt-at-colibow-a-and-c-pcrs">New attempt at Colibow A and C PCRs</H3><P>Mix</P><P>Phusion        25</P><P>Colibow A        1</P><P>o15.82                2.5</P><P>o15.83                2.5</P><P>Water                19</P><P>DMSO                1.5         (3%)</P><P>= Two tubes of 50 ul each -&gt; 1172 bp</P><P>Mix</P><P>Phusion        25</P><P>Colibow C        1</P><P>o15.86                2.5</P><P>o15.87                2.5</P><P>Water                19</P><P>DMSO                1.5         (3%)</P><P>= Two tubes of 50 ul each -&gt; 992 bp</P><P>This is stored in 4° for now (29/07)</P><H3 id="received-gibson-assembly-mixes-from-ihab">Received Gibson assembly mixes from Ihab</H3><P>I need better quality products before doing it.</P><H2 class="date" id="thursday-0730">Thursday, 07/30</H2><H3 id="gel-for-a-c-and-1">Gel for A, C and 1</H3><UL><LI>Sophie’s sample</LI></UL><TABLE><TBODY><TR class="odd"><TD align="left"><P>A</P></TD><TD align="left"><P>A</P></TD><TD align="left"><P>A</P></TD><TD align="left"><P>C</P></TD><TD align="left"><P>C</P></TD><TD align="left"><P>C</P></TD><TD align="left"><P>100+</P></TD><TD align="left"><P>1</P></TD><TD align="left"><P>1</P></TD><TD align="left"><P>1</P></TD><TD align="left"><P>1kb</P></TD></TR><TR class="even"><TD align="left"><P>1172</P></TD><TD align="left"><P>1172</P></TD><TD align="left"><P>1172</P></TD><TD align="left"><P>992</P></TD><TD align="left"><P>992</P></TD><TD align="left"><P>992</P></TD><TD align="left"><P>3965</P></TD><TD align="left"><P>3965</P></TD><TD align="left"><P>3965</P></TD></TR></TBODY></TABLE><P>50 ul sample + 10 ul LD -&gt; 40 ul in each well (3 wells)</P><P>Attention C sample accidentally added to the well containing 100 bp + ladder !!!</P><DIV class="figure"><P class="caption">Colibow A et C amp 29.07.jpg</P></DIV><P>Gel extraction</P><P>With Qiagen kit, product recovered in 30 ul of water.</P><P>Name: “product” X+ 30/07</P><P>The C product mixed with ladder was labeled Cl.</P><P>Titration</P><TABLE><TBODY><TR class="odd"><TD align="left"><P>Sample</P></TD><TD align="left"><P>1</P></TD><TD align="left"><P>A</P></TD><TD align="left"><P>C</P></TD><TD align="left"><P>Cl</P></TD></TR><TR class="even"><TD align="left"><P>c (ng/ul)</P></TD><TD align="left"><P>5.2</P></TD><TD align="left"><P>15.4</P></TD><TD align="left"><P>12.9</P></TD><TD align="left"><P>4.4</P></TD></TR></TBODY></TABLE><H3 id="gibson-assemblies-of-colibow">Gibson assemblies of Colibow</H3><P>General mix:</P><P>15 ul of Gibson supermix (MMII)</P><P>10-100 ng of backbone for a 6 kb fragment</P><P>Equimolar amount of DNA fragments</P><P>Total: 20 ul</P><P>Colibow PCR products</P><P>Using the most concentrated PCR products known to date.</P><TABLE><TBODY><TR class="odd"><TD align="left"><P>Nom</P></TD><TD align="left"><P>Taille (bp)</P></TD><TD align="left"><P>Concentration</P></TD><TD align="left"><P>Volume (ul)</P></TD><TD align="left"><P>Quantité (ng)</P></TD></TR><TR class="even"><TD align="left"><P>R6K pcr</P></TD><TD align="left"><P>2276</P></TD><TD align="left"><P>80</P></TD><TD align="left"><P>0.6</P></TD><TD align="left"><P>50</P></TD></TR><TR class="odd"><TD align="left"><P>Colibow A+ e (in water)</P></TD><TD align="left"><P>1172</P></TD><TD align="left"><P>15</P></TD><TD align="left"><P>1.7</P></TD><TD align="left"><P>25</P></TD></TR><TR class="even"><TD align="left"><P>Colibow Bp</P><P>(in EB)</P></TD><TD align="left"><P>909</P></TD><TD align="left"><P>29</P></TD><TD align="left"><P>0.9</P></TD><TD align="left"><P>25</P></TD></TR><TR class="odd"><TD align="left"><P>Colibow C+ e</P><P>(in water)</P></TD><TD align="left"><P>992</P></TD><TD align="left"><P>13</P></TD><TD align="left"><P>1.6</P></TD><TD align="left"><P>25</P></TD></TR></TBODY></TABLE><P>Colibow gBlocks</P><P>Using directly the gBlocks from IDT dna synthesis</P><TABLE><TBODY><TR class="odd"><TD align="left"><P>Nom</P></TD><TD align="left"><P>Taille (bp)</P></TD><TD align="left"><P>Concentration</P></TD><TD align="left"><P>Volume (ul)</P></TD><TD align="left"><P>Quantité (ng)</P></TD></TR><TR class="even"><TD align="left"><P>R6K pcr</P></TD><TD align="left"><P>2276</P></TD><TD align="left"><P>80</P></TD><TD align="left"><P>0.5</P></TD><TD align="left"><P>30</P></TD></TR><TR class="odd"><TD align="left"><P>Colibow A</P></TD><TD align="left"><P>1172</P></TD><TD align="left"><P>10</P></TD><TD align="left"><P>1.5</P></TD><TD align="left"><P>15</P></TD></TR><TR class="even"><TD align="left"><P>Colibow B</P></TD><TD align="left"><P>909</P></TD><TD align="left"><P>10</P></TD><TD align="left"><P>1.5</P></TD><TD align="left"><P>15</P></TD></TR><TR class="odd"><TD align="left"><P>Colibow C</P></TD><TD align="left"><P>992</P></TD><TD align="left"><P>10</P></TD><TD align="left"><P>1.5</P></TD><TD align="left"><P>15</P></TD></TR></TBODY></TABLE><P>Incubation during one hour at 50°C.</P><H3 id="lb-ampicillin-plates">3 LB ampicillin plates</H3><P>“30/07 ANTOINE”</P><P>50 ml of hot LB-agar.</P><H3 id="transformation-of-colibow-in-pir116-by-electroporation">Transformation of Colibow in Pir116 by Electroporation</H3><P>pColibow gBlock</P><P>pColibow PCR</P><P>Biobrick plasmid with RFP (3 ul of 10 pg tube, not dialysed)</P><P>Protocol from OWW:</P><UL><LI>Thaw cells from -80°C to ice for &gt;20 min</LI><LI>Chill cuvettes</LI><LI>Dialyse 6 ul of Gibson products on dWater during &gt;20 min</LI><LI>Mix 6 ul of dialysed DNA with 50 ul Pir116 electrocompetent cells</LI><LI>Mix with tip</LI><LI>Pulse (1.5 kV, 2 mm cuvette)</LI><LI>Add 1 ml LB (at 18h43)</LI></UL><P>Incubate for 1 hour at 37°C.</P><P>2 Kanamycine 25 plates for each colibow plasmid: 100 ul and 900 ul</P><P>1 Chloramphenicol plate for RFP control</P><P>Incubation overnight @37.</P><H3 id="transformation-of-pkt174-in-dh5a-by-heat-shock">Transformation of pKT174 in DH5a by Heat shock</H3><P>Protocol from addgene:</P><UL><LI>Thaw cells on ice (20 min)</LI><LI>3 ul DNA + 50 ul chemically competent cells, mix gently</LI><LI>20-30 min on ice</LI><LI>45s at 42°C</LI><LI>2 min on ice</LI><LI>Add 1 ml LB (at 18h50)</LI><LI>Incubate 1 hour @ 37°C</LI></UL><P>2 ampicillin plates: 100 ul and 900 ul.</P><P>Incubation overnight @37.</P><H3 id="gel-for-soe-2907">Gel for SOE 29/07</H3><TABLE><TBODY><TR class="odd"><TD align="left"><P>1 kb ladder</P></TD><TD align="left"><P>SOE</P></TD><TD align="left"><P>SOE</P></TD><TD align="left"><P>SOE</P></TD><TD align="left"><P>1 kb ladder</P></TD></TR></TBODY></TABLE><P>Expected size: 5600 for all of them.</P><DIV class="figure"><P class="caption">SOE+29.07.jpg</P></DIV><P>It didn’t work at all. Try to Gibson-assemble them.</P><H3 id="new-electroporation-of-pir116">New electroporation of Pir116</H3><P>Using the old electroporator (with the square cuvette holder).</P><P>Dialysis of 6 ul and sampling of 5 ul of Colibow gBlock assembly</P><P>Dialysis of 5 ul and sampling of 5 ul of 5 pg/ul</P><TABLE><TBODY><TR class="odd"><TD align="left"><P>Compound</P></TD><TD align="left"><P>x1</P></TD><TD align="left"><P>x8</P></TD></TR><TR class="even"><TD align="left"><P>10x Taq Buffer</P></TD><TD align="left"><P>2.5</P></TD><TD align="left"><P>20</P></TD></TR><TR class="odd"><TD align="left"><P>10 nM dNTPs</P></TD><TD align="left"><P>0.5</P></TD><TD align="left"><P>4</P></TD></TR><TR class="even"><TD align="left"><P>10 uM primer 90</P></TD><TD align="left"><P>0.5</P></TD><TD align="left"><P>4</P></TD></TR><TR class="odd"><TD align="left"><P>10 uM primer 94</P></TD><TD align="left"><P>0.5</P></TD><TD align="left"><P>4</P></TD></TR><TR class="even"><TD align="left"><P>taq polymerase</P></TD><TD align="left"><P>0.125</P></TD><TD align="left"><P>1</P></TD></TR><TR class="odd"><TD align="left"><P>water</P></TD><TD align="left"><P>17.875</P></TD><TD align="left"><P>167</P></TD></TR></TBODY></TABLE><P>Then 22 ul of mix is added to 3 ul of template (2 ul water + 1 ul sample for positive control). Template is obtained by soaking part of a colony in 20 ul water.</P><H4 id="program-saved-as-colony">Program (saved as Colony):</H4><P>95 (6’00)</P><P>95 (15) 49 (25) 68 (45) x30</P><P>68 (5’00)</P><H4 id="gel">Gel:</H4><P>P900        B1        B2        B3        ColibowB        100bp+</P><H4 id="results">Results:</H4><UL><LI>One band at 508 bp on the positive control.</LI><LI>No band at all on the colonies</LI></UL><P>-&gt; They are contaminations, the transformation did not work, probably because the Gibson itself did not work due to bad DNA concentration.</P><H2 class="date" id="wednesday-85">Wednesday, 8/5</H2><H4 id="pir116-electrocompetent-cells">Pir116 electrocompetent cells</H4><P>From the overnight culture, 1 ml of cells were diluted in 50 ml of LB and incubated at 37°C (10h30).</P><P>Centrifuge steps were done 10 min at 8500 rpm.</P><UL><LI>50 ml culture -&gt; centrifuge and remove well the supernatant</LI><LI>50 ml glycerol 10% -&gt; centrifuge</LI><LI>50 ml glycerol 10% -&gt; centrifuge</LI></UL><P>Cells in residual glycerol were aliquoted in eppendorf tubes (6 in the end) and frozen @-80.</P><H4 id="testing-of-these-ec-pir116-cells-function">Testing of these EC Pir116 cells function</H4><P>4 ul of pSB1C3 20 pg/ul were dialysed, 3 ul were picked after 20 min and mixed with 100 ul of cells.</P><P>Pulse duration: 5.7 ms</P><P>After 1 h of preculture @37, it was plated on a chloramphenicol 0.5 plate and grown @37.</P><H4 id="new-colibow-pcr">New colibow PCR</H4><P>Performed by Chloé and Émilie.</P><DIV class="figure"><P class="caption">1438872081.png</P></DIV><P>Program:</P><P>98 (30)</P><P>35x 98 (10), Gradient (30), 72 (2’20)</P><P>72 (5)</P><P>10 (hold)</P><P>Gradient:</P><P>59, 57.8, 55.3, 53.4</P><P>The very long extension time is used to avoid promoting small non-specific fragments.</P><P>Gel:</P><P>1% agarose, SYBRsafe, 2.5 ul sample + 0.5 ul LD</P><TABLE><TBODY><TR class="odd"><TD align="left"><P>L</P></TD><TD align="left"><P>A</P></TD><TD align="left"><P>A</P></TD><TD align="left"><P>A</P></TD><TD align="left"><P>A</P></TD><TD align="left"><P>L</P></TD><TD align="left"><P>B</P></TD><TD align="left"><P>B</P></TD><TD align="left"><P>B</P></TD><TD align="left"><P>B</P></TD><TD align="left"><P>L</P></TD><TD align="left"><P>C</P></TD><TD align="left"><P>C</P></TD><TD align="left"><P>C</P></TD><TD align="left"><P>C</P></TD><TD align="left"><P>L</P></TD></TR><TR class="even"><TD align="left"><P>1 kb</P></TD><TD align="left"><P>1172</P></TD><TD align="left"><P>1 kb</P></TD><TD align="left"><P>909</P></TD><TD align="left"><P>1 kb</P></TD><TD align="left"><P>992</P></TD></TR></TBODY></TABLE><DIV class="figure"><P class="caption">08.05 Colibow Amp gradient.jpg</P></DIV><H3 id="thursday-86">Thursday, 8/6</H3><H4 id="pcr-purification-of-colibow-gblocks">PCR purification of Colibow gBlocks</H4><P>The homologous tubes were mixed together, except for C2 that didn’t work.</P><P>Elution in 50 ul of water.</P><P>A second gel was ran in order to know whether the light band at the bottom is still present.</P><TABLE><TBODY><TR class="odd"><TD align="left"><P>1 kb ladder</P></TD><TD align="left"><P>A</P></TD><TD align="left"><P>B</P></TD><TD align="left"><P>C</P></TD></TR></TBODY></TABLE><P>Is it. This has to be taken into account when calculating the concentrations for the Gibson assembly.</P><P>Titration (in 50 ul of water):</P><P>A: 137 ng/ul</P><P>B: 167 ng/ul</P><P>C: 99 ng/ul</P><DIV class="figure"><P class="caption">08.06 Colibow gradient after purification.jpg</P></DIV><H3 id="new-r6k-pcr">New R6K PCR</H3><TABLE><TBODY><TR class="odd"><TD align="left"><P>Compound</P></TD><TD align="left"><P>1x</P></TD><TD align="left"><P>2x</P></TD></TR><TR class="even"><TD align="left"><P>water</P></TD><TD align="left"><P>71</P></TD><TD align="left"><P>142</P></TD></TR><TR class="odd"><TD align="left"><P>Phu buffer</P></TD><TD align="left"><P>20</P></TD><TD align="left"><P>40</P></TD></TR><TR class="even"><TD align="left"><P>dNTP</P></TD><TD align="left"><P>2</P></TD><TD align="left"><P>4</P></TD></TR><TR class="odd"><TD align="left"><P>80 primer</P></TD><TD align="left"><P>1</P></TD><TD align="left"><P>2</P></TD></TR><TR class="even"><TD align="left"><P>81 primer</P></TD><TD align="left"><P>1</P></TD><TD align="left"><P>2</P></TD></TR><TR class="odd"><TD align="left"><P>pR6K shortened (template)</P></TD><TD align="left"><P>1</P></TD><TD align="left"><P>2</P></TD></TR><TR class="even"><TD align="left"><P>DMSO</P></TD><TD align="left"><P>3</P></TD><TD align="left"><P>6</P></TD></TR><TR class="odd"><TD align="left"><P>Phusion polymerase</P></TD><TD align="left"><P>1</P></TD><TD align="left"><P>2</P></TD></TR></TBODY></TABLE><P>Program:</P><P>98 (30)</P><P>98 (10) 52 (30) 72 (1’30) x 35</P><P>72 (5’)</P><P>Problem: There was only &lt;1 ul of phusion left in the tube. The PCR did not work at all.</P><H4 id="new-attempt">New attempt</H4><P>Mix (made twice)</P><P>Phusion master mix        50 ul</P><P>80 primer                2 ul</P><P>81 primer                2 ul</P><P>pR6K                        3 ul</P><P>DMSO                        3 ul</P><P>Water                        40 ul</P><P>Program:</P><P>98 (30)</P><P>98 (10) 50 (30) 72 (1’30) x 35</P><P>72 (5’)</P><P>After this PCR, the product was:</P><UL><LI>ran on a gel: Ladder 1kb, tube 1, tube 2 (3 ul sample, 2 ul water, 1 ul LD)</LI><LI>digested by DPN1:</LI></UL><P>1 ul of DPN1 added directly to the product, then incubated for 15 minutes at 37° in the thermocycler and inactivated (5 min at 80°).</P><P>Gel results: [08.09 R6K pcr]</P><H2 class="date" id="monday-0810">Monday, 08/10</H2><H3 id="pcr-purification-of-r6k-amp">PCR purification of R6K amp</H3><P>With QIAGEN kit, performed by Chloé &amp; Émilie. At the end, the product was recovered in 50 ul of water and also 30 ul EB, for 80 ul total.</P><P>The titration was done using a 5 ul + 3 ul mix as a blank.</P><H4 id="products-summary">Products summary:</H4><TABLE><TBODY><TR class="odd"><TD align="left"><P>R6K pcr</P></TD><TD align="left"><P>67.4 ng/ul</P></TD><TD align="left"><P>2276 bp</P></TD></TR><TR class="even"><TD align="left"><P>Colibow A</P></TD><TD align="left"><P>137</P></TD><TD align="left"><P>1172 bp</P></TD></TR><TR class="odd"><TD align="left"><P>Colibow B</P></TD><TD align="left"><P>167</P></TD><TD align="left"><P>909 bp</P></TD></TR><TR class="even"><TD align="left"><P>Colibow C</P></TD><TD align="left"><P>99</P></TD><TD align="left"><P>992 bp</P></TD></TR></TBODY></TABLE><P>A and C have a small band at ~200 bp, so the actual concentration of the product is half of what indicated by the Nanodrop.</P><H3 id="gibson-assembly-of-colibow">Gibson assembly of Colibow</H3><P>Assuming half the DNA in A and C is the right one.</P><TABLE><TBODY><TR class="odd"><TD align="left"><P>Name</P></TD><TD align="left"><P>Volume (ul)</P></TD><TD align="left"><P>Final amount (pmol)</P></TD></TR><TR class="even"><TD align="left"><P>A</P></TD><TD align="left"><P>1.11</P></TD><TD align="left"><P>0.10</P></TD></TR><TR class="odd"><TD align="left"><P>B</P></TD><TD align="left"><P>0.4</P></TD><TD align="left"><P>0.11</P></TD></TR><TR class="even"><TD align="left"><P>C</P></TD><TD align="left"><P>1.30</P></TD><TD align="left"><P>0.10</P></TD></TR><TR class="odd"><TD align="left"><P>R6K</P></TD><TD align="left"><P>2.19</P></TD><TD align="left"><P>0.10</P></TD></TR></TBODY></TABLE><P>Added to a MMII Gibson assembly master mix and incubated during 1h at 50°C.</P><H4 id="gel-for-colibows-gibson">Gel for Colibow’s Gibson</H4><P>6 ul SYBRsafe in 20 ml agar 1%, with 1 kb ladder.</P><P>The sample consisted in 10 ul of Gibson product and 2 ul of LD.</P><P>Unfortunately the comb for making the wells went through the gel, resulting in loss of the sample.</P><H3 id="electroporation-of-pir116-e.-coli-with-newly-assembly-pcolibow">Electroporation of Pir116 E. coli with newly assembly pColibow</H3><UL><LI>Dialyse of the assembly product during 20 min. 5 ul were deposed and 5 ul were taken back.</LI><LI>Mixed with already tested Pir116 Electrocompetent cells</LI><LI>Program Ec2, bacterial, in 2 mm cuvette. Pulse time: 4.7 ms.</LI><LI>Incubation 1h @37</LI><LI>Plating on “pColibow Gibson” plates (K50 but K20 written on the box). 100 ul on the first box, centrifuged 900 ul on the second one.</LI><LI>Incubation overnight @37.</LI></UL><H2 class="date" id="tuesday-0811">Tuesday, 08/11</H2><H3 id="pir116-colibow-electroporation-results">Pir116 Colibow electroporation results</H3><P>We got a lot (~300) colonies on the 100 ul plate, and even more on the other plate (too many to distinguish). The colonies don't appear to be red at all. We can distinguish two types of colonies: bigger ones and smaller ones.</P><H3 id="colony-pcr-for-checking-colibow-transformation">Colony PCR for checking colibow transformation</H3><P>10 big colonies were picked (we're waiting for the small ones to be big enough to be tested) and diluted in 20 ul of water.</P><TABLE><THEAD><TR class="header"><TH align="left"><P>x1</P></TH><TH align="left"><P>x13</P></TH></TR></THEAD><TBODY><TR class="odd"><TD align="left"><P>10x Taq Buffer</P></TD><TD align="left"><P>2.5</P></TD><TD align="left"><P>32.5</P></TD></TR><TR class="even"><TD align="left"><P>10 nM dNTP</P></TD><TD align="left"><P>0.5</P></TD><TD align="left"><P>6.5</P></TD></TR><TR class="odd"><TD align="left"><P>Primer 90</P></TD><TD align="left"><P>0.5</P></TD><TD align="left"><P>6.5</P></TD></TR><TR class="even"><TD align="left"><P>Primer 94</P></TD><TD align="left"><P>0.5</P></TD><TD align="left"><P>6.5</P></TD></TR><TR class="odd"><TD align="left"><P>Taq</P></TD><TD align="left"><P>0.125</P></TD><TD align="left"><P>1.625</P></TD></TR><TR class="even"><TD align="left"><P>Water</P></TD><TD align="left"><P>17.87</P></TD></TR></TBODY></TABLE><P>Then, 22 ul mix + 3 ul template. <EM>Program:</EM> Same as last colony PCR (saved as Colony).</P><P><EM>Gel:</EM> 1,2,3,4,5,6,7,8,9,10, negative, positive. <EM>Negative:</EM> No template. <EM>Positive:</EM> Colibow B diluted gBlock.</P><P>[08.10 Colony PCR]</P><P>Results: The positive control is positive, but it didn't involve cell lysis. All the colonies are negative, which means that either the cell lysis didn't work, or they are not transformed properly.</P><P>Hypothesis: Too much template in R6K PCR, or the DPN1 digestion was not thorough enough.</P><P>The clones 1,2,5,6 and 7 were put into culture (4 ml LB Kan+) for further analysis. It included a negative control (un-transformed Pir116).</P><H3 id="gel-extraction-of-colibow-a-and-c">Gel extraction of Colibow A and C</H3><TABLE><TBODY><TR class="odd"><TD align="left"><P>A|A|1 kb ladder|C|C</P></TD></TR></TBODY></TABLE><P>[08.10 Colibow A and C]</P><P>The bands were cut and extracted with Qiagen kit. Recovery in 30 ul water. As Loading Dye was accidentally mixed with the B sampled, it was PCR-cleaned-up again and recovered in 30 ul water.</P><P>Titration:</P><DL><DT>A : 28.4 ng/ul
B : 212 ng/ul
C : 34 ng/ul</DT></DL><H2 class="date" id="wednesday-0812">Wednesday 08/12</H2><H3 id="new-new-r6k-pcr">New new R6K PCR</H3><P><EM>Mix</EM></P><P>Phusion master mix        50 ul</P><P>80 primer                2 ul</P><P>81 primer                2 ul</P><P>pR6K diluted 20 times                        1 ul</P><P>DMSO                        3 ul</P><P>Water                        42 ul</P><P>The goal is to minimize the amount of R6K backbone in the mix, to limit background colonies.</P><P><EM>Program:</EM></P><P>98 (30)</P><P>98 (10) 52 (30) 72 (1’30) x 35</P><P>72 (5’)</P><P><EM>Results:</EM> It did not work (gel not shown).</P><H3 id="new-new-new-r6k-pcr">New new new R6K PCR</H3><P><EM>Mix</EM></P><TABLE><THEAD><TR class="header"><TH align="left"><P>x1</P></TH><TH align="left"><P>x2</P></TH></TR></THEAD><TBODY><TR class="odd"><TD align="left"><P>Phusion master mix</P></TD><TD align="left"><P>50</P></TD><TD align="left"><P>100</P></TD></TR><TR class="even"><TD align="left"><P>primer 80</P></TD><TD align="left"><P>2</P></TD><TD align="left"><P>4</P></TD></TR><TR class="odd"><TD align="left"><P>primer 81</P></TD><TD align="left"><P>2</P></TD><TD align="left"><P>4</P></TD></TR><TR class="even"><TD align="left"><P>pR6K</P></TD><TD align="left"><P>0.5</P></TD><TD align="left"><P>1</P></TD></TR><TR class="odd"><TD align="left"><P>DMSO</P></TD><TD align="left"><P>3</P></TD><TD align="left"><P>6</P></TD></TR><TR class="even"><TD align="left"><P>Water</P></TD><TD align="left"><P>42.5</P></TD><TD align="left"><P>85</P></TD></TR></TBODY></TABLE><P><EM>Program:</EM></P><P>98 (30)</P><P>98 (10) 50 (30) 72 (1’30) x 35</P><P>72 (5’)</P><H3 id="analysis-of-colonies-from-pcolibow-transformation">Analysis of colonies from pColibow transformation</H3><P>All of the clones grew overnight. The negative control did not grow, meaning that the kanamycine we used is efficient and the Pir116 native cells are not resistant.</P><P>The cultures 1, 2, 5 and 6 were miniprepped (3 ml + 1 ml saved for Glycerol stock if needed) and recovered in 50 ul of water.</P><P>Titers:</P><DL><DT>I : 96
II : 105
V : 111
VI : 106</DT></DL><P>The rest was put back into culture at 37°.</P><H3 id="noti-digestion-of-the-plasmids">NotI digestion of the plasmids</H3><P>NotI cuts only once in both the pR6K vector and in the pColibow plasmid.</P><TABLE><THEAD><TR class="header"><TH align="left"><P>x1</P></TH><TH align="left"><P>x4</P></TH></TR></THEAD><TBODY><TR class="odd"><TD align="left"><P>Water</P></TD><TD align="left"><P>16</P></TD><TD align="left"><P>64</P></TD></TR><TR class="even"><TD align="left"><P>10x FD Buffer</P></TD><TD align="left"><P>2</P></TD><TD align="left"><P>8</P></TD></TR><TR class="odd"><TD align="left"><P>Plasmid</P></TD><TD align="left"><P>1</P></TD><TD align="left"><P>4</P></TD></TR><TR class="even"><TD align="left"><P>Not1</P></TD><TD align="left"><P>1</P></TD><TD align="left"><P>14</P></TD></TR></TBODY></TABLE><P>Incubation at 37° started at 17h10, stopped at 19h10.</P><H3 id="gel-after-noti-digestion">Gel after NotI digestion</H3><P>1 kb ladder, I, II, V, VI, 1 kb ladder. Expected results: 5190 bp if it's pColibow, 2229 bp if it's pR6K.</P><H3 id="gel-for-r6k-pcr">Gel for R6K PCR</H3><P>1B and R6K pcrs were ran on the same gel:</P><TABLE><TBODY><TR class="odd"><TD align="left"><P>1B | 1B | 1 kb ladder | R6K | R6K</P></TD></TR></TBODY></TABLE><P>2 ul water + 1 ul LD + 3 ul sample.</P><P>Results: [0.13 B1 and R6K]</P><P>he R6K PCR yielded two bands: one at around 1000 bp, and one at around 2500 bp. The bigger one is the good one, it could be gel-extracted if needed. However, we should stick to the previous version that will probably be more concentrated anyways due to the absence of off-product.</P><H2 class="date" id="thursday-0813">Thursday 08/13</H2><P>The new new new new new R6K pcr still did not work. There is a serious problem with one of the things we use because this PCR worked before.</P><H3 id="dpn1-digestion-of-the-previous-r6k-product">DPN1 digestion of the previous R6K product</H3><P>As it's no longer possible to even <EM>replicate</EM> the previous R6K pcr for some reason, I will stick with the small, diluted sample that I have and digest it with DPN1. That's the only thing I can do.</P><P>For R6K vol ~ 20 ul DPN1 FD 5ul 10x FD Buffer 3ul</P><P>37°C during 30 min, then 80°C for 20 min.</P><P>Titration: 24 ng/ul, with a high 280 absorbance due to DPN1 itself. I don't know why the total DNA concentration dropped.</P><H2 class="date" id="friday-0814">Friday 08/14</H2><H2 class="date" id="saturday-0815">Saturday 08/15</H2><H3 id="r6k-pcr-attempt-n">R6K PCR, attempt N</H3><P>Using either purified R6K plasmid or R6K PCR product.</P><P><EM>Mix</EM>: Master mix, 50 ul F (81), 1ul R (80), 1ul R6K, 1ul DMSO, 3 ul water, 44 ul</P><P>Each of them was done in duplicate, with a gradient (annealing temperature of 52 or 55°C).</P><P><EM>Program</EM> 98 (30) 98 (10) gradient (30) 72 (1'30) * 36 72 (5)</P><P><EM>Result</EM>: absolutely nothing.</P><H3 id="r6k-pcr-attempt-n1">R6K PCR, attempt N+1</H3><P><EM>Mix</EM>: Master mix, 50 ul F (81), 2ul R (80), 2ul R6K, 1ul DMSO, 3 ul water, 44 ul</P><P>Same prog, except 50°C annealing, 3'00 extension, 37 cycles.</P><P>It didn't work, once again.</P><P>3 ml LB + Kanamycine + R6K freezer stock cells for new miniprep, because the template is the only parameter that was not completely new.</P><H2 class="date" id="monday-0817">Monday 08/17</H2><H3 id="preparation-for-clonetegration">Preparation for clonetegration</H3><P>As the R6K setup definitely doesn't work, the working PCRs can't even be reproduce and we're stuck at the first step of a long protocol, I decided to change the strategy completely. Two new primers were order in order to change the vector and use pIT5-KH for clonetegration.</P><P>The strain carrying pIT5-KH (u15.39) was put into culture for miniprep in LB Kan+.</P><H3 id="new-r6k-gibson-tentative">New R6K gibson tentative</H3><P>Using newly digested R6K backbone (poor concentration).</P><TABLE><THEAD><TR class="header"><TH align="left"><P>Name</P></TH><TH align="left"><P>Concentration</P></TH><TH align="left"><P>Size</P></TH><TH align="left"><P>Volume</P></TH></TR></THEAD><TBODY><TR class="odd"><TD align="left"><P>A</P></TD><TD align="left"><P>28.4</P></TD><TD align="left"><P>1172</P></TD><TD align="left"><P>1.22</P></TD></TR><TR class="even"><TD align="left"><P>B</P></TD><TD align="left"><P>212</P></TD><TD align="left"><P>909</P></TD><TD align="left"><P>0.13 (0.5 actually)</P></TD></TR><TR class="odd"><TD align="left"><P>C</P></TD><TD align="left"><P>34</P></TD><TD align="left"><P>992</P></TD></TR><TR class="even"><TD align="left"><P>R6K</P></TD><TD align="left"><P>24</P></TD><TD align="left"><P>2276</P></TD><TD align="left"><P>0.86</P></TD></TR></TBODY></TABLE><P>This gives 0.01 pmol of each part.</P><P>As a control to measure undigested vector, 2.80 ul of linearized R6K vector + 17.2 ul of water.</P><H3 id="gblocks-assembly">gBlocks assembly</H3><TABLE><THEAD><TR class="header"><TH align="left"><P>Name</P></TH><TH align="left"><P>Concentration</P></TH><TH align="left"><P>Size</P></TH><TH align="left"><P>Volume</P></TH></TR></THEAD><TBODY><TR class="odd"><TD align="left"><P>A</P></TD><TD align="left"><P>10</P></TD><TD align="left"><P>1172</P></TD><TD align="left"><P>1.18</P></TD></TR><TR class="even"><TD align="left"><P>B</P></TD><TD align="left"><P>10</P></TD><TD align="left"><P>909</P></TD><TD align="left"><P>1.46</P></TD></TR><TR class="odd"><TD align="left"><P>C</P></TD><TD align="left"><P>10</P></TD><TD align="left"><P>992</P></TD><TD align="left"><P>1.13</P></TD></TR><TR class="even"><TD align="left"><P>R6K</P></TD><TD align="left"><P>10</P></TD><TD align="left"><P>2276</P></TD><TD align="left"><P>0.86</P></TD></TR></TBODY></TABLE><P>That's 0.004 pmol of each (barely enough...).</P><P>Started at 4 pm, ended at 5 pm and put in ice.</P><H3 id="pcr-for-checking-the-gibson">PCR for checking the gibson</H3><P>To know whether the Gibson Assembly really works or not, a bunch of PCR were made.</P><P>Junction 1: seqF1 + seqR4 = 799 bp Junction 2: seqF2 + seqR3 = 625 bp Junction 3: seqF4 + seqR1 = 808 bp Junction 4: seqF5 + jw06 = 829 bp</P><P>On 2 ul of Gibson product. Control : R2 + F3 = 508 bp</P><TABLE><THEAD><TR class="header"><TH align="left"><P>DT mix</P></TH><TH align="left"><P>50</P></TH></TR></THEAD><TBODY><TR class="odd"><TD align="left"><P>Template</P></TD><TD align="left"><P>15</P></TD></TR><TR class="even"><TD align="left"><P>Water</P></TD></TR></TBODY></TABLE><P>Then 18 ul of master mix + 1 ul of each primer.</P><P>95 (3') 95 (30) 52 (30) 72 (1 min) 72 (5')</P><P><EM>Results</EM>: 10 ul + 2 ul LD were deposited on a gel. gBlock I, II, III, IV, + ; 100 bp+ ; PCR I, II, III, IV, + [junctions] All of the junctions seem present, but for the PCR sample they are way sharper. Strangely, the control PCR did not work so well.</P><H2 class="date" id="tuesday-1808">Tuesday 18/08</H2><H3 id="transformation-of-both-gibson-assembly-product">Transformation of both Gibson Assembly product</H3><P>(as at least all the junctions work...). Target: Electrocompetent Pir116 cells. All of the gibson product (~5 ul) was dialysed, along with the control (PCR R6K digested by DPN1 + water). 8 ul of dialyse product were taken back after 20 min. Pulse times: Control: 5.7 ms Gibson gblocks: 5.6 ms Gibson PCR: 5.5 ms</P><H3 id="plates-with-kanamycine-20-ugml">Plates with Kanamycine 20 ug/ml</H3><P>100 ml LB agar + 40 ul Kanamycine.</P><H2 class="date" id="wednesday-1908">Wednesday 19/08</H2><H3 id="colibow-transformation-results">Colibow transformation results</H3><P>gBlock: nothing clear, a few number of what looks like colonies in the mess. PCR: Same with way more colonies. Control: negative.</P><P>They were re-streaked on LB + Kan 25 plates made for the occasion. A total of 16 colonies were tested.</P><H3 id="miniprep-of-pit5-kh">Miniprep of pIT5-KH</H3><P>Elution in 5* ul of water. It is very clean and the concentration is 167 ng/ul.</P><H3 id="colibow-a-and-c-pcr-for-pit5-cloning">Colibow A and C PCR for pIT5 cloning</H3><P>The goal is to make the A and C colibow parts compatible with pIT5-KH Gibson assembly. This PCR was performed by Constant.</P><P><EM>Colibow A:</EM> Primers 176 and 83.</P><P><EM>Colibow C:</EM> Primers 86 and 177.</P><TABLE><THEAD><TR class="header"><TH align="left"><P>Phusion master mix (&quot;B+M&quot;)</P></TH><TH align="left"><P>50</P></TH></TR></THEAD><TBODY><TR class="odd"><TD align="left"><P>Primer F</P></TD><TD align="left"><P>1</P></TD></TR><TR class="even"><TD align="left"><P>Primer R</P></TD><TD align="left"><P>1</P></TD></TR><TR class="odd"><TD align="left"><P>Colibow A/C</P></TD><TD align="left"><P>1 ul</P></TD></TR><TR class="even"><TD align="left"><P>DMSO</P></TD><TD align="left"><P>3</P></TD></TR><TR class="odd"><TD align="left"><P>Water</P></TD><TD align="left"><P>44</P></TD></TR></TBODY></TABLE><P><EM>Program</EM> 98 (30)</P><P>98 (10), 50 (30), 72 (2'20) * 37</P><P>72 (5')</P><P><EM>Results</EM> Only a slight smear, no visible PCR product. It didn't work (gel not shown).</P><H3 id="digestion-of-pit5-kh">Digestion of pIT5-KH</H3><P>The pIT5-KH vector was linearized by the restriction enzymes EcoR1 and Pst1. This allows excision of the pUC propagation ORI. There is still a R6K ORI that will not bee used here. Hence, the vector is no longer able to replicate and only the integrants will be replicated.</P><TABLE><THEAD><TR class="header"><TH align="left"><P>Plasmid</P></TH><TH align="left"><P>50 ul</P></TH></TR></THEAD><TBODY><TR class="odd"><TD align="left"><P>FD Green Buffer</P></TD><TD align="left"><P>5 ul</P></TD></TR><TR class="even"><TD align="left"><P>FD EcoR1</P></TD><TD align="left"><P>3.2 ul</P></TD></TR><TR class="odd"><TD align="left"><P>FD Pst1</P></TD></TR></TBODY></TABLE><P>This was incubated at 37°C for 20 minutes and not inactivated, as the gel extraction followed immediately.</P><H3 id="gel-extraction-of-digested-pit5-kh">Gel extraction of digested pIT5-KH</H3><P>Layout: 1 kb ladder, 3 merged wells with all of the digestion product.</P><P>Expected sizes: 1143 bp (to remove) and 5331 bp (to keep). The bands are present at the right size and strong.</P><P>After gel extraction, the product was eluted in 30 ul of pre-heated Elution Buffer.</P><P><EM>Titration</EM>: 19 ng/ul.</P><H3 id="re-streaking-of-pcolibow-r6k-transformation-product">Re-streaking of pColibow-R6K transformation product</H3><P>To figure out if they're resistant or not. 12 LB Kan25 plates were made, with 200 ml of LB-agar and 100 ul of Kanamycine 1000x.</P><P>Transformation from PCR product: Two plates from the 200 ul plate, one plate from the 800 ul plate. Transformation from gBlocks: Two plates from the 800 ul plate.</P><P>Each of these plates is divided in 4 parts, with one colony on each.</P><H2 class="date" id="thursday-2008">Thursday 20/08</H2><H3 id="colibow-r6k-rescue">Colibow R6K rescue</H3><P>It seems that it is somewhat resistant to Kan but it is not clear: it only grew at the most cell-concentrated spots. It was put back in the incubator in case slower colonies would show up.</P><H3 id="colony-pcr">Colony PCR</H3><P>A PCR with 4 primers was done: - We expect one band (jw005 + jw006 = 139 bp) if the backbone is present (unlikely due to the control plate being empty) - We expect two bands (jw005 + 96 = 714 bp and 92 + jw006 = 829 bp) if the right pColibow plasmid is present</P><TABLE><THEAD><TR class="header"><TH align="left"><P>Dream taq master mix</P></TH><TH align="left"><P>90</P></TH></TR></THEAD><TBODY><TR class="odd"><TD align="left"><P>jw05</P></TD><TD align="left"><P>9</P></TD></TR><TR class="even"><TD align="left"><P>jw06</P></TD><TD align="left"><P>9</P></TD></TR><TR class="odd"><TD align="left"><P>primer 92</P></TD><TD align="left"><P>9</P></TD></TR><TR class="even"><TD align="left"><P>primer 96</P></TD><TD align="left"><P>1</P></TD></TR><TR class="odd"><TD align="left"><P>Colibow A/C</P></TD><TD align="left"><P>1 ul</P></TD></TR><TR class="even"><TD align="left"><P>DMSO</P></TD><TD align="left"><P>3</P></TD></TR><TR class="odd"><TD align="left"><P>Water</P></TD><TD align="left"><P>44</P></TD></TR></TBODY></TABLE><P>Then 17 ul of mix was added to 3 ul of cell resuspension solution. Four colonies were tested for each transformation (1,2,3,4 and I,II,III,IV).</P><P>The program was the default DreamTaq program with 53°C of annealing temperature.</P><P><EM>Results:</EM> Absolutely nothing. Either the PCR didn't work, either the cells are not really resistant to kanamycin.</P><H3 id="pcr-for-colibow-pit5">PCR for Colibow pIT5</H3><P><EM>Gradient PCR</EM>.</P><P><EM>Primers</EM> A: 176 + 83 -&gt; 1202 bp B: 84 + 85 -&gt; 909 bp C: 86 + 177 -&gt; 1004 bp 176 and 177 were diluted again from stock.</P><TABLE><THEAD><TR class="header"><TH align="left"><P>Name</P></TH><TH align="left"><P>1x</P></TH><TH align="left"><P>6.5x</P></TH></TR></THEAD><TBODY><TR class="odd"><TD align="left"><P>2x Phusion Master Mix</P></TD><TD align="left"><P>50</P></TD><TD align="left"><P>325</P></TD></TR><TR class="even"><TD align="left"><P>DMSO</P></TD><TD align="left"><P>3</P></TD><TD align="left"><P>19.5</P></TD></TR><TR class="odd"><TD align="left"><P>Water</P></TD><TD align="left"><P>35</P></TD></TR></TBODY></TABLE><P>44 ul of mix without primers and template + 5 ul of each primer + 2 ul of template. The templates were diluted again from the gBlocks.</P><P><EM>Program</EM>:</P><P>98 (30)</P><P>98 (10), gradient (30), 72 (2'30)</P><P>72 (5')</P><P>It takes 3h30. Gradient: 60.2 (circled), 59, 57.1, 54.8</P><H2 class="date" id="friday-2108">Friday 21/08</H2><H3 id="results-of-colibow-pit5-pcr">Results of Colibow pIT5 pcr</H3><P>First gel: 1 kb ladder, A (circled), A, A, A, B (circled), B, B, B</P><P>Second gel: C (circled), C, C, C, 1 kb ladder</P><P>[colibow A B] [colibow C]</P><P>The PCR worked: We get the right bands (A: 1202, B: 909, C: 1004). As before, A and C have a low-size impurity.</P><H3 id="gel-extraction-of-colibow-pit5-parts">Gel extraction of Colibow pIT5 parts</H3><P>New gel: A, C, B (3 merged), 100 bp+ ladder, A (3 merged), C (3 merged). The well for B is huge. For each PCR, 50 ul of product was added to 5 ul of LD. The whole 200 ul don't fit in the triple ponds, so the first two ponds were used for A and C excess. [dirty]</P><P>This gel could not be used for extraction -&gt; Loss of the product :(</P><P>Next time, merge 2 ponds and put 100 ul in it. Do not prop the comb, or not so high.</P><H3 id="colibow-pit5-pcr-for-better-gel-extraction">Colibow pIT5 pcr for better gel extraction</H3><P><EM>What worked best last time with the gradient?</EM></P><P>Optimal annealing: - A: 60°C, 176 + 83 -&gt; 1202 bp - B: 55°C, 84 + 85 -&gt; 909 bp - C: 57°C, 86 + 177 -&gt; 1004 bp</P><TABLE><THEAD><TR class="header"><TH align="left"><P>Name</P></TH><TH align="left"><P>1x</P></TH><TH align="left"><P>3x</P></TH></TR></THEAD><TBODY><TR class="odd"><TD align="left"><P>2x Phusion Master Mix</P></TD><TD align="left"><P>50</P></TD><TD align="left"><P>150</P></TD></TR><TR class="even"><TD align="left"><P>DMSO</P></TD><TD align="left"><P>3</P></TD><TD align="left"><P>9</P></TD></TR><TR class="odd"><TD align="left"><P>Water</P></TD><TD align="left"><P>35</P></TD><TD align="left"><P>105</P></TD></TR></TBODY></TABLE><P>Then to 88 ul of mix, 5 ul of each primer and 2 ul of template were added.</P><P>Program:</P><P>98 (30)</P><P>98 (20) Gradient (30) 72 (2'30) x35</P><P>72 (5')</P><H2 class="date" id="monday-2408">Monday 24/08</H2><H3 id="analysis-of-pr6k-colibow-colonies">Analysis of pR6K-colibow colonies</H3><P>pR6K-colibow colonies from the previous transformation are resistant to kanamycine (checked by re-streaking). Some of these colonies are yellow.</P><P>They were put into culture and miniprepped (elution in 50 ul of EB). Titration: 19.8 ng/ul (clean).</P><P>The product were digested by NotI (fast-digest, 30 min at 37°C) and ran on a gel.</P><P><EM>Results</EM>: Nothing showed up on the gel, despite the very high deposited DNA amount. I don't know what happened, but it happens everytime.</P><H3 id="gel-extraction-of-colibow-for-pit5-vector">Gel extraction of Colibow for pIT5 vector</H3><P><EM>Layout:</EM> B, B, B, B, A, A, A, A, C, C, C, C No ladder, I already ran a gel of these products and I know what it looks like.</P><P><EM>Result:</EM> All the 3 PCRs worked very well, though there is a small band in all of them (pretty strong on C).</P><P>The good bands (909, 1202 and 1004 bp respectively) were extracted, purified and eluted in 30 ul of pre-heated Elution Buffer.</P><P><EM>Titration:</EM></P><TABLE><THEAD><TR class="header"><TH align="left"><P>A</P></TH><TH align="left"><P>22 ng/ul, not so clean (presence of organic molecules)</P></TH></TR></THEAD><TBODY><TR class="odd"><TD align="left"><P>B</P></TD><TD align="left"><P>30 ng/ul, clean</P></TD></TR><TR class="even"><TD align="left"><P>C</P></TD><TD align="left"><P>47 ng/ul, clean</P></TD></TR></TBODY></TABLE><H3 id="gibson-assembly">Gibson assembly</H3><P>Two reaction were done: one with strict equimolar concentrations, and one with a lower amount of vector as it's what takes all of the space.</P><TABLE><THEAD><TR class="header"><TH align="left"><P>Name</P></TH><TH align="left"><P>Concentration</P></TH><TH align="left"><P>Size</P></TH><TH align="left"><P>Volume</P></TH></TR></THEAD><TBODY><TR class="odd"><TD align="left"><P>A</P></TD><TD align="left"><P>22</P></TD><TD align="left"><P>1202</P></TD><TD align="left"><P>0.71</P></TD></TR><TR class="even"><TD align="left"><P>B</P></TD><TD align="left"><P>30</P></TD><TD align="left"><P>909</P></TD><TD align="left"><P>0.39</P></TD></TR><TR class="odd"><TD align="left"><P>C</P></TD><TD align="left"><P>47</P></TD><TD align="left"><P>1004</P></TD><TD align="left"><P>0.28</P></TD></TR><TR class="even"><TD align="left"><P>pIT5-KH</P></TD><TD align="left"><P>19</P></TD><TD align="left"><P>2276</P></TD><TD align="left"><P>3.63</P></TD></TR></TBODY></TABLE><P>Final amount: 0.004 nmol of each, which is way below the recommended concentration.</P><TABLE><THEAD><TR class="header"><TH align="left"><P>Name</P></TH><TH align="left"><P>Concentration</P></TH><TH align="left"><P>Size</P></TH><TH align="left"><P>Volume</P></TH></TR></THEAD><TBODY><TR class="odd"><TD align="left"><P>A</P></TD><TD align="left"><P>22</P></TD><TD align="left"><P>1202</P></TD><TD align="left"><P>1.03</P></TD></TR><TR class="even"><TD align="left"><P>B</P></TD><TD align="left"><P>30</P></TD><TD align="left"><P>909</P></TD><TD align="left"><P>0.57</P></TD></TR><TR class="odd"><TD align="left"><P>C</P></TD><TD align="left"><P>47</P></TD><TD align="left"><P>1004</P></TD><TD align="left"><P>0.40</P></TD></TR><TR class="even"><TD align="left"><P>pIT5-KH</P></TD><TD align="left"><P>19</P></TD><TD align="left"><P>2276</P></TD><TD align="left"><P>3.00</P></TD></TR></TBODY></TABLE><P>Final amount: 0.006 pmol of each.</P><H3 id="transformation-of-pit5-kh-colibow">Transformation of pIT5-KH-Colibow</H3><P>The Gibson assembly product was transformed in Mukit's DH5a electrocompetent cells (that didn't give good results in the past), and also in NEB turbo electrocompetent cells made by Barth and Mukit.</P><P>For each transformation, 8 ul of assembly was dialysed and electroporated (pulse went from 5.4 to 5.5). The cells were then plated on LB Kan 25 ug/ml plates (175 ul of cells).</P><H3 id="colony-pcr-of-r6k-colibow">Colony PCR of R6K colibow</H3><P>jw01 + seqR4 -&gt; 800 bp seqF5 + jw02 -&gt; 829 bp If the backbone is present, we should have a band from jw01 and jw02 at ~200 bp.</P><P><EM>Result</EM>: Nothing on the gel. Either the PCR did not work, or this is contamination. Gel not shown.</P><H3 id="new-pit5-kh-and-kl-culture">New pIT5-KH and KL culture</H3><P>To make more digested backbone in case today's gibson fails. Culture at 30°C in LB K50.</P><H2 class="date" id="tuesday-2508">Tuesday, 25/08</H2><H3 id="transformation-results">Transformation results</H3><P>pIT5-KH-Colibow did not transform (no colonies). The leftover of the recovery culture was plated just in case.</P><H3 id="colony-pcr-on-the-coloured-colonies-from-198">Colony PCR on the coloured colonies from 19/8</H3><P>Same setup as before. Still nothing, not even positive control. I'm tired of DreamTaq never working.</P><H3 id="preparation-of-new-pit5-vector">Preparation of new pIT5 vector</H3><P>KH did not grow very much. KL grew a little more. A glycerol stock was made for both. They were miniprep'd and eluted in 70 ul of water.</P><H3 id="digestion-of-vector">Digestion of vector</H3><P>70 ul of plasmid, 3,2 ul of EcoR1 and Pst1, and 7 ul of FD green buffer were incubated 20 minutes at 37°C.</P><H3 id="extraction">Extraction</H3><P><EM>Layout</EM>: KL, KL, KL, 1 kb ladder, KH, KH, KH.</P><P>On the gel, the KL samples migrated, so did the ladder, but the KH samples DID NOT MIGRATE. I can't believe this.</P><P>The KL sample was then extracted, purified and eluted in 40 ul of water. It yielded a concentration of 10 ng/ul. This is too low and useless.</P><H3 id="starting-cultures">Starting cultures</H3><UL><LI>pIT5-KH, 10 ml of LB + 4 ul Kanamycine, in triplicate. Grown at 30°C.</LI><LI>Pir116, in 3 ml of LB with a control. Grown at 37°C.</LI><LI>u15.25, in 7 ul LB + 7 ul Tetracycline. It contains the pFHC2938 plasmid that expresses the CRE recombinase. Grown at 30°C.</LI></UL><H3 id="pcr-of-colibow-gblocks-1">PCR of Colibow gBlocks</H3><TABLE><THEAD><TR class="header"><TH align="left"><P>Name</P></TH><TH align="left"><P>1x</P></TH><TH align="left"><P>10x</P></TH></TR></THEAD><TBODY><TR class="odd"><TD align="left"><P>2x Phusion Master Mix</P></TD><TD align="left"><P>50</P></TD><TD align="left"><P>500</P></TD></TR><TR class="even"><TD align="left"><P>DMSO</P></TD><TD align="left"><P>4</P></TD><TD align="left"><P>4.8</P></TD></TR></TBODY></TABLE><P>Then 264 ul of mix + 15 ul of each primer + 6 ul of template, which was then split in 3 tubes of 100 ul each.</P><P>Program: 98 (40), 56 (30), 72 (2'20), 98 (20), 58 (30), 72 (2'20) 98 (20), 60 (30), 72 (2'20) * 35 72 (5')</P><H2 class="date" id="wednesday-2608">Wednesday 26/08</H2><H3 id="checking-the-transformation-from-the-24th">Checking the transformation from the 24th</H3><P>There are colonies on the NEB turbo plates, but they're too small to be analyzed for now.</P><H3 id="preparation-of-kh">Preparation of KH</H3><P>KH integrates better than KL so we will focus on this one from now on. It didn't grow last night unfortunately.</P><H3 id="neb-turbo-and-pir116-competent-cells">NEB turbo and Pir116 competent cells</H3><P>The overnight cultures were diluted 100 times and grown until OD reaches 0.6. Wash cycle:</P><UL><LI>25 ml of sterile water</LI><LI>25 ml of sterile water</LI><LI>12.5 ml of 10% glycerol</LI><LI>Pir116 were aliquoted with 80 ul per tube.</LI><LI>NEB turbo were aliquoted with 50 ul per tube.</LI><LI>The cells were frozen at -80°C.</LI></UL><H3 id="miniprep-of-pfhc2938">Miniprep of pFHC2938</H3><P>Performed by Constant with JB's protocol. Elution in 60 ul of water. Titration: 268,9 ng/ul.</P><H3 id="results-of-colibow-double-pcr">Results of Colibow double PCR</H3><P>With 4% DMSO and high, progressive annealing temperatures, only the C PCR worked. A and B did not work, which is not surprising because the gradient-able PCR machine was not available so the annealing temperature of 60°C was not appropriate. 4% of DMSO did not help removing the non-specific binding, so we can stick to 3% in the future.</P><P>The C PCR product is still quite strong, so it has been gel-extracted. A total of 7 bands were cut and purified. Elution was performed in 30 ul of EB, and the final concentration was 80 ng/ul, which is pretty good.</P><H3 id="new-pcr-on-colibow-a">New PCR on Colibow A</H3><P>To have a better concentration on this part, a new PCR was done:</P><TABLE><THEAD><TR class="header"><TH align="left"><P>Name</P></TH><TH align="left"><P>1x</P></TH><TH align="left"><P>3x</P></TH></TR></THEAD><TBODY><TR class="odd"><TD align="left"><P>2x Phusion Master Mix</P></TD><TD align="left"><P>50</P></TD><TD align="left"><P>150</P></TD></TR><TR class="even"><TD align="left"><P>DMSO</P></TD><TD align="left"><P>3</P></TD><TD align="left"><P>9</P></TD></TR><TR class="odd"><TD align="left"><P>Water</P></TD><TD align="left"><P>35</P></TD><TD align="left"><P>105</P></TD></TR><TR class="even"><TD align="left"><P>176</P></TD><TD align="left"><P>5</P></TD><TD align="left"><P>15</P></TD></TR><TR class="odd"><TD align="left"><P>83</P></TD><TD align="left"><P>5</P></TD><TD align="left"><P>15</P></TD></TR><TR class="even"><TD align="left"><P>Colibow A gBlock</P></TD><TD align="left"><P>2</P></TD><TD align="left"><P>6</P></TD></TR></TBODY></TABLE><P>Program: 98 (30) 98 (20) 59 (30) 72 (3'00) * 5 98 (20) Gradient (30) 72 (2'20) * 30 72 (5'00)</P><P>Gradient: 58.8, 60.5, 61.2°C. This program was saved as &quot;double&quot;.</P><H3 id="result">Result</H3><P>This PCR worked well. On this gel 2 ul of sample were loaded. [08.27 Colibow A] There is still a lot of non-specific binding and a slight smear.</P><P>The bigger band was gel-extracted, eluted in 32 ul of water and titrated to 80 ng/ul. It's not perfectly clean on the nanodrop but it's fine.</P><H2 class="date" id="thursday-2708">Thursday, 27/08</H2><H3 id="miniprep-of-pit5-kh-1">Miniprep of pIT5-KH</H3><P>Done by Abdou, starting from two 10 ml overnight cultures in LB. For each the elution was done in 50 ul of pre-heated water. There are two samples, named Grand and Petit. Titration:</P><UL><LI>Grand: 205 ng/ul, clean</LI><LI>Petit: 216 ng/ul, clean</LI></UL><P>The Grand sample was digested by EcoR1, Pst1 and BamH1.</P><UL><LI>50 ul plasmid</LI><LI>3 ul EcoR1</LI><LI>3 ul Pst1</LI><LI>3 ul BamH1</LI><LI>6 ul of FD green buffer</LI></UL><P>This was incubated at 30°C during 20 min. A sample was ran on a gel to check that the digestion worked.</P><P>It was then purified using the Qiagen PCR cleanup kit and eluted in 30 ul of water. The final titer is 39 ng/ul.</P><H3 id="gibson-assembly-1">Gibson assembly</H3><TABLE><THEAD><TR class="header"><TH align="left"><P>Name</P></TH><TH align="left"><P>Concentration</P></TH><TH align="left"><P>Size</P></TH><TH align="left"><P>Volume</P></TH></TR></THEAD><TBODY><TR class="odd"><TD align="left"><P>A</P></TD><TD align="left"><P>80</P></TD><TD align="left"><P>1202</P></TD><TD align="left"><P>0.5</P></TD></TR><TR class="even"><TD align="left"><P>B</P></TD><TD align="left"><P>200</P></TD><TD align="left"><P>909</P></TD><TD align="left"><P>0.5</P></TD></TR><TR class="odd"><TD align="left"><P>C</P></TD><TD align="left"><P>80</P></TD><TD align="left"><P>1004</P></TD><TD align="left"><P>0.5</P></TD></TR><TR class="even"><TD align="left"><P>pIT5-KH</P></TD><TD align="left"><P>39</P></TD><TD align="left"><P>6000</P></TD><TD align="left"><P>4.5</P></TD></TR></TBODY></TABLE><P>At 18h the incubation was started at 50°C. At 19h05 is was stopped, put on ice and dialysed on a membrane.</P><H3 id="transformation-of-pcolibow">Transformation of pColibow</H3><P>Both Pir116 and Neb Turbo strains were transformed. For each, one negative control, one positive control (pSB1C3) and 8 ul of Gibson product were electroporated. Gibson product was dialysed for 15 min. Pir116 strains were recovered at 30°C to prevent integration while Neb Turbo were recovered at 37°C. After that, 150 ul were plated on LB K20 agar plates and kept at 30°C (done by Barth).</P><H2 class="date" id="friday-0828">Friday 08/28</H2><H3 id="transformation-results-1">Transformation results</H3><P>Barth made mistakes while plating, the negative control and positive control were inversed. Colonies are visible only on the Neb turbo plates with Gibson product.</P><H3 id="colony-pcr-to-check-the-integration">Colony PCR to check the integration</H3><P>To check for integration, a combination of four primer is used:</P><TABLE><THEAD><TR class="header"><TH align="left"><P>Name</P></TH><TH align="left"><P>1x</P></TH><TH align="left"><P>4.5x</P></TH></TR></THEAD><TBODY><TR class="odd"><TD align="left"><P>DT master mix</P></TD><TD align="left"><P>10</P></TD><TD align="left"><P>45</P></TD></TR><TR class="even"><TD align="left"><P>P1</P></TD><TD align="left"><P>1</P></TD><TD align="left"><P>4.5</P></TD></TR><TR class="odd"><TD align="left"><P>P2</P></TD><TD align="left"><P>1</P></TD><TD align="left"><P>4.5</P></TD></TR><TR class="even"><TD align="left"><P>P3</P></TD><TD align="left"><P>1</P></TD><TD align="left"><P>4.5</P></TD></TR><TR class="odd"><TD align="left"><P>P4</P></TD><TD align="left"><P>1</P></TD><TD align="left"><P>4.5</P></TD></TR><TR class="even"><TD align="left"><P>Water</P></TD><TD align="left"><P>3</P></TD></TR></TBODY></TABLE><P>8.5 ul of mix + 2 ul of colony in water.</P><P><EM>Expected:</EM></P><UL><LI>One band at 740 bp is there is no integration.</LI><LI>Two bands at 343 and 824 if there is one integration</LI><LI>Additional band at 427 in case of tandem integrations</LI></UL><P><EM>Results:</EM> Nothing. Even with colonies without chromosomal integration, at least one band should show up, so there is a problem either with the PCR or with the cell lysis.</P><H3 id="colony-pcr-to-check-the-presence-of-insert">Colony PCR to check the presence of insert</H3><P>In parallel, the same as always.</P><TABLE><THEAD><TR class="header"><TH align="left"><P>Name</P></TH><TH align="left"><P>1x</P></TH><TH align="left"><P>10x</P></TH></TR></THEAD><TBODY><TR class="odd"><TD align="left"><P>DT master mix</P></TD><TD align="left"><P>5</P></TD><TD align="left"><P>50</P></TD></TR><TR class="even"><TD align="left"><P>90</P></TD><TD align="left"><P>0.5</P></TD><TD align="left"><P>50</P></TD></TR><TR class="odd"><TD align="left"><P>94</P></TD><TD align="left"><P>0.5</P></TD><TD align="left"><P>50</P></TD></TR><TR class="even"><TD align="left"><P>Water</P></TD><TD align="left"><P>1</P></TD><TD align="left"><P>10</P></TD></TR></TBODY></TABLE><P>Then 7ul master mix + 3 ul.</P><P><EM>Expected:</EM> One band at 508 bp.</P><P><EM>Results</EM> There is one band in most colonies. The control is not noticeably positive. Gel layout: I, II, III, IV, V, VI, VII, VIII, gBlock B, 100 bp+. [Colony pIT5KH...]</P><P>There is one band in the colonies IV, V, VI, VII and VIII so it seems they are good and the cassette was integrated in the chromosome.</P><H3 id="new-pit5kh-digestion">New pIT5KH digestion</H3><P>In case they are not good. 50 ul of plasmid + 3 ul EcoR1 + 3 ul Pst1 + 5.6 ul FD buffer 37°C for 20 minutes. It was then stored at -20°C.</P><H3 id="preparation-of-tss-buffer">Preparation of TSS buffer</H3><P>For 100 ml:</P><UL><LI>10 g of PEG 8000</LI><LI>612 mg of MgCl2</LI><LI>5 ml of DMSO</LI><LI>LB to 100 ml</LI></UL><H3 id="culture-of-integrated-colibow">Culture of Integrated Colibow</H3><P>V, VI, VII and VIII were put to overnight culture in 5 ml of LB kanamycin.</P><H3 id="plates-kanamycine">Plates Kanamycine</H3><P>Made five plates with 100 ml of LB+agar and 80 ul of Kanamycine (concentration: 40 ug/ml)</P><H2 class="date" id="saturday-0829">Saturday 08/29</H2><P>Attempted to grow the Colibow integrant in 25 ml of LB-K20 for making TSS-competent cells. For some reason, nothing grew.</P><H2 class="date" id="sunday-0830">Sunday 08/30</H2><P>Same as yesterday. Still no growth at all. Plated the first 16 colonies on K40 to check whether they are really resistant.</P><H2 class="date" id="monday-0831">Monday 08/31</H2><P>The Colibow integrant cells grow slowly but they are indeed resistant to kanamycine. The presence of the phage integrase could be the reason why they do not grow well.</P><H3 id="screening-for-lb">Screening for LB</H3><P>Maybe the LB is the problem. Using two batches of LB-agar from two different days, a screening was performed, with and without kanamycine.</P><UL><LI>LB1: from the 24th</LI><LI>LB2: from the 19th</LI></UL><P>Each plate was made with 20 ml of LB-agar and 8 ul of kanamycine. The cells were taken from the plates.</P><H3 id="new-colibow-reaction">New colibow reaction</H3><P>The digestion of KHx was checked on a gel. It was then PCR-purified and eluted in 30 ul EB. Concentration: 118 ng/ul, clean.</P><H3 id="gibson-assembly-2">Gibson assembly</H3><TABLE><THEAD><TR class="header"><TH align="left"><P>Name</P></TH><TH align="left"><P>Concentration</P></TH><TH align="left"><P>Size</P></TH><TH align="left"><P>Volume</P></TH></TR></THEAD><TBODY><TR class="odd"><TD align="left"><P>A</P></TD><TD align="left"><P>80</P></TD><TD align="left"><P>1202</P></TD><TD align="left"><P>0.5</P></TD></TR><TR class="even"><TD align="left"><P>B</P></TD><TD align="left"><P>200</P></TD><TD align="left"><P>909</P></TD><TD align="left"><P>0.5</P></TD></TR><TR class="odd"><TD align="left"><P>C</P></TD><TD align="left"><P>80</P></TD><TD align="left"><P>1004</P></TD><TD align="left"><P>0.5</P></TD></TR><TR class="even"><TD align="left"><P>pIT5-KH</P></TD><TD align="left"><P>118</P></TD><TD align="left"><P>6000</P></TD><TD align="left"><P>4.5</P></TD></TR></TBODY></TABLE><P>That is 0.018 pmol per fragment.</P><H3 id="transformation">Transformation</H3><P>The gibson product was electroporated in Pir116 and NebTurbo. Pulse: 5.3 to 5.7. After recovery it was plated on LB Kan20 plates.</P><H3 id="new-culture-of-colibow">New culture of Colibow</H3><P>Using a bottle of LB from the lab at the 6th floor, and the colonies V, VI, VII, VIII of the re-streaked plate.</P><H2 class="date" id="tuesday-0901">Tuesday 09/01</H2><H3 id="tss-competent-cells-with-colibow">TSS competent cells with colibow</H3><P>From the overnight culture (which grew a little), dilution to the 1/50 in LB-Kan.</P><H3 id="reception-of-new-oligos">Reception of new oligos</H3><UL><LI>Two oligos for biobricking the promoter.</LI><LI>Four oligos for checking integration of the KH vector.</LI></UL><H3 id="result-of-yesterdays-transformation">Result of yesterday's transformation</H3><P>In the morning nothing was visible on the plates, but a couple of hours after, there are ~30 colonies on the Pir116 and NT plates. The Pir116 control seems contaminated with a few very small colonies.</P><P>Four Pir116 colonies were put into culture for miniprep and analysis.</P><H3 id="genomic-extraction-of-colibow">Genomic extraction of Colibow</H3><P>Since the colony PCR do not seem to work, this allows to remove one source of problems. Final concentrations:</P><UL><LI>V: 80 ng/ul</LI><LI>VI: 150 ng/ul</LI><LI>VII: 34 ng/ul</LI><LI>VIII: 38 ng/ul</LI></UL><H3 id="pcr-for-checking-the-integration">PCR for checking the integration</H3><P>With the extracted genome as a template.</P><TABLE><THEAD><TR class="header"><TH align="left"><P>Name</P></TH><TH align="left"><P>1x</P></TH><TH align="left"><P>4.5x</P></TH></TR></THEAD><TBODY><TR class="odd"><TD align="left"><P>DT master mix</P></TD><TD align="left"><P>10</P></TD><TD align="left"><P>45</P></TD></TR><TR class="even"><TD align="left"><P>P1</P></TD><TD align="left"><P>1</P></TD><TD align="left"><P>4.5</P></TD></TR><TR class="odd"><TD align="left"><P>P2</P></TD><TD align="left"><P>1</P></TD><TD align="left"><P>4.5</P></TD></TR><TR class="even"><TD align="left"><P>P3</P></TD><TD align="left"><P>1</P></TD><TD align="left"><P>4.5</P></TD></TR><TR class="odd"><TD align="left"><P>P4</P></TD><TD align="left"><P>1</P></TD><TD align="left"><P>4.5</P></TD></TR></TBODY></TABLE><P>Then 14 ul of mix + 6 ul of genomic DNA.</P><P>95 (1') 95 (20) 48 (20) 72 (1') *34 72 (4')</P><P><EM>Gel:</EM> Absolutely nothing! In fact the DNA concentration is waaay too high.</P><P>This PCR was done again by Barth with less template (0.5 ul of DNA).</P><P>[9/1 PCR for integration]</P><P>The gel is difficult to interpret, but it seems like the clone number 8 is integrated and VI and VII are not.</P><H3 id="biobricking-of-the-lox-array">Biobricking of the Lox Array</H3><P>PCR to add the biobrick prefix and suffix:</P><TABLE><THEAD><TR class="header"><TH align="left"><P>Phusion master mix</P></TH><TH align="left"><P>25</P></TH></TR></THEAD><TBODY><TR class="odd"><TD align="left"><P>Primer R</P></TD><TD align="left"><P>1</P></TD></TR><TR class="even"><TD align="left"><P>Primer L</P></TD><TD align="left"><P>1</P></TD></TR><TR class="odd"><TD align="left"><P>Colibow A</P></TD><TD align="left"><P>1 ul</P></TD></TR><TR class="even"><TD align="left"><P>DMSO</P></TD><TD align="left"><P>1.5</P></TD></TR><TR class="odd"><TD align="left"><P>Water</P></TD><TD align="left"><P>20.5</P></TD></TR></TBODY></TABLE><P>98 (30) 98 (20) 50 (25) 72 (15) * 35 72 (10)</P><P><EM>Gel</EM>: 100 bp+, PCR product. [09/01 biobricking] It's very good, and has to be cleaned up.</P><H2 class="date" id="wednesday-0902">Wednesday 09/02</H2><H3 id="vector-for-biobricking">Vector for biobricking</H3><P>pSB1C3-RFP: got 65 ul of miniprep from Mukit.</P><H3 id="purification-of-biobrick-pcr">Purification of Biobrick-pcr</H3><P>Elution in 35 ul of EB. Final titer: 20 ng/ul, clean.</P><H3 id="gibson-assembly-of-colibow-for-top10-e.-coli">Gibson assembly of Colibow for Top10 E. coli</H3><P>Due to the repetitive nature of the Colibow sequence, it seems better to use a Rec- strain with reduced homologous recombination, such as the Top10 strain. A new Gibson reaction was set up.</P><TABLE><THEAD><TR class="header"><TH align="left"><P>Name</P></TH><TH align="left"><P>Concentration</P></TH><TH align="left"><P>Size</P></TH><TH align="left"><P>Volume</P></TH></TR></THEAD><TBODY><TR class="odd"><TD align="left"><P>A</P></TD><TD align="left"><P>80</P></TD><TD align="left"><P>1202</P></TD><TD align="left"><P>0.67</P></TD></TR><TR class="even"><TD align="left"><P>B</P></TD><TD align="left"><P>30</P></TD><TD align="left"><P>909</P></TD><TD align="left"><P>1.35</P></TD></TR><TR class="odd"><TD align="left"><P>C</P></TD><TD align="left"><P>80</P></TD><TD align="left"><P>1004</P></TD><TD align="left"><P>0.56</P></TD></TR><TR class="even"><TD align="left"><P>pIT5-KH</P></TD><TD align="left"><P>118</P></TD><TD align="left"><P>6400</P></TD><TD align="left"><P>2.42</P></TD></TR></TBODY></TABLE><P>This is 0.014 pmol per fragment.</P><H3 id="miniprep-of-pcolibow-plasmid-from-pir116">Miniprep of pColibow plasmid from Pir116</H3><P>After elution in 50 ul of water, the concentrations are not very high (ranging from 13 to 20 ng/ul). A glycerol stock was made from the cultures.</P><P><EM>Gel</EM>: plasmid from colony 1, 2, 3, 4, 5, 6, 1kb ladder, 5 ul of Gibson product. Each time 5 ul of sample was deposited.</P><P>[9.2 Pir pCol]</P><P>It seems there is a plasmid in all colonies except 5. This plasmid is of the same size as the Gibson product, which seemed to work quite well.</P><P><EM>Analytical digest</EM> 6 ul of these plasmids were digested by 1 ul of NotI FD during 30 min at 37°C, in 20 ul total volume.</P><P>However, nothing was visible on the gel ran after digestion (there seems to be something in the ponds, though).</P><H2 class="date" id="transformation-of-top10-strain-with-pcolibow-assembly">Transformation of Top10 strain with pColibow assembly</H2><P>These cells are heat-shock competent, not electrocompetent. As a control, 1 ul of KHx was used. At the same time, a culture of Top10 in 5 ml of LB (with control) was launched. The transformation products were plated on Kan20 plates after 2 h of recovery.</P><H2 class="date" id="biobrick-construction">Biobrick construction</H2><P>The pSB1C3 vector and the insert were digested by EcoR1 and Pst1, during 20 min at 37°C.</P><P>The insert was PCR-purified again to remove the excised tails. Final concentration: 19 ng/ul in 30 ul of water.</P><P>The backbone was gel-extracted on a 0.6 % gel. Unfortunately, the digestion reaction did not work so good, and most of the vector has been cut by only one enzyme. The band at 2029 (double-digested vector) was cut off and purified. Final concentration: 7.2 ng/ul.</P></DIV></DIV></DIV></DIV></BODY></HTML>