Team:Paris Bettencourt/Notebook/VitaminA
Ferment It Yourself
iGEM Paris-Bettencourt 2O15
Background
Design
Project
Micronutrients
Continuity
Modeling
Manufacturing
Achievements
Results
Parts
Medal Fulfillments
Practices
Overview
SynBio Challenge
DIλ
Acceptance
Events
Rhizi
Collaborations
Safety
Safety Overview
Safety Form
Review Board
Notebook
Daily notebook
Protocols
Bibliography
Team
Members
Attributions
Acknowledgments
Press
Official Team Profile
Notebook - Vitamin A
July
August
September
Oligos
July 14th
Goal
Extract the integrative plasmid HO-Poly-KanMX4-HO from the E.Coli provided by AddGene (Accession number #51662).
Procedure
Liquid culture overnight in LB + Ampicillin.
Made a glycerol stock and stored it in the -20 freezer (g15.35)
Centrifuge the tube for 1 minute with 11000 rpm.
Miniprep
Throw out the supernatant
resuspend in 250 uL of resuspension solution in an Eppendorf
Add 250 uL of lysis solution for 2 min then add 350 uL of neutralization solution, and shake it lightly.
10 min of centrifugation at 14k rpm
supernatant is pour in a column and centrifugated for 30 sec at 14k rpm in a column
supernatant is discarded and 700uL of washing solution are added then the column is centrifugated at 14k rpm for 30s
supernatant is discarded and 500uL of washing solution are added then the column is centrifugated at 14k rpm for 30s
supernatant is discarded, the column is centrifugated at 14k rpm for 120s
put the column in another tube and add 45uL of DNAse RNAse free water in the middle of the column
wait 2 min
centri for 2 min at 10k rpm
discard the column
Measure concentration with Nanodrop
Results
Final DNA concentrations of the 4 tubes of miniprep, measured with Nanodrop:
tube HO-Poly-KanMX4-HO pl. 1 = 431.1 ng/uL
tube HO-Poly-KanMX4-HO pl. 2 = 313.6 ng/uL
tube HO-Poly-KanMX4-HO pl. 3 = 366.4 ng/uL
tube HO-Poly-KanMX4-HO pl. 4 = 261.7 ng/uL
July 15th
Goal
Test chromosomal integration in WT yeast SK1 with the integrative plasmid HO-Poly-KanMX4-HO.
Procedure
We followed the method described in "High-efficiency Yeast transformation using liAc/SS carrier DNA/PEG method" (Gietz 2007).
Inoculation of a single colony of the SK1 yeast strain in liquid YPD overnight on a rotatory shaker at 130 r.p.m and 30°C.
After 16 hours the titer of the cell culture was determined. The OD 600 nm of a 1/100 dilution (10 uL in 1 mL) was measured and the cell concentration determined using the formula (1*10^6 cells.mL^-1 will give an OD600nm of 0.1).
2.5*10^8 cells were added to 50 mL of pre-warmed YPD and incubated for 4.5 hours at 30°C and 130 rpm.
1.0 mL of salmon sperm carrier DNA was denaturated in a heat block at 99°C during 5 min, and chilled rapidly in ice.
The cells where harvested by centrifugation at 3000g for 5 min and resuspended in 25 mL of water and centrifuged again 5 min at 3000g. The washing process was repeated again, then the cells were resuspended in 1.0 mL of sterile water.
The suspension was transfered into a 1.5 mL eppendorf tube and centrifuged for 30s at 13,000g and the supernatant discarded
The cells were resuspended in 0.5 mL of sterile water. 100uL of the solution are pipette in a 1.5mL microcentrifuge then the transformation mix has been added (240uL of PEG 3350,36uL of liAc 1.0 M, 50 uL of the single stranded DNA carrier (2.0 mg.mL^-1, 35 uL of DNA plus sterile water up to a total of 360 uL.
tubes were placed at 42°C for 40 min.
tubes were centrifuged at 13 000g for 30s in a microcentrifuge and the supernanant removed with a micropipettor. Pellet was resuspended in YPD and incubate for 3 hours at 30°C to ensure good expression of the antibiotic resistance.
2, 20 and 200 uL of the cell suspension were plated on YPD agar + G418 and spread with glass beads.
plates were put to grow at 30°C for 3 days
Results
We had many transformants, with the resistance marker:
from left to right: dilutions 1/1 1/10 1/100
first line: without g418
second line: with g418
July 22nd
Received gBlocks vA-2, vA-3 and vA-4, which form the last parts of the polycistron, and the corresponding amplification oligos from IDT.
Resuspended gBlocks vA-2, vA-3 and vA-4 in 100 uL water, to reach a final concentration of 10 ng/uL.
Made aliquots of these gBlocks at concentration 1 ng/uL.
Resuspended primers o15.056, o15.076, o15.058, o15.059, o15.060, o15.061 in water to reach a final concentration of 100 uM for each.
Made aliquots of each primer at concentration 10 uM.
PCR amplification, using the following protocol:
1 uL of gBlock (1 ng)
2 uL forward primer
2 uL reverse primer
50 uL Master Mix 2X
45 uL water
We made 2 PCR tubes for each gBlocks, with the following primers:
vA-2 + o15.056 + o15.057
vA-3 + o15.058 + o15.059
vA-4 + o15.060 + o15.061
Settings PCR :
35 cycles amplification, using the following parameters:
time (min)
temperature (°C)
function
0:30
98
melting
0:10
98
melting
0:30
50
annealing
1:00
72
extension
10:00
72
extension
forever
10
storage
After PCR, we migrated the PCR product on TAE 1% agarose gel (100V), to check if the amplification product had the expected size.
From left to right:
100bp+ ladder, vA-2 (two wells), vA-3 (two wells), vA-4 (two wells) amplified at 52°C
We then made a PCR purification protocol using QIAGEN kit:
PCR purification protocol
Add 5 volumes of resuspension buffer to 1 volume of PCR product in an 1.5mL microcentrifuge tube, mix by pipetting up and down
Transfer in a centrifugation column
Centrifuge 2 min at 14000 rpm
Discard flow-through
Add 700μL of washing solution
Centrifuge 30 sec at 14000 rpm
Discard flow-through
Add 500μL of washing solution
Centrifuge 30 sec at 14000 rpm
Discard flow-through
Centrifuge 30 sec at 14000 rpm
Discard flow-through
Put the column in a sterile 1.5 mL Eppendorf tube
Add 45μL of DNAse/RNAse-free water on the membrane
Wait 2 minutes
Centrifuge 2 min at 10000 rpm
Discard column, DNA is saved in water DNA concentration measured with Nanodrop:
Part Name
[PCR product] (ng/μL)
vA-2 (tube 1)
126
vA-2 (tube 2)
97
vA-3 (tube 1)
129
vA-3 (tube 2)
76
vA-4 (tube 1)
202
vA-4 (tube 2)
108
July 28th
Goal
Retrieve TDH3 promoter from Biobrick BBa_K530008.
Procedure
PCR of TDH3:
1 uL of gBlock (1 ng)
2 uL forward primer o15.123
2 uL reverse primer o15.124
50 uL Master Mix 2X
45 uL water
Settings PCR:
30s at 95°C
35 times:
30s at 95°C
30s at 55°C
1m at 72°C
10m at 72°C
the tubes were then kept at 10°C
Results
Nanodrop : 57,9 ng/uL
August 3rd
We made the following PCR :
vA-1.1 + o15.142 + o15.141
vA-1.2 + o15.119 + o15.122
HO-Poly-KanMX4-HO plasmid + o15.135 + o15.143
The elongation time during the pCR was 1 minute for the two gBlocks, and 4 minutes for the plasmid.
August 4th
Results:
We migrated all the PCR products of our gBlocks on a gel .:
The bands for gBlocks vA-1.1, 1.2, 2 and 4 are at the expected position. The band for vA-1.1 is not very bright but still visible. There seems to be some unspecific binding for gBlock vA-3 as we can see two bands, one at the expected size and one smaller. We still decided to go on and try our Gibson Assembly, assuming the smaller DNA sequence wouldn't be that much of a problem as long as the right PCR product of gBlock vA-3 was present too.
DNA concentration was measured with a Nanodrop:
Part Name
[PCR product] (ng/μL)
vA-1.1
84
vA-1.2
22
HO-Poly-KanMX4-HO
25
August 6th
Gibson assembly The goal is to assemble all the parts together to get the plasmid with the 3 genes that are needed to produce beta carotene in Saccharomyces cerevisiae.
Gibson was performed on :
the PCR product of HO-Poly-KanMX4-HO, a plasmid from Addgene
PCR product of the gblock vA-1.1
PCR product of the gblock vA-1.2
PCR product of the gblock vA-2
PCR product of the gblock vA-3
PCR product of the gblock vA-4
5X ISO Buffer was prepared with the following recipe:
3 ml 1M Tris-HCl pH 7.5
+ 150 μl 2 M MgCl2
+ 240 μl  100 mM dNTP mix (25 mM each: dGTP, dCTP, dATP, dTTP)
+ 300 μl 1 M DTT
+ 1.5 g PEG-8000
+ 300 μl 100 mM NAD
dH20 to 6 ml
Prepare 1.2 ml of Gibson assembly master mix as follows:
320 μl 5X ISO Buffer
+ 0.64    μl 10 U/μl T5 exonuclease*
+ 20 μl  2 U/μl Phusion polymerase
+ 160 μl 40 U/μl Taq ligase
+     _ dH20 to 1.2 ml
We took 15 ul of the mix for our Gibson, and stored the rest at -20 C in 15 μl aliquots.  We calculated the required amount of each insert to put on the Gibson mix, and made a separate tube called “gBlock mix” containing the required concentration of each gBlock in 100 uL.
Component (size)
[PCR product]
Quantity required for Gibson
Volume put in “gBlock mix”
vA-1.1 (701 bp)
84 ng/uL
13 ng
15.5 uL
vA-1.2 (654 bp)
120 ng/uL
13 ng
10.8 uL
vA-2 (1560 bp)
126 ng/uL
25 ng
19.8 uL
vA-3 (1530 bp)
129 ng/uL
25 ng
19.4 uL
vA-4 (1584 bp)
108 ng/uL
25 ng
23.1 uL
plasmid HO-Poly-KanMX4-HO (6 kb)
103 ng/uL
100 ng
-
The “gBlock mix” contained 88,6 uL with all the gBlocks, and we added 11.4 uL of water to reach 100 uL.
The final Gibson mix contained:
15 uL Gibson master mix
1 uL HO-Poly-KanMX4-HO plasmid at 100 ng/uL
1 uL “gBlock mix”
3 uL water
Total = 20 uL.
The solution was put at 50°C for 60 minutes, then stored at 4°C overnight.
August 7th
Transformation
We transformed by electroporation an E. Coli (NEB turbo) prepared by Mukit that was kept at -80°C, with our Gibson product, and also with the HO-Poly-KanMX4-HO vector alone to make a control.
The bacteria recovered for 3 hours in SOC media after the electroporation. 100uL of the bacterial cultures were then plated on LB with or without Amp with different concentrations: 10^-1 10^-2 and 10^-3
August 8th
Results
We didn’t have any colonies after the overnight culture of the cells transformed with the Gibson product.  The E. Coli transformed with the HO-Poly-KanMX4-HO vector alone did grow very well (a lawn of bacteria on the plates with the 10^-1 and 10^-2 dilutions and more than 100 CFU on the 10^-3 plate).
Interpretation
What most likely happen is that the Gibson assembly failed, which could be due to the unspecific binding during the PCR of gBlock vA-3, or to secondary structures at the extremities of our gBlocks which could have made the binding difficult.
Or maybe the Gibson Assembly worked, but we shouldn’t have kept the product overnight before transforming E. Coli with it.
Now the plan is to make all the PCR again, to make fresh electro-competent cells, and to Gibson and transform on the same day.
August 10th
Plan of the PCR
gBlock vA-1.1 with primers o15.141 and o15.144
gBlock vA-1.2 with primers o15.119 and o15.120
gBlock vA-2 with primers o15.127 and o15.128
gBlock vA-3 with primers o15.129 and o15.130
gBlock vA-4 with primers o15.131 and o15.132
HO-Poly-KanMX4-HO with primers o15.135 and o15.143
Results of the PCR
gel electrophoresis.
5uL of each sample was mixed with 1uL of loading dye then run on a gel with TAE for 20 min
We can observe the expected bands for most of the gBlocks, except for gBlock vA-3 for which there is no clear band, and gBlock vA-1.1 which has a additional, unexpected band at a lower size, probably due to an unspecific binding of an oligo.
1uL of each sample was nanodroped and the concentrations are shown below:
Part Name
[PCR product] (ng/μL)
vA-1.1
204
vA-1.2
95
vA-2
77
vA-3
77
vA-4
134
HO-Poly-KanMX4-HO
61
Case of the gBlock vA-3
Since the gBlock vA-3 didn't amplify well on the last PCR, we tried to amplify it again, and this time observed 3 bands, showing unspecific binding of the oligos. We performed a PCR purification on this gBlock to take only the DNA in the band at the right size.  The concentration of the PCR product was measured with a Nanodrop:
Part Name
[PCR product] (ng/μL)
vA-3
87
New PCR and gel purification In case there could be a contamination in our aliquot of gBlock vA-3 at 1 ng/uL, we made a new aliquot from the mother solution (10 uL of gBlock vA-3 at 10 ng/uL + 90uL water). 6 tubes of 100 uL were then made following the PCR protocol (with elongation time=1:30 min, enzyme=phusion, 35 cycles, 57°C annealing temperature)
The PCR product was then put on a gel to migrate and extracted using the gel purification protocol .
All the product was then purified using the PCR purification protocol .
The concentration of the DNA was checked using the Nanodrop.
August 13th
We put some E. Coli NEB-Turbo to grow overnight in 2 mL LB, at 37°C with shaking.
August 14th
We made our E. Coli NEB-Turbo electrocompetent with the following protocol:
Preparation of electrocompetent cells protocol
The night before the transformation, start an overnight culture of cells.
5 ml LB.
The day of the transformation, dilute the cells 100X.
100 ml LB.
Grow at 37°C for about 2 hours.
Harvest the cells.
When the cells reach an OD600 of between 0.6 and 0.8.
Split the culture into 2x 50 ml falcon tubes, on ice.
Centrifuge at 4 °C for 10 min at 4000 rpm.
Wash and combine the cells.
Remove the supernatant.
Resuspend the cells in 2x 25 ml of ice cold water.
Combine the volumes in a single 50 ml falcon tube.
Wash the cells 2 more times.
Centrifuge at 4 °C for 10 min at 4000 rpm.
Resuspend in 50 ml of ice cold water.
Repeat.
Wash and concentrate the cells for electroporation.
Centrifuge at 4 °C for 10 min at 4000 rpm.
Resuspend in 1-2 ml of ice cold water.
We will use 200 ul of washed cells per transformation.
Gibson Assembly of the polycistron
We attempted to ligate by Gibson Assembly the parts vA-1.1, vA-1.2, vA-2, vA-3 and vA-4 in the HO-Poly-KanMX4-HO plasmid.  Since we didn't get good PCR products for gBlock vA-3, we decided to take it directly from our mother solution send by IDT, which is at 10 ng/uL. This gBlock already has 30 bp of overlap with vA-2 and vA-4, which may be enough for a Gibson Assembly.
For the other gBlocks and the plasmid, we took their PCR products to have a better concentration and a higher overlap with the neighboring parts (the gBlocks have 30 bp overlap between each other, and the PCR added an additional 30 bp of overlap thanks to the tails of our oligos). So there is 60 bp of overlap between the plasmid and vA-1.1, between vA-1.1 and vA-1.2, between vA-1.2 and vA-2, and between vA-4 and the plasmid.
We diluted the PCR products to have 100 ng of plasmid, and an equimolar concentration of every insert.
Part
Amplified with
Concentration (ng/μL)
Required amount for Gibson (ng)
Dilution
Volume taken for Gibson (uL)
HO-Poly-KanMX4-HO
o15.135
o15.143
61
100
none
2
vA-1.1
o15.141
o15.142
204
13
1/15
1
vA-1.2
o15.119
o15.122
95
13
1/7
1
vA-2
o15.056
o15.057
77
25
1/3
1
vA-3
none
10
from mother solution
25
none
2,5
vA-4
o15.060
o15.061
134
25
1/5
2
All those parts were put together in a tube, and amounted to a total of 8,5 uL. We then poured those 8,5 uL in the tube containing 15 uL of Gibson mix prepared by Ihab.
The mix was put at 50°C for 60 min.
We then made a dialyse of the Gibson product for 15 minutes.
Electroporation
We transformed both our fresh electro-competent NEB-Turbo cells prepared in the morning, and some electro-competent NEB-Turbo cells that had been prepared by Mukit and stored at -80°C, with our dialyzed Gibson product. We used two different kind of cells to see if transformation worked better on fresh cells or not.
We transformed the following with electroporation :
Fresh NEB-T cells + HO-Poly-KanMX4-HO plasmid linearized by PCR
Fresh NEB-T cells + Gibson product
Fresh NEB-T cells + circular HO-Poly-KanMX4-HO plasmid
NEB-T cells from freezer + Gibson product
NEB-T cells from freezer + Gibson product
The cells were then quickly resuspended in 1 mL LB and put at 37°C to recover, then plated in selective media (LB + Ampicillin).
August 15th
Results of the transformation with the Gibson product:
Nothing grew on the plates with cells that had been transformed with the Gibson product.
However we observed colonies for both the cells that had been transformed with the circular plasmid with no insert, and the linearized plasmid with no insert. While it was expected to see colonies for the former since the plasmid contains the Ampicillin resistance gene, it was not expected to see colonies of cells transformed with the linearized plasmid.
Interpretation
The presence of colonies in the plates of cells transformed by the linearized plasmid may be due to background, i.e. to the presence in our sample of circular plasmid that had not been linearized by the PCR.
The same background should then has been observed also in the plates of cells transformed with the Gibson product. But the concentration of plasmid we transformed the cells with was much higher in our controls than with our Gibson products. We should have transformed the cells with the same concentration of plasmid to have a real control.
We still plated the rest of the transformed cells, that had been kept overnight in a liquid culture at 37°C, to see if they would grow.
August 17th
Results of the transformation with the Gibson product:
Both the plates inoculated on August 14th and August 15th with cells transformed with our Gibson product had colonies, even though nothing had grew on the plates inoculated on August 14th after an overnight.
However, the colonies weren't homogeneous on the plates.
Interpretation Rather than multiple colonies of cells successfully transformed with our plasmid containing the insert, it looks like a maximum of 10 colonies on each plate were actually resistant to Ampicillin thanks to the bla gene, that they secreted beta-lactamase which hydrolyses and inactivates Ampicillin, and that this beta-lactamase diffusing into the medium allowed satellite colonies non possessing the plasmid to grow around the resistant colonies. This phenomenon is well-known when using Ampicillin selection.
There was still a possibility that the colony in the center of the cloud of satellites did possess the plasmid with our inserts, so we took 12 samples of those colonies from the plates - they had a bright white color, which distinguished them from the more grayish satellites - and put them into liquid LB + Ampicillin at 37°C overnight.
Ligation by PCR
Since the colonies we got on the plates weren't quite the ones expected - we expected colonies homogeneously spread on the plates - and looked like they could be contaminations, we tried to ligate our gBlocks this time with PCR . To isolate where a problem might be in the ligation, we cloned the parts two by two, three by three and all five together. However none of those worked: we ran a gel after the different PCR, and observed no band.
August 18th
Analysis of the colonies
After an overnight culture in liquid medium (LB + Ampicillin), the above colonies were miniprepped . We then cloned the plasmid and performed an analytical digestion on it, but it was hard to determine for sure from the gel whether the ligation/transformation had failed or not.
Gibson Assembly gBlocks two by two
Since there was a high chance our Gibson with all our gBlocks in the HO-Poly-KanMX4-HO plasmid didn't work, we tried to identify where the problem was by assembling only two parts at a time with Gibson Assembly, and see which parts wouldn't assemble. However, none of them worked.
August 19th
Sequencing
We sent the plasmid we got from mini-prepping the colonies resulting from our Gibson Assembly, to have final answer on whether or not they were the expected plasmid, or a contamination.
We followed GATC's instructions and sent the following:
5 uL purified plasmid at 90 ng/uL
5 uL primer at 5 uM
with primers o15.117, o15.118, o15.119 and o15.120.
August 21st
Sequencing results
GATC sequencing results arrived: in most of the tubes we had sent, 0 bp had been found with the primers we had sent. For only one tube we got a result of 150 bp, but the sequence had nothing to do with the sequence we expected.
It means our Gibson Assembly had failed, and the colonies that had grew and allowed satellites to grow were contaminations resistant to Ampicillin.
The HMG-CoA reductase gene
We finally received our HMG gBlock, with the CDS of the HMG-CoA reductase gene from the red yeast S. aureus, codon-optimized for S. cerevisiae, and synthesized by IDT. More information about this gene can be found here .
The gBlock was centrifuged 5 seconds at 3000 rpm, then resuspended in 100 uL of water to reach a concentration of 10 ng/uL.
The tube was mixed by briefly vortexing it, then put 20 minutes at 37°C accordingly to IDT's instructions. We then vortexed it again and centrifuged it, and made an aliquot at 1 ng/uL.
PCR
We made a PCR of the HMG gBlock. Two tubes were prepared:
1 ng gBlock HMG, with primers o15.137 and o15.138
10 ng gBlock HMG, with primers o15.137 and o15.138
We wanted to check whether a PCR had a better yield with more initial DNA template than 1 ng, as suggested by our advisors, which is why we tried with both 1ng and 10ng of initial concentration of template DNA.
The primers used on this PCR had tails containing the restriction sites of XbaI (on the 5' primer), and XhoI (on the 3' primer).
The elongation time was 2 minutes.
Results of the PCR:
We purified the PCR products and measured their concentration with a Nanodrop:
Part Name (initial amount)
[PCR product] (ng/μL)
HMG (1 ng)
4
HMG (10 ng)
300
For some reason the tube with a low concentration of HMG gBlock was hardly amplified at all, even though we had previously often managed to have a good amplification on 1 ng of DNA. Since we had a good yield in the second tube, we kept it for the following experiments.
p406ADH1 plasmid
Our HMG gBlock only contains the CDS of the gene. To add a strong yeast promoter (ADH1) and a terminator (CYC1) to this gene, we planned to insert it in the p406ADH1 vector which we got from AddGene ( Plasmid #15974 ). The plasmid was sent to us in a DH5alpha E. Coli.
We streaked this E. Coli containing the p406ADH1 plasmid on a plate with LB and Ampicillin and let it grow overnight at 37°C.
August 22nd
p406ADH1 plasmid
We took 2 isolated colonies of the E. Coli containing the p406ADH1 plasmid from our plate, and put them in a liquid culture of LB + Ampicillin at 37°C.
August 23rd
p406ADH1 plasmid
For some unknown reason, only one of the two liquid cultures grew overnight.
We miniprepped the bacteria that did grow, and measured the final concentration of p406ADH1 vector with a Nanodrop:
Plasmid Name
[Miniprep product] (ng/uL)
p406ADH1
200
August 24th
Digestion p406ADH1 plasmid and HMG gBlock
We digested p406ADH1 and the HMG gBlock with the restriction enzymes XbaI and XhoI. Those enzymes should cut the plasmid between the promoter ADH1 and the terminator Cyc1, and should cut the gBlock on both its extremities since we added the two restriction sites by PCR.
Digestion of p406ADH1:
Preparation of a 120 uL mix with:
15 uL plasmid (to have 3 ug)
4 uL XbaI
4 uL XhoI
4 uL FastAP
12 uL green Buffer FD
81 uL water (to reach 120 uL)
Digestion of HMG gBlock:
Preparation of a 80 uL mix with:
10 uL gBlock (to have 3 ug)
4 uL XbaI
4 uL XhoI
8 uL Buffer FD
54 uL water (to reach 80 uL)
Both mix were put at 37°C for 45 minutes.
We then PCR purified the digested HMG gBlock, and measured its concentration with a Nanodrop:
Part Name
[PCR product] (ng/uL)
digested HMG
50
We performed a gel electrophoresis. on the digested p406ADH1 plasmid, and obtained the following bands:
PCR gBlock 3 with primers o15.129 and o15.130 3 tubes with concentrations of gBlock of 1ng 1ng and 10ng
August 25th
After gel electrophoresis. the DNA is Gel Purified . The Nanodrop measurement then revealed that everything has been lost during the PCR purification .  Another PCR is launched with all the gBlocks and the right tails.
plan of the PCR
gBlock vA-1.1 with primers o15.141 and o15.142
gBlock vA-1.2 with primers o15.119 and o15.120
gBlock vA-2 with primers o15.127 and o15.128
gBlock vA-3 with primers o15.129 and o15.130
gBlock vA-4 with primers o15.131 and o15.132
HO-Poly-KanMX4-HO with primers o15.135 and o15.143
We also tried a Gibson Assembly again with the amplified gBlocks vA-1.1, 1.2, 2, 4 and the HO-Poly-KanMX4-HO vector ... which failed miserably.
on the right: 1kb ladder
on the left: Gibson product
no band at 1500 bp is visible (the exposition time is 5s)
August 26th
result of the PCR of the gBlocks (from left to right) vA-1.1, 1.2, 2, 3, 4, HO-Poly-KanMX4-HO and the last column is the ladder 1kb
The PCR for the gBlock 1.2, 2 and 4 worked well.
We are doing again the PCR for the gBlock 1.1, 3 and HO
The PCR with the gBlock 3 is done with 10ng of DNA
from left to right: gBlock 1.1, 3 (1ng), 3 (10ng), HO, 1kb ladder
The results are showing traces of unspecific binding for 1.1 and 3. The HO-Poly-KanMX4-HO band is alsmost invisible, predicting a low DNA concentration.
August 27th
Miniprep of the overnight cultures of the small and big colonies obtained after transformation.
Following the miniprep an analytical digest is performed with Pst1 to linearise the plasmid and then by Nde1 which is supposed to cut the plasmid in 2.
During the digestion the miniprep DNA concentration is recorded with a Nanodrop
colony which was grown for the miniprep
S1
S2
S3
S4
B1
B2
B3
B4
Nanodrop concentration
(ng.ul-1)
84
176
182
183
128
216
170
235
Then the results are tested with a gel electrophoresis .
GEL 1
from left to right 1kb ladder, small colony 1,2,3,4, big colony 1,2,3,4, small colony digested by Pst1 1,2,3,4, big colony digested by PST1 1,2,3,4
GEL 2
from left to right 1kb ladder, small colony digested by Nde1 1,2,3,4, big colony digested by Nde1 1,2,3,4
August 28th
PCR on the minipreped plasmid.
PCR with the oligos o.15.193 and o.15.194 and 10 ng of plasmid DNA.
The PCR product are then revealed with a gel electrophoresis.
from left to right: 1kb ladder thermoscientific, PCR of the plasmid on 4 different colonies(s2, b2, s4, b4)
The brightest band (at 2kb)is probably the band that we were expecting after PCR (the DNA piece that i supposed is amplified is composed of the promoter, the terminator and the insert).
August 29th
PCR on p406ADH1 with and without HMG
We made a gradient PCR on the plasmid p406ADH1 that was ligated with the HMG gBlock, as well as a PCR on the p406ADH1 without the insert. The primers o.15.193 and o.15.194 are located around the insert.
The expected bands should be at 1.3 kb for p406ADH1 + HMG, and at 0 kb for p406ADH1 without insert.
August 31th
PCR of the HMG gBlock with 1DMSO 2noDMSO
September 1st
Analytic digestion p406ADH1
The previous results made us suspicious about the actual length of the p406ADH1 plasmid we received from AddGene. So we performed an analytic digestion on it to see if we had bands at the expected size.
We digested the plasmid with:
PstI: expected bands at 2kb and 4 kb
XbaI: expected band at 6kb
XbaI + SacI: expected bands at 1.5kb and 4.5 kb
XbaI + XhoI: expected band at 6kb
BpiI: expected band at 2kb and 4kb
The Gene Ruler is 1kb from ThermoFischer.
The mix was put 30 min at 37°C for the digestion, then 5 min at 83°C to inactivate the enzymes.
Results
On this gel we can observe:
PstI: no bands
XbaI: bands at 8kb and >10 kb
XbaI + SacI: bands at 8kb, 6kb, and maybe 1.5 kb
XbaI + XhoI: band at 8 kb
BpiI: band at 5 kb and maybe 8 kb
September 2nd
Because of the inconsistency of the results regarding the nature of the plasmid p406ADH1 it was sent to sequencing with 2 different oligos.
PCR HO-KanMX4 plasmid and gBlocks vA-1.1, 2, and 3, with longer oligos
Since we have trouble with PCR, with often no bands or strong unspecific bands when we try to amplify the HO plasmid and the gBlocks vA-1.1, 2 and 3, we designed new oligos that were longer. It should at least reduce the unspecific binding. We performed the following PCR :
vA-1.1 + o15.227 + o12.228
vA-2 + o15.230 + o12.231
vA-3 + o15.232 + o12.233
HO-Poly-KanMX4-HO + o15.234 + o12.235
The annealing temperature was 57°C.
September 3rd
The results of the sequencing are showing that the plasmid that we are minipreping from the strain from addgene is indeed p406ADH1.
Results PCR with longer oligos
We migrated our PCR products that were amplified with longer oligos on a gel (see on the right).
We can only see a band (not very bright) for the gBlock vA-3. For once, there was no unspecific binding and we do not observe other bands at smaller size for this gBlock!
However, after we PCR-purified it, all the DNA was lost as the Nanodrop said the concentration was 0.
There was no band for the plasmid nor the gBlocks vA-1.1 and vA-2.
We launched those same PCR again, but this time we added 3% of DMSO in each tube, and lowered the annealing temperature to 51°C.
Digestion, ligation and transformation of the HMG gene in DH5alpha et NEBT
The digestion was performed on 300ng of HMG (PCR product) and on 100ng of plasmid P406ADH1 (miniprep product) by XhoI and Xba1.
25uL of both products were PCR purified and 25uL were kept unpurified.
2 ligations were performed.
The first one with the HMG and vector p406ADH1 digested and unpurified and the other one with the digested and purified HMG and p406ADH1.
The ligation was left for 1 hour at room temperature (18°C)
Dialysis was performed on all the ligation product for 15 min.
5uL of ligation products were then transformed by electroporation with 100uL of NEBT and DH5alpha.
Controls: NEBTurbo cells and Dh5alpha cells were also transformed with : undigested p406ADH1, p406ADH1 digested with Xba1 and xho1, nothing.
Cells were put to recover for 2 hours at 37°C.
The cells were then plated on LB+ampicillin.
September 4th
resultats du gel que arthur va envoyer.
gBlock VA1.1 and VA3 are showing no sign of amplification.
PCR again with the gBlocks VA1 and VA3 with a gradient PCR (50°C-55°C-60°C).
Here is the result of the transformation of september 4th.
The negative controls are free of colonies and the we have a lot of colonies in the positive controls.
We have a few colonies in the plates with bacteria transformed with the ligation product.
6 colonies are put to grow in liquid culture over night in LB + ampicillin.
September 5th
We performed a miniprep on 6 overnight cultures of the transformants, which we called C1, C2, C3, C4, C5 and C6.
Analytical digestion is then performed on the miniprep product with XbaI/XhoI (2 bands expected at 1.3 kb and 6 kb), and XbaI alone (1 band expected at 7.3 kb).
Here are the results of the gel electrophoresis :
We can observe very bright bands at 6 kb on all the digested minipreps, which is the size of the p406ADH1 plasmid without the insert. So the cells only grew because of the background (non digested plasmid), but it seems none of them has our gene. There are some bands higher than 6 kb on some wells, but they are probably due to circular plasmid that were not cut during the analytical digestion.
September 6th
The truncated gBlock
We finally noticed that one of the gBlock that we received from IDT (gBlock vA-1.2) doesn't have the sequence we ordered... It is truncated, and 16 bp are missing on the 3' end. We found out by comparing the sequence we had ordered and that was also written in a confirmation email from IDT, to the Fasta sequence available on the website. IDT admitted to their mistake.
We strongly suspect that these missing 16 bp are the reason (or at least one of the reasons) why several of our experiments failed. Indeed without those 16 bp, the 3' primer of this gBlock had little chance to bind (it had only 12 bp in common with the actual gBlock we received, with only 3 G/C). Also our 3' primer actually contained a tail that had 30 bp in common with the gBlock vA-2, for the Gibson Assembly. Without this, the Gibson was sure to fail.
We did observed bands at the right size when we amplified this gBlock during the summer though, but we suspect that the gBlock was always amplified by the 5' primer only.
So we ordered a new 3' primer, with a tail that put back the missing 16 bp.
Biobrick the HMG gene
In order to put the coding sequence of the HMG gene in the iGEM plasmid (pSB1C3), we amplified the HMG-gBlock with oligos that had the XbaI and SpeI restriction sites on their tails.
PCR :
HMG-gBlock + o15.137 + o15.192
September 7th
Attempt n°785464153 for the digestion, ligation and transformation of the HMG insert in the p406ADH1 vector.
A PCR is performed on HMG-gBlock with the oligos o15.138 and o15.137.
A PCR-Purification is performed on the PCR product.
The digestion of HMG and of the vector are performed with Xba1 and Xho1 (with no FAST AP or FAST AP buffer for the insert).
The digestion product are PCR purified to remove the small pieces of DNA (<60).
The products are then following a Ligation with 100ng of plasmid and 75ng of the insert(the molar ratio is plasmid 1:insert 3).
Electroporation .
5uL were added to a cuvette with 100uL of cells and the electroporation made an arc so i reduced the volume of DNA to 2uL and i had a poor time for the electroporation(4.3). I then decided to do a dialysis for 15 min before the 2 last transformations.
After dialysis i had some good time values (5.2 and 5.3).
The controls are : cells transformed with p406ADH1 digested with xba1 and xho1. Cells transformeds with he circular p406ADH1.
September 8th
after an overnight culture the cells transformed with the ligation product HMG/p406ADH1 have grown. Both negative and positive controls are ok. We are doing a colony PCR on 24 of the colonies that grew.
We are also doing the digestion again on the plasmid to be ready to redo the ligation transformation. After digestion of the plasmid it is run on a gel with the undigested plasmid. It seems that the digestion went well:
A culture of the E.coli that contains p406ADH1 in LB+ampicillin.
PCR on gBlocks vA-1.2, and 1.1-1.2 together
We found out that the reason many PCR and assembly didn't work is because the gBlock vA-1.2 that we've had synthesized by IDT is actually truncated. 16 bp were missing on the 3' end, and thus our 3' primers never bound to our gBlock. We had still observed clear bands when we had amplified it before, because it was (most probably) amplified by the 5' primer alone.
So we ordered a new primer that put back the missing 16 bp, and launched several PCR with it: one on the gBlock vA-1.2 alone, and one with vA-1.1 and 1.2 together (with the 5' of 1.1 and the 3' of 1.2).
Results
September 12th
A miniprep is performed on all the colonies that were showing a band at 1300 bp on the colony PCR of the 7 of september to extract any plasmid that could possibly got the insert even if the other PCR and digestion are showing no signs of it. A PCR is then performed:
primers 137 & 138
miniprep product
gBlock HMG
primers 253 & 254
miniprep product
miniprep p406ADH1
A gibson is also performed on the 5 parts with the intent to PCR the eventual results.
September 15th
SK1 and beta carotene producing yeasts are put to grow in YPD and in minimal media.
Oligos
o15.056
TCACAACAAGAATTTTGCAGAGGACCTAACCGAAGGAAAGTTTTCCTTCCCAACAATC
o15.057
GGACTTGATAGAATTTTGTCTGGTTGACTCGGATGCAATGGAGGACCGAACAAT
o15.058
AACGTTATTGTTCGGTCCTCCATTGC
o15.059
GTCCAGATTGACTCGAATGGGTGTAATGCCAGTATTTGACCAATGAATTGTAGCCTCAAG
o15.060
CTTGAGGCTACAATTCATTGGTCAAATAC
o15.061
TGTACGGGCGACAGTCACATCATGCCCCTGCAGGGTACCGCGAATTCCCC
o15.117
CGTCCAAGACATACTGCGTTTACG
o15.118
GTGTGCAACATTCCCACTACATTTTGAATG
o15.119
CATTCAAAATGTAGTGGGAATGTTGCAC
o15.120
TGGATTGTTGGGAAGGAAAACTTTCCTT
o15.122
AGCCTGGAGGAAGGGTTAGCATGGATGGAATGGATTGTTGGGAAGGAAAACTTTCCTT
o15.123
GCCCAGAATACCCTCCTTGACAGACAGTTTATTCCTGGCATCCACTAAATATAATG
o15.124
TATACTGCAGTTTGTTTGTTTATGTGTGTTTATTCGAAACTAAGTTC
o15.127
TCACAACAAGAATTTTGCAGAGGACCTAACCGAAGGAAAGTTTTCCTTCCCAACAATC
o15.128
GGACTTGATAGAATTTTGTCTGGTTGACTCGGATGCAATGGAGGACCGAACAAT
o15.129
AACGTTATTGTTCGGTCCTCCATTGC
o15.130
GTCCAGATTGACTCGAATGGGTGTAATGCCAGTATTTGACCAATGAATTGTAGCCTCAAG
o15.131
CTTGAGGCTACAATTCATTGGTCAAATAC
o15.132
TGTACGGGCGACAGTCACATCATGCCCCTGCAGGGTACCGCGAATTCCCC
o15.135
TCGCTATACTGGGGAATTCGCGGTACCCTGCAGGGGCATGATGTGACTGTCG
o15.137
TATATCTAGACCCGCCGCCACCATGCAAAG
o15.138
TATACTCGAGTCACTGTTGAGACCTCAAATCCTGTAAG
o15.141
TCCTCAACGTCGTCCATCAGTAAGCTCGCTGTGTGCAACATTCCCACTACATTTTGAATG
o15.142
GCCCAGAATACCCTCCTTGACAGCGTCCAAGACATACTGCGTTTACG
o15.143
TCGACACGTAAACGCAGTATGTCTTGGACGCTGTCAAGGAGGGTATTCTGGGC
o15.144
GTTTCGAATAAACACACATAAACAAACAAATCTAGAACTAGTGGATCCATGGATTAC
o15.192
TATAACTAGTTCACTGTTGAGACCTCAAATCCTGTAAG
o15.193
TATACTGCAGCCCGCCGCCACCATGCAAAG
o15.194
TATACTGCAGTCACTGTTGAGACCTCAAATCCTGTAAG
o15.227
CCGGGTTAATTAAGGCGCGCCAGATCTGTTCGTCCAAGACATACTGCGTTTACGTGTCG
o15.228
AACGTCGTCCATCAGTAAGCTCGCTGTGTGCAACATTCCCACTACATTTTGAATGACTTC
o15.230
CGAAGGAAAGTTTTCCTTCCCAACAATCCATTC
o15.231
GGACTTGATAGAATTTTGTCTGGTTGACTCGGATGCAATGGAGGACCGAACAATAACGT
o15.232
CATCCGAGTCAACCAGACAAAATTCTATCAAGT
o15.233
GATTGACTCGAATGGGTGTAATGCCAGTATTTGACCAATGAATTGTAGCCTCAAGAATGC
o15.234
TCGCTATACTGGGGAATTCGCGGTACCCTGCAGGGGCATGATGTGACTGTCGCCC
o15.235
TCGACACGTAAACGCAGTATGTCTTGGACGAACAGATCTGGCGCGCCTTAATTAACCC
o15.253
CGTATACTGCAGGCGCAATTAACCCTCACTAAAGGG
o15.254
GTCACTCTGCAGCTCACTATAGGGCGAATTGGGTAC
