<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Project_Phytase skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Paris Bettencourt/Project/Phytase</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="menuContainer"><DIV id="smallMenuTitle"><H1>Ferment It Yourself</H1><H3>iGEM Paris-Bettencourt 2O15</H3></DIV><UL><LI class="left"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Background" title="Background">Background</A></LI><LI class="left"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Design" title="Design">Design</A></LI><LI class="left"><BUTTON class="menuButton">Project</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Micronutrients">Micronutrients</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Continuity">Continuity</A></LI><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Manufacturing">Manufacturing</A></LI></DIV></UL><LI class="left"><BUTTON class="menuButton">Achievements</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Parts">Parts</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Medals">Medal Fulfillments</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Practices</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/SynBio_Challenge">SynBio Challenge</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/DILambda">DIλ</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Acceptance">Acceptance</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/Events">Events</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/Rhizi">Rhizi</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Collaborations">Collaborations</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Safety</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Safety">Safety Overview</A></LI><LI><A href="https://2015.igem.org/Safety/Final_Safety_Form?team_id=1678">Safety Form</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/ReviewBoard">Review Board</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Notebook</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Notebook">Daily notebook</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols">Protocols</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Bibliography">Bibliography</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Team</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Team">Members</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Acknowledgments">Acknowledgments</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Press">Press</A></LI><LI><A href="https://igem.org/Team.cgi?id=1678">Official Team Profile</A></LI></DIV></DIV><DIV id="riboflavin"><DIV class="contentContainer noBanner"><DIV id="banner"><H3>Phytase
</H3></DIV></DIV><DIV class="textWrapper"><DIV class="textContent black" id="caffeine"><DIV id="overview" style="text-align: center"><TABLE><TBODY><TR><TH class="noBottom"><H3>Background</H3></TH><TH class="noBottom"><H3>Aims</H3></TH><TH class="noBottom"><H3>Results</H3></TH></TR><DIV id="overview" style="text-align: justify"><TR><TD class="noTop"> The yeast <I>Saccharomyces cerevisiae</I> naturally produces phytase. 
      </TD><TD class="noTop"> We engineered <I>Saccharomyces cerevisiae</I> to produce higher amounts of phytase during the fermentation of idli batter to increase iron bioavailability.
      </TD><TD class="noTop"> Two repressor genes of the phytase biosynthesis pathway in <I>Saccharomyces cerevisiae</I> were successfully deleted to increase phytase production.
      </TD></TR></DIV></TBODY></TABLE></DIV></DIV><DIV class="textContent"><H2>Introduction</H2><DIV class="column-left"><P>
Anemia affects one third of the world's population and is mostly due to insufficient iron intake. Anemia and similar mineral deficiency diseases are primarily widespread in developing countries like India, partly resulting from the local diet that is mainly made up of cereal grains and seeds such as rice . In these types of food, iron bioavailability is substantially reduced by the presence of phytic acid (C<SUB>6</SUB>H<SUB>18</SUB>O<SUB>24</SUB>P<SUB>6</SUB>), which chelates minerals and forms insoluble salts which precludes their absorption in the gastrointestinal tract. </P><P>
Current research on increasing the bioavailability of iron or zinc involves the bioengineering of crop plants, which not only poses challenges in terms of the production of efficient genetically modified crops, but also requires extensive research for drawing any conclusions on strain sustainability. </P><P>
We propose an alternative strategy that focuses on the bioengineering of microorganisms involved in the fermentation of idli, a dish widely used as primary food source throughout much of India. Indeed, the lab model organism <I>Saccharomyces cerevisiae</I> is a strain present in the idli microbiome that naturally produces phytases. Phytases are enzymes that are able to hydrolyze phytic acid even when complexed with minerals, resulting in a greater mineral bioavailability.</P><P>
However the production of phytases in <I>Saccharomyces cerevisiae</I> is down-regulated by two genes: PHO80, present on chromosome 15 and PHO85, found on chromosome 16. The knockout of these genes would likely increase the yield of phytase production and therefore increase the general bioavailability of minerals in fermentation-based dishes such as idli.</P></DIV><DIV class="column-right"><P class="legend"><B>Figure 1:</B> Phytic acid in complex with calcium, magnesium, zinc and iron</P></DIV><DIV class="column-right"><P class="legend"><B>Figure 2:</B>Phytase hydrolyzes phytic acid.</P></DIV><H2>Experimental design</H2><DIV class="column-left"><P>We used a strain of <I>Saccharomyces cerevisiae</I>(SK1), a gift of the INSERM unit U1001. We designed a set of primers to delete the two phytase synthesis repressor genes PHO80 and PHO85 on chromosomes 15 and 16. We amplified the geneticin resitance gene (G418) from the pSB1C3 plasmid with primers designed to add 2 homology regions in order to insert the resistance gene inside the genes we wanted to delete.</P><P>After successful transformation and selection (fig5), we measured whether the level of phytase production was increased. This was done through the use of a quantification colorimetry kit, which measures the OD655 to determine how much phytic acid is found in different growth conditions. 

In order to do this, we used the kit to quantify the levels of phytic acid in rice, fermented rice and idli at different stages of its preparation using our different strains.</P></DIV><H2>Results</H2><P>The different tranformations were successful, as demonstrated through gel electrophoresis (fig5) and by the resistance to G418.
The kit that we used to titrate phytate was inefficient. It worked for none of our samples (giving unrealistic values).</P><DIV class="column-left"><P> In Table 1, we presenting a sample of the type of data that we obtained after using the kit. The values are very low in comparison with what is expected for such food samples. (for rice is is supposed to be equal to 6 mg.g<SUP>-1</SUP>, and here we observe a concentration of 5 µg.g<SUP>-1</SUP>, 10<SUP>-3</SUP> less).</P></DIV><DIV class="column-right"><SPAN class="caption"><B>Table 1:</B> Typical data obtained from the phytic acid titration kit.</SPAN></DIV><DIV class="column-right"><SPAN class="caption"><B>Figure 3:</B> Titration of phytic acid for different strains and food samples, under different conditions of growth. The strains X, Y and Z are the wild type strains used in the formation of the flour cube that we designed (see manufacturing).</SPAN></DIV><DIV class="column-left"><P> Figure 3 shows the variation of OD655nm before and after growth with different strains/food samples.The results do not show any significant decrease in concentration after growth or even between samples that were expected to present different levels of free phosphorus.</P></DIV><DIV class="column-right"><SPAN class="caption"><B>Figure 4:</B> Comparison of growth of different strains on YPD.</SPAN></DIV><DIV class="column-left"><P> The results for the comparative growth experiment show no significant decrease between the growth rate of the yeast before and after deletion of the gene (fig4).</P></DIV><DIV class="column-left"><SPAN class="caption"><B>Figure 5:</B> Electrophoresis gel for the transformation of pho80 and pho85.</SPAN></DIV><H2>Bibliography</H2>

Veide, J. &amp; Andlid, T. Improved extracellular phytase activity in Saccharomyces cerevisiae by modifications in the PHO system. International Journal of Food Microbiology 108, 60-67 (2006).



                      </DIV></DIV></DIV></BODY></HTML>