<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Notebook_Differentiationyeast skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Paris Bettencourt/Notebook/Differentiationyeast</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="menuContainer"><DIV id="smallMenuTitle"><H1>Ferment It Yourself</H1><H3>iGEM Paris-Bettencourt 2O15</H3></DIV><UL><LI class="left"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Background" title="Background">Background</A></LI><LI class="left"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Design" title="Design">Design</A></LI><LI class="left"><BUTTON class="menuButton">Project</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Micronutrients">Micronutrients</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Continuity">Continuity</A></LI><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Manufacturing">Manufacturing</A></LI></DIV></UL><LI class="left"><BUTTON class="menuButton">Achievements</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Parts">Parts</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Medals">Medal Fulfillments</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Practices</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/SynBio_Challenge">SynBio Challenge</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/DILambda">DIλ</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Acceptance">Acceptance</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/Events">Events</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/Rhizi">Rhizi</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Collaborations">Collaborations</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Safety</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Safety">Safety Overview</A></LI><LI><A href="https://2015.igem.org/Safety/Final_Safety_Form?team_id=1678">Safety Form</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/ReviewBoard">Review Board</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Notebook</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Notebook">Daily notebook</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols">Protocols</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Bibliography">Bibliography</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Team</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Team">Members</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Acknowledgments">Acknowledgments</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Press">Press</A></LI><LI><A href="https://igem.org/Team.cgi?id=1678">Official Team Profile</A></LI></DIV></DIV><DIV id="notebook"><DIV class="contentContainer"><DIV id="banner"><H3>Notebook - Differentiation in Yeast
</H3></DIV></DIV><DIV class="textWrapper"><DIV class="textContent {{{page_option}}}" id="caffeine"><DIV id="notebookMenu"><UL><LI><A href="#july">July</A></LI><LI><A href="#august">August</A></LI></UL></DIV><DIV id="notebookContent"><H2 class="date one" id="wednesday-0715">Wednesday 07/15</H2><P>Started to work on the packaging experiments for yeast.</P><P>Innoculation of S. cerevisiae BY4743 mCherry</P><P>Medium: 100 ml YPD with 400 ul Gentamycine (four 25 ml tubes)</P><P>Culture @ 30°C + Shaking</P><H2 class="date one" id="friday-0717">Friday 07/17</H2><P>Checked the yeast megaculture: growth is okay, but let’s wait for a third day to have more biomass.</P><P>Designed and ordered a lot of oligos for Yeastbow and Colibow projects.</P><H2 class="date one" id="saturday-0718">Saturday 07/18</H2><H3 id="manufacturing-experiment-part-1.">Manufacturing experiment, part 1.</H3><P>A strain of Yeast expressing mCherry was grown during three days at 30°C in 100 ml YPD.</P><P>In order to remove as much medium as possible, it was centrifuged at 4000 G during 5 min, twice, while discarding the supernatant each time.</P><P>Different conditions of drying were attempted:</P><UL><LI>In the tube, at the sun during 30 min (Tube1). This one was not completely dry when packaged.</LI><LI>Spread on tinfoil (Grattée)</LI><LI>Put on tinfoil without spreading (Air libre)</LI><LI>Put on tinfoil and covered with a plastic plate (Boîte)</LI><LI>Leftover in the tube (Tube2)</LI></UL><P>After drying these samples were packaged in paper bags and preserved in the fridge.</P><H2 class="date one" id="monday-0720">Monday 07/20</H2><P>Received oligos o15.74 and o15.75. They’re meant to amplify the Yeastbow source plasmids.</P><P>Prepared 9 YPD agar plates.</P><H3 id="manufacturing-experiment-part-2.">Manufacturing experiment, part 2.</H3><P>For each drying protocol, one sample was taken out of the fridge.</P><P>They were weighted, transferred in eppendorf tubes and ~1ml of LB was added (YPD not available at that time).</P><P>The « tube 1 » sample was not dry enough so a lot of it sticked to the package.</P><TABLE><TBODY><TR class="odd"><TD align="left"><P>ID</P></TD><TD align="left"><P>Boîte 2</P></TD><TD align="left"><P>Tube 1</P></TD><TD align="left"><P>Air libre 1</P></TD><TD align="left"><P>Grattée 1</P></TD></TR><TR class="even"><TD align="left"><P>Weight</P></TD><TD align="left"><P>12 mg</P></TD><TD align="left"><P>4.5 mg</P></TD><TD align="left"><P>9.2 mg</P></TD><TD align="left"><P>1.3 mg</P></TD></TR></TBODY></TABLE><P>This was then moved to YPD-agar plates and spread with sterile beads.</P><P>The plates were cultured at 30°C overnight for colony counting.</P><H2 class="date one" id="thursday-0716">Thursday 07/16</H2><P>Ordered two new oligos for Yeastbow cloning: o15.74 and o15.75</P><P>Yeast megaculture for wednesday: A little cell pellet is visible at the bottom.</P><P>-&gt; Titled the flasks so they’re shaked better.</P><P>Obtained two yeast strains that express CRE recombinase from Zoran, overnight at 30.</P><P>Designed and ordered new primers for yeastbow PCRs, based on Matt’s advice.</P><P>It was very difficult to find a good priming region since this seems to be in a terminator.</P><H2 class="date one" id="monday-0720-1">Monday 07/20</H2><H3 id="reception-of-pbrainbow-1.0-and-3.0-propogators-as-glycerol-stocks">Reception of pBrainbow 1.0 and 3.0 propogators as glycerol stocks</H3><P>Templates for PCR assembly of Yeastbow. Still need primers.</P><P>-&gt; Overnight culture for miniprep for PCR</P><UL><LI>pThy Brainbow 1.0</LI></UL><H3 id="reception-of-oligos-o15.74-and-o-15.75">Reception of oligos o15.74 and o 15.75</H3><P>For yeastbow first round of PCRs.</P><P>Reconstitution in dH2O, in the bottom drawer of the -20.</P><H3 id="reception-of-pbrainbow-ura3-for-yeastbow">Reception of pBrainbow URA3 for Yeastbow</H3><P>From Zack at Hersen lab.</P><P>It carries an AmpR.</P><P>Use 1 ul as a template for the PCR, and if it doesn’t work transform the rest into E. coli for propagation.</P><P>Stored in Barth’s styrofoam box.</P><H2 class="date one" id="tuesday-0721">Tuesday 07/21</H2><H3 id="results-for-the-preliminary-manufacturing-experiment">Results for the preliminary manufacturing experiment</H3><P>We get a lot of background. It’s almost sure that it is yeast, but no mCherry could be detected. The plasmid was likely lost due to the absence of antibiotics in the plating media.</P><P>It’s important to dilute before plating to know the number of colonies per mg.</P><P>All the plates are covered in yeast, even the less successful ones.</P><H3 id="reception-of-oligoso15.76-to-o15.96">Reception of oligos o15.76 to o15.96</H3><UL><LI>Yeastbow SOE</LI><LI>R6K linearization</LI><LI>Colibow gBlock Amp</LI><LI>Colibow Sequencing</LI></UL><P>The ultramers (o15.78, o15.79) for Leu2 flanking are not ready yet.</P><P>-&gt; Oligo reconstitution and dilution</P><P>-&gt; Miniprep of R6K vector</P><P>-&gt; PCR of R6K vector</P><P>-&gt; Miniprep of pBrainbow 1.0</P><P>-&gt; PCR of pBrainbow 1.0</P><P>-&gt; PCR of pThy Ura3</P><P>Reconstituted all primers to 100 ug/ml.</P><H3 id="miniprep-of-pr6k-and-pbrainbow-1.0">Miniprep of pR6K and pBrainbow 1.0</H3><P>For PCR, using Promega kit w/ double wash. Elution in 30 ul</P><P>Final concentration: 93 ng/ul</P><H2 class="date" id="pcr-pura3">PCR pURA3</H2><UL><LI>1 ul template</LI><LI>1 ul of each primer</LI><LI>25 ul of Phusion master mix</LI><LI>22 ul of dH2O</LI></UL><P>Template: pKT174 plasmid (AmpR).</P><P>We have ~5 ul * 2, for a total amount of 1 ug.</P><P>Use 1 ul for 1 PCR, then if needed transform the rest in E. coli for propagation.</P><P>Primers:</P><TABLE><TBODY><TR class="odd"><TD align="left"><P>URA F+</P></TD><TD align="left"><P>o15.74</P></TD><TD align="left"><P>56°</P></TD></TR><TR class="even"><TD align="left"><P>URA R</P></TD><TD align="left"><P>o15.77</P></TD><TD align="left"><P>58°</P></TD></TR></TBODY></TABLE><P>Polymerase: Phusion in the form of 2x mix.</P><P>Program: (saved as “Phusion”)</P><P>98 (30s)</P><P>98 (10s) 50 (25s) 72 (1 min) x 25</P><P>72 (8 min)</P><P>4° hold</P><P>Lasts 1h30, product size = 1684 bp</P><H2 class="date" id="miniprep-of-the-pthy1.0-plasmid">Miniprep of the pThy1.0 plasmid</H2><P>With promega kit.</P><P>Recovery in 30 ul.</P><H2 class="date" id="preparation-of-agarose-1-tae">Preparation of Agarose 1% TAE</H2><P>Kept at 55°C in the broken incubator.</P><H2 class="date" id="pcr-of-pthy1.0-cassette">PCR of pThy1.0 cassette</H2><UL><LI>1 ul template</LI><LI>1 ul of each primer</LI><LI>25 ul of Phusion master mix</LI><LI>22 ul of dH2O</LI></UL><P>Template: pThy1.0 (1 ul = 93 ng)</P><P>Primers:</P><TABLE><TBODY><TR class="odd"><TD align="left"><P>B1 Brain1 F</P></TD><TD align="left"><P>o15.76</P></TD><TD align="left"><P>55°</P></TD></TR><TR class="even"><TD align="left"><P>B1 Brain1 R+</P></TD><TD align="left"><P>o15.75</P></TD><TD align="left"><P>58°</P></TD></TR></TBODY></TABLE><P>Program: (modified from Phusion)</P><P>98 (30s)</P><P>98 (10s) 50 (25s) 72 (2’45’’) x 25</P><P>72 (8 min)</P><P>4° hold</P><P>Expected size: 3965 bp</P><H2 class="date one" id="wednesday-0722">Wednesday 07/22</H2><P>Gel for product size checking, in that order:</P><UL><LI>Generuler 1 kb+</LI><LI>PCR Thy1.0 -&gt; 3965 bp</LI><LI>PCR Ura3 -&gt; 1684 bp</LI></UL><P>Results:</P><UL><LI>PCR Thy1.0 is ~ 4000</LI><LI>PCR Ura3 is ~ 1700</LI></UL><P>It worked!</P><P>PCR purification</P><P>Using the kit.</P><P>Titration:</P><P>PCR Thy1.0: 32 ng/ul</P><P>PCR Ura3: 47 ng/ul</P><P>Overlap PCR</P><OL><LI>Equimolar amounts of purified fragments</LI></OL><P>Required volumic ratio Thy/Ura = (47 / 1684) / (32 / 3965) = 3.46</P><TABLE><TBODY><TR class="odd"><TD align="left"><P>PCR Thy1.0</P></TD><TD align="left"><P>17 ul</P></TD></TR><TR class="even"><TD align="left"><P>PCR Ura3</P></TD><TD align="left"><P>5 ul</P></TD></TR><TR class="odd"><TD align="left"><P>Water</P></TD><TD align="left"><P>3 ul</P></TD></TR><TR class="even"><TD align="left"><P>Phusion master mix</P></TD><TD align="left"><P>25 ul</P></TD></TR></TBODY></TABLE><OL><LI>Do not add any primers; the templates will prime each-other.</LI><LI>Run 5 PCR cycles without primers. Predicted annealing temperature ~ 61°C. Use 57° as an annealing temperature and 2’45’’ of elongation time.</LI><LI>Add primers directly to the mix.</LI></OL><P>B1 Brain1 F (o15.76) 55°C</P><P>Ura R (o15.77) 58°C</P><P>1 ul each, directly into the PCR product.</P><P>Expected product size: 5610 bp</P><H2 class="date" id="wednesday-0723">Wednesday 07/23</H2><H3 id="gel-for-soe-yeastbow-assembly">Gel for SOE yeastbow assembly</H3><P>1% agar TAE, with 1 kb+ generuler.</P><P>30 ul of product + 6 ul loading dye were loaded for gel extraction.</P><UL><LI>Smear at very large sizes</LI><LI>Band at ~1600, probably from Ura3 PCR</LI><LI>No visible band at ~6000 bp</LI></UL><P>The SOE didn’t work, so nothing was extracted.</P><H3 id="new-attempt-for-soe-yeastbow-assembly">New attempt for SOE yeastbow assembly</H3><P>The concentration of template was way too high.</P><P>PCR Thy1.0         6 ul</P><P>PCR Ura3         2 ul</P><P>Water                14 ul</P><P>Phusion mix        25 ul</P><P>o15.76                2 ul</P><P>o15.77                2 ul</P><P>« S+ » tube: Primers added from the beginning,</P><P>« S » tube: Primers added only after 6 cycles.</P><P>Program</P><P>98° 30s</P><P>98° 15s, 50° 25s, 72° 4’20’’ [35 times] -&gt; Shorter elongation time</P><P>72° 8 min</P><P>4° hold -&gt; 12 next time</P><P>Lid at 103°. It takes about four hours to complete.</P><H2 class="date" id="friday-0724">Friday, 07/24</H2><H3 id="reception-of-oligos-o15.78-and-o15.79">Reception of oligos o15.78 and o15.79</H3><P>For flanking yeastbow with homology regions</P><P>Reconstitution:</P><P>4 nmol oligos + 40 ul eau = 0.1 nmol/ul = 100 mM</P><H3 id="gel-for-previous-soe-2nd-attempt">Gel for previous SOE (2nd attempt)</H3><DIV class="figure"><P class="caption">Yeastbow SOE gel extraction Enhanced.jpg</P></DIV><P>SOE        SOE        SOE+        SOE+        1kb+ marker</P><P>2 barely visible bands around 5600 bp in SOE+.</P><P>It is not possible to know which one is the right one.</P><H3 id="gel-extraction-of-2-soe-bands">Gel extraction of 2 SOE+ bands</H3><P>“Grande”</P><P>“Petite”</P><P>Recovery in 30 ul EB</P><P>Grande : 22.5 ng/ul</P><P>Petite: 6.2 ng/ul</P><H3 id="pcr-of-extracted-product">PCR of extracted product</H3><P>Aim: recover the tiny amount of DNA in the gel.</P><P>Template: SOE G or P after purification (29 ul)</P><P>Primers: 76 and 77 (2 ul each)</P><P>Phusion: 50 ul</P><P>Water: 17 ul</P><P>Program:</P><P>98 (30)</P><P>98 (10), 54 (25), 72 (3’20’’) -&gt; 36x</P><P>72 (8’00)</P><P>12 (hold)</P><P>The increased annealing temperature (54 instead of 50) should lower the non-specific hybridation.</P><P>This PCR takes ~ 3h30.</P><H2 class="date" id="monday-0727">Monday, 07/27</H2><H3 id="gel-for-rescue-pcr-of-extracted-product">Gel for rescue PCR of extracted product</H3><P>Agar 1% + TAE + SYBRsafe</P><P>Laid 5 ul + 1 ul loading dye</P><P>Order: « Petite » - Marker 1kb+ - « Grande »</P><P>Results</P><P>Did. Not. Work.</P><H3 id="thirld-attempt-at-soeing">Thirld attempt at SOEing</H3><UL><LI>Redo the PCRs with gel extraction this time -&gt; No smear in pThy1.0 product</LI><LI>Try to Gibson assemble the two parts</LI><LI>Try again the SOE with a higher annealing temperature</LI></UL><P>New SOE:</P><P>PCR Thy1.0         3 ul</P><P>PCR Ura3         1 ul</P><P>Water                7 ul</P><P>Phusion mix        12.5 ul</P><P>o15.76                1 ul</P><P>o15.77                1 ul</P><P>Program</P><P>98° 30s</P><P>98° 20s, gradient 30s, 72° 3’10’’ [35 times] (NEB calculator recommends 60°C…)</P><P>72° 8 min</P><P>12° hold</P><P>Gradient: 56 to 62°C (bottom to top).</P><H3 id="new-pcrs">New PCRs</H3><P>Objective: Better quality of the SOE reagents, gel-extracted to get rid of the smear.</P><P>Larger number of cycles.</P><P>Ura3 PCR</P><P>pKT174        1ul</P><P>o15.74                1ul</P><P>o15.77                1ul</P><P>Water                22ul</P><P>Phusion mix        25ul</P><P>Program:</P><P>98 (30)</P><P>98 (10), 56 (25), 72 (1’00) x35</P><P>72 (8)</P><P>12 (hold) -&gt; 1684 bp</P><P>Thy PCR</P><P>pThy1.0        1ul</P><P>o15.75                1ul</P><P>o15.76                1ul</P><P>Water                22ul</P><P>Phusion mix        25ul</P><P>Program:</P><P>98 (30)</P><P>98 (10), 56 (25), 72 (1’00) x35</P><P>72 (8)</P><P>12 (hold) -&gt; 3965 bp</P><H2 class="date one" id="tuesday-0728">Tuesday 07/28</H2><H3 id="gel-extraction-for-pcr-ura-pcr-thy-and-soe-gradient-pcr">Gel extraction for PCR Ura, PCR Thy and SOE gradient PCR</H3><P>Agar 1%, SYBRsafe, TAE</P><TABLE><TBODY><TR class="odd"><TD align="left"><P>1</P></TD><TD align="left"><P>1</P></TD><TD align="left"><P>U</P></TD><TD align="left"><P>U</P></TD><TD align="left"><P>M</P></TD><TD align="left"><P>A</P></TD><TD align="left"><P>B</P></TD><TD align="left"><P>C</P></TD><TD align="left"><P>D</P></TD><TD align="left"><P>M</P></TD></TR></TBODY></TABLE><P>1: pThy pcr</P><P>U: pUra3 pcr</P><P>M: 1kb dna ladder</P><P>A, B, C, D: gradient PCR for SOE</P><P>It was at first run at 75 kV and then at 100 kV during 25 in total. The buffer in the tank was TBE, and as a result the gel was completely awful, with no possibility to read the size of the bands.</P><P>The products were also lost.</P><H3 id="new-pcrs-for-yeastbow">New PCRs for Yeastbow</H3><P>PCR Thy1.0</P><P>pThy1.0        1 ul</P><P>75                2.5 ul</P><P>76                2.5 ul</P><P>Phusion mix        25 ul</P><P>Water                19 ul</P><P>Program:</P><P>98 (60)</P><P>98 (10) 56 (30) 72 (120) x35</P><P>72 (600)</P><P>4 (hold)</P><P>Expected size: 3965. Should be over at about 18h30.</P><P>PCR Ura3</P><P>pKT174        1 ul</P><P>74                2.5 ul</P><P>77                2.5 ul</P><P>Phusion        25 ul</P><P>Water                19 ul</P><P>Program:</P><P>98 (60)</P><P>98 (10) 56 (30) 72 (60) x35</P><P>72 (600)</P><P>12 (hold)</P><P>Expected size: 1684 bp.</P><H2 class="date one" id="wednesday-0729">Wednesday 07/29</H2><H3 id="gel-for-yeastbow-1-and-u-and-extraction">Gel for Yeastbow 1 and U and extraction</H3><P>Agar 1%, SYBRsafe, TAE</P><TABLE><TBODY><TR class="odd"><TD align="left"><P>1kb ladder</P></TD><TD align="left"><P>1</P></TD><TD align="left"><P>1</P></TD><TD align="left"><P>U</P></TD><TD align="left"><P>U</P></TD><TD align="left"><P>1kb ladder</P></TD></TR><TR class="even"><TD align="left"><P>3965</P></TD><TD align="left"><P>3965</P></TD><TD align="left"><P>1684</P></TD><TD align="left"><P>1684</P></TD></TR></TBODY></TABLE><P>The PCR for U worked really well, but the PCR for 1 yielded an extremely low amount of DNA and a smear.</P><P>-&gt; Decrease annealing temp</P><P>-&gt; Increase extension time</P><P>pcr Ura3: 41 ng/ul * 30 ul of EB</P><P>pcr Thy1: 13 ng/ul * 30 ul of EB</P><H3 id="new-attempt-at-pcr-thy1-29-july">New attempt at PCR Thy1, 29 july</H3><P>Mix</P><P>Phusion        25</P><P>Thy                1</P><P>o15.75                2.5</P><P>o15.76                2.5</P><P>Water                19</P><P>= Two tubes of 50 ul each -&gt; 3965 bp</P><P>Program</P><P>98 (30)</P><P>98 (20), 60 (20), 72 (3’00’)</P><P>72 (5’00)</P><P>12 (hold)</P><H3 id="new-attempt-at-soe-29-july-with-pcr-products-from-28-july">New attempt at SOE, 29 july, with PCR products from 28 july</H3><P>Mix</P><P>Phusion        25</P><P>PCR 1                1</P><P>PCR U                1</P><P>o15.76                2.5</P><P>o15.77                2.5</P><P>Water                11</P><P>= Two tubes of 50 ul each -&gt; 5610 bp</P><P>Program</P><P>98 (30)</P><P>98 (20), 60 (20), 72 (3’00’)</P><P>72 (5’00)</P><P>12 (hold)</P><P>(Along with PCR 1)</P><H2 class="date one" id="thursday-0730">Thursday, 07/30</H2><H3 id="gel-for-soe-2907">Gel for SOE 29/07</H3><TABLE><TBODY><TR class="odd"><TD align="left"><P>1 kb ladder</P></TD><TD align="left"><P>SOE</P></TD><TD align="left"><P>SOE</P></TD><TD align="left"><P>SOE</P></TD><TD align="left"><P>1 kb ladder</P></TD></TR></TBODY></TABLE><P>Expected size: 5600 for all of them.</P><DIV class="figure"><P class="caption">SOE+29.07.jpg</P></DIV><P>It didn’t work at all. Try to Gibson-assemble them.</P><H2 class="date one" id="friday-0731">Friday 07/31</H2><H3 id="culture-of-pkt174-for-freezing-miniprep">Culture of pKT174 for freezing + miniprep</H3><P>In 4 ml LB + 4 ul ampicilline, @37.</P><H2 class="date two" id="saturday-81">Saturday, 8/1</H2><H3 id="the-culture-of-pkt174-grew">The culture of pKT174 grew:</H3><UL><LI>Miniprep (not titrated for now)</LI><LI>Glycerol stock @ -20</LI></UL><H3 id="new-strategy-based-on-gibson-assembly">New strategy based on Gibson assembly</H3><P>With minimal PCR steps. Low success rate, low mutation rate.</P><DIV class="figure"><P class="caption">yeastbow next assembly.png</P></DIV><UL><LI>Amplify pThy1.0 with o15.75 (58) and o15.78 (58) -&gt; 4006 bp</LI><LI>Amplify pUra3 with o15.74 (56) and o15.79 (56) -&gt; 1725 bp</LI><LI>Cleanup or extract (or try both)</LI><LI>Gibson assembly (39 bp overlap, Tm = 66°C) -&gt; 5692 bp. Works on Snapgene (see Yeastbow Vector.dna)</LI><LI>Ready to go</LI></UL><P>Method for PCR: Jake’s magic recipe.</P><P>Reagent</P><P>Volume</P><P> </P><P>1x</P><P>10x</P><P>Nuclease-free water</P><P>71 ul</P><P>710 ul</P><P>5x Phusion HF Buffer</P><P>20 ul</P><P> 200 ul</P><P>10 mM dNTPs</P><P>2 ul</P><P>20 ul</P><P>Forward Primer (10 uM)</P><P>1 ul</P><P>10 ul</P><P>Reverse Primer (10 uM)</P><P>1 ul</P><P>10 ul</P><P>Template</P><P>1 ul</P><P>10 ul</P><P>DMSO</P><P>3 ul</P><P>30 ul</P><P>Phusion DNA Polymerase</P><P>1 ul</P><P> 10 ul</P><P>Total Volume</P><P>100 ul</P><P>1000 ul</P><P>Thermocycler Protocol: NEB Phusion </P><P>98 (30)</P><P>98 (10) 52 (30) 72 (30/kb) *35</P><P>72 (5’)</P><P>10 (forever)</P><H4 id="gel08.01-yeastbow-new-method-pcrs.png">Gel</H4><P>L        M        L        R        M        R</P><P>2.5 ul sample + 0.5 LD</P><P>Marker: 1 kb</P><P>Both of the PCR worked quite well, the products are of the right size and there is no significant alien band.</P><P>-&gt; PCR purification with the QIAGEN kit</P><P>Recovery in 40 ul of water.</P><H4 id="titration">Titration</H4><P>Yeastbow R: 300 ng/ul (1725 bp)</P><P>Yeastbow L: 155 ng/ul (4006 bp)</P><H2 class="date two" id="monday-0803">Monday 08/03</H2><H3 id="gibson-assembly-of-yeastbow">Gibson assembly of yeastbow</H3><P>On ice: 15 ul master mix + 5 ul DNA, incubate at 50° for 1 hour.</P><P>Number of ng = pmols * bp * 0.65</P><TABLE><TBODY><TR class="odd"><TD align="left"><P>Fragment</P></TD><TD align="left"><P>ul</P></TD><TD align="left"><P>pmol</P></TD></TR><TR class="even"><TD align="left"><P>Yeastbow R</P></TD><TD align="left"><P>0.9</P></TD><TD align="left"><P>0.24</P></TD></TR><TR class="odd"><TD align="left"><P>Yeastbow R</P></TD><TD align="left"><P>4.1</P></TD><TD align="left"><P>0.24</P></TD></TR><TR class="even"><TD align="left"><P>Gibson master mix</P></TD><TD align="left"><P>15 ul</P></TD></TR></TBODY></TABLE><H2 class="date two" id="tuesday-84">Tuesday 8/4</H2><H4 id="transformation-of-yeastbow-in-the-cre-strains">Transformation of yeastbow in the CRE strains</H4><P>Culture in 50 ml 2x YPD.</P><P>Problem: According to Matt these strains already have Ura3. It is not possible to transform yeastbow in them.</P><P>-&gt; Transform the yeastbow in Ura- S. cerevisiae strains and add the CRE later (maybe by taking it from the yPH150/151)</P><P>Thanks to the fluorescence of the transformant, we attempted to transform the yeastbow cassette and select cells with mere fluorescence.</P><P>However, while in the process of doing the transformation, the centrifuge got stuck with my samples inside. I gave up at that point.</P><H4 id="freezer-stocks-of-yph150-and-yph151">Freezer stocks of yPH150 and yPH151</H4><P>500 ul glycerol 50% and 500 ul overnight culture.</P><P>Numbers 15.54 and 15.55 resp.</P><H4 id="yeastbow-strikes-back">Yeastbow strikes back</H4><P>CRE yeast strains we have:</P><UL><LI>yPH150: CRE under the Ura3 promoter (const)</LI><LI>yPH151: CRE under the Stl1 promoter (inducible)</LI></UL><P>These strains are based on BY4743. Both of them are auxotrophic for His, Leu and Lys, but no longer for Ura (as it was used for CRE transformation!)</P><P>All the work previously done on yeastbow is therefore useless, and it’s necessary to start over with His3 as a selection marker.</P><P>The Leu2 integration tails are correct, even if the Leu2 ORF has been destroyed.</P><P>Available plasmids with resistance:</P><P>pYMC32, pFA6, pPH75 from euroscarf toolbox. The names might not be correct.</P><P>For new primers were designed and ordered:</P><P>Amplify pThy1.0 with these primers:</P><P>o15.160 GCAATATATATATATATATTTCAAGGATATACCATTCTAACTAGCGAGCTCATAACTTCG (Tm=58, specific)</P><P>o15.161 GCTTATTTAGAAGTGGCGCGgacctctgcagaggaaggac (Tm=58, may hairpin but not likely, may dimerize, specific)</P><P>Product size: 4066 bp</P><P>Amplify pFA6-His3 with these primers:</P><P>o15.162 gtccttcctctgcagaggtcCGCGCCACTTCTAAATAAGC (may hairpin, may dimerize, specific) (Tm=56)</P><P>o15.163 AAGTTTATGTACAAATATCATAAAAAAAGAGAATCTTTCCTGGATGGCGGCGTTAGTATC (Tm=59, specific)</P><P>Product size: 1614 bp</P><P>Gibson assembly</P><P>Overlap: 40 bp, Tm = 70°C</P><P>Product size: 5640 bp</P><H2 class="date two" id="monday-0810">Monday, 08/10</H2><H3 id="received-plasmids-and-primers-for-yeastbow">Received plasmids and primers for Yeastbow</H3><H4 id="plan-with-soe">Plan with SOE:</H4><TABLE><TBODY><TR class="odd"><TD align="left"><P>Forward</P></TD><TD align="left"><P>Reverse</P></TD><TD align="left"><P>Template</P></TD><TD align="left"><P>PCR type</P></TD><TD align="left"><P>Tm</P></TD><TD align="left"><P>Extension Time</P></TD><TD align="left"><P>Product size</P></TD></TR><TR class="even"><TD align="left"><P>BB1F</P></TD><TD align="left"><P>BB1R</P></TD><TD align="left"><P>pThy1-Brainbow1.0</P></TD><TD align="left"><P>Phusion</P></TD><TD align="left"><P>60</P></TD><TD align="left"><P>2’</P></TD><TD align="left"><P>3965</P></TD></TR><TR class="odd"><TD align="left"><P>His3F</P></TD><TD align="left"><P>His3R</P></TD><TD align="left"><P>pPH21 or PH75</P></TD><TD align="left"><P>Phusion</P></TD><TD align="left"><P>60</P></TD><TD align="left"><P>1’</P></TD><TD align="left"><P>1532</P></TD></TR><TR class="even"><TD align="left"><P>Pho85F</P></TD><TD align="left"><P>Pho85R</P></TD><TD align="left"><P>PCR1 and 2</P></TD><TD align="left"><P>SOE</P></TD><TD align="left"><P>60</P></TD><TD align="left"><P>3’</P></TD><TD align="left"><P>5457</P></TD></TR></TBODY></TABLE><H4 id="alternative-setup">Alternative setup</H4><P>What do we have?</P><UL><LI>One brainbow plasmid: pThy1.0</LI><LI>One His3 resistance plasmid: pPH21</LI><LI>Two border primers without tails (not used here)</LI><LI>Two sets of central Gibson assembly primers: 161+162 or BB1R+His3F</LI><LI>Two sets of integration tails: Pho85F+Pho85R or 160+163</LI></UL><P>Every four combinations of central primers and integration tails were used separately.</P><H4 id="master-mix-for-both-pcrs">Master mix for both PCRs</H4><TABLE><TBODY><TR class="odd"><TD align="left"><P>Name</P></TD><TD align="left"><P>1x</P></TD><TD align="left"><P>5x</P></TD></TR><TR class="even"><TD align="left"><P>Water</P></TD><TD align="left"><P>40</P></TD><TD align="left"><P>200</P></TD></TR><TR class="odd"><TD align="left"><P>Phusion Master Mix</P></TD><TD align="left"><P>50</P></TD><TD align="left"><P>250</P></TD></TR><TR class="even"><TD align="left"><P>DMSO</P></TD><TD align="left"><P>3</P></TD><TD align="left"><P>15</P></TD></TR><TR class="odd"><TD align="left"><P>Template</P></TD><TD align="left"><P>1</P></TD><TD align="left"><P>5</P></TD></TR></TBODY></TABLE><H4 id="pcr">1.0 PCR</H4><P>Mix:</P><P>2 ul of each primers. In that order:</P><P>PB: PhoF + BB1 R</P><P>1B: 160 + BB1 R</P><P>P1: PhoF + 161</P><P>11: 160 + 161</P><P>Expected result:</P><P>~4066 bp</P><P>Program (as seen in previous working Yeastbow assembly):</P><P>98 (30)</P><P>98 (10) 52 (30) 72 (3’00) x35</P><P>75 (5)</P><H4 id="pph21-pcr">pPH21 PCR</H4><P>Mix:</P><P>2 ul of each primers. In that order:</P><P>HP: His3F + PhoR</P><P>1P: 162 + PhoR</P><P>H1: His3F + 163</P><P>11: 162 + 163</P><P>Expected result:</P><P>~1614 bp</P><P>Program (as seen in previous working Yeastbow assembly):</P><P>98 (30)</P><P>98 (10) 52 (30) 72 (2’00) x35</P><P>75 (5)</P><P>The tubes for this one are marked with a blue spot on the hinge of the PCR tube.</P><H3 id="gel-for-both-of-these-pcr">Gel for both of these PCR</H3><TABLE><TBODY><TR class="odd"><TD align="left"><P>1kb</P></TD><TD align="left"><P>PB</P></TD><TD align="left"><P>1B</P></TD><TD align="left"><P>P1</P></TD><TD align="left"><P>11</P></TD><TD align="left"><P>1kb</P></TD><TD align="left"><P>HP</P></TD><TD align="left"><P>1P</P></TD><TD align="left"><P>H1</P></TD><TD align="left"><P>11</P></TD><TD align="left"><P>1kb</P></TD></TR></TBODY></TABLE><P>The reactions containing the 161 primer for 1.0 generated small fragments, they can’t be used after mere PCR cleanup and would require gel extraction. Don’t use 161-162 as a central junction, use PB or 1B. They might require gel extraction anyways, or re-try with less template.</P><P>1P and H1 worked best for His3 PCR. Use H1.</P><P>Conclusion: Use 1B : 160 (Leu2 tail F) + BB1 Use H1: His3F + 163 (Leu2 tail R) Try PCR cleanup them and follow with Gibson assembly.</P><H2 class="date two" id="wednesday-0812">Wednesday, 08/12</H2><H3 id="yeastbow-assembly">Yeastbow assembly</H3><P>The following assembly was chosen according to the PCR screening:</P><UL><LI>Amplify pThy1.0 with 160 and BB1R (try to use less template)</LI><LI>Amplify pPH21 with His3F and 163.</LI></UL><P>Both of these products were purified.</P><UL><LI>1B was purified by JB, there is &quot;1B Caca&quot; written on the cap. Thanks, JB. It was eluted in 30 ul of water and the concentration is 119 ng/ul -&gt; Rh</LI><LI>H1 was purified, eluted in 30 ul of water and the concentration is 231.8 ng/ul -&gt; Lh</LI></UL><H2 class="date two" id="thursday-0813">Thursday, 08/13</H2><H3 id="yeast-transformation">Yeast transformation</H3><P>The pre-culture for yPH150/151 grew after two days. At 11 a.m. they were diluted in 50 ml of YPD 2x and put at 30°C for pre-transformation culture.</P><H3 id="gel-for-1b-pcr">Gel for 1B PCR</H3><P>1B and R6K pcrs were ran on the same gel:</P><TABLE><TBODY><TR class="odd"><TD align="left"><P>1B | 1B | 1 kb ladder | R6K | R6K</P></TD></TR></TBODY></TABLE><P>2 ul water + 1 ul LD + 3 ul sample.</P><P>Result: There is one clean band at around &gt;4000 bp. There is no significant smear above it, hence a better quality for Gibson assembly.</P><P>The PCR product was cleaned up and eluted in 30 ul of water. Concentration: 63 ng/ul.</P><H3 id="assembly-of-yeastbow-his3">Assembly of Yeastbow His3</H3><TABLE><THEAD><TR class="header"><TH align="left"><P>Part</P></TH><TH align="left"><P>Conc. (ng/ul)</P></TH><TH align="left"><P>size (bp)</P></TH><TH align="left"><P>volume (ul)</P></TH><TH align="left"><P>amount (pmol)</P></TH></TR></THEAD><TBODY><TR class="odd"><TD align="left"><P>L his3</P></TD><TD align="left"><P>63</P></TD><TD align="left"><P>4066</P></TD><TD align="left"><P>4.5</P></TD><TD align="left"><P>0.11</P></TD></TR><TR class="even"><TD align="left"><P>R his3</P></TD><TD align="left"><P>231</P></TD><TD align="left"><P>1614</P></TD><TD align="left"><P>0.5</P></TD><TD align="left"><P>0.11</P></TD></TR></TBODY></TABLE><P>+ 15 ul of MMII Gibson Assembly master mix.</P><P>Incubation 30 min at 50°C, then 5 min at 98 and 5 min at 72. The goal is to inactivate the exonuclease without inactivating the polymerase, to generate blunt-ends at the end.</P><H3 id="yph150151-transformation-with-yeastbow">yPH150/151 transformation with yeastbow</H3><UL><LI>Denature 1ml DNA carrier for 5 min at 99° and chill on ice</LI><LI>Centrifuge 5 min at 4000 G, recover in 25 ml sterile water</LI><LI>Centrifuge 5 min at 4000 G, recover in 25 ml sterile water</LI><LI>Centrifuge 5 min at 4000 G, recover in 1 ml sterile water</LI></UL><P>Transformation mix: PEG 3350 (50%): 580 ul (attention, should have been 480) LiAc 1.0M: 72 ul Carrier: 100 ul Plasmid DNA: 16 ul Sterile water: 52 ul</P><P>360 ul + 150 ul cells -&gt; vortex</P><P>Heatshock 50 min (instead of 40) at 42°C.</P><P>In the absence of appopriate medium, the result was centrifuged, the supernatant discarded and the pellet cultured in YPD overnight.</P><H2 class="date two" id="friday-0814">Friday 08/14</H2><H3 id="pcr-on-yeastbow-his3-assembly">PCR on Yeastbow His3 assembly</H3><TABLE><THEAD><TR class="header"><TH align="left"><P>Phusion master mix C</P></TH><TH align="left"><P>50</P></TH></TR></THEAD><TBODY><TR class="odd"><TD align="left"><P>160</P></TD><TD align="left"><P>2</P></TD></TR><TR class="even"><TD align="left"><P>163</P></TD><TD align="left"><P>2</P></TD></TR><TR class="odd"><TD align="left"><P>Assembly product</P></TD><TD align="left"><P>2</P></TD></TR><TR class="even"><TD align="left"><P>DMSO</P></TD><TD align="left"><P>3</P></TD></TR><TR class="odd"><TD align="left"><P>Water</P></TD></TR></TBODY></TABLE><P>Program: &quot;DMSO&quot; with 55°C annealing temperature, and 4 minutes of extension. Expected size: 5457 bp.</P><H3 id="preparation-of-selective-media-without-histidine">Preparation of selective media without histidine</H3><P>Base: 16.8g Glucose: 5g Supplement without histidine: 1.92g Agar: 15g Water: qsp 1000 ml This was then split between two 1L bottles and autoclaved.</P><H3 id="preparation-of-2x-ypd-medium">Preparation of 2x YPD medium</H3><P>Medium powder: 50g Water: qsp 500 ml</P><H3 id="preparation-of-lb-agar-medium">Preparation of LB agar medium</H3><P>Water: qsp 500 ml Medium powder: 25 g Agar: 7.5 g</P><H3 id="plating-of-yeastow-transformation-products">Plating of Yeastow transformation products</H3><P>The YPD culture were centrifugated and recovered in 1 ml sterile water. Each strain was plated twice on Histidine- medium + agar, once with 100 ul and once with 250 ul of transformation product. There were then grown at 30°C.</P><P>-&gt; Result: nothing grew. At all.</P><H2 class="date two" id="section">15/08</H2><H3 id="splicing-by-overlap-extension-with-his3">Splicing by Overlap Extension with His3</H3><TABLE><THEAD><TR class="header"><TH align="left"><P>Phusion master mix C</P></TH><TH align="left"><P>50</P></TH></TR></THEAD><TBODY><TR class="odd"><TD align="left"><P>160</P></TD><TD align="left"><P>2</P></TD></TR><TR class="even"><TD align="left"><P>163</P></TD><TD align="left"><P>2</P></TD></TR><TR class="odd"><TD align="left"><P>Lh</P></TD><TD align="left"><P>4.5 ul</P></TD></TR><TR class="even"><TD align="left"><P>Rh</P></TD><TD align="left"><P>0.5 ul</P></TD></TR><TR class="odd"><TD align="left"><P>DMSO</P></TD><TD align="left"><P>3</P></TD></TR><TR class="even"><TD align="left"><P>Water</P></TD></TR></TBODY></TABLE><P>At first the primers were not added to the mix.</P><P>98 (1'00) 98 (15) 52 (30) 72 (4'00) 5*</P><P>It took ~40 min.</P><P>Then the primers were added and the program for &quot;DMSO PCR&quot; was performed, with: - 54°C of annealing - 4' of extension - 37 cycles</P><P>Gel: 3 ul sample + 2 ul water + 1 ul LD [Successful]</P><P>It worked! There is a band at ~5500</P><H2 class="date" id="section-1">16/08</H2><H3 id="gel-extraction-of-the-soe-product">Gel-extraction of the SOE product</H3><P>60 ul of SOE product were deposed on a gel. Expected size: 5457 bp.</P><P>The migration failed miserably due to incorrect concentration of the TAE buffer. The deposited sample was lost. Fortunately, only a part of the SOE product was deposited.</P><H3 id="culture-of-yph150-and-yph151">Culture of yPH150 and ypH151</H3><P>For transformation on Monday. These strains were put to culture in 3 ml YDP, with a YDP control for contamination.</P><H3 id="pcr-cleanup-of-the-remaining-soe-product">PCR cleanup of the remaining SOE product</H3><P>Elution in 30 ul of water. Titration: 40 ng/ul, very clean.</P><H2 class="date" id="section-2">18/08</H2><H3 id="yeastbow-transformation">Yeastbow transformation</H3><P>Following the &quot;High-efficiency transformation of yeast protocol&quot;.</P><P>yPH150 did not grow, but it doesn't matter as it's not the most important strain (yPH151).</P><P>0D after overnight: 0.7 (=7.10^6 cells/ml)</P><P>2.5 ml of culture were diluted in 50 ml YPD at 11h30.</P><P><EM>Transformation mix (in ul)</EM> PEG, 1080 LiAc, 162 ssDNAcarrier, 225 plasmid, 22.5 -&gt; 200 ng per transformation sterile water, 130.5</P><P>Then 360 ul of mix per tube of cells. 1 control tube was made with 360 ul of sterile water.</P><P><EM>Heatshock</EM> from 20h45 to 21h20.</P><H2 class="date" id="section-3">19/08</H2><H3 id="reproducing-the-successful-soe">Reproducing the successful SOE</H3><P>Due to the small amount left of Lh product, only three tubes could be made. The tubes 1 and 2 contained exactly the same mix as the SOE of 15/08. The tube 3 was exactly the same except the primers 160 and 163 were replaced by Pho85 F and Pho85 R. These primers use a different homology region for yeast integration.</P><P>The protocol and program was exactly similar to what had worked before.</P><P><EM>Results:</EM> None of the worked (gel not shown). It's weird that this previously-working reaction doesn't work anymore.</P><H3 id="yeastbow-transformation-results">Yeastbow transformation results</H3><P>Nothing visible (wait 3 days). Make new SOE as a plan B.</P><H2 class="date" id="section-4">20/08</H2><H3 id="pcr-on-the-working-soe-product">PCR on the working SOE product</H3><P>Maybe we can get something good from what's left in the tube. A full annealing gradient was made with 8 samples.</P><TABLE><THEAD><TR class="header"><TH align="left"><P>Name</P></TH><TH align="left"><P>1x</P></TH><TH align="left"><P>8x</P></TH></TR></THEAD><TBODY><TR class="odd"><TD align="left"><P>2x Phusion Master Mix</P></TD><TD align="left"><P>10</P></TD><TD align="left"><P>80</P></TD></TR><TR class="even"><TD align="left"><P>DMSO</P></TD><TD align="left"><P>0.6</P></TD><TD align="left"><P>4.8</P></TD></TR><TR class="odd"><TD align="left"><P>160</P></TD><TD align="left"><P>1</P></TD><TD align="left"><P>8</P></TD></TR><TR class="even"><TD align="left"><P>163</P></TD><TD align="left"><P>1</P></TD><TD align="left"><P>8</P></TD></TR><TR class="odd"><TD align="left"><P>SOE product</P></TD><TD align="left"><P>1</P></TD><TD align="left"><P>8</P></TD></TR><TR class="even"><TD align="left"><P>Water</P></TD><TD align="left"><P>6.4</P></TD><TD align="left"><P>51.2</P></TD></TR></TBODY></TABLE><P><EM>Program</EM>: &quot;DMSO&quot; with 5 minutes of extension. 37 cycles. It takes like 6h.</P><P><EM>Gradient</EM>: 61, 60.2, 58.8, 56.7, 54.2, 52.1, 50.8, 50</P>

Results: It did not work. We cannot even repeat previously working experiment. There is definitely a problem with some materials or devices.

</DIV></DIV></DIV></DIV></BODY></HTML>