Team:Paris Bettencourt/Notebook/Phytase
Ferment It Yourself
iGEM Paris-Bettencourt 2O15
Background
Design
Project
Micronutrients
Continuity
Modeling
Manufacturing
Achievements
Results
Parts
Medal Fulfillments
Practices
Overview
SynBio Challenge
DIλ
Acceptance
Events
Rhizi
Collaborations
Safety
Safety Overview
Safety Form
Review Board
Notebook
Daily notebook
Protocols
Bibliography
Team
Members
Attributions
Acknowledgments
Press
Official Team Profile
Notebook - Phytase
August
September
August 10th
Design primers
Gene PHO85
5’Primer  of Kanamycin resistance gene with tails using to transformation with the PHO85 gene of the yeast.
5’-TATCATTATATATACATGGCTACGGTTTTTCGCTGACGGGCTGCGATAATCATTTGCA  TCCATACATTTTGATGGC -3’
3’Primer  of Kanamycin resistance gene with tails using to transformation with the PHO85 gene of the yeast.
3’-AAGGGATATATAGCGCGGCAAACTGGGCAAACTTGAGCAATACCACAGCAGTATAG  CGACCAGCATTC-5’
- Homology tail on gene PHO85
- Kanamycin resistance binding
Gene PHO80
5’Primer  of Kanamycin resistance gene with tails using to transformation with the PHO80 gene of the yeast.
5’-ATCATAAGACGAGGATATCCTTTGGAGACTCATAGAAATCATAATCATTTGCATCCAT  ACATTTTGATGGC-3’
3’Primer  of Kanamycin resistance gene with tails using to transformation with the PHO80 gene of the yeast.
3’-CTCAATCATGATTGCTTTCATAATACCCCACGAAAAATCACAGCAGTATAGCGACCA  GCATTC-5’
- Homology tail on gene PHO80
- Kanamycin resistance binding
Gene FRT + PHO85
5’Primer  of Kanamycin resistance gene with tails using to transformation with the PHO85 gene of the yeast, including FRT sequence to delete both of PHO80 and PHO85.
5’-TATCATTATATATACATGGCTACGGTTTTTCGCTGACGGGCTGCGGAAGTTCCTATTC  TCTAGAAAGTATAGGAACTTCATAATCATTTGCATCCATACATTTTGATGGC-3’
3’Primer  of Kanamycin resistance gene with tails using to transformation with the PHO85 gene of the yeast, including FRT sequence to delete both of PHO80 and PHO85.
3’-AAGGGATATATAGCGCGGCAAACTGGGCAAACTTGAGCAATACCACTTCAAGGATAT   GAAAGATCTCTTATCCTTGAAGCAGCAGTATAGCGACCAGCATTC-5’
- Homology tail on gene PHO85
- FRT sequence
- Kanamycin resistance binding
August 12nd
Culture
Inoculate 100µL of Saccharomyces cerevisiae SK1 on YPD medium overnight (at 30°C).
This yeast will be transformed.
PCR
3 PCR were realized on HO-Poly-KanMX4-HO plasmid to create a Kanamycin resistance marker, thanks to 3 pairs of primers wich have tails we’ll be use to knock out genes PHO80, PHO85 and both in the yeast.
Protocol:
PHO80
PHO85
FRT+ PHO85
Master mix (µL)
50
50
50
H2O DNAse Free (µL)
45
45
45
Resistance plasmid (µL)
1
1
1
PHO80 5'Primer (µL)
2
PHO80 3'Primer (µL)
2
PHO85 5'Primer (µL)
2
PHO85 3'Primer (µL)
2
PHO85 + FRT 5'Primer (µL)
2
PHO85 + FRT 3'Primer (µL)
2
Figure 1 : PCR cycle
August 13th
PCR Purification
Protocol : PCR purification
Result of PCR (August 12th)
The PCR product are then revealed with a gel electrophoresis.
We expected bands around 1.300bp. The band corresponding to marker with FRT is bigger than the two others strips because these have just the Kanamycin resistance with tails, and no FRT sequences.
Figure 2 :Result of PCR
Pre-culture
Plated one colony of Saccharomyces cerevisiae SK1 in 5mL liquid YPD medium and let's grow overnight.
August 14th
Transformation of yeast
Protocol: Heat shock transformation for yeast
August 17th
Result of plates:
There is a culture in plates.
The negative control is not well. The no change yeast grow in the YPD medium with the antibiotic.
We will repeat this control on an agar plate and not in a liquid medium.
We analyze anyway down results, the results of the new control will allow us to validate the result of our experiment or search which are our error and try again.
The positive control is well, yeast multiply on YPD medium plate without antibiotic. Yeasts are not dead, so the culture on other agar mediums are not contamination.
We see more colonies on the plates with yeast transforming PHO85 and FRT+PHO85.
We look only few colonies in the plates with yeast transforming PHO80.
The result is well, transformation works.
Figure 3 : Negative control
Figure 4 : Positive control and result of transformation
Design primers
We design primers to verifie our results on the plates.
Thanks to the colony PCR, we might determinate if the resistance is integrated into the yeast DNA.
Primer 5'-3' PHO80
ATCATAAGACGAGGATATCCTTTGGAG
Primer 3'-5' PHO80
CTCAATCATGATTGCTTTCATAATACCCC
Primer 5'-3' PHO85
TATCATTATATATACATGGCTACGGTTTTTCG
Primer 3'-5' PHO85
AAGGGATATATAGCGCGGCAAACTG
Primer 5'-3' FRT+PHO85
TATCATTATATATACATGGCTACGGTTTTTCG
Primer 3'-5' FRT+PHO85
AAGGGATATATAGCGCGGCAAACTG
August 18th
Verification of the new negative control
The verification of the negative control is good, any colony is watching. We can continue our experiments, it will be validated.
Figure 5 : Result of the new negative control
FRT problems
The transformation with the FRT may be run well, but the plasmid with the gene coding for flippase works only for E. coli. We can't use this plasmid because it will be rejected by the yeast.
Other transformation with CreLox system is possible.
CreLox is a gene which have the same fonction than the FRT, it not cut by the flippase but by the Cre recombinase.
We design two primers for the new transformation with CreLox system.
Primer 5'-3' CreLox + PHO85
5’-TATCATTATATATACATGGCTACGGTTTTTCGCTGACGGGCTGCGATAACTTCGTATAGCATACATTATACGAAGTTATATAATCATTTGCATCCATACATTTTGATGGC-3’
Primer 3'-5' CreLox + PHO85
3’-AAGGGATATATAGCGCGGCAAACTGGGCAAACTTGAGCAATACCAATAACTTCGTATAGCATACATTATACGAAGTTATCAGCAGTATAGCGACCAGCATTC-5’
- Homology tail on gene PHO85
- CreLox sequence
- Kanamycin resistance binding
August 19th
Colony PCR
Protocol:
PHO80
PHO85
FRT+ PHO85
dreamTaq 2X (µL)
3
3
3
H2O DNAse Free (µL)
9
9
9
Colony
1
1
1
PHO80 5'Primer (µL)
0.5
PHO80 3'Primer (µL)
0.5
PHO85 5'Primer (µL)
0.5
0.5
PHO85 3'Primer (µL)
0.5
0.5
Figure 6 : Colony PCR cycle
August 20th
Result of colony PCR (August 19th)
We see bands smaller than 500bp, but not the fragments we expected : this is probably contaminations. We start again the same PCR colony. There is DNA in holes too, which didn't migrate, probably the genomic DNA.
Figure 7 : Electrophoresis PCR colony
Colony PCR
Same to August 19th.
Result of second colony PCR
Figure 8 : second electrophoresis colony PCR
The DNA Ladder is good but DNA of PCR did not migrate. The ADN is in the holes. We think that the yeasts walls being thicker simple thermic shock does not break it. We will be carry a lysis whith NaOH in yeast, to push out DNA, to the primer can be fixed on it.
August 24th
Yeast lysis with NaOH
Protocol: Yeast lysis with NaOH
After the lysis of yeast we realize the new PCR in normal condition, the same as August 12nd.
August 25th
Result of PCR (August 24th)
Figure 9 : Third electrophoresis colony PCR with NaOH lysis
The DNA ladder migrate, but there was any amplification of the both genes.
We tested two PCR mix : the first did not work. The positive control worked with the second PCR mix. It is composed of HO plasmid and the oligos using for PHO85 gene.
August 26th
Colony PCR
To make the colony PCR, we need to lysis yeasts' wall. We realized the lysis with NaOH, but it did not work. So we realize a new lysis using the   " DNeasy Blood and Tissue Kit " with the zymolyase enzyme on the non-transformed yeasts to verifie the melting temperature of primers (try between 55°C and 65°C) and if the amplification of the genes work.
Phytic acid dosage
We dose the phytic acid in the fermented rice with the kit " Phytic Acid (Total Phosphorus) Assay Kit ".
August 27th
Result of PCR (August 26th)
Figure 10 : Electrophoresis colony PCR with temperature gradient on non-transformed yeasts lysed by the zymolyase
We made a PCR gradient on non-transformed yeast to know exactly wich temperature is better to a good fixation of the primers on the DNA.
The amplification failed, we supposed it is because our MasterMix did not work. We will try this PCR again.
Phytic acid dosage
We dose the phytic acid in fermented rice.
After analysis, the fermented rice have very little quantity of phosphorus and of phytic acid.
[Phosphorus] : 0.0758 mg/100g of rice.
[Phytic acid] : 0.269 mg/100g of rice.
Figure 11 : Phosphorus calibration curve for titration of phytic acid in fermented rice
Result of second PCR
Figure 12 : Second electrophoresis colony PCR with temperature gradient (non-transformed yeast)
We watch bands for the gene PHO80, at the good size : 882bp. But the gene PHO85, there was no amplifiction, and the positive control is negative : we only see aband bigger than 10,000bp and it is not what we expected.
We try again this PCR to see if the no amplification of the gene PHO85 it is a manipulation error or not.
August 28th
Phytic acid dosage in different strains
Figure 13 : Phosphorus calibration curve + results of OD in some strains
Figure 14 : Results of acid phytic dosage on fermented rice
Results of PCR
Figure 15 : Third electrophoresis gradient PCR (non-transformed yeast)
We watch bands for the both of genes, but not at the same temperatures. Thanks to the gradient, we can suppose that oligos have not exactly the same melting temperature, and it may be the reason why the previous PCR failed.
The controle postive with the HO plasmid worked, the band matches with the PHO85 gene size (1020bp).
Figure 16 : Electrophoresis PCR transformed yeasts
The positive control is negative, PCR does not therefore our work.
Transformation
We transform the yeast Saccharomyces cerevisiae SK1 with the resistance to geneticin and the Cre gene. We delete the PHO 85 genes and we replace it by the resistance gene.
Figure 17 :Principle of Cre recombinase
The Cre gene is the sequence that little be cut thanks Cre recombinase and recombined the gene without the sequence between the Cre sequences.
August 30th
Transformation on yeasts with Cre gene
First step, we realize the PCR of Cre and RFP gene with primers (created by Amaury) and plasmid containing RFP. We amplified the gene, its promoter and its terminater.
Protocol : PCR
Second step, we make PCR purification, and check the result with electrophoresis.
Protocol : PCR purification
Third step, we transformed yeast that is already integrated the resistance instead of PHO85 gene. We introduce the Cre and RFP genes instead of PHO80 gene
Protocol : Transformation of yeast
Fourth step, we cultived the transforming yeasts in YPD agar medium with geneticin overnight.
PCR and result
We did electrophoresis of transformation product. It's revealed bands around 10,000bp, while we expected 1,000bp. We think the DNA extraction in yeast doesn't work, and these bands are contamination.
Figure 18 : Colony PCR on transformed yeast with RFP
September 1st
Result of transformation
We watching any culture on plates YPD + Geneticin. The yeast are not resistant to geneticin, they had not introduced the resistance gene. Transformation does not work.
September 2nd
Transformation
We start again the transformation of august 28th, with the same protocol.
Titration of phytic acid
We test the quantity of phytic acid different strains, at dilution 1/5, 1/50 and 1/500.
Strains : g15,37 / g15,55 / g15,56 / g15,57 / g15,58 / Sook dall + HCl / Sook rice + HCl.
September 4th
After the transformation of September 2nd, any culture is observable on plates. The tansformation are not function. We restart transformation, but we transform with Cre and RFP genes. It is more interesting because we not forced to simply remove the resistance gene, we can make a double identification. It helps to identify yeast that is already integrated in place of the resistance gene PHO85 (they cultivate on agar with geneticin) and we also integrate the RFP in place of PHO80 gene (they will be red).
September 8th
PCR to product cassette Cre/RFP
We restart PCR with Cre sequence and RFP gene.
Figure 19 : Electrophoresis of Cre and RFP amplification
We see only one band. We can continue experiments with this sample and realize the transformation.
Analysis of results of titration phytic acid of September 2nd
We test the quantity of phytic acid different strains, at dilution 1/5, 1/50 and 1/500.
Strains : g15,37 / g15,55 / g15,56 / g15,57 / g15,58 / Sook dall + HCl / Sook rice + HCl.  We start to do the phosphorus scale.
Figure 20 : Phosphorus calibration curve
With calculations of the protocol, we can determine the concentration in our samples.
Protocol : Titration of phytic acid
Figure 21 : Table of results
Concentrations are very low. There are less to 1mg/100g d’idli.
September 11th
Transformation
We did the transformation on strain Saccharomyces cerevisiae SK1, using the protocol of heat shock yeast transformation .
September 14th
Genomic DNA extraction of transformed yeast
We try to extract genomic DNA of transformed yeast with the "Blood & Tissue extraction Kit"
We make PCR on it.
Result of PCR
The gel electrophoresis revealed any bands. We might suppose that the transformation did not work, but it's probably that we don't succeed to extract the yeast's DNA which prevent us to verify our results.
September 15th
Plate of yeast transformed with RFP
Figure 22 : Transformed S. cerevisiae SK1 on YPD agar + geneticin
Colonies are not isolated, we cannot distinguish all of them. The culture that was inoculate was too concentrate. We make suspensions low concentrate and inoculate again on YPD medium with geneticin at 200µg/mL.
September 16th
Plate of yeast transformed with RFP
Figure 23 : Transformed S. cerevisiae SK1 on YPD agar + geneticin
All colonies are the same, but there is no red coloration.
These result can be explicated by the fact that when this gene is introduced in the genomic DNA, it's not revealed, whereas it is in a plasmid.
Test new transformed yeast
We test our new transformed yeast which deletion of PHO80 and PHO85 genes and compare with yeast transformed previously (first with the geneticin resistance gene in place PHO85 gene and second with the  geneticin resistance gene in place PHO80). We put this yeast in YPD + geneticin medium (YPD + geneticinn for always keep the selection). We have grown the yeast with six concentration of phytic acid (0.005, 0.01, 0.02, 0.04, 0.1, 0.2 g/mL). We also realize a scale with phytic acid and YPD medium + geneticin.
Scale: C(phytic acid;YPD) (g.mL-1) 0,005 0,01 0,02 0,04 0,1
Figure 24 :Table manipulation
We don’t have enough reagents in our kit to do all manipulations we want, and it is too late to buy it. We choose to delete three concentrations by strain and in the scale.
Figure 25 :New table manipulation
Figure 26 :Result of test
Result analysis: We have only optic density result below zero. Normally is no possible. We suppose the reagents of the kit are old and it not work. Would have had to do again the tests but there is not reagents and time anymore.
