<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Notebook_VitaminB2 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Paris Bettencourt/Notebook/VitaminB2</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="menuContainer"><DIV id="smallMenuTitle"><H1>Ferment It Yourself</H1><H3>iGEM Paris-Bettencourt 2O15</H3></DIV><UL><LI class="left"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Background" title="Background">Background</A></LI><LI class="left"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Design" title="Design">Design</A></LI><LI class="left"><BUTTON class="menuButton">Project</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Micronutrients">Micronutrients</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Continuity">Continuity</A></LI><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Manufacturing">Manufacturing</A></LI></DIV></UL><LI class="left"><BUTTON class="menuButton">Achievements</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Parts">Parts</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Medals">Medal Fulfillments</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Practices</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/SynBio_Challenge">SynBio Challenge</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/DILambda">DIλ</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Acceptance">Acceptance</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/Events">Events</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/Rhizi">Rhizi</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Collaborations">Collaborations</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Safety</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Safety">Safety Overview</A></LI><LI><A href="https://2015.igem.org/Safety/Final_Safety_Form?team_id=1678">Safety Form</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/ReviewBoard">Review Board</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Notebook</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Notebook">Daily notebook</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols">Protocols</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Bibliography">Bibliography</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Team</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Team">Members</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Acknowledgments">Acknowledgments</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Press">Press</A></LI><LI><A href="https://igem.org/Team.cgi?id=1678">Official Team Profile</A></LI></DIV></DIV><DIV id="notebook"><DIV class="contentContainer"><DIV id="banner"><H3>Notebook - Vitamin B2
</H3></DIV></DIV><DIV class="textWrapper"><DIV class="textContent {{{page_option}}}" id="caffeine"><DIV id="notebookMenu"><UL><LI><A href="#july">July</A></LI><LI><A href="#august">August</A></LI><LI><A href="#september">September</A></LI></UL></DIV><DIV id="notebookContent"><H2 class="date one">July 13th</H2><UL><LI>Received gBlocks RibA, RibD, RibE, RibT25 and RibT48 and oligos for amplification from IDT.</LI>

        Diluted in water and used in PCR amplification, using the following protocol:
        <UL><LI>1 μL gBlock (0.1 to 1ng)
            </LI><LI>1 μL forward primer (10 μM)
            </LI><LI>1 μL reverse primer (10 μM)
            </LI><LI>22 μL DNAse/RNAse free water
            </LI><LI>25 μL LifeTech MasterMix (2X)
        </LI></UL><B>RibA</B> + <B>o15.001</B> (GCGCCCGAAGACTTATGCAG) + <B>o15.002</B>(GGCCCCGCGCATATGAAG)<B>RibD</B> + <B>o15.003</B>(CGCTATAGAAGACTTGAGAAGATCTG) + <B>o15.004</B>(GCGCGGCACCACATATGAAG)<B>RibE</B> + <B>o15.005</B>(CGGCTATAGAAGACTTGCGC) + <B>o15.006</B>(CGCGCCGGCATATGAAGA)<B>RibT25</B> + <B>o15.007</B>(CCGCGTATAGAAGACTGCTAGA) + <B>o15.008</B>(CAGCAGCATATGAAGACAACCC)<B>RibT48</B> + <B>o15.009</B>(GCGGTATAGAAGACTGCTAGAGA) + <B>o15.010</B>(CAGCAGCATATGAAGACAACCC)
        For each PCR reaction, a negative control without matrix DNA was prepared.12 cycles amplification, using the following parameters:
        <TABLE style="width:500px"><TBODY><TR><TH>time (min)</TH><TH>temperature (°C)</TH><TH>function</TH></TR><TR><TD>3:00</TD><TD>98</TD><TD>melting</TD></TR><TR><TD>0:30</TD><TD>98</TD><TD>melting</TD></TR><TR><TD>0:30</TD><TD>52</TD><TD>annealing</TD></TR><TR><TD>1:00</TD><TD>72</TD><TD>extension</TD></TR><TR><TD>10:00</TD><TD>72</TD><TD>extension</TD></TR><TR><TD>infinite</TD><TD>12</TD><TD>storage</TD></TR></TBODY></TABLE>

        PCR purification using QIAGEN kit

        <TABLE style="width:500px"><TBODY><TR><TD><UL><B>PCR purification protocol</B><LI>Add 5 volumes of resuspension buffer to 1 volume of PCR product in an 1.5mL microcentrifuge tube, mix by pipetting up and down
                        </LI><LI>Transfer in a centrifugation column
                        </LI><LI>Centrifuge 1 min at 14000 rpm
                        </LI><LI>Throw the filtration product
                        </LI><LI>Add 700μL of washing solution
                        </LI><LI>Centrifuge 1 min at 14000 rpm
                        </LI><LI>Throw the filtration product
                        </LI><LI>Add 500μL of washing solution
                        </LI><LI>Centrifuge 1 min at 14000 rpm
                        </LI><LI>Throw the filtration product
                        </LI><LI>Centrifuge 1 min at 14000 rpm
                        </LI><LI>Throw the filtration product
                        </LI><LI>Put the column in a sterile 1.5mL microcentrifuge tube
                        </LI><LI>Add 45μL of DNAse/RNAse free water on the membrane
                        </LI><LI>Wait 2 minutes
                        </LI><LI>Centrifuge 2min at 10000rpm
                        </LI><LI>Discard the column, DNA is saved in water
                </LI></UL></TD></TR></TBODY></TABLE>


    Concentrations measured with a Nanodrop:

    <TABLE style="width:500px"><TBODY><TR><TH>[PCR product with gBlock] (ng/μL)</TH><TH>[PCR product without gBlock] (ng/μL)</TH></TR><TR><TD>RibA</TD><TD>3.5</TD><TD>3.4</TD></TR><TR><TD>RibD</TD><TD>6.4</TD><TD>3.3</TD></TR><TR><TD>RibE</TD><TD>5.0</TD><TD>3.7</TD></TR><TR><TD>RibT25</TD><TD>3.5</TD><TD>3.8</TD></TR><TR><TD>RibT48</TD><TD>8.7</TD><TD>3.3</TD></TR></TBODY></TABLE>

    Almost no difference between the PCR and their corresponding negative control. 
    It could be the fallout of the limited number of cycles. (We only used 12 cycles, as advised by IDT)
    NEB  Tm calculator gave really different Tm  for the PCR primers than Geneious and IDT. 
    We decided to launch two PCR, both with 35 cycles, but with two different annealing
    temperature: 52 and 64°C.
    <H2 class="date one">July 14th</H2>

    Launched the two different PCR with different Tm.
    PCR with high annealing temperature, 35 cycles:<TABLE style="width:500px"><TBODY><TR><TH>time (min)</TH><TH>temperature (°C)</TH><TH>function</TH></TR><TR><TD>3:00</TD><TD>98</TD><TD>melting</TD></TR><TR><TD>0:30</TD><TD>98</TD><TD>melting</TD></TR><TR><TD>0:30</TD><TD>64</TD><TD>annealing</TD></TR><TR><TD>1:00</TD><TD>72</TD><TD>extension</TD></TR><TR><TD>10:00</TD><TD>72</TD><TD>extension</TD></TR><TR><TD>infinite</TD><TD>12</TD><TD>storage</TD></TR></TBODY></TABLE>

    PCR with low annealing temperature, 35 cycles:<TABLE style="width:500px"><TBODY><TR><TH>time (min)</TH><TH>temperature (°C)</TH><TH>function</TH></TR><TR><TD>3:00</TD><TD>98</TD><TD>melting</TD></TR><TR><TD>0:30</TD><TD>98</TD><TD>melting</TD></TR><TR><TD>0:30</TD><TD>64</TD><TD>annealing</TD></TR><TR><TD>1:00</TD><TD>72</TD><TD>extension</TD></TR><TR><TD>10:00</TD><TD>72</TD><TD>extension</TD></TR><TR><TD>infinite</TD><TD>12</TD><TD>storage</TD></TR></TBODY></TABLE>
    After PCR, we ran the PCR products on a TAE - 1% agarose gel (100V, 20min) to check if the amplification product correspond to the expected size of the gBlocks.<B>Figure 1: </B><I>PCR amplification confirmation by electrophoresis</I><I>From the left to the right:
        1kb Ladder, RibA, D, E, T25 and T48 amplified at 52°C and Rib A, D, E, T25 and T48 amplified at 64°C</I>


    The bands are not really specific and seems to be smeared over the gel.
    However, we can see that the extremity of each band correspond approximatively 
    to the expected size of our parts, except RibD amplified at 64°C for which no band has been detected.

    We purified our PCR product according to the previously used protocol, and then measure the DNA concentration using a Nanodrop.

    <TABLE style="width:500px"><TBODY><TR><TH>Part Name </TH><TH>PCR product amplified at 52°C concentration (ng/μL)</TH><TH>PCR product amplified at 64°C concentration (ng/μL)</TH></TR><TR><TD>RibA</TD><TD>54.3</TD><TD>132.7</TD></TR><TR><TD>RibD</TD><TD>50.5</TD><TD>140.2</TD></TR><TR><TD>RibE</TD><TD>67.1</TD><TD>78.2</TD></TR><TR><TD>RibT25</TD><TD>63.6</TD><TD>136.0</TD></TR><TR><TD>RibT48</TD><TD>61.2</TD><TD>143.1</TD></TR></TBODY></TABLE><H2 class="date one">July 16th</H2>

    Annealing of o15.011 (TCGACCATGCTTGTCTTCGAAGACTTGGGGGAT) and o15.012 (CTAGATCCCCCAAGTCTTCGAAGACAAGCATGG) using the following protocol:
    <TABLE><TBODY><TR><TD><B>Annealing Protocol</B><UL><LI>Phosphorylation of the oligos
                        </LI><LI>        5.6μL	DNAse/RNAse free water
                            </LI><LI>	6.0μL   o15.011 (10µM)
                            </LI><LI>	6.0μL   o15.012 (10µM)
                            </LI><LI>	2.0μL   10X T4 DNA ligase buffer
                            </LI><LI>	0.4μL   T4 PolyNucleotide Kinase
                                Total:   20μL

                        </LI></UL><LI>incubate 30min at 37°C
                    </LI><LI>add 1μL of 1M NaCl
                    </LI><LI>incubate 5min at 95°C
                    </LI><LI>let the mix cool down
                    </LI><LI>use 2μL of the mix as a 10X solution
                </LI></TD></TR></TBODY></TABLE>
    Inoculate LB + erythromycin (150ug/μL) with g15.21 containing pKV6.
    Inoculate M17 + erythromycin (10ug/μL) with g15.13 containing pSIP411.


    <H2 class="date one">July 17th</H2>
    Miniprep of g15.13 and g15.21 as describe below:
    <TABLE style="width:500px"><TBODY><TR><TD><B>Miniprep protocol using a PROMEGA kit </B><UL><LI>centrifuge an overnight culture of cells 10min at 4krpm
                    </LI><LI>throw the filtrate
                    </LI><LI>resuspend the pellet in 250μL of Cell Resuspension Solution then mix it
                    </LI><LI>transfer it in a 1.5mL microcentrifuge tube
                    </LI><LI>add 250mL of Cell lysis solution and mix by inverting several times
                    </LI><LI>incubate until the liquid is clear, maximum 5min
                    </LI><LI>add 10μL of Alkalyne protease solution, mix by inverting several times and incube 3 to 4 min
                    </LI><LI>add 350μL of Neutralisation Solution and mix by inverting several times
                    </LI><LI>centrifuge 10min at 14krpm
                    </LI><LI>add the supernatant in a column
                    </LI><LI>centrifuge 1min, 14krpm, then throw the filtrat
                    </LI><LI>add 750μL Washing Solution
                    </LI><LI>centrifuge 1min, 14krpm, then throw the filtrat
                    </LI><LI>add 250μL Washing Solution
                    </LI><LI>centrifuge 2min, 14krpm, then throw the filtrat
                    </LI><LI>centrifuge 1min, 14krpm
                    </LI><LI>transfer the column in a sterile microcentrifuge 1.5ml tube
                    </LI><LI>add 50μL of DNAse/RNAse free water right on the membrane of the filter, wait 1min
                    </LI><LI>centrifuge 1min, 14krpm
                    </LI><LI>throw the column, plasmid is saved in water
                </LI></UL></TD></TR></TBODY></TABLE>
    As g15.13 is a gram positive bacteria, we added 1mL of 10mg/mL Lysozyme at the same time that the cell lysis solution.
    
    Measurement of DNA concentration

    <H2 class="date one">July 20th</H2>


    -Annealing of o15.072 (TCGACCATGCTTGTCTTCGAAGACTTGGGGGAG) and o15.073 (AATTCTCCCCCAAGTCTTCGAAGACAAGCATGG) thanks to the protocol used previously.

    
    -PCR of pSIP411 ORI and resistance cassette (erythromycin) using o15.070 and o15.071.
    We will call it pSIPnew to have clearer explanations.
    The size of this fragment is 3.1kb, so the elongation phase was extended to 2min.
    a negative PCR control was made at the same time (without matrix DNA).
    PCR purification after the PCR and concentration measurement:<TABLE style="width:500px"><TBODY><TR><TH>Concentration (ng/μL)</TH></TR><TR><TD>pSIPnew</TD><TD>99.0</TD></TR><TR><TD>negative control</TD><TD>0.0</TD></TR></TBODY></TABLE><H1 class="date one">July 21st</H1>


    Digestion of amplified band of pSIPnew and pKV6 with respectively XbaI/SalI and EcoRI/SalI using the following protocol.

    <TABLE style="width:500px"><TBODY><TR><TD><B>Digestion Protocol</B><UL><LI>Prepare the following mix:</LI><LI>4μL of Enzyme 1</LI><LI>4μL of Enzyme 2</LI><LI>4μL of FastAP</LI><LI>12μL of Fast Digest buffer 10X</LI><LI>1 to 3 μg of DNA</LI><LI>up to 120μL of water</LI></UL><LI>mix by pipetting up and down</LI><LI>incubate at 37°C, 10 min for FD, 45 min for regular enzymes</LI><LI>incubate 10min at 70°C</LI></TD></TR></TBODY></TABLE>

    For pSIPnew: SalI and XbaI, 30μL of DNA (99ng/μL), 66μL of water.
    For pKV6: SalI and EcoRI, 10μL of DNA(270ng/μL), 86μL of water.
    PCR purification of both digested pKV6 and pSIPnew
    Measure the DNA concentration after PCR purification:<TABLE style="width:500px"><TBODY><TR><TH>Concentration (ng/μL)</TH></TR><TR><TD>pSIPnew</TD><TD>60.8</TD></TR><TR><TD>pKV6</TD><TD>26.4</TD></TR></TBODY></TABLE>

    Ligation of annealed oligos o15.011 and o15.012 with digested pSIP411new and o15.072 and o15.073 with digested pKV6 using the following protocol

    <TABLE style="width:500px"><TBODY><TR><TD><B>Ligation Protocol</B><UL><LI>Mix the following</LI><UL><LI>vector 100ng</LI><LI>insert 300ng</LI><LI>2µL T4 DNA ligase buffer 10X</LI><LI>1µL T4 DNA ligase</LI><LI>up to 20µL of water</LI></UL><LI>incubate 30 to 40 minutes at room temperature</LI></UL></TD></TR></TBODY></TABLE>

    For pKV6: 4µL of digested pKV6(26.4ng/µL), 7µL of annealed o15.072/o15.073, 6µL of water.
    For pSIPnew: 2µL of digested pKV6(60.8ng/µL), 6µL of annealed o15.011/o15.012, 9µL of water.
    pKV6 ligated with o15.072/o15.073 and pSIPnew ligated with o15.011/o15.012 will be respectively called p15.01 and p15.02.
    Transformation of p15.01, p15.02, pKV6(miniprep), pSIP411(miniprep) and a negative control (no DNA) in NEB DH5α chemically competent cells<TABLE style="width=500px"><TBODY><TR><TD><B>Heat Shock transformation protocol</B><UL><LI>Thaw frozen chemically competent cells (20µL aliquots) on ice for 10min.</LI><LI>add 2µL of ligation product (or 0.5µL of miniprep) and incubate the cells 30sec at 42°C.</LI><LI>put the cells back on ice for 2min.</LI><LI>add 200µL of LB to the cells and incubate 2 hours at 37°C.</LI><LI>plate the cells on LB + erythromycin (150µg/mL) or LB + erythromycin (10µg/mL), incubation at 37°C overnight.</LI></UL></TD></TR></TBODY></TABLE><H1 class="date one">July 22nd</H1><B>Transformation results</B><TABLE style="width:1000px"><TBODY><TR><TH>Transformed product</TH><TH>pKV6</TH><TH>pSIP411</TH><TH>p15.01</TH><TH>p15.02</TH><TH>negative control</TH></TR><TR><TD>Growth Results</TD><TD><UL><LI>(ery 150µg/mL):
                500 colonies, no background</LI></UL></TD><TD><UL><LI>(ery 10µg/mL):
                    no growth</LI><LI>(ery 150µg/mL):
                    lawn of bacteria</LI></UL></TD><TD><UL><LI>spreading of 170µL:
                        41 isolated colonies -&gt; isolation of three transformants: T1, T2 and T3</LI><LI>spreading of 50µL:
                        3 colonies on a lawny background</LI></UL></TD><TD><UL><LI>(ery 10µg/mL):
                    lawn of bacteria</LI><LI>(ery 150µg/mL):
                    lawny background</LI></UL></TD><TD> no growth </TD></TR></TBODY></TABLE><H1 class="date one">July 23rd</H1>


    Preparation of DH5α Electrocompetent cells, 50 tubes of 100µL.

    <TABLE style="width:500px"><TBODY><TR><TD><B>Electrocompetent Cells Preparation Protocol</B><UL><LI>Inoculate two 250mL LB flasks with 100µL of an overnight culture of DH5α</LI><LI>incubate until the the DO600 reach 0.5 to 0.7</LI><LI>place the cultures on ice for 15 minutes</LI><LI>pour the culture in cold sterile 50mL falcon tubes</LI><LI>centrifuge them for 10 minutes at 6000rpm</LI><LI>throw the supernatant</LI><LI>resuspend the cells in 50mL cold distilled water</LI><LI>centrifuge them for 10 minutes at 6000rpm</LI><LI>throw the supernatant</LI><LI>resuspend the cells in 25mL cold distilled water</LI><LI>centrifuge them for 10 minutes at 6000rpm</LI><LI>throw the supernatant</LI><LI>resuspend the cells in 12.5mL cold 10% glycerol</LI><LI>centrifuge them for 10 minutes at 6000rpm</LI><LI>throw the supernatant</LI><LI>resuspend the cells in 5mL cold 10% glycerol</LI><LI>make aliquots of the desire volume in microcentrifuge tubes and freeze them at -80°C</LI></UL></TD></TR></TBODY></TABLE>

    overnight cultures of 3 p15.01 transformants.





    <H1 class="date one">July 24th</H1>

    Miniprep of the 3 overnight cultures of p15.01 transformants T1, T2 and T3(respective concentration: 204.8, 160.7 and 267.7 ng/µL).
    Digestion of pKV6(negative control) and p15.01 with BbsI and Eco31I to control the insertion of the two BbsI sites in p15.01.
    <TABLE style="width:500px"><TBODY><TR><TD><B>Analytical digestion protocol</B><UL><LI>Prepare the following mix:
                        </LI><LI>2µL 10X Digestion buffer</LI><LI>0.5µL Eco31I</LI><LI>0.5µL BbsI</LI><LI>2µL of DNA (200ng)</LI><LI>15µL water</LI></UL><LI>incube 1h at 37°C</LI></TD></TR></TBODY></TABLE><B>HERE IS THE PICTURE OF THE CORRESPONDING GEL</B>
    Both T2 and T3 transformants present the same bands as pKV6, whereas T1 shows 3 bands at 2.7kb, 1.7kb and 1.4kb.
    Oligos used for sequencing were <B>o15.097</B>(CGGTAGAGCTCCCTTCTATGC) and <B>o15.098</B>(CTGGCACGACAGGTTTCCC).
    Overnight of T1 in LB+ery(150µg/mL).
    Dialyse of both ligation products(p15.01 and p15.02) and both native plasmids (pKV6 and pSIP411) on 0.025µm cellulose filter for 20 minutes.
    Transformation of DH5α by electroporation with the previously ligated p15.01 and p15.02.<TABLE style="width:500px"><TBODY><TR><TD><B>Electroporation Protocol</B><UL><LI>Thaw electrocompetent cells on ice</LI><LI>Add 2µL of ligation product or 0.5µL of native plasmid to the cells</LI><LI>Transfer the cells in an 0.2cm electroporation cuvette</LI><LI>put the cuvette in the electroporation device and pulse the cells at 2.5kV, 200 Ohms and 25µF</LI><LI>add 200µL of LB right after pulsing</LI><LI>recover 2 hours at 37°C</LI><LI>plate 200µL and 50µL of the cells on LB + erythromycin (200µg/mL), incubate overnight at 37°C</LI></UL></TD></TR></TBODY></TABLE>
    Cells were plated on two LB + erythromycin (200µg/mL) plates. The first was inoculated with 50µL, the second with
    µL.





    <H1 class="date one">July 25th</H1><B>Transformation results</B><TABLE style="width:500px"><TBODY><TR><TH>Transformed product</TH><TH>pKV6</TH><TH>pSIP411</TH><TH>p15.01</TH><TH>p15.02</TH><TH>negative control</TH></TR><TR><TD>Growth Results</TD><TD>lawn of unrecognized bacteria with dense colonies</TD><TD><UL><LI>200µL:
                        11 single colonies</LI><LI>50µL:
                        7 single colonies</LI></UL></TD><TD><UL><LI>200µL:
                        32 single colonies</LI><LI>50µL:
                        0 colony</LI></UL></TD><TD><UL><LI>200µL:
                        thin lawn of unrecognized bacteria with dense colonies</LI><LI>50µL:
                        thin lawn of unrecognized bacteria</LI></UL></TD><TD>
                thin lawn of unrecognized bacteria
            </TD></TR></TBODY></TABLE>

    As the negative control showed a growth, we discarded these results and decided to perform a second transformation (dialyse of the previous day ligation products + transformation) with the same ligation products, using the same protocol.
    p15.01 product was lost during dialysis.
    Time constants were 5.3ms for all the pulses.
    pKV6 was plated on LB + erythromycin (200µg/mL); pSIP411 and p15.02 were plated on LB + erythromycin (150µg/mL) and LB + erythromycin (200µg/mL).





    <H1 class="date one">July 27th</H1><B>Transformation results</B><TABLE style="width:500px"><TBODY><TR><TH>Transformed product</TH><TH>pKV6</TH><TH>pSIP411</TH><TH>p15.01</TH><TH>p15.02</TH><TH>negative control</TH></TR><TR><TD>Growth Results</TD><TD>158 single colonies</TD><TD><UL><LI>200µg/mL:
                        thin lawn of unrecognized bacteria</LI><LI>150µg/mL:
                        thin lawn of unrecognized bacteria</LI></UL></TD><TD>lost during dialysis
            </TD><TD><UL><LI>200µg/mL:
                        thin lawn of unrecognized bacteria</LI><LI>150µg/mL:
                        thin lawn of unrecognized bacteria</LI></UL></TD><TD>
                no growth
            </TD></TR></TBODY></TABLE>


    To prevent the transformation of the native pSIP411 (non-digested or re-circularized), we decided to PCR the 3kb fragment containing the ORI and the erythromycin resistance gene, digest it with SalI/XbaI, PCR purify it and then gel purify it.
    This would give us only the fragment we want to ligate and thus prevent background transformants to appear.
    
    For the PCR, <B>o15.070</B>(AAAAACCGCAGGGAGGCAAACAATGA) and <B>o15.071</B>(CGGATTAACGTTAAGAACTCTATTGAAGG) were used to amplify the 3 kb fragment, with an annealing temperature of 69°C (according to NEB Tm calculator) and an elongation time span of 2 minutes.
    All the PCR purified products were allowed to run on an 1% agarose gel, and gel extraction was performed on the band of approximately of size 3kb, according to the following protocol.<TABLE style="width:500px"><TBODY><TR><TD><B>Gel Extraction Protocol with QIAGEN kit</B><UL><LI>add three times the volume of QG solubilisation buffer with one volume (=mass) of gel extracted</LI><LI>incubate 10 minutes at 50°C, vortex every 2 or 3 minutes, until the gel is dissolved</LI><LI>transfer in a provided purification column, centrifuge 1 minute at 14000rpm then discard the filtrat</LI><LI>add 500µL of solubilisation buffer, centrifuge 1 minute at 14000rpm then discard the filtrat</LI><LI>add 750µL of washing buffer</LI><LI> centrifuge 1 minute at 14000rpm then discard the filtrat</LI><LI>add 500µL of washing buffer</LI><LI> centrifuge 1 minute at 14000rpm then discard the filtrat</LI><LI>centrifuge 1 minute at 14000rpm then discard the filtrat</LI><LI>put the column in a clean 1.5mL microcentrifuge tube</LI><LI>add 45µL of DNAse/RNAse free water right on the membrane of the column</LI><LI>wait a couple of minutes and then centrifuge 1 minute at 10000rpm</LI><LI>discard the column, DNA is saved in water</LI></UL></TD></TR></TBODY></TABLE>
    Concentration measured with Nanodrop was <B>INSERT CONCENTRATION HERE </B>

    To get a fresher batch of annealed oligos, we decided to anneal them again, using the same protocol.
    (o15.011+o15.012 and o15.072+o15.073).

    The three minipreps from p15.01 transformants were sent to sequence thanks to GATC LightRun NightXpress service.
    5µL (~100ng/µL) of all three minipreps were mixed with <B>o15.097</B> (forward primer, CGGTAGAGCTCCCTTCTATGC) or <B>o15.098</B> (reverse primer, CTGGCACGACAGGTTTCCC).<H1 class="date one">July 28th</H1>

    Measurement of DNA concentration after digestion/gel extraction:
    <TABLE style="width:500px"><TBODY><TR><TH>Concentration (ng/μL)</TH></TR><TR><TD>pKV6</TD><TD>44.8</TD></TR><TR><TD>pSIP411 3kb fragment</TD><TD>79.5</TD></TR></TBODY></TABLE>
    Ligation, dialyse and transformation by electroporation of pSIP411 3kb band + o15.011/o15.012, digested pKV6+o15.072/o15.073 and both native plasmids (pKV6 and pSIP411).
    Cells were then plated with beads on LB+erythromycin (200µg/mL) and incubated at 37°C.
    

    Sequencing results:
    sequencing confirm the insertion of the two BbsI sites in the T1 transformant(=p15.01 plasmid).
    Both T2 and T3 did not contain the insert.
    T1 was glycerol stocked for long-term conservation, and renamed as <B>g15.49</B>.<H1 class="date one">July 30th</H1><B>Transformation results</B><TABLE style="width:500px"><TBODY><TR><TH>Transformed product</TH><TH>pKV6</TH><TH>pSIP411</TH><TH>p15.01</TH><TH>p15.02</TH><TH>negative control</TH></TR><TR><TD>Growth Results</TD><TD>
                thin lawn of unrecognized bacteria
            </TD><TD>
                thin lawn of unrecognized bacteria
            </TD><TD>
                thin lawn of unrecognized bacteria
            </TD><TD>
                thin lawn of unrecognized bacteria
            </TD><TD>
                thin lawn of unrecognized bacteria
            </TD></TR></TBODY></TABLE>


    Golden Gate assembly of RibADET25 and RibADET48 in p15.01 and transformation of chemically competent NEB Turbo ''E.coli''.
    The two new plasmids are respectively named p15.06 and p15.07.
    
    Golden Gate was performed according to the following protocol:
    <TABLE style="width:500px"><TBODY><TR><TD><B>Golden Gate protocol</B><UL><LI>Mix the following in a PCR tube:</LI><UL><LI>1µL T4 DNA ligase buffer</LI><LI>1µL BbsI HF</LI><LI>2µL NEB GoldenGate Buffer 10X</LI><LI>200ng backbone</LI><LI>2:1 molarity ratio (insert:vector) of each insert</LI><LI>up to 20µL with water</LI></UL><LI>incubate using the following:</LI></UL></TD></TR><TR><TD><B>Time (minute)</B></TD><TD><B>Temperature (°C)</B></TD><TD><B>Function</B></TD></TR><TR><TD>3</TD><TD>37</TD><TD>Digestion</TD></TR><TR><TD>4</TD><TD>16</TD><TD>Ligation</TD></TR><TR><TD>5</TD><TD>50</TD><TD>Melting</TD></TR><TR><TD>5</TD><TD>80</TD><TD>Enzyme inactivation</TD></TR><TR><TD>infinite hold</TD><TD>12</TD><TD>Storage</TD></TR></TBODY></TABLE></UL>

    Heat shock transformation of both Golden Gate products, p15.01(miniprep), p15.02(ligation product) and pKV6. Recovery two hours at 37°C and plating on LB + erythromycin (150µg/mL).

    Inoculated LB with NEB Turbo, overnight at 37°C




    <H1 class="date one">July 31st</H1><B>Transformation results</B><TABLE style="width:500px"><TBODY><TR><TH>Transformed product</TH><TH>pKV6</TH><TH>pSIP411</TH><TH>p15.01</TH><TH>p15.02</TH><TH>p15.06</TH><TH>p15.07</TH><TH>negative control</TH></TR><TR><TD>Growth Results</TD><TD>
                204 isolated colonies (0.5 mm)
            </TD><TD>
                137 isolated colonies (0.5 to 1 mm)
            </TD><TD>
                half lawn, half isolated colonies (~500)
            </TD><TD>
                lawn
            </TD><TD>
                lawn and some isolated colonies
            </TD><TD>
                ~1000 tiny isolated colonies
            </TD><TD>
                no growth
            </TD></TR></TBODY></TABLE>

    Striked a sample of p15.02 lawn on LB + ery (150µg/mL).
    Striked a sample of p15.06 lawn on LB + ery (150µg/mL).
    Striked a single colony of p15.07 on LB + ery (150µg/mL).

    Incubated overnight at 37°C.

    Seeing all the electroporation results so far, we decided to prepare an new batch of electrocompetent NEB Turbo (using the same protocol).
    Overnight culture of g15.49 (containing p15.01) in LB + ery (150µg/mL).<H1 class="date two">August 1st</H1>


    Plating Results:
    <TABLE><TBODY><TR><TH>p15.02</TH><TH>p15.06</TH><TH>p15.07</TH></TR><TR><TD>big single colonies</TD><TD>big single colonies</TD><TD>big single colonies</TD></TR></TBODY></TABLE>

    Inoculated 5mL LB + ery (150µg/mL) with 3 p15.02 colonies, 4 p15.06 colonies and 3 p15.07 colonies, and a negative growth control (non-inoculated LB), incubated overnight at 37°C.

    Miniprep of the g15.49 overnight =&gt; p15.01 at 330ng/µL.






    <H1 class="date two">August 2nd</H1>

    All the cultures incubated on the 01/08 grew, except the negative control, and were minipreped.
    The three p15.02 transformants were sent for sequence with <B>o15.099</B> (forward primer, GCCAGCGGAATGCTTTCATCC) or  <B>o15.100</B> (reverse primer, CATGAATTAGTCTCGGACATTCTGC).

    <H1 class="date two">August 3rd</H1>

    Restriction Digestion of p15.01 with Eco31I and BbsI (using the digestion protocol) and PCR purification to remove the enzymes.
    Picked 10 single colonies from both p15.06 and p15.07 and launched overnight cultures in LB + ery (150µg/mL).<H1 class="date two">August 4th</H1>

    Miniprep of all the overnight cultures from clones of p15.06 and p15.07.
    We did restriction digestion with Eco31I on p15.06 and p15.07 minipreped.
    p15.06 and p15.07 restriction analysis should produce 6 fragments (first and second may be overlapping) (2.77, 2.7, 1.7, 1.3, 0.9 and 0.6 in kb size),
    whereas p15.01 restriction analysis should only produce two fragments(2.7, 3.4 in kb size)<B>Figure 2: </B><I>Restriction analysis of p15.01, p15.06 and p15.07 with Eco31I on 1% agarose gel
    From left to right: p15.01 non digested, p15.01 digested, p15.06 (6 colonies), p15.07 (2 colonies), generuler 1kb, p15.07 (6 colonies)</I>

    All the digested products are presenting only two bands (and some bands likely to come from Eco31I star activity)us are not containing the four inserts.
    

    We decided to perform an other golden gate/dialyse/transformation with p15.01, p15.06 and p15.07.
    According to several advice we received, we tried two different golden gate protocols, the first one using the 2:1 insert:backbone ratio, the second using a 3:1 ratio.
    Time constants for each electroporation was about 5.2 or 5.3 msec.
    With two BbsI enzyme tubes present in our stock, we tested both (A and B).
    After one hour of recovery, two volumes (50µL and 170 µL) of each transformed cells were plated on LB + ery (150µg/mL) and incubated overnight at 37°C.<H1 class="date two">August 5th</H1><B>Transformation results</B><TABLE style="width:500px"><TBODY><TR><TH>Transformed product</TH><TH>p15.01</TH><TH>p15.06, BbsI #A, 3:1 ratio</TH><TH>p15.06, BbsI #B, 2:1 ratio</TH><TH>p15.07, BbsI #A, 3:1 ratio</TH><TH>p15.07, BbsI #B, 2:1 ratio</TH><TH>Golden Gate negative control (golden gate reaction without inserts)</TH><TH>negative growth control</TH></TR><TR><TD>more than 500 colonies (1 to 3mm in diameter)</TD><TD><UL><LI>50µL:</LI>
                    0 colony
                    <LI>170µL:</LI>
                    5 colonies
                </UL></TD><TD><UL><LI>50µL:</LI>
                    16 colonies
                    <LI>170µL:</LI>
                    53 colonies
                </UL></TD><TD><UL><LI>50µL:</LI>
                    4 colonies
                    <LI>170µL:</LI>
                    65 colonies
                </UL></TD><TD><UL><LI>50µL:</LI>
                    29 colonies
                    <LI>170µL:</LI>
                    152 colonies
                </UL></TD><TD><UL><LI>BbsI #A, 170µL:</LI>
                    0 colony
                    <LI>BbsI #B, 170µL:</LI>
                    0 colony
                </UL></TD><TD><UL><LI>50µL:</LI>
                    0 colony
                    <LI>170µL:</LI>
                    5 colonies
                </UL></TD><TD><UL><LI>50µL:</LI>
                    0 colony
                    <LI>170µL:</LI>
                    0 colony
                </UL></TD></TR></TBODY></TABLE>


    To control the insertion, we used colony PCR with <B>o15.097</B>(CGGTAGAGCTCCCTTCTATGC) and <B>o15.098</B>(CTGGCACGACAGGTTTCCC).
    The colony PCR were performed on 10 single colonies of p15.06 #B, 10 single colonies of p15.07 # A, 10 single colonies of p15.07 #B and 3 single colonies of p15.06 #A.
    If the insertion worked, PCR should amplifiate a 4.8kb band, whereas if the insert is not present, a 0.6kb band should be observed.

    <TABLE style="width:500px"><TBODY><TR><TD><B>Colony PCR protocol</B><UL><LI>Prepare the following mix in this order:
                        </LI><LI>up to 20µL water</LI><LI>1µL DNA Template or some cells from a single colonies (save some cells on a plate)</LI><LI>0.5µL Forward Primer (10µM)</LI><LI>0.5µL Reverse Primer (10µM)</LI><LI>2µL dNTPs mix (10µM)</LI><LI>2µL DreamTaq Green Buffer (10X)</LI><LI>0.5µL DreamTaq (5U/µL)</LI></UL><LI>Run a PCR with a longer initial denaturation temperature phase (5 to 8 more minutes) to lyse the cells, and extend the elongation phase to 1 min/kb</LI></TD></TR></TBODY></TABLE>

    PCR analysis on 1% agarose 0.5X TAE gel.<B>Figure 3:</B><I>  Confirmation of Insertion of RibADET operon in p15.01 by Electrophoresis </I><UL><LI> a- From Left to Right:
            p15.07 DreamTaq #A (8 colonies) - 1kb ladder - p15.07 DreamTaq #A (2 colonies) - p15.07 DreamTaq #B (6 colonies)</LI><LI> b- From Left to Right:
            p15.06 DreamTaq #B (8 colonies) - 1kb ladder - p15.06 DreamTaq #B (2 colonies) - PCR negative control (no matrix DNA)  -</LI><LI> c- From Left to Right:
            p15.07 DreamTaq #B (4 colonies)  - 1kb ladder - p15.06 DreamTaq #A (3 colonies) - p15.01 </LI></UL>


    No band was visible. Several explanations are possible:
    <UL><LI>Colony PCR failed</LI><LI>The colonies did not contain p15.01, p15.06 or p15.07, however the presence of colonies after transformation but none in the negative control suggests that at least the native plasmid has been transformed succesfully.</LI></UL>
    Thus, we decided to do the experiment again the next day.

    <H1 class="date two">August 6th</H1>
    We performed again the colony PCR (with o15.097 and o15.098) on the same colonies with a different DreamTaq enzyme.

    PCR analysis on 1% agarose 0.5X TAE gel.<B>Figure 4:</B><I>  Confirmation of Insertion of RibADET operon in p15.01 by Electrophoresis </I><UL><LI> a- From Left to Right:
            p15.07 DreamTaq #A (9 colonies) - 1kb ladder - p15.07 DreamTaq #A (1 colonies) - p15.07 DreamTaq #B (6 colonies) </LI><LI> b- From Left to Right:
            of p15.07 DreamTaq #B (4 colonies) - p15.06 DreamTaq #A (3 colonies) - p15.01 -
            p15.06 DreamTaq #B (1 colonies) - 1kb ladder - p15.06 DreamTaq #B (7 colonies)</LI><LI> c- From Left to Right:
            p15.06 DreamTaq #B (2 colonies) - p15.01 - 1kb ladder - negatice control PCR (no matrix DNA) </LI><LI> d- zoomed view of two bands of figure 4b </LI></UL>

    Even when the gel quality is not really good, we observed a strange 8kb band in most of the wells and also in the negative insertion control. These results mean that the either the PCR failed or that the inserted plasmid is not the good one. 


    From these results, we decided to investigate the different tubes (3) of DreamTaq enzyme to look for a lost of function.
    
    So, we did PCR on two of our gBlocks (RibE and T25) with the three different DreamTaqs (Extension time: 2.00min).
    Then, to see whether the amplification worked or not, we run our PCR products on an 1% agarose gel.
    Here is a picture of the gel after 20min at 100V:<B>Figure 5:</B><I> PCR amplification using three DreamTaq mixes unraveled by electrophoresis</I>
    According to these results, we concluded that two among the three DreamTaqs did not work. It could explain why our precedent colony PCR failed miserably.

    We launched overnight cultures (LB + ery 150µg/mL) of:
    <UL><LI>4 colonies of p15.07, BbsI #A, 3:1 ratio</LI><LI>4 colonies of p15.07, BbsI #B, 2:1 ratio</LI><LI>2 colonies of p15.06, BbsI #A, 3:1 ratio</LI><LI>4 colonies of p15.06, BbsI #B, 2:1 ratio</LI></UL><H1 class="date two">August 7th</H1>

    Because of the poor digestion capacity we observed in BbsI, we investigated on the net if some other people had problems with the NEB BbsI enzyme. [This website: http://synbio.tsl.ac.uk/golden-gate-assembly-protocol/] explained that they detected that &quot;BbsI supplied by NEB (an isoschizomer of BpiI) loses activity extremely quickly and has a lot of star-activity&quot;. Using this information, we thought about using BpiI instead of BbsI.
    To confirm that BpiI would digest our plasmid the same way than BbsI, we digested p15.01 with BpiI to see if it would produce the same three bands than BbsI.
    Figure 6 shows the electrophoresis result of p15.01 after digestion with Eco31I and BpiI:<B>Figure 6:</B><I> Digestion of p15.01 with BpiI</I>The 3 expected bands (2.6, 1.7 and 1.4kb) are clearly visible, and no noisy band is detected at the same time, suggesting a low star activity. Thus, we decided to use BpiI to perform the next reactions.



    Miniprep of the 14 overnight cultures and restriction analysis of their plasmid with BpiI and Eco31I.
    Here is a simulation of an electrophoresis:<B>Figure 7:</B><I>  Electrophoresis simulation of the restriction analysis of transformant's plasmid with BpiI and Eco31I </I><I>From the left to the right:</I><I>Undigested p15.01, digested p15.01, p15.06 and p15.07 (Eco31I and BpiI)</I>
    Restriction analysis result after 25 min (100V, 1% agarose TAE gel):<B>Figure 8:</B><I>  Electrophoresis of the restriction analysis of transformant's plasmid with BpiI and Eco31I </I><I>From the left to the right:</I><I>p15.06 #A(2 transformants), p15.06 #B(4 transformants), generuler 1kb, p15.07 #A (4 transformants), p15.07 #B (4 transformants),</I>
    No well was showing the expected bands showing the insertion of the four gBlocks in the backbone.
    All the wells were presenting various coiled variants of the backbone and the non-inserted digested backbone.<H1 class="date two">August 8th</H1>
    Miniprep of g15.49. Extraction of 53.5ng/µL of p15.01.
    Golden Gate assembly of RibA, RibD, RibE and RibT25 or RibT48 in p15.01, with both 2:1 and 3:1 molarity ratios.
    Negative control is constituted of a digested p15.01 with BpiI and ligated without any digested insert.
    Instead of BbsI, we used BpiI.
    3µL of each Golden Gate products were transformed in electrocompetent NEB turbo <I>E.coli</I>.
    Cells were recovered one hour in LB at 37°C, and two volume (50µL and 170µL) were plated on LB + 150µg/mL of erythromycin.


    To check whether the assembly is working or not, we ran a gel to see if the Golden Gate is producing some expected bands.
    Here is a picture after 20 minutes of migration:<B>Figure 9: </B><I>Electrophoresis of p15.01, RibA, D, E and T after Golden gate assembly</I><I>From the left to the right:
        generuler 1kb - p15.06 1:2, p15.07 1:2, p15.06 1:3, p15.07 1:3, p15.01</I><B>PUT HERE A SIMULATION OF THE LIGATION</B><H1 class="date two">August 9th</H1>

    Miniprep of the two overnight cultures of g15.49: extracted 108 and 200 ng/µL.<B>Transformation results </B>(from 08/08 transformation)
    <TABLE style="width:500px"><TBODY><TR><TH>Transformed product</TH><TH>p15.06, 2:1 ratio</TH><TH>p15.06, 3:1 ratio</TH><TH>p15.07, 2:1 ratio</TH><TH>p15.07, 3:1 ratio</TH><TH>negative control</TH></TR><TR><TD><UL><LI>50µL:</LI>
                    0 colony
                    <LI>170µL:</LI>
                    12 colonies
                </UL></TD><TD><UL><LI>50µL:</LI>
                    0 colony
                    <LI>170µL:</LI>
                    0 colony
                </UL></TD><TD><UL><LI>50µL:</LI>
                    1 colony
                    <LI>170µL:</LI>
                    16 colonies
                </UL></TD><TD><UL><LI>50µL:</LI>
                    4 colonies
                    <LI>170µL:</LI>
                    7 colonies
                </UL></TD><TD><UL><LI>50µL:</LI>
                    0 colony
                    <LI>170µL:</LI>
                    0 colony
                </UL></TD></TR></TBODY></TABLE>

    We performed a colony PCR on 8 colonies from p15.06 1:2, p15.07 1:2, p15.07 1:3 and 1 colony of p15.01 (from 04/08 transformation), using o15.097 and o15.098.
    Instead of preparing ourselves the DreamTaq Mix, we used a 2X DreamTaq Master Mix, to which we only add both forward and reverse primers.
    Cells were saved on Lb + ery (150 µg/mL).
    Lyse was performed during 8 minutes, and elongation time span was extended to 3.3 minutes (instead of 6 - 1min/kb - human error...)<B>Figure 10: </B><I>Insertion control by colony PCR and electrophoresis</I><UL><LI>a-From left to right:
            p15.07 1:2 molarity ratio (8 colonies) - generuler 1kb - p15.07 1:3 molarity ratio (8 colonies)</LI><LI>b-From left to right:
            generuler 1kb, p15.01, p15.06 1:2 (8 colonies)</LI></UL><H1 class="date two">August 10th</H1>

    Colony PCR on 16 p15.06 (1:2) colonies, 16 p15.07 (1:2) colonies and  16 p15.07 (1:3) colonies.
    Elongation time span was extended to 3.30 minutes, and annealing was performed at 64°C.
    Because of a human mistake, thermocycler lid was inverted, letting the caps oped themselves when temperature increased. The PCR products were lost.

    Restriction digestion of p15.01 by BbsI or BpiI(Fast Digest), then by Eco31I.<B>Figure 11: </B><I>Electrophoresis of p15.01 after restriction digestion</I><I>From left to right: </I><I>non-digested p15.01 - p15.01 digested by BbsI (A and B) + Eco31I - p15.01 digested by BpiI + Eco31I</I>
    Though BbsI is a &quot;NEB time saver protocol enzyme&quot; but 5 to 15 minutes is not long enough for plasmid digestion.<H1 class="date two">August 11th</H1>

    PCR amplification of RibA, RibD, RibE and both RibT, elongation time was extended to 2.30 minutes and annealing temperature was 64°C.
    As the previous day, caps opened during the PCR because the thermocycler lid was inverted...

    Second PCR amplification of Rib gblocks. Final volume was 100µL, elongation time was 2.30 minutes and annealing temperature was 64°C.
    Electrophoresis was performed to check the gBlocks amplification.<B>Figure 12: </B><I>Rib gblocks amplification (64°C annealing) checked by electrophoresis</I><I>From left to right: 
        generuler 1kb - RibA - RibD - RibE - RibT25 - RibT48</I>


    It clearly appeared that something was wrong the amplification. Because of the absence of band for RibA and the double band for RibD, we decided to perform a third PCR using a lower annealing temperature (55°C).<B>Figure 13: </B><I>Rib gblocks amplification (55°C annealing) control by electrophoresis</I><I>From left to right: 
        generuler 1kb - RibA - RibD - RibE - RibT25 - RibT48</I>

    RibA, RibE and RibT48 were scarcely visible and RibD and RibT25 were presenting two bands.
    Using the different gel pictues we had, we decided to perform a fourth PCR amplification using for each gBlock the annealing temperature which gave the brightest and the sharpnest bands.
    For the amplification, we used the 60°C for RibA, RibD and RibT25, and 55°C for RibE and RibT48.

    Preparation of electrocompetent NEB turbo, saved in 200µL aliquots at -80°C.

    <H1 class="date two">August 12th</H1><B>Figure 14: </B><I>Rib gblocks amplification (launched on 11th) control by electrophoresis</I><I>From left to right: 
        generuler 1kb - RibA - RibD - RibE - RibT25 - RibT48</I>
    All the bands correspond to the expected size of the gBlocks, but RibA for which no band was visible.
    Thus, we decided to perform a gradient PCR (five tubes spread from 52°C to 64°C) on RibA to determine what was the best (or a least a good) temperature for annealing.<B>Figure 15: </B><I>RibA amplification checked by electrophoresis</I><I>From left to right: 
        generuler 1kb - RibA (Tm=52) - RibA (Tm=55) -  RibA (Tm=58) - RibA (Tm=61) - RibA (Tm=64) -</I>
    According to these results, we assumed that the five gblocks were there.
    PCR purification of the five gBlocks. For the multiple RibA, we purified them separately, giving four stocks of amplified gBlocks.
    Digestion of p15.01 and the Rib gBlocks separately, with BpiI (final volume = 30µL, about 100ng of DNA per tube). The digestion products were then PCR purified to get rid of the enzymes.<H1 class="date two">August 13th</H1>

    Ligation of p15.01 + RibA, RibD, RibE and RibT25 with 2:1 and 3:1 molarity ratio. Same for T48.
    Each ligation mix contained 20ng of the backbone, for a final volume of 20µL.
    Thus, the 2:1 mix contained 10ng of RibA, 12ng of RibD, 9ng of RibE and 6ng of RibT.
    3:1 mix contained respectively 15, 13, 13 and 8ng of RibA, RibD, RibE and RibT.

    All the ligation products were dialysed for 40 minutes.

    Transformation by electroporation of the four ligation products and for both digested and undigested p15.01 (5µL ligation product in 200µL electrocompetent cells).
    Time constant was 5.3 or 5.4 for all the transformations.
    Cells were recovered 2 hours in LB and then plated on LB + ery 150µg/mL.<H1 class="date two">August 14th</H1>

    Colony PCR of 15 colonies from each transformation. Annealing temperature was 64°C and elongation set to 3 minutes.<B>Figure 16: </B><I>Colony PCR of p15.06 (1:2 ratio) transformants</I><I>From left to right: 
        p15.01 (negative control) - 12 p15.06 (1:2) colonies - generuler 1kb - 3 p15.06 (1:2) colonies</I><B>Figure 17: </B><I>Colony PCR of p15.06 (1:3 ratio) transformants</I><I>From left to right: 
        p15.01 (negative control) - 8 p15.06 (1:3) colonies - generuler 1kb - 7 p15.06 (1:3) colonies</I><B>Figure 18: </B><I>Colony PCR of p15.07 (1:2 ratio) transformants</I><I>From left to right: 
        p15.01 (negative control) - 8 p15.07 (1:2) colonies - generuler 1kb - 7 p15.07 (1:2) colonies</I><B>Figure 19: </B><I>Colony PCR of p15.07 (1:3 ratio) transformants</I><I>From left to right: 
        p15.01 (negative control) - 8 p15.07 (1:2) colonies - generuler 1kb - 7 p15.07 (1:2) colonies</I>

    On all the gels, but p15.06 1:3, one can clearly see that all the bands are about 600bp heavy, as the negative control of insertion (p15.01).
    For p15.06 1:3, some bands are absent. After investigation, we figured out that the elongation time for the DreamTaq was 1kb/min instead of 30 seconds. Regarding to this information, we thought that maybe no band was visible because the insert was present but as the elongation time span was not long enough, the exponentially amplification could not work.
    To control these results, we launched overnight cultures of the four colonies corresponding to the absent bands (1, 6, 7 and 9).

    To obtain high yield of our five gBlocks, we performed a PCR using the optimal Tm we determined so far.<B>Figure 20: </B><I>Rib gBlocks PCR amplification</I><I>From left to right: 
        generuler 1kb - RibA - RibD - RibE - RibT25 - RibT48</I>

    Even with such an ugly gel (a very old stock of agarose was used), one can clearly see that four of the five gBlocks are correctly amplified. RibE expected size is 0.9kb, here an 1.5 and a 0.5 bands are visible.

    To determine the optimal Tm for RibE amplification, we performed a gradient PCR from 55°C to 64°C.<B>Figure 21: </B><I>RibE gradient PCR amplification</I><I>From left to right: 
        generuler 1kb - RibE amplified with the respective Tm: 55, 55.6, 56.7, 58.4, 60.3, 62.2, 63.3 and 64°C.</I>

    One can clearly see that only the two first wells are showing really visible bands.
    These two PCR products were PCR purified and saved for later use.<P><B>Lactobacillus plantarum Competent cells preparation:</B></P><UL><LI>
            Overnight culture (5mL) (37 degrees) (MRS) (G15.15 Lactobacillus plantarum NC8)
        </LI><LI>
            After 17hr (1:50 dilution) for log-phase cells
        </LI><LI>
            Growth for 4 hours (37 degrees) (MRS+1% glycine)
        </LI><LI><A href="http://openwetware.org/wiki/Lactobacillus_transformation_(Speer_2012)">Spear (2012)</A> (Open Wetware) (except, the first sucrose-glycerol wash replaced by water wash)
        </LI><LI>
            Stored in -80 degrees
        </LI></UL><H1 class="date two">August 15th</H1>

    Miniprep of the four overnight cultures and electrophoresis of the extracted DNA.
    Restriction analysis of the plasmid using BpiI and Eco31I.<B>Figure 22: </B><I>Restriction analysis of p15.06 transformants</I><I>From left to right: 
        generuler 1kb - p15.01 (negative control) - Transformant 1 - Transformant 2 - Transformant 3</I>


    The digestion profile is showing no difference with the negative control. It meant that none of these three transformants contained the inserts.

    PCR amplification of RibD (o15.003+o15.004), RibE (o15.005+o15.006) and both ribT25/48(o15.007+o15.008/o15.009+o15.010).<B>Figure 23: </B><I>Electrophoresis of Rib gBlock amplification</I><UL><LI>a-From left to right:
            generuler 1kb - 2 x RibD - 2 x RibE</LI><LI>b-From left to right:
            generuler 1kb - 2 x RibD - 2 empty wells - 2 x RibE </LI></UL>

    RibD was presenting a second amplification band and RibE, T25 and T48 were all not visible.
    We launched an other PCR for RibD and RibE using a different phusion master mix.<B>Figure 24: </B><I>Electrophoresis of digested an undigested p15.01(BpiI) and RibD and E amplification control</I><I>From left to right: 
        generuler 1kb - undigested p15.01 - digested p15.01 - 2 x RibE - RibD</I>

    As previously, RibD was presenting non expected bands. Geneious mismatch tool allowed us to look for non specific binding which could explain such a result. Allowing up to 8 mismatches, three other binding were detected, corresponding approximately to the size of the bands detected in the gel.

    PCR of the gBlocks with DMSO (1.5µL for 50µL reaction mix).<P><B>Result of lactobacillus plantarum transformation:</B></P><UL><LI>All the mini-prepped plasmid (pSIP409) was dialyzed for 20 minutes.</LI><LI>Frozen NC8 Competent cells were thawed for 20 minutes.</LI><LI>Frozen NC8 Competent cells were thawed for 20 minutes.</LI><LI>10ul of plasmid in 100 ul of competent cells</LI><LI>1mL of recovery MRS</LI><LI><I>Plates:</I> MRS + 10ug/mL of ERY
        </LI><LI><I>Electroporation Parameters:</I></LI><LI>Voltage: 2500 V</LI><LI>Capacitance: 25 uF</LI><LI>Resistance: 400 ohms</LI><LI>Cuvette: 2mm</LI><LI>10.1ms pulse (control)</LI><LI>and 10.4 ms pulse (sample)</LI></UL><LI>
            Recovery for 3.5 hr in (MRS)
        </LI><LI><I>Plating as follows:</I></LI><LI>200 uL of recovered cells (both sample and control) in (MRS+ERY(10ug/mL)) plates</LI><LI>Rest of the cells (for both sample and control) were:</LI><LI>centrifuged at 14000 rpm for 1 minute,</LI><LI> resuspended in a total of ~200 uL of MRS</LI><LI> and plated on (MRS+ERY(10ug/mL)) plates.</LI><H1 class="date two">August 16th</H1>

    15/08 gBlocks PCR amplification analysis by electrophoresis.<B>Figure 25: </B><I>gBlocks PCR amplification control by Electrophoresis</I><I>From left to right: 
        generuler 1kb - RibA - RibD - RibE - RibT25 - Ribt48</I>
    Gradient PCR amplification of RibD (annealing temperature from 52 to 65°C, total volume 20µL). Elongation time span was extended to 1.30 minutes.(overnight)

    <H1 class="date two">August 17th</H1>

    Electrophoresis of RibD gradient PCR.<B>Figure 26: </B><I>Electrophoresis of RibD amplification with eight annealing temperature</I><I>From left to right: 
        generuler 1kb - RibD (52 - 53 - 54.5 - 57 - 60 - 62.4 - 64 - 65°C)</I>

    As this gel was not presenting any band and we were short in amplified gBlocks, we decided to amplify all of them again.
    As several PCR on the lab (for different teams) were not working, we suspected that maybe some phusion MasterMixes were compromised.
    Gradient PCR amplification (Tm from 52 to 65°C : 52 - 53 - 54.5 - 57 - 60 - 62.4 - 64 - 65°C) of RibD, RibE and both RibT. Two phusion MM were used to prevent a negative result due to nonfunctional phusion MasterMix. Elongation time was extended to 1.30 minutes.<B>Figure 27: </B><I>Electrophoresis of RibD, RibE, RibT25 and RibT48 after gradient PCR amplification<UL><LI>a-From left to right:
            generuler 1kb - 8 x RibD - 8 x RibE</LI><LI>b-From left to right:
            8 x RibT25 - generuler 1kb - 8 x RibT48</LI></UL>
    Knowing that we finished the first phusion MasterMix for the seventh RibT25 PCR mix and used another one for the last RibT25 and for the eight RibT48, we suspected that the first MasterMix we used was not working anymore, and thus could explain these days PCR failures.<P><B>Result:</B></P><P>Sample plate with 200uL of recovered cells: ~45 colonies!</P><P>Control  plate with 200uL of (transformed with water) recovered cells: 0 colonies!</P><P>Sample plate with rest of the recovered cells: ~ 200 colonies</P><P>Control plate with rest of (transformed with water) recovered cells: 0 colonies!</P><H1 class="date two">August 18th</H1>

    PCR amplification of RibD, RibE, RibT25 and RibT48 (two mix for each, 1µL of dNTPS was added to increase the yield, 50µL of mix per tube).
    For annealing , we used respectively 60.5, 55, 64 and 62.2°C for RibD, RibE, RibT25 and RibT48.<B>Figure 28: </B><I>Electrophoresis of Rib gBlocks PCR products</I><I>From left to right: 
        RibD x2 - RibE x2 - generuler 1kb - RibT25 x2 - RibT48 x2</I>

    All the amplification products were showing bands corresponding to their expected size, but RibD was again presenting a non specific amplification band.
    Thus we decided to PCR RibD again with different annealing temperature (from 60 to 68°C, 7 tubes), 20µL total reaction volume, 1.30 minutes elongation.(overnight)<H1 class="date two">August 19th</H1>

    RibD gradient PCR amplification was controled by electrophoresis.<B>Figure 29: </B><I>Control of RibD gradient PCR amplification by Electrophoresis</I><I>From left to right: 
        generuler 1kb - 7x RibD</I>

    As a lot of PCR problems occurred at this period among the teams, the absence of band could be explained by the usage of a faulty phusion MasterMix.
    We received at this point a new batch of phusion MasterMix, that we used for all the next PCRs.


    PCR purification of RibE, RibT25 and ribT48 (amplified the 18/08).
    Purification products were run on a 1% agarose gel to control their purity.<B>Figure 30: </B><I>Electrophoresis of amplified and PCR purified RibE, RibT25 and Ribt48</I><I>From left to right: 
        generuler 1kb - 2x RibE - 2x RibT25 - 2x RibT48</I>

    Each gBlock is presenting a single band of the expected size. These amplification products were saved for later use.

    PCR amplification of RibD with 8 different annealing temperature (52 to 65°C).<B>Figure 31: </B><I>Electrophoresis of RibD gradient PCR amplification</I><I>From left to right: 
        generuler 1kb - RibD x8 (52, 53, 54.5, 57, 60, 62.4, 64, 65)</I>

    Visible bands corresponded to the following annealing temperature: 57, 60, 62 and 64°C. A non expected band with a size of about 0.3kb was still observed. So we poured all of the volume left of this four PCR products on a gel a performed a gel extraction to only get the 1.3kb band.<B>Figure 32: </B><I>Electrophoresis of RibD for gel extraction</I><I>In frame four bands were gel extracted</I><I>From left to right: 
        generuler 1kb - RibD x4</I>


    The yield was low, as expected, so we amplified again RibD in two tubes (100µL each, +1µL dNTPs, annealing temperature of 63°C).






    <H1 class="date two">August 20th</H1>

    19/08 RibD PCR amplification was run on a 1% agarose gel to allow the gel extraction of the 1.3kb band.<B>Figure 33: </B><I>Electrophoresis of RibD for gel extraction</I><I>Left: 
        8x RibD - generuler 1kb</I><I>Right: 
        8x RibD - generuler 1kb</I>

    Digestion of all the available amplified gBlocks by BpiI. Digestion mix total volume was 50µL with 1µg of each gBlock (such quantity was maybe too much), digestion span was extended to 25 minutes to compensate the amount of DNA.
    Digestion products were then PCR purified and saved for later use.

    After purification, concentration of each gBlock was really low, so we ran them on a gel to confirm their presence.</I><TABLE style="width:500px"><TBODY><TR><TH>gBlock</TH><TH>Concentration (ng/μL)</TH></TR><TR><TD>RIbA</TD><TD>23.5</TD></TR><TR><TD>RIbD</TD><TD>4.5</TD></TR><TR><TD>RIbE</TD><TD>19.9</TD></TR><TR><TD>RIbT25</TD><TD>22.6</TD></TR><TR><TD>RIbT48</TD><TD>21.8</TD></TR></TBODY></TABLE><B>Figure 34: </B><I>Digested gBlocks</I><I>From Left to Right: 
        RibA - RibD - RibE - RibT25 - RibT48 - generuler 1kb</I>

    Even if the bands are really faint, we can see that the five gblocks are presents and are not presenting any non expected bands.

    PCR amplification of RibD using o15.003 + o15.004 (previous yield after purification was really low).<H1 class="date two">August 21st</H1>

    Digestion and PCR purification of previously amplified RibD (20/08). 1.3kb band presence confirmed by electrophoresis. (see figure 35).

    Ligation of p15.01 (digested with BpiI), p15.01 (digested with BpiI) + RibADET25 (digested with BpiI) with two backbone:insert molarity ratios (1:2 and 1:3) to assemble p15.06, and p15.01 (digested with BpiI) + RibADET48 (digested with BpiI) with two molarity ratios (1:2 and 1:3) to assemble p15.07.
    Final volume was 20µL.

    1:2 assembly mix:<UL><LI>p15.01  40ng</LI><LI>RibA    20ng</LI><LI>RibD    18ng</LI><LI>RibE    12ng</LI><LI>RibT25  8ng</LI></UL>

    1:3 assembly mix:<UL><LI>p15.01  40ng</LI><LI>RibA    30ng</LI><LI>RibD    26ng</LI><LI>RibE    20ng</LI><LI>RibT25  16ng</LI></UL>


    To obtain the higher yield of ligated products, we increased the elongation time to 1 hour.

    After ligation, to see whether some gBlocks were ligated or not, a little amount of each ligation product was run on a 1% agarose TAE gel.<B>Figure 35: </B><I>Electrophoresis of the ligated products of p15.06 and p15.07 assembly</I><I>From Left to Right: 
        digested p15.01 - RibD digested and PCR purified - generuler 1kb - p15.06 1:2 - p15.06 1:3 -  p15.07 1:2 - p15.07 1:3</I>

    We could see several faint bands, corresponding approximately to 1.3, 2.8, 3.5 to 4, 4 to 5, 5.5 and a broad band from 8 to 10kb.<TABLE stle="width:500px"><TBODY><TR><TH>Assembled fragments</TH><TH>Size (kb)</TH></TR><TR><TD>A + D</TD><TD> 2.8</TD></TR><TR><TD>D + E</TD><TD> 2.2</TD></TR><TR><TD>E + T</TD><TD> 1.5</TD></TR><TR><TD>A + D + E</TD><TD> 3.7</TD></TR><TR><TD>D + E + T</TD><TD> 2.8</TD></TR><TR><TD>A + D + E + T</TD><TD> 4.3</TD></TR><TR><TD>A + D + E + T + p15.01</TD><TD> 10</TD></TR></TBODY></TABLE>
    Dialysed the ligation products for 25 minutes. 6µL of each ligation product were transformed in 100µL of NEB turbo electrocompetent cells. Time constants gave by the machine were about 5.2 to 5.4. Cells were recovered 2 hours in LB and then plated on LB + erythromycin (150µg/mL), 200µL in one plate and the rest pelleted and re-suspended in 200µL.<H1 class="date two">August 22nd</H1><B>Transformation results </B>(from 08/08 transformation)
    <TABLE style="width:500px"><TBODY><TR><TH>Transformed product</TH><TH>p15.06, 2:1 ratio</TH><TH>p15.06, 3:1 ratio</TH><TH>p15.07, 2:1 ratio</TH><TH>p15.07, 3:1 ratio</TH><TH>negative control</TH></TR><TR><TD><UL><LI>200µL:</LI>
                    ~300 colonies
                    <LI>rest:</LI>
                    ~1000 colonies
                </UL></TD><TD><UL><LI>200µL:</LI>
                    46 colonies
                    <LI>rest:</LI>
                    215 colonies
                </UL></TD><TD><UL><LI>200µL:</LI>
                    51 colonies
                    <LI>rest:</LI>
                    197 colonies
                </UL></TD><TD><UL><LI>200µL:</LI>
                    70 colonies
                    <LI>rest:</LI>
                    298 colonies
                </UL></TD><TD><UL><LI>200µL:</LI>
                    0 colony
                    <LI>rest:</LI>
                    0 colony
                </UL></TD></TR></TBODY></TABLE>


    Colony PCR of 15 mutants from each transformation with o15.097 and o15.098.
    Mix total volume was 20µL.
    Initial denaturation step (95°C) was extended to 8 minutes to lyse the cells. Annealing temperature was set to 64°C and elongation was extended to 6 minutes (1min/kb).
    After PCR, the PCR products were run on a gel to detect a possible insertion.<B>Figure 36: </B><I>Electrophoresis of colony PCR products after transformation with p15.06 and p15.07<UL><LI>a-From left to right:
            p15.01 - 7 x p15.06 1:2 - generuler 1kb - 8 x p15.06 1:2 </LI><LI>b-From left to right:
            p15.01 - 7 x p15.06 1:3 - generuler 1kb - 8 x p15.06 1:3 </LI><LI>c-From left to right:
            p15.01 - 7 x p15.07 1:2 - generuler 1kb - 8 x p15.07 1:2 </LI><LI>d-From left to right:
            p15.01 - 7 x p15.07 1:3 - generuler 1kb - 8 x p15.07 1:3 </LI></UL></I>


    The negative insertion control (p15.01) is not appearing, it was maybe due to a failure in the colony PCR protocol.
    The visible bands on the four gels were corresponding to the size of a non inserted plasmid.
    From these results, we concluded that the transformation did not worked (again...).<H1 class="date two">August 23rd</H1>

    We wanted to check whether the gblocks could bind two by two, three by three and the four at the same time.
    Each mix contained 1µL of the corresponding gBlock digested and purified (approximative concentration of each gBlock: 20ng/µL).
    Ligation was performed as usually, for a final volume of 20µL.

    After ligation, some ligation product was run on a 1% agarose TAE gel.<B>Figure 37: </B><I>Electrophoresis of Rib gblocks after ligation</I><I>From Left to Right: 
        generuler 1kb - A+D - D+E - E+T25 - E+T48 - A+D+E - D+E+T - A+D+E+T</I>

    On the gel, we can see that that several binding are working correctly. For example, for A+D, we can distinguish two bands, the first measuring about 1.5kb corresponding to RibA, and a second one about 2.5 to 3kb, corresponding approximately to the expected size (2.8kb) after ligation of RibA and Ribd. This band is also slightly visible in all the ligation products containing RibA and RibD.

    The ligation products were used as a DNA template for elongation to try to amplify the ligated gBlocks. This would help us to see which gBlock is correctly ligated and maybe to extract these PCR products and use them for transformation.
    PCR was performed with DreamTaq MasterMix, with an elongation time extended to 6 minutes (the time required to amplify the RibADET ligation product), with an annealing temperature of 55°C.<TABLE style="width:500px"><TBODY><TR><TH>ligation mix</TH><TH>primers used</TH></TR><TR><TD>A+D</TD><TD>o15.001+o15.004</TD></TR><TR><TD>D+E</TD><TD>o15.003+o15.006</TD></TR><TR><TD>E+T25</TD><TD>o15.005+o15.008</TD></TR><TR><TD>E+T48</TD><TD>o15.005+o15.010</TD></TR><TR><TD>A+D+E</TD><TD>o15.001+o15.006</TD></TR><TR><TD>D+E+T25</TD><TD>o15.003+o15.008</TD></TR><TR><TD>A+D+E+T25</TD><TD>o15.001+o15.008</TD></TR></TBODY></TABLE><B>Figure 38: </B><I>Electrophoresis of amplified ligation products</I><I>From Left to Right: 
        generuler 1kb - A+D - D+E - E+T25 - E+T48 - A+D+E - D+E+T - A+D+E+T</I>

    Only the amplification of A+D worked, showing three bands (two faint and a more visible one). The two first were likely to be the result of the amplification of RibA and RibD, and the brightest band the amplification of the ligated product.
    No other band was visible on the gel. This could be explained with the two following reasons:
    -the annealing temperature of one and/or the two primer(s) was not allowing the binding
    -the ligation did not work (overhangs were not good, ligation reaction did not work...).


    We received the primers to biobrick the four genes (RibA, RibD, RibE and RibT), the two promoters (p25 and p48) and the terminator (Tdhl).
    PCR amplification of the five gBlocks with respectively o15.027+o15.028, o15.029+o15.030, o15.031+o15.032, o15.033+o15.034 and o15.035+o15.034. 
    The promoters p25 and p48 were respectively amplified with o15.174+o15.175 and o15.172+o15.173.
    Terminator Tdhl was amplified with o15.170+o15.171.
    For amplification, was choose to perform a gradient PCR with four different annealing temperature: 70, 72, 73 and 74°C.
    Elongation was extended to 1.30 minutes.<B>Figure 39: </B><I>Electrophoresis of amplified parts for biocricking</I><I>Amplification products are presented from the cooler to the heater for each part.</I><UL><LI>a-From left to right:
            generuler 1kb - RibA x4 - RibD x4 - RibE x4 - RibT25 x4</LI><LI>b-From left to right:
            generuler 1kb - RibT48 x4 - Terminator Tdhl x4 - p25 x4 - p48 x4</LI></UL>
    RibA, RibE, both RibT and both promoters are presenting bands with the expected size. RibD is again showing the same second band and the terminator is invisible.
    We so decided to use these amplification products to latter transform our cells with.<H1 class="date two">August 24th</H1>

    Restriction digestion of the PCR amplified parts (with the BioBrick tails) with XbaI and SpeI.
    pSB1C3 was digested at the same time with XbaI and SpeI.
    Total volume was 30µL.
    DNA yield after digestion was about 6ng/µL.

    Overnight culture of pSB1C3+RFP in Lb+ Cm(20µg/mL).<H1 class="date two">August 25th</H1>

    Digestion of pSB1C3-RFP with EcoRI/XbaI and gel extraction of the 2.2kb band.<B>Figure 40: </B><I>Electrophoresis of digested pSB1C3</I><I>From left to right:
        generuler 1kb - digested pSB1C3 x2
        a: before extraction
        b: after extraction</I>


    Ligation, dialyse and transformation of the digested pSB1C3 with the BioBrick-tailed parts (RibA, RibE, RibT25, RibT48, p25 and p48).
    A 3:1 insert:backbone molarity ratio was used for the ligation step. 20ng of backbone was used per 20µL of reaction.
    Dialyse was perform for 30 minutes.
    For electroporation, 5µL of ligation product were added to 25µL of electrocompetent NEB turbo. Time constants were about 2.3 to 2.4. Cells were recovered for one hour and then the total 250µL of each recovery cultures were plated on LB agar + Cm (25µg/mL).


    Colony PCR of p15.06 and p15.07 transformations.
    Colony PCR failed, only a smear was visible on each column of the gel after electrophoresis.<H1 class="date two">August 26th</H1><B>Transformation results </B>(from August 28th transformation)
    <TABLE style="width:500px"><TBODY><TR><TH>Transformed product</TH><TH>RibA</TH><TH>RibE</TH><TH>RibT25</TH><TH>RibT48</TH><TH>p25</TH><TH>p48</TH><TH>negative control</TH></TR><TR><TD>97 single colonies</TD><TD>116 single colonies</TD><TD>171 single colonies</TD><TD>78 single colonies</TD><TD>127 single colonies</TD><TD>91 single colonies</TD><TD>0 single colony</TD></TR></TBODY></TABLE>

    Restriction digestion of p15.01 by FD BpiI and FastAP.
    About 3µg of p15.01 were digested for 1 hour in a 50µL mix.

    Colony PCR of BioBrick transformants (4 colony / plate) with o15.182+o15.183.
    First step was extended to 8 minutes to lyse the cells and the elongation time was extended to 2 minutes.<B>Figure 41: </B><I>Colony PCR of biobricked parts transformants</I><UL><LI>a-From left to right:
            RibA x4 - generuler 1kb - RibE x4 - generuler 1kb - RibT25 x4 - generuler 1kb - p15.01 digested with BpiI</LI><LI>b-From left to right:
            RibT48 x4 - generuler 1kb - p25 x4 - generuler 1kb - p48 x4 - generuler 1kb</LI></UL>
    One colony of RibA (gel a - 4th well) and one of p25 (gel b - 5th band) seemed to present the expecting band after insertion.
    We inoculated 2 tubes of 5mL LB + Cm 25µg/mL with the two colonies of RibA and p25. Cultures were incubated overnight at 37°C.<H1 class="date two">August 27th</H1>
    Miniprep of the two overnight cultures of RibA and p25.
    A sample of both miniprep were sent to sequence using o15.148 (TGCCACCTGACGTCTAAGAA, Forward sequencing ) or o15.149 (ATTACCGCCTTTGAGTGAGC, Reverse sequencing). These two primers are commonly used for pSB1C3 sequencing.

    Colony PCR of 25/08 biobrick-amplified parts transformation. Four colonies of BB-RibA, BB-RibE, BB-RibT25, BB-RibT48, BB-p25 and BB-p48 were analysed.
    PCR total volume was 20µL. Initial denaturation temperature step (95°C) was extended to 8 minutes to lyse the cells. Annealing temperature was 50°C and elongation time span was 2 minutes.


    At this point as our previous assembly protocols did not work, so we tried to PCR amplify the four gBlocks after digestion and ligation of the four of them at the same time, using o15.027 (GTTTCTTCGAATTCGCGGCCGCTTCTAGAGgcgcccgaagacttatgc) and o15.034 (GTTTCTTCCTGCAGCGGCCGCTACTAGTAcagcagcatatgaagac), respectively the RibA forward and RibT reverse primers with the biobrick prefix/suffix.<B>Figure 42: </B><I>PCR amplification of Rib gBlocks assembly
    From left to right:
    generuler 1kb - p15.06 1:2 - p15.06 1:3 - p15.07 1:2 - p15.07 1:3</I>
    It clearly appeared that none of the ligation products could have been amplified...<H1 class="date two">August 28th</H1>


    Sequencing results indicated that p25 was not present in pSB1C3.
    For the sequencing of BB-RibA, the sequencing was indicating that a huge part of the CDS was there without SNPs, but the other part was really noisy and presenting a lot of differences with the theoretical sequence.

    We troubleshot the golden gate protocol we used with an INSERM U1001 member, Antoine Decrulle.
    He gave us the following protocols that he was using at this time, his positive control and the enzymes he was using.
    From the results we get so far, two steps could explain the failures: digestion and ligation.
    Several points have been improved: using a High Concentration Ligase (T4 DNA ligase HC from Promega, #M1794) and adding some purified BSA (100X BSA, NEB, #B9000S).
    We also used two different protocols:
    -using the digestion buffer (G buffer 10x, Fermentas, #BG5)
    -using the ligation buffer coming with the HC ligase.

    So, when we refer to p15.06 L (or p15.07 L), one should understand p15.06 (or p15.07) assembled with the Ligase buffer protocol (same for p15.06 or p15.07 G).<B>Golden gate assembly with G buffer</B><TABLE style="width:500px"><TBODY><TR><TD><UL><LI>Prepare the following mix:</LI><LI>0.4µL of BpiI</LI><LI>0.2µL of Promega Ligase HC</LI><LI>1µL of BSA 10X</LI><LI>1µL of G buffer 10X</LI><LI>1µL of DTT(10mM)</LI><LI>1µL of ATP(10mM)</LI><LI>40 fmol of backbone</LI><LI>40 fmol of each insert</LI><LI>up to 10µL of water</LI></UL><LI>Use the following digestion/ligation cycles:</LI><LI>37°C, 5minutes (digestion)</LI><LI>20°C, 5 minutes (ligation)</LI>
                            repeat 50 times the two steps above<LI>50°C, 10 minutes</LI><LI>80°C, 10 minutes</LI></TD></TR></TBODY></TABLE><B>Golden gate assembly with L buffer</B><TABLE style="width:500px"><TBODY><TR><TD><UL><LI>Prepare the following mix:</LI><LI>0.4µL of BpiI</LI><LI>0.2µL of Promega Ligase HC</LI><LI>1µL of BSA 10X</LI><LI>1µL of 10X Ligase buffer (Promega)</LI><LI>1µL of DTT(10mM)</LI><LI>1µL of ATP(10mM)</LI><LI>40 fmol of backbone</LI><LI>40 fmol of each insert</LI><LI>up to 10µL of water</LI></UL><LI>Use the following digestion/ligation cycles:</LI><LI>37°C, 5 minutes</LI><LI>37°C, 2 minutes (digestion)</LI><LI>16°C, 5 minutes (ligation)</LI>
                            repeat 50 times the two steps above<LI>50°C, 10 minutes</LI><LI>80°C, 10 minutes</LI></TD></TR></TBODY></TABLE>

    We performed the golden gate assembly of p15.06 and p15.07 using both L and G protocols.
    Positive control was constituted of the pL2 backbone and of two inserts, pele1 and eYFP. The backbone is expressing LacI constitutively. In presence of IPTG, LacI loose its ability to inhibit the pLac promoter and allow the expression the protein promoted by pLac.<H1 class="date two">August 29th</H1>

    Dialyse and transformation of the Golden Gate products.
    Dialyse was performed for 15 minutes and 5µL of each Golden gate product were added to 50µL of electrocompetent cells.
    Cells were recovered 2 hours in LB, plated on LB + ery (p15.01, p15.06, p15.07, negative growth control) or LB+Cm+IPTG+Xgal (pL2) and incubated at 37°C.

    As we were craving for results, we launched a second Golden Gate assembly on the same products, using again both G and L protocols.<H1 class="date two">August 30th</H1>

    Miniprep of g15.49 and pSB1C3 x2 overnight cultures.<B>Transformation results </B>(from August 29th transformation)
    <TABLE style="width:500px"><TBODY><TR><TH>Transformed product</TH><TH>p15.06 L</TH><TH>p15.06 G</TH><TH>p15.07 L</TH><TH>p15.07 G</TH><TH>pL2 L</TH><TH>pL2 G</TH><TH>negative control #1</TH><TH>negative control #2</TH></TR><TR><TD>more than 600 colonies</TD><TD>95 single colonies</TD><TD>428 single colonies</TD><TD>23 single colonies</TD><TD>more than 600 colonies, 4 blue colonies</TD><TD>more than 600 colonies, 10 blue colonies</TD><TD>0 single colony</TD><TD>0 single colony</TD></TR></TBODY></TABLE>

    Colony PCR of 6 colonies from each p15.06 and p15.07 transformation with o15.097 and o15.098.
    initial denaturation step was extended to 8 minutes to lyse the cells and elongation was extended to 6 minutes.
    Annealing was performed at 55°C.<B>Figure 43: </B><I>Electrophoresis colony PCR products</I><UL><LI>a-From left to right:
            generuler 1kb - p15.01 - p15.06 L buffer</LI><LI>b-From left to right:
            generuler 1kb - p15.01 - p15.06 G buffer</LI><LI>c-From left to right:
            generuler 1kb - p15.01 - p15.07 L buffer</LI><LI>d-From left to right:
            generuler 1kb - p15.01 - p15.07 G buffer</LI></UL>
    The negative control amplification did not work, but one can clearly see that in between the 0.5kb bands we were used to see, a
    band of about 4.5kb is visible, which corresponded to the expected size of the amplification in case the insertion worked. But let's not be hasty, and let's wait for the sequencing results.

    The corresponding colonies (saved during the colony PCR) were restriked on LB + ery(150µg/mL) and incubated overnight.


    Dialyse, electrophoresis and transformation of the 29/28 Golden Gate products.<B>Figure 44: </B><I>Electrophoresis of Golden Gate products</I><I>From left to right:
        generuler 1kb - p15.01 - p15.06 G buffer - p15.06 L buffer - p15.07 G buffer - p15.07 L buffer</I>
    This gel is showing broad bands at about 10kb, which could represent the inserted p15.01.
    Cells were recovered two and half hours and 200µL of each transformation mix was plated on LB+ery(150µg/mL) (p15.01, p15.06, p15.07), LB+cm+IPTG+xGal (pL2) or LB+ery and LB+cm (negative growth control).<H1 class="date two">August 31th</H1>

    Sequencing results


    <B>Transformation results </B>(from August 30th transformation)
    <TABLE style="width:500px"><TBODY><TR><TH>Transformed product</TH><TH>p15.01</TH><TH>p15.06 L</TH><TH>p15.06 G</TH><TH>p15.07 L</TH><TH>p15.07 G</TH><TH>pL2 L</TH><TH>pL2 G</TH><TH>pL G</TH><TH>negative control #1</TH><TH>negative control #2</TH></TR><TR><TD>more than 3000 colonies</TD><TD>more than 1500 colonies</TD><TD>about 300 single colonies</TD><TD>more than 1500 colonies</TD><TD>about 300 single colonies</TD><TD>more than 1000 colonies, 3 blue colonies</TD><TD>more than 200 colonies, no blue colony</TD><TD>more than 1500 colonies, no blue colony</TD><TD>0 single colony</TD><TD>0 single colony</TD></TR></TBODY></TABLE>


    One or two single colonies of each suspected colony were inoculated in 5mL of LB + ery(150µg/mL) for 7 hours. 4mL of each culture were miniprep.
    Two miniprep of p15.06 G colony 4 were sent to sequence with the following gBlocks: o15.097 (CGGTAGAGCTCCCTTCTATGC), o15.110 (CAACACCGATAGCACCAACG), o15.112(CCACCGAAACGACCGTTAGC), o15.114(GCATGTACTTCAGCGTGTGC) and o15.116(CTTCAGGAGTTGCATGTTCAGC). This colony was the more likely to contain the insert, so we sequenced all the binding points in between the backbone and the gBlocks.
    One miniprep of p15.06 G colony 1 and two miniprep of p15.07 L colony 1 and 2 were sent to sequence with o15.097 and o15.116 to check the binding between p15.01/RibA and RibT/p15.01.

    Colony PCR of 8 colonies from p15.06 L and G and p15.07 L and G.<B>Figure 45: </B><I>Colony PCR of p15.06 and p15.07 transformants (29/08 transformation)</I><UL><LI> a- From Left to Right:
            p15.06 buffer L (8 colonies) - 1kb ladder - p15.06 buffer G (8 colonies)</LI><LI> b- From Left to Right:
            p15.07 buffer L (8 colonies) - p15.01 - 1kb ladder </LI><LI> c- From Left to Right:
            p15.07 buffer G (8 colonies) - p15.01 - 1kb ladder </LI></UL>

    Colony PCR failed for an unknown reason. We so launched a PCR for RibD to check the 3 Phusion MM and 1 DreamTaq MM.
    Denaturation temperature was set to 98°C for Phusion and 95°C for DreamTaq. Annealing was performed at 60 °C for Phusion and 50°C for DreamTaq, and elongation time span was 1:30 minutes and 2 minutes.<B>Figure 46: </B><I>PCR MasterMix control</I><I>From Left to Right:
        1kb ladder - Phusion 1 - Phusion 2 - Phusion 3 - DreamTaq</I>
    Seeing the results, we conclude that Phusion 2 and 3 were corrupted, as well as DreamTaq.
    The MM we used for the previous colony PCR was corrupted.<H1 class="date three">September 1st</H1>

    Colony PCR of 29/08 transformation. Eight colonies of both p15.06 L and G and both p15.07 L and G were used.
    PCR total volume was 20µL. Initial denaturation temperature step (95°C) was extended to 8 minutes to lyse the cells. Annealing temperature was 64°C and elongation time span was 6 minutes.<B>Figure 47: </B><I>Colony PCR analysis of p15.06 L and G and p15.07 L and G transformants by electrophoresis</I><I>From Left to Right:</I><LI><I> a- From Left to Right:
            p15.06 L x8 - generuler 1kb - p15.06 G x8</I></LI><LI> b- From Left to Right:
            p15.07 L x8 - generuler 1kb - p15.07 G x8</LI>

    On both gels, one can clearly see that some wells are containing several bands, we decided to investigate this behaviour (maybe an inserted transformant was picked at the same time as a non-inserted transformant).
    So, we inoculated the corresponding colonies (p15.06 G 3, 4 and 8, p15.07 G 2, 5 and 6) in 5mL Lb + agar + ery(150µg/mL).


    Colony PCR of 25/08 biobrick-amplified parts transformation with o15.148 and o15.149.
    Four colonies of BB-RibA, BB-RibE, BB-RibT25, BB-RibT48, BB-p25 and BB-p48 were analysed.
    PCR total volume was 20µL. First high temperature step (95°C) was extended to 8 minutes to lyse the cells. Annealing temperature was 50°C and elongation time span was 2 minutes.<B>Figure 48: </B><I>Colony PCR analysis of  BB-RibA, BB-RibE, BB-RibT25, BB-RibT48, BB-p25 and BB-48 transformants by electrophoresis</I><LI> a- From Left to Right:
        BB-RibA x4 - BB-RibE x4 - generuler 1kb - BB-RibT25 x4 - BB-RibT48 x4</LI><LI> b- From Left to Right:
        generuler 1kb - BB-p25 x4 - BB-p48 x4</LI>

    These results are all negative, as the expected size of a non-inserted amplification is about 0.3kb.
    Maybe more colonies should be screened to get an inserted transformant.

    We performed an electrophoresis on the 31/08 miniprep undigested to get an estimation about the size of the plasmid.<B>Figure 49: </B><I>Electrophoresis of p15.06 and p15.07 transformants presenting an interesting profile after colony PCR.</I><I>From Left to Right:
        generuler 1kb plus - p15.06 G2 #1 - p15.06 G4 #1 - p15.06 G4 #2 - p15.07 L6 #1 - p15.06 L6 #2</I>

    The gel is showing three bands with a size of approximately 10 to 20kb.
    p15.06 and p15.07 are about 10kb long, so the visible bands could correspond to p15.06 and p15.07.

    Moreover, on two plates containing the re-striking of the colonies used for the colony PCR, a really visible yellow dye was appearing around colonies.
    (Riboflavin appears yellow, just sayin')<B>Figure 50: </B><I>Photography of p15.06 and p15.07 transformants</I>


    Second colony PCR of 25/08 biobrick-amplified parts transformation with o15.148 and o15.149. Eight colonies of BB-RibA, BB-RibE, BB-RibT25, BB-RibT48, BB-p25 and BB-p48 were analysed.
    PCR total volume was 20µL. Initial denaturation temperature step (95°C) was extended to 8 minutes to lyse the cells. Annealing temperature was 50°C and elongation time span was 2 minutes.

    RibD PCR amplification with Biobrick prefix and suffix primers (respectively o15.029 and o15.030).
    As the amplification failed previously, we decided to test a broader range of annealing temperatures.
    NEB Tm calculator indicated that for the binding of the two primers (without their tail), an annealing temperature of 50°C is required. So we tried with 8 different temperatures from 48°C to 58°C. Elongation time span was set to 1.30 minutes.<H1 class="date three">September 2nd</H1><B>Figure 51: </B><I>Colony PCR of BioBrick transformants</I><LI><I> a- From Left to Right:
        BB-RibA x8 - generuler 1kb - BB-RibE x8 </I></LI><LI> b- From Left to Right:
        BB-RibT25 x8 - generuler 1kb - BB-RibT48 x8 </LI><LI> c- From Left to Right:
        BB-p25 x4 - generuler 1kb - BB-p48 x4  </LI>

    The expected size of a non inserted pSB1C3 amplification band with o15.182 and o15.183 is 0.3kb.
    Most of the wells are presenting a 0.3kb band, meaning that those colonies did not receive an inserted plasmid.
    However, on the first well of BB-RibA (gel a), one can see two bands above the 0.3kb band. These bands are not the result of a RibA insertion (RibA insertion amplification band should be about 1.7kb), but more probably the results of a primer off-binding.
    On the 8th well of BB-RibT25, a 0.9kb band is clearly visible, corresponding to the expected size of BB-RibT25 insertion in pSB1C3. The corresponding colony was striked on LB + Cm(25µg/mL) and incubated at 37°C overnight.
    Once more, no transformant containing BB-RibE, BB-RibT48, BB-p25 or BB-p48 were found.


    RibD gradient PCR amplification was with BB-tailed primers was analysed by electorphoresis.<B>Figure 52: </B><I>BB-RibD gradient PCR amplification</I><I>From Left to Right:
        RibD x8 (Tm: 48, 48.7, 49.9, 51.9, 54.2, 56 and 57.1°C) - generuler 1kb</I>

    One can clearly see that none of the amplification worked. These results could have been caused by a non working Phusion MM (again...) or because the annealing temperature was not good.


    Miniprep of p15.06 L colonies 6 and 8, p15.06 G colonies 3, 4, 5, 6 and 7, p15.07 G colonies 2, 3, 5 and 6.
    The miniprep results were anaylsed by electrophoresis.<B>Figure 53: </B><I>Electrophoresis of p15.06 and p15.07 transformants' miniprep</I><I>From Left to Right:
        p15.06 G 3, 4, 5, 6, 7 - p15.06 L 6, 8 - generuler 1kb plus - p15.07 G 2, 3, 5, 6</I>


    Seeing these results, we decided to perform an analytical digestion on those plasmids (200ng of DNA) with Eco31I.<B>Figure 54: </B><I>Restriction digestion of p15.06 and p15.07 transformants</I><UL><LI> a- From Left to Right:
            generuler 1kb - p15.01 - p15.06 - p15.07</LI><LI> b- From Left to Right:
            p15.06 G 3, 4, 5, 6, 7 - p15.06 L 6, 8 - p15.07 G 2 - generuler 1kb plus - p15.07 G 3, 5, 6</LI></UL>

    From the digestion, it appeared that p15.06 G 5 and 6 and p15.07 G 2, 3, 5, 6 were presenting the expected profile after insertion. At least the two brightest bands were visible, and the three other one were slightly visible or unvisible.
    We plated the corresponding colonies on LB + ery(150µG/mL) and incubated overnight at 37°C.

    Sequencing results from 31st of August did not give us any conclusive results of RibA, ribD, ribE and ribT’s insertion in p15.06 and p15.07 plasmids. As some primers gave confirmation of some of the inserts but some primers did not gave us any results at all, which may be due to off binding effect of the primers.


    <H1 class="date three">September 3rd</H1>

    Preparation of NEB Turbo electrocompetent cells (2mL final volume).


    Striking of the BB-RibT25 #8 on LB + Cm (25µg/mL).

    Seeing the poor results sequencing gave us, we decided to try whether the primers we were using were binding well or not.
    Also, as the 08/02 BB-RibD gradient PCR did not work, we decided to perform the PCR with both DreamTaq and Phusion.<TABLE style="width:500px"><TBODY><TR><TH>Amplified insert transition</TH><TH>Primers</TH><TH>Mix number</TH></TR><TR><TD>p15.01/RibT</TD><TD>o15.097+o15.108</TD><TD>1</TD></TR><TR><TD>RibT/RibE</TD><TD>o15.106+o15.107</TD><TD>2</TD></TR><TR><TD>RibE/RibD</TD><TD>o15.104+o15.105</TD><TD>3</TD></TR><TR><TD>RibD/RibA</TD><TD>o15.102+o15.103</TD><TD>4</TD></TR><TR><TD>RibA/p15.01</TD><TD>o15.098+o15.101</TD><TD>5</TD></TR></TBODY></TABLE>

    Amplification results were shown by electrophoresis.
    As we were expecting small bands, we used generuler 100bp and a 2% agarose gel. However, as you'll see right below.<B>Figure 55: </B><I>PCR amplification of the gBlock junction regions with Phusion and DreamTaq
    From Left to Right:
    Mix 1 - Mix 2 - Mix 3 - Mix 4 - Mix 5 - generuler 100bp - Mix 1 - Mix 2 - Mix 3 - Mix 4 - Mix 5</I>
    These results indicated us that some of the primers were not binding at the expected temperature (and that once again, the Phusion MasterMix was not working...).
    It also indicate that two of the couple of primers (couples 3 and 4) were binding well on the plasmid, however the sequencing results with the same plasmid and the forward primer of the couple number 3 (o15.105) gave a 0bp sequence. It showed that this primer is correctly binding and the negative sequencing result was not due to o15.105.
    

    Restriction digestion of pSB1C3 (3µg) with FD XbaI, FD BcuI (SpeI isoschizomer from ThermoScientific) and FastAP.
    The 2kb band of pSB1C3 was then gel extracted, purified and saved in water.<B>Figure 56: </B><I>pSB1C3 2kb band gel purification
    From Left to Right:
    generuler 1kb - pSB1C3 x4
    a- Before extraction
    b- After extraction</I>

    Ligation, dialyse and transformation of the digested and gel purified pSB1C3 with the BioBrick-tailed parts (RibA, RibE, RibT25, RibT48, p25 and p48).
    A 1:1 backbone:insert molarity ratio was use for the big inserts (RibA, E, T) and 1:3 for the smaller inserts.
    Electrocompetent NEB Turbo (electroporated without any DNA) were used as a negative control.

    Positive control was constituted by NEB Turbo transformed with pSB1C3.

    Dialysis was performed for 15 minutes, and 5µl of each dialysis product was transformed is 50µL of NEB Turbo electrocompetent cells.
    Cells were recover in LB for 1 hour and then plated on LB+ Cm(25µg/mL).

    Gradient PCR amplification of RibD from 58°C to 68°C with o15.029+o15.030 to add BioBrick tails around the gBlock.
    Elongation time span was set to 1.30 minutes.


    We received a chromosome integrative plasmid for <I>Lactobacillus plantarum</I> from Wageningen Centre for Food Science, Microbial Functionality and Safety Programme, and NIZO food research, Health and Safety Department, The Netherlands. (http://aem.asm.org/content/73/4/1126)

    <H1 class="date three">September 4th</H1>

    Liquid culture of a BB-RibT25 #8 single colony in LB+Cm (25µg/mL).

    Liquid culture of a single colony of pNZ5319<B>Transformation results </B>(from September third transformation)
    <TABLE style="width:500px"><TBODY><TR><TH>Transformed product</TH><TH>RibA</TH><TH>RibE</TH><TH>RibT25</TH><TH>RibT48</TH><TH>p25</TH><TH>p48</TH><TH>negative control</TH><TH>positive control</TH></TR><TR><TD>11 single colonies</TD><TD>2 single colonies</TD><TD>3 single colonies</TD><TD>1 single colony</TD><TD>28 single colonies</TD><TD>10 single colonies</TD><TD>0 single colony</TD><TD>30 red colonies</TD></TR></TBODY></TABLE>

    Colony PCR of 08/03 transformation with o15.182+o15.183 using DreamTaq.
    Eight colonies of BB-RibA, BB-p25 and BB-p48, one colony of BB-RibT48, two colonies of BB-RibE and three colonies of BB-RibT25 were analysed.
    Annealing was performed at 50°C and elongation time span was set to 2 minutes. Total volume was 20µL.<B>Figure 57: </B><I>Colony PCR of BioBrick transformants<UL><LI> a- From Left to Right:
        BB-RibA x8 - generuler 1kb - BB-RibE x2 - BB-RibT25 x3 - BB-RibT48 </LI><LI> b- From Left to Right:
            BB-Rp25 x8 - generuler 1kb - BB-p48 x8 </LI></UL></I>


    RibD gradient PCR (from 09/03) was analysed by electrophoresis.<B>Figure 58: </B><I>Colony PCR of BioBrick transformants
    From Left to Right:
    BB-RibD x8 (Tm = 58, 58.7, 60, 61.8, 64.1, 66.0, 67.3, 68°C) - generuler 1kb</I>

    These results were showing that the Master Mix, for once, was working well, but the three first wells were showing no band, but the wells with an annealing temperature above 60°C were presenting bands. However, the visible band is not the expected size.
    We were expecting a 1.6kb band, an we only got a 0.4kb band.
    So we simulate on geneious the binding of our primers allowing mismatches

    <P>
        PCR of mCerulean P25 and P48 gBlocks with respective oligos (223+224 and 225+226 respectively): gradient PCR from 45-70 degrees
    </P><P>
        PCR of pSIP409 plasmid with oligos for making the biobrick shuttle vector (182+183)<B style="color:red">(incorrect primers)</B>: gradient PCR from 45-70 degrees
    </P><P><B>Result:</B></P><P>Gel: wells with P25+mCerulean, P48+mCerulean and pSIP409</P><TABLE><TBODY><TR><TH>Well</TH><TH>P25</TH><TH>P25</TH><TH>P25</TH><TH>P25</TH><TH>P25</TH><TH>P25</TH><TH>P25</TH><TH>1kb plus</TH><TH>P48</TH><TH>P48</TH><TH>P48</TH><TH>P48</TH><TH>P48</TH><TH>P48</TH><TH>P48</TH><TH>P48</TH><TH>pSIP409</TH></TR><TR><TH>Temperature</TH><TD>45</TD><TD>46.8</TD><TD>49.9</TD><TD>59.9</TD><TD>64.8</TD><TD>68.0</TD><TD>70</TD><TD>-</TD><TD>45</TD><TD>46.8</TD><TD>49.9</TD><TD>54.4</TD><TD>59.9</TD><TD>64.8</TD><TD>68.0</TD><TD>70</TD><TD>54</TD></TR></TBODY></TABLE><H1 class="date three">September 5th</H1>

    Glycerol stock and miniprep of BB-RibT25 and pNZ319.

    Restriking of BB-RibT48 that showed a interesting band on LB+Cm (25µg/mL).

    Digestion of BB-Tdhl with XbaI and BcuI.
    Digestion product was PCR purified (the presence of the insert was controlled by electrophoresis) and then ligated with pSB1C3 (previously digested and gel extracted). The ligation product was then dialysed for 20 minutes.<B>Figure 59: </B><I>Colony PCR of BioBrick transformants
    From Left to Right:
    generuler 1kb - Tdhl terminator</I>

    5µL of Tdhl product and 1µL of pSB1C3 were trasnformed by electroporation in 50µL of NEB Turbo.
    Cells were recovered for two hours in 500µL of LB.
    After recovery, cells were plated on LB+Cm (25µg/mL) and incubated at 37°C.<P>PCR of mCerulean P25 at 55 and 60 degrees.</P><P>PCR of pSIP409 with correct primers. Extension time 2:00</P><TABLE><TBODY><TR><TD>p25 55 degree</TD><TD>p25 60 degree</TD><TD>1KB ladder</TD><TD>pSIP409 55 </TD><TD>pSIP409 60 </TD></TR></TBODY></TABLE><P>No band. Gel Lost.</P><H1 class="date three">September 6th</H1>
    Overnight cultures of pKV7, pLB65 and a <I>Lactococcus lactis subps cremoris MG1363</I>containing pSIP411(erythromycin resistance plasmid) in LB + respectively ampicilin, chloramphenicol and erythromycin (to prevent contamination).

    Here, we were using a erythromycin resistant MG1363 because we did not manage to grow the wild type MG1363 we received three times.<B>Transformation results </B>(from September third transformation)
    <TABLE style="width:500px"><TBODY><TR><TH>Transformed product</TH><TH>BB-Tdhl</TH><TH>negative control</TH><TH>positive control</TH></TR><TR><TD>1 red colony</TD><TD>0 single colony</TD><TD>107 red colonies</TD></TR></TBODY></TABLE>

    As pSB1C3 was gel purified, it was unexpected to see a red colony. The small amount of colonies might be due to the fact taht the cells stayed a long time in ice before being saved at -80°C.<H1 class="date three">September 7th</H1>

    Miniprep of pKV7, pLB65 and BB-RibT48.

    Restriction digestion of pSB1C3 with EcoRI and SpeI.<B>Figure 60: </B><I>Restriction digestion of pSB1C3 with EcoRI and SpeI
    From Left to Right:
    generuler 1kb - pSB1C3 digested</I>

    One can clearly see that something wrong happened during the digestion. As we changed XbaI for EcoRI from the previous digestion, we suspected that EcoRI was maybe sensitive to methylation of was not working.
    Thus, we decided to use XbaI instead of EcoRI, even if it could allow a recircularization of the plasmid (because of XbaI and SpeI compatible overhangs).

    PCR amplification of the two p15.06 and the four p15.07 presenting the four genes inserted with o15.027+o15.034 to BioBrick the four genes at the same time. Annealing was performed at 50°C and elongation was set to 2.45 minutes. RibT25+o15.033+o15.034 was used as a positive control.<B>Figure 61: </B><I>RibADET PCR amplification
    From Left to Right:
    generuler 1kb - p15.06 G5 - p15.06 G6 - p15.07 G2 - p15.07 G3 - p15.07 G5 - p15.07 G6 - BB-RibT25</I>

    It appeared that none of the amplification (even the positive control) worked.
    Once more, we assumed that either the master mix was wrong or something prevented the PCR to work as expected.


    Preparation of electrocompetent Lactococcus lactis

    Digestion of BB-RibA, BB-RibD, BB-RibE, BB-RibT25, BB-RibT48, BB-p25, BB-p48 and BB-Tdhl with XbaI and SpeI.
    Digestion products were PCR purified.
    Ligation and transformation by electroporation of pSB1C3 + one insert.
    200µL of cells were recovered for one hour then plated on LB+Cm(25µg/mL) and incubated overnight at 37°C.

    RibD PCR with o15.029+o15.030 to add the biobrick tails. Annealing was performed at 62°C and elongation time span was set to one minute.<H1 class="date three">September 8th</H1>


    RibD PCR with tails was analysed by electrophoresis.<B>Figure 62: </B><I>PCR analysis of RibD amplification by electrophoresis
    From Left to Right:
    generuler 1kb - RibD x4</I>

    One can clearly see that the 1.3kb band is present as expected, however a second band of 0.4kb is also visible.
    Thus, we gel purified the 1.3kb band.<B>Figure 63: </B><I>RibD gel purification
    a- Before band excision
    b- After gel excision
    From Left to Right:
    generuler 1kb - RibD x7</I>

    PCR amplification of p15.06 and p15.07 with o15.027+o15.034 to amplifiy the four genes with BioBrick tails. This time, we prepared the MasterMix ourselves.
    Annealing was performed at 50°C and elongation was set to 2.45 minutes.<B>Figure 64: </B><I>p15.06 and p15.07 PCR amplification
    From Left to Right:
    generuler 1kb - p15.06 G5 - p15.06 G6 - p15.07 G2 - p15.07 G3 - p15.07 G5 - p15.07 G6 - BB-RibT25</I>
    As the last PCR on p15.06 and p15.07, this PCR failed. Even the positive control is not showing any band.<B>Transformation results</B><TABLE style="width:1500px"><TBODY><TR><TH>Transformed product</TH><TH>pSB1C3</TH><TH>pSB1C3+BB-RibA</TH><TH>pSB1C3+BB-RibD</TH><TH>pSB1C3+BB-RibE</TH><TH>pSB1C3+BB-RibT25</TH><TH>pSB1C3+BB-RibT48</TH><TH>pSB1C3+BB-p25</TH><TH>pSB1C3+BB-p48</TH><TH>pSB1C3 gel purified and ligated</TH><TH>negative control</TH></TR><TR><TD>Growth Results</TD><TD>a beautiful red lawn</TD><TD>325 single colonies</TD><TD>213 single colonies</TD><TD>278 single colonies</TD><TD>450 single colonies</TD><TD>more than 800 single colonies</TD><TD>more than 2000 single colonies</TD><TD>more than 2000 single colonies</TD><TD>more than 200 white single colonies</TD><TD>0 colony</TD></TR></TBODY></TABLE><H1 class="date three">September 9th</H1>


    Preparation of Top10 (<I>E.coli</I>) electrocompetent cells.

    PCR amplification of p15.06 and p15.07 (transformants 5 and 6 and 2, 3, 5 and 6) with o15.027+o15.034 to amplifiy the four genes with BioBrick tails.
    Annealing was performed at 65 or 70°C and elongation was set to 2.30 minutes.
    Amplification was controlled by electrophoresis.<B>Figure 65: </B><I>p15.06 and p15.07 PCR amplification
    a- Annealing at 65°C
    b- Annealing at 70°C
    From Left to Right:
    generuler 1kb - positive control - p15.07 G2 - p15.07 G3 - p15.07 G5 - p15.07 G6 - p15.06 G5 - p15.06 G6</I>

    Restriction digestion of p15.07 G 2 and 5 with Eco31I for 20 minutes.
    Digestion was controled by electrophoresis on 1% agarose gel.<B>Figure 66: </B><I>p15.07 analytical digestion
    From Left to Right:
    generuler 1kb - p15.07 G2 - p15.07 G5</I>
    p15.07 G2 seemed to present interesting bands after digestion.


    PCR amplification of p15.06 and p15.07 (transformants 5 and 6 and 2, 3, 5 and 6) with o15.097+o15.098 to amplifiy the four genes. Annealing was performed at 64°C and elongation time span was set to 2.45 minutes.<B>Figure 67: </B><I>p15.06 and p15.07 PCR amplification control by electorphoresis
    From Left to Right:
    generuler 1kb - p15.07 G2 - p15.07 G3 - p15.07 G5 - p15.07 G6 - p15.06 G5 - p15.06 G6</I>
    This electrophoresis showed that, unfortunately, an other MasterMix was not working anymore... (an other team used it at the same time and none of their amplification worked).

    Colony PCR was performed on eight colonies from BB-RibA, BB-RibE, BB-p25 and BB-p48 from the 8th September transformation with o15.182+o15.183.
    Annealing temperature was set at 50°C and elongation time step was set to 2 minutes.
    Amplification was then controlled by electrophoresis.<B>Figure 68: </B><I>Colony PCR of inserted pSB1C3
    From Left to Right:<UL><LI> a- From Left to Right:
        negative control - positive control - BB-RibA x7 - generuler 1kb - BB-RibE x7</LI><LI> b- From Left to Right:
            negative control - positive control - BB-p25 x7 - generuler 1kb - BB-p48 x7</LI><LI> c- From Left to Right:
            generuler 1kb - negative control - positive control - BB-RibA - BB-RibE - BB-p25 - BB-p48</LI></UL></I>


    Dialyse and transformation by electroporation of p15.07 G2 and G5 in Top10 <I>E.coli</I>
    Cells were recovered for one hour and then 100µL were plated on LB+ery (150µG/mL).<H1 class="date three">September 10th</H1>


    Miniprep of p15.06 G4, G5 and G6, p15.07 G2, G3, G5, and G6.<B>Transformation results</B><TABLE style="width:500px"><TBODY><TR><TH>Transformed product</TH><TH>p15.07 G2</TH><TH>p15.07 G5</TH><TH>negative control</TH></TR><TR><TD>Growth Results</TD><TD>no growth</TD><TD>no growth</TD><TD>no growth</TD></TR></TBODY></TABLE>

    Colony PCR of 7th of September transformation. Seven colony of each BB-RibA, BB-RibE, BB-p25 and BB-p48 were amplified with o15.182+o15.183.
    Annealing was performed at 50°C and elongation was set to 2 minutes.
    Amplification was controlled by electrophoresis.<B>Figure 69: </B><I>Biobrick insertion control in pSB1C3 by colony PCR
    From Left to Right:<UL><LI> a- From Left to Right:
        pSB1C3 - BB-RibA x7 - generuler 1kb - BB-RibE x7</LI><LI> b- From Left to Right:
            pSB1C3 - BB-p25 x7 - generuler 1kb - BB-p48 x7 </LI></UL></I>

    None of the amplification worked.

    Digestion of p15.06 G5 and G6 and p15.07 G2, G3, G5 and G6 by SpeI (unique restriction site).<B>Figure 70: </B><I>Digestion of p15.06 and p15.07 with SpeI
    UnDigested: UD, Digested: D
    From Left to Right:
    generuler 1kb plus- p15.06 G5 UD - p15.06 G5 D - p15.06 G6 UD - p15.06 G6 D - p15.07 G2 UD - p15.07 G2 D - p15.07 G3 UD - p15.07 G3 D - p15.07 G5 UD - p15.07 G5 D - p15.07 G6 UD - p15.07 G6 D</I>

    The digested plasmid were showing several bands, likely to come from a mixed population between recombined plasmids and complete plasmids.


    Colony PCR of 15 colonies of p15.06 G4 with o15.097+o15.098.
    Annealing was performed at 55°C and elongation time span was set to 6 minutes.<B>Figure 71: </B><I>Colony PCR amplification control by electrophoresis
    From Left to Right:
    generuler 1kb - p15.06 G4 x15</I>

    All the wells were presenting an interesting band of about 4.5 kb. Thus, we decided to perform a restriction digestion on a miniprep of the same cells, with all the others miniprep of p15.06 and p15.07.

    Golden Gate assembly of p15.06 and p15.07 with the G protocol.<H1 class="date three">September 11th</H1>

    Digestion of p15.06 G4, G5 and G6 and p15.07 G2, G3, G5 and G6 by SpeI (unique restriction site).<B>Figure 72: </B><I>Digestion of p15.06 and p15.07 with SpeI
    UnDigested: UD, Digested: D
    From Left to Right:
    generuler 1kb plus - p15.06 G4 UD - p15.06 G4 D - p15.06 G5 UD - p15.06 G5 D - p15.06 G6 UD - p15.06 G6 D - p15.07 G2 UD - p15.07 G2 D - p15.07 G3 UD - p15.07 G3 D - p15.07 G5 UD - p15.07 G5 D - p15.07 G6 UD - p15.07 G6 D</I>


    Preparation of STBL 3 electrocompetent electrocompetent cells.


    Transformation of 9th September golden gate assembly mix of p15.06 and p15.07 in STBL3 <I>E.coli</I> after dialysis.
    Cells were recovered three hours and then plated on LB+ery(150µg/mL).

    Miniprep of p15.07 G2 and G5 (from 09/09 transformation in Top10), p25 #1, 2 and 5, p48 #3, 5 and 8.

    The three p25, p48 and p15.07 G2 and G5 were sent to sequencing.

    Restriction digestion of BB-RibD with XbaI and BcuI.<P>Remade the oligos: 186, 187, 223, 224.</P><P>Gradient PCR of mCerulean P25 and pSIP409 biobrick plasmid: 48-70 degrees, extension time 1:30.</P><H1 class="date three">September 12th</H1><B>Transformation results</B><TABLE style="width:500px"><TBODY><TR><TH>Transformed product</TH><TH>p15.06</TH><TH>p15.07</TH><TH>negative control</TH></TR><TR><TD>Growth Results</TD><TD>3 single colonies</TD><TD>4 single colonies</TD><TD>no growth</TD></TR></TBODY></TABLE>

    Colony PCR of three transformants of p15.06 and the four transformants of p15.07 with o15.097+o15.098. Annealing was performed at 55°C and elongation time step set to 5 minutes.<B>Figure 73: </B><I>Colony PCr of p15.06 and p15.07 transformants
    From Left to Right:
    generuler 1kb plus - p15.06 x3 - p15.07 x4</I>
    This electrophoresis did not present any band. Maybe the gel was run too long, and negative results ran away from the gel, or the PCR failed.

    Ligation, dialyse and transformation of pSB1C3 with BB-RibA, BB-RibD, BB-RibE, BB-Tdhl. During electroporation, BB-Tdhl showed an arch. Cells were recovered for one hour and then plated on LB+Cm(25µg/mL).

    Colony PCR of three transformants of p15.06 and the four transformants of p15.07 with o15.097+o15.098. Annealing was performed at 55°C and elongation time step set to 6 minutes.<P>Gel</P><P>pSIP409 Biobrick plasmid						mCerulean p25 45</P><TABLE><TBODY><TR><TD>48</TD><TD> </TD><TD> </TD><TD> </TD><TD>70</TD><TD>1 kb plus</TD><TD>48</TD><TD>70</TD></TR></TBODY></TABLE><H1 class="date three">September 13th</H1><B>Transformation results</B><TABLE style="width:500px"><TBODY><TR><TH>Transformed product</TH><TH>BB-RibA</TH><TH>BB-RibD</TH><TH>BB-RibE</TH><TH>BB-Tdhl</TH><TH>digested pSB1C3</TH><TH>negative control</TH></TR><TR><TD>Growth Results</TD><TD>0 colony</TD><TD>0 colony</TD><TD>0 colony</TD><TD>0 colony</TD><TD>0 colony</TD><TD>0 colony</TD></TR></TBODY></TABLE>


    Preparation of electrocompetent <I>Top10</I> and <I>STBL3</I>.

    Colony PCR on the four colonies of p15.06 and p15.07 from the 11th transformation with o15.097 + o15.098. Annealing was performed at 55°C and elongation time span set to 6 minutes.<B>Figure 74: </B><I>Colony PCR of p15.06 and p15.07 transformants
    From Left to Right:
    generuler 1kb plus - p15.06 x3 - p15.07 x4</I>
    The visible bands of about 0.8 kb are not corresponding neither to the a negative insertion nor to a positive insertion.
    These results should deserve further investigation, but for the lack of time, we had chosen to just ignore these results and kept the focus on getting a positive insertion.


    Restriction digestion of pSB1C3 (3µg) with FD XbaI, FD BcuI and FastAP.
    The 2kb band of pSB1C3 was then gel extracted, purified and saved in water.<B>Figure 75: </B><I>pSB1C3 2kb band gel purification
    From Left to Right:
    generuler 1kb - pSB1C3 x4
    a- Before extraction
    b- After extraction</I>

    Ligation, dialyse and transformation of pSB1C3  with BB-RibA, BB-RibD, BB-RibE, BB-Tdhl in Top10 <I>E.coli</I>.
    Two negatives control were performed:
    -non transformed Top10
    -Top10 transformed with the gel purified pSB1C3 after ligation.
    Positive control used was the native pSB1C3.
    50µL of cells were electroporated with 1µL of the dialysed ligation product.
    As we wanted to also test the STBL3 competence, we transformed pSB1C3 in STBL3 (and a negative control).
    150µL of each transformation was transformed on LB+Cm(25µg/mL).

    At this point, we also realized that we plated the previous transformation on LB + ery instead of LB + cm...<P>mCerulean P48 and pSIP409BB plasmid PCR purification</P><P>Digestion of mCerulean P48 and pSB1C3 with XbaI and SpeI.</P><P>Digestion of pSIP409BB with XbaI and DpnI (to eliminate the background original plasmid).</P><P>Ligation of (P48+pSB1C3)</P><P>Ligation of pSIP409BB plasmid (recircularization)</P><P>Transformation in Top10 E.coli electro-competent cells.</P><P>Electroporation parameters:</P><P>Voltage: 2500 V</P><P>Capacitance: 25 uF</P><P>Resistance: 400 ohms</P><P>Cuvette: 2mm</P><P>Recovery for 1h in 500mL LB</P><P>Plating: 50uL in one plate and rest of the cells concentrated in 100uL by centrifuging. (P48+pSB1C3) in  LB+Chloramphenicol(25ug/mL) and pSIP409BB in LB+(Erythromycin)

	</P><P>Effect of steam cooking on Idli microflora:</P><P>Idli was put for overnight fermentation in the standard way.</P><H1 class="date three">September 14th</H1>

    Preparation of <I> Lactococcus lactis subsp. cremoris MG1363</I>.

    Sequencing results of p25 showed that none of the suspect transformants contained the BioBrick. However, one transformant of p48 was presenting the promoter without any SNP.
    Sequencing results of p15.07 G2 and G5 were not exploitable. Among the 12 sequences we were expecting to get, only two were longer than 0bp (52 and 46bp long...).
    Analysis of the ab1 files showed that the signal was not specific, indicating that either the miniprep solution were not pure, or that the primers are binding at several places.<B>Transformation results</B><TABLE style="width:500px"><TBODY><TR><TH>Transformed product</TH><TH>BB-RibA</TH><TH>BB-RibD</TH><TH>BB-RibE</TH><TH>BB-Tdhl</TH><TH>digested pSB1C3</TH><TH>negative control</TH><TH>positive control</TH><TH>STBL3 + pSB1C3</TH></TR><TR><TD>Growth Results</TD><TD>about 600 colonies</TD><TD>about 100 colonies</TD><TD>about 500 colonies</TD><TD>about 350 colonies</TD><TD>10 red colonies</TD><TD>0 colony</TD><TD>about 3000 red colonies</TD><TD>about 3000 red colonies</TD></TR></TBODY></TABLE>

    Colony PCR was performed on 8 colonies of BB-RibA, BB-RibD, BB-RibE and BB-Tdhl with o15.182+o15.183.
    Annealing was performed at 50°C and elongation time span was set to 2 minutes.
    Amplification was then controlled by electrophoresis.
    Both gel were not presenting any band.

    PCR amplification of p15.06 G4, G5 and G6, p15.07 G2, G3, G5 and G6 and an other p15.06 presenting yellow dye with o15.097+o15.098.
    Annealing was performed at 64°C and elongation time span set to 3 minutes.
    Amplification was analysed by electrophoresis.<B>Figure 76: </B><I>PCR amplification of several p15.06 and p15.07
    From Left to Right:
    generuler 1kb plus - p15.06 G5 - p15.06 G6 - p15.07 G2 - p15.07 G3 - p15.07 G5 - p15.07 G6 - p15.06 G4 - p15.06 yellow</I>

    The about 5kb band of p15.06 G4 and GY and p15.07 G2, G5 and G6 was gel purified.<B>Figure 77: </B><I>Gel purification of p15.06 and p15.07
    From Left to Right:
    a- genereuler 1kb plus - p15.07 G2 x3 - p15.07 G5 x4 - p15.07 G6 x4
    b- genereuler 1kb plus - p15.06 G4 x4  - p15.06 GY x4</I>

    After gel purification, purity was analysed by electrophoresis.<B>Figure 78: </B><I>Gel purification control by electrophoresis</I>
    The bands seemed pure enough. The purified product was sent to sequence.

    Colony PCR on 8 transformants of BB-RibA, BB-RibD, BB-RibE and BB-Tdhl with o15.0182+o15.183.
    Annealing was performed at 50°C and elongation time span was set to 2 minutes.<B>Figure 79: </B><I>Colony PCR of BioBrick transformants
    From Left to Right:<UL><LI> a- From Left to Right:
        BB-RibA x8 - generuler 1kb  plus- BB-RibD x8</LI><LI> b- From Left to Right:
            BB-RibE x8 - generuler 1kb  plus- BB-Tdhl x8</LI></UL></I><B>Colonies appeared on the transformation plates, and were approximately counted as follows:</B><TABLE width="100%" border="1" cellspacing="0" cellpadding="4"><TBODY><TR><TD>50uL</TD><TD>100uL</TD><TD>Control (only pSB1C3)</TD><TD>Control (After DpnI digestion)</TD></TR><TR><TD>P48 in pSB1C3</TD><TD>3 + 2(Red)</TD><TD>13</TD><TD>13 + 3 (Red)</TD></TR><TR><TD>pSIP409BB</TD><TD>74</TD><TD>~250</TD><TD>-</TD><TD>~130</TD></TR></TBODY></TABLE><P>All transformants for P48 (except the ones which were red) were fluorescent blue due to the presence of mCerulean gene sequence in front of it. (tested in transilluminator as follows)</P>.
<P>3 transformants for pSIP409BB were picked and were mini-prepped, digested with BcuI (alternative for SpeI) and run on the gel.</P><P>The upper band for T1 and T2 probably corresponds to the uncut plasmid and the lower one to the linear one (it was expected to be ~2.9kb).</P><P>0.5g-0.6g each of fermented batter was inoculated in 5 different media (5mL each)</P><UL><LI>Electroporation parameters:</LI><LI>Voltage: 2500 V</LI><LI>Capacitance: 25 uF</LI><LI>Resistance: 400 ohms</LI><LI>Cuvette: 2mm</LI></UL>
Fermented batter was steamed and 0.5g-0.6g of idli was inoculated similarly in each of the above media.

After 3 days, 50mL of each of the culture was plated on respective plates (for both before and after the steaming process), along with control plates for each media and type (before and after).

    <H1 class="date three">September 15th</H1>

    Colony PCR on 16 transformants of BB-RibA, BB-RibD, BB-RibE and BB-Tdhl with o15.0182+o15.183.
    Annealing was performed at 50°C and elongation time span was set to 2 minutes.<B>Figure 80: </B><I>Colony PCR of BioBrick transformants
    From Left to Right:<UL><LI> a- From Left to Right:
        BB-RibA x8 - generuler 1kb plus - BB-RibA x8</LI><LI> b- From Left to Right:
            BB-RibD x8 - generuler 1kb plus - BB-RibD x8</LI><LI> c- From Left to Right:
            BB-RibE x8 - generuler 1kb plus- BB-RibE x8</LI><LI> d- From Left to Right:
            BB-Tdhl x8 - generuler 1kb plus - BB-Tdhl x8</LI></UL></I>

    BB-RibA was run too long, so the negative bands are not visible on the gel.
    All the bands of BB-RibE and BB-Tdhl are negative.
    12 among the 16 BB-RibD were presenting bands of the expected size. The 12 interesting colonies were sow in 5mL LB+cm(25µg/mL).

    PCR amplification of p15.06 G4, GY, p15.07 G2, G5 and G6 with o15.280+o15.281.<B>Figure 81: </B><I>Amplification control of p15.06 and p15.07
    From Left to Right:
    generuler 1kb plus - p15.07 G2 - p15.07 G5 - p15.07 G6 - p15.06 G4 - p15.06 GY</I>

    Cotransformation of <I>L. lactis subsp. cremoris MG1363</I> pLB65 + p15.06 G4, G5 or G6, p15.07 G2, G3 or G5 or p15.01.
    Plasmids were dialysed and electroporated in the cells.
    After 3 hours of recovery, cells were plated on M17 + 1% glucose + ery 5µg/mL.<P>Transformant T1 was glycerol stocked.</P><P>Fluorescent blue transformant from P48+mCerulean were re-streaked in a differential plating to establish the difference between the fluorescent transformant and the control E.coli cells.</P><P>3 colonies each (biological replicates) of both these strains were taken, and were grown in LB+Erythromycin (200ug/mL) and their absorbance and fluorescence was taken after 18h of growth as 3 technical replicates of each (total 9 measurements for each strain type), and also each one’s 10 time diluted form.
	The culture tubes at the end of 18h looked like the following under the fluorescence transilluminator.
	</P><P>In the manufacturing and Idli Notebook</P><H1 class="date three">September 16th</H1>

    Digestion of p15.06 G4, G5 and G6 and p15.07 G2, G3, G5; p15.06 GY with SpeI FD. A negative control without restriction enzyme was performed on p15.06 G4.
    We wanted to see whether the digestion profile were looking good or not, and if yes to extract the linear plasmid, recircularize it and transform it.<B>Figure 82: </B><I>Gel purification of linearized p15.06 and p15.07
    From Left to Right:
    generuler 1kb plus - negative control - p15.06 G4 - p15.06 G5 - p15.06 G6 - p15.07 G2 - p15.07 G3 - p15.07 G5 - p15.07 G6</I>

    After gel purification, the three linear plasmids were ligated, dialysed and then transformed in electrocompetent STBL3 <I>E. coli</I>.
    100µL of cells were transformed with all the ligation product (about 10µL), because the DNA concentration after gel extraction was really low.
    Cells were recovered to hours in LB at 37°C and then plated on LB+ery (150µg/mL).

    PCR amplification of p15.06 G4, GY, p15.07 G2, G5 and G6 with o15.097+o15.098.
    Gel purification of  of p15.06 G4, p15.07 G2 and G5/<B>Figure 83: </B><I>Gel purification of p15.06 and p15.07
    From Left to Right:
    generuler 1kb plus - p15.07 G2 x3- p15.07 G5 x3- p15.06 G4 x3</I>

    After gel purification, the purified product was PCR again with the same oligos in the same conditions to increase its concentration.<I>Lactococcus lactis MG1363</I> was grow in LB and in Idli water to obtain growth information.

    In order to measure the amount of riboflavin released in the media by the cells, we took the four <I> E. coli </I> strains producing a bright yellow dye and inoculate them in LB+ery (150µg/mL) or M9 + ery(150µg/mL).
    At the same time, we add riboflavin at different concentration in LB, M9 and water +ery (150µg/mL) to create a standard curve (0.05, 0.0725, 0.100, 0.125, 0.15, 0.175, 0.20µg/mL).
    A non producing E.coli was also sowed in Lb and M9 +ery(150µg/mL).
    The cells and the riboflavin solutions were incubated at 37°C.<H1 class="date three">September 17th</H1>

    LB, M9 and water containing different concentration of riboflavin were excited at 440nm and the light emitted at 565nm was measured. Fluorescence measures increased from about 80 to about 4300 in water and M9, and from 16000 to 21000 a.u..
    Increasing was nicely linear for M9 and water and somehow linear for LB.
    M9 and LB cultures of p15.06 G5, G6 and p15.07 G2 and G5 were centrifuged 10 minutes at 6000rpm.
    Fluorescence of the supernatant and of the three first 10 fold serial dilutions were measured.
    The non diluted supernatant returned a fluorescence 10 times higher than the highest fluorescence we get from the scale.
    The first ten-fold dilution returned a fluorescence slightly above the more concentrated solution of the dyeset.
    This virtual equality allowed us to approximately estimate the concentration of vitamin B2 in the supernatant.
    The highest concentration of the dyeset was 0.2µg/mL. Thus, the estimated concentration of riboflavin in the supernatant was about 2µg/mL.
    From this experiment, we decided to launch again the same king of experiment, but without forgetting to launch the negative control (Media + riboflavin non-producer <I>E.coli</I>), without using LB (the background of fluorescence is really high in this media and could alter the measurement) and making technical and biological replicates (respectively 8 an 4).<B>Lactococcus cotransformation results </B>(from September third transformation)
    <TABLE style="width:500px"><TBODY><TR><TH>Transformed product</TH><TH>p15.06 G4</TH><TH>p15.06 G5</TH><TH>p15.06 G6</TH><TH>p15.07 G2</TH><TH>p15.07 G3</TH><TH>p15.07 G5</TH><TH>negative control</TH><TH>positive control</TH></TR><TR><TD>0 colony</TD><TD>0 colony</TD><TD>0 colony</TD><TD>1 colony</TD><TD>3 colonies</TD><TD>0 colony</TD><TD>0 single colony</TD><TD>107 red colonies</TD></TR></TBODY></TABLE>
    The four colonies were inoculated in 5mL GM17 + ery (3µg/mL).</DIV></DIV></DIV></DIV></BODY></HTML>