<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Notebook_VitaminA skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Paris Bettencourt/Notebook/VitaminA</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="menuContainer"><DIV id="smallMenuTitle"><H1>Ferment It Yourself</H1><H3>iGEM Paris-Bettencourt 2O15</H3></DIV><UL><LI class="left"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Background" title="Background">Background</A></LI><LI class="left"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Design" title="Design">Design</A></LI><LI class="left"><BUTTON class="menuButton">Project</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Micronutrients">Micronutrients</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Continuity">Continuity</A></LI><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Manufacturing">Manufacturing</A></LI></DIV></UL><LI class="left"><BUTTON class="menuButton">Achievements</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Parts">Parts</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Medals">Medal Fulfillments</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Practices</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/SynBio_Challenge">SynBio Challenge</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/DILambda">DIλ</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Acceptance">Acceptance</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/Events">Events</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/Rhizi">Rhizi</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Collaborations">Collaborations</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Safety</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Safety">Safety Overview</A></LI><LI><A href="https://2015.igem.org/Safety/Final_Safety_Form?team_id=1678">Safety Form</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/ReviewBoard">Review Board</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Notebook</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Notebook">Daily notebook</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols">Protocols</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Bibliography">Bibliography</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Team</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Team">Members</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Acknowledgments">Acknowledgments</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Press">Press</A></LI><LI><A href="https://igem.org/Team.cgi?id=1678">Official Team Profile</A></LI></DIV></DIV><DIV id="notebook"><DIV class="contentContainer"><DIV id="banner"><H3>Notebook - Vitamin A
</H3></DIV></DIV><DIV class="textWrapper"><DIV class="textContent {{{page_option}}}" id="caffeine"><DIV id="notebookMenu"><UL><LI><A href="#july">July</A></LI><LI><A href="#august">August</A></LI><LI><A href="#september">September</A></LI><LI><A href="#oligos">Oligos</A></LI></UL></DIV><DIV id="notebookContent"><H1 class="date one">July 14th</H1><H2>Goal</H2>
Extract the integrative plasmid HO-Poly-KanMX4-HO from the E.Coli provided by AddGene (Accession number #51662).

<H2>Procedure</H2><OL><LI>Liquid culture overnight in LB + Ampicillin.</LI><LI>Made a glycerol stock and stored it in the -20 freezer (g15.35)</LI><LI>Centrifuge the tube for 1 minute with 11000 rpm.</LI><LI>Miniprep</LI><LI>Throw out the supernatant</LI><LI>resuspend in 250 uL of resuspension solution in an Eppendorf</LI><LI>Add 250 uL of lysis solution for 2 min then add 350 uL of neutralization solution, and shake it lightly.</LI><LI>10 min of centrifugation at 14k rpm</LI><LI>supernatant is pour in a column and centrifugated for 30 sec at 14k rpm in a column</LI><LI>supernatant is discarded and 700uL of washing solution are added then the column is centrifugated at 14k rpm for 30s</LI><LI>supernatant is discarded and 500uL of washing solution are added then the column is centrifugated at 14k rpm for 30s</LI><LI>supernatant is discarded, the column is centrifugated at 14k rpm for 120s</LI><LI>put the column in another tube and add 45uL of DNAse RNAse free water in the middle of the column</LI><LI>wait 2 min</LI><LI>centri for 2 min at 10k rpm</LI><LI>discard the column</LI></OL><LI>Measure concentration with Nanodrop</LI><H2>Results</H2>
Final DNA concentrations of the 4 tubes of miniprep, measured with Nanodrop:
<UL><LI>tube HO-Poly-KanMX4-HO pl. 1 = 431.1 ng/uL</LI><LI>tube HO-Poly-KanMX4-HO pl. 2 = 313.6 ng/uL</LI><LI>tube HO-Poly-KanMX4-HO pl. 3 = 366.4 ng/uL</LI><LI>tube HO-Poly-KanMX4-HO pl. 4 = 261.7 ng/uL</LI></UL><H1 class="date one">July 15th</H1><H2>Goal</H2>
Test chromosomal integration in WT yeast SK1 with the integrative plasmid HO-Poly-KanMX4-HO.

<H2>Procedure</H2>

We followed the method described in <B>&quot;High-efficiency Yeast transformation using liAc/SS carrier DNA/PEG method&quot;</B> (Gietz 2007).
<OL><LI>Inoculation of a single colony of the SK1 yeast strain in liquid YPD overnight on a rotatory shaker at 130 r.p.m and 30°C.</LI><LI>After 16 hours the titer of the cell culture was determined. The OD 600 nm of a 1/100 dilution (10 uL in 1 mL) was measured and the cell concentration determined using the formula (1*10^6 cells.mL^-1 will give an OD600nm of 0.1).</LI><LI>2.5*10^8 cells were added to 50 mL of pre-warmed YPD and incubated for 4.5 hours at 30°C and 130 rpm.</LI><LI>1.0 mL of salmon sperm carrier DNA was denaturated in a heat block at 99°C during 5 min, and chilled rapidly in ice.</LI><LI>The cells where harvested by centrifugation at 3000g for 5 min and resuspended in 25 mL of water and centrifuged again 5 min at 3000g. The washing process was repeated again, then the cells were resuspended in 1.0 mL of sterile water.</LI><LI>The suspension was transfered into a 1.5 mL eppendorf tube and centrifuged for 30s at 13,000g and the supernatant discarded</LI><LI>The cells were resuspended in 0.5 mL of sterile water. 100uL of the solution are pipette in a 1.5mL microcentrifuge then the transformation mix has been added (240uL of PEG 3350,36uL of liAc 1.0 M, 50 uL of the single stranded DNA carrier (2.0 mg.mL^-1, 35 uL of DNA plus sterile water up to a total of 360 uL.</LI><LI> tubes were placed at 42°C for 40 min.</LI><LI> tubes were centrifuged at 13 000g for 30s in a microcentrifuge and the supernanant removed with a micropipettor. Pellet was resuspended in YPD and incubate for 3 hours at 30°C to ensure good expression of the antibiotic resistance.</LI><LI>2, 20 and 200 uL of the cell suspension were plated on YPD agar + G418 and spread with glass beads.</LI><LI> plates were put to grow at 30°C for 3 days
</LI></OL><H2>Results</H2><DIV class="column-left">We had many transformants, with the resistance marker:
<UL><LI>from left to right: dilutions 1/1 1/10 1/100</LI><LI>first line: without g418 </LI><LI>second line: with g418</LI></UL></DIV><H1 class="date one">July 22nd</H1>Received gBlocks vA-2, vA-3 and vA-4, which form the last parts of the polycistron, and the corresponding amplification oligos from IDT.
Resuspended gBlocks vA-2, vA-3 and vA-4 in 100 uL water, to reach a final concentration of 10 ng/uL.
Made aliquots of these gBlocks at concentration 1 ng/uL.
Resuspended primers o15.056, o15.076, o15.058, o15.059, o15.060, o15.061 in water to reach a final concentration of 100 uM for each.
Made aliquots of each primer at concentration 10 uM.

<P>
PCR amplification, using the following protocol:
<UL><LI>1 uL of gBlock (1 ng)</LI><LI>2 uL forward primer</LI><LI>2 uL reverse primer</LI><LI>50 uL Master Mix 2X</LI><LI>45 uL water</LI></UL></P><P>
We made 2 <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR">PCR</A> tubes for each gBlocks, with the following primers:
<B>vA-2 + o15.056 + o15.057</B><B>vA-3 + o15.058 + o15.059</B><B>vA-4 + o15.060 + o15.061</B></P><P>
Settings <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR">PCR</A>:
35 cycles amplification, using the following parameters:
<TABLE style="width:25%"><TBODY><TR style="background-color:#E6E6E6"><TH>time (min)</TH><TH>temperature (°C)</TH><TH>function</TH></TR><TR><TD>0:30</TD><TD>98</TD><TD>melting</TD></TR><TR><TD>0:10</TD><TD>98</TD><TD>melting</TD></TR><TR><TD>0:30</TD><TD>50</TD><TD>annealing</TD></TR><TR><TD>1:00</TD><TD>72</TD><TD>extension</TD></TR><TR><TD>10:00</TD><TD>72</TD><TD>extension</TD></TR><TR><TD>forever</TD><TD>10</TD><TD>storage</TD></TR></TBODY></TABLE>

After PCR, we <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/Gel_Electrophoresis">migrated the PCR product on TAE 1% agarose gel</A> (100V), to check if the amplification product had the expected size.

<I>From left to right:
100bp+ ladder, vA-2 (two wells), vA-3 (two wells), vA-4 (two wells) amplified at 52°C</I></P><P>
We then made a <A href="">PCR purification protocol</A> using QIAGEN kit:

<TABLE style="width:50%"><TBODY><TR><TD><UL><B><A href="">PCR purification protocol</A></B></UL></TD></TR></TBODY></TABLE></P><P><LI>Add 5 volumes of resuspension buffer to 1 volume of PCR product in an 1.5mL microcentrifuge tube, mix by pipetting up and down
</LI><LI>Transfer in a centrifugation column
</LI><LI>Centrifuge 2 min at 14000 rpm
</LI><LI>Discard flow-through
</LI><LI>Add 700μL of washing solution
</LI><LI>Centrifuge 30 sec at 14000 rpm
</LI><LI>Discard flow-through
</LI><LI>Add 500μL of washing solution
</LI><LI>Centrifuge 30 sec at 14000 rpm
</LI><LI>Discard flow-through
</LI><LI>Centrifuge 30 sec at 14000 rpm
</LI><LI>Discard flow-through
</LI><LI>Put the column in a sterile 1.5 mL Eppendorf tube
</LI><LI>Add 45μL of DNAse/RNAse-free water on the membrane
</LI><LI>Wait 2 minutes
</LI><LI>Centrifuge 2 min at 10000 rpm
</LI><LI>Discard column, DNA is saved in water
    


DNA concentration measured with Nanodrop:
<TABLE style="width:27%" align="center"><TBODY><TR style="background-color:#E6E6E6"><TH>Part Name </TH><TH>[PCR product] (ng/μL)</TH></TR><TR align="center"><TD>vA-2 (tube 1)</TD><TD>126</TD></TR><TR align="center"><TD>vA-2 (tube 2)</TD><TD>97</TD></TR><TR align="center"><TD>vA-3 (tube 1)</TD><TD>129</TD></TR><TR align="center"><TD>vA-3 (tube 2)</TD><TD>76</TD></TR><TR align="center"><TD>vA-4 (tube 1)</TD><TD>202</TD></TR><TR align="center"><TD>vA-4 (tube 2)</TD><TD>108</TD></TR></TBODY></TABLE></LI></P><H1 class="date one">July 28th</H1><H2>Goal</H2>
Retrieve TDH3 promoter from Biobrick BBa_K530008.

<H2>Procedure</H2><B><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR">PCR</A> of TDH3:</B><UL><LI>1 uL of gBlock (1 ng)</LI><LI>2 uL forward primer o15.123</LI><LI>2 uL reverse primer o15.124</LI><LI>50 uL Master Mix 2X</LI><LI>45 uL water</LI></UL><B>Settings PCR:</B><UL><LI>30s at 95°C</LI><LI>35 times:
</LI><LI>30s at 95°C</LI><LI>30s at 55°C</LI><LI>1m at 72°C</LI></UL><LI>10m at 72°C</LI><LI>the tubes were then kept at 10°C</LI><H2>Results</H2>
Nanodrop : 57,9 ng/uL

<H1 class="date two">August 3rd</H1>
We made the following <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR">PCR</A>:
<B>vA-1.1 + o15.142 + o15.141</B><B>vA-1.2 + o15.119 + o15.122</B><B>HO-Poly-KanMX4-HO plasmid + o15.135 + o15.143</B>The elongation time during the pCR was 1 minute for the two gBlocks, and 4 minutes for the plasmid.


<H1 class="date two">August 4th</H1><B>Results: </B>We <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/Gel_Electrophoresis"> migrated all the PCR products of our gBlocks on a gel</A>.:
The bands for gBlocks vA-1.1, 1.2, 2 and 4 are at the expected position. The band for vA-1.1 is not very bright but still visible. There seems to be some unspecific binding for gBlock vA-3 as we can see two bands, one at the expected size and one smaller. We still decided to go on and try our Gibson Assembly, assuming the smaller DNA sequence wouldn't be that much of a problem as long as the right PCR product of gBlock vA-3 was present too.

DNA concentration was measured with a Nanodrop:
<TABLE style="width:25%" align="center"><TBODY><TR style="background-color:#E6E6E6"><TH>Part Name</TH><TH>[PCR product] (ng/μL)</TH></TR><TR align="center"><TD>vA-1.1</TD><TD>84</TD></TR><TR align="center"><TD>vA-1.2</TD><TD>22</TD></TR><TR align="center"><TD>HO-Poly-KanMX4-HO</TD><TD>25</TD></TR></TBODY></TABLE><H1 class="date two">August 6th</H1><B>Gibson assembly</B>

The goal is to assemble all the parts together to get the plasmid with the 3 genes that are needed to produce beta carotene in <I>Saccharomyces cerevisiae</I>.
Gibson was performed on : 
<UL><LI>the PCR product of HO-Poly-KanMX4-HO, a plasmid from Addgene </LI><LI>PCR product of the gblock vA-1.1</LI><LI>PCR product of the gblock vA-1.2</LI><LI>PCR product of the gblock vA-2</LI><LI>PCR product of the gblock vA-3</LI><LI>PCR product of the gblock vA-4</LI></UL>5X ISO Buffer was prepared with the following recipe:
3 ml 1M Tris-HCl pH 7.5
+ 150	μl	2 M MgCl2
+ 240	μl 	100 mM dNTP mix (25 mM each: dGTP, dCTP, dATP, dTTP)
+ 300	μl	1 M DTT
+ 1.5	g	PEG-8000
+ 300	μl	100 mM NAD
  dH20 to 6 ml

Prepare 1.2 ml of Gibson assembly master mix as follows:

320	μl	5X ISO Buffer
	+	0.64    μl	10 U/μl T5 exonuclease* 
	+ 20	μl 	2 U/μl Phusion polymerase
	+ 160	μl	40 U/μl Taq ligase
	+ 	   _	dH20 to 1.2	ml
We took 15 ul of the mix for our Gibson, and stored the rest at -20 C in 15 μl aliquots.

We calculated the required amount of each insert to put on the Gibson mix, and made a separate tube called “gBlock mix” containing the required concentration of each gBlock in 100 uL.

<TABLE cellspacing="2px" cellpadding="2px;" rules="all" style="border:solid 1px black;"><THEAD><TR style="background-color:#F2F2F2"><TH>Component (size)</TH><TH>[PCR product]</TH><TH>Quantity required for Gibson</TH><TH>Volume put in “gBlock mix”</TH></TR></THEAD><TBODY><TR><TD>vA-1.1 (701 bp)</TD><TD>84 ng/uL</TD><TD>13 ng</TD><TD>15.5 uL</TD></TR><TR><TD>vA-1.2 (654 bp)</TD><TD>120 ng/uL</TD><TD>13 ng</TD><TD>10.8 uL</TD></TR><TR><TD>vA-2 (1560 bp)</TD><TD>126 ng/uL</TD><TD>25 ng</TD><TD>19.8 uL</TD></TR><TR><TD>vA-3 (1530 bp)</TD><TD>129 ng/uL</TD><TD>25 ng</TD><TD>19.4 uL</TD></TR><TR><TD>vA-4 (1584 bp)</TD><TD>108 ng/uL</TD><TD>25 ng</TD><TD>23.1 uL</TD></TR><TR><TD>plasmid HO-Poly-KanMX4-HO (6 kb)</TD><TD>103 ng/uL</TD><TD>100 ng</TD><TD>-</TD></TR></TBODY></TABLE>The “gBlock mix” contained 88,6 uL with all the gBlocks, and we added 11.4 uL of water to reach 100 uL.

The final Gibson mix contained:
<UL><LI>15 uL Gibson master mix</LI><LI>1 uL HO-Poly-KanMX4-HO plasmid at 100 ng/uL</LI><LI>1 uL “gBlock mix”</LI><LI>3 uL water</LI></UL>
Total = 20 uL.

The solution was put at 50°C for 60 minutes, then stored at 4°C overnight.

<H1 class="date two">August 7th</H1><B>Transformation</B>We transformed by <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/Electroporation">electroporation</A> an E. Coli (NEB turbo) prepared by Mukit that was kept at -80°C, with our Gibson product, and also with the HO-Poly-KanMX4-HO vector alone to make a control.
The bacteria recovered for 3 hours in SOC media after the electroporation.
100uL of the bacterial cultures were then plated on LB with or without Amp with different concentrations: 10^-1 10^-2 and 10^-3
<H1 class="date two">August 8th</H1><B>Results</B>We didn’t have any colonies after the overnight culture of the cells transformed with the Gibson product. 
The E. Coli transformed with the HO-Poly-KanMX4-HO vector alone did grow very well (a lawn of bacteria on the plates with the 10^-1 and 10^-2 dilutions and more than 100 CFU on the 10^-3 plate).

<B>Interpretation</B> What most likely happen is that the Gibson assembly failed, which could be due to the unspecific binding during the PCR of gBlock vA-3, or to secondary structures at the extremities of our gBlocks which could have made the binding difficult.
Or maybe the Gibson Assembly worked, but we shouldn’t have kept the product overnight before transforming E. Coli with it.

Now the plan is to make all the PCR again, to make fresh electro-competent cells, and to Gibson and transform on the same day.

<H1 class="date two">August 10th</H1><H2>Plan of the <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR">PCR</A></H2><UL><LI>gBlock vA-1.1 with primers o15.141 and o15.144</LI><LI>gBlock vA-1.2 with primers o15.119 and o15.120</LI><LI>gBlock vA-2 with primers o15.127 and o15.128</LI><LI>gBlock vA-3 with primers o15.129 and o15.130</LI><LI>gBlock vA-4 with primers o15.131 and o15.132</LI><LI>HO-Poly-KanMX4-HO with primers o15.135 and o15.143</LI></UL><H2>Results of the PCR</H2><UL><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/Gel_Electrophoresis"> gel electrophoresis.</A></LI><LI>5uL of each sample was mixed with 1uL of loading dye then run on a gel with TAE for 20 min</LI>We can observe the expected bands for most of the gBlocks, except for gBlock vA-3 for which there is no clear band, and gBlock vA-1.1 which has a additional, unexpected band at a lower size, probably due to an unspecific binding of an oligo.</UL><UL><LI>1uL of each sample was nanodroped and the concentrations are shown below:</LI><TABLE style="width:25%" align="center"><TBODY><TR style="background-color:#E6E6E6"><TH>Part Name</TH><TH>[PCR product] (ng/μL)</TH></TR><TR align="center"><TD>vA-1.1</TD><TD>204</TD></TR><TR align="center"><TD>vA-1.2</TD><TD>95</TD></TR><TR align="center"><TD>vA-2</TD><TD>77</TD></TR><TR align="center"><TD>vA-3</TD><TD>77</TD></TR><TR align="center"><TD>vA-4</TD><TD>134</TD></TR><TR align="center"><TD>HO-Poly-KanMX4-HO</TD><TD>61</TD></TR></TBODY></TABLE></UL><H2>Case of the gBlock vA-3</H2>
Since the gBlock vA-3 didn't amplify well on the last PCR, we tried to amplify it again, and this time observed 3 bands, showing unspecific binding of the oligos. We performed a <A href="">PCR purification</A> on this gBlock to take only the DNA in the band at the right size.

The concentration of the PCR product was measured with a Nanodrop:
<TABLE style="width:25%" align="center"><TBODY><TR style="background-color:#E6E6E6"><TH>Part Name </TH><TH>[PCR product] (ng/μL)</TH></TR><TR align="center"><TD>vA-3</TD><TD>87</TD></TR></TBODY></TABLE><H3>New <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR">PCR</A> and gel purification
In case there could be a contamination in our aliquot of gBlock vA-3 at 1 ng/uL, we made a new aliquot from the mother solution (10 uL of gBlock vA-3 at 10 ng/uL + 90uL water).
6 tubes of 100 uL were then made following the <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR">PCR protocol</A> (with elongation time=1:30 min, enzyme=phusion, 35 cycles, 57°C annealing temperature)
The PCR product was then put on a gel to migrate and extracted using the <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/gel-purification">gel purification protocol</A>.
All the product was then purified using the <A href="">PCR purification protocol</A>.
The concentration of the DNA was checked using the Nanodrop.

</H3><H1 class="date two">August 13th</H1>We put some <I>E. Coli</I> NEB-Turbo to grow overnight in 2 mL LB, at 37°C with shaking.

<H1 class="date two">August 14th</H1>We made our <I>E. Coli</I> NEB-Turbo electrocompetent with the following protocol:

<TABLE style="width:50%"><TBODY><TR><TD><UL><B>Preparation of electrocompetent cells protocol</B><P><LI>The night before the transformation, start an overnight culture of cells.
        	</LI><LI>5 ml LB.</LI></P></UL><LI>The day of the transformation, dilute the cells 100X.
        	</LI><LI>100 ml LB.</LI><LI>Grow at 37°C for about 2 hours.</LI><LI>Harvest the cells.
        	</LI><LI>When the cells reach an OD600 of between 0.6 and 0.8.</LI><LI>Split the culture into 2x 50 ml falcon tubes, on ice.</LI><LI>Centrifuge at 4 °C for 10 min at 4000 rpm.</LI><LI>Wash and combine the cells.
        	</LI><LI>Remove the supernatant.</LI><LI>Resuspend the cells in 2x 25 ml of ice cold water.</LI><LI>Combine the volumes in a single 50 ml falcon tube.</LI><LI>Wash the cells 2 more times.
        	</LI><LI>Centrifuge at 4 °C for 10 min at 4000 rpm.</LI><LI>Resuspend in 50 ml of ice cold water.</LI><LI>Repeat.</LI><LI>Wash and concentrate the cells for electroporation.
        	</LI><LI>Centrifuge at 4 °C for 10 min at 4000 rpm.</LI><LI>Resuspend in 1-2 ml of ice cold water.</LI><LI>We will use 200 ul of washed cells per transformation.</LI></TD></TR></TBODY></TABLE><B>Gibson Assembly of the polycistron</B>We attempted to ligate by Gibson Assembly the parts vA-1.1, vA-1.2, vA-2, vA-3 and vA-4 in the HO-Poly-KanMX4-HO plasmid. 
Since we didn't get good PCR products for gBlock vA-3, we decided to take it directly from our mother solution send by IDT, which is at 10 ng/uL. This gBlock already has 30 bp of overlap with vA-2 and vA-4, which may be enough for a Gibson Assembly.
For the other gBlocks and the plasmid, we took their PCR products to have a better concentration and a higher overlap with the neighboring parts (the gBlocks have 30 bp overlap between each other, and the PCR added an additional 30 bp of overlap thanks to the tails of our oligos). So there is 60 bp of overlap between the plasmid and vA-1.1, between vA-1.1 and vA-1.2, between vA-1.2 and vA-2, and between vA-4 and the plasmid.

We diluted the PCR products to have 100 ng of plasmid, and an equimolar concentration of every insert.

<TABLE style="width:80%" align="center"><TBODY><TR style="background-color:#E6E6E6"><TH>Part</TH><TH>Amplified with</TH><TH>Concentration (ng/μL)</TH><TH>Required amount for Gibson (ng)</TH><TH>Dilution</TH><TH>Volume taken for Gibson (uL)</TH></TR><TR align="center"><TD>HO-Poly-KanMX4-HO</TD><TD>o15.135 o15.143</TD><TD>61</TD><TD>100</TD><TD>none</TD><TD>2</TD></TR><TR align="center"><TD>vA-1.1</TD><TD>o15.141 o15.142</TD><TD>204</TD><TD>13</TD><TD>1/15</TD><TD>1</TD></TR><TR align="center"><TD>vA-1.2</TD><TD>o15.119 o15.122</TD><TD>95</TD><TD>13</TD><TD>1/7</TD><TD>1</TD></TR><TR align="center"><TD>vA-2</TD><TD>o15.056 o15.057</TD><TD>77</TD><TD>25</TD><TD>1/3</TD><TD>1</TD></TR><TR align="center"><TD>vA-3</TD><TD>none</TD><TD>10 from mother solution</TD><TD>25</TD><TD>none</TD><TD>2,5</TD></TR><TR align="center"><TD>vA-4</TD><TD>o15.060 o15.061</TD><TD>134</TD><TD>25</TD><TD>1/5</TD><TD>2</TD></TR></TBODY></TABLE>

All those parts were put together in a tube, and amounted to a total of 8,5 uL. We then poured those 8,5 uL in the tube containing 15 uL of Gibson mix prepared by Ihab.
The mix was put at 50°C for 60 min.
We then made a dialyse of the Gibson product for 15 minutes.

<B><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/Electroporation">Electroporation</A></B>We transformed both our fresh electro-competent NEB-Turbo cells prepared in the morning, and some electro-competent NEB-Turbo cells that had been prepared by Mukit and stored at -80°C, with our dialyzed Gibson product. We used two different kind of cells to see if transformation worked better on fresh cells or not.

We transformed the following with <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/Electroporation">electroporation</A>:
<OL><LI>Fresh NEB-T cells + HO-Poly-KanMX4-HO plasmid linearized by PCR</LI><LI>Fresh NEB-T cells + Gibson product</LI><LI>Fresh NEB-T cells + circular HO-Poly-KanMX4-HO plasmid</LI><LI>NEB-T cells from freezer + Gibson product</LI><LI>NEB-T cells from freezer + Gibson product</LI></OL>

The cells were then quickly resuspended in 1 mL LB and put at 37°C to recover, then plated in selective media (LB + Ampicillin).

<H1 class="date two">August 15th</H1><B>Results of the transformation with the Gibson product:</B>Nothing grew on the plates with cells that had been transformed with the Gibson product.
However we observed colonies for both the cells that had been transformed with the circular plasmid with no insert, and the linearized plasmid with no insert. While it was expected to see colonies for the former since the plasmid contains the Ampicillin resistance gene, it was not expected to see colonies of cells transformed with the linearized plasmid.

<B>Interpretation</B>The presence of colonies in the plates of cells transformed by the linearized plasmid may be due to background, i.e. to the presence in our sample of circular plasmid that had not been linearized by the PCR.
The same background should then has been observed also in the plates of cells transformed with the Gibson product. But the concentration of plasmid we transformed the cells with was much higher in our controls than with our Gibson products. We should have transformed the cells with the same concentration of plasmid to have a real control.

We still plated the rest of the transformed cells, that had been kept overnight in a liquid culture at 37°C, to see if they would grow.

<H1 class="date two">August 17th</H1><B>Results of the transformation with the Gibson product:</B><DIV class="column-left">Both the plates inoculated on August 14th and August 15th with cells transformed with our Gibson product had colonies, even though nothing had grew on the plates inoculated on August 14th after an overnight.
However, the colonies weren't homogeneous on the plates.</DIV><B>Interpretation</B>Rather than multiple colonies of cells successfully transformed with our plasmid containing the insert, it looks like a maximum of 10 colonies on each plate were actually resistant to Ampicillin thanks to the <I>bla</I> gene, that they secreted beta-lactamase which hydrolyses and inactivates Ampicillin, and that this beta-lactamase diffusing into the medium allowed satellite colonies non possessing the plasmid to grow around the resistant colonies. This phenomenon is well-known when using Ampicillin selection.

There was still a possibility that the colony in the center of the cloud of satellites did possess the plasmid with our inserts, so we took 12 samples of those colonies from the plates - they had a bright white color, which distinguished them from the more grayish satellites - and put them into liquid LB + Ampicillin at 37°C overnight.

<B>Ligation by <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR">PCR</A></B>Since the colonies we got on the plates weren't quite the ones expected - we expected colonies homogeneously spread on the plates - and looked like they could be contaminations, we tried to ligate our gBlocks this time with <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR">PCR</A>. To isolate where a problem might be in the ligation, we cloned the parts two by two, three by three and all five together. However none of those worked: we ran a gel after the different PCR, and observed no band.

<H1 class="date two">August 18th</H1><B>Analysis of the colonies</B>After an overnight culture in liquid medium (LB + Ampicillin), the above colonies were <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/Miniprep_protocol_using_a_QIAGEN_kit">miniprepped</A>. We then <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR">cloned</A> the plasmid and performed an analytical digestion on it, but it was hard to determine for sure from the gel whether the ligation/transformation had failed or not.

<B>Gibson Assembly gBlocks two by two</B>Since there was a high chance our Gibson with all our gBlocks in the HO-Poly-KanMX4-HO plasmid didn't work, we tried to identify where the problem was by assembling only two parts at a time with Gibson Assembly, and see which parts wouldn't assemble. However, none of them worked.

<H1 class="date two">August 19th</H1><B>Sequencing</B>We sent the plasmid we got from mini-prepping the colonies resulting from our Gibson Assembly, to have final answer on whether or not they were the expected plasmid, or a contamination.
We followed GATC's instructions and sent the following:
<UL><LI>5 uL purified plasmid at 90 ng/uL</LI><LI>5 uL primer at 5 uM</LI></UL>
with primers o15.117, o15.118, o15.119 and o15.120.

<H1 class="date two">August 21st</H1><B>Sequencing results</B>GATC sequencing results arrived: in most of the tubes we had sent, 0 bp had been found with the primers we had sent. For only one tube we got a result of 150 bp, but the sequence had nothing to do with the sequence we expected.
It means our Gibson Assembly had failed, and the colonies that had grew and allowed satellites to grow were contaminations resistant to Ampicillin.

<B>The HMG-CoA reductase gene</B>We finally received our HMG gBlock, with the CDS of the HMG-CoA reductase gene from the red yeast <I>S. aureus</I>, codon-optimized for <I>S. cerevisiae</I>, and synthesized by IDT. More information about this gene can be found <A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/VitaminA">here</A>.
The gBlock was centrifuged 5 seconds at 3000 rpm, then resuspended in 100 uL of water to reach a concentration of 10 ng/uL.
The tube was mixed by briefly vortexing it, then put 20 minutes at 37°C accordingly to IDT's instructions. We then vortexed it again and centrifuged it, and made an aliquot at 1 ng/uL.

<B>PCR</B>We made a <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR">PCR</A> of the HMG gBlock. Two tubes were prepared:
<UL><LI>1 ng gBlock HMG, with primers o15.137 and o15.138</LI><LI>10 ng gBlock HMG, with primers o15.137 and o15.138</LI></UL>
We wanted to check whether a <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR">PCR</A> had a better yield with more initial DNA template than 1 ng, as suggested by our advisors, which is why we tried with both 1ng and 10ng of initial concentration of template DNA.
The primers used on this <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR">PCR</A> had tails containing the restriction sites of XbaI (on the 5' primer), and XhoI (on the 3' primer).
The elongation time was 2 minutes.

<B>Results of the PCR:</B>We purified the PCR products and measured their concentration with a Nanodrop:
<TABLE style="width:25%" align="center"><TBODY><TR style="background-color:#E6E6E6"><TH>Part Name (initial amount)</TH><TH>[PCR product] (ng/μL)</TH></TR><TR align="center"><TD>HMG (1 ng)</TD><TD>4</TD></TR><TR align="center"><TD>HMG (10 ng)</TD><TD>300</TD></TR></TBODY></TABLE>

For some reason the tube with a low concentration of HMG gBlock was hardly amplified at all, even though we had previously often managed to have a good amplification on 1 ng of DNA. Since we had a good yield in the second tube, we kept it for the following experiments.

<B>p406ADH1 plasmid</B>Our HMG gBlock only contains the CDS of the gene. To add a strong yeast promoter (ADH1) and a terminator (CYC1) to this gene, we planned to insert it in the p406ADH1 vector which we got from AddGene (<A href="https://www.addgene.org/15974/">Plasmid #15974</A>). The plasmid was sent to us in a DH5alpha E. Coli.
We streaked this E. Coli containing the p406ADH1 plasmid on a plate with LB and Ampicillin and let it grow overnight at 37°C.


<H1 class="date two">August 22nd</H1><B>p406ADH1 plasmid</B>We took 2 isolated colonies of the E. Coli containing the p406ADH1 plasmid from our plate, and put them in a liquid culture of LB + Ampicillin at 37°C.

<H1 class="date two">August 23rd</H1><B>p406ADH1 plasmid</B>For some unknown reason, only one of the two liquid cultures grew overnight.
We <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/Miniprep_protocol_using_a_QIAGEN_kit">miniprepped</A> the bacteria that did grow, and measured the final concentration of p406ADH1 vector with a Nanodrop:
<TABLE style="width:25%" align="center"><TBODY><TR style="background-color:#E6E6E6"><TH>Plasmid Name</TH><TH>[Miniprep product] (ng/uL)</TH></TR><TR align="center"><TD>p406ADH1</TD><TD>200</TD></TR></TBODY></TABLE><H1 class="date two">August 24th</H1><B>Digestion p406ADH1 plasmid and HMG gBlock</B>We digested p406ADH1 and the HMG gBlock with the restriction enzymes XbaI and XhoI. Those enzymes should cut the plasmid between the promoter ADH1 and the terminator Cyc1, and should cut the gBlock on both its extremities since we added the two restriction sites by PCR.

<DIV class="column-left"><U>Digestion of p406ADH1:</U>Preparation of a 120 uL mix with:
<UL><LI>15 uL plasmid (to have 3 ug)</LI><LI>4 uL XbaI</LI><LI>4 uL XhoI</LI><LI>4 uL FastAP</LI><LI>12 uL green Buffer FD</LI><LI>81 uL water (to reach 120 uL)</LI></UL></DIV><DIV class="column-right"><U>Digestion of HMG gBlock:</U>Preparation of a 80 uL mix with:
<UL><LI>10 uL gBlock (to have 3 ug)</LI><LI>4 uL XbaI</LI><LI>4 uL XhoI</LI><LI>8 uL Buffer FD</LI><LI>54 uL water (to reach 80 uL)</LI></UL></DIV>Both mix were put at 37°C for 45 minutes.
We then PCR purified the digested HMG gBlock, and measured its concentration with a Nanodrop:
<TABLE style="width:25%" align="center"><TBODY><TR style="background-color:#E6E6E6"><TH>Part Name</TH><TH>[PCR product] (ng/uL)</TH></TR><TR align="center"><TD>digested HMG</TD><TD>50</TD></TR></TBODY></TABLE>We performed a<A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/Gel_Electrophoresis"> gel electrophoresis.</A>on the digested p406ADH1 plasmid, and obtained the following bands:
<A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR">PCR</A> gBlock 3 with primers o15.129 and o15.130
3 tubes with concentrations of gBlock of 1ng 1ng and 10ng

<H1 class="date two">August 25th</H1>
After <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/Gel_Electrophoresis"> gel electrophoresis.</A> the DNA is <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/gel-purification">Gel Purified</A>.
The Nanodrop measurement then revealed that everything has been lost during the <A href="">PCR purification</A>.

Another <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR">PCR</A> is launched with all the gBlocks and the right tails.
<H2>plan of the <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR">PCR</A></H2><UL><LI>gBlock vA-1.1 with primers o15.141 and o15.142</LI><LI>gBlock vA-1.2 with primers o15.119 and o15.120</LI><LI>gBlock vA-2 with primers o15.127 and o15.128</LI><LI>gBlock vA-3 with primers o15.129 and o15.130</LI><LI>gBlock vA-4 with primers o15.131 and o15.132</LI><LI>HO-Poly-KanMX4-HO with primers o15.135 and o15.143</LI></UL>
We also tried a <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/Gibson_Assembly">Gibson Assembly</A> again with the amplified gBlocks vA-1.1, 1.2, 2, 4 and the HO-Poly-KanMX4-HO vector ... which failed miserably.


on the right: 1kb ladder
on the left: Gibson product
no band at 1500 bp is visible (the exposition time is 5s)<H1 class="date two">August 26th</H1>
result of the PCR of the gBlocks (from left to right) vA-1.1, 1.2, 2, 3, 4, HO-Poly-KanMX4-HO and the last column is the ladder 1kb

The PCR for the gBlock 1.2, 2 and 4 worked well.
We are doing again the <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR">PCR</A> for the gBlock 1.1, 3 and HO
The <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR">PCR</A> with the gBlock 3 is done with 10ng of DNA
from left to right: gBlock 1.1, 3 (1ng), 3 (10ng), HO, 1kb ladder
<DIV class="column-left" align="right">The results are showing traces of unspecific binding for 1.1 and 3. The HO-Poly-KanMX4-HO band is alsmost invisible, predicting a low DNA concentration.</DIV><H1 class="date two">August 27th</H1><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/Miniprep_protocol_using_a_QIAGEN_kit">Miniprep</A> of the overnight cultures of the small and big colonies obtained after transformation.

Following the miniprep an analytical digest is performed with Pst1 to linearise the plasmid and then by Nde1 which is supposed to cut the plasmid in 2.

During the digestion the miniprep DNA concentration is recorded with a Nanodrop
<TABLE style="width:100%"><TBODY><TR><TD><B>colony which was grown for the miniprep</B></TD><TD>S1</TD><TD>S2</TD><TD>S3</TD><TD>S4</TD><TD>B1</TD><TD>B2</TD><TD>B3</TD><TD>B4</TD></TR><TR><TD><B>Nanodrop concentration(ng.ul-1)</B></TD><TD>84</TD><TD>176</TD><TD>182</TD><TD>183</TD><TD>128</TD><TD>216</TD><TD>170</TD><TD>235</TD></TR></TBODY></TABLE>
Then the results are tested with a <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/Gel_Electrophoresis"> gel electrophoresis</A>.
GEL 1 <I>from left to right 1kb ladder, small colony 1,2,3,4, big colony 1,2,3,4, small colony digested by Pst1 1,2,3,4, big colony digested by PST1 1,2,3,4</I>GEL 2 <I>from left to right 1kb ladder, small colony digested by Nde1 1,2,3,4, big colony digested by Nde1 1,2,3,4</I><H1 class="date two">August 28th</H1><H2><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR">PCR</A> on the minipreped plasmid.</H2><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR">PCR</A> with the oligos o.15.193 and o.15.194 and 10 ng of plasmid DNA.
The PCR product are then revealed with a<A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/Gel_Electrophoresis"> gel electrophoresis.</A><I>from left to right: 1kb ladder thermoscientific, <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR">PCR</A> of the plasmid on 4 different colonies(s2, b2, s4, b4)</I> The brightest band (at 2kb)is probably the band that we were expecting after PCR (the DNA piece that i supposed is amplified is composed of the promoter, the terminator and the insert).

<H1 class="date two">August 29th</H1><B><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR">PCR</A> on p406ADH1 with and without HMG</B>We made a gradient <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR">PCR</A> on the plasmid p406ADH1 that was ligated with the HMG gBlock, as well as a PCR on the p406ADH1 without the insert. The primers o.15.193 and o.15.194 are located around the insert.
The expected bands should be at 1.3 kb for p406ADH1 + HMG, and at 0 kb for p406ADH1 without insert.
<H1 class="date two">August 31th</H1><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR">PCR</A> of the HMG gBlock with 1DMSO 2noDMSO
<H1 class="date three">September 1st</H1><B>Analytic digestion p406ADH1</B>The previous results made us suspicious about the actual length of the p406ADH1 plasmid we received from AddGene. So we performed an analytic digestion on it to see if we had bands at the expected size.
<DIV class="column-left" align="left">We <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/Digestion">digested</A> the plasmid with:
<UL><LI>PstI: expected bands at 2kb and 4 kb</LI><LI>XbaI: expected band at 6kb</LI><LI>XbaI + SacI: expected bands at 1.5kb and 4.5 kb</LI><LI>XbaI + XhoI: expected band at 6kb</LI><LI>BpiI: expected band at 2kb and 4kb</LI></UL>
The Gene Ruler is 1kb from ThermoFischer.
The mix was put 30 min at 37°C for the digestion, then 5 min at 83°C to inactivate the enzymes.

<B>Results</B>On this gel we can observe:
<UL><LI>PstI: no bands</LI><LI>XbaI: bands at 8kb and &gt;10 kb</LI><LI>XbaI + SacI: bands at 8kb, 6kb, and maybe 1.5 kb</LI><LI>XbaI + XhoI: band at 8 kb</LI><LI>BpiI: band at 5 kb and maybe 8 kb</LI></UL></DIV><H1 class="date three">September 2nd</H1>Because of the inconsistency of the results regarding the nature of the plasmid p406ADH1 it was sent to sequencing with 2 different oligos.
<B><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR">PCR</A> HO-KanMX4 plasmid and gBlocks vA-1.1, 2, and 3, with longer oligos</B>Since we have trouble with PCR, with often no bands or strong unspecific bands when we try to amplify the HO plasmid and the gBlocks vA-1.1, 2 and 3, we designed new oligos that were longer. It should at least reduce the unspecific binding.
We performed the following <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR">PCR</A>:
<UL><LI>vA-1.1 + o15.227 + o12.228</LI><LI>vA-2 + o15.230 + o12.231</LI><LI>vA-3 + o15.232 + o12.233</LI><LI>HO-Poly-KanMX4-HO + o15.234 + o12.235</LI></UL>
The annealing temperature was 57°C.

<H1 class="date three">September 3rd</H1>The results of the sequencing are showing that the plasmid that we are minipreping from the strain from addgene is indeed p406ADH1.<B>Results PCR with longer oligos</B><DIV class="column-left" align="left"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/Gel_Electrophoresis">We migrated</A> our PCR products that were amplified with longer oligos on a gel (see on the right).
We can only see a band (not very bright) for the gBlock vA-3. For once, there was no unspecific binding and we do not observe other bands at smaller size for this gBlock!
However, after we PCR-purified it, all the DNA was lost as the Nanodrop said the concentration was 0.
There was no band for the plasmid nor the gBlocks vA-1.1 and vA-2.

We launched those same <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR">PCR</A> again, but this time we added 3% of DMSO in each tube, and lowered the annealing temperature to 51°C.
</DIV><B>Digestion, <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/Ligation">ligation</A> and transformation of the HMG gene in DH5alpha et NEBT</B>The <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/Digestion">digestion</A> was performed on 300ng of HMG (PCR product) and on 100ng of plasmid P406ADH1 (miniprep product) by XhoI and Xba1.
25uL of both products were <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR_purification">PCR purified</A> and 25uL were kept unpurified.
2<A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/Ligation">ligations</A> were performed.
The first one with the HMG and vector p406ADH1 digested and unpurified and the other one with the digested and purified HMG and p406ADH1.
The ligation was left for 1 hour at room temperature (18°C)
Dialysis was performed on all the ligation product for 15 min.
5uL of ligation products were then transformed by <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/Electroporation">electroporation</A> with 100uL of NEBT and DH5alpha.
Controls: NEBTurbo cells and Dh5alpha cells were also transformed with : undigested p406ADH1, p406ADH1 digested with Xba1 and xho1, nothing.
Cells were put to recover for 2 hours at 37°C.
The cells were then plated on LB+ampicillin. 

<H1 class="date three">September 4th</H1>resultats du gel que arthur va envoyer.
gBlock VA1.1 and VA3 are showing no sign of amplification.
<A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR">PCR</A> again with the gBlocks VA1 and VA3 with a gradient PCR (50°C-55°C-60°C).
<B>Here is the result of the transformation of september 4th.</B>The negative controls are free of colonies and the we have a lot of colonies in the positive controls.
We have a few colonies in the plates with bacteria transformed with the ligation product.
6 colonies are put to grow in liquid culture over night in LB + ampicillin.
<H1 class="date three">September 5th</H1>We performed a <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/Miniprep_protocol_using_a_QIAGEN_kit">miniprep</A> on 6 overnight cultures of the transformants, which we called C1, C2, C3, C4, C5 and C6.
Analytical digestion is then performed on the miniprep product with XbaI/XhoI (2 bands expected at 1.3 kb and 6 kb), and XbaI alone (1 band expected at 7.3 kb).
Here are the results of the <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/Gel_Electrophoresis"> gel electrophoresis</A>:


We can observe very bright bands at 6 kb on all the digested minipreps, which is the size of the p406ADH1 plasmid without the insert. So the cells only grew because of the background (non digested plasmid), but it seems none of them has our gene. There are some bands higher than 6 kb on some wells, but they are probably due to circular plasmid that were not cut during the analytical digestion.


<H1 class="date three">September 6th</H1><B>The truncated gBlock</B>We finally noticed that one of the gBlock that we received from IDT (gBlock vA-1.2) doesn't have the sequence we ordered... It is truncated, and 16 bp are missing on the 3' end. We found out by comparing the sequence we had ordered and that was also written in a confirmation email from IDT, to the Fasta sequence available on the website. IDT admitted to their mistake.
We strongly suspect that these missing 16 bp are the reason (or at least one of the reasons) why several of our experiments failed. Indeed without those 16 bp, the 3' primer of this gBlock had little chance to bind (it had only 12 bp in common with the actual gBlock we received, with only 3 G/C). Also our 3' primer actually contained a tail that had 30 bp in common with the gBlock vA-2, for the Gibson Assembly. Without this, the Gibson was sure to fail.
We did observed bands at the right size when we amplified this gBlock during the summer though, but we suspect that the gBlock was always amplified by the 5' primer only.
So we ordered a new 3' primer, with a tail that put back the missing 16 bp.

<B>Biobrick the HMG gene</B>In order to put the coding sequence of the HMG gene in the iGEM plasmid (pSB1C3), we amplified the HMG-gBlock with oligos that had the XbaI and SpeI restriction sites on their tails. 
<A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR">PCR</A>:
<UL><LI>HMG-gBlock + o15.137 + o15.192</LI></UL><H1 class="date three">September 7th</H1><B>Attempt n°785464153 for the digestion, ligation and transformation of the HMG insert in the p406ADH1 vector.</B>A <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR"> PCR</A> is performed on HMG-gBlock with the oligos o15.138 and o15.137.
A <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/PCR-Purification"> PCR-Purification</A>is performed on the PCR product.
The <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/Digestion"> digestion</A> of HMG and of the vector are performed with Xba1 and Xho1 (with no FAST AP or FAST AP buffer for the insert).
The digestion product are PCR purified to remove the small pieces of DNA (&lt;60).
The products are then following a <A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/Ligation"> Ligation</A> with 100ng of plasmid and 75ng of the insert(the molar ratio is plasmid 1:insert 3).
<B><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols/Electroporation"> Electroporation</A>.</B>5uL were added to a cuvette with 100uL of cells and the electroporation made an arc so i reduced the volume of DNA to 2uL and i had a poor time for the electroporation(4.3). I then decided to do a dialysis for 15 min before the 2 last transformations.
 After dialysis i had some good time values (5.2 and 5.3).
The controls are : cells transformed with p406ADH1 digested with xba1 and xho1. Cells transformeds with he circular p406ADH1.<H1 class="date three">September 8th</H1>after an overnight culture the cells transformed with the ligation product HMG/p406ADH1 have grown. Both negative and positive controls are ok. We are doing a colony PCR on 24 of the colonies that grew.

We are also doing the digestion again on the plasmid to be ready to redo the ligation transformation. After digestion of the plasmid it is run on a gel with the undigested plasmid. It seems that the digestion went well:
A culture of the E.coli that contains p406ADH1 in LB+ampicillin.

<H3>PCR on gBlocks vA-1.2, and 1.1-1.2 together</H3>
We found out that the reason many PCR and assembly didn't work is because the gBlock vA-1.2 that we've had synthesized by IDT is actually truncated. 16 bp were missing on the 3' end, and thus our 3' primers never bound to our gBlock. We had still observed clear bands when we had amplified it before, because it was (most probably) amplified by the 5' primer alone.
So we ordered a new primer that put back the missing 16 bp, and launched several PCR with it: one on the gBlock vA-1.2 alone, and one with vA-1.1 and 1.2 together (with the 5' of 1.1 and the 3' of 1.2).

<B>Results</B><H1 class="date three">September 12th</H1>
A miniprep is performed on all the colonies that were showing a band at 1300 bp on the colony PCR of the 7 of september to extract any plasmid that could possibly got the insert even if the other PCR and digestion are showing no signs of it.
A PCR is then performed:
<TABLE cellpadding="0" cellspacing="0" class="c6"><TBODY><TR class="c3"><TD class="c1" colspan="1" rowspan="1"><P class="c2"><SPAN class="c0">primers 137 &amp; 138</SPAN></P></TD><TD class="c1" colspan="1" rowspan="1"><P class="c2"><SPAN class="c0">miniprep product</SPAN></P></TD><TD class="c1" colspan="1" rowspan="1"><P class="c2"><SPAN class="c0">gBlock HMG</SPAN></P></TD></TR><TR class="c3"><TD class="c1" colspan="1" rowspan="1"><P class="c2"><SPAN class="c0">primers 253 &amp; 254</SPAN></P></TD><TD class="c1" colspan="1" rowspan="1"><P class="c2"><SPAN class="c0">miniprep product</SPAN></P></TD><TD class="c1" colspan="1" rowspan="1"><P class="c2"><SPAN class="c0">miniprep p406ADH1</SPAN></P></TD></TR></TBODY></TABLE>
A gibson is also performed on the 5 parts with the intent to PCR the eventual results.
<H1 class="date three">September 15th</H1>SK1 and beta carotene producing yeasts are put to grow in YPD and in minimal media.

<H1 class="date one">Oligos</H1><TABLE width="100%"><TBODY><TR><TD width="8%">o15.056</TD><TD width="90%">TCACAACAAGAATTTTGCAGAGGACCTAACCGAAGGAAAGTTTTCCTTCCCAACAATC</TD></TR><TR><TD width="8%">o15.057</TD><TD width="90%">GGACTTGATAGAATTTTGTCTGGTTGACTCGGATGCAATGGAGGACCGAACAAT</TD></TR><TR><TD width="8%">o15.058</TD><TD width="90%">AACGTTATTGTTCGGTCCTCCATTGC</TD></TR><TR><TD width="8%">o15.059</TD><TD width="90%">GTCCAGATTGACTCGAATGGGTGTAATGCCAGTATTTGACCAATGAATTGTAGCCTCAAG</TD></TR><TR><TD width="8%">o15.060</TD><TD width="90%">CTTGAGGCTACAATTCATTGGTCAAATAC</TD></TR><TR><TD width="8%">o15.061</TD><TD width="90%"> TGTACGGGCGACAGTCACATCATGCCCCTGCAGGGTACCGCGAATTCCCC</TD></TR><TR><TD width="8%">o15.117</TD><TD width="90%"> CGTCCAAGACATACTGCGTTTACG</TD></TR><TR><TD width="8%">o15.118</TD><TD width="90%"> GTGTGCAACATTCCCACTACATTTTGAATG</TD></TR><TR><TD width="8%">o15.119</TD><TD width="90%"> CATTCAAAATGTAGTGGGAATGTTGCAC</TD></TR><TR><TD width="8%">o15.120</TD><TD width="90%">TGGATTGTTGGGAAGGAAAACTTTCCTT</TD></TR><TR><TD width="8%">o15.122</TD><TD width="90%"> AGCCTGGAGGAAGGGTTAGCATGGATGGAATGGATTGTTGGGAAGGAAAACTTTCCTT</TD></TR><TR><TD width="8%">o15.123</TD><TD width="90%"> GCCCAGAATACCCTCCTTGACAGACAGTTTATTCCTGGCATCCACTAAATATAATG</TD></TR><TR><TD width="8%">o15.124</TD><TD width="90%">TATACTGCAGTTTGTTTGTTTATGTGTGTTTATTCGAAACTAAGTTC</TD></TR><TR><TD width="8%">o15.127</TD><TD width="90%"> TCACAACAAGAATTTTGCAGAGGACCTAACCGAAGGAAAGTTTTCCTTCCCAACAATC </TD></TR><TR><TD width="8%">o15.128</TD><TD width="90%"> GGACTTGATAGAATTTTGTCTGGTTGACTCGGATGCAATGGAGGACCGAACAAT </TD></TR><TR><TD width="8%">o15.129</TD><TD width="90%"> AACGTTATTGTTCGGTCCTCCATTGC </TD></TR><TR><TD width="8%">o15.130</TD><TD width="90%">GTCCAGATTGACTCGAATGGGTGTAATGCCAGTATTTGACCAATGAATTGTAGCCTCAAG</TD></TR><TR><TD width="8%">o15.131</TD><TD width="90%">CTTGAGGCTACAATTCATTGGTCAAATAC</TD></TR><TR><TD width="8%">o15.132</TD><TD width="90%"> TGTACGGGCGACAGTCACATCATGCCCCTGCAGGGTACCGCGAATTCCCC </TD></TR><TR><TD width="8%">o15.135</TD><TD width="90%"> TCGCTATACTGGGGAATTCGCGGTACCCTGCAGGGGCATGATGTGACTGTCG </TD></TR><TR><TD width="8%">o15.137</TD><TD width="90%"> TATATCTAGACCCGCCGCCACCATGCAAAG </TD></TR><TR><TD width="8%">o15.138</TD><TD width="90%"> TATACTCGAGTCACTGTTGAGACCTCAAATCCTGTAAG </TD></TR><TR><TD width="8%">o15.141</TD><TD width="90%"> TCCTCAACGTCGTCCATCAGTAAGCTCGCTGTGTGCAACATTCCCACTACATTTTGAATG </TD></TR><TR><TD width="8%">o15.142</TD><TD width="90%"> GCCCAGAATACCCTCCTTGACAGCGTCCAAGACATACTGCGTTTACG </TD></TR><TR><TD width="8%">o15.143</TD><TD width="90%"> TCGACACGTAAACGCAGTATGTCTTGGACGCTGTCAAGGAGGGTATTCTGGGC </TD></TR><TR><TD width="8%">o15.144</TD><TD width="90%"> GTTTCGAATAAACACACATAAACAAACAAATCTAGAACTAGTGGATCCATGGATTAC </TD></TR><TR><TD width="8%">o15.192</TD><TD width="90%"> TATAACTAGTTCACTGTTGAGACCTCAAATCCTGTAAG </TD></TR><TR><TD width="8%">o15.193</TD><TD width="90%">TATACTGCAGCCCGCCGCCACCATGCAAAG</TD></TR><TR><TD width="8%">o15.194</TD><TD width="90%">TATACTGCAGTCACTGTTGAGACCTCAAATCCTGTAAG</TD></TR><TR><TD width="8%">o15.227</TD><TD width="90%"> CCGGGTTAATTAAGGCGCGCCAGATCTGTTCGTCCAAGACATACTGCGTTTACGTGTCG </TD></TR><TR><TD width="8%">o15.228</TD><TD width="90%"> AACGTCGTCCATCAGTAAGCTCGCTGTGTGCAACATTCCCACTACATTTTGAATGACTTC </TD></TR><TR><TD width="8%">o15.230</TD><TD width="90%"> CGAAGGAAAGTTTTCCTTCCCAACAATCCATTC </TD></TR><TR><TD width="8%">o15.231</TD><TD width="90%"> GGACTTGATAGAATTTTGTCTGGTTGACTCGGATGCAATGGAGGACCGAACAATAACGT </TD></TR><TR><TD width="8%">o15.232</TD><TD width="90%"> CATCCGAGTCAACCAGACAAAATTCTATCAAGT </TD></TR><TR><TD width="8%">o15.233</TD><TD width="90%"> GATTGACTCGAATGGGTGTAATGCCAGTATTTGACCAATGAATTGTAGCCTCAAGAATGC </TD></TR><TR><TD width="8%">o15.234</TD><TD width="90%">TCGCTATACTGGGGAATTCGCGGTACCCTGCAGGGGCATGATGTGACTGTCGCCC</TD></TR><TR><TD width="8%">o15.235</TD><TD width="90%">TCGACACGTAAACGCAGTATGTCTTGGACGAACAGATCTGGCGCGCCTTAATTAACCC</TD></TR><TR><TD width="8%">o15.253</TD><TD width="90%"> CGTATACTGCAGGCGCAATTAACCCTCACTAAAGGG </TD></TR><TR><TD width="8%">o15.254</TD><TD width="90%"> GTCACTCTGCAGCTCACTATAGGGCGAATTGGGTAC </TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></BODY></HTML>