<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Project_Continuity skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Paris Bettencourt/Project/Continuity</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="menuContainer"><DIV id="smallMenuTitle"><H1>Ferment It Yourself</H1><H3>iGEM Paris-Bettencourt 2O15</H3></DIV><UL><LI class="left"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Background" title="Background">Background</A></LI><LI class="left"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Design" title="Design">Design</A></LI><LI class="left"><BUTTON class="menuButton">Project</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Micronutrients">Micronutrients</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Continuity">Continuity</A></LI><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Manufacturing">Manufacturing</A></LI></DIV></UL><LI class="left"><BUTTON class="menuButton">Achievements</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Parts">Parts</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Medals">Medal Fulfillments</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Practices</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/SynBio_Challenge">SynBio Challenge</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/DILambda">DIλ</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Acceptance">Acceptance</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/Events">Events</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/Rhizi">Rhizi</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Collaborations">Collaborations</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Safety</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Safety">Safety Overview</A></LI><LI><A href="https://2015.igem.org/Safety/Final_Safety_Form?team_id=1678">Safety Form</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/ReviewBoard">Review Board</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Notebook</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Notebook">Daily notebook</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols">Protocols</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Bibliography">Bibliography</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Team</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Team">Members</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Acknowledgments">Acknowledgments</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Press">Press</A></LI><LI><A href="https://igem.org/Team.cgi?id=1678">Official Team Profile</A></LI></DIV></DIV><DIV id="continuity"><DIV class="contentContainer"><DIV id="banner"><H3>Continuity
</H3></DIV></DIV><DIV class="textWrapper"><DIV class="textContent {{{page_option}}}" id="caffeine"><H1>Purpose</H1><DIV class="column-left"><P>In order to gain the trust of the public, we believe the technology should belong to everyone. In a manner similar to the open-source software industry, people should be able to improve our project or create their own versions of it. This idea of openness is very common among the community of synthetic biologists, but a lot of pitfalls have to be overcome to make it a sustainable reality.</P><P>A parallel could be drawn with electronics in the 1960's, when computer programming was extremely low-level and belonged to the realm of academia. Since then, it has reached a far wider population thanks to the creation of frameworks allowing for abstraction of the most technical parts. How could the same principles be applied to synthetic biology, in the context of metabolic engineering and vitamins production?</P><P>Even though a lot of lab strains designed for easier modification have been designed in the past, they usually have a very general purpose and biotechnology remains a matter of specialists where every modification has to be made from scratch. We imagined a repurposed organism made especially for the quick construction of these <EM>self-replicative tiny factories</EM>, that could be easily used by startups, community labs or just by enthusiasts. In the following section we discuss the constraints associated with it, and what such an organism could look like.</P></DIV><DIV class="column-right">
For the product to be usable, several specifications have to be taken into account:
<UL><LI><B>It must be easily extendable:</B>Our micro-organism should be a chassis allowing for quick addition of standard cassettes</LI><LI><B>It must be modular:</B> The different metabolic pathways should be independant so they can be put together without going through tedious troubleshooting,</LI><LI><B>It must survive in the real world:</B> To make our micro-organism more resistant to contamination, we need to design it so our modifications come with a minimal fitness cost.</LI><LI><B>It must be all-in-one:</B> For people with limited equipment, having only one strain that does everything is a huge advantage, because only one bioreactor and one production line is needed. This makes it accessible to community labs or NGOs that would want to start producing their own version of our product.</LI></UL></DIV><H1 id="our-design">Our design</H1><H2 id="from-the-lab-to-the-world">From the lab to the world</H2><DIV class="column-left"><P>For a biological product to leave the benches and actually reach the population, it's essential to foresee its life in the hands of the people who will cultivate it and make sure it stays alive for the future. Our design must therefore provide strategies to create a durable, usable product. On paper, the plan is simple: the manufacturers grow the micro-organism, distribute it and save a little fraction with which to start a new culture. This could in principle continue ad infinitum, but in reality the universal rules of biology soon kick back in.</P><P>Let's consider the following scenario: a wild type organism sneaks into the incubator and starts to replicate along with the engineered organism. Our microbe cannot compete: this contaminant has been selected precisely for its ability to sneak into environments and replicate, during hundreds of years, while our microbe has the burden of producing tons of enzymes to make the precious vitamins. Additionally, unnatural proteins and metabolites can have toxic effects when their production rate is high. After a couple of growth cycles, the worst seems unavoidable: the microorganism that will be distributed will not be the right one. Not only does it not produce nutrients, but it might not ferment the rice well or even be pathogenic.</P><P>These contamination events bring a lot of hassle for the manufacturer, so our design must provide solutions for making them as rare as possible.</P><P>Our approaches is based on two strategies:
<UL><LI><B>Reducing the fitness burden:</B> To make our micro-organism more resistant to contamination, we need to design it so our modifications come with a minimal fitness cost.</LI><LI><B>Identifying the contamination:</B> If a contamination occurs, it is essential that it does not go unnoticed. Our design must allow the manufacturer to detect contamination, and check that what he is growing is exactly what he wants to grow.</LI></UL></P></DIV><DIV class="column-right"><P>It seems impossible to make a strain that fullfills its nutrient-producing functions while growing as fast as the wild type, so we found a workaround: the cells that people use are not the cells that people grow.

We embedded a differentiation system into our organism, so the vitamin-producing pathways are only expressed after a recombination event. First, the cells that are grown are almost identical to the wild-type cells. The battle against contaminants is now a fair fight. The only difference is that they express a gene allowing for their identification, that we will further refer to as <EM>the ID gene</EM>.
Before distribution, the differentiation is induced and the cells start to produce vitamins in high quantity. They lose the ID gene in the process.</P><P class="caption"><B>Our model for decreasing the fitness burden of our organism during the production phase.</B> The cells do not produce any vitamin during the manufacturing process. The vitamin production is triggered before the distribution.</P></DIV><P>We protected our product against foreign organisms, but one threat remains: our organism's own mutants. If a mutation occurs in the active site of an enzyme, or in the promoter of an operon, the functionality of the organism might be impaired. How can we prevent our organism from mutating?</P><P>Fortunately, our friends at the Vanderbilt University iGEM team worked precisely on that problem this summer. We worked hand in hand with them to see what a real-life application of their invention would mean practically. They invented an algorithm to scan the sequences looking for regions that are likely to mutate, and proposed alternative versions of our sequences.
As they worked on this project while we were working on ours, we obviously could not use their optimized sequences for our constructs. However, we relied on gene synthesis for a lot of parts, so it would not have been a problem to use the optimized sequences instead. Their algorithm is therefore a valuable tool for any synthetic biologist willing to create durable products, and we applaud their work.
</P><H2>The ID gene</H2><DIV class="column-left" style="width:40%"><P>The quality control of our system is possible thanks to an ID gene, which consists of the fluorescent protein mCherry expressed at low levels. We chose this protein because its fluorescence colour is easily distinguishable from the media that will be used for growth, hence a better signal.</P><P>The aim of this gene is to provide a quick and reliable way to determine whether the strain that will be distributed is the one intended. After growing the micro-organism in a bioreactor, a sample is taken in order to start a new culture from it. We suggest that, while doing so, the sample has to pass a quality check where its fluorescence is measured.</P><P>If the sample displays fluorescence, the culture is sent for packaging and a new culture can be launched. If the fluorescence in not sufficient, the sample is discarded and a new blister stock of the original strain is used to start the culture.</P><P>
This fluorescence measurement is a good example of real-life use for the DIλ spectrophotometer, a low-budget device that is developed by our neighbours at the Openlab. Heavy development is currently ongoing to make it capable of fluorescence measurements.
</P><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/DILambda" class="readMore buttonCyan">Click here to learn more about the DIλ spectrophotometer</A></DIV><DIV class="column-right" style="width:55%"><SPAN class="caption"><B>How quality control will be performed.</B> The transfer of the inoculate should be done in sterile conditions in a container that also plays the role of fluorimeter cuvette. This ensures that even low-budget labs will always distribute the right strain to people.</SPAN></DIV><H2>An extendable system</H2><DIV class="column-left" style="width: 35%"><P>Our differentiation system is inspired by the Brainbow system, initially developed for tracking the axons of neurons in mammalian's brain. We modified it so it becomes extendable (<B>A</B>).</P><P>This system is randomized on a single-cell level, so each cell produce one —and only one—, vitamin pathway. In most research work, metabolic engineering has been done only one target compound at a time, and little is known about what happens when production pathways are used simultaneously in the same cell (<B>B</B>).
Having one cell expressing only one pathway should theoretically preclude unexpected interactions between different pathways, thus making an extendable framework where every synthesis function is decoupled (<B>C</B>).
The different vitamin-producing pathways can be prototyped separately on a classical lab strain, and it is then easy to put them all together in the same chassis for a multi-functional organism.</P></DIV><H2 id="the-chassis">The chassis</H2><P>Let us see how it works under the hood.
Before addition of any metabolic pathways, this is what our empty chassis would look like. The following cassette is integrated in the chromosome. All proteins' coding regions are preceded by a RBS (<EM>Ribosome Binding Site</EM>) and followed by a transcription terminator.</P><P><STRONG>Constitutive promoter:</STRONG> Thanks to this promoter, a RNA transcript of the cassette will be produced until the first terminator is reached.</P><P><STRONG>The Lox Array:</STRONG> 
The original LoxP site comes from the phage P1.
When an enzyme called the <EM>CRE recombinase</EM> is expressed, all the DNA between two Lox sites is deleted. Each lox site is made of one <EM>overlap region</EM> (in bold) surrounded by two complementary <EM>flanking regions</EM>. The middle of the sequence can be modified, but two LoxP sites will recombine together only if the sequence is exactly identical for both (Richier 2015). The flanking regions cannot be mutated and determine the specificity for one recombination enzyme.</P><P>Here are the four orthogonal Lox sites we used:
<UL style="font-size:13px"><LI><B>LoxP:</B>    ATAACTTCGTATA<STRONG>ATGTATGC</STRONG>TATACGAAGTTAT</LI><LI><B>Lox2272:</B> ATAACTTCGTATA<STRONG>AAGTATCC</STRONG>TATACGAAGTTAT</LI><LI><B>LoxN:</B>    ATAACTTCGTATA<STRONG>AGGTATAC</STRONG>TATACGAAGTTAT</LI><LI><B>Lox5171:</B> ATAACTTCGTATA<STRONG>ATGTGTAC</STRONG>TATACGAAGTTAT</LI></UL></P><P><STRONG>The ID gene: </STRONG>This gene is entirely optional but can be used as a barcode to identify the strain. This allows for quality control of what is inoculated when a new production culture is started.</P><P><STRONG>The landing pad: </STRONG>
This part allows for easy integration of new gene cassettes into the system.

</P><P><STRONG>CRE-recombinase: </STRONG>The CRE recombinase should be integrated in the chromosome as well, so we do not have to use an antibiotic for maintaining the plasmid. It has to be under the control of an inducible promoter. The expression of CRE will trigger the differentiation.</P><H2 id="landing-pads">The Landing Pad</H2><DIV class="column-left"><P>Starting from this chassis, up to four metabolic pathways can be added by using the attB sequence as a landing pad. Like the Lox sites, this sequence comes from a bacteriophage: the PhiC31 phage uses it to integrate itself in the genome of the host. To insert a new sequence in this landing pad, all you need to do is build a plasmid with the matching &quot;attP&quot; site and express the PhiC31 integrase.
<UL style="font-size:13px"><LI>attB: GTGCGGGTGCCAGGGCGTGCCC<STRONG>TT</STRONG>GGGCTCCCCGGGCGCGTACTCCA</LI><LI>attP: AGTGCCCCAACTGGGGTAACCT<STRONG>TT</STRONG>GAGTTCTCTCAGTTGGGGGCGT</LI></UL>
When inserting something in the landing pad, a new landing pad should be added for subsequent integration. This landing pad should be orthogonal to the first one to avoid multiple successive integrations. The same integrase can be used, the central <STRONG>TT</STRONG> just has to be replaced by <STRONG>CC</STRONG> to make the two sites orthogonal.</P><P>In summary, a new gene to be added in the system should have the following standard structure (<B>A</B>):
<UL><LI>An attP sequence different from the one that was used just before,</LI><LI>A Lox sequence (Lox sequences should be added in the same order they come in the Lox Array),</LI><LI>The operon to be expressed,</LI><LI>An attB sequence, orthogonal to the attP used for integration,</LI><LI>A selection system (not depicted here for clarity).</LI></UL>
When the phage PhiC31 integrase is expressed, this plasmid will be integrated in the locus (<B>B</B>). The CRISPR-Cas9 system from <EM>S. pyogenes</EM> should work well for selecting the cells who integrated the plasmid(Jiang, Bikard 2013), as the attB contains the protospacer adjacent motif &quot;NGG&quot; next to the two central bases (Mojica 2009). It is therefore possible to kill the cells who still have an intact attB site, just by using CRISPR spacers targeting the following sequences:
<UL style="font-size:13px"><LI>GCGGGTGCCAGGGCGTGCCCTTGGGCTCCC for killing cells who have not integrated anything in the first attB version,</LI><LI>GCGGGTGCCAGGGCGTGCCCCCGGGCTCCC, for the second attB version.</LI></UL>
It has the advantage of leaving no scar, thus reducing the number of recombination sites present in the locus.
After integration, the new cassette becomes a new part of the system (<B>C</B>).
</P></DIV><DIV class="column-right"><SPAN class="caption"><B>The proposed process for addition of new operons to the chassis.</B> A standard cassette is integrated in the PhiC31 locus. It then becomes a new possible outcome for the random differentiation, thanks to the addition of a new LoxP site.</SPAN></DIV><H2 id="division-of-labour">Division of labour</H2><DIV class="column-left"><P>Now that the different genes have been added to the chassis, it is time to see it in action.</P><P>The CRE recombinase will cut the LoxP sites in the middle, remove the region in-between, and join the two remaining halves of LoxP sites together (Nagy 2000). This only occurs if the overlap sequence are exactly identical (Missirlis 2006). This means that, in the picture of the right, only LoxP sites of same colour would recombine. Given the configuration of this system, any LoxP recombination event would result in the loss of several other LoxP site, in a such way that further recombinations are not possible. The pair of LoxP sites that undergo recombination is therefore chosen randomly by each cell.</P><P>Depending on which region is excised, one random coding region settles next to the promoter and starts to be expressed. For a chassis containing four different operons, the mother cells differentiates in four different daughter cells, each of them expressing one operon.</P><P>Even if the chassis is not completely filled, it still works: the number of different daughter cells is always equal to the number of inserted cassettes, and the probability of each is adjusted accordingly.</P></DIV><DIV class="column-right"><P class="caption"><B>Overview of the differentiation process.</B> For a complete organism with four metabolic operons, there are four possible outcomes, each of them leading to the expression of only one operon. In other words, each cell randomly choses one operon to express.
</P></DIV><H3>How to induce the differentiation?</H3>
There are different ways the CRE recombinase can be induced.
<H4>Chemical induction</H4>
A chemical would be one of the most predictable, efficient way to differentiate the cells. However, it requires to have access to this chemical, and to open the reactor which can be impractical for community labs with low resources to maintain sterility. It is nevertheless the solution of choice for funded factories. Carbohydrates such as glucose, arabinose or lactose seem to be the best options since they are not toxic.
<H4>Heat</H4>
On the other hand, heat does not require to open the bioreactor, so it's ideal when sterile conditions are not easy to obtain.
Numerous heat-sensitive promoters exist, such as the Heat-Shock Promoter that is present in the registry (<PARTINFO>BBa_K338001</PARTINFO>).
<H3>The problem of leakiness</H3><P>Most of the promoters from living organisms are <EM>leaky</EM>, i.e. they still lead to a small amount of transcription even in the absence of inducer. In our case, it means that the CRE recombinase will be expressed from time to time in some cells and it could result in the differentiation of some of them. If their number is low, it should not have any consequence as the differentiated cells are very unlikely to take over, but leakiness may be an obstacle if it affects a large proportion of the cells.</P><H3>The perfect expression level</H3><P>We could try to differentiate all the cells as quickly as possible, or let the cells differentiate slowly in a prograssive manner. Provided the differentiated cells grow significantly slower than the mother cell, the strategy that generates the largest amount of vitamins is not obvious.
To answer this question, we created a mathematical and computational model of the situation. Given the growth rate of the mother cells and the daughter cells, it is possible to calculate the optimal differentiation rate, and chose the strength of the promoter accordingly.
</P><A href="https://2015.igem.org/Team:Paris_Bettencourt/Modeling" class="readMore buttonCyan">Click here to learn more about the model</A><H1>Results</H1><H2>Construction of the system</H2><P>We succesfully assembled a prototype version of this system in the model bacteria <EM>Escherichia coli</EM>.
The initial promoter is a strong constitutive promoter from the biobricks registry, BBa_J23119.The genes involved in vitamin production are replaced with fluorescent proteins, allowing for easy monitoring of their production. Our construct contains mCherry as a reporter gene, and two other fluorescent proteins to mimick pathways operons. It also has a phage PhiC31 integration site for subsequent addition of new genes.</P><P>In theory, the cells with this cassette integrated in the chromosome are expected to emit a red fluorescence. Upon induction of the CRE-recombinase, they should lose the red fluorescence and start to express either mCerualean (a cyan fluorescent protein) or mVenus (a yellow fluorescent protein). Each cell should express only one of those two proteins at the same time.</P><SPAN class="caption"><B>Map of the DNA sequence we constructed.</B> The promoter BBaJ23199 is constitutive. mCherry, mCerulean, mVenus are fluorescent proteins of different colours.</SPAN><DIV class="column-left"><H3>Chromosomal integration</H3><P>This cassette was constructed by gene synthesis and Gibson assembly and assembled in a self-integrating plasmid vector (Saint-Pierre, 2013). This vector uses the integrase of the phage HK022 to integrate itself in <EM>E. coli</EM>'s chromosome. This plasmid was electroporated in the bacteria and the HK022 integrase was induced.</P><P>To check that the cassette has correctly been integrated in the right locus, we performed an analytical PCR on the whole genome of the transformants, with a set of four primers which allows for amplification of the junction between the vector and the chromosome.
</P><P>Here is presented the result of this PCR using the genome of six clones of transformants as a template.</P><P>It tells us that the vector was succesfully integrated at the right locus. It also shows that there have been only one integration and no tandem integrations, which would have resulted in an additional band.</P></DIV><DIV class="column-right"><SPAN class="caption"><B>Gel electrophoresis after PCR for checking the integration.</B> We amplified the junctions between the artificial cassette and <I>E. coli </I>'s chromosome. For every screened clone, the two bands have the expected sizes, which proves that the cassette is integrated in the correct locus.</SPAN></DIV><DIV class="column-left"><SPAN class="caption"><B>Gel electrophoresis after PCR for checking the presence of the three fluorescent proteins.</B> For the clone shown here, it means that all three proteins ORFs are present on the chromosome, even though only the first one is actually expressed.</SPAN></DIV><DIV class="column-right"><H3>Integrity of the cassette</H3><P>We then performed three other PCRs with pairs of primers binding on the ORF of the three fluorescent proteins. As a <STRONG>positive control</STRONG>, we performed the same PCR on the pure fragment that have been used for the assembly.
This way we ensured that the cassette was present in its entirety in the chromosome.</P><H3>Sequencing of the Lox Array</H3><P>To investigate whether unexpected recombination occured within the LoxP sites due to homologous recombination, we performed Sanger sequencing on the first part of the integrated cassette, where the Lox Array is. This way we could make sure that it was still intact and contained no PCR-induced mutations.</P></DIV><H2>Impact of the Lox array on the transcription</H2><DIV class="column-left"><P>The Lox array was the most difficult region to construct. As it is a very repetitive sequence with numerous dyad repeats, it is tedious to synthesize, amplify and assemble. That's why we created biobrick <A href="http://parts.igem.org/Part:BBa_K1678005">BBa_K1678005</A>. It contains the promoter followed by the four orthogonal Lox sites. We sequenced this biobrick to confirm that it contains no mutation.</P><P>We characterized this new biobrick's function by assembling it in pSB1C3 with the part <PARTINFO>BBa_K516030</PARTINFO> which contains a RBS, the mRFP coding sequence and a double terminator.
For comparison, the biobrick <PARTINFO>BBa_J23119</PARTINFO> was assembled with the same mRFP cassette on the same vector.</P><P>As in prokaryotes the 30S subunit of the ribosome binds directly to the RBS, the LoxP array does not theoretically interfere with translation. It can however interfere with the transcription.
During the transcription, the RNA polymerase has to go through the LoxP array, which is made of repetitive sequences that are likely to form a hairpin. We show that this has an impact on the transcription efficiency (Mann-Whitney-Wilcoxon test, p-value &lt; 10<SUP>-6</SUP>). However, it still allows for strong protein expression as the average expression level was equal to 91% of the expression level of the BBa_J23119 promoter. The fraction of RNA polymerases that go through the Lox array should be more than enough for our design.</P></DIV><DIV class="column-right"><P><SPAN class="legend"><B>Characterization of the promoter followed by the four LoxP sites.</B>
Using standard biobrick assembly, three plasmids were constructed and transformed into <I>E. coli</I>:</SPAN><UL style="font-size:13px"><LI>The promoter directly connected to the mRFP sequence (RBS + ORF + Terminator),</LI><LI>The promoter connected to the Lox array, connected to the mRFP sequence,</LI><LI>The promoter alone, without any fluorescent proteins as a negative control.</LI></UL><SPAN class="legend">
The cells were diluted to an OD<SUB>600</SUB> of 0.01, grown to exponential phase and the fluorescence was measured on a TECAN plate reader when the OD reached 0.3. The excitation wavelength was 585 nm and the detection wavelength was 615 nm.
</SPAN></P></DIV><H2>Expression of the proteins</H2><DIV class="column-left"><SPAN class="legend"><B>Expression of the first protein of the cassette in the chromosome.</B>
A &quot;mother cell&quot; with our differentiation system integrated in the chromosome was grown to exponential phase and its fluorescence was measured when OD<SUB>600</SUB> reached 0.3. As a negative control, a cell without fluorescent underwent the same treatment.
Excitation wavelength: 585 nm.
Detection wavelength: 615 nm.
</SPAN></DIV><DIV class="column-right"><P>Because this differentiation system requires that only one copy of the sequence is present in the cell, we measured the expression level of mCherry on cells with the chromosomally integrated cassette.</P><P>
The cells exhibit clear fluorescence (Mann-Whitney test, p-value &lt; 10<SUP>-6</SUP>), even though it was not visible to the naked eye. The mCherry fluorescent protein is therefore a suitable reporter for quality control of the strain.
</P></DIV><H2>Induction of the differentiation</H2><DIV class="column-left"><P>We then aimed to trigger the differentiation of our newly constructed strain. A strain carrying the plasmid pFHC2938, that allows for expression of the CRE recombinase upon arabinose induction (Nielsen 2006), was aquired.
Unfortunately, all our attempts at transforming our strain with it have been unsuccesful, even when resorting to very efficient techniques such as electroporation. It always lead to either nothing, or a lawn of bacteria that did not seem to carry the antibiotic resistance using for selecting the plasmid. To troubleshoot this transformation, we made three hypothesis:
<UL><LI>Is there something in the cell that interferes transformation?</LI><LI>Is the strain not suitable for CRE expression, e.g. there are LoxP sites somewhere that result in deletions in the genome?</LI><LI>Is the plasmid the wrong one?</LI></UL></P><P>
To figure this out, we transformed our strain with the CRE recombinase plasmid along with a standard pSB1C3-mRFP plasmid. The transformation of the CRE-recombinase gave a lawn, while the control plasmid gave clear colonies. We then performed the same 4-primer PCR that was used to check for integration on the bacteria present on the plates. The bacteria transformed with pSB1C3-mRFP still contained the integrated cassette, while the bacteria on the pFHC2938 plate did not display any band, meaning that they were contaminants. This means that the transformation process is not the problem.</P><P>We then transformed a non-modified strain without our integrated cassete with the pFHC2938 plasmid. At the same time, the integrative plasmid was transformed into another <I>E. coli</I> strain (STBL). None of the transformations yielded any colonies, meaning that the problem came from the plasmid and not from the strain.</P></DIV><DIV class="column-right"><SPAN class="caption"><B>Troubleshooting of the transformation.</B><STRONG>A.</STRONG>  Strain carrying the integrated cassette before transformation. <EM>B.</EM> Cells picked from the lawn after pFHC2938 transformation. <STRONG>C.</STRONG> Colony picked after the transformation with a control plasmid (pSB1C3-mRFP).</SPAN><P class="caption">Unmodified Top10 lab strain after transformation with pFHC2938. The great nothingness.</P></DIV><H1>Outlook</H1><H2>More than just operons</H2><P>The system we invented, as presented here, works primarily when the whole vitamin pathway fits in one operon. It was notably the case for our pathway for vitamin A, where all the required enzymes are tied together in one big polycistron.
It is still possible to implement this differentiation system for a pathway that needs several promoters to function. For this, we can put the different operons under promoters that are activated by another factor, and this factor is put in the differentiation system.
Two related technologies, <EM>CRISPR interference</EM> and <EM>CRISPR activation</EM> (Bikard 2013), appear as an ideal way to do this. It relies on a mutant of Cas9 defective for nuclease activity (dCas9), that can be targeted at about any place in the genome. This can lead to either repression of transcription, or activation by fusing a transcription activation factor to dCas9. It works on both prokaryotes and eukaryotes (Perez-Pinera 2013), and is extremely versatile and programmable. As seen in our phytase project, inactivation of a gene is sometimes useful for nutrient production.
</P><P>By expressing dCas9 and replacing the fluorescent proteins in our construct with CRISPR arrays, it could be possible to widely change the expression profile of the micro-organism while keeping the advantages of differentiation.</P><H1>Attribution</H1>
This project was designed and accomplished by Antoine Vigouroux in consultation with Jason Bland and Ihab Boulas.

Most of the strains (DH5alpha, Top10, NEB turbo, Pir116) were kindly provided by Inserm U1001. Plasmids pFHC2938 and pMEV250 were provided by Jason Bland and Aleksandra Nivina at Didier Mazel's lab at Institut Pasteur. Plasmids pL1F2 and pR6K-shortened were provided by Antoine Decrulle and Ihab Boulas at Inserm U1001. The pIT5-KH vector was provided by Lun Cui at David Bikard's lab at Institut Pasteur.

Special thanks to all the people who gave me an hand during this project, and all the Paris Bettencourt team for making this adventure so much fun.

</DIV></DIV></DIV></BODY></HTML>