<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Project_VitaminB12 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Paris Bettencourt/Project/VitaminB12</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="menuContainer"><DIV id="smallMenuTitle"><H1>Ferment It Yourself</H1><H3>iGEM Paris-Bettencourt 2O15</H3></DIV><UL><LI class="left"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Background" title="Background">Background</A></LI><LI class="left"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Design" title="Design">Design</A></LI><LI class="left"><BUTTON class="menuButton">Project</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Micronutrients">Micronutrients</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Continuity">Continuity</A></LI><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Manufacturing">Manufacturing</A></LI></DIV></UL><LI class="left"><BUTTON class="menuButton">Achievements</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Parts">Parts</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Medals">Medal Fulfillments</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Practices</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/SynBio_Challenge">SynBio Challenge</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/DILambda">DIλ</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Acceptance">Acceptance</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/Events">Events</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/Rhizi">Rhizi</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Collaborations">Collaborations</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Safety</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Safety">Safety Overview</A></LI><LI><A href="https://2015.igem.org/Safety/Final_Safety_Form?team_id=1678">Safety Form</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/ReviewBoard">Review Board</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Notebook</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Notebook">Daily notebook</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols">Protocols</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Bibliography">Bibliography</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Team</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Team">Members</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Acknowledgments">Acknowledgments</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Press">Press</A></LI><LI><A href="https://igem.org/Team.cgi?id=1678">Official Team Profile</A></LI></DIV></DIV><DIV id="riboflavin"><DIV class="contentContainer noBanner"><DIV id="banner"><H3>Vitamin B12 (cobalamin)
</H3></DIV></DIV><DIV class="textWrapper"><DIV class="textContent black" id="caffeine"><DIV id="overview" style="text-align: center"><TABLE><TBODY><TR><TH class="noBottom"><H3>Background</H3></TH><TH class="noBottom"><H3>Aims</H3></TH><TH class="noBottom"><H3>Results</H3></TH></TR><DIV id="overview" style="text-align: justify"><TR><TD class="noTop">Vitamin B12 is already widely naturally synthesized by some natural producers such as <I>Propionibacterium freudenrichii</I> which is GRAS (Generally Recognized As Safe) organism. It is however quite impossible to quickly and cheaply quantify it.
      </TD><TD class="noTop">Introducing <I>Propionibacterium</I> in our idli batter would allow us to highly increase its quantity of vitamin B12. Quantifying vitamin B12 in fermented food in a cheap way was aimed by designing a biosensor based on a vitamin B12 riboswitch associated with a fluorescent protein.
      </TD><TD class="noTop"><UL><LI>We successfully grew <I>Propinibacterium freudenreichii</I> in the idli batter</LI><LI>Used a riboswitch sensor to measure vitamin B12</LI><LI>Biobricked this riboswitch sensor: <A href="http://parts.igem.org/Part:BBa_K1678007">BBa_K1678007</A></LI></UL></TD></TR></DIV></TBODY></TABLE></DIV></DIV><DIV class="textContent"><H1 class="date one">Cobalamin (Vitamin B12)</H1><DIV class="column-left"><P>
Cobalamin is a vitamin involved in the metabolism of every cell in the human body. It is needed for development and maintenance of the nervous system and brain. Vitamin B12 is involved in DNA synthesis and regulation of the transcription. Neither humans nor animals are able to synthesize this vitamin. Foods of animal source are the only natural source of cobalamin in human diet and Cobalamin deficiency results in heavy symptoms including brain damage. It makes it the cause of one of the major vitamin deficiency in developing countries.</P><P>
We tackled this issue by the re-engineering the food microbiome of idli which is a very popular dish used as primary food source throughout India. Since the microbial ecosystem of idli doesn't produce vitamin B12, we introduced <I>Propionibacterium freudenreichii</I>, a bacterium that has been reported as a Cobalamin producer (Takahashi-Iñiguez et al. 2012) and assessed its fitness in the idli environment.</P></DIV><DIV class="column-right"><SPAN class="caption">Vitamin B12 (or cobalamin) molecule.</SPAN></DIV><H2>B12 daily requirement and toxicity </H2><UL><LI><B>Children:</B> 0.4-1.8 µg/day</LI><LI><B>Pregnant Women:</B> 2.6 µg/day</LI><LI><B>Men:</B> 2.4 µg/day</LI><LI><B>Cobalamin</B> is not toxic even in doses well above the recommended daily allowance because it is a water soluble vitamin.</LI><LI><B>However</B> this comes with a cost as water soluble vitamins are not stored in the body but <B>excreted</B> through urine.</LI></UL><H2><I>Propionibacterium freudenreichii</I> grows in idli</H2><DIV class="column-left"><P><I>Propionibacterium freudenreichii</I> is a bacterium that has the Generally Recognized As Safe (GRAS) status, and is commonly found is cheese such as Emmental. It is a natural big producer of vitamin B12, conferring high amounts of this vitamin on cheese. 
Since the biosynthesis of vitamin B12 is a complex process, we decided to engineer the microbiome of idli by directly adding this bacterium to the batter, in the hope of increasing its vitamin B12 content.</P><P>As a matter of fact, <I>P. freudenreichii</I> was found to grow well in idli (see figure on the right).</P></DIV><DIV class="column-right"><SPAN class="caption">Comparison of growth of <I>P. freudenreichii</I> in YEL (regular growth medium) and 100 times diluted idli batter.</SPAN></DIV><H1 class="date two" id="biosensor">Cobalamin Biosensor</H1><DIV class="column-left"><H2>Background</H2><P>
In order to avoid a costly and labor intensive HPLC to analyze production of cobalamin (B12) by our strains, we wanted to develop a reliable assay which would be cheap, easy to use, precise, specific and sensitive. We chose to develop a riboswitch-based biosensor, as others have been developed for B12 which we could adapt or improve. </P></DIV><DIV class="column-right"><H2>Riboswitch / Fluorescent Protein</H2><P>
The Cobalamin Biosensor is based on a riboswitch taken from a transcribed fragment upstream of a cobalamin biosynthesis gene, cbiB, which is found in Propionibacterium shermanii and has been demonstrated to be sensitive to B12. We associated this sequence with mRFP1; as B12 acts as a translation repressor in this system, the concentration of B12 is inversely linked with RFP fluorescence. In this way, the B12 concentration can be quickly and easily measured.</P></DIV><H2>Design/Construction</H2><P>
The riboswitch expression cassette was originally ordered as a G block consisting of the riboswitch upstream of EGFP under control of the P(bla) promoter and ending with the rrnB terminator. Following insertion of this cassette into pSB1C3, we had no GFP expression at all, even in the absence of cobalamin. Sequencing of the cassette revealed to amino acid substitutions in the EGFP gene. </P><P>
Rather than attempt to determine if the EGFP was non-functional due to the mutations, or due to non-function of the riboswitch, we extracted the riboswitch and inserted it into a pSB1C3 plasmid containing an mRFP1 gene (BBa_J04450) using Gibson assembly; this 213 bp sequence was cloned just upstream of the mRFP1 gene, replacing the original RBS and the START codon, leaving the Lac promoter/Lac O untouched. The Lac promoter was left because we were not sure if the riboswitch sequence did indeed contain a promoter. This novel plasmid was cotransformed in a chemically competent E. coli DH5alpha strain along with a compatible plasmid containing the Venus fluorescent protein whose expression is not regulated by B12, and act as our positive control.</P><DIV style="align: center">
Map of the riboswitch/RFP cassette.
</DIV><P>The 213 sequence is the one that has been used in Xuan Zhu 2015 paper. We might have identified - using the BLAST tool from NCBI - that it was actually containing 24 bases - at the end - coding for the beginning of the CbiB protein, including the start codon (GTG), this has been a crucial information actually because it gave us the warranty that we were placing our fluorescent protein in frame with this original start codon. We might also have identified the RBS, 3 bases upstream.</P><H2> Experiments </H2><DIV class="column-left"><P>
To test the functionality of our construct, we measured the fluorescence level emitted by our strain in the presence of increasing levels of Adocobalamin. Cells were grown in M9-glucose; fluorescence of mRFP1 and Venus and the OD600 were measured on a TECAN plate reader. Overall, we do observe a slight decrease in RFP expression as the adocobalamin concentration is increased, as would be expected if the riboswitch functions properly.
</P></DIV><DIV class="column-right"><P>
However, we do observe an unexpected peak in RFP expression. This peak was reproduced during repetitions of the experiment in a manner dependent on the samples’ placement on the plate reader. Therefore, this is likely an artefact of the experiment and requires a redesign of the experiment.
</P></DIV><DIV style="text-align: center">


Cobalamin detection with our biosensor. <B>Left:</B> mRFP1/OD = f([AdoCbl]); Exc=585nm and  Em=635nm. <B>Right:</B> Venus/OD = f([AdoCbl]) = Positive control; Exc=480nm and  Em=520nm. OD was mesured at 600nm.
</DIV><DIV class="column-left"><P>
Normalizing the mRFP1 expression to the Venus expression gives you the power to get rid of any noise coming from the bacterial metabolism, or it’s environment.
</P><P>
We were waiting our solution containing the E. coli sensor to reach an OD of 0.1 and dilute it 1:20 with M9-Glucose. Then this diluted solution containing our bacterial sensor was inoculated (75uL) with increasing concentrations of AdoCbl  diluted in M9-Glucose (75uL) in plate reader wells covered by 50uL mineral oil. M9 was used because it is free of B12 and thus their is no interference in our measure.</P></DIV><DIV class="column-right"><P>
Our riboswitch has been demonstrated to be sensitive to different forms of cobalamin:
<UL><LI>Natural forms: AdenosylCbl (AKA AdoCbl or Coenzyme B12), MethylCbl, and HydroxyCbl</LI><LI>Hemisynthetic form: CyanoCbl</LI></UL></P><P>
Some other experience would have to be done to determine if our riboswitch construction is sensitive to those Cobalamin forms.
</P><P>
All the new results experiences will be updated in the biobrick page. (<A href="http://parts.igem.org/Part:BBa_K1678007">BBa_K1678007</A>). Check for updates if you’re interested in.
</P></DIV><H2> Use the construction with a food sample </H2>


In order to analyze a B12 concentration in any sample would need to calibrate the system doing a standard curve, Fluo/OD = f (B12 concentration).

If it turns out that the tool is sensing CyanoCobalamin, in order to mesure the whole quantity of bioactive Cobalamin in a sample you would have to convert any form in CyanoCbl before assay by heating it in acetate buffer with KCN.<H2> Apply the same model for other biosensors </H2>


Using Riboswitch for biosensing could be applied to other vitamins such  as :

-B1 Vitamin (Thiamin) using the thiamine pyrophosphate (TPP) sensing riboswitch
-B2 Vitamin (Riboflavin) using flavin mononucleotide (FMN) sensing riboswitch; FMN is a precursor of B2

The mechanism of all riboswitchs is not very well understood yet, so as a cautious mesure the approach to design a riboswitch based biosensor would be to insert before any fluorescent protein a wide sequence containing :

1- The natural promoter
2- The aptamer (binding) part from the riboswitch
3- The RBS
4- The beginning of the protein next to the riboswitch (including a start codon)<DIV style="text-align: center">
General scheme for Riboswitch/Fluorescent protein based biosensor.</DIV>


Riboswitch is a complex mRNA system where all the element listed above are intricate in a fine manner, disturbing this trying to shorter the sequence could be fatal in the operation of the biosensing construct.

The fluorescent protein next to the riboswitch system sequence must form a fusion protein with the natural truncated protein. A critical point is obviously to place the fluorescent protein whiteout its start codon in frame with this natural original truncated protein.</DIV></DIV></DIV></BODY></HTML>