<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Project_VitaminA skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Paris Bettencourt/Project/VitaminA</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="menuContainer"><DIV id="smallMenuTitle"><H1>Ferment It Yourself</H1><H3>iGEM Paris-Bettencourt 2O15</H3></DIV><UL><LI class="left"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Background" title="Background">Background</A></LI><LI class="left"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Design" title="Design">Design</A></LI><LI class="left"><BUTTON class="menuButton">Project</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Micronutrients">Micronutrients</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Continuity">Continuity</A></LI><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Manufacturing">Manufacturing</A></LI></DIV></UL><LI class="left"><BUTTON class="menuButton">Achievements</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Parts">Parts</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Medals">Medal Fulfillments</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Practices</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/SynBio_Challenge">SynBio Challenge</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/DILambda">DIλ</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Acceptance">Acceptance</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/Events">Events</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/Rhizi">Rhizi</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Collaborations">Collaborations</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Safety</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Safety">Safety Overview</A></LI><LI><A href="https://2015.igem.org/Safety/Final_Safety_Form?team_id=1678">Safety Form</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/ReviewBoard">Review Board</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Notebook</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Notebook">Daily notebook</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols">Protocols</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Bibliography">Bibliography</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Team</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Team">Members</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Acknowledgments">Acknowledgments</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Press">Press</A></LI><LI><A href="https://igem.org/Team.cgi?id=1678">Official Team Profile</A></LI></DIV></DIV><DIV id="retinol"><DIV class="contentContainer noBanner"><DIV id="banner"><H3>Vitamin A (retinol)
</H3></DIV></DIV><DIV class="textWrapper"><DIV class="textContent black" id="caffeine"><DIV id="overview" style="text-align: center"><TABLE><TBODY><TR><TH class="noBottom"><H3>Background</H3>
      It has been shown that <I>S. cerevisiae</I> can be engineered to produce vitamin A with the addition of 3 genes.
      </TH><TH class="noBottom"><H3>Aims</H3>
      Evaluate the yeast growth and its vitamin production in idli.
      Improve the yield of vitamin A in <I>S. cerevisiae</I>.
      </TH><TH class="noBottom"><H3>Results</H3>
      Showed that the vitamin A producing strain grows as fast as the WT in idli.
      Proved that the yeast produces significant amounts of vitamin A in idli.
      Designed a plan to increase the vitamin A synthesis.
    </TH></TR></TBODY></TABLE></DIV></DIV><DIV class="textContent"><DIV class="column-left" align="justify"><H1 class="date one">Motivation</H1><P><B>Current situation</B>Vitamin A deficiency is a <A href="https://2015.igem.org/Team:Paris_Bettencourt/Background">crucial issue</A> in India, affecting millions of people.
The government has developed different programs to provide people with vitamin A supplements, but they are not very convenient (people need to go to a center everyday to receive it), only help a small portion of the population, and the retinol present in the supplements is not as healthy as the ß-carotene found in food. Another potential solution is Golden Rice, a rice that have been genetically engineered to synthesize vitamin A. However, the use of Golden Rice is rather controversial and has not been implemented in India.

<B>Our idea</B>Our idea is to have vitamin A produced by the microbes involved in food fermentation and not by the cereal itself. It is much easier, cheaper and faster to genetically engineer micro-organisms than plants. And for the consumer it is much less intrusive and constraining to have a starter of yeast and bacteria which they can chose to add or not to their food at anytime, rather than to have to change their entire crops as proposed by the Golden Rice project.


</P><H1 class="date two" id="design">Design</H1>
To produce vitamin A in idli, a popular Indian fermented rice cake, we chose to use the yeast <I>Saccharomyces cerevisiae</I> since it is commonly found in idli batter (Soni and Sandhu, 1989 and Nout, 2009), which reduces the likelihood of affecting the idli's taste. Though <I>S. cerevisiae</I> doesn’t naturally produce ß-carotene, it has been shown that with the introduction of two carotenogenic genes from the carotenoid-producing ascomycete <I>Xanthophyllomyces dendrorhous</I>, <I>S. cerevisiae</I> could synthesize ß-carotene (Verwaal et al., 2007). These two genes are <I>crtYB</I> which codes for phytoene synthase and lycopene cyclase, and <I>crtI</I>, which encodes phytoene desaturase.
<P>Additional overexpression of crtE (GGPP synthase) from <I>X. dendrorhous</I>, and an additional copy of a truncated 3-hydroxy-3-methylglutaryl-coenzyme A reductase gene (<I>tHMG1</I>) from <I>S. cerevisiae</I> were both reported to increase the carotenoid production levels in <I>S. cerevisiae</I> (Verwaal et al., 2007). A more recent study also showed that ß-carotene synthesis in this yeast could also be increased with codon-optimization of <I>crtI</I> and <I>crtYB</I>, and by introducing the HMG-CoA reductase (mva) from <I>Staphyloccocus aureus</I> rather than the truncated HMG-CoA reductase (tHMG1) from <I>S. cerevisiae</I> (Li, 2013).</P></DIV><DIV class="column-right" align="left">HMG-CoA: 3-hydroxy-3-methylglutaryl-coenzyme A
HMG1 and HMG2 (paralogs): HMG-CoA reductase
IPP: isopentenyl pyrophosphate
DMAPP: dimethylallyl pyrophosphate
GPP: geranyl diphosphate
FPP: farnesyl pyrophosphate
GGPP: geranylgeranyl-diphosphate
CrtE: GGPP synthase
CrtYB: lycopene cyclase/phytoene synthase
CrtI: phytoene desaturase

</DIV><H1 class="date three" id="design">The polycistron</H1><DIV class="column-left" align="justify">In 2014, Beekwilder &amp; als. assembled the <I>crtE</I>, <I>crtYB</I>, and <I>crtI</I> genes into a polycistronic construct where the individual Crt proteins were separated by the T2A sequences of the <I>Thosea asigna</I> virus. This polycistron is under the regulation of the strong yeast promoter TDH3, and the terminator TEF1. The three genes are involved in the lasts steps of the ß-carotene synthesis and it has have been shown that their addition to <I>Saccharomyces cerevisiae</I> was enough to make it produce ß-carotene.</DIV><DIV class="column-right" align="left"><B>2A sequences or cis-acting hydrolase element:</B>
2A-like sequences are able to force the ribosome to &quot;skip&quot; a codon. The ribosome releases the part that it has already translated and to keep translating the mRNA. It allows transcription of multiple proteins from only 1 mRNA with 1 promoter, like bacterial polycistronic elements, but also with only one kozac sequence (yeast RBS) which ensures that the quantities of all the translated product are the same.</DIV><H1 class="date one">Comparative growth of WT and Vitamin A producing Yeast</H1><DIV class="column-right" align="justify">
No differences were found between the growth rate of the 2 yeasts. This was a concern because a fitness cost would mean that wild type yeast would have slowly replaced the carotenoid producing yeast in the culture. 
We are using the strain IME167: Mata ura3-52 HIS3 LEU2 TRPl MAL2-8c SUC2 pUDE269 (2u URA3 pTDH3-crtYB-T2Al-crtl-T2A2-crtE-tTEF) described in &quot;Polycistronic expression of a beta-carotene biosynthetic pathway in Saccharomyces cerevisiae coupled to beta-ionone production&quot; (Beekwilder et al. 2014). This strain is carrying a high copy plasmid with the polycistronic construct.
We were afraid that wild type yeast would have been able to out-compete the vitamin A producer yeast because of the burden that the synthesis could have put on the cell. So we compared the growth speed of both the producer and a WT strain of <I>S. cerevisiae</I> (SK1) in YPD by measuring the variations of OD600nm over time.</DIV><H1 class="date one">Absorption spectrum of yeast extracts</H1><DIV class="column-left">In order to quantify the production of carotenoids of the IME167 strain we extracted the carotenoid content of the cells with hexane. The control was the same SK1 strain that has been used in the growth experiment.

The peak of absorbance at 449nm can be linked to the concentration of carotenoids using the following formula : <B>Total carotenoids (in µg/g[dw]) = (A449 x mL hexane) / (0.2702 x g [dw])</B>
Here the dry weight is 0.8g and the volume of hexane used to extract the vitamin is 2mL. There is a peak of absorbance reaching 1.2 at 449nm. 
The total carotenoid content is then (1.2*2)/(0.2702*0.8) = 11ug/g of dry weight.
</DIV><H1 class="date two">Further improvements </H1><DIV class="column-left" align="justify"><B>An optimized polycistron</B>The current strain is not producing enough ß-carotene and to meet to daily requirement we would have to add more yeasts in the batter and this could have effects on the taste and texture of idli. This is why we aimed to strongly increase the ß-carotene yield of those yeast.
For this purpose, we designed a construct very similar to theirs, except that we moved the <I>crtE</I> gene to the first place of the polycistron, in order to increase the carotenoid yield. Indeed, it has been shown that the efficiency of translation decreases after every 2A sequence (de Felipe et al. 2006), and that an increase of CrtE may improve the ß-carotene production (Verwaal et al. 2007). We kept the same 2A sequences between the cistrons, as well as the same terminator TEF1. In order to synthesize the whole construct though, we had to change the TDH3 promoter: like most yeast promoters it has a very low GC content, which makes it very difficult to synthesize. So we used the ADH1 promoter instead, which is another strong promoter for yeast.
We also codon-optimized the three genes for <I>S. cerevisiae</I>, using the IDT codon-optimization tool, in order to increase the genes expression. The study from Li et al. (2013) had shown that the optimization of 5 codons in the sequence of <I>crtI</I>, and 8 codons in the sequence of <I>crtYB</I> had increased the ß-carotene production in <I>S. cerevisiae</I> by 200%, so we had high hopes that codon-optimizing the whole genes would lead to an even better yield.
The <A onclick="swal({
  title: 'Codon-optimized polycistron',
text:'Show the sequence of the codon-optimized polycistron: ADH1 promoter / crtE/YB/I / TDH3 terminator',
  showCancelButton: true,
  confirmButtonText: 'Open',
  cancelButtonText: 'Cancel',
  closeOnConfirm: true,
allowOutsideClick: true,
closeOnCancel: true
},
function(isConfirm){
  if (isConfirm) {
   window.location = 'https://static.igem.org/mediawiki/2015/c/c2/ParisBettencourt_sequence_polycistron.gb';
  }
});">whole construct</A> we designed was synthesized by IDT in 5 gBlocks.
<B>An optimized HMG gene</B>Additionally, we codon-optimized for <I>S. cerevisiae</I> the  <A onclick="swal({
  title: 'Codon-optimized HMG-CoA reductase',
text:'Show the sequence of the HMG-CoA reductase (mva) gene from S. aureus codon-optimized for S. cerevisiae',
  showCancelButton: true,
  confirmButtonText: 'Open',
  cancelButtonText: 'Cancel',
  closeOnConfirm: true,
allowOutsideClick: true,
closeOnCancel: true
},
function(isConfirm){
  if (isConfirm) {
   window.location = 'https://static.igem.org/mediawiki/2015/6/66/ParisBettencourt_sequence_HMG.gb';
  }
});">HMG-CoA reductase gene</A> from <I>S. aureus</I> that had been used by Li &amp; al. in 2013. Indeed, their study had shown that <I>S. cerevisiae</I> transformed with this gene had a better ß-carotene yield than the ones transformed by the tHMG1 from <I>S. cerevisiae</I>; it is highly probable than a codon-optimized version of this gene from <I>S. aureus</I> would produce even more ß-carotene.
</DIV><DIV class="column-right" align="justify"><B>Chromosomal integration</B>In the strain containing the polycistron designed by Beekwilder, the polycistronic construct is on a plasmid (pUDC082). But since in our final product we would like our yeast to grow on non-selective media and to keep the polycistron, we designed a way to integrate the construct in the yeast chromosome. Our plan was to use the HO-Poly-KanMX4-HO plasmid (AddGene plasmid #51662) as a backbone for our construct: it's a yeast plasmid for chromosomal integration into the HO locus, with a selection marker for yeast (KanMX4). This plasmid also has an origin of replication for <I>E. coli</I> and a selection marker for bacteria (Ampicillin).

Map of the HO-Poly-KanMX4-HO plasmid containing our optimized polycistron:
<B>Mutation optimization</B>Another very different way to improve the sequence of the pathway would be to replace the regions that are the most likely to mutate by alternative, more stable versions. Indeed, our strain is meant to be released in the environment without containment and to be consumed by people, so if mutations were to happen in the genes we have engineered, the yeast's ability to produce ß-carotene could be greatly reduced, or lost.
To address this problem, we made a collaboration with the Vanderbilt iGEM Team 2015, who invented an algorithm to scan the sequences looking for regions that are likely to mutate. Thanks to their software they were able to find alternative versions of the <A onclick="swal({
  title: 'Mutation-optimized Polycistron',
text:'Show the sequence of the Mutation-optimized Polycistron crtE/YB/I',
  showCancelButton: true,
  confirmButtonText: 'Open',
  cancelButtonText: 'Cancel',
  closeOnConfirm: true,
allowOutsideClick: true,
closeOnCancel: true
},
function(isConfirm){
  if (isConfirm) {
   window.location = 'https://static.igem.org/mediawiki/2015/f/fc/ParisBettencourt_sequence_mutationotimied_polycistron.gb';
  }
});">polycistronic construct crtE/YB/I</A> and <A onclick="swal({
  title: 'Mutation-optimized HMG-CoA reductase',
text:'Show the sequence of the mutation-optimized HMG-CoA reductase',
  showCancelButton: true,
  confirmButtonText: 'Open',
  cancelButtonText: 'Cancel',
  closeOnConfirm: true,
allowOutsideClick: true,
closeOnCancel: true
},
function(isConfirm){
  if (isConfirm) {
   window.location = 'https://static.igem.org/mediawiki/2015/a/ad/ParisBettencourt_sequence_mutationoptimized_HMG.gb';
  }
});">HMG-CoA</A> genes that were more robust and durable.
</DIV></DIV></DIV></DIV></BODY></HTML>