Team:Paris Bettencourt/Notebook/Riboswitch
Ferment It Yourself
iGEM Paris-Bettencourt 2O15
Background
Design
Project
Micronutrients
Continuity
Modeling
Manufacturing
Achievements
Results
Parts
Medal Fulfillments
Practices
Overview
SynBio Challenge
DIλ
Acceptance
Events
Rhizi
Collaborations
Safety
Safety Overview
Safety Form
Review Board
Notebook
Daily notebook
Protocols
Bibliography
Team
Members
Attributions
Acknowledgments
Press
Official Team Profile
Notebook
Vitamin A
Vitamin B2
Vitamin B12
Phytase
Riboswitch
Differentiation on E. coli
Differentiation on S. cerevisiae
Manufacturing
Idli and Micro-organisms
July
August
September
July 27th
Goal
E coli DH5-alpha transformation with RFP from iGEM plate 4 - 2H (pSB1A3 backbone)
Procedure
I took iGEM plate 4 - 2H (RFP, BBa_J04450 in PSB1A3 backbone) and resuspend it in 10 uL of distilled water.
Used E coli competent cells from NEB 2 tubes c2987 (50uL) : 1 control (C) and one for transformation (T)
Introduced 2 uL (200-300pg/ul) of my suspended plasmid solution in T.
Waiting 30 min in ice
Heat shock 30 sec 42°C
Then 5min in ice
Add 200uL SOC medium
Recovery 2 hours then plate it on 3 different 100ug/mL Ampicillin LB
Incubate 37°C till tomorrow
Results
The day after I had colonies growing, nothing on my control
I took 2 different colonies and do a 8 hours culture in two different tubes T1 an T2 in LB + Amp 100ug/mL, in order to do a glycerol stock.
July 29th
Goal
Making chemically competent cells
Procedure
I followed the following method
The day before, start an overnight Liquid Culture of the E. coli strain. (5mL of LB + Antibiotics + Single colony)
The next day, set the Centrifuge at 4°C and place on ice a tube containing 15mL of 100mM CaCl2 and a tube containing 4mL of 100mM CaCl2 + 15% Glycerol.
In an Erlenmeyer, add 50mL of LB and 250uL of the Liquid Culture, (for quicker growth, use a larger Erlenmeyer)
Put the Erlenmeyer to shake at 37°C until the OD(600) reaches 0.4 - 0.5.
When the OD(600) is reached, transfer in a 50mL Falcon tube and centrifuge 10min at 4°C.
Remove the supernatant and resuspend in 15mL of ice cold 100mM CaCl2. -Leave on ice for 4h.
Centrifuge 10min at 4°C.
Remove the supernatant and resuspend in 4mL of ice cold 100mM CaCl2 + 15% Glycerol.
Aliquot in 1.5mL eppendorf tubes and store at -80°C until use.
July 30th
Goal 1
PCR my G block and backbone
Procedure
PCR1 = Backbone + 068 and 069 primers, before purify
PCR2 = FINAL + 068 and 069 primers, before purify
Then PCR purification
Nanodrop : PCR 1 = 61ng/uL ; PCR2 = 25,4 ng/uL
Goal 2
Digestion/Ligation/Transformation
Procedure
First, digestion :
PCR1  ADN : 40uL = 2,4 ug 4uL EcoRI, PstI, FastAp 12 uL FD buffer 56 uL water TOT =120uL
PCR2 ADN : 40uL = 1ug 6 uL EcoRI, PstI 12 uL FD buffer 64 uL water  TOT= 128uL
10 min at 37°C
Then PCR purification
Nanodrop : nanodrop : PCR 1 = 20,7 ng/uL ;PCR2 = 25,7
For the ligation :
Vector PCR1 100 ng = 5 uL;  Insert PCR 2 300 ng = 12uL T4 ligase buffer 2uL ; T4 ligase 1 uL ; Total 20 uL
20-40 min on the bench
Nanodrop : L= 270 ng/uL
Then I did a heat shock transformation (following the iGEM protocol : http://parts.igem.org/Help:Protocols/Transformation )
C + = 10 ng/uL of RFP in pS1C3, 2 uL transformed (10pg/ul RFP Control (pSB1C3 w/ BBa_J04450)
C - = nothing but efficients cells
T =  Plasmid construct     2uL transformed
Each with 50 uL of chemically competents cells done yesterday.
Then I did 5 different plates :
5 plates with 25ug/mL chloramphenicol in LB
200 uL control + , 20 uL control +
200 uL T , 20 uL T
200 uL C -
Incubation overnight at 37°C
July 31th
Goal 1
Check the transformations I've done yesterday
Procedure
According to my plates my competent cells are working, red colonies on the C+. Transformation of my construction didn’t worked, I’ll start again all the process : PCR, purify, digest, ligate, integrate (heat shock), overnight culture.
Goal 2
PCR my backbone and my insert
Procedure
PCR1' = Backbone + 068 and 069 primers, before purify
PCR2' = FINAL + 068 and 069 primers, before purify
After PCR and after purification : 1'p = 66,5  2'p =90,5 ng/uL
The gel migration shows a bad result for PCR 1’ (Linearized pSB1C3)
August 1st
Goal
PCR my backbone and my insert again : 1" and 2"
Procedure
PCR then purify
August 5th
Goal
Ligation/transformations (insertion of FINAL block from IDT in the pSB1C3 backbone, FINAL is my Riboswitch associated with an EGFP)
Procedure
Used 3 different insert PCR products
Ligation after digestion and purification of PCR2’ = 94,6ng/uL , PCR2’’= 57,5 ng/uL , PCR2’’’=83,4 ng/uL ; Backbone pSB1C3 = 37,9ng/uL
'
"
"'
Insert
4
7
5
Backbone
5
5
5
Buffer
2
2
2
Ligase
1
1
1
Water
8
5
7
TOTAL
20uL
20uL
20uL
I did then a transformation using the iGEM protocol (Heat Shock) with 50uL of chemically competent cells + 3 uL of Ligation Product for each (pSB1C3-TOTAL)
C + : pSB1C3-RFP
C - : just competent cells
Plated on Cm 25 ug/mL
5 plates : ‘,’’,’’’,C+,C-
Results
C- : nothing
C+ : lot of red colonies
‘ : lot
‘’ : lot
‘’’ : lot
I should have done a control transformation with just the backbone
August 6th
Goal 1
Cultivate my transformants
Procedure
There is red and white transformant colonies, all resistant to Cm 25
I selected some white colonies overnight culture
On plate LB Cm —> it grew (some red, maybe contamination)
On broth LB Cm —> it grew (some red, maybe contamination)
Inoculated at 4:00 pm in LB Cm 25 bacteria from a culture ( red —> RFP ) with different AdoCbl concentrations in order to detect the minimal amount of B12 viable for e.coli : 5; 10; 20; 40; 80; 160 ng/mL
Goal 2
PCR the pSB1C3 backbone and my FINAL riboswitch construction
Procedure
2 PCR each
I1 and I2 using primers 150 and 151
LP1 and LP2 using primers 152 and 153
Purification then nano drop : good for I2 : 112.4  ng/uL
Results
It shows a good length only for I2 at 1300 bp.
Goal 3
Do the PCR again
Procedure
Template
Primers
MasterMix
Water
TOTAL
Annealing T°C
Elongation time
I3
2ng = 2 uL
1 uL
25 uL
22 uL
50uL
56°C
2 min
I4
2ng = 2 uL
2 uL
25 uL
21 uL
50uL
LP3
25ng = 1uL
1 uL
25 uL
23 uL
50uL
LP4
25ng = 1uL
2 uL
25 uL
22 uL
50uL
Results
After PCR purify, nanodrop (ng/uL)
I3
I4
LP3
LP4
30.7
177.6
86.7
93.9
Gel migration
Very good for LP4, good for LP3
Nothing for I3 and I4
August 7th
Goal 1
Digestion of LP4 and I2
Procedure
LP4
I2
EcoRI
4uL
4uL
PstI
4uL
4uL
FastAP
4uL
-
FD Buffer
12uL
8uL
DNA
2 ug (90 ng/uL) —> 20 uL
2ug (100 ng/uL) —> 20 uL
Water
76 uL
44 uL
TOTAL
120uL
80uL
New nano drop : LP4 = 87,4 ; I2 = 90,4 ng/uL
Waited 15 min at 37°C
After digestion and PCR purification, nanodrop :
38.2 ng/uL for I2 (d. and p.)
31.2 ng/uL for LP4 (d. and p.)
Goal 2
Ligation of LP4dp and I2dp
Procedure
L*
Vector LP4 dp
100 ng (31.2 ng/uL) —> 3uL
Insert I2 dp
300 ng (38.2 ng/uL) —> 8uL
T4 L,Buffer
2uL
T4 ligase
1uL
Water
6uL
TOTAL
20 uL
Waited 40 min on the bench (25°C)
Goal 3
Transformation of L*
Procedure
C-
C+
T*
ChemicallyCompetent Cells
50 uL
50 uL
50 uL
L*
-
-
3 uL
pSB1C3 with RFP 100ng/uL
-
3 uL
-
iGEM Transfo protocol
90 sec heat shock
On saturday they were growing but no fluorescence
August 10th
Goal 1
Colony PCR my T*
Procedure
Pick up 16 colonies on my two T* plates
Put some bacteria in PCR tube + 2 uL of 148 and 149 (iGEM verification forward and backward) ;
10uM + 50 uL of Master Mix Phusion (last in the tubes) + 45 uL of water (first in the tube)
Waiting for a 1577 bp fragment on a gel  Gel migration with a GR 1kb
It is quiet bad
August 11th
Goal 1
Do a new PCR of FINAL (G-block) and of pSB1C3 associated with RFP
Procedure
Template
Primers
Master Mix Phusion
Water
Time of elongation (72°C)
T°C annealing
Lenght
Primers
Nanodrop after purify (ng/uL)
LP1*
5ng
2uL
25uL
QSP for 50uL
2 min
56°C
2078
152 - 153
68.3
LP2*
10ng
2uL
25uL
QSP for 50uL
2 min
56°C
2078
152 - 153
133.5
LP3*
2ng
2uL
25uL
QSP for 50uL
2 min
56°C
2078
152 - 153
127.8
F1*
2ng
2uL
25uL
QSP for 50uL
1 min
58°C
1314
150 - 151
98
F2*
3ng
2uL
25uL
QSP for 50uL
1 min
58°C
1314
150 - 151
102
F3*
1ng
2uL
25uL
QSP for 50uL
1 min
58°C
1314
150 - 151
58
Results
Gel migration wit 1kb GR
Goal 2
Digestion of pSB1C3 + RFP
Procedure
Backbone+RFP
Plasmid (100ng/uL)
10uL
EcoRI
2uL
PstI
2uL
Buffer
6uL
Water
38uL
FastAP
2uL
TOTAL
60uL
Results
Gel migration : order = 1kb GR - Digested product x3
Then I cuted the 2100 fragment and did the gel extraction protocol
Goal 3
Digestion of pSB1C3 linearized (LP1,2,3 *) and F1,2,3 *
Procedure
F2d
LP2d
LP3d
ADN
10 uL
10 uL
10 uL
EcorRI
2 uL
2 uL
2 uL
PstI
2 uL
2 uL
2 uL
Buffer
4 uL
6 uL
6 uL
Water
22 uL
38 uL
38 uL
FastAP
_
2uL
2uL
TOTAL
40
60
60
Then 30min at 37°C
Goal 4
Ligations and transformations
Procedure
L1
L2
L3
Vector 100ng
LP2d (40.1ng/uL),2uL
LP3d (21.1ng/uL),4 uL
Backbone from digestion + electrophoresis extraction (9.1ng/uL), 10 uL
Insert 300 ng (8.1ng/uL) (F2d)
30 uL
20 uL
10 uL
T4 ligase buffer
4 uL
4 uL
2 uL
T4 ligase
2 uL
2 uL
1 uL
Water
0
10 uL
0
TOTAL
40 uL
40 uL
20 uL
Waited 20 min on the bench after mixing to do the ligation
Used 3 uL for the transformations with 50 uL of competent cells
(iGEM protocol) Waited 15 min instead of 30 min after put the ligation product (on ice)
Heat shock 1 min 30
2 hours recovery
Plated C- (just chemically competents cells Heat Shocked) C (LP2 dp ! no ligation) L1, L2, L3
Results
Grew on L1, L2, L3 ; nothing on the two others
Number of colonies :
L1 ++ (50)
L2 ++ (50)
L3 +++ (100)
August 12th
Goal 1
Ligation of LP2 alone and transfo for a negative control
Procedure
Vector : LP2 (40.1ng/uL) 2 uL
No insert
T4 ligase buffer : 4 uL
T4 ligase : 2 uL
Water : 32 uL
TOTAL : 40 uL
Transfo : use 3 uL of ligated product
Result after culture on LB + Cm : nothing for the C and lot of colonies for L2
Goal 2
Colony PCR on yesterday transformations plates
Procedure
Using primers 148 - 149 (10uM), 2 uL each
Dream Taq Master Mix : 10 uL
Water : 6uL
TOTAL : 20uL
Colony n°
1
2
3
4
5
6
7
8
9
10
11
12
13
Comes from
L2-1
L2-2
L2-3
L2-4
L2-5
L1-1
L1-2
L1-3
L1-4
L3-1
L3-2
L3-3
L3-4
Results
Nothing : ?
350 bp : recircularization (expected : 320 bp )
Expected = 1700 ! —> nothing
August 13th
Goal 1
Digestion
Procedure
I will digest F1* this time and try to insert it in LP2* digested too
F1*d
LP2*d
ADN
2 ug (60,6 ng/uL) = 35 uL
2 ug (108,3 ng/uL) = 20 uL
EcorI
4 uL
4 uL
PstI
4 uL
4 uL
FastAP
-
4 uL
Buffer
8 uL
12 uL
Water
29 uL
76uL
TOTAL
80 uL
120 uL
Then incubated 1 h at 37°C
Goal 2
Ligation
Procedure
L4
LC (control, no insert)
Insert
195 ng (35 ng/uL) = 6 uL
-
Vector
100 ng (35ng/uL) = 3uL
3 uL
Ligase
1 uL
1 uL
Ligase Buffer
2 uL
2 uL
Water
8 uL
14 uL
TOTAL
20 uL
20 uL
Then waited 40 min on the bench
And transformed 2uL of each tube in 50 uL of chemically competent cells.
Then plated 3 petri dish (Cm 25ng/uL : 100 uL of L4; 150 uL of L4; 100 uL of LC
37°C overnight.
Result
Number of colonies : 100 uL of L4 = 200 ; 150 uL of L4 = 300 ; 100 uL of LC = 3 The control is showing that my T4 colonies might have insertions
Goal 3
4 colony PCR on 24 colonies from L2 done yesterday with the control
Procedure
Made 4 tubes with 1 mL of water and introduced 6 colonies per tube in order to be able to do more screening. Let the tubes on the bench 1 hour (osmotic shock), then centrifugation 5 min 14 K rpm. Then took 8 uL of each suspension and put it in 4 PCR tubes.
PCR1
PCR2
PCR3
C
Colonies
1-6 ;8 uL
7-12 ; 8 uL
13-18 ; 8 uL
19-24 ; 8 uL
Master Mix
10 uL
10 uL
10 uL
10 uL
Primers 148 - 149
1 uL x 2
1 uL x 2
1 uL x 2
1 uL x 2
Water
6 uL
6 uL
6 uL
6 uL
TOTAL
20 uL
20 uL
20 uL
20 uL
Result = nothing on the electrophoresis migration.
It might be a problem in the protocol, maybe 1mL is to much and it’s over diluted so not enough template ?
Maybe the cells didn’t exploded ?
August 14th
Goal
Colony PCR (16 colonies from T4)
Procedure
Suspended T4 colony in 20 uL of water
6 uL
Primers 148 - 149
1 uL x2
Master Mix Phusion
8 uL
TOTAL
16 uL
x 16
Anealing 51°C
Elongation time : 1:30 min
35 cycles
Selected colony 6 for overnight culture in LB broth + Cm25
Also colony 5 for a control
The electrophoresis shows a bad level of insertion ( It is strange considering that my control - had just 3 colonies growing )
Why 2 bands in the 6th colony ? Maybe multiple plasmid insertion some with insertion, some without.
What to do?
Work on the 6th colony
Change my restriction enzymes
Use DpnI
August 17th
Goal
Miniprep and digest colonie 6 and 5 for control
Procedure
Using Promega SV Miniprep Kit
Nanodrop after
6 = 55 ng/uL
5 = 51 ng/uL
Digestion : Hind III enzyme (1 digestion site in my construction, inside my insert)
5
6
Hind III
4 uL
4 uL
FastAP
4 uL
4 uL
FD buffer
8 uL
8 uL
DNA (50ng/uL)
20 uL
20 uL
Water
44 uL
44 uL
TOTAL
80 uL
80 uL
Mix, Incubate 30 min 37°C PCR purification
August 18th
Goal1
New digestion
Procedure
F2d'
F3d'
LP1d'
LP3d'
DNA
55ng/uL = 20uL
76 ng/uL = 15uL
76 ng/uL = 15uL
82ng/uL = 15uL
FastAP
2uL
2uL
EcoRI
2uL
2uL
XbaI
2uL
2uL
SpeI
2uL
2uL
2uL
2uL
FD buffer
4uL
4uL
6uL
6uL
Water
12uL
17uL
33uL
33uL
TOTAL
40uL
40uL
60uL
60uL
Nanodrop after purify
41 ng/uL
21 ng/uL
16 ng/uL
68 ng/uL
20 min at 37°C
Goal2
Ligation
Procedure
La (F2d’-LP1d’)
La c
Lb (F3d’-LP3d’)
Lb c
F2dp’
6uL
LP1dp’
7uL
7uL
F3dp’
15uL
LP3dp’
2uL
2uL
Ligase
1uL
1uL
1uL
1uL
L buffer
2uL
2uL
2uL
2uL
Water
4uL
10uL
15uL
TOTAL
20uL
20uL
20uL
20uL
Transformed 5uL in 50 uL of competent cells - Heat shock Then plate it
August 19th
Results
Yesterday plates : La (250uL) : 50
Lb (250uL) : 0
La c (50uL) : 1 colony
Lb c (50uL) : 1 colony
Nothing but competent cells : NOTHING
Goal
Colony PCR
Procedure
Cells in 10 uL of water
2 uL
Primers
1uL x 2
MasterMixDreamTaq
10 uL
Water
6 uL
Total
20 uL
Expected 1577bp
Result
Then m iniprep of 22, 23 (+24?), 24
Also plated LB + Cm
And liquid culture
August 21th
Goal
Digestion of my mini prep from yesterday
Procedure
Nanodrop :
22 = 23.5ng/uL
23 = 53.7ng/uL
24 = 25.5ng/uL
22
23
24
DNA
16
16
16
Buffer
2
2
2
Hind III
1
1
1
Nco I
1
1
1
TOTAL
20
20
20
Expected length
2040 bp (no insertion, recircularisation)
1315 and 2026 bp (insertion)
2040 bp
37°C 20 min
Gel order : GR1kb , 22, 23, 24, GR1kb
Exactly what I was expecting
Band 23 : 3700 = plasmid undigested ??
New gel
Order : GR 1kb, 23, 23 d, 24, 24d
PS : 23 is two bands now maybe because the digestion is now finished (it was on the bench for one hour)
23 sent to sequencing
August 31th
Goal 1
PCR for my Gibson 1 = Replace the RBS and first codon of the RFP in the classical pSB1C3 associated with Part:BBa J04450 by the riboswitch sequence
Procedure
PCR
Insert
Backbone
Template
1ng = 2uL
1 uL
1ng = 2uL
1uL
Primers
209-210
1uL
210-211
1uL
MasterMix
25 uL
25 uL
Water
23 uL
24 uL
23 uL
24 uL
TOTAL
50uL
50uL
Elongation time
30 sec
2 min
Expected length
243
3149
Results
Gel order : GR 1kb, B1, B2, I1, I2
Bad result for B1 , B2
Goal 2
New PCR for the backbone
Procedure
B2
B3
B4
Template
3uL (ng)
3uL (ng)
3uL (ng)
MasterMixPhusion
25 uL
25 uL
25 uL
DMSO
1uL
1uL
1uL
Primers
1uL
1uL
1uL
Water
20 uL
20 uL
20 uL
T°C annealing
56°C
58°C
60°C
Result
So bad
September 1st
Goal 1
PCR FINAL for a new transformation
Procedure
F1
F2
F3
Template
2ng
5ng
10ng
Primers 150-151
2uL
2uL
2uL
MasterMix
25uL
25 uL
25 uL
Water
19 uL
16uL
11uL
TOTAL
50 uL
50 uL
50 uL
T°C gradient : 0 - 59°C
1 - 58.8°C
2 - 58.4°C
3 - 57.9°C
4 - 57.2°C
5- 56.6°C
Result
Gel order : GR1KB-10-11-12-13-14-15-   GR1KB-20-21-22-23-24-25-   GR1KB-30-31-32-33-34-35
Goal 2
Gibson 1 - New PCR
Procedure
B5
B6
B7
I3
I4
I5
Template
1ng
3ng
6ng
1ng
3ng
6ng
Primers
2uL
2uL
2uL
2uL
2uL
2uL
MasterMixPhusion
25uL
25uL
25uL
25uL
25uL
25uL
Water
20 uL
20 uL
20 uL
20 uL
20 uL
20 uL
TOT
50 uL
50 uL
50 uL
50 uL
50 uL
50 uL
Primers used
211-212
209-210
T°C annealing
69°C
66°C
Elongation time
2min
15sec
Results
Gel order :  - - - B5 B6 B7 I4 I5 ( I3 was lost )
September 2nd
Goal 1
Digestion/Ligation
Procedure
The PCR product was purified before.
Digestion :
F11d
pSB1C3d
DNA 1:3
55ng/uL, 1.5 ug —> 30 uL
96 ng/uL, 0.75 ug —> 8 uL
Fast AP
2uL
EcoRI
2uL
2uL
SpeI
2uL
2uL
FD Buffer
4uL
6uL
Water
2uL
40uL
TOTAL
40uL
60uL
Nanodrop after purif
8.6
11 (after gel extract)
I extracted the 3kb band from the gel
Ligation :
La (F2d’-LP1d’)
La c
F11d
25 uL
pSB1C3d
10uL
10uL
Ligase
1uL
1uL
L buffer
4uL
4uL
Water
25uL
TOTAL
40uL
40uL
Then on the bench for 1 hour
Goal 2
Back to my Gibson 1
Procedure
~ 60 ng of backbone in ~ 5 uL for assembly so concentrations as low as 12 ng/uL are usually fine.
G11
G12
B5
100ng — > 3uL
100ng — > 3uL
I5
7.7ng (equimolar) —> 1uL
I2
7.7ng (equimolar according to NEB Bio Calculator) —> 1uL
GibsonMix
15uL
15uL
Water
1 uL
1 uL
TOTAL
20uL
20uL
Then 60 min to 50°C
Then I did a transformation by Heat Shock using 2 uL of the product and I plated it.
September 3rd
Result
Result of my plating from yesterday
G11: 200uL —> 300 colonies (3 were red)
20 uL —> 40 colonies (2 red)
G12 : 200uL —> 30 colonies (9 were red)
20uL —> 3 colonies (1 red)
Goal 1
Colony PCR (on colonies that don’t show any red fluo)
Procedure
16 colonies :  Master Mix dream Taq 10uL , Primers Fwd 148 1uL , Primer Rv 149 1uL, Water 8uL , TOTAL 20uL
1 min 45 elongation expected length 1566; T°Cannealing 52°C; 95°C 8min first
Result
Colony 1 to 16, GR 1kb between 8 and 9
Liquid culture in M9 (without B12) with Cm 25 for colonies 3, 4, 5, 7, 8, 9 , 10
Goal 2
Colony PCR on the red colonies
Procedure
Master Mix dream Taq 10uL , Primers Fwd 148 1uL , Primer Rv 149 1uL, Water 8uL , TOTAL 20uL
Result
Colony 11’ is quiet interesting, the 3' too
Liquid culture in M9 Cm25 of 1’, 2’, 3’, 11’ without B12 and with 5 mg/L of adenosylcobalamin and in LB Cm25
September 4th
Goal 1
PCR my new G-block for Gibson 2
Procedure
R (Riboswitch)
B (pSB1C3)
Template DNA
2ng
2ng
Primers 1 (10uM)
2uL (213)
2uL (215)
Primers 2 (10uM)
2uL (214)
2uL (216)
Phusion Master Mix
25 uL
25 uL
Water
19 uL
19 uL
TOTAL
50 uL
50 uL
T°C annealing
67°C
66°C
Time elongation
25 sec
1 min 10
Length
476
3149
Result
Nothing
