Team:Paris Bettencourt/Notebook/Differentiation
Ferment It Yourself
iGEM Paris-Bettencourt 2O15
Background
Design
Project
Micronutrients
Continuity
Modeling
Manufacturing
Achievements
Results
Parts
Medal Fulfillments
Practices
Overview
SynBio Challenge
DIλ
Acceptance
Events
Rhizi
Collaborations
Safety
Safety Overview
Safety Form
Review Board
Notebook
Daily notebook
Protocols
Bibliography
Team
Members
Attributions
Acknowledgments
Press
Official Team Profile
Notebook - Differentiation
July
August
September
Advancement on E. coli
The whole brainbow system will be integrated chromosomally at the GalK site. It will be synthesized as three gBlocks.
It should allow expression of mCherry first, then upon expression of the CRE recombinase, differentiation in two states (YFP and CFP).
It also contains a landing pad that allow insertion of a third state, which would subsequently lower the probability of states 1 and 2 so the three outcomes are possible.
Writing the artificial gene
Promoter J23199 from the Biobrick collection
4 LoxP sites used together in mammalian brainbow plasmids
RBS from Ihab
ORF for mCerulean and mVenus from Ihab
ORF for mCherry from Antoine
rrnBT1 terminator used on the pOSIP plasmids
Landing Pad (PhiC31 attB TT site)
Check for secondary structure in the RBS Done
Check for RBS in the LoxP array Done
Split to have two ~1500 bp gblocks Done
Design overlaps for Gibson in R6K vector Done
Check for Biobrick restriction sites Done
Fix gBlocks problems
Palindroms between LoxP sites Done
Repeats in terminator: replaced with Lambda T0 terminator Done
Repeat in RBS Done
More GC in the LoxP array Done
More GC at the end of fragment 1Done
Repeat at the end of mVenus and mCerulean Done
More GC at the beginning of fragment 2 Done
It is very difficult to solve these -> make 3 fragments Done
Fragment A: 0 - 1143
Fragment B: 1105 - 2006
Fragment C: 1981 - 2946
Change the RBS for Fragment A -> “New RBS” Done
Order gBlocks Done 07/13
Design + order oligos for gBlocks amplification Done
Overlaps melting points are 62, 67, 54, 74.
The overlap between fragments B and C should be increased to >62.
R6K LinR
tagcattatacctaggactgagctagctgtcaaggcaaatttgcggccgcaag
o15.80
R6K LinF
CGGGCGCGTACTCCAgaagggcatcgatggc
o15.81
Colibow A F
ttgacagctagctcagtcctag
o15.82
Colibow A R
GGCCATTCACATCACCATC
o15.83
Colibow B F
GCCGATTCTTGTTGAACTTG
o15.84
Colibow B R
CCATGGTACCTCCTCCTTACTTCTATAACTTC
o15.85
Colibow C F
GATACTTTATACGAAGTTATAGAAGTAAGGAGGAG
o15.86
Colibow C R
gccatcgatgcccttcTGGAGTACGCGCCCG
o15.87
Overlap melting points: 62, 67, 62, 72.
Design + order oligos for cassette sequencing Done
>50 bp before the interest region
800 bp max contigs
Obtain Pir+ strain Done
Overnight of Pir+ strain Done
Glycerol of Pir+ strain Done
Cloning inside the replication vector
Order oligos linR and linF Done
Obtain R6K vector from Ihab Done
Overnight culture of the R6K vector propagation strain
        Failed New attempt with less harsh growth conditions. Done
Only 20 ug/ml of Kanamycine
6 ul of Thyamine in 3 ul of LB
Culture at 30°C
Miniprep of the R6K vector Done
Glycerol of R6K strain Done
Linearization of the R6K vector by PCR Done
Primer linF:
CCCTTGGGCTCCCCGGGCGCGTACTCCAgaagggcatcgatggc
(28 bases: longest possible overlap without having a hairpin at 50°)
Tm = 55°
Primer linR:
tagcattatacctaggactgagctagctgtcaaggcaaatttgcggccgcaag
(33 bases overlap)
Tm = 62°
Product length: 2276 bp
Synthetize, reconstitute and dilute primers Done
Run gel for size checking Done
PCR purification Done
Obtain gBlocks Done
PCR of colibow fragments A, B and C Done
PCR purification of amplicon Done
Obtain Gibson mix Done
Make overnight culture of Pir+ strain Done
Make electrocompetent cells out of the Pir+ strain Done
Gibson assembly of 3 gblocks with the R6K backbone. Done
The vector can only replicate in Pir+ strain. Transform into Pir+ strain Got colonies
Check transformant: colony PCR before culture Failed
Liquid culture + miniprep
Analytical digestion or sequencing (find enzymes)
SeqF1
o15.88
cttagtacgttagccatgagg
SeqF2
o15.89
CTAATTTTCCATCTGATGGCC
SeqF3
o15.90
CAAGCTCACGCTCAAATTC
SeqF4
o15.91
ACAACCATTACCTGTCGACG
SeqF5
o15.92
CGACATTAGGGTATGGGCTG
SeqR1
o15.93
TATAAACATTATGGCTATTATAG
SeqR2
o15.94
TTTAGAGAGTTTTGACTGCG
SeqR3
o15.95
TTTGCCAGTCGTACAGATGAA
SeqR4
o15.96
TCTTCTTCTGCATCACCGGGC
Chromosomal integration with GalK
PCR of the purified R6K plasmid. Done
PCR-linerization of the R6K backbone
Miniprep du backbone R6K (20 ng/ul) -> 5 ul in PCR
Primers: 15.80 (LinR) et 15.81(LinF)  -> 1 ul each
Eau qsp 25 ul
Master mix 25 ul
1’30’’ extension, 50°C annealing.
Find oligos (already there) Done
IntR:
GTTTGCGCGCAGTCAGCGATATCCATTTTCGCGAATCCGGAGTGTAAGAACCATATGAATATCCTCCTTAGTTCCTATTCCG
Alternatively, with only one binding site:
GTTTGCGCGCAGTCAGCGATATCCATTTTCGCGAATCCGGAGTGTAAGAAttagccatggtccatatgaatatcctccttag
IntF:
TTCATATTGTTCAGCGACAGCTTGCTGTACGGCAGGCACCAGCTCTTCCGGGTTGAACTGCGGATCTTGCGGC
Italique: Homology region with E. coli GalK site.
Length: 4908 bp
Attention, autre site potentiel d’amorçage, produit de 3515 bp
Purification
Overnight culture de E. coli Lambda Red
Competent cells out of Lambda Red strain
Transformation of E. coli with pKD46 that carries Lambda Red recombinase
Transformation of the Colibow PCR product
Function testing
Tecan + Flux cytometry
CRE recombinase expression
pFHC2938 should work.
It expresses CRE and has a temperature sensitive ORI (30°C)
Monday 07/06
Research about the synthetic integron
It might make the landing pads better than with brainbow because the recombination site is always the same.
Plate culture of E. coli pIT5-KL and pIT5-KH
Very important: grow @30
-> Make a liquid culture of each and freeze at -80
-> Miniprep from pIT5-KL for first construct
These two strains produce the vector plasmids for clonetegration. They have to be linearized with EcoR1 and Pst1, and then Gibson assembled to get the self-integrating plasmids.
The integrase they carry is expressed only at 37 degrees.
They are resistant to Kanamycine.
KH integrates into the HK022 phage’s site, while KL integrates into the Lambda phage site. We will probably only need one of them but I took both just in case. If needed there is a whole collection of such vectors.
Plate culture of E. coli pE-FLP
Grow @ 30
This strain produces the pE-FLP plasmid, which expresses the flippase. It will be useful to remove the backbone after the clonetegration. It’s resistant to ampicillin.
It has a temperature-sensitive ORI and will disappear if grown at 37.
New primers
LC81, LC85, LC82, LC86: used in the PCR to check integration with pIT5 KH.
LC81, LC87, LC83, LC84: used in the PCR to check integration with pIT5 KL.
Thursday 07/09
Prepared 10 tubes of 1 ml Kanamycine, 50 mg/ml
Inoculated pIT5-KL and pIT5-KH E. coli in 2 ml LB Kan+50
Inoculated pE-FLP E. coli in 2ml LB Amp+100
Cultured these three tubes @ 30
Monday 07/20
Reception of two plates from Ihab
Shortened R6K vector -> Kan+ and thyamine auxotrophy. Thyamine 1000x is available in an eppendorf tube.
Pir 116 strain for propagation of the vector.
-> Overnight culture for miniprep for PCR and gleezing
R6K in LB Kan Thyamine (in antibiotics box)
Pir116 in LB with a control tube
Started cultures for Colibow
pThy 1.0 aka pBrainbow 1.0 in 3 ml LB-Amp -> Miniprepcr
pR6K in LB-Kan-Thyamine -> Miniprepcr and Glycerol freezing
Pir116 for replication of pR6K-vectors -> LB
Control without innoculation -> LB
All of them, culture @ 37° overnight.
No need to culture pKT174 because we already have the purified version. It may be a good idea to transform DH5a with it in case we run out of plasmid.
Reception of oligos o15.76 to o15.96
Yeastbow SOE
R6K linearization
Colibow gBlock Amp
Colibow Sequencing
The ultramers (o15.78, o15.79) for Leu2 flanking are not ready yet.
-> Oligo reconstitution and dilution
-> Miniprep of R6K vector
-> PCR of R6K vector
-> Miniprep of pBrainbow 1.0
-> PCR of pBrainbow 1.0
-> PCR of pThy Ura3
Reconstituted all primers to 100 ug/ml.
Miniprep of pR6K and pBrainbow 1.0
For PCR, using Promega kit w/ double wash. Elution in 30 ul
Final concentration: 93 ng/ul
Re-start of the R6K culture
The first culture failed: let’s try again with less harsh conditions.
Only 20 ug/ml of Kanamycine
6 ul of Thyamine in 3 ul of LB
Culture at 30°C
Wednesday 07/22
PCR-linerization of the R6K backbone
Miniprep du backbone R6K (20 ng/ul) -> 5 ul in PCR
Primers: 15.80 (LinR) et 15.81(LinF)  -> 1 ul each
Eau qsp 25 ul
Master mix 25 ul
1’30’’ extension, 50°C annealing -> 2276 bp
Bad news from IDT about the gene synthesis
« I would like to notify you about a best effort for gBlock ‘Colibow C’. Even after multiple attempts at trying to correct this issue, the Final QC data for the best prep is not meeting our quality standards. The final prep (mfg# 188635671) has desired peak.  A secondary peak is also present. The target mass was found in the chromatogram, but not as the most abundant product in any of the chromatographic peaks for mass spectrometry, and the sequence has been verified.
If you are using this sequence for cloning and/or qPCR you should not have any problems with this sequence. It may require you to screen additional colonies. However, if your intentions with using this gBlock was for other purposes, we might need to cancel the gBlock and redesign. I have provided the trace below. »
Thursday, 07/23
Glycerol stock for pR6K
1 ml of overnight culture of E. coli pR6K.
1 ml of glycerol.
-> -20°C
Gel for linearized R6K pcr
1% agar TAE, with 1 kb+ generuler.
5 ul PCR product + 1 ul LB.
-> band at the right size (2276 bp) , the PCR worked.
linearized R6K pcr cleanup
Using the Qiagen kit, taken back in 50 ul water
Titration: about 80 ng/ul, but very bad 260/230 ratio (presence of organic molecules).
Pir116 electrocompetent for bowcoli transformation
2 ml overnight culture in 100 ml LB, at 37°C w/ shaking.
When OD reaches 0.6, they were put in ice for half an hour.
Electrocompetent cells were made by Mukit along with DH5a competent cells.
Tuesday, 07/28
Reception of gBlocks Colibow A, B, C
Reconstitution to the concentration of 10 ng/ul (from spec sheet):
Centrifuge @ 11kG
Add 100 ul of RNase-free water
Vortex
Incubate at 50°C for 20 minutes
Vortex/Centrifuge
PCR of Colibow gBlocks
Colibow A
Primers: 82 (56°) + 83 (54°) -> 1172 bp
Colibow B
Primers: 84 (53°) + 85 (54°) -> 909 bp
Colibow C
Primers: 86 (54°) + 87 (57°) -> 992 bp
Reaction in 50 ul:
Compound
Volume (ul)
Water
19
Phusion 2x
25
Primer 1
2.5
Primer 2
2.5
gBlock diluted 10 times
1 ul
Program:
98 (30)
98 (10) 58 (25) 72 (45) x35
72 (600)
12 (hold)
In parallel from running the gel, the PCR products were purified with the QIAGEN kit and eluted in 40 ul of water.
Titration
Fragment A: 90 ng/ul
Fragment B: 88 ng/ul
Fragment C: 85 ng/ul
Gel plan
1% agar, SYBRsafe, 5 ul of PCR product + 1 ul of LD.
The 100 bp+ marker was used.
The apparatus didn’t work. It stopped two times, the gel stayed to diffuse in the tank for 30 minutes. At the end it was imaged even though it was far from finished.
Ladder 100 bp+
Colibow A
Colibow B
Colibow C
Expected size
1172
909
992
Conclusion
There are a lot of non-specific binding.
Due to the awful quality of the gel, it’s not possible to determine their size or which bands are the good ones. The top ones seem more consistent regarding their relative sizes.
To do:
Run the whole purified PCR product in a gel and perform a gel extraction. Hopefully the right band can be determined this time.
Try the PCR again with a higher annealing temperature.
Gibson assemble directly the gBlocks fragments.
Wednesday, 07/29
Gel for Colibow A, B, C and extraction
Agar 1%, SYBRsafe, TAE
Ladder 100 bp+
Colibow A
Colibow B
Colibow C
Expected size
1172
909
992
Colibow Amp Extracted.jpg
For each sample, the top band was extracted. The Colibow B sample was extracted twice: Bg (grande, top band) and Bp (petite, bottom band).
The bottom band is the right one.
-> add DMSO 3% next time
Titration (in 30 ul EB)
Name
1
U
A
Bp
Bg
C
C (ng/ul)
13
41
6
29
11
12.1
The concentrations of Colibow A and C are critically low. It is necessary to fix the PCR and reduce the non-specific priming before performing the Gibson.
New attempt at Colibow A and C PCRs
Mix
Phusion        25
Colibow A        1
o15.82                2.5
o15.83                2.5
Water                19
DMSO                1.5         (3%)
= Two tubes of 50 ul each -> 1172 bp
Mix
Phusion        25
Colibow C        1
o15.86                2.5
o15.87                2.5
Water                19
DMSO                1.5         (3%)
= Two tubes of 50 ul each -> 992 bp
This is stored in 4° for now (29/07)
Received Gibson assembly mixes from Ihab
I need better quality products before doing it.
Thursday, 07/30
Gel for A, C and 1
Sophie’s sample
A
A
A
C
C
C
100+
1
1
1
1kb
1172
1172
1172
992
992
992
3965
3965
3965
50 ul sample + 10 ul LD -> 40 ul in each well (3 wells)
Attention C sample accidentally added to the well containing 100 bp + ladder !!!
Colibow A et C amp 29.07.jpg
Gel extraction
With Qiagen kit, product recovered in 30 ul of water.
Name: “product” X+ 30/07
The C product mixed with ladder was labeled Cl.
Titration
Sample
1
A
C
Cl
c (ng/ul)
5.2
15.4
12.9
4.4
Gibson assemblies of Colibow
General mix:
15 ul of Gibson supermix (MMII)
10-100 ng of backbone for a 6 kb fragment
Equimolar amount of DNA fragments
Total: 20 ul
Colibow PCR products
Using the most concentrated PCR products known to date.
Nom
Taille (bp)
Concentration
Volume (ul)
Quantité (ng)
R6K pcr
2276
80
0.6
50
Colibow A+ e (in water)
1172
15
1.7
25
Colibow Bp
(in EB)
909
29
0.9
25
Colibow C+ e
(in water)
992
13
1.6
25
Colibow gBlocks
Using directly the gBlocks from IDT dna synthesis
Nom
Taille (bp)
Concentration
Volume (ul)
Quantité (ng)
R6K pcr
2276
80
0.5
30
Colibow A
1172
10
1.5
15
Colibow B
909
10
1.5
15
Colibow C
992
10
1.5
15
Incubation during one hour at 50°C.
3 LB ampicillin plates
“30/07 ANTOINE”
50 ml of hot LB-agar.
Transformation of Colibow in Pir116 by Electroporation
pColibow gBlock
pColibow PCR
Biobrick plasmid with RFP (3 ul of 10 pg tube, not dialysed)
Protocol from OWW:
Thaw cells from -80°C to ice for >20 min
Chill cuvettes
Dialyse 6 ul of Gibson products on dWater during >20 min
Mix 6 ul of dialysed DNA with 50 ul Pir116 electrocompetent cells
Mix with tip
Pulse (1.5 kV, 2 mm cuvette)
Add 1 ml LB (at 18h43)
Incubate for 1 hour at 37°C.
2 Kanamycine 25 plates for each colibow plasmid: 100 ul and 900 ul
1 Chloramphenicol plate for RFP control
Incubation overnight @37.
Transformation of pKT174 in DH5a by Heat shock
Protocol from addgene:
Thaw cells on ice (20 min)
3 ul DNA + 50 ul chemically competent cells, mix gently
20-30 min on ice
45s at 42°C
2 min on ice
Add 1 ml LB (at 18h50)
Incubate 1 hour @ 37°C
2 ampicillin plates: 100 ul and 900 ul.
Incubation overnight @37.
Gel for SOE 29/07
1 kb ladder
SOE
SOE
SOE
1 kb ladder
Expected size: 5600 for all of them.
SOE+29.07.jpg
It didn’t work at all. Try to Gibson-assemble them.
New electroporation of Pir116
Using the old electroporator (with the square cuvette holder).
Dialysis of 6 ul and sampling of 5 ul of Colibow gBlock assembly
Dialysis of 5 ul and sampling of 5 ul of 5 pg/ul
Compound
x1
x8
10x Taq Buffer
2.5
20
10 nM dNTPs
0.5
4
10 uM primer 90
0.5
4
10 uM primer 94
0.5
4
taq polymerase
0.125
1
water
17.875
167
Then 22 ul of mix is added to 3 ul of template (2 ul water + 1 ul sample for positive control). Template is obtained by soaking part of a colony in 20 ul water.
Program (saved as Colony):
95 (6’00)
95 (15) 49 (25) 68 (45) x30
68 (5’00)
Gel:
P900        B1        B2        B3        ColibowB        100bp+
Results:
One band at 508 bp on the positive control.
No band at all on the colonies
-> They are contaminations, the transformation did not work, probably because the Gibson itself did not work due to bad DNA concentration.
Wednesday, 8/5
Pir116 electrocompetent cells
From the overnight culture, 1 ml of cells were diluted in 50 ml of LB and incubated at 37°C (10h30).
Centrifuge steps were done 10 min at 8500 rpm.
50 ml culture -> centrifuge and remove well the supernatant
50 ml glycerol 10% -> centrifuge
50 ml glycerol 10% -> centrifuge
Cells in residual glycerol were aliquoted in eppendorf tubes (6 in the end) and frozen @-80.
Testing of these EC Pir116 cells function
4 ul of pSB1C3 20 pg/ul were dialysed, 3 ul were picked after 20 min and mixed with 100 ul of cells.
Pulse duration: 5.7 ms
After 1 h of preculture @37, it was plated on a chloramphenicol 0.5 plate and grown @37.
New colibow PCR
Performed by Chloé and Émilie.
1438872081.png
Program:
98 (30)
35x 98 (10), Gradient (30), 72 (2’20)
72 (5)
10 (hold)
Gradient:
59, 57.8, 55.3, 53.4
The very long extension time is used to avoid promoting small non-specific fragments.
Gel:
1% agarose, SYBRsafe, 2.5 ul sample + 0.5 ul LD
L
A
A
A
A
L
B
B
B
B
L
C
C
C
C
L
1 kb
1172
1 kb
909
1 kb
992
08.05 Colibow Amp gradient.jpg
Thursday, 8/6
PCR purification of Colibow gBlocks
The homologous tubes were mixed together, except for C2 that didn’t work.
Elution in 50 ul of water.
A second gel was ran in order to know whether the light band at the bottom is still present.
1 kb ladder
A
B
C
Is it. This has to be taken into account when calculating the concentrations for the Gibson assembly.
Titration (in 50 ul of water):
A: 137 ng/ul
B: 167 ng/ul
C: 99 ng/ul
08.06 Colibow gradient after purification.jpg
New R6K PCR
Compound
1x
2x
water
71
142
Phu buffer
20
40
dNTP
2
4
80 primer
1
2
81 primer
1
2
pR6K shortened (template)
1
2
DMSO
3
6
Phusion polymerase
1
2
Program:
98 (30)
98 (10) 52 (30) 72 (1’30) x 35
72 (5’)
Problem: There was only <1 ul of phusion left in the tube. The PCR did not work at all.
New attempt
Mix (made twice)
Phusion master mix        50 ul
80 primer                2 ul
81 primer                2 ul
pR6K                        3 ul
DMSO                        3 ul
Water                        40 ul
Program:
98 (30)
98 (10) 50 (30) 72 (1’30) x 35
72 (5’)
After this PCR, the product was:
ran on a gel: Ladder 1kb, tube 1, tube 2 (3 ul sample, 2 ul water, 1 ul LD)
digested by DPN1:
1 ul of DPN1 added directly to the product, then incubated for 15 minutes at 37° in the thermocycler and inactivated (5 min at 80°).
Gel results: [08.09 R6K pcr]
Monday, 08/10
PCR purification of R6K amp
With QIAGEN kit, performed by Chloé & Émilie. At the end, the product was recovered in 50 ul of water and also 30 ul EB, for 80 ul total.
The titration was done using a 5 ul + 3 ul mix as a blank.
Products summary:
R6K pcr
67.4 ng/ul
2276 bp
Colibow A
137
1172 bp
Colibow B
167
909 bp
Colibow C
99
992 bp
A and C have a small band at ~200 bp, so the actual concentration of the product is half of what indicated by the Nanodrop.
Gibson assembly of Colibow
Assuming half the DNA in A and C is the right one.
Name
Volume (ul)
Final amount (pmol)
A
1.11
0.10
B
0.4
0.11
C
1.30
0.10
R6K
2.19
0.10
Added to a MMII Gibson assembly master mix and incubated during 1h at 50°C.
Gel for Colibow’s Gibson
6 ul SYBRsafe in 20 ml agar 1%, with 1 kb ladder.
The sample consisted in 10 ul of Gibson product and 2 ul of LD.
Unfortunately the comb for making the wells went through the gel, resulting in loss of the sample.
Electroporation of Pir116 E. coli with newly assembly pColibow
Dialyse of the assembly product during 20 min. 5 ul were deposed and 5 ul were taken back.
Mixed with already tested Pir116 Electrocompetent cells
Program Ec2, bacterial, in 2 mm cuvette. Pulse time: 4.7 ms.
Incubation 1h @37
Plating on “pColibow Gibson” plates (K50 but K20 written on the box). 100 ul on the first box, centrifuged 900 ul on the second one.
Incubation overnight @37.
Tuesday, 08/11
Pir116 Colibow electroporation results
We got a lot (~300) colonies on the 100 ul plate, and even more on the other plate (too many to distinguish). The colonies don't appear to be red at all. We can distinguish two types of colonies: bigger ones and smaller ones.
Colony PCR for checking colibow transformation
10 big colonies were picked (we're waiting for the small ones to be big enough to be tested) and diluted in 20 ul of water.
x1
x13
10x Taq Buffer
2.5
32.5
10 nM dNTP
0.5
6.5
Primer 90
0.5
6.5
Primer 94
0.5
6.5
Taq
0.125
1.625
Water
17.87
Then, 22 ul mix + 3 ul template. Program: Same as last colony PCR (saved as Colony).
Gel: 1,2,3,4,5,6,7,8,9,10, negative, positive. Negative: No template. Positive: Colibow B diluted gBlock.
[08.10 Colony PCR]
Results: The positive control is positive, but it didn't involve cell lysis. All the colonies are negative, which means that either the cell lysis didn't work, or they are not transformed properly.
Hypothesis: Too much template in R6K PCR, or the DPN1 digestion was not thorough enough.
The clones 1,2,5,6 and 7 were put into culture (4 ml LB Kan+) for further analysis. It included a negative control (un-transformed Pir116).
Gel extraction of Colibow A and C
A|A|1 kb ladder|C|C
[08.10 Colibow A and C]
The bands were cut and extracted with Qiagen kit. Recovery in 30 ul water. As Loading Dye was accidentally mixed with the B sampled, it was PCR-cleaned-up again and recovered in 30 ul water.
Titration:
A : 28.4 ng/ul
B : 212 ng/ul
C : 34 ng/ul
Wednesday 08/12
New new R6K PCR
Mix
Phusion master mix        50 ul
80 primer                2 ul
81 primer                2 ul
pR6K diluted 20 times                        1 ul
DMSO                        3 ul
Water                        42 ul
The goal is to minimize the amount of R6K backbone in the mix, to limit background colonies.
Program:
98 (30)
98 (10) 52 (30) 72 (1’30) x 35
72 (5’)
Results: It did not work (gel not shown).
New new new R6K PCR
Mix
x1
x2
Phusion master mix
50
100
primer 80
2
4
primer 81
2
4
pR6K
0.5
1
DMSO
3
6
Water
42.5
85
Program:
98 (30)
98 (10) 50 (30) 72 (1’30) x 35
72 (5’)
Analysis of colonies from pColibow transformation
All of the clones grew overnight. The negative control did not grow, meaning that the kanamycine we used is efficient and the Pir116 native cells are not resistant.
The cultures 1, 2, 5 and 6 were miniprepped (3 ml + 1 ml saved for Glycerol stock if needed) and recovered in 50 ul of water.
Titers:
I : 96
II : 105
V : 111
VI : 106
The rest was put back into culture at 37°.
NotI digestion of the plasmids
NotI cuts only once in both the pR6K vector and in the pColibow plasmid.
x1
x4
Water
16
64
10x FD Buffer
2
8
Plasmid
1
4
Not1
1
14
Incubation at 37° started at 17h10, stopped at 19h10.
Gel after NotI digestion
1 kb ladder, I, II, V, VI, 1 kb ladder. Expected results: 5190 bp if it's pColibow, 2229 bp if it's pR6K.
Gel for R6K PCR
1B and R6K pcrs were ran on the same gel:
1B | 1B | 1 kb ladder | R6K | R6K
2 ul water + 1 ul LD + 3 ul sample.
Results: [0.13 B1 and R6K]
he R6K PCR yielded two bands: one at around 1000 bp, and one at around 2500 bp. The bigger one is the good one, it could be gel-extracted if needed. However, we should stick to the previous version that will probably be more concentrated anyways due to the absence of off-product.
Thursday 08/13
The new new new new new R6K pcr still did not work. There is a serious problem with one of the things we use because this PCR worked before.
DPN1 digestion of the previous R6K product
As it's no longer possible to even replicate the previous R6K pcr for some reason, I will stick with the small, diluted sample that I have and digest it with DPN1. That's the only thing I can do.
For R6K vol ~ 20 ul DPN1 FD 5ul 10x FD Buffer 3ul
37°C during 30 min, then 80°C for 20 min.
Titration: 24 ng/ul, with a high 280 absorbance due to DPN1 itself. I don't know why the total DNA concentration dropped.
Friday 08/14
Saturday 08/15
R6K PCR, attempt N
Using either purified R6K plasmid or R6K PCR product.
Mix: Master mix, 50 ul F (81), 1ul R (80), 1ul R6K, 1ul DMSO, 3 ul water, 44 ul
Each of them was done in duplicate, with a gradient (annealing temperature of 52 or 55°C).
Program 98 (30) 98 (10) gradient (30) 72 (1'30) * 36 72 (5)
Result: absolutely nothing.
R6K PCR, attempt N+1
Mix: Master mix, 50 ul F (81), 2ul R (80), 2ul R6K, 1ul DMSO, 3 ul water, 44 ul
Same prog, except 50°C annealing, 3'00 extension, 37 cycles.
It didn't work, once again.
3 ml LB + Kanamycine + R6K freezer stock cells for new miniprep, because the template is the only parameter that was not completely new.
Monday 08/17
Preparation for clonetegration
As the R6K setup definitely doesn't work, the working PCRs can't even be reproduce and we're stuck at the first step of a long protocol, I decided to change the strategy completely. Two new primers were order in order to change the vector and use pIT5-KH for clonetegration.
The strain carrying pIT5-KH (u15.39) was put into culture for miniprep in LB Kan+.
New R6K gibson tentative
Using newly digested R6K backbone (poor concentration).
Name
Concentration
Size
Volume
A
28.4
1172
1.22
B
212
909
0.13 (0.5 actually)
C
34
992
R6K
24
2276
0.86
This gives 0.01 pmol of each part.
As a control to measure undigested vector, 2.80 ul of linearized R6K vector + 17.2 ul of water.
gBlocks assembly
Name
Concentration
Size
Volume
A
10
1172
1.18
B
10
909
1.46
C
10
992
1.13
R6K
10
2276
0.86
That's 0.004 pmol of each (barely enough...).
Started at 4 pm, ended at 5 pm and put in ice.
PCR for checking the gibson
To know whether the Gibson Assembly really works or not, a bunch of PCR were made.
Junction 1: seqF1 + seqR4 = 799 bp Junction 2: seqF2 + seqR3 = 625 bp Junction 3: seqF4 + seqR1 = 808 bp Junction 4: seqF5 + jw06 = 829 bp
On 2 ul of Gibson product. Control : R2 + F3 = 508 bp
DT mix
50
Template
15
Water
Then 18 ul of master mix + 1 ul of each primer.
95 (3') 95 (30) 52 (30) 72 (1 min) 72 (5')
Results: 10 ul + 2 ul LD were deposited on a gel. gBlock I, II, III, IV, + ; 100 bp+ ; PCR I, II, III, IV, + [junctions] All of the junctions seem present, but for the PCR sample they are way sharper. Strangely, the control PCR did not work so well.
Tuesday 18/08
Transformation of both Gibson Assembly product
(as at least all the junctions work...). Target: Electrocompetent Pir116 cells. All of the gibson product (~5 ul) was dialysed, along with the control (PCR R6K digested by DPN1 + water). 8 ul of dialyse product were taken back after 20 min. Pulse times: Control: 5.7 ms Gibson gblocks: 5.6 ms Gibson PCR: 5.5 ms
Plates with Kanamycine 20 ug/ml
100 ml LB agar + 40 ul Kanamycine.
Wednesday 19/08
Colibow transformation results
gBlock: nothing clear, a few number of what looks like colonies in the mess. PCR: Same with way more colonies. Control: negative.
They were re-streaked on LB + Kan 25 plates made for the occasion. A total of 16 colonies were tested.
Miniprep of pIT5-KH
Elution in 5* ul of water. It is very clean and the concentration is 167 ng/ul.
Colibow A and C PCR for pIT5 cloning
The goal is to make the A and C colibow parts compatible with pIT5-KH Gibson assembly. This PCR was performed by Constant.
Colibow A: Primers 176 and 83.
Colibow C: Primers 86 and 177.
Phusion master mix ("B+M")
50
Primer F
1
Primer R
1
Colibow A/C
1 ul
DMSO
3
Water
44
Program 98 (30)
98 (10), 50 (30), 72 (2'20) * 37
72 (5')
Results Only a slight smear, no visible PCR product. It didn't work (gel not shown).
Digestion of pIT5-KH
The pIT5-KH vector was linearized by the restriction enzymes EcoR1 and Pst1. This allows excision of the pUC propagation ORI. There is still a R6K ORI that will not bee used here. Hence, the vector is no longer able to replicate and only the integrants will be replicated.
Plasmid
50 ul
FD Green Buffer
5 ul
FD EcoR1
3.2 ul
FD Pst1
This was incubated at 37°C for 20 minutes and not inactivated, as the gel extraction followed immediately.
Gel extraction of digested pIT5-KH
Layout: 1 kb ladder, 3 merged wells with all of the digestion product.
Expected sizes: 1143 bp (to remove) and 5331 bp (to keep). The bands are present at the right size and strong.
After gel extraction, the product was eluted in 30 ul of pre-heated Elution Buffer.
Titration: 19 ng/ul.
Re-streaking of pColibow-R6K transformation product
To figure out if they're resistant or not. 12 LB Kan25 plates were made, with 200 ml of LB-agar and 100 ul of Kanamycine 1000x.
Transformation from PCR product: Two plates from the 200 ul plate, one plate from the 800 ul plate. Transformation from gBlocks: Two plates from the 800 ul plate.
Each of these plates is divided in 4 parts, with one colony on each.
Thursday 20/08
Colibow R6K rescue
It seems that it is somewhat resistant to Kan but it is not clear: it only grew at the most cell-concentrated spots. It was put back in the incubator in case slower colonies would show up.
Colony PCR
A PCR with 4 primers was done: - We expect one band (jw005 + jw006 = 139 bp) if the backbone is present (unlikely due to the control plate being empty) - We expect two bands (jw005 + 96 = 714 bp and 92 + jw006 = 829 bp) if the right pColibow plasmid is present
Dream taq master mix
90
jw05
9
jw06
9
primer 92
9
primer 96
1
Colibow A/C
1 ul
DMSO
3
Water
44
Then 17 ul of mix was added to 3 ul of cell resuspension solution. Four colonies were tested for each transformation (1,2,3,4 and I,II,III,IV).
The program was the default DreamTaq program with 53°C of annealing temperature.
Results: Absolutely nothing. Either the PCR didn't work, either the cells are not really resistant to kanamycin.
PCR for Colibow pIT5
Gradient PCR.
Primers A: 176 + 83 -> 1202 bp B: 84 + 85 -> 909 bp C: 86 + 177 -> 1004 bp 176 and 177 were diluted again from stock.
Name
1x
6.5x
2x Phusion Master Mix
50
325
DMSO
3
19.5
Water
35
44 ul of mix without primers and template + 5 ul of each primer + 2 ul of template. The templates were diluted again from the gBlocks.
Program:
98 (30)
98 (10), gradient (30), 72 (2'30)
72 (5')
It takes 3h30. Gradient: 60.2 (circled), 59, 57.1, 54.8
Friday 21/08
Results of Colibow pIT5 pcr
First gel: 1 kb ladder, A (circled), A, A, A, B (circled), B, B, B
Second gel: C (circled), C, C, C, 1 kb ladder
[colibow A B] [colibow C]
The PCR worked: We get the right bands (A: 1202, B: 909, C: 1004). As before, A and C have a low-size impurity.
Gel extraction of Colibow pIT5 parts
New gel: A, C, B (3 merged), 100 bp+ ladder, A (3 merged), C (3 merged). The well for B is huge. For each PCR, 50 ul of product was added to 5 ul of LD. The whole 200 ul don't fit in the triple ponds, so the first two ponds were used for A and C excess. [dirty]
This gel could not be used for extraction -> Loss of the product :(
Next time, merge 2 ponds and put 100 ul in it. Do not prop the comb, or not so high.
Colibow pIT5 pcr for better gel extraction
What worked best last time with the gradient?
Optimal annealing: - A: 60°C, 176 + 83 -> 1202 bp - B: 55°C, 84 + 85 -> 909 bp - C: 57°C, 86 + 177 -> 1004 bp
Name
1x
3x
2x Phusion Master Mix
50
150
DMSO
3
9
Water
35
105
Then to 88 ul of mix, 5 ul of each primer and 2 ul of template were added.
Program:
98 (30)
98 (20) Gradient (30) 72 (2'30) x35
72 (5')
Monday 24/08
Analysis of pR6K-colibow colonies
pR6K-colibow colonies from the previous transformation are resistant to kanamycine (checked by re-streaking). Some of these colonies are yellow.
They were put into culture and miniprepped (elution in 50 ul of EB). Titration: 19.8 ng/ul (clean).
The product were digested by NotI (fast-digest, 30 min at 37°C) and ran on a gel.
Results: Nothing showed up on the gel, despite the very high deposited DNA amount. I don't know what happened, but it happens everytime.
Gel extraction of Colibow for pIT5 vector
Layout: B, B, B, B, A, A, A, A, C, C, C, C No ladder, I already ran a gel of these products and I know what it looks like.
Result: All the 3 PCRs worked very well, though there is a small band in all of them (pretty strong on C).
The good bands (909, 1202 and 1004 bp respectively) were extracted, purified and eluted in 30 ul of pre-heated Elution Buffer.
Titration:
A
22 ng/ul, not so clean (presence of organic molecules)
B
30 ng/ul, clean
C
47 ng/ul, clean
Gibson assembly
Two reaction were done: one with strict equimolar concentrations, and one with a lower amount of vector as it's what takes all of the space.
Name
Concentration
Size
Volume
A
22
1202
0.71
B
30
909
0.39
C
47
1004
0.28
pIT5-KH
19
2276
3.63
Final amount: 0.004 nmol of each, which is way below the recommended concentration.
Name
Concentration
Size
Volume
A
22
1202
1.03
B
30
909
0.57
C
47
1004
0.40
pIT5-KH
19
2276
3.00
Final amount: 0.006 pmol of each.
Transformation of pIT5-KH-Colibow
The Gibson assembly product was transformed in Mukit's DH5a electrocompetent cells (that didn't give good results in the past), and also in NEB turbo electrocompetent cells made by Barth and Mukit.
For each transformation, 8 ul of assembly was dialysed and electroporated (pulse went from 5.4 to 5.5). The cells were then plated on LB Kan 25 ug/ml plates (175 ul of cells).
Colony PCR of R6K colibow
jw01 + seqR4 -> 800 bp seqF5 + jw02 -> 829 bp If the backbone is present, we should have a band from jw01 and jw02 at ~200 bp.
Result: Nothing on the gel. Either the PCR did not work, or this is contamination. Gel not shown.
New pIT5-KH and KL culture
To make more digested backbone in case today's gibson fails. Culture at 30°C in LB K50.
Tuesday, 25/08
Transformation results
pIT5-KH-Colibow did not transform (no colonies). The leftover of the recovery culture was plated just in case.
Colony PCR on the coloured colonies from 19/8
Same setup as before. Still nothing, not even positive control. I'm tired of DreamTaq never working.
Preparation of new pIT5 vector
KH did not grow very much. KL grew a little more. A glycerol stock was made for both. They were miniprep'd and eluted in 70 ul of water.
Digestion of vector
70 ul of plasmid, 3,2 ul of EcoR1 and Pst1, and 7 ul of FD green buffer were incubated 20 minutes at 37°C.
Extraction
Layout: KL, KL, KL, 1 kb ladder, KH, KH, KH.
On the gel, the KL samples migrated, so did the ladder, but the KH samples DID NOT MIGRATE. I can't believe this.
The KL sample was then extracted, purified and eluted in 40 ul of water. It yielded a concentration of 10 ng/ul. This is too low and useless.
Starting cultures
pIT5-KH, 10 ml of LB + 4 ul Kanamycine, in triplicate. Grown at 30°C.
Pir116, in 3 ml of LB with a control. Grown at 37°C.
u15.25, in 7 ul LB + 7 ul Tetracycline. It contains the pFHC2938 plasmid that expresses the CRE recombinase. Grown at 30°C.
PCR of Colibow gBlocks
Name
1x
10x
2x Phusion Master Mix
50
500
DMSO
4
4.8
Then 264 ul of mix + 15 ul of each primer + 6 ul of template, which was then split in 3 tubes of 100 ul each.
Program: 98 (40), 56 (30), 72 (2'20), 98 (20), 58 (30), 72 (2'20) 98 (20), 60 (30), 72 (2'20) * 35 72 (5')
Wednesday 26/08
Checking the transformation from the 24th
There are colonies on the NEB turbo plates, but they're too small to be analyzed for now.
Preparation of KH
KH integrates better than KL so we will focus on this one from now on. It didn't grow last night unfortunately.
NEB turbo and Pir116 competent cells
The overnight cultures were diluted 100 times and grown until OD reaches 0.6. Wash cycle:
25 ml of sterile water
25 ml of sterile water
12.5 ml of 10% glycerol
Pir116 were aliquoted with 80 ul per tube.
NEB turbo were aliquoted with 50 ul per tube.
The cells were frozen at -80°C.
Miniprep of pFHC2938
Performed by Constant with JB's protocol. Elution in 60 ul of water. Titration: 268,9 ng/ul.
Results of Colibow double PCR
With 4% DMSO and high, progressive annealing temperatures, only the C PCR worked. A and B did not work, which is not surprising because the gradient-able PCR machine was not available so the annealing temperature of 60°C was not appropriate. 4% of DMSO did not help removing the non-specific binding, so we can stick to 3% in the future.
The C PCR product is still quite strong, so it has been gel-extracted. A total of 7 bands were cut and purified. Elution was performed in 30 ul of EB, and the final concentration was 80 ng/ul, which is pretty good.
New PCR on Colibow A
To have a better concentration on this part, a new PCR was done:
Name
1x
3x
2x Phusion Master Mix
50
150
DMSO
3
9
Water
35
105
176
5
15
83
5
15
Colibow A gBlock
2
6
Program: 98 (30) 98 (20) 59 (30) 72 (3'00) * 5 98 (20) Gradient (30) 72 (2'20) * 30 72 (5'00)
Gradient: 58.8, 60.5, 61.2°C. This program was saved as "double".
Result
This PCR worked well. On this gel 2 ul of sample were loaded. [08.27 Colibow A] There is still a lot of non-specific binding and a slight smear.
The bigger band was gel-extracted, eluted in 32 ul of water and titrated to 80 ng/ul. It's not perfectly clean on the nanodrop but it's fine.
Thursday, 27/08
Miniprep of pIT5-KH
Done by Abdou, starting from two 10 ml overnight cultures in LB. For each the elution was done in 50 ul of pre-heated water. There are two samples, named Grand and Petit. Titration:
Grand: 205 ng/ul, clean
Petit: 216 ng/ul, clean
The Grand sample was digested by EcoR1, Pst1 and BamH1.
50 ul plasmid
3 ul EcoR1
3 ul Pst1
3 ul BamH1
6 ul of FD green buffer
This was incubated at 30°C during 20 min. A sample was ran on a gel to check that the digestion worked.
It was then purified using the Qiagen PCR cleanup kit and eluted in 30 ul of water. The final titer is 39 ng/ul.
Gibson assembly
Name
Concentration
Size
Volume
A
80
1202
0.5
B
200
909
0.5
C
80
1004
0.5
pIT5-KH
39
6000
4.5
At 18h the incubation was started at 50°C. At 19h05 is was stopped, put on ice and dialysed on a membrane.
Transformation of pColibow
Both Pir116 and Neb Turbo strains were transformed. For each, one negative control, one positive control (pSB1C3) and 8 ul of Gibson product were electroporated. Gibson product was dialysed for 15 min. Pir116 strains were recovered at 30°C to prevent integration while Neb Turbo were recovered at 37°C. After that, 150 ul were plated on LB K20 agar plates and kept at 30°C (done by Barth).
Friday 08/28
Transformation results
Barth made mistakes while plating, the negative control and positive control were inversed. Colonies are visible only on the Neb turbo plates with Gibson product.
Colony PCR to check the integration
To check for integration, a combination of four primer is used:
Name
1x
4.5x
DT master mix
10
45
P1
1
4.5
P2
1
4.5
P3
1
4.5
P4
1
4.5
Water
3
8.5 ul of mix + 2 ul of colony in water.
Expected:
One band at 740 bp is there is no integration.
Two bands at 343 and 824 if there is one integration
Additional band at 427 in case of tandem integrations
Results: Nothing. Even with colonies without chromosomal integration, at least one band should show up, so there is a problem either with the PCR or with the cell lysis.
Colony PCR to check the presence of insert
In parallel, the same as always.
Name
1x
10x
DT master mix
5
50
90
0.5
50
94
0.5
50
Water
1
10
Then 7ul master mix + 3 ul.
Expected: One band at 508 bp.
Results There is one band in most colonies. The control is not noticeably positive. Gel layout: I, II, III, IV, V, VI, VII, VIII, gBlock B, 100 bp+. [Colony pIT5KH...]
There is one band in the colonies IV, V, VI, VII and VIII so it seems they are good and the cassette was integrated in the chromosome.
New pIT5KH digestion
In case they are not good. 50 ul of plasmid + 3 ul EcoR1 + 3 ul Pst1 + 5.6 ul FD buffer 37°C for 20 minutes. It was then stored at -20°C.
Preparation of TSS buffer
For 100 ml:
10 g of PEG 8000
612 mg of MgCl2
5 ml of DMSO
LB to 100 ml
Culture of Integrated Colibow
V, VI, VII and VIII were put to overnight culture in 5 ml of LB kanamycin.
Plates Kanamycine
Made five plates with 100 ml of LB+agar and 80 ul of Kanamycine (concentration: 40 ug/ml)
Saturday 08/29
Attempted to grow the Colibow integrant in 25 ml of LB-K20 for making TSS-competent cells. For some reason, nothing grew.
Sunday 08/30
Same as yesterday. Still no growth at all. Plated the first 16 colonies on K40 to check whether they are really resistant.
Monday 08/31
The Colibow integrant cells grow slowly but they are indeed resistant to kanamycine. The presence of the phage integrase could be the reason why they do not grow well.
Screening for LB
Maybe the LB is the problem. Using two batches of LB-agar from two different days, a screening was performed, with and without kanamycine.
LB1: from the 24th
LB2: from the 19th
Each plate was made with 20 ml of LB-agar and 8 ul of kanamycine. The cells were taken from the plates.
New colibow reaction
The digestion of KHx was checked on a gel. It was then PCR-purified and eluted in 30 ul EB. Concentration: 118 ng/ul, clean.
Gibson assembly
Name
Concentration
Size
Volume
A
80
1202
0.5
B
200
909
0.5
C
80
1004
0.5
pIT5-KH
118
6000
4.5
That is 0.018 pmol per fragment.
Transformation
The gibson product was electroporated in Pir116 and NebTurbo. Pulse: 5.3 to 5.7. After recovery it was plated on LB Kan20 plates.
New culture of Colibow
Using a bottle of LB from the lab at the 6th floor, and the colonies V, VI, VII, VIII of the re-streaked plate.
Tuesday 09/01
TSS competent cells with colibow
From the overnight culture (which grew a little), dilution to the 1/50 in LB-Kan.
Reception of new oligos
Two oligos for biobricking the promoter.
Four oligos for checking integration of the KH vector.
Result of yesterday's transformation
In the morning nothing was visible on the plates, but a couple of hours after, there are ~30 colonies on the Pir116 and NT plates. The Pir116 control seems contaminated with a few very small colonies.
Four Pir116 colonies were put into culture for miniprep and analysis.
Genomic extraction of Colibow
Since the colony PCR do not seem to work, this allows to remove one source of problems. Final concentrations:
V: 80 ng/ul
VI: 150 ng/ul
VII: 34 ng/ul
VIII: 38 ng/ul
PCR for checking the integration
With the extracted genome as a template.
Name
1x
4.5x
DT master mix
10
45
P1
1
4.5
P2
1
4.5
P3
1
4.5
P4
1
4.5
Then 14 ul of mix + 6 ul of genomic DNA.
95 (1') 95 (20) 48 (20) 72 (1') *34 72 (4')
Gel: Absolutely nothing! In fact the DNA concentration is waaay too high.
This PCR was done again by Barth with less template (0.5 ul of DNA).
[9/1 PCR for integration]
The gel is difficult to interpret, but it seems like the clone number 8 is integrated and VI and VII are not.
Biobricking of the Lox Array
PCR to add the biobrick prefix and suffix:
Phusion master mix
25
Primer R
1
Primer L
1
Colibow A
1 ul
DMSO
1.5
Water
20.5
98 (30) 98 (20) 50 (25) 72 (15) * 35 72 (10)
Gel: 100 bp+, PCR product. [09/01 biobricking] It's very good, and has to be cleaned up.
Wednesday 09/02
Vector for biobricking
pSB1C3-RFP: got 65 ul of miniprep from Mukit.
Purification of Biobrick-pcr
Elution in 35 ul of EB. Final titer: 20 ng/ul, clean.
Gibson assembly of Colibow for Top10 E. coli
Due to the repetitive nature of the Colibow sequence, it seems better to use a Rec- strain with reduced homologous recombination, such as the Top10 strain. A new Gibson reaction was set up.
Name
Concentration
Size
Volume
A
80
1202
0.67
B
30
909
1.35
C
80
1004
0.56
pIT5-KH
118
6400
2.42
This is 0.014 pmol per fragment.
Miniprep of pColibow plasmid from Pir116
After elution in 50 ul of water, the concentrations are not very high (ranging from 13 to 20 ng/ul). A glycerol stock was made from the cultures.
Gel: plasmid from colony 1, 2, 3, 4, 5, 6, 1kb ladder, 5 ul of Gibson product. Each time 5 ul of sample was deposited.
[9.2 Pir pCol]
It seems there is a plasmid in all colonies except 5. This plasmid is of the same size as the Gibson product, which seemed to work quite well.
Analytical digest 6 ul of these plasmids were digested by 1 ul of NotI FD during 30 min at 37°C, in 20 ul total volume.
However, nothing was visible on the gel ran after digestion (there seems to be something in the ponds, though).
Transformation of Top10 strain with pColibow assembly
These cells are heat-shock competent, not electrocompetent. As a control, 1 ul of KHx was used. At the same time, a culture of Top10 in 5 ml of LB (with control) was launched. The transformation products were plated on Kan20 plates after 2 h of recovery.
Biobrick construction
The pSB1C3 vector and the insert were digested by EcoR1 and Pst1, during 20 min at 37°C.
The insert was PCR-purified again to remove the excised tails. Final concentration: 19 ng/ul in 30 ul of water.
The backbone was gel-extracted on a 0.6 % gel. Unfortunately, the digestion reaction did not work so good, and most of the vector has been cut by only one enzyme. The band at 2029 (double-digested vector) was cut off and purified. Final concentration: 7.2 ng/ul.
