Team:Paris Bettencourt/Protocols
Ferment It Yourself
iGEM Paris-Bettencourt 2O15
Background
Design
Project
Micronutrients
Continuity
Modeling
Manufacturing
Achievements
Results
Parts
Medal Fulfillments
Practices
Overview
SynBio Challenge
DIλ
Acceptance
Events
Rhizi
Collaborations
Safety
Safety Overview
Safety Form
Review Board
Notebook
Daily notebook
Protocols
Bibliography
Team
Members
Attributions
Acknowledgments
Press
Official Team Profile
Protocols
Yeast DNA extraction  using Blood and tissue kit
Yeast DNA extraction using DNeasy® Blood & Tissue Kit
Centrifuge a maximum of cells for 5 minutes at 300g (190rpm). Resuspend in 200µL PBS. Add 20µL zymolyase.
Add 200µL Buffer AL. Mix thoroughly by vortexing.
Add 200µL ethanol (96%). Mix thoroughly by vortexing.
Pipet the mixture into a DNeasy Mini spin column placed in a 2mL collection tube. Centrifuge at 6000g (8000rpm) for 1 minute. Discard the flow-through and collection tube.
Place the spin column in a new 2mL collection tube. Add 500µL Buffer AW1. Centrifuge for 1 minute at 6000g. Discard the flow-through and collection tube.
Place the spin column in a new 2mL collection tube. Add 500µL Buffer AW2. Centrifuge for 3 minutes at 20,000g. Discard the flow-through and collection tube.
Transfer the spin column to a new 1.5mL or 2mL microcentrifuge tube.
Elute the DNA by adding 200µL Buffer AE or sterile water to the center of the spin column membrane. Incubate for 1 minute at room temperature. Centrifuge for 1 minute at 6000g.
Repeat this step for increased DNA yield.
PCR protocol
PCR protocol (Phusion)
Prepare the following mix
2µL primer Forward (10 µM)
2µL primer Reverse (10 µM)
0.1 to 1 ng of template DNA
25µL of Life Technologies Phusion High-Fidelity PCR Master Mix with HF Buffer
complete to 50µL of DNase/RNase-free water
Launch 30 to 35 PCR cycles using the following parameters:
time (min)
temperature (°C)
function
3:00
98
melting
0:30
98
melting
0:30
XX°C
annealing
1 minute/kb
72
extension
10:00
72
extension
infinite hold
12
storage
Gel purification with the QIAquick® Gel Extraction Kit
Gel-Purification protocol with the QIAquick® Gel Extraction Kit
Excise the DNA band from the gel as precisely as possible and put it in a micro-centrifuge tube.
Add 3 volume of QG buffer (100mg of gel ~ 100ul) in the tube and incubate for 10 min at 50°C (or 10 min on a vortex).
Add one volume of isopropanol and mix.
Centrifugate the tube for 2 min at 14 000 rpm.
Carefully pipette the supernanant in a QIAquick column.
Put the column in a 2ml microcentrifuge tube and centrifuge it for 1 min at 14 000 rpm.
Discard the flow-through.
Add 750 uL of PE buffer in the column and centrifuge for 1 min and discard flow-through.
Centrifuge again for 2 min and put the column in a new 1.5uL microcentrifuge tube.
Elute the DNA by putting 50uL of water in the middle of the column and let it stand for 2 min.
Centrifugate for 2min at 10 000 rpm
Gel Electrophoresis with SYBR safe
Gel-Electrophoresis protocol
Preparation of the Gel (50mL)
Mix 0.5 grams of agarose with 50mL 0.5% TAE solution
Heat the Erlenmeyer (conical flask) in a microwave until the liquid becomes totally transparent.
Let it rest until you can take the Erlenmeyer in your hand without pain for 10 seconds.
Add 5uL of SYBR® Safe (10000X) and mix gently until the solution homogenizes.
Pour the liquid in a gel mold with the desired number of wells and wait 20-30 min (or until the gel is solidified).
Put the gel inside the electrophoresis machine and pour TAE 0.5% until the gel is totally submerged.
Preparation of the samples
Mix 5uL of each sample with 1uL of 6X loading dye in microcentrifuge tubes (or any hydrophobic surface like parafilm)
Pipette 5-6uL of each sample inside a different well.
Pipette 5uL of the appropriate ladder in one of the wells (Some ladders have a dye in it, but some don't, so one has to add gel loading dye to these ones)
Close the machine and launch the migration with the desired voltage
After the desired migration time turn off the machine, remove the lid and put the gel inside the gel visualizer.
Titration of phytic acid
Titration of phytic acid using Megazyme© Kit
Preparation of reagent solutions (not supllied) :
Trichloroacetic acid  (50% w/v) : 100mL
Add 50g of thrichloroacetic to 60mL of distilled water and dissolve. Make to volume (100mL) with distilled water. (Stable for > 6 moths at 4°C)
Hydrochloric acid (0.66M) : 1L
Add 54.5mL of hydrochloric acid to 945.5mL of distilled wtaer an mix. (Store at room temperature)
Sodium hydroxide (0.75M) : 200mL
Add 6g of sodium hydroxide pellets to 180mL of distilled water and dissolve. Make to volume (200mL) with distilled water. (Store at room temperature)
Phytic acid :
Pure phytic acid control sample may be required.
Sample extraction :
Weigh 1g of sample material.
Add 20mL of hydrochloric acid (0.66M).
Cover with foil and incubate for a minimum of 3 hours at room temperature.
Transfer 1mL of extract to a 1.5mL microcentrifuge tube.
Centrifuge 10 minutes at 13,000rpm.
Transfer 0.5mL of the resulting extract supernatant to a fresh 1.5mL microfuge tube.
Add 0.5mL of sodium hydroxide solution (0.75M) to neutralise.
Enzymatic dephosphorylation reaction :
Colourimetric determination of phosphorous :
Preparation of phosphorous calibration curve :
Calculation :
Phosphorous calibration curve :
Determine the absorbance of each phosphorous standard. Substract the absorbance of STD0 from the absorbance of the others STD, therby obtaining ΔA(phosphorous).       Calculate M as follows, for each standard:  \[ \begin{align} M = \frac{P(\mu g)}{\Delta A (phosphorous)} \end{align} \]       Use "Mean M" to calculate the phosphorous content of test samples.
Phosphorous / phytic acid content :
Determine the absorbance of the both "Free Phosphorous" and "Total Phosphorous". Substract the absorbance of the "Free Phosphorous" sample from the absorbance of the "Total Phosphorous" sample. Obtaining ΔA(phosphorous). \[ \begin{align} C^{o} = \frac{mean(M)\times 20 \times F}{10000 \times 1.0 \times \nu} \times \Delta A(phosphorous) \end{align} \]
Where :
\(20\) = original sample extract volume
\(F\) = dilution factor
\(10000\) = conversion from µg/g to g/100g
\(1.0\) = wigh of original sample material
\(\nu\) = sample volume
It follows for phosphorous :
\[ \begin{align} C^{o} ~= \frac{mean(M)\times 20 \times 55.6}{10000 \times 1.0 \times \nu} \times \Delta A(phosphorous)\\  ~= mean(M) \times 0.1112 \times \Delta A(phosphorous) \end{align} \] It follows for phytic acid :
\[ \begin{align} C^{o} = \frac{phosphorus}{0.282} \end{align} \]
Vitamin B12 titration
Disrupt the microorganisms in a 0.1 N phosphate buffer solution containing 0.01% KCN at pH 5.5 for 20 min at 100°C.
Vitamin B12 formed intracellularly is determined spectrophotometrically in the dicyano-form at wavelength of 367 nm.
PCR purification
Add 5 volumes of resuspension buffer to 1 volume of PCR product in an 1.5mL microcentrifuge tube, mix by pipetting up and down
Transfer in a centrifugation column
Centrifuge 1 min at 14000 rpm
Throw the filtration product
Add 700μL of washing solution
Centrifuge 1 min at 14000 rpm
Throw the filtration product
Add 500μL of washing solution
Centrifuge 1 min at 14000 rpm
Throw the filtration product
Centrifuge 1 min at 14000 rpm
Throw the filtration product
Put the column in a sterile 1.5mL microcentrifuge tube
Add 45μL of DNAse/RNAse free water on the membrane
Wait 2 minutes
Centrifuge 2min at 10000rpm
Discard the column, DNA is saved in water
Analytical digestion protocol
Prepare the following mix:
2µL 10X Digestion buffer
0.5µL Eco31I
0.5µL BbsI
2µL of DNA (200ng)
15µL water
incubate 1h at 37°C
Annealing Protocol
Phosphorylation of the oligos
5.6μL   DNAse/RNAse free water
6.0μL   oligo 1 (10µM)
6.0μL   oligo 2 (10µM)
2.0μL   10X T4 DNA ligase buffer
0.4μL   T4 PolyNucleotide Kinase
Total:   20μL
incubate 30min at 37°C
add 1μL of 1M NaCl
incube 5min at 95°C
let the mix cool down
use 2μL of the mix as a 10X solution
Chemical test for competent cells
20-30 sec at 8-10 krpm for the DNA tube (from the kit).
Thow cell competent on ice. Label 2 ml eppendorf µtube for each concentration and put it on ice.
Add 1µl of DNA into each µtube.
Add 50µl of competent cell into each tube. Flick gently to mix. Incubate in ice for 30 minutes. Pre-heat waterbath at 42°C.
Heat-shock cell by placing in waterbath 1 minute. CAUTION : The top need to be close to the water level, but not immerge in.
Keep back the tube to ice for 5 minutes.
Add 200µl of SOC media, incubate 37°C for 2h. Prepare agar plate of the media that you want, and label it (3 plate for each concentration).
Add 20µl of each tube n  the appropriate plate. Incubate 37°C for OVN (12-16h).
BANANAAAA
Line up ice cream scoops next to each other in an oval deep dish or a banana boat.
Cut the ends of the banana off (about 1/4 inch) while still in the peel.
Slice in half longways.
Pop each half of the banana out of the peel onto each side of the ice cream row, pressing down and in a little so it'll stay put.
Top the vanilla ice cream with the pineapple, the chocolate with the chocolate syrup and the strawberry with the strawberry sauce.
Spoon the wet walnuts over all three scoops of ice cream.
Top each scoop with some whipped cream and a cherry for each.
Enjoy!
Electroporation
Thaw electrocompetent cells on ice
Add 2µL of ligation product or 0.5µL of native plasmid to the cells
Transfer the cells in an 0.2mm electroporation cuvette
put the cuvette in the electroporation device and pulse the cells at 2.5kV, 200 Ohms and 25µF
add 200µL of LB right after pulsing
recover 2 hours at 37°C
plate 200µL and 50µL of the cells on LB + erythromycin (200µg/mL), incubate overnight at 37°C
Electrocompetent Cells
Inoculate a 250mL LB flasks with 2.5mL of an overnight culture of cells
Incubate until the the DO600 reach 0.6 to 0.8
Place the cultures on ice for 15 minutes
Pour the culture in cold sterile 50mL falcon tubes
Centrifuge for 10 minutes at 4000rpm, 4°C
Throw the supernatant
Resuspend the cells in 50mL cold distilled water
Place the cells on ice for 10 minutes
Centrifuge them for 10 minutes at 4000rpm, 4°C
Throw the supernatant
Resuspend the cells in 12.5mL cold 10% glycerol
Place the cells on ice for 10 minutes
Centrifuge them for 10 minutes at 4000rpm, 4°C
Throw the supernatant
Resuspend the cells in 5mL cold 10% glycerol
Make aliquots of 50 to 100µL in microcentrifuge tubes and freeze them at -80°C
Digestion
Prepare the following mix:
4μL of Enzyme 1 Fast Digest
4μL of Enzyme 2 Fast Digest
4μL of FastAP
12μL of Fast Digest buffer 10X
1 to 3 μg of DNA
up to 120μL of water
mix by pipetting up and down
incube 10 to 20 minutes at 37°C
incube 10 minutes at 68°C to inactivate the enzymes
Lactobacillus plantarum electrocompetent cells
Inoculate 5ml MRS medium with L. plantarum freezer stock.
Grow overnight at 30°C without shaking.
Add 1.25g glycine to two flasks of 50ml MRS medium.
Shake to dissolve.
Add 1ml overnight culture to each flask (1:50 dlution).
Culture for ~3 hours at 37°C with shaking.
Centrifuge culture for 5min at 4000g
Resuspend in 25ml ice-cold DI water.
Repeat it.
Centrifuge for 5min at 4000g
Resuspend in 5ml 50mM EDTA.
Incubate on ice for 5 minutes
Add 25ml ice-cold DI water.
Centrifuge culture for 5min at 4000g
Resuspend in 25ml ice-cold DI water.
Centrifuge culture for 5min at 4000g
Resuspend in 25ml ice-cold 0.5M Sucrose, 10% glycerol.
Repeat it.
Centrifuge culture for 5min at 4000g
Resuspend in 0.8ml ice-cold 0.5M Sucrose, 10% glycerol.
Aliquot 90μL of cell concentrate into ~20 microcentrifuge tubes.
Keep on ice until use (within the next two hours).
Add 10μL of plasmid DNA to the 90μL of cell concentrate.
Keep on ice for 5 minutes.
Put 1mm cuvettes on ice too.
Pipette the cell/DNA mixture into the cuvette
Electrporate at 1200 volts
Time constant should be ~5.0
Immediately transfer the electroporated cells to 900μL of MRS medium.
Incubate at 30°C for 2-3 hours.
Plate on medium with the appropriate antibiotic.
Incubate at 30°C for two days.
Pick a colony
Heat Shock Transformation
Thaw frozen chemically competent cells (20µL aliquots) on ice for 10min.
add 2µL of ligation product (or 0.5µL of miniprep) and incubate the cells 30sec at 42°C.
put the cells back on ice for 2min.
add 200µL of LB to the cells and incubate 2 hours at 37°C.
plate the cells on LB + erythromycin (150µg/mL) or LB + erythromycin (10µg/mL), incubation at 37°C overnight.
Vitamin A Titration
Take 1g of food (with particle < 1mm of diameter) in bottle protected of the light, and add 15ml of hexane. Mix it.
Add again 15ml of hexane and shake during 5-10 minutes (do the same time, if you want compare two food sample).
In an other bottle protected of the light, do a filtration (coarse cellulose filter) of the previous solution.
Take the batter who don't pass the filter, and 15ml of hexane, shake 5 minutes, and filtered it like previously, to take all B-caroten as possible from the food sample.
Put in the fridge at -20°C to conserve it.
Obtain the result with a spectrophotometer with wavelenght of 450 nm (with a blank of hexane).
Vitamin A Titration using HPLC
Suspend cells in 1 mL of sterile water.
Add 0.5 to 0.75 mm glass beads and vortex for 3 minutes.
Add 2.5 mL of 0.2% (wt/vol) pyrogallol dissolved in methanol and vortex for 3 minutes.
Add 1.25 mL of 60% (wt/vol) KOH and vortex for 10 seconds.
Incubate for 1 h at 75ºC, vortexing every 15 minutes.
Add appropriate amount of hexane.
Centrifuge tubes for 5 minutes at 2,800 rpm.
Pipette 1 mL of hexane into a cuvette.
Lactococcus lactiselectrocompetent preparation
Prepare the following solutions
Glycine/Sucrose solution
80mL of 50% sucrose solution
29.2mL of 20% glycine solution
7.6mL of water
Electroporation buffer
0.125g of NaHPO4
34.2mL of 50% sucrose solution
100µL of MgCl2 1M
65.5mL of water
Electroporation buffer + glycreol
187.5µL of 80% glycerol
812.5µL of electroporation buffer
Protocol:
Inoculate 10mL of M17 + 1% glucose (GM17) with some Lactococcus and incubate overnight at 30°C
Inoculate a 100mL flask of GM17 with the 10mL of overnight culture
Incubate the flask at 30°C until DO reach 0.4 to 0.5
Add 100mL of the glycine/sucrose solution
Incubate 1 hour at 30°C with shaking
Split the 100mL culture in twyo 50mL centrifuge tubes
Centrifuge the cells 10 minutes at 6000 rpm
Throw the supernatant and work on ice for the rest of the protocol
Resuspend the cells in 20mL of electroporation buffer
Centrifuge the cells 10 minutes at 6000 rpm and then throw the supernatant
Resuspend the cells in 20mL of electroporation buffer
Centrifuge the cells 10 minutes at 6000 rpm and then throw the supernatant
Resuspend the cells in 0.5mL of electroporation buffer
Centrifuge the cells 10 minutes at 6000 rpm and then throw the supernatant
Resuspend the cells in 1mL of cold electroporation buffer + glycerol
Aliquots the cells in eppendorf tubes
Save the cells at -80°C
Yeast Lysis with NaOH
20 µl NaOH into PCR tubes
Pick colonies into NaOH
Incubate at 95°C for ~45 minutes
Centrifugate at 8000 krpm for 10 minutes
Use 1 µl supernatant as template in a (10 µl) PCR
Heat Shock Transformation for Yeast
After growth, determine the titer of the yeast culture by using spectrophotometer : pipette 10µL of cells into 1mL of water in spectrophotometer cuvette and measure the OD at 600nm.
Add 2.5x108 cells to 50mL of 2X YPD in a culture flask.
Incubate the flask in a shaking incubator at 30°C until the cell culture is at least 2x107 cells.mL-1
Denature 1mL of carrier DNA at 99°C for 5min and chill immediately in ice.
Harvest the yeast cells by centrifugation at 3,000g for 5min.
Resuspend the pellet in 25mL of sterile water and centrifuge at 3,000g for 5min at 20°C. Repeat this wash with sterile water 2 times.
Resuspend the last pellet in 1mL of sterile water.
Transfer the cell suspension to a 1.5mL microcentrifuge tube.
Centrifuge for 30s at 13,000g and discard the supernatant.
Resuspend the cells in 1mL of sterile water and pipette 100µL samples into 1.5mL microcentrifuge tubes, one for each transformation.
For each transformation :
240µL of PEG 3350 (50% (w/v))
36µL of LiAc 1.0M
50µL of single-stranded carrier DNA (2.0mg.mL-1)
6µL of PCR product
28µL of water DNAse Free
Place the tubes at 42°C for 40min.
Centrifuge the tubes at 13,000g for 30s in a microcentrifuge tube and remove the supernatant.
Pipette 1mL of YPD liquid medium into the transformation tube, and vortex mix to resuspend pellet.
Incubate 3h at 30°C to ensure good antibiotic expression.
Plate 2, 20 and 200µL of the cell suspension onto YPD medium with 200µm.mL-1 antibiotic G418.
Incubate the plates at 30°C for 3 days.
Ligation
Mix the following:
vector 100ng
insert 300ng
2µL T4 DNA ligase buffer 10X
1µL T4 DNA ligase
up to 20µL of water
incubate 10 to 15 minutes at 22°C
incubate 5 minutes at 70°C
Vitamin B2 Titration using HPLC
Titration protocols for vitamin A Here, I find  2 different protocols to determine the quantity of B2 using HPLC.
I°) In a 250 ml conical flask, put 5g of food sample and 65 ml of 0.1M HCl.
Put it in 100°C for 30 minutes.
Cooling, then adjuste Ph to 4.5 with 2.5M sodium acetate.
Add 50ml Beta amylase, 500ml Takadiastase in water (small quantity). Put it at 37°C for 18h.
Dilutate with 125ml water.
Fitred with cellulose paper. And do a 2nd filtration with 0.2µm filter paper.
II°) Take Xg of food sample and add 15ml of 0.1M HCl.
Incubate at 100°C for 30 minutes.
Cooling, then add 2.5M sodium acetate (to have a Ph4.5) and add 10mg of Takadiastase. Incubate it at 50°C during 3h.
Filtred with Albert No 135.
Made up to 25ml with milli Q water.
Filtered with 0.2µm nylon filter.
Idli Recipe
Soak rice and dall separately for 4-5 hours. (for 1.5 volume of rice, put the same volume of water, and 0.5 volume of dall, put the same volume of water).
Rinse both, mix it, add 2/3 volume of water of this volume.
Blend this mix, and let ferment  for 12-16 hours.
Then Cook it with idli cooker for 15 minutes.
In the case where we add micro-organisms, we have 3 options:
Add µorganisms during the soak phase
Add µorganisms after the blending phase.
Let soak µorganisms in the same time that rices and dall, and add it for the blending phase.
The third was mainly used.
Miniprep protocol
centrifuge an overnight culture of cells 10min at 4krpm
throw the filtrate
resuspend the pellet in 250μL of Cell Resuspension Solution then mix it
transfer it in a 1.5mL microcentrifuge tube
add 250mL of Cell lysis solution and mix by inverting several times
incube until the liquid is clear, maximum 5min
add 10μL of Alkalyne protease solution, mix by inverting several times and incube 3 to 4 min
add 350μL of Neutralisation Solution and mix by inverting several times
centrifuge 10min at 14krpm
add the supernatant in a column
centrifuge 1min, 14krpm, then throw the filtrat
add 750μL Washing Solution
centrifuge 1min, 14krpm, then throw the filtrat
add 250μL Washing Solution
centrifuge 2min, 14krpm, then throw the filtrat
centrifuge 1min, 14krpm
transfer the column in a sterile microcentrifuge 1.5ml tube
add 50μL of DNAse/RNAse free water right on the membrane of the filter, wait 1min
centrifuge 1min, 14krpm
throw the column, plasmid is saved in water
Analytical digestion
Annealing
Chemical Test for Competent Cell
Digestion
Electroporation
Electrocompetent Cells
L. plantarum Electrocompetent Cells
Gel Electrophoresis with SYBR safe
Gel purification
Heat Shock Transformation
Heat shock transformation for yeast
Idli Recipe
Ligation
Miniprep protocol
PCR
PCR purification
Titration of phytic acid
Vitamin A Titration
Vitamin A Titration using HPLC
Vitamin B2 Titration using HPLC
Vitamin B12 Titration using spectrophotry
Yeast DNA extraction
Yeast Lysis with NaOH
Miscellaneous
Banana Split
