<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Paris Bettencourt/Protocols</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="menuContainer"><DIV id="smallMenuTitle"><H1>Ferment It Yourself</H1><H3>iGEM Paris-Bettencourt 2O15</H3></DIV><UL><LI class="left"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Background" title="Background">Background</A></LI><LI class="left"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Design" title="Design">Design</A></LI><LI class="left"><BUTTON class="menuButton">Project</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Micronutrients">Micronutrients</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Continuity">Continuity</A></LI><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Manufacturing">Manufacturing</A></LI></DIV></UL><LI class="left"><BUTTON class="menuButton">Achievements</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Parts">Parts</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Medals">Medal Fulfillments</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Practices</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/SynBio_Challenge">SynBio Challenge</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/DILambda">DIλ</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Acceptance">Acceptance</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/Events">Events</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/Rhizi">Rhizi</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Collaborations">Collaborations</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Safety</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Safety">Safety Overview</A></LI><LI><A href="https://2015.igem.org/Safety/Final_Safety_Form?team_id=1678">Safety Form</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/ReviewBoard">Review Board</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Notebook</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Notebook">Daily notebook</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols">Protocols</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Bibliography">Bibliography</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Team</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Team">Members</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Acknowledgments">Acknowledgments</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Press">Press</A></LI><LI><A href="https://igem.org/Team.cgi?id=1678">Official Team Profile</A></LI></DIV></DIV><DIV id="protocols"><DIV class="contentContainer"><DIV id="banner"><H3>Protocols
</H3></DIV></DIV><DIV class="textWrapper"><DIV class="textContent {{{page_option}}}" id="caffeine"><DIV class="textBox" id="YeastDNAextractiondneasybloodandtissuekit"><H3>Yeast DNA extraction  using Blood and tissue kit</H3><B>Yeast DNA extraction using DNeasy® Blood &amp; Tissue Kit</B><UL><LI>Centrifuge a maximum of cells for 5 minutes at 300g (190rpm). Resuspend in 200µL PBS. Add 20µL zymolyase.</LI><LI>Add 200µL Buffer AL. Mix thoroughly by vortexing.</LI><LI>Add 200µL ethanol (96%). Mix thoroughly by vortexing.</LI><LI>Pipet the mixture into a DNeasy Mini spin column placed in a 2mL collection tube. Centrifuge at 6000g (8000rpm) for 1 minute. Discard the flow-through and collection tube.</LI><LI>Place the spin column in a new 2mL collection tube. Add 500µL Buffer AW1. Centrifuge for 1 minute at 6000g. Discard the flow-through and collection tube.</LI><LI>Place the spin column in a new 2mL collection tube. Add 500µL Buffer AW2. Centrifuge for 3 minutes at 20,000g. Discard the flow-through and collection tube.</LI><LI>Transfer the spin column to a new 1.5mL or 2mL microcentrifuge tube.</LI><LI>Elute the DNA by adding 200µL Buffer AE or sterile water to the center of the spin column membrane. Incubate for 1 minute at room temperature. Centrifuge for 1 minute at 6000g.</LI><LI>Repeat this step for increased DNA yield.</LI></UL></DIV><DIV class="textBox" id="PCR"><H3>PCR protocol</H3><B>PCR protocol (Phusion)</B><UL><LI>Prepare the following mix
        </LI><LI>2µL primer Forward (10 µM)</LI><LI>2µL primer Reverse (10 µM)</LI><LI>0.1 to 1 ng of template DNA</LI><LI>25µL of <I>Life Technologies</I> Phusion High-Fidelity PCR Master Mix with HF Buffer</LI><LI>complete to 50µL of DNase/RNase-free water</LI></UL><LI>Launch 30 to 35 PCR cycles using the following parameters:<TABLE style="width:25%"><TBODY><TR><TH>time (min)</TH><TH>temperature (°C)</TH><TH>function</TH></TR><TR><TD>3:00</TD><TD>98</TD><TD>melting</TD></TR><TR><TD>0:30</TD><TD>98</TD><TD>melting</TD></TR><TR><TD>0:30</TD><TD>XX°C</TD><TD>annealing</TD></TR><TR><TD>1 minute/kb</TD><TD>72</TD><TD>extension</TD></TR><TR><TD>10:00</TD><TD>72</TD><TD>extension</TD></TR><TR><TD>infinite hold</TD><TD>12</TD><TD>storage</TD></TR></TBODY></TABLE></LI></DIV><DIV class="textBox" id="GelPurification"><H3>Gel purification with the QIAquick® Gel Extraction Kit</H3><B>Gel-Purification protocol with the QIAquick® Gel Extraction Kit</B><UL><LI>Excise the DNA band from the gel as precisely as possible and put it in a micro-centrifuge tube.
      </LI><LI>Add 3 volume of QG buffer (100mg of gel ~ 100ul) in the tube and incubate for 10 min at 50°C (or 10 min on a vortex).
      </LI><LI>Add one volume of isopropanol and mix.
      </LI><LI>Centrifugate the tube for 2 min at 14 000 rpm.
      </LI><LI>Carefully pipette the supernanant in a QIAquick column.
      </LI><LI>Put the column in a 2ml microcentrifuge tube and centrifuge it for 1 min at 14 000 rpm.
      </LI><LI>Discard the flow-through.
      </LI><LI>Add 750 uL of PE buffer in the column and centrifuge for 1 min and discard flow-through.
      </LI><LI>Centrifuge again for 2 min and put the column in a new 1.5uL microcentrifuge tube.
      </LI><LI>Elute the DNA by putting 50uL of water in the middle of the column and let it stand for 2 min.
      </LI><LI>Centrifugate for 2min at 10 000 rpm
     </LI></UL></DIV><DIV class="textBox" id="Electrophoresis"><H3>Gel Electrophoresis with SYBR safe</H3><B>Gel-Electrophoresis protocol</B><UL><LI>Preparation of the Gel (50mL)
        </LI><LI>Mix 0.5 grams of agarose with 50mL 0.5% TAE solution</LI><LI>Heat the Erlenmeyer (conical flask) in a microwave until the liquid becomes totally transparent.</LI><LI>Let it rest until you can take the Erlenmeyer in your hand without pain for 10 seconds.</LI><LI>Add 5uL of SYBR® Safe (10000X) and mix gently until the solution homogenizes.</LI><LI>Pour the liquid in a gel mold with the desired number of wells and wait 20-30 min (or until the gel is solidified).</LI><LI>Put the gel inside the electrophoresis machine and pour TAE 0.5% until the gel is totally submerged.</LI></UL><LI>Preparation of the samples</LI><UL><LI>Mix 5uL of each sample with 1uL of 6X loading dye in microcentrifuge tubes (or any hydrophobic surface like parafilm) </LI><LI>Pipette 5-6uL of each sample inside a different well.</LI><LI>Pipette 5uL of the appropriate ladder in one of the wells (Some ladders have a dye in it, but some don't, so one has to add gel loading dye to these ones)</LI><LI>Close the machine and launch the migration with the desired voltage</LI><LI>After the desired migration time turn off the machine, remove the lid and put the gel inside the gel visualizer.</LI></UL></DIV><DIV class="textBox" id="Titrationacidphytic"><H3>Titration of phytic acid</H3><B>Titration of phytic acid using Megazyme© Kit</B><UL><LI><B>Preparation of reagent solutions (not supllied) :</B></LI><UL><LI><I><B>Trichloroacetic acid  (50% w/v) : 100mL</B></I></LI>
           Add 50g of thrichloroacetic to 60mL of distilled water and dissolve. Make to volume (100mL) with distilled water. (Stable for &gt; 6 moths at 4°C)<LI><I><B>Hydrochloric acid (0.66M) : 1L</B></I></LI>
           Add 54.5mL of hydrochloric acid to 945.5mL of distilled wtaer an mix. (Store at room temperature)<LI><I><B>Sodium hydroxide (0.75M) : 200mL</B></I></LI>
           Add 6g of sodium hydroxide pellets to 180mL of distilled water and dissolve. Make to volume (200mL) with distilled water. (Store at room temperature)<LI><I><B>Phytic acid :</B></I></LI>
           Pure phytic acid control sample may be required.</UL><LI><B>Sample extraction :</B></LI><UL><LI>Weigh 1g of sample material.</LI><LI>Add 20mL of hydrochloric acid (0.66M).</LI><LI>Cover with foil and incubate for a minimum of 3 hours at room temperature.</LI><LI>Transfer 1mL of extract to a 1.5mL microcentrifuge tube.</LI><LI>Centrifuge 10 minutes at 13,000rpm.</LI><LI>Transfer 0.5mL of the resulting extract supernatant to a fresh 1.5mL microfuge tube.</LI><LI>Add 0.5mL of sodium hydroxide solution (0.75M) to neutralise.</LI></UL><LI><B>Enzymatic dephosphorylation reaction :</B></LI><LI><B>Colourimetric determination of phosphorous :</B></LI><LI><B>Preparation of phosphorous calibration curve :</B></LI><LI><B>Calculation :</B></LI><UL><LI><B>Phosphorous calibration curve :</B></LI>
      Determine the absorbance of each phosphorous standard. Substract the absorbance of STD0 from the absorbance of the others STD, therby obtaining ΔA(phosphorous).
      Calculate M as follows, for each standard: 
\[
\begin{align}
M = \frac{P(\mu g)}{\Delta A (phosphorous)}
\end{align}
\]
      Use &quot;Mean M&quot; to calculate the phosphorous content of test samples.<LI><B>Phosphorous / phytic acid content :</B></LI>
      Determine the absorbance of the both &quot;Free Phosphorous&quot; and &quot;Total Phosphorous&quot;. Substract the absorbance of the &quot;Free Phosphorous&quot; sample from the absorbance of the &quot;Total Phosphorous&quot; sample. Obtaining ΔA(phosphorous).
\[
\begin{align}
C^{o} = \frac{mean(M)\times 20 \times F}{10000 \times 1.0 \times \nu} \times \Delta A(phosphorous)
\end{align}
\]
<I>Where :</I>
\(20\) = original sample extract volume
\(F\) = dilution factor
\(10000\) = conversion from µg/g to g/100g
\(1.0\) = wigh of original sample material
\(\nu\) = sample volume

<B><I>It follows for phosphorous :</I></B>
\[
\begin{align}
C^{o} ~= \frac{mean(M)\times 20 \times 55.6}{10000 \times 1.0 \times \nu} \times \Delta A(phosphorous)\\
 ~= mean(M) \times 0.1112 \times \Delta A(phosphorous)
\end{align}
\]

<B><I>It follows for phytic acid :</I></B>
\[
\begin{align}
C^{o} = \frac{phosphorus}{0.282}
\end{align}
\]

</UL></UL></DIV><DIV class="textBox" id="TitrationofVitaminB12bySpectrophotometer"><H3>Vitamin B12 titration</H3><LI> Disrupt the microorganisms in a 0.1 N phosphate buffer solution containing 0.01% KCN at pH 5.5 for 20 min at 100°C.</LI><LI> Vitamin B12 formed intracellularly is determined spectrophotometrically in the dicyano-form at wavelength of 367 nm. </LI></DIV><DIV class="textBox" id="PCRpurification"><H3>PCR purification</H3><UL><LI>Add 5 volumes of resuspension buffer to 1 volume of PCR product in an 1.5mL microcentrifuge tube, mix by pipetting up and down
  </LI><LI>Transfer in a centrifugation column</LI><LI>Centrifuge 1 min at 14000 rpm</LI><LI>Throw the filtration product</LI><LI>Add 700μL of washing solution</LI><LI>Centrifuge 1 min at 14000 rpm</LI><LI>Throw the filtration product</LI><LI>Add 500μL of washing solution</LI><LI>Centrifuge 1 min at 14000 rpm</LI><LI>Throw the filtration product</LI><LI>Centrifuge 1 min at 14000 rpm</LI><LI>Throw the filtration product</LI><LI>Put the column in a sterile 1.5mL microcentrifuge tube</LI><LI>Add 45μL of DNAse/RNAse free water on the membrane</LI><LI>Wait 2 minutes</LI><LI>Centrifuge 2min at 10000rpm</LI><LI>Discard the column, DNA is saved in water</LI></UL></DIV><DIV class="textBox" id="AnalyticalDigestion"><H3>Analytical digestion protocol</H3><UL><LI>Prepare the following mix:
    </LI><LI>2µL 10X Digestion buffer</LI><LI>0.5µL Eco31I</LI><LI>0.5µL BbsI</LI><LI>2µL of DNA (200ng)</LI><LI>15µL water</LI></UL><LI>incubate 1h at 37°C</LI></DIV><DIV class="textBox" id="Annealing"><H3>Annealing Protocol</H3><UL><LI>Phosphorylation of the oligos</LI><UL><LI>5.6μL   DNAse/RNAse free water</LI><LI>6.0μL   oligo 1 (10µM)</LI><LI>6.0μL   oligo 2 (10µM)</LI><LI>2.0μL   10X T4 DNA ligase buffer</LI><LI>0.4μL   T4 PolyNucleotide Kinase</LI><B>Total:   20μL</B></UL><LI>incubate 30min at 37°C</LI><LI>add 1μL of 1M NaCl</LI><LI>incube 5min at 95°C</LI><LI>let the mix cool down</LI><LI>use 2μL of the mix as a 10X solution</LI></UL></DIV><DIV class="textBox" id="ChemicalTestCompetentCells"><H3>Chemical test for competent cells</H3><UL><LI>20-30 sec at 8-10 krpm for the DNA tube (from the kit).</LI><LI>Thow cell competent on ice. Label 2 ml eppendorf µtube for each concentration and put it on ice.</LI><LI>Add 1µl of DNA into each µtube.</LI><LI>Add 50µl of competent cell into each tube. Flick gently to mix. Incubate in ice for 30 minutes. Pre-heat waterbath at 42°C.</LI><LI>Heat-shock cell by placing in waterbath 1 minute. CAUTION : The top need to be close to the water level, but not immerge in.</LI><LI>Keep back the tube to ice for 5 minutes.</LI><LI>Add 200µl of SOC media, incubate 37°C for 2h. Prepare agar plate of the media that you want, and label it (3 plate for each concentration).</LI><LI>Add 20µl of each tube n  the appropriate plate. Incubate 37°C for OVN (12-16h).</LI></UL></DIV><DIV class="textBox" id="BananaSplit"><H3>BANANAAAA</H3><UL><LI>Line up ice cream scoops next to each other in an oval deep dish or a banana boat.
    </LI><LI>Cut the ends of the banana off (about 1/4 inch) while still in the peel.
    </LI><LI>Slice in half longways.
    </LI><LI>Pop each half of the banana out of the peel onto each side of the ice cream row, pressing down and in a little so it'll stay put.
    </LI><LI>Top the vanilla ice cream with the pineapple, the chocolate with the chocolate syrup and the strawberry with the strawberry sauce.
    </LI><LI>Spoon the wet walnuts over all three scoops of ice cream.
    </LI><LI>Top each scoop with some whipped cream and a cherry for each.
    </LI><LI>Enjoy!
  </LI></UL></DIV><DIV class="textBox" id="Electroporation"><H3>Electroporation</H3><UL><LI>Thaw electrocompetent cells on ice
      </LI><LI>Add 2µL of ligation product or 0.5µL of native plasmid to the cells
      </LI><LI>Transfer the cells in an 0.2mm electroporation cuvette
      </LI><LI>put the cuvette in the electroporation device and pulse the cells at 2.5kV, 200 Ohms and 25µF
      </LI><LI>add 200µL of LB right after pulsing
      </LI><LI>recover 2 hours at 37°C
      </LI><LI>plate 200µL and 50µL of the cells on LB + erythromycin (200µg/mL), incubate overnight at 37°C
    </LI></UL></DIV><DIV class="textBox" id="ElectrocompetentCells"><H3>Electrocompetent Cells</H3><UL><LI>Inoculate a 250mL LB flasks with 2.5mL of an overnight culture of cells</LI><LI>Incubate until the the DO600 reach 0.6 to 0.8</LI><LI>Place the cultures on ice for 15 minutes</LI><LI>Pour the culture in cold sterile 50mL falcon tubes</LI><LI>Centrifuge for 10 minutes at 4000rpm, 4°C</LI><LI>Throw the supernatant</LI><LI>Resuspend the cells in 50mL cold distilled water</LI><LI>Place the cells on ice for 10 minutes</LI><LI>Centrifuge them for 10 minutes at 4000rpm, 4°C</LI><LI>Throw the supernatant</LI><LI>Resuspend the cells in 12.5mL cold 10% glycerol</LI><LI>Place the cells on ice for 10 minutes</LI><LI>Centrifuge them for 10 minutes at 4000rpm, 4°C</LI><LI>Throw the supernatant</LI><LI>Resuspend the cells in 5mL cold 10% glycerol</LI><LI>Make aliquots of 50 to 100µL in microcentrifuge tubes and freeze them at -80°C</LI></UL></DIV><DIV class="textBox" id="Digestion"><H3>Digestion</H3><UL><LI>Prepare the following mix:
  </LI><LI>4μL of Enzyme 1 Fast Digest</LI><LI>4μL of Enzyme 2 Fast Digest</LI><LI>4μL of FastAP</LI><LI>12μL of Fast Digest buffer 10X</LI><LI>1 to 3 μg of DNA</LI><LI>up to 120μL of water</LI></UL><LI>mix by pipetting up and down</LI><LI>incube 10 to 20 minutes at 37°C</LI><LI>incube 10 minutes at 68°C to inactivate the enzymes</LI></DIV><DIV class="textBox" id="ElectrocompetentLactobacillus"><H3><I>Lactobacillus plantarum</I> electrocompetent cells</H3><UL><LI> Inoculate 5ml MRS medium with L. plantarum freezer stock.</LI><LI> Grow overnight at 30°C without shaking.</LI><LI> Add 1.25g glycine to two flasks of 50ml MRS medium.</LI><LI> Shake to dissolve.</LI><LI> Add 1ml overnight culture to each flask (1:50 dlution).</LI><LI> Culture for ~3 hours at 37°C with shaking.</LI><LI> Centrifuge culture for 5min at 4000g</LI><LI> Resuspend in 25ml ice-cold DI water.</LI><LI> Repeat it.</LI><LI> Centrifuge for 5min at 4000g</LI><LI> Resuspend in 5ml 50mM EDTA.</LI><LI> Incubate on ice for 5 minutes</LI><LI> Add 25ml ice-cold DI water.</LI><LI> Centrifuge culture for 5min at 4000g</LI><LI> Resuspend in 25ml ice-cold DI water.</LI><LI> Centrifuge culture for 5min at 4000g</LI><LI> Resuspend in 25ml ice-cold 0.5M Sucrose, 10% glycerol.</LI><LI> Repeat it.</LI><LI> Centrifuge culture for 5min at 4000g</LI><LI> Resuspend in 0.8ml ice-cold 0.5M Sucrose, 10% glycerol.</LI><LI> Aliquot 90μL of cell concentrate into ~20 microcentrifuge tubes.</LI><LI> Keep on ice until use (within the next two hours).</LI><LI> Add 10μL of plasmid DNA to the 90μL of cell concentrate.</LI><LI> Keep on ice for 5 minutes.</LI><LI> Put 1mm cuvettes on ice too.</LI><LI> Pipette the cell/DNA mixture into the cuvette</LI><LI> Electrporate at 1200 volts</LI><LI> Time constant should be ~5.0</LI><LI> Immediately transfer the electroporated cells to 900μL of MRS medium.</LI><LI> Incubate at 30°C for 2-3 hours.</LI><LI> Plate on medium with the appropriate antibiotic.</LI><LI> Incubate at 30°C for two days.</LI><LI> Pick a colony</LI></UL></DIV><DIV class="textBox" id="HeatShockTransformation"><H3>Heat Shock Transformation</H3><UL><LI>Thaw frozen chemically competent cells (20µL aliquots) on ice for 10min.</LI><LI>add 2µL of ligation product (or 0.5µL of miniprep) and incubate the cells 30sec at 42°C.</LI><LI>put the cells back on ice for 2min.</LI><LI>add 200µL of LB to the cells and incubate 2 hours at 37°C.</LI><LI>plate the cells on LB + erythromycin (150µg/mL) or LB + erythromycin (10µg/mL), incubation at 37°C overnight.</LI></UL></DIV><DIV class="textBox" id="VitaminATitration"><H3>Vitamin A Titration</H3><UL><LI>Take 1g of food (with particle &lt; 1mm of diameter) in bottle protected of the light, and add 15ml of hexane. Mix it.</LI><LI>Add again 15ml of hexane and shake during 5-10 minutes (do the same time, if you want compare two food sample).</LI><LI>In an other bottle protected of the light, do a filtration (coarse cellulose filter) of the previous solution.</LI><LI>Take the batter who don't pass the filter, and 15ml of hexane, shake 5 minutes, and filtered it like previously, to take all B-caroten as possible from the food sample.</LI><LI>Put in the fridge at -20°C to conserve it.</LI><LI>Obtain the result with a spectrophotometer with wavelenght of 450 nm (with a blank of hexane).</LI></UL></DIV><DIV class="textBox" id="VitaminATitrationHPLC"><H3>Vitamin A Titration using HPLC</H3><UL><LI>Suspend cells in 1 mL of sterile water.</LI><LI>Add 0.5 to 0.75 mm glass beads and vortex for 3 minutes.</LI><LI>Add 2.5 mL of 0.2% (wt/vol) pyrogallol dissolved in methanol and vortex for 3 minutes.</LI><LI>Add 1.25 mL of 60% (wt/vol) KOH and vortex for 10 seconds.</LI><LI>Incubate for 1 h at 75ºC, vortexing every 15 minutes.</LI><LI>Add appropriate amount of hexane.</LI><LI>Centrifuge tubes for 5 minutes at 2,800 rpm.</LI><LI>Pipette 1 mL of hexane into a cuvette. </LI></UL></DIV><DIV class="textBox" id="ElectrocompetentLactococcus"><H3><I>Lactococcus lactis</I>electrocompetent preparation</H3>
  Prepare the following solutions
  <UL><LI>Glycine/Sucrose solution</LI><UL><LI>80mL of 50% sucrose solution</LI><LI>29.2mL of 20% glycine solution</LI><LI>7.6mL of water</LI></UL><LI>Electroporation buffer
    </LI><LI>0.125g of NaHPO<SUB>4</SUB></LI><LI>34.2mL of 50% sucrose solution</LI><LI>100µL of MgCl<SUB>2</SUB> 1M </LI><LI>65.5mL of water</LI></UL><LI>Electroporation buffer + glycreol
    </LI><LI>187.5µL of 80% glycerol</LI><LI>812.5µL of electroporation buffer</LI>

  Protocol:<UL><LI>Inoculate 10mL of M17 + 1% glucose (GM17) with some <I>Lactococcus</I> and incubate overnight at 30°C</LI><LI>Inoculate a 100mL flask of GM17 with the 10mL of overnight culture</LI><LI>Incubate the flask at 30°C until DO reach 0.4 to 0.5</LI><LI>Add 100mL of the glycine/sucrose solution</LI><LI>Incubate 1 hour at 30°C with shaking</LI><LI>Split the 100mL culture in twyo 50mL centrifuge tubes</LI><LI>Centrifuge the cells 10 minutes at 6000 rpm</LI><LI>Throw the supernatant and work on ice for the rest of the protocol</LI><LI>Resuspend the cells in 20mL of electroporation buffer</LI><LI>Centrifuge the cells 10 minutes at 6000 rpm and then throw the supernatant</LI><LI>Resuspend the cells in 20mL of electroporation buffer</LI><LI>Centrifuge the cells 10 minutes at 6000 rpm and then throw the supernatant</LI><LI>Resuspend the cells in 0.5mL of electroporation buffer</LI><LI>Centrifuge the cells 10 minutes at 6000 rpm and then throw the supernatant</LI><LI>Resuspend the cells in 1mL of cold electroporation buffer + glycerol</LI><LI>Aliquots the cells in eppendorf tubes</LI><LI>Save the cells at -80°C</LI></UL></DIV><DIV class="textBox" id="YeastLysisNaOH"><H3>Yeast Lysis with NaOH</H3><UL><LI> 20 µl NaOH into PCR tubes</LI><LI> Pick colonies into NaOH
  </LI><LI> Incubate at 95°C for ~45 minutes</LI><LI> Centrifugate at 8000 krpm for 10 minutes</LI><LI> Use 1 µl supernatant as template in a (10 µl) PCR</LI></UL></DIV><DIV class="textBox" id="HeatShockYeast"><H3>Heat Shock Transformation for Yeast</H3><UL><LI>After growth, determine the titer of the yeast culture by using spectrophotometer : pipette 10µL of cells into 1mL of water in spectrophotometer cuvette and measure the OD at 600nm.</LI><LI>Add 2.5x10<SUP>8</SUP> cells to 50mL of 2X YPD in a culture flask.</LI><LI>Incubate the flask in a shaking incubator at 30°C until the cell culture is at least 2x10<SUP>7</SUP> cells.mL<SUP>-1</SUP></LI><LI>Denature 1mL of carrier DNA at 99°C for 5min and chill immediately in ice.</LI><LI>Harvest the yeast cells by centrifugation at 3,000g for 5min.</LI><LI>Resuspend the pellet in 25mL of sterile water and centrifuge at 3,000g for 5min at 20°C. Repeat this wash with sterile water 2 times.</LI><LI>Resuspend the last pellet in 1mL of sterile water.</LI><LI>Transfer the cell suspension to a 1.5mL microcentrifuge tube.</LI><LI>Centrifuge for 30s at 13,000g and discard the supernatant.</LI><LI>Resuspend the cells in 1mL of sterile water and pipette 100µL samples into 1.5mL microcentrifuge tubes, one for each transformation.</LI><LI>For each transformation :</LI><UL><LI>240µL of PEG 3350 (50% (w/v))</LI><LI>36µL of LiAc 1.0M</LI><LI>50µL of single-stranded carrier DNA (2.0mg.mL<SUP>-1</SUP>)</LI><LI>6µL of PCR product </LI><LI>28µL of water DNAse Free</LI></UL><LI>Place the tubes at 42°C for 40min.</LI><LI>Centrifuge the tubes at 13,000g for 30s in a microcentrifuge tube and remove the supernatant.</LI><LI>Pipette 1mL of YPD liquid medium into the transformation tube, and vortex mix to resuspend pellet.</LI><LI>Incubate 3h at 30°C to ensure good antibiotic expression.</LI><LI>Plate 2, 20 and 200µL of the cell suspension onto YPD medium with 200µm.mL<SUP>-1</SUP> antibiotic G418.</LI><LI>Incubate the plates at 30°C for 3 days.</LI></UL></DIV><DIV class="textBox" id="Ligation"><H3>Ligation</H3><UL><LI>Mix the following:

      </LI><LI>vector 100ng</LI><LI>insert 300ng</LI><LI>2µL T4 DNA ligase buffer 10X</LI><LI>1µL T4 DNA ligase</LI><LI>up to 20µL of water</LI></UL><LI>incubate 10 to 15 minutes at 22°C</LI><LI>incubate 5 minutes at 70°C</LI></DIV><DIV class="textBox" id="VitaminB2TitrationHPLC"><H3>Vitamin B2 Titration using HPLC</H3><B>Titration protocols for vitamin A </B>

Here, I find  2 different protocols to determine the quantity of B2 using HPLC.

I°) In a 250 ml conical flask, put 5g of food sample and 65 ml of 0.1M HCl.
Put it in 100°C for 30 minutes.
Cooling, then adjuste Ph to 4.5 with 2.5M sodium acetate.
Add 50ml Beta amylase, 500ml Takadiastase in water (small quantity). Put it at 37°C for 18h.
Dilutate with 125ml water.
Fitred with cellulose paper. And do a 2nd filtration with 0.2µm filter paper.

II°) Take Xg of food sample and add 15ml of 0.1M HCl.
Incubate at 100°C for 30 minutes.
Cooling, then add 2.5M sodium acetate (to have a Ph4.5) and add 10mg of Takadiastase.
Incubate it at 50°C during 3h.
Filtred with Albert No 135.
Made up to 25ml with milli Q water.
Filtered with 0.2µm nylon filter.</DIV><DIV class="textBox" id="Idli"><H3>Idli Recipe</H3><UL><LI> Soak rice and dall separately for 4-5 hours. (for 1.5 volume of rice, put the same volume of water, and 0.5 volume of dall, put the same volume of water).</LI><LI> Rinse both, mix it, add 2/3 volume of water of this volume.</LI><LI> Blend this mix, and let ferment  for 12-16 hours.</LI><LI> Then Cook it with idli cooker for 15 minutes.</LI></UL>


In the case where we add micro-organisms, we have 3 options:
<OL><LI>Add µorganisms during the soak phase</LI><LI>Add µorganisms after the blending phase.</LI><LI>Let soak µorganisms in the same time that rices and dall, and add it for the blending phase.</LI></OL>
The third was mainly used. </DIV><DIV class="textBox" id="MiniprepprotocolQIAGENkit"><H3>Miniprep protocol</H3><UL><LI>centrifuge an overnight culture of cells 10min at 4krpm
</LI><LI>throw the filtrate
</LI><LI>resuspend the pellet in 250μL of Cell Resuspension Solution then mix it
</LI><LI>transfer it in a 1.5mL microcentrifuge tube
</LI><LI>add 250mL of Cell lysis solution and mix by inverting several times
</LI><LI>incube until the liquid is clear, maximum 5min
</LI><LI>add 10μL of Alkalyne protease solution, mix by inverting several times and incube 3 to 4 min
</LI><LI>add 350μL of Neutralisation Solution and mix by inverting several times
</LI><LI>centrifuge 10min at 14krpm
</LI><LI>add the supernatant in a column
</LI><LI>centrifuge 1min, 14krpm, then throw the filtrat
</LI><LI>add 750μL Washing Solution
</LI><LI>centrifuge 1min, 14krpm, then throw the filtrat
</LI><LI>add 250μL Washing Solution
</LI><LI>centrifuge 2min, 14krpm, then throw the filtrat
</LI><LI>centrifuge 1min, 14krpm
</LI><LI>transfer the column in a sterile microcentrifuge 1.5ml tube
</LI><LI>add 50μL of DNAse/RNAse free water right on the membrane of the filter, wait 1min
</LI><LI>centrifuge 1min, 14krpm
</LI><LI>throw the column, plasmid is saved in water
</LI></UL></DIV><DIV id="protocolsGallerie"><DIV class="box"><DIV class="innerBox yellow"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols#AnalyticalDigestion"><DIV class="ptext"><P>Analytical digestion</P></DIV></A></DIV></DIV><DIV class="box"><DIV class="innerBox cyan"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols#Annealing"><DIV class="ptext"><P>Annealing</P></DIV></A></DIV></DIV><DIV class="box"><DIV class="innerBox magenta"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols#ChemicalTestCompetentCells"><DIV class="ptext"><P>Chemical Test for Competent Cell</P></DIV></A></DIV></DIV><DIV class="box"><DIV class="innerBox magenta"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols#Digestion"><DIV class="ptext"><P>Digestion</P></DIV></A></DIV></DIV><DIV class="box"><DIV class="innerBox cyan"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols#Electroporation"><DIV class="ptext"><P>Electroporation</P></DIV></A></DIV></DIV><DIV class="box"><DIV class="innerBox yellow"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols#ElectrocompetentCells"><DIV class="ptext"><P>Electrocompetent Cells</P></DIV></A></DIV></DIV><DIV class="box"><DIV class="innerBox cyan"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols#ElectrocompetentLactobacillus"><DIV class="ptext"><P><I>L. plantarum</I> Electrocompetent Cells</P></DIV></A></DIV></DIV><DIV class="box"><DIV class="innerBox cyan"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols#Electrophoresis"><DIV class="ptext"><P>Gel Electrophoresis with SYBR safe</P></DIV></A></DIV></DIV><DIV class="box"><DIV class="innerBox cyan"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols#GelPurification"><DIV class="ptext"><P>Gel purification</P></DIV></A></DIV></DIV><DIV class="box"><DIV class="innerBox cyan"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols#HeatShockTransformation"><DIV class="ptext"><P>Heat Shock Transformation</P></DIV></A></DIV></DIV><DIV class="box"><DIV class="innerBox cyan"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols#HeatShockYeast"><DIV class="ptext"><P>Heat shock transformation for yeast</P></DIV></A></DIV></DIV><DIV class="box"><DIV class="innerBox yellow"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols#Idli"><DIV class="ptext"><P>Idli Recipe</P></DIV></A></DIV></DIV><DIV class="box"><DIV class="innerBox yellow"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols#Ligation"><DIV class="ptext"><P>Ligation</P></DIV></A></DIV></DIV><DIV class="box"><DIV class="innerBox cyan"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols#MiniprepprotocolQIAGENkit"><DIV class="ptext"><P>Miniprep protocol</P></DIV></A></DIV></DIV><DIV class="box"><DIV class="innerBox yellow"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols#PCR"><DIV class="ptext"><P>PCR</P></DIV></A></DIV></DIV><DIV class="box"><DIV class="innerBox yellow"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols#PCRpurification"><DIV class="ptext"><P>PCR purification</P></DIV></A></DIV></DIV><DIV class="box"><DIV class="innerBox cyan"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols#Titrationacidphytic"><DIV class="ptext"><P>Titration of phytic acid</P></DIV></A></DIV></DIV><DIV class="box"><DIV class="innerBox cyan"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols#VitaminATitration"><DIV class="ptext"><P>Vitamin A Titration</P></DIV></A></DIV></DIV><DIV class="box"><DIV class="innerBox magenta"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols#VitaminATitrationHPLC"><DIV class="ptext"><P>Vitamin A Titration using HPLC</P></DIV></A></DIV></DIV><DIV class="box"><DIV class="innerBox yellow"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols#VitaminB2TitrationHPLC"><DIV class="ptext"><P>Vitamin B2 Titration using HPLC</P></DIV></A></DIV></DIV><DIV class="box"><DIV class="innerBox magenta"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols#TitrationofVitaminB12bySpectrophotometer"><DIV class="ptext"><P>Vitamin B12 Titration using spectrophotry</P></DIV></A></DIV></DIV><DIV class="box"><DIV class="innerBox magenta"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols#YeastDNAextractiondneasybloodandtissuekit"><DIV class="ptext"><P>Yeast DNA extraction </P></DIV></A></DIV></DIV><DIV class="box"><DIV class="innerBox magenta"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols#YeastLysisNaOH"><DIV class="ptext"><P>Yeast Lysis with NaOH</P></DIV></A></DIV></DIV></DIV><H2 style="color: white"> Miscellaneous </H2><DIV id="protocolsGallerie"><DIV class="box"><DIV class="innerBox cyan"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols#BananaSplit"><DIV class="ptext"><P>Banana Split</P></DIV></A></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>