<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Project_VitaminB2 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Paris Bettencourt/Project/VitaminB2</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="menuContainer"><DIV id="smallMenuTitle"><H1>Ferment It Yourself</H1><H3>iGEM Paris-Bettencourt 2O15</H3></DIV><UL><LI class="left"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Background" title="Background">Background</A></LI><LI class="left"><A href="https://2015.igem.org/Team:Paris_Bettencourt/Design" title="Design">Design</A></LI><LI class="left"><BUTTON class="menuButton">Project</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Micronutrients">Micronutrients</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Continuity">Continuity</A></LI><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Project/Manufacturing">Manufacturing</A></LI></DIV></UL><LI class="left"><BUTTON class="menuButton">Achievements</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Parts">Parts</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Medals">Medal Fulfillments</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Practices</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/SynBio_Challenge">SynBio Challenge</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/DILambda">DIÎ»</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Acceptance">Acceptance</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/Events">Events</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Practices/Rhizi">Rhizi</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Collaborations">Collaborations</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Safety</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Safety">Safety Overview</A></LI><LI><A href="https://2015.igem.org/Safety/Final_Safety_Form?team_id=1678">Safety Form</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/ReviewBoard">Review Board</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Notebook</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Notebook">Daily notebook</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Protocols">Protocols</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Bibliography">Bibliography</A></LI></DIV><LI class="right"><BUTTON class="menuButton">Team</BUTTON></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Team">Members</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Acknowledgments">Acknowledgments</A></LI></DIV><DIV class="subright"><LI><A href="https://2015.igem.org/Team:Paris_Bettencourt/Press">Press</A></LI><LI><A href="https://igem.org/Team.cgi?id=1678">Official Team Profile</A></LI></DIV></DIV><DIV id="riboflavin"><DIV class="contentContainer noBanner"><DIV id="banner"><H3>Vitamin B2 (riboflavin)
</H3></DIV></DIV><DIV class="textWrapper"><DIV class="textContent black" id="caffeine"><DIV id="overview" style="text-align: center"><TABLE><TBODY><TR><TH class="noBottom"><H3>Background</H3></TH><TH class="noBottom"><H3>Aims</H3></TH><TH class="noBottom"><H3>Results</H3></TH></TR><DIV id="overview" style="text-align: justify"><TR><TD class="noTop">Some microorganisms (e.g. <I>B. subtilis</I>) naturally produce riboflavin in high amount.
      </TD><TD class="noTop">Make Idli batter microorganisms produce high amount of riboflavin during fermentation to increase the vitamin B2 content of the batter.
      </TD><TD class="noTop"><UL><LI>Successfully cloned the full riboflavin biosynthesis pathway of <I>B. subtilis</I> in <I>E. coli</I> and made it over-produce vitamin B2 and proceeded to chromosomaly integrate the full pathway in <I>Lactococcus lactis</I></LI><LI>BioBrick submitted to the BioBrick Registry: <A href="http://parts.igem.org/Part:BBa_K1678001"> BBa_K1678001</A>, <A href="http://parts.igem.org/Part:BBa_K1678002"> BBa_K1678002</A>, <A href="http://parts.igem.org/Part:BBa_K1678003"> BBa_K1678003</A>, <A href="http://parts.igem.org/Part:BBa_K1678004"> BBa_K1678004</A></LI></UL></TD></TR></DIV></TBODY></TABLE></DIV></DIV><DIV class="textContent"><H2>Introduction</H2><DIV class="column-left" align="justify"><P>Riboflavin derived coenzymes (FMN and FAD) are the cofactors of numerous oxydo-reduction enzymes and are also used in the energy transduction process. It is also used in antioxidative reactions and is required for the metabolism of several other vitamins.</P><P>Riboflavin deficiency is a devastating problem in India. For socio-economic reasons, indians are not consuming riboflavin-rich food (dairy products, offal, eggs, almonds...).
Only 13% of the households meet the riboflavin dietary requirements and more than 70% of women and children of low-income groups (2009 Indian Council of Medical Research report) and 66% of children from middle-income groups have biochemical evidence of riboflavin deficiency (S. Swaminathan &amp; al., European J. Clin. Nut., 2013).</P><P>The most recognizable manifestation of advanced riboflavin deficiency are orolingual, dermal, corneal and hematological.
In earlier stages, riboflavin deficiency decreases psychomotor abilities, induces fatigue as well as itching and burning in the eyes. As it is involved in iron absorbtion, B2 deficiency often results in anemia. During pregnancy, a riboflavin deficiency can lead to limb-reduction in infants (M. S. Bamji et al., Bulletin of Nutrition Foundation of India, 1993).</P><P>Here we present how we readapted the riboflavin biosynthesis pathway from a natural riboflavin overproducer, <I>Bacillus subtilis</I> to engineer lactic acid bacteria (LAB) in order to make them produce high quantities of riboflavin while fermenting Idli batter.</P><H1 class="date one" id="design">Design</H1>

A broad range of micro-organisms have been characterized in Idli batter, mainly, Gram positive bacteria such as <I>Leuconostoc, Weissella, Pediococcus, Lactobacillus</I> (C. Saravanan &amp;<I>al</I>, J Food Sci Technol, 2015) and also <I>Lactococcus</I> ('Applied Microbiology',  Sanjai Saxena).
These bacteria producing neglectable amounts of riboflavin, far from meeting nutritional requirements.<P>LAB, such as <I>Lactococcus</I> and <I>Lactobacillus</I>, are used worldwide to ferment food. As both are present in Idli batter, we choose to work on <I>Lactobacillus plantarum</I>.

Also, to prevent most of the horizontal gene transfer events and to make the insertion more stable and resilient, we decided to  integrate the genes coding for the enzymes of the pathway into the chromosome.</P><P>First, we identified the different enzymes required to produce riboflavin in <I>B. subtilis</I> (K. -P. Stahmann, Appl. Mic.Biotech, 2000)(JB Perkins, J. of Ind. Mic. &amp; Biotech., 1999)(A. G. Vitreschak, Nuc. Acid Res., 2002).
The riboflavin biosynthesis pathway is detailed bellow.</P><P>Genetic sequences of <I>Bacillus subtilis</I> four enzymes were codon optimized for <I>Lactobacillus plantarum</I> on IDT website tool.
The enzyme was expressed by a promoter rather than a single operon expression to allow a precise tuning of the gene transcription. Two metabolic bottlenecks were identified in the pathway (M. Birkenmeier, Biotech. Lett., 2014). The first bottleneck correspond to the GTP cyclohydrolase activity of RibA. Overcoming the first bottleneck creates a second bottleneck corresponding to RibT's lumazine synthase activity.
To promote a differential expression of the four genes, we used the two synthetic <I>Lactobacillus</I> promoters p25 and p48 (respectively medium and strong expressing promoters) (I. Rud, Microbiology, 2006).
RibA expression was therefor promoted by p48, RibD and RibE by p25 and RibT by either p25 or p48.
For translation initiation, we used the consensus RBS for <I>Lactobacillus</I> (Tauer et al. Microbial Cell Factories 2014, 13:150).
T<SUB>ldh</SUB> terminator from <I>L. buchneri</I> lactate dehydrogenase gene was used to stop the transcription (Spath et al. Microbial Cell Factories 2012, 11:141).</P></DIV><DIV class="column-right" align="left">
GTP: Guanosine triphosphate
DARPP: 2, 5-diamino-6-ribosylamino-4 (3H)-pyrimidinone-5'-phosphate
ARPP: 5-amino-6-ribosylamino-2,4 (1H, 3H)-pyrimidinone-5'-phosphate
ArPP: 5-amino-6-ribosylamino-2,4 (1H, 3H)-pyrimidinedione-5'-phosphate
ArP: 5-amino-6-ribitylamino-2,4 (1H, 3H)-pyrimidinone
Ribu-5-P: Ribulose 5 Phosphate
DHBP: 3, 4-dihydroxy-2-butanone 4-phosphate
DRL: 6, 7-dimethyl-8-ribityl-lumazine

RibA: GTP cyclohydrolase II / 3,4-dihydroxy-2-butanone 4-phosphate synthase
RibD: Pyrimidine deaminase/reductase
RibE: Riboflavin synthase, beta-chain
RibT: lumazine synthase, (Riboflavin synthase, alpha-chain)</DIV><DIV class="column-left" align="left"><H1 class="date one" id="results">Results</H1>
The chromosome integration plasmid pKV6 was first altered to allow the insertion of the four genes by Golden Gate Assembly. This new plasmid was called p15.01.
Then, the four genes were successfully inserted in p15.01 and cloned in <I>E. coli</I>.<B>Figure 1: </B><I>pKV6 is an integrative plasmid we got from Christian Solem for repetitive marker free chromosomal integration. </I><B>Figure 2: </B><I>p15.01 is a derivative plasmid from pKV6, where a pair of BbsI restriction enzyme sites were introduced for Golden gate assembly. </I><B>Figure 3: </B><I>p15.06 is a derivative plasmid from p15.01 after golden gate assembly of Vitamin B2 pathway. Here RibA cassette has P48 synthetic promoter and all the other cassettes RibD, RibE, RibT have P25 synthetic promoter. </I><B>Figure 4: </B><I>p15.07 is a derivative plasmid from p15.01 after golden gate assembly of Vitamin B2 pathway. Here RibA and RibT cassettes have P48 synthetic promoter and rest two cassettes RibD, RibE have P25 synthetic promoter.</I></DIV><DIV class="column-right" align="justify">

After cloning, cells began to release riboflavin in their surrounding media, suggesting that the expression system was functioning in E. coli as well. 
Riboflavin concentration corresponding to the fluorescence (at 565nm) of the supernatant was estimated on a standard curve.<B>Figure 5: </B><I>Standard curve of riboflavin fluorescence</I>

From the standard curve, we derived a linear equation mapping riboflavin concentration to fluorescence (fluorescence = 16344*x + 418).<B>Figure 6: </B><I>Over-production of riboflavin by E.coli </I><I>E. coli</I> strains over-produce riboflavin up to 6Âµg/mL of media.<B>Figure 7: </B><I>E. coli strain diffusing riboflavin in plate</I><B>Figure 8: </B><I>Riboflavin revelation by fluorescence</I>

We characterized the functioning of promoter p48 in <I>E. coli</I> and BioBrick it (<A href="http://parts.igem.org/Part:BBa_K1678004">BBa_K1678004</A>).
Activity of p48 was shown by the expression of the fluorescent protein mCerulean.<B>Figure 9: </B><I>Fluorescence of mCerulean promoted by p48</I></DIV><DIV class="column-left" align="left"><H1 class="date one" id="discussion">Discussion</H1>
Though successfully we managed to produce Riboflavin (Vitamin B2) in <I>E. coli</I>, which was the vital obstacle to overcome but due to time constraints this project could have proceeded to introduce this B2 producing plasmid into <I>Lactococcus lactis</I> as mentioned by Christian Solem et al, 2013 and integrate the whole pathway into <I>L. lactis</I> chromosome. We used the same plasmid used by them for repetitive, marker-free, site specific chromosomal integration of foreign DNA, in this case vitamin B2. So for the further development of the project we can proceed to chromosomal integration of the vitamin B2 in <I>Lactococcus</I> and as once the plasmid is integrated, the site can be used for repetitive chromosomal integration. We can integrate Vitamin A and Vitamin B12 pathways in the same way thus probably creating a single cell with multiple vitamins producing capability. 
</DIV><DIV class="column-right" align="left"><H1 class="date one" id="attribution">Attributions</H1>
This project was designed and accomplished by Barthelemy Caron and Shazzad Hossain Mukit in consultation with Jason Bland, Ihab Boulas, Jake Wintermute and Antoine Decrulle. Most of the strains (DH5alpha, Top10, NEB turbo, STABLE) were kindly provided by Inserm U1001. Plasmids pKV6, PKV7 and pLB65 were provided by Christian Solem and Jianming Liu of National Food Institute, DTU (Denmark). Plasmid pNZ5319 for L.plantarum chromosomal integration was provided by Michiel Kleerebezem and Ingrid van Alen of NIZO food research, The Netherlands. pL2 backbone and two inserts, pele1 and eYFP, for positive control of Golden Gate assembly were provided by Antoine Decrulle. Special thanks to all the people for their important advice, and all the Paris Bettencourt team members for making this adventure such an amazing experience.
</DIV></DIV></DIV></DIV></BODY></HTML>