<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Valencia_Biocampus_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Valencia_Biocampus/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Valencia_Biocampus/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Valencia_Biocampus/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Valencia_Biocampus/Project&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Valencia_Biocampus/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Valencia Biocampus/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="container" id="headerWrapper"><UL id="header-top-menu" class="header-top-menu pull-right"><LI id="menu_a_project"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Project">Project</A></LI><LI id="menu_a_team"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Team">The Team</A></LI><LI id="menu_a_results"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Results">Results</A></LI><LI id="menu_a_modeling"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Modeling">Modeling</A></LI><LI id="menu_a_biobricks"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Biobricks">Biobricks</A></LI></UL><UL id="header-top-menu2" class="header-top-menu pull-right"><LI id="menu_a_human-practices"><A href="https://2014.igem.org/Team:Valencia_Biocampus/HumanPractices">Human Practices</A></LI><LI id="menu_a_interlab-study"><A href="https://2014.igem.org/Team:Valencia_Biocampus/InterlabStudy">Interlab Study</A></LI><LI id="menu_a_achievements"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Achievements">Achievements</A></LI><LI id="menu_a_attributions"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Attributions">Attributions</A></LI><LI id="menu_a_art"><A href="https://2014.igem.org/Team:Valencia_Biocampus/ArtAndMore">Art&amp;More</A></LI></UL></DIV></DIV><DIV class="container withBackground"><H1><SPAN class="mw-headline" id="The_St2OOL_project"> The St<SUP>2</SUP>OOL project</SPAN></H1><P><UL id="myTab" class="nav nav-tabs" data-tabs="tabs"><LI class="active"><A href="#Overview" data-toggle="tab">Overview</A></LI><LI><A href="#Stability" data-toggle="tab"><SPAN class="corporative bold">St</SPAN>ability</A></LI><LI><A href="#Standardization" data-toggle="tab"><SPAN class="corporative bold">St</SPAN>andardization</A></LI><LI><A href="#Orthogonality" data-toggle="tab"><SPAN class="corporative bold">O</SPAN>rthogonality</A></LI><LI><A href="#OpenLicense" data-toggle="tab"><SPAN class="corporative bold">O</SPAN>pen <SPAN class="corporative bold">L</SPAN>icense</A></LI><LI><A href="#TheSeat" data-toggle="tab">The Seat of the <SPAN class="corporative bold">St<SUP>2</SUP>OOL</SPAN></A></LI></UL></P><DIV id="my-tab-content" class="tab-content"><DIV class="tab-pane fade in active" id="Overview"><H2 style="text-align:center;padding:0;margin:0;">Overview</H2><DIV class="watercolored" style="margin-top:-50px;"><P>
      Synthetic Biology implies an engineering perspective on biotechnology. Similarly to man-made objects, cells are expected to be decoupled, modified and even built from scratch. However, there is a general assent on the current difficulties for fully -and predictably- engineering living organisms, which are always subjected to strong evolutionary constraints. The St<SUP>2</SUP>OOL project aims at deeply studying four of the key engineering pillars of Synthetic Biology. <B>St<SUP>2</SUP>OOL</B> stands for <B>St</B>andardization, <B>St</B>ability, <B>O</B>rthogonality and <B>O</B>pen Licence.
      </P><P><STRONG>We have performed a vast range of experimental studies to find out how standard, stable, orthogonal and patentable ten selected Biobrick parts are</STRONG>. The results of our work -such as those on standardization and orthogonality-, surprised us, since a huge amount of data from different techniques clearly indicate that engineering principles should not be taken for granted in Synthetic Biology. In a second approach, still under way, we are currently isolating through functional metagenomics new biological parts -promoters-, not because of their strength but because of their particularly standard, stable or orthogonal behavior. Taken together, the results of our project are expected to contribute in answering this key question: <STRONG>Is life fully engineerable?</STRONG> and if not, can we improve the &quot;engineerability&quot; of life?


    </P></DIV></DIV><DIV class="tab-pane fade" id="Stability"><DIV class="row" style="margin-bottom:-15px"><H2><SPAN class="mw-headline" id="Stability"> Stability </SPAN></H2><DIV class="col-sm-8"><P>
A computer works optimally at 10ºC to 35ºC, but it still works very well in a much wider range of temperatures. What about engineered bacteria? What are the limits of their -expected- behavior? We wanted to test two things: How stable is <EM> Escherichia coli </EM>when subjected to sub-optimal conditions (stability of the chassis), and what is the behavior of ten different Biobricks in two different strains of<EM> E. coli, </EM>XL1 Blue and DH5α (stability of the output signals), under those conditions. Stability was determined by subjecting either untransformed or transformed cells to different types of stress, including a range of extreme or cycling temperatures, radiation, pH, salinity or vacuum.  By measuring bacterial growth, we have been able to characterize the robustness of <EM> E. coli </EM> as a chassis for Synthetic Biology. By subjecting Biobrick-transformed cells to environmental stresses and then measuring their outputs, we have been able to determine which are the boundaries of Biobricks and how narrow or large are the conditions under which they behave as expected.
</P></DIV></DIV></DIV><DIV class="tab-pane fade" id="Standardization"><H2><SPAN class="mw-headline" id="Standardization"> Standardization </SPAN></H2><P>
  One should not assume that a functional module working fine in one cell type will work the same way in even a closely related cell type <A href="http://books.google.es/books?hl=es&amp;lr=&amp;id=9iKDQEFM0ScC&amp;oi=fnd&amp;pg=PR13&amp;dq=Modelling+Biological+Populations+in+Space+and+Time&amp;ots=eVAhPiiIT9&amp;sig=fGjzEYakHztERdAs9gw5eKfIfbI#v=onepage&amp;q=Modelling%20Biological%20Populations%20in%20Space%20and%20Time&amp;f=false" target="_blank">(Renshaw, 1993)</A>. Whereas traditional engineering practices typically rely on the standardization of parts, the uncertain and intricate nature of biology makes standardization in the synthetic biology field difficult. Beyond typical circuit design issues, synthetic biologists must also account for cell death, crosstalk, mutations, intracellular and extracellular conditions, noise and other biological phenomena. As the number of system components grows, it becomes increasingly difficult to coordinate component inputs and outputs to produce the overall desired behaviour <A href="http://www.nature.com/nrm/journal/v10/n6/pdf/nrm2698.pdf" target="_blank">(Purnick &amp; Weiss, 2009)</A>. In the St²OOL Project, we wanted to check how standard biobricks are. For this goal, we have used a set of strains of <I>Escherichia coli</I>  transformed, in single transformation, with <A href="https://2014.igem.org/Team:Valencia_Biocampus/Biobricks" target="_blank">ten different Biobricks</A> whose outputs have been measured by fluorometry, colorimetry or luminometry. The aim of this approach is simple and fits with the answer to the following question:
</P><P>
  Will the Biobricks behave the same way independently of the host strain they have been transformed into? The answer to this question, in our result section! 
</P></DIV><DIV class="tab-pane fade" id="Orthogonality"><H2><SPAN class="mw-headline" id="Orthogonality"> Orthogonality </SPAN></H2><DIV class="row"><P>
    Do biobricks behave independently when cotransformed in a cell? In other words, can we consider them to be orthogonal? Orthogonality is a mathematical
    concept that refers to the independence of behavior. Translating this concept into the biological realm, two constructions can be considered orthogonal
    when they only interact at specific and predictable interphases and do not disturb each other. This is a desirable feature of Biobricks if one wants to
    build synthetic circuits with predictable and robust behaviors.
</P><P>
    To study this pillar of synthetic biology we have analyzed different combinations of Biobrick parts in cotransformed <EM> Escherichia coli </EM> cells. We have studied their behavior in comparison with simple transformants, and therefore determined whether their output was the predicted one. We did that at two different levels: the population level -with standard fluorometry assays-, and the cell level -by using flow cytometry-.
</P><P>
But we also studied orthogonality at a third level: the proteome. We have analyzed and compared the whole proteome of three different strains of <I>E. coli</I>: a wild-type strain, a strain expressing a simple Biobrick part, and a strain carrying the empty cloning plasmid. With this experiment, we have determined the orthogonality of a Biobrick part with respect with the other parts naturally present in the chassis.   
</P></DIV></DIV><DIV class="tab-pane fade" id="OpenLicense"><H2><SPAN class="mw-headline" id="Open_License"> Open License </SPAN></H2><P>
    We did not want the Human Practices section to be a cosmetic addendum to our iGEM project. In The St<SUP>2</SUP>OOL, we have performed a critical but
    constructive analysis of four of the pillars of the competition. Three of them are technical, engineering assumptions (<STRONG>stability</STRONG>, <STRONG>standardization</STRONG>,
    <STRONG>orthogonality</STRONG>). The fourth leg of our St<SUP>2</SUP>OOL is for us at least as important as the other three, and deals with the answer to this difficult
    question: who owns synthetic constructions?</P><P>
    iGEM is linked to the Registry of Standard Biological Parts, a non-profit organization (as iGEM Foundation itself) archiving and distributing thousands of
    BioBrick parts for free. No patents, no licenses, no royalties. Just Open Access (either totally free access or under
    certain conditions such as a creative commons subscription). As stated in their Website, it’s “an open community that runs and grows on the Get &amp; Give
    (&amp; Share) philosophy. Users get parts, samples, data, and tools from the Registry to work on their synthetic biology projects. They will give back to
    the Registry the new parts they have made, as well as data and experience on new and existing parts”. But of course, the world of Biotechnology/SynBio is
    far more than iGEM and a range of Intellectual Property protection formulas do exist. Is iGEM an island of Free Access in an ocean of patentable SynBio
    achievements?
</P><DIV style="text-align:center; padding:20px;"><A href="https://2014.igem.org/Team:Valencia_Biocampus/HumanPractices" class="btn btn-lg btn-default">Human Practices Section </A></DIV></DIV><DIV class="tab-pane fade" id="TheSeat"><H2><SPAN class="mw-headline" id="The_Seat_of_the_St.C2.B2OOL"> The Seat of the St²OOL </SPAN></H2><P>
    By checking how standard, stable, orthogonal or patentable Biobricks are we have performed an analysis of the four engineering pillars of Synthetic Biology. But we wanted to
    combine these pillars, these four legs of our St<SUP>2</SUP>OOL, with a truly synthetic and constructive approach.
</P><P>
    In The Seat of the St<SUP>2</SUP>OOL we designed a functional metagenomic strategy aiming at isolating promoter sequences from natural
    environments. The goal was to characterize the promoter sequences in order to select those with the highest stability, standard and orthogonal behaviors. We have
    worked with soil and compost samples, from which we have isolated total metagenomic DNA, which was then digested, ligated into an expression vector and
    used to build a library in <EM>E. coli</EM>. We planned to subject transformants –clones with a promoter-like natural sequence- to a range of stresses and
    subtransformation steps to find out their stability and standard/orthogonal behavior. To the best of our knowledge, this is the first Synthetic Biology
    functional metagenomic approach aiming at selecting biological parts not because of their “strength” but because of their compatibility with other parts
    and suitability for engineering purposes.</P><P>
    Finally, one technical –yet important- detail: we have not used neither commercial kits nor manual protocols for isolating metagenomic DNA. We have built our own robot, <A href="https://2014.igem.org/Team:Valencia_Biocampus/Results#TheSeatSection" target="_blank">&quot;the TOOL&quot;</A>, able to automatically isolate metagenomic DNA from environmental samples.
</P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Valencia_Biocampus/Project">http://2014.igem.org/Team:Valencia_Biocampus/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Valencia_Biocampus/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Valencia_Biocampus/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Valencia_Biocampus/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Valencia_Biocampus/Project&amp;oldid=374379" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>