<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Valencia_Biocampus_ResultsDemoTestATOPE"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Valencia_Biocampus/ResultsDemoTestATOPE">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Valencia_Biocampus/ResultsDemoTestATOPE&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Valencia_Biocampus/ResultsDemoTestATOPE&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Valencia_Biocampus/ResultsDemoTestATOPE&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Valencia_Biocampus/ResultsDemoTestATOPE" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Valencia Biocampus/ResultsDemoTestATOPE</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="container" id="headerWrapper"><UL id="header-top-menu" class="header-top-menu pull-right"><LI id="menu_a_project"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Project">Project</A></LI><LI id="menu_a_team"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Team">The Team</A></LI><LI id="menu_a_results"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Results">Results</A></LI><LI id="menu_a_modeling"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Modeling">Modeling</A></LI><LI id="menu_a_biobricks"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Biobricks">Biobricks</A></LI></UL><UL id="header-top-menu2" class="header-top-menu pull-right"><LI id="menu_a_human-practices"><A href="https://2014.igem.org/Team:Valencia_Biocampus/HumanPractices">Human Practices</A></LI><LI id="menu_a_interlab-study"><A href="https://2014.igem.org/Team:Valencia_Biocampus/InterlabStudy">Interlab Study</A></LI><LI id="menu_a_achievements"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Achievements">Achievements</A></LI><LI id="menu_a_attributions"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Attributions">Attributions</A></LI><LI id="menu_a_art"><A href="https://2014.igem.org/Team:Valencia_Biocampus/ArtAndMore">Art&amp;More</A></LI></UL></DIV></DIV><DIV class="container withBackground"><H1>
   Our Results<UL id="myTab" class="nav nav-tabs" data-tabs="tabs"><LI class="active"><A href="#StabilitySection" data-toggle="tab"><SPAN class="corporative bold">St</SPAN>ability</A></LI><LI><A href="#StandardizationSection" data-toggle="tab"><SPAN class="corporative bold">St</SPAN>andardization</A></LI><LI><A href="#OrthogonalitySection" data-toggle="tab"><SPAN class="corporative bold">O</SPAN>rthogonality</A></LI><LI><A href="#OpenLicenseSection" data-toggle="tab"><SPAN class="corporative bold">O</SPAN>pen <SPAN class="corporative bold">L</SPAN>icense</A></LI><LI><A href="#TheSeatSection" data-toggle="tab">The Seat of the <SPAN class="corporative bold">St<SUP>2</SUP>OOL</SPAN></A></LI><LI><A href="#ModelingSection" data-toggle="tab">Modeling</A></LI></UL></H1><H2>Stability</H2><P><STRONG><EM>Escherichia coli</EM> is the lab rat of the bacterial world.</STRONG>
               We have performed the most complete characterization of the stability of <EM>E. coli</EM> in order to both confirm its optimality as a chassis for Synthetic Biology, and to determine the behavior of Biobricks under sub-optimal environmental conditions. We have subjected wild type strains to a range of stressful conditions (see just below &quot;the limits of <EM>E. coli</EM>&quot;) and measured how our Biobricks behaved.
            </P></DIV></DIV><DIV class="tab-pane fade" id="StandardizationSection"><H2>Standardization</H2></DIV><DIV class="tab-pane fade" id="OrthogonalitySection"><DIV class="col-sm-3 tocWrapperiGEM"><UL id="tocifyiGEM-header0" class="tocifyiGEM-header nav nav-list"><LI class="tocifyiGEM-item" data-unique="Orthogonalityresults" style="cursor: pointer;"><A>Orthogonality results</A></LI></UL><UL id="tocifyiGEM-header1" class="tocifyiGEM-header nav nav-list"><LI class="tocifyiGEM-item" data-unique="1.StandardFluorimetryassays" style="cursor: pointer;"><A>1. Standard Fluorimetry assays</A></LI><UL class="tocifyiGEM-subheader nav nav-list" data-tag="3" style="display: none;"><LI class="tocifyiGEM-item" data-unique="Generalprotocol(1)" style="cursor: pointer;"><A> General protocol(1) </A></LI><LI class="tocifyiGEM-item" data-unique="Results(111)" style="cursor: pointer;"><A> Results (111)</A></LI></UL></UL><UL id="tocifyiGEM-header2" class="tocifyiGEM-header nav nav-list"><LI class="tocifyiGEM-item" data-unique="2.Flowcytometry" style="cursor: pointer;"><A>2. Flow cytometry</A></LI><UL class="tocifyiGEM-subheader nav nav-list" data-tag="3" style="display: none;"><LI class="tocifyiGEM-item" data-unique="Orthogonalityatthecelllevel" style="cursor: pointer;"><A>Orthogonality at the cell level</A></LI><LI class="tocifyiGEM-item" data-unique="Orthogonalitychangesintime" style="cursor: pointer;"><A>Orthogonality changes in time</A></LI><LI class="tocifyiGEM-item" data-unique="Fluorescenserateswithincultures:averystableimbalance" style="cursor: pointer;"><A>Fluorescense rates within cultures: a very stable imbalance</A></LI></UL></UL><UL id="tocifyiGEM-header3" class="tocifyiGEM-header nav nav-list"><LI class="tocifyiGEM-item active" data-unique="3.Proteomics" style="cursor: pointer;"><A>3. Proteomics</A></LI><UL class="tocifyiGEM-subheader nav nav-list" data-tag="3" style="display: block;"><LI class="tocifyiGEM-item" data-unique="Generalprotocol(2)" style="cursor: pointer;"><A>General protocol (2)</A></LI><LI class="tocifyiGEM-item" data-unique="Resultsproteomics" style="cursor: pointer;"><A>Results proteomics</A></LI></UL></UL><UL id="tocifyiGEM-header4" class="tocifyiGEM-header nav nav-list"><LI class="tocifyiGEM-item" data-unique="4.Conclusions" style="cursor: pointer;"><A>4. Conclusions</A></LI></UL></DIV></DIV></DIV><DIV class="col-sm-9 pull-right"><H2>Orthogonality results</H2><P>
               In Synthetic Biology, <B>two constructions can be considered orthogonal when they only interact at specific and predictable interphases</B>, and do not disturb
               each other. We have studied this desirable feature of Biobrick parts by combining two constructions in the same cell and comparing this output with the one
               produced by simple transformations. To do this, <B>we used both standard fluorimetry assays and flow cytometry</B>, which allowed to get information at the cell level. Then, we wanted to go one step further: what is the effect of a simple transformation (a plasmid with a Biobrick part) into the cell architecture? To get some insights about this,
               <B>we performed a proteomic analysis</B> in which the whole proteome of an <EM>E. coli</EM> strain transformed with a Biobrick part was
               compared to that of the non-transformed, control strain. Last, but not least, <A href="https://2014.igem.org/Team:Valencia_Biocampus/Modeling#OrthogonalityTab" target="_blank"><STRONG>a detailed set of equations </STRONG></A>modeling the behavior of cells carrying two Biobrick parts was developed.
            </P><H2>1. Standard Fluorimetry assays</H2><P>
               We performed experiments with two different combinations of Biobrick parts containing a fluorescent protein under the control of a promoter sequence from
               Anderson’s promoter collection: Bb1 (GFP with strong promoter J23104) + Bb2 (RFP with strong promoter J23104) and Bb2 + Bb3 (GFP with less strong promoter
               J23110). In all cases, several controls were used: wild type (non-transformed) cells, simple transformations with either the biobrick or an empty plasmid,
               and co-transformations of the biobricks and the empty plasmids. It has to be noted that there was a high background emission of red fluorescence by the cells in all cases. Our results are shown in Figures 1 and 2.
            </P><H3> General protocol(1) </H3><UL><LI>
                  Strike the desired strains on LB plates with the appropriate antibiotic. Allow to grow by incubating at 37°C at least 24 hours.
               </LI><LI>
                  Gather a large amount of bacteria with an inoculation loop and transfer to 1 mL of LB. Incubate during 20 minutes at room temperature.
               </LI><LI>
                  Inoculate 100 µL of bacterial suspension in a tube containing 3 mL of LB with antibiotic. Perform three biological replicas of each combination of Biobrick
                  parts.
               </LI><LI>
                  Incubate until OD<SUB>600</SUB> is between 0.1-0.3.
               </LI><LI>
                  Measure OD<SUB>600</SUB>.
               </LI><LI>
                  Measure GFP fluorescence (exc.= 493nm; emis.=505nm) and RFP fluorescence (exc.= 576 nm; emis.= 592 nm) in a FP6200 spectrofluorimeter (Jasco, Easton, MD) fluorometer using standard plastic cuvettes.
               </LI><LI>
                  With excel, normalize fluorescence by cell density, calculate the average and standard deviation taking into account the three biological replicas, and represent the data.
               </LI></UL><H3> Results (111)</H3><DIV class="thumbnail col-sm-11 center-block"><DIV class="caption"><STRONG>
                  Figure 1. Fluorescence emission in XL1-Blue <EM>Escherichia coli</EM> strains co-expressing Biobrick parts 1, 2, and 3 reveals asymmetry of behaviors (lack of
                  orthogonality).
                  </STRONG>
                  Left, cells containing both Biobrick parts 1 and 2, Biobrick part 1 and an empty kanamycin plasmid (e.k.p), Biobrick part 2 and an empty ampicillin plasmid
                  (e.a.p.), Biobrick part 1, and Biobrick part 2. Right: Cells containing both Biobrick parts 2 and 3, Biobrick part 2 and an empty kanamycin plasmid
                  (e.k.p), Biobrick part 3 and an empty ampicillin plasmid (e.a.p.), Biobrick part 2, and Biobrick part 3. XL1-Blue <EM>E. coli</EM> cells containing no
                  plasmids or empty plasmids were used as controls. Both green (505 nm) and red (592 nm) fluorescence intensity (FI) were measured for all samples and
                  normalized by the optical density at 600 nm (OD600).
               </DIV></DIV><DIV class="thumbnail col-sm-7 center-block"><DIV class="caption"><STRONG>
                  Figure 2. Fluorescence emission by DH5α <EM>E. coli</EM> strains co-expressing Biobrick parts 1 and 2 also reveals non orthogonal expression of
                  Biobrick parts.
                  </STRONG>
                  Cells containing both Biobrick parts 1 and 2, Biobrick part 1 and an empty kanamycin plasmid (e.k.p), Biobrick part 2 and an empty ampicillin plasmid
                  (e.a.p.), Biobrick part 1, and Biobrick part 2 were analyzed. XL1-Blue <EM>E. coli</EM> cells containing no plasmids or empty plasmids were used as
                  controls. Both green (505 nm) and red (592 nm) fluorescence intensity (FI) were measured for all samples and normalized by the optical density at 600 nm
                  (OD600).
               </DIV></DIV><P><STRONG>When in co-transformation, Bb1 and Bb2 showed a clearly asymmetric expression</STRONG>
               . <STRONG>Bb1 was highly expressed in comparison to Bb2 (2-fold in strain XL1-Blue and 3-fold in DH5α</STRONG>). A
               similar phenomenon was observed when either Bb1 or Bb2 were cotransformed with empty plasmids. Again, less fluorescence was detected in comparison to the
               simple transformants. Surprisingly, this fluorescence decay was strikingly different between both <EM>E. coli</EM> strains: Bb1 strongly lowered its
               expression in DH5α, whereas Bb2 showed a higher decay in XL1-Blue cells.
            </P><P>
               However, the result was completely different when Bb2 and Bb3 were present in the same cell. In the XL1-Blue strain, the fluorescence intensity of Bb2 was
               as high as that of the single transformant, whereas the fluorescence of Bb3 –containing a less strong promoter- dropped dramatically. Fluorescence
               intensity of Bb3 was not detectable in strain DH5α under our experimental conditions.  
            </P><P><STRONG>
               With these results with co-transformants, we have to conclude that Biobrick parts, even if controlled by the same promoter and cloned into the same
               plasmid, are not expressed at similar rates.
               </STRONG>
               But, is this lack of orthogonality stable and reproducible? In other words, is the asymmetric output always the same or does it fluctuate in time or among cells? We were able to answer this question by using flow cytometry. Check our results below!
            </P><H2>2. Flow cytometry</H2><P>
               To study cell-level fluorescence we used <B>flow cytometry</B>.  To prepare the samples, XL1-Blue cultures (wild type, simple transformants for Bb1 and Bb2, and cotransformants with Bb1+Bb2) were grown in LB with the appropriate antibiotic, harvested and resuspended in 1mL PBS.  From now on, we will refer to Bb1 as GFP, and Bb2 as RFP.
            
            </P><H3>Orthogonality at the cell level</H3><P>
            Our first result, shown in Figure 3, is the <B>confirmation of non-orthogonal behavior as deduced by a higher expression of GFP in comparison to RFP in cotransformed XL1-Blue cultures</B>. There is four to five times more expression of GFP in comparison to RFP (Fig 3D).  These results are similar to those we previously obtained by fluorimetry (with younger cultures, 5 h), in which an imbalanced GFP:RFP rate of about 2:1 was detected.
            It has to be stressed that, unexpectedly, only a fraction (around 10%) of the cells exhibited activity of both reporter proteins, indicating that <B>the fluorescence of a culture as it is usually measured (fluorimetry) represents in fact and average of a very diverse pool of individual outputs</B>, including mono-active cells. 
            We plan to repeat the assay using the reverse conditions: GFP in a Kan resistant plasmid, and RFP in an Amp resistant plasmid.  
            </P><DIV class="thumbnail col-sm-7 center-block"><DIV class="caption"><STRONG>
                  Figure 3. Dot plots of XL1-Blue reveal a non-orthogonal behavior.
                  </STRONG>
                  Dot plots obtained by flow cytometry. A, B, C &amp; D correspond to wild type, GFP transformant, RFP transformant and cotransformed cells, respectively.  FITC-A, or X axis, indicates green fluorescence (505 nm), whereas PE-A, or Y axis, indicates red fluorescence (592 nm).
               </DIV></DIV><H3>Orthogonality changes in time</H3>
            Figure 4 shows a clear correlation of fluorescense and time in mono-transformants.  Fluorescence is very low for all samples at 5h (approximately 30% of cells transformed with Bb1 express GFP but only 1% of cells transformed with Bb2 express RFP); moderate at 10h for GFP mono-transformants (41% and 1.5% respectively); and maximum after 20h, when GFP is expressed by 80% of the cells, and RFP is expressed by approximately 45% of the cells. Regarding co-transformants, fluorescence also increased with time but even after 20h only a fraction (10%) of doubly fluorescent cells were detected.
            <DIV class="thumbnail col-sm-9 center-block"><DIV class="caption"><STRONG>
                  Figure 4. XL1-Blue dot plots indicating an increase of fluorescent protein expression with time. 
                  </STRONG>
                  Samples of the four variations (wild type, GFP transformants, RFP transformants, and cotransformed cells) are taken at 5, 10 and 20 hours.  FITC-A, or X axis, represents green fluorescence (505 nm).  PE-A, or Y axis, represents red fluorescence (592 nm).
               </DIV></DIV><P>
               These results indicate that <B>higher fluorescence levels are reached when cells are reaching stationary phase</B> and that <B>even long incubation times fail to yield a majoritary population of co-transformed cells with dual GFP and RFP activity</B>.
            </P><H3>Fluorescense rates within cultures: a very stable imbalance</H3>
            Our last assay aimed at answering this paradoxical question: <B>is the lack of orthogonality a stable trait?</B> In other words, is the GFP:RFP imbalance always the same? The answer, according to Figure 5 is clear: <B>Yes, independent cultures exhibit a striking similarity in their flow cytometry plots</B>. Almost half of the cell population only displays GFP activity, whereas only a relatively small fraction is either displaying RFP fluorescence or both.
            <DIV class="thumbnail col-sm-7 center-block"><DIV class="caption"><STRONG>
                  Figure 5. Independent cultures of cotransformed cells indicate the lack of orthogonality is stable.
                  </STRONG>
                  Analysis by flow cytometry of XL1-Blue cotransformed cells.  One dot plot is shown for each independent culture.  FITC-A indicates green fluorescence (505 nm).  PE-A indicates red fluorescence (592 nm). Below: diagrams representing the percentage of cells that express no fluorescence, RFP, GFP, or both.
               </DIV></DIV><H2>3. Proteomics</H2>
            Our experiments involving <I>E. coli</I> cells co-expressing two different Biobrick parts revealed a lack of orthogonality between them. But <B>what is going on with all the other parts naturally present in the chassis?</B> Does the expression of a Biobrick part interact or interfere with any of the genes expressed by the host? In order to test this hypothesis, <B>we performed a proteomic analysis on the simplest scenario</B>: we compared the proteome of the DH5α <I>E. coli</I> strain expressing Biobrick part 1 (consisting of a green fluorescent protein under the control of a constitutive promoter) with that of the same strain carrying the empty cloning plasmid and the non-transformed wild-type strain.
            <H3>General protocol (2)</H3><UL><LI>
                  Set up three independent 5-mL cultures of the strains named before as we explained above for the standard fluorimetry assays.
               </LI><LI>When an OD value between 0.2 and 0.4 is reached, pellet the cells at 4ºC and then wash them with sterile ice-cooled PBS buffer.</LI><LI>Resuspend the cells in 1 mL of ice-cooled sterile PBS buffer.</LI></UL><P>
               After this, we kept the tubes on ice and brought them to the Proteomics lab of the University of Valencia. The proteomic analysis was carried out in the SCSIE_university of Valencia Proteomics Unit , a member of ISCIII ProteoRed Proteomics Platform. Once there, proteins were isolated and quantified with the supervision of the technicians. Finally, proteins were digested and labeled by using the iTRAQ technology, which allowed us to work with all the samples in the mass spectrometer at a time and then assign each identification to a particular sample.
            </P><H3>Results proteomics</H3>
               A general view on the protein profile of each sample was firstly observed on an SDS-PAGE gel (Figure 6). Strong differences were detected, as expected, in the bands corresponding to the GFP and the β-lactamase encoded by the ampicillin-resistance gene of the pUC57 plasmid used for cloning. 
               <DIV class="thumbnail col-sm-7 center-block"><DIV class="caption"><STRONG>
                     Figure 6. SDS-PAGE gel showing the protein profile of the samples used for proteomics analysis.
                     </STRONG>
                     WT: wild-type <I>E. coli</I> strain DH5α (2 replicates); Bb1: Biobrick part 1-expressing strain (3 replicates); ᴓ: strain carrying the empty cloning plasmid. Red and green arrows indicate β-lactamase and GFP, respectively.
                  </DIV></DIV><P>
                  An average of <B>1500 proteins per sample were identified</B> with the mass spectrometer. Among them, <B>472 proteins could be properly quantified</B>. Figure 7 shows the variation of a subset of proteins among samples. Notice that chaperone proteins are specially over-represented in the strain expressing Bb1, whereas proteins involved in response to oxidative stress are particularly abundant in the strains carrying the empty plasmid.
               </P><DIV class="thumbnail col-sm-7 center-block"><DIV class="caption"><STRONG>
                     Figure 7. Representation of a subset of differentially expressed proteins among samples.
                     </STRONG>
                     Red and green colors indicate strong over-representation and under-representation, respectively. Blue arrows indicating β-lactamase and GFP; orange arrows indicating chaperone proteins; purple arrows indicating oxidative stress-related proteins.
                  </DIV></DIV><P>
                  But this apparent variation in the proteome changes when applying some statistics. A relative comparison of the average expression level of each one of the quantified proteins in the three groups of samples (WT strain, Bb1-expressing strain, and empty plasmid-carrying strain) is shown in Figure 8. Those proteins displaying the highest alterations in their expression levels are marked with numbers and identified in the table on the right.
               </P><DIV class="thumbnail col-sm-7 right"><DIV class="caption"><STRONG>                   Figure 8. Differential expression level of the proteins quantified by iTRAQ mass spectrometry.
                     </STRONG>
                     Each of the 472 quantified proteins has been represented for each group of samples: WT (blue line), Bb1-expressing strain (green line), and e.a.p. (empty Amp plasmid)-carrying strain (red line). Proteins on the X axis ordered in decreasing confidence level.
                  </DIV></DIV><P>
                  As deduced in Figure 7, <B>a very low number of proteins strongly changed their expression</B> in the strains carrying Bb1 or the empty plasmid. Several statistical tests (FDR-corrected ANOVA, T-test, p-value-weighted analysis) were performed with these proteins, and only three of them proved to be significant in all the tests. These proteins were the <B>GFP</B> (only present in Bb1 strain), the <B>β-lactamase</B> (present in both Bb1 and ᴓ strains), and the <B>chaperone protein DnaJ</B>, which was particularly abundant in the strain expressing Bb1. The high expression of chaperone proteins was expected, since many molecules of GFP need to be properly folded in order to avoid the formation of inclusion bodies and cellular toxicity.
                  From this experiment, <B>we have to conclude that the expression of a simple Biobrick part such as a fluorescent protein in an <I>E. coli</I> cell does not alter the proteomic architecture of the chassis in a significant way. Therefore, Biobrick part 1 could be considered orthogonal with respect to chassis' parts</B>.
               </P><H2>4. Conclusions</H2><UL><LI>Surprisingly, standard fluorimetry assays show an <B>assymetric output</B> of red and green fluorescent proteins <B>in co-transformed cells</B>.</LI><LI><B>Flow cytometry</B> confirms this asymmetric activity and strongly suggests the co-existence of <B>sub-populations of cells with different outputs</B>. This asymmetry proved <B>very reproducible</B> among cultures, and matched the predictions of our <A href="https://2014.igem.org/Team:Valencia_Biocampus/Modeling#OrthogonalityTab" target="_blank"><STRONG>theoretical model.</STRONG></A></LI><LI>The activity of fluorescent proteins <B>dramatically changes over time</B>, reaching a peak when cells are in the stationary phase. Despite being under the control of the same promoter, GFP is expressed earlier than RFP. <B>To the question “are two Biobricks orthogonal?” we thus suggest to add this other question: “when are they orthogonal?”</B></LI><LI><B>Proteomic analysis</B> reveals the <B>robustness of <I>E. coli</I></B> confronted to heterologous gene expression: only minor proteomic changes were found compared to the controls. An <B>orthogonal relation</B> between Biobrick part 1 and the proteomic architecture (chassis) is thus confirmed.</LI></UL></DIV><DIV class="tab-pane fade" id="OpenLicenseSection"><H2> Open License <SPAN class="small">(and Responsible Research and Innovation)</SPAN></H2><P>
      Not unlike scientific research, RRI is about common sense, about combining self-profit with others’ profit, about taking care of our planet. We believe that being responsible is not the opposite of being selfish; responsibility is just a broad-minded way of selfishness that points at the long-term benefit.
      And there is no long-term benefit without a sustainable society in a sustainable environment. What is good for our neighbors and for our home must be good for us as well.
   </P><H3> Our Proposal: Responsible Research and Innovation as a tool to choose within the IP ecosystem.</H3><P>
      Since we, our Human Practices sub-team, are a biotechnology and a law student, we are also glad to see that another key concept in IP is a beautiful science-law metaphor: Jane Calvert &amp; Drew Endy's <STRONG>“diverse ecology” scenario</STRONG>. This image fits well with our interdisciplinary spirit and we found it very useful because it evokes very elegantly the range of different solutions out there to choose from in order to use a scientific invention in a given way.
   </P></DIV><DIV class="tab-pane fade" id="TheSeatSection"><H2> The seat of the ST<SUP>2</SUP>OOL</H2><P>
      One of the goals of our project was to select standard, stable, and orthogonal parts naturally occurring in nature. To do that, we carried out a <B>functional
      metagenomics strategy</B> aiming at selecting promoters from a library of metagenomic DNA. As a first step, we wanted to isolate metagenomic DNA from
      environmental samples, but instead of using traditional manual kits, <B>we decided to build a robot able to automatically extract metagenomic DNA for us</B>… It
      is a pleasure for our team to introduce you our <B>TOOL</B> robot:
   </P></DIV><DIV class="tab-pane fade" id="ModelingSection"><H2>Modeling</H2><P>
      We summarize here our main contributions modeling The ST$^2$OOL. Our hypothesis, formulae and analysis can be found in the <A href="Modeling#">modeling section</A>. 
      </P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Valencia_Biocampus/ResultsDemoTestATOPE">http://2014.igem.org/Team:Valencia_Biocampus/ResultsDemoTestATOPE</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Valencia_Biocampus/ResultsDemoTestATOPE" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Valencia_Biocampus/ResultsDemoTestATOPE" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Valencia_Biocampus/ResultsDemoTestATOPE&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Valencia_Biocampus/ResultsDemoTestATOPE&amp;oldid=370173" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>