<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Valencia_Biocampus_ProjectTestingNewVersion"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Valencia_Biocampus/ProjectTestingNewVersion">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Valencia_Biocampus/ProjectTestingNewVersion&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Valencia_Biocampus/ProjectTestingNewVersion&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Valencia_Biocampus/ProjectTestingNewVersion&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Valencia_Biocampus/ProjectTestingNewVersion" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Valencia Biocampus/ProjectTestingNewVersion</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="container" id="headerWrapper"><UL id="header-top-menu" class="header-top-menu pull-right"><LI id="menu_a_project"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Project">Project</A></LI><LI id="menu_a_team"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Team">The Team</A></LI><LI id="menu_a_results"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Results">Results</A></LI><LI id="menu_a_modeling"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Modeling">Modeling</A></LI><LI id="menu_a_biobricks"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Biobricks">Biobricks</A></LI></UL><UL id="header-top-menu2" class="header-top-menu pull-right"><LI id="menu_a_human-practices"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Results#OpenLicense">Human Practices</A></LI><LI id="menu_a_interlab-study"><A href="https://2014.igem.org/Team:Valencia_Biocampus/InterlabStudy">Interlab Study</A></LI><LI id="menu_a_achievements"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Achievements">Achievements</A></LI><LI id="menu_a_attributions"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Attributions">Attributions</A></LI><LI id="menu_a_art"><A href="https://2014.igem.org/Team:Valencia_Biocampus/ArtAndMore">Art&amp;More</A></LI></UL></DIV></DIV><DIV class="container withBackground"><H1><SPAN class="mw-headline" id="The_ST2OOL_project"> The ST<SUP>2</SUP>OOL project</SPAN></H1><P><UL id="myTab" class="nav nav-tabs" data-tabs="tabs"><LI class="active"><A href="#Overview" data-toggle="tab">Overview</A></LI><LI><A href="#Stability" data-toggle="tab"><SPAN class="corporative bold">ST</SPAN>ability</A></LI><LI><A href="#Standardization" data-toggle="tab"><SPAN class="corporative bold">ST</SPAN>andardization</A></LI><LI><A href="#Orthogonality" data-toggle="tab"><SPAN class="corporative bold">O</SPAN>rthogonality</A></LI><LI><A href="#OpenLicense" data-toggle="tab"><SPAN class="corporative bold">O</SPAN>pen <SPAN class="corporative bold">L</SPAN>icense</A></LI><LI><A href="#TheSeat" data-toggle="tab">The Seat of the <SPAN class="corporative bold">ST<SUP>2</SUP>OOL</SPAN></A></LI></UL></P><DIV id="my-tab-content" class="tab-content"><DIV class="tab-pane fade in active" id="Overview"><H2 style="text-align:center;padding:0;margin:0;">Overview</H2><DIV class="watercolored" style="margin-top:-50px;"><P>
      Synthetic Biology implies an engineering perspective on biotechnology. Similarly to man-made objects, cells are expected to be decoupled, modified and even built from scratch. However, there is a general assent on the current difficulties for fully -and predictably- engineering living organisms, which are always subjected to strong evolutionary constraints. The ST<SUP>2</SUP>OOL project aims at deeply studying four of the key engineering pillars of Synthetic Biology. ST<SUP>2</SUP>OOL stands for STandardization, STability, Orthogonality and Open Licence.
      </P><P>
      The first approach, an analytical one, will consist of a vast range of experimental studies to find out how standard, stable, orthogonal and patentable several selected Biobrick are. The second approach, a synthetic one, will include functional metagenomics: several environmental libraries will be set and screened in E. coli in order to select new biological parts -promoters-, not because of their strength but because of their particularly standard, stable or orthogonal behavior. Taken together, the results of our project are expected to contribute in answering this key question: Is life fully engineerable?


    </P></DIV></DIV><DIV class="tab-pane fade" id="Stability"><DIV class="row" style="margin-bottom:-15px"><H2><SPAN class="mw-headline" id="Stability"> Stability </SPAN></H2><DIV class="col-sm-7"><P>
Our subteam aims to test the stability of ten different Biobricks transformed in two different strains of<EM> Escherichia coli, </EM>XL1 Blue and DH5α    <EM>. </EM>Stability is determined by subjecting the transformed cells to different types of stress and comparing the output with non-transformed ones
    grown in the same conditions, as well as with cells transformed with an empty plasmid (without Biobrick).
</P></DIV></DIV><H3><SPAN class="mw-headline" id="Temperature_Stress">Temperature Stress</SPAN></H3><P>
    We start out by establishing an overnight culture of each strain, every one of which will be subjected to the different temperatures. The experiment will
    be carried out in an improvised miniature system we just developed; PCR tubes filled with 75 µL of culture are placed in a thermo cycler, in which a
    gradient of temperatures from 30-50ºC will be set. In each round, four replicas of two strains are to be tested simultaneously at 12 different
    temperatures:
</P><DIV class="thumbnail col-sm-5 center-block"><DIV class="caption"><P>four replicas of two strains tested simultaneously at 12 different temperatures</P></DIV></DIV><P>
    After an incubation period at the given temperatures the tubes containing the culture will be collected to measure the O.D. Fluorescence will also be
    measured and later normalized to O.D.<SUB>600</SUB>, to compare the expression of the different Biobricks.
</P><H3><SPAN class="mw-headline" id="Material_Fatigue"> Material Fatigue </SPAN></H3><P>
The idea is analogous to material fatigue in material sciences; in <A href="http://en.wikipedia.org/wiki/Materials_science" title="Materials science">materials science</A>, fatigue is the weakening of a material caused by
    repeatedly applied loads. If the loads are above a certain threshold, microscopic cracks will begin to form at the stress concentrators such as the
    surface, persistent slip bands and grain interfaces. In our case, we chose to induce the fatigue by subjecting the cells to temperature fluctuation, every
    minute the temperature will be changed from 37ºC (<EM>E.coli</EM> optimum growth temperature) to 41ºC and back to 37ºC again, this way, the temperature
    oscillates every minute from 37 to 41ºC for 4 hours. What will be the effect of such thermal fluctuations?
</P><H3><SPAN class="mw-headline" id="pH_.26_Salinity"> pH &amp; Salinity </SPAN></H3><P>
    One of the best known factors that conditions bacterial growth is proton and salt concentration in the media. This experiment, as simple as may seem,
    offers the opportunity to test cell viability as well as the Biobrick behaviour at different pH and salt concentrations. Overnight cultures of both strains
    will be resuspended in different LB media with pH ranging from 5 to 9 and incubated for 90 minutes, after which O.D.<SUB>600 </SUB>and fluorescence of each
    tube is to be measured. The design of the experiment is the same for salts, ranging from LB medium without salt to an extra 4% of salt, maintaining the
    normal pH of the LB medium for all the tubes.
</P><H3><SPAN class="mw-headline" id="UV_Radiation"> UV Radiation </SPAN></H3><P>
    As per today, high doses of UV light are proven detrimental to life, but, to what extent is it injurious to an <EM>E.coli</EM> cell? Is transformation a
    factor that enhances the lethality of UV radiation or, contrary to what logic dictates, helps resist the UV radiation? Curiosity led us to design an
    experiment which would answer these questions: overnight cultures of the strains will be diluted and spot-plated on LB agar. Subsequently, the spots will
    be subjected to timed pulses of UV radiation (intensity of 340 µW/cm<SUP>2) </SUP>with a control that receives no UV irradiation at all. Thereafter the
    plates will be incubated and fluorescence will be measured and normalized to O.D.<SUB>600.</SUB></P><H3><SPAN class="mw-headline" id="Vacuum"> Vacuum </SPAN></H3><P>
    A simple experiment which mainly consists of exposing the cells, spot-plated on Petri dishes, to vacuum during 48 hours will be carried out; the output
    from these, as always, compared to cells spot-plated and grown in optimal conditions, through OD<SUB>600</SUB> and fluorescence assays.
</P></DIV><DIV class="tab-pane fade" id="Standardization"><H2><SPAN class="mw-headline" id="Standardization"> Standardization </SPAN></H2><P>
  One should not assume that a functional module working fine in one cell type will work the same way in even a closely related cell type (Renshaw, 1993). Whereas traditional engineering practices typically rely on the standardization of parts, the uncertain and intricate nature of biology makes standardization in the synthetic biology field difficult. Beyond typical circuit design issues, synthetic biologists must also account for cell death, crosstalk, mutations, intracellular and extracellular conditions, noise and other biological phenomena. As the number of system components grows, it becomes increasingly difficult to coordinate component inputs and outputs to produce the overall desired behaviour (Purnick &amp; Weiss, 2009). In the ST²OOL Project, we wanted to check how standard biobricks are. For this goal, we are using a set of strains of <I>Escherichia coli</I>  transformed, in single transformation, with <A href="https://2014.igem.org/Team:Valencia_Biocampus/Biobricks">ten different Biobricks</A> whose outputs will be measured by fluorometry, colorimetry or luminometry. The aim of this approach is simple and fits with the answer to the following question:
</P><P>
  Will the Biobricks behave the same way independently of the host strain they have been transformed into? Will we obtain the same output from all of them? The answer to these questions… in two months! 
</P><H3><SPAN class="mw-headline" id="References"> References </SPAN></H3><P><A href="http://www.nature.com/nrm/journal/v10/n6/pdf/nrm2698.pdf">Purnick, P. E. M. &amp; Weiss, R. <I>The second wave of synthetic biology: from modules to systems</I>. Nature Reviews <B>10</B>, 410-422 (2009)</A>.
</P><P><A href="http://books.google.es/books?hl=es&amp;lr=&amp;id=9iKDQEFM0ScC&amp;oi=fnd&amp;pg=PR13&amp;dq=Modelling+Biological+Populations+in+Space+and+Time&amp;ots=eVAhPiiIT9&amp;sig=fGjzEYakHztERdAs9gw5eKfIfbI#v=onepage&amp;q=Modelling%20Biological%20Populations%20in%20Space%20and%20Time&amp;f=false">Purnick, P. E. M. &amp; Weiss, R. Renshaw, E. <I>Modelling Biological Populations in Space and Time</I>. Cambridge University Press (1993)</A>.
</P></DIV><DIV class="tab-pane fade" id="Orthogonality"><H2><SPAN class="mw-headline" id="Orthogonality"> Orthogonality </SPAN></H2><DIV class="row"><P>
    Do biobricks behave independently when cotransformed in a cell? In other words, can we consider them to be orthogonal? Orthogonality is a mathematical
    concept that refers to the independence of behavior. Translating this concept into the biological realm, two constructions can be considered orthogonal
    when they only interact at specific and predictable interphases and do not disturb each other. This is a desirable feature of Biobricks if one wants to
    build synthetic circuits with predictable and robust behaviors.
</P><P>
    To study this pillar of synthetic biology we are analyzing different combinations of Biobricks in cotransformed cells. That way, we can study the
    expression of pairs of Biobricks and compare them with simple transformations, and therefore determine whether the presence of two constructions in the
    same cell affects their behavior. In order to do so, we will measure the output of each Biobrick (i.e. fluorescense) in both the cotransformants and the
    controls.
</P><P>
    We are going to study three combinations, taking in to account that each pair of biobricks is controlled by the same promoter and only differs in the
    protein expressed and the antibiotic resistance:
</P><UL><LI>
        Constitutive promoter I (strong) – GFP &amp; Constitutive promoter I (strong) – RFP
    </LI><LI>
        Constitutive promoter II (less strong) – GFP &amp; Constitutive promoter I (strong) – RFP
    </LI><LI>
        Ptrc – lacZ &amp; Ptrc – luciferase
    </LI></UL><P>
    Besides individually transformed cells, we will use a control consisting of cotransformed cells with a Biobrick-bearing plasmid and an empty one.
</P><P>
    We are currently working on all this. But we have another, very ambitious experiment. We plan to determine the effect of the production of a single
    heterologous protein on the proteomic architecture of the host cell. For this endeavor, we just submitted our samples for a proteomic quantification.
</P></DIV></DIV><DIV class="tab-pane fade" id="OpenLicense"><H2><SPAN class="mw-headline" id="Open_License"> Open License </SPAN></H2><P>
    We did not want the Human Practices section to be a cosmetic addendum to our iGEM project. In The St<SUP>2</SUP>OOL, we have performed a critical but
    constructive analysis of four of the pillars of the competition. Three of them are technical, engineering assumptions (stability, standardization,
    orthogonality). The fourth leg of our St<SUP>2</SUP>OOL is for us at least as important as the other three, and deals with the answer to this difficult
    question: who owns synthetic bio-constructions?</P><P>
    iGEM is linked to the Registry of Standard Biological Parts, a non-profit organization (as iGEM Foundation itself) archiving and distributing thousands of
    BioBrick parts for free (<A href="http://parts.igem.org/Main_Page">http://parts.igem.org/Main_Page</A>). No patents, no licenses, no royalties. Just Open Access (either totally free access or under
    certain conditions such as a creative commons subscription). As stated in their Website, it’s “an open community that runs and grows on the Get &amp; Give
    (&amp; Share) philosophy. Users get parts, samples, data, and tools from the Registry to work on their synthetic biology projects. They will give back to
    the Registry the new parts they have made, as well as data and experience on new and existing parts”. But of course, the world of Biotechnology/SynBio is
    far more than iGEM and a range of Intellectual Property protection formulas do exist. Is iGEM an island of Free Access in an ocean of patentable SynBio
    achievements? How can one deal with the variety of IP forms?
</P><P style="text-align:center"><A href="https://2014.igem.org/Team:Valencia_Biocampus/HumanPractices" class="btn btn-lg btn-default">Human Practices Section</A></P></DIV><DIV class="tab-pane fade" id="TheSeat"><H2><SPAN class="mw-headline" id="The_Seat_of_the_ST.C2.B2OOL"> The Seat of the ST²OOL </SPAN></H2><P>
    By checking how standard, stable, orthogonal or patentable Biobricks are we have performed an analysis of four engineering pillars. But we wanted to
    combine these pillars, these four legs of our ST<SUP>2</SUP>OOL, with a truly synthetic and constructive approach.
</P><P>
    In The Seat of the ST<SUP>2</SUP>OOL we are developing a functional metagenomic strategy in which we aim at isolating promoter sequences from natural
    environments. We will then characterize these sequences in order to select those with the highest stability, standard and orthogonal behaviors. We are
    working with soil and compost samples, from which we will isolate total metagenomic DNA, which will then be digested, ligated into an expression vector and
    used to built a library in <EM>E. coli</EM>. Transformants –clones with a promoter-like natural sequence- will be subjected to a range of stresses and
    subtransformation steps to find out their stability and standard/orthogonal behavior. To the best of our knowledge, this is the first Synthetic Biology
    functional metagenomic approach aiming at selecting biological parts not because of their “strength” but because of their compatibility with other parts
    and suitability for engineering purposes.</P><P>
    Finally, one technical –yet important- detail: we are not using neither commercial kits nor manual protocols for isolating metagenomic DNA. We are building
    our own robot…
</P><DIV class="row"><DIV class="thumbnail col-sm-5 center-block"><DIV class="caption"><P>Our robot design</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Valencia_Biocampus/ProjectTestingNewVersion">http://2014.igem.org/Team:Valencia_Biocampus/ProjectTestingNewVersion</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Valencia_Biocampus/ProjectTestingNewVersion" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Valencia_Biocampus/ProjectTestingNewVersion" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Valencia_Biocampus/ProjectTestingNewVersion&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Valencia_Biocampus/ProjectTestingNewVersion&amp;oldid=227631" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>