<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Valencia_Biocampus_Results"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Valencia_Biocampus/Results">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Valencia_Biocampus/Results&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Valencia_Biocampus/Results&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Valencia_Biocampus/Results&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Valencia_Biocampus/Results" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Valencia Biocampus/Results</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="container" id="headerWrapper"><UL id="header-top-menu" class="header-top-menu pull-right"><LI id="menu_a_project"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Project">Project</A></LI><LI id="menu_a_team"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Team">The Team</A></LI><LI id="menu_a_results"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Results">Results</A></LI><LI id="menu_a_modeling"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Modeling">Modeling</A></LI><LI id="menu_a_biobricks"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Biobricks">Biobricks</A></LI></UL><UL id="header-top-menu2" class="header-top-menu pull-right"><LI id="menu_a_human-practices"><A href="https://2014.igem.org/Team:Valencia_Biocampus/HumanPractices">Human Practices</A></LI><LI id="menu_a_interlab-study"><A href="https://2014.igem.org/Team:Valencia_Biocampus/InterlabStudy">Interlab Study</A></LI><LI id="menu_a_achievements"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Achievements">Achievements</A></LI><LI id="menu_a_attributions"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Attributions">Attributions</A></LI><LI id="menu_a_art"><A href="https://2014.igem.org/Team:Valencia_Biocampus/ArtAndMore">Art&amp;More</A></LI></UL></DIV></DIV><DIV class="container withBackground"><H1>
   Our Results<UL id="myTab" class="nav nav-tabs" data-tabs="tabs"><LI class="active"><A href="#StabilitySection" data-toggle="tab"><SPAN class="corporative bold">St</SPAN>ability</A></LI><LI><A href="#StandardizationSection" data-toggle="tab"><SPAN class="corporative bold">St</SPAN>andardization</A></LI><LI><A href="#OrthogonalitySection" data-toggle="tab"><SPAN class="corporative bold">O</SPAN>rthogonality</A></LI><LI><A href="#OpenLicenseSection" data-toggle="tab"><SPAN class="corporative bold">O</SPAN>pen <SPAN class="corporative bold">L</SPAN>icense</A></LI><LI><A href="#TheSeatSection" data-toggle="tab">The Seat of the <SPAN class="corporative bold">St<SUP>2</SUP>OOL</SPAN></A></LI><LI><A href="#ModelingSection" data-toggle="tab">Modeling</A></LI></UL></H1><H2>Stability</H2><P><STRONG><EM>Escherichia coli</EM> is the lab rat of the bacterial world.</STRONG>
               We have performed the most complete characterization of the stability of <EM>E. coli</EM> in order to both confirm its optimality as a chassis for Synthetic Biology, and to determine the behavior of Biobricks under sub-optimal environmental conditions. We have subjected wild type strains to a range of stressful conditions (see just below &quot;the limits of <EM>E. coli</EM>&quot;) and measured how our Biobricks behaved.
            </P><P><A href="https://static.igem.org/mediawiki/2014/0/07/List_of_Protocols_Stability.pdf" target="_blank"><STRONG> List of protocols here</STRONG></A></P><H2>1 . The Limits of <EM>E.coli</EM></H2><H3>Temperature</H3><P><EM>E.coli</EM>
               is a fully studied and characterized bacterium. The optimal growth of this species is 37-38ºC with continuous agitation (200 r.p.m.) but out of this range
               the growth slows down. First of all, we measured the growth of <EM>E.coli</EM> at temperatures ranging from 30 to 50ºC. To achieve our goal, we used a
               fully original technique: instead of an incubator, we used a thermal cycler programmed to display a gradient of temperatures.
            </P><P>This proved a very easy and reproductible way of very precisely incubating at increasing temperatures small (PCR tubes) bacterial cultures.</P><P>
               The results obtained are shown below:
            </P><DIV class="thumbnail col-sm-11 center-block"><DIV class="caption"><STRONG>Figure 1:</STRONG>
                  Growth of <EM>E. coli</EM> strains <STRONG>DH5α </STRONG>(left) and <STRONG>XL1-Blue</STRONG> (right) under different temperatures (30ºC-50ºC) set in a thermal cycler. The two strains display clearly different temperature profiles.
               </DIV></DIV><H4>Conclusion</H4><EM>Is</EM> E. coli stable<EM> at a wide range of temperatures?</EM><STRONG>Yes it is!</STRONG> We have detected some variations between the two strains. For example, DH5α resists low temperatures better than XL1-Blue, but <STRONG>both grow at a very large range of non-optimal temperatures.</STRONG><H3> Material fatigue (1)</H3><P>
               Our modeling sub-team suggested an original idea. Is there something similar to material fatigue in biology? For example, how would an <EM>E.coli</EM> culture behave if subjected to a temperature fluctuation? We carried out some experiments with our thermal cycler by cycling the temperature of the medium every minute (37-41ºC) for 3h and maintaining controls at fixed temperatures (typically 37 and 41ºC). The results of these experiments are shown in Figure 2. 
            </P><DIV class="thumbnail col-sm-9 center-block"><DIV class="caption"><STRONG>Figure 2:</STRONG><STRONG>XL1-Blue</STRONG> (left) and <STRONG>DH5α</STRONG> subjected to cycling (37-41ºC) temperature conditions. Compared to the controls, the former strain did not exhibit any significant change in the growth rate, as deduced by the OD<SUB>600</SUB> after 3 hours (DH5α) and 15 hours (XL1-Blue). By contrast, DH5α significantly increased its growth compared to both controls. The statistical parameters are cited in the text. 
                  <P>
                     The ANOVA analysis performed with the 16 replica of each strain revealed no significant differences in the growth of Xl1-Blue (F=0.870, p=0.433). However, the differences in the growth of DH5α did have statistical significance (F=63.95, p&lt;0.001).
               </P></DIV></DIV><H4>Conclusion</H4><EM>Is </EM>E. coli<EM> sensitive to fatigue of materials in the form of cycling conditions?</EM><STRONG>Yes! at least DH5α, but in a positive way. Cycling temperatures resulted in an emergent property: increased growth</STRONG>. In order to check whether this effect was due to a mechanical resuspension (convection) linked to the cycling conditions, we performed a specific experiment with dead, antibiotic-treated <EM>E. coli </EM>cells (Kanamicine and Cloranfenicol) subjected to extreme (30 to 50ºC) thermal fluctuations. The following figure shows that <STRONG>convection forces might not be the reason of such increased growth</STRONG> (no significant differences found in a t-Student test for 2 groups and 16 replicas; t=1.371, p=0.180). <STRONG>A biological explanation may be hypothesized: cycling conditions allow a range of optimal enzymatic activities to be met.</STRONG><DIV class="thumbnail col-sm-5 center-block"><DIV class="caption"><STRONG>Figure 3:</STRONG>
                  O.D.600 of antibiotic-killed <EM>E. coli</EM> suspensions of strain XL1-Blue incubated under cycling (30ºC-50ºC) and average (40ºC) conditions. No significant differences suggest that convection forces are not the main key players in the increased growth of <I>E. coli</I> incubated under thermally cycling condition. t-Student test 95% showed no significant differences, as explained in the text.
               </DIV></DIV><H3> UV Radiation (1) </H3><P>
               UV radiation generally carries out its biological effect by damaging nucleic acids. Depending on the radiation intensity and exposure time the damage caused to the genome is reversible; cells can repair their DNA and continue to grow, although they might carry point mutations. Since UV studies are scarce, our team decided to assay the maximum resistance of <EM>E. coli</EM> strains to UV radiation of a fixed intensity (340 µW/cm<SUP>2</SUP>). The experiment consisted of subjecting spot plated <EM>E.coli</EM> strains to timed pulses of UV light and comparing the output to a control set of bacteria which received no radiation at all.
            </P><DIV class="thumbnail col-sm-11 center-block"><DIV class="caption"><STRONG>Figure 4:</STRONG>
                  Characterization of <STRONG>XL1-Blue </STRONG>(left) and <STRONG>DH5α</STRONG> (right) subjected to UV light (340 µW/cm<SUP>2</SUP>). Both strains are rather sensitive, but DH5α resists UV light better than XL1-Blue.
               </DIV></DIV><P><STRONG>Therefore, <EM>E. coli</EM> proved very sensitive to UV light </STRONG> We also studied how sensitive these strains are when transformed with our Biobricks. As shown in Fig 5, Biobrick-expressing strains were more sensitive to
               UV light compared to plasmid-only strains. However,<STRONG> Biobrick7 provided some resistance to radiation in DH5α compared with the controls.</STRONG></P><DIV class="thumbnail col-sm-11 center-block"><DIV class="caption"><STRONG>Figure 5:</STRONG>
                  Resistance of<STRONG> DH5α </STRONG>(left) and <STRONG>XL1-Blue</STRONG> (right) to UV pulses (340 µW/cm<SUP>2</SUP>). X axis values are given in seconds. Both strains are very sensitive to UV light with a maximum resistance of around 6 seconds.
               </DIV></DIV><H3>pH &amp; Salinity (1)</H3><P>
               We also characterized <EM>E. coli</EM> resistance to a range of extreme pH values and salt concentrations. As shown in Table 1, <STRONG>none of the extreme conditions tested resulted in a total inhibition of <EM>E. coli’</EM>s growth.</STRONG></P><DIV class="col-sm-7 center-block small"><P class="text-centered"><STRONG>Table 1. Growth of XL1-Blue and DH5α under different pH and salt concentrations</STRONG></P><TABLE class="table table-bordered table-condensed" style="background:#f0f0f0"><TBODY><TR><TD><P><STRONG>pH</STRONG></P></TD><TD><P><STRONG>5</STRONG></P></TD><TD><P><STRONG>6</STRONG></P></TD><TD><P><STRONG>7</STRONG></P></TD><TD><P><STRONG>8</STRONG></P></TD><TD><P><STRONG>9</STRONG></P></TD></TR><TR><TD><P>
                              XL1-Blue
                           </P></TD><TD><P>
                              +
                           </P></TD><TD><P>
                              ++
                           </P></TD><TD><P>
                              +++
                           </P></TD><TD><P>
                              ++
                           </P></TD><TD><P>
                              +
                           </P></TD></TR><TR><TD><P>
                              DH5α
                           </P></TD><TD><P>
                              ++
                           </P></TD><TD><P>
                              +++
                           </P></TD><TD><P>
                              +++
                           </P></TD><TD><P>
                              ++
                           </P></TD><TD><P>
                              +
                           </P></TD></TR><TR class="background-burdeos"><TD><P><STRONG>Salt</STRONG></P></TD><TD><P><STRONG>Without salt</STRONG></P></TD><TD><P><STRONG>LB </STRONG></P></TD><TD><P><STRONG>LB+1% salt</STRONG></P></TD><TD><P><STRONG>LB+2% salt</STRONG></P></TD><TD><P><STRONG>LB+4%salt</STRONG></P></TD></TR><TR><TD><P>
                              XL1-Blue
                           </P></TD><TD><P>  +++
                           </P></TD><TD><P>
                              +++
                           </P></TD><TD><P>
                              ++
                           </P></TD><TD><P>
                              +
                           </P></TD><TD><P>
                              +
                           </P></TD></TR><TR><TD><P>
                              DH5α
                           </P></TD><TD><P>
                              +++
                           </P></TD><TD><P>
                              ++
                           </P></TD><TD><P>
                              +
                           </P></TD><TD><P>
                              +
                           </P></TD><TD><P>
                              +
                           </P></TD></TR></TBODY></TABLE></DIV><H3> Vacuum (1) </H3><P>
               We designed a simple experiment to subject <I>E.coli</I> to Mars-like vacuum. It consisted of streaking Petri dishes and incubating them in deep vacuum 
               during 48 hours. We also maintained control plates, grown at atmospheric pressure. After this incubation period, we measured the diameter of three colonies of each Petri dish. The results, indicating <STRONG> only a moderate decrease in colony size after two days of vacuum incubation </STRONG> are shown below: 
            </P><DIV class="thumbnail col-sm-11 center-block"><DIV class="caption"><STRONG>Figure 6:</STRONG>
                  Mean Colony Diameter of <STRONG>XL1</STRONG> (left) and <STRONG> DH5α</STRONG> (right) grown under vacuum and atmospheric pressure. Growth differences were statistically significant according to a t-Student test for two groups and three replicas (XL1: t=4.24, p=0.013&lt;0.05 ; DH5α: t=9.98, p&lt;0.001).
               </DIV></DIV><H3> BL21.  Chromosomal T7 polymerase gene is unstable! </H3><P>
               BL21 is an <EM>Escherichia coli </EM>strain that bears the T7 phage polymerase gene into its genome. Therefore, it is used to express proteins that are
               controlled by T7 phage promoters, as is the case of three of our Biobricks (Biobricks 8, 9 &amp; 10). However, when we transformed the strain with the
               Biobricks (blue and yellow chromoproteins), we could not measure any signal. After many tests, we hypothesized that the T7 phage polymerase gene was
               unstable and did not resist stressful situations such as competence or transformation protocols.
            </P><P>
               We performed a PCR screening to find out whether the T7 gene was present. We used colonies of BL21 transformed with Biobrick 8 (blue chromoprotein) and
               grew them on LBA medium. We performed a colony-PCR with a couple of T7 specific oligos designed by ourselves targeting a 1.5 kb region of the T7 polymerase
               coding sequence. 
            </P><DIV class="thumbnail col-sm-7 center-block"><DIV class="caption"><STRONG> Figure 7: </STRONG> 0.8% agarose gel electrophoresis. We saw that the bacteria forming the colonies did not have the gene that codes for the polymerase. </DIV></DIV><P>
            We then decided to check whether our glycerol stock of bacteria contained the gene. Maybe during the process of competence the culture was contaminated. To our surprise, the gene was present in every single colony we analyzed. Therefore, we decided to do a final colony PCR using the bacteria from the glycerol stock as positive controls, and using colonies from the non-transformed competent stock.
            </P><DIV class="thumbnail col-sm-7 center-block"><DIV class="caption"><STRONG> Figure 8:</STRONG> 0,8% agarose gel electrophoresis. In the BL21-glicerol stock the gene was present. Nonetheless, almost all BL21-competent cells lack T7 phage polymerase gene. 
               </DIV></DIV><P><STRONG>Our results reveal that BL21 loses the T7 polymerase gene at some point during the process of making the cells competent</STRONG>, as only three out of the ten colonies analyzed did result in (a rather faint) amplification of the gene.
            </P><H2>2 . Behavior of different Biobricks in sub-optimal conditions</H2><P>
               We show below a summary of our Biobrick characterization program, in which we have subjected selected Biobrick-expressing strains to a range of environmental stresses. <STRONG>Our results clearly show that whereas UV light, pH variation and salt content do not significantly affect Biobrick expression, sub-optimal temperatures can in some cases decrease the Biobrick output. Anaerobic, vacuum culture resulted in a dramatic decrease of Biobrick 2 (RFP) activity.</STRONG></P><H3> Temperature Stress </H3><P>
               37ºC and 200 rpm are the standard growth conditions of <EM>E.coli</EM>. Varying these conditions and measuring the growth after three hours, (O.D.<SUB>600</SUB>) as well as fluorescence (Biobrick output), was used to test the effect on Biobrick expression (all data are the average of at least three measurements; fluorescence values are normalized to OD<SUB>600</SUB>). We also built a <B>model of the temperature stability</B> of the constructions used in these experiments, and compared to the data. Interestingly, our model captures the shape of the temperature profile and predicts that additional variations at lower temperatures are independent of the chosen Biobrick.  
            </P><DIV class="thumbnail col-sm-7 center-block"><DIV class="caption"><STRONG>Figure 9A:</STRONG> The strain XL1-Blue transformed with a Biobrick which encodes GFP (Biobrick 1, Bb1) was grown at different temperatures in a thermocycler for three hours, the cells were collected and O.D<SUB>600</SUB> was measured. Fluorescence was also determined and compared to the non-transformed/wildtype or control strain. As the plot shows, Biobrick expression is affected by growth at different temperatures, fluorescence (normalized to O.D<SUB>600</SUB>) is higher around <I>E.coli</I> optimal growth temperature (37-41 ºC); over or below this rather short temperature range fluorescence decreases. The green line represents the theoretical prediction of our model, which 
                  captures the shape of the temperature profile. 
                  <A href="https://2014.igem.org/Team:Valencia_Biocampus/Modeling#StabilityTab " target="_blank"><STRONG> Click here to see our model. </STRONG></A></DIV></DIV><DIV class="thumbnail col-sm-7 center-block"><DIV class="caption"><STRONG>Figure 9B:</STRONG> The strain XL1-Blue transformed with a Biobrick which encodes GFP (Biobrick 3, Bb3) placed under the control of a weak promoter. The micro-cultures were grown at different temperatures in a thermo cycler for three hours, the cells were collected and O.D<SUB>600</SUB> was measured. Fluorescence was also determined and compared to the non-transformed/wildtype or control strain. Fluorescence values in the different temperatures tested are very similar to the control and only slightly higher around optimal temperatures. This indicates a low expression of Bb3 in all cases. The green line represents the theoretical prediction of our model, which 
                  captures the shape of the temperature profile.
                  <A href="https://2014.igem.org/Team:Valencia_Biocampus/Modeling#StabilityTab " target="_blank"><STRONG>Click here to see our model.</STRONG></A></DIV></DIV><H3> Material fatigue (2) </H3><P>
               As described in the first parts of this section “The limits of <EM>E. coli</EM>”, the strain DH5α grows better under temperature cycling conditions &quot;thermal fatigue&quot; compared to the optimal growth temperature (37ºC), but the same effect was not observed for the strain XL1-Blue. Hence, we decided to check what was the effect of Biobrick expression in XL1-Blue (Fig 10). 
            </P><DIV class="thumbnail col-sm-5 center-block"><DIV class="caption"><STRONG>Figure 10:</STRONG> Biobrick 1 expression under different temperatures, including temperature fluctuation (from 37ºC to 41ºC, every minute each) in a thermocycler. Controls were incubated at 37 and 41ºC. After an incubation period of 3 hours, O.D.<SUB>600</SUB> alongside fluorescence was measured. The results show that the expression of the Biobrick is not affected by cycling temperatures. ANOVA test 95% shows no significant
                  differences, as explained in the text.
               </DIV></DIV><P><B>No significant differences were found among treatments</B> according to an ANOVA test for three groups and 16 replicas (F=1.682, p=0.200) at a 95% level of significance.
            </P><H3>pH &amp; Salinity (2)</H3><P>
               Large proteins, including enzymes, are affected by pH. Usually, the catalytic properties of the enzymes are lost and metabolism is halted, which eventually leads to death. The pH range determined for <EM>E.coli</EM> optimal growth is 6 to 7, but does Biobrick expression affect this? The results of the experiment in which the pH of the medium was varied are displayed below. The results show that, as expected, <B>Bb1</B> displays a higher fluorescence and <B>is active regardless the pH</B> value of the medium.
            </P><H4> DH5α pH </H4>
            The figure shows our experimental results compared to the prediction of our <A href="https://2014.igem.org/Team:Valencia_Biocampus/Modeling#StabilityTab" target="_blank"> theoretical model</A>. The model predicts that fluorescence is lowest at the pH with lowest growth; while it is mildly dependent on the pH from 6 to 8.
            <DIV class="thumbnail col-sm-9 center-block"><DIV class="caption"><STRONG>Figure 11:</STRONG> The plot displayed above shows the variation in fluorescence of the transformed DH5α cells grown in LB with pH ranging from 5 to 8. Cells were transformed with GFP constructs, one of which is controlled by a strong constitutive promoter (BB1) and the other, BB3, controlled by a weak housekeeping promoter. The output from these is compared to a cell transformed with an empty plasmid (Ø Ampicillin). 
                  <P><A href="https://2014.igem.org/Team:Valencia_Biocampus/Modeling#StabilityTab" target="_blank"><STRONG>Click here to see our model.</STRONG></A></P></DIV></DIV><P><B>The differences found among the range of pH values were significant for BB1</B> (F=2.654, p=0.075) <B>but not significant for BB3</B> (F=7.6, p=0.001) at a 95% level of confidence. 
            </P><H4> DH5α salinity </H4>
            Regarding salinity, a major environmental factor, we also tested the stability of our Biobricks by measuring their output in increasingly salty media (Fig. 12). Again, there were <B>no significant differences associated with salt concentration in the case of BB1</B> (ANOVA values: F=0.832, p=0.491) but they were <B>significnat for BB3</B> (ANOVA values: F=3.149,
            p=0.045).
    <DIV class="thumbnail col-sm-9 center-block"><DIV class="caption"><STRONG>Figure 12:</STRONG> Using the same approach as the previous experiment, the same strains (DH5α) were grown in mediums with different salt concentrations (LB without salt, regular LB, and LB supplemented with 1% and 2% NaCl). Fluorescence of the GFP Biobricks was measured and results normalized to the O.D.<SUB>600</SUB>. As before, Bb1 was significantly expressed compared to the control at values relatively independent of the salt concentration. ANOVA 95% test
                  equal mean values hypothesis rejected, as explained in the text.
               </DIV></DIV><H3> UV Radiation (2) </H3><P>
               We display below the results of Biobrick behavior upon irradiation by measuring fluorescence of the surviving cells. We found out that the <B>fluorescence was not affected by UV exposure</B> (Figures 13 and 14), since the t-Student test for two groups and three replica showed no significant differences (BB1: t=1.011, p=0.418; BB3:
               t=1.240, p=0.341)
            </P><H4> GFP </H4><DIV class="thumbnail col-sm-11 center-block"><DIV class="caption"><STRONG>Figure 13:</STRONG> Biobrick 1 and 3 behaviors before and after a 2 second UV pulse. Left, DH5α; Right, XL1-Blue. The activity of both Biobricks is not affected by previous exposure to UV. In both cases, t-Student test 95% shows no significant differences, as explained in the text.
               </DIV></DIV><H4> RFP </H4>
            Similar results were obtained with BB2, finding, again, no significant differences under the different treatments (DH5α: t=-0.852, p=0.499; XL1-Blue: t=0.428, p=0.754).
            <DIV class="thumbnail col-sm-11 center-block"><DIV class="caption"><STRONG>Figure 14:</STRONG> Biobrick 2 behavior before and after a 2 second UV pulse. Left, DH5α; Right, XL1-Blue. The activity this Biobrick is not affected by previous exposure to UV. Unexpectedly Bb2 does not show any significant fluorescence even in the non-UV-treated samples. In both cases, t-Student test 95% shows no significant differences.
               </DIV></DIV><H3> Vacuum (2) </H3><P>
               Similarly to the experiment carried out in the section “Limits of <I>E.coli</I>” with vacuum, we plated the strains and incubated them in deep vacuum for 48 hours. After this incubation period, cells were recollected and O.D.600 as well as (normalized) fluorescence, determined. Surprisingly, <B>Biobrick 2 proved extremely sensitive to vacuum</B> in both strains, since normalized fluorescence dramatically decreased compared to the normal pressure-incubated cultures (Figure 15).
            </P><H4> RFP </H4><DIV class="thumbnail col-sm-11 center-block"><DIV class="caption"><STRONG>Figure 15:</STRONG> Effect of vacuum on the expression of the Biobrick 2 (RFP) in two strains of <EM>E.coli</EM>. In both cases vacuum-cultured cells display a much lesser normalized fluorescence that the normal pressure-incubated ones (all assays were conducted at 37ºC).
               </DIV></DIV></DIV></DIV><DIV class="tab-pane fade" id="StandardizationSection"><H2>Standardization</H2><P><STRONG>Biobrick parts are supposed to be standard</STRONG>, this is, they are expected to <STRONG>behave in a similar and predictable way</STRONG> in different host organisms
            or “chassis”. In this part of the project we used six different <EM>Escherichia coli</EM> strains commonly used in molecular biology labs (<STRONG>DH5α</STRONG>, <STRONG>XL1-Blue</STRONG>, <STRONG>JM109</STRONG>, <STRONG>DH10B</STRONG>, <STRONG>HB101</STRONG>, <STRONG>BL21(DE3)</STRONG>) to test whether the
            behavior of our Biobrick parts was homogeneous among them (<STRONG>Bb1</STRONG> to <STRONG>Bb6</STRONG>, <STRONG>BBa_K1468000</STRONG>-<STRONG>BBa_K1468005</STRONG> in the <A href="http://parts.igem.org/Catalog" target="_blank">Registry of Standard Biological Parts</A>). Surprisingly, <STRONG>only one of the constructions</STRONG> we tested displayed a relatively standard output among the different strains (<STRONG>BB5</STRONG>).
         </P><H3>General protocol</H3><UL><LI>Strike the desired strains on LB plates or LB plates supplemented with the appropriate antibiotic (depending on the bacteria we are working with: the wild type, sensitive to the antibiotic, or the transformed ones, resistant to the selective agent). Allow to grow by incubating at 37°C at least 24 hours.</LI><LI>Gather a large amount of bacteria with an inoculation loop and transfer to 1 mL of LB or LB supplemented with the antibiotic. Incubate during 20 minutes at 37°C and 200 rpm.</LI><LI>Inoculate 100 µL of bacterial suspension in a tube containing 3 mL of LB with antibiotic. Perform four biological replica of each culture. The
               expression of Bb4 was induced with a heat shock at 42 °C for 2 min. The expression of Bb5 and Bb6 was induced by adding doxycycline and IPTG, respectively,
               to the medium.
            </LI><LI>Incubate until OD<SUB>595</SUB> is between 0.15-0.4 (1 hour aprox.).</LI><LI>Measure OD<SUB>595</SUB>.</LI><LI>In the case of Biobrick parts 4, 5 and 6, <STRONG>sonicate an aliquot of the culture</STRONG> (it allows us working with the pretein extract, avoiding the interference of cell density in the measurement of the blue, product of the XGal reaction), eliminate cell debris by centrifugation (2 min, max. rpm), add X-Gal, incubate 6 min at RT, and stop the
               reaction by adding Na<SUB>2</SUB>CO<SUB>3</SUB>.
            </LI><LI>Measure either GFP fluorescence (<STRONG>Bb1</STRONG> and <STRONG>Bb3</STRONG>;  λex=493nm;  λem=505nm), RFP fluorescence (<STRONG>Bb2</STRONG>;  λex=576 nm;  λem=592 nm), or absorbance at 630 nm
               (<STRONG>Bb4</STRONG>, <STRONG>Bb5</STRONG>, and <STRONG>Bb6</STRONG>). Fluorescence measurements were performed in a Hitachi F7000 plate reader, whereas a spectrophotometer was used for absorbance
               measurements.
               </LI>
               With Excel, normalize fluorescence by cell density, and calculate the average and standard deviation taking into account the four biological replicas.
               Represent the data.
            </UL><H3>Results</H3>
         The behavior of all the Biobrick parts we tested is shown in <STRONG>Figure 1</STRONG>. From our results, it is evident that the expression of all but one of the
         constructions, <STRONG>Bb5</STRONG>, clearly differed depending on the host strain of choice. Even for <STRONG>Bb5</STRONG>, using an ANOVA test for six groups and 4 replicas, one can reject the null hypothesis of equal mean values (F=4.253, p=0.010) which means that the expresion of the Biobrick is indeed different. Each strain proved to be more “efficient” at hosting a different Biobrick
         part. For instance, <STRONG>Bb1</STRONG> and <STRONG>Bb5</STRONG> are highly expressed in strain <STRONG>JM109</STRONG>, whereas the expression of the other biobricks was low or nearly undetectable. Note
         that <STRONG>Bb3</STRONG>, containing the weak promoter J23110, could not be detected in four out of the six strains. It is also remarkable the fact that in <STRONG>1A</STRONG> and in <STRONG>1B</STRONG> (<STRONG>Bb1</STRONG> and <STRONG>Bb2</STRONG> are under the control of the same promoter) the pattern obtained is almost equal among strains except in <STRONG>JM109</STRONG>. 
         <DIV class="thumbnail col-sm-10 center-block"><DIV class="caption"><STRONG>Figure 1. Behavior of Biobrick parts in six different <EM>E. coli</EM> strains.</STRONG>
               Fluorescence intensity of Bb1, Bb2, and Bb3 (A, B, and C, respectively) was corrected by the fluorescence of cells containing an empty plasmid. Absorbance values of Bb4, Bb5, and Bb6 (D, E, and F, respectively) was corrected by the absorbance displayed by non-transformed cells (bacteria containing an empty plasmid where not used because the white-blue screening sytem present in the vector).
            </DIV></DIV><P><A href="https://2014.igem.org/Team:Valencia_Biocampus/Modeling#StandardizationTab" target="_blank"><STRONG>Standarization modeling, click here to see</STRONG></A></P><H3>Discussion and conclusions</H3><P>
            We demonstrate here a difference in the output of the Biobricks depending on the host strain. If we assume that they are in the <STRONG>same physiological state</STRONG> this results could be related  to epistasis, since epistasis describes how gene interactions can affect phenotypes (<A href="http://www.nature.com/scitable/topicpage/epistasis-gene-interaction-and-phenotype-effects-460" target="_blank">Miko, I. Epistasis: Gene interaction and phenotype effects. Nature Education (2008) 1(1):197</A>). In other words, <STRONG>the behaviour of a particular gene strongly depends on the presence of others</STRONG>, on the <A href="https://static.igem.org/mediawiki/2014/5/58/Genetic_backgrounds_of_each_Escherichia_coli_strain_used_in_The_ST2OOL_Project.pdf" target="_blank">genetic background of each <EM>Escherichia coli</EM> strain</A>.
         </P><P>
            The different behavior can be a consequence of the <STRONG>synthesis rate</STRONG>, which also depends on the genetic background. The measures were made in the exponential phase of growing, where the bacteria are constantly dividing, in such a way that the protein synthesized is also being divided. Maybe, if the same experiment was carried out when the bacterial cells had reached the stationary phase, the output obtained could be more homogeneous. We are currently testing that.
         </P><P>
            In any case, with the results we obtained, we have to conclude that <STRONG>Biobrick parts do not behave in a standard way even whitin strains of the same bacterial species</STRONG>.
      </P></DIV><DIV class="tab-pane fade" id="OrthogonalitySection"><DIV class="col-sm-12 pull-right"><H2>Orthogonality results</H2><P>
               In Synthetic Biology, <B>two constructions can be considered orthogonal when they only interact at specific and predictable interphases</B>, and do not disturb
               each other. We have studied this desirable feature of Biobrick parts by combining two constructions in the same cell and comparing this output with the one
               produced by simple transformations. To do this, <B>we used both standard fluorimetry assays and flow cytometry</B>, which allowed to get information at the cell level. Then, we wanted to go one step further: what is the effect of a simple transformation (a plasmid with a Biobrick part) into the cell architecture? To get some insights about this,
               <B>we performed a proteomic analysis</B> in which the whole proteome of an <EM>E. coli</EM> strain transformed with a Biobrick part was
               compared to that of the non-transformed, control strain. Last, but not least, <A href="https://2014.igem.org/Team:Valencia_Biocampus/Modeling#OrthogonalityTab" target="_blank"><STRONG>a detailed set of equations </STRONG></A>modeling the behavior of cells carrying two Biobrick parts was developed.
            </P><H2>1. Standard Fluorimetry assays</H2><P>
               We performed experiments with two different combinations of Biobrick parts containing a fluorescent protein under the control of a promoter sequence from
               Anderson’s promoter collection: Bb1 (GFP with strong promoter J23104) + Bb2 (RFP with strong promoter J23104) and Bb2 + Bb3 (GFP with less strong promoter
               J23110). In all cases, several controls were used: wild type (non-transformed) cells, simple transformations with either the biobrick or an empty plasmid,
               and co-transformations of the biobricks and the empty plasmids. It has to be noted that there was a high background emission of red fluorescence by the cells in all cases. Our results are shown in Figures 1 and 2.
            </P><H3> General protocol(1) </H3><UL><LI>
                  Strike the desired strains on LB plates with the appropriate antibiotic. Allow to grow by incubating at 37°C at least 24 hours.
               </LI><LI>
                  Gather a large amount of bacteria with an inoculation loop and transfer to 1 mL of LB. Incubate during 20 minutes at room temperature.
               </LI><LI>
                  Inoculate 100 µL of bacterial suspension in a tube containing 3 mL of LB with antibiotic. Perform three biological replicas of each combination of Biobrick
                  parts.
               </LI><LI>
                  Incubate until OD<SUB>600</SUB> is between 0.1-0.3.
               </LI><LI>
                  Measure OD<SUB>600</SUB>.
               </LI><LI>
                  Measure GFP fluorescence (exc.= 493nm; emis.=505nm) and RFP fluorescence (exc.= 576 nm; emis.= 592 nm) in a FP6200 spectrofluorimeter (Jasco, Easton, MD) fluorometer using standard plastic cuvettes.
               </LI><LI>
                  With excel, normalize fluorescence by cell density, calculate the average and standard deviation taking into account the three biological replicas, and represent the data.
               </LI></UL><H3> Fluorimetry Results </H3><DIV class="thumbnail col-sm-11 center-block"><DIV class="caption"><STRONG>
                  Figure 1. Fluorescence emission in XL1-Blue <EM>Escherichia coli</EM> strains co-expressing Biobrick parts 1, 2, and 3 reveals asymmetry of behaviors (lack of
                  orthogonality).
                  </STRONG>
                  Left, cells containing both Biobrick parts 1 and 2, Biobrick part 1 and an empty kanamycin plasmid (e.k.p), Biobrick part 2 and an empty ampicillin plasmid
                  (e.a.p.), Biobrick part 1, and Biobrick part 2. Right: Cells containing both Biobrick parts 2 and 3, Biobrick part 2 and an empty kanamycin plasmid
                  (e.k.p), Biobrick part 3 and an empty ampicillin plasmid (e.a.p.), Biobrick part 2, and Biobrick part 3. XL1-Blue <EM>E. coli</EM> cells containing no
                  plasmids or empty plasmids were used as controls. Both green (505 nm) and red (592 nm) fluorescence intensity (FI) were measured for all samples and
                  normalized by the optical density at 600 nm (OD600).
               </DIV></DIV><DIV class="thumbnail col-sm-7 center-block"><DIV class="caption"><STRONG>
                  Figure 2. Fluorescence emission by DH5α <EM>E. coli</EM> strains co-expressing Biobrick parts 1 and 2 also reveals non orthogonal expression of
                  Biobrick parts.
                  </STRONG>
                  Cells containing both Biobrick parts 1 and 2, Biobrick part 1 and an empty kanamycin plasmid (e.k.p), Biobrick part 2 and an empty ampicillin plasmid
                  (e.a.p.), Biobrick part 1, and Biobrick part 2 were analyzed. XL1-Blue <EM>E. coli</EM> cells containing no plasmids or empty plasmids were used as
                  controls. Both green (505 nm) and red (592 nm) fluorescence intensity (FI) were measured for all samples and normalized by the optical density at 600 nm
                  (OD600).
             </DIV></DIV><P><STRONG>When in co-transformation, Bb1 and Bb2 showed a clearly asymmetric expression</STRONG>
               . <STRONG>Bb1 was highly expressed in comparison to Bb2 (2-fold in strain XL1-Blue and 3-fold in DH5α</STRONG>). A
               similar phenomenon was observed when either Bb1 or Bb2 were cotransformed with empty plasmids. Again, less fluorescence was detected in comparison to the
               simple transformants. Surprisingly, this fluorescence decay was strikingly different between both <EM>E. coli</EM> strains: Bb1 strongly lowered its
               expression in DH5α, whereas Bb2 showed a higher decay in XL1-Blue cells.
            </P><P>
               However, the result was completely different when Bb2 and Bb3 were present in the same cell. In the XL1-Blue strain, the fluorescence intensity of Bb2 was
               as high as that of the single transformant, whereas the fluorescence of Bb3 –containing a less strong promoter- dropped dramatically. Fluorescence
               intensity of Bb3 was not detectable in strain DH5α under our experimental conditions.  
            </P><P><STRONG>
               With these results with co-transformants, we have to conclude that Biobrick parts, even if controlled by the same promoter and cloned into the same
               plasmid, are not expressed at similar rates.
               </STRONG>
               But, is this lack of orthogonality stable and reproducible? In other words, is the asymmetric output always the same or does it fluctuate in time or among cells? We were able to answer this question by using flow cytometry. Check our results below!
            </P><H2>2. Flow cytometry</H2><P>
               To study cell-level fluorescence we used <B>flow cytometry</B>.  To prepare the samples, XL1-Blue cultures (wild type, simple transformants for Bb1 and Bb2, and cotransformants with Bb1+Bb2) were grown in LB with the appropriate antibiotic, harvested and resuspended in 1mL PBS.  From now on, we will refer to Bb1 as GFP, and Bb2 as RFP.
            </P><H3>Orthogonality at the cell level</H3>
            Our first result, shown in Figure 3, is the <B>confirmation of non-orthogonal behavior as deduced by a higher expression of GFP in comparison to RFP in cotransformed XL1-Blue cultures</B>. There is four to five times more expression of GFP in comparison to RFP (Fig 3D).  These results are similar to those we previously obtained by fluorimetry (with younger cultures, 5 h), in which an imbalanced GFP:RFP rate of about 2:1 was detected.
            It has to be stressed that, unexpectedly, only a fraction (around 10%) of the cells exhibited activity of both reporter proteins, indicating that <B>the fluorescence of a culture as it is usually measured (fluorimetry) represents in fact and average of a very diverse pool of individual outputs</B>, including mono-active cells. 
            We plan to repeat the assay using the reverse conditions: GFP in a Kan resistant plasmid, and RFP in an Amp resistant plasmid.  
            <DIV class="thumbnail col-sm-7 center-block"><DIV class="caption"><STRONG>
                  Figure 3. Dot plots of XL1-Blue reveal a non-orthogonal behavior.
                  </STRONG>
                  Dot plots obtained by flow cytometry. A, B, C &amp; D correspond to wild type, GFP transformant, RFP transformant and cotransformed cells, respectively.  FITC-A, or X axis, indicates green fluorescence (505 nm), whereas PE-A, or Y axis, indicates red fluorescence (592 nm).
               </DIV></DIV><H3>Orthogonality changes in time</H3>
            Figure 4 shows a clear correlation of fluorescense and time in mono-transformants.  Fluorescence is very low for all samples at 5h (approximately 30% of cells transformed with Bb1 express GFP but only 1% of cells transformed with Bb2 express RFP); moderate at 10h for GFP mono-transformants (41% and 1.5% respectively); and maximum after 20h, when GFP is expressed by 80% of the cells, and RFP is expressed by approximately 45% of the cells. Regarding co-transformants, fluorescence also increased with time but even after 20h only a fraction (10%) of doubly fluorescent cells were detected.
            <DIV class="thumbnail col-sm-9 center-block"><DIV class="caption"><STRONG>
                  Figure 4. XL1-Blue dot plots indicating an increase of fluorescent protein expression with time. 
                  </STRONG>
                  Samples of the four variations (wild type, GFP transformants, RFP transformants, and cotransformed cells) are taken at 5, 10 and 20 hours.  FITC-A, or X axis, represents green fluorescence (505 nm).  PE-A, or Y axis, represents red fluorescence (592 nm).
               </DIV></DIV><P>
               These results indicate that <B>higher fluorescence levels are reached when cells are reaching stationary phase</B> and that <B>even long incubation times fail to yield a majoritary population of co-transformed cells with dual GFP and RFP activity</B>.
            </P><H3>Fluorescense rates within cultures: a very stable imbalance</H3>
            Our last assay aimed at answering this paradoxical question: <B>is the lack of orthogonality a stable trait?</B> In other words, is the GFP:RFP imbalance always the same? The answer, according to Figure 5 is clear: <B>Yes, independent cultures exhibit a striking similarity in their flow cytometry plots</B>. Almost half of the cell population only displays GFP activity, whereas only a relatively small fraction is either displaying RFP fluorescence or both.
            <DIV class="thumbnail col-sm-7 center-block"><DIV class="caption"><STRONG>
                  Figure 5. Independent cultures of cotransformed cells indicate the lack of orthogonality is stable.
                  </STRONG>
                  Analysis by flow cytometry of XL1-Blue cotransformed cells.  One dot plot is shown for each independent culture.  FITC-A indicates green fluorescence (505 nm).  PE-A indicates red fluorescence (592 nm). Below: diagrams representing the percentage of cells that express no fluorescence, RFP, GFP, or both.
               </DIV></DIV><H2 id="Orthogonality-Proteomics">3. Proteomics</H2>
            Our experiments involving <I>E. coli</I> cells co-expressing two different Biobrick parts revealed a lack of orthogonality between them. But <B>what is going on with all the other parts naturally present in the chassis?</B> Does the expression of a Biobrick part interact or interfere with any of the genes expressed by the host? In order to test this hypothesis, <B>we performed a proteomic analysis on the simplest scenario</B>: we compared the proteome of the DH5α <I>E. coli</I> strain expressing Biobrick part 1 (consisting of a green fluorescent protein under the control of a constitutive promoter) with that of the same strain carrying the empty cloning plasmid and the non-transformed wild-type strain.
            <H3>General protocol (2)</H3><UL><LI>
                  Set up three independent 5-mL cultures of the strains named before as we explained above for the standard fluorimetry assays.
               </LI><LI>When an OD value between 0.2 and 0.4 is reached, pellet the cells at 4ºC and then wash them with sterile ice-cooled PBS buffer.</LI><LI>Resuspend the cells in 1 mL of ice-cooled sterile PBS buffer.</LI></UL><P>
               After this, we kept the tubes on ice and brought them to the Proteomics lab of the University of Valencia. The proteomic analysis was carried out in the SCSIE_university of Valencia Proteomics Unit , a member of ISCIII ProteoRed Proteomics Platform. Once there, proteins were isolated and quantified with the supervision of the technicians. Finally, proteins were digested and labeled by using the iTRAQ technology, which allowed us to work with all the samples in the mass spectrometer at a time and then assign each identification to a particular sample.
            </P><H3> Proteomics Results</H3>
               A general view on the protein profile of each sample was firstly observed on an SDS-PAGE gel (Figure 6). Strong differences were detected, as expected, in the bands corresponding to the GFP and the β-lactamase encoded by the ampicillin-resistance gene of the pUC57 plasmid used for cloning. 
               <DIV class="thumbnail col-sm-7 center-block"><DIV class="caption"><STRONG>
                     Figure 6. SDS-PAGE gel showing the protein profile of the samples used for proteomics analysis.
                     </STRONG>
                     WT: wild-type <I>E. coli</I> strain DH5α (2 replicates); Bb1: Biobrick part 1-expressing strain (3 replicates); ᴓ: strain carrying the empty cloning plasmid. Red and green arrows indicate β-lactamase and GFP, respectively.
                  </DIV></DIV><P>
                  An average of <B>1500 proteins per sample were identified</B> with the mass spectrometer. Among them, <B>472 proteins could be properly quantified</B>. Figure 7 shows the variation of a subset of proteins among samples. Notice that chaperone proteins are specially over-represented in the strain expressing Bb1, whereas proteins involved in response to oxidative stress are particularly abundant in the strains carrying the empty plasmid.
               </P><DIV class="thumbnail col-sm-7 center-block"><DIV class="caption"><STRONG>
                     Figure 7. Representation of a subset of differentially expressed proteins among samples.
                     </STRONG>
                     Red and green colors indicate strong over-representation and under-representation, respectively. Blue arrows indicating β-lactamase and GFP; orange arrows indicating chaperone proteins; purple arrows indicating oxidative stress-related proteins.
                  </DIV></DIV><P>
                  But this apparent variation in the proteome changes when applying some statistics. A relative comparison of the average expression level of each one of the quantified proteins in the three groups of samples (WT strain, Bb1-expressing strain, and empty plasmid-carrying strain) is shown in Figure 8. Those proteins displaying the highest alterations in their expression levels are marked with numbers and identified in the table on the right.
               </P><DIV class="thumbnail col-sm-7 right"><DIV class="caption"><STRONG>
                     Figure 8. Differential expression level of the proteins quantified by iTRAQ mass spectrometry.
                     </STRONG>
                     Each of the 472 quantified proteins has been represented for each group of samples: WT (blue line), Bb1-expressing strain (green line), and e.a.p. (empty Amp plasmid)-carrying strain (red line). Proteins on the X axis ordered in decreasing confidence level.
                  </DIV></DIV><P>
                  As deduced in Figure 7, <B>a very low number of proteins strongly changed their expression</B> in the strains carrying Bb1 or the empty plasmid. Several statistical tests (FDR-corrected ANOVA, T-test, p-value-weighted analysis) were performed with these proteins, and only three of them proved to be significant in all the tests. These proteins were the <B>GFP</B> (only present in Bb1 strain), the <B>β-lactamase</B> (present in both Bb1 and ᴓ strains), and the <B>chaperone protein DnaJ</B>, which was particularly abundant in the strain expressing Bb1. The high expression of chaperone proteins was expected, since many molecules of GFP need to be properly folded in order to avoid the formation of inclusion bodies and cellular toxicity.
                  From this experiment, <B>we have to conclude that the expression of a simple Biobrick part such as a fluorescent protein in an <I>E. coli</I> cell does not alter the proteomic architecture of the chassis in a significant way. Therefore, Biobrick part 1 could be considered orthogonal with respect to chassis' parts</B>.
               </P><H2>4. Conclusions</H2><UL><LI>Surprisingly, standard fluorimetry assays show an <B>assymetric output</B> of red and green fluorescent proteins <B>in co-transformed cells</B>.</LI><LI><B>Flow cytometry</B> confirms this asymmetric activity and strongly suggests the co-existence of <B>sub-populations of cells with different outputs</B>. This asymmetry proved <B>very reproducible</B> among cultures, and matched the predictions of our <A href="https://2014.igem.org/Team:Valencia_Biocampus/Modeling#OrthogonalityTab" target="_blank"><STRONG>theoretical model.</STRONG></A></LI><LI>The activity of fluorescent proteins <B>dramatically changes over time</B>, reaching a peak when cells are in the stationary phase. Despite being under the control of the same promoter, GFP is expressed earlier than RFP. <B>To the question “are two Biobricks orthogonal?” we thus suggest to add this other question: “when are they orthogonal?”</B></LI><LI><B>Proteomic analysis</B> reveals the <B>robustness of <I>E. coli</I></B> confronted to heterologous gene expression: only minor proteomic changes were found compared to the controls. An <B>orthogonal relation</B> between Biobrick part 1 and the proteomic architecture (chassis) is thus confirmed.</LI></UL></DIV></DIV><DIV class="tab-pane fade" id="OpenLicenseSection"><H2> Open License <SPAN class="small">(and Responsible Research and Innovation)</SPAN></H2><P>
      Not unlike scientific research, RRI is about common sense, about combining self-profit with others’ profit, about taking care of our planet. We believe that being responsible is not the opposite of being selfish; responsibility is just a broad-minded way of selfishness that points at the long-term benefit.
      And there is no long-term benefit without a sustainable society in a sustainable environment. What is good for our neighbors and for our home must be good for us as well.
   </P><H3> Our Proposal: Responsible Research and Innovation as a tool to choose within the IP ecosystem.</H3><P>
      Since we, our Human Practices sub-team, are a biotechnology and a law student, we are also glad to see that another key concept in IP is a beautiful science-law metaphor: Jane Calvert &amp; Drew Endy's <STRONG>“diverse ecology” scenario</STRONG>. This image fits well with our interdisciplinary spirit and we found it very useful because it evokes very elegantly the range of different solutions out there to choose from in order to use a scientific invention in a given way.
   </P><P>
      Having reached this point, it is obvious which the next step is: there is a whole ecosystem of IP figures (see the poetic cover illustration by Paula, another team member), and one has to be chosen. How? Our bet is simple: by using <STRONG>RRI principles</STRONG> (mutual learning, human resources management, fundamental rights respect, research formation and transparency) as the main guide in the decision process.
   </P><P>
      The following figure illustrates what we propose:
   </P><DIV class="thumbnail col-sm-7 center-block"><DIV class="caption"><STRONG>Figure 1.</STRONG>
         RRI-based decision process on a diverse ecology scenario. From a diverse array of IP protection possibilities (diverse ecology scenario), we propose RRI as a framework to choose the most suitable one. In order to assist in the decision process, experts’ opinion is the major force, and we propose here two tools to contribute to this process: a common language exemplified by our dictionaries (Annex I), and a formula-based assistance procedure (Annex II).
      </DIV></DIV><P>
      In our view, the decision process has to fall upon the responsibility of the experts or examiners, as it is the case today. But we propose that a formation
      on RRI for patent examiners and other IP actors is introduced. In the same way that scientists need a basic law background for transferring their knowledge
      into applied solutions, we envisage a future in which everybody involved in the production of a given IP figure (a patent, for example, from scientists and
      engineers to ethics specialists, lawyers and other experts) should share a common interdisciplinary formation on RRI. A common language is the first step
      towards a common goal.
   </P><P>
      Additionally, we propose in this report two tools that might contribute to avoid misunderstandings and to make a simplified decision process available for
      a wide audience. In order to accomplish the former objective, we have prepared the <STRONG>two common-languages short dictionaries</STRONG> listed in <STRONG>Annex II</STRONG>. To achieve
      the latter, we propose, for the first time for the best of our knowledge, <STRONG>a math-based equation</STRONG> that could contribute to make IP issues available to a wide
      audience (for a complete description see <STRONG>Annex I</STRONG>). We have prepared the equation in two steps. First, we combined the patentability requirements (novelty, inventive step, industrial
      application) in a mathematically logical way to reach our draft equation (<STRONG>F<SUB>0</SUB></STRONG>). Then, we submitted the formula to several experts to have their feedback, and thus
      modify <STRONG>F<SUB>0</SUB></STRONG> accordingly to yield <STRONG>F<SUB>1</SUB></STRONG> (this reflexive process, combining design with improvement rounds, elegantly evokes a biotechnology strategy called
      <EM>directed evolution</EM>).
   </P><DIV class="thumbnail col-sm-7 center-block"><DIV class="caption"><STRONG>Figure 2.</STRONG>
         Human decisions are only acceptable if no help is used? Left, chess player. Right, graphic representation of the equation <STRONG>F<SUB>0</SUB></STRONG><SUB> we initially proposed for patentability determination, as a function of the Industrial Application (<STRONG>I</STRONG>) and the Inventive step (<STRONG>A</STRONG>). See <STRONG>Annex II</STRONG> for details.
      </SUB></DIV></DIV><P>
      The very idea of using maths to decide whether something is patentable may seem odd. The opinion of the law experts was useful; but we wanted to have a
      broader feedback from other social actors. We thus carried out a survey to probe the opinion of <EM>iGEMers</EM>, scientists/engineers and law students, but also of
      general population to compare their view on our proposal. The <STRONG>C section of Annex II</STRONG> describes the results of this survey that can be summarize as follows: the
      vast majority of the 526 respondents (70 to around 80%) found that a math-based approach could be a useful tool, but should not substitute humans as the final decision makers. We
      could not agree more on that.
   </P><DIV class="caption" style="text-align:center">
      Original watercolor by Paula Villaescusa.
   </DIV><P style="text-align:center"><A href="https://static.igem.org/mediawiki/2014/8/84/AnnexIFinal.pdf" target="_blank" class="btn btn-lg btn-default"> Annex I. Patentability index</A><A href="https://static.igem.org/mediawiki/2014/5/5c/AnnexII._.pdf" target="_blank" class="btn btn-lg btn-default"> Annex 2. Brief dictionary of IP and SB</A></P></DIV><DIV class="tab-pane fade" id="TheSeatSection"><H2> The seat of the ST<SUP>2</SUP>OOL</H2><P>
      One of the goals of our project was to select standard, stable, and orthogonal parts naturally occurring in nature. To do that, we carried out a <B>functional
      metagenomics strategy</B> aiming at selecting promoters from a library of metagenomic DNA. As a first step, we wanted to isolate metagenomic DNA from
      environmental samples, but instead of using traditional manual kits, <B>we decided to build a robot able to automatically extract metagenomic DNA for us</B>… It
      is a pleasure for our team to introduce you our <B>TOOL</B> robot:
   </P><H2>
      “The TOOL” robot
   </H2><H3>How does it work?</H3><P>
      The main goal of this machine was the automation of the DNA isolation process. This robot can be used for both <EM>E. coli</EM> (mini/maxipreps) and for DNA extraction from metagenomic samples.
   </P><P>
      The robotic system was developed in a <B>continuous</B> philosophy and all the components were assembled on a piece of wood. In short, it consisted of the following <B>modules</B>:
   </P><UL><LI><B>The control and powering system</B>: the system was controlled by an Arduino Mega 2560 microchip, powered by a computer via USB. The other components
         of the machine were powered by an ATX power supply.
      </LI><LI><B>The injection system</B>, which consisted of several screw stepper motors controlled by an A4988 driver. We chose this kind of motors because they
         allowed a quite accurate control of the position and the speed of the injections. A torque of 35N*m was also included. The system had three-way
         stopcocks in order to refill the syringes easily.
      </LI><LI><B>The temperature modules</B>: high temperatures were achieved with a system that combined hot water and Peltier cells, whereas low temperatures were
         achieved by ice cooling. Temperature was measured with a dbs1820 temperature sensor.
      </LI><LI><B>The mixing</B>: a homogeneous mixture of all the reagents was obtained thanks to the use of tubes allowing a turbulent regime.
      </LI></UL><P>
         The complete process was as follows:
      </P><OL><LI>Manually inject a suspension of the cells into 3 mL of ice-cold solution I.</LI><LI>From this step until step 8, the procedure is automatic (indicated in italics).</LI><LI><EM>200 µL of solution II are injected and mixed during the flow into the silicone tubes.</EM></LI><LI><EM>The mixture is moved through the silicone tubes to a water bath at 65 ºC and incubated for 30 minutes.</EM></LI><LI><EM>60 µL of ice-cold Potassium Acetate 3M (pH 5.0) are injected and mixed during the flow into the silicone tubes.</EM></LI><LI><EM>The mixture is moved through the silicone tubes to an ice bath and incubated 20 minutes at -20ºC.</EM></LI><LI><EM>The resulting suspension is then filtered through a membrane and insoluble particles are removed.</EM></LI><LI><EM>An equal volume of isopropanol is injected and mixed with to the filtered solution during the flow into the silicone tubes, and the mixture is incubated at room temperature for 5 minutes. </EM></LI><LI>At this point, the solution is collected in Eppendorf tubes and these tubes are centrifuged at maximum speed for 10 minutes.</LI><LI>The supernatant is discarded and the pellet washed with 500 µL of 70% (v/v) ethanol.</LI><LI>The tubes are centrifuged again at maximum speed for 3 minutes and the supernatant is discarded.</LI><LI>Finally, the pellet is dried and resuspended in 50 µL of water or elution buffer.</LI></OL><H3>Results (The Seat)</H3><P>
         As a <B>proof of concept</B>, we used a suspension of <B>soil bacteria</B> (obtained after several mild centrifugations of a suspension of compost in PBS sterile buffer)
         to isolate metagenomic DNA. Up to now, we have been able to perform steps 1-8 with our robot, and we are still working hard on the implementation of the
         final steps of the protocol. Figure 1A shows the Nanodrop analysis on the metagenomic DNA we obtained from the suspension. <B>We isolated pure</B> (A260/A280 and
         A260/A230 ratios close to 2) <B>and quite concentrated</B> (90.5 ng/µL) <B>metagenomic DNA</B>, the integrity of which was also checked on a 0.8% agarose gel (Figure
         1B). Interestingly, we also observed the bands corresponding to the 16S and 23S ribosomal RNA on the gel.
      </P><DIV class="thumbnail col-sm-7 center-block"><DIV class="caption"><STRONG>Figure 1. A)</STRONG> Screenshot of the Nanodrop output of compost metagenomic DNA isolated with the robot as described in the main text. <STRONG>B)</STRONG> TBE 0.8% agarose gel showing the nucleic acids isolated with our robot. Lane 1: M.W. marker; lanes 2-5: metagenomic DNA aliquots. Arrows indicate metagenomic DNA (upper part) and 16S/23S ribosomal RNA.
         </DIV></DIV><H3> Small budget, big ideas </H3><P>
         Table 1 shows a list of all the components and materials used to build the robot. <B>We decided to buy cheap and easy-to-find materials, so anyone can re-build or improve the robot!</B> We only spent around 500€:
      </P><DIV class="col-sm-7 center-block small"><P><STRONG>Table 1.Growth of XL1-Blue and DH5α under different pH and salt concentrations</STRONG>Materials and components used during the construction of “The TOOL” robot. The price of each component is indicated on the right.
            </P><TABLE class="table table-bordered table-condensed"><TBODY><TR><TD><STRONG>MATERIAL</STRONG></TD><TD><STRONG>NUMBER</STRONG></TD><TD><STRONG>Price (€)</STRONG></TD></TR><TR><TD>
                        ARDUINO MEGA 2560
                     </TD><TD>
                        1
                     </TD><TD>
                        12
                     </TD></TR><TR><TD>
                        WIRES
                     </TD><TD>
                        100
                     </TD><TD>
                        10
                     </TD></TR><TR><TD>
                        PROTOBOARD
                     </TD><TD>
                        1
                     </TD><TD>
                        3
                     </TD></TR><TR><TD>
                        DRIVER A4988
                     </TD><TD>
                        5
                     </TD><TD>
                        15
                     </TD></TR><TR><TD>
                        SCREW STEPPER MOTOR
                     </TD><TD>
                        4
                     </TD><TD>
                        130
                     </TD></TR><TR><TD>
                        SYRINGE
                     </TD><TD>
                        4
                     </TD><TD>
                        1
                     </TD></TR><TR><TD>
                        SCREW S
                     </TD><TD>
                        50
                     </TD><TD>
                        5
                     </TD></TR><TR><TD>
                        ALUMINUM BARS
                     </TD><TD>
                        1
                     </TD><TD>
                        3
                     </TD></TR><TR><TD>
                        WOOD TABLE
                     </TD><TD>
                        3
                     </TD><TD>
                        3
                     </TD></TR><TR><TD>
                        CLAMP
                     </TD><TD>
                        8
                     </TD><TD>
                        3
                     </TD></TR><TR><TD>
                        DRAWER GUIDE
                     </TD><TD>
                        4
                     </TD><TD>
                        3
                     </TD></TR><TR><TD>
                        SILICONE TUBES
                     </TD><TD>
                        5
                     </TD><TD>
                        1
                     </TD></TR><TR><TD>
                        PERISTALTIC PUMP
                     </TD><TD>
                        2
                     </TD><TD>
                        10
                     </TD></TR><TR><TD>
                        BEAKERS
                     </TD><TD>
                        3
                     </TD><TD>
                        7
                     </TD></TR><TR><TD>
                        SAW
                     </TD><TD>
                        12
                     </TD></TR><TR><TD>
                        FISH
                     </TD><TD>
                        1
                     </TD><TD>
                        2
                     </TD></TR><TR><TD>
                        DRILL
                     </TD><TD>
                        1
                     </TD><TD>
                        20
                     </TD></TR><TR><TD>
                        STOPCOCK 3 WAYS
                     </TD><TD>                       6
                     </TD><TD>
                        12
                     </TD></TR><TR><TD>
                        SCROLL SAW
                     </TD><TD>
                        1
                     </TD><TD>
                        8
                     </TD></TR><TR><TD>
                        FILTER
                     </TD><TD>
                        1 per sample
                     </TD><TD>
                        -
                     </TD></TR><TR><TD>
                        FILTER HOLDER (own design)
                     </TD><TD>
                        1
                     </TD><TD>
                        4
                     </TD></TR><TR><TD>
                        AID SILICONE TUBES
                     </TD><TD>
                        4
                     </TD><TD>
                        5
                     </TD></TR><TR><TD>
                        RESISTOR (400 h)
                     </TD><TD>
                        4
                     </TD><TD>
                        3
                     </TD></TR><TR><TD>
                        CAPACITOR(0.5F)
                     </TD><TD>
                        1
                     </TD><TD>
                        2
                     </TD></TR><TR><TD>
                        LIMIT SWITCH
      </TD><TD>
                        2
                     </TD><TD>
                        3
                     </TD></TR><TR><TD>
                        WELDER
                     </TD><TD>
                        1
                     </TD><TD>
                        15
                     </TD></TR><TR><TD>
                        TIN COIL
                     </TD><TD>
                        1
                     </TD><TD>
                        11
                     </TD></TR><TR><TD>
                        RULE
                     </TD><TD>
                        1
                     </TD><TD>
                        3
                     </TD></TR><TR><TD>
                        LUB
                     </TD><TD>
                        1
                     </TD><TD>
                        6
                     </TD></TR><TR><TD>
                        SILICONE PISTOL
                     </TD><TD>
                        1
                     </TD><TD>
                        6
                     </TD></TR><TR><TD>
                        SILICONE
                     </TD><TD>
                        4
                     </TD><TD>
                        1
                     </TD></TR><TR><TD>
                        SCREW DRIVER
               </TD><TD>
                        2
                     </TD><TD>
                        5
                     </TD></TR><TR><TD>
                        ATX POWER SUPPLY
                     </TD><TD>
                        1
                     </TD><TD>
                        10
                     </TD></TR><TR><TD><A href="https://www.google.es/search?es_sm=93&amp;q=dbs1820+sensor+temperature&amp;sa=X&amp;ei=TFcYVM_LJ4XqaP7VgsgB&amp;ved=0CBwQ7xYoAA">
                        dbs1820 SENSOR
                        </A></TD><TD>
                        2
                     </TD><TD>
                        6
                     </TD></TR><TR><TD>
                        PELTIER CELL
                     </TD><TD>
                        3
                     </TD><TD>
                        12
                     </TD></TR><TR><TD>
                        RELAY
                     </TD><TD>
                        1
                     </TD><TD>
                        6
                     </TD></TR><TR><TD>
                        PERISTALTIC PUMP
                     </TD><TD>
                        2
                     </TD><TD>
                        16
                     </TD></TR><TR><TD>
                        TRANSISTOR
                     </TD><TD>
                        2
                  </TD><TD>
                        1
                     </TD></TR><TR><TD>
                        Hours of work…
                     </TD><TD>
                        A lot!
                     </TD><TD>
                        A lot!
                     </TD></TR></TBODY></TABLE></DIV><H3>Selecting natural standard promoters</H3><P>
            Once we had the metagenomic DNA extraction performed by our robot, we wanted to clone a library of fragments coming from this DNA in the pAcGFP1-1 plasmid
            (Clontech Laboratories, Inc.). This plasmid contains a GFP protein placed downstream the MCS, so <B>if a promoter sequence is inserted in the polylinker, the
            expression of the fluorescent protein is triggered</B> and thus fluorescent transformants can be easily detected.
         </P><P>
            We digested both the plasmid and the metagenomic DNA with the restriction enzymes EcoRI and BamHI and then purified the resulting fragments (Figure 2).
         </P><DIV class="thumbnail col-sm-3 center-block"><DIV class="caption"><STRONG>Figure 2. TBE 0.8% agarose gel showing the plasmid and the metagenomic DNA fragments obtained.</STRONG> Lane 1: MW marker; lane 2: undigested mgDNA; lane 3:  digested mgDNA; lane 4: digested pAcGFP1-1 plasmid.
            </DIV></DIV><P>
            We set up a ligation reaction, incubated it overnight, and then transformed <EM>E. coli</EM> DH5α competent cells. We obtained around 300 colonies, but
            none was fluorescent… We performed a colony PCR with oligonucleotides flanking the MCS of the pAcGFP1-1 plasmid in order to check whether the transformant
            colonies were carrying <B>inserts of metagenomic DNA</B>, and <B>confirmed their presence</B> as you can see in Figure 3.
         </P><DIV class="thumbnail col-sm-8 center-block"><DIV class="caption"><STRONG>Figure 3. TBE 0.8% agarose gel showing the results of the colony-PCRs.</STRONG> Lanes 1 and 10: MW marker; lanes 2-9 and 11-16: independent colonies; lane 17: negative control.
            </DIV></DIV><P>
            We are working now on improving the efficiency of the transformation, so we can increase the probability of &quot;catching&quot; a good promoter sequence… we hope to succeed soon!
         </P></DIV><DIV class="tab-pane fade" id="ModelingSection"><H2>Modeling</H2><P>
      We summarize here our main contributions modeling The ST$^2$OOL. Our hypothesis, formulae and analysis can be found in the <A href="Modeling#">modeling section</A>. 
      </P><P><A href="Modeling#StabilityTab"><STRONG>Stability</STRONG></A>
      We have modeled the variation of activity per cell of Biobricks with temperature and pH changes by considering variation in their transcription 
      and translation rates. Our model fits the cell fluorescence trend as a function of temperature. <STRONG>We have found that transcription rate increases with temperature due to a higher number of free σ factors, and translation rate decreases when temperature is above the optimal</STRONG>. Unfortunately, this is not all the story. Our model dramatically fails to fit the amount of variation, which is much larger experimentally than the predicted one. Let us compare theoretical predictions and experimental results. In both studies, XL1-Blue strain transformed with Biobricks Bb1 or Bb3, we find that the ratio between theoretical prediction and experimental result is the same at sub-optimal temperatures (in fact, accidentally too close given the error bars). <STRONG>This fact suggests that the extra temperature dependence not included in our model should come from cell mechanisms independent on parameters of the Biobricks (promoter strength, ...)</STRONG>. Quite on the contrary, the comparison of theoretical predictions and experimental results at larger 
      temperatures show that our model has not yet captured the dependence of the magnitude of the decrease in activity with the Biobrick parameters.
      </P><P><A href="Modeling#StandardizationTab"><STRONG>Standardization</STRONG></A><STRONG>We have built an app that computes the standardization-stability index and generates a promoter sequence for a given expression rate</STRONG>.
      Our wet-lab results show that Biobrick 5 is the most standard one. Our model predicts that systems with less modules or factors will be the more stable and standard. Biobrick 5 has a tetracycline depending promoter, whose limitating step is the binding of tetracycline to TetR. In this scenario, the intracellular concentration of aTc and the binding to the TetR repressor do not depend on the strain, <STRONG>so there is only one factor in which the strain could have an important effect: the TetR intracellular concentration, which has a constitutive expression</STRONG>. Analyzing the other Biobricks, we found that the strains with less expression in the constitutive promoter are those ones with higher expression when we use the Ptet promoter (but JM109 and BB1), which agrees with our prediction.
      </P><P><A href="Modeling#OrthogonalityTab"><STRONG>Orthogonality</STRONG></A>
      We have studied deterministic and stochastic models of the plasmid growth to learn about the unspecific interaction between two plasmids in a cell. We found that this system is unstable and the only way to keep the plasmid levels is by introducing a resistance. However, it is not a good way to maintain those levels: bacteria which have non-desired levels of plasmids are killed. This leads to a lower yield. Also, <STRONG>there is an important variability, which can drive the system away from the expected behavior</STRONG>.
      In conclusion, <STRONG>plasmids with an identical origin of replication are not orthogonal per se</STRONG>. Although we can introduce use different resistances to tune the system, it may not behave as expected due to a great variability. As a solution, we could use plasmids with different replication mechanisms, but they could show different dynamics, making the system even more difficult to control. Therefore, new plasmids must be designed to make engineered cells more reliable and predictable, in order to progress in synthetic biology.</P><P><A href="Modeling#OpenLicenseTab"><STRONG>Open License</STRONG></A><STRONG>We have modeled the susceptibility to patent a human creation and define the factors that influence it</STRONG>. Inventive is the most relevant variable but it is regulated by the industrial application. The highest inventive step can be reduced by almost a factor 2 with the lowest industrial application. On the contrary, the lowest inventive step is not affected by the industrial application. After computing all the discussed factors, the patentability index can run from 0 to 10. The reader can find a detailed discussion of the questionnaires, tests, analysis of the index and references in the area <A href="HumanPractices">Human Practices</A>.
      </P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Valencia_Biocampus/Results">http://2014.igem.org/Team:Valencia_Biocampus/Results</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Valencia_Biocampus/Results" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Valencia_Biocampus/Results" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Valencia_Biocampus/Results&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Valencia_Biocampus/Results&amp;oldid=377289" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>