"
Page
Discussion
View source
History
teams
Log in
 
Team:Valencia Biocampus/ProjectTestingNewVersion
From 2014.igem.org
Project
The Team
Results
Modeling
Biobricks
Human Practices
Interlab Study
Achievements
Attributions
Art&More
The ST2OOL project
Overview
STability
STandardization
Orthogonality
Open License
The Seat of the ST2OOL
Overview
Synthetic Biology implies an engineering perspective on biotechnology. Similarly to man-made objects, cells are expected to be decoupled, modified and even built from scratch. However, there is a general assent on the current difficulties for fully -and predictably- engineering living organisms, which are always subjected to strong evolutionary constraints. The ST2OOL project aims at deeply studying four of the key engineering pillars of Synthetic Biology. ST2OOL stands for STandardization, STability, Orthogonality and Open Licence.
The first approach, an analytical one, will consist of a vast range of experimental studies to find out how standard, stable, orthogonal and patentable several selected Biobrick are. The second approach, a synthetic one, will include functional metagenomics: several environmental libraries will be set and screened in E. coli in order to select new biological parts -promoters-, not because of their strength but because of their particularly standard, stable or orthogonal behavior. Taken together, the results of our project are expected to contribute in answering this key question: Is life fully engineerable?
Stability
Our subteam aims to test the stability of ten different Biobricks transformed in two different strains of Escherichia coli, XL1 Blue and DH5α . Stability is determined by subjecting the transformed cells to different types of stress and comparing the output with non-transformed ones     grown in the same conditions, as well as with cells transformed with an empty plasmid (without Biobrick).
Temperature Stress
We start out by establishing an overnight culture of each strain, every one of which will be subjected to the different temperatures. The experiment will     be carried out in an improvised miniature system we just developed; PCR tubes filled with 75 µL of culture are placed in a thermo cycler, in which a     gradient of temperatures from 30-50ºC will be set. In each round, four replicas of two strains are to be tested simultaneously at 12 different     temperatures:
four replicas of two strains tested simultaneously at 12 different temperatures
After an incubation period at the given temperatures the tubes containing the culture will be collected to measure the O.D. Fluorescence will also be     measured and later normalized to O.D.600, to compare the expression of the different Biobricks.
Material Fatigue
The idea is analogous to material fatigue in material sciences; in materials science , fatigue is the weakening of a material caused by     repeatedly applied loads. If the loads are above a certain threshold, microscopic cracks will begin to form at the stress concentrators such as the     surface, persistent slip bands and grain interfaces. In our case, we chose to induce the fatigue by subjecting the cells to temperature fluctuation, every     minute the temperature will be changed from 37ºC (E.coli optimum growth temperature) to 41ºC and back to 37ºC again, this way, the temperature     oscillates every minute from 37 to 41ºC for 4 hours. What will be the effect of such thermal fluctuations?
pH & Salinity
One of the best known factors that conditions bacterial growth is proton and salt concentration in the media. This experiment, as simple as may seem,     offers the opportunity to test cell viability as well as the Biobrick behaviour at different pH and salt concentrations. Overnight cultures of both strains     will be resuspended in different LB media with pH ranging from 5 to 9 and incubated for 90 minutes, after which O.D.600 and fluorescence of each     tube is to be measured. The design of the experiment is the same for salts, ranging from LB medium without salt to an extra 4% of salt, maintaining the     normal pH of the LB medium for all the tubes.
UV Radiation
As per today, high doses of UV light are proven detrimental to life, but, to what extent is it injurious to an E.coli cell? Is transformation a     factor that enhances the lethality of UV radiation or, contrary to what logic dictates, helps resist the UV radiation? Curiosity led us to design an     experiment which would answer these questions: overnight cultures of the strains will be diluted and spot-plated on LB agar. Subsequently, the spots will     be subjected to timed pulses of UV radiation (intensity of 340 µW/cm2) with a control that receives no UV irradiation at all. Thereafter the     plates will be incubated and fluorescence will be measured and normalized to O.D.600.
Vacuum
A simple experiment which mainly consists of exposing the cells, spot-plated on Petri dishes, to vacuum during 48 hours will be carried out; the output     from these, as always, compared to cells spot-plated and grown in optimal conditions, through OD600 and fluorescence assays.
Standardization
One should not assume that a functional module working fine in one cell type will work the same way in even a closely related cell type (Renshaw, 1993). Whereas traditional engineering practices typically rely on the standardization of parts, the uncertain and intricate nature of biology makes standardization in the synthetic biology field difficult. Beyond typical circuit design issues, synthetic biologists must also account for cell death, crosstalk, mutations, intracellular and extracellular conditions, noise and other biological phenomena. As the number of system components grows, it becomes increasingly difficult to coordinate component inputs and outputs to produce the overall desired behaviour (Purnick & Weiss, 2009). In the ST²OOL Project, we wanted to check how standard biobricks are. For this goal, we are using a set of strains of Escherichia coli transformed, in single transformation, with ten different Biobricks whose outputs will be measured by fluorometry, colorimetry or luminometry. The aim of this approach is simple and fits with the answer to the following question:
Will the Biobricks behave the same way independently of the host strain they have been transformed into? Will we obtain the same output from all of them? The answer to these questions… in two months!
References
Purnick, P. E. M. & Weiss, R. The second wave of synthetic biology: from modules to systems. Nature Reviews 10, 410-422 (2009) .
Purnick, P. E. M. & Weiss, R. Renshaw, E. Modelling Biological Populations in Space and Time. Cambridge University Press (1993) .
Orthogonality
Do biobricks behave independently when cotransformed in a cell? In other words, can we consider them to be orthogonal? Orthogonality is a mathematical     concept that refers to the independence of behavior. Translating this concept into the biological realm, two constructions can be considered orthogonal     when they only interact at specific and predictable interphases and do not disturb each other. This is a desirable feature of Biobricks if one wants to     build synthetic circuits with predictable and robust behaviors.
To study this pillar of synthetic biology we are analyzing different combinations of Biobricks in cotransformed cells. That way, we can study the     expression of pairs of Biobricks and compare them with simple transformations, and therefore determine whether the presence of two constructions in the     same cell affects their behavior. In order to do so, we will measure the output of each Biobrick (i.e. fluorescense) in both the cotransformants and the     controls.
We are going to study three combinations, taking in to account that each pair of biobricks is controlled by the same promoter and only differs in the     protein expressed and the antibiotic resistance:
Constitutive promoter I (strong) – GFP & Constitutive promoter I (strong) – RFP
Constitutive promoter II (less strong) – GFP & Constitutive promoter I (strong) – RFP
Ptrc – lacZ & Ptrc – luciferase
Besides individually transformed cells, we will use a control consisting of cotransformed cells with a Biobrick-bearing plasmid and an empty one.
We are currently working on all this. But we have another, very ambitious experiment. We plan to determine the effect of the production of a single     heterologous protein on the proteomic architecture of the host cell. For this endeavor, we just submitted our samples for a proteomic quantification.
Open License
We did not want the Human Practices section to be a cosmetic addendum to our iGEM project. In The St2OOL, we have performed a critical but     constructive analysis of four of the pillars of the competition. Three of them are technical, engineering assumptions (stability, standardization,     orthogonality). The fourth leg of our St2OOL is for us at least as important as the other three, and deals with the answer to this difficult     question: who owns synthetic bio-constructions?
iGEM is linked to the Registry of Standard Biological Parts, a non-profit organization (as iGEM Foundation itself) archiving and distributing thousands of     BioBrick parts for free ( http://parts.igem.org/Main_Page ). No patents, no licenses, no royalties. Just Open Access (either totally free access or under     certain conditions such as a creative commons subscription). As stated in their Website, it’s “an open community that runs and grows on the Get & Give     (& Share) philosophy. Users get parts, samples, data, and tools from the Registry to work on their synthetic biology projects. They will give back to     the Registry the new parts they have made, as well as data and experience on new and existing parts”. But of course, the world of Biotechnology/SynBio is     far more than iGEM and a range of Intellectual Property protection formulas do exist. Is iGEM an island of Free Access in an ocean of patentable SynBio     achievements? How can one deal with the variety of IP forms?
Human Practices Section
The Seat of the ST²OOL
By checking how standard, stable, orthogonal or patentable Biobricks are we have performed an analysis of four engineering pillars. But we wanted to     combine these pillars, these four legs of our ST2OOL, with a truly synthetic and constructive approach.
In The Seat of the ST2OOL we are developing a functional metagenomic strategy in which we aim at isolating promoter sequences from natural     environments. We will then characterize these sequences in order to select those with the highest stability, standard and orthogonal behaviors. We are     working with soil and compost samples, from which we will isolate total metagenomic DNA, which will then be digested, ligated into an expression vector and     used to built a library in E. coli. Transformants –clones with a promoter-like natural sequence- will be subjected to a range of stresses and     subtransformation steps to find out their stability and standard/orthogonal behavior. To the best of our knowledge, this is the first Synthetic Biology     functional metagenomic approach aiming at selecting biological parts not because of their “strength” but because of their compatibility with other parts     and suitability for engineering purposes.
Finally, one technical –yet important- detail: we are not using neither commercial kits nor manual protocols for isolating metagenomic DNA. We are building     our own robot…
Our robot design
Retrieved from " http://2014.igem.org/Team:Valencia_Biocampus/ProjectTestingNewVersion "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
