<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Valencia_Biocampus_Modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Valencia_Biocampus/Modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Valencia_Biocampus/Modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Valencia_Biocampus/Modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Valencia_Biocampus/Modeling&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Valencia_Biocampus/Modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Valencia Biocampus/Modeling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="container" id="headerWrapper"><UL id="header-top-menu" class="header-top-menu pull-right"><LI id="menu_a_project"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Project">Project</A></LI><LI id="menu_a_team"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Team">The Team</A></LI><LI id="menu_a_results"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Results">Results</A></LI><LI id="menu_a_modeling"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Modeling">Modeling</A></LI><LI id="menu_a_biobricks"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Biobricks">Biobricks</A></LI></UL><UL id="header-top-menu2" class="header-top-menu pull-right"><LI id="menu_a_human-practices"><A href="https://2014.igem.org/Team:Valencia_Biocampus/HumanPractices">Human Practices</A></LI><LI id="menu_a_interlab-study"><A href="https://2014.igem.org/Team:Valencia_Biocampus/InterlabStudy">Interlab Study</A></LI><LI id="menu_a_achievements"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Achievements">Achievements</A></LI><LI id="menu_a_attributions"><A href="https://2014.igem.org/Team:Valencia_Biocampus/Attributions">Attributions</A></LI><LI id="menu_a_art"><A href="https://2014.igem.org/Team:Valencia_Biocampus/ArtAndMore">Art&amp;More</A></LI></UL></DIV></DIV><DIV class="container withBackground"><H1><SPAN class="mw-headline" id="Modeling"> Modeling </SPAN></H1><P><UL id="myTab" class="nav nav-tabs" data-tabs="tabs"><LI class="active"><A href="#MainTab" data-toggle="tab">Main</A></LI><LI><A href="#StabilityTab" data-toggle="tab">Stability</A></LI><LI><A href="#StandardizationTab" data-toggle="tab">Standardization</A></LI><LI><A href="#OrthogonalityTab" data-toggle="tab">Orthogonality</A></LI><LI><A href="#OpenLicenseTab" data-toggle="tab">Open License</A></LI><LI><A href="#OtherTab" data-toggle="tab">Other projects</A></LI><LI><A href="#ReferencesTab" data-toggle="tab">References</A></LI></UL></P><DIV id="my-tab-content" class="tab-content"><DIV class="tab-pane fade in active" id="MainTab"><DIV class="col-sm-12"><H2>Presentation</H2><P>
    In this section, we present our models to describe the four legs of the ST$^2$OOL. We have closely followed all the experimental activities, so our models do not pretend to 
completely characterize a single dynamical process of a cell, but instead we intend to capture the dependences of our systems on the variables explored 
by our experiments.  
</P><P><A href="Modeling#StabilityTab"><STRONG>Stability</STRONG></A> We have modeled the variation of the activity of cells transformed with Biobricks, when subjected to different temperatures or pH, by considering changes in the Biobrick transcription 
and translation rates. 
</P><P><A href="Modeling#StandardizationTab"><STRONG>Standardization</STRONG></A> We have modeled the expression in different strains by comparing the transcription rates. We have built an app that computes the standardization-stability index and generates the promoter sequence for a given expression rate.
</P><P><A href="Modeling#OrthogonalityTab"><STRONG>Orthogonality</STRONG></A>
We have studied deterministic and stochastic models of the plasmid growth in order to learn about the unspecific interaction between two plasmids in a cell.
</P><P><B><A href="Modeling#OpenLicenseTab"><STRONG>Open License</STRONG></A></B> We have modeled the susceptibility to patent a human creation and define the factors that influence it .
</P></DIV></DIV><DIV class="tab-pane fade" id="StabilityTab"><H2>Stability</H2><P>
    Living organisms have genetic and biochemical tools that allow them to respond to stress. These tools normally up-regulate some stress-specific genes, and, directly or
    indirectly, down-regulate the rest of normal genes (most of them housekeeping genes). On the other hand, every <EM>Escherichia coli</EM> component works worse under non-optimal conditions
    (for example, nutrients uptake or ATP production), so that, every <EM>E. coli</EM> function will be reduced.
    We model the relative expression of biobricks when subjected to variations of temperature, pH and salt concentration. 
</P><H3><SPAN class="mw-headline" id="Protein_production"> Protein production </SPAN></H3><P>
    The mRNA and protein production of a cell can be described by the kinetic equations:
</P><P>
$$ \begin{eqnarray}
\frac{d N_{mRNA} }{dt}&amp;=&amp; P \cdot  r_T - N_{mRNA} \cdot  \lambda_{mRNA} \nonumber ,\\
\frac{d N_{Pr} }{dt}&amp;=&amp; N_{mRNA} \cdot r_P - N_{Pr} \cdot  \lambda_{Pr},
\nonumber
\end{eqnarray} 
$$
</P><P>
where $N_{\it{mRNA}} (N_{Pr} )$ is the number of biobrick mRNA (protein) molecules per cell, 
$\lambda_{mRNA} $ ($\lambda_{Pr}$) is the mRNA (protein) decay rate,  $r_T$ ($r_P$) is 
the transcription (translation) rate, P is the mean number of plasmids per cell.  
</P><P>
Assuming stationary messenger and protein content during the cell cycle, 
$\frac{d N_{mRNA} }{dt}=0$ and $\frac{d N_{Pr} }{dt}=0$, we get 
</P>
$$
\begin{equation}
N_{Pr}  = \frac{ r_T \cdot r_P }{ \lambda_{mRNA}\cdot \lambda_{Pr} } P .
\end{equation}
$$
<P>
This equation describes the amount of total synthesized protein as a function of calculable rates and the number of free plasmids. It can be easily shown that, after many generations, the mean number of proteins per cell is 1.5 times the protein production during a cell cycle.  
</P><H3><SPAN class="mw-headline" id="1.29_TEMPERATURE"> 1) TEMPERATURE </SPAN></H3><P>
    The expression level of our heterologous gene can be calculated by equations 1 and 2. Our main goal is to find which of the terms will change most
    significantly with the different stress factors. We are interested in the activity in comparison to a reference one, which we take as the activity of <EM>E. coli</EM> bacteria grown on the LB medium at <U>38.7 <SUP>o</SUP>C, pH 7, 1% of NaCl, no UV irradiation and 1 atm of pressure.</U></P><P>
$$
\begin{equation}
\frac{A_{c}}{A^0_{c}}  = \prod_i \frac{V_i}{V^0_i}  .
\end{equation}
$$
where $V$ is any of the factors contributing to the activity and the index $0$ refers to the reference case.
</P><H3><SPAN class="mw-headline" id="Transcription_rate"> Transcription rate </SPAN></H3><P>
    Among several factors, we will first discuss the transcription rate. In the range of temperatures (or pH) that we have studied, the polymerase (RNAP) is
    sufficiently stable to assume that the elongation rate is not affected and the interaction energies RNAP-promoter do not change significantly (for example, the promoter
    specific interaction only changes 16%, from 33 <SUP>o</SUP>C to 44 <SUP>o</SUP>C). We did not find any significant difference in the nucleotide
    compositions of <EM>E. coli</EM> under stress [2]. Therefore, main variations in the transcription rate are due to the probability of RNAP-promoter bounds 
$$
\begin{equation}
r_T (T)=\frac{1}{\frac{1}{r_I(\Delta\epsilon)}+\frac{1}{r_E}}Prob_B (\Delta\epsilon, T) \propto Prob_B (T).
\nonumber
\end{equation}
$$
    On the other hand, down-regulation affects the expression level of our Biobrick, because it works as a housekeeping gene (with a normal constitutive promoter for the RNAP). For example, <EM>E. coli</EM> has a common σ factor for housekeeping genes (σ70), and, another one (σ32) for temperature
    stress. In the case of temperature stress, the bacteria will synthetize σ32, which will bind RNA polymerase (RNAP) cores that won’t be free for σ70, and so
    they will not be able to synthetize mRNA of the normal housekeeping genes.
</P><P>
    Therefore, we just need to find a relation between the amount of σ factors and the expression level, which we derive next with thermodynamic arguments, and
    also the variation of σ factors with temperature, which can be determined experimentally.
</P><P>
The RNAP is bound to a promoter (unspecific DNA) with $\epsilon_{S}$ ($\epsilon_{NS}$) free energy. There are 
$N_{NS}$ nonspecific sites on the <EM>E. coli</EM> genome (sequences of 41bp) and R free E$\sigma$70, with $R \ll N_{NS}$ .
The combinatory number of unbound states, with energy R$\epsilon_{NS}$, is 
$$
\begin{equation}
\frac{N_{NS}!}{R!(N_{NS}-R)!} \simeq    \frac{N^R_{NS}}{R!} .
\nonumber
\end{equation}
$$
Similarly, the combinatory number with one bound state, with energy $\epsilon_{S}$+(R-1)$\epsilon_{NS}$, is 
$$
 \begin{equation}
\frac{N_{NS}!}{(R-1)!(N_{NS}-R-1)!} \simeq    \frac{N^{R-1}_{NS}}{(R-1)!} .
\nonumber
\end{equation}
$$
Then, the probability of finding a RNAP bound to a promoter is with the enter
$$
\begin{equation}
Prob_B  = \frac {\frac{R}{N_{NS}} e^{-\frac{\Delta\epsilon}{k_BT }}}{1+\frac{R}{N_{NS}} e^{-\frac{\Delta\epsilon}{k_BT }}},
\end{equation}
$$
with $\Delta\epsilon = \epsilon_{S} - \epsilon_{NS}$, calculated with the energy matrix developed in [3] and $k_B$ is 
the Bolztmann constant. R $\simeq$ 1000 and $N_{NS} \simeq 10^7$ for <EM>E. coli</EM>.  

Using the measured variation of free E$\sigma$70, in arbitrary units, with temperature, in $^o$C, [4]
$$
\begin{equation}
E\sigma70  = 0.027 T^2 - 2.12 T + 42.2,
\nonumber
\end{equation}
$$
we can determine the relative change of transcription rate
$$
\begin{equation}
\frac{r_T}{r^0_T}  = \frac{Prob_B}{Prob^0_B}  = \frac { 1+\frac{R^0}{N_{NS}} e^{-\frac{\Delta\epsilon}{k_BT }} } { \frac {R^0}{R}+\frac{R^0}{N_{NS}} e^{-\frac{\Delta\epsilon}{k_BT }}} ,
\end{equation}
$$
with
$$ 
\begin{equation}
\frac{R^0}{R}  = \frac {0.027 (T^0)^2 - 2.12 T^0 + 42.2} {0.027 T^2 - 2.12 T + 42.2} .
\end{equation}
$$
</P><H3><SPAN class="mw-headline" id="Synthesis_rate"> Synthesis rate </SPAN></H3><P>
Modeling synthesis rate follows the lines discussed in transcription rate, but the working regime is quite different.
The energies of the mRNA-ribosome sequences interactions are much larger than in the RNAP-promoter ones. 
The average difference of free energies (computed with the RBS calculator for random sequences of <EM>E. coli</EM> genes) is 
$\Delta\epsilon = -20.79 kT$, which means that, in the equation:
$$ 
\begin{equation}
\frac{r_T}{r^0_T}  = \frac{Prob_B}{Prob^0_B}  = \frac { 1+\frac{R^0}{N_{NS}} e^{-\frac{\Delta\epsilon}{k_BT }} } { \frac {R^0}{R}+\frac{R^0}{N_{NS}} e^{-\frac{\Delta\epsilon}{k_BT }}} ,
\end{equation}
$$
The term $\frac{R^0}{N_{NS}} e^{-\frac{\Delta\epsilon}{k_BT }}&gt;&gt;&gt;\frac {R^0}{R}$. With this, and knowing that there is no effect on the ribosomal protein composition with temperature changes [5], we conclude that the most signifficant factor should be the elongation rate.
</P>
Therefore, 
$$ 
\begin{equation}
\frac{r_P}{r^0_P}  = \frac{1+\frac{r^0_E}{r^0_I}}{\frac{r^0_E}{r_E}+\frac{r^0_E}{r^0_I}},
\end{equation}
$$
where $r_E$ ($r_I$) is the elongation (initiation) rate and the superscript denotes our reference case.
In <EM>E. coli</EM>, most of the ribosomes are translating (4 active every one free) [6], thus $\frac{r^0_E}{r^0_I} \simeq\frac{1}{4}$ and 
$$
\begin{equation}
\frac{r_P}{r^0_P}  = \frac{\frac{5}{4}}{\frac{r^0_E}{r_E}+\frac{1}{4}}.
\end{equation}
$$
<P>
We discuss next the dependence of elongation rates with temperature. We use the relation of amino acid concentration with
temperature [7]. Using data from [8], we know that small variations on the ternary complex does not affects the elongation rate. The slowests steps are: 
1) GTP hydrolysis on ternary complex (0.01 s), 2) dipeptide bond formation (0.03s) and 3) translocation (0.022s). 
$$ 
\begin{equation}
r_E  = \frac{1}{\frac{1}{A_1}e^{\frac{E^1_a}{RT}}+\frac{1}{A_2}e^{\frac{E^2_a}{RT}}+\frac{1}{A_3}e^{\frac{E^3_a}{RT}}},
\end{equation}
$$
where $E^i_a$ is (32, 75.2, 29.260) kJ/mol and $A^{-1}_i$ is (2.5 $\cdot 10^7$, 1.6 $\cdot 10^{14}$, 3.9 $\cdot 10^6$).  
</P><H3><SPAN class="mw-headline" id="Summary"> Summary </SPAN></H3><P>
Our model includes two main contributions to the changes of the enzymatic activity due to temperature variation: transcription and synthesis rates. In the case of the GFP, 
the folding free energy term is small and the relative enzymatic activity can be simplified to our master equation for stability
$$
\begin{equation}
\frac{Ac}{Ac^0}  = \frac{r_T}{r^0_T} \frac{r_P}{r^0_P} .
\end{equation}
$$
</P><H3><SPAN class="mw-headline" id="2.29_pH"> 2) pH </SPAN></H3><P>
Contrary to the temperature situations, internal pH hardly varies in <EM>E. coli</EM> when the external pH does. Therefore, we focus on the intrinsic <EM>E. coli</EM> element that may be affected by this type of stress, subsequently influencing Biobrick expression. The main factor is the number of free RNAP, which will be reduced due to the synthesis of stress response genes. We estimate the ratio of free RNAP under different conditions by the ratio of their growth rates
$$
\begin{equation}
\frac{r_T}{r^0_T}  = \frac{Prob_B}{Prob^0_B}  = \frac { 1+\frac{R^0}{N_{NS}} e^{-\frac{\Delta\epsilon}{k_BT }} } { \frac {R^0}{R}+\frac{R^0}{N_{NS}} e^{-\frac{\Delta\epsilon}{k_BT }}} ,
\end{equation}
$$
with
$$ 
\begin{equation}
\frac{R^0}{R}  = \frac{growth^0} {growth} .
\end{equation}
$$


</P><H3><SPAN class="mw-headline" id="Comparison_with_wet-lab_data"> Comparison with wet-lab data </SPAN></H3><P><B><A href="Results#StabilitySection">Our model captures the shape of the observed temperature profile and predicts that additional variations at lower temperatures are independent of the chosen Biobrick</A></B>. We have found that transcription rate decreases with temperature as a consequence of the reduction of free RNAP; and synthesis rate grows above the optimal temperature. Their combination leads to the observed variation of fluorescence with temperature. Unfortunately, this is not the whole story. Our model fails to fit the amount of variation, which is larger experimentally than the predicted one. By comparing our theoretical predictions and experimental results of XL1 Blue strain transformed with Bb1 or Bb3, we find that the ratio between theoretical prediction and experimental results is the same at under-optimal temperatures (in fact, accidentally too close given the error bars). This fact suggests that the temperature dependence not included in our model should come from cell mechanisms; independent from Biobrick parameters (promoter strength, ...). On the contrary, the comparison of theoretical predictions and experimental results at higher temperatures show that our model might require more dynamics which depends on Biobrick parameters. 
</P><P><B><A href="Results#StabilitySection">Our model captures the shape of the observed pH profile</A></B>. We have found that the lowest fluorescence happens at the pH with lowest growth, while it is mildly dependent on the pH from 6 to 8.
</P></DIV><DIV class="tab-pane fade" id="StandardizationTab"><H2>Standardization</H2><P>

We will now discuss Biobrick expression in different <EM>E. coli</EM> strains. The effect of the changes in the number of free RNAP on mRNA synthesis rate depend on the energy of the promoter. The relative transcription rate in different strains can be written as
$$
\begin{equation}
\frac{r_T}{r'_T}  = \frac{Prob_B}{Prob'_B}  = \frac { 1+\frac{R'}{N_{NS}} e^{-\frac{\Delta\epsilon}{k_BT }} } { \frac {R'}{R}+\frac{R'}{N_{NS}} e^{-\frac{\Delta\epsilon}{k_BT }}} ,
\end{equation}
$$
where we can see that the dominance of the second term over the first one, controlled by the promoter strength, would lead to similar rate independently on the particular strain chosen. Comparing this formula with the relative transcription rate in stability, we find that promoters whose  binding energy makes them more standard also implies more stable promoters. Strong promoter sequences experience a decrease in synthesis rates when subjected to an increase of promoter energy. We use these results and the energy matrix derived in [3] to create an application that  is able to compute promoter sequences for a given relative (to the maximum) expression rate and associated standard and stable rate. The physical parameters behind are  the promoter strength (using the data of Anderson’ promoters) and the binding energy and the tool create a sequence of a constitutive promoter with the appropriated trade-off between strength and energy, therefore, with the appropriated trade-off between stability &amp; standardization (ST$^2$)and rate of expression. We define the ST$^2$ index as $$ F_{st2} = (F - 0.012)/0.51 $$ with $$F = \frac{R'}{N_{NS}} e^{-\frac{\Delta\epsilon}{k_BT }}, $$ where $F_{sty}$ has been scaled to run from 1 to 0. We have used a fitting formula for the rate of expression relative to the maximum rate [3] as a function of the binding energy $\Delta\epsilon$,  $$ r_T = 2.29 + 0.27 \frac{\Delta\epsilon}{kT}. $$  With this approach, we can infer the ST$^2$ level and the promoter sequence for a given expression rate. We show below our app in javascript. 

</P><DIV class="col-sm-9 center-block" style="padding:20px;"><P><B>Generate a promoter sequence and compute the ST$^2$ (Stability &amp; Standardization) index for a given expression rate</B></P><P>Move the cursor to a given expression rate (as a fraction of its maximum) and press Generate a chain</P><P style="text-align:center"><SPAN class="label label-success" id="percent1">25%</SPAN><BUTTON type="button" name="button" id="button1" data-loading-text="Generating..." class="btn btn-default">Generate a chain</BUTTON></P><P class="text-centered">Promoter sequence (from -41 to -1 nt) is:</P><P class="text-centered">ST$^2$ index:
                    </P></DIV><H3><SPAN class="mw-headline" id="Comparison_with_wet-lab_data_2"> Comparison with wet-lab data </SPAN></H3><P><B><A href="Results#StandardizationSection">Our wet-lab results show that Biobrick 5 is the most standard one.</A></B>Our model predicts that systems with less modules or factors will be more stable and standard. Biobrick 5 has a tetracycline depending promoter, whose limitating step is the binding of tetracycline to TetR. In this scenario, the intracellular concentration of aTc and the binding to the TetR repressor do not depend on the strain, so there is only one factor in which the strain could have an important effect: the TetR intracellular concentration, which has a constitutive expression. Analyzing the other Biobricks, we found that the strains with less expression of the constitutive promoter are those ones with higher expression when we use the Ptet promoter (but JM109 and BB1), which agree with our prediction.


        </P></DIV><DIV class="tab-pane fade" id="OrthogonalityTab"><DIV class="col-sm-9 pull-right"><H2>Orthogonality</H2><P>
Plasmid dynamics have been studied exhaustively over these years using mathematical models: from how plasmid growth is regulated to plasmid partitioning after cell division. However, little is known about how two identical plasmids interact with each other. This part of our modelling project aims to shed light upon this issue using previous models of ColE1 plasmids and the properties of two different plasmids with same origin of replication when these are introduced in a cell. We employed both deterministic and stochastic models to try to answer whether two plasmids behave independently, and developed some approximations to highlight important features of the system.

</P><H3>
Deterministic model
</H3><P>
This model is taken from [16]. Here, plasmid levels are regulated both by an RNA molecule that inhibits plasmid replication and by the dilution of plasmids as a result of cell growth and division:

$$ \frac{d P}{dt} = r \cdot P \cdot R(s) - \frac{\log 2}{\tau} \cdot P, $$

$$ \frac{d s}{dt} = \alpha \cdot P - \beta \cdot s, $$

where $P$ is the number of plasmids, $s$ is the number of RNA inhibitors, $r$ is the plasmid growth rate, $\tau$ is the cell growth rate, $\alpha$ is the RNA inhibitor synthesis rate
, $\beta$ is the RNA inhibitor decay rate and $R(s)$ is the repression effect of RNA inhibitor with 
$$ R(s) = \frac{1}{(1+\frac{s}{s_0})^n},$$
where $s_0$ is the value of RNA inhibitor necessary to repress half of the replication events and $n$ is the cooperativity. Plasmid levels are regulated both by a negative feedback and by cell growth. If we consider several plasmids, we get

$$ \frac{d P_i}{dt} = r_i \cdot P_i \cdot R(s) - \frac{\log 2}{\tau} \cdot P_i,$$

$$ \frac{d s}{dt} = \alpha \cdot \sum \limits_{i=1}^n{P_i} - \beta \cdot s,$$

with a steady state solution

$$ s_{ss} = R^{-1}(\frac{\log 2}{\tau \cdot r_i}),$$

$$ \sum \limits_{i=1}^n{P_{i,ss}} = (\frac{ \beta \cdot s_{ss}}{\alpha}).$$

In a simplified way, with two identical plasmids, we get:
$$ P_{1,ss} + P_{2,ss} = K.$$


It is noticeable that plasmid levels are not independent. Small perturbations can increase the relative level of one plasmid, favoring its growth. As they are repressed as a function of the total number of plasmids but they grow depending on their number, the system is prone to introduce asymmetries

$$ \frac{d P_1}{d P_2} = \frac{r_1 \cdot P_1}{r_2 \cdot P_2} .$$


If growth rates are constant, the slope is determined by the initial conditions
$$ m = \frac{P_{01}}{P_{02}} .$$

</P><H4>Quasi-steady state approach</H4>
To simplify our equations, we assume inhibitor dynamics are faster than plasmid growth, so RNA levels are a direct representation of plasmid levels

$$ \frac{d P_i}{dt} = r_i \cdot P_i \cdot R(P_i) - \frac{\log 2}{\tau} \cdot P_i .$$

<H4>Dependence on initial conditions:</H4>
To test our hypothesis, we integrate our differential equations using numerical methods with several initial conditions. As expected, small differences in initial proportions are amplified by the dynamics.

<DIV class="thumbnail col-sm-11 center-block"><DIV class="caption"><STRONG>Figure 1. Dependence on initial conditions</STRONG> A) Steady state distribution of plasmids from different initial conditions. As seen, small differences in initial conditions are amplified. B) Phase space map of plasmid growth. 
  </DIV></DIV><H3>
Stochastic model
</H3>
The deterministic model assumes negligible noise. However, this might not be the case and small variations in plasmid levels could be highly amplified. Hence, we now discuss the importance of noise in plasmid growth. We used the <B>Stochastic Simulation Algorithm (SSA)</B> developed by Gillespie (see [13]). In short, SSA simulates by iteration which processes occur at any time using a random number generator.

First of all, we used the algorithm to develop a code assuming growth of the plasmids and cell division as independent processes. The steps in the algorithm are:<LI>Compute the propensity function of each reaction and the overall propensity function:
$$ \alpha_{P_i \rightarrow P_i+1} = r_i \cdot P_i \cdot R(P_i), $$
$$ \alpha_{0} = \sum \limits_{i=1}^{n}\alpha_{P_i \rightarrow P_{i+1}}. $$
</LI><LI>Compute when the next chemical reaction takes place :
$$ t = \frac{1}{\alpha_{0}} \cdot \log(\frac{1}{\theta_1}),$$
where $\theta_1$ is a random number from a uniform distribution U(0,1) we make use of the random distribution for exponential reactions.</LI><LI>Compute which reaction happens:
$$ \frac{1}{\alpha_{0}} \sum \limits_{i=1}^{j-1}{\alpha_{P_i \rightarrow P_i+1}} &lt; \theta_2 \leq \frac{1}{\alpha_{0}} \sum \limits_{i=1}^{j}{\alpha_{P_i \rightarrow P_i+1}},$$
where $\theta_2$ is a random number given by the uniform distribution U(0,1).
</LI><LI>Plasmid partition after cell division at t = $\tau$ 
We show below the evolution of a two plasmids system for two different initial conditions: $P_1=P_2=10$ and $P_1 = 10$ and $P_2 = 50$.  <DIV class="thumbnail col-sm-11 center-block"><DIV class="caption"><STRONG>Figure 2. Stochastic simulation (solid lines) versus deterministic solution (solid + dots lines) for different initial conditions (left $P_1=P_2=10$, right $P_1 = 10$ and $P_2 = 50$)</STRONG> This figure depicts the differences between these two modeling approaches. When several cells are taken into account, the dynamics are similar to the deterministic solution. However, the intracellular levels of plasmids can vary from the deterministic solution as shown here. Each plasmid is represented by a color and the total level of plasmids of the deterministic solution is in black.
  </DIV></DIV>
As expected, the deterministic model gives a mean version of the dynamics. Single cell dynamics shows oscillatory behaviour. The levels of each plasmid can vary but their sum remains unchaged (black line)

<H4>
A lineage model
</H4>

Aiming to obtain a better description, we studied plasmid levels in a lineage from a single cell. 
As tracking bacteria and their plasmid levels is an important issue, we implemented a tree plot in our code using <A href="http://www.mathworks.co.uk/matlabcentral/fileexchange/35623-tree-data-structure-as-a-matlab-class"><B>a matlab package</B></A>. 
Plasmids were given to each daughter cell following a gaussian distribution with a standard deviation of $\frac{1}{\sqrt{n}}$

<DIV class="thumbnail col-sm-11 center-block"><DIV class="caption"><STRONG>Figure 3. Plasmid dynamics over a lineage (I)</STRONG> As cells divide, they can suffer from variations in their number of copies of a plasmid, influencing the levels of daughter cells. Left picture shows the evolution in the levels of each plasmid (in different colors) and right picture is a lineage tree to track each cell.
  </DIV></DIV>
Also we were able to track the plasmid levels of each bacteria at the time of their division.
<DIV class="thumbnail col-sm-11 center-block"><DIV class="caption"><STRONG>Figure 4. Plasmid dynamics over a lineage (II)</STRONG> Variations of plasmid copy number in each lineage can be track using a lineage tree. Each tree shows the level of its associated plasmid.
  </DIV></DIV>
The distribution at the beginning and at the end of the simulation is:
<DIV class="thumbnail col-sm-11 center-block"><DIV class="caption"><STRONG>Figure 5. Heat maps of time evolution of plasmid distribution in a lineage</STRONG> Here we show the distributions of the plasmid levels in a lineage for the first 500 simulations (left picture) and the last 500 simulations (right picture), from a single cell with the initial conditions $P_1 = P_2 = 75$. Notice the 
broadening of the distribution with the lineage growth.
  </DIV></DIV>
Comparing with a distribution of orthogonal plasmids, we find that while both are symmetric (respect to the $P_1 = P_2$ line), 
non orthogonal plasmids lead to broader heat maps. 

<DIV class="thumbnail col-sm-5 center-block"><DIV class="caption"><STRONG>Figure 6. Plasmid distribution in a system with no influence between different plasmids</STRONG> When each plasmid represses itself, the distribution gets narrower and the time-dependent evolution is lost.
  </DIV></DIV><H4>
A step beyond: Resistance
</H4>
As a proof of concept, we modelled the bacterial time of division as a function of both plasmids, being minimal when both plasmid levels are similar. For this simple regulation, we obtained that the average level of plasmids in a population is the same for each DNA molecule. 
<DIV class="thumbnail col-sm-11 center-block"><DIV class="caption"><STRONG>Figure 7. Plasmid dynamics over a lineage in presence of resistance and plasmid-dependent growth </STRONG> Introduction of a new feature to the system, a plasmid-dependent growth, cells shows loss of synchrony in growth, favouring cells keeping similar levels of both plasmids. Left picture shows the plasmid copy number over time and right picture shows the lineage tree as a chronogram.
</DIV></DIV>
This way to tune the plasmid levels is unable to make them equal, as depicted in the following picture. Also, there are variations in their growth rates.
<DIV class="thumbnail col-sm-11 center-block"><DIV class="caption"><STRONG>Figure 8. Resistance-induced plasmid levels modulation (left) and doubling cell time (right)</STRONG> Resistance is able to equal the levels of both plasmids, even there was a difference in initial conditions, although there may be a small difference. This is achieved by decreasing the doubling time of cells with similar levels of each plasmid, obtaining a distribution in these times as shown in the right picture.
</DIV></DIV><H4>
Approaches for less computational expensive
</H4>
Although these simulations are powerful and helpful and yield informative results, they are computationally expensive. This could become an important pitfall when dealing with a huge number of cells or long times. We tried to bypass this problem by using an approach if we are working with populations as explained in [16].  We added a degradation process for each plasmid following the equations: 
$$ \alpha_{P_i \rightarrow P_i-1} = \frac{log(2)}{\tau} \cdot P_i$$
This approach can be used to study cell dynamics in a population. With the system of equations, we capture the global dynamics for a cell population. 
If there is no resistance, we find dynamical examples where a plasmid is lost. 
<DIV class="thumbnail col-sm-5 center-block"><DIV class="caption"><STRONG>Figure 9. Plasmid levels in cell lineages without resistance modulation using a less computational approach.</STRONG> Simulation of all features are time-consuming and computation expensive. However, simpler approaches can lead to similar results. In this particular example, the drift of a plasmid is shown.
  </DIV></DIV><H3><SPAN class="mw-headline" id="Comparison_with_wet-lab_data_3"> Comparison with wet-lab data </SPAN></H3><P>
The interaction between two plasmids in the same cell is still an uncharted territory. A relevant example is shown in [15], where the authors obtained an emergent behaviour as result of an unexpected relation between the bacterial physiology and plasmid dynamics. Using models in single plasmid dynamics, we switch between deterministic and stochastic models to learn about the unspecific interaction between two plasmids in a cell. <A href="Results#OrthogonalitySection"><B>There is an important variability, which drives the system away from orthogonality. Plasmids with identical origin of replication are not orthogonal</B><I>per se</I></A>. Although we can introduce different resistances to tune the system, it 
is hard to stabilize the system due to the large variability. Plasmids with different mechanisms of replication might solve it but it is not guaranteed. Certainly, new plasmids must be designed to make engineered cells more reliable and predictable.
</P></LI></DIV></DIV><DIV class="tab-pane fade" id="OpenLicenseTab"><H2>Open License</H2><P>
We model the susceptibility to patent a human creation by 
$$
P.I. = M \cdot N \cdot A ^{log(\frac{A+I}{2})}, 
$$
where P.I. is the patentability index, M is the morality, which is a non-legal term accepted in the E.U., and N refers to novelty. Both factors are described by a discrete variable which can be valued 0 or 1 depending on the results of a questionnaire. A is the inventive step and I is the industrial application which can be valued from 1 to 10 depending of the 
results of a test. 
</P><P>
The Inventive Step is the most relevant variable but is regulated by the industrial application. The highest inventive step can be reduced by almost a factor 2 with the lowest industrial application. On the contrary, the lowest inventive step is not affected by the industrial application. After computing all the discussed factors, the patentability index can run from 0 to 10. The reader can find a detailed discussion of the questionnaires, tests, analysis of the index and references in the <B><A href="HumanPractices"><STRONG>Open License section</STRONG></A></B>.
</P></DIV><DIV class="tab-pane fade" id="OtherTab"><H2>Applications to other projects</H2><P>
    In our project, we have developed a tool that allows us to predict the stability of <EM>E. coli </EM>carrying a reporter gene (in particular, the stability when subjected to different temperatures). The variation of activity with temperature was described by:
</P>
$$
\begin{equation}
\frac{A_{c}}{A^0_{c}}  = \prod_i \frac{V_i}{V^0_i} ,
\end{equation}
$$
 where $V_i$ is a temperature dependent factor, and A<SUB>c</SUB><SUP>0</SUP> is the activity in reference conditions (in this case, 37 ºC).
<P>
 We found (<A href="#StabilityTab">see Stability</A>):
$$
\begin{equation}
\frac{Ac}{Ac^0}  = \frac{N_Pr}{N^0_Pr} =\frac{r_T}{r^0_T} \frac{r_P}{r^0_P},
\end{equation}
$$
with
$$
\begin{equation}
\frac{r_T}{r^0_T}  = \frac{Prob_B}{Prob^0_B}  = \frac { 1+\frac{R^0}{N_{NS}} e^{-\frac{\Delta\epsilon}{k_BT }} } { \frac {R^0}{R}+\frac{R^0}{N_{NS}} e^{-\frac{\Delta\epsilon}{k_BT }}},
\end{equation}
$$
where
$$
\begin{equation}
\frac{R^0}{R}  = \frac {0.027 (T^0)^2 - 2.12 T^0 + 42.2} {0.027 T^2 - 2.12 T + 42.2} ,
\end{equation}
$$
and with $$
\begin{equation}
\frac{r_P}{r^0_P}  = \frac{\frac{5}{4}}{\frac{r^0_E}{r_E}+\frac{1}{4}}, 
\end{equation}
$$
where
$$
\begin{equation}
r_E  = \frac{1}{\frac{1}{A_1}e^{\frac{E^1_a}{RT}}+\frac{1}{A_2}e^{\frac{E^2_a}{RT}}+\frac{1}{A_3}e^{\frac{E^3_a}{RT}}}.
\end{equation}
$$
In more complex systems, one expects more temperature dependent factors, and varying more significantly. For an enzyme, the activity is:
$$
\begin{equation}
A_{c}  = k  e^{-\frac{E_a}{RT } }  \frac{N_{Pr}}{1+e^{\frac{\Delta G_F}{RT } } } .
\end{equation}
$$
 where N<SUB>Pr</SUB> is the number of total proteins, ΔG is the free energy of the folding process, k is the Arrhenius constant of the chemical reaction
    and E<SUB>a</SUB> is the activation energy of the catalyzed reaction.
</P><P>
 A crude estimation can be obtained by assuming: a) all genes are controlled by the same promoter, b) same folding energy for all proteins, and c) same activation energy 
for all reactions. The activation energy for each reaction (mean activation energy is from [1]) is:
$$
\begin{equation}
\frac{A_{c}}{A^0_{c}}  = {(\frac{r_P}{r^0_P} )^n}{({\frac{r_P}{r^0_P}}{\frac{\frac{1}{1+e^{\frac{\Delta G_F}{RT } } }}{\frac{1}{1+e^{\frac{\Delta G_F}{RT^0 } } }}}{\frac{e^{-\frac{E_a}{RT } }}{e^{-\frac{E_a}{RT^0 } }}})^m},
\end{equation}
$$
where n is the number of different mRNAs, m is the number of proteins, E$_ a$ is the activation energy of the catalyzed reaction and $\Delta$G is the protein folding energy. Using:
$$
\begin{equation}
\Delta G_F (L,T)  = \Delta H(L) + \Delta C_p (L)(T-T_h) - T\Delta S(L) - \Delta C_p (L)T\ln\frac{T}{T_S} \nonumber,
\end{equation}
$$
with
$$
\begin{eqnarray}
\Delta H(L) &amp;=&amp; -5.03 L - 41.6 (kJ/mol)\nonumber, \\
\Delta C_p(L) &amp;=&amp; -0.062 L + 0.53 (kJ/mol)\nonumber, \\
\Delta S(L) &amp;=&amp; -16.8 L -85 (J/mol)\nonumber.
\end{eqnarray}
$$
and T<SUB>h</SUB> = 373.5 K, T<SUB>s</SUB> = 385 K and L = 300 aa is the <EM>E. coli</EM> mean protein length is 300 aa, we computed the 
relative activity and relative protein number for n mRNAs and m proteins. The results are shown below: 
</P><DIV class="thumbnail col-sm-7 center-block"><DIV class="caption"><STRONG>Figure 1:</STRONG>
    Variation of activity with temperature for a system that has 1, 2, 3, 4 and 5 genes.
  </DIV></DIV><DIV class="thumbnail col-sm-7 center-block"><DIV class="caption"><STRONG>Figure 2:</STRONG>
    Variation of synthetized protein with temperature for a system that has 1, 2, 3, 4 and 5 genes.
  </DIV></DIV><DIV class="thumbnail col-sm-7 center-block"><DIV class="caption"><STRONG>Figure 3:</STRONG>
    Variation of activity with temperature for a system that has 1, 2, 3, 4 and 5 enzymes, but only one mRNA.
  </DIV></DIV><H3><DIV class="thumbnail col-sm-7 center-block"><DIV class="caption"><STRONG>Figure 4:</STRONG>
    Variation of activity with temperature of a system that has 5 proteins,
using 1, 2, 3, 4 and 5 mRNA.
  </DIV></DIV></H3><H3><SPAN class="mw-headline" id="Summary_2"> Summary </SPAN></H3><P>
    - Stability decreases with increasing modules number. In case of many modules, stability is larger when one of the modules is the limiting step.
</P><P>
    - Stability grows with decreasing number of mRNA at lower temperatures.  On the other hand, higher temperatures and lower number of mRNA show higher activity.
</P><H3><SPAN class="mw-headline" id="Application_to_other_iGEM_projects"> Application to other iGEM projects </SPAN></H3><P><A href="https://2010.igem.org/Team:Slovenia/ABOUT_US" target="blank">Team Slovenia 2010 (DNA CODING BEYOND TRIPLETS)</A>
developed a system to use a DNA sequence as a scaffold for enzymes, in order to optimize a metabolic pathway (<A href="https://2010.igem.org/Team:Slovenia" target="blank"> TS10 </A>). We apply our model as follows:
</P><P>
$$
\begin{equation}
\frac{A_{c}}{A^0_{c}}  = {(\frac{r_P}{r^0_P} )^n}{({\frac{r_P}{r^0_P}}{\frac{\frac{1}{1+e^{\frac{\Delta G_F}{RT } } }}{\frac{1}{1+e^{\frac{\Delta G_F}{RT^0 } } }}}{\frac{e^{-\frac{E_a}{RT } }}{e^{-\frac{E_a}{RT^0 } }}})^m}{({{\frac{\frac{1}{1+e^{\frac{\Delta G_Z}{RT } } }}{\frac{1}{1+e^{\frac{\Delta G_Z}{RT^0 } } }}}}{{\frac{\frac{1}{1+e^{\frac{\Delta G_I}{RT } } }}{\frac{1}{1+e^{\frac{\Delta G_I}{RT^0 } } }}}})^z}{{\frac{\frac{1}{1+e^{\frac{\Delta G_D}{RT } } }}{\frac{1}{1+e^{\frac{\Delta G_D}{RT^0 } } }}}}
\end{equation}
$$
where n is the number of required transcripts, m is the number of required translated proteins, z is the number of required zinc-finger—DNA interactions,  E$_a$ is the activation energy, $\Delta$G$_F$ is the folding energy of any enzyme required for the pathway, $\Delta$G$_Z$ is the  folding energy of every zinc-finger domain [17], $\Delta$G$_I$ is the interaction energy of a normal zinc-finger-DNA interaction [17] and $\Delta$G$_D$ is the DNA melting energy.
</P><P>
The goal was to look for a more efficient synthesis of violacein. Using the mean activation energy (50 kJ/mol) for any catalyzed reaction [1] we get 
</P><DIV class="thumbnail col-sm-7 center-block"><DIV class="caption"><STRONG>Figure 5:</STRONG>
    Variation of activity with temperature of a metabolic pathway with 1, 2, 3, 4 and 5 enzymes.
  </DIV></DIV>
We conclude that the new modules (zinc-finger and DNA) do not significantly affect the stability.
<P><A href="https://2013.igem.org/Team:Heidelberg/Team" target="blank">Team Heidelberg 2013 (PHILOSOPHER’S STONE)</A>
developed a system to synthesize non ribosomal peptides. One of the simplest mechanisms was the synthesis of indigoidine (<A href="https://2013.igem.org/Team:Heidelberg" target="blank">TH13 wiki</A>) :
</P>
    For this synthesis, we need: 1) acquisition of Glu (assuming that the bacteria acquires from the external medium with a specific transporter), 2) transform Glu in cGlu and 3) transform cGlu in Ind (assuming that there ir one lower step). We get:
$$
\begin{equation}
\frac{A_{c}}{A^0_{c}}  = {(\frac{r_P}{r^0_P} )^3}{(\frac{r_P}{r^0_P})^3}{({\frac{\frac{1}{1+e^{\frac{\Delta G_F}{RT } } }}{\frac{1}{1+e^{\frac{\Delta G_F}{RT^0 } } }}}{\frac{e^{-\frac{E_a}{RT } }}{e^{-\frac{E_a}{RT^0 } }}})^2}{\frac{\frac{1}{1+e^{\frac{\Delta G_T}{RT } } }}{\frac{1}{1+e^{\frac{\Delta G_T}{RT^0 } } }}}{\frac{e^{-\frac{E_aT}{RT } }}{e^{-\frac{E_aT}{RT^0 } }}}
\end{equation}
$$
where E$_a$ is the activation energy of every chemical reaction, E$_{aT}$ is the activation energy of the transport process, $\Delta$G$_F$ is the folding energy of any enzyme required for the pathway and $\Delta$G$_T$ is the folding energy of the transporter.
<P>
We use the mean activation energy (50 kJ/mol) for any catalyzed reaction [1] and the activation energy for the Lysine transport through membranes derived from [18], 
E$_{aT}$=33.244 kJ/mol, we get 
</P><DIV class="thumbnail col-sm-7 center-block"><DIV class="caption"><STRONG>Figure 6:</STRONG>
    Variation of the Ind production with temperature compared to the variation of activity of the systems composed by 2 or 3 enzymes.
  </DIV></DIV><P>
In this case, we observe deviations to the expectation: the optimum temperature and the maximum value of Ac/Ac$^0$ are lower than in the n,m = 3 case. This implies a significant contribution of the Glu transport module.
</P></DIV><DIV class="tab-pane fade" id="ReferencesTab"><H2>References</H2><OL><LI>
    Ghosh K., Dill K. (2010). Cellular proteomes have broad distributions of protein stability. Biophys J <STRONG>99</STRONG>: 3996–4002.
</LI><LI>
Soini, J. <EM>et al.</EM> (2005) Transient increase of ATP as a response to temperature up-shift in Escherichia coli. Microb. Cell Fact.    <STRONG>4</STRONG>, 9.
</LI><LI>
Brewster R.C., Jones D.L., Phillips R.(2012) Tuning promoter strength through RNA polymerase binding site design in Escherichia coli. PLoS Comput Biol    <STRONG>8</STRONG>: e1002811.
</LI><LI>
Skelly S. <EM>et al.</EM> (1987) Correlation between the 32-kDa s factor levels and in vitro expression of Escherichia coli heat shock genes. Proc.
    Natl. Acad. Sci. USA <STRONG>84</STRONG>:8365–8369.
</LI><LI>
Zaritsky, A. (1982). Effects of growth temperature on ribosomes and other physiological properties of Escherichia coli. J. Bacteriol.    <STRONG>151</STRONG>:485-486.
</LI><LI>
Bremer, H., Dennis, P. P. (1996) Modulation of chemical composition and other parameters of the cell by growth rate. Neidhardt, et al. eds. Escherichia coli and Salmonella typhimurium: Cellular and Molecular Biology, 2nd ed. chapter 97.
</LI><LI>
Piperno J.R., Oxender D.L. (1968) Amino acid transport systems in Escherichia coli K-12. J Biol Chem. <STRONG>243</STRONG>(22):5914–5920.
</LI><LI>
Thompson R.C., Dix D.B., Karim A.M. (1986) The reactions of ribosomes with elongation factor Tu·GTP complexes. Aminoacyl-tRNA-independent reactions in the elongation cycle determine the accuracy of protein synthesis. J Biol Chem, <STRONG>261</STRONG>:4868–4874.
</LI><LI>
Bilgin N.<EM> et al.</EM> (1992) Kinetic properties of Escherichia coli ribosomes with altered forms of S12. J Mol Biol. <STRONG>224</STRONG>
    (4):1011-27.
</LI><LI>
Grigorenko B.L.<EM> et al.</EM> (2008) Mechanism of the chemical step for the guanosine triphosphate (GTP) hydrolysis catalyzed by elongation factor Tu. Biochim Biophys Acta <STRONG>1784</STRONG>:1908–1917.</LI><LI>
Gindulyte,A. <EM>et al.</EM> (2006) The transition state for formation of the peptide bond in the ribosome. Proc. Natl. Acad. Sci. U. S. A.,  <STRONG>103</STRONG>:13327–13332.
</LI><LI>
Wilson,K. S., Noller, H.F. (1998) Mapping the position of translational elongation factor EF-G in the ribosome by directed hydroxyl radical probing. Cell, <STRONG>92</STRONG>: 131–139.
</LI><LI>Erban R, Chapman SJ and Maini PK (2007) <I>A PRACTICAL GUIDE TO STOCHASTIC SIMULATIONS OF REACTION-DIFFUSION PROCESSES </I><A href="https://people.maths.ox.ac.uk/erban/Education/StochReacDiff.pdf"><B>link</B></A></LI><LI>Klumpp E (2011)<I> Growth-Rate Dependence Reveals Design Principles of Plasmid Copy Number Control</I> PLoS One Vol 6 Issue 5 <A href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0020403"><B>link</B></A></LI><LI>Marguet P, Tanouchi Y, Splitz E, Smith C and You L (2010) <I>Oscillations by Minimal Bacterial Suicide Circuits Reveal Hidden Facets of Host-Circuit Phisology </I> PLoS One Vol 5 Issue 7 <A href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0011909"><B>link</B></A></LI><LI>Paulsson J and Ehrenberg M (2001) <I>Noise in a minimal regulatory network: plasmid copy number control</I> Cambridge Press <A href="http://journals.cambridge.org/action/displayFulltext?type=1&amp;fid=75292&amp;jid=QRB&amp;volumeId=34&amp;issueId=01&amp;aid=75291"><B>link</B></A></LI><LI>Shiro I and Kuldell N (2007) <I>Zinc Finger proteins: from atomic contact to cellular function</I>. Landes Bioscience. <A href="http://www.landesbioscience.com/iu/Iuchi_9781587066429.pdf"><B>link</B></A></LI><LI>Berlin, R.D (1973) <I>Temperature dependence of nucleoside membrane transport in rabbit alveolar macrophages and polymorphonuclear leukocytes</I>. J. Biol. Chem., 248:4724-4730. <A href="http://www.ncbi.nlm.nih.gov/pubmed/4718740"><B>link</B></A></LI></OL></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Valencia_Biocampus/Modeling">http://2014.igem.org/Team:Valencia_Biocampus/Modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Valencia_Biocampus/Modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Valencia_Biocampus/Modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Valencia_Biocampus/Modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Valencia_Biocampus/Modeling&amp;oldid=369026" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>