<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UChicago_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UChicago/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UChicago/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UChicago/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UChicago/Project&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UChicago/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UChicago/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="wrapper"><NAV><UL><LI class="navHome"><A id="top" href="/Team:UChicago">UCHICAGO</A></LI><LI class="navBar">
                    |
                </LI><LI class="navIgem"><A href="https://2013.igem.org">iGEM 2013</A></LI><LI class="navTeam"><A href="/Team:UChicago/Team">Team</A></LI><LI class="navProject"><A href="/Team:UChicago/Project">Project</A></LI><LI class="navNotebook"><A href="/Team:UChicago/Notebook">Notebook</A></LI><LI class="navSafety"><A href="/Team:UChicago/Safety">Safety</A></LI><LI class="navSponsor"><A href="/Team:UChicago/Sponsor_Us">Sponsor Us!</A></LI></UL></NAV><DIV class="whiteBox"><DIV class="subContent"><DIV id="" class="subNav"><UL><LI class=""><A href="/Team:UChicago/Parts">See Our Dream Parts</A></LI><LI class=""><A href="/Team:UChicago/Plan">Our Summer Plan</A></LI></UL></DIV><H2><SPAN class="mw-headline" id="Project_Overview"><B>Project Overview</B></SPAN></H2><P>Each year, over two billion pounds of feather waste is produced as a byproduct of commercial poultry processing. Most of this waste is chemically treated or steam pressure-cooked into nutrient-poor animal feed (1). Nonetheless, recent research has shown that feather keratin can be used to produce biodegradable plastics (2), fertilizers (3), detergents (4), and pharmaceuticals (5). However, traditional methods of keratin degradation rely on chemicals such as sulfides and limes which are energetically costly and pose serious environmental and health concerns. Therefore, researchers have turned to feather-degrading bacteria as an alternative to traditional chemical methods (6). Although keratin-degrading bacterial strains were isolated from poultry waste nearly two decades ago and some keratin-degrading proteases (keratinases) have been identified, keratinase production in heterologous hosts has been stymied by poor protease expression. By developing an efficient process for the digestion of keratin, we would not only drastically reduce the cost of disposal but also allow the keratin to be used in products like animal feed, fertilizers and biodegradable plastics. Our final product would be a one-step continuous process in which feather waste could be continually added to <I>B. subtilis</I> cultures.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:750px;"><DIV class="thumbcaption">A simple schematic of our project</DIV></DIV></DIV><P>
In order to improve keratinase expression in Bacillus subtilis, a host suited to industrial use, we decided to build a BioBrick for the best characterized and most efficient keratinase gene. The keratin-degrading bacterial strain that is best characterized in the literature is Bacillus licheniformis PWD-1. The keratinase, called KerA, responsible for this bacterium’s activity was isolated and characterized by Shih et al. along with its signal peptide and promoter (7). The DNA sequence of kerA is 1458 bases long and codes for a serine protease. Like many secreted proteases in the Bacillus genus, KerA has a signal peptide, which is cleaved before KerA crosses the single plasma membrane of B. licheniformis. Additionally, the kerA promoter is activated when keratin is the sole source of carbon and nitrogen. Shih et al. successfully transformed B. subtilis with kerA and expressed it with its own promoter as well as with P43, a constitutive promoter, and a combination of the two in a high copy number backbone. All transformants expressed keratinase activity in the presence and absence of keratin (8).
</P><P>
Building on the success of Shih et al., our team designed a BioBrick to improve kerA expression in a heterologous host. We purchased two gBlocks from IDT to assemble the kerA DNA sequence, which included the signal peptide. In order to increase the level of KerA produced in B. subtilis, we planned to use BioBrick part BBa_K143053, which includes Pveg, a constitutive promoter. Moreover, a high copy number backbone with an origin of replication for B. subtilis is needed for the largest production of KerA. Since the current B. subtilis plasmids in the iGEM registry are single copy plasmids that integrate into the genome, our team designed a high copy number backbone compatible with B. subtilis based on pUB110 (9). 
</P><P>
We also wanted to determine if KerA expression in E. coli BL21-DE3 was comparable to that of B. subtilis. Unlike the members of the Bacillus genus, E. coli has two membranes. The kerA signal peptide could allow transport across the inner membrane but not the outer membrane, trapping KerA in the periplasm, where the amino acid modifications could render the keratinase inactive. Moreover, it is possible that E. coli does not contain the cellular machinery necessary to cleave the kerA signal peptide, producing an inactive form of the enzyme. Thus, we designed an alternate kerA BioBrick that was missing the signal peptide. To optimize expression of the keratinase in E. coli, we would have used a constitutive E. coli promoter BioBrick and a high copy number backbone.
</P><H2><SPAN class="mw-headline" id="Sources"> Sources </SPAN></H2><P>(1) El-Nagar, K., Saleh, S. M., &amp; Ramadan, A. R. (2006). <A href="http://www.tx.ncsu.edu/jtatm/volume5issue2/articles/El-Nagar/El_nagar_Full_171-05.pdf" class="external text" rel="nofollow">Utilization of feather waste to improve the properties of the Egyptian cotton fabrics.</A> Jour. of Textile and Apparel Technology and Management, 2, 1-12.
</P><P>
(2) Jin, E., Reddy, N., Zhu, Z., &amp; Yang, Y. (2011). <A href="http://pubs.acs.org/doi/abs/10.1021/jf1039519" class="external text" rel="nofollow">Graft polymerization of native chicken feathers for thermoplastic applications.</A> Journal of agricultural and food chemistry, 59(5), 1729-1738.
</P><P>
(3) Hadas, A., &amp; Kautsky, L. (1994). <A href="http://link.springer.com/article/10.1007%2FBF00748776" class="external text" rel="nofollow">Feather meal, a semi-slow-release nitrogen fertilizer for organic farming.</A> Fertilizer research, 38(2), 165-170.
</P><P>
(4) Riffel, A., Lucas, F., Heeb, P., &amp; Brandelli, A. (2003). <A href="http://www.ncbi.nlm.nih.gov/pubmed/12677362" class="external text" rel="nofollow">Characterization of a new keratinolytic bacterium that completely degrades native feather keratin.</A> Archives of Microbiology, 179(4), 258-265.
</P><P>
(5) Tiwary, E., &amp; Gupta, R. (2010). <A href="http://www.ncbi.nlm.nih.gov/pubmed/20597544" class="external text" rel="nofollow">Extracellular expression of keratinase from Bacillus licheniformis ER-15 in Escherichia coli.</A> Journal of agricultural and food chemistry, 58(14), 8380-8385.
</P><P>
(6) Williams, C. M., Richter, C. S., Mackenzie, J. M., &amp; Shih, J. C. (1990). <A href="http://aem.asm.org/content/56/6/1509" class="external text" rel="nofollow">Isolation, identification, and characterization of a feather-degrading bacterium.</A> Applied and Environmental Microbiology, 56(6), 1509-1515.
</P><P>
(7) Lin, X., Kelemen, D. W., Miller, E. S., &amp; Shih, J. C. (1995). <A href="http://www.ncbi.nlm.nih.gov/pubmed/7747965" class="external text" rel="nofollow">Nucleotide sequence and expression of kerA, the gene encoding a keratinolytic protease of Bacillus licheniformis PWD-1.</A> Applied and environmental microbiology,61(4), 1469-1474.
</P><P>
(8) Lin, X., Wong, S. L., Miller, E. S., &amp; Shih, J. C. H. (1997). <A href="http://www.ncbi.nlm.nih.gov/pubmed/9366094" class="external text" rel="nofollow">Expression of the Bacillus licheniformis PWD-1 keratinase gene in B. subtilis.</A> Journal of Industrial Microbiology and Biotechnology, 19(2), 134-138.
</P><P>
(9) Boe, L. A. R. S., Gros, M. F., te Riele, H. E. I. N., Ehrlich, S. D., &amp; Gruss, A. (1989). <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC210059/" class="external text" rel="nofollow">Replication origins of single-stranded-DNA plasmid pUB110.</A> Journal of bacteriology, 171(6), 3366-3372.
</P><H2><SPAN class="mw-headline" id="Results"> Results </SPAN></H2><P>Check out <A href="/Team:UChicago/Parts">our BioBrick designs.</A></P></DIV><DIV id="The_Advisors" class="divider"><P><A href="#top">^ top</A></P></DIV><DIV class="button wrapperContent"><H3><A href="/Team:UChicago/Sponsor_Us">Sponsor Us for Next Year's Competition!</A></H3></DIV></DIV><DIV id="footer"><DIV class="footerAttribute"><P class="mediawiki">Powered by <A href="http://www.mediawiki.org">MediaWiki</A></P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:UChicago/Project">http://2013.igem.org/Team:UChicago/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UChicago/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UChicago/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UChicago/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UChicago/Project&amp;oldid=330191" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>