Warning:  mkdir(): Permission denied in /websites/2013.igem.org/wiki/includes/GlobalFunctions.php on line 2176
Warning:  wfMkdirParents: failed to mkdir "/websites/2013.igem.org/wiki/images/thumb/b/b2/Keratinase_gene_diagrams.jpg" mode 511 in /websites/2013.igem.org/wiki/includes/GlobalFunctions.php on line 2179
Team:UChicago/Plan - 2013.igem.org
"
Page
Discussion
View source
History
teams
Log in
 
Team:UChicago/Plan
From 2013.igem.org
UCHICAGO
|
iGEM 2013
Team
Project
Notebook
Safety
Sponsor Us!
Return to Project Overview
Contents
1 Summer Plan
1.1 Constructing the kerA BioBricks
1.2 Constructing the pUB110 backbone
1.3 Assembling plasmid for expression in B. subtilis
1.4 Assembling plasmid for expression in E. coli BL21-De3
1.5 Testing KerA expression and activity in B. subtilis
1.6 Testing KerA expression and activity in E. coli BL21-De3
1.7 Improve KerA activity by mutagenesis
Summer Plan
Constructing the kerA BioBricks
As we approached synthesizing our construct, we realized that it artificial synthesis the kerA gene was cheaper than obtaining the B. subtilis PWD-1 strain and isolating the gene from there. Furthermore, the natural kerA sequence contained cut sites that were incompatible with 3A assembly (EcoRI and a PstI). Therefore, we opted use the Gibson assembly method with geneblocks ordered from IDT to synthesize the kerA gene to our liking. Once synthesized, we amplified with PCR and checked the DNA fragment's length using gel electrophoresis to confirm we had the right DNA.
Once we had sufficient kerA DNA to work with, we decided it would be best to insert the BioBrick into an Amp resistant backbone. To do so, we digested the PCR product and the backbone with EcoRI and PstI. We set up a ligation and transformed the plasmid into competent E. coli DH5-alpha. Unfortunately, we were unable to get any transformants. Prior to the transformation with the plasmid containing kerA, we tested the transformation efficiency of our competent cells so we discarded the possibility that we were performing the transformations incorrectly. Therefore, we thought that maybe the problem was with the assembly, PCR, digestion, or ligation of kerA. We tried using different conditions (temperatures, incubation time, and different enzymes) to eliminate our sources of error, but we were unable to get any transformants.
We were contemplating making several variants of our construct depending on how our needs changed as the project progressed. For example, the inclusion of the secretion signal peptide depended on which compartment of the cell was best suited to the proper folding of keratinase. Another example would be the inclusion of a His-tag for detection & purification purposes—which could later be removed in the final product. Some of the kerA variants we had planned for are pictured below. The DNA sequences for the 4 variants can also be found here .
Error creating thumbnail: Unable to create destination directory
[top]
Constructing the pUB110 backbone
In order to express kerA in B. subtilis in high amounts, we need to use a high copy number plasmid. Since high copy number plasmids for B. subtilis are not available from the iGEM HQ, we decided to design our own. We derived our design from pUB110 because it has been widely used in B. subtilis
Adapted from: Boe, L. A. R. S., Gros, M. F., te Riele, H. E. I. N., Ehrlich, S. D., & Gruss, A. (1989). Replication origins of single-stranded-DNA plasmid pUB110. Journal of bacteriology, 171(6), 3366-3372.
The original pUB110 plasmid is pictured above on the left, while our proposed BioBrick version is modeled on the right. Because the pUB110 vector includes EcoRI and XbaI sites, we had to find a way to remove them before we turned the vector into a BioBrick backbone. We chose to digest pUB110 with NdeI and AflII, which would remove the Pre gene along with the EcoRI and XbaI sites from the vector. Luckily, this gene is not essential for replication in B. subtilis. We then ran the digest in a gel and did a gel extraction to isolate the part of the vector that we needed.
In order to add in the BioBrick prefix and suffix to the vector, we designed a linker that was synthesized as two separate oligos. The DNA sequence of the two oligos are as follows:
5' attcgtcttaaggaattcgcggccgcttctagagtactagtagcggccgctgcagcatatgtcatac 3'  5' gtatgacatatgctgcagcggccgctactagtactctagaagcggccgcgaattccttaagacgaat 3'
We annealed the two oligos in duplex buffer (recipe from IDT) to use the duplex as a linker. Once the oligos were annealed, the linker was digested with NdeI and AflII. We then set up an overnight ligation of the digested, gel purified pUB110 and digested pUB110 linker, which would ideally lead to the formation of a backbone compatible with BioBricks. Unfortunately, we ran out of time and we were unable to get any transformants with our modified vector. To make sure that we weren't doing anything wrong with the transformation procedure of B. subtilis, we tried to  transform the original pUB110. The transformation was successful, which led us to think that the problem may be with our modified vector. The failure of transformations could have been attributed to incomplete digestions, too little DNA retrieved from the gel extraction, or incomplete ligation. We attempted to address these potential problems by using new AflII and NdeI restriction enzymes with varying incubation times, different kits for gel extractions, and overnight ligations. Despite all our attempts, we were unable to transform B. subtilis with our modified backbone.
If our transformation of the modified pUB110 vector worked, we would have gone on to pick transformants, grow them overnight to amplify the quantity of backbone DNA, and then do a miniprep to isolate the modified pUB110 backbone. We would then send the vector for sequencing. Once we verified that the backbone sequence were correct, we would have tested expression of the backbone. Using the NEB assembly kit, we would have digested an upstream part or promoter, a RFP BioBrick, and the modified pUB110 backbone. Once the construct were assembled, we would have transformed it into B. subtilis and checked that it worked by using it to express RFP.
[top]
Assembling plasmid for expression in B. subtilis
In order to express kerA in B. subtilis, we would have used the NEB assembly kit for the following parts:
Upstream part: constitutive promoter Pveg +RBS spoVG + SacB ( BBa_K541501 ) found in a Cam resistant vector.
Downstream part: kerA variants without the signal peptide in the Amp resistant vector.
Destination plasmid: the modified pUB110 with Kan resistance.
Schematic of the BioBrick® assembly method. Source: New England Bio Labs
Assembly of our parts
Note: In order to test expression of kerA with its native signal peptide, we would have instead used the constitutive promoter Pveg + RBS spoVG ( BBa_K143053 ) as the upstream part.
[top]
Assembling plasmid for expression in E. coli BL21-De3
To test expression in E. coli, we would have used the constitutive expression cassette ( BBa_K314100 ) as the upstream part, the kerA variants without the signal peptide, and a high copy number vector with Kan resistance as a destination plasmid. If kerA expression in E. coli were toxic, we would have instead used the lac induced expression cassette ( BBa_K314103 ) as the upstream part.
[top]
Testing KerA expression and activity in B. subtilis
Ideally, we would have gone on to grow our B. subtilis kerA transformants on keratin plates (made with LB agar + keratin derived from feathers) to determine secretion and activity of KerA protein. As a negative control for non-specific protease activity we would have instead used non-transformed B. subtilis.  As a positive control, we would have also gotten B. licheniformis PWD-1 and grown it on the keratin plates. We would have used the clearing of keratin from the plate as a sign of kerA activity and secretion.
Source:
Wawrzkiewicz, K., Wolski, T., & Łobarzewski, J. (1991). Screening the keratinolytic activity of dermatophytes in vitro. Mycopathologia, 114(1), 1-8.
To be completely certain that kerA (his tagged) is being expressed in B. subtilis, we would have also done a western blot on the cell lysis as well as on the liquid medium in which the transformants were grown. The western blot of the medium would indicate whether there was KerA secretion.
In order to have a more quantitative test of KerA activity, we would have used a keratin azure assay, which demonstrates keratinase activity based on a colorimetric reading. In this assay, keratin azure is used, which is a keratin protein that is conjugated to a specific molecule such that when it is digested by a proteinase, it breaks down into a colored reaction product. The concentration of this product, and thus the activity of the proteinase, can then be deduced by measuring the change in color using a spectrophotometer.
[top]
Testing KerA expression and activity in E. coli BL21-De3
In E. coli transformants, we would have detected keratinase (his tagged) expression in the cell lysate by western blot. If expression of his tagged KerA is detected, we would have proceeded to do a keratin azure assay on the cell lysate.
[top]
Improve KerA activity by mutagenesis
Our ideal enzymatic degradation method was to grow B. subtilis directly from the feathers, so that feather waste could be continually added. To do this, KerA's activity and its secretion should occur at a similar temperature. However, optimal KerA secretion occurs between 25 to 37°C, while its highest activity is observed at 50°C. In the future, we would like to use error prone PCR to introduce mutations in kerA as a way to isolate kerA variants with increased activity at lower temperatures. We would then transform the mutated kerA gene into B. subtilis. The KerA mutants would then be tested using a keratin azure assay at temperatures ranging from 25 to 37°C to quantitatively determine keratinase activity.
[top]
Sponsor Us for Next Year's Competition!
Return to Project Overview
Powered by MediaWiki
Retrieved from " http://2013.igem.org/Team:UChicago/Plan "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
