<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UChicago_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UChicago/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UChicago/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UChicago/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UChicago/Notebook&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UChicago/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UChicago/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="wrapper"><NAV><UL><LI class="navHome"><A id="top" href="/Team:UChicago">UCHICAGO</A></LI><LI class="navBar">
                    |
                </LI><LI class="navIgem"><A href="https://2013.igem.org">iGEM 2013</A></LI><LI class="navTeam"><A href="/Team:UChicago/Team">Team</A></LI><LI class="navProject"><A href="/Team:UChicago/Project">Project</A></LI><LI class="navNotebook"><A href="/Team:UChicago/Notebook">Notebook</A></LI><LI class="navSafety"><A href="/Team:UChicago/Safety">Safety</A></LI><LI class="navSponsor"><A href="/Team:UChicago/Sponsor_Us">Sponsor Us!</A></LI></UL></NAV><DIV class="whiteBox"><DIV id="" class="subNav"><UL><LI class=""><A href="/Team:UChicago/Protocols">Go to Protocols</A></LI></UL></DIV><DIV class="subContent"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#June"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">June</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Week_3"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Week 3</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#Tuesday.2C_June_18.2C_2013"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">Tuesday, June 18, 2013</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-4"><A href="#Week_4"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Week 4</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Wednesday.2C_June_26.2C_2013"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Wednesday, June 26, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Friday.2C_June_28.2C_2013"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Friday, June 28, 2013</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#July"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">July</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Week_1"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Week 1</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Wednesday.2C_July_3.2C_2013"><SPAN class="tocnumber">2.1.1</SPAN><SPAN class="toctext">Wednesday, July 3, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Thursday.2C_July_4.2C_2013"><SPAN class="tocnumber">2.1.2</SPAN><SPAN class="toctext">Thursday, July 4, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Friday.2C_July_5.2C_2013"><SPAN class="tocnumber">2.1.3</SPAN><SPAN class="toctext">Friday, July 5, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Saturday.2C_July_6.2C_2013"><SPAN class="tocnumber">2.1.4</SPAN><SPAN class="toctext">Saturday, July 6, 2013</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Week_2"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Week 2</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Monday.2C_July_8.2C_2013"><SPAN class="tocnumber">2.2.1</SPAN><SPAN class="toctext">Monday, July 8, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Wednesday.2C_July_10.2C_2013"><SPAN class="tocnumber">2.2.2</SPAN><SPAN class="toctext">Wednesday, July 10, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#Thursday.2C_July_11.2C_2013"><SPAN class="tocnumber">2.2.3</SPAN><SPAN class="toctext">Thursday, July 11, 2013</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Week_3_2"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Week 3</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Week_4_2"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Week 4</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#Tuesday.2C_July_23.2C_2013"><SPAN class="tocnumber">2.4.1</SPAN><SPAN class="toctext">Tuesday, July 23, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-20"><A href="#Wednesday.2C_July_24.2C_2013"><SPAN class="tocnumber">2.4.2</SPAN><SPAN class="toctext">Wednesday, July 24, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-21"><A href="#Friday.2C_July_26.2C_2013"><SPAN class="tocnumber">2.4.3</SPAN><SPAN class="toctext">Friday, July 26, 2013</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#Week_5"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Week 5</SPAN></A></LI><LI class="toclevel-3 tocsection-23"><A href="#Monday.2C_July_29.2C_2013"><SPAN class="tocnumber">2.5.1</SPAN><SPAN class="toctext">Monday, July 29, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-24"><A href="#Tuesday.2C_July_30.2C_2013"><SPAN class="tocnumber">2.5.2</SPAN><SPAN class="toctext">Tuesday, July 30, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-25"><A href="#Wednesday.2C_July_31.2C_2013"><SPAN class="tocnumber">2.5.3</SPAN><SPAN class="toctext">Wednesday, July 31, 2013</SPAN></A></LI><LI class="toclevel-1 tocsection-26"><A href="#August"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">August</SPAN></A></LI><LI class="toclevel-2 tocsection-27"><A href="#Week_1_2"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Week 1</SPAN></A></LI><LI class="toclevel-3 tocsection-28"><A href="#Thursday.2C_August_1.2C_2013"><SPAN class="tocnumber">3.1.1</SPAN><SPAN class="toctext">Thursday, August 1, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-29"><A href="#Friday.2C_August_2.2C_2013"><SPAN class="tocnumber">3.1.2</SPAN><SPAN class="toctext">Friday, August 2, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-30"><A href="#Saturday.2C_August_3.2C_2013"><SPAN class="tocnumber">3.1.3</SPAN><SPAN class="toctext">Saturday, August 3, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-31"><A href="#Sunday.2C_August_4.2C_2013"><SPAN class="tocnumber">3.1.4</SPAN><SPAN class="toctext">Sunday, August 4, 2013</SPAN></A></LI><LI class="toclevel-2 tocsection-32"><A href="#Week_2_2"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Week 2</SPAN></A></LI><LI class="toclevel-3 tocsection-33"><A href="#Monday.2C_August_5.2C_2013"><SPAN class="tocnumber">3.2.1</SPAN><SPAN class="toctext">Monday, August 5, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-34"><A href="#Tuesday.2C_August_6.2C_2013"><SPAN class="tocnumber">3.2.2</SPAN><SPAN class="toctext">Tuesday, August 6, 2013</SPAN></A></LI><LI class="toclevel-4 tocsection-35"><A href="#Advisor_Meeting"><SPAN class="tocnumber">3.2.2.1</SPAN><SPAN class="toctext">Advisor Meeting</SPAN></A></LI><LI class="toclevel-3 tocsection-36"><A href="#Wednesday.2C_August_7.2C_2013"><SPAN class="tocnumber">3.2.3</SPAN><SPAN class="toctext">Wednesday, August 7, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-37"><A href="#Notes_from_meeting_with_Karyl"><SPAN class="tocnumber">3.2.4</SPAN><SPAN class="toctext">Notes from meeting with Karyl</SPAN></A></LI><LI class="toclevel-3 tocsection-38"><A href="#Thursday.2C_August_8.2C_2013"><SPAN class="tocnumber">3.2.5</SPAN><SPAN class="toctext">Thursday, August 8, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-39"><A href="#Friday.2C_August_9.2C_2013"><SPAN class="tocnumber">3.2.6</SPAN><SPAN class="toctext">Friday, August 9, 2013</SPAN></A></LI><LI class="toclevel-2 tocsection-40"><A href="#Week_3_3"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Week 3</SPAN></A></LI><LI class="toclevel-3 tocsection-41"><A href="#Monday.2C_August_12.2C_2013"><SPAN class="tocnumber">3.3.1</SPAN><SPAN class="toctext">Monday, August 12, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-42"><A href="#Tuesday.2C_August_13.2C_2013"><SPAN class="tocnumber">3.3.2</SPAN><SPAN class="toctext">Tuesday, August 13, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-43"><A href="#Wednesday.2C_August_14.2C_2013"><SPAN class="tocnumber">3.3.3</SPAN><SPAN class="toctext">Wednesday, August 14, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-44"><A href="#Thursday.2C_August_15.2C_2013"><SPAN class="tocnumber">3.3.4</SPAN><SPAN class="toctext">Thursday, August 15, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-45"><A href="#Friday.2C_August_16.2C_2013"><SPAN class="tocnumber">3.3.5</SPAN><SPAN class="toctext">Friday, August 16, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-46"><A href="#Saturday.2C_August_17.2C_2013"><SPAN class="tocnumber">3.3.6</SPAN><SPAN class="toctext">Saturday, August 17, 2013</SPAN></A></LI><LI class="toclevel-2 tocsection-47"><A href="#Week_4_3"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">Week 4</SPAN></A></LI><LI class="toclevel-3 tocsection-48"><A href="#Sunday.2C_August_18.2C_2013"><SPAN class="tocnumber">3.4.1</SPAN><SPAN class="toctext">Sunday, August 18, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-49"><A href="#Tuesday.2C_August_20.2C_2013"><SPAN class="tocnumber">3.4.2</SPAN><SPAN class="toctext">Tuesday, August 20, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-50"><A href="#Wednesday.2C_August_21.2C_2013"><SPAN class="tocnumber">3.4.3</SPAN><SPAN class="toctext">Wednesday, August 21, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-51"><A href="#Thursday.2C_August_22.2C_2013"><SPAN class="tocnumber">3.4.4</SPAN><SPAN class="toctext">Thursday, August 22, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-52"><A href="#Friday.2C_August_23.2C_2013"><SPAN class="tocnumber">3.4.5</SPAN><SPAN class="toctext">Friday, August 23, 2013</SPAN></A></LI><LI class="toclevel-2 tocsection-53"><A href="#Week_5_2"><SPAN class="tocnumber">3.5</SPAN><SPAN class="toctext">Week 5</SPAN></A></LI><LI class="toclevel-3 tocsection-54"><A href="#Monday.2C_August_26.2C_2013"><SPAN class="tocnumber">3.5.1</SPAN><SPAN class="toctext">Monday, August 26, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-55"><A href="#Tuesday.2C_August_27.2C_2013"><SPAN class="tocnumber">3.5.2</SPAN><SPAN class="toctext">Tuesday, August 27, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-56"><A href="#Wednesday.2C_August_28.2C_2013"><SPAN class="tocnumber">3.5.3</SPAN><SPAN class="toctext">Wednesday, August 28, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-57"><A href="#Thursday.2C_August_29.2C_2013"><SPAN class="tocnumber">3.5.4</SPAN><SPAN class="toctext">Thursday, August 29, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-58"><A href="#Friday.2C_August_30.2C_2013"><SPAN class="tocnumber">3.5.5</SPAN><SPAN class="toctext">Friday, August 30, 2013</SPAN></A></LI><LI class="toclevel-1 tocsection-59"><A href="#September"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">September</SPAN></A></LI><LI class="toclevel-2 tocsection-60"><A href="#Week_1_3"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Week 1</SPAN></A></LI><LI class="toclevel-3 tocsection-61"><A href="#Tuesday.2C_September_3.2C_2013"><SPAN class="tocnumber">4.1.1</SPAN><SPAN class="toctext">Tuesday, September 3, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-62"><A href="#Wednesday.2C_September_4.2C_2013"><SPAN class="tocnumber">4.1.2</SPAN><SPAN class="toctext">Wednesday, September 4, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-63"><A href="#Thursday.2C_September_5.2C_2013"><SPAN class="tocnumber">4.1.3</SPAN><SPAN class="toctext">Thursday, September 5, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-64"><A href="#Friday.2C_September_6.2C_2013"><SPAN class="tocnumber">4.1.4</SPAN><SPAN class="toctext">Friday, September 6, 2013</SPAN></A></LI><LI class="toclevel-2 tocsection-65"><A href="#Week_2_3"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Week 2</SPAN></A></LI><LI class="toclevel-3 tocsection-66"><A href="#Monday.2C_September_9.2C_2013"><SPAN class="tocnumber">4.2.1</SPAN><SPAN class="toctext">Monday, September 9, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-67"><A href="#Tuesday.2C_September_10.2C_2013"><SPAN class="tocnumber">4.2.2</SPAN><SPAN class="toctext">Tuesday, September 10, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-68"><A href="#Wednesday.2C_September_11.2C_2013"><SPAN class="tocnumber">4.2.3</SPAN><SPAN class="toctext">Wednesday, September 11, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-69"><A href="#Thursday.2C_September_12.2C_2013"><SPAN class="tocnumber">4.2.4</SPAN><SPAN class="toctext">Thursday, September 12, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-70"><A href="#Friday.2C_September_13.2C_2013"><SPAN class="tocnumber">4.2.5</SPAN><SPAN class="toctext">Friday, September 13, 2013</SPAN></A></LI><LI class="toclevel-3 tocsection-71"><A href="#Saturday.2C_September_14.2C_2013"><SPAN class="tocnumber">4.2.6</SPAN><SPAN class="toctext">Saturday, September 14, 2013</SPAN></A></LI><LI class="toclevel-2 tocsection-72"><A href="#Week_3_4"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Week 3</SPAN></A></LI><LI class="toclevel-3 tocsection-73"><A href="#Monday.2C_September_16.2C_2013"><SPAN class="tocnumber">4.3.1</SPAN><SPAN class="toctext">Monday, September 16, 2013</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="June">June</SPAN></H2><H3><SPAN class="mw-headline" id="Week_3">Week 3</SPAN></H3><H4><SPAN class="mw-headline" id="Tuesday.2C_June_18.2C_2013">Tuesday, June 18, 2013</SPAN></H4><P><B>First iGEM Meeting of the Summer!</B></P><UL><LI>All student members gathered in the lab. 
</LI></UL><UL><LI>We toured the lab, prep room, and incubator room, and we discussed the lab rules.
</LI></UL><UL><LI>We also split into teams to discuss lab logistics: contacting funding sources, ordering reagents/materials, compiling protocols, recording our electronic lab notebook, and making plates/media.
</LI></UL><H3><SPAN class="mw-headline" id="Week_4">Week 4</SPAN></H3><H4><SPAN class="mw-headline" id="Wednesday.2C_June_26.2C_2013">Wednesday, June 26, 2013</SPAN></H4><P><B>Planning Wet Lab Work</B></P><UL><LI>We split into planning sub-teams to plan out different parts of the project with protocols, timelines, and ordering reagents.
</LI></UL><DL><DD><UL><LI>Plasmid Design
</LI><LI>Media and Competent Cells
</LI><LI>E. coli and B. subtilis Transformation
</LI><LI>Keratinase Assay and His tagging
</LI><LI>Running DNA gels, SDS-PAGE gels, and western blots
</LI></UL></DD></DL><UL><LI>We will also be holding weekly meetings with our graduate student advisers and biweekly meetings with our faculty advisers to provide updates. 
</LI><LI>Wet lab work begins tomorrow!
</LI></UL><H4><SPAN class="mw-headline" id="Friday.2C_June_28.2C_2013">Friday, June 28, 2013</SPAN></H4><P><B>Researching the Sequence of kerA</B></P><UL><LI>Planning subteams continued to order reagents and compile protocols.
</LI><LI>The plasmid design team decided on gibson assembly to produce the kerA biobrick with geneblocks ordered from IDT.
</LI><LI>kerA sequence was obtained from “Nucleotide Sequence and Expression of kerA, the Gene Encoding a Keratinolytic Protease of Bacillus licheniformis PWD-1” (Lin et al. 1995). 
</LI></UL><DL><DD><UL><LI>Two putative promoters (red), a possible ribosomal binding site (green), and a transcriptional terminator sequence (light blue, single underline) are indicated. The putative starting residues of the preprotein (pre = blue), proprotein (pro = pink), and mature protein (mature = purple) are indicated.
</LI><LI>EcoRI (orange) and PstI (blue) sites within the gene are indicated and must be removed to produce the kerA biobrick.
</LI></UL></DD></DL><DIV class="dividerNotebook"><P><A href="#top">[top]</A></P></DIV><H2><SPAN class="mw-headline" id="July">July</SPAN></H2><H3><SPAN class="mw-headline" id="Week_1">Week 1</SPAN></H3><H4><SPAN class="mw-headline" id="Wednesday.2C_July_3.2C_2013">Wednesday, July 3, 2013</SPAN></H4><P><B>BioBrick Design Presentation and Grad Adviser Meeting</B></P><UL><LI>Designed a biobrick with:
</LI></UL><DL><DD><UL><LI>C terminus His-tagged kerA
</LI></UL></DD><DD><UL><LI>CATCATCATCATCATCAT
</LI></UL></DD></DL><UL><LI>Prefix: gaattcgcggccgcttctagag
</LI><LI>Suffix: tactagtagcggccgctgcag
</LI></UL><UL><LI>Moreover, the bio brick must include
</LI></UL><DL><DD><UL><LI>Eliminated EcoRI and PstI restriction sites from kerA nucleotide seq
</LI></UL></DD><DD><UL><LI>EcoRI (gaattc--814) and PstI (ctgcag--978)
</LI><LI>Conserved aa seq + biobrick prefix + suffix
</LI></UL></DD></DL><UL><LI>Secretion signal peptide
</LI><LI>Inducible promoter
</LI></UL><UL><LI>We also presented a feasible timeline of our project, which included:
</LI></UL><DL><DD><UL><LI>Assembly of biobrick and transformation into E. coli 
</LI><LI>Transformation of B. subtilis
</LI><LI>Keratinase isolation and keratinase assays
</LI><LI>kerA mutagenesis for optimization of the keratinase
</LI></UL></DD></DL><UL><LI>We also split up basic lab duties!
</LI></UL><DL><DD><UL><LI>Ivan &amp; Ethan are in charge of the IBC protocol.
</LI></UL></DD><DD><UL><LI>We can’t start wet lab work until our IBC protocol is finished!
</LI></UL></DD></DL><UL><LI>Ethan is in charge of minutes.
</LI><LI>Susan is in charge of ordering invoices.
</LI><LI>Annie is in charge of emailing advisors and members to plan meetings
</LI><LI>Chloe is getting the B. subtilis gene that expressing keratinase.
</LI></UL><H4><SPAN class="mw-headline" id="Thursday.2C_July_4.2C_2013">Thursday, July 4, 2013</SPAN></H4><P><B>Milk Agar Plates</B></P><UL><LI>The purpose of milk agar plates is often to detect protease activity, including the activity of keratinases. 
</LI><LI>A measurement of any baseline protease activity of our parent B. subtilis strain (WB700), which does not endogenously express any keratinases, is desired. 
</LI><LI>Excess protease activity of our parent B. subtilis strain may prevent our use of milk agar plates as an assay for keratinase expression of our future transformants with the kerA gene.
</LI><LI>Therefore, the parent B. subitilis strain (WB700) was streaked on the milk agar plates to observe baseline protease activity that could prevent us from detecting keratinase expression after transformation.
</LI><LI><A href="https://2013.igem.org/Team:UChicago/Protocols#Making_Agar_Plates" class="external text" rel="nofollow">Milk Agar Plates</A> (in LB):
</LI></UL><DL><DD><UL><LI>Total volume of  600 ml (makes 20-25 plates)
</LI><LI>20g (30*.6) nonfat dry milk powder
</LI><LI>500mL 2X <A href="https://2013.igem.org/Team:UChicago/Protocols#Making_LB_.28500mL.29" class="external text" rel="nofollow">LB medium</A></LI><LI>250mL 2X <A href="https://2013.igem.org/Team:UChicago/Protocols#Making_Agar_Plates" class="external text" rel="nofollow">agar</A></LI></UL></DD></DL><UL><LI>Ideally, there should be a minimal zone of clearing around our B. subtilis colonies.
</LI><LI>Next Steps:
</LI></UL><DL><DD><UL><LI>Observe the results of our milk agar plates with B. subtilis colonies
</LI></UL></DD></DL><H4><SPAN class="mw-headline" id="Friday.2C_July_5.2C_2013">Friday, July 5, 2013</SPAN></H4><P><B>Primer Design for kerA biobrick</B></P><UL><LI>Designed primers for directed mutagenesis of our kerA biobrick using <A href="http://www.bioinformatics.org/primerx/" class="external text" rel="nofollow">PrimerX</A>:
</LI></UL><OL><LI>Primer pair 1 (EcoRI mutation, aat --&gt; aac, asparagine)
<DL><DD>a. Forward: 5' GGTTAAAGTACTGAACTCAAGCGGAAGCGG 3'
</DD><DD>b. Reverse: 5' CCGCTTCCGCTTGAGTTCAGTACTTTAACC 3'
</DD></DL></LI><LI>Primer pair 2 (Pst1 mutation, gca--&gt;gcg, alanine)
<DL><DD>a. Forward: 5' GTTGTCGTTGTAGCTGCGGCAGGGAACAGCGGATC 3'
</DD><DD>b. Reverse: 5' GATCCGCTGTTCCCTGCCGCAGCTACAACGACAAC 3'
</DD></DL></LI></OL><UL><LI>Used APE for checking codons and mutating restriction sites
</LI></UL><H4><SPAN class="mw-headline" id="Saturday.2C_July_6.2C_2013">Saturday, July 6, 2013</SPAN></H4><P><B>Primer Design Part II</B></P><UL><LI>Designed primers to isolate non-coding and coding sequence of kerA and add prefix and suffix
</LI></UL><DL><DD><UL><LI>forward primer + xbaI site:
</LI></UL></DD><DD> 5' TATCTAGAGGTCTATTCATACTTTCGAA 3'
<UL><LI>reverse primer + speI site (reverse complement):
</LI></UL></DD><DD> 5' ATTGATCATTGGACAACCTTCATCAG 3'
</DD></DL><UL><LI>We still need to design additional primers that will include the secretion signal peptide (in case it needs to be secreted out of B. subtilis) and kerA (From start to stop codon).
</LI><LI>These primers should include igem biobrick prefix and his tag (before stop codon) + suffix.
</LI></UL><H3><SPAN class="mw-headline" id="Week_2">Week 2</SPAN></H3><H4><SPAN class="mw-headline" id="Monday.2C_July_8.2C_2013">Monday, July 8, 2013</SPAN></H4><P><B>Milk Agar Plate Results</B></P><UL><LI>The results of our milk agar assay from 7/04/13 of the non-transformed B. subtilis strain WB700:
</LI></UL><UL><LI>The zone of clearing around the B. subtilis colonies indicates that our parent strain breaks down milk. Its prominent endogenous protease activity prevents us from using the milk agar assay as a screen for successfully transformed kerA-expressing B. subtilis. 
</LI></UL><UL><LI>Alternative method to screen for transformed B. subtilis needed: 
</LI></UL><OL><LI>The feather assay used in Saha and Dhanasekaran’s 2010 paper, “Isolation and Screening of Keratinolytic Actinobacteria form Keratin Waste Dumped Soil in Tiruchirappalli and Nammakkal, Tamil Nadu, India”
</LI><LI>Quantitative assay using azokeratin
</LI></OL><P>
Our future plans: 
</P><UL><LI>For our promoter for kerA, Karyl, one of our graduate advisors, recommended P43, a constitutive promoter, to transform into B. subtilis.
</LI></UL><UL><LI>We also decided to use pUB110, a high copy number plasmid used in S. aureus and B. subtilis, to introduce kerA into our parent B. subtilis strain.
</LI><LI>pUB110 will also be made into a biobrick.
</LI></UL><UL><LI>We have to choose a suitable plasmid (either pSB1A3, pSB1C3, pSB1T3, or pSB1K3.m1) for our E. coli transformations.
</LI></UL><H4><SPAN class="mw-headline" id="Wednesday.2C_July_10.2C_2013">Wednesday, July 10, 2013</SPAN></H4><P><B>Grad Advisor Meeting on further developed Keratinase Project</B></P><UL><LI>We presented the primers we designed to our advisers, who suggested we double check them for self dimerization and 3’ overlaps with: 
</LI></UL><DL><DD><UL><LI><A href="http://www.idtdna.com/analyzer/applications/oligoanalyzer/" class="external free" rel="nofollow">http://www.idtdna.com/analyzer/applications/oligoanalyzer/</A></LI><LI><A href="http://bioinfo.ut.ee/primer3/" class="external free" rel="nofollow">http://bioinfo.ut.ee/primer3/</A></LI></UL></DD></DL><UL><LI>We discovered that our forward primer forms a self-dimer and must be redesigned.
</LI></UL><UL><LI>We also need lysozyme to digest the peptidoglycan of B. subtilis in order to miniprep it.
</LI></UL><UL><LI>We decided to use the iGEM plasmid pSB1K3.m1 for our E. coli transformations.
</LI></UL><UL><LI>To screen for keratinase activity, instead of using the feather assay, which takes up to a month for visible results to appear, we decided to use the azokeratin assay as the secondary screening method.
</LI></UL><H4><SPAN class="mw-headline" id="Thursday.2C_July_11.2C_2013">Thursday, July 11, 2013</SPAN></H4><P><B>Meeting with Purdue iGEM team</B></P><P>Get Excited for the UChicago X Purdue collaboration! Our first meeting with the Purdue team went well: 
</P><UL><LI>The Purdue team will send us a form to provide feedback on their igem biobrick characterization and protocol forms on Monday.
</LI></UL><UL><LI>The biobrick forms are in PDF format.
</LI></UL><UL><LI>Their biobrick characterization contains info like biobrick sequence, plasmid and microorganism in which it’s inserted, ect.
</LI></UL><UL><LI>The protocol form is a checklist of the protocols carried out to characterize the biobrick. It doesn’t contain much info.
</LI></UL><UL><LI>Nora asked them to add a section stating the purpose for using a specific protocol, as in, what specifically about the biobrick are they trying to characterize.
</LI></UL><H3><SPAN class="mw-headline" id="Week_3_2">Week 3</SPAN></H3><H3><SPAN class="mw-headline" id="Week_4_2">Week 4</SPAN></H3><H4><SPAN class="mw-headline" id="Tuesday.2C_July_23.2C_2013">Tuesday, July 23, 2013</SPAN></H4><P>Today we made LB media with kanamycin and chloramphenicol. There are 7 plates of each kind of antibiotic marker left over; we put them in the 4ºC fridge. We autoclaved all the media that we made today and yesterday.
</P><P>We transformed the plasmid with promoter Veg (in the pSB1C3 vector) into TOP10 E. coli competent cells. We got the DNA (part BBa_K143012) from well 23B on the distribution kit. The bacteria have been plated onto a chloramphenicol plate. They have been left to incubate at 37ºC in Nora’s lab (GCIS W519).
</P><P><I>Future plans:</I></P><UL><LI>Select a few colonies and inoculate overnight in 3 mL cultures (These will eventually be used to make glycerol stocks and for DNA miniprep). Do this later on in the day... ~3 pm onwards.
</LI><LI>Store the plate in 4ºC afterwards in case it is needed in the future.
</LI><LI>Make liquid LB media (no agar!) with 50 µg/mL chloramphenicol.
</LI></UL><H4><SPAN class="mw-headline" id="Wednesday.2C_July_24.2C_2013">Wednesday, July 24, 2013</SPAN></H4><P>2 colonies were selected from yesterday's pVeg transformants to be grown in an overnight liquid culture. However, we redid the transformation using a slightly different procedure because there were very few transformants:
</P><OL><LI> Let the cells thaw on ice.
</LI><LI> Add your plasmid to the competent cells.
</LI><LI> Leave on ice for 10 mins.
</LI><LI> Place on the 42ºC heat block for 45 s to heat shock.
</LI><LI> Put on ice for 2 minutes.
</LI><LI> Add 1 mL SOC media.
</LI><LI> Incubate in the 37ºC shaker for 1 hour. 
</LI><LI> Centrifuge at 8 rpm for 30 seconds. Decant all but ~100 µL supernatant and resuspend.
</LI><LI> Plate onto chloramphenicol (or antibiotic of choice) plates and use glass beads to spread out the colonies.
</LI></OL><P><I>Future plans:</I></P><UL><LI>Make DH5a competent cells; our poor transformation results could be due to incompetent cells.
</LI><LI>Isolate Pveg (for B. subtilis) promoter from transformants grown overnight
</LI></UL><DL><DD><UL><LI>Do DNA miniprep of the transformants
</LI><LI>Cut w/ EcoRI and SpeI
</LI><LI>Run in 0.9% or 1% agarose gel (make gel w/ large wells, load 50 uL of sample)
</LI><LI>If there is a band of right size of Pveg, do a gel extraction
</LI><LI>Put at -20ºC
</LI><LI>If there is no correct band, set overnight cultures of transformants from the transfromation done by Ivan today (7/24/13)
</LI></UL></DD></DL><UL><LI>And for Friday's meeting (7/26/13)
</LI></UL><DL><DD><UL><LI>Isolate Pveg (for B. subtilis) promoter as described above if it was unsuccessful the day before. Use the overnight cultures set up on 7/25/13.
</LI><LI>Check if the transformation w/ the E. coli constitutive promoter was successful. If yes, store plate at 4ºC.
</LI></UL></DD></DL><H4><SPAN class="mw-headline" id="Friday.2C_July_26.2C_2013">Friday, July 26, 2013</SPAN></H4><P><I><B>To do:&lt;/i&gt;</B></I></P><UL><LI> Make overnight cultures of colonies from Annie’s transformation.
</LI><LI> Transform 18O and 18E again, this time into the competent cells made yesterday by Ethan and Annie. Those plasmids should be in the -20ºC.--DO MONDAY, Ethan’s doing with transformation efficiency 7/29
</LI><LI> Check the efficiency of our competent cells by using the transformation efficiency kit from iGEM (check the instructions on the iGEM wiki).--DO MONDAY
</LI><LI> Do a gel purification of the digestion of the PVeg promoter &amp; check the size.
</LI></UL><DL><DD><UL><LI> The digested DNA is sitting in the fridge. Use ALL of it.
</LI></UL></DD></DL><H3><SPAN class="mw-headline" id="Week_5">Week 5</SPAN></H3><H4><SPAN class="mw-headline" id="Monday.2C_July_29.2C_2013">Monday, July 29, 2013</SPAN></H4><P>&lt;i&gt;To do:</P><UL><LI> Set O.N. culture of 18O (2X 3mL)--done by Annie (18O is the E. coli constitutive promoter)
</LI><LI> Check transformation efficiency, 18E of competent cells made by Ethan
</LI><LI> Run gel with Pveg digestion (cut w/ EcoRI and SpeI) 
</LI><LI> Redo the digestion with 10uL of DNA miniprep of Pveg w/ EcoRI and SpeI for 2 hr
</LI><LI> Run in 2% agarose
</LI></UL><DL><DD><UL><LI> If there are bands--purify promoter
</LI><LI> If there are no bands, set up O.N. culture of Ivan’s transformation of Pveg
</LI></UL></DD></DL><UL><LI> Set O.N. culture of B. subtilis BD 366 w/ pUB110 (2X 3mL)--done by Nora
</LI></UL><P><I>Getting done today:</I></P><UL><LI> Re-digest pVeg, digest 5ul of miniprep 1 2hrs at 37 deg C:
</LI></UL><DL><DD><UL><LI> 2 ul 10X buffer 2.1
</LI><LI> 1 ul EcoRI HF
</LI><LI> 1 ul SpeI
</LI><LI> 5 ul mini prep 1
</LI><LI> H2O to 20 ul
</LI></UL></DD></DL><UL><LI> Also set previously done digest by Klevin for 2 hrs at 37 deg C
</LI></UL><P>--Done by Annie and Nora
</P><UL><LI> Transformed 50 uls of new competent cells and old competent cells with 0.5 ul of 18E resuspension and plated 50 ul of new competent cells as a negative control
</LI></UL><P>--Done by Ethan
</P><P><I>Next Steps:</I></P><UL><LI> Run digest of Pveg on 2% agarose gel
</LI><LI> Do miniprep of B. subtilis pUB110
</LI><LI> Do miniprep of 18O transformants
</LI></UL><H4><SPAN class="mw-headline" id="Tuesday.2C_July_30.2C_2013">Tuesday, July 30, 2013</SPAN></H4><UL><LI> Ran gel of Pveg miniprep digest (Klevin’s at total 3hrs, and Nora’s/Annie’s at 2hrs digestion) with EcoRI and SpeI
</LI></UL><DL><DD><UL><LI> 10 ul ladder
</LI><LI> 20 ul of samples + 5 ul of Orange G dye
</LI></UL></DD></DL><UL><LI> Final gel: Nora’s and Annie’s (N&amp;A) had faint band at 100 band marker, we assume this is the fragment
</LI><LI> Nora purified gel fragment, with Qiagen kit, eluted in 30ul water in order to obtain a higher concentration of DNA.
</LI></UL><UL><LI> Miniprep were prepared of 2 B. subtilis samples, pUB110 and 18O. They were stored at -20ºC.
</LI><LI> A transformation of 18E will need to be redone; incorrect selection marker was used.
</LI></UL><UL><LI> Ethan is testing the competency of the TOP10 cells he made.
</LI></UL><H4><SPAN class="mw-headline" id="Wednesday.2C_July_31.2C_2013">Wednesday, July 31, 2013</SPAN></H4><P>
gBlock Fragment Gibson Assembly calculations:
</P><TABLE border="1"><TBODY><TR><TD>total fmol (resuspended in 20 ul H2O)</TD><TD>final concentration</TD></TR><TR><TD>gene block 1</TD><TD>648 fmol</TD><TD>.032 pmol/ul</TD></TR><TR><TD>gene block 2</TD><TD>455.4 fmol</TD><TD>.023 pmol/ul</TD></TR></TBODY></TABLE><P>
Gibson assembly mix (Trial 1):
</P><DL><DD><UL><LI>4 ul gblock 1
</LI><LI>6 ul gblock 2
</LI><LI>10 ul master mix
</LI></UL></DD></DL><P>We suspended pub110 insert oligonucleotides in H2O (to 100 uM) and annealed them at 94°C to make duplex dna
</P><P><I>Next steps:</I></P><DL><DD><UL><LI> Check the concentration on the oligos against the duplex DNA on the nanodrop to confirm annealing.
</LI></UL></DD></DL><DIV class="dividerNotebook"><P><A href="#top">[top]</A></P></DIV><H2><SPAN class="mw-headline" id="August">August</SPAN></H2><H3><SPAN class="mw-headline" id="Week_1_2">Week 1</SPAN></H3><H4><SPAN class="mw-headline" id="Thursday.2C_August_1.2C_2013">Thursday, August 1, 2013</SPAN></H4><DL><DD><UL><LI>Nora’s running PCR for Pveg, 18O, kerA assembly in Kron Lab
</LI></UL></DD></DL><P>
Concentration results from nanodrop:
</P><DL><DD><UL><LI>Forward pUB110 ssDNA: 1596.12 ng/µL
</LI><LI>Reverse pUB110 ssDNA: 1913.96 ng/µL
</LI><LI>Annealed pUB110 dsDNA: 2114.50 ng/µL*
</LI></UL></DD></DL><TABLE border="1"><TBODY><TR><TD>sample</TD><TD>ng/ul</TD><TD>A260</TD><TD>A280</TD><TD>A230</TD><TD>260/230</TD><TD>extinction coefficient</TD></TR><TR><TD>oligo-f</TD><TD>1596</TD><TD>48.4</TD><TD>25.6</TD><TD>22.3</TD><TD>2.17</TD><TD>33</TD></TR><TR><TD>oligo-r</TD><TD>1914</TD><TD>58.0</TD><TD>30.0</TD><TD>26.4</TD><TD>2.2</TD><TD>33</TD></TR><TR><TD>annealed</TD><TD>2159</TD><TD>42.3</TD><TD>22.3</TD><TD>13.8</TD><TD>2.25</TD><TD>50</TD></TR></TBODY></TABLE><P>
A = E * b * c
</P><P>C_annealed = [A / (50 * b)]
2159*(50/33)  = 3271
</P><P>Mass concentration of forward and reverse oligo mixture (ng/ul) = (1596 + 1914)/2 = 1755 ng/ul
</P><DL><DD><UL><LI>If none of the DNA anneals, the extinction coefficient is 33 and the nanodrop should read 1755*(50/33) = 2659 ng/ul
</LI><LI>If DNA was completely annealed, the extinction coefficient is 50 and the reading on the nanodrop should be 1755 ng/ul
</LI></UL></DD><DD><UL><LI>Therefore, the percentage of annealing is (2659 - 2159)/(2659 - 1755) = 500/905 = 55 %
</LI></UL></DD></DL><P><I>The assembled DNA was then digested in the following conditions:</I></P><DL><DD><UL><LI>0.5 ul pUB110 insert
</LI><LI>0.5 ul NdeI
</LI><LI>0.5 ul AflII (N)
</LI><LI>1.5 ul 2.1 buffer
</LI><LI>12 ul H2O
</LI><LI>5 ul pUB110 miniprep #1
</LI><LI>0.5 ul NdeI
</LI><LI>0.5 ul AflII (N)
</LI><LI>1.5 ul 2.1 buffer
</LI><LI>7.5 ul H2O
</LI></UL></DD></DL><H4><SPAN class="mw-headline" id="Friday.2C_August_2.2C_2013">Friday, August 2, 2013</SPAN></H4><UL><LI>Today we set up digests of pUB110 and 18O.
</LI></UL><DL><DD><UL><LI>Conditions:
</LI></UL></DD><DD><UL><LI>18O promoter:
</LI></UL></DD></DL><DL><DD><UL><LI>5 ul 18O miniprep insert
</LI><LI>0.5 ul EcoRI
</LI><LI>0.5 ul PstI
</LI><LI>1.5 ul 2.1 buffer
</LI><LI>7.5 ul H2O
</LI><LI>Incubate at 37°C beginning at 3 pm
</LI></UL></DD></DL><DL><DD><UL><LI>pUB110 promoter:
</LI></UL></DD><DD><UL><LI>5 ul pUB110 miniprep insert
</LI><LI>0.5 ul Afl II (blue-labelled)
</LI><LI>0.5 ul Nde I (Nora’s)
</LI><LI>1.5 ul 2.1 buffer
</LI><LI>7.5 ul H2O
</LI><LI>Incubate at 37°C for 2 hrs
</LI></UL></DD></DL><UL><LI>Yields of pVeg fragment DNA have been low; we will start over by picking more pVeg transformant colonies and cloning more DNA.
</LI></UL><H4><SPAN class="mw-headline" id="Saturday.2C_August_3.2C_2013">Saturday, August 3, 2013</SPAN></H4><H4><SPAN class="mw-headline" id="Sunday.2C_August_4.2C_2013">Sunday, August 4, 2013</SPAN></H4><H3><SPAN class="mw-headline" id="Week_2_2">Week 2</SPAN></H3><H4><SPAN class="mw-headline" id="Monday.2C_August_5.2C_2013">Monday, August 5, 2013</SPAN></H4><UL><LI>Digest pUB110 promoter for Pre gene (bit.ly/16v7PRO a planning doc):
</LI></UL><DL><DD><UL><LI>One of the enzymes may have lost activity, so use NdeI sample taken from Nora instead
</LI></UL></DD></DL><DL><DD><UL><LI>7.5 ul H2O, nuclease-free
</LI><LI>5 ul pUB110 miniprep insert, mp1
</LI><LI>1.5 ul 2.1 buffer, labelled “in-use”
</LI><LI>0.5 ul Afl II, blue-labelled Ethan’s
</LI><LI>0.5 ul Nde I, Nora’s
</LI><LI>incubate at 37°C at 1pm to 3pm, placed in 4°C 
</LI></UL></DD></DL><P>Ran a gel of the following DNA samples:
</P><DL><DD><UL><LI>lane 1: pVeg mp1 (2 ul)
</LI><LI>lane 2: pVeg mp2 (2 ul)
</LI><LI>lane 3: pVeg mp3 (2 ul)
</LI><LI>lane 4: pVeg mp4 (2 ul)
</LI><LI>lane 5: empty
</LI><LI>lane 6: 18O digest (15 ul)
</LI><LI>lane 7: ladder (8 ul)
</LI><LI>lane 8: pUB119 digest (15 ul)
</LI></UL></DD></DL><P>Based on the results, it seems that none of our digests are working. It may be that the enzymes we are using have lost activity, so we may have to obtain fresh ones.
</P><P>The following transformations were carried out by Annie:
</P><DL><DD><UL><LI>BBa_K314103 → Plate 1 : 4D in pSB1C3  (1638 bp)
</LI><LI>BBa_K314100 → Kit 1 : 1G (518 bp)
</LI><LI>BBa_K541501 → 11E 
</LI><LI>Plated 500ul per plate, incubated o/n in Nora’s lab
</LI><LI>Shaker was off when we arrived so unsure if that will affect anything...
</LI></UL></DD></DL><P>
In other news, the BL21 competent cells were dropped off by Karyl.
</P><P><I>To Do:</I></P><UL><LI>Run gel of sample from gel purified pUB110 from last week
</LI><LI>Re-do Digestion of pUB110 and single-enzyme digestion of pUB110, and digestion of pVeg miniprep (overnight) (Ethan)
</LI><LI>Gel purification of last week’s questionable pUB110 fragment (Klevin)
</LI></UL><H4><SPAN class="mw-headline" id="Tuesday.2C_August_6.2C_2013">Tuesday, August 6, 2013</SPAN></H4><UL><LI>Running the gel with inserts
</LI></UL><DL><DD><UL><LI>Running 2% agarose gel at 120 volts
</LI></UL></DD><DD><UL><LI>Lane 1: biobrick insert
</LI><LI>Lane 2: puB110 from gel extract (Klevin’s)
</LI><LI>Lane 3: pub110 from gel extract (Ethan’s)
</LI><LI>Lane 4: skipped
</LI><LI>Lane 5: pub110 insert forward
</LI><LI>Lane 6: ladder
</LI></UL></DD></DL><P>All but pUB110 insert forward ran off of the gel (sorry!)
Rerunning:
</P><DL><DD><UL><LI>Lane 1: ladder
</LI><LI>Lane 2: KerA biobrick
</LI><LI>Lane 3: pub110 insert forward
</LI><LI>Lane 4: pUB110 from gel extract (Klevin’s)
</LI><LI>Lane 5: pUB110 from gel extract (Ethan’s)
</LI></UL></DD></DL><UL><LI>Klevin set up the overnight culture of the casettes
</LI><LI>Susan gel extracts the PB110
</LI><LI>Ethan will pick up reagents for B. subtilis transformation
</LI></UL><H5><SPAN class="mw-headline" id="Advisor_Meeting">Advisor Meeting</SPAN></H5><OL><LI> Find distribution part with only pVeg and RBS no S.S. 
</LI><LI> Figure out how to buy keratin (non-hydrolyzed) 
</LI><LI> Figure out how to make keratin infused plate
</LI><LI> Figure out what the f1 origin does, does it allow high copy number?
</LI><LI> Figure out if the Pre gene is necessary by trying B. subtilis transformation with pUB110 undigested
</LI><LI> Double check if pVeg (with the S.S.) is in frame with KerA gene; try these combos, all with pVeg + RBS
</LI><LI> KerA + pVeg, both with S.S.
</LI><LI> KerA + pVeg, with pVeg only with S.S.
</LI><LI> KerA + pVeg, with KerA only with S.S.
</LI></OL><LI> Feathers dissolve in DMSO and precipitate it in acetone (Source: <A href="http://onlinelibrary.wiley.com/doi/10.1046/j.1365-2672.1999.00485.x/full" class="external free" rel="nofollow">http://onlinelibrary.wiley.com/doi/10.1046/j.1365-2672.1999.00485.x/full</A>)
</LI><LI>Characterization of alkaline protease from Bacillus: <A href="http://link.springer.com/content/pdf/10.1007%2FBF00172544.pdf" class="external free" rel="nofollow">http://link.springer.com/content/pdf/10.1007%2FBF00172544.pdf</A></LI><LI>Japanese 1989 assay for Keratin: <A href="http://link.springer.com/content/pdf/10.1007%2FBF00263997.pdf" class="external free" rel="nofollow">http://link.springer.com/content/pdf/10.1007%2FBF00263997.pdf</A></LI><DL><DD><UL><LI>Making keratin agarose plates 1985: <A href="http://www.sciencedirect.com/science/article/pii/0003269785902817" class="external free" rel="nofollow">http://www.sciencedirect.com/science/article/pii/0003269785902817</A></LI><LI>Making keratin powder (first step of making keratin sponges): <A href="http://www.sciencedirect.com/science/article/pii/S0142961203010846" class="external free" rel="nofollow">http://www.sciencedirect.com/science/article/pii/S0142961203010846</A></LI></UL></DD></DL><LI> Add picture of 11E in slide?
</LI><LI> linker insert (prefix/suffix), we need to miniprep it and check it on gel to see if it’s there, put in excess. If that doesn’t work, phosphorylate or titrate it
</LI><LI> NEED TO DO B. SUBTILIS TRANSFORMATION ASAP WITH PUB110 
</LI><LI> Will B. subtilis secrete KerA as the same as licheniformis
</LI><LI> Figure out another assay for keratinase secretion
</LI><LI>replica plate to filter paper
</LI><LI>ust be unhydrolyzed
</LI><LI>google “high yield purificwool assay?
</LI><LI>recipes for embedding keratin
</LI><LI>grind up protein to put in agar to see clearing zone
</LI><LI>check for cheapest source, keratin from ‘body building’ powder
</LI><LI>feather powder
</LI><LI>keratin mation of keratin from wool fiber” req. formic acid
</LI><LI> random vs. non-random mutagenesis
</LI><LI> Screening Fungi for synthesis of keratin expre., feather meal <A href="http://www.vsrdjournals.com/vsrd/Issue/2011_3_Mar/2_Tom_Sinoy_Research_Article_Mar_2011.pdf" class="external free" rel="nofollow">http://www.vsrdjournals.com/vsrd/Issue/2011_3_Mar/2_Tom_Sinoy_Research_Article_Mar_2011.pdf</A></LI><LI> Wool to keratin
</LI><LI> Find papers where they use them for sole carbon sources
</LI><H4><SPAN class="mw-headline" id="Wednesday.2C_August_7.2C_2013">Wednesday, August 7, 2013</SPAN></H4><UL><LI>Competent cells have been made!
</LI><LI>Ran 1% gel with 6 minipreps of 11E, 1G, and 4D. The results are good. Image is in Google drive.
</LI></UL><DL><DD><UL><LI>lane1: ladder (8ul)
</LI><LI>lane2: 11E mp1 (2ul)
</LI><LI>lane3: 11E mp2 (2ul)
</LI><LI>lane4: 1G mp1 (2ul)
</LI><LI>lane5: 1G mp2 (2ul)
</LI><LI>lane6: 4D mp2 (2ul)
</LI><LI>lane7: 4D mp1 (2ul)
</LI></UL></DD></DL><UL><LI>Set up pUB110, pUB110 linker digestion o/n → see 8/8 for photo of digest- Nora 
</LI><LI>Set up B. subtilis o/n culture- Ethan
</LI><LI>Ethan heat shocked Ivan’s competency cells 37ºC for 40s and then heat shocked again at 42ºC for 40s. In between heat shocks, he put it on ice. Transformed competency kit as test.-Ethan
</LI><LI>Requested Pveg + RBS part from iGEM, will arrive Tues 8/13 in agar slab, just plate and culture-Alice
</LI></UL><P>Nanodrop concentrations
</P><OL><LI> Linker gel extract: 115.2 ng/ul (definitely should be visible on gel)
</LI><LI> pUB110 gel extract, more concentrated: 4.5 ng/ul (too low)
</LI></OL><P>-Alice
</P><H4><SPAN class="mw-headline" id="Notes_from_meeting_with_Karyl">Notes from meeting with Karyl</SPAN></H4><OL><LI> Karyl suggests using quikchange to add his tag and suffix (no linker necessary)
</LI><LI>Need unmethylated and maybe use DpnI to eliminate template plasmid DNA
</LI><LI>Depending where active site is in KerA, quikchange may not work (ie if the active site is near the ends, there might be mutagenesis of the active end)
</LI><LI> If we have time, there are other peptides to consider besides the His-tag
</LI><LI> Perhaps try 3% gels for promoters
</LI><LI> Karyl says that our B. subtilis transformation protocol is okay to use
</LI></OL><H4><SPAN class="mw-headline" id="Thursday.2C_August_8.2C_2013">Thursday, August 8, 2013</SPAN></H4><UL><LI>Digesting 2.5 and 5.0 ul of pub110 (miniprep # 2) starting at 12:45 pm--will leave over the weekend.
</LI></UL><DL><DD><UL><LI>If eluted with water, can be used for PCR.
</LI></UL></DD></DL><H4><SPAN class="mw-headline" id="Friday.2C_August_9.2C_2013">Friday, August 9, 2013</SPAN></H4><P>pUB110 digest (3rd Trial)
</P><UL><LI>Digested 2.5 and 5.0 ul of pUB110 (miniprep # 2) starting at 12:45 pm. 
</LI></UL><DL><DD><UL><LI>Digestion was left in incubator over the weekend
</LI></UL></DD></DL><UL><LI>Note: for better PCR results, elute with water
</LI></UL><H3><SPAN class="mw-headline" id="Week_3_3">Week 3</SPAN></H3><H4><SPAN class="mw-headline" id="Monday.2C_August_12.2C_2013">Monday, August 12, 2013</SPAN></H4><P>Bacillus subtilis WB700 Transformation:
</P><DL><DD><UL><LI>Grew up overnight culture of B. subtilis strain WB700 in LB (to OD600 = 5.3) for transformation
</LI></UL></DD></DL><DL><DD><UL><LI>Made two 25 ml tubes of pre-transformation medium and added subtilis cells 1 and 2 ml respectively to OD .25 and .50
</LI></UL></DD></DL><DL><DD><UL><LI>note: The OD 0.5 tube fell off of the shaker some time in the first 3 hours of incubation and fell behind the 0.25 OD tube.
</LI></UL></DD></DL><TABLE class="wikitable"><TBODY><TR><TD> Time (h)
</TD><TD> OD 600 (sample 1)
</TD><TD> OD 600 (sample 1)
</TD></TR><TR><TD> 0
</TD><TD> 0.25
</TD><TD> 0.5
</TD></TR><TR><TD>4
</TD><TD>1.78
</TD><TD>0.76
</TD></TR><TR><TD> 6
</TD><TD> 2.7
</TD><TD> 1.9
</TD></TR></TBODY></TABLE><DL><DD><UL><LI>After the 6 hours we spun the cells down in two 25 ml tubes and resuspended in 5 ml of the supernatant + 1 ml (sterile) glycerol
</LI></UL></DD></DL><DL><DD><UL><LI>Took 0.2 ml of resuspended cells and add to 1 ml (per transformation) of transformation medium 
</LI></UL></DD></DL><DL><DD><UL><LI>Added DNA for transformation to tubes
</LI></UL></DD></DL><P>Transformation: 
</P><TABLE class="wikitable"><TBODY><TR><TD> Cell Sample
</TD><TD> ul of 100 ng/ul pUB110 miniprep
</TD></TR><TR><TD> 1
</TD><TD> 0 ul (- control)
</TD></TR><TR><TD> 1
</TD><TD> 1 ul
</TD></TR><TR><TD> 1
</TD><TD> 5 ul
</TD></TR><TR><TD> 2
</TD><TD> 1 ul
</TD></TR><TR><TD> 2
</TD><TD> 5 ul
</TD></TR></TBODY></TABLE><UL><LI>Then, we
</LI></UL><DL><DD><UL><LI>Froze the rest of the competent cells in liquid nitrogen in case the transformations work
</LI></UL></DD></DL><DL><DD><UL><LI>Incubated transformations for 2 hours at 37°C
</LI></UL></DD></DL><DL><DD><UL><LI>Spun down the transformation mixture, resuspended in 50-100 ul, and spreaded on kanamycin plates
</LI></UL></DD></DL><P>
Overnight digestions of kerA: 
</P><UL><LI>10 ul kerA
</LI></UL><UL><LI>1.5 ul digest buffer
</LI></UL><UL><LI>0.5 ul EcoRI (F.D.)
</LI></UL><UL><LI>0.5 ul SpeI (F.D.)
</LI></UL><UL><LI>2.5 ul H2O
</LI></UL><P>Overnight digestions of pUB110
</P><UL><LI>7.5 ul pUB110
</LI></UL><UL><LI>1.5 ul copycutter 2.1 buffer
</LI></UL><UL><LI>0.5 ul AflII (from nora)
</LI></UL><UL><LI>0.5 ul Nde I (F.D.)
</LI></UL><UL><LI>5 ul H2O
</LI></UL><H4><SPAN class="mw-headline" id="Tuesday.2C_August_13.2C_2013">Tuesday, August 13, 2013</SPAN></H4><DL><DD><UL><LI>Running out of pUB110 DNA, so we inoculated some overnights to clone some more.
</LI></UL></DD></DL><P><I>Next steps:</I></P><DL><DD><UL><LI>Miniprep of 3 tubes of pUB110 tomorrow.
</LI><LI>O/N culture of pSB1 C, and K plasmid backbones for E. coli
</LI></UL></DD></DL><H4><SPAN class="mw-headline" id="Wednesday.2C_August_14.2C_2013">Wednesday, August 14, 2013</SPAN></H4><P><I>To Do:</I></P><DL><DD><UL><LI>Transform pSB1A3 into competent DH5a cells 
</LI><LI>Miniprep pUB110 from B. subtilis O.N. culture, then digest with AflII and NdeI to get rid of the Pre gene.
</LI><LI>Make more kanR agar plates because Ethan has taken the rest for B. subtilis transformation 
</LI><LI>Make for LB broth and sterile-ly aliquot to 15ml tubes, to prevent any future large contamination issues.
</LI><LI>Need to make spreadsheet to log the progress of each BioBrick step 
</LI><LI>Figure out how to submit DNA samples for sequencing.
</LI><LI>Order primers after Karyl has checked them.
</LI></UL></DD></DL><DL><DD><UL><LI>Advisor meeting today 7pm, some questions we'd like to address:
</LI></UL></DD><DD><UL><LI>Our his-tag is all CAT, will that be an issue? 
</LI><LI>What B. subtilis strain can we borrow from Karyl that contains no knocked-out proteases?
</LI><LI>Team organization--how to split up work efficiently.
</LI><LI>Updates on putting kerA in Amp plasmid backbone and linker and pUB110 together.
</LI></UL></DD></DL><P>Ask the listhost questions if you have them and when you have answers, reply all! Comment in the lab notebook with questions and reply if you have answers!
</P><DL><DD><UL><LI>We did minipreps to isolate more pUB110 DNA from the O/N cultures and set up a digestion. Enzymes should be heat inactivated at 80C prior to ligation.
</LI><LI>We Found a Biobrick for RFP to put into B. subtilis, which we are doing to check for expression in B. subtilis. We’re using BBa_E1010 in pSB1C3, which has chloramphenicol resistance. We chose the C resistant one over the kanamycin resistant plasmid because we have more plates that are C resistant. The biobrick is located: plate 3, well 6G-Nicole.
</LI></UL></DD></DL><H4><SPAN class="mw-headline" id="Thursday.2C_August_15.2C_2013">Thursday, August 15, 2013</SPAN></H4><P><B>Digest of pUB110 and kerA-pSB1A3 Ligation</B></P><P>B.subtilis Backbone
</P><UL><LI>Ran a 1% agarose gel of 8/14’s pUB110 miniprep #2 O.N. digest, started 12pm (for eventual gel purfication)
</LI></UL><DL><DD><UL><LI>Used GelPilot loading dye, 5x from QIAGEN kit, 120V 12:47pm -
</LI><LI>Loading 7.5ul loading dye to 30ul of pUB110 digest
</LI><LI>lane 1: ladder (8ul)
</LI><LI>lane 2: pUB110 digest + dye (~19ul)
</LI><LI>lane 3: pUB110 digest + dye (~19ul)
</LI></UL></DD></DL><UL><LI>Results of the gel reveal that the digest was not successful. There was only one band around 4.5kb region, meaning no digestion occurred.
</LI></UL><UL><LI>Re-digested 8/14 pUB110 mp2 and mp1
</LI></UL><TABLE class="wikitable"><TBODY><TR><TD> Item
</TD><TD> Vol (ul)
</TD></TR><TR><TD> 2.1 buffer
</TD><TD> 3
</TD></TR><TR><TD> H20
</TD><TD> 15
</TD></TR><TR><TD> AflII Enzyme
</TD><TD> 1
</TD></TR><TR><TD> NdeI Enzyme
</TD><TD> 1
</TD></TR><TR><TD> pUB110
</TD><TD> 10
</TD></TR></TBODY></TABLE><DL><DD><UL><LI>Incubated in 37C overnight
</LI></UL></DD></DL><UL><LI>Note: Do not take the enzymes out of the freezer until you need them. Add them near the freezer. If this second digest doesn’t work, then there may be problems with the enzymes.
</LI><LI>We need more AFLII enzyme.
</LI></UL><P>Restocking
</P><UL><LI>Made 100ml stock EtBr bath and used empty tip top to hold the bath. Remember to cover in aluminum foil. 5ul EtBr/100ml ddH2O.
</LI></UL><UL><LI>Since our concentrations are so low, we need to alter our gel purification protocol:
</LI></UL><P>：*Remember to heat up elution buffer to 50C before use and let the binding buffer sit on the column for a few minutes, let the EtOH evaporate off too.
</P><DL><DD><UL><LI>Try using smaller columns and eluting twice with warm elution buffer. Take the total volume you want and divide it in half. First round of elution, let the EB sit for 10min and then spin down. And then do it again. The problem with the spin column 50 tubes is that the silica is too large compared to the volume of elution going through, so DNA tends to get struck. Flow-through is probably not the problem.
</LI></UL></DD></DL><DL><DD><UL><LI>Note: Gel purification does not work with 2% agarose gels
</LI><LI>Note ON AGAROSE GELS: It’s okay to reuse agarose gels, we just need to run the sample off. They can be re-used for a few weeks. The EtBr will need to be added to the TAE buffer in the apparatus though and re-added each time.
</LI></UL></DD></DL><P>
E.coli kerA Plasmid. 
</P><UL><LI>Ligated kerA to pSB1A3, started 12pm, and transformed into E. coli
</LI></UL><DL><DD><UL><LI>First, Heat inactivated SpeI (from 8/14’s digestion) at 80C for 20min: 12:10pm - 12:30pm
</LI></UL></DD></DL><TABLE class="wikitable"><TBODY><TR><TD> Ligase buffer (fixed)
</TD><TD> 1ul
</TD></TR><TR><TD> T4 DNA ligase (fixed)
</TD><TD> 0.5ul
</TD></TR><TR><TD> Vector (pSB1A3) 2155bp, 25ng/ul
</TD><TD> 4.7ul
</TD></TR><TR><TD> Insert (kerA) 1182bp, ~50ng/ul (assumption)
</TD><TD> 3.8ul
</TD></TR><TR><TD> Total
</TD><TD> 10 ul
</TD></TR></TBODY></TABLE><UL><LI>Incubated at RT for 20min, took 2ul for transformation of DH5a cells. 
</LI></UL><UL><LI>Incubated with competent cells 4:10pm - 5:10pm and then plated on ampR plates.
</LI></UL><UL><LI>The rest of the ligation mix is incubating for 3:30pm-6:05pm and then heat to inactivate ligase, 65C for 10min. This ligation mix can be used again if the prior transformation doesn’t work.
</LI></UL><P>
E.coli Cassettes, B.subtilis Promoter.
</P><P>
Digest from Aug 8 photo taken from 8/8 lab notebok
</P><P>lane1: ladder (8ul);  lane2: 11E (2ul);  lane3: 1G Cons cassette (2ul);  lane4: 4D Lac cassette (2ul)
</P><UL><LI>Refer to BioBricks as 4D LacCass (for Lac inducible expression cassette), 1G ConsCass (for constitutive expression cassette), 11E VegRSac (for Constitutive promoter Pveg with RBS spoVG and SacB) for labelling.
</LI></UL><UL><LI>All the bands we need are on present on the 1G ConsCass and 4D LacCass, which means the digest is good for 3A Assembly. All we need now is the kerA and pSB1A3 ligation and the primers.
</LI></UL><UL><LI>We need to double-check the sizes of the insert and see if they are on the gel
</LI></UL><P>
Paper Work
</P><P>
Things to do:
</P><UL><LI>Figure out how to send things to sequencing
</LI><LI>Fill out expense sheet for ordering primers
</LI><LI>Order primers!! Kim and Arthur are out but the expense form needs to be filled out
</LI><LI>See Arthur and talk to him about paying the conference fees
</LI><LI>Finish coding website and upload to iGEM wiki source
</LI></UL><H4><SPAN class="mw-headline" id="Friday.2C_August_16.2C_2013">Friday, August 16, 2013</SPAN></H4><UL><LI>Need more AflII enzyme to do the pUB110 digestion.
</LI><LI>Tag: B.subtilisBackbone. pUB110 gel purification of samples 1 and 2 (from mini-prep) // double checking if enzymes are working ETHAN IS RUNNING THIS
</LI><LI>Tag: E.colikerAPlasmid. kerA to pSB1A3 (ampR plasmid) O.N. cultures of 3 picked colonies (mark with sharpie on back of petri with number to note which ones you’ve picked) not now, maybe Monday, read up on VF2/VR primers instead (Happens later)
</LI><LI>Tag: B.subtilisPromoter. Agar slab, re-streak
</LI><LI>Tag: E.colikerAPlasmid, B.subtilisBackbone. Using the columns that Ethan saved, he did gel extraction with warmed EB
</LI><LI>Tag: PaperWork.
</LI></UL><DL><DD><UL><LI>Figuring out how many of and of what samples to send to sequencing (3 samples of the kerA-pSB1A3 plasmid, 3 samples of linker-pUB110, if that ever gets done)
</LI><LI>Melissa leaves office by 4pm, 
</LI><LI>Monday, we can send samples for sequencing (can we just hand them the plate--check the DNA core website for details).
</LI></UL></DD></DL><UL><LI>If someone can make a mailchimp campaign and start sending out requests for donations
</LI><LI>Isolated more pUB110: miniprep sample #3
</LI></UL><H4><SPAN class="mw-headline" id="Saturday.2C_August_17.2C_2013">Saturday, August 17, 2013</SPAN></H4><P>pUB110 linker and RBS agar stab, O.N. cultures of E. coli with kerA-pSB1A3, and Transformation of DH5a cells with RFP (for amplification)
</P><UL><LI>Alice put the linker (pre-fix and suffix ONLY for pUB110)+RBS (ribosome binding site) agar stab in the incubator.
</LI><LI>Annie made ampicillin stock 100mg/ml: Dissolved 5mg ampicillin in 50 µl of dH20. It’s in the -20ºC.
</LI><LI>Annie made 5 overnight cultures of kerA-pSB1A3 in 3ml LB and 3µl of amp stock (8/17 O.N. kerA-pSB1A3 #1-5). The colonies are labeled on the agar plate. Overnight cultures are put in the shaker in Nora’s Lab.
</LI><LI>Made and autoclaved 1L 1XLB. Ivan Poured 15 camR plates.
</LI><LI>Ivan and Alice transformed RFP into E coli. dh5α. (For purpose of amplification since we don’t have enough of it)
</LI></UL><UL><LI>Next Steps:
</LI></UL><DL><DD><UL><LI>Pick colonies from pUB110 linker-RBS agar stab and set up O.N. cultures
</LI><LI>Miniprep 3 of the O.N. kerA-pSB1A3 cultures
</LI><LI>Pick colonies from RFP transformation of DH5α E. coli
</LI></UL></DD></DL><H3><SPAN class="mw-headline" id="Week_4_3">Week 4</SPAN></H3><H4><SPAN class="mw-headline" id="Sunday.2C_August_18.2C_2013">Sunday, August 18, 2013</SPAN></H4><P>
Restocking
</P><UL><LI> Poured 12 cam and amp plates each, using 250ml of each
</LI></UL><DL><DD><UL><LI>Note: when washing anything that will touch bacteria in the future, don’t use antimicrobial soap that’s provided. wash thoroughly with ddH2O instead.
</LI></UL></DD></DL><P>Amount of antibiotics added: 
</P><TABLE class="wikitable"><TBODY><TR><TD> Amp 100ug/ml
</TD><TD> added 0.025g/250ul Amp to 250ml LB-agar
</TD></TR><TR><TD> Cam 25ug/ml
</TD><TD> added 0.0063g/125ul Cam to 250ml LB-agar
</TD></TR></TBODY></TABLE><P>
B.subtilis Backbone
</P><UL><LI>Ran an overnight digest of 8/16 pUB110 mp3 (8/18 pUB110 mp3 Digest)
</LI></UL><DL><DD><UL><LI>7.5ul nuclease-free H2O
</LI><LI>5ul pUB110 mp3
</LI><LI>1.5ul 2.1 buffer
</LI><LI>0.5ul AflII
</LI><LI>0.5ul NdeI
</LI></UL></DD></DL><UL><LI>Incubated from 5:30pm - 10:30am
</LI></UL><P>
E.coli kerA Plasmid
</P><UL><LI>The overnight cultures set up yesterday showed no growth, perhaps because the caps were tightened too much.
</LI><LI>We set up three more overnight cultures (protocol) of 8/15’s kerA-pSB1A3 transformation (8/18 O.N. kerA-pSB1A3 #6-8)
</LI></UL><DL><DD><UL><LI>Each of the three colonies were put in 3 mL LB + 3 ul 50mg/ul ampicilin at 9:45 am
</LI><LI>the caps of 8/17’s O.N. kerA-pSB1A3 cultures #3, #4, #5 were loosened in hopes that the colonies will grow. 8/17’s O.N. kerA-pSB1A3 cultures #1, #2 were discarded because there were no pellets.
</LI><LI>The cultures were checked at 6pm. However, there was still no cell growth
</LI></UL></DD><DD><UL><LI>Perhaps the colonies were not Amp resistant, and we must make new ampicillin plates.
</LI></UL></DD></DL><P>
B.substilis Promoter
</P><UL><LI>Plate Pveg + RBS if there's growth.
</LI></UL><UL><LI>The pUB110 linker and RBS agar stab was incubated yesterday at 37C overnight but no growth is visible. The plate will continue to incubate until tomorrow. If there’s no growth by tomorrow morning, we must request another agar stab.
</LI></UL><P>
B.substilis RFP
</P><UL><LI>Made three overnight cultures of the RFP BioBrick in DH5a (8/18 O.N. RFP #1, #2, #3)
</LI><LI>Each colony was put in 3 ml LB + 1.5 ul of 50mg/ul chloramphenicol on a shaker at 37C at 11:30 am 
</LI></UL><P>
Paper Work 
</P><UL><LI>making binder full of necessary info and protocols
</LI></UL><P>
Next Steps:
</P><UL><LI>Make new ampicillin plates
</LI><LI>Re-do kerA-pSB1A3 transformation on new amp plates
</LI></UL><H4><SPAN class="mw-headline" id="Tuesday.2C_August_20.2C_2013">Tuesday, August 20, 2013</SPAN></H4><P><B>kerA-pSB1A3 ligation and RFP digestion results</B></P><UL><LI>Measured the concentration of the RFP digested yesterday
</LI></UL><DL><DD><UL><LI>[8/19 RFP Digest] = 74.7 ug/ul 
</LI></UL></DD></DL><UL><LI>Ran a 2% gel of the RFP digest
</LI></UL><DL><DD><UL><LI>Lane 2: Ladder (8 ul)
</LI><LI>Lane 4: 8/19 RFP mp#2 Digest (2 ul) + loading dye (2 ul)
</LI><LI>Added 5/10 ul EtBr to the buffer
</LI></UL></DD></DL><DL><DD><UL><LI>There is only 1 band in lane 4.
</LI></UL></DD></DL><UL><LI>Email iGEM to request new Pveg+RBS construct purchased
</LI></UL><UL><LI>Ligated kerA and pSB1A3 with control experiments (Ivan)
</LI><LI>The ligations were set up as follows:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TD>uncut w/o ligase <B>Control</B></TD><TD>cut w/ ligase <B>Control</B></TD><TD>3:1
</TD><TD>5:1
</TD><TD>10:1
</TD></TR><TR><TD>Vol. Insert kerA digest 8/19 (uL)
</TD><TD>N/A
</TD><TD>N/A
</TD><TD>1
</TD><TD>1.5
</TD><TD>3
</TD></TR><TR><TD>Vol. Vector pSB1A3 digest (uL)
</TD><TD>(18E)** 1
</TD><TD>1
</TD><TD>1
</TD><TD>1
</TD><TD>1
</TD></TR><TR><TD>Vol. Buffer (uL)
</TD><TD>1
</TD><TD>1
</TD><TD>1
</TD><TD>1
</TD><TD>1
</TD></TR><TR><TD>Vol. Ligase (uL)
</TD><TD>0.5
</TD><TD>0.5
</TD><TD>0.5
</TD><TD>0.5
</TD><TD>0.5
</TD></TR><TR><TD>Vol. H2O (uL)
</TD><TD>8.5
</TD><TD>7.5
</TD><TD>6.5
</TD><TD>6
</TD><TD>5.5
</TD></TR></TBODY></TABLE><P>used 18E (e. coli promoter from plate 5) instead of pSB1A3, has plasmid <A href="http://parts.igem.org/File:PBca1020-r0040.jpg" class="external text" rel="nofollow">BBa_J61002</A> w/ AmpR
</P><UL><LI>The concentrations of the components of the ligations were as follows: 
</LI></UL><DL><DD><UL><LI>kerA 8/19 digest: 90ng/ul
</LI><LI>pSB1A3 digest: 24ng/ul
</LI><LI>18E miniprep: 963ng/ul
</LI></UL></DD></DL><UL><LI>The ligations incubated at RT overnight
</LI><LI>Another kerA-pSB1A3 ligation is being run in parallel in case the first does not work (Ethan):
</LI></UL><DL><DD><UL><LI>Incubated overnight at 4°C
</LI><LI>ligation reactions:
</LI><LI>ker A gel extraction (8/16)  = 115 ng/ul (~1 kb)
</LI><LI>ker A gel extract (8/13) = 75 ng/ul
</LI><LI>pSB1A3 digest = 25 ng/ul (~ 2 kb)
</LI></UL></DD></DL><TABLE class="wikitable"><TBODY><TR><TD>reaction
</TD><TD>1
</TD><TD>2
</TD></TR><TR><TD>kerA
</TD><TD>1 ul kerA (8/16) = 115 ng
</TD><TD>2 ul kerA (8/13) = 150 ng
</TD></TR><TR><TD>pSB1A3
</TD><TD>1 ul pSB1A3 = 25 ng
</TD><TD>0.5 ul pSB1A3 = 12.5 ng
</TD></TR><TR><TD>molar ratio (insert/vector)
</TD><TD>2.3
</TD><TD>6
</TD></TR><TR><TD>1 ul 10X ligation buffer
</TD><TD>1 ul 10X ligation buffer
</TD></TR><TR><TD>0.5 ul ligase
</TD><TD>0.5 ul ligase
</TD></TR><TR><TD>6.5 ul water
</TD><TD>6 ul water
</TD></TR></TBODY></TABLE><UL><LI>Digested (<A href="https://2013.igem.org/Team:UChicago/Protocols#Enzymatic_digestion" class="external text" rel="nofollow">Enzymatic Digestion Protocol</A>) kerA overnight (Ethan)
</LI></UL><DL><DD><UL><LI>10 ul kerA
</LI><LI>2 ul digest buffer
</LI><LI>0.5 ul EcoRI (F.D.)
</LI></UL></DD></DL><P>0.5 ul SpeI (F.D.)
7 ul H2O
</P><UL><LI>Digested pSB1A3 overnight (Ethan)
</LI></UL><DL><DD><UL><LI>5 ul pSB1A3 (linear)
</LI><LI>1 ul digest buffer
</LI><LI>0.5 ul EcoRI (F.D.)
</LI><LI>0.5 ul SpeI (F.D.)
</LI><LI>3 ul H2O
</LI></UL></DD></DL><H4><SPAN class="mw-headline" id="Wednesday.2C_August_21.2C_2013">Wednesday, August 21, 2013</SPAN></H4><P><B>Transformation with kerA-pSB1A3 with control experiments and Transformation of B. subtilis WB700 with pUB110</B></P><UL><LI>Transformed E. coli ([<A href="https://2013.igem.org/Team:UChicago/Protocols#Ivan.E2.80.99s_protocol_for_his_competent_cells_in_PCR_tubes:" class="external text" rel="nofollow">Transformation protocol</A>]) with yesterday’s kerA-pSB1A3 ligation
</LI></UL><DL><DD><UL><LI>used 8/19 kerA digest (90ng/ul), the only pSB1A3 digest (24ng/ul), and 18E miniprep (963ng/ul)
</LI><LI>incubated overnight at 37C starting 2:30pm
</LI><LI>the following experiments, including controls, were performed
</LI></UL></DD></DL><TABLE class="wikitable"><TBODY><TR><TD><B>plate/tube #</B></TD><TD>1
</TD><TD>2
</TD><TD>3
</TD><TD>4 (has small crack)
</TD><TD>5
</TD></TR><TR><TD>tube
</TD><TD>3:1
</TD><TD>5:1
</TD><TD>10:1
</TD><TD>cut vector w/ ligase - <B>control</B></TD><TD>uncut vector w/o ligase - <B>control</B></TD></TR><TR><TD>Insert (kerA digest 8/19)
</TD><TD>1ul
</TD><TD>1.5ul
</TD><TD>3ul
</TD><TD>N/A
</TD><TD>N/A
</TD></TR><TR><TD>Vector (pSB1A3 digest)
</TD><TD>1ul
</TD><TD>1ul
</TD><TD>1ul
</TD><TD>1ul
</TD><TD>(18E)** 1ul
</TD></TR><TR><TD>Buffer
</TD><TD>1ul
</TD><TD>1ul
</TD><TD>1ul
</TD><TD>1ul
</TD><TD>1ul
</TD></TR><TR><TD>Ligase
</TD><TD>0.5ul
</TD><TD>0.5ul
</TD><TD>0.5ul
</TD><TD>0.5ul
</TD><TD>0.5ul
</TD></TR><TR><TD>H2O
</TD><TD>6.5ul
</TD><TD>6ul
</TD><TD>5.5ul
</TD><TD>7.5ul
</TD><TD>8.5ul
</TD></TR></TBODY></TABLE><P>used 18E (e. coli promoter from plate 5) instead of pSB1A3, has plasmid <A href="http://parts.igem.org/File:PBca1020-r0040.jpg" class="external text" rel="nofollow">BBa_J61002</A> w/ AmpR
</P><UL><LI>About control experiments:
</LI></UL><DL><DD><UL><LI><B>Uncut vector (BBa_J61002) withOUT ligase</B>: checks for viability of competent cells and antibiotic resistance of the plasmid, as well as transformation procedure itself
</LI></UL></DD><DD><UL><LI>expect colonies
</LI><LI>since we are using a linearized plasmid, is there another uncut vector we could use that has the same antibiotic resistance so we can check the competency of our cells?
</LI></UL></DD></DL><UL><LI><B>cut vector WITH ligase</B>: shows background due to vector recircularization (and uncut vector)
</LI></UL><DL><DD><UL><LI>expect few colonies (few more than without ligase), since only the plasmids which did not get cut during the digest or the plasmids which did get cut but are able to recircularize with the ligase will be transformed
</LI><LI>this is the most important control. theoretically, there should be no colonies, but there is always some background, so as long as we have a lot more colonies on the vector with insert (actual ligation reaction) plate, it's okay.
</LI></UL></DD></DL><UL><LI>The following controls were not carried out but should be and are useful for future experiments:
</LI></UL><DL><DD><UL><LI><B>cut vector withOUT ligase</B>: shows background due to uncut vector
</LI></UL></DD><DD><UL><LI>expect few colonies, since only the plasmids which did not get cut during the digest will be transformed
</LI><LI>only do this one if you have extra cut vector, if we don't set it up it's not a big deal, since cut vector with ligase will just show the total background due to uncut vector and vector recircularization.
</LI></UL></DD></DL><UL><LI><B>insert ONLY WITH ligase</B>: checks for contamination of intact plasmid in ligation or transformation reagents
</LI></UL><DL><DD><UL><LI>expect no colonies, since if there are colonies it indicates contamination of intact plasmid carrying antibiotic resistance in reagents since only the insert was used which should not be able to recircularize into a construct which can be transformed
</LI><LI>this control is optional, but if we are concerned about contamination of reagents with plasmids this could help, so I think we should do it if there is extra insert
</LI></UL></DD></DL><UL><LI>More info on control ligation experiments can be found <A href="http://www.addgene.org/plasmid_protocols/DNA_ligation/" class="external text" rel="nofollow">here</A>.
</LI></UL><UL><LI>Analysis of yesterday’s Agarose gel of digested RFP in lane 4:
</LI></UL><DL><DD><UL><LI>the fragment is about 2 kb long
</LI><LI>The BBa_E1010 insert (RFP) is 706bp long, while the vector is 2070 bp long.
</LI><LI>It seems like we have our vector present (the approximately 2kb band in lane 4) but not our insert.
</LI></UL></DD></DL><UL><LI>To check if the BioBrick actually has RFP, plate E. coli with the plasmid with IPTG.
</LI><LI>After checking the kit again, we realized that the BioBrick we isolated from plate 3 well 6G is BBa_R0080, not BBa_E1010.
</LI></UL><UL><LI>The insert in BBa_R0080 is 149 bp, which explains why we couldn’t see it on the gel because it was run off. 
</LI></UL><UL><LI>Transformed B. subtilis (Transformation Protocol) with pUB110
</LI><LI>make O.N. cultures tomorrow in LB Medium
</LI></UL><H4><SPAN class="mw-headline" id="Thursday.2C_August_22.2C_2013">Thursday, August 22, 2013</SPAN></H4><P><B>Results of kerA-pSB1A3 transformation and control experiments and Transformation of E. coli with RFP</B></P><UL><LI>Transformation Plate Results of kerA-pSB1A3
</LI></UL><TABLE class="wikitable"><TBODY><TR><TD><B>plate/tube #</B></TD><TD>1
</TD><TD>2
</TD><TD>3
</TD><TD>4 (has small crack)
</TD><TD>5
</TD></TR><TR><TD>tube
</TD><TD>3:1
</TD><TD>5:1
</TD><TD>10:1
</TD><TD>cut vector w/ ligase - <B>control</B></TD><TD>uncut vector w/o ligase - <B>control</B></TD></TR><TR><TD>Insert (kerA digest 8/19)
</TD><TD>1ul
</TD><TD>1.5ul
</TD><TD>3ul
</TD><TD>N/A
</TD><TD>N/A
</TD></TR><TR><TD>Vector (pSB1A3 digest)
</TD><TD>1ul
</TD><TD>1ul
</TD><TD>1ul
</TD><TD>1ul
</TD><TD>(18E)** 1ul
</TD></TR><TR><TD>Buffer
</TD><TD>1ul
</TD><TD>1ul
</TD><TD>1ul
</TD><TD>1ul
</TD><TD>1ul
</TD></TR><TR><TD>Ligase
</TD><TD>0.5ul
</TD><TD>0.5ul
</TD><TD>0.5ul
</TD><TD>0.5ul
</TD><TD>0.5ul
</TD></TR><TR><TD>H2O
</TD><TD>6.5ul
</TD><TD>6ul
</TD><TD>5.5ul
</TD><TD>7.5ul
</TD><TD>8.5ul
</TD></TR><TR><TD><B>Colonies</B></TD><TD><B>&gt;10</B></TD><TD><B>&gt;10</B></TD><TD><B>&gt;10</B></TD><TD><B>0</B></TD><TD><B>0*</B></TD></TR></TBODY></TABLE><P>used 18E (e. coli promoter from plate 5) instead of pSB1A3, has plasmid <A href="http://parts.igem.org/File:PBca1020-r0040.jpg" class="external text" rel="nofollow">BBa_J61002</A> w/ AmpR
</P><UL><LI>The positive control (Tube #5) did not work, even though there should have been colonies. Perhaps there were no colonies because the 18E tube was not actually being the miniprep. The concentration did seem too high.
</LI></UL><UL><LI>Set up three <A href="https://2013.igem.org/Team:UChicago/Protocols#Making_Overnight_Cultures" class="external text" rel="nofollow">O.N. cultures</A> of kerA-pSB1A3 E. coli in 3 ml LB with ampicillin
</LI></UL><UL><LI>Transformed E. coli with RFP BBa_E1010, well 12N, plate 3 (12N3)
</LI></UL><DL><DD><UL><LI>Followed <A href="https://2013.igem.org/Team:UChicago/Protocols#Ivan.E2.80.99s_protocol_for_his_competent_cells_in_PCR_tubes:" class="external text" rel="nofollow">PCR tube competent cell protocol</A></LI><LI>Spread on plates with chloramphenicol and IPTG
</LI></UL></DD></DL><H4><SPAN class="mw-headline" id="Friday.2C_August_23.2C_2013">Friday, August 23, 2013</SPAN></H4><P><B>Sending kerA-pSB1A3 Transformants for Sequencing and overnight cultures of RFP transformants</B></P><UL><LI>Isolated Plasmid DNA by Miniprep of 8/18’s O.N. kerA-pSB1A3 cultures #4 and #6 and measured their concentrations to send for sequencing:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TD>colony
</TD><TD>ng/ul
</TD></TR><TR><TD>4
</TD><TD>150 ng/ul
</TD></TR><TR><TD>6
</TD><TD>185 ng/ul
</TD></TR></TBODY></TABLE><UL><LI>The university's DNA sequencing facility recommends that the concentration must be at least <B>size (kb)/10 = [ug/ul]</B></LI></UL><DL><DD><UL><LI>3.35kb / 10 = 0.335ug/ul → 335ng/ul
</LI><LI>The concentrations of the kerA-pSB1A3 above are too low.
</LI></UL></DD></DL><UL><LI>Instead we sent the 3:1 ligation plate (see 8/22/13) for colony picking and sequencing of 5 samples.
</LI><LI>Primers VF2/VR, which are used for sequencing, need to be at 4uM
</LI></UL><DL><DD>dilution: (100uM)(2ul)=(4uM)(50ul) ⇒ 2ul primer in 48ul dH2O
</DD></DL><UL><LI>Results of yesterday’s RFP Transformation Plate:
</LI></UL><DL><DD><UL><LI>The colonies are not red. The transformation does not seem to have worked.
</LI><LI>After 5 hours of additional incubation 11am-4pm (20hrs total), white mold started growing on the plate. 
</LI><LI>Plate is not cam resistant?
</LI></UL></DD></DL><UL><LI><A href="https://2013.igem.org/Team:UChicago/Protocols#Ivan.E2.80.99s_protocol_for_his_competent_cells_in_PCR_tubes:" class="external text" rel="nofollow">Transformed E. coli</A> again onto another cam plate, more recently made (<A href="https://2013.igem.org/Team:UChicago/Protocols#Making_Agar_Plates" class="external text" rel="nofollow">agar plates protocol</A>)
</LI></UL><DL><DD><UL><LI>RT SOC incubation 4:50 - 5:50
</LI><LI>Incubation on 37C shaker from 6 pm
</LI></UL></DD></DL><UL><LI>Set up two colonies from yesterday’s RFP transformation plate for <A href="https://2013.igem.org/Team:UChicago/Protocols#Making_Overnight_Cultures" class="external text" rel="nofollow">overnight culture</A> in 3 mL LB with chloramphenicol to double check if they are cam resistant colonies at  5:30pm.
</LI></UL><H3><SPAN class="mw-headline" id="Week_5_2">Week 5</SPAN></H3><H4><SPAN class="mw-headline" id="Monday.2C_August_26.2C_2013">Monday, August 26, 2013</SPAN></H4><P><B>pUB110 digestions, keratinase assay, and quikchange mutagenesis research</B></P><P>B.subtilis Backbone
</P><UL><LI>Used FastDigest enzymes and green dye to digest pUB110 for 1 hour
</LI><LI>Ran the pUB110 digest (8/26 pUB110 digest with FastDigest enzymes) on a 1% agarose gel:
</LI></UL><DL><DD><UL><LI>lane 1: ladder (10ul)
</LI><LI>lane 2: fastdigest (20ul)
</LI></UL></DD></DL><DL><DD><UL><LI>The pUB110 digest does not seem to have worked.
</LI><LI>Will try again tomorrow or Wednesday
</LI></UL></DD></DL><UL><LI>Expected sizes:
</LI></UL><DL><DD><UL><LI>uncut pUB110: ~4500 bp
</LI><LI>vector: ~3500 bp
</LI><LI>AflII - NdeI: ~600 bp
</LI><LI>NdeI - NdeI: ~400 bp
</LI></UL></DD></DL><P>
B.substilis RFP
</P><UL><LI>Realized that BBa_E1010 has no promoter so we shouldn’t expect RFP expression anyway. 
</LI><LI>Realized that we might not even need the RFP since the instructions for submitting a new biobrick backbone says we only need the prefix/suffix.
</LI></UL><P>
Keratinase Assay
</P><UL><LI>Finished compiling keratin plate/keratinase assay information
</LI><LI>Called Western Live Poultry and Alliance Poultry Farm to ask for chicken feathers. Will be stopping by tomorrow with garbage bags, gloves, and boots to pick up some really dirty feathers that we’ll need to wash before use.
</LI></UL><P>
Quikchange Mutagenesis
</P><UL><LI>For the optimization of keratinase activity
</LI><LI>Quikchange mutagenesis protocols to look at:
</LI></UL><OL><LI><A href="http://openwetware.org/wiki/Site-directed_mutagenesis" class="external free" rel="nofollow">http://openwetware.org/wiki/Site-directed_mutagenesis</A></LI><LI><A href="http://www.stanford.edu/~loening/protocols/Site_Directed_Mutagenesis.pdf" class="external free" rel="nofollow">http://www.stanford.edu/~loening/protocols/Site_Directed_Mutagenesis.pdf</A> (w/ Pfu)
</LI></OL><OL><LI><A href="https://www.neb.com/products/e0554-q5-site-directed-mutagenesis-kit" class="external free" rel="nofollow">https://www.neb.com/products/e0554-q5-site-directed-mutagenesis-kit</A></LI><LI><A href="https://www.neb.com/protocols/2012/08/29/protocol-for-q5-high-fidelity-2x-master-mix-m0492" class="external free" rel="nofollow">https://www.neb.com/protocols/2012/08/29/protocol-for-q5-high-fidelity-2x-master-mix-m0492</A></LI></OL><UL><LI>Compiled a protocol for <A href="https://2013.igem.org/Team:UChicago/Protocols#Reusing_Electroporation_Cuvettes" class="external text" rel="nofollow">re-using cuvettes</A>.
</LI></UL><H4><SPAN class="mw-headline" id="Tuesday.2C_August_27.2C_2013">Tuesday, August 27, 2013</SPAN></H4><P><B>Troubleshooting pUB110 Digest</B></P><UL><LI>FastDigest enzyme digest may not have worked because we used 1.5/2.1 buffer instead of FastDigest buffer
</LI><LI>We also used too much of the miniprep (5 ul instead of 1.5 ul)when we should have used less.
</LI></UL><H4><SPAN class="mw-headline" id="Wednesday.2C_August_28.2C_2013">Wednesday, August 28, 2013</SPAN></H4><P><B> pUB110 digestion and B. subtilis electroporation</B></P><UL><LI>Bacillus subtilis electroporation: following protocols stated below
</LI></UL><DL><DD><UL><LI>Grew wb700 overnight to OD600 = 4.1 and 7.4
</LI><LI>Started two 10 ml cultures (@ 12 pm) with 
</LI></UL></DD><DD><UL><LI>OD600 = 4.1 (500 ul) and 
</LI><LI>OD600= 1.3 (750 ul) 
</LI><LI>x to OD600 = 0.2 (OD after dilution)
</LI></UL></DD></DL><UL><LI>4.1 * (0.05) = .205 OD
</LI><LI>1.3 * (0.075) = 0.0975 OD ???? too low
</LI><LI>subtilis doubling time = 25-40 minutes
</LI></UL><DL><DD><UL><LI>1:30→ 0.4 abs
</LI><LI>2:30→ 0.85 abs and 1.0 abs
</LI></UL></DD></DL><UL><LI>Electroporated at 5:30pm, following Eppendorf and its original “High osmolarity improves the electro-transformation efficiency of the gram-positive bacteria Bacillus subtilis and Bacillus licheniformis” (Xue et al. 1999)
</LI></UL><UL><LI>Measured concentration of pBU110 miniprep: [pUB110 miniprep 1] = 367ng/ul sample
</LI><LI>Digested pUB110 overnight with FastDigest enzymes: incubated 6pm
</LI></UL><DL><DD><UL><LI>1.5 ul pUB110
</LI><LI>1.5 ul FastDigest buffer
</LI><LI>0.5 ul AflII
</LI><LI>0.5 ul NdeI
</LI><LI>11 ul nuclease-free H2O
</LI></UL></DD></DL><UL><LI>Next Steps:
</LI></UL><DL><DD><UL><LI>Run 1% agarose gel of 8/28 pUB110 FastDigest and 1 ul of undigested miniprep
</LI></UL></DD></DL><H4><SPAN class="mw-headline" id="Thursday.2C_August_29.2C_2013">Thursday, August 29, 2013</SPAN></H4><P><B>pUB110 digest and Results of B. subtilis electroporation</B></P><UL><LI>8/28 pUB110 O.N. digest with FastDigest enzymes: incubated 6pm-11:30am
</LI></UL><DL><DD><UL><LI>1.5ul (miniprep 1: 367ng/ul sample) pUB110
</LI><LI>1.5ul FastDigest buffer
</LI><LI>0.5ul AflII
</LI><LI>0.5ul NdeI
</LI><LI>11ul nuclease-free H2O
</LI></UL></DD></DL><UL><LI>Ran a 1% agarose gel of 8/28 pUB110 digest (11:45 am -  at 120 V)
</LI></UL><DL><DD><UL><LI>lane 1: previously failed digest from 8/27 (5ul dye + 15ul sample)
</LI><LI>lane2: 2 log ladder (10ul)
</LI><LI>lane3: 8/28 pUB110 digest (15ul)
</LI><LI>lane4: miniprep pUB110 (1ul dye + 1ul sample)
</LI></UL></DD></DL><UL><LI>Gel purified (<A href="https://2013.igem.org/Team:UChicago/Protocols#Gel_extraction_with_the_QIAGEN_kit" class="external text" rel="nofollow">Qiagen gel extraction protocol</A>) the lower two bands: grouped them together into tubes called Upper Band (UB) and Lower Band pUB110 (LB) digest
</LI></UL><DL><DD><UL><LI>Lane 1: Ladder
</LI><LI>Lane 2: 8/28 pUB110 digest
</LI></UL></DD></DL><DL><DD><UL><LI>Measured the concentrations with nanodrop:
</LI></UL></DD><DD><UL><LI>LB: 3.2ng/ul
</LI><LI>UB: 5.2ng/ul
</LI></UL></DD></DL><UL><LI>pUB110 re-digest (with FastDigest conditions, same as 8/26/13)
</LI></UL><DL><DD><UL><LI>used 2ul pUB110 sample with fastdigest conditions, incubated 1hr at 37C (total volume 20ul)
</LI><LI>measured concentrations of digest: 8.1 ng/ul
</LI></UL></DD></DL><P>E.coli kerA Plasmid
</P><UL><LI>Troubleshooting kerA-pSB1A3 ligations and transformations: 
</LI></UL><DL><DD><UL><LI>try using phosphatase to minimize self-ligations
</LI><LI>next digest: scale up 30ul volume, making sure to use water bath in heat block, do it with controls as well (single enzyme and etc)
</LI></UL></DD></DL><H4><SPAN class="mw-headline" id="Friday.2C_August_30.2C_2013">Friday, August 30, 2013</SPAN></H4><P><B>kerA-pSB1A3 Sequencing Results</B></P><P>E.colikerAPlasmid
</P><UL><LI>We recieved our sequencing results of our kerA-pSB1A3 colonies
</LI><LI>When we analyzed the sequences for alignment with our kerA DNA sequence, it was apparent that our kerA biobrick was not ligated into the plasmid.
</LI></UL><DIV class="dividerNotebook"><P><A href="#top">[top]</A></P></DIV><H2><SPAN class="mw-headline" id="September">September</SPAN></H2><H3><SPAN class="mw-headline" id="Week_1_3">Week 1</SPAN></H3><H4><SPAN class="mw-headline" id="Tuesday.2C_September_3.2C_2013">Tuesday, September 3, 2013</SPAN></H4><P><B>Promoter+RFP transformation, kerA-pSB1A3 ligation and digestion, kerA digestion, pSB1A3 digestion</B></P><P>
B.substilisRFP
</P><UL><LI>Transformed E. coli with BBa_J23100 (constitutive promoter family + RFP) for pSB1A3 backbone
</LI></UL><DL><DD><UL><LI>Followed <A href="https://2013.igem.org/Team:UChicago/Protocols#Ivan.E2.80.99s_protocol_for_his_competent_cells_in_PCR_tubes:" class="external text" rel="nofollow">transformation protocol for E. coli PCR tubes</A></LI><LI>For making stock pSB1A3 backbone and for trying ligation without gel extraction
</LI><LI>Plates dried in the incubator, but <A href="http://www.protocol-online.org/biology-forums-2/posts/6843.html" class="external text" rel="nofollow">bubbles at bottom</A> appeared, which could imply contamination
</LI><LI>Incubation at 37C overnight
</LI><LI>The following morning there were no colonies. 
</LI><LI>Must try transformation again except with control experiments.
</LI></UL></DD></DL><P>
E.coli kerA Plasmid
</P><UL><LI>Perhaps we should treat the plasmid with antarctic phosphatase to prevent self-ligation
</LI><LI>Digested kerA and pSB1A3 (<A href="https://2013.igem.org/Team:UChicago/Protocols#Enzymatic_digestion" class="external text" rel="nofollow">Ezymatic Digestion Protocol</A>) overnight at 37°C
</LI></UL><TABLE class="wikitable"><TBODY><TR><TD>Reaction
</TD><TD>1
</TD><TD>2
</TD></TR><TR><TD>6 ul linearized pSB1A3 backbone = 150 ng
</TD><TD>2.5 ul kerA (408 ng/ul) = 800 ng DNA
</TD></TR><TR><TD>1.5 ul buffer
</TD><TD>1.5 ul buffer
</TD></TR><TR><TD>0.5 ul EcoRI
</TD><TD>0.5 ul EcoRI
</TD></TR><TR><TD>0.5 SpeI
</TD><TD>0.5 SpeI
</TD></TR><TR><TD>7.5 ul water
</TD><TD>7.5 ul water
</TD></TR></TBODY></TABLE><UL><LI>Next Steps:
</LI></UL><DL><DD><UL><LI>kerA-pSB1A3 ligations and transformation of E. coli
</LI><LI>Start O.N. cultures of RFP transformations and cut amp concentration to 50 ug/ul
</LI></UL></DD></DL><H4><SPAN class="mw-headline" id="Wednesday.2C_September_4.2C_2013">Wednesday, September 4, 2013</SPAN></H4><P><B>Results of RFP Transformation, kerA-pSB1A3 ligations and transformation of E. coli, and testing transformation efficiency of E. coli cells</B></P><P>
E.coli kerA Plasmid
</P><UL><LI>Ligated yesterday’s digested kerA and pSB1A3 overnight at 4°C (<A href="https://2013.igem.org/Team:UChicago/Protocols#Ligation_Protocol" class="external text" rel="nofollow">Ligation Protocol</A>)
</LI></UL><DL><DD><UL><LI>9 ul kerA digest (40 ng/ul)
</LI><LI>1 ul pSB1A3 digest (12.5 ng/ul)
</LI><LI>1.5 ul buffer
</LI><LI>0.5 ul ligase
</LI><LI>3 ul water
</LI></UL></DD></DL><P>
B.substilis RFP
</P><UL><LI>BBa_J23100 RFP transformation resulted in no colonies. Must try again.
</LI><LI>Other biobricks with AmpR plasmids we can try transformations with:
</LI></UL><DL><DD><UL><LI>J23119 18A, plate 5 (in pSB1A3)
</LI><LI>J23100 18C, plate 5 re-transform (in BBa_J61002)
</LI><LI><A href="http://parts.igem.org/Part:BBa_J23101" class="external text" rel="nofollow">J23101</A> 18E, plate 5 (in BBa_J61002)
</LI></UL></DD></DL><P>
kerA BioBrick Troubleshooting
</P><UL><LI>Perhaps, there is something wrong with our kerA?
</LI><LI>Redid kerA biobrick PCR (gibson assembly with geneblocks)
</LI></UL><UL><LI>Measured concentrations of kerA biobrick PCR with nanodrop:
</LI></UL><DL><DD><UL><LI>kerA biobrick PCR #1: 475ng/ul
</LI><LI>kerA biobrick PCR #2: 414ng/ul
</LI><LI>kerA digest 8/21: 50ng/ul
</LI></UL></DD></DL><UL><LI>Tested transformation efficiency of our competent cells with iGEM’s <A href="http://parts.igem.org/Help:Transformation_Efficiency_Kit" class="external text" rel="nofollow">Transformation Efficiency Kit</A>: using 5pg/ul, 10pg/ul, 20pg/ul
</LI></UL><UL><LI>Dephosphorylated restriction digests using antarctic phosphorylase:
</LI></UL><DL><DD><UL><LI>15 ul kerA/pSB1A3 digest (9/3)
</LI><LI>1.7 ul buffer
</LI><LI>0.5 ul phosphorylase
</LI><LI>Incubated at 37°C (3:30-4:00 pm)
</LI><LI>Heat inactivated enzyme at 70°C (5 minutes)
</LI></UL></DD></DL><UL><LI>Ligated dephosphorylated pSB1A3 and kerA digests and incubated overnight at 4C
</LI></UL><TABLE class="wikitable"><TBODY><TR><TD>1
</TD><TD>2
</TD><TD>3
</TD></TR><TR><TD>2 ul pSB1A3 Digest (18 ng)
</TD><TD>2 ul pSB1A3 Digest (18 ng)
</TD><TD>2 ul pSB1A3 Digest (18 ng)
</TD></TR><TR><TD>4 ul kerA (188 ng)
</TD><TD>8 ul kerA
</TD><TD>1 ul buffer
</TD></TR><TR><TD>1 ul buffer
</TD><TD>1.05 ul buffer
</TD><TD>0.5 ul digest
</TD></TR><TR><TD>0.5 ul ligase
</TD><TD>0.5 ul ligase
</TD><TD>6.5 ul H2O
</TD></TR><TR><TD>2.5 ul H2O
</TD></TR></TBODY></TABLE><UL><LI>Set <A href="https://2013.igem.org/Team:UChicago/Protocols#Making_Overnight_Cultures" class="external text" rel="nofollow">overnight culture (protocol)</A> of B. subtilis for electroporation tomorrow
</LI><LI>Autoclaved materials and tips and Made more chloramphenicol plates (<A href="https://2013.igem.org/Team:UChicago/Protocols#Making_Agar_Plates" class="external text" rel="nofollow">protocol</A>)
</LI></UL><UL><LI>Transformed kerA-pSB1A3 ligation into DH5Alpha E. coli
</LI></UL><DL><DD><UL><LI>Following <A href="https://2013.igem.org/Team:UChicago/Protocols#Ivan.E2.80.99s_protocol_for_his_competent_cells_in_PCR_tubes:" class="external text" rel="nofollow">PCR tube transformation protocol</A></LI></UL></DD></DL><OL><LI>1ul DNA into thawed competent cells on ice per tube (3 sets of 3 tubes)
</LI><LI>Incubate set #1, 2 on ice for 30min, set #3 on ice for 10min
</LI><LI>Heat shock in PCR machine 45s at 42C
</LI><LI>Recover on ice for 5min
</LI><LI>Add to 1mL LB and incubate on shaker for 37C for 1hr
</LI><LI>Spin down at 8rpm for 30s and plate on Cam plates
</LI></OL><H4><SPAN class="mw-headline" id="Thursday.2C_September_5.2C_2013">Thursday, September 5, 2013</SPAN></H4><P><B> Results of Transformation efficiency, B. subtilis electroporation (2nd Trial), and RFP transformation</B></P><UL><LI>Results of RFP Control with Transformation Efficiency Kit testing PCR competent cells
</LI></UL><DL><DD><UL><LI>Negative Control Plage
</LI></UL></DD></DL><DL><DD><UL><LI>There weren't any colonies on the negative plate, and we got colonies on the positive plates.
</LI><LI>We can conclude that the competent cells worked. 
</LI><LI>However, the number of colonies from plate to plate for the same concentrations weren't consistent. That may be due to inconsistent volume of competent cells in the PCR tubes.
</LI></UL></DD></DL><UL><LI>Electroporation of B. subtilis WB700 (2nd Trial): following &quot;High osmolarity improves the electro-transformation efficiency of the gram-positive bacteria Bacillus subtilis and Bacillus licheniformis&quot; (Xue et al. 1999)
</LI></UL><DL><DD><UL><LI>Calculating Doubling Time Constant
</LI></UL></DD></DL><DL><DD><UL><LI>Average doubling time = 1.156 
</LI><LI>~69min doubling time from 8/28 OD measurements, using the 4.1OD → 0.205OD → 0.85OD or 1OD, taking ave
</LI></UL></DD></DL><UL><LI><U>Doubling time (formula from wikipedia)</U></LI></UL><P>N(t) = 0.85abs
C = 0.133abs (see below)
d = 1.156
t = 1.156 * [log(.85/.133)/log2] = <B><U>3.09 hrs</U></B></P><DL><DD><I>rough estimate of doubling time to double check above figure:</I></DD><DD>need for 0.9nm OD
</DD><DD>3.46nm/26ml
</DD><DD>starting OD calculated at 0.133 (w/o dilution)
</DD><DD> x2 = 0.266
</DD><DD> x2 = 0.532
</DD><DD> x2 = 1.064 ~roughly
</DD></DL><P><B>O/N culture OD (dilution)</B>: 0.346nm (1:10 dilution in water, 100ul in 900ul water)
original OD in 15ml tube is 3.46nm
</P><P><B>Diluted Initial OD (w/o dilution)</B>: 0.133nm (1ml each in 25ml sorbitol and 25ml LB)
</P><P><B>Desired OD (w/o dilution)</B>: 0.85nm
</P><TABLE class="wikitable"><TBODY><TR><TD>0.5M Sorbitol in LB, 25 mL
</TD><TD>LB, 25 mL
</TD></TR><TR><TD>10am-11:30am
</TD><TD>0.133 nm
</TD><TD>0.133 nm
</TD></TR><TR><TD>11:30 am
</TD><TD>0.031*10 = 0.31 nm
</TD><TD>0.064*10 = 0.64 nm
</TD></TR><TR><TD>12-12:20 pm
</TD><TD>N/A
</TD><TD>0.104nm*10 = 1.04 nm (used Ethan's spec)
</TD></TR><TR><TD>12-1:35pm
</TD><TD>0.111nm*10 = 1.11 nm (used Nora's spec)
</TD><TD>N/A
</TD></TR></TBODY></TABLE><P><B>LB only</B></P><P>N(t) = 0.85abs
</P><P>C = 0.64abs (see below)
</P><P>d = 0.662
</P><P>t = 0.662 * [log(.85/.64)/log2] = 0.27 *60min = <B>16min → 20min</B></P><P>[copy, paste into google search] log(.85/.64)/log(2) *0.662
</P><P><B>LB 0.5M sorbitol</B></P><P>N(t) = 0.85abs
</P><P>C = 0.31abs (see below)
</P><P>d = 1.229
</P><P>t = 1.229 * [log(.85/.31)/log2] = 1.79 * 60min = <B>1hr 47min → 1hr 50min</B></P><P>[copy, paste into google search] log(.85/.31)/log(2) *1.229
</P><P><A href="https://2013.igem.org/Team:UChicago/Protocols#B._subtilis_Electroporation_Protocol" class="external text" rel="nofollow">B. subtilis Electroporation Protocol</A> from Xue et al. 1999:
</P><OL><LI>Cool cells in 50ml Falcon tubes in ice-water, 10min
</LI><LI>Spin down 3,000rpm for 5min (Nora’s lab)
</LI><LI>Resuspend in 650ul electroporation medium and transfer to 1.5ml tubes
</LI><LI>Spin down 5,000xg for 5min (BSLC lab)
</LI><LI>Wash 3x with electroporation medium (500ul)
</LI><LI>Resuspend in 1/40 of culture volume of electroporation medium
</LI></OL><DL><DD><UL><LI>resuspend in 26/40 = 650ul
</LI></UL></DD></DL><DL><DD><UL><LI>dilution pUB110 #1 244ng/ul: (244ng/ul)(x ul) = (10ng/ul)(200ul) → x = <B>8.2ul</B></LI></UL></DD></DL><P>in 0.1cm gap
</P><P><B>Re-using Cuvettes</B>: (<A href="https://2013.igem.org/Team:UChicago/Protocols#Reusing_Electroporation_Cuvettes" class="external text" rel="nofollow">protocol</A>)
500ul H2O mix up and down <B>3x</B>, EtOH mix up and down with 10ul pipette, make sure you see flakes of dead cells on the first mixing, <B>2x</B>, pour EtOH out as much as possible and air dry for 10min, wipe cap with EtOH too.
</P><TABLE class="wikitable"><TBODY><TR><TD><B>number</B></TD><TD><B>contains</B></TD><TD><B>kV</B></TD><TD><B>time constant</B></TD><TD><B>notes</B></TD></TR><TR><TD>1
</TD><TD>LBS + DNA
</TD><TD>2.0
</TD><TD>4.08
</TD><TD>no arc
</TD></TR><TR><TD>2
</TD><TD>LB + DNA
</TD><TD>2.0
</TD><TD>N/A
</TD><TD>didn't wash after initial use, arc'd, immediately filled with EtOH and set aside
</TD></TR><TR><TD>3
</TD><TD>LBS + DNA
</TD><TD>2.1
</TD><TD>3.96
</TD><TD>new cuvette
</TD></TR><TR><TD>4
</TD><TD>LBS + DNA
</TD><TD>2.0
</TD><TD>1.70
</TD><TD>incubated sample to see what hapens; washed with 1mL H2O 3x, lots of EtOH 2x, dried out carefully, on plain plate
</TD></TR><TR><TD>5
</TD><TD>LBS + DNA
</TD><TD>2.0
</TD><TD>4.16
</TD><TD>no arc
</TD></TR><TR><TD>6
</TD><TD>LB + DNA
</TD><TD>2.1
</TD><TD>3.06
</TD><TD>washed 2x H2O instead of 3x, wasn't as careful, arc'd, incubated
</TD></TR><TR><TD>7
</TD><TD>LB + DNA
</TD><TD>2.1
</TD><TD>4.2
</TD><TD>washed carefully, no arc
</TD></TR><TR><TD>8
</TD><TD>LB (N)
</TD><TD>2.1
</TD><TD>4.2
</TD><TD>washed carefully, no arc
</TD></TR></TBODY></TABLE><P>Incubated tubes at 6:40pm-9:40pm. Incubated plates at 10pm.
</P><UL><LI>Plated on 50ug/ml Kan plates → may be issue since protocol recommends 20ug/ml Kan plates?
</LI></UL><H4><SPAN class="mw-headline" id="Friday.2C_September_6.2C_2013">Friday, September 6, 2013</SPAN></H4><P><B>Results of B. subtilis electroporation and kerA-pSB1A3 ligations (all trials)</B></P><P>E.coli kerA Plasmid
</P><UL><LI>Ran a <A href="https://2013.igem.org/Team:UChicago/Protocols#Making_a_1.25_agarose_gel_for_gel_purification" class="external text" rel="nofollow">1% agarose gel</A> of kerA ligations to see if the ligations actually worked
</LI></UL><DL><DD><UL><LI>lane1: ladder (10ul)
</LI><LI>lane2: kerA pcr (0.5ul of 457ng/ul in 5ul)
</LI><LI>lane3: kerA-pSB1A3 ligation  8/15(5ul) [ looks like there some ligation product, but transformation didn’t work ]
</LI><LI>lane4: cut vector w/ ligase 8/20 (8ul)
</LI><LI>lane5: 3:1 ligation 8/20 (8ul)
</LI><LI>lane6: 5:1 ligation 8/20 (8ul)
</LI><LI>lane7: 10:1 ligation 8/20 (8ul)
</LI></UL></DD><DD> kerA-pSB1A3 ligation digest 8/20 (3:1 ratio tube)
</DD><DD> kerA size: ~1199 bp
</DD><DD> pSB1A3 size: ~2155 bp
</DD><DD> expected size: = ~3354 bp
</DD></DL><P>
B.subtilis Backbone
</P><UL><LI>Made <A href="https://2013.igem.org/Team:UChicago/Protocols#Making_Overnight_Cultures" class="external text" rel="nofollow">overnight cultures</A> of B. subtilis BD 366 in 3 mL LB with kanamycin to isolate pUB110 (from 7pm - 1pm)
</LI></UL><UL><LI>B. subtilis Electroporation plates:
</LI></UL><UL><LI>There were no colonies on the negative plate, while there is only one colony on the other plates. It was incubated for longer, and restreaked the following day.
</LI></UL><UL><LI>Transformed (<A href="https://2013.igem.org/Team:UChicago/Protocols#Ivan.E2.80.99s_protocol_for_his_competent_cells_in_PCR_tubes:" class="external text" rel="nofollow">protocol</A>) E. coli with (three) kerA-pSB1A3 ligations and (two) RFP biobrick
</LI></UL><DL><DD><UL><LI>incubated 30 min on ice
</LI><LI>plated on Amp plates
</LI></UL></DD></DL><H3><SPAN class="mw-headline" id="Week_2_3">Week 2</SPAN></H3><H4><SPAN class="mw-headline" id="Monday.2C_September_9.2C_2013">Monday, September 9, 2013</SPAN></H4><P><B>O.N. cultures of past RFP biobricks, kerA-pSB1A3 ligations, and electroporations; Prolonged incubation of electroporations; and Re-streaking RFP biobricks, electroporations, and most recent kerA-pSB1A3 ligation</B></P><UL><LI>At 5pm set up O.N. cultures of:
</LI></UL><TABLE class="wikitable"><CAPTION>overnight cultures 9/09
</CAPTION><TBODY><TR><TD>cam: RFP 8/22
</TD><TD>overnight culture of a RFP biobrick in pSB1C3 backbone
</TD></TR><TR><TD>amp: RFP1 9/6
</TD><TD>overnight culture of a RFP biobrick in non-pSB1A3 backbone
</TD></TR><TR><TD>amp: RFP2 9/6
</TD><TD>overnight culture of a RFP biobrick in pSB1A3 backbone
</TD></TR><TR><TD>amp: 10:1 ligation #1
</TD><TD>old ligation plates, from 8/20, colony 1
</TD></TR><TR><TD>amp: 10:1 ligation #2
</TD><TD>old ligation plates, from 8/20, colony 2
</TD></TR><TR><TD>amp: 5:1 ligation #1
</TD><TD>old ligation plates, from 8/20
</TD></TR><TR><TD>amp: 5:1 ligation #2
</TD><TD>old ligation plates, from 8/20
</TD></TR><TR><TD>amp: 9/6 L2
</TD><TD>Ethan’s most recent ligation pSB1A3-kerA, only plate with colonies
</TD></TR><TR><TD>kan: AY #3 EP #1
</TD><TD>one of my electroporation plates, colony 1
</TD></TR><TR><TD>kan: AY #3 EP #2
</TD><TD>one of Alice's electroporation plates, colony 2
</TD></TR><TR><TD>kan: EB N EP
</TD><TD>Ethan’s electroporation plates, negative control
</TD></TR><TR><TD>kan: EB 10ng EP #1
</TD><TD>Ethan’s electroporation plates, negative control, used 10ng of plasmid DNA, colony 1
</TD></TR><TR><TD>kan: EB 10ng EP #2
</TD><TD>Ethan’s electroporation plates, negative control, used 10ng of plasmid DNA, colony 2
</TD></TR><TR><TD>kan: EB 10ng EP #3
</TD><TD>Ethan’s electroporation plates, negative control, used 10ng of plasmid DNA, colony 3
</TD></TR><TR><TD>kan: EB 50ng EP #1
</TD><TD>Ethan’s electroporation plates, negative control, used 50ng of plasmid DNA, colony 1
</TD></TR><TR><TD>kan: EB 50ng EP #2
</TD><TD>Ethan’s electroporation plates, negative control, used 50ng of plasmid DNA, colony 2
</TD></TR><TR><TD>kan: EB 50ng EP #3
</TD><TD>Ethan’s electroporation plates, negative control, used 50ng of plasmid DNA, colony 3
</TD></TR><TR><TD>kan: EB 4x EP
</TD><TD>Ethan’s electroporation plates, most recent plate that had colonies - did four overnight cultures
</TD></TR></TBODY></TABLE><UL><LI>Incubated the following plates longer from 10:30 am and checked again at 4 pm:
</LI></UL><TABLE class="wikitable"><CAPTION>plate incubated longer
</CAPTION><TBODY><TR><TD>kan: AY #1 EP
</TD><TD>one of Alice’s electroporation plates, wanted to see if more colonies would grow
</TD></TR><TR><TD>kan: AY #3 EP
</TD><TD>one of Alice’s electroporation plates, wanted to see if more colonies would grow
</TD></TR><TR><TD>kan: EB 10ng EP
</TD><TD>Ethan’s electroporation plates, wanted to see if more colonies would grow
</TD></TR><TR><TD>kan: EB 50ng EP
</TD><TD>Ethan’s electroporation plates, wanted to see if more colonies would grow
</TD></TR><TR><TD>kan: EB N EP
</TD><TD>Ethan’s electroporation plates, wanted to see if more colonies would grow
</TD></TR></TBODY></TABLE><UL><LI>Restreaked and incubated until 9pm:
</LI></UL><TABLE class="wikitable"><CAPTION>re-streaked
</CAPTION><TBODY><TR><TD>amp: RFP1 9/6
</TD><TD>From BBa_J23100, non-pSB1A3 backbone, from Chloe’s RIBS class’ BioBrick used, original plate was messy plating
</TD></TR><TR><TD>amp: RFP2 9/6
</TD><TD>From BBa_J23119, pSB1A3 backbone, from BioBrick kit, original plate was messy plating
</TD></TR><TR><TD>kan: AY #3 EP
</TD><TD>plate number 3, my electroporation plate with colonies; replated to try to get more colonies
</TD></TR><TR><TD>amp: 9/6 L2
</TD><TD>Ethan’s most recent ligation pSB1A3-kerA, only plate with colonies
</TD></TR></TBODY></TABLE><UL><LI>The feathers we ordered are here! Must pick them up tonight and we can make our feather keratinase assay.
</LI></UL><UL><LI>Made chloramphenicol plates 250 mL in total (<A href="https://2013.igem.org/Team:UChicago/Protocols#Making_Agar_Plates" class="external text" rel="nofollow">protocol</A>)
</LI></UL><UL><LI>Ligated kerA and antarctic phosphatase treated pSB1A3 and incubated overnight at 4°C (<A href="https://2013.igem.org/Team:UChicago/Protocols#Ligation_Protocol" class="external text" rel="nofollow">protocol</A>):
</LI></UL><TABLE class="wikitable"><TBODY><TR><TD>1
</TD><TD>2
</TD><TD>3
</TD><TD>4
</TD><TD>5
</TD></TR><TR><TD>1 ul pSB1A3 Digest (9 ng)
</TD><TD>1 ul pSB1A3 digest (9 ng)
</TD><TD>1 ul pSB1A3 digest (9 ng)
</TD><TD>1 ul pSB1A3 digest (9 ng)
</TD><TD>1 ul pSB1A3 digest (9 ng)
</TD></TR><TR><TD>0.5 ul ul KerA (220 ng)
</TD><TD>8 ul KerA (188 ng)
</TD><TD>1.05 ul buffer
</TD><TD>1 ul buffer
</TD><TD>1 ul buffer
</TD></TR><TR><TD>1 ul buffer
</TD><TD>1 ul buffer
</TD><TD>0.5 ul digest
</TD><TD>0.5 ul digest
</TD><TD>0.5 ul digest
</TD></TR><TR><TD>0.5 ul ligase
</TD><TD>0.5 ul ligase
</TD><TD>6.5 ul h2o
</TD><TD>6.5 ul h2o
</TD><TD>6.5 ul h2o
</TD></TR><TR><TD>2.5 ul h2o
<P>total DNA = 22 ng/ul
</P></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Tuesday.2C_September_10.2C_2013">Tuesday, September 10, 2013</SPAN></H4><P><B>Miniprep of 9/09’s O.N. cultures</B></P><UL><LI>Isolated plasmid DNA of yesterday’s O.N. cultures (see below) by <A href="https://2013.igem.org/Team:UChicago/Protocols#Miniprep_Protocols" class="external text" rel="nofollow">miniprep</A></LI></UL><TABLE class="wikitable"><CAPTION> 9/10/13 Minipreps
</CAPTION><TBODY><TR><TD>cam: RFP 8/22
</TD><TD>overnight culture of a RFP biobrick in pSB1C3 backbone
</TD></TR><TR><TD>amp: RFP1 9/6
</TD><TD>overnight culture of a RFP biobrick in non-pSB1A3 backbone
</TD></TR><TR><TD>amp: RFP2 9/6
</TD><TD>overnight culture of a RFP biobrick in pSB1A3 backbone
</TD></TR><TR><TD>amp: 10:1 ligation #1
</TD><TD>old ligation plates, from 8/20, colony 1
</TD></TR><TR><TD>amp: 10:1 ligation #2
</TD><TD>old ligation plates, from 8/20, colony 2
</TD></TR><TR><TD>amp: 5:1 ligation #1
</TD><TD>old ligation plates, from 8/20
</TD></TR><TR><TD>amp: 5:1 ligation #2
</TD><TD>old ligation plates, from 8/20
</TD></TR><TR><TD>amp: 9/6 L2
</TD><TD>Ethan’s most recent ligation pSB1A3-kerA, only plate with colonies
</TD></TR><TR><TD>kan: AY #3 EP #1
</TD><TD>one of Alice's electroporation plates, colony 1
</TD></TR><TR><TD>kan: AY #3 EP #2
</TD><TD>one of my electroporation plates, colony 2
</TD></TR><TR><TD>kan: EB N EP
</TD><TD>Ethan’s electroporation plates, negative control
</TD></TR><TR><TD>kan: EB 10ng EP #1
</TD><TD>Ethan’s electroporation plates, negative control, used 10ng of plasmid DNA, colony 1
</TD></TR><TR><TD>kan: EB 10ng EP #2
</TD><TD>Ethan’s electroporation plates, negative control, used 10ng of plasmid DNA, colony 2
</TD></TR><TR><TD>kan: EB 10ng EP #3
</TD><TD>Ethan’s electroporation plates, negative control, used 10ng of plasmid DNA, colony 3
</TD></TR><TR><TD>kan: EB 50ng EP #1
</TD><TD>Ethan’s electroporation plates, negative control, used 50ng of plasmid DNA, colony 1
</TD></TR><TR><TD>kan: EB 50ng EP #2
</TD><TD>Ethan’s electroporation plates, negative control, used 50ng of plasmid DNA, colony 2
</TD></TR><TR><TD>kan: EB 50ng EP #3
</TD><TD>Ethan’s electroporation plates, negative control, used 50ng of plasmid DNA, colony 3
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Wednesday.2C_September_11.2C_2013">Wednesday, September 11, 2013</SPAN></H4><P><B>pUB110 digestion and gel analysis</B></P><UL><LI>digested pUB110 minipreps with Nde I to test if electroporation was successful
</LI><LI>Ran a 1% agarose gel of the digest
</LI><LI>The DNA was a smear!!!! (genomic DNA?)
</LI><LI>The pUB110 minipreps were unsuccessful.
</LI></UL><H4><SPAN class="mw-headline" id="Thursday.2C_September_12.2C_2013">Thursday, September 12, 2013</SPAN></H4><P><B>Gibson Assembly of kerA, pUB110 ligation with linker (suffix and prefix), and Transformation of E. coli with 9/09 kerA-pSB1A3 ligations</B></P><UL><LI>Ran a PCR 1:10 gibson assembly of kerA
</LI></UL><DL><DD><UL><LI>1 ul prefix primer
</LI><LI>1 ul suffix primer
</LI><LI>2.5 ul buffer
</LI><LI>1 ul mgcl2
</LI><LI>0.5 ul polymerase
</LI><LI>1 ul dntps
</LI><LI>17 ul h2o
</LI><LI>→ annealing temperatures (55°C, 58°C, &amp; 62°C)
</LI></UL></DD></DL><UL><LI>Ligated pUB110 digest with gel purified linker (pUB110 suffix and prefix): <A href="https://2013.igem.org/Team:UChicago/Protocols#Ligation_Protocol" class="external text" rel="nofollow">Ligation protocol</A></LI></UL><DL><DD><UL><LI>Concentrations of pUB110 upper band, pUB110 lower band, and linker
</LI></UL></DD><DD><UL><LI>linker gel extract = 115 ng/ul = 2.64 pmol/ul = 2640 femtomoles
</LI></UL></DD><DD><UL><LI>0.5 microliters of linker = 57 ng
</LI><LI>The pUB110 linker is 67 b.p.
</LI></UL></DD></DL><UL><LI>pUB110 extract LB: 3.2ng/ul (8/29) = 1.1 fmol/ul
</LI><LI>pUB110 extract UB: 5.2ng/ul (8/29) = 1.7 fmol/ul
</LI><LI>pUB110 is 4500 kb
</LI></UL><UL><LI>Final amounts used:
</LI></UL><DL><DD><UL><LI>10 ul of LB and UB = (11 fmol and 17 fmol, respectively)
</LI><LI>linker should be ~ 50 fmol (2 ul of 1:100 dilution)
</LI></UL></DD></DL><UL><LI>Transformed E. coli with 9/09 kerA-pSB1A3 ligations #1 - #5 (<A href="https://2013.igem.org/Team:UChicago/Protocols#Ivan.E2.80.99s_protocol_for_his_competent_cells_in_PCR_tubes:" class="external text" rel="nofollow">Transformation protocol</A>)
</LI></UL><H4><SPAN class="mw-headline" id="Friday.2C_September_13.2C_2013">Friday, September 13, 2013</SPAN></H4><P><B>Results of kerA gibson assembly and kerA ligation into pGEM-T Easy Vector</B></P><UL><LI>The PCR of kerA was successful.
</LI><LI>Ligated kerA into <A href="http://www.promega.com/resources/protocols/technical-manuals/0/pgem-t-and-pgem-t-easy-vector-systems-protocol/?__utma=1.1998193980.1380160105.1380160105.1380160105.1&amp;__utmb=1.1.10.1380160105&amp;__utmc=1&amp;__utmx=-&amp;__utmz=1.1380160105.1.1.utmcsr=google|utmccn=%28organic%29|utmcmd=organic|utmctr=%28not%20provided%29&amp;__utmv=-&amp;__utmk=166957246" class="external text" rel="nofollow">pGEM-T Easy Vector</A> for eventual transformation into E. coli
</LI></UL><P>(ng of vector × kb size of insert)/(kb size of vector) × insert:vector molar ratio = ng insert
</P><P>(25 ng vector x 1 kb insert)/(3 kb vector) x 1:1 = 8.3 ng of insert = ~1.5 ul of PCR
</P><TABLE class="wikitable"><CAPTION>Ligation Reactions
</CAPTION><TBODY><TR><TD>1
</TD><TD>2
</TD><TD>3
</TD><TD>4
</TD><TD>5
</TD><TD>6
</TD></TR><TR><TD>1 ul buffer
</TD><TD>1 ul buffer
</TD><TD>1 ul buffer
</TD><TD>1 ul buffer
</TD><TD>1 ul buffer
</TD><TD>1 ul buffer
</TD></TR><TR><TD>0.5 ul pgem
</TD><TD>0.5 ul pgem
</TD><TD>0.5 ul pgem
</TD><TD>0.5 ul pgem
</TD><TD>0.5 ul pgem
</TD><TD>0.5 ul pgem
</TD></TR><TR><TD>1.5 ul PCR 1 (9/12)
</TD><TD>1.5 ul PCR 2 (9/12)
</TD><TD>1.5 ul PCR 3 (9/12)
</TD><TD>0.5 ul PCR 3 (9/12)
</TD><TD>3.0 ul PCR 3 (9/12)
</TD><TD>5.0 ul PCR 3 (9/12)
</TD></TR><TR><TD>0.5 ul ligase
</TD><TD>0.5 ul ligase
</TD><TD>0.5 ul ligase
</TD><TD>0.5 ul ligase
</TD><TD>0.5 ul ligase
</TD><TD>0.5 ul ligase
</TD></TR><TR><TD>6.5 ul h2o
</TD><TD>6.5 ul h2o
</TD><TD>6.5 ul h2o
</TD><TD>7.5 ul h2o
</TD><TD>5.0 ul h2o
</TD><TD>3.0 ul h2o
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Saturday.2C_September_14.2C_2013">Saturday, September 14, 2013</SPAN></H4><P><B>B. subtilis electroporation with control experiments</B></P><UL><LI>Note: on re-using Cuvettes:
</LI></UL><DL><DD><UL><LI>500ul H2O mix up and down <B><U>3x</U></B>, 
</LI><LI>EtOH mix up and down with 10ul pipette, make sure you see flakes of dead cells on the first mixing, <B><U>2x</U></B>, 
</LI><LI>pour EtOH out as much as possible and air dry for 10min, wipe cap with EtOH too.
</LI></UL></DD></DL><UL><LI>Electroporation Experiments:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TD>number
</TD><TD>contains
</TD><TD>kV
</TD><TD>time constant
</TD><TD>notes
</TD></TR><TR><TD>1
</TD><TD>neg control
</TD><TD>2.1
</TD><TD>3.88
</TD><TD>no arc
</TD></TR><TR><TD>2
</TD><TD>LB + 50ng DNA
</TD><TD>2.1
</TD><TD>4.20
</TD><TD>no arc
</TD></TR><TR><TD>3
</TD><TD>UB + 50ng DNA
</TD><TD>2.1
</TD><TD>4.00
</TD><TD>no arc
</TD></TR><TR><TD>4
</TD><TD>positive control
</TD><TD>2.1
</TD><TD>3.54
</TD><TD>no arc
</TD></TR><TR><TD>5
</TD><TD>LB + 100ng DNA
</TD><TD>2.1
</TD><TD>3.96
</TD><TD>no arc
</TD></TR><TR><TD>6
</TD><TD>UB + 100ng DNA
</TD><TD>2.1
</TD><TD>3.70
</TD><TD>no arc
</TD></TR></TBODY></TABLE><DL><DD><UL><LI>Negative control: 6ul of dH2O
</LI><LI>LB Lig: 760ng/ul, diluted 1ul DNA w/ 14.2ul H2O, then added 1ul for 50ng, and 2ul for 100ng
</LI><LI>UB Lig: 690ng/ul, diluted 1ul DNA w/ 12.8ul H2O, then added 1ul for 50ng, and 2ul for 100ng
</LI><LI>Positive control: added 5ul, pUB110, 10ng/ul
</LI></UL></DD></DL><H3><SPAN class="mw-headline" id="Week_3_4">Week 3</SPAN></H3><H4><SPAN class="mw-headline" id="Monday.2C_September_16.2C_2013">Monday, September 16, 2013</SPAN></H4><P><B>Results of Electroporation experiments with controls</B></P><UL><LI>Results of yesterday’s electroporation experiments:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TD>Positive control
</TD><TD>2 colonies + a smear of E coli
</TD></TR><TR><TD>Negative control
</TD><TD>1 colony
</TD></TR><TR><TD>UB 50
</TD><TD>no distinct colonies, but some bacterial growth
</TD></TR><TR><TD>UB 100
</TD><TD>no distinct colonies, but some bacterial growth
</TD></TR><TR><TD>LB 50
</TD><TD>~10 colonies
</TD></TR><TR><TD>LB 100
</TD><TD>1 colony
</TD></TR></TBODY></TABLE><UL><LI>The electroporations seem to have not worked.
</LI></UL><UL><LI>Also, this may be insignificant, but there is a cam plate labeled “cam 9/9     AY + spmol tube 6” with 2-3 red colonies on it. 
</LI><LI>The positive plates were put in the fridge.
</LI></UL></DIV><DIV class="dividerNotebook"><P><A href="#top">[top]</A></P></DIV><DIV class="button wrapperContent"><H3><A href="/Team:UChicago/Sponsor_Us">Sponsor Us for Next Year's Competition!</A></H3></DIV><DIV id="" class="subNav"><UL><LI class=""><A href="/Team:UChicago/Protocols">Go to Protocols</A></LI></UL></DIV></DIV><DIV id="footer"><DIV class="footerAttribute"><P class="mediawiki">Powered by <A href="http://www.mediawiki.org">MediaWiki</A></P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:UChicago/Notebook">http://2013.igem.org/Team:UChicago/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UChicago/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UChicago/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UChicago/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UChicago/Notebook&amp;oldid=235040" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>