<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NYMU-Taipei_Project_Speedy_protein_degrader"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NYMU-Taipei/Project/Speedy_protein_degrader">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NYMU-Taipei/Project/Speedy_protein_degrader&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NYMU-Taipei/Project/Speedy_protein_degrader&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NYMU-Taipei/Project/Speedy_protein_degrader&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NYMU-Taipei/Project/Speedy_protein_degrader" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NYMU-Taipei/Project/Speedy protein degrader</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><TABLE id="nav" style="border:1px solid gray;width:965px"><TBODY><TR><TD class="right"><A href="/Team:NYMU-Taipei" title="Team:NYMU-Taipei"> Home</A></TD><TD class="left" style="width:120px;"><A href="/Team:NYMU-Taipei/Project" title="Team:NYMU-Taipei/Project"> Project Overview</A></TD><TD class="left"><A href="/Team:NYMU-Taipei/Project/Speedy_reporter" title="Team:NYMU-Taipei/Project/Speedy reporter"> Speedy reporter</A></TD><TD class="left"><A href="/Team:NYMU-Taipei/Project/Speedy_switch" title="Team:NYMU-Taipei/Project/Speedy switch"> Speedy switch</A></TD><TD class="left" style="width:120px;"><STRONG class="selflink"> Speedy protein degrader</STRONG></TD><TD class="both" style="width:180px;"><A href="/Team:NYMU-Taipei/Experiments" title="Team:NYMU-Taipei/Experiments"> Experiments and Parts</A></TD><TD class="both"><A href="/Team:NYMU-Taipei/Applications" title="Team:NYMU-Taipei/Applications"> Applications</A></TD><TD class="both"><A href="/Team:NYMU-Taipei/FAQ" title="Team:NYMU-Taipei/FAQ"> F.A.Q</A></TD><TD class="left"><A href="/Team:NYMU-Taipei/Team" title="Team:NYMU-Taipei/Team"> About Us</A></TD></TR></TBODY></TABLE><DIV style="float:right;padding:10px;"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Abstract"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Abstract</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Motivation"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Motivation</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Design:_Controlled_protein_degradation_in_bacteria"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Design: Controlled protein degradation in bacteria</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#SsrA_tag"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">SsrA tag</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Circuit_Design"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Circuit Design</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-6"><A href="#Data_analysis"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Data analysis</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Advantages"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Advantages</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Reference"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Reference</SPAN></A></LI></TD></TR></TBODY></TABLE></DIV><H1><SPAN class="mw-headline" id="Abstract"><FONT color="blue">Abstract</FONT></SPAN></H1><UL><LI> We developed the fluorescent proteins with <B>degradation tags</B> which accelerate the reporter turnover. This provides <B>better temporal resolution</B> of gene expression profiling in single cells.
</LI></UL><P>A central goal of synthetic biology is to explore the design principles that embedded in the architecture and operation of biological systems. To understand how biological systems are genetically assembled, synthetic biologists need tools to quantitatively describe the gene expression in model organisms. Fluorescent proteins(FP), along with modern microscopes, allow real-time gene expression profiling at the level of individual living cells. Though the detection of FPs benefits from their stability, this advantage turns awkward when we are monitoring the dynamic cellular events which required measurements at higher temporal resolution. One approach to achieve such demand is specific and accelerated reporter degradation. Thus, we focus on the design of fusion proteins with degradation tags. Previous studies suggest the LVA tag has hitherto been most efficient tag to shorten the half-life of tagged protein in <I>Escherichia coli</I>. We added LVA tags to the C-terminal of different fluorescent proteins. Our results indicate that the half-life of fluorophores are significantly reduced. Therefore, we improved the temporal resolution of reporter proteins. This provides a promising tool for studying transient gene expression in the future.
</P><H1><SPAN class="mw-headline" id="Motivation"><FONT color="blue">Motivation</FONT></SPAN></H1><UL><LI>How can we <B>improve the temporal sampling</B> of biological assay? We applied <B>destabilized fluorescent proteins</B> to achieve this goal.
</LI></UL><P>Fluorescent proteins, for example, green fluorescent protein (GFP), are now widely used to visualize a protein’s distribution and dynamics in a subcellular compartment. The remarkable feature of the fluorescent proteins (FP) is that the fluorophore forms spontaneously (Chalfie et al., 1994) without the need of any substrate or cofactor (Heim et al., 1994). In addition, FP variants can be fused to virtually any protein of interest; thus, they can be used in many species for detection purposes within single living cells. FPs, together with modern microscopes, has become essential tools for studying spatial and temporal dynamics of cellular processes at high resolution.
</P><P>The biophysical characteristics of fluorescent probes provide opportunities and, set limitations for fusion-protein studies. The fluorophore of GFP and its variants is contained within a barrel of beta-sheet protein (Ormo et al., 1996). This compact structure renders FPs with high photostability under a variety of conditions, even under treatment with protease (Chalfie et al., 1994). Once stable FPs formed, they are cleared from the cells in tens of hours (Andersen et al., 1998). This property allows prolonged imaging of cells, and easier detection when FPs accumulated. However, the stability of FPs limits its application in some studies which needs fast reporter turnover, for example, studies of transient gene expression. This also impedes the measurements of temporal expression pattern and behavior of proteins, because proteins at different stages of their lifetime are being detected. To overcome this limitation, we tried to develop destablized FPs.
</P><P>There are several ways to generate FP variants with different spectral and expression properties. One approach is mutagenesis studies. Though this approach is fruitful (Delagrave et al., 1995; Heim et al., 1995; Zacharias et al., 2002), it seems impractical to screen possible mutations of each FP in ever-increasing FP libraries. Alternatively, we planed to control proteolysis by adding degradation tags to FPs (Andersen et al., 1998). This approach can be easily generalized to several FPs; thus, it provides extensibility to future applications.
</P><H1><SPAN class="mw-headline" id="Design:_Controlled_protein_degradation_in_bacteria"><FONT color="blue">Design: Controlled protein degradation in bacteria</FONT></SPAN></H1><UL><LI>We added the <B>degradation tag</B> to each fluorescent protein. This approach effectively <B>accelerates reporter turnover</B>.
</LI></UL><P>The flow of genetic information can—and sometimes need to — depend not only on its controlled synthesis but also equally on its controlled degradation. Indeed, protein degradation in natural systems is essential for cellular functions. Previous study showed that 2.7% of proteins are degraded per generation in <I>Escherichia coli</I> (<I>E. coli.</I>) growing in logarithmic phase (Fox and Brown, 1961). Control of protein turnover is required for cell cycle progression, signal transduction, and rapid responses to environmental challenges (Grunenfelder et al., 2001; Hengge-Aronis, 2002; Neher et al., 2003). Therefore, regulated degradation is expected to be critical for the development of synthetic circuits.
</P><P>To control the half-life of specific proteins, we utilized the regulated proteolysis machineries   naturally employed to bacterial systems. Regulated degradation is ubiquitous for prokaryotic and eukaryotic cells (Hershko and Ciechanover, 1998; Jenal and Hengge-Aronis, 2003), because they need to clear damaged or aberrant proteins while ignoring functional ones. A common strategy for substrate selection is adding degradation tags to target proteins. Then, certain proteases execute energy dependent degradation of those proteins. One prominent type of degradation tag in prokaryotes is the ssrA tag, a tmRNA encoded by ssrA gene. This tagging system is engaged in protein quality control throughout all eubacteria (Karzai et al., 2000). Though it was first described as a mechanism to clear obstructed ribosomes (Keiler et al., 1996), recent studies suggested ssrA-mediated tagging also plays a regulatory role in the expression of lac operon (Abo et al., 2000). This motivated us to control the protein expression via manipulating controlled proteolysis.
</P><H3><SPAN class="mw-headline" id="SsrA_tag">SsrA tag</SPAN></H3><P>SsrA tag, encoded by the ssrA RNA, is important for protein quality control in bacteria. SsrA RNA is recognized as tmRNA because it has characteristics of both tRNA and mRNA (Atkins and Gesteland, 1996; Keiler et al., 1996). When <I>E. coli.</I> ribosomes stall during translation due to absence of a proper stop codon, ssrA ribosome-rescue system mediates C-terminal modification by adding the sequence AANDENYALAA to the the incomplete peptide (Keiler et al., 1996). SsrA tagging frees these ribosomes for other mRNAs, and targets the defective polypeptides for degradation by ClpXP and other ATP-dependent proteases (Gottesman et al., 1998). Previous studies suggested that 0.5% of protein products in <I>E. coli.</I>. receive ssrA tags (Lies and Maurizi, 2008). Bacterial strains lacking functional ssrA gene show slower growth (Oh and Apirion, 1991), reduction in motility (Komine et al., 1994), and subsided pathogenesis (Julio et al., 2000). These results indicate that ssrA tagging system play a major role in bacterial physiology.
</P><P>The SsrA tag contains binding sites for the SspB adaptor and ClpXP protease (Levchenko et al., 2000). The SspB adaptor enhances substrate delivery by tethering SsrA-tagged proteins to the ClpX, which unfolds the tagged protein, and then translocates the denatured polypeptide into ClpP for degradation (Fig.1)(Sauer et al., 2004). The binding affinity of the ssrA tag to the SspB and ClpX plays key role in modulating substrate choice (Flynn et al., 2004) and the efficiency of degradation (Hersch et al., 2004; McGinness et al., 2006). One of the mutants, called LVA tag, further accelerates the degradation of tagged proteins compared to the wild-type LAA tag (Andersen et al., 1998). LVA tag has been applied to construct a fast synthetic gene oscillator, because it decreases protein lifetime and increases temporal resolution (Stricker et al., 2008). Thus, we added the LVA tag to FPs to maximize the temporal resolution of these reporters.
</P><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Fig.1 Molecular mechanism of SsrA-SspB system</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Circuit_Design">Circuit Design</SPAN></H3><UL><LI>We designed four fluorescence protein circuits. In each following panel, the upper one is the conrol, and the lower one is the one with LVA tag. 
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:617px;"><DIV class="thumbcaption">Fig.2 GFP(BBa_K411233) and GFP with LVA tag(BBa_K411237)</DIV></DIV></DIV><DIV class="thumb tnone"><DIV class="thumbinner" style="width:617px;"><DIV class="thumbcaption">Fig.3 RFP(BBa_K411234) and RFP with LVA tag(BBa_K411238)</DIV></DIV></DIV><DIV class="thumb tnone"><DIV class="thumbinner" style="width:617px;"><DIV class="thumbcaption">Fig.4 CFP(BBa_K411231) and CFP with LVA tag(BBa_K411235)</DIV></DIV></DIV><DIV class="thumb tnone"><DIV class="thumbinner" style="width:617px;"><DIV class="thumbcaption">Fig.5 YFP(BBa_K411232) and YFP with LVA tag(BBa_K411236)</DIV></DIV></DIV><P>
For details of this part, please see: <A href="/Team:NYMU-Taipei/Experiments#Parts" title="Team:NYMU-Taipei/Experiments">Designed Parts</A>, <A href="/Team:NYMU-Taipei/Project/Speedy_protein_degrader/Sequences" title="Team:NYMU-Taipei/Project/Speedy protein degrader/Sequences">Sequences</A></P><H1><SPAN class="mw-headline" id="Data_analysis"><FONT color="blue">Data analysis</FONT></SPAN></H1><P><A href="/Team:NYMU-Taipei/Experiments/Speedy_degrader#Results" title="Team:NYMU-Taipei/Experiments/Speedy degrader">This page</A> contains the experimental data gathered through reporting assays.
</P><H1><SPAN class="mw-headline" id="Advantages"><FONT color="blue">Advantages</FONT></SPAN></H1><P>1.<B>Universal</B>: 
</P><P>The SsrA-SspB system is employed in all eubacteria (Karzai et al., 2000) whose genomes have been sequenced. Thus, the ‘speedy degrader’ can be easily implemented in other prokaryotic systems, for example, <I>Bacillus subtilis</I>. Actually, the components of SsrA system has been transplanted into a <I>Saccharomyces cerevisiae</I> strain that allows for tunable degradation of a tagged protein (Grilly et al., 2007). The striking instance suggested that it is plausible to apply the ‘speedy degrader’ even in eukaryotic systems.
</P><P>2.<B>Less interference to the protein function</B>: 
</P><P>Most fusion protein studies require an available functional assay. Because the peptide tag may be detrimental to the structure of target protein, it is critical to assure that the tagged protein behaves as the native protein. A nonfunctional FP-tagged protein will be uninformative in most, if not all, biological studies. Fortunately, ssrA tag does not affect the structure or thermodynamic stability of attached proteins (Gottesman et al., 1998). Therefore, the degradation tag will be a suitable component to integrate with other synthetic devices.
</P><H1><SPAN class="mw-headline" id="Reference"><FONT color="blue">Reference</FONT></SPAN></H1><UL><LI>Abo, T., Inada, T., Ogawa, K., and Aiba, H. (2000). SsrA-mediated tagging and proteolysis of LacI and its role in the regulation of lac operon. <I>EMBO J</I> 19, 3762-3769.
</LI></UL><UL><LI>Andersen, J.B., Sternberg, C., Poulsen, L.K., Bjorn, S.P., Givskov, M., and Molin, S. (1998). New unstable variants of green fluorescent protein for studies of transient gene expression in bacteria. <I>Appl Environ Microbiol</I> 64, 2240-2246.
</LI></UL><UL><LI>Atkins, J.F., and Gesteland, R.F. (1996). A case for trans translation. <I>Nature</I> 379, 769-771.
</LI></UL><UL><LI>Chalfie, M., Tu, Y., Euskirchen, G., Ward, W.W., and Prasher, D.C. (1994). Green fluorescent protein as a marker for gene expression. <I>Science</I> 263, 802-805.
</LI></UL><UL><LI>Delagrave, S., Hawtin, R.E., Silva, C.M., Yang, M.M., and Youvan, D.C. (1995). Red-shifted excitation mutants of the green fluorescent protein. <I>Biotechnology (N Y)</I> 13, 151-154.
</LI></UL><UL><LI>Flynn, J.M., Levchenko, I., Sauer, R.T., and Baker, T.A. (2004). Modulating substrate choice: the SspB adaptor delivers a regulator of the extracytoplasmic-stress response to the AAA+ protease ClpXP for degradation. <I>Genes Dev</I> 18, 2292-2301.
</LI></UL><UL><LI>Fox, G., and Brown, J.W. (1961). Protein degradation in Escherichia coli in the logarithmic phase of growth. <I>Biochim Biophys Acta</I> 46, 387-389.
</LI></UL><UL><LI>Gottesman, S., Roche, E., Zhou, Y., and Sauer, R.T. (1998). The ClpXP and ClpAP proteases degrade proteins with carboxy-terminal peptide tails added by the SsrA-tagging system. <I>Genes Dev</I> 12, 1338-1347.
</LI></UL><UL><LI>Grilly, C., Stricker, J., Pang, W.L., Bennett, M.R., and Hasty, J. (2007). A synthetic gene network for tuning protein degradation in Saccharomyces cerevisiae. <I>Mol Syst Biol</I> 3, 127.
</LI></UL><UL><LI>Grunenfelder, B., Rummel, G., Vohradsky, J., Roder, D., Langen, H., and Jenal, U. (2001). Proteomic analysis of the bacterial cell cycle. <I>Proc Natl Acad Sci U S A</I> 98, 4681-4686.
</LI></UL><UL><LI>Heim, R., Cubitt, A.B., and Tsien, R.Y. (1995). Improved green fluorescence. <I>Nature</I> 373, 663-664.
</LI></UL><UL><LI>Heim, R., Prasher, D.C., and Tsien, R.Y. (1994). Wavelength mutations and posttranslational autoxidation of green fluorescent protein. <I>Proc Natl Acad Sci U S A</I> 91, 12501-12504.
</LI></UL><UL><LI>Hengge-Aronis, R. (2002). Signal transduction and regulatory mechanisms involved in control of the sigma(S) (RpoS) subunit of RNA polymerase. <I>Microbiol Mol Biol Rev</I> 66, 373-395
</LI></UL><UL><LI>Hersch, G.L., Baker, T.A., and Sauer, R.T. (2004). SspB delivery of substrates for ClpXP proteolysis probed by the design of improved degradation tags. <I>Proc Natl Acad Sci U S A</I> 101, 12136-12141.
</LI></UL><UL><LI>Hershko, A., and Ciechanover, A. (1998). The ubiquitin system. <I>Annu Rev Biochem</I> 67, 425-479.
</LI></UL><UL><LI>Jenal, U., and Hengge-Aronis, R. (2003). Regulation by proteolysis in bacterial cells. <I>Curr Opin Microbiol</I> 6, 163-172.
</LI></UL><UL><LI>Julio, S.M., Heithoff, D.M., and Mahan, M.J. (2000). ssrA (tmRNA) plays a role in Salmonella enterica serovar Typhimurium pathogenesis. <I>J Bacteriol</I> 182, 1558-1563.
</LI></UL><UL><LI>Karzai, A.W., Roche, E.D., and Sauer, R.T. (2000). The SsrA-SmpB system for protein tagging, directed degradation and ribosome rescue. <I>Nat Struct Biol</I> 7, 449-455.
</LI></UL><UL><LI>Keiler, K.C., Waller, P.R., and Sauer, R.T. (1996). Role of a peptide tagging system in degradation of proteins synthesized from damaged messenger RNA. <I>Science</I> 271, 990-993.
</LI></UL><UL><LI>Komine, Y., Kitabatake, M., Yokogawa, T., Nishikawa, K., and Inokuchi, H. (1994). A tRNA-like structure is present in 10Sa RNA, a small stable RNA from Escherichia coli. <I>Proc Natl Acad Sci U S A</I> 91, 9223-9227.
</LI></UL><UL><LI>Levchenko, I., Seidel, M., Sauer, R.T., and Baker, T.A. (2000). A specificity-enhancing factor for the ClpXP degradation machine. <I>Science</I> 289, 2354-2356.
</LI></UL><UL><LI>Lies, M., and Maurizi, M.R. (2008). Turnover of endogenous SsrA-tagged proteins mediated by ATP-dependent proteases in Escherichia coli. <I>J Biol Chem</I> 283, 22918-22929.
</LI></UL><UL><LI>McGinness, K.E., Baker, T.A., and Sauer, R.T. (2006). Engineering controllable protein degradation. <I>Mol Cell</I> 22, 701-707.
</LI></UL><UL><LI>Neher, S.B., Flynn, J.M., Sauer, R.T., and Baker, T.A. (2003). Latent ClpX-recognition signals ensure LexA destruction after DNA damage. <I>Genes Dev</I> 17, 1084-1089.
</LI></UL><UL><LI>Oh, B.K., and Apirion, D. (1991). 10Sa RNA, a small stable RNA of Escherichia coli, is functional. <I>Mol Gen Genet</I> 229, 52-56.
</LI></UL><UL><LI>Ormo, M., Cubitt, A.B., Kallio, K., Gross, L.A., Tsien, R.Y., and Remington, S.J. (1996). Crystal structure of the Aequorea victoria green fluorescent protein. <I>Science</I> 273, 1392-1395.
</LI></UL><UL><LI>Sauer, R.T., Bolon, D.N., Burton, B.M., Burton, R.E., Flynn, J.M., Grant, R.A., Hersch, G.L., Joshi, S.A., Kenniston, J.A., Levchenko, I., et al. (2004). Sculpting the proteome with AAA(+) proteases and disassembly machines. <I>Cell</I> 119, 9-18.
</LI></UL><UL><LI>Stricker, J., Cookson, S., Bennett, M.R., Mather, W.H., Tsimring, L.S., and Hasty, J. (2008). A fast, robust and tunable synthetic gene oscillator. <I>Nature</I> 456, 516-519.
</LI></UL><UL><LI>Zacharias, D.A., Violin, J.D., Newton, A.C., and Tsien, R.Y. (2002). Partitioning of lipid-modified monomeric GFPs into membrane microdomains of live cells. <I>Science</I> 296, 913-916.
</LI></UL><DIV id="bottomlinkbox"><A id="bottomlink" href="https://2010.igem.org/Team:NYMU-Taipei">NYMU-Taipei iGEM 2010 Team</A></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:NYMU-Taipei/Project/Speedy_protein_degrader">http://2010.igem.org/Team:NYMU-Taipei/Project/Speedy_protein_degrader</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NYMU-Taipei/Project/Speedy_protein_degrader" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NYMU-Taipei/Project/Speedy_protein_degrader" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NYMU-Taipei/Project/Speedy_protein_degrader&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NYMU-Taipei/Project/Speedy_protein_degrader&amp;oldid=203062" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>