<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NYMU-Taipei_FAQ"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NYMU-Taipei/FAQ">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NYMU-Taipei/FAQ&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NYMU-Taipei/FAQ&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NYMU-Taipei/FAQ&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NYMU-Taipei/FAQ" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NYMU-Taipei/FAQ</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><TABLE id="nav" style="border:1px solid gray;width:965px"><TBODY><TR><TD class="right"><A href="/Team:NYMU-Taipei" title="Team:NYMU-Taipei"> Home</A></TD><TD class="left" style="width:120px;"><A href="/Team:NYMU-Taipei/Project" title="Team:NYMU-Taipei/Project"> Project Overview</A></TD><TD class="left"><A href="/Team:NYMU-Taipei/Project/Speedy_reporter" title="Team:NYMU-Taipei/Project/Speedy reporter"> Speedy reporter</A></TD><TD class="left"><A href="/Team:NYMU-Taipei/Project/Speedy_switch" title="Team:NYMU-Taipei/Project/Speedy switch"> Speedy switch</A></TD><TD class="left" style="width:120px;"><A href="/Team:NYMU-Taipei/Project/Speedy_protein_degrader" title="Team:NYMU-Taipei/Project/Speedy protein degrader"> Speedy protein degrader</A></TD><TD class="both" style="width:180px;"><A href="/Team:NYMU-Taipei/Experiments" title="Team:NYMU-Taipei/Experiments"> Experiments and Parts</A></TD><TD class="both"><A href="/Team:NYMU-Taipei/Applications" title="Team:NYMU-Taipei/Applications"> Applications</A></TD><TD class="both"><STRONG class="selflink"> F.A.Q</STRONG></TD><TD class="left"><A href="/Team:NYMU-Taipei/Team" title="Team:NYMU-Taipei/Team"> About Us</A></TD></TR></TBODY></TABLE><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#General_Questions"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">General Questions</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Speedy_switch"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Speedy switch</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Speedy_reporter"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Speedy reporter</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Speedy_degrader"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Speedy degrader</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="General_Questions"><FONT color="blue">General Questions</FONT></SPAN></H2><DIV class="qa We have created 45 parts this year, and have tested and used many of the provided biobricks. More details about the parts we have created can be found in &lt;a href="> Parts.&quot;&gt;
<DIV class="q"><SPAN>Q.</SPAN> How many biobricks do you test or construct the new biobricks? </DIV><DIV class="a"><SPAN>A.</SPAN> We have created 45 parts this year, and have tested and used many of the provided biobricks. More details about the parts we have created can be found in <A href="/Team:NYMU-Taipei/Experiments#Parts" title="Team:NYMU-Taipei/Experiments"> Parts</A>. </DIV></DIV><DIV class="qa We want to provide a tool that can be used in synthetic biology to reveal the intricate rules involved in biological systems. Furthermore, we have created a tool that can speed up our exploration of gene regulation in vivo."><DIV class="q"><SPAN>Q.</SPAN> Could you please tell me the significance of your project? What kind of problems you want to solve? </DIV><DIV class="a"><SPAN>A.</SPAN> We want to provide a tool that can be used in synthetic biology to reveal the intricate rules involved in biological systems. Furthermore, we have created a tool that can speed up our exploration of gene regulation in vivo. </DIV></DIV><DIV class="qa Discovering the rules of biological system is the basis of much research in synthetic biology today. However, due to the nature of bacteria and other living organisms, finding these rules require a lot of time. Our project provides a tool that can not only help discover details about the biological world, it also works to speed up conventional research. By speeding up research, we can reveal the nature of organisms."><DIV class="q"><SPAN>Q.</SPAN> Tell me more about the rule of synthetic biology in your slides. What is the connection between rule and speedy? </DIV><DIV class="a"><SPAN>A.</SPAN> Discovering the rules of biological system is the basis of much research in synthetic biology today. However, due to the nature of bacteria and other living organisms, finding these rules require a lot of time. Our project provides a tool that can not only help discover details about the biological world, it also works to speed up conventional research. By speeding up research, we can reveal the nature of organisms. </DIV></DIV><DIV class="qa When studing a cell population, we can only take the average of all of the cells expression. This way of studying an organism is synonymous with studying humans using millions as the smallest unit. However, it is often the uniquness of a specific individual that reveals the most information. Assuming that the technical aspects of single cell study can be soon overcome, we believe that to truly understand the minute details of research requires precise work on single cell research. Our project can aid this research by providing control over the cell's actions."><DIV class="q"><SPAN>Q.</SPAN> What are the differences between a single cell and cell population? </DIV><DIV class="a"><SPAN>A.</SPAN> When studing a cell population, we can only take the average of all of the cells expression. This way of studying an organism is synonymous with studying humans using millions as the smallest unit. However, it is often the uniquness of a specific individual that reveals the most information. Assuming that the technical aspects of single cell study can be soon overcome, we believe that to truly understand the minute details of research requires precise work on single cell research. Our project can aid this research by providing control over the cell's actions. </DIV></DIV><DIV class="qa We put emphasis on a single cell simply because few have considered single cell research. Neither single cell research nor population research is better than the other. They are separate from each other; both types have advantages and disadvatages. We research on animals and humans using both population and individual research methods. This should be reflected synthetic biology. Both single cell studies and population studies should be vital components of synthetic biology."><DIV class="q"><SPAN>Q.</SPAN> Why we put our emphasis on a single cell? If we can get result just by a simple method in cell population, what is the need for us to use a single cell? What's the importance of using a single cell? </DIV><DIV class="a"><SPAN>A.</SPAN> We put emphasis on a single cell simply because few have considered single cell research. Neither single cell research nor population research is better than the other. They are separate from each other; both types have advantages and disadvatages. We research on animals and humans using both population and individual research methods. This should be reflected synthetic biology. Both single cell studies and population studies should be vital components of synthetic biology. </DIV></DIV><H2><SPAN class="mw-headline" id="Speedy_switch"><FONT color="blue">Speedy switch</FONT></SPAN></H2><DIV class="qa Riboswitch is a part of mRNA which can bind to a small target molecule. This reaction can then affect the activity of the gene following this riboswitch due to change in secondary structure. In essence, a riboswitch acts similarly to the promoter region of DNA and can regulate the translation of proteins from this mRNA"><DIV class="q"><SPAN>Q.</SPAN> What is a Riboswitch? </DIV><DIV class="a"><SPAN>A.</SPAN> Riboswitch is a part of mRNA which can bind to a small target molecule. This reaction can then affect the activity of the gene following this riboswitch due to change in secondary structure. In essence, a riboswitch acts similarly to the promoter region of DNA and can regulate the translation of proteins from this mRNA </DIV></DIV><DIV class="qa Due to the fact that proteins can bind to small molecules to induce effects, riboswitches may not be essential to life. However, we can use the ability of riboswitches to bind to small molecules to regulate the genes that are downstream. Using this method, DNA is pre-transcribed, and when the genes downstream are needed, an inducer can be introduced to activate the riboswitch, thus translating the needed genes."><DIV class="q"><SPAN>Q.</SPAN> Why do we need Riboswitches </DIV><DIV class="a"><SPAN>A.</SPAN> Due to the fact that proteins can bind to small molecules to induce effects, riboswitches may not be essential to life. However, we can use the ability of riboswitches to bind to small molecules to regulate the genes that are downstream. Using this method, DNA is pre-transcribed, and when the genes downstream are needed, an inducer can be introduced to activate the riboswitch, thus translating the needed genes. </DIV></DIV><DIV class="qa When promoters are used, the cell has to go through the entire process of DNA to RNA to Protein. As a result, This is a slow way to measure the reaction time of a certain experiments. For example, to measure the effect of a substance on the production of GFP after it has been introduced into the cell, we would have to wait for the cell to transcribe RNA from DNA, then GFP from RNA. If we used a riboswitch, transcription of RNA to DNA has already been accomplished. We when we introduce a substance that combines with a riboswitch, all the cell has to do is translate protein from mRNA essentially skipping the first part of the central dogma. Using this way we can speed up the reaction time of a cell towards a particular substance"><DIV class="q"><SPAN>Q.</SPAN> What are the advantages of Riboswitches over Promoters </DIV><DIV class="a"><SPAN>A.</SPAN> When promoters are used, the cell has to go through the entire process of DNA to RNA to Protein. As a result, This is a slow way to measure the reaction time of a certain experiments. For example, to measure the effect of a substance on the production of GFP after it has been introduced into the cell, we would have to wait for the cell to transcribe RNA from DNA, then GFP from RNA. If we used a riboswitch, transcription of RNA to DNA has already been accomplished. We when we introduce a substance that combines with a riboswitch, all the cell has to do is translate protein from mRNA essentially skipping the first part of the central dogma. Using this way we can speed up the reaction time of a cell towards a particular substance </DIV></DIV><DIV class="qa Currently the main limitation of riboswitches is that there are only a few substances that we know can bind with  riboswitches. Furthermore, sometimes RNA sequences of protein  ligated to the end of the riboswitch interfere with the function of riboswitches. As a result, there are many reactions that we cannot test. However, if we can find more riboswitches and are able to pick and choose the riboswitch as well as the protein sequence that can be attached to its end, we can speed up a variety of different experiments. "><DIV class="q"><SPAN>Q.</SPAN> What are its limitations?* </DIV><DIV class="a"><SPAN>A.</SPAN> Currently the main limitation of riboswitches is that there are only a few substances that we know can bind with  riboswitches. Furthermore, sometimes RNA sequences of protein  ligated to the end of the riboswitch interfere with the function of riboswitches. As a result, there are many reactions that we cannot test. However, if we can find more riboswitches and are able to pick and choose the riboswitch as well as the protein sequence that can be attached to its end, we can speed up a variety of different experiments.  </DIV></DIV><DIV class="qa GFP is a widely common and thoroughly tested protein. Its fluorescence makes it very easy to tell if the riboswitch is working or not. By using GFP we can first determine the viability of a particular riboswitch before we ligate other protein coding regions to the end of it."><DIV class="q"><SPAN>Q.</SPAN> Why do we use GFP to test riboswitch first? What are the advantages? </DIV><DIV class="a"><SPAN>A.</SPAN> GFP is a widely common and thoroughly tested protein. Its fluorescence makes it very easy to tell if the riboswitch is working or not. By using GFP we can first determine the viability of a particular riboswitch before we ligate other protein coding regions to the end of it. </DIV></DIV><DIV class="qa Although there are many different riboswitches, we needed a very specific type of riboswitch in our experiment. We had several requirements that had to be fulfilled:
&lt;/p&gt;
&lt;ul&gt;&lt;li&gt;The inducer for this riboswitch cannot be metabolized by E.coli. If the induced is metabolized, it may be digested before it has the chance to bind with the riboswitch.
&lt;/li&gt;&lt;li&gt;The inducer cannot exist in naturally occuring E.coli, otherwise we would have no means of turning this switch on or off.
&lt;/li&gt;&lt;li&gt;The DNA of the riboswitch cannot contain a E,X,S, or P cutting site, otherwise we would splice it accidently during transformation.
&lt;/li&gt;&lt;/ul&gt;
&lt;p&gt;After scanning through many papers documenting riboswitches, only the theophylline riboswitch matched our descriptions."><DIV class="q"><SPAN>Q.</SPAN> There are many different riboswitches .Why did you choose the theophylline riboswitch? </DIV><DIV class="a"><SPAN>A.</SPAN> Although there are many different riboswitches, we needed a very specific type of riboswitch in our experiment. We had several requirements that had to be fulfilled:
<UL><LI>The inducer for this riboswitch cannot be metabolized by E.coli. If the induced is metabolized, it may be digested before it has the chance to bind with the riboswitch.
</LI><LI>The inducer cannot exist in naturally occuring E.coli, otherwise we would have no means of turning this switch on or off.
</LI><LI>The DNA of the riboswitch cannot contain a E,X,S, or P cutting site, otherwise we would splice it accidently during transformation.
</LI></UL><P>After scanning through many papers documenting riboswitches, only the theophylline riboswitch matched our descriptions. </P></DIV></DIV><DIV class="qa When a cell is in a bad environment(in our device, theophylline is toxic for E. coli.), the speed of mRNA degration is slower than under normal condition. Even though less mRNA with riboswitch is degraded, we still use a constitutive promoter to ensure that a greater amount of RNA is produced."><DIV class="q"><SPAN>Q.</SPAN> Will the untranslated mRNA with Riboswitch be degraded in the cell? </DIV><DIV class="a"><SPAN>A.</SPAN> When a cell is in a bad environment(in our device, theophylline is toxic for E. coli.), the speed of mRNA degration is slower than under normal condition. Even though less mRNA with riboswitch is degraded, we still use a constitutive promoter to ensure that a greater amount of RNA is produced. </DIV></DIV><DIV class="qa Downstream the mRNA reporting aptamer so that we can study the movement of mRNA before inducing the riboswitch to begin translation to protein"><DIV class="q"><SPAN>Q.</SPAN> In the whole picture, where is the riboswitch sequence located? Upstream or downstream the mRNA reporting aptamer? </DIV><DIV class="a"><SPAN>A.</SPAN> Downstream the mRNA reporting aptamer so that we can study the movement of mRNA before inducing the riboswitch to begin translation to protein </DIV></DIV><DIV class="qa One of our references: &lt;i&gt;Shana Topp and Justin P. Gallivan(2007)Guiding Bacteria with Small Molecules and RNA. JACS&lt;/i&gt; also use theophylline riboswitch as their material. Their data show that after adding theophylline into riboswitch can really work. For more reference, please check out our part: speedy switch."><DIV class="q"><SPAN>Q.</SPAN> Could you please give me some references to support your theophylline riboswitch can work? </DIV><DIV class="a"><SPAN>A.</SPAN> One of our references: <I>Shana Topp and Justin P. Gallivan(2007)Guiding Bacteria with Small Molecules and RNA. JACS</I> also use theophylline riboswitch as their material. Their data show that after adding theophylline into riboswitch can really work. For more reference, please check out our part: speedy switch. </DIV></DIV><DIV class="qa By using the speedy switch, we can control the flow of genetic information. We can stop and start protein translation at will. When we consider this level of control over a single cell, it becomes much easier to research how a single cell works in different enviroments, with different genetic information"><DIV class="q"><SPAN>Q.</SPAN> Could you please explain the relationship between speedy switch and single cell?  </DIV><DIV class="a"><SPAN>A.</SPAN> By using the speedy switch, we can control the flow of genetic information. We can stop and start protein translation at will. When we consider this level of control over a single cell, it becomes much easier to research how a single cell works in different enviroments, with different genetic information </DIV></DIV><H2><SPAN class="mw-headline" id="Speedy_reporter"><FONT color="blue">Speedy reporter</FONT></SPAN></H2><DIV class="qa We chose eIF4A as our mRNA binding protein due to its dumbell structure. This structure makes it very simple to cleave in the middle which maintaining the interaction between the two parts of the split eIF4A structure."><DIV class="q"><SPAN>Q.</SPAN> Why choose eIF4A as our mRNA binding protein rather than other proteins that are natively produced by bacteriaÂ ? </DIV><DIV class="a"><SPAN>A.</SPAN> We chose eIF4A as our mRNA binding protein due to its dumbell structure. This structure makes it very simple to cleave in the middle which maintaining the interaction between the two parts of the split eIF4A structure. </DIV></DIV><DIV class="qa According to research papers, there is very low background fluorescence when the mRNA aptamer is not added into the solution. This means that the probability of random combination of split GFP-eIF4A is extremely low."><DIV class="q"><SPAN>Q.</SPAN> How can you confirm that fluorescence is emitted via GFP-eIF4A binding on the mRNA aptamer rather than random combination of the split GFP-eIF4A? </DIV><DIV class="a"><SPAN>A.</SPAN> According to research papers, there is very low background fluorescence when the mRNA aptamer is not added into the solution. This means that the probability of random combination of split GFP-eIF4A is extremely low. </DIV></DIV><DIV class="qa While these ideas may not be the newest or the most innovative, they are still scarcely seen in the grand scheme of synthetic biology. SpeedyBac combines several smaller devices into a combination that can help increase the rate at which knowledge about synthetic biology is growing. We wish to bring unconspicuous ways to shorten lab times to the teams at iGEM and across the world."><DIV class="q"><SPAN>Q.</SPAN> It's seems you just copy the paper,what's your new idea or contribution? </DIV><DIV class="a"><SPAN>A.</SPAN> While these ideas may not be the newest or the most innovative, they are still scarcely seen in the grand scheme of synthetic biology. SpeedyBac combines several smaller devices into a combination that can help increase the rate at which knowledge about synthetic biology is growing. We wish to bring unconspicuous ways to shorten lab times to the teams at iGEM and across the world. </DIV></DIV><DIV class="qa They are both inducible in our system. We wish to use these promoters to check mRNA reporter system works to report mRNA when they are produced."><DIV class="q"><SPAN>Q.</SPAN> Plac or Ptet promoter is inducible or consitutive in your system? Why does mRNA reporting part ues two kinds of inducible promoter? </DIV><DIV class="a"><SPAN>A.</SPAN> They are both inducible in our system. We wish to use these promoters to check mRNA reporter system works to report mRNA when they are produced. </DIV></DIV><DIV class="qa We skip the process of translation because our split GFP is constitutive in the cell. However, these GFPs still only light up when we induce the production of RNA aptamer. This way we can essentially skip the waiting time for protein folding."><DIV class="q"><SPAN>Q.</SPAN> Could you please tell me what is mRNA reprting part different from the conventional way? </DIV><DIV class="a"><SPAN>A.</SPAN> We skip the process of translation because our split GFP is constitutive in the cell. However, these GFPs still only light up when we induce the production of RNA aptamer. This way we can essentially skip the waiting time for protein folding. </DIV></DIV><DIV class="qa IPTG is the inducer that starts production of our eIF4A aptamer."><DIV class="q"><SPAN>Q.</SPAN> In your result assay part, why do you add IPTG? </DIV><DIV class="a"><SPAN>A.</SPAN> IPTG is the inducer that starts production of our eIF4A aptamer. </DIV></DIV><DIV class="qa The split form of GFP can reconstitute in the presence of RNA aptamer and begin fluorescing in about 3 mins. A normal GFP needs about 3 hours to emit its fluroscence."><DIV class="q"><SPAN>Q.</SPAN> What is the time saved when using split-GFP instead of normal GFP? </DIV><DIV class="a"><SPAN>A.</SPAN> The split form of GFP can reconstitute in the presence of RNA aptamer and begin fluorescing in about 3 mins. A normal GFP needs about 3 hours to emit its fluroscence. </DIV></DIV><DIV class="qa It is a linker between EGFP and eIF4A that does not interfere with the protein folding of either protein. Also, Glycine and Serine are used as the linker due to their flexibility."><DIV class="q"><SPAN>Q.</SPAN> What is the function and feature of the linkerÂ ? </DIV><DIV class="a"><SPAN>A.</SPAN> It is a linker between EGFP and eIF4A that does not interfere with the protein folding of either protein. Also, Glycine and Serine are used as the linker due to their flexibility. </DIV></DIV><DIV class="qa Using our method, we can study the cell in vivo, rather than having to lyse the cell in order to get to the proteins."><DIV class="q"><SPAN>Q.</SPAN> What is the difference between your method and the antibody technique? </DIV><DIV class="a"><SPAN>A.</SPAN> Using our method, we can study the cell in vivo, rather than having to lyse the cell in order to get to the proteins. </DIV></DIV><DIV class="qa Conventionally mRNA and Protein is produced at the same time. This may cause the GFP that binds with mRNA and RFP that binds with protein to produce the phenomenon known as FRET. However in our experiment, we have added a riboswitch in between the mRNA level and protein levels. Using this switch we can control the flow of gene expression. We can study both mRNA and protein without having to worry about their interaction, nor the interaction of GFP and RFP. "><DIV class="q"><SPAN>Q.</SPAN> In your experiment, will the RFP and GFP FRET? </DIV><DIV class="a"><SPAN>A.</SPAN> Conventionally mRNA and Protein is produced at the same time. This may cause the GFP that binds with mRNA and RFP that binds with protein to produce the phenomenon known as FRET. However in our experiment, we have added a riboswitch in between the mRNA level and protein levels. Using this switch we can control the flow of gene expression. We can study both mRNA and protein without having to worry about their interaction, nor the interaction of GFP and RFP.  </DIV></DIV><H2><SPAN class="mw-headline" id="Speedy_degrader"><FONT color="blue">Speedy degrader</FONT></SPAN></H2><DIV class="qa SsrA tag is encoded by the ssrA RNA. SsrA RNA is recognized as tmRNA because it has characteristics of both tRNA and mRNA. In nature, it labels incomplete peptides, and then directs tagged proteins to several proteases"><DIV class="q"><SPAN>Q.</SPAN> What does SsrA stand for? </DIV><DIV class="a"><SPAN>A.</SPAN> SsrA tag is encoded by the ssrA RNA. SsrA RNA is recognized as tmRNA because it has characteristics of both tRNA and mRNA. In nature, it labels incomplete peptides, and then directs tagged proteins to several proteases </DIV></DIV><DIV class="qa Because the SsrA-SspB system is ubiquitous for eubacteria, our devices can be easily apply to other prokaryotic systems."><DIV class="q"><SPAN>Q.</SPAN> There are many ways to degrade proteins. Among these methods, why choose the SsrA tag? What are the benefits? </DIV><DIV class="a"><SPAN>A.</SPAN> Because the SsrA-SspB system is ubiquitous for eubacteria, our devices can be easily apply to other prokaryotic systems. </DIV></DIV><DIV class="qa We want to apply the ssrA tags to different fluorescent proteins. We contribute biobricks for iGEM, providing instant fast-degraded reporters for other iGEM teams."><DIV class="q"><SPAN>Q.</SPAN> Many scientists have already done research on the SsrA taq. What is the purpose of our experiments? </DIV><DIV class="a"><SPAN>A.</SPAN> We want to apply the ssrA tags to different fluorescent proteins. We contribute biobricks for iGEM, providing instant fast-degraded reporters for other iGEM teams. </DIV></DIV><DIV class="qa Previous studies suggest LVA tag has been the most efficient tag to shorten the half-life of tagged protein in Escherichia coli."><DIV class="q"><SPAN>Q.</SPAN> Why do we choose LVA tag for degradation among tags with the same function? </DIV><DIV class="a"><SPAN>A.</SPAN> Previous studies suggest LVA tag has been the most efficient tag to shorten the half-life of tagged protein in Escherichia coli. </DIV></DIV><DIV id="bottomlinkbox"><A id="bottomlink" href="https://2010.igem.org/Team:NYMU-Taipei">NYMU-Taipei iGEM 2010 Team</A></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:NYMU-Taipei/FAQ">http://2010.igem.org/Team:NYMU-Taipei/FAQ</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NYMU-Taipei/FAQ" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NYMU-Taipei/FAQ" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NYMU-Taipei/FAQ&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NYMU-Taipei/FAQ&amp;oldid=201058" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>