"
Page
Discussion
View source
History
teams
Log in
 
Team:UIUC-Illinois/Project/Future/Scaffold
From 2012.igem.org
Header
Home
2012
2012e
2011
2010
2010
Software
2009
2009
Software
2008
Team
Undergrads
Advisors
Blog
About Us
Overview
Project
Overview
PUF
RNA
Scaffold
Data
Results
PUF Data
RNA
Scaffold
Data
Biobricks
Ancillary
Overview
Petrobrick
Testing
Assembly
Line
Notebook
Protocols
Meeting
Notes
Lab
Notebook
Outreach
Overview
Safety
Overview
Attributions
Overview
Scaffold
RNA Scaffold
Overview
RNA Scaffold Design
PUF Tethering Design
RNA Scaffold Overview
In order to provide a direct application for the RNA binding abilities of PUF, an RNA scaffold was designed with the idea of serving as a platform for an enzyme conveyor belt. The array of enzymatic pathways which could be enhanced by a scaffold are numerous, though, we projected to increase efficiency and production of a resveratrol derivative called piceatannol.
The project consists of a couple parts, each a proof of concept and build-up of previous ones. The start of the project consisted of designing an RNA scaffold which is best tailored to PUF binding in a spatially specific manner. Once the scaffold was designed, synthesized, and purified it was important to show not only that PUF can bind specifically to its designated sites, but that the scaffold can support a concept such as a biological conveyor belt.
One assay which was designed to prove this was incubation of the RNA scaffold with non-specific endonucleases bound to PUF. The length of digested RNA parts would prove that PUF was binding specifically and appropriately to the designated sequences. Another assay would include tethering a split-fluorescent protein to wild-type and mutant PUF. An in-vitro gel-shift assay, or EMSA, would once again prove that PUF is binding the the appropriate sites. More importantly, an in-vivo experiment which shows fluorescence with presence of the scaffold and darkness without the scaffold would prove efficient enzymatic pathways could be achieved.
RNA Scaffold Overview
In order to provide a direct application for the RNA binding abilities of PUF, an RNA scaffold was designed with the idea of serving as a platform for an enzyme conveyor belt. The array of enzymatic pathways which could be enhanced by a scaffold are numerous, though, we projected to increase efficiency and production of a resveratrol derivative called piceatannol.
The project consists of a couple parts, each a proof of concept and build-up of previous ones. The start of the project consisted of designing an RNA scaffold which is best tailored to PUF binding in a spatially specific manner. Once the scaffold was designed, synthesized, and purified it was important to show not only that PUF can bind specifically to its designated sites, but that the scaffold can support a concept such as a biological conveyor belt.
One assay which was designed to prove this was incubation of the RNA scaffold with non-specific endonucleases bound to PUF. The length of digested RNA parts would prove that PUF was binding specifically and appropriately to the designated sequences. Another assay would include tethering a split-fluorescent protein to wild-type and mutant PUF. An in-vitro gel-shift assay, or EMSA, would once again prove that PUF is binding the the appropriate sites. More importantly, an in-vivo experiment which shows fluorescence with presence of the scaffold and darkness without the scaffold would prove efficient enzymatic pathways could be achieved.
RNA Scaffold Design
Fig. 1.[1] The design of our RNA scaffold was based upon a scaffold created by the Pam Silver research lab at Harvard. This group was the only one to develop such a construct (pictured above) and prove its effectiveness so it was built upon in order to serve as the application for our own RNA binding proteins.
The d0 scaffold which is shown above has two hairpin loops with binding sites for two distinct RNA binding proteins, MS2 and PP7. Although the Silver group developed even more complicated scaffolds, we decided to work with the simplest (d0) as a way to prove not only that PUF can bind specifically, but that the scaffold can be used for efficient production of compounds in bacterial cells.
Fig. 2. The sequence in Fig. 2 is the DNA sequence coding for the d0 scaffold. The first highlighted potion is the T7 promoter followed by the MS2 binding site. The next highlighted region shows the PP7 binding site followed by the T7 terminator.
Fig. 3. The corresponding image of the d0 RNA secondary structure as predicted by the LocARNA tool of Freiburg RNA Tools . The software accurately depicts what the d0 structure looks like in the literature, therefore it was a reliable program which could be used to modify and visualize RNA secondary structures.
Fig. 4. Modifications to the d0 sequence were made by replacing the PP7 and MS2 binding sites with WT PUF-PIN and 6-2/7-2 PUF-PIN binding sites.
Fig. 5. The resulting scaffold of the changed sequence.
Fig. 6. The scaffold was further modified after research suggested that PUF binds best to nucleotides with an angle of curvature similar to its own of approximately 20o turn per repeat [3]. The hairpin loops were changed in order to accommodate this from 8 nucleotides to 18 nucleotides achieving a 20o turn per nucleotide effect. Sequences of the stem loop were further modified in order to keep GC content from being too high and to make a more stable structure. This DNA sequence was then synthesized through IDT’s miniGENE option.
Fig. 7. The final structure of the scaffold used in the project.
Fig. 8.[1] The figure above shows an important theoretical concept of the RNA scaffold. Assuming that the RNA binding proteins bind specifically, a spatial control of “cargo” can be made to serve various functions. In our project, we envisioned enzymes of the piceatannol pathway to churn out product due to the spatial proximity of one enzyme next to another, yielding increased reaction kinetics. However, in order to prove that such a concept is possible, a few assays need to be done.
Fig. 9.[1] A gel-shift in-vitro assay as the one pictured would properly demonstrate that distinct RNA binding proteins are binding to the scaffold and thus proper scaffold functioning. With addition of the scaffold, each separate binding protein causes a change in overall size resulting in a different band from each protein by itself. Due to assay shown being ran on a native gel, secondary structures and changes in electronegative affinities might explain the unusual effect of larger constructs being localized further down the gel.
Fig. 10.[2] This figure is shown from the Prof. Wang’s lab in UNC which characterized PUF-PIN (PIN being a non-specific endonuclease) functioning at various pH levels. A similar assay with our scaffold would show specific binding of PUF to its destined sites on the scaffold due to the presence of RNA fragments of expected lengths.
References:
[1]:
Organization of Intracellular Reactions with Rationally Designed RNA Assemblies
Camille J. Delebecque,  Ariel B. Lindner,  Pamela A. Silver,  and Faisal A. Aldaye
Science 22 July 2011: 333 (6041), 470-474.Published online 23 June 2011 [DOI:10.1126/science.1206938]
[2]:
Engineering RNA Endonucleases with Customized Sequence Specificities
Rajarshi Choudhury, Daniel Dominguez, Yang Wang and Zefeng Wang
Department of Pharmacology and Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, April 11, 2012
[3]:
Yeming Wang,  Laura Opperman,  Marvin Wickens,  and Traci M. Tanaka Hall. Structural basis for specific recognition of multiple mRNA targets by a PUF regulatory protein. PNAS 2009 ; published ahead of print November 9, 2009,doi:10.1073/pnas.0812076106
PUF Tether Design
Fig. 1.[1] An example of a conclusive experiment which would prove that spatial control of enzymes is possible is depicted above. With addition of split fluorescent parts to each separate RNA binding protein observed fluorescence with expression of the scaffold is expected. This result would prove spatial control of enzymes is possible and a desired effect, such as an enzyme conveyer belt, is achievable.
Fig. 2.[1] Fluorescence microscopy would show that activity of the fluorescent protein only occurs with addition of the RNA scaffold. The scaffold brings the split-fluorescent parts to close proximity resulting in a drastic effect such as an almost 100 fold in fluorescence.
Fig. 3. In order to tether split-fluorescent proteins to PUF, primer extension was used with BioBricks from the Parts Registry. Specifically, parts BBa_K157005 (Split-Cerulean-CFP) and BBa_K157006 (Split-Cerulean-nCFP) were worked with after the 2010 Slovenia iGEM team showed conclusive results testing them. Linker sequences used in literature which bound PUF to endonucleases were included in the primer sequences to create the most optimal construct [2]. Primes on the outside of the gene parts included restriction enzymes compatible with standard assembly methods for future biobricking work.
References:
[1]:
Organization of Intracellular Reactions with Rationally Designed RNA Assemblies
Camille J. Delebecque,  Ariel B. Lindner,  Pamela A. Silver,  and Faisal A. Aldaye
Science 22 July 2011: 333 (6041), 470-474.Published online 23 June 2011 [DOI:10.1126/science.1206938]
[2]:
Engineering RNA Endonucleases with Customized Sequence Specificities  Rajarshi Choudhury, Daniel Dominguez, Yang Wang and Zefeng Wang  Department of Pharmacology and Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, April 11, 2012
Retrieved from " http://2012.igem.org/Team:UIUC-Illinois/Project/Future/Scaffold "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
