<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UIUC-Illinois_Notebook_Protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UIUC-Illinois/Notebook/Protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UIUC-Illinois/Notebook/Protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UIUC-Illinois/Notebook/Protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UIUC-Illinois/Notebook/Protocols&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UIUC-Illinois/Notebook/Protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UIUC-Illinois/Notebook/Protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3></DIV></DIV></DIV><TITLE>Header</TITLE><DIV id="headernavigation"><UL id="headernavbar"><LI><A href="https://2012.igem.org/Team:UIUC-Illinois">Home</A></LI><LI><A href="https://2012.igem.org/Team:UIUC-Illinois">2012</A></LI><LI><A href="https://2012e.igem.org/Team:UIUC_Illinois">2012e</A></LI><LI><A href="https://2011.igem.org/Team:UIUC-Illinois">2011</A></LI><LI><A href="https://2010.igem.org/Team:UIUC-Illinois">2010</A></LI><LI><A href="https://2010.igem.org/Team:UIUC-Illinois-Software">2010  Software</A></LI><LI><A href="https://2009.igem.org/Team:Illinois">2009</A></LI><LI><A href="https://2009.igem.org/Team:Illinois-Tools">2009  Software</A></LI><LI><A href="https://2008.igem.org/Team:Illinois">2008</A></LI></UL><LI><A href="https://2012.igem.org/Team:UIUC-Illinois/Team/AboutUs">Team</A></LI><LI><A href="https://2012.igem.org/Team:UIUC-Illinois/Team/Undergrads">Undergrads</A></LI><LI><A href="https://2012.igem.org/Team:UIUC-Illinois/Team/Advisors">Advisors</A></LI><LI><A href="http://igemuiuc2012.blogspot.com/">Blog</A></LI><LI><A href="https://2012.igem.org/Team:UIUC-Illinois/Team/AboutUs">About Us</A></LI><LI><A href="https://2012.igem.org/Team:UIUC-Illinois/Team/AboutUs">Overview</A></LI><LI><A href="https://2012.igem.org/Team:UIUC-Illinois/Project">Project</A></LI><LI><A href="https://2012.igem.org/Team:UIUC-Illinois/Project">Overview</A></LI><LI><A href="https://2012.igem.org/Team:UIUC-Illinois/Project/Design">PUF</A></LI><LI><A href="https://2012.igem.org/Team:UIUC-Illinois/Project/Future/Scaffold">RNA  Scaffold</A></LI><LI><A href="https://2012.igem.org/Team:UIUC-Illinois/Results">Data</A></LI><LI><A href="https://2012.igem.org/Team:UIUC-Illinois/Results">Results</A></LI><LI><A href="https://2012.igem.org/Team:UIUC-Illinois/Results">PUF Data</A></LI><LI><A href="https://2012.igem.org/Team:UIUC-Illinois/Results/Scaffold">RNA Scaffold Data</A></LI><LI><A href="https://2012.igem.org/Team:UIUC-Illinois/Results/Biobricks">Biobricks</A></LI><LI><A href="https://2012.igem.org/Team:UIUC-Illinois/Project/Future">Ancillary</A></LI><LI><A href="https://2012.igem.org/Team:UIUC-Illinois/Project/Future">Overview</A></LI><LI><A href="https://2012.igem.org/Team:UIUC-Illinois/Project/Future/Petrobrick">Petrobrick Testing</A></LI><LI><A href="https://2012.igem.org/Team:UIUC-Illinois/Project/Future/AssemblyLine">Assembly  Line</A></LI><LI><A href="https://2012.igem.org/Team:UIUC-Illinois/Notebook">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:UIUC-Illinois/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:UIUC-Illinois/Notebook/MeetingNotes">MeetingNotes</A></LI><LI><A href="https://2012.igem.org/Team:UIUC-Illinois/Notebook/LabNotebook">LabNotebook</A></LI><LI><A href="https://2012.igem.org/Team:UIUC-Illinois/Outreach">Outreach</A></LI><LI><A href="https://2012.igem.org/Team:UIUC-Illinois/Outreach">Overview</A></LI><LI><A href="https://2012.igem.org/Team:UIUC-Illinois/Safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:UIUC-Illinois/Safety">Overview</A></LI><LI><A href="https://2012.igem.org/Team:UIUC-Illinois/Attributions">Attributions</A></LI><LI><A href="https://2012.igem.org/Team:UIUC-Illinois/Attributions">Overview</A></LI></DIV><P><TITLE>Protocols</TITLE></P><DIV id="protocol-container"><DIV id="protocolselection"><CENTER><H2>Protocol Selection</H2></CENTER><LI><A name="prot1">Bootcamp Protocols</A></LI><LI><A name="prot2">Digestions</A></LI><LI><A name="prot3">Gel Purification</A></LI><LI><A name="prot4">Inoculation</A></LI><LI><A name="prot5">Ligation</A></LI><LI><A name="prot6">Making Electrocompetent E.Coli</A></LI><LI><A name="prot7">Making Electrophoresis Gels</A></LI><LI><A name="prot8">Making TAE Buffers</A></LI><LI><A name="prot9">Miniprep</A></LI><LI><A name="prot10">PCR Protocols</A></LI><LI><A name="prot11">Storage of Cells</A></LI><LI><A name="prot12">Subculturing Plates</A></LI><LI><A name="prot13">Transformation of E.Coli</A></LI><LI><A name="prot16">mCherry Experiments</A></LI><LI><A name="prot14">4CL:STS Sequencing</A></LI><LI><A name="prot15">Biosynthesis of piceatannol</A></LI></DIV><DIV id="protocoloverview"><CENTER><H2>UIUC iGEM Protocols</H2></CENTER><CENTER><P>The standard protocols for each technique used in our project endeavors have been documented. Unless further noted <A href="http://www.scs.illinois.edu/rao/index.php">all procedures are based off of those used by the lab of C. V. Rao</A>.</P></CENTER><DIV id="prot1" style="display:none"><CENTER><H1>Bootcamp Protocols</H1></CENTER><H2>Making LB for plates</H2>
To make 1 Liter of LB: 
1. Always use a 10:5:5 ratio of Tryptone to Yeast Extract to NaCl. 
2. Add dry ingredients first. 
3. Use a 2L flask. 
4. Add the following 
- 10 g Tryptone  
- 5 g Yeast Extract 
- 5 g NaCl 
- 1.5 g Agar (NOT agarose!)
4. Then add 1 L of MilliQ water. 
5. Autoclave by total volume.
6. Pour 25 mL on each plate (just enough to cover the bottom). <H2>Making Liquid Media </H2>
1. Always use a 5-2.5-2.5 ratio of Tryptone to Yeast Extract to NaCl 
2. Add dry ingredients first, then add MilliQ water 
3. No Agar! 
4. Autoclave by total volume <H2>Making Glycerol Stock </H2>
To make 400 mL of 10% glycerol you will need:

-40 mL glycerol
-360 mL of MilliQ water 

To make 400 mL of 20% glycerol you will need:

-80 mL glycerol 
-320 mL of MilliQ water 

1. Use flasks and bottles before graduated cylinders (they take forever to mix!)
2. Need a stir plate and a large stir bar 
3. Stir until mixture is homogeneous 
4. Don’t autoclave! <H2>Making ddH2O</H2>
- Use the small bottles. 
- Autoclave water by total volume. <H2>Autoclaving </H2>
- Robert is the source of official help on all things about the Autoclave. 
1. Use the autoclave tape. 
2. Look at the Betastar for settings. 
3. Go by materials and total volume. 
4. Make sure the autoclave tape has changed colour by the end of the cycle. Check conditions and settings then repeat steps 1-4 if tape has not changed colour. <H2>Ligation </H2>
We used the <A href="http://ginkgobioworks.com/support/BioBrick_Assembly_Manual.pdf">Ginkgo bioworks protocol</A>
1. Control: 
- 2 uL of Circular PSB1C3 plasmid 13 uL ddH2O 
- 2 uL T4 ligase buffer
- 1 uL T4 ligase 
2. P&amp;C: 
- 2 uL of Circular PSB1C3 plasmid
- 2 uL of PUF, 11 uL ddH2O
- 2 uL T4 ligase buffer
- 1 uL T4 ligase
3. L&amp;P:
- 2 uL of linear PSB1C3 plasmid
- 2 uL of PUF
- 11 uL ddH2O
- 2 uL T4 ligase buffer
- 1 uL T4 ligase</DIV><DIV id="prot2" style="display:none"><CENTER><H1>Digestions</H1></CENTER>
Reaction Mix:
- 500 ng – Template
- 5 µl – 10x Buffer
- 1 µl – NEB Enzyme 1 (10 Units/µl) = 10 Units in 50 µl reaction
- 1 µl – NEB Enzyme 2 (10 Units/µl) = 10 Units in 50 µl reaction
- dH2O to 50 µl
- BSA
- TOTAL – 50 µl
Procedure:
1. Calculate how much template is needed for digestion. Then calculate how much water needs to be added to make a 50 µl reaction.
2. Pipette appropriate amounts of dH20, template, and 10x Buffer in that order into a PCR tube (200 µl). Mix reagents together by vortexing and then tapping tube on desk to keep reagents on the bottom of the PCR tube.
3. Add appropriate amounts of Enzyme 1 and Enzyme 2. Add .5ul BSA, or other volume depending on BSA content in enzymes. Once added gently swirl around reaction mix with pipette tip.
4. Incubate at 37o C for 1 – 2 hours
5. Incubate at 80 oC for 20 minutes to deactivate the restriction enzymes 
6. Run a gel of your digestions on a gel with low temperature melting agarose. Run at 100V for 1 hr. 30 min.
7. Look at gel under 320nm UV light. Conduct gel extraction purification.
NOTE: When digesting your vector it may be useful to use 2 µl of each enzyme in the 50 µl reaction and extending incubation time.</DIV><DIV id="prot3" style="display:none"><CENTER><H1>Gel Purification</H1></CENTER>
What you need:
- The sample of interest, typically a DNA double-digest (50 uL)
- A low-melt gel
- Razor blade
- Autoclaved 1.5 mL centrifuge tubes
- Agarose-dissolving buffer (ADB)
- Big Zymo-spin columns in collection tubes
- DNA wash buffer
- Autoclaved MilliQ water
Procedure:
1. Place each sample into a 1.5 mL centrifuge tube
2. To each tube, add 3 volumes of ADB (i.e for 100 uL gel slice add 300 uL ADB)
3. Incubate at 37-55˚C in the warm water bath until the gel is completely dissolved (if DNA fragment is larger than 8kb, add one additional gel volume of water to the tube following incubation for better DNA recovery)
4. Transfer the melted agarose solution to a Zymo-spin column in a collection tube
5. Centrifuge for 30 seconds and discard liquid flow-through
6. To the column, add 200uL wash buffer
7. Centrifuge for 30 seconds and discard the liquid flow-through
8. Repeat steps 6-7
9. Centrifuge the empty column for 30 seconds
10. Place column in a new 1.5 mL centrifuge tube
11. Gently add 30uL autoclaved water to the column
12. Let column stand for 2 minutes
13. Centrifuge for 30-60 seconds to elute pure DNA into the tube and store at -20˚C until use</DIV><DIV id="prot4" style="display:none"><CENTER><H1>Inoculation</H1></CENTER>
What you need:
- Falcon tubes
- The appropriate liquid media
- The appropriate antibiotic
- An agar plate containing the cells of interest or a frozen stock solution
- Autoclaved pipet tips
Procedure: 
1. To an autoclaved glass test tube with lid, add: 2-5 mL liquid media, 2-5 ul antibiotic (dilute to 1/1000)
2. Using a pipet tip,  gently poke a single colony of plated cells, or scratch cells onto the stick form the frozen freezer stock (taking colonies from a plate typically yields better results)
3. Stir cells into the liquid media
4. Label the test tubes and place them in 37˚C warm room overnight to grow (12-16 hrs)</DIV><DIV id="prot5" style="display:none"><CENTER><H1>Ligations</H1></CENTER>
Set-up:
1. Run a gel of your digestions and take a picture.
2. Estimate the intensities of your insert and vector where your insert (usually less intense) is considered 1 and your vector is x times as bright.
3. Once intensity is determined perform the following equation:

L = Ligation volume 
S = Size of fragments 
I = Intensity of fragments 
V = Volume ran down gel 
i = Insert 
v = Vector

4. Ligation volume (Lv) for the vector should be approximately 2 µl.
5. The ratio for the ligation in the previous equation is 3:1 insert:vector but can be done 6:1 by replacing the 3 with a 6.
Reaction Mix:
- 2 µl – Vector
- Li µl – Insert
- 2 µl – 10x Buffer
- dH20 to 19 µl
- 1 µl – T4 DNA Ligase
- TOTAL – 20 µl 

Procedure:
1. Add appropriate amounts of dH20, Vector, and Insert in a PCR tube (200 µl). Once added, heat mixture to 65o C for 5 min. This will remove all items that are already annealed to each other causing inefficient ligation.
2. Add then appropriate amounts of 10x Buffer and T4 DNA Ligase. Make sure not to pipette up and down or vortex to mix, this will shear the ligase. Gently swirl with the tip of your pipette to mix.
3. Incubate reaction at room temperature for 30 minutes.
4. After incubation heat inactivate the ligase by another incubation at 80o C for 20 min.
NOTE: The ligase buffer should smell like wet dog. If you smell nothing from the tube, then the buffer is old and useless.
Can also use the Ginkgo Bioworks protocol: 
1. Get the 10X T4 DNA ligase buffer and the T4 DNA ligase from the 20 degree storage. 
2. Put them on ice to thaw. 
3. In a PCR tube add 11 uL of water. 
4. Add 2 uL of each digest to the tube. 
5. Use a pipet to thoroughly mix the T4 DNA ligase buffer and resuspend the precipitate in the bottom. Then add 2 uL to the PCR tube. 
6. Add 1 uL of T4 DNA ligase. 
7. Flick the tube to mix it. If necessary, put it in the microcentrifuge to get all the liquid back to the bottom. </DIV><DIV id="prot6" style="display:none"><CENTER><H1>Making electrocompetent E.Coli cells</H1></CENTER>

What you need: 
- 50mL subculture of E.coli cells
- 50mL conicial tubes
- Cold 10% glycerol solution  
- Autoclaved 1.5mL centrifuge tubes
Procedure:
1. Divide liquid cell subculture into 50mL conical tubes and centrifuge for 10 minute at 4000rpm 
2. Discard supernatant and add 25mL cold 10% glycerol   
3. Stir the mixture by mixing and centrifuge for an additional 10 minutes
4. Repeat steps 2-3 two more times
5. Discard the supernatant and resuspend the cells in 1mL 10% glycerol
6. Transfer to 1.5mL centrifuge tubes and centrifuge for 2 minutes
7. Discard the supernantant and resuspend the cells in 2 volumes (cell volumes) 10% glycerol
8. Store at -80˚C</DIV><DIV id="prot7" style="display:none"><CENTER><H1>Making Gel Electrophoresis gel solutions</H1></CENTER><H2>To check PCR results</H2>

What you need:
- High-melt agarose
- TAE buffer 1X
- Autoclaved container 

Procedure:
1. Add 3.2g high-melt agarose to 400 mL 1X TAE buffer (50 uL agarose gel, 250 mL 1X TAE buffer for the small gel box) 
2. Microwave flask or bottle (cap placed loosely if bottle) of mixed solution to dissolve gel in 1 minute increments until all agarose is dissolved
3. BE CAREFUL AS LIQUID CAN BE SUPERHEATED<H2>To do a gel purification</H2>
What you need:
- Low-melt agarose
- TAE buffer 1X
- Autoclaved container 
Procedure:
1. Add 4.0g low-melt agarose to 400 mL 1X TAE buffer
2. Microwave flask or bottle (cap placed loosely if bottle) of mixed solution to dissolve gel in 1 minute increments until all agarose is dissolved
3. BE CAREFUL AS LIQUID CAN BE SUPERHEATED</DIV><DIV id="prot8" style="display:none"><CENTER><H1>Buffer Protocols</H1></CENTER><H2>Making 50X TAE buffer</H2>

What you need: 
- Tris base
- Glacial acetic acid
- Na2EDTA•2H2O
- MilliQ water

Procedure:
1. In a 1000mL pyrex bottle add and mix: 
- 242 Tris base
- 57.1mL glacial acetic acid
- 37.2g Na2EDTA•2H2O
- 1000mL MilliQ water
2. Mix thoroughly, no sterilization necessary<H2>Making 1X TAE buffer</H2>

What you need: 
- 50X TAE buffer 
- MilliQ water 

Procedure:

1. Add 20mL 50X TAE buffer to 980mL MilliQ water and mix to make 1L of 1X TAE buffer stock </DIV><DIV id="prot9" style="display:none"><CENTER><H1>Miniprep</H1></CENTER>
A miniprep is done to purify plasmids from a strain 
What you need:
- An overnight culture of your plasmid-containing cells
- Autoclaved 1.5mL centrifuge tubes
- Resuspension solution containing RNase A
- Lysis solution
- Neutralization solution
- Large Zymo-spin columns with collection tubes
- DNA wash buffer
- Autoclaved MilliQ water
Procedure:
1. To a 1.5 mL centrifuge tube, add 1.5 mL of overnight culture
2. Centrifuge at full speed for 2 minutes and discard supernatant
3. Resuspend the cells by pipetting in 250 uL resuspension solution
4. Add 250 uL lysis solution and mix by inversion (about 30 inversions)
5. Add 350 uL neutralization solution and mix by inversion
6. Centrifuge the mixture at full speed for 5 minutes
7. Transfer the supernatant to a Zymo-spin column with collection tube
8. Centrifuge for 1 minute and discard the flow-through in collection tube
9. Add 500 uL wash buffer to column
10. Centrifuge for 1 minute and discard the supernatant
11. Repeat steps 9-10
12. Spin the empty column for an additional minute
13. Place the column in a new 1.5 mL centrifuge tube
14. Add 50 uL autoclaved water directly to the column
15. Let the column incubate at room temperature for  2 minutes
16. Spin the column in the centrifuge for 2 minutes
17. Store the liquid at -20˚C until use</DIV><DIV id="prot10" style="display:none"><CENTER><H1>PCR Protocols</H1></CENTER><H2>PCR</H2>
What you need:
- A container of ice
- Small PCR tubes
- Template DNA containing the gene of interest
- DNA primers designed to amplify the gene of interest
- Phusion DNA polymerase
- dNTP solution
- DMSO (optional)
- 5X HF or GC buffer
- Autoclaved MilliQ water
Procedure:
1. Keep all components on ice as you work
2. Label clearly one small PCR tube for each sample you wish to run. 
3. For each sample, mix the following:
- 0.75 uL template DNA 
- 1uL each of the two primers
- 0.5uL DNA polymerase
- 1uL dNTPs
- 1.5 uL DMSO (optional)
- 10uL 5X HF/CG buffer
- 33.5 uL autoclaved water
4. Use your pipet to mix thoroughly
5. Load samples in the PCR machine and select the phusion protocol, using an extension time of roughly one minute per kb of DNA to be amplified. However – consult the bootcamp optimized PCR protocol if necessary. 
Note: When running PCR with the supermix use 25 uL of supermix, 1.5 uL of DMSO (optional), 0.5uL DNA polymerase, 1uL each of the two primers, 0.75 uL of template, and autoclaved water to 50uL volume. <H2>PCR Cleanup</H2>
What you need:
- Large Zymo-spin columns in collection tubes
- The completed PCR of interest
- DNA binding buffer
- DNA wash buffer
- DNA wash buffer
- Autoclaved 1.5 mL centrifuge tubes
- Autoclaved MilliQ water
Procedure:
1. To the PCR liquid, add 2 volumes of DNA binding buffer
2. Mix briefly by pipetting and add all of the mixture to a large Zymo-spin column
3. Centrifuge at full speed for 30 seconds
4. Discard the waste liquid in the collection tube
5. To the column, add 200 uL wash buffer
6. Centrifuge for 30 seconds and discard the waste liquid
7. Repeat steps 5-6
8. Centrifuge the empty column for 1 minute
9. Replace the collection tube with 1.5 mL centrifuge tube
10. Gently add 30uL of autoclaved water to the column
11. Let the column incubate at room temperature for 2 minutes
12. Centrifuge 1 minute
13. Discard the column 
14. Label the collected liquid and store it in the -20˚C freezer until use<H2>Colony PCR</H2>
What you need:
- Autoclaved 1.5 mL centrifuge tubes
- Autoclaved MilliQ water
- Green master mix
- Primers complementary to the gene of interest, that specifically reveal both its presence and its location
- PCR strip tubes
- An agar plate containing colonies to be tested
- Autoclaved toothpicks
Procedure:
1. In a 1.5mL centrifuge tube, mix 12.5uL*(# of samples) of autoclaved water, 12.5uL*(# of samples) of green mix, and 1uL*(# of samples of each primer
2. Add 27uL of the mixture to each tube of the PCR strip tubes
3. Number the colonies you wish to test as well as the tubes of liquid
4. With an autoclaved toothpick, gently poke each colony and stir it into the corresponding liquid 
5. Be sure all liquid is in the bottom of the tubes (spin tubes in centrifuge if it isn’t)
6. Run the samples in the PCR machine, using the colony PCR program with an extension time of roughly one minute per kb of DNA to be amplified by your primers
7. Analyze samples on a gel<H2>PCR Troubleshooting</H2>

Ran a diagnostic test to optimize buffer and temperature for the PCR. After doing the PCR (it’s nice to do this in linked PCR tubes) we run on a gel to test which worked best.

Buffers tested: GC + DMSO, HF + DMSO, GC, buffer G supermix + DMSO (3 tubes of each)

Made a master mix of primers, phusion, template DNA, and dNTPs. This was then used this for all the tubes of GC+DMSO, HF+DMSO, and GC.
 
Master Mix: 
- 10 uL dNTP’s
- 2 uL forward primers
- 2 uL reverse primers
- 2 uL template DNA
- 1 uL phusion 

GC + DMSO: 
- 18 uL H2O
- 6 uL GC buffer
- 5.1 uL of master mix
- 0.9 uL DMSO
- now divide into 3 tubes (10 uL into each PCR tube) 

HF + DMSO:
- 18 uL H2O
- 6 uL HF buffer
- 5.1 uL of master mix
- 0.9 uL DMSO
Now divide into 3 tubes (10 uL into each PCR tube)

GC:
- 18.9 uL H2O
- 6 uL GC buffer
- 5.1 uL master mix
Now divide into 3 tubes (10 uL into each PCR tube) 

Supermix:
- 15 uL buffer G
- 12 uL H2O
- 0.9 uL DMSO
- 0.6 uL forward primer
- 0.6 uL reverse primer
- 0.6 uL template DNA
- 0.3 uL phusion 

PCR program: 

1. 98 degrees Celsius for 3 minutes 
2. 98 degrees C for 15 seconds 
3. Temperature gradient 48-55-63 degrees for 30 seconds 
4. 72 degrees C for 1 minute 
5. Go to step 2, repeat 30 times 
6. 72 degrees C for 5 minutes 
Gel results: Only buffer G +DMSO works and it works at all temperatures.<H2>Optomized PCR</H2>
Worked using buffer G +DMSO supermix at 63o C. 
Supermix (for three reactions. 10uL/rxn):
- 15 uL buffer G
- 12 uL H2O
- 0.9 uL DMSO
- 0.6 uL forward primer
- 0.6 uL reverse primer
- 0.6 uL template DNA
- 0.3 uL phusion 

PCR program (running time of just under 2 hours): 
1. 98 degrees Celsius for 3 minutes 
2. 98 degrees C for 15 seconds 
3. 63 degrees C for 30 seconds 
4. 72 degrees C for 1 minute 
5. Go to step 2, repeat 30 times 
6. 72 degrees C for 5 minutes </DIV><DIV id="prot11" style="display:none"><CENTER><H1>Storing cells in Glycerol Solution</H1></CENTER>
What you need:
- 1.5 mL cryogenic vials
- 50% glycerol solution (can also use 20%) 
- An overnight culture of the cells of interest
Procedure:
1. To a cryogenic vial, add 600 uL 50% glycerol solution and 400 uL liquid cell culture
2. Mix thoroughy and store in the -80˚C freezer until use 

Unless noted, <A href="http://www.scs.illinois.edu/rao/index.php">all procedures are based on those in the lab of Christopher Rao</A>. </DIV><DIV id="prot12" style="display:none"><CENTER><H1>Subculturing E. Coli</H1></CENTER>
What you need: 
- Autoclaved 250 mL flasks 
- Liquid Media 
- Antibiotic, if necessary (not for DH5a c competent cells) 
- An overnight culture of the cells of interest 

Procedure: 

1. Into your autoclaved flask add:
 - 50 mL liquid media 
 - 50 ul antibiotic solution if necessary 
 - 500 uL overnight liquid culture. 

Note: there is a 1:100 ratio of cells to media. Furthermore there is a 1:1000 ratio of antibiotic to media. When subculturing DH5alpha, antibiotic solutions are not required. 

2. Incubate in the 37˚C warm room on a stirrer for 4-6 hours 
Note: Need to check the cells 2 hours after inoculation to test their optical density (OD). The best competent cells come from cells in exponential phase of their growth. Therefore an OD of 0.4-0.8 is desired and this happens after 2-4 hours of incubation. </DIV><DIV id="prot13" style="display:none"><CENTER><H1>Electroporation and Transformation</H1></CENTER>
You will need:
- 100 µl – Electrocompetent Cells
- 2-5 µl – Ligation Reaction (10 – 100 ng DNA)
- 1 ml – SOC broth/ LB liquid media
- Electrocuvette

Procedure:
1. Gather all your supplies and keep them on ice (or in 4o C room). This includes the cuvettes. 
2. Thaw frozen electrocompetent cells on ice and place in electrocuvette. Then add ligation reaction and tap the cuvette to mix well. 3. DO NOT pipette up and down.
4. Keep on ice for 1 min.
5. Pulse once (On machine use Ec1 for 1mm cuvettes and Ec2 for 2mm cuvettes).
6. IMMEDIATELY after add 1ml of SOC broth and pipette up and down gently.
7. Incubate recovering cells in 37o C for 1 hr.
Plating:
1. After incubation, plate 200 µl of transformants on to appropriate selective plate and incubate at 37o C for 18+ hrs.
2. If no colonies are formed plate out the rest of the transformants (~800 µl) and incubate at 37o C for 24 hrs</DIV><DIV id="prot14" style="display:none"><CENTER><H2>To sequence the 4CL:STS part from the parts registry</H2></CENTER>

1. PCR the 4CL:STS mini prep DNA in the plastic box labeled “Other” in the -20C (this will be your template in the PCR rxn) Please do 6 total rxns for a total of 300 ul.
For (3) 50 ul rxns:
	- 99.75 ul H2O
	- 30.00 ul HF Buffer
	- 3.00 ul dNTPs
	- 7.50 ul primer 1 (VF2 is the name 20 bp, TempMelt= 63.8C GC%= 50)
	- 7.50 ul primer 2 (VF is the name, 20 bp, TempMelt= 63.4C, GC%= 50)
	- 1.50 ul of template (microfuge tube of 4CL:STS mini prep in the “Other” box in -20C)
	- 0.75 ul of Phusion (preferably Phusion Hot start)
	- 150 ul total
2. Divide this 150 ul among 3 small PCR tubes
Put them in the PCR machine with the conditions:
	1. 98 deg C for 3:00 min
	2. 98 deg C for 0:15 sec
        3. 64 deg C for 0:15 sec
        4. 72 deg C for 1:30 min
        * Go back to step 2, repeat 34 times*
        5. 72 deg C for 5:00 min
        6. 12 deg C for infinty 
3. Make a 1% Agarose gel to gel purify the PCR, use the small comb that creates large wells, use the 1kb ladder (1.5 ul) in one lane. In another lane use all of the PCR product + 15 ul of 6X loading dye to dye the product. MAKE SURE IT IS A THICK GEL (use approx 50-70 ml of gel)
	Band calculations:
	- VF2 amplifies from approx 2020 bp of pSB1A3 to 2155 of pSB1A3= 135 bp
	- VF amplifies from approx 160 bp of pSB1A3 to 0 bp of pSB1A3= 160 bp
	- 4CL:STS part is 1815 bp long
	- TOTAL PCR band product: 2110 bp *look for this size band*
4. If you get a 2110 bp band, gel purify this band by cutting out the band and following gel purification protocol.	
if the band looks bright, elute with 50ul of H2O and incubate for 1:00 min
	- If the band looks faint, elute with 30 ul of H2O and incubate for 5:00 min
	- If you can barely see the band, elute with 15 ul of H2O and incubate for 5:00 min
5. Make a second 1% Agarose gel to test the purification product, use the small comb that creates small wells, use the 1kb ladder (1.5 ul) in one lane. In another lane use 1 ul of the purification product + 1 ul of 6X loading dye to dye the product (this step is to make sure you didn’t lose your product in the purification step)
- Now these are the details for the next step. You don’t have to rush on this because hopefully I’ll be back to help you start this https://unicorn.biotec.illinois.edu/Sequencing_Protocol.pdf
This is a protocol for the sequencing PCR. We’ll need 75-100ng of DNA in order to start this step, so please do maybe 6 PCRs in step 1 to get enough DNA for this.</DIV><DIV id="prot15" style="display:none"><CENTER><H2>Biosynthesis of Piceatannol from Resveratrol in E. Coli</H2></CENTER>

1. Grow BM3 wt, BM3 10, BM3 13, in 4ml of LB + Amp media overnight from gylcerol stock in -80 C freezer
2. Take 250ul of this culture and a dd to 25ml of new LB + 25 ul Amp; grow at 37 C for approx. 2 hours.
3. When OD@600nm = 0.4, add isopropyl-Dthiogalactopyranoside (IPTG) to a final concentration of 0.50 mM. For our 25mL samples we added 12.5 ul of IPTG 1M.
4. Get to mid log phase OD@600nm &gt; 0.8, this should take approx. 2 hours.
5. Transfer contents to Falcon tubes.
6. Pellet for 5:00min at 13.2k rpm and resuspend cells in 25ml of this media:
	- 500ml 1% gylcerol Tris buffer with salene
	-Tris Buffred Saline (TBS), pH 7.4 of 10 mM Tris and 150 mM NaCl.
- 500 ml of TBS can be prepared by dissolving 0.605 g of Tris base and 4.35 g of NaCl in 500 ml of distilled water. Adjust the pH before use.
	- Add 5ml of glycerol
6. Autoclave in 1L container
7. Add 0.75 mg resveratrol (solubility 0.03 g/L) meaning a concentration of 0.03 g/L
8. Grow at 30 C for 36 h, taking 50ul samples of culture
9. Dip the capillary tubes in the 50ul sample and spot on TLC plate
10. Put plate in developing chamber, add ~10ml of Normal buffer to the solution. Cover with the aluminum foil. Develop to ~1cm from the top.</DIV><DIV id="prot16" style="display:none"><CENTER><H2>mCherry Experimental Protocols</H2></CENTER>
1) Inoculate 2 of each of the following cultures in 5ml LB: 
	BL21 (no antibiotic)
	A. 
            mcherry + Wild Type Puf Binding Site in Protet plasmid &amp; Wild Type Puf+Pin 
            (5ul amp* + 5ul cm*)
            mcherry + Wild Type Puf Binding Site in Protet plasmid &amp; Mutant Puf+Pin 
            (5ul amp + 5ul cm)
           B.
            mcherry + Mutant Puf Binding Site in Protet plasmid &amp; Wild Type Puf+Pin 
            (5ul amp + 5ul cm)
	mcherry + Mutant Puf Binding Site in Protet plasmid &amp; Mutant Puf+Pin 
            (5ul amp + 5ul cm)
           C.
	mcherry + Control Binding Site in Protet plasmid &amp; Wild Type Puf+Pin 
            (5ul amp + 5ul cm) 
            mcherry + Control Binding Site in Protet plasmid &amp; Mutant Puf+Pin 
            (5ul amp + 5ul cm)
Grow cultures for 14-16hrs at 37 degrees C

Note: 
*amp: Ampicillin 100 
*cm: Chloramphenicol 34
2) Subculture each culture by adding 50ul of overnight culture into 5ml LB (with appropriate antibiotics as step one). Since there are two of each culture, label one of each duplicate “induced” and the other duplicate “un-induced”. 

Grow cultures for 3 hrs at 37 degrees C
3) Induce the cultures labeled “induced” by adding 1.5ul of the IPTG stock (stock is 1M) for a final concentration of 300 uM IPTG. 

Grow cultures for 6 hrs at 37 degrees C
4) Use the Fluorescence plate reader</DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:UIUC-Illinois/Notebook/Protocols">http://2012.igem.org/Team:UIUC-Illinois/Notebook/Protocols</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UIUC-Illinois/Notebook/Protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UIUC-Illinois/Notebook/Protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UIUC-Illinois/Notebook/Protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UIUC-Illinois/Notebook/Protocols&amp;oldid=281836" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>