<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETH_Zurich_Modeling_SingleCell"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ETH_Zurich/Modeling/SingleCell">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETH_Zurich/Modeling/SingleCell&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Modeling/SingleCell&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Modeling/SingleCell&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/Modeling/SingleCell" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETH Zurich/Modeling/SingleCell</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV class="stickToTop" id="naviBox" style="text-align:center; line-height: 0px; font-size: 0px; z-index: 500;"><DIV style="position: relative; margin: 0 auto; display:inline-block; left: 0px; right: 0px;"><DIV align="center" id="naviMenu" style="left: 0px; right: 0px; margin: 0pt auto; text-align: left;"><UL class="dropdown dropdown-horizontal mainmenu"><LI class="first"><A href="/Team:ETH_Zurich">Home</A></LI><LI><A href="/Team:ETH_Zurich/Overview/Project" class="dir">Project <I>SmoColi</I></A></LI><LI class="first"><A href="/Team:ETH_Zurich/Overview/Project">What is SmoColi?</A></LI><LI><A href="/Team:ETH_Zurich/Overview/CircuitDesign">The Circuit</A></LI><LI><A href="/Team:ETH_Zurich/Overview/Informationprocessing">Information processing</A></LI><LI><A href="/Team:ETH_Zurich/Overview/Dialogue">Dialogue between Biology and Modeling</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Achievements">Achievements</A></LI></UL><LI><A href="/Team:ETH_Zurich/Achievements/Data_page">Data Page</A></LI><LI><A href="/Team:ETH_Zurich/Biology" class="dir">Biology</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Biology">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Biology/Detector">Network Elements</A></LI><LI><A href="/Team:ETH_Zurich/Biology/MolecularMechanism">Genetic Design</A></LI><LI><A href="/Team:ETH_Zurich/Biology/Cloning">Materials and Methods</A></LI><LI><A href="/Team:ETH_Zurich/Biology/Journal">Lab Journal</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Biology/Validation">Results</A></LI><LI><A href="/Team:ETH_Zurich/Process/Microfluidics/Overview" class="dir">Microfluidics</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Process/Microfluidics/Overview" class="dir">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Process/Microfluidics">Channel Design Development</A></LI><LI><A href="/Team:ETH_Zurich/Process/Validation">Final Design and Validation</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Process/Microfluidics/Proof">Proof of Concept</A></LI><LI><A href="/Team:ETH_Zurich/Modeling" class="dir">Modeling</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Modeling">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/SingleCell">Single-Cell Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Analysis">System Analysis</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Stochastic">Stochastic Analysis</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Microfluidics">Reaction-Diffusion Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Combined">Combined 3D Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Analytical_Approximation">Analytical Approximation</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Parameters">Parameters</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/References">References</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Modeling/Code">Code</A></LI><LI><A href="/Team:ETH_Zurich/Team/Human practices" class="dir">Human Practices</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Team/Human practices">Outreach</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Biology/Safety">Safety</A></LI><LI><A href="/Team:ETH_Zurich/Team" class="dir">Team</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Team">Team SmoColi</A></LI><LI><A href="/Team:ETH_Zurich/Team/Media">Photo Gallery</A></LI><LI><A href="/Team:ETH_Zurich/Team/Acknowledgements">Acknowledgements &amp; Contributions</A></LI><LI class="last"><A href="https://igem.org/Team.cgi?year=2011&amp;team_name=ETH_Zurich" target="_blank">Official Team Overview</A></LI><LI id="loginMenu"><A href="#" id="loginLink">Log In</A></LI><LI style="display:none;" id="editMenu"><A href="#" id="editLink" class="dir">Edit</A></LI><LI class="first"><A href="#" id="historyLink">History</A></LI><LI><A href="/Special:Upload" target="_blank">Upload File</A></LI><LI class="last"><A href="#" id="logoutLink">Log Out</A></LI><LI class="dir last" style="background-image: none; position: relative;"><FIELDSET><FORM method="post" id="searchForm" action=""><LABEL for="s">Search:</LABEL></FORM></FIELDSET></LI></DIV></DIV></DIV><DIV style="position: fixed; top: 37px; left: 0px; padding: 15px; width:220px; bottom: 0px; z-index: 600; overflow: auto;"><DIV style="width: 100%; text-align: center"><B><A href="#" onclick="document.getElementById('easteregg').style.display='block'" style="text-decoration:none;color:#000000;">Can you feel the smoke tonight?</A></B> </DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Single-Cell_Model"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Single-Cell Model</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Overview"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Initial_States"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Initial States</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Sensors"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Sensors</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Acetaldehyde_Sensor"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Acetaldehyde Sensor</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Xylene_Sensor"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Xylene Sensor</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Initial_Model_Based_on_.5B6.5D"><SPAN class="tocnumber">4.2.1</SPAN><SPAN class="toctext">Initial Model Based on [6]</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Model_Based_on_own_Measurement_Data"><SPAN class="tocnumber">4.2.2</SPAN><SPAN class="toctext">Model Based on own Measurement Data</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-9"><A href="#Band_Detector"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Band Detector</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Toxicity_and_parameter_tuning"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Toxicity and parameter tuning</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Filter_.28Alarm.29"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Filter (Alarm)</SPAN></A></LI></TD></TR></TBODY></TABLE></DIV><DIV style="bottom: 0; left: 235px; overflow: auto; position: fixed; right: 0px; top: 32px;"><DIV style="margin-left: 20px; margin-right: 20px; padding-top: 20px;"><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="Single-Cell_Model"> Single-Cell Model </SPAN></H1><P><B>The single-cell model of our system consists of three connected but easily separable modules: the sensor, the band detector and the filter. The sensor detects toxic substances from cigarette smoke. The band detector produces green fluorescent protein upon a detection of a certain range (band) of toxic substance concentration. After the toxic substance concentration crosses a certain threshold, the filter turns the system red (red fluorescent protein is expressed). The single-cell model helped us understand in detail whether and how our synthetic circuit works, especially to explore the characteristics of the GFP band and get a feeling of how it is related to concentration of our input substance. </B></P></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="Overview"> Overview </SPAN></H1><DIV class="thumb tright"><DIV class="thumbinner" style="width:462px;"><DIV class="thumbcaption"><B>Figure 1</B>: Molecular mechanism of SmoColi with acetaldehyde sensor. The model with the xylene sensor is similar.</DIV></DIV></DIV><P>The single cell model is a mathematical model of the species involved in our synthetic system of a single SmoColi cell. The change of the species concentrations in time is given by non-linear ordinary differential equations (ODEs), most of which follow Hill kinetics. The parameters we used in the model are derived from literature supported by experimental evidence. Some of the parameters are slightly adjusted to meet our own experimental approaches. The single cell model was simulated using numerical Matlab solvers.
</P><P>Our model has several important strengths which are:<B>1. MODULARITY</B>
Our system consists of three modules:</P><UL><LI><A href="/Sensor_Module" title="Sensor Module">Sensor Module</A></LI><LI><A href="/Band_Detector_Module" title="Band Detector Module" class="mw-redirect">Band Detector Module</A></LI><LI><A href="/Filter_(Alarm)_Module" title="Filter (Alarm) Module">Filter (Alarm) Module</A></LI></UL><P>The modules are almost independent, related between each other just by a single species. Most of the parameter values used within the modules do not depend on the other modules as well. The independence of the three modules enables their reusage in other systems. So if we want for example to add another sensor module to the existing band detector, the only thing we should change in the band detector is few parameter values for the linker species. This important property allows us to use the band detector + alarm with different sensors, making our synthetic cells a biological quantifiers of a range of substances.
<B>2. ROBUSTNESS</B>
We did a <A href="/Team:ETH_Zurich/Modeling/Analysis" title="Team:ETH Zurich/Modeling/Analysis">robustness analysis</A> for our system and found out that it is indeed robust. This means that even upon parameter variation, the GFP band does not dissapear, just gets shifted or varies in thickness. Each of our species eventually ends up in a single stable steady state, there are no bifurcations nor oscillations in our system. So, we can freely vary the parameters found in literature to meet our experimental approaches and we can be sure that our system will still show the same global behaviour.
</P></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="Initial_States"> Initial States </SPAN></H1><P>The first step was to obtain the inital concentrations for all the species in our circuit. It is crucial to obtain biologically realistic values since we use them in the subsequent ODE solvers.These values describe the <B>state of our cells at the start of the experiment</B>, i.e. before connecting the channel to the acetaldehyde filled reservoir. When the cells are grown, they don't have any acetaldehyde in their environment and this affects the concentrations of intracellular species whose production depends on it.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:852px;"><DIV class="thumbcaption"><B>Figure 1</B>: Initial internal concentrations of cellular species</DIV></DIV></DIV></DIV><P>
The obtained values are in line with our expectations based on the architecture of our engineered system. By analyzing the circuit we notice that when acetaldehyde is absent, the only proteins that are produced are TetR, LacI and LuxI, this being precisely what we see in the graph.
</P></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="Sensors">Sensors</SPAN></H1><P>The sensor module of our system detects toxic substance found in cigarette smoke. Upon binding of the substance to the sensor molecule, a series of downstream gene regulation events take place. We are dealing with two toxic substances (acetaldehyde and xylene). These two are detected by different sensor molecules with different mechanisms of action. Therefore, we designed two models, one for acetaldehyde detection, another for xylene detection. The models differ only by their sensor module, the band detector and the filter modules are same for both. 
</P><H2><SPAN class="mw-headline" id="Acetaldehyde_Sensor"> Acetaldehyde Sensor </SPAN></H2><P>As the name says, this sensor detects acetaldehyde from the air. Acetaldehyde binds to the transcriptional repressor AlcR, inhibiting TetR, which further inhibits CI and LacI production. Thus, the acetaldehyde binding to AlcR leads to CI and LacI expression (double inhibition equals activation).
</P><P>There has been experimental evidence about the AlcR repression activity when it binds to acetaldehyde <A href="/Team:ETH_Zurich/Modeling/References#Ref1" title="Team:ETH Zurich/Modeling/References">[5]</A>. However, the exact mechanism of acetaldehyde binding to AlcR is not known. In Weber et al. <A href="/Team:ETH_Zurich/Modeling/References#Ref1" title="Team:ETH Zurich/Modeling/References">[5]</A> the acetaldehyde-inducible gene expression was assessed by measuring the expression of acetaldehyde adjustable SEAP for different acetaldehyde concentrations. We <B>fitted the experimental data</B> points for SEAP expression to the Hill equation and used the parameter values that were obtained from the fit to describe the repression kinetics of TetR.
The ODE for TetR is the following:
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Equation system 1</B>: ODE for TetR</DIV></DIV></DIV></DIV><P>
TetR gene is repressed by the AlcR-acetaldehyde complex (AlcR_AcA). The input acetaldehyde concentration is implicitly considered in the equation, due to the fitting procedure. The meaning and the values of the parameters are summarized in the <A href="/Team:ETH_Zurich/Modeling/Parameters" title="Team:ETH Zurich/Modeling/Parameters">Parameters</A> section.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:852px;"><DIV class="thumbcaption"><B>Figure 2</B>: Repression of TetR by the AlcR-Acetaldehyde complex as a function of the input acetaldehyde concentration</DIV></DIV></DIV></DIV><P>We can see in the graph that TetR is expressed only at very small (close to zero) acetaldehyde concentrations, and it gets almost completely repressed already when the acetaldehyde concentration is 200 uM.
</P><H2><SPAN class="mw-headline" id="Xylene_Sensor"> Xylene Sensor </SPAN></H2><P>The sensor that detects xylene is XylR. It is a hexamer which gets activated upon binding of xylene. Active XylR triggers CI and LacI expression. 
</P><P>At the beginning no measurement data was available, such that we modeled the xylene receptor activation based on the model published in <A href="/Team:ETH_Zurich/Modeling/References#Ref6" title="Team:ETH Zurich/Modeling/References">[6]</A>.
Later during the course of creating SmoColi, we obtained own measurement data, based on which we redefined both the model structure as well as the parameter values. We here present both, our initial as well as our final model of xylene activation.
</P><H3><SPAN class="mw-headline" id="Initial_Model_Based_on_.5B6.5D"> Initial Model Based on <A href="/Team:ETH_Zurich/Modeling/References#Ref6" title="Team:ETH Zurich/Modeling/References">[6]</A></SPAN></H3><P>Our initial model incooperated both active and inactive Xylene (Xyla and XylI respectively). Xylene is found in two forms in this model: bound to XylR and free xylene. We consider the free xylene as a state (Xyl) and the initial xylene concentration as an input parameter (Xyl<SUB>ini</SUB>). Thus, the bound xylene concentration can be found by substracting the free xylene concentration from the initial one (Xyl<SUB>ini</SUB> - Xyl). 
</P><P>The ODEs for active and inactive XylR, as well as for xylene are given below and the parameter values are summarized in the <A href="/Team:ETH_Zurich/Modeling/Parameters" title="Team:ETH Zurich/Modeling/Parameters">Parameters</A> section.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Equation system 2a</B>: ODEs for the states involved in the xylene sensor. XylRi stands for inactive XylR, XylRa for active XylR, Xyl for unbound xylene</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Model_Based_on_own_Measurement_Data"> Model Based on own Measurement Data </SPAN></H3><P>Based on our measurement data (see <A href="/Team:ETH_Zurich/Biology/Validation" title="Team:ETH Zurich/Biology/Validation">Team:ETH_Zurich/Biology/Validation</A>), we redefined our parameter values. In this process we also simplified the model as well as adapted it to our synthetic system. Different to the Equation System 2a, our new model assumes the concentration of xylene to be much higher than the concentration of XylR. Consequently, the concentration of free xylene can be considered to be approximately equal to the externally provided concentration (Xyl=Xyl<SUB>ini</SUB>=const), and the last equation in the previous model disappears.
</P><P>Furthermore, different to the system under consideration in <A href="/Team:ETH_Zurich/Modeling/References#Ref6" title="Team:ETH Zurich/Modeling/References">[6]</A>, we could find no indication in the measurement data of our synthetic system for a special dissociation kinetic of the activated XylR (the Hill coefficient is nearly 1, see <A href="/Team:ETH_Zurich/Modeling/References#Ref6" title="Team:ETH Zurich/Modeling/References">[6]</A>), such that we assumed standard mass action association and dissociation kinetics. Finally we modeled both, activated and inactivated XylR, as hexamers, further simplifying the model structure.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:552px;"><DIV class="thumbcaption"><B>Equation system 2b</B>:ODEs for the states involved in the redefined xylene sensor. XylRi stands for inactive XylR, XylRa for active XylR, Xyl<SUB>ini</SUB> for the provided xylene concentration</DIV></DIV></DIV></DIV><P>To compare the model to the measurement data, we assumed the model being in quasi steady state. In this case it is true that
the measured value for the dissociation constant is equal to the ratio of the dissociation and the association rate constant:
</P><P>K<SUB>D</SUB>=r<SUB>R<SUB>XylR</SUB></SUB> / r<SUB>XylR</SUB>=144μM. 
</P><P>Since we could not measure any temporal data with a resolution high enough to separate the value of the dissociation rate constant from the association rate constant, we kept the value obtained from <A href="/Team:ETH_Zurich/Modeling/References#Ref6" title="Team:ETH Zurich/Modeling/References">[6]</A> for the association rate constant and could then redefine the value for the dissociation rate constant based on our measurement data to
</P><P>r<SUB>R<SUB>XylR</SUB></SUB>=144μM * 4*10<SUP>-5</SUP> 1/μM min = 5.7*10<SUP>-3</SUP> 1/min. 
</P><P>The rest of the parameter values were kept from the previous model (see <A href="/Team:ETH_Zurich/Modeling/Parameters" title="Team:ETH Zurich/Modeling/Parameters">parameters</A> section).
</P></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="Band_Detector"> Band Detector </SPAN></H1><P>The band detector is an amplitude band-pass filter that gives a GFP output when the toxic input substance is in a certain concentration range. The filter is a feed-forward system with 2 branches, interconnected by a NOR gate. 

After the toxic substance is detected by the sensor module, the system splits into a slow branch and a fast branch.
</P><UL><LI>The <B>fast branch</B> is the <B>amplitude low-pass filter</B>. This means that it will repress the production of GFP when the toxic  substance concentration goes above the high threshold. In other words, GFP is produced only at low toxic substance concentrations. High concentrations of toxic substance trigger LacIm1 expression, GFP production being thus inhibited.
</LI><LI>The <B>slow branch</B> is the <B>amplitude high-pass filter</B> which represses GFP when the toxic substance concentration falls below the low threshold or in other words GFP is produced only at high levels of toxic substance.
</LI></UL><P> 
By combining these 2 filters, GFP is produced when neither LacI<SUB>M1</SUB> or LacI represses it, which is the region between the two thresholds. This region is the range of the toxic substance concentration which can trigger the GFP pulse. The ODEs for the species involved in the band detector are given below.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Equation system 3</B>: ODEs for the states involved in the band detector module (CI, LacI and GFP)</DIV></DIV></DIV></DIV><P>
We simulated the models for 5000 minutes in order to obtain the steady state GFP value. We repeated this simulation for a range of input acetaldehyde concentrations (from 0 to 5000 μM). In the following figures we can see how the steady state concentration of GFP changes for different input concentrations of acetaldehyde. For the acetaldehyde model the band is formed when the acetaldehyde concentration is approximately in the range of 500-1500 μM. The peak of the GFP pulse is observed when the acetaldehyde concentration is 1000 μM.
</P><SPAN id="GFPvsAcAl"><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:852px;"><DIV class="thumbcaption"><B>Figure 2</B>: GFP vs. increasing acetaldehyde input (both in μM). The band appears when the acetaldehyde concentration is in the range of 500-1500 μM</DIV></DIV></DIV></DIV></SPAN><H2><SPAN class="mw-headline" id="Toxicity_and_parameter_tuning"> Toxicity and parameter tuning </SPAN></H2><P>The acetaldehyde concentration range where the band is formed might be reasonable in our model, but not in reality. We tested the toxicity of acetaldehyde by adding 10 mM to a culture of E.coli cells and we observed that at this concentration the cells die. By taking into account the fact that a fraction of the acetaldehyde, due to its volatility, will diffuse out of the culture medium, we concluded that the maximum concentration the cells can tolerate is around 2 mM. This means that although in our model the band appears at a lower concentration: 500-1500 uM that does not affect the cells, the bacteria that are closer in the channel to the acetaldehyde source will still die.

Clearly, this tells us that we need to explore the parameter space for some of the constants to find suitable values. This is not surprising since for some of the parameters we made assumptions based on values for similar molecules. We carefully studied the equations involved in the band detector module and decided to make the following changes:
</P><UL><LI><B>TetR production rate (α<SUB>TetR</SUB>)</B>: From the curve fitting <A href="/Team:ETH_Zurich/Modeling/References#Ref1" title="Team:ETH Zurich/Modeling/References">[1]</A>, we got the TetR production rate to be 3.93 uM/min and we used this in our simulation. However, the measurements for which we did the fitting were done in mammalian cells, not in &quot;E. coli&quot;. Since we express TetR on the same plasmid with other genes, for which we consider production rate to be 1 uM/min, we also changed α<SUB>TetR</SUB> to 1 uM/min.
</LI><LI><B>TetR repression coefficient (β<SUB>TetR</SUB>)</B>: We didn't have any supporting evidence for this, but we assumed it to be 0.01 uM. This was the value used in Basu et al. <A href="/Team:ETH_Zurich/Modeling/References#Ref1" title="Team:ETH Zurich/Modeling/References">[1]</A>, but there however the repressor is LuxR, not TetR. We increased TetR repression coefficient to 0.1 uM since we assume that TetR is a less strong repressor than LuxR. With the modified value we get more realistic results.
</LI></UL><P>
With these 2 changes, the acetaldehyde concentration range where the band appears is much lower than in the previous case. We can see that the band forms between 25 μM and 100 μM of acetaldehyde input, with the peak at 46 μM (see Figure 4).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:852px;"><DIV class="thumbcaption"><B>Figure 4</B>:The band for acetaldehyde after parameter fine tuning is now shifted to the left. It appears when acetaldehyde concentration range is 30-100 uM</DIV></DIV></DIV></DIV><P>In both cases, the shape of the band looks similar, but it is approximately 10 times narrower in the case where we used the modified parameters. However, in both cases, its left side is steeper than its ride side, that is it appears more quickly than it dissapears. 
</P></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="Filter_.28Alarm.29"> Filter (Alarm) </SPAN></H1><P>This module acts like a high pass amplitude filter for the toxic input substance, since it produces RFP only when the toxic substance concentration goes above a certain threshold. At high acetaldehyde concentrations, TetR is repressed, so the repression of CI by TetR can not be established. At high xylene concentrations, XylR is activated and it triggers CI expression. 

In both models, at high concentrations of the toxic substance CI is expressed and it inhibits the production of LuxI, which means that AHL is not produced. Once AHL is not there, it can not bind to LuxR, so LuxR can not repress RFP anymore. This results in RFP expression; all the cells turn red. As mentioned, this module of the model acts as a high pass amplitude filter for the toxic substance, or low pass amplitude filter for AHL, since AHL is produced only when the toxic substance levels are low.

The ODEs for the states involved in the second module of the system are given below:
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Equation system 4</B>: ODEs for the states involved in the Filter module. AHLi represents the intracellular AHL concentration, AHLe is the extracellular one. R represents the complex between LuxR and the intracellular AHL. Short explanation of the parameters is given in the Parameter section.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:852px;"><DIV class="thumbcaption"><B>Figure 4</B>: The intracellular AHL and the RFP concentration plotted against Acetaldehyde</DIV></DIV></DIV></DIV><P> 
The RFP steady state concentrations plotted against increasing toxic substance concentrations are presented in the above figures. For the acetaldehyde model, RFP appears after acetaldehyde crosses the threshold of 500 μM.

This plot shows the behavior of a singe cell. However, in our final system we have a population of cells immobilized in agarose that communicate whith eachother through the diffusing AHL. Thus, in the final setup the channel will turn red only when all the cells stop producing AHL and all the existing AHL is degraded in the channel.
</P></DIV><DIV style="position: relative; height: 149px; text-align: center; clear: both; margin: 20px auto 20px auto;" class="boxWrap"><TABLE style="height: 100%; width: 100%; border-spacing: 0px; " cellspacing="0"><TBODY><TR style="font-weight: bold;"><TD style="text-align: center; border-right: 1px solid black;">Back to iGEM</TD><TD colspan="4" style="text-align: center;"><SPAN id="SponsorsFooter">Our Sponsors</SPAN></TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="visualClear"> </DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:ETH_Zurich/Modeling/SingleCell">http://2011.igem.org/Team:ETH_Zurich/Modeling/SingleCell</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETH_Zurich/Modeling/SingleCell" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETH_Zurich/Modeling/SingleCell" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETH_Zurich/Modeling/SingleCell&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETH_Zurich/Modeling/SingleCell&amp;oldid=259303" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>