"
Page
Discussion
View source
History
teams
Log in
 
Team:ETH Zurich/Biology/MolecularMechanism
From 2011.igem.org
Home
Project SmoColi
What is SmoColi?
The Circuit
Information processing
Dialogue between Biology and Modeling
Achievements
Data Page
Biology
Overview
Network Elements
Genetic Design
Materials and Methods
Lab Journal
Results
Microfluidics
Overview
Channel Design Development
Final Design and Validation
Proof of Concept
Modeling
Overview
Single-Cell Model
System Analysis
Stochastic Analysis
Reaction-Diffusion Model
Combined 3D Model
Analytical Approximation
Parameters
References
Code
Human Practices
Outreach
Safety
Team
Team SmoColi
Photo Gallery
Acknowledgements & Contributions
Official Team Overview
Log In
Edit
History
Upload File
Log Out
Search:
Can you feel the smoke tonight?
 
Contents
1 Genetic Design
2 Plasmid design
2.1 pSB6A5
3 Cloning strategy
3.1 General procedure
3.2 Sources of genes and promoters
3.3 Cloning successes and problems
4 Choice and design of ribosome binding sites
5 References
Genetic Design
In a System where so many trancriptional regulators effect each other, their respective levels are very important factor in the establishment of a running system. The expression rate is controlled with a multi plasmid operation where the genes are assembled on different plasmids with different copy numbers. Also we put a lot of effort in the design of the ribosome binding site. Below we provide a description of the plasmid design, the strategy we used to clone the necessary parts and the method of choosing the ribosome binding sites.
Plasmid design
Table 1: Plasmid design for the AlcR system Transcriptional terminators included for each gene. Plasmid copy numbers adapted from [1] .
Table 2: Plasmid design for the XylR system Transcriptional terminators included for each gene. Plasmid copy numbers adapted from [1] . For a detailed description of the pCK04AxylR plasmid see [3] .
Table 3: Plasmid design for the AraC test system Transcriptional terminators included for each gene. Plasmid copy numbers adapted from [1] .
For the biological implementation of the SmoColi system, we constructed a three-plasmid system with different antibiotic resistances. This setup was chosen in order to clone the whole SmoColi system into the same bacterial cells.
The vectors used in our setup are the parts pSB6A5, pSB3C5, pSB4C5, pSB3K3 and pSB4K5. These plasmids contain different origins of replication, thus resulting in different copy numbers of the corresponding plasmid in E.coli. pBR322, p15A and pSC101 origins are used as a stable three-plasmid system [2] . The copy number influences the amount of protein produced by the cell and plays a crucial role in our design, especially for the bandpass filter. Depending on the function of a particular protein in our system we had to evaluate on which plasmid the corresponding gene should be cloned.
In the xylene-responsive system we included an artificial degradation pathway (xylWCMABN) so that we could establish a xylene gradient. For this purpose the plasmid pCK04AxylR was added [3] . To avoid incompatibilities of plasmids we adapted the rest of our system and changed the composition of the genes on each vector compared to the AlcR system.
For proof of concept of the bandpass filter, we also constructed a system with AraC as an arabinose-dependent sensor. Here we again change the plasmid design in order to match the requirements of the test system. Therefore we introduced a plasmid (pSB4C5) containing the araC regulator gene and the degradation pathway for arabinose, araBAD. Since AraC is activating PBAD upon sensing arabinose, the design for the other two plasmids could be adapted from the design of the xylene system. Only Pu promoters were exchanged with PBAD.
pSB6A5
Figure 1: Improvement of the plasmid pSB6A1 to pSB6A5
The pSB6A5 plasmid we constructed by reducing the size of the antibotic (Ampicillin) and origin of replication (pMB1) casettes of pSB6A1 to a minimum and by introducing terminators around the multiple cloning site. PMB1 is a medium copy orign with about 15-20 copies per cell. For the characterization of pSB6A5 click here
Cloning strategy
General procedure
Figure 1: Cloning strategy for SmoColi
For cloning our parts we tried to follow the strategies proposed in the registry of standard biological parts [4] as much as possible . The general cloning scheme was performed as follows:
If available, genes and promoters in SmoColi were obtained from the iGEM spring 2011 distribution kit. A codon-optimized version of alcR and Ptet-lacIM1 were synthesized. Palc- and PU-promoters were obtained by PCR.
After isolation of the DNA, ribosome binding sites and prefix (5' gaattcgcggccgcttctagag 3') respectively suffix (5' tactagtagcggccgctgcag 3') were added to the gene by polymerase chain reaction. The PCR products were purified and afterwards digested with XbaI-PstI.
Plasmid vectors containing the corresponding promoter were linearized by SpeI-PstI and the XbaI-PstI fragment containing the gene of interest was ligated into the backbone.
In a next step we isolated a EcoRI-SpeI fragment with promoter, ribosome binding site and gene and inserted it into a EcoRI-XbaI cut vector containing a transcriptional terminator.
In the end all the constructed parts were assembled on the corresponding vectors for the final SmoColi system. Some parts (especially lacI in an actively expressed form) appeared to be negatively selected by E.coli. So we had to assemble the parts in a way that lacI was always repressed.
Sources of genes and promoters
Table 4: Promoters of SmoColi: source and final constructs of the different used promoters
Table 3: Genes of SmoColi: source and final constructs of the different used genes
Cloning successes and problems
All parts of the system were successfully assembled by this strategy, except for lacI and cI under control of the Tet promoter. Whenever we were sequencing a clone of either of those constructs, we always found protein inactivating mutations within the primer annealing part - either being deletions of the Shine-Dalgarno sequence of the RBS or frame shift mutations within the first bases of the coding regions. We first believed the problem to lie in the synthesis of the very long oligonucleotides, for adding the whole RBS and prefix in front of the genes. But also performing a whole plasmid PCR with short primers did not solve the problem and we still only got non-protein-generating sequences. We believe this to be due to lethality of high overexpression levels of LacI or CI, leading to a selection for clones mutated in a way that lacI or cI is not expressed. Thus one always has to make sure to clone lacI or cI behind downregulated promoters, for example for PTet you have to make sure that there is TetR present in the cell. Those problems unforunately delayed the construction of the complete system to only shortly before Wikifreeze, thus we hope to produce as many results as possible till the Jamboree.
Choice and design of ribosome binding sites
Table 5: Different ribosome binding sites and their estimated strengths which are included in our design. The strengths of the sequences were estimated with a RBS strength calculator [7] . In the last column it is listed in front of which genes a particular RBS was added.
Another important role in the synthesis of proteins is played by the ribosome binding site (RBS), the location where the ribosomes are bound to the mRNA at the initiation of the translation process. The ribosome binding sites which we included in our system were designed in several different ways:
For the bandpass filter we implemented the ones which are also used for the plasmids in [5] .
For the other parts we either tried to use naturally occurring ribosome binding sites or took specific ones from the Anderson library of ribosome binding sites [6] .
The strength of all the ribosome binding sites used was estimated with a RBS calculator. Although the efficiency of protein synthesis is not directly proportional to the estimated values, they can still indicate whether a certain RBS is more or less strong. Below you can find a list of the different ribosome binding sites used, their estimated binding strengths and in front of which genes they were put.
References
[1] OpenWetWare Arking: JCAOligoTutorial8, 0 Jul 2008, 04:32 UTC. 21 Sep 2011, 01:32
[2] Karl Friehs: Plasmid Copy Number and Plasmid Stability, Advances in Biochemical Engineering/Biotechnology, 2004, 86: 22-192
[3] Sven Panke, Juan M. Sánchez-Romero, and Víctor de Lorenzo: Engineering of Quasi-Natural Pseudomonas putida Strains for Toluene Metabolism through an ortho-Cleavage Degradation Pathway, Appl Environ Microbiol, February 1998, 64: 748-751
[4] Registry of Standard Biological Parts Help:contents 21 Sep 2011, 02:59
[5] Subhayu Basu, Yoram Gerchman1, Cynthia H. Collins, Frances H. Arnold & Ron Weiss: A synthetic multicellular system for programmed pattern formation, Nature, 2005, 434: 1130-11342
[6] J. Christopher Anderson Ribosome Binding Sites/Prokaryotic/Constitutive/Community Collection Registry of Standard Biological Parts, 21 Sep 2011, 08:49
Back to iGEM
Our Sponsors
 
Retrieved from " http://2011.igem.org/Team:ETH_Zurich/Biology/MolecularMechanism "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
