<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETH_Zurich_Biology_Detector"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ETH_Zurich/Biology/Detector">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETH_Zurich/Biology/Detector&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Biology/Detector&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Biology/Detector&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/Biology/Detector" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETH Zurich/Biology/Detector</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV class="stickToTop" id="naviBox" style="text-align:center; line-height: 0px; font-size: 0px; z-index: 500;"><DIV style="position: relative; margin: 0 auto; display:inline-block; left: 0px; right: 0px;"><DIV align="center" id="naviMenu" style="left: 0px; right: 0px; margin: 0pt auto; text-align: left;"><UL class="dropdown dropdown-horizontal mainmenu"><LI class="first"><A href="/Team:ETH_Zurich">Home</A></LI><LI><A href="/Team:ETH_Zurich/Overview/Project" class="dir">Project <I>SmoColi</I></A></LI><LI class="first"><A href="/Team:ETH_Zurich/Overview/Project">What is SmoColi?</A></LI><LI><A href="/Team:ETH_Zurich/Overview/CircuitDesign">The Circuit</A></LI><LI><A href="/Team:ETH_Zurich/Overview/Informationprocessing">Information processing</A></LI><LI><A href="/Team:ETH_Zurich/Overview/Dialogue">Dialogue between Biology and Modeling</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Achievements">Achievements</A></LI></UL><LI><A href="/Team:ETH_Zurich/Achievements/Data_page">Data Page</A></LI><LI><A href="/Team:ETH_Zurich/Biology" class="dir">Biology</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Biology">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Biology/Detector">Network Elements</A></LI><LI><A href="/Team:ETH_Zurich/Biology/MolecularMechanism">Genetic Design</A></LI><LI><A href="/Team:ETH_Zurich/Biology/Cloning">Materials and Methods</A></LI><LI><A href="/Team:ETH_Zurich/Biology/Journal">Lab Journal</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Biology/Validation">Results</A></LI><LI><A href="/Team:ETH_Zurich/Process/Microfluidics/Overview" class="dir">Microfluidics</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Process/Microfluidics/Overview" class="dir">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Process/Microfluidics">Channel Design Development</A></LI><LI><A href="/Team:ETH_Zurich/Process/Validation">Final Design and Validation</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Process/Microfluidics/Proof">Proof of Concept</A></LI><LI><A href="/Team:ETH_Zurich/Modeling" class="dir">Modeling</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Modeling">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/SingleCell">Single-Cell Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Analysis">System Analysis</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Stochastic">Stochastic Analysis</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Microfluidics">Reaction-Diffusion Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Combined">Combined 3D Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Analytical_Approximation">Analytical Approximation</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Parameters">Parameters</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/References">References</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Modeling/Code">Code</A></LI><LI><A href="/Team:ETH_Zurich/Team/Human practices" class="dir">Human Practices</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Team/Human practices">Outreach</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Biology/Safety">Safety</A></LI><LI><A href="/Team:ETH_Zurich/Team" class="dir">Team</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Team">Team SmoColi</A></LI><LI><A href="/Team:ETH_Zurich/Team/Media">Photo Gallery</A></LI><LI><A href="/Team:ETH_Zurich/Team/Acknowledgements">Acknowledgements &amp; Contributions</A></LI><LI class="last"><A href="https://igem.org/Team.cgi?year=2011&amp;team_name=ETH_Zurich" target="_blank">Official Team Overview</A></LI><LI id="loginMenu"><A href="#" id="loginLink">Log In</A></LI><LI style="display:none;" id="editMenu"><A href="#" id="editLink" class="dir">Edit</A></LI><LI class="first"><A href="#" id="historyLink">History</A></LI><LI><A href="/Special:Upload" target="_blank">Upload File</A></LI><LI class="last"><A href="#" id="logoutLink">Log Out</A></LI><LI class="dir last" style="background-image: none; position: relative;"><FIELDSET><FORM method="post" id="searchForm" action=""><LABEL for="s">Search:</LABEL></FORM></FIELDSET></LI></DIV></DIV></DIV><DIV style="position: fixed; top: 37px; left: 0px; padding: 15px; width:220px; bottom: 0px; z-index: 600; overflow: auto;"><DIV style="width: 100%; text-align: center"><B><A href="#" onclick="document.getElementById('easteregg').style.display='block'" style="text-decoration:none;color:#000000;">Can you feel the smoke tonight?</A></B> </DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Network_Elements"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Network Elements</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Acetaldehyde_Sensor"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Acetaldehyde Sensor</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Where_it_comes_from..."><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Where it comes from...</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#How_we_used_it"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">How we used it</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-5"><A href="#Xylene_Sensor"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Xylene Sensor</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Where_it_comes_from..._2"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Where it comes from...</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#How_we_used_it_2"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">How we used it</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Arabinose_Sensor"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Arabinose Sensor</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Where_it_comes_from..._3"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Where it comes from...</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#How_we_used_it_3"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">How we used it</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Bandpass"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Bandpass</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Where_it_comes_from..._4"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Where it comes from...</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#How_we_used_it_4"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">How we used it</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#Inverter"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Inverter</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Where_it_comes_from..._5"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Where it comes from...</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#How_we_used_it_5"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">How we used it</SPAN></A></LI><LI class="toclevel-1 tocsection-17"><A href="#Quorum_Sensing_of_the_Alarm_system"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Quorum Sensing of the Alarm system</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Where_it_comes_from..._6"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">Where it comes from...</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#How_we_used_it_6"><SPAN class="tocnumber">7.2</SPAN><SPAN class="toctext">How we used it</SPAN></A></LI><LI class="toclevel-1 tocsection-20"><A href="#References"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE></DIV><DIV style="bottom: 0; left: 235px; overflow: auto; position: fixed; right: 0px; top: 32px;"><DIV style="margin-left: 20px; margin-right: 20px; padding-top: 20px;"><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="Network_Elements"> Network Elements </SPAN></H1><P><B>SmoColi can be divided into different network elements: the major sub-networks being a sensor, a band-pass filter and a quorum sensing mechanism.</B>
Cigarette smoke contains a lot of different toxic and carcinogenic components. In nature there are many sensors for such components, mostly to induce their degradation. One is the acetaldehyde system in <I>aspergillus nidulans</I>: the activator AlcR binds to its operator site if acetaldehyde is present <A href="#Ref1">[1]</A>. Another one is the xylene sensing system in <I>Pseudomonas putida</I><A href="#Ref2">[2]</A> working as an activator. Since our system is constructed in a modular fashion an can be coupled with any sensor, an arabinose sensing system was established as a proof of principle.
We combined a smoke-sensitive band-pass filter with GFP output with a quorum-sensing based mechanism alarming the user at high smoke levels by expressing RFP.
</P></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="Acetaldehyde_Sensor">Acetaldehyde Sensor </SPAN></H1><H3><SPAN class="mw-headline" id="Where_it_comes_from..."> Where it comes from... </SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 1: Structures of the AlcR(1-60)-DNA complex</B><A href="#Ref13">[13]</A></DIV></DIV></DIV><P>In <I>Aspergillus nidulans</I>, the <I>alcR</I> gene encodes a regulatory protein which activates the expression of the ethanol utilization (alc) pathway if the co-inducer acetaldehyde is present. Ethanol is converted to acetaldehyde by the alcohol dehydrogenase and further metabolized to acetate by the aldehyde dehydrogenase. Both genes <I>alcA</I> and <I>aldA</I> are regulated by AlcR. AlcR also regulates its own expression by autoactivation <A href="#Ref1">[1]</A>.
The AlcR DNA binding domain contains a zinc bi-nuclear cluster, which can bind either to symmetric and asymmetric sites with same affinity. In contrast to other members of the Zn<SUB>2</SUB>Cys<SUB>6</SUB>, the DNA binding domain is stretch of 16 amino acid residues between the third and fourth cysteine. AlcR binds as a monomer, but 2 proteins can bind to inverted repeats in a noncooperative manner. Additional DNA sequences upstream of the zinc cluster were identified to be responsible for high-affinity binding,  <A href="#Ref4">[4]</A>.
</P><H3><SPAN class="mw-headline" id="How_we_used_it"> How we used it </SPAN></H3><P>Because fungal activators normally do not work in <I>E. coli</I>, the system was modified to work as a bacterial repressor. The mechanism of how AlcR activates transcription is not completely known, which makes the process of redesigning it very challenging. A second challenge is the different codon usage in <I>Apergillus nidulans</I> and <I>E. coli</I>. The codon adaptation index (CAI) of <I>Aspergillus nidulans</I> AlcR is 0.7 in <I>E. coli</I>, especially at the beginning of the gene a few very rare codons are present. To get a fully <I>E. coli</I>-optimized gene, we ordered the gene codon-optimized. To get a partly codon-optimized version, we exchanged the rare codons at the beginning of the <I>Aspergillus nidulans</I> protein with PCR.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 2: Design of the promoter for the AlcR-system in SmoColi</B>, the operon sequence is designed between the -10 and the -35 region (blue) of a strong promoter with inverted and direct repeats.</DIV></DIV></DIV><P><SPAN id="AlcR_Promoter">For</SPAN> a redesigned bacterial P<SUB>Alc</SUB> promoter, natural operator sites of AlcR from <I>Aspergillus nidulans</I> were placed between the -10 and -35 region of a constitutive <I>E.coli</I> promoter. Two versions of the synthetic promoter were implemented, one with direct-repeat targets and one with inverted <A href="#Ref1">[1]</A>. The idea is that in the presence of acetaldehyde, AlcR will bind to the operon and block the binding of the RNA polymerase. This only works if the mechanism of AlcR transcriptional activation is based on acetaldehyde induced DNA binding and does not just involve conformational changes. When implementing the SmoColi genetic network, an additional inverter (TetR) is added in front of the bandpass, so that the acetaldehyde signal is passed into the bandpass in a positive way. Thus the AlcR promoter (P<SUB>Alc</SUB>) is cloned in front of the TetR repressor to introduce the bandpass.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 3: Circuit operations for SmoColi with acetaldehyde sensor</B>, AlcR works as an repressor, therefore a inverter (TetR) is introduced to convert the repression of the input to an activation.</DIV></DIV></DIV><P> 
Acetaldehyde is reduced by the native <I>E. coli</I> enzyme aldehyde dehydrogenase, generating a concentration gradient in our tube. At the position with the required concentration of acetaldehyde for the bandpass-filter, GFP is expressed, at all other positions it is repressed, resulting in a single GFP band. By tuning the input of acetaldehyde in the tube, we get the GFP band at different positions. Thus, with constantly increasing acetaldehyde concentration, we get a moving band from one side to the other side of the channel. Finally,if the acetaldehyde concentration exceeds a certain threshold, the whole channel turns red, due to the quorum-sensing based RFP expression.
</P></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="Xylene_Sensor"> Xylene Sensor </SPAN></H1><H3><SPAN class="mw-headline" id="Where_it_comes_from..._2"> Where it comes from... </SPAN></H3><P>The <I>xylR</I> gene in <I>Pseudomonas putida</I> encodes a 566 aa regulatory protein that activates the degradation of toluene and xylene. In <I>P. putida</I> the complete upper Tol pathway is under the control of the toluene-responsive promoter P<SUB>U</SUB>. In the presence of xylene, XylR hexamerically binds to P<SUB>U</SUB> promoter and activates it <A href="#Ref5">[5]</A>. This activates expression of the  xylene degradation cassette <I>xylC-xylM-xylA-xylB-xylN</I>, where <I>xylC</I> encodes for the benzaldehyde dehydrogenase, <I>xylM</I> and <I>xylA</I> for subunits of the xylene oxygenase and <I>xylB</I> for benzyl alcohol dehydrogenase <A href="#Ref6">[6]</A>. XlyN is not required for the degradation of xylene, but it is part of the whole transcriptional unit. The DNA binding of XylR occurs at a 40 bp upstream activating sequence (UAS) located 150 bp upstream from the transcriptional start site of the σ<SUP>54</SUP> P<SUB>U</SUB> promoter. DNA looping between the UAS and the -12/-24 motifs of the σ<SUP>54</SUP>-RNAP facilitates the activation of the holoenzyme by XylR <A href="#Ref7">[7]</A>.
</P><H3><SPAN class="mw-headline" id="How_we_used_it_2"> How we used it </SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 4: Circuit operations of SmoColi with xylene sensor</B></DIV></DIV></DIV><P>Xylene is not degraded naturally by <I>E. coli</I>. To generate a concentration gradient of xylene we engineered its degradation in SmoColi by including the upper Tol pathway. XylR is a prokaryotic enhancer binding protein, thus is completely functional in <I>E. coli</I>. As an input into the SmoColi system, LacI<SUB>M1</SUB> and CI are put under the control of the P<SUB>U</SUB> promoter, enhancing their expression in the presence of xylene. 
</P></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="Arabinose_Sensor"> Arabinose Sensor </SPAN></H1><H3><SPAN class="mw-headline" id="Where_it_comes_from..._3"> Where it comes from... </SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 5: Structures of the AraC heterodimere with L-Arabinose</B><A href="#Ref14">[14]</A></DIV></DIV></DIV><P>The AraC regulatory protein is a activator of the genes for the araBAD, the <I>araE</I> and the <I>araFGH</I> operon in <I>E.coli</I>.   <I>araB, araA</I> and <I>araD</I> encode for the metabolic enzymes which degrade L-Arabinose to D-xylulose-5-phosphate. In absence of arabinose AraC does bind to two DNA site separated over 200 bp from each other (I<SUB>1</SUB> and O<SUB>2</SUB> of the P<SUB>BAD</SUB> promoter) <A href="#Ref8">[8]</A>. By dimer formation of the two DNA bound AraC molecules the DNA is looped and transcription blocked. Binding of arabinose leads to binding of arabinose to I<SUB>1</SUB> and I<SUB>2</SUB>, I<SUB>2</SUB> bound AraC directly interacts with the RNA polymerase and activates it. <I>araF, araG</I> and <I>araH</I> and <I>araE</I> encode for proteins needed for uptake of arabinose. If some arabinose enters the cell, the uptake of arabinose is activated leading and more and more arabinose entering the cell. This positive feedback loop leads to an all or nothing response of arabinose in the cells.
</P><H3><SPAN class="mw-headline" id="How_we_used_it_3"> How we used it </SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 6: Circuit operations of SmoColi with arabinose sensor</B></DIV></DIV></DIV><P>To show that our system can be activated by different signals, we introduced a arabinose sensor. Due to the fact that AraC s well established in contract AlcR and XlyR, the arabinose system can be seen as proof of principle for the bandpass and the alarm system for our smoke sensors. The LacI<SUB>M1</SUB> and the CI regulators were under the control of the P<SUB>BAD</SUB> promoter, enhancing their expression in the presence of arabinose and the absence of glucose. 
The all or nothing response of the arabinose system makes the natural system unusable for our bandpass, because we need an intermediate input signal. In order that we can still use the arabinose system we use the BW27783 (BW25113 DE(<I>araFGH</I>) Φ(∆<I>araEp</I>) (P<SUB>CP8</SUB>-<I>araE</I>)) <I>E.coli</I> strain <A href="#Ref9">[9]</A>. This strain has a knock-out of the <I>araBAD</I> (degradation), <I>araFGH</I> (low capacity, high affinity transporter) and the promoter of the <I>araE</I> gene (high capacity, low affinity transporter) has been exchanged from P<SUB>BAD</SUB> to P<SUB>CP8</SUB>, which is a constitutive promoter. The elimination of the positive feedback loop within this strains leads to a titratable response on arabinose, allowingintermediate input concentrations for the bandpass filter. By inactivation of the kanamycin casette, we created the BW27783 from the BW27749, which was provided to us by Khlebnikov et al. (Done according to the method of: <A href="#Ref10">[10]</A>)
</P></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="Bandpass"> Bandpass </SPAN></H1><H3><SPAN class="mw-headline" id="Where_it_comes_from..._4"> Where it comes from... </SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:408px;"><DIV class="thumbcaption"><B>Figure 7: A Bandpassfilter</B>, is a combination of a high pass filter and a low pass filter.</DIV></DIV></DIV><P>In 2005 Ron Weiss et al. constructed the first band-pass filter device in synthetic biology. This was done by using the combination of a high-pass and a low-pass filter. An input signal is transmitted by cell to cell communication while the signal processing is done using an intracellular feed-forward loop. Filters are created by using different transcriptional regulator proteins with different repression efficiencies and binding affinities for corresponding promoters.
</P><H3><SPAN class="mw-headline" id="How_we_used_it_4"> How we used it </SPAN></H3><P>We used his pioneering network design as a basis in the planning process for our band-pass filter implementation <A href="#Ref11">[11]</A>. Instead of cell-cell communication via AHL for the input signal, we included our different smoke detectors which detect small volatile molecules like acetaldeyhde, xylene or arabinose.
</P></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="Inverter"> Inverter </SPAN></H1><H3><SPAN class="mw-headline" id="Where_it_comes_from..._5"> Where it comes from... </SPAN></H3><P>An inverter is a part which converts a repressor signal into an activator signal, or vise versa. For SmoColi we used TetR as an inverter. TetR binds and blocks the TetR promoter (P<SUB>Tet</SUB>, <A href="http://partsregistry.org/Part:BBa_R0040" class="external text" rel="nofollow">BBa_R0040</A>).
</P><H3><SPAN class="mw-headline" id="How_we_used_it_5"> How we used it </SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:408px;"><DIV class="thumbcaption"><B>Figure 8: What would happen without an inverter:</B> No GFP formation..</DIV></DIV></DIV><P>When the inverter is not present, the band does not appear because the activation threshold of the high pass filter is at a higher value than the repression threshold of the low pass filter, thus there is no region where both branches produce an output signal. If the input signal (a repressor) is inverted we get again a nice bandpass filter. In our circuit the inverter is only needed for AlcR, for AraC and XylR the input signal is correct (they are already activators).
</P></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="Quorum_Sensing_of_the_Alarm_system"> Quorum Sensing of the Alarm system </SPAN></H1><H3><SPAN class="mw-headline" id="Where_it_comes_from..._6"> Where it comes from... </SPAN></H3><P>The luminescence operon in <I>Vibrio fischeri</I> is regulated by the transcriptional activator (LuxR). The lux box is a 20 bp inverted repeat sequence at a position centered 42.5 bases upstream of the transcriptional start. LuxR binds to the lux box in presents of AHL and enhance its expression. LuxR is needed to activate the σ<SUP>70</SUP> RNA Polymerase. The <I>Vibrio fischeri</I> lux operon consists of seven genes, the first one is <I>luxI</I>. The <I>luxI</I> gene encodes for the enzyme for AHL synthesis. The autoinducer AHL is secreted by the cells to coordinate gene expression based on the local density of the bacterial population.  Furthermore transcription of luciferase in the lux operon is induced, leading to bioluminescence.
</P><H3><SPAN class="mw-headline" id="How_we_used_it_6"> How we used it </SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 9: Promoter for luxR</B>, the operon (red) designed between the -10 and the -35 region (blue) of the promoter.</DIV></DIV></DIV><P>For our system we used a promoter which is repressed by luxR! (<A href="http://partsregistry.org/Part:BBa_R0061" class="external text" rel="nofollow">BBa_R0061</A>) It was designed by placing the lux box between and partially overlapping the consensus -35 and -10 site of an artificial lacZ promoter. In the presence of AHL, LuxR binds to the lux box and inhibits binding of the RNA Polymerase. <A href="#Ref12">[12]</A></P></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="References"> References </SPAN></H1><P><SPAN id="Ref1">[1] <A href="http://www.ncbi.nlm.nih.gov/pubmed/2834622" class="external text" rel="nofollow">R. Locklngton, C. Scazzocchio, D. Sequeval, M. Mathieu, B. Felenbok: <B>Regulation of alcR, the positive regulatory gene of the ethanol utilization regulon of Aspergillus nidulans</B>, Mol Microbiol., 1987, 1: 275-81</A></SPAN></P><P><SPAN id="Ref2">[2] <A href="http://aem.asm.org/cgi/content/abstract/64/2/748" class="external text" rel="nofollow">Sven Panke, Juan M. Sánchez-Romero, and Víctor de Lorenzo: <B>Engineering of Quasi-Natural Pseudomonas putida Strains for Toluene Metabolism through an ortho-Cleavage Degradation Pathway</B>, Appl Environ Microbiol, 1998, 64: 748-751</A></SPAN></P><P><SPAN id="Ref3">[3] <A href="http://aem.asm.org/cgi/content/abstract/64/6/2032" class="external text" rel="nofollow">Sven Panke, Bernard Witholt, Andreas Schmid, and Marcel G. Wubbolts: <B>Towards a Biocatalyst for (S)-Styrene Oxide Production: Characterization of the Styrene Degradation Pathway of Pseudomonas sp. Strain VLB120</B>, Appl Environ Microbiol, 1998, 64: 2032-2043</A></SPAN></P><P><SPAN id="Ref4">[4] <A href="http://www.ncbi.nlm.nih.gov/pubmed/9182587" class="external text" rel="nofollow">Lenouvel F, Nikolaev I, Felenbok B.: <B>In vitro recognition of specific DNA targets by AlcR, a zinc binuclear cluster activator different from the other proteins of this class</B>, J Biol Chem. 1997, 272(24):15521-6.</A></SPAN></P><P><SPAN id="Ref5">[5] <A href="http://www.springerlink.com/content/r65l105rg21t1l73/" class="external text" rel="nofollow">Manabu Gomada, Sachiye Inouye, Hiromasa Imaishi, Atsushi Nakazawa and Teruko Nakazawa: <B>Analysis of an upstream regulatory sequence required for activation of the regulatory gene xylS in xylene metabolism directed by the TOL plasmid of Pseudomonas putida</B>, Molecular and General Genetics MGG, 1992, 233:419-426</A></SPAN></P><P><SPAN id="Ref6">[6] <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC210316/" class="external text" rel="nofollow">S Harayama, M Rekik, M Wubbolts, K Rose, R A Leppik, and K N Timmis:<B> Characterization of five genes in the upper-pathway operon of TOL plasmid pWW0 from Pseudomonas putida and identification of the gene products</B>, 1989, 171(9): 5048–5055</A></SPAN></P><P><SPAN id="Ref7">[7] <A href="http://nar.oxfordjournals.org/content/31/23/6926.full" class="external text" rel="nofollow">Marc Valls and Víctor de Lorenzo: <B>Transient XylR binding to the UAS of the Pseudomonas putida σ<SUP>54</SUP> promoter Pu revealed with high intensity UV footprinting in vivo</B>, Nucl. Acids Res., 2003, 31 (23): 6926-6934</A></SPAN></P><P><SPAN id="Ref8">[8] <A href="http://gene.bio.jhu.edu/Ourspdf/108.pdf" class="external text" rel="nofollow">D. A. Hodgson, C. M. Thomas: <B>SGM symposium 61: Signals, switches, regulons and cascades: control of bacterial gene expression</B>. Cambridge University Press. ISBN 0 521 81388 3 ©SGM 2002</A></SPAN></P><P><SPAN id="Ref9">[9] <A href="http://mic.sgmjournals.org/content/147/12/3241/F4.expansion.html" class="external text" rel="nofollow">A. Khlebnikov, K. Datsenko, T. Skaug, B. Wanner and J. Keasling: <B>Homogeneous expression of the P<SUB>BAD</SUB> promoter in Escherichia coli by constitutive expression of the low-affinity high-capacity AraE transporter,</B> Microbiology, 2001, (147): 3241–3247</A></SPAN></P><P><SPAN id="Ref10">[10] <A href="http://www.ncbi.nlm.nih.gov/pubmed/10829079" class="external text" rel="nofollow">Datsenko, KA, BL Wanner: <B>One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products</B>, 2000 Proc. Natl. Acad. Sci. U.S.A. 97(12):6640-5</A></SPAN></P><P><SPAN id="Ref11">[11] <A href="http://www.nature.com/nature/journal/v434/n7037/full/nature03461.html" class="external text" rel="nofollow">Subhayu Basu, Yoram Gerchman, Cynthia H. Collins, Frances H. Arnold &amp; Ron Weiss: <B>A synthetic multicellular system for programmed pattern formation</B>, Nature 2005, 434: 1130-11342</A></SPAN></P><P><SPAN id="Ref12">[12] <A href="http://jb.asm.org/cgi/content/abstract/182/3/805" class="external text" rel="nofollow">Kristi A. Egland and E. P. Greenberg: <B>Conversion of the Vibrio fischeri Transcriptional Activator, LuxR, to a Repressor</B>, Journal of Bacteriology, 2000, 182: 805-811</A></SPAN></P><P>Pymol:
</P><P><SPAN id="Ref13">[13] <A href="http://www.ncbi.nlm.nih.gov/Structure/mmdb/mmdbsrv.cgi?uid=49005" class="external text" rel="nofollow"><B>http://www.ncbi.nlm.nih.gov/Structure/mmdb/mmdbsrv.cgi?uid=49005</B> NCBI Protein Structure summary, 18.10.11</A></SPAN></P><P><SPAN id="Ref14">[14] <A href="http://www.ncbi.nlm.nih.gov/Structure/mmdb/mmdbsrv.cgi?uid=52511" class="external text" rel="nofollow"><B>http://www.ncbi.nlm.nih.gov/Structure/mmdb/mmdbsrv.cgi?uid=52511</B> NCBI Protein Structure summary, 18.10.11</A></SPAN></P></DIV><DIV style="position: relative; height: 149px; text-align: center; clear: both; margin: 20px auto 20px auto;" class="boxWrap"><TABLE style="height: 100%; width: 100%; border-spacing: 0px; " cellspacing="0"><TBODY><TR style="font-weight: bold;"><TD style="text-align: center; border-right: 1px solid black;">Back to iGEM</TD><TD colspan="4" style="text-align: center;"><SPAN id="SponsorsFooter">Our Sponsors</SPAN></TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="visualClear"> </DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:ETH_Zurich/Biology/Detector">http://2011.igem.org/Team:ETH_Zurich/Biology/Detector</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETH_Zurich/Biology/Detector" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETH_Zurich/Biology/Detector" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETH_Zurich/Biology/Detector&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETH_Zurich/Biology/Detector&amp;oldid=260966" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>