<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETH_Zurich_Biology_Cloning"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ETH_Zurich/Biology/Cloning">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETH_Zurich/Biology/Cloning&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Biology/Cloning&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Biology/Cloning&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/Biology/Cloning" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETH Zurich/Biology/Cloning</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV class="stickToTop" id="naviBox" style="text-align:center; line-height: 0px; font-size: 0px; z-index: 500;"><DIV style="position: relative; margin: 0 auto; display:inline-block; left: 0px; right: 0px;"><DIV align="center" id="naviMenu" style="left: 0px; right: 0px; margin: 0pt auto; text-align: left;"><UL class="dropdown dropdown-horizontal mainmenu"><LI class="first"><A href="/Team:ETH_Zurich">Home</A></LI><LI><A href="/Team:ETH_Zurich/Overview/Project" class="dir">Project <I>SmoColi</I></A></LI><LI class="first"><A href="/Team:ETH_Zurich/Overview/Project">What is SmoColi?</A></LI><LI><A href="/Team:ETH_Zurich/Overview/CircuitDesign">The Circuit</A></LI><LI><A href="/Team:ETH_Zurich/Overview/Informationprocessing">Information processing</A></LI><LI><A href="/Team:ETH_Zurich/Overview/Dialogue">Dialogue between Biology and Modeling</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Achievements">Achievements</A></LI></UL><LI><A href="/Team:ETH_Zurich/Achievements/Data_page">Data Page</A></LI><LI><A href="/Team:ETH_Zurich/Biology" class="dir">Biology</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Biology">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Biology/Detector">Network Elements</A></LI><LI><A href="/Team:ETH_Zurich/Biology/MolecularMechanism">Genetic Design</A></LI><LI><A href="/Team:ETH_Zurich/Biology/Cloning">Materials and Methods</A></LI><LI><A href="/Team:ETH_Zurich/Biology/Journal">Lab Journal</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Biology/Validation">Results</A></LI><LI><A href="/Team:ETH_Zurich/Process/Microfluidics/Overview" class="dir">Microfluidics</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Process/Microfluidics/Overview" class="dir">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Process/Microfluidics">Channel Design Development</A></LI><LI><A href="/Team:ETH_Zurich/Process/Validation">Final Design and Validation</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Process/Microfluidics/Proof">Proof of Concept</A></LI><LI><A href="/Team:ETH_Zurich/Modeling" class="dir">Modeling</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Modeling">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/SingleCell">Single-Cell Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Analysis">System Analysis</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Stochastic">Stochastic Analysis</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Microfluidics">Reaction-Diffusion Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Combined">Combined 3D Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Analytical_Approximation">Analytical Approximation</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Parameters">Parameters</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/References">References</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Modeling/Code">Code</A></LI><LI><A href="/Team:ETH_Zurich/Team/Human practices" class="dir">Human Practices</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Team/Human practices">Outreach</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Biology/Safety">Safety</A></LI><LI><A href="/Team:ETH_Zurich/Team" class="dir">Team</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Team">Team SmoColi</A></LI><LI><A href="/Team:ETH_Zurich/Team/Media">Photo Gallery</A></LI><LI><A href="/Team:ETH_Zurich/Team/Acknowledgements">Acknowledgements &amp; Contributions</A></LI><LI class="last"><A href="https://igem.org/Team.cgi?year=2011&amp;team_name=ETH_Zurich" target="_blank">Official Team Overview</A></LI><LI id="loginMenu"><A href="#" id="loginLink">Log In</A></LI><LI style="display:none;" id="editMenu"><A href="#" id="editLink" class="dir">Edit</A></LI><LI class="first"><A href="#" id="historyLink">History</A></LI><LI><A href="/Special:Upload" target="_blank">Upload File</A></LI><LI class="last"><A href="#" id="logoutLink">Log Out</A></LI><LI class="dir last" style="background-image: none; position: relative;"><FIELDSET><FORM method="post" id="searchForm" action=""><LABEL for="s">Search:</LABEL></FORM></FIELDSET></LI></DIV></DIV></DIV><DIV style="position: fixed; top: 37px; left: 0px; padding: 15px; width:220px; bottom: 0px; z-index: 600; overflow: auto;"><DIV style="width: 100%; text-align: center"><B><A href="#" onclick="document.getElementById('easteregg').style.display='block'" style="text-decoration:none;color:#000000;">Can you feel the smoke tonight?</A></B> </DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Material_and_methods"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Material and methods</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Double_digest_for_cloning"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Double digest for cloning</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Ligation"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Ligation</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Transformation"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Transformation</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#PCR"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">PCR</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Colony_PCR"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Colony PCR</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Preparation_of_glycerol_stocks"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Preparation of glycerol stocks</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#AlcR_testing"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">AlcR testing</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Chemically_competent_E._coli_.28JM_101.29"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">Chemically competent E. coli (JM 101)</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Chemically_competent_E._coli_.28DH5.CE.B1.29"><SPAN class="tocnumber">1.9</SPAN><SPAN class="toctext">Chemically competent E. coli (DH5α)</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#SDS_page"><SPAN class="tocnumber">1.10</SPAN><SPAN class="toctext">SDS page</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Western_Blot"><SPAN class="tocnumber">1.11</SPAN><SPAN class="toctext">Western Blot</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Diffusion_test_in_tubes"><SPAN class="tocnumber">1.12</SPAN><SPAN class="toctext">Diffusion test in tubes</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Mediums"><SPAN class="tocnumber">1.13</SPAN><SPAN class="toctext">Mediums</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Antibiotics"><SPAN class="tocnumber">1.14</SPAN><SPAN class="toctext">Antibiotics</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-16"><A href="#References"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE></DIV><DIV style="bottom: 0; left: 235px; overflow: auto; position: fixed; right: 0px; top: 32px;"><DIV style="margin-left: 20px; margin-right: 20px; padding-top: 20px;"><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="Material_and_methods"> Material and methods </SPAN></H1><P><B>A lot of lab work was done to assemble SmoColi .</B></P></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H2><SPAN class="mw-headline" id="Double_digest_for_cloning"> Double digest for cloning </SPAN></H2><TABLE border="1" align="center" style="text-align:left;"><TBODY><TR><TD>Variant 1
</TD><TD>Variant 2
</TD></TR><TR><TD>Plasmid
</TD><TD><UL><LI> x µl Plasmid (~2 µg if possible)
</LI><LI> 2 µl EcoRI      
</LI><LI> 2 µl XbaI
</LI><LI> 5 µl buffer 4
</LI><LI> 0.5 µl BSA
</LI><LI> adjust to 10 µl ddH<SUB>2</SUB>O
</LI></UL></TD><TD><UL><LI> x µl Plasmid (~2 µg if possible)
</LI><LI> 3 µl PstI
</LI><LI> 2 µl Spe I
</LI><LI> 5 µl buffer 2
</LI><LI> 0.5  µl BSA
</LI><LI> adjust to 10 µl ddH<SUB>2</SUB>O
</LI></UL></TD></TR><TR><TD>Insert
</TD><TD><UL><LI> x µl Insert (~2 µg if possible)
</LI><LI> 2 µl EcoRI 
</LI><LI> 3 µl Spe I  
</LI><LI> 5 µl	buffer 1
</LI><LI> 0.5 µl BSA 
</LI><LI> adjust to 10 µl ddH<SUB>2</SUB>O
</LI></UL></TD><TD><UL><LI> x µl Insert (~2 µg if possible)
</LI><LI> 3 µl XbaI
</LI><LI> 2 µl Pst 1   
</LI><LI> 5 µl buffer 3
</LI><LI> 0.5 µl BSA 
</LI><LI> adjust to 10 µl ddH<SUB>2</SUB>O
</LI></UL></TD></TR></TBODY></TABLE><CENTER> ⇒ incubate for 2 h at 37°C and gel purify </CENTER></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H2><SPAN class="mw-headline" id="Ligation"> Ligation </SPAN></H2><OL><LI> Ligation mix
</LI><LI> 1:5 ratio of plasmid backbone : Insert 
</LI><LI> 1 µl 10 x T4 Ligase buffer (vortex)
</LI><LI> 0.5 µl T4 Ligase
</LI><DL><DD> adjust to 10 µl with ddH<SUB>2</SUB>O
</DD></DL><LI> Let the mixture stay for 1h at room temperature
</LI><LI> Denature the ligase at 65 °C for 5 min
</LI></OL></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H2><SPAN class="mw-headline" id="Transformation"> Transformation </SPAN></H2><OL><LI> Thaw competent cells on ice.
</LI><LI> Add 200 µl cells to 10 µl ligation mix
</LI><LI> Place the mixture  put them on ice for 30 min
</LI><LI> Heatshock 30 sec at 42 °C
</LI><LI> Place the mixture again on ice for 2 min
</LI><LI> Add 4 x SOC medium (800 µl)
</LI><LI> Let them grow for 60 min at 37 °C
</LI><LI> Spin cells down for 5 min at low speed
</LI><LI> Remove the supernatant, resuspend the cells
</LI><LI> Spread 100 µl of the cells onto the plates.
</LI></OL></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H2><SPAN class="mw-headline" id="PCR"> PCR </SPAN></H2><P><B>PCR mixture</B></P><UL><LI> 10 µl 10x Phusion buffer
</LI><LI> 1 µl 10 mM dNTPs
</LI><LI> 2.5 µl 10 µM forward primer
</LI><LI> 2.5 µl 10 µM reverse primer
</LI><LI> 1 ng DNA template
</LI><LI> 0.2 µl Phusion polymerase
</LI><LI><I>optional DMSO</I></LI></UL><P>adjust to 50 µl with ddH<SUB>2</SUB>O
</P><P><B>PCR procedure</B></P><UL><LI> initial denaturation: 98 °C for 30 sec
</LI><LI> 35 cycles
</LI><LI> denaturation: 98 °C for 5 sec
</LI><LI> annealing: annealing temperature (see <A href="#Primer_list">Primer list</A>) for 10 sec
</LI><LI> extension: 72 °C for 30 sec per kb
</LI></UL><LI> final extension: 72°C for 5 min
</LI><LI> storage at 4°C
</LI></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H2><SPAN class="mw-headline" id="Colony_PCR"> Colony PCR </SPAN></H2><P>Pick a colony and resuspend in 10 µl ddH<SUB>2</SUB>O
</P><P><B>PCR mixture</B></P><UL><LI> 1 µl colony
</LI><LI> 1 µl 10x taq buffer
</LI><LI> 0.8 µl dNTPs
</LI><LI> 0.05 µl 10 µM forward primer
</LI><LI> 0.05 µl 10 µM reverse primer
</LI><LI> 0.1 µl Taq
</LI></UL><P>adjust to 10 µl with ddH<SUB>2</SUB>O
</P><P><B>PCR procedure</B></P><UL><LI> initial denaturation: 95 °C for 30 sec
</LI><LI> 25 cycles
</LI><LI> denaturation: 95 °C for 30 sec
</LI><LI> annealing: 55 °C for 30 sec
</LI><LI> extension: 68 °C for 90 sec
</LI></UL><LI> final extension: 68 °C for 5 min
</LI><LI> storage at 4 °C
</LI></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H2><SPAN class="mw-headline" id="Preparation_of_glycerol_stocks"> Preparation of glycerol stocks </SPAN></H2><P>From the overnight cultures 15 % glycerol were made and stored at -20 °C
</P></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H2><SPAN class="mw-headline" id="AlcR_testing"> AlcR testing </SPAN></H2><UL><LI> preparatory overnight culture of JM101 with AlcR-testsystem in LB medium
</LI><LI> cultures in M9 minimal medium
</LI><LI> all final experiments were done in 15 ml falcons with 3 ml cells
</LI><LI> AlcR production was induced with either 0, 1, 100 ng/ml anhydrotetracycline added at 4 °C
</LI><LI> For AlcR activation 0, 10, 1000 µM acetaldehyde was added at 4 °C
</LI><LI> Cells were incubated at 37 °C in closed tubes
</LI><LI> Fluorescence of GFP was measured after 1 h, 2 h and over night, all measurements were done in triplicates
</LI></UL></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H2><SPAN class="mw-headline" id="Chemically_competent_E._coli_.28JM_101.29"> Chemically competent <I>E. coli</I> (JM 101) </SPAN></H2><P><B>1 M MOPS</B></P><UL><LI> 10.46 g MOPS
</LI></UL><P>Total 50 ml
</P><P><B>1 M CaCl<SUB>2</SUB>•2H<SUB>2</SUB>O</B></P><UL><LI> 7.35 g CaCl<SUB>2</SUB></LI></UL><P>Total 50 ml
</P><P><B>3 M KOAc</B></P><UL><LI> 14.72 g KOAc
</LI></UL><P>Total 50 ml
</P><P><B>TFBI</B></P><UL><LI> 1.2 g RbCl
</LI><LI> 0.99 g MnCl<SUB>2</SUB>•4H<SUB>2</SUB>O
</LI><LI> 1 ml 1 M CaCl<SUB>2</SUB></LI><LI> 1 ml 3 M KOAc
</LI><LI> 15.33 ml glycerol 100 %
</LI></UL><P>Total 100 ml; bring to pH 5.8 with 0.2 N acetic acid; filter sterilize
</P><P><B>TFBII</B></P><UL><LI> 0.048 g RbCl
</LI><LI> 3 ml 1 M CaCl<SUB>2</SUB></LI><LI> 0.4 ml 1 M MOPS pH 7
</LI><LI> 7 ml glycerol 100 %
</LI></UL><P>filter sterilize
</P><P><B>Preparation</B></P><UL><LI> add 0.5 ml fresh O/N culture to 250 ml LB medium
</LI><LI> shake at 37 °C until OD<SUB>600</SUB> is 0.5-0.7
</LI><LI> cool on ice, spin down at 4 °C/ 3000rpm/ 5 min, put pellet on ice
</LI><LI> resuspend pellet in 5 ml TFBI with pipet, add 70 ml cool TFBI, incubate on ice for 2-4 h
</LI><LI> spin down at 4 °C/3krpm/ 5min
</LI><LI> resuspend pellet in 10 ml TFBII
</LI><LI> aliquot suspension into pre-cooled tubes, 200 µl each
</LI><LI> freeze aliquots immediately in dry ice, store at -80 °C
</LI></UL></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H2><SPAN class="mw-headline" id="Chemically_competent_E._coli_.28DH5.CE.B1.29"> Chemically competent <I>E. coli</I> (DH5α) </SPAN></H2><P>Done according to the <A href="http://openwetware.org/wiki/Preparing_chemically_competent_cells_%28Inoue%29" class="external text" rel="nofollow">Inoue protocol</A> on OpenWetWare
</P></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H2><SPAN class="mw-headline" id="SDS_page"> SDS page </SPAN></H2><P>(D-BSSE MolBioSkript 2011)
</P><P><B>Cast gels</B></P><TABLE border="1" align="center" style="text-align:center;"><CAPTION> For 4 gels:
</CAPTION><TBODY><TR><TD>Compound
</TD><TD>12% Running gel
</TD><TD>5% Stacking gel
</TD></TR><TR><TD>H<SUB>2</SUB>O (ml)
</TD><TD>6.7
</TD><TD>3.5
</TD></TR><TR><TD>1.5 M Tris-HCl, pH 8.8 (ml)
</TD><TD>5
</TD></TR><TR><TD>0.5 M Tris-HCl, pH 6.8 (ml)
</TD><TD>3
</TD></TR><TR><TD>10% (w/v) SDS (ml)
</TD><TD>0.2
</TD><TD>0.08
</TD></TR><TR><TD>Acrylamide/Bis-acrylamide (30%/ 0.8% w/v) (ml)
</TD><TD>8
</TD><TD>1.32
</TD></TR><TR><TD>10% (w/v) ammoniumpersulfate (APS) (ml)
</TD><TD>0.1
</TD><TD>0.1
</TD></TR><TR><TD>TEMED (&amp;mikro; l)
</TD><TD>30
</TD><TD>15
</TD></TR></TBODY></TABLE><P><B>Preparation</B></P><OL><LI> Clean glass plates with 30% EtOH (front- and back-glass plate, 10-spike comb)
</LI><LI> Assemble them in SDS-PAGE gel casting apparatus
</LI><LI> Mix the ingredients for the running gel except for APS and TEMED
</LI><LI> Add APS and TEMED to the lower gels solution, mix by gently 
</LI><LI> Pour the gels quickly with a 2 mL pipette. Leave about 1 cm below the bottom of the comb for the stacking gel and avoid formation of air-bubbles.
</LI><LI> Overlay your gels very carefully with isopropanol. The isopropanol helps to avoid drying-out of gels during polymerization and puts weight on the gel thereby helping to obtain a smooth surface.
</LI><LI> Wait until all gels have polymerized completely 
</LI><LI> Mix the components of the stacking gel while the running gel is polymerizing, leave out the APS and TEMED 
</LI><LI> Remove the isopropanol completely, wash with 0.5 mL of H<SUB>2</SUB>O to get rid of all isopropanol
</LI><LI> Add APS and TEMED to the stacking gel solution, mix by gently shaking the Falcon tube
</LI><LI> Pour the stacking gel on top of the running gel
</LI><LI> Insert the comb without introducing air-bubbles.
</LI><LI> Wait until gels have completely polymerized (approx. 30 min)
</LI><LI> Remove the gels from the casting apparatus, remove the combs, and clean the gels from gel-remnants
</LI></OL><P><B>SDS-PAGE</B></P><OL><LI> Assemble SDS-PAGE apparatus
</LI><LI> Load 5 µl protein marker into the first well
</LI><LI> Mix 50 μl samples with 50 µl Laemmli buffer
</LI><LI> Incubate the mix at 95°C for 10 min in a heating block
</LI><LI> Load 10 µl of your samples into the following wells (non-induced, induced…)
</LI><LI> Connect your gel chamber to the power supply, run the gel with 120 V fixed for around 40 min, watch the advance of the blue front
</LI><LI> When the blue front is ca. 0.5 cm before the end of the gel, stop the run and remove the gels from the apparatus
</LI><LI> Remove the gel from the glass-plates
</LI><LI> Add ca. 100 ml Coomassie staining solution to the gel and incubate on the shaker for ca. 30 min
</LI><LI> Rinse the gel with ddH<SUB>2</SUB>O
</LI><LI> Add ca. 100 ml destaining solution to the gel, incubate for until the protein-bands are clearly visible and background signals are almost completely disappeared
</LI></OL></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H2><SPAN class="mw-headline" id="Western_Blot"> Western Blot </SPAN></H2><P>The western blot was performed using the QIAGEN Penta-His HRP Conjugate Kit. A detailed description of the different steps can be found in <A href="#Ref1">[1]</A>.
</P><P>
Some <B>important notes</B> on the protocol:
</P><UL><LI> Prior to performance of the western blot, a SDS-PAGE gel was done as described in the chapter above.  
</LI><LI> We used the tank-blotting procedure as described in the protocol provided by the company.
</LI><LI> For immunodetection (chromogenic method) we used a HRP Conjugate Substrate Kit from Bio-Rad <A href="#Ref2">[2]</A>.
</LI></UL></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H2><SPAN class="mw-headline" id="Diffusion_test_in_tubes"> Diffusion test in tubes </SPAN></H2><UL><LI> agarose with <I>E. coli</I> with IPTG inducible GFP were filled in different tubes
</LI><LI> one end of the tube was put in a IPTG solution
</LI></UL></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H2><SPAN class="mw-headline" id="Mediums"> Mediums </SPAN></H2><P><B>Agar plates</B></P><UL><LI> 18g/l Agar added per liter respective medium
</LI></UL><P><B>SOB</B></P><UL><LI> 0.5 % (w/v) yeast extract
</LI><LI> 2 % (w/v) tryptone
</LI><LI> 10 mM NaCl
</LI><LI> 2.5 mM KCl
</LI><LI>20 mM MgSO<SUB>4</SUB></LI></UL><P>adjust to pH 7.5 by adding 1M NaOH
</P><P><B>SOC</B></P><UL><LI> SOB
</LI><LI> 20 mM glucose
</LI></UL><P><B>LB medium</B></P><UL><LI> 10 g Bacto-tryptone
</LI><LI> 5 g yeast extract
</LI><LI> 10 g NaCl 
</LI></UL><P>Total 1 l
</P><P><B>M9 minimal medium</B></P><P>10x M9
</P><UL><LI>12.8 g Na<SUB>2</SUB>HPO<SUB>4</SUB></LI><LI> 3 g KH<SUB>2</SUB>PO<SUB>4</SUB></LI><LI> 0.5 g NaCl
</LI><LI> 1 g NH<SUB>4</SUB>Cl 
</LI></UL><P>Total 100 ml 
</P><P>autoclave seperatly
</P><UL><LI> 10 x M9
</LI><LI> 1 M MgSO<SUB>4</SUB></LI><LI> 1 M CaCl<SUB>2</SUB></LI><LI> 1 % (w/v) thiamine solution
</LI><LI> 20 % (w/v) glucose
</LI></UL></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H2><SPAN class="mw-headline" id="Antibiotics"> Antibiotics </SPAN></H2><UL><LI> Ampicilin: 50ug/ml for low copy and 150ug/ml for high copy plasmids
</LI><LI> Chloramphenicol: 34ug/ml for low copy and 100ug/ml for high copy plasmids
</LI><LI> Kanamycin: 10ug/ml for low copy and 30ug/ml for high copy plasmids
</LI></UL></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="References"> References </SPAN></H1><P><SPAN id="Ref1">[1] <A href="http://www.qiagen.com/literature/handbooks/literature.aspx?id=1000171" class="external text" rel="nofollow">QIAGEN Handbooks and Protocols</A></SPAN></P><P><SPAN id="Ref2">[2] <A href="http://www3.bio-rad.com/B2B/BioRad/jsp/default.do?BV_SessionID=@@@@0347918889.1319835173@@@@&amp;BV_EngineID=ccchadfemklihldcfngcfkmdhkkdfll.0&amp;catalogNumber=170-6431&amp;contentDescrip=contains+premixed+4-chloro-1-naphthol%2C+hydrogen+peroxide+solutions%2C+color+development+buffer%3B+makes+1+L+color+development+solution&amp;nextPage=BRCatalogPage&amp;oid=10008" class="external text" rel="nofollow">Bio-Rad HRP Conjugate Substrate Kit</A></SPAN></P></DIV><DIV style="position: relative; height: 149px; text-align: center; clear: both; margin: 20px auto 20px auto;" class="boxWrap"><TABLE style="height: 100%; width: 100%; border-spacing: 0px; " cellspacing="0"><TBODY><TR style="font-weight: bold;"><TD style="text-align: center; border-right: 1px solid black;">Back to iGEM</TD><TD colspan="4" style="text-align: center;"><SPAN id="SponsorsFooter">Our Sponsors</SPAN></TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="visualClear"> </DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:ETH_Zurich/Biology/Cloning">http://2011.igem.org/Team:ETH_Zurich/Biology/Cloning</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETH_Zurich/Biology/Cloning" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETH_Zurich/Biology/Cloning" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETH_Zurich/Biology/Cloning&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETH_Zurich/Biology/Cloning&amp;oldid=255987" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>