<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETH_Zurich_Biology_MolecularMechanism"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ETH_Zurich/Biology/MolecularMechanism">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETH_Zurich/Biology/MolecularMechanism&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Biology/MolecularMechanism&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Biology/MolecularMechanism&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/Biology/MolecularMechanism" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETH Zurich/Biology/MolecularMechanism</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV class="stickToTop" id="naviBox" style="text-align:center; line-height: 0px; font-size: 0px; z-index: 500;"><DIV style="position: relative; margin: 0 auto; display:inline-block; left: 0px; right: 0px;"><DIV align="center" id="naviMenu" style="left: 0px; right: 0px; margin: 0pt auto; text-align: left;"><UL class="dropdown dropdown-horizontal mainmenu"><LI class="first"><A href="/Team:ETH_Zurich">Home</A></LI><LI><A href="/Team:ETH_Zurich/Overview/Project" class="dir">Project <I>SmoColi</I></A></LI><LI class="first"><A href="/Team:ETH_Zurich/Overview/Project">What is SmoColi?</A></LI><LI><A href="/Team:ETH_Zurich/Overview/CircuitDesign">The Circuit</A></LI><LI><A href="/Team:ETH_Zurich/Overview/Informationprocessing">Information processing</A></LI><LI><A href="/Team:ETH_Zurich/Overview/Dialogue">Dialogue between Biology and Modeling</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Achievements">Achievements</A></LI></UL><LI><A href="/Team:ETH_Zurich/Achievements/Data_page">Data Page</A></LI><LI><A href="/Team:ETH_Zurich/Biology" class="dir">Biology</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Biology">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Biology/Detector">Network Elements</A></LI><LI><A href="/Team:ETH_Zurich/Biology/MolecularMechanism">Genetic Design</A></LI><LI><A href="/Team:ETH_Zurich/Biology/Cloning">Materials and Methods</A></LI><LI><A href="/Team:ETH_Zurich/Biology/Journal">Lab Journal</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Biology/Validation">Results</A></LI><LI><A href="/Team:ETH_Zurich/Process/Microfluidics/Overview" class="dir">Microfluidics</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Process/Microfluidics/Overview" class="dir">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Process/Microfluidics">Channel Design Development</A></LI><LI><A href="/Team:ETH_Zurich/Process/Validation">Final Design and Validation</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Process/Microfluidics/Proof">Proof of Concept</A></LI><LI><A href="/Team:ETH_Zurich/Modeling" class="dir">Modeling</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Modeling">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/SingleCell">Single-Cell Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Analysis">System Analysis</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Stochastic">Stochastic Analysis</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Microfluidics">Reaction-Diffusion Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Combined">Combined 3D Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Analytical_Approximation">Analytical Approximation</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Parameters">Parameters</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/References">References</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Modeling/Code">Code</A></LI><LI><A href="/Team:ETH_Zurich/Team/Human practices" class="dir">Human Practices</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Team/Human practices">Outreach</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Biology/Safety">Safety</A></LI><LI><A href="/Team:ETH_Zurich/Team" class="dir">Team</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Team">Team SmoColi</A></LI><LI><A href="/Team:ETH_Zurich/Team/Media">Photo Gallery</A></LI><LI><A href="/Team:ETH_Zurich/Team/Acknowledgements">Acknowledgements &amp; Contributions</A></LI><LI class="last"><A href="https://igem.org/Team.cgi?year=2011&amp;team_name=ETH_Zurich" target="_blank">Official Team Overview</A></LI><LI id="loginMenu"><A href="#" id="loginLink">Log In</A></LI><LI style="display:none;" id="editMenu"><A href="#" id="editLink" class="dir">Edit</A></LI><LI class="first"><A href="#" id="historyLink">History</A></LI><LI><A href="/Special:Upload" target="_blank">Upload File</A></LI><LI class="last"><A href="#" id="logoutLink">Log Out</A></LI><LI class="dir last" style="background-image: none; position: relative;"><FIELDSET><FORM method="post" id="searchForm" action=""><LABEL for="s">Search:</LABEL></FORM></FIELDSET></LI></DIV></DIV></DIV><DIV style="position: fixed; top: 37px; left: 0px; padding: 15px; width:220px; bottom: 0px; z-index: 600; overflow: auto;"><DIV style="width: 100%; text-align: center"><B><A href="#" onclick="document.getElementById('easteregg').style.display='block'" style="text-decoration:none;color:#000000;">Can you feel the smoke tonight?</A></B> </DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Genetic_Design"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Genetic Design</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Plasmid_design"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Plasmid design</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#pSB6A5"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">pSB6A5</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-4"><A href="#Cloning_strategy"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Cloning strategy</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#General_procedure"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">General procedure</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Sources_of_genes_and_promoters"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Sources of genes and promoters</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Cloning_successes_and_problems"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Cloning successes and problems</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Choice_and_design_of_ribosome_binding_sites"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Choice and design of ribosome binding sites</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#References"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE></DIV><DIV style="bottom: 0; left: 235px; overflow: auto; position: fixed; right: 0px; top: 32px;"><DIV style="margin-left: 20px; margin-right: 20px; padding-top: 20px;"><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="Genetic_Design"> Genetic Design </SPAN></H1><P><B>In a System where so many trancriptional regulators effect each other, their respective levels are very important factor in the establishment of a running system.</B> The expression rate is controlled with a multi plasmid operation where the genes are assembled on different plasmids with different copy numbers. Also we put a lot of effort in the design of the ribosome binding site. Below we provide a description of the plasmid design, the strategy we used to clone the necessary parts and the method of choosing the ribosome binding sites.
</P></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="Plasmid_design"> Plasmid design </SPAN></H1><DIV class="thumb tright"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Table 1: Plasmid design for the AlcR system</B> Transcriptional terminators included for each gene. Plasmid copy numbers adapted from <A href="#Ref1">[1]</A>.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Table 2: Plasmid design for the XylR system</B> Transcriptional terminators included for each gene. Plasmid copy numbers adapted from <A href="#Ref1">[1]</A>. For a detailed description of the pCK04AxylR plasmid see <A href="#Ref3">[3]</A>.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Table 3: Plasmid design for the AraC test system</B> Transcriptional terminators included for each gene. Plasmid copy numbers adapted from <A href="#Ref1">[1]</A>.</DIV></DIV></DIV><P>
For the biological implementation of the SmoColi system, we constructed a <B>three-plasmid system</B> with different antibiotic resistances. This setup was chosen in order to clone the whole SmoColi system into the same bacterial cells.
</P><P>The vectors used in our setup are the parts pSB6A5, pSB3C5, pSB4C5, pSB3K3 and pSB4K5. These plasmids contain different origins of replication, thus resulting in different copy numbers of the corresponding plasmid in <I>E.coli</I>. pBR322, p15A and pSC101 origins are used as a stable three-plasmid system <A href="#Ref2">[2]</A>. The copy number influences the amount of protein produced by the cell and plays a crucial role in our design, especially for the bandpass filter. Depending on the function of a particular protein in our system we had to evaluate on which plasmid the corresponding gene should be cloned. 
</P><P>In the xylene-responsive system we included an artificial degradation pathway (<I>xylWCMABN</I>) so that we could establish a xylene gradient. For this purpose the plasmid pCK04AxylR was added <A href="#Ref3">[3]</A>. To avoid incompatibilities of plasmids we adapted the rest of our system and changed the composition of the genes on each vector compared to the AlcR system.
</P><P>For proof of concept of the bandpass filter, we also constructed a system with AraC as an arabinose-dependent sensor. Here we again change the plasmid design in order to match the requirements of the test system. Therefore we introduced a plasmid (pSB4C5) containing the <I>araC</I> regulator gene and the degradation pathway for arabinose, <I>araBAD</I>. Since AraC is activating P<SUB>BAD</SUB> upon sensing arabinose, the design for the other two plasmids could be adapted from the design of the xylene system. Only P<SUB>u</SUB> promoters were exchanged with P<SUB>BAD</SUB>.
</P><H2><SPAN class="mw-headline" id="pSB6A5">pSB6A5</SPAN></H2><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 1: Improvement of the plasmid pSB6A1 to pSB6A5</B></DIV></DIV></DIV><P>The pSB6A5 plasmid we constructed by <B>reducing the size</B> of the antibotic (Ampicillin) and origin of replication (pMB1) casettes of pSB6A1 to a minimum and by introducing <B>terminators around the multiple cloning site</B>. PMB1 is a medium copy orign with about 15-20 copies per cell. For the characterization of pSB6A5 <A href="/Team:ETH_Zurich/Biology/Validation#Copy_number_test" title="Team:ETH Zurich/Biology/Validation">click here</A></P></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="Cloning_strategy">Cloning strategy </SPAN></H1><H2><SPAN class="mw-headline" id="General_procedure"> General procedure </SPAN></H2><DIV class="thumb tleft"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><B>Figure 1: Cloning strategy for SmoColi</B></DIV></DIV></DIV><P>For cloning our parts we tried to follow the strategies proposed in the registry of standard biological parts <A href="#Ref4">[4]</A> as much as possible . The general cloning scheme was performed as follows:
</P><UL><LI> If available, genes and promoters in SmoColi were obtained from the iGEM spring 2011 distribution kit. A codon-optimized version of <I>alcR</I> and <I>P<SUB>tet</SUB>-lacI<SUB>M1</SUB></I> were synthesized. <I>P<SUB>alc</SUB></I>- and <I>P<SUB>U</SUB></I>-promoters were obtained by PCR. 
</LI></UL><UL><LI> After isolation of the DNA, ribosome binding sites and prefix (5' gaattcgcggccgcttctagag 3') respectively suffix (5' tactagtagcggccgctgcag 3') were added to the gene by polymerase chain reaction. The PCR products were purified and afterwards digested with <I>XbaI-PstI</I>. 
</LI></UL><UL><LI> Plasmid vectors containing the corresponding promoter were linearized by <I>SpeI-PstI</I> and the <I>XbaI-PstI</I> fragment containing the gene of interest was ligated into the backbone. 
</LI></UL><UL><LI> In a next step we isolated a <I>EcoRI-SpeI</I> fragment with promoter, ribosome binding site and gene and inserted it into a <I>EcoRI-XbaI</I> cut vector containing a transcriptional terminator.
</LI></UL><UL><LI> In the end all the constructed parts were assembled on the corresponding vectors for the final SmoColi system. Some parts (especially <I>lacI</I> in an actively expressed form) appeared to be negatively selected by <I>E.coli</I>. So we had to assemble the parts in a way that <I>lacI</I> was always repressed.
</LI></UL><H2><SPAN class="mw-headline" id="Sources_of_genes_and_promoters"> Sources of genes and promoters </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Table 4: Promoters of SmoColi:</B> source and final constructs of the different used promoters</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Table 3: Genes of SmoColi: </B>source and final constructs of the different used genes</DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Cloning_successes_and_problems"> Cloning successes and problems </SPAN></H2><P>All parts of the system were successfully assembled by this strategy, except for <I>lacI</I> and <I>cI</I> under control of the Tet promoter. Whenever we were sequencing a clone of either of those constructs, we always found protein inactivating mutations within the primer annealing part - either being deletions of the Shine-Dalgarno sequence of the 
RBS or frame shift mutations within the first bases of the coding regions. We first believed the problem to lie in the synthesis of the very long oligonucleotides, for adding the whole RBS and prefix in front of the genes. But also performing a whole plasmid PCR with short primers did not solve the problem and we still only got non-protein-generating sequences.
We believe this to be due to lethality of high overexpression levels of LacI or CI, leading to a selection for clones mutated in a way that <I>lacI</I> or <I>cI</I> is not expressed. Thus one always has to <B>make sure to clone <I>lacI</I> or <I>cI</I> behind downregulated promoters</B>, for example for P<SUB>Tet</SUB> you have to make sure that there is TetR present in the cell. Those problems unforunately delayed the construction of the complete system to only shortly before Wikifreeze, thus we hope to produce as many results as possible till the Jamboree.
</P></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="Choice_and_design_of_ribosome_binding_sites"> Choice and design of ribosome binding sites </SPAN></H1><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Table 5: Different ribosome binding sites and their estimated strengths</B> which are included in our design. The strengths of the sequences were estimated with a RBS strength calculator <A href="#Ref7">[7]</A>. In the last column it is listed in front of which genes a particular RBS was added.</DIV></DIV></DIV><P>Another important role in the synthesis of proteins is played by the ribosome binding site (RBS), the location where the ribosomes are bound to the mRNA at the initiation of the translation process.
The ribosome binding sites which we included in our system were designed in several different ways:
</P><UL><LI> For the bandpass filter we implemented the ones which are also used for the plasmids in <A href="#Ref5">[5]</A>. 
</LI></UL><UL><LI> For the other parts we either tried to use <B>naturally occurring ribosome binding sites</B> or took specific ones from the Anderson library of ribosome binding sites <A href="#Ref6">[6]</A>.
</LI></UL><P>The strength of all the ribosome binding sites used was estimated with a <B>RBS calculator</B>. Although the efficiency of protein synthesis is not directly proportional to the estimated values, they can still indicate whether a certain RBS is more or less strong. Below you can find a list of the different ribosome binding sites used, their estimated binding strengths and in front of which genes they were put.
</P></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="References"> References </SPAN></H1><P><SPAN id="Ref1">[1] <A href="http://openwetware.org/wiki/Arking:JCAOligoTutorial8" class="external text" rel="nofollow">OpenWetWare <B>Arking: JCAOligoTutorial8</B>, 0 Jul 2008, 04:32 UTC. 21 Sep 2011, 01:32</A></SPAN></P><P><SPAN id="Ref2">[2] <A href="http://www.springerlink.com/content/3fj1xxl9p42kx8l5/" class="external text" rel="nofollow">Karl Friehs: <B>Plasmid Copy Number and Plasmid Stability</B>, Advances in Biochemical Engineering/Biotechnology, 2004, 86: 22-192</A></SPAN></P><P><SPAN id="Ref3">[3] <A href="http://aem.asm.org/cgi/content/abstract/64/2/748" class="external text" rel="nofollow">Sven Panke, Juan M. Sánchez-Romero, and Víctor de Lorenzo: <B>Engineering of Quasi-Natural Pseudomonas putida Strains for Toluene Metabolism through an ortho-Cleavage Degradation Pathway</B>, Appl Environ Microbiol, February 1998, 64: 748-751</A></SPAN></P><P><SPAN id="Ref4">[4] <A href="http://partsregistry.org/Help:Contents" class="external text" rel="nofollow">Registry of Standard Biological Parts <B>Help:contents</B> 21 Sep 2011, 02:59</A></SPAN></P><P><SPAN id="Ref5">[5] <A href="http://www.nature.com/nature/journal/v434/n7037/full/nature03461.html" class="external text" rel="nofollow">Subhayu Basu, Yoram Gerchman1, Cynthia H. Collins, Frances H. Arnold &amp; Ron Weiss: <B>A synthetic multicellular system for programmed pattern formation</B>, Nature, 2005, 434: 1130-11342</A></SPAN></P><P><SPAN id="Ref6">[6] <A href="http://partsregistry.org/Ribosome_Binding_Sites/Prokaryotic/Constitutive/Anderson" class="external text" rel="nofollow">J. Christopher Anderson <B>Ribosome Binding Sites/Prokaryotic/Constitutive/Community Collection</B> Registry of Standard Biological Parts, 21 Sep 2011, 08:49</A></SPAN></P></DIV><DIV style="position: relative; height: 149px; text-align: center; clear: both; margin: 20px auto 20px auto;" class="boxWrap"><TABLE style="height: 100%; width: 100%; border-spacing: 0px; " cellspacing="0"><TBODY><TR style="font-weight: bold;"><TD style="text-align: center; border-right: 1px solid black;">Back to iGEM</TD><TD colspan="4" style="text-align: center;"><SPAN id="SponsorsFooter">Our Sponsors</SPAN></TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="visualClear"> </DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:ETH_Zurich/Biology/MolecularMechanism">http://2011.igem.org/Team:ETH_Zurich/Biology/MolecularMechanism</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETH_Zurich/Biology/MolecularMechanism" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETH_Zurich/Biology/MolecularMechanism" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETH_Zurich/Biology/MolecularMechanism&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETH_Zurich/Biology/MolecularMechanism&amp;oldid=256255" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>