<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETH_Zurich_Modeling_Stochastic"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ETH_Zurich/Modeling/Stochastic">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETH_Zurich/Modeling/Stochastic&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Modeling/Stochastic&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Modeling/Stochastic&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/Modeling/Stochastic" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETH Zurich/Modeling/Stochastic</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV class="stickToTop" id="naviBox" style="text-align:center; line-height: 0px; font-size: 0px; z-index: 500;"><DIV style="position: relative; margin: 0 auto; display:inline-block; left: 0px; right: 0px;"><DIV align="center" id="naviMenu" style="left: 0px; right: 0px; margin: 0pt auto; text-align: left;"><UL class="dropdown dropdown-horizontal mainmenu"><LI class="first"><A href="/Team:ETH_Zurich">Home</A></LI><LI><A href="/Team:ETH_Zurich/Overview/Project" class="dir">Project <I>SmoColi</I></A></LI><LI class="first"><A href="/Team:ETH_Zurich/Overview/Project">What is SmoColi?</A></LI><LI><A href="/Team:ETH_Zurich/Overview/CircuitDesign">The Circuit</A></LI><LI><A href="/Team:ETH_Zurich/Overview/Informationprocessing">Information processing</A></LI><LI><A href="/Team:ETH_Zurich/Overview/Dialogue">Dialogue between Biology and Modeling</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Achievements">Achievements</A></LI></UL><LI><A href="/Team:ETH_Zurich/Achievements/Data_page">Data Page</A></LI><LI><A href="/Team:ETH_Zurich/Biology" class="dir">Biology</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Biology">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Biology/Detector">Network Elements</A></LI><LI><A href="/Team:ETH_Zurich/Biology/MolecularMechanism">Genetic Design</A></LI><LI><A href="/Team:ETH_Zurich/Biology/Cloning">Materials and Methods</A></LI><LI><A href="/Team:ETH_Zurich/Biology/Journal">Lab Journal</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Biology/Validation">Results</A></LI><LI><A href="/Team:ETH_Zurich/Process/Microfluidics/Overview" class="dir">Microfluidics</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Process/Microfluidics/Overview" class="dir">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Process/Microfluidics">Channel Design Development</A></LI><LI><A href="/Team:ETH_Zurich/Process/Validation">Final Design and Validation</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Process/Microfluidics/Proof">Proof of Concept</A></LI><LI><A href="/Team:ETH_Zurich/Modeling" class="dir">Modeling</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Modeling">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/SingleCell">Single-Cell Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Analysis">System Analysis</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Stochastic">Stochastic Analysis</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Microfluidics">Reaction-Diffusion Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Combined">Combined 3D Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Analytical_Approximation">Analytical Approximation</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Parameters">Parameters</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/References">References</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Modeling/Code">Code</A></LI><LI><A href="/Team:ETH_Zurich/Team/Human practices" class="dir">Human Practices</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Team/Human practices">Outreach</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Biology/Safety">Safety</A></LI><LI><A href="/Team:ETH_Zurich/Team" class="dir">Team</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Team">Team SmoColi</A></LI><LI><A href="/Team:ETH_Zurich/Team/Media">Photo Gallery</A></LI><LI><A href="/Team:ETH_Zurich/Team/Acknowledgements">Acknowledgements &amp; Contributions</A></LI><LI class="last"><A href="https://igem.org/Team.cgi?year=2011&amp;team_name=ETH_Zurich" target="_blank">Official Team Overview</A></LI><LI id="loginMenu"><A href="#" id="loginLink">Log In</A></LI><LI style="display:none;" id="editMenu"><A href="#" id="editLink" class="dir">Edit</A></LI><LI class="first"><A href="#" id="historyLink">History</A></LI><LI><A href="/Special:Upload" target="_blank">Upload File</A></LI><LI class="last"><A href="#" id="logoutLink">Log Out</A></LI><LI class="dir last" style="background-image: none; position: relative;"><FIELDSET><FORM method="post" id="searchForm" action=""><LABEL for="s">Search:</LABEL></FORM></FIELDSET></LI></DIV></DIV></DIV><DIV style="position: fixed; top: 37px; left: 0px; padding: 15px; width:220px; bottom: 0px; z-index: 600; overflow: auto;"><DIV style="width: 100%; text-align: center"><B><A href="#" onclick="document.getElementById('easteregg').style.display='block'" style="text-decoration:none;color:#000000;">Can you feel the smoke tonight?</A></B> </DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Stochastic_Analysis"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Stochastic Analysis</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Method"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Method</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Simulations"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Simulations</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Analysis_of_Results"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Analysis of Results</SPAN></A></LI></UL></TD></TR></TBODY></TABLE></DIV><DIV style="bottom: 0; left: 235px; overflow: auto; position: fixed; right: 0px; top: 32px;"><DIV style="margin-left: 20px; margin-right: 20px; padding-top: 20px;"><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="Stochastic_Analysis"> Stochastic Analysis </SPAN></H1><P><B>Besides the parameter space search, we performed another type of analysis in order to verify that our system is robust and that it is monostable (i.e. it has only one steady state). We were especially interested in the GFP band, whether it is always present and whether the amount of GFP produced in the cells for different concentrations of Acetaldehyde has large fluctuations. Because gene expression is an intrinsically stochastic process, we performed stochastic simulations to see how our system reacts to noise and how it responds to perturbations.</B><B>The following results show that our system is indeed robust to noise and it does not have bifurcations.</B></P></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="Method"> Method </SPAN></H1><P>For the stochastic simulations we used the <A href="http://magnet.systemsbiology.net/software/Dizzy/" class="external text" rel="nofollow">Dizzy</A> software, to which we gave as an input an SBML file with the description of our system. Moving from deterministic to stochastic simulations we had to change several things in the SBML file. Firstly, we exported the SBML file to a CMDL format.Next, we converted the species values as well as the production rates and repression coefficients from concentrations (in μM) to number of molecules. Since our initial system was described in terms of ODEs, we also had to change this by separating the system into distinct reaction channels (i.e. separate degradation from activation/repression) and specifying the reaction rates.

With the new description of the system, we could start the simulations. For analyzing the robustness of the system, we chose 5 strategic Acetaldehyde concentrations (see Figure 1), and for each of them we performed stochastic simulations. We selected the following Acetaldehyde concentrations:
</P><UL><LI> 0 μM (0 molecules). At this concentration the GFP output should approach 0 μM.
</LI><LI> 25 μM (30110 molecules). At this concentration GFP rises quickly, so it should be very sensitive to noise.
</LI><LI> 46 μM (55402 molecules). This is the concentration that gives the peak GFP value.
</LI><LI> 100 μM (120440 molecules). At this concentration GFP is decreasing, so it should be sensitive to noise.
</LI><LI> 300 μM (361320 molecules). At this concentration the GFP output should again approach 0 μM. 
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B>Figure 1</B>: GFP versus Acetaldehyde. The red squares denote the 5 Acetaldehyde concentrations for which we simulated the time course of GFP steady state in stochastic mode.</DIV></DIV></DIV></DIV><P>In order to be sure that we are indeed simulating the time course of the &quot;stochastic steady state&quot;, and that we are not in the transient regime, we implemented the following approach:
</P><UL><LI> We ran a deterministic simulation for 5000 minutes (~ 83 hours) to get to the deterministic steady state. We took over the concentration values for the last time point as new initial conditions for the next simulation.
</LI><LI> We ran a stochastic simulation for 10000 minutes (~ 7 days), to make sure that we are somewhere around the “stochastic steady state”. Once again we took over the concentration values for the last time point as new initial conditions for the next simulation.
</LI><LI> We ran a long stochastic simulation for 100000 minutes (~ 70 days) storing the molecule numbers of the species every minute. 
</LI></UL><P>
For the stochastic simulations we used the &quot;Gillespie-direct&quot; option, since this gives an exact solution, compared to the other solvers which only approximate the solution.
</P></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="Simulations"> Simulations </SPAN></H1><P>After we obtained the samples with Dizzy, we loaded the data in Matlab to analyze it. To get an initial feeling of how the system behaves in the stochastic regime, we plotted the GFP molecule numbers as function of time. In Figure 2 we can see that for the peak of the band (Acetaldehyde = 46 μM), GFP values fluctuate around a mean value which is about 23500 molecules.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B>Figure 2</B>: Stochastic simulation of GFP steady state for an Acetaldehyde concentration of 46 μM.</DIV></DIV></DIV></DIV><P>Next, in order to see how the values are distributed and whether there are multiple steady states, we plotted the normalized histograms for the 5 data sets. We used 150 bins for plotting the histograms for the 3 intermediate Acetaldehyde values and 28 and 61 bins for the first and last Acetaldehyde value, respectively. In these last 2 cases we had to reduce the number of bins because the range of the GFP molecule values was narrower.
</P><TABLE border="0" width="100%"><TBODY><TR valign="top"><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 3</B>: Distribution of the number of GFP molecules for an Acetaldehyde input of 0 μM</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 4</B>: Distribution of the number of GFP molecules for an Acetaldehyde input of 25 μM</DIV></DIV></DIV></DIV></TD></TR><TR valign="top"><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 5</B>: Distribution of the number of GFP molecules for an Acetaldehyde input of 46 μM</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 6</B>: Distribution of the number of GFP molecules for an Acetaldehyde input of 100 μM</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 7</B>: Distribution of the number of GFP molecules for an Acetaldehyde input of 300 μM</DIV></DIV></DIV></DIV></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="Analysis_of_Results"> Analysis of Results </SPAN></H1><P>From the time course plot (Figure 2) we can already see that the values always fluctuate around one mean value which leads us to believe that the system is stable. Since the numbers of GFP molecules do not jump between two or more distinct values we can assume that the system is not bistable. The plot alone however does not constitute conclusive evidence, so for this reason we generated and analyzed the distributions of GFP values.

All of the 5 distributions have a Gaussian-like shape, which means that the GFP values cluster around one mean value. The distributions are relatively narrow, which means there are no large fluctuations and there are unimodal, which corresponds to a having no multiple stead states in the deterministic regime. 
</P><TABLE style="margin: 1em auto 1em auto; width: 80%;" class="wikitable sortable"><TBODY><TR><TH>Acetaldehyde concentration
</TH><TH>GFP mean (no. of molecules)
</TH><TH>GFP mean (μM)
</TH><TH>Deterministic GFP value (μM)
</TH><TH>GFP standard deviation (no. of molecules)
</TH><TH>GFP standard deviation (μM)
</TH><TH>GFP relative standard deviation (%)
</TH></TR><TR><TD>0 μM
</TD><TD>12.2903
</TD><TD>0.0102 μM
</TD><TD>0.01 μM
</TD><TD>3.5473
</TD><TD>0.0029 μM
</TD><TD>28.86 %
</TD></TR><TR><TD>25 μM
</TD><TD>1954.4
</TD><TD>1.6227 μM
</TD><TD>1.7157 μM
</TD><TD>544.6184
</TD><TD>0.4522 μM
</TD><TD>27.86 %
</TD></TR><TR><TD>46 μM
</TD><TD>23491
</TD><TD>19.5046 μM
</TD><TD>19.6046 μM
</TD><TD>669.5291
</TD><TD>0.5559 μM
</TD><TD>2.85 %
</TD></TR><TR><TD>100 μM
</TD><TD>2981.7
</TD><TD>2.4757 μM
</TD><TD>2.4830 μM
</TD><TD>243.6979
</TD><TD>0.2023 μM
</TD><TD>8.17 %
</TD></TR><TR><TD>300 μM
</TD><TD>40.0056
</TD><TD>0.0332 μM
</TD><TD>0.0332 μM
</TD><TD>7.1426
</TD><TD>0.0059 μM
</TD><TD>17.85 %
</TD></TR></TBODY></TABLE><P>In the table above we can see that the mean values of the distributions of GFP molecule numbers (in concentration) obtained from the stochastic simulations are very similar to the deterministic ones obtained previously by simulating the system with an ODE solver. This confirms once again the fact that the deterministic steady state corresponds to the stochastic one, thus it sees that we do not have bifurcations.. 

Analyzing the relative standard deviations (RSD), we notice that the highest value was obtained for an input concentration of acetaldehyde equal to 0 μM. This is due to the fact that in this case GFP is repressed, so there is only leaky GFP expression. As the GFP output is on the order of a few up to tens of molecules, the ratio between the value of the mean and the amplitude of the fluctuations is much higher than the absolute value of the fluctuations. 

The lowest RSD corresponds to the acetaldehyde concentration that gives the GFP peak (46 μM). This is in accordance to the central limit theorem, since for higher values of the mean the relative value of the fluctuations decreases. We can notice that for 46 μM of acetaldehyde the standard deviation is on the same order of magnitude as for 25 and 100 μM of acetaldehyde, however the RSD is much smaller for the peak value.

The RSD is higher in the region where the GFP increases compared the region where it decreases despite the fact the value of the mean is comparable. This can be explained by taking into account the steepness of the slope which is higher for 25 than for 100 μM of acetaldehyde. On the left side of the peak the GFP value changes more quickly than on the right side, meaning that it is more prone to larger fluctuations. 
</P></DIV><DIV style="position: relative; height: 149px; text-align: center; clear: both; margin: 20px auto 20px auto;" class="boxWrap"><TABLE style="height: 100%; width: 100%; border-spacing: 0px; " cellspacing="0"><TBODY><TR style="font-weight: bold;"><TD style="text-align: center; border-right: 1px solid black;">Back to iGEM</TD><TD colspan="4" style="text-align: center;"><SPAN id="SponsorsFooter">Our Sponsors</SPAN></TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="visualClear"> </DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:ETH_Zurich/Modeling/Stochastic">http://2011.igem.org/Team:ETH_Zurich/Modeling/Stochastic</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETH_Zurich/Modeling/Stochastic" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETH_Zurich/Modeling/Stochastic" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETH_Zurich/Modeling/Stochastic&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETH_Zurich/Modeling/Stochastic&amp;oldid=257112" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>