<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETH_Zurich_Overview_Informationprocessing"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ETH_Zurich/Overview/Informationprocessing">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETH_Zurich/Overview/Informationprocessing&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Overview/Informationprocessing&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Overview/Informationprocessing&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/Overview/Informationprocessing" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETH Zurich/Overview/Informationprocessing</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV class="stickToTop" id="naviBox" style="text-align:center; line-height: 0px; font-size: 0px; z-index: 500;"><DIV style="position: relative; margin: 0 auto; display:inline-block; left: 0px; right: 0px;"><DIV align="center" id="naviMenu" style="left: 0px; right: 0px; margin: 0pt auto; text-align: left;"><UL class="dropdown dropdown-horizontal mainmenu"><LI class="first"><A href="/Team:ETH_Zurich">Home</A></LI><LI><A href="/Team:ETH_Zurich/Overview/Project" class="dir">Project <I>SmoColi</I></A></LI><LI class="first"><A href="/Team:ETH_Zurich/Overview/Project">What is SmoColi?</A></LI><LI><A href="/Team:ETH_Zurich/Overview/CircuitDesign">The Circuit</A></LI><LI><A href="/Team:ETH_Zurich/Overview/Informationprocessing">Information processing</A></LI><LI><A href="/Team:ETH_Zurich/Overview/Dialogue">Dialogue between Biology and Modeling</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Achievements">Achievements</A></LI></UL><LI><A href="/Team:ETH_Zurich/Achievements/Data_page">Data Page</A></LI><LI><A href="/Team:ETH_Zurich/Biology" class="dir">Biology</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Biology">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Biology/Detector">Network Elements</A></LI><LI><A href="/Team:ETH_Zurich/Biology/MolecularMechanism">Genetic Design</A></LI><LI><A href="/Team:ETH_Zurich/Biology/Cloning">Materials and Methods</A></LI><LI><A href="/Team:ETH_Zurich/Biology/Journal">Lab Journal</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Biology/Validation">Results</A></LI><LI><A href="/Team:ETH_Zurich/Process/Microfluidics/Overview" class="dir">Microfluidics</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Process/Microfluidics/Overview" class="dir">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Process/Microfluidics">Channel Design Development</A></LI><LI><A href="/Team:ETH_Zurich/Process/Validation">Final Design and Validation</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Process/Microfluidics/Proof">Proof of Concept</A></LI><LI><A href="/Team:ETH_Zurich/Modeling" class="dir">Modeling</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Modeling">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/SingleCell">Single-Cell Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Analysis">System Analysis</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Stochastic">Stochastic Analysis</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Microfluidics">Reaction-Diffusion Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Combined">Combined 3D Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Analytical_Approximation">Analytical Approximation</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Parameters">Parameters</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/References">References</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Modeling/Code">Code</A></LI><LI><A href="/Team:ETH_Zurich/Team/Human practices" class="dir">Human Practices</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Team/Human practices">Outreach</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Biology/Safety">Safety</A></LI><LI><A href="/Team:ETH_Zurich/Team" class="dir">Team</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Team">Team SmoColi</A></LI><LI><A href="/Team:ETH_Zurich/Team/Media">Photo Gallery</A></LI><LI><A href="/Team:ETH_Zurich/Team/Acknowledgements">Acknowledgements &amp; Contributions</A></LI><LI class="last"><A href="https://igem.org/Team.cgi?year=2011&amp;team_name=ETH_Zurich" target="_blank">Official Team Overview</A></LI><LI id="loginMenu"><A href="#" id="loginLink">Log In</A></LI><LI style="display:none;" id="editMenu"><A href="#" id="editLink" class="dir">Edit</A></LI><LI class="first"><A href="#" id="historyLink">History</A></LI><LI><A href="/Special:Upload" target="_blank">Upload File</A></LI><LI class="last"><A href="#" id="logoutLink">Log Out</A></LI><LI class="dir last" style="background-image: none; position: relative;"><FIELDSET><FORM method="post" id="searchForm" action=""><LABEL for="s">Search:</LABEL></FORM></FIELDSET></LI></DIV></DIV></DIV><DIV style="position: fixed; top: 37px; left: 0px; padding: 15px; width:220px; bottom: 0px; z-index: 600; overflow: auto;"><DIV style="width: 100%; text-align: center"><B><A href="#" onclick="document.getElementById('easteregg').style.display='block'" style="text-decoration:none;color:#000000;">Can you feel the smoke tonight?</A></B> </DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Information_processing"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Information processing</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Signal_Pre-Processing"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Signal Pre-Processing</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Signal_Detection"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Signal Detection</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Signal_Processing"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Signal Processing</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#I:_The_Band-Pass_Filter"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">I: The Band-Pass Filter</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#II:_The_Alarm_Signal"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">II: The Alarm Signal</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-7"><A href="#References"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE></DIV><DIV style="bottom: 0; left: 235px; overflow: auto; position: fixed; right: 0px; top: 32px;"><DIV style="margin-left: 20px; margin-right: 20px; padding-top: 20px;"><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="Information_processing"> Information processing </SPAN></H1><P>SmoColi processes a <B>gaseous</B> into a <B>visible signal</B>. At first the gaseous signal molecules will be dissolved in our reservoir with water. Out of this dissolution our signal molecule diffuses into the agarose forming a <B>spatial concentration gradient</B> out of spatial absolute value in the reservoir. The signal molecules were either imported actively or by diffusion into our SmoColi cells. These signal is in the cells <B>processed to a visible signal</B> via a signal cascade. The visible signal is either a spatially located green band or red signal.
</P><CENTER><B>Figure 1: Information processing of SmoColi,</B> click on the different processing steps to read more about them</CENTER></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="Signal_Pre-Processing">Signal Pre-Processing </SPAN></H1><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 2: Tube mapping an absolute acetaldehyde concentration to a spatial gradient.</B> Acetaldehyde is diffusing from a sample medium into the tube. The SmoColi cells inside the tube degrade acetaldehyde and, thus, counteract the diffusion, leading to a spatial steady state acetaldehyde gradient which is characteristic for the acetaldehyde concentration in the sample medium.</DIV></DIV></DIV><P>The first step for both designs is to map an <B>absolute acetaldehyde (xylene) concentration into a spatial signal</B> for further analysis. For this pre-processing step we constructed a flow-channel consisting of a tube filled with agarose and the SmoColi cells (see Figure 3), and which contains initially no acetaldehyde (xylene). The channel is then connected at one side to the sample medium for which the acetaldehyde (xylene) concentration should be measured, leading to acetaldehyde (xylene) diffusing from the sample medium into the tube.
</P><P>Acetaldehyde is naturally degraded in E. coli by the acetaldehyde dehydrogenase. For the second design we engineered SmoColi to also degrade xylene by integrating the upper TOL pathway into the cells. The degradation of acetaldehyde (xylene) counteracts its diffusion into the channel, leading to the establishment of a spatial steady state acetaldehyde (xylene) concentration gradient. The shape and hight of the gradient only depends on the cell density, the <B>degradation</B> rate per cell, the <B>diffusion</B> constant, and the acetaldehyde (xylene) concentration in the sample medium. Thus, by increasing or decreasing the acetaldehyde (xylene) concentration in the sample medium and keeping the other parameters unchanged, the gradient is shifted to the right or to the left. In the following we describe how we use this spatial shift of the gradient to detect and quantify the concentration of acetaldehyde (xylene) in the sample medium.
</P></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="Signal_Detection"> Signal Detection </SPAN></H1><P>Inside the tube, acetaldehyde (xylene) diffuses through the cellular membranes of the SmoColi cells and associates with its constitutively expressed receptor AlcR (XylR). Upon binding, the AlcR-acetaldehyde (XylR-xylene) complex can bind to its cognate operator, and inhibits (activates) the transcription of the genes under the control of the P<SUB>AlcR</SUB> (P<SUB>U</SUB>) promoter.
</P><P>Since for the further signal processing it was necessary to have a POPS signal proportional to the signaling molecule concentration, and since the acetaldehyde sensor inhibits transcription, we decided to include an additional unit to invert the transduced signal downstream of acetaldehyde (this element is not necessary for the xylene sensor): we put TetR under the transcriptional control of AlcR, and since TetR acts as a transcriptional inhibitor for downstream proteins, the transduced acetaldehyde signal has a similar qualitative input/output relation as the transduced xylene sensor without inverter.
</P></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="Signal_Processing"> Signal Processing </SPAN></H1><H2><SPAN class="mw-headline" id="I:_The_Band-Pass_Filter"> I: The Band-Pass Filter </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 3: Circuit operations for SmoColi exposed to low acetaldehyde concentrations</B> resulting in a GFP band. The cells are exposed, depending on their position in the channel, to different signaling molecule concentrations. The sizes of the clouds indicate the local acetaldehyde (lilac) and AHL (orange) concentrations, receptively. For each spatial position, interactions being not active and proteins being not expressed in the corresponding cells are indicated by being shaded away.</DIV></DIV></DIV><P>To realize the spatial green fluorescence band indicating the acetaldehyde (xylene) concentration (see Figure 4)), we implemented a signal processing unit, which transduces the incoming signal if and only if the signal is neither too strong nor too weak (band-pass filter <A href="#Ref1">[1]</A>):
</P><P>TetR represses (XylR enhances) the expression of the LacI<SUB>M1</SUB> repressor (codon-modified LacI) and the lambda repressor CI. Thus, high acetaldehyde (xylene) concentrations result in high cytoplasmic levels of CI and of LacI<SUB>M1</SUB>. Since LacI<SUB>M1</SUB> represses the expression of the green fluorescent protein (GFP), no green fluorescence signal can be detected for high concentrations. 
</P><P>Low acetaldehyde (xylene) concentrations result in low LacI<SUB>M1</SUB> and CI concentrations. Although in this case, LacI<SUB>M1</SUB> does not repress GFP production, GFP is transcriptional repressed by the non-codon modified LacI, which is expressed at low CI concentrations.  Only intermediate signals results in a moderate level of CI and LacI<SUB>M1</SUB>. The repression of CI is significantly higher than the one of  LacI<SUB>M1</SUB>. While moderate levels of CI effectively shuts off LacI expression, the LacI<SUB>M1</SUB> concentration is to low to repress GFP
</P><H2><SPAN class="mw-headline" id="II:_The_Alarm_Signal"> II: The Alarm Signal </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 4: Circuit operations for SmoColi exposed to high acetaldehyde concentrations</B> resulting in an alarm (red fluorescence) signal. For further information, see Figure 5.</DIV></DIV></DIV><P>To realize the alarm signal for toxic acetaldehyde (xylene) concentrations (see Figure 5), we put the N-Acyl homoserine lactone (AHL) Synthase LuxI under the transcriptional control of CI, which is thus expressed only at low acetaldehyde (xylene) concentrations. LuxI syntesizes the small signaling molecule AHL which can quickly diffuse through the cell membrane in the extracellular space. Due to its relatively high diffusion constant, AHL diffuses through the whole tube and, thus, is present even in cells where no AHL is produced. Inside a cell, AHL can bind to its constitutively expressed receptor protein LuxR, and the thus activated LuxR represses the transcription of a red fluorescent protein (RFP). The transcriptional activator LuxR is used as an repressor in that case. Therefore we used a constructed artificial lacZ promoter with the lux box between -10 and -35 region. Thus, if the acetaldehyde concentration is below a threshold at least for a large enough minority of the cells in the channel, RFP is repressed in all cells. Only if the acetaldehyde concentration is above the threshold for all cells, no AHL is present in the channel and RFP is produced. Thus, the alarm signal RFP can be used to signal critical and potentially dangerous acetaldehyde (xylene) concentrations.
</P></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="References"> References </SPAN></H1><P><SPAN id="Ref1">[1] <A href="http://www.nature.com/nature/journal/v434/n7037/full/nature03461.html" class="external text" rel="nofollow">Subhayu Basu, Yoram Gerchman1, Cynthia H. Collins, Frances H. Arnold &amp; Ron Weiss: <B>A synthetic multicellular system for programmed pattern formation</B>, Nature 2005, 434: 1130-11342</A></SPAN></P></DIV><DIV style="position: relative; height: 149px; text-align: center; clear: both; margin: 20px auto 20px auto;" class="boxWrap"><TABLE style="height: 100%; width: 100%; border-spacing: 0px; " cellspacing="0"><TBODY><TR style="font-weight: bold;"><TD style="text-align: center; border-right: 1px solid black;">Back to iGEM</TD><TD colspan="4" style="text-align: center;"><SPAN id="SponsorsFooter">Our Sponsors</SPAN></TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="visualClear"> </DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:ETH_Zurich/Overview/Informationprocessing">http://2011.igem.org/Team:ETH_Zurich/Overview/Informationprocessing</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETH_Zurich/Overview/Informationprocessing" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETH_Zurich/Overview/Informationprocessing" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETH_Zurich/Overview/Informationprocessing&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETH_Zurich/Overview/Informationprocessing&amp;oldid=259028" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>