<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETH_Zurich_Biology_Validation"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ETH_Zurich/Biology/Validation">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETH_Zurich/Biology/Validation&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Biology/Validation&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Biology/Validation&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/Biology/Validation" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETH Zurich/Biology/Validation</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV class="stickToTop" id="naviBox" style="text-align:center; line-height: 0px; font-size: 0px; z-index: 500;"><DIV style="position: relative; margin: 0 auto; display:inline-block; left: 0px; right: 0px;"><DIV align="center" id="naviMenu" style="left: 0px; right: 0px; margin: 0pt auto; text-align: left;"><UL class="dropdown dropdown-horizontal mainmenu"><LI class="first"><A href="/Team:ETH_Zurich">Home</A></LI><LI><A href="/Team:ETH_Zurich/Overview/Project" class="dir">Project <I>SmoColi</I></A></LI><LI class="first"><A href="/Team:ETH_Zurich/Overview/Project">What is SmoColi?</A></LI><LI><A href="/Team:ETH_Zurich/Overview/CircuitDesign">The Circuit</A></LI><LI><A href="/Team:ETH_Zurich/Overview/Informationprocessing">Information processing</A></LI><LI><A href="/Team:ETH_Zurich/Overview/Dialogue">Dialogue between Biology and Modeling</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Achievements">Achievements</A></LI></UL><LI><A href="/Team:ETH_Zurich/Achievements/Data_page">Data Page</A></LI><LI><A href="/Team:ETH_Zurich/Biology" class="dir">Biology</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Biology">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Biology/Detector">Network Elements</A></LI><LI><A href="/Team:ETH_Zurich/Biology/MolecularMechanism">Genetic Design</A></LI><LI><A href="/Team:ETH_Zurich/Biology/Cloning">Materials and Methods</A></LI><LI><A href="/Team:ETH_Zurich/Biology/Journal">Lab Journal</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Biology/Validation">Results</A></LI><LI><A href="/Team:ETH_Zurich/Process/Microfluidics/Overview" class="dir">Microfluidics</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Process/Microfluidics/Overview" class="dir">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Process/Microfluidics">Channel Design Development</A></LI><LI><A href="/Team:ETH_Zurich/Process/Validation">Final Design and Validation</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Process/Microfluidics/Proof">Proof of Concept</A></LI><LI><A href="/Team:ETH_Zurich/Modeling" class="dir">Modeling</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Modeling">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/SingleCell">Single-Cell Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Analysis">System Analysis</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Stochastic">Stochastic Analysis</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Microfluidics">Reaction-Diffusion Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Combined">Combined 3D Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Analytical_Approximation">Analytical Approximation</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Parameters">Parameters</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/References">References</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Modeling/Code">Code</A></LI><LI><A href="/Team:ETH_Zurich/Team/Human practices" class="dir">Human Practices</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Team/Human practices">Outreach</A></LI><LI class="last"><A href="/Team:ETH_Zurich/Biology/Safety">Safety</A></LI><LI><A href="/Team:ETH_Zurich/Team" class="dir">Team</A></LI><LI class="first"><A href="/Team:ETH_Zurich/Team">Team SmoColi</A></LI><LI><A href="/Team:ETH_Zurich/Team/Media">Photo Gallery</A></LI><LI><A href="/Team:ETH_Zurich/Team/Acknowledgements">Acknowledgements &amp; Contributions</A></LI><LI class="last"><A href="https://igem.org/Team.cgi?year=2011&amp;team_name=ETH_Zurich" target="_blank">Official Team Overview</A></LI><LI id="loginMenu"><A href="#" id="loginLink">Log In</A></LI><LI style="display:none;" id="editMenu"><A href="#" id="editLink" class="dir">Edit</A></LI><LI class="first"><A href="#" id="historyLink">History</A></LI><LI><A href="/Special:Upload" target="_blank">Upload File</A></LI><LI class="last"><A href="#" id="logoutLink">Log Out</A></LI><LI class="dir last" style="background-image: none; position: relative;"><FIELDSET><FORM method="post" id="searchForm" action=""><LABEL for="s">Search:</LABEL></FORM></FIELDSET></LI></DIV></DIV></DIV><DIV style="position: fixed; top: 37px; left: 0px; padding: 15px; width:220px; bottom: 0px; z-index: 600; overflow: auto;"><DIV style="width: 100%; text-align: center"><B><A href="#" onclick="document.getElementById('easteregg').style.display='block'" style="text-decoration:none;color:#000000;">Can you feel the smoke tonight?</A></B> </DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Experimental_results"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Experimental results</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#XylR_-_the_xylene_sensor"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">XylR - the xylene sensor</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Comparison_of_the_two_promoter_versions"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Comparison of the two promoter versions</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#BBa_K625003_characterization"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">BBa_K625003 characterization</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Dose_response_curve"><SPAN class="tocnumber">2.2.1</SPAN><SPAN class="toctext">Dose response curve</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-6"><A href="#AraC_-_the_arabinose_sensor"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">AraC - the arabinose sensor</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#AlcR-_the_acetaldehyde_sensor"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">AlcR- the acetaldehyde sensor</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Experiments_and_results_for_fungal_AlcR"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Experiments and results for fungal AlcR</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Bandpass-Characterization_of_lacIM1"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Bandpass-Characterization of lacI<SUB>M1</SUB></SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Alarm"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Alarm</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Copy_number_test"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Copy number test</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#References"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE></DIV><DIV style="bottom: 0; left: 235px; overflow: auto; position: fixed; right: 0px; top: 32px;"><DIV style="margin-left: 20px; margin-right: 20px; padding-top: 20px;"><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="Experimental_results"> Experimental results </SPAN></H1><P><B>For the 3 subparts of SmoColi we designed additional test systems to test the functionality of the single parts. Alike all new or improved biobricks were tested and characterized before they were implemented in the subparts.</B> For the sensors we tested the repressors or activators in the presence of the corresponding small molecule. For the P<SUB>u</SUB> promoter we show a<A href="/Team:ETH_Zurich/Biology/Validation#Results_of_BBa_K625003_characterization" title="Team:ETH Zurich/Biology/Validation"> dose response for xylene</A> over air. In the case of arabinose, by knocking out the transporter, we could also change the ON-/OFF character of AraC to a <A href="#AraC_-_the_arabinose_sensor">dose response for arabinose</A>. The fungal AlcR sensor however seems not to work as an network element in <I>E.coli</I>.
</P></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="XylR_-_the_xylene_sensor"> XylR - the xylene sensor </SPAN></H1><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 1: P<SUB>u</SUB> promoter</B></DIV></DIV></DIV><P>E. coli strain JM101 was transformed with two plasmids containing the transcriptional regulator XylR, the degradation cassette <I>xylMABN</I> and a GFP reporter coupled to <A href="http://partsregistry.org/Part:BBa_K625002" class="external text" rel="nofollow">BBa_K625002</A> respectively <A href="http://partsregistry.org/Part:BBa_K625003" class="external text" rel="nofollow">BBa_K625003</A>. <A href="http://partsregistry.org/Part:BBa_K625002" class="external text" rel="nofollow">BBa_K625002</A> is an improved version of the P<SUB>u</SUB> promoter <A href="http://partsregistry.org/Part:BBa_I723138" class="external text" rel="nofollow">BBa_I723138</A> were we inserted a stop codon to prevent fusion proteins. <A href="http://partsregistry.org/Part:BBa_K625003" class="external text" rel="nofollow">BBa_K625003</A> is a minimal version of the P<SUB>u</SUB> promoter, where the partial XylU sequence is deleted. The regulator and degradation pathway was provided by Sven Panke on the plasmid pCK04AxylR according to <A href="#Ref5">[5]</A>. The reporter plasmid was constructed by using <A href="http://partsregistry.org/Part:BBa_K625005" class="external text" rel="nofollow">BBa_K625005</A> as a backbone. 
</P><P>We inoculated 50 mL of LB medium with an overnight culture of the co-transformed strains and set the OD<SUB>600</SUB> to 0.1. Upon reaching the exponential growth phase, the cultures were induced with m-xylene. Therefore we put a sterile test tube with m-xylene  into the flask and tightly sealed it with parafilm in order to get an <B>air-induced response</B> (see Figure 4 below). This way of inducing was chosen due to the very high volatility of Xylene, when dissolved in water, which makes it impossible to set its concentration in an open system.
</P><P>OD<SUB>600</SUB> and GFP fluorescence was measured after different time periods in a 96-well plate. Afterwards the data was normalized in order to get more consistent results. 
</P><H2><SPAN class="mw-headline" id="Comparison_of_the_two_promoter_versions">Comparison of the two promoter versions</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 2:  Characterization of BBa_K625002 and BBa_K625003</B><I>m</I>-xylene was added by air-induction.</DIV></DIV></DIV><P>After 3 h we could observe a clear increase in GFP fluorescence when <I>m</I>-xylene was added in a test tube. Thus GFP production can be induced by <I>m</I>-xylene present from the air. The results of the experiments can be seen in Figure 8. By comparing <A href="http://partsregistry.org/Part:BBa_K625002" class="external text" rel="nofollow">BBa_K625002</A> and <A href="http://partsregistry.org/Part:BBa_K625003" class="external text" rel="nofollow">BBa_K625003</A>, we can see higher expression rates for the shortened promoter <A href="http://partsregistry.org/Part:BBa_K625003" class="external text" rel="nofollow">BBa_K625003</A> compared to the longer one, both in induced and noninduced state. The increase of promoter strength is thought to be some unknown mechanism of the deleted part of the promoter like mRNA secondary structure formation or a change of mRNA degradation rate.
Nevertheless, we could also see a higher level of induction in the setup with <A href="http://partsregistry.org/Part:BBa_K625003" class="external text" rel="nofollow">BBa_K625003</A> (around 7-fold) compared to the <A href="http://partsregistry.org/Part:BBa_K625002" class="external text" rel="nofollow">BBa_K625002</A> setup (around 5-fold). Thus both of the provided promoters can be used for transcriptional regulation with XylR with different strength.
</P><H2><SPAN class="mw-headline" id="BBa_K625003_characterization"><A href="http://partsregistry.org/Part:BBa_K625003" class="external text" rel="nofollow">BBa_K625003</A> characterization</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption"><B>Figure 4: Erlenmeyer flask with xylene induced GFP.</B></DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption"><B>Figure 3: Timecourse of GFP expression from <A href="http://partsregistry.org/Part:BBa_K625003" class="external text" rel="nofollow">BBa_K625003</A></B></DIV></DIV></DIV><P>The short version of the P<SUB>U</SUB> promoter was characterized in more detail. These experiments were performed in M9 medium containing 0.5 % glucose, in the hope of less leaky transcription through more defined medium. The extent of xylene induction was modulated by performing the induction over air not only with pure xylene, but also with mixtures of xylene and paraffin oil, which gives lower xylene concentration within the medium due to a lower partial pressure. Therefore a 50 ml shaking flask culture of JM101 containing the sensor plasmid PcK04AxylR and a medium copy plasmid with a sfGFP reporter under <A href="http://partsregistry.org/Part:BBa_K625003" class="external text" rel="nofollow">BBa_K625003</A> control was induced in exponential phase with different dilutions of xylene in paraffin oil. The specific fluorescence was measured in a platereader and dose-responsiveness could be shown for all timepoints (Figure 3), with the overnight one having the lowest detection limit. 
</P><P>These promising results made us <A href="/Team:ETH_Zurich/xylene" title="Team:ETH Zurich/xylene"> calculate</A> the xylene distribution in our thermodynamically closed system, to get a proper dose response for XylR. (The induction with 25 % xylene in paraffin oil in excess would for example correspond to a concentration in the medium of 775 μM, which means a responsiveness in the micromolar range, which is expected)
</P><P>
In another experiment (some shaking flasks containing the test tubes for induction shown in Figure 4), the dose response curve for the same cells was collected in the same way after overnight growth (Figure 5). Xylene was already detected in the lower micromolar range. For the highest induction rates in the low millimolar range, a certain toxicity was already observed, as can be seen by a decreased OD<SUB>600</SUB> (Figure 6). The promoter was found to be rather strong, as it showed expression rates of about a quarter of a <A href="http://partsregistry.org/Part:BBa_K180000" class="external text" rel="nofollow">Tac</A> promoter induced with 100μM IPTG (positive control) and its leakyness was found to be negligible (negative control). (see Figure 7)
</P><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 295px;"><DIV class="gallerytext"><P><B>Figure 5: Characterization of BBa_K625003</B> different concentration of m-xylene were added by air-induction. sfGFP expression could be induced by lower micromolar concentrations of xylene.
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 295px;"><DIV class="gallerytext"><P><B>Figure 6: Influence of xylene on the cell growth of E.coli.</B> Xylene shows toxic effects on <I>E.coli</I> in the lower millimolar range.
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 295px;"><DIV class="gallerytext"><P><B>Figure 7: Positive and negative control </B> in comparison to fully xylene-induced (+) and non induced (-) GFP expression by <A href="http://partsregistry.org/Part:BBa_K625003" class="external text" rel="nofollow">BBa_K625003</A>. (Positive control being fully induced Lac promoter, negative promoter the same strain without GFP)
</P></DIV></DIV></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Dose_response_curve">Dose response curve</SPAN></H3><P>We fitted the activation data in dependency of the xylene concentration to a Hill function. Since for a xylene concentration of 3100 μM the measured OD<SUB>600</SUB> value was significantly reduced compared to the other xylene concentrations, this data point was removed from the analysis (this concentration can be considered to be toxic for the cells). For a xylene concentration of 1550 μM the fluorescence signal was considered to have reached steady state (see Figure 5). Thus, we defined the relative activation for a given xylene concentration as the ratio between a given fluorescence signal and the fluorescence signal for a xylene concentration of 1550 μM (see Figure 8).
</P><P>To this data (see Figure 8) we fitted the Hill function
</P><P>
with K<SUB>D</SUB> the dissociation constant and n the Hill coefficient. We obtained optimal values of
</P><P>K<SUB>D</SUB>=144.1 μM≈144 μM;
</P><P>n=1.055≈1
</P><P>Since the Hill coefficient is close to 1, we can assume non-cooperative activation of XylR. This data was subsequently used to redefine the corresponding parameters for the <A href="/Team:ETH_Zurich/Modeling/SingleCell#Xylene_Sensor" title="Team:ETH Zurich/Modeling/SingleCell"> xylene model </A>.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"><B>Figure 8: Dose response curve of XylR activation depending on xylene.</B> A Hill type function (blue solid curve) was fitted to the measured data (black diamonds). The dotted black lines display the position of the K<SUB>D</SUB> value.</DIV></DIV></DIV></DIV></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="AraC_-_the_arabinose_sensor"> AraC - the arabinose sensor </SPAN></H1><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 9: Characterization of arabinose sensor for different concentration of arabinose. </B></DIV></DIV></DIV><P>The <I>E.coli</I> AraC sensor was used as a proof of concept input for our SmoColi system, as it is a well established expression system with an easily diffusing and easy to handle inducer. 
</P><P>To test the sensor, the <A href="/Team:ETH_Zurich/Biology/Detector#How_we_used_it_3" title="Team:ETH Zurich/Biology/Detector"> BW27783 </A><I>E.coli</I> strain (containing a genomic version of AraC) was transformed with a medium copy plasmid containing a sfGFP reporter under control of the P<SUB>BAD</SUB> promoter <A href="http://partsregistry.org/Part:BBa_I13453" class="external text" rel="nofollow">BBa_I13453</A>. During exponential phase, LB liquid cultures were induced with various amounts of arabinose. Specific fluorescence was measured after 2, 4 and 16 hours. The results shown in Figure 9 exhibited an interesting behaviour. The values after 2 hours are in perfect correspondence with the data from the original publication introducing the strain <A href="#Ref2">[2]</A>. For the datasets after 4 and 16 hours, the dynamic range of the fluorescent output is moved towards higher concentrations. At the moment we believe those effects to be due to nonlinear dynamics due to active arabinose import, which will be further explored by a modelling investigation.
</P></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="AlcR-_the_acetaldehyde_sensor"> AlcR- the acetaldehyde sensor </SPAN></H1><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 10: Design of the AlcR testsystem</B> AlcR gets produced upon addition of anhydrotetracycline. A 6x His-tag was added to the C-terminal end of AlcR to test the expression of <I>alcR</I>.</DIV></DIV></DIV><P>As described in the <A href="/Team:ETH_Zurich/Biology/Detector#AlcR_Promoter" title="Team:ETH Zurich/Biology/Detector"> Smoke detectors section</A> we designed two artificial AlcR-dependent promoters which get repressed upon binding of AlcR to acetaldehyde. For testing of our design we created a system with a sfGFP output in order to characterize the gene expression with different levels of AlcR production and acetaldehyde induction.
</P><P>The system works in the following way (see Figure 10): 
</P><P>A) In normal medium conditions the TetR transcription factor is produced and binds to its cognate promoter, thus inhibiting the expression of <I>alcR</I>. In this case no AlcR is present and GFP is produced. 
</P><P>B) Upon addition of increasing amounts of anhydrotetracycline, TetR gets released from the promoter and AlcR is produced. The amount of AlcR should increase with increasing amounts of anhydrotetracycline. 
</P><P>C) By addition of acetaldehyde AlcR binds to the alc-promoter and the GFP response decreases.
</P><H2><SPAN class="mw-headline" id="Experiments_and_results_for_fungal_AlcR">Experiments and results for fungal AlcR</SPAN></H2><P>In a first experiment, the <I>alc</I>-promoters were tested by using a non-codon optimized natural variant of <I>alcR</I>. Because of this we tested the compatibility of the codon usage in the variant from <I>Aspergillus nidulans</I> with the one from <I>E. coli</I>. We received a codon adaptation index (CAI) of 0.7 <A href="#Ref3">[3]</A>. Nevertheless, five rare arginine codons in <I>E.coli</I> were present within the first 20 amino acids of the gene. After a first experiment without any significant results (data not shown), the first three rare arginine codons of <I>alcR</I> were exchanged by PCR and a fully codon-optimized variant was ordered. All of the following experiments were performed with <I>alcR</I> codon-optimized within the first 20 amino acids.
</P><P>The AlcR test was established as a two plasmid system containing ampicillin and kanamycin resistances and pMB1 respectively pSC101 origins of replication. These plasmids were co-transformed into <I>E. coli</I> JM101 strain. M9 medium was inoculated with an overnight culture of the strain. The bacteria were induced with different amounts of acetaldehyde (0, 10, 1000 µM) and anhydrotetracycline (0, 1, 100 ng/mL) during the exponential  growth phase. In order to avoid evaporation of acetaldehyde, all equipment was cooled down to -20 degrees prior to preparation of the stock solutions. All solutions were afterwards prepared in tubes at 4 °C. Finally, the cell suspension was added to the solutions and the tubes were tightly closed to prevent loss of acetaldehyde. Fluorescence and OD<SUB>600</SUB> measurements were performed in a 96-well plate several hours after induction.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:392px;"><DIV class="thumbcaption"><B>Figure 11: Test results for <A href="/Team:ETH_Zurich/Biology/Detector#AlcR_Promoter" title="Team:ETH Zurich/Biology/Detector"> P<SUB>alc1</SUB></A></B></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:416px;"><DIV class="thumbcaption"><B>Figure 12: Test results for <A href="/Team:ETH_Zurich/Biology/Detector#AlcR_Promoter" title="Team:ETH Zurich/Biology/Detector"> P<SUB>alc2</SUB></A></B></DIV></DIV></DIV><P>As visible on the two images above, no significant change in GFP fluorescence could be observed neither between different acetaldehyde concentrations nor between different AlcR induction levels. Because of these negative results and the non codon-optimized <I>alcR</I>, we performed expression tests for the protein in <I>E. coli</I> strain JM101. In these experiments, gene expression was induced with different amounts of anhydrotetracycline during the exponential growth phase of the bacteria. After cultivation, cells were harvested and lysed either by using a Retsch mill or by lysozyme. After resuspension of the cell extract we performed both SDS-PAGE gels and western blots to check for <I>alcR</I> expression. Buffer was added to the cell debris and heated to 95 °C for 10 minutes in order to resolubilize potential inclusion bodies.
</P><P>AlcR has a molecular mass of about 96 kDa and thus its band was suspected to be on the same level as the red box marked in the SDS-PAGE gel (see Figure 13) <A href="#Ref4">[4]</A>. Despite increasing amounts of anhydrotetracycline (wells 1 and 3: 0 ng/mL, 2 and 4: 1 ng/mL,  3 and 6: 100 ng/mL), no change could be observed in this area of the gel for any of the probes. A reason for this could be that <I>alcR</I> is not expressed at all. 
</P><P>To check again whether <I>alcR</I> is expressed at all we performed a western blot. Therefore, a 6x His-tag was added to the C-terminal end of AlcR and we used an anti-His antibody kit in order to detect it. A probe with another concentrated 6x His-tagged protein was added as positive control so that we could exclude systematic errors during the blotting procedure (see Figure 14).  
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 13: SDS-PAGE performed with cell extract supernatant for AlcR expression testing</B> The red box indicates the level where a band of AlcR should be expected.</DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 14: Western blot of cell extract supernatant for AlcR expression testing</B> The visible bands are from a His-tagged positive control protein.</DIV></DIV></DIV></DIV><P>Because we could not show any expression of <I>alcR</I> at all we suppose that there is no AlcR produced in the cells, probably due to the usage of rare codons in the gene. Due to a long delay in the synthesis of the codon-optimized part, we were not able to test our system with this variant until now.
</P></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="Bandpass-Characterization_of_lacIM1"> Bandpass-Characterization of lacI<SUB>M1</SUB></SPAN></H1><DIV class="thumb tleft"><DIV class="thumbinner" style="width:392px;"><DIV class="thumbcaption"><B>Figure 15: Design for the LacI<SUB>M1</SUB> characterization</B></DIV></DIV></DIV><P>LacI<SUB>M1</SUB> (<A href="http://partsregistry.org/Part:BBa_K625000" class="external text" rel="nofollow">Bba_K625000</A>) is a LacI variant with the same protein sequence as the wild-type LacI, but with a completely codon-modified DNA sequence. <A href="#Ref6">[6]</A> This enables to use both sequences on two different plasmids (with two different promoters) without recombination happening between them, which is necessary for in our system to establish the bandpass filter. By entering this BioBrick into the registry, we wanted to check its functionality. Thus we engineered a test system for our BioBrick by adding a lac-dependent GFP reporter. A Tet promoter (P<SUB>Tet</SUB>) was put in front of <I>lacI<SUB>M1</SUB></I> and TetR constitutively produced, thus preventing <I>lacI<SUB>M1</SUB></I> from expression.
</P><P>In the non-induced state of the system, <I>lacI<SUB>M1</SUB></I> expression is repressed and the cells produce GFP.
If anhydrotetracycline is added, TetR is released from the Tet promoter (P<SUB>tet</SUB>) and LacI is produced. LacI inhibits gfp transcription and the fluorescence decreases. Addition of IPTG although leads to the opposite case: LacI gets inhibited and the fluorescence increases.
</P><P>The test system for <A href="http://partsregistry.org/Part:BBa_K625000" class="external text" rel="nofollow">BBa_K625000</A> was established as a single plasmid system. In the experiments we used the two different plasmids pSB1AK3 and pSB3C5 as backbones of the designed network. Experiments were performed in <I>E.coli</I> strain JM101, which was grown in M9 medium until exponential growth and then induced with increasing amounts of anhydrotetracyline (0 ng/mL, 1 ng/mL, 100 ng/mL) and IPTG (0 µM, 1 µM, 10 µM, 100 µM, 1000 µM). The cells were incubated in a 96-well plate at 37 °C and every 15 minutes OD<SUB>600</SUB> and GFP fluorescence was measured. 
</P><P><B>Results</B></P><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>pSB3C5</CAPTION></TABLE><P><B>Figure 16: time course of fluorescence</B> depending on different expression levels of the transcriptional regulator LacI<SUB>M1</SUB>. pSB3C5 was used as plasmid backbone for the test system.
</P><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>pSB1AK3</CAPTION></TABLE><P><B>Figure 17: time course of fluorescence</B> depending on different expression levels of the transcriptional regulator LacI<SUB>M1</SUB>. pSB1AK3 was used as plasmid backbone for the test system.
</P><P>
In general the test system for characterization of our BioBrick parts <A href="http://partsregistry.org/Part:BBa_K625000" class="external text" rel="nofollow">BBa_K625000</A> respectively <A href="http://partsregistry.org/Part:BBa_K625001" class="external text" rel="nofollow">BBa_K625001</A> worked as expected. In both system we can see that the fluorescence decreases with increasing amounts of anhydrotetracycline. This indicates that LacI<SUB>M1</SUB> is induced and thus represses the expression of <I>gfp</I>. By addition of IPTG LacI<SUB>M1</SUB> gets inhibited and therefore the GFP response increses. This tendency can be observed both in figure 16 and figure 17.
</P><P>At a concentration of 100 ng/mL anhydrotetracycline, only a slight increase in fluorescence can be observed for increasing IPTG concentrations. We are supposing that the expression of <I>lacI<SUB>M1</SUB></I> is fully induced at this level and thus the IPTG amount present in the system is not sufficient anymore to derepressing all of the LacI<SUB>M1</SUB>.
</P><P>In the case where no anhydrotetracycline is present in the medium, there can still be observed a change in fluorescence depending on the level of IPTG. The same pattern of induction can be observed here as in the weakly induced systems. The reason for this is likely due to the leaky expression of the <I>lacI<SUB>M1</SUB></I> gene, resulting in a small amount of LacI<SUB>M1</SUB> always present in the cells and thus inhibiting the GFP production.
</P><P>After about 300 min for pSB3C5 and about 200 min for pSB1AK3, the cells enter stationary phase and the flourescence intensity and OD<SUB>600</SUB> do not increase anymore.
</P></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="Alarm"> Alarm </SPAN></H1><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 18: Design of the quorum sensing based alarm system.</B></DIV></DIV></DIV><P>When finally having <A href="https://2011.igem.org/Team:ETH_Zurich/Biology/MolecularMechanism#Cloning_successes_and_problems" class="external text" rel="nofollow">finished cloning</A>  the complete system with arabinose as an inducer, we were really astonished that all colonies on the cotransformation of the three final plasmids turned red on standard LB agar plates, thus having a fully induced alarm signal. From the logic design, an alarm signal would only be expected at very high arabinose concentrations. This made us design test systems for the bandpass and the alarm separately, whereas here are the results for the latter one. 
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 19: Corepression by AHL of mCherry </B> driven by the <A href="https://2011.igem.org/Team:ETH_Zurich/Biology/Detector#How_we_used_it_6" class="external text" rel="nofollow">P<SUB>lux</SUB></A> promoter</DIV></DIV></DIV><P><I>E.coli</I> JM101 was transformed with a medium copy test plasmid containing the construct P<SUB>const</SUB>-<I>luxR</I>-terminator-P<SUB>lux</SUB>-<I>rfp</I>-terminator (The <A href="https://2011.igem.org/Team:ETH_Zurich/Biology/Detector#How_we_used_it_6" class="external text" rel="nofollow">P<SUB>lux</SUB></A> being an engineered version of the natural one, where the operon site for LuxR is placed between the -10 and -35 region of the promoter, rendering LuxR a repressor). After grown to exponential phase, they were corepressed with different concentrations of commercial AHL. Upon bindung of AHL to LuxR it is supposed to repress RFP production, thus this should be a test system for the two parts (P<SUB>const</SUB>-luxR and P<SUB>lux</SUB>-mCherry contained on the plasmid, which are also used in the final system.
</P><P>The corepression was performed with a wide range of AHL concentrations during the exponential growth phase, and specific fluorescence measured 5 hours later on a platereader. As can be seen in Figure 19, we could not find any response upon AHL sensing. Our hypothesis is that the engineered P<SUB>lux</SUB> is really sensitive on its surroundings and our construct with mCherry might impair with the regulation mechanism. This promoter has already previously raised intrigues, as nicely documented by the <A href="http://partsregistry.org/Part:BBa_R0061:Experience#iGEM_Chiba_2010" class="external text" rel="nofollow">2010 iGEM Chiba</A> team. This hyothesis will be tested by another experiment with a test construct containing the wild type Lux promoter around our P<SUB>const</SUB>-luxR construct.
</P></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="Copy_number_test"> Copy number test </SPAN></H1><P><B>Results for BBa_K625005</B></P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:212px;"><DIV class="thumbcaption"><B>Figure 20: Agarose gel</B> with 2-log-ladder loaded in in lane one, digests of triplicates of the EcoRI digests of pSB6A1 (lane 2-4) and pSB6A5 (lane5-7).</DIV></DIV></DIV><P><A href="http://partsregistry.org/Part:BBa_K625005" class="external text" rel="nofollow">pSB6A5</A>  is the version 5 BioBrick vector backbone of the medium copy (minimal pBR322 origin) pSB6A1 plasmid. This means, that the cassettes for the ORI and antibiotic resistance have been minimized (reduction of the size and transcriptional terminators flanking the prefix and the suffix have been added, to trancriptionally insulate the inserted parts from the vector backbone machinery and vice versa. This minimization yielded a reduction in size from 4022 bp to 2743 bp. 
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:212px;"><DIV class="thumbcaption"><B>Figure 21: Agarose gel</B> with 2-log-ladder loaded in in lane one, digests of triplicates of the EcoRI digests of pSB6A1 (lane 2-4) and pSB6A5 (lane5-7).</DIV></DIV></DIV><P>For comparison of the copy number of the plasmids, triplicates of OD normalized bacterial cultures containing the respective plasmids were miniprepped. The DNA concentration was then determined by an agarose DNA gel electrophoresis of EcoRI digests of the purified plasmids (Figure 21). This experiment showed a clear reduction in copy number of the minimized version of the original plasmid (Figure 20). Possible reasons for this could be deletion of some unknown regulatory sequences during minimization of the ORI or transcription from within the insert into the origin region in psB6A1 impacting the replication machinery.
</P></DIV><DIV class="boxWrap" style="margin: 20px auto 0px auto; padding: 15px;"><H1><SPAN class="mw-headline" id="References"> References </SPAN></H1><P><SPAN id="Ref1">[1] <A href="http://www.pnas.org/content/97/12/6640.full.pdf+html" class="external text" rel="nofollow">Kirill A. Datsenko and Barry L. Wanner: <B>One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products</B> 2000, Proc Natl Acad Sci USA, 97: 6640-5</A></SPAN></P><P><SPAN id="Ref2">[2] <A href="http://mic.sgmjournals.org/content/147/12/3241/F4.expansion.html" class="external text" rel="nofollow">A. Khlebnikov, K. Datsenko, T. Skaug, B. Wanner and J. Keasling: <B>Homogeneous expression of the P<SUB>BAD</SUB> promoter in Escherichia coli by constitutive expression of the low-affinity high-capacity AraE transporter,</B> Microbiology, 2001, (147): 3241–3247</A></SPAN></P><P><SPAN id="Ref3">[3] <A href="http://www.ncbi.nlm.nih.gov/pubmed/3072264" class="external text" rel="nofollow">Beatrice Felenbok et al. <B>The ethanol regulon in <I>Aspergillus nidulans</I>: characterization and sequence of the positive regulatory gene alcR Gene</B> 1988 ; 73(2): 385-96</A></SPAN></P><P><SPAN id="Ref4">[4] <A href="http://www.ncbi.nlm.nih.gov/pubmed/11550794" class="external text" rel="nofollow">Beatrice Felenbok, Michel Flipphi and Igor Nikolaev: <B>Ethanol Catabolism in <I>Aspergillus nidulans</I>: A Model System for Studying Gene Regulation, Progress in Nucleic Acid Research and Molecular Biology</B>, 69: 149-204</A></SPAN></P><P><SPAN id="Ref5">[5] <A href="http://aem.asm.org/cgi/content/abstract/64/2/748" class="external text" rel="nofollow">S. Panke, J. M. Sanchez Romero, V. de Lorenzo: <B>Engineering of Quasi-Natural Pseudomonas putida Strains for Toluene Metabolism through an ortho-Cleavage Degradation Pathway</B> , Applied and Environmental Microbiology, 1998, 64: 748-751</A></SPAN></P><P><SPAN id="Ref6">[6] <A href="http://www.nature.com/nature/journal/v434/n7037/full/nature03461.html" class="external text" rel="nofollow">Subhayu Basu, Yoram Gerchman1, Cynthia H. Collins, Frances H. Arnold &amp; Ron Weiss: <B>A synthetic multicellular system for programmed pattern formation</B>, Nature 2005, 434: 1130-11342</A></SPAN></P></DIV><DIV style="position: relative; height: 149px; text-align: center; clear: both; margin: 20px auto 20px auto;" class="boxWrap"><TABLE style="height: 100%; width: 100%; border-spacing: 0px; " cellspacing="0"><TBODY><TR style="font-weight: bold;"><TD style="text-align: center; border-right: 1px solid black;">Back to iGEM</TD><TD colspan="4" style="text-align: center;"><SPAN id="SponsorsFooter">Our Sponsors</SPAN></TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="visualClear"> </DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:ETH_Zurich/Biology/Validation">http://2011.igem.org/Team:ETH_Zurich/Biology/Validation</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETH_Zurich/Biology/Validation" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETH_Zurich/Biology/Validation" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETH_Zurich/Biology/Validation&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETH_Zurich/Biology/Validation&amp;oldid=261719" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>