<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_HAFS-Korea_Experiment skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:HAFS-Korea/Experiment</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="menuContainer"><UL><LI><A href="https://2015.igem.org/Team:HAFS-Korea">Home</A></LI><LI><A href="https://2015.igem.org/Team:HAFS-Korea/Tour">Tour</A></LI><LI><A href="https://2015.igem.org/Team:HAFS-Korea/Project">Project
            					</A></LI><LI><A href="https://2015.igem.org/Team:HAFS-Korea/Purpose">Purpose</A></LI><LI><A href="https://2015.igem.org/Team:HAFS-Korea/Design">Description</A></LI><LI><A href="https://2015.igem.org/Team:HAFS-Korea/Experiment">Experiment</A></LI><LI><A href="https://2015.igem.org/Team:HAFS-Korea/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:HAFS-Korea/Modeling">Modeling</A></LI></UL><LI><A href="https://2015.igem.org/Team:HAFS-Korea/Expansion">Expansion
            					</A></LI><LI><A href="https://2015.igem.org/Team:HAFS-Korea/Social">Social Implication</A></LI><LI><A href="https://2015.igem.org/Team:HAFS-Korea/Practices">Human Practice</A></LI><LI><A href="https://2015.igem.org/Team:HAFS-Korea/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:HAFS-Korea/Team">Team
            					</A></LI><LI><A href="https://2015.igem.org/Team:HAFS-Korea/Introduction">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:HAFS-Korea/Attributions">Attribution</A></LI><LI><A href="https://2015.igem.org/Team:HAFS-Korea/Collaborations">Collaboration</A></LI></DIV><DIV id="contentContainer"><H1>Experiment</H1><P>The overall experimental procedure goes like the following:
<OL><LI>Creating Construct 1 comprised of E.coli gene set (signal, lpp, and ompA genes)</LI><LI>Creating Construct 2 comprised of trichoderma reesei bgl I gene</LI><LI>Ligation of the two constructs</LI><LI>Transformation of the recombinant vector</LI><LI>Confirmation of gene expression using flag tag and his tag</LI></OL></P><H2>I. Creating a construct comprised of e.coli gene set (signal, lpp, and ompA genes)</H2><H4><UL><LI>ompA signal sequence</LI><LI>1st 9 amino acid sequence from lpp gene</LI><LI>46th~159th amino acid sequence from ompA gene</LI><LI>pET28a vector</LI><LI>sincere prayer (... regardless of whether you are a Buddhist, Christian, or Islam)</LI></UL></H4><H3>1. Preparation of E.coli</H3><OL><LI>The colony of E.coli is prepared using conventional laboratory methods-- bacteria culture in solid medium.</LI><LI>Before the experiment, E.coli must be incubated for 24 hours in the incubator to obtain sufficient amount of colony to ensure genomic extraction.</LI><LI>The incubator must be kept within proper environment for effective growth, including humidity, temperature, and acidity. </LI></OL><H3>2. Extraction of pET28a</H3><OL><LI>DNA extraction</LI><LI>
The vector that is used for carrying the three genes is pET28a. This vector is plasmid carried by E.coli that has certain sites for promoter, kanamycin-resistant gene, and the sequences that can be cut by the same enzymes used in cutting the genes of interest. This is extracted from the E.coli genome using DNA-miniprep. This technique first requires cell lysis and the following extraction of DNA by using spin column binding method. It effectively assays only the DNA from the cytoplasm based on the biochemical nature of nucleotide, whose phosphate has a minus charge due to its covalent bond structure. In order to make sure that sufficient amount of the plasmid is prepared for its use, 6 different samples are prepared.</LI><LI>b. Enzyme Cutting (Nco1)</LI><LI>
Once the plasmid is successfully extracted, cut the specific sequences using appropriate
enzymes, NcoI and Hind III in this case. When NcoI and Hind III are added to a 1.5ml microtube with the extracted plasmid sample and adequate butter, each enzyme cuts the base sequence in one location of the plasmid, thereby separating the splitted sequence into two sticky ends. The mixture is then incubated in the incubator for 3 hours.</LI></OL><H3>3. Extraction of the three genes of interest (signal, lpp, and ompA)</H3><OL><LI>Extraction of E.coli genome</LI><LI>
The previously grown E.coli is transferred to a liquid medium. Then the E.coli genome is also extracted through DNA-miniprep, same as above.</LI><LI>Enzyme Cutting (Nco1 and Hind III)</LI><LI>
Once the E.coli genome is successfully extracted, cut the specific sequences using Nco1 and Hind III, the same enzymes used for cutting pET28a. The genome is splitted into several DNA fragments that have specific sticky ends. Out of these fragments, extract the gene of interest through gel electroporosis. Since the number of the bases of the gene of interest is already known, the specific gene can be extracted by examining the specific bands that appear on the gel and cutting them out. This process should be repeated three times, for the three genes of interest, which in this case are the signal, lpp, and ompA gene.
(* Caution! Be careful not to touch the gel with bare hands! It is carcinogenic.)</LI><LI>PCR</LI><LI>
The copies of the three genes of interest are multiplied through PCR. Each gene of interest is mixed with a buffer and a specific primer, a short DNA fragment that is complementary with the single strand of denaturated DNA. This primer must pair with both Nco1 and Hind III, the enzymes used previously to cut the sticky ends of the genes, because the specific primer detects the specific sequence in the sticky end, thereby initiating the polymerase reaction. PCR is necessary in increasing the possibility of success in the subsequent DNA ligation.</LI><LI>DNA Ligation</LI><LI>
The three genes with the same sticky ends are then linked through DNA ligation, in the order of the signal gene, lpp gene, and ompA gene. This is where the biological nature of sticky ends comes in use -- when the two genes are jointed together in a right location, the single-stranded part of each sticky end binds to the complementary single-stranded part of another sticky end via hydrogen bonding. As A pairs with T and C pairs with T, the linkage between the two single strands allow to be jointed firmly. The bond between the adjacent nucleotides is formed by DNA ligase. The ligation process takes place in the 18C incubator for 1 hour, in a mixture of the three genes and DNA ligase. </LI></OL><H3>4. Ligation of pET28a and three genes of interest</H3><OL><LI>DNA Ligation</LI><LI>
The pET28a and the construct comprised of three genes of interest are ligated. Because they were both cut with Nco1 and Hind III prior to this stage, the ligation is able to take place. The procedure goes as above. This vector is not the final form of our project, since it consists of only the three genes that accounts for the production of lpp-ompA complex. It merely serves as a temporary storage for construct 1.</LI></OL><H2>II. Creating a construct comprised of trichoderma reesei bgl I gene</H2><H4><UL><LI>trichoderma reesei bgl I gene sequence</LI><LI>another prayer (accompanied with some sort of ritual -- trichoderma is very picky)</LI></UL></H4><H3> 1. Preparation of trichoderma reesei</H3><OL><LI>Colony of trichoderma reesei is prepared through conventional laboratory method.(24 celcius degrees, about 4-5 days, on potato agarose gel)</LI></OL><P>â†’ three different kinds of Trichoderma Reesei. They have different colors ! cute!
</P><H3>2. Extraction of bgl I gene</H3><OL><LI>Extraction of trichoderma reesei genome</LI><LI>
In order for the gene of interest to be extracted, the entire genome must be extracted by using genomic DNA prep kit for fungus. This is a process similar to the previous DNA mini-prep. Prepare as many samples as possible since extraction of fungus DNA is trickier than that of E.coli DNA. (It actually took five trials of experiment trying to extract the fungi genome.)</LI><LI>Enzyme Cutting (Hind III)</LI><LI>
The bgl I can only be obtained through cutting specific DNA sequence in the extracted genome. </LI><LI>The procedure of enzyme cutting follows in the same way. The both ends of bgl I have sequences cut by Hind III. </LI><LI>PCR</LI><LI>
The pure form of bgl I gene is amplified through PCR, using the same primer for Hind III. The ultimate product of this process is a mass of bgl I with sticky ends that can later attach to the sticky ends in Construct 1.</LI></OL><H2>III. Ligation of the two constructs</H2><OL><LI>Ligation</LI><LI>Enzyme Cutting (Hind III)</LI><LI>
The construct 1 designed previously -- pET28a vector with the three genes included -- is cut with Hind III. While the site previously treated with Nco1 remains intact, the site treated with Hind III is revealed again, splitting into two sticky ends. The bgl I is already in the form of sticky end by Hind III, so it does not need to be cut again.</LI><LI>Ligation</LI><LI>
Once the two parts of the final vector is both treated with Hind III, thus having same sticky ends, ligation can be undertaken. The sticky ends, one from E.coli genes and the other from bgl1, are joined together through both automatic and voluntary forces: the hydrogen bonding nature of complementary base pairs and the linkage created by DNA ligase. When the procedure mentioned above is followed, the final product of the ligation is one vector that contains both the construct 1 and construct 2.</LI></OL><H2>IV. Transformation of the recombinant vector</H2><OL><LI>Transformation</LI><LI>Addition of the vector into the media</LI><LI>
This process involves granting E.coli an opportunity to take up the plasmid. The final pET28a samples are first mixed with competent cells; the type in the experiment was dh 5 alpha. Then the cells and vectors are spreaded through solid LB media, which is also prepared in several plates to ensure success. As E.coli grow, they uptake the vector through one of their methods of incorporating foreign DNA molecule into their genome: transformation. Since this process is both non-selective and arbitrary in nature, it is very important to make different samples (This is the part that requires you to become a devout believer). Many trials of experiment were carried out with the testing of positive, negative, and transformation.</LI></OL><H2>V. Confirmation of gene expression using flag tag and his tag</H2><OL><LI>Using flag tag and his tag</LI><LI>
Once pET28a is successfully transformed into DNA of E.coli, an index marker that verifies the success in the expression of lpp-ompA complex and cellulase is needed. Two types of index markers, flag tag and his tag, are already included in the vector and therefore will express themselves in E.coli. As a type of chromatography, their usage is to purify proteins because they are expressed as specific antibodies on the cellular membrane of E.coli, along with the lpp-ompA complex. By detecting a small, specific part of their antigens called epitopes, those antibodies can attach to specific proteins. Therefore, as E.coli are placed in a container filled with protein mixture and are allowed to pass through, the protein mixture will lack of certain protein that is attached to the antibody. After electroporosis of the remaining mixture, it is determined whether the lpp-ompA complex and cellulase has been successfully expressed.</LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>