<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCLouvain_Experimental skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCLouvain/Experimental</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>&quot;</P><DIV class="container-fluid"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2016.igem.org/Team:UCLouvain">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-hashpopup="true" aria-expanded="false">About Us  </A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/Sponsors">Sponsors</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-hashpopup="true" aria-expanded="false">Project  </A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/Context">Context</A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/Background">Background</A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/Experimental">Experimental Approach</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-hashpopup="true" aria-expanded="false">Lab Work </A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/Homemade">Homemade product</A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/Perspectives">Perspectives</A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/Parts">Parts</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-hashpopup="true" aria-expanded="false">Human practices </A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/Story">Our story</A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/Media">In the media</A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/EE16">EE16</A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:UCLouvain/Medals">Medals</A></LI></DIV><ARTICLE><DIV class="container"><DIV class="row"><H2>Experimental Approach</H2><P>Based on the previously mentioned informations, we built a strong experimental approach that would allow us to create not only leaky mutants but regulable leaky mutants. We first generated porin mutant libraries and then we used a selection method to select the mutants of interest.</P></DIV><DIV class="row"><DIV class="col-sm-6"><H3>Mutant library construction</H3><P>Mutagenesis tools are a very efficient mean to generate mutant proteins with new or optimized properties and functions. Not only these methods can help you probe through a construct for structurally or functionally essential residues and thereby help the understanding of proteins but it also is a powerful engineering tool. Through a proper experimental design it can generate mutant proteins with improved, optimized or novel properties and allow any ambitious synthetic biologists to mess around with protein engineering.</P><P>In our particular case, we wanted to show the rapid emergence of a new function coupled with a regulatory system through a strong but simple experimental design. And for that our lab had the perfect tool to bring to iGEM. Researcher from the Institute of Life Sciences at UCLouvain have recently developed a new method to build plasmid libraries by seamless cloning of degenerated oligonucleotides, called QuickLib (Galka et al., submitted).</P><P>As you can see in figure 1, the QuickLib method is a simple combination of the use of degenerated oligonucleotides to induce randomized mutation through PCR and the Gibson assembly method (Galka et al, sumbitted). We first generate linear fragment with a targeted randomized region and overlapping ends through PCR. Than those linear fragments are circularized back to a plasmid form thanks to the hybridization of the overlapping region and the combined action of the exonuclease, DNA polymerase and DNA ligase as described by Gibson et al. (2009).</P><P>We strongly believed that this new tool can find great purpose in synthetic biology as it allows to quickly and easily create synthetic plasmid libraries with high diversity. By including smart use of random mutagenesis and this tool in our experimental design we believed we could outmatch rational design throughput.</P></DIV><DIV class="col-sm-6"><P>Figure 1: QuickLib Method : A. PCR with oligonucleotides designed to generate an overlapping region and a targeted degenerated region. B. Gibson Assembly. In red on the figure: degenerated region; in blue: overlapping regions. (Galka et al., submitted)</P></DIV></DIV><DIV class="row"><DIV class="col-sm-6"><H3>SEMI-RATIONAL TARGETED MUTAGENESIS METHOD:</H3><P>A fully randomized mutagenesis approach certainly allows to englobe every single possible mutation and possibly every single mutation combination but going through such a gigantic library would be time consuming and not very efficient. Furthermore such a library would include a very high number of uninteresting mutants. That is why we decided to target mutations to regions in our protein that were previously identified (Spagnolo et al., 2010) (cf. <A href="https://2016.igem.org/Team:UCLouvain/Background">background</A>) as playing a role in the hermeticity of the gating portion of the protein.</P><P>We not only wanted to create leaky porins but regulable leaky porins and for maximum effectiveness we thought that our mutagenesis design would have to encompass this regulatory objective. As shown by Mathonet et al. (2006), we decided that the easiest way to achieve regulation would be to use the histidine affinity for transition metal ions, such as Ni<SUP>2+</SUP> or Zn<SUP>2+</SUP>, and the potential conformational effect the binding of the two together has on folded proteins. Those conformational changes could be a great mean to open and close our gates and thus create a regulatory system. With all this in mind, we decided not only to target certain region but to limit the degeneracy and favor histidine content in those targeted regions.</P><P>Two Gating regions were identified by Spagnulo et al. (2010) and we decided to design two primers over each of these regions. One lightly degenerated allowing only the wild type codon, histidine codons and the intermediary codons. The other one, heavily degenerated, includes any codons but STOP codons. We were expecting higher viability of the proteins with the lightly degenerated primers but the heavily degenerated primers create libraries with higher diversity. We thus build a total of four libraries with two oligonucleotides designed over each gating region as shown the figure 2.</P></DIV><DIV class="col-sm-6"><P>Figure 2: Library Construction Model</P></DIV></DIV><DIV class="row"><P>Our final design represented 65 536 amino acid combination, ranging from the wild type sequence to a sequence including only histidine codons for the lightly degenerated primers and 67 108 864 amino acid combination, ranging again from the wild type sequence to a sequence including only histidine codons for heavily degenerated primers. The figure 3 illustrates the effective design of lightly degenerated primer for our first library (Library 1 on figure 2).</P><P>In comparison to a fully randomized mutagenesis approach, our “semi-rational targeted mutagenesis approach”, as we decided to call it, should proportionally yield a lot more valuable mutants.
We haven’t seen any project using this kind of approach and we wanted to show that, even in a short amount of time, random mutagenesis can be a wonderful tool for synthetic biologists. It is, as we found, a very interesting way to work on protein design problems.</P><P>Once the libraries were created, all we needed was a strong selection design and the time to reduce our diversity down to a few mutants that would fit our needs.</P></DIV><P style="text-align:center">Figure 3: Design of the degenerated primers</P><DIV class="row"><DIV class="col-sm-6"><H3>Selection method</H3><P>Our selection method was based on the work of Spagnuolo et al. (2010) as explained in the <A href="https://2016.igem.org/Team:UCLouvain/Background">background</A> section of this wiki. Its particularity resides in the fact that through two different selective growth media we’re able to select the open phenotype on one hand and the closed one on the other hand.</P><P>Selection of the open phenotype is achieved with a minimal media, containing maltose sugars of polymerization degree 4 through 6 as sole carbon source in a ΔLamB strain of E. coli.</P><P>Selection of the closed phenotype on the other hand is achieved on regular LB Growth medium with vancomycin.</P><P>To select an inducible opening or closing of the porin, we compared viability on the two previously mentioned media in presence and absence of our transition metal ions (Zn<SUP>2+</SUP> and Ni<SUP>2+</SUP>). If, on the same medium, a mutant survives in presence of the ions and dies in its absence or vice-versa, it possibly contains a porin for which the gate opening or closing is regulated by Zn<SUP>2+</SUP> or Ni<SUP>2+</SUP>. The mutants selected through this step should be further investigated and characterized. </P><P>These successive steps of selection are illustrated on figure 4.</P></DIV><DIV class="col-sm-6"><P>Figure 4: Selection Process</P></DIV></DIV><DIV class="row"><P>Here is our selection process in a table form:</P><TABLE><TBODY><TR><TH>1st selection</TH><TH>2nd selection</TH><TH>Result after the two selections</TH></TR><TR><TD>Restrictive medium
                          + Maltodextrin
                          + Zn</TD><TD>Nutritive medium
                          Vancomycin</TD><TD rowspan="2">Opening induction of the gate with Zn</TD></TR><TR><TD>Nutritive medium
                          + Vancomycin</TD><TD>Restrictive medium
                          + Maltodextrin
                          + Zn</TD></TR><TR><TD>Restrictive medium
                              + Maltodextrin
                              + Ni</TD><TD>Nutritive medium
                              + Vancomycin</TD><TD rowspan="2">Opening induction of the gate with Ni</TD></TR><TR><TD>Nutritive medium
                              + Vancomycin</TD><TD>Restrictive medium
                              + Maltodextrin
                              + Ni</TD></TR><TR><TD>Restrictive medium
                              + Maltodextrin</TD><TD>Nutritive medium
                              + Vancomycin
                              + Zn</TD><TD rowspan="2">Closing induction of the gate with Zn</TD></TR><TR><TD>Nutritive medium
                              + Vancomycin
                              + Zn</TD><TD>Restrictive medium
                              + Maltodextrin</TD></TR><TR><TD>Restrictive medium
                              + Maltodextrin</TD><TD>Nutritive medium
                              + Vancomycin
                              + Ni</TD><TD rowspan="2">Closing induction of the gate with Ni</TD></TR><TR><TD>Nutritive medium
                              + Vancomycin
                              + Ni</TD><TD>Restrictive medium
                              + Maltodextrin</TD></TR></TBODY></TABLE><P>After obtaining and isolating the open/closed regulated pIV porins, we would have to test the secretion of a selection of easily detectable proteins present in the periplasm but not able to go through the second membrane of E. coli. If those proteins can be detected in the extracellular environment only in the presence of such a mutant in the right ion conditions than we achieved our goal. A new verified biobrick could be proposed as a universal aspecific regulable secretion tool.</P><H3>Bibliography</H3><P>Galka, P., Jamez, E., Joachim, G., and Soumillion, P. Building plasmid libraries by seamless cloning of degenerated oligonucleotides. Submitted for publication.</P><P>Gibson, D.G., Young, L., Chuang, R.Y., Venter, J.C., Hutchison, C.A., Smith, H.O. (2009). &quot;Enzymatic assembly of DNA molecules up to several hundred kilobases&quot;. Nature Methods. 6 (5): 343–345.</P><P>Mathonet P, Barrios H, Soumillion P, Fastrez J. (2006). Selection of allosteric β-lactamase mutants featuring an activity regulation by transition metal ions. Protein Science,15(10):2335-43.</P><P>Spagnuolo, J., Opalka, N., Wen, W. X., Gagic, D., Chabaud, E., Bellini, P., Rakonjac, J. (2010). Identification of the gate regions in the primary structure of the secretin pIV. Molecular Microbiology, 76(1), 133–150. doi:10.1111/j.1365-2958.2010.07085.x</P></DIV></DIV></ARTICLE>

&quot;
    </DIV></DIV></DIV></DIV></DIV></BODY></HTML>