Team:Bielefeld-CeBiTec/Description
Toggle navigation
NavBar
PROJECT
Description
Library
Overview
Scaffolds
Design and Construction
Create your own Library
Mutation
Overview
Error prone Polymerase I
Genome wide mutator
Selection
Overview
Bacterial Two-Hybrid System
Alternative Selection System
Modeling
Overview
Model I
Model II
RESULTS
Overview
Library
Overview
Assembly
Phagemid Display
Sequencing
Mutation
Overview
Assembly
Reversion
Sequencing
Selection
Overview
Assembly
Expression Control
Interaction Control
Binding Control
in vivo Selection
Modeling
Fermentation
Parts
Best Basic Part
Best Composite Part
Best Part Collection
Improve a part
Proof of Concept
Demonstrate your Work
Medal Fulfillment
HUMAN PRACTICES
Overview
Engagement
Dictionary
Educational Outreach
Crowdfunding and Press
Entrepreneurship
Integrated Practices
Achievements
COLLABORATIONS
Overview
Düsseldorf
Lethbridge
Freiburg
NOTEBOOK
Overview
Lab Notebook
Library
Mutation
Selection
Primers
Lab Protocols
Media and Buffers
TEAM
Team Profile
Members
Bielefeld University
Contact
ACKNOWLEDGEMENT
Overview
Sponsors and Partners
Attributions
Project Description
The only way to do great work is to love what you do - Steve Jobs
Motivation
As long as mankind remembers, different diseases struck from time to time and demanded millions of lives.              Maybe the most fatal of these epidemic outbreaks was the 1918 flu pandemic, which killed between 50-100 million people worldwide.             Derived from a simple influenza virus only a few mutations were necessary to change this virus into one of the deadliest threats              that ever existed. Pandemic viral infections like this were the reason for our project selection. Our highest             motivation was to create a system to counteract these             extremely high risk potential slumbering in commonly known and seemingly not to dangerous viruses. Of course the influenza virus              stated 1918 a much higher risk than it does today. But still viruses are very hard to treat and especially members of the family flaviviridae like Zika or Dengue virus, show a high risk potential because of their high mutation rate.
We found a way to create in a short period of time antibody-like proteins.              Evobodies are binding             proteins that are able to be quickly adapted to altered targets like viral hull proteins and              re-establish binding properties in extremely short periods of time             The limiting factors in this process are the rate at which mutations happen in the gene of the Evobody              and screening of different proteins. This is the reason why we wanted to build a mutation inducing system             which is not only able to change basepairs in vivo at a very high frequency,              but is also specific enough to provide stability in culture             and does not interfere too much with growth properties of the individual cell.
The revolutionary part of Evobody generation is the combination of an in vivo mutagenesis             and a selection system which is also capable of screening our mutants during the process of              cell cultivation. Due to our constant interaction of altered binding proteins and the in vivo selection, we were eager to increase the mutation rate while retaining normal growth conditions. That is             why we did not only calculate the mutation rate of our two different mutagenesis approaches,              but also determined the growth rate under             different conditions.
Generation of binding proteins by directed evolution
This year the iGEM Team Bielefeld-CeBiTec aims to create a method for generating synthetic binding proteins,     our so-called Evobodies. This works by creating a library of binding proteins and increasing their affinity     towards a target by directed evolution (Figure 1). As a starting point, we randomize the binding regions of     synthetic antibody-like proteins (Figure 1A). Following we screen this library for affinity towards a target by     using a bacterial two-hybrid system (Figure 1B). To further increase the Evobodies affinity, we combine the selection     via the two-hybrid system with an in vivo mutagenesis system (Figure 1C). Doing this we hope to generate strong     and specific binding proteins by combining the powerful genetics of E. coli with the biological idea of     antibody generation and maturation in vertebrates.
Figure 1: Schematic Evobody generation process. A library of synthetic binding proteins (A) is screened by a bacterial two hybrid(B) for interaction with a traget protein. Existing interaction between both proteins leads to expression of a resistance gene thus granting the individual cell a selection advantage. After screening the library all suviving clones binding proteins are mutagenized in vivo (C) which results in the generation of new binding properties. Continuous selection cycles of selection and mutation increase the Evobodies binding affinity towards high affine binding proteins.
We designed our Evobody approach as an alternative to conventional methods for the generation of binding proteins.     In our vision it should be possible to clone each protein encoding sequence into one of our plasmids, let our system     do the work and get a high affinity binding protein, which can be either used for medical, diagnostic or scientific applications.
The starting point - synthetic binding protein library
As starting point, we want to create a library of many binding proteins with a high chance to contain a protein     with the potential to bind our target protein. In doing so we choose the core region of the antibody-mimetic mono-     and nanobodies. In the coding region of those proteins, we randomized the loop regions, which are known to bind     other proteins to obtain our library.
The randomization strategy as well as the choice of the protein scaffold is a key part of library generation.     We identified amino acids, which are present in most protein-protein interaction areas and created a randomization     scheme so that only these amino acids are encoded in the antibody-mimetics binding region. Read more about our     library design here .
Figure 2: Library of initial binding proteins. Expression of the initial binding proteins with variable regions highlighted in seperate colors (turquoise, orange, white, green, pink and blue) and constant regions in olive. The theoretical variability for each scaffold is 1.073.741.824.
Survival of the fittest - bacterial two-hybrid
In the next step, the binding protein library should be screened for proteins with an innate affinity for our target.     We want to realize this by using a bacterial two-hybrid system (Figure 3). Therefore, our target protein (1) is fused to a DNA     binding domain (2), which localises upstream of a reporter cassette (3). The binding protein (4) is fused to a RNA polymerase     subunit (5). Interaction between the binding and the target protein leads to recruitment of the RNA polymerase to the promoter     region of the reporter cassette and activates the reporter gene expression. By using an antibiotic resistance as a reporter     gene the output of the  bacterial two hybrid system should lead to the survival E. coli cells carrying a good binding     protein and the death of all cells with a bad binding protein.
Figure 3: Bacterial-two hybrid system.Interaction between the binding protein (4) and the target protein (1)       lead to recruitment of RNA polymerase (5) to the promoter upstream of the reporter cassette (3) and subsequent       expression of the reporter gene. In this case the reporter is beta-lactamase, which expression leads to degradation       of ampicillin (blue squares) and survival of the bacteria.
From the two-hybrid system we expect foremost a selection of the binding protein library. Furthermore, we expect a correlation     between binding - target protein affinity and the activated gene expression strength of the reporter. By using an antibiotic     resistance protein as a reporter we predict increased levels of the resistance protein inside a cell with a high affinity     binding protein. The outcome of this should be an increase of individual fitness for bacteria with good binding proteins,     which should lead to a higher growth rate under strong selective pressure. The complete two-hybrid system should cumulate in a     correlation between binding protein affinity and bacterial growth rate, which should lead to selection of a few bacteria with     strong binding proteins. Find out more on our selection subpage .
Accessing the sequence space - in vivo mutagenesis
After selection of the bulk of our library we will increase the affinity of our Evobodies in a process similar to the affinity     maturation of antibodies ( Teng und Papavasiliou 2007 ). As     addressed above we will select our Evobodies by increasing the selection pressure. At the same time, we will use an in vivo mutagenesis system. Thereby ,we can increase the sequence diversity beyond the limits of our library. Slightly modifications of      binding proteins identified during the initial selection will are the basis for the directed  evolution.
Figure 4: In vivo mutagenesis system. By using an in vivo mutagenesis system a single Evobody coding       sequence can be evolved to various different variants, each with a unique binding site. The single starting sequence is       replicated during growth and thereby mutations are incorporated either through error-prone polymerase I or a combination       of global mutator genes. The process results in the creating of the library of binding proteins with different binding properties.
In detail, we will compare two different possibilities to diversify our binding proteins. The first approach is the use of an     error-prone polymerase I in an otherwise Pol I temperature-sensitive E. coli strain.     ( Camps et al. 2003 )     Growth at a non-permissive temperature should result in accumulation of mutations in the part of the genom maintained by DNA polymerase I.     The interesting idea behind this approach is the fact that large parts of plasmids carrying an origin of replication from the     ColE1-familiy are replicated by the polymerase I. ( Camps et al. 2003 ; Camps 2010 )     Because of this, the usage of the error-prone polymerase I should mutant mainly our Evobody sequence on a plasmid. Thereby off-target     mutations, which are a major obstacle of in vivo mutagenesis, should be minimized. ( Camps et al. 2003 )
Our other approach is based on creating a plasmid borne hypermutator system by modulating the E. coli DNA fidelity systems.     ( Badran und Liu 2015 ) We will express known mutator genes under tight regulation from a plasmid. By using a plasmid borne mutator     system, in contrast to the more classical approaches of incorporating the mutator genes directly inside the genom.     ( Agilent Technologies ; Greener et al. 1997 )      Thereby we want to circumvent the known problems with globally increased mutation rate, which are genetic instability or general unviability.
Over the course of our project we want to find out which mutagenesis system is most suitable to our directed evolution approach.     Therefore we will compare both possibilities in terms of mutagenesis rate, -spectrum, -controllability and -specifity. How will     we do this? Find out on our mutation mainpage.
Library
Mutation
Selection
Modeling
Results
Improve a part
Mutator gene dnaQ926 - BBa_K1333108
We used iGEM SYSU-China 2014's dnaQ926 ( BBa_K1333108 ) in our genome wide mutator.
See here or go directly to the partsreg to see how we contribute to BBa_K1333108 characterization.
Improve a part
Literature
Agilent Technologies: XL1-Red Competent Cells. Instruction Manual 2015.
Badran, Ahmed H.; Liu, David R. (2015): Development of potent in vivo mutagenesis plasmids with broad mutational spectra. In: Nature communications 6, S. 8425. DOI: 10.1038/ncomms9425.
Camps, Manel (2010): Modulation of ColE1-like plasmid replication for recombinant gene expression. In: Recent patents on DNA & gene sequences 4 (1), S. 58–73.
Camps, Manel; Naukkarinen, Jussi; Johnson, Ben P.; Loeb, Lawrence A. (2003): Targeted gene evolution in Escherichia coli using a highly error-prone DNA polymerase I. In: Proceedings of the National Academy of Sciences of the United States of America 100 (17), S. 9727–9732. DOI: 10.1073/pnas.1333928100.
Fijalkowska, I. J.; Schaaper, R. M. (1996): Mutants in the Exo I motif of Escherichia coli dnaQ: defective proofreading and inviability due to error catastrophe. In: Proceedings of the National Academy of Sciences of the United States of America 93 (7), S. 2856–2861.
Greener, A.; Callahan, M.; Jerpseth, B. (1997): An efficient random mutagenesis technique using an E. coli mutator strain. In: Molecular biotechnology 7 (2), S. 189–195. DOI: 10.1007/BF02761755.
Teng, Grace; Papavasiliou, F. Nina (2007): Immunoglobulin somatic hypermutation. In: Annual review of genetics 41, S. 107–120. DOI: 10.1146/annurev.genet.41.110306.130340.
