Team:Bielefeld-CeBiTec/Results/Mutation/Assembly
Toggle navigation
NavBar
PROJECT
Description
Library
Overview
Scaffolds
Design and Construction
Create your own Library
Mutation
Overview
Error prone Polymerase I
Genome wide mutator
Selection
Overview
Bacterial Two-Hybrid System
Alternative Selection System
Modeling
Overview
Model I
Model II
RESULTS
Overview
Library
Overview
Assembly
Phagemid Display
Sequencing
Mutation
Overview
Assembly
Reversion
Sequencing
Selection
Overview
Assembly
Expression Control
Interaction Control
Binding Control
in vivo Selection
Modeling
Fermentation
Parts
Best Basic Part
Best Composite Part
Best Part Collection
Improve a part
Proof of Concept
Demonstrate your Work
Medal Fulfillment
HUMAN PRACTICES
Overview
Engagement
Dictionary
Educational Outreach
Crowdfunding and Press
Entrepreneurship
Integrated Practices
Achievements
COLLABORATIONS
Overview
Düsseldorf
Lethbridge
Freiburg
NOTEBOOK
Overview
Lab Notebook
Library
Mutation
Selection
Primers
Lab Protocols
Media and Buffers
TEAM
Team Profile
Members
Bielefeld University
Contact
ACKNOWLEDGEMENT
Overview
Sponsors and Partners
Attributions
Results basic parts & assembly of the mutators
Overview
We want to construct parts for in vivo mutagensis and assemble them to two mutators systems, which pose the main    driving force for our ongoing Evobody evolution system.
In this article we will sum up our design and construction of the mutator constructs used for in vivo characterization by reversion assays and high-throughput sequencing as well as the associated reporter systems.
Promoter
Error prone polymerase I
Genome wide mutator
Reporter
Promoter
We started by choosing the araC-PBAD regulatory unit from BBa_K808000 . This promoter is known for low basal expression and is therfore the     perfect regulatory unit for the expression of potentially toxic mutator genes. Furthermore we wanted to modify this promoter to     yield an even lower basal activity.    Therefore, we screened the synthetic PBAD library from iGEM DTU-Denmark 2013 and chose the promoter from Col.13. Advantages of this promoter    are the very low basal activity while still maintaining a high activity when induced.
We obtained our modified PBAD BBa_K2082112 through site-directed mutagenesis of BBa_K808000 .
Figure 1: Alignment between promoter sequence from BBa_K808000 (wt-PBAD )       and the modified PBAD sequence.
In addition to the low basal activity, the PBAD has the advantage of tuneable experssion strength, that is controlled by addition of different amounts of arabinose.    We estimated that controllable amounts of mutagenic proteins     would also result in different amount of mutagenesis, thus enabling a tuneable mutagenesis.
In addition to the titrable promoter activity by adding arabinose PBAD can be repressed by adding glucose. Glucose decreases intracellular     3',5'-cyclic AMP concentration, thereby lowering expression of PBAD promoter (Miyada et al. 1984).
We characterized our modified PBAD BBa_K2082112 by adding the RFP expression system BBa_K516032 downstream and measurement of RFP after different amounts     of time and with various amounts of arabinose added. The measurement was performed in the Top10 strain, which is deficient for arabinose metabolism (araD139 Δ(ara-leu)7697).
Figure 3: Measured expression of BBa_K2082112 in comparison to BBa_K808000 . RFP expression via BBa_K516032 was used for promoter characterization, and the promoter was induced by addition of arabinose. After 300 min RFP fluorescence was normalized       on OD600.
No difference in leakiness could be observed. The addition of 20 mM glucose represses the promoter slightly and decreases promoter strength by about 65 % to 52 %, compared to no addition. Addition of arabinose activates the promoter, in which increased amounts of inductor leads to increased promoter activity.    In total is the modified PBAD BBa_K2082112 a bit weaker than BBa_K808000 .    The main aim in decreasing promoter basal activity was not proven. Perhaps the used method has to be optimized further for detection the already small basal activity of BBa_K808000 and compare it with our modified promoter.
Error-prone polymerase I
For the polymerase I we contacted Manel Camps, the leading expert on this field. ( Read more about our     correspondence with Manel Camps here ) He told us that the two-plasmid mutagenesis system is a complex mechanism.     This means that changes in the expression levels of error-prone polymerase (e.g. overexpression) oftentimes yields no     additional mutagenesis effect. Because of this we decided to use the complete genetic construct provided by M. Camps as     device for mutagenesis with the polymerase I. The devices contains the error-prone polymerase I under control of the lac-promoter.
Figure 2: Schematic view of the error-prone polymerase I under control of a lac-promoter.
We made the error-prone polymerase I coding sequence aviable asexpression device available as basic part ( BBa_K2082106 ).     As negative controls we also added the wild type E. coli polymerase I coding sequence ( BBa_K2082107 )
We characterized BBa_K2082106 and BBa_K2082107 under    control of a lac-promoter.    We used the expression devices for the error-prone polymerase I and the wild type polymerase I in the plasmid pHSG, which resemebles the iGEM     backbone pSB4C5, because of its pSC101 origin of replication.
Genome wide mutator
We wanted to port the mutator plasmid described by Badran and coworkers (Badran and Liu 2015) to the iGEM standard.
This is a mutation device consisting of six genes which modulate the E. coli fidelity mechanism     (for a detailed explanation see here ).     Because of the potential hazardous effect of the potent mutator genes we decided to use the described, tightly regulated     arabinose promoter. In addition we thought that low level expression of the mutagenic protein is enough to achieve a mutator phenotype.     Furthermore the expression strength of our arabinose promoter under complete induction is high enough to obtain sufficient     amounts of proteins even when more is needed.
Therefore, we decided that it is more beneficial for the controllability of our mutator constrauct to use a very weak ribosome binding     site. We used BBa_B0031 as appropriately pretty weak RBS.
As for the genes for our mutator plasmid we build upon the work done by Badran and coworkers, who already tested several     combinations of mutator genes and found a combination with high mutagenesis rate and broad spectrum.
The six genes are dnaQ926, dam, seqA, emrR, ugi and P. marinus cda1. dnaQ926 , dam and emrR are already in the iGEM parts registry so we ended up using them.     (see here how our results further characterise these parts)
We added the coding sequences for seqA , ugi and cda1 via gene synthesis.
We assembled the mutagenesis genes by Gibson Assembly and added BBa_B0031 in the     process in front of each mutation gene. To easily assemble six genes the synthesis was done in two steps. At first dnaQ926, dam and seqA respectively emrR, ugi and cda1 were combined. Sadly the first assembly could only be isolated without the dnaQ926 RBS. In the     following step both synthesis intermediates were combined by BioBrick Assembly . To add the RBS in front of dnaQ926 and to add a terminator behind the construct     the complete genes were cloned into BBa_K516031 . The complete coding sequences with     ribosome binding site were cloned downstream of our modified PBAD to obtain the final construct BBa_K208117 .
Figure 4: Schematic view of the cloning process leading to BBa_K208117 .
To assess the effect of seperate expression of dnaQ926, which is a very strong mutator in itself, this gene alone was cloned into BBa_K516031 ,     added downstream of our modified PBAD, which yielded BBa_K2082116 .
Figure 5: Schematic view of BBa_K2082116 .
Reporter
To characterise our mutation systems we did reversion assays . Therefore, we needed reporter proteins, which activity could     be easily measured. We decided on beta-lactamase and gfp as reporters because of the easy detection assays.
Before deciding where to put the stop-codon we put a lot of effort into identifying suitable positions. We had two concerns:
Can we make sure that the stop-codon is outside of the okazaki-fragments?
Can we make sure that every single base pair change, that does not change the stop codon to another stop codon, reverts the reporter back to function?
First first points is due to the irregular distribution of mutations when using the error-prone polymerase I. Because of the enzymes role in processing     okazaki fragments these region are hotspots to mutations. The mutation rate of a hotspot region would appear higher than that of the main part of the mutated region.
To solve this problem we choose to create two different reporters, where the positions of the stop codons are separated at least 120 bp from each other.
The next concern was that there are positions inside proteins where the amino acids is remarkable important for the proteins function (e.g. amino acids     inside the catalytic center) or folding. We wanted to avoid these important positions, because at these positions only reversions back to the original amino     acid would reconstitute the protein. The catalytic center for both of our reporter proteins are known (Figure 7) and can be avoided. To avoid positions     responsible for folding of the proteins the impact of point mutation on the folding was modelled using FoldX . Each amino acid was in silico mutated to each other amino acid and the protein stability was calculated. The complete results for the positions 100 to 109 are depicted in Figure 6a, the impact of alanine replacement on every position was mapped on the    GFP crystal structure as an overview (Figure 6b). Striking is the importance of Gly104, where every change in amino acid greatly destabilizes the protein. Based on these modelling results (complete scan like Figure 6a) two positions for introduction of a stop codon were determined (Figure 7).    In addition, a stop position inside gfp already used in reversion experiments were used as well. (Wang et al. 2004)
Figure 6: Extensive in silico stability scan for GFP (A) and mapping of the stability changes for a complete in silico alanine scan (B). The stability changes for mutating each position inside GFP were calculated usign FoldX. The stability changes are calculated in ΔΔG (difference of mutant folding energy to wild type folding energy, low means more stable).      As overview the complete values for mutating to alanine were mapped on the GFP structure. (PDB: 2WUR)
Figure 7: Coding sequence for beta-lactamase and gfp with indicated function-essential amino acids as well as introduced stop-codons.
These stop codons were introduced by site-directed mutagenesis of BBa_K608010 and the beta lactamase from pSB1AK3 creating     the following BioBricks:
beta-lactamase
K30∗
L79∗
BBa_K2082151
BBa_K2082152
gfp
K3∗
Y106∗
K107∗
BBa_K2082113
BBa_K2082114
BBa_K2082115
The stop-GFP's were analysed by flow cytometry (Figure 8) and complete loss of fluorescence by introduction of a stop-codon was observed.
A)
B)
C)
Figure 8: Characterisation of stop-GFP by flow cytometry. The different stop-GFPs (green) were analysed by flow cytometry and compared      to wt-GFP (red, BBa_E0040 ).
Read on and see how we used our assembled mutators
Reversion
Sequencing
References
Badran, Ahmed H.; Liu, David R. (2015): Development of potent in vivo mutagenesis plasmids with broad mutational spectra. In: Nature communications 6, S. 8425. DOI: 10.1038/ncomms9425.
Miyada, C. G.; Stoltzfus, L.; Wilcox, G. (1984): Regulation of the araC gene of Escherichia coli: catabolite repression, autoregulation, and effect on araBAD expression. In: Proceedings of the National Academy of Sciences of the United States of America 81 (13), S. 4120–4124.
Wang, Clifford L.; Harper, Ryan A.; Wabl, Matthias (2004): Genome-wide somatic hypermutation. In: Proceedings of the National Academy of Sciences of the United States of America 101 (19), S. 7352–7356. DOI: 10.1073/pnas.0402009101.
