Team:Bielefeld-CeBiTec/Results/Mutation
Toggle navigation
NavBar
PROJECT
Description
Library
Overview
Scaffolds
Design and Construction
Create your own Library
Mutation
Overview
Error prone Polymerase I
Genome wide mutator
Selection
Overview
Bacterial Two-Hybrid System
Alternative Selection System
Modeling
Overview
Model I
Model II
RESULTS
Overview
Library
Overview
Assembly
Phagemid Display
Sequencing
Mutation
Overview
Assembly
Reversion
Sequencing
Selection
Overview
Assembly
Expression Control
Interaction Control
Binding Control
in vivo Selection
Modeling
Fermentation
Parts
Best Basic Part
Best Composite Part
Best Part Collection
Improve a part
Proof of Concept
Demonstrate your Work
Medal Fulfillment
HUMAN PRACTICES
Overview
Engagement
Dictionary
Educational Outreach
Crowdfunding and Press
Entrepreneurship
Integrated Practices
Achievements
COLLABORATIONS
Overview
Düsseldorf
Lethbridge
Freiburg
NOTEBOOK
Overview
Lab Notebook
Library
Mutation
Selection
Primers
Lab Protocols
Media and Buffers
TEAM
Team Profile
Members
Bielefeld University
Contact
ACKNOWLEDGEMENT
Overview
Sponsors and Partners
Attributions
Results Mutation
New is always better
Result Mutagensis
In the Evobody generating process we want to obtain a high affinity binding protein by directed evolution.      To further evolve binding proteins with an innate ability to interact with our target we want to mutagenize      them and thereby enable an ongoing adaptation to the target.
We successfully cloned two different approaches to in vivo mutagenesis, demonstrated and quantified      mutagenesis by reversion assays and determined the mutation spectrum by next-generation sequencing .
Cloning
We added several in vivo mutagensis parts to the iGEM parts registry
BioBrick Number
construct
BBa_K2082106
error-type polymerase I
BBa_K2082107
wild-type polymerase I
BBa_K2082116
dnaQ926
BBa_K2082117
M6
In vivo Characterization
From our mutators we quantified the mutation by reversion assays using a stop beta lactamase.
Figure 1: Mutagensis as reversion rate for the error prone polymerase I in comparison to wild type polymerase I. Frequency of ampicillin resistance colonies per complete cell count was determined.
Figure 2: Mutagenesis rate of our genome wide mutators  as reversion rate for the error prone polymerase I in comparison to wild type polymerase I. Frequency of ampicillin resistance colonies per complete cell count was determined.
Furthermore we determined the mutation rate of our error prone polymerase I as well as from our genome wide mutator      BioBrick BBa_K2082117 by high-throughput sequencing.
Figure 3: Mutagenesis rate of our genome wide mutators BBa_K2082116 and BBa_K2082117 .
Figure 4: Mutagenic spectrum of our error prone polymerase I and our genome wide mutators BBa_K2082117 and BBa_K2082116
Assembly
Reversion
Sequencing
