<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Results_Mutation_Reversion skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Results/Mutation/Reversion</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container main"><DIV id="header"><NAV id="navigationbar" class="navbar navbar-default custom_navbar_style navbar-fixed-top"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="#">NavBar</A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-center"><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description"> PROJECT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description">Description</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds">Scaffolds</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design">Design and Construction</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/CreateYours">Create your own Library</A></LI></UL><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/EpPolI">Error prone PolymeraseI</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/Global">Genome wide mutator</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">Bacterial Two-Hybrid System</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/SplitBetaLactamaseHitchHiker">Alternative Selection System</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_I">Model I</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_II">Model II</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results"> RESULTS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Phage">Phagemid Display</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">Reversion</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/Cloning">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/ExpressionControl">Expression Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/InteractionControl">Interaction Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/BindingControl">Binding Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/KnockOutKnochIn"><I>in vivo</I> Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Modeling">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Fermentation">Fermentation</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Best Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Best Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Best Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts/Improve">Improve a part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Demonstrate your Work</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Medal_Fulfillment">Medal Fulfillment</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold"> HUMAN PRACTICES </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Dictionary">Dictionary</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks">Educational Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Crowdfunding">Crowdfunding and Press</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Achievements</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations"> COLLABORATIONS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Duesseldorf">DÃ¼sseldorf</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Lethbridge">Lethbridge</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Freiburg">Freiburg</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments"> NOTEBOOK </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook">Lab Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">Primers</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols">Lab Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Media">Media and Buffers</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team"> TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team">Team Profile</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Members">Members</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Contact">Contact</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement"> ACKNOWLEDGEMENT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Sponsors_And_Partners">Sponsors and Partners</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></DIV></DIV></NAV></DIV></DIV><DIV class="container main"><DIV class="jumbotron" style="background-image: url(https://static.igem.org/mediawiki/2016/0/02/Bielefeld_CeBiTec_2016_10_14_X_projectheader.png)"><DIV class="jumbotron-text"><H1 style="margin-bottom: 0px; text-align:left">Determine Mutation Rates by Reversion Assays</H1></DIV></DIV><DIV class="container text_header" id="overview"><H3>Overview</H3></DIV><DIV class="container text">
			We performed reversion assays to investigate our mutagenic constructs. In the <A href="#firststeps">first step</A>, we confirmed that we obtain functional 
			revertants by sanger sequencing. Afterwards we quantified mutation rate of our <A href="#eppol">Error Prone Polymerase I</A> and our
			<A href="#m6">Genome Wide Mutator</A>. At last we try to integrate the mutation feature into our complete system by assessing the mutation while continuous cultivation.
		</DIV><DIV class="container text_header" id="eppol"><H3>Introduction</H3></DIV><DIV class="container text">
			To quantify mutagenesis rate exactly is a very tedious task, because a multitude of factors affect the mutation rate. The most important are:
			<UL><LI>the experimental setup</LI><LI>the exact used method</LI></UL>
			Classical approaches to measure mutation rate are distinguished in reversion or forward assays, respectively. The mutation reverts a stop 
			from a reporter (reversion assay) or mutates a protein in such a way that it gains a function e.g. the organism becomes resistant to 
			antibiotics (mostly rifampicin forward assay). Since the rise of high-throughput sequencing, this technique starts to replace reversion 
			assays as prime method for assessing the mutation rate due to higher sensitivity, resolution and additional information. 
			
			As the error prone polymeraseÂ I is mostly specific for ColE1 plasmids we used a reporter on a plasmid. Therefore, we decided to 
			perform reversion assays with a stop beta-lactamase on a ColE1 plasmid.
		</DIV><DIV class="container text_header" id="firststeps"><H3>First steps</H3></DIV><DIV class="container text">
			Before going on and quantifying the mutagenesis rate we wanted to show that, our mutagenesis process yield functional revertants. 
			Therefore, we applied a modified version of the mutagenesis protocol described in the literature (Alexander et al. 2014). In 
			summary, we transformed the stop beta-lactamase plasmid pLA230 into <I>E.Â coli</I> JS200 cells carrying pHSG-EPPolI plasmids. 
			After regeneration, the cells were used to inoculate a 37Â Â°C prewarmed LB culture. After 24 hours, cells were plated in 
			various dilution level at LB plates containing ampicillin. 10 colonies were picked, grown at 30Â Â°C, the plasmids were isolated and sequenced.
			<FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 1: Sequencing of  revertants, showing reversion of the stop codon.</B>  revertant colonies were grown in selective media 
					and sequenced. 
				</FIGCAPTION></FIGURE>
			The sequencing results show that the obtained revertants are really revertants, which mutated one base inside the stop codon to obtain 
			fully functional beta lactamase. In some cases the reversion is not completely obvious and is only hinted as two different peaks at the same position in the interferogram.
		</DIV><DIV class="container text_header" id="eppol"><H3>Error Prone Polymerase I</H3></DIV><DIV class="container text">
			The function of the error prone polymeraseÂ I in our experimental setup was validated by the identification of <I>E.Â coli</I> clones 
			with different mutations. Further characteriation of the error-prone polymeraseÂ I involves the determination of the mutation rate 
			and adapting for optimal use in the Evobody generation system. 
			
			A literature review revealed the technique of mutation accumulation, where the fraction of revertants f at two points with a large 
			difference in cell count N is measured. Examples are measurement in the beginning and in the end of a cultivation. 
			<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Equation 1: Determination of mutation rate by mutant accumulation (Pope et al. 2008).</B> 
					and sequenced. 
				</FIGCAPTION></FIGURE></CENTER>
			However, critical points have to be considered, when calculating the mutation rate. The reporter is encoded on plasmid with copy 
			numbers per cell ranging from 100 to 300. We postulate that one reversed beta-lactamase gene can confer ampicillin resistance to 
			the cell. Therefore, the observed mutation rate is 100 to 300 fold higher than the true mutagenesis rate. Additionally, potential 
			reversion sites on each plasmid contribute to the underestimation of the mutation rate. The stop codon TAA can mutate into nine 
			different codons, which include two new stop codon (TGA, TAG). In total, there are seven mutation possibilities on every plasmid 
			which could revert the stop beta-lactamase back to its functional state. These considerations lead to the modification of equation 
			one to equation and.
			<FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Equation 2: Modified equation to determine the mutation rate by mutant accumulation considering a reporter with a stop codon on a plasmid.</B></FIGCAPTION></FIGURE>
			To measure the mutation rate similar conditions for each replicate were choosen. The setup looked as following. JS200 cells 
			carrying either pHSG-EP Pol I or pHSG-WT Pol I were transformed with 10 ng pLA230, regenerated for 60 min in SOC and transferred 
			LB with chloramphenicol and kanamycin. After 24 h the cell count was determined by OD600, the cells were serial diluted and appropriate 
			dilutions were plated on LB plates with ampicillin and grown at 30Â°C. 
			
			The reversion frequency f was determined by counting ampicillin resistant colonies and normalizing this value by the number of cells 
			measured via OD600. Therefore, we assumed OD<SUB>600</SUB> as 5Ã10<SUP>8</SUP> cells. This experiment was done in five biological replicates 
			both with the error prone polymerase I as well as with the wild type polymerase I.
			<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Equation 2: Modified equation to determine the mutation rate by mutant accumulation considering a reporter with a stop codon on a plasmid.</B></FIGCAPTION></FIGURE></CENTER>
			An increase in reversion frequency by a factor of 2.7Ã10<SUP>3</SUP> when using the EP Pol I in contrast to WT Pol I 
			was observed. In this evolution of the reversion experiment the controls pre mutagenesis were not plated, which prevents 
			exact determination of the change in reversion frequency. This leads to us using plating to determine the viable cell count in addition to the revertant count.
			
			Due to obvious difference between the error-prone and the wild type polymerase I, we reduced consumption of material by only 
			characterizing the error prone polymerase in further detail. Major obstacles when trying to measure revertants in wild-type 
			cultures were the trait offs in respect to the number of plated cells. Plating too few cell leads to no revertants, while 
			plating too much cells leads to an unspecific growth of a strange bacterial lawn of not resistant cells on top of other dead 
			cells. However, this experiment clearly showed that the number of mutations caused by the wild type polymerase is not measureable 
			in this experiment.
			
			Our next experiment was tailored to the Evobody generation process. In this proof of concept  experiment Evobody-producing 
			<I>E.Â coli</I> are selected based on their different growth rates. Simultaneously,the Evobody coding sequence was mutagenized. 
			The recommended mutagenesis protocol for the EP Pol I includes periodic retransformation of the mutagenized plasmids. However, 
			this is not suitable to our approach. We started with a library and therefore avoided transformations to keep the diversity as 
			high as possible. Transformations are known bottle necks, while working with libraries. Therefore, we investigated the mutation 
			accumulation rate by inoculating LB culture with colonies grown at 30Â°C and growing them to saturation at 37Â°C. To calculate the 
			mutation accumulation according to equation 2 the reversion frequency were determined at two different points of the cultivation.
			<FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 2: Reversion frequency of <I>E.Â coli</I> JS200 with EP Pol I at different time points.</B> JS200 carrying the 
			EPÂ PolÂ I and the stop lactamase plasmid were cultivated for 24Â h in LB at 37Â°C (biological replicates A-C). 
			The reversion frequency was determined after 0Â h and 24Â h by plating and counting the fraction of ampicillin resistance 
			colonies. The reversion frequency increases over time by the factor four.
				</FIGCAPTION></FIGURE>
			This cultivation experiment revealed two interesting results: 
			<UL><LI>The mutation frequency within JS200 colonies containing the EPÂ PolÂ I grown at the permissive temperature 
				of 30Â°C is still notable</LI><LI>The reversion frequency increases after cultivation at 37Â°C by the factor of four.</LI></UL>
			Both results show striking similarity to a previously determined mutation rate at the permissive temperature (Camps, personal communication). 
			Camps told us, that mutations still occur at 30Â°C, but the rate is reduced to only 25% of the mutation rate at 37Â°C.
			<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 3: Reversion frequency after 0 h and 24 h (red) and the calculated difference (blue).</B> The reversion frequency of EP Pol in 
					JS200 was determined after 0 h and 24 h by plating (red). For calculating the mutation rate via equation <A>one</A> the difference 
					in mutation frequency was determined (blue). The error bars represent the standard deviation of three biological replicates.
				</FIGCAPTION></FIGURE></CENTER>
			
			Another observation is the huge variability in the reversion frequency at the starting point. This variability is much lower for 
			the mutation rate after 24 h. This shows that mutations, which occur early into a mutagenic process, take an overproportional part 
			of the whole culture. Therefore, exact determination of the mutation rate is only possible by measuring the mutation accumulation 
			(f<SUB>2</SUB>-f<SUB>1</SUB>) and not the absolute mutation count (Pope et al. 2008). Using the results from Figure 3 for f<SUB>2</SUB>-
			f<SUB>1</SUB> = (9.47Â±2.28)Ã10<SUP>-5</SUP> and Figure 2 for N<SUB>0h</SUB> = 0.1 OD<SUB>600</SUB> and N<SUB>24h</SUB> = 
			2.44 OD<SUB>600</SUB> and a plasmid copy number of 10 (Camps et al. 2003) in equation 2 this cumulates to a mutation rate of
			
			This measures mutation rate is three orders of magnitude over the basal mutation rate of <I>E.Â coli</I> mutation rate (Lee et al. 2012). 
			It strongly differs from the rate described by Camps of 8.1Ã10<SUP>-4</SUP> per bp which converts to 4.8Ã10<SUP>-5</SUP> per bp 
			per generation under the assumption that Camps cultivation process involves ~16 generations. 
			
			Furthermore, Camps uses a completely different approach in quantifiying the mutagenesis rate by starting from one measurement of the 
			reversion frequency f and using the plasmid copynumber (10) and reversion sites (7) and conversion between codon and base (3) 
			(Camps et al. 2003). His equitation looks like
			
			When using this formula we obtain a mutation rate of 1.11Ã10<SUP>-5</SUP> per bp, which is 72-fold lower than the one calculated by Camps. 
			In our opinion, the method of mutant accumulation is a more reliable method to discern mutation rate. Especially our experiment 
			where we used three independent cultures is a very descriptive example for this. Although the total reversion frequency at the start 
			as well as at our end point differs strongly between the cultures, the change in reversion frequency between both measured time points 
			is surprisingly low.
		</DIV><DIV class="container text_header" id="m6"><H3>Genome Wide Mutator</H3></DIV><DIV class="container text">
			We performed reversion assays with our genome wide mutators <I>dnaQ926</I>(<A href="http://parts.igem.org/Part:BBa_K2082116">BBa_K2082116</A>) 
			and M6(<A href="http://parts.igem.org/Part:BBa_K2082116">BBa_K2082117</A>) again using the stop lactamase plasmid pLA230. 
			
			Our first approach was to cotransform the reporter plasmid and pSB1C3:dnaQ into <I>EÂ coli</I> Top10 and inoculating cultures 
			containing arabinose or glucose directly with the transformed cells. This yielded a reversion frequency of 2.6Ã10<SUP>-5</SUP> 
			induced and 1.3Ã10<SUP>-6</SUP> uninduced after 12 h. Because we did not plate the cells out for exact determination of the cell 
			count this result is only a first hint at an increases mutation rate through activation of our mutator.
			
			Our next step was to transform the mutator and the reporter plasmids in <I>EÂ coli</I> Top10, plate this on LBCmKan, inoculate LBCmKan 
			cultures and induced or repress this culture when reaching mid-log phase. The cells were plated after 24 h and counted. 
			From this experiment, the plasmids for our high-throughput experiment were prepared. 

<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 4: Reversion frequency measured in our first experiment with the genome wide mutators.</B> The reversion rate is probably 
			not correct (see text), because no plating to count the exact cell count was done. This was included into our next experiments.
			The results show an increased reversion frequency in samples supplemented with glucose. In theory the addition of glucose 
			should repress the P<SUB>BAD</SUB> activity, thus decreasing expression of mutagenic proteins.
				</FIGCAPTION></FIGURE></CENTER>

			For the observed smaller reversion frequency of the activated mutagenesis parts there can be two explanations.
			
			The exact cell count was not determined by plating but only by OD<SUB>600</SUB>. Because expression of the mutator proteins 
			decreases cell viability (Badran and Liu 2015) the measured OD<SUB>600</SUB> value probably contains large amount of dead cells, 
			therefore greatly inflating our supposed cell count. Badran found that induction of their M6 variant decreases cell viability 
			to ~5 %. This would result that the accessed total cell count is 20x too high in the induced samples, resulting in a 20x to 
			low reversion frequency. 
			
			Another possibility is that the arabinose promoter was active through very low levels of arabinose in LB. It is possible 
			that even very low levels of leakiness can lead to some mutagenic proteins. For example in the case of <I>dnaQ926</I> one 
			protein is enough to create one proofreading deficient polymerase III complex. In normal <I>E.Â coli</I> 
			strains this should be no problem but the Top10 strain is deficient for the arabinose metabolism and therefore low levels of 
			arabinose are not metabolized but activate promoter activity.
			
			We concluded that in order to obtain more reliable data we had to plate dilutions with low amounts of cells in order to 
			determine the viable cell count in our samples. To combat the supposed problem of promoter leakiness in LB media we dicided 
			to use LB supplemented with 20 mM glucose. This is analog to the protocol from Badran and coworkers who used a defined media with glucose.
			
			Using all this experience, we designed another experiment to determine the mutation rate.
			
			We transform <I>E.coli</I> Top10 with the reporter plasmid as well as the mutator plasmid (pSB1C3:BBa_K2082116 or BBa_K2082117), plate a portion 
			and use another portion to inoculate a LBCmKan culture with 20 mM glucose, let this culture grow till mid-log phase and induce 
			or repress the P<SUB>BAD</SUB>. 24 h after induction the cells were plated on LBCmKan or LBAmp each with 20 mM glucose and 
			counted revertants and the total viable cell number. This experiment was performed for each of our mutator in six replicates.
			<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 5: Figure: Beta lactamase reversion frequency using <I>dnaQ926</I> (BBa_K2082116) or M6 (BBa_K2082117) as mutator.</B></FIGCAPTION></FIGURE></CENTER><CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 6: Statistic of the beta lactamase reversion experiment using <I>dnaQ926</I> (BBa_K2082116) or M6 (BBa_K2082117) as mutator.</B></FIGCAPTION></FIGURE></CENTER>
			For the growth term in Equation 2 we measure the cell count right after transformation and while plating after 24 h. Therefore we 
			could determine N<SUB>2</SUB>/N<SUB>1</SUB>. As plasmid copy number we used n=40 as determined in our <A href="2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Sequencing">NGS experiment</A>.
			<CENTER><TABLE><TBODY><TR><TD>f<SUB>2</SUB>-f<SUB>1</SUB></TD><TD>N<SUB>2</SUB>/N<SUB>1</SUB></TD><TD>Î¼ [bp<SUP>-1</SUP>Ãgeneration<SUP>-1</SUP>]</TD></TR><TR><TD rowspan="2"><I>dnaQ926</I> (BBa_K2082116)</TD><TD>Induced</TD><TD>(3.92Â±1.91)Ã10<SUP>-4</SUP></TD><TD>70.11</TD><TD>(2.3Â±1.12)Ã10<SUP>-6</SUP></TD></TR><TR><TD>Repressed</TD><TD>(2.30Â±1.41)Ã10<SUP>-5</SUP></TD><TD>5530,43</TD><TD>(6.67Â±4.09)Ã10<SUP>-8</SUP></TD></TR><TR><TD rowspan="2">M6 (BBa_K2082117)</TD><TD>Induced</TD><TD>(1.86Â±1.12)Ã10<SUP>-3</SUP></TD><TD>660,85</TD><TD>(7.16Â±4.31)Ã10<SUP>-6</SUP></TD></TR><TR><TD>Repressed</TD><TD>(0.82Â±1.04)Ã10<SUP>-5</SUP></TD><TD>3538,52</TD><TD>(2.51Â±3.18)Ã10<SUP>-8</SUP></TD></TR></TBODY></TABLE></CENTER><CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 7: Mutagensis rate of <I>dnaQ926</I> (BBa_K2082116) and M6 (BBa_K2082117).</B></FIGCAPTION></FIGURE></CENTER>
			We determined the mutation rate of our genome wide mutator BBa_K2082117 to be (7.16Â±4.31)Ã10<SUP>-6</SUP> bp<SUP>-1</SUP>Ãgeneration<SUP>-1</SUP> induced and (2.51Â±3.18)Ã10<SUP>-8</SUP> bp<SUP>-1</SUP>Ãgeneration<SUP>-1</SUP> when repressed.
			This is almost exactly the same rate Badran and coworkers determined for their iteration of the M6 mutator (Badran and Liu, 2015), who determined a mutation rate of
			6.2Ã10<SUP>-6</SUP> bp<SUP>-1</SUP>Ãgeneration<SUP>-1</SUP> induced and 3Ã10<SUB>-9</SUB> bp<SUP>-1</SUP>Ãgeneration<SUP>-1</SUP>.
			Therfore our aim to reconstruct this mutator with definded iGEM parts was successful.
			Furthermore we showed the muation rate of BBa_K1333108 in our construct BBa_K2082116 to be (2.3Â±1.12)Ã10<SUP>-6</SUP> bp<SUP>-1</SUP>Ãgeneration<SUP>-1</SUP> induced and (6.67Â±4.09)Ã10<SUP>-8</SUP> bp<SUP>-1</SUP>Ãgeneration<SUP>-1</SUP>.
			Thereby we show the functionality of this already existing part.
		</DIV><DIV class="container text_header"><H3>Summary</H3></DIV><DIV class="container text">
			We showed mutation using the error prone polymerase I, which is 4-5 orders of magnitude higher than basal mutation rate. 
			
			We determined the mutation rate of the error prone polymerase I and our genome wide mutator systems. Both systems produced mutations with a 
			frequency several orders of magnitude over the background. Therefore, our parts can be used be <I>in vivo</I> generation of protein libraries 
			to optimze enzymes or proteins for various purposes.
			
			After determining the mutation rate of our construct we also investigated them with <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Sequencing">high-throughput sequencing</A></DIV><CENTER><BUTTON><A class="button_link" href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Sequencing">Sequencing</A></BUTTON></CENTER><DIV class="container text_header"><H3>References</H3></DIV><DIV class="container text"><UL><LI>Alexander, David L.; Lilly, Joshua; Hernandez, Jaime; Romsdahl, Jillian; Troll, Christopher J.; Camps, Manel (2014): Random mutagenesis by error-prone pol plasmid replication in Escherichia coli. In: Methods in molecular biology (Clifton, N.J.) 1179, S. 31â44. DOI: 10.1007/978-1-4939-1053-3_3.</LI><LI>Badran, Ahmed H.; Liu, David R. (2015): Development of potent in vivo mutagenesis plasmids with broad mutational spectra. In: Nature communications 6, S. 8425. DOI: 10.1038/ncomms9425.</LI><LI>Camps, Manel; Naukkarinen, Jussi; Johnson, Ben P.; Loeb, Lawrence A. (2003): Targeted gene evolution in Escherichia coli using a highly error-prone DNA polymerase I. In: Proceedings of the National Academy of Sciences of the United States of America 100 (17), S. 9727â9732. DOI: 10.1073/pnas.1333928100.</LI><LI>Lee, Heewook; Popodi, Ellen; Tang, Haixu; Foster, Patricia L. (2012): Rate and molecular spectrum of spontaneous mutations in the bacterium Escherichia coli as determined by whole-genome sequencing. In: Proceedings of the National Academy of Sciences of the United States of America 109 (41), S. E2774-83. DOI: 10.1073/pnas.1210309109.</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>