Team:Bielefeld-CeBiTec/Results
Toggle navigation
NavBar
PROJECT
Description
Library
Overview
Scaffolds
Design and Construction
Create your own Library
Mutation
Overview
Error prone Polymerase I
Genome wide mutator
Selection
Overview
Bacterial Two-Hybrid System
Alternative Selection System
Modeling
Overview
Model I
Model II
RESULTS
Overview
Library
Overview
Assembly
Phagemid Display
Sequencing
Mutation
Overview
Assembly
Reversion
Sequencing
Selection
Overview
Assembly
Expression Control
Interaction Control
Binding Control
in vivo Selection
Modeling
Fermentation
Parts
Best Basic Part
Best Composite Part
Best Part Collection
Improve a part
Proof of Concept
Demonstrate your Work
Medal Fulfillment
HUMAN PRACTICES
Overview
Engagement
Dictionary
Educational Outreach
Crowdfunding and Press
Entrepreneurship
Integrated Practices
Achievements
COLLABORATIONS
Overview
DÃ¼sseldorf
Lethbridge
Freiburg
NOTEBOOK
Overview
Lab Notebook
Library
Mutation
Selection
Primers
Lab Protocols
Media and Buffers
TEAM
Team Profile
Members
Bielefeld University
Contact
ACKNOWLEDGEMENT
Overview
Sponsors and Partners
Attributions
Results
Life is like a mirror - we get the best results when we smile at it
Library
We implement a basic diversity of initial binding proteins by creating a high-quality library consisting of over one hundred thousand Monobodies and over onehundredsixty thousand Nanobodies , respectively. Construction of our part collection was a foundation for our project and a cornerstone for utilizing libraries in iGEM.
We verified our binding protein libraries with Phagemid display and high-throughput sequencing techniques.
Figure 1: Randomized CDR of 24 Nanobody colonies. Top to bottom: Ordered sequence, chromatogram and sequencing result. Randomized regions show the occurence of the expected bases.
Mutation
In the Evobody generating process, we want to obtain a high affinity binding protein by directed evolution. To further evolve binding proteins with an innate ability to interact with our target, we want to mutagenize them and thereby enable an ongoing adaptation to the target.  We successfully cloned two different approaches for in vivo mutagenesis, demonstrated and quantified mutagenesis by reversion assays (figure 2) and determined the mutation spectrum by high-throughput sequencing.  Furthermore, we determined the mutation rate of our error prone polymerase I as well as of our genome wide mutator   BioBrick BBa_K2082117 by high-throughput sequencing.
Figure 2: Mutagenesis rate of our our error prone polymerase I and our genome wide mutator BBa_K2082117
Selection
The main selection system applied for the whole Evobody generator was a bacterial two-hybrid system. This method is able to select whole bacterial cultures based on their binding strength of the Evobody against a desired target. A lot of parameters of the system were checked by showing the interaction possibility of the positive controls, the binding of the cI DNA binding domain and the expression rate of the fusion proteins itself. However, the most important achievement was the demonstration of a whole working bacterial two-hybrid system.
Figure 3: Selection system
Modeling
With our mutagenesis system, we want to find the best adapted E. coli culture, and we want to find this culture as quickly as possible. Therefore, we looked for a tool that can predict, if the desired culture can be found at all, and, if yes, which culture will be the winner. Additionally, the prediction should provide optimal parameters for the laboratory, the best cultivation time and inoculation number. The program computes a minimal cultivation period of 8 hours and at least 11 inoculations, when an initial binder exists (figure 4).
Figure 4: Time between inoculations.
Fermentation
One of the fundamental aspects when establishing a directed evolution approach is to grant an advantage to the individuals with favorable properties. In our setup, we coupled correct binding properties of our Evobody proteins to a growth advantage in the form of a gradual ampicillin resistance. To determine the applicability of this resistance and to analyze how selective pressure can be applied, we did several fermentations to characterize the used strain
Library
Mutation
Selection
Modeling
