<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container main"><DIV id="header"><NAV id="navigationbar" class="navbar navbar-default custom_navbar_style navbar-fixed-top"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="#">NavBar</A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-center"><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description"> PROJECT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description">Description</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds">Scaffolds</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design">Design and Construction</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/CreateYours">Create your own Library</A></LI></UL><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/EpPolI">Error prone PolymeraseI</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/Global">Genome wide mutator</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">Bacterial Two-Hybrid System</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/SplitBetaLactamaseHitchHiker">Alternative Selection System</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_I">Model I</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_II">Model II</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results"> RESULTS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Phage">Phagemid Display</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">Reversion</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/Cloning">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/ExpressionControl">Expression Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/InteractionControl">Interaction Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/BindingControl">Binding Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/KnockOutKnochIn"><I>in vivo</I> Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Modeling">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Fermentation">Fermentation</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Best Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Best Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Best Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts/Improve">Improve a part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Demonstrate your Work</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Medal_Fulfillment">Medal Fulfillment</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold"> HUMAN PRACTICES </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Dictionary">Dictionary</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks">Educational Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Crowdfunding">Crowdfunding and Press</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Achievements</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations"> COLLABORATIONS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Duesseldorf">Düsseldorf</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Lethbridge">Lethbridge</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Freiburg">Freiburg</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments"> NOTEBOOK </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook">Lab Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">Primers</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols">Lab Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Media">Media and Buffers</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team"> TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team">Team Profile</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Members">Members</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Contact">Contact</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement"> ACKNOWLEDGEMENT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Sponsors_And_Partners">Sponsors and Partners</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></DIV></DIV></NAV></DIV></DIV><DIV class="container main"><DIV class="jumbotron" style="background-image: url(https://static.igem.org/mediawiki/2016/8/89/Bielefeld_CeBiTec_2016_10_14_X_jessi2.png)"><DIV class="jumbotron-text"><H1 style="margin-bottom: 0px; text-align:left">Project Description</H1><H2 style="color:#ffffff; text-align:left">The only way to do great work is to love what you do - Steve Jobs</H2></DIV></DIV><DIV class="container text_header"><H3>Motivation</H3></DIV><DIV class="container text">
			
            As long as mankind remembers, different diseases struck from time to time and demanded millions of lives. 
            Maybe the most fatal of these epidemic outbreaks was the 1918 flu pandemic, which killed between 50-100 million people worldwide.
            Derived from a simple influenza virus only a few mutations were necessary to change this virus into one of the deadliest threats 
            that ever existed. Pandemic viral infections like this were the reason for our project selection. Our highest
            motivation was to create a system to counteract these
            extremely high risk potential slumbering in commonly known and seemingly not to dangerous viruses. Of course the influenza virus 
            stated 1918 a much higher risk than it does today. But still viruses are very hard to treat and especially members of the family <I>flaviviridae</I>
            like Zika or Dengue virus, show a high risk potential because of their high mutation rate.
            We found a way to create in a short period of time antibody-like proteins. 
            Evobodies are binding
            proteins that are able to be quickly adapted to altered targets like viral hull proteins and 
            re-establish binding properties in extremely short periods of time
            The limiting factors in this process are the rate at which mutations happen in the gene of the Evobody 
            and screening of different proteins. This is the reason why we wanted to build a mutation inducing system
            which is not only able to change basepairs <I>in vivo</I> at a very high frequency, 
            but is also specific enough to provide stability in culture
            and does not interfere too much with growth properties of the individual cell.
        </DIV><DIV class="container text">
        
            The revolutionary part of Evobody generation is the combination of an <I>in vivo</I> mutagenesis
            and a selection system which is also capable of screening our mutants during the process of 
            cell cultivation. Due to our constant interaction of altered binding proteins and the <I>in vivo</I> 
            selection, we were eager to increase the mutation rate while retaining normal growth conditions. That is
            why we did not only calculate the mutation rate of our two different mutagenesis approaches, 
            but also determined the growth rate under
            different conditions.
        
		</DIV><DIV class="container text_header"><H3>Generation of binding proteins by directed evolution</H3></DIV><DIV class="container text">
			This year the iGEM Team Bielefeld-CeBiTec aims to create a method for generating synthetic binding proteins, 
			our so-called Evobodies. This works by creating a library of binding proteins and increasing their affinity 
			towards a target by directed evolution (Figure 1). As a starting point, we randomize the binding regions of 
			synthetic antibody-like proteins (Figure 1A). Following we screen this library for affinity towards a target by 
			using a bacterial two-hybrid system (Figure 1B). To further increase the Evobodies affinity, we combine the selection 
			via the two-hybrid system with an <I>in vivo</I> mutagenesis system (Figure 1C). Doing this we hope to generate strong 
			and specific binding proteins by combining the powerful genetics of <I>E. coli</I> with the biological idea of 
			antibody generation and maturation in vertebrates.
			<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 1: Schematic Evobody generation process</B>. A library of synthetic binding proteins (A) is screened by a bacterial two hybrid(B) for interaction with a traget protein. Existing interaction between both proteins leads to expression of a resistance gene thus granting the individual cell a selection advantage. After screening the library all suviving clones binding proteins are mutagenized <I>in vivo</I> (C) which results in the generation of new binding properties. Continuous selection cycles of selection and mutation increase the Evobodies binding affinity towards high affine binding proteins.  
				</FIGCAPTION></FIGURE></CENTER>
			We designed our Evobody approach as an alternative to conventional methods for the generation of binding proteins. 
			In our vision it should be possible to clone each protein encoding sequence into one of our plasmids, let our system 
			do the work and get a high affinity binding protein, which can be either used for medical, diagnostic or scientific applications.
		</DIV><DIV class="container text_header"><H3>The starting point - synthetic binding protein library</H3></DIV><DIV class="container text">
			As starting point, we want to create a library of many binding proteins with a high chance to contain a protein 
			with the potential to bind our target protein. In doing so we choose the core region of the antibody-mimetic mono- 
			and nanobodies. In the coding region of those proteins, we randomized the loop regions, which are known to bind 
			other proteins to obtain our library.
			The randomization strategy as well as the choice of the protein scaffold is a key part of library generation. 
			We identified amino acids, which are present in most protein-protein interaction areas and created a randomization 
			scheme so that only these amino acids are encoded in the antibody-mimetics binding region. Read more about our 
			library design <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">here</A>.<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 2: Library of initial binding proteins.</B> Expression of the initial binding proteins with variable regions highlighted in seperate colors (turquoise, orange, white, green, pink and blue) and constant regions in olive. The theoretical variability for each scaffold is 1.073.741.824.
				</FIGCAPTION></FIGURE></CENTER></DIV><DIV class="container text_header"><H3>Survival of the fittest - bacterial two-hybrid</H3></DIV><DIV class="container text">	
			In the next step, the binding protein library should be screened for proteins with an innate affinity for our target. 
			We want to realize this by using a bacterial two-hybrid system (Figure 3). Therefore, our target protein (1) is fused to a DNA 
			binding domain (2), which localises upstream of a reporter cassette (3). The binding protein (4) is fused to a RNA polymerase 
			subunit (5). Interaction between the binding and the target protein leads to recruitment of the RNA polymerase to the promoter 
			region of the reporter cassette and activates the reporter gene expression. By using an antibiotic resistance as a reporter 
			gene the output of the  bacterial two hybrid system should lead to the survival <I>E. coli</I> cells carrying a good binding 
			protein and the death of all cells with a bad binding protein.
			<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 3: Bacterial-two hybrid system.</B>Interaction between the binding protein (4) and the target protein (1) 
					lead to recruitment of RNA polymerase (5) to the promoter upstream of the reporter cassette (3) and subsequent 
					expression of the reporter gene. In this case the reporter is beta-lactamase, which expression leads to degradation 
					of ampicillin (blue squares) and survival of the bacteria.
				</FIGCAPTION></FIGURE></CENTER>
			From the two-hybrid system we expect foremost a selection of the binding protein library. Furthermore, we expect a correlation 
			between binding - target protein affinity and the activated gene expression strength of the reporter. By using an antibiotic 
			resistance protein as a reporter we predict increased levels of the resistance protein inside a cell with a high affinity 
			binding protein. The outcome of this should be an increase of individual fitness for bacteria with good binding proteins, 
			which should lead to a higher growth rate under strong selective pressure. The complete two-hybrid system should cumulate in a 
			correlation between binding protein affinity and bacterial growth rate, which should lead to selection of a few bacteria with 
			strong binding proteins. Find out more on our <A href="”" https:="">selection subpage</A>.
		</DIV><DIV class="container text_header"><H3>Accessing the sequence space - <I>in vivo</I> mutagenesis</H3></DIV><DIV class="container text">	
			After selection of the bulk of our library we will increase the affinity of our Evobodies in a process similar to the affinity 
			maturation of antibodies (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description#Teng2007">Teng und Papavasiliou 2007</A>). As 
			addressed above we will select our Evobodies by increasing the selection pressure. At the same time, we will use an <I>in vivo</I> 
			mutagenesis system. Thereby ,we can increase the sequence diversity beyond the limits of our library. Slightly modifications of  
			binding proteins identified during the initial selection will are the basis for the directed  evolution.
			<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 4: <I>In vivo</I> mutagenesis system.</B> By using an <I>in vivo</I> mutagenesis system a single Evobody coding 
					sequence can be evolved to various different variants, each with a unique binding site. The single starting sequence is 
					replicated during growth and thereby mutations are incorporated either through error-prone polymerase I or a combination 
					of global mutator genes. The process results in the creating of the library of binding proteins with different binding properties.
				</FIGCAPTION></FIGURE></CENTER>
			In detail, we will compare two different possibilities to diversify our binding proteins. The first approach is the use of an 
			error-prone polymerase I in an otherwise Pol I temperature-sensitive <I>E. coli</I> strain. 
			(<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description#Camps2003">Camps et al. 2003</A>) 
			Growth at a non-permissive temperature should result in accumulation of mutations in the part of the genom maintained by DNA polymerase I. 
			The interesting idea behind this approach is the fact that large parts of plasmids carrying an origin of replication from the 
			ColE1-familiy are replicated by the polymerase I. (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description#Camps2003">Camps et al. 2003</A>; <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description#Camps2010">Camps 2010</A>) 
			Because of this, the usage of the error-prone polymerase I should mutant mainly our Evobody sequence on a plasmid. Thereby off-target 
			mutations, which are a major obstacle of <I>in vivo</I> mutagenesis, should be minimized. (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description#Camps2003">Camps et al. 2003</A>)
			Our other approach is based on creating a plasmid borne hypermutator system by modulating the <I>E. coli</I> DNA fidelity systems. 
			(<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description#Badran2015">Badran und Liu 2015</A>) We will express known mutator genes under tight regulation from a plasmid. By using a plasmid borne mutator 
			system, in contrast to the more classical approaches of incorporating the mutator genes directly inside the genom. 
			(<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description#AgilentTech">Agilent Technologies</A>; <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description#Greener1997">Greener et al. 1997</A>)  
			Thereby we want to circumvent the known problems with globally increased mutation rate, which are genetic instability or general unviability.
			Over the course of our project we want to find out which mutagenesis system is most suitable to our directed evolution approach. 
			Therefore we will compare both possibilities in terms of mutagenesis rate, -spectrum, -controllability and -specifity. How will 
			we do this? Find out on our <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">mutation</A> mainpage.
		</DIV><CENTER><BUTTON><A class="button_link" href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library" role="button">Library</A></BUTTON><BUTTON><A class="button_link" href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation" role="button">Mutation</A></BUTTON><BUTTON><A class="button_link" href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation" role="button">Selection</A></BUTTON><BUTTON><A class="button_link" href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model" role="button">Modeling</A></BUTTON><BUTTON><A class="button_link" href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results" role="button">Results</A></BUTTON></CENTER><DIV class="container text_header"><H2>Improve a part</H2><H3>Mutator gene dnaQ926 - <A href="http://parts.igem.org/Part:BBa_K1333108">BBa_K1333108</A></H3></DIV><DIV class="container text">
			We used <A href="https://2014.igem.org/Team:SYSU-China">iGEM SYSU-China 2014's</A> dnaQ926 (<A href="http://parts.igem.org/Part:BBa_K1333108">BBa_K1333108</A>) in our genome wide mutator.
			
			See <A href="">here</A> or go directly to the <A href="http://parts.igem.org/Part:BBa_K1333108">partsreg</A> to see how we contribute to BBa_K1333108 characterization.
		</DIV><CENTER><BUTTON><A class="button_link" href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts/Improve" role="button">Improve a part</A></BUTTON></CENTER><DIV class="container text_header"><H3>Literature</H3></DIV><DIV class="container text"><UL><LI id="AgilentTech">Agilent Technologies: XL1-Red Competent Cells. Instruction Manual 2015.</LI><LI id="Badran2015">Badran, Ahmed H.; Liu, David R. (2015): Development of potent in vivo mutagenesis plasmids with broad mutational spectra. In: Nature communications 6, S. 8425. DOI: 10.1038/ncomms9425.</LI><LI id="Camps2010">Camps, Manel (2010): Modulation of ColE1-like plasmid replication for recombinant gene expression. In: Recent patents on DNA &amp; gene sequences 4 (1), S. 58–73.</LI><LI id="Camps2003">Camps, Manel; Naukkarinen, Jussi; Johnson, Ben P.; Loeb, Lawrence A. (2003): Targeted gene evolution in Escherichia coli using a highly error-prone DNA polymerase I. In: Proceedings of the National Academy of Sciences of the United States of America 100 (17), S. 9727–9732. DOI: 10.1073/pnas.1333928100.</LI><LI id="Fijalkowsk1996">Fijalkowska, I. J.; Schaaper, R. M. (1996): Mutants in the Exo I motif of Escherichia coli dnaQ: defective proofreading and inviability due to error catastrophe. In: Proceedings of the National Academy of Sciences of the United States of America 93 (7), S. 2856–2861.</LI><LI id="Greener1997">Greener, A.; Callahan, M.; Jerpseth, B. (1997): An efficient random mutagenesis technique using an <I>E. coli</I> mutator strain. In: Molecular biotechnology 7 (2), S. 189–195. DOI: 10.1007/BF02761755.</LI><LI id="Teng2007">Teng, Grace; Papavasiliou, F. Nina (2007): Immunoglobulin somatic hypermutation. In: Annual review of genetics 41, S. 107–120. DOI: 10.1146/annurev.genet.41.110306.130340.</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>