Team:Bielefeld-CeBiTec/Proof
Toggle navigation
NavBar
PROJECT
Description
Library
Overview
Scaffolds
Design and Construction
Create your own Library
Mutation
Overview
Error prone Polymerase I
Genome wide mutator
Selection
Overview
Bacterial Two-Hybrid System
Alternative Selection System
Modeling
Overview
Model I
Model II
RESULTS
Overview
Library
Overview
Assembly
Phagemid Display
Sequencing
Mutation
Overview
Assembly
Reversion
Sequencing
Selection
Overview
Assembly
Expression Control
Interaction Control
Binding Control
in vivo Selection
Modeling
Fermentation
Parts
Best Basic Part
Best Composite Part
Best Part Collection
Improve a part
Proof of Concept
Demonstrate your Work
Medal Fulfillment
HUMAN PRACTICES
Overview
Engagement
Dictionary
Educational Outreach
Crowdfunding and Press
Entrepreneurship
Integrated Practices
Achievements
COLLABORATIONS
Overview
DÃ¼sseldorf
Lethbridge
Freiburg
NOTEBOOK
Overview
Lab Notebook
Library
Mutation
Selection
Primers
Lab Protocols
Media and Buffers
TEAM
Team Profile
Members
Bielefeld University
Contact
ACKNOWLEDGEMENT
Overview
Sponsors and Partners
Attributions
Proof of Concept
Life is like a mirror - we get the best results when we smile at it
Proof of Concept
We developed a novel in vivo system for generating binding proteins in E. coli via directed evolution. The concept of our system subdivides into a library for initial diversity, a system for mutagenesis to further increase the diversity and a selection system to identify high affinity binding proteins.
Library
At first, we designed and created two libraries of partly random binding protein sequences in E. coli based on Monobodies and Nanobodies , respectively. Each library reached a size of over one hundred thousand clones posing a solid start point of our system. High diversity of the library was confirmed by stade-of-the-art high-throughput sequencing . Applicability of our library was confirmed by finding multiple potential binders against diverse targets . This library poses a valuable resource for our following directed evolution. Moreover, we provide an unprecedented part collection to the whole iGEM community enabling the application of libraries in future projects .
Mutagenesis
Furthermore, we used an error prone polymerase I for targeted mutagenesis of the binding protein encoding plasmid. This system is well suited for the diversification of initial binding proteins derived from our library. This mutagenesis system was functionally characterized by various reversion experiments . Precise mutation rate and mutation types were analyzed via high-throuput sequencing . In contrast to genome-wide mutation system, our system is especially useful to future iGEM projects to apply mutagenesis of BioBricks within the standard plasmid.
Selection system
Finally, we used a bacterial two hybrid system to give cells with fitting binding proteins to the target protein an advantage in growth by developing an antibiotic resistance. Functionality of this system was demonstrated by several experiments, ranging expression controls over binding controls to interaction controls and in vivo tests. In summary, we provide a functional bacterial two hybrid system for other iGEM teams to work with.
In a nutshell, we provide evidence that all devices of our project work as expected . Moreover, we made it iteratively even better by modeling our system, integrating experts and public reviews . Industry scale applicability was indicated by continuous fermentation experiments and a business plan .
In particular, we reached all of our milestones: (1) We designed and created two functional libraries with high diversities, (2) we assembled a working system for plasmid-targeted mutagenesis and (3) we implemented a functional bacterial two hybrid selection system.
