Team:Bielefeld-CeBiTec/Project/Library/CreateYours
Toggle navigation
NavBar
PROJECT
Description
Library
Overview
Scaffolds
Design and Construction
Create your own Library
Mutation
Overview
Error prone Polymerase I
Genome wide mutator
Selection
Overview
Bacterial Two-Hybrid System
Alternative Selection System
Modeling
Overview
Model I
Model II
RESULTS
Overview
Library
Overview
Assembly
Phagemid Display
Sequencing
Mutation
Overview
Assembly
Reversion
Sequencing
Selection
Overview
Assembly
Expression Control
Interaction Control
Binding Control
in vivo Selection
Modeling
Fermentation
Parts
Best Basic Part
Best Composite Part
Best Part Collection
Improve a part
Proof of Concept
Demonstrate your Work
Medal Fulfillment
HUMAN PRACTICES
Overview
Engagement
Dictionary
Educational Outreach
Crowdfunding and Press
Entrepreneurship
Integrated Practices
Achievements
COLLABORATIONS
Overview
DÃ¼sseldorf
Lethbridge
Freiburg
NOTEBOOK
Overview
Lab Notebook
Library
Mutation
Selection
Primers
Lab Protocols
Media and Buffers
TEAM
Team Profile
Members
Bielefeld University
Contact
ACKNOWLEDGEMENT
Overview
Sponsors and Partners
Attributions
Library Project
Create Your Own Library
Implementing diversity
For our directed evolution system of binding proteins, we designed and constructed two libraries, containig proposed randomized CDS for Monobodies or Nanobodies as initial Evobodies. The detailed construction of our libraries is discribed in "Design and Construction" . Plasmid libraries, coding for useful proteins, are a powerful tool in synthetic biology, but underestimated in iGEM, yet. Unfortunately, the construction of a library is a difficult and time consuming task. To supply this tool for future iGEM teams, we provided our libraries to the iGEM community and want to implement a new category in iGEM parts registry for libraries, coding several different proteins categories, like enzymes, binding proteins or aptamers. By doing so we want to establish libraries in iGEM and pave the way for every team to access the advantages of large diversity libraries as well as creating and submitting their own.
As a starting point, we submitted our Monobody and Nanobody libraries as plasmid mixes. These collections comprise over 100,000 (Monobodies), respectively over 160,000 (Nanobodies) of distinct sequences for binding proteins out of a theoretically size of over one billion (1,073,741,824) possible sequences. Additionally, we constructed fundamental frameworks composed of the Monobody or Nanobody constant regions, which can easily be extended for creating a specific binding protein or even your own library, as it is discribed in the following text. For that you only had to design your specific variable regions and get them synthesized in one or more parts.
Create your Monobody library:
To create your own Monobody library, we constructed a fundamental framework ( BBa_K2082000 ) composed of the Monobody constant regions and RFP instead of variable regions, which can easily be exchanged with randomized variable segments. Due to this, an easy selection trough visual control is possible. To insert self-constructed variable regions, you have to design a randomized insert with fitting overlaps to the adjacent constant Monobody sequences (Fig. 1). To integrate them, by replacement of RFP, the backbone with constant parts ( BBa_K2082000 ) should be amplified by using the primers MB-bb-fw and MB-bb-rev . Afterwards the variable fragments, containing overlaps, can get inserted in the gained backbone by Gibson assembly.
Figure 1: Overview of the Monobody construction. An individual insert can get integrated in the Monobody CDS. Constant regions are colored in green, variable regions are colored in blue.
Create your Nanobody library:
For creation of a unique Nanobody library you can use our framework, containing the constant Nanobody segments ( BBa_K2082001 ). In this case, it is only necessary to create one variable segment, because in the used scaffold , we have only randomized the sequence of the third Complementarity-determining region (CDR), which is the main binding CDR . Amplifying BBa_K2082001 with primers NB-bb-fw and NB-bb-rev results to a backbone, to insert a double stranded and overlapping variable insert by Gibson assembly, to built a unique Nanobody library.
Figure 1: Overview of the Nanobody construction.Variable CDR 3 is colored in blue.
Implementing diversity
It would push the iGEM parts registry forward and help future iGEM teams, if it would be possible to submit several plasmid libraries, encoding different binding proteins, enzymes, aptamers, or other useful proteins. So all following iGEM teams can build on it and screen the diversity for their desired protein with custom properties. For that reason, it has to be possible, to enter bases of the IUPAC nucleotide code including explicitly random bases, e.g. M, W, X and Y as a part sequence.
To distribute the submitted Monobody and Nanobody libraries for further iGEM teams by the iGEM headquarter, it is necessary to conserve the library diversity as effectively as possible. For that, recloning by chemical or electro transformation is the wrong way. It would be the best, to multiply the plasmid mixes by PCR. Using the primers LIB-bb-fw and LIB-bb-rev . After distribution the operator can realign the linearized Plasmid by using biobrick or Gibson assembly. To ensure, a high yield of different bacterial clones it would be rational, to use high efficient competent cells to gain an optimal variability.
Results
