Team:Bielefeld-CeBiTec/Project/Mutation/EpPolI
Toggle navigation
NavBar
PROJECT
Description
Library
Overview
Scaffolds
Design and Construction
Create your own Library
Mutation
Overview
Error prone Polymerase I
Genome wide mutator
Selection
Overview
Bacterial Two-Hybrid System
Alternative Selection System
Modeling
Overview
Model I
Model II
RESULTS
Overview
Library
Overview
Assembly
Phagemid Display
Sequencing
Mutation
Overview
Assembly
Reversion
Sequencing
Selection
Overview
Assembly
Expression Control
Interaction Control
Binding Control
in vivo Selection
Modeling
Fermentation
Parts
Best Basic Part
Best Composite Part
Best Part Collection
Improve a part
Proof of Concept
Demonstrate your Work
Medal Fulfillment
HUMAN PRACTICES
Overview
Engagement
Dictionary
Educational Outreach
Crowdfunding and Press
Entrepreneurship
Integrated Practices
Achievements
COLLABORATIONS
Overview
Düsseldorf
Lethbridge
Freiburg
NOTEBOOK
Overview
Lab Notebook
Library
Mutation
Selection
Primers
Lab Protocols
Media and Buffers
TEAM
Team Profile
Members
Bielefeld University
Contact
ACKNOWLEDGEMENT
Overview
Sponsors and Partners
Attributions
Error Prone DNA Polymerase I
The mechanism of DNA replication takes place in the cell nucleus. The polymerases are responsible for the replication of the entire DNA. One of these polymerases is the DNA polymerase I, which plays a role in lagging-strand replication of chromosomal DNA. Moreover, the DNA polymerase I also has the task to do proofreading and plays a role in replicating ColE1 plasmids (Camps et. al 2003). ColE1 plasmids are characterized by the ColE1 origin of replication (ori). During the lagging-strand synthesis the DNA polymerase processes RNA primer (˜20 nt) and fills gaps during DNA repair reactions (Allen et. al 2011). Sometimes errors occur during the synthesis of a DNA strand. These mistakes may have big effects. They can lead to diseases or death of the organism, when a gene is damaged that is essential for survival. To reduce the frequency of mistakes the DNA polymerase I checks its work by using its own proofreading functionality.
EP-Pol I
Structure of EP-Pol I
To produce more diversity in our library we use an in vivo mutagenesis system, which is based on an error prone DNA polymerase I (EP-Pol I) (Troll et. al 2011). Thus, the mutagenesis system generates mutations during replication. It is designed as a two plasmid system and is beneficial to our approach due to the selectivity towards a specific plasmid. Only the ColE1 plasmid encoding the target protein sequence is mutated.
Prof. Dr. Manel Camps from the University of Santa Cruz investigated a DNA polymerase I, called EP-Pol I, which lacks in correct synthesis and in proof-reading (Camps et. al 2003). In contrast to the normal DNA polymerase I the EP-Pol I has three point mutations, I709N, A759R and D424A (Camps et. al 2003)(figure 1). The I709N mutation is located in motif A. This is a conserved sequence in the palm domain of the polymerase active site. The mutations lead to an enlargement of the substrate-binding pocket, which is a possible explanation for the increased error rate (Camps et. al 2003). The D424A mutation in the exonuclease domain leads to deactivation of the proofreading activity of the EP-Pol I. The amino acid replacements A759R is located in the O-helix, which is a conserved sequence (motif B) that is located close to the polymerase active site on dNTP binding. This may stabilize the enzyme with similar conformations, which leads to missed integration of nucleotides (Camps et. al 2003).
Figure 1: Structures of the klenow fragment of DNA polymerase I with the three substitutions. The blue one is I709N, red is A759R and orange is D424A (Beese et. al 2011).
Mutation
According to Allen et al. 2011, the EP-Pol I produces evenly distributed mutations and generates base pair substitutions (transitions) and transversions. Another advantage of the EP-Pol I is that it only replicates and thus mutates plasmids that have a ColE1 ori and that the EP-Pol I processes Okazaki fragments (Allen et. al 2011). Therefore, no significant increase in the mutation rate of the chromosomal DNA was observed (Troll et. al 2014). During replication, the EP-Pol I is replaced by DNA-Polymerase III. This leads to the conclusion that with increasing distance from the replacement point the frequency of EP-Pol I mutations decrease (Allen et. al 2011). In this paper, they also wrote that the EP-Pol I synthesizes 400 nt to 500 nt after the ori, but during a skype conversation with Manel Camps, he said that this is not the case. It seems that the mutations are randomly distributed over the plasmid and that only a few base pairs at the start are synthesized by DNA-Polymerase I. According to Alexander et. al (2014) the EP-Pol I generates more than one mutation per kb. The substitution of the four different bases has various probabilities (figure 1).
Figure 2: Base substition. The substitution frequencies for all bases made by the EP-Pol I are shown. Figure adapted from (Badran et. al 2015).
EP-Pol I system
For an efficient use of the EP-Pol I it is recommended to use the E. coli JS200 strain, that carries a knockout of the native Pol I as well as a temperature sensitive DNA polymerase I, which can compensate the knockout at a certain temperature (Camps et. al 2003). With the use of JS200 it is possible to switch between EP-Pol I (mutation phase) and DNA polymerase I (no mutation phase). At 30 °C the DNA polymerase I is active and synthesizes the plasmids, because it is more efficient and faster than the low fidelity EP-Pol I (Camps et. al 2003). When changing to 37 °C the DNA polymerase I is no longer active and the EP-Pol I begins to mutate the DNA sequence (Alexander et. al 2014).
To effectively occupy the EP-Pol I we use a two-plasmid system developed by Manel Camps (figure 3). One plasmid has a pSC101 ori (Pol I-independent) and the EP-Pol I sequence and the other plasmid has the ColE1 ori and our Evobody sequence. Therefore, only the plasmid with the Evobody sequence will be mutated. This is a directed evolution, because only the target DNA sequence and not all the available DNA is mutated (Alexander et. al 2014). By applying this system to the JS200 strain we are able to switch between a mutation and a no-mutation phase.
Figure 3: The two plasmid system. The EP-Pol I is expressed from the plasmid with the pSB101 ori and this replicate parts of the plasmid with the ColE1 plasmid.
Global mutagenesis
References
Alexander, David L.; Lilly, Joshua; Hernandez, Jaime; Romsdahl, Jillian; Troll, Christopher J.; Camps, Manel (2014): Random mutagenesis by error-prone pol plasmid replication in Escherichia coli. In: Methods in molecular biology (Clifton, N.J.) 1179, S. 31–44. DOI: 10.1007/978-1-4939-1053-3_3.
Allen, Jennifer M.; Simcha, David M.; Ericson, Nolan G.; Alexander, David L.; Marquette, Jacob T.; van Biber, Benjamin P. et al. (2011): Roles of DNA polymerase I in leading and lagging-strand replication defined by a high-resolution mutation footprint of ColE1 plasmid replication. In: Nucleic acids research 39 (16), S. 7020–7033. DOI: 10.1093/nar/gkr157..
Badran AH, 2015. Development of potent in vivo mutagenesis plasmids with broad mutational spectra. Nature Communications, 2015, 6, 8425.
Camps, Manel; Naukkarinen, Jussi; Johnson, Ben P.; Loeb, Lawrence A. (2003): Targeted gene evolution in Escherichia coli using a highly error-prone DNA polymerase I. In: Proceedings of the National Academy of Sciences of the United States of America 100 (17), S. 9727–9732. DOI: 10.1073/pnas.1333928100.
Troll, Chris; Alexander, David; Allen, Jennifer; Marquette, Jacob; Camps, Manel (2011): Mutagenesis and functional selection protocols for directed evolution of proteins in E. coli. In: Journal of visualized experiments : JoVE (49). DOI: 10.3791/2505.
Beese, L. S.; Friedman, J. M.; Steitz, T. A. (1993): Crystal structures of the Klenow fragment of DNA polymerase I complexed with deoxynucleoside triphosphate and pyrophosphate. In: Biochemistry 32 (51), S. 14095–14101.
Troll, Christopher; Yoder, Jordan; Alexander, David; Hernandez, Jaime; Loh, Yueling; Camps, Manel (2014): The mutagenic footprint of low-fidelity Pol I ColE1 plasmid replication in E. coli reveals an extensive interplay between Pol I and Pol III. In: Current genetics 60 (3), S. 123–134. DOI: 10.1007/s00294-013-0415-9.
