<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Integrated_Practices skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Integrated Practices</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container main"><DIV id="header"><NAV id="navigationbar" class="navbar navbar-default custom_navbar_style navbar-fixed-top"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="#">NavBar</A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-center"><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description"> PROJECT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description">Description</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds">Scaffolds</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design">Design and Construction</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/CreateYours">Create your own Library</A></LI></UL><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/EpPolI">Error prone PolymeraseI</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/Global">Genome wide mutator</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">Bacterial Two-Hybrid System</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/SplitBetaLactamaseHitchHiker">Alternative Selection System</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_I">Model I</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_II">Model II</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results"> RESULTS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Phage">Phagemid Display</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">Reversion</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/Cloning">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/ExpressionControl">Expression Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/InteractionControl">Interaction Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/BindingControl">Binding Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/KnockOutKnochIn"><I>in vivo</I> Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Modeling">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Fermentation">Fermentation</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Best Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Best Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Best Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts/Improve">Improve a part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Demonstrate your Work</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Medal_Fulfillment">Medal Fulfillment</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold"> HUMAN PRACTICES </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Dictionary">Dictionary</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks">Educational Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Crowdfunding">Crowdfunding and Press</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Achievements</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations"> COLLABORATIONS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Duesseldorf">Düsseldorf</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Lethbridge">Lethbridge</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Freiburg">Freiburg</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments"> NOTEBOOK </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook">Lab Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">Primers</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols">Lab Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Media">Media and Buffers</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team"> TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team">Team Profile</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Members">Members</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Contact">Contact</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement"> ACKNOWLEDGEMENT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Sponsors_And_Partners">Sponsors and Partners</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></DIV></DIV></NAV></DIV></DIV><DIV class="container main"><DIV class="jumbotron" style="background-image: url(https://static.igem.org/mediawiki/2016/b/ba/Bielefeld_CeBiTec_2016_10_19_X_hp.png)"><DIV class="jumbotron-text"><H1 style="margin-bottom: 0px; text-align:left">Integrated Human Practices</H1></DIV></DIV><DIV class="container text">
During the conceptualization of our project, we contacted several experienced researches and leading persons in the industry. Provided information was especially helpful for the construction of the selection system. Several important advices were collected by contacting experts on various fields per email. Unfortunately, most of the contacted persons did not respond. For the identification of the optimal mutagenesis system for plasmids in our project, we contacted <A href="#camps">Manel Camps</A>, who is the leading expert on this topic. To get in contact with additional experts from project associated field, we attended the <A href="#symposium">CeBiTec Symposium</A> about medical bioinformatics.

Different approaches were applied to communicate synthetic biology and our project efficiently to the public. One example is our participation in the <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks#postcards">post card action</A> initiated by the team from Düsseldorf. We identified critical issues within the process during our participation in an industrial meet-up, the <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks#nrw">NRW day</A>, the <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks#academy">pupils academy</A> and the discussion with an anthropologist. To optimize our presentation skills we started an evolution-based project. Lectures were given in front of <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks#schoolPresentation">high school students</A> and the communication of information was tested afterwards. Through several iterations, we were able to increase the transferred knowledge about synthetic biology and our project significantly.

</DIV><DIV class="container text_header"><H3 id="symposium">CeBiTec Symposium &quot;Microbial Genomics and Metagenomics in Human Health and Disease&quot;</H3></DIV><DIV class="container text"> From July 4th to 6th we had the chance to present our project to scientific audience from all over Germany and Europe at the <A href="http://www.cebitec.uni-bielefeld.de/index.php/events/conferences/459-2016-04-07-11th-cebitec-symposium">11th CeBiTec symposium</A>. The conference with the topic &quot;Microbial Genomics and Metagenomics in Human Health and Disease&quot; occurs at the &quot;Center of Interdisciplinary Research&quot; of Bielefeld University. Among many interesting lectures and workshops, there was a poster session and much time for the scientific exchange. We were proud to have the opportunity to show our newly designed poster (figure 1) to many interested visitors. Participating experts from the field of biology and medicine provided us with a lot of useful tips and their opinions about our project. 
</DIV><FIGURE class="figure"><FIGCAPTION><B>Figure 1</B>: Poster for the poster presentation</FIGCAPTION></FIGURE><FIGURE class="figure"><FIGCAPTION><B>Figure 2</B>: Poster presentation</FIGCAPTION></FIGURE><DIV class="container text_header"><H3><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Dictionary">Dictionary</A></H3></DIV><DIV class="container text">	
As we got several understand problems with non-researchers and researchers, we started to write a <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Dictionary">dictionary</A>. Thus we had to think about how to explain technical terms in understanding words. Writing the dictionary helped us to deal with difficult words. Furthermore we hope that our dictionary will help other iGEM teams, when they are need to explain their project easily.
</DIV><DIV class="container text_header"><H3><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks">Educational Outreach</A></H3></DIV><DIV class="container text">	
With writing our dictionary our journey for an easier understandable project wasn’t over. We also started <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks#schoolPresentation">“Directed Evolution of Educational Outreach”</A>, where we went to several schools presenting our project and iteratively improve our project explanation. Also we taught some <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks#schoolPresentation">pupils</A> about synthetic biology and learned what difficulties in understanding basic research experiments have non-researchers to overcome with. Our last hurdle were the <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks#postcards">post card action</A> and the <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks#nrw">NRW day</A>. During the postcard action we had to find a way to explain synthetic biology to everyone and that on a 150 cm<SUP>2</SUP> tiny piece of paper. On the <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks#nrw">NRW day</A> we presented our project to the public and tried to explain synthetic biology. During that day we distributed our postcards.
</DIV><DIV class="container text_header"><H3 id="experts">Expert Contacts</H3></DIV><DIV class="container text">
To adjust our project to the needs of the public and industry, we contacted several experts. Some experts are working in the field of diagnostics and some devote with <I>in vivo</I> mutagenesis.
</DIV><DIV class="container text_header"><H4>Searching for a target</H4></DIV><DIV class="container text">
Searching for an application field was a difficult task do deal with. Our system is designed for a wide field. But since we need an issue to focus on and optimizing our system to, we asked several experts for their opinion and a review to our mutagenesis system. 
<A href="https://www.virologie.med.tum.de/en/research-groups-tum/ulrike-protzer/">Prof Dr. Med. Ulrike Protzer</A>, professor at the technical university in Munich, director of the institute of virology and currently working at the Helmholtz center Munich (German center for environmental health), said, that she could imagine an application in the virology. We could develop a diagnostic assay, since the production of our Evobodies in <I>E. coli</I> would be very low cost. Thus, we could provide a cheap diagnostic for third world countries like Africa. 

Something equally said <A href="https://www.klinikum.uni-heidelberg.de/AG-Bartenschlager.104953.0.html">Prof. Dr. Ralf Bartenschlager</A>, professor at the university of Heidelberg and member of the department of infectious diseases, who mentioned emerging viruses. Emerging viruses are viruses that occur suddenly. Most of them are unknown or difficult to find. They can spread in no time. Good examples  are the Zika virus, the MERS-Corona virus, the Ebola virus or newly evolved influenza viruses. As a basic researcher, he thinks that our method can perfectly complement modern –omics methods by providing an antibody.
Further topics that are mentioned by other experts can be seen in the following figure 3:
</DIV><CENTER><FIGURE><FIGCAPTION><B>Figure 3</B>: Overview of expert answers.</FIGCAPTION></FIGURE></CENTER><DIV class="container text">
To be able to use the mutagenesis system as a detection system we wanted to provide our reporter gene with a fluorescent dye. Thus, we were able to detect a cell that produce a suitable Evobody. We could measure the strength of the fluorescence or the resistance level to conclude to the affinity of our target and Evobody. 

Experts told us that we must consider the restricted protein folding in E. coli, and look out for disulfide bonds we decided to use Monobodies and Nanobodies in our library. Proteins with disulfide bonds mostly cannot be folded correctly in E. coli and therefore does not work with our <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">bacterial two-hybrid system</A>. Additionally, we planned to use our mutagenesis system as an antibody refinery like <A href="https://www.klinikum.uni-heidelberg.de/AG-Bartenschlager.104953.0.html">Prof. Dr. Bartenschlager</A> advocated. Thus, we were producing antibody mimetics and provided their sequence to other researchers.
</DIV><DIV class="container text_header"><H4 id="camps">Correspondence with Manel Camps </H4></DIV><DIV class="container text"><A href="http://www.metx.ucsc.edu/research/camps.html">Prof Dr. Manel Camps</A> from the University of Santa Cruz is the leading expert concerning the use of the error prone polymerase I (EP-Pol I). He designed the two-plasmid system (Camps et. al 2003) and further characterized it over the years (Camps et. al 2003; Camps and Loeb 2005; Allen et. al, 2011; Troll et. al, 2011; Troll et. al 2014; Alexander et. al 2014). Since we developed an in vivo mutagenesis system, based on the EP-Pol I, we contacted Mr. Camps. During E-Mail correspondence and Skype conversation he reviewed our concept and helped us to integrate the EP-Pol I to the most effect in our project. Additionally, Mr. Camps kindly provided us the EP-Pol I next to different controls we used for characterisation of our mutagenesis parts.

According to our correspondence with Mr. Camps we adapted our project accordingly. 

One modification is the positioning of our target sequence on the plasmid. First, we wanted to place the target sequence behind the origin of replication, due to older information speaking of an increased mutation rate closely after the ColE1-ori (Camps et. al 2003). In our Skype conference Mr. Camps told us that the mutation is distributed evenly across the whole ColE1-plasmid with exception of the first very short region behind the origin of replication and regions of processed okazaki primers. 
After this information we decided to drop our plan to move the BioBrick site directly behind the ColE1-ori and continued using the iGEM pSB1-series.

Finally, we asked Mr. Camps about an estimation of our concept. He said that our approach to use the EP-Pol I as the in vivo mutation machinery is a very excellent idea. Other in vivo approaches like the common used XL1 red system are not as efficient as mutating with the EP-Pol I in terms of a balanced mutation spectrum. But more importantly the EP-Pol I gives us control about the mutation target. Since the EP-Pol I mostly targets ColE1 plasmids, we protect the chromosome and thus house-keeping genes from being mutated. That is a huge advantage compared to common in vivo mutagenesis approaches.

Additionally, for further questions after the conversation round, we stayed in e-mail contact with Mr. Camps and asked him when problems occured.

Because our determination of the mutation rate does not confirm the rate written in the literature, we asked for possible explanations. He stressed out the importance to start the mutagenesis directly after transformation. Furthermore it could be advantageous to place the cultures under replicative stress by letting them grow for several days until reaching hypersaturation. Thus, the cultures would experience saturate mismatch repair.

Furthermore, we corresponded with Mr. Camps about the determination of the mutation rate and our idea to use high-throughput sequencing for this issue. He said that next generation sequencing would be a good experiment, especially to show where the mutational hotspots of the EP-Pol I lies. Therefore, we designed our high-throughput sequencing experiment.

We were also interested in the exact functionality of the chromosomal temperature-sensitive DNA polymerase I from E. coli JS200. We wanted to know whether he already tested if and how many mutations occur at the permissive temperature while still containing the plasmid encodes error-prone polymerase I. Mr. Camps estimates the mutagenesis rate under these conditions to be about 25% of the mutagenesis performance at 37°C. 

At last, we wanted to know Mr. Camps opinion of our idea to change the expression rate of the EP-Pol I. He responded that after reaching a certain threshold the mutagenesis rate would not be improved anymore. This means that raising the expression strength would not change the mutagenesis rate.
</DIV><DIV class="container text_header"><H3>Upscaling, Process Development and Business Plan</H3></DIV><DIV class="container text">
As the future prospects of our project were of great interest to us we consulted several experts (Prof. Dr. Karl Friehs, Dr. Joe Max Risse, Prof. Dr. Thomas Noll, Dr. Martin Schleef) on the topic to effectively upscale our project. Together we discussed what has to be done to implement our system into a market environment especially which <A href="2016.igem.org/Team:Bielefeld-CeBiTec/Entrepreneurship">aspects</A> had to be considered in detail. The result was a business plan that in detail describes the measurements to be taken to found and effectively run a company. This also involves a summary of factors we decided on after carefully discussing different possibilities and methods with the consulted experts. Thus we collected all information considered vital for a company to be competitive on the market.
</DIV><FIGURE class="figure"><FIGCAPTION><B>Figure 4</B>: Flowchart of the main process.</FIGCAPTION></FIGURE><DIV class="container text_header"><H3>References</H3></DIV><DIV class="container text"><UL><LI>Alexander, David L.; Lilly, Joshua; Hernandez, Jaime; Romsdahl, Jillian; Troll, Christopher J.; Camps, Manel (2014): Random mutagenesis by error-prone pol plasmid replication in Escherichia coli. In: Methods in molecular biology (Clifton, N.J.) 1179, S. 31–44. DOI: 10.1007/978-1-4939-1053-3_3.</LI><LI>Allen, Jennifer M.; Simcha, David M.; Ericson, Nolan G.; Alexander, David L.; Marquette, Jacob T.; van Biber, Benjamin P. et al. (2011): Roles of DNA polymerase I in leading and lagging-strand replication defined by a high-resolution mutation footprint of ColE1 plasmid replication. In: Nucleic acids research 39 (16), S. 7020–7033. DOI: 10.1093/nar/gkr157.</LI><LI>Camps, Manel; Loeb, Lawrence A. (2005): Critical role of R-loops in processing replication blocks. In: Frontiers in bioscience : a journal and virtual library 10, S. 689–698.</LI><LI>Camps, Manel; Naukkarinen, Jussi; Johnson, Ben P.; Loeb, Lawrence A. (2003): Targeted gene evolution in Escherichia coli using a highly error-prone DNA polymerase I. In: Proceedings of the National Academy of Sciences of the United States of America 100 (17), S. 9727–9732. DOI: 10.1073/pnas.1333928100.</LI><LI>Troll, Chris; Alexander, David; Allen, Jennifer; Marquette, Jacob; Camps, Manel (2011): Mutagenesis and functional selection protocols for directed evolution of proteins in E. coli. In: Journal of visualized experiments : JoVE (49). DOI: 10.3791/2505.</LI><LI>Troll, Christopher; Yoder, Jordan; Alexander, David; Hernandez, Jaime; Loh, Yueling; Camps, Manel (2014): The mutagenic footprint of low-fidelity Pol I ColE1 plasmid replication in E. coli reveals an extensive interplay between Pol I and Pol III. In: Current genetics 60 (3), S. 123–134. DOI: 10.1007/s00294-013-0415-9.</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>