Team:Bielefeld-CeBiTec/Collaborations/Lethbridge
Toggle navigation
NavBar
PROJECT
Description
Library
Overview
Scaffolds
Design and Construction
Create your own Library
Mutation
Overview
Error prone Polymerase I
Genome wide mutator
Selection
Overview
Bacterial Two-Hybrid System
Alternative Selection System
Modeling
Overview
Model I
Model II
RESULTS
Overview
Library
Overview
Assembly
Phagemid Display
Sequencing
Mutation
Overview
Assembly
Reversion
Sequencing
Selection
Overview
Assembly
Expression Control
Interaction Control
Binding Control
in vivo Selection
Modeling
Fermentation
Parts
Best Basic Part
Best Composite Part
Best Part Collection
Improve a part
Proof of Concept
Demonstrate your Work
Medal Fulfillment
HUMAN PRACTICES
Overview
Engagement
Dictionary
Educational Outreach
Crowdfunding and Press
Entrepreneurship
Integrated Practices
Achievements
COLLABORATIONS
Overview
Düsseldorf
Lethbridge
Freiburg
NOTEBOOK
Overview
Lab Notebook
Library
Mutation
Selection
Primers
Lab Protocols
Media and Buffers
TEAM
Team Profile
Members
Bielefeld University
Contact
ACKNOWLEDGEMENT
Overview
Sponsors and Partners
Attributions
Collaborations
A joy that's shared is a joy made double
Collaboration - Lethbridge
Summary
Discussed practical obstancles in creation of a binding protein library
Compared the different approaches of the bacterial two-hybrid system
Bielefeld provided Lethbridge with positive control in various strength for characterization of their two hybrid system
Lethbridge send the influenza A NS1 peptide as a potential target for generation of Evobodies
Skype Protocol: Lethbridge - Bielefeld
Protocol from 2016-08-26
Participants Lethbridge: Andrew, Taylor, Rhys
Paricipants Bielefeld: Marius, Sebastian, Carsten, Boas
Each team shortly describes their projects
Lethbridge
Library: Nanobodies
bioinformatic approach to predict good binding amino acids
theoretical variability of ~1018 possible sequences accounting for CDR1, CDR2 and CDR3, however only ~105-6 will be sampled during screening.
Screen via bacterial two hybrid
Nanobody fused to α-subunit of RNA-polymerase on plasmid 1
target protein fused to λcI-protein on plasmid 2
readout via mRFP1 expression relative to mTagBFP expression and cell sorting
system inspired by the invitrogen BacterioMatch II
Target
Collaboration with local EMS to investigate microbiome in ambulence vehicle
Identified microbes via DNA isolation of swabs from various points in ambulences and Nanopore next generation sequencing.
Important pathogens identified in sequencing or other studies selected for antibody production via bacteria-two-hybrid selection.
Bielefeld
Library of nano- and monobodies
randomization of CDR3-loop (nanobodies) respectively three loops (monobodies)
theoretical variability of 109 binding proteins
Error prone mutation system to further increase the variability
Screen via bacterial two hybrid
binding protein fused to ω-subunit of RNA-polymerase on plasmid 1
target protein fused to 434cI on plasmid 2
selection via β-lactamase and cell survival (RFP expression as control)
Target
contacted experts with respect to possible targets
viral envelope proteins (e.g. zika)
cellular proteins
Cooperation plans were discussed
Exchange of libraries with detailed cloning strategy
Exchange of controls for the bacterial two hybrid system
Mutual validation of the bacterial two hybrid
Protocol from 2016-08-26
Participants Lethbridge: Andrew, Rhys, Graeme
Paricipants Bielefeld: Marius, Pascal, Carsten, Boas
Each team describes their progress and current obstacles
Lethbridge
Library
complete randomisation of CDR loops
very high library size (Nanopore sequencing)
transformation efficiency is limiting (~40,000 clones)
transformation strategy: electroporation
Two-Hybrid
fluorescent protein gBlock contains mutations => use of streptomycin as reporter and selection via survivability
measurement of the basal activity of the reporter (weak constitutive promoter)
genomic integration of reporter gene is the target => only one copy per cell would be nice => would further decrease basal expression level
start selection cycles directly with target protein
Target: domain III of Zika E-protein
general design
two plasmid analog to the Agilent BacterioMatch II Two-Hybrid System
initial plan: use two reporters for copy number control
monobodies still contain S-S bridges
Mutual discussion about the molecular mechanism of the two hybrid
Question: Doesn't a strong antibody-mimetic - target protein affinity hold the RNA-Pol back?
No literature addressing this issue was found
Possible explanations
protein interaction is broken by RNA-Pol
DNA is pulled through the enzyme which stays at the promoter position
Bielefeld
Library
randomisation strategy (leads to 109 variants each)
high quality library by using only specifi amino acids
relative small library size => potential to use a considerable amount of the whole library
transformation strategy: electroporation, large agar plates
Two-Hybrid
characterization of the two hybrid system by using the HA4 - SH2 binding pair in addition to mutants with decreasing strength
=> correlation between binding strength and two hybrid output
talked about the design of the two hybrid reporter site
modified promoter for very low basal expression
optimized distance between cI binding site and promoter
start selection cycles directly with target protein
Target: domain III of Zika E-protein
general design
two plasmid analog to the Agilent BacterioMatch II Two-Hybrid System
initial plan: use two reporters for copy number control
monobodies still contain S-S bridges
Protocol from 2016-09-29
Participants Lethbridge: Andrew
Paricipants Bielefeld: Marius, Pascal, Carsten
Current status of each project
Lethbridge
Library
Optimization of competent cells with cfu ~ 108 cfu/μg
currently 40.000 different clones
confirmed by deep sequencing
Two-Hybrid
constitutive fluorescence A as control of plasmid copy number
bacterial two hybrid activatable fluorescence B as signal of the bacterial two hybrid
currently no strong increase of fluorescence B detectable
Bielefeld
Library
variable regions are cloned into the scaffolds
NGS of the variable regions is planned
Two-Hybrid
devices are completely assembled but no reporter activity is detected
control of the single parts of the two hybrid system
control of the DNA binding of cI
which cI OR's does Lethbridge use? suggestion that lethbridge checks which OR’s they use for λcI and maybe add missing OR's
change of 434 cI to the partsreg λcI + binding site
control of the positive control binding by native PAGE and affinity chromatography
Cooperation plans were discussed
Exchange of libraries with detailed cloning strategy
Exchange of controls for the bacterial two hybrid system
Mutual validation of the bacterial two hybrid
Discussion points
modelling of complete bacterial two hybrid on protein structure level
=> could yield information about the optimal distance between cI binding site and promoter
control of the bacterial two hybrid system
positive controls in the HA4-SH2 binding pair from Bielefeld
direct fusion between cI and the RNA-Pol subunit as a positive control?
GFP inbetween as expression control?
target
Lethbridge uses the domain III of zika E protein as target
Bielefeld uses the complete zika E protein as target
