Team:Bielefeld-CeBiTec/Project/Selection/SplitBetaLactamaseHitchHiker
Toggle navigation
NavBar
PROJECT
Description
Library
Overview
Scaffolds
Design and Construction
Create your own Library
Mutation
Overview
Error prone Polymerase I
Genome wide mutator
Selection
Overview
Bacterial Two-Hybrid System
Alternative Selection System
Modeling
Overview
Model I
Model II
RESULTS
Overview
Library
Overview
Assembly
Phagemid Display
Sequencing
Mutation
Overview
Assembly
Reversion
Sequencing
Selection
Overview
Assembly
Expression Control
Interaction Control
Binding Control
in vivo Selection
Modeling
Fermentation
Parts
Best Basic Part
Best Composite Part
Best Part Collection
Improve a part
Proof of Concept
Demonstrate your Work
Medal Fulfillment
HUMAN PRACTICES
Overview
Engagement
Dictionary
Educational Outreach
Crowdfunding and Press
Entrepreneurship
Integrated Practices
Achievements
COLLABORATIONS
Overview
Düsseldorf
Lethbridge
Freiburg
NOTEBOOK
Overview
Lab Notebook
Library
Mutation
Selection
Primers
Lab Protocols
Media and Buffers
TEAM
Team Profile
Members
Bielefeld University
Contact
ACKNOWLEDGEMENT
Overview
Sponsors and Partners
Attributions
Selection
Alternative Selection System
Overview
While our main approach, the bacterial two-hybrid system , is the optimal approach for our selection, we proposed two alternative systems to apply positive selective pressure towards generation of functional Evobodies. In contrast to the bacterial two-hybrid system which relies on the expression of a selection marker, these methods work independent of transcriptional regulation. To screen bacteria in vivo for protein-protein interaction we proposed the usage of a split-protein system. For our split-protein selection system, we focused on two major kinds of split-protein selection systems that are both working with fusion proteins. The first one, later referred to as “split-protein system”, is realised by splitting a fully functional protein in two parts, fusing one half to our binding protein and the other half to our target protein. The second approach is known as “Hitchhiker-system”. It is based on a beta-lactamase, that is missing a transport signal, fused to the target protein and a transport signal fused to the binding protein.
Choice of Antibiotic Resistance
As a selective marker gene we chose the ampicillin resistance gene (amp) coding for TEM-1 beta-lactamase. Beta-lactamases are well-studied resistance enzymes of gram-negative bacteria against beta-lactam antibiotics. Beta-lactam antibiotics inhibit the synthesis of the peptidoglycan layer in the periplasm and therefore the proliferation of bacteria. If the beta-lactamase is present in the periplasm it can hydrolyze the beta-lactam ring, rendering the antibiotic useless (Wilke et al. 2005). A benefit of beta-lactamase as a resistance for our selection is, that it can grant resistance even at a low expression rate (Matagne et al. 1998). Therefore, even Evobodies with a low affinity to the target protein might be able to grow on low concentrations of ampicillin. In a process of continuously increasing the selective pressure, the chances of acquiring a functional Evobody is a lot better if low affinity Evobody bacteria are not eliminated instantly but instead are granted the possibility to adapt the Evobody to the target protein. Apart from our projects primary selection, the bacterial two-hybrid system , we proposed two additional methods to screen for binding Evobodies.
Split-Protein System
Figure 2: Illustration of the split-protein system. Only in case of the Evobody showing affinity towards the target protein the functionality of the split beta-lactamase(10) is restored, granting a ampicillin resistance to the bacterium that generates the Evobody.
The split-protein system works, like the name indicates, with two parts of a former functional protein, which are fused to the two proteins that should be screened for an interaction. A lot of different proteins are able to be split into half and to reassemble when fused to two interacting proteins (Ladant, Karimova 2000). By using a visual reporter like the green fluorescent protein or an selection marker like a beta-lactamase, the split-protein selection system can work as an indicator to screen cultures for protein-protein interaction (Shekhawat, Ghosh 2011). In our case however, we utilized it as an in vivo selection system (Porter, JR et al. 2007) to give a growth advantage to those bacteria producing the best Evobody.
The split-protein approach relies on the Evobody and the target protein each forming a translational fusion with one half of the split-protein, respectively. It is important to design the split-protein in a way that each half by itself is not functional. As proposed in literature, we dissected the beta-lactamase between Gly196 and Leu198, deleting 197. This site is supposed to be across from the active site of the enzyme while dividing it into fragments of similar size (Galarneau et al. 2002). In case of successful binding between an Evobody and the target protein, the two halves of the beta-lactamase should be sterically close enough to restore its functionality, thus granting ampicillin resistance to the host bacterium.
Hitchhiker-System
Same as the split-protein system, the hitchhiker-system is used to study protein-protein interactions. But instead of splitting the protein itself the TEM beta-lactamase without any natural export signal is fused to one binding protein while a signal peptide for the Tat (Twin-Arginine Translocation) pathway is fused to the other protein. In case of successful binding between the two proteins, the beta-lactamase is exported via the Tat pathway into the periplasm where it provides a functional antibiotic resistance. The Tat hitchhiker selection also enacts as a control for the protein folding, as only correctly folded proteins are exported (Waraho, DeLisa 2009).
Alongside the secretory pathway (Sec pathway), The twin-arginine translocation pathway (Tat pathway) is used in E. coli to export proteins through the membrane. In contrast to the Sec pathway, only folded proteins are exported via the Tat pathway. The name of the Tat pathway is due to the twin-arginine leader motif (S/TRRXFLK) of the signal peptide. Responsible for the translocation in E. coli are four cytoplasmic membrane proteins: TatA, TatB, TatC and TatE, where TatE appears to be homologous to TatA (Natale et al. 2008). However, the exact functionality has not been completely researched yet.  The signal peptide used for the hitchhiker export originates from pre-trimethylamine N-oxide (TMAO) reductase (TorA). This enzyme is known to be exported by the Tat translocase in E. coli. While this signal is fused to the Evobody, a TEM-1 beta-lactamase with a deleted natural secretion signal is fused to the target protein. If the Evobody is binding the target protein and the proteins are folded correctly, the complex is exported into the periplasm due to the signal peptide, which is removed during the process (Blaudeck et al. 2001). That way, only bacteria with functional Evobody are able to export the beta-lactamase into the periplasm where it operates as a resistance to ampicillin.
Results
References
Blaudeck, N.; Sprenger, G. A.; Freudl, R.; Wiegert, T. (2001): Specificity of signal peptide recognition in tat-dependent bacterial protein translocation. In Journal of bacteriology 183 (2), pp. 604–610. DOI: 10.1128/JB.183.2.604-610.2001.
Galarneau, Andre; Primeau, Martin; Trudeau, Louis-Eric; Michnick, Stephen W. (2002): Beta-lactamase protein fragment complementation assays as in vivo and in vitro sensors of protein protein interactions. In Nature biotechnology 20 (6), pp. 619–622. DOI: 10.1038/nbt0602-619.
Ladant, Daniel; Karimova, Gouzel (2000): Genetic systems for analyzing protein–protein interactions in bacteria. In Research in Microbiology 151 (9), pp. 711–720. DOI: 10.1016/S0923-2508(00)01136-0.
Natale, Paolo; Bruser, Thomas; Driessen, Arnold J. M. (2008): Sec- and Tat-mediated protein secretion across the bacterial cytoplasmic membrane--distinct translocases and mechanisms. In Biochimica et biophysica acta 1778 (9), pp. 1735–1756. DOI: 10.1016/j.bbamem.2007.07.015.
Porter, JR; Stains, C. I.; Segal, D. J.; Ghosh, I. (2007): Split beta-lactamase sensor for the sequence-specific detection of DNA methylation. In Analytical chemistry 79 (17), pp. 6702–6708. DOI: 10.1021/ac071163.
Shekhawat, S. S.; Ghosh, I. (2011): Split-Protein Systems: Beyond Binary Protein-Protein Interactions. In Current opinion in chemical biology 15 (6), pp. 789–797. DOI: 10.1016/j.cbpa.2011.10.014.
Waraho, Dujduan; DeLisa, Matthew P. (2009): Versatile selection technology for intracellular protein-protein interactions mediated by a unique bacterial hitchhiker transport mechanism. In Proceedings of the National Academy of Sciences of the United States of America 106 (10), pp. 3692–3697. DOI: 10.1073/pnas.0704048106.
Publication bibliography Blaudeck, N.; Sprenger, G. A.; Freudl, R.; Wiegert, T. (2001): Specificity of signal peptide recognition in tat-dependent bacterial protein translocation. In Journal of bacteriology 183 (2), pp. 604–610. DOI: 10.1128/JB.183.2.604-610.2001.
Galarneau, Andre; Primeau, Martin; Trudeau, Louis-Eric; Michnick, Stephen W. (2002): Beta-lactamase protein fragment complementation assays as in vivo and in vitro sensors of protein protein interactions. In Nature biotechnology 20 (6), pp. 619–622. DOI: 10.1038/nbt0602-619.
Ladant, Daniel; Karimova, Gouzel (2000): Genetic systems for analyzing protein–protein interactions in bacteria. In Research in Microbiology 151 (9), pp. 711–720. DOI: 10.1016/S0923-2508(00)01136-0.
Natale, Paolo; Bruser, Thomas; Driessen, Arnold J. M. (2008): Sec- and Tat-mediated protein secretion across the bacterial cytoplasmic membrane--distinct translocases and mechanisms. In Biochimica et biophysica acta 1778 (9), pp. 1735–1756. DOI: 10.1016/j.bbamem.2007.07.015.
Porter, JR; Stains, C. I.; Segal, D. J.; Ghosh, I. (2007): Split beta-lactamase sensor for the sequence-specific detection of DNA methylation. In Analytical chemistry 79 (17), pp. 6702–6708. DOI: 10.1021/ac071163.
Shekhawat, S. S.; Ghosh, I. (2011): Split-Protein Systems: Beyond Binary Protein-Protein Interactions. In Current opinion in chemical biology 15 (6), pp. 789–797. DOI: 10.1016/j.cbpa.2011.10.014.
Waraho, Dujduan; DeLisa, Matthew P. (2009): Versatile selection technology for intracellular protein-protein interactions mediated by a unique bacterial hitchhiker transport mechanism. In Proceedings of the National Academy of Sciences of the United States of America 106 (10), pp. 3692–3697. DOI: 10.1073/pnas.0704048106.
Wilke, Mark S.; Lovering, Andrew L.; Strynadka, Natalie C. J. (2005): Beta-lactam antibiotic resistance: a current structural perspective. In Current opinion in microbiology 8 (5), pp. 525–533. DOI: 10.1016/j.mib.2005.08.016.
Matagne A, Lamotte-Brasseur J, Frère JM. Catalytic properties of class A beta-lactamases: efficiency and diversity. Biochemical Journal. 1998;330(Pt 2):581-598.
