<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Project_Modeling_Model_II skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Project/Modeling/Model II</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container main"><DIV id="header"><NAV id="navigationbar" class="navbar navbar-default custom_navbar_style navbar-fixed-top"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="#">NavBar</A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-center"><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description"> PROJECT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description">Description</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds">Scaffolds</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design">Design and Construction</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/CreateYours">Create your own Library</A></LI></UL><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/EpPolI">Error prone PolymeraseI</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/Global">Genome wide mutator</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">Bacterial Two-Hybrid System</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/SplitBetaLactamaseHitchHiker">Alternative Selection System</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_I">Model I</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_II">Model II</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results"> RESULTS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Phage">Phagemid Display</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">Reversion</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/Cloning">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/ExpressionControl">Expression Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/InteractionControl">Interaction Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/BindingControl">Binding Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/KnockOutKnochIn"><I>in vivo</I> Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Modeling">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Fermentation">Fermentation</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Best Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Best Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Best Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts/Improve">Improve a part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Demonstrate your Work</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Medal_Fulfillment">Medal Fulfillment</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold"> HUMAN PRACTICES </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Dictionary">Dictionary</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks">Educational Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Crowdfunding">Crowdfunding and Press</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Achievements</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations"> COLLABORATIONS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Duesseldorf">Düsseldorf</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Lethbridge">Lethbridge</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Freiburg">Freiburg</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments"> NOTEBOOK </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook">Lab Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">Primers</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols">Lab Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Media">Media and Buffers</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team"> TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team">Team Profile</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Members">Members</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Contact">Contact</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement"> ACKNOWLEDGEMENT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Sponsors_And_Partners">Sponsors and Partners</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></DIV></DIV></NAV></DIV></DIV><DIV class="container main"><DIV class="jumbotron" style="background-image: url(https://static.igem.org/mediawiki/2016/8/89/Bielefeld_CeBiTec_2016_10_14_X_jessi2.png)"><DIV class="jumbotron-text"><H1 style="margin-bottom: 0px; text-align:left">ModelII</H1><H2 style="color:#ffffff; text-align:left">Competition model </H2></DIV></DIV><DIV class="container text">
Raising different adapted <I>E. coli</I> cultures together in one flask, one can observe that they compete for the limited space and nutrients. To determine how much better one <I>E. coli</I> culture must be adapted in comparison to its rivals and how long our system takes to produce a winner <I>E. coli</I> culture, we decided to model the competitive behavior.

The growth behavior of one <I>E. coli</I> culture can be modeled with a logistic growth function (Bacaër 2011):
</DIV><DIV class="container text">
were <I>x</I> stands for the population [OD],
<I>r</I> for the reproduction rate [h<SUP>-1</SUP>] and
<I>C</I> for the maximal capacity [OD].
 
 For different adapted <I> E. coli</I> cultures equation (15) must be modified to:
</DIV><DIV class="container text">
where <I>x<SUB>1</SUB>,...,x<SUB>n</SUB></I> stands for the different adapted <I>E. colis</I> and
<I>r<SUB>1</SUB>,...,r<SUB>n</SUB></I> for the different reproduction rates of the different adapted <I>E. colis</I>.

The growth behavior of one adapted <I>E. coli</I> culture depends on its specific reproduction (<I>r<SUB>1</SUB>,..,r<SUB>n</SUB></I>) rate. But one adapted <I> E. coli </I> culture can only reach the maximal capacity <I>C</I>. Thus, the change in growth also depends on the amount of free capacity ((<I>C</I>-(<I>x<SUB>1</SUB>+...+x<SUB>n</SUB></I>))/<I>C</I>).  
</DIV><DIV class="container text_header"><H2>Experimental validation</H2></DIV><DIV class="container text">
A theoretically implemented model must always be validated with experimentally measured data. To improve model II we did a row of cultivation experiments. One part of the cultivation experiments was used for fitting the growth parameter <I>r</I>, and separate experiments were used for validating the model.
</DIV><DIV class="container text_header"><H3>Cultivation experiments</H3></DIV><DIV class="container text">
For the determination of the experimental growth rate and for the production of a standard growth curve of the JS200 strain with an active error prone polymerase I we cultivated JS200 for around 18 hours at 37°C. We also wanted to investigate how a JS200 cell grows and, thus, beta-lactamase reacts, when it encounters different ampicillin concentrations. Therefore, the cultures were provided with a plasmid that carries a beta-lactamase sequence. Additionally, we let the cultures grow with 1, 3, 5 and 10 g ampicillin per liter, respectively (figure 1 (A)). As a control we also cultivated JS200 with an active wild-type polymerase I (figure 1 (B)). All experiments were done with three biological replicates. The measurements were taken every hour. The results can be seen here:
</DIV><CENTER><DIV><FIGURE><FIGCAPTION><B>Figure 1</B>: Cultivation of an <I>E. coli</I> culture with a plasmid that holds a wild-type polymerase and a culture with a plasmid that contains the error-prone polymerase. The cultivation was done with different ampicillin (amp) concentrations.
	
The cultivations were performed at 37°C in presence of 1, 3, 5 or 10 g ampicillin per liter, respectively.
	</FIGCAPTION></FIGURE></DIV></CENTER><CENTER><DIV><FIGURE><FIGCAPTION><B>Figure 2</B>: The measarment curves were fitted with the fitting programm.Therefore, we get the different reproduction rates.
	</FIGCAPTION></FIGURE></DIV></CENTER><DIV class="container text">
These measurements were used to calculate which reproduction rates can be reached. The maximal reproduction rate of the JS200 strain with the error prone polymerase is 0.67h<SUP>-1</SUP> . It were determined through fitting (figure 2) with the logistical function with the dynamical <I>r</I>.

A close look at our experimentally measured error prone growth curves (figure 1 (A)) led to the following interesting observation: Although the curves showed different growth behavior and diverging lag phases their rates seemed to undergo a shift, once the ampicillin was out of the system. Without ampicillin pressure the growth rates seem to be similar (figure 1 (A), exponential phase).
To account for varying growth behavior at the beginning and an almost uniform behavior after a certain time, we had to work with a dynamical <I>r</I>. After fitting the growth curves of our theoretical model to the experimentally measured ones, we observed a change of the growth rate r at an OD of about 0.38. 

As we could not find information about the behavior of different adapted <I>E. colis</I> together in one flask, we performed another experiment for further improvement of our mathematical model. 
Therefore, we cultivated two different adapted JS200 cultures, culture A and culture B, together in one flask. To be able to compare the two cultures we transformed a GFP labeled plasmid in culture A and an RFP labeled plasmid in culture B. Culture B had a little disadvantage with respect to growth in comparison to culture A. 
As a control we also cultivated the cultures alone. The experiments were performed with 1:1 A to B and 1:9 A to B proportion of the two different cultures to be able to study the behavior in detail.
</DIV><CENTER><DIV><FIGURE><FIGCAPTION><B>Figure 3</B>: Growth (A), GFP (B) and RFP (C) curves of differently well growing cultures and culture mixtures.
 
Culture A is a well growing <I>E. col</I> culture containing a GFP labeled plasmid and culture B a badly growing culture containing a RFP labeled plasmid. A 1:1 B is a culture mixture which contains an equal amount of culture A and B and A 9:1 B is a culture mixture which contains a proportion of 9:1 A to B.
	</FIGCAPTION></FIGURE></DIV></CENTER><DIV class="container text">
As can be seen in figure 2 (A) the culture mixtures do not reach the same high plateau as the separately cultivated culture A. Still they reach a higher plateu as the separately cultivated culture B. 
This means that the reproduction rate is influenced by different cultures. 
Also in figure 2 (B) it can be seen that the curve of the culture mixtures lies in between culture A and B, but the curve resembling the culture mixtures with A 9:1 B is nearly the same as the curve representing culture A. 
Thus, it can be seen that in the culture mixture culture A grows normally, but can not reach the same final OD. 
It can also be seen that in the 1:1 culture mixture the GFP concentration is not that much lower compared to the concentration in the separately cultivated culture A.
This means that culture B does not influence the growing of culture A. 
Figure 2 (C) shows that in the culture mixtures also culture B grows, but not as fast. So it never reaches the same OD amount as the single culture B. 

In this experiment we observed that the differently growing <I>E. coli </I> cultures compete with respect to space and nutrients. That is why we decided to use a modified logistical growth equation for the modelling of the competition.

While comparing all growth curves, we made an additional observation. There is always a little fluctuation in the growth behavior. 
This leads to the assumption that even when one strain is cultivated more than once its growth behavior will vary from time to time. 
So we incorporated a fluctuating <I>r</I>, following a normal distribution.
</DIV><CENTER><BUTTON><A class="button_link" href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Modeling" role="button">Results</A></BUTTON></CENTER><DIV class="container text_header"><H3>References</H3></DIV><DIV class="container text"><UL><LI>Bacaër, Nicolas (2011): A Short History of Mathematical Population Dynamics. London: Springer-Verlag London Limited. Online accessible at http://site.ebrary.com/lib/alltitles/docDetail.action?docID=10445161.</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>