<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Composite_Part skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Composite Part</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container main"><DIV id="header"><NAV id="navigationbar" class="navbar navbar-default custom_navbar_style navbar-fixed-top"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="#">NavBar</A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-center"><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description"> PROJECT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description">Description</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds">Scaffolds</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design">Design and Construction</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/CreateYours">Create your own Library</A></LI></UL><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/EpPolI">Error prone PolymeraseI</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/Global">Genome wide mutator</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">Bacterial Two-Hybrid System</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/SplitBetaLactamaseHitchHiker">Alternative Selection System</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_I">Model I</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_II">Model II</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results"> RESULTS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Phage">Phagemid Display</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">Reversion</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/Cloning">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/ExpressionControl">Expression Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/InteractionControl">Interaction Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/BindingControl">Binding Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/KnockOutKnochIn"><I>in vivo</I> Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Modeling">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Fermentation">Fermentation</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Best Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Best Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Best Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts/Improve">Improve a part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Demonstrate your Work</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Medal_Fulfillment">Medal Fulfillment</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold"> HUMAN PRACTICES </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Dictionary">Dictionary</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks">Educational Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Crowdfunding">Crowdfunding and Press</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Achievements</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations"> COLLABORATIONS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Duesseldorf">Düsseldorf</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Lethbridge">Lethbridge</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Freiburg">Freiburg</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments"> NOTEBOOK </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook">Lab Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">Primers</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols">Lab Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Media">Media and Buffers</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team"> TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team">Team Profile</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Members">Members</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Contact">Contact</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement"> ACKNOWLEDGEMENT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Sponsors_And_Partners">Sponsors and Partners</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></DIV></DIV></NAV></DIV></DIV><DIV class="container main"><DIV class="jumbotron" style="background-image: url(https://static.igem.org/mediawiki/2016/0/02/Bielefeld_CeBiTec_2016_10_14_X_projectheader.png)"><DIV class="jumbotron-text"><H1 style="margin-bottom: 0px; text-align:left">Best Composite Part</H1></DIV></DIV><DIV class="container text_header"><H3>BBa_K2082231: (Part of) The Selection</H3></DIV><DIV class="container text"><CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 1: Illustration of the part BBa_K2082231.</B></FIGCAPTION></FIGURE></CENTER>
What would be a whole library and a very high created variety, if you do not have something to get the parts, which are important for the specific situation? A selection system is a very important aspect of our system for the creation of binding proteins (Evobodies) for a lot of different situations. Perhaps an Evobody against a transcription factor for scientific researches is needed or a fast countermeasure against a new upcoming virus. Therefore, a bacterial two-hybrid system was created to select those Evobodies out of the whole created variety which are interacting with the specific protein.

Our best composite part <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2082231">BBa_K2082231</A> is one part of these bacterial two-hybrid system.  It contains a SH2 domain fused with the repressor protein cI of the phage 434. Also the whole reporter for the selection system is available on this part. To test the functionality of this part a lot of measurements were accomplished. 
At first the expression of the fusion protein SH2-cI was shown, with a <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols#sds_page">SDS-PAGE</A> and a <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols#western">Western blot assay</A>.  The results revealed an expressed protein with a mass about 25 kDa. This was the expected mass for the fusion protein and proofed that the designed protein is expressed as expected.  
<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 2: EMSA results. </B> In alternaty application the DNA fragment with the 434 OR1 binding site and the DNA fragment with the lambda binding site were used as DNA sample. +/- = with/without SH2-cI fusion protein. Only with the sample with the 434 binding site and addition of the SH2-cI protein a band shift could be seen.
</FIGCAPTION></FIGURE></CENTER>
The next step was the proof, if the designed DNA binding domain cI of the fusion protein really binds at their specific sequence. Therefore, a <I> in vitro</I> experiment with an electrophoretic mobility shift assay (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols#emsa">
EMSA</A>) was realized.  Only with addition of the cI protein the DNA fragment runs slower in the gel, which is a hint for an interaction between the cI protein and the binding site. Also another DNA sequence was tested as well with the lambda phage binding site instead of the binding site of the phage 434. This experiments results in no differences of the run speed of the DNA fragment in the gel and therefore no interaction of the cI protein with the DNA. Besides the <I>in vitro</I> proof also an <I>in vivo</I>  proof was accomplished.  Through a cloning step of a GFP generator under the control of an Anderson promoter upstream of the reporter, an experiment was created, where the RFP gene of the reporter is highly expressed, when cI is not binding at the specific binding site.  It was measured in a fluorescence-activated cell scanner (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols#facs">FACS</A>), that the RFP intensity of a non cI producing cell is about 35% higher than in a cell of with cI protein expression. Therefore, the designed DNA binding domain is functional.
<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 3: Results of the BLItz experiment.</B> Illustrated are the measured base line, the association and the dissociation of the protein-protein interaction in the BLItz. The addition of the second protein is marked at 60 seconds after measuring the base line. The SH2-cI fusion protein was added at the start without washing (green), after one (purple) and two (yellow) washing steps with NaCl and after a washin step with a high concentrated HCl solution (upper blue). Also BSA was measured with 1 μg/ml (lower blue) and 10 μ/ml (red) protein.</FIGCAPTION></FIGURE></CENTER>
A bacterial two-hybrid system is built on the interaction of two proteins and converts it in transcriptional activity of a reporter gene. The SH2 protein is our chosen positive control and should interact with the monobody HA4 designed in Chicago.  Therefore, the interaction of the fusion protein SH2-cI with the HA4 fusion protein of the part <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2082221">BBa_K2082221</A>   was examined.  After purification of the proteins an affinity chromatography was made to validate the interaction. With the <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols#maldi">
MALDI-TOF</A> the sample of the affinity chromatography was analyzed. Both proteins could be detected in the same sample. Therefore, an interaction of the two proteins was possible. To validate the results the two purified proteins where measured in the bio-layer interferometry system (BLItz) of ForteBio. Directly after adding the SH2-cI fusion protein to the immobilized HA4 protein, interferences in the white light refraction was measured. Those interferences will be released if something is binding at the immobilized protein. The negative control with BSA does not result in any interference. That is the final evidence that the expected interaction of SH2 and HA4 also worked with our designed proteins. 
<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 4: Tecan-Results of the <I>in vivo</I> reporter activity.</B> Two culters of <I>E. coli</I> were measured on their ability to produce RFP through the reporter construct. A culture with one fusion protein SH2-cI expressing is compared with a culture carrying both fusion proteins SH2-cI and HA4-RpoZ. The RFP intensity of the culture with both fusion proteins is significantly higher than in the culture with only one fusion protein.</FIGCAPTION></FIGURE></CENTER>
The final functionality of the BioBrick as a part of the bacterial two-hybrid system was researched by a comparison of the RFP intensity of two different bacterial cultures. The first culture was only carrying the described plasmid <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2082231">BBa_K2082231</A>. The second culture also carrying the plasmid with the second fusion protein HA4-RpoZ. The comparison of the RFP intensity in the <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols#tecan">
Tecan</A> plate reader revealed a visible difference in these two cultures. The cells carrying both fusion proteins produce about 48% more RFP than the cells with only one plasmid. A two sided t-test approved the assumption that the difference between these two cultures is very significant. Therefore, an <I>in vivo</I> activation of our bacterial two-hybrid system is possible.

Therefore, the part is characterized. The fusion protein will be expressed and worked like expected. The DNA binding domain is binding at the designed OR1 binding region upstream of the promoter. Also an interaction of SH2 and HA4 was approved. But the most important aspect, the SH2-cI fusion protein is a functional part of the bacterial two-hybrid system and is able to activate the reporter gene <I> in vivo</I>  in interplay with the HA4-RpoZ fusion protein.  We created a part for the iGEM community to create a possibility of <I>in vivo</I> selection through a transcriptional activation bacterial two-hybrid system. 

</DIV><DIV class="container text_header"><H3>Achievements</H3></DIV><DIV class="container text"><TABLE class="table" style="border:none;"><TBODY style="border:none;"><TR style="border:none;"><TD style="text-align:left; border:none;">Our part will be  <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/ExpressionControl" target="_blank">expressed</A></TD></TR><TR style="border:none;"><TD style="text-align:left; border:none;">Successful <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/InteractionControl" target="_blank">interaction</A> between our SH2 domain with the HA4 domain of the HA4-RpoZ protein!</TD></TR><TR style="border:none;"><TD style="text-align:left; border:none;"> The DNA binding domain cI successfully <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/BindingControl" target="_blank">binds</A> at the 434 OR1 binding site!</TD></TR><TR style="border:none;"><TD style="text-align:left; border:none;">We achieved a functional <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/KnockOutKnochIn">bacterial two-hybrid system</A> with this part.
	</TD></TR><TR style="border:none;"><TD style="text-align:left; border:none;">Together with <A href="http:// parts.igem.org/wiki/index.php?title=Part:BBa_K2082221">BBa_K2082221</A>  we offer the iGEM community a functional system for <I>in vivo</I> selection!</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>