Team:Bielefeld-CeBiTec/test
Toggle navigation
NavBar
PROJECT
Description
Library
Overview
Scaffolds
Design and Construction
Create your own Library
Mutation
Overview
Error prone Polymerase I
Genome wide mutator
Selection
Overview
Bacterial Two-Hybrid System
Alternative Selection System
Modeling
Overview
Model I
Model II
RESULTS
Overview
Library
Overview
Assembly
Phagemid Display
Sequencing
Mutation
Overview
Assembly
Reversion
Sequencing
Selection
Overview
Assembly
Expression Control
Interaction Control
Binding Control
in vivo Selection
Modeling
Fermentation
Parts
Best Basic Part
Best Composite Part
Best Part Collection
Improve a part
Proof of Concept
Demonstrate your Work
Medal Fulfillment
HUMAN PRACTICES
Overview
Engagement
Dictionary
Educational Outreach
Crowdfunding and Press
Entrepreneurship
Integrated Practices
Achievements
COLLABORATIONS
Overview
Düsseldorf
Lethbridge
Freiburg
NOTEBOOK
Overview
Lab Notebook
Library
Mutation
Selection
Primers
Lab Protocols
Media and Buffers
TEAM
Team Profile
Members
Bielefeld University
Contact
ACKNOWLEDGEMENT
Overview
Sponsors and Partners
Attributions
Project overview
Motivation
???
Generation of binding proteins by directed evolution
This year the iGEM Team Bielefeld-CeBiTec aims to create a system for generation of synthetic binding protein     the so called Evobodies. The system works by creating a library of binding proteins and increase their affinity     towards a target by directed evolution. As a starting point we randomise the binding regions of synthetic     antibody-mimetic frameworks. Following we screen this library for affinity towards a target by using a bacterial     two-hybrid system. To further increase the evobodies affinity, we combine the selection via the two-hybrid system     with an in vivo mutagenesis system. Doing this we hope to generate strong and specific binding proteins by combining     the powerful genetic and the speed of E. coli with the biological idea of antibody generation and maturation in vertebrates.
We aim to create this system as an alternative to conventional binding generation systems. We hope that it is better?! Was soll hier hin
The starting point - synthetic binding protein library
As staring point, we use the antibody-mimetic scaffolds of mono- and nanobodies. In these proteins we randomise     the CDR-like binding regions in that only amino acids known to contribute to binding are expressed there.     Together we create a library of binding proteins with a relative small size (~109 variants), but restriction     to specific amino acids raises the overall library quality. To read more about our library design go to the library subpage.
Survival of the fittest - bacterial two-hybrid
In the next step, the binding protein library should be screened for proteins with a innate affinity for our target. We want to realise     this by using a bacterial two-hybrid system. Therefore, the binding protein is fused to a RNA polymerase subunit and the target protein     is fused to the DNA-binding phage repressor cI protein. In the complete two-hybrid system the target-cI fusion is located on the cognate     DNA-site. If the binding protein binds to the target protein the RNA polymerase is recruited to the cI binding site. There it finds a     very weak promoter and increases the transcription of the downstream gene. All in all should the interaction between the binding and the     target protein lead to increased gene expression of a reporter gene.
From the two-hybrid system we expect foremost a selection of the binding protein library. But furthermore we expect a correlation     between binding protein – target affinity and the thus activated gene expression of the reporter. By using an antibiotic resistance     protein as a reporter we predict increased levels of the resistance protein inside a cell with a good binding protein. The outcome of     this should be an increase of individual fitness for the bacteria, which should lead to a higher growth rate under strong selective pressure.     The complete two-hybrid system should cumulate in a relationship between binding protein affinity and bacterial growth rate, which should     lead to selection of a few bacteria with strong binding proteins.
Accessing the sequence space - in vivo mutagenesis
After selection of the bulk of our library we will increase the affinity of our evobodies in a process similar to the affinity maturation     of antibodies. As addressed above we will select our evobodies by increasing the selection pressure. At the same time, we will use an in vivo mutagenesis system to further adapt our evobodies. Thereby we can increase the accessed sequence space and may modify useful binding proteins     from our library using the power of evolution.    Technical this leads to us using a low-fidelity polymerase I in an otherwise Pol I temperature-sensitive E. coli strain or the classical     approach of global increased mutation rate by modulatiog the E. coli DNA fidelity systems. Especially the first method is attractive,     because the mutation is mostly limited to plasmids carrying the ColE1-ori. But to identifiy the best mutation system for a directed evolution     approach we will compare both system in terms of mutagenesis rate, -spectrum, -controllability and -specifity.
