Team:Bielefeld-CeBiTec/Results/Library/Phage
Toggle navigation
NavBar
PROJECT
Description
Library
Overview
Scaffolds
Design and Construction
Create your own Library
Mutation
Overview
Error prone Polymerase I
Genome wide mutator
Selection
Overview
Bacterial Two-Hybrid System
Alternative Selection System
Modeling
Overview
Model I
Model II
RESULTS
Overview
Library
Overview
Assembly
Phagemid Display
Sequencing
Mutation
Overview
Assembly
Reversion
Sequencing
Selection
Overview
Assembly
Expression Control
Interaction Control
Binding Control
in vivo Selection
Modeling
Fermentation
Parts
Best Basic Part
Best Composite Part
Best Part Collection
Improve a part
Proof of Concept
Demonstrate your Work
Medal Fulfillment
HUMAN PRACTICES
Overview
Engagement
Dictionary
Educational Outreach
Crowdfunding and Press
Entrepreneurship
Integrated Practices
Achievements
COLLABORATIONS
Overview
Düsseldorf
Lethbridge
Freiburg
NOTEBOOK
Overview
Lab Notebook
Library
Mutation
Selection
Primers
Lab Protocols
Media and Buffers
TEAM
Team Profile
Members
Bielefeld University
Contact
ACKNOWLEDGEMENT
Overview
Sponsors and Partners
Attributions
Library Results
Phagemid Display
Screening the Initial Diversity
The core of our system is the combination of an initial library with in vivo mutation and selection, to optimize Evobodies by directed evolution. To get in this evolutional process, some binding proteins must show at least a marginal affinity to the chosen target protein. Low affinity binders get mutated and selected in vivo during  culture growth under an increasing selective pressure.  To make sure the designed binding proteins can attach to several target proteins and the Monobody and Nanobody libraries can deliver initial Evobodies, which can be optimized by mutation and selection, we performed a phagemid display , using phagemid pAK100 and M13 derived helper phages.
After assembling the pAK100 backbone with the CDS of our designed binding proteins via Gibson cloning, the phagemid was transformed into electrocompetent E. coli ER2738 and plated on LB agar, containing chloramphenicol and tetracycline. Thus, we generated a comparative small library of about 3000 clones for each, Monobody and Nanobody coding phagemids, as a test library for the use in the phagemid display.  The infection of the eluted and incubated culture with M13 derived helper phages and the following purification of the composed phages leads to pelleted phages, diluted in PBS. The UV-VIS absorbance of an 1:50 aliquot at 269 nm and 320 nm was measured, to calculate the phage concentration, based on the following formula:
For the Monobody carrying phages we gained a concentration of  2.992 x 1013 phages per mL in an elution volume of 1.5 mL. For the Nanobody carrying phages a concentration of 1.599 x 1013 phages per mL was achieved in an equal elution volume.
These phages were used to accomplish one stage of phage panning on immuno-tubes, coated with several different proteins, from transcription factors to enzymes and protein domains. We explicit relinquished further panning rounds, to simulate the conditions of our system. Thereby just an initial binder is required to get optimized by in vivo mutation and selection. After an exhaustive washing procedure and the following elution of phages with affinity imparting binding proteins an exponential growing culture of ER2738 was infected and plated on LB agar, containing chloramphenicol.  After incubation at 37°C, several colonies were detectable on agar plates (Fig. 1).
Figure 1: Agar plate, showing E. coli ER2738 colonies after one panning round of phagemid display. Every colony carries a phagemid with the CDS for a Monobody with binding affinity to FGE protein
As an example, we isolated and sequenced one of the Monobody CDSs coding for a formylglycine-generating enzyme (FGE) binder (Fig. 2).
Figure 2: Example sequence of a Monobody, identified as FGE binder by phagemid display. Top to bottom: Ordered sequence, chromatogram and sequencing result.
Summarized, the phagemid display was a good opportunity, to screen our Monobody and Nanobody libraries at initial binding proteins for our directed evolution system. We could show, that in a small extract of 3.000 variable Monobodies, as well as Nanobodies, out of a theoretical library size of about 1 billion CDS contains some potential candidates for getting an optimal Evobody by directed evolution!
Sequencing
