<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Project_Library_Scaffolds skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Project/Library/Scaffolds</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container main"><DIV id="header"><NAV id="navigationbar" class="navbar navbar-default custom_navbar_style navbar-fixed-top"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="#">NavBar</A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-center"><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description"> PROJECT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description">Description</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds">Scaffolds</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design">Design and Construction</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/CreateYours">Create your own Library</A></LI></UL><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/EpPolI">Error prone PolymeraseI</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/Global">Genome wide mutator</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">Bacterial Two-Hybrid System</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/SplitBetaLactamaseHitchHiker">Alternative Selection System</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_I">Model I</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_II">Model II</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results"> RESULTS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Phage">Phagemid Display</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">Reversion</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/Cloning">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/ExpressionControl">Expression Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/InteractionControl">Interaction Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/BindingControl">Binding Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/KnockOutKnochIn"><I>in vivo</I> Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Modeling">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Fermentation">Fermentation</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Best Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Best Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Best Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts/Improve">Improve a part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Demonstrate your Work</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Medal_Fulfillment">Medal Fulfillment</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold"> HUMAN PRACTICES </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Dictionary">Dictionary</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks">Educational Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Crowdfunding">Crowdfunding and Press</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Achievements</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations"> COLLABORATIONS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Duesseldorf">DÃ¼sseldorf</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Lethbridge">Lethbridge</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Freiburg">Freiburg</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments"> NOTEBOOK </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook">Lab Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">Primers</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols">Lab Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Media">Media and Buffers</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team"> TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team">Team Profile</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Members">Members</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Contact">Contact</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement"> ACKNOWLEDGEMENT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Sponsors_And_Partners">Sponsors and Partners</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></DIV></DIV></NAV></DIV></DIV><DIV class="container main"><DIV class="jumbotron" style="background-image: url(https://static.igem.org/mediawiki/2016/8/89/Bielefeld_CeBiTec_2016_10_14_X_jessi2.png)"><DIV class="jumbotron-text"><H1 style="margin-bottom: 0px; text-align:left">Library Project</H1><H2 style="color:#ffffff; text-align:left">Scaffolds</H2></DIV></DIV><DIV class="container text_header"><H3>What is a Library?</H3></DIV><DIV class="container text">
Our project is based on the construction of an adequate library, which is defined as a collection of identical plasmids that only vary in the protein coding sequences (CDS). These CDS are designed to possess optimized and region specific randomized subregions displaying high variability. Upon transformation of the randomized plasmids into <I>E. coli</I> a heterogeneous culture is obtained with each colony carrying a different insert translating into different binding proteins (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds#Osoegawa">Osoegawa <I>et al.</I>, 2001</A>).

One major advantage of our library is the availability of a wide range of different binding proteins as starting point for the directed evolution process. Completely new antibodies or variations of existing ones could serve as starting material. 

By creating such a large variety, we are aiming for the highest chance for a possible binder. By adjusting the variable regions and their nucleic acid scheme we can specify the target further  but still have leverage to cover a wide range of possible targets in the beginning (see <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design">Library - Design and Construction</A>).
In order to identify suitable binding proteins for an appropriate scaffold of our library, we compared different types of antibodies and antibody mimetics from various sources. Antibody mimetics also show better accessibility to genetic engineering like construction of fusion proteins or incorporation of modifiable amino acids thus allowing the combination with fluorophores or drugs for targeted drug delivery (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds#Skerra">Skerra<I>et al.</I>, 2007</A>). After deep and thorough investigation we decided to use Monobodies and Nanobodies. Although commonly used antibodies might be more robust, they are outperformed ten times by Mono- and Nanobodies in their smaller size (15 kDa vs. 150 kDa) and stability. This enables to penetrate depth, even in hard tissue (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds#Ahmad">Ahmad<I>et al.</I>, 2012</A>). Additionally, Nano- and Monobody binding proteins are rapidly cleared from the blood and have lower retention time in non-target tissues (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds#Ahmad">Ahmad <I>et al.</I>, 2012</A>).
</DIV><DIV class="container text_header"><H3>Monobodies</H3></DIV><DIV class="container text"> 
Monobodies, also called adnectines, are synthetic antibody mimetics that originate from the tenth human fibronectin type 3 domain. Fibronectin is a rather large protein that is essential in the formation of the extracellular matrix and cell-cell interactions (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds#Koide2007">Koide and Koide, 2007</A>). In 1998 the research group of Akiko Koide discovered that the fibronectin scaffold possesses similar binding properties like antibodies (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds#Koide1998">Koide <I>et al.</I>, 1998</A>). Since then continuous research lead to appliance in treatment of diseases like glioblastoma multiform (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds#Bloom">Bloom <I>et al.</I>, 2009</A>) or chronic myelogenous leukaemia (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds#Grebien">Grebien <I>et al.</I>, 2011</A>). The structure of Monobodies is similar to immunoglobulin domains (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds#Lipovsek2011">Lipovsek <I>et al.</I>, 2011</A>). The original three dimensional structure of the scaffold is a Î²-sandwich but with seven Î²-strands instead of nine which are common in the variable domain of the heavy chain domain of regular antibodies and three surface loops on each end (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds#Koide2007">Koide and Koide, 2007</A>).  They have a high stability and due to their small size, 10 kDa, and they penetrate deep into tissue (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds#Koide2007">Koide and Koide, 2007</A>).  Another outstanding feature that they can be functionally expressed in the <I>E. coli</I> cytoplasm and thereby can be produced simple and at low cost in culture. This is due to the absence of disulphide bonds which are unlikely to form in the <I>E. coli</I> cytoplasm. This makes them compatible with virtually any display technologies (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds#Koide2007">Koide and Koide, 2007</A>) and is therefore a suitable canditate for our <I>in vivo</I> system. By randomising the amino acid sequence of the CDR-like loops an initial library is created. The extensive loop sequence variation is tolerated by the structure so it stays stable (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds#Batori">Batori <I>et al.</I>, 2002</A>). Thus allowing binding of the target with high affinity and selectivity (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds#Skerra">Skerra <I>et al.</I>, 2007</A>).
</DIV><FIGURE class="figure"><B>Fig. 1: Protein structure of an Monobody.</B> Constant regions colored in green, variable regions colored in blue.</FIGURE><DIV class="container text_header"><H3>Nanobodies</H3></DIV><DIV class="container text">
Nanobodies are single domain antibodies that originate from camelids like Alpaca and Camel or cartilaginous fish like sharks (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds#Stanfield">Stanfield <I>et al.</I>, 2004</A>; <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds#Linden">van der Linden <I>et al.</I>, 1999</A>.

The nanobody family consist of conventional heterotetrameric antibodies with low-affinity binders as well as unique functional heavy (H)-chain antibodies (HCAbs) with high affinity binders (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds#Harmsen">Harmsen <I>et al.</I>, 2007</A>. Because a single polypeptide chain is expressed, they are better suited for phage display than Fragment antigen-binding (Fab) fragments (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds#Holt">Holt <I>et al.</I>, 2003</A>). The H chain of these homodimeric antibodies consists of one antigen-binding domain, the VHH, and two constant domains. HCAbs fail to incorporate light (L) chains owing to the deletion of the first constant domain and a reshaped surface at the VHH side, which normally associates with L chains in conventional antibodies (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds#Muyldermans">Muyldermans <I>et al.</I>, 2003</A>). 

Therefore, the production takes place in liquid cell culture and thus production in large quantities is simple with higher yields than Fab fragments (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds#Holt">Holt <I>et al.</I>, 2003</A>). They consist of a single monomeric variable antibody domain and a universal Nanobody-framework (in our case cAbBCII10). With a size of 12-15 kDa they are slightly bigger than Monobodies but still able to penetrate hard tissues like solid tumours (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds#Holt">Holt <I>et al.</I>, 2008</A>). Research lead to various applications in the therapeutically field. For example, a humanised scFv is in phase II testing to reduce mortality and myocardial infarction in patients undergoing graft surgery for coronary artery bypass (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds#Saerens">Saerens<I>et al.</I>, 2003</A>). Also therapeutically application against the influenza A H5N1 virus has been shown in mice where it reduced the replication rate significantly.  Even in oral immunotherapy Nanobodies are used (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds#Harmsen">Harmsen <I>et al.</I>, 2007</A>). Also many diagnostically applications were made possible like super-resolution microscopy using a Nanobody tagged with GFP or tagging by binding proteins, enzymes or antigens with such a complex (Rothbauer U. et al, 2007). They can also be modified to work as a biosensor (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds#Saerens">Saerens<I>et al.</I>, 2003</A>).
A high expression rate was shown in <I>E. coli</I> and they are still functional and stable even in absence of conserved disulphide bonds (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds#Saerens">Saerens<I>et al.</I>, 2003</A>)  which are unlikely to form in the <I>E. coli</I> cytoplasm. Their high tolerance against acidic conditions, temperature compared to mouse monoclonal antibodies (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds#Linden">van der Linden <I>et al.</I>, 1999</A>), high solubility (Arbabi Ghahroudi M. et al, 1997)  and very few cleavage sites for enzymes allow new areas of application (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds#Harmsen">Harmsen <I>et al.</I>, 2007</A>) . The Nanobody loop structure can bind to antigen structures inaccessible to human antibodies, for example catalytic centres in enzymes. Also fusion proteins and complexes can be created leaving both parts still active. Example are a complex of a shark Nanobody and lysozyme (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds#Stanfield">Stanfield <I>et al.</I>, 2004</A>) or a Nanobody conjugated branched Gold nanoparticle for photothermal therapy against tumour cells (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds#Broek">van de Broek<I>et al.</I>, 2011</A>). 
</DIV><FIGURE class="figure"><B>Fig. 2: Protein structure of an Nanobody.</B> Constant regions colored in green, variable regions colored in blue.</FIGURE><CENTER><BUTTON><A class="button_link" href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design" role="button">Design and Construction</A></BUTTON></CENTER><DIV class="container text_header"><H3>References</H3></DIV><DIV class="container text"><UL><LI id="Ahmad">Ahmad, Zuhaida Asra; Yeap, Swee Keong; Ali, Abdul Manaf; Ho, Wan Yong; Alitheen, Noorjahan Banu Mohamed; Hamid, Muhajir (2012): scFv antibody: principles and clinical application. In: <I>Clinical &amp; developmental immunology</I> 2012, S. 980250. DOI: 10.1155/2012/980250.</LI><LI id="Batori"> Batori, V.; Koide, A.; Koide, S. (2002): Exploring the potential of the monobody scaffold. Effects of loop elongation on the stability of a fibronectin type III domain. In: <I>Protein Engineering Design and Selection</I> 15 (12), S. 1015â1020. DOI: 10.1093/protein/15.12.1015. </LI><LI id="Bloom">Bloom, Laird; Calabro, Valerie (2009): FN3: a new protein scaffold reaches the clinic. In: <I>Drug discovery today</I> 14 (19-20), S. 949â955. DOI: 10.1016/j.drudis.2009.06.007.</LI><LI id="Grebien">Grebien, Florian; Hantschel, Oliver; Wojcik, John; Kaupe, Ines; Kovacic, Boris; Wyrzucki, Arkadiusz M. et al. (2011): Targeting the SH2-kinase interface in Bcr-Abl inhibits leukemogenesis. In: <I>Cell</I> 147 (2), S. 306â319. DOI: 10.1016/j.cell.2011.08.046.</LI><LI id="Harmsen"> Harmsen, M. M.; Haard, H. J. de (2007): Properties, production, and applications of camelid single-domain antibody fragments. In: <I>Applied microbiology and biotechnology</I> 77 (1), S. 13â22. DOI: 10.1007/s00253-007-1142-2.</LI><LI id="Holt"> Holt, Lucy J.; Herring, Chris; Jespers, Laurent S.; Woolven, Benjamin P.; Tomlinson, Ian M. (2003): Domain antibodies: proteins for therapy. In: Trends in biotechnology 21 (11), S. 484â490. DOI: 10.1016/j.tibtech.2003.08.007.</LI><LI id="Koide1998">Koide, A.; Bailey, C. W.; Huang, X.; Koide, S. (1998): The fibronectin type III domain as a scaffold for novel binding proteins. In: <I>Journal of Molecular Biology</I> 284 (4), S. 1141â1151. DOI: 10.1006/jmbi.1998.2238.</LI><LI id="Koide2007">Koide, Akiko; Koide, Shohei (2007): Monobodies: antibody mimics based on the scaffold of the fibronectin type III domain. In: <I>Methods in molecular biology (Clifton, N.J.)</I> 352, S. 95â109. DOI: 10.1385/1-59745-187-8:95.</LI><LI id="Lipovsek2011">Lipovsek, D. (2011): Adnectins: engineered target-binding protein therapeutics. In: <I>Protein engineering, design &amp; selection</I> : PEDS 24 (1-2), S. 3â9. DOI: 10.1093/protein/gzq097.</LI><LI id="Muyldermans"> Muyldermans, Serge (2013): Nanobodies: natural single-domain antibodies. In: <I>Annual review of biochemistry</I> 82, S. 775â797. DOI: 10.1146/annurev-biochem-063011-092449.</LI><LI id="Osoegawa">Osoegawa, K.; Jong, P. J. de; Frengen, E.; Ioannou, P. A. (2001): Construction of bacterial artificial chromosome (BAC/PAC) libraries. In: <I>Current protocols in molecular biology</I> Chapter 5, Unit 5.9. DOI: 10.1002/0471142727.mb0509s55.</LI><LI id="Saerensâ&gt;Saerens, Dirk; Ghassabeh, Gholamreza Hassanzadeh; Muyldermans, Serge (2008): Single-domain antibodies as building blocks for novel therapeutics. In: &lt;i&gt;Current opinion in pharmacology&lt;/i&gt; 8 (5), S. 600â608. DOI: 10.1016/j.coph.2008.07.006.&lt;/li&gt;
&lt;li id=" skerra="">Skerra, Arne (2007): Alternative non-antibody scaffolds for molecular recognition. In: <I>Current opinion in biotechnology</I> 18 (4), S. 295â304. DOI: 10.1016/j.copbio.2007.04.010.</LI><LI id="Stanfield"> Stanfield, Robyn L.; Dooley, Helen; Flajnik, Martin F.; Wilson, Ian A. (2004): Crystal structure of a shark single-domain antibody V region in complex with lysozyme. In: <I>Science</I> (New York, N.Y.) 305 (5691), S. 1770â1773. DOI: 10.1126/science.1101148.</LI><LI id="Broekâ&gt; van de Broek, Bieke; Devoogdt, Nick; D'Hollander, Antoine; Gijs, Hannah-Laura; Jans, Karolien; Lagae, Liesbet et al. (2011): Specific cell targeting with nanobody conjugated branched gold nanoparticles for photothermal therapy. In: ACS &lt;i&gt;nano&lt;/i&gt; 5 (6), S. 4319â4328. DOI: 10.1021/nn1023363.&lt;/li&gt;
&lt;li id=" linden=""> van der Linden, R.H.J.; Frenken, L.G.J.; Geus, B. de; Harmsen, M. M.; Ruuls, R. C.; Stok, W. et al. (1999): Comparison of physical chemical properties of llama VHH antibody fragments and mouse monoclonal antibodies. In: Biochimica et Biophysica Acta (BBA) - <I>Protein Structure and Molecular Enzymology</I> 1431 (1), S. 37â46. DOI: 10.1016/S0167-4838(99)00030-8.</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>