<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Project_Mutation_Global skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Project/Mutation/Global</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container main"><DIV id="header"><NAV id="navigationbar" class="navbar navbar-default custom_navbar_style navbar-fixed-top"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="#">NavBar</A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-center"><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description"> PROJECT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description">Description</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds">Scaffolds</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design">Design and Construction</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/CreateYours">Create your own Library</A></LI></UL><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/EpPolI">Error prone PolymeraseI</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/Global">Genome wide mutator</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">Bacterial Two-Hybrid System</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/SplitBetaLactamaseHitchHiker">Alternative Selection System</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_I">Model I</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_II">Model II</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results"> RESULTS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Phage">Phagemid Display</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">Reversion</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/Cloning">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/ExpressionControl">Expression Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/InteractionControl">Interaction Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/BindingControl">Binding Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/KnockOutKnochIn"><I>in vivo</I> Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Modeling">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Fermentation">Fermentation</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Best Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Best Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Best Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts/Improve">Improve a part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Demonstrate your Work</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Medal_Fulfillment">Medal Fulfillment</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold"> HUMAN PRACTICES </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Dictionary">Dictionary</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks">Educational Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Crowdfunding">Crowdfunding and Press</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Achievements</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations"> COLLABORATIONS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Duesseldorf">Düsseldorf</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Lethbridge">Lethbridge</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Freiburg">Freiburg</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments"> NOTEBOOK </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook">Lab Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">Primers</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols">Lab Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Media">Media and Buffers</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team"> TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team">Team Profile</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Members">Members</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Contact">Contact</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement"> ACKNOWLEDGEMENT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Sponsors_And_Partners">Sponsors and Partners</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></DIV></DIV></NAV></DIV></DIV><DIV class="container main"><DIV class="container text_header"><H1>Global mutagenesis</H1></DIV><DIV class="container text_header"><H3>Impairing the DNA fidelity mechanisms</H3></DIV><DIV class="container text_header"><H3>Abstract</H3></DIV><DIV class="container text">
			Organisms ensure the fidelity of their DNA by three overlapping pathways namely base selection, proofreading and mismatch repair. 
			These pathways cumulate in replication fidelity of ~10<SUP>-10</SUP> mutations per bp per generation in <I>E. coli</I> (Lee et al. 2012). 
			Overexpression of dominant negative variants of major players in this system can increase the mutation rate substantially. Recently, 
			mutation rates up to 6.2×10<SUP>-6</SUP> mutations per bp and per generation were obtained by using a plasmid-borne mutator system 
			(Badran, Liu 2015). We aim to make this mutator plasmid available for the whole iGEM community by rebuilding it with defined iGEM parts 
			(see our <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2082117">BBa_K208117</A>). We envision an application in the 
			directed evolution of our binding protein. Furthermore, implementation in other directed evolution approaches could be feasibly 
			in which our plasmid pose an <I>in vivo</I> alternative to error-prone PCR (Wilson, Keefe 2001) or exogene mutagens (reviewed in Wong et al. 2006; Steensels et al. 2014).</DIV><DIV class="container text_header"><H3>Maintenance of DNA fidelity and creating <I>in vivo</I> mutagenesis by affecting the participating proteins</H3></DIV><DIV class="container text"> 
			DNA is the single blueprint for all proteins of an organism. Minor changes in the DNA sequence are mostly without significant consequences, 
			but in some cases the consequences are detrimental e.g. by rendering encoded crucial proteins useless. Therefore, an effective system of DNA 
			repair mechanisms emerged during evolution. The DNA fidelity system is composed of three main mechanisms, which contribute to the high fidelity. 
			The first fidelity mechanism lies in selecting the correct base for incorporation during replication of DNA. This process of base selection 
			reduces the error rate by 10<SUP>5</SUP> mutations per bp per generation. During replication, the acting polymerase (mainly polymerase III in <I>E. coli</I>) 
			directly corrects falsely incorporated bases. This process is called proofreading and reduces the error rate by 10<SUP>2</SUP> mutations per bp per generation. 
			Other mutations in the DNA e.g. introduced through replication errors or other mutagenic sources are repaired by the mismatch repair systems. 
			This system reduces the error rate by 10<SUP>3</SUP> mutations per bp per generation. (Schaaper 1993; Fijalkowska et al. 2012)  All these fidelity 
			mechanisms contributed to the error rate reduction, leading to a final rate of about 10<SUP>-10</SUP> mutations per bp per generation. (Lee et al. 2012).
			  			
			Many variants of the participating proteins with (partial) loss-of-function were identified and described in the literature 
			(Degnen, Cox 1974; Nghiem et al. 1988; Schaaper, Radman 1989; Fowler, Schaaper 1997; Junop et al. 2003). The common phenotype upon the 
			mutants was a strongly increased mutation rate. It did not take long before better-characterized hypermutator strains arose as a tool to study 
			the effect of random mutations on proteins or to improve proteins trough means of directed evolution 
			(Greener et al. 1997; Selifonova et al. 2001; Stefan et al. 2001). A common issue among  different applications of genomic mutator genes is the 
			general genetic instability and frequent unviability of these strains. For example, the  <I>E. coli</I> strain, which chromosomally harbors 
			the strongest known mutator <I>dnaQ926</I> is almost completely unviable without additional mutations (Fijalkowska, Schaaper 1996).
			Therefore, recent development was directed to the rational design of a plasmid-borne mutator system. Combination of strong mutator capabilities with a 
			tight inducible promoter should result in a controllable mutagenesis system, generally circumventing many issues of chromosome-based hypermutator strains. 
			Furthermore, the rational combination of different groups of mutator genes enables compositions, which have a very <A href="#">balanced mutagenesis spectra</A>
			Our mutation system was designed to use an untargeted mutation approach as an alternative to the error-prone polymerase I. Therefore, we reconstructed 
			the previously described mutator plasmid (Badran, Liu 2015), which consists of six different genes. 
			<FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 1: Schematic view of the mutator genes on a plasmid developed by Badran and coworkers (Badran, Liu 2015).</B> This mutator plasmid uses six  
					genes under tight control through an arabinose promoter. The different DNA fidelity mechanisms affected by the genes are indicated by coloring.
				</FIGCAPTION></FIGURE>	
			Overexpression of these mutator genes rises the mutation 
			frequency in <I>E. coli</I> to 6.2×10<SUP>-6</SUP> per bp per generation with induction and only ~4×10<SUP>-9</SUP> per bp per generation without induction. 
			Moreover, the mutation spectrum is very diverse.
							
			<FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 2: Mutation spectrum of the mutagenesis plasmid developed by Badran and coworkers (Badran and Liu 2015).</B></FIGCAPTION></FIGURE></DIV><DIV class="container text_header"><H3>DnaQ926</H3></DIV><DIV class="container text"> 
			The <I>dnaQ</I> gene encodes  ε-subunit of <I>E. coli</I> DNA polymerase III, which is responsible for the proof reading activity 
			of this enzyme. DnaQ confers  3'-exonuclease activity, which excises falsely incorporated bases during replication. DnaQ926 contains two amino 
			acid substitutions  (D12A, E14A) in the Exo I region, which prevents coordination of an essential metal ion and therefore abolishes the 
			proofreading activity (Fijalkowska, Schaaper 1996). The overall structure of DnaQ and  interactions with other proteins of the DNA polymerase III 
			complex are not affected by these substitutions. <I>dnaQ926</I> acts as a dominant negative gene causing a strong mutator phenotype even in <I>E. coli</I> 
			strains with chromosomal wild type <I>dnaQ</I> (Cox, Horner 1986). DnaQ926 competes against DnaQ about incorporation into the DNA polymerase 
			III complex. Incorporation of DnaQ926 leads to proofreading deficient replication complexes. Therefore, overexpression or plasmid-based expression 
			of <I>dnaQ926</I> is supposed to yield an increased mutation effect. The huge  amount of mutations introduced by <I>dnaQ926</I>  overwhelms the <I>E. coli</I> 
			mismatch repair system leading to the manifestation of mutations from various sources (Schaaper, Radman 1989; Damagnez et al. 1989) 
			The exact rate and mutagenesis spectrum of DnaQ926 depends on the used cultivation media. Rich media leads high mutation frequency (3.7×10<SUP>4</SUP> 
			over background) with mostly transitions, while minimal media cause less mutations (4.8×10<SUP>2</SUP> over background) and mostly transversions 
			(mainly AT → TA). The faster growth of bacteria in rich media decreases the time the time to remove errors before cell division via the repair 
			mechanisms (Schaaper 1988).
			<FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 3: Structure of DnaQ with indicated mutation sites.</B> In DnaQ926 the acidic amino acids D12A, E14A (red) are changed to alanine.
					Thereby complexation of the function-essential manganese ion (grey) is abolished due to missing negative charges at the amino acid residues. 
					(PDB: 2IDO) (Kirby et al. 2006)
				</FIGCAPTION></FIGURE></DIV><DIV class="container text_header"><H3>DNA adenine methylase (Dam) and SeqA</H3></DIV><DIV class="container text"> 
			A mechanism of mismatch repair is the Dam-directed mismatch repair (DDMR). Parental and daughter DNA strands are modified 
			by methylation at GATC sites. These methylations are added after replication by the DNA adenine methylase (Dam). In the timeframe 
			between daughter strand synthesis and Dam methylation the repair mechanisms can distinguish between the correct base on the methylated 
			parent strand (template) and the falsely incorporated base on the non-methylated daughter strand (product). This modification 
			is used by the mismatch repair proteins MutS, MutL and MutH, which repair the daughter strand relying on information from the 
			parent strand (Horst et al. 1999). 
			
			SeqA is a GATC-binding protein, which also controls the methylation state of DNA. It counteracts Dam by demethylating DNA. This results 
			in hypomethylated DNA on both strands, which further decreases the efficiency of DNA mismatch repair (Yang et al. 2004).
			
			Combined overexpression of <I>dam</I> and <I>seqA</I> strongly shifts the methylation state of the DNA. Methylation of the parental strand by SeqA 
			is reduced while the methylation of the daughter strand via Dam is increased. Both effects abrogate the methylation difference between 
			parental and daughter strand and prevent effective mismatch repair. 
		</DIV><DIV class="container text_header"><H3>EmrR</H3></DIV><DIV class="container text"> 
			As mentioned before, the main factor of replication fidelity is the correct base incorporation during replication. 
			This is mainly dependent on the α-subunit of DNA polymerase III, but the intracellular concentrations of the 
			single nucleotidetriphosphates are important too (ref?). A strong imbalance between the single nucleotide-triphosphates or 
			a high concentration of base analogs can increase the rate of erroneous base incorporation (Gon et al. 2011). 
			
			This metabolic imbalance is achieved by the overexpression of <I>emrR</I>. The transcription repressor EmrR reduces the 
			expression of the repressor of the <I>emrAB</I> operon. This operon encodes an efflux pump, which is linked to multidrug resistance (Lomovskaya et al. 1995). 
			However, the native function of EmrAB is the export of intermediates of the purine and pyrimidine metabolism. 
			An overexpression of <I>emrR</I> reduces the concentration of EmrA and EmrB leading to increased concentration of nucleotide 
			precursors inside the cell. At high concentrations, these nucleotide precursors can act as base analogs. Mutations are 
			caused either incorporation of the nucleotide precursor or by failure during the repair attempt of an erroneous base incorporation(Gabrovsky et al. 2005).
		</DIV><DIV class="container text_header"><H3>Uracil glycosylase inhibitor and <I>Pteromyzon marinus</I> cytidin deaminase 1</H3></DIV><DIV class="container text"> 
			Cytidin deaminases (<I>cda</I>) are DNA editing enzymes in eukaryotes, which have multiple functions like the generation of antibody diversity, 
			defence against retroviruses and demethylation during development (Lada et al. 2011). 
			The main activity of CDAs is the deamination from cytidin to uracil in CG base pairs. The resulting UG base pairs split in replication 
			in one CG and one UA basepair. Therefore, the main mutations created by CDAs are C → T transitions. <FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 4: Deamination of cytidin by cytidin deasminases and possible resulting mutations by replication of base excision repair (BER).</B> 
					Cytidin deaminase (CDA) deaminases cytidin to uracil. The resulting GU base pair leads to a UA and a CG base pair during replication. 
					Alternatively, the uracil is repaired via the base excision repair. Thereby, uracil-DNA-glycosylase (UNG) cuts out uracil and the 
					abasic site (AP-site) inside the DNA gets repaired by specialized endonucleases and polymerases.
				</FIGCAPTION></FIGURE>
			Organisms have created mechanism to repair mismatches as UG pairs via the base excision repair (BER). Thereby the mismatched base is 
			cut out by the uracil-DNA-glycosylase (UNG). The resulting abasic site (AP-site) is cleaved by an AP-endonuclease and the patch is filled 
			by a variety of specialized proteins (Krokan, Bjoras 2013). 
			Overexpression of eukaryotic cytidin deaminase increases the mutation rate in bacteria. The CDA1 of <I>Pteromyzon marinus</I> was identified 
			in a comprehensive screen as the most potent cytidin deaminase when expressed in <I>E. coli</I>. Overexpression of <I>P. marinus</I><I>cda1</I> 
			increases the mutation rate to 10<SUP>-6</SUP>-10<SUP>-5</SUP> per bp. In <I>ung</I> deficient strains the effect was amplified by a factor of 50 
			due to the lack of UNG induced BER (Lada et al. 2011). 
			
			The CDS of <I>P. marinus</I><I>cda1</I> was codon optimized for <I>E. coli</I> integrated into our mutagenesis plasmid to increase the expression. 
			Moreover, the uracil-DNA-glycosylase inhibitor (<I>ugi</I>) was added. This short protein mimics the structure of abasic sides, binds to UNG and inhibits 
			the UNG mediated BER pathway (Serrano-Heras et al. 2007).
			<FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 5: Structural alignment of UNG-UGI and UNG-DNA complexes.</B> The <I>E. coli</I> UNG-UGI complex (PDB: 1LQG) was aligned to 
					<I>Deinococcus radiodurans</I> UNG-DNA complex (PDB: 4UQM). The structural mimicry of UGI (red) to DNA with an abasic side (blue) and 
					the binding to UNG (green) is highlighted. 
				</FIGCAPTION></FIGURE>
			See here how we characterized our genome wide mutator by <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">reversion assays</A> and <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Sequencing">high-throughput sequencing</A><CENTER><BUTTON><A class="button_link" href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation" role="button">Results</A></BUTTON></CENTER></DIV><DIV class="container text_header"><H3>References</H3></DIV><DIV class="container text"><UL><LI>Badran, Ahmed H.; Liu, David R. (2015): Development of potent in vivo mutagenesis plasmids with broad mutational spectra. In Nature communications 6, p. 8425. DOI: 10.1038/ncomms9425.
				</LI><LI>Cox, E. C.; Horner, D. L. (1986): DNA sequence and coding properties of mutD(dnaQ) a dominant Escherichia coli mutator gene. In Journal of molecular biology 190 (1), pp. 113–117.
				</LI><LI>Damagnez, V.; Doutriaux, M. P.; Radman, M. (1989): Saturation of mismatch repair in the mutD5 mutator strain of Escherichia coli. In Journal of bacteriology 171 (8), pp. 4494–4497.
				</LI><LI>Degnen, G. E.; Cox, E. C. (1974): Conditional mutator gene in Escherichia coli: isolation, mapping, and effector studies. In Journal of bacteriology 117 (2), pp. 477–487.
				</LI><LI>Fijalkowska, I. J.; Schaaper, R. M. (1996): Mutants in the Exo I motif of Escherichia coli dnaQ: defective proofreading and inviability due to error catastrophe. In Proceedings of the National Academy of Sciences of the United States of America 93 (7), pp. 2856–2861.</LI><LI>Fijalkowska, Iwona J.; Schaaper, Roel M.; Jonczyk, Piotr (2012): DNA replication fidelity in Escherichia coli: a multi-DNA polymerase affair. In FEMS microbiology reviews 36 (6), pp. 1105–1121. DOI: 10.1111/j.1574-6976.2012.00338.x.</LI><LI>Fowler, R. G.; Schaaper, R. M. (1997): The role of the mutT gene of Escherichia coli in maintaining replication fidelity. In FEMS microbiology reviews 21 (1), pp. 43–54.</LI><LI>Gabrovsky, Vanessa; Yamamoto, Mitsuko Lynn; Miller, Jeffrey H. (2005): Mutator effects in Escherichia coli caused by the expression of specific foreign genes. In Journal of bacteriology 187 (14), pp. 5044–5048. DOI: 10.1128/JB.187.14.5044-5048.2005.</LI><LI>Gon, Stephanie; Napolitano, Rita; Rocha, Walter; Coulon, Stephane; Fuchs, Robert P. (2011): Increase in dNTP pool size during the DNA damage response plays a key role in spontaneous and induced-mutagenesis in Escherichia coli. In Proceedings of the National Academy of Sciences of the United States of America 108 (48), pp. 19311–19316. DOI: 10.1073/pnas.1113664108.</LI><LI>Greener, A.; Callahan, M.; Jerpseth, B. (1997): An efficient random mutagenesis technique using an E. coli mutator strain. In Molecular biotechnology 7 (2), pp. 189–195. DOI: 10.1007/BF02761755.</LI><LI>Horst, J. P.; Wu, T. H.; Marinus, M. G. (1999): Escherichia coli mutator genes. In Trends in microbiology 7 (1), pp. 29–36.</LI><LI>Junop, Murray S.; Yang, Wei; Funchain, Pauline; Clendenin, Wendy; Miller, Jeffrey H. (2003): <I>In vitro</I> and <I>in vivo</I> studies of MutS, MutL and MutH mutants: correlation of mismatch repair and DNA recombination. In DNA repair 2 (4), pp. 387–405.</LI><LI>Kirby, T. W.; Harvey, S.; DeRose, E. F.; Chalov, S.; Chikova, A. K.; Perrino, F. W. et al. (2006): Structure of the E. coli Pol III epsilon-Hot proofreading complex.</LI><LI>Krokan, Hans E.; Bjoras, Magnar (2013): Base excision repair. In Cold Spring Harbor perspectives in biology 5 (4), pp. a012583. DOI: 10.1101/cshperspect.a012583.</LI><LI>Lada, A. G.; Krick, C. Frahm; Kozmin, S. G.; Mayorov, V. I.; Karpova, T. S.; Rogozin, I. B.; Pavlov, Y. I. (2011): Mutator effects and mutation signatures of editing deaminases produced in bacteria and yeast. In Biochemistry. Biokhimiia 76 (1), pp. 131–146.</LI><LI>Lee, Heewook; Popodi, Ellen; Tang, Haixu; Foster, Patricia L. (2012): Rate and molecular spectrum of spontaneous mutations in the bacterium Escherichia coli as determined by whole-genome sequencing. In Proceedings of the National Academy of Sciences of the United States of America 109 (41), p. E2774-83. DOI: 10.1073/pnas.1210309109.</LI><LI>Lomovskaya, O.; Lewis, K.; Matin, A. (1995): EmrR is a negative regulator of the Escherichia coli multidrug resistance pump EmrAB. In Journal of bacteriology 177 (9), pp. 2328–2334.</LI><LI>Nghiem, Y.; Cabrera, M.; Cupples, C. G.; Miller, J. H. (1988): The mutY gene: a mutator locus in Escherichia coli that generates G.C----T.A transversions. In Proceedings of the National Academy of Sciences of the United States of America 85 (8), pp. 2709–2713.</LI><LI>Schaaper, R. M. (1988): Mechanisms of mutagenesis in the Escherichia coli mutator mutD5. Role of DNA mismatch repair. In Proceedings of the National Academy of Sciences 85 (21), pp. 8126–8130. DOI: 10.1073/pnas.85.21.8126.</LI><LI>Schaaper, R. M. (1993): Base selection, proofreading, and mismatch repair during DNA replication in Escherichia coli. In The Journal of biological chemistry 268 (32), pp. 23762–23765.</LI><LI>Schaaper, R. M.; Radman, M. (1989): The extreme mutator effect of Escherichia coli mutD5 results from saturation of mismatch repair by excessive DNA replication errors. In The EMBO journal 8 (11), pp. 3511–3516.</LI><LI>Selifonova, O.; Valle, F.; Schellenberger, V. (2001): Rapid evolution of novel traits in microorganisms. In Applied and environmental microbiology 67 (8), pp. 3645–3649. DOI: 10.1128/AEM.67.8.3645-3649.2001.</LI><LI>Serrano-Heras, Gemma; Ruiz-Maso, Jose A.; del Solar, Gloria; Espinosa, Manuel; Bravo, Alicia; Salas, Margarita (2007): Protein p56 from the Bacillus subtilis phage phi29 inhibits DNA-binding ability of uracil-DNA glycosylase. In Nucleic acids research 35 (16), pp. 5393–5401. DOI: 10.1093/nar/gkm584.</LI><LI>Steensels, Jan; Snoek, Tim; Meersman, Esther; Picca Nicolino, Martina; Voordeckers, Karin; Verstrepen, Kevin J. (2014): Improving industrial yeast strains: exploiting natural and artificial diversity. In FEMS microbiology reviews 38 (5), pp. 947–995. DOI: 10.1111/1574-6976.12073.</LI><LI>Stefan, Alessandra; Radeghieri, Annalisa; Gonzalez Vara y Rodriguez, Antonio; Hochkoeppler, Alejandro (2001): Directed evolution of β-galactosidase from Escherichia coli by mutator strains defective in the 3′→5′ exonuclease activity of DNA polymerase III. In FEBS Letters 493 (2-3), pp. 139–143. DOI: 10.1016/S0014-5793(01)02293-1.</LI><LI>Wilson, D. S.; Keefe, A. D. (2001): Random mutagenesis by PCR. In Current protocols in molecular biology / edited by Frederick M. Ausubel … [et al.] Chapter 8, p. Unit8.3. DOI: 10.1002/0471142727.mb0803s51.</LI><LI>Wong, Tuck Seng; Zhurina, Daria; Schwaneberg, Ulrich (2006): The diversity challenge in directed protein evolution. In Combinatorial chemistry &amp; high throughput screening 9 (4), pp. 271–288.</LI><LI>Xue, Yali; Wang, Qiuju; Long, Quan; Ng, Bee Ling; Swerdlow, Harold; Burton, John et al. (2009): Human Y chromosome base-substitution mutation rate measured by direct sequencing in a deep-rooting pedigree. In Current biology : CB 19 (17), pp. 1453–1457. DOI: 10.1016/j.cub.2009.07.032.</LI><LI>Yang, Hanjing; Wolff, Erika; Kim, Mandy; Diep, Amy; Miller, Jeffrey H. (2004): Identification of mutator genes and mutational pathways in Escherichia coli using a multicopy cloning approach. In Molecular microbiology 53 (1), pp. 283–295. DOI: 10.1111/j.1365-2958.2004.04125.x.</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>