<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Results_Mutation_Assembly skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Results/Mutation/Assembly</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container main"><DIV id="header"><NAV id="navigationbar" class="navbar navbar-default custom_navbar_style navbar-fixed-top"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="#">NavBar</A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-center"><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description"> PROJECT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description">Description</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds">Scaffolds</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design">Design and Construction</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/CreateYours">Create your own Library</A></LI></UL><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/EpPolI">Error prone PolymeraseI</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/Global">Genome wide mutator</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">Bacterial Two-Hybrid System</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/SplitBetaLactamaseHitchHiker">Alternative Selection System</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_I">Model I</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_II">Model II</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results"> RESULTS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Phage">Phagemid Display</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">Reversion</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/Cloning">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/ExpressionControl">Expression Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/InteractionControl">Interaction Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/BindingControl">Binding Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/KnockOutKnochIn"><I>in vivo</I> Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Modeling">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Fermentation">Fermentation</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Best Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Best Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Best Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts/Improve">Improve a part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Demonstrate your Work</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Medal_Fulfillment">Medal Fulfillment</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold"> HUMAN PRACTICES </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Dictionary">Dictionary</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks">Educational Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Crowdfunding">Crowdfunding and Press</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Achievements</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations"> COLLABORATIONS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Duesseldorf">Düsseldorf</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Lethbridge">Lethbridge</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Freiburg">Freiburg</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments"> NOTEBOOK </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook">Lab Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">Primers</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols">Lab Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Media">Media and Buffers</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team"> TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team">Team Profile</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Members">Members</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Contact">Contact</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement"> ACKNOWLEDGEMENT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Sponsors_And_Partners">Sponsors and Partners</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></DIV></DIV></NAV></DIV></DIV><DIV class="container main"><DIV class="jumbotron" style="background-image: url(https://static.igem.org/mediawiki/2016/0/02/Bielefeld_CeBiTec_2016_10_14_X_projectheader.png)"><DIV class="jumbotron-text"><H1 style="margin-bottom: 0px; text-align:left">Results basic parts &amp; assembly of the mutators</H1></DIV></DIV><DIV class="container text_header" id="promoter"><H3>Overview</H3></DIV><DIV class="container text">
			We want to construct parts for <I>in vivo</I> mutagensis and assemble them to two mutators systems, which pose the main
			driving force for our ongoing Evobody evolution system.
			
			In this article we will sum up our design and construction of the mutator constructs used for <I>in vivo</I> characterization by 
			<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">reversion assays</A> and <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Sequencing">high-throughput sequencing</A> as well as the associated reporter systems.
			<UL><LI><A href="#promoter">Promoter</A></LI><LI><A href="#eppol">Error prone polymerase I</A></LI><LI><A href="#m6">Genome wide mutator</A></LI><LI><A href="#reporter">Reporter</A></LI></UL></DIV><DIV class="container text_header" id="promoter"><H3>Promoter</H3></DIV><DIV class="container text">
			We started by choosing the araC-P<SUB>BAD</SUB> regulatory unit from 
			<A href="http://parts.igem.org/Part:BBa_K808000">BBa_K808000</A>. This promoter is known for low basal expression and is therfore the 
			perfect regulatory unit for the expression of potentially toxic mutator genes. Furthermore we wanted to modify this promoter to 
			yield an even lower basal activity.
			Therefore, we screened the <A href="http://parts.igem.org/PBAD_SPL">synthetic P<SUB>BAD</SUB> library</A> 
			from <A href="https://2013.igem.org/Team:DTU-Denmark">iGEM DTU-Denmark 2013</A> and chose the promoter from Col.13. Advantages of this promoter
			are the very low basal activity while still maintaining a high activity when induced.
			
			We obtained our modified P<SUB>BAD</SUB><A href="http://parts.igem.org/Part:BBa_K2082112">BBa_K2082112</A> through site-directed mutagenesis of 
			<A href="http://parts.igem.org/Part:BBa_K808000">BBa_K808000</A>.
			<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 1: Alignment between promoter sequence from <A href="http://parts.igem.org/Part:BBa_K808000">BBa_K808000</A> (wt-P<SUB>BAD</SUB> ) 
					and the modified P<SUB>BAD</SUB> sequence.</B></FIGCAPTION></FIGURE></CENTER>
			In addition to the low basal activity, the P<SUB>BAD</SUB> has the advantage of tuneable experssion strength, that is controlled by addition of different amounts of arabinose.
			We estimated that controllable amounts of mutagenic proteins 
			would also result in different amount of mutagenesis, thus enabling a tuneable mutagenesis.
			
			In addition to the titrable promoter activity by adding arabinose P<SUB>BAD</SUB> can be repressed by adding glucose. Glucose decreases intracellular 
			3',5'-cyclic AMP concentration, thereby lowering expression of P<SUB>BAD</SUB> promoter (Miyada et al. 1984).
			
			
			We characterized our modified P<SUB>BAD</SUB><A href="http://parts.igem.org/Part:BBa_K2082112">BBa_K2082112</A> by adding the RFP expression system 
			<A href="http://parts.igem.org/Part:BBa_K516032">BBa_K516032</A> downstream and measurement of RFP after different amounts 
			of time and with various amounts of arabinose added. The measurement was performed in the Top10 strain, which is deficient for arabinose metabolism (araD139 Δ(ara-leu)7697).
			<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 3: Measured expression of <A href="http://parts.igem.org/Part:BBa_K2082106">BBa_K2082112</A> in comparison to <A href="http://parts.igem.org/Part:BBa_K808000">BBa_K808000</A>.</B>
					RFP expression via <A href="http://parts.igem.org/Part:BBa_K516032">BBa_K516032</A> was used for promoter characterization, and the promoter was induced by addition of arabinose. After 300 min RFP fluorescence was normalized 
					on OD<SUB>600</SUB>.
				</FIGCAPTION></FIGURE></CENTER>
			No difference in leakiness could be observed. The addition of 20 mM glucose represses the promoter slightly and decreases promoter strength by about 65 % to 52 %, compared to no addition. Addition of arabinose activates the promoter, in which increased amounts of inductor leads to increased promoter activity.
			In total is the modified P<SUB>BAD</SUB><A href="http://parts.igem.org/Part:BBa_K2082106">BBa_K2082112</A> a bit weaker than <A href="http://parts.igem.org/Part:BBa_K808000">BBa_K808000</A>.
			The main aim in decreasing promoter basal activity was not proven. Perhaps the used method has to be optimized further for detection the already small basal activity of
			<A href="http://parts.igem.org/Part:BBa_K808000">BBa_K808000</A> and compare it with our modified promoter.
		</DIV><DIV class="container text_header" id="eppol"><H3>Error-prone polymerase I</H3></DIV><DIV class="container text">
			For the polymerase I we contacted Manel Camps, the leading expert on this field. (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Integrated_Practices">Read more about our 
			correspondence with Manel Camps here</A>) He told us that the two-plasmid mutagenesis system is a complex mechanism. 
			This means that changes in the expression levels of error-prone polymerase (e.g. overexpression) oftentimes yields no 
			additional mutagenesis effect. Because of this we decided to use the complete genetic construct provided by M. Camps as 
			device for mutagenesis with the polymerase I. The devices contains the error-prone polymerase I under control of the 
			<I>lac</I>-promoter.  
			<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 2: Schematic view of the error-prone polymerase I under control of a <I>lac</I>-promoter.</B></FIGCAPTION></FIGURE></CENTER>
			We made the error-prone polymerase I coding sequence aviable asexpression device available as basic part (<A href="http://parts.igem.org/Part:BBa_K2082106">BBa_K2082106</A>). 
			As negative controls we also added the wild type <I>E. coli</I> polymerase I coding sequence (<A href="http://parts.igem.org/Part:BBa_K2082107">BBa_K2082107</A>)

			We characterized <A href="http://parts.igem.org/Part:BBa_K2082106">BBa_K2082106</A> and <A href="http://parts.igem.org/Part:BBa_K2082107">BBa_K2082107</A> under
			control of a <I>lac</I>-promoter.
			We used the expression devices for the error-prone polymerase I and the wild type polymerase I in the plasmid pHSG, which resemebles the iGEM 
			backbone pSB4C5, because of its pSC101 origin of replication.
		</DIV><DIV class="container text_header" id="m6"><H3>Genome wide mutator</H3></DIV><DIV class="container text">
			We wanted to port the mutator plasmid described by Badran and coworkers (Badran and Liu 2015) to the iGEM standard.
			This is a mutation device consisting of six genes which modulate the <I>E. coli</I> fidelity mechanism 
			(for a detailed explanation see <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/Global">here</A>). 
			Because of the potential hazardous effect of the potent mutator genes we decided to use the described, tightly regulated 
			arabinose promoter. In addition we thought that low level expression of the mutagenic protein is enough to achieve a mutator phenotype. 
			Furthermore the expression strength of our arabinose promoter under complete induction is high enough to obtain sufficient 
			amounts of proteins even when more is needed.
			
			Therefore, we decided that it is more beneficial for the controllability of our mutator constrauct to use a very weak ribosome binding 
			site. We used <A href="http://parts.igem.org/Part:BBa_B0031">BBa_B0031</A> as appropriately pretty weak RBS.
			
			As for the genes for our mutator plasmid we build upon the work done by Badran and coworkers, who already tested several 
			combinations of mutator genes and found a combination with high mutagenesis rate and broad spectrum. 
			
			The six genes are <I>dnaQ926</I>, <I>dam</I>, <I>seqA</I>, <I>emrR</I>, <I>ugi</I> and <I>P. marinus cda1</I>. 
			<A href="http://parts.igem.org/Part:BBa_K1333108"><I>dnaQ926</I></A>, <A href="http://parts.igem.org/Part:K1500994"><I>dam</I></A> and 
			<A href="http://parts.igem.org/Part:K1500995"><I>emrR</I></A> are already in the iGEM parts registry so we ended up using them. 
			(see <A href="">here</A> how our results further characterise these parts)
			
			We added the coding sequences for <A href="http://parts.igem.org/Part:BBa_K2082103"><I>seqA</I></A>, <A href="http://parts.igem.org/Part:BBa_K2082104"><I>ugi</I></A> 
			and <A href="http://parts.igem.org/Part:BBa_K2082105"><I>cda1</I></A> via gene synthesis.
			
			We assembled the mutagenesis genes by Gibson Assembly and added <A href="http://parts.igem.org/Part:BBa_B0031">BBa_B0031</A> in the 
			process in front of each mutation gene. To easily assemble six genes the synthesis was done in two steps. At first <I>dnaQ926</I>, <I>dam</I> and 
			<I>seqA</I> respectively <I>emrR</I>, <I>ugi</I> and <I>cda1</I> were combined. Sadly the first assembly could only be isolated without the <I>dnaQ926 RBS</I>. In the 
			following step both synthesis intermediates were combined by <A href="2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols">BioBrick Assembly</A>. To add the RBS in front of <I>dnaQ926</I> and to add a terminator behind the construct 
			the complete genes were cloned into <A href="http://parts.igem.org/Part:BBa_K516031">BBa_K516031</A>. The complete coding sequences with 
			ribosome binding site were cloned downstream of our modified P<SUB>BAD</SUB> to obtain the final construct <A href="http://parts.igem.org/ Part:BBa_K2082117">BBa_K208117</A>.
			<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 4: Schematic view of the cloning process leading to <A href="http://parts.igem.org/Part:BBa_K2082117">BBa_K208117</A>.</B></FIGCAPTION></FIGURE></CENTER>
			To assess the effect of seperate expression of <I>dnaQ926</I>, which is a very strong mutator in itself, this gene alone was cloned into <A href="http://parts.igem.org/Part:BBa_K516031">BBa_K516031</A>, 
			added downstream of our modified P<SUB>BAD</SUB>, which yielded <A href="http://parts.igem.org/Part:BBa_K2082116">BBa_K2082116</A>.
			<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 5: Schematic view of <A href="http://parts.igem.org/Part:BBa_K2082116">BBa_K2082116</A>.</B></FIGCAPTION></FIGURE></CENTER></DIV><DIV class="container text_header" id="reporter"><H3>Reporter</H3></DIV><DIV class="container text">
			To characterise our mutation systems we did <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">reversion assays</A>. Therefore, we needed reporter proteins, which activity could 
			be easily measured. We decided on <I>beta-lactamase</I> and <I>gfp</I> as reporters because of the easy detection assays.
			
			Before deciding where to put the stop-codon we put a lot of effort into identifying suitable positions. We had two concerns:
			<UL><LI>Can we make sure that the stop-codon is outside of the okazaki-fragments?</LI><LI>Can we make sure that every single base pair change, that does not change the stop codon to another stop codon, reverts the reporter back to function?</LI></UL>
			First first points is due to the irregular distribution of mutations when using the error-prone polymerase I. Because of the enzymes role in processing 
			okazaki fragments these region are hotspots to mutations. The mutation rate of a hotspot region would appear higher than that of the main part of the mutated region. 
			
			To solve this problem we choose to create two different reporters, where the positions of the stop codons are separated at least 120 bp from each other.
			
			The next concern was that there are positions inside proteins where the amino acids is remarkable important for the proteins function (e.g. amino acids 
			inside the catalytic center) or folding. We wanted to avoid these important positions, because at these positions only reversions back to the original amino 
			acid would reconstitute the protein. The catalytic center for both of our reporter proteins are known (Figure 7) and can be avoided. To avoid positions 
			responsible for folding of the proteins the impact of point mutation on the folding was modelled using <A href="http://foldxsuite.crg.eu/">FoldX</A>. Each amino acid was <I>in silico</I> 
			mutated to each other amino acid and the protein stability was calculated. The complete results for the positions 100 to 109 are depicted in Figure 6a, the impact of alanine replacement on every position was mapped on the
			GFP crystal structure as an overview (Figure 6b). Striking is the importance of Gly104, where every change in amino acid greatly destabilizes the protein. Based on these modelling results (complete scan like Figure 6a) two positions for introduction of a stop codon were determined (Figure 7).
			In addition, a stop position inside <I>gfp</I> already used in reversion experiments were used as well. (Wang et al. 2004)
			<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 6: Extensive <I>in silico</I> stability scan for GFP (A) and mapping of the stability changes for a complete <I>in silico</I> alanine scan (B).</B> 
					The stability changes for mutating each position inside GFP were calculated usign FoldX. The stability changes are calculated in ΔΔG (difference of mutant folding energy to wild type folding energy, low means more stable).
					As overview the complete values for mutating to alanine were mapped on the GFP structure. (PDB: 2WUR)
				</FIGCAPTION></FIGURE></CENTER><CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 7: Coding sequence for <I>beta-lactamase</I> and <I>gfp</I> with indicated function-essential amino acids as well as introduced stop-codons.</B></FIGCAPTION></FIGURE></CENTER>
			These stop codons were introduced by site-directed mutagenesis of <A href="http://parts.igem.org/Part:BBa_K608010">BBa_K608010</A> and the beta lactamase from pSB1AK3 creating 
			the following BioBricks:
			<CENTER><TABLE><TBODY><TR><TD rowspan="2"><I>beta-lactamase</I></TD><TD>K30<SUP>∗</SUP></TD><TD>L79<SUP>∗</SUP></TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K2082151">BBa_K2082151</A></TD><TD><A href="http://parts.igem.org/Part:BBa_K2082152">BBa_K2082152</A></TD></TR><TR><TD rowspan="2"><I>gfp</I></TD><TD>K3<SUP>∗</SUP></TD><TD>Y106<SUP>∗</SUP></TD><TD>K107<SUP>∗</SUP></TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K2082113">BBa_K2082113</A></TD><TD><A href="http://parts.igem.org/Part:BBa_K2082114">BBa_K2082114</A></TD><TD><A href="http://parts.igem.org/Part:BBa_K2082115">BBa_K2082115</A></TD></TR></TBODY></TABLE></CENTER>
			The stop-GFP's were analysed by flow cytometry (Figure 8) and complete loss of fluorescence by introduction of a stop-codon was observed.
			<FIGURE class="figure"><TABLE><TBODY><TR><TD>A)</TD><TD>B)</TD><TD>C)</TD></TR></TBODY></TABLE><FIGCAPTION class="figure-caption"><B>Figure 8: Characterisation of stop-GFP by flow cytometry.</B> The different stop-GFPs (green) were analysed by flow cytometry and compared
					to wt-GFP (red, <A href="http://parts.igem.org/Part:BBa_E0040">BBa_E0040</A>).
				</FIGCAPTION></FIGURE><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">Read on and see how we used our assembled mutators</A></DIV><CENTER><BUTTON><A class="button_link" href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion" role="button">Reversion</A></BUTTON><BUTTON><A class="button_link" href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Sequencing">Sequencing</A></BUTTON></CENTER><DIV class="container text_header"><H3>References</H3></DIV><DIV class="container text"><UL><LI>Badran, Ahmed H.; Liu, David R. (2015): Development of potent in vivo mutagenesis plasmids with broad mutational spectra. In: Nature communications 6, S. 8425. DOI: 10.1038/ncomms9425.</LI><LI>Miyada, C. G.; Stoltzfus, L.; Wilcox, G. (1984): Regulation of the araC gene of Escherichia coli: catabolite repression, autoregulation, and effect on araBAD expression. In: Proceedings of the National Academy of Sciences of the United States of America 81 (13), S. 4120–4124.</LI><LI>Wang, Clifford L.; Harper, Ryan A.; Wabl, Matthias (2004): Genome-wide somatic hypermutation. In: Proceedings of the National Academy of Sciences of the United States of America 101 (19), S. 7352–7356. DOI: 10.1073/pnas.0402009101.</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>