<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Project_Library_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Project/Library/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container main"><DIV id="header"><NAV id="navigationbar" class="navbar navbar-default custom_navbar_style navbar-fixed-top"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="#">NavBar</A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-center"><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description"> PROJECT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description">Description</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds">Scaffolds</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design">Design and Construction</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/CreateYours">Create your own Library</A></LI></UL><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/EpPolI">Error prone PolymeraseI</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/Global">Genome wide mutator</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">Bacterial Two-Hybrid System</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/SplitBetaLactamaseHitchHiker">Alternative Selection System</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_I">Model I</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_II">Model II</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results"> RESULTS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Phage">Phagemid Display</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">Reversion</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/Cloning">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/ExpressionControl">Expression Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/InteractionControl">Interaction Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/BindingControl">Binding Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/KnockOutKnochIn"><I>in vivo</I> Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Modeling">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Fermentation">Fermentation</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Best Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Best Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Best Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts/Improve">Improve a part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Demonstrate your Work</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Medal_Fulfillment">Medal Fulfillment</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold"> HUMAN PRACTICES </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Dictionary">Dictionary</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks">Educational Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Crowdfunding">Crowdfunding and Press</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Achievements</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations"> COLLABORATIONS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Duesseldorf">Düsseldorf</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Lethbridge">Lethbridge</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Freiburg">Freiburg</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments"> NOTEBOOK </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook">Lab Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">Primers</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols">Lab Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Media">Media and Buffers</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team"> TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team">Team Profile</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Members">Members</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Contact">Contact</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement"> ACKNOWLEDGEMENT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Sponsors_And_Partners">Sponsors and Partners</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></DIV></DIV></NAV></DIV></DIV><DIV class="container main"><DIV class="jumbotron" style="background-image: url(https://static.igem.org/mediawiki/2016/8/89/Bielefeld_CeBiTec_2016_10_14_X_jessi2.png)"><DIV class="jumbotron-text"><H1 style="margin-bottom: 0px; text-align:left">Library Project</H1><H2 style="color:#ffffff; text-align:left">Design and Construction</H2></DIV></DIV><DIV class="container text_header"><H3>Library Construction Area</H3></DIV><DIV class="container text">As starting point of our directed evolution system for binding proteins we needed a numerousness of partly randomized Monobodies and Nanobodies, respectively, to emerge our Evobodies. An Evobody is a binding protein resulting from the combination of a <A href="2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds">scaffold</A>, be it Monobody or Nanobody, with a continuously mutation and selection system, and thus represents the fusion of a semi rational design approach with evolution. 
We designed two libraries for Monobodies and Nanobodies, respectively. A library is a collection of identical plasmids that only vary in the protein coding sequences (CDS). These CDS are designed to possess optimized and planned randomized subregions displaying high variability. The result is a wide variety of plasmids with different inserts. One major advantage of a library is the availability of a wide range of different binding proteins as starting material for the evolution process, which ensures that candidates with limited but evolvable affinity are present from the outset. After transformation into <I>E. coli</I> a heterogeneous culture is created and after plating the transformation each colony carries a different insert encoding a different binding protein (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design#Osoegawa">Osoegawa <I>et al.</I>, 2001</A>). 
By using specific mixtures of bases for defined positions in the oligonucleotide synthesis resulting in degenerate codons coding for a defined mix of amino acids in the variable regions of our scaffolds we achieve a theoretical diversity of 1,073,741,824 proteins sequences for each library. 
Since the practical diversity of our library is limited by the transformation efficiency of <I>E. coli</I>, the theoretical library size needed to be chosen carefully to maximize the binding the potential while avoiding dilution in unproductive (i.e. non-folding or non-binding) sequence space. Furthermore, due to our evolution system, only a weak affinity is required in the first place which can then mature to high affinity. We decided to optimize the library by using restricted yet proven codon or amino acids set, respectively, for the complementary determining regions (CDRs). As a realistic library size, we obtained for a diversity of about one billion varieties. For constructing the coding sequence, we utilized the IUPAC ambiguity base naming scheme for the ordering of our oligonucleotides at a vendor, which did support the equimolar mixing of nucleotides. 

First of all, in our library design, we avoided stop codons to guarantee the synthesis of the complete binding proteins. By exclusion of cysteine encoding triplets, we ensured the absence of disulfide bonds and compatibility with the intracellular and extracellular environment. Furthermore, we preferred amino acids that are beneficial to a high binding affinity. Evolution has optimized natural proteins for specific biological functions. Proteins with similar tasks and features also show similar structures and contain related amino acids. Likewise, the binding hot spots for protein-protein interactions are often enriched in tyrosine, tryptophan and arginine (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design#Bogan">Bogan and Thorn, 1998</A>). Especially tyrosine is overrepresented in high affinity CDRs in functional antibodies. Round about 40 % of their sequence consists of tyrosine. Another ~30 % is assembled of small amino acids like serine, glycine, alanine and threonine (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design#Mian">Mian <I>et al.</I>, 1991</A>; <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design#Zemlin">Zemlin <I>et al.</I>, 2003</A>). Scientific studies also have shown the importance of tyrosine in the CDS of synthetic binding proteins (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design#Fellouse2004">Fellouse <I>et al.</I>, 2004</A>). The tyrosine side chain bears most of the contacts necessary for high affinity antigen recognition in synthetic libraries of binding proteins. Firstly, because of its big size to fill large volumes with just a few angels of torsion (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design#Fellouse2007">Fellouse <I>et al.</I>, 2007</A>), while smaller amino acids provide the necessary space and protein conformation flexibility (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design#Koide">Koide <I>et al.</I>, 2010</A>). The size of tyrosine also leads to many van der Waals´ contacts and the -OH group provides electrostatic interactions for binding (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design#Mian">Mian <I>et al.</I>, 1991</A>). The amphipathic nature of the tyrosine is also helpful in the different hydrophobic and hydrophilic environments in antibody-antigen complexes (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design#Mian">Mian <I>et al.</I>, 1991</A>). 
In contrast to other high affinity providing amino acids, tyrosine does not show an outstanding flexibility. This fact leads to another benefit of tyrosine in binding proteins by providing a high specificity (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design#Koide">Koide <I>et al.</I>, 2010</A>). Beside high affinity, specificity is another important requirement for a good Evobody. 

On these grounds, we designed the randomized codons TMY, KMY, RMR, YWY, NWY and WMY. 
Table 1 shows the used randomized IUPAC nucleotide naming and the encoded bases: 
<TABLE class="table"><THEAD><TR><TH>Degenerated base designation</TH><TH>Actual bases coded</TH></TR></THEAD><TBODY><TR><TD>M</TD><TD>A/C</TD></TR><TR><TD>Y</TD><TD>C/T</TD></TR><TR><TD>K</TD><TD>G/T</TD></TR><TR><TD>W</TD><TD>A/T</TD></TR><TR><TD>R</TD><TD>A/G</TD></TR><TR><TD>M</TD><TD>A/C</TD></TR></TBODY></TABLE>
Table 2: used randomized codons and resulting amino acids:

<TABLE class="table"><THEAD><TR><TH>Designed randomized codon</TH><TH>Actual amino acids encoded</TH></TR></THEAD><TBODY><TR><TD>TMY</TD><TD>Tyrosine, Serine </TD></TR><TR><TD>KMY</TD><TD>Tyrosine, Serine, Alanine, Aspartic Acid </TD></TR><TR><TD>WMY</TD><TD>Tyrosine, Serine, Threonine, Asparagine </TD></TR><TR><TD>RMR</TD><TD> Threonine, Alanine, Lysine, Glutamic Acid </TD></TR><TR><TD>YWY</TD><TD>Phenylalanine, Serine, Isoleucine, Threonine </TD></TR><TR><TD>NWY</TD><TD>Phenylalanine, Leucine, Isoleucine, Valine, Tyrosine, Histidine, Asparagine, Aspartic Acid</TD></TR></TBODY></TABLE>
Finally, we achieved a theoretic variability of 1,073,741,824 different molecules for Monobodies and Nanobodies, respectively. 

 </DIV><DIV class="container text_header"><H3>Implementation of the Evobody Libraries</H3></DIV><DIV class="container text">
The standard plasmid pSB1K3 was extended with the required parts of the <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">selection system</A>. Finally, the respective binding protein got fused with RpoZ, the omega domain of RNA polymerase, via a c-Myc-linker to enable selection for good target affinity.  In the first steps, the constant regions of our binding protein scaffolds (Monobody and Nanobody) were ordered as gene as synthesized IDT G-Blocks® and inserted in the vector via Gibson assembly. 
After finding the optimal randomized codon scheme for Monobodies and Nanobodies, respectively, we had to decide how to generate the library. Most of the gene synthesis companies do not offer long gene fragments with partly randomized sequences and if they do the synthesis of a whole library is prohibitively expensive (about 5-10 k€) and time consuming. Therefore, we decided to do the gene synthesis of the library ourselves and ordered a small number of larger, partly randomized oligonucleotides (from Metabion). 
The ordered oligonucleotides had complementary overlaps for annealing. Double stranded fragments were generated by a single touch-down fill in reaction to achieve and maintain maximum diversity. After generating the double stranded library fragments, these were cloned in the library vector using a Gibson assembly, due to the incorporated overlap with the vector. 
<B>Monobodies:</B></DIV><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 1: Overview of the Monobody construction.</B>Constant regions are colored green, variable regions are colored blue. Small letters denote the oligonucleotides which were used for library assembly.</FIGCAPTION></FIGURE><DIV class="container text">
Variable Monobody Oligonucleotides:
<TABLE class="table"><THEAD><TR><TH> Oligonucleotide </TH><TH> Length [bp]</TH><TH> Sequence</TH></TR></THEAD><TBODY><TR><TD> a </TD><TD> MB-V1-1 </TD><TD>60 </TD><TD>TCTTGGGACGCTCCGGCTGTTACCGTTNWYYWYTACNWYATTACTTATGGCGAGACTGGC </TD></TR><TR><TD> b </TD><TD> MB-V1-2 </TD><TD>75</TD><TD>GGTAGCGGTAGATTTAGAACCCGGAACYKYGAAYKYCTGRKARKARKMRKARKAGCCAGTCTCGCCATAAGTAAT </TD></TR><TR><TD> c </TD><TD> MB-V2-1 </TD><TD>75 </TD><TD>GTTCCGGGTTCTAAATCTACCGCTACTATCTCTGGTCTGTCTCCGGGTGTTGACTATACCATCACCGTTTACGCT </TD></TR><TR><TD> d </TD><TD> MB-V2-2 </TD><TD>80 </TD><TD>GGTACGGTAGTTGATAGAGATCGGAGARKARKARKARKMRKARKARKMRKARKARKAAGCGTAAACGGTGATGGTATAGT </TD></TR></TBODY></TABLE>
To create our Monobody library, we constructed the fundamental framework (<A href="parts.igem.org/wiki/index.php?title=Part:BBa_K2082004">BBa_K2082004</A>) composed of the Monobody constant regions with an inserted RFP as a place holder for the variable regions, which can easily be exchanged. Therefore, we ordered a synthesized IDT G- Blocks® and assembled it using Gibson cloning. To achieve a complete Monobody from our submitted part, the RFP can be replaced with randomized variable regions allowing an easy visual control. To insert the variable regions, we amplified the backbone, including Monobody constant regions, the cMyc-linker and <I>rpoZ</I> by using the primers <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">MB-bb-fw</A> and <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">MB-bb-rev</A>. After annealing the single stranded oligonucleotides V1-1 (a) + V1-2 (b) filling in the parts resulting in the double stranded fragment V1 and annealing oligonucleotides V2-1 (c) + V2-2 (d) and fill-in yielding V2 and a Gibson assembly with V1, V2 and the backbone combined the parts. 
<B>Nanobodies:</B></DIV><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 1: Overview of the Nanobody construction.</B>Constant regions are colored green, variable regions are colored blue. Small letters denote the oligonucleotides which were used for library assembly.</FIGCAPTION></FIGURE><DIV class="container text">
Variable Nanobody Oligonucleotides:
<TABLE class="table"><THEAD><TR><TH> Oligonucleotide </TH><TH> Length [bp]</TH><TH> Sequence</TH></TR></THEAD><TBODY><TR><TD> a </TD><TD> NB-F2-1</TD><TD>102 </TD><TD>GACACCGCTATCTACTACTGCGCTGCTWMYKMYWMYKMYWMYTMYKMYWMYKMYTMYWMYKMYWMYKMYWMYKMYTGGGGTCAGGGTACGCAGGTTACCGTT </TD></TR><TR><TD> b </TD><TD> NB-F2-2</TD><TD>65</TD><TD>CTGTCAGGGGCGGGGTTTTTTTTTCTCTAGTAAGAAGAAACGGTAACCTGCGTACCCTGACCCCA</TD></TR></TBODY></TABLE>
Similar to Monobodies, we built up a universal framework for Nanobodies, containing rpoZ, c-Myc-linker and a constant Nanobody segment (<A href="parts.igem.org/wiki/index.php?title=Part:BBa_K2082001">BBa_K2082001</A>). As opposed to the Monobody CDS in this case we only designed one variable insert, because in the used scaffold, we have only randomized the third Complementarity-determining region (CDR3), which is known to be the main binding CDR (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design#Krebber">Krebber <I>et al.</I>, 1997</A>). Amplifying <A href="parts.igem.org/wiki/index.php?title=Part:BBa_K2082001">BBa_K2082001</A> with primers <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">NB-bb-fw</A> and <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">NB-bb-rev</A> results in a backbone, which was used to insert the annealed and replenished variable fragment F2 = F2-1 (a) + F2-2 (b) by Gibson assembly.</DIV><DIV class="container text_header"><H3>Proof of Concept</H3></DIV><DIV class="container text">The core of our system is the combination of the library with a mutation system and a selection system, to optimize binding proteins to Evobodies. To get in this directed evolution process, some binding proteins must show at least a marginal affinity to the chosen target protein. Low affinity binders then get mutated and selected in vivo during plasmid replication and cell growth under a rising selective pressure. To make sure, that the designed binding proteins work and that the initial libraries contained some useful Monobodies and Nanobodies, respectively, we performed a <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Phage">phagemid display</A>, using phagemid pAK100 and M13 derived helper phages.
</DIV><CENTER><BUTTON><A class="button_link" href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/CreateYours" role="button">Create your own library</A></BUTTON></CENTER><DIV class="container text_header"><H3>References</H3></DIV><DIV class="container text"><UL><LI id="Bogan">
Bogan, A. A.; Thorn, K. S. (1998): Anatomy of hot spots in protein interfaces. In: <I>Journal of Molecular Biology</I> 280 (1), S. 1–9. DOI: 10.1006/jmbi.1998.1843. </LI><LI id="Fellouse2004">Fellouse, Frederic A.; Wiesmann, Christian; Sidhu, Sachdev S. (2004): Synthetic antibodies from a four-amino-acid code: a dominant role for tyrosine in antigen recognition. In: <I> Proceedings of the National Academy of Sciences of the United States of America</I> 101 (34), S. 12467–12472. DOI: 10.1073/pnas.0401786101.</LI><LI id="Fellouse2007">Fellouse, Frederic A.; Esaki, Kaori; Birtalan, Sara; Raptis, Demetrios; Cancasci, Vincenzo J.; Koide, Akiko et al. (2007): High-throughput generation of synthetic antibodies from highly functional minimalist phage-displayed libraries. In: Journal of Molecular Biology 373 (4), S. 924–940. DOI: 10.1016/j.jmb.2007.08.005.</LI><LI id="Koide">Koide, Shohei; Sidhu, Sachdev S. (2009): The importance of being tyrosine: lessons in molecular recognition from minimalist synthetic binding proteins. In: <I>ACS chemical biology</I> 4 (5), S. 325–334. DOI: 10.1021/cb800314v.</LI><LI id="Krebber">Krebber, Anke; Bornhauser, Susanne; Burmester, Jörg; Honegger, Annemarie; Willuda, Jörg; Bosshard, Hans Rudolf; Plückthun, Andreas (1997): Reliable cloning of functional antibody variable domains from hybridomas and spleen cell repertoires employing a reengineered phage display system. In: <I>Journal of Immunological Methods</I> 201 (1), S. 35–55. DOI: 10.1016/S0022-1759(96)00208-6.</LI><LI id="Mian">Mian, I.Saira; Bradwell, Arthur R.; Olson, Arthur J. (1991): Structure, function and properties of antibody binding sites. In: <I>Journal of Molecular Biology</I> 217 (1), S. 133–151. DOI: 10.1016/0022-2836(91)90617-F.</LI><LI id="Osoegawa">Osoegawa, K.; Jong, P. J. de; Frengen, E.; Ioannou, P. A. (2001): Construction of bacterial artificial chromosome (BAC/PAC) libraries. In: <I>Current protocols in molecular biology</I> Chapter 5, Unit 5.9. DOI: 10.1002/0471142727.mb0509s55.</LI><LI id="Zemlin">Zemlin, Michael; Klinger, Martin; Link, Jason; Zemlin, Cosima; Bauer, Karl; Engler, Jeffrey A. et al. (2003): Expressed Murine and Human CDR-H3 Intervals of Equal Length Exhibit Distinct Repertoires that Differ in their Amino Acid Composition and Predicted Range of Structures. In: <I>Journal of Molecular Biology</I> 334 (4), S. 733–749. DOI: 10.1016/j.jmb.2003.10.007.</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>