<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Results_Mutation_Sequencing skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Results/Mutation/Sequencing</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container main"><DIV id="header"><NAV id="navigationbar" class="navbar navbar-default custom_navbar_style navbar-fixed-top"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="#">NavBar</A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-center"><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description"> PROJECT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description">Description</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds">Scaffolds</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design">Design and Construction</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/CreateYours">Create your own Library</A></LI></UL><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/EpPolI">Error prone PolymeraseI</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/Global">Genome wide mutator</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">Bacterial Two-Hybrid System</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/SplitBetaLactamaseHitchHiker">Alternative Selection System</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_I">Model I</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_II">Model II</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results"> RESULTS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Phage">Phagemid Display</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">Reversion</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/Cloning">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/ExpressionControl">Expression Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/InteractionControl">Interaction Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/BindingControl">Binding Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/KnockOutKnochIn"><I>in vivo</I> Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Modeling">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Fermentation">Fermentation</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Best Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Best Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Best Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts/Improve">Improve a part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Demonstrate your Work</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Medal_Fulfillment">Medal Fulfillment</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold"> HUMAN PRACTICES </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Dictionary">Dictionary</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks">Educational Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Crowdfunding">Crowdfunding and Press</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Achievements</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations"> COLLABORATIONS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Duesseldorf">Düsseldorf</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Lethbridge">Lethbridge</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Freiburg">Freiburg</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments"> NOTEBOOK </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook">Lab Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">Primers</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols">Lab Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Media">Media and Buffers</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team"> TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team">Team Profile</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Members">Members</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Contact">Contact</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement"> ACKNOWLEDGEMENT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Sponsors_And_Partners">Sponsors and Partners</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></DIV></DIV></NAV></DIV></DIV><DIV class="container main"><DIV class="jumbotron" style="background-image: url(https://static.igem.org/mediawiki/2016/0/02/Bielefeld_CeBiTec_2016_10_14_X_projectheader.png)"><DIV class="jumbotron-text"><H1 style="margin-bottom: 0px; text-align:left">Investigate Mutation Rates by High-throughput Sequencing</H1></DIV></DIV><DIV class="container text_header" id="overview"><H3>Determine Mutation by High-throughput Sequencing</H3></DIV><DIV class="container text">
			After showing mutation by reversion, we wanted to analyze our mutation parts by next-generation sequencing (NGS) in more detail. NGS presents a method to measure mutation rates and the mutation spectrum with extreme precision. Examples for this application are the determination of <I>E. coli</I> (Lee et al. 2012), Yeast (Zhu et al. 2014) and Human (Xue et al. 2009) mutational rates and spectrum.

We used Illumina sequencing (MiSeq) to correctly asses the mutation frequency and spectrum of the error prone polymerase I and our genome wide mutator. Furthermore, we used the obtained data to quantify plasmid copy numbers and ratios between plasmids with the same origin of replication inside one cell. Different plasmids were analyzed by re-sequencing and <I>de novo</I> assembly, respectively.
<UL><LI><A href="#eppol">Error prone polymerase I</A></LI><LI><A href="#m6">genome wide mutator pSB1C3:BBa_K2082116 or BBa_K2082117</A></LI><LI><A href="#assembly"><I>de novo</I> assembly of the reporter plasmid pLA230</A></LI></UL>
Each sample was prepared as a separate library using specific tags in the adapters for multiplexing. At first, mutations within the initial plasmid were identified. Moreover, we distinguished between sequencing errors and true mutations by including a control. 
</DIV><DIV class="container text_header" id="eppol"><H3>Library generation, sequencing and initial computation</H3></DIV><DIV class="container text">
			We used plasmids from one of our reversion <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">experiments</A> as template for this sequencing experiment. Out of each sample a sequencing library was constructed prepared using the <A href="http://www.illumina.com/products/by-type/sequencing-kits/library-prep-kits/nextera-dna.html">Illumina Nextera DNA Library Preparation Kit</A>. DNA fragments with a length between 500-1,000 bp were isolated. Subtracting the length of both terminal adapter, the DNA fragments contain 360 to 860 bp fragments derived from our plasmids. 
			<FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 1: Preperated MiSeq library analyzed with the Agilent Bioanalyzer 2100.</B> The results show a fragment length between 500 and 1,000 bp. 
				</FIGCAPTION></FIGURE>
			The sequencing revealed 2x300 nt (paired-end) reads. Reads were analyzed with <A href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/">FastQC</A> showing ~10,000 reads per library with an average read length of 300 nt (Figure). 
The average quality distribution of the reads (Figure) was continuously very high  until base 250. 
			<FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 2: Figure: Distribution of read length of the library and quality distribution of the reads pre trimming analyzed with FastQC.</B></FIGCAPTION></FIGURE>
			To obtain only high quality reads we trimmed the reads with <A href="http://www.usadellab.org/cms/?page=trimmomatic">Trimmomatic</A> (Bolger et al. 2014) by cutting off after four consecutive bases with a quality score lower than 25. The trimmed library was analyzed as well, consisting of X reads with an average length of 220 bp (Figure) and end-to-end quality higher than 25 (Figure). A quality score over 25 means an error probability of 0.0032 (Illumina 2014). The quality score of the whole trimmed library is around 38, resulting in an error probability of 0.00016. Assuming  a library size of 2.5×10<SUP>6</SUP> bp this leads to about 400 sequencing errors.
						<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 3: Distribution of read length of the library and quality distribution of the reads after trimming analyzed with FastQC.</B></FIGCAPTION></FIGURE></CENTER>
			The reads were mapped with <A href="https://www.qiagenbioinformatics.com/products/clc-genomics-workbench/">CLC genomics workbench 9.5</A> to our plasmid references. The coverage and base distribution per position was calculated. However, it was not possible to assign the reads originating from the origin of replication initiation from the two different plasmids in some samples correctly, because both plasmid contain a very similar sequence. This region was excluded for further analysis involving the read coverage depth. As foundation for further analysis TSV files containing the information about the mapping at each position of the reference were exported from CLC Genomics Workbench. TSV files are TAB-delimited text files containing the number of supporting reads for each base at each position (Figure 4).
				<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 4: Structure of generated TSV files.</B></FIGCAPTION></FIGURE></CENTER></DIV><DIV class="container text_header" id=""><H3>Determination of mutation rate through NGS sequencing results</H3></DIV><DIV class="container text">
We used plasmid combinations according to our <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">reversion experiments</A> as reference for the mapping. These experiments consist of one plasmid carrying the mutagenesis genes (either <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/EpPolI">EP-PolI</A> one of the <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/Global">genome wide mutator systems</A>) and one reporter plasmid with a stop beta-lactamase. Therefore, both plasmids were present in our sequencing library. As a control for measuring sequencing errors our reporter plasmid was sequenced in parallel in a separate library.
			</DIV><DIV class="container text_header" id="eppol"><H3>Error prone polymerase I</H3></DIV><DIV class="container text">
			The plasmids for the sequencing libraries were made by transforming 10 ng of the stop beta-lactamase plasmid <A href="https://www.addgene.org/11724/">pLA230</A> in JS200 cells, carrying the error prone polymerase I. Afterwards, the cells were grown for 24 h at 37 °C in prewarmed LB-broth media supplemented with chloramphenicol and kanamycin. Finally, the cells were plated for the <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">reversion assays</A> or the plasmids were isolated for sequencing. Alongside the experiment was performed using wild type polymerase I instead of the error-prone polymerase I.
<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 5: Coverage Plots for pLA230 and pHSG-EPPolI.</B> Coverage per base for pLA230 and pHSG-EPPolI were depicted. Important plasmid features are shown as arrows</FIGCAPTION></FIGURE></CENTER><B>Table 1: Average coverage of the single plasmids in the plasmid mixes.</B><CENTER><TABLE><TBODY><TR><TD>pLA230</TD><TD>pHSG-PolI</TD><TD>ratio (pLA230/pHSG-PolI)</TD></TR><TR><TD>WT</TD><TD>575.26</TD><TD>98.98</TD><TD>5.8</TD></TR><TR><TD>EP</TD><TD>675.33</TD><TD>141.13</TD><TD>4.8</TD></TR></TBODY></TABLE></CENTER>
The average coverage-ratio between pLA230 and pHSG-PolI was 4.8 for the error-prone polymerase and 5.8 for the wild-type polymerase. This confirms values from previously published works that used the error-prone polymerase inside the polymerase I temperature-sensitive <I>E. coli</I> JS200 and observed a decrease in copy number of ColE1 plasmids (Camps et al. 2003). However,  the ratio in our results is lower than the published value (Camps, 2003). Furthermore, it is interesting that the published copy number of ColE1 plasmids ranging from 100 to 300 and copy numbers of pSC101-based plasmids, like pHSG, are only between 6 and 8 (Kornberg und Baker 1992; Cabello et al. 1976; Hasunuma und Sekiguchi 1977). Based on this data, we expected  ratios between 12.5 and 50. The observed difference between our sequencing based copy number estimation and the previously published values, could be explained by the application of different techniques. Assuming the copy number of the controlled regulation of pSC101 plasmid is correct (6-8) this results in copy numbers for the ColE1 plasmid of 34-46 for the wild type and 29-38 for the error prone polymerase I. In our opinion the determination of plasmid ratios via NGS is more reliable than older methods. Therefore, we will use our determined value of 40 copies with a normal polymerase and 33 copies with the error prone polymerase I as values for our calculations in the <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">reversion assay</A> article. 

To assess the mutations incorporated by the error prone polymerase I we wrote a custom python script which 
<UL><LI>Imports a TSV file</LI><LI>Counts coverage for all non-reference bases at every positions as well as the total coverage</LI><LI>Normalizes the counted mutations by the total coverage of the analyzed plasmid</LI><LI>Constructs mutation and coverage plots for all sequences of interest</LI></UL>
This script calculates the mutation frequency as mutation frequency per sequenced base. The calculation was done for all mappings to our reporter plasmid. The results provide an overview about the basal mutation rate in combination with the sequencing error and the effect of the error prone polymerase I.
<FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 6: Mutation frequency determined by high-throughput sequencing.</B> The mutation frequency was determined as observed mutation per sequenced bases. The control is our reporter plasmid grown without any kind of mutator. Therefore, this sample was used to control for sequencing errors and other background effects. The higher numbers of differences to the reference in other samples is supposed to represent true mutations. Analyzed samples are our reporter plasmid grown in <I>E. coli</I> JS200 with plasmid expressed wild type polymerase I or error prone polymerase I, respectively.
				</FIGCAPTION></FIGURE>
			The calculated mutation rate for the control is 7.55×10<SUP>-4</SUP> mutations per base pair, which is higher than the expected sequencing error of 1.6×10<SUP>-4</SUP> (Illumina 2014). The mutation rate of the wild type polymerase I is 7.3×10<SUP>-4</SUP> and therefore very similar to the control. Use of EP Pol I increases the mutation rate to 10.3×10<SUP>-4</SUP> mutations per base pair. Under the assumption that the difference between the wild type polymerase I sample and the control (0.25×10<SUP>-4</SUP>) is the measurement inaccuracy, the error prone sample lies over 10 times this inaccuracy value above the control.

However, it is also possible, that the overexpression of proteins participating in DNA repair like wild type polymerase I leads to a decreased mutation rate.

Furthermore, we used another script than looks for mutations by scanning the generated TSV files for bases, which deviate from the consensus base. All this mutations are summed up, deviating which kind of mutation (reference base &amp;arr; mutated base) is present. This results in an extensive overview of the mutations introduced by the error-prone polymerase but also the sequencing errors. 
<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 7: Observed mutation distribution of the different sequencing libraries. </B> 
					All mutations (n) inside the sequencing data were classified as  the appropriate mutation type. The squares show the percentage of mutations from the reference base to a mutated base. 
				</FIGCAPTION></FIGURE></CENTER>
			We observed a significant change in mutational spectrum between the error prone polymerase I and the control (p = 1.43×10<SUP>-6</SUP>, χ<SUP>2</SUP> = 44.89) but no difference between the wild type polymerase I and the control (χ<SUP>2</SUP> = 3.86). Especially the strong increase of the transitions G→A and C→T is notable. But also the general portion of transition of the complete mutations increases from 28.9 % and 32.7 % in the control respectively in the wild type polymerase I to 43.4 % in the error prone polymerase I strain.
		<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 8: Proportion of transitions and transversions observed from all mutations on our reporter plasmid causedby the error prone, wild type polymerase in <I>E. coli</I> JS200 or our reporter plasmid in <I>E. coli</I> Top10, respectively.</B></FIGCAPTION></FIGURE></CENTER>
			The mutational spectrum of the error prone polymerase I was investigated by Troll and coworkers (Troll et al. 2014) using a total of 123 sanger sequenced clones.
				<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 9: Mutagenic spectrum of the error prone polymerase I as determined by Troll and coworkers (Troll et al. 2014).</B></FIGCAPTION></FIGURE></CENTER>
			In comparison to the published results our experiments revealed a different picture of the mutagenic capabilities of the error prone polymerase I. We found a prevalence for transitions, while Troll found 83 % transversions.

The difference between both results can probably be explained by the different methods applied. This assumption is supported by vastly different mutation spectra previously observed by comparing different methods (Badran and Liu, 2015). When investigation the <I>rpoB</I> gene of forward mutants in a rifampicin assay with sanger sequencing and a small sample size (n &lt; 215) the mutation spectrum differs very strongly from the one determined with NGS and a much larger sample size (n &lt; 112580). Generally, NGS is a more reliable method due to the high  amount of observed (mutation)events. Furthermore, NGS is independent from different selection strategy like selection of the occurrence of an antibiotic resistance. This preselection before sequencing is needed to filter out the very large amount (&gt;99 %) of unmutated sequences. However, this selection  introduces a mutational bias. Therefore, NGS with a high sample size and the possibility to sequence a library, which was created by mutation without any kind of selection pressure, is the optimal way to determine mutation spectra. Nevertheless, there are some biases in NGS data generation as well. The GC content of the fragment was shown to effect the mapability of a ultra short NGS read (Dohm et al., 2008). However, most of these effects do not impact the results significantly anymore due to an increased read length and improved sequencing chemistry.
</DIV><DIV class="container text_header" id="m6"><H3>Genome wide mutator</H3></DIV><DIV class="container text">
Our genome wide mutator system was analyzed via NGS as well. We used the M6 mutator and the <I>dnaQ926</I> mutator gene alone. The plasmids were used from a corresponding reversion experiment. 

The experimental setup was as following:

The M6 mutator BBa_K2082117 or the <I>dnaQ926</I> device BBa_K2082116 were cotransformed with the reporter plasmid pLA230 into <I>E. coli</I> Top10 and plated on LB agar plates with chloramphenicol, kanamycin and glucose. 

From a single colony LB media with chloramphenicol and kanamycin was inoculated, grown to mid-log phase, induced or repressed with arabinose or glucose and plated after 24 h. Plasmids for sequencing were isolated 24 h after induction/repression, yielding four libraries for NGS.

The mutation rate and the mutation spectrum was determined as describe above.
				<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 10: Mutation rate after induction/repression of <A href="http://parts.igem.org/Part:BBa_K2082117">BBa_K2082117</A>.</B></FIGCAPTION></FIGURE></CENTER>
			The mutation rate increases from the control to the repressed M6 plasmid sample and further to the induced M6 plasmid sample. Using the difference between control and the wild type polymerase sample to measure the inaccuracy of the method. The repressed plasmid sample is 1.8 times and the induced plasmid sample is 7.1 times the measurement inaccuracy above the control. This means that <A href="http://parts.igem.org/Part:BBa_K2082117">BBa_K2082117</A> mutates more strongly than the background, thus is a mutator. Furthermore, the induction of <A href="http://parts.igem.org/Part:BBa_K2082117">BBa_K2082117</A> works as seen in an increase in mutation rate in comparison to the uninduced sample. Finally, the used promoter is active even in the not-induced state as seen by the increase in mutation rate between the control and the repressed sample.
Based on these results we modified our reversion assays in that we kept the repressed control under glucose repression at all times. In contrast: in this experiment we cultivated at first in LB media and added glucose in the repressed sample simultaneous to the addition of arabinose in the induced sample.

Additionally, we wanted to determine the mutagenic spectra of our mutators. This was done as described above.
				<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 11: Mutational spectrum of the <A href="http://parts.igem.org/Part:BBa_K2082117">M6</A> and <A href="http://parts.igem.org/Part:BBa_K2082116">dnaQ926</A> mutators.</B></FIGCAPTION></FIGURE></CENTER>
			Based on the χ<SUP>2</SUP> test no significant difference between the spectrum of the control and the induced mutators could be observed. (χ<SUP>2</SUP><SUB><I>dnaQ926</I></SUB> = 0.85, χ<SUP>2</SUP><SUB>M6</SUB> = 3.78) 

Nevertheless, the mutation spectrum is comparable to the published mutation spectrum of the M6 plasmid used by Badran and coworkers (Badran and Liu, 2015).
				<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 12: Mutational spectrum of our M6 system <A href="http://parts.igem.org/Part:BBa_K2082117">BBa_K2082117</A> and the one used by Badran and coworkers (Badran and Liu, 2015).</B></FIGCAPTION></FIGURE></CENTER></DIV><DIV class="container text_header" id="assembly"><H3><I>de novo</I> assembly of pLA230</H3></DIV><DIV class="container text">
pLA230 was used as control. To make sure our used sequence was correct, we assembled pLA230 <I>de novo</I> from the control dataset and compared it with the published <A href="https://www.addgene.org/11724/">plasmid sequence</A>. We found one point mutation at position and detected an insertion behind the beta lactamase coding sequence.
<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 13: Alignment of the <I>de novo</I> assembled pLA230 sequence and the pLA230 sequence from <A href="https://www.addgene.org/11724/">Addgene</A>.</B> The alignment shows a point mutation and the deletion inside the pLA230 sequence. 
					</FIGCAPTION></FIGURE></CENTER></DIV><DIV class="container text_header"><H3>Summary</H3></DIV><DIV class="container text">
			We analyzed the error prone polymerase I and our genome wide mutators with next generation sequencing.
			
			We showed that the error prone polymerase I has a visible increased mutation rate (10x over background). Furthermore we determined the mutation spectrum of the error prone polymerase I and found a prevalence of transitions. 
			
			For our genome wide mutator system we found an increased mutation rate when using the M6 BioBrick BBa_K2082117 up to 7x over background. The uninduced sample also showed visible increased mutation (~2x over background), confirming our 
			assumption of considerable leakiness of our promoter, when not consequently repressed with glucose.
			We also determined the mutation spectrum of the genome wide mutator and it shows the same mutageic spectra as the MP6 plasmid used by Badran and coworkers(Badran and Liu 2015).
		</DIV><DIV class="container text_header"><H3>References</H3></DIV><DIV class="container text"><UL><LI>Badran, Ahmed H.; Liu, David R. (2015): Development of potent in vivo mutagenesis plasmids with broad mutational spectra. In: Nature communications 6, S. 8425. DOI: 10.1038/ncomms9425.</LI><LI>Bolger, Anthony M.; Lohse, Marc; Usadel, Bjoern (2014): Trimmomatic: a flexible trimmer for Illumina sequence data. In: Bioinformatics (Oxford, England) 30 (15), S. 2114–2120. DOI: 10.1093/bioinformatics/btu170.</LI><LI>Cabello, F.; Timmis, K.; Cohen, S. N. (1976): Replication control in a composite plasmid constructed by in vitro linkage of two distinct replicons. In: Nature 259 (5541), S. 285–290.</LI><LI>Camps, Manel; Naukkarinen, Jussi; Johnson, Ben P.; Loeb, Lawrence A. (2003): Targeted gene evolution in Escherichia coli using a highly error-prone DNA polymerase I. In: Proceedings of the National Academy of Sciences of the United States of America 100 (17), S. 9727–9732. DOI: 10.1073/pnas.1333928100.</LI><LI>Hasunuma, K.; Sekiguchi, M. (1977): Replication of plasmid pSC101 in Escherichia coli K12: requirement for dnaA function. In: Molecular &amp; general genetics : MGG 154 (3), S. 225–230.</LI><LI>Illumina (2014): Understanding Illumina Quality Scores. Quality scores are an efficient way to communicate small error probabilities. Online verfügbar unter http://www.illumina.com/content/dam/illumina-marketing/documents/products/technotes/technote_understanding_quality_scores.pdf.</LI><LI>Kornberg, A.; Baker, T. A. (1992): DNA replication. In: Freeman, New York.</LI><LI>Lee, Heewook; Popodi, Ellen; Tang, Haixu; Foster, Patricia L. (2012): Rate and molecular spectrum of spontaneous mutations in the bacterium Escherichia coli as determined by whole-genome sequencing. In: Proceedings of the National Academy of Sciences of the United States of America 109 (41), S. E2774-83. DOI: 10.1073/pnas.1210309109.</LI><LI>Troll, Christopher; Yoder, Jordan; Alexander, David; Hernandez, Jaime; Loh, Yueling; Camps, Manel (2014): The mutagenic footprint of low-fidelity Pol I ColE1 plasmid replication in E. coli reveals an extensive interplay between Pol I and Pol III. In: Current genetics 60 (3), S. 123–134. DOI: 10.1007/s00294-013-0415-9.</LI><LI>Xue, Yali; Wang, Qiuju; Long, Quan; Ng, Bee Ling; Swerdlow, Harold; Burton, John et al. (2009): Human Y chromosome base-substitution mutation rate measured by direct sequencing in a deep-rooting pedigree. In: Current biology : CB 19 (17), S. 1453–1457. DOI: 10.1016/j.cub.2009.07.032.</LI><LI>Zhu, Yuan O.; Siegal, Mark L.; Hall, David W.; Petrov, Dmitri A. (2014): Precise estimates of mutation rate and spectrum in yeast. In: Proceedings of the National Academy of Sciences of the United States of America 111 (22), S. E2310-8. DOI: 10.1073/pnas.1323011111.</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>