Team:Bielefeld-CeBiTec/Project/Modeling/Model I
Toggle navigation
NavBar
PROJECT
Description
Library
Overview
Scaffolds
Design and Construction
Create your own Library
Mutation
Overview
Error prone Polymerase I
Genome wide mutator
Selection
Overview
Bacterial Two-Hybrid System
Alternative Selection System
Modeling
Overview
Model I
Model II
RESULTS
Overview
Library
Overview
Assembly
Phagemid Display
Sequencing
Mutation
Overview
Assembly
Reversion
Sequencing
Selection
Overview
Assembly
Expression Control
Interaction Control
Binding Control
in vivo Selection
Modeling
Fermentation
Parts
Best Basic Part
Best Composite Part
Best Part Collection
Improve a part
Proof of Concept
Demonstrate your Work
Medal Fulfillment
HUMAN PRACTICES
Overview
Engagement
Dictionary
Educational Outreach
Crowdfunding and Press
Entrepreneurship
Integrated Practices
Achievements
COLLABORATIONS
Overview
Düsseldorf
Lethbridge
Freiburg
NOTEBOOK
Overview
Lab Notebook
Library
Mutation
Selection
Primers
Lab Protocols
Media and Buffers
TEAM
Team Profile
Members
Bielefeld University
Contact
ACKNOWLEDGEMENT
Overview
Sponsors and Partners
Attributions
Model I
Selection system
Model I describes the in vivo bacterial two hybrid selection system :
Figure 1: Selection pathway.
Plasmid 1 and plasmid 2 each produce one protein of the bacterial two hybrid system, respectively. When both proteins come together and bind on the DNA the reporter gene is transcribed, here beta-lactamase. The beta-lactamase then diffuses from the cytoplasm into the periplasm, where it encounters the antibiotic ampicillin and deactivates it. For detailed description see bacterial two hybrid system .
The selection pathway can be illustrated by the following reaction sequence:
Figure 2: Reaction pathway of the selection system.
The 0 stands for degeneration of proteins or complexes and the + for the interaction of two proteins or complexes.
The reaction sequence starts with the expression of the Evobody-RpoZ translational fusion construct [(a)] on the one hand and the target protein-DNA binding domain translational fusion construct [(b)] on the other hand.   Both resulting fusion proteins are degraded over time [(c)].   In the figure the degeneration is marked with a 0.   Furthermore, together with the DNA binding site for the DNA binding domain (DNABD) both fusion proteins form the bacterial two hybrid complex [(d)].   When the complex is built the RNA polymerase gets recruited and transcribes the beta-lactamase.   Just like the fusion proteins the beta-lactamase will be degraded after a while [(e)].   When the beta-lactamase and ampicillin meet in the periplasm the ampicillin will be deactivated and the cell survives.
Based on the reaction sequence the chemical reaction equations can be written down:
(I) kexprP1 stands for the expression rate of plasmid 1, which holds the Evobody-RpoZ sequence [s-1].
kdegEvo stands for the degeneration rate of the Evobody-RpoZ protein [s-1].
(II) kexprP2 stands for the expression rate of plasmid 2, which holds the target-DNA binding domain (DNABD) protein [s-1].
kdegTarget stands for the degeneration of the target-DNABD protein [s-1].
(III) kass stands for the association rate of the bacterial two hybrid complex [(s*M)-1]. kdiss stands for the disassociation rate of the bacterial two hybrid complex [s-1].
(IV) kexprbla stands for the expression rate of beta-lactamase (bla) [s-1].
(V) ktranspbla stands for the transport rate of beta-lactamase from cytoplasm to periplasm [s-1].
(VI) kdegbla stands for the degeneration rate of the beta-lactamase [s-1].
(VII) ktranspamp stands for the transport rate of ampicillin from medium to cytoplasm and back [s-1].
(VIII) kdiff stands for the diffusion rate of beta-lactamase and ampicillin [(s*M)-1].
Differential equations
Next the language of chemistry has to be translated into the language of mathematics. A system of ordinary differential equations is generated. As this system has to be solved numerically, we have chosen a notation the software package MATLAB® understands:
standexpr stands for standard expression [M].
[Evobody] or Evobody stands for Evobody-RpoZ protein [M].
[Target] or Target stands for target-DNABD protein [M].
[DNA] stands for DNA [M].
[ampPeri] stands for ampicillin in periplasm [M].
[ampMedium] stands for ampicillin in medium [M].
[blaCyt] stands for beta-lactamase in cytoplasm [M].
[blaPeri] stands for beta-lactamase in periplasm [M].
[Evobody-Target] stands for Evobody-Target protein complex [M].
[Target-DNA] stands for Target-DNA complex [M].
[Evobody-Target-DNA] stands for Evobody-Target-DNA complex [M].
kdissEvoTarget stands for disassociation rate of the Evobody-Target complex [s-1].
kassEvoTarget stand for association rate of the Evobody-Target complex [(s*M)-1].
kdissTargetDNA stand for disassociation rate of the Evobody-Target complex [s-1].
kassTargetDNA stand for association rate of the Evobody-Target complex [(s*M)-1].
kdegTarget stands for degeneration rate of the target protein [s-1].
kdegEvo stands for degeneration rate of the Evobodies [s-1].
kexprblaETD stands for the expression rate of beta-lactamase with the bacterial two hybrid complex [s-1].
Diffusion
The deactivation reaction of ampicillin through beta-lactamase is a diffusion-controlled reaction. An equation that considers the rate at which the reactants diffuse is needed (Atkins and De Paula 2010):
Figure 3: Diffusion of two proteins without consideration of an active centrum.
This equation describes the general diffusion of two molecules (figure 3).  But in our case the enzyme beta-lactamase only reacts with ampicillin, if its active centrum is in close proximity to ampicillin (figure 4 B).  When beta-lactamase and ampicillin meet correctly, ampicillin is deactivated as shown in figure 4, section B to D. Section E indicates a wrong encounter of beta-lactamase and ampicillin. That's why we decided to modify the equation according to (Bisswanger 2008):
Figure 4: Diffusion of two proteins with consideration of an active centrum.
Rmin is defined as the distance in which the two reactant molecules, here ampicillin and beta-lactamase, react. This is the case, when they are in very close proximity. D is the sum of the diffusion coefficients of the two reactants in the solution. In our case the solution is the periplasm. The term diffusion describes the movement of a substance in a solution; the bigger the diffusion coefficient, the larger the diffusion velocity.  The angle α restricts the reaction area of the enzyme to its active center (see opening in figure 4). So a reaction between beta-lactamase and ampicillin only takes place, when ampicillin meets the active center of beta-lactamase.
Finally, the equation for the diffusion rate kdiff must be multiplied with the Avogadro constant NA to gain the unit [(s*M)-1].
Rmin was computated under the assumption that beta-lactamase and ampicillin react when they are in close vicinity and that the molecules are nearly spherical. Therefore, the minimal radius of a sphere that could contain the given mass of protein was estimated using the formula from (Erickson 2009):
for V in nm3, M in Dalton and Rmin in nanometer.
For beta-lactamase we obtained 2.0277 nm and for ampicillin 0.4649 nm.
To compute the diffusion coefficients we used the program WinHydroPro v1.00 PUB  [Ortega, 2011]. We received 2.852*10-9 for beta-lactamase and 1.237*10-8 for ampicillin. The angle α was computed with the open-source program PyMOL (PyMOl 2013). α was determined to be 25°.  In the end we computed the reproduction rate in dependence of the ampicillin concentration in the periplasm, adapting the formula of Potsdam's:
where n is the ampicillin concentration in periplasm. The time point 20 minutes was chosen, because the division rate of an E. coli cell is around 20 minutes.
Experimental validation
A theoretically implemented model must always be validated with experimentally measured data. Therefore, we designed and conducted an amount of appropriate experiments. Firstly, we performed a beta-lactamase PCA colorimetric assay, and, secondly, a cultivation experiment. The cultivation experiment was used to measure the beta-lactamase concentration with the Nitrocefin assay over time. The experiments were essential to adapt model I.
Beta-lactamase PCA colorimetric assay
To quantify the amount of active beta-lactamase in the periplasm of an E. coli bacterium and thus validate our mathematical program in this aspect, we performed an [osmotic shock] and a beta-lactamase PCA colorimetric assay (André Galarneau et. al 2002). The osmotic shock leads to the disruption of the periplasm and spreading of membrane proteins. To yield the maximum amount of active beta-lactamase an osmotic shock is used that does not use detergents that are harmful for the activity of beta-lactamase (Jalalirad 2013). With the beta-lactamase PCA colorimetric assay we determined the amount of beta-lactamase through the reduction of Nitrocefin. When Nitrocefin is split by beta-lactamase it changes its color from yellow to red like in figure 5. To measure how the beta-lactamase production changes over time we cultivated E. coli with a beta-lactamase plasmid for 6 hours and took probes every hour. Afterwards we did a lysis and a Nitrocefin assay. The results can be seen here:
Figure 5: Nitrocefin assay.
From left to right: water control and time points t0 to t6 of a E. coli cultivation.  During a cultivation period samples were taken from the culture every hour (t0 to t6). Cells in this sample were lysed, and measurements with a Nitrocefin assay were conducted.
Figure 6: Growth curve of a culture, which carries a beta-lactamase plasmid.
In figure 5 it can be seen that the beta-lactamase concentration rises with proceeding cultivation. Also we measured the beta-lactamase concentration of the whole culture at every time point and computed with the use of the OD (figure 6) the beta-lactamase concentration for one cell. 1 OD is converted to 8 * 108 cells. One cell comprises around 1.27848 * 10-6 M beta-lactamase in the periplasm. We adapted our system accordingly.
Model II
References
Atkins, Peter W.; Paula, Julio de (2010): Physical chemistry. 9. ed. Oxford u.a.: Oxford Univ. Press.
Bisswanger, Hans (2008): Enzyme kinetics. Principles and methods. 2., rev. and updated ed. Weinheim: Wiley-VCH. Online accessible at http://site.ebrary.com/lib/alltitles/docDetail.action?docID=10301967.
Erickson, Harold P. (2009): Size and shape of protein molecules at the nanometer level determined by sedimentation, gel filtration, and electron microscopy. In: Biological procedures online 11, S. 32–51. DOI: 10.1007/s12575-009-9008-x.
Galarneau, André; Primeau, Martin; Trudeau, Louis-Eric; Michnick, Stephen W. (2002): |[beta]|-Lactamase protein fragment complementation assays as in vivo and in vitro sensors of protein|[ndash]|protein interactions. In: Nature Biotechnology 20 (6), S. 619–622. DOI: 10.1038/nbt0602-619.
Jalalirad, Reza (2013): Selective and efficient extraction of recombinant proteins from the periplasm of Escherichia coli using low concentrations of chemicals. In: Journal of industrial microbiology & biotechnology 40 (10), S. 1117–1129. DOI: 10.1007/s10295-013-1307-1.
