<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Entrepreneurship skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Entrepreneurship</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container main"><DIV id="header"><NAV id="navigationbar" class="navbar navbar-default custom_navbar_style navbar-fixed-top"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="#">NavBar</A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-center"><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description"> PROJECT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description">Description</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds">Scaffolds</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design">Design and Construction</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/CreateYours">Create your own Library</A></LI></UL><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/EpPolI">Error prone PolymeraseI</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/Global">Genome wide mutator</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">Bacterial Two-Hybrid System</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/SplitBetaLactamaseHitchHiker">Alternative Selection System</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_I">Model I</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_II">Model II</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results"> RESULTS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Phage">Phagemid Display</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">Reversion</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/Cloning">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/ExpressionControl">Expression Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/InteractionControl">Interaction Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/BindingControl">Binding Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/KnockOutKnochIn"><I>in vivo</I> Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Modeling">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Fermentation">Fermentation</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Best Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Best Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Best Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts/Improve">Improve a part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Demonstrate your Work</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Medal_Fulfillment">Medal Fulfillment</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold"> HUMAN PRACTICES </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Dictionary">Dictionary</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks">Educational Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Crowdfunding">Crowdfunding and Press</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Achievements</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations"> COLLABORATIONS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Duesseldorf">DÃ¼sseldorf</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Lethbridge">Lethbridge</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Freiburg">Freiburg</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments"> NOTEBOOK </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook">Lab Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">Primers</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols">Lab Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Media">Media and Buffers</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team"> TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team">Team Profile</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Members">Members</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Contact">Contact</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement"> ACKNOWLEDGEMENT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Sponsors_And_Partners">Sponsors and Partners</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></DIV></DIV></NAV></DIV></DIV><DIV class="container main"><DIV class="jumbotron" style="background-image: url(https://static.igem.org/mediawiki/2016/b/ba/Bielefeld_CeBiTec_2016_10_19_X_hp.png)"><DIV class="jumbotron-text"><H1 style="margin-bottom: 0px; text-align:left">Entrepreneurship</H1><H2 style="color:#ffffff; text-align:left">A brief history of the future</H2></DIV></DIV><CENTER><A href="#ProcessDevelopment"><H3>Process Development</H3></A><A href="#UpscalingProcess"><H3>Upscaling Process</H3></A><A href="#BusinessPlan"><H3>Business Plan</H3></A></CENTER><DIV class="container text_header"><H3>Development Process</H3></DIV><DIV class="container text">
The developmentof a flowchart of our process and procedures supports the initial overview of our project. Furthermore, this overview is a major requirement to successful upscale our idea and bring it into actual use for researchers and diagnosticians all over the world. Therefore, we talked to several experts (<A href="âhttp://ekvv.uni-bielefeld.de/pers_publ/publ/PersonDetail.jsp?personId=11015â">Prof. Dr. Karl Friehs</A> (fermentation technology), <A href="âhttp://ekvv.uni-bielefeld.de/pers_publ/publ/PersonDetail.jsp?personId=163841â">Prof Dr. Thomas Noll</A> (cell culture technology), <A href="âhttp://ekvv.uni-bielefeld.de/pers_publ/publ/PersonDetail.jsp?personId=11021â">Dr. Joe Max Risse</A> (fermentation technology) and <A href="âhttp://cms.plasmidfactory.com/ueber-plasmidfactory/ihr-ansprechpartner/â">Dr. Martin Schleef</A> (Plasmid Factory) in large scale production within biotechnology. Results from these discussions were transferred to our procedures to reach a bigger scale. One major issue was identified easily: the whole process would be too complex. Therefore, the two central processes of generating of the Evobody encoding sequence and the final production of the Evobody itself are presented in separate flowcharts to present our unique system of in vivo mutagenesis and selection.
Our starting point is the main procedure that gives an overview. This makes up the skeleton of our project. It describes the steps that are necessary to get an Evobody sequence and how to continue after the production of the Evobody itself. It involves the procedures that are applied when an order for a new target is issued up to the finished product that is then send out (Fig. 1).
</DIV><CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><I>Fig. 1: Flowchart of the main process</I></FIGCAPTION></FIGURE></CENTER><DIV class="container text">
Second is the generation of an Evobody sequence that shows the greatest binding properties to our target. This is achieved with our system. It involves the generation of the library, the system for mutating the sequence of the Evobody and the constant selection for the best binder.
</DIV><CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><I>Fig. 2: Flowchart for the generation of an Evobody sequence</I></FIGCAPTION></FIGURE></CENTER><DIV class="container text">
Last but not least is the actual process of producing our Evobody, the extraction and cleanup of the Evobody before it can be shipped. It starts with the amplification of the sequence for the best binder that is the result of the second process and ends with the purified and stabilized binding protein. Our Evobody.
</DIV><CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><I>Fig. 3: Flowchart for the production of an Evobody</I></FIGCAPTION></FIGURE></CENTER><DIV class="container text_header"><H3>Upscaling Process</H3></DIV><DIV class="container text">
In order to transfer our project to an industrial level we had to compare the differences between the scales of processes in production and a research group, respectively. For the process of upscaling we consulted different experts from different fields: <A href="âhttp://ekvv.uni-bielefeld.de/pers_publ/publ/PersonDetail.jsp?personId=11015â">Prof. Dr. Karl Friehs</A> (fermentation technology), <A href="âhttp://ekvv.uni-bielefeld.de/pers_publ/publ/PersonDetail.jsp?personId=163841â">Prof Dr. Thomas Noll</A> (cell culture technology), <A href="âhttp://ekvv.uni-bielefeld.de/pers_publ/publ/PersonDetail.jsp?personId=11021â">Dr. Joe Max Risse</A> (fermentation technology) and <A href="âhttp://cms.plasmidfactory.com/ueber-plasmidfactory/ihr-ansprechpartner/â">Dr. Martin Schleef</A> (Plasmid Factory). Our main concerns are the approval by the government institutions and the efficient production of our Evobody protein. This takes the method of cultivation as well as the materials used into consideration.
The most in important thing is the process of creating the best binder, our Evobody, is still rather small and can be done on the level of a single lab. In case of binding protein production, a large scale fermentation process is needed for production in large quantities. We discussed two approaches: a repeated batch and a continuous culture, respectively. The repeated batch is easier and there is no dilution of e.g. antibiotics over time. In the continuous batch we would face a lower m<LOWER>y</LOWER>(the rate of growth) which means constant pressure on the cells, it is reproducible and we have constant conditions during the production process .
The suggested cultivation time is supposed to be between 10 and 12 generation times. Also adding another carbon source like glucose or glycerin (that does not need autoclavation but sterile filtration) would improve the yield and reduce the costs. Using a defined medium would just elongate the process in terms of analysis, control and preparation measurement. Considering the generation of the sequence of the perfect binder the culture should be harvested in the stationary phase, because there the concentration of plasmid DNA is the highest.

</DIV><CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><I>Graph 1. : Amount of pDNA in comparison to OD600 (cell number) during cultivation (Dr. Schleef)</I></FIGCAPTION></FIGURE></CENTER><DIV class="container text">
Therefore, when using a repeated batch letting only the last batch grow into the stationary phase would speed up the process as well as the yield.
The type of <I>E. coli</I> used in production is also crucial. A fast growing and well characterized  strain has to be used in order to speed up production as well as approval. Also the characteristics of the strain when our plasmids are transformed into the cells have to be determined to further optimize the system because characteristics may change due to that.
Another hurdle we had to consider was the approval by the government institutions. They differentiate between generics and biosimilars. Generics are a complete synthetic copy and thus the approval is shortened for its effect and side effects are already known. However, biosimialars root from a different amino acid sequence but the same target as the original. Because we are changing the sequence in search for the best binder, we belong to the second category. That means we have to go through all phases of testing. But aside from the pharmaceutical field our Evobodies can be freely used in research. In research our advantage is that it is <I>in silico</I> which means fast, flexible and a guaranteed outcome. Therefore, research labs can instantly react to new or mutated pathogens. Additionally, they can focus on how they plan to kill the pathogens instead of spending valuable time on finding ways to bind the target. Their possibilities are large because of the high stability of our Evobody various enzymes and proteins can be fused to it.
</DIV><DIV class="container text_header"><H3>Business Plan</H3></DIV><DIV class="container text"><U><DIV class="container text_header"><H4><CENTER>Executive summary:</CENTER></H4></DIV></U>
Our goal is to introduce Evobodys (custom binding proteins) to the market. We produce customer specific binding proteins for all areas of application. Our system is fast and cost efficient. Thus, a competitive product is available to everybody. Even small research labs can work with it and medical facilities can use it as a head start against new or reemerging pathogens. Due to its focus on research use we do not have to go through the phases of approval resulting in accessibility to everybody. By marketing on websites and magazines with scientific content we effectively get in contact with our target group. To sum it up, we can provide two things to potential customers. First a binding protein generating system to researchers and the specific customized binding proteins themselves.
</DIV><U><DIV class="container text_header"><H4><CENTER>Idea:</CENTER></H4></DIV></U><DIV class="container text">
The impact of antibodies in modern medicine is on a permanent rise. However, major problems are costs and time for development and production, as well as ethical discussions concerning the immunization of animals that are killed during the harvesting process. We want to establish an alternative using antibody-like binding proteins that are generated <I>in vivo</I> in <I>E. coli</I>. Profiting of the short generation time and the exponential growth of bacteria, we want to generate and optimize binding proteins in a much shorter period of time while also being more cost efficient. Furthermore, no animals are to suffer our process. The binding proteins can subsequently be utilized in both, diagnostics and maybe even in therapy against pathogens. Due to the ability of our system to adapt quickly to a specific target protein under evolutionary pressure it is especially useful in concern of quickly evolving and newly arising viral pathogens.
Our goal is on one hand to develop binding proteins in <I>E. coli</I> in a process of directed evolution that can be utilized in diagnostic techniques and target-mediated drug delivery. On the other hand we want to add a production process to sell custom ordered binding proteins. The concept of our system is structured into the following aspects: At first, we created a library of random binding protein sequences in bacteria forming the starting point of our project. After that we used a special plasmid which generates a higher mutation rate in a specific region. Finally, we developed a selection system to select proteins that show the best affinity to the target protein. Mediated by the binding of our improved protein to the target, the cell in concern is granted a selective advantage directly increasing its evolutionary fitness.
The potential of our project is a quick response to suddenly emerging viruses. We provide a quick response to these threats. Whether the pathogens are new, mutated from an animal only pathogen to a human pathogen or already known viruses that changed or adapted in a way that the present treatment is no longer effective. Our goal is to reduce the time for the development of an binding protein and also make the immunization of animals obsolete. This not only benefits human in the detection and treatment of diseases but also stops the killing of animals. Due to its time efficiency the spread of the pathogen is inhibited earlier, further reducing the number of people become diseased and preventing any damage caused by the pathogens. Our target audience are patients with a viral infection. The corresponding doctors acting as the intermediates. We also plan to sell stocks to governments in order to fight a sudden outbreak in the fastest and most efficient way with a minimum of infections. Health is and will always be one of the greatest treasures in life and we want to preserve it.
</DIV><U><DIV class="container text_header"><H4><CENTER>Market and competition:</CENTER></H4></DIV></U><DIV class="container text">
In the beginning, the market was rather small due to its focus mostly on small research teams. But due to the recent of new viruses like Denge, Ebola and most recently Zika the demand risen significantly. The more frequent appearances of several epidemic viruses are alarming and as a reaction to that more and more researchers choose to further tackle the new threat. So the demand is on the rise. The market itself is always changing as soon as a new method is introduced. Several groups are testing new procedures and systems. When it proves to be sufficient, it is fast adapted by everybody in the field. The competition we will face are producers and license holder of current antibodys and antibody production methods. These still involve immunization of animals like rabbits or mice. This can only be done by specially equipped labsor ordered from a producer of antibodies. These already available methods are time consuming and expensive. We offer a cheap and fast method that can be applied even in small labs. We can also easily modify our system to the clients likes and demands while current producers are more limited in their product range. Because our system is aimed mostly at research groups, there are only a few barriers to enter the market. We are acting as the intermediates before clinical applications. For research use only products have very few limitations for example the declaration of dangerous chemicals or GMOs which handling procedures have to be added.
</DIV><U><DIV class="container text_header"><H4><CENTER>Goals:</CENTER></H4></DIV></U><DIV class="container text">
We intend to provide researchers with the sufficient equipment to start their research and quickly produce results that can be medically applied. The system was designed to focus on a wide range of targets. Furthermore, it can be used in various fields of research and in a lot of contexts. Our vision is easy accessible binding proteins that are quickly available and affordable for small labs. Therefore, even a small research groups with a limited budget can use it. Our short time goal is to quickly introduce our system to the market and collect as much feedback as possible to further improve our Evobody production. The longtime goal is to establish our system as the starting point for all target or binding protein related research. We also want to develop our range of products and offer sites for fusion proteins to widen the field of applications.
</DIV><U><DIV class="container text_header"><H4><CENTER>Strategy:</CENTER></H4></DIV></U><DIV class="container text">
What makes our system unique is that we do not need a specific template sequence for ourbinding protein. Due to other randomization and mutation systems we gradually generate the perfect binder regardless of the origin of the target. So everyone who  need a binder to its target can easily get one himself in a simple and quick manner. Our system would be the next step in target related research that even spreads to the medical field because of its application in target drug delivery, the tagging of certain cells like cancer cells and the inactivation or killing of viruses and pathogens. We aim to provide the starting point and make fast responses to new threats possible. The process from a  new developed antibody to a real life application is time consuming due to the requirement of going through the three phases of approval. For example the FDA requires tests on animals, tests on volunteers (Phase 1) to determine its safety and side effects, phase 2 that tests its effectiveness by comparing it to different types of treatments and phase 3 that further examines its effectiveness in a large scale study that also explores the necessary dosage and combination with other drugs. The complete process is shown in the following table:<SUP>1</SUP></DIV><CENTER><TABLE><TBODY><TR><TH>Step</TH><TH>Requirment</TH></TR><TR><TD>Animal Testing</TD><TD>-preclinical testing on laboratory animals</TD></TR><TR><TD>IND Application</TD><TD>-review by a panel of scientists and non-scientists in hospitals and research institutions that oversees clinical research. </TD></TR><TR><TD>Phase 1 Testing</TD><TD>-studies conducted in 20 to 80 healthy volunteers to determine what the drug's most frequent side effects are and how the drug is metabolized and excreted</TD></TR><TR><TD>Phase 2 Testing</TD><TD> -obtain preliminary data on whether the drug works in people (from a few dozen to about 300) who have a certain disease or condition by comparing patients receiving the drug with similar patients receiving a different treatment  -short-term side effects are studied. Typically, the number of subjects in Phase 2 studies ranges from a few dozen to about 300</TD></TR><TR><TD>Phase 3 Testing</TD><TD>-begins if evidence of effectiveness is shown in Phase 2
-gathering more information about safety and effectiveness, studying different populations and different dosages and using the drug in combination with other drugs
-number of subjects ranges from several hundred to about 3,000 people</TD></TR><TR><TD>Review Meeting</TD><TD>-postmarket requirement and commitment studies to gather additional information about a product's safety, efficacy, or optimal use</TD></TR><TR><TD>NDA Application</TD><TD>-collection of all the data from the previous steps</TD></TR><TR><TD>Application Reviewed</TD><TD>-the FDA has 60 days to decide whether to file it so that it can be reviewed
</TD></TR><TR><TD>Evaluation</TD><TD>- FDA review team is assigned to evaluate the sponsor's research on the drug's safety and effectiveness
</TD></TR><TR><TD>Inspection of labelling and facilities of production</TD><TD>-reviews information on how to use the drug
- inspects the facilities where the drug will be manufactured
</TD></TR><TR><TD>Approval
</TD><TD>-FDA reviewers will approve the application or issue a complete response letter
</TD></TR></TBODY></TABLE></CENTER><U><DIV class="container text_header"><H4><CENTER>Marketing:</CENTER></H4></DIV></U><DIV class="container text">
Our product would be the custom binding protein that specifically meets the customerâs wishes. We will focus on quality not quantity to satisfy our customer. The package would also include a detailed procedure with the basic steps, tips, tricks and possible modifications for further use and handling. The price will be affordable due to the main expenses being labor cost and the materials for purification. The contents of our product itself are cheap and produced by ourselves. We will distribute our product, the lyophilized binding protein, by mail order which provides a fast and efficient way. Advertising will mainly takes place on websites with scientific content in the field of microbiology and molecular biology as well as scientific journals or magazines. Because of this a rather small budget is needed and our product is presented specifically to subject related audience. This provides us a customer oriented and cost efficient form of marketing.
</DIV><U><DIV class="container text_header"><H4><CENTER>Organisation:</CENTER></H4></DIV></U><DIV class="container text">
The management structure of our company will contain all necessary assignments while keeping it simple to avoid unnecessary costs.
</DIV><CENTER><FIGURE class="figure"><I>Fig. :1 Managment Structure</I></FIGURE></CENTER><DIV class="container text">
We will mostly employ molecular biologists, biotechnologists and biotechnical assistants. The most advantageous location would be an enterprise zone because of good accessibility to highways and thus faster transportation.
</DIV><U><DIV class="container text_header"><H4><CENTER>Law:</CENTER></H4></DIV></U><DIV class="container text">
For our laboratory we would need a safety level 1 laboratory which involves registration with the government, regularly controls and special equipment to ensure the safety of personnel, nature and humans. Aside from that no additional permit is necessary. We will call our company âEvobody Incorporatedâ which also indicates our legal form.
</DIV><U><DIV class="container text_header"><H4><CENTER>Finances:</CENTER></H4></DIV></U><CENTER><FIGURE class="figure"><I>Fig. :2 Profit calculation first year</I></FIGURE><FIGURE class="figure"><I>Fig. :3 Profit calculation second year</I></FIGURE><FIGURE class="figure"><I>Fig. :4 Profit calculation third year</I></FIGURE></CENTER><DIV class="container text">
The calculation shows that it is possible to make profit after 15 month. That is based on slowly rising orders. It is possible that we reach this point earlier or later depending on how fast our company gains public attention and thus more orders.
</DIV><U><DIV class="container text_header"><H4><CENTER>SWOT-analysis:</CENTER></H4></DIV></U><DIV class="container text">
The strength of our product lies in its speed and customizability. The customer can specifically request his binder for a specific application. On the other hand our binder could also cover a wider range due to the mutation system. It depends on the length of cultivation. Moreover, we will offer our Evobodys at a competitive price due to our system. We have to tackle the competition of antibody producers with our system which is new and approaches the topic of creating antibodys from a new perspective and offers various new appliances. One of the major things to work on would be spreading the reliability of our system and the quality of our binders to build up trust in our system. By keeping a low price and great range of applications we give smaller research groups with low budgets the opportunity to work in this field. Another point is the fast response to pathogens to relive medical facilities and reduce victims. Possible competitors are companies that already produce antibodies upon request. Although their range of binders is limited and expensive compared to our product range.
</DIV><DIV class="container text"><I> 1 http://www.fda.gov/drugs/resourcesforyou/consumers/ucm143534.htm</I></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>