<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Results_Selection_InteractionControl skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Results/Selection/InteractionControl</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container main"><DIV id="header"><NAV id="navigationbar" class="navbar navbar-default custom_navbar_style navbar-fixed-top"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="#">NavBar</A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-center"><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description"> PROJECT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description">Description</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds">Scaffolds</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design">Design and Construction</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/CreateYours">Create your own Library</A></LI></UL><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/EpPolI">Error prone PolymeraseI</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/Global">Genome wide mutator</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">Bacterial Two-Hybrid System</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/SplitBetaLactamaseHitchHiker">Alternative Selection System</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_I">Model I</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_II">Model II</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results"> RESULTS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Phage">Phagemid Display</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">Reversion</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/Cloning">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/ExpressionControl">Expression Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/InteractionControl">Interaction Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/BindingControl">Binding Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/KnockOutKnochIn"><I>in vivo</I> Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Modeling">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Fermentation">Fermentation</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Best Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Best Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Best Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts/Improve">Improve a part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Demonstrate your Work</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Medal_Fulfillment">Medal Fulfillment</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold"> HUMAN PRACTICES </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Dictionary">Dictionary</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks">Educational Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Crowdfunding">Crowdfunding and Press</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Achievements</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations"> COLLABORATIONS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Duesseldorf">DÃ¼sseldorf</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Lethbridge">Lethbridge</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Freiburg">Freiburg</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments"> NOTEBOOK </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook">Lab Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">Primers</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols">Lab Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Media">Media and Buffers</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team"> TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team">Team Profile</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Members">Members</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Contact">Contact</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement"> ACKNOWLEDGEMENT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Sponsors_And_Partners">Sponsors and Partners</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></DIV></DIV></NAV></DIV></DIV><DIV class="container main"><DIV class="container text_header"><H1>Interaction</H1></DIV><DIV class="container text_header"><H3>Protein purification with IMPACT</H3></DIV><DIV class="container text"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 1: Bradford-Assay.</B> The protein concentration was measured by a Bradford-Assay. With the fitting of the known BSA-samples the concentrations of the purified SH2-cI protein (red) and of the purified HA4-RpoZ protein (blue) could be measured. The purified proteins were diluted by the factor 100 to accomplish evaluable results. </FIGCAPTION></FIGURE>		
The first step for the interaction control of HA4 and SH2 required purified proteins. For the purification we used the <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols#impact">IMPACT</A> (Intein Mediated Purification with an Affinity Chitin-binding Tag)
		Kit from NEB (New England Biolabs).An intein-tag and a chitin-binding domain were translational fused to our protein of interest and integrated in the vector 
		pTXB1. After cultivation of the pTXB1-carrying <I>E. coli</I> strain ER2566, we did a cell lysis via French Press to prevent the from being damaged. The actually purification happens in a chitin column.
		The chitin binding domain, which is fused at our protein of interest, mediated a binding between our protein and the chitin beads in the column. After some wash steps all other proteins should be removed and only the bounded protein should be left in the column. A buffer with DTT activates at last the self-cleavage of the intein-tag at our protein and the protein disengages from the column and can be
		collected in a new tube.
		
		The <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols#bradford">Bradford-Assay</A> of our purificated proteins revealed a very high concentration. Compared with a standard series of different BSA concentrations
		we calculated a protein amount of 15 mgâml for the HA4-RpoZ fusion protein and 20 mgâml for the SH2-cI fusion protein (figure 1). These results coincides with the <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols#sds_page">SDS-PAGE</A>.
		There is a strong band on lane in the gel indicating proteins of about 25 kDa for SH2-cI and about 22 kDa for HA4-RpoZ, respectively. These results a very similar to the results from the <A href="https://2016.igem.org/wiki/index.php?title=Team:Bielefeld-CeBiTec/Results/Selection/ExpressionControl">expression control</A> and show that the two proteins you can the 
		see in the gel are definitely the two fusion-proteins. Therefore, it is sure, that the purification worked very well.
		</DIV><DIV class="container text_header"><H3>Affinity Chromatography</H3></DIV><DIV class="container text">
		After the purification has worked it was necessary to proofe if our controls are interact with each other. The first attempt was an affinity chromatography with these two proteins. For an affinity chromotography it is important that you immobilize
		one of your proteins. Therefore, the IMPACT experiment was repeated only for the SH2-cI fusion protein. The headed discoveries demonstrated that it is possible to bind a protein with chitin binding domain at the chitin column.
		Therefore, we used this to immobilize our SH2-cI protein at the chitin beads of the column. But before we induced the self-cleavage of the intein-tag of our protein we make a additional step with adding also purificated HA4-RpoZ in the column with one additional washing step.
		No interaction of HA4 and SH2 would lead to total clean up of our HA4 from the column after washing it. However, a binding of HA4 with our designed SH2 domain would hold some HA4 back in the column. After self-cleavage of the intein-tag the bounded HA4 proteins would also be detectable 
		in the purificated protein solution. 
		
		To prove this adoption we measured both purificated proteins SH2-cI and HA4-RpoZ and our affinity chromatography protein solution in the MALDI-TOF(Matrix-assisted Laser Desorption/Ionisation-Time Of Flight) to compare the protein mass of the measurement with the expectet masses.
		<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 2: MALDI-Results for the purified HA4-RpoZ protein.</B></FIGCAPTION></FIGURE><FIGURE class="figure"><FIGCAPTION class="figure-caption">Figure 3: MALDI-Results for the purified SH2-cI protein.</FIGCAPTION></FIGURE></CENTER>
		MALDI-TOF features two very definite masses for both single purificated proteins HA4-RpoZ and SH2-cI. If we compare these masses with the expectet ones,the masses 22.2 kDa and 24.98 kDa are detectable. But next to them two masses are very present with
		about half of the expectet masses, therefore, 11.1 kDa for HA4-RpoZ and a very high amount of 12.49 kDa for SH2-cI. In MALDI-TOF measurement the proteins are ionised by a implemented laser. Through this ionisation it is possible, that beside only one created 
<FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 4: MALDI-Results of the purified SH2-cI protein and the affinity chromatography.</B> By the comparison of the purified SH2-cI protein with the elution of the affinity chromatography two peaks differ from each other. These peaks are marked with the black box.</FIGCAPTION></FIGURE>

		charge for your protein, two or even three charges are possible. Whereas on the x-axes of the MALDI created diagramm the mass per charge is plotted, the accurate kDa-size is divided by the amount of charges on the protein. Therefore, the two very high present masses 
		different from the expectet mass are the same protein with two instead of one charge on it. The MALDI results verificate the results of the IMPACT Kit and evince, that HA4-RpoZ as well as SH2-cI are really purificated with a very high concentration.
		If the MALDI results of the purificated SH2-cI are compared with the results of the purificated SH2-cI after the affinity chromatography the digramms are broadly the same except of two peaks. It was detected that the two peaks have protein masses of 22,14 kDa and 11,07 kDa.
		A look back to the masses of the purificated HA4 protein, nearly the same two peaks are detected as well.  This is a very big clue on the fact, that some HA4 proteins are also in the same protein solution then SH2-cI after elution. Therefore it is possible, that SH2-cI and HA4-RpoZ are 
		interacting with each other. 
		</DIV><DIV class="container text_header"><H3>Interaction Measurement with BLItz</H3></DIV><DIV class="container text">	
		With the affinity chromatography experiment a first clue of an interaction between SH2 and HA4 was given. To validate the results a second experiment was implemented. The direct interaction of SH2 and HA4 was measured with the <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols#blitz">BLItz System</A>
		of ForteBio. For the BLItz (Bio-layer interferometry) measurement it is necessary to immobilize one of your proteins at the biosensor surface of your tip with a biocompatible matrix. The BLItz system works by emiting white light down the biosensor and collecting any light reflected back.
		Reflected wavelength are are affected by the thickness of the coating on the optical layer. Some of these wavelength show some interferences, which are captured by a spectrometer as a unique spectral signature. Any change in the number of molecules bound to the biosensor causes a shift in the interference
		pattern that is measured in real time. Therefore, it is possible to measure if two proteins are realy interacting with each other. 
		In our case the HA4 protein was fixed with a amino matrix directly at the biosensor. As the targets we choose obviously the SH2 protein and as a negativ control BSA, thereby it is excluded, that HA4 do not bind with every protein.
		The figure 5 shows the results of the BLItz experiment. At first the basic interference was defined by doing a measurement with only <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Media">PBS buffer</A>. After 60 seconds the second protein was added to the PBS buffer. The green, black, orange and purple curve describe the results of adding SH2. 
		Directly after that point of SH2 addition the graph raises constantly. Therefore, a interference between the basic line and protein addition can be measured. Such an interference is only visible, if their is an interaction between our fixed protein (HA4) and our target (SH2). This is a proof, that these two 
		proteins are interacting with each other.
		<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 5: Results of the BLItz experiment.</B> Illustrated are the measured base line, the association and the dissociation of the protein-protein interaction in the BLItz. The addition of the second protein is marked at 60 seconds after measuring the base line. The SH2-cI fusion protein was added at the start without washing (green), after one (purple) and two (yellow) washing steps with NaCl and after a washin step with a high concentrated HCl solution (upper blue). Also BSA was measured with 1 Î¼g/ml (lower blue) and 10 Î¼/ml (red) protein.</FIGCAPTION></FIGURE></CENTER>
 The reason 4 graphs with different interference strength are visible after SH2 addition can be attributed to the washing procedure after every measurement. The used washing solutions like sodium chloride are weak washing procedures to not damage the immobilized protein.
		This results in a not complete closed dissociation. Therefore, a bit targets are left binding on our protein. If the new base line is measured, the base interference is much higher based on the not dissolved targets. After the next addition of our SH2 target the difference between the base interference and new 
		interference is smaller then on previous tests. Therefore, with every run the results a bit weaker than before.
		To test the binding of HA4 to other proteins to exclude a general binding of HA4, the BLItz experiment would be repeated with BSA as target. Both graphs red and blue reveal no differences in the interference line after addition of the BSA protein. This demonstrate, that the interaction of SH2 and HA4 is a very specific 
		interaction. 
		
		These results leads to the same outcome than the affinity chromatography experiment, our positive controls SH2 and HA4 have a high affinity to each other and a strongly interaction.
<CENTER><BUTTON><A class="button_link" href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/BindingControl" role="button">Binding Control</A></BUTTON></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>