<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Results_Selection_BindingControl skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Results/Selection/BindingControl</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container main"><DIV id="header"><NAV id="navigationbar" class="navbar navbar-default custom_navbar_style navbar-fixed-top"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="#">NavBar</A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-center"><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description"> PROJECT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description">Description</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds">Scaffolds</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design">Design and Construction</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/CreateYours">Create your own Library</A></LI></UL><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/EpPolI">Error prone PolymeraseI</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/Global">Genome wide mutator</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">Bacterial Two-Hybrid System</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/SplitBetaLactamaseHitchHiker">Alternative Selection System</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_I">Model I</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_II">Model II</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results"> RESULTS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Phage">Phagemid Display</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">Reversion</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/Cloning">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/ExpressionControl">Expression Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/InteractionControl">Interaction Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/BindingControl">Binding Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/KnockOutKnochIn"><I>in vivo</I> Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Modeling">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Fermentation">Fermentation</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Best Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Best Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Best Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts/Improve">Improve a part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Demonstrate your Work</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Medal_Fulfillment">Medal Fulfillment</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold"> HUMAN PRACTICES </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Dictionary">Dictionary</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks">Educational Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Crowdfunding">Crowdfunding and Press</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Achievements</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations"> COLLABORATIONS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Duesseldorf">DÃ¼sseldorf</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Lethbridge">Lethbridge</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Freiburg">Freiburg</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments"> NOTEBOOK </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook">Lab Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">Primers</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols">Lab Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Media">Media and Buffers</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team"> TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team">Team Profile</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Members">Members</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Contact">Contact</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement"> ACKNOWLEDGEMENT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Sponsors_And_Partners">Sponsors and Partners</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></DIV></DIV></NAV></DIV></DIV><DIV class="container main"><DIV class="container text_header"><H1>Binding of the DNA binding protein cI</H1></DIV><DIV class="container text_header"><H3><I>In vitro</I> binding</H3></DIV><DIV class="container text">
		The next step of the working bacterial two-hybrid system is the examination of the interaction between the DNA binding domain and the possible binding site upstream of the promoter region. An <I>in vitro</I> interaction
		was demonstrated via electrophoretic mobility shift assay (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols#emsa">EMSA</A>) with the single OR1 binding site (<A href="http://parts.igem.org/Part:BBa_K2082231">BBa_K2082231</A>) and the double OR1 and OR2 binding site (<A href="http://parts.igem.org/Part:BBa_K2082239">BBa_K2082239</A>)
		of the phage 434. 

The DNA fragment without addition of the protein SH2-cI runs way faster than after adding the SH2-cI protein (figure 1A). This observation indicates an interaction between 434 cI an the binding site OR1. Binding of the protein to the DNA increased the weigth of the DNA fragment in the gel, which results in the slower movement and the visible band at the higher position in the gel.
		To disprove the possibilities of general DNA binding of the cI protein, another fragment was tested with nearly the same fragment size than the OR1 fragment but with a different sequence. However, the agarose gel did not reveal any recognizable differences between addition of SH2-cI and no addition of SH2-cI. This result confirmed our conjecture, that there is only an interaction between
		cI and the cI binding site. 
The binding at the single binding site OR1 of cI is compared with the binding at the double binding site OR1 and OR2(figure 1B). In both cases, a shift by the SH2-cI protein with the DNA is noticeable, but it is not possible to see a stronger shift with the doubled binding site. Therefore, the construct with one 434 binding site is sufficiently for 
		binding with the cI protein. Moreover, it was demonstrated that one binding site upstream of the promoter is enough for a strong binding and adequate for the bacterial two-hybrid system used.
<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 1: EMSA results. (A)</B> In alternaty application the DNA fragment with the 434 OR1 binding site and the DNA fragment with the lambda binding site were used as DNA sample. +/- = with/without SH2-cI fusion protein.<B>(B)</B> The DNA fragments containing the single OR1 and the double OR1 and OR2 binding site were applied. The SH2-cI protein was added with an amount of 0, 0.5, 5 and 50 pmol.</FIGCAPTION></FIGURE></CENTER></DIV><DIV class="container text_header"><H3><I>In vivo</I> binding</H3></DIV><DIV class="container text"><CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 3: Illustration of the <I>in vivo</I> experiment.</B> The experiment is designed by an GFP generator upstream of the reporter. If no cI is binding at the OR1 binding site, the RNA polymerase, which is bound at the anderson promoter of the GFP can read over the binding site and express the GFP and the RFP gene(<B>A</B>). If cI is binding, the RNA polymerase is stopped by the binding protein and only the GFP can be expressed (<B>B</B></FIGCAPTION></FIGURE></CENTER>

		A new construct was produced as <I>in vivo</I> binding control. A GFP gene under the control of a constitutive Anderson promoter (<A href="http://parts.igem.org/Part:BBa_K608010">BBa_K608010</A>) was cloned directly upstream of the reporter construct of the single reporter part <A href="http://parts.igem.org/Part:BBa_K2082211">BBa_K2082211</A> and of the reporter and SH2-cI combined part
		<A href="http://parts.igem.org/Part:BBa_K2082231">BBa_K2082231</A> without any terminator. The expectation was given, that the Anderson promoter would be able to increase the transcription rate of the RFP if the RNA polymerase can not be stopped. The polymerase is able to read over the second promoter and leads to an expression of the normally weakly expressed RFP gene.
		If the cI protein would bind at the binding site between the GFP gene and the optimized <I>lacZ</I> promoter (<A href="http://parts.igem.org/Part:BBa_K2082210">BBa_K2082210</A>), it is possible that the polymerase has to stop the expression due to steric inhibition. Therefore, the relation of RFP to GFP expression could give a clue if cI is binding <I>in vivo</I>.
		To analyze this hypothesis the GFP and RFP expression are measured from a liquid culture of two <I>E. coli</I> cultures carrying different plasmids. The first culture was transformed with the GFP coupled part <A href="http://parts.igem.org/Part:BBa_K2082211">BBa_K2082211</A> and the second with the GFP coupled part <A href="http://parts.igem.org/Part:BBa_K2082231">BBa_K2082231</A>. 
		The main difference of these two parts is the ability of BBa_K2082231 of producing the SH2-cI fusion protein.
		
		A first measurement was performed in the Infinite M200 plate reader (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols#tecan">Tecan</A>). The culture without producing the cI protein shows a bit higher RFP intensity (figure 3). Integrating the error bars in the analysis, it is not possible to say if their is a real difference between the two constructs designed for this experiment.
		<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 3: Tecan results.</B> In the Tecan plate reader the RFP intensity was measured. The <I>E. coli</I> cultures with (red) and without (black) producing the cI repressor were analyzed. Error bars are implemented.</FIGCAPTION></FIGURE></CENTER>
		Therefore, a second measurement was conducted in the <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols#facs">FACS</A> (fluorescence-activated cell scanning) for direct measurement of single cell events and verification of the results given by the TECAN. At first, the GFP expression of the two designed constructs was compared with the native reporter<A href="http://parts.igem.org/Part:BBa_K2082211">BBa_K2082211</A>.
		In total, 50,000 cells were measured in the FACS system, which revealed a much higher GFP intensity produced by the GFP gene carrying cells (figure 4A). This supports the supposition, that GFP was correctly cloned upstream of the reporter sequences. A comparison of the RFP intensity of the reporter without the GFP gene with the modified version exhibits an about 85% stronger RFP production in the cells with the GFP gene
		and the reporter (figure 4B). The RFP expression increases with the read over of the RNA polymerase, after docking at the constitutive Anderson promoter region upstream of the GFP gene. 
		
		A comparison of the cI-SH2 and GFP producing cells with the cells only producing GFP also demonstrated differences in the RFP intensity (fugure 4C). The cells with cI-SH2 proteins had an about 35% lower RFP signal after measuring the RFP intensity in 50,000 single cells. Normalized on the GFP production, the differences raises to 44% lower RFP intensity per measured GFP intensity.
		<CENTER><FIGURE class="figure-img" class="figure"><FIGCAPTION class="figure-caption"><B>Figure 4: FACS measurement.</B> The three bacterial cultures with the single reporter (blue), the GFP and the reporter (red) and the GFP, reporter and carrying the functional cI-SH2 fusion protein were measured. Illustrated  is the GFP(A) or RFP(B,C) intensity in dependence of the measured cell events. The arithmetic mean is pictured under the graph.</FIGCAPTION></FIGURE></CENTER>
		Therefore, an <I>in vivo</I> validation of the DNA binding domain is also possible. One possible explanation for the reduced RFP production copared to the native reporter level with the binding of cI is the possibility that the binding protein could compete with the RNA polymerase for the DNA sequence with the binding site. Therefore, the polymerase
		represses the binding cI protein and continues the expression of the genes. Under normal conditions the cI protein binds at four binding sites and builds an octamere structure to completely repress the transcription activity. The single binding site OR1 could lead to a lower binding strength of cI and therefore, a higher chance that the RNA polymerase could repress the cI protein.
		All in all the binding of cI was shown under <I>in vivo</I> and <I>in vitro</I> conditions.
		</DIV><CENTER><BUTTON><A class="button_link" href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/KnockOutKnochIn" role="button"><I>in vivo</I> selection</A></BUTTON></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>