Team:Bielefeld-CeBiTec/Results/Library/Assembly
Toggle navigation
NavBar
PROJECT
Description
Library
Overview
Scaffolds
Design and Construction
Create your own Library
Mutation
Overview
Error prone Polymerase I
Genome wide mutator
Selection
Overview
Bacterial Two-Hybrid System
Alternative Selection System
Modeling
Overview
Model I
Model II
RESULTS
Overview
Library
Overview
Assembly
Phagemid Display
Sequencing
Mutation
Overview
Assembly
Reversion
Sequencing
Selection
Overview
Assembly
Expression Control
Interaction Control
Binding Control
in vivo Selection
Modeling
Fermentation
Parts
Best Basic Part
Best Composite Part
Best Part Collection
Improve a part
Proof of Concept
Demonstrate your Work
Medal Fulfillment
HUMAN PRACTICES
Overview
Engagement
Dictionary
Educational Outreach
Crowdfunding and Press
Entrepreneurship
Integrated Practices
Achievements
COLLABORATIONS
Overview
Düsseldorf
Lethbridge
Freiburg
NOTEBOOK
Overview
Lab Notebook
Library
Mutation
Selection
Primers
Lab Protocols
Media and Buffers
TEAM
Team Profile
Members
Bielefeld University
Contact
ACKNOWLEDGEMENT
Overview
Sponsors and Partners
Attributions
Library Results
Assembly
Assembly
As our library is made of binding proteins possessing variable as well as constant regions we had to assemble them in a special manner. For the assembly of our library we had to anneal the fragments containing the variable regions that we ordered through oligonucleotide synthesis.
As for Monobodies we assembled the fragments called V1-1 and V1-2 to V1 and V2-1 and V2-2 to V2, and then again V1 and V2 to have the variable region essential for the construct. The correct size of the whole assembled construct was estimated by a gelelectrophoresis (figure 2) and then later sequenced . For the Monobody construct we deciced to set the design up in a way that one can exchange RFP for the fragment containing the variable (randomized) regions (Figure 1a, 1b). The size of the annealed V1 fragment is 114 bp, the size of V2 is 132 bp. These are the fragments inserted into our Monobody construct which can be used to build the library .This construct was submitted by us as BBa_K2082000 for the simple Monobody and as BBa_K2082004 as the Monobody construct equipped for use in our system .
Figure 1a: Monobody construct with RFP. Construct for the Monobody with RFP insert exchangable for variable regions.
Figure 1b: Monobody construct with RFP and fusion to RNA-Polymerase omega subunit. Construct for the Monobody with RFP insert exchangable for variable regions and usable in our system.
To get further insight into the theoretical assembly take a look here .
Figure 2: Gelelectrophoresis of assembled construct for Monobodies. Photography of 1% agarose gel dyed with ethidium bromide. Loading of the samples (from right to left): Monobody construct with variable region (MB), 1kb DNA Ladder (by New England Biolabs ).
We further annealed F2-1 and F2-2 to the region containing the variable region called F2 for the Nanobody construct. To get more insight take a look here .
The size of the annealed F2 fragment is ought to be 143 bp. The size of the assembled construct was checked by gelectrophoresis (figure 4) first and sequencing . This then can be inserted into the Nanobody construct equipped for use in our system ( BBa_K2082001 ) (Figure 3) to create the Nanobody library.
Figure 3: Complete Nanobody construct with variable regions and fusion to RNA-Polymerase omega subunit. Construct for Nanobody to have the variable region inserted in, in fusion with the omega subunit of the RNA-Polymerase omega subunit.
Figure 4: Gelelectrophoresis of assembled construct for Nanobodies. Photography of 1% agarose gel dyed with ethidium bromide. Loading of the samples (from right to left): 1kb DNA Ladder (by New England Biolabs , NB construct with variable region.
As we were sure our constructs were right, we strieved for having as much clones as possible. To create enough room for the high quantities of colonies we produced, we used big agar plates (picture below). They consisted of 750ml LB-Agar and provided space for the amount of about 15 normal agar plates. We totally recommend larger plates for every team with a library approach!
Comparison of Polymerases
As our variable regions (for each Monobodies and Nanobodies ) were ordered through oligonucleotide synthesis, the strands naturally were single stranded. For the correct assembly of the variable regions into the construct, they had to be annealed. For this purpose we tried using the Klenow Fragment (3'→5' exo-) by New England Biolabs at first. However, the cloning of the Klenow annealed fragments was not optimal.
For example, when inserting the variable regions for the Nanobodies, the primers called F2-1 and F2-2 had to be annealed, to then be cloned into the Nanobody construc t (missing the variable regions). When separating the samples (annealed oligonucleotides) by gel electrophoresis the reason why was hinted as no fragments of 143 bp were found in the bands on which the annealed sample were loaded (see Figure 5).
Figure 5: Gelelectrophoresis of annealed oligonucleotides. Photography of 1% agarose gel dyed with ethidium bromide. Loading of the samples (from right to left): Primer F2-1, Primer F2-2, 100bp plus ladder (by New England Biolabs ), empty space, three times the by the Klenow Fragment annealed parts, empty space, 100bp plus ladder (by New England Biolabs ).
We then concluded to use other Polymerases to anneal the oligonucleotides. For that we annealed two oligonucleotides called V1-1 and V1-2 to a fragment called V1  as well as V2-1 and V2-2 to a fragment called V2. These were used to bring in the variable region of the Monobody construct, inhabitating the randomized regions essential for binding proteins in the following.
For this purpose we used the following polymerases: Q5 High-Fidelity DNA Polymerase (by New England Biolabs ), KOD DNA Polymerase (by Merck Millipore ), Phusion High-Fidelity DNA Polymerase (by New England Biolabs ), GoTaq G2 (by Promega ).
In Figure 6.1 one can see the result of the annealing of the oligonucleotides V1-1 and V1-2 to the fragment V1 and V2-1 and V2-2 to the fragment V2 by the four polymerases.
Figure 6.1: Gelelectrophoresis of annealed oligonucleotides. Photography of 1% agarose gel dyed with ethidium bromide. Loading of the samples (from right to left): Low Molecular Weight DNA Ladder (by New England Biolabs ), Fragment V1 annealed with GoTaq G2 (by Promega ), Fragment V1 annealed with KOD DNA Polymerase (by Merck Millipore ), Fragment V1 annealed with Phusion High-Fidelity DNA Polymerase (by New England Biolabs ), Fragment V1 annealed with Q5 High-Fidelity DNA Polymerase (by New England Biolabs ), Fragment v2 annealed with GoTaq G2 (by Promega ), Fragment V2 annealed with KOD DNA Polymerase (by Merck Millipore ), Fragment V2 annealed with Phusion High-Fidelity DNA Polymerase (by New England Biolabs ), Fragment V2 annealed with Q5 High-Fidelity DNA Polymerase (by New England Biolabs ).
The same was done for the annealing of the oligonucleotides F2-1 and F2-2 to fragment F2 containing the variable region for the Nanobody. The gel photography can be viewed in figure 6.2.
Figure 6.2: Gelelectrophoresis of annealed oligonucleotides. Photography of 1% agarose gel dyed with ethidium bromide. Loading of the samples (from right to left): Low Molecular Weight DNA Ladder (by New England Biolabs ), Fragment F2 annealed with GoTaq G2 (by Promega ), Fragment F2 annealed with KOD DNA Polymerase (by Merck Millipore , Fragment F2 annealed with Phusion High-Fidelity DNA Polymerase (by New England Biolabs ), Fragment F2 annealed with Q5 High-Fidelity DNA Polymerase (by New England Biolabs ).
After choosing the Q5 High-Fidelity DNA Polymerase (by New England Biolabs ) for the annealing of the oligonucleotides we once again checked for the right size of the fragments mentioned above (V1, V2 and F2) on an 1% agarose gel (see Figure 7).
Figure 7: Gelelectrophoresis of Q5-Polymerase annealed oligonucleotides. Photography of 1% agarose gel dyed with ethidium bromide. Loading of the samples (from right to left): Low Molecular Weight DNA Ladder (by New England Biolabs ), three times Fragment V1 annealed with Q5 High-Fidelity DNA Polymerase (by New England Biolabs ), two times Fragment V2 annealed with Q5 High-Fidelity DNA Polymerase (by New England Biolabs ), two times Fragment F2 annealed with Q5 High-Fidelity DNA Polymerase (by New England Biolabs ), Low Molecular Weight DNA Ladder (by New England Biolabs ).
We so chose to procee d with further annealing with the Q5 High-Fidelity DNA Polymerase (by New England Biolabs ) as the bands were clearly sharper in the gel as well as the transformation efficiency was seemingly higher. The use of this polymerase also resulted in an up to nearly 8 fold greater (Figure 8) number of clones counted in later cloning compared to cloning steps where other polymerases were employed
Figure 8: Comparison of averaged number of colonies. After the transformation of assembled constructs, grown colonies were counted. The averaged values with standard error are shown in this bar graph.
To put it in a nutshell, the construction of the plasmids for our library was successful and delivered the right constructs to set up our library. You can view an example Monobody generated by us here (BBa_K2082009), or a part of our Monobody library here (BBa_K2082005). Likewise, you can find an example Nanobody here (BBa_K2082010), or a part of our Nanobody library here (BBa_K2082006).
The general visualisation of the constructs assembled can be found in figure 9 and 10.
Figure 9: General scheme of Monobody construct with variable regions. A schematic view of how variable regions are embedded in the Monobody construct.
Figure 10:General scheme of Nanobody construct with variable regions. A schematic view of how variable regions are embedded in the Nanobody construct.
Phage
