Team:Bielefeld-CeBiTec/Project/Library/Overview
Toggle navigation
NavBar
PROJECT
Description
Library
Overview
Scaffolds
Design and Construction
Create your own Library
Mutation
Overview
Error prone Polymerase I
Genome wide mutator
Selection
Overview
Bacterial Two-Hybrid System
Alternative Selection System
Modeling
Overview
Model I
Model II
RESULTS
Overview
Library
Overview
Assembly
Phagemid Display
Sequencing
Mutation
Overview
Assembly
Reversion
Sequencing
Selection
Overview
Assembly
Expression Control
Interaction Control
Binding Control
in vivo Selection
Modeling
Fermentation
Parts
Best Basic Part
Best Composite Part
Best Part Collection
Improve a part
Proof of Concept
Demonstrate your Work
Medal Fulfillment
HUMAN PRACTICES
Overview
Engagement
Dictionary
Educational Outreach
Crowdfunding and Press
Entrepreneurship
Integrated Practices
Achievements
COLLABORATIONS
Overview
DÃ¼sseldorf
Lethbridge
Freiburg
NOTEBOOK
Overview
Lab Notebook
Library
Mutation
Selection
Primers
Lab Protocols
Media and Buffers
TEAM
Team Profile
Members
Bielefeld University
Contact
ACKNOWLEDGEMENT
Overview
Sponsors and Partners
Attributions
Overview
Motivation
As every house needs a cornerstone we need our binding protein library. The demand of easily accessible, fast adapting and specific binding proteins is ever-growing. To utilize the advantages of binding proteins for our project, a library, adapting the end-users (or more specific: our) needs, was designed by us.
When we were faced the challenge of designing a binding protein library, earlier this year, we glanced at existing iGEM projects and were surprised by not finding approaches in similar manner. Actually - our research proved our approach to be unprecedented in this competition. Introducing and providing one therefor contributes to the community as well. In this way, we hope to show the benefits to pave the way for a higher usage of libraries in iGEM.
Overview
To achieve generating the perfect binder an initial library is essential for our project. Thus, we set up the foundation to build our in vivo mutagenesis system on.
Our library layout is based on synthetic binding proteins as well as naturally occurring binding proteins. To start with, we chose two different scaffolds: Nanobodies and Monobodies. Each possessing advantages compared to conventional antibodies, making them suitable for our system. Read more about our used scaffolds.
Setting up a humongous library, containing all different kinds of variants, seemed tempting at first, but having a more defined and optimized library was the more rational choice. For doing so we involved only specific amino acids in the randomized regions of our binding proteins, to optimize binding of targets ( Fellouse et al., 2004 ; Koide, 2009 ). Hence, we kept the theoretical variability to a maximum of about one billion (1,073,741, 824), just including estimatingly promising initial binders. See our Design and Construction for further insight.
Figure 1: Library of initial binding proteins. Expression of the initial binding proteins with variable regions highlighted in seperate colors (turquoise, orange, white, green, pink and blue). The theoretical variability for each scaffold is 1,073,741,824.
Literature
Fellouse, Frederic A., 2004. Synthetic Antibodies from a Four-Amino-Acid Code: A Dominant Role for Tyrosine in Antigen Recognition. Proceedings of the National Academy of Sciences of the United States of America, 2004, 101, 34, 12467.
Koide, S 2009. The Importance of Being Tyrosine: Lessons in Molecular Recognition from Minimalist Synthetic Binding Proteins. ACS CHEMICAL BIOLOGY, 2009, 4, 5, 325.
