Team:Bielefeld-CeBiTec/Results/Library/Overview
Toggle navigation
NavBar
PROJECT
Description
Library
Overview
Scaffolds
Design and Construction
Create your own Library
Mutation
Overview
Error prone Polymerase I
Genome wide mutator
Selection
Overview
Bacterial Two-Hybrid System
Alternative Selection System
Modeling
Overview
Model I
Model II
RESULTS
Overview
Library
Overview
Assembly
Phagemid Display
Sequencing
Mutation
Overview
Assembly
Reversion
Sequencing
Selection
Overview
Assembly
Expression Control
Interaction Control
Binding Control
in vivo Selection
Modeling
Fermentation
Parts
Best Basic Part
Best Composite Part
Best Part Collection
Improve a part
Proof of Concept
Demonstrate your Work
Medal Fulfillment
HUMAN PRACTICES
Overview
Engagement
Dictionary
Educational Outreach
Crowdfunding and Press
Entrepreneurship
Integrated Practices
Achievements
COLLABORATIONS
Overview
DÃ¼sseldorf
Lethbridge
Freiburg
NOTEBOOK
Overview
Lab Notebook
Library
Mutation
Selection
Primers
Lab Protocols
Media and Buffers
TEAM
Team Profile
Members
Bielefeld University
Contact
ACKNOWLEDGEMENT
Overview
Sponsors and Partners
Attributions
Library Results
Overview
Overview
We implement a basic diversity of initial binding proteins by creating a high-quality library consisting of over one hundred thousand Monobodies and over one hundred sixty thousand Nanobodies, respectively. Thereby we set up the foundation to build our system on and set a cornerstone for utilizing and generating libraries in iGEM.
We verified the functionality of our library by several approaches.
Assembly
We did not just show that our design and construction of our libraries was successful we also adapted the way to assemble them. Thus we established an unprecedented approach in this competition and provide future iGEM teams a great part collection to build their own libraries .
Figure 1: Gelelectrophoresis of assembled construct for Monobodies. Photography of 1% agarose gel stained with ethidium bromide. Loading of the samples (from right to left): Monobody construct with variable region (MB), 1kb DNA Ladder (by New England Biolabs ).
Phagemid Display
We showed that our libraries consists of several initial binders even in small extracts. Thus, the Evobody can get in the directed evolution and can be optimized by mutation and selection .
Figure 2: Example sequence of a Monobody, identified as FGE binder by phagemid display. Top to bottom: Ordered sequence, chromatogram and sequencing result.
Sequencing
Sanger sequencing was applied to check the correctness of assemblies and to examine the appearance of the different chosen bases for positions of the variable regions of our binding proteins, respectively. Also we calculated the actual library sizes by high-throughput sequencing technologies.
Figure 3: Nanobody randomized CDR3 of 24 colonies. Top to bottom: Ordered sequence, chromatogram and sequencing result. Randomized regions show the occurence of the expected bases.
Assembly
Phage
Sequencing
