<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Project_Library_CreateYours skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Project/Library/CreateYours</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container main"><DIV id="header"><NAV id="navigationbar" class="navbar navbar-default custom_navbar_style navbar-fixed-top"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="#">NavBar</A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-center"><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description"> PROJECT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description">Description</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds">Scaffolds</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design">Design and Construction</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/CreateYours">Create your own Library</A></LI></UL><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/EpPolI">Error prone PolymeraseI</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/Global">Genome wide mutator</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">Bacterial Two-Hybrid System</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/SplitBetaLactamaseHitchHiker">Alternative Selection System</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_I">Model I</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_II">Model II</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results"> RESULTS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Phage">Phagemid Display</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">Reversion</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/Cloning">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/ExpressionControl">Expression Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/InteractionControl">Interaction Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/BindingControl">Binding Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/KnockOutKnochIn"><I>in vivo</I> Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Modeling">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Fermentation">Fermentation</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Best Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Best Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Best Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts/Improve">Improve a part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Demonstrate your Work</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Medal_Fulfillment">Medal Fulfillment</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold"> HUMAN PRACTICES </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Dictionary">Dictionary</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks">Educational Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Crowdfunding">Crowdfunding and Press</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Achievements</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations"> COLLABORATIONS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Duesseldorf">DÃ¼sseldorf</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Lethbridge">Lethbridge</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Freiburg">Freiburg</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments"> NOTEBOOK </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook">Lab Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">Primers</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols">Lab Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Media">Media and Buffers</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team"> TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team">Team Profile</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Members">Members</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Contact">Contact</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement"> ACKNOWLEDGEMENT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Sponsors_And_Partners">Sponsors and Partners</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></DIV></DIV></NAV></DIV></DIV><DIV class="container main"><DIV class="jumbotron" style="background-image: url(https://static.igem.org/mediawiki/2016/8/89/Bielefeld_CeBiTec_2016_10_14_X_jessi2.png)"><DIV class="jumbotron-text"><H1 style="margin-bottom: 0px; text-align:left">Library Project</H1><H2 style="color:#ffffff; text-align:left">Create Your Own Library</H2></DIV></DIV><DIV class="container text_header"><H3>Implementing diversity</H3></DIV><DIV class="container text">

For our directed evolution system of binding proteins, we designed and constructed two libraries, containig proposed randomized CDS for 
<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds">Monobodies</A> or <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds">Nanobodies</A> as initial Evobodies. The detailed construction of our libraries is discribed in <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design">&quot;Design and Construction&quot;</A>. Plasmid libraries, coding for useful proteins, are a powerful tool in synthetic biology, but underestimated in iGEM, yet. Unfortunately, the construction of a library is a difficult and time consuming task. To supply this tool for future iGEM teams, we provided our libraries to the iGEM community and want to implement a new category in iGEM parts registry for libraries, coding several different proteins categories, like enzymes, binding proteins or aptamers. By doing so we want to establish libraries in iGEM and pave the way for every team to access the advantages of large diversity libraries as well as creating and submitting their own.
As a starting point, we submitted our Monobody and Nanobody libraries as plasmid mixes. These collections comprise over 100,000 (Monobodies), respectively over 160,000 (Nanobodies) of distinct sequences for binding proteins out of a theoretically size of over one billion (1,073,741,824) possible sequences. Additionally, we constructed fundamental frameworks composed of the Monobody or Nanobody constant regions, which can easily be extended for creating a specific binding protein or even your own library, as it is discribed in the following text. For that you only had to design your specific variable regions and get them synthesized in one or more parts.</DIV><DIV class="container text_header"><H3>Create your Monobody library:</H3></DIV><DIV class="container text">
To create your own <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds">Monobody</A> library, we constructed a fundamental framework (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2082000">BBa_K2082000</A>) composed of the Monobody constant regions and RFP instead of variable regions, which can easily be exchanged with randomized variable segments. Due to this, an easy selection trough visual control is possible. To insert self-constructed variable regions, you have to design a randomized insert with fitting overlaps to the adjacent constant Monobody sequences (Fig. 1). To integrate them, by replacement of RFP, the backbone with constant parts (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2082000">BBa_K2082000</A>) should be amplified by using the primers <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">MB-bb-fw</A> and <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">MB-bb-rev</A>. Afterwards the variable fragments, containing overlaps, can get inserted in the gained backbone by Gibson assembly.</DIV><DIV><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 1: Overview of the Monobody construction.</B> An individual insert can get integrated in the Monobody CDS. Constant regions are colored in green, variable regions are colored in blue.</FIGCAPTION></FIGURE></DIV><DIV class="container text_header"><H3>Create your Nanobody library: </H3></DIV><DIV class="container text">
For creation of a unique <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds">Nanobody</A> library you can use our framework, containing the constant Nanobody segments (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2082001">BBa_K2082001</A>). In this case, it is only necessary to create one variable segment, because in the used <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds">scaffold</A>, we have only randomized the sequence of the third Complementarity-determining region (CDR), which is the <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design">main binding CDR</A>. Amplifying <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2082001">BBa_K2082001</A> with primers <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">NB-bb-fw</A> and <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">NB-bb-rev</A> results to a backbone, to insert a double stranded and overlapping variable insert by Gibson assembly, to built a unique Nanobody library.
</DIV><DIV><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 1: Overview of the Nanobody construction.</B>Variable CDR 3 is colored in blue.</FIGCAPTION></FIGURE></DIV><DIV class="container text_header"><H3>Implementing diversity</H3></DIV><DIV class="container text">
It would push the iGEM parts registry forward and help future iGEM teams, if it would be possible to submit several plasmid libraries, encoding different binding proteins, enzymes, aptamers, or other useful proteins. So all following iGEM teams can build on it and screen the diversity for their desired protein with custom properties. For that reason, it has to be possible, to enter bases of the IUPAC nucleotide code including explicitly random bases, e.g. M, W, X and Y as a part sequence.
To distribute the submitted Monobody and Nanobody libraries for further iGEM teams by the iGEM headquarter, it is necessary to conserve the library diversity as effectively as possible. For that, recloning by chemical or electro transformation is the wrong way. It would be the best, to multiply the plasmid mixes by PCR. Using the primers <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">LIB-bb-fw</A> and <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">LIB-bb-rev</A>. After distribution the operator can realign the linearized Plasmid by using biobrick or Gibson assembly. To ensure, a high yield of different bacterial clones it would be rational, to use high efficient competent cells to gain an optimal variability.
<CENTER><BUTTON><A class="button_link" href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview" role="button">Results</A></BUTTON></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>