<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Results_Modeling skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Results/Modeling</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container main"><DIV id="header"><NAV id="navigationbar" class="navbar navbar-default custom_navbar_style navbar-fixed-top"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="#">NavBar</A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-center"><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description"> PROJECT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description">Description</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds">Scaffolds</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design">Design and Construction</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/CreateYours">Create your own Library</A></LI></UL><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/EpPolI">Error prone PolymeraseI</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/Global">Genome wide mutator</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">Bacterial Two-Hybrid System</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/SplitBetaLactamaseHitchHiker">Alternative Selection System</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_I">Model I</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_II">Model II</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results"> RESULTS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Phage">Phagemid Display</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">Reversion</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/Cloning">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/ExpressionControl">Expression Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/InteractionControl">Interaction Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/BindingControl">Binding Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/KnockOutKnochIn"><I>in vivo</I> Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Modeling">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Fermentation">Fermentation</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Best Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Best Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Best Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts/Improve">Improve a part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Demonstrate your Work</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Medal_Fulfillment">Medal Fulfillment</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold"> HUMAN PRACTICES </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Dictionary">Dictionary</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks">Educational Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Crowdfunding">Crowdfunding and Press</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Achievements</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations"> COLLABORATIONS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Duesseldorf">DÃ¼sseldorf</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Lethbridge">Lethbridge</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Freiburg">Freiburg</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments"> NOTEBOOK </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook">Lab Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">Primers</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols">Lab Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Media">Media and Buffers</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team"> TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team">Team Profile</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Members">Members</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Contact">Contact</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement"> ACKNOWLEDGEMENT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Sponsors_And_Partners">Sponsors and Partners</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></DIV></DIV></NAV></DIV></DIV><DIV class="container main"><DIV class="jumbotron" style="background-image: url(https://static.igem.org/mediawiki/2016/0/02/Bielefeld_CeBiTec_2016_10_14_X_projectheader.png
)"><DIV class="jumbotron-text"><H1 style="margin-bottom: 0px; text-align:left">Results</H1><H2 style="color:#ffffff; text-align:left">Modeling</H2></DIV></DIV><DIV class="container text">
We built a conglomeration of two models in order to find the best cultivation time and inoculation number.
This approach made it possible to predict when the desired proportion of the best adapted <I>E. coli</I> culture is reached.
</DIV><DIV class="container text_header"><H3>Model I</H3></DIV><DIV class="container text">
The model for the <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">bacterial two hybrid system</A>, <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_I">modelI</A>, needs the affinity value of the bacterial two hybrid complex as an input. 
It then simulates the events in the cell which are essential for the bacterial two hybrid system and the resulting production of beta-lactamase. 
As an extension the encounter of beta-lactamase and ampicillin, an antibiotic that harms the cell, is modeled. If the beta-lactamase concentration in the cell is too low and the ampicillin concentration is too high, the cell dies. If the proportion of ampicillin does not lead to death a high ampicillin concentration will cause growth inhibitions.

To ensure that our program can deal with all extreme cases, we made a simulation run of three different affinities: an optimal affinity, a moderate affinity and a bad affinity. The time scale for the events in the cell is limited to 20 minutes, because after 20 minutes it is assumed that the cell divides:
</DIV><FIGURE><FIGCAPTION><B>Figure 1:</B> (A) Beta-lactamase concentration in periplasm for optimal, moderate and bad affinity. (B) Beta-lactamase concentration in periplasm for an optimal, a moderate and a bad affinity. (C) Growth curves for optimal, moderate and bad affinity.
	</FIGCAPTION></FIGURE><DIV class="container text">
In figure 1 (A) the beta-lactamase concentrations in the periplasm for the 3 affinities are shown. 
In the first 200 seconds the beta-lactamase concentration does not rise very strongly, because the products for the bacterial two hybrid complex (B2H) must firstly be produced. When enough products for the B2H complex are present, the probability for the formation of the B2H complex gets high and the reporter gene, here the <I>beta-lactamase gene</I>, is often transcribed. Thus the production of beta-lactamase rises. 

After 20 minutes the strain with the optimal affinity comprises a significantly higher beta-lactamase concentration than the other two strains. 
Furthermore, the beta-lactamase curve resembling moderate affinity has an observable higher end concentration than the strain equipped with the bad affinity.
These results are as assumed, because the change in affinity for the two components of the B2H complex causes a particular expression of the reporter gene, here beta-lactamase. 
When the bacterial two hybrid complex dissociates quickly, the expression of the reporter gene is lower (bad affinity) than in the case of a more stable complex (moderate or optimal affinity).

In comparison to the beta-lactamase curves, figure 1 (B) shows the active ampicillin concentrations for the different affinities in the periplasm. For the first 200 seconds, the ampicillin concentration for all affinities extremely rises, because the beta-lactamase concentration is too low at this time and thus the probability for an encounter of beta-lactamase and ampicillin is low. After around 200 seconds the beta-lactamase starts to significantly deactivate ampicillin and the active ampicillin concentration falls. All ampicillin curves have a very strong decline, due to the fact that beta-lactamase is highly active.

As a result our program computes a final concentration of active ampicillin of 1.2682*10<SUP>-6</SUP>M for optimal affinity, 2.3616*10<SUP>-6</SUP>M for moderate affinity and 5.9811*10<SUP>-6</SUP>M for bad affinity. The final active ampicillin concentration is then converted into a reproduction rate for the particular <I>E. coli</I> cell (figure 1 (C)).
</DIV><DIV class="container text_header"><H3>ModelII</H3></DIV><DIV class="container text">
The computed reproduction rates, 1.0532h<SUP>-1</SUP> for optimal, 0.7248h<SUP>-1</SUP> for moderate and 0.1569h<SUP>-1</SUP> for bad affinity from model I are transferred to <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_II">modelII</A>, the competition model. This model simulates how <I>E. colis</I> with the given reproduction rates grow together in one flask. 
Furthermore, to model the several successive [inoculation steps] of our project, we simulated a transfer of a determined proportion of every culture after some time, immediately followed by a new cultivation round. In the laboratory, the pipetted amount of culture is assumed to be binomially distributed. Since the assumptions for applying an approximation according to the de Moivre-Laplace theorem (Rosenkrantz 1983) are fulfilled (n, corresponding to the number of cells, being large), we use the normal distribution as an approximation to the binomial distribution. The growth behavior under several inoculations is depicted in the following movie:
</DIV><CENTER><VIDEO width="1000px" src="https://static.igem.org/mediawiki/2016/f/fe/Bielefeld_CeBiTec_2016_10_18_Mod_curves_movie.mp4" autoplay="" controls=""> Your browser can not play this video.</VIDEO></CENTER><DIV class="container text">
As a result modelII creates a 3D plot that shows how long the cultivation must last and how often an inoculation must be done till the desired proportion of the best adapted <I>E. coli</I> culture is reached. For our system we want to reach a proportion of 95% of the total culture mix. The 3D plots for our 3 test affinities can be seen below:
</DIV><CENTER><FIGURE><FIGCAPTION><B>Figure 14</B>: Time between inoculations.
	</FIGCAPTION></FIGURE></CENTER><DIV class="container text">
Since the pipetting in the inoculation step follows a normal distribution and the r also fluctuates, we get slightly different results when running the program. That's why we take the time period and inoculation number at which the 95% mark was reached for all 3D plots as reference points.
For our three affinities the program predicts a minimal cultivation period of 8 hours and at least 11 inoculations. 
 </DIV><DIV class="container text_header"><H3>References</H3></DIV><DIV class="container text"><UL><LI>Rosenkrantz, R. D. (Hg.) (1983): E. T. Jaynes: Papers on Probability, Statistics and Statistical Physics. Dordrecht: Springer (Synthese Library, Studies in Epistemology, Logic, Methodology, and Philosophy of Science, 158). Online accessible at http://dx.doi.org/10.1007/978-94-009-6581-2.</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>