Team:Bielefeld-CeBiTec/Project/Library/Scaffolds
Toggle navigation
NavBar
PROJECT
Description
Library
Overview
Scaffolds
Design and Construction
Create your own Library
Mutation
Overview
Error prone Polymerase I
Genome wide mutator
Selection
Overview
Bacterial Two-Hybrid System
Alternative Selection System
Modeling
Overview
Model I
Model II
RESULTS
Overview
Library
Overview
Assembly
Phagemid Display
Sequencing
Mutation
Overview
Assembly
Reversion
Sequencing
Selection
Overview
Assembly
Expression Control
Interaction Control
Binding Control
in vivo Selection
Modeling
Fermentation
Parts
Best Basic Part
Best Composite Part
Best Part Collection
Improve a part
Proof of Concept
Demonstrate your Work
Medal Fulfillment
HUMAN PRACTICES
Overview
Engagement
Dictionary
Educational Outreach
Crowdfunding and Press
Entrepreneurship
Integrated Practices
Achievements
COLLABORATIONS
Overview
Düsseldorf
Lethbridge
Freiburg
NOTEBOOK
Overview
Lab Notebook
Library
Mutation
Selection
Primers
Lab Protocols
Media and Buffers
TEAM
Team Profile
Members
Bielefeld University
Contact
ACKNOWLEDGEMENT
Overview
Sponsors and Partners
Attributions
Library Project
Scaffolds
What is a Library?
Our project is based on the construction of an adequate library, which is defined as a collection of identical plasmids that only vary in the protein coding sequences (CDS). These CDS are designed to possess optimized and region specific randomized subregions displaying high variability. Upon transformation of the randomized plasmids into E. coli a heterogeneous culture is obtained with each colony carrying a different insert translating into different binding proteins ( Osoegawa et al., 2001 ).
One major advantage of our library is the availability of a wide range of different binding proteins as starting point for the directed evolution process. Completely new antibodies or variations of existing ones could serve as starting material.
By creating such a large variety, we are aiming for the highest chance for a possible binder. By adjusting the variable regions and their nucleic acid scheme we can specify the target further  but still have leverage to cover a wide range of possible targets in the beginning (see Library - Design and Construction ).
In order to identify suitable binding proteins for an appropriate scaffold of our library, we compared different types of antibodies and antibody mimetics from various sources. Antibody mimetics also show better accessibility to genetic engineering like construction of fusion proteins or incorporation of modifiable amino acids thus allowing the combination with fluorophores or drugs for targeted drug delivery ( Skerraet al., 2007 ). After deep and thorough investigation we decided to use Monobodies and Nanobodies. Although commonly used antibodies might be more robust, they are outperformed ten times by Mono- and Nanobodies in their smaller size (15 kDa vs. 150 kDa) and stability. This enables to penetrate depth, even in hard tissue ( Ahmadet al., 2012 ). Additionally, Nano- and Monobody binding proteins are rapidly cleared from the blood and have lower retention time in non-target tissues ( Ahmad et al., 2012 ).
Monobodies
Monobodies, also called adnectines, are synthetic antibody mimetics that originate from the tenth human fibronectin type 3 domain. Fibronectin is a rather large protein that is essential in the formation of the extracellular matrix and cell-cell interactions ( Koide and Koide, 2007 ). In 1998 the research group of Akiko Koide discovered that the fibronectin scaffold possesses similar binding properties like antibodies ( Koide et al., 1998 ). Since then continuous research lead to appliance in treatment of diseases like glioblastoma multiform ( Bloom et al., 2009 ) or chronic myelogenous leukaemia ( Grebien et al., 2011 ). The structure of Monobodies is similar to immunoglobulin domains ( Lipovsek et al., 2011 ). The original three dimensional structure of the scaffold is a β-sandwich but with seven β-strands instead of nine which are common in the variable domain of the heavy chain domain of regular antibodies and three surface loops on each end ( Koide and Koide, 2007 ).  They have a high stability and due to their small size, 10 kDa, and they penetrate deep into tissue ( Koide and Koide, 2007 ).  Another outstanding feature that they can be functionally expressed in the E. coli cytoplasm and thereby can be produced simple and at low cost in culture. This is due to the absence of disulphide bonds which are unlikely to form in the E. coli cytoplasm. This makes them compatible with virtually any display technologies ( Koide and Koide, 2007 ) and is therefore a suitable canditate for our in vivo system. By randomising the amino acid sequence of the CDR-like loops an initial library is created. The extensive loop sequence variation is tolerated by the structure so it stays stable ( Batori et al., 2002 ). Thus allowing binding of the target with high affinity and selectivity ( Skerra et al., 2007 ).
Fig. 1: Protein structure of an Monobody. Constant regions colored in green, variable regions colored in blue.
Nanobodies
Nanobodies are single domain antibodies that originate from camelids like Alpaca and Camel or cartilaginous fish like sharks ( Stanfield et al., 2004 ; van der Linden et al., 1999 .
The nanobody family consist of conventional heterotetrameric antibodies with low-affinity binders as well as unique functional heavy (H)-chain antibodies (HCAbs) with high affinity binders ( Harmsen et al., 2007 . Because a single polypeptide chain is expressed, they are better suited for phage display than Fragment antigen-binding (Fab) fragments ( Holt et al., 2003 ). The H chain of these homodimeric antibodies consists of one antigen-binding domain, the VHH, and two constant domains. HCAbs fail to incorporate light (L) chains owing to the deletion of the first constant domain and a reshaped surface at the VHH side, which normally associates with L chains in conventional antibodies ( Muyldermans et al., 2003 ).
Therefore, the production takes place in liquid cell culture and thus production in large quantities is simple with higher yields than Fab fragments ( Holt et al., 2003 ). They consist of a single monomeric variable antibody domain and a universal Nanobody-framework (in our case cAbBCII10). With a size of 12-15 kDa they are slightly bigger than Monobodies but still able to penetrate hard tissues like solid tumours ( Holt et al., 2008 ). Research lead to various applications in the therapeutically field. For example, a humanised scFv is in phase II testing to reduce mortality and myocardial infarction in patients undergoing graft surgery for coronary artery bypass ( Saerenset al., 2003 ). Also therapeutically application against the influenza A H5N1 virus has been shown in mice where it reduced the replication rate significantly.  Even in oral immunotherapy Nanobodies are used ( Harmsen et al., 2007 ). Also many diagnostically applications were made possible like super-resolution microscopy using a Nanobody tagged with GFP or tagging by binding proteins, enzymes or antigens with such a complex (Rothbauer U. et al, 2007). They can also be modified to work as a biosensor ( Saerenset al., 2003 ).
A high expression rate was shown in E. coli and they are still functional and stable even in absence of conserved disulphide bonds ( Saerenset al., 2003 )  which are unlikely to form in the E. coli cytoplasm. Their high tolerance against acidic conditions, temperature compared to mouse monoclonal antibodies ( van der Linden et al., 1999 ), high solubility (Arbabi Ghahroudi M. et al, 1997)  and very few cleavage sites for enzymes allow new areas of application ( Harmsen et al., 2007 ) . The Nanobody loop structure can bind to antigen structures inaccessible to human antibodies, for example catalytic centres in enzymes. Also fusion proteins and complexes can be created leaving both parts still active. Example are a complex of a shark Nanobody and lysozyme ( Stanfield et al., 2004 ) or a Nanobody conjugated branched Gold nanoparticle for photothermal therapy against tumour cells ( van de Broeket al., 2011 ).
Fig. 2: Protein structure of an Nanobody. Constant regions colored in green, variable regions colored in blue.
Design and Construction
References
Ahmad, Zuhaida Asra; Yeap, Swee Keong; Ali, Abdul Manaf; Ho, Wan Yong; Alitheen, Noorjahan Banu Mohamed; Hamid, Muhajir (2012): scFv antibody: principles and clinical application. In: Clinical & developmental immunology 2012, S. 980250. DOI: 10.1155/2012/980250.
Batori, V.; Koide, A.; Koide, S. (2002): Exploring the potential of the monobody scaffold. Effects of loop elongation on the stability of a fibronectin type III domain. In: Protein Engineering Design and Selection 15 (12), S. 1015–1020. DOI: 10.1093/protein/15.12.1015.
Bloom, Laird; Calabro, Valerie (2009): FN3: a new protein scaffold reaches the clinic. In: Drug discovery today 14 (19-20), S. 949–955. DOI: 10.1016/j.drudis.2009.06.007.
Grebien, Florian; Hantschel, Oliver; Wojcik, John; Kaupe, Ines; Kovacic, Boris; Wyrzucki, Arkadiusz M. et al. (2011): Targeting the SH2-kinase interface in Bcr-Abl inhibits leukemogenesis. In: Cell 147 (2), S. 306–319. DOI: 10.1016/j.cell.2011.08.046.
Harmsen, M. M.; Haard, H. J. de (2007): Properties, production, and applications of camelid single-domain antibody fragments. In: Applied microbiology and biotechnology 77 (1), S. 13–22. DOI: 10.1007/s00253-007-1142-2.
Holt, Lucy J.; Herring, Chris; Jespers, Laurent S.; Woolven, Benjamin P.; Tomlinson, Ian M. (2003): Domain antibodies: proteins for therapy. In: Trends in biotechnology 21 (11), S. 484–490. DOI: 10.1016/j.tibtech.2003.08.007.
Koide, A.; Bailey, C. W.; Huang, X.; Koide, S. (1998): The fibronectin type III domain as a scaffold for novel binding proteins. In: Journal of Molecular Biology 284 (4), S. 1141–1151. DOI: 10.1006/jmbi.1998.2238.
Koide, Akiko; Koide, Shohei (2007): Monobodies: antibody mimics based on the scaffold of the fibronectin type III domain. In: Methods in molecular biology (Clifton, N.J.) 352, S. 95–109. DOI: 10.1385/1-59745-187-8:95.
Lipovsek, D. (2011): Adnectins: engineered target-binding protein therapeutics. In: Protein engineering, design & selection : PEDS 24 (1-2), S. 3–9. DOI: 10.1093/protein/gzq097.
Muyldermans, Serge (2013): Nanobodies: natural single-domain antibodies. In: Annual review of biochemistry 82, S. 775–797. DOI: 10.1146/annurev-biochem-063011-092449.
Osoegawa, K.; Jong, P. J. de; Frengen, E.; Ioannou, P. A. (2001): Construction of bacterial artificial chromosome (BAC/PAC) libraries. In: Current protocols in molecular biology Chapter 5, Unit 5.9. DOI: 10.1002/0471142727.mb0509s55.
Skerra, Arne (2007): Alternative non-antibody scaffolds for molecular recognition. In: Current opinion in biotechnology 18 (4), S. 295–304. DOI: 10.1016/j.copbio.2007.04.010.
Stanfield, Robyn L.; Dooley, Helen; Flajnik, Martin F.; Wilson, Ian A. (2004): Crystal structure of a shark single-domain antibody V region in complex with lysozyme. In: Science (New York, N.Y.) 305 (5691), S. 1770–1773. DOI: 10.1126/science.1101148.
van der Linden, R.H.J.; Frenken, L.G.J.; Geus, B. de; Harmsen, M. M.; Ruuls, R. C.; Stok, W. et al. (1999): Comparison of physical chemical properties of llama VHH antibody fragments and mouse monoclonal antibodies. In: Biochimica et Biophysica Acta (BBA) - Protein Structure and Molecular Enzymology 1431 (1), S. 37–46. DOI: 10.1016/S0167-4838(99)00030-8.
