<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Fermentation skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Fermentation</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container main"><DIV id="header"><NAV id="navigationbar" class="navbar navbar-default custom_navbar_style navbar-fixed-top"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="#">NavBar</A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-center"><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description"> PROJECT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description">Description</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds">Scaffolds</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design">Design and Construction</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/CreateYours">Create your own Library</A></LI></UL><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/EpPolI">Error prone PolymeraseI</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/Global">Genome wide mutator</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">Bacterial Two-Hybrid System</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/SplitBetaLactamaseHitchHiker">Alternative Selection System</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_I">Model I</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_II">Model II</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results"> RESULTS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Phage">Phagemid Display</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">Reversion</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/Cloning">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/ExpressionControl">Expression Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/InteractionControl">Interaction Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/BindingControl">Binding Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/KnockOutKnochIn"><I>in vivo</I> Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Modeling">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Fermentation">Fermentation</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Best Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Best Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Best Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts/Improve">Improve a part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Demonstrate your Work</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Medal_Fulfillment">Medal Fulfillment</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold"> HUMAN PRACTICES </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Dictionary">Dictionary</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks">Educational Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Crowdfunding">Crowdfunding and Press</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Achievements</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations"> COLLABORATIONS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Duesseldorf">DÃ¼sseldorf</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Lethbridge">Lethbridge</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Freiburg">Freiburg</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments"> NOTEBOOK </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook">Lab Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">Primers</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols">Lab Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Media">Media and Buffers</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team"> TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team">Team Profile</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Members">Members</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Contact">Contact</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement"> ACKNOWLEDGEMENT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Sponsors_And_Partners">Sponsors and Partners</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></DIV></DIV></NAV></DIV></DIV><DIV class="container main"><DIV class="jumbotron" style="background-image: url(https://static.igem.org/mediawiki/2016/0/02/Bielefeld_CeBiTec_2016_10_14_X_projectheader.png
)"><DIV class="jumbotron-text"><H1 style="margin-bottom: 0px; text-align:left">Fermentation</H1><H2 style="color:#ffffff; text-align:left">Culture drives great results. - Jack Welch</H2></DIV></DIV><DIV class="container text_header"><H3>Introduction</H3></DIV><DIV class="container text">
        One of the fundamental aspects when establishing a directed evolution approach is to grant an advantage to the individuals with favorable
        properties. In our setup we coupled correct binding properties of our Evobody proteins to a growth advantage in the form of a gradual
        ampicillin resistance. To determine the applicability of this resistance and how selective pressure can be applied, we did several fermentations
        to characterize the used strain. 
</DIV><DIV class="container text_header"><H3>Shaker flask cultivation with varying antibiotic concentrations</H3></DIV><DIV class="container text">
        When communicating with industry associated experts, one question we were constantly asked was:&quot;Why do you use ampicillin? Is ampicillin an adequate
        selection marker?&quot; We thought about the possibility to use other antibiotics as our resistance marker, but as a huge part of our selective system was based on the beta-lactamase, we were hesitating to change our system.
		So, to determine whether or not ampicillin was suitable for our projects goals, we designed a series of experiments of cultivations using 
        different setups regarding 
        resistancy and a broad range of different ampicillin concentrations. The strain which we used was meant to immitate our final construct. 
        That is why we transformed <I>E. coli</I> JS200 with pHSG-EPPolI and pSB1AK3. We cultivated 25 ml LB in 250 ml shaker flasks at 37Â°C and 150 RPM.
        Inoculation was done to gain an initial optical density of 0.1. The ampicillin concentration was varied between 0 g/l and 10 g/l.
        Additionally kanamycin and chloramphenicol were added at adequate concentrations to grant plasmid stability. All Experiments were also
        done with a JS200 strain containing the pHSG-WTPolI plasmid to determine possible differences when cultivated under ampicillin.
        <FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 1: Growth curve of JS200 containing pWTPolI and pSB1AK3.</B>Growth curve of JS200 strains containing pHSG-pWTPolI and pSB1AK3
  The cells were cultivated between 10-16 h in LB Medium. The medium was supplemented with adequate concentrations of kanamycin, chloramphenicol and 0, 0.1, 1.0, 3.0, 5.0 and 10.0 g/l ampicillin.</FIGCAPTION></FIGURE><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 2: Growth curve of JS200 containing pEPPolI and pSB1AK3.</B>Growth curve of JS200 strains containing an error prone DNA polymerase I pHSG  and pSB1AK3
  The cells were cultivated between 12-18 h in LB Medium. The medium was supplemented with adequate concentrations of kanamycin, chloramphenicol and 0, 0.1, 1.0, 3.0, 5.0 and 10.0 g/l ampicillin. </FIGCAPTION></FIGURE>
        As you can see in Figures 1 and 2 higher ampicillin concentrations have a major impact on the growth properties of the strain containing the
        error prone DNA polymerase I. However the strain containing the plasmid carrying the wildtype DNA polymerase I is almost not influenced by
        a raised ampicillin concentration. The differences in ampicillin sensibility may be caused by the low fidelity property of the <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2082106">error prone 
        polymerase I</A>. The different activities directly influence the ColE1-like plasmid pSB1AK3 and connected to this, the beta-lactamase presence in culture.
        Based on this results, we decided that ampicillin is indeed an adequat selective marker for our system, as long as we keep the quantitative aspect of the beta-lactamase expression as low as possible. 
		To achieve this, we integrated our reporter construct into the genome of the <I>E. coli</I> strain we were using. 
		As it can be seen in Fig. 1 and Fig. 2, the lower the plasmid copy number, the more influence has ampicillin.
		
		It can be seen in Figure 2 that a higher ampicillin concentration is directly associated to the elongation of the lag phase of the respective culture. 
		As our selection construct is meant to express beta-lactamase corellating with the binding properties of our Evobodies, 
                we switched to different fermentation methods to change ampicillin concentration over time.
                <UL><LI>
                        Continous cultivation with a secondary feed in a turbidostat
        </LI><LI>
                        Repeated batch fermentation with different ampicillin concentrations per cycle
                    </LI></UL>
		However, as our selection system directly correlates to the expression of the beta-lactamase, we can assume that a selective advantage through higher beta-lactamase expression would have the same impact as a lower ampicillin concentration.
		But what does this mean in context of the ability of ampicillin to work as a selection marker? 
        
        <DIV class="container text_header"><H3>Usage of a low budget turbidostat to increase selective pressure</H3></DIV><DIV class="container text">
            To further characterize the mutation system used in our evolutionary process, we used a low budget turbidostat provided by the departement of molecular 
            biotechnology of the University of Potsdam. After doing reversion tests in shaker flasks we took the leap to the turbidostat and applied selective pressure
            in form of antibiotics. By doing a continuous fermentation while constantly increasing the ampicillin concentration in the Medium
            feed, we were able to optimize the revertant frequency in culture. Inoculation was done with <I>E. coli</I> JS200 <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2082106">pEPPolI</A> and freshly transformed 
            <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2082151">earlyStop beta-lactamase</A>. 
            
            The complete setup can be seen in figure 3. After autoclaving for sterilization the antibiotics were added under sterile conditions. To
            increase the selective pressure a  second feed bottle was connected to the primary feed and 2 g/l Amp LB has been pumped with a rate of about
            18 ml/h. The primary feed was adjusted to keep the OD in the bioreactor at 0.5. To sustain a minimum of thermal degradation the 2 g/l Amp LB was stored
            on ice for the duration of cultivation. A clean inoculation was done through a septum in the fermenter lid. To provide constant conditions the whole fermenter 
            was placed in a 37Â°C incubator and 10x Amp LB was pumped through an opening in the incubator seal.
            <FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 3: Turbidostat overview.</B> Setup of continuous fermentation using a turbidostat: 1. Waste bottle 
      2. Fermenter and computational unit containing stirrer, temperature sensor, optical density sensor, 20 ml reactor vial and arduino based controller. 3. Power supply 
  and pump unit, containing air and feed pump. 4. Primary feed bottle. 5. Secondary feed pump. 6. Laptop with user interface. 7. Chilled 2 g/l Amp LB reservoir.</FIGCAPTION></FIGURE>
            
            As you can see in figure 4
            the culture reached the target OD within 7 h and dropped rapidly when the medium feed diluted the cell suspension. However, the population regenerated quickly and reached the target 
            OD of 0.5 after about 18 h. After fermentation the population was screened for revertants to calculate reversion frequency. The results of these experiments can be read <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">here</A>.
            <FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 4: Growth curve of <I>E. coli</I> JS200 containing <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2082106">pEPPolI</A> and <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2082151">earlyStop beta-lactamase</A>.</B>Growth curve of 
  <I>E. coli</I> JS200 containing <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2082106">pEPPolI</A> and <A href="BBa_K2082151">earlyStop beta-lactamase</A>. Cultivation was done at 37Â°C for 18 h and target OD was set
  to 0.5.</FIGCAPTION></FIGURE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>