<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Results_Selection_KnockOutKnochIn skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Results/Selection/KnockOutKnochIn</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container main"><DIV id="header"><NAV id="navigationbar" class="navbar navbar-default custom_navbar_style navbar-fixed-top"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="#">NavBar</A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-center"><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description"> PROJECT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description">Description</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds">Scaffolds</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design">Design and Construction</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/CreateYours">Create your own Library</A></LI></UL><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/EpPolI">Error prone PolymeraseI</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/Global">Genome wide mutator</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">Bacterial Two-Hybrid System</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/SplitBetaLactamaseHitchHiker">Alternative Selection System</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_I">Model I</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_II">Model II</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results"> RESULTS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Phage">Phagemid Display</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">Reversion</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/Cloning">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/ExpressionControl">Expression Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/InteractionControl">Interaction Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/BindingControl">Binding Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/KnockOutKnochIn"><I>in vivo</I> Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Modeling">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Fermentation">Fermentation</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Best Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Best Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Best Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts/Improve">Improve a part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Demonstrate your Work</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Medal_Fulfillment">Medal Fulfillment</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold"> HUMAN PRACTICES </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Dictionary">Dictionary</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks">Educational Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Crowdfunding">Crowdfunding and Press</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Achievements</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations"> COLLABORATIONS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Duesseldorf">DÃ¼sseldorf</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Lethbridge">Lethbridge</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Freiburg">Freiburg</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments"> NOTEBOOK </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook">Lab Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">Primers</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols">Lab Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Media">Media and Buffers</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team"> TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team">Team Profile</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Members">Members</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Contact">Contact</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement"> ACKNOWLEDGEMENT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Sponsors_And_Partners">Sponsors and Partners</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></DIV></DIV></NAV></DIV></DIV><DIV class="container main"><DIV class="container text_header"><H1><I>In vivo</I> selection</H1></DIV><DIV class="container text">
		Strong evidence was provided, that both created fusion proteins of our bacterial two-hybrid system are <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/ExpressionControl">expressed</A> in the cell. Moreover, the positive control HA4 and SH2 are <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/InteractionControl">interacting</A> with each other and the chosen DNA 
		binding domain cI of the phage 434 is <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/BindingControl">binding</A> at the specific binding site OR1 upstream of the promoter. The parts <A href="http://parts.igem.org/Part:BBa_K2082221">BBa_K2082221</A> with the functional HA4-rpoZ fusion protein, <A href="http://parts.igem.org/Part:BBa_K2082231">BBa_K2082231</A> with the functional SH2-cI fusion protein and 
		the reporter of the bacterial two-hybrid system had to be combined in one <I>E. coli</I> cell. However, the created HA4 mutations and the Gal4-RpoZ (<A href="http://parts.igem.org/Part:BBa_K2082226">BBa_K2082226</A>) as the negative control were tested together with BBa_K2082231. Even after a successful transformation with both plasmids no visible difference between the five approaches was observed.
		Moreover, exchanging the binding site by the binding site of the lambda cI did not produce new results.
		
		The use of the same pUC19-derived pMB1 origin of replication within pSB1C3 and pSB1K3 could cause problems, because<I> E. coli</I> is not able to control the amount of plasmid in the cell if both plasmids have the same ori. The next step was the exchang of the backbone of the part with SH2-cI and the reporter from pSB1K3 to <A href="http://parts.igem.org/Part:pSB4C5">pSB4C5</A>. The pSB4C5 vector uses the pSC101 origin of replication.
		For different antibiotic resistances the coding sequence of the other fusion protein HA4-RpoZ was cloned in the pSB1K3 vector. To test the efficiency of the new constructs, both were transformed in the <I>E. coli</I> strain KRX together. A comparison of a culture with cells carrying both plasmids and a culture with only the pSB4C5 plasmid containing the reporter and the SH2-cI fusion protein reveals visible differences in the RFP fluorescence intensity. The culture with both plasmids has a higher 
		visual red color. To validate the results, the two cultures were measured in the <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols#tecan">Tecan</A> plate reader (figure 1).
<FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 1: Tecan-results <I>in vivo</I> reporter activity.</B> Two culturs of <I>E. coli</I> were measured on their ability to produce RFP through the reporter construct. A culture with one fusion protein SH2-cI expressing is compared with a culture carrying both fusion proteins SH2-cI and HA4-RpoZ. The RFP intensity of the culture with both fusion proteins is significantly higher than in the culture with only one fusion protein.</FIGCAPTION></FIGURE>
Like expected, the two measured samples differ from each other. A comparison of the RFP intensity demonstrates an about 48%  higher RFP signal if the bacteria carrying both fusion proteins SH2-cI and HA4-RpoZ except only the SH2-cI. To calculate if the difference can be 
		pointed out as significant, an unpaired two sample t-test was applied. With a  calculated t-value of 4.62 and the corresponding p-value of 0.0099 the possibility, that the difference between these two fluorescence signals are caused by chance is lower than 1%.  
		Due to the fact that the results are very significant the evidence is given, that <I>in vivo</I> selection with our designed bacterial two-hybrid system is possible.
		
		Although it is shown that the bacterial two-hybrid system is working, the output signal is lower than expected. Two possibilities could increase the transcription activity of the bacterial two-hybrid selection system. The first opportunity is the reduction of the background activity of the promoter. The used promoter <A href="http://parts.igem.org/Part:BBa_K2082210">BBa_K2082210</A> is a little bit leaky. Therefore, without any interaction of the two fusion proteins the gene is a little bit expressed.
		In the measurement the reporter was brought in the pSB4C5 plasmid. For the plasmid itself a very high amount of DNA was measured after every plasmid isolation. Therefore, we expect it to be a high copy plasmid. Although the promoter is only a little bit leaky, the high amount of plasmid leads to a high number of the reporter gene transcripts. And if every reporter is a little bit leaky, the sole number of the reporter causes a high signal of RFP. This is seen in the figure X,  considering that the RFP signal is extremly high, without any transcription activation through the two fusion proteins.
		The second possibility for the low activation rate of the selection system is the native <I>rpoZ</I> gene in the <I>E. coli</I> cell. RpoZ is a subunit of the RNA polymerase I core complex. Under normal conditions the RpoZ protein would be guided by the betaÂ´-subunit to the core complex of the polymerase as the last unit.  If <I>E. coli</I> produces a lot of native RpoZ it competes with the RpoZ coupled with the HA4 Monobody. Therefore, it is possible, that most of the most polymerases build their core with the native RpoZ instead the RpoZ-HA4 fusion protein. In this complex it is not able to be recruited by the second 
		fusion protein and the transcription activation of the reporter stay off. 
		
		An interesting fact is, that the RNA polymerase do not necessarily needs the RpoZ protein of their functionality (Gosh et al., 2001; Mathew et al., 2005). Therefore, the next step for the bacterial two-hybrid system is a knock-out of the native <I>rpoZ</I> gene combined with a knock-in of the beta-lactamase containing reporter(<A href="http://parts.igem.org/Part:BBa_K2082238">BBa_K2082238</A>) in the genome of our working strain JS200 of <I>E. coli</I>. With this attempt it is possible to decrease the background activity of the reporter by only one copy of the reporter left in the cell and to eliminate the expression of the native RpoZ
		and preventing the competiton of the native RpoZ with the RpoZ-HA4 fusion protein in one step. 
		<FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 2: Illustration of the CRISPR/Cas-method for a knock-out/knock-in experiment.</B> At first the Guide-RNA is bound at the Cas-Protein. Then the guide-RNA guides the Cas-protein to the specific point at the DNA, in our case the <I>rpoZ</I> gene, and make a cut (<B>A</B>). A designed DNA fragment containing the knock-in sequence flanked by two regions directly upstream and downstream of the knock-out gene is moving to the broken DNA sequence(<B>B</B>). Through homologous recombination the new DNA fragment interacts with the flanked regions of the knock-out gene (<B>C</B>) and replaced it completely (<B>D</B>).</FIGCAPTION>
The realization of such an experiment was made with the two plasmid CRISPR/Cas-system. The first plasmid encodes the Cas protein and the second the small guide RNA sequence. If the guide RNA is expressed, it can guide the Cas protein to one specific sequence on the DNA. Now the Cas protein is able to cut the DNA at this specific sequence. Under normal conditions the cell would recognize the broken genome and would begin with a degeneration process of the whole cell. However, adding the right linear DNA sequence in addition to both plasmids within the bacterium, a special repair mechanism 
		can cut out the broken sequence in the genome and add instead the added DNA sequence at the same place. In this experiment a DNA sequence containing the reporter was created, which is able to repair the genome after the Cas protein did a cut inside the <I>rpoZ</I> gene. Therefore the bacterium needs to cut out the <I>rpoZ</I> gene completely and has to integrate the designed DNA sequence to be alive.
		Based on the beta-lactamase gene, which was implemented by the knock-in of the reporter, the cultures were plated on LB agar with low concentration of ampicillin on it. Some of the growing cultures were picked for a colony PCR validation (figure 2).
		
		Some bands on the gel are 2,300 to 2,400 bp in size. This is the expected length of the genomic sequence of the flanked <I>rpoZ</I> gene. Therefore, these results are negative. However, five bands on the gel are longer with about 3000 bp. With a length of about 1000 bp of the reporter and each time 1000 base-pairs of the two flanked fragments of the designed DNA construct, the expected length for a successful integration of the reporter gene in the genome was given. 
		Therefore, the knock-out, knock-in experiment was successful and the reporter was perfectly integrated into the JS200 genome.
<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 3: Results of the genomic cPCR of the possible knock-out/knock-in strains.</B> For this gel 13 colonies of the ampicillin containing agar plates were picked. The expected band of the knock-out/knock-in strain was about 3000 bp in size. With the 1 kb ladder as the length standard, it is seen that the colonies 2, 6, 7, 9 and 13 have the expected result.</FIGCAPTION></FIGURE></CENTER>
		Although the integration was successful, it was not possible to test the advantages of this modification <I>in vivo</I>, due to the time limitations. The different HA4 mutants could not be tested, because the cloning from pSB1C3 to pSB1K3 took long. Unfortunately the final measurement of the system with the mutants could not be completed.
		Nevertheless, we can proudly report the, that we designed a functional bacterial two-hybrid system and we paved the way for upcoming experiments with the system.
		</FIGURE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>