<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Project_Mutation_EpPolI skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Project/Mutation/EpPolI</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container main"><DIV id="header"><NAV id="navigationbar" class="navbar navbar-default custom_navbar_style navbar-fixed-top"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="#">NavBar</A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-center"><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description"> PROJECT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description">Description</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds">Scaffolds</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design">Design and Construction</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/CreateYours">Create your own Library</A></LI></UL><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/EpPolI">Error prone PolymeraseI</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/Global">Genome wide mutator</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">Bacterial Two-Hybrid System</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/SplitBetaLactamaseHitchHiker">Alternative Selection System</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_I">Model I</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_II">Model II</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results"> RESULTS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Phage">Phagemid Display</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">Reversion</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/Cloning">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/ExpressionControl">Expression Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/InteractionControl">Interaction Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/BindingControl">Binding Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/KnockOutKnochIn"><I>in vivo</I> Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Modeling">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Fermentation">Fermentation</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Best Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Best Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Best Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts/Improve">Improve a part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Demonstrate your Work</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Medal_Fulfillment">Medal Fulfillment</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold"> HUMAN PRACTICES </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Dictionary">Dictionary</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks">Educational Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Crowdfunding">Crowdfunding and Press</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Achievements</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations"> COLLABORATIONS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Duesseldorf">Düsseldorf</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Lethbridge">Lethbridge</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Freiburg">Freiburg</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments"> NOTEBOOK </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook">Lab Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">Primers</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols">Lab Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Media">Media and Buffers</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team"> TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team">Team Profile</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Members">Members</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Contact">Contact</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement"> ACKNOWLEDGEMENT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Sponsors_And_Partners">Sponsors and Partners</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></DIV></DIV></NAV></DIV></DIV><DIV class="container main"><DIV class="container text_header"><H1>Error Prone DNA PolymeraseI</H1></DIV><DIV class="container text">
The mechanism of DNA replication takes place in the cell nucleus. The polymerases are responsible for the replication of the entire DNA. One of these polymerases is the DNA polymeraseI, which plays a role in lagging-strand replication of chromosomal DNA. Moreover, the DNA polymeraseI also has the task to do proofreading and plays a role in replicating ColE1 plasmids (Camps et. al 2003). ColE1 plasmids are characterized by the ColE1 origin of replication (ori). During the lagging-strand synthesis the DNA polymerase processes RNA primer (˜20 nt) and fills gaps during DNA repair reactions (Allen et. al 2011). Sometimes errors occur during the synthesis of a DNA strand. These mistakes may have big effects. They can lead to diseases or death of the organism, when a gene is damaged that is essential for survival. To reduce the frequency of mistakes the DNA polymeraseI checks its work by using its own proofreading functionality.
</DIV><DIV class="container text_header"><H2>EP-PolI</H2></DIV><DIV class="container text_header"><H3>Structure of EP-PolI</H3></DIV><DIV class="container text">
To produce more diversity in our library we use an <I>in vivo</I> mutagenesis system, which is based on an error prone DNA polymeraseI (EP-PolI) (Troll et. al 2011). Thus, the mutagenesis system generates mutations during replication. It is designed as a two plasmid system and is beneficial to our approach due to the selectivity towards a specific plasmid. Only the ColE1 plasmid encoding the target protein sequence is mutated. 
<A href="http://www.metx.ucsc.edu/research/camps.html
"> Prof. Dr. Manel Camps</A> from the University of Santa Cruz investigated a DNA polymeraseI, called EP-PolI, which lacks in correct synthesis and in proof-reading (Camps et. al 2003). In contrast to the normal DNA polymeraseI the EP-PolI has three point mutations, I709N, A759R and D424A (Camps et. al 2003)(figure 1). The I709N mutation is located in motif A. This is a conserved sequence in the palm domain of the polymerase active site. The mutations lead to an enlargement of the substrate-binding pocket, which is a possible explanation for the increased error rate (Camps et. al 2003). The D424A mutation in the exonuclease domain leads to deactivation of the proofreading activity of the EP-PolI. The amino acid replacements A759R is located in the O-helix, which is a conserved sequence (motif B) that is located close to the polymerase active site on dNTP binding. This may stabilize the enzyme with similar conformations, which leads to missed integration of nucleotides (Camps et. al 2003). 

<FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 1</B>: Structures of the klenow fragment of DNA polymeraseI with the three substitutions. The blue one is I709N, red is A759R and orange is D424A (Beese et. al 2011).</FIGCAPTION></FIGURE></DIV><DIV class="container text_header"><H3> Mutation </H3></DIV><DIV class="container text">
According to Allen et al. 2011, the EP-PolI produces evenly distributed mutations and generates base pair substitutions (transitions) and transversions. Another advantage of the EP-PolI is that it only replicates and thus mutates plasmids that have a ColE1 ori and that the EP-PolI processes Okazaki fragments (Allen et. al 2011). Therefore, no significant increase in the mutation rate of the chromosomal DNA was observed (Troll et. al 2014). During replication, the EP-PolI is replaced by DNA-PolymeraseIII. This leads to the conclusion that with increasing distance from the replacement point the frequency of EP-PolI mutations decrease (Allen et. al 2011). In this paper, they also wrote that the EP-PolI synthesizes 400 nt to 500 nt after the ori, but during a skype conversation with Manel Camps, he said that this is not the case. It seems that the mutations are randomly distributed over the plasmid and that only a few base pairs at the start are synthesized by DNA-PolymeraseI. According to Alexander et. al (2014) the EP-PolI generates more than one mutation per kb. The substitution of the four different bases has various probabilities (figure 1). 
</DIV><CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 2</B>: Base substition. The substitution frequencies for all bases made by the EP-PolI are shown. Figure adapted from (Badran et. al 2015).
                    </FIGCAPTION></FIGURE></CENTER><DIV class="container text_header"><H3> EP-PolI system</H3></DIV><DIV class="container text">
For an efficient use of the EP-PolI it is recommended to use the E. coli JS200 strain, that carries a knockout of the native PolI as well as a temperature sensitive DNA polymeraseI, which can compensate the knockout at a certain temperature (Camps et. al 2003). With the use of JS200 it is possible to switch between EP-PolI (mutation phase) and DNA polymeraseI (no mutation phase). At 30°C the DNA polymeraseI is active and synthesizes the plasmids, because it is more efficient and faster than the low fidelity EP-PolI (Camps et. al 2003). When changing to 37°C the DNA polymeraseI is no longer active and the EP-PolI begins to mutate the DNA sequence (Alexander et. al 2014). 

To effectively occupy the EP-PolI we use a two-plasmid system developed by Manel Camps (figure 3). One plasmid has a pSC101 ori (Pol I-independent) and the EP-PolI sequence and the other plasmid has the ColE1 ori and our Evobody sequence. Therefore, only the plasmid with the Evobody sequence will be mutated. This is a directed evolution, because only the target DNA sequence and not all the available DNA is mutated (Alexander et. al 2014). By applying this system to the JS200 strain we are able to switch between a mutation and a no-mutation phase. 
</DIV><CENTER><FIGURE class="figure"><CENTER><FIGCAPTION class="figure-caption"><B>Figure 3</B>: The two plasmid system. The EP-Pol I is expressed from the plasmid with the pSB101 ori and this replicate parts of the plasmid with the ColE1 plasmid.
                    </FIGCAPTION></CENTER></FIGURE></CENTER><CENTER><BUTTON><A class="button_link" href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/Global" role="button">Global mutagenesis</A></BUTTON></CENTER><DIV class="container text_header"><H3>References</H3></DIV><DIV class="container text"><UL><LI>
Alexander, David L.; Lilly, Joshua; Hernandez, Jaime; Romsdahl, Jillian; Troll, Christopher J.; Camps, Manel (2014): Random mutagenesis by error-prone pol plasmid replication in Escherichia coli. In: Methods in molecular biology (Clifton, N.J.) 1179, S. 31–44. DOI: 10.1007/978-1-4939-1053-3_3.</LI><LI>Allen, Jennifer M.; Simcha, David M.; Ericson, Nolan G.; Alexander, David L.; Marquette, Jacob T.; van Biber, Benjamin P. et al. (2011): Roles of DNA polymerase I in leading and lagging-strand replication defined by a high-resolution mutation footprint of ColE1 plasmid replication. In: Nucleic acids research 39 (16), S. 7020–7033. DOI: 10.1093/nar/gkr157..</LI><LI>Badran AH, 2015. Development of potent in vivo mutagenesis plasmids with broad mutational spectra. <I>Nature Communications, 2015, 6, 8425</I>.</LI><LI> Camps, Manel; Naukkarinen, Jussi; Johnson, Ben P.; Loeb, Lawrence A. (2003): Targeted gene evolution in Escherichia coli using a highly error-prone DNA polymerase I. In: Proceedings of the National Academy of Sciences of the United States of America 100 (17), S. 9727–9732. DOI: 10.1073/pnas.1333928100.</LI><LI>Troll, Chris; Alexander, David; Allen, Jennifer; Marquette, Jacob; Camps, Manel (2011): Mutagenesis and functional selection protocols for directed evolution of proteins in E. coli. In: Journal of visualized experiments : JoVE (49). DOI: 10.3791/2505.</LI><LI>Beese, L. S.; Friedman, J. M.; Steitz, T. A. (1993): Crystal structures of the Klenow fragment of DNA polymerase I complexed with deoxynucleoside triphosphate and pyrophosphate. In: Biochemistry 32 (51), S. 14095–14101. </LI><LI>Troll, Christopher; Yoder, Jordan; Alexander, David; Hernandez, Jaime; Loh, Yueling; Camps, Manel (2014): The mutagenic footprint of low-fidelity Pol I ColE1 plasmid replication in E. coli reveals an extensive interplay between Pol I and Pol III. In: Current genetics 60 (3), S. 123–134. DOI: 10.1007/s00294-013-0415-9.</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>