<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Project_Modeling_Model_I skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Project/Modeling/Model I</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container main"><DIV id="header"><NAV id="navigationbar" class="navbar navbar-default custom_navbar_style navbar-fixed-top"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="#">NavBar</A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-center"><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description"> PROJECT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description">Description</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds">Scaffolds</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design">Design and Construction</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/CreateYours">Create your own Library</A></LI></UL><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/EpPolI">Error prone PolymeraseI</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/Global">Genome wide mutator</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">Bacterial Two-Hybrid System</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/SplitBetaLactamaseHitchHiker">Alternative Selection System</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_I">Model I</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_II">Model II</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results"> RESULTS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Phage">Phagemid Display</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">Reversion</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/Cloning">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/ExpressionControl">Expression Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/InteractionControl">Interaction Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/BindingControl">Binding Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/KnockOutKnochIn"><I>in vivo</I> Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Modeling">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Fermentation">Fermentation</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Best Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Best Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Best Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts/Improve">Improve a part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Demonstrate your Work</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Medal_Fulfillment">Medal Fulfillment</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold"> HUMAN PRACTICES </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Dictionary">Dictionary</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks">Educational Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Crowdfunding">Crowdfunding and Press</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Achievements</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations"> COLLABORATIONS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Duesseldorf">Düsseldorf</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Lethbridge">Lethbridge</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Freiburg">Freiburg</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments"> NOTEBOOK </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook">Lab Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">Primers</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols">Lab Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Media">Media and Buffers</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team"> TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team">Team Profile</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Members">Members</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Contact">Contact</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement"> ACKNOWLEDGEMENT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Sponsors_And_Partners">Sponsors and Partners</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></DIV></DIV></NAV></DIV></DIV><DIV class="container main"><DIV class="jumbotron" style="background-image: url(https://static.igem.org/mediawiki/2016/8/89/Bielefeld_CeBiTec_2016_10_14_X_jessi2.png)"><DIV class="jumbotron-text"><H1 style="margin-bottom: 0px; text-align:left">ModelI</H1><H2 style="color:#ffffff; text-align:left">Selection system</H2></DIV></DIV><DIV class="container text">
ModelI describes the <I>in vivo</I><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">bacterial two hybrid selection system</A>:
 </DIV><CENTER><FIGURE><FIGCAPTION><B>Figure 1</B>: Selection pathway. 
	
	Plasmid 1 and plasmid 2 each produce one protein of the bacterial two hybrid system, respectively. When both proteins come together and bind on the DNA the reporter gene is transcribed, here beta-lactamase. The beta-lactamase then diffuses from the cytoplasm into the periplasm, where it encounters the antibiotic ampicillin and deactivates it. For detailed description see <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">bacterial two hybrid system</A>.
	</FIGCAPTION></FIGURE></CENTER><DIV class="container text">
 The selection pathway can be illustrated by the following reaction sequence:
 </DIV><CENTER><FIGURE><FIGCAPTION><B>Figure 2</B>: Reaction pathway of the selection system.
	
The 0 stands for degeneration of proteins or complexes and the + for the interaction of two proteins or complexes.
	</FIGCAPTION></FIGURE></CENTER><DIV class="container text">
 The reaction sequence starts with the expression of the Evobody-RpoZ translational fusion construct [(a)] on the one hand and the target protein-DNA binding domain translational fusion construct [(b)] on the other hand. 
 Both resulting fusion proteins are degraded over time [(c)]. 
 In the figure the degeneration is marked with a 0. 
 Furthermore, together with the DNA binding site for the DNA binding domain (DNABD) both fusion proteins form the bacterial two hybrid complex [(d)]. 
 When the complex is built the RNA polymerase gets recruited and transcribes the beta-lactamase. 
 Just like the fusion proteins the beta-lactamase will be degraded after a while [(e)]. 
 When the beta-lactamase and ampicillin meet in the periplasm the ampicillin will be deactivated and the cell survives.

Based on the reaction sequence the chemical reaction equations can be written down:

</DIV><DIV class="container text">
(I) <I>k<SUB>exprP1</SUB></I> stands for the expression rate of plasmid 1, which holds the Evobody-RpoZ sequence [<I>s</I><SUP>-1</SUP>].
k<SUB>degEvo</SUB> stands for the degeneration rate of the Evobody-RpoZ protein [<I>s</I><SUP>-1</SUP>].

(II) <I>k<SUB>exprP2</SUB></I> stands for the expression rate of plasmid 2, which holds the target-DNA binding domain (DNABD) protein [<I>s</I><SUP>-1</SUP>].
<I>k<SUB>degTarget</SUB></I> stands for the degeneration of the target-DNABD protein [<I>s</I><SUP>-1</SUP>].

(III) <I>k<SUB>ass</SUB></I> stands for the association rate of the bacterial two hybrid complex [(<I>s*M</I>)<SUP>-1</SUP>].
<I>k<SUB>diss</SUB></I> stands for the disassociation rate of the bacterial two hybrid complex [<I>s</I><SUP>-1</SUP>].

(IV) <I>k<SUB>exprbla</SUB></I> stands for the expression rate of beta-lactamase (bla) [<I>s</I><SUP>-1</SUP>].

(V) <I>k<SUB>transpbla</SUB></I> stands for the transport rate of beta-lactamase from cytoplasm to periplasm [<I>s</I><SUP>-1</SUP>].

(VI) <I>k<SUB>degbla</SUB></I> stands for the degeneration rate of the beta-lactamase [<I>s</I><SUP>-1</SUP>].

(VII) <I>k<SUB>transpamp</SUB></I> stands for the transport rate of ampicillin from medium to cytoplasm and back [<I>s</I><SUP>-1</SUP>].

(VIII) <I>k<SUB>diff</SUB></I> stands for the diffusion rate of beta-lactamase and ampicillin [(<I>s*M</I>)<SUP>-1</SUP>].
</DIV><DIV class="container text_header"><H3>Differential equations</H3></DIV><DIV class="container text">
Next the language of chemistry has to be translated into the language of mathematics. A system of ordinary differential equations is generated. As this system has to be solved numerically, we have chosen a notation the software package MATLAB® understands:
</DIV><DIV class="container text">
 	stand<SUB>expr</SUB> stands for standard expression [M].
	
[Evobody] or Evobody stands for Evobody-RpoZ protein [M].

[Target] or Target stands for target-DNABD protein [M].

[DNA] stands for DNA [M].

[amp<SUB>Peri</SUB>] stands for ampicillin in periplasm [M].
 
[amp<SUB>Medium</SUB>] stands for ampicillin in medium [M].

[bla<SUB>Cyt</SUB>] stands for beta-lactamase in cytoplasm [M].

[bla<SUB>Peri</SUB>] stands for beta-lactamase in periplasm [M].

[Evobody-Target] stands for Evobody-Target protein complex [M].

[Target-DNA] stands for Target-DNA complex [M].

[Evobody-Target-DNA] stands for Evobody-Target-DNA complex [M].
<I>k<SUB>dissEvoTarget</SUB></I> stands for disassociation rate of the Evobody-Target complex [<I>s</I><SUP>-1</SUP>].
<I>k<SUB>assEvoTarget</SUB></I> stand for association rate of the Evobody-Target complex [(<I>s*M</I>)<SUP>-1</SUP>].
<I>k<SUB>dissTargetDNA</SUB></I> stand for disassociation rate of the Evobody-Target complex [<I>s</I><SUP>-1</SUP>].
<I>k<SUB>assTargetDNA</SUB></I> stand for association rate of the Evobody-Target complex [(<I>s*M</I>)<SUP>-1</SUP>].
<I>k<SUB>degTarget</SUB></I> stands for degeneration rate of the target protein [<I>s</I><SUP>-1</SUP>].
<I>k<SUB>degEvo</SUB></I> stands for degeneration rate of the Evobodies [<I>s</I><SUP>-1</SUP>].
<I>k<SUB>exprblaETD</SUB></I> stands for the expression rate of beta-lactamase with the bacterial two hybrid complex [<I>s</I><SUP>-1</SUP>].
</DIV><DIV class="container text_header"><H3>Diffusion</H3></DIV><DIV class="container text">
The deactivation reaction of ampicillin through beta-lactamase is a diffusion-controlled reaction. An equation that considers the rate at which the reactants diffuse is needed (Atkins and De Paula 2010):
</DIV><CENTER><FIGURE><FIGCAPTION><B>Figure 3</B>: Diffusion of two proteins without consideration of an active centrum.
	</FIGCAPTION></FIGURE></CENTER><DIV class="container text">
This equation describes the general diffusion of two molecules (figure 3). 
But in our case the enzyme beta-lactamase only reacts with ampicillin, if its active centrum is in close proximity to ampicillin (figure 4 B). 
When beta-lactamase and ampicillin meet correctly, ampicillin is deactivated as shown in figure 4, section B to D. Section E indicates a wrong encounter of beta-lactamase and ampicillin.
That's why we decided to modify the equation according to (Bisswanger 2008):
</DIV><CENTER><FIGURE><FIGCAPTION><B>Figure 4</B>: Diffusion of two proteins with consideration of an active centrum.
	</FIGCAPTION></FIGURE></CENTER><DIV class="container text"><I>R<SUB>min</SUB></I> is defined as the distance in which the two reactant molecules, here ampicillin and beta-lactamase, react. This is the case, when they are in very close proximity. D is the sum of the diffusion coefficients of the two reactants in the solution. In our case the solution is the periplasm. The term diffusion describes the movement of a substance in a solution; the bigger the diffusion coefficient, the larger the diffusion velocity. 
The angle <I>α</I> restricts the reaction area of the enzyme to its active center (see opening in figure 4). So a reaction between beta-lactamase and ampicillin only takes place, when ampicillin meets the active center of beta-lactamase. 

Finally, the equation for the diffusion rate <I>k<SUB>diff</SUB></I> must be multiplied with the Avogadro constant <I>N<SUB>A</SUB></I> to gain the unit [(<I>s*M</I>)<SUP>-1</SUP>].
<I>R<SUB>min</SUB></I> was computated under the assumption that beta-lactamase and ampicillin react when they are in close vicinity and that the molecules are nearly spherical. Therefore, the minimal radius of a sphere that could contain the given mass of protein was estimated using the formula from (Erickson 2009):
</DIV><DIV class="container text">
for V in nm<SUP>3</SUP>, M in Dalton and R<SUB>min</SUB> in nanometer.

For beta-lactamase we obtained 2.0277 nm and for ampicillin 0.4649 nm.

To compute the diffusion coefficients we used the program WinHydroPro v1.00 PUB  [Ortega, 2011]. We received 2.852*10<SUP>-9</SUP> for beta-lactamase and 1.237*10<SUP>-8</SUP> for ampicillin.
The angle <I>α</I> was computed with the open-source program PyMOL (PyMOl 2013). <I>α</I> was determined to be 25°.

In the end we computed the reproduction rate in dependence of the ampicillin concentration in the periplasm, adapting the formula of Potsdam's:
 </DIV><DIV class="container text">
 where <I>n</I> is the ampicillin concentration in periplasm. The time point 20 minutes was chosen, because the division rate of an <I>E. coli</I> cell is around 20 minutes.
 </DIV><DIV class="container text_header"><H3>Experimental validation</H3></DIV><DIV class="container text">
A theoretically implemented model must always be validated with experimentally measured data.
Therefore, we designed and conducted an amount of appropriate experiments.
Firstly, we performed a beta-lactamase PCA colorimetric assay, and, secondly, a cultivation experiment. The cultivation experiment was used to measure the beta-lactamase concentration with the Nitrocefin assay over time. The experiments were essential to adapt model I.

</DIV><DIV class="container text_header"><H4>Beta-lactamase PCA colorimetric assay</H4></DIV><DIV class="container text">
To quantify the amount of active beta-lactamase in the periplasm of an <I>E. coli</I> bacterium and thus validate our mathematical program in this aspect, we performed an [osmotic shock] and a <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols#assays#Nitro_assay">beta-lactamase PCA colorimetric assay</A> (André Galarneau et. al 2002). The osmotic shock leads to the disruption of the periplasm and spreading of membrane proteins. To yield the maximum amount of active beta-lactamase an osmotic shock is used that does not use detergents that are harmful for the activity of beta-lactamase (Jalalirad 2013). With the beta-lactamase PCA colorimetric assay we determined the amount of beta-lactamase through the reduction of Nitrocefin. When Nitrocefin is split by beta-lactamase it changes its color from yellow to red like in figure 5. To measure how the beta-lactamase production changes over time we cultivated <I>E. coli</I> with a beta-lactamase plasmid for 6 hours and took probes every hour. Afterwards we did a lysis and a Nitrocefin assay. The results can be seen here:
</DIV><CENTER><FIGURE><FIGCAPTION><B>Figure 5</B>: Nitrocefin assay.
	
	From left to right: water control and time points t0 to t6 of a <I>E. coli</I> cultivation. 
During a cultivation period samples were taken from the culture every hour (t0 to t6). Cells in this sample were lysed, and measurements with a Nitrocefin assay were conducted.
	</FIGCAPTION></FIGURE></CENTER><CENTER><FIGURE><FIGCAPTION><B>Figure 6</B>: Growth curve of a culture, which carries a beta-lactamase plasmid.
	</FIGCAPTION></FIGURE></CENTER><DIV class="container text">
In figure 5 it can be seen that the beta-lactamase concentration rises with proceeding cultivation. Also we measured the beta-lactamase concentration of the whole culture at every time point and computed with the use of the OD (figure 6) the beta-lactamase concentration for one cell. 1 OD is converted to 8*10<SUP>8</SUP> cells. One cell comprises around 1.27848*10<SUP>-6</SUP>M beta-lactamase in the periplasm. We adapted our system accordingly.
 </DIV><CENTER><BUTTON><A class="button_link" href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_II" role="button">ModelII</A></BUTTON></CENTER><DIV class="container text_header"><H3>References</H3></DIV><DIV class="container text"><UL><LI>Atkins, Peter W.; Paula, Julio de (2010): Physical chemistry. 9. ed. Oxford u.a.: Oxford Univ. Press.</LI><LI>Bisswanger, Hans (2008): Enzyme kinetics. Principles and methods. 2., rev. and updated ed. Weinheim: Wiley-VCH. Online accessible at http://site.ebrary.com/lib/alltitles/docDetail.action?docID=10301967.</LI><LI>Erickson, Harold P. (2009): Size and shape of protein molecules at the nanometer level determined by sedimentation, gel filtration, and electron microscopy. In: Biological procedures online 11, S. 32–51. DOI: 10.1007/s12575-009-9008-x.</LI><LI>Galarneau, André; Primeau, Martin; Trudeau, Louis-Eric; Michnick, Stephen W. (2002): |[beta]|-Lactamase protein fragment complementation assays as in vivo and in vitro sensors of protein|[ndash]|protein interactions. In: Nature Biotechnology 20 (6), S. 619–622. DOI: 10.1038/nbt0602-619.</LI><LI>Jalalirad, Reza (2013): Selective and efficient extraction of recombinant proteins from the periplasm of Escherichia coli using low concentrations of chemicals. In: Journal of industrial microbiology &amp; biotechnology 40 (10), S. 1117–1129. DOI: 10.1007/s10295-013-1307-1.</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>