Team:Bielefeld-CeBiTec/Results/Selection
Toggle navigation
NavBar
PROJECT
Description
Library
Overview
Scaffolds
Design and Construction
Create your own Library
Mutation
Overview
Error prone Polymerase I
Genome wide mutator
Selection
Overview
Bacterial Two-Hybrid System
Alternative Selection System
Modeling
Overview
Model I
Model II
RESULTS
Overview
Library
Overview
Assembly
Phagemid Display
Sequencing
Mutation
Overview
Assembly
Reversion
Sequencing
Selection
Overview
Assembly
Expression Control
Interaction Control
Binding Control
in vivo Selection
Modeling
Fermentation
Parts
Best Basic Part
Best Composite Part
Best Part Collection
Improve a part
Proof of Concept
Demonstrate your Work
Medal Fulfillment
HUMAN PRACTICES
Overview
Engagement
Dictionary
Educational Outreach
Crowdfunding and Press
Entrepreneurship
Integrated Practices
Achievements
COLLABORATIONS
Overview
Düsseldorf
Lethbridge
Freiburg
NOTEBOOK
Overview
Lab Notebook
Library
Mutation
Selection
Primers
Lab Protocols
Media and Buffers
TEAM
Team Profile
Members
Bielefeld University
Contact
ACKNOWLEDGEMENT
Overview
Sponsors and Partners
Attributions
Results Selection
The purpose of our project was developing a system to generate Evobodies that are able to bind any given target protein. While a library and the mutation system grant high variability of Evobodies, we needed a method to screen for those that actually show affinity to the target protein. That is why we decided to utilize an in vivo selection system. The beauty of this approach is, that selective advantage for functional Evobodies ensures the lone survival of bacteria providing exactly those, eventually.
We were able to successfully clone the two constructs for the fusion proteins as well as several controls for our bacterial two-hybrid system . Furthermore, we checked if our fusion proteins were expressed as estimated. This was analyzed mainly using SDS PAGE and Western Blot.
Figure 1. SDS PAGE for expression control: Result of a SDS PAGE with samples from left to right: Color prestained marker, HA4 Evobody wild type, HA4 Y87A mutant Evobody, HA4 R38A mutant Evobody, HA4 R38A E52A, SH2:cI + HA4 Evobody, SH2:cI434, SH2:cILambda
In addition, we were able to show that our fusion proteins actually interacted . This was achieved via a BLItz experiment:
Figure 2: Results of the BLItz experiment. Illustrated are the measured base line, the association and the dissociation of the protein-protein interaction in the BLItz. The addition of the second protein is marked at 60 seconds after measuring the base line. The SH2-cI fusion protein was added at the start without washing (green), after one (purple) and two (yellow) washing steps with NaCl and after a washin step with a high concentrated HCl solution (upper blue). Also BSA was measured with 1 μg/ml (lower blue) and 10 μ/ml (red) protein.
To test the affinity of our DNA-binding domain (DBD) towards its supposed binding site we examined their interaction in vitro using EMSA. The results depict the expected band shift strongly suggesting interaction between DBD and its binding site.
Figure 3: EMSA results. In alternaty application the DNA fragment with the 434 OR1 binding site and the DNA fragment with the lambda binding site were used as DNA sample. +/- = with/without SH2-cI fusion protein.
An in vivo experiment designed by us supports the EMSA results. A construct containing the green fluorescent protein (GFP) upstream and a red fluorescent protein (RFP) downstream of the DBD binding site has been used. Since we observed a lower RFP/GFP ratio in the presence of the DBD, the DBD appears to bind to its target site, thereby preventing the polymerase to express RFP.
Figure 4: Tecan results. In the Tecan plate reader the RFP intensity was measured. The E. coli cultures with (red) and without (black) producing the cI repressor were analyzed. Error bars are implemented.
Last but not least we established to show a higher expression rate when using both fusion proteins in contrast to a negative control with only one, indicating the functionality of the bacterial two-hybrid system in general.
Figure 5: Tecan-Results in vivo reporter activity. Two culters of E. coli were measured on their ability to produce RFP through the reporter construct. A culture with one fusion protein SH2-cI expressing is compared with a culture carrying both fusion proteins SH2-cI and HA4-RpoZ. The RFP intensity of the culture with both fusion proteins is significantly higher than in the culture with only one fusion protein.
Assembly
Expression Control
Interaction Control
Binding Control
in vivo selection
