Team:Bielefeld-CeBiTec/Project/Library
Toggle navigation
NavBar
PROJECT
Description
Library
Overview
Scaffolds
Design and Construction
Create your own Library
Mutation
Overview
Error prone Polymerase I
Genome wide mutator
Selection
Overview
Bacterial Two-Hybrid System
Alternative Selection System
Modeling
Overview
Model I
Model II
RESULTS
Overview
Library
Overview
Assembly
Phagemid Display
Sequencing
Mutation
Overview
Assembly
Reversion
Sequencing
Selection
Overview
Assembly
Expression Control
Interaction Control
Binding Control
in vivo Selection
Modeling
Fermentation
Parts
Best Basic Part
Best Composite Part
Best Part Collection
Improve a part
Proof of Concept
Demonstrate your Work
Medal Fulfillment
HUMAN PRACTICES
Overview
Engagement
Dictionary
Educational Outreach
Crowdfunding and Press
Entrepreneurship
Integrated Practices
Achievements
COLLABORATIONS
Overview
DÃ¼sseldorf
Lethbridge
Freiburg
NOTEBOOK
Overview
Lab Notebook
Library
Mutation
Selection
Primers
Lab Protocols
Media and Buffers
TEAM
Team Profile
Members
Bielefeld University
Contact
ACKNOWLEDGEMENT
Overview
Sponsors and Partners
Attributions
Library Project
Overview
Motivation
As every house needs a cornerstone, we need our binding protein library. The demand for easily accessible, fast adapting and specific binding proteins is ever-growing. To utilize the advantages of binding proteins for our project, a library adapted to the end-users (or more specific: our) needs, was designed by us.
When we were faced with the challenge of designing a binding protein library, earlier in 2016 year, we browsed existing iGEM projects and were surprised that we did not find preexisting examples for libraries. Actually - our investigation showed that our approach is unprecedented in this competition. Introducing and providing libraries of binding proteins therefore establishes a new entity in the iGEM BioBrick collection and provides new opportunities for the community. With our work we also hope to show how synthetic libraries can be in synthetic biology and pave the way for the usage of libraries in iGEM.
Overview
To generate the perfect binder an initial genetic library coding for binding proteins with a diversified binding-surface is essential for our project. Similar to the initial immune response this set of molecules forms the foundation for our in vivo mutagenesis system, which mimics the antibody maturation process.
We based our library layout on a synthetic binding protein named Monobody (adenctin) as well as on a naturally occurring binding protein named Nanobody. Each possesses advantages compared to conventional antibodies, making them suitable for our system. Read more about our used scaffolds.
Setting up a humongous library, containing all different kinds of variants, seemed tempting or even impossible at first, but after researching the literature we opted for a more defined and optimized library. For doing so, we restricted the randomization of the binding regions of our scaffold proteins to specific amino acids ( Fellouse et al., 2004 ; Koide, 2009 ). With this choice we were able to randomize larger parts of the scaffold to generate antibody like binding properties while at the same time keeping the theoretical library diversity at a maximum of about one billion, 1,073,741, 824 variants to be precise. Note that the number of variants can be further increased in our evolutionary selection system. See our Design and Construction for further insight.
Figure 1: Library of initial binding proteins. Expression of the initial binding proteins with variable regions highlighted in separate colors (turquoise, orange, white, green, pink and blue). The theoretical library diversity for each scaffold is 1,073,741,824.
Scaffold
Design and Construction
Create your own library
Results
References
Fellouse, Frederic A., 2004. Synthetic Antibodies from a Four-Amino-Acid Code: A Dominant Role for Tyrosine in Antigen Recognition. Proceedings of the National Academy of Sciences of the United States of America, 2004, 101, 34, 12467.
Koide, S 2009. The Importance of Being Tyrosine: Lessons in Molecular Recognition from Minimalist Synthetic Binding Proteins. ACS CHEMICAL BIOLOGY, 2009, 4, 5, 325.
