<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Collaborations_Lethbridge skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Collaborations/Lethbridge</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container main"><DIV id="header"><NAV id="navigationbar" class="navbar navbar-default custom_navbar_style navbar-fixed-top"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="#">NavBar</A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-center"><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description"> PROJECT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description">Description</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds">Scaffolds</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design">Design and Construction</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/CreateYours">Create your own Library</A></LI></UL><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/EpPolI">Error prone PolymeraseI</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/Global">Genome wide mutator</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">Bacterial Two-Hybrid System</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/SplitBetaLactamaseHitchHiker">Alternative Selection System</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_I">Model I</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_II">Model II</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results"> RESULTS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Phage">Phagemid Display</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">Reversion</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/Cloning">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/ExpressionControl">Expression Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/InteractionControl">Interaction Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/BindingControl">Binding Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/KnockOutKnochIn"><I>in vivo</I> Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Modeling">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Fermentation">Fermentation</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Best Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Best Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Best Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts/Improve">Improve a part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Demonstrate your Work</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Medal_Fulfillment">Medal Fulfillment</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold"> HUMAN PRACTICES </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Dictionary">Dictionary</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks">Educational Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Crowdfunding">Crowdfunding and Press</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Achievements</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations"> COLLABORATIONS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Duesseldorf">Düsseldorf</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Lethbridge">Lethbridge</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Freiburg">Freiburg</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments"> NOTEBOOK </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook">Lab Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">Primers</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols">Lab Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Media">Media and Buffers</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team"> TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team">Team Profile</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Members">Members</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Contact">Contact</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement"> ACKNOWLEDGEMENT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Sponsors_And_Partners">Sponsors and Partners</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></DIV></DIV></NAV></DIV></DIV><DIV class="container main"><DIV class="container main"><DIV class="jumbotron" style="background-image: url(https://static.igem.org/mediawiki/2016/f/f7/Bielefeld_CeBiTec_2016_10_14_X_hands.png)"><DIV class="jumbotron-text"><H1 style="margin-bottom: 0px; text-align:left">Collaborations</H1><H2 style="color:#ffffff; text-align:left">A joy that's shared is a joy made double</H2></DIV></DIV><DIV class="container text_header"><H1>Collaboration - <A href="https://2016.igem.org/Team:Lethbridge">Lethbridge</A></H1></DIV><DIV class="container text_header"><H3>Summary</H3></DIV><DIV class="container text"><UL><LI>Discussed practical obstancles in creation of a binding protein library</LI><LI>Compared the different approaches of the bacterial two-hybrid system</LI><LI>Bielefeld provided Lethbridge with positive control in various strength for characterization of their two hybrid system</LI></UL><UL><LI>Lethbridge send the influenza A NS1 peptide as a potential target for generation of Evobodies</LI></UL></DIV><DIV class="container text_header"><H3>Skype Protocol: Lethbridge - Bielefeld</H3></DIV><H2>Protocol from 2016-08-26</H2><DIV class="container text">
				Participants Lethbridge: Andrew, Taylor, Rhys
				
				Paricipants Bielefeld: Marius, Sebastian, Carsten, Boas 
				
				Each team shortly describes their projects
				<H3>Lethbridge</H3><UL><LI>Library: Nanobodies
						</LI><LI>bioinformatic approach to predict good binding amino acids</LI><LI>theoretical variability of ~10<SUP>18</SUP> possible sequences accounting for CDR1, CDR2 and CDR3, however only ~10<SUP>5-6</SUP> will be sampled during screening.</LI></UL><LI>Screen via bacterial two hybrid
						</LI><LI>Nanobody fused to α-subunit of RNA-polymerase on plasmid 1</LI><LI>target protein fused to λcI-protein on plasmid 2</LI><LI>readout via mRFP1 expression relative to mTagBFP expression and cell sorting</LI><LI>system inspired by the invitrogen BacterioMatch II</LI><LI>Target
						</LI><LI>Collaboration with local EMS to investigate microbiome in ambulence vehicle</LI><LI>Identified microbes via DNA isolation of swabs from various points in ambulences and Nanopore next generation sequencing.</LI><LI>Important pathogens identified in sequencing or other studies selected for antibody production via bacteria-two-hybrid selection.</LI><H3>Bielefeld</H3><UL><LI>Library of nano- and monobodies
						</LI><LI>randomization of CDR3-loop (nanobodies) respectively three loops (monobodies)</LI><LI>theoretical variability of 10<SUP>9</SUP> binding proteins</LI></UL><LI>Error prone mutation system to further increase the variability</LI><LI>Screen via bacterial two hybrid  
						</LI><LI>binding protein fused to ω-subunit of RNA-polymerase on plasmid 1</LI><LI>target protein fused to 434cI on plasmid 2</LI><LI>selection via β-lactamase and cell survival (RFP expression as control)</LI><LI>Target
						</LI><LI>contacted experts with respect to possible targets</LI><LI>viral envelope proteins (e.g. zika)</LI><LI>cellular proteins</LI>
				Cooperation plans were discussed
				<OL><LI>Exchange of libraries with detailed cloning strategy</LI><LI>Exchange of controls for the bacterial two hybrid system</LI><LI>Mutual validation of the bacterial two hybrid</LI></OL></DIV><H2>Protocol from 2016-08-26</H2><DIV class="container text">
				Participants Lethbridge: Andrew, Rhys, Graeme
				
				Paricipants Bielefeld: Marius, Pascal, Carsten, Boas
				
				Each team describes their progress and current obstacles
				<H3>Lethbridge</H3><UL><LI>Library
						</LI><LI>complete randomisation of CDR loops</LI><LI>very high library size (Nanopore sequencing)</LI><LI>transformation efficiency is limiting (~40,000 clones)</LI><LI>transformation strategy: electroporation</LI></UL><LI>Two-Hybrid
						</LI><LI>fluorescent protein gBlock contains mutations =&gt; use of streptomycin as reporter and selection via survivability</LI><LI>measurement of the basal activity of the reporter (weak constitutive promoter)</LI><LI>genomic integration of reporter gene is the target =&gt; only one copy per cell would be nice =&gt; would further decrease basal expression level</LI><LI>start selection cycles directly with target protein</LI><LI>Target: domain III of Zika E-protein</LI><LI>general design
						</LI><LI>two plasmid analog to the Agilent BacterioMatch II Two-Hybrid System</LI><LI>initial plan: use two reporters for copy number control</LI><LI>monobodies still contain S-S bridges</LI>
				Mutual discussion about the molecular mechanism of the two hybrid 
				<UL><LI>Question: Doesn't a strong antibody-mimetic - target protein affinity hold the RNA-Pol back?</LI><LI>No literature addressing this issue was found</LI><LI>Possible explanations
						</LI><LI>protein interaction is broken by RNA-Pol</LI><LI>DNA is pulled through the enzyme which stays at the promoter position</LI></UL><H3>Bielefeld</H3><UL><LI>Library
						</LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design">randomisation strategy</A> (leads to 10<SUP>9</SUP> variants each)</LI><LI>high quality library by using only specifi amino acids</LI><LI>relative small library size =&gt; potential to use a considerable amount of the whole library</LI><LI>transformation strategy: electroporation, large agar plates</LI></UL><LI>Two-Hybrid
						</LI><LI>characterization of the two hybrid system by using the HA4 - SH2 binding pair in addition to mutants with decreasing strength</LI><LI>=&gt; correlation between binding strength and two hybrid output</LI><LI>talked about the design of the two hybrid reporter site
								</LI><LI>modified promoter for very low basal expression</LI><LI>optimized distance between cI binding site and promoter</LI><LI>start selection cycles directly with target protein</LI><LI>Target: domain III of Zika E-protein</LI><LI>general design
						</LI><LI>two plasmid analog to the Agilent BacterioMatch II Two-Hybrid System</LI><LI>initial plan: use two reporters for copy number control</LI><LI>monobodies still contain S-S bridges</LI><H2>Protocol from 2016-09-29</H2><DIV class="container text">
				Participants Lethbridge: Andrew
				
				Paricipants Bielefeld: Marius, Pascal, Carsten 
				
				Current status of each project
				<H3>Lethbridge</H3><UL><LI>Library
						</LI><LI>Optimization of competent cells with cfu ~ 10<SUP>8</SUP> cfu/μg</LI><LI>currently 40.000 different clones</LI><LI>confirmed by deep sequencing</LI></UL><LI>Two-Hybrid
						</LI><LI>constitutive fluorescence A as control of plasmid copy number</LI><LI>bacterial two hybrid activatable fluorescence B as signal of the bacterial two hybrid</LI><LI>currently no strong increase of fluorescence B detectable</LI><H3>Bielefeld</H3><UL><LI>Library
						</LI><LI>variable regions are cloned into the scaffolds</LI><LI>NGS of the variable regions is planned</LI></UL><LI>Two-Hybrid
						</LI><LI>devices are completely assembled but no reporter activity is detected</LI><LI>control of the single parts of the two hybrid system</LI><LI>control of the DNA binding of cI</LI><LI>which cI OR's does Lethbridge use? suggestion that lethbridge checks which OR’s they use for λcI and maybe add missing OR's</LI><LI>change of 434 cI to the partsreg λcI + binding site</LI><LI>control of the positive control binding by native PAGE and affinity chromatography</LI>
				Cooperation plans were discussed
				<OL><LI>Exchange of libraries with detailed cloning strategy</LI><LI>Exchange of controls for the bacterial two hybrid system</LI><LI>Mutual validation of the bacterial two hybrid</LI></OL><UL><LI>Discussion points
						</LI><LI>modelling of complete bacterial two hybrid on protein structure level</LI><LI>=&gt; could yield information about the optimal distance between cI binding site and promoter</LI><LI>control of the bacterial two hybrid system
								</LI><LI>positive controls in the HA4-SH2 binding pair from Bielefeld</LI><LI>direct fusion between cI and the RNA-Pol subunit as a positive control?</LI><LI>GFP inbetween as expression control?</LI></UL><LI>target
								</LI><LI>Lethbridge uses the domain III of zika E protein as target</LI><LI>Bielefeld uses the complete zika E protein as target</LI></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>