Team:Bielefeld-CeBiTec/Experiments/Protocols
Toggle navigation
NavBar
PROJECT
Description
Library
Overview
Scaffolds
Design and Construction
Create your own Library
Mutation
Overview
Error prone Polymerase I
Genome wide mutator
Selection
Overview
Bacterial Two-Hybrid System
Alternative Selection System
Modeling
Overview
Model I
Model II
RESULTS
Overview
Library
Overview
Assembly
Phagemid Display
Sequencing
Mutation
Overview
Assembly
Reversion
Sequencing
Selection
Overview
Assembly
Expression Control
Interaction Control
Binding Control
in vivo Selection
Modeling
Fermentation
Parts
Best Basic Part
Best Composite Part
Best Part Collection
Improve a part
Proof of Concept
Demonstrate your Work
Medal Fulfillment
HUMAN PRACTICES
Overview
Engagement
Dictionary
Educational Outreach
Crowdfunding and Press
Entrepreneurship
Integrated Practices
Achievements
COLLABORATIONS
Overview
Düsseldorf
Lethbridge
Freiburg
NOTEBOOK
Overview
Lab Notebook
Library
Mutation
Selection
Primers
Lab Protocols
Media and Buffers
TEAM
Team Profile
Members
Bielefeld University
Contact
ACKNOWLEDGEMENT
Overview
Sponsors and Partners
Attributions
Protocols
Used Protocols
Protocols
Standard protocols
Blunt end ligation
For "intramolecular" ligation of a PCR product the 5'-ends are phosphorylated with T4 Polynucleotidekinase (PNK)
Set up the following reaction:
reagent
volume [ µL]
10x T4 DNA ligase buffer
2.5
T4 PNK
1
PEG 4000 50% (w/v)
2.5
DNA (100 to 600 ng)
x
ddH2O
to 25
Incubate at 37 °C, 30 min
Heat inactivate at 65 °C, 20 min
Add 1 µL T4 DNA ligase after reaction has cooled down to room temperature
Incubate at room temperature for at least 2 h, overnight also works.
Next step: Transformation via heat shock
Colony PCR
One Reaction mix contains
5 µL 5x GoTaq buffer (Promega)
1 µL MgCl2 (25 mM stock)
0.5 µL dNTPs (10 mM each)
0.25 µL primer mix (prefix/suffix primers or sequencing primers) 100 mM
17.625 µL ddH2O
0.125 µL GoTaq polymerase (Promega)
0.5 µL template
PCR program
Cell lysis and denaturation: 5 min, 95 °C
30 cycles
Denaturation: 10 s, 95 °C
Hybridisation: 30 s, annealing temperature
Elongation: 60 s/kb of product, 72 °C
Final elongation: 5 min, 72 °C
Template alternatives
Pick a colony with sterile tip, elute in 100 µL ddH2O or buffer, store at 4 °C during PCR, plate if insert is of correct size
Pick colony, streak at marked position on a new plate and solute remaining cells on the tip in the PCR tube with reaction mixture, cultivate if insert is of correct size
Gel electrophoresis for control of fragment size
DNA Purification
We used the following kits for DNA purification. For each kit the specific protocol is described seperately:
Thermo Scientific GeneJET Gel Extraction Kit
Note: all centrifugations should be carried out in a table-top microcentrifuge at > 12000 x g/10.000-14.000 rpm
Preparation
Excise gel slice containing the DNA fragment using a clean scalpel or razor blade. Cut as close to the DNA band as possible to minimize the gel volume. Place the gel slice into a pre-weighed 1.5 mL tube. Record the weight of the gel slice. Note: If the purified fragment will be used for cloning reactions, avoid damaging the DNA through UV light exposure. Minimize UV exposure to a few seconds or keep the gel slice on a glass or plastic plate during UV illumination.
Add 1:1 volume of Binding Buffer to the gel slice (volume:weigt)
Incubate the gel mixture at 50-60 °C for 10 min or until the gel slice is completely dissolved. Mix the tube by inversion every few minutes to facilitate the melting process. Ensure that the gel slice is completely dissolved. Vortex the gel buffer mixture briefly before loading on the column.
Check the color of the solution. A yellow color indicates an optimal pH for DNA binding. If the color of the solution is orange or violet, add 10 µL of 3 M sodium acetate, pH 5.2 solution and mix. The color of the mix will become yellow.
Optional: Use this step only when DNA fragment is <500 bp or >10 kb long.
If the DNA fragment is <500 bp, add 1:2 volume of 100 % isopropanol to the solubilized gel solution. Mix thoroughly.
If the DNA fragment is >10 kb, add 1:2 volume of water to the solubilized gel solution. Mix thoroughly.
Binding of DNA
Transfer up to 800 µL of the solubilized gel solution (from step 3/4) to the GeneJET purification column. Centrifuge 1 min at 11,000 x g. Discard the flow-through and place the column back into the same collection tube. Note: If the total volume exceeds 800 µL, the solution can be added to the column in stages. After each application, centrifuge the column for 30-60 s and discard the flow through after each spin. Repeat until the entire volume has been applied to the column membrane. Do not exceed 1 g of total agarose gel slice per column.
Add 100 µL of Binding Buffer to the GeneJet purification column. Centrifuge for 1 min. Discard the flow-through and place the column back into the same collection tube.
Washing
Add 700 µL of Wash Buffer to the GeneJet purification column. Centrifuge for 1 min. Discard the flow-through and place the column back into the same collection tube. (We altered the protocol to to wash steps a 350 µL optional)
Centrifuge the empty GeneJet purification column for an additional 1 min to completely remove residual wash buffer
Elution
Transfer the GeneJET purification column into a clean 1.5 mL microcentrifuge tube. Add 50 µL of Elution Buffer to the center of the purification column membrane. Centrifuge for 1 minute. Note: for low DNA amounts the elution volumes can be reduced to increase DNA concentration. An elution volume between 20-50 µL does not significantly reduce the DNA yield. However, elution volumes less than 10 µL are not recommended. If DNA fragment is >10 kb, preheat Elution Buffer to 65 °C before applying to column. If the elution volume is 10 µL and DNA amount is <5 µg, incubate column for 1 min at room temperature before centrifugation.(We altered the protocol to 30 µL Elution Buffer preheated to 60 °C and 1 min incubation before centrifuging for all sizes.)
Discard the GeneJet purification column and store the purified DNA at -20 °C.
Qiagen QIAquick Gel Extraction Kit (Centrifugation)
Note: All centrifugation steps are carried out at 17.900 x g (13.000 rpm) in a conventional table-top microcentrifuge
Preparation
Excise the DNA fragment from the agarose gel with a clean, sharp scalpel
Weigh the gel slice in a colorless tube. Add 3 volumes of Buffer QG to 1 volume gel (100 mg gel - 100 µL). The maximum amount of gel per spin column is 400 mg. For >2 % agarose gels, add 6 volumes Buffer QG
Incubate at 50°C for 10 min (or until the gel slice has completely dissolved). Vortex the tube every 2-3 minutes to help dissolve the gel. After the gel slice has dissolved completely, check that the color of the mixture is yellow(similar to Buffer QG without dissolved agarose). If the color of the mixture is orange or violet, add 10 µL of 3 M sodium acetate, pH 5.0, ad mix. The mixture turns yellow.
Binding of DNA
Add 1 volume isopropanol to the sample and mix.
Place a QIAquick spin column in a provided 2 mL collection tube. To bind DNA, apply the sample to the QIAquick column and centrifuge for 1 min until all the samples have passed through the column. Discard the flow-through and place the QIAquick column back into the same tube. For sample volumes >800 µL, load and spin again.
Washing
If DNA will subsequently be used for sequencing,
in vitro
transcription or microinjection, add 500 µL Buffer QG to the QIAquick column and centrifuge for 1 min. Discard flow-through and place the QIAquick column back into the same tube
To wash, add 750 µL Buffer PE to QIAquick column and centrifuge for 1 min. Discard flow-through and place the QIAquick column back into the same tube. If the DNA will be used for salt-sensitive applications (sequencing, blut-ended-ligation) let the column stand 2-5 minutes after addition of Buffer PE. Centrifuge the QIAquick column in the provided 2 mL collection tube for 1 min to remove residual wash buffer
Place QIAquick column into a clean 1.5 mL microcentrifuge tube
Elution
To elute DNA, add 50 µL Buffer EB (10 mM TRIS-HCl, pH 8.5) or water to the center of the QIAquick membrane and centrifuge the column for 1 min. For increased DNA concentration, add 30 µL Buffer EB to the center of the QIAquick column, let the column stand for 1 min, and then centrifuge for 1 min. After the addition of Buffer EB to the QIAquick membrane, increasing the incubation time up to 4 min can increase the yield of purified DNA.
If the purified DNA is to be analyzed on a gel, add 1 volume of loading dye to 5 volumes of purified DNA. Mix the solution by pipetting up and down before loading the gel.
Analytik Jena InnuPREP Gel Extraction Kit
Preparation
Excise the DNA fragment from the agarose gel with a sharp scalpel. Note: minimize the agarose gel slice. Check the weight. Do not use more than 300 mg gel slice for one spin filter.
Transfer the gel slice into a 1.5 mL or 2.0 mL reaction tube and add the 650 µL Gel Solubilizer.
Incubate for 10 min at 50°C until the agarose gel slice is completely dissolved.
Binding of DNA
Add 50 µL Binding Optimizer and mix the suspension by vortexing or piepetting sometimes up and down.
Apply the sample onto the spin filter (green) located in a 2.0 mL Receiver Tube. Close the cap and centrifuge at 10.000 x g (12.000 rpm) for 1 min. Discard the filtrate and re-use the Receiver Tube. Place the Spin Filter back into the 2.0 mL Receiver Tube.
Washing
Open the Spin Filter and add 700 µL Washing Solution LS, close the cap and centrifuge at 10.000 x g (12.000 rpm) for 1 min. Discard the filtrate and re-use the Receiver Tube. Place the Spin Filter back into the 2.0 mL Receiver Tube. Repeat the entire step.
Centrifuge at max speed for 2 min to remove all traces of ethanol. Discard the 2.0 mL Receiver Tube.
Elution
Place the Spin Filter into a 1.5 mL elution Tube. Carefully open the cap of the Spin Filter and add 30-50 µL Elution Buffer (optionally preheated to 50°C). Incubate at room temperature for 1 min. Centrifuge at 6.000 x g (8.000 rpm) for 1 min. A second elution step will increase the yield of extracted DNA. Note: The DNA can be eluted with a lower or a higher volume of Elution Buffer. (...) Store the extracted DNA at +4 °C. For long time storage placing at -20°C us recommended
Promega Wizard SV Gel and PCR CleanUp System
Gel Slice/PCR product preparation
Following electrophoresis, excise DNA band from gel and place gel slice in a 1.5 mL microcentrifuge tube.
Add 10 µL of Membrane Binding solution per 10 mg of gel slice. Vortex and incubate at 50-65 °C until gel slice is copletely dissolved.
Alternative for PCR amplifications: Add an equal volume of Membran Binding Solution to the PCR amplification
Binding of DNA
Insert SV Minicolumn into Collection Tube.
Transfer dissolved gel mixture or prepared PCR product to the Minicolumn assembly. Incubate at room temperature for 1 min.
Centrifuge at 16.000 x g for 1 minute. Discard flow through and reinsert Minicolumn into Collection Tube.
Washing
Add 700 µL Membrane Wash Solution (ethanol added). Centrifuge at 16.000 x g for 1 minute. Discard flow-through and reinsert Minicolumn into Collection Tube.
Repeat previous Step with 500 µL Membrane Washing Solution. Centrifuge at 16.000 x g for 5 minutes.
Empty the Collection Tube and recentrifuge the column assembly for 1 minute with the microcentrifuge lid open (or off) to allow evaporation of any residual ethanol.
Elution
Carefully transfer Minicolumn to a clean 1.5 mL microcentrifuge tube.
Add 50 µL of Nuclease-Free Water to the Minicolumn. Incubate at room temperature for 1 minute. Centrifuge at 16.000 x g for 1 minute.
Discard Minicolumn and store DNA at 4 °C or -20°C.
Gibson Assembly
Modified from Gibson et al. (2009)
This assembly method is an isothermal, single-reaction method for assembling multiple overlapping DNA molecules. By coordinating the activity of a 5' exonuclease, a DNA polymerase and a DNA ligase two adjacent DNA fragments with complementary terminal sequence overlaps can be joined into a covalently sealed molecule, without the use of any restriction endonuclease.
Preparation of DNA molecules for in vitro recombination
Generate the complenentary sequence overlaps by PCR using the Phusion DNA-polymerase. If necessary add 5 M Betain in the reaction mix by reducing the amount of H2O to decrease the number of false PCR products.
Seperate the PCR product of interest by gel electrophoresis, extract the products via Gel Extraction .
In vitro recombination
Preparing assembly mixture
320 µL 5x isothermal reaction buffer
0.64 µL of 10 U ml-1 T5 exonuclease (for DNA molecules overlapping by greater than 150 bp add 3.2 µL of 10 U mL-1 T5 exonuclease)
20 µL of 2 U mL-1 Phusion DNA polymerase
160 µL of 40 U mL-1 taq DNA ligase
add ddH2O up to a final volume of 1.2 mL
Aliquote 15 µL of the assembly mixture and store at -20 °C
That 15 µL assembly mixture on ice until use
Add 5 µL of the purified DNA molecules in equimolar amounts (between 10 and 100 ng of each fragment).
Incubate the mixture at 50 °C for 15 to 60 min with 60 min being optimal.
Transformation via heat shock or electroporation without cleaning up the assembly product.
Q5 PCR
Q5
Q5 master mix (25 µL)
5 µL 5 x Q5 buffer
0.5 µL 10 mM dNTPs
Up to 50 ng*µL-1template
0.1 µL Q5 DNA polymerase
0.5-1 µL primer 10 µM each
Ad 25 µL dH2O
Q5 standard protocol
Initial denaturation: 98°C - 30 s
30-35 cycles of
Denaturation: 98 °C - 20 s
Annealing: 50-75 °C - 30 s
Elongation: 72 °C - 20 s/kb
Final elongation: 72 °C - 5 min
Storage: 4 °C
Generating competent cells
We generated chemocompetent, ultra-chemocompetent and electrocompetent E. coli using the following protocols:
Chemo competent cells
Preparation
Cool 5 50 mL Falcons
Cool TSS buffer
Cool centrifuge
Protocol
Inoculate 2 x 3 mL LB with bacterial stock over night at 37 °C and 200 rpm
Inoculate 1 x 250 mL LB with the over night cultures in 1 L flask at 37 °C and 140 rpm
Incubate to OD600 0.4-0.6
Devide the culture into 5 cooled 50 mL falcons and incubate on ice for 10 min
Onwards all steps at 4 °C
Centrifuge at 3000 rpm, 4 °C, 10 min
Discard supernatant
Pipette remaining media out carefully
Add 10% volume cold TSS buffer of original cultur, about 1-2 mL
Resuspend pellet
Aliquot 100 µL and drop in liquid nitrogen immediatly
Store at -80 °C
Electro competent cells
Preparation
Autoclave 550 mL LB-Medium
Cool 1 L bidest. H2O
Cool 50 mL 10% glycerine
Cool 10 50 mL Falcons
Protocol
Inoculate 2 x 3 mL LB with bacterial stock, incubate over night at 37 °C and 200 rpm
Inoculate 2 x 250 mL LB with the over night culture in 1 L flask at 37 °C and 140 rpm
Cultivate until OD6000.4-0.6
Store on ice for 15-30 min
Onwards all steps at 4 °C
Divide the cultures into the pre-cooled 50 mL Falcons and centrifuge at 4000 rpm, 4 °C, 15 min with slow acceleration and decceleration
Discard supernatant
Resuspend in 5 mL cooled bidest H2O shaking gently
Pool two suspensions, add bidest. H2 to 50 mL and centrifuge again at 4000 rpm, 4°C, 15 min
Discard supernatant
Resuspend pellet in 5 mL cooled bidest H2O
Add bidest H2O up to 50 mL and centrifuge again at 4000 rpm, 4 °C, 15 min
Discard supernatant
Resuspend in 5 mL cooled 10% glycerine
Transfer suspensions in two 50 mL falcons and centrifuge again at 4000 rpm, 4 °C, 15 min
Discard supernatant
Add 1 volume similar to the pellet of 10% glycerine and resuspend
Aliquot in 50 µL and freeze in liquid nitrogen immediately
Store at -80 °C
Ultra chemo competent cells
Preparation
Inoculate 10 mL LB medium in a 100 mL flask with 50 µL cells from glycerol culture, e.g. E. coli DH5a MCR
Incubate over night at 37 °C and 180 rpm
Protocol
Precool centrifuge and media
Inoculate 250 mL SOB-Medium in a 1 L flask with 2.5 mL of the preculture
Incubate at 19 °C  and 180 rpm up to OD600 0.5 (about 24-36 hours)
Cool flask 10 min in ice water
Onwards all steps at 4 °C
Devide the culture in 5 50 mL cooled falcons
Centrifuge 5 min at 4 °C and 5000 x g
Discard supernatant
Resuspend in each 6 mL icecold TB Buffer
Incubate 10 min on ice
Centrifuge 5 min at 4 °C and 5000 x g
Discard supernatant
Resuspend in each 6 mL icecold TB Buffer
Add 420 µL DMSO and vortex immediately
Aliquot 150 µL and freeze in liquid nitrogen immediately
Store at - 80 °C
Plasmid Isolation
For Plasmid Isolation we used the following kits. The methods are described seperately:
PureYieldTM Plasmid Miniprep System (Promega)
Prepare Lysate
Add 600 µL of bacterial culture to a 1.5 mL microcentrifuge tube
Alternative for larger cultures: Centrifuge 1.5 mL bacterial culture for 30 s at maximum speed in a microcentrifuge. Discard the supernatant. Add an additional 1.5 mL bacterial culture to the same tube and repeat previous step. Add 600 µL of TE buffer or water to the pellet, and resuspend completely.
Add 100 µL of Cell Lysis Buffer (blue), and mix by inverting the tube 6 times.
Add 350 µL of cold (4-8 °C) Neutralization Solution and mix thoroughly by inverting.
Centrifuge at maximum speed in a microcentrifuge for 3 minutes.
Transfer the supernatant (about 900 µL) to a PureYieldTM Minicolumn without disturbing the cell debris pellet
Place the minicolumn into a Collection Tube and centrifuge at maximum speed in a microcentrifuge for 15 s
Discard the flowthrough, and place the minicolumn into the same Collection Tube.
Wash
Add 200 µL of Endotoxin Removal Wash (ERB) to the minicolumn. Centrifuge at maximum speed in a microcentrifuge for 15 s
Add 400 µL of Column Wash Solution (CWC)to the minicolumn. Centrifuge at maximum speed in a microcentrifuge for 30 seconds
Elute
Transfer the microcolumn to a clean 1.5 mL microcentrifuge tube, then add 30 µL of Elution Buffer or nuclease-free water directly to the minicolumn matrix. Let stand for 1 minute at room temperature
Centrifuge for 15 seconds to elute the plasmid DNA. Cap the microcentrifuge tube, and store eluted plasmid DNA at -20°C
Thermo Scientific GeneJET Plasmid Miniprep Kit (Centrifugation)
All purification steps should be carried out at room temperature.
All centrifugations should be carried out in a table-top microcentrifuge at >12.000 x g (10.000-14.000 rpm)
Use 1-5 mL of E. coli culture in LB media for purification of high-copy plasmids.
Use up to 10 mL culture for low-copy plasmids
Protocol
Resuspend the pelleted cells in 250 µL of the Resuspension Solution. Transfer the cell suspension to a microcentrifuge tube. The bacteria should be resuspended completely by vortexing or pipetting up and down until no cells clumps remain.Note: Ensure RNase A has been added to the Resuspension Solution.
Add 250 µL of the Lysis Solution and mix throughly by inverting the tube 4-6 tines until the solution becomes viscous and slightly clear. Note: Do not vortex to avoid shearing of chromosomal DNA. Do not incubate for more than 4 min to avoid denaturation of supercoiled DNA.
Add 350 µL of the Neutralization Solution and mix immediately and thoroughly by inverting the tube 4-6 times. Note:It is important to mix throughly and gently after the addition of the Neutralization Solution to avoid localized precipitation of bacterial cell debris. The neutralized bacterial lysate should become cloudy.
Centrifuge for 5 min to pellet cell debris and chromosomal DNA
Transfer the supernatant to the supplied GeneJET spin column by decanting or pipetting. Avoid disturbing or transferring the white precipitate.
Centrifuge for 1 min. Discard the flow-through and place the column back into the same collection tube.
Add 500 µL of the Wash Solution (diluted with ethanol prior to use) to the GeneJET spin column. Centrifuge for 30-60 seconds and discard the flow-through. Place the column back into the same collection tube.
Repeat the wash procedure using 500 µL of the Wash Solution
Discard the flow-through and centrifuge an additional 1 min to remove residual Wash Solution. This step is essential to avoid residual ethanol in plasmid preps.
Transfer the GeneJET spin column into a fresh 1.5 mL microcentrifuge tube. Add 50 µL of the Elution Buffer to the center of GeneJET spin column membrane to elute the plasmid DNA. Take care not to contact the membrane with the pipette tip. incubate for 2 min at room temperature and centrifuge for 2 min.
Discard the column and store the purified plasmid DNA at -20°C.
Analytik Jena innuPREP Plasmid Mini Kit (Centrifugation)
Starting material: 0.5-5.0 mL
Pellet cells
Centrifuge: max speed, 1 min
Discard supernatant completely
Resuspend cells
Add 250 µL Resuspension Buffer
Vortex
Lysis (do not vortex)
Add 250 µL Lysis Buffer
Mix: invert tube 6-8 times
Lysis time: <5 min
Neutralization
Add 350 µL Neutralization Buffer
Mix: invert tube 6-8 times
Centrifuge max speed, 8 min
Binding of DNA
Add Spin Filter to Receiver Tube
Add clarified sample Spin Filter
Centrifuge at 11.000 x g (12.000 rpm) 1 min
Washing
Add 500 µL Washing Solution A
Centrifuge 1 min at 11.000 g (12.000 rpm)
Add 700 µL Washing Solution B
Centrifuge 11.000 x g (12.000 rpm) 1 min
Remove ethanol
Discard filtrate
Add Spin Filter to Receiver Tube
Centrifuge Max speed, 2 min
Elution
Add Spin Filter to a 1.5 mL reaction tube
Add 50-100 µL Elution Buffer P
Incubation 1 min at room temperature
Centrifuge at 11.000 x g (12.000 rpm) 1 min
QIAprep Spin Miniprep Kit (Qiagen)
All centrifugation steps are carried out at 13.000 rpm (17.900m x g) in a conventional table-top microcentrifuge
Pellet 1-5 mL bacterial overnight culture by centrifugation at >8.000 rpm (6.800 x g) for 3 min at room temperature (15-25 °C)
Resuspend pelleted bacterial cells in 250 µL Buffer P1 and transfer to a microcentrifuge tube.
Add 250 µL Buffer P2 and mix thoroughly by inverting the tube 4-6 times until the solution becomes clear. Do not allow the lysis reaction to proceed for more than 5 min . If using LyseBlue reagent, the solution will turn blue.
Add 350 µL Buffer N3 and mix immediately and thoroughly by inverting the tube 4-6 times. If using LyseBlue reagent, the solution will turn colorless.
Centrifuge for 10 minutes at 13.000 rpm (17.900 x g) in a table-top microcentrifuge.
Apply the supernatant to the QIAprep spin column by decanting or pipetting. Centrifuge for 30-60 s and discard the flow-through.
Wash the QIAprep spin column by adding 500 µL Buffer PB. Centrifuge for 30-60 s and discard the flow-through
Wash the QIAprep spin column by adding 750 µL Buffer PB. Centrifuge for 30-60 s and discard the flow-through. Transfer the QIAprep spin column to the collection tube.
Centrifuge for 1 min to remove residual wash buffer.
Place the QIAprep column in a clean 1.5 mL microcentrifuge tube. To elute DNA, add 50 µL Buffer EB(10mM Tris-Cl, pH8.5) or water to the center of the QIAprep spin column, let it stand for 1 min, and centrifuge for 1 min.
Standard BioBrick Assembly
Modified from Silver Lab
Minimum size of the BioBricks for this methods is 150 bp, whereas it should be at least 500 bp bigger or smaller than the used backbone. The BioBricks, which complies with these conditions, is used as the insert and is assembled into the prefix or suffix of the other BioBrick called vector. The methods differ in prefix and suffix insertion.
Suffix insertion
Digestion of inserted Biobrick
Acquire > 700 ng DNA (10 µL volume)
Add 1 µL 10 x NEBuffer
Add 0.5 µL XbaI
Add 0.5 µL PstI
Digest 1 h at 37 °C
Inactivate 20 min at 80 °C
Clean up via gel electrophoresis, cut and extract from gel
Digestion of the vector
Acquire about 700 ng DNA (10 µL volume)
Add 1 µL 10 x NEBuffer 2.1
Add 0.5 µL SpeI
Add 0.5 µL  PstI
Digest 1 h at 37 °C
Inactivate 20 min at 80 °C
Add 1 µL Antarctic Phosphatase (AP) and 1.2 µL 10 x AP reaction buffer
Incubate 1 h at 37°C
Clean up with PCR clean up kit
Ligation
Mix 50-200 ng vector with 3-10 fold molar access of insert
Add 2 µL T4-Ligase-Buffer, 1 µL T4-DNA-Ligase
Incubate for a minimum 20-30 min at room temperature (we usually ligate over night)
Inactivate 5 min at 70 °C
Store at -20 °C or transform
Prefix insertion
Follow protocol of Suffix insertion, replace the restriction enzymes as following:
Digest insert with EcoRI and SpeI
Digest vector with EcoRI and XbaI
Transformation via Electroporation
Thaw 50 µL electrocompetent E. coli cells on ice, dilute with icecold 50 µL glycerine (10%) if necessary
Add 0.5-5 µL plasmid to 50 µL electrocompetent cells
Store cells on ice for 1 minute
Electroporate at U = 2.5 kV, C = 25 µF, R = 400 Ω
Transfer transformation reaction to 450 µL SOC-Medium and incubate 1 h at 37 °C
Plate on selective LB-Medium
Incubate over night at 37 °C
Transformation via Heat Shock
Thaw 100 µL chemo competent E. coli cells on ice
Add 0.5-5 µL plasmid to 100 µl chemocompetent cells
Store cells on ice for 10-30 min on ice
Heat shock for 90 seconds at 42 °C
Store reaction on ice for 60 seconds
Optional: Preheat SOC medium to 37 °C
Transfer reaction to 1 ml SOC medium and incubate at 37 °C for at least 1 hour
Centrifuge 3 minutes at 12000 rpm and plate on selective LB medium
Incubate at 37 °C over night
Other protocols
SDS-PAGE
Pouring the polyacrylamide gel:
For 2 resolving gel (14 %) :
4.7 mL  Bisacrylamid/Acrylamid (0.8 %; , 30 %;, at the ratio of 37.5:1)
2.7 mL H2O
2.5 mL 0.5 M TRIS-HCl, pH 6.8
0.1 mL 10% SDS
Add 50 µL 10 % ammonium persulfate and 10 µL TEMED and mix.
Pour the solution quickly into the gel casting form. Leave about 2 centimeters below the bottom of the comb for the stacking gel.
Layer isopropanol on top of the gel.
Leave the resolving gel at room temperature for 30-45 minutes to polymerize.
Remove isopropanol and wash the surface of the separating gel with H2O. Wait until the surface is dry.
For 2 stacking gel (4 %) :
1.3 mL Bisacrylamid/Acrylamid (0.8 %; , 30 %;, at the ratio of 37.5:1)
6.1 mL H2O
2.5 mL 0.5 M TRIS-HCl, pH 6.8
0.1 mL 10% SDS
50 µL 10 % ammonium persulfate
5 µL TEMED
Add 50 µL 10 % ammonium persulfate and 5 µL TEMED to each aliquote and mix.
Insert comb without getting bubbles stuck underneath.
Leave the gel at room temperature for 30-45 minutes to polymerize.
For storage:
Remove sealing and store the gel wrapped in moistened paper towel at 4 °C.
Preparing the sample:
Mix your protein mixture 3:1 with PBJR buffer (15 µL protein solution + 5 µL PBJR buffer )
Heat for 5 minutes at 95 °C
Running the gel:
Remove sealing, put the polymerized gel into gel box and pour 1x SDS-PAGE running buffer into the negative and positive electrode chamber.
Remove comp without destroying the gel pockets.
Pipet the SDS running buffer in the gel pockets up and down for flushing the gel pockets.
Pipet slowly 20 µL of the sample into the gel pockets.
Make sure to include at least one lane with molecular weight standards (PageRuler Prestained Protein Ladder™ (Fa. Fermentas)) to determinate the molecular weight of the sample.
Connect the power lead and run the gel with 180 V until the distance of the lowest molecular weight standard lane to the gel end is down to 0.5 cm.
Staining the polyacrylamide gel (Colloidal Coomassie Brilliant Blue staining):
After finishing the SDS-PAGE remove gel from gel casting form and transfer it into a box.
Add 100 mL of the Colloidal Coomaassie Brilliant Blue staining solution to your polyacrylamid gel.
Incubate the gel in the solution at room temperature until the protein bands got an intensive blue color. Shake the gel continuously during incubation.
Remove the staining solution.
Wash the gel with destaining buffer .
Incubate the gel in H2O (2-6 h) for bleaching the background. Shake the gel continuously during incubation.  If necessary replace the colored water with new one.
MALDI-TOF/TOF sample preparation
Tryptic digest of gel lanes for analysis with MALDI-TOF:
Be careful that skin scales or hair do not contaminate your sample. So wear hand gloves and tie up your hair.
Reaction tubes have to be cleaned with 60 % (v/v) CH3CN and 0.1 % (v/v) TFA. Afterwards the solution has to be removed completely followed by evaporation of the tubes under a fume hood. Alternatively, microtiter plates from Greiner® (REF 650161) can be used without washing. If you work with reaction tubes from Eppendorf, you do not need the wash step either.
Cut out the protein lanes of a Coomassie-stained SDS-PAGE using a clean scalpel. Gel parts are transferred to the washed reaction tubes. If necessary, cut the parts to smaller slices
Gel slices should be washed two times. Therefore add 200 µL 30 % (v/v) acetonitrile in 0.1 M ammonium hydrogen carbonate each time and shake lightly for 20 minutes till the gel slices are destained. Remove supernatant and discard to special waste
Dry gel slices at least 30 minutes in a Speedvac.
Rehydrate gel slices in 15 µL trypsin solution followed by short centrifugation.
Trypsin-solution : 1 µL trypsin + 14 µL 10 mM NH4HCO3
For this solution solubilize lyophilized trypsin in 200 µl of provided buffer and activate Trypsin for 15 minutes at 30 °C. For further use it can be stored at -20 °C.
Gel slices have to be incubated 30 minutes at room temperature, followed by incubation at 37 °C overnight
Dry gel slices at least 60 minutes in a Speedvac.
According to the size of the gel slice, add 5 - 20 µL 50 % (v/v) ACN / 0.1 % (v/v) TFA
Samples can be used for MALDI measurement or stored at -20 °C
Preparation and Spotting for analysis of peptides on Bruker AnchorChips:
Spot 0.5 - 1 µL of sample aliquot
Add 1 µL HCCA matrix solution to the spotted sample aliquots. Pipet up and down approximately five times to obtain a sufficient mixing. Be careful not to contact the AnchorChip. Note: Most of the sample solvent needs to be gone in order to achieve a sufficiently low water content. When the matrix solution is added to the previously spotted sample aliquot at a too high water content in the mixture, it will result in undesired crystallization of the matrix outside the anchor spot area.
Dry the prepared spots at room temperature
Spot external calibrants on the adjacent calibrant spot positions. Use the calibrant stock solution (Bruker’s “ Peptide Calibration Standard II ”, Part number #222570), add 125 µL of 0.1 % TFA (v/v) in 30 % ACN to the vial. Vortex and sonicate the vial.
Mix the calibrant stock solution in a 1:200 ratio with HCCA matrix and deposit 1 µL of the mixture onto the calibrant spots.
Analyze samples in ultrafleXtreme by Bruker Daltonics.
Phagemid display
Preparing immune-tubes for phage panning:
Dilute 45 μg of your target protein in 1.5 mL 1 M NaHCO3
Incubate rolling over night at 4 °C and discard the rest
Coat free surfaces by incubating 1 h filled with 5 mg/mL BSA
Wash three times with PBST
Store at 4 °C in the dark
Phagemid display:
Insert a variable library of binding protein CDSs in the phagemid vector pAK100 by Gibson or restriction cloning
Transformate the vector in E. coli ER2738
Remaining colonies form the phagemid library, screened for binders in the following procedure
Wash up all colonies with 5 mL LB and inoculate 200 mL LB, containing chloramphenicol, up to OD 0.1
Incubate the culture, shaking at 37°C, up to OD 0.5
Add 10 x Phages per E. coli cell to infect the bacteria
Incubate 1.5 h at 37°C, then add kanamycin
Incubate further 1.5 h at 37°C
Centrifuge at 5,000 g for 10 min and decant the supernatant
Add 25 %vol PEG NaCl and mix well
Precipitation of produced phages at 4 °C, over night
Centrifuge at 12,000 g and 4°C for 30 min and discard the supernatant
Resuspend the pellet with 3 mL PBS and centrifuge at 13,000 g and room temperature for 10 min and decant the supernatant
Add 20 %vol PEG NaCl and precipitate for 1 h on ice
Centrifuge at 4 °C and 13,000 g for 10 min and discard supernatant
Resuspend the pellet in 500 μL PBS
Centrifuge at 13,000 g and room temperature for 10 min
Measure UV-VIS absorbance of an 1:50 aliquot at 269 nm and 320 nm to calculate the phage concentration
Add 1x1012 to 5x1012 Phages diluted in 1 mL PBST containing 1 mg/mL BSA on the prepared immune-tubes and incubate rolling at 4°C for 1 h
Wash the immune-tubes carefully with PBST for at least 10 times and discard the washing buffer
Eluate remaining phages by adding 1 mL Glycine-HCl, containing 1 mg/mL BSA by incubating rolling for 10 min at room temperature
Neutralize the eluate by adding 150 μL TRIS at pH 9
Add 10 μL eluted phages to 140 μL of E. coli ER2738 at OD 0.5, incubate for 10 min at room temperature and streak out at LB agar containing tetracycline and chloramphenicol
Electrophoretic Mobility Shift Assay (EMSA)
Required solutions:
5x EMSA buffer (100 mM Na2HPO4, 375 mM KCl, 25 % Glycerol, adjust ph to 8.0)
EMSA running buffer (20 mM Na2HPO4, adjust ph to 8.0)
3 % agarose gel prepared with EMSA running buffer
MgCl2: 25 mM
EDTA: 2,5 mM
BSA: 10 mg/ml
Dithiotreitol
salmon sperm
Cy3 labeled DNA
protein
Reaction:
0.05 pmol Cy3 labeled DNA
5 pmol purified protein of interest
4 µl 5x EMSA buffer
2 µl MgCl2 final concentration: 2.5 mM
0.8 µl EDTA final concentration: 0.1 mM
1 µl BSA final amount: 10 µg
1 µl salmon sperm final amount: 1 µg
optional: 5 mM DTT
mix all components of the reaction carefully and add up with water to 20 µl H2O.
Incubate at room temperature in the dark for 15 min.
Keep in mind that Cy3 is light sensitive, always keep your samples in the dark after addition of the DNA fragment.
Apply reaction to 3 % agarose gel.
Run gel at 80 V for approximately 80 min at 4 °C. (Adjust voltage and duration to size of your DNA fragement.)
Detect Cy3 with a fluorescence scanner. In our case it was the Typhoon 8600 variable Mode Imager.
CRISPR/Cas9-Knockout
Design of the guide RNA: 20 nt of interest with 3' protospacer adjacent sequence (PAM) NGG, where N is any nucleotide. Preference is given to:
sequences with purines occupying the last four (3') bases of the protospacer.
sequences on the non-coding strand.
sequences in which the last 12 nt protospacer + 3 nt PAM (15 nt total) are unique in the genome (check by BLAST with all four possible NGG sequences)
For knock-out of the rpoZ gene of E. coli the following sequence can be used:
5'- CGACCAGTACCAGGTCAAAACGG -3'
Design two 24 nt oligonucleotide (4 nt 5' sticky end + 20 nt spacer sequence) with the sticky ends ACGC on the forward primer and AAAC on the reverse primer. For the rpoZ knock-out the two primers are:
Guide-fw: 5'-ACGCCGACCAGTACCAGGTCAAAA-3'
Guide-rv: 5'-AAACTTTTGACCTGGTACTGGTCG-3'
The sticky ends are designed for the BbsI sites.
For single spacer, anneal single spacer oligos as follows:
Resuspend both oligos to 100µM in water
Mix 5µl Guide-for + 5µl Guide-rev + 90 µl 30mM HEPES, pH 7.8
Heat to 95°C for 5min, then ramp to 4°C at 0.1°C/sec
Insert annealed spacer by Golden Gate assembly,
Golden Gate reaction mixture:
Backbone
X µl
100 ng
Insert
0.3 µl
From 10-fold diluted annealed oligo stock
T4 Ligase Buffer (NEB)
2 µl
T4 Ligase (NEB)
1 µl
400 U/µl stock is sufficient; add last
BbsI
1 µl
Stored at -80°C
H2O
Y µl
In sum
20 µl
Golden Gate Program:
37°C , 10 min
16°C , 10 min
Goto step 1, 9 times
50°C , 5 min
65°C , 20 min
4°C , forever
Transform 3 µl of each reaction to E. coli DH4α by heat shock(manufacturer's protocol) or electroporation(manufacturer's protocol)
Plate 10% of recovery culture on selective plates with 10 µl of 0.5M IPTG and 40 µl of 20 mg/ml Bluo-gal (in DMSO).
Pick white colonies to selective LB (for pTarget use streptomycine) and recover plasmid.
CRISPR/Cas knock-out/knock-in:
Transformation of pCas9 in willed knock-out strain via electroporation:
Preheat SOC medium with 0.1% arabinose by 30°C
Thaw 50 µL electrocompetent E. coli cells (the knock-out/knock-in strain)on ice, dilute with icecold 50 µL glycerine (10%) if necessary
Add 1-5 µL plasmid to 50 µL electrocompetent cells
Store cells on ice for 1 minute
Electroporate at U = 2.5 kV, C = 25 µF, R = 400 Ω
Transfer transformation reaction to 950 µL prepared SOC-Medium and incubate 1 h at 30 °C
Plate on LB-Medium prepared with kanamycine and 0.1% arabinose
Incubate over night at 30 °C
Create electrocompetent cells of the knock-out/knock-in strain carrying the pCas plasmid. ATTENTION: cultivation of the cells by 30°C in LB with 0,1% arabinose!
PCR with the BioBrick BBa_K2082251 :
Q5 master mix (25 µL):
5 µL 5 x Q5 buffer
0.5 µL 10 mM dNTPs
Up to 50 ng/µL-template ( BBa_K2082251 )
0.1 µL Q5 DNA polymerase
0.5 µl forward-primer: CC_FragAB_Reporter_fw
0.5 µl reverse-primer: CC_FragAB_Reporter_rv
Ad 25 µL dH2O
Q5 standard protocol:
Initial denaturation: 98°C - 30 s
30-35 cycles of:
Denaturation: 98 °C - 20 s
Annealing: 58 °C - 30 s
Elongation: 72 °C - 20 s/kb
Final elongation: 72 °C - 5 min
Storage: 4 °C
Purification of the PCR product (manufacturer's protocol)
Transformation of the purified PCR product and the guide-RNA-carrying plasmid pTarget in the pCas-carrying competent knock-out/knock-in strain of E. coli via electroporation
Preheat SOC medium with 0.1% arabinose by 30°C
Thaw 50 µL electrocompetent pCas-carrying E. coli cells (the knock-out/knock-in strain)on ice, dilute with icecold 50 µL glycerine (10%) if necessary
Add 1-3 µL pTarget plasmid to 50 µL electrocompetent cells
Add 2 µL purified PCR product to 50 µL electrocompetent cells
Store cells on ice for 1 minute
Electroporate at U = 2.5 kV, C = 25 µF, R = 400 Ω
Transfer transformation reaction to 950 µL prepared SOC-Medium and incubate 1 h at 30 °C
Plate first half of the transformed cells on LB-Medium prepared with kanamycine, streptomycine and 0.1% arabinose
Plate second half of the transformed cells on LB-Medium prepared with kanamycine, streptomycine and 0.1% arabinosen + 20µg/ml ampicillin
Incubate over night at 30 °C
Pick colonies from the ampicillin-prepared LB plate and accomplish a colony PCR:
5 µL 5x GoTaq buffer (Promega)
1 µL MgCl2 (25 mM stock)
0.5 µL dNTPs (10 mM each)
0.25 µL primer mix (prefix/suffix primers or sequencing primers) 100 mM
17.625 µL ddH2O
0.125 µL GoTaq polymerase (Promega)
0.5 µL template
PCR program
Cell lysis and denaturation: 5 min, 95 °C
30 cycles
Denaturation: 10 s, 95 °C
Hybridisation: 30 s, annealing temperature
Elongation: 60 s/kb of product, 72 °C
Final elongation: 5 min, 72 °C
Template alternatives
Pick a colony with sterile tip, elute in 100 µL ddH2O or buffer, store at 4 °C during PCR, plate if insert is of correct size
Pick colony, streak at marked position on a new plate and solute remaining cells on the tip in the PCR tube with reaction mixture, cultivate if insert is of correct size
Gel electrophoresis for control of fragment size
Colonies with a fragment size of about 3,000 bp are the cells, with knock-out of the E. coli rpoZ gene and knock-in of the reporter BBa_K2082238
Bradford Assay
Pipette 100 µl each of sample buffer, standard  solutions and samples into clean test tubes.
Add  5 ml 1X - staining solution
Mix by inverting repeatedly
Measure the OD 595 of the standard solutions  and samples after 5 to 30 mins against the  reference (sample buffer in 1X - staining dye).
Plot and compare OD 595 of standard solutions  to amount of protein u sed. The amount of  protein in sample can be read on calibration
Intein Mediated Purification with an Affinity Chitin-binding Tag (IMPACT)
Plasmid preparation
For a C-termninal fusion of your protein with the Intein Tag use the pTXB1 vektor, for a N-terminal fusion use the pTYB11 vektor.
Cloning of the protein of interest directly before (pTXB1) or behind (pTYB11) the intein tag.
Finished vektor with elektroporation or chemical transformation in E. coli expression strain ER2566 for cultivation
Cell growth
Preculture: Inoculate some cells in 10 ml LB-medium (+ 200 µg/ml Ampicillin), growth by 37°C over nigth
Main cultivation: Inoculate 250 ml LB (+ 200 µg/ml Ampicillin) with the preculture on an OD600 of 0.1. Growth by  37°C until it reaches an OD600 between 0.5 and 0.8. Addition of 0.5 mM IPTG and growth over night at 19°C.
Cell harvesting
Transfer of all cells to a big centrifuge pot and centrifugation for 10-20 min at 4°C and 4,500 rpm.
Resuspend the pellet in 20-30 ml Impact Lysis buffer and transfer it to BlueCaps.
Cell disruption with the French Press (at least 2 times; max volume 35-40 ml) and keep the suspension on ice.  Store 50 µl of the suspension at -20°C for SDS-PAGE.
Centrifugation of the cell remains (4,500 rpm, 1 h, 4°C) and collect the supernatant in BlueCaps on ice. Store 50 µl of the suspension at -20°C for SDS-PAGE.
Preparation of the columns
Seal the bottom of the column with Parafilm and pipet 5 ml water in it. Mark the fill level and let the water run out
Fill the column with chitin beads up to the marked place and put a filter designator on the material.
Equilibrate the column with 10 volumes of Impact column buffer (Low-Salt; only 2.92 g on 1 L instead of 58.44 g).
Loading of the column
Give the clean supernatant of the cell harvesting on the column and let it flow through the column very slowly (flow through rate of a maximum of 0.5-1 ml/min). Store 50 µl of the flow through at -20°C for SDS-PAGE.
Washing of the column
Wash the column with 10 volumes of Impact column buffer (High-Salt).
Induction of the cleavage
Washing of the column with 3 volumes of Impact cleavage buffer . This step should not take more time than 30-60 min.
Cleavage for 16-40 h at 23°C in the dark
For better cleavage the amino acid which is fused with the intein tag should be Tyr, Phe, Gln, Asn, Thr, Lys, Ala, His, Leu or Met for C-terminal cleavage or Met, Ala, Gln, Gly, Leu, Asn, Trp, Phe or Tyr for N-terminal cleavage.
Elution
Elution with 1-2 volumes of Impact column buffer (Low-Salt) in fresh BlueCap. Store 50 µl of the elution at -20°C for SDS-PAGE.
Usage of a suitable protein filter to get a higher protein concentration. Centrifugation for 45 min at 4500-5000 rpm and 4°C until you have 250 µl left.
Wash the protein solution 2-3 times with the Impact protein-wash buffer .
Bradford-Assay to calculate the protein concentration.
SDS-PAGE with 8 µg of the protein mixtures and 2 µg of the eluate
The protein can be stored with 50% glycerin at -20°C.
Western Blot
Transfere
Run a SDS gel
Soak 6 filter papers and a nitrocellulose membrane in your transfere buffer for 15 minutes.
Stack 3 filter papers, the membrane, the SDS gel and another 3 filter papers in your semi-dry blotter. The nitrocellulose membrane should be nearest to the positive electrode.
Blot the gel to the membrane according to the specifications of your blotter.
Mark your Protein Ruler with a pencil, so it will be visible after staining the membrane.
Staining
Block the membrane for 1 hour in transfere buffer with 3% BSA.
Dilute your primary antibody in transfere buffer with 3% BSA and incubate the membrane with it for 1.5 hours.
Wash the membrane 5 times for 5 minutes each with fresh transfere buffer.
Dilute your secondary antibody in transfere buffer with 3% BSA and incubate the membrane with it for 1 hour.
Wash the membrane 5 times for 5 minutes each with fresh transfere buffer.
For chemiluminescent detection, follow the instructions for the reagent you are using. Acquire image.
Bio-layer interferometry (BLItz)
Immobilisation
Activate sensor by incubating it in 250 µL EDC/NHS solution for 2 minutes.
Immobilize your first protein on the sensor by incubating the sensore in 250 µL adequate diluted protein solution for 5 minutes.
Wash the sensor by incubating it in 250 µL of the buffer you are using to dilute your protein for 2 minutes.
Measuring
Measure a baseline by incubating the sensor in 250 µL of your dilution buffer for 5 minutes.
Incubate the sensor in 250 µL adequate diluted protein solution of your second protein for 10 minutes to measure the association of your second to your first protein.
Wash the sensor in 250 µL of your dilution buffer for 5 minutes to measure the dissociation of your second protein from your first protein.
To reactivate the sensor, incubate it for 5 minutes in 250 µL 1M NaOH.
