<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Experiments_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Experiments/Protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container main"><DIV id="header"><NAV id="navigationbar" class="navbar navbar-default custom_navbar_style navbar-fixed-top"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="#">NavBar</A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-center"><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description"> PROJECT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description">Description</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds">Scaffolds</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design">Design and Construction</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/CreateYours">Create your own Library</A></LI></UL><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/EpPolI">Error prone PolymeraseI</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/Global">Genome wide mutator</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">Bacterial Two-Hybrid System</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/SplitBetaLactamaseHitchHiker">Alternative Selection System</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_I">Model I</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_II">Model II</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results"> RESULTS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Phage">Phagemid Display</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">Reversion</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/Cloning">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/ExpressionControl">Expression Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/InteractionControl">Interaction Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/BindingControl">Binding Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/KnockOutKnochIn"><I>in vivo</I> Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Modeling">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Fermentation">Fermentation</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Best Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Best Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Best Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts/Improve">Improve a part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Demonstrate your Work</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Medal_Fulfillment">Medal Fulfillment</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold"> HUMAN PRACTICES </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Dictionary">Dictionary</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks">Educational Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Crowdfunding">Crowdfunding and Press</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Achievements</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations"> COLLABORATIONS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Duesseldorf">Düsseldorf</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Lethbridge">Lethbridge</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Freiburg">Freiburg</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments"> NOTEBOOK </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook">Lab Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">Primers</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols">Lab Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Media">Media and Buffers</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team"> TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team">Team Profile</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Members">Members</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Contact">Contact</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement"> ACKNOWLEDGEMENT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Sponsors_And_Partners">Sponsors and Partners</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></DIV></DIV></NAV></DIV></DIV><DIV class="container main"><DIV class="jumbotron" style="background-image: url(https://static.igem.org/mediawiki/2016/a/ab/Bielefeld_CeBiTec_2016_10_14_X_notebook.png)"><DIV class="jumbotron-text"><H1 style="margin-bottom: 0px; text-align:left">Protocols</H1><H2 style="color:#ffffff; text-align:left">Used Protocols</H2></DIV></DIV><DIV class="container text_header"><H3>Protocols</H3></DIV><DIV class="container text"><H1>Standard protocols</H1><DIV class="panel-group scientific" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#BluntEndLigation">Blunt end ligation</A></H4></DIV><DIV id="BluntEndLigation" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI> For &quot;intramolecular&quot; ligation of a PCR product the 5'-ends are phosphorylated with T4 Polynucleotidekinase (PNK)</LI><LI> Set up the following reaction:</LI><TABLE><TBODY><TR><TH>reagent</TH><TH>volume [ µL]</TH></TR><TR><TD>10x T4 DNA ligase buffer</TD><TD>2.5</TD></TR><TR><TD>T4 PNK </TD><TD>1</TD></TR><TR><TD>PEG 4000 50% (w/v)</TD><TD>2.5</TD></TR><TR><TD>DNA (100 to 600 ng)</TD><TD>x </TD></TR><TR><TD>ddH<SUB>2</SUB>O </TD><TD>to 25 </TD></TR></TBODY></TABLE><LI> Incubate at 37 °C, 30 min</LI><LI> Heat inactivate at 65 °C, 20 min</LI><LI> Add 1 µL T4 DNA ligase after reaction has cooled down to room temperature </LI><LI> Incubate at room temperature for at least 2 h, overnight also works. </LI><LI> Next step: <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#HeatShock">Transformation via heat shock</A></LI></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#gotaq_colony_pcr">Colony PCR</A></H4></DIV><DIV id="gotaq_colony_pcr" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>One Reaction mix contains</LI><UL><LI>5 µL 5x GoTaq buffer (Promega)</LI><LI>1 µL MgCl<SUB>2</SUB> (25 mM stock)</LI><LI>0.5 µL dNTPs (10 mM each) </LI><LI>0.25 µL primer mix (prefix/suffix primers or sequencing primers) 100 mM</LI><LI>17.625 µL ddH<SUB>2</SUB>O </LI><LI>0.125 µL GoTaq polymerase (Promega)</LI><LI>0.5 µL template </LI></UL><LI>PCR program </LI><UL><LI>Cell lysis and denaturation: 5 min, 95 °C</LI><LI>30 cycles</LI><UL><LI>Denaturation: 10 s, 95 °C </LI><LI>Hybridisation: 30 s, annealing temperature </LI><LI>Elongation: 60 s/kb of product, 72 °C </LI></UL><LI>Final elongation: 5 min, 72 °C </LI></UL><LI>Template alternatives </LI><UL><LI>Pick a colony with sterile tip, elute in 100 µL ddH<SUB>2</SUB>O or buffer, store at 4 °C during PCR, plate if insert is of correct size</LI><LI>Pick colony, streak at marked position on a new plate and solute remaining cells on the tip in the PCR tube with reaction mixture, cultivate if insert is of correct size </LI></UL><LI>Gel electrophoresis for control of fragment size </LI></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#DNAPurification">DNA Purification</A></H4></DIV><DIV id="DNAPurification" class="panel-collapse collapse"><DIV class="panel-body">
              
We used the following kits for DNA purification. For each kit the specific protocol is described seperately:  <DIV class="panel-group inneraccordion" id="accordion2"><DIV class="panel panel-default"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion2" href="#collapseDNAPurificationA"><U>Thermo Scientific GeneJET Gel Extraction Kit   	</U></A></H4><DIV id="collapseDNAPurificationA" class="panel-collapse collapse in"><DIV class="panel-body"><UL><LI>Note: all centrifugations should be carried out in a table-top microcentrifuge at &gt; 12000 x g/10.000-14.000 rpm</LI><LI>Preparation</LI><UL><LI>Excise gel slice containing the DNA fragment using a clean scalpel or razor blade. Cut as close to the DNA band as possible to minimize the gel volume. Place the gel slice into a pre-weighed 1.5 mL tube. Record the weight of the gel slice. Note: If the purified fragment will be used for cloning reactions, avoid damaging the DNA through UV light exposure. Minimize UV exposure to a few seconds or keep the gel slice on a glass or plastic plate during UV illumination. </LI><LI>Add 1:1 volume of Binding Buffer to the gel slice (volume:weigt)</LI><LI>Incubate the gel mixture at 50-60 °C for 10 min or until the gel slice is completely dissolved. Mix the tube by inversion every few minutes to facilitate the melting process. Ensure that the gel slice is completely dissolved. Vortex the gel buffer mixture briefly before loading on the column. </LI><LI>Check the color of the solution. A yellow color indicates an optimal pH for DNA binding. If the color of the solution is orange or violet, add 10 µL of 3 M sodium acetate, pH 5.2 solution and mix. The color of the mix will become yellow. </LI><LI><I>Optional</I>: Use this step only when DNA fragment is &lt;500 bp or &gt;10 kb long. 
</LI><LI>If the DNA fragment is &lt;500 bp, add 1:2 volume of 100 % isopropanol to the solubilized gel solution. Mix thoroughly.</LI><LI>If the DNA fragment is &gt;10 kb, add 1:2 volume of water to the solubilized gel solution. Mix thoroughly. </LI></UL></UL><LI>Binding of DNA</LI><UL><LI>Transfer up to 800 µL of the solubilized gel solution (from step 3/4) to the GeneJET purification column. Centrifuge 1 min at 11,000 x g. Discard the flow-through and place the column back into the same collection tube. Note: If the total volume exceeds 800 µL, the solution can be added to the column in stages. After each application, centrifuge the column for 30-60 s and discard the flow through after each spin. Repeat until the entire volume has been applied to the column membrane. Do not exceed 1 g of total agarose gel slice per column. </LI><LI>Add 100 µL of Binding Buffer to the GeneJet purification column. Centrifuge for 1 min. Discard the flow-through and place the column back into the same collection tube.</LI></UL><LI>Washing</LI><UL><LI>Add 700 µL of Wash Buffer to the GeneJet purification column. Centrifuge for 1 min. Discard the flow-through and place the column back into the same collection tube. (We altered the protocol to to wash steps a 350 µL optional)</LI><LI>Centrifuge the empty GeneJet purification column for an additional 1 min to completely remove residual wash buffer </LI></UL><LI>Elution</LI><UL><LI>Transfer the GeneJET purification column into a clean 1.5 mL microcentrifuge tube. Add 50 µL of Elution Buffer to the center of the purification column membrane. Centrifuge for 1 minute. Note: for low DNA amounts the elution volumes can be reduced to increase DNA concentration. An elution volume between 20-50 µL does not significantly reduce the DNA yield. However, elution volumes less than 10 µL are not recommended. If DNA fragment is &gt;10 kb, preheat Elution Buffer to 65 °C before applying to column. If the elution volume is 10 µL and DNA amount is &lt;5 µg, incubate column for 1 min at room temperature before centrifugation.(We altered the protocol to 30 µL Elution Buffer preheated to 60 °C and 1 min incubation before centrifuging for all sizes.) </LI><LI>Discard the GeneJet purification column and store the purified DNA at -20 °C. </LI></UL></DIV></DIV></DIV><DIV class="panel panel-default"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion2" href="#collapseDNAPurificationB"><U>Qiagen QIAquick Gel Extraction Kit (Centrifugation)   	</U></A></H4><DIV id="collapseDNAPurificationB" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Note: All centrifugation steps are carried out at 17.900 x g (13.000 rpm) in a conventional table-top microcentrifuge </LI><LI>Preparation</LI><UL><LI>Excise the DNA fragment from the agarose gel with a clean, sharp scalpel </LI><LI>Weigh the gel slice in a colorless tube. Add 3 volumes of Buffer QG to 1 volume gel (100 mg gel - 100 µL). The maximum amount of gel per spin column is 400 mg. For &gt;2 % agarose gels, add 6 volumes Buffer QG</LI><LI>Incubate at 50°C for 10 min (or until the gel slice has completely dissolved). Vortex the tube every 2-3 minutes to help dissolve the gel. After the gel slice has dissolved completely, check that the color of the mixture is yellow(similar to Buffer QG without dissolved agarose). If the color of the mixture is orange or violet, add 10 µL of 3 M sodium acetate, pH 5.0, ad mix. The mixture turns yellow. </LI></UL><LI>Binding of DNA</LI><UL><LI>Add 1 volume isopropanol to the sample and mix.</LI><LI>Place a QIAquick spin column in a provided 2 mL collection tube. To bind DNA, apply the sample to the QIAquick column and centrifuge for 1 min until all the samples have passed through the column. Discard the flow-through and place the QIAquick column back into the same tube. For sample volumes &gt;800 µL, load and spin again. </LI></UL><LI>Washing</LI><UL><LI>If DNA will subsequently be used for sequencing, </LI><LI>in vitro</LI> transcription or microinjection, add 500 µL Buffer QG to the QIAquick column and centrifuge for 1 min. Discard flow-through and place the QIAquick column back into the same tube <LI>To wash, add 750 µL Buffer PE to QIAquick column and centrifuge for 1 min. Discard flow-through and place the QIAquick column back into the same tube. If the DNA will be used for salt-sensitive applications (sequencing, blut-ended-ligation) let the column stand 2-5 minutes after addition of Buffer PE. Centrifuge the QIAquick column in the provided 2 mL collection tube for 1 min to remove residual wash buffer </LI><LI>Place QIAquick column into a clean 1.5 mL microcentrifuge tube </LI></UL><LI>Elution</LI><UL><LI>To elute DNA, add 50 µL Buffer EB (10 mM TRIS-HCl, pH 8.5) or water to the center of the QIAquick membrane and centrifuge the column for 1 min. For increased DNA concentration, add 30 µL Buffer EB to the center of the QIAquick column, let the column stand for 1 min, and then centrifuge for 1 min. After the addition of Buffer EB to the QIAquick membrane, increasing the incubation time up to 4 min can increase the yield of purified DNA.</LI><LI>If the purified DNA is to be analyzed on a gel, add 1 volume of loading dye to 5 volumes of purified DNA. Mix the solution by pipetting up and down before loading the gel. </LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion2" href="#collapseDNAPurificationC"><U>Analytik Jena InnuPREP Gel Extraction Kit   	</U></A></H4><DIV id="collapseDNAPurificationC" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Preparation</LI><UL><LI>Excise the DNA fragment from the agarose gel with a sharp scalpel. Note: minimize the agarose gel slice. Check the weight. Do not use more than 300 mg gel slice for one spin filter.</LI><LI>Transfer the gel slice into a 1.5 mL or 2.0 mL reaction tube and add the 650 µL Gel Solubilizer. </LI><LI>Incubate for 10 min at 50°C until the agarose gel slice is completely dissolved. </LI></UL><LI>Binding of DNA </LI><UL><LI>Add 50 µL Binding Optimizer and mix the suspension by vortexing or piepetting sometimes up and down. </LI><LI>Apply the sample onto the spin filter (green) located in a 2.0 mL Receiver Tube. Close the cap and centrifuge at 10.000 x g (12.000 rpm) for 1 min. Discard the filtrate and re-use the Receiver Tube. Place the Spin Filter back into the 2.0 mL Receiver Tube.</LI></UL><LI>Washing</LI><UL><LI>Open the Spin Filter and add 700 µL Washing Solution LS, close the cap and centrifuge at 10.000 x g (12.000 rpm) for 1 min. Discard the filtrate and re-use the Receiver Tube. Place the Spin Filter back into the 2.0 mL Receiver Tube. Repeat the entire step. </LI><LI>Centrifuge at max speed for 2 min to remove all traces of ethanol. Discard the 2.0 mL Receiver Tube.</LI></UL><LI>Elution</LI><UL><LI>Place the Spin Filter into a 1.5 mL elution Tube. Carefully open the cap of the Spin Filter and add 30-50 µL Elution Buffer (optionally preheated to 50°C). Incubate at room temperature for 1 min. Centrifuge at 6.000 x g (8.000 rpm) for 1 min. A second elution step will increase the yield of extracted DNA. Note: The DNA can be eluted with a lower or a higher volume of Elution Buffer. (...) Store the extracted DNA at +4 °C. For long time storage placing at -20°C us recommended </LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion2" href="#collapseDNAPurificationD"><U>Promega Wizard SV Gel and PCR CleanUp System   	</U></A></H4><DIV id="collapseDNAPurificationD" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Gel Slice/PCR product preparation</LI><UL><LI>Following electrophoresis, excise DNA band from gel and place gel slice in a 1.5 mL microcentrifuge tube.</LI><LI>Add 10 µL of Membrane Binding solution per 10 mg of gel slice. Vortex and incubate at 50-65 °C until gel slice is copletely dissolved.</LI><LI>Alternative for PCR amplifications: Add an equal volume of Membran Binding Solution to the PCR amplification </LI></UL><LI>Binding of DNA </LI><UL><LI>Insert SV Minicolumn into Collection Tube.</LI><LI>Transfer dissolved gel mixture or prepared PCR product to the Minicolumn assembly. Incubate at room temperature for 1 min. </LI><LI>Centrifuge at 16.000 x g for 1 minute. Discard flow through and reinsert Minicolumn into Collection Tube.</LI></UL><LI>Washing </LI><UL><LI>Add 700 µL Membrane Wash Solution (ethanol added). Centrifuge at 16.000 x g for 1 minute. Discard flow-through and reinsert Minicolumn into Collection Tube.</LI><LI>Repeat previous Step with 500 µL Membrane Washing Solution. Centrifuge at 16.000 x g for 5 minutes.</LI><LI>Empty the Collection Tube and recentrifuge the column assembly for 1 minute with the microcentrifuge lid open (or off) to allow evaporation of any residual ethanol. </LI></UL><LI>Elution</LI><UL><LI>Carefully transfer Minicolumn to a clean 1.5 mL microcentrifuge tube.</LI><LI>Add 50 µL of Nuclease-Free Water to the Minicolumn. Incubate at room temperature for 1 minute. Centrifuge at 16.000 x g for 1 minute. </LI><LI>Discard Minicolumn and store DNA at 4 °C or -20°C. </LI></UL></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#gibson_assembly">Gibson Assembly</A></H4></DIV><DIV id="gibson_assembly" class="panel-collapse collapse"><DIV class="panel-body"><LI>Modified from Gibson et al. (2009) </LI><LI>This assembly method is an isothermal, single-reaction method for assembling multiple overlapping DNA molecules. By coordinating the activity of a 5' exonuclease, a DNA polymerase and a DNA ligase two adjacent DNA fragments with complementary terminal sequence overlaps can be joined into a covalently sealed molecule, without the use of any restriction endonuclease.</LI><LI>Preparation of DNA molecules for <I>in vitro </I> recombination </LI><UL><LI> Generate the complenentary sequence overlaps by PCR using the Phusion DNA-polymerase. If necessary add 5 M Betain in the reaction mix by reducing the amount of H<SUB>2</SUB>O to decrease the number of false PCR products. </LI><LI>Seperate the PCR product of interest by gel electrophoresis, extract the products via <A href="https://2015.igem.org/team:bielefeld-cebitec/protocols#dnapurification" target="blank">Gel Extraction</A>. </LI></UL><LI><I>In vitro</I> recombination </LI><UL><LI>Preparing assembly mixture </LI><UL><LI>320 µL 5x isothermal reaction buffer </LI><LI>0.64 µL of 10 U ml<SUP>-1</SUP> T5 exonuclease (for DNA molecules overlapping by greater than 150 bp add 3.2 µL of 10 U mL<SUP>-1</SUP> T5 exonuclease) </LI><LI>20 µL of 2 U mL<SUP>-1</SUP> Phusion DNA polymerase </LI><LI>160 µL of 40 U mL<SUP>-1</SUP> taq DNA ligase </LI><LI>add ddH<SUB>2</SUB>O up to a final volume of 1.2 mL </LI></UL><LI>Aliquote 15 µL of the assembly mixture and store at -20 °C </LI><LI>That 15 µL assembly mixture on ice until use </LI><LI>Add 5 µL of the purified DNA molecules in equimolar amounts (between 10 and 100 ng of each fragment).</LI><LI>Incubate the mixture at 50 °C for 15 to 60 min with 60 min being optimal. </LI></UL><LI>Transformation via <A href="https://2015.igem.org/team:bielefeld-cebitec/protocols#heatshock" target="blank">heat shock</A> or <A href="https://2015.igem.org/team:bielefeld-cebitec/protocols#electrotransformation" target="blank">electroporation</A> without cleaning up the assembly product.</LI></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#q5_pcr">Q5 PCR</A></H4></DIV><DIV id="q5_pcr" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Q5 </LI><UL><LI>Q5 master mix (25 µL)</LI><UL><LI>5 µL 5 x Q5 buffer </LI><LI>0.5 µL 10 mM dNTPs </LI><LI>Up to 50 ng*µL<SUP>-1</SUP>template </LI><LI>0.1 µL Q5 DNA polymerase </LI><LI>0.5-1 µL primer 10 µM each </LI><LI>Ad 25 µL dH<SUB>2</SUB>O </LI></UL><LI>Q5 standard protocol </LI><UL><LI>Initial denaturation: 98°C - 30 s </LI><LI>30-35 cycles of</LI><UL><LI>Denaturation: 98 °C - 20 s </LI><LI>Annealing: 50-75 °C - 30 s </LI><LI>Elongation: 72 °C - 20 s/kb </LI></UL><LI>Final elongation: 72 °C - 5 min </LI><LI>Storage: 4 °C </LI></UL></UL><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#GeneratingCompetentCells">Generating competent cells</A></H4></DIV><DIV id="GeneratingCompetentCells" class="panel-collapse collapse"><DIV class="panel-body">
We generated chemocompetent, ultra-chemocompetent and electrocompetent <I>E. coli</I> using the following protocols:
<DIV class="panel-group inneraccordion" id="accordion5"><DIV class="panel panel-default"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion5" href="#collapseCompisA"><U>Chemo competent cells 	</U></A></H4><DIV id="collapseCompisA" class="panel-collapse collapse in"><DIV class="panel-body"><UL><LI>Preparation</LI><UL><LI>Cool 5 50 mL Falcons </LI><LI>Cool TSS buffer </LI><LI>Cool centrifuge </LI></UL><LI>Protocol</LI><UL><LI>Inoculate 2 x 3 mL LB with bacterial stock over night at 37 °C and 200 rpm </LI><LI>Inoculate 1 x 250 mL LB with the over night cultures in 1 L flask at 37 °C and 140 rpm </LI><LI>Incubate to OD<SUB>600</SUB> 0.4-0.6 </LI><LI>Devide the culture into 5 cooled 50 mL falcons and incubate on ice for 10 min </LI><LI>Onwards all steps at 4 °C </LI><LI>Centrifuge at 3000 rpm, 4 °C, 10 min </LI><LI>Discard supernatant </LI><LI>Pipette remaining media out carefully </LI><LI>Add 10% volume cold TSS buffer of original cultur, about 1-2 mL</LI><LI>Resuspend pellet</LI><LI>Aliquot 100 µL and drop in liquid nitrogen immediatly </LI><LI>Store at -80 °C</LI></UL><DIV class="panel panel-default"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion5" href="#collapseCompisB"><U>Electro competent cells 	</U></A></H4><DIV id="collapseCompisB" class="panel-collapse collapse in"><DIV class="panel-body"><UL><LI>Preparation</LI><UL><LI>Autoclave 550 mL LB-Medium </LI><LI>Cool 1 L bidest. H<SUB>2</SUB>O </LI><LI>Cool 50 mL 10% glycerine </LI><LI>Cool 10 50 mL Falcons </LI></UL><LI>Protocol</LI><UL><LI>Inoculate 2 x 3 mL LB with bacterial stock, incubate over night at 37 °C and 200 rpm </LI><LI>Inoculate 2 x 250 mL LB with the over night culture in 1 L flask at 37 °C and 140 rpm </LI><LI>Cultivate until OD<SUB>600</SUB>0.4-0.6 </LI><LI>Store on ice for 15-30 min </LI><LI>Onwards all steps at 4 °C </LI><LI>Divide the cultures into the pre-cooled 50 mL Falcons and centrifuge at 4000 rpm, 4 °C, 15 min with slow acceleration and decceleration </LI><LI>Discard supernatant </LI><LI>Resuspend in 5 mL cooled bidest H<SUB>2</SUB>O shaking gently </LI><LI>Pool two suspensions, add bidest. H<SUB>2</SUB> to 50 mL and centrifuge again at 4000 rpm, 4°C, 15 min </LI><LI>Discard supernatant </LI><LI>Resuspend pellet in 5 mL cooled bidest H<SUB>2</SUB>O </LI><LI>Add bidest H<SUB>2</SUB>O up to 50 mL and centrifuge again at 4000 rpm, 4 °C, 15 min </LI><LI>Discard supernatant </LI><LI>Resuspend in 5 mL cooled 10% glycerine </LI><LI>Transfer suspensions in two 50 mL falcons and centrifuge again at 4000 rpm, 4 °C, 15 min </LI><LI>Discard supernatant </LI><LI>Add 1 volume similar to the pellet of 10% glycerine and resuspend </LI><LI>Aliquot in 50 µL and freeze in liquid nitrogen immediately </LI><LI>Store at -80 °C </LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion5" href="#collapseCompisC"><U>Ultra chemo competent cells 	</U></A></H4><DIV id="collapseCompisC" class="panel-collapse collapse in"><DIV class="panel-body"><UL><LI>Preparation</LI><UL><LI>Inoculate 10 mL LB medium in a 100 mL flask with 50 µL cells from glycerol culture, e.g. <I>E. coli </I> DH5a MCR </LI><LI>Incubate over night at 37 °C and 180 rpm </LI></UL><LI>Protocol</LI><UL><LI>Precool centrifuge and media </LI><LI>Inoculate 250 mL SOB-Medium in a 1 L flask with 2.5 mL of the preculture </LI><LI>Incubate at 19 °C  and 180 rpm up to OD<SUB>600</SUB> 0.5 (about 24-36 hours) </LI><LI>Cool flask 10 min in ice water </LI><LI>Onwards all steps at 4 °C </LI><LI>Devide the culture in 5 50 mL cooled falcons </LI><LI>Centrifuge 5 min at 4 °C and 5000 x g </LI><LI>Discard supernatant </LI><LI>Resuspend in each 6 mL icecold TB Buffer </LI><LI>Incubate 10 min on ice </LI><LI>Centrifuge 5 min at 4 °C and 5000 x g </LI><LI>Discard supernatant </LI><LI>Resuspend in each 6 mL icecold TB Buffer </LI><LI>Add 420 µL DMSO and vortex immediately </LI><LI>Aliquot 150 µL and freeze in liquid nitrogen immediately </LI><LI>Store at - 80 °C </LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#PlasmidIsolation">Plasmid Isolation</A></H4></DIV><DIV id="PlasmidIsolation" class="panel-collapse collapse"><DIV class="panel-body">
 For Plasmid Isolation we used the following kits. The methods are described seperately: 
<DIV class="panel-group inneraccordion" id="accordion3"><DIV class="panel panel-default"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion2" href="#PlasmidIsolationA"><U>PureYield<SUP>TM</SUP> Plasmid Miniprep System (Promega)   	</U></A></H4><DIV id="PlasmidIsolationA" class="panel-collapse collapse in"><DIV class="panel-body"><UL><LI>Prepare Lysate</LI><UL><LI>Add 600 µL of bacterial culture to a 1.5 mL microcentrifuge tube </LI><LI>Alternative for larger cultures: Centrifuge 1.5 mL bacterial culture for 30 s at maximum speed in a microcentrifuge. Discard the supernatant. Add an additional 1.5 mL bacterial culture to the same tube and repeat previous step. Add 600 µL of TE buffer or water to the pellet, and resuspend completely. </LI><LI>Add 100 µL of Cell Lysis Buffer (blue), and mix by inverting the tube 6 times.</LI><LI>Add 350 µL of cold (4-8 °C) Neutralization Solution and mix thoroughly by inverting.</LI><LI>Centrifuge at maximum speed in a microcentrifuge for 3 minutes. </LI><LI>Transfer the supernatant (about 900 µL) to a PureYield<SUP>TM</SUP> Minicolumn without disturbing the cell debris pellet </LI><LI>Place the minicolumn into a Collection Tube and centrifuge at maximum speed in a microcentrifuge for 15 s </LI><LI>Discard the flowthrough, and place the minicolumn into the same Collection Tube.</LI></UL><LI>Wash</LI><UL><LI>Add 200 µL of Endotoxin Removal Wash (ERB) to the minicolumn. Centrifuge at maximum speed in a microcentrifuge for 15 s </LI><LI>Add 400 µL of Column Wash Solution (CWC)to the minicolumn. Centrifuge at maximum speed in a microcentrifuge for 30 seconds </LI></UL><LI>Elute</LI><UL><LI>Transfer the microcolumn to a clean 1.5 mL microcentrifuge tube, then add 30 µL of Elution Buffer or nuclease-free water directly to the minicolumn matrix. Let stand for 1 minute at room temperature </LI><LI>Centrifuge for 15 seconds to elute the plasmid DNA. Cap the microcentrifuge tube, and store eluted plasmid DNA at -20°C </LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion2" href="#PlasmidIsolationB"><U>Thermo Scientific GeneJET Plasmid Miniprep Kit (Centrifugation)   	</U></A></H4><DIV id="PlasmidIsolationB" class="panel-collapse collapse in"><DIV class="panel-body"><UL><LI>All purification steps should be carried out at room temperature.</LI><LI>All centrifugations should be carried out in a table-top microcentrifuge at &gt;12.000 x g (10.000-14.000 rpm)</LI><LI>Use 1-5 mL of <I>E. coli</I> culture in LB media for purification of high-copy plasmids.</LI><LI>Use up to 10 mL culture for low-copy plasmids</LI><LI>Protocol</LI><UL><LI>Resuspend the pelleted cells in 250 µL of the Resuspension Solution. Transfer the cell suspension to a microcentrifuge tube. The bacteria should be resuspended completely by vortexing or pipetting up and down until no cells clumps remain.Note: Ensure RNase A has been added to the Resuspension Solution.</LI><LI>Add 250 µL of the Lysis Solution and mix throughly by inverting the tube 4-6 tines until the solution becomes viscous and slightly clear. Note: Do not vortex to avoid shearing of chromosomal DNA. Do not incubate for more than 4 min to avoid denaturation of supercoiled DNA.</LI><LI>Add 350 µL of the Neutralization Solution and mix immediately and thoroughly by inverting the tube 4-6 times. Note:It is important to mix throughly and gently after the addition of the Neutralization Solution to avoid localized precipitation of bacterial cell debris. The neutralized bacterial lysate should become cloudy.</LI><LI>Centrifuge for 5 min to pellet cell debris and chromosomal DNA </LI><LI>Transfer the supernatant to the supplied GeneJET spin column by decanting or pipetting. Avoid disturbing or transferring the white precipitate.</LI><LI>Centrifuge for 1 min. Discard the flow-through and place the column back into the same collection tube.</LI><LI>Add 500 µL of the Wash Solution (diluted with ethanol prior to use) to the GeneJET spin column. Centrifuge for 30-60 seconds and discard the flow-through. Place the column back into the same collection tube.</LI><LI>Repeat the wash procedure using 500 µL of the Wash Solution </LI><LI>Discard the flow-through and centrifuge an additional 1 min to remove residual Wash Solution. This step is essential to avoid residual ethanol in plasmid preps.</LI><LI>Transfer the GeneJET spin column into a fresh 1.5 mL microcentrifuge tube. Add 50 µL of the Elution Buffer to the center of GeneJET spin column membrane to elute the plasmid DNA. Take care not to contact the membrane with the pipette tip. incubate for 2 min at room temperature and centrifuge for 2 min. </LI><LI>Discard the column and store the purified plasmid DNA at -20°C.</LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion2" href="#PlasmidIsolationC"><U>Analytik Jena innuPREP Plasmid Mini Kit (Centrifugation)   	</U></A></H4><DIV id="PlasmidIsolationC" class="panel-collapse collapse in"><DIV class="panel-body"><UL><LI>Starting material: 0.5-5.0 mL</LI><LI>Pellet cells </LI><UL><LI>Centrifuge: max speed, 1 min </LI><LI>Discard supernatant completely </LI></UL><LI>Resuspend cells </LI><UL><LI>Add 250 µL Resuspension Buffer</LI><LI>Vortex</LI></UL><LI>Lysis (do not vortex) </LI><UL><LI>Add 250 µL Lysis Buffer </LI><LI>Mix: invert tube 6-8 times</LI><LI>Lysis time: &lt;5 min</LI></UL><LI>Neutralization</LI><UL><LI>Add 350 µL Neutralization Buffer </LI><LI>Mix: invert tube 6-8 times</LI><LI>Centrifuge max speed, 8 min </LI></UL><LI>Binding of DNA </LI><UL><LI>Add Spin Filter to Receiver Tube </LI><LI>Add clarified sample Spin Filter </LI><LI>Centrifuge at 11.000 x g (12.000 rpm) 1 min </LI></UL><LI>Washing</LI><UL><LI>Add 500 µL Washing Solution A </LI><LI>Centrifuge 1 min at 11.000 g (12.000 rpm) </LI><LI>Add 700 µL Washing Solution B </LI><LI>Centrifuge 11.000 x g (12.000 rpm) 1 min </LI></UL><LI>Remove ethanol</LI><UL><LI>Discard filtrate </LI><LI>Add Spin Filter to Receiver Tube </LI><LI>Centrifuge Max speed, 2 min </LI></UL><LI>Elution</LI><UL><LI>Add Spin Filter to a 1.5 mL reaction tube </LI><LI>Add 50-100 µL Elution Buffer P </LI><LI>Incubation 1 min at room temperature </LI><LI>Centrifuge at 11.000 x g (12.000 rpm) 1 min </LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion2" href="#PlasmidIsolationD"><U>QIAprep Spin Miniprep Kit (Qiagen)   	</U></A></H4><DIV id="PlasmidIsolationD" class="panel-collapse collapse in"><DIV class="panel-body"><UL><LI>All centrifugation steps are carried out at 13.000 rpm (17.900m x g) in a conventional table-top microcentrifuge </LI><LI>Pellet 1-5 mL bacterial overnight culture by centrifugation at &gt;8.000 rpm (6.800 x g) for 3 min at room temperature (15-25 °C) </LI><LI>Resuspend pelleted bacterial cells in 250 µL Buffer P1 and transfer to a microcentrifuge tube. </LI><LI>Add 250 µL Buffer P2 and mix thoroughly by inverting the tube 4-6 times until the solution becomes clear. Do not allow the lysis reaction to proceed for more than 5 min . If using LyseBlue reagent, the solution will turn blue. </LI><LI>Add 350 µL Buffer N3 and mix immediately and thoroughly by inverting the tube 4-6 times. If using LyseBlue reagent, the solution will turn colorless.</LI><LI>Centrifuge for 10 minutes at 13.000 rpm (17.900 x g) in a table-top microcentrifuge.</LI><LI>Apply the supernatant to the QIAprep spin column by decanting or pipetting. Centrifuge for 30-60 s and discard the flow-through.</LI><LI>Wash the QIAprep spin column by adding 500 µL Buffer PB. Centrifuge for 30-60 s and discard the flow-through </LI><LI>Wash the QIAprep spin column by adding 750 µL Buffer PB. Centrifuge for 30-60 s and discard the flow-through. Transfer the QIAprep spin column to the collection tube.</LI><LI>Centrifuge for 1 min to remove residual wash buffer.</LI><LI>Place the QIAprep column in a clean 1.5 mL microcentrifuge tube. To elute DNA, add 50 µL Buffer EB(10mM Tris-Cl, pH8.5) or water to the center of the QIAprep spin column, let it stand for 1 min, and centrifuge for 1 min.</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#BioBrickAssembly">Standard BioBrick Assembly</A></H4></DIV><DIV id="BioBrickAssembly" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Modified from Silver Lab</LI><LI>Minimum size of the BioBricks for this methods is 150 bp, whereas it should be at least 500 bp bigger or smaller than the used backbone. The BioBricks, which complies with these conditions, is used as the insert and is assembled into the prefix or suffix of the other BioBrick called vector. The methods differ in prefix and suffix insertion. </LI><LI>Suffix insertion</LI><UL><LI>Digestion of inserted Biobrick </LI><UL><LI> Acquire &gt; 700 ng DNA (10 µL volume) </LI><LI> Add 1 µL 10 x NEBuffer </LI><LI> Add 0.5 µL XbaI </LI><LI> Add 0.5 µL PstI </LI><LI> Digest 1 h at 37 °C </LI><LI> Inactivate 20 min at 80 °C </LI><LI> Clean up via gel electrophoresis, cut and extract from gel </LI></UL><LI> Digestion of the vector </LI><UL><LI> Acquire about 700 ng DNA (10 µL volume) </LI><LI> Add 1 µL 10 x NEBuffer 2.1 </LI><LI> Add 0.5 µL SpeI </LI><LI> Add 0.5 µL  PstI </LI><LI> Digest 1 h at 37 °C </LI><LI> Inactivate 20 min at 80 °C</LI><LI> Add 1 µL Antarctic Phosphatase (AP) and 1.2 µL 10 x AP reaction buffer </LI><LI> Incubate 1 h at 37°C </LI><LI> Clean up with PCR clean up kit </LI></UL><LI> Ligation </LI><UL><LI> Mix 50-200 ng vector with 3-10 fold molar access of insert</LI><LI> Add 2 µL T4-Ligase-Buffer, 1 µL T4-DNA-Ligase </LI><LI> Incubate for a minimum 20-30 min at room temperature (we usually ligate over night) </LI><LI> Inactivate 5 min at 70 °C </LI><LI> Store at -20 °C or transform </LI></UL></UL><LI> Prefix insertion </LI><UL><LI> Follow protocol of Suffix insertion, replace the restriction enzymes as following: </LI><UL><LI> Digest insert with EcoRI and SpeI </LI><LI> Digest vector with EcoRI and XbaI </LI></UL></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#ElectroTransformation">Transformation via Electroporation</A></H4></DIV><DIV id="ElectroTransformation" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Thaw 50 µL electrocompetent <I>E. coli</I> cells on ice, dilute with icecold 50 µL glycerine (10%) if necessary</LI><LI>Add 0.5-5 µL plasmid to 50 µL electrocompetent cells</LI><LI>Store cells on ice for 1 minute</LI><LI>Electroporate at U = 2.5 kV, C = 25 µF, R = 400 Ω</LI><LI>Transfer transformation reaction to 450 µL SOC-Medium and incubate 1 h at 37 °C</LI><LI>Plate on selective LB-Medium</LI><LI>Incubate over night at 37 °C</LI></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#chemo_trafo">Transformation via Heat Shock</A></H4></DIV><DIV id="chemo_trafo" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Thaw 100 µL chemo competent <I>E. coli</I> cells on ice </LI><LI>Add 0.5-5 µL plasmid to 100 µl chemocompetent cells </LI><LI>Store cells on ice for 10-30 min on ice </LI><LI>Heat shock for 90 seconds at 42 °C </LI><LI>Store reaction on ice for 60 seconds </LI><LI>Optional: Preheat SOC medium to 37 °C</LI><LI>Transfer reaction to 1 ml SOC medium and incubate at 37 °C for at least 1 hour</LI><LI>Centrifuge 3 minutes at 12000 rpm and plate on selective LB medium </LI><LI>Incubate at 37 °C over night </LI></UL></DIV></DIV></DIV><H1>Other protocols</H1><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#sds_page">SDS-PAGE</A></H4></DIV><DIV id="sds_page" class="panel-collapse collapse"><DIV class="panel-body"><LI><U> Pouring the polyacrylamide gel: </U></LI><UL><LI> For 2 <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media#resolvingGel" target="_blank">resolving gel (14 %)</A>: </LI><UL><LI>4.7 mL  Bisacrylamid/Acrylamid (0.8 %; , 30 %;, at the ratio of 37.5:1)</LI><LI>2.7 mL H<SUB>2</SUB>O</LI><LI>2.5 mL 0.5 M TRIS-HCl, pH 6.8</LI><LI>0.1 mL 10% SDS</LI></UL></UL><LI> Add 50 µL 10 % ammonium persulfate and 10 µL TEMED and mix. </LI><LI> Pour the solution quickly into the gel casting form. Leave about 2 centimeters below the bottom of the comb for the stacking gel. </LI><LI> Layer isopropanol on top of the gel. </LI><LI> Leave the resolving gel at room temperature for 30-45 minutes to polymerize. </LI><LI> Remove isopropanol and wash the surface of the separating gel with H<SUB>2</SUB>O. Wait until the surface is dry.</LI><LI> For 2 <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media##stackingGel" target="_blank">stacking gel (4 %)</A>: </LI><UL><LI>1.3 mL Bisacrylamid/Acrylamid (0.8 %; , 30 %;, at the ratio of 37.5:1)</LI><LI>6.1 mL H<SUB>2</SUB>O</LI><LI>2.5 mL 0.5 M TRIS-HCl, pH 6.8</LI><LI>0.1 mL 10% SDS</LI><LI>50 µL 10 % ammonium persulfate</LI><LI>5 µL TEMED</LI></UL><LI> Add 50 µL 10 % ammonium persulfate and 5 µL TEMED to each aliquote and mix. </LI><LI> Insert comb without getting bubbles stuck underneath.</LI><LI> Leave the gel at room temperature for 30-45 minutes to polymerize.</LI><LI> For storage:</LI><UL><LI> Remove sealing and store the gel wrapped in moistened paper towel at 4 °C. </LI></UL><LI><U> Preparing the sample: </U></LI><UL><LI> Mix your protein mixture 3:1 with <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media#PBJR" target="_blank">PBJR buffer</A> (15 µL protein solution + 5 µL <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media#PBJR" target="_blank">PBJR buffer</A>) </LI><LI> Heat for 5 minutes at 95 °C</LI></UL><LI><U> Running the gel: </U></LI><UL><LI> Remove sealing, put the polymerized gel into gel box and pour 1x <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media#SDSrunning" target="_blank">SDS-PAGE running buffer</A> into the negative and positive electrode chamber. </LI><LI> Remove comp without destroying the gel pockets. </LI><LI> Pipet the SDS running buffer in the gel pockets up and down for flushing the gel pockets.  </LI><LI> Pipet slowly 20 µL of the sample into the gel pockets. </LI><LI> Make sure to include at least one lane with molecular weight standards (PageRuler Prestained Protein Ladder™ (Fa. Fermentas)) to determinate the molecular weight of the sample. </LI><LI> Connect the power lead and run the gel with 180 V until the distance of the lowest molecular weight standard lane to the gel end is down to 0.5 cm. </LI></UL><LI><U> Staining the polyacrylamide gel (Colloidal Coomassie Brilliant Blue staining): </U></LI><UL><LI> After finishing the SDS-PAGE remove gel from gel casting form and transfer it into a box. </LI><LI> Add 100 mL of the <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media#stainSDS" target="_blank">Colloidal Coomaassie Brilliant Blue staining solution</A> to your polyacrylamid gel. </LI><LI> Incubate the gel in the solution at room temperature until the protein bands got an intensive blue color. Shake the gel continuously during incubation. </LI><LI> Remove the staining solution. </LI><LI> Wash the gel with <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media#collapseTen" target="_blank">destaining buffer</A>.</LI><LI> Incubate the gel in H<SUB>2</SUB>O (2-6 h) for bleaching the background. Shake the gel continuously during incubation.  If necessary replace the colored water with new one.
       </LI></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#maldi">MALDI-TOF/TOF sample preparation</A></H4></DIV><DIV id="maldi" class="panel-collapse collapse"><DIV class="panel-body"><LI><U> Tryptic digest of gel lanes for analysis with MALDI-TOF: </U></LI><UL><LI>Be careful that skin scales or hair do not contaminate your sample. So wear hand gloves and tie up your hair.</LI><LI>Reaction tubes have to be cleaned with 60 % (v/v) CH<SUB>3</SUB>CN and 0.1 % (v/v) TFA. Afterwards the solution has to be removed completely followed by evaporation of the tubes under a fume hood. Alternatively, microtiter plates from Greiner® (REF 650161) can be used without washing. If you work with reaction tubes from Eppendorf, you do not need the wash step either. </LI><LI>Cut out the protein lanes of a Coomassie-stained SDS-PAGE using a clean scalpel. Gel parts are transferred to the washed reaction tubes. If necessary, cut the parts to smaller slices</LI><LI> Gel slices should be washed two times. Therefore add 200 µL 30 % (v/v) acetonitrile in 0.1 M ammonium hydrogen carbonate each time and shake lightly for 20 minutes till the gel slices are destained. Remove supernatant and discard to special waste </LI><LI> Dry gel slices at least 30 minutes in a Speedvac. </LI><LI> Rehydrate gel slices in 15 µL <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media#trypsinsolution" target="_blank">trypsin solution</A> followed by short centrifugation. </LI><UL><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media#trypsinsolution" target="_blank">Trypsin-solution</A>: 1 µL trypsin + 14 µL 10 mM NH<SUB>4</SUB>HCO<SUB>3</SUB></LI><UL><LI> For this solution solubilize lyophilized trypsin in 200 µl of provided buffer and activate Trypsin for 15 minutes at 30 °C. For further use it can be stored at -20 °C. </LI></UL></UL><LI> Gel slices have to be incubated 30 minutes at room temperature, followed by incubation at 37 °C overnight </LI><LI> Dry gel slices at least 60 minutes in a Speedvac. </LI><LI> According to the size of the gel slice, add 5 - 20 µL 50 % (v/v) ACN / 0.1 % (v/v) TFA </LI><LI> Samples can be used for MALDI measurement or stored at -20 °C </LI></UL><LI><U> Preparation and Spotting for analysis of peptides on Bruker AnchorChips: </U></LI><UL><LI> Spot 0.5 - 1 µL of sample aliquot </LI><LI> Add 1 µL <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media#hccamatrix" target="_blank">HCCA matrix solution</A> to the spotted sample aliquots. Pipet up and down approximately five times to obtain a sufficient mixing. Be careful not to contact the AnchorChip. Note: Most of the sample solvent needs to be gone in order to achieve a sufficiently low water content. When the matrix solution is added to the previously spotted sample aliquot at a too high water content in the mixture, it will result in undesired crystallization of the matrix outside the anchor spot area. </LI><LI> Dry the prepared spots at room temperature </LI><LI> Spot external calibrants on the adjacent calibrant spot positions. Use the calibrant stock solution (Bruker’s “<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media#MALDIstandard" target="_blank">Peptide Calibration Standard II</A>”, Part number #222570), add 125 µL of 0.1 % TFA (v/v) in 30 % ACN to the vial. Vortex and sonicate the vial.</LI><LI> Mix the calibrant stock solution in a 1:200 ratio with HCCA matrix and deposit 1 µL of the mixture onto the calibrant spots. </LI><LI> Analyze samples in ultrafleXtreme by Bruker Daltonics. </LI></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#phagemid">Phagemid display</A></H4></DIV><DIV id="phagemid" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI><U>Preparing immune-tubes for phage panning:</U></LI><UL><LI> Dilute 45 μg of your target protein in 1.5 mL 1 M NaHCO3 </LI><LI>Incubate rolling over night at 4 °C and discard the rest </LI><LI>Coat free surfaces by incubating 1 h filled with 5 mg/mL BSA</LI><LI>Wash three times with PBST</LI><LI>Store at 4 °C in the dark</LI></UL><UL><LI><U>Phagemid display:</U></LI><LI>Insert a variable library of binding protein CDSs in the phagemid vector pAK100 by Gibson or restriction cloning</LI><LI>Transformate the vector in <I>E. coli</I> ER2738</LI><LI>Remaining colonies form the phagemid library, screened for binders in the following procedure
</LI><LI>Wash up all colonies with 5 mL LB and inoculate 200 mL LB, containing chloramphenicol, up to OD 0.1</LI><LI>Incubate the culture, shaking at 37°C, up to OD 0.5</LI><LI>Add 10 x Phages per <I>E. coli</I> cell to infect the bacteria</LI><LI>Incubate 1.5 h at 37°C, then add kanamycin</LI><LI>Incubate further 1.5 h at 37°C</LI><LI>Centrifuge at 5,000 g for 10 min and decant the supernatant</LI><LI>Add 25 %vol PEG NaCl and mix well</LI><LI>Precipitation of produced phages at 4 °C, over night</LI><LI>Centrifuge at 12,000 g and 4°C for 30 min and discard the supernatant</LI><LI>Resuspend the pellet with 3 mL PBS and centrifuge at 13,000 g and room temperature for 10 min and decant the supernatant</LI><LI>Add 20 %vol PEG NaCl and precipitate for 1 h on ice</LI><LI>Centrifuge at 4 °C and 13,000 g for 10 min and discard supernatant</LI><LI>Resuspend the pellet in 500 μL PBS</LI><LI>Centrifuge at 13,000 g and room temperature for 10 min</LI><LI>Measure UV-VIS absorbance of an 1:50 aliquot at 269 nm and 320 nm to calculate the phage concentration</LI><LI>Add 1x1012 to 5x1012 Phages diluted in 1 mL PBST containing 1 mg/mL BSA on the prepared immune-tubes and incubate rolling at 4°C for 1 h</LI><LI>Wash the immune-tubes carefully with PBST for at least 10 times and discard the washing buffer</LI><LI>Eluate remaining phages by adding 1 mL Glycine-HCl, containing 1 mg/mL BSA by incubating rolling for 10 min at room temperature</LI><LI>Neutralize the eluate by adding 150 μL TRIS at pH 9</LI><LI>Add 10 μL eluted phages to 140 μL of <I>E. coli</I> ER2738 at OD 0.5, incubate for 10 min at room temperature and streak out at LB agar containing tetracycline and chloramphenicol </LI></UL><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#EMSA">Electrophoretic Mobility Shift Assay (EMSA)</A></H4></DIV><DIV id="EMSA" class="panel-collapse collapse"><DIV class="panel-body"><P><UL><LI>Required solutions:</LI><UL><LI>5x EMSA buffer (100 mM Na<SUB>2</SUB>HPO<SUB>4</SUB>, 375 mM KCl, 25 % Glycerol, adjust ph to 8.0)</LI><LI>EMSA running buffer (20 mM Na<SUB>2</SUB>HPO<SUB>4</SUB>, adjust ph to 8.0)</LI><LI>3 % agarose gel prepared with EMSA running buffer</LI><LI>MgCl<SUB>2</SUB>: 25 mM</LI><LI>EDTA: 2,5 mM</LI><LI>BSA: 10 mg/ml</LI><LI>Dithiotreitol</LI><LI>salmon sperm</LI><LI>Cy3 labeled DNA</LI><LI>protein</LI></UL><LI>Reaction:</LI><UL><LI>0.05 pmol Cy3 labeled DNA</LI><LI>5 pmol purified protein of interest</LI><LI>4 µl 5x EMSA buffer</LI><LI>2 µl MgCl<SUB>2</SUB> final concentration: 2.5 mM</LI><LI>0.8 µl EDTA final concentration: 0.1 mM</LI><LI>1 µl BSA final amount: 10 µg</LI><LI>1 µl salmon sperm final amount: 1 µg</LI><LI>optional: 5 mM DTT</LI></UL><LI>mix all components of the reaction carefully and add up with water to 20 µl H<SUB>2</SUB>O. </LI><LI>Incubate at room temperature in the dark for 15 min.</LI><LI>Keep in mind that Cy3 is light sensitive, always keep your samples in the dark after addition of the DNA fragment.</LI><LI>Apply reaction to 3 % agarose gel. </LI><LI>Run gel at 80 V for approximately 80 min at 4 °C. (Adjust voltage and duration to size of your DNA fragement.) </LI><LI>Detect Cy3 with a fluorescence scanner. In our case it was the Typhoon 8600 variable Mode Imager.</LI></UL></P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#crispr">CRISPR/Cas9-Knockout</A></H4></DIV><DIV id="crispr" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Design of the guide RNA: 20 nt of interest with 3' protospacer adjacent sequence (PAM) NGG, where N is any nucleotide. Preference is given to:
</LI><LI> sequences with purines occupying the last four (3') bases of the protospacer.</LI><LI> sequences on the non-coding strand.</LI><LI> sequences in which the last 12 nt protospacer + 3 nt PAM (15 nt total) are unique in the genome (check by BLAST with all four possible NGG sequences)</LI></UL><LI>For knock-out of the <I>rpoZ</I> gene of <I>E. coli</I> the following sequence can be used:
</LI><LI> 5'- CGACCAGTACCAGGTCAAAACGG -3'</LI><LI> Design two 24 nt oligonucleotide (4 nt 5' sticky end + 20 nt spacer sequence) with the sticky ends ACGC on the forward primer and AAAC on the reverse primer. For the <I>rpoZ</I> knock-out the two primers are:
</LI><LI> Guide-fw: 5'-ACGCCGACCAGTACCAGGTCAAAA-3'</LI><LI> Guide-rv: 5'-AAACTTTTGACCTGGTACTGGTCG-3'</LI>
The sticky ends are designed for the <I>Bbs</I>I sites.
<LI>For single spacer, anneal single spacer oligos as follows:
</LI><LI> Resuspend both oligos to 100µM in water</LI><LI> Mix 5µl <B>Guide-for</B> + 5µl <B>Guide-rev</B> + 90 µl 30mM HEPES, pH 7.8</LI><LI> Heat to 95°C for 5min, then ramp to 4°C at 0.1°C/sec</LI><LI>Insert annealed spacer by Golden Gate assembly, 
Golden Gate reaction mixture:
</LI><LI><TABLE><TBODY><TR><TD>Backbone</TD><TD>X µl</TD><TD>100 ng</TD></TR><TR><TD>Insert</TD><TD>0.3 µl</TD><TD>From 10-fold diluted annealed oligo stock</TD></TR><TR><TD>T4 Ligase Buffer (NEB)</TD><TD>2 µl</TD></TR><TR><TD>T4 Ligase (NEB)</TD><TD>1 µl</TD><TD>400 U/µl stock is sufficient; add last</TD></TR><TR><TD><I>Bbs</I>I</TD><TD>1 µl</TD><TD>Stored at -80°C</TD></TR><TR><TD>H2O</TD><TD>Y µl</TD></TR><TR><TD>In sum</TD><TD>20 µl</TD></TR></TBODY></TABLE></LI>
Golden Gate Program:
<UL><LI> 37°C , 10 min</LI><LI> 16°C , 10 min</LI><LI> Goto step 1, 9 times</LI><LI> 50°C , 5 min</LI><LI> 65°C , 20 min</LI><LI> 4°C , forever</LI></UL><LI> Transform 3 µl of each reaction to <I>E. coli</I> DH4α by heat shock(manufacturer's protocol) or electroporation(manufacturer's protocol)</LI><LI> Plate 10% of recovery culture on selective plates with 10 µl of 0.5M IPTG and 40 µl of 20 mg/ml Bluo-gal (in DMSO).</LI><LI> Pick white colonies to selective LB (for pTarget use streptomycine) and recover plasmid.</LI>
CRISPR/Cas knock-out/knock-in:
<UL><LI> Transformation of pCas9 in willed knock-out strain via electroporation:
</LI><LI>Preheat SOC medium with 0.1% arabinose by 30°C</LI><LI>Thaw 50 µL electrocompetent E. coli cells (the knock-out/knock-in strain)on ice, dilute with icecold 50 µL glycerine (10%) if necessary</LI><LI>Add 1-5 µL plasmid to 50 µL electrocompetent cells</LI><LI>Store cells on ice for 1 minute</LI><LI>Electroporate at U = 2.5 kV, C = 25 µF, R = 400 Ω</LI><LI>Transfer transformation reaction to 950 µL prepared SOC-Medium and incubate 1 h at 30 °C</LI><LI>Plate on LB-Medium prepared with kanamycine and 0.1% arabinose</LI><LI>Incubate over night at 30 °C</LI></UL><LI>Create electrocompetent cells of the knock-out/knock-in strain carrying the pCas plasmid. <B>ATTENTION:</B> cultivation of the cells by 30°C in LB with 0,1% arabinose!</LI><LI>PCR with the BioBrick <A href="http://parts.igem.org/Part:BBa_K2082251">BBa_K2082251</A>:
</LI><LI>Q5 master mix (25 µL):
</LI><LI>5 µL 5 x Q5 buffer</LI><LI>0.5 µL 10 mM dNTPs</LI><LI>Up to 50 ng/µL-template (<A href="http://parts.igem.org/Part:BBa_K2082251">BBa_K2082251</A>)</LI><LI>0.1 µL Q5 DNA polymerase</LI><LI>0.5 µl forward-primer:<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">CC_FragAB_Reporter_fw</A></LI><LI>0.5 µl reverse-primer:<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">CC_FragAB_Reporter_rv</A></LI><LI>Ad 25 µL dH2O </LI><LI>Q5 standard protocol:
</LI><LI>Initial denaturation: 98°C - 30 s </LI><LI>30-35 cycles of:
</LI><LI>Denaturation: 98 °C - 20 s</LI><LI>Annealing: 58 °C - 30 s </LI><LI>Elongation: 72 °C - 20 s/kb</LI><LI>Final elongation: 72 °C - 5 min</LI><LI>Storage: 4 °C</LI><LI>Purification of the PCR product (manufacturer's protocol)</LI><LI>Transformation of the purified PCR product and the guide-RNA-carrying plasmid pTarget in the pCas-carrying competent knock-out/knock-in strain of <I>E. coli</I> via electroporation</LI><UL><LI>Preheat SOC medium with 0.1% arabinose by 30°C</LI><LI>Thaw 50 µL electrocompetent pCas-carrying E. coli cells (the knock-out/knock-in strain)on ice, dilute with icecold 50 µL glycerine (10%) if necessary</LI><LI>Add 1-3 µL pTarget plasmid to 50 µL electrocompetent cells</LI><LI>Add 2 µL purified PCR product to 50 µL electrocompetent cells</LI><LI>Store cells on ice for 1 minute</LI><LI>Electroporate at U = 2.5 kV, C = 25 µF, R = 400 Ω</LI><LI>Transfer transformation reaction to 950 µL prepared SOC-Medium and incubate 1 h at 30 °C</LI><LI>Plate first half of the transformed cells on LB-Medium prepared with kanamycine, streptomycine and 0.1% arabinose</LI><LI>Plate second half of the transformed cells on LB-Medium prepared with kanamycine, streptomycine and 0.1% arabinosen + 20µg/ml ampicillin</LI><LI>Incubate over night at 30 °C</LI></UL><LI>Pick colonies from the ampicillin-prepared LB plate and accomplish a colony PCR:</LI><UL><LI>5 µL 5x GoTaq buffer (Promega)</LI><LI>1 µL MgCl<SUB>2</SUB> (25 mM stock)</LI><LI>0.5 µL dNTPs (10 mM each) </LI><LI>0.25 µL primer mix (prefix/suffix primers or sequencing primers) 100 mM</LI><LI>17.625 µL ddH<SUB>2</SUB>O </LI><LI>0.125 µL GoTaq polymerase (Promega)</LI><LI>0.5 µL template </LI></UL><LI>PCR program </LI><UL><LI>Cell lysis and denaturation: 5 min, 95 °C</LI><LI>30 cycles</LI><UL><LI>Denaturation: 10 s, 95 °C </LI><LI>Hybridisation: 30 s, annealing temperature </LI><LI>Elongation: 60 s/kb of product, 72 °C </LI></UL><LI>Final elongation: 5 min, 72 °C </LI></UL><LI>Template alternatives </LI><UL><LI>Pick a colony with sterile tip, elute in 100 µL ddH<SUB>2</SUB>O or buffer, store at 4 °C during PCR, plate if insert is of correct size</LI><LI>Pick colony, streak at marked position on a new plate and solute remaining cells on the tip in the PCR tube with reaction mixture, cultivate if insert is of correct size </LI></UL><LI>Gel electrophoresis for control of fragment size </LI><LI>Colonies with a fragment size of about 3,000 bp are the cells, with knock-out of the <I>E. coli rpoZ</I> gene and knock-in of the reporter <A href="http://parts.igem.org/Part:BBa_K2082238">BBa_K2082238</A></LI></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#Bradford">Bradford Assay</A></H4></DIV><DIV id="Bradford" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Pipette 100 µl each of sample buffer, standard 
solutions and samples into clean test tubes.</LI><LI>Add 
5 ml 1X
-
staining solution</LI><LI>Mix by inverting repeatedly</LI><LI>Measure the OD
595
of the standard solutions 
and samples after 5 to 30 mins against the 
reference (sample buffer in 1X
-
staining dye).</LI><LI>Plot and compare OD
595
of standard solutions 
to amount of protein u
sed. The amount of 
protein in sample can be read on calibration </LI></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#Impact">Intein Mediated Purification with an Affinity Chitin-binding Tag (IMPACT)</A></H4></DIV><DIV id="Impact" class="panel-collapse collapse"><DIV class="panel-body"><H3>Plasmid preparation</H3><UL><LI>For a C-termninal fusion of your protein with the Intein Tag use the pTXB1 vektor, for a N-terminal fusion use the pTYB11 vektor.</LI><LI>Cloning of the protein of interest directly before (pTXB1) or behind (pTYB11) the intein tag.</LI><LI>Finished vektor with elektroporation or chemical transformation in <I>E. coli</I> expression strain ER2566 for cultivation</LI></UL><H3>Cell growth</H3><UL><LI>Preculture: Inoculate some cells in 10 ml LB-medium (+ 200 µg/ml Ampicillin), growth by 37°C over nigth</LI><LI>Main cultivation: Inoculate 250 ml LB (+ 200 µg/ml Ampicillin) with the preculture on an OD600 of 0.1. Growth by  37°C until it reaches an OD600 between 0.5 and 0.8. Addition of 0.5 mM IPTG and growth over night at 19°C.</LI></UL><H3>Cell harvesting</H3><UL><LI>Transfer of all cells to a big centrifuge pot and centrifugation for 10-20 min at 4°C and 4,500 rpm.</LI><LI>Resuspend the pellet in 20-30 ml <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Media">Impact Lysis buffer</A> and transfer it to BlueCaps.</LI><LI>Cell disruption with the French Press (at least 2 times; max volume 35-40 ml) and keep the suspension on ice.  Store 50 µl of the suspension at -20°C for SDS-PAGE.</LI><LI>Centrifugation of the cell remains (4,500 rpm, 1 h, 4°C) and collect the supernatant in BlueCaps on ice. Store 50 µl of the suspension at -20°C for SDS-PAGE.</LI></UL><H3>Preparation of the columns</H3><UL><LI>Seal the bottom of the column with Parafilm and pipet 5 ml water in it. Mark the fill level and let the water run out</LI><LI>Fill the column with chitin beads up to the marked place and put a filter designator on the material. </LI><LI>Equilibrate the column with 10 volumes of <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Media">Impact column buffer</A> (Low-Salt; only 2.92 g on 1 L instead of 58.44 g). </LI></UL><H3>Loading of the column</H3><UL><LI>Give the clean supernatant of the cell harvesting on the column and let it flow through the column very slowly (flow through rate of a maximum of 0.5-1 ml/min). Store 50 µl of the flow through at -20°C for SDS-PAGE.</LI></UL><H3>Washing of the column</H3><UL><LI>Wash the column with 10 volumes of <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Media">Impact column buffer</A>(High-Salt).</LI></UL><H3>Induction of the cleavage</H3><UL><LI>Washing of the column with 3 volumes of <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Media">Impact cleavage buffer</A>. This step should not take more time than 30-60 min.</LI><LI>Cleavage for 16-40 h at 23°C in the dark</LI><LI>For better cleavage the amino acid which is fused with the intein tag should be Tyr, Phe, Gln, Asn, Thr, Lys, Ala, His, Leu or Met for C-terminal cleavage or Met, Ala, Gln, Gly, Leu, Asn, Trp, Phe or Tyr for N-terminal cleavage. </LI></UL><H3>Elution</H3><UL><LI>Elution with 1-2 volumes of <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Media">Impact column buffer</A> (Low-Salt) in fresh BlueCap. Store 50 µl of the elution at -20°C for SDS-PAGE.</LI><LI>Usage of a suitable protein filter to get a higher protein concentration. Centrifugation for 45 min at 4500-5000 rpm and 4°C until you have 250 µl left.</LI><LI>Wash the protein solution 2-3 times with the <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Media">Impact protein-wash buffer</A>.</LI><LI>Bradford-Assay to calculate the protein concentration.</LI><LI>SDS-PAGE with 8 µg of the protein mixtures and 2 µg of the eluate</LI><LI>The protein can be stored with 50% glycerin at -20°C.</LI></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#Western">Western Blot</A></H4></DIV><DIV id="Western" class="panel-collapse collapse"><DIV class="panel-body"><H3>Transfere</H3><UL><LI>Run a SDS gel</LI><LI>Soak 6 filter papers and a nitrocellulose membrane in your transfere buffer for 15 minutes.</LI><LI>Stack 3 filter papers, the membrane, the SDS gel and another 3 filter papers in your semi-dry blotter. The nitrocellulose membrane should be nearest to the positive electrode.</LI><LI>Blot the gel to the membrane according to the specifications of your blotter.</LI><LI>Mark your Protein Ruler with a pencil, so it will be visible after staining the membrane.</LI></UL><H3>Staining</H3><UL><LI>Block the membrane for 1 hour in transfere buffer with 3% BSA.</LI><LI>Dilute your primary antibody in transfere buffer with 3% BSA and incubate the membrane with it for 1.5 hours. </LI><LI>Wash the membrane 5 times for 5 minutes each with fresh transfere buffer.</LI><LI>Dilute your secondary antibody in transfere buffer with 3% BSA and incubate the membrane with it for 1 hour. </LI><LI>Wash the membrane 5 times for 5 minutes each with fresh transfere buffer.</LI><LI>For chemiluminescent detection, follow the instructions for the reagent you are using. Acquire image.
</LI></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#BLItz">Bio-layer interferometry (BLItz)</A></H4></DIV><DIV id="BLItz" class="panel-collapse collapse"><DIV class="panel-body"><H3>Immobilisation</H3><UL><LI>Activate sensor by incubating it in 250 µL EDC/NHS solution for 2 minutes. </LI><LI>Immobilize your first protein on the sensor by incubating the sensore in 250 µL adequate diluted protein solution for 5 minutes.</LI><LI>Wash the sensor by incubating it in 250 µL of the buffer you are using to dilute your protein for 2 minutes.</LI></UL><H3>Measuring</H3><UL><LI>Measure a baseline by incubating the sensor in 250 µL of your dilution buffer for 5 minutes.</LI><LI>Incubate the sensor in 250 µL adequate diluted protein solution of your second protein for 10 minutes to measure the association of your second to your first protein.</LI><LI>Wash the sensor in 250 µL of your dilution buffer for 5 minutes to measure the dissociation of your second protein from your first protein.</LI><LI>To reactivate the sensor, incubate it for 5 minutes in 250 µL 1M NaOH.</LI></UL></DIV></DIV></DIV></UL></DIV></DIV></DIV></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>