Team:Bielefeld-CeBiTec/Project/Selection/Motivation
Toggle navigation
NavBar
PROJECT
Description
Library
Overview
Scaffolds
Design and Construction
Create your own Library
Mutation
Overview
Error prone Polymerase I
Genome wide mutator
Selection
Overview
Bacterial Two-Hybrid System
Alternative Selection System
Modeling
Overview
Model I
Model II
RESULTS
Overview
Library
Overview
Assembly
Phagemid Display
Sequencing
Mutation
Overview
Assembly
Reversion
Sequencing
Selection
Overview
Assembly
Expression Control
Interaction Control
Binding Control
in vivo Selection
Modeling
Fermentation
Parts
Best Basic Part
Best Composite Part
Best Part Collection
Improve a part
Proof of Concept
Demonstrate your Work
Medal Fulfillment
HUMAN PRACTICES
Overview
Engagement
Dictionary
Educational Outreach
Crowdfunding and Press
Entrepreneurship
Integrated Practices
Achievements
COLLABORATIONS
Overview
Düsseldorf
Lethbridge
Freiburg
NOTEBOOK
Overview
Lab Notebook
Library
Mutation
Selection
Primers
Lab Protocols
Media and Buffers
TEAM
Team Profile
Members
Bielefeld University
Contact
ACKNOWLEDGEMENT
Overview
Sponsors and Partners
Attributions
Selection
Overview
Motivation
Selection is a key mechanism of evolution like Darwin defined it. Biodiversity and life as we know it in all its complexity is scientificly not explicable without it. We wanted to design a system able to generate an Evobody that can gain affinity towards any target protein it is confronted with. So what better way is there to design our system than copying one of evolutions core principles and letting the bacteria with the best Evobody emerge by itself?
Having a library of Evobodies as well as a mutation system is a great way to guarantee a high variety of binding proteins.    However, there is still the need of a screening method to separate Evobodies that show affinity to the target protein from those    that do not. As we wanted our whole system to work in vivo utilizing directed evolution we settled for three different    approaches to screen for functional Evobodies:
Overview
Figure 1: Illustration of the bacterial two-hybrid system. At first two fusion proteins were expressed. The first protein contains a DNA binding domain (cI(434)) with the target protein (1). The second fusion protein contains the activation domain (RpoZ) with our created binding protein (2). At first the DNA binding domain binds at the specific binding site upstream of the reporter (3). If the binding protein can interact with our target (4) the RNA Polymerase I can be recruited and binds to the promoter (5). The result is the expression of the reporter gene, in example the beta-lactamase(6). No binding between the target and the binding protein (7) leads to no expression of the reporter gene (8). At the end only the bacteria survive with the activated reporter. These bacteria you can use to produce a lot of the binding proteins against your specific target (9).
The bacterial two-hybrid system , our main approach, consists of two fusion proteins and a reporter gene that grants selective advantage. One    fusion protein is the target protein fused to a DNA-binding domain while the other comprises the Evobody consisting of a binding protein with a RNA polymerase subunit. The    DNA-binding domain that is connected to the target protein binds upstream of the reporter gene, in our case coding for beta-lactamase.   If there is a certain affinity between the Evobody and the target protein, the polymerase subunit of the Evobody recruits the full enzyme and subsequently the beta-lactamase   is expressed ( Badran et al. 2016 ). Cultivated in an increasing concentration of ampicillin, only those bacteria expressing enough beta-lactamase will prevail. Since the expression    of beta-lactamase correlates to the affinity between Evobody and binding protein we can use the best grown bacteria to determine the sequence of the Evobody    which then could be used for production.
Figure 2: Illustration of the split-protein system. Only in case of the Evobody showing affinity towards the target protein the functionality of the split beta-lactamase(10) is restored, granting a ampicillin resistance to the bacterium that generates the Evobody.
The split-protein approach is also based on beta-lactamase as a resistance to ampicillin. The general idea is, to split a reporter protein    into two non-functional subunits. Then, a translational fusion of one half of the split-protein respectively and one of the two allegedly binding proteins   is created. In case of successful binding between the proteins, the fused split-protein halves should therefore restore the functional reporter protein. As afore mentioned we   chose beta-lactamase as a reporter, as it allows the application of selective pressure. Both, the Evobody and the target protein are fused to a different    half of the split-beta-lactamase. In case of a successful binding Evobody the bacterium in concern should be granted an ampicillin resistance due to a restored and    functional beta-lactamase ( Galarneau et al. 2002 ).
The TAT-hitchhiker selection works on a similar premise as the split-protein. Yet, in this case the beta-lactamase is not divided into two subunits    but instead its natural export signals are removed. this modified beta-lactamase forms a translational unit with the target protein whilst a signal peptide    for the Tat (Twin-Arginine Translocation) pathway is attached to the Evobody. Only in case of a funtionally binding Evobody to the target protein is the    beta-lactamase exported into the periplasm where it can counteract the ampicillin. Since the Tat pathway only exports correctly folded proteins, this approach also    provides an option to check if the proteins are expressed as we predicted ( Waraho, Dujduan & DeLisa, Matthew P. 2009 ).
In summary, we wanted a selection system to screen for Evobodies with high affinity to the target protein while being functional in vivo. We chose three approaches, of which the bacterial two-hybrid system proved to be the most promising system .
Bacterial Two-Hybrid System
Alternative Selection Systems
Results
References
Badran, Ahmed H.; Guzov, Victor M.; Huai, Qing; Kemp, Melissa M.; Vishwanath, Prashanth; Kain, Wendy et al. (2016): Continuous evolution of Bacillus thuringiensis toxins overcomes insect resistance. In: Nature 533 (7601), S. 58–63. DOI: 10.1038/nature17938.
Galarneau, Andre; Primeau, Martin; Trudeau, Louis-Eric; Michnick, Stephen W. (2002): Beta-lactamase protein fragment complementation assays as in vivo and in vitro sensors of protein protein interactions. In: Nature biotechnology 20 (6), S. 619–622. DOI: 10.1038/nbt0602-619.
Waraho, Dujduan; DeLisa, Matthew P. (2009): Versatile selection technology for intracellular protein-protein interactions mediated by a unique bacterial hitchhiker transport mechanism. In: Proceedings of the National Academy of Sciences of the United States of America 106 (10), S. 3692–3697. DOI: 10.1073/pnas.0704048106.
