<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Project_Selection_SplitBetaLactamaseHitchHiker skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Project/Selection/SplitBetaLactamaseHitchHiker</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container main"><DIV id="header"><NAV id="navigationbar" class="navbar navbar-default custom_navbar_style navbar-fixed-top"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="#">NavBar</A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-center"><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description"> PROJECT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description">Description</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds">Scaffolds</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design">Design and Construction</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/CreateYours">Create your own Library</A></LI></UL><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/EpPolI">Error prone PolymeraseI</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/Global">Genome wide mutator</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">Bacterial Two-Hybrid System</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/SplitBetaLactamaseHitchHiker">Alternative Selection System</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_I">Model I</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_II">Model II</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results"> RESULTS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Phage">Phagemid Display</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">Reversion</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/Cloning">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/ExpressionControl">Expression Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/InteractionControl">Interaction Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/BindingControl">Binding Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/KnockOutKnochIn"><I>in vivo</I> Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Modeling">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Fermentation">Fermentation</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Best Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Best Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Best Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts/Improve">Improve a part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Demonstrate your Work</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Medal_Fulfillment">Medal Fulfillment</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold"> HUMAN PRACTICES </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Dictionary">Dictionary</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks">Educational Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Crowdfunding">Crowdfunding and Press</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Achievements</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations"> COLLABORATIONS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Duesseldorf">DÃ¼sseldorf</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Lethbridge">Lethbridge</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Freiburg">Freiburg</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments"> NOTEBOOK </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook">Lab Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">Primers</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols">Lab Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Media">Media and Buffers</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team"> TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team">Team Profile</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Members">Members</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Contact">Contact</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement"> ACKNOWLEDGEMENT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Sponsors_And_Partners">Sponsors and Partners</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></DIV></DIV></NAV></DIV></DIV><DIV class="container main"><DIV class="jumbotron" style="background-image: url(https://static.igem.org/mediawiki/2016/8/89/Bielefeld_CeBiTec_2016_10_14_X_jessi2.png)"><DIV class="jumbotron-text"><H1 style="margin-bottom: 0px; text-align:left">Selection</H1><H2 style="color:#ffffff; text-align:left">Alternative Selection System</H2></DIV></DIV><DIV class="container text_header"><H3>Overview</H3></DIV><DIV class="container text"> While our main approach, the <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">bacterial two-hybrid system</A>, is the optimal approach for our selection, we proposed two alternative systems to apply positive selective pressure towards generation of functional Evobodies. In contrast to the bacterial two-hybrid system which relies on the expression of a selection marker, these methods work independent of transcriptional regulation. To screen bacteria <I>in vivo</I> for protein-protein interaction we proposed the usage of a split-protein system. For our split-protein selection system, we focused on two major kinds of split-protein selection systems that are both working with fusion proteins. The first one, later referred to as âsplit-protein systemâ, is realised by splitting a fully functional protein in two parts, fusing one half to our binding protein and the other half to our target protein. The second approach is known as âHitchhiker-systemâ. It is based on a beta-lactamase, that is missing a transport signal, fused to the target protein and a transport signal fused to the binding protein. </DIV><DIV class="container text_header"><H3>Choice of Antibiotic Resistance</H3></DIV><DIV class="container text">As a selective marker gene we chose the ampicillin resistance gene (<I>amp</I>) coding for TEM-1 beta-lactamase. Beta-lactamases are well-studied resistance enzymes of gram-negative bacteria against beta-lactam antibiotics. Beta-lactam antibiotics inhibit the synthesis of the peptidoglycan layer in the periplasm and therefore the proliferation of bacteria. If the beta-lactamase is present in the periplasm it can hydrolyze the beta-lactam ring, rendering the antibiotic useless (Wilke et al. 2005). A benefit of beta-lactamase as a resistance for our selection is, that it can grant resistance even at a low expression rate (Matagne et al. 1998). Therefore, even Evobodies with a low affinity to the target protein might be able to grow on low concentrations of ampicillin. In a process of continuously increasing the selective pressure, the chances of acquiring a functional Evobody is a lot better if low affinity Evobody bacteria are not eliminated instantly but instead are granted the possibility to adapt the Evobody to the target protein. Apart from our projects primary selection, the <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">bacterial two-hybrid system</A>, we proposed two additional methods to screen for binding Evobodies.</DIV><DIV class="container text_header"><H3 id="split">Split-Protein System</H3></DIV><CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 2: Illustration of the split-protein system.</B> Only in case of the Evobody showing affinity towards the target protein the functionality of the split beta-lactamase(<B>10</B>) is restored, granting a ampicillin resistance to the bacterium that generates the Evobody.
</FIGCAPTION></FIGURE></CENTER><DIV class="container text">The split-protein system works, like the name indicates, with two parts of a former functional protein, which are fused to the two proteins that should be screened for an interaction. A lot of different proteins are able to be split into half and to reassemble when fused to two interacting proteins (Ladant, Karimova 2000). By using a visual reporter like the green fluorescent protein or an selection marker like a beta-lactamase, the split-protein selection system can work as an indicator to screen cultures for protein-protein interaction (Shekhawat, Ghosh 2011). In our case however, we utilized it as an in vivo selection system (Porter, JR et al. 2007) to give a growth advantage to those bacteria producing the best Evobody. 
The split-protein approach relies on the Evobody and the target protein each forming a translational fusion with one half of the split-protein, respectively. It is important to design the split-protein in a way that each half by itself is not functional. As proposed in literature, we dissected the beta-lactamase between Gly196 and Leu198, deleting 197. This site is supposed to be across from the active site of the enzyme while dividing it into fragments of similar size (Galarneau et al. 2002). In case of successful binding between an Evobody and the target protein, the two halves of the beta-lactamase should be sterically close enough to restore its functionality, thus granting ampicillin resistance to the host bacterium.
</DIV><DIV class="container text_header"><H3 id="hitch">Hitchhiker-System</H3></DIV><DIV class="container text">Same as the split-protein system, the hitchhiker-system is used to study protein-protein interactions. But instead of splitting the protein itself the TEM beta-lactamase without any natural export signal is fused to one binding protein while a signal peptide for the Tat (Twin-Arginine Translocation) pathway is fused to the other protein. In case of successful binding between the two proteins, the beta-lactamase is exported via the Tat pathway into the periplasm where it provides a functional antibiotic resistance. The Tat hitchhiker selection also enacts as a control for the protein folding, as only correctly folded proteins are exported (Waraho, DeLisa 2009).
Alongside the secretory pathway (Sec pathway), The twin-arginine translocation pathway (Tat pathway) is used in E. coli to export proteins through the membrane. In contrast to the Sec pathway, only folded proteins are exported via the Tat pathway. The name of the Tat pathway is due to the twin-arginine leader motif (S/TRRXFLK) of the signal peptide. Responsible for the translocation in E. coli are four cytoplasmic membrane proteins: TatA, TatB, TatC and TatE, where TatE appears to be homologous to TatA (Natale et al. 2008). However, the exact functionality has not been completely researched yet. 
The signal peptide used for the hitchhiker export originates from pre-trimethylamine N-oxide (TMAO) reductase (TorA). This enzyme is known to be exported by the Tat translocase in E. coli. While this signal is fused to the Evobody, a TEM-1 beta-lactamase with a deleted natural secretion signal is fused to the target protein. If the Evobody is binding the target protein and the proteins are folded correctly, the complex is exported into the periplasm due to the signal peptide, which is removed during the process (Blaudeck et al. 2001). That way, only bacteria with functional Evobody are able to export the beta-lactamase into the periplasm where it operates as a resistance to ampicillin.
<CENTER><BUTTON><A class="button_link" href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection" role="button">Results</A></BUTTON></CENTER></DIV><DIV class="container text_header"><H3>References</H3></DIV><DIV class="container text"><UL><LI>Blaudeck, N.; Sprenger, G. A.; Freudl, R.; Wiegert, T. (2001): Specificity of signal peptide recognition in tat-dependent bacterial protein translocation. In Journal of bacteriology 183 (2), pp. 604â610. DOI: 10.1128/JB.183.2.604-610.2001.</LI><LI>Galarneau, Andre; Primeau, Martin; Trudeau, Louis-Eric; Michnick, Stephen W. (2002): Beta-lactamase protein fragment complementation assays as in vivo and in vitro sensors of protein protein interactions. In Nature biotechnology 20 (6), pp. 619â622. DOI: 10.1038/nbt0602-619.</LI><LI>Ladant, Daniel; Karimova, Gouzel (2000): Genetic systems for analyzing proteinâprotein interactions in bacteria. In Research in Microbiology 151 (9), pp. 711â720. DOI: 10.1016/S0923-2508(00)01136-0.</LI><LI>Natale, Paolo; Bruser, Thomas; Driessen, Arnold J. M. (2008): Sec- and Tat-mediated protein secretion across the bacterial cytoplasmic membrane--distinct translocases and mechanisms. In Biochimica et biophysica acta 1778 (9), pp. 1735â1756. DOI: 10.1016/j.bbamem.2007.07.015.</LI><LI>Porter, JR; Stains, C. I.; Segal, D. J.; Ghosh, I. (2007): Split beta-lactamase sensor for the sequence-specific detection of DNA methylation. In Analytical chemistry 79 (17), pp. 6702â6708. DOI: 10.1021/ac071163.</LI><LI>Shekhawat, S. S.; Ghosh, I. (2011): Split-Protein Systems: Beyond Binary Protein-Protein Interactions. In Current opinion in chemical biology 15 (6), pp. 789â797. DOI: 10.1016/j.cbpa.2011.10.014.</LI><LI>Waraho, Dujduan; DeLisa, Matthew P. (2009): Versatile selection technology for intracellular protein-protein interactions mediated by a unique bacterial hitchhiker transport mechanism. In Proceedings of the National Academy of Sciences of the United States of America 106 (10), pp. 3692â3697. DOI: 10.1073/pnas.0704048106.</LI><LI>Publication bibliography
Blaudeck, N.; Sprenger, G. A.; Freudl, R.; Wiegert, T. (2001): Specificity of signal peptide recognition in tat-dependent bacterial protein translocation. In Journal of bacteriology 183 (2), pp. 604â610. DOI: 10.1128/JB.183.2.604-610.2001.</LI><LI>Galarneau, Andre; Primeau, Martin; Trudeau, Louis-Eric; Michnick, Stephen W. (2002): Beta-lactamase protein fragment complementation assays as in vivo and in vitro sensors of protein protein interactions. In Nature biotechnology 20 (6), pp. 619â622. DOI: 10.1038/nbt0602-619.</LI><LI>Ladant, Daniel; Karimova, Gouzel (2000): Genetic systems for analyzing proteinâprotein interactions in bacteria. In Research in Microbiology 151 (9), pp. 711â720. DOI: 10.1016/S0923-2508(00)01136-0.</LI><LI>Natale, Paolo; Bruser, Thomas; Driessen, Arnold J. M. (2008): Sec- and Tat-mediated protein secretion across the bacterial cytoplasmic membrane--distinct translocases and mechanisms. In Biochimica et biophysica acta 1778 (9), pp. 1735â1756. DOI: 10.1016/j.bbamem.2007.07.015.</LI><LI>Porter, JR; Stains, C. I.; Segal, D. J.; Ghosh, I. (2007): Split beta-lactamase sensor for the sequence-specific detection of DNA methylation. In Analytical chemistry 79 (17), pp. 6702â6708. DOI: 10.1021/ac071163.</LI><LI>Shekhawat, S. S.; Ghosh, I. (2011): Split-Protein Systems: Beyond Binary Protein-Protein Interactions. In Current opinion in chemical biology 15 (6), pp. 789â797. DOI: 10.1016/j.cbpa.2011.10.014.</LI><LI>Waraho, Dujduan; DeLisa, Matthew P. (2009): Versatile selection technology for intracellular protein-protein interactions mediated by a unique bacterial hitchhiker transport mechanism. In Proceedings of the National Academy of Sciences of the United States of America 106 (10), pp. 3692â3697. DOI: 10.1073/pnas.0704048106.</LI><LI>Wilke, Mark S.; Lovering, Andrew L.; Strynadka, Natalie C. J. (2005): Beta-lactam antibiotic resistance: a current structural perspective. In Current opinion in microbiology 8 (5), pp. 525â533. DOI: 10.1016/j.mib.2005.08.016.</LI><LI>Matagne A, Lamotte-Brasseur J, FrÃ¨re JM. Catalytic properties of class A beta-lactamases: efficiency and diversity. Biochemical Journal. 1998;330(Pt 2):581-598.</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>