Team:Bielefeld-CeBiTec/Results/Mutation/Reversion
Toggle navigation
NavBar
PROJECT
Description
Library
Overview
Scaffolds
Design and Construction
Create your own Library
Mutation
Overview
Error prone Polymerase I
Genome wide mutator
Selection
Overview
Bacterial Two-Hybrid System
Alternative Selection System
Modeling
Overview
Model I
Model II
RESULTS
Overview
Library
Overview
Assembly
Phagemid Display
Sequencing
Mutation
Overview
Assembly
Reversion
Sequencing
Selection
Overview
Assembly
Expression Control
Interaction Control
Binding Control
in vivo Selection
Modeling
Fermentation
Parts
Best Basic Part
Best Composite Part
Best Part Collection
Improve a part
Proof of Concept
Demonstrate your Work
Medal Fulfillment
HUMAN PRACTICES
Overview
Engagement
Dictionary
Educational Outreach
Crowdfunding and Press
Entrepreneurship
Integrated Practices
Achievements
COLLABORATIONS
Overview
Düsseldorf
Lethbridge
Freiburg
NOTEBOOK
Overview
Lab Notebook
Library
Mutation
Selection
Primers
Lab Protocols
Media and Buffers
TEAM
Team Profile
Members
Bielefeld University
Contact
ACKNOWLEDGEMENT
Overview
Sponsors and Partners
Attributions
Determine Mutation Rates by Reversion Assays
Overview
We performed reversion assays to investigate our mutagenic constructs. In the first step , we confirmed that we obtain functional     revertants by sanger sequencing. Afterwards we quantified mutation rate of our Error Prone Polymerase I and our Genome Wide Mutator . At last we try to integrate the mutation feature into our complete system by assessing the mutation while continuous cultivation.
Introduction
To quantify mutagenesis rate exactly is a very tedious task, because a multitude of factors affect the mutation rate. The most important are:
the experimental setup
the exact used method
Classical approaches to measure mutation rate are distinguished in reversion or forward assays, respectively. The mutation reverts a stop     from a reporter (reversion assay) or mutates a protein in such a way that it gains a function e.g. the organism becomes resistant to     antibiotics (mostly rifampicin forward assay). Since the rise of high-throughput sequencing, this technique starts to replace reversion     assays as prime method for assessing the mutation rate due to higher sensitivity, resolution and additional information.
As the error prone polymerase I is mostly specific for ColE1 plasmids we used a reporter on a plasmid. Therefore, we decided to     perform reversion assays with a stop beta-lactamase on a ColE1 plasmid.
First steps
Before going on and quantifying the mutagenesis rate we wanted to show that, our mutagenesis process yield functional revertants.     Therefore, we applied a modified version of the mutagenesis protocol described in the literature (Alexander et al. 2014). In     summary, we transformed the stop beta-lactamase plasmid pLA230 into E. coli JS200 cells carrying pHSG-EPPolI plasmids.     After regeneration, the cells were used to inoculate a 37 °C prewarmed LB culture. After 24 hours, cells were plated in     various dilution level at LB plates containing ampicillin. 10 colonies were picked, grown at 30 °C, the plasmids were isolated and sequenced.
Figure 1: Sequencing of  revertants, showing reversion of the stop codon. revertant colonies were grown in selective media       and sequenced.
The sequencing results show that the obtained revertants are really revertants, which mutated one base inside the stop codon to obtain     fully functional beta lactamase. In some cases the reversion is not completely obvious and is only hinted as two different peaks at the same position in the interferogram.
Error Prone Polymerase I
The function of the error prone polymerase I in our experimental setup was validated by the identification of E. coli clones     with different mutations. Further characteriation of the error-prone polymerase I involves the determination of the mutation rate     and adapting for optimal use in the Evobody generation system.
A literature review revealed the technique of mutation accumulation, where the fraction of revertants f at two points with a large     difference in cell count N is measured. Examples are measurement in the beginning and in the end of a cultivation.
Equation 1: Determination of mutation rate by mutant accumulation (Pope et al. 2008). and sequenced.
However, critical points have to be considered, when calculating the mutation rate. The reporter is encoded on plasmid with copy     numbers per cell ranging from 100 to 300. We postulate that one reversed beta-lactamase gene can confer ampicillin resistance to     the cell. Therefore, the observed mutation rate is 100 to 300 fold higher than the true mutagenesis rate. Additionally, potential     reversion sites on each plasmid contribute to the underestimation of the mutation rate. The stop codon TAA can mutate into nine     different codons, which include two new stop codon (TGA, TAG). In total, there are seven mutation possibilities on every plasmid     which could revert the stop beta-lactamase back to its functional state. These considerations lead to the modification of equation     one to equation and.
Equation 2: Modified equation to determine the mutation rate by mutant accumulation considering a reporter with a stop codon on a plasmid.
To measure the mutation rate similar conditions for each replicate were choosen. The setup looked as following. JS200 cells     carrying either pHSG-EP Pol I or pHSG-WT Pol I were transformed with 10 ng pLA230, regenerated for 60 min in SOC and transferred     LB with chloramphenicol and kanamycin. After 24 h the cell count was determined by OD600, the cells were serial diluted and appropriate     dilutions were plated on LB plates with ampicillin and grown at 30°C.
The reversion frequency f was determined by counting ampicillin resistant colonies and normalizing this value by the number of cells     measured via OD600. Therefore, we assumed OD600 as 5×108 cells. This experiment was done in five biological replicates     both with the error prone polymerase I as well as with the wild type polymerase I.
Equation 2: Modified equation to determine the mutation rate by mutant accumulation considering a reporter with a stop codon on a plasmid.
An increase in reversion frequency by a factor of 2.7×103 when using the EP Pol I in contrast to WT Pol I     was observed. In this evolution of the reversion experiment the controls pre mutagenesis were not plated, which prevents     exact determination of the change in reversion frequency. This leads to us using plating to determine the viable cell count in addition to the revertant count.
Due to obvious difference between the error-prone and the wild type polymerase I, we reduced consumption of material by only     characterizing the error prone polymerase in further detail. Major obstacles when trying to measure revertants in wild-type     cultures were the trait offs in respect to the number of plated cells. Plating too few cell leads to no revertants, while     plating too much cells leads to an unspecific growth of a strange bacterial lawn of not resistant cells on top of other dead     cells. However, this experiment clearly showed that the number of mutations caused by the wild type polymerase is not measureable     in this experiment.
Our next experiment was tailored to the Evobody generation process. In this proof of concept  experiment Evobody-producing E. coli are selected based on their different growth rates. Simultaneously,the Evobody coding sequence was mutagenized.     The recommended mutagenesis protocol for the EP Pol I includes periodic retransformation of the mutagenized plasmids. However,     this is not suitable to our approach. We started with a library and therefore avoided transformations to keep the diversity as     high as possible. Transformations are known bottle necks, while working with libraries. Therefore, we investigated the mutation     accumulation rate by inoculating LB culture with colonies grown at 30°C and growing them to saturation at 37°C. To calculate the     mutation accumulation according to equation 2 the reversion frequency were determined at two different points of the cultivation.
Figure 2: Reversion frequency of E. coli JS200 with EP Pol I at different time points. JS200 carrying the     EP Pol I and the stop lactamase plasmid were cultivated for 24 h in LB at 37°C (biological replicates A-C).     The reversion frequency was determined after 0 h and 24 h by plating and counting the fraction of ampicillin resistance     colonies. The reversion frequency increases over time by the factor four.
This cultivation experiment revealed two interesting results:
The mutation frequency within JS200 colonies containing the EP Pol I grown at the permissive temperature      of 30°C is still notable
The reversion frequency increases after cultivation at 37°C by the factor of four.
Both results show striking similarity to a previously determined mutation rate at the permissive temperature (Camps, personal communication).     Camps told us, that mutations still occur at 30°C, but the rate is reduced to only 25% of the mutation rate at 37°C.
Figure 3: Reversion frequency after 0 h and 24 h (red) and the calculated difference (blue). The reversion frequency of EP Pol in       JS200 was determined after 0 h and 24 h by plating (red). For calculating the mutation rate via equation one the difference       in mutation frequency was determined (blue). The error bars represent the standard deviation of three biological replicates.
Another observation is the huge variability in the reversion frequency at the starting point. This variability is much lower for     the mutation rate after 24 h. This shows that mutations, which occur early into a mutagenic process, take an overproportional part     of the whole culture. Therefore, exact determination of the mutation rate is only possible by measuring the mutation accumulation     (f2-f1) and not the absolute mutation count (Pope et al. 2008). Using the results from Figure 3 for f2-    f1 = (9.47±2.28)×10-5 and Figure 2 for N0h = 0.1 OD600 and N24h =     2.44 OD600 and a plasmid copy number of 10 (Camps et al. 2003) in equation 2 this cumulates to a mutation rate of
This measures mutation rate is three orders of magnitude over the basal mutation rate of E. coli mutation rate (Lee et al. 2012).     It strongly differs from the rate described by Camps of 8.1×10-4 per bp which converts to 4.8×10-5 per bp     per generation under the assumption that Camps cultivation process involves ~16 generations.
Furthermore, Camps uses a completely different approach in quantifiying the mutagenesis rate by starting from one measurement of the     reversion frequency f and using the plasmid copynumber (10) and reversion sites (7) and conversion between codon and base (3)     (Camps et al. 2003). His equitation looks like
When using this formula we obtain a mutation rate of 1.11×10-5 per bp, which is 72-fold lower than the one calculated by Camps.     In our opinion, the method of mutant accumulation is a more reliable method to discern mutation rate. Especially our experiment     where we used three independent cultures is a very descriptive example for this. Although the total reversion frequency at the start     as well as at our end point differs strongly between the cultures, the change in reversion frequency between both measured time points     is surprisingly low.
Genome Wide Mutator
We performed reversion assays with our genome wide mutators dnaQ926( BBa_K2082116 )     and M6( BBa_K2082117 ) again using the stop lactamase plasmid pLA230.
Our first approach was to cotransform the reporter plasmid and pSB1C3:dnaQ into E coli Top10 and inoculating cultures     containing arabinose or glucose directly with the transformed cells. This yielded a reversion frequency of 2.6×10-5 induced and 1.3×10-6 uninduced after 12 h. Because we did not plate the cells out for exact determination of the cell     count this result is only a first hint at an increases mutation rate through activation of our mutator.
Our next step was to transform the mutator and the reporter plasmids in E coli Top10, plate this on LBCmKan, inoculate LBCmKan     cultures and induced or repress this culture when reaching mid-log phase. The cells were plated after 24 h and counted.     From this experiment, the plasmids for our high-throughput experiment were prepared.
Figure 4: Reversion frequency measured in our first experiment with the genome wide mutators. The reversion rate is probably     not correct (see text), because no plating to count the exact cell count was done. This was included into our next experiments.    The results show an increased reversion frequency in samples supplemented with glucose. In theory the addition of glucose     should repress the PBAD activity, thus decreasing expression of mutagenic proteins.
For the observed smaller reversion frequency of the activated mutagenesis parts there can be two explanations.
The exact cell count was not determined by plating but only by OD600. Because expression of the mutator proteins     decreases cell viability (Badran and Liu 2015) the measured OD600 value probably contains large amount of dead cells,     therefore greatly inflating our supposed cell count. Badran found that induction of their M6 variant decreases cell viability     to ~5 %. This would result that the accessed total cell count is 20x too high in the induced samples, resulting in a 20x to     low reversion frequency.
Another possibility is that the arabinose promoter was active through very low levels of arabinose in LB. It is possible     that even very low levels of leakiness can lead to some mutagenic proteins. For example in the case of dnaQ926 one     protein is enough to create one proofreading deficient polymerase III complex. In normal E. coli strains this should be no problem but the Top10 strain is deficient for the arabinose metabolism and therefore low levels of     arabinose are not metabolized but activate promoter activity.
We concluded that in order to obtain more reliable data we had to plate dilutions with low amounts of cells in order to     determine the viable cell count in our samples. To combat the supposed problem of promoter leakiness in LB media we dicided     to use LB supplemented with 20 mM glucose. This is analog to the protocol from Badran and coworkers who used a defined media with glucose.
Using all this experience, we designed another experiment to determine the mutation rate.
We transform E.coli Top10 with the reporter plasmid as well as the mutator plasmid (pSB1C3:BBa_K2082116 or BBa_K2082117), plate a portion     and use another portion to inoculate a LBCmKan culture with 20 mM glucose, let this culture grow till mid-log phase and induce     or repress the PBAD. 24 h after induction the cells were plated on LBCmKan or LBAmp each with 20 mM glucose and     counted revertants and the total viable cell number. This experiment was performed for each of our mutator in six replicates.
Figure 5: Figure: Beta lactamase reversion frequency using dnaQ926 (BBa_K2082116) or M6 (BBa_K2082117) as mutator.
Figure 6: Statistic of the beta lactamase reversion experiment using dnaQ926 (BBa_K2082116) or M6 (BBa_K2082117) as mutator.
For the growth term in Equation 2 we measure the cell count right after transformation and while plating after 24 h. Therefore we     could determine N2/N1. As plasmid copy number we used n=40 as determined in our NGS experiment .
f2-f1
N2/N1
μ [bp-1×generation-1]
dnaQ926 (BBa_K2082116)
Induced
(3.92±1.91)×10-4
70.11
(2.3±1.12)×10-6
Repressed
(2.30±1.41)×10-5
5530,43
(6.67±4.09)×10-8
M6 (BBa_K2082117)
Induced
(1.86±1.12)×10-3
660,85
(7.16±4.31)×10-6
Repressed
(0.82±1.04)×10-5
3538,52
(2.51±3.18)×10-8
Figure 7: Mutagensis rate of dnaQ926 (BBa_K2082116) and M6 (BBa_K2082117).
We determined the mutation rate of our genome wide mutator BBa_K2082117 to be (7.16±4.31)×10-6 bp-1×generation-1 induced and (2.51±3.18)×10-8 bp-1×generation-1 when repressed.    This is almost exactly the same rate Badran and coworkers determined for their iteration of the M6 mutator (Badran and Liu, 2015), who determined a mutation rate of    6.2×10-6 bp-1×generation-1 induced and 3×10-9 bp-1×generation-1.    Therfore our aim to reconstruct this mutator with definded iGEM parts was successful.    Furthermore we showed the muation rate of BBa_K1333108 in our construct BBa_K2082116 to be (2.3±1.12)×10-6 bp-1×generation-1 induced and (6.67±4.09)×10-8 bp-1×generation-1.    Thereby we show the functionality of this already existing part.
Summary
We showed mutation using the error prone polymerase I, which is 4-5 orders of magnitude higher than basal mutation rate.
We determined the mutation rate of the error prone polymerase I and our genome wide mutator systems. Both systems produced mutations with a     frequency several orders of magnitude over the background. Therefore, our parts can be used be in vivo generation of protein libraries     to optimze enzymes or proteins for various purposes.
After determining the mutation rate of our construct we also investigated them with high-throughput sequencing
Sequencing
References
Alexander, David L.; Lilly, Joshua; Hernandez, Jaime; Romsdahl, Jillian; Troll, Christopher J.; Camps, Manel (2014): Random mutagenesis by error-prone pol plasmid replication in Escherichia coli. In: Methods in molecular biology (Clifton, N.J.) 1179, S. 31–44. DOI: 10.1007/978-1-4939-1053-3_3.
Badran, Ahmed H.; Liu, David R. (2015): Development of potent in vivo mutagenesis plasmids with broad mutational spectra. In: Nature communications 6, S. 8425. DOI: 10.1038/ncomms9425.
Camps, Manel; Naukkarinen, Jussi; Johnson, Ben P.; Loeb, Lawrence A. (2003): Targeted gene evolution in Escherichia coli using a highly error-prone DNA polymerase I. In: Proceedings of the National Academy of Sciences of the United States of America 100 (17), S. 9727–9732. DOI: 10.1073/pnas.1333928100.
Lee, Heewook; Popodi, Ellen; Tang, Haixu; Foster, Patricia L. (2012): Rate and molecular spectrum of spontaneous mutations in the bacterium Escherichia coli as determined by whole-genome sequencing. In: Proceedings of the National Academy of Sciences of the United States of America 109 (41), S. E2774-83. DOI: 10.1073/pnas.1210309109.
