<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Project_Selection_Motivation skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Project/Selection/Motivation</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container main"><DIV id="header"><NAV id="navigationbar" class="navbar navbar-default custom_navbar_style navbar-fixed-top"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="#">NavBar</A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-center"><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description"> PROJECT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description">Description</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds">Scaffolds</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design">Design and Construction</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/CreateYours">Create your own Library</A></LI></UL><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/EpPolI">Error prone PolymeraseI</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/Global">Genome wide mutator</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">Bacterial Two-Hybrid System</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/SplitBetaLactamaseHitchHiker">Alternative Selection System</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_I">Model I</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_II">Model II</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results"> RESULTS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Phage">Phagemid Display</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">Reversion</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/Cloning">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/ExpressionControl">Expression Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/InteractionControl">Interaction Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/BindingControl">Binding Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/KnockOutKnochIn"><I>in vivo</I> Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Modeling">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Fermentation">Fermentation</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Best Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Best Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Best Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts/Improve">Improve a part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Demonstrate your Work</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Medal_Fulfillment">Medal Fulfillment</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold"> HUMAN PRACTICES </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Dictionary">Dictionary</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks">Educational Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Crowdfunding">Crowdfunding and Press</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Achievements</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations"> COLLABORATIONS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Duesseldorf">Düsseldorf</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Lethbridge">Lethbridge</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Freiburg">Freiburg</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments"> NOTEBOOK </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook">Lab Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">Primers</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols">Lab Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Media">Media and Buffers</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team"> TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team">Team Profile</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Members">Members</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Contact">Contact</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement"> ACKNOWLEDGEMENT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Sponsors_And_Partners">Sponsors and Partners</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></DIV></DIV></NAV></DIV></DIV><DIV class="container main"><DIV class="jumbotron" style="background-image: url(https://static.igem.org/mediawiki/2016/8/89/Bielefeld_CeBiTec_2016_10_14_X_jessi2.png)"><DIV class="jumbotron-text"><H1 style="margin-bottom: 0px; text-align:left">Selection</H1><H2 style="color:#ffffff; text-align:left">Overview</H2></DIV></DIV><DIV class="container text_header"><H3>Motivation</H3></DIV><DIV class="container text"> Selection is a key mechanism of evolution like Darwin defined it. Biodiversity and life as we know it in all its complexity is scientificly not explicable without it. We wanted to design a system able to generate an Evobody that can gain affinity towards any target protein it is confronted with. So what better way is there to design our system than copying one of evolutions core principles and letting the bacteria with the best Evobody emerge by itself?
  Having a <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">library</A> of Evobodies as well as a <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">mutation system</A> is a great way to guarantee a high variety of binding proteins. 
  However, there is still the need of a screening method to separate Evobodies that show affinity to the target protein from those 
  that do not. As we wanted our whole system to work <I>in vivo</I> utilizing directed evolution we settled for three different 
  approaches to screen for functional Evobodies:
</DIV><DIV class="container text_header"><H3>Overview</H3></DIV><CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 1: Illustration of the bacterial two-hybrid system.</B> At first two fusion proteins were expressed. The first protein contains a DNA binding domain (cI(434)) with the target protein (<B>1</B>). The second fusion protein contains the activation domain (RpoZ) with our created binding protein (<B>2</B>). At first the DNA binding domain binds at the specific binding site upstream of the reporter (<B>3</B>). If the binding protein can interact with our target (<B>4</B>) the RNA Polymerase I can be recruited and binds to the promoter (<B>5</B>). The result is the expression of the reporter gene, in example the beta-lactamase(<B>6</B>). No binding between the target and the binding protein (<B>7</B>) leads to no expression of the reporter gene (<B>8</B>). At the end only the bacteria survive with the activated reporter. These bacteria you can use to produce a lot of the binding proteins against your specific target (<B>9</B>).
</FIGCAPTION></FIGURE></CENTER><DIV class="container text">The <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">bacterial two-hybrid system</A>, our main approach, consists of two fusion proteins and a reporter gene that grants selective advantage. One 
  fusion protein is the target protein fused to a DNA-binding domain while the other comprises the Evobody consisting of a binding protein with a RNA polymerase subunit. The 
  DNA-binding domain that is connected to the target protein binds upstream of the reporter gene, in our case coding for beta-lactamase.
  If there is a certain affinity between the Evobody and the target protein, the polymerase subunit of the Evobody recruits the full enzyme and subsequently the beta-lactamase
  is expressed (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation#badran">Badran et al. 2016</A>). Cultivated in an increasing concentration of ampicillin, only those bacteria expressing enough beta-lactamase will prevail. Since the expression 
  of beta-lactamase correlates to the affinity between Evobody and binding protein we can use the best grown bacteria to determine the sequence of the Evobody 
  which then could be used for production.
  <CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 2: Illustration of the split-protein system.</B> Only in case of the Evobody showing affinity towards the target protein the functionality of the split beta-lactamase(<B>10</B>) is restored, granting a ampicillin resistance to the bacterium that generates the Evobody.
</FIGCAPTION></FIGURE></CENTER>
  The <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/SplitBetaLactamaseHitchHiker#split">split-protein</A> approach is also based on beta-lactamase as a resistance to ampicillin. The general idea is, to split a reporter protein 
  into two non-functional subunits. Then, a translational fusion of one half of the split-protein respectively and one of the two allegedly binding proteins
  is created. In case of successful binding between the proteins, the fused split-protein halves should therefore restore the functional reporter protein. As afore mentioned we
  chose beta-lactamase as a reporter, as it allows the application of selective pressure. Both, the Evobody and the target protein are fused to a different 
  half of the split-beta-lactamase. In case of a successful binding Evobody the bacterium in concern should be granted an ampicillin resistance due to a restored and 
  functional beta-lactamase (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation#galarneau">Galarneau et al. 2002</A>).
  
  The <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/SplitBetaLactamaseHitchHiker#hitch">TAT-hitchhiker selection</A> works on a similar premise as the split-protein. Yet, in this case the beta-lactamase is not divided into two subunits 
  but instead its natural export signals are removed. this modified beta-lactamase forms a translational unit with the target protein whilst a signal peptide 
  for the Tat (Twin-Arginine Translocation) pathway is attached to the Evobody. Only in case of a funtionally binding Evobody to the target protein is the 
  beta-lactamase exported into the periplasm where it can counteract the ampicillin. Since the Tat pathway only exports correctly folded proteins, this approach also 
  provides an option to check if the proteins are expressed as we predicted (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation#waraho">Waraho, Dujduan &amp; DeLisa, Matthew P. 2009</A>).
  
  In summary, we wanted a selection system to screen for Evobodies with high affinity to the target protein while being functional <I>in vivo</I>. We chose three approaches, of which the <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">bacterial two-hybrid system</A> proved to be the <A href="*">most promising system</A>.
 </DIV><CENTER><BUTTON><A class="button_link" href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System" role="button">Bacterial Two-Hybrid System</A></BUTTON><BUTTON><A class="button_link" href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/SplitBetaLactamaseHitchHiker" role="button">Alternative Selection Systems</A></BUTTON><BUTTON><A class="button_link" href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection" role="button">Results</A></BUTTON></CENTER><DIV class="container text_header"><H3>References</H3></DIV><DIV class="container text"><UL><LI id="badran">Badran, Ahmed H.; Guzov, Victor M.; Huai, Qing; Kemp, Melissa M.; Vishwanath, Prashanth; Kain, Wendy et al. (2016): Continuous evolution of Bacillus thuringiensis toxins overcomes insect resistance. In: Nature 533 (7601), S. 58–63. DOI: 10.1038/nature17938.</LI><LI id="galarneau">Galarneau, Andre; Primeau, Martin; Trudeau, Louis-Eric; Michnick, Stephen W. (2002): Beta-lactamase protein fragment complementation assays as in vivo and in vitro sensors of protein protein interactions. In: Nature biotechnology 20 (6), S. 619–622. DOI: 10.1038/nbt0602-619.</LI><LI id="waraho">Waraho, Dujduan; DeLisa, Matthew P. (2009): Versatile selection technology for intracellular protein-protein interactions mediated by a unique bacterial hitchhiker transport mechanism. In: Proceedings of the National Academy of Sciences of the United States of America 106 (10), S. 3692–3697. DOI: 10.1073/pnas.0704048106.</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>