<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Project_Selection_Bacterial_Two-Hybrid_System skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Project/Selection/Bacterial Two-Hybrid System</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container main"><DIV id="header"><NAV id="navigationbar" class="navbar navbar-default custom_navbar_style navbar-fixed-top"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="#">NavBar</A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-center"><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description"> PROJECT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Description">Description</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Scaffolds">Scaffolds</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/Design">Design and Construction</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library/CreateYours">Create your own Library</A></LI></UL><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/EpPolI">Error prone PolymeraseI</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Mutation/Global">Genome wide mutator</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Motivation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System">Bacterial Two-Hybrid System</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/SplitBetaLactamaseHitchHiker">Alternative Selection System</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Model">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_I">Model I</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Modeling/Model_II">Model II</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results"> RESULTS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Phage">Phagemid Display</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Library/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Assembly">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Reversion">Reversion</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Mutation/Sequencing">Sequencing</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/Cloning">Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/ExpressionControl">Expression Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/InteractionControl">Interaction Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/BindingControl">Binding Control</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Selection/KnockOutKnochIn"><I>in vivo</I> Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Results/Modeling">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Fermentation">Fermentation</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Best Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Best Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Best Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Parts/Improve">Improve a part</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Demonstrate your Work</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Medal_Fulfillment">Medal Fulfillment</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold"> HUMAN PRACTICES </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Gold">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Dictionary">Dictionary</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/School_Talks">Educational Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Crowdfunding">Crowdfunding and Press</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Achievements</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations"> COLLABORATIONS </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Duesseldorf">Düsseldorf</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Lethbridge">Lethbridge</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Collaborations/Freiburg">Freiburg</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments"> NOTEBOOK </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments">Overview</A></LI><LI class="navbar_button dropdown dropdown-submenu"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook">Lab Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Library">Library</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Mutation">Mutation</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Notebook/Selection">Selection</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Primers">Primers</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Protocols">Lab Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Experiments/Media">Media and Buffers</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team"> TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team">Team Profile</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Members">Members</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Team/Contact">Contact</A></LI><LI class="navbar_button dropdown"><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement"> ACKNOWLEDGEMENT </A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Acknowledgement">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Sponsors_And_Partners">Sponsors and Partners</A></LI><LI><A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></DIV></DIV></NAV></DIV></DIV><DIV class="container main"><DIV class="jumbotron" style="background-image: url(https://static.igem.org/mediawiki/2016/8/89/Bielefeld_CeBiTec_2016_10_14_X_jessi2.png)"><DIV class="jumbotron-text"><H1 style="margin-bottom: 0px; text-align:left">Selection</H1><H2 style="color:#ffffff; text-align:left">Bacterial Two-Hybrid System</H2></DIV></DIV><DIV class="container text_header"><H3>Motivation and Overview</H3></DIV><DIV class="container text">
It is utmost importance to our project to separate bacteria  with high affinity binding proteins from bacteria with low or moderate affinity binding proteins.  Therefore, we need an efficient selection system. One approach 
is based on the concept of the very common yeast two-hybrid system, but optimized for bacterial selection directly in <I>E. coli</I> (<A href="#Badran2016">Badran et al. 2016</A>).
			
An overview of the system in action is given in Figure(1).
			<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 1: Illustration of the bacterial two-hybrid system.</B> At first two fusion proteins were expressed. The first protein contains a DNA binding domain (cI(434)) with the target protein (<B>1</B>). The second fusion protein contains the activation domain (RpoZ) with our created binding protein (<B>2</B>). At first the DNA binding domain binds at the specific binding site upstream of the reporter (<B>3</B>). If the binding protein can interact with our target (<B>4</B>) the RNA Polymerase I can be recruited and binds to the promoter (<B>5</B>). The result is the expression of the reporter gene, in example the beta-lactamase(<B>6</B>). No binding between the target and the binding protein (<B>7</B>) leads to no expression of the reporter gene (<B>8</B>). At the end only the bacteria survive with the activated reporter. These bacteria you can use to produce a lot of the binding proteins against your specific target (<B>9</B>).
</FIGCAPTION></FIGURE></CENTER></DIV><DIV class="container text_header"><H3>Theoretical background</H3></DIV><DIV class="container text">
The bacterial two-hybrid system is a possibility to separate cells from each other through differences in protein-protein interaction strength(<A href="#Hu2000">Hu et al. 2000</A>). The whole system works <I>in vivo</I> and gives bacteria with strong interaction of two chosen proteins a high advantage in their growth ability. This growth difference distinguishes such bacteria from bacteria with very weak protein interactions. This method can be applied to our Evobody generation system by constructing a <B>hybrid transcriptional activator system</B>. This system comprises two fusion proteins. One protein contains a DNA binding domain fused with our target protein (<B>1</B>). The second protein consists of our Evobody fused to a subunit of the RNA polymerase(<B>2</B>). The DNA binding domain
of the first fusion protein binds to a specific DNA pattern upstream of the promoter sequence of a reporter gene (<B>3</B>).
If there is an interaction between the target protein and the Evobody (<B>4</B>), the interaction leads to a recruitment of the fused subunit of the RNA polymerase to the promoter (<B>5</B>). Therefore, the RNA polymerase can only express the reporter gene, for example a beta-lactamase, if the interaction of the target and the Evobody is strong enough (<B>6</B>). Otherwise, a weak or unavailable interaction (<B>7</B>) leads to no reporter gene transcription at all (<B>8</B>). Therefore, it is possible to raise the antibiotic concentration to increase the selection pressure on cells. The results are optimized Evobodies with high affinity towards the target (<B>9</B>).
		</DIV><DIV class="container text_header"><H3>Our system in detail</H3></DIV><DIV id="DBD" class="container text_header"><H4>The DNA binding domain</H4></DIV><DIV class="container text">
Our system is based on two fusion proteins. The first protein consists of our target protein fused with a DNA binding domain. In very early hybrid transcriptional activation system the frequently binding domains were different zinc finger proteins (<A href="#Joung2000">Joung et al. 2000</A>). Each of these proteins has a so called zinc finger domain, which is able to bind at specific DNA sequences (<A href="#Klug1987">Klug &amp; Rhodes 1987</A>). The zinc finger domain of the murein transcription factor Zif268 was most frequently used in two-hybrid systems. Three individual zinc finger motifs collectively bind a nine base-pair long sequence (<A href="#Pavletich1991">Pavletich &amp; Pabo 1991</A>).  Due to the development of more complicated experiments more zinc finger proteins per experiment were needed. This results in a big prone for faults, because the specificities of individiual zinc finer proteins can overlap and can depend on the context of surrounding zinc fingers and DNA (<A href="#Ramirez2008">Ramirez et al. 2008</A>). Off-target effects were observed instead of a high affinity to the intended target. The revision of the zinc fingers make them on the one hand way more effective but also on the other hand way more complicated to use (<A href="#Maeder2008">Maeder et al. 2008</A>; <A href="#Sander2011">Sander et al. 2011</A>).

Therefore, DNA binding domains, which are a lot easier to use for bacterial two-hybrid systems are necessary. Examples are, the repressor proteins cI of the phages lambda (<A href="#Dove2004">Dove &amp; Hochschild 2004</A>) and 434 (<A href="#Hays2000">Hays et al. 2000</A>), respectively. The protein family cI are repressor proteins that compete with the phage Cro proteins for DNA binding. 
The main function of them is the binding at the binding sites OR1 and OR2 on the DNA. If they bind at these sites they prevent that Cro can bind at the binding sites OR2 and OR3 and thereby inhibit the expression of the <I>cro</I> gene (<A href="#Badran2016">Brooks &amp; Clark 1967</A>). 
Further researches revealed that next to the binding sequences 
 
OR1 and OR2, also OL1 and OL2 downstream of the OR1 and OR2 sites are necessary for complete repression of the <I>cro</I> gene. Therefore, cI binds at the DNA as an octamere (<A href="#Dodd2001">Dodd et al. 2001</A>).

The sites OR1 and OR2 are important, because the binding domain is only used to anchor the fusion protein close to the promoter. Afterwards, gene expression is increased by attracting the activation domain which is fused to the binding protein (<A href="#Joung2000">Joung et al. 2000</A>). All cI protein have specific binding sequences namely OR1 and OR2. These cI binding sequences are species specific and therefore different between diverse varying phages isolates.  However, the design of a hybrid transcriptional activation system with cI as binding domain is easier than the use of a zinc finger protein. This is the due to the binding sequences for a specific cI, which does not require optimization like zinc fingers do.

Comparison of cI proteins to zinc finger proteins revealed a much higher expression rate of a reporter in a designed bacterial two hybrid system, if cI proteins is used as the DNA binding domain (<A href="#Badran2016">Badran et al. 2016</A>). Due to the easier design and higher expected profit, we decided to use the cI protein of the phage 434, as the DNA binding domain for our system.
		</DIV><DIV id="AD" class="container text_header"><H4>The activation domain</H4></DIV><DIV class="container text">
The second fusion protein of the hybrid transcriptional activation system is our Evobody fused to an activation domain.	This domain is the impeller of the system. If the activation domain is recruited to the promoter through the interaction of the binding protein (the Evobody) and the DNA anchored target protein the gene expression of the reporter gene is strongly increased(<A href="#Dove2004">Dove &amp; Hochschild 2004</A>). The most commonly used activation domain is the alpha subunit of the DNA-dependent RNA polymerase A or in short RpoA. Several early publications about bacterial one- or two-hybrid systems are based on this subunit as activator (<A href="#Joung2000">Joung et al. 2000</A>;<A href="#Dove2004">Dove &amp; Hochschild 2004</A>, <A href="#Durai2006">Durai et al. 2006</A>). RpoA is a subunit of the core complex of the DNA-dependent RNA polymerase. The interaction with the promoter as well as the connection to the regulatory proteins is established via RpoA. Therefore, it is essential for the initiation of the subunit assembly of the RNA polymerase (<A href="#Ishihama1992">Ishihama 1992</A>).
<CENTER><FIGURE class="figure"><CENTER><FIGCAPTION class="figure-caption"><B>Figure 6: RpoA and RpoZ highlighted in the RNA polymerase I crystal structure.</B> Image designed by PyMol.</FIGCAPTION></CENTER></FIGURE></CENTER>
RpoZ, the much smaller omega subunit of the RNA polymerase, which could also be used as transcriptional activator. It is mainly important for the facilitation and stabilization of the assembly of the RNA polymerase (<A href="#Mathew2006">Mathew &amp; Chatterji 2006</A>). RpoZ outperformed RpoA and a special phage polymerase subunit in a gene activation comparison (<A href="#Badran2016">Badran et al. 2016</A>). Therefore, we decided to use the RpoZ as activator domain. 
		</DIV><DIV id="Controls" class="container text_header"><H4>Choice of the positive controls</H4></DIV><DIV class="container text">	
Every system should be validate by analyzing a positive control. The two hybrid transcriptional activation system converts the binding affinity of two proteins in corresponding expression intensity. A well-documented protein-protein interaction is the binding of the regulator protein Gal4 of <I>Saccharomyces cerevisiae</I> with a modified version of the Gal11 regulatory protein Gal11P (<A href="#Jeong2001">Jeong et al. 2001</A>). The transformation from Gal11 to Gal11P results in a mutation in only one amino acid (<A href="#Himmelfarb1990">Himmelfarb et al. 1990</A>), which allows the binding of Gal11P in the dimerization domain of Gal4 (<A href="#Hidalgo2001">Hidalgo et al. 2001</A>).
			<FIGURE class="figure"><CENTER><FIGCAPTION class="figure-caption"><B>Figure 7: Comparison of Gal11 and Gal11P.</B>The only difference of Gal11 and Gal11P is the amino acid mutation N342A.</FIGCAPTION></CENTER></FIGURE>
In addition to the Gal4-Gal11P interaction a second positiv control should be tested. Having our <A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Library">library</A> in mind we looked for a control that was very similar to our Evobodies. One protein with good structural similarities 

is the antibody mimic or monobody HA4. This monobody was designed by a working group in Chicago searching for an antibody mimic, that can bind the SH2 domain of the tyrosine kinase Abelson with low nanomolar affinity (<A href="#Wojcik2010">Wojcik et al. 2010</A>). The results of this work show an equally well interaction <I>in vitro</I> and <I>in vivo</I>, respectively. To validate the potential of the hybrid transcriptional activation system HA4 mutants are also necessary. Single amino acid substitutions change in most cases the structure of the protein and lower or increase the interaction affinity with other proteins (<A href="#Brender2015">Brender &amp; Zhang 2015</A>). In single mutation R38A in HA4 causes a reduction of transcriptional activation of the reporter gene of 35 to 40%. The mutation Y87A is even more detrimental leading to only 5 to 10% of the native binding affinity (<A href="#Badran2016">Badran et al. 2016</A>).	 

These controls allowed a detail characterization of our system. It is possible to show that different binding affinities  result in differences in the reporter gene expression.   
		</DIV><DIV id="Reporter" class="container text_header"><H4>Design of the reporter</H4></DIV><DIV class="container text">	
A selection system should be characterized based on different proteins with known mutual affinity values. A strong correlation between the binding affinity of the interacting proteins and the reporter gene activity is required. The selection of hogh affiniy Evobodies necessitates a reporter that confers growth advantage to the cells of interest. A good choice for such a reporter is an antibiotic resistance gene like <I>bla</I> against ampicillin (<A href="#Livermore1995">Livermore 1995</A>) or <I>tetA</I> against tetracycline (<A href="#Roberts1996">Roberts 1996</A>), respectively. Increase of the antibiotic concentration would lead to a decrease of a bacterial population. Finally, the best Evobodies should allow the survival of the cells.
<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 8: Illustration of the antibiotic pressure.</B>At first all bacteria are alive(<B>1</B>). The first concentration of the antibiotics like ampicillin(<B>2</B>) leads to the first struggling bacteria, who have not enough beta-lactamase transcription to fight against it(<B>3</B>). Every increase of the antibiotica concentration (<B>4</B>) would kill morer and more bacteria, until the one bacterium with the best transcriptional activation rate of the reporter gene is the only one alive(<B>5</B>)</FIGCAPTION></FIGURE></CENTER>
The native <I>lacZ</I> pomoter was modified to achieve a ten times greater expression rate when bound by a cI-RpoZ fusion protein (<A href="#Badran2016">Badran et al 2016</A>). Moreover, the distance of the 434 cI binding site to the transcription start site was optimized by the working group (<A href="#Badran2016">Badran et al 2016</A>). The optimal distance of the binding site is exactly 61 basepairs upstream of the transcription start site (<A href="#Badran2016">Badran et al. 2016</A>).

The final design of our promoter is given in figure (9). 
<CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 9: Final promoter region.</B> The final reporter contains an optimized PlacZ with an OR1 DNA binding domain 61 base-pairs upstream of the transcription start site for optimal interaction conditions for the bacterial two-hybrid system.</FIGCAPTION></FIGURE></CENTER><CENTER><BUTTON><A class="button_link" href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/SplitBetaLactamaseHitchHiker" role="button">Alternative Selection Systems</A></BUTTON></CENTER><DIV class="container text_header"><H3>References</H3></DIV><DIV class="container text"><UL><LI id="Badran2016">Badran, A. H., Guzov, V. M. &amp; Huai, Q. et al. (2016) Continuous evolution of Bacillus thuringiensis toxins overcomes insect resistance. Nature 533 (7601), 58–63.</LI><LI id="Brender2015">Brender, J. R. &amp; Zhang, Y. (2015) Predicting the Effect of Mutations on Protein-Protein Binding Interactions through Structure-Based Interface Profiles. PLoS computational biology 11 (10), e1004494.</LI><LI id="Brooks1967">Brooks, K. &amp; Clark, A. J. (1967) Behavior of lambda bacteriophage in a recombination deficienct strain of Escherichia coli. Journal of virology 1 (2), 283–293.</LI><LI id="Dodd2001">Dodd, I. B., Perkins, A. J., Tsemitsidis, D. &amp; Egan, J. B. (2001) Octamerization of lambda CI repressor is needed for effective repression of P(RM) and efficient switching from lysogeny. Genes &amp; development 15 (22), 3013–3022.</LI><LI id="Dove2004">Dove, S. L. &amp; Hochschild, A. (2004) A bacterial two-hybrid system based on transcription activation. Methods in molecular biology (Clifton, N.J.) 261, 231–246.</LI><LI id="Durai2006">Durai, S., Bosley, A., Abulencia, AB., Chandraseqaran S. &amp; Ostermeier M. (2006) A bacterial one-hybrid selection system for interrogating zinc finger-DNA interactions. Comb Chem High Throughput Screen. 2006 May;9(4), 301-11</LI><LI id="Hays2000">Hays, L. B., Chen, Y. S. &amp; Hu, J. C. (2000) Two-hybrid system for characterization of protein-protein interactions in E. coli. BioTechniques 29 (2), 288-90, 292, 294 passim.</LI><LI id="Hidalgo2001">Hidalgo, P., Ansari, A. Z. &amp; Schmidt, P. et al. (2001) Recruitment of the transcriptional machinery through GAL11P: structure and interactions of the GAL4 dimerization domain. Genes &amp; development 15 (8), 1007–1020.</LI><LI id="Himmelfarb1990">Himmelfarb, H. J., Pearlberg, J., Last, D. H. &amp; Ptashne, M. (1990) GAL11P: a yeast mutation that potentiates the effect of weak GAL4-derived activators. Cell 63 (6), 1299–1309.</LI><LI id="Hu2000">Hu, J. C., Kornacker, M. G. &amp; Hochschild, A. (2000) Escherichia coli one- and two-hybrid systems for the analysis and identification of protein-protein interactions. Methods (San Diego, Calif.) 20 (1), 80–94.</LI><LI id="Ishiama1992">Ishihama, A. (1992) Role of the RNA polymerase alpha subunit in transcription activation. Molecular microbiology 6 (22), 3283–3288.</LI><LI id="Jeong2001">Jeong, C. J., Yang, S. H., Xie, Y., Zhang, L., Johnston, S. A. &amp; Kodadek, T. (2001) Evidence that Gal11 protein is a target of the Gal4 activation domain in the mediator. Biochemistry 40 (31), 9421–9427.</LI><LI id="Joung2000">Joung, J. K., Ramm, E. I. &amp; Pabo, C. O. (2000) A bacterial two-hybrid selection system for studying protein-DNA and protein-protein interactions. Proceedings of the National Academy of Sciences of the United States of America 97 (13), 7382–7387.</LI><LI id="Klug1987">Klug, A. &amp; Rhodes, D. (1987) Zinc fingers: a novel protein fold for nucleic acid recognition. Cold Spring Harbor symposia on quantitative biology 52, 473–482.</LI><LI id="Livermore1995">Livermore, D. M. (1995) beta-Lactamases in laboratory and clinical resistance. Clinical microbiology reviews 8 (4), 557–584.</LI><LI id="Maeder2008">Maeder, M. L., Thibodeau-Beganny, S. &amp; Osiak, A. et al. (2008) Rapid &quot;open-source&quot; engineering of customized zinc-finger nucleases for highly efficient gene modification. Molecular cell 31 (2), 294–301.</LI><LI id="Mathew2006">Mathew, R. &amp; Chatterji, D. (2006) The evolving story of the omega subunit of bacterial RNA polymerase. Trends in microbiology 14 (10), 450–455.</LI><LI id="Pavletich1991">Pavletich, N. P. &amp; Pabo, C. O. (1991) Zinc finger-DNA recognition: crystal structure of a Zif268-DNA complex at 2.1 A. Science (New York, N.Y.) 252 (5007), 809–817.</LI><LI id="Ramirez2008">Ramirez, C. L., Foley, J. E. &amp; Wright, D. A. et al. (2008) Unexpected failure rates for modular assembly of engineered zinc fingers. Nature methods 5 (5), 374–375</LI><LI id="Roberts2008">Roberts, M. C. (1996) Tetracycline resistance determinants: mechanisms of action, regulation of expression, genetic mobility, and distribution. FEMS microbiology reviews 19 (1), 1–24</LI><LI id="Sander2011">Sander, J. D., Dahlborg, E. J. &amp; Goodwin, M. J. et al. (2011) Selection-free zinc-finger-nuclease engineering by context-dependent assembly (CoDA). Nature methods 8 (1), 67–69.</LI><LI id="Wojcik2010">Wojcik, J., Hantschel, O. &amp; Grebien, F. et al. (2010) A potent and highly specific FN3 monobody inhibitor of the Abl SH2 domain. Nature structural &amp; molecular biology 17 (4), 519–527.</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>