Team:Bielefeld-CeBiTec/Model
Toggle navigation
NavBar
PROJECT
Description
Library
Overview
Scaffolds
Design and Construction
Create your own Library
Mutation
Overview
Error prone Polymerase I
Genome wide mutator
Selection
Overview
Bacterial Two-Hybrid System
Alternative Selection System
Modeling
Overview
Model I
Model II
RESULTS
Overview
Library
Overview
Assembly
Phagemid Display
Sequencing
Mutation
Overview
Assembly
Reversion
Sequencing
Selection
Overview
Assembly
Expression Control
Interaction Control
Binding Control
in vivo Selection
Modeling
Fermentation
Parts
Best Basic Part
Best Composite Part
Best Part Collection
Improve a part
Proof of Concept
Demonstrate your Work
Medal Fulfillment
HUMAN PRACTICES
Overview
Engagement
Dictionary
Educational Outreach
Crowdfunding and Press
Entrepreneurship
Integrated Practices
Achievements
COLLABORATIONS
Overview
Düsseldorf
Lethbridge
Freiburg
NOTEBOOK
Overview
Lab Notebook
Library
Mutation
Selection
Primers
Lab Protocols
Media and Buffers
TEAM
Team Profile
Members
Bielefeld University
Contact
ACKNOWLEDGEMENT
Overview
Sponsors and Partners
Attributions
Modeling
Overview
As biological research studies are more and more based on both, laboratory research and mathematical computations, we decided to follow this encouraging development and, thus, build a mathematical model of our system. This abstract representation of a biological process has the purpose to understand the underlying functionality in the first place. Moreover, a variety of distinct conditions can easily be tested and arbitrarily often repeated. Once the feedback loop between model and experiment has led to a satisfying validation, the model can be applied to predict specific biological events of interest. New knowledge can be achieved very quickly.
Goal
With our mutagenesis system, we want to find the best adapted E. coli culture, and we want to find this culture as quickly as possible. Therefore we long for a tool that can predict us, if the desired culture can be found at all, and, if yes, which culture will be the winner. Additionally the prediction should provide us optimal parameters for the laboratory, the best cultivation time and inoculation number.
These considerations lead to a mathematical model that has to comprise different levels - a micro layer and a macro layer. On the one hand, the model has to deal with the experimental parameters affinity, expression strength, ampicillin concentration, and portray the corresponding processes in an E. coli cell. On the other hand, the model has to transfer this gained knowledge to the growth behavior of different E. coli strains and predict what happens, if different cultures have to contest for space and nutrients.
The Model
To reach this goal we built two models, model I and model II .  Both models are based on chemical and biological mechanisms. Model I represents our bacterial two-hybrid selection system and model II the [cultivation step]. After careful validation we could finally apply this model conglomeration to predict how long a cultivation must last and how often an inoculation must be done till the desired proportion of the best adapted E. coli culture is reached.
The models can be used separately. In the following the workflow of our connected models is shown:
Figure 1: Workflow of the modeling process.
The interaction of model I and model II are shown. From left to right: The model I produces a reproduction rate r - with every run. After several runs the different reproduction rates are transferred to model II, which translates these rates into growth curves.
First, the selection system was implemented (model I). The basic structure is based on iGEM Potsdam 's modeling work of 2011. The model I represents the bacterial two-hybrid selection system and produces the reproduction rate r of a single cell with a given affinity under selection pressure with ampicillin (amp). The reproduction rate is computed for many different affinities and then transmitted to the competition model (model II). The competition model depicts the growth behavior of different adapted E. colis in one flask under multiple inoculations.
Model I
Model II
Results
Verhulst.m
selectionSystem.m
Verhulst_competetive.m
double_time.m
find_best.m
solve_program.m
