"
Page
Discussion
View source
History
teams
Log in
 
Team:UGA-Georgia/RBS
From 2014.igem.org
HOME
PROJECT
Overview Geraniol Modeling RBS Library Parts
WET LAB
Safety Notebook Protocols
HUMAN PRACTICES
Outreach Seminars
TEAM
Official Team Profile Members Attributions
Ribosome Binding Site Library
The ribosome binding sites of archaea are not well characterized. Creating and characterizing a RBS library will be the first of its kind, allowing researchers the ability to express proteins of interest at variable levels of expression in Methanococcus.
Characterization of regulatory sequences – mCherry as a quantitative fluorescent reporter
To characterize the variability of expression per RBS, we must use some sort of quantitative approach. Use of the red fluorescent protein, mCherry, along with the quantitative measurements of a plate reader were our choices for characterizing our library of sequences. It’s important that we picked a red fluorescent protein because methanogens are naturally auto-fluorescent in the blue-green range, due to Coenzyme F420.
Last year, we created a first-draft vector ( BBa_K1138002 ) that contained mCherry in the pAW50 vector. The largest issue with this construct was that the mCherry gene contained an internal Pst1 site, therefore making it BioBrick incompatible. Fluorescence readings from this construct were notably unreproducible, which will be elaborated on in the novel protocol for fluorescence reading section below. To improve upon this construct, we fixed the internal Pst1 site of mCherry to establish BioBrick compatibility and picked up a new constitutively expressing vector highly optimized for use in M. maripaludis, pMEV4.
The pMEV4-mCherry vector (fig. 1) shown below is the construct we’ve designed to create and characterize our library of RBS’s. The primary differences between this vector and the pAW50-mCherry vector of 2013 is the internal restriction site has been removed from the mCherry gene, and the Puromycin resistance gene has an independent promoter. These fixes are particularly useful as this vector is now BioBrick compatible, and more reliably able to function under selective pressure.
Fig. 1 This is the pMEV4-mCherry vector which contains two constitutive promoters, one immediately upstream of the RBS and mCherry gene and the other upstream of the regions of puromycin and ampicillin resistance.
The region labeled ‘RBS 1-39’ is the 12 base-pair region including the first base of the start codon and the 11 bases immediately prior which will be subject to mutation. Primers were designed for a mutation on each base along a 12 base-pair mutation region (fig 2). This region includes the RBS (1-18), spacer (19-33), and first base of the start codon (34-36). Additionally, two primers were designed (37 & 38) as theoretical ‘perfect’ and ‘negative’ RBS sequences. The theoretical ‘perfect’ and ‘negative’ RBS sequences (hereafter referred to as 37 & 38, BBa_K1383001 & BBa_K138002 , respectively) were derived based off of 16S rRNA data found on “Methanoccocus maripaludis Strain S2, Complete Sequence" by Hendrickson. We chose to include these sequences in our library as effective positive and negative controls of RBS binding. The sequence in fig. 2 labeled ‘Native RBS’ (hereafter referred to as Native, BBa_K1383000 ) is a known functional RBS in methanogens that primarily, and usually solely, is used for creating synthetic parts. To characterize every sequence of the library, our approach is to first create three pools of libraries; 1) All RBS region mutation sequences, 2) All spacer region mutation sequences, and 3) All start codon mutation sequences. Anaerobic transformation in M. maripaludis is both time-consuming and meticulous, so by pooling libraries we reduce the number of transformations that need to be done. After picking a statistically appropriate number of colonies from the transformants to maximize likelihood of including every sequence, we will allow all of the cultures to grow under optimal growth conditions, then quantify fluorescence of mCherry using a plate reader. Then we will have all of the clones sequenced and correlate the sequence to the relative strength of fluorescence. However, we encountered some issues when it came to quantifying mCherry production from M. maripaludis that hasn’t been addressed in any previous literature which led to the development of the novel mCherry fluorescence quantification protocol for production in M. maripaludis, as described below.
Fig. 2 This is a chart detailing the mutations of the library we wish to compile on each base along a 12 base-pair mutation region. This region includes the RBS (1-18), spacer (19-33), and first base of the start codon (34-36). Additionally, two primers were designed (37 & 38) as theoretical ‘perfect’ and ‘negative’ RBS sequences.
Development of a novel mCherry fluorescence maturation and quantification protocol for M. maripaludis.
Simply using the process described above taking 100ul of supernatant from a culture and placing it into a plate reader for fluorescence reading constantly produced inconsistent, unreproducible results. Looking into the literature, there were no previous studies of fluorescence quantification in a biological system similar to ours. To begin our troubleshooting, we looked into the broth-medium for growth of Methanococcus. An untypical compound used in some mediums of obligate anaerobes is Resazurin. Resazurin is a blue dye that is used as an oxidation-reduction indicator. When resazurin is reduced, for example by O2, it changes to a bright pink color. This compound is usually put in obligate anaerobic mediums so one can visually tell if any oxygen may have accidentally entered a culture, therefore spoiling it. Resazurin, in its reduced pink state, is highly red fluorescent and likely has been causing great inconsistencies and false positives. The solution to this was fortunately an easy fix, since resazurin isn’t necessary for growth, we began making media without resazurin and took extra caution not to introduce any oxygen to cultures. At this point, fluorescent screening was providing the same result for all cultures, wild type and pMEV4-mCherry transformants, all negative. We were not convinced of any issue of the plasmid or its expression in M. maripaludis, so we began looking specifically into mCherry. What many people, including ourselves, do not consider about fluorescent proteins, is that some of them require oxygen exposure for proper maturation of the fluorophore, which is in fact the case for mCherry. This was clearly a conflict of interest considering M. maripaludis is an obligate anaerobe. After much trial and error, we developed an effective and sufficient method for oxygen maturation of mCherry from M. maripaludis cultures. The detailed protocol may be found on the protocols page, but the general idea is that cultures needed to be spun down, suspended in non-lysis buffer, and left shaking overnight in a microcentrifuge tube (fig X). After this process was complete, we were getting our expected, consistent results of fluorescence quantification.
Fig. 3 Photos of our "theoretical best", Native, and "theoretical worst" RBS sequences respectively, by column after oxygen exposure. The top row is from 100mL cultures, the middle row is from 25mL cultures, and the last row is from 5mL cultures.
Results
Given the constraint of time, we were only able to characterize the three control sequences, namely; Native, 37 ‘Positive’, and 38 ‘Negative’. The graph below (fig. 4) illustrates the quantitative values of mCherry of our three control RBS sequences, as well as a hard negative control, wild-type M. maripaludis. All data points were created by taking the mean average of triplicates for every sample.
Fig. 4 This graph details the results of our fluorescence experiment. Showing that 14 had the highest fluorescence, then 15, then the native.
Discussion and Outlook
Our hypothesis was partly supported by the data presented above. The 37 ‘Positive’ was the strongest of our controls as expected, however, the 38 ‘Negative’ had a much higher, consistent fluorescent reading than expected, even greater than that of the Native RBS. This may be due to a misannotation of the 16S rRNA gene in the M. maripaludis genome or errors in synthesis of the RBS sequence. These are only speculations and will require further investigation. Now that we’ve solved the fluorescence reading issue, and characterized our controls, we may now begin characterizing the rest of the RBS library by reviving the transformed libraries from frozen stocks. After characterizing the library and correlating sequences to their relative levels of expression, we may search for patterns of expression in the sequence to determine which, if any, are critical bases for the binding of the RBS. We plan to submit a range of  characterized variable expression sequences to the iGEM parts registry when they have been completed.
Reference
Hendrickson, E.L., Whitman, W.B., et al. "Methanococcus Maripaludis Strain S2, Complete Sequence." National Center for Biotechnology Information. U.S. National Library of Medicine, 2004. Web. 17 Oct. 2014. http://www.ncbi.nlm.nih.gov/nuccore/45050763 .
Retrieved from " http://2014.igem.org/Team:UGA-Georgia/RBS "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
