"
Page
Discussion
View source
History
teams
Log in
 
Team:UGA-Georgia/Geraniol
From 2014.igem.org
HOME
PROJECT
Overview Geraniol Modeling RBS Library Parts
WET LAB
Safety Notebook Protocols
HUMAN PRACTICES
Outreach Seminars
TEAM
Official Team Profile Members Attributions
Geraniol
What is Geraniol?
Geraniol is a 10-carbon monoterpene-alcohol (fig.1) naturally occurring in plants such as rose and lemongrass. Due to its rose-like scent, it has a long history of usage approved by FDA in the fragrance and food industry. In recent years, potential usages of geraniol start expanding to biofuel, organic insect repellent and tumor suppressor. Currently, geraniol is commercially available through extraction mainly from plants. However, it is an inefficient model for geraniol production, as geraniol represents only a small portion of the plant biomass.
Fig. 1 The chemical structure of geraniol consisting of two five carbon isoprenoid units and a terminal hydroxy group. This structure has been designed to resemble a rose and with its "stem" since geraniol is associated with the scent of roses.
Why Geraniol?
When UGA-iGEM begun, we had the opportunity to work with the methanogenic arhcaeon, Methanococcus maripaludis, and after reading through the literature we were able to develop this project where we may open doors towards greater sustainability in biofuel production at drastically reduced costs. We chose geraniol in particular because of the two central points when designing this project: 1) Geraniol has a growing number of applications like those mentioned above, and 2) Methanococcus maripaludis naturally synthesizes the immediately prior intermediate, geranyl-pyrophoshphate, which is catalyzed into geraniol and iPP by the enzyme geraniol synthase (figure 2).
Fig. 2 This shows the chemical structures of the immediately prior intermediates in both the isoprenoid membrane lipid pathway and geraniol.
How do we employ synthetic biology to produce Geraniol in Methanococcus maripaludis?
There are several homologues of geraniol synthase (GS) found in different plants. After reading through the literature, we chose the GS gene from Ocimum basilicum, otherwise known as sweet basil, based off of its enzyme kinetics and rate of turnover. This gene was codon optimized based on the codon usage for M. maripaludis. Codon optimization is the process of selecting an organism’s most commonly used codon(s) for particular amino acid residues. This gene was synthesized where it may be cloned into the pAW42 vector. pAW42 is a constitutively expressing vector designed for function in both E. coli and M. maripaludis (figure 3). The GS gene, however, is optimized for expression in M. maripaludis. We use E. coli for cloning work and for rapid amplification of our plasmid, this is simply because these basic processes are significantly more time-consuming in anaerobic organisms.
Fig.3 This is the pAW42-GS vector that contains two constitutive promoters, one immediately upstream of the RBS and geraniol synthase gene and the other upstream of the regions of puromycin and ampicillin resistance.
After successful cloning of the GS gene into the pAW42 vector in E. coli, plasmids were extracted , and measured for density by fluorometer. These plasmids were then transformed into wild-type M. maripaludis cells by mediation of polyethylene glycol. Colonies of transformants were grown on puromycin-containing anaerobic bottle plates for selection, and cultures were grown from the picked colonies under continued selective pressure.
Cultures were grown to an OD600 of 0.8 before they were subject to centrifugation. Supernatant was separated from the cell pellet, so that we may do extraction (by separator funnel) procedure of both intracellular and extracellular content. Extraction of geraniol was done by using hexanes as an organic solvent in a separatory funnel. The extractions were concentrated via evaportation under a stream of N2 gas, and evaluated for geraniol content using Gas Chromatography/Mass Spectrometry (GC/MS).
Fig. 4 This is an image of balch tubes of GS before the batch tube extraction protocol has been performed.
The two chromatographs shown in figure 4 are the results of running a geraniol standard and an extracted M. maripaludis sample. Mass Spec confirmed the peak in both of these instances to be the presence of geraniol ((trans)-3,7-Dimethyl-2,6-octadien-1-ol). A wild type M. maripaludis culture was also extracted using hexanes and run through GC/MS and no peak was present. We were able to calculate the production of geraniol in our model as 0.2% of lipid dry weight by plotting the concentration of various standards against the integral area of the peak created and obtaining a linear relationship.
Fig. 5 These chromatographs indicate a peak that was caused by the presence of geraniol.
Now, that leads us to this year’s project, The New Archaea-Type. Our focus began with the question, “How can we produce more geraniol?” Our first approach was addressing the extraction method, we experimented with different solvents including DCM, butanol, diethyl ether, hexanes, and 50% hexanes 50% acetone solution, and concluded that DCM was both the most efficient at recovering geraniol from Methanococcus-formate medium, and was the most volatile for quicker concentration times. Next, we delved into the extraction (by balch tube) protocol, and developed a new one [protocol (balch tube extraction)] which involves less transfer of material between containers, and a more reliable method of mixing phases for the consistent extraction of geraniol.
Since geraniol shares its intermediate with the lipid biosynthesis pathway in Methanococcus, we decided to pick up a new project – Constructing a genome-scale model of M. maripaludis and completing flux balance analysis to optimize flux to particular biomass . Ideally, a complete flux balance model will allow us to optimize for production of geraniol.
Retrieved from " http://2014.igem.org/Team:UGA-Georgia/Geraniol "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
