<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CGU_Taiwan_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CGU Taiwan/Experiments</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Home | CGU_Taiwan</TITLE></P><DIV class="navbar navbar-inverse" role="banner"><DIV class="container"><DIV class="collapse navbar-collapse"><UL class="nav navbar-nav navbar-right"><LI class="header-list"><A href="https://2015.igem.org/Team:CGU_Taiwan/Description"><FONT size="4">Project</FONT></A></LI><LI class="dropdown header-list"><A href="#" class="list-arrow"><FONT size="4">Experiment </FONT></A></LI><LI><A href="https://2015.igem.org/Team:CGU_Taiwan/Experiments">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:CGU_Taiwan/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:CGU_Taiwan/Results">Result</A></LI><LI><A href="https://2015.igem.org/Team:CGU_Taiwan/Safety">Safety</A></LI></UL><LI class="dropdown header-list"><A href="#" class="list-arrow"><FONT size="4">Parts </FONT></A></LI><LI><A href="https://2015.igem.org/Team:CGU_Taiwan/Parts">Team Parts</A></LI><LI><A href="https://2015.igem.org/Team:CGU_Taiwan/Basic_Part">Basic Parts</A></LI><LI><A href="https://2015.igem.org/Team:CGU_Taiwan/Composite_Part">Composite Parts</A></LI><LI><A href="https://2015.igem.org/Team:CGU_Taiwan/Part_Collection">Part Collection</A></LI><LI class="header-list"><A href="https://2015.igem.org/Team:CGU_Taiwan/Practices"><FONT size="4">Human Practice</FONT></A></LI><LI class="header-list"><A href="https://2015.igem.org/Team:CGU_Taiwan/Collaborations"><FONT size="4">Collaboration</FONT></A></LI><LI class="header-list"><A href="https://2015.igem.org/Team:CGU_Taiwan/Team"><FONT size="4">About Us </FONT></A></LI><LI class="header-list"><A href="https://2015.igem.org/Team:CGU_Taiwan/Attributions"><FONT size="4">Attribution</FONT></A></LI><LI class="dropdown header-list"><A href="#" class="list-arrow"><FONT size="4">Notebook</FONT></A></LI><LI><A href="https://2015.igem.org/Team:CGU_Taiwan/Timeline">Overview &amp; Timeline</A></LI><LI><A href="https://2015.igem.org/Team:CGU_Taiwan/Notebook">Lab Notes</A></LI></DIV></DIV></DIV><P><TITLE>Home | CGU_Taiwan</TITLE></P><DIV class="container"><DIV class="single-blog blog-details two-column " id="GPCR"><H1 class="post-title bold">Protocols</H1><H2>Yeast With IL-8 Receptor</H2></DIV><DIV id="accordion-container"><DIV class="panel-group" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse1">Agar plate and medium</A></H4></DIV><DIV id="collapse1" class="panel-collapse collapse "><DIV class="panel-body">
																1.	YPD agar plate  Total volume: 400ml  Number of plates: 20 
																<TABLE class="protocol-table"><TBODY><TR><TD>Bacto-yeast extract</TD><TD>4g</TD></TR><TR><TD>Bacto-peptone</TD><TD>8g</TD></TR><TR><TD>Dextrose</TD><TD>8g</TD></TR><TR><TD>Bacto-agar</TD><TD>8g</TD></TR><TR><TD>+ddH<FONT style=" vertical-align:sub;font-size:0.5em;">2</FONT>O to</TD><TD>400ml</TD></TR></TBODY></TABLE>
																2.	YPD medium (lipid)  Total volume：200ml
																<TABLE class="protocol-table"><TBODY><TR><TD>Bacto-yeast extract</TD><TD>2g</TD></TR><TR><TD>Bacto-peptone</TD><TD>4g</TD></TR><TR><TD>Dextrose</TD><TD>4g</TD></TR><TR><TD>+ddH<FONT style=" vertical-align:sub;font-size:0.5em;">2</FONT>O to</TD><TD>200ml</TD></TR></TBODY></TABLE>
																3.	synthetic complete media(SCM) plate
																<TABLE class="protocol-table"><TBODY><TR><TD>0.2%Adenine</TD><TD>10ml</TD></TR><TR><TD>Bacto-yeast nitrogen base (without a.a)</TD><TD>6.7g</TD></TR><TR><TD>Dextrose</TD><TD>20g</TD></TR><TR><TD>Amino acid mixture</TD><TD>Depend on your selection</TD></TR><TR><TD>agar</TD><TD>20g</TD></TR><TR><TD>Total volume</TD><TD>1L</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse2">Plamid miniprep</A></H4></DIV><DIV id="collapse2" class="panel-collapse collapse"><DIV class="panel-body">
																1.	Transfer for 1.5ml of bacterial culture to a microcentrifuge tube.
																2.	Centrifuge for I min at 13,000 rpm , room temperature in a microcentrifuge and discard supernatant.
																3.	Add 200μl of PD1 Buffer and resuspend the cell pellet by vortexing.
																4.	Add 200μl PD2 Buffer and mix gently by inverting the tube 10 times. Do not vortex and pipetting.
																5.	Allow mixture to stand for 2min at room temperature until lysate clears.
																6.	Add 300μl of PD3 Buffer and mix immediately by inverting the tube 10 times. Do not vortex.
																7.	Centrifuge for 2min at 13,000rpm , RT.
																8.	Place a PD Column in a 2ml Collection Tube.
																9.	Apply the clear lysate from step 7 to the PD Column.
																10.	Centrifuge at 13,000rpm for 30s.
																11.	Discard the flow-through and return the PD Column to the 2ml Collection Tube.
																12.	Add 400μl of W1 Buffer in the PD Column.
																13.	Centrifuge at 13,000rpm ,RT for 30s.
																14. Discard the flow-through and return the PD Column to the 2ml Collection Tube.
																15.	Add 600μl of Wash Buffer (Ethanol added) to PD Column.
																16.	Centrifuge at 13,000rpm , RT for 30s.
																17.	Discard the flow-through and return the PD Column to the 2ml Collection Tube.
																18.	Centrifuge again for 3min at 13,000rpm , RT to dry the column matrix.
																19.	Transfer the dried PD Column to a clean 1,5ml microcentrifuge tube.
																20.	Add 50μl ddH<FONT style=" vertical-align:sub;font-size:0.5em;">2</FONT>O directly onto the centre of the membrane. 
																21.	Allow to stand for 2min until the liquid is absorbed.
																22.	Centrifuge for 2min at 13,000rpm , RT to elute purified DNA.
																23.	Measure the concentration of plasmid.</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse3">PCR</A></H4></DIV><DIV id="collapse3" class="panel-collapse collapse"><DIV class="panel-body">
																1.	PCR program
																<TABLE class="protocol-table"><TBODY><TR><TD>Step</TD><TD>Temperature</TD><TD>Time</TD></TR><TR><TD>Step1</TD><TD>95℃</TD><TD>5min </TD></TR><TR><TD>Step2</TD><TD>95℃</TD><TD>30s </TD></TR><TR><TD>Step3</TD><TD>Primer annealing Tm</TD><TD>30s </TD></TR><TR><TD>Step4→Step2 for n cycle</TD><TD>72℃</TD><TD>Depend on product length </TD></TR><TR><TD>Step5</TD><TD>72℃</TD><TD>5min</TD></TR><TR><TD>Step6(Hold on)</TD><TD>Depend on PCR machine</TD><TD>1hr</TD></TR></TBODY></TABLE>
																2.	PCR reagent 
																<TABLE class="protocol-table"><TBODY><TR><TD>10x Dream Taq buffer</TD><TD>2.5μl</TD></TR><TR><TD>2.5mM dNTP</TD><TD>0.5μl</TD></TR><TR><TD>10μM primer(F)</TD><TD>0.5μl</TD></TR><TR><TD>10μM primer(R)</TD><TD>0.5μl</TD></TR><TR><TD>Template(150ng)</TD><TD>Depend on concentration</TD></TR><TR><TD>Taq polymerase</TD><TD>0.5μl</TD></TR><TR><TD>ddH<FONT style=" vertical-align:sub;font-size:0.5em;">2</FONT>O</TD><TD>it depends</TD></TR><TR><TD>Total volume</TD><TD>25μl</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse4">Clean up after digestion</A></H4></DIV><DIV id="collapse4" class="panel-collapse collapse"><DIV class="panel-body">
																1.	Transfer up to 100μl reaction product to a microcentrifuge tube.
																2.	Add 5 volumes of PG Buffer to 1 volume of the sample and mix by vortexing. 
																3.	Place a Spin Column in a Collection Tube. Apply the sample mixture from previous step into the Spin Column. 
																4.	Centrifuge at 10000 x g (13,000 rpm) for 30 seconds. Discard the flow-through and place the Spin Column back in the Collection Tube.
																5.	Add 600μl of Wash Buffer (ethanol added) in the Spin Column. Centrifuge at 10000 x g (13,000 rpm) for 30 seconds. 
																6.	Discard the flow-through and place the Spin Column back into the Collection Tube. 
																7.	Centrifuge again for 2 minutes at full speed (14.000 rpm) to dry the column matrix. 
																8.	Transfer dried column in a new microcentrifuge tube (not provided).Add 25μl of ddH<FONT style=" vertical-align:sub;font-size:0.5em;">2</FONT>O into the center of the column matrix. 
																9.	Stand for 2 minutes until ddH<FONT style=" vertical-align:sub;font-size:0.5em;">2</FONT>O absorbed by the matrix. 
																10.	Centrifuge for 2 minutes at full speed to elute purified DNA.
																11.	Measure the concentration.</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse5">Gel extraction</A></H4></DIV><DIV id="collapse5" class="panel-collapse collapse"><DIV class="panel-body">
																1.	Excise the agarose gel slice containing DNA fragments and removing extra agarose to minimize the size of the gel slice.
																2.	Transfer up to 300 mg of the gel slice into a microcentrifuge tube (not provided). Add 500μl of PG Buffer to the sample and mix by vortexing.
																3.	Incubate at 55℃ for 15 minutes until the gel slice has been completely dissolved. During incubation, invert the tube every 2-3 min. 
																4.	Place a Spin Column in a Collection Tube. Apply 800μl of the sample mixture from previous step into the Spin Column.
																5.	Centrifuge at 10000 x g (13,000 rpm) for 30 seconds. Discard the flow-through and place the Spin Column back in the Collection Tube. Note: If the sample mixture is more than 800μl, repeat this DNA Binding Step. 
																6.	Add 500μl Wash Buffer (ethanol added) to the spin Column.
																7.	Centrifuge at max. 10,000 xg (13,000 rpm) for 30 seconds.
																8.	Discard the flow-through and place the spin Column back into the Collection Tube.
																9.	Centrifuge again for 2 minutes at full speed (14,000 rpm) to dry the column matrix. 
																10.	Transfer dried column in a new microcentrifuge tube. Add 25μl of ddH<FONT style=" vertical-align:sub;font-size:0.5em;">2</FONT>O into the center of the column matrix. 
																11.	Stand for 2 minutes until ddH<FONT style=" vertical-align:sub;font-size:0.5em;">2</FONT>O absorbed by the matrix. 
																12.	Centrifuge for 2 minutes at full speed to elute purified DNA.
																13.	Measure the concentration.</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse6">Extraction of DNA with  phenol chloroform</A></H4></DIV><DIV id="collapse6" class="panel-collapse collapse"><DIV class="panel-body">
																1.	Add an equal volume (300ul) of phenol : chloroform to the nucleic acid sample in a 1.5ml eppendorf.
																2.	Mix the contents of the tube vigorously.
																3.	Centrifuge the mixture at max speed for 5 min at R.T.
																4.	Transfer the aqueous phase to a fresh tube.
																5.	Recover the nucleic acid by ethanol precipitation.
																6.	Add 1/10 volume of sodium acetate(pH5.2) and mix well.
																7.	Add 2.5 times volume of ice-cold 100% ethanol and mix well.
																8.	Place at -80℃ for 30min to allow the precipitation to form.
																9.	Centrifuge at max speed for 15min at 4℃.
																10.	Carefully decant the supernatant.
																11.	Wash the pellet with 1ml 70% ethanol.
																12.	Centrifuge at max speed for 10min at 4℃.
																13.	Carefully decant supernatant.
																14.	Air dry the pellet at 45℃ for 10min.
																15.	Resuspend pellet in 25ul ddH<FONT style=" vertical-align:sub;font-size:0.5em;">2</FONT>O.</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse7">Yeast transformation</A></H4></DIV><DIV id="collapse7" class="panel-collapse collapse"><DIV class="panel-body">
																1.	The day before transformation , grow yeast in 5ml of YPD with 0.002% Adenine at 30℃,overnight.
																2.	After 12hr , measure the optical density under wavelength 600nm.
																3.	Dilute to OD<FONT style=" vertical-align:sub;font-size:0.5em;">600</FONT>=0.05 in 20ml of YPD+A medium.
																4.	Measure the optical density under wavelength 600nm.
																5.	Prepare yeast competent cells   (1)	Collect all of the log phase yeast culture in 15ml tube   (2)	Centrifuge at 1000g,RT,10min   (3)	Discard the supernatant   (4)	Add 2ml of 0.1M LiAc to resuspend the cell pellet (gently pipette)   (5)	Centrifuge at 1000g,RT,10min   (6)	Discard the supernatant   (7)	Add 800μl of 0.1M LiAc to resuspend the cell pellet (gently pipette)
																
																6.	Heat Salmon sperm DNA (ssDNA) at 95℃ for 10 min and then incubate on ice immediately.
																7.	Mix following component in a 15ml tube<TABLE class="protocol-table"><TBODY><TR><TD>Sample1</TD><TD>Negative control</TD></TR><TR><TD>40%PEG8000 in 0.1M LiAc </TD><TD>700μl</TD><TD>700μl</TD></TR><TR><TD>LiAc treated yeast </TD><TD>100μl </TD><TD>100μl</TD></TR><TR><TD>ssDNA</TD><TD>5μl</TD><TD>5μl</TD></TR><TR><TD>PCR product</TD><TD>20μl </TD><TD>0μl</TD></TR><TR><TD>ddH<FONT style=" vertical-align:sub;font-size:0.5em;">2</FONT>O</TD><TD>0μl</TD><TD>20μl </TD></TR></TBODY></TABLE>
																8.	Rotate at 30℃ for 2hr
																9.	Add 100μl of DMSO and gently pipeting. 
																10.	Heat shock at 42℃ for 20min and incubate on ice for 2min immediately.
																11.	Add 5ml of YPD and stand at RT for 5min. 
																12.	Centrifuge at 1000g,RT, 10min
																13.	Discard the supernatant.
																14.	Resuspend in 1ml of ddH<FONT style=" vertical-align:sub;font-size:0.5em;">2</FONT>O. 
																15.	Repeat step 12-13 one more time.
																16.	Resuspend in 100μl of ddH<FONT style=" vertical-align:sub;font-size:0.5em;">2</FONT>O.
																17.	Take 100ul to plate onto your selection plate.
																18.	Incubate 3 plates at 30℃ for 3 days.</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse8">Enzyme digestion</A></H4></DIV><DIV id="collapse8" class="panel-collapse collapse"><DIV class="panel-body"><TABLE class="protocol-table"><TBODY><TR><TD>Insert </TD><TD>400ng</TD></TR><TR><TD>10x NEB buffer #4 </TD><TD>5μl</TD></TR><TR><TD>Restriction enzyme </TD><TD>1μl</TD></TR><TR><TD>ddH<FONT style=" vertical-align:sub;font-size:0.5em;">2</FONT>O </TD><TD>It depends</TD></TR><TR><TD>Total volume </TD><TD>50μl</TD></TR></TBODY></TABLE><TABLE class="protocol-table"><TBODY><TR><TD>Vector </TD><TD>2μg</TD></TR><TR><TD>10x NEB buffer #4 </TD><TD>5μl</TD></TR><TR><TD>Restriction enzyme </TD><TD>1μl</TD></TR><TR><TD>ddH<FONT style=" vertical-align:sub;font-size:0.5em;">2</FONT>O </TD><TD>It depends</TD></TR><TR><TD>Total volume  </TD><TD>50μl</TD></TR></TBODY></TABLE>
																Condition : 37℃ for 1hr
															</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse9">Ligation and transformation of <I>E.coli</I></A></H4></DIV><DIV id="collapse9" class="panel-collapse collapse"><DIV class="panel-body">
															Ligation of E.coli
															<TABLE class="protocol-table"><TBODY><TR><TD>&amp;nbsp </TD><TD>Vector:insert=1:3 </TD><TD>Vector only</TD></TR><TR><TD>Vector</TD><TD>50-100ng </TD><TD>50-100ng</TD></TR><TR><TD>Insert</TD><TD>Depend on concentration </TD><TD>0μl</TD></TR><TR><TD>Ligase </TD><TD>1μl </TD><TD>1μl</TD></TR><TR><TD>10x ligation buffer </TD><TD>2μl</TD><TD>2μl</TD></TR><TR><TD>ddH<FONT style=" vertical-align:sub;font-size:0.5em;">2</FONT>O </TD><TD>It depends </TD><TD>It depends</TD></TR><TR><TD>Total volume </TD><TD>20μl</TD><TD>20μl</TD></TR></TBODY></TABLE>
															Condition: 16-24℃ for 2-4hr
															Transformation of E.coli
															1.	Thaw 2/3 of the competent cells and add the plasmid into the competent cells.
															2.	Vortex the competent cells for 1s.
															3.	Incubate the competent cells on ice for 10 min.
															4.	Transfer the 100 μl competent cells on your selection plates and incubate at 37℃ overnight.</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse10">Fast extraction of gDNA of yeast</A></H4></DIV><DIV id="collapse10" class="panel-collapse collapse"><DIV class="panel-body">
																1.	Add 100μl of 200mM LiAc with 1% SDS in a 1.5ml eppendorf.
																2.	Use pipette tip to transfer yeast colony into the eppendorf and mix by vortex.
																3.	Incubate at 70℃ for 15min.
																4.	Add 300μl of 96% EtOH and vortex 4-5 times.
																5.	Centrifuge at 15,000g for 3min.
																6.	Discard the supernatant.
																7.	Dry the pellet at 45℃ about 0.5-1hr until it becomes translucent.
																8.	Resuspend the pellet with 12μl of ddH<FONT style=" vertical-align:sub;font-size:0.5em;">2</FONT>O.							9.	Centrifuge at 15,000g for 1min.
																10.	Collect the supernatant for PCR reaction.
																11.	Measure the concentration of gDNA.</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse11">Transformation of plasmid into yeast</A></H4></DIV><DIV id="collapse11" class="panel-collapse collapse"><DIV class="panel-body">
																1.	The day before transformation , grow yeast in 5ml of YPD with 0.002% Adenine at 30℃, overnight.
																2.	The next day , prepare yeast competent cells   (1)	Collect 1.5ml of yeast culture in eppendorf.(x2 tubes)   (2)	Centrifuge at 1000g,RT,10min.   (3)	Discard the supernatant.   (4)	Add 200μl of 0.1M LiAc to resuspend the cell pellet. (vortex)   (5)	Centrifuge at 1000g,RT,10min.   (6)	Discard the supernatant.   (7)	Add 200μl of 0.1M LiAc to resuspend the cell pellet. (vortex)

																3.	Heat Salmon sperm DNA (ssDNA) at 95℃ for 10 min and then incubate on ice immediately.
																4.	Mix following component in a 15ml tube <TABLE class="protocol-table"><TBODY><TR><TD> </TD><TD>Sample1</TD><TD>Negative control</TD></TR><TR><TD>40%PEG8000 in 0.1M LiAc </TD><TD>700μl</TD><TD>700μl</TD></TR><TR><TD>LiAc treated yeast</TD><TD>100μl</TD><TD>100μl</TD></TR><TR><TD>ssDNA</TD><TD>5μl</TD><TD>5μl</TD></TR><TR><TD>Plasmid</TD><TD>1μl </TD><TD>0μl</TD></TR><TR><TD>ddH<FONT style=" vertical-align:sub;font-size:0.5em;">2</FONT>O</TD><TD>0μl</TD><TD>1μl</TD></TR></TBODY></TABLE>
																5.	Rotate at 30℃ for 2hr.
																6.	Add 100μl of DMSO and gently pipeting.
																7.	Heat shock at 42℃ for 20min.
																8.	Add 5ml of YPD and stand at RT for 5min. 
																9.	Centrifuge at 1000g,RT, 10min.
																10.	Discard the supernatant.
																11.	Resuspend in 1ml of ddH<FONT style=" vertical-align:sub;font-size:0.5em;">2</FONT>O. 
																12.	Repeat step 9-10 one more time.
																13.	Resuspend in 1ml of ddH<FONT style=" vertical-align:sub;font-size:0.5em;">2</FONT>O.
																14.	Take 50-100μl to plate onto your selection plate.
																15.	Incubate selection plates at 30℃ for 3 days.</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse12">Protein extraction of yeast</A></H4></DIV><DIV id="collapse12" class="panel-collapse collapse"><DIV class="panel-body">
																1.	Resuspend cells in 100μl lysis buffer and add 900μl ddH<FONT style=" vertical-align:sub;font-size:0.5em;">2</FONT>O.
																2.	Incubate sample for 5 min on ice and put acetone into -20℃ at the same time.
																3.	Add 60μl 100% TCA which is a 6% final solution.
																4.	Vortex and incubate on -20℃ for 20min.
																5.	Centrifuge at 4℃ for 10min at 13000rpm.
																6.	Discard the supernatant.
																7.	Wash the pellet with 1ml (-20℃) acetone to remove acid and salt.(pipetting gently until pellet float up)
																8.	Centrifuge at 4℃ for 10min at 13000rpm.
																9.	Discard the supernatant and air dry for 2hr.
																10.	Resuspend the pellet in 60μl  2xSDS-PAGE protein sample buffer with 5% 2-mercaptoethanol.
																11.	Add 16μl tris-base to make every sample turn blue.
																12.	Boil sample for 10min, at 95℃.
																13.	Centrifuge at RT for 1min at 13000rpm.
																14.	Store samples at -20℃.</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse13">Induction with α factor</A></H4></DIV><DIV id="collapse13" class="panel-collapse collapse"><DIV class="panel-body">
																1.	Day1, grow yeast in 5ml YPD(+A) overnight.
																2.	Day2, measure the the OD<FONT style=" vertical-align:sub;font-size:0.5em;">600</FONT> and dilute to OD<FONT style=" vertical-align:sub;font-size:0.5em;">600</FONT>= 0.05 in 5ml of YPD+A
																3.	Grow to OD<FONT style=" vertical-align:sub;font-size:0.5em;">600</FONT>= 0.3-0.5
																4.	Add 100μg/ml ofαfactor and grow for 2hr in 30℃
																5.	Observe under microscope.</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse14">Induction of yeast with galactose</A></H4></DIV><DIV id="collapse14" class="panel-collapse collapse"><DIV class="panel-body">
																1.	One day before induction , incubate yeast in 5ml YPD+A overnight.
																2.	The next day measure OD<FONT style=" vertical-align:sub;font-size:0.5em;">600</FONT>.
																3.	Dilute to OD<FONT style=" vertical-align:sub;font-size:0.5em;">600</FONT>=0.05 and subculture until OD<FONT style=" vertical-align:sub;font-size:0.5em;">600</FONT> between 0.3 to 0.4
																4.	Centrifuge at RT, 1000g for 10min
																5.	Discard the supernatant and wash with 10ml YEP(+Raf).
																6.	Repeat step4-6 for 5 times.
																7.	Resuspend in 10ml YEP(+Raf)
																8.	Dilution:1/20,1/50,1/100 and incubate at 30℃ overnight.
																9.	The third day , measure OD<FONT style=" vertical-align:sub;font-size:0.5em;">600</FONT> of culture and choose one tube whose OD<FONT style=" vertical-align:sub;font-size:0.5em;">600</FONT>=0.3-0.5, and dilute to OD<FONT style=" vertical-align:sub;font-size:0.5em;">600</FONT>=0.1
																10.	Centrifuge at RT,1000g for 10min.
																11.	Resuspend the pellet with YEP(Raf/Gal)
																12.	Collect the culture at 3hr,4hr,5hr,6hr,7hr,8hr.</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse15">SDS-PAGE and Western blot</A></H4></DIV><DIV id="collapse15" class="panel-collapse collapse"><DIV class="panel-body">
																1.	Preparation of 10% SDS Running gel<TABLE class="protocol-table"><TBODY><TR><TD>40% acryl-bis acryl</TD><TD>2.5ml</TD></TR><TR><TD>1.5M Tris pH 8.8</TD><TD>2.5ml</TD></TR><TR><TD>ddH<FONT style=" vertical-align:sub;font-size:0.5em;">2</FONT>O</TD><TD>4.8ml</TD></TR><TR><TD>10% SDS</TD><TD>100μl</TD></TR><TR><TD>10% APS</TD><TD>100μl</TD></TR><TR><TD>TEMED</TD><TD>4μl</TD></TR><TR><TD>Total volume</TD><TD>10ml</TD></TR></TBODY></TABLE>
																2.	Preparation of 5% SDS stacking gel
																<TABLE class="protocol-table"><TBODY><TR><TD>40% acryl-bis acryl</TD><TD>0.625ml</TD></TR><TR><TD>1.5M Tris pH 8.8</TD><TD>0.625ml</TD></TR><TR><TD>ddH<FONT style=" vertical-align:sub;font-size:0.5em;">2</FONT>O</TD><TD>3.65ml</TD></TR><TR><TD>10% SDS</TD><TD>50μl</TD></TR><TR><TD>10% APS</TD><TD>50μl</TD></TR><TR><TD>TEMED</TD><TD>5μl</TD></TR><TR><TD>Total volume</TD><TD>10ml</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse15-1">Immunofluorescence Analysis</A></H4></DIV><DIV id="collapse15-1" class="panel-collapse collapse"><DIV class="panel-body">
																1. Inoculate yeast cells overnight in a 30 °C shaker.
																2. Subculture cells at OD600=0.1 in 5 ml YPD or defined media.
                                                                3. Continue to shake at 30 °C for 4-6 hr to OD600=0.49.
                                                                4. Add 0.6 ml of 37% formaldehyde solution directly to the cells and continue to incubate with shaking for 90 min (to be exact) at 30 °C.
                                                                5. Transfer cells to a 15 ml conical tube and pellet by centrifugation at 2,000 x g for 2 min at 4 °C.                                            6. Aspirate supernatant and wash cells with ice-cold 5 ml phosphate buffer (0.1 M K2HPO4 in H2O, pH 6.5). <FONT style="color:red">Be gentle at this step because 
                                                                yeast cells are very fragile after fixation by formaldehyde.</FONT>
                                                                7. Repellet cells and wash with ice-cold 5 ml sorbitol buffer (1.2 M sorbitol in phosphate buffer).
                                                                8. Pellet cells and aspirate supernatant. Resuspend in 1 ml sorbitol buffer.
                                                                9. Pre-warm the cells at 30 °C for 5 min.
                                                              10. Add 1 μl of zymolyase(1U/μl), mix gently and incubate on 30 °C heat blot for 1 hr.
                                                              11. Check the digestion under a light microscope.
                                                              12. When up to 80% of cells are digested, Pellet cells 2,000 x g for 2 min at 4 °C. Re-suspend cells gently in 0.5 ml ice-cold sorbitol buffer.
                                                              13. Repeat step 12.
                                                              14. Place 20 μl of cell suspension into each well on the slide coated with Poly-L-lysine. Incubate in a wet chamber for 30 min.
                                                              15. Immediately immerse slide in ice-cold methanol for 6-7 min.
                                                              16. Remove and immerse immediately in ice-cold acetone for 30 sec.
                                                              17. Dry the slides on 30 °C heat blot.<FONT style="color:red">From this step on, don’t let the wells dry.</FONT>                                                18. Wash wells several times with blocking buffer (5 % BSA in 1x PBS (pH 8.0) and incubate a moist chamber for 1 hr.
                                                              19. Remove supernatant and add 20 μl of primary antibody (diluted in blocking buffer) in each well. Incubate in a moist chamber for 2 hr.
                                                              20. Aspirate excess solution and wash 5 times with blocking buffer for 1 hr.
                                                              21. Add 20 µl of florescence conjugated-secondary antibody (diluted in blocking buffer) in each well.
                                                              22. Place in a dark/moist chamber for 1 hr. <FONT style="color:red">Keep slides in the dark as much as possible to prevent bleaching of florescence.</FONT>
                                                              23. Aspirate excess solution and wash 5 times with blocking buffer for 1 hr.
                                                              24. Wash once with 1x PBS.
                                                              25. Dilute DAPI (1:1,000) in 1x PBS.
                                                              26. Add DAPI solution to wells for 2 min.
                                                              27. Aspirate excess solution and wash once with 1x PBS.
                                                              28. Do not allow the slide to dry and add 2µl 50% Glycerol to the well and immediately cover with cover slide.
                                                              29. Seal the edge of the slides and examine under a fluorescence microscope or confocal microscope.</DIV></DIV></DIV></DIV></DIV><DIV class="single-blog blog-details two-column post-content" id="RNA"><H2>Toehold Switch As RNA Senor</H2></DIV><DIV id="accordion-container"><DIV class="panel-group" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse16">Digestion and gel extraction</A></H4></DIV><DIV id="collapse16" class="panel-collapse collapse"><DIV class="panel-body">	
																1.	Set up the enzyme digestion as following 
																<TABLE class="protocol-table"><TBODY><TR><TD>plasmid (0.2mg/ml)</TD><TD>5 μl</TD></TR><TR><TD>10X buffer</TD><TD>1 μl</TD></TR><TR><TD>Each of restriction Enzyme (10 units/μl)</TD><TD>2 μl</TD></TR><TR><TD>ddH<FONT style=" vertical-align:sub;font-size:0.5em;">2</FONT>O </TD><TD>2 μl</TD></TR><TR><TD>Final volume</TD><TD>10 μl</TD></TR></TBODY></TABLE>  Incubate the digestions at 37°C for 1 hour. 
																
																2.	Make a 160ml 1.2% agarose gel in 1X TAE with 12 μL DNA View.    (1)	Measure 1.92g agarose powder to a 500ml-bottle, and add 160ml 1X TAE buffer.    (2)	Melt agarose solution in microwave until the powder is dissolved. Be careful not to over-heat it, or else too much water may evaporate and result in gels of the incorrect agarose concentration.   (3)	Let gel solution cool down to 60-70°C (very warm but not hot).   (4)	Add 12 μl of DNA View.   (5)	Pour agarose to the gel holder with the gel comb (25-well tooth) in place. After the agarose gel is solidified, transfer it to the gel running chamber. Fill the box with 1X TAE buffer enough to cover the whole gel. 
																
																4.	Briefly centrifuge the digestion reactions and add 3 μL 6X loading dye to each sample. 
																5.	Run digestion reactions on agarose gels in the following order:    (1)	1kb-DNA ladder, 5μl   (2)	Uncut DNA control, 4 μl    (3)	Digested wild-type sample, 18 μl

																*Uncut DNA control: In microcentrifuge tubes add 5 μl Uncut DNA, 5 μl of ddH<FONT style=" vertical-align:sub;font-size:0.5em;">2</FONT>O, and 2 μl of 6X DNA loading dye. Mix well by gentle pipetting. 
																6.	Visualize DNA bands in agarose gels with a camera - equipped gel documentation system. Take a picture. 
																7.	Bring agarose gel to UV light box and cut out the wanted bands. Place the gel slices into clean microcentrifuge tubes. 
																8.	Dissolve the gel slices with 500 μl buffer for 10 min at 550℃. Invert the tubes 3 times every 3 min. 
																9.	Take gel extraction columns and fit them into empty collection tubes. Transfer the gel/DNA solutions to columns. 
																10.	Centrifuge columns at 13000rpm for 30 sec. Pour off all flow-through and put column back in the original collection tube. 
																11.	Add 500 μl wash buffer (contains ethanol) to column. 
																12.	Centrifuge at 13000rpm for 30sec. 
																13.	Pour off all flow-through and put column back into original collection tube. 
																14.	Centrifuge again for 2 min at 13000rpm to remove all ethanol. 
																15.	Transfer columns to clean microcentrifuge tubes. 
																16.	Add 30μl elution buffer to the center of the column membrane. 
																17.	Centrifuge for 2 min at 13000rpm to elute DNA off the column. 
																18.	Store eluted DNA in -200°C freezer . </DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse17">Ligation</A></H4></DIV><DIV id="collapse17" class="panel-collapse collapse"><DIV class="panel-body">
																1.	Quantitate the concentration of DNA fragments you have using a spectrophotometer. (？ng/μl) 
																2.	Calculate how much insert should be added into the ligation reaction. We will use 10ng of digested vector and the molar ratio of vector: insert should be 1:5 (insert:？μl) 
																3. Set up the ligation reaction as follows: <TABLE class="protocol-table"><TBODY><TR><TD>Plasmid backbone (10ng/μl)</TD><TD>1 μl</TD></TR><TR><TD>Insert DNA	</TD><TD>? μl</TD></TR><TR><TD>10X ligation buffer </TD><TD>2 μl</TD></TR><TR><TD>ATP</TD><TD>1 μl</TD></TR><TR><TD>T4 DNA ligase</TD><TD>1 μl</TD></TR><TR><TD>ddH<FONT style=" vertical-align:sub;font-size:0.5em;">2</FONT>O</TD><TD>? μl</TD></TR><TR><TD>Final volume</TD><TD>20 μl</TD></TR></TBODY></TABLE>																
																Incubate reactions at room temperature for 30 min.</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse18">Mini-prep</A></H4></DIV><DIV id="collapse18" class="panel-collapse collapse"><DIV class="panel-body">
																[The afternoon before]—pick colonies from bacteria plates. Transfer a little bit of bacteria from colony to 1.5 ml LBamp medium, incubate the broth at 37℃ for &gt;16 hours (overnight) with shaking at 225~250 rpm. 
																1.	Make a 1.2% agarose gel. 
																2.	Transfer overnight bacterial cultures to fresh tubes and centrifuge at 13000 rpm for 1 min. 
																3.	Resuspend each bacterial pellet in 200 μl cold buffer PD1 by pipetting. No cell clumps should be visible after resuspension of the pellet. 
																4.	Add 200 μl buffer PD2, mix well by gently inverting tubes 4-6 times—DO NOT SHAKE or VORTEX (vortexing may cause genomic DNA to dissociate from cell membrane pieces and contaminate the plasmid DNA preparation). Stand for 2 min at room temperature until lysate is clear. 
																5.	Add 300 μl buffer PD3, mix immediately by inverting tubes 4-6 times. 
																6.	Centrifuge for 5 min at 13000 rpm at room temperature. 
																7.	Transfer supernatants to fresh PD spin columns and spin for 1 min at 13000 rpm. Discard the flow through and put the spin column back on the collection tubes. 
																8.	Add 200 μl buffer W1 to the columns, centrifuge for 1 min at 13000 rpm. Discard the flow through and put columns back over collection tubes. 
																9.	Add 600 μl wash buffer to the columns, centrifuge for 1 min at 13000 rpm. Discard the flow through and put columns back over collection tubes. 
																10.	Centrifuge columns for 3 more mins at 13000rpm to remove residual liquid. (The wash buffer contains ethanol, and a small amount of ethanol left may inhibit subsequent enzymatic reactions.) 
																11.	Place the columns over new microcentrifuge tubes. Add 30 μl preheated buffer EB, let incubate for 5 min, centrifuge at 13000 rpm for 2min. Collect eluate. 
																12.	Test if the minipreps were successful by running some samples on the gel. For each miniprep sample, mix 5 μl miniprep plasmid with 1 μl 6X loading dye and run on 1.2% agarose gels. </DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse19">Plamid miniprep </A></H4></DIV><DIV id="collapse19" class="panel-collapse collapse"><DIV class="panel-body">			
																Digestion
																1.	Set up the enzyme digestion as following：<TABLE class="protocol-table"><TBODY><TR><TD colspan="2">for PCR product digestion</TD></TR><TR><TD>DNA product</TD><TD>45 μl</TD></TR><TR><TD>10X Buffer</TD><TD>7 μl</TD></TR><TR><TD>10X BSA</TD><TD>7 μl</TD></TR><TR><TD>ddH<FONT style=" vertical-align:sub;font-size:0.5em;">2</FONT>O</TD><TD>9 μl</TD></TR><TR><TD>Restriction enzyme 1</TD><TD>1 μl</TD></TR><TR><TD>Restriction enzyme 2</TD><TD>1 μl</TD></TR><TR><TD>Final volume</TD><TD>70 μl</TD></TR></TBODY></TABLE>
																Incubate at 37℃ in a dry heat-block for 1 hr.
																<TABLE class="protocol-table"><TBODY><TR><TD colspan="2">for plamsid backbone digestion</TD></TR><TR><TD>Plamsid backbone(1 μg/μl)</TD><TD>3 μl</TD></TR><TR><TD>10X Buffer</TD><TD>2 μl</TD></TR><TR><TD>10X BSA</TD><TD>2 μl</TD></TR><TR><TD>ddH<FONT style=" vertical-align:sub;font-size:0.5em;">2</FONT>O</TD><TD>11 μl</TD></TR><TR><TD>Restriction enzyme 1</TD><TD>1 μl</TD></TR><TR><TD>Restriction enzyme 2</TD><TD>1 μl</TD></TR><TR><TD>Final volume</TD><TD>20 μl</TD></TR></TBODY></TABLE>																
																Incubate at 37℃ in a dry heat-block for 1 hr.
																
																
																2. After incubation, place sample into 72℃ dry baths and reacts 10 min to inactive the digest reaction .
																Clean-up：
																1.	Add 5 volumes of PG buffer to 1 volume of the sample and mix by vortexing.
																2.	Place a spin column in a collection tube and apply the sample mixture into spin column.
																3.	Centrifuge at 13000 rpm for 30 seconds. Discard the flow-through and place the spin column back in the collection tube.
																4.	Add 600 μl of wash buffer (ethanol added) in the spin column. Centrifuge at 13000 rpm for 30 seconds.
																5.	Discard the flow-through and place the spin column back into the collection tube.
																6.	Centrifuge again for 2 minutes to dry the column matrix.
																7.	Transfer dried column in a new Eppendorf tube. Add 30 μl of elution buffer into the center of the column matrix. Let it sit for two more minutes.
																8.	Centrifuge for 2 minutes to elute purified DNA.</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse20">PCR reaction and product confirmation</A></H4></DIV><DIV id="collapse20" class="panel-collapse collapse"><DIV class="panel-body">			
																1.	Set up the PCR reaction as following：<TABLE class="protocol-table"><TBODY><TR><TD> </TD><TD>Vol./reaction</TD><TD>Final Conc.</TD></TR><TR><TD>2X Taq Master Mix</TD><TD>25 μl</TD><TD>1X</TD></TR><TR><TD>Forward Primer（1μM）</TD><TD>5 μl</TD><TD>100 nM</TD></TR><TR><TD>Reverse Primer（1μM）</TD><TD>5 μl</TD><TD>100 nM</TD></TR><TR><TD>※Template DNA（0.2ng/μl）</TD><TD>1 μl</TD><TD>4 pg/μl</TD></TR><TR><TD>ddH<FONT style=" vertical-align:sub;font-size:0.5em;">2</FONT>O</TD><TD>14 μl	</TD><TD> </TD></TR><TR><TD>final volume</TD><TD>50 μl</TD><TD> </TD></TR></TBODY></TABLE>
																2.	Label the tubes. Make a mix for all reactions and then transfer the amounts needed for each tube into the tubes. Include an extra reaction for no-DNA control! 
																3.	PCR condition: <TABLE class="protocol-table"><TBODY><TR><TD>Step</TD><TD>Temperature</TD><TD>Time</TD><TD>Function</TD></TR><TR><TD>Pre-incubation(1 cycle)</TD><TD>95℃</TD><TD>5 minutes</TD><TD>Pre-heat, to allow the double-stranded template DNA being denatured completely </TD></TR><TR><TD rowspan="3">Amplification(30 cycles)</TD><TD>95℃</TD><TD>1 minutes</TD><TD>dsDNA denaturation</TD></TR><TR><TD>50℃</TD><TD>1 minutes</TD><TD>Primer annealing</TD></TR><TR><TD>72℃</TD><TD>1 minutes</TD><TD>DNA extension</TD></TR><TR><TD>Post-incubation(1 cycle)</TD><TD>72℃</TD><TD>10 minutes</TD><TD>DNA Extension, to allow all the unfinished DNA having chance to be fully synthesized</TD></TR><TR><TD>Preservation</TD><TD>4℃</TD><TD> </TD><TD>To preserve sample</TD></TR></TBODY></TABLE>
																4.	During incubation of PCR reaction, prepare a 1.2% agarose gel.   (1)	Measure 1.92 g agarose powder to a 500 ml-bottle, and add 160 ml 1x TAE buffer.    (2)	Melt agarose solution in microwave until the powder is dissolved. Be careful not to over-heat it, otherwise, you would evaporate water and make incorrect concentration of agarose gel.   (3)	Put the bottle on the hot plate, stir with stirring-bar until the bottle is cooled down to 60-70 oC.   (4)	Add 7.5 μl DNA view .   (5)	Pour agarose to the gel holder with the gel comb (25-well tooth) in place. After the agarose gel is solidified, transfer it to the gel running chamber. Fill the box with 1xTAE buffer enough to cover the whole gel.
																
																5.	Load the following samples in order into the wells of 1.2% agarose gel:    (1)	1kb-DNA ladder, 6 μl   (2)	PCR control, 5 μl   (3)	DNA PCR product, 5 μl</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse21">Preparation LB broth and LB plates</A></H4></DIV><DIV id="collapse21" class="panel-collapse collapse"><DIV class="panel-body"><TABLE class="protocol-table"><TBODY><TR><TD colspan="2"> LB broth component</TD></TR><TR><TD>Tryptone</TD><TD>10 g</TD></TR><TR><TD>Yeast extracts</TD><TD>5 g</TD></TR><TR><TD>NaCl</TD><TD>5 g</TD></TR><TR><TD>＊Agar (LB plate)</TD><TD>15 g</TD></TR><TR><TD>H2O</TD><TD>1 L</TD></TR></TBODY></TABLE>
																LB broth preparation: 
																1.	Measure 5 g LB powder, and then pour it to 500 ml Erlenmeyer flask with 200 ml water pre-loaded 
																2.	Add water up to 250ml when the LB powder melts completely.
																LB plates preparation: 
																1.	Measure 5 g LB powder, and then pour it to 500 ml Erlenmeyer flask with 200 ml water pre-loaded.
																2.	Add water up to 250 ml when the LB powder melts completely.
																3.	Pour 3.75 g agar powder and throw into a stir bar.
																4.	Autoclave at 121℃for 30 min (if the container is a bottle with a screw cap, be sure to remember to unscrew the little screw cap)
																5.	After sterilization, put it in 60℃water bath until the solution temperature dropped to 60℃. 
																6.	Add 250 μl ampicillin (100mg / ml) (the final concentration 100μg / ml) and stir mixing (stir gently, you should avoid bubbles)
																7.	Pour about 20-25 ml medium to each of the 90-mm plate (should avoid bubbles).
																8.	After agar plate solidification, packed into bags and mark the date and name. </DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse22">Restriction enzyme mapping</A></H4></DIV><DIV id="collapse22" class="panel-collapse collapse"><DIV class="panel-body">	
																1.	Note the concentrations of DNA labeled on the tube. 
																2.	Set up restriction digests (final volume is 10 μl for each reaction) <TABLE class="protocol-table"><TBODY><TR><TD>Plasmid (0.2 mg/ml)</TD><TD>5 μl</TD></TR><TR><TD>10X buffer</TD><TD>1 </TD></TR><TR><TD>Restriction Enzyme</TD><TD>(10 units/μl) 1 μl</TD></TR><TR><TD>ddH<FONT style=" vertical-align:sub;font-size:0.5em;">2</FONT>O</TD><TD>3 μl</TD></TR><TR><TD>Final volume</TD><TD>10 μl</TD></TR></TBODY></TABLE>
																Incubate at 37℃ in a dry heat-block for 1 hr. 
																
																3.	During the incubation of the digestion reactions, prepare a 1.2% agarose gel:    (1)	Measure 1.92 g agarose powder to a 500ml-bottle, and add 160 ml 1X TAE buffer.    (2)	Melt agarose solution in microwave until the powder is dissolved. Be careful not to over-heat it, or else too much water may evaporate and result in gels of the incorrect agarose concentration.  
																   (3)	Let gel solution cool down to 60-70°C (very warm but not hot).    (4)	Add 12 μl of DNA View.    (5)	Pour agarose to the gel holder with the gel comb (25-well tooth) in place. After the agarose gel is solidified, transfer it to the gel running chamber. Fill the box with 1X TAE buffer enough to cover the whole gel.  
																
																4.	Briefly centrifuge the reactions, add 2 μl of 6X loading dye and mix by pipetting. 
																5.	Load the following samples in the indicated order into the wells of the agarose gel: 
																
																   (1)	1kb-DNA ladder, 5μl   (2)	Uncut wild-type DNA control, 4μl   (3)	Digested sample 8μl 
																
																*Uncut DNA control: In microcentrifuge tubes add 5μl Uncut DNA, 5 μl of ddH<FONT style=" vertical-align:sub;font-size:0.5em;">2</FONT>O, and 2 μl of 6X DNA loading dye. Mix well by gentle pipetting.
																6.	Turn on the power supply and run at 100V for 30 min. 
																7.	Carefully lift the gel and visualize all DNA fragments under camera-equipped gel documentation system. Take a picture of the gel. 
																8.	Figure out the restriction enzyme maps of our test plamsids based on our digests and compares them to the predicted version we drew. 
																9.	Judge if the plasmids are correct </DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse23">Transformation (all steps before heat-shock needs to be on ice!!)</A></H4></DIV><DIV id="collapse23" class="panel-collapse collapse"><DIV class="panel-body">
																1.	Thaw a tube of DH10βcompetent bacteria on ice. Pre-cool two clean microcentrifuge tubes and ligation reactions. 
																2.	Transfer 50μl each of bacteria cells into two tubes. Do not pipet bacteria! 
																3.	Add 2μl ligation product to bacteria. Incubate DNA/bacteria mix on ice for 30 min. 
																4.	Heat shock competent bacteria at 42°C for 1min. 
																5.	Incubate DNA/bacteria mix on ice for 3min. 
																6.	Add 300μl LB broth to each tube. Incubate bacteria at 370C for 30 min-1h. 
																7.	Plate all bacteria/LB mix onto LB-Amp plates: transfer all LB/bacteria mix onto plates, add 5-10 glass plating beads to each plate, close lid of plates and shake gently so that the bacteria/LB mix is evenly distributed on the plates. Shake until there is no visible liquid left. Pour glass beads into beaker with ethanol (do not throw away or put back together with clean, unused beads!!) 
																8.	Incubate plates at 370C overnight. 
																9.	Come in tomorrow morning to count colonies!! </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>