Team:CGU Taiwan/Description
Home | CGU_Taiwan
Project
Experiment
Protocols
Modeling
Result
Safety
Parts
Team Parts
Basic Parts
Composite Parts
Part Collection
Human Practice
Collaboration
About Us
Attribution
Notebook
Overview & Timeline
Lab Notes
Home | CGU_Taiwan
Project
Motivation
  The World Health Organization estimates that, around the world, there were 4 million new cancer cases and 8.2 million cancer-related deaths in 2012.
  In Taiwan, according to cancer statistics released by the Health Promotion Administration (HPA), one is diagnosed with cancer every 5 minutes and 26 seconds in 2012, and the record was 14 seconds faster than the previous year. The major cancers in Taiwan includes colon, lung, liver, breast, oral cancers, however, while the other cancers are decreasing their influences, oral cancer had the highest mortality rate increase from 2002 to 2012!
  There are many causes of oral cancer, and 90% of oral cavity cancer is due to chewing betel nuts. Up to 200 million people have this bad oral habit around the world, and the population centers in the southeastern Asia. This is probably why southeast Asia has the most number of oral cancer patients worldwide.
  Oral cancer is not so successfully controlled, as it is mostly detected by direct observation, which can easily lead to erroneous detection, also, lots of patients refuse further evaluation for fear of invasive biopsy. Consequently, oral cancer is mostly diagnosed when it is already at the 4th stage, and this significantly reduces the survival rate.
  As oral cancer is such a severe and important diseases in Taiwan and in Asia, we want to put our efforts in improving the condition by constructing an objective and noninvasive early detector, which can sense the potential overexpressed salivary biomarkers simply by testing patients’ saliva. Also, by helping to establish correct health concept through health education, we want to raise public’s awareness of oral cancer.
  Through building an objective and non-invasive detector, we hope we can do some changes and make this world a better place with less pain and less late stage oral cancer diagnosis.
Yeast With IL-8 Receptor
Overview
Many papers recently report that interleukin 8 (IL8) is one of the potential salivary biomarkers in oral cavity cancer, which has significantly different levels between control and patient groups. It is proposed that IL8 can induce angiogenesis and play an important role in tumorigenesis.              IL8 receptor belongs to GPCR family. Nowadays, many research characterizes GPCR-mediated pathway in yeast and its application in clinical detection. For the above reason, we choose yeast as our bio-detector and constructed a yeast strain optimized for expressing human GPCR (IL8 receptor, also named CXCR1).
Engineered Yeast pheromone sensing signaling pathway
  One of the most well-known MAPK cascade in yeast is pheromone sensing signaling pathway. In wild-type yeast, ste2/ste3 receptor binds to pheromone α/a factor and downstream pathway is activated for mating response
  In our experiment design, CXCR1 is overexpressed in yeast and mediates MAPK cascade. We got a strain BY4741 with GFP-tagged fus1 gene from Dr.Schuyler ‘s lab. In the engineered pathway, far1 gene is deleted to prevent cell cycle arrest in G1 phase. We also replaced the last five amino acid at the C-terminal of Gpa1 by human Gαi2 to promote the interaction  between CXCR1and Gpa1.
CXCR 1
IL8 binds to two receptors: CXCR1 and CXCR2. CXCR1 (also named with IL8RA and CD181) has high affinity in binding with IL8, whereas, CXCR2 nonspecifically binds to many chemokines. So we choose the high affinity binding receptor CXCR1 for expression in yeast. CXCR1 belongs to the GPCR rhodopsin-like family and has seven-transmembrane domains, with the N-terminus exposed to the extracellular space and the C-terminus exposed to cytoplasm. Different from ordinary GPCR ligand binding sites, in which of CXCR1 locate on N-terminal and extracellular loop 3 of CXCR1             To improve heterologous expression of CXCR1 in yeast, we inserted two different signal peptide sequences upstream of the CXCR1 sequence.             One is the rhodopsin signal peptide (called Rho), and the other is a combination of leucine rich repeat 32 protein signal peptide and rho (called Lucy-rho). In addition, the receptor sequence has been optimized to fit to the translational system in yeast.
Toehold Switches as RNA sensors
Background
Toehold switch is a hairpin structure RNA. The reporter gene is also assembled downstream to the hairpin, and helps indicate the result of the experiment. The hairpin structure hides the ribosome binding site and the starting codon, so the translation of the reporter gene is repressed.             The toehold switch structure is designed to detect the target RNAs, it contains the sequences perfectly complementary to the target RNA in the former half of the hairpin. When the target RNA is present, the toehold switch binds to it and this binding opens the hairpin.  As a result, the initially hidden ribosome binding site and the starting codon will be exposed. The reporter gene at downstream of the switch can now be expressed.
Toehold switch sequence design
Seven oral cancer biomarkers are found in a previous research [3], and four of them are chosen to be the target of our project, which are IL-8, IL-1β, dual specificity phosphatase 1(DUSP1), and spermine N1-acetyltransferase(SAT). We design our toeholds by getting our target sequences from the NCBI website, and then we define and design our target region into 30 base pairs. It is very important that it contains about equal amounts of ATCGs, and we need to make sure they don’t match with human genome; also, they cannot contain restriction enzyme sites in the sequences. When we complete the stem loop, we add luciferase, which is our reporter, at the end of the sequences. Finally, predict the structure with RNA Structure website (http://rna.urmc.rochester.edu/RNAstructureWeb) to confirm the secondary structure of our toehold switches.
Biobricks assemble
To construct the sensor and test its quality, we built two plasmids which contain toehold switches and triggers respectively.              In the toehold switch, we sent the designed sequences to the IDT Company, and had the toehold switch synthesized with a T7 promoter (BBa_R0085) upstream to it. In order to quantify our detection, we assemble the toehold switch and reporter protein, luciferase. The T7-toehold part is digested with EcoRI and SpeI, then ligated to the luciferase (BBa_I712019) provided by iGEM, which is digested with EcoRI and XbaI. The T7-toehold switch-luciferase is later subcloned to pSB1C3 backbone.             As for the trigger part, we construct it to validate the function of toehold switch. The designed trigger is synthesized by the IDT Company, and digested with XbaI and PstI. It then ligate downstream to the T7 promoter (BBa_R0085) digested by SpeI and PstI.
Prototype Design
Devices
Master Mold Fabrication:
All designs were drawn with solidwork  software and printed out as film photomasks. Precision engraving machine engraving PMMA master mold.
PDMS Replication and molding=>Loading Channel Molding:
PDMS Sylgard 184 from (Dow Corning, USA) pre-polymer mixed with its crosslinker at a 10 : 1 ratio, was poured onto the master mold.After heating 80 degrees celsius in the oven for one hour. The PDMS device is cutout from the mold, and holes are excised with a biopsy punch to allow tubing to be inserted into the device. Lastly, the device is bonded to a glass substrate via O2 plasma for support.
How the device works?
First,we use syringe to inject saliva from inlet to outlet.Second,inject  yeast to outlet.Because we want yeast and saliva to mix well,we inject magnetic beads into outlet. Then,place the chip on the stirring hotplate stir until saliva is mixed up with the ????.And we hope magnetic beads will make yeast evenly distributed in chamber.             Our ultimate goal is to see the yeast will radiate GFP and we can analyze it by fluorescence spectrophotometer.
Future Work
Given the fact that chewing betel nuts, smoking and drinking are the most significant causes of oral cancer and that people who keep these bad habits are labor workers that need refreshment, there are certain occupations that cover the most consumers; for instance, cab drivers, truck drivers and constructional workers. To approach these high-risk population with our early detection devices, we want to establish the detector-buses that tour around communities, especially to sites such as cab stands, travel centers, constructional cites, and even rehabs where we can encounter people that have ever been or still are consumers. After getting the results, we can then inform the subject if he or she should get further examination.             The advantage of our detector-buses are that their mobility offers chances to benefit people all over the country and that they are able to provide a moderate environment for the devices equipped on them so that the results wouldn’t be affected by conditions like temperature or humidity.              Apart from the detector-buses that are mainly for the specific groups of people mentioned above, we also want to set up our early detection devices in clinics or community health centers so that the general public can also gain access to them.
References
Brown, A.J., Dyos, S.L., Whiteway, M.S., White, J.H., Watson, M.A., Marzioch, M., Clare, J.J., Cousens, D.J., Paddon, C., Plumpton, C., et al. (2000). Functional coupling of mammalian receptors to the yeast mating pathway using novel yeast/mammalian G protein alpha-subunit chimeras. Yeast (Chichester, England) 16, 11-22.
Cheng, Y.S., Rees, T., and Wright, J. (2014). A review of research on salivary biomarkers for oral cancer detection. Clinical and translational medicine 3, 3.
Green, A.A., Silver, P.A., Collins, J.J., and Yin, P. (2014). Toehold switches: de-novo-designed regulators of gene expression. Cell 159, 925-939.
Ladds, G., Goddard, A., and Davey, J. (2005). Functional analysis of heterologous GPCR signalling pathways in yeast. Trends in Biotechnology 23, 367-373.
Park, S.H., Das, B.B., Casagrande, F., Tian, Y., Nothnagel, H.J., Chu, M., Kiefer, H., Maier, K., De Angelis, A.A., Marassi, F.M., et al. (2012). Structure of the chemokine receptor CXCR1 in phospholipid bilayers. Nature 491, 779-783.
Pardee, K., Green, A.A., Ferrante, T., Cameron, D.E., DaleyKeyser, A., Yin, P., and Collins, J.J. (2014). Paper-based synthetic gene networks. Cell 159, 940-954.
