Team:CGU Taiwan/Results
Home | CGU_Taiwan
Project
Experiment
Protocols
Modeling
Result
Safety
Parts
Team Parts
Basic Parts
Composite Parts
Part Collection
Human Practice
Collaboration
About Us
Attribution
Notebook
Overview & Timeline
Lab Notes
Home | CGU_Taiwan
Results
Yeast With IL-8 Receptor
Overview
1. Identification of FUS1-GFP strain with α factor induction
2. FAR1 gene deletion and replacement by KanMX gene
3. Gpa1(Gαsubunit) modification to improve the affinity between GPCR and Gpa1
4. Construction of new yeast integrating plasmids
5. Protein expression with galactose induction
6. Evaluating the function of the receptor with IL-8 treatment
1. Identification of FUS1-GFP strain with α factor induction
An initial test was performed in order to confirm that FUS1-GFP strain can express bright green fluorescence protein with α factor induced.
We also select a strain which highly expresses GFP as our positive control, and a wild type yeast strain as negative control.
Figure 1. Pictures taken under UV yield and 2 seconds exposure time
Note: The left three one is with αfactor while the right withoutαfactor
2. FAR1 gene deletion and replacement by KanMX gene
One task of our project is FAR1 gene deletion, which makes for cell-cycle arrest. Because FAR1 gene will also be activated in yeast endogensis MAPK pathway. To achieve this goal, we adopt PCR method to construct recombinant fragment.
Figure 1. A schematic diagram of homologous recombination for gene knockout
KANMX gene was amplified with primers P1 and P2 . It is worth to say the fragments containing flanking DNA fragment so that recognition and recombination exchange Far1 gene with KanMX . The expected size of the KanMX is about 1900 bp which corresponded to the sizes observed on gel.
After transformed amplified PCR product into FUS1-GFP strain , we did PCR to check whether homologous recombination successfully or not.PCR ran with forward primer outside of the inserted region and reverse primer within the KanMX  respectively. The expected length of check PCR product is about 1300bp , and from the gel result , it indicated that homologous recombination worked.
Figure 2. the result of PCR KanMX fragment on gel
Figure 3 . Electrophoresis showing result of FAR1△::KanMX
3. Gpa1(Gαsubunit) modification to improve the affinity between GPCR and Gpa1
Furthermore, we also considered the affinity between CXCR1 (hGPCR) and the endogenous yeast G-proteinαsubunit (Gpa1) . To conquer the field , we intended to replace C terminal five amino acids of Gpa1 with IL-8 specific Gα-CXCR1 (called GNAI2 in human). We chose PCR to achieve Gpa1 gene modification.              The selection gene Leu2 template was amplified by PCR from plasmid pRS405 containing this selection gene with primer P1 and P2 , and then the PCR product was transformed into FAR1△::KanMX-FUS1-GFP strain.               To check the result of homologous recombination , we did PCR with primer P3 and P4 to check whether Gpa1 gene modification was succeeded.
Figure 1 . A schematic diagram of PCR method of Gpa1 chimera
Figure 2. Electrophoresis showing the result of PCR Leu2 fragment
Note: The expected length of PCR is about 1600bp and it worked.
Figure 3 . Electrophoresis showing the result of homologous recombination
Note: The negative control was PCR product with template--Gpa1 endogensis gene while the lane1/2/3/4 were PCR product with template—Gpa1 chimera . The expected length was about 600bp and it met the expection.
4. Construction of new yeast integrating plasmids
Plasmids cloning vectors that can “shuttle” DNA between yeast and bacteria we used is called pGAL426. Distinctly, it has different selection markers in yeast and bacteria. Rho-CXCR1 and Lucy-rho-CXCR1 was inserted into this shuttle vector separately and finally transformed into FAR1△::KanMX-FUS1-GFP Gpa1 chimera strain.
Figure 1(a) The result of pGal426 Lucy-rho-CXCR1 with BamHI and XhoI digestion.
Figure 1(b) The result of pGAL426 rho-CXCR1 with BamHI and XhoI digestion.
Note: These two restriction enzyme site designed on insert sequence was used to check the ligation performance by enzyme digestion. Theoretically, there would be two bands.
5. Protein expression with inducer
The galactose induction of GPCR expression was performed as described in protocol. Proteins from the yeast whole-cell extract were separated by electrophoresis on SDS-PAGE and transferred to membrane to run Western Blot.
Figure 1 Western blot analysis of rho-CXCR1 expression
Figure 2 Western blot analysis of Lucy-rho-CXCR1 expression
6. Evaluating the function of the receptor with IL-8 treatment
To test whether the constructed GPCR was functional on the yeast cell membrane, we firstly prepare the spheroplasts to increase the effiency of binding ligand without cell wall. After preparation , we tested the sensitivity toward 10μM IL-8 and observed under microscope. Although the signal of GFP was not too high to distinguish easily by eye, we still saw the difference between test strain and negative control (no IL-8 treatment) via camera.
Toehold Switch As RNA Senor
Result toehold secondary structure prediction
The pictures are the secondary structure prediction of four toehold switches that we design, which we want to use to sense the four biomarkers we chose to detect in oral cancer’s patients’ saliva. We use the website RNA structure :
http://rna.urmc.rochester.edu/RNAstructureWeb/Servers/Predict1/Predict1.html
to predict their secondary structures. These structures must contain hairpin structure, which could have the switch on-off mechanism to sense the designed target RNA and trigger RNA we to test its efficiency, and further the sensitivity and specificity of this detection method. Also, the secondary structure can’t be annealed randomly, and the linker and trigger RNA can’t be complementary. These four secondary structures of the sequences are correspond with our expectation. Then we can further do the experiment of construction of toehold switches.
Toehold switch sequence design
Seven oral cancer biomarkers are found in a previous research [3], and four of them are chosen to be the target of our project, which are IL-8, IL-1β, dual specificity phosphatase 1(DUSP1), and spermine N1-acetyltransferase(SAT). We design our toeholds by getting our target sequences from the NCBI website, and then we define and design our target region into 30 base pairs. It is very important that it contains about equal amounts of ATCGs, and we need to make sure they don’t match with human genome; also, they cannot contain restriction enzyme sites in the sequences. When we complete the stem loop, we add luciferase, which is our reporter, at the end of the sequences. Finally, predict the structure with RNA Structure website (http://rna.urmc.rochester.edu/RNAstructureWeb) to confirm the secondary structure of our toehold switches.
Biobricks assemble
To construct the sensor and test its quality, we built two plasmids which contain toehold switches and triggers respectively.              In the toehold switch, we sent the designed sequences to the IDT Company, and had the toehold switch synthesized with a T7 promoter (BBa_R0085) upstream to it. In order to quantify our detection, we assemble the toehold switch and reporter protein, luciferase. The T7-toehold part is digested with EcoRI and SpeI, then ligated to the luciferase (BBa_I712019) provided by iGEM, which is digested with EcoRI and XbaI. The T7-toehold switch-luciferase is later subcloned to pSB1C3 backbone.             As for the trigger part, we construct it to validate the function of toehold switch. The designed trigger is synthesized by the IDT Company, and digested with XbaI and PstI. It then ligate downstream to the T7 promoter (BBa_R0085) digested by SpeI and PstI.
Plasmid construction we have done
We construct toehold switch part in pSB1C3 backbone, and construct trigger in pSB1AK8 backbone for use of different antibiotic to check whether the bacteria uptake both plasmids in it.
These two plasmids are what we have constructed. One is toehold switch with T7 promoter and luciferase in pSB1C3 backbone; the other is trigger RNA with T7 promoter in pSB1AK8 backbone. The pSB1C3 T7-toehold-luciferase construct also contains chloramphenicol antibiotic for selection. On the other hand, the pSB1C3 T7-trigger construct contain chloramphenicol antibiotic for selection. The two plasmids are constructed in different backbones for further cotransformation selection, this way we can check whether the two plasmids are both successfully uptaken by the bacteria or not.
Restriction enzyme digestion
(A) 1. Luciferase cut with EcoRI and XbaI.             Lane 1 and 4, 3 μg of luciferase digested by EcoRI and XbaI ; Lane 2, 1K DNA marker ;  Lane 3, 3 μg  of luciferase not digested by EcoRI and  XbaI.             (B) Restriction enzyme digestion of synthetic toehold and trigger RNA with EcoRI and SpeI. Samples were run in 1% agarose gel. Lane 1, 1kb DNA marker；Lane 2-5, 125 ng of synthetic DNA fragment that would transcribe into toehold sensors (SAT、DUSP1、IL-1β、IL-8 ) were digested by EcoRI and SpeI；Lane 6-9, 125 ng of DNA fragment that would transcribe into trigger RNAs (SAT、DUSP1、IL-1β、IL-8 ) were digested by EcoRI and SpeI.              (C) T7 promoter digestion with restriction enzyme PstI and SpeI.  Lane 1, marker ; Lane 2, 3 μg T7 promoter digested by restriction enzyme PstI and SpeI.
Preparation of parts to construct two vectors system
    We use enzyme to digest, and cut out the bands we want, and then run electrophoresis then amplifying the toehold and trigger RNA we would use.  In Figure A, the backbone, pSB1AK8, is 3426 base pairs, when it be digested with restriction enzyme and added luciferase, which is 1653 base pairs, it would be 4937 base pairs. Then we use enzyme EcoRI and XbaI to digest the plasmid. The results match our expectations.
    According to Figure B, toehold and trigger RNA. To cut out the toehold and trigger RNAs, we digest them with EcoRI and SpeI. The toehold switch sensor are all 167 base pairs, the trigger RNAs are 149 base pairs. The results match our expectation. In Figure C, we use restriction enzyme PstI and SpeI to digest. Then cut out the T7 promoter, thus further we could amplify the T7 promoter. The T7 promoter part in pSB1AK8 is about 3K .
    After we confirm the sequences are correct, we digest toehold switches, luciferases from pSB1AK8, trigger RNAs, and T7 promoters from pSB1C3. Then we construct two vectors, one contains T7 promoter luciferase and toehold switch sequence in pSB1C3 backbone ; the other vector, pSB1AK8, contains T7 promoter trigger RNAs sequence. After construct these two vectors, we sent it to the biotechnology company to confirm the sequence. The confirmed final plasmids are showed in Figure 7. In the future, we will cotransform these two vector into BL21*DE3 and test the luciferase’s brightness, then we can lysate the bacteria, which already have these two plasmids, and use ELISA reader to test whether there is light or not. Further to build the sensitivity standard curve.
  Discussion
    In Figure , the size of cut is about 5 K. Uncut DNA’s structure is supercoiled, it should be below the cut DNA. After discussing with the advisor, we still don’t know the reason why the uncut doesn’t meet our expectation. We predict that the structure of uncut plasmid didn’t fold perfectly, so it didn't run rapidly.
Luciferase brightness test
After construct two plasmids, we want to test the sensor efficiency of the toehold switch structure. So we then transform the two plasmids into BL21 bacteria then break the bacteria to get lysate. We use PCR to make a positive control, a fragment of sequence contains T7 promoter, toehold switch, and luciferase. And our negative control for this part is T7 promoter with luciferase.
We lysed the bacteria to get the lysate. There are two groups, one is toehold switch only; the other is toehold switch treated with trigger, which is made by cotransformation of the two constructed plasmids. The two plasmids can be selected for that they were constructed with different backbones  which contains different antibiotics resistance. Thus, the bacteria can survive the environment with ampicillin and chloramphenicol antibiotics added if the two plasmids are successfully cotransformed.
PCR for T7-luciferase
In the figure, each lane are added with 10 μl of PCR products, we use temperature gradient (50°C, 53°C, 56°C, 59°C, 62°C, 65°C) to test the product of which temperature condition is suitable to be our negative control of in vivo functional assay.
To do the in vivo part of functional assay, we must have control condition. This figure shows the process that we use temperature gradient to choose which temperature is the most appropriate for us to do the experiment of the in vivo part of functional assay. Finally, we choose the result of 53 °C for PCR. Because the product of 53 °C condition is much clearer, also much brighter then other conditions.
