<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CGU_Taiwan_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CGU Taiwan/Notebook</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Home | CGU_Taiwan</TITLE></P><DIV class="navbar navbar-inverse" role="banner"><DIV class="container"><DIV class="collapse navbar-collapse"><UL class="nav navbar-nav navbar-right"><LI class="header-list"><A href="https://2015.igem.org/Team:CGU_Taiwan/Description"><FONT size="4">Project</FONT></A></LI><LI class="dropdown header-list"><A href="#" class="list-arrow"><FONT size="4">Experiment </FONT></A></LI><LI><A href="https://2015.igem.org/Team:CGU_Taiwan/Experiments">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:CGU_Taiwan/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:CGU_Taiwan/Results">Result</A></LI><LI><A href="https://2015.igem.org/Team:CGU_Taiwan/Safety">Safety</A></LI></UL><LI class="dropdown header-list"><A href="#" class="list-arrow"><FONT size="4">Parts </FONT></A></LI><LI><A href="https://2015.igem.org/Team:CGU_Taiwan/Parts">Team Parts</A></LI><LI><A href="https://2015.igem.org/Team:CGU_Taiwan/Basic_Part">Basic Parts</A></LI><LI><A href="https://2015.igem.org/Team:CGU_Taiwan/Composite_Part">Composite Parts</A></LI><LI><A href="https://2015.igem.org/Team:CGU_Taiwan/Part_Collection">Part Collection</A></LI><LI class="header-list"><A href="https://2015.igem.org/Team:CGU_Taiwan/Practices"><FONT size="4">Human Practice</FONT></A></LI><LI class="header-list"><A href="https://2015.igem.org/Team:CGU_Taiwan/Collaborations"><FONT size="4">Collaboration</FONT></A></LI><LI class="header-list"><A href="https://2015.igem.org/Team:CGU_Taiwan/Team"><FONT size="4">About Us </FONT></A></LI><LI class="header-list"><A href="https://2015.igem.org/Team:CGU_Taiwan/Attributions"><FONT size="4">Attribution</FONT></A></LI><LI class="dropdown header-list"><A href="#" class="list-arrow"><FONT size="4">Notebook</FONT></A></LI><LI><A href="https://2015.igem.org/Team:CGU_Taiwan/Timeline">Overview &amp; Timeline</A></LI><LI><A href="https://2015.igem.org/Team:CGU_Taiwan/Notebook">Lab Notes</A></LI></DIV></DIV></DIV><P><TITLE>Home | CGU_Taiwan</TITLE></P><DIV class="container"><DIV class="single-blog blog-details two-column " id="Y"><H1 class="post-title bold">Lab Note</H1><H2>Yeast With IL-8 Receptor</H2></DIV><DIV id="accordion-container"><DIV class="panel-group" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseY71">2015.07.01</A></H4></DIV><DIV id="collapseY71" class="panel-collapse collapse "><DIV class="panel-body">
																Operator: Wan-Yun, Jin-Ting, Jin-Ye
																Goal: 
																  1. Extraction of gDNA of Far1∆::KANMX strain  2. Detection the concentration of fast extracted gDNA of ∆Far1 strain  3. PCR gDNA of Far1∆ ::KANMX  4. Electrophoresis to check PCR product
																Experiment steps:&lt; Extraction of gDNA of Far1∆::KANMX strain&gt;
																Consult the protocol<PROTOCOL of="" fast="" extraction="" of="" gdna="" of="" yeast=""><DETECTION the="" concentration="" of="" fast="" extracted="" gdna=""><TABLE class="protocol-table"><TBODY><TR><TD>Strains </TD><TD>260/280 </TD><TD>260/230 </TD><TD>C(ng/μl) </TD></TR><TR><TD>Far1∆::KANMX strain </TD><TD>1.62 </TD><TD>0.97 </TD><TD>44.7 </TD></TR></TBODY></TABLE><PCR gdna="" of="" far1="">
																1.	Design of primers<TABLE class="protocol-table"><TBODY><TR><TD> Name</TD><TD>Pur. </TD><TD>Seq.(5’-3’) </TD><TD>Size (mer.) </TD><TD> MW (g/mol)</TD><TD>Tm (℃)</TD><TD>GC (%)</TD><TD>Nmole</TD><TD>μl for 100μM </TD></TR><TR><TD>dFAR1 F’</TD><TD>Desalt </TD><TD>ggTTTTgTTAggCgggCAAg </TD><TD>20 </TD><TD>6244.1 </TD><TD>53.8 </TD><TD>55</TD><TD>21.25</TD><TD>212.50</TD></TR><TR><TD>dFAR1 R’</TD><TD>Desalt </TD><TD>CATTAACTgCTATTTACgACgC </TD><TD>22 </TD><TD>6669.4 </TD><TD>51.1 </TD><TD>41</TD><TD>17.12</TD><TD>171.20</TD></TR></TBODY></TABLE>
																2.	PCR program
																<TABLE class="protocol-table"><TBODY><TR><TD>Step </TD><TD>Temperature </TD><TD>Time</TD></TR><TR><TD>Step1 </TD><TD>95℃ </TD><TD>5min</TD></TR><TR><TD>Step2 </TD><TD>95℃ </TD><TD>30s</TD></TR><TR><TD>Step3 </TD><TD>52℃ </TD><TD>30s</TD></TR><TR><TD>Step4→step2 for 30 cycle </TD><TD>72℃ </TD><TD>2min </TD></TR><TR><TD>Step5 </TD><TD>72℃ </TD><TD>5min </TD></TR><TR><TD>Step6 (hold on) </TD><TD>10℃ </TD><TD>1hr </TD></TR></TBODY></TABLE>
																3.PCR reagent
														
																<TABLE class="protocol-table"><TBODY><TR><TD>10x Dream Taq buffer </TD><TD>2.5μl</TD></TR><TR><TD>2.5mM dNTP</TD><TD>0.5μl</TD></TR><TR><TD>10mM primer(F)</TD><TD>0.5μl</TD></TR><TR><TD>10mM primer(R)</TD><TD>0.5μl </TD></TR><TR><TD>template (Far1∆::KANMX strain gDNA) </TD><TD>3.4μl</TD></TR><TR><TD>Taq polymerase</TD><TD>0.5μl</TD></TR><TR><TD>ddH2O </TD><TD>17.1μl</TD></TR><TR><TD>Total volume</TD><TD>25μl</TD></TR></TBODY></TABLE>&lt; Electrophoresis to check PCR product&gt;
																Material:   DNA marker: 100bp ladder 8μl  DNA sample: 2μlPCR product + 1μl 6x loading buffer
																Condition:   0.5xTBE buffer 100V
																Time:  30min
																Result:  M：Marker；#1：Far1 ∆ for annealing at 52℃  There is no band appears in the gel electrophoresis.
																Conclusion:  Due to the Figure 1 in result, we have to check the temperature of primers annealing and the design of primers to solve the problem. Next, we use gradient PCR to find out the exactly temperature of primer annealing and we add positive control as well.
																
																
															</PCR></DETECTION></PROTOCOL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseY72">2015.07.02</A></H4></DIV><DIV id="collapseY72" class="panel-collapse collapse "><DIV class="panel-body">
																Operator: Wan-Yun, Jin-Ting, Jin-Ye
																Goal:   1. Extraction of gDNA of FAR1∆::KANMX strain   2. Detection the concentration of fast extracted gDNA of FAR1∆::KANMX strain  3. First round of PCR   4. Electrophoresis to check first round-PCR product  5. Second round of PCR 
																Experiment steps:&lt; Extraction of gDNA of FAR1∆::KANMX strain&gt;
																Consult the protocol <PROTOCOL of="" fast="" extraction="" of="" gdna="" of="" yeast=""><DETECTION the="" concentration="" of="" fast="" extracted="" gdna=""><TABLE class="protocol-table"><TBODY><TR><TD>Strains </TD><TD>260/280 </TD><TD>260/230 </TD><TD>C(ng/μl) </TD></TR><TR><TD>Far1∆::KANMX </TD><TD>1.72 </TD><TD>0.78 </TD><TD>42.7 </TD></TR><TR><TD>Positive control </TD><TD>1.63 </TD><TD>0.76 </TD><TD>37.1 </TD></TR></TBODY></TABLE><FIRST round="" of="" pcr="">
																1. Information of primers<TABLE class="protocol-table"><TBODY><TR><TD> Name</TD><TD>Pur. </TD><TD>Seq.(5’-3’) </TD><TD>Size (mer.) </TD><TD> MW (g/mol)</TD><TD>Tm (℃)</TD><TD>GC (%)</TD><TD>Nmole</TD><TD>μl for 100μM </TD></TR><TR><TD>dFAR1 F’</TD><TD>Desalt </TD><TD>ggTTTTgTTAggCgggCAAg </TD><TD>20 </TD><TD>6244.1 </TD><TD>53.8 </TD><TD>55</TD><TD>21.25</TD><TD>212.50</TD></TR><TR><TD>dFAR1 R’</TD><TD>Desalt </TD><TD>CATTAACTgCTATTTACgACgC </TD><TD>22 </TD><TD>6669.4 </TD><TD>51.1 </TD><TD>41</TD><TD>17.12</TD><TD>171.20</TD></TR></TBODY></TABLE>
																2.PCR program<TABLE class="protocol-table"><TBODY><TR><TD>Step </TD><TD>Temperature </TD><TD>Time</TD></TR><TR><TD>Step1 </TD><TD>95℃ </TD><TD>5min</TD></TR><TR><TD>Step2 </TD><TD>95℃ </TD><TD>30s</TD></TR><TR><TD>Step3 </TD><TD>Gradient42℃-46℃-50℃ </TD><TD>30s</TD></TR><TR><TD>Step4→step2 for 30 cycle </TD><TD>72℃ </TD><TD>2min </TD></TR><TR><TD>Step5 </TD><TD>72℃ </TD><TD>5min </TD></TR><TR><TD>Step6 (hold on) </TD><TD>10℃ </TD><TD>1hr </TD></TR></TBODY></TABLE>
																3.PCR reagent<TABLE class="protocol-table"><TBODY><TR><TD>10x Dream Taq buffer </TD><TD>2.5μl</TD></TR><TR><TD>2.5mM dNTP</TD><TD>0.5μl</TD></TR><TR><TD>10mM primer(F)</TD><TD>0.5μl</TD></TR><TR><TD>10mM primer(R)</TD><TD>0.5μl </TD></TR><TR><TD>template (Far1∆::KANMX strain gDNA) </TD><TD>3.4μl</TD></TR><TR><TD>Taq polymerase</TD><TD>0.5μl</TD></TR><TR><TD>ddH2O </TD><TD>17.1μl</TD></TR><TR><TD>Total volume</TD><TD>25μl</TD></TR></TBODY></TABLE>&lt; Electrophoresis to check first round-PCR product (25μl ul)&gt;
																Material:  DNA marker: 100bp ladder 8μl  DNA sample: 2μl PCR product + 1μl 6x loading buffer
																Condition:   0.5xTBE buffer 100V 
																Time:   30min
																Result:
																M: Marker; #1: FAR1∆ annealing at 42 ℃; #2: FAR1∆ annealing at 46 ℃;
																#3: FAR1∆ annealing at 50 ℃; #4: FAR1∆ annealing at 52 ℃(positive control)
																<SECOND round="" of="" pcr="">
																1.PCR program<TABLE class="protocol-table"><TBODY><TR><TD>Step </TD><TD>Temperature </TD><TD>Time</TD></TR><TR><TD>Step1 </TD><TD>95℃ </TD><TD>5min</TD></TR><TR><TD>Step2 </TD><TD>95℃ </TD><TD>30s</TD></TR><TR><TD>Step3 </TD><TD>46℃</TD><TD>30s</TD></TR><TR><TD>Step4→step2 for 30 cycle </TD><TD>72℃ </TD><TD>2min </TD></TR><TR><TD>Step5 </TD><TD>72℃ </TD><TD>5min </TD></TR><TR><TD>Step6 (hold on) </TD><TD>10℃ </TD><TD>1hr </TD></TR></TBODY></TABLE>
																2.PCR reagent
																<TABLE class="protocol-table"><TBODY><TR><TD>10x Dream Taq buffer </TD><TD>5μl</TD></TR><TR><TD>2.5mM dNTP</TD><TD>1μl</TD></TR><TR><TD>10mM primer(F)</TD><TD>1μl</TD></TR><TR><TD>10mM primer(R)</TD><TD>1μl </TD></TR><TR><TD>template (First round PCR product) </TD><TD>1μl</TD></TR><TR><TD>Taq polymerase</TD><TD>1μl</TD></TR><TR><TD>ddH2O </TD><TD>40μl</TD></TR><TR><TD>Total volume</TD><TD>50μl</TD></TR></TBODY></TABLE></SECOND></FIRST></DETECTION></PROTOCOL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseY73">2015.07.03</A></H4></DIV><DIV id="collapseY73" class="panel-collapse collapse "><DIV class="panel-body">
																Operator: Wan-Yun, Jin-Ting, Jin-Ye
																Goal:   1. 1.Electrophoresis to check second round-PCR product  2. Transform PCR product into FUS::GFP(His) strain
																Experiment steps:&lt; Electrophoresis to check second round-PCR product (50μl)&gt;
																Material:  DNA marker: 100bp ladder 8μl
																  DNA sample:2μl PCR product + 1ul 6x loading buffer
																Condition: 0.5xTBE buffer 100V
																Time: 30min
																Result:
																Figure 1. Gel electrophoresis of FAR1∆
																M: Marker; #1: FAR1∆ annealing at 46 ℃<TRANSFORM pcr="" product="" into="" fus::gfp="">
																Consult the protocol &lt; protocol of yeast transformation&gt;
																Use strain name: FUS1-GFP(His)
																Selection plate: YPD+G418</TRANSFORM></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseY76">2015.07.06</A></H4></DIV><DIV id="collapseY76" class="panel-collapse collapse "><DIV class="panel-body">
																Operator: Wan-Yun, Jin-Ting, Jin-Ye
																Goal:   1. Check transformation result  2. Second round PCR   3. Electrophoresis to check PCR product  4. Incubate E.coli with shuttle vector-p426GAL1 from stock

																Experiment steps:<CHECK transfprmation="" resule="">
																1.	Take plates out from incubator and observe growth of colony
																2.	Conclusion: We failed to transformation of PCR product so we should it again.<SECOND round="" pcr="">
																Consult the experiment record &lt;2015.7.2 Experiment Record&gt;&lt; Electrophoresis to check second round-PCR product&gt;
																Material:  DNA marker: 100bp ladder 8μl   DNA sample:2μlPCR product + 1μl 6x loading buffer
																Condition: 0.5xTBE buffer 100V 
																Time: 30min
																Result:

																
																
																Conclusion: Its expected length is 1.9kb and it worked.&lt; Incubate E.coli with shuttle vector-p426GAL1 from stock&gt;
																1.	Take stock E.coli with shuttle vector-p426GAL1 from -80℃ fridge.
																2.	Plate E.coli on LB+Amp plates.
																3.	Incubate in 37℃ overnight.</SECOND></CHECK></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseY77">2015.07.07</A></H4></DIV><DIV id="collapseY77" class="panel-collapse collapse "><DIV class="panel-body">
																Operator: Wan-Yun, Jin-Ting, Jin-Ye
																Goal:   1.	Transform PCR product into FUS::GFP (His) strain
																Experiment steps:<TRANSFORM pcr="" product="" into="" fus::gfp="">
																Consult the experiment record &lt;2015.7.3 Experiment Record&gt;</TRANSFORM></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseY79">2015.07.09</A></H4></DIV><DIV id="collapseY79" class="panel-collapse collapse "><DIV class="panel-body">
																Operator: Wan-Yun, Jin-Ting, Jin-Ye
																Goal:   1. Miniprep plasmid of p426GAL1   2. Measure concentration of plasmid.
																Experiment steps:&lt; Miniprep plasmid of p426GAL1
																Consult the protocol <PROTOCOL of="" miniprep="" plamid=""><MEASURE concentration="" of="" plasmid=""><TABLE class="protocol-table"><TBODY><TR><TD> </TD><TD>concentration </TD><TD>260/280 </TD><TD>260/230 </TD></TR><TR><TD>P426GAL1/1</TD><TD>130.9ng/μl</TD><TD>1.91 </TD><TD>2.36 </TD></TR><TR><TD>P426GAL1/2</TD><TD>121.3ng/μl</TD><TD>1.89 </TD><TD>2.26 </TD></TR></TBODY></TABLE></MEASURE></PROTOCOL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseY712">2015.07.12</A></H4></DIV><DIV id="collapseY712" class="panel-collapse collapse "><DIV class="panel-body">
															Operator: Wan-Yun, Jin-Ting, Jin-Ye
															Goal:   1. Second round PCR   2. Incubate FAR1△::KANMX strain in 5ml YPD+A medium.
															Experiment steps:<SECOND round="" pcr="">
															Consult the experiment record &lt;2015.7.2 Experiment Record&gt;</SECOND></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseY717">2015.07.17</A></H4></DIV><DIV id="collapseY717" class="panel-collapse collapse "><DIV class="panel-body">
															Operator: Wan-Yun, Jin-Ting, Jin-Ye
															Goal: 
															1. Transform PCR product into FUS1-GFP strain 
															Experiment steps: &lt; Transform PCR product into FUS1-GFP strain&gt; 
															Consult the experiment record &lt;2015.7.3 Experiment Record&gt;</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseY720">2015.07.20</A></H4></DIV><DIV id="collapseY720" class="panel-collapse collapse "><DIV class="panel-body">
																Operator: Wan-Yun, Jin-Ting, Jin-Ye
																Goal:   1. Maintain the colonies of far1Δ::KanMX-FUS1-GFP.  2. 2nd round PCR for far1Δ::KanMX  3. Phenol chloroform and EtOH precipitation of 2nd round PCR product															
																Experiment steps:&lt; Maintain the colonies of far1Δ::KanMX-FUS1-GFP &gt;  1.	Choose 10 colonies to transfer the new YPD+G418 plate.  2.	Check plates after two days.&lt;2nd round PCR for far1Δ::KanMX 
																1.PCR program <TABLE class="protocol-table"><TBODY><TR><TD>Step </TD><TD>Temperature </TD><TD>Time</TD></TR><TR><TD>Step1 </TD><TD>95℃ </TD><TD>5min</TD></TR><TR><TD>Step2 </TD><TD>95℃ </TD><TD>30s</TD></TR><TR><TD>Step3 </TD><TD>46℃</TD><TD>30s</TD></TR><TR><TD>Step4→step2 for 30 cycle </TD><TD>72℃ </TD><TD>2min </TD></TR><TR><TD>Step5 </TD><TD>72℃ </TD><TD>5min </TD></TR><TR><TD>Step6 (hold on) </TD><TD>10℃ </TD><TD>1hr </TD></TR></TBODY></TABLE>
																2.PCR reagent<TABLE class="protocol-table"><TBODY><TR><TD>10x Dream Taq buffer </TD><TD>5μl</TD></TR><TR><TD>2.5mM dNTP</TD><TD>1μl</TD></TR><TR><TD>10mM primer(F)</TD><TD>1μl</TD></TR><TR><TD>10mM primer(R)</TD><TD>1μl </TD></TR><TR><TD>template (First round PCR product)  </TD><TD>1μl</TD></TR><TR><TD>Taq polymerase</TD><TD>1μl</TD></TR><TR><TD>ddH2O </TD><TD>40μl</TD></TR><TR><TD>Total volume</TD><TD>50μl</TD></TR></TBODY></TABLE>&lt; Phenol chloroform and EtOH precipitation of 2nd round PCR product &gt;
																Consult the protocol </DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseY721">2015.07.21</A></H4></DIV><DIV id="collapseY721" class="panel-collapse collapse "><DIV class="panel-body">
															Operator: Wan-Yun, Jin-Ting, Jin-Ye
															Goal:   1.	Transform PCR product(Far1Δ::KanMX) into FUS1-GFP strain   2.	Extraction of gDNA of yeast   3.	Check PCR for far1Δ::KanMX-FUS1-GFP  4.	Digestion of p426 Gal 
															Experimental steps: 
 															&lt; Transform PCR product into FUS1-GFP strain&gt; 
															Consult the experiment record &lt;2015.7.3 Experiment Record&gt;
															Because last time , negative control also had grown colony so we did transformation again. At the same time , we selected colony on selection plate to do PCR check.&lt; Extraction of gDNA of yeast&gt;
															Consult the protocol &lt; protocol of fast extraction of gDNA of yeast&gt;&lt; Check PCR for Far1Δ::KanMX- FUS-GFP &gt;
															1.PCR program<TABLE class="protocol-table"><TBODY><TR><TD>Step </TD><TD>Temperature </TD><TD>Time</TD></TR><TR><TD>Step1 </TD><TD>95℃ </TD><TD>5min</TD></TR><TR><TD>Step2 </TD><TD>95℃ </TD><TD>30s</TD></TR><TR><TD>Step3 </TD><TD>46℃</TD><TD>30s</TD></TR><TR><TD>Step4→step2 for 30 cycle </TD><TD>72℃ </TD><TD>90sec </TD></TR><TR><TD>Step5 </TD><TD>72℃ </TD><TD>5min </TD></TR><TR><TD>Step6 (hold on) </TD><TD>10℃ </TD><TD>1hr </TD></TR></TBODY></TABLE>
															2.PCR reagent<TABLE class="protocol-table"><TBODY><TR><TD>10x Dream Taq buffer </TD><TD>2.5μl</TD></TR><TR><TD>2.5mM dNTP</TD><TD>0.5μl</TD></TR><TR><TD>10mM primer(F)</TD><TD>0.5μl</TD></TR><TR><TD>10mM primer(R)</TD><TD>0.5μl </TD></TR><TR><TD>template (First round PCR product) </TD><TD>8μl</TD></TR><TR><TD>Taq polymerase</TD><TD>0.5μl</TD></TR><TR><TD>ddH2O </TD><TD>12.5μl</TD></TR><TR><TD>Total volume</TD><TD>25μl</TD></TR></TBODY></TABLE><TABLE class="protocol-table"><TBODY><TR><TD>far1Δ::KanMX-FUS-GFP </TD><TD>260/280 </TD><TD>260/230 </TD><TD>ng/μl </TD></TR><TR><TD>1 </TD><TD>1.76 </TD><TD>0.91 </TD><TD>111.7 </TD></TR><TR><TD>2 </TD><TD>2.05 </TD><TD>1.32 </TD><TD>158.2 </TD></TR><TR><TD>3 </TD><TD>2.02 </TD><TD>1.29 </TD><TD>120.4 </TD></TR><TR><TD>4 </TD><TD>1.98 </TD><TD>1.08 </TD><TD>89.9 </TD></TR><TR><TD>5 </TD><TD>2.05 </TD><TD>0.95 </TD><TD>66.9 </TD></TR><TR><TD>6 </TD><TD>2.07 </TD><TD>1.45 </TD><TD>182.6 </TD></TR><TR><TD>7 </TD><TD>1.86 </TD><TD>0.90 </TD><TD>95.4 </TD></TR><TR><TD>8 </TD><TD>2.08 </TD><TD>1.45 </TD><TD>166.4 </TD></TR><TR><TD>9 </TD><TD>2.18 </TD><TD>0.42 </TD><TD>22.5 </TD></TR><TR><TD>10 </TD><TD>2.07 </TD><TD>1.30 </TD><TD>102.4 </TD></TR></TBODY></TABLE>&lt; Digestion of p426 Gal &gt;<TABLE class="protocol-table"><TBODY><TR><TD> </TD><TD>Eco RI(μl) </TD><TD>Bam HI(μl) </TD><TD>Uncut(μl) </TD></TR><TR><TD>ddH2O </TD><TD>20.5 </TD><TD>20.5 </TD><TD>21.5 </TD></TR><TR><TD>10x NEB buf. #4 </TD><TD>2.5 </TD><TD>2.5 </TD><TD>2.5 </TD></TR><TR><TD>DNA(200ng) </TD><TD>1.5 </TD><TD>1.5 </TD><TD>1.5 </TD></TR><TR><TD>Enzyme </TD><TD>0.5 </TD><TD>0.5 </TD><TD>-- </TD></TR><TR><TD>total </TD><TD>25 </TD><TD>25 </TD><TD>25 </TD></TR></TBODY></TABLE>
															1.	Incubate at 37 for 1 hr.
															2.	Run the sample on the 1% agarose gel at 100 V for 30 min.(10 μl sample +2 μl loading dye)
															</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseY723">2015.07.23</A></H4></DIV><DIV id="collapseY723" class="panel-collapse collapse "><DIV class="panel-body">
															Operator: Wan-Yun, Jin-Ting, Jin-Ye
															Goal:   1.	Digestion of p426 Gal and rho-CXCR1  2.	Gel extraction of p426 Gal(vector)  3.	Clean up of rho-CXCR1(insert)
															Experimental steps: &lt; Digestion of p426 Gal and rho-CXCR1&gt;<TABLE class="protocol-table"><TBODY><TR><TD> </TD><TD>121 ng/μl </TD><TD>131 ng/μl </TD></TR><TR><TD>p426 Gal (2 μg) </TD><TD>16.5μl </TD><TD>15.2μl </TD></TR><TR><TD>10x NEB buf. 4 </TD><TD>5μl </TD><TD>5μl </TD></TR><TR><TD>Bam HI </TD><TD>2μl </TD><TD>2μl</TD></TR><TR><TD>Eco RI </TD><TD>2μl </TD><TD>2μl</TD></TR><TR><TD>ddH2O </TD><TD>24.5μl </TD><TD>25.8μl </TD></TR><TR><TD>Total </TD><TD>50μl </TD><TD>50 μl </TD></TR></TBODY></TABLE><TABLE class="protocol-table"><TBODY><TR><TD>Rho-CXCR1 </TD><TD>8(400 ng) </TD></TR><TR><TD>10x NEB buf. 4 </TD><TD>5μl </TD></TR><TR><TD>Bam HI </TD><TD>1μl </TD></TR><TR><TD>Eco RI </TD><TD>1μl</TD></TR><TR><TD>ddH2O </TD><TD>35μl </TD></TR><TR><TD>Total </TD><TD>50μl </TD></TR></TBODY></TABLE>&lt; Gel extraction of p426 Gal &gt;
															Consult the protocol &lt; protocol of gel extraction&gt;&lt; Clean up of rho-CXCR1&gt;
															Consult the protocol </DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseY724">2015.07.24</A></H4></DIV><DIV id="collapseY724" class="panel-collapse collapse "><DIV class="panel-body">
															Operator: Wan-Yun, Jin-Ting, Jin-Ye
															Goal:   1.	Ligation of the rho-CXCR1 and p426 Gal  2.	Transform the ligation product into the E.coli
															Experimental steps:&lt; Ligation of the rho-CXCR1 and p426 Gal&gt;<TABLE class="protocol-table"><TBODY><TR><TD> </TD><TD>1:3 </TD><TD>Vector only </TD></TR><TR><TD>Vector(11.1 ng/μl) </TD><TD>10μl </TD><TD>10μl </TD></TR><TR><TD>Insert(9.1 ng/μl) </TD><TD>7μl </TD><TD>0μl </TD></TR><TR><TD>Ligase </TD><TD>1μl </TD><TD>1μl </TD></TR><TR><TD>Ligation buff. </TD><TD>2μl </TD><TD>2μl </TD></TR><TR><TD>ddH2O </TD><TD>0μl </TD><TD>7μl </TD></TR><TR><TD>total </TD><TD>20μl </TD><TD>20μl </TD></TR></TBODY></TABLE>
															Incubate the ligation product at R.T. for 2 hr.&lt; Transform the ligation product into the competent cell&gt;
															Consult the protocol &lt; protocol of ligation and transformation of E.coli&gt;</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseY727">2015.07.27</A></H4></DIV><DIV id="collapseY727" class="panel-collapse collapse "><DIV class="panel-body">
															Operator: Wan-Yun, Jin-Ting
															Goal:   1.	Fast extraction of gDNA of Far1Δ::KanMX-FUS1-GFP  2.	Check PCR for far1Δ::KanMX-FUS-GFP
															Experimental steps: 
															&lt; Fast extraction of gDNA of far1Δ::KanMX-FUS-GFP &gt;
															Consult the protocol <PROTOCOL of="" fast="" extraction="" of="" gdna="" of="" yeast=""><TABLE class="protocol-table"><TBODY><TR><TD>Strains  </TD><TD>260/280 </TD><TD>260/230 </TD><TD>C(ng/μl)  </TD></TR><TR><TD>FAR1∆::KANMX  </TD><TD>1.85 </TD><TD>1.19 </TD><TD>177.3 </TD></TR><TR><TD>FAR1∆::KANMX  </TD><TD>1.83 </TD><TD>1.19 </TD><TD>163.8 </TD></TR><TR><TD>Positive control  </TD><TD>1.96 </TD><TD>1.61 </TD><TD>235.3 </TD></TR></TBODY></TABLE>&lt; Check PCR for Far1Δ::KanMX-FUS1-GFP&gt;
															1.PCR program<TABLE class="protocol-table"><TBODY><TR><TD>Step </TD><TD>Temperature </TD><TD>Time</TD></TR><TR><TD>Step1 </TD><TD>95℃ </TD><TD>5min</TD></TR><TR><TD>Step2 </TD><TD>95℃ </TD><TD>30s</TD></TR><TR><TD>Step3 </TD><TD>46℃</TD><TD>30s</TD></TR><TR><TD>Step4→step2 for 30 cycle </TD><TD>72℃ </TD><TD>90sec </TD></TR><TR><TD>Step5 </TD><TD>72℃ </TD><TD>5min </TD></TR><TR><TD>Step6 (hold on) </TD><TD>10℃ </TD><TD>1hr </TD></TR></TBODY></TABLE>
															2.PCR reagent<TABLE class="protocol-table"><TBODY><TR><TD> </TD><TD>FAR1∆::KANMX  </TD><TD>Positive control  </TD></TR><TR><TD>10x Dream Taq buffer </TD><TD>2.5μl</TD><TD>2.5μl</TD></TR><TR><TD>2.5mM dNTP</TD><TD>0.5μl</TD><TD>0.5μl</TD></TR><TR><TD>10mM primer(F)</TD><TD>0.5μl</TD><TD>0.5μl</TD></TR><TR><TD>10mM primer(R)</TD><TD>0.5μl </TD><TD>0.5μl </TD></TR><TR><TD>template (First round PCR product) </TD><TD>0.92μl</TD><TD>0.64μl</TD></TR><TR><TD>Taq polymerase</TD><TD>0.5μl</TD><TD>0.5μl</TD></TR><TR><TD>ddH2O </TD><TD>19.58μl</TD><TD>19.86μl</TD></TR><TR><TD>Total volume</TD><TD>25μl</TD><TD>25μl</TD></TR></TBODY></TABLE></PROTOCOL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseY728">2015.07.28</A></H4></DIV><DIV id="collapseY728" class="panel-collapse collapse "><DIV class="panel-body">
															Operator: Wan-Yun, Jin-Ting, Jin-Ye
															Goal:   1.Fast extraction of pGAL426 rho-CXCR1
															Experimental steps: &lt; Fast extraction of plasmid DNA &gt;  1.	Add 50 μl of STE buffer(100 mM NaCl, 10 mM Tris buffer, pH 7.0, 1 mM EDTA) into each new Eppendorf tube.  2.	Add 50 μl of Phenol chloroform and vortex vigorously for 30 sec.  3.	Centrifuge at 13,000 g for 5 min.  4.	Remove the 10 μl of supernatant to a new Eppendorf tube and run the sample on the 1 % agarose gel.</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseY729">2015.07.29</A></H4></DIV><DIV id="collapseY729" class="panel-collapse collapse "><DIV class="panel-body">
																Operator: Wan-Yun, Jin-Ting
																Goal:   1.	Miniprep of p426 Gal-rho-CXCR1  2.	Enzyme digestion to check p426 Gal-rho-CXCR1 
																Experimental steps: &lt; Miniprep of p426 Gal-rho-CXCR1&gt;
																Consult the protocol <PROTOCOL of="" miniprep="" plamid="">&lt; Enzyme digestion for the p426 Gal-rho-CXCR1 check &gt;
																1.	Enzyme digestion<TABLE class="protocol-table"><TBODY><TR><TD>p426 Gal-rho-CXCR1 </TD><TD>#7,9 </TD><TD>+,#8,12,16 </TD></TR><TR><TD>10x NEB buff.4 </TD><TD>2.5μl </TD><TD>2.5μl </TD></TR><TR><TD>Eco RI </TD><TD>0.5μl </TD><TD>0.5μl </TD></TR><TR><TD>Bam HI </TD><TD>0.5μl </TD><TD>0.5μl </TD></TR><TR><TD>DNA </TD><TD>2μl </TD><TD>1μl </TD></TR><TR><TD>ddH2O </TD><TD>19.5μl </TD><TD>20.5μl </TD></TR><TR><TD>total </TD><TD>25μl </TD><TD>25μl </TD></TR></TBODY></TABLE>
																2.	Loading 10 μl sample and 2 μl DNA loading dye into the 1 % agarose gel.
																</PROTOCOL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseY730">2015.07.30</A></H4></DIV><DIV id="collapseY730" class="panel-collapse collapse "><DIV class="panel-body">
																Operator: Wan-Yun, Jin-Ting, Jin-Ye
																Goal:   1.	Transform pSB1C3 BBa_J04450 into the competent cell  2.	Transform the plasmid (pRS405) into the competent cell
																Experimental steps: &lt; Transform pSB1C3 BBa_J04450 into the competent cell&gt;
																Consult the protocol <PROTOCOL of="" ligation="" and="" transformation="" of="" e.coli="">
																Selection plate: LB+ Cam plate&lt; Transform the plasmids (pRS405) into the competent cell&gt;
																Selection plate: LB+ Amp plate</PROTOCOL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseY83">2015.08.03</A></H4></DIV><DIV id="collapseY83" class="panel-collapse collapse "><DIV class="panel-body">
																Operator: Wan-Yun, Jin-Ting, Jin-Ye
																Goal:   1. Miniprep of pRS405 and run the 1% agarose gel  2. Digestion of pRS405 with Xho I, Bam HI and Eco RI  3.	Digestion of p426 Gal with Xho I  4.	Inoculate the bacteria with BBa_J04450 into the LB+ chloramphanicle broth
																Experiment steps:&lt; Miniprep of pRS405 and run the 1% agarose gel &gt;
																Consult the protocol &lt; protocol of miniprep plamid&gt;&lt; Digestion of pRS405 with Xho I, Bam HI and Eco RI &gt;<TABLE class="protocol-table"><TBODY><TR><TD> </TD><TD>uncut </TD><TD>Xho I </TD><TD>Bam HI </TD><TD>Eco RI </TD></TR><TR><TD>pRS405 (V105○1) </TD><TD>0.5 </TD><TD>0.5 </TD><TD>0.5 </TD><TD>0.5 </TD></TR><TR><TD>10x NEB buff.4 </TD><TD>2.5 </TD><TD>2.5 </TD><TD>2.5 </TD><TD>2.5 </TD></TR><TR><TD>Enzyme</TD><TD>- </TD><TD>0.5 </TD><TD>0.5</TD><TD>0.5  </TD></TR><TR><TD>ddH2O</TD><TD>22.5 </TD><TD>21.5 </TD><TD>21.5 </TD><TD>21.5</TD></TR><TR><TD>total </TD><TD>25 μl </TD><TD>25 μl</TD><TD>25 μl </TD><TD>25 μl </TD></TR></TBODY></TABLE>
																Reaction condition: 37℃, 1hr &lt; Digestion of p426 Gal with Xho I &gt;<TABLE class="protocol-table"><TBODY><TR><TD> .</TD><TD>uncut </TD><TD>Xho I </TD></TR><TR><TD>p426 Gal (131 ng/μl)</TD><TD>0.5</TD><TD>0.5</TD></TR><TR><TD>10x NEB buff.4</TD><TD>2.5 </TD><TD>2.5</TD></TR><TR><TD> Enzyme</TD><TD>- </TD><TD>0.5 </TD></TR><TR><TD>ddH2O</TD><TD>22.5</TD><TD>21.5</TD></TR><TR><TD>total</TD><TD>25 μl </TD><TD>25 μl </TD></TR></TBODY></TABLE>
																Reaction condition: 37℃, 1hr </DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseY84">2015.08.04</A></H4></DIV><DIV id="collapseY84" class="panel-collapse collapse "><DIV class="panel-body">
																Operator: Wan-Yun, Jin-Ting
																Goal:   1.	1st PCR of rho-CXCR1  2.	Electrophoresis to check the PCR product  3.	Miniprep of pSB1C3 BBa_J04450
																Experiment steps:&lt;1st PCR of rho-CXCR1&gt;
																1.	PCR program
																<TABLE class="protocol-table"><TBODY><TR><TD>Step </TD><TD>Temp. </TD><TD>Time </TD></TR><TR><TD>Step1 </TD><TD>95℃ </TD><TD>5 min </TD></TR><TR><TD>Step2 </TD><TD>95℃ </TD><TD>30 sec </TD></TR><TR><TD>Step3</TD><TD>40,44,48℃ </TD><TD>30 sec</TD></TR><TR><TD>Step4→Step2 for 30 cycle  </TD><TD>72℃</TD><TD>80sec  </TD></TR><TR><TD>Step5</TD><TD>72℃ </TD><TD>5 min</TD></TR><TR><TD>Step6(Hold on) </TD><TD>10℃</TD><TD>1 hr</TD></TR></TBODY></TABLE>							2.	PCR reagent
																<TABLE class="protocol-table"><TBODY><TR><TD> 10x Dream Taq buffer </TD><TD>2.5μl  </TD></TR><TR><TD>2.5mMdNTP  </TD><TD>0.5μl </TD></TR><TR><TD>10μMprimer(F’) </TD><TD>0.5μl </TD></TR><TR><TD> 10μMprimer(R’) </TD><TD>0.5 μl </TD></TR><TR><TD>Rho-CXCR1</TD><TD>3μl </TD></TR><TR><TD>Taq polymerase </TD><TD>0.5 μl </TD></TR><TR><TD>ddH2O</TD><TD>17.5μl </TD></TR><TR><TD>Total volume</TD><TD>25 μl </TD></TR></TBODY></TABLE>&lt; Electrophoresis to check the PCR product&gt;&lt; Electrophoresis to check second round-PCR product&gt;
																Material:   DNA marker: 100bp ladder 8μl   DNA sample:2μlPCR product + 1μl 6x loading buffer
																Condition:   0.5xTBE buffer 100V 
																Time:  30min
																Result:
																Conclusion: It expected length is 1200bp and it worked.&lt; Miniprep of pSB1C3 BBa_J04450&gt;
																Consult the protocol </DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseY810">2015.08.10</A></H4></DIV><DIV id="collapseY810" class="panel-collapse collapse "><DIV class="panel-body">
																Operator: Wan-Yun, Jin-Ting, Jin-Ye
																Goal:   1.	Electrophoresis to check p426Gal-rho-CXCR1 digestion product again  2. Do digestion of Lucy-rho-CXCR1 (insert) and p426Gal (vector)  3. Clean up digestion product of Lucy-rho-CXCR1  4. Gel extraction of digestion product of p426Gal  5. Ligation insert and vector  6. Transform the construct into competent cell
																Experiment steps:&lt; Electrophoresis to check p426 Gal-rho-CXCR1 digestion product again&gt;
																Material:   DNA marker: 1kb ladder 6ul   DNA sample:2ul p426Gal-rho-CXCR1 digestion product(BamHI and XhoI) + 1ul 6x loading buffer
																Condition:   0.5xTBE buffer 100V 
																Time:   30min
																Result:
																Note: pGAL426 rho-CXCR1 enzyme digestion product with BamHI and XhoI<TABLE class="protocol-table"><TBODY><TR><TD> </TD><TD>Lucy-rho-CXCR1 (insert) </TD><TD>p426Gal (vector) </TD><TD>p426Gal (vector) </TD></TR><TR><TD>10x NEB buffer#4 </TD><TD>5μl	 </TD><TD>5μl	 </TD><TD>5μl </TD></TR><TR><TD>BamHI </TD><TD>1μl </TD><TD>1μl </TD><TD>1μl	 </TD></TR><TR><TD>XhoI </TD><TD>1μl </TD><TD>1μl </TD><TD>1μl	 </TD></TR><TR><TD>DNA	 </TD><TD>25μl (400ng)	 </TD><TD>7μl (2ug) </TD><TD>7μl	 </TD></TR><TR><TD>ddH2O </TD><TD>18μl	 </TD><TD>34μl	 </TD><TD>34μl </TD></TR><TR><TD>Total volume </TD><TD>50μl	 </TD><TD>50μl	 </TD><TD>50μl	 </TD></TR></TBODY></TABLE>
																Reaction condition : 37℃ for 1hr&lt; Clean up digestion product of Lucy-rho-CXCR1&gt;
																Consult the protocol <PROTOCOL of="" clean="" up="" after="" digestion=""><TABLE class="protocol-table"><TBODY><TR><TD>Conc	 </TD><TD>260/280	 </TD><TD>260/230	 </TD></TR><TR><TD>10.0ng/ul	 </TD><TD>1.95	 </TD><TD>0.94	 </TD></TR></TBODY></TABLE>&lt; Gel extraction of digestion product of p426Gal&gt;
																Consult the protocol <PROTOCOL of="" gel="" extraction=""><TABLE class="protocol-table"><TBODY><TR><TD>Conc.	 </TD><TD>260/280 </TD><TD>260/230 </TD></TR><TR><TD>11.2ng/ul </TD><TD>1.75 </TD><TD>0.20 </TD></TR></TBODY></TABLE>&lt; Ligation insert and vector&gt;
																Vector:insert=1:3<TABLE class="protocol-table"><TBODY><TR><TD> </TD><TD>Test tube </TD><TD>Vector only	 </TD></TR><TR><TD>Vector(11.2ng/ul) </TD><TD>9μl </TD><TD>9μl </TD></TR><TR><TD>Insert(10.0ng/ul) </TD><TD>6μl </TD><TD>0μl </TD></TR><TR><TD>Ligase </TD><TD>1μl </TD><TD>1μl </TD></TR><TR><TD>10xLigation buffer </TD><TD>2μl </TD><TD>2μl </TD></TR><TR><TD>ddH2O </TD><TD>2μl </TD><TD>8μl </TD></TR><TR><TD>Total volume </TD><TD>20μl </TD><TD>20μl </TD></TR></TBODY></TABLE>
																Reaction condition: Room temperature for 2-4hr&lt; Transform the construct into competent cell&gt;
																Selection plate: LB+Cam</PROTOCOL></PROTOCOL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseY811">2015.08.11</A></H4></DIV><DIV id="collapseY811" class="panel-collapse collapse "><DIV class="panel-body">
																Operator: Wan-Yun, Jin-Ting, Jin-Ye
																Goal:   1.	Obeserve colony growth to confirm transformation result  2.	PCR Gpa1-Leu2  3.	Electrophoresis to check PCR product  4.	Second PCR round of Gpa1-Leu2 with first product  5.	Extraction of DNA with Phenol/Chloroform
																Experiment steps:&lt; Obeserve colony growth to confirm transformation result&gt;
 																Select 12 colonies and incubate in LB broth(with Amp) overnight,37℃&lt; PCR Gpa1-Leu2&gt;
																1.PCR program
																<TABLE class="protocol-table"><TBODY><TR><TD>Step </TD><TD>Temperature </TD><TD>Time</TD></TR><TR><TD>Step1 </TD><TD>95℃ </TD><TD>5min</TD></TR><TR><TD>Step2 </TD><TD>95℃ </TD><TD>30s</TD></TR><TR><TD>Step3 </TD><TD>48℃,53℃,59℃,62℃ </TD><TD>90s</TD></TR><TR><TD>Step4→step2 for 30 cycle </TD><TD>72℃ </TD><TD>5min </TD></TR><TR><TD>Step5 </TD><TD>72℃ </TD><TD>5min </TD></TR><TR><TD>Step6 (hold on) </TD><TD>10℃ </TD><TD>1hr </TD></TR></TBODY></TABLE>
																2.PCR reagent 
																<TABLE class="protocol-table"><TBODY><TR><TD>10x Dream Taq buffer </TD><TD>2.5μl</TD></TR><TR><TD>2.5mM dNTP</TD><TD>0.5μl</TD></TR><TR><TD>10mM primer(F)</TD><TD>0.5μl</TD></TR><TR><TD>10mM primer(R)</TD><TD>0.5μl </TD></TR><TR><TD>Template(pRS405,150ng) </TD><TD>0.55μl</TD></TR><TR><TD>Taq polymerase</TD><TD>0.5μl</TD></TR><TR><TD>ddH2O </TD><TD>20μl</TD></TR><TR><TD>Total volume</TD><TD>25μl</TD></TR></TBODY></TABLE><ELECTROPHORESIS to="" check="" pcr="" product="">&lt; Electrophoresis to check second round-PCR product (50μl)&gt;
																Material:   DNA marker: 1kb ladder 6μl   DNA sample:2μl PCR product + 1μl 6x loading buffer
																Condition:   0.5xTBE buffer 100V 
																Time:  30min
																Result:
																Conclution: Its expected length is 1502bp and it worked.&lt; Second PCR round of Gpa1-Leu2 with first product&gt;<TABLE class="protocol-table"><TBODY><TR><TD>Step </TD><TD>Temperature </TD><TD>Time</TD></TR><TR><TD>Step1 </TD><TD>95℃ </TD><TD>5min</TD></TR><TR><TD>Step2 </TD><TD>95℃ </TD><TD>30s</TD></TR><TR><TD>Step3 </TD><TD>53℃ </TD><TD>30s</TD></TR><TR><TD>Step4→step2 for 30 cycle </TD><TD>72℃ </TD><TD>5min </TD></TR><TR><TD>Step5 </TD><TD>72℃ </TD><TD>5min </TD></TR><TR><TD>Step6 (hold on) </TD><TD>10℃ </TD><TD>1hr </TD></TR></TBODY></TABLE>
																2.	PCR reagent 
																<TABLE class="protocol-table"><TBODY><TR><TD>10x Dream Taq buffer </TD><TD>5μl</TD></TR><TR><TD>2.5mM dNTP</TD><TD>1μl</TD></TR><TR><TD>10mM primer(F)</TD><TD>1μl</TD></TR><TR><TD>10mM primer(R)</TD><TD>1μl </TD></TR><TR><TD>template (First round PCR product) </TD><TD>1μl</TD></TR><TR><TD>Taq polymerase</TD><TD>1μl</TD></TR><TR><TD>ddH2O </TD><TD>40μl</TD></TR><TR><TD>Total volume</TD><TD>50μl</TD></TR></TBODY></TABLE>&lt; Extraction of DNA with Phenol/Chloroform&gt;
																Consult the protocol </ELECTROPHORESIS></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseY812">2015.08.12</A></H4></DIV><DIV id="collapseY812" class="panel-collapse collapse "><DIV class="panel-body">
																Operator: Wan-Yun, Jin-Ting, Jin-Ye
																Goal:     1. Miniprep of p426Gal-Lucy-rho-CXCR1  2. Electrophoresis to confirm whether transformed successfully or not  3. Digestion of p426Gal-Lucy-rho-CXCR1  4. Electrophoresis to confirm which colony been transformed successlly  5. Transform Gpa1 PCR product into FAR1::KANMX-FUS1-GFP strain&lt; Miniprep of p426Gal-Lucy-rho-CXCR1&gt;
																Consult the protocol <PROTOCOL of="" miniprep="" plamid="">&lt; Electrophoresis to confirm whether transformed successfully or not&gt;
																Material: DNA marker: 1kb ladder 6ul 
																DNA sample: plasmid 2ul + 3ul 6x loading buffer
																Condition: 0.5xTBE buffer 100V 
																Time: 40min
																Result:
																Note: This gel result shows size difference of plasmid transformed or untransformed.&lt; Digestion of p426Gal-Lucy-rho-CXCR1&gt;<TABLE class="protocol-table"><TBODY><TR><TD>DNA(p426Gal-Lucy-rho-CXCR1 #2 #3 #7) </TD><TD>1μl </TD></TR><TR><TD>BamHI </TD><TD>0.5μl </TD></TR><TR><TD>XhoI </TD><TD>&gt;0.5μl </TD></TR><TR><TD>10x NEB buffer #4	 </TD><TD>2.5μl </TD></TR><TR><TD>ddH2O </TD><TD>20.5μl </TD></TR><TR><TD>Total volume </TD><TD>25μl </TD></TR></TBODY></TABLE>
																Condition:  incubate at 37℃ for 1hr&lt; Electrophoresis to confirm which colony been transformed successlly&gt;
																Material: DNA marker: 1kb ladder 6μl
																DNA sample: plasmid 5μl + 3μl 6x loading buffer
																Condition: 0.5xTBE buffer 100V 
																Time: 30min
																Result:
																Note: We use BamHI and XhoI to check whether the construct was built successfully.&lt; Transform Gpa1 PCR product into FAR1::KANMX-FUS1-GFP strain&gt;
																Consult the protocol <PROTOCOL of="" yeast="" transformation="">
																Yeast strain: FAR1::KANMX-FUS1-GFP
																Selection plate : YPD+G418 plate</PROTOCOL></PROTOCOL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseY813">2015.08.13</A></H4></DIV><DIV id="collapseY813" class="panel-collapse collapse "><DIV class="panel-body">
																Operator: Wan-Yun, Jin-Ting, Jin-Ye
																Goal:   1. Digestion of pSB1C3 BBa_J04450   2. Electrophoresis to check digestion product  3. Digestion of pSB1C3 BBa_J04450 and rho-CXCR1  4. Gel extraction of vector and clean up insert  5. Ligation and transformation
																Experiment steps:&lt; Digestion of pSB1C3 BBa_J04450&gt;<TABLE class="protocol-table"><TBODY><TR><TD> </TD><TD>Both </TD><TD>Only XbaI </TD><TD>Only SpeI </TD><TD>Uncut </TD></TR><TR><TD>pSB1C3 BBa_J04450#3 </TD><TD>1μl</TD><TD>1μl</TD><TD>1μl </TD><TD>1μl</TD></TR><TR><TD>10x NEB buffer #4 </TD><TD>2.5μl</TD><TD>2.5μl</TD><TD>2.5μl</TD><TD>0μl</TD></TR><TR><TD>XbaI </TD><TD>0.3μl</TD><TD>0.3μl</TD><TD>0μl</TD><TD>0μl</TD></TR><TR><TD>SpeI </TD><TD>0.3μl</TD><TD>0μl</TD><TD>0.3μl</TD><TD>0μl</TD></TR><TR><TD>ddH2O </TD><TD>21μl</TD><TD>21.2μl</TD><TD>21.2μl</TD><TD>24μl </TD></TR><TR><TD>Total volume </TD><TD>25μl </TD><TD>25μl </TD><TD>25μl </TD><TD>25μl </TD></TR></TBODY></TABLE>
																Condition:  at 37℃ fpr 1hr&lt; Electrophoresis to check digestion product&gt;
																Material: DNA marker: 1kb ladder 6μl 
																DNA sample: 10μl + 3μl 6x loading buffer
																Condition: 0.5xTBE buffer 100V 
																Time: 30min
																Result:
																Note: It was up to our expection.&lt; Digestion of pSB1C3 BBa_J04450 and rho-CXCR1&gt;<TABLE class="protocol-table"><TBODY><TR><TD> </TD><TD>Rho-CXCR1(400ng) </TD><TD>BBa_J04450(2μg)x2 tubes </TD></TR><TR><TD>DNA </TD><TD>8μl</TD><TD>9.6μl </TD></TR><TR><TD>10x NEB buffer#4</TD><TD>5μl </TD><TD>5μl</TD></TR><TR><TD>XbaI </TD><TD> 1μl</TD><TD>		2μl </TD></TR><TR><TD>SpeI </TD><TD>1μl </TD><TD>		2μl </TD></TR><TR><TD>ddH2O </TD><TD>35μl </TD><TD>		31.5μl </TD></TR><TR><TD>Total volume </TD><TD>50μl </TD><TD>		50μl </TD></TR></TBODY></TABLE>
																Condition: at 37℃ for 1hr&lt; Gel extraction of vector&gt;
																Consult the protocol &lt; protocol of gel extraction&gt;<TABLE class="protocol-table"><TBODY><TR><TD>Conc.	 </TD><TD>260/280	 </TD><TD>260/230 </TD></TR><TR><TD>6.3ng/μl </TD><TD>	1.62 </TD><TD>	0.17 </TD></TR></TBODY></TABLE>&lt; Clean up the insert&gt;
																Consult the protocol &lt; protocol of clean up after digestion&gt;<TABLE class="protocol-table"><TBODY><TR><TD>Conc. </TD><TD>260/280 </TD><TD>		260/230 </TD></TR><TR><TD>4.9ng/μl </TD><TD>1.48 </TD><TD>		0.68 </TD></TR></TBODY></TABLE>&lt; Ligation and transformation&gt;<TABLE class="protocol-table"><TBODY><TR><TD> </TD><TD>Construct </TD><TD>		Vector only </TD></TR><TR><TD>Veator </TD><TD>6μl </TD><TD>		6μl </TD></TR><TR><TD>Insert </TD><TD>13μl </TD><TD>		0μl </TD></TR><TR><TD>10x ligation buffer </TD><TD>2μl </TD><TD>	2μl </TD></TR><TR><TD>Ligase </TD><TD>1μl </TD><TD>		1μl </TD></TR><TR><TD>ddH2O </TD><TD>0.5μl </TD><TD>		11μl </TD></TR><TR><TD>Total volume </TD><TD>22μl </TD><TD>		20μl </TD></TR></TBODY></TABLE>
																Condition: room temperature 20℃ for 2hr&lt; Transform the construct into competent cell&gt;
																Consult the protocol <PROTOCOL of="" ligation="" and="" transformation="" of="" e.coli="">
																Selection plate: LB+CAM plates </PROTOCOL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseY817">2015.08.17</A></H4></DIV><DIV id="collapseY817" class="panel-collapse collapse "><DIV class="panel-body">
																Operator: Wan-Yun, Jin-Ye
																Goal:   1.Fast and dirty extraction of gDNA of Gpa1 chimera yeast  2. Do PCR to check location of Gpa1 chimera   3. Electrophoresis to check PCR product  4. Do colony PCR to check location of insert (rho-CXCR1 and Lucy-rho-CXCR1)  5. Electrophoresis to check colony PCR product
																Experiment steps:&lt; Fast and dirty extraction of gDNA of Gpa1 chimera yeast&gt;
																Consult the protocol <PROTOCOL of="" fast="" extraction="" of="" gdna="" of="" yeast=""><TABLE class="protocol-table"><TBODY><TR><TD>Strain name </TD><TD>Conc.(ng/μl) </TD><TD>260/280 </TD><TD>260/230 </TD></TR><TR><TD>Gpa1 chimera </TD><TD>277.6 </TD><TD>2.00 </TD><TD>1.49 </TD></TR><TR><TD>Far1△::KANMX-FUS1-GFP </TD><TD>303.6 </TD><TD>2.08 </TD><TD>1.71 </TD></TR></TBODY></TABLE><DO pcr="" to="" check="" location="" of="" gpa1="" chimera="">
																1.PCR program<TABLE class="protocol-table"><TBODY><TR><TD>Step </TD><TD>Temperature </TD><TD>Time</TD></TR><TR><TD>Step1 </TD><TD>95℃ </TD><TD>5min</TD></TR><TR><TD>Step2 </TD><TD>95℃ </TD><TD>30s</TD></TR><TR><TD>Step3 </TD><TD>48℃,52℃,55℃,59℃</TD><TD>30s</TD></TR><TR><TD>Step4→step2 for 30 cycle </TD><TD>72℃ </TD><TD>50sec </TD></TR><TR><TD>Step5 </TD><TD>72℃ </TD><TD>5min </TD></TR><TR><TD>Step6 (hold on) </TD><TD>10℃ </TD><TD>1hr </TD></TR></TBODY></TABLE>
																2.PCR reagent <TABLE class="protocol-table"><TBODY><TR><TD> </TD><TD>Gpa1 chimera </TD><TD>FAR1△::KANMX-FUS1-GFP  </TD></TR><TR><TD>10x Dream Taq buffer </TD><TD>2.5μl</TD><TD>2.5μl</TD></TR><TR><TD>2.5mM dNTP</TD><TD>0.5μl</TD><TD>0.5μl</TD></TR><TR><TD>10mM primer(F)</TD><TD>0.5μl</TD><TD>0.5μl</TD></TR><TR><TD>10mM primer(R)</TD><TD>0.5μl </TD><TD>0.5μl </TD></TR><TR><TD>Template(150ng) </TD><TD>0.55μl</TD><TD>0.5μl</TD></TR><TR><TD>Taq polymerase</TD><TD>0.5μl</TD><TD>0.5μl</TD></TR><TR><TD>ddH2O </TD><TD>20μl</TD><TD>20μl</TD></TR><TR><TD>Total volume</TD><TD>25μl</TD><TD>25μl</TD></TR></TBODY></TABLE>&lt; Electrophoresis to check PCR product&gt;
																Electrophoresis
																Material: DNA marker: 1kb ladder 6μl 
																DNA sample: 2μl + 3μl 6x loading buffer
																Condition: 0.5xTBE buffer 100V 
																Time: 30min
																Result:
																Note: Gpa1 chimera PCR check with different annealing Tm.&lt; Do colony PCR to check location of insert (rho-CXCR1 and Lucy-rho-CXCR1)&gt;<TABLE class="protocol-table"><TBODY><TR><TD> </TD><TD>pSB1C3 rho-CXCR1 </TD><TD>pSB1C3 Lucy-rho-CXCR1 </TD><TD>pSB1C3 BBa_J04450#3 </TD></TR><TR><TD>DNA template </TD><TD>Colonyx7 </TD><TD>Colonyx7 </TD><TD>0.7 </TD></TR><TR><TD>10μM VF2 primer </TD><TD>0.5μl </TD><TD>0.5μl0.5μl </TD><TD>0.5μl </TD></TR><TR><TD>10μM VR primer </TD><TD>0.5μl </TD><TD>0.5μl </TD><TD>0.5μl </TD></TR><TR><TD>2x Taq mix </TD><TD>12.5μl </TD><TD>12.5μl </TD><TD>12.5μl </TD></TR><TR><TD>ddH2O </TD><TD>11.5μl </TD><TD>11.5μl </TD><TD>10.8μl </TD></TR><TR><TD>Total volume </TD><TD>25μl </TD><TD>25μl </TD><TD>25μl </TD></TR></TBODY></TABLE>&lt; Electrophoresis to check colony PCR product&gt;
																Material: DNA marker: 6μl 1kb ladder 
																DNA sample: 2μl + 4μl 6x loading buffer
																Condition: 0.5xTBE buffer 100V 
																Time: 30min
																Result:
																Note: The middle lane was positive control---pSB1C3 BBa_J04450 and other lanes were loaded with construct built. However , the gel result was out of our expection and we assumed that they were self-ligation.</DO></PROTOCOL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseY818">2015.08.18</A></H4></DIV><DIV id="collapseY818" class="panel-collapse collapse "><DIV class="panel-body">
																Operator: Wan-Yun, Jin-Ting, Jin-Ye
																Goal:   1.Induction of three yeast strains with α factor  2.Observe GFP under microscope.&lt; Induction of three yeast strains with α factor&gt;
																Consult the protocol <PROTOCOL of="" induction="" with="" α="" factor="">&lt; Observe GFP under microscope&gt;	
															</PROTOCOL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseY825">2015.08.25</A></H4></DIV><DIV id="collapseY825" class="panel-collapse collapse "><DIV class="panel-body">
																Operator: Jin-Ting, Jin-Ye
																Goal:   1.Transform pGal426 Lucy-rho-CXCR1 into strain FAR1△::KANMX-FUS1-GFP Gpa1 chimera
																Experiment step:&lt; Transform pGal426 Lucy-rho-CXCR1 into strain FAR1△::KANMX-FUS1-GFP Gpa1 chimera&gt;
																Consult the protocol <PROTOCOL of="" transformation="" of="" plasmid="" into="" yeast="">
																Selection plate: Ura-+Leu- +G418 plate</PROTOCOL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseY827">2015.08.27</A></H4></DIV><DIV id="collapseY827" class="panel-collapse collapse "><DIV class="panel-body">
																Operator: Jin-Ting, Jin-Ye
																Goal:   1.Indution of yeast with αfactor
																Experiment record: <INDUTION of="" yeast="" with="" αfactor="">
																Consult protocol <PROTOCOL of="" induction="" with="" α="" factor="">
																Strain:
																positive control: YGR254W/YJR009C/YMR186W/YPR035W/YAL038W
																Negative control: wild type
																Test strain: FUS1-GFP</PROTOCOL></INDUTION></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseY828">2015.08.28~08.29</A></H4></DIV><DIV id="collapseY828" class="panel-collapse collapse "><DIV class="panel-body">
																Operator: Wan-Yun, Jin-Ting, Jin-Ye
																Goal:   1.Use galactose to induce yeast experess CXCR1
																Experiment record:
																Consult the protocol<PROTOCOL of="" induction="" of="" yeast="" with="" galactose="">
																Strain: FAR1△::KANMX-FUS1-GFP Gpa1 chimera-pGAL426 rho-CXCR1
															</PROTOCOL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseY831">2015.08.31</A></H4></DIV><DIV id="collapseY831" class="panel-collapse collapse "><DIV class="panel-body">
																Operator: Wan-Yun, Jin-Ting, Jin-Ye
																Goal:   1.	Protein extraction of induced yeast
																Experiment steps:
																Consult the protocol <PROTOCOL of="" protein="" extraction="" of="" yeast="">
																Protein sample : cell culture with different induction time(3h,4h,5h,6h,7h,8h)	
															</PROTOCOL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseY92">2015.09.02</A></H4></DIV><DIV id="collapseY92" class="panel-collapse collapse "><DIV class="panel-body">
																Operator: Jin-Ting, Jin-Ye
																Goal:   1.SDS-PAGE and Western blot
																Experiment steps:
																Consult the protocol <PROTOCOL of="" sds-page="" and="" western="" blot="">
																Protein sample : product of protein extraction last week
																Positive control : HEP G2 cell lysate （from human）
															</PROTOCOL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseY93">2015.09.03~09.04</A></H4></DIV><DIV id="collapseY93" class="panel-collapse collapse "><DIV class="panel-body">
																Operator: Wan-Yun, Jin-Ting, Jin-Ye
																Goal:   1. Use galactose to induce yeast express Lucy-rho-CXCR1 and rho-CXCR1  2. Treat yeast with IL-8
																Experiment steps:&lt; Use galactose to induce yeast express Lucy-rho-CXCR1 and rho-CXCR1&gt;
																Consult the protocol <PROTOCOL of="" induction="" of="" yeast="" with="" galactose="">
																Test strain:  ①Far1△::KANMX-FUS1-GFP Gpa1 chimera-pGAL426 Lucy-rho-CXCR1  ②Far1△::KANMX-FUS1-GFP Gpa1 chimera-pGAL426 rho-CXCR1
																Medium used: YEP(Raf) and YEP(Raf/Gal)&lt; Treat yeast with IL-8&gt;
																1.	Collect 100μl yeast with galactose induction in 3hr. 
																2.	Add 8μl IL-8(10μlM) to cell culture.
																3.	Take 10μl of total to observer under microscope.
															</PROTOCOL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseY97">2015.09.07~09.08</A></H4></DIV><DIV id="collapseY97" class="panel-collapse collapse "><DIV class="panel-body">
																Operator: Wan-Yun, Jin-Ting, Jin-Ye
																Goal:   1.Protein extraction of yeast with galactose induced  2.Do SDS-PAGE and Western Blot
																Experiment steps:&lt; Protein extraction of yeast with galactose induced&gt;
																Consult the protocol &lt; protocol of protein extraction of yeast&gt;
																Yeast strain:   ①Far1△::KANMX-FUS1-GFP Gpa1 chimera-pGAL426 Lucy-rho-CXCR1  ②Far1△::KANMX-FUS1-GFP Gpa1 chimera-pGAL426 rho-CXCR1
																Sample : yeast with galactose induced by 0hr, 1hr, 2hr, 3hr, 5hr, 8hr&lt; Do SDS-PAGE and Western Blot&gt;
																Consult the protocol &lt; protocol of SDS-PAGE and Western blot&gt;
																Negative control: yeast with galactose induced by 0hr
																Positive control: HEPG2 cell lysate</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseY99">2015.09.09</A></H4></DIV><DIV id="collapseY99" class="panel-collapse collapse "><DIV class="panel-body">
																Operator: Jin-Ting, Jin-Ye
																Goal:   1.Do galactose induction of yeast   2.Treat yeast with IL-8
																Experiment steps:
																Consult the protocol <PROTOCOL of="" induction="" of="" yeast="" with="" galactose="">
																Strain used:   ①Far1△::KANMX-FUS1-GFP Gpa1 chimera  ②Far1△::KANMX-FUS1-GFP Gpa1chimera pGAL426 rho-CXCR1  ③Far1△::KANMX-FUS1-GFP Gpa1 chimera pGAL 426 Lucy-rho-CXCR1
																Medium used: SCM(Ura-)+Raf and SCM(Ura-)+Raf/Gal
																Negative control: Far1△::KANMX-FUS1-GFP Gpa1 chimera&lt; Treat yeast with IL-8&gt;
																1.	Collect 100μl yeast culture with galactose induced by 3hr.
																2.	Add 8μl IL-8 (10μM) to cell culture.
																3.	Take 10μl of total to observe under microscope.
																Positive control: YGR254W
																Negative control: Wild type yeast
																Positive control 2: FAR1△::KANMX-FUS1-GFP with α factor induced 
															</PROTOCOL></DIV></DIV></DIV></DIV></DIV><DIV class="single-blog blog-details two-column " id="T"><H2>Toehold Switch As RNA Sensor</H2></DIV><DIV id="accordion-container"><DIV class="panel-group" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT620">2015.06.20~2015.06.25</A></H4></DIV><DIV id="collapseT620" class="panel-collapse collapse "><DIV class="panel-body">
																A, IL8 ; B, IL1β ; C, DUSP1 (dual specificity phosphatase 1) ; 
																D, SAT (spermidine/spermine N1-acetyltransferase EST
																We use the website RNA structure to predict the secondary structure of these four toehold switches whether would have hairpin structures we want. Luckily, all of its results are match our expectation.</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT625">2015.06.25</A></H4></DIV><DIV id="collapseT625" class="panel-collapse collapse "><DIV class="panel-body">
																 Sent our designed sequences to IDT.
															</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT714">2015.07.01~2015.07.14</A></H4></DIV><DIV id="collapseT714" class="panel-collapse collapse "><DIV class="panel-body">
																Amplify parts DNA we need.  
															</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT715">2015.07.15</A></H4></DIV><DIV id="collapseT715" class="panel-collapse collapse "><DIV class="panel-body">
																Receive IDT DNA.
															</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT727">2015.07.27</A></H4></DIV><DIV id="collapseT727" class="panel-collapse collapse "><DIV class="panel-body">
																Transform BBa_I712019.
															</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT728">2015.07.28</A></H4></DIV><DIV id="collapseT728" class="panel-collapse collapse "><DIV class="panel-body">
																Small scale incubation.
															</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT729">2015.07.29</A></H4></DIV><DIV id="collapseT729" class="panel-collapse collapse "><DIV class="panel-body">
																  Mini prep BBa_I712019 (Luciferase).
                                                                  Concentration=186.2 (ng/ μl)
															</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT730">2015.07.30</A></H4></DIV><DIV id="collapseT730" class="panel-collapse collapse "><DIV class="panel-body">
																  Digest 4 toehold switches with EcoRI and XbaI and 4 trigger DNAs with XbaI and SpeI.
                                                                  <TOEHOLD switches=""><TABLE class="protocol-table"><TBODY><TR><TD>Plasmid DNA (25 ng/ μl)</TD><TD>5 μl</TD></TR><TR><TD>NEB #2</TD><TD>1 μl</TD></TR><TR><TD>EcoRI (20 U/μl)</TD><TD>0.2 μl</TD></TR><TR><TD>XbaI (20 U/μl</TD><TD>0.2 μl</TD></TR><TR><TD>ddH2O</TD><TD>3.6 μl</TD></TR><TR><TD>Total volume</TD><TD>10 μl</TD></TR></TBODY></TABLE>
                                                                Incubate at 37˚C for 3 hr. 
                                                                
                                                                <TRIGGER rnas=""><TABLE class="protocol-table"><TBODY><TR><TD>Plasmid DNA ( 25 ng/ μl))</TD><TD>5 μl</TD></TR><TR><TD>NEB #2</TD><TD>1 μl</TD></TR><TR><TD>EcoR I (20 U/μl)</TD><TD>0.2 μl</TD></TR><TR><TD>SpeI (20 U/μl)</TD><TD>0.2 μl</TD></TR><TR><TD>ddH2O</TD><TD>3.6 μl</TD></TR><TR><TD>Total volume</TD><TD>10 μl</TD></TR></TBODY></TABLE>
                                                                Incubate at 37˚C for 3 hr. 
															</TRIGGER></TOEHOLD></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT731">2015.07.31</A></H4></DIV><DIV id="collapseT731" class="panel-collapse collapse "><DIV class="panel-body">
																  Do the electrophoresis of digested IDT products and gel extraction
                                                                  
                                                                  Restriction enzyme digestion of synthetic toehold and trigger RNA with EcoRI and SpeI. Samples were run in 1% agarose gel. Lane 1, 1kb DNA 
                                                                  marker；Lane 2-5, 125 ng of synthetic DNA fragment that would transcribe into toehold sensors (SAT、DUSP1、IL-1β、IL-8 ) were digested by 
                                                                  EcoRI and SpeI；Lane 6-9, 125 ng of DNA fragment that would transcribe into trigger RNAs (SAT、DUSP1、IL-1β、IL-8 ) were digested by 
                                                                  EcoRI and SpeI. 
                                                                  All the results are around 100 bp, are the same as our expectation. 
															</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT803">2015.08.03</A></H4></DIV><DIV id="collapseT803" class="panel-collapse collapse "><DIV class="panel-body">
																Ask NYMU_Taiwan team for confirmed BBa_I712019, because we mistook the wrong antibiotic and lose all the parts DNA.
															</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT804">2015.08.04</A></H4></DIV><DIV id="collapseT804" class="panel-collapse collapse "><DIV class="panel-body">
																Get the parts and transform parts BBa_I712019.
                                                                 <DIGEST bba_i712019=""><TABLE class="protocol-table"><TBODY><TR><TD>Vector DNA</TD><TD>2 μl</TD></TR><TR><TD>Competent cell (DH10β)</TD><TD>100 μl</TD></TR></TBODY></TABLE></DIGEST></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT805">2015.08.05</A></H4></DIV><DIV id="collapseT805" class="panel-collapse collapse "><DIV class="panel-body">
																Do small scale incubation.
															</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT806">2015.08.06</A></H4></DIV><DIV id="collapseT806" class="panel-collapse collapse "><DIV class="panel-body">
																  MIniprep BBa_I712019.
															</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT810">2015.08.10</A></H4></DIV><DIV id="collapseT810" class="panel-collapse collapse "><DIV class="panel-body">
																 Small scale parts plasmids BBa_I712019 (Luciferase) digestion 
                                                                 <DIGEST bba_i712019=""><TABLE class="protocol-table"><TBODY><TR><TD>Plasmid DNA (25 ng/ μl))</TD><TD>2.5 μl</TD></TR><TR><TD>NEB #2</TD><TD>1 μl</TD></TR><TR><TD>EcoRI (20 U/μl)</TD><TD>0.2 μl</TD></TR><TR><TD>SpeI (20 U/μl)</TD><TD>0.2 μl</TD></TR><TR><TD>ddH2O</TD><TD>6.1 μl</TD></TR><TR><TD>Total volume</TD><TD>10 μl</TD></TR></TBODY></TABLE>                                                      Incubate at 37˚C for 3 hr. 
															</DIGEST></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT811">2015.08.11</A></H4></DIV><DIV id="collapseT811" class="panel-collapse collapse "><DIV class="panel-body">
																    Do the electrophoresis of BBa_I712019 (Luciferase) to check the size and condition for experiment and gel extraction to check the size.
                                                                    Mini-prep trigger RNAs in pSB1AK8 backbone (total 20 tubes).
															</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT812">2015.08.12</A></H4></DIV><DIV id="collapseT812" class="panel-collapse collapse "><DIV class="panel-body">
																 Largely cut luciferase in pSB1AK8 and purify it, then we got linear luciferase. 
                                                                 <DIGEST bba_i712019=""><TABLE class="protocol-table"><TBODY><TR><TD>Plasmid DNA (142.5 ng/ μl)</TD><TD>3 μl</TD></TR><TR><TD>Cut smart</TD><TD>3 μl</TD></TR><TR><TD>XbaI (20 U/μl)</TD><TD>0.6 μl</TD></TR><TR><TD>EcoRI (20 U/μl)</TD><TD>0.6 μl</TD></TR><TR><TD>ddH2O</TD><TD>22.8 μl</TD></TR><TR><TD>Total volume</TD><TD>30 μl</TD></TR></TBODY></TABLE>
                                                                Incubate at 37˚C for 3 hr.
                                                                Lane 1 and 4, 3 μg of luciferase digested by EcoRI and XbaI ; Lane 2, 1K DNA marker ;  Lane 3, 3 μg  of luciferase not digested by EcoRI and  
                                                                XbaI.In the picture, uncut plasmid is higher than the cut ones. After discussing with our advisor, we suppose that the uncut plasmid is not 
                                                                functional fold, so it runs slower than the plasmid with digestion.
                                                                Ligation toehold switches with luciferase (pSB1AK8).
                                                                Ligation trigger RNAs into pSB1AK8 backbone.</DIGEST></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT813">2015.08.13</A></H4></DIV><DIV id="collapseT813" class="panel-collapse collapse "><DIV class="panel-body">
																     Transform two kinds of construct into DH10α bacteria respectively to do mass culture.
															</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT814">2015.08.14</A></H4></DIV><DIV id="collapseT814" class="panel-collapse collapse "><DIV class="panel-body">
																     Sent our E. coli and primer (VF2) to sequence for check our assembled result.
															</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT817">2015.08.17</A></H4></DIV><DIV id="collapseT817" class="panel-collapse collapse "><DIV class="panel-body">
																     Have synthetic toeholds (DUSP1, SAT, IL-8) -luciferase in pSB1AK8 sequence be confirmed.
															</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT818">2015.08.18</A></H4></DIV><DIV id="collapseT818" class="panel-collapse collapse "><DIV class="panel-body">		     Small scale  T7 promoter digestion.
															</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT819">2015.08.19</A></H4></DIV><DIV id="collapseT819" class="panel-collapse collapse "><DIV class="panel-body">
																     Large scale T7 promoter digestion, and gel extraction.<DIGEST t7="" promoter=""><TABLE class="protocol-table"><TBODY><TR><TD>Plasmid DNA</TD><TD>30 μl</TD></TR><TR><TD>NEB #2</TD><TD>5 μl</TD></TR><TR><TD>PstI (20 U/μl)</TD><TD>3 μl</TD></TR><TR><TD>SpeI (20 U/μl)</TD><TD>2.6 μl</TD></TR><TR><TD>ddH2O</TD><TD>10 μl</TD></TR><TR><TD>Total volume</TD><TD>50 μl</TD></TR></TBODY></TABLE>
                                                                Incubate at 37˚C for 3 hr.
                                                                  Lane 1, marker ; Lane 2, 3 μg T7 promoter digested by restriction enzyme PstI and SpeI.
															</DIGEST></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT820">2015.08.20</A></H4></DIV><DIV id="collapseT820" class="panel-collapse collapse "><DIV class="panel-body">
																     Toehold+luciferase RNA enzyme digestion, gel extraction.
                                                                     <TABLE class="protocol-table"><TBODY><TR><TD>SAT1 </TD><TD>241.7 (ng/ μl)</TD></TR><TR><TD>DUSP</TD><TD>236.2 (ng/ μl)</TD></TR><TR><TD>IL8</TD><TD>181.2(ng/ μl)</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT821">2015.08.21</A></H4></DIV><DIV id="collapseT821" class="panel-collapse collapse "><DIV class="panel-body">
																  Ligation trigger RNA to T7 promoter in pSB1AK8 backbone.
                                                                 <T7 promoter="" ligation=""><TABLE class="protocol-table"><TBODY><TR><TD>T7 promoter linear (119.6 ng/ μl)  </TD><TD>15 μl</TD></TR><TR><TD>Plasmid DNA  </TD><TD>2 μl</TD></TR><TR><TD>NEB #2</TD><TD>1.5 μl</TD></TR><TR><TD>PstI (20 U/μl)</TD><TD>3 μl</TD></TR><TR><TD>SpeI (20 U/μl)</TD><TD>3 μl</TD></TR><TR><TD>ddH2O</TD><TD>5.5 μl</TD></TR><TR><TD>Total volume</TD><TD>30 μl</TD></TR></TBODY></TABLE>
                                                                 The trigger RNA construct we finally complete. The structure contain T7 promoter, trigger, also Ampicillin antibiotic sequence easy for us to do                                                                   
                                                                 the selection.  
															</T7></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT822">2015.08.22</A></H4></DIV><DIV id="collapseT822" class="panel-collapse collapse "><DIV class="panel-body">
																  Synthetic toehold–luciferase (IL1β) sequences confirmation.
                                                                  T7 promoter-trigger RNA (SAT) in pSB1AK8 backbone sequences confirmation success, but T7 promoter-trigger RNA (DUSP1, IL1β,IL8) in pSB
                                                                  1AK8 backbone sequences confirmation failed.
															</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT823">2015.08.23</A></H4></DIV><DIV id="collapseT823" class="panel-collapse collapse "><DIV class="panel-body">
																  Reconstruct the T7 promoter-trigger RNA (IL8,IL1 β,DUSP) in pSB1AK8 backbone.
															</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT824">2015.08.24</A></H4></DIV><DIV id="collapseT824" class="panel-collapse collapse "><DIV class="panel-body">
																   T7 promoter-trigger RNA(DUSP1, IL-1β,IL-8) in pSB1AK8 quick screen 6 colonies each.
                                                                   1.	Add100 μl lysis buffer into Eppendorf tube.
                                                                   2.	Use stick to take a little bit bacteria into lysis buffer.
                                                                   3.	Add 1:1 phenol/chloroform and shake it.
                                                                   4.	Centrifuge(12000g, 5mins).
                                                                   5.	Take 10μl to electrophoresis.</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT825">2015.08.25</A></H4></DIV><DIV id="collapseT825" class="panel-collapse collapse "><DIV class="panel-body">
																 Toehold switches digestion and ligation with pSB1C3 backbone.<DIGESTION of="" sat1=""><TABLE class="protocol-table"><TBODY><TR><TD>Plasmid DNA (SAT1)</TD><TD>45 μl</TD></TR><TR><TD>NEB#2</TD><TD>6 μl</TD></TR><TR><TD>EcoRI (20 U/μl)</TD><TD>4 μl</TD></TR><TR><TD>PstI (20 U/μl)</TD><TD>4 μl</TD></TR><TR><TD>ddH2O</TD><TD>1 μl</TD></TR><TR><TD>Total volume</TD><TD>60 μl</TD></TR></TBODY></TABLE><DIGESTION of="" dusp1=""><TABLE class="protocol-table"><TBODY><TR><TD>Plasmid DNA (DUSP1)</TD><TD>45 μl</TD></TR><TR><TD>NEB#2</TD><TD>6 μl</TD></TR><TR><TD>EcoRI (20 U/μl</TD><TD>4 μl</TD></TR><TR><TD>PstI (20 U/μl</TD><TD>4 μl</TD></TR><TR><TD>ddH2O</TD><TD>1 μl</TD></TR><TR><TD>Total volume</TD><TD>60 μl</TD></TR></TBODY></TABLE><DIGESTION of="" il8=""><TABLE class="protocol-table"><TBODY><TR><TD>Plasmid DNA (IL8)</TD><TD>45 μl</TD></TR><TR><TD>NEB#2</TD><TD>6 μl</TD></TR><TR><TD>EcoRI (20 U/μl</TD><TD>4 μl</TD></TR><TR><TD>PstI (20 U/μl</TD><TD>4 μl</TD></TR><TR><TD>ddH2O</TD><TD>1 μl</TD></TR><TR><TD>Total volume</TD><TD>60 μl</TD></TR></TBODY></TABLE><DIGESTION of="" il1β=""><TABLE class="protocol-table"><TBODY><TR><TD>Plasmid DNA (IL1β)</TD><TD>45 μl</TD></TR><TR><TD>NEB#2</TD><TD>6 μl</TD></TR><TR><TD>EcoRI (20 U/μl</TD><TD>4 μl</TD></TR><TR><TD>PstI (20 U/μl</TD><TD>4 μl</TD></TR><TR><TD>ddH2O</TD><TD>1 μl</TD></TR><TR><TD>Total volume</TD><TD>60 μl</TD></TR></TBODY></TABLE><LIGATION of="" toehold="" switches="" to="" luciferase=""><TABLE class="protocol-table"><TBODY><TR><TD>BBa_I712074 linear</TD><TD>5 μl</TD></TR><TR><TD>Toehold switch DNA</TD><TD>4 μl</TD></TR><TR><TD>T4 ligase</TD><TD>1 μl</TD></TR><TR><TD>PEG400</TD><TD>2 μl</TD></TR><TR><TD>Total volume</TD><TD>20 μl</TD></TR></TBODY></TABLE>
                                                               The construct of toehold switches part we done. The structure include T7 promoter, toehold switches, and reporter gene—the luciferase. What’s 
                                                               more, the plasmids contain the chloramphenicol resistance gene for selection.
															</LIGATION></DIGESTION></DIGESTION></DIGESTION></DIGESTION></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT826">2015.08.26</A></H4></DIV><DIV id="collapseT826" class="panel-collapse collapse "><DIV class="panel-body">
															  Transformation
                                                              <TABLE class="protocol-table"><TBODY><TR><TD>Vector DNA (158 ng/μl)</TD><TD>1 μl</TD></TR><TR><TD>Competent cell (DH10β)</TD><TD>100 μl</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT827">2015.08.27</A></H4></DIV><DIV id="collapseT827" class="panel-collapse collapse "><DIV class="panel-body">
																   Pick single colony and do the quick screen to check if the parts are inserted. 
															</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT828">2015.08.28</A></H4></DIV><DIV id="collapseT828" class="panel-collapse collapse "><DIV class="panel-body">
													        Small scale incubation.
															</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT831">2015.08.31</A></H4></DIV><DIV id="collapseT831" class="panel-collapse collapse "><DIV class="panel-body">
																    Do the miniprep.
                                                                    DNA Concentration= 174ng/μl  
															</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT91">2015.09.01</A></H4></DIV><DIV id="collapseT91" class="panel-collapse collapse "><DIV class="panel-body">
																   Cotransformation (DUSP1, IL-8, SAT) in the condition of Amp, CAM, Amp+CAM.
															</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT92">2015.09.02</A></H4></DIV><DIV id="collapseT92" class="panel-collapse collapse "><DIV class="panel-body">
																    T7-trigger (IL-1 β) ligation.
                                                                    
                                                                    In the figure, each lane are 10 μl of PCR products, we use temperature gradient (50°C, 53°C, 56°C, 59°C, 62°C, 65°C) to test the product of 
                                                                    which temperature condition is suitable.
                                                                   we choose the result of 53 °C for PCR. Because the product of 53 °C condition is much clearer, also much brighter then other conditions.</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT93">2015.09.03</A></H4></DIV><DIV id="collapseT93" class="panel-collapse collapse "><DIV class="panel-body">
																Cotransformation (DUSP1, IL-8, SAT) in the condition of Amp+CAM .
                                                                   <TABLE class="protocol-table"><TBODY><TR><TD>Toehold plasmid</TD><TD>100 ng</TD></TR><TR><TD>Trigger plasmid</TD><TD>100 ng</TD></TR><TR><TD>Competent cell (BL21DE3)</TD><TD>100 μl</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT96">2015.09.06</A></H4></DIV><DIV id="collapseT96" class="panel-collapse collapse "><DIV class="panel-body">
																1.	Grow the overnight culture before the day of experiment.
                                                                2.	Dilute 200 μl overnight culture with 10 ml LB broth and incubate for 2 hours.
                                                                3.	Control the culture O.D between 0.6~0.8. 
                                                                4.	Spin down(6000g,3 mins) 9 ml culture in Eppendorf tube.
                                                                5.	Wash the pellet with 1 ml PBS and spin down (6000 g, 3 min).
                                                                6.	Add 500μl 1x luciferase passive lysis buffer and move to new mini beads beater tube with enough mini beads.
                                                                7.	Put the tube into mini beads beater and shock 40 sec/time.
                                                                8.	Shock 4~6 times and 2 mins on ice between every shock.
                                                                9.	Spin down(12000g, 5mins)the lysate and move the supernatant into new Eppendorf tube.
                                                              10.	Add 1ml 1X protein assay dye into cuvette and mix with 1μl lysate sample.
                                                              11.	Put the sample into DU800 spectrometer and get the concentration data.
                                                              12.	Normalize the quantity of protein(0.72 mg/ml) in each Eppendorf tube.
                                                              13.	Add 50μl luciferin to each sample and put into GloMax® luminometer.
                                                              14.	Get the number of luciferase activity.</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT97">2015.09.07</A></H4></DIV><DIV id="collapseT97" class="panel-collapse collapse "><DIV class="panel-body">
																Cotransformation (DUSP1, IL-8, SAT) in the condition of Amp+CAM again to have another plate of PCR T7-luciferase (positive).
                                                                   <TABLE class="protocol-table"><TBODY><TR><TD>Toehold plasmid</TD><TD>100 ng</TD></TR><TR><TD>Trigger plasmid</TD><TD>100 ng</TD></TR><TR><TD>Competent cell (BL21DE3)</TD><TD>100 μl</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT98">2015.09.08</A></H4></DIV><DIV id="collapseT98" class="panel-collapse collapse "><DIV class="panel-body">
																Luciferase assay
															</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT99">2015.09.09</A></H4></DIV><DIV id="collapseT99" class="panel-collapse collapse "><DIV class="panel-body">
																Cotransformation (DUSP1, IL-8, SAT) in the condition of Amp+CAM .
                                                                </DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT910">2015.09.10</A></H4></DIV><DIV id="collapseT910" class="panel-collapse collapse "><DIV class="panel-body">														 Luciferase assay
                                                             Failed!!!  No luciferase express.
															</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT911">2015.09.11</A></H4></DIV><DIV id="collapseT911" class="panel-collapse collapse "><DIV class="panel-body">
															Do the electrophoresis of negative control (T7 promoter with toehold switch).
															</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT912">2015.09.12</A></H4></DIV><DIV id="collapseT912" class="panel-collapse collapse "><DIV class="panel-body">
															    Miniprep to prepare the toehold switch IL1β. 
                                                                 Luciferase assay twice.
															</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT913">2015.09.13</A></H4></DIV><DIV id="collapseT913" class="panel-collapse collapse "><DIV class="panel-body">
															 Miniprep sensors and triggers in pSB1C3 (Il8,SAT1,DUSP1) and IL1β.
                                                                         Luciferase assay twice.
															</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT914">2015.09.14</A></H4></DIV><DIV id="collapseT914" class="panel-collapse collapse "><DIV class="panel-body">
															Luciferase assay once.
															</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT916">2015.09.16</A></H4></DIV><DIV id="collapseT916" class="panel-collapse collapse "><DIV class="panel-body">
															Luciferase assay once
															</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapseT917">2015.09.17</A></H4></DIV><DIV id="collapseT917" class="panel-collapse collapse "><DIV class="panel-body">
															Luciferase assay twice.
															</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>