Team:CGU Taiwan/Notebook
Home | CGU_Taiwan
Project
Experiment
Protocols
Modeling
Result
Safety
Parts
Team Parts
Basic Parts
Composite Parts
Part Collection
Human Practice
Collaboration
About Us
Attribution
Notebook
Overview & Timeline
Lab Notes
Home | CGU_Taiwan
Lab Note
Yeast With IL-8 Receptor
2015.07.01
Operator: Wan-Yun, Jin-Ting, Jin-Ye
Goal:
  1. Extraction of gDNA of Far1∆::KANMX strain
  2. Detection the concentration of fast extracted gDNA of ∆Far1 strain
  3. PCR gDNA of Far1∆ ::KANMX
  4. Electrophoresis to check PCR product
Experiment steps:
< Extraction of gDNA of Far1∆::KANMX strain>
Consult the protocol
Strains
260/280
260/230
C(ng/μl)
Far1∆::KANMX strain
1.62
0.97
44.7
1. Design of primers
Name
Pur.
Seq.(5’-3’)
Size (mer.)
MW (g/mol)
Tm (℃)
GC (%)
Nmole
μl for 100μM
dFAR1 F’
Desalt
ggTTTTgTTAggCgggCAAg
20
6244.1
53.8
55
21.25
212.50
dFAR1 R’
Desalt
CATTAACTgCTATTTACgACgC
22
6669.4
51.1
41
17.12
171.20
2. PCR program
Step
Temperature
Time
Step1
95℃
5min
Step2
95℃
30s
Step3
52℃
30s
Step4→step2 for 30 cycle
72℃
2min
Step5
72℃
5min
Step6 (hold on)
10℃
1hr
3.PCR reagent
10x Dream Taq buffer
2.5μl
2.5mM dNTP
0.5μl
10mM primer(F)
0.5μl
10mM primer(R)
0.5μl
template (Far1∆::KANMX strain gDNA)
3.4μl
Taq polymerase
0.5μl
ddH2O
17.1μl
Total volume
25μl
< Electrophoresis to check PCR product>
Material:
  DNA marker: 100bp ladder 8μl
  DNA sample: 2μlPCR product + 1μl 6x loading buffer
Condition:
  0.5xTBE buffer 100V
Time:
  30min
Result:
  M：Marker；#1：Far1 ∆ for annealing at 52℃
  There is no band appears in the gel electrophoresis.
Conclusion:
  Due to the Figure 1 in result, we have to check the temperature of primers annealing and the design of primers to solve the problem. Next, we use gradient PCR to find out the exactly temperature of primer annealing and we add positive control as well.
2015.07.02
Operator: Wan-Yun, Jin-Ting, Jin-Ye
Goal:
  1. Extraction of gDNA of FAR1∆::KANMX strain
  2. Detection the concentration of fast extracted gDNA of FAR1∆::KANMX strain
  3. First round of PCR
  4. Electrophoresis to check first round-PCR product
  5. Second round of PCR
Experiment steps:
< Extraction of gDNA of FAR1∆::KANMX strain>
Consult the protocol
Strains
260/280
260/230
C(ng/μl)
Far1∆::KANMX
1.72
0.78
42.7
Positive control
1.63
0.76
37.1
1. Information of primers
Name
Pur.
Seq.(5’-3’)
Size (mer.)
MW (g/mol)
Tm (℃)
GC (%)
Nmole
μl for 100μM
dFAR1 F’
Desalt
ggTTTTgTTAggCgggCAAg
20
6244.1
53.8
55
21.25
212.50
dFAR1 R’
Desalt
CATTAACTgCTATTTACgACgC
22
6669.4
51.1
41
17.12
171.20
2.PCR program
Step
Temperature
Time
Step1
95℃
5min
Step2
95℃
30s
Step3
Gradient42℃-46℃-50℃
30s
Step4→step2 for 30 cycle
72℃
2min
Step5
72℃
5min
Step6 (hold on)
10℃
1hr
3.PCR reagent
10x Dream Taq buffer
2.5μl
2.5mM dNTP
0.5μl
10mM primer(F)
0.5μl
10mM primer(R)
0.5μl
template (Far1∆::KANMX strain gDNA)
3.4μl
Taq polymerase
0.5μl
ddH2O
17.1μl
Total volume
25μl
< Electrophoresis to check first round-PCR product (25μl ul)>
Material:
  DNA marker: 100bp ladder 8μl
  DNA sample: 2μl PCR product + 1μl 6x loading buffer
Condition:
  0.5xTBE buffer 100V
Time:
  30min
Result:
M: Marker; #1: FAR1∆ annealing at 42 ℃; #2: FAR1∆ annealing at 46 ℃;
#3: FAR1∆ annealing at 50 ℃; #4: FAR1∆ annealing at 52 ℃(positive control)
1.PCR program
Step
Temperature
Time
Step1
95℃
5min
Step2
95℃
30s
Step3
46℃
30s
Step4→step2 for 30 cycle
72℃
2min
Step5
72℃
5min
Step6 (hold on)
10℃
1hr
2.PCR reagent
10x Dream Taq buffer
5μl
2.5mM dNTP
1μl
10mM primer(F)
1μl
10mM primer(R)
1μl
template (First round PCR product)
1μl
Taq polymerase
1μl
ddH2O
40μl
Total volume
50μl
2015.07.03
Operator: Wan-Yun, Jin-Ting, Jin-Ye
Goal:
  1. 1.Electrophoresis to check second round-PCR product
  2. Transform PCR product into FUS::GFP(His) strain
Experiment steps:
< Electrophoresis to check second round-PCR product (50μl)>
Material:
  DNA marker: 100bp ladder 8μl   DNA sample:2μl PCR product + 1ul 6x loading buffer                 Condition: 0.5xTBE buffer 100V
Time: 30min
Result:
Figure 1. Gel electrophoresis of FAR1∆
M: Marker; #1: FAR1∆ annealing at 46 ℃
Consult the protocol < protocol of yeast transformation>
Use strain name: FUS1-GFP(His)
Selection plate: YPD+G418
2015.07.06
Operator: Wan-Yun, Jin-Ting, Jin-Ye
Goal:
  1. Check transformation result
  2. Second round PCR
  3. Electrophoresis to check PCR product
  4. Incubate E.coli with shuttle vector-p426GAL1 from stock
Experiment steps:
1. Take plates out from incubator and observe growth of colony
2. Conclusion: We failed to transformation of PCR product so we should it again.
Consult the experiment record <2015.7.2 Experiment Record>
< Electrophoresis to check second round-PCR product>
Material:
  DNA marker: 100bp ladder 8μl
  DNA sample:2μlPCR product + 1μl 6x loading buffer
Condition: 0.5xTBE buffer 100V
Time: 30min
Result:
Conclusion: Its expected length is 1.9kb and it worked.
< Incubate E.coli with shuttle vector-p426GAL1 from stock>
1. Take stock E.coli with shuttle vector-p426GAL1 from -80℃ fridge.
2. Plate E.coli on LB+Amp plates.
3. Incubate in 37℃ overnight.
2015.07.07
Operator: Wan-Yun, Jin-Ting, Jin-Ye
Goal:
  1. Transform PCR product into FUS::GFP (His) strain
Experiment steps:
Consult the experiment record <2015.7.3 Experiment Record>
2015.07.09
Operator: Wan-Yun, Jin-Ting, Jin-Ye
Goal:
  1. Miniprep plasmid of p426GAL1
  2. Measure concentration of plasmid.
Experiment steps:
< Miniprep plasmid of p426GAL1
Consult the protocol
 
concentration
260/280
260/230
P426GAL1/1
130.9ng/μl
1.91
2.36
P426GAL1/2
121.3ng/μl
1.89
2.26
2015.07.12
Operator: Wan-Yun, Jin-Ting, Jin-Ye
Goal:
  1. Second round PCR
  2. Incubate FAR1△::KANMX strain in 5ml YPD+A medium.
Experiment steps:
Consult the experiment record <2015.7.2 Experiment Record>
2015.07.17
Operator: Wan-Yun, Jin-Ting, Jin-Ye
Goal:
1. Transform PCR product into FUS1-GFP strain
Experiment steps:
< Transform PCR product into FUS1-GFP strain>
Consult the experiment record <2015.7.3 Experiment Record>
2015.07.20
Operator: Wan-Yun, Jin-Ting, Jin-Ye
Goal:
  1. Maintain the colonies of far1Δ::KanMX-FUS1-GFP.
  2. 2nd round PCR for far1Δ::KanMX
  3. Phenol chloroform and EtOH precipitation of 2nd round PCR product
Experiment steps:
< Maintain the colonies of far1Δ::KanMX-FUS1-GFP >
  1. Choose 10 colonies to transfer the new YPD+G418 plate.
  2. Check plates after two days.
<2nd round PCR for far1Δ::KanMX
1.PCR program
Step
Temperature
Time
Step1
95℃
5min
Step2
95℃
30s
Step3
46℃
30s
Step4→step2 for 30 cycle
72℃
2min
Step5
72℃
5min
Step6 (hold on)
10℃
1hr
2.PCR reagent
10x Dream Taq buffer
5μl
2.5mM dNTP
1μl
10mM primer(F)
1μl
10mM primer(R)
1μl
template (First round PCR product)
1μl
Taq polymerase
1μl
ddH2O
40μl
Total volume
50μl
< Phenol chloroform and EtOH precipitation of 2nd round PCR product >
Consult the protocol
2015.07.21
Operator: Wan-Yun, Jin-Ting, Jin-Ye
Goal:
  1. Transform PCR product(Far1Δ::KanMX) into FUS1-GFP strain
  2. Extraction of gDNA of yeast
  3. Check PCR for far1Δ::KanMX-FUS1-GFP
  4. Digestion of p426 Gal
Experimental steps: < Transform PCR product into FUS1-GFP strain>                 Consult the experiment record <2015.7.3 Experiment Record>                Because last time , negative control also had grown colony so we did transformation again. At the same time , we selected colony on selection plate to do PCR check.
< Extraction of gDNA of yeast>
Consult the protocol < protocol of fast extraction of gDNA of yeast>
< Check PCR for Far1Δ::KanMX- FUS-GFP >
1.PCR program
Step
Temperature
Time
Step1
95℃
5min
Step2
95℃
30s
Step3
46℃
30s
Step4→step2 for 30 cycle
72℃
90sec
Step5
72℃
5min
Step6 (hold on)
10℃
1hr
2.PCR reagent
10x Dream Taq buffer
2.5μl
2.5mM dNTP
0.5μl
10mM primer(F)
0.5μl
10mM primer(R)
0.5μl
template (First round PCR product)
8μl
Taq polymerase
0.5μl
ddH2O
12.5μl
Total volume
25μl
far1Δ::KanMX-FUS-GFP
260/280
260/230
ng/μl
1
1.76
0.91
111.7
2
2.05
1.32
158.2
3
2.02
1.29
120.4
4
1.98
1.08
89.9
5
2.05
0.95
66.9
6
2.07
1.45
182.6
7
1.86
0.90
95.4
8
2.08
1.45
166.4
9
2.18
0.42
22.5
10
2.07
1.30
102.4
< Digestion of p426 Gal >
 
Eco RI(μl)
Bam HI(μl)
Uncut(μl)
ddH2O
20.5
20.5
21.5
10x NEB buf. #4
2.5
2.5
2.5
DNA(200ng)
1.5
1.5
1.5
Enzyme
0.5
0.5
--
total
25
25
25
1. Incubate at 37 for 1 hr.
2. Run the sample on the 1% agarose gel at 100 V for 30 min.(10 μl sample +2 μl loading dye)
2015.07.23
Operator: Wan-Yun, Jin-Ting, Jin-Ye
Goal:
  1. Digestion of p426 Gal and rho-CXCR1
  2. Gel extraction of p426 Gal(vector)
  3. Clean up of rho-CXCR1(insert)
Experimental steps:
< Digestion of p426 Gal and rho-CXCR1>
 
121 ng/μl
131 ng/μl
p426 Gal (2 μg)
16.5μl
15.2μl
10x NEB buf. 4
5μl
5μl
Bam HI
2μl
2μl
Eco RI
2μl
2μl
ddH2O
24.5μl
25.8μl
Total
50μl
50 μl
Rho-CXCR1
8(400 ng)
10x NEB buf. 4
5μl
Bam HI
1μl
Eco RI
1μl
ddH2O
35μl
Total
50μl
< Gel extraction of p426 Gal >
Consult the protocol < protocol of gel extraction>
< Clean up of rho-CXCR1>
Consult the protocol
2015.07.24
Operator: Wan-Yun, Jin-Ting, Jin-Ye
Goal:
  1. Ligation of the rho-CXCR1 and p426 Gal
  2. Transform the ligation product into the E.coli
Experimental steps:
< Ligation of the rho-CXCR1 and p426 Gal>
 
1:3
Vector only
Vector(11.1 ng/μl)
10μl
10μl
Insert(9.1 ng/μl)
7μl
0μl
Ligase
1μl
1μl
Ligation buff.
2μl
2μl
ddH2O
0μl
7μl
total
20μl
20μl
Incubate the ligation product at R.T. for 2 hr.
< Transform the ligation product into the competent cell>
Consult the protocol < protocol of ligation and transformation of E.coli>
2015.07.27
Operator: Wan-Yun, Jin-Ting
Goal:
  1. Fast extraction of gDNA of Far1Δ::KanMX-FUS1-GFP
  2. Check PCR for far1Δ::KanMX-FUS-GFP
Experimental steps: < Fast extraction of gDNA of far1Δ::KanMX-FUS-GFP >                Consult the protocol
Strains
260/280
260/230
C(ng/μl)
FAR1∆::KANMX
1.85
1.19
177.3
FAR1∆::KANMX
1.83
1.19
163.8
Positive control
1.96
1.61
235.3
< Check PCR for Far1Δ::KanMX-FUS1-GFP>
1.PCR program
Step
Temperature
Time
Step1
95℃
5min
Step2
95℃
30s
Step3
46℃
30s
Step4→step2 for 30 cycle
72℃
90sec
Step5
72℃
5min
Step6 (hold on)
10℃
1hr
2.PCR reagent
 
FAR1∆::KANMX
Positive control
10x Dream Taq buffer
2.5μl
2.5μl
2.5mM dNTP
0.5μl
0.5μl
10mM primer(F)
0.5μl
0.5μl
10mM primer(R)
0.5μl
0.5μl
template (First round PCR product)
0.92μl
0.64μl
Taq polymerase
0.5μl
0.5μl
ddH2O
19.58μl
19.86μl
Total volume
25μl
25μl
2015.07.28
Operator: Wan-Yun, Jin-Ting, Jin-Ye
Goal:
  1.Fast extraction of pGAL426 rho-CXCR1
Experimental steps:
< Fast extraction of plasmid DNA >
  1. Add 50 μl of STE buffer(100 mM NaCl, 10 mM Tris buffer, pH 7.0, 1 mM EDTA) into each new Eppendorf tube.
  2. Add 50 μl of Phenol chloroform and vortex vigorously for 30 sec.
  3. Centrifuge at 13,000 g for 5 min.
  4. Remove the 10 μl of supernatant to a new Eppendorf tube and run the sample on the 1 % agarose gel.
2015.07.29
Operator: Wan-Yun, Jin-Ting
Goal:
  1. Miniprep of p426 Gal-rho-CXCR1
  2. Enzyme digestion to check p426 Gal-rho-CXCR1
Experimental steps:
< Miniprep of p426 Gal-rho-CXCR1>
Consult the protocol
< Enzyme digestion for the p426 Gal-rho-CXCR1 check >
1. Enzyme digestion
p426 Gal-rho-CXCR1
#7,9
+,#8,12,16
10x NEB buff.4
2.5μl
2.5μl
Eco RI
0.5μl
0.5μl
Bam HI
0.5μl
0.5μl
DNA
2μl
1μl
ddH2O
19.5μl
20.5μl
total
25μl
25μl
2. Loading 10 μl sample and 2 μl DNA loading dye into the 1 % agarose gel.
2015.07.30
Operator: Wan-Yun, Jin-Ting, Jin-Ye
Goal:
  1. Transform pSB1C3 BBa_J04450 into the competent cell
  2. Transform the plasmid (pRS405) into the competent cell
Experimental steps:
< Transform pSB1C3 BBa_J04450 into the competent cell>
Consult the protocol
Selection plate: LB+ Cam plate
< Transform the plasmids (pRS405) into the competent cell>
Selection plate: LB+ Amp plate
2015.08.03
Operator: Wan-Yun, Jin-Ting, Jin-Ye
Goal:
  1. Miniprep of pRS405 and run the 1% agarose gel
  2. Digestion of pRS405 with Xho I, Bam HI and Eco RI
  3. Digestion of p426 Gal with Xho I
  4. Inoculate the bacteria with BBa_J04450 into the LB+ chloramphanicle broth
Experiment steps:
< Miniprep of pRS405 and run the 1% agarose gel >
Consult the protocol < protocol of miniprep plamid>
< Digestion of pRS405 with Xho I, Bam HI and Eco RI >
 
uncut
Xho I
Bam HI
Eco RI
pRS405 (V105○1)
0.5
0.5
0.5
0.5
10x NEB buff.4
2.5
2.5
2.5
2.5
Enzyme
-
0.5
0.5
0.5
ddH2O
22.5
21.5
21.5
21.5
total
25 μl
25 μl
25 μl
25 μl
Reaction condition: 37℃, 1hr
< Digestion of p426 Gal with Xho I >
.
uncut
Xho I
p426 Gal (131 ng/μl)
0.5
0.5
10x NEB buff.4
2.5
2.5
Enzyme
-
0.5
ddH2O
22.5
21.5
total
25 μl
25 μl
Reaction condition: 37℃, 1hr
2015.08.04
Operator: Wan-Yun, Jin-Ting
Goal:
  1. 1st PCR of rho-CXCR1
  2. Electrophoresis to check the PCR product
  3. Miniprep of pSB1C3 BBa_J04450
Experiment steps:
<1st PCR of rho-CXCR1>
1. PCR program
Step
Temp.
Time
Step1
95℃
5 min
Step2
95℃
30 sec
Step3
40,44,48℃
30 sec
Step4→Step2 for 30 cycle 
72℃
80sec
Step5
72℃
5 min
Step6(Hold on)
10℃
1 hr
2. PCR reagent
10x Dream Taq buffer
2.5μl
2.5mMdNTP 
0.5μl
10μMprimer(F’)
0.5μl
10μMprimer(R’)
0.5 μl
Rho-CXCR1
3μl
Taq polymerase
0.5 μl
ddH2O
17.5μl
Total volume
25 μl
< Electrophoresis to check the PCR product>
< Electrophoresis to check second round-PCR product>
Material:
  DNA marker: 100bp ladder 8μl
  DNA sample:2μlPCR product + 1μl 6x loading buffer
Condition:
  0.5xTBE buffer 100V
Time:
  30min
Result:
Conclusion: It expected length is 1200bp and it worked.
< Miniprep of pSB1C3 BBa_J04450>
Consult the protocol
2015.08.10
Operator: Wan-Yun, Jin-Ting, Jin-Ye
Goal:
  1. Electrophoresis to check p426Gal-rho-CXCR1 digestion product again
  2. Do digestion of Lucy-rho-CXCR1 (insert) and p426Gal (vector)
  3. Clean up digestion product of Lucy-rho-CXCR1
  4. Gel extraction of digestion product of p426Gal
  5. Ligation insert and vector
  6. Transform the construct into competent cell
Experiment steps:
< Electrophoresis to check p426 Gal-rho-CXCR1 digestion product again>
Material:
  DNA marker: 1kb ladder 6ul
  DNA sample:2ul p426Gal-rho-CXCR1 digestion product(BamHI and XhoI) + 1ul 6x loading buffer
Condition:
   0.5xTBE buffer 100V
Time:
  30min                 Result:
Note: pGAL426 rho-CXCR1 enzyme digestion product with BamHI and XhoI
 
Lucy-rho-CXCR1 (insert)
p426Gal (vector)
p426Gal (vector)
10x NEB buffer#4
5μl
5μl
5μl
BamHI
1μl
1μl
1μl
XhoI
1μl
1μl
1μl
DNA
25μl (400ng)
7μl (2ug)
7μl
ddH2O
18μl
34μl
34μl
Total volume
50μl
50μl
50μl
Reaction condition : 37℃ for 1hr
< Clean up digestion product of Lucy-rho-CXCR1>
Consult the protocol
Conc
260/280
260/230
10.0ng/ul
1.95
0.94
< Gel extraction of digestion product of p426Gal>
Consult the protocol
Conc.
260/280
260/230
11.2ng/ul
1.75
0.20
< Ligation insert and vector>
Vector:insert=1:3
 
Test tube
Vector only
Vector(11.2ng/ul)
9μl
9μl
Insert(10.0ng/ul)
6μl
0μl
Ligase
1μl
1μl
10xLigation buffer
2μl
2μl
ddH2O
2μl
8μl
Total volume
20μl
20μl
Reaction condition: Room temperature for 2-4hr
< Transform the construct into competent cell>
Selection plate: LB+Cam
2015.08.11
Operator: Wan-Yun, Jin-Ting, Jin-Ye
Goal:
  1. Obeserve colony growth to confirm transformation result
  2. PCR Gpa1-Leu2
  3. Electrophoresis to check PCR product
  4. Second PCR round of Gpa1-Leu2 with first product
  5. Extraction of DNA with Phenol/Chloroform
Experiment steps:
< Obeserve colony growth to confirm transformation result>
Select 12 colonies and incubate in LB broth(with Amp) overnight,37℃
< PCR Gpa1-Leu2>
1.PCR program
Step
Temperature
Time
Step1
95℃
5min
Step2
95℃
30s
Step3
48℃,53℃,59℃,62℃
90s
Step4→step2 for 30 cycle
72℃
5min
Step5
72℃
5min
Step6 (hold on)
10℃
1hr
2.PCR reagent
10x Dream Taq buffer
2.5μl
2.5mM dNTP
0.5μl
10mM primer(F)
0.5μl
10mM primer(R)
0.5μl
Template(pRS405,150ng)
0.55μl
Taq polymerase
0.5μl
ddH2O
20μl
Total volume
25μl
< Electrophoresis to check second round-PCR product (50μl)>
Material:
  DNA marker: 1kb ladder 6μl
  DNA sample:2μl PCR product + 1μl 6x loading buffer
Condition:
  0.5xTBE buffer 100V
Time:
  30min
Result:
Conclution: Its expected length is 1502bp and it worked.
< Second PCR round of Gpa1-Leu2 with first product>
Step
Temperature
Time
Step1
95℃
5min
Step2
95℃
30s
Step3
53℃
30s
Step4→step2 for 30 cycle
72℃
5min
Step5
72℃
5min
Step6 (hold on)
10℃
1hr
2. PCR reagent
10x Dream Taq buffer
5μl
2.5mM dNTP
1μl
10mM primer(F)
1μl
10mM primer(R)
1μl
template (First round PCR product)
1μl
Taq polymerase
1μl
ddH2O
40μl
Total volume
50μl
< Extraction of DNA with Phenol/Chloroform>
Consult the protocol
2015.08.12
Operator: Wan-Yun, Jin-Ting, Jin-Ye
Goal:
     1. Miniprep of p426Gal-Lucy-rho-CXCR1
  2. Electrophoresis to confirm whether transformed successfully or not
  3. Digestion of p426Gal-Lucy-rho-CXCR1
  4. Electrophoresis to confirm which colony been transformed successlly
  5. Transform Gpa1 PCR product into FAR1::KANMX-FUS1-GFP strain
< Miniprep of p426Gal-Lucy-rho-CXCR1>
Consult the protocol
< Electrophoresis to confirm whether transformed successfully or not>
Material: DNA marker: 1kb ladder 6ul
DNA sample: plasmid 2ul + 3ul 6x loading buffer
Condition: 0.5xTBE buffer 100V
Time: 40min
Result:
Note: This gel result shows size difference of plasmid transformed or untransformed.
< Digestion of p426Gal-Lucy-rho-CXCR1>
DNA(p426Gal-Lucy-rho-CXCR1 #2 #3 #7)
1μl
BamHI
0.5μl
XhoI
>0.5μl
10x NEB buffer #4
2.5μl
ddH2O
20.5μl
Total volume
25μl
Condition:  incubate at 37℃ for 1hr
< Electrophoresis to confirm which colony been transformed successlly>
Material: DNA marker: 1kb ladder 6μl
DNA sample: plasmid 5μl + 3μl 6x loading buffer
Condition: 0.5xTBE buffer 100V
Time: 30min
Result:
Note: We use BamHI and XhoI to check whether the construct was built successfully.
< Transform Gpa1 PCR product into FAR1::KANMX-FUS1-GFP strain>
Consult the protocol
Yeast strain: FAR1::KANMX-FUS1-GFP
Selection plate : YPD+G418 plate
2015.08.13
Operator: Wan-Yun, Jin-Ting, Jin-Ye
Goal:
  1. Digestion of pSB1C3 BBa_J04450
  2. Electrophoresis to check digestion product
  3. Digestion of pSB1C3 BBa_J04450 and rho-CXCR1
  4. Gel extraction of vector and clean up insert
  5. Ligation and transformation
Experiment steps:
< Digestion of pSB1C3 BBa_J04450>
 
Both
Only XbaI
Only SpeI
Uncut
pSB1C3 BBa_J04450#3
1μl
1μl
1μl
1μl
10x NEB buffer #4
2.5μl
2.5μl
2.5μl
0μl
XbaI
0.3μl
0.3μl
0μl
0μl
SpeI
0.3μl
0μl
0.3μl
0μl
ddH2O
21μl
21.2μl
21.2μl
24μl
Total volume
25μl
25μl
25μl
25μl
Condition:  at 37℃ fpr 1hr
< Electrophoresis to check digestion product>
Material: DNA marker: 1kb ladder 6μl
DNA sample: 10μl + 3μl 6x loading buffer
Condition: 0.5xTBE buffer 100V
Time: 30min
Result:
Note: It was up to our expection.
< Digestion of pSB1C3 BBa_J04450 and rho-CXCR1>
 
Rho-CXCR1(400ng)
BBa_J04450(2μg)x2 tubes
DNA
8μl
9.6μl
10x NEB buffer#4
5μl
5μl
XbaI
1μl
2μl
SpeI
1μl
2μl
ddH2O
35μl
31.5μl
Total volume
50μl
50μl
Condition: at 37℃ for 1hr
< Gel extraction of vector>
Consult the protocol < protocol of gel extraction>
Conc.
260/280
260/230
6.3ng/μl
1.62
0.17
< Clean up the insert>
Consult the protocol < protocol of clean up after digestion>
Conc.
260/280
260/230
4.9ng/μl
1.48
0.68
< Ligation and transformation>
 
Construct
Vector only
Veator
6μl
6μl
Insert
13μl
0μl
10x ligation buffer
2μl
2μl
Ligase
1μl
1μl
ddH2O
0.5μl
11μl
Total volume
22μl
20μl
Condition: room temperature 20℃ for 2hr
< Transform the construct into competent cell>
Consult the protocol
Selection plate: LB+CAM plates
2015.08.17
Operator: Wan-Yun, Jin-Ye
Goal:
  1.Fast and dirty extraction of gDNA of Gpa1 chimera yeast
  2. Do PCR to check location of Gpa1 chimera
  3. Electrophoresis to check PCR product
  4. Do colony PCR to check location of insert (rho-CXCR1 and Lucy-rho-CXCR1)
  5. Electrophoresis to check colony PCR product
Experiment steps:
< Fast and dirty extraction of gDNA of Gpa1 chimera yeast>
Consult the protocol
Strain name
Conc.(ng/μl)
260/280
260/230
Gpa1 chimera
277.6
2.00
1.49
Far1△::KANMX-FUS1-GFP
303.6
2.08
1.71
1.PCR program
Step
Temperature
Time
Step1
95℃
5min
Step2
95℃
30s
Step3
48℃,52℃,55℃,59℃
30s
Step4→step2 for 30 cycle
72℃
50sec
Step5
72℃
5min
Step6 (hold on)
10℃
1hr
2.PCR reagent
 
Gpa1 chimera
FAR1△::KANMX-FUS1-GFP
10x Dream Taq buffer
2.5μl
2.5μl
2.5mM dNTP
0.5μl
0.5μl
10mM primer(F)
0.5μl
0.5μl
10mM primer(R)
0.5μl
0.5μl
Template(150ng)
0.55μl
0.5μl
Taq polymerase
0.5μl
0.5μl
ddH2O
20μl
20μl
Total volume
25μl
25μl
< Electrophoresis to check PCR product>
Electrophoresis
Material: DNA marker: 1kb ladder 6μl
DNA sample: 2μl + 3μl 6x loading buffer
Condition: 0.5xTBE buffer 100V
Time: 30min
Result:
Note: Gpa1 chimera PCR check with different annealing Tm.
< Do colony PCR to check location of insert (rho-CXCR1 and Lucy-rho-CXCR1)>
 
pSB1C3 rho-CXCR1
pSB1C3 Lucy-rho-CXCR1
pSB1C3 BBa_J04450#3
DNA template
Colonyx7
Colonyx7
0.7
10μM VF2 primer
0.5μl
0.5μl0.5μl
0.5μl
10μM VR primer
0.5μl
0.5μl
0.5μl
2x Taq mix
12.5μl
12.5μl
12.5μl
ddH2O
11.5μl
11.5μl
10.8μl
Total volume
25μl
25μl
25μl
< Electrophoresis to check colony PCR product>
Material: DNA marker: 6μl 1kb ladder
DNA sample: 2μl + 4μl 6x loading buffer
Condition: 0.5xTBE buffer 100V
Time: 30min
Result:
Note: The middle lane was positive control---pSB1C3 BBa_J04450 and other lanes were loaded with construct built. However , the gel result was out of our expection and we assumed that they were self-ligation.
2015.08.18
Operator: Wan-Yun, Jin-Ting, Jin-Ye
Goal:
  1.Induction of three yeast strains with α factor
  2.Observe GFP under microscope.
< Induction of three yeast strains with α factor>
Consult the protocol
< Observe GFP under microscope>
2015.08.25
Operator: Jin-Ting, Jin-Ye
Goal:
  1.Transform pGal426 Lucy-rho-CXCR1 into strain FAR1△::KANMX-FUS1-GFP Gpa1 chimera
Experiment step:
< Transform pGal426 Lucy-rho-CXCR1 into strain FAR1△::KANMX-FUS1-GFP Gpa1 chimera>
Consult the protocol
Selection plate: Ura-+Leu- +G418 plate
2015.08.27
Operator: Jin-Ting, Jin-Ye
Goal:
  1.Indution of yeast with αfactor
Experiment record:
Consult protocol
Strain:
positive control: YGR254W/YJR009C/YMR186W/YPR035W/YAL038W
Negative control: wild type
Test strain: FUS1-GFP
2015.08.28~08.29
Operator: Wan-Yun, Jin-Ting, Jin-Ye
Goal:
  1.Use galactose to induce yeast experess CXCR1
Experiment record:
Consult the protocol
Strain: FAR1△::KANMX-FUS1-GFP Gpa1 chimera-pGAL426 rho-CXCR1
2015.08.31
Operator: Wan-Yun, Jin-Ting, Jin-Ye
Goal:
  1. Protein extraction of induced yeast
Experiment steps:
Consult the protocol
Protein sample : cell culture with different induction time(3h,4h,5h,6h,7h,8h)
2015.09.02
Operator: Jin-Ting, Jin-Ye
Goal:
  1.SDS-PAGE and Western blot
Experiment steps:
Consult the protocol
Protein sample : product of protein extraction last week
Positive control : HEP G2 cell lysate （from human）
2015.09.03~09.04
Operator: Wan-Yun, Jin-Ting, Jin-Ye
Goal:
  1. Use galactose to induce yeast express Lucy-rho-CXCR1 and rho-CXCR1
  2. Treat yeast with IL-8
Experiment steps:
< Use galactose to induce yeast express Lucy-rho-CXCR1 and rho-CXCR1>
Consult the protocol
Test strain:
  ①Far1△::KANMX-FUS1-GFP Gpa1 chimera-pGAL426 Lucy-rho-CXCR1
  ②Far1△::KANMX-FUS1-GFP Gpa1 chimera-pGAL426 rho-CXCR1
Medium used: YEP(Raf) and YEP(Raf/Gal)
< Treat yeast with IL-8>
1. Collect 100μl yeast with galactose induction in 3hr.
2. Add 8μl IL-8(10μlM) to cell culture.
3. Take 10μl of total to observer under microscope.
2015.09.07~09.08
Operator: Wan-Yun, Jin-Ting, Jin-Ye
Goal:
  1.Protein extraction of yeast with galactose induced
  2.Do SDS-PAGE and Western Blot
Experiment steps:
< Protein extraction of yeast with galactose induced>
Consult the protocol < protocol of protein extraction of yeast>
Yeast strain:
  ①Far1△::KANMX-FUS1-GFP Gpa1 chimera-pGAL426 Lucy-rho-CXCR1
  ②Far1△::KANMX-FUS1-GFP Gpa1 chimera-pGAL426 rho-CXCR1
Sample : yeast with galactose induced by 0hr, 1hr, 2hr, 3hr, 5hr, 8hr
< Do SDS-PAGE and Western Blot>
Consult the protocol < protocol of SDS-PAGE and Western blot>
Negative control: yeast with galactose induced by 0hr
Positive control: HEPG2 cell lysate
2015.09.09
Operator: Jin-Ting, Jin-Ye
Goal:
  1.Do galactose induction of yeast
  2.Treat yeast with IL-8
Experiment steps:
Consult the protocol
Strain used:
  ①Far1△::KANMX-FUS1-GFP Gpa1 chimera
  ②Far1△::KANMX-FUS1-GFP Gpa1chimera pGAL426 rho-CXCR1
  ③Far1△::KANMX-FUS1-GFP Gpa1 chimera pGAL 426 Lucy-rho-CXCR1
Medium used: SCM(Ura-)+Raf and SCM(Ura-)+Raf/Gal
Negative control: Far1△::KANMX-FUS1-GFP Gpa1 chimera
< Treat yeast with IL-8>
1. Collect 100μl yeast culture with galactose induced by 3hr.
2. Add 8μl IL-8 (10μM) to cell culture.
3. Take 10μl of total to observe under microscope.
Positive control: YGR254W
Negative control: Wild type yeast
Positive control 2: FAR1△::KANMX-FUS1-GFP with α factor induced
Toehold Switch As RNA Sensor
2015.06.20~2015.06.25
A, IL8 ; B, IL1β ; C, DUSP1 (dual specificity phosphatase 1) ;
D, SAT (spermidine/spermine N1-acetyltransferase EST
We use the website RNA structure to predict the secondary structure of these four toehold switches whether would have hairpin structures we want. Luckily, all of its results are match our expectation.
2015.06.25
Sent our designed sequences to IDT.
2015.07.01~2015.07.14
Amplify parts DNA we need.
2015.07.15
Receive IDT DNA.
2015.07.27
Transform BBa_I712019.
2015.07.28
Small scale incubation.
2015.07.29
Mini prep BBa_I712019 (Luciferase).
Concentration=186.2 (ng/ μl)
2015.07.30
Digest 4 toehold switches with EcoRI and XbaI and 4 trigger DNAs with XbaI and SpeI.
Plasmid DNA (25 ng/ μl)
5 μl
NEB #2
1 μl
EcoRI (20 U/μl)
0.2 μl
XbaI (20 U/μl
0.2 μl
ddH2O
3.6 μl
Total volume
10 μl
Incubate at 37˚C for 3 hr.
Plasmid DNA ( 25 ng/ μl))
5 μl
NEB #2
1 μl
EcoR I (20 U/μl)
0.2 μl
SpeI (20 U/μl)
0.2 μl
ddH2O
3.6 μl
Total volume
10 μl
Incubate at 37˚C for 3 hr.
2015.07.31
Do the electrophoresis of digested IDT products and gel extraction
Restriction enzyme digestion of synthetic toehold and trigger RNA with EcoRI and SpeI. Samples were run in 1% agarose gel. Lane 1, 1kb DNA                                                                    marker；Lane 2-5, 125 ng of synthetic DNA fragment that would transcribe into toehold sensors (SAT、DUSP1、IL-1β、IL-8 ) were digested by                                                                    EcoRI and SpeI；Lane 6-9, 125 ng of DNA fragment that would transcribe into trigger RNAs (SAT、DUSP1、IL-1β、IL-8 ) were digested by                                                                    EcoRI and SpeI.
All the results are around 100 bp, are the same as our expectation.
2015.08.03
Ask NYMU_Taiwan team for confirmed BBa_I712019, because we mistook the wrong antibiotic and lose all the parts DNA.
2015.08.04
Get the parts and transform parts BBa_I712019.
Vector DNA
2 μl
Competent cell (DH10β)
100 μl
2015.08.05
Do small scale incubation.
2015.08.06
MIniprep BBa_I712019.
2015.08.10
Small scale parts plasmids BBa_I712019 (Luciferase) digestion
Plasmid DNA (25 ng/ μl))
2.5 μl
NEB #2
1 μl
EcoRI (20 U/μl)
0.2 μl
SpeI (20 U/μl)
0.2 μl
ddH2O
6.1 μl
Total volume
10 μl
Incubate at 37˚C for 3 hr.
2015.08.11
Do the electrophoresis of BBa_I712019 (Luciferase) to check the size and condition for experiment and gel extraction to check the size.
Mini-prep trigger RNAs in pSB1AK8 backbone (total 20 tubes).
2015.08.12
Largely cut luciferase in pSB1AK8 and purify it, then we got linear luciferase.
Plasmid DNA (142.5 ng/ μl)
3 μl
Cut smart
3 μl
XbaI (20 U/μl)
0.6 μl
EcoRI (20 U/μl)
0.6 μl
ddH2O
22.8 μl
Total volume
30 μl
Incubate at 37˚C for 3 hr.
Lane 1 and 4, 3 μg of luciferase digested by EcoRI and XbaI ; Lane 2, 1K DNA marker ;  Lane 3, 3 μg  of luciferase not digested by EcoRI and                                                                   XbaI.In the picture, uncut plasmid is higher than the cut ones. After discussing with our advisor, we suppose that the uncut plasmid is not                                                                  functional fold, so it runs slower than the plasmid with digestion.
Ligation toehold switches with luciferase (pSB1AK8).
Ligation trigger RNAs into pSB1AK8 backbone.
2015.08.13
Transform two kinds of construct into DH10α bacteria respectively to do mass culture.
2015.08.14
Sent our E. coli and primer (VF2) to sequence for check our assembled result.
2015.08.17
Have synthetic toeholds (DUSP1, SAT, IL-8) -luciferase in pSB1AK8 sequence be confirmed.
2015.08.18
Small scale  T7 promoter digestion.
2015.08.19
Large scale T7 promoter digestion, and gel extraction.
Plasmid DNA
30 μl
NEB #2
5 μl
PstI (20 U/μl)
3 μl
SpeI (20 U/μl)
2.6 μl
ddH2O
10 μl
Total volume
50 μl
Incubate at 37˚C for 3 hr.
Lane 1, marker ; Lane 2, 3 μg T7 promoter digested by restriction enzyme PstI and SpeI.
2015.08.20
Toehold+luciferase RNA enzyme digestion, gel extraction.
SAT1
241.7 (ng/ μl)
DUSP
236.2 (ng/ μl)
IL8
181.2(ng/ μl)
2015.08.21
Ligation trigger RNA to T7 promoter in pSB1AK8 backbone.
T7 promoter linear (119.6 ng/ μl)
15 μl
Plasmid DNA
2 μl
NEB #2
1.5 μl
PstI (20 U/μl)
3 μl
SpeI (20 U/μl)
3 μl
ddH2O
5.5 μl
Total volume
30 μl
The trigger RNA construct we finally complete. The structure contain T7 promoter, trigger, also Ampicillin antibiotic sequence easy for us to do                                                                                                                                     the selection.
2015.08.22
Synthetic toehold–luciferase (IL1β) sequences confirmation.
T7 promoter-trigger RNA (SAT) in pSB1AK8 backbone sequences confirmation success, but T7 promoter-trigger RNA (DUSP1, IL1β,IL8) in pSB                                                                   1AK8 backbone sequences confirmation failed.
2015.08.23
Reconstruct the T7 promoter-trigger RNA (IL8,IL1 β,DUSP) in pSB1AK8 backbone.
2015.08.24
T7 promoter-trigger RNA(DUSP1, IL-1β,IL-8) in pSB1AK8 quick screen 6 colonies each.
1. Add100 μl lysis buffer into Eppendorf tube.
2. Use stick to take a little bit bacteria into lysis buffer.
3. Add 1:1 phenol/chloroform and shake it.
4. Centrifuge(12000g, 5mins).
5. Take 10μl to electrophoresis.
2015.08.25
Toehold switches digestion and ligation with pSB1C3 backbone.
Plasmid DNA (SAT1)
45 μl
NEB#2
6 μl
EcoRI (20 U/μl)
4 μl
PstI (20 U/μl)
4 μl
ddH2O
1 μl
Total volume
60 μl
Plasmid DNA (DUSP1)
45 μl
NEB#2
6 μl
EcoRI (20 U/μl
4 μl
PstI (20 U/μl
4 μl
ddH2O
1 μl
Total volume
60 μl
Plasmid DNA (IL8)
45 μl
NEB#2
6 μl
EcoRI (20 U/μl
4 μl
PstI (20 U/μl
4 μl
ddH2O
1 μl
Total volume
60 μl
Plasmid DNA (IL1β)
45 μl
NEB#2
6 μl
EcoRI (20 U/μl
4 μl
PstI (20 U/μl
4 μl
ddH2O
1 μl
Total volume
60 μl
BBa_I712074 linear
5 μl
Toehold switch DNA
4 μl
T4 ligase
1 μl
PEG400
2 μl
Total volume
20 μl
The construct of toehold switches part we done. The structure include T7 promoter, toehold switches, and reporter gene—the luciferase. What’s                                                                 more, the plasmids contain the chloramphenicol resistance gene for selection.
2015.08.26
Transformation
Vector DNA (158 ng/μl)
1 μl
Competent cell (DH10β)
100 μl
2015.08.27
Pick single colony and do the quick screen to check if the parts are inserted.
2015.08.28
Small scale incubation.
2015.08.31
Do the miniprep.
DNA Concentration= 174ng/μl
2015.09.01
Cotransformation (DUSP1, IL-8, SAT) in the condition of Amp, CAM, Amp+CAM.
2015.09.02
T7-trigger (IL-1 β) ligation.
In the figure, each lane are 10 μl of PCR products, we use temperature gradient (50°C, 53°C, 56°C, 59°C, 62°C, 65°C) to test the product of                                                                      which temperature condition is suitable.
we choose the result of 53 °C for PCR. Because the product of 53 °C condition is much clearer, also much brighter then other conditions.
2015.09.03
Cotransformation (DUSP1, IL-8, SAT) in the condition of Amp+CAM .
Toehold plasmid
100 ng
Trigger plasmid
100 ng
Competent cell (BL21DE3)
100 μl
2015.09.06
1. Grow the overnight culture before the day of experiment.
2. Dilute 200 μl overnight culture with 10 ml LB broth and incubate for 2 hours.
3. Control the culture O.D between 0.6~0.8.
4. Spin down(6000g,3 mins) 9 ml culture in Eppendorf tube.
5. Wash the pellet with 1 ml PBS and spin down (6000 g, 3 min).
6. Add 500μl 1x luciferase passive lysis buffer and move to new mini beads beater tube with enough mini beads.
7. Put the tube into mini beads beater and shock 40 sec/time.
8. Shock 4~6 times and 2 mins on ice between every shock.
9. Spin down(12000g, 5mins)the lysate and move the supernatant into new Eppendorf tube.
10. Add 1ml 1X protein assay dye into cuvette and mix with 1μl lysate sample.
11. Put the sample into DU800 spectrometer and get the concentration data.
12. Normalize the quantity of protein(0.72 mg/ml) in each Eppendorf tube.
13. Add 50μl luciferin to each sample and put into GloMax® luminometer.
14. Get the number of luciferase activity.
2015.09.07
Cotransformation (DUSP1, IL-8, SAT) in the condition of Amp+CAM again to have another plate of PCR T7-luciferase (positive).
Toehold plasmid
100 ng
Trigger plasmid
100 ng
Competent cell (BL21DE3)
100 μl
2015.09.08
Luciferase assay
2015.09.09
Cotransformation (DUSP1, IL-8, SAT) in the condition of Amp+CAM .
2015.09.10
Luciferase assay                                                              Failed!!!  No luciferase express.
2015.09.11
Do the electrophoresis of negative control (T7 promoter with toehold switch).
2015.09.12
Miniprep to prepare the toehold switch IL1β.
Luciferase assay twice.
2015.09.13
Miniprep sensors and triggers in pSB1C3 (Il8,SAT1,DUSP1) and IL1β.
Luciferase assay twice.
2015.09.14
Luciferase assay once.
2015.09.16
Luciferase assay once
2015.09.17
Luciferase assay twice.
