<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_IIT_Kharagpur_Project skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:IIT Kharagpur/Project</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="jumbotron jumbo-project"><DIV class="menu"><UL><LI class="menu-logo"><A href="https://2015.igem.org/Team:IIT_Kharagpur"><DIV class="menu-logo-text">IGEM IIT  Kharagpur</DIV></A></LI><LI class="menu-li"><A href="https://2015.igem.org/Team:IIT_Kharagpur"><DIV class="menu-text">Home</DIV></A></LI><LI class="menu-li"><A href="https://2015.igem.org/Team:IIT_Kharagpur/Project"><DIV class="menu-text">Project</DIV></A></LI><LI><A href="https://2015.igem.org/Team:IIT_Kharagpur/Project">About</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Kharagpur/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Kharagpur/Modeling">Modeling</A></LI></UL><LI class="menu-li"><A href="https://2015.igem.org/Team:IIT_Kharagpur/Team"><DIV class="menu-text">Team</DIV></A></LI><LI><A href="https://2015.igem.org/Team:IIT_Kharagpur/Team">Members</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Kharagpur/Attributions">Attributions</A></LI><LI class="menu-li"><A href="https://2015.igem.org/Team:IIT_Kharagpur/Notebook"><DIV class="menu-text">Notebook</DIV></A></LI><LI class="menu-li"><A href="https://2015.igem.org/Team:IIT_Kharagpur/Practices"><DIV class="menu-text">Practices</DIV></A></LI><LI><A href="https://2015.igem.org/Team:IIT_Kharagpur/Practices#meetups">Meetups</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Kharagpur/Practices#collabs">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Kharagpur/Practices#surveys">Surveys</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Kharagpur/Practices#interviews">Interviews</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Kharagpur/Practices#workshops">Workshops</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Kharagpur/Practices#guidebook">Guidebook</A></LI><LI class="menu-li"><A href="https://2015.igem.org/Team:IIT_Kharagpur/Safety"><DIV class="menu-text">Safety</DIV></A></LI><LI class="menu-li"><A href="https://2015.igem.org/Team:IIT_Kharagpur/Blog"><DIV class="menu-text">Blog</DIV></A></LI></DIV></DIV><DIV class="body-wrapper" style="background:url('https://static.igem.org/mediawiki/2015/7/70/IIT_Kharagpur_bg_pattern_swirl.png')"><DIV class="container project-mid1"><H2> Causes of food spoilage</H2><DIV class="row"><DIV class="col-lg-5 col-md-5 project-description">
                Food spoilage can simply be defined as deterioration in the
quality of a food item, making it undesirable and unfit for consumption.
Spoiled food does not only cause serious economic loss but can also lead to
critical health issues. As scary it may sound, a food item can be pretty easily
spoiled because of constant biotic and abiotic stresses around us - light,
heat, humidity, biochemical reactions and microbial actions, for example. While
abiotic factors do damage the flavor and texture of an item, what causes the
most harm is the growth of these tiny microbes (invisble to naked eye!) such as
bacteria, yeast and moulds on your food. It's the diverse repertoire of enzymes
produced (Braun and others 1999; Loureiro 2000; Ragaert and others 2007) by
these microorganisms that ruins your food and puts you at risk.
 			</DIV></DIV><H2> Quorum Sensing</H2><DIV class="row"><DIV class="col-lg-12 col-md-12 project-description">
              Bacteria use signaling molecules for inter and intracellular communication. This phenomenon of bacterial cell-to-cell communication called <I>quorum sensing</I>, is used to control a broad range of activities. For example, quorum sensing is used by the bacteria to modulate the expression of gene expression resulting in phenotypic changes and better adjustment to environmental conditions during growth. <REF:> Recent explosion in studies of such complex chemically coordinated system has made clear that the ability to communicate both within and between species is critical for bacterial survival and interactions in natural habitat. Bacteria often take many actions, the decision of which is based upon assesment of its local cell density.
              
              Cell-to-cell communication depends on the production of, secretion of, and response to small, diffusible signal molecules called <I>autoinducers</I>. These signal molecules are produced and secreted at a basal level during bacterial growth.
Their concentration in the environmental medium or matrix increases as the bacterial population
expands, and when it reaches a threshold level (quorum level) it induces phenotypic effects by regulating quorum-sensing dependent target gene expression <REF:>.  This phenomenon occurs without any external intervention and is also referred as auto induction.
 			</REF:></REF:></DIV></DIV><H2> Quorum Sensing in Food Spoilage</H2><DIV class="row"><DIV class="col-lg-12 col-md-12 project-description">
				Quorum sensing or cell-to-cell communication is employed by a diverse group of bacteria including those commonly associated with food to communicate with each other by producing the signaling molecules, autoinducers. It has been observed that the bacterial spoilage of some food products is influenced by quorum-sensing-regulated phenotypes. Several proteolytic, lipolytic, chitinolytic, and pectinolytic activities associated with the deterioration of foods are regulated by quorum sensing. Moreover, several types of signaling molecules (like AHL) have been detected in different spoiled food products. Milk and dairy products are easily susceptible to spoilage by psychrotrophic bacteria such as pseudomonads. These gram-negative bacteria produce extracellular proteinases, lipases, lecithinases, and glycosidases responsible for food spoilage<REF:>. The involvement of quorum sensing in regulation and expression of these biochemical substances was elucidated by a study in Serratia proteamaculans strain B5a. The inoculation of pasteurized milk with wild-type S.proteamaculans caused spoilage after 18 h of storage at room temperature, while the inoculation with a mutant having an inactivated sprI (inducer (HSL) producing gene) gene did not result in spoilage. However, the addition of 3-oxo-C6-HSL to milk inoculated with the sprI mutant caused its spoilage, implying the role of signaling molecules in the spoilage of milk.

                Food spoiling bacteria can produce AHLs even when they are present at a lower concentration in the food substrate. The AHLs (mainly 3-oxo-C6-HSL) have been produced by the inoculated (under conditions that simulated food environments of 5 ◦C, reduced oxygen, and 4% NaCl) and indigenous members of the <I>Enterobacteriaceae</I> at lower concentrations of 106 CFU/mL and 105 to 106 CFU/g, respectively.
                
                The pectinolytic activity of <I>Pseudomonadaceae</I> or <I>Enterobacteriaceae</I> (mostly <I>Erwinia spp.</I>) growing to high-cell densities (108 to 109 CFU g/L) in fruits and vegetables causes enzymatic browning, off-tastes, off-odors, and/or texture breakdown resulting in their spoilage (Liao 1989). Erwinia and Pseudomonas produce various pectinolytic enzymes, namely, pectin lyases, pectate lyase, polygalacturonase, and pectin methyl esterases, which are responsible for the spoilage of ready-to-eat vegetables, also produce a broad range of AHLs (mainly 3-oxo-C6-HSL and C6-HSL) (Rasch and others 2005).

 			</REF:></DIV></DIV><TABLE class="table table-striped table-hover"><CAPTION>Table: Bacterial food spoilage influenced by quorum-sensing-regulated phenotypes.</CAPTION><THEAD><TR><TD>Organism</TD><TD>Food product</TD><TD>Signal-dependent phenotype</TD><TD>Signaling molecules</TD><TD>References</TD></TR></THEAD><TDATA><TBODY><TR><TD>Pseudomonas fluorescens 395</TD><TD>Milk</TD><TD>Proteolytic milk spoilage</TD><TD>C4-HSL and 3OC8-HSL</TD><TD>Liu and others (2007)</TD></TR><TR><TD>Serratia proteomaculans strainB5a</TD><TD>Milk</TD><TD>Lipolytic and proteolytic milk spoilage</TD><TD>3-oxo-C6- HSL</TD><TD>Christensen and others  (2003)</TD></TR><TR><TD>Pseudomonas fluorescens</TD><TD>Milk</TD><TD>Proteolytic milk spoilage</TD><TD>L-HSL α-amino-γ -butyrolactones</TD><TD>Dunstall and others(2005)</TD></TR><TR><TD>Pseudomonas phosphoreum and Aeromonas spp</TD><TD>Cod fillets</TD><TD>Chitinolytic activity</TD><TD>3-hydroxy-C8-HSL</TD><TD>Flodgaard and others(2005)</TD></TR><TR><TD>Erwinia carotovora</TD><TD>Vegetables</TD><TD>Cellulolytic and proteolytic Spoilage</TD><TD>3-oxo-C6- HSL</TD><TD>Jones and others (1993)</TD></TR><TR><TD>Pectobacterium sp. A2JM</TD><TD>Bean sprouts</TD><TD>Pectinolytic and proteolytic Spoilage</TD><TD>3-oxo-C6- HSL</TD><TD>Rasch and others (2005</TD></TR><TR><TD>Serratia plymuthica RVH1</TD><TD>Vegetables</TD><TD>Chitinase and protease Activity</TD><TD>3-oxo-C6- HSL and C6-HSL</TD><TD>Van Houdt and others
(2007)</TD></TR><TR><TD>Hafnia alvei and Serratia spp.</TD><TD>Vaccum packed Meat</TD><TD>Proteolytic spoilage</TD><TD>N-3-oxohexanoyl HSL</TD><TD>Bruhn and others
(2004)</TD></TR><TR><TD>Pseudomonas spp.</TD><TD>Meat</TD><TD>Biofilm formation and proteolytic spoilage</TD><TD>AHLs</TD><TD>Jay and others (2003)</TD></TR><TR><TD>Photobacterium phosphoreum and Aeromonas spp.</TD><TD>Cod fillets</TD><TD>Chitinolytic spoilage</TD><TD>3-hydroxy-C8-HSL</TD><TD>Flodgaard and others
(2005)</TD></TR></TBODY></TDATA></TABLE><H2> Mechanism (the LuxR/LuxI system)</H2><DIV class="row"><DIV class="col-lg-12 col-md-12 project-description">
				The typical LuxR/LuxI system was first discovered in <I>Vibrio fischeri</I> as the first quorum sensing circuit. There are five luciferase structural genes (luxCDABE) and two regulatory genes (luxR and luxI) required for quorum sensing–controlled light emission in V. fischeri. The genes are arranged in two adjacent but divergently transcribing units. luxR is transcribed to the left, and the luxICDABE operon is transcribed to the right. The LuxI protein (square) is responsible for synthesis of the HSL autoinducer N-(3-oxohexanoyl)-homoserine lactone (hexagons). As the cell population density increases, the concentration of the autoinducer increases both intra and extracellularly. At a critical autoinducer concentration (1-10 ug/ml), the LuxR protein (shown with a circle) binds the autoinducer. This LuxR-autoinducer complex then binds to the luxICDABE promoter and activates transcription of this operon. This action results in an exponential increase in autoinducer synthesis via the increase in transcription of luxI and an exponential increase in light production via the increase in transcription of luxCDABE. The LuxR-autoinducer complex also binds at the luxR promoter, but in this case the complex represses the transcription of luxR, thereby compensating the positive action at the luxICDABE promoter. The oval represents a bacterial cell.

 			</DIV></DIV><H2> Our Project</H2><DIV class="row"><DIV class="col-lg-12 col-md-12 project-description">
              Does the doctor's heavy fee and prescription tabloid adds more to your frustration than what is brought by the pathological state of affairs from food infection? If only you could have been a bit more careful, everything would have been fine! What does it take to find the bad food - a sniff, small bite, a drop on your tongue, or the fundamental theorem of projecting last time’s experience over two more days! &quot;Old kitchen's wives tales are not always helpful are they? But what alternative do we have!”, you say, &quot;quick and efficient, while not being too expensive&quot;. That is why you need to pay attention to what follows!
              
iGEM IIT Kharagpur targeted the ancient cause of microbial animosity among mankind i.e. microbial spoilage of food and subsequent pathological disorder that follows. Out of the long list of possible candidates, we chose to detect bacteria which releases AHL(Acyl Homoserine Lactone) molecules as a quorum signal as a qualitative idea of the microbial population in the system. As explained in project background, quorum signal molecules regulate expression of certain genes (different in every organism), light illuminating peptides in Vibrio fischerie for example. We decided to develop a modified strain from <I>E.coli</I> DH5-alpha strain. The modification includes presence of a singular plasmid which consists of luxR gene put under a constitutive promoter, a luxpR sequence which is a luxR-AHL complex inducible promoter site. Our idea is based around the premise that AHL is constitutively produced by bacteria and as their population grows the AHL concentration grows. Since AHL is freely diffusible through cell membrane, its intracellular concentration is equal to extracellular concentration. In presence of luxR protein to which it has a binding affinity, it forms an activating complex which activates the gene downstream of the luxPR sequence. The gene which we have introduced is CrtEBI, a cluster of three protein coding regions encoding CrtE, CrtB, and CrtI. These proteins convert FPP (farnesyl pyrophosphate), a molecule produced naturally by E.coli  to red colored pigment called lycopene. This red signal acts as a red alert against the spoiled food.

Our wet lab team decided to make use of three biobrick assemblies (as mentioned
above) available in iGEM part repository. The final biobrick planned is to have
parts from all three plasmids (2 genes and 1 promoter). 

 			</DIV></DIV></DIV><DIV class="container-fluid footer1"><DIV class="footer-content"><DIV class="row"><DIV class="col-lg-4 col-md-4 col-sm-4" style="margin-top:34px;"><A href="">Department of Biotechnology 
							IIT Kharagpur</A></DIV><DIV class="col-lg-4 col-md-4 col-sm-4"><H3>Connect</H3><BUTTON type="button" class="btn btn-default mail-button"><A href="mailto:igemiitkharagpur15@gmail.com?Subject=Hello%20again" target="_top">Mail Us</A></BUTTON></DIV></DIV></DIV></DIV></DIV><DIV class="container-fluid footer2"><DIV class="row"><DIV class="col-lg-12"><DIV class="footer-cr">
						Team IIT_KHARAGPUR
					</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>