Team:Tianjin/Note/R-R
TEAM TIANJIN
PLASTERMINATOR
Toggle navigation
OVERVIEW
PROJECTS
OUTLINE
CONSORTIUM
PROTEIN ENGINEERING
R-R SYSTEM
PROOF
MODEL
RESULTS
ACHIEVEMENTS
DEMONSTRATE
PARTS
FUTURE WORK
PEOPLE
TEAM
ATTRIBUTION
OUTREACH
HUMAN PRACTICE
COLLABORATIONS
SAFETY
NOTES
NOTEBOOK
PROTOCOL
Team Tianjin-Attribution
Notes for R-R System
Week1(8/24/2016-8/30/2016)
We used PCR to amplify the CpxR promoter and RFP gene from plasmid pUC57, and then we recycled the amplified fragment from the agarose gel. Then we used Xba1 and Pst1 enzymes to cut the plasmid pUC19 and CpxR-RFP fragment.
We linked the cut plasmid and CpxR-RFP fragment together and transformed the recombinant plasmid to E.coli. We used PCR to amplify the PETase gene and then recycled them from the agarose gel.
We cultured the grown-up E.coli which had been transformed into recombinant pUC19 into liquid LB+Amp culture medium overnight.
We isolated the recombinant plasmid from the E.coli cultured last night. Then we use Xba1 and Pst1 enzyme to cut the plasmid to verify the plasmid was successfully constructed. The result was we succeeded.
We cut the recombinant plasmid pUC19 with enzyme EcoR1 and Sac1 and then we recycled it from the agarose gel. We stored the recycled product in -30℃ in order to wait for the PETase gene transformed into it.
In order to verify the inclusion body sensing effects of CpxR promoter, we selected a colony of E.coli with recombinant plasmid pUC19 and cultured them in 37℃ for 6 hours and then rose the temperature to 42℃ and culture it overnight.
The result of the verification experiment last night was unsuccessful for there was only ultralow red fluorescence was detected, which was considered the basic expression of RFP.
We redesigned the experiment and set 3 groups:
1. E.coli with standard RFP gene from our own laboratory.
2. E.coli with our recombinant plasmid pUC19 and we cultured them in 37℃ all through.
3. E.coli with our recombinant plasmid pUC19 and we first cultured them in 37℃ and after 6 hours transferred them to 42℃.
The result was still unsuccessful for the 2nd and 3rd group showed ultralow red fluorescence and only 1st group showed high enough red fluorescence.
We redesigned the experiment again. We decided to transformed the recombinant plasmid pUC19 and pET21a which was from the protein modification group and had PETase gene in it into E.coli at the same time.
We cut the pUC57 with enzyme Xba1 and Pst1 and recycled the skeleton part from the agarose gel.
Show More
Week2(8/31/2016-9/6/2016)
We linked the remained cut CpxR-RFP fragment into the skeleton and then transformed the recombinant pUC57 and the pET21a into E.coli at the same time.
The transformation last night turned to be a failure. We tried it again.
The transformation last day seemed to be successful for the colonies were visible in LB+Amp plate. However, we use PCR to verify and it turned out that the fragment had not been linked into the plasmid.
We finally gave up the former design and decided to link the PETase gene into the plasmid pUC19. However, we did not have the key enzyme Sal1 so we started to construct the TPA positive feedback system.
We first prepared the TPA standard solution (5g/L) for further use. Then we use PCR to amplify the TPA-sensing leader sequence, PGK1 promoter, CYC1 terminator, RFP gene, TPA regulation protein gene (tpaR), TPA transporting protein gene (tpaK). Then we cut the fragments above and plasmid pRS413, pRS415, and pYES2 with corresponding enzymes and recycled the fragments from agarose gel.
We linked the fragments together by this way:
1. pYES2-leader-PGK1-RFP.
2. pRS413-PGK1-tpaK-CYC1.
3. pRS415-PGK1-tpaR-CYC1
Then we used PCR to verify the success and all of the plasmids were correctly constructed. Then we transformed the there plasmids into Saccharomyces cerevisiae.
The key enzyme Sal1 arrived and we isolate the plasmid pET21a. Then we use BamH1 and Sal1 to cut both plasmid and PETase gene, then linked them together and transformed the recombinant plasmid into E.coli.
Show More
Week3(9/7/2016-9/13/2016)
We cultured the transformed E.coli and isolated the plasmid. Then we use PCR to amplify the whole fragment in pET21a from T7 promoter to T7 terminator. Then we recycled this fragment from agarose gel.
The transformed Saccharomyces cerevisiae had grown to visible colony in Sc-Ura-Leu-His plate. Then we use colony PCR to verify the plasmids had been transformed into the cells. The result is successful so that we streaked more plates.
We cut the T7 promoter-PETase gene-T7 terminator fragment with enzymes EcoR1 and Sac1. Then we linked it to the already cut plasmid pUC19 (cut in August 28th). Then we transformed the recombinant plasmid into E.coli.
We cultured the transformed Saccharomyces cerevisiae into Sc-Ura-Leu-His culture medium in 30℃. We added TPA standard solution in this way:
1. Group 1: did not add TPA.
2. Group 2: added 1000μL TPA standard solution.
3. Group 3: added 100μL TPA standard solution.
4. Group 4: added 10μL TPA standard solution.
5.Group 5: added 1μL TPA standard solution.
We cultured the transformed E.coli into LB+Amp culture medium. Then add 1.5μL IPTG to induce the expression of PETase gene.
We first detected the red fluorescence of E.coli, however, the experiment group had almost no increase of red fluorescence relative to control group. We changed the induction wavelength and scan the whole wavelength of emission, but we did not receive any result we expected.
The TPA positive feedback system seemed to have minor effection for there were a little increment of red fluorescence of the 5th group relative to the 1st one.
We doubted that it might be the RFP in the kit was useless. We isolated the pET21a and used PCR to amplify the RFP gene.
We cut the RFP gene and pET21a with enzymes Xba1 and Sac1, then we linked them and transformed it into E.coli.
We cultured the transformed E.coli and added IPTG to induce the expression of RFP, and this time the red fluorescence was clear enough that could be seen directly.
Show More
Week4(9/14/2016-9/20/2016)
We started to construct another regulation way, the E.coli lysis regulation pathway. We first used colony PCR to obtain the ddpX gene from the E.coli genome and recycled the ddpX from the agarose gel.
We found that there was no enzyme cleavage site between the CpxR promoter and RFP gene in the part we use. We had to design the primers and amplified the CpxR promoter by PCR.
We used PCR to amplify the CpxR promoter. Then we recycled it from agarose gel.
We cut the CpxR promoter with enzymes Xba1 and BamH1, ddpX gene with enzymes BamH1 and EcoR1, first batch of pET21a with Xba1 and EcoR1, second batch of pET21a with BamH1 and EcoR1.
Then we linked these fragment in the following two ways:
1. pET21a-CpxR-ddpX.
2. pET21a-ddpX.
Show More
Week5(9/21/2016-9/27/2016)
We used PCR to amplify the whole fragments in pET21a (from CpxR to T7 terminator). However, the band in the agarose gel was disperse so that we were unable to recycle it.
We used colony PCR to verify if the pET21a had been correctly constructed, the result is yes.
We changed the DNA polymerase and annealing temperature several times and redid the PCR, however, the disperse band were always existed.
We cultured the E.coli transformed into the pET21a-ddpX fragment and detect the OD600 in order to verify the lysis effection of ddpX.
Considering the pYES2 is multicopy plasmid so that the copy number would affect the RFP expression level, we decided to change the pYES2 to single-copy plasmid pRS416. Since the pRS416 does not have terminator in its backbone, we used PCR to amplify the CYC1 terminator from plasmid pYES2.
We cut the pYES2 with enzyme Hind3 and EcoR1, CYC1 with EcoR1 and Sal1, pRS416 with Hind3 and Sal1. Then we linked the three part together.
We transformed the three plasmids into Saccharomyces cerevisiae together.
The new primers using to amplify the CpxR-ddpX-T7 terminator fragment arrived and we redid the PCR. However, the disperse band was still existed.
The transformation of Saccharomyces cerevisiae turned out to be a failure because no colony was found on the Sc-Ura-Leu-His plate.
Show More
Week6(9/28/2016-10/2/2016)
We redid the inclusion body reporting experiment, and this time we directly observed the color of bacterial after centrifugation (12000rpm, 1min). The group with PETase gene and CpxR-RFP fragment showed the deepest red.
We redid all the constructing process of ddpX induced cell lysis system and finally we obtained the positive result.
We test the ddpX induced cell lysis system by measuring the OD600 of culture medium.
Show More
