Team:Tianjin/Protocol
TEAM TIANJIN
PLASTERMINATOR
Toggle navigation
OVERVIEW
PROJECTS
OUTLINE
CONSORTIUM
PROTEIN ENGINEERING
R-R SYSTEM
PROOF
MODEL
RESULTS
ACHIEVEMENTS
DEMONSTRATE
PARTS
FUTURE WORK
PEOPLE
TEAM
ATTRIBUTION
OUTREACH
HUMAN PRACTICE
COLLABORATIONS
SAFETY
NOTES
NOTEBOOK
PROTOCOL
Team Tianjin-Attribution
General Protocols
1.Plasmid Extraction
We use the TIANprep Mini Plasmid Kit made by TIANGEN Biotech Co.,Ltd. to extract plasmid. Here is the protocol.
Add ethanol (96-100%) to Buffer PW before use, check bottle tag for the adding volume.
1. Column equilibration: Place a Spin Column CP3 in a clean collection tube, and add 500 μl Buffer BL to CP3. Centrifuge for 1 min at 12,000 rpm (~13,400 × g) in a table-top microcentrifuge. Discard the flow-through, and put the Spin Column CP3 back into the collection tube. (Please use freshly treated spin column).
2. Harvest 1-5 ml bacterial cells in a microcentrifuge tube by centrifugation at 12,000 rpm (~13,400 × g) in a conventional, table-top microcentrifuge for 1 min at room temperature (15-25°C), then remove all traces of supernatant by inverting the open centrifuge tube until all medium has been drained (For large volume of bacterial cells, please harvest to one tube by several centrifugation step.)
3. Re-suspend the bacterial pellet in 250 μl Buffer P1 (Ensure that RNase A has been added). The bacteria should be resuspended completely by vortex or pipetting up and down until no cell clumps remain.
Note: No cell clumps should be visible after resuspension ofthe pellet, otherwise incomplete lysis will lower yield and purity. 4. Add 250 μl Buffer P2 and mix gently and thoroughly by inverting the tube 6-8 times.
Note: Mix gently by inverting the tube. Do not vortex, as this will result in shearing of genomic DNA. If necessary, continue inverting the tube until the solution becomes viscous and slightly clear. Do not allow the lysis reaction to proceed for more than 5 min. If the lysate is still not clear, please reduce bacterial pellet.
5. Add 350 μl Buffer P3 and mix immediately and gently by inverting the tube 6-8 times. The solution should become cloudy. Centrifuge for 10 min at 12,000 rpm (~13,400 × g) in a table-top microcentrifuge.
Note: To avoid localized precipitation, mix the solution thoroughly, immediately after addition of Buffer P3. If there is still white precipitation in the supernatant, please centrifuge again.
6. Transfer the supernatant from step 5 to the Spin Column CP3 (place CP3 in a collection tube) by decanting or pipetting. Centrifuge for 30-60 s at 12,000 rpm (~13,400 × g). Discard the flow-through and set the Spin Column CP3 back into the Collection Tube.
7. (Optional, actually we hardly ever use) Wash the Spin Column CP3 by adding 500 μl Buffer PD and centrifuge for 30-60 s at 12,000 rpm (~13,400 × g). Discard the flow-through and put Spin Column CP3 back to the collection tube.
This step is recommended to remove trace nuclease activity when using endA+ strains such as the JM series, HB101 and its derivatives, or any wild-type strain, which have high levels of nuclease activity or high carbohydrate content.
8. Wash the Spin Column CP3 by adding 600 μl Buffer PW (ensure that ethanol (96%-100%) has been added) and centrifuge for 30-60 s at 12,000 rpm (~13,400 × g). Discard the flow-through, and put the Spin Colum CP3 back into the Collection Tube.
9. Repeat Step 8.
10. Centrifuge for an additional 2 min at 12,000 rpm (~13,400 × g) to remove residual wash Buffer PW.
Note: Residual ethanol from Buffer PW may inhibit subsequent enzymatic reactions. We suggest open CP3 lid and stay at room temperature for a while to get rid of residual ethanol.
11. Place the Spin Column CP3 in a clean 1.5 ml microcentrifuge tube. To elute DNA, add 50-100 μl Buffer EB to the center of the Spin Column CP3, incubate for 2 min, and centrifuge for 2 min at 12,000 rpm (~13,400 × g).
Note: If the volume of eluted buffer is less than 50 μl, it may affect recovery efficiency. The pH value of eluted buffer will have some influence in eluting; Buffer EB or distilled water (pH 7.0-8.5) is suggested to elute plasmid DNA. For long-term storage of DNA, eluting in Buffer EB and storing at -20°C is recommended, since DNA stored in water is subject to acid hydrolysis. Repeat step 11 to increase plasmid recovery efficiency.
Show More
2.DNA Purification
We use the TIANquick Midi Purification Kit made by TIANGEN Biotech Co.,Ltd. to purify the DNA products of PCR and restriction endonuclease cutting. Here is the protocol.
Add ethanol (96-100%) to Buffer PW before use (see bottle label for volume).
1. Column equilibration: add 500 μl Buffer BL to the Spin Column CB2 (put Spin Column CB2 into a collection tube). Centrifuge for 1 min at 12,000 rpm (~13,400 × g). Discard the flow-through, and then place Spin Column CB2 back into the collection tube (please use freshly treated spin column).
2. Add 5 volumes of Buffer PB to 1 volume of the PCR reaction or enzymatic reaction and mix. It is not necessary to remove mineral oil or kerosene.
Note: For example, add 250 μl Buffer PB to 50 μl PCR reaction (not including oil).
3. Transfer the mixture to the Spin Column CB2, incubate at room temperature (15-25°C) for 2 min. Centrifuge for 30-60 s at 12,000 rpm (~13,400 × g) in a table-top microcentrifuge. Discard the flow-through, and then place Spin Column CB2 back into the same collection tube.
Note: The maximum loading volume of the column is 800 μl. For sample volumes greater than 800 μl simply load again.
4. Add 600 μl Buffer PW (ensure that ethanol (96-100%) has been added) to the Spin Column CB2 and centrifuge for 30-60 s at 12,000 rpm (~13,400 × g). Discard the flow-through, and place Spin Column CB2 back in the same collection tube.
Note: If the purified DNA is used for the subsequent salt sensitive experiments, such as ligation or sequencing experiment, it is suggested to stand for 2-5 min after adding Buffer PW, and then centrifuge.
5. Repeat step 4.
6. Centrifuge at 12,000 rpm (~13,400 × g) for 2 min to remove residual Buffer PW. Discard the flow-through, and allow the column to air dry with the cap open for several minutes to dry the membrane.
Note: Residual ethanol from Buffer PW may inhibit subsequent experiment (enzymatic or PCR reactions).
7. Place the Spin Column CB2 in a clean 1.5 ml microcentrifuge tube. Add 30-50 μl Buffer EB to the center of membrane, incubate for 2 min, and centrifuge for 2 min at 12,000 rpm (~13,400 × g).
Note: If the volume of eluted buffer is less than 30 μl, it may affect recovery efficiency. The pH value of eluted buffer will have big influence in eluting; distilled water (pH 7.0-8.5, adjusted with NaOH) is suggested to elute plasmid DNA, pH<7.0 will decrease elution efficiency. For long-term storage of DNA, eluting in Buffer EB and storing at -20°C is recommended, since DNA stored in water is subject to acid hydrolysis. Repeat step 7 to increase plasmid recovery efficiency.
Show More
3.Agarose Gel Electrophoresis Products Recycling
We use the TIANgel Midi DNA Purification Kit made by TIANGEN Biotech Co.,Ltd. to recycle the DNA products from the agarose gel. Here is the protocol.
Add ethanol (96-100%) to Buffer PW before use (see bottle label for volume).
1. Column equilibration: add 500 μl Buffer BL to the Spin Column CA2 (put Spin Column CA2 into a collection tube). Centrifuge for 1 min at 12,000 rpm (~13,400 × g) in a table-top microcentrifuge. Discard the flow-through, and put Spin Column CA2 back into the collection tube (please use freshly treated spin column).
2. Cut the DNA fragment from agarose gel with a clean, sharp scalpel. Weigh the gel slice in a clean tube.
3. Add equivalent volume of Buffer PN to the gel (If the gel is 0.1 g, it is defaulted to be 100 μl, then add 100 μl Buffer PN). Incubate at 50°C by inverting up and down the tube until the agarose gel dissolves completely. If the agarose gel does not dissolve completely, incubate for longer period or add additional Buffer PN until all the agarose gel dissolved completely (If the agarose gel is too large, please cut the agarose gel into several pieces in advance).
Note: If DNA fragment is <300 bp, it is recommended to add isopropanol which is 1/2 volume of Buffer PN to the agarose gel sample after the gel completely dissolved. Cooling the solution at room temperature (15-25°C) and then add the solution to Spin Column CA2 since silica membrane of the column adsorbs DNA best at room temperature.
4. When the gel dissolved completely and the solution temperature turns to room temperature (15-25°C), transfer the mixture to the Spin Column CA2 (put Spin Column CA2 into a collection tube). Let the column stand for 2 min at room temperature (15-25°C), then centrifuge for 30-60 s at 12,000 rpm (~13,400 × g) in a table-top microcentrifuge. Discard the flow-through; place the Spin Column CA2 back into the collection tube again.
Note: The maximum loading volume of the column is 800 μl. For sample volumes greater than 800 μl simply load again.
5. Wash the Spin Column CA2 with 600 μl Buffer PW (ensure that ethanol (96-100%) has been added) and centrifuge for 30-60 s at 12,000 rpm (~13,400 × g). Discard the flow-through and place the Spin Column CA2 back into the collection tube.
Note: If the purified DNA is used for the salt sensitive experiments, such as direct sequencing and blunt-ended ligation, let the column stand for 2-5 min after adding Buffer PW, and then centrifuge.
6. Repeat Step 5.
7. Place the Spin Column CA2 back to the collection tube and centrifuge at 12,000 rpm (~13,400 × g) for 2 min to remove residual wash buffer. Discard the flow-through, and place column with the cap open for several minutes to air dry the membrane.
Note: Residual ethanol from Buffer PW will influence the subsequent enzymatic reaction (enzyme digestion, PCR etc).
8. Transfer the Spin Column CA2 to a clean 1.5 ml microcentrifuge tube. Add appropriate volume of Buffer EB to the center of the membrane, incubate at room temperature (15-25°C) for 2 min, then centrifuge at 12,000 rpm (~13,400 × g) for 2 min.
Note: The elution volume should not be less than 30 μl since smaller volume will affect recovery efficiency. The pH value of eluted buffer will affect eluting. If purified DNA is used for sequencing, it is recommended to choose ddH2O (pH 7.0-8.5) to elute DNA, pH<7.0 will decrease the elution efficiency. Obtained DNA should be stored at -20°C to prevent degradation. Buffer (10 mM Tris-Cl, pH 8.0) could also be used for DNA elution. For higher yield, pipette the eluate to the center of the membrane again, incubate 2 min and centrifuge at 12,000 rpm (~13,400 × g) for 2 min.
Show More
4.Agarose Gel Electrophoresis
1. Preparation of TAE buffer: Add 242g Tris, 37.2g Na2EDTA·2H2O, 800mL ddH2O, after all the solute dissolve, add 57.1mL acetic acid and add ddH2O to make the total volume 1L.
2. Preparation of sample: Add DNA loading buffer to the DNA solution according to the dilution ratio of particular buffer.
3. Preparation of agarose gel: Add agarose powder 10g/L, 1×TAE buffer 100mL (variable, according to need), heat the mixture by microwave oven until the agarose was dissolved. After the solution cool down to touchable temperature, add 50-100μL/L Goldenview Nucleic acid dye to the solution. Then pour the solution to the gel mould with gel comb inserted and wait for its concretion.
4. Add samples to the gel pore: After the formation of gel, pull out the gel comb and take the gel out of the mould. Immerge the gel with 1×TAE buffer in the electrophoresis chamber. Using pipette to add marker and samples to different pore. (The content of pore depends on the gel comb, there are 3 kinds of volume, 10μL, 20μL, and 50μL) Do not stick the bottom and side of gel pore to prevent the leakage.
5. Turn on the electrical source to start the electrophoresis, the voltage is set at 150-160V and the electrophoresis time is set at 8-12min.
6. After the electrophoresis process end, the gel is observed under blue light or ultraviolet.
Show More
5.Restriction Endonuclease Digestion
Prepare the system:
Total volume: 50μL
Restriction endonuclease: 2μL respectively
10×Cut Smart Buffer: 5μL
DNA to be cut: 30μL
ddH2O: 13μL
Reaction time: 2h
Reaction temperature: 37℃
Show More
6.DNA Fragments Ligation
Prepare the system:
Total volume: 20μL
T4 DNA ligase: 1μL
5×Ligase Buffer: 4μL
DNA to be linked: (c1V1/L1): (c2V2/L2)=5:1. (c1: Concentration of cut DNA fragments; c2: Concentration of cut plasmid; V1: Volume of cut DNA fragments; V2: Volume of cut plasmid; L1: Length of cut DNA fragments; L2: Length of cut plasmid)
ddH2O: add to make the total volume 20μL
Reaction time: 2h
Reaction temperature: 22℃
Show More
7.PCR
Prepare the system:
Total volume: 50μL
Q5 DNA Polymerase: 0.5μL
5×Q5 DNA Polymerase Buffer: 10μL
Template: 1μL
dNTP: 1μL
Primers: Sense Primer and Anti-sense Primer, respectively 2.5μL
ddH2O: 32.5μL
Cycles: 25-35
Pre-denaturation: 98℃,30s
Denaturation: 98℃,5-10s
Annealing: Depend on the primers, generally 45-65℃,10-30s
Extension: 72℃,20-30s/kb
Fully extension: 72℃,2min
Product Storage: 4℃
Note: Different DNA polymerase has different protocol, this is only the case of Q5 DNA polymerase.
Show More
8.Colony PCR
Prepare the system:
Total volume: 20μL
Colony PCR System: 2×Taq Master Mix (Dye added) 10μL
Primers: Sense Primer and Anti-sense Primer, respectively 1μL
ddH2O: 8μL
select single colony with stick and immerse the stick into the system for 1 min. Preparing the bacterial liquid and then add 1μL of it is also feasible.
Cycles: 25-35
Pre-denaturation: 94℃,5min
Denaturation: 94℃,30s
Annealing: Depend on the primers, generally 45-65℃,20-60s
Extension: 72℃, depend on the length of amplified fragment, generally 20-60s
Fully extension: 72℃,7min
Product Storage: 4℃
Note: Preparation of bacteria liquid Prepare the system:
Total volume: 50μL
ddH2O: 50μL
select single colony with stick and immerse the stick into the system for 1 min
Cycles: 3-4
Pre-denaturation: 99℃,4min
Renaturation: 4℃,4min
Product Storage: 4℃
Show More
9.TransT1 Transformation
1. Mix 50μL competent E.coli cell with 10μL plasmids, and place them in ice bath for 30 min.
2. Heat shock at 42℃ for 30 s.
3. Put the system back on ice for 2 min.
4. Add 500 μl of LB without antibiotics and incubate at 37 ℃ for 30-60 min with the shaking speed of 200rpm.
5. Centrifuge the culture medium at 4000 rpm for 3min.
6. Discard the supernatant and resuspend the sediment, use this resuspending liquid to spread the plate with antibiotics.
Show More
Protocols for Microbial Consortia
1.16s-DNA
Prepare the system:
Total volume: 20μL
Colony PCR System: 2×Taq Master Mix (Dye added) 10μL
ddH2O: 8.2μL
primer: 0.4μL each
bacteria liquid:1μL
The primer could copy the special 16s-DNA from this kind of bacteria.
The bacteria liquid is pretreatment.
Cycles: 25-35
Pre-denaturation: 94℃,2min
Denaturation: 94℃,30s
Annealing: Depend on the primers, generally 45-65℃,20-60s
Extension: 72℃, depend on the length of amplified fragment, generally 20-60s
Fully extension: 72℃,4min
Product Storage: 4℃
Show More
2. Preparation of W medium
Initial medium
KH2PO4: 1.7 g/L
Na2HPO4: 9.8 g/L
(NH4)2SO4: 1.0 g/L
MgSO4·7H2O: 0.1 g/L
FeSO4·7H2O: 0.95 mg/L
MgO: 10.75 mg/L
CaCO3: 2.0 mg/L
ZnSO4·7H2O: 1.44 mg/L
CuSO4·5H2O: 0.25 mg/L
CoSO4·7H2O: 0.25 mg/L
H3BO3: 0.06 mg/L
HCl: 51.3ml/L
We modify initial formula for convenience, and modified formula is shown as following table:
KH2PO4: 1.7 g/L
Na2HPO4: 9.8 g/L
(NH4)2SO4: 1.0 g/L
NH4Cl: 0.865 g/L
MgSO4·7H2O: 0.1 g/L
MgCl2: 0.025 g/L
Mother solution: 1 ml/L
Formula of mother solution
FeSO4·7H2O: 0.95 g/L
CoCl2·6H2O: 0.236 g/L
CaCl2: 2.22 g/L
ZnSO4·7H2O: 1.44 g/L
CuSO4: 0.16 g/L
H3BO3: 0.06 g/L
Show More
Protocols for Protein Engineering
1.Preparation of the optimized yeast culture medium
Glucose:22g/L
Yeast Nitrogen Base:6.7g/L
Dispense mixture:1.224g/L
Agar:15g/L
His/Trp/Leu mixed solution:10ml/L
Show More
2.pNPA assay
A Buffer
Glycerol:400μl
Gal:1000μl
SC-Ura:3600μl
B Buffer:
10mM pNPA Buffer:50μl
PBS(pH=7.4):98μl
Show More
Protocols for Cell-Free Protein Synthesis
1.Cell-free protein synthesis system(50 µL)
ddH2O:7.9µL
Feeding buffer : 25µL
Mg2+ solution  : 1.1µL
Gene( plasmid as template) : 1µL
Lysate   : 15µL
Show More
2.Preparation of pNPA degradation reaction system
Substrate: 0.2mM pNPA water solution (0.0362g pNPA dissolving in ddH2O)
Reaction system: 10 times diluted Enzyme solution(unpurified) mixed with the equal volume substrate. (pH=7)
After static reaction at 39℃ for 8 hours, we detected the characteristic adsorption peak of the product ,pNP, which has no other characteristic adsorption peak except in 400nm.
Show More
3.Preparation of PET degradation reaction system
Reaction system: PET was put into 20 times diluted Enzyme solution(unpurified).(pH=7)
After static reaction at 39℃ for 2 days, we detected the characteristic adsorption peak of the product ,MHET, which has no other characteristic adsorption peak except in 260nm. And detect it every other day in the future 3-4 days.
Show More
Protocols for Cyanobacteria
1.Restriction Endonuclease Cutting and Dephosphorylation
Prepare the system:
Total volume: 100μL
Restriction endonuclease : 3μL respectively
10×fastdigest Buffer: 10μL
Plasmids to be cut: 2μg
ddH2O: add to make the total volume 100μL
fast AP: add when reacting for 1.5h
Total reaction time: 2h
Reaction temperature: 37℃
Show More
2.Phosphorylation
Prepare the system:
Total volume: 20μL
ATP:2μL
10×Buffer A:2μL
PNK:1μL
ddH2O: add to make the total volume 200μL
Reaction time:0.5h
Reaction temperature: 37℃
Show More
3.Ligation of Phosphorylated DNA Fragments and Dephosphorylated Plasmids
Prepare the system:
Total volume: 20μL
DNA fragment: 3μL
Plasmids to be ligase : 1μL
T4 Buffer: 2μL
PEG4000:2μL
T4 DNA ligase: 1μL
PCR H2O:11μL
Reaction time: >1h
Reaction temperature: 22℃
Show More
4.A-Tailing Reaction
Prepare the system:
Total volume: 20μL
Talling-A reaction Buffer: 4μL
DNA fragment: 15μL
Taq DNA Polymerase: 1μL
Reaction time: >0.5h
Reaction temperature: 72℃
Show More
5.TA cloning
Prepare the system:
Total volume: 10μL
5×Buffer:2μL
Vector:0.5μL
DNA fragment: 7μL
T4 DNA ligase: 0.5μL
Reaction time: 0.5-1h
Reaction temperature: 22℃
Show More
6.Electroporation
1.Cultures (OD630=0.6–0.7) which grow approaching the plateau are harvested.
2.According to the quantity of transformation, one plasmid need 8-10mL cultures. Cells are harvested by centrifugation at3500rpm for 13min at 4°C.
3.Discard the supernatant, resuspend with 20mL sterilized ultrapure water. Then, centrifuge at 3500rpm for 13min at 4°C.
4.Repeat step3.
5.Add 200mL sterilized ultrapure water for each plasmid.
6. Plasmid(2μg) is added to 200mL condensed cultures in a sterilized EP tube. Then transfer the mixture into a 2mm-gap electroporation cuvette.
7.Place them on the ice for 1-3min.
8. Bacteria are pulsed at 2.5kV(1.8kV).
9. 200μL BG-11 medium(without antibiotics) is added into the electroporation mixture and incubate for 5h.
10. Use the liquid to spread the plate with antibiotics.
Show More
7.Preparation of BG-11 medium
NaNO3: 1.5 g/L
K2HPO4·3H2O: 0.04 g/L
MgSO4·7H2O: 0.075 g/L
CaCl2·2H2O: 0.036 g/L
Citric acid: 0.006 g/L
Ferric ammonium citrate: 0.006 g/L
EDTA(disodium magnesium salt): 0.001 g/L
Na2CO3: 0.02 g/L
Trace metal mix A5+Co: 1 mL/L
Deionized water: 1000 mL
pH after autoclaving and cooling: 7.4
Show More
