<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tianjin_Note_Consortium2 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tianjin/Note/Consortium2</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>TEAM TIANJIN</TITLE></P><DIV class="container-fluid"><DIV class="navbar-header"><DIV class="nameTeamTianjin">PLASTERMINATOR</DIV><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:Tianjin" id="Overview">OVERVIEW</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Projects">PROJECTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Description" id="Outline">OUTLINE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Experiment/Consortium" id="Consortium">CONSORTIUM</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Experiment/Protein_Engineering" id="ProteinEngineering">PROTEIN ENGINEERING</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Experiment/R-R" id="RRSystem">R-R SYSTEM</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Proof" id="Proof">PROOF</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Model" id="Model">MODEL</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Results">RESULTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Achievements" id="Achievements">ACHIEVEMENTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Demonstrate" id="Demonstrate">DEMONSTRATE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Parts" id="Parts">PARTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Future" id="FutureWork">FUTURE WORK</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="People">PEOPLE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Team" id="Team">TEAM</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Attributions" id="Attribution">ATTRIBUTION</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Outreach">OUTREACH</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Community/Overview" id="HumanPractice">HUMAN PRACTICE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Collaborations" id="Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Safety" id="Safety">SAFETY</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Notes">NOTES</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Notes" id="Notebook">NOTEBOOK</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Protocol" id="Protocol">PROTOCOL</A></LI></DIV></DIV><P><TITLE>Team Tianjin-Attribution</TITLE></P><DIV class="wrapper-white"><DIV id="page" class="hfeed"><DIV id="main"><DIV id="content" class="home blog single-author one-column content" role="main"><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><HEADER class="entry-header"><DIV class="entry-title" align="center">Notes for Microbial Consortium</DIV></HEADER><H1 class="entry-title">Week1(7/24/2016-7/30/2016)</H1><DIV class="entry-content"><DIV class="note-content"><LI><H1 style="font-size:135%">Optimization of Culture Conditions</H1><B><H1 style="font-size:108%">  Jul.26th</H1><DIV style="padding-left:32px;">Prepare M9 medium with TPA and culture <I>Pseudomonas putida KT2440</I> at 30℃ &amp; 200 rpm, checked the bacterial concentration at OD<SUB>600</SUB>.</DIV><B><H1 style="font-size:108%">  Jul.27th</H1><DIV style="padding-left:32px;">1.Extract 10 ml LB medium to 5 test tubes and add 0μl, 50μl,, 100μl,, 250μl,, 500μl EG to them, respectively.
2.Add 5μl bacteria solution of <I>Pseudomonas putida KT2440</I> to five test tubes above.
3.Culture them at 30℃ &amp; 200 rpm, checked the bacterial concentration at OD<SUB>600</SUB>. </DIV><B><H1 style="font-size:108%">  Jul.28th</H1><DIV style="padding-left:32px;">Add 6g TPA, 2.9gNaOH and 1.2g glucose to M9 medium, and culture them in the improved M9 medium and M9 medium of Jul.26 at 30℃ &amp; 200 rpm, checked the bacterial concentration at OD<SUB>600</SUB> and detected the concentration of TPA by UV at OD<SUB>242</SUB>.</DIV><B><H1 style="font-size:108%">  Jul.30th</H1><DIV style="padding-left:32px;">Cultured different bacteria in M9 medium with sodium terephthalate and different concentration of glucose at 30℃ &amp; 200 rpm, checked the bacterial concentration at OD<SUB>600</SUB> and detected the concentration of TPA by UV at OD<SUB>242</SUB>.</DIV></B></B></B></B></LI></DIV><A class="expand-btn">Show More</A></DIV></ARTICLE><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><HEADER class="entry-header"><H1 class="entry-title">Week2(7/31/2016-8/6/2016)</H1></HEADER><DIV class="entry-content"><DIV class="note-content2"><LI><H1 style="font-size:135%">Optimization of Culture Conditions</H1><B><H1 style="font-size:108%">  Jul.31th</H1><DIV style="padding-left:32px;">Cultured <I>Rhodococcus jostii RHA1</I> in M9 medium with sodium terephthalate and glucose at 30℃ &amp; 200 rpm, checked the bacterial concentration at OD<SUB>600</SUB> and detected the concentration of TPA by UV at OD<SUB>242</SUB>.</DIV><B><H1 style="font-size:108%">  Aug.1th</H1><DIV style="padding-left:32px;">Cultured <I>Rhodococcus jostii RHA1</I> in W medium with sodium terephthalate and glucose at 30℃ &amp; 200 rpm, checked the bacterial concentration at OD<SUB>600</SUB> and detected the concentration of TPA by UV at OD<SUB>242</SUB>.</DIV><B><H1 style="font-size:108%">  Aug.2th</H1><DIV style="padding-left:32px;">Cultured <I>Pseudomonas putida KT2440</I> in W medium with sodium terephthalate and glucose at 30℃ &amp; 200 rpm, checked the bacterial concentration at OD<SUB>600</SUB> and detected the concentration of TPA by UV at OD<SUB>242</SUB>.</DIV><B><H1 style="font-size:108%">  Aug.3th</H1><DIV style="padding-left:32px;">Cultured <I>Pseudomonas putida KT2440</I> , <I>Rhodococcus jostii RHA1</I> , <I>Pseudomonas putida KT2440</I> and <I>Rhodococcus jostii RHA1</I>in W medium with sodium terephthalate and glucose at 30℃ &amp; 200 rpm, checked the bacterial concentration at OD<SUB>600</SUB> and detected the concentration of TPA by UV at OD<SUB>242</SUB>.</DIV><B><H1 style="font-size:108%">  Aug.4th</H1>        Observed the growth of 1-day pre-cultured bacteria in LB at 30℃:
<DIV style="padding-left:32px;">All above were kept culturing for one more day and were checked on the next day. The next step was to explore the optimal condition for each bacterium and to co-culture each teo of them to see whether the consortium could work well.
The most probable pairs:
1. P.putida KT2440 + R.jostii RHA1;
2. B.subtilis 168 + R.jostii RHA1;
3. B.subtills 168 + P.putida KT2440;
4. R.jostii RHA1 + Y.lipolytia;
5. P.putida KT2440 + Y.lipolytia;
6. E.coli(RFP) + Y.lipolytia.</DIV><B><H1 style="font-size:108%">  Aug.5th</H1><DIV style="padding-left:32px;">Cultured different bacteria in M9 medium with sodium terephthalate at 30℃ &amp; 200 rpm, checked the bacterial concentration at OD<SUB>600</SUB> and detected the concentration of TPA by UV at OD<SUB>242</SUB>:
The growth of each bacterium was not as expected, so we considered culturing bacteria in improved W medium.</DIV><DIV style="padding-left:32px;">Co-cultured different pairs in improved W medium in the same condition (using two tubes in each group):</DIV><DIV style="padding-left:32px;">Co-cultured different pairs in LB medium in the same condition(using two tubes in each group):</DIV><B><H1 style="font-size:108%">  Aug.6th</H1><DIV style="padding-left:32px;">Co-cultured different pairs in improved W medium in the same condition (using two tubes in each group):</DIV><DIV style="padding-left:32px;">Microscopic examination of each bacterium and each pair by means of Fuchsin or Gram dye.
To our surprise, there were some pairs (P.p + R.j) living well in the same tube and also there were some pairs we were not sure. It seemed that P.putida KT2440 was the dominant bacterium when it was co-cultured with others. We were thinking about limiting its growth by control ingredients in different media.
</DIV></B></B></B></B></B></B></B></LI><LI><H1 style="font-size:135%">Construction of PBBR</H1><B><H1 style="font-size:108%">  Aug.5th</H1>        Extraction of plasmid pBBR1MCS-2<B><H1 style="font-size:108%">  Aug.6th</H1>            Cut pBBR1MCS-2 with restricted enzymes Xba1 and Sac1 and checked by agarose gel electrophoresis:         Failed. After gel recovery, the concentration of digested pBBR1MCS-2 was 2.4 ng/μl, which was too low to be used in the next step.</B></B></LI><LI><H1 style="font-size:135%">Gene Knockout of <I>Escherichia coli</I></H1><DIV style="padding-left:32px;">We tried to copy tet gene(tetracycline resistance gene) in two overlap area of the fragment(about 500bp and 1000bp,called ‘tet-1’ and ‘tet-2’) and the homologous arms of the knockout gene-atpF and atpH(about 450bp each,called ‘left’ and ‘right’)with the help of PCR.</DIV></LI></DIV><A class="expand-btn2">Show More</A></DIV></ARTICLE><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><H1 class="entry-title">Week3(8/7/2016-8/13/2016)</H1><DIV class="entry-content"><DIV class="note-content3"><LI><H1 style="font-size:135%">Optimization of Culture Conditions</H1><B><H1 style="font-size:108%">  Aug.7th</H1><DIV style="padding-left:32px;">For each pair, we checked the bacterial concentration at OD_600 and detected the concentration of TPA by UV at Abs_242.According to results of the UV tests, if R.j existed, whatever the strategy was, the concentration of TPA would significantly decrease. Especially for pairs R.j+P.p and R.j + B.s, results showed greater reductions in the concentration of TPA compared with control group.</DIV><DIV style="padding-left:32px;">Pre-culture and acclimatize P.putida KT2440 in a condition with 5% EG
To our surprise, this wild type of P.putida could grow well in this condition, we considered increasing the concentration of EG in the condition.
</DIV><B><H1 style="font-size:108%">  Aug.9th</H1><DIV style="padding-left:32px;">1.Extract 5ml YPD medium to 8 test tubes, respectively.
2.Added bacteria solution as following table (use two tubes each group)</DIV><DIV style="padding-left:32px;">3.Cultured them at 30℃ &amp; 200 rpm, checked the bacterial concentration at OD<SUB>600</SUB> and detected the concentration of TPA by UV at OD<SUB>242</SUB></DIV><B><H1 style="font-size:108%">  Aug.10th</H1><DIV style="padding-left:32px;">1.Prepared 300ml W0 medium and add 1.5g Yeast Extract, then regarded it W1 medium.
2.Extracted 8ml W1 medium to 16 test tubes, respectively.
3.Added bacteria solution as following table (use two tubes each group).</DIV><DIV style="padding-left:32px;">4.Cultured them at 30℃ &amp; 200 rpm, checked the bacterial concentration at OD<SUB>600</SUB> and detected the concentration of TPA by UV at OD<SUB>242</SUB>.</DIV><B><H1 style="font-size:108%">  Aug.12th</H1><DIV style="padding-left:32px;">1.Prepared 200ml W0 medium and add 0.4g Urea, then regarded it W2 medium.
2.Extracted 8ml W2 medium to 16 test tubes, respectively.
3.Added bacteria solution as following table (use two tubes each group)</DIV><DIV style="padding-left:32px;">4.Cultured them at 30℃ &amp; 200 rpm, checked the bacterial concentration at OD<SUB>600</SUB> and detected the concentration of TPA by UV at OD<SUB>242</SUB>.</DIV><DIV style="padding-left:32px;">1.Prepared 200ml W0 medium and add 0.4g KNO3, then regarded it W3 medium.
2.Extracted 8ml W3 medium to 16 test tubes, respectively.
3.Added bacteria solution as following table (use two tubes each group)</DIV><DIV style="padding-left:32px;">4.Cultured them at 30℃ &amp; 200 rpm, checked the bacterial concentration at OD<SUB>600</SUB> and detected the concentration of TPA by UV at OD<SUB>242</SUB>.</DIV><B><H1 style="font-size:108%">  Aug.13th</H1><DIV style="padding-left:32px;">1.Extract 5ml W medium to 6 test tubes and add 20μL, 50μL, 80μL EG to them, respectively;
2.Add 10 μL bacteria solution of <I>Pseudomonas putida KT2440</I> to the six test tubes, respectively;
3. Cultured them at 30℃ &amp; 200 rpm, checked the bacterial concentration at OD<SUB>600</SUB>.</DIV><DIV style="padding-left:32px;">Added bacteria solution to W2, W3 medium as following table (use two tubes each group)</DIV></B></B></B></B></B></LI><LI><H1 style="font-size:135%">Construction of PBBR</H1><B><H1 style="font-size:108%">  Aug.7th</H1><DIV style="padding-left:32px;">Repeated the digestion and agarose gel electrophoresis:
Also Failed. After gel recovery, the concentration of digested pBBR1MCS-2 was only 2.4 ng/μl. We needed to increase the concentration of pBBR1MCS-2 in EB Solution.</DIV><B><H1 style="font-size:108%">  Aug.8th</H1><DIV style="padding-left:32px;">Colony PCR of P.putida KT2440 to get genes AcoA &amp; AceA:</DIV><DIV style="padding-left:32px;">Result:
Failed. Only primer dimers existed.</DIV><B><H1 style="font-size:108%">  Aug.9th</H1><DIV style="padding-left:32px;">Repeated the colony PCR above:</DIV><DIV style="padding-left:32px;">Result:
Also Failed. Only primer dimers existed. We considered using boiled bacteria as the template in the next PCR.
</DIV><B><H1 style="font-size:108%">  Aug.11th</H1><DIV style="padding-left:32px;">Extraction of plasmid pBBR1MCS-2.</DIV><DIV style="padding-left:32px;">The nucleic acid concentration of new EB solutions increased a lot. Increasing the amount of E.coli and decreasing the amount of EB did work.</DIV><B><H1 style="font-size:108%">  Aug.12th</H1><DIV style="padding-left:32px;">PCR of P.putida KT2440 to get genes AcoA &amp; AceA:</DIV><DIV style="padding-left:32px;">Result:
Successful PCR. Obvious bright bands located at 900+ and 1300+ bp (AcoA 978bp and AceA 1326 bp). Using boiled P.putida as the template in the PCR worked well.
</DIV><B><H1 style="font-size:108%">  Aug.13th</H1><DIV style="padding-left:32px;">Cut pBBR1MCS-2 with restricted enzymes Xba1 and Sac1 and checked by agarose gel electrophoresis.
Result: Obvious bright bands located over 5000 bp.
After gel recovery, we got the EB solution with cut plasmid pBBR1MCS-2, but the concentration is only 6.6 ng/μl, which is still quite low.

</DIV></B></B></B></B></B></B></LI><LI><H1 style="font-size:135%">Experiments about <I>Bacillus subtilis</I></H1><B><H1 style="font-size:108%">  Aug.9th</H1><DIV style="padding-left:32px;">Inoculated <I>B.subtilis 168</I> for transformation.</DIV><B><H1 style="font-size:108%">  Aug.10th</H1><DIV style="padding-left:32px;">Plasmids of PETase and plasmid of pHP13-p43 in <I>E.coli</I> were isolated. Enzyme digestion using <I>EcoR I</I> and <I>BamH I</I>, then do gel extraction and then link them.</DIV><B><H1 style="font-size:108%">  Aug.11th</H1><DIV style="padding-left:32px;">The plasmids from DNA ligation were transferred into <I>B.subtilis 168</I>, then cultivated on chloramphenicol containing LB plates to observe whether successful.</DIV><B><H1 style="font-size:108%">  Aug.12th</H1><DIV style="padding-left:32px;"><I>B.subtilis</I> Transformation was observed failure，3 times.
</DIV></B></B></B></B></LI><LI><H1 style="font-size:135%">Gene Knockout of <I>Escherichia coli</I></H1><DIV style="padding-left:32px;">Instead of enzyme-cut and link up, we used the overlap of each fragment to increase our aim gene(‘tet’, ‘left-right’, ‘left-tet-right’).</DIV></LI></DIV><A class="expand-btn3">Show More</A></DIV></ARTICLE><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><H1 class="entry-title">Week4(8/14/2016-8/20/2016)</H1><DIV class="entry-content"><DIV class="note-content4"><LI><H1 style="font-size:135%">Optimization of Culture Conditions</H1><B><H1 style="font-size:108%">  Aug.15th</H1><DIV style="padding-left:32px;">1.Prepared 200ml W0 medium and add 0.6g sucrose, then regarded it W4 medium.
2.Extracted 8ml W4 medium to 16 test tubes, respectively.
3.Added bacteria solution as following table (use two tubes each group)</DIV><DIV style="padding-left:32px;">4.Cultured them at 30℃ &amp; 200 rpm, checked the bacterial concentration at OD<SUB>600</SUB> and detected the concentration of TPA by UV at OD<SUB>242</SUB>.</DIV><B><H1 style="font-size:108%">  Aug.16th</H1><DIV style="padding-left:32px;">Prepared 200ml W0 medium.
2.Extracted 8ml W0 medium to 16 test tubes, respectively.
3.Added bacteria solution as following table (use two tubes each group)</DIV><DIV style="padding-left:32px;">4.Cultured them at 30℃ &amp; 200 rpm, checked the bacterial concentration at OD<SUB>600</SUB> and detected the concentration of TPA by UV at OD<SUB>242</SUB>.</DIV></B></B></LI><LI><H1 style="font-size:135%">Construction of PBBR</H1><B><H1 style="font-size:108%">  Aug.14th</H1><DIV style="padding-left:32px;">Ligation of cut pBBR1MCS-2 and CFP and Chemical transformation of pBBR into E.coli:
Ligation sites: Sac1 and Xba1</DIV><DIV style="padding-left:32px;">Result:
The ligation was successful, because we found the special band which represented CFP during the verification by PCR. We also successfully transformed the plasmid into E.coli which was cultured on the medium with kanamycin. However, the expression of CFP was not detected, which needed to be figure out.
</DIV><B><H1 style="font-size:108%">  Aug.15th</H1><DIV style="padding-left:32px;">Cut pBBR1MCS-2 with restricted enzymes EcoR1 and Sac1 and checked by agarose gel electrophoresis: 	
Result: Obvious bright bands located over 5000 bp.
After gel recovery, we got the EB solution with cut plasmid pBBR1MCS-2, and the concentration is 14.7 ng/μl.
</DIV><B><H1 style="font-size:108%">  Aug.16th</H1><DIV style="padding-left:32px;">Extraction of plasmid pBBR1MCS-2.</DIV><B><H1 style="font-size:108%">  Aug.17th</H1><DIV style="padding-left:32px;">Ligation of cut pBBR1MCS-2 with AcoA and AceA and Chemical transformation of pBBRAA into E.coli:
Ligation sites: Sac1 and EcoR1</DIV><DIV style="padding-left:32px;">Result:
The ligation was successful, because we found the special band which represented AcoA and AceA during the verification by PCR. We also successfully transformed the plasmid into E.coli which was cultured on the medium with kanamycin.
</DIV></B></B></B></B></LI><LI><H1 style="font-size:135%">Experiments about <I>Bacillus subtilis</I></H1><B><H1 style="font-size:108%">  Aug.14th</H1><DIV style="padding-left:32px;"><I>B.subtilis</I> Transformation was observed failure.
Reconstruct the plasmids.
</DIV><B><H1 style="font-size:108%">  Aug.15th</H1><DIV style="padding-left:32px;"><I>E.coli</I> transformation was observed failure.
Enzyme digestion again.<I>E.coli</I> transformation again.
</DIV><B><H1 style="font-size:108%">  Aug.16th</H1><DIV style="padding-left:32px;"><I>E.coli</I> transformation was observed failure.
Plasmid was enzyme digested and not observed correct band.
</DIV><B><H1 style="font-size:108%">  Aug.17th</H1><DIV style="padding-left:32px;">Plasmids of PETase was enzyme digestion again.
Inoculate <I>B.subtilis DB104</I>.
</DIV><B><H1 style="font-size:108%">  Aug.19th</H1><DIV style="padding-left:32px;">Inoculate <I>E.coli</I> of pHP13-p43.
</DIV></B></B></B></B></B></LI><LI><H1 style="font-size:135%">Gene Knockout of <I>Escherichia coli</I></H1><DIV style="padding-left:32px;">We aimed to put the left-tet-right gene and plasmid which have λ-red gene into the the genome of E.coli. After two screening, we could gain the target strains which knocked out the atpF and atpH gene.</DIV></LI></DIV><A class="expand-btn4">Show More</A></DIV></ARTICLE><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><H1 class="entry-title">Week5(8/21/2016-8/27/2016)</H1><DIV class="entry-content"><DIV class="note-content4"><LI><H1 style="font-size:135%">Optimization of Culture Conditions</H1><B><H1 style="font-size:108%">  Aug.22th</H1>        1.Prepared 200ml W0 medium and add 0.3g NaOAc, then regarded it W7 medium.        2.Extracted 8ml W7 medium to 16 test tubes, respectively.        3.Added bacteria solution as following table (use two tubes each group)      4.cultured them at 30℃ and check growing situation and the situation of TPA degradation.

</B></LI><LI><H1 style="font-size:135%">Experiments about <I>Bacillus subtilis</I></H1><B><H1 style="font-size:108%">  Aug.24th</H1><DIV style="padding-left:32px;">Cultivated transformed <I>E.coli</I> on chloramphenicol containing LB plates to observe whether successful.
</DIV></B></LI><LI><H1 style="font-size:135%">Gene Knockout of <I>Escherichia coli</I></H1><DIV style="padding-left:32px;">We cultivateD Saccharomyces cerevisiae and E.coli which has knocked out atpF and atpH gene and introduced GFP gene in special culture medium which the only carbon source was xylose.</DIV></LI></DIV><A class="expand-btn4">Show More</A></DIV></ARTICLE><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><HEADER class="entry-header"><H1 class="entry-title">Week6(8/28/2016-9/3/2016)</H1></HEADER><DIV class="entry-content"><DIV class="note-content4"><LI><H1 style="font-size:135%">Experiments about <I>Bacillus subtilis</I></H1><B><H1 style="font-size:108%">  Aug.29th</H1><DIV style="padding-left:32px;"><I>B.subtilis DB104</I> transformation.
</DIV><B><H1 style="font-size:108%">  Aug.30th</H1><DIV style="padding-left:32px;"><I>B.subtilis DB104</I> transformation is successful.
</DIV><B><H1 style="font-size:108%">  Sep.2th</H1><DIV style="padding-left:32px;">Construct plamid of MHETase without success.
Transform into <I>E.coli</I>, in chloramphenicol containing LB culture.

</DIV></B></B></B></LI></DIV><A class="expand-btn4">Show More</A></DIV></ARTICLE><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><HEADER class="entry-header"><H1 class="entry-title">Week7(9/4/2016-9/10/2016)</H1></HEADER><DIV class="entry-content"><DIV class="note-content4"><LI><H1 style="font-size:135%">Experiments about <I>Bacillus subtilis</I></H1><B><H1 style="font-size:108%">  Sep.8th</H1><DIV style="padding-left:32px;">Inoculate PETase transformed <I>B.stubtilis 168</I> in 3 flask culture.
1.LB, chloramphenicol, PETase transformed <I>B.stubtilis 168</I>, PET film.
2.LB, chloramphenicol, PETase transformed <I>B.stubtilis 168</I>, no PET film.
3.LB, chloramphenicol, <I>B.stubtilis 168</I>, PET film.

</DIV><B><H1 style="font-size:108%">  Sep.9th</H1><DIV style="padding-left:32px;">Construct plamid of MHETase without success.
</DIV><B><H1 style="font-size:108%">  Sep.10th</H1><DIV style="padding-left:32px;">Construct plamid of MHETase without success.

</DIV></B></B></B></LI></DIV><A class="expand-btn4">Show More</A></DIV></ARTICLE><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><HEADER class="entry-header"><H1 class="entry-title">Week8(9/11/2016-9/17/2016)</H1></HEADER><DIV class="entry-content"><DIV class="note-content4"><LI><H1 style="font-size:135%">Optimization of Culture Conditions</H1><B><H1 style="font-size:108%">  Sep.12th</H1><DIV style="padding-left:32px;">Optimize the growing environment of Bacillus stubtilis by change medium components.Firstly, we prepared 400ml W medium, then, we devided the medium into four pieces averagely and number them No.1, No.2, No.3, No.4. Next, we added 0.1g NaCl, 0.1g NH4Cl, 0.3g sucrose to No.2, No.3, No.4, respectively. After that, we Extracted 5 ml No.1, No.2, No.3, No.4 and added them to four test tubes, then, we added 10 μL bacteria solution of <I>Bacillus stubtilis 168</I> to four test tubes, respectively. Eventually, we cultured them at 37℃ and check growing situation.</DIV></B></LI><B><H1 style="font-size:108%">  Sep.16th</H1>        1.Prepared 200ml W0 medium and add 0.2g NH4Cl, then regarded it W8 medium.        2.Extracted 10ml W8 medium to 16 test tubes, respectively.        3.Added bacteria solution as following table (use two tubes each group)        4.Cultured them at 30℃ &amp; 200 rpm, checked the bacterial concentration at OD<SUB>600</SUB> and detected the concentration of TPA by UV at OD<SUB>242</SUB>.
</B><LI><B><H1 style="font-size:135%">Experiments about <I>Bacillus subtilis</I></H1><B><H1 style="font-size:108%">  Sep.11th</H1><DIV style="padding-left:32px;">Construct plamid of MHETase without success.

</DIV><B><H1 style="font-size:108%">  Sep.12th</H1><DIV style="padding-left:32px;">Construct plamid of MHETase without success.
</DIV></B></B></B></LI></DIV><A class="expand-btn4">Show More</A></DIV></ARTICLE><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><HEADER class="entry-header"><H1 class="entry-title">Week9(9/18/2016-9/24/2016)</H1></HEADER><DIV class="entry-content"><DIV class="note-content4"><LI><H1 style="font-size:135%">Optimization of Culture Conditions</H1><B><H1 style="font-size:108%">  Sep.19th</H1>        1. Prepared 1L W medium and add 2.5g TPA and 1.1875g NaOH.        2. devided the medium into five pieces averagely and add chemicals as following table        3. Extracted 5ml W9, W10, W11, W12, W13 medium to 40 test tubes, respectively.        4. Added bacteria solution as following table (use two tubes each group)        5. cultured them at 30℃ &amp; 200 rpm, checked the bacterial concentration at OD<SUB>600</SUB> and detected the concentration of TPA by UV at OD<SUB>242</SUB>.<B><H1 style="font-size:108%">  Sep.21th</H1>        Repeat experiments of W9, W10 the day before yesterday.




<B><H1 style="font-size:108%">  Sep.22th</H1>        1. Use five mediums of Sep.19th and W0, W8 medium of Sep.16th.        2. Extracted 5ml W0, W8, W9, W10, W11, W12, W13 medium to 56 test tubes, respectively.        3. Added bacteria solution as following table (use two tubes each group)        4. cultured them at 30℃ &amp; 200 rpm, checked the bacterial concentration at OD<SUB>600</SUB> and detected the concentration of TPA by UV at OD<SUB>242</SUB>.


<B><H1 style="font-size:108%">  Sep.23th</H1>        1. Prepared 400L W medium and add 1.2g KNO3 and 1.2g glucose.        2. devided the medium into four pieces averagely and add chemicals as following table        3. Extracted 5ml W0, W14, W15, W16 medium to 34 test tubes, respectively.        4. Added bacteria solution as following table (use two tubes each group)        5. cultured them at 30℃ &amp; 200 rpm, checked the bacterial concentration at OD<SUB>600</SUB> and detected the concentration of TPA by UV at OD<SUB>242</SUB>.</B></B></B></B></LI><LI><H1 style="font-size:135%">Experiments about <I>Bacillus subtilis</I></H1><B><H1 style="font-size:108%">  Sep.19th</H1><DIV style="padding-left:32px;">Construct plasmid of MHETase without success.
Cultivated MHETase transformed <I>B.stubtilis 168</I> on Erythromycin containing LB plates to observe, without success.

</DIV></B></LI><LI><H1 style="font-size:135%">16SrDNA</H1><B><H1 style="font-size:108%">  Sep.19th</H1><DIV style="padding-left:32px;">we firstly tried to cultivate each of them(<I>Rhodococcus RHA1, Pseudomonas putida KT2440, bacillus subtilis 168</I>) in LB culture medium for amplification.

</DIV><B><H1 style="font-size:108%">  Sep.21th</H1><DIV style="padding-left:32px;">We use orthogonal test to prove each bacteria had each special DNA stripe from the method of bacteria colony-PCR.
</DIV><B><H1 style="font-size:108%">  Sep.22th</H1><DIV style="padding-left:32px;">Then we cultivate each of them and the mixture of three in W0 culture medium for amplification.
</DIV><B><H1 style="font-size:108%">  Sep.23th</H1><DIV style="padding-left:32px;">We did the same 4*3(bacteria liquid and six primer sequences) orthogonal test.

</DIV><B><H1 style="font-size:108%">  Sep.24th</H1><DIV style="padding-left:32px;">We cultivate each of them and the mixture of three in in modified W0 culture medium which changed the carbon source from glucose to sugar for amplification. 

</DIV></B></B></B></B></B></LI></DIV><A class="expand-btn4">Show More</A></DIV></ARTICLE><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><HEADER class="entry-header"><H1 class="entry-title">Week10(9/25/2016-10/1/2016)</H1></HEADER><DIV class="entry-content"><DIV class="note-content4"><LI><H1 style="font-size:135%">16SrDNA</H1><B><H1 style="font-size:108%">  Sep.25th</H1><DIV style="padding-left:32px;">We still did the same 4*3(bacteria liquid and six primer sequences) orthogonal test.
</DIV></B></LI></DIV><A class="expand-btn4">Show More</A></DIV></ARTICLE></DIV></DIV></DIV></DIV><DIV class="side-nav"><UL><LI><A class="topLink" href="#Topp" style="color:#5555FF">Top</A></LI><LI><A class="topLink" href="#Week1" style="color:#5555FF">Week1</A></LI><LI><A class="topLink" href="#Week2" style="color:#5555FF">Week2</A></LI><LI><A class="topLink" href="#Week3" style="color:#5555FF">Week3</A></LI><LI><A class="topLink" href="#Week4" style="color:#5555FF">Week4</A></LI><LI><A class="topLink" href="#Week5" style="color:#5555FF">Week5</A></LI><LI><A class="topLink" href="#Week6" style="color:#5555FF">Week6</A></LI><LI><A class="topLink" href="#Week7" style="color:#5555FF">Week7</A></LI><LI><A class="topLink" href="#Week8" style="color:#5555FF">Week8</A></LI><LI><A class="topLink" href="#Week9" style="color:#5555FF">Week9</A></LI><LI><A class="topLink" href="#Week10" style="color:#5555FF">Week10</A></LI></UL></DIV><DIV class="alert alert-info"><DIV class="container-fluid"><DIV class="alert-icon"><I class="material-icons">info_outline</I></DIV><BUTTON type="button" class="close" data-dismiss="alert" aria-label="Close"><SPAN aria-hidden="true"><I class="material-icons">clear</I></SPAN></BUTTON><B>Notice:</B> This page is currently under construction. Contents in this page are temporaory and will be modified several times before the final release.    — 2016 iGEM Team Tianjin
    </DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>