<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tianjin_Experiment_R-R skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tianjin/Experiment/R-R</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>TEAM TIANJIN</TITLE></P><DIV class="container-fluid"><DIV class="navbar-header"><DIV class="nameTeamTianjin">PLASTERMINATOR</DIV><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:Tianjin" id="Overview">OVERVIEW</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Projects">PROJECTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Description" id="Outline">OUTLINE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Experiment/Consortium" id="Consortium">CONSORTIUM</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Experiment/Protein_Engineering" id="ProteinEngineering">PROTEIN ENGINEERING</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Experiment/R-R" id="RRSystem">R-R SYSTEM</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Proof" id="Proof">PROOF</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Model" id="Model">MODEL</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Results">RESULTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Achievements" id="Achievements">ACHIEVEMENTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Demonstrate" id="Demonstrate">DEMONSTRATE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Parts" id="Parts">PARTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Future" id="FutureWork">FUTURE WORK</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="People">PEOPLE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Team" id="Team">TEAM</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Attributions" id="Attribution">ATTRIBUTION</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Outreach">OUTREACH</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Community/Overview" id="HumanPractice">HUMAN PRACTICE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Collaborations" id="Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Safety" id="Safety">SAFETY</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Notes">NOTES</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Notes" id="Notebook">NOTEBOOK</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Protocol" id="Protocol">PROTOCOL</A></LI></DIV></DIV><DIV class="section clearfix object-non-visible" data-animation-effect="fadeIn"><DIV class="container" style="width:70vw;margin-top:80px"><DIV class="col-md-12"><H1 id="about" class="title text-center"><SPAN>R-R system</SPAN></H1><H2><B>Overview</B></H2><P style="font-size:18px" id="ReportingSystem">R-R system (namely reporting and regulation system), is used in our project in order to make the expression of PET degrading enzyme visible and regular. As its name implies, this system consists of two independent part, reporting and regulation. We test our reporting part in <I>E.coli</I> and regulation part in <I>Saccharomyces cerevisiae</I> .</P><DIV class="row"><DIV class="col-md-3"><DIV align="center"><FIGURE><FIGCATION>Fig.1. Structure of part <I>BBa_K339007</I></FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-9"><H3><B>1. Reporting System</B></H3><P style="font-size:18px">The basis of our reporting system is the part <A href="http://parts.igem.org/Part:BBa_K339007" target="_blank"><I>BBa_K339007</I></A>, Designed by Emily Hicks from Group iGEM10_Calgary. This part can sense the CpxR protein, which will form spontaneously in <I>E.coli</I> when inclusion body and misfolding protein present in the periplasm of <I>E.coli</I>, and then start expressing RFP so that we can detect red fluorescence. As we all know, the inclusion body will inevitably form when we overexpress heterologous protein like PETase in <I>E.coli</I>. Therefore, the emission of red fluorescence can report the overexpression of PETase. What is more, this device can be modified to report overexpression of any heterologous protein only if the PETase gene is replaced by another heterologous gene. After the red fluorescence is detected, we could start the purification of protein.</P></DIV></DIV><DIV class="row"><DIV class="col-md-8"><DIV align="center"><FIGURE id="CellLysisBasedRegulationSystem"><FIGCATION>Fig.2. Brief Structure of our reporting system based on inclusion body sensing CpxR promoter</FIGCATION></FIGURE></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-5"><H3><B>2. Cell Lysis Based Regulation System</B></H3><P style="font-size:18px">The regulation system consists of two section. The first section is based on the already mentioned reporting system. We change the RFP gene to the novel ddpX (D-alanyl-D-alanine dipeptidase) gene from <I>E.coli</I> genome. The ddpX gene can hydrolyze the D-Ala-D-Ala structure in peptidoglycan molecule and cause damage to the cell wall of <I>E.coli</I>. Under normal condition, this gene only express when the cell is in starvation mode in order to use hydrolysate alanine as carbon source. However, if we overexpress this gene, the cell wall will be dissolved and finally cell lysis will happen. Therefore, in this system, when the PETase is overexpressed, the spontaneously forming inclusion body will induce the expression of ddpX and cause cell lysis. It will provide us with a novel and convenient and way of protein purification when you use <I>E.coli</I> as chassis.</P></DIV><DIV class="col-md-7"><DIV align="center"><FIGURE><FIGCATION>Fig.3. Brief Structure of our regulation system based on cell lysis</FIGCATION></FIGURE></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-5"><DIV align="center"><FIGURE><FIGCATION>Fig.4. Mechanism of TPA positive feedback system</FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-7"><H3><B>3. TPA Positive Feedback Based Regulation System</B></H3><P style="font-size:18px">The next section is based on the TPA-inducing promoter. Considering the TPA degrading ability of <I>Rhodococcus jostii RHA1</I>, we believe there should be promoter that can sense and be induced by TPA. Luckily we find these three gene that have something to do with TPA degrading in <I>Rhodococcus jostii RHA1</I> can be induced significantly by TPA. The reason why these promoter can be induced by TPA is they have a leader sequence before the promoter sequence, we name it TPA inducible leader sequence (TILS). The gene of TPA transporting protein and regulation protein are also transformed into <I>Saccharomyces cerevisiae</I>. The TPA regulation protein is belong to the IclR family. This novel protein can combine the TILS and induce the expression of downstream gene when it combine the TPA molecule. Therefore, we insert the TILS before the enhanced promoter PGK1 so that we can make our promoter inducible by TPA.</P></DIV></DIV><H2><B>Theoretical Background</B></H2><DIV class="row"><DIV class="col-md-6"><H3><B>1. The Cpx Regulation System<A class="btn popover-info ref-title-link" data-toggle="popover" data-placement="top" title="Reference" data-content="Physiologie der Mikroorganismen, Humboldt Universitat zu Berlin, Chausseestr. Misfolded maltose binding protein MalE219 induces the CpxRA envelope stress response by stimulating phosphoryl transfer from CpxA to CpxR. Research in Microbiology 160 (2009) 396-400.">[1]</A></B></H3><P style="font-size:18px">In order to adapt to their changing environment，Escherichia coli bacterium need plenty of regulatory systems. The Cpx system is a three-component regulatory system which is kind of similar to the lactose operon.</P><P style="font-size:18px">The Cpx system consists of the histidine kinase CpxA, the response regulator CpxR and the periplasmic CpxP protein. CpxA is composed of a large periplasmic domain and a highly conserved cytosolic catalytic domain. Both domains are connected via two trans-membrane helices. CpxA has autophosphorylation, phosphor-transfer and phosphatase activities .Sensing envelope perturbation by an unknown feature, CpxA transmits a signal via a phosphorelay to CpxR, which in response acts as a transcription regulator of genes, whose products are mainly involved in envelope protein folding, detoxification and biofilm formation. The Cpx stress response is controlled by feedback inhibition CpxP acts at the initiation point of signal transduction by reducing CpxA auto-phosphorylation activity in the reconstituted CpxRA system.</P><P style="font-size:18px">The Cpx pathway is activated by a large number of different signals including elevated pH, increasing osmolarity, metals, altered membrane composition, overproduction of outer membrane lipoproteins and misfolded variants of maltose binding protein.</P><P style="font-size:18px" id="DdpXcelllysiseffect">When the stress is at lower level, the CpxP protein combine with the CpxA to prevent CpxA from phosphorylating CpxR and when the stress changes at higher level, some signals lead to activation of the Cpx pathway. Particularly, misfolded protein such as MalE219 interacts directly with the periplasmic domain of CpxA, resulting in stimulation of CpxA phosphotransfer activity towards CpxR. </P></DIV><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.5. The Cpx inclusion body responding system in <I>E.coli</I>&amp;nbsp<SUP>[1]</SUP></FIGCATION></FIGURE></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-5"><DIV align="center"><FIGURE><FIGCATION>Fig.6. The function of ddpX in <I>E.coli</I> under starvation conditions <SUP>[2]</SUP></FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-7"><H3><B>2. DdpX cell lysis effect<A class="btn popover-info ref-title-link" data-toggle="popover" data-placement="top" title="Reference" data-content="Ivan A. D. Lwssard and Christopher T. Walsh. VanX, a bacterial D-alanyl-D-alanine dipeptidase: Resistance, immunity, or survival function? Proc. Natl. Acad. Sci. USA. Vol. 96, pp. 11028–11032, September, 1999.">[2]</A></B></H3><P style="font-size:18px">DdpX, namely D-alanyl-D-alanine dipeptidase, is a kind of peptidoglycan hydrolase which can hydrolyze the D-Ala-D-Ala part in peptidoglycan molecule. As we all know, the cell wall of bacterial mainly consists of peptidoglycan, so the ddpX can hydrolyze the cell wall of bacterial.</P><P style="font-size:18px">It cannot be more strange that many bacterial own this kind of seemly dangerous gene in their genome. In fact, this gene also has many benefits to bacterial. In gram-positive bacterial, Vancomycin, a kind of antibiotics, can cause cell lysis because it can combine the D-Ala-D-Ala residue of peptidoglycan in cell wall and block the cross-linking of peptidoglycan. Some gram-positive bacterial like Enterococcus faecalis and Streptomyces toyocaensis have developed the resistance to the vancomycin because they have VanX gene, the homologue of ddpX gene, which can hydrolyze the D-Ala-D-Ala and transfer the D-Ala-D-Ala residue of peptidoglycan to D-Ala-D-Lac residue so that the vancomycin cannot combine the peptidoglycan.</P><P style="font-size:18px" id="TPAPositiveFeedbackMechanism">However, in gram-negative bacterial like <I>E.coli</I>, which own the robust outer membrane that can resist the vancomycin, the hydrolase ddpX with the same effects also exists. This is strange because the gram-negative bacterial have no necessity to own this kind of seemly dangerous hydrolase. Actually the ddpX in <I>E.coli</I> has another vital use when they are under starvation conditions. The ddpX can hydrolyze the D-Ala-D-Ala in their cell wall to produce the D-Ala as the carbon source to maintain their life. This mechanism is only carried out when they are under starvation conditions. If the ddpX gene is overexpressed, the cell wall will be damaged and cell lysis will occur. </P></DIV></DIV><DIV class="row"><DIV class="col-md-12"><H3><B>3. TPA Positive Feedback Mechanism<A class="btn popover-info ref-title-link" data-toggle="popover" data-placement="top" title="Reference" data-content="Hirofumi Hara, Lindsay D. Eltis, Julian E. Davies. Transcriptomic Analysis Reveals a Bifurcated Terephthalate Degradation Pathway in Rhodococcus sp. Strain RHA1. Journal of Bacteriology, Mar. 2007, 189(5), 1641–1647.">[3]</A><A class="btn popover-info ref-title-link" data-toggle="popover" data-placement="top" title="Reference" data-content="Molina-Henares, A. J., T. Krell, M. E. Guazzaroni, A. Segura, and J. L. Ramos. 2006. Members of the IclR family of bacterial transcriptional regulators function as activators and/or repressors. FEMS Microbiol. Rev. 30: 157–186.">[4]</A></B></H3><P style="font-size:18px">As we all know, PET is solid in normal condition. So it’s not easy for microorganisms to realize if there is any PET in the environment. For this reason, we designed the following regulating path.</P><P style="font-size:18px">We aim at finding a way to offer bacterial the ability to sense TPA so that it can produce more enzyme when TPA degraded by PETase exists in the environment.
We find the similar mechanism in the <I>Rhodococcus jostii RHA1</I>, which can make use of TPA as carbon source. We speculate that there must be the pathway we want in the <I>Rhodococcus jostii RHA1</I>. By the way, RHA1 is also well used in microbial consortia part of our project. In<I> Rhodococcus</I>, the distinct expression patterns of the TPA gene clusters indicate that they are independently regulated. The cluster contains gene encoding putative regulatory protein, namely tpaR. This gene encodes the regulatory protein of the IclR family, based on the presence of a conserved signature region. The regulator has helix-turn-helix domain and encodes regulator for its respective operons, which is consistent with the case for IclR-type regulatory proteins for other aromatic catabolism pathways. IclR-type positive regulators bind a sequence before their promoter DNA in the existence of inductor and start the transcription of downstream gene, so we need to express the regulator too.<A class="btn popover-info ref-title-link" data-toggle="popover" data-placement="top" title="Reference" data-content="Molina-Henares, A. J., T. Krell, M. E. Guazzaroni, A. Segura, and J. L. Ramos. 2006. Members of the IclR family of bacterial transcriptional regulators function as activators and/or repressors. FEMS Microbiol. Rev. 30: 157–186.">[4]</A> Then the gene followed the promoter will be regulated by TPA.
We find a promoter from the upstream of a gene named tpaAa regulated by TPA. It will express 300 times more when TPA exist. So we plan to transform the three genes into <I>Saccharomyces cerevisiae</I>. They respectively encode TPA transporter, TPA regulation protein and RFP bonded with the TILS. Then we can detect the intension of the red signal to measure the expression of the protein in distinct concentrations of TPA. </P></DIV></DIV><H2 id="ConstructionofReportingSystem"><B>Experiment Design</B></H2><DIV class="row"><DIV class="col-md-5"><H3><B>1. Construction of Reporting System</B></H3><P style="font-size:18px">We use a common expression vector plasmid, pUC19, in <I>E.coli</I> to load our device, which consists of heterologous gene part (in this circumstance, PETase gene) and inclusion body reporting part. First of all, we transform the plasmid with part <I><I>BBa_K339007</I></I> from the kit shipped to us using the protocol in the instruction from iGEM official website. Then we use PCR to amplify this part with restriction endonuclease cutting sites <I>Xba1</I> and <I>Pst1</I> respectively on sense and anti-sense primers. Then we use corresponding restriction endonuclease to cut the part and plasmid pUC19 and then use T4 DNA ligase to link them together. The next step is to transform the PETase gene into the same plasmid. The initial gene synthetized does not has promoter and terminator so it cannot express. We have to cut the PETase gene and plasmid pET21A with <I>BamH1 </I> and Sal1 enzyme and link them together to transform the PETase gene into pET21A and then use PCR to amplify the T7 promoter-PETase gene-T7 terminator fragment added the restriction endonuclease cutting sites <I>EcoR1 </I> and Sac1. In this way, after we cut the recombinant plasmid pUC19 and T7 promoter-PETase gene-T7 terminator fragment with corresponding restriction endonucleases and link them together, we can obtain the complete device we want. </P></DIV><DIV class="col-md-7"><DIV align="center"><FIGURE><FIGCATION>Fig.7. The construction process of our reporting system</FIGCATION></FIGURE></DIV></DIV></DIV><DIV class="col-md-12"><H3><B>2. Verification of RFP in the part <I><I>BBa_K339007</I></I></B></H3><P style="font-size:18px" id="MethodofRedFluorescenceAssay">The verification of RFP is carried out by using PCR to amplify the RFP gene with restriction endonuclease cutting sites <I>Xba1</I> and Sac1 added and then cut the RFP and plasmid pET21a with corresponding restriction endonuclease. Then the cut fragments are linked together and transformed into <I>E.coli</I> to express. Then we can detect the red fluorescence.</P></DIV><DIV class="col-md-12"><H3><B id="CultureandExpressionCondition">3. Method of Red Fluorescence Assay</B></H3><P style="font-size:18px">The red fluorescence is detected by 96-well Microplate Reader. The excitation wavelength is set at 584nm and the emission wavelength is set at 607nm. Considering the RFP has an advantage that it can be directly observed by bare eyes, we also use centrifugation to precipitate the bacterial and observe the color of sediment. The red color can be observed if the RFP is expressed. All the experiment including the latter mentioned regulation system use this assay method.</P></DIV><DIV class="col-md-12"><H3><B id="ConstructionofCellLysis">4. Culture and Expression Condition of <I>E.coli</I> in this experiment</B></H3><P style="font-size:18px">Tradition culture medium LB (5g/L yeast extracts, 10g/L peptone, 10g/L NaCl) is also used by us. Because of the ampicillin resistance gene in the plasmid pUC19 and pET21A, ampicillin (100μg/mL) is added to screen for the correctly transformed bacterial. 5mL bacterial are cultured in test tube at 37℃ with 200rpm shaking speed. IPTG is added to induce the expression of PETase gene after 6 hours.</P></DIV><DIV class="row"><DIV class="col-md-7"><DIV align="center"><FIGURE><FIGCATION>Fig.8. The construction process of our cell lysis based regulation system</FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-5"><H3><B>5. Construction of Cell Lysis Based Regulation System</B></H3><P style="font-size:18px">This system has a great similarity to the reporting system above. Therefore it is easy to construct because we only need to change the RFP gene to the ddpX gene. However, there is no restriction endonuclease cutting site between the CpxR and RFP gene sequence according to the part map from the iGEM official website, so we have to use PCR to amplify the CpxR promoter solely and add restriction endonuclease cutting sites <I>Xba1</I> and <I>BamH1 </I> respectively in both end. The ddpX gene is obtained from the <I>E.coli</I> genome using colony PCR and the <I>BamH1 </I> and <I>EcoR1 </I> restriction endonuclease cutting sites are added respectively to both end. Then the three fragments, CpxR promoter, ddpX gene, and cut plasmid pET21a are linked together. Then the whole part is amplified by PCR with <I>Xba1</I> and <I>Pst1</I> restriction endonuclease cutting sites added respectively to both end. This way, we can easily cut down the former CpxR-RFP fragment and add the new CpxR-ddpX fragment to the plasmid pUC19. </P></DIV></DIV><DIV class="col-md-12"><H3 id="MethodofCellLysisAssay"><B>6. Verification of ddpX Gene Effect</B></H3><P style="font-size:18px">Just like the verification of RFP mentioned before, the verification of ddpX is carried out in the similar way. The pET21a plasmid is cut by <I>BamH1 </I> and <I>EcoR1 </I> instead of <I>Xba1</I> and <I>EcoR1 </I>, so that the ddpX can be linked to the cut plasmid pET21a solely. Then we can detect if the cell lysis occurs.</P></DIV><DIV class="col-md-12"><H3><B id="ChassisselectionforTPAPositive">7. Method of Cell Lysis Assay</B></H3><P style="font-size:18px">Cell lysis can be reflected by the OD<SUB>600</SUB> of culture medium. The lower the value of OD<SUB>600</SUB> is than the wild type <I>E.coli</I> at the same condition, the stronger the cell lysis effect will be. The OD<SUB>600</SUB> is detected by 96-well Microplate Reader. In order to know the OD<SUB>600</SUB> value continuously, the detection process works through the time of bacterial growth and we will obtain the OD<SUB>600</SUB>-Growing time curve. </P></DIV><DIV class="col-md-12"><H3><B id="ConstructionofTPAPositiveFeedback">8. Chassis selection for TPA Positive Feedback Based Regulation System</B></H3><P style="font-size:18px">As the explanation before, the TPA positive feedback system is derived from the TPA degradation metabolic pathway in <I>Rhodococcus jostii RHA1</I>. Considering the difficulty of conducting gene-scale operation in this unusual organism, we directly synthetize all the gene including tpaK, tpaR, and TILS. At first we want to use <I>E.coli</I> to test this device because of the easy and familiar operation. However, in this situation, we have to transform at least 3 plasmids and this cannot be more difficult for <I>E.coli</I>. Therefore, we use another familiar organism, <I>Saccharomyces cerevisiae</I>, as the chassis. In the preliminary experiment, we successfully transform 3 plasmids into <I>Saccharomyces cerevisiae</I>. </P></DIV><DIV class="row"><DIV class="col-md-5"><H3><B>9. Construction of TPA Positive Feedback Based Regulation System</B></H3><P style="font-size:18px">We use common plasmids of <I>Saccharomyces cerevisiae</I>, pRS413, pRS415 and pYES2, to respectively load the TPA transporting protein gene, TPA regulation protein gene and TPA induced RFP gene. First of all, we use PCR to amplify all of these fragments and add different restriction endonuclease cutting sites. Then we cut the plasmids with corresponding restriction endonucleases. Then these cut fragments are linked according to the designed order and transformed into <I>Saccharomyces cerevisiae</I>. We screen for the correctly transformed cell by using the Sc-Ura-Leu-His plate.</P></DIV><DIV class="col-md-7"><DIV align="center"><FIGURE><FIGCATION>Fig.9. The construction process of our TPA Positive Feedback Based regulation system</FIGCATION></FIGURE></DIV></DIV></DIV><DIV class="col-md-12"><H3><B id="FormerBiobrickVarification">10. Culture and Expression Condition of <I>Saccharomyces cerevisiae</I> in this experiment</B></H3><P style="font-size:18px">Traditional YPD culture medium (22g/L glucose, 20g/L peptone, 10g/L yeast extracts) is used by us. Sc-Ura-Leu-His culture medium (22g/L glucose, 6.7g/L yeast nitrogen base, 1.224g/L nutrient deficiency mixture without Ura, His, Leu and Trp, 5mg/L Trp) is used to screen for correctly transformed cell. All the cells are cultured in 5mL medium at 30℃ with shaking speed of 200rpm. To induce the expression of RFP, we add TPA with different concentration. We first make up TPA standard solution with TPA concentration of 5g/L. Then we respectively add 0, 1μL, 10μL, 100μL, 1mL standard solution to the culture medium. </P></DIV><DIV class="col-md-12"><H2><B>Former Biobrick Varification</B></H2><P style="font-size:18px" id="ExpectedResults">We applied the former part <A href="http://parts.igem.org/Part:BBa_K339007" target="_blank"><I>BBa_K339007</I></A> constructed by iGEM10_Calgary in our R-R system. This composite part consists of a CpxR responsive promoter (<A href="http://parts.igem.org/Part:BBa_K135000" target="_blank"><I>BBa_K135000</I></A>), a ribosome binding site (<A href="http://parts.igem.org/Part:BBa_B0034" target="_blank"><I>BBa_B0034</I></A>), a mRFP gene (<A href="http://parts.igem.org/Part:BBa_E1010" target="_blank"><I>BBa_E1010</I></A>), and two terminators (<A href="http://parts.igem.org/Part:BBa_B0010" target="_blank"><I>BBa_B0010</I></A> and <A href="http://parts.igem.org/Part:BBa_B0012" target="_blank"><I>BBa_B0012</I></A>). The CpxR responsive promoter can respond to the CpxR protein which appears when there are misfold protein in the periplasm of the E.coli and start the transcription of the downstream mRFP gene so that the red fluorescence can be detected. 
We applied this part in our reporting system. We transformed the PETase gene together with this part and induced the expression of the PETase gene. To make the reporting effect visible. We centrifuged the culture medium with the speed of 12000rpm for 1min, and directly observed by bare eyes.


Therefore, when the exogenous gene overexpress in the E.coli, the CpxR induced mRFP gene also express and lead to the emission of red fluorescence. The little red fluorescence in the bacterial without transformed into the PETase gene is largely likely to be caused by the basic expression of CpxR protein in the E.coli. However, when the exogenous gene overexpress, the formed inclusion body can significantly increase the expression of CpxR protein to induce the expression of mRFP. </P><H2><B>Expected Results</B></H2><P style="font-size:18px" id="References">PETase and MHETase are two key enzymes in our project. However, as heterologous proteins, the expression of these two enzymes face many problems just like expressing other heterologous proteins before including the formation of inclusion body, the lack of regulation pathway, etc. We design this R-R system in order to express the two enzymes visibly and regularly. </P><LI><P style="font-size:18px">First, we hope to directly observe the expression condition of our enzyme by color, when the inclusion body form, which means the overexpressing, the red color can be observed. </P></LI><LI><P style="font-size:18px">Second, when inclusion body form, the normal way to solve this problem is to use lysozyme and ultrasonic to break the cell and purify the protein, which is complex and time-consuming. We expect the cell lysis will automatically occur when the inclusion body form by using ddpX gene.  </P></LI><LI><P style="font-size:18px">Third, we expect the chassis organism can sense the existence of TPA, the hydrolyze product of PET and using TPA as the induction of PETase gene. Thus if the degradation process start, this process can be even enhanced until the PET is used up.</P></LI><H2><B>References</B></H2><P style="font-size:16px"><I>[1]Physiologie der Mikroorganismen, Humboldt Universitat zu Berlin, Chausseestr. Misfolded maltose binding protein MalE219 induces the CpxRA envelope stress response by stimulating phosphoryl transfer from CpxA to CpxR. Research in Microbiology 160 (2009) 396-400.
[2]Ivan A. D. Lwssard and Christopher T. Walsh. VanX, a bacterial D-alanyl-D-alanine dipeptidase: Resistance, immunity, or survival function? Proc. Natl. Acad. Sci. USA. Vol. 96, pp. 11028–11032, September, 1999.
[3]Hirofumi Hara, Lindsay D. Eltis, Julian E. Davies. Transcriptomic Analysis Reveals a Bifurcated Terephthalate
Degradation Pathway in Rhodococcus sp. Strain RHA1. Journal of Bacteriology, Mar. 2007, 189(5), 1641–1647.
[4]Molina-Henares, A. J., T. Krell, M. E. Guazzaroni, A. Segura, and J. L. Ramos. 2006. Members of the IclR family of bacterial transcriptional regulators function as activators and/or repressors. FEMS Microbiol. Rev. 30: 157–186.</I></P></DIV><DIV class="col-md-2 text-center"><A type="button" class="btn btn-next" href="https://2016.igem.org/Team:Tianjin/Note/R-R"><P>See our Notes</P></A></DIV><DIV class="col-md-2 text-center"><A type="button" class="btn btn-next" href="https://2016.igem.org/Team:Tianjin/Demonstrate#rrsystem"><P>See our Results</P></A></DIV></DIV><DIV class="sideNavbar" style="{border-color: #03a9f4}"><DIV id="jquery-accordion-menu" class="jquery-accordion-menu blue "><UL id="list"><LI><A href="#Overview">Overview </A></LI><LI><A class="topLink" href="#ReportingSystem">Reporting System</A></LI><LI><A class="topLink" href="#CellLysisBasedRegulationSystem">Cell Lysis Based Regulation System </A></LI><LI><A class="topLink" href="#TPAPositiveFeedbackBasedRegulationSystem">TPA Positive Feedback Based Regulation System</A></LI></UL><LI><A href="#TheoreticalBackground">Theories</A></LI><LI><A class="topLink" href="#TheCpxRegulationSystem">The Cpx Regulation System</A></LI><LI><A class="topLink" href="#DdpXcelllysiseffect">DdpX cell lysis effect</A></LI><LI><A class="topLink" href="#TPAPositiveFeedbackMechanism">TPA Positive Feedback Mechanism</A></LI><LI><A href="#ExperimentDesign1">Experiment Design 1</A></LI><LI><A class="topLink" href="#ConstructionofReportingSystem">Construction of Reporting System</A></LI><LI><A class="topLink" href="#VerificationofRFP">Verification of RFP in the part BBa_K339007 </A></LI><LI><A class="topLink" href="#MethodofRedFluorescenceAssay">Method of Red Fluorescence Assay</A></LI><LI><A class="topLink" href="#CultureandExpressionCondition">Culture and Expression Condition </A></LI><LI><A class="topLink" href="#ConstructionofCellLysis">Cell Lysis Based Regulation System</A></LI><LI><A href="#ExperimentDesign2">Experiment Design 2</A></LI><LI><A class="topLink" href="#VerificationofddpXGeneEffect">Verification of ddpX Gene Effect</A></LI><LI><A class="topLink" href="#MethodofCellLysisAssay">Method of Cell Lysis Assay</A></LI><LI><A class="topLink" href="#ChassisselectionforTPAPositive">Chassis selection for TPA Positive System</A></LI><LI><A class="topLink" href="#ConstructionofTPAPositiveFeedback">TPA Positive Feedback Based Regulation System</A></LI><LI><A class="topLink" href="#CultureandExpressionConditionsc">Culture and Expression Condition of S.c</A></LI><LI><A class="topLink" href="#FormerBiobrickVarification">Part Varification</A></LI><LI><A class="topLink" href="#ExpectedResults">Expected Results</A></LI><LI><A class="topLink" href="#References">References </A></LI></DIV><DIV class="sideNavCtrl"><DIV class="btn btn-info btn-raised sideNavShowBtn" id="sideNavShowBtn">Hide Navigation</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>