<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tianjin_Proof skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tianjin/Proof</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>TEAM TIANJIN</TITLE></P><DIV class="container-fluid"><DIV class="navbar-header"><DIV class="nameTeamTianjin">PLASTERMINATOR</DIV><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:Tianjin" id="Overview">OVERVIEW</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Projects">PROJECTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Description" id="Outline">OUTLINE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Experiment/Consortium" id="Consortium">CONSORTIUM</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Experiment/Protein_Engineering" id="ProteinEngineering">PROTEIN ENGINEERING</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Experiment/R-R" id="RRSystem">R-R SYSTEM</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Proof" id="Proof">PROOF</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Model" id="Model">MODEL</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Results">RESULTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Achievements" id="Achievements">ACHIEVEMENTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Demonstrate" id="Demonstrate">DEMONSTRATE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Parts" id="Parts">PARTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Future" id="FutureWork">FUTURE WORK</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="People">PEOPLE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Team" id="Team">TEAM</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Attributions" id="Attribution">ATTRIBUTION</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Outreach">OUTREACH</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Community/Overview" id="HumanPractice">HUMAN PRACTICE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Collaborations" id="Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Safety" id="Safety">SAFETY</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Notes">NOTES</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Notes" id="Notebook">NOTEBOOK</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Protocol" id="Protocol">PROTOCOL</A></LI></DIV></DIV><P><TITLE>Worthy</TITLE></P><DIV class="section clearfix object-non-visible" data-animation-effect="fadeIn"><DIV class="container"><DIV class="col-md-12"><H1 id="about" class="title text-center">Proof</H1><DIV class="row"><DIV class="col-md-10"><P style="font-size:18px"><B>
    Our project mainly consists of three integrated sections--Protein Modification, Microbial Consortium and Reporting-Regulation System. In each section there are many concepts related to our experiment design. Our project focused on proving these concepts by constructing new parts. Our proof of concepts are divided into three sections respectively belong to the three sections of our whole project. The detailed proof method and relative constructed devices are as follows.</B></P></DIV></DIV><DIV class="row"><H2><B>Protein Engineering</B></H2><H3><B>Outline</B></H3><P style="font-size:18px">The real goal of the protein engineering part in our project was to create a more active form of PETase protein. We designed and expressed 22 mutations in cell-free protein synthesis system, a brand new way to do the high-throughout assay. And from the 22 mutations, we successfully screened out a mutation I208V with hydrolysis activity improved by two-fold compared to the wild-type PETase.</P><H3><B>Rational Design</B></H3><H3>Serine-based Catalytic Triad Mechanism &amp; 3D Model Simulation</H3><P style="font-size:18px">Since PETase was found to contain a <B>GWSMG</B> motif in accordance with the <B>GXGXG</B> motif, which is characteristic and highly conserved in α/β hydrolase fold family, we simulated a best fit model for PETase by SWISSMODEL with a template as Thc_Cut2. As expected, the homology model of PETase displays a canonical α/β hydrolase fold with a Ser<SUP>160</SUP>-His<SUP>237</SUP>-Asp<SUP>206</SUP> catalytic triad and a preformed oxyanion hole (Fig.1), suggesting a classic serine hydrolase mechanism.
                        </P><DIV class="col-md-8"><DIV align="center"><FIGURE><FIGCATION>Fig.1 Simulated 3D structure for PETase</FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-12"><H3>Mutation Design</H3><P style="font-size:18px">Based on the simulated 3D structure, we found the 208th amino acid residue isoleucine situates right over the catalytic triad(Fig.2). It is apperant that the interaction of catalytic triad with substrate would be inhibited due to the space oppcupied by 208th isoleucine. So we decided to substitue 208th isoleucine with another smaller amino acid.
                        
                        However, as the isoleucine is the most hydrophobic amino acid, it takes risk to substitue it with another one, as it may lead to the hydrophobic amino acid decreasing. On balance, we chose valine as substitution, for its hydrophobicity is quite close to isoleucine, and the volume of valine is much smaller than that of isoleucine.
                        
                        The 3D structure of mutant I208V is as Fig.3. Comparing to wild-type PETase, the mutant I208V exposes the catalytic triad more widely. And the assay shows the stratety worked well for improving PET hydrolysis activity. The results are as follows.
                            </P></DIV><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.2 3D Structure for Wild-type PETase</FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.3 3D Structure for Mutant I208V PETase</FIGCATION></FIGURE></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12"><H3>Plasmid construction and expression in CFPS system</H3><P style="font-size:18px">Ligaion of digested pRset_CFP-1, digested CFP gene and digested PETase(I208V) gene in accordance with the 1:5:5 molecular ratio. The newly constructed plasmid is called pRset_CFP-1_PETase-I208V.Then, the plasmids pRset_CFP-1_PETase-I208V was put into the CFPS(Cell-Free Protein Synthesis) to synthesis the enzymes we expected. </P><DIV class="col-md-8"><DIV align="center"><FIGURE><FIGCATION>Fig.4 Plasmid construction and expression in CFPS ststem</FIGCATION></FIGURE></DIV></DIV></DIV></DIV><DIV class="row"><H3>Degradation data</H3><P style="font-size:18px">After the enzymes were expressed in the system successfully, we used the mutants we got to degrade PET and detected the degradation product, MHET, which has no other characteristic adsorption peak except at 260nm. 

    We detect the absorption peak of culture medium after culturing with PET film added for 5d. The absorption degree-wavelength curve is as follows.</P><DIV class="col-md-8"><DIV align="center"><FIGURE><FIGCATION>Fig.5 Spectra scanning for the degradation product</FIGCATION></FIGURE></DIV></DIV></DIV><DIV class="row"><P style="font-size:18px">We used Cyan fluorescence protein (CFP) as a real-time approach to monitor protein expression quantity by detecting the fluorescence signal in the plate-reader. The emission wavelength of the CFP is 479nm and the absorption wavelength is 435nm. This is the real-time detection of emission wavelength at 479nm. We detected the fluorescence absorption for 12 hours before adding PET film substrate.</P><DIV class="col-md-8"><DIV align="center"><FIGURE><FIGCATION>Fig.6 Screened plasmids expressions in the CFPS system</FIGCATION></FIGURE></DIV></DIV></DIV><DIV class="row"><P style="font-size:18px">To exclude the influence of different expression quantity, we divided the quantity of production of PET degradation of each mutant by its expression quantity to gain the relative activity, since the final concentration of product is positively relevant to absorption at 260nm and protein expression quantity is positively relevant to the absorption of CFP at emission wavelength.</P></DIV><DIV class="row"><DIV class="col-md-8"><DIV align="center"><FIGURE><FIGCATION>Fig.7 Relative enzyme activity of Plasmid pRset_CFP-1_PETase-I208V</FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-12"><P style="font-size:18px">So our result shows the rational design actually worked well and the mutant I208V shows a relatively improved hyfrolysis activity comparing to wild-type. Though as we can see in Fig.3, the substitution of isoleucine with valine is actually a very small change with a methyl on the 208<SUP>th</SUP> amino residue. However it leads to a two-fold improvement of enzyme activity, revealing that as we expected the 208<SUP>th</SUP> is a crucial site and the space adjacent to active center matters a lot. </P></DIV></DIV><H2><B>R-R System</B></H2><H3>Introduction</H3><P style="font-size:18px">
R-R (Reporting and Regulation)system is constructed in order to report the expression of PETase gene and regulate the expression process. In this section there are two main concepts we need to prove. The first is whether the ddpX gene can cause cell lysis and the other is whether the ddpX gene can cause cell lysis under the CpxR promoter when inclusion body form led by the overexpression of PETase gene. We constructed the part <A href="http://parts.igem.org/Part:BBa_K2110008" target="_blank"><I>BBa_K2110008</I></A> to prove the validity of this concept.</P><H3>Constructing Process</H3><P style="font-size:18px">
This part is modified and improved from the former part <A href="http://parts.igem.org/Part:BBa_K339007" target="_blank"><I>BBa_K339007</I></A> by changing the original mRFP gene to the novel ddpX gene (Part: <A href="http://parts.igem.org/Part:BBa_K2110004" target="_blank"><I>BBa_K2110004</I></A>). The ddpX gene was obtained by colony PCR of <I>E.coli</I> and was verified by sequencing. Considering there is no restriction endonuclease cutting site among the subparts, we used PCR to amplify the CpxR promoter-RBS sequence and linked it with the plasmid backbone and the ddpX gene. Then the new part was obtained. </P><H3>Proof Result</H3><P style="font-size:18px">
The novel ddpX gene is transformed into <I>E.coli</I> under the regulation of T7 promoter and lactose operon. We use IPTG to induce the overexpression of ddpX. Then we detected the OD<SUB>600</SUB> of culture medium to show the cell lysis effect of ddpX.</P><DIV class="col-md-8"><DIV align="center"><FIGURE><FIGCATION>Fig.8 OD<SUB>600</SUB>-culturing time curve of different groups of inclusion body induced cell lysis system</FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-12"><P style="font-size:18px">
From this image, we can see that in the first 5 hours, the OD<SUB>600</SUB> of each group is almost the same because of the rich nutrition and the ddpX cannot take effect immidiately. However, after 5 hours, the group with ddpX transformed and IPTG induced showed the fastest decrease among all the groups. This is because of the cell lysis effect of ddpX. Comparing the group with ddpX gene and IPTG inducted with the group with wildtype bacterial and the same amount of IPTG added we can draw the conclusion that the ddpX gene can cause cell lysis significantly. Therefore, when we replace the mRFP gene in the part <A href="http://parts.igem.org/Part:BBa_K339007" target="_blank"><I>BBa_K339007</I></A> with the ddpX gene, we can cause the cell lysis when the exogenous genes overexpress.
                             </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>