<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tianjin_Result_Consortium skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tianjin/Result/Consortium</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>TEAM TIANJIN</TITLE></P><DIV class="container-fluid"><DIV class="navbar-header"><DIV class="nameTeamTianjin">PLASTERMINATOR</DIV><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:Tianjin" id="Overview">OVERVIEW</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Projects">PROJECTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Description" id="Outline">OUTLINE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Experiment/Consortium" id="Consortium">CONSORTIUM</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Experiment/Protein_Engineering" id="ProteinEngineering">PROTEIN ENGINEERING</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Experiment/R-R" id="RRSystem">R-R SYSTEM</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Proof" id="Proof">PROOF</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Model" id="Model">MODEL</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Results">RESULTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Achievements" id="Achievements">ACHIEVEMENTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Demonstrate" id="Demonstrate">DEMONSTRATE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Parts" id="Parts">PARTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Future" id="FutureWork">FUTURE WORK</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="People">PEOPLE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Team" id="Team">TEAM</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Attributions" id="Attribution">ATTRIBUTION</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Outreach">OUTREACH</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Community/Overview" id="HumanPractice">HUMAN PRACTICE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Collaborations" id="Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Safety" id="Safety">SAFETY</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Notes">NOTES</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Notes" id="Notebook">NOTEBOOK</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Protocol" id="Protocol">PROTOCOL</A></LI></DIV></DIV><P><TITLE>Worthy</TITLE></P><DIV class="section clearfix object-non-visible" data-animation-effect="fadeIn"><DIV class="container"><DIV class="col-md-12"><H1 id="about" class="title text-center">Results of <SPAN>Consortium</SPAN></H1><H2><B id="DetailedresultsofConsortium">Detailed results of Consortium</B></H2><H2><B id="OptimizationofCultureConditions">Optimization of Culture Conditions</B></H2><H3><B id="Explorationforanappropriatemedium">1. Exploration for an appropriate medium</B></H3><P style="font-size:18px"> (1) Co-culture in LB medium </P><DIV class="col-md-12"><P style="font-size:15px;text-align:center">Table.1. Microscopic examination of Co-culture in LB medium </P></DIV><P style="font-size:18px"> Results of microscopic examination indicate that co-culturing these bacteria is impossible in LB medium.</P><P style="font-size:18px"> (2) Co-culture in YPD medium </P><DIV class="row"><DIV class="col-md-6"><P style="font-size:15px;text-align:center">Table.2. Microscopic examination of Co-culture in YPD medium </P></DIV><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.1 <I>Rhodococcus jostii RHA1</I> and <I>Y.lipolytica</I> under the microscope</FIGCATION></FIGURE></DIV></DIV></DIV><P style="font-size:18px"> Results of microscopic examination indicate that co-culturing these bacteria is impossible in YPD medium except <I>Rhodococcus jostii RHA1</I> and <I>Y.lipolytica</I>. </P><P style="font-size:18px"> (3) Co-culture in M9 medium</P><P style="font-size:18px"> We cultured bacteria as experiments for three days, and checked the bacterial concentration at OD<SUB>600</SUB>(See Fig.2 in details) and detectd the concentration of TPA by UV at OD<SUB>242</SUB>(See Fig.3 in details).</P><DIV class="row"><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.2 OD<SUB>600</SUB> in M9 medium after three-day culture</FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.3 Concentration of TPA in M9 medium after three-day culture</FIGCATION></FIGURE></DIV></DIV></DIV><P style="font-size:18px"> After analyzing graphs, easily, we can get a conclution that <I>Rhodococcus jostii RHA1 </I>can degrade TPA particularly well and<I> Pseudomonas putida KT2440 </I>can not degrading TPA. Obviously, <I>Rhodococcus jostii RHA1</I> and <I>Pseudomonas putida KT2440</I> can not live with each other in M9 medium because Group R.j + P.p is the same as control regarding concentration of TPA.</P><P style="font-size:18px"> (4) Co-culture in W medium </P><P style="font-size:18px"> We cultured bacteria as experiments for three days, and checked the bacterial concentration at OD<SUB>600</SUB>(See Fig.4 in details) and detectd the concentration of TPA by UV at OD<SUB>242</SUB>(See Fig.5 in details), then, observed some sample with microscope(See Fig.6 in details)</P><DIV class="row"><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.4 OD<SUB>600</SUB> in W medium after three-day culture</FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.5 Concentration of TPA in W medium after three-day culture</FIGCATION></FIGURE></DIV></DIV></DIV><P style="font-size:18px"> By Fig.4 and Fig.5, we can find that TPA of both Group R.j and Group R.j + P.p was degraded markedly, and degrading quantity of Group R.j + P.p is slightly larger than Group R.j. Then, by Fig.4, Fig.5, Fig.6, Fig.7, <I>Rhodococcus jostii RHA1</I> and <I>Pseudomonas putida KT2440</I> can be co-cultured in W medium. Besides, <I>Bacillus stubtilis</I> can grow better in W medium compared with M9 medium.</P><DIV class="row"><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.6 <I>Rhodococcus jostii RHA1</I> and <I>Pseudomonas putida KT2440</I> under the microscope
(Dyed by basic fuchsin)</FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.7<I>Rhodococcus jostii RHA1</I> and <I>Pseudomonas putida KT2440</I> under the microscope
(Dyed by Gram stain)
</FIGCATION></FIGURE></DIV></DIV></DIV><H3><B id="Adjustmentofnitrogensource">2. Adjustment of nitrogen source</B></H3><P style="font-size:18px">We choose two groups of experiments to show how to change growing condition of bacterial consortium by adjusting nitrogen source. Cell growth curve and TPA concentrations changes in W2 and W3 medium are shown as Fig.8, Fig.9 and Fig.10, Fig.11, respectively.</P><DIV class="row"><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.8 Cell growth curve with different strategies in W2 medium</FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.9 TPA concentrations changes with different strategies in W2 medium</FIGCATION></FIGURE></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.10 Cell growth curve with different strategies in W3 medium</FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.11 TPA concentrations changes with different strategies in W3 medium</FIGCATION></FIGURE></DIV></DIV></DIV><H3><B id="Temperature">3. Temperature</B></H3><P style="font-size:18px">We cultured<I> Rhodococcus jostii RHA1</I>,<I> Pseudomonas putida KT2440</I>, <I>Bacillus stubtilis 168</I> in W9 medium at 30 degrees and 37 degrees respectively. Growing conditions of bacteria the next day are shown as Fig.12.</P><DIV class="row"><DIV class="col-md-12"><DIV align="center"><FIGURE><FIGCATION>Fig.12 OD600 in W9 medium the next day at 30 and 37 degrees</FIGCATION></FIGURE></DIV></DIV></DIV><P style="font-size:18px">We found that<I> Bacillus stubtilis</I> grew a little better at 37 degrees, however, <I>Rhodococcus jostii</I> could not grow at 37 degrees. So we finally chose 30 degrees as culture degree of bacterial consortium.</P><H3><B id="InfluenceofCl">4. Influence of Cl<SUP>-</SUP></B></H3><P style="font-size:18px">By accident, I found <I>Bacillus stubtilis</I> could not grow well without Cl<SUP>-</SUP>, but <I>Rhodococcus jostii RHA1</I>,<I> Pseudomonas putida KT2440</I> grew as usual. Therefore, we prepared a series of medium(add NaCl to W medium without NH4Cl ) to explore influence of Cl<SUP>-</SUP>. Growing conditions of<I> Bacillus stubtilis</I> the next day are shown as Fig.13. </P><DIV class="row"><DIV class="col-md-12"><DIV align="center"><FIGURE><FIGCATION>Fig.13 OD600 with different concentration of Cl ions the next day</FIGCATION></FIGURE></DIV></DIV></DIV><P style="font-size:18px">From Fig.13, we conclude that increasing the concentration of Cl<SUP>-</SUP>can contribute to growth of <I>Bacillus stubtilis</I>, but after exceeding certain concentration, acceleration of Cl<SUP>-</SUP>disappears.</P><H3><B id="Adjustmentofcarbonsource">5. Adjustment of carbon source</B></H3><P style="font-size:18px">We added sucrose to improve growing condition of <I>Bacillus stubtilis</I>, then, Cell growth curve and TPA concentrations changes in W9 medium are shown as Fig.14, Fig.15, respectively.</P><DIV class="row"><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.14 Cell growth curve with different strategies in W9 medium</FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.15 TPA concentrations changes with different strategies in W9 medium</FIGCATION></FIGURE></DIV></DIV></DIV><H3><B id="Optimummedium">6.Optimum medium</B></H3><P style="font-size:18px">Basing on experiments above, we synthesize some conclusion and design some better media to give bacterial consortium a better surrounding to degrading PET. After processing data, we find W18 medium (without PET films) can make our consortium work better(Fig.16 and Fig.17).</P><DIV class="row"><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.16 Cell growth curve with different strategies in W18 medium(without PET films)</FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.17 TPA concentrations changes with different strategies in W9 medium(without PET films)</FIGCATION></FIGURE></DIV></DIV></DIV><P style="font-size:18px">To our delight, in the medium, <I>Bacillus stubtilis</I> can grow better than experiments before and group RPB can degrading TPA more efficiently. Eventually, we succeed in construct a relatively more stable bacterial consortium.</P><P style="font-size:18px">Later, in order to confirm coexist of three bacteria, we firstly observed sample under the microscope(Fig.18 and Fig.19).</P><DIV class="row"><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.18 Bacterial consortium under the microscope (Dyed by basic fuchsin)</FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.19 Bacterial consortium under the microscope (Dyed by Gram stain)</FIGCATION></FIGURE></DIV></DIV></DIV><P style="font-size:18px">Regrettably, results of microscopic examinations can not confirm coexist of three bacteria certainly, therefore, we use 16SrDNA to examine further. </P><H3><B id="TheSpecificityofPrimers">7.results of the first colony-pcr of 16s-rDNA: provident the specificity of the primers</B></H3><DIV class="row"><DIV class="col-md-6"><P style="font-size:18px"> The first round of colony-pcr helped us to prove the specificity of the primers. The number was collected in advance in Table 3. The result was showed in Figure 20. Only on three parallel lanes of the gel(number 1,5 and 9),three set of DNA molecules of known size(1000bp for number 1, 1500bp for number 5, 1600bp for number 9) were run. </P></DIV><DIV class="col-md-6"><P style="font-size:15px;text-align:center">Table.3. the number of 16s-rDNA in the first colony-pcr : the provement of primer</P></DIV></DIV><DIV class="row"><DIV class="col-md-12"><DIV align="center"><FIGURE><FIGCATION>Fig.20.the result of the first colony-pcr(Marker：2K-PLUS)</FIGCATION></FIGURE></DIV></DIV></DIV><P style="font-size:18px"> If number 1 had a set of DNA molecules rather than number 2/3, we got the conclusion that the primers—‘RJ-S/A’ could only get the 16s-rDNA gene of Rhodococcus RHA1 rather than ones in Pseudomonas putida KT2440 and Bacillus subtilis 168. Also,DNA molecules in number 5 and 9 could prove the specificity of the primers of other two bacteria. </P><H3><B id="ThePossibilityofConsortium">8.results of the second colony-pcr: provident the possibility of consortium</B></H3><DIV class="row"><DIV class="col-md-6"><P style="font-size:18px"> At the second round, we were going to prove the possibility of consortium. We cultivate each of them and the mixture of three in the modified W0 culture medium which changed the carbon source from glucose to sugar for amplification. The numbers and formula were collected in the Table 4. The result was showed in Figure 21.</P></DIV><DIV class="col-md-6"><P style="font-size:15px;text-align:center">Table.4. the number of 16s-rDNA in the second colony-pcr : the possibility of consortium</P></DIV></DIV><DIV class="row"><DIV class="col-md-12"><DIV align="center"><FIGURE><FIGCATION>Fig.21.the result of the second colony-pcr(Marker：2K-PLUS，after 2-day cultivation)</FIGCATION></FIGURE></DIV></DIV></DIV><P style="font-size:18px"> On six parallel lanes of the gel(number 1,5,9,10,11,12), six set of DNA molecules of known size( 1000bp for number 1 and 10 ; 1500bp for number 5 and 11; 1600bp for number 9 and 12) were run. From the DNA band of number 1,5 and 9，we could analyze that three bacteria growed well in each modified W0 culture medium. From the DNA band of number 10,11 and 12 , we could delightedly prove that three bacteria growed well together in the modified W0 culture medium.  </P><P style="font-size:18px"> In conclusion, we cultivated consortium of three bacteria in the modified W0 culture medium by the validation of 16s-rDNA. </P><H2><B id="ModificationofB.s">Modification of <I>Bacillus subtilis</I></B></H2><H3><B id="thedegreeofPETdegradation">1.the degree of PET degradation in<I> Bacillus subtilis</I></B></H3><DIV class="row"><DIV class="col-md-3"><P style="font-size:18px"> The Plasmid-PHP13-p43 is isolated and enzyme digested using EcoR I and BamH I restriction enzyme. After gel extraction to get right band, construction of pHP13-p43 + PETase and pHP13-p43 + MHETase are caught out.(Figure 22).</P></DIV><DIV class="col-md-8"><DIV align="center"><FIGURE><FIGCATION>Fig.22. the construction of recombinant plasmid</FIGCATION></FIGURE></DIV></DIV><P style="font-size:18px"> Then we transport the recombinant plasmid in <I>Bacillus subtilis 168</I> and<I> Bacillus subtilis DB104</I>. We cultivate four different strain(wild <I>Bacillus subtilis 168</I>，recombinant <I>Bacillus subtilis 168</I> ，wild <I> Bacillus subtilis DB104</I>，recombinant <I> Bacillus subtilis DB104</I>) and blank control group in LB medium with PET slices. After 7 days, we measure the absorbancy of the product from 230nm to 300nm (Figure 23). </P><DIV class="row"><DIV class="col-md-12"><DIV align="center"><FIGURE><FIGCATION>Fig.23.the product of degradation in different <I> Bacillus subtilis</I></FIGCATION></FIGURE></DIV></DIV><P style="font-size:18px"> The curves of degradation of PET in different <I>Bacillus subtilis</I> pass through a maximum when the absorbancy is within 260-280nm. Also, we found the improvement in recombinant one(B.s 168+PETase/ B.s DB104+PETase) compared to three control group(B.s 168/B.s DB104/control). That was the basic provement of degradation.</P><DIV class="row"><DIV class="col-md-12"><DIV align="center"><FIGURE><FIGCATION>Fig.24.the degrading ability in different <I> Bacillus subtilis</I></FIGCATION></FIGURE></DIV></DIV><P style="font-size:18px"> From the absorbancy at 265nm in the picture, we can compare the the degree of PET degradation.Easily to see in Figure 24, <I>Bacillus subtilis DB104</I> has a better secretory ability than <I>Bacillus subtilis 168</I> in consideration of the higher absorbancy at 265nm, which represent the absorption peak of MHET. Also, the result helped us to affirm that PETase gene expressed well in <I>Bacillus subtilis</I> after using codon optimization.  </P><H3><B id="thedegreeofPNPAdegradation1">2.The provement of </B></H3><H3><B id="thedegreeofPNPAdegradation">3.the degree of PNPA degradation——new discovery</B></H3><P style="font-size:18px">We make the first step of PNPA degradation after the success of transport. However, we fail to find the absorption peak of PNP at 400nm in experimental group.At first ,we  attribute this result to PNP decomposition(Figure 25).Then, we search on the Internet and find that Bacillus subtilis has the ability to degree PNP.So we cannot use the degree of PNPA degradation to measure enzymatic activity. </P><DIV class="row"><DIV class="col-md-8"><DIV align="center"><FIGURE><FIGCATION>Fig.25.Proposed degradation pathway for PNP in<I> Bacillus subtilis</I></FIGCATION></FIGURE></DIV></DIV></DIV><H2><B id="ModificationofP.p">Modification of<I>Pseudomonas putida KT2440</I></B></H2><H3><B id="ProofofoverexpressioninP.p">Proof of overexpression in <I>P.putida KT2440</I></B></H3><P style="font-size:18px"> After the experiment design was finally confirmed, we started to establish the overexpression vector based on the shuttle plasmid pBBR1MCS-2 as we described on the Experiment Page. The aim of this overexpression vector is to make KT2440 utilize EG more efficient and tend to accumulate PHA in bacterial body.</P><P style="font-size:18px"> After plasmid pBBRAA (pBBR linked with AcoA and AceA) was successfully transformed into E.coli, we amplified and extracted it from E.coli to prove that the target genes had been successfully ligated into pBBR. We proved it by PCR of pBBRAA and the result is showed as below.</P><DIV class="row"><DIV class="col-md-8"><DIV align="center"><FIGURE><FIGCATION>Fig.26.PCR of AcoA and AceA form plasmid pBBRAA</FIGCATION></FIGURE></DIV></DIV></DIV><P style="font-size:18px">The PCR’s results confirmed that the target genes were ligated into pBBR, which meant the overexpression was successfully established. Then, we transformed it into P.putida KT2440 by electroporation.</P><P style="font-size:18px">The electroporation succeeded and we started to test the overexpression vector. Due to the complexity of culture medium, we couldn’t use gas chromatography to detect the decrease of EG’s concentration, because compared with the standard sample, there were too many miscellaneous and overlapping peaks to distinguish the right peak. Under these circumstances, we redesigned the experiment to indirectly detect the increase of EG’s intake by test the bacterial concentration at OD600 with the same initial concentration of EG and bacteria.</P><DIV class="row"><DIV class="col-md-8"><P style="font-size:15px;text-align:center">Table.5</P></DIV></DIV><DIV class="row"><DIV class="col-md-12"><P style="font-size:18px">After 12-hour pre-culture, we detected the bacterial concentration at OD600 for 24h, and here is the result.</P></DIV></DIV><DIV class="row"><DIV class="col-md-8"><P style="font-size:15px;text-align:center">Table.27</P></DIV></DIV><DIV class="row"><DIV class="col-md-12"><P style="font-size:18px">From the figure, we could obviously find the bacterial concentration of P.putida with pBBRAA is _____ times higher than the wild type. It illustrated that our overexpression vector had worked well.
</P></DIV></DIV><H2><B id="AControllableLipidProducer">A Controllable Lipid Producer</B></H2><H3><B id="TheEfficiencyofLysisSystem">The efficiency of Lysis System</B></H3><DIV class="row"><DIV class="col-md-4"><P style="font-size:18px">For the construction of <I>Synechocystis 6803</I>-Inducible Lysis System, three lysis genes were synthesized by GENEWIZ (Figure 28), and we ligased them through overlap. After splicing those to the T vector, we put Pni, a promoter responding to Ni<SUP>2+</SUP>, to the upstream of lysis genes to make a nickel sensing signal system. Up to now, we constructed the expression vector successfully and transformed it into Synechocystis 6803 via electroporation.</P></DIV><DIV class="col-md-8"><DIV align="center"><FIGURE><FIGCATION>Fig.28. the construction of plasmid p3031</FIGCATION></FIGURE></DIV></DIV><P style="font-size:18px">We hope that after adding nickel sulfate to the cultures, cyanobacteria could split in a few days, releasing plenty of lipid to feed others as the source of carbon in our mixed bacteria system.</P><H2><B>Summary</B></H2><P style="font-size:22px"><B>In this part of experiment, we have achieved the following results:</B></P><P style="font-size:18px">
1.We successfully established a special bacteria consortium-<I>Bacillus stubtilis 168</I> or <I>DB104</I>, <I>Rhodococcus jostii RHA1</I> and <I>Pseudomonas putida KT2440</I> for this enzyme catalysis reaction. We use 16s-rDNA to prove the existence of special bacteria consortium.2.We optimized culture conditions by changing carbon source, nitrogen source and some ions. After checking the growth situations and conditions of the degrading PET, TPA and EG, we explore a suitable culture condition to co-culture our bacteria consortium.
3.We constructed pHP13-P43-PETase/MHETase plasmid for <I>Bacillus stubtilis 168</I> or <I>DB104</I>. The recombinant strain had the ability to degree PET. Also, we found PNPA couldn’t characterize of enzyme hydrolysis rate.
4. In order to increase the <I>Pseudomonas putida KT2440</I>’s intake of EG, we found the main influential genes---AceA and AcoA and successfully established an overexpression vector in <I>Pseudomonas putida KT2440</I>, which makes this bacterium can utilize EG more efficiently.
</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>