<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tianjin_Experiment_Consortium skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tianjin/Experiment/Consortium</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>TEAM TIANJIN</TITLE></P><DIV class="container-fluid"><DIV class="navbar-header"><DIV class="nameTeamTianjin">PLASTERMINATOR</DIV><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:Tianjin" id="Overview">OVERVIEW</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Projects">PROJECTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Description" id="Outline">OUTLINE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Experiment/Consortium" id="Consortium">CONSORTIUM</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Experiment/Protein_Engineering" id="ProteinEngineering">PROTEIN ENGINEERING</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Experiment/R-R" id="RRSystem">R-R SYSTEM</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Proof" id="Proof">PROOF</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Model" id="Model">MODEL</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Results">RESULTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Achievements" id="Achievements">ACHIEVEMENTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Demonstrate" id="Demonstrate">DEMONSTRATE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Parts" id="Parts">PARTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Future" id="FutureWork">FUTURE WORK</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="People">PEOPLE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Team" id="Team">TEAM</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Attributions" id="Attribution">ATTRIBUTION</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Outreach">OUTREACH</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Community/Overview" id="HumanPractice">HUMAN PRACTICE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Collaborations" id="Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Safety" id="Safety">SAFETY</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Notes">NOTES</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Notes" id="Notebook">NOTEBOOK</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Protocol" id="Protocol">PROTOCOL</A></LI></DIV></DIV><DIV class="section clearfix object-non-visible" data-animation-effect="fadeIn"><DIV class="container" style="width:75vw;margin-right:10%"><DIV class="col-md-12"><H1 id="about" class="title text-center"><SPAN>Bacteria Consortium</SPAN></H1><H2><B id="Overview">Overview</B></H2><P style="font-size:18px" id="OptimizationofCultureConditions">After Yoshida and his co-workers found and isolated <I>Ideonella sakaiensis 201-F6</I>, which produced two enzymes to degrades PET, we kept very high interests at their works and also came up with many ordinary ideas to increase the efficiency of degradation reaction. Bacteria consortium is one of the most creative ideas.</P><P style="font-size:18px">
The inspiration of this idea comes from nature and also learns from nature. Actually, bacteria never exist alone in our nature, they co-work and cooperate together to achieve an aim or live better in a special condition. Thinking from this point, we established a special bacteria consortium for this enzyme catalysis reaction.
</P><DIV class="row"><DIV class="col-md-7"><DIV align="center"><FIGURE><FIGCATION>Fig.1      Division of work in bacteria consortium</FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-5"><H3><B>1. Optimization of Culture Conditions</B></H3><P style="font-size:18px">In order to improve efficiency of degrading PET, we are determined to co-culture <I> Pseudomonas putida KT2440, Rhodococcus jostii RHA1 </I>and <I>Bacillus stubtilis 168</I> (or <I> Bacillus stubtilis DB 104</I>). In our bacteria consortium, work of degradation is divided several parts as follows:
1.<I>Rhodococcus jostii RHA1 </I> is responsible for degrading TPA (terephthalic acid) to remove substrate inhibition;
2. <I> Pseudomonas putida KT2440</I> is responsible for degrading EG (ethylene glycol) to remove substrate inhibition, and contribute to produce degradable plastics PHA (polyhydroxyalkanoate).
3. <I>Bacillus stubtilis 168</I> (or <I> Bacillus stubtilis DB 104</I>) is responsible for secreting PETase and MHETase as the main player of degrading PET.

</P></DIV></DIV><DIV class="row"><DIV class="col-md-4"><P style="font-size:18px">
Whereas, if our bacteria consortium want to achieve their aim, they must work in harmony, therefore, it is necessary find a appropriate environment where these bacteria can normally or supernormally work together. </P><P style="font-size:18px" id="ModificationofP.p1">Primarily, we try several kinds of medium and decide to use W medium in the end; next, we optimize culture conditions by change carbon source, nitrogen source and some ions, then, we check growing situations and conditions of the degrading PET, TPA and EG; eventually, 1we can find out a suitable culture condition to co-culture our bacteria consortium.


</P></DIV><DIV class="col-md-8"><DIV align="center"><FIGURE><FIGCATION>Fig.2      Idea about optimization of culture conditions</FIGCATION></FIGURE></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12"><H3><B>2. Modification of <I> Pseudomonas putida KT2440</I></B></H3><P style="font-size:18px"><I>P.putida KT2440</I> is one of bacteria which can utilize ethylene glycol (EG) at a high speed and meanwhile produces mcl-PHA. In 1988, Lageveen and his co-workers first found mcl-PHA in <I>P.putida KT2440</I>. And then, the metabolism of producing PHA in <I>P.putida KT2440</I> was reasearched, which found the gene AcoA was the key gene in the procedure. José Manuel Borrero-de Acuña and his co-workers improved the yield by 33% by overexpressing AcoA.
</P><P style="font-size:18px" id="Modification of B.s1">
From Björn Mückschel’s works, Ethylene Glycol Metabolism by Pseudomonas putida was found. The key enzymes were identified by comparative proteomics. In P. putida JM37, tartronate semialdehyde synthase (Gcl), malate synthase (GlcB), and isocitrate lyase (AceA) were found to be induced in the presence of ethylene glycol or glyoxylic acid. Under the same conditions, strain KT2440 showed induction of AceA only. 
</P><P style="font-size:18px"> From those studies, we decided to overexpress AcoA and AceA in <I>P.putida KT2440</I> to help utilize EG as energy source for its growth.
</P></DIV></DIV><DIV class="row"><DIV class="col-md-12"><H3><B id="AControllableLipidProducer1">3. Modification of <I> Bacillus subtilis</I></B></H3><P style="font-size:18px">After some attempt in <I>E.coli</I> and yeast, we look for a new type of host cells- <I>B.subtilis</I> for more secretion. In our experiment, the genes encoding two enzymes are for the first time expressed in <I>S.cerevisiae</I>. Increased yields of PETase and MHETase enzymes are achieved when <I>B. subtilis</I> strains 168 and DB104 (deficient in two and three extracellular proteases, respectively<SUP>[1]</SUP>) were transformed with the recombinant plasmid with the help of the enhanced promoter-p43.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12"><H3><B>4. A Controllable Lipid Producer
 </B></H3><P style="font-size:18px" id="DegradationofTPA">Cyanobacteria are excellent organisms for biofuel production. We thus have selected Cyanobacterium <I>Synechocystis sp. PCC 6803</I> as the source of carbon in our mixed bacteria system. Our target is simply to make the cyanobacteria lyse at the appropriate time by transforming a plasmid contained three bacteriophage-derived lysis genes which were placed downstream of a nickel-inducible signal transduction system into the <I>Synechocystis 6803</I>.</P><P style="font-size:18px" id="TheoreticalBackground">In this part of our project, Cyanobacterium Synechocystis sp. PCC 6803  was selected as a model organism as the source of carbon in our mixed bacteria system. We simply to establish a cell wall disruption process which could make the cyanobacteria lyse at the appropriate time. </P></DIV></DIV><DIV class="row"><DIV class="col-md-12"><H2><B>Theoretical Background</B></H2><DIV class="row"><DIV class="col-md-12"><H3><B>1. Degradation of Terephthalate</B></H3><P style="font-size:18px"><I>Rhodococcus sp. strain RHA1</I> is thought to be capable of degrading a wide range of aromatic compounds including terephthalate acid (TPA). in 2006, a reliable pathway consisting of Distinct ring cleavage dioxygenase systems and protocatechuate (PCA) pathway was come up with, and the proposed degradation pathway for TPA is shown as below<SUP>[2]</SUP>.</P></DIV></DIV><DIV class="row"><DIV class="col-md-8"><DIV align="center"><FIGURE><FIGCATION id="DegradationofEG">Fig.3    Proposed degradation pathway for TPA</FIGCATION></FIGURE></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.4   Postulated pathway for the metabolism of ethylene glycol in <I>Pseudomonas putida </I>strains KT2440 and JM37. The enzymes and/or metabolites identified in response to ethylene glycol in KT2440 are depicted in black. Additional pathways identified in strain JM37 are shown in gray. Detailed descriptions of the pathways are given in the text.<SUP>[3]</SUP></FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-6"><H3><B>2. Degradation of Ethylene Glycol
</B></H3><P style="font-size:18px">By employing growth and bioconversion experiments, directed mutagenesis, and proteome analysis, it is found that <I>Pseudomonas putida KT2440</I> does not grow within 2 days of incubation, compared to <I>Pseudomonas putida JM37</I> which can grow rapidly under the same conditions. The key enzymes and specific differences between the two strains were identified by comparative proteomics. In <I>P. putida JM37</I>, tartronate semialdehyde synthase (Gcl), malate synthase (GlcB), and isocitrate lyase (AceA) were found to be induced in the presence of ethylene glycol or glyoxylic acid. Under the same conditions, strain KT2440 showed induction of AceA only. Postulated pathway for the metabolism of ethylene glycol in Pseudomonas putida strains KT2440 and JM37 is shown left<SUP>[3]</SUP>.</P></DIV></DIV><DIV class="row"><DIV class="col-md-6"><H3><B id="ProductionofPHA">3. Production of Polyhydroxyalkanoate</B></H3><P style="font-size:18px"><I>Pseudomonas putida </I>is a natural producer of medium chain length polyhydroxyalkanoates (mcl-PHA), a polymeric precursor of bioplastics. A genome-based in silico model for <I>P. putida KT2440</I> metabolism was employed to identify potential genetic targets to be engineered for the improvement of mcl-PHA production using glucose as sole carbon source. Here, overproduction of pyruvate dehydrogenase subunit AcoA in the <I>P. putida KT2440</I> wild type led to an increase of PHA production. In controlled bioreactor batch fermentations PHA production was increased by 33% in the acoA overexpressing wild type in comparison to <I>P. putida KT2440</I>.</P><P style="font-size:18px" id="AdvantagesofB.s">
Transcriptome analyses of engineered PHA producing P. putida in comparison to its parental strains revealed the induction of genes encoding glucose 6-phosphate dehydrogenase and pyruvate dehydrogenase. In addition, NADPH seems to be quantitatively consumed for efficient PHA synthesis, since a direct relationship between low levels of NADPH and high concentrations of the biopolymer were observed. In contrast, intracellular levels of NADH were found increased in PHA producing organisms. Central metabolism of <I>P. putida KT2440</I> is shown right<SUP>[4]</SUP>.</P></DIV><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.5    Central metabolism of <I>P. putida KT2440<SUP>[4]</SUP></I></FIGCATION></FIGURE></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12"><H3><B>4. Advantages of <I>B. subtilis</I> strains
 </B></H3><P style="font-size:18px" id="EnhancedPromoter"><I>Bacillus subtilis</I> has an excellent secretion ability, displays fast growth, and is a nonpathogenic bacterium free of endotoxin <SUP>[5] </SUP>. It can, therefore, be used in food, enzyme, and pharmaceutical industries and can replace <I>Escherichia coli</I> for protein expression. Furthermore, the extracellular heterogeneous proteins secreted from <I>B. subtilis</I> are more convenient for recovery and purification in large-scale production during downstream processing <SUP>[6] </SUP>. </P></DIV></DIV><DIV class="row"><DIV class="col-md-12"><H3><B>5. Enhanced Promoter-p43
 </B></H3><P style="font-size:18px" id="TheCooperationofTwo"> In order to increase secretion, some enhanced promoters are necessary. However, native gene in a high-copy number plasmid was found to be unstable in <I>B. subtilis</I><SUP>[5] </SUP>. To optimize the production and the stability of the expression vectors, both the promoter and the signal sequence of PETase were replaced by <I>B. subtilis</I> P43 promoter, a constitutively expressed promoter. This overcame the plasmid instability problem. </P></DIV></DIV><DIV class="row"><DIV class="col-md-3"><DIV align="center"><FIGURE><FIGCATION>Fig.6  construction of plasmid of cooperation of p43+psacB promoter</FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-9"><H3><B>6. The Cooperation of Two Promoters-p43+p<SUB>sacB</SUB></B></H3><P style="font-size:18px" id="LipidProducer">Interestingly, the cooperation of two promoters in <I>B. subtilis</I> are easily found. For example, An endoglucanase from Bacillus akibai I-1 was successfully overexpressed in <I>Bacillus subtilis</I> 168 by the help of p43 promoter and the expression level of the recombinant enzyme was greatly enhanced by using the sucrose-inducible psacB promoter<SUP>[7] </SUP>.The construction of plasmid is in the Figure 1 . Thus, we are willing to try whether the combination of p43 and psacB can make a difference in the secretion of enzyme.</P></DIV></DIV><DIV class="row"><DIV class="col-md-8"><H3><B>7. Lipid Producer
 </B></H3><P style="font-size:18px"> Photosynthetic microorganisms, including eukaryotic algae and cyanobacteria, are being optimized to overproduce numerous biofuel. According to previous data, algae accumulate large quantities of lipid as storage materials, but they do this when under stress and growing slowly. By contrast, cyanobacteria accumulate lipids in thylakoid membranes, which are associated with high levels of photosynthesis and a rapid growth rate. Thus, photo-synthetic bacteria have a natural advantage for producing lipids at a high rate. Furthermore, being prokaryotes can be improved by genetic manipulations much more readily than can eukaryotic algae. <SUP>[8]</SUP> Therefore, we decided to do something to make cyanobacteria ,the lipid producer, more appropriate for our project.
</P><P style="font-size:18px" id="LipidRecoveryFromBiomass"><I>Synechococcus elongatus PCC7942</I> has larger capacity of lipid production than <I>Synechocystis sp. PCC6803</I> but accumulates most of the product in the cell because of the imbalance of the rates of lipid production and secretion. Initially, we intended to do something to increase lipid secretion by knocking the <I>wzt</I> gene<SUP>[9]</SUP> (Akihiro Kato et al. 2016), however, <I>Synechococcus elongatus PCC7942</I> wasn’t able to revive in two-week shaking cultivation. So we turned into <I>Synechocystis sp.PCC 6803</I>. </P></DIV><DIV class="col-md-4"><DIV align="center"><FIGURE><FIGCATION>Fig.7    Lipid contents</FIGCATION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-md-5"><DIV align="center"><FIGURE><FIGCATION>Fig.8   The functions of holin and endolysins in degrading the cell wall</FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-7"><H3><B>8. Lipid Recovery From Biomass</B></H3><P style="font-size:18px">The first goal of our research was to facilitate lipid recovery from biomass. The scientific community widespread disrupts the cyanobacterial cell envelope to achieve the goal. <SUP>[10]</SUP> (Seog JL et al. 1998)However, all these methods are not economical for large amounts of biomass or add additional cost and reduce the overall utility of the process. Our target is simply to make the cyanobacteria lyse at the appropriate time.</P><P style="font-size:18px" id="ControlTheLysisSystem">
We found that the cyanobacterial cell envelope is composed of 4 layers: the external surface layers ;the outer membrane; the polypeptidoglycan which is considerably thick, and the cytoplasmic membrane. <SUP>[11]</SUP> ( Hoiczyk E et al. 2000)To break up the peptidoglycan layer, we applied the holin-endolysin lysis strategy used by bacteriophages to exit bacterial cells<SUP>[12]</SUP> (Wang IN et al. 2000).Endolysins are peptidoglycan-degrading enzymes that attack the covalent linkages of the peptidoglycans that maintain the integrity of the cell wall. In addition to endolysins, some auxiliary lysis factors are involved in cleaving the oligopeptide linkages between the peptidoglycan and the outer membrane lipoprotein. Holins are small membrane proteins that produce nonspecific lesions (holes) in the cytoplasmic membrane from within, allow the endolysins and auxiliary lysis factors to gain access to the polypeptidoglycan layers, and trigger the lysis process. In this way, the cell wall is easy to break up.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12"><H3><B>9. Control The Lysis System</B></H3><P style="font-size:18px">To control the appropriate time, a nickel sensing/responding signal system<SUP>[13]</SUP> (Garcia-Dominguez M et al. 2000) was used to control the timing of the expression of phage lysis genes in Synechocystis 6803.</P><P style="font-size:18px" id="FindanAppropriateMedium">
Our strategy for achieving our target is to construct a expression vector pCPC3031-Ni-13-19-15  introduced the Salmonella phage P22 lysis cassette (<I>13-19-15</I>) with a Spectinomycin selection marker  downstream of the promoter Pni, a nickel responding signal operon. Synechocystis 6803 with the pCPC3031-Ni-13-19-15 will lyse after Ni2+ addition.
</P></DIV></DIV><DIV style="display: none"><H2><B id="ExperimentDesign">Experiment Design</B></H2><H1 style="font-size:200%"><B>Optimization of Culture Conditions</B></H1><DIV class="row"><DIV class="col-md-12"><H3><B>1. Find an Appropriate Medium</B></H3><P style="font-size:18px">Co-culture different pairs in improved W medium[17] in the same condition (using two tubes in each group): </P><P style="font-size:18px">Co-culture different pairs in improved M9 medium in the same condition (using two tubes in each group):</P><P style="font-size:18px">Co-culture different pairs in LB medium in the same condition(using two tubes in each group):</P><P style="font-size:18px">Co-culture different pairs in YPD medium in the same condition (using two tubes in each group):</P><P style="font-size:18px" id="NitrogenSource">
(PS: OD<SUB>600</SUB> of all the above bacteria solutions are 0.60, and these bacteria solutions are got by diluting all kinds of bacteria cultured in LB or YPD medium; and similarly hereinafter.)

Culture all groups at 30℃ &amp; 200 rpm for 3 days, check the bacterial concentration at OD<SUB>600</SUB> and observe these groups with microscope. 
(P.S. We always extract 200μL samples to each well in 96-well plate, and similarly hereinafter.)

Eventually, we find W medium is most promising to co-culture these bacteria.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12"><H3><B>2. Nitrogen Source</B></H3><P style="font-size:18px">We added some kinds of nitrogenous organic and inorganic compounds to W medium to improve initial W medium, and strategies of improvement are shown as follows.</P><P style="font-size:18px" id="CarbonSource">Add bacteria solution to media above as following table (use two tubes each group)</P><P style="font-size:18px">Culture them at 30℃ &amp; 200 rpm, checked the bacterial concentration at OD<SUB>600</SUB> and detect the concentration of TPA by UV at OD<SUB>242</SUB>, and then observe some samples with microscope.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12"><H3><B>3. Carbon Source</B></H3><P style="font-size:18px">We added some different kinds of carbonic organic compounds to W medium to improve initial W medium, and strategies of improvement are shown as follows.</P><P style="font-size:18px" id="MgandCl">Add bacteria solution to media above as following table (use two tubes each group)</P><P style="font-size:18px">Culture them at 30℃ &amp; 200 rpm, checked the bacterial concentration at OD<SUB>600</SUB> and detect the concentration of TPA by UV at OD<SUB>242</SUB>, and then observe some samples with microscope.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12"><H3><B>4. Mg<SUP>2+ </SUP>and Cl<SUP>-</SUP></B></H3><P style="font-size:18px">We add Mg<SUP>2+ </SUP> and Cl<SUP>-</SUP> to improve initial W medium, and strategies of improvement is shown as follows.</P><P style="font-size:18px" id="OrthogonalExperiments">Add bacteria solution to media above as following table (use two tubes each group)</P><P style="font-size:18px">Culture them at 30℃ &amp; 200 rpm, checked the bacterial concentration at OD<SUB>600</SUB> and detect the concentration of TPA by UV at OD<SUB>242</SUB>, and then observe some samples with microscope.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12"><H3><B>5. Orthogonal Experiments</B></H3><P style="font-size:18px">We added some different kinds of nitrogenous organic or inorganic compounds and carbonic organic compounds to W medium to improve initial W medium, and strategies of improvement are shown as follows.</P><P style="font-size:18px" id="Temperture">(PS: Because concentration of TPA is lower actually, we decrease concentration of sodium terephthalate.)

Add bacteria solution to mediums above as following table (use two tubes each group)
</P><P style="font-size:18px">Culture them at 30℃ &amp; 200 rpm, checked the bacterial concentration at OD<SUB>600</SUB> and detect the concentration of TPA by UV at OD<SUB>242</SUB>, and then observe some samples with microscope.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12"><H3><B>6. Temperture</B></H3><P style="font-size:18px">Culture bacteria as following strategies at 37℃ &amp; 200 rpm
</P><P style="font-size:18px">Checked the bacterial concentration at OD<SUB>600</SUB>.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12"><H3><B id="PreliminaryExperimentsforDegradingPET">7. Preliminary Experiments for Degrading PET</B></H3><P style="font-size:18px">We prepared media as following strategies:</P><P style="font-size:18px">(PS: Because concentration of TPA is lower actually, we decrease concentration of sodium terephthalate.)

Add bacteria solution to mediums above as following table (use two tubes each group)
</P><P style="font-size:18px">Culture them at 30℃ &amp; 200 rpm, checked the bacterial concentration at OD<SUB>600</SUB> and detect the concentration of TPA by UV at OD<SUB>242</SUB>, and then observe some samples with microscope.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12"><H3><B id="DegradationofPET">8. Degradation of PET</B></H3><P style="font-size:18px">Culture bacteria as following strategies:</P><P style="font-size:18px">(PS: Later, we substitute <I>Bacillus stubtilis DB 104</I> for <I>Bacillus stubtilis 168</I>.)

Culture them at 30℃ &amp; 200 rpm, checked the bacterial concentration at OD<SUB>600</SUB> and detect the concentration of TPA by UV at OD<SUB>242</SUB>, then, observe some sample with microscope.

We prepare some medium</P><P style="font-size:18px" id="Appendix">(W medium + 3g/L KNO3 + 3g/L glucose + 3g/L sucrose + 2.5g/L sodium terephthalate), 
then we add 20μL <I>Rhodococcus jostii RHA1</I>, 0.5μL <I>Pseudomonas putida KT2440</I>, 10μL <I>Bacillus stubtilis 168</I> (PETase) and 10μL <I>Bacillus stubtilis DB104</I> (MHETase), after that, Culture them at 30℃ &amp; 200 rpm, checked the bacterial concentration at OD<SUB>600</SUB> and detect the concentration of TPA by UV at OD<SUB>242</SUB>, then, observe some sample with microscope.
</P></DIV></DIV><DIV class="row"><DIV class="col-md-12"><H3><B>9. Appendix</B></H3><P style="font-size:18px">(1) We prepare a series of TPA standard solution and measure OD242, then plot standard curve. </P><DIV class="col-md-8"><DIV align="center"><FIGURE><FIGCATION>Fig.9.Standard curve of TPA with ELIASA</FIGCATION></FIGURE></DIV></DIV></DIV><DIV class="col-md-12"><P style="font-size:18px" id="the16SrDNA">
    (2)By dilution plating procedure[18], we find that concentration of <I>Bacillus stubtilis 168</I> or 
<I>Bacillus stubtilis DB104</I> is approximately 0.5 * 10^8 when OD600=0.8; and <I>Rhodococcus jostii RHA1</I> and <I>Pseudomonas putida KT2440</I> are 1*10^8 and1.2*10^8 when OD600=0.8, because dilution plating procedure is relatively imprecise, these relationships between OD and concentration of bacteria are just for reference.

(3)Measure OD242
        We firstly extract bacteria solutions to centrifugal tubes, then, centrifuge and extract supernatant 200μL to 96-well plate to measure OD242. </P></DIV></DIV><DIV class="row"><DIV class="col-md-6"><H3><B>10. 16S rDNA</B></H3><P style="font-size:18px">
    We validated our ideas that three kinds of bacteria were cooperating through the method of 16sRNA. The interesting thing is, three bacteria were living in harmony in modified W0 culture medium which changed the carbon source from glucose to sugar.
we firstly tried to cultivate each of them(Rhodococcus RHA1, Pseudomonas putida KT2440, bacillus subtilis 168) in LB culture medium for amplification. Then we take a little of culture for next work.we design 2 primers in each bacteria which could amplify special 16s-DNA from whole genome.(Table 1)
</P></DIV><DIV class="col-md-6"><P style="font-size:18px;text-align:center">
Table.1   16S rDNA Primers Information
</P></DIV></DIV><DIV class="row"><DIV class="col-md-6"><P style="font-size:18px;text-align:center">
Table.2   The number of 16s-rDNA in the first colony-pcr : the provement of primers
</P></DIV><DIV class="col-md-6"><P style="font-size:18px">
    We use orthogonal test to prove each bacteria had each special DNA stripe from the method of bacteria colony-pcr.In other words, we designed a 3*3 orthogonal test with the help of the bacteria liquid of three kinds and six primer sequences matched ribosome DNA of specific bacteria. The result was each bacteria had each special DNA stripe.
   Then we cultivate each of them and the mixture of three in W0 culture medium for amplification. Then we did the same 4*3(bacteria liquid and six primer sequences) orthogonal test. However, three bacteria failed to mix well, only rhodococcus lived.The number is collected in Table 2.

</P></DIV></DIV><DIV class="row"><DIV class="col-md-6"><P style="font-size:18px" id="OverexpressionofAcoAandAceA">
   We cultivate each of them and the mixture of three in in modified W0 culture medium which changed the carbon source from glucose to sugar for amplification. Then we still did the same 4*3(bacteria liquid and six primer sequences) orthogonal test. Luckily, three bacteria mixed well after cultivating for 12 hours. The number is collected in Table 3.

</P></DIV><DIV class="col-md-6"><P style="font-size:18px;text-align:center">
Table.3   The number of 16s-rDNA in the second colony-pcr: the possibility of consortium
</P></DIV></DIV></DIV><H1 style="font-size:200%"><B id="ModificationofP.p">Modification of <I> Pseudomonas putida KT2440</I></B></H1><DIV class="row"><DIV class="col-md-8"><H3><B>1. Overexpression of AcoA and AceA in <I>P.putida KT2440</I></B></H3><P style="font-size:18px" id="Electroporation">AcoA and AceA are the crucial genes for production of PHA and utilization of EG, so overexpressing these two genes is beneficial to improve the efficiency of degradation reaction and accumulation of PHA. Based on this idea, we established a expression vector which can be used in <I>P.putida KT2440</I>.

Based on the shuttle plasmid pBBR1MCS-2, we established the overexpression vector. The target genes were obtained by colony PCR and were ligated to the plasmid by T4 DNA ligase. The target genes are leaded by a T3 promoter.
After we established the vector, we transformed it to competent cell of E.coli and amplified it.
</P></DIV><DIV class="col-md-4"><DIV align="center"><FIGURE><FIGCATION>Fig.10   Overexpression vector in <I>P.putida KT2440</I></FIGCATION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-md-12"><H3><B>2. Electroporation of <I>P.putida KT2440</I></B></H3><P style="font-size:18px" id="DetectionofPHA">Procedure:
(1) 10 ml LB ＋ 0.1 ml Overnight culture of <I>P.putida KT2440</I>; 
(2) Centrifugation at 5000 rpm for 10 min at 4 degrees centigrade;
(3) Wash the precipitation (<I>P.putida KT2440</I>) with ice cold Elec-Buffer * 3;
(4) Re-suspend in 20 microliter Elec-Buffer;
(5) 4 microliter or 9 microliter suspension + 1 microliter of DNA(50 ng);
(6) Place 5 microliter or 10 microliter samples in Electrode Pin(5.6 mm diameter?) (50 microliter in Electrode Pin(1 mm diameter)?);
(7) Electroporation at 1.2 kV, 200 Ω, and 25 μF.
(8) Add 1-2 ml SOC Medium and culture at 30 degrees centigrade for 2 h 
(9) Spread on selection plate (LB agar with Kanamycin)

P.S. Elec-Buffer is 10% Glycerol.

</P></DIV></DIV><DIV class="row"><DIV class="col-md-12"><H3><B>3. Detection of PHA in <I>P.putida KT2440</I></B></H3><P style="font-size:18px" id="RecombinantPlasmidConstruction">Procedure:
(1)	take a certain amount of bacteria in the extraction bottle, add 15mL chloroform for each 1g bacteria, and place the extraction bottle in a high-pressure reaction kettle at 100 degrees Centigrade to react for 4h.
(2)	After cooling, filter.
(3)	Slowly add 10 times the volume of ethanol, cooling and stirring.
(4)	Place at 4 degrees centigrade over night.
(5)	After pressure reduction and pressure filtration, dehydrate the PHA in vacuum drying oven for 24h, weigh and record.

</P></DIV></DIV><H1 style="font-size:200%"><B id="ModificationofB.s">Modification of <I>Bacillus subtilis</I></B></H1><DIV class="row"><DIV class="col-md-8"><H3><B>1. Recombinant Plasmid Construction and Transformation</B></H3><P style="font-size:18px"><I>E.coli</I> with gene of PETase and MHETase and vector of pHP13-p43 has been inoculated in tubes of LB culture. The plasmid is constructed in advance（Figure 13）.

The Plasmids were isolated and enzyme digested using <I>EcoR I</I> and <I>BamH I</I> restriction enzyme. After gel extraction to get right band, construction of pHP13-p43 + PETase and pHP13-p43 + MHETase were caught out. The successfully constructed vectors, pHP13-p43 + PETase and pHP13-p43 + MHETase, were respectively transferred into <I>E.coli</I> and then <I>E.coli</I> were cultivated on chloramphenicol-containing LB plates to filter positive colony.（Figure 14） The positive colony of PETase and of MHETase were inoculated into tubes containing LB with chloramphenicol. Then well-constructed vectors were isolated. Vectors were respectively transferred into <I>B.subtilis</I> and then <I>B.subtilis</I> were cultivated on erythromycin-containing LB plates to filter positive colony. Above all, we got 2 strains of <I>B.subtilis</I> respectively secret PETase and MHETase.
</P></DIV><DIV class="col-md-4"><DIV align="center"><FIGURE><FIGCATION>Fig.11   the construction of pHP13-P43 plasmid in advance</FIGCATION></FIGURE></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-10"><DIV align="center"><FIGURE id="BioactivityofPETaseandMHETase"><FIGCATION>Fig.12   the construction of pHP13-P43-PETase/MHETase plasmid</FIGCATION></FIGURE></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12"><H3><B>2. Bioactivity of PETase and MHETase</B></H3><P style="font-size:18px">In order to test the bioactivity of PETase and MHETase, we have two methods. First, we use hydrolysis of pNPA through degradation rate. We cultivated wild <I>B.subtilis</I> and recombinant one in modified LB culture medium, which add PNPA solution through filtering(pNPA has a very low solubility in water).
After some days, we take 1 ml bacteria liquid each and centrifuge them for 12000r/min. Then we take the supernatant and measure solution absorbance at 400 nm, which is an obvious absorption peak of p-nitrophenol. If there is an an obvious absorption peak at 400 nm, we prove the bioactivity of PETase and MHETase in <I>B.subtilis</I>.
Second, we make orthogonal test. We cultivated wild <I>B.subtilis</I> and two recombinant one in LB culture medium（Figure 15）. We put PET in wild one and one of the recombinant <I>B.subtilis</I>. Then, we imitate of the first methods and measure solution absorbance at 260 and 240 nm , which is an obvious absorption peak of MHET and TPA.
</P><DIV class="row"><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.13.cultivation of  wild <I>B.subtilis</I> and two recombinant one in LB culture medium</FIGCATION></FIGURE></DIV></DIV></DIV></DIV></DIV><H1 style="font-size:200%"><B id="AControllableLipidProducer">A Controllable Lipid Producer</B></H1><DIV class="row"><DIV class="col-md-12"><H3><B id="Lysisgenes">1. Lysis genes(<I>13-19-15</I>)</B></H3><P style="font-size:18px">P22 gp13,P22 gp19 and P22 gp 15 are holins, endolysins and auxiliary lysis factors respectively. To construct the holin-endolysin lysis system,  they should be connected together with defined sequence. First of all, we obtained the three lysis genes which were synthesized by GENEWIZ separately. </P><P style="font-size:18px" id="ANickelSensingSignalSystem">
Then we use PCR to amplify this part. TA cloning and ligation of Blunt-ended DNA on the T vector were our original idea.However, <I>19</I> and <I>15</I> were spliced via TA cloning according to our presumption. <I>13</I> and <I>19-15</I> were ligated PCR overlap extension method of Warrens et al.<SUP>[19]</SUP>( Warrens AN et al. 1997)</P></DIV></DIV><DIV class="row"><DIV class="col-md-8"><DIV align="center"><FIGURE><FIGCATION>Fig.14   TA cloning and blunt end ligation</FIGCATION></FIGURE></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-6"><H3><B>2.A Nickel Sensing/Responding Signal System(pCPC3031-Ni)</B></H3><P style="font-size:18px" id="References">Ni activates the transcription of downstream genes of Pni and positively autoregulates its own synthesis. The amount of mRNA increased about 20-fold within 4 h after Ni addition. First, Pni was cloned into pCPC3031 and was amplified by using PCR. Then we cut the plasmid with Nru I, after that the lysis genes, <I>13-19-15</I>, were placed downstream of Pni. What’ more, we handed in the plasimids for sequencing, which confirmed its correctness.</P></DIV><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.15    Overlap PCR</FIGCATION></FIGURE></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12"><H2><B>References</B></H2><P style="font-size:16px">[1] He Wang, Ruijin Yang3,Xiao Hua, Wei Zhao, Wenbin Zhang. Functional Display of Active β-Galactosidase on <I>Bacillus subtilis</I> Spores Using Crust Proteins as Carriers

[2]Hirofumi Hara, Lindsay D. Eltis, Julian E. Davies, and William W. Mohn (2007): Transcriptomic Analysis Reveals a Bifurcated Terephthalate Degradation Pathway in <I>Rhodococcus sp. Strain RHA1</I>. In JOURNAL OF BACTERIOLOGY, Mar. 2007, p. 1641–1647. doi:10.1128/JB.01322-06

[3]Bjorn Mückschel,a Oliver Simon,b Janosch Klebensberger,a Nadja Graf,c Bettina Rosche,d Josef Altenbuchner,c Jens Pfannstiel,b Armin Huber,b and Bernhard Hauera (2012): Ethylene Glycol Metabolism by <I>Pseudomonas putida</I>. In Applied and Environmental Microbiology 78 (24), pp. 8531–8539. 

[4]José Manuel Borrero-de Acuña, Agata Bielecka, Susanne Häussler, Max Schobert, Martina Jahn, Christoph Wittmann, Dieter Jahn1 and Ignacio Poblete-Castro (2014): Production of medium chain length polyhydroxyalkanoate in metabolic flux optimized <I>Pseudomonas putida</I>. In Microbial Cell Factories 13 (88).

[5] Sen-Lin Liu, Kun Du. Enhanced expression of an endoglucanase in <I>Bacillus subtilis</I> by using the sucrose-inducible sacB promoter and improved properties of the recombinant enzyme

[6] Ruiqiong Ye, June-Hyung Kim, Byung-Gee Kim, Steven Szarka, Elaine Sihota, Sui-Lam Wong High-Level Secretory Production of Intact, Biologically Active Staphylokinase from <I>Bacillus subtilis</I>

[7] Zhongjun Chen,Cai Heng ,Zhengying Li ,Xinle Liang , Shangguan Xinchen Expression and secretion of a single-chain sweet protein monellin in <I>Bacillus subtilis</I> by sacB promoter and signal peptide

[8]Espaux L, Mendez-Perez D, Li R, Keasling JD (2015) Synthetic biology for microbial production of lipid-based biofuels. Curr Opin Chem Biol. 29:58-65

[9]Akihiro Kato, Kazuhide Use, Nobuyuki Takatani, Kazutaka Ikeda, Miyuki Matsuura, Kouji Kojima (2016) Modulation of the balance of fatty acid production and secretion is crucial for enhancement of growth and productivity of the engineered mutant of the <I>cyanobacterium Synechococcus</I> elongates. Biotechnol Biofuels9:91-101.

[10]Seog JL, Byung-Dae Y, O. H-M (1998) Rapid method for the determination of lipid fromthe green alga <I>Botryococcus braunii</I>. Biotechnol Tech 12:553–556.

[11]Hoiczyk E, HanselA(2000) <I>Cyanobacterial</I> cell walls: News from an unusual prokaryotic envelope. J Bacteriol 182:1191–1199.

[12]Wang IN, Smith DL, Young R (2000) Holins: The protein clocks of bacteriophage infections. Annu Rev Microbiol 54:799–825.

[13]Garcia-Dominguez M, Lopez-Maury L, Florencio FJ, Reyes JC (2000) A gene clusterinvolved in metal homeostasis in the <I>cyanobacterium Synechocystis</I> sp. strain PCC 6803. J Bacteriol 182:1507–1514.

[14] Kang Zhou, Kangjian Qiao, Steven Edgar &amp; Gregory Stephanopoulos.Distributing a metabolic pathway among a microbial consortium enhances production of natural products. doi:10.1038/nbt.3095

[15] Junjie Yang,Bingbing Sun,He Huang,Biao Chen,Chongmao Xu,Xin Wang,Jinle Liu ,Liuyang Diao .Multiple-site genetic modifications in <I>Escherichia coli</I> using lambda-Red recombination and I-SceI cleavage. doi:10.1007/s10529-015-1878-1

[16] T. B. Causey, S. Zhou, K. T. Shanmugam, and L. O. Ingram.Engineering the metabolism of <I>Escherichia coli</I> W3110 for the conversion of sugar to redox-neutral and oxidized products: Homoacetate production.

[17]MASASHI SETO, KAZUHIDE KIMBARA, MINORU SHIMURA, TAKASHI HATTA, MASAO FUKUDA, AND KEIJI YANO (1995): A Novel Transformation of Polychlorinated Biphenyls by <I>Rhodococcus</I> sp. Strain RHA1. In APPLIED AND ENVIRONMENTAL MICROBIOLOGY 61 (9), p. 3353–3358.

[18]Qian Ma (2015): The proteomic and metabolomic analyses of the consortia for vitamin C fermentation. Tianjin university.

[19]Warrens AN, Jones MD, Lechler RI (1997) Splicing by overlap extension by PCR using asymmetric amplification: An improved technique for the generation of hybrid proteins of immunological interest. Gene 186:29–35.</P></DIV></DIV></DIV></DIV><DIV class="col-md-12"><DIV class="col-md-2 text-center"><A type="button" class="btn btn-next" href="https://2016.igem.org/Team:Tianjin/Note/Consortium"><P>See our Notes about Consortium </P></A></DIV><DIV class="col-md-2 text-center"><A type="button" class="btn btn-next" href="https://2016.igem.org/Team:Tianjin/Note/6803"><P>See our Notes about Cyanobacteria</P></A></DIV><DIV class="col-md-2 text-center"><A type="button" class="btn btn-next" href="https://2016.igem.org/Team:Tianjin/Demonstrate#microbialconsortium"><P>See our Results</P></A></DIV></DIV><DIV class="sideNavbar" style="{border-color: #03a9f4}"><DIV id="jquery-accordion-menu" class="jquery-accordion-menu blue "><UL id="list"><LI><A href="#Overview">Overview </A></LI><LI><A class="topLink" href="#OptimizationofCultureConditions">Optimization of Culture Conditions </A></LI><LI><A class="topLink" href="#ModificationofP.p1">Modification of P.p</A></LI><LI><A class="topLink" href="#Modification of B.s1">Modification of B.s </A></LI><LI><A class="topLink" href="#AControllableLipidProducer1">A Controllable Lipid Producer </A></LI></UL><LI><A href="#">Theoretical Background</A></LI><LI><A class="topLink" href="#DegradationofTPA">Degradation of TPA </A></LI><LI><A class="topLink" href="#DegradationofEG">Degradation of EG</A></LI><LI><A class="topLink" href="#ProductionofPHA">Production of PHA </A></LI><LI><A class="topLink" href="#AdvantagesofB.s">Advantages of B.s </A></LI><LI><A class="topLink" href="#EnhancedPromoter">Enhanced Promoter-p43 </A></LI><LI><A class="topLink" href="#TheCooperationofTwo">The Cooperation of Two Promoters</A></LI><LI><A class="topLink" href="#LipidProducer">Lipid Producer </A></LI><LI><A class="topLink" href="#LipidRecoveryFromBiomass">Lipid Recovery From Biomass </A></LI><LI><A class="topLink" href="#ControlTheLysisSystem">Control The Lysis System </A></LI><LI><A href="#">Experiment Design </A></LI><LI><A href="#">Optimization of Culture Conditions one </A></LI><LI><A class="topLink" href="#FindanAppropriateMedium">Find an Appropriate Medium </A></LI><LI><A class="topLink" href="#NitrogenSource">Nitrogen Source </A></LI><LI><A class="topLink" href="#CarbonSource">Carbon Source </A></LI><LI><A class="topLink" href="#MgandCl">Mg<SUP>2+</SUP> and <SUP>Cl-</SUP></A></LI><LI><A class="topLink" href="#OrthogonalExperiments">Orthogonal Experiments </A></LI><LI><A href="#">Optimization of Culture Conditions two </A></LI><LI><A class="topLink" href="#Temperture">Temperture </A></LI><LI><A class="topLink" href="#PreliminaryExperimentsforDegradingPET">Preliminary Experiments for Degrading PET </A></LI><LI><A class="topLink" href="#DegradationofPET">Degradation of PET </A></LI><LI><A class="topLink" href="#Appendix">Appendix </A></LI><LI><A class="topLink" href="#the16SrDNA">16S rDNA </A></LI><LI><A href="#">Modification of P.p</A></LI><LI><A class="topLink" href="#OverexpressionofAcoAandAceA">Overexpression of AcoA and AceA in P.P</A></LI><LI><A class="topLink" href="#Electroporation">Electroporation of P.p</A></LI><LI><A class="topLink" href="#DetectionofPHA">Detection of PHA</A></LI><LI><A href="#">Modification of B.s</A></LI><LI><A class="topLink" href="#RecombinantPlasmidConstruction">Recombinant Plasmid Construction and Transformation </A></LI><LI><A class="topLink" href="#BioactivityofPETaseandMHETase">Bioactivity of PETase and MHETase </A></LI><LI><A href="#">A Controllable Lipid Producer </A></LI><LI><A class="topLink" href="#Lysisgenes">Lysis genes(13-19-15) </A></LI><LI><A class="topLink" href="#ANickelSensingSignalSystem">A Nickel Sensing/Responding Signal System(pCPC3031-Ni) </A></LI><LI><A class="topLink" href="#References">References </A></LI></DIV><DIV class="sideNavCtrl"><DIV class="btn btn-info btn-raised sideNavShowBtn" id="sideNavShowBtn">Hide Navigation</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>