<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tianjin_Note_Protein_Engineering skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tianjin/Note/Protein Engineering</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>TEAM TIANJIN</TITLE></P><DIV class="container-fluid"><DIV class="navbar-header"><DIV class="nameTeamTianjin">PLASTERMINATOR</DIV><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:Tianjin" id="Overview">OVERVIEW</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Projects">PROJECTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Description" id="Outline">OUTLINE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Experiment/Consortium" id="Consortium">CONSORTIUM</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Experiment/Protein_Engineering" id="ProteinEngineering">PROTEIN ENGINEERING</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Experiment/R-R" id="RRSystem">R-R SYSTEM</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Proof" id="Proof">PROOF</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Model" id="Model">MODEL</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Results">RESULTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Achievements" id="Achievements">ACHIEVEMENTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Demonstrate" id="Demonstrate">DEMONSTRATE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Parts" id="Parts">PARTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Future" id="FutureWork">FUTURE WORK</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="People">PEOPLE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Team" id="Team">TEAM</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Attributions" id="Attribution">ATTRIBUTION</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Outreach">OUTREACH</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Community/Overview" id="HumanPractice">HUMAN PRACTICE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Collaborations" id="Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Safety" id="Safety">SAFETY</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Notes">NOTES</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Notes" id="Notebook">NOTEBOOK</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Protocol" id="Protocol">PROTOCOL</A></LI></DIV></DIV><P><TITLE>Team Tianjin-Attribution</TITLE></P><DIV class="wrapper-white"><DIV id="page" class="hfeed"><DIV id="main"><DIV id="content" class="home blog single-author one-column content" role="main"><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><HEADER class="entry-header"><DIV class="entry-title" align="center">Notes For Protein Engineering</DIV></HEADER><H1 class="entry-title">Section 1</H1><DIV class="entry-content"><DIV class="note-content"><P><LI>PCR (Q5 DNA Polymerase Protocol) was applied to amplify single stranded DNA of site mutation for overlap in next step, 48 wells on plate was used. Gel electrophoresis implied that N205L_Reverse, S92A_Reverse, Q119A_Forward, Q199A_Reverse, S207T_Forward and M154A_Reverse were not properly amplified. After several trials and errors, annealing temperature was slightly reduced. Forward and reverse single stranded DNA of all 24 mutation sites were properly prepared.</LI><LI>All PCR product was purified (DNA Purification Protocol).</LI><LI>Overlapping forward and reverse single stranded DNA of all 24 mutation sites (Q5 DNA Polymerase Protocol). Considering several sites had bad gel electrophoresis results, touchdown PCR was introduced. We set the annealing temperature varying from 62 degrees centigrade to 55 degrees centigrade, which meant temperature would touch down 0.2 degrees centigrade every cycle in 35 PCR cycles.</LI><LI>After several trials and errors, optimized condition was found. All mutation sites PCR product were inspected through gel electrophoresis. Sites W159H, W159A, R90T, R90I, R90A, S92A, S207A were discarded for their bad stripe images implied the original primer might had been degraded due to exposed to relatively high temperature (our laboratory room temperature could occasionally rise to above 30 degrees centigrade). These sites were prepared again later.</LI><LI>Genome was extracted from Saccharomyces cerevisiae, and was used as template to prepare alpha-factor signal peptide. PCR was firstly applied with Q5 DNA Polymerase Protocol as annealing temperature was 65 degrees centigrade. Gel electrophoresis implied that alpha-factor signal peptide was not properly prepared. The second time we used Pfu DNA Polymerase Protocol and annealing temperature was set to 50 degrees centigrade. Gel electrophoresis implied alpha-factor signal peptide was properly prepared. Product was purified (DNA Purification Protocol).
</LI></P></DIV><A class="expand-btn">Show More</A></DIV><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><HEADER class="entry-header"><H1 class="entry-title">Section 2</H1></HEADER><DIV class="entry-content"><DIV class="note-content2"><P><LI>Help Cell-Free team prepare site mutation DNA segments. Q5 DNA Polymerase Protocol and Pfu Polymerase Protocol were both applied. To attain the best quality PCR product, more than 8 times PCR was performed on 96-wells plates, each time more than 20 wells were used. Touchdown PCR was also introduced.</LI><LI>Help Consortia team perform PCR on pBBR plasmid.</LI><LI>PCR (Pfu DNA Polymerase Protocol) was applied to prepare CYC1 terminator and was extracted from gel (Gel DNA Purification Protocol).</LI><LI>PCR (Pfu DNA Polymerase Protocol) was applied to prepare PGK promoter and was extracted from gel (Gel DNA Purification Protocol). After several trials and errors we found the rational condition to prepare PGK promoter gene.</LI><LI>Colony PCR was applied to help Consortia team verify genes on E.Coli genome.</LI><LI>Plasmid pRS416 was performed PCR with primers of alpha-factor signal peptide to check whether pRS416 was correctly linked with alpha-factor signal peptide. Result was negative, one hypothetical explanation was our plasmid donated by other laboratory fellows might have been damaged after a long period (at least 2 years) storage in fridge. We decide to substitute pRS415 for pRS416.</LI><LI>Help Consortia team perform PCR to prepare several genes.</LI></P></DIV><A class="expand-btn2">Show More</A></DIV><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><H1 class="entry-title">Section 3</H1><DIV class="entry-content"><DIV class="note-content3"><P><LI>PCR (Q5 DNA Polymerase Protocol) was applied to amplify PETase wild type gene, for future use.</LI><LI>We tried to combine all components including PGK promoter, CYC1 terminator, PETase mutant gene as well as alpha-factor signal peptide on plasmid pUC19. We tried to apply DNA Ligase Protocol to all 4 components simultaneously but found the efficiency was rather low. After several trials and errors we successfully prepared recombinant plasmid though it took several days. </LI><LI>We tried to link PETase mutant I208V gene to plasmid pRS415, and check whether it was successful use the same method described before. Gel electrophoresis confirmed that pRS415 was correctly linked with PETase mutant I208V gene. We also noticed that there exist 2 EcoRI cutting sites on pRS416 (one on resistance gene), which led to the failure in last week.</LI></P></DIV><A class="expand-btn3">Show More</A></DIV><H1 class="entry-title">Detailed Notes</H1><DIV class="entry-content"><DIV class="note-content3"><A class="media" href="https://static.igem.org/mediawiki/2016/b/b5/T--Tianjin--Notes_of_Protein_Engineering.pdf">Detailed Notes</A></DIV><A class="expand-btn3">Show More</A></DIV></ARTICLE></ARTICLE></ARTICLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>