<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tianjin_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tianjin/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>TEAM TIANJIN</TITLE></P><DIV class="container-fluid"><DIV class="navbar-header"><DIV class="nameTeamTianjin">PLASTERMINATOR</DIV><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:Tianjin" id="Overview">OVERVIEW</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Projects">PROJECTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Description" id="Outline">OUTLINE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Experiment/Consortium" id="Consortium">CONSORTIUM</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Experiment/Protein_Engineering" id="ProteinEngineering">PROTEIN ENGINEERING</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Experiment/R-R" id="RRSystem">R-R SYSTEM</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Proof" id="Proof">PROOF</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Model" id="Model">MODEL</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Results">RESULTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Achievements" id="Achievements">ACHIEVEMENTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Demonstrate" id="Demonstrate">DEMONSTRATE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Parts" id="Parts">PARTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Future" id="FutureWork">FUTURE WORK</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="People">PEOPLE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Team" id="Team">TEAM</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Attributions" id="Attribution">ATTRIBUTION</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Outreach">OUTREACH</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Community/Overview" id="HumanPractice">HUMAN PRACTICE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Collaborations" id="Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Safety" id="Safety">SAFETY</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Notes">NOTES</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Notes" id="Notebook">NOTEBOOK</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Protocol" id="Protocol">PROTOCOL</A></LI></DIV></DIV><DIV class="section clearfix object-non-visible" data-animation-effect="fadeIn"><DIV class="container" style="width:75%"><DIV class="col-md-12"><H1 id="about" class="title text-center">Results</H1><H2><B id="SystemWorkingUnderReal">System Working Under Real World Conditions.</B></H2><P style="font-size:18px" id="UsingtheRealWorld">
Synthetic biology aims at using biological part to construct different kinds of devices to realize different functions in real world. Experiments in the laboratory is idealized so it cannot represent the real world conditions all the time. In order to simulate the real world condition without taking the biological materials outside the lab, we can use PET products bought from the regular shop instead of the chemical shop. Considering the organisms are living with each other in real world, we applied the microbial consortium to simulate the real world condition as much as possible.
</P><H3><B>Using the Real World Substrate</B></H3><DIV class="row"><DIV class="col-md-6"><P style="font-size:18px">

Our project this year is about biodegradation of the widely used palstic PET. As a polymer, PET is hard to degrade and the degrading process is time-consuming. Due to the mild condition of the biodegradation reaction and the restriction of growing of organisms, it is even slower although it is more environmental friendly. Therefore, we use the substitute pNPA (p-nitrophenyl acetate), a kind of small molecular ester as the reagent of enzymatic reaction. However, pNPA is not what we really want to degrade, so we need to measure the PET degrading effects of our enzyme. Actually, pNPA is mainly used to screen for the mutated PETase gene because of the high requirement of efficiency of this step. When we obtain the mutants we want, we used them to degrade PET. In order to simulate the real world condition as much as possible, we used the PET film bought from the online plastic products shop instead of the test reagent bought from the chemical company. The film is originally used as packing material. We soak the cut film into the 75% ethanol solution and wash them with double distilled water before degrading experiments.

</P></DIV></DIV><H3><B>SEM image of degraded PET</B></H3><P style="font-size:18px">
The main characteration method of degradation of PET applied by us is measuring the absorption peak of product MHET or TPA. However, we think it will be much more intuitive if we can directly observe the degrading results. We used the scanning electron microscope (SEM) to directly observe the surface of PET film after dealt with PETase. We took pictures under different magnifications and the results are as follows.


</P><DIV class="row"><DIV class="col-md-4"><DIV align="center"><FIGURE><FIGCATION>Fig.1. SEM image of PET film surface cultured with <I>Bacillus subtilis</I> with no PETase gene transformed. (magnification: 50k)</FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-4"><DIV align="center"><FIGURE><FIGCATION>Fig.2. SEM image of PET film surface cultured with <I>Bacillus subtilis</I> with PETase gene transformed for 7 days. (magnification: 50k)</FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-4"><DIV align="center"><FIGURE><FIGCATION>Fig.3. SEM image of PET film surface cultured with <I>Bacillus subtilis</I> with PETase gene transformed for 19 days. (magnification: 50k)</FIGCATION></FIGURE></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-4"><DIV align="center"><FIGURE><FIGCATION>Fig.4. SEM image of PET film surface cultured with <I>Bacillus subtilis</I> with no PETase gene transformed. (magnification: 100k)</FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-4"><DIV align="center"><FIGURE><FIGCATION>Fig.5. SEM image of PET film surface cultured with <I>Bacillus subtilis</I> with PETase gene transformed for 7 days. (magnification: 100k)</FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-4"><DIV align="center"><FIGURE><FIGCATION>Fig.6. SEM image of PET film surface cultured with <I>Bacillus subtilis</I> with PETase gene transformed for 19 days. (magnification: 100k)</FIGCATION></FIGURE></DIV></DIV></DIV><P style="font-size:18px">

From these images, we can easily find that after cultured with Bacillus subtilis trasformed with PETase for only 7 days, the PET film showed clearly holes on its surface. What is more, with the extension of cultured time, the holes become more clear, as showed in the figure 3 and 6. These images undoubtedly verify that the PET film is degraded by the PETase synthetized by Bacillus subtilis.

</P><H3><B id="Usingthe">Using the Real World Simulated Microbial Consortium</B></H3><P style="font-size:18px">
The inspiration of the idea of microbial consortium comes from nature. Actually, bacteria never exist alone in our nature, they co-work and cooperate together to achieve an aim or live better in a special condition. Thinking from this point, we established a special bacteria consortium for this enzyme catalysis reaction in order to not only improve the degradation effect, but also simulate the real world condition. In this system ,we applied three kinds of organisms, <I>Pseudomonas putida KT2440</I>, <I>Rhodococcus jostii RHA1</I> and<I> Bacillus stubtilis 168</I> (or <I>Bacillus stubtilis DB 104</I>), to degrade the PET and the harmful product EG and TPA to carbon dioxide or generate biodegradable plastic PHA. The culture medium of this consortium contains significantly less carbon source than common culture medium, which has a clearly similarity with the nature environment. We also decided to introduce the Cyanobacterial into the system in the future in order to make this system use only solar energy to work.
</P><DIV class="row"><H2 id="proteinengineering"><B>Results of Protein Engineering</B></H2><P style="font-size:18px" id="Detailedresults">By expressing our twenty-two PETase mutants and doing the assay in our high-throughput strategyâ€”Cell-free protein synthesis, we successfully screened out a mutant with two-fold PET degradation activity compared to wild-type PETase. </P><P style="font-size:18px">We attempted to use two substrate for our mutant assay, one is PET film and the other is  p-nitrophenyl acetate (pNPA). pNP-aliphatic esters is a kind of universal assay substrate of PET degradation activity for its rapid reaction and visible color of hydrolysis product. However, as pNPA is soluble, the using of pNPA may not allow us to select out the more efficient mutant improved by being designed to enhance the surface hydrophobicity. Most notably, according to Yoshida et.al, the assay result of activity comparison between PETase, LC cutinase(LCC), and F.solani cutinase(FsC) shows that the activity for pNP-aliphatic esters of PETase is lower than that of LCC and FsC, however, the activity of PETase against PET film is 5.5 and 88 times as high as that of LCC and FsC[1]. Hence aiming at breaking PET plastic and solving real-condition problem, we finally decided to use PET film as our substrate and detect the hydrolysis product mono(2-hydroxyethyl) terephthalic acid (MHET) at an absorption of 260nm. Due to lacking of MHET standard (which is not a commercial agent), we failed to draw calibration curve of MHET at 260nm. So we compared the activity of mutants and wild-type with unit absorption, which is positively related to concentration of the detecting product. The result data are as follows.
 </P></DIV><DIV class="row"><DIV class="col-md-8"><DIV align="center"><FIGURE><FIGCATION>Fig.1 Spectra scanning for the degradation product MHET</FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-6"><DIV align="center"><P style="font-size:15px">The experiments were implemented at 39 centidregrees and statically reacted for 5 days with three parallels. The spectra was detected in a plate reader. This group includes 10 mutants, a wily-type PETase as a positive control and a blank pRset plasmid in CFPS system along with PET film as a negative control. The data is original with influence of expression quantity included. </P></DIV></DIV><DIV class="col-md-6"><DIV align="center"><P style="font-size:15px"><B>A</B>: A few mutants are of relatively low activity compared to wild-type, some may even be inactivated as their absorption curve shapes are not exactly the same with that of wild-type. But mutant I208V is much more efficient than wild-type. Also R90I and S207T showed higher activity compared to wild-type. <B>B</B>: We selected the mutants with higher degradation quantity. The position of peak and the value of absorption are noted on the curve.
        </P></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8"><DIV align="center"><FIGURE><FIGCATION>Fig.2 Screened plasmids expressions in the CFPS systemWe used Cyan fluorescence protein (CFP) as a real-time approach to monitor protein expression quantity by detecting the fluorescence signal in the plate-reader. The emission wavelength of the CFP is 479nm and the absorption wavelength is 435nm. This is the real-time detection of emission wavelength at 479nm. We detected the fluorescence absorption for 12 hours before adding PET film substrate.</FIGCATION></FIGURE></DIV></DIV></DIV><DIV class="row"><P style="font-size:18px">To exclude the influence of different expression quantity, we divided the quantity of production of PET degradation of each mutant by its expression quantity to gain the relative activity, since the final concentration of product is positively relevant to absorption at 260nm and protein expression quantity is positively relevant to the absorption of CFP at emission wavelength.</P><DIV class="row"><DIV class="col-md-8"><DIV align="center"><FIGURE><FIGCATION>Fig.3 Relative activities of enzymes</FIGCATION></FIGURE></DIV></DIV></DIV></DIV><P style="font-size:18px">We successfully screened two mutants (I208V &amp; R90A ) with higher enzyme activity by site-directed mutation.<B>You can find our part pages here:Â Â <A href="http://parts.igem.org/Part:BBa_K2110105"> I208V(BBa_K2110105)</A>Â Â Â Â <A href="http://parts.igem.org/Part:BBa_K2110104">R90A(BBa_K2110104)</A></B></P><DIV class="row"><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.4 3D Structure for Wild-type PETase</FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.5 3D Structure for Mutant I208V PETase</FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-12"><P style="font-size:18px">The 208<SUP>th</SUP> (with signal peptide excluded) amino acid residue isoleucine situates right over the catalytic triad(Fig.4). It is apperant that the interaction of catalytic triad with substrate would be inhibited due to the space oppcupied by 208th isoleucine. That's why we decided to substitue 208th isoleucine with another smaller amino acid.
                   
                    
                        Our result shows the rational design actually worked well and the mutant I208V shows a relatively improved hyfrolysis activity comparing to wild-type. Though as we can see in Fig.3, the substitution of isoleucine with valine is actually a very small change with a methyl on the 208<SUP>th</SUP> amino residue. However it leads to a two-fold improvement of enzyme activity, revealing that as we expected the 208<SUP>th</SUP> is a crucial site and the space adjacent to active center matters a lot. 
                                 </P></DIV></DIV><H2 id="microbialconsortium"><B id="Explorationforanappropriatemedium">Results of Microbial Consortium</B></H2><H3><B id="OptimizationofCultureConditions">Optimization of Culture Conditions</B></H3><H4><B>1. Exploration for an appropriate medium</B></H4><P style="font-size:18px"> (1) Co-culture in LB medium </P><DIV class="row"><DIV class="col-md-12"><DIV class="row"><DIV class="col-md-8"><P style="font-size:15px;text-align:center">Table.1. Microscopic examination of Co-culture in LB medium </P></DIV></DIV><P style="font-size:18px"> Results of microscopic examination indicate that co-culturing these bacteria is impossible in LB medium.</P></DIV></DIV><P style="font-size:18px"> (2) Co-culture in YPD medium </P><DIV class="row"><DIV class="col-md-8"><P style="font-size:15px;text-align:center">Table.2. Microscopic examination of Co-culture in YPD medium </P></DIV></DIV><DIV class="row"><DIV class="col-md-6"><P style="font-size:18px"> Results of microscopic examination indicate that co-culturing these bacteria is impossible in YPD medium except <I>Rhodococcus jostii RHA1</I> and <I>Y.lipolytica</I>. </P></DIV><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.1 <I>Rhodococcus jostii RHA1</I> and <I>Y.lipolytica</I> under the microscope</FIGCATION></FIGURE></DIV></DIV></DIV><P style="font-size:18px"> (3) Co-culture in M9 medium</P><P style="font-size:18px"> We cultured bacteria as experiments for three days, and checked the bacterial concentration at OD<SUB>600</SUB>(See Fig.2 in details) and detectd the concentration of TPA by UV at OD<SUB>242</SUB>(See Fig.3 in details).</P><DIV class="row"><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.2 OD<SUB>600</SUB> in M9 medium after three-day culture</FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.3 Concentration of TPA in M9 medium after three-day culture</FIGCATION></FIGURE></DIV></DIV></DIV><P style="font-size:18px"> After analyzing graphs, easily, we can get a conclution that <I>Rhodococcus jostii RHA1 </I>can degrade TPA particularly well and<I> Pseudomonas putida KT2440 </I>can not degrading TPA. Obviously, <I>Rhodococcus jostii RHA1</I> and <I>Pseudomonas putida KT2440</I> can not live with each other in M9 medium because Group R.j + P.p is the same as control regarding concentration of TPA.</P><P style="font-size:18px"> (4) Co-culture in W medium </P><P style="font-size:18px"> We cultured bacteria as experiments for three days, and checked the bacterial concentration at OD<SUB>600</SUB>(See Fig.4 in details) and detectd the concentration of TPA by UV at OD<SUB>242</SUB>(See Fig.5 in details), then, observed some sample with microscope(See Fig.6 in details)</P><DIV class="row"><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.4 OD<SUB>600</SUB> in W medium after three-day culture</FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.5 Concentration of TPA in W medium after three-day culture</FIGCATION></FIGURE></DIV></DIV></DIV><P style="font-size:18px"> By Fig.4 and Fig.5, we can find that TPA of both Group R.j and Group R.j + P.p was degraded markedly, and degrading quantity of Group R.j + P.p is slightly larger than Group R.j. Then, by Fig.4, Fig.5, Fig.6, Fig.7, <I>Rhodococcus jostii RHA1</I> and <I>Pseudomonas putida KT2440</I> can be co-cultured in W medium. Besides, <I>Bacillus stubtilis</I> can grow better in W medium compared with M9 medium.</P><DIV class="row"><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.6 <I>Rhodococcus jostii RHA1</I> and <I>Pseudomonas putida KT2440</I> under the microscope
(Dyed by basic fuchsin)</FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.7<I>Rhodococcus jostii RHA1</I> and <I>Pseudomonas putida KT2440</I> under the microscope
(Dyed by Gram stain)
</FIGCATION></FIGURE></DIV></DIV></DIV><H4><B id="Adjustmentofnitrogensource">2. Adjustment of nitrogen source</B></H4><P style="font-size:18px">We choose two groups of experiments to show how to change growing condition of bacterial consortium by adjusting nitrogen source. Cell growth curve and TPA concentrations changes in W2 and W3 medium are shown as Fig.8, Fig.9 and Fig.10, Fig.11, respectively.</P><DIV class="row"><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.8 Cell growth curve with different strategies in W2 medium</FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.9 TPA concentrations changes with different strategies in W2 medium</FIGCATION></FIGURE></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.10 Cell growth curve with different strategies in W3 medium</FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.11 TPA concentrations changes with different strategies in W3 medium</FIGCATION></FIGURE></DIV></DIV></DIV><P style="font-size:18px">By analyzing data we have had, Bacillus stubtilis grow best in W3 medium, and in W2 and W3 medium, group RPB can both degrading TPA well. Because Bacillus stubtilis is main player to secret PETase and MHETase, we decide to add KNO3 to adjust nitrogen source at last.</P><H4><B id="Temperature">3. Temperature</B></H4><P style="font-size:18px">We cultured<I> Rhodococcus jostii RHA1</I>,<I>Pseudomonas putida KT2440</I>, <I>Bacillus stubtilis 168</I> in W9 medium at 30 degrees and 37 degrees respectively. Growing conditions of bacteria the next day are shown as Fig.12.</P><DIV class="row"><DIV class="col-md-12"><DIV align="center"><FIGURE><FIGCATION>Fig.12 OD600 in W9 medium the next day at 30 and 37 degrees</FIGCATION></FIGURE></DIV></DIV></DIV><P style="font-size:18px" id="InfluenceofCl">We found that<I> Bacillus stubtilis</I> grew a little better at 37 degrees, however, <I>Rhodococcus jostii</I> could not grow at 37 degrees. So we finally chose 30 degrees as culture degree of bacterial consortium.</P><H4><B>4. Influence of Cl<SUP>-</SUP></B></H4><P style="font-size:18px">By accident, I found <I>Bacillus stubtilis</I> could not grow well without Cl<SUP>-</SUP>, but <I>Rhodococcus jostii RHA1</I>,<I> Pseudomonas putida KT2440</I> grew as usual. Therefore, we prepared a series of medium(add NaCl to W medium without NH4Cl ) to explore influence of Cl<SUP>-</SUP>. Growing conditions of<I> Bacillus stubtilis</I> the next day are shown as Fig.13. </P><DIV class="row"><DIV class="col-md-12"><DIV align="center"><FIGURE><FIGCATION>Fig.13 OD600 with different concentration of Cl ions the next day</FIGCATION></FIGURE></DIV></DIV></DIV><P style="font-size:18px" id="Adjustmentofcarbonsource">(PS: Because W9 medium includes NH4Cl that disrupts our exploration to influence of Cl-, so we replace NH4Cl with (NH4)2SO4 which consists of same concentration of NH4+.)From Fig.13, we conclude that increasing the concentration of Cl<SUP>-</SUP>can contribute to growth of <I>Bacillus stubtilis</I>, but after exceeding certain concentration, acceleration of Cl<SUP>-</SUP>disappears.</P><H4><B>5. Adjustment of carbon source</B></H4><P style="font-size:18px">We added sucrose to improve growing condition of <I>Bacillus stubtilis</I>, then, Cell growth curve and TPA concentrations changes in W9 medium are shown as Fig.14, Fig.15, respectively.</P><DIV class="row"><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.14 Cell growth curve with different strategies in W9 medium</FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.15 TPA concentrations changes with different strategies in W9 medium</FIGCATION></FIGURE></DIV></DIV></DIV><H4><B id="Optimummedium">6.Optimum medium</B></H4><P style="font-size:18px">Basing on experiments above, we synthesize some conclusion and design some better media to give bacterial consortium a better surrounding to degrading PET. After processing data, we find W18 medium (without PET films) can make our consortium work better(Fig.16 and Fig.17).</P><DIV class="row"><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.16 Cell growth curve with different strategies in W18 medium(without PET films)</FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.17 TPA concentrations changes with different strategies in W9 medium(without PET films)</FIGCATION></FIGURE></DIV></DIV></DIV><P style="font-size:18px">To our delight, in the medium, <I>Bacillus stubtilis</I> can grow better than experiments before and group RPB can degrading TPA more efficiently. Eventually, we succeed in construct a relatively more stable bacterial consortium.</P><P style="font-size:18px">Later, in order to confirm coexist of three bacteria, we firstly observed sample under the microscope(Fig.18 and Fig.19).</P><DIV class="row"><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.18 Bacterial consortium under the microscope (Dyed by basic fuchsin)</FIGCATION></FIGURE></DIV></DIV><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.19 Bacterial consortium under the microscope (Dyed by Gram stain)</FIGCATION></FIGURE></DIV></DIV></DIV><P style="font-size:18px" id="TheSpecificityofPrimers">Regrettably, results of microscopic examinations can not confirm coexist of three bacteria certainly, therefore, we use 16SrDNA to examine further. </P><H4><B id="xctezhi">7.results of the first colony-pcr of 16s-rDNA: provident the specificity of the primers</B></H4><DIV class="row"><DIV class="col-md-6"><P style="font-size:18px"> The first round of colony-pcr helped us to prove the specificity of the primers. The number was collected in advance in Table 3. The result was showed in Figure 20. Only on three parallel lanes of the gel(number 1,5 and 9),three set of DNA molecules of known size(1000bp for number 1, 1500bp for number 5, 1600bp for number 9) were run. </P></DIV><DIV class="col-md-6"><P style="font-size:15px;text-align:center">Table.3. the number of 16s-rDNA in the first colony-pcr : the provement of primer</P></DIV></DIV><DIV class="row"><DIV class="col-md-10"><DIV align="center"><FIGURE><FIGCATION>Fig.20.the result of the first colony-pcr(Markerï¼š2K-PLUS)</FIGCATION></FIGURE></DIV></DIV></DIV><P style="font-size:18px" id="ThePossibilityofConsortium"> If number 1 had a set of DNA molecules rather than number 2/3, we got the conclusion that the primersâ€”â€˜RJ-S/Aâ€™ could only get the 16s-rDNA gene of Rhodococcus RHA1 rather than ones in Pseudomonas putida KT2440 and Bacillus subtilis 168. Also,DNA molecules in number 5 and 9 could prove the specificity of the primers of other two bacteria. </P><H4><B>8.results of the second colony-pcr: provident the possibility of consortium</B></H4><DIV class="row"><DIV class="col-md-6"><P style="font-size:18px"> At the second round, we were going to prove the possibility of consortium. We cultivate each of them and the mixture of three in the modified W0 culture medium which changed the carbon source from glucose to sugar for amplification. The numbers and formula were collected in the Table 4. The result was showed in Figure 21.</P></DIV><DIV class="col-md-6"><P style="font-size:15px;text-align:center">Table.4. the number of 16s-rDNA in the second colony-pcr : the possibility of consortium</P></DIV></DIV><DIV class="row"><DIV class="col-md-10"><DIV align="center"><FIGURE><FIGCATION>Fig.21.the result of the second colony-pcr(Markerï¼š2K-PLUSï¼Œafter 2-day cultivation)</FIGCATION></FIGURE></DIV></DIV></DIV><P style="font-size:18px"> On six parallel lanes of the gel(number 1,5,9,10,11,12), six set of DNA molecules of known size( 1000bp for number 1 and 10 ; 1500bp for number 5 and 11; 1600bp for number 9 and 12) were run. From the DNA band of number 1,5 and 9ï¼Œwe could analyze that three bacteria growed well in each modified W0 culture medium. From the DNA band of number 10,11 and 12 , we could delightedly prove that three bacteria growed well together in the modified W0 culture medium.  </P><P style="font-size:18px"> In conclusion, we cultivated consortium of three bacteria in the modified W0 culture medium by the validation of 16s-rDNA. </P><H3><B id="ModificationofB.s">Modification of <I>Bacillus subtilis</I></B></H3><H4><B id="thedegreeofPETdegradation">1.the degree of PET degradation in<I> Bacillus subtilis</I></B></H4><DIV class="row"><DIV class="col-md-3"><P style="font-size:18px"> The Plasmid-PHP13-p43 is isolated and enzyme digested using EcoR I and BamH I restriction enzyme. After gel extraction to get right band, construction of pHP13-p43 + PETase and pHP13-p43 + MHETase are caught out.(Figure 22).</P></DIV><DIV class="col-md-8"><DIV align="center"><FIGURE><FIGCATION>Fig.22. the construction of recombinant plasmid</FIGCATION></FIGURE></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12"><P style="font-size:18px"> Then we transport the recombinant plasmid in <I>Bacillus subtilis 168</I> and<I> Bacillus subtilis DB104</I>. We cultivate four different strain(wild <I>Bacillus subtilis 168</I>ï¼Œrecombinant <I>Bacillus subtilis 168</I> ï¼Œwild <I> Bacillus subtilis DB104</I>ï¼Œrecombinant <I> Bacillus subtilis DB104</I>) and blank control group in LB medium with PET slices. After 7 days, we measure the absorbancy of the product from 230nm to 300nm (Figure 23). </P></DIV></DIV><DIV class="row"><DIV class="col-md-8"><DIV align="center"><FIGURE><FIGCATION>Fig.23.the product of degradation in different <I> Bacillus subtilis</I></FIGCATION></FIGURE></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12"><P style="font-size:18px"> The curves of degradation of PET in different <I>Bacillus subtilis</I> pass through a maximum when the absorbancy is within 260-280nm. Also, we found the improvement in recombinant one(B.s 168+PETase/ B.s DB104+PETase) compared to three control group(B.s 168/B.s DB104/control). That was the basic provement of degradation.</P></DIV></DIV><DIV class="row"><DIV class="col-md-8"><DIV align="center"><FIGURE><FIGCATION>Fig.24.the degrading ability in different <I> Bacillus subtilis</I></FIGCATION></FIGURE></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12"><P style="font-size:18px" id="thedegreeofPNPAdegradation"> From the absorbancy at 265nm in the picture, we can compare the the degree of PET degradation.Easily to see in Figure 24, <I>Bacillus subtilis DB104</I> has a better secretory ability than <I>Bacillus subtilis 168</I> in consideration of the higher absorbancy at 265nm, which represent the absorption peak of MHET. Also, the result helped us to affirm that PETase gene expressed well in <I>Bacillus subtilis</I> after using codon optimization.  </P></DIV></DIV><H4><B>2.the degree of PNPA degradationâ€”â€”new discovery</B></H4><P style="font-size:18px">We make the first step of PNPA degradation after the success of transport. However, we fail to find the absorption peak of PNP at 400nm in experimental group.At first ,we  attribute this result to PNP decomposition(Figure 25).Then, we search on the Internet and find that Bacillus subtilis has the ability to degree PNP.So we cannot use the degree of PNPA degradation to measure enzymatic activity. </P><DIV class="row"><DIV class="col-md-8"><DIV align="center"><FIGURE><FIGCATION>Fig.25.Proposed degradation pathway for PNP in<I> Bacillus subtilis</I></FIGCATION></FIGURE></DIV></DIV></DIV><H3><B id="ModificationofP.p">Modification of<I>Pseudomonas putida KT2440</I></B></H3><H4><B id="ProofofoverexpressioninP.p">Proof of overexpression in <I>P.putida KT2440</I></B></H4><P style="font-size:18px"> After the experiment design was finally confirmed, we started to establish the overexpression vector based on the shuttle plasmid pBBR1MCS-2 as we described on the Experiment Page. The aim of this overexpression vector is to make KT2440 utilize EG more efficient and tend to accumulate PHA in bacterial body.</P><P style="font-size:18px"> After plasmid pBBRAA (pBBR linked with AcoA and AceA) was successfully transformed into E.coli, we amplified and extracted it from E.coli to prove that the target genes had been successfully ligated into pBBR. We proved it by PCR of pBBRAA and the result is showed as below.</P><DIV class="row"><DIV class="col-md-8"><DIV align="center"><FIGURE><FIGCATION>Fig.26.PCR of AcoA and AceA form plasmid pBBRAA</FIGCATION></FIGURE></DIV></DIV></DIV><P style="font-size:18px">The PCRâ€™s results confirmed that the target genes were ligated into pBBR, which meant the overexpression was successfully established. Then, we transformed it into P.putida KT2440 by electroporation.However, we met some problem unsolved in the electroporation.So we need some improvement in the future.</P><DIV class="row"><DIV class="col-md-5"><H3><B id="AControllableLipidProducer">A Controllable Lipid Producer</B></H3><H4><B id="TheEfficiencyofLysisSystem">The efficiency of Lysis System</B></H4><P style="font-size:18px">For the construction of <I>Synechocystis 6803</I>-Inducible Lysis System, three lysis genes were synthesized by GENEWIZ (Figure 28), and we ligased them through overlap. After splicing those to the T vector, we put Pni, a promoter responding to Ni<SUP>2+</SUP>, to the upstream of lysis genes to make a nickel sensing signal system. 
Up to now, we constructed the expression vector successfully and transformed it into Synechocystis 6803 via electroporation.</P></DIV><DIV class="col-md-6"><DIV align="center"><FIGURE><FIGCATION>Fig.28. the construction of plasmid p3031</FIGCATION></FIGURE></DIV></DIV></DIV><P style="font-size:18px">We hope that after adding nickel sulfate to the cultures, cyanobacteria could split in a few days, releasing plenty of lipid to feed others as the source of carbon in our mixed bacteria system.</P><H2 id="rrsystem"><B>Results of R-R system</B></H2><H3><B id="Resultsofinclusionbody">Results of inclusion body based reporting system.</B></H3><P style="font-size:18px">The red fluorescence can be observed by bare eyes. We set up four groups: 
A. No <I>E.coli</I>.
B. <I>E.coli</I> with empty plasmid pUC19.
C. <I>E.coli</I> with pUC19+CpxR-RFP.
D. <I>E.coli</I> with pUC19+CpxR-RFP+PETase gene.
The result is as the following picture. (After centrifugation with the speed of 12000rpm for 1min)
</P><DIV class="row"><DIV class="col-md-6"><P style="font-size:15px;text-align:center">
Fig.1. Directed observed fluorescence of inclusion body based reporting system. (From left to right: A, B, C, D)
</P></DIV></DIV><P style="font-size:18px">
We found that the group with part <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K339007" target="_blank"><I>BBa_K339007</I></A> and PETase gene showed deepest red in all of the four groups. The group with only part <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K339007" target="_blank"><I>BBa_K339007</I></A> also showed a little red, we speculate that this might be caused by the basic expression of some genes in <I>E.coli</I> and some of the expression products formed inclusion bodies. It might also be caused by the basic expression of the RFP gene because the CpxR promoter might also, though not so strongly, start the transcription even without the induction of CpxR protein. The group with empty plasimd did not showed any red color and the group with no <I>E.coli</I> cultured did not have any sediment at the bottom of tube.
</P><H3><B>2.Results of cell lysis effect of ddpX gene.</B></H3><P style="font-size:18px">We first did a experiment to measure the cell lysis effect of ddpX gene. We simply use the IPTG inducible T7 promoter to regulate the expression of ddpX gene. We totally set four groups:
1. <I>E.coli </I> wildtype.
2. <I>E.coli </I> wildtype added IPTG.
3. <I>E.coli </I> with ddpX gene, no induction.
4. <I>E.coli </I> with ddpX gene, IPTG added as induction.
Then we continuously measure the OD<SUB>600</SUB> of the culture medium by 96-well Microplate Reader and draw the OD<SUB>600</SUB>-culturing time curve by Matlab. The graph is showed below.

</P><DIV class="row"><DIV class="col-md-8"><DIV align="center"><FIGURE><FIGCATION>Fig.2. OD<SUB>600</SUB>-culturing time curve of different groups of inclusion body induced cell lysis system.
</FIGCATION></FIGURE></DIV></DIV></DIV><P style="font-size:18px" id="Resultsofinclusionbodybased"> 
From this image, we can see that in the first 5 hours, the OD<SUB>600</SUB> of each group is almost the same because of the rich nutrition and the ddpX cannot take effect immidiately. However, after 5 hours, the group with ddpX transformed and IPTG induced showed the fastest decrease among all the groups. This is because of the cell lysis effect of ddpX. Comparing the group with ddpX gene and IPTG inducted with the group with wildtype bacterial and the same amount of IPTG added we can draw the conclusion that the ddpX gene can cause cell lysis significantly. However, the data of another two groups are a little strange. We think it is because of the basic expression of ddpX can partly hydrolyze the peptidoglycan in the cell wall and provide cells with D-Ala as carbon source. This is exactly the physical use the the ddpX gene when the cell is under starvation condition, <A href="https://2016.igem.org/Team:Tianjin/Experiment/R-R#ddpx" target="_blank">see more details about this explanation in our experiment page</A>. Therefore the group with ddpX gene but no induction can live longer and better when the nutritions are deficient.
</P><H2><B>Summary</B></H2><P style="font-size:18px">

1.We implemented the cell-free protein synthesis system as a rapid high-throughput assay method.
    
2.We designed twenty-two PETase muatnts and successfully screened two mutants of the wild type <I>PETase</I> with higher enzyme activity.

3.We successfully established a special bacteria consortium-<I>Bacillus stubtilis 168</I> or <I>DB104</I>, <I>Rhodococcus jostii RHA1</I> and <I>Pseudomonas putida KT2440</I> for this enzyme catalysis reaction. We use 16s-rDNA to prove the existence of special bacteria consortium.
4.We optimized culture conditions by changing carbon source, nitrogen source and some ions. After checking the growth situations and conditions of the degrading PET, TPA and EG, we explore a suitable culture condition to co-culture our bacteria consortium.

5.We constructed pHP13-P43-PETase/MHETase plasmid for <I>Bacillus stubtilis 168</I> or <I>DB104</I>. The recombinant strain had the ability to degree PET. Also, we found PNPA couldnâ€™t characterize of enzyme hydrolysis rate.

6. In order to increase the <I>Pseudomonas putida KT2440</I>â€™s intake of EG, we found the main influential genes---AceA and AcoA and successfully established an overexpression vector in <I>Pseudomonas putida KT2440</I>, which makes this bacterium can utilize EG more efficiently.

7.We successfully applied and improved the previous part BBa_K339007 designed and constructed by the Group iGEM10_Calgary to construct our inclusion body based reporting system and inclusion body based cell lysis regulation system.  
8.The red fluorescence can even be directly observed by bare eyes when we express the PETase gene in <I>E.coli</I> which was transformed into the CpxR-RFP fragment. 
9.We measured the cell lysis effect of the novel ddpX gene and combined it with the PETase expression system in <I>E.coli</I>.</P></DIV></DIV><DIV class="sideNavbar" style="{border-color: #03a9f4}"><DIV id="jquery-accordion-menu" class="jquery-accordion-menu blue "><UL id="list"><LI><A class="topLink" href="#Overview">Overview </A></LI><LI><A href="#SystemWorkingUnderReal">Under Real World</A></LI><LI><A class="topLink" href="#UsingtheRealWorld">Substrate</A></LI><LI><A class="topLink" href="#Usingthe">Microbial Consortium </A></LI></UL><LI><A class="topLink" href="#proteinengineering">Protein Engineering</A></LI><LI><A href="#microbialconsortium">Microbial Consortium</A></LI><LI><A href="#OptimizationofCultureConditions">Optimization of Culture Conditions</A></LI><LI><A class="topLink" href="#Explorationforanappropriatemedium">Appropriate medium</A></LI><LI><A class="topLink" href="#Adjustmentofnitrogensource">nitrogen source</A></LI><LI><A class="topLink" href="#Temperature">Temperature</A></LI><LI><A class="topLink" href="#InfluenceofCl">Influence of Cl</A></LI><LI><A class="topLink" href="#Adjustmentofcarbonsource">carbon source</A></LI><LI><A class="topLink" href="#Optimummedium">Optimum medium</A></LI><LI><A href="#xctezhi">The 16s-rDNA Method</A></LI><LI><A class="topLink" href="#TheSpecificityofPrimers">Specificity of Primers</A></LI><LI><A class="topLink" href="#ThePossibilityofConsortium">Possibility of consortium</A></LI><LI><A href="#ModificationofB.s">Modification of B.s</A></LI><LI><A class="topLink" href="#thedegreeofPETdegradation">PET degradation</A></LI><LI><A class="topLink" href="#thedegreeofPNPAdegradation">PNPA degradation</A></LI><LI><A class="topLink" href="#ModificationofP.p">Modification of P.p</A></LI><LI><A class="topLink" href="#AControllableLipidProducer">Controllable Lipid Producer</A></LI><LI><A href="#rrsystem">R-R system</A></LI><LI><A class="topLink" href="#Resultsofinclusionbody">reporting system.</A></LI><LI><A class="topLink" href="#Resultsofinclusionbodybased">regulation system</A></LI><LI><A class="topLink" href="#Summary3">Summary</A></LI></DIV><DIV class="sideNavCtrl"><DIV class="btn btn-info btn-raised sideNavShowBtn" id="sideNavShowBtn">Hide Navigation</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>