<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tianjin_Note_R-R skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tianjin/Note/R-R</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>TEAM TIANJIN</TITLE></P><DIV class="container-fluid"><DIV class="navbar-header"><DIV class="nameTeamTianjin">PLASTERMINATOR</DIV><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:Tianjin" id="Overview">OVERVIEW</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Projects">PROJECTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Description" id="Outline">OUTLINE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Experiment/Consortium" id="Consortium">CONSORTIUM</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Experiment/Protein_Engineering" id="ProteinEngineering">PROTEIN ENGINEERING</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Experiment/R-R" id="RRSystem">R-R SYSTEM</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Proof" id="Proof">PROOF</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Model" id="Model">MODEL</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Results">RESULTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Achievements" id="Achievements">ACHIEVEMENTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Demonstrate" id="Demonstrate">DEMONSTRATE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Parts" id="Parts">PARTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Future" id="FutureWork">FUTURE WORK</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="People">PEOPLE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Team" id="Team">TEAM</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Attributions" id="Attribution">ATTRIBUTION</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Outreach">OUTREACH</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Community/Overview" id="HumanPractice">HUMAN PRACTICE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Collaborations" id="Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Safety" id="Safety">SAFETY</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Notes">NOTES</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Notes" id="Notebook">NOTEBOOK</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Protocol" id="Protocol">PROTOCOL</A></LI></DIV></DIV><P><TITLE>Team Tianjin-Attribution</TITLE></P><DIV class="wrapper-white"><DIV id="page" class="hfeed"><DIV id="main"><DIV id="content" class="home blog single-author one-column content" role="main"><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><HEADER class="entry-header"><DIV class="entry-title" align="center">Notes for R-R System</DIV></HEADER><H1 class="entry-title">Week1(8/24/2016-8/30/2016)</H1><DIV class="entry-content"><DIV class="note-content"><LI>We used PCR to amplify the <B><I>CpxR</I></B> promoter and <B><I>RFP</I></B> gene from plasmid <B><I>pUC57</I></B>, and then we recycled the amplified fragment from the agarose gel. Then we used<B><I> Xba1</I></B> and <B><I>Pst1</I></B> enzymes to cut the plasmid <B><I>pUC19</I></B> and <B><I>CpxR-RFP</I></B> fragment. </LI><LI>We linked the cut plasmid and<B><I> CpxR-RFP</I></B> fragment together and transformed the recombinant plasmid to <B><I>E.coli</I></B>.
We used PCR to amplify the <B><I>PETase</I></B> gene and then recycled them from the agarose gel.</LI><LI>We cultured the grown-up<B><I> E.coli</I></B> which had been transformed into recombinant <B><I>pUC19</I></B> into liquid LB+Amp culture medium overnight.</LI><LI>We isolated the recombinant plasmid from the<B><I> E.coli</I></B> cultured last night. Then we use<B><I> Xba1</I></B> and <B><I>Pst1</I></B> enzyme to cut the plasmid to verify the plasmid was successfully constructed. The result was we succeeded.</LI><LI>We cut the recombinant plasmid <B><I>pUC19</I></B> with enzyme<B><I> EcoR1</I></B> and <B><I>Sac1</I></B> and then we recycled it from the agarose gel. We stored the recycled product in -30℃ in order to wait for the <B><I>PETase</I></B> gene transformed into it.</LI><LI>In order to verify the inclusion body sensing effects of <B><I>CpxR</I></B> promoter, we selected a colony of <B><I>E.coli</I></B> with recombinant plasmid<B><I> pUC19</I></B> and cultured them in 37℃ for 6 hours and then rose the temperature to 42℃ and culture it overnight.</LI><LI>The result of the verification experiment last night was unsuccessful for there was only ultralow red fluorescence was detected, which was considered the basic expression of <B><I>RFP</I></B>.</LI><LI>We redesigned the experiment and set 3 groups:
1. E.coli with standard <B><I>RFP</I></B> gene from our own laboratory.
2. E.coli with our recombinant plasmid<B><I> pUC19</I></B> and we cultured them in 37℃ all through.
3. E.coli with our recombinant plasmid <B><I>pUC19</I></B> and we first cultured them in 37℃ and after 6 hours transferred them to 42℃.
</LI><LI>The result was still unsuccessful for the 2nd and 3rd group showed ultralow red fluorescence and only 1st group showed high enough red fluorescence.</LI><LI>We redesigned the experiment again. We decided to transformed the recombinant plasmid <B><I>pUC19</I></B> and<B><I> pET21a</I></B> which was from the protein modification group and had <B><I>PETase</I></B> gene in it into<B><I> E.coli</I></B> at the same time.</LI><LI>We cut the<B><I> pUC57</I></B> with enzyme <B><I>Xba1</I></B> and <B><I>Pst1</I></B> and recycled the skeleton part from the agarose gel.</LI></DIV><A class="expand-btn">Show More</A></DIV><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><HEADER class="entry-header"><H1 class="entry-title">Week2(8/31/2016-9/6/2016)</H1></HEADER><DIV class="entry-content"><DIV class="note-content2"><LI>We linked the remained cut <B><I>CpxR-RFP</I></B> fragment into the skeleton and then transformed the recombinant <B><I>pUC57</I></B> and the <B><I>pET21a</I></B> into<B><I> E.coli</I></B> at the same time.</LI><LI>The transformation last night turned to be a failure. We tried it again.</LI><LI>The transformation last day seemed to be successful for the colonies were visible in LB+Amp plate. However, we use PCR to verify and it turned out that the fragment had not been linked into the plasmid.</LI><LI>We finally gave up the former design and decided to link the<B><I> PETase</I></B> gene into the plasmid<B><I> pUC19</I></B>. However, we did not have the key enzyme <B><I>Sal1</I></B> so we started to construct the TPA positive feedback system.</LI><LI>We first prepared the TPA standard solution (5g/L) for further use. Then we use PCR to amplify the <B><I>TPA-sensing leader sequence</I></B>,<B><I> PGK1 promoter</I></B>, <B><I>CYC1 terminator</I></B>, <B><I>RFP gene</I></B>, TPA regulation protein gene (<B><I>tpaR</I></B>), TPA transporting protein gene (<B><I>tpaK</I></B>). Then we cut the fragments above and plasmid <B><I>pRS413</I></B>, <B><I>pRS415</I></B>, and <B><I>pYES2</I></B> with corresponding enzymes and recycled the fragments from agarose gel.</LI><LI>We linked the fragments together by this way:<B><I>1. pYES2-leader-PGK1-RFP.
2. pRS413-PGK1-tpaK-CYC1.
3. pRS415-PGK1-tpaR-CYC1</I></B></LI><LI>Then we used PCR to verify the success and all of the plasmids were correctly constructed. Then we transformed the there plasmids into <B><I>Saccharomyces cerevisiae</I></B>.
</LI><LI>The key enzyme <B><I>Sal1</I></B> arrived and we isolate the plasmid <B><I>pET21a</I></B>. Then we use <B><I>BamH1</I></B> and <B><I>Sal1</I></B> to cut both plasmid and <B><I>PETase</I></B> gene, then linked them together and transformed the recombinant plasmid into <B><I>E.coli</I></B>.</LI></DIV><A class="expand-btn2">Show More</A></DIV><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><H1 class="entry-title">Week3(9/7/2016-9/13/2016)</H1><DIV class="entry-content"><DIV class="note-content3"><LI>We cultured the transformed<B><I> E.coli</I></B> and isolated the plasmid. Then we use PCR to amplify the whole fragment in <B><I>pET21a</I></B> from <B><I>T7 promoter</I></B> to <B><I>T7 terminator</I></B>. Then we recycled this fragment from agarose gel.</LI><LI>The transformed <B><I>Saccharomyces cerevisiae</I></B> had grown to visible colony in Sc-Ura-Leu-His plate. Then we use colony PCR to verify the plasmids had been transformed into the cells. The result is successful so that we streaked more plates.</LI><LI>We cut the <B><I>T7 promoter-PETase gene-T7 terminator</I></B> fragment with enzymes <B><I>EcoR1</I></B> and <B><I>Sac1</I></B>. Then we linked it to the already cut plasmid <B><I>pUC19</I></B> (cut in August 28th). Then we transformed the recombinant plasmid into<B><I> E.coli</I></B>.</LI><LI>We cultured the transformed <B><I>Saccharomyces cerevisiae</I></B> into Sc-Ura-Leu-His culture medium in 30℃. We added TPA standard solution in this way:
1. Group 1: did not add TPA.
2. Group 2: added 1000μL TPA standard solution.
3. Group 3: added 100μL TPA standard solution.
4. Group 4: added 10μL TPA standard solution.
5.Group 5: added 1μL TPA standard solution.
</LI><LI>We cultured the transformed <B><I>E.coli</I></B> into LB+Amp culture medium. Then add 1.5μL IPTG to induce the expression of <B><I>PETase</I></B> gene.</LI><LI>We first detected the red fluorescence of <B><I>E.coli</I></B>, however, the experiment group had almost no increase of red fluorescence relative to control group. We changed the induction wavelength and scan the whole wavelength of emission, but we did not receive any result we expected.</LI><LI>The TPA positive feedback system seemed to have minor effection for there were a little increment of red fluorescence of the 5th group relative to the 1st one.</LI><LI>We doubted that it might be the<B><I> RFP</I></B> in the kit was useless. We isolated the <B><I>pET21a</I></B> and used PCR to amplify the <B><I>RFP </I></B>gene.</LI><LI>We cut the <B><I>RFP</I></B> gene and <B><I>pET21a</I></B> with enzymes <B><I>Xba1</I></B> and <B><I>Sac1</I></B>, then we linked them and transformed it into <B><I>E.coli</I></B>.</LI><LI>We cultured the transformed <B><I>E.coli</I></B> and added IPTG to induce the expression of<B><I> RFP</I></B>, and this time the red fluorescence was clear enough that could be seen directly.</LI></DIV><A class="expand-btn3">Show More</A></DIV><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><H1 class="entry-title">Week4(9/14/2016-9/20/2016)</H1><DIV class="entry-content"><DIV class="note-content4"><LI>We started to construct another regulation way, the <B><I>E.coli</I></B> lysis regulation pathway. We first used colony PCR to obtain the <B><I>ddpX</I></B> gene from the <B><I>E.coli</I></B> genome and recycled the <B><I>ddpX</I></B> from the agarose gel.</LI><LI>We found that there was no enzyme cleavage site between the<B><I> CpxR</I></B> promoter and <B><I>RFP</I></B> gene in the part we use. We had to design the primers and amplified the <B><I>CpxR</I></B> promoter by PCR.</LI><LI>We used PCR to amplify the<B><I> CpxR</I></B> promoter. Then we recycled it from agarose gel.</LI><LI>We cut the<B><I> CpxR</I></B> promoter with enzymes <B><I>Xba1</I></B> and<B><I> BamH1</I></B>, <B><I>ddpX</I></B> gene with enzymes <B><I>BamH1</I></B> and<B><I> EcoR1</I></B>, first batch of <B><I>pET21a</I></B> with<B><I> Xba1</I></B> and<B><I> EcoR1</I></B>, second batch of <B><I>pET21a</I></B> with<B><I> BamH1</I></B> and <B><I>EcoR1</I></B>.</LI><LI>Then we linked these fragment in the following two ways:<B><I>1. pET21a-CpxR-ddpX.
2. pET21a-ddpX.</I></B></LI></DIV><A class="expand-btn4">Show More</A></DIV><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><H1 class="entry-title">Week5(9/21/2016-9/27/2016)</H1><DIV class="entry-content"><DIV class="note-content5"><LI>We used PCR to amplify the whole fragments in<B><I> pET21a</I></B> (from <B><I>CpxR </I></B>to <B><I>T7 terminator</I></B>). However, the band in the agarose gel was disperse so that we were unable to recycle it.</LI><LI>We used colony PCR to verify if the <B><I>pET21a</I></B> had been correctly constructed, the result is yes.</LI><LI>We changed the DNA polymerase and annealing temperature several times and redid the PCR, however, the disperse band were always existed.</LI><LI>We cultured the <B><I>E.coli</I></B> transformed into the <B><I>pET21a-ddpX</I></B> fragment and detect the OD<SUB>600</SUB> in order to verify the lysis effection of <B><I>ddpX</I></B>. </LI><LI>Considering the <B><I>pYES2</I></B> is multicopy plasmid so that the copy number would affect the <B><I>RFP</I></B> expression level, we decided to change the <B><I>pYES2</I></B> to single-copy plasmid <B><I>pRS416</I></B>. Since the <B><I>pRS416</I></B> does not have terminator in its backbone, we used PCR to amplify the <B><I>CYC1</I></B> terminator from plasmid <B><I>pYES2</I></B>.</LI><LI>We cut the <B><I>pYES2</I></B> with enzyme <B><I>Hind3</I></B> and <B><I>EcoR1</I></B>, <B><I>CYC1</I></B> with<B><I> EcoR1</I></B> and <B><I>Sal1</I></B>,<B><I> pRS416</I></B> with <B><I>Hind3</I></B> and<B><I> Sal1</I></B>. Then we linked the three part together.</LI><LI>We transformed the three plasmids into <B><I>Saccharomyces cerevisiae</I></B> together. </LI><LI>The new primers using to amplify the<B><I> CpxR-ddpX-T7</I></B> terminator fragment arrived and we redid the PCR. However, the disperse band was still existed. </LI><LI>The transformation of Saccharomyces cerevisiae turned out to be a failure because no colony was found on the Sc-Ura-Leu-His plate. </LI></DIV><A class="expand-btn5">Show More</A></DIV><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><HEADER class="entry-header"><H1 class="entry-title">Week6(9/28/2016-10/2/2016)</H1></HEADER><DIV class="entry-content"><DIV class="note-content6"><LI>We redid the inclusion body reporting experiment, and this time we directly observed the color of bacterial after centrifugation (12000rpm, 1min). The group with <B><I>PETase</I></B> gene and <B><I>CpxR-RFP</I></B> fragment showed the deepest red.</LI><LI>We redid all the constructing process of ddpX induced cell lysis system and finally we obtained the positive result.</LI><LI>We test the <B>ddpX</B> induced cell lysis system by measuring the OD<SUB>600</SUB> of culture medium.</LI></DIV><A class="expand-btn6">Show More</A></DIV></ARTICLE></ARTICLE></ARTICLE></ARTICLE></ARTICLE></ARTICLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>