<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tianjin_Protocol skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tianjin/Protocol</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>TEAM TIANJIN</TITLE></P><DIV class="container-fluid"><DIV class="navbar-header"><DIV class="nameTeamTianjin">PLASTERMINATOR</DIV><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:Tianjin" id="Overview">OVERVIEW</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Projects">PROJECTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Description" id="Outline">OUTLINE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Experiment/Consortium" id="Consortium">CONSORTIUM</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Experiment/Protein_Engineering" id="ProteinEngineering">PROTEIN ENGINEERING</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Experiment/R-R" id="RRSystem">R-R SYSTEM</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Proof" id="Proof">PROOF</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Model" id="Model">MODEL</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Results">RESULTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Achievements" id="Achievements">ACHIEVEMENTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Demonstrate" id="Demonstrate">DEMONSTRATE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Parts" id="Parts">PARTS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Future" id="FutureWork">FUTURE WORK</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="People">PEOPLE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Team" id="Team">TEAM</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Attributions" id="Attribution">ATTRIBUTION</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Outreach">OUTREACH</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Community/Overview" id="HumanPractice">HUMAN PRACTICE</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Collaborations" id="Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Safety" id="Safety">SAFETY</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" id="Notes">NOTES</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Notes" id="Notebook">NOTEBOOK</A></LI><LI><A href="https://2016.igem.org/Team:Tianjin/Protocol" id="Protocol">PROTOCOL</A></LI></DIV></DIV><P><TITLE>Team Tianjin-Attribution</TITLE></P><DIV class="wrapper-white"><DIV id="page" class="hfeed"><DIV id="main"><DIV id="content" class="home blog single-author one-column content" role="main"><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><HEADER class="entry-header"><DIV class="entry-title" align="center" style="font-size:42px;margin-top:30px">General Protocols</DIV></HEADER><H2 class="entry-title">1.Plasmid Extraction</H2><DIV class="entry-content"><DIV class="note-content">	
	
We use the<B> TIANprep Mini Plasmid Kit </B>made by <B>TIANGEN Biotech Co.,Ltd.</B> to extract plasmid. Here is the protocol.
Add <B>ethanol (96-100%)</B> to Buffer <B>PW </B>before use, check bottle tag for the adding volume.
1. Column equilibration: Place a Spin Column<B> CP3 </B>in a clean collection tube, and add <B>500 μl</B> Buffer <B>BL</B> to <B>CP3</B>. Centrifuge for <B>1 min</B> at <B>12,000 rpm (~13,400 × g)</B> in a table-top microcentrifuge. Discard the flow-through, and put the Spin Column <B>CP3</B> back into the collection tube. (Please use freshly treated spin column).
2. Harvest <B>1-5 ml</B> bacterial cells in a microcentrifuge tube by centrifugation at <B>12,000 rpm (~13,400 × g)</B> in a conventional, table-top microcentrifuge for <B>1 min</B> at room temperature (<B>15-25°C</B>), then remove all traces of supernatant by inverting the open centrifuge tube until all medium has been drained (For large volume of bacterial cells, please harvest to one tube by several centrifugation step.)
3. Re-suspend the bacterial pellet in <B>250 μl</B> Buffer<B> P1 </B>(Ensure that <B>RNase A </B>has been added). The bacteria should be resuspended completely by vortex or pipetting up and down until no cell clumps remain.
Note: No cell clumps should be visible after resuspension ofthe pellet, otherwise incomplete lysis will lower yield and purity.
4. Add <B>250 μl</B> Buffer<B> P2 </B>and mix gently and thoroughly by inverting the tube 6-8 times.
Note: Mix gently by inverting the tube. Do not vortex, as this will result in shearing of genomic DNA. If necessary, continue inverting the tube until the solution becomes viscous and slightly clear. Do not allow the lysis reaction to proceed for more than 5 min. If the lysate is still not clear, please reduce bacterial pellet.
5. Add <B>350 μl</B> Buffer <B>P3</B> and mix immediately and gently by inverting the tube 6-8 times. The solution should become cloudy. Centrifuge for <B>10 min</B> at<B> 12,000 rpm (~13,400 × g)</B> in a table-top microcentrifuge.
Note: To avoid localized precipitation, mix the solution thoroughly, immediately after addition of Buffer <B>P3</B>. If there is still white precipitation in the supernatant, please centrifuge again.
6. Transfer the supernatant from step 5 to the Spin Column <B>CP3</B> (place <B>CP3 </B>in a collection tube) by decanting or pipetting. Centrifuge for <B>30-60 s </B>at <B>12,000 rpm (~13,400 × g)</B>. Discard the flow-through and set the Spin Column <B>CP3 </B>back into the Collection Tube.
7. (Optional, actually we hardly ever use) Wash the Spin Column <B>CP3 </B>by adding <B>500 μl</B> Buffer <B>PD </B>and centrifuge for <B>30-60 s</B> at <B>12,000 rpm (~13,400 × g)</B>. Discard the flow-through and put Spin Column <B>CP3 </B>back to the collection tube.
This step is recommended to remove trace nuclease activity when using <B><I>endA+</I></B> strains such as the<B><I> JM series</I></B>, <B><I>HB101</I></B> and its derivatives, or any wild-type strain, which have high levels of nuclease activity or high carbohydrate content.
8. Wash the Spin Column <B>CP3 </B>by adding <B>600 μl</B> Buffer <B>PW</B> (ensure that <B>ethanol (96%-100%)</B> has been added) and centrifuge for <B>30-60 s </B>at <B>12,000 rpm (~13,400 × g)</B>. Discard the flow-through, and put the Spin Colum<B> CP3</B> back into the Collection Tube.
9. Repeat Step 8.
10. Centrifuge for an additional <B>2 min</B> at <B>12,000 rpm (~13,400 × g) </B>to remove residual wash Buffer <B>PW</B>.
Note: Residual ethanol from Buffer <B>PW </B>may inhibit subsequent enzymatic reactions. We suggest open <B>CP3 </B>lid and stay at room temperature for a while to get rid of residual ethanol.
11. Place the Spin Column <B>CP3 </B>in a clean <B>1.5 ml </B>microcentrifuge tube. To elute DNA, add <B>50-100 μl</B> Buffer <B>EB </B>to the center of the Spin Column <B>CP3</B>, incubate for <B>2 min</B>, and centrifuge for <B>2 min</B> at <B>12,000 rpm (~13,400 × g)</B>.
Note: If the volume of eluted buffer is less than <B>50 μl</B>, it may affect recovery efficiency. The pH value of eluted buffer will have some influence in eluting; Buffer <B>EB</B> or distilled water (<B>pH 7.0-8.5</B>) is suggested to elute plasmid DNA. For long-term storage of DNA, eluting in Buffer<B> EB</B> and storing at<B> -20°C</B> is recommended, since DNA stored in water is subject to acid hydrolysis. Repeat step 11 to increase plasmid recovery efficiency.

</DIV><A class="expand-btn">Show More</A></DIV><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><HEADER class="entry-header"><H2 class="entry-title">2.DNA Purification</H2></HEADER><DIV class="entry-content"><DIV class="note-content2">
We use the <B>TIANquick Midi Purification Kit</B> made by <B>TIANGEN Biotech Co.,Ltd.</B> to purify the DNA products of PCR and restriction endonuclease cutting. Here is the protocol.
Add <B>ethanol (96-100%)</B> to Buffer <B>PW </B>before use (see bottle label for volume).
1. Column equilibration: add <B>500 μl</B> Buffer <B>BL</B> to the Spin Column <B>CB2</B> (put Spin Column <B>CB2 </B>into a collection tube). Centrifuge for <B>1 min</B> at <B>12,000 rpm (~13,400 × g)</B>. Discard the flow-through, and then place Spin Column <B>CB2</B> back into the collection tube (please use freshly treated spin column).
2. Add <B>5 volumes</B> of Buffer <B>PB</B> to <B>1 volume</B> of the PCR reaction or enzymatic reaction and mix. It is not necessary to remove mineral oil or kerosene.
Note: For example, add <B>250 μl </B>Buffer <B>PB</B> to <B>50 μl</B> PCR reaction (not including oil).
3. Transfer the mixture to the Spin Column<B> CB2</B>, incubate at room temperature (<B>15-25°C</B>) for <B>2 min</B>. Centrifuge for <B>30-60 s</B> at <B>12,000 rpm (~13,400 × g)</B> in a table-top microcentrifuge. Discard the flow-through, and then place Spin Column <B>CB2</B> back into the same collection tube.
Note: The maximum loading volume of the column is <B>800 μl</B>. For sample volumes greater than <B>800 μl</B> simply load again.
4. Add <B>600 μl</B> Buffer<B> PW</B> (ensure that <B>ethanol (96-100%)</B> has been added) to the Spin Column<B> CB2</B> and centrifuge for<B> 30-60 s</B> at <B>12,000 rpm (~13,400 × g)</B>. Discard the flow-through, and place Spin Column <B>CB2</B> back in the same collection tube.
Note: If the purified DNA is used for the subsequent salt sensitive experiments, such as ligation or sequencing experiment, it is suggested to stand for<B> 2-5 min</B> after adding Buffer<B> PW</B>, and then centrifuge.
5. Repeat step 4.
6. Centrifuge at <B>12,000 rpm (~13,400 × g)</B> for <B>2 min </B>to remove residual Buffer<B> PW</B>. Discard the flow-through, and allow the column to air dry with the cap open for several minutes to dry the membrane.
Note: Residual ethanol from Buffer <B>PW</B> may inhibit subsequent experiment (enzymatic or PCR reactions).
7. Place the Spin Column <B>CB2</B> in a clean <B>1.5 ml</B> microcentrifuge tube. Add<B> 30-50 μl </B>Buffer <B>EB</B> to the center of membrane, incubate for <B>2 min</B>, and centrifuge for <B>2 min</B> at <B>12,000 rpm (~13,400 × g)</B>.
Note: If the volume of eluted buffer is less than <B>30 μl</B>, it may affect recovery efficiency. The pH value of eluted buffer will have big influence in eluting; distilled water (<B>pH 7.0-8.5</B>, adjusted with<B> NaOH</B>) is suggested to elute plasmid DNA, <B>pH&lt;7.0</B> will decrease elution efficiency. For long-term storage of DNA, eluting in Buffer <B>EB</B> and storing at <B>-20°C </B>is recommended, since DNA stored in water is subject to acid hydrolysis. Repeat step 7 to increase plasmid recovery efficiency.

</DIV><A class="expand-btn2">Show More</A></DIV><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><H2 class="entry-title">3.Agarose Gel Electrophoresis Products Recycling</H2><DIV class="entry-content"><DIV class="note-content3">


We use the <B>TIANgel Midi DNA Purification Kit</B> made by <B>TIANGEN Biotech Co.,Ltd.</B> to recycle the DNA products from the agarose gel. Here is the protocol.
Add <B>ethanol (96-100%)</B> to Buffer <B>PW</B> before use (see bottle label for volume).
1. Column equilibration: add <B>500 μl</B> Buffer <B>BL</B> to the Spin Column <B>CA2 </B>(put Spin Column <B>CA2 </B>into a collection tube). Centrifuge for<B> 1 min</B> at <B>12,000 rpm (~13,400 × g)</B> in a table-top microcentrifuge. Discard the flow-through, and put Spin Column<B> CA2</B> back into the collection tube (please use freshly treated spin column).
2. Cut the DNA fragment from agarose gel with a clean, sharp scalpel. Weigh the gel slice in a clean tube. 
3. Add equivalent volume of Buffer <B>PN </B>to the gel (If the gel is <B>0.1 g</B>, it is defaulted to be <B>100 μl</B>, then add <B>100 μl </B>Buffer <B>PN</B>). Incubate at<B> 50°C</B> by inverting up and down the tube until the agarose gel dissolves completely. If the agarose gel does not dissolve completely, incubate for longer period or add additional Buffer <B>PN </B>until all the agarose gel dissolved completely (If the agarose gel is too large, please cut the agarose gel into several pieces in advance). 
Note: If DNA fragment is <B>&lt;300 bp</B>, it is recommended to add isopropanol which is <B>1/2 volume </B>of Buffer <B>PN</B> to the agarose gel sample after the gel completely dissolved. Cooling the solution at room temperature (<B>15-25°C</B>) and then add the solution to Spin Column <B>CA2 </B>since silica membrane of the column adsorbs DNA best at room temperature. 
4. When the gel dissolved completely and the solution temperature turns to room temperature (<B>15-25°C</B>), transfer the mixture to the Spin Column<B> CA2</B> (put Spin Column <B>CA2</B> into a collection tube). Let the column stand for <B>2 min</B> at room temperature (<B>15-25°C</B>), then centrifuge for <B>30-60 s</B> at <B>12,000 rpm (~13,400 × g)</B> in a table-top microcentrifuge. Discard the flow-through; place the Spin Column <B>CA2 </B>back into the collection tube again. 
Note: The maximum loading volume of the column is<B> 800 μl</B>. For sample volumes greater than <B>800 μl</B> simply load again.
5. Wash the Spin Column <B>CA2</B> with <B>600 μl</B> Buffer <B>PW </B>(ensure that <B>ethanol (96-100%)</B> has been added) and centrifuge for <B>30-60 s</B> at <B>12,000 rpm (~13,400 × g)</B>. Discard the flow-through and place the Spin Column <B>CA2</B> back into the collection tube. 
Note: If the purified DNA is used for the salt sensitive experiments, such as direct sequencing and blunt-ended ligation, let the column stand for <B>2-5 min</B> after adding Buffer<B> PW</B>, and then centrifuge. 
6. Repeat Step 5. 
7. Place the Spin Column<B> CA2</B> back to the collection tube and centrifuge at <B>12,000 rpm (~13,400 × g)</B> for <B>2 min </B>to remove residual wash buffer. Discard the flow-through, and place column with the cap open for several minutes to air dry the membrane. 
Note: Residual ethanol from Buffer <B>PW</B> will influence the subsequent enzymatic reaction (enzyme digestion, PCR etc). 
8. Transfer the Spin Column<B> CA2</B> to a clean <B>1.5 ml</B> microcentrifuge tube. Add appropriate volume of Buffer<B> EB </B>to the center of the membrane, incubate at room temperature (<B>15-25°C</B>) for <B>2 min</B>, then centrifuge at <B>12,000 rpm (~13,400 × g)</B> for <B>2 min</B>. 
Note: The elution volume should not be less than <B>30 μl</B> since smaller volume will affect recovery efficiency. The pH value of eluted buffer will affect eluting. If purified DNA is used for sequencing, it is recommended to choose ddH<SUB>2</SUB>O (<B>pH 7.0-8.5</B>) to elute DNA,    <B>pH&lt;7.0</B> will decrease the elution efficiency. Obtained DNA should be stored at<B> -20°C</B> to prevent degradation. Buffer (<B>10 mM Tris-Cl, pH 8.0</B>) could also be used for DNA elution. For higher yield, pipette the eluate to the center of the membrane again, incubate <B>2 min</B> and centrifuge at <B>12,000 rpm (~13,400 × g)</B> for <B>2 min</B>.
 
</DIV><A class="expand-btn3">Show More</A></DIV><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><H2 class="entry-title">4.Agarose Gel Electrophoresis</H2><DIV class="entry-content"><DIV class="note-content4">

1. Preparation of <B>TAE buffer</B>: Add <B>242g Tris, 37.2g Na2EDTA·2H2O, 800mL ddH<SUB>2</SUB>O</B>, after all the solute dissolve, add <B>57.1mL acetic acid </B>and add <B>ddH<SUB>2</SUB>O</B> to make the total volume <B>1L</B>.
2. Preparation of sample: Add <B>DNA loading buffer</B> to the DNA solution according to the dilution ratio of particular buffer.
3. Preparation of agarose gel: Add <B>agarose powder 10g/L, 1×TAE buffer 100mL </B>(variable, according to need), heat the mixture by microwave oven until the agarose was dissolved. After the solution cool down to touchable temperature, add <B>50-100μL/L Goldenview Nucleic acid dye</B> to the solution. Then pour the solution to the gel mould with gel comb inserted and wait for its concretion.
4. Add samples to the gel pore: After the formation of gel, pull out the gel comb and take the gel out of the mould. Immerge the gel with <B>1×TAE buffer</B> in the electrophoresis chamber. Using pipette to add marker and samples to different pore. (The content of pore depends on the gel comb, there are 3 kinds of volume, <B>10μL, 20μL, and 50μL</B>) Do not stick the bottom and side of gel pore to prevent the leakage. 
5. Turn on the electrical source to start the electrophoresis, the voltage is set at <B>150-160V</B> and the electrophoresis time is set at <B>8-12min</B>.
6. After the electrophoresis process end, the gel is observed under <B>blue light or ultraviolet</B>.

        

 </DIV><A class="expand-btn4">Show More</A></DIV><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><H2 class="entry-title">5.Restriction Endonuclease Digestion</H2><DIV class="entry-content"><DIV class="note-content5">

Prepare the system:
<LI>Total volume: <B>50μL</B></LI><LI>Restriction endonuclease:<B> 2μL</B> respectively</LI><LI>10×Cut Smart Buffer:<B> 5μL</B></LI><LI>DNA to be cut: <B>30μL</B></LI><LI>ddH<SUB>2</SUB>O:<B> 13μL</B></LI><LI>Reaction time: <B>2h</B></LI><LI>Reaction temperature: <B>37℃</B></LI></DIV><A class="expand-btn5">Show More</A></DIV><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><HEADER class="entry-header"><H2 class="entry-title">6.DNA Fragments Ligation</H2></HEADER><DIV class="entry-content"><DIV class="note-content6">

Prepare the system:
<LI>Total volume: <B>20μL</B></LI><LI>T4 DNA ligase: <B>1μL</B></LI><LI>5×Ligase Buffer:<B> 4μL</B></LI><LI>DNA to be linked: <B>(c1V1/L1): (c2V2/L2)=5:1</B>. (c1: Concentration of cut DNA fragments; c2: Concentration of cut plasmid; V1: Volume of cut DNA fragments; V2: Volume of cut plasmid; L1: Length of cut DNA fragments; L2: Length of cut plasmid)</LI><LI>ddH<SUB>2</SUB>O: add to make the total volume <B>20μL</B></LI><LI>Reaction time:<B> 2h</B></LI><LI>Reaction temperature: <B>22℃</B></LI></DIV><A class="expand-btn6">Show More</A></DIV><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><HEADER class="entry-header"><H2 class="entry-title">7.PCR</H2></HEADER><DIV class="entry-content"><DIV class="note-content7">

Prepare the system:
<LI>Total volume: <B>50μL</B></LI><LI>Q5 DNA Polymerase: <B>0.5μL</B></LI><LI>5×Q5 DNA Polymerase Buffer: <B>10μL</B></LI><LI>Template: <B>1μL</B></LI><LI>dNTP:<B> 1μL</B></LI><LI>Primers: Sense Primer and Anti-sense Primer, respectively <B>2.5μL</B></LI><LI>ddH<SUB>2</SUB>O: <B>32.5μL</B></LI><LI>Cycles: <B>25-35</B></LI><LI>Pre-denaturation:<B> 98℃,30s</B></LI><LI>Denaturation: <B>98℃,5-10s</B></LI><LI>Annealing: Depend on the primers, generally <B>45-65℃,10-30s</B></LI><LI>Extension: <B>72℃,20-30s/kb</B></LI><LI>Fully extension: <B>72℃,2min</B></LI><LI>Product Storage: <B>4℃</B></LI><LI>Note: Different DNA polymerase has different protocol, this is only the case of Q5 DNA polymerase.</LI></DIV><A class="expand-btn7">Show More</A></DIV><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><HEADER class="entry-header"><H2 class="entry-title">8.Colony PCR</H2></HEADER><DIV class="entry-content"><DIV class="note-content8">

Prepare the system:
<LI>Total volume: <B>20μL</B></LI><LI>Colony PCR System: 2×Taq Master Mix (Dye added) <B>10μL</B></LI><LI>Primers: Sense Primer and Anti-sense Primer, respectively <B>1μL</B></LI><LI>ddH<SUB>2</SUB>O: <B>8μL</B></LI><LI>select single colony with stick and immerse the stick into the system for <B>1 min</B>. Preparing the bacterial liquid and then add 1μL of it is also feasible.
</LI><LI>Cycles:<B> 25-35</B></LI><LI>Pre-denaturation: <B>94℃,5min</B></LI><LI>Denaturation:<B> 94℃,30s</B></LI><LI>Annealing: Depend on the primers, generally <B>45-65℃,20-60s</B></LI><LI>Extension:<B> 72℃</B>, depend on the length of amplified fragment, generally <B>20-60s</B></LI><LI>Fully extension: <B>72℃,7min</B></LI><LI>Product Storage: <B>4℃</B></LI>
Note: Preparation of bacteria liquid
Prepare the system:
<LI>Total volume: <B>50μL</B></LI><LI>ddH<SUB>2</SUB>O: <B>50μL</B></LI><LI>select single colony with stick and immerse the stick into the system for <B>1 min</B></LI><LI>Cycles:<B> 3-4</B></LI><LI>Pre-denaturation: <B>99℃,4min</B></LI><LI>Renaturation: <B>4℃,4min</B></LI><LI>Product Storage: <B>4℃</B></LI></DIV><A class="expand-btn8">Show More</A></DIV><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><HEADER class="entry-header"><H2 class="entry-title">9.TransT1 Transformation</H2></HEADER><DIV class="entry-content"><DIV class="note-content9">

1. Mix <B>50μL</B> competent <I>E.coli</I> cell with <B>10μL</B> plasmids, and place them in ice bath for <B>30 min</B>.
2. Heat shock at <B>42℃</B> for <B>30 s</B>.
3. Put the system back on ice for <B>2 min</B>.
4. Add <B>500 μl</B> of LB without antibiotics and incubate at <B>37 ℃</B> for <B>30-60 min</B> with the shaking speed of <B>200rpm</B>.
5. Centrifuge the culture medium at <B>4000 rpm</B> for <B>3min</B>.
6. Discard the supernatant and resuspend the sediment, use this resuspending liquid to spread the plate with antibiotics.



 </DIV><A class="expand-btn9">Show More</A></DIV><DIV class="entry-title" align="center" style="font-size:42px">Protocols for Microbial Consortia</DIV><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><HEADER class="entry-header"><H2 class="entry-title">1.16s-DNA </H2></HEADER><DIV class="entry-content"><DIV class="note-content11">

Prepare the system:
<LI>Total volume: <B>20μL</B></LI><LI>Colony PCR System: 2×Taq Master Mix (Dye added) <B>10μL</B></LI><LI>ddH<SUB>2</SUB>O: <B>8.2μL</B></LI><LI>primer: <B>0.4μL </B> each</LI><LI>bacteria liquid:<B>1μL</B></LI><LI>The primer could copy the special 16s-DNA from this kind of bacteria.</LI><LI>The bacteria liquid is pretreatment. </LI><LI>Cycles:<B> 25-35</B></LI><LI>Pre-denaturation: <B>94℃,2min</B></LI><LI>Denaturation: <B>94℃,30s</B></LI><LI>Annealing: Depend on the primers, generally <B>45-65℃,20-60s</B></LI><LI>Extension: <B>72℃</B>, depend on the length of amplified fragment, generally <B>20-60s</B></LI><LI>Fully extension: <B>72℃,4min</B></LI><LI>Product Storage:<B> 4℃</B></LI></DIV><A class="expand-btn11">Show More</A></DIV><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><HEADER class="entry-header"><H2 class="entry-title">2. Preparation of W medium </H2></HEADER><DIV class="entry-content"><DIV class="note-content15">

Initial medium 
<LI><B>KH<SUB>2</SUB>PO<SUB>4</SUB>:	1.7 g/L</B></LI><LI>Na<SUB>2</SUB>HPO<SUB>4</SUB>:	9.8 g/L</LI><LI>(NH<SUB>4</SUB>)<SUB>2</SUB>SO<SUB>4</SUB>:	1.0 g/L</LI><LI>MgSO<SUB>4</SUB>·7H<SUB>2</SUB>O:	0.1 g/L</LI><LI>FeSO<SUB>4</SUB>·7H<SUB>2</SUB>O:	0.95 mg/L</LI><LI>MgO:	10.75 mg/L</LI><LI>CaCO<SUB>3</SUB>:	2.0 mg/L</LI><LI>ZnSO<SUB>4</SUB>·7H<SUB>2</SUB>O:	1.44 mg/L</LI><LI>CuSO<SUB>4</SUB>·5H<SUB>2</SUB>O:	0.25 mg/L</LI><LI>CoSO<SUB>4</SUB>·7H<SUB>2</SUB>O:	0.25 mg/L</LI><LI>H<SUB>3</SUB>BO<SUB>3</SUB>:	0.06 mg/L</LI><LI>HCl:	51.3ml/L</LI>

We modify initial formula for convenience, and modified formula is shown as following table:

<LI><B>KH<SUB>2</SUB>PO<SUB>4</SUB>:	1.7 g/L</B></LI><LI>Na<SUB>2</SUB>HPO<SUB>4</SUB>:	9.8 g/L</LI><LI>(NH<SUB>4</SUB>)<SUB>2</SUB>SO4:	1.0 g/L</LI><LI>NH<SUB>4</SUB>Cl:	0.865 g/L</LI><LI>MgSO<SUB>4</SUB>·7H<SUB>2</SUB>O:	0.1 g/L</LI><LI>MgCl<SUB>2</SUB>:	0.025 g/L</LI><LI>Mother solution:	1 ml/L</LI>
Formula of mother solution
<LI><B>FeSO<SUB>4</SUB>·7H<SUB>2</SUB>O:	0.95 g/L</B></LI><LI>CoCl<SUB>2</SUB>·6H<SUB>2</SUB>O:	0.236 g/L</LI><LI>CaCl<SUB>2</SUB>:	2.22 g/L</LI><LI>ZnSO<SUB>4</SUB>·7H<SUB>2</SUB>O:	1.44 g/L</LI><LI>CuSO<SUB>4</SUB>:	0.16 g/L</LI><LI>H<SUB>3</SUB>BO<SUB>3</SUB>:	0.06 g/L</LI></DIV><A class="expand-btn15">Show More</A></DIV><DIV class="entry-title" align="center" style="font-size:42px">Protocols for Protein Engineering</DIV><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><HEADER class="entry-header"><H2 class="entry-title">1.Preparation of the optimized yeast culture medium </H2></HEADER><DIV class="entry-content"><DIV class="note-content23"><LI><B>Glucose:22g/L</B></LI><LI>Yeast Nitrogen Base:6.7g/L</LI><LI>Dispense mixture:1.224g/L</LI><LI>Agar:15g/L</LI><LI>His/Trp/Leu mixed solution:10ml/L</LI></DIV><A class="expand-btn23">Show More</A></DIV><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><HEADER class="entry-header"><H2 class="entry-title">2.pNPA assay</H2></HEADER><DIV class="entry-content"><DIV class="note-content24">
A Buffer  
<LI><B>Glycerol:400μl</B></LI><LI> Gal:1000μl</LI><LI>SC-Ura:3600μl</LI>
B Buffer:
<LI><B>10mM pNPA Buffer:50μl</B></LI><LI> PBS(pH=7.4):98μl</LI></DIV><A class="expand-btn24">Show More</A></DIV><DIV class="entry-title" align="center" style="font-size:42px">Protocols for Cell-Free Protein Synthesis</DIV><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><HEADER class="entry-header"><H2 class="entry-title">1.Cell-free protein synthesis system(50 µL) </H2></HEADER><DIV class="entry-content"><DIV class="note-content12"><LI>ddH<SUB>2</SUB>O:<B>7.9µL</B></LI><LI>Feeding buffer :   <B>25µL</B></LI><LI>Mg<SUP>2+</SUP> solution  :  <B>1.1µL</B></LI><LI>Gene( plasmid as template) :<B> 1µL</B></LI><LI>Lysate   :  <B>15µL</B></LI></DIV><A class="expand-btn12">Show More</A></DIV><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><HEADER class="entry-header"><H2 class="entry-title">2.Preparation of pNPA degradation reaction system </H2></HEADER><DIV class="entry-content"><DIV class="note-content13">

Substrate: <B>0.2mM pNPA </B>water solution (<B>0.0362g pNPA </B>dissolving in ddH<SUB>2</SUB>O)
Reaction system: <B>10 times</B> diluted Enzyme solution(unpurified) mixed with the equal volume substrate. (<B>pH=7</B>)
After static reaction at <B>39℃ </B>for <B>8 hours</B>, we detected the characteristic adsorption peak of the product ,pNP, which has no other characteristic adsorption peak except in <B>400nm</B>.


 </DIV><A class="expand-btn13">Show More</A></DIV><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><HEADER class="entry-header"><H2 class="entry-title">3.Preparation of PET degradation reaction system </H2></HEADER><DIV class="entry-content"><DIV class="note-content14">

Reaction system: PET was put into <B>20 times </B>diluted Enzyme solution(unpurified).(<B>pH=7</B>)
After static reaction at <B>39℃</B> for <B>2 days</B>, we detected the characteristic adsorption peak of the product ,MHET, which has no other characteristic adsorption peak except in <B>260nm</B>. And detect it every other day in the future 3-4 days.


 </DIV><A class="expand-btn14">Show More</A></DIV><DIV class="entry-title" align="center" style="font-size:42px">Protocols for Cyanobacteria</DIV><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><HEADER class="entry-header"><H2 class="entry-title">1.Restriction Endonuclease Cutting and Dephosphorylation </H2></HEADER><DIV class="entry-content"><DIV class="note-content16">

Prepare the system:
<LI>Total volume: <B>100μL</B></LI><LI>Restriction endonuclease :<B> 3μL respectively</B></LI><LI>10×fastdigest Buffer:<B> 10μL</B></LI><LI>Plasmids to be cut: <B>2μg</B></LI><LI>ddH<SUB>2</SUB>O: add to make the total volume <B>100μL</B></LI><LI>fast AP: add when reacting for <B>1.5h</B></LI><LI>Total reaction time: <B>2h</B></LI><LI>Reaction temperature:<B> 37℃</B></LI></DIV><A class="expand-btn16">Show More</A></DIV><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><HEADER class="entry-header"><H2 class="entry-title">2.Phosphorylation </H2></HEADER><DIV class="entry-content"><DIV class="note-content17">

Prepare the system:
<LI>Total volume: <B>20μL</B></LI><LI>ATP:<B>2μL</B></LI><LI>10×Buffer A:<B>2μL</B></LI><LI>PNK:<B>1μL</B></LI><LI>ddH<SUB>2</SUB>O: add to make the total volume<B> 200μL</B></LI><LI>Reaction time:<B>0.5h</B></LI><LI>Reaction temperature:<B> 37℃</B></LI></DIV><A class="expand-btn17">Show More</A></DIV><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><HEADER class="entry-header"><H2 class="entry-title">3.Ligation of Phosphorylated DNA Fragments and Dephosphorylated Plasmids </H2></HEADER><DIV class="entry-content"><DIV class="note-content18">

Prepare the system:
<LI>Total volume: <B>20μL</B></LI><LI>DNA fragment: <B>3μL</B></LI><LI>Plasmids to be ligase : <B>1μL</B></LI><LI>T4 Buffer:<B> 2μL</B></LI><LI>PEG4000:<B>2μL</B></LI><LI>T4 DNA ligase: <B>1μL </B></LI><LI>PCR H<SUB>2</SUB>O:<B>11μL</B></LI><LI>Reaction time:<B> &gt;1h</B></LI><LI>Reaction temperature:<B> 22℃</B></LI></DIV><A class="expand-btn18">Show More</A></DIV><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><HEADER class="entry-header"><H2 class="entry-title">4.A-Tailing Reaction </H2></HEADER><DIV class="entry-content"><DIV class="note-content19">

Prepare the system:
<LI>Total volume:<B> 20μL</B></LI><LI>Talling-A reaction Buffer:<B> 4μL</B></LI><LI>DNA fragment: <B>15μL</B></LI><LI>Taq DNA Polymerase: <B>1μL</B></LI><LI>Reaction time: <B>&gt;0.5h</B></LI><LI>Reaction temperature: <B>72℃</B></LI></DIV><A class="expand-btn19">Show More</A></DIV><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><HEADER class="entry-header"><H2 class="entry-title">5.TA cloning </H2></HEADER><DIV class="entry-content"><DIV class="note-content20">

Prepare the system:
<LI>Total volume:<B> 10μL</B></LI><LI>5×Buffer:<B>2μL</B></LI><LI>Vector:<B>0.5μL</B></LI><LI>DNA fragment:<B> 7μL</B></LI><LI>T4 DNA ligase:<B> 0.5μL</B></LI><LI>Reaction time: <B>0.5-1h</B></LI><LI>Reaction temperature:<B> 22℃</B></LI></DIV><A class="expand-btn20">Show More</A></DIV><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><HEADER class="entry-header"><H2 class="entry-title">6.Electroporation </H2></HEADER><DIV class="entry-content"><DIV class="note-content21">

1.Cultures (<B>OD630=0.6–0.7</B>) which grow approaching the plateau are harvested.
2.According to the quantity of transformation, one plasmid need <B>8-10mL</B> cultures. Cells are harvested by centrifugation at<B>3500rpm</B> for <B>13min</B> at<B> 4°C</B>.  
3.Discard the supernatant, resuspend with <B>20mL</B> sterilized ultrapure water. Then, centrifuge at <B>3500rpm</B> for <B>13min</B> at <B>4°C</B>.
4.Repeat step3.
5.Add <B>200mL </B>sterilized ultrapure water for each plasmid.
6. Plasmid(<B>2μg</B>) is added to <B>200mL</B> condensed cultures in a sterilized EP tube. Then transfer the mixture into a <B>2mm</B>-gap electroporation cuvette.
7.Place them on the ice for<B> 1-3min</B>.
8. Bacteria are pulsed at <B>2.5kV(1.8kV)</B>.
9. <B>200μL </B>BG-11 medium(without antibiotics) is added into the electroporation mixture and incubate for <B>5h</B>.
10. Use the liquid to spread the plate with antibiotics.





 </DIV><A class="expand-btn21">Show More</A></DIV><ARTICLE id="post-4252" class="post-4252 post type-post status-publish format-standard has-post-thumbnail hentry category-150 tag-174 tag-xinjiang tag-173"><HEADER class="entry-header"><H2 class="entry-title">7.Preparation of BG-11 medium </H2></HEADER><DIV class="entry-content"><DIV class="note-content22"><LI><B>NaNO<SUB>3</SUB>: 1.5 g/L</B></LI><LI>K<SUB>2</SUB>HPO<SUB>4</SUB>·3H<SUB>2</SUB>O: 0.04 g/L</LI><LI>MgSO<SUB>4</SUB>·7H<SUB>2</SUB>O: 0.075 g/L</LI><LI>CaCl<SUB>2</SUB>·2H<SUB>2</SUB>O: 0.036 g/L</LI><LI>Citric acid: 0.006 g/L</LI><LI>Ferric ammonium citrate: 0.006 g/L</LI><LI>EDTA(disodium magnesium salt): 0.001 g/L</LI><LI>Na<SUB>2</SUB>CO<SUB>3</SUB>: 0.02 g/L</LI><LI>Trace metal mix A5+Co: 1 mL/L</LI><LI>Deionized water: 1000 mL</LI><LI>pH after autoclaving and cooling: 7.4 </LI></DIV><A class="expand-btn22">Show More</A></DIV></ARTICLE></ARTICLE></ARTICLE></ARTICLE></ARTICLE></ARTICLE></ARTICLE></ARTICLE></ARTICLE></ARTICLE></ARTICLE></ARTICLE></ARTICLE></ARTICLE></ARTICLE></ARTICLE></ARTICLE></ARTICLE></ARTICLE></ARTICLE></ARTICLE></ARTICLE></ARTICLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>