Team:Tianjin/Note/Consortium
TEAM TIANJIN
PLASTERMINATOR
Toggle navigation
OVERVIEW
PROJECTS
OUTLINE
CONSORTIUM
PROTEIN ENGINEERING
R-R SYSTEM
PROOF
MODEL
RESULTS
ACHIEVEMENTS
DEMONSTRATE
PARTS
FUTURE WORK
PEOPLE
TEAM
ATTRIBUTION
OUTREACH
HUMAN PRACTICE
COLLABORATIONS
SAFETY
NOTES
NOTEBOOK
PROTOCOL
Team Tianjin-Attribution
Notes for Bacterial Consortium
Week1(7/24/2016-7/30/2016)
Optimization of Culture Conditions
  Jul.26th
Prepare M9 medium with TPA and culture Pseudomonas putida KT2440 at 30℃ & 200 rpm, checked the bacterial concentration at OD600.
  Jul.27th
1.Extract 10 ml LB medium to 5 test tubes and add 0μl, 50μl,, 100μl,, 250μl,, 500μl EG to them, respectively.
2.Add 5μl bacteria solution of Pseudomonas putida KT2440 to five test tubes above.
3.Culture them at 30℃ & 200 rpm, checked the bacterial concentration at OD600.
  Jul.28th
Add 6g TPA, 2.9gNaOH and 1.2g glucose to M9 medium, and culture them in the improved M9 medium and M9 medium of Jul.26 at 30℃ & 200 rpm, checked the bacterial concentration at OD600 and detected the concentration of TPA by UV at OD242.
  Jul.30th
Cultured different bacteria in M9 medium with sodium terephthalate and different concentration of glucose at 30℃ & 200 rpm, checked the bacterial concentration at OD600 and detected the concentration of TPA by UV at OD242.
Show More
Week2(7/31/2016-8/6/2016)
Optimization of Culture Conditions
  Jul.31th
Cultured Rhodococcus jostii RHA1 in M9 medium with sodium terephthalate and glucose at 30℃ & 200 rpm, checked the bacterial concentration at OD600 and detected the concentration of TPA by UV at OD242.
  Aug.1th
Cultured Rhodococcus jostii RHA1 in W medium with sodium terephthalate and glucose at 30℃ & 200 rpm, checked the bacterial concentration at OD600 and detected the concentration of TPA by UV at OD242.
  Aug.2th
Cultured Pseudomonas putida KT2440 in W medium with sodium terephthalate and glucose at 30℃ & 200 rpm, checked the bacterial concentration at OD600 and detected the concentration of TPA by UV at OD242.
  Aug.3th
Cultured Pseudomonas putida KT2440 , Rhodococcus jostii RHA1 , Pseudomonas putida KT2440 and Rhodococcus jostii RHA1in W medium with sodium terephthalate and glucose at 30℃ & 200 rpm, checked the bacterial concentration at OD600 and detected the concentration of TPA by UV at OD242.
  Aug.4th
        Observed the growth of 1-day pre-cultured bacteria in LB at 30℃:
All above were kept culturing for one more day and were checked on the next day. The next step was to explore the optimal condition for each bacterium and to co-culture each teo of them to see whether the consortium could work well.
The most probable pairs:
1. P.putida KT2440 + R.jostii RHA1;
2. B.subtilis 168 + R.jostii RHA1;
3. B.subtills 168 + P.putida KT2440;
4. R.jostii RHA1 + Y.lipolytia;
5. P.putida KT2440 + Y.lipolytia;
6. E.coli(RFP) + Y.lipolytia.
  Aug.5th
Cultured different bacteria in M9 medium with sodium terephthalate at 30℃ & 200 rpm, checked the bacterial concentration at OD600 and detected the concentration of TPA by UV at OD242: The growth of each bacterium was not as expected, so we considered culturing bacteria in improved W medium.
Co-cultured different pairs in improved W medium in the same condition (using two tubes in each group):
Co-cultured different pairs in LB medium in the same condition(using two tubes in each group):
  Aug.6th
Co-cultured different pairs in improved W medium in the same condition (using two tubes in each group):
Microscopic examination of each bacterium and each pair by means of Fuchsin or Gram dye. To our surprise, there were some pairs (P.p + R.j) living well in the same tube and also there were some pairs we were not sure. It seemed that P.putida KT2440 was the dominant bacterium when it was co-cultured with others. We were thinking about limiting its growth by control ingredients in different media.
Construction of PBBR
  Aug.5th
        Extraction of plasmid pBBR1MCS-2
  Aug.6th
            Cut pBBR1MCS-2 with restricted enzymes Xba1 and Sac1 and checked by agarose gel electrophoresis:
        Failed. After gel recovery, the concentration of digested pBBR1MCS-2 was 2.4 ng/μl, which was too low to be used in the next step.
Gene Knockout of Escherichia coli
We tried to copy tet gene(tetracycline resistance gene) in two overlap area of the fragment(about 500bp and 1000bp,called ‘tet-1’ and ‘tet-2’) and the homologous arms of the knockout gene-atpF and atpH(about 450bp each,called ‘left’ and ‘right’)with the help of PCR.
Show More
Week3(8/7/2016-8/13/2016)
Optimization of Culture Conditions
  Aug.7th
For each pair, we checked the bacterial concentration at OD_600 and detected the concentration of TPA by UV at Abs_242.According to results of the UV tests, if R.j existed, whatever the strategy was, the concentration of TPA would significantly decrease. Especially for pairs R.j+P.p and R.j + B.s, results showed greater reductions in the concentration of TPA compared with control group.
Pre-culture and acclimatize P.putida KT2440 in a condition with 5% EG To our surprise, this wild type of P.putida could grow well in this condition, we considered increasing the concentration of EG in the condition.
  Aug.9th
1.Extract 5ml YPD medium to 8 test tubes, respectively.
2.Added bacteria solution as following table (use two tubes each group)
3.Cultured them at 30℃ & 200 rpm, checked the bacterial concentration at OD600 and detected the concentration of TPA by UV at OD242
  Aug.10th
1.Prepared 300ml W0 medium and add 1.5g Yeast Extract, then regarded it W1 medium.
2.Extracted 8ml W1 medium to 16 test tubes, respectively.
3.Added bacteria solution as following table (use two tubes each group).
4.Cultured them at 30℃ & 200 rpm, checked the bacterial concentration at OD600 and detected the concentration of TPA by UV at OD242.
  Aug.12th
1.Prepared 200ml W0 medium and add 0.4g Urea, then regarded it W2 medium.
2.Extracted 8ml W2 medium to 16 test tubes, respectively.
3.Added bacteria solution as following table (use two tubes each group)
4.Cultured them at 30℃ & 200 rpm, checked the bacterial concentration at OD600 and detected the concentration of TPA by UV at OD242.
1.Prepared 200ml W0 medium and add 0.4g KNO3, then regarded it W3 medium.
2.Extracted 8ml W3 medium to 16 test tubes, respectively.
3.Added bacteria solution as following table (use two tubes each group)
4.Cultured them at 30℃ & 200 rpm, checked the bacterial concentration at OD600 and detected the concentration of TPA by UV at OD242.
  Aug.13th
1.Extract 5ml W medium to 6 test tubes and add 20μL, 50μL, 80μL EG to them, respectively;
2.Add 10 μL bacteria solution of Pseudomonas putida KT2440 to the six test tubes, respectively;
3. Cultured them at 30℃ & 200 rpm, checked the bacterial concentration at OD600.
Added bacteria solution to W2, W3 medium as following table (use two tubes each group)
Construction of PBBR
  Aug.7th
Repeated the digestion and agarose gel electrophoresis:
Also Failed. After gel recovery, the concentration of digested pBBR1MCS-2 was only 2.4 ng/μl. We needed to increase the concentration of pBBR1MCS-2 in EB Solution.
  Aug.8th
Colony PCR of P.putida KT2440 to get genes AcoA & AceA:
Result: Failed. Only primer dimers existed.
  Aug.9th
Repeated the colony PCR above:
Result: Also Failed. Only primer dimers existed. We considered using boiled bacteria as the template in the next PCR.
  Aug.11th
Extraction of plasmid pBBR1MCS-2.
The nucleic acid concentration of new EB solutions increased a lot. Increasing the amount of E.coli and decreasing the amount of EB did work.
  Aug.12th
PCR of P.putida KT2440 to get genes AcoA & AceA:
Result: Successful PCR. Obvious bright bands located at 900+ and 1300+ bp (AcoA 978bp and AceA 1326 bp). Using boiled P.putida as the template in the PCR worked well.
  Aug.13th
Cut pBBR1MCS-2 with restricted enzymes Xba1 and Sac1 and checked by agarose gel electrophoresis.
Result: Obvious bright bands located over 5000 bp.
After gel recovery, we got the EB solution with cut plasmid pBBR1MCS-2, but the concentration is only 6.6 ng/μl, which is still quite low.
Experiments about Bacillus subtilis
  Aug.9th
Inoculated B.subtilis 168 for transformation.
  Aug.10th
Plasmids of PETase and plasmid of pHP13-p43 in E.coli were isolated. Enzyme digestion using EcoR I and BamH I, then do gel extraction and then link them.
  Aug.11th
The plasmids from DNA ligation were transferred into B.subtilis 168, then cultivated on chloramphenicol containing LB plates to observe whether successful.
  Aug.12th
B.subtilis Transformation was observed failure，3 times.
Gene Knockout of Escherichia coli
Instead of enzyme-cut and link up, we used the overlap of each fragment to increase our aim gene(‘tet’, ‘left-right’, ‘left-tet-right’).
Show More
Week4(8/14/2016-8/20/2016)
Optimization of Culture Conditions
  Aug.15th
1.Prepared 200ml W0 medium and add 0.6g sucrose, then regarded it W4 medium.
2.Extracted 8ml W4 medium to 16 test tubes, respectively.
3.Added bacteria solution as following table (use two tubes each group)
4.Cultured them at 30℃ & 200 rpm, checked the bacterial concentration at OD600 and detected the concentration of TPA by UV at OD242.
  Aug.16th
Prepared 200ml W0 medium.
2.Extracted 8ml W0 medium to 16 test tubes, respectively.
3.Added bacteria solution as following table (use two tubes each group)
4.Cultured them at 30℃ & 200 rpm, checked the bacterial concentration at OD600 and detected the concentration of TPA by UV at OD242.
Construction of PBBR
  Aug.14th
Ligation of cut pBBR1MCS-2 and CFP and Chemical transformation of pBBR into E.coli: Ligation sites: Sac1 and Xba1
Result:
The ligation was successful, because we found the special band which represented CFP during the verification by PCR. We also successfully transformed the plasmid into E.coli which was cultured on the medium with kanamycin. However, the expression of CFP was not detected, which needed to be figure out.
  Aug.15th
Cut pBBR1MCS-2 with restricted enzymes EcoR1 and Sac1 and checked by agarose gel electrophoresis:
Result: Obvious bright bands located over 5000 bp.
After gel recovery, we got the EB solution with cut plasmid pBBR1MCS-2, and the concentration is 14.7 ng/μl.
  Aug.16th
Extraction of plasmid pBBR1MCS-2.
  Aug.17th
Ligation of cut pBBR1MCS-2 with AcoA and AceA and Chemical transformation of pBBRAA into E.coli:
Ligation sites: Sac1 and EcoR1
Result:
The ligation was successful, because we found the special band which represented AcoA and AceA during the verification by PCR. We also successfully transformed the plasmid into E.coli which was cultured on the medium with kanamycin.
Experiments about Bacillus subtilis
  Aug.14th
B.subtilis Transformation was observed failure.
Reconstruct the plasmids.
  Aug.15th
E.coli transformation was observed failure.
Enzyme digestion again.
E.coli transformation again.
  Aug.16th
E.coli transformation was observed failure.
Plasmid was enzyme digested and not observed correct band.
  Aug.17th
Plasmids of PETase was enzyme digestion again.
Inoculate B.subtilis DB104.
  Aug.19th
Inoculate E.coli of pHP13-p43.
Gene Knockout of Escherichia coli
We aimed to put the left-tet-right gene and plasmid which have λ-red gene into the the genome of E.coli. After two screening, we could gain the target strains which knocked out the atpF and atpH gene.
Show More
Week5(8/21/2016-8/27/2016)
Optimization of Culture Conditions
  Aug.22th
        1.Prepared 200ml W0 medium and add 0.3g NaOAc, then regarded it W7 medium.
        2.Extracted 8ml W7 medium to 16 test tubes, respectively.
        3.Added bacteria solution as following table (use two tubes each group)
      4.cultured them at 30℃ and check growing situation and the situation of TPA degradation.
Experiments about Bacillus subtilis
  Aug.24th
Cultivated transformed E.coli on chloramphenicol containing LB plates to observe whether successful.
Gene Knockout of Escherichia coli
We cultivateD Saccharomyces cerevisiae and E.coli which has knocked out atpF and atpH gene and introduced GFP gene in special culture medium which the only carbon source was xylose.
Show More
Week6(8/28/2016-9/3/2016)
Experiments about Bacillus subtilis
  Aug.29th
B.subtilis DB104 transformation.
  Aug.30th
B.subtilis DB104 transformation is successful.
  Sep.2th
Construct plamid of MHETase without success.
Transform into E.coli, in chloramphenicol containing LB culture.
Show More
Week7(9/4/2016-9/10/2016)
Experiments about Bacillus subtilis
  Sep.8th
Inoculate PETase transformed B.stubtilis 168 in 3 flask culture.
1.LB, chloramphenicol, PETase transformed B.stubtilis 168, PET film.
2.LB, chloramphenicol, PETase transformed B.stubtilis 168, no PET film.
3.LB, chloramphenicol, B.stubtilis 168, PET film.
  Sep.9th
Construct plamid of MHETase without success.
  Sep.10th
Construct plamid of MHETase without success.
Show More
Week8(9/11/2016-9/17/2016)
Optimization of Culture Conditions
  Sep.12th
Optimize the growing environment of Bacillus stubtilis by change medium components.Firstly, we prepared 400ml W medium, then, we devided the medium into four pieces averagely and number them No.1, No.2, No.3, No.4. Next, we added 0.1g NaCl, 0.1g NH4Cl, 0.3g sucrose to No.2, No.3, No.4, respectively. After that, we Extracted 5 ml No.1, No.2, No.3, No.4 and added them to four test tubes, then, we added 10 μL bacteria solution of Bacillus stubtilis 168 to four test tubes, respectively. Eventually, we cultured them at 37℃ and check growing situation.
  Sep.16th
        1.Prepared 200ml W0 medium and add 0.2g NH4Cl, then regarded it W8 medium.
        2.Extracted 10ml W8 medium to 16 test tubes, respectively.
        3.Added bacteria solution as following table (use two tubes each group)
        4.Cultured them at 30℃ & 200 rpm, checked the bacterial concentration at OD600 and detected the concentration of TPA by UV at OD242.
Experiments about Bacillus subtilis
  Sep.11th
Construct plamid of MHETase without success.
  Sep.12th
Construct plamid of MHETase without success.
Show More
Week9(9/18/2016-9/24/2016)
Optimization of Culture Conditions
  Sep.19th
        1. Prepared 1L W medium and add 2.5g TPA and 1.1875g NaOH.
        2. devided the medium into five pieces averagely and add chemicals as following table
        3. Extracted 5ml W9, W10, W11, W12, W13 medium to 40 test tubes, respectively.
        4. Added bacteria solution as following table (use two tubes each group)
        5. cultured them at 30℃ & 200 rpm, checked the bacterial concentration at OD600 and detected the concentration of TPA by UV at OD242.
  Sep.21th
        Repeat experiments of W9, W10 the day before yesterday.
  Sep.22th
        1. Use five mediums of Sep.19th and W0, W8 medium of Sep.16th.
        2. Extracted 5ml W0, W8, W9, W10, W11, W12, W13 medium to 56 test tubes, respectively.
        3. Added bacteria solution as following table (use two tubes each group)
        4. cultured them at 30℃ & 200 rpm, checked the bacterial concentration at OD600 and detected the concentration of TPA by UV at OD242.
  Sep.23th
        1. Prepared 400L W medium and add 1.2g KNO3 and 1.2g glucose.
        2. devided the medium into four pieces averagely and add chemicals as following table
        3. Extracted 5ml W0, W14, W15, W16 medium to 34 test tubes, respectively.
        4. Added bacteria solution as following table (use two tubes each group)
        5. cultured them at 30℃ & 200 rpm, checked the bacterial concentration at OD600 and detected the concentration of TPA by UV at OD242.
Experiments about Bacillus subtilis
  Sep.19th
Construct plasmid of MHETase without success.
Cultivated MHETase transformed B.stubtilis 168 on Erythromycin containing LB plates to observe, without success.
16SrDNA
  Sep.19th
we firstly tried to cultivate each of them(Rhodococcus RHA1, Pseudomonas putida KT2440, bacillus subtilis 168) in LB culture medium for amplification.
  Sep.21th
We use orthogonal test to prove each bacteria had each special DNA stripe from the method of bacteria colony-PCR.
  Sep.22th
Then we cultivate each of them and the mixture of three in W0 culture medium for amplification.
  Sep.23th
We did the same 4*3(bacteria liquid and six primer sequences) orthogonal test.
  Sep.24th
We cultivate each of them and the mixture of three in in modified W0 culture medium which changed the carbon source from glucose to sugar for amplification.
Show More
Week10(9/25/2016-10/1/2016)
16SrDNA
  Sep.25th
We still did the same 4*3(bacteria liquid and six primer sequences) orthogonal test.
Show More
