<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UESTC-China_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UESTC-China/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>team</TITLE></P><DIV class="demo" style="padding: 0 0; z-index:9999"><DIV class="containe" style="z-index:9999"><DIV class="row" style="z-index:9999"><DIV class="col-md-12" style="z-index:9999"><NAV class="navbar navbar-default navbar-mobile bootsnav" style="z-index:9999"><DIV class="navbar-header" style="height:60px; z-index:9999; border-bottom-style:solid; border-bottom-width:1px; border-bottom-color:#d9d9d9"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#navbar-menu" style="margin-top:13px;"><I class="fa fa-bars"> UESTC-China</I></BUTTON></DIV><DIV class="collapse navbar-collapse" id="navbar-menu" style="border-bottom-style:solid; border-bottom-width:1px" style="z-index:9999"><UL class="nav navbar-nav" data-in="fadeInDown" data-out="fadeOutUp"><LI><A href="https://2018.igem.org/Team:UESTC-China">HOME</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/achievement">ACHIEVEMENT</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">PROJECT</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/Demonstrate">Demonstrate</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">PART</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/Parts">Basic Part</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/Improve">Improve Part</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">MODEL</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/Model">Overview</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/Model_butanol">Butanol System Model</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/Model_h2">Hydrogen System Model</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/Attributions">ATTRIBUTIONS</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">H  P</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/Human_Practices">Integrate Human Practice</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/Public_Engagement">Engagement</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/Software">Gene Card Online</A></LI><LI class="dropdown"><A href="https://2018.igem.org/Team:UESTC-China/team" class="dropdown-toggle" data-toggle="dropdown">TEAM</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/team">Team</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/team_teamintroduction">Team Introduction</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">NOTEBOOK</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/Notebook">Day Note</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/notebook_protocol">Protocol</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/InterLab">Interlab</A></LI></DIV></NAV></DIV></DIV></DIV></DIV><DIV class="team" style="margin-top:-30px; position:relative; width:100%; z-index:1; background-color:#fff"><DIV class="neirong" style="position:relative; z-index:1 ;padding:50px 8% 50px 1%; background-color:#fff"><DIV class="row member"><DIV class="col-md-3 col-sm-4"><DIV class="fixed" style="position:fixed; top:150px; "><DIV class="fl_l"><H3>Demonstrate</H3></DIV><MENU id="tocc" class="hide-on-med-and-down" style="padding: 0.1px; left: 0px; z-index:1; font-family:'Candara',sans-serif;"><UL class="fl_l" style="margin-top:200px; opacity:0; 

max-width: 350px;"><LI><A href="#">1. Pathway construction</A></LI><LI><A href="#">2. From straw to glucose</A></LI><LI><A href="#">3. Butanol production</A></LI><LI><A href="#">4. Hydrogen production</A></LI><LI><A href="#">5. Work going on</A></LI><LI><A href="#">6. References</A></LI></UL></MENU></DIV></DIV><DIV class="main col-md-9 col-sm-8 col-xs-12" style="padding-left:80px;"><UL class="fl_r"><LI><DIV class="bigtitle">
            1. Pathway construction
        </DIV><DIV class="zhengwen">
            For efficient expression of multiple enzymes in <I>E. coli</I>, codon optimization of all target genes were performed before DNA synthesis. The obtained genes were subsequently cloned into different expression vectors by using Gibson Assembly and Golden Gate strategies. The resulting vectors piGEM2018-Module001, piGEM2018-Module002, and piGEM2018-Module003 are listed in Table 1. 
        </DIV><DIV class="xstitle">Table 1   Illustration of the three constructed vectors.</DIV><DIV class="zhengwen"><TABLE class="table table-hover"><TBODY><TR><TD>No.</TD><TD>	Vector	</TD><TD><I>E. coli</I> resistance</TD><TD>	Description</TD></TR><TR><TD>1</TD><TD>	piGEM2018-Module001</TD><TD>	Amp	</TD><TD>BBa_J23100-RBS-pelB+5D-Xyn10D-Fae1A-RBS-pelB+5D-Xyl3A-RBS-pelB+5D-Cex-RBS-pelB+5D-CenA-Ter</TD></TR><TR><TD>2</TD><TD>	piGEM2018-Module002	</TD><TD>Kan	</TD><TD>Ter-Ter-RBS-Fdh-RBS-FRE_adhE-FRE_ackA-RBS-AtoB-RBS-AdhE2-RBS-Crt-RBS-Hbd-Ter</TD></TR><TR><TD>3	</TD><TD>piGEM2018-Module003	</TD><TD>Kan	</TD><TD>BBa_J23100-RBS-FhlA-RBS-HydA-Ter</TD></TR></TBODY></TABLE></DIV><DIV class="zhengwen">
            Before DNA sequencing, those vectors were verified by restriction enzyme digestion. After electrophoresis analysis, the samples which contained the desired bands were selected and sent for sequencing. The sequencing results showed that all the above constructed vectors were successful. (Fig. 1)
        </DIV><DIV class="tu">
            Fig. 1 Double restriction enzyme digestion of three constructed vectors analyzed by using agarose gel electrophoresis. 
        </DIV><DIV class="tu">(a) piGEM2018-Module001 digested by <I>Eco32</I>Ⅰ+<I>Nco</I>Ⅰ(lane 1), piGEM2018-Module001 digested by <I>Nco</I>Ⅰ+<I>Xho</I>Ⅰ(lane 2); </DIV><DIV class="tu">(b) piGEM2018-Module002 digested by <I>Pst</I>Ⅰ+<I>Kpn</I>Ⅰ(lane 1), piGEM2018-Module002 digested by <I>Hind</I>Ⅲ+<I>Kpn</I>Ⅰ(lane 2); </DIV><DIV class="tu">(c) piGEM2018-Module003 digested by <I>EcoR</I>Ⅰ+<I>Nco</I>Ⅰ(lane 1), piGEM2018-Module003 digested by <I>BamH</I>Ⅰ+<I>Bgl</I>Ⅱ(lane 2).</DIV></LI><LI><DIV class="bigtitle">
            2. From straw to glucose
        </DIV><DIV class="zhengwen"><I>E. coli</I> BL21(DE3) carrying piGEM2018-Module001 was used to degrade straw to glucose. Three extracellular expressed enzymes ferulic acid esterase, xylanase and cellulase are involved in the pathway. The expression of the three target proteins were verified by enzyme activity using fermentation liquid and enzyme crude extract, respectively. The results showed that the activities of all three enzymes in fermentation liquid fraction are too low to be detected, which could be due to the fact of low enzyme concentration. We also determined the enzyme activity using enzyme crude extract. The enzyme activity in crude fraction obtained from the strain carrying piGEM2018-Module001 was higher than that without the corresponding vector for all enzymes. 
        </DIV><DIV class="tu">
            Fig. 2 Enzyme activity detection using crude extract fraction of strain with or without. Module001：All sample were collected after fermentation for 24 h. 
(a) The activity of xylanase. U/mL definition: 1 ml of enzyme solution (at 40 °C pH = 6.0) 1 h decomposition of xylan to produce 1 mg xylose, regarded as an enzyme unit [1]; 
(b) The activity of ferulic acid esterase. U/mL definition: The amount of enzyme (at 40 °C pH = 6.4) required to degrade 1 μmol of 4-nitrophenyl trans-ferulate per minute is one Unit of ferulic acid esterase activity [2]; 
(c) The activity of total cellulase. U/mL definition: 1 ml of enzyme solution (at 40 °C pH = 7.0) hydrolyzed the filter paper per minute to produce 1.0 μmol of glucose, which is one Unit of total cellulase activity [3]; 
(d) The activity of endoglucanase. U/mL definition: 1 ml of enzyme solution (at 40 ° C pH = 7.0) hydrolyzed CMC-Na per minute to produce 1.0 μmol of glucose, which is one Unit of endoglucanase activity [4].</DIV><DIV class="zhengwen">
            In order to further evaluate the extracellular expression of ferulic acid esterase, xylanase and cellulose, the formation of the corresponding intermediates were determined. GC-MS was used to detect ferulic acid. We shaked <I>E. coli</I> BL21(DE3) carrying piGEM2018-Module001 with corn stalk and the ferulic acid production was monitored by periodically taking samples from the fermentation liquid of <I>E. coli</I> BL21(DE3) carrying piGEM2018-Module001. Carvacrol was used as an internal standard for GC measurements [5]. GC results showed that the peak of ferulic acid was appeared after fermentation for 24h with the strain carrying piGEM2018-Module001, while no ferulic acid was detected in the sample of <I>E. coli</I> BL21(DE3) without vector (Fig. 3). The ferulic acid peak was further analyzed by using GC-MS (Fig. 4).
        </DIV><DIV class="tu">
            Fig. 3 The production of ferulic acid determined using GC. The supernatants of fermented liquid of <I>E. coli</I> BL21(DE3) with or without  piGEM2018-Module001were used for the determination of ferulic acid, respectively. Samples were shaking with corn straw and taken from reaction mixtureat 0h, 24h.
        </DIV><DIV class="tu">
            Fig. 4  Mass spectrums of ferulic acid derivative. Sample was the supernatants of <I>E. coli</I> BL21(DE3) carrying piGEM2018-Module001 after fermentation for 24h.
        </DIV><DIV class="zhengwen">
            The activity of cellulase was determined by using Congo Red assay [6].  As shown in Fig. 5, the strains carrying piGEM2018-Module001 displayed a zone of clearance. Such a zone of clearance was also observed in the positive control carried out by using commercial cellulase, while no zone of clearance was observed with same <I>E. coli</I> strain carrying empty vector. The results indicated that cellulase was successfully extracellularly expressed in <I>E. coli</I> BL21(DE3) with piGEM2018-Module001. 
        </DIV><DIV class="tu">
            Fig. 5  Activity determination of cellulase using Congo Red assay. (a) CMC agar plate before staining with Congo Red. (b)CMC agar plate after staining with Congo Red. Module001: BL21(DE3) carrying piGEM2018-Module001（OD600: 1, 3, 5 from left to right); Positive Control: commercial cellulase (Enzyme concentration: 0.2, 0.3, 0.4 mg/ml from left to right); Negative Control: BL21 (DE3) carrying empty vector (OD600: 1, 3, 5 from left to right)
        </DIV></LI><LI><DIV class="bigtitle">
            3. Butanol production
        </DIV><DIV class="zhengwen"><I>E. coli</I> BL21(DE3) carrying piGEM2018-Module002 was used to convert  glucose to butanol. Five enzymes AtoB, Hbd, Crt, Ter and AdhE2 are involved in the pathway, which was validated by monitoring the production of butanol using GC. To test whether this multi-enzyme conversion system could work in our <I>E. coli</I> successfully or not, we detected the production curve of butanol.
        </DIV><DIV class="zhengwen">
            The butanol production was determined by periodically taking samples from the supernatant of fermented liquid of BL21(DE3). Isobutanol was used as an internal standard [7]. The results showed that the peak of butanol was appeared after fermentation for 24h with the strain carrying piGEM2018-Module002, while no butanol peak was observed for the negative control <I>E. coli</I> BL23(DE3) with empty vector(Fig. 6).
        </DIV><DIV class="tu">
            Fig. 6   The production of butanol determined using GC. The supernatants of fermented liquid of <I>E. coli</I> BL21(DE3) with or without piGEM2018-Module002 were used for the determination of butanol, respectively. For anaerobic growth, precultures were adjusted to OD600 10 with 2 mL of fresh medium with 50mg/mL kanamycin. The culture was transferred to a sealed 10ml vacuum tube. Cultures were shaken (200 rpm) at 37 °C for 24 h. Then samples were taken from reaction mixture at 0h, 24h.
        </DIV><DIV class="zhengwen">
            To improve butanol production, the fermentation conditions were optimized by using orthogonal array design by designing 17 three factor analysis. After a serious of butanol-production detection under different cultivating condition, we finally provided the necessary data for modeling. Then detecting the butanol production in 24h under the optimized conditions. (Fig. 7) And the final maximum yield of butanol is 0.406 g / L in 26 hours.
        </DIV><DIV class="tu">
            Fig. 7  The butanol production curve at optimized conditions (temperature=34℃, initial OD=13, initial pH=8)
        </DIV></LI><LI><DIV class="bigtitle">
            4. Hydrogen production
        </DIV><DIV class="zhengwen">
            To verify whether the piGEM2018-Module003 plasmid was working normally in <I>E. coli</I> DH5α, we set up an anaerobic fermentation unit. The produced gases flow through the following setup to remove carbon dioxide as much as possible to get hydrogen.
        </DIV><DIV class="tu">
            Fig. 8 Overall setup of the hydrogen collection device.
        </DIV><DIV class="zhengwen">
            The collected gas is qualitatively detected by combustion test. As seen from Video 1, one can clearly hear the sound of boom, while no similar phenomenon was observed for the control group.
        </DIV><DIV style="padding:20px 10%;"><VIDEO controls="controls" width="100%"><SOURCE src="https://static.igem.org/mediawiki/2018/8/80/T--UESTC-China--H2.mp4" type="video/mp4"> 当前浏览器不支持 video直接播放，点击这里下载视频：
                <A href="https://static.igem.org/mediawiki/2018/d/d9/T--UESTC-China--H2_%281%29.mp4">Download Video</A></SOURCE></VIDEO></DIV><DIV class="tu">
            Video 1. The video of gas combustion test. Samples were collected from the engineered <I>E. coli</I> DH5α carrying the plasmid piGEM2018-Module003 and wild type plasmid after shaking 48h in TB medium.
        </DIV><DIV class="zhengwen">
Next, we used gas collection devices to quantitatively detect hydrogen production. The recombinant strain and the original DH5α strain were fermented continuously for 36 hours in a 200 ml 1% (W/V) glucose M9 medium. Fig. 9 illustrates that our <I>E. coli</I> DH5α with piGEM2018-Module003 has higher hydrogen production than the original strain. At the same time, the hydrogen production of the recombinant bacteria increased exponentially with time (0-36h). Up to the end of fermentation, the yield of recombinant bacteria was 2.17mL/hr, which was 2.4 times higher than that of the original strain.
        </DIV><DIV class="tu">
            Fig. 9  A diagram of the volume of hydrogen generated over time in a gas collection device. Recombinant strain and original strain were fermented in 200 ml 1% (W/V) glucose M9 medium.
        </DIV><DIV class="zhengwen">
            Apart from the experiments, here we also carried out some theoretical modeling to provide some guidance to optimize the experimental condition. As demonstrated as following, 1 mol glucose can produce 2 mol hydrogen from the theoretical point of view [8]. In order to explore the optimal conditions for hydrogen production from <I>E. coli</I> DH5α with piGEM2018-Module003 more efficiently, we used vacuum vascularization (10mL) with 1mL 1% (w/v) glucose M9 medium and 100 microliters of recombinant bacteria growing to logarithmic metaphase to ferment under different conditions. Next, we measured the amount of hydrogen and formation rate following the 40hr anaerobic fermentation. The results showed that under the condition of 30°C, 1.13% (W/V), pH=6.8, our recombinant strain has the highest hydrogen production. Based on this, we compare our hydrogen production with relevant literatures. The results are as follows (Fig. 10).
        </DIV><DIV class="tu">
            Fig. 10 Comparing our hydrogen production with relevant literatures.
        </DIV><DIV class="zhengwen">
Fig. 10 shows that our <I>E. coli</I> DH5α with piGEM2018-Module003 hydrogen production is higher than most of the knockout and overexpression methods for <I>E. coli</I>'s own genes, but there is still a gap compared with the methods for cloning some heterologous genes into <I>E. coli</I>. 
        </DIV></LI><LI><DIV class="bigtitle">
            5. Work going on
        </DIV><DIV class="smtitle">
            Import suicide gene
        </DIV><DIV class="zhengwen">
The risks of our project are about the antibiotic resistance genes in case of the accident of incorrectly operation. However, it’s not uncontrollable. We plan to add a “suicide gene” leading the bacteria to express endolysin to kill themselves when they escape our cultivate medium.</DIV><DIV class="smtitle">
            Import resistance gene
        </DIV><DIV class="zhengwen">
            GroESL, which is a combination of GroEL and GroES gene has been shown to increase butanol production in <I>E. coli</I> [12]. We have test the tolerance of <I>E. coli</I> BW25113 with GroESL gene and original BW25113 under the condition of 1% (v/V) butanol concentration. The result shows that GroESL has a positive effect on the butanol tolerance of <I>E. coli</I> (Fig. 11). In the future, we will use <I>E. coli</I> BW25113 combining piGEM2018-Module002 with GroESL gene to produce butanol.   
        </DIV><DIV class="tu">Fig. 11  After adding 0.3ml butanol in 30ml LB medium, the measurement of optical density (OD600) deference between DH5α with GroESL and Negative Control in 24 hour.</DIV><DIV class="smtitle">
            Knock out relevant gene
        </DIV><DIV class="zhengwen">
            Due to limitations of experimental conditions, we are not able to knock out competitive pathway. However, many by-products will produce in the process of butanol production, namely frdABCD for succinate, ldhA for lactate, pta-ack for acetate, and adhE for ethanol (Fig. 12)[13]. If we could knock out these competitive pathway, then our butanol production will be greatly improved.
        </DIV><DIV class="tu">Fig. 12   <I>E. coli</I> self-metabolic pathway</DIV></LI><LI><DIV class="bigtitle">
            7. References
        </DIV><DIV class="zhengwen"><DIV>[1]Shen C, Li R, Hu T &amp; Yan K. 2011. Determination of xylanase activity with DNS method. Dyeing &amp; Finishing, 2: 35-39.</DIV><DIV>[2]Zhang S, Pei X &amp; Wu Z. 2009. Cloning and expression of feruloyl esterase A from Aspergillus niger, and establishment of fast activity detection methods. Chinese Journal of Applied &amp; Environmental Biology, 2: 276-279.</DIV><DIV>[3]Luciano Silveira MH, Rau M, Pinto da Silva Bon E &amp; Andreaus J. 2012. A simple and fast method for the determination of endo- and exo-cellulase activity in cellulase preparations using filter paper. Enzyme and Microbial Technology, 51: 280-285.
</DIV><DIV>[4]Wood TM &amp; Bhat KM. 1988. Methods for measuring cellulase activities. Methods in Enzymology, 160: 87-112.
</DIV><DIV>[5]Li Q, Gan G, Wang G &amp; Liu Y. 2007. Determination of ferulic acid in ligusticum chuanxiong Hort.Oil by GC. Lishizhen Medicine &amp; Materia Medica Research, 7: 1687-1688.
</DIV><DIV>[6]Lakhundi SS. Synthetic biology approach to cellulose degradation[D]. Edinburgh: University of Edinburgh, 2012.
</DIV><DIV>[7]Atsumi S, Cann AF, Connor MR, Shen CR, Smith KM, Brynildsen MP &amp; Liao JC. 2008. Metabolic engineering of E. coli for 1-butanol production. Metabolic Engineering, 10: 305-311.
</DIV><DIV>[8]Hallenbeck PC &amp; Ghosh D. 2012. Improvements in fermentative biological hydrogen production through metabolic engineering. Journal of Environmental Management, 95: 360-364.</DIV><DIV>[9]Yoshida A, Nishimura T, Kawaguchi H, Inui M &amp; Yukawa H. 2006. Enhanced hydrogen production from glucose using ldh- and frd-inactivated Escherichia coli strains. Applied Microbiology and Biotechnology, 73: 67-72.</DIV><DIV>[10]Akhtar MK &amp; Jones PR. 2009. Construction of a synthetic YdbK-dependent pyruvate:H2 pathway in Escherichia coli BL21(DE3). Metabolic Engineering, 11: 139-147.</DIV><DIV>[11]Chittibabu G, Nath K &amp; Das D. 2006. Feasibility studies on the fermentative hydrogen production by recombinant escherichia coli, bl-21. Process Biochemistry, 41: 682-688.
</DIV><DIV>[12]Abdelaal AS, Ageez AM, El AEHAA &amp; Abdallah NA. 2015. Genetic improvement of n-butanol tolerance in Escherichia coli by heterologous overexpression of groESL operon from Clostridium acetobutylicum. 3 Biotech, 5: 401-410.</DIV><DIV>[13]Shen C &amp; Liao J. 2008. Metabolic engineering of Escherichia coli for 1-butanol and 1-propanol production via the keto-acid pathways. Metabolic Engineering, 10: 312-320.</DIV></DIV></LI></UL></DIV></DIV></DIV></DIV><DIV class="hehe" style="background-color:balck"><DIV class="container"><DIV class="row"><DIV class="col-md-12 " style="text-alien: center; vertical-alien: middle; font-size: 127%; text-align: center;">Copyright © 2018 iGEM UESTC-China </DIV><DIV class="col-md-12" style="text-alien: center; vertical-alien: middle; font-size: 127%; text-align: center;"><DIV class="social-icons" style="font-size: 170%;">   </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>