<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UESTC-China_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UESTC-China/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>team</TITLE></P><DIV class="demo" style="padding: 0 0; z-index:9999"><DIV class="containe" style="z-index:9999"><DIV class="row" style="z-index:9999"><DIV class="col-md-12" style="z-index:9999"><NAV class="navbar navbar-default navbar-mobile bootsnav" style="z-index:9999"><DIV class="navbar-header" style="height:60px; z-index:9999; border-bottom-style:solid; border-bottom-width:1px; border-bottom-color:#d9d9d9"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#navbar-menu" style="margin-top:13px;"><I class="fa fa-bars"> UESTC-China</I></BUTTON></DIV><DIV class="collapse navbar-collapse" id="navbar-menu" style="border-bottom-style:solid; border-bottom-width:1px" style="z-index:9999"><UL class="nav navbar-nav" data-in="fadeInDown" data-out="fadeOutUp"><LI><A href="https://2018.igem.org/Team:UESTC-China">HOME</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/achievement">ACHIEVEMENT</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">PROJECT</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/Demonstrate">Demonstrate</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">PART</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/Parts">Basic Part</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/Improve">Improve Part</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">MODEL</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/Model">Overview</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/Model_butanol">Butanol System Model</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/Model_h2">Hydrogen System Model</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/Attributions">ATTRIBUTIONS</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">H  P</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/Human_Practices">Integrate Human Practice</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/Public_Engagement">Engagement</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/Software">Gene Card Online</A></LI><LI class="dropdown"><A href="https://2018.igem.org/Team:UESTC-China/team" class="dropdown-toggle" data-toggle="dropdown">TEAM</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/team">Team</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/team_teamintroduction">Team Introduction</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">NOTEBOOK</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/Notebook">Day Note</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/notebook_protocol">Protocol</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-China/InterLab">Interlab</A></LI></DIV></NAV></DIV></DIV></DIV></DIV><DIV class="team" style="margin-top:-30px; position:relative; width:100%; z-index:1; background-color:#fff"><DIV class="neirong" style="position:relative; z-index:1 ;padding:50px 8% 50px 1%; background-color:#fff"><DIV class="row member"><DIV class="col-md-3 col-sm-4"><DIV class="fixed" style="position:fixed; top:200px; "><DIV class="fl_l" style="opacity: 0;"><H3>DESIGN</H3></DIV><MENU id="tocc" class="hide-on-med-and-down" style="padding: 0.1px; left: 0px; z-index:1; font-family:'Candara',sans-serif;"><UL class="fl_l" style="margin-top: 200px; max-width: 396px; opacity: 0;"><LI class=""><A href="#">1. Overview</A></LI><LI class=""><A href="#">2. Construction of straw to energy pathway</A></LI><LI class=""><A href="#">3. References</A></LI></UL></MENU></DIV></DIV><DIV class="main col-md-9 col-sm-8 col-xs-12" style="padding-left:100px;"><UL class="fl_r"><LI><DIV class="bigtitle">
            1. Overview
        </DIV><DIV class="zhengwen">
            How to integrate the two related pathways which is involved in the straw degradation and the synthesis of butanol and hydrogen? According to previous studies, we designed three module vectors in the present work. The first module piGEM2018-Module001 was designed to achieve the transform from straw to glucose, and it performed the function of bifunctional enzyme. 
The other two modules piGEM2018-Module002 and piGEM2018-Module003 were designed to produce two types of clean energy, butanol and hydrogen from glucose (Fig. 1). After we checked if these three modules could work independently, we aimed to combine them together to achieve efficient conversion of straw to butanol and hydrogen in the near future. 
        </DIV><DIV class="tu">
            Fig. 1 Schematic illustration of energy conversion system.
        </DIV></LI><LI><DIV class="bigtitle">
            2. Construction of straw to energy pathway
        </DIV><DIV class="zhengwen">
            There are three modules synthesized in our energy conversion systems, including straw degradation, butanol production and hydrogen production (Fig. 2 ). Prior to the DNA synthesis, we carried out the codon optimization for the purpose to gain better expression for the target genes in <I>E. coli</I>.
        </DIV><DIV class="tu">
            Fig. 2 Schematic illustration of the straw degradation, butanol production and hydrogen production pathway. Xyn10D-Fae1A: Bifunctional xylanase/ferulic acid esterase; Cex: exo-β-1,4-glucanase gene; CenA: endo-β-1,4-glucanase gene; Xyl3A: xylosidase/arabinofuranosidase; FhlA: Enterobacter cloacae Fe-hydrogena; HydA: Hydrogen enzyme; Fdh: formate dehydrogenase; AtoB: acetyl-CoA acetyltransferase; Hbd: 3-hydroxybutyryl-CoA dehydrogenase; Crt: crotonase; Ter: Trans-2-enoyl-CoA reductase; AdhE2: aldehyde/alcohol dehydrogenase.
        </DIV><DIV class="smtitle">
            From Straw to glucose
        </DIV><DIV class="zhengwen">
            As shown in Fig. 2, the structure of straw is compact and is difficult to be decomposed, because cellulose is wrapped by hemicellulose, and is interspersed with lignin via ferulate ester bond (Fig. 3 ). It has been found that the presence of lignin would give rise to the reduced cellulase activity. Therefore, it is necessary and significant to pretreat the straw to make it separate with the lignin. The commonly used methods included physical method, chemical method and biological method. The physical method is not suitable for the industrial production because of its complication and high cost. Chemical methods often cause environmental pollution due to the usage of large amount of acid or alkali. Biological method has been unable to treat straw directly because of its low enzyme activity with the presence of lignin. Nevertheless, we know from some literatures that ferulic acid esterase, xylanase and β-glucosidase have synergistic effect, which greatly accelerated the degradation rate of straw [1]. 
        </DIV><DIV class="tu">
            Fig. 3 Schematic diagram of straw structure [2].
        </DIV><DIV class="zhengwen">
            The enzymes Xyn10D-Fae1A, Xyl3A, Cex and CenA are the key enzymes to degrade straw into glucose and other products shown in Fig. 4a. Xyn10D-Fae1A is the bifunctional enzyme, which has the activity of both ferulic acid esterase and xylanase. Ferulic acid esterase (abbreviated as Fae1A) can break the ferulic acid ester bond, which is responsible for attaching the complex cellular cell wall structures. Xylanase can hydrolyze xylan existing in plant cell wall, in which it can cut the β-1,4 glycosidic bond between the xylose residues in the xylan backbone [3]. From BioBrick, we found the endoglucanase CenA, (BBa_K118023) and exoglucanase Cex, (BBa_K118022). Cellulose will be degraded into glucose with the help of Xyl3A, CenA and Cex [4]. Besides, an extra signal peptide pelB followed by five aspartate repeats (pelB+5D seen in Fig. 4b) were introduced to facilitate the extracellular expression of the four enzymes in <I>E. coli</I> [5].
        </DIV><DIV class="tu">
            Fig. 4 Biodegradation pathway of straw to glucose and corresponding vector. 
(a) Pathway of straw degradation and glucose production. (b) Schematic map of piGEM2018-Module001. Xyn10D-Fae1A: Bifunctional xylanase/ferulic acid esterase; Xyl3A: xylosidase/arabinofuranosidase; Cex: exo-β-1,4-glucanase gene; CenA: endo-β-1,4-glucanase gene; RBS: ribosomebinding site; BBa_J23100: constitutive promoter family member; pelB+5D: an extra signal peptide pelB followed by five aspartate repeats; Ter: Trans-2-enoyl-CoA reductase.

        </DIV><DIV class="smtitle">
            Butanol production From glucose to butanol
        </DIV><DIV class="zhengwen">
           In order to meet the needs of industrial production of butanol, a butanol synthesis pathway existed in native <I>Clostridium</I> was rebuilt in <I>E. coli.</I> Basing on previous studying [6], we adopted condon optimization and gene stacking strategy to further optimize this butanol production pathway (Fig. 5a).
        </DIV><DIV class="zhengwen">
            FRE_adhE and FRE_ackA is the upstream of AdhE and AckA gene, which includes the promoter, ribosome binding site (RBS) and the entire transcription factor (TF) binding site. They has been adopted by Wen et al. to enhance the production of butanol in <I>E. coli</I> without adding revulsant, antibiotics and under anaerobic fermentation conditions, which is hard for industrial production [7]. We want to not only produce large amounts of butanol, but also meet the needs of lower cost production. Therefore, We assemble these parts together and construct our plasmid of butanol production (Fig. 5b).
        </DIV><DIV class="tu">
            Fig. 5 Biosynthesis pathway of glucose to butanol and corresponding vector. 
(a) Pathway of butanol biosynthesis. (b) Schematic map of piGEM2018-Module002. (c) Schematic map of piGEM2018-Module004. Ter: trans-2-enoyl-CoA reductase; RBS: ribosomebinding site; Fdh: formate dehydrogenase; FRE_adhE: The fermentation control element of <I>E. coli</I>; FRE_ackA: The fermentation control element of <I>E. coli</I>; AtoB: acetyl-CoA acetyltransferase; AdhE2, aldehyde/alcohol dehydrogenase; Crt: crotonase; Hbd: 3-hydroxybutyryl-CoA dehydrogenase; BBa_J23100: constitutive promoter family member; GroEL: Chaperone protein; GroES: Chaperone protein.
        </DIV><DIV class="zhengwen">
            The accumulation of butanol exerts a greatly inhibitory effect on cell growth during butanol production process. Thus, we will import GroESL gene from Clostridium acetobutylicum to allow <I>E. coli</I> expresses heat shock protein groESL chaperone, which has shown the function for improving the yield of butanol by [8] (Fig. 5c).
        </DIV><DIV class="smtitle">
           Hydrogen production From glucose to hydrogen 
        </DIV><DIV class="zhengwen"><I>E. coli</I> is a microorganism capable of producing hydrogen under anaerobic conditions, which can convert the methyl formate to hydrogen gas and carbon dioxide (Fig. 6a) via an enzyme complex called formate hydrogenlyase (FHL) [9]. It has been reported that FhlA, an activating protein of FHL, could enhance the production capacity of hydrogen [9]. Additionally, structural analysis showed that FHL from <I>E. coli</I> belongs to [Ni-Fe] hydrogenase family. Its enzymatic activity is about 100 times lower than another hydrogenase from green algae and some bacteria. Therefore we introduced the gene encoding the hydrogenase from Enterobacter cloacae IIT-BT 08 into <I>E. coli</I> to further increase hydrogen production [10] (Fig. 6b).
        </DIV><DIV class="tu">
            Fig. 6 Biodegradation pathway of glucose to hydrogen and corresponding vector. 
(a) Pathway of hydrogen biosynthesis. (b) Schematic map of piGEM2018-Module003. BBa_J23100: constitutive promoter family member; FhlA: Enterobacter cloacae Fe-hydrogena; HydA: Hydrogen enzyme; Ter: Trans-2-enoyl-CoA reductase.

        </DIV></LI><LI><DIV class="bigtitle">
            3. References
        </DIV><DIV class="zhengwen">[1]Liu P. Study on the degradation of corn stover by biological method and the heat resistance modification of ferulic acid esterase [D]. Jinan：Qilu University of Technology, 2016.
            [2]Doherty WOS, Mousavioun P &amp; Fellows CM. 2011. Value-adding to cellulosic ethanol: Lignin polymers. Industrial Crops and Products, 33: 259-276.
            [3]Dodd D, Kocherginskaya SA, Spies MA, Beery KE, Abbas CA, Mackie RI &amp; Cann IK. 2009. Biochemical analysis of a β-D-xylosidase and a bifunctional xylanase-ferulic acid esterase from a xylanolytic gene cluster in Prevotella ruminicola 23. Journal of Bacteriology, 191: 3328-3338.
            [4]Lakhundi SS. Synthetic biology approach to cellulose degradation[D]. Edinburgh: University of Edinburgh, 2012.
            [5]Kim SK, Park YC, Lee HH, Jeon ST, Min WK &amp; Seo JH. 2015. Simple amino acid tags improve both expression and secretion of Candida antarctica lipase B in recombinant Escherichia coli. Biotechnology and Bioengineering, 112: 346-355.
            [6]Atsumi S, Cann AF, Connor MR, Shen CR, Smith KM, Brynildsen MP &amp; Liao JC. 2008. Metabolic engineering of Escherichia coli for 1-butanol production. Metabolic Engineering, 10: 305-311.
            [7]Wen RC &amp; Shen CR. 2016. Self-regulated 1-butanol production in Escherichia coli based on the endogenous fermentative control. Biotechnology for Biofuels, 9: 267.
            [8]Zingaro KA &amp; Papoutsakis ET. 2013. GroESL overexpression imparts Escherichia coli tolerance to i-, n-, and 2-butanol, 1, 2, 4-butanetriol and ethanol with complex and unpredictable patterns. Metabolic Engineering, 15: 196-205.
            [9]Yoshida A, Nishimura T, Kawaguchi H, Inui M &amp; Yukawa H. 2005. Enhanced hydrogen production from formic acid by formate hydrogen lyase-overexpressing Escherichia coli strains. Applied and Environmental Microbiology, 71: 6762-6768. 
            [10]Chittibabu G, Nath K &amp; Das D. 2006. Feasibility studies on the fermentative hydrogen production by recombinant Escherichia coli BL-21. Process Biochemistry, 41: 682-688.
</DIV></LI></UL></DIV></DIV></DIV></DIV><DIV class="hehe" style="background-color:balck"><DIV class="container"><DIV class="row"><DIV class="col-md-12 " style="text-alien: center; vertical-alien: middle; font-size: 127%; text-align: center;">Copyright © 2018 iGEM UESTC-China </DIV><DIV class="col-md-12" style="text-alien: center; vertical-alien: middle; font-size: 127%; text-align: center;"><DIV class="social-icons" style="font-size: 170%;">   </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>