Team:UESTC-China/project design
team
 UESTC-China
HOME
ACHIEVEMENT
PROJECT
Introduction
Design
Demonstrate
Improve
PART
Part
Improve
MODELING
ATTRIBUTIONS
H  P
Our Story
Supporting Research
Engagement
TEAM
Team
Team Introduction
Collaborations
NOTEBOOK
Day Note
Protocol
Safety
Interlab
Overview
Construct the pathway
Improve the function
References
Overview
How to integrate the two pathways related to both straw degradation and the synthesis of biofuel? After reading many literature, especially for the work of Dodd et al., (2009) Maeda et al., (2018) and Wen et al., (2016) , we designed our system. In the system, three module plasmids were designed separately. This could endow our system with multiple functions by sequential combination of these three modules. In the end, our system could degrade straw successfully and transform the cellulose into valuable biofuel.
Figure 1. Schematic illustration of our system.
 
Overview
There are three modules in our systems, inculding straw degradation, hydrogen production and butanol production. To get a good expression of these genes in E. coli, we did codon optimization before DNA synthesis.
Figure 2. Pathway of our system.
1、   straw degradation and glucose production
Figure 3. Pathway of straw degradation and glucose production
The enzymes Xyn10D-fae1A, Xyl3A, cex and cenA are the key enzymes to degrade straw into glucose and other products. Xyn10D-fae1A is a bifunctional enzyme，which has the activity of ferulic esterase and xylanase. Feruloyl esterase can hydrolyze ferulic acid ester groups, which are responsible for attaching in complex cellular cell wall structures. Xylanase can hydrolyze plant cell wall component xylan. It can cut the β-1,4 glycosidic bond between the xylose residues in the xylan backbone. Feruloyl esterase, xylanase  and Xyl3A also have a synergistic effect[1].
From BioBrick, we found endoglucanase cenA, (BBa_K118023)and exoglucanase cex, (BBa_K118022).  Cellulose will be degraded into glucose with the help of  Xyl3A, cenA and cex[2].
Besides, to achieve extracellular expression, an extra signal peptide pelB together with five aspartate repeats were adopted to facilitate the extracellular expression of the four enzymes in E. coli (Kim et al., 2015)[3].
Figure 4. Schematic illustration of pelB+5D
2、   hydrogen production
Escherichia coli is a microorganism capable of producing hydrogen under anaerobic conditions,whcich can convert the formateinto to hydrogen gas and carbon dioxide, via an enzyme complex called formate hydrogenlyase (FHL). On this basis, we decided to enhance the hydrogen production capacity of E. coli by overexpressing FhlA, an activating protein gene of the FHL system[4]. Meanwhile, we think about the hydrogenase in FHL system and find that it belongs to [Ni-Fe] hydrogenase with low efficiency. In green algae and some bacteria, there is another hydrogenase - [Fe] hydrogenase, whose catalytic center has a unique non-prion H cluster, so it is 100 times more active than other hydrogenases. Therefore, we decided to introduce the [Fe] hydrogenase gene Hyda of Enterobacter cloacae IIT-BT 08 into E. coli to further increase hydrogen production. [5]
2、   butanol production
There is a butanol synthesis pathway in native Clostridium, in order to meet the needs of industrial production, we hope to achieve continuous production of butanol in E.coli.  According to literature which studies the screening of same enzymes of different species, we finally tried to assemble a relatively superior combination[6], which mainly uses the following six genes to construct a biosynthetic pathway from glucose to butanol：
Figure 5. Pathway of butanol production. the gene are listed as follow:Fdh, formate dehydrogenase; AtoB, acetyl-CoA acetyltransferase;  Hbd, 3-hydroxybutyryl-CoA dehydrogenase; Crt, crotonase; ter, trans-2-enoyl-CoA reductase; AdhE2, aldehyde/alcohol dehydrogenase;
Meanwhile, FRE is the transcriptional and translational regulatory element upstream of these mixed acid fermentation genes, including the promoter, ribosome binding site (RBS) and the entire transcription factor (TF) binding site. Wen et al. have proved that using its original regulatory element of ackA and adhE (FREadhE, FREackA) has a good effect on butanol production, without adding revulsant, antibiotics and under anaerobic fermentation conditions[7] . Therefore, We assemble these parts together and  construct our plasmid of butanol production.
 
Improve the function
In addition to the activity of the enzyme in E.coli, culture conditions represent another crucial factor for the improvement of butanol production efficiency. We used response surface methodology to explore the optimal temperature, optimal initial pH, and optimal initial OD for the fermentation process. The Box-Behnken Design method (BBD) was used to design the experiment, and the BBD experimental data was analyzed by regression analysis to obtain a binomial regression model to explain the fermentation process[8]. The theoretical optimal conditions for the fermentation process are obtained by finding the optimal solution for the model[9].
Besides, the accumulation of butanol exerts a great inhibitory effect on cell growth during the production process. Thus, we will import GroESL gene from Clostridium acetobutylicum to allow E.coli express the heat shock protein groESL chaperone, which has proved that it could improve the yield of butanol[10].
 
References
[1]Dodd, D., Kocherginskaya, S. A., Spies, M. A., Beery, K. E., Abbas, C. A., Mackie, R. I., & Cann, I. K. (2009). Biochemical analysis of a β-D-xylosidase and a bifunctional xylanase-ferulic acid esterase from a xylanolytic gene cluster in Prevotella ruminicola 23. Journal of bacteriology, 191(10), 3328-3338.
[2]Lakhundi, S. S. (2012). Synthetic biology approach to cellulose degradation.
[3]Kim, S. K., Park, Y. C., Lee, H. H., Jeon, S. T., Min, W. K., & Seo, J. H. (2015). Simple amino acid tags improve both expression and secretion of Candida antarctica lipase B in recombinant Escherichia coli. Biotechnology and bioengineering, 112(2), 346-355.
[4]Yoshida, A., Nishimura, T., Kawaguchi, H., Inui, M., & Yukawa, H. (2005). Enhanced hydrogen production from formic acid by formate hydrogen lyase-overexpressing Escherichia coli strains. Applied and Environmental Microbiology, 71(11), 6762-6768.
[5]Chittibabu, G., Nath, K., & Das, D. (2006). Feasibility studies on the fermentative hydrogen production by recombinant Escherichia coli BL-21. Process Biochemistry, 41(3), 682-688.
[6] Atsumi, S., Cann, A. F., Connor, M. R., Shen, C. R., Smith, K. M., Brynildsen, M. P., ... & Liao, J. C. (2008). Metabolic engineering of Escherichia coli for 1-butanol production. Metabolic engineering, 10(6), 305-311.
[7]Wen, R. C., & Shen, C. R. (2016). Self-regulated 1-butanol production in Escherichia coli based on the endogenous fermentative control. Biotechnology for biofuels, 9(1), 267.
[8] Shukor, H., Al-Shorgani, N. K. N., Abdeshahian, P., Hamid, A. A., Anuar, N., Rahman, N. A., & Kalil, M. S. (2014). Production of butanol by Clostridium saccharoperbutylacetonicum N1-4 from palm kernel cake in acetone–butanol–ethanol fermentation using an empirical model. Bioresource technology, 170, 565-573.
[9]Pan, C. M., Fan, Y. T., Xing, Y., Hou, H. W., & Zhang, M. L. (2008). Statistical optimization of process parameters on biohydrogen production from glucose by Clostridium sp. Fanp2. Bioresource technology, 99(8), 3146-3154.
[10]Zingaro, K. A., & Papoutsakis, E. T. (2013). GroESL overexpression imparts Escherichia coli tolerance to i-, n-, and 2-butanol, 1, 2, 4-butanetriol and ethanol with complex and unpredictable patterns. Metabolic engineering, 15, 196-205.
 
Copyright © 2018 iGEM UESTC_China
     
