"
Page
Discussion
View source
History
teams
Log in
 
Team:UniSalento Lecce/Data
From 2013.igem.org
Home
Team
Team
University/Department
Project
Why NickBusters?
Experimental data and results
Biobricks and parts
Applications
Human practices
Notebook
Weekly Journal
Protocols
Safety
Extra
Model
Sponsors
Acknowledgments
1. Submitted parts agarose gel analysis
2. HpNikR characterization data
1. HpNikR PCR
2. HpNikR expression in BL21(DE3) cells
3. Cytosol/membrane separation by Zerial method
4. HpNikR purification by Ni-NTA resin
5. ICP-AES assay
6. ATR-FTIR spectroscopy assay
7. Raman spectroscopy assay
8. CryoTEM microscopy of purified HpNikR
3. Nickel responsive promoters characterization data
1. PCR amplification of the promoters
2. pnikr
3. Double divergent promoter (pnikR-pexbB)
Experimental data and results
1 - Submitted parts agarose gel analysis
Here you can find a PCR and/or a restriction analysis of these parts:
BBa_K1151001 , BBa_K1151005 , BBa_K1151009 , BBa_K1151010 , BBa_K1151011 , BBa_K1151036 , BBa_K1151038 , made to evaluate their identity.
The molecular weight of these parts has been evaluated with the Invitrogen 1kb DNA Ladder (15615-016)
BBa_K1151001
The K1151001 digestion with EcoRI and PstI displays a profile confirming its molecular weight.
The K1151001 PCR with VF2 ( BBa_BBa_G00100 ) and VR ( BBa_BBa_G00101 ) displays a profile confirming its molecular weight.
BBa_K1151005
The K1151005 digestion with EcoRI and PstI displays a profile confirming its molecular weight.
The K1151005 PCR with VF2 ( BBa_BBa_G00100 ) and VR ( BBa_BBa_G00101 ) displays a profile confirming its molecular weight.
BBa_K1151009
The K1151009 digestion with EcoRI and PstI displays a profile confirming its molecular weight.
The K1151009 PCR with VF2 ( BBa_BBa_G00100 ) and VR ( BBa_BBa_G00101 ) displays a profile confirming its molecular weight.
BBa_K1151010
The K1151010 digestion with EcoRI and PstI displays a profile confirming its molecular weight.
The K1151010 PCR with VF2 ( BBa_BBa_G00100 ) and VR ( BBa_BBa_G00101 ) displays a profile confirming its molecular weight.
BBa_K1151011
The K1151011 digestion with EcoRI and PstI displays a profile confirming its molecular weight.
The K1151011 PCR with VF2 ( BBa_BBa_G00100 ) and VR ( BBa_BBa_G00101 ) displays a profile confirming its molecular weight.
BBa_K1151036
The K1151036 PCR with VF2 ( BBa_BBa_G00100 ) and VR ( BBa_BBa_G00101 ) displays a profile confirming its molecular weight.
BBa_K1151038
The K1151038 digestion with EcoRI displays a profile confirming its molecular weight.
2 - HpNikR characterization data
We received this part by Dr. Alberto Danielli, a researcher in molecular biology at the Department of Pharmacology and Biotechnology at the University of Bologna. He sent us the gene encoding HpNikR (from the genome of H. pylori G27) inserted into a pET-15b plasmid.
For studying HpNikR, submitted as BBa_K1151000 , we proceeded with the following workflow. Go to the Protocols page to see our experimental procedures.
2.1 - HpNikR PCR
This part was amplified by PCR from the plasmid pETNikR.
Primers used (including Biobrick Prefix and Suffix, lowercase):
nikRFor:
gtttcttcgaattcgcggccgcttctagATGGATACACCCAATAAAGACG
nikRRev:
gtttcttcctgcagcggccgctactagtattatta CTATTCATTGTGTTCAAAG
2.2 - HpNikR expression using BL21(DE3) cells
First we made ​​competent BL21 cells and we transformed it with the plasmid containing HpNikR. We then proceeded with the normal protocol of induction with IPTG for a time of 1, 2 and 4 hours.
2.3 - Cytosol/membrane separation by Zerial method
To confirm that NikR is a cytosolic protein (not expressed in multivesicular bodies, and then in membrane) we performed a separation membrane-cytosol (Zerial method) (sample: 2-hours induced cells).
2.4 - HpNikR purification by Ni-NTA resin
NikR (sample: 2-hours induced cells) can be purified by Ni-NTA resin, which has a high affinity for histidine residues.
2.5 - ICP-AES assay
Inductively-Coupled Plasma Atomic Emission Spectroscopy (ICP-AES) is a type of emission spectroscopy that uses the inductively coupled plasma to produce excited atoms and ions that emit electromagnetic radiation at wavelengths characteristic of a particular element. The intensity of this emission is indicative of the concentration of the element within the sample. These are the samples (also here the final volume of each is 100 ul) analyzed, see the Protocols page for Incubation protocol:
100 ul Incubation buffer
1,5 ul Nickel sulfate + 98,5 ul Incubation buffer
1,2 ul (1 ug) NikR + 0,3 ul Nickel sulfate + 98,5 ul Incubation buffer
6,1 ul (5 ug) NikR + 1,5 ul Nickel sulfate + 92,4 ul Incubation buffer
12,2 ul (10 ug) NikR + 3 ul Nickel sulfate + 84,8 ul Incubation buffer
Sephadex resin after incubation with the sample n.4
Although it remains difficult to identify accurately the binding NikR-nickel stoichiometry we can say, however, that between the two variables exists a direct relationship.
2.6 - ATR-FTIR spectroscopy assay
To complete our analysis on the protein, we studied the conformational changes which HpNikR goes under in the bond with Ni2+ ions through Attentuated Total Reflectance Fourier Transform Infrared Spectroscopy (ATR-FTIR). With this technique it is possible to acquire the infrared absorption spectra of our control (Apo-HpNikR) and of our protein treated with NiSO4 (complex HpNikR:Ni), so as to obtain the IR differential spectrum. By means of these it will be possible to highlight how some spectrum peaks suffer changes in infrared absorption due to changes in protein conformation induced by the nickel binding.
Our hypothesis: The analysis of the spectrum of the native protein shows two peaks at values ​​of wave number of 1261 cm-1 and 800 cm-1; these peaks are absent in the protein in the presence of nickel and everything is confirmed by the differential spectrum. These peaks could be assigned to a tyrosine residue that due to the interaction with nickel undergoes a deprotonation of the hydroxyl functional group. So it can be assumed that the bond to the metal induces a change necessary to the regulator NikR for the interaction with DNA.
2.7 - Raman spectroscopy assay
Raman spectroscopy is a spectroscopic method based on the interaction between electromagnetic radiation used to observe vibrational, rotational, and other low-frequency modes in a system.
With Raman spectroscopy analysis it is possible to state that in the 1000-1600 cm-1 region the difference in signal comes from amide groups (I, II e III). Instead, to high frequences, in the 2900-3200 cm-1 region the difference in signal comes from the stretching of the CH2 and CH3 bonds.
Conformational deformations could be seen in the amide regions, but further investigations are necessary.
2.8 - CryoTEM microscopy of purified HpNikR
The images above shown are for explicative use only. For a full resolution image click here
The high quality of the CryoTEM image displays:
We made a correct sample preparation
We set up correct set of experimental and instrumental conditions
There is high probability that the image is relative to purified HpNikR protein. Assuming that ribosomal particles, whose molecular weight is around 200 kDa, have a diameter around 20 nm, the particle of diameter of around 1-2 nm which can be seen may coincide with a particle of MW around 20 kDa. HpNikR MW is around 17 kDa.
3 - Nickel responsive promoters characterization data
3.1 - PCR amplification of the promoters
We obtained the promoters from PCR with the following primers:
IntergenicaFor: gtttcttcgaattcgcggccgcttctagagTGAGAAAAATCCTTTTTTG
pnikrev: gtttcttcctgcagcggccgctactagtaTGAGAAAAATCCTTTTTTG
pnikfor: gtttcttcgaattcgcggccgcttctagagAATTCAAACGCTCTTATG
pexbfor: gtttcttcgaattcgcggccgcttctagagACTGGATTTAAATGGTTG
pexbrev:gtttcttcctgcagcggccgctactagtaGCACCCTATAAGAAGGCATC
Fluorescence Decay assays
We set up these experiments to evaluate the shutdown of the fluorescence signal, through a series of measurements conducted at fluorometer. We added a fixed quantity of IPTG and nickel to BL21 cells transformed with the plasmid containing this part, and exploited the ability of Holo-NikR to bind the pnikR and the double divergent promoter, thus repressing the transcription of GFP. (Nickel and IPTG added per flask: 0.3 ul from stock 10 ug / ul; 5 ul 1M)
3.2 - pnikR
To characterize this part we used BBa_K1151036 . These data shows the ability of HpNikR to regulate pnikR, regulating GFP expression.
3.3 - Double divergent promoter pnikR-pexbB
To characterize this part we used BBa_K1151038 . These data shows the ability of HpNikR to bind to the divergent promoter, with GFP cds cloned downstream of pexbB (ExbB promoter).
All the graphs show the difference between the 3 control samples (LB, LB + Ni2+, LB + IPTG) and the Ni-IPTG incubated sample regarding variations in time-dependent fluorescence signal. The expression of the repressor, HpNikR ( BBa_K1151000 ), under the control of a pLac promoter, has effects on the fluorescence level of GFP only in the cultures incubated with nickel. The response of repression of GFP is found in both the constructs ( BBa_K1151036 and BBa_K1151038 ) with HpNiKR responsive promoters and it is quite clear despite the long half-life of GFP. The proof showed the correct operation of the Biobricks, in particular the good functionality of the repression mechanism, nickel-dependent, operated by NikR on the two promoters, that composed a divergent intergenic region of Helicobacter pylori, pnikR and pexbB. Measurements were made, with a Tecan Infinite 200 PRO Multimode Reader, at the two typical wavelenghts of GFP in order to have a complete and secure framework of the experiments.
Sponsors
Contacts
Facebook
Twitter
Google+
E-mail: igem.unisalento@gmail.com
Design by Mattia Corvaglia - www.mattiacorvaglia.com - Visitors:
Back to Top
Retrieved from " http://2013.igem.org/Team:UniSalento_Lecce/Data "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
