<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UniSalento_Lecce_Overview"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UniSalento_Lecce/Overview">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UniSalento_Lecce/Overview&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UniSalento_Lecce/Overview&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UniSalento_Lecce/Overview&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UniSalento_Lecce/Overview" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UniSalento Lecce/Overview</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><NAV><UL id="css3menu1" class="topmenu"><LI class="topmenu"><A href="https://2013.igem.org/Team:UniSalento_Lecce" title="Home" style="height:24px;line-height:24px;">
Home
</A></LI><LI class="toproot"><A href="#" title="Team" style="height:24px;line-height:24px;">
Team
</A></LI><LI><A href="https://2013.igem.org/Team:UniSalento_Lecce/Team" title="Team">
Team
</A></LI><LI><A href="https://2013.igem.org/Team:UniSalento_Lecce/University" title="University/Department">
University/Department
</A></LI></UL></NAV></P></DIV></DIV><LI class="toproot"><A href="#" title="Project" style="height:24px;line-height:24px;"><SPAN>
Project
</SPAN></A></LI><LI><A href="#" title="Why NickBusters?">
Why NickBusters?
</A></LI><LI><A href="https://2013.igem.org/Team:UniSalento_Lecce/Data" title="Experimental data and results">
Experimental data and results
</A></LI><LI><A href="https://2013.igem.org/Team:UniSalento_Lecce/Parts" title="Biobricks and parts">
Biobricks and parts
</A></LI><LI><A href="https://2013.igem.org/Team:UniSalento_Lecce/Applications" title="Applications">
Applications
</A></LI></DIV><LI class="topmenu"><A href="https://2013.igem.org/Team:UniSalento_Lecce/Human_Practices" title="Human practices" style="height:24px;line-height:24px;"><SPAN>
Human practices
</SPAN></A></LI><LI class="toproot"><A href="#" title="Nootebook" style="height:24px;line-height:24px;"><SPAN>
Notebook
</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:UniSalento_Lecce/Notebook" title="Weekly Journal">
Weekly Journal
</A></LI><LI><A href="https://2013.igem.org/Team:UniSalento_Lecce/Protocols" title="Protocols">
Protocols
</A></LI><LI class="topmenu"><A href="https://2013.igem.org/Team:UniSalento_Lecce/Safety" title="Safety" style="height:24px;line-height:24px;"><SPAN>
Safety
</SPAN></A></LI><LI class="toproot"><A href="#" title="Extra" style="height:24px;line-height:24px;">
Extra
</A></LI><LI><A href="https://2013.igem.org/Team:UniSalento_Lecce/Modeling" title="Model">
Model
</A></LI><LI><A href="https://2013.igem.org/Team:UniSalento_Lecce/Sponsors" title="Sposors">
Sponsors
</A></LI><LI><A href="https://2013.igem.org/Team:UniSalento_Lecce/Attributions" title="Acknowledgments">
Acknowledgments
</A></LI><DIV id="main"><DIV id="main_wrap"><DIV id="container"><DIV class="fixsidebar"><UL class="nav nav-list bs-docs-sidenav" style="list-style:none"><LI><A href="#o1" style="text-decoration: none">1. Background</A></LI><LI><A href="#o2" style="text-decoration: none">2. Nickel sensing </A></LI><LI class="caption"><A href="#o2_1" style="text-decoration: none; color: #777777;">1. HpNikR: a pleiotropic regulator</A></LI><LI class="caption"><A href="#o2_2" style="text-decoration: none; color: #777777;">2. HpNikR-controlled promoters</A></LI></UL></DIV><LI><A href="#o3" style="text-decoration: none">3. Storage system</A></LI><LI class="caption"><A href="#o3_1" style="text-decoration: none; color: #777777;">1. Hpn: a storage protein in H. pylori</A></LI></DIV><LI><A href="#o4" style="text-decoration: none">4. Nickel-dependant Quorum Sensing Network</A></LI><LI><A href="#o5" style="text-decoration: none">5. General working scheme</A></LI><LI><A href="#o6" style="text-decoration: none">6. Bibliography</A></LI></DIV><DIV class="split2_3r"><H2 style="text-align:center">Project overview</H2><SECTION id="intro"><H3>1 - Background</H3><P>
Nickel is a very widespread heavy metal and its levels in waters and other environmental matrixes are highly related to human activities. A lot of industrial applications leads to an increase of nickel amounts in the environment, and the growing nickel contamination led to an increase in Nickel related allergies (more than the 15% of the population suffers from nickel allergy) and other pathologies concerning the reproductive, cardiac and respiratory systems. Furthermore, Nickel compounds have been classified as <A href="http://monographs.iarc.fr/ENG/Monographs/vol100C/mono100C-10.pdf" style="color: #8ac101">carcinogenic</A> by the International Agency for Research on Cancer, IARC.The risks for human health made it necessary, as for many heavy metals, to remove this metal from the environment. Nowadays nickel can be chemically removed from fluids using a chelating agent, known as H2DMG (dimethylglyoxime),which helps the precipitation of the nickel metal ions for its dosing and removal.We thought of a living organism carrying a nickel sensing device, coupled via <I>Quorum Sensing</I> to a second system (i.e. another bacterial population) which will remove the nickel from the environment. Our two-population system could be implemented in a water purification plant, allowing a regulated heavy metal removal without the need of chemicals (see the <A href="https://2013.igem.org/Team:UniSalento_Lecce/Applications" style="color: #8ac101">Applications</A> page).
</P></SECTION><SECTION id="intro"><H3>2 - Nickel sensing</H3><P>
The sensing device is based on a set of genetic parts from the pathogen <I>Helicobacter pylori</I>. This microorganism, which colonises the stomach, requires a large amount of nickel to maintain its homeostasis in such a hard environment. Particularly, Ni++ ions are essential for Urease activity, the key enzyme for <I>H.pylori</I> survival in the acidic environment of the stomach, as well as for a respiratory hydrogenase. Urease makes up to 10% of the total cell protein synthesis, so that nickel homeostasis becomes an essential step for the pathogen survival. 
</P></SECTION><SECTION id="intro"><H3>2.1 - HpNikR: a pleiotropic regulator</H3><P><I>Helicobacter</I> controls this homeostasis through a nickel regulator, <B>HpNikR</B>, a homologue of NikRs from different bacterial species (including <I>E.coli</I>). However, in <I>H.pylori</I> NikR acts as a pleiotropic regulator, being a rare case of master switching regulator in bacteria. Its strong reliance on nickel concentrations and its multiple responsive elements, both positively or negatively controlled, make us think of developing a synthetic regulon for nickel sensing.</P>
We cloned HpNikR from <I>H.pylori</I> strain G27 from the plasmid pET-NikR, sent us by prof. Alberto Danielli (from University of Bologna), the gene was cloned using the following primers (including Biobrick Prefix and Suffix, lowercase):<P style="text-align:center">
nikrFor: gtttcttcgaattcgcggccgcttctag<B>ATGGATACACCCAATAAAGACG</B>
nikrRev: gtttcttcctgcagcggccgctactagtattatta<B>CTATTCATTGTGTTCAAAG</B></P><P>(Here the <A href="https://2013.igem.org/Team:UniSalento_Lecce/Data" style="color: #8ac101">PCR electrophoretic profile</A>). Thus we obtained <A href="http://parts.igem.org/Part:BBa_K1151000" style="color: #8ac101">BBa_K1151000</A>, and its derivative coding device, pLac-regulated <A href="http://parts.igem.org/Part:BBa_K1151006" style="color: #8ac101">BBa_K1151006</A>.
</P></SECTION><SECTION id="intro"><H3>2.2 - HpNikR-controlled promoters</H3><P>
HpNikR, as said above, has multiple target promoters, some of them being upregulated, some being downregulated, in response to nickel concentration in the environment.
</P></SECTION><SECTION id="intro"><H3>2.2.1 - Negative promoters</H3><P>
As shown above, nickel-bound NikR can downregulate the expression of various targets. We obtained a genomic region from <I>H.pylori</I> G27 containing a divergent operon with two NikR-regulated promoters. The plasmid was sent us again by prof. Alberto Danielli as pNKTB. The promoters control the expression of the exbBD operon (for siderophore uptake, called Pexb) and for nikR expression itself (PnikR). According to the functional segmentation of the divergent operon (as in Delany et al., see bibliography below), we tried to separate the promoters, using the following primers (including Biobrick Prefix and Suffix, lowercase):</P><P style="text-align:center">
IntergenicaFor: gtttcttcgaattcgcggccgcttctagag<B>TGAGAAAAATCCTTTTTTG</B>
pnikrev: gtttcttcctgcagcggccgctactagta<B>TGAGAAAAATCCTTTTTTG</B>
pnikfor: gtttcttcgaattcgcggccgcttctagag<B>AATTCAAACGCTCTTATG</B>
pexbfor: gtttcttcgaattcgcggccgcttctagag<B>ACTGGATTTAAATGGTTG</B>
pexbrev:gtttcttcctgcagcggccgctactagta<B>GCACCCTATAAGAAGGCATC</B></P><P>
(Here the <A href="https://2013.igem.org/Team:UniSalento_Lecce/Data" style="color: #8ac101">PCR Results</A>) So we obtained the whole divergent operon  and Pexb and Pnikr. These promoters were cloned upstream of <A href="http://parts.igem.org/Part:BBa_E0240" style="color: #8ac101">BBa_E0240</A>, becoming parts <A href="http://parts.igem.org/Part:BBa_K1151009" style="color: #8ac101">BBa_K1151009</A>,<A href="http://parts.igem.org/Part:BBa_K1151010" style="color: #8ac101">BBa_K1151010</A>, <A href="http://parts.igem.org/Part:BBa_K1151011" style="color: #8ac101">BBa_K1151011</A>.Here you can see (link to experimental data) characterisation data for <A href="http://parts.igem.org/Part:BBa_K1151036" style="color: #8ac101">BBa_K1151036</A> and <A href="http://parts.igem.org/Part:BBa_K1151038" style="color: #8ac101">BBa_K1151038</A>: constructs made up by the promoters cloned upstream of GFP cds together with <A href="http://parts.igem.org/Part:BBa_K1151006" style="color: #8ac101">BBa_K1151006</A>, HpNikR coding device.
</P></SECTION><SECTION id="intro"><H3>2.2.2 - Positive promoter</H3><P>We also tried a positively-regulated promoter, from UreABCDE operon (PureA). We got it synthetised from IDT, obtaining <A href="http://parts.igem.org/Part:BBa_K1151005" style="color: #8ac101">BBa_K1151005</A>.We weren't able to characterise BBa_K1151005 activity.
</P></SECTION><SECTION id="intro"><H3>3 - Storage system</H3><P>
The second part of our project aimed to develop a bacterial population able to store nickel, basing its activity on the Hpn protein from <I>H. pylori</I> and a <I>Quorum Sensing</I> signalling: this last natural process is the basis of interaction between the two bacterial communities. The storage system of our design allows to accumulate the metal by bioconcentration: once accumulation occurred, it is necessary to dispose of the bacterial biomass containing the metal. See the <A href="https://2013.igem.org/Team:UniSalento_Lecce/Applications" style="color: #8ac101">Applications</A> page for a design of purification plant implementation.
</P></SECTION><SECTION id="intro"><H3>3.1 - Hpn protein</H3><P>
Hpn is an histidine-rich protein whose role is to store Nickel ions in the cytoplasm of <I>Helicobacter</I>. In conditions of Nickel starvation, this protein supplies the ions to the nickel-dependent machineries. We thought we can use this protein to make <I>E.coli</I> accumulate nickel. Hpn has been demonstrated not to be toxic even if highly induced in this heterologous system (Ge et al., see Bibliography). Hpn coding sequence has been synthetised by IDT, and submitted as <A href="http://parts.igem.org/Part:BBa_K1151001" style="color: #8ac101">BBa_K1151001</A>, and its derivative translational unit <A href="http://parts.igem.org/Part:BBa_K1151004" style="color: #8ac101">BBa_K1151004</A>, generator <A href="http://parts.igem.org/Part:BBa_K1151025" style="color: #8ac101">BBa_K1151025</A> and coding device <A href="http://parts.igem.org/Part:BBa_K1151013" style="color: #8ac101">BBa_K1151013</A>. We weren't able to characterise these parts activities.
</P></SECTION><SECTION id="intro"><H3>4 - Nickel-dependent Quorum Sensing Network</H3><P>
At first, our project was designed to work on a single plasmid, for both the sensing and metal-accumulation functions. To increase the safety of our device, giving more control to the human operator, we chose to divide the two phases of the project into two different bacterial populations. The activation of the storage depends on the activation of the sensing population. So, in order to integrate the two systems,  we exploited the mechanism of intercellular communication of Quorum Sensing. Evaluating the effectiveness of operation of various QS BioBricks present in the Registry, we chose <A href="http://parts.igem.org/Part:BBa_K805016" style="color: #8ac101">BBa_K805016</A>, <A href="http://parts.igem.org/Part:BBa_C0261" style="color: #8ac101">BBa_C0261</A> and <A href="http://parts.igem.org/Part:BBa_R0062" style="color: #8ac101">BBa_R0062</A>, from <I>Alivibrio fischerii</I>, based and dependent on the synthesis of AHLs as signaling molecules.In our system, the communication start is Ni2+ and IPTG dependent, both necessary for the synthesis of <A href="http://parts.igem.org/Part:BBa_K1151000" style="color: #8ac101">HpNikR</A>, the genic master regulator which activates the translation of LuxI, the QS inducer. The LuxI protein catalyzes the synthesis of <I>[N-]acyl-homoserine lactones</I> (AHLs), the signalling molecule, from <I>S-adenosylmethionine</I> (SAM) in the sensing population. Quorum sensing molecules reach the other bacterial population by diffusion.Here we built a system designed on the principle of positive feedback. In fact, the storage population integrates the gene LuxR within the plasmid we designed: LuxR codes for the receiver protein. Placing the transcription of the LuxI gene under the control of the promoter LuxPR, we obtained an AHL signal amplification phenomenon.
</P><P>
Under the same promoter and within the same synthetic operon we also placed the gene of the nickel-binding protein, <A href="http://parts.igem.org/Part:BBa_K1151001" style="color: #8ac101">Hpn</A>, thereby  obtaining a triple control of the protein synthesis process in the accumulator population, that depends upon the concentrations of Ni2+, IPTG and AHLs.
</P></SECTION><SECTION id="intro"><H3>5 - General working scheme</H3><P>Click on the image to zoom!</P></SECTION><SECTION id="intro"><H3>6 - Bibliography</H3><P><UL><LI><B>Abraham et al</B>, TheÂ metal- andÂ DNA-bindingÂ activitiesÂ ofÂ HelicobacterÂ pyloriÂ NikR. J Inorg Biochem.Â 2006; 100(5-6): 1005-14.</LI><LI><B>Benanti et al</B>, Helicobacter pylori NikR Protein Exhibits Distinct Conformations When Bound to Different Promoters. J Biol Chem. 2011; 18;286: 15728â15737.</LI><LI><B>Contreras et al</B>, Characterization of the roles of NikR, a nickel-responsive pleiotropic autoregulator of Helicobacter pylori; Molecular Microbiology, 49 (4) (2003), 947â963. </LI><LI><B>Delany et al</B>, In vitro analysis of protein-operator interactions of the NikR and Fur Metal-Responsive Regulators of coregulated genes in Helicobacter pylori. J Bacteriol.Â 2005 Nov;187(22):7703-15.</LI><LI><B>DanielliÂ et al</B>, BuiltÂ shallowÂ toÂ maintainÂ homeostasisÂ andÂ persistentÂ infection:Â insightÂ into theÂ transcriptionalÂ regulatory networkÂ of theÂ gastricÂ humanÂ pathogenÂ HelicobacterÂ pylori. PLoS Pathog.Â 2010, 10; 6(6):e1000938.</LI><LI><B>Danielli andÂ Scarlato</B>, RegulatoryÂ circuits inÂ HelicobacterÂ pylori: networkÂ motifsÂ andÂ regulators involvedÂ in metal-dependent responses. FEMS Microbiol Rev, 2010; 34(5):738-52.</LI><LI><B>Denkhaus and Salnikow</B>, Nickel essentiality, toxicity and carcinogenity; Critical Reviews in Oncology/Hematology, 42 (2002) 35â56.</LI><LI><B>Dian et al</B>, Structural Basis of the Nickel Response in Helicobacter pylori: Crystal Structures of HpNikR in Apo and Nickel-bound States. J Mol Biol.Â 2006; 25;361(4):715-30.</LI><LI><B>Dosanjh et al</B>, Helicobacter pylori NikRâs Interaction with DNA: A Two-Tiered Mode of Recognition. Biochem. 2009, 48, 527â536.</LI><LI><B>Dosanjh andÂ  Michel</B>, MicrobialÂ nickelÂ metalloregulation:Â NikRsÂ forÂ nickelÂ ions. Curr Opin Chem Biol.Â 2006 (2):123-30.</LI><LI><B>Ernst et al</B>, The nickel-responsive regulator NikR controls activation and repression of gene transcription in Helicobacter pylori. Infect Immun. 2005; 73(11):7252-8.</LI><LI><B>Gadd</B>, Metals, minerals and microbes: geomicrobiology and bioremediation; Microbiology, 156 (2010), 609â643.
</LI><LI><B>Ge et al</B>, Expression and characterization of a histidine-rich protein, Hpn: potential for Ni2+ storage in Helicobacter pylori; Biochemistry Journal 393 (2006), 285â293.</LI><LI><B>Ge et al</B>, ThermodynamicÂ andÂ kineticÂ aspectsÂ ofÂ metalÂ bindingÂ to theÂ histidine-richÂ protein,Â Hpn. J Am Chem Soc.Â 2006; 128(35): 11330-1.</LI><LI><B>Gilbert et al</B>, Protein Hpn: cloning and characterization of a Histidine-Rich Metal-Binding Polypeptide in Helicobacter pylori and Helicobacter mustelae. Infect Immun.Â 1995 Jul; 63(7):2682-8.</LI><LI><B>Muller et al</B>, Hierarchical regulation of the NikR-mediated nickel response in Helicobacter pylori. Nucleic Acids Res. 2011; 39(17):7564-75.</LI><LI><B>Mutsunori et al</B>, Activation of Helicobacter pylori ureA promoter by a hybrid Escherichia coliâH. pylori rpoD gene in E. coli. Gene 239 (1999) 351â359.</LI><LI><I>âNickel, Elementalâ</I> on <B>TOXNET- Databases on toxicology, hazardous chemicals, environmental health, and toxic releases.</B></LI><LI><B>Seshadri et al</B>, Roles of His-Rich Hpn and Hpn-Like Proteins in Helicobacter pylori Nickel Physiology. J Bacteriol.Â 2007 Jun; 189 (11): 4120-6.</LI><LI><B>van Vliet et al</B>, Nickel-responsiveÂ inductionÂ ofÂ ureaseÂ expressionÂ inÂ Helicobacter pyloriÂ isÂ mediatedÂ at theÂ transcriptionalÂ level. Infect Immun.Â 2001 Aug;69(8):4891-7.</LI><LI><B>Zambelli et al</B>, High affinityÂ Ni2+Â bindingÂ selectivelyÂ promotesÂ bindingÂ ofÂ HelicobacterÂ pyloriÂ NikRÂ to itsÂ targetÂ ureaseÂ promoter. J Mol Biol.Â 2008 Nov 28;383(5):1129-43.</LI></UL></P></SECTION></DIV></DIV><FOOTER><DIV id="main_wrap"><DIV class="split2_3r" style="width:50%;text-align:right;padding-right:50px"><H2 style="margin-right:70px;color:#cccccc">Sponsors</H2></DIV><DIV class="split2l" style="width:30%;text-align:left;margin-left:50px"><H2 style="margin-left:40px;color:#cccccc">Contacts</H2><TABLE style="float:left;background-color:transparent"><TBODY><TR><TD><FONT style="font-size:15px;color:#cccccc">Facebook</FONT></TD></TR><TR><TD><FONT style="font-size:15px;color:#cccccc">Twitter</FONT></TD></TR><TR><TD><FONT style="font-size:15px;color:#cccccc">Google+</FONT></TD></TR></TBODY></TABLE><P style="margin-top:20px"><FONT style="font-size:15px">
					E-mail: 
				</FONT><A href="mailto:igem.unisalento@gmail.com" style="color:#cccccc;background:none;font-size: 15px">
					igem.unisalento@gmail.com
				</A></P></DIV><DIV class="nosplit"><P style="text-align:center;color:#cccccc">Design by Mattia Corvaglia - <A href="http://www.mattiacorvaglia.com" style="color:#cccccc">www.mattiacorvaglia.com</A> - Visitors: 
				</P></DIV></DIV></FOOTER><P id="back-top"><A href="#top">Back to Top</A></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:UniSalento_Lecce/Overview">http://2013.igem.org/Team:UniSalento_Lecce/Overview</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UniSalento_Lecce/Overview" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UniSalento_Lecce/Overview" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UniSalento_Lecce/Overview&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UniSalento_Lecce/Overview&amp;oldid=294429" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>