<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Northwestern_experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Northwestern/experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Northwestern Template</TITLE><TITLE>Northwestern Template </TITLE></P><DIV class="nav_container"><DIV class="dropdown"><BUTTON class="dropbtn">Human Practices</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Northwestern/humanpractices">Overview</A><A href="https://2017.igem.org/Team:Northwestern/HP/Silver">Silver</A><A href="https://2017.igem.org/Team:Northwestern/HP/Gold_Integrated">Gold</A><A href="https://2017.igem.org/Team:Northwestern/Collaborations">Collaborations</A><A href="https://2017.igem.org/Team:Northwestern/Engagement">Engagement </A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Parts</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Northwestern/Parts">Overview</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Lab</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Northwestern/experiments">Experiments </A><A href="https://2017.igem.org/Team:Northwestern/Model">Modeling</A><A href="https://2017.igem.org/Team:Northwestern/Results">Results</A><A href="https://2017.igem.org/Team:Northwestern/Lab">Notebook</A><A href="https://2017.igem.org/Team:Northwestern/Safety"> Safety </A><A href="https://2017.igem.org/Team:Northwestern/InterLab"> Interlab </A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Project</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Northwestern/project">Description</A><A href="https://2017.igem.org/Team:Northwestern/Design">Design </A><A href="https://2017.igem.org/Team:Northwestern/Medals">Medal Criteria</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Team</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Northwestern/Team#"> About Us </A><A href="https://2017.igem.org/Team:Northwestern/Attributions">Attributions </A></DIV></DIV><A href="https://2017.igem.org/Team:Northwestern"><DIV class="dropdown"><BUTTON class="dropbtn">Home</BUTTON></DIV></A></DIV><P><TITLE>Home</TITLE>
button{border: 0; background-color: #D3D3D3
}
</P><DIV><H3><CENTER>Packaging Cas9</CENTER></H3><P style="padding-top:2%; padding-right: 15%; padding-left:15%; font-size:14px;" class="big">In order to successfully package Cas9 into Outer Membrane Vesicles, we needed a source of OMVs.  We transformed our saCas9 encoding plasmids into a hyper-vesiculating strain of E. coli known as JC8031.  This strain is genetically engineered to create a large amount of OMVs and as a result, any substance whose concentration is high in the periplasm of the cell would also be expected to exist within the generated OMVs. The Cas9 itself also has a His6 tag attached for identification purposes, as well as a signal sequence meant to direct and secrete the protein into the periplasm.</P><H3><CENTER>Signaling Sequences</CENTER></H3><P style="padding-top:2%; padding-right: 15%; padding-left:15%; font-size:14px;" class="big">The following table includes the signalling sequences that we used and tested over the course of the summer.  They are divided into the secretion pathway taken by the sequences, where an ambiguous sequence simply has a less defined pathway or may use multiple pathways.</P><P style="padding-top:2%; padding-right: 15%; padding-left:15%; font-size:14px;" class="big">In order to determine which secretion tags would be of most use to us, and because the tag is expected to detach from the Cas9 protein after successful delivery to the periplasm, we used the Signal IP server to identify the location of the cleavage sites.  This analysis gave us information regarding where the cleavage of the peptide from the protein would most likely occur and the likelihood of this event occurring.  Below is an example of a Signal IP for PelB, one of the tags that clearly surpassed the threshold.
</P><P style="padding-top:2%; padding-right: 15%; padding-left:15%; font-size:14px;" class="big">In addition to the signalling sequences, we also fused the Cas9 protein to ClyA, a pore-forming toxin, which was expected to bore through the inner membrane of the cell and lodge itself in the outer membrane, eventually having an OMV for around it.  This way the Cas9 would follow the ClyA into the OMVs.
</P><H3><CENTER>Types of Cas9 Protein</CENTER></H3><P style="padding-top:2%; padding-right: 15%; padding-left:15%; font-size:14px;" class="big">
Two different types of Cas9 were characterized by our team over the summer.  The saCas9 already existed in the iGEM registry having been used by previous teams.  However, in order to explore more options and find a more compatible Cas9, we also used cjCas9 protein.  This Cas9 has two main benefits to our project.  The first is that the size of cjCas9 is comparatively much smaller than the saCas9, making it more likely that this protein will fit inside the JC8031 OMVs.  Secondly, the cjCas9 has been demonstrated to have much less off-target effects and thereby target the gene of interest with much more efficiency than saCas9.  These two differences make cjCas9 the apparent optimal protein for our project, but further comparative testing of these two proteins give more information of their individual benefits.</P><H3><CENTER>Cas9 Functionality Test</CENTER></H3><P style="padding-top:2%; padding-right: 15%; padding-left:15%; font-size:14px;" class="big">
One important experiment performed during the course of this project was a Cas9 functionality test to prove that our cas9 was functionally operational inside the JC8031 cells.  The protocol of this experiment involved four separate transformations into JC8031 using three different plasmids:
<STRONG>1.saCas9 and mRFP-gRNA</STRONG>This was a double transformation of one plasmid encoding the Cas9 protein and one plasmid that encoded for both mRFP and a guide RNA for the Cas9.  The gRNA was designed to target the mRFP, and therefore was expected to result in the cutting of the mRFP sequence.
<STRONG>2.mRFP-gRNA ONLY</STRONG>This was our negative control, since without any Cas9 to cut the mRFP, the result should be fluorescing cells.
<STRONG>3.saCas9 ONLY</STRONG>This was our positive control, since there should be no fluorescing cells without any mRFP plasmid.
<STRONG>4.saCas9 and mRFP-gRNA (template)</STRONG>This was also a control where the guide RNA on the second plasmid was simply a template sequence that was NOT designed to target the mRFP sequence.  As a result, the Cas9 should not be able to cut the mRFP.

The expected results of this experiment would be that we receive fluorescent colonies for samples 2 and 4, since they do not have any functional combination of gRNA and Cas9, while we see non-fluorescent colonies for samples 1 and 3 because they lack an intact mRFP sequence.<STRONG> For further information, see the <A href="https://2017.igem.org/Team:Northwestern/Results">Results Page</A>.</STRONG></P><H3><CENTER>Fractionation and Western Blots</CENTER></H3><P style="padding-top:2%; padding-right: 15%; padding-left:15%; font-size:14px;" class="big">
The following is a summary of the protocol developed with the purpose of testing for the existence of Cas9 in the periplasm of the JC8031 cells:
-Lyse outer membrane with lysozyme and retrieve periplasmic fraction.
-Lyse remaining spheroplasts via boiling and retrieve the cytoplasmic fraction.
-Run a western blot with both fractions including the following two controls:
-Test for presence of GroEL (with GroEL antibody).
-Test for presence of MalE (with MBP antibody).

The purpose of the last two controls is to make sure that the two fractions retrieved after lysing the cells are, for the most part, mutually exclusive.  GroEL is a protein commonly found in the cytoplasm of these cells, while MalE is typically mostly found in the periplasm.  Therefore, if we detect any MalE or GroEL concentration in an unexpected fraction, there was most likely some leakage that occurred during the experiment.<STRONG> For further information, see the <A href="https://2017.igem.org/Team:Northwestern/Results">Results Page</A>.</STRONG></P><CENTER><H3>References</H3></CENTER><P style="padding-top:2%; padding-right: 15%; padding-left:15%; padding-bottom:5%; font-size:14px;" class="big"><B>1.</B> Kim, E. et al. In vivo genome editing with a small Cas9 orthologue derived from Campylobacter jejuni. Nat. Commun. 8, 14500 doi: 10.1038/ncomms14500 (2017).

</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>