"
Page
Discussion
View source
History
teams
Log in
Â 
Team:MoWestern Davidson/notebooks
From 2009.igem.org
Home
Team
Project Overview
Project Design
Wet Lab
Mathematical Models
Physical Models
Parts
Notebooks
Human Practice
Outcomes
Acknowledgements
Contents
1 Safety
2 Week 1 (May 25-May 29)
3 Week 2 (June 1-June 5)
4 Week 3 (June 8-June 12)
5 Week 4 (June 15- June 19)
6 Week 5 (June 22-June 26)
7 Week 6 (June 29- July 3)
8 Week 7 (July 6-July 10)
9 Week 8 (July 13-July 17)
10 Week 9 (July 20-July 24)
11 Week 10 (July 27-July 31)
12 Fall Semester
Safety
Would any of your project ideas raise safety issues in terms of researcher safety, public safety, or environmental safety?
Our project raises issues of researcher safety with regard to exposure to biohazardous material (ethidium bromide, acrylamide) and ultraviolet radiation.  It also raises environmental safety issues with the disposal of recombinant bacteria and biohazardous material. The parts and devices we made pose no hazards.
Is there a local biosafety group, committee, or review board at your institution?
The Biology Departments at Missouri Western and Davidson College have policies concerning general laboratory safety, treatment and disposal of biohazardous materials, protection of laboratory personnel, and safety training of student laboratory researchers. We followed all local guidelines and regulations.
What does your local biosafety group think about your project?
The Departments at each of our institutions are satisfied that the iGEM project was conducted in accordance with the established policies regarding laboratory safety.
Do any of the new BioBrick parts that you made this year raise any safety issues?
None of our BioBrick parts raises safety issues beyond those associated with reagents used to manipulate DNA and working with a laboratory strain of E. coli.
Week 1 (May 25-May 29)
The Missouri Western students came to Davidson to finalize the research agenda for the summer. Both the biology and math aspects of the project were discussed in detail. By the end of the week, the team had unified around applications of the Satisfiability (SAT) problem to the genome of E.coli via the use of frameshift suppressor tRNAs.
We needed to design and engineer a reporter gene that the tRNA would suppress. Therefore, our research would consist of 2 main tracks: tRNA project and the 5mer Reporter Project.
Week 2 (June 1-June 5)
tRNA Project
We began to plan the construction of a 1-SAT model that would include a modified reporter gene suppressed by its respective tRNA.
The two campuses decided to split up the list of tRNAs according to percent suppression. Missouri Western took CUACC, AGGAC, CCAAU, CUAGC, CUACU, and CCACC. Davidson took CUAGU, CCACU, CGGUC, CCCUC, CCAUC-9 and CCAUC-10. We requested the sequence for our tRNAs from Dr. J. Christopher Anderson. After receiving them, we planned their construction via oligo assembly.
By the end of the week, we had planned to use 4 total oligos for our tRNAs.
5mer Reporter Project
In order to minimize major changes made to the protein structures and gene, we decided to place our 5 base pair addition upstream of our reporter protein. We accomplished this using PCR. We manipulated the PCR primers by designing our forward primer to include ATG, our 5mer, and the first half of the reporter protein up to a restriction site of our choice.
We wanted 1 fluorescence gene and 1 antibiotic resistance gene on each campus. After careful consideration of which reporters had the most "hardy" restriction sites, we decided upon RFP and tetracycline resistance protein on the Davidson campus; and GFP and chloramphenicol resistance on the Missouri campus.
Week 3 (June 8-June 12)
tRNA Project
After editing the first draft of 4 oligos, we devised a tRNA constructed of  7 oligos of shorter lengths to reduce the chance of mutations. We assembled our tRNA from these oligos and ligated/transformed the assembly. By the end of the week, we had obtained colonies from these transformations.
5mer Reporter Project
We chose NcoI as the restriction site in the RFP and BamHI as the site in Tet Resistance. We ordered the oligos of our primers. The forward primer was different for each of the 5 different 5mers on Davidson's campus and 6 different 5mer's on Missouri's campus. We only needed one reverse primer for each reporter gene we used. We carried our PCR using wild type RFP and Tet plasmids as our respective reporter templates.
We only used an aliquot amount to run a gel and verify that the sizes of the PCR fragments included the beginning of the reporter (with ATG and our 5mer) to the restriction site we chose.
Week 4 (June 15- June 19)
tRNA Project
We found that the variable to negative ratio of tRNA colonies was about 2 to 1. We saved liquid cultures of these and we re-ligated and re-transformed the tRNAs again.
Later that week, each lab member "adopted" a specific 5mer and tRNA for the rest of the summer. After verifying and plating tRNAs of correct insert size, we prepared samples of some tRNAs for sequencing.
5mer Reporter Project
We ligated cleaned and digested PCR inserts into vectors and transformed them. The vectors had been digested with EcoRI and either BamHI (Tet) or NcoI (RFP). Unfortunately, no colonies appeared on plates the next day. We repeated PCR to construct the first half of the reporter. We ended the week by verifying the insert sizes and cleaning the insert.
Week 5 (June 22-June 26)
tRNA Project
Sequencing for the tRNAs initially gave us 2 100% matches. The rest of the tRNAs had at least 1 ambigious nucleotide. We  decided to resequence these tRNAs. We also decided that we would use pBad as the promoter for these tRNAs. We decided that to use the pBad promoter for the tRNAs because it maintains low expression of the tRNA in cells, which is best for the them. We also entered our tRNAs into the parts registry. Suppressor tRNA Parts
5mer Reporter Project
We re-verified our PCR inserts and after cleaning/digesting them, re-ligated and re-transformed them. We obtained colonies this second time. After PCR screening the colonies, we chose samples that looked the right size for sequencing.  At the end of the week, we entered our 5mer reporters into the parts registry. 5mer Reporter Parts
Week 6 (June 29- July 3)
tRNA Project
We digested the tRNAs with XbaI and PstI to isolate the insert. We gel purified the insert, and ligated it to pBad vector. By the end of the week, we PCR screened the colonies from the ligations.
5mer Reporter Project
The sequencing that returned from our 5mer Reporter was quite messy. We decided to use a different mini prep kit (switched from Zyppy to Promega) to re-sequence our 5mers.
Week 7 (July 6-July 10)
tRNA Project
After mini prepping viable clones of the pBad-tRNA ligation, we decided that we wanted to see how the cells would function with more than one pBad-tRNA gene. During the week, digested pBad-tRNA plasmid with XbaI/PstI and SpeI/PstI to construct an insert and vector respectively.
5mer Reporter Project
We sent our 5mer Reporters for sequencing in glycerol stocks. Every member of our lab sent the different clones of their Tet and RFP 5mer reporters.
Week 8 (July 13-July 17)
tRNA Project
We entered pBad-tRNA into the registry and made their glycerol stocks for our freezer.
Over the past week we noted that the sizes of our plasmids appeared much larger than they actually were. Part of this incongruity resulted from the use of a different molecular weight marker with our gels. After sequencing our double pBad-tRNA ligation, however, we discovered a 32 bp insertion after the promoter. This insertion was palidromic and contained the NcoI restriction site. Further digestion with EcoRI and NcoI yielded the inserts expected from this mutation.
We traced the source of this mutation back to the pBad in our freezer stocks. We had to discard all minipreps and cells containing this promoter and start afresh.
5mer Reporter Project
We decided to test between pLac and pTet as the promoter for translation of our reporter. We digested and gel purified pLac, pTet, and RBS. Then, we ligated pLac to RBS and pTet to RBS. pLac+RBS produced colonies from the first ligation, but pTet+RBS produced colonies on the second ligation. These colonies were PCR screened and we found 3 pLac+RBS colonies to miniprep and digest for insert check.
We also recieved confirmation of sequencing for our 5mer reporters this week. We had 5 100% matches for RFP and 3 100% matches for Tet. However, all of the RFP sequences had a 25 bp insertion between the stop codon and the Bio Brick suffix. Because this insertion is not within the gene, we will continue to use this clone.
Week 9 (July 20-July 24)
5mer Reporter Project
We cut one of our good Tet clones and one good RFP clones with XbaI and PstI to make insert that will go downstream of the promoter and RBS. After we ran this on a gel, we gel purified the inserts.
We also digested pLac+RBS (with EcoRI and PstI) and ran our viable clones on a gel to check the insert size of the colonies from ligation. The insert looked about 50bp longer than we predicted, so we miniprepped the clones and decided to send it off for sequencing.
We decided to make vectors out of the pLac-RBS just in case the sequences yield 100% matches. We also went ahead and ligated this promoter and RBS to every Tet-5mer and RFP-5mer we had. When we PCR screened colonies resulting from the transformation, we discovered we had some successful ligations. We sequenced some of the clones resulting for each promoter.
Additionally, we ligated and transformed pTet with RBS. We PCR screened almost all the colonies that grew from this ligation and grew of up cultures of viable pTet+RBS. After mini prepping these contructs, we digested them to check insert size. The sizes of the inserts seemed larger than we had anticipated so we sent off seven clones for sequencing.
Assuming that we could still proceed with the ligation of our tRNA to the reporters, we digested the Tet-5mer and RFP-5mers with XbaI and PstI to make an insert. We will ligate this insert downstream of the pBad-tRNA.
tRNA Project
We obtained colonies from our ligations of pBad-tRNAs and PCR screened the colonies. We discovered that there were several successful ligations. We are sending some clones for sequencing and will also digest them with SpeI and PstI so we can ligate our reporter downstream of this gene.
Week 10 (July 27-July 31)
Davidson team traveled to St. Joseph, MO to summarize our findings, create the wiki and decide what we need to do next.
While we were in Missouri, the team standardized the names and descriptions for our tRNAs and 5mers. We also finalized and entered all clones into Registry at MIT and the GCAT-alog of frozen stocks .
In addition, we came up with our theme, and wiki design.
We also brainstormed some designs for the final construct. Some of the ideas were to use one promoter (pLac) followed by RBS and the reporter and then the tRNA. Another idea, which we thought was better, was to contruct pBad-tRNA upstream of pLac-RBS-reporter (or downstream of the reporter). We did not want to use pLac as the promoter for tRNA because we assumed that being a strong promoter, pLac may cause fatal overexpression of the tRNA. Instead, we used pBad as the promoter.
This same week in Missouri, our team also received sequencing back of the promoter-RBS and of the complete constructs from the previous week. We discovered that what we had assumed to be the pLac-RBS, was actually pLac-pTet. Somewhere along the way, a mistake was made in the ligation. We realized this was the reason we were not getting expression of our reporter.
On the other hand, most of the pBad-tRNAs constructs were correct.
One of our members, Olivia, was working one more week by herself and she completed the final constructs for the summer.
We had to make a poster for the poster session on campus in early September.
Several members on both campuses decided to continue trying to build the full constructs during the semester. Their findings are described below.
Fall Semester
This Semester's work:
1) We started with the pBad Promoter
Initially started with weaker promoter because we did not want to flood cells with suppressor tRNAs.
Concerned about lethality and binding of suppressor tRNAs to non-target mRNAs.
2) pBad results
1 out of 5 clones gave visible fluorescence
A second clone produced detectable fluorescence by fluorometer
3) We switched to a stronger promoter: pTet
We hypothesized that stronger pTet promoter would increase suppression of 5mer, allowing increased transcription of RFP. This has been supported with our data.
We will compare successful frame shift suppression with pBad promoter and pTet promoter driving tRNA production. pTet appears better than pBad.
4) Testing of second generation constructs
Is there a difference in frame shift suppression if the promoter+tRNA construct is upstream or downstream of the 5mer-RFP reporter? Yes, promoter and tRNA downstream gives stronger suppression, probably because we do not have a TT after the RFP and so the tRNA can be transcribed from both promoters.
Compare pTet and pBad promoters for frame shift suppressor phenotypes. The pTet promoter gives a better suppression phenotype.
Retrieved from " http://2009.igem.org/Team:MoWestern_Davidson/notebooks "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
