"
Page
Discussion
View source
History
teams
Log in
 
Team:MoWestern Davidson/project physicalmodel
From 2009.igem.org
Home
Team
Project Overview
Project Design
Wet Lab
Mathematical Models
Physical Models
Parts
Notebooks
Human Practice
Outcomes
Acknowledgements
Contents
1 Frameshift Suppressor tRNA Structure
1.1 How Nucleotides Bond
1.2 The Cloverleaf Model
1.3 The Spatial tRNA Structural Motif
2 Using the PDB Format to Design a Frameshift Suppressor tRNA
2.1 The PDB Format Defined
2.2 Modifying a PDB File
3 Building a Physical Model
3.1 Preparing a Model for Production
3.2 Model Production
3.2.1 Rapid Prototyping
3.2.2 The Milwaukee School of Engineering
Frameshift Suppressor tRNA Structure
All tRNAs, whether they contain a 3- or 5-base anticodon, conform to a general structure. They feature a series of loops and stems folded into a tertiary L-shape, with the amino acid acceptor arm opposite the anticodon. The structure of tRNA molecules is important because their unique structural motifs are recognized by the ribosome and enzymatic pathways, and allow tRNAs to carry out their life-sustaining function.
How Nucleotides Bond
Two nucleotides can interact in two ways: by covalent bond or hydrogen bond, the latter being about one-twentieth the strength of the former. Nucleotides are linked covalently by a phosphodiester bond between the oxygen off of the 3′ carbon of the ribose sugar of one nucleotide and the oxygen off of the 5′ carbon of another. This creates a backbone consisting of alternating sugars and phosphates. If positioned in the correct alignment, two nucleotides may hydrogen bond. This allows a strand of RNA to base pair with itself and assume the shape of a tRNA. tRNA molecules often exhibit non Watson and Crick base pairing both within the molecule and between the mRNA codon and tRNA anticodon. This "wobble" phenomenon allows cells to synthesize fewer tRNAs than the 64 (4³) that would be required if each tRNA recognized only one codon. This "wobble" sometimes involves modified bases. For instance inosine, a post-transcriptional modification of adenine, base pairs with adenine, cytosine, and uracil. This explains the genetic code's tendency to allow the 3′ end base of mRNA transcripts to vary and still code for the same amino acid.
The cloverleaf model shows how the RNA strand base pairs with itself to form loops and stems, but does not accurately depict its structural motif.
The tertiary structure of a tRNA molecule shows how the stems and loops fold together.
The Cloverleaf Model
The structure of tRNA molecules is often represented by showing how stems and loops are formed by base pairing within a two-dimensional context—useful for depicting secondary structure, but unable to show three dimensional folding. Each tRNA molecule consists of five arms, and each, with the exception of the acceptor arm, contains both stems and loops. A stem is formed by self base pairing along the RNA strand, and loops are formed from non-paired nucleotides. The 3′ terminus of the TRNA strand protrudes beyond the 5′ terminus by several nucleotides, but the 5′ terminus is base paired. The appropriate amino acid is connected to the 3′ terminus by an amino acyl tRNA synthetase, an enzyme that uses the unique structural motif of each tRNA to consistently pair it with its designated amino acid. Moving from the 3′ terminus in the 5′ direction, the TΨC arm is encountered. It is so named because of the presence of the sequence 5′-TΨC-3′, where Ψ stands for the base pseudouracil, an isomer of uracil formed from post-transcriptional modification. Continuing in the 5′ direction, the variable loop is encountered. This portion of the tRNA family of molecules shows high size variation, from only a few nucleotides to well over a dozen. Next is the anticodon arm, which contains the anticodon itself. The anticodon recognizes its mRNA complement, and may contain a wobble base pairing at the 5′ end residue of the anticodon. This wobble base pairing often utilizes the abundant assortment of non-standard nucleotides. The final arm before the 5′ terminus is the D arm. It contains the D base, or dihydrouracil.
The Spatial tRNA Structural Motif
Base pairing between arms stabilizes the structure of the tRNA molecule into a L-shape. The acceptor arm is positioned at one and of the L, and the anticodon at the other. The size and sequence of each arm influences the spacial structure of the tRNA, and allows enzymes responsible for maintaining function to recognize each tRNA.
Using the PDB Format to Design a Frameshift Suppressor tRNA
The PDB format is a standard file type used in molecular modeling and known structures to code for the Cartesian space coordinate triple for each atom within the molecule. It also includes portions describing which atoms combine to form nucleotides or amino acids, the locations of hydrogen bonds, and other information pertaining to the structure of molecules. A PDB file coding for a theorized model of a 5-base anticodon was made by modifying the anticodon loop of the existing PDB file 1EHZ—yeast phenylalanine tRNA. This particular tRNA was one of the first to undergo extensive study, and remains a model molecule among the countless tRNA molecules within the diverse variety of life.
a sample of code used to modify 1EHZ
The PDB Format Defined
PDB is an acronym for Protein Data Bank. Although predominately a resource for protein files, it does include some nucleic acid files. Coordinate files following the PDB format are in plain text, and carry the ".pdb" extension. A PDB file itself contains a series of columns of data aligned to prescribed boundaries. A PDB file contains an expanse of information pertaining to the molecule in question. Among them are authorship of the file, method of data collection, primary sequence, secondary structural features, the location of hydrogen bonds and disulfide bridges, placement of hetero groups such as solvents, and most importantly the Cartesian space coordinate triples that code for individual atom placement. The PDB file 1EHZ is about ninety 8½″×11″ (A4) pages in length, while a file coding for a ribosome-tRNA-mRNA complex exceeds 2000.
Modifying a PDB File
In order to insert two nucleotides into a closed loop, it was necessary to move nucleotides adjacent to the insertion. Two tools in an Excel spreadsheet facilitated this operation: a translation tool and rotation tool. These allowed the movement of nucleotides while preserving the relative positions of their component atoms. The translation tool operates by selecting a particular atom and defining its desired final position. The distance traveled in each the x, y, and z directions is calculated, and is used to find the new locations of all other atoms to be moved. The rotation tool uses the cross product property of vectors to create 90° rotations. This same set of tools allowed for a reverse engineering approach to move a copy of a desired nucleotide within the molecule to a new location. Final adjustments were necessary to ensure that all atoms were bonding properly and that a chemically reasonable structure resulted.
Building a Physical Model
Above is the physical model produced—the yeast phenylalanine tRNA modified to have a 5-base anticodon. The anticodon (colored by base) has the sequence 5′-CUGAA-3′, and the 5′-CCA-3′ acceptor stem is colored yellow.
Preparing a Model for Production
There are a number of types of models available to suit whatever purpose the model is to serve. Space-filling models depict individual atoms as spheres with proper van der Waals radius that overlap to indicate bonding. Ball and stick models show atoms as spheres and bonds as sticks connecting them. Backbone models connect particular monomer atoms, such as the alpha carbons in proteins, to create a representative but abbreviated structure of the polymer. In addition to shape characteristics, it is possible to note particular features of the desired molecule such as hydrogen bonds and disulfide bridges. The use of color allows one to highlight particular features of a molecule, such as the anticodon of a tRNA. It also allows for the communication of which element a particular atom belongs to, usually by using the CPK coloring scheme in which hydrogen is white, oxygen is red, etc.
For the tRNA model we chose to use a CPK color scheme with the exception of two key features we wished to highlight: the acceptor stem and anticodon. The anticodon has a different color for each nucleotide, and the nucleotides of the acceptor stem share the same color.
Model Production
Rapid Prototyping
Rapid prototyping is the use of computer-guided machinery to construct a three dimensional structure. Methods vary from the use of laser-solidified photopolymers to 3D printing. 3D printing is commonly used to produce space-filling models, and works by depositing a small layer of plaster powder representing a cross section of the final model followed by a binding agent. The machine continues building layer by layer until it completes the job. The model is then sealed for durability.
The Milwaukee School of Engineering
Models used in this project were courteously produced by the Center for BioMolecular Modeling at the Milwaukee School of Engineering (MSOE) under the direction of Dr. Tim Herman. MSOE also provided educational materials and models for use prior to and during the modeling process. This addition to the project would not have been possible without their willing assistance.
Retrieved from " http://2009.igem.org/Team:MoWestern_Davidson/project_physicalmodel "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
