<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NTHU_Formosa_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NTHU Formosa/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="wrap"><HEADER><DIV class="inner relative"><NAV id="navigation"><UL id="main-menu"><LI class="current-menu-item"><A href="https://2018.igem.org/Team:NTHU_Formosa#0">HOME</A></LI><LI class="parent"><A href="https://2018.igem.org/Team:NTHU_Formosa#1">Project</A></LI><LI><A href="https://2018.igem.org/Team:NTHU_Formosa/Description"> Introduction</A></LI><LI><A href="https://2018.igem.org/Team:NTHU_Formosa/Design">  Design</A></LI><LI><A href="https://2018.igem.org/Team:NTHU_Formosa/Safety"> Biosafety</A></LI><LI><A class="parent" href="https://2018.igem.org/Team:NTHU_Formosa/Applied_Design"> Applied Design</A></LI></UL><LI class="parent"><A href="https://2018.igem.org/Team:NTHU_Formosa#2">Experiment</A></LI><LI><A href="https://2018.igem.org/Team:NTHU_Formosa/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:NTHU_Formosa/Parts">Parts &amp; Component</A></LI><LI><A href="https://2018.igem.org/Team:NTHU_Formosa/Composite_Part">Composite Part</A></LI><LI><A href="https://2018.igem.org/Team:NTHU_Formosa/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:NTHU_Formosa/InterLab">InterLab</A></LI><LI class="parent"><A href="https://2018.igem.org/Team:NTHU_Formosa/Model">Modeling</A></LI><LI class="parent"><A href="https://2018.igem.org/Team:NTHU_Formosa#4">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:NTHU_Formosa/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:NTHU_Formosa/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:NTHU_Formosa/Human_Practices">Integrated Human Practices</A></LI><LI class="parent"><A href="https://2018.igem.org/Team:NTHU_Formosa#5">About us</A></LI><LI><A href="https://2018.igem.org/Team:NTHU_Formosa/Team">Members</A></LI><LI><A href="https://2018.igem.org/Team:NTHU_Formosa/Sponsor">Sponsors</A></LI><LI><A href="https://2018.igem.org/Team:NTHU_Formosa/Attributions">Attributions</A></LI><LI><A href="https://igem.org/2018_Judging_Form?team=NTHU_Formosa">Judging Form</A></LI><LI><A href="https://2018.igem.org/Team:NTHU_Formosa/Medal">Medal</A></LI></NAV></DIV></HEADER></DIV><DIV class="w3-container w3-padding-64 w3-center"><DIV class="w3-content" style="font-family: " microsoft="" yahei=""><H2 class="w3-justify" style="font-size:60px;  padding-left: 385px"><B>Design</B></H2></DIV></DIV><P class="w3-center" style="font-family:Quicksand; font-size:30px;"><B>Autonomous bioluminescence output in mammalian cell system induced by extracellular soluble stimuli</B></P><DIV class="w3-container w3-padding-24 w3-center"><DIV class="w3-content"><P class="w3-justify" style="font-size:20px;">
 　 Claiming the versatility of such programmable signaling pathways inducible by extracellular soluble molecules following autonomous regulation of the downstream gene expression, we propose a customer-oriented application in diagnostic aspect with considerately
        high commercial potential. In the extracellular space such as bloodstream, various soluble proteins serve as biomarkers representing the physiological or pathological conditions of organisms (Altintas and Tothill, 2013; Rapisuwon et al., 2016).
        For the purpose of diagnosis, the blood test is one of the most common ways for detecting biomarkers but it suffers from several inevitable drawbacks such as invasiveness, time-consuming procedure, demand for medical staff service, non-real-time
        tracking and so on. These disadvantages may discourage people from the periodic medical checkup. In terms of non-invasive real-time health monitoring, great varieties of wearable devices and smartwatches, working as personal daily fitness trackers,
        have been successively developed. The global market of wearable medical devices is estimated to reach $35 billion USD soon. However, most of the current devices are only capable of providing limited diagnosis or information, such as heart rate,
        steps taken, calories, quality of sleep and a few other personal metrics, which cannot precisely represent the physiological states relating to innumerable life-threatening diseases.

         　 To address this long-standing worldwide issue, we developed engineered cells that are capable of receiving extracellular soluble biomarkers as stimuli and autonomously produce bioluminescence signal output for real-time and non-invasively diagnosis
        purpose based on our programmable signaling pathway. To our knowledge, this is the first programmable system for non-invasive real-time diagnosis capable of receiving the soluble biomarkers as input stimulation from extracellular space and triggering
        the autonomous bioluminescence output. Powerful and versatile, our design is anticipated to revolutionize the strategies of non-invasive real-time diagnosis. </P><P class="w3-justify" style="font-size:20px;"> 　 We begin by describing the design of our programmable signaling systems that can be triggered by both membrane-bound ligands and soluble ligands from extracellular space. In this system, nanobodies, single variable domain antibody fragments (VHH)
        derived from heavy-chain-only antibodies in Camelidae (Wang et al., 2016), is used to recognize wide varieties of ligands. Nanobodies against various antigens have been developed owing to its unique features--high affinity and specificity for
        antigen, thermostability, nanoscale size, and soluble characteristic in aqueous solution (Hu et al., 2017). According to the information from iCAN database (<A href="http://ican.ils.seu.edu.cn">Institute collection and Analysis of Nanobody</A>),
        over two thousand nanobodies are available for recognizing different antigens including molecular-bound molecules and soluble molecules. Aiming to recognize extracellular molecules, nanobodies are anchored on the cell surface by tagging them with
        transmembrane domain (TM) of plasma membrane proteins (Fig. 1). Based on previous studies that found antigens can serve as scaffolds to induce dimerization of split nanobodies (Tang et al., 2013, 2015), our programmable signaling is designed to
        be triggered by antigens/nonabodies binding. Therefore, nanobody is split into N-terminal fragment and C-terminal fragment and is tagged, respectively, with distinct TMs on the cell surface (Fig. 1). Theoretically, antigens act as scaffolds to
        induce dimerization of a pair of transmembrane proteins-tagged-split-nanobodies (Fig. 1a,b).
         　 We next conjugate split Nla tobacco etch virus (TEV) proteases to the intracellular site of our designed transmembrane proteins. The dimerization triggered by antigens, in turn, induces the formation of biologically activated TEV complexes
        to cleave their substrates carrying unique amino acid sequences (TEV cutting site, TCS; Fig. 1c). This inducible cleavage releases a transcription activator (in this case, tTA) from the plasma membrane to the nucleus (Fig. 1d), where soluble tTA
        binds to TRE3G promoter (pTRE3G) and drives the expression of GOIs (Gene of interests) as outputs of our programmable signaling system (Fig. 1e). With this design, our programmable signaling pathways can be modified to identify different forms
        of molecules from extracellular space by swapping available distinct nanobodies on the cell surface. Moreover, using different gene of interests, the engineered cells are able to elicit customized responses (Fig. 1). To check whether our construct
        works, we chose HEK293T as the reporter cells, and replace a segment of detector from nanobody to GFP-binding protein (GBP), with the usage of Green fluorescent protein (GFP) as the substrate (See <A href="https://2018.igem.org/Team:NTHU_Formosa/Results">results</A>).</P><P class="w3-justify" style="font-size: 22px; padding-left: 90px"><B>Figure 1. Schematic representation of our programmable signaling pathways. </B></P><P class="w3-justify" style="font-size:20px;"> 　 As for the output signal, bioluminescence is applied in our design. Conventionally, the production of bioluminescence requires additional chemical substrates, luciferins for example, which undergo an enzyme-catalyzed oxidation resulting in emitting
        bioluminescence signal (Mezzanotte et al., 2017). Injecting luciferins into living hosts is the invasive procedures we struggled to avoid. Therefore, we adopted a codon-optimized autonomous bioluminescence system, lux gene cassette, originated
        from microbes. This codon-optimized lux bioluminescence system scavenges endogenous intermediated metabolites stock within mammalian cells and converts them into substrates by enzymes, hLuxC, hLuxD, hLuxE, and frp (Close et al., 2010; Xu et al.,
        2014). To this end, CMV promoter is designated to continuously drive the expression of hLuxC, hLuxD, hLuxE, and frp for substrates production, while TRE3G promoter is used to inducibly drive the expression of hLuxA and hLuxB to form luciferase
        dimer (See <A href="https://2018.igem.org/Team:NTHU_Formosa/Results">results</A>). Bioluminescent output can be obtained in the presence of both substrates and inducibly formed luciferase when both promoters are activated.</P><P class="w3-justify" style="font-size: 22px; padding-left: 340px"><B>Figure 2. Biowatcher Reporter cells </B></P><P class="w3-justify" style="font-size:20px;">1. Mesenchymal stem cells (MSCs):
        Connective tissue cells which are multipotent and able to perform cell renewal.
        2. Nanobodies:
         Single variable domain derived from antibody fragment, with the size of 3nm (15kDa). Their small size enables them to pass through body tissues faster. Also, the simple structure results in higher affinity to antigen in contrast to the complete
        antibody.
         3. Luciferase:
         The class of oxidative enzymes that catalyzes luciferin for producing bioluminescence. In the autonomous lux gene system, the wavelength of bioluminescence output, namely the emitted light, is 490nm, which is close to blue-green colored
        </P><P class="w3-justify" style="font-size:25px;"><B>Reference: </B></P><P class="w3-justify" style="font-size:20px;">
        1. Altintas, Z., and Tothill, I. (2013). Biomarkers and biosensors for the early diagnosis of lung cancer. Sensors Actuators, B Chem. 188, 988–998.
         2. Close, D.M., Patterson, S.S., Ripp, S., Baek, S.J., Sanseverino, J., and Sayler, G.S. (2010). Autonomous bioluminescent expression of the bacterial luciferase gene cassette (lux) in a mammalian cell line. PLoS One 5.
         3. Hu, Y., Liu, C., and Muyldermans, S. (2017). Nanobody-based delivery systems for diagnosis and targeted tumor therapy. Front. Immunol. 8.
         4. Mezzanotte, L., van ‘t Root, M., Karatas, H., Goun, E.A., and Löwik, C.W.G.M. (2017). In Vivo Molecular Bioluminescence Imaging: New Tools and Applications. Trends Biotechnol. 35, 640–652.
         5. Rapisuwon, S., Vietsch, E.E., and Wellstein, A. (2016). Circulating biomarkers to monitor cancer progression and treatment. Comput. Struct. Biotechnol. J. 14, 211–222.
         6. Tang, J.C.Y., Szikra, T., Kozorovitskiy, Y., Teixiera, M., Sabatini, B.L., Roska, B., and Cepko, C.L. (2013). A nanobody-based system using fluorescent proteins as scaffolds for cell-specific gene manipulation. Cell 154, 928–939.
         7. Tang, J.C.Y., Rudolph, S., Dhande, O.S., Abraira, V.E., Choi, S., Lapan, S.W., Drew, I.R., Drokhlyansky, E., Huberman, A.D., Regehr, W.G., et al. (2015). Cell type-specific manipulation with GFP-dependent Cre recombinase. Nat. Neurosci. 18,
        1334–1341.
         8. Tang, J.C.Y., Drokhlyansky, E., Etemad, B., Rudolph, S., Guo, B., Wang, S., Ellis, E.G., Li, J.Z., Cepko, C.L., and Brooker, C. (2016). Detection and manipulation of live antigen-expressing cells using conditionally stable nanobodies. Aust.
        J. Pharm. 97, 5.
         9. Wang, Y., Fan, Z., Shao, L., Kong, X., Hou, X., Tian, D., Sun, Y., Xiao, Y., and Yu, L. (2016). Nanobody-derived nanobiotechnology tool kits for diverse biomedical and biotechnology applications. Int. J. Nanomedicine 11, 3287–3303.
         10. Xu, T., Ripp, S., Sayler, G.S., and Close, D.M. (2014). Expression of a humanized viral 2A-Mediated lux operon efficiently generates autonomous bioluminescence in human cells. PLoS One 9.
        </P></DIV></DIV><DIV id="myModal" class="modal"><DIV class="modal-content"><SPAN class="close">×</SPAN><P>Description text in the box. Image can also be included</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>