<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NTHU_Formosa_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NTHU Formosa/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="wrap"><HEADER><DIV class="inner relative"><NAV id="navigation"><UL id="main-menu"><LI class="current-menu-item"><A href="https://2018.igem.org/Team:NTHU_Formosa#0">HOME</A></LI><LI class="parent"><A href="https://2018.igem.org/Team:NTHU_Formosa#1">Project</A></LI><LI><A href="https://2018.igem.org/Team:NTHU_Formosa/Description"> Introduction</A></LI><LI><A href="https://2018.igem.org/Team:NTHU_Formosa/Design">  Design</A></LI><LI><A href="https://2018.igem.org/Team:NTHU_Formosa/Safety"> Biosafety</A></LI><LI><A class="parent" href="https://2018.igem.org/Team:NTHU_Formosa/Applied_Design"> Applied Design</A></LI></UL><LI class="parent"><A href="https://2018.igem.org/Team:NTHU_Formosa#2">Experiment</A></LI><LI><A href="https://2018.igem.org/Team:NTHU_Formosa/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:NTHU_Formosa/Parts">Parts &amp; Component</A></LI><LI><A href="https://2018.igem.org/Team:NTHU_Formosa/Composite_Part">Composite Part</A></LI><LI><A href="https://2018.igem.org/Team:NTHU_Formosa/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:NTHU_Formosa/InterLab">InterLab</A></LI><LI class="parent"><A href="https://2018.igem.org/Team:NTHU_Formosa/Model">Modeling</A></LI><LI class="parent"><A href="https://2018.igem.org/Team:NTHU_Formosa#4">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:NTHU_Formosa/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:NTHU_Formosa/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:NTHU_Formosa/Human_Practices">Integrated Human Practices</A></LI><LI class="parent"><A href="https://2018.igem.org/Team:NTHU_Formosa#5">About us</A></LI><LI><A href="https://2018.igem.org/Team:NTHU_Formosa/Team">Members</A></LI><LI><A href="https://2018.igem.org/Team:NTHU_Formosa/Sponsor">Sponsors</A></LI><LI><A href="https://2018.igem.org/Team:NTHU_Formosa/Attributions">Attributions</A></LI><LI><A href="https://igem.org/2018_Judging_Form?team=NTHU_Formosa">Judging Form</A></LI><LI><A href="https://2018.igem.org/Team:NTHU_Formosa/Medal">Medal</A></LI></NAV></DIV></HEADER></DIV><DIV class="w3-container w3-padding-24 w3-center"><DIV class="w3-content"><H2 class="w3-center" style="font-size:60px; font-family:Quicksand;"><B>Modeling</B></H2><P class="w3-justify"><B><BIG><BIG>Motivation</BIG></BIG></B></P><P class="w3-justify"> 　  Our sensing module design relies on a special form of antibody, called nanobodies. Nanobodies are single variable domain antibody fragments (VHH) derived from heavy-chain-only antibodies in camelidae. Nanobodies are used to recognize wide varieties of ligands. Due to its unique features, such as high affinity and specificity for antigen, thermostability, nanoscale size, and soluble characteristic in aqueous solution, over two thousand nanobodies are available for recognizing different antigens including membrane-bound molecules and soluble molecules, including biomarkers, according to the information from iCAN database (Institute collection and Analysis of Nanobody).

 　  Here we use nanobodies as the extracellular domain on our sensing module. We split the nanobodies into N-terminal and C-terminal fragments. Based on previous studies (Tang et al., 2013, 201), antigen binding induces the dimerization between N-terminal and C-terminal and thus triggers downstream gene expressions. Therefore, we tag our split nanobodies with a transmembrane domain on our sensing module along with other proteins. Theoretically, antigens will induce dimerization of these split sensing module and turn on downstream events. 
</P><P class="w3-justify"><B><BIG><BIG>Overview</BIG></BIG></B></P><P class="w3-justify"> 　  To prove the concept, we use the most well-studied nanobodies, GFP, and its antigen, GFP binding protein, GBP. Since trigger of BioWatcher system depends on the dimerization of the split nanobodies as they approach and binds to the antigen, prior to any of our experiment, we use simulation to model our design and see if dimerization of GBP N and C terminals happens at the presence of GFP. The GFP-GBP(N) complex were first aligned to reference (3OGO) to guarantee the possible binding interface, the simulation was focus on the binding interface of GBP(C) to GFP-GBP(N) complex.
</P><P class="w3-justify"><B><BIG><BIG>Methodology</BIG></BIG></B></P><P class="w3-justify"> 　  To best describe the conformation of GBP(C) and GFP-GBP(N) complex in real cellular environment, OpenMM Python API is used for molecular dynamics simulation, and the suggested binding conformation is visualized by VMD (Visual Molecular Dynamics). </P><P class="w3-justify"><B>Molecular Dynamics Simulation</B></P><P class="w3-justify">

 　  Although the GFP-binding domains of GBP(N) and GBP(C) are well-studied structures, the GBP(N) and GBP(C) linkers have no homology model structures. Therefore, they were built directly from Discovery Studio as very long unstructured loops. 
 　  At the initial condition, the soluble folded parts of GFP and GBP(N) have been balanced in the prior 200ns simulation. A stable binding interface between them has been found. The whole simulation would perform 200ns to find a stable structure of GBP(C) binding interface.
</P><P class="w3-justify"><B>Molecular Dynamics Simulation Details</B></P><P class="w3-justify">
Package: OpenMM Python API
Forcefield: CHARMM36m with supplementary force objects from CHARMM-GUI
Electrostatics: PME
VDW: 6-12 LJ with 1-4 scaling and force-switching
Switching: 1.0 nm 
Cutoff: 1.2 nm
Integrator: LangevinIntegrator 
Temperature: 310 K
Ion concentrations: 0.15 M NaCl
Barostat: MonteCarloMembraneBarostat
Volume changing frequency: 100 time steps
Time step: 0.002 ps
NPT production run: 200 ns</P><P class="w3-justify"><B>Sequence of the system to be built</B></P><P class="w3-justify">
GFP:
VSKGEELFTGVVPILVELDGDVNGHKFSVSGEGEGDATYGKLTLKFICTTGKLPVPWPTLVTTLTYGVQC
FSRYPDHMKQHDFFKSAMPEGYVQERTIFFKDDGNYKTRAEVKFEGDTLVNRIELKGIDFKEDGNILGHK
LEYNYNSHNVYIMADKQKNGIKVNFKIRHNIEDGSVQLADHYQQNTPIGDGPVLLPDNHYLSTQSALSKD
PNEKRDHMVLLEFVTAAGIT

GBP(N):
DVQLQESGGGSVQAGEALRLSCVGSGYTSINPYMAWFRQAPGKEREGVAAISSGGQYTYYADSVKGRFTI
SRDNAKNTMYLQMPSLKPDDSAKYYCAADFRRGGSWNVDPLRYDYQHWGQGTQVTVSS

GBP(C):
DVQLQESGGGSVQAGGSLRLSCAASGFPFSNYCMGWFRQAPGKEREGVATISRLGMFTEYADSVQGRFII
SRDNAQNMVFLQMNNLTPEDTAIYYCAAVSTSSSDCRPRLPSQEYTYWGQGTQVTVSSQ

GBP(N) Linker:
SAGGGGGSNAVGQDTQEVIVVP

GBP(C) Linker:
SAGGNAVGQDTQEVIVVP

Helix: 
HSLPFKVVVISAILALVVLTIISLIILIMLWQKKPRYE
</P><P class="w3-justify"><B>VMD (Visual Molecular Dynamics)</B></P><P class="w3-justify">
 　  VMD is a computer program designed for visualizing the result of molecular dynamics simulations, especially for protein and lipid bilayer system. Also, VMD can
animate and analyze the trajectory of a MD simulation then shows molecules in variety drawing methods and materials. So we used VMD to present the suggested binding conformation by a 3D model.
</P><P class="w3-justify"><B><BIG><BIG>Results &amp; Conclusions</BIG></BIG></B></P><P class="w3-justify">   After 200ns simulation, the suggested binding conformation of the complex is shown above. Here in the videos, we showed the N-terminal GBP (showed in yellow) and C-terminal GBP (showed in pink) anchored on the cell membrane separately. At the presence of GFP (showed in green), dimerization of the N-terminal and C-terminal GBP are formed when they are pulled together. Based on the results of the simulation that approve the design of our sensing module, we ran <A href="https://2018.igem.org/Team:NTHU_Formosa/Results">further experiments</A> and confirm the complete design for BioWatcher cells. 
</P><P class="w3-justify"><B><BIG><BIG>Reference:</BIG></BIG></B></P><P class="w3-justify" style="font-size:20px;">
1. Altintas, Z., and Tothill, I. (2013). Biomarkers and biosensors for the early diagnosis of lung cancer. Sensors Actuators, B Chem. 188, 988–998.</P></DIV></DIV><DIV id="myModal" class="modal"><DIV class="modal-content"><SPAN class="close">×</SPAN><P>Description text in the box. Image can also be included</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>