<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bologna_Modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Bologna/Modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Bologna/Modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bologna/Modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bologna/Modeling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bologna/Modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bologna/Modeling</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><TABLE style="color:#fffff1;background-color:#f6efcd;" cellpadding="1" cellspacing="1" border="0" width="100%" align="center"><TBODY><TR><TH align="center"><A href="/Team:Bologna" title="Team:Bologna">HOME</A></TH><TH align="center"><A href="/Team:Bologna/Project" title="Team:Bologna/Project">PROJECT</A></TH><TH align="center"><A href="/Team:Bologna/Team" title="Team:Bologna/Team">TEAM</A></TH><TH align="center"><A href="/Team:Bologna/Software" title="Team:Bologna/Software">SOFTWARE</A></TH><TH align="center"><STRONG class="selflink">MODELING</STRONG></TH><TH align="center"><A href="/Team:Bologna/Wetlab" title="Team:Bologna/Wetlab">WET LAB</A></TH><TH align="center"><A href="/Team:Bologna/Notebook" title="Team:Bologna/Notebook">LAB-BOOK</A></TH><TH align="center"><A href="/Team:Bologna/Parts" title="Team:Bologna/Parts">SUBMITTED PARTS</A></TH><TH align="center"><A href="/Team:Bologna/Biosafety" title="Team:Bologna/Biosafety">BIOSAFETY AND PROTOCOLS</A></TH></TR></TBODY></TABLE><DIV style="text-align:justify"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Model-based_analysis_of_the_genetic_Flip-Flop"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Model-based analysis of the genetic Flip-Flop</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#The_genetic_Flip-Flop"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">The genetic Flip-Flop</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Mathematical_Model"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Mathematical Model</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Equibrium_conditions"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Equibrium conditions</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Bifurcation_analysis"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Bifurcation analysis</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Procedure_for_Ki-index_identification"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Procedure for Ki-index identification</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Numerical_simulations"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Numerical simulations</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-8"><A href="#SR_Latch"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">SR Latch</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#UV_Radiation:_SOS_system"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">UV Radiation: SOS system</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Bibliography"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Bibliography</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Model-based_analysis_of_the_genetic_Flip-Flop"> Model-based analysis of the genetic Flip-Flop </SPAN></H1><H2><SPAN class="mw-headline" id="The_genetic_Flip-Flop"> The genetic Flip-Flop </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Figure 1. Scheme of the genetic Flip-Flop</DIV></DIV></DIV><DIV style="text-align:justify"><P>The molecular circuit in Figure 1 can switch between two different stable states (LacI-ON and TetR-ON), driven by two external stimuli (UVc and IPTG). LacI-ON represents the stable state in which LacI gene is active and LacI protein represses  the TetR gene expression, with a positive feedback. Therefore, the LacI-ON state coincides with the TetR-OFF condition. On the contrary, the TetR-ON represents the state with the TetR gene active and the LacI gene silenced (LacI-OFF). Owing to the coexistence of two stable states (bistability), this circuit is capable of serving as a binary memory [Collins et al. 2000]. We denominated it  a Flip-Flop since it works as a <A href="/Team:Bologna/Modeling#SR_Latch" title="Team:Bologna/Modeling">SR Latch</A>: LacI state is the  output and TetR state is the  output. Uvc is the set signal and IPTG is the reset signal. Indeed, IPTG stimulation inhibits LacI repressor, thus can cause the transition from the LacI-ON state to the TetR-ON. UVc radiation, inactivating LexA repressor through the <A href="/Team:Bologna/Modeling#UV_Radiation:_SOS_system" title="Team:Bologna/Modeling">SOS response</A> [Friedberg et al. 1995] can cause the opposite transition from LacI-ON to TetR-ON.
The core elements of the genetic program are two mutually regulated promoters. The promoter transcriptional strength and the repressor binding affinity to the promoter determinate such relevant circuit properties as the bistability and the response to inputs. To quote the promoters in terms of strength and sensitivity to repressor we resorted to the following model-based analysis and <A href="/Team:Bologna/Modeling#Numerical_simulations" title="Team:Bologna/Modeling">numerical simulations</A>.
</P><P><A href="https://2008.igem.org/Team:Bologna/Modeling" class="external text" rel="nofollow"><I>Up</I></A></P><H2><SPAN class="mw-headline" id="Mathematical_Model"> Mathematical Model </SPAN></H2><P><B><FONT size="3">Model equations</FONT></B></P><P>
The Flip-Flop circuit in Figure 1 can be modeled by the following equations:
</P><P>Symbol definition is listed in Table 1.
</P><P>A common motif in repressor proteins is the presence of a dimeric nucleotide-binding site with dimeric structure. In accordance to this general structure the cooperativity coefficients () were assumed equal to 2. The maximum velocity of repressor synthesis () accounts for the strength of the unregulated promoter and RBS. The value of the affinity constant for the binding of repressor to the promoter strictly depends on the sequence of operator site (OS block).
</P><P><B><FONT size="3">Adimensional equations</FONT></B></P><P>
The equations (1.1) and (1.2) can be written dimensionless:
</P><P>Where:
</P><P><A href="https://2008.igem.org/Team:Bologna/Modeling" class="external text" rel="nofollow"><I>Up</I></A></P><H2><SPAN class="mw-headline" id="Equibrium_conditions"> Equibrium conditions </SPAN></H2><P>In the absence of stimuli, the adimensional concentrations of LacI () and TetR () at equilibrium are related by the equations: 
</P><P>
To obtain these relations the UVc-dependent term in equation (1.4) was ignored ( ). This is justified by the high binding constant of LexA for its operator, and the consequent negligible contribution to the LacI synthesis.
Equations (1.6) and (1.7) can have one or three solutions that represent the equilibrium conditions of the circuit. The solutions, i.e. the equilibrium conditions, can be graphically identified as the intersections between  and   nullclines (see Figure 2). The case of multiple equilibrium conditions (bistability case) is shown in Figure 2 panel b. TetR-ON and LacI-ON are stable equilibria separated by the unstable one (saddle point). Due to the bistability the circuit can operate as a binary memory.  
The existence of a bistability condition depends on the value of  and    parameters. If  decrease (see Figure 2 panel a) a saddle-node bifurcation can occur, TetR-ON equilibrium vanishes and remain only the stable equilibrium LacI-ON.  The contrary occurs when  is decreased (Figure 2 panel c). Thus bistability is guaranteed only for a limited range of   and  values. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption">Figure 2. Equilibrium conditions</DIV></DIV></DIV></DIV><P><A href="https://2008.igem.org/Team:Bologna/Modeling" class="external text" rel="nofollow"><I>Up</I></A></P><H2><SPAN class="mw-headline" id="Bifurcation_analysis"> Bifurcation analysis </SPAN></H2><P>Assuming that LacI-ON exists the corresponding equilibrium value of   is higher than 1 (see Figure 2), then it can be assumed that   and the equation (1.7) simplies to:
</P><P>
Substituting this expression in equation (1.6) one obtain:
</P><P>Then:
</P><P>
To be real the solutions of equation (1.10) it is necessary that   (see Figure 3). Under this condition the existence of the LacI-ON state is assured. When   the system undergoes a saddle-node bifurcation (LacI-ON and saddle point go in collision) and the two equilibrium points vanish.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 3. Saddle-Node bifurcation</DIV></DIV></DIV></DIV><P>
An analogous result can be obtained for the existence of the TetR-ON state. Thus, a sufficient condition for bistability is:
</P><P>Figure 4 shows the log-log plot of (1.11) and (1.12) 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 4. Range of bistability for the genetic Flip-Flop</DIV></DIV></DIV></DIV><P>As shown in Figure 4, the coexistence of two stable equilibria (LacI-ON and TerR-ON) is guaranteed for  and greater than 3 in a large range of values (bistability range). Imposing a value for  greater than 3 it is possible to establish a range for  to have a bistable behaviour. More the (,) point is near the bifurcation lines more the transition from bistability to monostability is probably. To have a good robustness we fixed  and  equal to 10. For this value the circuit is sufficently distant from the bifurcation lines to avoid random memory switching but it is possible to set (or to reset) the memory with proper stimulation. 
</P><P><A href="https://2008.igem.org/Team:Bologna/Modeling" class="external text" rel="nofollow"><I>Up</I></A></P><H2><SPAN class="mw-headline" id="Procedure_for_Ki-index_identification"> Procedure for Ki-index identification </SPAN></H2><P>
The procedure will be described for LacI, analogous procedure can be applied to the TetR case. The value of -index can be identified comparing the experimental responses of the open loop and closed loop circuits: 
</P><TABLE align="center"><TBODY><TR><TD><UL><LI> Open loop circuit
</LI></UL><UL><LI> Closed loop circuit
</LI></UL></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 5. Molecular circuits for the Ki-index experimental determination</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>The LacI concentration in the open loop circuit is given by:
</P><P>
Thus the equilibrium condition is:
</P><P>
The time derivative of LacI concentration in the close loop circuit follows:
</P><P>
Which gives the equilibrium condition:
</P><P>
That can be rewritten 
</P><P>
The affinity constant can consequently be derived from this expression
</P><P>
Inserting the (1.14) and (1.18) in the  -index definition one obtain:
</P><P>
We assume that GFP is proportion to  , then
</P><P>
We introduce the ratio  between the fluorescence in open loop and in closed loop:
</P><P>
After measuring the ratio   it is possible to calculate   by the curve in Figure 6 and then it is possible to establish by Figure 4  the   range that guarantees bistability. In the presence of an experimentally characterized library of regulated promoter, the procedure can be adopted to design genetic Flip-Flop with desired behaviors.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 6.  Ki-index  for LacI</DIV></DIV></DIV></DIV><P><A href="https://2008.igem.org/Team:Bologna/Modeling" class="external text" rel="nofollow"><I>Up</I></A></P><H2><SPAN class="mw-headline" id="Numerical_simulations"> Numerical simulations</SPAN></H2><P>Model equations (1.4) and (1.5) have been used to simulate: i) the sensitivity of the circuit to initial state and ii) the response to  and <A href="/Team:Bologna/Wetlab#UV_Induction" title="Team:Bologna/Wetlab">UVc stimulations</A>. Numerical simulations were performed in Simulink (MathWorks). Simulator scheme is shown in Figure 7. To ensure circuit bistability parameters  and  were both fixed to 10.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 7. Simulink simulator scheme</DIV></DIV></DIV></DIV><P>The trajectories in the plane of the states (see Figure 8) clearly show the separatrix line (red line) and two attraction basins (one for the LacI-ON and the other for TetR-ON equilibrium). Depending on the initial state the memory can reach one of two stable equilibria (blu curve). The memory switches from a stable equilibrium to the other one when an external perturbation move the state over the separatrix entering it in the other attraction basin.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 8. State-Plane Trajectories</DIV></DIV></DIV></DIV><P>To simulate the  reset, we fixed the initial state to the LacI-ON ( = 10 and  = 0.2) and we simulate the application of different levels of . In the simulations we referred the  to the  concentration. An induction of maximum 5 hours was considered (see Figure 9).  Until  level is lower than 1.2 times the , the memory does not reset to TetR-ON  and when the induction finish, the state returns in about 5 hours to LacI-ON. To observe the memory switching is necessary to overcome the separatrix. This occurs for &gt; 1.2 times . With a minimal  dose the switching is very slow (about 5 hour). To accelerate the transition it is sufficient to expose the circuit for 1 hour to  = 3 times . 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 9. Circuit response to IPTG pulse</DIV></DIV></DIV></DIV><P>To simulate the UVc set to LacI-ON of the memory, we assumed the TetR-ON as initial condition ( = 0.2 and  = 10),  parameter was fixed equal to 10, the rate  was equal to 30 and   was equal to 2. We supposed to radiate the memory for few minutes with different UVc energies. The cell radiation starts the SOS response that we simple modeled as a temporary RecA-mediated inhibition of LexA repressor. The memory set to LacI-ON does not take place until the UVc energy is lower than a threshold of 16  (see Figure 10). When the UVc exceeds this threshold, the memory switches from TetR-ON to LacI-ON. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 10. Circuit response to UVc radiation</DIV></DIV></DIV></DIV><P><A href="https://2008.igem.org/Team:Bologna/Modeling" class="external text" rel="nofollow"><I>Up</I></A></P><H1><SPAN class="mw-headline" id="SR_Latch"> SR Latch </SPAN></H1><P>In digital circuits, a Flip-Flop is a term referring to an electronic circuit (a bistable multivibrator) that has two stable states and thereby is capable of serving as one bit of memory. Today, the term Flip-Flop has come to mostly denote non-transparent (clocked or edge-triggered) devices, while the simpler transparent ones are often referred to as <B>latches</B>; however, as this distinction is quite new, the two words are sometimes used interchangeably.
</P><P>A Flip-Flop is usually controlled by one or two control signals and/or a gate or clock signal. The output often includes the complement as well as the normal output. As Flip-Flops are implemented electronically, they require power and ground connections.
</P><P>Flip-Flops can be either simple (transparent) or clocked. Simple Flip-Flops can be built around a pair of cross-coupled inverting elements: vacuum tubes, bipolar transistors, field effect transistors, inverters, and inverting logic gates have all been used in practical circuits — perhaps augmented by some gating mechanism (an enable/disable input). The more advanced clocked (or non-transparent) devices are specially designed for synchronous (time-discrete) systems; such devices therefore ignores its inputs except at the transition of a dedicated clock signal (known as clocking, pulsing, or strobing). This causes the Flip-Flop to either change or retain its output signal based upon the values of the input signals at the transition. Some Flip-Flops change output on the rising edge of the clock, others on the falling edge.
</P><P>Our project simulates a SR Latch (Figure 11), the most fundamental latch, where S and R stand for set and reset. It can be constructed from a pair of cross-coupled NOR logic gates. The stored bit is present on the output marked Q (or the complement <SPAN style="text-decoration:overline">Q</SPAN>).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Figure 11. SR latch</DIV></DIV></DIV></DIV><P>
Normally, in storage mode, the S and R inputs are both low, and feedback maintains the Q and <SPAN style="text-decoration:overline">Q</SPAN> outputs in a constant state. If S (Set) is pulsed high while R is held low, then the Q output is forced high, and stays high when S returns low; similarly, if R (Reset) is pulsed high while S is held low, then the Q output is forced low, and stays low when R returns low (Figure 12).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Figure 12. SR latch truth table</DIV></DIV></DIV></DIV><P>
The R = S = 1 combination is called a restricted combination because, as both NOR gates then output zeros, it breaks the logical equation Q = not <SPAN style="text-decoration:overline">Q</SPAN>. The combination is also inappropriate in circuits where both inputs may go low simultaneously (i.e. a transition from restricted to keep). The output would lock at either 1 or 0 depending on the propagation time relations between the gates (a race condition). In certain implementations, it could also lead to longer ringings (damped oscillations) before the output settles, and thereby result in undetermined values (errors) in high-frequency digital circuits. This condition is therefore sometimes avoided.
</P><P><A href="https://2008.igem.org/Team:Bologna/Modeling" class="external text" rel="nofollow"><I>Up</I></A></P><H1><SPAN class="mw-headline" id="UV_Radiation:_SOS_system">UV Radiation: SOS system </SPAN></H1><P>Ultraviolet is that part of electromagnetic radiation bounded by the lower wavelength extreme of the visible spectrum and the upper end of the X-ray radiation band. The spectral range of ultraviolet radiation  is,  by definition, between 100  and 400 nm  and  is   invisible   to human  eyes.  The  UV spectrum is subdivided into three bands: UV-A (long-wave) from 315 to 400 nm, UV-B (medium-wave) from 280 to 315 nm, UV-C (short-wave) from 100 to 280 nm. A strong germicidal effect is provided by the radiation in the short-wave UV-C band. 
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:339px;"><DIV class="thumbcaption">Figure 13. After dimerization of two adjacent thymines blue light gives energy to photolyase to repair the damage</DIV></DIV></DIV><P>The maximum UV germicidal effect  coincides with the peak absorbance of DNA (near 260nm) due to the dimerization of two adjacent thymines.
E.Coli cells have a system that recovery DNA damage when it occurs. The best studied transcriptional response to DNA damage is the SOS response [Friedberg et al 1995, Walker et al. 1996].  
This systems can be divided into two class: the SOS Photoreactivation repair and the  SOS respond triggered by RecA protein. The first uses the photolyase, a poorly expressed enzyme(encoded by genes phrA and phrB)  which  binds the pyrimidine dimers and uses the blue light to split them apart as showed in Figure 13.
Otherwise single stranded DNA produced by several DNA-damaging agents can be bound by RecA protein, resulting in conversion of this protein to its activated form. The RecA repair system doesn’t need light and Lexa protein controls the expression of 43 genes [Courcelle et al. (2001)]  that cooperate together to repair the extensive genetic  damage (Figure 14). RecA and LexA proteins play an important rule for the regulatory of SOS Recombination System.  A LexA binding site is present in all the SOS promoters genes' and it works as a repressor of SOS system. In presence of DNA damage (DNA Single Strains) RecA becomes active and interacts with LexA protein, the repressor of the SOS genes [Wagner et al., 1999] . This interaction triggers the autocatalytic cleavage of LexA and consequent destruction of its ability to function as a repressor, which results in the derepression of SOS genes [Mustard and Little 2000; Fernandez De Henestrosa et al. 2000]. When the damage is repaired, DNA single strains are not present in the cell and the RecA protein no longer promotes the auto-cleavage of the LexA which is restored to its initial repression level (Figure 15).</P><TABLE align="center"><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:270px;"><DIV class="thumbcaption">Figure 14. SOS System controlled by LexA</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Figure 15. Genes repressed by LexA</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P><A href="https://2008.igem.org/Team:Bologna/Modeling" class="external text" rel="nofollow"><I>Up</I></A></P><H1><SPAN class="mw-headline" id="Bibliography"> Bibliography </SPAN></H1><UL><LI>Mads Kaern, William J.Blake, and J.J.Collins. The Engineering of Gene Regulatory Networks, Annu.Rev.Biomed.Eng.2003 5:179-206.
</LI><LI>Timothy S.Gardner, Charles R.Cantor &amp; James J.Collins. Construction of a genetic toggle switch in Escherichia coli, Nature Vol.403 339-342.
</LI><LI>Nitzan Rosenfeld, Jonathan W.Young, Uri Alon, Peter S.Swain, Michael B.Elowitz. Gene Regulation at the Single-Cell Level. Science Vol 307 1962-1965.
</LI><LI>Sergej V.Aksenov. Dynamics of the inducing signal for the SOS regulatory system in Escherichia coli after ultraviolet irradiation. Mathematical Bioscienses 157 (1999) 269-286.
</LI><LI>Sandeep Krishna, Sergei Maslov, Kim Sneppen. UV-induced mutagenesis in Escherichia coli SOS response: A quantitative model. PLoS Comput Biol 3(3): e41. doi:10.1371/journal.pcbi.0030041.
</LI><LI>S.V.Aksenov et al. Mathematical model of SOS response regulation of an excision repair deficient mutant of Escherichia coli after ultraviolet light irradiation. J.theor.Biol (1997)186, 251-260.
</LI><LI>M.Sassanfar, J.W.Roberts. Nature of the SOS inducing signal in Escherichia coli: the involvement of DNA replication. J.Molec.Biol. 212 (1990) 79.
</LI><LI>R.Brent and M.Ptashne. Mechanism of action of the lexA gene production. Prot.Nat.Acad.Sci.USA Vol.78, No 7, 4204-4208, 1981.
</LI><LI>Friedberg, E.C., Walker, G.C., and Siede, W. DNA repair and mutagenesis. ASM Press, Washington, D.C. 1995.
</LI><LI>Walker, G.C. The SOS Response of Escherichia coli. In Escherichia coli and salmonella: Cellular and molecular biology (ed. F.C. Neidhardt, R. Curtiss III, J.L. Ingraham, E.C.C. Lin, K.B. Low, B. Magasanik, W.S. Reznikoff, M. Riley, M. Schaechter and H.E. Umbarger), pp. 1400-1416. ASM Press, Washington, D.C. 1996
</LI><LI>Courcelle, J., A. Khodursky, B. Peter, P. O. Brown and P. C. Hanawalt, 2001 Comparative gene expression profiles following UV exposure in wild-type and SOS-deficient Escherichia coli. Genetics 158: 41–64.
</LI><LI>J. Wagner, P. Gruz, S.-R. Kim, M. Yamada, K. Matsui, R.P.P. Fuchs and T. Nohmi, The dinB gene encodes a novel Escherichia coli DNA polymerase (DNA pol IV) involved in mutagenesis. Mol. Cell 4 (1999), pp. 281–286.
</LI><LI>Mustard, J. A., Little, J. W. (2000). Analysis of Escherichia coli RecA Interactions with LexA, lambda CI, and UmuD by Site-Directed Mutagenesis of recA. J. Bacteriol. 182: 1659-1670.
</LI><LI>Fernandez De Henestrosa, A. R., Ogi, T., Aoyagi, S., Chafin, D., Hayes, J. J., Ohmori, H. &amp; Woodgate, R. (2000). Identification of additional genes belonging to the LexA regulon in Escherichia coli. Mol Microbiol 35, 1560–1572.
</LI></UL><A href="https://2008.igem.org/Team:Bologna/Modeling" class="external text" rel="nofollow"><I>Up</I></A></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Bologna/Modeling">http://2008.igem.org/Team:Bologna/Modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bologna/Modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bologna/Modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bologna/Modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bologna/Modeling&amp;oldid=103226" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>