"
Page
Discussion
View source
History
teams
Log in
 
Team:GeorgiaTech/WeekTen
From 2010.igem.org
Georgia Institute of Technology iGEM Team 2010 Homepage
10/3-10/9
10/3/10
For Today:
Run on a gel
aoxa, aoxb next to the troubleshooting pcrs from today, specifically of the PCRS using Ompa-F and Aoxa/b-R since they are similar sizes
Take out plates from incubator. Record observations
Run colony PCRs on at least 20 colonies per construct (yes 20)
Check which primers to use for colony pcr! We can likely use the primers we already have
Run results on gel
 
Results: plated colonies from 10/2/10 (Mitesh & Rob):
Observed that HybB-ompA-Aox1(a/b) constructs were successfully transformed- strong colony growth and red colonies (strange- given that these were supposedly constructs without RFP). The color strongly suggests RFP. Red colony patterns show no consistent trends- some are densely red in the middle of the plate, others mainly around the edges- red intensity varied widely even on the same plate.
 
 
Colony PCR of HybB-ompA-Aox constructs
Picked colonies from plates of HybB-ompA-Aox-RFP and added to 25 ul of water each,10 colonies from Aox1a containing plates, and 10 from Aox1b containing plates:
 
HybB-ompA-Aox1a-RFP: 1.5 mL Tubes 1-10
HybB-ompA-Aox1b-RFP: 1.5 mL Tubes 11-20
 
Used 5 ul for PCR, and added 250 ul of LB media to the rest.
 
Colony PCRs (for each colony): (Rob, Mitesh & Scott)
5 uL of cells
25.5 uL H2O
10 uL GOTAQ 5X Reaction buffer
5 uL HybB-F forward primer
5 uL RFP-R reverse primer
1 uL dNTP 10 mM - (thawed & kept on ice)
0.5 uL polymerase enzyme, TAQ
Total Volume= 50 uL
 
Colony PCRs (for each colony):
5 uL of cells
25.5 uL H2O
10 uL GOTAQ 5X Reaction buffer
5 uL HybB-F forward primer
5 uL Aox1a or b-R reverse primer
1 uL dNTP 10 mM - (thawed & kept on ice)
0.5 uL polymerase enzyme, TAQ
Total Volume= 50 uL
Important: For the PCR tubes, the ones labeled 1,3,5,7,9 are treated with Aox1a reverse primer.
All the even number (2,4,6,...16,18,20) labeled have RFP-R reverse primer in them. And the tubes labeled 11,13,15,17,19 have Aox 1bR reverse primer in them.
PCR start time: 6 PM
 
For Monday:
Check colony pcr on a gel
calculate the predicted sizes and compare to gel results
Continue with colony PCRS if more info is required
Check triple ligations on gel
 
10/4/2010
Protocols
 
1 % Gel to Check colony PCRs from 10/3/2010
Started at 9 am
Order:ladder:1-6|ladder|7-12|ladder|13-14
Important: For the PCR tubes, the ones labeled 1,3,5,7,9 are treated with Aox1a reverse primer.
All the even number (2,4,6,...16,18,20) labeled have RFP-R reverse primer in them. And the tubes labeled 11,13,15,17,19 have Aox 1bR reverse primer in them.
PCR start time: 6 PM
gel start at 10:34 am
 
Predicted Bands:
Hyb-F + RFP-R approx 2100-2200  bp
Hyb-F + Aoxa-R - approx 1500
Hyb-F + Aoxb-R approx 1500
 
Observations- 2200 bp band is seen, but other bands are also observed for the Hyb-F and RFP-R reactions
 
Suggestions:
Troubleshoot by PCR sequencing the basic products (e.g. sequence just HybB in one tube, in the next, just OmpA; in the next, just AOX; in the next, just RFP). This will tell us whether each gene is actually present in the cells PCRed.
Start over again entirely, from the very beginning, digesting each individual gene, ligating, transforming.
Start using controls. When possible, include positive and negative controls.
For example, when doing a double-ligation, for instance, perform experiment alongside designed to fail (e.g. lacking one of the ligation genes) for a negative control.
To see if the plasmid pSB1A3 is causing our problems, transformed this plasmid alone directly into E. coli and plate.
If pSB1A3 turns out to be the problem, can attempt to use a different plasmid backbone. We’ve researched and found plasmid pSB1C3 to have the desired characteristics, including lack of a  promoter, beginning EcoRI and ending SpeI restriction sites.
 
Plan to go forward:
1. Transform “plain” plasmid pSB1A3 into E. coli today 4 OCT 2010. Check for RFP expression.
2. In tandem with step 3, troubleshoot by sequencing the basic gene products in the “bad” HybB-OmpA-AOX contsructs, looking specifically for each gene.
3. In tandem with step 2, re-do the entire construction process from the individual genes up.
4. Transform cells using new biobrick vectors, like psb1c3, and use those instead of psb1a3!
 
NOTE: Need to make more media plates!
 
Protocols
 
Transformation of NB cells using pSB1a3
10 սL Nova Blue cells + 5 սL of pSB1A3 from MiniPrep
 
1. Left cells and plasmid on ice.
2. Added plasmid (5սL) to cells. Mix gently by swirling pipette tip in mixture (DO NOT ASPIRATE).
3. Left cells on ice for 30 min.
4. Applied heat shock of 45 seconds in 42C bath.
5. Put tubes on ice for 2 min.
6. Added 250 սL of LB (room temp.)
7. Incubated 1 hour at 37 C
8. Plated 100սL (x2 plates) and left plates in the 37 degrees incubator  
9. Incubated overnight at 37C.
 
Troubleshooting PCRs
For each test (e.g. HybB, OmpA, AOX1a, etc.)
2 uL of cells
9.4 uL H2O
4 uL GOTAQ 5X Reaction buffer
2 uL forward primer
2 uL reverse primer
0.4 uL dNTP 10 mM - (thawed & kept on ice)
0.2 uL polymerase enzyme, GOTAQ
Total Volume= 20 uL  
 
Combinations
A: HybB-OmpA-AOX1a-pSB1A3 in NovaBlue cells, ligated 2 OCT 2010 (Margo + Scott)
Selected 1 red colony, suspended in 20 uL sterile dH2O
1: HybB forward & reverse primers
2: OmpA forward & reverse primers
3: AOX1a forward & reverse primers
4: RFP forward & reverse primers
 
Selected 1 white colony, suspended in 20 uL sterile dH2O
5: HybB forward & reverse primers
6: OmpA forward & reverse primers
7: AOX1a forward & reverse primers
8: RFP forward & reverse primers
 
B: HybB-OmpA-AOX1b-pSB1A3 in NovaBlue cells, ligated 2 OCT 2010 (Margo + Scott)
Selected 1 red colony, suspended in 20 uL sterile dH2O
1: HybB forward & reverse primers
2: OmpA forward & reverse primers
3: AOX1a forward & reverse primers
4: RFP forward & reverse primers
 
Selected 1 white colony, suspended in 20 uL sterile dH2O
5: HybB forward & reverse primers
6: OmpA forward & reverse primers
7: AOX1a forward & reverse primers
8: RFP forward & reverse primers
 
P: “plain” mini-prepped pSB1A3 plasmid
        RFP forward & reverse primers
 
Ran a PCR on the samples using 1KB ladder -- ran A 1-7
Lane         1: ladder
2: A1
3: A2
4: A3
5: A4
6: A5
7: A6
8: A7
 
 
Note: need to make a big batch of 1% gels tomorrow since we are running so many samples.
Need more agarose from Gaucher Lab. COMPLETED
Need more EtBr. COMPLETED
Need a 100bp Ladder to see some of this stuff (small constructs) -- we might need to get Ryan to order some.
 
10-5-10
Obtain some more agarose and EtBr from Gaucher lab.
Make 1% gels.
See Protocols page for Preparing 1% Agarose Gels
Ran a gel on the remaining PCR samples from 10-4-10
Following 1 KB ladder, lanes should read in order:
A8, P, B1, B2, B3, B4, B5, B6, B7, B8.
No digital picture available, but one was printed and is in the lab, labelled with the date.
 
Starting construct building strategy over again
Reactions:
HyBb F,R (template biobrick)
OmpA F,R (template ompa vector)
Aox1a F,R (template synthesized gene)
Aox1a F,R2 (template synthesized gene)
Aox1b F,R (template synthesized gene)
Aox1b F,R2 (template synthesized gene)
RFP F,R (template RFP plasmid) (from 9.27.2010 miniprep)
RFP F2,R  (template RFP plasmid) (from 9.27.2010 miniprep)
 
PCR Protocol (Scott)
26.5 uL H2O
10 uL PHUSION 5X Reaction buffer
5 uL forward primer
5 uL reverse primer
1 uL dNTP 10 mM - (thawed & kept on ice)
2 uL template DNA
0.5 uL polymerase enzyme, PHUSION
Total Volume= 50 uL
started 4:33pm
 
10/6/2010
Goals
The gel from 10/5/2010 of the troubleshooting pcrs, specifically the reaction of pSB1A3 with RFP-F and R, suggests that the vector used for ligation has RFP. We will use anothe rvector from biobrick
We want to transform cells with the new biobrick vector today, miniprep the cells tomorrow morning, digest, and ligate to our hyb.ompa+aox that we have or are making now. Finish by friday is the goal.
Sequence hyb-RFP (for bronze medal requirements)
begin hyb-ompa-rfp building when primers come in
I saw Richard’s sequence was for psb1c3, but we are using psb1a3; this may a primer and digests! Investigate!
Sequence white colonies from the plates on 10/3/2010
For today:
Reconstitute new biobrick vector (psb1A7) COMPLETED
Transform NB cells using new biobrick vector COMPLETED
Check PCRs from 10/5/2010 on gel (tubes in yellow pcr box labeled 1-8 and dated 10.5) COMPLETED
Make Amp plates
Autoclave pipette tips
This involves testing which tips are compatible with ours (we have lots of random tips)
 
Notes on Vector-
Here is an experience from another team concerning transcription of a product during an “off state”:
 
Our team ( Davidson College ) has used this vector in E. coli strain JM109 extensively. Clones are Amp resistant and are recovered in reasonable yields from mini preps. One drawback is that the vector has read-through transcription that comes into the cloned BioBrick part from both directions. This problem is a major one when an "off state" is critical for the control of a device. Parts without a promoter are expressed in this vector, regardless of orientation (see pSB1A7 Part Design for details). Reverse read-through might be from the Amp resistance gene. Forward read-through might be from a cryptic promoter, since no known genes in the backbone are in the plus orientation.
-- Kahaynes 15:48, 23 October 2006 (EDT)
 
Alternative vectors:
pSB1A7 (2010 Kit plate 3, 15G well, name BBa_K126000)
Reasons:
There are double terminators on either side of the multiple cloning site that block any read through or unwanted transcription from the plasmid promoters (plasmid promoters are involved in antibiotic production)
Ampicillin resistance
Already in kit
 
 
Protocols
1. Resuspend psb1A7 DNA in 10uL of autoclaved water.
 
 
3. Run gels of PCR’s on 10-5-10.
 
Order:
HyBb F,R (template biobrick)
OmpA F,R (template ompa vector)
Aox1a F,R (template synthesized gene)
Aox1a F,R2 (template synthesized gene)
Aox1b F,R (template synthesized gene)
Aox1b F,R2 (template synthesized gene)
RFP F,R (template RFP plasmid) (from 9.27.2010 miniprep)
RFP F2,R  (template RFP plasmid) (from 9.27.2010 miniprep)
 
Growing colony of pSB1a3 transformed cells (Scott)
I picked 3 colonies from the plate marked “MFC 9.15.2010 C1 “ from 9.15.2010, with the psb1a3 transformed cells that are slighty pink. I did 3 mL + 3 uL of Carb, and put them in the incubator.
 
PCR of pSB1A3 (direct from well aliquot) using RFP-F,R
27.5 uL H2O
10 uL GoTaq 5X Reaction buffer
5 uL forward primer (RFP-F)
5 uL reverse primer (RFP-R)
1 uL dNTP 10 mM - (thawed & kept on ice)
1 uL template DNA
0.5 uL polymerase enzyme, GoTaq
Total Volume= 50 uL
Left in block B- Thursday people need to retieve it- Need to gel purify- scroll down to see a lit of things to do for Thursday. These steps are also highligted in blue below.
PCR of mRFP miniprep (direct from well aliquot) using RFP-F3,R
26.5 uL H2O
10 uL GoTaq 5X Reaction buffer
5 uL forward primer (RFP-F3)
5 uL reverse primer (RFP-R)
1 uL dNTP 10 mM - (thawed & kept on ice)
2 uL template DNA
0.5 uL polymerase enzyme, GoTaq
Total Volume= 50 uL
Left in block B- Thursday people need to retieve it  - Scott knows what to do with this. I don’t. -Debika
 
 
Transformation of NB using pSB1A3 direct from Gel aliquot) -completed (Christina, Debika)
10 սL Nova Blue cells + 5 սL of pSB1A3 from MiniPrep
 
1. Left cells and plasmid on ice.(Scott)
2. Added plasmid (5սL) to cells. Mix gently by swirling pipette tip in mixture (DO NOT ASPIRATE).
3. Left cells on ice for 30 min.
4. Applied heat shock of 45 seconds in 42C bath. (Debika)
5. Put tubes on ice for 2 min. (I messed up and put in LB in a minute instead of 2 minutes. Left it in ice again for a minute after adding LB). Don’t know how that will affect our results. Sorry if this transformation doesn’t work).
6. Added 250 սL of LB (room temp.)
7. Incubated 1 hour at 37 C
8. Plated 100սL (x2 plates) and left plates in the 37 degrees incubator. (Christina)  
9. Incubated overnight at 37C.  
 
Transformation of NB using pSB1A7 direct from Gel aliquot) -completed (Christina)
10 սL Nova Blue cells + 5 սL of pSB1A3 from MiniPrep
Add 1uL of resuspended psb1A7 in 10uL of competent cells (Novablue). (thawed on ice)
See Protocols page for Heat Shock Transformation
 
Note: The incubation step was skipped, which likely will lead to little growth
 
Meeting Notes
Priority List
Critical
hyb.ompa.aox construct (in pSB1A7)
hyb.ompa.RFP construct (in pSB1A7) (Scott)
Sequencing of constructs and conformation with standard
Biobrick
Need
Calorimetric/Heat  (Rob, Christian)
Periplasmic Expression Picture (Fluorescent Microscope) (Scott)
Cold Shock Quantification (hybB)
Modeling (Gita, Mitesh)
Liquid culture
Solid Media
        Want
Confirm pSB1A3 has RFP in it
Pick colony from plates, liquid culture. (COMPLETED)(SCOTT)  “MFC 9.15.2010 C1 “
Thursday: Mini-prep, off to sequencing. Ask Ryan or Megan
    ii     Aliquot from well, PCR (COMPLETED, DEBIKA, left in right side block overnight)
Thursday: gel purify, off to sequencing. Ask Ryan or Megan
 
Detailed Plan
Today (wed)
Critical Actions
hyb.ompa.rfp construct
Start PCR of RFP using new primer using a plasmid with ryan’s rfp (pcold and rfp, or the gaucher lab rfp plasmid) (Debika, completed)
 
Want Actions
confirm PSB1A3 has RFP
Scott
Pick colony from original psb1a3 plates  “MFC 9.15.2010 C1 “ and put in liq culture (completed)
Debika
Start PCR of psb1a3 directly from well, using primers RFP-F and RFP-R (Debika, completed)
Christina, Debika,
transformation of NB cells using original psb1a3 aliquot (completed)
 
Other Actions
order psb1a7 sequencing primers (the forward psb1a3-F primer is ok, but we need a reverse primer)
 
Thursday
perform a heat shock transformation of psb1a7 -- this could not be done on Wed. because no plates were left to perform this. Had to make plates first.
Add 1uL of resuspended psb1A7 in 10uL of competent cells (Novablue). (thawed on ice)
See Protocols page for Heat Shock Transformation
 
Mini-prep of psB1A3, send sequence w/ RFP F,R
Gel-purify psb1A3 pcr, send off to sequencing w/ RFP F,R
Friday
Start liquid culture of psB1A7
 
Digest - Debika (12-2 pm)
Ligate, transform - Mitesh, Scott (5pm)
Saturday
Colony PCR of constructs (10 each)
Mini-prep of psb1A7- Scott (9am)
Run gel
hybB F, AOX R
hybB F, ompA R
ompA F, AOX R
 
Need controls for everything from now on!
 
Controls: (Can’t read Scott’s handwriting, sorry Scott! This stuff is still on the whiteboard in the lab)
Ligate
leave out a piece of the ligation so we know it won’t work (- control)
continue to use this throughout the transformation
Transformation control
Just digest psB1A7 use it to transform (negative control
 
10/7/2010 (Thursday)
GOALS:
 
perform a heat shock transformation of psb1a7 -- this could not be done on Wed. because no plates were left to perform this. Had to make plates first. (completed Christina, Rob)
Mini-prep of psB1A3 (completed, Christina, Rob)
send sequence w/ RFP F,R
Gel-purify psb1A3 pcr, send off to sequencing w/ RFP F,R
Make gel in Hammer Lab Chamber (the orange one) (gel completed, christina)
Gel tray is max 80 or so ml
perform gel extraction (I can do this in afternoon, Scott)
 
Results and Observations from plates made on 10-6-10:
psb1a3 had growth; no pink colonies (may be slight, can ask Ryan as she is expert)
made a starter culture from these plates and see if they turn pink.
psb1a7 had no growth - plates thrown out.
 
Protocols:
 
Moved the plates of the psb1a7, psb1a3 transformations to fridge -- psb1A7 plates thrown out. (Scott)
 
Make starter cultures from psb1A3/NB plates made on 10/6/10.(Christina)
 
perform a heat shock transformation of psb1a7 -- this could not be done on Wed. because no plates were left to perform this. Had to make plates first (Christina)
Add 1uL of resuspended psb1A7 in 10uL of competent cells (Novablue). (thawed on ice)
See Protocols page for Heat Shock Transformation
 
Mini-prep of psB1A3 from starter cultures grown on 9-6-10 (Christina)
 
Autoclaved a variety of pipette tip boxes to stock up(Christina)
 
make a new 1% gel in the orange pcr chamber (the chamber is the casting tray -- GENIUS!) (Christina)
See Protocols page for Preparing 1% Agarose Gels
 
PCR purifcation of psb1a3 pcr wiith RFP-F, RFP-R
See Protocols page for PCR Purifacation
 
Gel extraction of PCR from psb1a3 using primers RFP-F, RFP-R (Scott, Gita)
See Protocols page for Gel Extraction
Note: To elute DNA, added 30  μL water (pH 7.0 – 8.5), let the column stand for 1 min, and then centrifuged for 1 min.
 
RE Digest Recipe for RFP-F3R (that Debika did this yesterday)(Christian is doing the digest today)
12.5 uL H20
5uL 10X Promega Buffer B
6 uL RFP-F3R (167.4 ng/uL 10.6.2010)
5 uL BSA (1ug/uL, to a final conc of .1mg/ml)
0.75  uL SpeI
0.75  uL XmaI
Total=50 ul total
Run 3 hours in the heating block in our lab (37c, put water in the heating blocks)
Start 5pm
 
10/8/2010
Goals
Nanospec the gel extract done on 10/7/2010 (of psb1a3 RFP-FR, 700 bp band)
Run 4 uL of the gel extract on the gel and take picture (to prove psb1a3 has rfp!)
Send off for sequencing:
gel extract with RFP FR
the mini prep of psb1a3 with RFP FR (mini prep from thursday)
Repeat transformation of psb1a7!
PCR purification of digest from (10/7/2010) of RFP F3R
See Protocols page for PCR Purification
run on gel to check for 700 bp band
if ok, then
ligate to Hyb.ompa and psb1a3 or psb1a7 (which vector)?
Observations
The transformation of psb1a7 into NB did not work. We should check whether the strain is compatible with the plasmid.  
The colonies from the transformation of psb1a3 directly from well into NB have turned slightly pink.
Liquid cultures of psb1a3 colonies from plates from 10.6.2010 appear brown with no pink. However, when we pelleted them for miniprep, the pellet was pink!
 
minprep of psb1a3 liq culture from palte from10.6.2010
 
Transformation of pSB1A7 in NB cells (with + control)
Megan suggested we use fresh NB cells (from Gaucher cells), use a positive control (known good plasmids)
2 reactions:
10 սL Nova Blue cells + 5 սL of mRFP from Gaucher Lab (labeled mRFP mini prep 3/11/09)
10 սL Nova Blue cells + 2սL of psb1a7 from  well aliquot
See Protocols page for Heat Shock Transformation.
 
PCR purification of digest from (10/7/2010) of RFP F3R (Gita)
See Protocols page for PCR Purification.
notes: elute in 30 uL!
 
Nanospec
purified digest of RFP-F3R (labeled with 10.8.2010) = 18.1 ng/uL
Gel extract of RFP FR (tube labeled 1, date 10.7.2010) = 84 ng/uL
Gel extract of RFP FR (tube labeled 2, date 10.7.2010) = 91.3 ng/uL
psb1a3 mini prep
tube 1: 64 ng/uL
tube 2 :56 ng/uL
 
Run on 1% gel
Gel extractions from 10.7.2010 of psb1a3+ RFP-FR (2 tubes)
purified, digested RFP-F3R from today, 10.8.2010
 
 
For tomorrow
high priority
pick colonies of psb1a7 (if succesful) and transfer to liquid culture
tranform cells using psb1c3, since it is the vector we have to submit in.
ligate RFP F3R to hyb.ompa and psb1a7 (likely have to be done sunday)
autoclave more water
Find some chloramphenicol since we have to submit parts in psb1c3 and it has chloramphenicol resistance
low priority
run pcr on psb1a3 minipreps today to check for rfp
 
10/9/2010
Goals
Troubleshoot psb1a7, psb1a3
List of wells, inserts, and sequences ( http://partsregistry.org/assembly/libraries.cgi?id=31 )
Notes
Troubleshoot psb1a7
does it have a cbbd toxic gene in it?
No, it has an insert coding for a antibody chain, part: K126000
( http://partsregistry.org/wiki/index.php?title=Part:BBa_K126000 ) (http://partsregistry.org/assembly/plates.cgi?id=1259)
 
can we digest (is it digested) and ligate it directly from the well to our parts?
should we pcr it first?
do we have a ccdb tolerant strain?
 
Trobuleshoot psb1a3
psb1a3 has RFP (http://partsregistry.org/partsdb/get_part.cgi?part=pSB1A3)
BBa_J04450 = RFP
tranform cells using psb1c3, since it is the vector we have to submit in.
Have plasmid backbone from kit (Where is it?)
Has RFP insert for in well Plate 1 - A3 ( http://partsregistry.org/assembly/plates.cgi?id=1257 )
 
Protocols
RE Double Digest Recipe for pSB1A3 (from 10/8/2010)
6.5 uL H20
3uL 10X Promega Buffer E
16 uL pSB1A3 (10.8.2010, 64 ng/uL)(tube 1)
3 uL BSA (1ug/uL, to a final conc of .1mg/ml)
0.75 uL SpeI
0.75 uL EcoRI
Total=30 ul total
Run 3 hours in the heating block in our lab (37c, put water in the heating blocks)
Predict a insert (1000 bp) and plasmid (2000 bp)
RE Digest Recipe for pSB1A3 (from 10/8/2010)
5.0 uL H20
2.5 uL 10X Promega Buffer E
16 uL pSB1A3 (10.8.2010, 50 ng/uL)(tube 2)
3 uL BSA (1ug/uL, to a final conc of .1mg/ml)
0.75 uL SpeI
Total=30 ul total
Run 3 hours in the heating block in our lab (37c, put water in the heating blocks)
Digest start::12pm
For Tomorrow:
PREFORM PCR on WHITE colonies to check for what is in them: triple ligation plates from Oct
Ask Gaucher lab if they have 1) ccdb resistance strains 2) vector lacking promoters and have amp resistance 3) the three tubes of plasmid backbone from the standard iGem kit
Need more Promega Buffer E
 
Retrieved from " http://2010.igem.org/Team:GeorgiaTech/WeekTen "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
