"
Page
Discussion
View source
History
teams
Log in
 
Team:GeorgiaTech/Project
From 2010.igem.org
Georgia Institute of Technology iGEM Team 2010 Project
Overview
Microorganisms respond to environmental stress through several means for survival. This research project aims to engineer Escherichia coli cells to generate heat in response to a cold-shock. Altering a cell’s reaction to external variations prolongs its ability to survive such a shock, thereby ensuring an evolutionary advantage over other species. Studying and manipulating a prokaryote’s stress response can broaden the prospective on evolutionary mechanisms in general. This research venture will advance biological engineering by furthering current attempts at constructing and understanding complex gene networks. Applications of synthetic, thermogenerating bacteria range from the investigation of cold resistance evolution to the development of a new generation of inexpensive biosensors for environmental monitoring and contaminant detection. The experiment will focus on producing an over-expression of alternative oxidase (AOX), an enzyme associated with thermogenesis in plants. In the future, we hope to link a decrease in ambient temperature with a “huddling” cell aggregation leading to biofilm formation.
Background
The synthesis of ATP is coupled with the flow of electrons through the electron transport chain to a terminal electron acceptor, terminal oxidase via ubiquinone and cytochromes. In addition to this pathway, all higher plants contain a cyanide-resistant pathway involving an enzyme, alternative oxidase. In this route, electrons branch from the cytochrome pathway at ubiquinone (before cytochrome) and terminate with an alternative oxidase.
Unlike the free energy from the cytochrome pathway, the free energy generated by the flow of electrons from ubiquinone to alternative oxidase does not result in the generation of ATP but instead is lost as heat. Due to an increase in the alternative oxidase activity in certain plants, heat is generated (up to 20ºC above ambient) to volatilize foul-smelling attractants for insects engaged in the pollination process [1][2]. The occurrence of AOX is found in plants, as well as in many fungi and protists.
Arabidopsis thaliana is widely researched partly due to the breadth of knowledge on the plant including the completion of its genome sequence. Additionally, its AOX pathway has been extensively studied. The two major classes of AOX genes are expressed in mRNA, AOX1 and AOX2. The AOX1 gene family is stress induced and found in both monocotyledons and eudicotyledons while the AOX2 gene family is prominent only in eudicotyledons and expression depends on the tissue and developmental stage [3]. It has been demonstrated that over-expressing AOX1 leads to large increases in AOX protein and alternative pathway capacity in tobacco cells [4]. The presence of thermogenic monocots (such as Arum maculatum) also indicates that in simple prokaryotic systems, AOX1 expression would be sufficient for thermogenesis.
Objectives and Techniques
Cloning AOX genes into E.coli
Kumar and Soll have demonstrated that the gene for alternative oxidase from Arabidopsis thaliana can be cloned into E. Coli [5]. We chose, however, to use AOX from a thermogenic plant, the Voodoo lily, which can produce temperatures 20c above ambient. Two forms of Aox will be synthesized for bacterial expression by the company Mr. Gene. The forms are overexpressed in certain tissues during the thermogenesis event in Voodoo Lily.
Strains and Growth Conditions
The E. coli strains used will be Novablue and the expression strain BL21. The strains will be grown on LB media. Ampicillin(100 ug/ml) will be added to the growth medium when indicated.
Overexpressing AOX
The expression level of AOX corresponds to the amount of heat produced. AOX protein content in the thermogenic stages of flowering was at least 10-fold higher than that of pre and post-flowering stages, when heat is not produced [6]. To produce high levels of a protein, it is often useful to clone the gene downstream of a well characterized promoter. Inducing transcription from the regulated promoter thus results in elevated expression of the downstream gene product. By choosing a temperature-sensitive promoter, AOX production can be regulated and stopped once a cell has reached an optimal temperature. E.coli’s major cold shock protein CS7.4, is cold shock-induced via the temperature sensitive cspA promoter [7]. Other promoters that are more active around 15-20°C than at 37°C with a 100-fold or greater increase in gene expression have been identified. They include nusA, infA, and infB of the nus-inf operon, pnp of the S15 operon, recA, and aceE,F, and hypbB [8]. Qoronfleh et. al. have demonstrated that naturally occurring cold-inducible promoters in E. coli could be exploited to develop more effective systems for low-temperature expression of heterologous proteins in plasmid-based vectors. Based on previous characterization by UCSF in 2005, we chose the HybB promoter due to distinct on/off states between 37°C and 20°C.
Future Focus
Biofilm Formation
In a large number of microbial species, non-optimal growth conditions and cellular stress stimulate biofilm formation [9]. In E. coli, lower temperatures promote several adhesive factors [10]. These factors are largely dependent on a stress-response gene, RpoS, initiated in the σs subunit of RNA polymerase. Translation of σs increases at 20°C suggesting that RpoS-dependent gene products play a role in cellular adaptation to a cold shock [11]. As a defense mechanism against an environmental stress, cells can produce various extracellular polysaccharides (EPS) that can be manipulated for adhesion and biofilm formation. One EPS signal molecule regulated by RpoS is cyclic-di-guanosine monophoshpate (c-di-GMP). c-di-GMP is synthesized by diguanylate cyclases (DGCs) which is associated with the GGDEF domain [12]. The GGDEF protein YddV can stimulate cell aggregation and EPS production via its diguanylate cyclase activity. Overexpression of the YddV protein results in a major rearrangement of the bacterial cell’s global gene expression pattern causing the culture to transition from planktonic to biofilm growth through upregulation of EPS-related genes and RpoS [13]. YddV increases EPS production through stimulation of specific gene transcriptions leading to expression of the protein CsgD. CsgD activates curli fiber production, which is an adhesion factor in E. coli.
Cold shock induced biofilm formation can be heightened by a mutation in the ompR gene, ompR234. This mutation initiates transcription of the CsgD protein thereby resulting in an increase in biofilm formation [14]. The resulting cold activated biofilm formation will be paired with expression of the alternative oxidase (AOX1) gene. Pairing cellular group formation with heat production will provide evolutionary insight and allow for the engineering of microorganisms holding utility for institutions such as the Department of Energy and Department of Defense.
Attributions and Contributions
Soll et al had demonstrated successful cloning of AOX from Arabidopsis into E. coli. However, Arabidopsis does not use AOX as a significant source of heat generation. As Georgia Tech’s inaugural iGEM team, we chose to clone the AOX gene from Voodoo lily into bacteria because the AOX pathway plays an important role in thermogenesis in this plant species. Previous work done by the UCSF iGEM team of 2005 helped us determine hyBb as an optimal cold-shock promoter for our experiments.  We would like to thank our advisors Dr. Gaucher, Dr. Weitz, Dr. Styczynski, Dr. Cole and graduate  mentors Richard In-Ho Joh, Ryan Randall, and Catherine Rivet.
References
[1] Meusse B. “Thermogenic Respiration in Aroids.” Annual Review of Plant Physiology 26  (1975): 117-126.  [2] Wagner A et al. “Regulation of Thermogenesis in Flowering Araceae: The Role of the Alternative Oxidase.” Biochimica et Biophysica Acta 1777 (2008): 993-1000. [3] Juszczuk I, Rychter A. “Alternative Oxidase in Higher Plants.” Acta Biochemica Polonica 50 (2003): 1257–1271. [4] Maxwell D, Yong W, McIntosh L. “The Alternative Oxidase Lowers Mitochondrial Reactive Oxygen Production in Plant Cells.” Proceedings of the National Academy of Sciences 96 (1999): 8271-8276. [5] Kumar A, Soll D. “Arabidopsis Alternative Oxidase Sustains Escherichia Coli Respiration.” Proceedings of the National Academy of Sciences 89 (1992): 10842-10846.  [6] Grant NM, Miller RE, Watling JR, Robinson SA. “Synchronicity of the Thermogenic Activity, Alternative Pathway Respiratory Flux, AOX Protein Content, and Carbohydrates in Receptacle Tissues of Sacred Lotus During Floral Development.” Journal of Experimental Botany 59 (2008): 705-714.  [7] Tanabe, H et. al. “Identification of the Promoter Region of the Escherichia coli Major Cold Shock Gene, cspA.” Journal of Bacteriology 174 (1992) 3867-3873. [8] Qoronfleh M, Debouck C, Keller J. “Identification and Characterization of Novel Low- Temperature-Inducible Promoters of Escherichia coli.” Journal of Bacteriology 174 (1992) 7902-7909.  [9] Landini P. “Cross-talk Mechanisms in Biofilm Formation and Responses to Environmental and Physiological Stress in Escherichia coli.” Research in Microbiology 160 (2009): 259-266. [10] Romling U, Sierralta W, Eriksson K, Normark S. “Multicellular and Aggregative Behaviour of Salmonella typhimurium Strains is Controlled by Mutations in the agfD Promoter.” Molecular Microbiology 28 (1998): 249-264. [11] Vaillancourt P. “E. coli Gene Expression Protocols.” Methods in Molecular Biology 205 (2003): 1-7.  [12] Pruss B, Besemann C, Denton A, Wolfe A. “A Complex Transcription Network Controls the Early Stages of Biofilm Development by Escherichia coli.” Journal of Bacteriology 188 (2006): 3731–3739.  [13] Mendez-Ortiz M, Hyodo M, Hayakawa Y, Membrillo-Hernandez J. “Genome-wide Transcriptional Profile of Escherichia coli in Response to High Levels of the Second Messenger 30,50-Cyclic Diguanylic Acid.” Journal of Biological Chemistry 281 (2006): 8090-8099. [14] Brombacher E, Dorel C, Zehnder A, Landini P. “The Curli Biosynthesis Regulator CsgD Co-ordinates the Expression of Both Positive and Negative Determinants for Biofilm Formation in Escherichia coli.” Microbiology 149 (2003): 2847-2857.
Retrieved from " http://2010.igem.org/Team:GeorgiaTech/Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
