<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BostonU_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BostonU/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="menuwrap"><UL id="menubartable" class="menu-type"><LI class="menu-column"><A href="https://2017.igem.org/Team:BostonU/Team" class="dropbtn">People</A><DIV class="dropdown-content"><A class="menuitem" href="https://2017.igem.org/Team:BostonU/Team">Team</A><A class="menuitem" href="https://2017.igem.org/Team:BostonU/Attributions">Attributions</A></DIV></LI><LI class="menu-column"><A href="https://2017.igem.org/Team:BostonU/Description" class="dropbtn">Research</A><DIV class="dropdown-content"><A class="menuitem" href="https://2017.igem.org/Team:BostonU/Description">Overview</A><A class="menuitem" href="https://2017.igem.org/Team:BostonU/Experiments">Experiments</A><A class="menuitem" href="https://2017.igem.org/Team:BostonU/Results">Results</A><A class="menuitem" href="https://2017.igem.org/Team:BostonU/Model">Modeling</A><A class="menuitem" href="https://2017.igem.org/Team:BostonU/Notebook">Notebook</A><A class="menuitem" href="https://2017.igem.org/Team:BostonU/Protocols">Protocols</A><A class="menuitem" href="https://2017.igem.org/Team:BostonU/Safety">Safety</A></DIV></LI><LI class="menu-column"><A href="https://2017.igem.org/Team:BostonU/Engagement" class="dropbtn">Beyond the Bench</A><DIV class="dropdown-content"><A class="menuitem" href="https://2017.igem.org/Team:BostonU/HP/Silver">Human Practices</A><A class="menuitem" href="https://2017.igem.org/Team:BostonU/HP/Gold_Integrated">Integrated Human Practices</A><A class="menuitem" href="https://2017.igem.org/Team:BostonU/Engagement">Public Engagement</A><A class="menuitem" href="https://2017.igem.org/Team:BostonU/Collaborations">Collaborations</A><A class="menuitem" href="https://2017.igem.org/Team:BostonU/InterLab">InterLab Contribution</A></DIV></LI><LI class="menu-column"><A href="https://2017.igem.org/Team:BostonU/JudgingCriteria" class="dropbtn">Achievements</A><DIV class="dropdown-content"><A class="menuitem" href="https://2017.igem.org/Team:BostonU/JudgingCriteria">Overview</A><A class="menuitem" href="https://2017.igem.org/Team:BostonU/Parts">Parts</A><A class="menuitem" href="https://2017.igem.org/Team:BostonU/JudgingCriteria">Judging Criteria</A></DIV></LI></UL></DIV><DIV id="backgroundimage1"><DIV class="background-gradient-down"><P class="wide-heading-type mainwrap align-center">EXPERIMENTS</P></DIV></DIV><DIV id="panel1" class="link-slideup"><P class="inline-heading-type mainwrap">Characterizing Our Cell Free System</P><P class="body-type mainwrap">
In order to characterize the fluorescence capabilities of our cell-free system, we measured the fluorescence from a plasmid coding for a  constitutively active deGFP at varying concentrations. Our plasmid of choice was pBEST, which comes from the Noireaux Lab, whose cell-free protocol we used to make our in house cell-free. pBEST was shown by the Noireaux lab to have high performance in cell-free [1]. The plasmid was designed modularly, so each part could be replaced using a simple digestion ligation reaction. </P><P class="body-type mainwrap"> </P><P class="body-type mainwrap">
The plasmid was added to cell-free at 0 nM, 10 nM, 20 nM, 30 nM, and 40 nM concentrations. Fluorescence was measured over eight hours. The data was then used to inform our <A href="https://2017.igem.org/Team:BostonU/Model">model</A>.</P><P class="body-type mainwrap"> </P><P class="inline-heading-type mainwrap">Characterizing Toehold Activity Our Cell Free System</P><P class="body-type mainwrap">We began by adapting our source plasmid to contain a toehold as its RBS. We did this by using primers: one set of master primers, which add in XbaI sites used to clone our final product into our final plasmid. The other set of primers added in the toehold or trigger architecture. The toehold switch and trigger sequences came from the best performing forward engineered toehold switch designed by Green et al [2]. We combined the two PCR products into a single linear piece of DNA using overlap extension PCR. </P><P class="body-type mainwrap"> </P><P class="body-type mainwrap">After this step, our plan was to test these linear PCR products in the cell-free transcription translation system. We wanted to test linear fragments beacuse it would drastically cut down on time and cost, and thus improve throughput. However, we observed basal levels of expression from linearized deGFP compared to the plasmid deGFP. After performing a literature search, we hypothesized that either our linear fragments were being degraded by exonucleases present in our cell-free system or RNAse present in our cell-free system degrading our RNA. In order to prevent exonuclease activity, we added purified gamS protein to our cell-free, which inhibits exonuclease activity. We also added commercial RNase inhibitor. However, after running cell-free tests with gamS, RNAse inhibitor, and linear DNA, we still saw only basal levels of expression. From this point onward we tested all our constructs in cell-free as plasmids. </P><P class="body-type mainwrap"> </P><P class="body-type mainwrap">We moved on to adding our triggers as plasmid DNA. This resulted in expression higher than basal levels, however fluorescence still failed to reach the level of the constitutively active deGFP gene and was actually quite low.We hypothesized that too much of the cell-free's transcriptional machinery was being allocated to transcribing the trigger DNA and not enough was being allocated towards transcribing the toehold. We then decided to add the trigger as RNA instead as DNA. We transcribed our trigger DNA into RNA using the Ampliscribe T7 Flash Transcription Kit before adding it into the cell-free system. Adding our trigger as RNA showed a more significant increase in fluorescence. Results from this experiment can be seen on the <A href="#">results</A> page.</P><P class="body-type mainwrap"> </P><P class="body-type mainwrap"> </P><P class="body-type mainwrap">Once toeholds were showing positive results in cell-free, our next goal was to characterize recombinase activity in cell-free. </P><P class="body-type mainwrap"> </P><P class="inline-heading-type mainwrap">Characterizing Recombinase Activity in Our Cell Free System</P><P class="body-type mainwrap">In order to determine how recombinases function in cell-free, we obtained a commercially available Cre recombinase protein from New England Biolabs. We designed a reporter plasmid with the same design as pBEST, but with a premature terminator before the deGFP gene. This terminator was flanked with recombinase recognition sites, and in the presence of Cre should be excised, allowing for deGFP expression. The figure below shows the reporter architecture.</P><P class="body-type mainwrap"> </P><P class="body-type mainwrap">We set up a cell-free reaction in which we added one unit of the Cre recombinase to the reporter plasmid. We also added one unit of the Cre recombinase to a positive control reaction containing a constitutive deGFP plasmid. The reporter with Cre showed only background fluorescence as compared to a reaction with no DNA. The Cre with the constitutive deGFP plasmid showed only 25% fluorescence as compared to a constitutive deGFP reaction with no Cre. You can view these results <A href="#">here</A>. </P><P class="body-type mainwrap"> </P><P class="body-type mainwrap">To try to avoid the decrease in expression caused by cell-free, we attempted running the recombination reaction on the reporter before running the cell-free experiment. However, we saw similar poor levels of expression. Alterations to the protocol did not improve our results. Since we hypothesized that some component of the solution in which Cre recombinase was stored was affecting cell-free efficiency, our next experiments aimed at characterizing cell-free recombinase activity from a plasmid.</P><P class="body-type mainwrap"> </P><P class="body-type mainwrap">For this set of experiments, we used a BxbI recombinase in the pBEST plasmid. We exchanged the deGFP with the BxbI sequence, achieving a constitutive BxbI plasmid. The reporter we used was a promoter inversion promoter. In the presence of BxbI, the formerly inverted promoter should be moved into the proper orientation, allowing for deGFP expression. </P><P class="body-type mainwrap"> </P><P class="body-type mainwrap">We ran a cell-free reaction with the constitutive BxbI plasmid added to the reporter plasmid. Again, this only showed background expression as compared to a reaction containing no DNA. A reaction containing the constitutive BxbI plasmid added to the constitutive deGFP showed fluorescence at about 66% compared to a reaction with just the constitutive deGFP. View these results <A href="#">here</A>.  We believe that this is indicative that both the deGFP and BxbI recombinase are being transcribed and translated. Because of limited machinery, the deGFP expression would be decreased to allow for BxbI expression. Future work will be aimed at proving this hypothesis. In addition, a literature search revealed that only a small subset of previously tested recombinases were shown to have functionality in cell-free [3]. Future work will also aim at discovering which recombinase show the best functionality in our cell-free system.</P><P class="body-type mainwrap"> </P><P class="inline-heading-type mainwrap">Additional Experiments and Future Directions</P><P class="body-type mainwrap">A large amount of time during our project was devoted to developing a more modular version of the pBEST plasmid in order to allow for easy exchange of toehold sequences and genes and the addition of fluorescent fusion proteins. We wanted to tag our recombinases with fluorescent proteins so that we could monitor their level of expression in our cell-free system. Though we were not able to complete this, here we provide a detailed methodology of our plans and what could be accomplished in the future: </P><P class="body-type mainwrap"> </P><P class="body-type mainwrap">Using the pBEST plasmid as a foundation, we introduced two linker regions so that we could add in any toehold regardless of the downstream coding sequence as well as being able to create a fusion protein in any combination. This should not affect the coding sequences, but provide enough universal nucleotides for any primer to attach to and facilitate overhang PCRs. The first linker comes between the toehold and the coding sequence. Our toehold switches are sourced from the Green et. al. forward engineered switch, all of which have a 21-nt linker between the switch and the start codon of the downstream gene. We replaced the 5 bases that were originally between the RBS and start codon with this universal 21-nt linker. The linker is composed of low-molecular-weight amino acids. However, we needed to make a slight alteration to preserve the NcoI site between the ribosome binding site and coding seqeunce, which we use for cloning. </P><P class="body-type mainwrap"> </P><P class="body-type mainwrap">With the desire to create fusion proteins in mind, our second region consisted of 30-nt sourced from within our lab and verified to work in dozens of constructs. This would allow us to tag the recombinases with fluorescent proteins and track their synthesis overtime. At the end of our time working on the iGEM project, we were working on characterizing the effects of adding these linker regions to the pBEST plasmid’s activity in cell-free. Once this is verified, we could tag our recombinase proteins and verify that they are being expressed in the cell-free system. From here we can better troubleshoot our system and gain functional recombinases in cell-free. </P><P class="body-type mainwrap"> </P><P class="body-type mainwrap">Finally, we would exchange the deGFP in our toehold switch plasmids with the functional recombinase. This would accomplish our overall goal of using RNA detection to drive recombinase based logic.</P><P class="body-type mainwrap"> </P><P class="body-type mainwrap">[1] Shin, Jonghyeon, and Vincent Noireaux. &quot;Efficient cell-free expression with the endogenous E. Coli RNA polymerase and sigma factor 70.&quot; Journal of biological engineering 4.1 (2010): 8.</P><P class="body-type mainwrap"> </P><P class="body-type mainwrap">[2] Green, Alexander A., et al. &quot;Toehold switches: de-novo-designed regulators of gene expression.&quot; Cell 159.4 (2014): 925-939.</P><P class="body-type mainwrap"> </P><P class="body-type mainwrap">[3] Artavanis, Georgios, et al. &quot;The role of single occupancy effects on integrase dynamics in a cell-free system.&quot; bioRxiv (2016): 059675.</P><P class="body-type mainwrap"> </P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>