<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Hong_Kong_HKU_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Hong Kong HKU/Results</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><HEADER id="main_menu"><NAV><UL><LI id="line"><A href="https://2015.igem.org/Team:Hong_Kong_HKU">HOME</A></LI><LI id="line"><A href="https://2015.igem.org/Team:Hong_Kong_HKU/Team">TEAM</A></LI><LI id="line"><A href="#">PROJECT</A></LI><LI style="margin-top: 3px"><A href="https://2015.igem.org/Team:Hong_Kong_HKU/Description">Description</A></LI><LI style="margin-top: 3px"><A href="https://2015.igem.org/Team:Hong_Kong_HKU/Experiments">Experiments</A></LI><LI style="margin-top: 3px"><A href="https://2015.igem.org/Team:Hong_Kong_HKU/Results">Results</A></LI><LI style="margin-top: 3px"><A href="https://2015.igem.org/Team:Hong_Kong_HKU/Design">Design</A></LI></UL><LI id="line"><A href="#">PARTS</A></LI><LI style="margin-top: 3px"><A href="https://2015.igem.org/Team:Hong_Kong_HKU/Parts">Team Parts</A></LI><LI style="margin-top: 3px"><A href="https://2015.igem.org/Team:Hong_Kong_HKU/Basic_Part">Basic Parts</A></LI><LI style="margin-top: 3px"><A href="https://2015.igem.org/Team:Hong_Kong_HKU/Composite_Part">Composite  Parts</A></LI><LI id="line"><A href="https://2015.igem.org/Team:Hong_Kong_HKU/Modeling">MODELING</A></LI><LI id="line"><A href="https://2015.igem.org/Team:Hong_Kong_HKU/Notebook">NOTEBOOK</A></LI><LI id="line"><A href="https://2015.igem.org/Team:Hong_Kong_HKU/Attributions">ATTRIBUTIONS</A></LI><LI id="line"><A href="https://2015.igem.org/Team:Hong_Kong_HKU/Practices">HUMANPRACTICES</A></LI><LI id="line"><A href="https://2015.igem.org/Team:Hong_Kong_HKU/Safety">SAFETY</A></LI></NAV></HEADER></P><DIV id="globalWrapper"><DIV id="content"><DIV id="contentcontainer"><H3>RESULT</H3><H2>Design and Order gBlock</H2><P>After the construction and optimization of our killing switch system that aims to work in E. coli BL21(DE3), we originally planned to use the BioBricks provided and start transformation and cloning. Yet, we had difficulties in transformation some major components like pC (BBa_J23106), cas9 (BBa_K1218011), not to mention that pR (BBa_R0051 with pSB1A2) could be hardly grown on Amp<SUP>R</SUP> plate and nicely on CM30 plate. We had to request for those BioBricks from HQ which indeed took much time for shipping to Hong Kong. Throughout 1-month transformation work, we did not have much ready-to-use BioBricks at the end. Read <A href="https://static.igem.org/mediawiki/2015/8/8c/Materials_at_hand.pdf">Biobriack Documantation HKU2015</A> to see what BioBricks we had done experiments with and the results.</P><P> In order to finish the project on time, we abandoned the cloning method and turned to gBlock, a synthetic double-stranded DNA product provided by IDT company. Our whole construct was divided into 5 gBlock fragments and the gBlocks were ordered. This strategy was chosen since our design included several new parts such as Cas9 and guiding RNA that target specific sequences. The whole construct also involved in numerous minor changes, thus traditional cloning and assembly of available biobricks cannot meet our needs.</P></DIV><DIV id="contentcontainer"><H6 style="text-align: center;">Fig 1. Span of gBlock corresponding to the killing switch construct. For easy understanding, the length of DNA fragments in the figure are not in actual proportion. </H6><P>These gBlocks are DNA fragments synthesized according to our order, and can be later assembled by Hi-Fi assembly. Fig. 1 shows the span of the gBlock fragments and their length in base pair. gBlock 1 initiates with prefix and gblock 5 ends with suffix, thus after assembly with pSB1C3 backbone, the whole construct forms a circular plasmid and can be transformed into targeted organisms.</P><H2>Design and Order gBlock Primer</H2><H6 style="text-align: center;">Fig. 2  Relative position of primers and gBlocks</H6><P>While ordering the gBlock DNAs, we also designed and ordered the 10 forward and reverse primers that matched the 5 gBlocks respectively in order to amplify them through PCR. Fig. 2 illustrates the relative positions of gBlocks and primers in a circular plasmid. Since the forward/ reverse primers have overlapping regions on the preceding/ following gBlocks, after amplification all fragments would have overlapping sequences that enable Hi-Fi assembly. The sequences and working temperatures of each primers are provided in the table below. All primers were designed to have around 40 base pairs to provide enough specificity for the annealing DNA. Yet the arrangement exceeded the optimal length of primer design (which should be 18-24 bp) and the primers might have a higher chance to generate self-dimers or form secondary structure.</P><TABLE id="t01" style="width:100%"><TBODY><TR><TH>Overlaps</TH><TH>Oligo (Uppercase = gene-specific primer) 5'-&gt;3'</TH><TH>Anneals</TH><TH>F/R</TH><TH>3' Tm</TH><TH>3' Ta</TH></TR><TR><TD>gBlock 4</TD><TD><FONT color="#ea662d">tgggcctttctgcgtttataTACTAGAGTTTACGGCTAG</FONT></TD><TD>gBlock 5</TD><TD>Fwd</TD><TD>55.1°C</TD><TD>58.1°C</TD></TR><TR><TD>pSBCc3</TD><TD><FONT color="#454545">tgcagcggccgctactagtaAAAAAAAGCACCGACTCG</FONT></TD><TD>gBlock 5</TD><TD>Rev</TD><TD>58.3°C</TD><TD>58.1°C</TD></TR><TR><TD>pSB1C3</TD><TD><FONT color="#454545">attcgcggccgcttctagagTTATGACAACTTGACGGC</FONT></TD><TD>gBlock 1</TD><TD>Fwd</TD><TD>57.1°C</TD><TD>58.4°C</TD></TR><TR><TD>gBlock 2</TD><TD><FONT color="#eeb400">cgtgcaaataatcaatgtctCTAGTAATAGCAAAGTGTGAC</FONT></TD><TD>gBlock 1</TD><TD>Rev</TD><TD>55.4°C</TD><TD>58.4°C</TD></TR><TR><TD>gBlock 3</TD><TD><FONT color="#0d6a98">aactgtcaaagttgttgatgAATTAGTAAAAGTGATGGGTCG</FONT></TD><TD>gBlock 4</TD><TD>Fwd</TD><TD>57.7°C</TD><TD>59.9°C</TD></TR><TR><TD>gBlock 5</TD><TD><FONT color="#4f854d">gctagccgtaaactctagtaTATAAACGCAGAAAGGCC</FONT></TD><TD>gBlock 4</TD><TD>Rev</TD><TD>56.7°C</TD><TD>59.9°C</TD></TR><TR><TD>gBlock 2</TD><TD><FONT color="#eeb400">caatctgatcgcattgtcgcTGGGTCTGACCCCTAACT</FONT></TD><TD>gBlock 3</TD><TD>Fwd</TD><TD>62.5°C</TD><TD>60.9°C</TD></TR><TR><TD>gBlock 4</TD><TD><FONT color="#ea662d">acccatcacttttactaattCATCAACAACTTTGACAGTTTG</FONT></TD><TD>gBlock 3</TD><TD>Rev</TD><TD>57.9°C</TD><TD>60.9°C</TD></TR><TR><TD>gBlock 1</TD><TD><FONT color="#d40909">tcacactttgctattactagAGACATTGATTATTTGCACGG</FONT></TD><TD>gBlock 2</TD><TD>Fwd</TD><TD>58.5°C</TD><TD>61.5°C</TD></TR><TR><TD>gBlock 3</TD><TD><FONT color="#0d6a98">aaagttaggggtcagacccaGCGACAATGCGATCAGATTG</FONT></TD><TD>gBlock 2</TD><TD>Rev</TD><TD>62.3°C</TD><TD>61.5°C</TD></TR><TR style="background-color: #aeaeae;"><TD>---</TD><TD>CTCTAGAAGCGGCCGCGA</TD><TD>pSB1C3</TD><TD>Rev</TD><TD>68.1°C</TD><TD>67.5°C</TD></TR><TR style="background-color: #aeaeae;"><TD>---</TD><TD>TACTAGTAGCGGCCGCTG</TD><TD>pSB1C3</TD><TD>Fwd</TD><TD>64.5°C</TD><TD>67.5°C</TD></TR></TBODY></TABLE><H6 style="text-align: center;"> Table 1. Sequences of the 10 ordered forward and reverse primers. The lowercase letters correspond to the overlapping sequences and the capital letters represent the annealing sequences of the adjacent gBlock fragments. Tm stands for melting temperature, and Ta stands for annealing temperature. </H6><H2>PCR of gBlock (i)</H2><P>We started PCR amplification after the arrival of both gBlocks and primers. Related figures are included in a table below. For the detailed preparation and concentration of reaction agents, please refer to our <A href="https://2015.igem.org/Team:Hong_Kong_HKU/Notebook">Notebook</A> page.</P><TABLE id="t01"><TBODY><TR><TH>Stage 1</TH><TH colspan="3">Stage 2</TH><TH colspan="2">Stage 3</TH></TR><TR><TD>Time (mm:ss)</TD><TD>0:30</TD><TD>0:10</TD><TD>0:30</TD><TD>1:30</TD><TD>2:30</TD><TD>Infinite</TD></TR><TR><TD>Temperature (degree celcius)</TD><TD>98.0</TD><TD>98.0</TD><TD>Tx</TD><TD>72.0</TD><TD>72.0</TD><TD>4.0</TD></TR><TR><TD>Cycle</TD><TD>1</TD><TD colspan="3">30</TD><TD>1</TD><TD>/</TD></TR></TBODY></TABLE><H6 style="text-align: center;">Table 2. The reaction figures for 1st PCR</H6><P>For Tx, it is 3 degree Celsius higher than the lower Tm in that of the paired forward and reverse primers, to each gBlocks respectively. </P><UL><P>- gBlock 1: 58.4 degree</P><P>- gBlock 2: 61.5 degree</P><P>- gBlock 3: 60.9 degree</P><P>- gBlock 4: 59.9 degree</P><P>- gBlock 5: 58.1 degree</P></UL><P> The result product of our first PCR was run in 1% agarose gel and is provided below.  gBlock 1, 3, 4 and 5 were fine with the band size though some non-specific band is also shown. Yet gBlock 4 was somehow vague to be seen under UV. gBlock 2 showed wrong band size. Gel purification for gBlock 1, 3, 4, 5 was done except for gBlock 5, and their DNA concentrations are low. </P><H6 style="text-align: center;">Fig. 3  The result of the 1st PCR.</H6><P>The correct band size for gBlock 1-5 should be</P><UL><P>- gBlock 1: 1120bp</P><P>- gBlock 2: 1889bp</P><P>- gBlock 3: 1500bp</P><P>- gBlock 4: 2000bp</P><P>- gBlock 5: 683bp</P></UL><H2>PCR of gBlock (ii)</H2><P>PCR was performed once more for gBlock 1, 2, 3 and 4, but this time with DMSO added to eliminate any possible secondary structures that may account for gBlock amplification failure. PCR reaction settings were the same as PCR 1st. The result product of PCR 1st was run in 1% agarose gel and is provided below. gBlock 1, 3, 4 all showed correct band size and the DNA concentration increased compared to the 1st PCR after gel extraction. 
  gBlock 2 still showed wrong band size.</P><H6 style="text-align: center;">Fig. 4 The result of 2nd  PCR.</H6><P>Since we could not gain correctly amplified gBlock 2, the original stock of it was run in agarose gel in order to identify the problem. Yet the results either showed an extremely thin shadow of correct size, or sometimes no band at all. We concluded that there might be some degrees of degeneration of the original stock due to vortex and frequent thawing. Thus, we decided to order gBlock 2 of the same sequence from IDT company again.</P><H2>Hi-Fi Assembly (i)</H2><P>As we were unable to amplify gBlock 2, and the newly synthesized one was not yet arrived, the first Hi-Fi Assembly was done with the purified product of amplified gBlock 1, 3, 4, 5, and the original stock of gBlock 2. Backbone pSB1C3 was also added to complete the construct. Since gBlock 1 and 3, after amplification, carried some overlapping region with the stock gBlock 2, there’s still a possibility for successful Hi-Fi assembly. For detailed preparation and concentration of reaction agents, please see our <A href="https://2015.igem.org/Team:Hong_Kong_HKU/Notebook">Notebook</A> page.</P><H2>Transformation of Hi-Fi Product (i)</H2><P>A positive control was done along with the 1st Hi-Fi assembly. Both products were transformed into DH10B competent cells on CM30 plates with 10 uL of 20% arabinose stock and 10 uL of 0.1% tryptophan stock added on the plates, to prevent the activation of the killing switch system.</P><P>Results were confirmed that only the positive control showed &gt;30 colonies while HiFi product transformation failed. We thus ran a gel to test HiFi products and several bands instead of one single band were observed; this led to the conclusion that the gBlocks didn’t make it to assemble to each other. The photo the result is provided below as Fig. 5.</P><H6 style="text-align: center;">Fig. 5  The result of 1st Hi-Fi assembly </H6><H2>Transformation of Hi-Fi Product (ii)</H2><P>As during the first transformation of Hi-Fi product we didn’t add tryptophan and arabinose during the recovery state, there’s a chance that DH10B might be killed by activated Cas 9 system when it first encountered the plasmids, leading to transformation failure. To exclude this possibility, transformation was done again but with 0.02% arabinose and 0.001% tryptophan added during the recovery step.</P><P> Results were confirmed that 14 small pink colonies were found. We picked the colonies and miniprep was done, with good yielding of DNA concentration. All miniprepped HiFi products were tested by running a 0.7% agarose gel. But each product showed 2 bands of 2000 bp and 3000 bp instead of just a single band of around 9000 bp. We suspected that the competent cells or the HiFi products were contaminated with RFP prior to transformation. Photo is provided below as Fig. 6</P><H6 style="text-align: center;">Fig. 6  The result of 2nd transformation of Hi-Fi product transformation</H6><H2>Hi-Fi Assembly (ii)</H2><P>We wondered if the overlapping regions were too long for assembly of all PCR products of gBlocks that might affect efficiency of HiFi assembly. Therefore, we used the stock of gBlock 2 and gBlock 4 to connect the ampilfied PCR products of gBlock 1, gBlock 3 and gBlock 5 and pSB1C3.</P><P>i.e. (PCR1) + (Stock2) + (PCR3) + (Stock4) + (PCR5) + (pSB1C3)</P><H2>Transform of Hi-Fi Product (ii)</H2><P>Such HiFi product of alternative samples were transformed into DH10B with arabinose and tryptophan added during the recovery step and spread on the CM30 plate. Yet, there were no colony found even the transformation setting was unchanged. We deduced that overlapping region was not an issue to HiFi failure which gave an outcome of wrong band size. We became more convinced that gBlock 2 was somehow degraded.</P><H2>PCR of gBlock (iii)</H2><P>We were almost desperate until the new gBlock 2 arrived on 14 Sep, using DMSO and Q5, we successfully amplified the new gBlock 2 with a correct band size after careful resuspension and sample handling. Fig. 7 showed the band size of around 2000 bp reveals that this gBlock might give a chance for HiFi assembly again before the deadline. The extract DNA after gel purification had a concentration of 12.1 ng/uL and 8.7 ng/uL.</P><H6 style="text-align: center;">Fig. 7 Result of the 3rd PCR of gBlock</H6><P>At the same time, we also tried to examine the band intensity of the PCR products of 5 gBlocks by running a 0.7% agarose gel. As expected, they were of correct band sizes and gBlock 2 and gBlock 4 had a lower intensity than others (see Fig. 8)</P><H6 style="text-align: center;">Fig. 8 Band intensities of all gBlock PCR products</H6><H2>HiFi Assembly (iii)</H2><P>PCR products of the 5 gBlocks were assembled in pSB1C3 and the HiFi products were immediately tested by running a 0.7% agarose gel. The expected band size did not show while we found 4 bands of smaller sizes (see Fig. 9). We doubt whether the HiFi efficiency was too low so the whole product had a critically low concentration, or if we totally failed during HiFi assembly. </P><H6 style="text-align: center;">Fig. 9 Result of the 3rd HiFi assembly</H6><H2>Transformation of HiFi product (iii)</H2><P>It was performed in both DH10B and BL21(DE3), same as previous transformation, arabinose and tryptophan were added right after heat shock and spread on CM30 plate to ensure the killing system was not activated. And there were many colonies grown on the plate for BL21(DE3) while DH10B showed none (see Fig. 10). </P><H6 style="text-align: center;">Fig. 10 Bl21(DE3) plates and DH10B plates showing the result of the 3rd transformation of HiFi product </H6><H2>Transformation of HiFi product (IV)</H2><P>Using the same HiFi product, transformation into DH10B was done again to ensure our construct could be cloned. Still arabinose and tryptophan were added with more careful handling, and there were more than 30 colonies grown on CM30 plates as shown in Fig.11. But unlike BL21(DE3), the number of colonies for DH10B was much smaller than that for BL21(DE3) though here we had got bigger colonies. </P><H6 style="text-align: center;">Fig. 11 Result of the 4th transformation of HiFi product </H6><H2>HiFi product testing</H2><P>10 colonies from BL21(DE3) were picked and the cells were miniprepped. The concentration of the product ranged from 15.7 ng/uL to 61.5 ng/uL. The miniprep product was tested by digestion cut using NcoI and HindIII. It was compared with the uncut one of the same product. For the uncut plasmid, there were bands of 2000 bp, 4000 bp and 7000 bp, we wondered whether this was the result of supercoiling and circular form of the DNA which made it run faster in the gel, assuming the DNA was our HiFi product of around 9000 bp. For the cut plasmid, there were 2 bands of 1000 bp and 2000 bp, rather than 4 expected bands (see Fig. 12) We’re unable to explain such result and need more time in the future to dig out the reason.</P><H6 style="text-align: center;">Fig. 12 Digestion cut of HiFi product compared to uncut product </H6><P>To avoid supercoiling of HiFi product for testing, we performed single digest cut using EcoRI to linearize the plasmid. From the gel photo shown in Fig. 13, all HiFi products showed only one band of 3000 bp. This made us wonder what exactly our HiFi product is, as making it linear did not give a result of a band of 9000 bp.</P><H6 style="text-align: center;">Fig.13 Single digestion cut of HiFi product using EcoRI </H6><P>We had run the gel again using uncut HiFi product, and it shows fragments of 1000 bp, 3000 bp and 7000 bp. Fig. 14 is the gel photo of the uncut plasmid. The fragments were purified for sequencing in future. Our team was eager to know what might go wrong in the HiFi product sequence even though we did not have time to do so before the deadline.</P><H6 style="text-align: center;">Fig. 14 The 3 fragments to be extracted for sequencing in future </H6><H2>Final Step</H2><P>We had freeze-dried HiFi product after miniprep for composite part submission. Since it is a big plasmid, details of the components could be found in the Registry page (BBa_K1774000). Due to limited time, it was a pity that we could not idenify and characterize the Hi-Fi product and make sure about its content. We will do so in the future and isolate other parts in our plasmid. Yet, the details of these parts, including some basic and composite parts such as cas protein and sgRNA, were well documented in registry.
</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>