<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Hong_Kong_HKU_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Hong Kong HKU/Design</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><HEADER id="main_menu"><NAV><UL><LI id="line"><A href="https://2015.igem.org/Team:Hong_Kong_HKU">HOME</A></LI><LI id="line"><A href="https://2015.igem.org/Team:Hong_Kong_HKU/Team">TEAM</A></LI><LI id="line"><A href="#">PROJECT</A></LI><LI style="margin-top: 3px"><A href="https://2015.igem.org/Team:Hong_Kong_HKU/Description">Description</A></LI><LI style="margin-top: 3px"><A href="https://2015.igem.org/Team:Hong_Kong_HKU/Experiments">Experiments</A></LI><LI style="margin-top: 3px"><A href="https://2015.igem.org/Team:Hong_Kong_HKU/Results">Results</A></LI><LI style="margin-top: 3px"><A href="https://2015.igem.org/Team:Hong_Kong_HKU/Design">Design</A></LI></UL><LI id="line"><A href="#">PARTS</A></LI><LI style="margin-top: 3px"><A href="https://2015.igem.org/Team:Hong_Kong_HKU/Parts">Team Parts</A></LI><LI style="margin-top: 3px"><A href="https://2015.igem.org/Team:Hong_Kong_HKU/Basic_Part">Basic Parts</A></LI><LI style="margin-top: 3px"><A href="https://2015.igem.org/Team:Hong_Kong_HKU/Composite_Part">Composite  Parts</A></LI><LI id="line"><A href="https://2015.igem.org/Team:Hong_Kong_HKU/Modeling">MODELING</A></LI><LI id="line"><A href="https://2015.igem.org/Team:Hong_Kong_HKU/Notebook">NOTEBOOK</A></LI><LI id="line"><A href="https://2015.igem.org/Team:Hong_Kong_HKU/Attributions">ATTRIBUTIONS</A></LI><LI id="line"><A href="https://2015.igem.org/Team:Hong_Kong_HKU/Practices">HUMANPRACTICES</A></LI><LI id="line"><A href="https://2015.igem.org/Team:Hong_Kong_HKU/Safety">SAFETY</A></LI></NAV></HEADER></P><DIV id="globalWrapper"><DIV id="content"><DIV id="contentcontainer"><H3 class="title" align="center">BACKGROUND OF CRISPR/CAS 9<P>CRISPR-Cas (clustered regularly interspaced short palindromic repeats, CRISPR-associated) is an adaptive immune system against invasion of foreign DNA commonly found in bacteria and archaea. Just like animals' adaptive immune system, it involve memorization and removal of the invading DNA such as bacteriophage and plasmid DNA. The CRISPR-Cas system is generally encoded by chromosomal cas gene which code for CRISPR-associated proteins and an array of non-coding repeat-spacer sequence consists of sequences complementary to the invading DNA. Expression and post-processing of the array gives rise to a CRISPR RNA (crRNA) which plays a critical role in guiding the CRISPR-Cas complex<SUP>1,2</SUP>.</P><P>The CRISPR-Cas system can mainly be classified into three families. Our project have make use of one of the well-studied type II system, CRISPR-Cas9 originated from <I>Streptococcus pyogenes </I> SF370. Similar to other types of CRISR-Cas system, CRISPR-Cas9 consists of Cas9 endonuclease and crRNA containing 20nt complementary to the target DNA, however different from type I and III, CRISPR-Cas9 also require a trans-activating crRNA (tracRNA) which form a RNA duplex with crRNA (tracRNA:crRNA) to guide the Cas9 endonuclease. Base pairing of tracRNA:crRNA with target sequence immediate upstream of protospacer adjacent motif (PAM) initiates double strand break by RuvC and NHN nuclease domain of Cas9 endonuclease<SUP>1-5</SUP>. By redesigning the 20 nt guiding sequence, Cas9 can be reprogrammed to target plasmid DNA or essential genes to induce lethality.</P></H3><H6 style="text-align: center;">Fig. 1 (a) Schematic of wildtype CRISPR-Cas9 (b) Reprogrammed CRISPR-Cas9 using sgRNA<SUP>6</SUP></H6><P> In the first part of the project, we would like to examine whether CRISPR-Cas9 could fits the role of killing switch. Thus we have designed a sequence that target an essential gene, DNA polymerase III alpha subunit (dnaE) to introduce lethality.  In order to enhance the efficiency of our device, a single guide RNA (sgRNA) targeting DNA polymerase III alpha subunit (dnaE) was adopted instead of using the wild type tracRNA and CRISPR array<SUP>3</SUP>. With the advantage of gBlock DNA synthesis, we have also attempted to improve the translation efficiency by optimizing the Cas9 codon for expression in E. coli using IDT oligo-optimizer.In the future, we would also like to experiment on the effectiveness of Cas9 in destroying plasmid DNA.</P><H3 class="title" align="center">BACKGROUND OF CRISPR/CAS 3<P>CRISPR-Cas3 belongs to the type I system, which makes up of 95% of all CRISPR system<SUP>7</SUP>. Type I system has six different subtype (I-A to I-F), depending on which Cas protein the system encodes<SUP>8</SUP>. The origin of CRISPR-Cas 3 is Escherichia coli K12 substr. MG1655.</P><P>Unlike type II system, type I system requires crispr RNA (crRNA) with 32-nucleotides complementary to the target, complex Cascade (CRISPR associated complex for antiviral defense), and Cas3 protein<SUP>9</SUP>. The Cascade, which comprises of CasA – CasE proteins, is used to process the pre-mature crRNA and bind to the crRNA to locate the target. Cas 3 protein contains the nickase and helicase activity, making it an ideal enzyme to unwind and degrade the target DNA<SUP>8</SUP>.</P><P>In order to degrade the DNA, firstly, the Cascade promotes the binding of crRNA to the target DNA strand, causing the non-target strand looped out. This generates a ‘R-loop’ structure<SUP>9</SUP>. The Cascade then recruits the Cas3 protein, which is activated by the R-loop.  Since the Cas3 protein encodes N-terminal HD nuclease and C-terminal Type A (3’ to 5’) helicase, it unpacks the genes and degrades the target DNA<SUP>7</SUP>. 
</P></H3><H6 style="text-align: center;">Fig. 2 Model of crRNA expression and interference<SUP>8</SUP></H6><P>The crRNA targets the DNA polymerase III alpha subunit. The promoter in design is also changed from constitutive to regulated one. Codon is optimized for better expression in E.coli.</P><H3 class="title" align="center">CIRCUIT DESIGN<P>To ensure the efficacy of the killswitch and minimize erroneous destruction of DNA in bacteria, double regulation is essential as a tight control in our device. Thus, our circuit consists of 2 modules, one is regulated by L-arabinose and the other one is regulated by L-tryptophan. Only when both modules are on, the killswitch would be triggered to destroy the target bacteria and the DNA. </P></H3><H2>(I) Crispr cas 9: cas9 + sgRNA</H2></DIV><DIV id="contentcontainer"><P>The first part of the construct is composed of AraC protein, araBAD promoter and λ repressor protein CI. AraC is a gene regulatory protein which determines the transcription of CI repressor downstream of the promoter pBAD based on the L-arabinose level in the environment<SUP>10</SUP>. pBAD is used as it has low basal expression while maintaining large dynamic range. CI prevents lytic development so as to maintain the lysogenic state<SUP>11</SUP>. In the circuit, CI represses the promoter pR which is essential to perform lytic functions, and thus the expression of cas9 protein.</P><H4><FONT color="blue">OFF state: + arabinose</FONT></H4><UL><H6>1. AraC originally forms a DNA loop after binding trans to the O2 and I1 half-sites<SUP>12</SUP>. When arabinose is present and binds to AraC, the arabinose-AraC complex binds cis to the I1 and I2 half-sites at pBAD instead to unwind the loop (see Fig. 3).</H6><H6>2. pBAD is therefore induced to transcribe CI repressor. At the same time, it allows the transient derepression of pC.</H6><H6>3. Since CI repressor is on, the expression of cas9 protein from pR is prevented.</H6></UL><H6 style="text-align: center;">Fig. 3 Binding of araC to specific half sites in the presence and absence of arabinose<SUP>12</SUP></H6><H4><FONT color="Red">On state: - arabinose</FONT></H4><UL><H6>1. Bacteria escape from the medium and end up somewhere without arabinose supply, and arabinose taken up by the bacteria is continuously consumed by other activities pentose phosphate pathway and binary fission.</H6><H6>2. When the concentration of arabinose drops, the DNA loop is formed. Hence, pBAD is repressed and so does the transcription of CI repressor.</H6><H6>3. The expression of cas9 from pR is therefore induced without the repression of CI, activating the cas system.</H6></UL><P>In the second module, trpR gene and promoter pTrp promoter take their roles in activating or repressing the transcription of sgRNA. trpR codes for a tryptophan-inducible regulatory protein, trp aporepressor<SUP>13</SUP>. Forming a complex with its co-repressor L-tryptophan, trpR is activated and located within pTrp to block the transcription of sgRNA as a result.</P><H4><FONT color="blue">OFF state: + tryptophan</FONT></H4><UL><H6>1. When tryptophan is present in the environment, it binds to trpR regulatory region and activates it (see Fig. 2).</H6><H6>2. The active repressor complex then binds at pTrp so that RNA polymerase cannot bind.</H6><H6>3. This prevents initiation of transcription of sgRNA downstream and switches off the cas system.</H6></UL><H6 style="text-align: center;">Fig. 4 Activation of trpR by tryptophan to switch off gene expression<SUP>14</SUP></H6><H4><FONT color="Red">On state: - tryptophan</FONT></H4><UL><H6>1. When there is a lack of tryptophan in the environment, trpR is inactive.</H6><H6>2. RNA polymerase can then bind to pTrp.</H6><H6>3. Transcription of sgRNA is achieved and this turns the cas system on.</H6></UL><H2>(II) Crispr cas 3: cas3-casABCDE + crRNA</H2></DIV><DIV id="contentcontainer"><P>The 2 modules of this circuit design is basically same as the previous one. For the first part, when there is arabinose, the DNA loop around pBAD is unwinded. Therefore, CI is induced to repress the transcription of cas3 as well as casABCDE. In other words, cas3 and casABCDE are expressed when the environment lacks arabinose, inhibiting the CI repressor and hence inducing pR for transcription of cas proteins. In the second part, with the presence of tryptophan, trpR is activated and it prevents crRNA to be transcribed. In order to express crRNA, tryptophan should be added to initiate the transcription.</P><H5>References</H5><H6>[1] Chylinski, K., Le Rhun, A., &amp; Charpentier, E. 2013. The tracrRNA and Cas9 families of type II CRISPR-Cas immunity systems. RNA Biol. 10:726–737</H6><H6>[2] Ran, F.A. et al. 2013. Genome engineering using the CRISPR-Cas9 system. Nat. Protoc. 8:2281–2308.</H6><H6>[3] Sander, J.D. &amp; Joung, J.K. 2014. CRISPR-Cas systems for editing, regulating and targeting genomes. Nat. Biotechnol. 32:347–355.</H6><H6>[4] M. Jineket al. 2012. A programmable dual-RNA-guided DNA endonuclease in adaptive bacterial immunity. Science. 337:816.</H6><H6>[5] Deltcheva, E. et al. 2011. CRISPR RNA maturation by trans-encoded small RNA and host factor RNase III. Nature. 471:602–607.</H6><H6>[6] Bortesi, L. &amp; Fischer, R. 2015. The CRISPR/Cas9 system for plant genome editing and beyond. Biotechnology Advances. 33:41–52.</H6><H6>[7] Huo, Y et al. 2014. Structures of CRISPR Cas3 offer mechanistic insights into Cascade-activated DNA unwinding and degradation. Nat Struct Mol Biol. 9:771-777.</H6><H6>[8] Rath, D., Lina, A, Rath, A. &amp; Lundgren, M. 2015. The CRISPR-Cas immune system: Biology, mechanisms and applications. Biochimie. 117:119-128</H6><H6>[9] Hochstrasser, M.L..2014. CasA mediates Cas3-catalyzed target degradation during CRISPR RNA-guided interference. Proceedings of the National Academy of Sciences. 111(18): 6618-6623.</H6><H6>[10] Caliando, B.J. and Voigt, C.A. 2015. Targeted DNA degradation using a CRISPR device stably carried in the host genome. Nat. Comm. 6:6989.</H6><H6>[11] Lewis, D. et al. 2011. Multilevel autoregulation of λ repressor protein CI by DNA looping in vitro. PNAS. 108;36:14807-14812.</H6><H6>[12] Schleif, R. 2003. AraC protein: a love-hate relationship. BioEssays. 25:274-282.</H6><H6>[13] Gunsalus, R.P. and Yanofsky, C. 1980. Nucleotide sequence and expression of Escherichia coli trpR, the structural gene for the trp aporepressor. proc. Natl. Acad. Sci. USA. 77;12:7117-7121.</H6><H6>[14] Alberts, B. et al. 2007. Molecular Biology of the Cell. 5th Edition. Garland Science.</H6></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>