<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Stuttgart_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Stuttgart/Protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2017.igem.org/Team:Stuttgart">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Sponsoring">Sponsoring</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Background-Information">Background
                            Information</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Protocols">Protocols &amp; Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/InterLab">Interlab Study</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Model">Modelling</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Composite Part">Composite Part</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Results </A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Esterases and Lipases">Esterases and Lipases</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Keratinases">Keratinases</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Rose and Limonene Fragrance">Rose and Limonene Fragrance</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Demonstrate">Achievements</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/HP/Gold_Integrated">Integrated and Gold</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/HP/Silver">Silver HP</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Awards </A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Measurement">Measurement</A></LI><LI><A href="https://igem.org/2017_Judging_Form?team=Stuttgart">Judging Form</A></LI></DIV></DIV><H1 align="middle"> Protocols &amp; Experiments </H1><DIV class="container"><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><H3>Preparation of chemically competent DH5alpha <I>E. coli</I> cells</H3><H4>Material:</H4><P><UL><LI>LB media</LI><LI>TSS buffer </LI><LI>DH5alpha <I>E. coli</I> cells (o/n colonies on agar plates)</LI></UL></P><H4>Method:</H4><P><OL><LI>Pick one colony of the plate and transfer into 5 mL of LB media. Grow the culture over night for 16-18 hours at 37°C </LI><LI>Transfer 1 mL of the overnight culture into a shaking flask with 99 mL of LB media. Measure optical density (OD) at 600 nm and incubate culture at 37°C (shaking) to an OD of 0,5. </LI><LI>Divide the 100 mL into two 50 mL tubes and incubate 10 min on ice. </LI><LI>Spin the tubes at 3000 rpm for 10 minutes at 4°C </LI><LI>Resuspend the pellet of competent cells with 10 % TSS buffer (5 mL). </LI><LI>Aliquot 100 µL of the cell solution into 1.5 mL microtubes (all steps on ice!). </LI><LI>Store the competent cells at -80°C</LI></OL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><H3>Heat-shock Transformation of <I>E. coli</I> cells</H3><H4>Material:</H4><P><UL><LI>SOC media </LI><LI>Agar plates with appropriate antibiotic</LI></UL></P><H4>Method:</H4><P><OL><LI>Thaw chemically competent cells on ice. </LI><LI>Transfer 50 µL of the cells into a 1.5 mL microtube, add 1 µL of the desired DNA and incubate on ice for 30 minutes.  </LI><LI>Place the tube in a 42 deg°C water bath for 60 seconds. </LI><LI>After heat shock leave cells on ice for 5 minutes. </LI><LI>Add 950 µL of SOC media and shake cells for 2 hours at 37°C</LI><LI>Pipet 100 µL of the cells onto an appropriate plate and spread them using sterile glass beads. 
  Incubate overnight at 37 deg. Celsius and hope for colonies in the morning to prove a successful transformation. </LI></OL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><H3>Preparation of LB media</H3><H4>Material:</H4><P><UL><LI>Tryptone</LI><LI>NaCl</LI><LI>Yeast extract</LI></UL></P><H4>Method:</H4><P><OL><LI>Fill a container (bottle) to about 60/70 % its volume with destilled water. </LI><LI>Add 10 g/L Tryptone, 10 g/L NaCl and 5 g/L of yeast extract. </LI><LI>Stir properly and fill up the remaining volume with distilled water. </LI><LI>Treat LB media by autoclave. </LI></OL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><H3>Preparation of LB agar</H3><H4>Material:</H4><P><UL><LI>Tryptone</LI><LI>NaCl</LI><LI>Yeast extract</LI><LI>Agar</LI></UL></P><H4>Method:</H4><P><OL><LI>Fill a container (bottle) to about 60/70 % its volume with destilled water. </LI><LI>Add 10 g/L Tryptone, 10 g/L NaCl, 5 g/L of yeast extract and 20 g/L agar. </LI><LI>Stir properly and fill up the remaining volume with distilled water. </LI><LI>Treat by autoclave. </LI></OL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><H3>Preparation of M9 media</H3><H4>Salt solution:</H4><P><UL><LI>3 g Na<SUB>2</SUB>HPO<SUB>4</SUB></LI><LI> 1,5 g KH<SUB>2</SUB>PO<SUB>4</SUB></LI><LI>0,5 g NH<SUB>4</SUB>Cl</LI><LI>0,25 g NaCl</LI><LI>1 mL MgSO<SUB>4</SUB></LI><LI>50 µL CaCl<SUB>2</SUB></LI></UL></P><H4>Sugar solution:</H4><P><UL><LI>20 g/LGlycerol</LI><LI>5 g/L Yeast </LI><LI>10 mg Thiamine</LI><LI>1 mL Micronutrients</LI></UL></P><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><H3>Glycerol stocks – storage of bacterial strains</H3><H4>Material:</H4><P><UL><LI>Glycerol</LI></UL></P><H4>Method:</H4><P><OL><LI>Mix 700 µL of overnight culture with 300 µL glycerol. </LI><LI>Store at -80°C</LI></OL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><H3>Polymerase chain reaction (PCR)</H3><H4>Material:</H4><P><UL><LI>Primer forward &amp; reverse</LI><LI>Template DNA</LI><LI>NEB Q5® High-Fidelity 2X Master Mix (dNTPs + Polymerase)</LI><LI>Distilled water</LI><LI>PCR-Cycler</LI></UL></P><H4>Method:</H4><P><OL><LI>All steps have to be performed on ice.</LI><LI>50 µL approach (mix well):</LI></OL></P><TABLE align="middle" hspace="10"><TBODY><TR><TH>Components</TH><TH>Volume</TH><TH>Concentration</TH></TR><TR><TD>Q5 Master Mix</TD><TD>25 µL</TD><TD>1x</TD></TR><TR><TD> 10 µM fw primer</TD><TD>2,5 µL</TD><TD>0,5 µM</TD></TR><TR><TD> 10 µM rv primer</TD><TD> 2,5 µL </TD><TD>0,5 µM</TD></TR><TR><TD> Template DNA</TD><TD>&lt;1000 ng</TD></TR><TR><TD> Water</TD><TD>remaining volume to 50 µL</TD></TR></TBODY></TABLE><P>PCR-cycler conditions:</P><TABLE align="middle" hspace="10"><TBODY><TR><TH>Step</TH><TH>Cycles</TH><TH>Temperature</TH><TH>Time</TH></TR><TR><TD>Denaturation</TD><TD>1</TD><TD>98°C</TD><TD>30 sec</TD></TR><TR><TD>Annealing</TD><TD>25-35</TD><TD>98°C</TD><TD>5-10 sec</TD></TR><TR><TD>Elongation</TD><TD>72°C</TD><TD>20-30sec/kb</TD></TR><TR><TD>Final extension</TD><TD>1</TD><TD>72°C</TD><TD>2 min</TD></TR><TR><TD>Hold</TD><TD>4-10°C</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><H3>Determination of DNA concentration</H3><P><UL><LI>DNA concentration is determined by using a Nanodrop ()</LI><LI>The absorbance at 260 nm is converted to concentration using the Lambert – Beer Equation by the program </LI><LI>The purity of the sample is confirmed by the 260/280 ratio (contamination with proteins) and the 260/230 ratio (presence of co-purified contaminants)</LI><LI>For pure DNA the 260/280 ratio should be around 1.8 and the 260/280 ratio should be around 1.8 – 2.2. (ND-1000-v3.4-users-manual, Thermo Scientific) </LI></UL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><H3>Mini-Prep (based on Fast-n-Easy Plasmid Mini-Prep Kit Jena Bioscience)</H3><H4>Material:</H4><P><UL><LI>Lysis buffer</LI><LI>Neutralization buffer</LI><LI>Column-Activation buffer</LI><LI>Wash buffer</LI><LI>Elution buffer</LI><LI>Binding column</LI></UL></P><H4>Method:</H4><P><OL><LI>Harvest the over-night culture by centrifugation (3000 g for 10 minutes.)</LI><LI>Activate the Binding Column by adding 100 µl of Activation buffer and centrifugation at 10000 g for 30 seconds. </LI><LI>For cell lysis resuspend the cell pellet in 300 µl Lysis buffer (pipetting or vortexing).</LI><LI>Add 300 µl of Neutralization buffer and mix by inverting the tube (4 – 6 times).</LI><LI>Centrifuge at 10000 g for 5 minutes.</LI>
The colour of the supernatant should be yellow.<LI>Transfer the supernatant into the Binding Column and centrifuge at 10000 g for 30 seconds. Discard the flow-through.</LI><LI>Add 500 µl Washing buffer to the column and centrifuge at 10000 g for 30 seconds. Discard the flow-through.</LI><LI>Place the Binding Column into a clean microtube an add 30 µl Elution buffer.</LI><LI>Incubate for 1 minute and centrifuge at 10000 g for 1 minute to elute DNA.</LI><LI>The eluted DNA could be used directly or should be stored at -20°C.</LI></OL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><H3>Restriction digest</H3><H4>Material:</H4><P><UL><LI>Plasmid DNA</LI><LI>Restriction Enzymes</LI><LI>Restriction buffer</LI><LI>H2O</LI><LI>Ice</LI></UL></P><H4>Method:</H4><P><OL><LI>All steps must be performed on ice.</LI><LI>For a 20 µl double digest approach following amount are needed:</LI></OL><TABLE><TBODY><TR><TH>Components</TH><TH>Volume</TH><TH>Concentration</TH></TR><TR><TD>Restriction buffer (10X)</TD><TD>2 µL</TD><TD>1X</TD></TR><TR><TD>Restriction enzyme 1</TD><TD>1 µL</TD></TR><TR><TD>Restriction enzyme 2</TD><TD>1 µL</TD></TR><TR><TD>Plasmid DNA</TD><TD>x µL</TD><TD>1 µg total</TD></TR><TR><TD>H2O</TD><TD>add to 20 µL</TD></TR></TBODY></TABLE><LI>Digest the approach for 1h at 37 °C.</LI><LI>To stop the reaction, incubate the reaction for 20 minutes at 80 °C.</LI></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><H3>Gibson Assembly</H3><H4>Material:</H4><P><UL><LI>Gibson Assembly Master Mix</LI><LI>Backbone DNA</LI><LI>Insert DNA</LI><LI>Ice</LI></UL></P><H4>Method:</H4><P><OL><LI>Calculate the amount of backbone and insert DNA</LI><LI>Mix a 5 µL approach</LI><LI>Add this approach to a 15 µL approach of Gibson-Assembly Master Mix and mix gently</LI><LI>Incubate the mixture for 1 h at 50°C</LI><LI>Transform 2 µL of 1:2 dilution of the approach into 50 µL of competent cells</LI></OL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><H3>Ligation</H3><H4>Material:</H4><P><UL><LI>Insert DNA</LI><LI>Vektor DNA</LI><LI>Ligation buffer</LI><LI>H<SUB>2</SUB>O</LI></UL></P><H4>Method:</H4><P><OL><LI>All steps must be performed on ice.</LI><LI>For a 20 µL ligation approach following amount are needed: (molar ratio vector DNA/insert DNA (3:1))</LI></OL><TABLE><TBODY><TR><TH>Components</TH><TH>Volume</TH><TH>Concentration</TH></TR><TR><TD>Ligation buffer</TD><TD>2 µL</TD><TD>1X</TD></TR><TR><TD>Ligase</TD><TD>1 µL</TD></TR><TR><TD>Vector DNA</TD><TD>x µL</TD></TR><TR><TD>Plasmid DNA</TD><TD>x µL</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>add to 20 µL</TD></TR></TBODY></TABLE><OL><LI>Mix the reaction gently</LI><LI>Incubate the reaction mix for one hour at room temperature</LI><LI>To stop the reaction, incubate the reaction for 30 min at 65°C</LI><LI>Transfer 2 µL of the reaction into 50 µL competent cells</LI></OL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><H3>Cell-Lysis</H3><H4>Material:</H4><P><UL><LI>Cells</LI><LI>Tris-HCL buffer</LI><LI>Glas-sand</LI><LI>Cell Mill</LI></UL></P><H4>Method:</H4><P><OL><LI>Harvest the cells by centrifugation (3000 g for 10 min)</LI><LI>Resuspend the pellet in 1 mL of TRIS-HCL buffer (pH 8,0)</LI><LI>Transfer the resuspended pellet into a reaction tube with glas-sand (0,25 mL)</LI><LI>Lyse the cells in a cell mill (3 cycles)</LI><LI>Centrifuge the approach for 30 min at 13000 rpm and 4°C </LI><LI>Use the supernatant for further experiments  </LI></OL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><H3>Gel-Extraction (based on Gel DNA Recovery Kit from Zymo Research)</H3><H4>Material:</H4><P><UL><LI>Extraction buffer (ADB buffer)</LI><LI>Washing buffer</LI><LI>Elution buffer</LI><LI>Spin column</LI></UL></P><H4>Method:</H4><P><OL><LI>Cut out the area of the gel containing the DNA fragment of interest. </LI><LI>Add ADB buffer (3 volumes to 1 volume gel).</LI><LI>Dissolve the gel by incubation at 37 – 55 °C for 5 – 10 minutes.</LI><LI>Transfer the solution to a spin column and centrifuge for 30 seconds. Discard the flow-through.</LI><LI>Add 200 µl Washing buffer and centrifuge for 30 seconds. Discard the flow-through.</LI><LI>Repeat the wash step.</LI><LI>Place the column in a clean microtube and add more than 6 µl Elution buffer. Centrifuge for 1 minute to elute DNA.</LI></OL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><H3>Enzyme activity assay of the esterases EstCS2 (BBa_K1149002) and LipB
</H3><P>To determine the enzyme activity of EstCS2 and LipB a photometric assay was prepared. Two wildtypes were compared with the enzyme activity of the esterases. The results were obtained from biological triplicates. 
</P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><H4>Principle</H4></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-7 col-md-7"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:730px;"><DIV class="thumbcaption"><B>Figure 1:</B> Principle of Esterase activity assay.
</DIV></DIV></DIV></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><H4>Material:</H4><P><UL><LI>Agar plates</LI><LI>Spectrometer</LI><LI>Photometer</LI><LI>Centrifuge</LI><LI>96 well plates</LI></UL></P><H4>Chemicals:</H4><P><UL><LI>LB media</LI><LI>PBS buffer</LI><LI>Arabinose stock solution [500 mM]</LI><LI>IPTG stock solution [1 M]</LI><LI>p-nitrophenyl butyrate [50 mM] diluted in acetonitril</LI></UL></P><H4>Method:</H4><P>Preparation of the cell lysate</P><P><OL><LI>Pick a colony from the agar plates</LI><LI>Grow the colony in 5 mL LB media (+antibiotic) over night</LI><LI>Dilute the overnight culture 1:100 in 15 mL LB media (+antibiotic)</LI><LI>Grow the cells until the OD600 reached 0,2</LI><LI>Induce gene expression with IPTG/arabinose with different concentrations 
(0 mM, 1 mM, 2 mM, 3 mM) over night</LI><LI>Grow the cell culture until the OD600 reached 6</LI><LI>Harvest the cells by centrifugation (10 min, 3000 rpm)</LI><LI>Resuspend the cell pellet in 10 mL PBS buffer</LI></OL></P><P>Enzyme activity assay: Continous spectrometric rate determination </P><P><OL><LI>100 µL of the supernatant (1:10 dilution) or resuspended cell pellet and 10 µL p-nitrophenyl butyrate with different concentrations [2.5; 5; 10; 15; 20 mM] are mixed in a 96 well plate</LI><LI>Photometric enzyme activity test with the following conditions: wavelength: 405 nm, 
temperature: 37 °C
time: measurement every 45 seconds for 30 min
96 well plate is shaked (200 rpm) </LI><LI>Calculation of the enzyme activity out of the absorption rates. 1 unit (U) was defined as the amount of enzyme that catalyzes the reaction of 1 nanomol of substrate per minute. </LI></OL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><H3>Determination of protein concentration - BCA Assay (Pierce™ BCA Protein Assay Kit – Thermo Scientific)</H3><H4> Material:</H4><P><UL><LI>BCA reagent A and B (provided in Assay Kit)</LI><LI>BSA stock solution (provided in Assay Kit)</LI><LI>96-Well microplates</LI><LI>Microplate reader</LI><LI>Microplate incubator/shaker</LI></UL></P><H4>Method:</H4><P><OL><LI>Preparation of serial dilution of BSA-standard:</LI><TABLE><TBODY><TR><TH>Vial</TH><TH>Volume of Diluent (µL)</TH><TH>Volume and Source of BSA (µL)</TH><TH>Final BSA Concentration (µg/mL)</TH></TR><TR><TD>A</TD><TD>0</TD><TD>300 of Stock</TD><TD>2000</TD></TR><TR><TD>B</TD><TD>125</TD><TD>375 of Stock</TD><TD>1500</TD></TR><TR><TD>C</TD><TD>325</TD><TD>325 of Stock</TD><TD>1000</TD></TR><TR><TD>D</TD><TD>175</TD><TD>175 of vial B dilution</TD><TD>750</TD></TR><TR><TD>E</TD><TD>325</TD><TD>325 of vial C dilution</TD><TD>500</TD></TR><TR><TD>F</TD><TD>325</TD><TD>325 of vial E dilution</TD><TD>250</TD></TR><TR><TD>G</TD><TD>325</TD><TD>325 of vial F dilution</TD><TD>125</TD></TR><TR><TD>H</TD><TD>400</TD><TD>100 of vial G dilution</TD><TD>25</TD></TR><TR><TD>I</TD><TD>400</TD><TD>0</TD><TD>0=Blank</TD></TR></TBODY></TABLE><DL><DD>Table: User Guide, Pierce™ BCA Protein Assay Kit – Thermo Scientific</DD><LI>Preparation of Assay Working Reagent: mix 50 parts of Reagent A with 1 part of Reagent B.</LI><LI>If necessary dilute your samples to fit into the working range of 20-2000µg/mL.</LI><LI>Pipette 25 µL of each standard and sample into a microplate and add 200 µL of Working Reagent.</LI><LI>Mix for 30 seconds on plate shaker and incubate at 37°C for another 30 minutes.</LI><LI>After cooling down to room temperature, measure the absorbance at 562 nm.</LI></DL></OL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><H3>Semi-quantitative hair degradation assay for keratinases</H3><P>To prove any enzyme activity a semi-quantitative hair degradation assay was performed.
First, cultures of <I>E. coli</I> containing kerA, kerUS, kerP plasmid and one wild-type <I>E. coli</I> were grown at 37°C in sterile LB broth. Chloramphenicol (final concentration 35 µg/mL) was added to the cultures containing kerA and kerUS. Kanamycin (final concentration 50 µg/mL) was added to the cultures containing the kerP. After incubation OD600 was measured before inducing cultures containing kerA and kerUS with IPTG with a final concentration of 1mM.
Human hair was reduced to smaller pieces and then dried for 1 hour at 65°C. Afterwards the hair was distributed in 0.05 g aliquots and the full amount of each culture was added. 
The cultures + hair were incubated 120 hours at 37°C with slightly shaking.</P><H3>Skim milk plate assay for keratinases</H3><P>This assay was performed to show qualitative enzyme activity. 
The different keratinases (kerA, kerUS and kerP) should degrade the casein in the milk, seen as clear zones around the cultures itself or the supernatant of the cells.
</P><H4>Preparation of skim milk plates:</H4><P>First 100 mL LB/agar and 5 g skim milk powder in 125 mL distilled water was prepared. Both solutions were autoclaved and mixed after cooling down. 
For the plates used for kerA and kerUS cultures chloramphenicol (35 µg/mL) was added – in case of kerP kanamycin (50 µg/mL) was used to prepare plates. 
The plates were poured and stored at 4°C.</P><H4>Assay:</H4><P><UL><LI>kerA/kerUS: Before spreading both keratinases on the plates, cells were induced with 1mM IPTG o/n at 37°C. 
  After incubation 10 µL of the induced cell cultures and supernatant, cell culture without IPTG induction and a wild-type <I>E. coli</I> control was spread on skim milk plates containing chloramphenicol. 
  The plates were incubated at room temperature for four days.
</LI><LI>kerP: 10 µL of cell culture, supernatant and a wild-type <I>E. coli</I> control was spread on skim milk plates containing kanamycin. 
  The plates were incubated at room temperature for four days.</LI></UL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><H3>Enzyme activity assay of keratinases</H3><H4> Material:</H4><P><UL><LI>96 well plates </LI><LI>Arabinose stock solution [500 nM]</LI><LI>IPTG stock solution [1 M]</LI><LI>Azo-keratin solution (substrate) </LI><LI>Microplate incubator/shaker</LI></UL></P><H4>Method:</H4><P><OL><LI>Grow the cells in 30 ml shaking flaks and induce the gene expression with IPTG/arabinose (1 mM)
</LI><LI>Harvest the cells by centrifugation (3000 g for 10 min)</LI><LI>Use the supernatant or the lysed cells (see cell lyse) for the activity assay</LI><LI>200 µL of culture supernatant or lysed cell supernatant and 1,6 mL of centrifugated substrate are mixed and incubated at 37°C (300 rpm).
Technical triplicates are performed</LI><LI>Extinction of the samples are measured after 0h, 3h and 24h at 440 nm</LI></OL></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>