<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Stuttgart_Safety skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Stuttgart/Safety</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2017.igem.org/Team:Stuttgart">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Sponsoring">Sponsoring</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Background-Information">Background
                            Information</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Protocols">Protocols &amp; Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/InterLab">Interlab Study</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Model">Modelling</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Composite Part">Composite Part</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Results </A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Esterases and Lipases">Esterases and Lipases</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Keratinases">Keratinases</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Rose and Limonene Fragrance">Rose and Limonene Fragrance</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Demonstrate">Achievements</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/HP/Gold_Integrated">Integrated and Gold</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/HP/Silver">Silver HP</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Awards </A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Measurement">Measurement</A></LI><LI><A href="https://igem.org/2017_Judging_Form?team=Stuttgart">Judging Form</A></LI></DIV></DIV><H1 align="middle"> Biosafety </H1><DIV class="container"><DIV class="row section"><DIV class="col-xs-12 col-sm-9 col-md-9"><P>Biosafety is an important part in the field of ‘Synthetic Biology’ that has to be considered thoroughly. 
             Genetically modified organisms must be prevented from spreading out into the environment after they fulfilled their designated function. 
             In our project we constructed a genetically modified tube cleaner, based on a holistic approach of genetically modified <I>E.coli</I>  (for reasons of simplicity we will call them fictionally E.pipelight). 
             Eventually, the cells will be flushed through the tubes into wastewater and could potentially spread out into the environment from there.
          </P></DIV><DIV class="col-xs-12 col-sm-3 col-md-3"><DIV class="thumb tleft"><DIV class="thumbinner" style=";"><DIV class="thumbcaption"><B>Figure 1: </B>E.Pipelight.</DIV></DIV></DIV></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><P>To gain further insight on this topic, we had an interesting expedition to a local wastewater treatment plant from the department of wastewater technology at the University Stuttgart.  
             We determined that our E.pipelight cells would not be retained or filtered there. 
             On the contrary, as wastewater treatment plants also use microorganisms, this leads to the possibility of interaction between our genetically modified E.pipelight and these other microorganisms. 
             To avoid an exchange of DNA between these organisms which might lead to unpredictable consequences for the environment, we had to consider a biosafety system that guaranteed the elimination of the E.pipelight right at their location of function – the clogging of drains. 
             Due to that we would use an <I>E. coli</I> strain which is a synthetic auxotroph. This means that the E.pipelight will require the presence of a particular molecule that ensures for their viability (Lopez and Anderson 2015).
           </P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-6 col-md-6"><P>We wanted to use a chromosomal deletion of the asd-gene, which encodes for the aspartate-β-semi aldehyde-dehydrogenase. 
             This enzyme is involved in lysine, threonine and methionine biosynthesis and works together with dihydrodipicolinate synthase (DHDPS). 
             DHDPS is encoded by the dapA gene and is the first enzyme of the pathway to synthesize diaminopimelinacid (DPA) in <I>E.coli</I> and could be also be considered for deletion (Born and Blanchard 1999, Richaud et al. 1986). 
             DPA is an essential component of the peptidoglycan crosslinkage of the cell wall of gram negative bacteria (Schleifer and Kandler, 1972). Deleteting the asd gene will disrupt the diaminopimelinacid
             pathway leading to impaired cell wall synthesis and subsequent cell lysis (Born and Blanchard 1999). 
             In the past the application of DPA auxotroph <I>E.coli</I> strain has been already proven to be successful in a strain that is called <I>Escherichia coli</I> chi1776, a descendant of <I>Escherichia coli</I> K-12. 
             This strain was specifically constructed and approved as a safe bacterial strain for use with plasmid cloning vectors (Alexander et al. 1980). 
           </P></DIV><DIV class="col-xs-12 col-sm-6 col-md-6"><DIV class="thumb tleft"><DIV class="thumbinner" style=";"><DIV class="thumbcaption"><B>Figure 2: </B>E.Pipelight.</DIV></DIV></DIV></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><P>The risk that these deletion could be repealed by another gain-of-function mutation is very low. 
                  Nevertheless this case has to be kept in mind. Another problem that could occur, is that plasmids which last after cell lysis in the tube for a certain period of time, might be up-taken from another microorganism and interfere with them. 
                  Again this occurrence is very unlikely, yet we have to avoid every potential risk, no matter how rarely it could occur. 
                  For this we may have to modify our next generation of E.pipelight by performing our genetically engineered modifications, not in a plasmid but right away in the chromosome of the <I>E.coli</I>.  
                  This could be achieved via already well established and precise tools such as CRISPR-Cas or Lambda-Red.  
                  As the chromosome is less stable than the plasmid without its hosts, the possibility that another micro-organism would incorporate a functional chromosome will be decreased once again. 
                </P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><FOOT><P>REFERENCES
                <OL><LI>Alexander WJ, Alexander LS, Curtiss R., 3rd Bactericidal activity of human serum against <I>Escherichia coli</I> chi1776. Infect Immun. 1980;28:837–841.
              </LI><LI>Born, T.L., and Blanchard, J.S. 1999. Structure/function studies on enzymes in the diaminopimelate pathway of bacterial cell wall biosynthesis. Curr. Opin. Chem. Biol. 3: 607–613. doi:10. 1016/S1367-5931(99)00016-2. PMID:10508663.
              </LI><LI>Lopez, G. and Anderson, J. (2015). Synthetic Auxotrophs with Ligand-Dependent Essential Genes for a BL21(DE3) Biosafety Strain. ACS Synthetic Biology, 4(12), pp.1279-1286
              </LI><LI>Richaud, F., Richaud, C., Ratet, P., and Patte, J.-C. 1986. Chromosomal location and nucleotide sequence of the <I>Escherichia coli</I> dapA gene. J. Bacteriol. 166(1): 297–300. PMID:3514578.
              </LI><LI>Schleifer KH, Kandler O. Peptidoglycan types of bacterial cell walls and their taxonomic implications. Bacteriol Rev. 1972 Dec;36(4):407–477.
              </LI></OL></P></FOOT></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>