<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Stuttgart_Esterases_and_Lipases skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Stuttgart/Esterases and Lipases</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2017.igem.org/Team:Stuttgart">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Sponsoring">Sponsoring</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Background-Information">Background
                            Information</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Protocols">Protocols &amp; Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/InterLab">Interlab Study</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Model">Modelling</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Composite Part">Composite Part</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Results </A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Esterases and Lipases">Esterases and Lipases</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Keratinases">Keratinases</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Rose and Limonene Fragrance">Rose and Limonene Fragrance</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Demonstrate">Achievements</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/HP/Gold_Integrated">Integrated and Gold</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/HP/Silver">Silver HP</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Awards </A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Measurement">Measurement</A></LI><LI><A href="https://igem.org/2017_Judging_Form?team=Stuttgart">Judging Form</A></LI></DIV></DIV><H1 align="middle"> Results </H1><DIV class="container"><DIV class="row section"><H3>Esterases and Lipases</H3><H4>Enzyme activity assay: Esterases</H4><DIV class="col-xs-12 col-sm-10 col-md-10"><P>In the following figures enzyme activities of the supernatant of the cells containing the plasmid with the genes for the esterases EstCS2 and LIpB are shown. All results were obtained from biological triplicates. The detailed method for the activity determination is shown <A href="https://2017.igem.org/Team:Stuttgart/Protocols"> here.</A>
                 At the N-terminal end of the esterase EstCS2 a signal peptide (PelB) is added. Therefore the gene induction leads to enzyme expression and the expression of the signalpeptide and the secretion of the enzyme is enabled. 
              </P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-8 col-md-8"><DIV class="thumb tleft"><DIV class="thumbinner" style="px;"><DIV class="thumbcaption"><B>Figure 1:</B> Illustration of the esterase activity [U] of the cells containing the EstCS2 gene in the supernatant dependend on time and substrate concentration (2.5, 5, 10, 15, 20 mM). 
The standard abbreviation was calculated from biological triplicates.
</DIV></DIV></DIV></DIV><DIV class="col-xs-12 col-sm-4 col-md-4"><P>The enzyme activity of EstCS2 raises with the induction level for gene expression and with the substrate concentration. 
  A maximum of 246,5 U is gained with 20 mM p-Nitrophenyl butyrate and 1 mM arabinose. 
  The enzyme activities between the induction levels of 2 mM arabinose and 3 mM arabinose isn’t different. 
  The enzyme activity is highest with an induction level of 1 mM arabinose instead of an induction level of 2 or 3 mM arabinose. 
  This could be explained with less movement flexibility of the esterases if high gene expression rates are induced. Additionally inclusion bodies could build if high transcription rates are induced and the secretion system isn't capable to secrete the produced enzymes fast enough. (LINK Sekretion von Proteinen)
  We would further recommend to induce the gene expression with less arabinose concentrations and investigate the enzyme activity with 0.1 mM, 0.5 mM and 1 mM arabinose. </P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-8 col-md-8"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:;"><DIV class="thumbcaption"><B>Figure 2:</B> Illustration of the esterase activity [U] of E. coli wilde types in the supernatant dependend on time and substrate concentration (2.5, 5, 10, 15, 20 mM). 
      The standard abbreviation was calculated from biological triplicates. 
    </DIV></DIV></DIV></DIV><DIV class="col-xs-12 col-sm-4 col-md-4"><P>In comparison we investigated the enzyme activity of two E. coli wilde type strains (E. coli DH5α and MG1655). 
        We aimed to aquire knowledge about the esterase activity of the wild typs to identify the effect of genetically engineered organisms. 
        The figures show less enzyme activity of the wild types in comparison to the genetically engineered E. coli that contain the plasmid with the esterase genes. 
        This means that the investigated EstCS2 is an appropriate enzyme to degradade fat layers on hair.
       </P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-8 col-md-8"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:;"><DIV class="thumbcaption"><B>Figure 3:</B> Illustration of the esterase activity [U] of the cells containing the LipB gene in the supernatant dependend on time and substrate concentration (2.5, 5, 10, 15, 20 mM). 
       The standard abbreviation was calculated from biological triplicates.
     </DIV></DIV></DIV></DIV><DIV class="col-xs-12 col-sm-4 col-md-4"><P>Additionally we investigated the enzyme activity of the supernatant of the cells with the plasmid containg the LipB gene. 
         To compare the enzyme activities of the cells with the different esterases, we used the same induction levels and substrate concentrations for the assays. 
         The figures show that the enzyme activity of the supernatant isn’t higher than the enzyme activity of the supernatant of the wild type cells. 
         These results can be declared with the absence of a signal peptide at the N-terminal side of the esterase gene. 
         Thus, no enzyme secretion is performed and less enzyme activity can be detected in the supernatant. 
         This leads to the conclusion that a signal peptide has to be added at the N-terminal side of the esterase gene to obtain enzyme secretion and extracellular enzyme activity. 
       </P></DIV></DIV></DIV><DIV class="container"><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><H4>Gibson Assembly PelB-LipB</H4></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><P>In this part of the project we aimed to assemble the esterase LipB with the signal sequence PelB to achieve extracellular secretion of the enzyme. 
      Therefore we amplified the fragments PelB-Vector and LipB. 
      <A href="http://parts.igem.org/Part:BBa_K1149002"> BBa_K1149002</A> was used as a backbone that also includes the PelB signal sequence and pet19-LipB as the template for the amplification of the PelB gene. 
      Subsequently we performed a Gibson Assembly to assemble these fragments. 
      In conclusion, we tried to replace the EstCS2 gene from <A href="http://parts.igem.org/Part:BBa_K1149002"> BBa_K1149002</A>  with the LipB gene from pet19-LipB. 
      With this method we didn’t only attach the signal sequence PelB to  the LipB but also cloned the genes into the iGEM vector (pSB1C3). 
      sfGFP was also assembled with the esterases. 
      In our project sfGFP didn’t interfere with the esterase activities so we decided not to remove the fragment.
 </P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-4 col-md-4"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:;"><DIV class="thumbcaption"><B>Figure 4:</B><A href="http://parts.igem.org/Part:BBa_K1149002"> BBa_K1149002</A>   - iGEM Team Imperial College 2013.
    </DIV></DIV></DIV></DIV><DIV class="col-xs-12 col-sm-4 col-md-4"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:;"><DIV class="thumbcaption"><B>Figure 5:</B> Assembled PelB-LipB - improved Part.
</DIV></DIV></DIV></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-10 col-md-10"><P>Figure 6 shows the fragment sizes of the amplified fragments LipB and PelB-Vector. 
  The picture confirms the expected fragment sizes of 588 bp (LipB fragment) and 3069 bp (PelB-Vector fragment).  
  Afterwards Gibson Assembly was performed and the assembled fragments were transformed (Figure 7). 
  Subsequently we executed a colony PCR to verify which colonies contain the assembled LipB-PelB fragment. 
  For the colony PCR we used the iGEM Primers VF2 (<A href="http://parts.igem.org/Part:BBa_G00100">BBa_G00100</A>) and VR (<A href="http://parts.igem.org/Part:BBa_G00101">BBa_G00101</A>). 
  Figure 8 proves that three colonies (number 4, 5, 8) contain the plasmid with the pBAD-RBS-PelB-LipB-sfGFP-fragment with an expected PCR fragment size of 1515 bp. 
  As a positive control we used the plasmid <A href="http://parts.igem.org/Part:BBa_K1149002">BBa_K1149002 </A>. 
  The primers amplified pBAD-RBS-PelB-LipB-sfGFP which is 2703 bp long. Colonies with the number 1 – 3, 6, 7, 9 – 15 still enclose the original plasmid with EstCS2 instead of LipB.     We can see a small fragment of 250 bp in every colony. 
  This could be explained with the annealing temperature of 57 °C that was used for the PCR.     Primers could bind to unspecific base pairs of the plasmid and amplify small fragments. 
  Nevertheless the Gibson Assembly was successful and we improved a previes iGEM part. 
  In our next step we performed the esterase enzyme activity assay with the positive colonies to determine the enzyme activity of LipB with the assembled PelB signal peptide. 
  Afterwards the PelB-LipB-colonies need to be sequenced. 
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:;"><DIV class="thumbcaption"><B>Figure 6:</B> 1 - 15: Colony PCR products - PelB-LipB-sfGFP (amplified with the iGEM primers Vf2 and VR); 16 - 18: Control - EstCS2 - sfGFP (amplified with iGEM primers VF2 and VR	</DIV></DIV></DIV></DIV><DIV class="col-xs-12 col-sm-2 col-md-2"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:;"><DIV class="thumbcaption"><B>Figure 7:</B> LipB PCR-fragment:expected size 588 bp, (2)Pet19-LipB:expected size 6263 bp, (3)iGEM vector (PelB) 
          PCR-Fragment: expected size 3069 bp, (4) <A href="http://parts.igem.org/Part:BBa_K1149002" bba_k1149002="">:expected size 4773 bp.
	</A></DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:;"><DIV class="thumbcaption"><B>Figure 8:</B> Transformation after Gibson Assembly of PelB-LipB.
	</DIV></DIV></DIV></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-6 col-md-6"><P>Figure 9 shows the results from the enzyme activity assay with the assembled fragment PelB-LipB. 
  Predicted positive colonies that contain the PelB-LipB fragment (figure xy) were grown in 5 mL LB media with antibiotics. 
  Gene expression was directly induced with 1 mM arabinose. 
  In comparison to the cells containing the plasmid with the EstCS2 gene instead of the LipB gene, we can detect lower activity of LipB. 
  We have to keep in mind that the preparation of the cells for the activity assay containing PelB-LipB is different than the protocol that we used for the preparation of the cells containing PelB-EstCS2. 
  The cells containing PelB-EstCS2 were grown overnight in 5 mL LB, diluted (1:100) in 15 mL LB and induced over night. 
  Therefore we can’t abosolutely compare the enzyme assay results of PelB-LipB with PelB-EstCS2. 
  Nevertheless our new construct which contains the PelB-LipB fragment is active and an appropriate enzyme to support the fat layer degradation. 
</P></DIV><DIV class="col-xs-12 col-sm-6 col-md-6"><DIV class="thumb tleft"><DIV class="thumbinner" style="px;"><DIV class="thumbcaption"><B>Figure 9:</B> : Illustration of the esterase activity [U] of PelB-LipB  in the supernatant dependend on time and substrate concentration [2.5, 5, 10, 15, 20 mM). The standard abbreviation was calculated from biological triplicates.
    </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>