<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Stuttgart_Keratinases skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Stuttgart/Keratinases</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2017.igem.org/Team:Stuttgart">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Sponsoring">Sponsoring</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Background-Information">Background
                            Information</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Protocols">Protocols &amp; Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/InterLab">Interlab Study</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Model">Modelling</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Composite Part">Composite Part</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Results </A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Esterases and Lipases">Esterases and Lipases</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Keratinases">Keratinases</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Rose and Limonene Fragrance">Rose and Limonene Fragrance</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Demonstrate">Achievements</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/HP/Gold_Integrated">Integrated and Gold</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/HP/Silver">Silver HP</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Awards </A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Measurement">Measurement</A></LI><LI><A href="https://igem.org/2017_Judging_Form?team=Stuttgart">Judging Form</A></LI></DIV></DIV><H1 align="middle"> Results </H1><DIV class="container"><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><H3>Keratinases</H3></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-10 col-md-10"><H4>BCA Assay</H4><P>Protein concentration of supernatant and cell disruption of <I>E.coli</I> carrying different keratinase plasmids (KerP, KerUS and KerA in different vectors) was measured with BCA Assay method (see chapter methods).
                The figure 1 below shows the BSA standard curve (blank corrected, arithmetic average of triplicates).
                Table 1 and 2 show the results of sample measurements (OD values: blank corrected, arithmetic average of triplicates).
                The concentrations are calculated with the calibration curve (BSA standard curve).
                Expectetly protein concentration is much more higher in cell disruption than in supernatants.
                It is also noticeable that protein concentraion of <I>E.coli</I> wild type is very high compared to engineered cultures. 
                Unfortunately we do not have OD values, but we explain it by a higher amount of wild type cells after cultivation.
                </P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-6 col-md-6"><P>Table 1: supernatant (dilution 1:25):</P><TABLE><TBODY><TR><TH>sample</TH><TH>OD562</TH><TH>concentration [µg/mL]</TH></TR><TR><TD>KerP psD1K3</TD><TD>0,194</TD><TD>3066,667</TD></TR><TR><TD>KerP psD1C3</TD><TD>0,195</TD><TD>3091,667</TD></TR><TR><TD>KerUS pet28</TD><TD>0,191</TD><TD>2991,667</TD></TR><TR><TD>KerUS psD1C3</TD><TD>0,207</TD><TD>3391,667</TD></TR><TR><TD>KerUS BB</TD><TD>0,226</TD><TD>3866,667</TD></TR><TR><TD>KerA pet28</TD><TD>0,194</TD><TD>3083,333</TD></TR><TR><TD>KerA psB1C3</TD><TD>0,213</TD><TD>3550,000</TD></TR><TR><TD>KerA BB</TD><TD>0,201</TD><TD>3241,667</TD></TR><TR><TD>wild type <I>E.coli</I></TD><TD>0,248</TD><TD>4416,667</TD></TR></TBODY></TABLE></DIV><DIV class="col-xs-6 col-sm-6 col-md-6"><P>Table 2: cell disruption (dilution 1:25):</P><TABLE><TBODY><TR><TH>sample</TH><TH>OD562</TH><TH>concentration [µg/mL]</TH></TR><TR><TD>KerP psD1K3</TD><TD>0,514</TD><TD>11075,000</TD></TR><TR><TD>KerP psD1C3</TD><TD>0,611</TD><TD>13491,667</TD></TR><TR><TD>KerUS pet28</TD><TD>0,405</TD><TD>8358,333</TD></TR><TR><TD>KerUS psD1C3</TD><TD>0,285</TD><TD>5350,000</TD></TR><TR><TD>KerUS BB</TD><TD>0,385</TD><TD>7841,667</TD></TR><TR><TD>KerA pet28</TD><TD>0,482</TD><TD>10266,667</TD></TR><TR><TD>KerA psB1C3</TD><TD>0,585</TD><TD>12858,333</TD></TR><TR><TD>KerA BB</TD><TD>0,410</TD><TD>8475,000</TD></TR><TR><TD>wild type <I>E.coli</I></TD><TD>0,615</TD><TD>13600,000</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="row section"><DIV class="col-xs-7 col-sm-6 col-md-6"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:;"><DIV class="thumbcaption"><B>Figure 1:</B> BSA standard calibration curve.
	</DIV></DIV></DIV></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><H4>Enzyme activity based on extinction of Azo-Keratin conversion </H4><P>
  To evaluate which keratinase shows the most efficient enzyme activity we focused on three different keratinases that originated from distinct organisms and were known to be successfully expressed in <I>E.coli</I> before (Gupta and Gupta 2010, Hu et al. 2013, Jaouadi et al. 2013). KerUS that is originally expressed by <I>Bacillus brevis</I> and KerA that originates from <I>Bacillus Licheniformis</I> are both Keratinases that the iGEM Team Canmore has worked with in previous projects. They were so kind to send us these enzymes on different backbones (pet28, pSB1C3). KerP from <I>Pseudomonas aeruginosa</I> was synthesized and cloned into a pSB1K3 and a pSB1C3 backbone. 
  </P><P>
  To avoid confusions, we have to state that clonations from KerUS and KerA DNA that were located on a pSB1C3 backbone and which we obtained from iGEM Team Canmore 2016 were not successful. During transformation process of these plasmids into our competent Dh5α cells, two transposons interrupted our keratine coding sequence and rendered these cells dysfunctional (fig. 8 and fig. 9). This was proved by sequencing the DNA of both enzymes after transformation in our lab. Therefore, cell modifications by exchanging various promoters (BBa_J23115, BBa_J23119, BBa_K206000) and signal sequences (pelB and OmpA) were not successful (see gels in notebook) and no enzyme activity assay could be performed using these <I>E.coli</I> strains.
  </P></DIV></DIV><DIV class="row section"><DIV class="col-xs-4 col-sm-4 col-md-4"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:;"><DIV class="thumbcaption"><B>Figure 2:</B> : Extinction of Azo-keratine conversion over time of three different keratinases (kerUS, kerA, kerP) each of them located in different vectors (pet28, pSB1C3 and pSB1K3). Measurements of enzyme activity was performed at three timepoints (0h, 3h, 24h). Intracellular enzyme activity was measured.
 </DIV></DIV></DIV></DIV><DIV class="col-xs-4 col-sm-4 col-md-4"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:;"><DIV class="thumbcaption"><B>Figure 3:</B> : Extinction of Azo-keratine conversion over time of three different keratinases (kerUS, kerA, kerP) each of them located in different vectors (pet28, pSB1C3 and pSB1K3). Measurements of enzyme activity was performed at three timepoints (0h, 3h, 24h). Extracellular enzyme activity was measured.
 </DIV></DIV></DIV></DIV><DIV class="col-xs-4 col-sm-4 col-md-4"><P>
To overcome this problem, we performed clonation using KerUS and KerA plasmids from Team Canmore 2016 again. KerUS and KerA were located once on a pet28b+ backbone and on a pSB1C3 backbone. Clonation was performed using NEB 5-alpha competent <I>E.coli</I> this time. Another sequencing was performed to assure correct protein expression and to verify that the transposon incidence occurred during the transformation in our lab. 
  </P><P>
Transformation of the keratinases was successful this time and enzyme activity assays using these strains with additionally including KerP on a pSB1K3 and on a pSB1C3 backbone were performed.
 </P><P>
Enzyme activity was observed by performing an enzyme activity using keratin azure substrate (Sigma-Aldrich). 
During active keratinolytic activity azo-keratine is converted, so that azo dye is released. 
This extinction can be measured in the filtrate at 440 nm. 
  </P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><P>
    As a control we used water + substrate. 
    We refered higher enzyme activity to an increasing extinction rate. 
    This extinction rate was measured over time at three time points (0h, 3h and 24h) and was compared between KerA, KerUS and KerP located on the different backbones. 
    To verify if the signal peptides on the keratinase constructs (pelB for KerUS and KerA, and natural signal sequence of <I>Pseudomonas aeruginosa</I> for KerP) were definitely functional, we compared the extinction time of keratine azure intracellular (fig.2) and extracellular (fig.3) over time. 
    Cells were lysated and the intracellular enzyme activity was evaluated by an increasing extinction rate. 
    Same was done using the supernatant of the keratinases, to see how many active enzymes were extracellularly secreted and active. 
  </P><P>
In both different assays KerP located in the pSB1C3 backbone showed high activity if we can refer it to the amount of converted keratine azure. Interestingly KerP in pSB1K3 showed the second highest enzyme activity intracellularly, while it showed barely enzyme activity extracellularly. As they both have the same natural signal sequence, we wonder if this might be due to the different antibiotic resistances on the plasmid. 
  </P><P>
KerUS on the pet28b+ backbone seemed to show lower enzyme activity intracellulary, but enzyme activity was clearly high on the extracellular level. This may lead to the assumption that KerUS is quite efficiently secreted out of the cell and still active.
  </P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><H4>Total intracellular and extracellular protein concentration of different keratinases</H4><P>
Nevertheless, to make valid postulations we have to put these substrate extinctions in relation to the protein concentration or their OD600 of the investigated enzymes respectively. For this we measured total protein concentrations via BSA Assay as already described above, intracellular (fig. 4) and extracellular (fig. 5). 
  </P></DIV></DIV><DIV class="row section"><DIV class="col-xs-4 col-sm-4 col-md-4"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:;"><DIV class="thumbcaption"><B>Figure 4:</B> Intracellular total protein concentration of three different keratinases (kerUS, kerA, kerP) each of them located in different vectors (pet28, pSB1C3 and pSB1K3).
 </DIV></DIV></DIV></DIV><DIV class="col-xs-4 col-sm-4 col-md-4"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:;"><DIV class="thumbcaption"><B>Figure 5:</B> Extracellular total protein concentration of three different keratinases (kerUS, kerA, kerP) each of them located in different vectors (pet28, pSB1C3 and pSB1K3).
 </DIV></DIV></DIV></DIV><DIV class="col-xs-4 col-sm-4 col-md-4"><P>
It is important to mention that enzyme activity assays from fig. 2 and fig. 3 were unfortunately independently performed. 
The total protein concentration extracellular in fig. 5, just gives an idea of how much the distinct keratinase strains express proteins in total, with KerA pet28b+ and KerUS pSB1C3 secreting the highest total protein amount, but cannot be put in relation to the extinction rates. 
Nevertheless, to get an idea - and as we had the OD600 for the extracellular assay measured -, we put it in relation to their OD600 (fig. 7). 
  </P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><H4>Intracellular and extracellular enzyme activity of different keratinases</H4><P>
Intracellular extinction rate was performed simultaneously to the protein concentration determination and could be set in relation to each other (fig. 6). KerP in pSB1C3 backbone and KerA in pet28b+ backbone showed highest protein amount intracellular, with KerUS in pSB1C3 vector showing the lowest expression. This goes in line with our extinction rates observing these keratinases. 
  </P></DIV></DIV><DIV class="row section"><DIV class="col-xs-4 col-sm-4 col-md-4"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:;"><DIV class="thumbcaption"><B>Figure 6:</B> Ratio extinction azokeratine/protein concentration of intracellular keratinases over time. Shown is the extinction (440nm) of converted azo-keratine in relation to the total protein concentration (c) of three different keratinases (KerUS, KerA, KerP) each of them located on different vectors (pet28, pSB1C3 and pSB1K3).
 </DIV></DIV></DIV></DIV><DIV class="col-xs-4 col-sm-4 col-md-4"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:px;"><DIV class="thumbcaption"><B>Figure 7:</B> Ratio extinction azokeratine/OD600 of extracellular keratinases over time. 
   Shown is the extinction (440nm) of converted azo-keratine in relation to total protein concentration (c) of three different keratinases (KerUS, KerA, KerP) each of them located on different vectors (pet28+, pSB1C3 and pSB1K3). 

 </DIV></DIV></DIV></DIV><DIV class="col-xs-4 col-sm-4 col-md-4"><P>
Regarding the time courses of enzyme activity intracellular (in relation to the protein concentration) in fig. 6 and extracellular (in relation the their OD600) in fig. 7, we see that KerUS pSB1C3 shows the significantly highest enzyme activity, while extracellular KerUS in pet28b+ shows highest enzyme activity extracellularly. 
This might lead to the assumption that KerUS is highly expressed in the cell on a pet28b+ backbone but can be also successfully secreted when being located on the pSB1C3 backbone.
</P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><H4>Transposons</H4></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-6 col-md-8"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:;"><DIV class="thumbcaption"><B>Figure 8: Alignment with sequencing data of KerA and sequence of KerA (with standard iGEM primers). </B></DIV></DIV></DIV></DIV><DIV class="col-xs-12 col-sm-6 col-md-4"><P>Alignment with sequencing data of KerA and sequence of KerA (with standard iGEM primers).
Forward primer binds between insert point 3467 and insert point 300. 
Revers primer binds between insert point 687 and insert point 1587. 
There is a gap between 300 and 687 (387 bp long) at the begining of KerA gene.
</P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-6 col-md-8"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:;"><DIV class="thumbcaption"><B>Figure 9: Alignment with sequencing data of KerUS and sequence of KerUS (with standard iGEM primers). </B></DIV></DIV></DIV></DIV><DIV class="col-xs-12 col-sm-6 col-md-4"><P>Alignment with sequencing data of kerUS and sequence of KerUS (with standard iGEM primers). 
  Forward primer binds between insert point 3489 and insert point 277. 
  Revers primer binds between insert point 783 and insert point 1605. 
  There is a gap between 277 and 783 (506 bp long) at the beginning of KerUS gene (fig. 9)
</P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><H4>Future outlook</H4></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><P>
Due to the unfortunate transposon incidence we were not able to show quantitative results. 
As this occured just two weeks before wiki freeze we were not able to perform several biological and technical triplicates. 
Nevertheless we were able to show qualitative results, as succesful enzyme activity was proven after new transformation of the keratinases. 
This was shown with all three different keratinases KerA, KerUS and KerP. 
Seeing that KerUS and KerP ( the keratinase which resulted in our first own biobrick) are having the best enzyme activity we suggest to continue enzyme activity assays with these two keratinases. 
Additionaly we want to mention that alternatives for the Azo-Keratin Assay should be considered, as Azo-Keratin did not properly dissolve in the TRIS-HCL buffer. 
Therefore alternative assays and substrates such as skim milk assays and the substrate Azocasein should be kept in mind. 
As another consequence of the transposon interruption we were not able to exchange different promotors and signal sequences effieciently. 
Therefore we see great potential in improving enzyme activity by varying these parts in future experiments.
</P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><FOOT><P>REFERENCES</P><OL><LI>
Hu, Hong; He, Jun; Yu, Bing; Zheng, Ping; Huang, Zhiqing; Mao, Xiangbing et al. (2013): Expression of a keratinase (kerA) gene from Bacillus licheniformis in Escherichia coli and characterization of the recombinant enzymes. In: Biotechnology letters 35 (2), S. 239–244.
</LI><LI>
Jaouadi, Nadia Zaraî; Rekik, Hatem; Badis, Abdelmalek; Trabelsi, Sahar; Belhoul, Mouna; Yahiaoui, Amina Benkiar et al. (2013): Biochemical and molecular characterization of a serine keratinase from Brevibacillus brevis US575 with promising keratin-biodegradation and hide-dehairing activities. In: PloS one 8 (10), e76722.
</LI><LI>
Sharma, Richa; Gupta, Rani (2010): Extracellular expression of keratinase Ker P from Pseudomonas aeruginosa in E. coli. In: Biotechnology letters 32 (12), S. 1863–1868.
</LI></OL></FOOT></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>