<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Stuttgart_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Stuttgart/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2017.igem.org/Team:Stuttgart">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Sponsoring">Sponsoring</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Background-Information">Background
                            Information</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Protocols">Protocols &amp; Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/InterLab">Interlab Study</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Model">Modelling</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Composite Part">Composite Part</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Results </A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Esterases and Lipases">Esterases and Lipases</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Keratinases">Keratinases</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Rose and Limonene Fragrance">Rose and Limonene Fragrance</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Demonstrate">Achievements</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/HP/Gold_Integrated">Integrated and Gold</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/HP/Silver">Silver HP</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Awards </A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:Stuttgart/Measurement">Measurement</A></LI><LI><A href="https://igem.org/2017_Judging_Form?team=Stuttgart">Judging Form</A></LI></DIV></DIV><H1 align="middle"> Notebook </H1><DIV class="container"><DIV class="row section"><H2>Esterases and Lipases</H2><DIV class="col-xs-12 col-sm-10 col-md-10"><P><U>03.08.2017</U><UL><LI>Transformation of <A href="http://parts.igem.org/Part:BBa_K1149002">BBa_K1149002</A> and  <A href="http://parts.igem.org/Part:BBa_K1149003">BBa_K1149003</A></LI></UL></P><P><U>04.08.17</U><UL><LI>Single colonies (BBa_K1149002 and BBa_K1149003) are plated on agar plates and incubated at 37 °C over night</LI></UL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-9 col-md-9"><P><U>09.08.17</U><UL><LI>Growing of a single colony (<A href="http://parts.igem.org/Part:BBa_K1149002">BBa_K1149002</A> and <A href="http://parts.igem.org/Part:BBa_K1149003">BBa_K1149003</A>) in 5 mL LB media + chloramphenicol (35 µg/mL) at 37 °C</LI></UL></P><P><U>10.08.17</U><UL><LI>Preparation of miniprep (Jena Biosciences Kit) and glycerol stock (storage: -80 °C) (<A href="http://parts.igem.org/Part:BBa_K1149003">BBa_K1149002</A> and BBa_K1149002)</LI><LI>SOC media preparation</LI><LI>Transformation pet19-LipB</LI></UL></P><P><U>21.08.17</U><UL><LI>Preparation: preliminary test of esterase assay with EstCS2: <A href="http://parts.igem.org/Part:BBa_K1149002">BBa_K1149002</A> - preparation of o/n cultures (37°C)</LI></UL></P><P><U>22.08.17</U><UL><LI>Preparation: preliminary test of esterase assay with EstCS2: <A href="http://parts.igem.org/Part:BBa_K1149002">BBa_K1149002</A> - glycerol stocks and induction with arabinose</LI></UL></P><P><U>23.08.17</U><UL><LI>Preliminary test of esterase assay with EstCS2: <A href="http://parts.igem.org/Part:BBa_K1149002">BBa_K1149002</A> (link results) </LI></UL></P></DIV><DIV class="col-xs-8 col-sm-9 col-md-9"><P><U>24.08.17</U><UL><LI>Transformation of LipB into Zymo Research competent mix and go cells</LI><LI>Preparation of LB medium and new agar plates</LI></UL></P><P><U>25.08.17</U><UL><LI>Single colonies of LipB are plated on agar plates and incubated at 37 °C over night
</LI></UL></P><P><U>28.08.17</U><UL><LI>Preparation of electro-competent cells </LI><LI>Growing of a single colony (LipB) in 5 mL LB media + chloramphenicol (35 µg/mL) at 37°C
</LI></UL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><P><U>31.08.17</U><UL><LI>Transformation/electroporation with iGEM competent cells test kit DNA (o/n incubation, 37°C - transformation failed)</LI></UL></P><P><U>01.09.17</U><UL><LI>Transformation/electroporation with pUC19 plasmid (o/n incubation, 37°C - transformation failed)</LI></UL></P><P><U>06.09-08.09.17</U><UL><LI>Enzyme activity assay (cell pellet and supernatant) with <A href="http://parts.igem.org/Part:BBa_K1149002">BBa_K1149002</A> and pet19-LipB</LI></UL></P><P><U>12.09.17</U><UL><LI>Preparation of chemo-competent cells</LI></UL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-9 col-md-9"><P><U>13.09.17</U><UL><LI>Transformation with iGEM competent cells test kit DNA and pUC19 plasmid (o/n incubation, 37°C) - transformation successful </LI></UL></P><P><U>14.09.17</U><UL><LI>Calculation - efficiency of the chemo-competent cells/pUC19:
efficiency E =(56cfu/0.001ng)*1000ng/(µL)=5.6*((10)^7 )cfu/µL</LI></UL></P><P><U>18.09.17</U><UL><LI>Preparation InterLab study - transformation of 8 parts into DH5alpha <I>E. coli</I> cells</LI></UL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><P><U>14.09.17</U><UL><LI>Calculation - efficiency of the chemo-competent cells/pUC19:
efficiency E =(56cfu/0.001ng)*1000ng/(µL)=5.6*((10)^7 )cfu/µL</LI></UL></P><P><U>18.09.17</U><UL><LI>Preparation InterLab study - transformation of 8 parts into DH5alpha <I>E. coli</I> cells</LI></UL></P><P><U>19.09.17 – 21.09.17 </U><UL><LI>InterLab study LUDOX measurement</LI><LI>Enzyme activity assay with <A href="http://parts.igem.org/Part:BBa_K1149002">BBa_K1149002</A> of the supernatant - measurement in biological triplicates </LI><LI>InterLab study Fluorescein measurement</LI></UL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-10 col-sm-9 col-md-9"><P><U>19.09.17</U><UL><LI>Preparation PCR Purification Lipase:</LI></UL><OL><LI>prtE-f4: 1410 bp</LI><LI>prtF_f5: 1691 bp</LI><LI>LARD_f2: 683 bp</LI><LI>prtD_f3: 683 bp</LI><LI>LARD_f2: 683 bp</LI><LI>pBAD_f1: 1660 bp</LI></OL></P><P><U>22.09.17</U><UL><LI>InterLab study sample measurement (link results)</LI></UL></P><P><U>27.09.17 - 29.09.17</U><UL><LI>Enzyme activity assay with pet19-LipB of the supernatant - measurement in biological triplicates</LI></UL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><P><U>22.09.17</U><UL><LI>InterLab study sample measurement (link results)</LI></UL></P><P><U>27.09.17 - 29.09.17</U><UL><LI>Enzyme activity assay with pet19-LipB of the supernatant - measurement in biological triplicates</LI></UL></P><P><U>06.10.17</U><UL><LI>Collaboration iGEM team Heidelberg: preparation mutagenesis plasmid activity assay - preparation of media, antibiotic-stocks and sugar-stocks + agar plates with different antibiotics</LI></UL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-9 col-md-9"><P><U>10.10.17</U><UL><LI>Collaboration iGEM team Heidelberg: preparation of different cultures + induction of the cells with arabinose (o/n incubation, 37°C)</LI></UL></P><P><U>11.10.17</U><UL><LI>Collaboration iGEM team Heidelberg: spread the o/n cultures on prepared agar plates (o/n incubation, 37°C)</LI></UL></P><P><U>12.10.17</U><UL><LI>Gibson Assembly of LipB in psB1C3 backbone</LI></UL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-9 col-md-9"><P><U>19.10.17</U><UL><LI>PCR - amplification of LipB - cloning of LipB in the iGEM standard plasmid (<A href="http://parts.igem.org/Part:BBa_K1149002">BBa_K1149002</A> is used)</LI><LI>PCR - amplification of <A href="http://parts.igem.org/Part:BBa_K1149002">BBa_K1149002</A> (without EstCS2)</LI></UL></P><P><U>20.10.17</U><UL><LI>SDS page, Gibson Assembly, Transformation (Zymos) - cloning of LipB in the iGEM standard plasmid (BBa_K1149002 is used)</LI></UL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-9 col-md-9"><P><U>23.10.17</U><UL><LI>5x pelB LipB with 5ml LB over night at 37°C </LI></UL></P><P><U>24.10.17</U><UL><LI>Miniprep (Jena Bioscience Kit)</LI><LI>Induction of enzyme expression for the enzyme activity assay for the Gibson Assembly (PelB-LipB), LipB in iGEM standard plasmid (BBa_K1149002 is used)
</LI><LI>Sequencing of the Gibson Assembly (PelB-LipB)</LI></UL></P><P><U>25.10.17</U><UL><LI>Enzyme activity assay (PelB-LipB), LipB in iGEM standard plasmid (<A href="http://parts.igem.org/Part:BBa_K1149002">BBa_K1149002</A> is used)</LI></UL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-9 col-md-9"><P><U>26.10.17</U><UL><LI>OD<SUB>600</SUB> determination of Lipase TliA</LI></UL></P><P><U>27.10.17</U><UL><LI>Colony PCR of the Gibson Assembly product (PelB-LipB), SDS-PAGE</LI></UL></P></DIV></DIV></DIV><DIV class="container"><DIV class="row section"><H2>Keratinases</H2><DIV class="col-xs-12 col-sm-10 col-md-10"><P><U>26.07.17</U><UL><LI>Transformation of kerUS (<A href="http://parts.igem.org/Part:BBa_K11498000">BBa_K1498000</A>)</LI><LI>Preparation of antibiotic stock solutions: chloramphenicol (35 mg/mL) and ampicillin (100 mg/mL)</LI></UL></P><P><U>27.07.17</U><UL><LI>Growing of a single colony (kerUS (<A href="http://parts.igem.org/Part:BBa_K11498000">BBa_K1498000</A>)) in 5 mL LB media chloramphenicol (35 µg/mL) at 37 °C</LI><LI>Single colonies are plated on agar plates and incubated at 37 °C over night</LI><LI>Growing of a single colony (<A href="http://parts.igem.org/Part:BBa_K1149002">BBa_K149002</A> and <A href="http://parts.igem.org/Part:BBa_K1149003">BBa_K149003</A>) in 5 mL LB media + chloramphenicol (35 µg/mL) at 37 °C</LI><LI>Transfer of <A href="http://parts.igem.org/Part:BBa_K11498000">BBa_K1498000</A> into glycerin culture and storage at -80° freezer</LI></UL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-9 col-md-9"><P><U>28.07.17</U><UL><LI>Preparation of: miniprep (Jena Biosciences Kit)and glycerol stock (kerUS (BBa_K1498000))</LI></UL></P><P><U>16.08.17</U><UL><LI>Transformation of three promotors: BBa_J23102, BBa_J2314 and BB-K206000</LI></UL></P><P><U>17.08.17</U><UL><LI>Transformation of three promotors: BBa_J23102, BBa_J2314 and BB-K206000 again, after nothing grew on the agar plates</LI><LI>Competent cell test - not succesfull</LI></UL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><P><U>22.08.17</U><UL><LI>Preparing LB (5ml tubes) and (chemical) competent cells (link prtokoll openwetware)</LI></UL></P><P><U>23.08.17</U><UL><LI>preparing primer for overlapping PCR to get restriction sides to kerP:
-&gt; preparing and dilute, following IDT protocols.</LI></UL></P><P>PCR-cycler conditions:</P><TABLE align="middle" hspace="10"><TBODY><TR><TH>Step</TH><TH>Cycles</TH><TH>Temperature</TH><TH>Time</TH></TR><TR><TD>Denaturation</TD><TD>98°C</TD><TD>30 sec</TD></TR><TR><TD>Annealing</TD><TD>35</TD><TD>68°C</TD><TD>30 sec</TD></TR><TR><TD>Elongation</TD><TD>72°C</TD><TD>30 sec</TD></TR><TR><TD>final extension</TD><TD>1</TD><TD>72°C</TD><TD>2 min</TD></TR><TR><TD>hold</TD><TD>4-10°C</TD></TR></TBODY></TABLE><L><UL>PCR-Purification with JenaBioScience-kit</UL></L></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-9 col-md-9"><P><U>24.08.17</U><UL><LI>Ligation: Digest kerP/pSB1K3, kerP + linearizied plasmid backbone pSB1K3</LI><LI>Transformation of three promotors: BBa_J23102, BBa_J2314 and BB-K206000 again into Zymo Research Competent mix and go cells this time, after nothing grew on the agar plates
</LI></UL></P><P>
     To add our keratinase genes with promotors we transformed BBa_J23115, BBa_J23119 and an inducible pBAD promotor BBa_K206000 and cut cleaned up plasmids with EcoR1 and Spe1 for Standard Assembly. 
     After digest with second enzyme, for J23115 and J23119 only plasmid backbones were detected. 
      In lines of K206000 two fragments were detected, as expected (insert and backbone) (figure ). 
</P></DIV><DIV class="col-xs-12 col-sm-3 col-md-3"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:270px;"><DIV class="thumbcaption"><B>Figure 1:</B> Agarose-gel electrophoresis of ligation products.
</DIV></DIV></DIV></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-10 col-md-10"><P><U>25.08.17</U><UL><LI>New transformation of three promotors: BBa_J23102, BBa_J2314 and BB-K206000 again into Zymo Research Competent mix and go on new agar plates from 24.8.17</LI></UL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-8 col-sm-8 col-md-8"><P><U>28.08.17</U><UL><LI>Preparation of electro-competent cells </LI><LI>Preparation of electro-competent cells </LI><LI>Preparation of new agar plates with Ampicilline and Chloramphenicole
</LI><LI>preculturing <I>E.coli</I> for electroporethic competent cell assay
</LI><LI>Single colonies of promotor (BB_K206000) are plated on agar plates and incubated at 37 °C over night ( other promotors didn’t grow again)</LI><LI>Transformation of KerA and KerUS on pSB1C3 vector from Canada into Zymo competent mix and go cells (KeratinaseA: BBa_K1717000 , KeratinaseUS: BBa_K1717173)</LI></UL></P><P><U>29.08.17</U><UL><LI>Single colonies of (KeratinaseA: BBa_K1717000 , KeratinaseUS: BBa_K1717173) are plated on agar plates and incubated at 37 °C 
</LI><LI>Growing of a single colony (BBa_K206000) in 5 mL LB media + ampicilline (100 µg/mL) at 37°C
</LI></UL></P></DIV><DIV class="col-xs-4 col-sm-4 col-md-4"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:;"><DIV class="thumbcaption"><B>Figure 2:</B> Digest of kerP plasmids.
	</DIV></DIV></DIV></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-9 col-md-9"><P><U>30.08.17</U><UL><LI>Transformation of psB1K3-KerP</LI><LI>Repeat Ligation with another backbone: Digest kerP/pSB1K3, kerP + linearizied plasmid backbone pSB1K3</LI><LI>Transformation of three other promotor (BBa_J23100, BBa_J23119, BBa_J23119) and one RBS (BBa_B0034) into Zymo Competent Mix and Go cells
</LI><LI>Preparation Mini-Prep and glycerol storage at -80°C freezer of promotor (BBa_K206000): final concentration: 77,8 ng/µl
</LI><LI>Growing of a single colony (KeratinaseA: BBa_K1717000 , KeratinaseUS: BBa_K1717173) in 5 mL LB media + chloramphenicol (35 µg/mL) at 37°C
</LI></UL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><P><U>31.08.17</U><UL><LI>Transformation of Ligation-product kerP_pSB1K3 with electroporation
preparing competent cells (protocol igem)</LI><LI>Preparation Mini-Prep and glycerol storage at -80°C freezer of (KeratinaseA: BBa_K1717000 , KeratinaseUS: BBa_K1717173): final concentration: BB-K1717000: 305 ng/µl, BBa_K1717173: 232,4 ng/µl
</LI></UL></P><P><U>01.09.17</U><UL><LI>Mini-prep (jenabioscience kit): BBa_23119, BBa_23115, RBS BBa_B0034</LI></UL></P><P><U>04.09.17</U><UL><LI>Transformation of BBa_J04450 -&gt; making pSB1K3 backbone</LI></UL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-9 col-md-9"><P><U>05.09.17</U><UL><LI>Colony-PCR with colony kerP_pSB1K3</LI><LI>Transformation of two signal peptides: pelB: BBa_K208004 and OmpA: BBa_ K103006 in Zymo Competent mix and go cells
</LI></UL></P><P><U>06.09.17</U><UL><LI>Transformation of BBa_J04450 cut (with E and P) and kerP-&gt; Zymo Mix &amp; Go</LI><LI>Single colonies of signal peptides BBa_K208004 and BBa_ K103006 are plated on agar plates and incubated at 37 °C </LI></UL></P><P><U>07.09.17</U><UL><LI>Primer-Design and ordering</LI><LI>Growing of a single colony (BBa_K208004 and BBa_ K103006) in 5 mL LB media + chloramphenicol (35 µg/mL) at 37°C
</LI><LI>Prepration of LB medium and agar plates</LI></UL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><P><U>08.09.17</U><UL><LI>Preparation Mini-Prep and glycerol storage at -80°C freezer of signal peptides (BBa_K208004 and BBa_ K103006): BBa_K208004: final concentration: 268,7 ng/µl and BBa_K103006: 187,9 ng/µl
</LI></UL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-8 col-sm-8 col-md-8"><P><U>11.09.17</U><UL><LI>pSB1K3_kerP + 5ml LB over night at 37°C, BBa-J04450 + 5ml LB over night at 37°C</LI></UL></P><P><U>13.09.17</U><UL><LI>Mini-prep kerP and BBa_J04450</LI></UL></P><P><U>15.09.17</U><UL><LI>Overlapping PCR for kerP,kerA,kerUS and signal peptids OmpA/pelB</LI></UL></P></DIV><DIV class="col-xs-4 col-sm-4 col-md-4"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:;"><DIV class="thumbcaption"><B>Figure 3:</B></DIV></DIV></DIV></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-9 col-md-9"><P>PCR-cycler conditions:</P><TABLE align="middle" hspace="10"><TBODY><TR><TH>Step</TH><TH>Cycles</TH><TH>Temperature</TH><TH>Time</TH></TR><TR><TD>Denaturation</TD><TD>98°C</TD><TD>30 sec</TD></TR><TR><TD>Annealing</TD><TD>35</TD><TD>68°C</TD><TD>30 sec</TD></TR><TR><TD>Elongation</TD><TD>72°C</TD><TD>2 min</TD></TR><TR><TD>final extension</TD><TD>1</TD><TD>72°C</TD><TD>2 min</TD></TR><TR><TD>hold</TD><TD>4-10°C</TD></TR></TBODY></TABLE><P><U>18.09.17</U><UL><LI>Agarose gel with PCR products,
repeat overlapping PCR for kerP,kerA,kerUS and signal peptids OmpA/pelB
split of for better range of annealing temperature (65°C)
</LI><LI>Preparation PCR for KeratinaseA: BBa_K1717000 , KeratinaseUS: BBa_K1717173
</LI></UL></P><P><U>19.09.17</U></P><P><UL><LI>Preparation PCR Purification:</LI></UL><OL><LI>OA-A: 73,4 ng/µl</LI><LI>KA-OA:90,4ng/µl</LI><LI>KA-PB: 77,4 ng/µl</LI><LI>PB-A: 22,8 ng/µl</LI><LI>KUS-PB: 110,8 ng/µl</LI><LI>PB/US: 13,9 ng/µl</LI><LI>KUS-OA:  81,9 ng/µl</LI><LI>OA-US: 99.5 ng/µl</LI></OL></P><P><UL><LI>Gibson Calculater: </LI></UL><OL><LI>Vektor: 163 ng/µl</LI><LI>KerA+OmpA: 173 ng/µl</LI><LI>OA-A: 162 ng/µl</LI><LI>Kerus-OmpA, Vektor: 176 ng/µl</LI><LI>KUS-OA: 188 ng/µl</LI><LI>OA-US: 175 ng/µl</LI></OL><UL><LI>Transformation into Zymo competent mix and go cells afterwards.</LI></UL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-9 col-md-9"><P><U>06.10.17</U><UL><LI>Preparation of Keratinase-Assay. Due to troubles to dilute Azo-Keratine, Assay was not successful and has to be repeated.</LI><LI>Preparation of Azo-Keratine (milling) in Tris/HCL Buffer (pH 8) (2,94 g Tris into 500 ml HCL Buffer)</LI></UL></P><P><U>10.10.17</U><UL><LI>Keratinase assay -&gt; preparing substrate azure keratin, problems with insoluble substrate</LI></UL></P><P><U>12.10.17</U><UL><LI>Gibson Assembly KerA-ompA, ompA-KerA, KerUS-ompA and ompA-KerUS in psB1C3 backbone</LI></UL></P><P><U>16.10.17</U><UL><LI>Preparation semi-quantitative keratinase-assay: spread kerUS, KerA and KerP on chloramphenicol/kanamycin agar plates (o/n incubation, 37°C)</LI><LI>Preparation skim-milk-plate assay: preparation of skim-milk-agar-plates with chloramphenicol/kanamycin</LI></UL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><P><U>17.10.17</U><UL><LI>Loading sceme on gel for restriction digest for kerA and KerUS
        M-KerA-KerUS-BBa_J23115-BBa_J23119_BBa_K206000</LI></UL></P><P><U>18.10.17</U><UL><LI>Preparation semi-quantitative keratinase-assay: preparation of o/n cultures (37°C)</LI><LI>Preparation skim-milk-plate assay: preparation of o/n cultures (37°C)</LI><LI>Preparation of miniprep (Jena Bioscience), standard assembly (used restriction enzymes: EcoRI, XbaI, SpeI) and transformation of: KerA-OmpA and KerUS-OmpA with each of them combined to three promotors: BBa_J23119, BBa_J23115, BBa_K206000.
</LI></UL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-9 col-md-9"><P><U>19.10.17</U><UL><LI>Semi-quantitative keratinase-assay: add 0.005 g of dried hair (65 °C, 1h) to o/n cultures (KerUS and KerA are induced with 1mM IPTG before) - 5 days incubation, 37°C (link results)</LI><LI>Skim-milk-plate assay kerP - spread on prepared skim-milk agar plates (supernatant and cells) - 4 days incubation, room temperature </LI></UL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-10 col-md-10"><P><U>20.10.17</U><UL><LI>Skim-milk-plate assay kerUS and KerA - spread on prepared skim-milk agar plates (supernatant and cells) - 4 days incubation, room temperature (link results)</LI><LI> Sequencing of keratinase plasmids: pSB1C3-KerA-ompA, pSB1C3-KerUS-ompA, pSB1C3-KerA_Kanada and pSB1C3-KerUS-Kanada</LI></UL></P><P><U>24.10.17</U><UL><LI>Ligation with kerP digest and pSB1C3 backbone</LI></UL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-9 col-md-9"><P><U>25.10.17</U><UL><LI>Analysis of Sequencing (GATC) of different fragments:</LI></UL><OL><LI>KerA-OmpA: FW: 1057 bp, RV: 772 bp</LI><LI>KerUS-OmpA: FW: 1048 bp, RV: 789 bp</LI><LI>KerA (BBa_K1717000): FW: 1107 bp, RV: 837 bp</LI><LI>KeratinaseUS (BBa_K1717173): FW: 1148 bp</LI></OL><UL><LI>Measurment of DNA concentration with Nano-Drop:</LI></UL><OL><LI>Biobasic KerUS pet28b+:30,3 ng/µl</LI><LI>Biobasic KerA pet28b+:74,0 ng/µl</LI><LI>KerUS pet28b+:1286,7 ng/µl</LI><LI>KerA pet28b+:2465,4 ng/µl</LI><LI>KerUS pSB1C3:1588,2 ng/µl</LI><LI>KerA pSB1C3:1826,2 ng/µl</LI></OL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-9 col-md-9"><UL><LI>GATC Sequencing of:</LI></UL><OL><LI>KerP pet28b+ </LI><LI>KerUS pSB1C3</LI><LI>KerA pSB1C3</LI><LI>KerA pet28b+</LI><LI>KerUS pet28b+</LI></OL><P><U>26.10.17</U><UL><LI>Preparation of Lipase Assay with 0M and 3M induction of IPTG and five different substrate concentrations: 2,5; 5; 10 ; 15; 20 µg/ml</LI></UL></P><P><U>30.10.17</U><UL><LI>Cell Lysis of different Keratinases strains (look it up from the days before)</LI></UL></P></DIV></DIV><DIV class="container"><DIV class="row section"><H2>Rose and Limonene Fragrance</H2><DIV class="col-xs-12 col-sm-10 col-md-10"><P><U>12.07.17</U><UL><LI>Preparation of: miniprep (Jena Biosciences Kit)and glycerol stock (pET28a-KDC-YjgB-ARO8: 351,9 ng/µl and pET28a-ATF1: 352,5 ng/µl)</LI><LI>  Preparation of Kanamycin stocks (50µg/ml)</LI></UL></P><P><U>14.07.2017</U><UL><LI>Preparation of LB-Agar plates with kanamycin (50 µg/mL)</LI></UL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-9 col-md-9"><P><U>28.08.17</U><UL><LI>Preparation of LB-Agar plates with kanamycin (50 µg/mL)</LI><LI> Sequencing of rose fragrance plasmids from Guo et al. (pET28a-KDC-YjgB-ARO8 and pET28a-ATF1) </LI></UL></P><P><U>13.09.17</U><UL><LI>Overlap-PCR of 1.pBAD (BBa_K206000), 2.KDC-YjgB-ARO8, 3.ATF1 and 4.pBAD-KDC-YjgB-ARO8</LI><LI>PCR-Purification and agarose-gel-electrophoresis of PCR products, PCR 4 (pBad-KDC-YjgB-ARO8) not sucessfull</LI></UL><UL><LI>Measurment of DNA concentration with Nano-Drop:</LI></UL><OL><LI>KDC-YjgB-ARO8:54,4 ng/µl</LI><LI>ATF1:57,4 ng/µl</LI><LI>pBAD:140,7 ng/µL</LI></OL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><P><U>14.09.17</U><UL><LI>Preparation PCR Lemonen and Agarose-Gel
</LI></UL></P><P><U>19.09.17</U><UL><LI>Preparation of LB-Agar plates with kanamycin (50 µg/mL) and chloramphenicol (35 µg/mL)</LI><LI>Repeat of PCR 4 (pBAD-KDC-YjgB-ARO8)</LI><LI>PCR-Purification and agarose-gel-electrophoresis of PCR product 4</LI><LI>PCR 4 (pBAD-KDC-YjgB-ARO8) not successful</LI><LI>Transformation of Limonene-plasmid and over-night incubation on agar-plates with kanamycin (50 µg/mL) at 37°C </LI></UL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-6 col-md-6"><P><U>21.09.17</U><UL><LI>Gibson Assembly of rose PCR-overlap products pBAD, KDC-YjgB-ARO8 and ATF1 in psB1K3 backbone</LI></UL><UL><LI>Gibson Calculater: </LI></UL><OL><LI>pSB1K3: 0,55 µL</LI><LI>pBAD: 0,74 µL</LI><LI>KDC-YjgB-ARO8: 1,91 µL</LI><LI>ATF1: 1,81 µL</LI></OL><UL><LI>Transformation of assembled rose-plasmid and over-night incubation on agar-plates with kanamycin (50 µg/mL) at 37°C</LI></UL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-9 col-md-9"><P><U>22.09.17</U><UL><LI>Rose-plasmid Transformation successful</LI><LI>Single colonies are plated on agar plates and incubated at 37 °C over night</LI></UL></P><P><U>25.09.17</U><UL><LI>Verification of transformed rose-plasmid by colony-PCR</LI><LI>Agarose-gel-electrophoresis of colony-PCR product - colony-PCR not successful</LI></UL></P><P><U>26.09.17</U><UL><LI>Repeat of colony-PCR with transformed rose-plasmid</LI><LI>Agarose-gel-electrophoresis of colony-PCR product - colony-PCR not successful</LI></UL></P><P><U>27.09.17</U><UL><LI>Repeat of colony-PCR with transformed rose-plasmid</LI><LI>Agarose-gel-electrophoresis of colony-PCR product - colony-PCR not successful</LI></UL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-9 col-md-9"><P><U>28.09.17</U><UL><LI>Mini-prep of transformed rose-plasmid</LI></UL><UL><LI>Nanodrop Measurement: </LI></UL><OL><LI>Colony 1: 192,1 ng/µL</LI><LI>Colony 2: 159,8 ng/µL</LI><LI>Colony 3: 159,1 ng/µL</LI><LI>Colony 4: 100,1 ng/µL</LI><LI>Colony 5: 118,2 ng/µL</LI></OL><UL><LI>Restriction assay of isolated rose-plasmid, cut with SpeI</LI><LI>Verification of restriction product by agarose-gel-electrophoresis - not successful</LI></UL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-12 col-md-12"><P><U>29.09.17</U><UL><LI>Repeat Gibson Assembly of rose PCR-overlap products pBAD, KDC-YjgB-ARO8 and ATF1</LI><LI>Transformation of assembled rose-plasmid and over-night incubation on agar-plates with kanamycin (50 µg/mL) at 37°C - transformation not successful </LI></UL></P><P><U>05.10.17</U><UL><LI>Repeat Restriction assay of isolated rose-plasmid, cut with EcoRI - not successful</LI><LI>Verification of restriction product by agarose-gel-electrophoresis - not successful</LI></UL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-9 col-md-9"><P><U>12.10.17</U><UL><LI>Gibson Assembly of limonene PCR-products ()  in psB1C3 backbone</LI><LI>Transformation of assembled rose-plasmid and over-night incubation on agar-plates with kanamycin (50 µg/mL) at 37°C - transformation not successful </LI></UL></P><P><U>17.10.17</U><UL><LI>Gibson-Assembly:</LI></UL><OL><LI>PSB1C3: 1,75 µl</LI><LI>F1: 1,36 µl</LI><LI>F2: 1,89 µl</LI></OL></P></DIV></DIV><DIV class="row section"><DIV class="col-xs-12 col-sm-9 col-md-9"><P><U>18.10.17</U><UL><LI>Repeat of colony-PCR with transformed rose-plasmid, temperature range from 58-70 °C</LI><LI>Agarose-gel-electrophoresis of colony-PCR product - colony-PCR not successful</LI><LI>M9 media preparation</LI></UL></P><P><U>20.10.17</U><UL><LI>Repeat overlap-PCR of pBad and KDC-YjgB-ARO8</LI><LI>Verification of PCR products by agarose-gel-electrophoresis</LI></UL></P><P><U>23.10.17</U><UL><LI> Sequencing of transformed rose fragrance plasmid (pSB1K3-pBad-KDC-YjgB-ARO8-ATF1) </LI><LI> Sequencing of transformed limonene fragrance plasmid (pSB1C3-pBad) </LI></UL></P><P><U>25.10.17</U><UL><LI>Repeat Gibson Assembly of rose PCR-overlap products pBAD, KDC-YjgB-ARO8 and ATF1, plasmid backbones pSB1K3 and pSB1C3</LI><LI>Transformation of assembled rose-plasmid and over-night incubation on agar-plates with kanamycin (50 µg/mL) and chloramphenicol (35 µg/mL) at 37°C - transformation not successful </LI></UL></P><P><U>26.10.17</U><UL><LI>Repeat transfomation of rose-plasmid in competent NEB-cells and and over-night incubation on agar-plates with kanamycin (50 µg/mL) and chloramphenicol (35 µg/mL) at 37°C - transformation successful </LI></UL></P><P><U>27.10.17</U><UL><LI>Verification of transformed rose-plasmid by colony-PCR</LI><LI>Agarose-gel-electrophoresis of colony-PCR product - colony-PCR not successful</LI></UL></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>