<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_LCG-UNAM-Mexico_Notebook_2008-May"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:LCG-UNAM-Mexico/Notebook/2008-May">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:LCG-UNAM-Mexico/Notebook/2008-May&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:LCG-UNAM-Mexico/Notebook/2008-May&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:LCG-UNAM-Mexico/Notebook/2008-May&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:LCG-UNAM-Mexico/Notebook/2008-May" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:LCG-UNAM-Mexico/Notebook/2008-May</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><P><TITLE>LCG-UNAM-Mexico:Notebook/Mayo</TITLE><TABLE width="82%" border="0" cellspacing="0" cellpadding="0"><TBODY><TR bgcolor="#D5EDB3"><TD height="50" colspan="3" id="logo" valign="bottom" align="center" nowrap="nowrap">LCG-UNAM-Mexico</TD></TR><TR bgcolor="#D5EDB3"><TD height="51" colspan="3" id="tagline" valign="top" align="center">iGEM 2008 TEAM</TD></TR><TR bgcolor="#99CC66"><TD colspan="7" id="dateformat" height="20">  </TD></TR><TR><TD width="165" bgcolor="#5C743D">  </TD></TR><TR><TD width="165" bgcolor="#5C743D"><A href="https://2008.igem.org/Team:LCG-UNAM-Mexico" class="navText">Home</A></TD></TR><TR><TD width="165" bgcolor="#5C743D"><A href="https://2008.igem.org/Team:LCG-UNAM-Mexico/Project" class="navText">Our project</A></TD></TR><TR><TD width="165" bgcolor="#5C743D"><A href="https://2008.igem.org/Team:LCG-UNAM-Mexico/Modeling" class="navText">Modeling</A></TD></TR><TR><TD width="165" bgcolor="#5C743D"><A href="https://2008.igem.org/Team:LCG-UNAM-Mexico/Experiments" class="navText">Wet Lab</A></TD></TR><TR><TD width="165" bgcolor="#5C743D"><A href="https://2008.igem.org/Team:LCG-UNAM-Mexico/Parts" class="navText">Parts Submitted to the Registry</A></TD></TR><TR><TD width="165" bgcolor="#5C743D"><A href="https://2008.igem.org/Team:LCG-UNAM-Mexico/Notebook" class="navText">Notebook</A></TD></TR><TR><TD width="165" bgcolor="#5C743D"><A href="https://2008.igem.org/Team:LCG-UNAM-Mexico/Story" class="navText">Our story</A></TD></TR><TR><TD width="165" bgcolor="#5C743D"><A href="https://2008.igem.org/Team:LCG-UNAM-Mexico/Team" class="navText">About us</A></TD></TR></TBODY></TABLE>      <TABLE border="0" cellspacing="0" cellpadding="2" width="590"><TBODY><TR><TD class="pageName"><STRONG>May</STRONG></TD></TR></TBODY></TABLE></P>2008-05-07<DIV align="justify"><DL><P><B><U>GROUP SESSION:</U></B><B>Preliminary Projects Group Presentation (2nd Round)</B><DT>Discussion of Papers &amp; Ideas: <I>Sprinzak &amp; Elowitz, 2005. </I></DT><DD>Combinatorial experiment. </DD><DD>Three TF and 5 binding sites. </DD><DD>Combinatorial and phenotypes analysis. </DD><DD>With a few well characterized elements much can be done. </DD><B>Things to consider:</B><LI><DT><B>Next steps, future models:</B></DT></LI><UL><LI>Protein level phosphorylation (example: Phytocrome found in cyanobacteria that get phosphorylated when activated with a specific wavelength and eventually trigger the activation of TF.</LI><LI>miRNA (only in eukaryotes, not found in yeast)</LI></UL><LI><B>Sequence-specific Ribosomes.</B> (few well characterized) </LI><LI><B>Repressilator:</B> Three genes that repress each other. </LI><LI><B>Signaling cascades </B></LI><UL><LI>Not exploited yet.</LI><LI>Response time delay. </LI><LI>It is mentioned as the simplest system. </LI><LI>The noise is accumulated in these systems. </LI><LI>Design a signaling cascade and see how noise affects and spreads throughout the system. </LI><LI>Compare the noise of a two element cascade with the one of a three-element cascade. </LI><LI>There is evidence that to reduce noise in a noisy system, noise is introduced into the noise, and as a result the system becomes ordered. </LI><LI>Increase copies of the elements and see how does the noise is affected. <I>Theory says that noise should decrease, see if this system converges to a deterministic system.</I></LI><LI>Compare strains with different protein production ratios. <B>*EXPLOIT MORE THIS POINT*</B>. </LI><LI>See if it is easy to do. However in this system it is very unlikely to find multi-stationary states. </LI><LI>Measuring noise is a potential problem. Do we have access to technology to measure it?</LI></UL><LI><B>Experimental Technique:</B> Promoters recombination. </LI><DD>Suggestion: Use PCR to avoid direct recombination. </DD><LI>Investigate the difficulty to study the competition between the two enzymes for same metabolite in a metabolic pathway.</LI><LI>In positive circuits it is possible to see multiple states.</LI><LI>It is not enough to have the network topology, the context also influences the outcome. 
We need to check whether the module works in the desired conditions. Two modules working separately doesn't mean that they will work when joined together.</LI></P>2008-05-11<DIV align="justify"><P><B><U>GROUP SESSION:</U></B><STRONG>Ideas:</STRONG><LI>Counting in module 2 (even and odd)        2nd Option</LI><LI>Mosaic                               REJECTED</LI><LI>Singing bacteria.                         Researching</LI><LI>Pong                                Researching</LI><LI>Balance between populations             2nd Option</LI><LI>Counting days                         REJECTED</LI><B>Pong Research: 
Possibilities </B> <B>Bacteria motility:</B> 
The motile cell will trigger a repellent in the cell population to which it is approaching, along with a reporter, then it would move to the second cell population, and in this way they  will share roles.<LI><STRONG>Repellents</STRONG> -&gt; weak organic acids such as formic, acetic acid, benzoic, and salicylic <I>[works by decreasing intracellular pH]</I>. Amino acids as leucine and valine, indol. Also alcohols, polyols, and  heavy cations (Co and Ni)</LI><LI>The stimulus is detected by a membrane receptor (MCP methyl accepting chemotaxis proteins) TSR, Tar, Trg, Tap <I>[These 4 described in <I>E.coli</I>]</I>. </LI><LI>The signaling cascade is initialized when a conformational change is induced by the ligand in the cytoplasmic side of the protein. This change is recognized by a complex formed by CheA-CheW. CheA is a kinase that is inhibited when the ligand recognizes attractors and is activated when repellent is present. CheA donates its phosphate to CheY and CheB, CheY-P interacts with a complex that has a switch function at the base engine scourge, this switch is responsible for the change between CW [clockwise] and CCW [counterclockwise]. Studied in S. thypimurium and E. coli.</LI><LI>We still face the problem of degrading the existing repellent [activator], and accumulation of noise. </LI><LI>The point would be to generate a molecule in which we can control its expression, maybe a hormone, but we would need  to generate a chimera to recognize our hormone and trigger the cascade that indicates repression to CW-swim.
As we see, it is the same as the project submitted by the Rice University but with the difference that it will be a repellent version. So if we try to do this project it will be just doing another version of something already done. </LI><LI>We could use two cell types, one that produces the signal and another which is responsible for degrading it, but working under induction, which are activated when the motile bacteria reach any of the poles. Something like flageline receptors or any membrane protein of the motile bacteria. 
</LI><LI><B>Using an <I>E.coli</I> attractor</B>-&gt; we could use the Rice University biobrick [iGem 2006-seek and destroy e. - coli]. Where <I>B. subtilis</I> produces a pheromone that serves as <I>E.coli</I> attractor, which use a chimeric receptor that recognizes the pheromone and the intramembranal MCP domain inducing the chemotaxis route. When it finds a <I>B. subtilis</I>, it expresses GFP or kills <I>B. subtilis</I>. </LI><LI><B>By quorum sensing and a signal degrader that is not disseminated.</B>
Using AHL, LuxR and degradation AHL; playing with  thresholds and proteins lifes.
Using the Imperial College oscillator model, but in the same cell.
</LI><LI><B>Using efflux pumps. </B><B>How complex are they?</B><I>Few genes (~ 3). </I>
Finding one that expresses something that can be interpreted as a sign.<B>Tetracycline pumps</B>
We have a  Tetracycline-sensitive promoter in bioparts (R0040)- it is constitutively produced and is suppressed by the presence of tetracycline. <B>Papers:</B><I>Ramos et al, 2005: Big Review of tetR . </I> 
We use tetR as a repressor of a tetA, which exports tetR into the medium. Playing with thresholds and the lifetime of the repressor of tetA we could manage cell populations to oscillate in order to launch signals from one side to another.</LI><B>Singing Bacteria Research
Possibilities:</B><LI>Cl Channels (bop / hop), activated by light. Measured  with electrodes. We need halotolerant bacteria.</LI><LI>Efflux tetracycline Pumps. Same, measured with electrodes. You can use <I>E. coli</I>. We do not know how significative will the fluctuations be.</LI><B>Notes:</B><UL><LI><I>Halobacterium salinarum</I> NRC1 and R1 have the two genes:<A href="http://www.halolex.mpg.de/public/RegionViewer?strain=Halobacterium_salinarum.R1.public_MPIB&amp;contig=CHR&amp;start=155343&amp;stop=157167">hop </A>(825bp) and <A href="http://www.halolex.mpg.de/public/RegionViewer?strain=Halobacterium_salinarum.R1.public_MPIB&amp;contig=CHR&amp;start=1081741&amp;stop=1083529">bop</A> (789pb)</LI><LI>Modified it replicates in about an hour, the WT in about 12hrs(NRC1)</LI><LI>Small genes (&gt;3Kb)</LI><LI>We could use Halorodopsines and Bacteriorodopsines, and as an alternative efflux pumps contained in plasmids and that work on the antibiotics, so we could transform light into sound or only measure the concentrations.</LI><LI>We love the idea :P</LI><LI>The electrode part is not difficult</LI><LI>We would make a model of the system to regulate both pumps</LI><LI>Program a script for the “singing sound”</LI><LI>Rhodopsines react to different light frequencies(colors) generating different concentrations in the medium and depending on those concentration, they could “say” the color of the light that it is receiving.</LI><LI>There is a paper that has much to do with our problem: <A href="http://www.biophysj.org/cgi/content/abstract/83/4/1749">  &quot;Cl  Concentration Dependence of Photovoltage Generation by Halorhodopsin from <I>Halobacterium salinarum</I>&quot;</A></LI><LI>It is possible to move this system into <I>E. coli</I>(Luis found a <A href="http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B6T36-4BTY92K-6&amp;_user=945819&amp;_rdoc=1&amp;_fmt=&amp;_orig=search&amp;_sort=d&amp;view=c&amp;_acct=C000048981&amp;_version=1&amp;_urlVersion=0&amp;_userid=945819&amp;md5=925a785c26b2dc68677ad594dea82ed1">paper </A>that proves it)</LI><LI>There is a specific<A href="http://www.dsmz.de/microorganisms/html/media/medium000097.html"> medium for Halobacterium</A> and its cost is $12 USD</LI><LI>Paper: <I>“Genetic Transfer in Halobacterium volcanii&quot; Moshe Mevarech &amp; Ruth Werczberger. Journal of bacteriology 1985. Vol 162 No 1.</I>. In the Methods section the author speak about some culture media.  They point out some auxotroph mutants. <B>It took 6-10 days to grow</B></LI><LI>Paper: <I>&quot;Bacteriorhodopsin production by cell recycle culture of Halobacterium halobium&quot; Sang Yup Lee et. al. Biotechnology Letters 1998. Vol 20 No 8.</I> Talk about some cultures of <I>Halobacterium salinarum</I> R1. </LI><LI>
Different culture media</LI><LI><A href="http://www.biochem.mpg.de/en/rd/oesterhelt/web_page_list/Org_Hasal/"> 
General information about <I>Halobacterium salinarum</I></A></LI><LI><I>Halobacterium salinarum</I> divides itself each 4-6h (it could last even a day).</LI><LI>About the <A href="http://www.pnas.org/cgi/reprint/97/22/12176.pdf"> general genome</A>, with an image about the membrane proteins. Watch out the two CI</LI><LI><I>Halobacterium</I><A href="http://www.pnas.org/cgi/content/full/97/22/12176"> was sequenced</A></LI><LI><I>Halobacterium salinarum</I><A href="http://redalyc.uaemex.mx/redalyc/pdf/579/57937307.pdf">growing conditions</A>: 50ºC, pH 7.2, [NaCl]=3.5-4.3 M</LI></UL></P></DIV>2008-05-12<DIV align="justify"><P><B><U>GROUP SESSION:</U></B><STRONG>Preliminary Projects Groups Presentation(3rd Round)
Points to review…</STRONG><B>Singing Bacteria:</B><UL><LI>How to control the salinity. </LI><LI>What happens if you knock out this channels? Does a compensatory mechanism exist in the cell?</LI><LI>Wave length at which the cell reacts. Do cells react to colors?</LI><LI>If we use tetracycline, use a naturally resistant mutant</LI><LI>We need to know if the cell is transporting tetracycline in an efficient way.</LI><LI>Why don’t we use efflux pumps but with heavy metals? They seem to be auto-regulated and the insertion is specific  </LI></UL><B>PONG:</B><UL><LI>Usin metal pumps</LI><LI>Magnetic proteins?</LI></UL></P></DIV>2008-05-21<DIV align="justify"><P><B><U>GROUP SESSION:</U></B><STRONG>Project Analysis and Tasks Assignation </STRONG><B>Two models:</B><LI>One with bacteriorhodopsin or halorhodopsin, using Cl-channels regulated by light.
The regulation of those being at a transcriptional level or with the light frequency</LI><LI>Another with metal efflux pumps. 
The regulation of those being at a transcriptional or at a phosphorilation level or any other specific regulation of the pump</LI><SPAN class="font-size: small">*The circuit will be incorporated the channels regulation. We will measure the ions concentration fluctuations in the medium </SPAN><B>Why would our project be relevant??</B><LI>It is possible to couple it with another systems. For example, it could help to know if a nutrient is lacking </LI><LI>We will take the medium into account for the modeling. </LI><LI>We will model a channel, and this is new(at least in the beginning), it has never been done</LI><LI>It is possible to regulate it with accuracy if we manage to reduce the noise. </LI><LI>We could use it in other projects (like PONG). </LI><LI>We could see at which level could we modulate the internal part of the system. </LI><LI>Bacteria are capable of modifying their medium as we wish. </LI><B>TASKS:</B><DL><DT>Study the bioparts:</DT><UL><DD><LI>Take into account the external signal</LI></DD><DD><LI>Regulation</LI></DD><DD><LI>Mainly a cascade, ‘cause it is easier </LI></DD><DD><LI><B>People in charge: </B>Martin, Libertad &amp; Minerva</LI></DD></UL><DT>Study the pumps in <I>E. coli</I>:</DT><DD><UL><LI><B>People in charge: </B>Mariana, Mariana-GS, Jimena &amp; Isaac</LI></UL></DD><DT>Investigate about the electrodes:</DT><DD><UL><LI><B>People in charge: </B>Carlos, Enrique &amp; Luis</LI></UL></DD><DT> Writing:</DT><DD><UL><LI><B>People in charge: </B>Sur, Daniela &amp; Atahualpa</LI></UL></DD></DL></P></DIV>2008-05-28<DIV align="justify"><P><B><U>GROUP SESSION:</U></B><B>Points to check:</B><UL><LI>Define a model</LI><LI>Manage the medium to make fluctuations and measure it.
Machines to measure fluctuations of even one ion. (Mariana Reyes will contact the Chemestry Faculty. There’s no machine in the IBT, at least no with such sensitivity. Electrodes aren’t expensive, neither hard to obtain, the hard part will be passing the info into the computer. We can get one adapter in 800 UK pounds</LI></UL><B>Wet Lab Group:</B><UL><LI>Mariana </LI><LI>Martin</LI><LI>Libertad </LI><LI>Jimena </LI><LI>Luis </LI><LI>Atahualpa </LI><LI>Minerva </LI><LI>Isaac </LI><LI>Enrique </LI></UL><B>Dry Lab Group:</B><UL><LI>Mariana GS </LI><LI>Daniela</LI><LI>Carlos </LI><LI>Sur </LI></UL></P></DIV>         <DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:LCG-UNAM-Mexico/Notebook/2008-May">http://2008.igem.org/Team:LCG-UNAM-Mexico/Notebook/2008-May</A>&quot;</DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:LCG-UNAM-Mexico/Notebook/2008-May" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:LCG-UNAM-Mexico/Notebook/2008-May" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:LCG-UNAM-Mexico/Notebook/2008-May&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:LCG-UNAM-Mexico/Notebook/2008-May&amp;oldid=84150" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DL></DIV></DIV></DIV></DIV></BODY></HTML>