<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_LCG-UNAM-Mexico_Parameters"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:LCG-UNAM-Mexico/Parameters">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:LCG-UNAM-Mexico/Parameters&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:LCG-UNAM-Mexico/Parameters&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:LCG-UNAM-Mexico/Parameters&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:LCG-UNAM-Mexico/Parameters" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:LCG-UNAM-Mexico/Parameters</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><P><TITLE>LCG-UNAM-Mexico:Modeling</TITLE><TABLE width="82%" border="0" cellspacing="0" cellpadding="0"><TBODY><TR bgcolor="#D5EDB3"><TD height="50" colspan="3" id="logo" valign="bottom" align="center" nowrap="nowrap">LCG-UNAM-Mexico</TD></TR><TR bgcolor="#D5EDB3"><TD height="51" colspan="3" id="tagline" valign="top" align="center">iGEM 2008 TEAM</TD></TR><TR bgcolor="#99CC66"><TD colspan="7" id="dateformat" height="20">  </TD></TR><TR><TD width="165" bgcolor="#5C743D">  </TD></TR><TR><TD width="165" bgcolor="#5C743D"><A href="https://2008.igem.org/Team:LCG-UNAM-Mexico" class="navText">Home</A></TD></TR><TR><TD width="165" bgcolor="#5C743D"><A href="https://2008.igem.org/Team:LCG-UNAM-Mexico/Project" class="navText">Our project</A></TD></TR><TR><TD width="165" bgcolor="#5C743D"><A href="https://2008.igem.org/Team:LCG-UNAM-Mexico/Modeling" class="navText">Modeling</A></TD></TR><TR><TD width="165" bgcolor="#5C743D"><A href="https://2008.igem.org/Team:LCG-UNAM-Mexico/Experiments" class="navText">Wet Lab</A></TD></TR><TR><TD width="165" bgcolor="#5C743D"><A href="https://2008.igem.org/Team:LCG-UNAM-Mexico/Parts" class="navText">Parts Submitted to the Registry</A></TD></TR><TR><TD width="165" bgcolor="#5C743D"><A href="https://2008.igem.org/Team:LCG-UNAM-Mexico/Notebook" class="navText">Notebook</A></TD></TR><TR><TD width="165" bgcolor="#5C743D"><A href="https://2008.igem.org/Team:LCG-UNAM-Mexico/Story" class="navText">Our story</A></TD></TR><TR><TD width="165" bgcolor="#5C743D"><A href="https://2008.igem.org/Team:LCG-UNAM-Mexico/Team" class="navText">About us</A></TD></TR></TBODY></TABLE>    </P><DIV align="center">  </DIV><TABLE border="0" cellspacing="0" cellpadding="2" width="590"><TBODY><TR><TD class="pageName"><DIV align="center">
      Parameters &amp; kinetics </DIV></TD></TR><TR><TD valign="top" class="bodyText"><P align="center"><A href="#reactions">Biochemical Reactions</A> | <A href="#initial">Defining the Initial State of the Model </A></P><P align="justify">
              The complete model uses 18 kinetic parameters and 11 biochemical  reactions. We got 13 of these parameters researching the literature and we estimated the range of values for 2 of them. The remaining 3 we adjusted to the observed results. Reaction kinetics were gotten from the literature,  and if no evidence was found then we assumed it to be Law of Mass Action.1. <SPAN class="style4">Degradation of AHL by AiiA</SPAN></P></TD></TR><TR><TD width="61"> </TD></TR><TR><TD colspan="2"><DIV align="left" class="style5">AiiA + AHL → AiiA</DIV></TD></TR><TR><TD width="88" valign="top"><STRONG>Kinetics:</STRONG></TD><TD width="228">Michaelis-Menten<SUP>1,2</SUP></TD></TR><TR><TD valign="top"><STRONG>Parameters:</STRONG></TD><TD><EM>k</EM><SUB>1cat</SUB> = 27.97 s<SUP>-1</SUP><EM>K</EM><SUB>1m</SUB> = 3.723 mM = 224.20427E5 molecules</TD></TR><TR><TD valign="top"><STRONG>Flux:</STRONG></TD></TR></TBODY></TABLE><P align="justify">2. <SPAN class="style4">Complex formation and dissociation between AHL and LuxR </SPAN></P><TABLE width="575" border="0"><TBODY><TR><TD width="61"> </TD></TR><TR><TD colspan="2"><DIV align="left" class="style5">AHL + LuxR ↔ AHL:LuxR</DIV></TD></TR><TR><TD width="88" valign="top"><STRONG>Kinetics:</STRONG></TD><TD width="361">Mass Action<SUP>3</SUP></TD></TR><TR><TD valign="top"><STRONG>Parameters:</STRONG></TD><TD><P><EM>k</EM><SUB>2</SUB> = 10 <SUP>-5</SUP> molecules<SUP>-1</SUP> s<SUP>-1</SUP><EM>k</EM><SUB>-2</SUB> = 3.33 x 10 <SUP>-3</SUP> s<SUP>-1</SUP></P></TD></TR><TR><TD valign="top"><STRONG>Flux:</STRONG></TD></TR><TR><TD valign="top"><STRONG>Notes:</STRONG></TD><TD><P align="justify">The complex formation is slow and its dissociation is fast, so with few AHL and LuxR the complex concentration is negligible. </P></TD></TR></TBODY></TABLE><P align="justify">
            2.1. <SPAN class="style4"><STRONG>Dimer formation and dissociation between AHL:LuxR complexes</STRONG></SPAN></P><TABLE width="577" border="0"><TBODY><TR><TD width="61"> </TD></TR><TR><TD colspan="2"><DIV align="left" class="style5">2 AHL:LuxR ↔ (AHL:LuxR):(AHL:LuxR)</DIV></TD></TR><TR><TD width="88" valign="top"><STRONG>Kinetics:</STRONG></TD><TD width="398">Mass Action<SUP>3</SUP></TD></TR><TR><TD valign="top"><STRONG>Parameters:</STRONG></TD><TD><P><EM>k</EM><SUB>2.1</SUB> = 10 <SUP>-5</SUP> molecules<SUP>-1</SUP> s<SUP>-1</SUP><EM>k</EM><SUB>-2.1</SUB> = 10 <SUP>-2</SUP> s<SUP>-1</SUP></P></TD></TR><TR><TD valign="top"><STRONG>Flux:</STRONG></TD></TR></TBODY></TABLE><P align="justify">
            3.1. <SPAN class="style4">CI synthesis induced by AHL and LuxR complexes dimer</SPAN></P><TABLE width="577" border="0"><TBODY><TR><TD width="61"> </TD></TR><TR><TD colspan="2"><DIV align="left" class="style5">ρcI + (AHL:LuxR):(AHL:LuxR) → ρcI + (AHL:LuxR):(AHL:LuxR) + CI</DIV></TD></TR><TR><TD valign="top"><STRONG>Kinetics:</STRONG></TD><TD>Mass Action<SUP>3</SUP></TD></TR><TR><TD valign="top"><STRONG>Parameters:</STRONG></TD><TD><P><EM>k</EM><SUB>3on</SUB> = 10 <SUP>-2</SUP> molecules<SUP>-1</SUP> s<SUP>-1</SUP></P></TD></TR><TR><TD valign="top"><STRONG>Flux:</STRONG></TD></TR></TBODY></TABLE><P align="justify">3.2. <SPAN class="style4">Constitutive CI synthesis</SPAN></P><TABLE width="577" border="0"><TBODY><TR><TD width="61"> </TD></TR><TR><TD colspan="2"><DIV align="left" class="style5">ρcI → ρcI + CI</DIV></TD></TR><TR><TD width="88" valign="top"><STRONG>Kinetics:</STRONG></TD><TD width="359">Mass Action<SUP>3</SUP></TD></TR><TR><TD valign="top"><STRONG>Parameters:</STRONG></TD><TD><P><EM>k</EM><SUB>3off</SUB> = 4 x 10 <SUP>-2</SUP> s<SUP>-1</SUP></P></TD></TR><TR><TD valign="top"><STRONG>Flux:</STRONG></TD></TR><TR><TD valign="top"><STRONG>Notes:</STRONG></TD><TD><DIV align="justify">To give more stability to the <EM>off</EM> state in the model, the rate constant in the presence of the inducer is  <A href="https://2008.igem.org/Team:LCG-UNAM-Mexico/Notebook/2008-August_2#21ago">lower than the constitutive rate constant</A>, regardless the implication  of a greater threshold to achieve the <EM>on </EM>state<SUP>3</SUP>.</DIV></TD></TR></TBODY></TABLE><P align="justify">4. <SPAN class="style4">Natural degradation of  CI</SPAN></P><TABLE width="577" border="0"><TBODY><TR><TD width="61"> </TD></TR><TR><TD colspan="2"><DIV align="left" class="style5">CI → Ø</DIV></TD></TR><TR><TD width="88" valign="top"><STRONG>Kinetics:</STRONG></TD><TD width="359">Mass Action</TD></TR><TR><TD valign="top"><STRONG>Parameters:</STRONG></TD><TD><P><EM>k</EM><SUB>4</SUB> = 0.002888  s<SUP>-1</SUP></P></TD></TR><TR><TD valign="top"><STRONG>Flux:</STRONG></TD></TR><TR><TD valign="top"><STRONG>Notes:</STRONG></TD><TD><DIV align="justify">The half life of CI with LAA tail is 4 minutes<SUP>8</SUP>. Andersen JB <EM>et al.</EM><SUP>9</SUP>  conclude that LAA tail and LVA tail modified the half life of GFP in a  similar extent. Given this value,<A href="https://2008.igem.org/Team:LCG-UNAM-Mexico/Notebook/2008-August_2#21ago"> the rate constant was calculated</A>.</DIV></TD></TR></TBODY></TABLE><P align="justify">4.1. <SPAN class="style4">Dimer formation and dissociation between   CI molecules </SPAN></P><TABLE width="577" border="0"><TBODY><TR><TD width="61"> </TD></TR><TR><TD colspan="2"><DIV align="left" class="style5">2 CI ↔ CI:CI</DIV></TD></TR><TR><TD width="88" valign="top"><STRONG>Kinetics:</STRONG></TD><TD width="359">Mass Action</TD></TR><TR><TD valign="top"><STRONG>Parameters:</STRONG></TD><TD><P><EM>k</EM><SUB>4.1</SUB> = 0.00001 molecules<SUP>-1</SUP> s<SUP>-1</SUP><EM>k</EM><SUB>-4.1</SUB> = 0.01  s<SUP>-1</SUP></P></TD></TR><TR><TD valign="top"><STRONG>Flux:</STRONG></TD></TR><TR><TD valign="top"><STRONG>Notes:</STRONG></TD><TD><DIV align="justify">Kenneth <EM>et al.</EM> estimated the change in free Gibbs energy in this reaction (with wildtype CI) as -11.1 kcal/mol,<SUP>10</SUP> which leads to an equilibrium constant of 8.32186E16 molecules<SUP>-1</SUP>.  This implies that the forward rate constant should be at least sixteen orders of magnitude greater than the reverse rate constant, which means a constant repression of RcnA even with the constitutive CI synthesis.  A parameter scan was run to determine the range of values that gives the desired behavior and the rate constants were chosen arbitrarily  within this range. These values are comparable to others typical biochemical parameters. It has been shown that kinetic parameters can be modified by changing amino acid sequences (for example, CI half life is reduced by adding a LVA tail in the C-terminal), it’s proposed that  it’s possible to engineer the protein to reach an acceptable dissociation constant. </DIV></TD></TR></TBODY></TABLE><P align="justify">6. <SPAN class="style4">RcnA production </SPAN></P><TABLE width="577" border="0"><TBODY><TR><TD width="61"> </TD></TR><TR><TD colspan="2"><DIV align="left" class="style5">ρ → ρ + RcnA</DIV></TD></TR><TR><TD width="88" valign="top"><STRONG>Kinetics:</STRONG></TD><TD width="359">Cooperative inhibition (Hill kinetics)<SUP>4,5,6,7</SUP></TD></TR><TR><TD valign="top"><STRONG>Parameters:</STRONG></TD><TD><P><EM>n</EM><SUB>5</SUB> =1.9</P></TD></TR><TR><TD valign="top"><STRONG>Flux:</STRONG></TD></TR><TR><TD valign="top"><STRONG>Notes:</STRONG></TD><TD><DIV align="justify">ΔG<SUB>CI:CI-OR1</SUB>=-11.6 kcal/mol
                        ΔG<SUB>CI:CI-OR2</SUB>=-10.1 kcal/mol
                          ΔG<SUB>CI:CI-OR1-OR2</SUB>=-23.8 kcal/mol<EM>ν</EM><SUB>6</SUB>(<EM>Pl</EM>)=20mM/h=3346.111 molecules/s with 20 promoter copies (ρ<SUB>0</SUB>)<SUP>7</SUP>.
                        The promoter in our construction is Pr, which is similar to Pl, the one used to estimated the parameter<SUP>7</SUP>.</DIV></TD></TR></TBODY></TABLE><P align="justify">
            7. <SPAN class="style4">Nickel efflux by RcnA </SPAN></P><TABLE width="577" border="0"><TBODY><TR><TD width="61"> </TD></TR><TR><TD colspan="2"><DIV align="left" class="style5">RcnA + Ni<SUB>int</SUB> → RcnA + Ni<SUB>ext</SUB></DIV></TD></TR><TR><TD width="88" valign="top"><STRONG>Kinetics:</STRONG></TD><TD width="359">Mass Action</TD></TR><TR><TD valign="top"><STRONG>Parameters:</STRONG></TD><TD><P><EM>k</EM><SUB>7</SUB> = 500 molecules<SUP>-1</SUP> s<SUP>-1</SUP></P></TD></TR><TR><TD valign="top"><STRONG>Flux:</STRONG></TD></TR><TR><TD valign="top"><STRONG>Notes:</STRONG></TD><TD><DIV align="justify"><A href="https://2008.igem.org/Team:LCG-UNAM-Mexico/Simulation#nickel">Biologically possible range of values estimated through the model.</A></DIV></TD></TR></TBODY></TABLE><P align="justify">8.<SPAN class="style4"> Natural degradation of RcnA </SPAN></P><TABLE width="577" border="0"><TBODY><TR><TD width="61"> </TD></TR><TR><TD colspan="2"><DIV align="left" class="style5">RcnA → Ø</DIV></TD></TR><TR><TD width="88" valign="top"><STRONG>Kinetics:</STRONG></TD><TD width="359">Mass Action</TD></TR><TR><TD valign="top"><STRONG>Parameters:</STRONG></TD><TD><P><EM>k</EM><SUB>8</SUB> = 1.666E-4 s<SUP>-1</SUP></P></TD></TR><TR><TD valign="top"><STRONG>Flux:</STRONG></TD></TR><TR><TD valign="top"><STRONG>Notes:</STRONG></TD><TD><DIV align="justify">This kinetic parameter wasn’t found in our bibliographic search and <A href="https://2008.igem.org/Team:LCG-UNAM-Mexico/Notebook/2008-August_2#letter">personal communication with Peter T. Chivers</A>  (Washington University School of Medicine) confirmed that this  parameter is unknown. The value used is the degradation rate of LacY,  the lactose permease of <EM>E. coli</EM>, which is also a transmembran protein.<SUP>11</SUP></DIV></TD></TR></TBODY></TABLE><P align="justify">
            9.<SPAN class="style4"> Nickel import by unknown channel </SPAN></P><TABLE width="577" border="0"><TBODY><TR><TD width="61"> </TD></TR><TR><TD colspan="2"><DIV align="left" class="style5">Unk + Ni<SUB>ext</SUB> → Unk + Ni<SUB>int</SUB></DIV></TD></TR><TR><TD width="88" valign="top"><STRONG>Kinetics:</STRONG></TD><TD width="359">Mass Action</TD></TR><TR><TD valign="top"><STRONG>Parameters:</STRONG></TD><TD><P><EM>k</EM><SUB>9</SUB> = 500 molecules<SUP>-1</SUP> s<SUP>-1</SUP></P></TD></TR><TR><TD valign="top"><STRONG>Flux:</STRONG></TD></TR><TR><TD valign="top"><STRONG>Notes:</STRONG></TD><TD><DIV align="justify"><A href="https://2008.igem.org/Team:LCG-UNAM-Mexico/Simulation#nickel">Biologically possible range of values estimated through the model</A>.</DIV></TD></TR></TBODY></TABLE><P><STRONG>NOTE:</STRONG> The average volume of an <EM>E. coli </EM>cell is 10<SUP>-15</SUP> liters.</P><P class="style2">Defining the initial state of the system</P><P align="justify"> The initial concentrations of the constitutive proteins (AiiA,  LuxR, CI -constitutive synthesis- and CI:CI -due to constitutive  synthesis-) were estimated based on the efficiency rate of their  promoters, number of promoters per cell, degradation rate of their  mRNAs, translation efficiency and degradation rate of the proteins.  Initial concentrations of AHL:LuxR complex, the dimer of complexes, CI  and CI:CI due to complex activation were set to 0, given these  are all due to the action of AHL. Number of copies of both <EM>cI</EM> and <EM>rcnA</EM>  promoters are 10 based on plasmid copy number. RcnA and Unk were  estimated experimentally and set consistent to the observed rate.  Concentration of AHL and nickel is determined by us to obtain the  desired results. </P><P align="justify"><SPAN class="style3">AHL:</SPAN> It’s an arbitrary and adjustable value. Different outcomes can be observed <A href="https://2008.igem.org/Team:LCG-UNAM-Mexico/Simulation">manipulating this initial value</A>.<SPAN class="style3">
              Nickel (total):</SPAN> It’s an arbitrary and adjustable value. Different outcomes can be observed <A href="https://2008.igem.org/Team:LCG-UNAM-Mexico/Simulation">manipulating this initial value</A>.<SPAN class="style3">
              Unk:</SPAN> Both the Unk concentration and its rate constant are unknown. They are arbitrarily defined in such a way that it is consistent with the <A href="https://2008.igem.org/Team:LCG-UNAM-Mexico/Simulation#nickel">desired flux.</A>    <STRONG>[Unk]</STRONG>= 3,315 molecules<SPAN class="style3">
            ρ and ρCI:</SPAN> Their concentration is defined by the copy number of the plasmids that contain them.    <STRONG>[ρ]</STRONG>= 10 molecules    <STRONG>[ρcI]</STRONG>= 10 molecules<SPAN class="style3">CI and CI:CI:</SPAN> Given the <A href="#cisynthesis">constitutive synthesis</A> and <A href="#cidegradation">degradation</A> rate of CI, as well as its <A href="#cidimer">dimerization constant</A>, CI and CI:CI concentrations are estimated in absence of AHL.    <STRONG>[CI]</STRONG>= 138 molecules    <STRONG>[CI:CI]</STRONG>= 19 molecules<SPAN class="style3">
             RcnA:</SPAN> Given the <A href="#rcnaproduction">synthesis</A> and <A href="#rcnadegradation">degradation</A> rate of RcnA, as well as the <A href="#ciciconstitutive">constitutive concentration of CI:CI</A>, RcnA concentration is estimated in absence of AHL.    <STRONG>[RcnA]</STRONG>= 33150 molecules<SPAN class="style3">
             AiiA:</SPAN> The constant concentration of AiiA is calculated taking into account the following parameters retrieved from literature:        -       pLac average transcription rate<SUP>12</SUP>:           0.003 s<SUP>-1</SUP>        -       mRNA average degradation rate<SUP>13</SUP>:         0.00766 s<SUP>-1</SUP>        -       Average translation rate<SUP>13</SUP>:                     0.31333 s<SUP>-1</SUP>        -       AiiA degration rate:                               0.00012 s<SUP>-1</SUP>
            The half life of RcnA with LVA tail is approximately 2 minutes<SUP>14</SUP>; Andersen JB et al. found that this tail reduces the half life of GFP forty-eight times.9 Therefore the half life of wildtype AiiA can be estimated to 96 minutes.   <STRONG> [AiiA]</STRONG>= 10000 molecules<SPAN class="style3">
             LuxR:</SPAN> The constant concentration of LuxR is calculated taking into account the following parameters retrieved from literature:        -       pTet average transcription rate<SUP>12,15</SUP>:        0.003 s<SUP>-1</SUP>        -       mRNA average degradation rate<SUP>13</SUP>:         0.00766 s<SUP>-1</SUP>        -       LuxR translation rate<SUP>16</SUP>:                          0.556 s<SUP>-1</SUP>        -       LuxR degration rate<SUP>16</SUP>:                           9.627E-5 s<SUP>-1</SUP>   <STRONG> [LuxR]</STRONG>= 22000 molecules</P><P align="justify"><SPAN class="style2"><STRONG>References</STRONG></SPAN><STRONG>1.     </STRONG>Wang LH <EM>et al</EM>. (2004) <STRONG>Specificity and Enzyme Kinetics of the Quorum-quenching <EM>N-</EM>Acyl Homoserine Lactone Lactonase (AHL-Lactonase). </STRONG>J Biol Chem <STRONG>279:</STRONG>4, 13645-13651. <STRONG>2.     </STRONG>Hee Kim <EM>et al. </EM>(2005) <STRONG>The molecular structure and catalytic mechanism of a quorum-quenching N-acyl-L-homoserine lactone hydrolase.</STRONG> Proc Natl Acad Sci USA 102:49, 17606-17611. <STRONG>3.     </STRONG>Goryachev AB, Toh DJ, Lee T (2006). <STRONG>Systems  analysis of a quorum sensing network: Design constraints imposed by the  functional requirements, network topology and kinetic constants.</STRONG> Biosystems 83, 178-187. <STRONG>4.     </STRONG>Babic AC, Little JW (2007) <STRONG>Cooperative binding by CI repressor is dispensable in a phage </STRONG><STRONG>λ </STRONG><STRONG>variant. </STRONG>Proc Natl Acad Sci USA 104: 17741-17746. <STRONG>5.     </STRONG>Ackers GK, Johnson AD, Shea MA (1982). <STRONG>Quantitative model for gene regulation by </STRONG><STRONG>λ</STRONG><STRONG>phage repressor.</STRONG>Proc Natl Acad Sci USA 79: 1129-1133. <STRONG>6.     </STRONG>Reinitz J, Vaisnys JR (1990) <STRONG>Theoretical and Experimental Analysis of the Phage Lambda Genetic Switch Implies Missing Levels of Co-operativity</STRONG>. J Theor Biol 145: 295-318. <STRONG>7.     </STRONG>Iadevaia S, Mantzaris NV (2006) <STRONG>Genetic Network Driven Control of PHBV Copolymer Composition. </STRONG>J Biotechnol 122: 99-121. <STRONG>8.     </STRONG>Elowitz MB &amp; Leibler S (2000). <STRONG>A synthetic oscillatory network of transcriptional regulators. </STRONG>Nature 403 335-338. <STRONG>9.     </STRONG>Andersen JB <EM>et al </EM> (1998). <STRONG>New Unstable of Green Fluorescent Protein for Studies of Transient Gene Expression in Bacteria. </STRONG> Appl Environ Microbiol 64,6: 2240-2246. <STRONG>10.   </STRONG>Kenneth S.Koblan and Gary K. Ackers (1991) <STRONG>Energetics of Subunit Dimerization in Bacteriophage </STRONG><STRONG>λ </STRONG><STRONG>cI </STRONG><STRONG>Repressor: Linkage to</STRONG><STRONG>Protons, Temperature, and KCl.</STRONG> Biochemistry 1991, 30, 7817-7821. <STRONG>11.   </STRONG>M. Santillán and M. C. Mackey (2004). <STRONG>Influence of catabolite repression and inducer exclusion on the bistable behavior of the lac operon.</STRONG> Biophys J. 86: 1282-1292 <STRONG>12.   </STRONG>Malan, T. P., A. Kolb, H. Buc, and W. R. McClure (1984). <STRONG>Mechanism of CRP-cAMP activation of lac operon transcription initiation activation of the P1 promoter.</STRONG> J. Mol. Biol. 180:881–909. <STRONG>13.   </STRONG>Kennell, D., and H. Riezman (1977). <STRONG>Transcription and translation initiation frequencies of the <EM>Escherichia coli</EM> lac operon.</STRONG> J. Mol. Biol. 114:1–21. <STRONG>14.   </STRONG>Christopher Batten. <STRONG>Modeling the Lux/AiiA Relaxation Oscillator</STRONG><STRONG>.</STRONG> Unpublished (<A href="http://www.mit.edu/%7Ecbatten/work/ssbc04/modeling-ssbc04.pdf">http://www.mit.edu/~cbatten/work/ssbc04/modeling-ssbc04.pdf</A>). <STRONG>15.   </STRONG>Bologna Cesena Campus, iGEM 2007 WIKI. (<U><A href="https://2007.igem.org/Bologna">https://2007.igem.org/Bologna</A></U>) <STRONG>16.   </STRONG>KULeuven team, iGEM 2008 WIKI. Dr. Coli, the bacterial drug delivery system. (<A href="https://2008.igem.org/Team:KULeuven/Model/CellDeath" target="_blank">https://2008.igem.org/Team:KULeuven/Model/CellDeath</A>) </P><DIV align="justify"> </DIV>        <DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:LCG-UNAM-Mexico/Parameters">http://2008.igem.org/Team:LCG-UNAM-Mexico/Parameters</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:LCG-UNAM-Mexico/Parameters" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:LCG-UNAM-Mexico/Parameters" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:LCG-UNAM-Mexico/Parameters&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:LCG-UNAM-Mexico/Parameters&amp;oldid=105461" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>