"
Page
Discussion
View source
History
teams
Log in
 
Team:LCG-UNAM-Mexico/Notebook/2008-June 2
From 2008.igem.org
LCG-UNAM-Mexico:Notebook/June_2
LCG-UNAM-Mexico
iGEM 2008 TEAM
  
 
 
Home
Our project
Modeling
Wet Lab
Parts Submitted to the Registry
Notebook
Our story
About us
 
 
 
 
 
 
June
2008-06-18
Final design
Scheme
The first plasmid contains the efflux pump for Nickel (RcnA), which  will maintain its natural regulation dependent of RcnR and  additionally, it will contain a promoter regulated by the repressor of lambda  phage, cI as well as a resistance as a marker of the plasmid.
The second plasmid contains everything needed for regulating RcnA dependent on an external signal (AHL). Both luxR and aiiA will be synthesised constitutively. LuxR with a strong promoter (pTetR), as we do not want the presence of LuxR to be limiting and aiiA under the control of a moderate or weak promoter (pLacZ) for it is a very efficient enzyme, and we don't want it to degrade all AHL and prevent the signal from being transmitted. And cI*, which is a version of cI tagged with an LVA tail for rapid degradation, regulated by a promoter dependent of LuxR +  AHL. It will also contain a resistance as a marker.
When AHL is added, it will bind LuxR and stimulate the production of cI*, which in  turn will represses the transcription of rcnA. Like cI*, the signal produced by AHL will be short-lived since aiiA will be degradating it constantly, so the system quickly returns to its initial state.
Parts
Defining bioparts we will use or where to get what is necessary.
Part: BBa_I729006
Part of Quorum sensing used by the team Chiba in iGEM2007. Both tetR  and LacI + pL are constitutive promoters, but since LacI + pL is  a very strong promoter, it will probably be replaced. This biopart will be  responsible for the regulation by luxR  and the action of the system by AHL.  Instead of GFP (Subpart E0040), the BBa_C0051 part that  codes for the protein cI + LVA will be inserted, which will join the regulatory region of cI  (biopart BBa_R0051) in the other plasmid.
(Previous experience: none)
Part:BBa_C0051
Region coding for the repressor cI, of lambda phage, tagged with an LVA tail for rapid degradation. cI joins the  regulator cI (BBa_R0051)
(Previous experience: none)
Part:BBa_R0051
 
Promoter regulated by cI based on the pR promoter of lambda phage. The  promoter has two binding sites for the cI repressor of lambda phage  (BBa_C0051). The binding of cI leads to the suppression of the transcript synthesis.
(Previous experience: it works)
The sequence of the 3 previously mentioned bioparts is in the Registry of Standard Biological Parts and according to the information provided by the registry, DNA is available.
Part: BBa_G00510
gatttctgcatagccagacttggg
This is the forward primer for C0051, 24 bp long.
Part: BBa_G00511
cactgactagcgataactttccccac
Reverse primer for C0051, 26 bp long.
Vectors
Possibilities in bioparts:
Name
Description
pSB3C5
Low to medium copy    BioBrick standard vector
pSB3T5
Low to medium copy    BioBrick standard vector
pSB4A3
pSB4A3
pSB4C5
Low copy BioBrick    standard vector
pSB4A1
pSB4A1
pSB4A5
Low copy BioBrick    standard vector
pSB4T5
Low copy BioBrick    standard vector
BBa_I739202
pCK01BB1
Primers
Build or find oligos that we could use for our constructions.
We need:
• rcnA (with its regulatory region; no promoter).
• cI* with its AHL-LuxR dependent promoter.
• LuxR with its constitutive tetR promoter.
• AiiA with its constitutive promoter (lac is proposed, it is a moderate promoter).
• Promoter dependent of cI.
 In all cases, we have to check whether they already exist (in bioparts or elsewhere) and evaluate them.
 
 
Sequence
Tm
Deg.
Restr. Site
Bioparts
(pTetR)luxR/(p.c.strong)aiiA
Upper
 
62.5 ºC
1
None
?
Lower
 
63.8 ºC
1
None
?
pcI
Upper
 
61.9-76.2 ºC
864
None
?
Lower
 
66.5 ºC
1
None
?
pLacZ
Upper
5' GCACCCAGGCTTTACACTTT 3'
64.7 ºC
1
None
?
Lower
5' TGTTATCCGCTCACAATTCCA 3'
60.3 ºC
1
None
?
cI*
Upper
5' GATTTCTGCATAGCCAGACTTGGG 3'
62.9 ºC
1
None
BBa_G00510
Lower
5' CACTGACTAGCGATAACTTTCCCCAC 3'
61.9 ºC
1
None
BBa_G00511
rcnA
Upper
5' CACTATTAATCTACTGGGGGGTAG3'
64.2ºC
1
None
 
Lower
5' AGTTATCGCATTATGCCCATG 3'
65.8ºC
1
None
 
Promoters
Investigate more about the proposed promoters and define if they are the most optimal depending on our needs.
Promoter
Biopart
Constitutive?
Strength
Notes
pTetR
BBa_R0040
In  tetracycline presence or  TetR absence
medium
Recomended by our advisor Miguel
pLuxR-HSL
BBa_R0062
Over-regulated by LuxR-HSL (increases its expression).
weak (constitutive)/medium (LuxR-HSL)
luxR could bring some trouble if it becomes a part of the sistem
pLacIQ
BBA_I14032
Yes
high
¿Is there a biopart? It could be  the promoter for luxR
pCyc
BBa_I766555
Yes
medium
Yeast promoter
J23112
BBa_J23113
Yes
1
 
J23103
BBa_J23113
Yes
17
 
J23113
BBa_J23113
Yes
21
 
J23109
BBa_J23113
Yes
106
 
J23117
BBa_J23113
Yes
162
 
J23114
BBa_J23113
Yes
256
 
J23115
BBa_J23113
Yes
387
 
J23116
BBa_J23113
Yes
396
Constitutive promoters family
J23105
BBa_J23113
Yes
62
J23110
BBa_J23113
Yes
844
 
J23107
BBa_J23113
Yes
908
 
J23106
BBa_J23113
Yes
1185
 
J23108
BBa_J23113
Yes
1303
 
J23118
BBa_J23113
Yes
1429
 
J23111
BBa_J23113
Yes
1487
 
J23101
BBa_J23113
Yes
1791
 
J23104
BBa_J23113
Yes
1831
 
J23102
BBa_J23113
Yes
2179
 
J23100
BBa_J23113
Yes
2547
 
Facts about  kinetics & other things...
Investigate more about the elements of the system to begin building an outline for the model and defining if design is theoretically feasible.
NOTES: For LuxR to bind HSL and enable the transcription of cI, HLS should be at a micromolar concentration.
Not all bioparts have been previously used, most DNA is  available but there is still no record their functionality. We need to evaluate the DNA quality to ensure that there will be no problems.
2008-06-24
MODELING:
Variables
Concentrations of:
LuxR  (constant).
aiiA  (constant).
AHL  (arbitrary).
cI* (according to aiiA, AHL & LuxR).
RcnA (according to cI*).
We need to determine the initial concentrations and lifetime of proteins involved, as well as the efficiency of AiiA (kinetics in general).
The concentration of Nickel (NiCl2) in the medium that the cells can tolerate according to Rodrigue et al. (2005) before inhibiting growth is 4 μ M for the strain lacking rcnA, 10 μM in the wildtype and up to 100-fold more in a strain with a multicopy gene.
 
 
Concentration
Life span (half-life)
Substrate affinity
Notes
AHL
?
3 hrs.
?
Conflictive information
LuxR
?
60 min- (~40-100)
?
 
2 min (35 min +AHL)
 
aiiA
?
24 hrs
?
 
cI*
?
?
?
 
RcnA
?
?
?
 
Assumption 1: Once there is nickel in the medium, RcnR will not interfere in the pump regulation. This because there will be large concentrations of metal, so we can assume that RcnR will always be bound to a molecule of nickel and it will therefore be unable to suppress the transcription of rcnA; the noise that the few RcnR free molecules can cause, will be indistinguishable from normal behaviour of the pump.
Assumption  2: Any decrease in the concentration of AHL is due to aiiA. It is  believed that the natural degradation of this molecule is irrelevant in the time  scale analysis. Either way, a process will not be distinguishable from  the other and even when the first is estimated, it would not be very  informative for the analysis, so we intend to  take this assumption as  true.
Assumption 3: The transcription of cI* depends solely on the concentration of AHL. LuxR is not a limiting step, ie, it is in a constant concentration and in sufficient amount to always be ready to associate with AHL. Only to simplify the analysis, at least for our first approach.
Initial outline:
(v1) AHL0
(v2) aiiA + AHL -> aiiA
(v3) AHL + LuxR -> cI*
(v4,v5) ρ + cI* <--> ρ.cI*
ρ -> ρ +RcnA
RcnA + Ni -> RcnA
RcnA -> Ø
 
2008-06-26
WET LAB:
1. Take the sequences (fasta format)
2. Once you have the sequence find appropriate reading frames
3. Make the restriction map
-- Nedcutter, check the page for NewEngland Biolabs (because we are going to use enzymes from that company)
For rcnA and rcnR, the regulatory region that was among the two genes was not explained.
We have to take the whole sequence in fasta format and use it in a program called Gene Construction Kit. This shows reading frames and restriction sites.
 
In fruitfly.org: 9005/seq_tools/promoter.html we can look for primers and we can adjust parameters. We can also analyze the  stability energy, and seek the lowest point of stability. This point is generally the -10box. To find inverted repeats, we shall use the program StemLoop of the parcel of GCG (genetics computer group). This program calls in the sequence in a GCG format. To find direct repeats we will use the program "repeat". For rcnR and rcnA we found three direct repeats between the -10 box and the translation start of rcnA.We suggest that this is a regulatory region. Based on this, we designed the primers, trying to preserve the regulatory region and changing its promoter.
 
Primer design.
The region should be rich in GC, of about 20 nucleotides with a 50% GC content at least and it should finish in G. The program can also show the double chain to facilitate the design of oligo lower. If they are rich in AT, they can be longer primers to increase its Tm.
 
The most popular program at the center is Oligo. Here we open a new window and paste the sequence. This will open two windows. The first one with the Tm, and the other one with the free energy. The program can calculate all oligos and show potential couples with its parameters. We can also specify were we want the oligo to be located. Once the program generates it, we can analyze its biochemical properties.
Trying to k Delta G so it won't be lower than -10.
The differences between the TMS should not be greater than 5 degrees. Enzymes used in PCR use magnesium chloride. The most reliable and processed use magnesium acetate. It is said that 10mM of dinucleotidos is an optimal concentration for PCR, .4 mM is used in the lab. Once we have the oligo, we add a site at the far restraining 5 '. And add nucleotides in the 5 'end to ensure that the enzyme is positioned correctly and efficiently cut. These nucleotides are different for each enzyme, and they also protect the  5' end.
 
Two plasmids are used as a basis prK415 P. Our advisor, Miguel, already has isolated DNA and this DNA will be used to transform and to have a  the plasmid reserved. 2ul of the plasmid and competent cells treated with calcium chloride. The theory says that positive ions are attached to the membrane, so the membrane has a positive charge. As DNA has a negative charge, once they are mixed at 4 degrees Celsius for 20 minutes, we are going to take the tube and put it at 42 degrees centigrade. This stress produces holes in the membrane and many things will be capable or entering or exiting through the membrane, including DNA. Then it remains 2 more minutes at this temperature. The we return it to ice for 5 more minutes to recover. Later, the cells are placed in 1ml of rich medium (LB) were they are allowed to grow at 37 degrees for one hour at 300 revolutions per minute (this allows them to recover).
 
100ul are taken and used to plate in petri dishes with the antibiotic. It is left to grow for an entire day and at the end, isolated colonies should appear.
Bacteria with kanamycin 5ml of two strains, one with a deletion in rcnA and another one with any deletion except for rcnA . For 6 hours, the bacteria will have an exponential growth. Genomic DNA will be extracted.
 
The contents of the tube will be put in an eppendorf, we centrifuge and then we withdraw the liquid medium with a syringe. Before we lyse de cells, we need to wash with TE 10 1 (Tris 10uM EDTA 1uM), with pH 8. Vortex, to separate and disintegrate. Again, we centrifuge and remove supernatant. To lyse, we add 400-450 ul TE5020pH8 and SDS 10% and K proteinase. We leave it at 37 degrees for 20 minutes. The medium goes from an opaque color to a light color when lysis happens. We add  ethanol 100% once we have lysed the cells and we vortex. In the presence of ethanol DNA is precipitated, so we add 1ml of ethanol. Then we centrifuge for a few minutes and we have pellet. We wash three times with ethanol 70%, which solubilised salts and the small molecules (including RNA). We remove all the ethanol, this tube is placed in a specific centrifuge. The vacuum from this centrifuge will remove the remain solvent. It is necessary to remove all the ethanol, because this affects the pH. TE 10 1 RNAs 10mg per ml, this Stock solution is divided 1000 times and  50ul approx are added. To check the quality of the DNA extracted, we use an agarose gel.
 
Transforming bioparts:
Bacteria needed to extract DNA plasmid. Centrifuge, wash and put solution 1. Glucose, TRIS, EDTA and sometimes RNAs 1. Sodium hydroxide and SDS in the solution 2, sodium hydroxide denatures the DNA. Solution 3 with sodium acetate neutralizes the base. Wash and dry every time.
 
               
Retrieved from " http://2008.igem.org/Team:LCG-UNAM-Mexico/Notebook/2008-June_2 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
