Team:Columbia NYC/Notebook
THE PROJECT
DESCRIPTION
DESIGN
EXPERIMENTS
RESULTS
TEAM MEMBERS
TEAM PARTS
THE NOTEBOOK
ATTRIBUTIONS
COLLABORATIONS
HUMAN PRACTICE
LAB SAFETY
OUR SPONSORS
NOTEBOOK
The following is the notebook that we kept throughout the course of the project:
May 26th- June 18th Planning and Brainstorming
June 19th – June 23rd Planning/Preparing Cells and Buffers
June 24th: To Do This Week
· Regulatory Research (Sam and Hudson)
· Quorum Sensing Promoters
· Potential Fail safes (look at other iGEM teams) (either repressors or self destruction models)
· Last Secretion Modeling
· TEV Protease (Sam)
· PAM cytosolic sequence (Kong)
· Protocol of Supernatant protein concentration (NTU Taida Team) (Sam)
· Primer Design and Ordering (Whole Team)
· Presentation Prep
· On Type II Secretion System (Ppt) (Kong)
· Describe general system
· Describe our SP's roles in it and normalized function
· Make list of sources
· On Complex Communities (Ppt) (Kenya)
· Talk about complexity in natural habitats—hypothermal vent, soil, guts
· What's the benefit of these communities? How do they contribute to the ecosystem? Physiology?
· Bacteria involved, methods, what's natural, etc. Just WOW factors
· Assign Someone the strain notebook
June 28th- July 9th:
We then started to do the designing for the primers and then realized that we are not going to use Gibson assembly because our gene is too small, and if we went on using this the gene can get lost at some point, so it was decided that we use regular cloning instead. We will order the primers tonight (6/29). We have received the other primers that were ordered on Friday. We have set them up by matching them with their partners; whether they were forward or reverse we had a total of 26 and G1. We then re-suspended them with water (this depended on the OD of the gene) so for example if one was 24.7 then we would pipette 24.7 uL. Then we grabbed the larger tubes of PCR and then added water into it (for this we pipetted 0.80 uL) of water and then would add 20 uL of the gene into the corresponding tube. After that was transferred we obtained PCR tubes and placed into the DNA. That was placed on ice and that procedure was done on ice. Then in those PCR tubes which eight tubes we added the diluted version into the PCR tubes that all contained already: 10 uL of Q5 polymerase and 7 uL of dH2O. Then in each tube (8) 1 uL of G1 dilution was added to each. Then each specific PCR tube was numbered, where the forward and the reverse was added into it a total of 1 uL of the forward and uL of the reverse. The first tube was PhoA, second tube was Lp-3050-PYY, third tube was PYY, fourth tube was PYY His tag, fifth tube was Ghrelin, sixth tube was Ghrelin His tag, seventh tube was GLP-1, and eighth tube was GLP-1 His tag. Then they were placed into the PCR machine for 40 minutes, it was placed in block A. After the period of 40 minutes it was removed from the machine and was placed into the freezer 20 degrees Celsius.
The results: The first lane is the ladder, where it goes up by intervals of 100. Then the first lane is shady, so not much is seen (we will most likely edit this).This was the PhoA-GLP-1. The second lane is higher and that is correct because you have the signal peptide attached to the peptide. This was the Lp-3050-PYY. The third lane is perfect. PYY (this was in the lane).The fourth lane is higher than the third lane and it should be because it contains a His-tag, therefore it is correct. In this lane you have PYY and the His-Tag. The fifth lane is a double lane (we will have to modify this), this is Ghrelin. The sixth lane is good (again higher because of the His-tag), this is Ghrelin with the His-Tag. The seventh lane is good this is GLP-1. The eighth lane is again higher because of the attachment of the His-tag this is GLP-1 with the His-Tag. For the lanes that were not seen as clear as they have should, it is concluded that it could have been due to the annealing temperature. Most of them were in the same range of the temperature around 60-65, but because some were off in the range, the bands probably affected by this. This is just a suggestion as to why they don’t appear on the gel, but we will modify this and try to figure out why this is the case.
July 10th:
The following primers were being used:
Name
Primer number
PhoA-GLP1
1,2
Lp-3050
5,6
PYY
7,8
Ghrelin
11,12
GLP1
15,16
The template that was being used is: 1 G-block and 99 uL od dH2O
The reaction to do the PCR used the following components: (this used a total of 50 uL)
1 uL
template
2.5 uL
P1
2.5 uL
P2
19 uL
dH2O
25 uL
Q5 Mastermix
The temperatures are the following with the time intervals (also this was running for 30 cycles)
Temperature (Celsius)
Time (seconds)
98
50
98
10
62
20
72
20
72
2 min
4
Infinity
Re-did Ghrelin (one through five)
Listed below are the temperatures and the time intervals and also the number of cycles which is 30
Temperature (Celsius)
Time (seconds)
98
30
98
10
67-55
30
72
20
72
2 min
4
Infinity
Statements: after this part of the experiment, it was seen that PhoA-GLP runs at lower temperature and that Ghrelin runs better at higher temperature
PCR part 2
Name
Primer number
PhoA-GLP1: +His, +DS
36,30
Lp-3050-PYY: +His, +DS
37,32
PYY: +His, +DS
31,32
Ghrelin: +His, +DS
27,28
GLP1: +His, +DS
29,30
Templates used for the these are 1-5 (the order goes down vertically)
Then a gel was done the order was the following and the results were the following:
M 1 2 3 4 5 6 7 8 9 10
a b c d e 1 2 3 4 5
The results were a-e were successful, best ghrelin (1-5) at 5 it suggesting lower temperature for PCR (60 degrees or less)
Then purification was done
Then digesting procedure and calculations were done:
Nanodrop (ng/uL)
Mass
After equation (uL)
28.9
63+93+33=189 bp
26
137.2
123+102+33=258 258bp
20
21.1
102+33=135 bp
36
27.5
84+33=117 bp
28
19.1
93+33=126 bp
49=45
The digested continuation is:
Protocol:
37 C
1:15:00
68 C
20 min
12 C
Infinity
After this purification was done
Ligation Reaction:
Protocol Mix:
uL
Materials needed
2
T4 buffer
1
ligase
1.25
Backbone digest
y
Insert digest
20-y
dH2O
The BB= 6000 bp = 69.1 ng/uL
(the math is found in the hard cover notebook)= 29 uL of backbone
For x DNA
uL
Material needed
43
dH2O
1
Enzyme 1
1
Enzyme 2
5
Smart cut buffer
The "y" is the following
Y
uL
dH2O
PhoA-GLP1
3.15
12.6
Lp-3050-PYY
4..3
11.5
PYY
2.25
13.5
Ghrelin
1.95
13.8
GLP
2.1
13.65
The temperature and time intervals are as followed:
Temperature (C)
Time
16
12 hr 30 min
65
10 min
12
infinity
July 13th:
Did PCR for colony and Mini-Prep
Colony PCR
uL
Materials Needed
16
P1
16
P2
10
Master Mix (Kapper Hifi)
8.8
8% DMSO
Mini-Prep PCR
uL
Materials Needed
1
P1
1
P2
10
Master Mix (Q5)
7
8% DMSO
1
template
Temperature C
Time intervals
95
3 minutes
98
20 seconds
65
15 seconds
72
6 min
72
1 min
4
infinity
30 cycles
Temperature C
Time intervals
98
30 minutes
98
20 seconds
62
15 seconds
72
1 min
72
2 min
4
infinity
30 cycles
2. Transformations
o On the plate labeled (+)
· This is a turbo strain (NEB) it is a positive control to check if the plasmid that Sonja gave us is working
3. Running a gel
a. The order for the gel is the following: The image is located under the pictures and observations page
M
The ladder
1
Mini-Prep BB (this is from last week) (it contains the plasmid and the DNA)
2
PCR BB (this was made today) (it is the plasmid only and it is unpurified)
4. We are incubating
a. Before the incubating begin, the following had to be done:
i. The PCR tube that was done this morning was taken out and then 1 uL of Dpn1 was added to the tube, this is to digest the background of the PCR product of the Mini-Prep
Temperature C
Time (intervals)
37
1 hour
80
20 minutes
4
Infinity
5. Inoculating over night
a. So 2 culture tubes were obtained each tube had 5 mL of LB and then when you insert the bacteria it is MegaX and Nissle (it was just poked not an exact amount was taken out)
b. This was done to make electro component cells tomorrow
6. Colony PCR was done (after 4 hours of running) and it was placed on a gel to run
a. The order of the gel goes in this specific order:
M
ladder
1
Mini-Prep
2
Kappa high 5 PCR
b. The results showed that it failed so therefore we are not going to use this one
7. Purification is being done on the PCR BB
8. Nano-drop of the PCR BB and it was 6.1 ng/uL
9. Then we did Genewiki, where we filled out PCR tubes (10 of them) with the sequences from Friday labeled a-e
a. The amount inserted into the tubes was more concentration, now I don’t know how much that can be affected
b. There was no insertion of an peptide or sequence into tube 10
10. We did the nano-drop of the amplified PCR since it was to low and I got the reading to be 476.7 ng/uL (without purification)
a. Then a gel was made and the order of the gel was:
b.
M
ladder
1
1st PCR
2
2nd PCR
11. More PCR
12. Then started to do transformations
13. Waited for an hour and then transformed them
14. Check tomorrow
Continuation Wet Work:
1. Colony PCR the colonies
a. Re-did a couple for Ghrelin since it wasn't working
b. The PCR was done for the following:
2. Ran a gel after the Colony PCR was over to check if the products were present
3. Grew electrocomptent cells
4. Transformations
5. Inoculated the colonies that work
a. Overnight
6. Cycled ligation
7. Nano-drop
Plan for the week of 7/31st
Tuesday: Overnight lacto bb in e.coli (with constructs), Overnight lacto as well, Mini-prep move into lacto itself, Colony PCR to verify
Wednesday: Quorum sensing stuff (ordering), Mini-prep and transforming into lacto, Overnight nissle constructs
Monday: Shipping primers and Quorum sensing compounds
Thursday: Colony PCR lacto, Grow up immediately, Induce nissle + lacto overnight, and Quorum sensing stuff
Friday: Concentrate, Dot blotting, and Quorum sensing
Design: Promoter- lux box- sp-hormone, Promoter-lux box-factor, Promoter- cp-qs things Into e.coli and lacto
Week of August 5th:
Lactobacillus was grown over night to do electrocomptent cells the next day. Ran a colony PCR for the plates that were left in the fridge for a week. After the colony PCR was done a gel was done to verify. The gel verification was good. The primers that were used for this colony PCR were from the sanger sequence (51 and 52). Each plate 3 colonies were chosen to do the PCR (expect for the fifth one because that one didn’t have any colonies present). Re-inoculated lactobacillus because it was too late to start electrocomptent cells. Growing electrocomptent cells for lactobacillus (200 mL and 1 mL of culture). Re-organized the glycerol stocks on the excel sheet. Did a glycerol stock for lactobacillus retueri. Making more glycerol stock for when needed in the lab 20% glycerol in water. Made plates MRS agar (4 plates). Ran a colony PCR with the correct primers:
a. From each plate 3 colonies were chosen
b. The plates that worked were 3 and 4
c. Plates 1 and 2 didn’t work
d. They should be re-done
e. A gel was done to verify if the product was present
Transformations were done with the electrocomptent cells that were grown and in that the ligation mixes were placed
Week of August 18th:
Doing LCR
a. It ran for 2 hours
b. There are 12 tubes
i. The following was inserted into each tube
Amount (uL)
What was added
2.5
Amp Buffer
2
Amp Ligase
1
BO
2
S.P.
10
G.P.
7.5
H2O
c. The tubes were labeled:
Tube Number
Name
1
PhoA-PYY
2
PhoA-Ghrelin
3
PhoA-GLP1
4
pelB-PYY
5
pelB-Ghrelin
6
pelB-GLP1
7
Lp-PYY
8
Lp-Ghrelin
9
Lp-GLP1
10
M6-PYY
11
M6-Ghrelin
12
M6-GLP1
2. Primers that were used were for the following tubes:
Tube Number
Primers
1
43
2
47
3
39
4
49
5
48
6
40
7
45
8
49
9
41
10
46
11
50
12
42
3. Can't complete the ligation because the primers needed to do the next PCR are not in yet
a. Because of this we expect that the primers come in tomorrow morning
i. If the primers come in tomorrow morning then we can continue to do the ligation
4. In a flask LB was added
a. 100 mL
5. When the colonies were inoculated from the plates of lactobacillus out of the eight colonies only one of them grew.
a. So with that one that grew we took 3 mL of MRS and then 100 uL of the culture and placed it in the culture tube
i. And then placed it in the incubator (37 degrees)
1. Should check on it tomorrow morning
1. Purified the PCR that was done last night
2. After purification Nano-drop was done
a. On all 12 samples
3. Another 2 PCR were done
a. This is because they have different annealing temperatures
b. The set up for the PCR was:
1
PhoA + PYY
NO RDS: 79 |8
4
PelB + PYY
NO RDS: 80 | 8
7
Lp-3050 + PYY
NO RDS: 81 | 8
10
M6 + PYY
NO RDS: 82 | 8
2
PhoA + Ghrelin
NO RDS: 79 | 12
5
PelB + Ghrelin
NO RDS: 80 | 12
8
Lp-3050 + Ghrelin
NO RDS: 81 | 12
11
M6 + Ghrelin
NO RDS: 82 | 12
3
PhoA + GLP1
NO RDS: 79 | 16
6
PelB + GLP1
NO RDS: 80 | 16
9
Lp-3050 + GLP1
NO RDS: 81 | 16
12
M6 + GLP1
NO RDS: 82 | 16
4. After the PCR was done purification was done
a. Instead of Nano-dropping we ran a gel with those purified samples
i. The results were that some bands appeared and other didn't
1. There were 2 gels the first gel was PYY (that took 4 rows) and the next one was Ghrelin (that took the next 4 rows)
2. The other gel contained GLP1
b. So it was suggested since some of those bands didn’t work that we run another gel with the samples that were done this morning
i. Those samples were purified from the LCR and were labeled (the signal peptide, gut hormone, Pho. DNA)
1. In that gel the samples above were used and 15 uL of the DNA was applied, no water was used
a. Now not all samples were used PhoA-GLP1 and LP3050-PYY worked they were the controls
b. The results are as follow for the gel that was done:
i. Some bands were present and others weren't
5. Because the gel approved that some of the bands were not present the following was done
a. LCR was repeated
i. It was set up today
1. Incubating of the primers was done
2. Incubating of the hormones was done
3. PCR
4. Slow annealing was done
5. Nano-drop
6. Purification
Week of August 24th:
a. We are amplifying all parts from their respective g-blocks using the correct primers
i. This is only for part 1,2, and 4 cause 3 is has not arrived yet
After the PCR was done it was purified
It was placed in the box
1. So in order to continue the products have to grow in the glycerol stocks that were made
a. The problem is that the sequences were not sequenced so we need to sequence them to check which one works
b. A colony PCR was done were water, robust, primers 51 and 52 were used, and the template was obtained from the glycerol stock which was then added into 15 uL of water
c. The PCR ran for approximately 46 minutes
d. After the PCR was done it purified
e. It was sent off to sequencing to two different companies to see the results
1. Phosphorylated primers were done before running a PCR
Part 1
Part 2
Part 3
Part 4
14 uL H20
14 uL H20
14 uL H20
14 uL H20
1 uL T4 PNK
1 uL T4 PNK
1 uL T4 PNK
1 uL T4 PNK
5 uL T4 PNK Buffer
5 uL T4 PNK Buffer
5 uL T4 PNK Buffer
5 uL T4 PNK Buffer
Primers (57,60)
Primers (61,62)
Primers (63,64)
Primers (65,58)
2. After the PCR tubes were set up they were incubated for an hour under 37 degrees
3. Once this was over a PCR was set and it is running for 57 minutes
a. The PCR was set up this way: (parts 1,2, and 4 were only set up because we don’t currently have part 3)
Part 1
Part 2
Part 4
4 uL primer
4 uL primer
4 uL primer
1 uL gblock
1 uL gblock
1 uL gblock
10 uL Q5
10 uL Q5
10 uL Q5
5 uL DMSO
5 uL DMSO
5 uL DMSO
b. Ran a gel afterwards and the results were the following:
i. The values are where they are suppose to be
c. Purification was done on the PCR products
i. After this was done nano-drop was done
4. Made cultures to grow overnight
a. This was based on the sequencing and the results were that:
i. 3 culture tubes were made
1. PYY
2. GLP1
3. PhoA-GLP1
5. LCR was set up
uL
What was used
5
Part 1
5
Part 4
2
Primer #70 (blue cap one not the diluted one) BO
2
10x Amp Buffer
2
Amp Ligase
4
DMSO
a. This will run for 2 hours
i. After the reaction was completed the product was placed in the 4 degree fridge
1. PCR purification was done for the ligation that was done yesterday and placed into the fridge
a. It was diluted in 30 uL of water
i. Nano-drop was done and it was 39.6 ng/uL
2. A PCR was done for part 1+4 for digestion
Part 1
Part 4
1 uL primer 55
1 uL primer 74
1 uL primer 56
1 uL primer 72
1 uL part 1 of PCR product
1 uL part 4 of PCR product
7 uL DMSO
7 uL DMSO
10 uL Q5
10 uL Q5
a. This was purified and nano-drop
3. Then digestion was done
a. First thing that has to be done is to incubate at 37 degrees Celsius for an hour
b. The PCR tubes for the digestions are the following:
i.
Part 1 and 4
25 uL part 1/4
1 uL Nde1
5 uL Cut Smart Buffer
19 uL water
Part 1
25 uL part 1
1 uL Nde1
1 uL Kpn1
5 uL Cut Smart Buffer
19 uL water
Part 4
25 uL part 4
1 uL Spe1
1 uL Nde1
5 uL Cut Smart Buffer
34 uL water
PhoA-GLP1 Part 1 and 4
10 uL mini-prep
1 uL Nde1
5 uL Cut Smart Buffer
34 uL water
PhoA-GLP1 Part 1
10 uL mini-prep
1 uL Nde1
1 uL Kpn1
5 uL Cut Smart Buffer
34 uL water
PhoA-GLP1 part 4
10 uL mini-prep
1 uL Spe1
1 uL Nde1
5 uL Cut Smart Buffer
34 uL water
c. After the digestion was completed
i. Purification was done
ii. Nano-drop was done as well
The backbone values were too low, the insertions were all good values
Therefore, the conclusion is to give the elute more time instead of a minute do 5 minutes
a. This procedure will be done tomorrow again
4. Inoculation was done yesterday of PhoA-GLP1
Continuation from the week:
PCR was done for Part 2+3 and part 2+3+4
a. The setup was the following (the first one is for part 2+3 and the second chart is for part 2+3
b. 4):
uL
What is added?
1
Primer 73
1
Primer 71
1
Part 2+3 PCR product
7
DMSO
10
Q5
i.
uL
What is added?
1
Primer 73
1
Primer 72
1
Part 2+3+4 LCR product
7
DMSO
10
Q5
ii. After this PCR was done the gel was done and the results were not great you had smears going up and the product wasn't even there
1. So therefore, this PCR was done again to verify the results and this time it was better then the first one
2. This was purified and nano-dropped and placed into the box
2. PCR was set up
a. This time it contained 11 reactions to be done:
Pel B GLP1
7 uL DMSO
10 uL Q5
Template #83
Primer #80
Primer #16
PhoA Ghrelin
7 uL DMSO
10 uL Q5
Template #83
Primer #79
Primer #12
Lp3050 PYY
7 uL DMSO
10 uL Q5
Template #83
Primer #81
Primer #8
M6 GLP1
7 uL DMSO
10 uL Q5
Template #84
Primer #82
Primer #16
Lp3050 Ghrelin
7 uL DMSO
10 uL Q5
Template #84
Primer #81
Primer #12
PelB PYY
7 uL DMSO
10 uL Q5
Template #84
Primer #80
Primer #8
Lp3050 GLP1
7 uL DMSO
10 uL Q5
Template #85
Primer #81
Primer #16
pelB Ghrelin
7 uL DMSO
10 uL Q5
Template #85
Primer #80
Primer #12
M6 PYY
7 uL DMSO
10 uL Q5
Template #85
Primer #82
Primer #8
M6 Ghrelin
7 uL DMSO
10 uL Q5
Template #86
Primer #82
Primer #12
PhoA PYY
7 uL DMSO
10 uL Q5
Template #86
Primer #79
Primer #8
b. After the PCR was completed it was purified and then it was nano-dropped it was then labeled and place into the box
3. Then a colony PCR was done
a. This was based on the plates that were done for the transformations
i. There were three plates and one of them was the positive control plate
1. The colonies were picked and diluted in 15 uL of water
2. Then the PCR was set up
a. The following was added in the PCR:
b.
uL
What is added?
1
Primer 51
1
Primer 52
1
Colony diluted templates
7
water
10
Q5
c. After the PCR was done it was sent off for sequencing
i. We await for those results
4. Cells were grown overnight
1. Midi-Prep was done on the flask in the incubator
a. This was left overnight to grow
2. PCR was done to add in the restriction sites
a. The PCR was set the following way:
Pel B GLP1
7 uL DMSO
10 uL Q5
Purified
Pel B GLP1
Primer #35
Primer #30
PhoA Ghrelin
7 uL DMSO
10 uL Q5
Purified
PhoA Ghrelin
Primer #36
Primer #28
Lp3050 PYY
7 uL DMSO
10 uL Q5
Purified
Lp3050 PYY
Primer #37
Primer #32
M6 GLP1
7 uL DMSO
10 uL Q5
Purified
M6 GLP1
Primer #38
Primer #30
Lp3050 Ghrelin
7 uL DMSO
10 uL Q5
Purified Lp3050 Ghrelin
Primer #37
Primer #28
PelB PYY
7 uL DMSO
10 uL Q5
Purified
PelB PYY
Primer #35
Primer #32
Lp3050 GLP1
7 uL DMSO
10 uL Q5
Purified
Lp3050 GLP1
Primer #37
Primer #37
pelB Ghrelin
7 uL DMSO
10 uL Q5
Purified
pelB Ghrelin
Primer #35
Primer #35
M6 PYY
7 uL DMSO
10 uL Q5
Purified
M6 PYY
Primer #38
Primer #38
M6 Ghrelin
7 uL DMSO
10 uL Q5
Purified
M6 Ghrelin
Primer #38
Primer #28
PhoA PYY
7 uL DMSO
10 uL Q5
Purified
PhoA PYY
Primer #36
Primer #32
3. After the PCR was completed it was purified and then it will be nano-dropped
a. The DNA concentrations were to high
i. This could mean that it wasn’t that purify
ii. So we ran it on a gel and the results weren't that great
4. Therefore, we ran this PCR again with different templates
a. The templates were the g-blocks and instead of using Q5 Kapa Hifi was used
b. After the PCR was over it was run on a gel
i. The results were not that great there are still some smears in the gel
c. So it seems that the problem is that the primers the reverse ones are not appearing on the g-block which seems to be the problem
d. This was reviewed over again and now a new set of primers is going to be ordered
