<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BIT-China_Project_Transduction skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BIT-China/Project/Transduction</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>BIT-CHINA</TITLE></P><DIV class="screen-hidden"><NAV class="navbar navbar-default navbar-fixed-top"><DIV class="container-fluid"><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav"><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project</A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Project/Description"><SPAN>Description</SPAN></A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Project/Sense"><SPAN>Receptor Expression</SPAN></A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Project/Transduction"><SPAN>Pathway optimization</SPAN></A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Project/Detection"><SPAN>Signal Output</SPAN></A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Project/Results"><SPAN>Result</SPAN></A></LI></UL><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Modeling</A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Model"><SPAN>Overview</SPAN></A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Model/Docking"><SPAN>Docking</SPAN></A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Model/GPCRpathway"><SPAN>GPCR pathway</SPAN></A></LI><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Laboratory</A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Material"><SPAN>Material</SPAN></A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Notebook"><SPAN>Notebook</SPAN></A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Protocol"><SPAN>Protocol</SPAN></A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Equipment"><SPAN>Equipment</SPAN></A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Safety"><SPAN>Safety</SPAN></A></LI><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Achievement
              </A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Parts"><SPAN>Parts</SPAN></A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Medal"><SPAN>Medal</SPAN></A></LI><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">HP</A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/HP/Silver"><SPAN>HP-Silver</SPAN></A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/HP/Gold_Integrated"><SPAN>integrated＆Gold</SPAN></A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Engagement"><SPAN>Engagement</SPAN></A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Collaborations"><SPAN>Collaboration</SPAN></A></LI><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team
              </A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Member"><SPAN>Member</SPAN></A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Attributions"><SPAN>Attribution</SPAN></A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Sponsor"><SPAN>Sponsor</SPAN></A></LI></DIV></DIV></NAV></DIV><SECTION class="content_container" id="mytop"><H2 class="title-h2">Pathway optimization</H2><DIV class="section-upline cd-section" id="knockout"><H3 class="title-h3">gene knockout</H3><P class="my-content-p">In order to amplify the intensity of the signal, we need to knock out some genes in yeast, including <I>ste2</I>, <I>sst2</I> and <I>far1</I> which exist in the endogenous mating pheromone response pathways in yeast and respond to the mating signal and transduction of downstream signal.</P><DIV class="my-img-box" style="justify-content: flex-start;"><TABLE class="table-co"><THEAD><TR><TH style="width: 50%">Gene</TH><TH>Function</TH></TR></THEAD><TBODY><TR><TD><I>ste2</I></TD><TD>The GPCR of yeast pheromone receptor is Ste2 protein, which is an important factor during yeasts mating.</TD></TR><TR><TD><I>sst2</I></TD><TD>Sst2 protein is a phosphatase catalyzing GTP and producing GDP which will inhibit the release of G<SUB>γ</SUB>、<SUB>β</SUB> subunits. Gene knockout of <I>sst2</I> can improve the pheromone GPSTP.</TD></TR><TR><TD><I>far1</I></TD><TD>α factor is an active factor of the yeast pheromone GPSTP, which can cause the yeast cell cycle arresting in the G1 phase through Far1 protein, resulting in that cell growth is inhibited.</TD></TR></TBODY></TABLE></DIV><P class="my-content-p">Gene knockout can be achieved by homologous recombination.</P></DIV><DIV class="section-upline cd-section" id="homologous"><H3 class="title-h3">homologous recombination</H3><P class="my-content-p">By analyzing <I>ste2</I>, <I>sst2</I> and <I>far1</I> gene sequences from the NCBI, we ensured the appropriate homologous area in the upstream and the downstream of these genes in <I>saccharomyces cerevisiae</I> genome. In order to knock out these genes effectively, we selected different lengths of homologous arms, 50bp, 200bp and 500bp. According to our experiment result, we found that 200bp and 500bp homologous arms worked more efficiently.</P><DIV class="my-content-box"><SPAN>Fig.1 Homologous combination knock-out strategy</SPAN></DIV><P class="my-content-p">We designed 3 pairs of primers whose templates were the upstream homologous arm, the marker and the downstream homologous arm respectively. The markers were <I>Histone</I> synthesis gene, <I>Uracil</I> synthesis gene and <I>Tryptophan</I> synthesis gene, abbreviated as <I>his</I>, <I>ura</I> and <I>trp</I>. Initially, we got 3 fragments by PCR and they had their own overlapping areas with corresponding to each homologous arms. </P><P class="my-content-p">Secondly, the recombination fragment obtained from OE-PCR was transformed to the <I>CEN.PK2-1C</I> which has been already knocked out the endogenous genes <I>ura, his</I> and <I>trp</I>. Additionally, the colony was chosen by the relevant nutritional deficiency medium. </P></DIV><DIV class="section-upline cd-section" id="verification"><H3 class="title-h3">verification</H3><P class="my-content-p">To verify whether the gene was actually knocked out and to avoid false positive colonies, we designed primer1, 2, 3 and 4 for each gene, as shown in Fig.2. The primer 1 and primer 4 were designed according to the yeast genome. The primer 2 was designed according to the selection maker and primer 3 was to the gene which would be knocked out. </P><DIV class="my-content-box"><SPAN>Fig.2 Verification strategy</SPAN></DIV><P class="my-content-p">We got positive result using the primer 1 and 2, meanwhile the negative result would be detected by using primer 1 and 3. Then we sequenced the PCR product using primer 1 and 4 to further confirm the realness of the results.</P></DIV></SECTION><A class="my-backTop" href="#mytop"><SPAN>TOP</SPAN></A><NAV id="cd-vertical-nav"><UL><LI><A href="#knockout" data-number="1"><SPAN class="cd-label">gene knockout</SPAN></A></LI><LI><A href="#homologous" data-number="2"><SPAN class="cd-label">homologous recombination</SPAN></A></LI><LI><A href="#verification" data-number="3"><SPAN class="cd-label">verification</SPAN></A></LI></UL></NAV><DIV class="footer-all"><DIV class="footer-right"><H5 class="title-h4">Contact</H5><P class="my-content-p3">Institute of Biotransformation and synthetic biosystem</P><P class="my-content-p3">School of life science </P><P class="my-content-p3">Beijing Institute of Technology</P><P class="my-content-p3">100081, Beijing</P><P class="my-content-p3">Email: lichun@bit.edu.cn</P></DIV></DIV><DIV id="footer-info" class="inner"><P class="my-content-p3" style="color: #777; text-align: center !important;">Copyright  © 2017 BIT-China</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>