<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Botchan_Lab_Tokyo_Project skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Botchan Lab Tokyo/Project</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><UL><DIV id="menu_bar"><A href="https://2017.igem.org/Team:Botchan_Lab_Tokyo">HOME</A><DIV id="drop_down_menu"><BUTTON class="buttonbar"><SPAN style="border-bottom: 2px solid black">PROJECT</SPAN></BUTTON><DIV id="project_contents"><A href="https://2017.igem.org/Team:Botchan_Lab_Tokyo/Project">RESULTS</A><A href="https://2017.igem.org/Team:Botchan_Lab_Tokyo/Notebook">LABORATORY NOTEBOOK</A><A href="https://2017.igem.org/Team:Botchan_Lab_Tokyo/Model">MODELING</A></DIV></DIV><DIV id="drop_down_menu1"><BUTTON class="buttonbar1"><SPAN style="border-bottom: 2px solid black">LAB</SPAN></BUTTON><DIV id="lab_contents"><A href="https://2017.igem.org/Team:Botchan_Lab_Tokyo/Team">TEAM</A><A href="https://2017.igem.org/Team:Botchan_Lab_Tokyo/Attributions">ATTRIBUTIONS</A><A href="https://2017.igem.org/Team:Botchan_Lab_Tokyo/InterLab">INTERLAB</A></DIV></DIV><DIV id="drop_down_menu2"><BUTTON class="buttonbar2"><SPAN style="border-bottom: 2px solid black">HUMAN PRACTICE</SPAN></BUTTON><DIV id="humanpractice_contents"><A href="https://2017.igem.org/Team:Botchan_Lab_Tokyo/HP/Silver">SILVER</A><A href="https://2017.igem.org/Team:Botchan_Lab_Tokyo/HP/Gold_Integrated">GOLD</A><A href="https://2017.igem.org/Team:Botchan_Lab_Tokyo/Collaborations">COLLABOLATIONS</A></DIV></DIV></DIV></UL></P><DIV id="top-logo-project"><P><CENTER><SPAN style="font-family:Monotype Corsiva;color:#ffffff;font-size:50px;">PROJECT</SPAN></CENTER></P><DIV class="linkbox-project"><A href="#link1-project">overview</A><A href="#link2-project">system</A><A href="#link3-project">results</A><A href="#link4-project">Indophenol results</A><A href="#link5-project">protocol</A></DIV></DIV><DIV id="inpagelink-project"><A href="#link1-project">overview</A><A href="#link2-project">system</A><A href="#link3-project">results</A><A href="#link4-project">Indophenol results</A><A href="#link5-project">protocol</A></DIV><DIV id="divarea2-project"><DIV id="box1-project"><H1><CENTER>Overview of the project</CENTER></H1>
Our team's project goal is to synthesizing amino acids from nitrogen in the air. We thought of constructing the pathway  in <I>Escherichia coli</I>, and by introducing the products to intestine, it will be possible for people to use the synthesized amino acids. We devided into following three specific pathways.
The First pathway is expressing nitrogenase to create ammonia from nitrogen gas. To express nitrogenase inside <I>E.coli</I>, we introduced there nif gene group and make them express. Although we want synthesized ammonia to be metabolized for synthesizing amino acids, acuumulated ammonia will be maybe used in ammonia metabolic pathway originally in <I>E.coli</I>.
Therefore, as the second pathway, we used an inhibitor called MSX to inhibit ammonia to be metabolized by <I>E.coli</I> and make them acuumulated.
And finally, by introducing and expressing genes of amino acid dehydrogenase, synthesize amino acids from ammonia. But this time we wanted to target on glutamic acid dehydrogenase and synthesize it.
The above is the overview of our project.

</DIV><DIV id="box2-project"><H1><CENTER>System</CENTER></H1><P id="link2-project"><CENTER><I>NifBHDKENX hesA NifVSU</I></CENTER><B><I>NifB</I></B>: synthesize the enzymes needed for making the structure.
Synthesizing FeMoco structure needs Fe and S when synthesizing the small Fe-S fragment, and these small Fe-S fragments become FeMoco structure catalyzed by the enzyme purify <I>NifB</I>. (This structure starts by <I>NifSU</I> in high possibity?)
<B><I>NifH</I></B>：the <I>nifH</I>-encoded Fe protein (an essential factor for FeMoco maturation) allows the accumulation of a precursor form of FeMoco on <I>NifEN</I>.<I>NifH</I> is used when nitrogen Fe protein (the factor necessary for maturing FeMoco) accumulate pre-FeMoco structure in <I>NifE,N</I>.
<B><I>NifEN</I></B>：FeMoco<I>NifEN</I> is functioned as a scaffold protein for the maturation of FeMoco
FeMoco catalyzed with <I>NifB</I> is send to <I>NifEN</I>, and functioned as scaffold protein for maturing FeMoco.<B><I>NifX</I></B>: Functions as modifying factor after translation, which control the important residue of protein for cluster formation.<B><I>Nif DK</I></B>: Codes molybdenum-iron protein.
Nitrogenase is formed by Fe-protein and MoFe-protein encoded by <I>NifH</I>.<B><I>NifUS</I></B>: <I>NifUS</I> is necessary for synthesizing Fe-s fragments, which is necessary as the one of the steps of synthesizing FeMoco structure. And synthesis of Fe-S fragments is done by the genes encoded by <I>NifUS</I>.<B><I>NifV</I></B>: Encode the homocitrate synthase that supplies homocitrate for the biogenesis of FeMoco
Homocritrate is used for stabilizing FeMoco structure and bind with one of the FeMoco cofactors.<B><I>hesA</I></B>: Forms acyladenylate complex that makes Sulfur move, and gets involved in the ATP dependant process which activates C-terminal of ubiquitin-like protein.</P></DIV><DIV id="box3-project"><H1><CENTER>Results</CENTER></H1><P id="link3-project"><CENTER>pHY <I>NifB~hesA</I></CENTER>

We amplified the genes of <I>nifB~hesA</I> from <I> Paenibacillus polymyxa</I> ATCC strain 15970 and the sequences which include the ampicillin resistance genes from the plasmid pHY. The following figure 1 shows the result of electrophoresis.
</P>
We used the same marker for all of the electrophoresis.
<CENTER>Figure 1 Electrophoresis of PCR product</CENTER>
1 maker /2,3 pHY300PKL/4,5 pUC19/6,7 pUC-<I>NifB~hesA</I>/8,9 pHY-<I>NifB~hesA</I>
The amplifications of both samples were confirmed by electrophoresis.
We purified those products of PCR and confirmed by electrophoresis. The result of the electrophoresis is shown in the figure 2 below.
<CENTER>Figure 2 Electrophoresis of purification product</CENTER>
1 maker/2 pHY-<I>NifB~hesA</I> /3 pUC-<I>NifB~hesA</I>
From this result, the target fragments were comfirmed.
After purification, we combined the end together by homologous recombination, and transformed. The result of this process is shown below.
<CENTER>Figure 3 Electrophoresis of purification product</CENTER>
4 maker/5 pHY300PKL/6 pUC19
We got some colonies and from the colony, we extract plasmid and conducted electrophoresis. The result of the electrophoresis is shown below.
<CENTER>Figure 4 Transformation colony</CENTER>
According to the result, we confirmed that we succeeded the construction of the vector.
We also conducted insert check PCR and confirmed the amplification of the target fragments.
<CENTER>Figure 5 Electrophoresis of plasmid extraction</CENTER>
From this result, we confirmed the success of the construction of the vector that includes the target fragments, and the transformation to the <I>E.coli</I>. Next, we checked if this <I>E.coli</I> could make ammonia from nitrogen in the air by indophenol method.
<CENTER>Figure 6 Electrophoresis of insert check PCR product</CENTER>
We amplified the genes of <I>nifVUS</I> from <I>Rahnella aquatilis</I> NBRC strain 105701 and the sequences that include the kanamycin resistance genes from the plasmid pMW. We comfirmed the result by electrophoresis.
<CENTER>Figure 7 Electrophoresis of PCR product</CENTER>
1 maker/2 <I>NifVSU</I>
The amplification of both samples were confirmed through electrophoresis
We purified these products of PCR and we conducted electrophoresis. The result of electrophoresis is shown below.
<CENTER>Figure 8 Electrophoresis of Plasmid extraction product</CENTER>
From this result, our target fragments were comfirmed.
We conducted the restriction digest using Sac 1 and Xba 1 to the products of the PCR, and ligated the end of the each fragment, and transformed it to the <I>E.coli</I> JM109. We got the result below.
<CENTER>Figure 9 Transformation colony</CENTER>
We amplificated the genes from the colonies we got by PCR to confirm the success of construction of the vector and the transformation of the vector to the <I>E.coli</I>.
We found the six colonies which we succeeded the transformation.
<CENTER>Figure 10.11 Electrophoresis of insert check colony PCR product</CENTER>
1 maker/2 <I>Rahnella aquatilis</I> NBRC 105701/3~46 insert check colony PCR product.From the result, we could confirmed our target fragments.

We got insert check colony PCR product, we have a sample that may contain inserts , we extract plasmid and conducted electrophoresis. The result of the electrophoresis is shown below.
<CENTER>Figure 12 Electrophoresis of Plasmid extraction product</CENTER>
1 maker/ 2 ~7 <I>NifVSU</I> Plasmid extraction product
From the result, we could confirmed our target fragments.

We amplified the genes of <I>pheDH</I> from <I>Bacillus badius</I> NBRC strain 15713 and the genes of <I>gluDH</I> from <I>Bacillus licheniformis</I> NBRC strain 12200 by PCR. We comfirmed the result by electrophoresis and got the result below.
<CENTER>Figure 13 Electrophoresis of PCR product</CENTER>
1 maker/2.3 <I>pheDH</I>/4.5 <I>gluDH</I>
From the result, we could confirmed our target fragments.
</DIV><DIV id="box4-project"><CENTER><H1>Indophenol results</H1></CENTER><P id="link4-project">
Indophenol method is a way to quantify of ammonia by coloration. This time, we used indophenol method and measured the activity of nitrogenase that synthesizes ammonia.
</P><CENTER>Figure 1. The result of 24 hours cultivation of strain ATCC 15970</CENTER>
The micro tubes above are 100-fold dilution and the micro tube below is 10-fold dilution. From left, 1 Nitrogen-free medium only (negative control), 2 none, 3 none+MSX,4 gul, 5 gul+MSX,6 NH<SUB>4</SUB>Cl,7 NH4Cl+MSX.

In the samples of 24-hour cultivation, the bacteria didn’t grow enough and the nitrogen fixation hasn’t preceded enough. Among the sample ‘none’, ‘none+MSX’, ‘gul’, and ‘gul+MSX’, only ‘ none’ has colored. This is considered because by putting MSX, the amount of the bacteria decrease, so there are no bacteria colored in 24 hours. We don’t know why the glutamic acid-added-sample didn’t colored.
<CENTER>Figure 2. The result of 48 hours cultivation of strain ATCC 15970</CENTER>
These samples are all 100-fold dilution. 1 Negative control. 2 None, 3 none+MSX, 4 gul, 5 gul+MSX, 6 NH<SUB>4</SUB>Cl, 7 NH4Cl+MSX from left.

In the samples of 48-hour cultivation, other samples started to color. This is considered that as the bacteria grow and color, they start to synthesize ammonia. From this, we can say that that the genes from <I>Paenibacillus polymyxa</I> has correctly expressed in <I>E.coli</I> and has got nitrogenase activity.
<CENTER>Figure.3 The result of 72 hours cultivation of strain ATCC 15970</CENTER>
These samples are all 100-fold dilution. 1 Negative control. 2 None, 3 none+MSX, 4 gul, 5 gul+MSX, 6 NH<SUB>4</SUB>Cl, 7 NH4Cl+MSX from left.
<CENTER>Figure 4. The result of the absorbance of strain ATCC15970</CENTER>
In the samples of 72-hour cultivation, the color of the sample ‘none’ has become lighter, on the other hand, the color of the sample ‘none+MSX’ became darker. The coloration of MSX is because the bacteria which of glutamine synthase were inhibited by MSX and ammonia was accumulated have increased. The reason of the color of the sample ‘none’ got lighter is considered that the increase of ammonium concentration made the nitrogenase deactivation.
<UL><LI>The color of the sample ‘none+MSX’ is darker compared to the sample with nothing, ‘none’. From this result, we can consider that 
     glutamine synthase was inhibited by MSX and the ammonia has accumulated.</LI><LI>When adding glutamic acid, we can estimate that the equilibrium of glutamine and glutamic acid collapse and the production of glutamine increase, making the accumulation of ammonia. However, from Figure 4, there is no increase of the activity. MSX wasn’t effective when glutamic acid was added. We can think that when glutamic acid was added, the nitrogen source that <I>Paenibacillus polymyxa</I> can use increases, the amount of bacteria increases.Because it is not measuring turbidity, we cannot confirm the result of adding glutamic acid.</LI><LI>When NH4Cl was added, it was dyed in dark blue, but the concentration of NH4Cl was high, so we are not sure if <I>Paenibacillus 
     polymyxa</I> synthesized ammonia.</LI></UL><CENTER>Figure5. The result of indophenol method after 24-hour cultivation of strain ATCC15970 and strain JM109 (<I>nifB~hesA</I>). 10-fold dilution samples.</CENTER><OL><LI><I>ATCC15970</I>(none)</LI><LI><I>ATCC15970</I>(none+MSX)</LI><LI><I>NifB~hesA</I>(none)</LI><LI><I>NifB~hesA</I>(none+MSX) </LI><LI><I>NifB~hesA</I>(none+ homocitrate 50 µL)</LI><LI><I>NifB~hesA</I>(none+MSX+ homocitrate 50 µL)</LI><LI><I>NifB~hesA</I>(none+ homocitrate 5 µL)</LI><LI><I>NifB~hesA</I>(none+MSX+ homocitrate 5 µL)</LI><LI><I>NifB~hesA</I>(none+ homocitrate 0.5 µL)</LI><LI><I>NifB~hesA</I>(none+MSX+ homocitrate 0.5 µL)</LI></OL>
Because these are 24-hour cultivation samples, there is no big change of the color, but some samples are colored lightly so we can know that ammonia was synthesized. The samples with MSX colored more darkly by the effect of MSX.
<CENTER>Figure6. The result of indophenol method after 48-hour cultivation of strain ATCC15970 and strain JM109 (<I>nifB~hesA</I>). 10-fold dilution samples.</CENTER><OL><LI><I>ATCC15970(none)</I></LI><LI><I>ATCC15970</I>(none+MSX)</LI><LI><I>NifB~hesA</I>(none)</LI><LI><I>NifB~hesA</I>(none+MSX) </LI><LI><I>NifB~hesA</I>(none+ homocitrate 50 µL)</LI><LI><I>NifB~hesA</I>(none+MSX+ homocitrate 50 µL)</LI><LI><I>NifB~hesA</I>(none+ homocitrate 5 µL)</LI><LI><I>NifB~hesA</I>(none+MSX+ homocitrate 5 µL)</LI><LI><I>NifB~hesA</I>(none+ homocitrate 0.5 µL)</LI><LI><I>NifB~hesA</I>(none+MSX+ homocitrate 0.5 µL)</LI></OL>
All samples colored darker than 24-hour cultivation samples. From the result, the difference of the amount of synthesized ammonia between the sample without MSX and the samples with MSX became conspicuous. 
<CENTER>Figure 7. The result of indophenol method after 24-hour cultivation of strain ATCC15970, strain JM109 (<I>nifB~hesA</I>), and strain JM109 (only pHY). 10-fold dilution samples.</CENTER>
The top row: strain ATCC15970. From left, 1 none, 2 none+MSX, 3 homocitrate, 4 homocitrate +MSX.
The second row: strain JM109 (only pHY). From left, 5 none, 6 none+MSX’, 7 homocitrate, 8 homocitrate +MSX.
The third row: strain JM109 (pHY + <I>nifB~hesA</I>). From left, 9 none, 10 none+MSX, 11 homocitrate, 12 homocitrate +MSX.<OL>
・In strain ATCC15970, The effect of MSX hasn’t appeared, but the synthesis of ammonia can be confirmed by coloration. As you can know 
     from the result of figure 1,2,3 (10-fold dilution), MSX become effective as time passes.
・<I>E.coli</I> with plasmids but no insert of target genes (empty vectors), the sample ‘none’ wasn’t colored and the samples that 
     MSX was added colored. This is because synthesized ammonia is used for glutamine synthesis, making the sample ‘none’ un-colored.     When we inhibited glutamine synthase by MSX, ammonia is accumulated. Therefore, in terms of the coloration of the samples of 
     transformed <I>E.coli</I> with target genes, the sample ‘none’ is thought to be because of nitrogenase. And comparing the samples 
     with insert DNA and without insert DNA (empty vector), both with MSX, the coloration is darker in the samples with insert DNA, so the 
     effect of MSX can be confirmed. 
・When adding citric acid, transformed <I>E.coli</I> didn’t colored. Homocitric acid is important substance of nitrogenase formation, 
     but it seems homocitric acid worked as negative factor.
・The samples with both citric acid and MSX colored into blue. This is thought to be due to the effect of MSX.</OL><CENTER>Figure 8. The result of indophenol method after 48-hour cultivation of strain ATCC15970, strain JM109 (<I>nifB~hesA</I>), and strain JM109 (only pHY). 10-fold dilution samples.</CENTER>
The top row: strain ATCC15970. From left, 1 none, 2 none+MSX, 3 homocitrate, 4 homocitrate +MSX.
The second row: strain JM109 (only pHY). From left, 5 none, 6 none+MSX, 7 homocitrate, 8 homocitrate +MSX.
The third row: strain JM109 (pHY + <I>nifB~hesA</I>). From left, 9 none, 10 none+MSX, 11 homocitrate, 12 homocitrate +MSX.<UL><LI>In 48-hour cultivation samples, they colored darker compared to 24-hour cultivation samples. We considered that the amount of bacteria 
    increased and led the increase of the amount of accumulation of ammonia. However, we can’t see the difference of color by the effect 
    of MSX in strain ATCC. </LI></UL></DIV><DIV id="box5-project"><H1><CENTER>Protocol</CENTER></H1><P>We used 4 different strains<I>NifB～hesA</I><I>Paenibacillus polymyxa</I> ATCC 15970<I>NifVSU</I><I>Rahnella aquatilis</I> NBRC 105701<I>PheDH</I><I>Bacillus badius</I> NBRC 15713<I>GluDH</I><I>Bacillus licheniformis</I> NBRC 12200</P>
We used the same marker for the result of electrophoresis.
<B>LD preparation</B>
LD medium contain following 
<DIV align="center"><DIV id="table1"><TABLE border="1"><TBODY><TR><TD><B>Materials</B></TD><TD><B>Volume</B></TD></TR><TR><TD>NaCl</TD><TD>2.5g</TD></TR><TR><TD>yeast extract</TD><TD>5.0g</TD></TR><TR><TD>typtone</TD><TD>10g</TD></TR><TR><TD>dH<SUB>2</SUB>O</TD><TD>1L</TD></TR><TR><TD>pH 6.8～7.2</TD></TR></TBODY></TABLE></DIV></DIV><B>Glutamic free medium　preparation</B>
Glutamic free medium contain following<PRE>
Na<SUB>2</SUB>HPO<SUB>4</SUB>,      　　　      10.4g
 
KH<SUB>2</SUB>PO<SUB>4</SUB>,      　　　        3.4g

CaCl<SUB>2</SUB>·2H<SUB>2</SUB>O,      　　　    26mg

30 mg MgSO<SUB>4</SUB>,      　　　   30mg

MnSO<SUB>4</SUB>,      　　　        0.3mg

Ferric citrate,      　　　36mg
 
Na<SUB>2</SUB>MoO<SUB>4</SUB>·2H<SUB>2</SUB>O      　　　   7.6mg

p-aminobenzoic acid,      10μg

biotin      　　　         5μg

glucose                      4g

distilled water      　　　  1L 
</PRE><B>PCR or colony PCR</B>
Mix PCR solutions and run the PCR machine in a program which is detailed below.

<DIV align="center"><DIV id="table1"><TABLE border="1"><TBODY><TR><TD><B>PCR Solution</B></TD><TD>Primer-F 10µM</TD><TD>template</TD><TD>Primer-R 10 µM</TD><TD>2mM dNTPs</TD><TD>2 x PCR Buffer</TD><TD>KOD Fx Neo DNA Polymerase</TD><TD>DW</TD><TH>Total</TH></TR><TR><TD><B>Volume (µL)</B></TD><TD>1.5</TD><TD>x</TD><TD>1.5</TD><TD>10</TD><TD>25</TD><TD>1</TD><TD>11</TD><TH>50+x</TH></TR></TBODY></TABLE></DIV></DIV><PRE>
Cycle 1:(1 x)	Step 1	94℃ for 2:00 		

Cycle 2:(35 x)	Step１	98℃ for 0:10	Step 2	68℃ for 02:00(pUC)  
				                     for 10:00(<I>nif BHDKEVX-hesA</I>)
　　　　　　　　　　　　　　　　　　　　　　　　　　 for 06:00(<I>nifVSU</I>)
                                                     for 04:00(<I>pheDH,gluDH</I>)
Cycle 3:(1 x)	Step 1	4℃ for ∞		
 
primer　5’→3’ 
pUC19-f　gaatcagggg ataacgcagg aaaga 
pUC19-r　atccgctcat gagacaataa ccctg 
pUC19-<I>nif</I>-f　cagggttatt gtctcatgag cggat 
pUC19-<I>nif</I>-r　tctttcctgc gttatcccct gattc 
pHY300PLK-f 　cgtgtttttcttggaattgtgctgt 
pHY300PLK-r　 tcgacctgcagatctctagaagctt 
pHY-<I>nif</I>-f  aagcttctagagatctgcaggtcga 
pHY-<I>nif</I>-r  acagcacaattccaagaaaaacacg<I>nifVSU</I>-f　 cac <U>GAG CTC</U> ttgtacgacatcacccgaacc
                 SacI
<I>nifVSU</I>-r　tccc <U>CCC GGG</U> tcagcctccctgagcctgaag
                 XmaI 
pheDH-f　tccc <U>CCC GGG</U> ggctaaataaaagcgttcaac 
                XmaI 
pheDH-r　tgc <U>TCT AGA</U> ttagttgcgaatatcccattt
               XbaI 
gluDH-f　tccc <U>CCC GGG</U> aaaatcttacaatgttcgttc
                XmaI 
gluDH-r　tgc <U>TCT AGA</U> ttaaacgactccctgagcgat
               XbaI
</PRE><B>Electrophoresis</B> <B>１</B>.Prepare agarose gel  1. Measure 0.2 g of agarose (1-2%) to 20 mL of TAE in a beaker 2. Microwave until all agarose has dissolved 3. Allow agarose solution to cool down 4. Pour solution into gel tray. 5. Leave until gel has solidified then remove the com<B>２</B>. Insert the gel into the electrophoresis chamber with the wells closest to the negative (black) electrode.  
<B>３</B>. Gradually add 1xTBE (Tris-Borate-EDTA; electrophoresis buffer) to the chamber until the buffer just covers the top of the gel.
<B>４</B>．Add 2 µl of 6 x loading buffer to 5 µl of each sample.<B>５</B>．Add 5 µl of each sample with 2 µl of 10 x loading buffer and 0.5~5kb marker .<B>６</B>．Start electrophoresis at 100V.<B>７</B>．Stop at appropriate time. <B>８</B>．Put the gel in the container which is filled with EtBr<B>Cloning</B>
We used in-fusion cloning<B>１</B>.We refined PCR mixture by GFX　pulification kit.<B>２</B>. Set up the in-Fusion cloning reaction:<DIV align="center"><DIV id="table1"><TABLE border="1"><TBODY><TR><TD>5X In-Fusion HD Enzyme Premix </TD><TD>２µl</TD></TR><TR><TD>Linearized vector </TD><TD>1 µl</TD></TR><TR><TD> PCR fragment</TD><TD>3 µl</TD></TR><TR><TD>typtone</TD><TD>4 µl</TD></TR><TR><TD>dH2O </TD><TD>1L</TD></TR><TR><TH>total volume </TH><TH>10 µl</TH></TR></TBODY></TABLE></DIV></DIV><B>３</B>. Incubate the reaction for 15 min at 50 °C, then place on ice.<B>４</B>. store the cloning reaction at refrigerator(-20℃) over night.<B>Transformation</B><B>１</B>.Thaw competent cells(JM109) on ice<B>２</B>. Add plasmid DNA (1 µl) to competent cells. leave them on ice for 30 minutes<B>３</B>. Keep them in heating block(42℃) for 45 seconds, then cool on ice for 1 minutes<B>４</B>. Add 500 µl SOC liquid culture medium, then incubate with shaking for 1hour at 37°C<B>５</B>. Seed cells onto LB+ampicillin agar plate. Incubate for 24 hours at 37℃<B>Gel purification</B>
We used illustra GFX PCR DNA and Gel Band Purification Kit<B>１</B>.Sample Capture
a.	Using a clean scalpel, long wavelength (365 nm) ultraviolet light and minimal exposure time, cut out an agarose band containing the sample of interest. 1.Sample Capture
b.	Add 10 µl Capture buffer type 3 for each 10 mg of gel slice, for example, add 300 µl Capture buffer type 3 to each 300 mg gel slice.
c.	Mix by inversion and incubate at 60°C for 15–30 minutes until the agarose is completely dissolved. Mix by inversion every 3 minutes.
d.	For each purification that is to be performed, place one GFX MicroSpin column into one Collection tube.<B>２</B>. Sample Binding
a. Centrifuge Capture buffer type 3- sample mix briefly to collect the liquid at the bottom of the tube.
b. Transfer up to 800 µl Capture buffer type 3- sample mix onto the assembled GFX MicroSpin column and Collection tube.
c. Incubate at room temperature for 1 minute.
d. Spin the assembled column and Collection tube at 16 000 × g for 30 seconds.
e. Discard the flow through by emptying the Collection tube. Place the GFX MicroSpin column back inside the Collection tube.
f. Repeat Sample Binding steps b. to e. as necessary until all sample is loaded.<B>３</B>. Wash &amp; Dry
 a. Add 500 µl Wash buffer type 1 to the GFX MicroSpin column.
b. Spin the assembled column and Collection tube at 16 000 × g for 30 seconds
c. Discard the Collection tube and transfer the GFX MicroSpin column to a fresh DNase-free 1.5 ml microcentrifuge tube (supplied by user).<B>４</B>.Elution
a. Add 10–50 µl Elution buffer type 4 OR type 6 to the center of the membrane in the assembled GFX MicroSpin column and sample Collection tube.
b. Incubate the assembled GFX MicroSpin column and sample Collection tube at room temperature for 1 minute.
c. Spin the assembled column and sample Collection tube at 16 000 × g for 1 minute to recover the purified DNA.
d. Proceed to downstream application. Store the purified DNA at -20°C.<B>Purification of plasmid DNA</B>
We used QIAprep Spin Miniprep Kit<B>１</B>. Resuspend pelleted bacterial cells in 250 μl Buffer P1 and transfer to a
microcentrifuge tube.<B>２</B>. Add 250 μl Buffer P2 and gently invert the tube 4–6 times to mix.<B>３</B>. Add 350 μl Buffer N3 and invert the tube immediately but gently 4–6 times.<B>４</B>. Centrifuge for 10 min at 13,000 rpm (~17,900 x g) in a table-top microcentrifuge.<B>５</B>. Apply the supernatants from step 4 to the QIAprep spin column by decanting or
pipetting.<B>６</B>. Centrifuge for 30–60 s. Discard the flow-through.<B>７</B>. (Optional): Wash the QIAprep spin column by adding 0.5 ml Buffer PB and
centrifuging for 30–60s. Discard the flow-through.<B>８</B>. Wash QIAprep spin column by adding 0.75 ml Buffer PE and centrifuging for
30–60s.<B>９</B>. Discard the flow-through, and centrifuge for an additional 1 min to remove residual
wash buffer.<B>１０</B>. Place the QIAprep column in a clean 1.5 ml microcentrifuge tube. To elute DNA,
add 50 μl Buffer EB (10 mM Tris・Cl, pH 8.5)  to the center of each QIAprep
spin column, let stand for 1 min, and centrifuge for 1 min.

※pre-heat Buffer EB (or water) to 70°C prior to eluting DNA<B>Restriction digests</B>
We used sac1 and Xma1 as restriction enzyme.<B>１</B>.Add 68 ul of DNA to be digested into a 1.5 ml microcentrifuge tube.<B>２</B>.Add 10 ul of 10x buffer.<B>３</B>.Add 20 ul of pSMW218 or PCR product.<B>４</B>.Add 2.0 ul of Sac1.<B>５</B>.Incubate the restriction digest at 37℃ for 5 hour.
Purification of DNA from an enzymatic reaction by GFX　pulification kit.<B>１</B>. 1.Add 68 ul of DNA to be digested into a 1.5 ml microcentrifuge tube.<B>２</B>.Add 10 ul of 10x buffer.<B>３</B>.Add 20 ul of pSMW218 or PCR product.<B>４</B>.Add 2.0 ul of Xma1.<B>５</B>.Incubate the restriction digest at 37℃ for overnight.
Purification of DNA from an enzymatic reaction by GFX pulification kit.</DIV><DIV id="box6-project"><H2><CENTER>Sponsor</CENTER></H2><H3>Contact</H3></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>