<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Marburg_Project_Notebook_October"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Marburg:Project:Notebook:October">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Marburg:Project:Notebook:October&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Marburg:Project:Notebook:October&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Marburg:Project:Notebook:October&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Marburg:Project:Notebook:October" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Marburg:Project:Notebook:October</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="navbox"><NAV class="menu center first" aria-haspopup="true" aria-controls="p1"><UL id="p1" class="first"><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg" class="a">Home</A></LI><LI class="current"><A href="https://2014.igem.org/Team:Marburg:Project" class="b">Project</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Safety" class="c">Safety</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Team" class="d">Team</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Policy_Practices" class="e">Policy &amp; Practices</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Attributions" class="f">Attributions</A></LI></UL></NAV><NAV class="menu center second" aria-haspopup="true" aria-controls="p1"><UL id="p1" class="second"><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:Silver" class="d">SilverSURF</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:Tumor" class="c">CancerSURF</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:NRPS" class="e">RiboSURF</A></LI><LI class="current"><A href="https://2014.igem.org/Team:Marburg:Project:Notebook" class="f">Notebook</A></LI><LI class="last" aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:Parts" class="g">Parts</A></LI></UL></NAV></DIV><DIV class="mr_content"><H1>

Notebook: October

</H1><UL id="submenu"><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook">March</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:April">April</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:May">May</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:June">June</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:July">July</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:August">August</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:September">September</A></LI><LI class="current"><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:October">October</A></LI><LI><A href="#01.10.2014">01.10.2014</A></LI><LI><A href="#02.10.2014">02.10.2014</A></LI><LI><A href="#03.10.2014">03.10.2014</A></LI><LI><A href="#04.10.2014">04.10.2014</A></LI><LI><A href="#05.10.2014">05.10.2014</A></LI><LI><A href="#06.10.2014">06.10.2014</A></LI><LI><A href="#07.10.2014">07.10.2014</A></LI><LI><A href="#08.10.2014">08.10.2014</A></LI><LI><A href="#09.10.2014">09.10.2014</A></LI><LI><A href="#10.10.2014">10.10.2014</A></LI><LI><A href="#11.10.2014">11.10.2014</A></LI><LI><A href="#12.10.2014">12.10.2014</A></LI><LI><A href="#13.10.2014">13.10.2014</A></LI><LI><A href="#14.10.2014">14.10.2014</A></LI><LI><A href="#15.10.2014">15.10.2014</A></LI><LI><A href="#16.10.2014">16.10.2014</A></LI></UL><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:Methods">Methods</A></LI><DIV class="notebooky-entry"><H2 class="title"><A name="01.10.2014">01.10.2014</A></H2><FIELDSET class="exp13"><LEGEND><A name="exp13.1">18.72 Isolation of flagella from Hag-D2-Strep</A></LEGEND><DIV class="aim"><P>Aim: Isolation of filaments for assays with StrepDARPidin</P></DIV><DIV class="exp-content"><P>After the purification steps until the ammonium sulphate precipitation a SDS-gel with coomassie staining was done and WT3610 as well as Hag-D2_3 were used as a control. </P><P>The gel shows that there is a band after the ammonium sulfate precipitation which is running to low to be flagellin. The isolation was repeated several times with the same result so that we think of unstable flagellins. Additionally, a <I>B. subtilis</I> PY79 strain with a Strep-Tag integrated into the normal Hag-gene was used for isolation of flagella. They might be more stable than Hag-D2-Strep filaments.</P></DIV></FIELDSET><FIELDSET class="exp25"><LEGEND><A name="exp25.4c">25.4 Fluorometer assay with A549</A></LEGEND><DIV class="aim"><P>Aim: Checking interaction of StrepDARPidin with EpCAM on A549 Lung carcinoma cells</P></DIV><DIV class="exp-content"><P>In order to check the interaction of StrepDARPidin with the EpCAM on  A549 Lung Carcinoma cells we planned an ELISA-Like assay. Yesterday row H 1-12 of  a black Fluotrac600 96-well plate was coated with 100000 cells A549/ well  (counted with Neubauer-Chamber) overnight at 37°C. </P><P>We intended to incubate them with different concentrations of  StrepDARPidin and then target the N-terminal His-Tag with and Anti-His antibody  Alexa488 conjugated (1:50 endvolumen). In the end the signal in comparison to  blank and negative control would be generated in an ELISA reader by stimulation  of the ALexa488 at 480 nm and measurement at 520 nm.</P><P>In well H1-6 gel filtration purified StrepDARPidin with 87 µM was  tested, in well H 7-12 Ni-NTA purified StrepDARPidin with 280 µM.</P><P>Following concentrations of StrepDARPidin dissolved in PBS (pH 7,4) +2,5  % glycerin were used:</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Well
Col</TH><TH scope="col">Well Row</TH><TH scope="col">Blank  corrected raw data (485, 520)</TH><TH scope="col">Purification</TH><TH scope="col">StrepDARPidin  concentration [M]</TH></TR><TR><TH scope="row">Neg. Control</TH><TD>2</TD><TD>G</TD><TD>14073</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Blank B</TH><TD>3</TD><TD>G</TD><TD> </TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Antibody signal</TH><TD>4</TD><TD>G</TD><TD>249059</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">A549</TH><TD>1</TD><TD>H</TD><TD>10418</TD><TD>GeFi</TD><TD>2 µM</TD></TR><TR><TH scope="row">A549</TH><TD>2</TD><TD>H</TD><TD>19020</TD><TD>GeFi</TD><TD>1 µM</TD></TR><TR><TH scope="row">A549</TH><TD>3</TD><TD>H</TD><TD>6673</TD><TD>GeFi</TD><TD>100 nM</TD></TR><TR><TH scope="row">A549</TH><TD>4</TD><TD>H</TD><TD>8559</TD><TD>GeFi</TD><TD>10 nM</TD></TR><TR><TH scope="row">A549</TH><TD>5</TD><TD>H</TD><TD>5743</TD><TD>GeFi</TD><TD>100 pM</TD></TR><TR><TH scope="row">A549</TH><TD>6</TD><TD>H</TD><TD>9033</TD><TD>GeFi</TD><TD>10 pM</TD></TR><TR><TH scope="row">A549</TH><TD>7</TD><TD>H</TD><TD>9090</TD><TD>Ni-NTA</TD><TD>2 µM</TD></TR><TR><TH scope="row">A549</TH><TD>8</TD><TD>H</TD><TD>8796</TD><TD>Ni-NTA</TD><TD>1 µM</TD></TR><TR><TH scope="row">A549</TH><TD>9</TD><TD>H</TD><TD>5356</TD><TD>Ni-NTA</TD><TD>100 nM</TD></TR><TR><TH scope="row">A549</TH><TD>10</TD><TD>H</TD><TD>6684</TD><TD>Ni-NTA</TD><TD>10 nM</TD></TR><TR><TH scope="row">A549</TH><TD>11</TD><TD>H</TD><TD>6930</TD><TD>Ni-NTA</TD><TD>100 pM</TD></TR><TR><TH scope="row">A549</TH><TD>12</TD><TD>H</TD><TD>10237</TD><TD>Ni-NTA</TD><TD>10 pM</TD></TR></TBODY></TABLE><P>The 96-well plate was taken out of the incubator and the medium was  carefully discarded. 100 µL of the dilutions were transferred into the wells  and incubated for 45 min. at room temperature under the Flow hood.</P><P>An Anti-His-Alexa488 conjugate (1 mg/mL stock) with 5 µL concentrate was  dissolved in 40 µL  PBS. 1 µL of the  antibody was added after washing to each well and incubated for 20 min at room  temperature in the dark.</P><P>As a negative control t-cells clones were used and incubated the same  way with 1 µM StrepDARPidin and 1 µL antibody (Well G2). Well G1 was left  empty, well G3 was filled with 100 µL PBS+Glycerin and G4 with PBS+2,5 %  glycerin with 1 µL Antibody.</P><P>After 20 min incubation the wells were washed carefully with 100 µL PBS  to wash out unbound elements.</P><P>The plate was measured in the ELISA reader:</P><P>We are able to see that the fluorescence signal decreases with  increasing dilution. Unfortunately the negative control was very high as well.  The t-cells were incubated in Eppis because of their non-adherent features.  They were incubated with StrepDARPidin and Antibody before. The supernatant was  discarded after centrifugation at 1500x g but the cells did not stay at the  pellet’s position so that it was not possible to remove the whole preincubated  material which might be left when measuring the control.</P><P>Additionally it is noticeable that there is the same trend for both  purification methods. The lower signal from NTA-purified StrepDARPidin can be  explained by the fact that the concentration was higher but the purity also  lower so that even less StrepDARPidin was found in the protein aliquods.  Unfortunately the negative control had a high signal as well. The T-cell clones  derive from human t-cells which carry the genes for  almost every protein and marker for human  cells. In the process of immortalization it is possible that genes are  expressed which do not belong to typical t-cells. A different negative control  has to be used. In order to increase the signal more protein with a different  serial dilution has to be used for reproducing the conditions. </P></DIV></FIELDSET><FIELDSET class="exp23"><LEGEND><A name="exp23.35">23.35 Mutagenesis PCR of pSB1C3 Hag-D2-Strep clone 1 and 2</A></LEGEND><DIV class="exp-content"><P>Mutagenesis Primer arrived → PCR with pSB1C3-Hag-<I>Kpn</I>I-Strep clone 1 and 2 (appr. 20 ng/µl) in 30 µl reaction</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="row">Plasmid</TH><TD>1</TD></TR><TR><TH scope="row">Water </TH><TD>13,2</TD></TR><TR><TH scope="row">Phusion  Buffer (5x)</TH><TD>6</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>1</TD></TR><TR><TH scope="row">dNTPs</TH><TD>1</TD></TR><TR><TH scope="row">Primer  81 (1:50)</TH><TD>3,9</TD></TR><TR><TH scope="row">Primer  82 (1:50)</TH><TD>3,9</TD></TR></TBODY></TABLE><P>1 min elongation time in standard  protocol was used. Whole PCR product was loaded on an agarose gel and purified  via gel extraction with resulting concentrations of 16 and 17 ng/µl. </P></DIV></FIELDSET><FIELDSET class="exp23"><LEGEND><A name="exp23.36">23.36 Gibson assembly and transformation mutated pSB1C3 Hag-D2-Strep clone 1 and 2</A></LEGEND><DIV class="exp-content"><P>5 µl of the purified plasmid were  used for a Gibson assembly reaction (total DNA amount appr. 80-90 ng). The  complete Gibson mix was later transformed into chemically competent <EM>E. coli</EM> XL-1-blue cells.</P></DIV></FIELDSET><FIELDSET class="exp"><LEGEND><A name="exp_">Growing cultures for electron microscopy</A></LEGEND><DIV class="aim"><P>Aim: cultures in different states of growth for electron microscopy of flagella</P></DIV><DIV class="exp-content"><P>The  structure of the flagella of the mutant strains <EM>Bacillus subtilis</EM> 3610 D2-Strep and <EM>Bacillus subtilis</EM> 3610 D2-cup should be investigated by electron  microscopy. Thus cultures of strain WT3619, 3610 D2-Strep and 3610 D2-Cup were  inoculated in 5 ml LB medium. The cultures were incubated shaking at 37°C until  they reached the exponential phase with an OD<SUB>600</SUB> of 0.7 - 0.8, and  until the cells reached the stationary phase. The cells were kept on the same  OD by storage on ice until further processing for electron microscopy.</P></DIV></FIELDSET><FIELDSET class="exp24"><LEGEND><A name="exp24.0">24. Streptavidin-Cup1-1 Fusion</A></LEGEND><DIV class="aim"><P>Aim: Fusion of Streptavidin to Cup1-1 </P></DIV></FIELDSET><FIELDSET class="exp24"><LEGEND><A name="exp24.1">24.1 amplification of Streptavidin and cup for Strep-cup</A></LEGEND><DIV class="aim"><P>Aim: amplification of Strep-Cup for the fusionprotein Streptavidin-Cup1-1</P></DIV><DIV class="exp-content"><P>The  truncated cup1-1 gene should be amplified with overhangs to the  streptavidin-gene and an overhang with a restriction site for <EM>Nco</EM>I. The used template was piGEM029.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix PCR</TH><TH scope="col">Single reaction (µl)</TH></TR><TR><TH scope="row">piGEM029</TH><TD>1</TD></TR><TR><TH scope="row">Primer iGEM078 (1:50)</TH><TD>6,25</TD></TR><TR><TH scope="row">Primer iGEM080 (1:50)</TH><TD>6,25</TD></TR><TR><TH scope="row">5x HF buffer</TH><TD>10</TD></TR><TR><TH scope="row">dNTP mix (10 mM each)</TH><TD>1</TD></TR><TR><TH scope="row">Phusion DNA polymerase 1:10</TH><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>24,5</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>30 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The Streptavidin was amplified from the plasmid piGEM028, the  StrepDARPidin plasmids, with an overhang to cup and an overhang with an <EM>Xho</EM>I restriction site.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix PCR</TH><TH scope="col">Single reaction (µl)</TH></TR><TR><TH scope="row">piGEM029</TH><TD>1</TD></TR><TR><TH scope="row">Primer iGEM077 (1:50)</TH><TD>6,25</TD></TR><TR><TH scope="row">Primer iGEM079 (1:50)</TH><TD>6,25</TD></TR><TR><TH scope="row">5x HF buffer</TH><TD>10</TD></TR><TR><TH scope="row">dNTP mix (10 mM each)</TH><TD>1</TD></TR><TR><TH scope="row">Phusion DNA polymerase 1:10</TH><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>24,5</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>40 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The amplification of cup yielded in 3 different bands from which the  middle, most severe band had the expected size (170 bp). The rest of the PCR  reaction was separated on a 1% agarose gel, the band in the correct size was  cut out and purified via a Gel Ex kit. Streptavidin with the size of 415 bp was  amplified correctly and the DNA fragment was purified via a Gel Ex kit.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.102">13.102 Ligation of piGEM002/007/008/009 with the IPTG inducible promoter</A></LEGEND><DIV class="aim"><P>Aim: insertion of the promoter into the nose plasmids</P></DIV><DIV class="exp-content"><P>The annealed oligos  were not phosphorylated. Thus additional ATP and a polynucleotide kinase (PNK)  were given into the ligase reaction of the <EM>Nco</EM>I  and <EM>Sac</EM>I digested plasmids  piGEM002, piGEM007, piGEM008 and piGEM009 with the annealed oligos  iGEM071/iGEM072 that form the IPTG inducible promoter. The correct relations  between vector and insert were calculated according to the formula, where V(T) is the  total volume of added DNA, L(P) is the length of the plasmid, L(I) is the  length of the insert, C(x) for the concentration of the plasmid and the insert  and R is the ratio between vector and insert, which was set as 3 in this case.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Ligation I (µl)</TH><TH scope="col">Ligation II (µl)</TH><TH scope="col">Ligation III (µl)</TH><TH scope="col">Ligation IV (µl)</TH></TR><TR><TH scope="row">piGEM-002 (281 ng/µl)</TH><TD>7,9</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">piGEM007 (ca 30  ng/µl)</TH><TD>-</TD><TD>9,7</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">piGEM008 (ca 30  ng/µl)</TH><TD>-</TD><TD>-</TD><TD>9,2</TD><TD>-</TD></TR><TR><TH scope="row">piGEM009 (ca 30 ng/µl)</TH><TD>-</TD><TD> </TD><TD>-</TD><TD>9,5</TD></TR><TR><TH scope="row">IPTG inducible promoter (iGEM071/ iGEM072) (54.7  ng/µl)</TH><TD>2,1</TD><TD>0,3</TD><TD>0,3</TD><TD>0,3</TD></TR><TR><TH scope="row">ATP (10mM)</TH><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">PNK</TH><TD>0,5</TD><TD>0,5</TD><TD>0,5</TD><TD>0,5</TD></TR><TR><TH scope="row">T4 Ligase</TH><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Ligation Buffer 10x</TH><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR><TH scope="row">Water</TH><TD>5,5</TD><TD>5,5</TD><TD>5,5</TD><TD>5,5</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>20</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><P>The reaction was incubated for  one hour at room temperature and subsequently used for the transformation of  competent <EM>E. coli</EM> XLIBlue. These cells were plated out on  LB-ampicillin and incubated at 37°C over night.</P></DIV></FIELDSET><FIELDSET class="exp22"><LEGEND><A name="exp22.7">22.17 amplification of KSI with amyE flankII and overhang to the IPTG inducible promoter</A></LEGEND><DIV class="aim"><P>Aim: create an overhang in front of the antiholin gene for the modified lac promoter</P></DIV><DIV class="exp-content"><P>To introduce the IPTG inducible promoter into the killswitch module I it had to be amplified again with a new primer, which created an overhang of the antiholin gene to the new promoter. The already built KSI with the too short promoter was used as a template for the PCR.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix PCR (µl)</TH><TH scope="col">Single reaction (µl)</TH></TR><TR><TH scope="row">module KSI (amyEI-antiholin-amEII)</TH><TD>1</TD></TR><TR><TH scope="row">Primer iGEM037 (1:50)</TH><TD>6,25</TD></TR><TR><TH scope="row">Primer iGEM075 (1:50)</TH><TD>6,25</TD></TR><TR><TH scope="row">5x HF buffer</TH><TD>10</TD></TR><TR><TH scope="row">dNTP mix (10 mM each)</TH><TD>1</TD></TR><TR><TH scope="row">Phusion DNA polymerase 1:10</TH><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>24,5</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>90 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The expected band in the size of ca 1350 bp was visible. The whole reaction was separated on a 1% agarose gel, the right band was cut out and the fragment was purified with a Gel Ex kit.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="02.10.2014">02.10.2014</A></H2><FIELDSET class="exp21"><LEGEND><A name="exp21.6">21.6 new Strep-Bead pulldown</A></LEGEND><DIV class="aim"><P>Aim: Checking interactions between Hag-(D2-)Strep and Streptavidin beads</P></DIV><DIV class="exp-content"><P>The purified Hag-D2-Strep which was left from crystallization and the purified Hag-Strep were used to make a pulldown with Streptavidin beads. We wanted to see if the Strep-Tag in the flagellin monomers can interact with the Streptavidin beads.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col"> </TH><TH scope="col">A280</TH><TH scope="col">MW [kDA]</TH><TH scope="col">Ex.coeff.</TH><TH scope="col">Conc. [mM]</TH></TR><TR><TH scope="row">FliS/ Hag-D2-Strep</TH><TD>28,8</TD><TD>60</TD><TD>23000</TD><TD>0,9 (0,9 nmol/µL)</TD></TR><TR><TH scope="row">FliS/ Hag-Strep</TH><TD>206</TD><TD>47,7</TD><TD>17420</TD><TD>1,18 (1,18 nmol/ µL)</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col"> </TH><TH scope="col">1</TH><TH scope="col">2</TH><TH scope="col">3</TH><TH scope="col">4</TH><TH scope="col">5</TH><TH scope="col">used</TH></TR><TR><TH scope="row">GST-bead</TH><TD>x</TD><TD> </TD><TD> </TD><TD>x</TD><TD>x</TD><TD>20 µL slurry</TD></TR><TR><TH scope="row">Strep-bead</TH><TD> </TD><TD>x</TD><TD>x</TD><TD> </TD><TD> </TD><TD>30 µL slurry</TD></TR><TR><TH scope="row">Hag-D2-Strep</TH><TD>x</TD><TD> </TD><TD>x</TD><TD> </TD><TD>x</TD><TD>4 nmol → 4,5 µL</TD></TR><TR><TH scope="row">Hag-Strep</TH><TD> </TD><TD>x</TD><TD> </TD><TD>x</TD><TD> </TD><TD>4 nmol → 3,4 µL</TD></TR></TBODY></TABLE><UL class="list"><LI>Pd-column + bead-slurry/ 1 min. 4000 rpm, RT</LI><LI>Add 400 µL GeFi(old) - wash/ 1 min. 4000 rpm, RT</LI><LI>Discard Flow through, add 400 µL GeFi(old) + Protein → 20 min on Turning wheel at RT</LI><LI>1 min at 4000 rpm, RT</LI><LI>2 x 400 µL &quot;Strep-wash&quot; </LI><LI>Elution with 40 µL 'Strep-Elution'</LI><LI>5 min. incubation at RT</LI><LI>1 min. 4000 rpm at RT, Elution in 10 µL Loading Buffer in 1,5 mL Eppi</LI><LI>Remaining on filter resuspended in 40 µL water + 10 µL SDS loading buffer
      ( 5µL  unstained protein ladder was used)</LI></UL><TABLE width="100%" border="1"><TBODY><TR><TH scope="col"> </TH><TH scope="col">Sample</TH></TR><TR><TH scope="row">1</TH><TD>Strep-beads without protein</TD></TR><TR><TH scope="row">2</TH><TD>Strep-beads + Hag-Strep</TD></TR><TR><TH scope="row">3</TH><TD>Strep-beads + Hag-D2-Strep</TD></TR><TR><TH scope="row">4</TH><TD>GST-beads + Hag-Strep</TD></TR><TR><TH scope="row">5</TH><TD>GST-beads + Hag-D2-Strep</TD></TR><TR><TH scope="row">6</TH><TD>Strep-beads + Hag-Strep</TD></TR><TR><TH scope="row">7</TH><TD>Strep-beads + Hag-D2-Strep</TD></TR><TR><TH scope="row">8</TH><TD>GST-beads + Hag-Strep</TD></TR><TR><TH scope="row">9</TH><TD>GST-beads + Hag-D2-Strep</TD></TR></TBODY></TABLE><P>The  Strep-pulldown showed us that the Strep-Tag interacts fine with the  Streptavidin-beads. The next step would be to make a competitive pulldown to  test the interaction between Hag-(D2-)Strep and StrepDARPidin. </P></DIV></FIELDSET><FIELDSET class="exp23"><LEGEND><A name="exp23.37">23.37 Addition of the first two biobricks iGEM Team Marburg to the registry</A></LEGEND><DIV class="exp-content"><P>Added two bricks to the registry to get numbers for the shipment of the samples.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Part  Number</TH><TH scope="col">Description</TH></TR><TR><TH scope="row">Part:BBa_K1329000</TH><TD>StrepDARPidin</TD></TR><TR><TH scope="row">Part:BBa_K1329001</TH><TD>Hag-KpnI</TD></TR></TBODY></TABLE><P>Fill 20µl in PCR Cup, add lab  tape with Biobrick number, wrap in parafilm, place in 50 ml falcon.</P></DIV></FIELDSET><FIELDSET class="exp23"><LEGEND><A name="exp23.38">23.38 Sequencing results of 1C3-Hag-KpnI-Cup-1</A></LEGEND><DIV class="exp-content"><P>Sequencing of 1C3-Hag-KpnI-Cup-1 clone 3 is positive, can also be sent to the registry.</P></DIV></FIELDSET><FIELDSET class="exp23"><LEGEND><A name="exp23.39">23.39 Analyze successful Gibson assembly of Hag-D2-Strep clone 1 and 2 (mutagenesis)</A></LEGEND><DIV class="exp-content"><P>Inoculate 16 clones from the Gibson assembly (1.10.) in Lb-Cm for plasmid preparation and test restriction.</P><P>Small colonies on the Gibson  transformation plates are visible in the morning; further incubation until  colonies can be inoculated in LB-Cm. Plasmids will then be isolated from the  cultures and digested with <I>Pst</I>I in order to analyze the removal of the <I>Pst</I>I  restriction site in the insert.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.103">13.103 Screening of the transformands from the ligation of the nose plasmids with the IPTG inducible promoter</A></LEGEND><DIV class="aim"><P>Aim: screening for positive clones with the finished plasmids</P></DIV><DIV class="exp-content"><P>The only plate that was covered with transformands was XLIBlue piGEM002+lac. The ligation with piGEM007+lac yielded in no transformands at all, piGEM008+lac showed two colonies and piGEM009+lac only one. The low number of transformands probably correlated with the low amount of vector used for the ligation. For the further ligations the vectors were used directly after the restriction and heat inactivation of the enzymes, without further purification with a Gel Ex kit. </P><P>A  colony PCR was carried out.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix colony PCR</TH><TH scope="col">Single reaction</TH></TR><TR><TH scope="row">Template</TH><TD>part of a colony</TD></TR><TR><TH scope="row">Primer iGEM006 (1:10)</TH><TD>1</TD></TR><TR><TH scope="row">Primer iGEM071 (1:10)</TH><TD>1</TD></TR><TR><TH scope="row">5x HF buffer</TH><TD>4</TD></TR><TR><TH scope="row">dNTP mix (10 mM each)</TH><TD>0,5</TD></TR><TR><TH scope="row">Phusion DNA polymerase 1:10</TH><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>12,5</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>10  min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1 min</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The colony PCR for positive clones showed no  results. It was repeated for piGEM002+lac and nine further clones were picked.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix colony PCR</TH><TH scope="col">Single reaction</TH></TR><TR><TH scope="row">Template</TH><TD>part of a colomy</TD></TR><TR><TH scope="row">Primer iGEM006 (1:50)</TH><TD>3,125</TD></TR><TR><TH scope="row">Primer iGEM059 (1:50)</TH><TD>3,125</TD></TR><TR><TH scope="row">5x HF buffer</TH><TD>4</TD></TR><TR><TH scope="row">dNTP mix (10 mM each)</TH><TD>0,3</TD></TR><TR><TH scope="row">Phusion DNA polymerase 1:10</TH><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>8,45</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>10 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1 min</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>This  time bands of the expected size (745 bp) could be noticed. Clone 14 was picked  and restreaked on a LB-Amp plate.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.1">13.104 amplification of the strong constitutive promoter for Gram-positives from gDNA of <EM>Bacillus cereus</EM></A></LEGEND><DIV class="aim"><P>Aim: amplification of the constitutive promoter</P></DIV><DIV class="exp-content"><P><EM>Bacillus cereus</EM> gDNA  was received from ZIEL, TU München. Based on this genomic DNA the promoter  should be amplified with primers that lead to overhangs containing a <EM>Sac</EM>I and a <EM>Nco</EM>I restriction site.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix PCR</TH><TH scope="col">Single reaction (µl)</TH></TR><TR><TH scope="row">gDNA <EM>Bacillus  cereus</EM> (1:10)</TH><TD>1</TD></TR><TR><TH scope="row">Primer iGEM073 (1:50)</TH><TD>6,25</TD></TR><TR><TH scope="row">Primer iGEM074 (1:50)</TH><TD>6,25</TD></TR><TR><TH scope="row">5x HF buffer</TH><TD>10</TD></TR><TR><TH scope="row">dNTP mix (10 mM each)</TH><TD>0,5</TD></TR><TR><TH scope="row">Phusion DNA polymerase 1:10</TH><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>25</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>30 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>3 min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The amplification of the constitutive promoter worked (gel photo at 13.103). The fragment was in the size between 200 and 300 bp like expected (260 bp).</P></DIV></FIELDSET><FIELDSET class="exp24"><LEGEND><A name="exp24.2">24.2 Gibson assembly with Streptavidin and cup</A></LEGEND><DIV class="aim"><P>Aim: Fusion of the genes of streptavidin and cup</P></DIV><DIV class="exp-content"><P>In order to fuse the genes for streptavidin and cup, which both had overhangs for each other, a Gibson assembly was carried out. Two ratios of DNA addition were tested: equal volumes of streptavidin and cup were added to the Gibson mix and the other ratio was calculated according to the formula in 13.102 with R = 1. A third Gibson reaction was prepared with the digested vector pET16b, since it was thought, the primers for strep-cup were designed with overhangs to this vector. Although the vector maps show only the overhangs with restriction sites for <I>Nco</I>I and <I>Sac</I>I the reaction was prepared with the vector as well.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">equal amounts volume  (µL)</TH><TH scope="col">equal amounts DNA  (µl)</TH><TH scope="col">with vector (µl)</TH></TR><TR><TH scope="row">streptavidin (49.5  ng/µl)</TH><TD>2,5</TD><TD>1,17</TD><TD>0,5</TD></TR><TR><TH scope="row">cup (4.9 ng/µl)</TH><TD>2,5</TD><TD>3,83</TD><TD>1,5</TD></TR><TR><TH scope="row">pET16b <EM>Nco</EM>I/<EM>Sac</EM>I</TH><TD>-</TD><TD>-</TD><TD>3</TD></TR><TR><TH scope="row">Gibson mix</TH><TD>15</TD><TD>15</TD><TD>-</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>20</TD><TD>-</TD></TR></TBODY></TABLE><P>The Gibson assembly was separated on a 1 % agarose gel and in addition to the bands of the single parts streptavidin (cup went out of the gel due to its small size) a very light band in the right size of 561 bp was visible. Since the yield of correct product was so low, the Gibson reaction was repeated double, separated on an agarose gel, cut out and purified via Gel Extraction.</P></DIV></FIELDSET><FIELDSET class="exp25"><LEGEND><A name="exp25.5">25.5 new Fluorometer assay with A549</A></LEGEND><DIV class="aim"><P>Aim: Checking interaction of StrepDARPidin with EpCAM on A549 Lung carcinoma cells using a new serial dilution</P></DIV><DIV class="exp-content"><P>In order to check the interaction of StrepDARPidin with the EpCAM on A549 Lung Carcinoma cells we planned an ELISA-Like assay again. Yesterday row F 1-12 of a black Fluotrac600 96-well plate was coated with 100000 cells A549/ well (counted with Neubauer-Chamber) overnight at 37°C. 100000 cells of 3T3 fibroblasts were used as negative control and as well used to coat well G6.</P><P>We intended to incubate them with different concentrations of  StrepDARPidin and then target the N-terminal His-Tag with and Anti-His antibody  Alexa488 conjugated (1:50 endvolumen). </P><P>In wells H1-6 gel filtration purified StrepDARPidin with 87 µM was  tested, in well H 7-12 Ni-NTA purified StrepDARPidin with 280 µM.</P><P>Following concentrations of StrepDARPidin dissolved in PBS (pH 7,4) +2,5  % glycerin were used:</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Well
Col</TH><TH scope="col">Well Row</TH><TH scope="col">raw data (485, 520)</TH><TH scope="col">Purification</TH><TH scope="col">StrepDARPidin  concentration [M]</TH></TR><TR><TH scope="row">A549</TH><TD>1</TD><TD>F</TD><TD>5782</TD><TD>GeFi</TD><TD>50 µM</TD></TR><TR><TH scope="row">A549</TH><TD>2</TD><TD>F</TD><TD>4912</TD><TD>GeFi</TD><TD>25 µM</TD></TR><TR><TH scope="row">A549</TH><TD>3</TD><TD>F</TD><TD>3903</TD><TD>GeFi</TD><TD>2,5 µM</TD></TR><TR><TH scope="row">A549</TH><TD>4</TD><TD>F</TD><TD>3513</TD><TD>GeFi</TD><TD>250 nM</TD></TR><TR><TH scope="row">A549</TH><TD>5</TD><TD>F</TD><TD>3613</TD><TD>GeFi</TD><TD>25 nM</TD></TR><TR><TH scope="row">A549</TH><TD>6</TD><TD>F</TD><TD>3211</TD><TD>GeFi</TD><TD>2,5 nM</TD></TR><TR><TH scope="row">A549</TH><TD>7</TD><TD>F</TD><TD>7340</TD><TD>Ni-NTA</TD><TD>50 µM</TD></TR><TR><TH scope="row">A549</TH><TD>8</TD><TD>F</TD><TD>5329</TD><TD>Ni-NTA</TD><TD>25 µM</TD></TR><TR><TH scope="row">A549</TH><TD>9</TD><TD>F</TD><TD>5846</TD><TD>Ni-NTA</TD><TD>2,5 µM</TD></TR><TR><TH scope="row">A549</TH><TD>10</TD><TD>F</TD><TD>5718</TD><TD>Ni-NTA</TD><TD>250 nM</TD></TR><TR><TH scope="row">A549</TH><TD>11</TD><TD>F</TD><TD>3852</TD><TD>Ni-NTA</TD><TD>25 nM</TD></TR><TR><TH scope="row">A549</TH><TD>12</TD><TD>F</TD><TD>6828</TD><TD>Ni-NTA</TD><TD>2,5 nM</TD></TR><TR><TH scope="row">Empty well</TH><TD>5</TD><TD>G</TD><TD>5699</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">negative control 3T3 fibroblasts</TH><TD>6</TD><TD>G</TD><TD>9983</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Blank</TH><TD>7</TD><TD>G</TD><TD>1760</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">positive control AB</TH><TD>8</TD><TD>G</TD><TD>60182</TD><TD>-</TD><TD>-</TD></TR></TBODY></TABLE><P>The 96-well plate was taken out of the incubator and the medium was  carefully discarded. 100 µL of the dilutions were transferred into the wells  and incubated for 45 min. at room temperature under the Flow hood.</P><P>An Anti-His-Alexa488 conjugate (1 mg/mL stock) with 5 µL concentrate was  dissolved in 40 µL  PBS. 1 µL of the  antibody was added after washing to each well and incubated for 20 min at room  temperature in the dark.</P><P>As a negative control t-cells clones were used and incubated the same  way with 1 µM StrepDARPidin and 1 µL antibody (Well G8). Well G5 was left  empty, well G7 was filled with 100 µL PBS+Glycerin and G4 with PBS+2,5 %  glycerin with 1 µL Antibody. As negative Bacillus Subtilis WT culture was used  which was already lysed causing the huge signal. A different EpCAM-negative  cell line has to be used.</P><P>After 20 min incubation the wells were washed carefully with 100 µL PBS  to wash out unbound elements.</P><P>The plate was measured in the ELISA reader:</P><P>We are able to see that the fluorescence signal decreases with  increasing dilution. Unfortunately the negative control was very high as well.  The t-cells were incubated in Eppis because of their non-adherent features.  They were incubated with StrepDARPidin and Antibody before. The supernatant was  discarded after centrifugation at 1500x g but the cells did not stay at the  pellet’s position so that it was not possible to remove the whole preincubated  material which might be left when measuring the control.</P><P>Additionally it is noticeable that there is the same trend for both  purification methods. The lower signal from NTA-purified StrepDARPidin can be  explained by the fact that the concentration was higher but the purity also  lower so that even less StrepDARPidin was found in the protein aliquods. As  negative Bacillus Subtilis WT culture was used which was already lysed causing  the huge signal. A different EpCAM-negative cell line has to be used. </P><P>For the next measurement row C of the plate was coated with 100000 A549 cells &amp; Caco-2 cells per well overnight and the same amount of 3T3 fibroblasts as negative control.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="03.10.2014">03.10.2014</A></H2><FIELDSET class="exp21"><LEGEND><A name="exp21.7">21.7 competitive Strep-Bead pulldown</A></LEGEND><DIV class="aim"><P>Aim: Checking interactions between Hag-(D2-)Strep and StrepDARPidin</P></DIV><DIV class="exp-content"><P>The purified Hag-D2-Strep which was left from crystallization and the purified Hag-Strep were used to make a pulldown with purified StrepDARPidin. We wanted to see if the Strep-Tag in the flagellin monomers can interact with the StrepDARPidin, especially with the Streptavidin part.</P><TABLE width="100%" border="1"><TBODY><TR><TD> </TD><TD><STRONG>A280</STRONG></TD><TD><STRONG>MW [kDA]</STRONG></TD><TD><STRONG>Ex.coeff.</STRONG></TD><TD><STRONG>Conc. [M]</STRONG></TD></TR><TR><TD><STRONG>FliS/ Hag-D2-Strep</STRONG></TD><TD>28,8</TD><TD>60</TD><TD>23000</TD><TD>0,9 mM(0,9 nmol/µL)</TD></TR><TR><TD><STRONG>FliS/ Hag-Strep</STRONG></TD><TD>206</TD><TD>47,7</TD><TD>17420</TD><TD>1,18 Mm (1,18 nmol/ µL)</TD></TR><TR><TD><STRONG>StrepDARPidin</STRONG></TD><TD>6,061</TD><TD>31,8</TD><TD>57410</TD><TD>105,5 µM</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TD> </TD><TD><STRONG>1</STRONG></TD><TD><STRONG>2</STRONG></TD><TD><STRONG>3</STRONG></TD><TD><STRONG>4</STRONG></TD><TD><STRONG>5</STRONG></TD><TD><STRONG>used</STRONG></TD></TR><TR><TD><STRONG>StrepDARPidin</STRONG></TD><TD> </TD><TD> </TD><TD>X</TD><TD>X</TD><TD>X</TD><TD>4 nmol → 40 µl</TD></TR><TR><TD><STRONG>Hag-D2-Strep</STRONG></TD><TD> </TD><TD>X</TD><TD> </TD><TD>X</TD><TD> </TD><TD> 2 nmol → 2,25 µl</TD></TR><TR><TD><STRONG>Hag-Strep</STRONG></TD><TD>X</TD><TD> </TD><TD>X</TD><TD> </TD><TD> </TD><TD>2 nmol → 1,7 µl</TD></TR></TBODY></TABLE><UL class="list"><LI>40 µL (105,5 µM) StrepDARPidin + Hag-(D2-)Strep à PRECIPITATES!</LI><LI>Hag-(D2-)Strep &amp; StrepDARPidin in 1xPBS+2,5%  glycerine stable.</LI><LI>15 min, 13000 RPM-1min</LI><LI>Supernatant 1:2 and 1:10 on SDS-PAGE, samples heated 5  min at 95°C</LI><LI>Pellet resuspended in 10 µL LB</LI></UL><TABLE width="100%" border="1"><TBODY><TR><TD> </TD><TD><STRONG>Sample</STRONG></TD></TR><TR><TD><STRONG>1</STRONG></TD><TD>Hag-Strep</TD></TR><TR><TD><STRONG>2</STRONG></TD><TD>Hag D2-Strep</TD></TR><TR><TD><STRONG>3</STRONG></TD><TD>Hag Strep + StrepDARPidin</TD></TR><TR><TD><STRONG>4</STRONG></TD><TD>Hag-D2-Strep + StrepDARPidin</TD></TR><TR><TD><STRONG>5</STRONG></TD><TD>StrepDARPidin</TD></TR><TR><TD><STRONG>6</STRONG></TD><TD>Hag-Strep + StrepDARPidin [1:5]</TD></TR><TR><TD><STRONG>7</STRONG></TD><TD>Hag-D2-Strep + StrepDARPidin [1:5]</TD></TR><TR><TD><STRONG>8</STRONG></TD><TD>Hag-Strep + StrepDARPidin [1:1]</TD></TR><TR><TD><STRONG>9</STRONG></TD><TD>Hag-D2-Strep + StrepDARPidin [1:1]</TD></TR></TBODY></TABLE><P>The competitive pulldown shows that just a small amount of protein remains in the supernatant. Hag-D2-Strep seems to be more stable than Hag-Strep. The highest amount of protein precipitates. We have assumed that the Strep-Tag/ Streptavidin-binding is very strong and causes maybe misfolding of the flagellin after binding which causes the precipitation. We plan to isolate the flagella in order to see if the same reaction takes place.</P></DIV></FIELDSET><FIELDSET class="exp21"><LEGEND><A name="exp21.8">21.8 Western blot with anti-Strep-Tag-AB</A></LEGEND><DIV class="aim"><P>Aim: Checking interactions between Hag-D2-Strep &amp; Anti-Strep-Tag-AB</P></DIV><DIV class="exp-content"><P>To 2ml of culture of Bacillus subtilis (OD=1.01) 120µl H20 and 80 µl were added for lysation. 10 µl of the lysate  supplied with Roti-Load (Roth, K 929.1) were loaded on a 12% polyacrylamide gel. The electrophoresis ran at max. 130 V. After finishing, the gel was blotted onto PVDF membrane (6 cm * 9 cm) at 4°C at 250mA for about 1.5 hours. Then, the membrane was washed in BSA solution (3% BSA in TBS/Tween-20, Roth, 9127.1) over night at 4°C, before it was treated with the strep-antibody (Precision Protein StrepTactin-HRP Conjugate (Bio Rad, #161-0381): 1:5000) for 1 hr at RT. The membrane was washed 3 times for 10 minutes in TBS/Tween-20. The membrane was rinsed for 1 minute with ECL solution with a chemiluminescent substrate, which was detected by a Fusion Fx Vilber Lourmat imaging system.</P><P>It could be seen that in comparison to the wildtype the Hag-D2-Strep strain gave a positive signal for the StrepTag, which was incorporated into the flagellin.</P></DIV></FIELDSET><FIELDSET class="exp23"><LEGEND><A name="exp23.40">23.40 Plasmid isolation and test restriction of Hag-D2-Strep clone 1 and 2 (mutagenesis)</A></LEGEND><DIV class="aim"><P>Aim: Analyze the removal of the <I>Pst</I>I restriction site </P></DIV><DIV class="exp-content"><P>Plasmid preparation of the inoculated E. coli cultures was performed after over night incubation. 500-600 ng of all plasmids have been digested with <I>Pst</I>I for 2h (37°C). In case the <I>Pst</I>I restriction site is removed from the insert, only one fragment (linearized plasmid) should occur in the gel. If the mutagenesis PCR failed and the restriction site is still inside the plasmid 2 fragments (3000bp + 470 bp) should occur in the gel.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col"> </TH><TH scope="col">1x</TH><TH scope="col">18x</TH></TR><TR><TH scope="row">Water</TH><TD>14,5</TD><TD>261</TD></TR><TR><TH scope="row">CutSmart  Buffer (10x)</TH><TD>2</TD><TD>36</TD></TR><TR><TH scope="row"><I>Pst</I>I</TH><TD>0,5</TD><TD>9</TD></TR><TR><TH scope="row">Plasmid</TH><TD>3</TD><TD> </TD></TR></TBODY></TABLE><P>The result of the restriction was analyzed on a 1% agarose gel:</P><P>Negative clones: 1.2, 1.4, 2.3 and 2.9</P><P>Only one fragment should  occur in the gel in case of <I>Pst</I>I linearization.</P></DIV></FIELDSET><FIELDSET class="exp24"><LEGEND><A name="exp24.3">24.3 amplification of StrepCup from the purified Gibson fragment</A></LEGEND><DIV class="aim"><P>Aim: increase the amount of StrepCup to work on</P></DIV><DIV class="exp-content"><P>The purified Gibson assembly product was used as a template for further amplification of the StrepCup construct.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix PCR</TH><TH scope="col">Single reaction (µl)</TH></TR><TR><TH scope="row">StrepCup (1.5 ng/µl)</TH><TD>3</TD></TR><TR><TH scope="row">Primer iGEM077 (1:50)</TH><TD>6,25</TD></TR><TR><TH scope="row">Primer iGEM080 (1:50)</TH><TD>6,25</TD></TR><TR><TH scope="row">5x HF buffer</TH><TD>10</TD></TR><TR><TH scope="row">dNTP mix (10 mM each)</TH><TD>0,5</TD></TR><TR><TH scope="row">Phusion DNA polymerase 1:10</TH><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>23</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>50 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>In addition to the expected band at 541 bp a smaller on was visible. The fragments of the right size were cut out, pooled and purified with a Gel Ex kit.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.105">13.105 repeated restriction of the nose plasmids and ligation with the lac promoter and the constitutive promoter</A></LEGEND><DIV class="aim"><P>Aim: insert the different promoters into the plasmids for further characterization</P></DIV><DIV class="exp-content"><P>Since the last ligation yielded in barely any colonies on the plates, the vectors were not further purified after restriction with <I>Nco</I>I and <I>Sac</I>I, which would result in a high loss of DNA. The ligation of piGEM007, piGEM008, piGEM009 with the IPTG inducible and the strong constitutive promoter and piGEM002 with the strong constitutive promoter was carried out. The DNA ratios were calculated according to the previously used formula (13.102).</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Ligation I (µl)</TH><TH scope="col">Ligation II (µl)</TH><TH scope="col">Ligation III (µl)</TH><TH scope="col">Ligation IV (µl)</TH><TH scope="col">Ligation V (µl)</TH><TH scope="col">Ligation VI (µl)</TH><TH scope="col">Ligation VII (µl)</TH></TR><TR><TH scope="row">piGEM-002 (281 ng/µl)</TH><TD>6,577</TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TH scope="row">piGEM007 (ca 100  ng/µl)</TH><TD> </TD><TD>7,6</TD><TD> </TD><TD> </TD><TD>8,6</TD><TD> </TD><TD> </TD></TR><TR><TH scope="row">piGEM008 (ca 100  ng/µl)</TH><TD> </TD><TD> </TD><TD>7,16</TD><TD> </TD><TD> </TD><TD>8,3</TD><TD> </TD></TR><TR><TH scope="row">piGEM009 (ca 100 ng/µl)</TH><TD> </TD><TD> </TD><TD> </TD><TD>6,76</TD><TD> </TD><TD> </TD><TD>8</TD></TR><TR><TH scope="row">IPTG inducible promoter (iGEM071/ iGEM072) (54.7  ng/µl)</TH><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD>1,4</TD><TD>1,7</TD><TD>2</TD></TR><TR><TH scope="row">constitutive promoter (63.2 ng/µl)</TH><TD>3,43</TD><TD>2,4</TD><TD>2,84</TD><TD>3,24</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TH scope="row">ATP (10mM)</TH><TD>0,5</TD><TD>0,5</TD><TD>0,5</TD><TD>0,5</TD><TD>0,5</TD><TD>0,5</TD><TD>0,5</TD></TR><TR><TH scope="row">PNK</TH><TD>0,5</TD><TD>0,5</TD><TD>0,5</TD><TD>0,5</TD><TD>0,5</TD><TD>0,5</TD><TD>0,5</TD></TR><TR><TH scope="row">T4 Ligase</TH><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Ligation Buffer 10x</TH><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR><TH scope="row">Water</TH><TD>6</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>6</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>20</TD><TD>20</TD><TD>20</TD><TD>20</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><P>The  reactions were incubated at room temperature for one hour and subsequently used  to transform <EM>E. coli</EM> XLIBlue cells  that were plated out on LB-ampicillin plates and incubated at 37°C.</P></DIV></FIELDSET></DIV><H2 class="title"><A name="04.10.2014">04.10.2014</A></H2><FIELDSET class="exp22"><LEGEND><A name="exp22.7">22.17 amplification of amyE-flankI with overhang to the IPTG inducible promoter</A></LEGEND><DIV class="aim"><P>Aim: introduce a compatible overhang for fusion PCR/Gibson</P></DIV><DIV class="exp-content"><P>In order to fuse the parts of KSI together, analogue overhangs on each part were needed. Thus, amyE flank I was amplified anew with the new primers and the old flank as template.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix PCR</TH><TH scope="col">Single reaction (µl)</TH></TR><TR><TH scope="row">amyE flank I </TH><TD>1</TD></TR><TR><TH scope="row">Primer iGEM034 (1:50)</TH><TD>6,25</TD></TR><TR><TH scope="row">Primer iGEM076 (1:50)</TH><TD>6,25</TD></TR><TR><TH scope="row">5x HF buffer</TH><TD>10</TD></TR><TR><TH scope="row">dNTP mix (10 mM each)</TH><TD>0,5</TD></TR><TR><TH scope="row">Phusion DNA polymerase 1:10</TH><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>25</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1 min</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The amyE flank with the overhang to the promoter was amplified successfully. The reaction mix was purified with an Omega Gel Extraction kit.</P></DIV></FIELDSET><FIELDSET class="exp22"><LEGEND><A name="exp22.18">22.18 FusionPCR/Gibson of amyE-flank I, lac promoter and antiholin-amyE-flank II</A></LEGEND><DIV class="aim"><P>Aim: Fusion of the single parts to create KS module I</P></DIV><DIV class="exp-content"><P>The parts were ready for the assembly. The same FusionPCR/Gibson assembly mixture was used like the last time to build KSI (22.10). The primers were added to the reaction after a first step of incubation at 50°C for 0.5 hours. Then the PCR was carried out as usual.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix PCR</TH><TH scope="col">Single reaction (µl)</TH></TR><TR><TH scope="row">amyE flank I (ca 10 ng/µl)</TH><TD>1</TD></TR><TR><TH scope="row">IPTG inducible promoter (iGEM071/072; ca 10 ng/µl)</TH><TD>1</TD></TR><TR><TH scope="row">antiolin-amyE flank II (10 ng/µl)</TH><TD>1</TD></TR><TR><TH scope="row">Primer iGEM034 (1:50)</TH><TD>6,25</TD></TR><TR><TH scope="row">Primer iGEM076 (1:50)</TH><TD>6,25</TD></TR><TR><TH scope="row">5x HF buffer</TH><TD>10</TD></TR><TR><TH scope="row">dNTP mix (10 mM each)</TH><TD>0,7</TD></TR><TR><TH scope="row">Phusion DNA polymerase 1:10</TH><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>22,8</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>2 min 10 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The fusion of the parts with this method did not work. Instead of a  distinct band, a smear over the whole lane was visible.</P></DIV></FIELDSET><FIELDSET class="exp24"><LEGEND><A name="exp24.4">24.4 Restriction of StrepCup and Ligation with pET16b</A></LEGEND><DIV class="aim"><P>Aim: insert the StrepCup construct in the overproduction plasmid pET16b</P></DIV><DIV class="exp-content"><P>The purified StrepCup fragment was digested with the enzymes <I>Nco</I>I and <I>Xho</I>I, since the Gibson assembly of Strep and Cup with the vector yielded in only one transformand, which was considered as a spontaneous mutation in relation to the control plate, which carried one colony more.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Attempt I (µL)</TH></TR><TR><TH scope="row">StrepCup 170 ng/µl)</TH><TD>15</TD></TR><TR><TH scope="row"><I>Nco</I>I</TH><TD>0,5</TD></TR><TR><TH scope="row">XhoI</TH><TD>0,5</TD></TR><TR><TH scope="row">Cutsmart Buffer 10x</TH><TD>2</TD></TR><TR><TH scope="row">Water</TH><TD>2</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD></TR></TBODY></TABLE><P>The reaction was incubated for one hour at 37°C and inactivated at 85°C  for 20 minutes afterwards.</P><P>The ratio of DNA addition to the ligation was again calculated according  to the formula from 13.102.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Ligation I (µl)</TH></TR><TR><TH scope="row">StrepCup <I>Nco</I>I/<I>Xho</I>I  (170.8 ng/µl)</TH><TD>0,5</TD></TR><TR><TH scope="row">pET16b <I>Nco</I>I/<I>Xho</I>I  (18.1 ng/µl)</TH><TD>16</TD></TR><TR><TH scope="row">ATP (10mM)</TH><TD>0,5</TD></TR><TR><TH scope="row">PNK</TH><TD>0,5</TD></TR><TR><TH scope="row">T4 Ligase</TH><TD>0,5</TD></TR><TR><TH scope="row">Ligation Buffer 10x</TH><TD>2</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD></TR></TBODY></TABLE><P>The reaction was incubated at room temperature for 1 h and was afterwards  completely used to transform cells of <EM>E.  coli</EM> XLIBlue, which were plated out on LB-ampicillin.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.106">13.106 Screening of transformands after ligation of nose plasmids with the lac promoter and the constitutive promoter</A></LEGEND><DIV class="aim"><P>Aim: check for correct plasmids</P></DIV><DIV class="exp-content"><P>Colony PCRs were performed to screen the clones for the right plasmids. Every plate carried enough colonies to pick multiple clones. Five clones of every plate was picked and this time instead of using the common gfp-reverse primer iGEM006 the primers specific for the degradation tags of the different nose plasmids were used. The ligation with the constitutive promoter should be repeated, since the fragment was not digested with <I>Nco</I>I and <I>Sac</I>I before.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix PCR</TH><TH scope="col">02+const</TH><TH scope="col">07+const</TH><TH scope="col">08+const</TH><TH scope="col">09+const</TH><TH scope="col">07+lac</TH><TH scope="col">08+lac</TH><TH scope="col">09+lac</TH></TR><TR><TH scope="row">Template</TH><TD>colony</TD><TD>colony</TD><TD>colony</TD><TD>colony</TD><TD>colony</TD><TD>colony</TD><TD>colony</TD></TR><TR><TH scope="row">iGEM073 (1:50)</TH><TD>3,125</TD><TD>3,125</TD><TD>3,125</TD><TD>3,125</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TH scope="row">iGEM059 (1:50)</TH><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD>3,125</TD><TD>3,125</TD><TD>3,125</TD></TR><TR><TH scope="row">iGEM006 (1:50)</TH><TD>3,125</TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TH scope="row">iGEM021 (1:50)</TH><TD> </TD><TD>3,125</TD><TD> </TD><TD> </TD><TD>3,125</TD><TD> </TD><TD> </TD></TR><TR><TH scope="row">iGEM022 (1:50)</TH><TD> </TD><TD> </TD><TD>3,125</TD><TD> </TD><TD> </TD><TD>3,125</TD><TD> </TD></TR><TR><TH scope="row">iGEM023 (1:50)</TH><TD> </TD><TD> </TD><TD> </TD><TD>3,125</TD><TD> </TD><TD> </TD><TD>3,125</TD></TR><TR><TH scope="row">5x HF buffer</TH><TD>4</TD><TD>4</TD><TD>4</TD><TD>4</TD><TD>4</TD><TD>4</TD><TD>4</TD></TR><TR><TH scope="row">dNTP mix (10 mM each)</TH><TD>0,3</TD><TD>0,3</TD><TD>0,3</TD><TD>0,3</TD><TD>0,3</TD><TD>0,3</TD><TD>0,3</TD></TR><TR><TH scope="row">Phusion DNA polymerase 1:10</TH><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>8,45</TD><TD>8,45</TD><TD>8,45</TD><TD>8,45</TD><TD>8,45</TD><TD>8,45</TD><TD>8,45</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>20</TD><TD>20</TD><TD>20</TD><TD>20</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><P>The ligation with the constitutive promoter had to be repeated, since it was not digested before and could not be ligated correctly. The  bands visible on the gel were all of the wrong size, the expected fragment  would have a size of 1039 bp. Theoretically, there should not be an amplificate  without insertion of the promoter but the primers probably bound randomly.  However, some clones with the lac promoter were positive (745 bp), even when  the bands were very light. Some clones additionally exhibited other fragments  which were probably the result of unspecific primer annealing. The normal  plasmids piGEM007, piGEM008 and piGEM009 were used as a positive control for  the reverse primers with the ssrA-tags.</P></DIV></FIELDSET><FIELDSET class="exp25"><LEGEND><A name="exp25.6a">25.6a Immunofluorescence microscopy  at AG Grosse lab</A></LEGEND><DIV class="aim"><P>Aim: Checking the interaction of StrepDARPidin with EpCAM-positive cell lines A549 &amp; Caco-2 via immunofluorescence Antibody staining</P></DIV><DIV class="exp-content"><P>We wanted to see if our StrepDARPidin construct binds to EpCAM localized in the cell membrane by staining the cells with an Anti-His-Antibody conjugated with Alexa488 after incubation with our StrepDARPidin which should bind to the N-terminal His-Tag of the StrepDARPidin. DAPI was used to stain the cell core and Phalloidin-Rhodamin to stain the membrane-associated F-actin.</P><P>Cover slips fitting into the wells of a 24-well plate were coated in  water with gelatin overnight at 4°C. A549 &amp; Caco-2 cells were cultured 2  days before in a small T25 culture flask in DMEM+10% /20% FCS + L-Glutamine  until full confluence was reached. </P><P>The cells were trypsinized and counted. The cover slips were incubated at  37°C for an hour with 100000 cells per cover slip in medium so that the cells  were able to attach to the slips. After an hour the medium was discarded and  the slips were washed with PBS several times and the cells observed in the  microscope.</P><P> 150 µL of a 50 µM StrepDARPidin  solution in PBS with 2,5% Glycerin (to keep the protein stable) were used to  cover the wells with A549 and Caco-2 cells for half an hour at room temperature.  As a negative control we used cells not incubated with the protein.</P><P>After discarding the protein solution and wash with PBS the cells were  fixed with 3,7% formaldehyde dissolved in PBS for 5 min at room temperature.</P><P>After washing PBS+5%BSA was used to block unspecific binding (20 min  +wash). The cells were permeabilized with Triton X-100 0,1% in PBS for a  minute.</P><P>For the staining an Penta-Anti-His-Alexa488 conjugate (QIAGEN) 1:25  (0,2mg/ mL) was used in combination with Phalloidin-Rhodamin (1:200, 300 units  stock) and 20 nM DAPI. After washing the slips they were mounted with DePex and  dried overnight in the fridge. The microscopy was done with Matthias Plessner  (AG Grosse).</P></DIV></FIELDSET><DIV class="notebooky-entry"><H2 class="title"><A name="05.10.2014">05.10.2014</A></H2><FIELDSET class="exp25"><LEGEND><A name="exp25.6b">25.6b Immunofluorescence microscopy  at AG Grosse lab</A></LEGEND><DIV class="aim"><P>Aim: Checking the interaction of StrepDARPidin with EpCAM-positive cell lines A549 &amp; Caco-2 via immunofluorescence Antibody staining</P></DIV><DIV class="exp-content"><P>The dried samples were analysed under a Laser scanning microscope Zeiss LSM Series.</P><P>The dried samples were analysed under a Laser scanning microscope Zeiss LSM Series.</P><P>The microscopy pictures show that there is a colocalization of StrepDARPidin and membrane-associated F-actin. The Anti-His-Alexa488 antibody (green) stains the StrepDARPidin incubated A540 and Caco-2 cells generating a high intensity fluorescence signal at the cell membrane. Unspecific His-staining was detected in the nucleus of all cell lines both in presence and absence of StrepDARPidin. The negative control 3T3 fibroblasts stay unstained at the membrane.</P><P>We were able to prove that the StrepDARPidin is binding specifically to EpCAM-positive cells.</P></DIV></FIELDSET><FIELDSET class="exp24"><LEGEND><A name="exp24.5">24.5 Screening of transformands from Gibson of StrepCup and pET16b</A></LEGEND><DIV class="aim"><P>Aim: check for positive clones</P></DIV><DIV class="exp-content"><P>Eight colonies were visible on the trafo plate. The clones were screened for the positive plasmid construct by a colony PCR.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix colony PCR</TH><TH scope="col">Single reaction</TH></TR><TR><TH scope="row">Template</TH><TD>part of a colony</TD></TR><TR><TH scope="row">Primer iGEM077 (1:50)</TH><TD>3,125</TD></TR><TR><TH scope="row">Primer iGEM080 (1:50)</TH><TD>3,125</TD></TR><TR><TH scope="row">5x HF buffer</TH><TD>4</TD></TR><TR><TH scope="row">dNTP mix (10 mM each)</TH><TD>0,3</TD></TR><TR><TH scope="row">Phusion DNA polymerase 1:10</TH><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>8,45</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>10 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>50 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>Fragments of the expected size (561 bp) could be seen for nearly all clones except clone 7 (see gel at 22.19). These clones were assumed to be positive. For further work clone 6 was chosen.</P></DIV></FIELDSET><FIELDSET class="exp22"><LEGEND><A name="exp22.19">22.19 FusionPCR of amyE-flank I, lac promoter and antiholin-amyE-flank II</A></LEGEND><DIV class="aim"><P>Aim: Fusion of the single parts to create KS module I</P></DIV><DIV class="exp-content"><P>This time, a simple fusion PCR was performed to fuse the separate parts of the killswitch module I together, the primers were included directly and the PCR reaction was started as usual.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix PCR</TH><TH scope="col">Single reaction (µl)</TH><TH scope="col">Single reaction with  overhang primers</TH></TR><TR><TH scope="row">amyE flank I (ca 10 ng/µl)</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">IPTG inducible promoter (iGEM071/072; ca 10 ng/µl)</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">antiholin-amyE flank II (10 ng/µl)</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Primer iGEM034 (1:50)</TH><TD>6,25</TD><TD>6,25</TD></TR><TR><TH scope="row">Primer iGEM037 (1:50)</TH><TD>6,25</TD><TD>6,25</TD></TR><TR><TH scope="row">Primer iGEM075 (1:50)</TH><TD>-</TD><TD>6,25</TD></TR><TR><TH scope="row">Primer iGEM076 (1:50)</TH><TD>-</TD><TD>6,25</TD></TR><TR><TH scope="row">5x HF buffer</TH><TD>10</TD><TD>10</TD></TR><TR><TH scope="row">dNTP mix (10 mM each)</TH><TD>0,7</TD><TD>0,7</TD></TR><TR><TH scope="row">Phusion DNA polymerase 1:10</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>22,8</TD><TD>22,8</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>2 min 10 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The fusion PCR of KSI did not  work. The reaction just containing the primer iGEM034 and 037, which anneal to  the outer sides of the amyE flanks yielded in no amplificate at all. The  addition of primers iGEM075 and 076 that are the overhangs between the separate  modules resulted in a band in the height of ca. 500 bp, which probably was the  amplificated amyE flank I_lac fragment.</P></DIV></FIELDSET><FIELDSET class="exp23"><LEGEND><A name="exp23.41">23.41 Repeat test restriction</A></LEGEND><DIV class="exp-content"><P>New test restriction with only 200 ng plasmid DNA since the last restriction showed a lot of undigested DNA. In order to test for the removal of the <I>Pst</I>I restriction site 8 clones will be digested with <I>Pst</I>I and <I>Pst</I>I/<I>Eco</I>RI.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col"> </TH><TH scope="col">1x</TH><TH scope="col">18x</TH></TR><TR><TH scope="row">Water</TH><TD>16,5</TD><TD>148,5 / 144,0</TD></TR><TR><TH scope="row">CutSmart  Buffer (10x)</TH><TD>2</TD><TD>18</TD></TR><TR><TH scope="row"><I>Pst</I>I</TH><TD>0,5</TD><TD>4,5</TD></TR><TR><TH scope="row">i&gt;EcoRI</TH><TD>0,5</TD><TD>4,5</TD></TR><TR><TH scope="row">Plasmid</TH><TD>1</TD><TD> </TD></TR></TBODY></TABLE><P><I>Pst</I>I restriction should lead to a linearized plasmid, i&gt;EcoRI/<I>Pst</I>I shows the expected size of 1500 and 2000 bp. Additionally some undigested plasmid is left in the <I>Pst</I>I restriction.</P><UL><LI>Contamination of plasmid? New digestion with new batch of <I>Pst</I>I and <I>Eco</I>RI.</LI><LI><I>Pst</I>I restriction lead to a linearized plasmid  and <I>Pst</I>I/<I>Eco</I>RI double digest lead to two fragments of a size of 1400 and 2000  bp</LI><LI>Due to limited time clones 1.1 and 2.1 will be  sent for sequencin</LI></UL><P>Result: <I>Pst</I>I is contaminated with i&gt;EcoRI.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="06.10.2014">06.10.2014</A></H2><FIELDSET class="exp13"><LEGEND><A name="exp13.107">13.107 colony PCR on transformed Bacillus subtilis PY79</A></LEGEND><DIV class="aim"><P>Aim: check for positive transformation</P></DIV><DIV class="exp-content"><P>On the plate with the transformed PY79 was a colony visible, which probably was tansformed with piGEM035 (nose plasmid without ssrA-tag and Cu sensitive promoter). The colony was restraked on LB-chloramphenicol and used for a colony PCR to check the correct transformation. For checking the insertion into the amyE locus, the colony was streaked onto LB-starch agar together with the wildtype.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix colony PCR</TH><TH scope="col">Single reaction</TH></TR><TR><TH scope="row">Colony in 100 µl PBS</TH><TD>5</TD></TR><TR><TH scope="row">Primer Flo 56 (1:50)</TH><TD>6,25</TD></TR><TR><TH scope="row">Primer Flo 54 (1:50)</TH><TD>6,25</TD></TR><TR><TH scope="row">5x HF buffer</TH><TD>10</TD></TR><TR><TH scope="row">dNTPs (40 mM)</TH><TD>0,5</TD></TR><TR><TH scope="row">Phusion DNA polymerase 1:10</TH><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>21</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1 min</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The wildtype PY79 was taken as a negative control, the PY79 that had already been transformed with piGEM034 as a positive control with the primer iGEM057 for the Ag promoter. Since not even the positive control was positive, the reaction should be defective. The PCR was repeated and the amount of colony taken to boil it in 1x PBS was slightly lower, so there was no turbidity visible in the PBS.</P></DIV></FIELDSET><FIELDSET class="exp23"><LEGEND><A name="exp23.42">23.42 Sequencing of 1C3 Hag-KpnI- D2-Strep</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Label  number</TH><TH scope="col">Plasmid</TH><TH scope="col">Primer  (Premix)</TH></TR><TR><TH scope="row">AGB0023-564</TH><TD>1C3  Hag-KpnI- D2-Strep 1.1</TD><TD>iGEM  53</TD></TR><TR><TH scope="row">AGB0023-565</TH><TD>1C3  Hag-KpnI- D2-Strep 1.1</TD><TD>iGEM  54</TD></TR><TR><TH scope="row">AGB0023-566</TH><TD>1C3  Hag-KpnI- D2-Strep 2.1</TD><TD>iGEM  53</TD></TR><TR><TH scope="row">AGB0023-567</TH><TD>1C3  Hag-KpnI- D2-Strep 2.1</TD><TD>iGEM  54</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp23"><LEGEND><A name="exp23.43">23.43 Analysis of the sequencing of 1C3 Hag-KpnI- D2-Strep</A></LEGEND><DIV class="exp-content"><P>Sequencing results of both plasmids are positive. Send to registry tomorrow.</P></DIV></FIELDSET><FIELDSET class="exp23"><LEGEND><A name="exp23.44">23.44 Addition of the two new biobricks to the registry (short/long description and sequence)</A></LEGEND><DIV class="exp-content"><P>see above</P></DIV></FIELDSET><FIELDSET class="exp24"><LEGEND><A name="exp24.6">24.6 Miniprep and restriction of pET16b_StrepCup</A></LEGEND><DIV class="aim"><P>Aim: check the plasmid for correct insertion of StrepCup</P></DIV><DIV class="exp-content"><P>Clone 6 was picked to inoculate a miniprep in LB-ampicillin. The plasmid was isolated after 8 hours of incubation at 37°C. A control restriction was carried out with the enzymes <I>Nco</I>I, <I>Eco</I>RI and <I>Pst</I>I; a negative clone would result in bands of the size of 394 bp, 748 bp and 4569 bp; a positive clone would yield bands in size of 748 bp, 865 bp and 4569 bp. Lane 2 showed the fragments in the expected size.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.108">13.108 Gibson assembly of piGEM008 + Cu promoter and ligation of the nose plasmids with the constitutive promoter</A></LEGEND><DIV class="aim"><P>Aim: insert the different promoters into the nose plasmids</P></DIV><DIV class="exp-content"><P>One plasmid with a metal sensitive promoter was still missing: piGEM008 + Cu promoter. A Gibson assembly with the digested vector and the Copper sensitive promoter was carried out in two parallels: one with purified vector (low concentration) and one with unpurified vector, only heat inactivated after restriction (high concentration).</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Gibson purified  vector (µL)</TH><TH scope="col">Gibson unpurified  vector (µl)</TH><TH scope="col">Control  (µL)</TH></TR><TR><TH scope="row">Purified piGEM008  (89.6 ng/µL)</TH><TD>4</TD><TD> </TD><TD>4</TD></TR><TR><TH scope="row">Unpurified piGEM008 (196.7 ng/µl)</TH><TD> </TD><TD>3,2</TD><TD> </TD></TR><TR><TH scope="row">Cu promotor</TH><TD>1</TD><TD>1</TD><TD> </TD></TR><TR><TH scope="row">Gibson mix</TH><TD>15</TD><TD>15</TD><TD>15</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><P>The reaction was incubated at  room temparature for 15 seconds and on 50°C for one hour. The whole reaction  mixes were used to transform competent <EM>E.  coli</EM> XLIBlue cells, which were plated out on LB-ampicillin plates.</P><P>The ligation with the  constitutive promoter was carried out with unpurified vector.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Ligation I (µl)</TH><TH scope="col">Ligation II (µl)</TH><TH scope="col">Ligation III (µl)</TH><TH scope="col">Ligation IV (µl)</TH></TR><TR><TH scope="row">piGEM-002 (201.8  ng/µl)</TH><TD>6,77</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TH scope="row">piGEM007 (ca 143.4  ng/µl)</TH><TD> </TD><TD>7,4</TD><TD> </TD><TD> </TD></TR><TR><TH scope="row">piGEM008 (ca 196.7  ng/µl)</TH><TD> </TD><TD> </TD><TD>6,8</TD><TD> </TD></TR><TR><TH scope="row">piGEM009 (ca 218.3 ng/µl)</TH><TD> </TD><TD> </TD><TD> </TD><TD>6,6</TD></TR><TR><TH scope="row">constitutive promoter (49.3 ng/µl)</TH><TD>3,23</TD><TD>2,6</TD><TD>3,2</TD><TD>3,4</TD></TR><TR><TH scope="row">T4 Ligase</TH><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Ligation Buffer 10x</TH><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR><TH scope="row">Water</TH><TD>7</TD><TD>7</TD><TD>7</TD><TD>7</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>20</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><P>The reactions were incubated for  one hour at room temperature and used to transform competent <EM>E. coli</EM> XLIBlue cells, which were plated  out on LB-ampicillin and incubated at 37°C.</P></DIV></FIELDSET><FIELDSET class="exp22"><LEGEND><A name="exp22.20">22.20 Gibson assembly with separate KSI parts</A></LEGEND><DIV class="aim"><P>Aim: Fusion of the single parts to create KS module I</P></DIV><DIV class="exp-content"><P>Since the first attempts to combine the modules for KSI did not work, a simple Gibson assembly was performed with the separate parts. One reaction also contained the primers iGEM075 and iGEM076 which should introduce the overhangs between the parts.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Gibson (µL)</TH><TH scope="col">Gibson with  additional primer (µl)</TH></TR><TR><TH scope="row">amyE flank I (ca 10 ng/µl)</TH><TD>1,6</TD><TD>1</TD></TR><TR><TH scope="row">IPTG inducible promoter (iGEM071/072; ca 10 ng/µl)</TH><TD>1,6</TD><TD>1</TD></TR><TR><TH scope="row">antiholin-amyE flank II (10 ng/µl)</TH><TD>1,6</TD><TD>1</TD></TR><TR><TH scope="row">Primer iGEM075 (1:50)</TH><TD> </TD><TD>0,5</TD></TR><TR><TH scope="row">Primer iGEM076 (1:50)</TH><TD> </TD><TD>0,5</TD></TR><TR><TH scope="row">H2O</TH><TD> </TD><TD>1</TD></TR><TR><TH scope="row">Gibson Mix</TH><TD>15</TD><TD>15</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>20</TD></TR></TBODY></TABLE></DIV><P>Applied onto a gel nothing could be seen.</P></FIELDSET><FIELDSET class="exp25"><A name="exp25.7">25.7 new Fluorometer assay with A549 and Caco-2</A><DIV class="aim"><P>Aim: Checking interaction of StrepDARPidin with EpCAM on A549 Lung carcinoma cells &amp; Caco-2 cells using a new serial dilution</P></DIV><DIV class="exp-content"><P>This time Caco-2 (C1-6) and A549 (C7-12) were used to coat row C of a black 96-well plate (100000 cells/ well). 100000 cells of 3T3 fibroblasts were used as negative control and as well used to coat 2 wells (G10 &amp; G11) which was done overnight.</P><P>The same protocol like in 22.5 was used or GeFi-purified StrepDARPidin.</P><P>Following concentrations of StrepDARPidin dissolved in PBS (pH 7,4) +2,5 % glycerin were used:</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Well Col</TH><TH scope="col">Well Row</TH><TH scope="col">raw data (485, 520)</TH><TH scope="col">Purification</TH><TH scope="col">StrepDARPidin  concentration [M]</TH></TR><TR><TH scope="row">Caco-2</TH><TD>1;1</TD><TD>C</TD><TD>344</TD><TD>-</TD><TD>25 µM</TD></TR><TR><TH scope="row">Caco-2</TH><TD>2;2</TD><TD>C</TD><TD>538</TD><TD>-</TD><TD>2.5 µM</TD></TR><TR><TH scope="row">Caco-2</TH><TD>3;3</TD><TD>C</TD><TD>386</TD><TD>-</TD><TD>250 nM</TD></TR><TR><TH scope="row">Caco-2</TH><TD>4;4</TD><TD>C</TD><TD>319</TD><TD>-</TD><TD>25 nM</TD></TR><TR><TH scope="row">Caco-2</TH><TD>5;5</TD><TD>C</TD><TD>281</TD><TD>-</TD><TD>2.5 nM</TD></TR><TR><TH scope="row">Caco-2</TH><TD>6;6</TD><TD>C</TD><TD>259</TD><TD>-</TD><TD>250 pM</TD></TR><TR><TH scope="row">A549</TH><TD>7;7</TD><TD>C</TD><TD>422</TD><TD>-</TD><TD>25 µM</TD></TR><TR><TH scope="row">A549</TH><TD>8;8</TD><TD>C</TD><TD>396</TD><TD>-</TD><TD>2,5 µM</TD></TR><TR><TH scope="row">A549</TH><TD>9;9</TD><TD>C</TD><TD>273</TD><TD>-</TD><TD>250 nM</TD></TR><TR><TH scope="row">A549</TH><TD>10;10</TD><TD>C</TD><TD>247</TD><TD>-</TD><TD>25 nM</TD></TR><TR><TH scope="row">A549</TH><TD>11;11</TD><TD>C</TD><TD>288</TD><TD>-</TD><TD>2,5 nM</TD></TR><TR><TH scope="row">A549</TH><TD>12;12</TD><TD>C</TD><TD>272</TD><TD>-</TD><TD>250 pM</TD></TR><TR><TH scope="row">Blank</TH><TD>5;9</TD><TD>G</TD><TD>197</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">negative control 3T3 fibroblast</TH><TD>6;10</TD><TD>G</TD><TD>355</TD><TD>-</TD><TD>25 µM</TD></TR><TR><TH scope="row">negative control 3T3 fibroblast</TH><TD>7;11</TD><TD>G</TD><TD>305</TD><TD>-</TD><TD>25 µ</TD></TR><TR><TH scope="row">positive AB-control</TH><TD>8,12</TD><TD>G</TD><TD>3950</TD><TD>-</TD><TD>-</TD></TR></TBODY></TABLE><P>The serial dilution shows that the signal is decreasing for both cell lines. The Caco-2 Cells have a higher signal. The experiment should be repeated.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="07.10.2014">07.10.2014</A></H2><FIELDSET class="exp13"><LEGEND><A name="exp13.109">13.109 Screening for positive transformands from the Gibson and ligation</A></LEGEND><DIV class="aim"><P>Aim: check for positive insertion of the promoters</P></DIV><DIV class="exp-content"><P>The plates all carried sufficient colonies to pick up to five clones per transformand. A colony PCR was carried out with specific primers for the constitutive promoter (piGEM073) and the reverse primer for the gfp with or without the degradation tags.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix PCR</TH><TH scope="col">02+const</TH><TH scope="col">07+const</TH><TH scope="col">08+const</TH><TH scope="col">09+const</TH><TH scope="col">08+Cu</TH></TR><TR><TH scope="row">Template</TH><TD>colony</TD><TD>colony</TD><TD>colony</TD><TD>colony</TD><TD>colony</TD></TR><TR><TH scope="row">iGEM055 (1:50)</TH><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD>3,125</TD></TR><TR><TH scope="row">iGEM073 (1:50)</TH><TD>3,125</TD><TD>3,125</TD><TD>3,125</TD><TD>3,125</TD><TD> </TD></TR><TR><TH scope="row">iGEM006 (1:50)</TH><TD>3,125</TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TH scope="row">iGEM021 (1:50)</TH><TD> </TD><TD>3,125</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TH scope="row">iGEM022 (1:50)</TH><TD> </TD><TD> </TD><TD>3,125</TD><TD> </TD><TD>3,125</TD></TR><TR><TH scope="row">iGEM023 (1:50)</TH><TD> </TD><TD> </TD><TD> </TD><TD>3,125</TD><TD> </TD></TR><TR><TH scope="row">5x HF buffer</TH><TD>4</TD><TD>4</TD><TD>4</TD><TD>4</TD><TD>4</TD></TR><TR><TH scope="row">dNTP mix (10 mM each)</TH><TD>0,3</TD><TD>0,3</TD><TD>0,3</TD><TD>0,3</TD><TD>0,3</TD></TR><TR><TH scope="row">Phusion DNA polymerase 1:10</TH><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>8,45</TD><TD>8,45</TD><TD>8,45</TD><TD>8,45</TD><TD>8,45</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>20</TD><TD>20</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>2 min 10 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>As a  positive control <EM>E. coli</EM> DH5α with  the nose plasmids containing the metallosensitive promoter were used. This  control only acted as a control for the right vector, since no plasmid was  available with the constitutive promoter, which could be used as positive  control for the promoter. For every construct positive clones were visible  except for piGEM009 + const. The positive fragment for the constitutive  promoter should be at ca 1000 bp and the positive construct of piGEM008 + Cu  should yield a 923 bp fragment. Both of the fragments could be seen on the gel.  Further work was done with the clones 4 of piGEM002 + const, 1 of piGEM007 +  const, 5 of piGEM008 + const and 3 of piGEM008 + Cu.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.110">13.110 Plasmid Maxi Prep of Nose plasmids with Ag and Cu promoter</A></LEGEND><DIV class="aim"><P>Aim:  Obtain enough plasmid to transform <EM>Bacillus  subtilis</EM></P></DIV><DIV class="exp-content"><P>To  transform Bacillus a huge amount of plasmid is needed. For this reason 100 ml  culture with <EM>E. coli</EM> for every nose  plasmid with a metallosensitive promoter was harvested and the plasmids were  isolated according to the protocol in the Qiagen Plasmid Plus Maxi Kit. One  exception was made, since no QIAvac plus 24 was available and no vacuum pump.  Thus the filtered cell lysate was loaded onto the  columns by centrifugation. </P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="08.10.2014">08.10.2014</A></H2><FIELDSET class="exp23"><LEGEND><A name="exp23.45">23.45 Sending the two new biobricks to the registry</A></LEGEND><DIV class="exp-content"><P>Preparation of both bricks:</P><UL><LI>15 µl plasmid DNA (app. 200 ng/µl) in PCR cup</LI><LI>Tape with Biobrick number around the PCR cup</LI><LI>Parafilm wrapped around the whole Eppendorf cup</LI><LI>50 ml Falkon with Shipping number, team name and  shipping date</LI></UL><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Part  Number</TH><TH scope="col">Name</TH></TR><TR><TH scope="row">Part:BBa_K1329000</TH><TD>StrepDARPidin</TD></TR><TR><TH scope="row">Part:BBa_K1329001</TH><TD>Hag-KpnI</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp23"><LEGEND><A name="exp23.46">23.46 PCR amplification of the Arc1p-C domain fragment and the PheA domain fragment</A></LEGEND><DIV class="aim"><P>Aim: Fragments are needed for the cloning of two new biobricks</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col"> </TH><TH scope="col">Arc1p (µl)</TH><TH scope="col">PheA (µl)</TH></TR><TR><TH scope="row">Water</TH><TD>24</TD><TD>24</TD></TR><TR><TH scope="row">HF Buffer 5x</TH><TD>10</TD><TD>10</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Primer for (1:50)</TH><TD>6.5</TD><TD>6.5</TD></TR><TR><TH scope="row">Primer rev (1:50)</TH><TD>6.5</TD><TD>6.5</TD></TR><TR><TH scope="row">Template (20 ng/µl)</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">dNTP mix (10 mM each)</TH><TD>1</TD><TD>1</TD></TR></TBODY></TABLE><P>Both fragments could be amplified (in a duplicate) and will be used for further cloning processes. The gel for the PheA domain did not run straight, but the approximate expected size can be seen.</P></DIV></FIELDSET><FIELDSET class="exp23"><LEGEND><A name="exp23.47">23.47 Restriction of pSB1C3 and the amplified fragments with <I>Xba</I>I and <I>Pst</I>I</A></LEGEND><DIV class="aim"><P>Aim: Digested plasmid and DNA fragments are needed</P></DIV><DIV class="exp-content"><P>The iGEM vector pSB1C3 and the fragments resulting from the previous PCR will be digested with <I>Xba</I>I and <I>Pst</I>I for a following ligation and transformation.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Reaction Mix</TH><TH scope="col">pSB1C3 (µl)</TH><TH scope="col">Arc1p (µl)</TH><TH scope="col">PheA (µl)</TH></TR><TR><TH scope="row"><I>Pst</I>I</TH><TD>0.5</TD><TD>0.5</TD><TD>0.5</TD></TR><TR><TH scope="row"><I>Xba</I>I</TH><TD>0.5</TD><TD>0.5</TD><TD>0.5</TD></TR><TR><TH scope="row">Buffer 2.1 10x</TH><TD>2.0</TD><TD>2.0</TD><TD>2.0</TD></TR><TR><TH scope="row">Water</TH><TD>14</TD><TD>12</TD><TD>9</TD></TR></TBODY></TABLE><P>Digestion at room temperature over night.</P><FIELDSET class="exp13"><LEGEND><A name="exp13.111">13.111 Bacillus Transformation with isolated plasmids</A></LEGEND><DIV class="aim"><P>Aim: Transformation of <I>Bacillus subtilis</I> PY79 with the nose-plasmids</P></DIV><DIV class="exp-content"><P>The plasmids isolated with the maxi prep kit were used to transform competent <I>Bacillus subtilis</I> PY79. Ca 15 µg of each plasmid were given to 100 µl of cells in a test tube, which was incubated shaking at 37°C for 30 min before plating the cells out on LB-chloramphenicol plates.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.111">Control PCR with nose-lac plasmids</A></LEGEND><DIV class="aim"><P>check the plasmids for correct insertion of the lac promoter</P></DIV><DIV class="exp-content"><P>The plasmids isolated with the maxi prep kit were used to transform competent <I>Bacillus subtilis</I> PY79. Ca 15 µg of each plasmid were given to 100 µl of cells in a test tube, which was incubated shaking at 37°C for 30 min before plating the cells out on LB-chloramphenicol plates.</P></DIV><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix PCR</TH><TH scope="col">02 + lac °C</TH><TH scope="col">07 + lac</TH><TH scope="col">08 + lac</TH><TH scope="col">09 + lac</TH></TR><TR><TH scope="row">piGEM002/7/8/9 + lac (ca 40 ng/µl)</TH><TD>1,5</TD><TD>1,5</TD><TD>1,5</TD><TD>1,5</TD></TR><TR><TH scope="row">iGEM073 (1:50)</TH><TD>3,125</TD><TD>3,125</TD><TD>3,125</TD><TD>3,125</TD></TR><TR><TH scope="row">iGEM006 (1:50)</TH><TD>3,125</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">iGEM021 (1:50)</TH><TD>-</TD><TD>3,125</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">iGEM022 (1:50)</TH><TD>-</TD><TD>-</TD><TD>3,125</TD><TD>-</TD></TR><TR><TH scope="row">iGEM023 (1:50)</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>3.125</TD></TR><TR><TH scope="row">5x HF buffer</TH><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD></TR><TR><TH scope="row">dNTP mix (10 mM each)</TH><TD>0,5</TD><TD>0,5</TD><TD>0,5</TD><TD>0,5</TD></TR><TR><TH scope="row">Phusion DNA polymerase 1:10</TH><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>23.5</TD><TD>23.5</TD><TD>23.5</TD><TD>23.5</TD></TR><TR><TH scope="row">Total</TH><TD>50</TD><TD>50</TD><TD>50</TD><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">PCR Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1 min</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The PCR did not work. The high bands hat could be seen were probably the plasmid templates.</P></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="09.10.2014">09.10.2014</A></H2><FIELDSET class="exp24"><LEGEND><A name="exp24.7">24.7 Test expression of StrepCup in <I>E. coli</I> BL21(DE3)</A></LEGEND><DIV class="aim"><P>Aim: check the production of StrepCup</P></DIV><DIV class="exp-content"><P>30 ml LB-ampicillin were inoculated with cells of <I>E. coli</I> BL21(DE3) pET16b_StrepCup from plate. This culture was incubated shaking at 37°C until they reached an OD<SUB>600</SUB> of 0.58. A preinduction sample was taken (1.2 ml = (0.7 * 0.7) / (0.7 * 0.58)), the cells were pelleted by centrifugation at 14000 rpm for 1 minute, followed by resuspension in 80 µl water and 20 µl 5X SDS loading dye. The culture was induced with 30 µl IPTG for 3 hours. Another sample was taken afterwards, at this time point the culture reached an OD<SUB>600</SUB> of 1.2 (0.583 ml = (0.7 * 0.7 / 0.7 * 1.2)). These samples were boiled at 95°C for 10 minutes and then analyzed by SDS-PAGE (12% SDS-polyacrylamide gel).
	</P><P>It could be noticed that there was a difference between the preinduction and the induction sample. According to the Expasy Compute pI/Mw tool StrepCup should be around 18.6 kDa. The protein expressed was slightly bigger, but considering the protein page ruler is just an orientation guide and not 100% accurate the thick band in the induction sample was determined as StrepCup (ca 20 kDa).
	</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.113">13.113 control PCR of nose plasmids with lac promoter / constitutive promoter</A></LEGEND><DIV class="aim"><P>Aim: check nose - lac/const plasmids for correct insert
	</P></DIV><DIV class="exp-content"><P>Since there could have been a confusion after isolating the plasmids, which could have been the reason for the last negative control PCR on the lac plasmids, every reverse primer for the nose plasmids with the ssrA-tags and without the degradation tag were tested. The following table went for every of the nose plasmids with the lac-promoter, not only for the shown piGEM002 + lac In addition to that, a control PCR was performed with the the three existing nose plasmids with constitutive promoter piGEM002/007 and 008, another control PCR was done for piGEM008 + Cu sensitive promoter, which was the last one missing of the nose-plasmids with the metallosensitive promoters.</P><P>Additionally, a colony PCR was made with further 6 clones picked from the ligation plate for piGEM009 + const to check for positive clones.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix PCR</TH><TH scope="col">02+lac with iGEM006</TH><TH scope="col">02+lac with iGEM021</TH><TH scope="col">02+lac with iGEM022</TH><TH scope="col">02+lac with iGEM023</TH><TH scope="col">02+const</TH><TH scope="col">08+Cu</TH><TH scope="col">cPCR 09 + const</TH></TR><TR><TH scope="row">colony</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>part</TD></TR><TR><TH scope="row">piGEM002 (ca 40 ng/µl)</TH><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">piGEM002/007/008 + const. %#040;ca 40 ng/µl)</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>1</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">piGEM008 + Cu</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">iGEM059 (1:50)</TH><TD>3,3</TD><TD>3,3</TD><TD>3,3</TD><TD>3,3</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">iGEM073 (1:50)</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>3,3</TD><TD>-</TD><TD>3,125</TD></TR><TR><TH scope="row">iGEM055 (1:50)</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>3,3</TD><TD>-</TD></TR><TR><TH scope="row">iGEM006 (1:50)</TH><TD>3,3</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>3,3</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">iGEM021 (1:50)</TH><TD>-</TD><TD>3,3</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">iGEM022 (1:50)</TH><TD>-</TD><TD>-</TD><TD>3,3</TD><TD>-</TD><TD>-</TD><TD>3,3</TD><TD>-</TD></TR><TR><TH scope="row">iGEM023 (1:50)</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>3,3</TD><TD>-</TD><TD>-</TD><TD>3,125</TD></TR><TR><TH scope="row">5x HF buffer</TH><TD>5</TD><TD>5</TD><TD>5</TD><TD>5</TD><TD>5</TD><TD>5</TD><TD>5</TD></TR><TR><TH scope="row">dNTP mix (10 mM each)</TH><TD>0,3</TD><TD>0,3</TD><TD>0,3</TD><TD>0,3</TD><TD>0,3</TD><TD>0,3</TD><TD>0,3</TD></TR><TR><TH scope="row">Phusion DNA polymerase 1:10</TH><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>11,1</TD><TD>11,1</TD><TD>11,1</TD><TD>11,1</TD><TD>11,1</TD><TD>11,1</TD><TD>11,1</TD></TR><TR><TH scope="row">Total</TH><TD>25</TD><TD>25</TD><TD>25</TD><TD>25</TD><TD>25</TD><TD>25</TD><TD>25</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">PCR Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>5 min/ colony PCR separate: 10 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>70 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>infinite</TD></TR></TBODY></TABLE><P>For the colony PCR several positive clones could be seen that exposed a band in the height of ca 1000 bp, clone 9 was chosen to work with in the further steps. The control PCR on the nose - lac plasmids was negative for all plasmids. Some showed two very thin bands of ca 750 bp and 1000 bp size. The expected fragment should have a size of approx. 818 bp. These findings were striking, since the colony PCR for the clones from which the plasmids were isolated were positive.</P><P>The control PCR for the plasmids piGEM002/007 + const and piGEM008 + Cu were positive. The clones containing these plasmids were used to inoculate 100 ml LB+ampicillin for a maxi prep.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.114">13.114 control restriction of nose plasmids + lac and new colony PCR of picked clones</A></LEGEND><DIV class="aim"><P>Aim: verify the absence of the lac promoter in the nose plasmids like indicated by the control PCR</P></DIV><DIV class="exp-content"><P>The colony PCR for the clones containing the nose plasmids were positive but the control PCR with the isolated plasmids were negative. Since colony PCRs are a less reliable method than the control PCR on the isolated plasmid or a control restriction, the plasmids should be negative. To be sure a control restriction with <I>Bam</I>HI and <I>Hind</I>III was carried out with these plasmids. The positive plasmids should result in fragments of 4194 bp and 2620 bp size, the negative plasmid should not be digested by <I>Hind</I>III, because the only restriction site for <I>Hind</I>III is in the IPTG inducible promoter. As a control piGEM030 was cut with <I>Hind</I>III and <I>Nco</I>I (1172 bp, 5246 bp and 241 bp) and piGEM008 + const was cut with <I>Bam</I>HI and <I>Nco</I>I (4189 bp + 2757 for a positive plasmid, 4189 bp and 2518 bp for a negative plasmid).
	<TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">piGEM030 (µL)</TH><TH scope="col">lac plasmids (µL)</TH><TH scope="col">piGEM008 + const (µL)</TH></TR><TR><TH scope="row">piGEM030</TH><TD>5</TD><TD>5</TD><TD>5</TD></TR><TR><TH scope="row">nose plasmids + lac promoter</TH><TD>-</TD><TD>5</TD><TD>-</TD></TR><TR><TH scope="row">piGEM008 + const</TH><TD>-</TD><TD>-</TD><TD>5</TD></TR><TR><TH scope="row"><I>Nco</I>I-HF</TH><TD>0,5</TD><TD>-</TD><TD>0,5</TD></TR><TR><TH scope="row"><I>Hind</I>III</TH><TD>1</TD><TD>1</TD><TD>-</TD></TR><TR><TH scope="row"><I>Bam</I>HI-HF</TH><TD>-</TD><TD>0,5</TD><TD>0,5</TD></TR><TR><TH scope="row">CutSmart 10x</TH><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR><TH scope="row">Water</TH><TD>11,5</TD><TD>11,5</TD><TD>12</TD></TR><TR><TH scope="row">Total</TH><TD>20</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE></P><P>The reactions were incubated at 37°C for over one hour.</P><P>Additionally a new colony PCR was done with new picked clones from the ligation plates of the plasmids with the lac promoter.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix colony PCR</TH><TH scope="col">piGEM002 + lac (µL)</TH><TH scope="col">piGEM007 + lac (µL)</TH><TH scope="col">piGEM008 + lac (µL)</TH><TH scope="col">piGEM009 + lac (µL)</TH><TH scope="col">Positive control piGEM002 + Ag (µL)</TH></TR><TR><TH scope="row">Template</TH><TD>part of a colony</TD><TD>part of a colony</TD><TD>part of a colony</TD><TD>part of a colony</TD><TD>part of a colony</TD></TR><TR><TH scope="row">iGEM073 (1:50)</TH><TD>3,125</TD><TD>3,125</TD><TD>3,125</TD><TD>3,125</TD><TD>-</TD></TR><TR><TH scope="row">iGEM057 (1:50)</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>3,125</TD></TR><TR><TH scope="row">iGEM006 (1:50)</TH><TD>3,125</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>3,125</TD></TR><TR><TH scope="row">piGEM021 + const</TH><TD>-</TD><TD>3,125</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">piGEM022 + const</TH><TD>-</TD><TD>-</TD><TD>3,125</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">piGEM023 + const</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>3,125</TD><TD>-</TD></TR><TR><TH scope="row">5x HF buffer</TH><TD>5</TD><TD>5</TD><TD>5</TD><TD>5</TD><TD>5</TD></TR><TR><TH scope="row">dNTP mix (10 mM each)</TH><TD>0,3</TD><TD>0,3</TD><TD>0,3</TD><TD>0,3</TD><TD>0,3</TD></TR><TR><TH scope="row">Phusion DNA polymerase 1:10</TH><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>12,45</TD><TD>12,45</TD><TD>12,45</TD><TD>12,45</TD><TD>12,45</TD></TR><TR><TH scope="row">Total</TH><TD>20</TD><TD>20</TD><TD>20</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">PCR Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>10 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>60 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The control restriction of the nose plasmids was negative, the plasmid just seemed to be linearized and not cut into two fragments. That means that the <I>Hind</I>III could not cut since the IPTG inducible promoter was not contained in the plasmids. However, also the control piGEM030 did not show the expected bands. It could be possible that <I>Hind</I>III did not work correctly, since it was no HF enzyme and used in the CutSmart buffer of NEB, but according to the NEB Double Digest Finder <I>Hind</I>III was able to cut in CutSmart only with the half of its optimal activity. That was the reason, why the double amount of <I>Hind</I>III was chosen.</P><P>Some samples of the new colony PCR showed bands slightly higher than 750 bp, which would resemble the size of the expected fragment of ca 800 bp, but since the controls with the nose plasmids containing the Ag sensitive promoter (all confirmed positive by control PCR on isolated plasmids) were not all positive, it was uncertain, that the visible fragments really were the ones that indicate a positive clone. For the further cloning, the plasmids were isolated and digested directly.</P></DIV></FIELDSET><FIELDSET class="exp25"><LEGEND><A name="exp25.9">25.9 new Fluorometer assay with A549 and Caco-2</A></LEGEND><DIV class="aim"><P>Aim: Checking interaction of StrepDARPidin with EpCAM on A549 Lung carcinoma cells &amp; Caco-2 cells using a new serial dilution</P></DIV><DIV class="exp-content"><P>This time Caco-2 (A1-6) and A549 (B1-6) were used to coat row C of a black 96-well plate (100000 cells/ well). 100000 cells of 3T3 WT fibroblasts were used as negative control and as well used to coat C5.</P><P>The same protocol like in 22.5 was used or GeFi-purified StrepDARPidin.</P><P>Following concentrations of StrepDARPidin dissolved in PBS pH 7,4 +2,5 % glycerin were used:</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Well Col</TH><TH scope="col">Well Row</TH><TH scope="col">Blank corrected raw data (85, 520)</TH><TH scope="col">StrepDARPidin concentration (M)</TH></TR><TR><TD>Caco-2</TD><TD>1</TD><TD>A</TD><TD>1096</TD><TD>25 µM</TD></TR><TR><TD>Caco-2</TD><TD>2</TD><TD>A</TD><TD>380</TD><TD>25 µM</TD></TR><TR><TD>Caco-2</TD><TD>3</TD><TD>A</TD><TD>745</TD><TD>2,5 µM</TD></TR><TR><TD>Caco-2</TD><TD>4</TD><TD>A</TD><TD>565</TD><TD>2,5 µM</TD></TR><TR><TD>Caco-2</TD><TD>5</TD><TD>A</TD><TD>539</TD><TD>250 nM</TD></TR><TR><TD>Caco-2</TD><TD>6</TD><TD>A</TD><TD>635</TD><TD>250 nM</TD></TR><TR><TD>Caco-2</TD><TD>7</TD><TD>A</TD><TD>300</TD><TD>25 nM</TD></TR><TR><TD>Caco-2</TD><TD>8</TD><TD>A</TD><TD>280</TD><TD>25 nM</TD></TR><TR><TD>Caco-2</TD><TD>9</TD><TD>A</TD><TD>1063</TD><TD>2,5 nM</TD></TR><TR><TD>Caco-2</TD><TD>10</TD><TD>A</TD><TD>1024</TD><TD>2,5 nM</TD></TR><TR><TD>Caco-2</TD><TD>11</TD><TD>A</TD><TD>328</TD><TD>250 pM</TD></TR><TR><TD>Caco-2</TD><TD>12</TD><TD>A</TD><TD>488</TD><TD>250 pM</TD></TR><TR><TD>A549</TD><TD>1</TD><TD>B</TD><TD>377</TD><TD>25 µM</TD></TR><TR><TD>A549</TD><TD>2</TD><TD>B</TD><TD>444</TD><TD>25 µM</TD></TR><TR><TD>A549</TD><TD>3</TD><TD>B</TD><TD>365</TD><TD>2,5 µM</TD></TR><TR><TD>A549</TD><TD>4</TD><TD>B</TD><TD>563</TD><TD>2,5 µM</TD></TR><TR><TD>A549</TD><TD>5</TD><TD>B</TD><TD>255</TD><TD>250 nM</TD></TR><TR><TD>A549</TD><TD>6</TD><TD>B</TD><TD>294</TD><TD>250 nM</TD></TR><TR><TD>A549</TD><TD>7</TD><TD>B</TD><TD>376</TD><TD>25 nM</TD></TR><TR><TD>A549</TD><TD>8</TD><TD>B</TD><TD>330</TD><TD>25 nM</TD></TR><TR><TD>A549</TD><TD>9</TD><TD>B</TD><TD>1244</TD><TD>2,5 nM</TD></TR><TR><TD>A549</TD><TD>10</TD><TD>B</TD><TD>531</TD><TD>2,5 nM</TD></TR><TR><TD>A549</TD><TD>11</TD><TD>B</TD><TD>1049</TD><TD>250 pM</TD></TR><TR><TD>A549</TD><TD>12</TD><TD>B</TD><TD>864</TD><TD>250 pM</TD></TR><TR><TD>Blank</TD><TD>1</TD><TD>C</TD><TD>216</TD><TD>-</TD></TR><TR><TD>Empty Well</TD><TD>2</TD><TD>C</TD><TD>767</TD><TD>-</TD></TR><TR><TD>Empty Well</TD><TD>3</TD><TD>C</TD><TD>848</TD><TD>-</TD></TR><TR><TD>Pos. control AB</TD><TD>4</TD><TD>C</TD><TD>4873</TD><TD>-</TD></TR><TR><TD>negative control 3T3 fibroblasts</TD><TD>5</TD><TD>C</TD><TD>512</TD><TD>25 µM</TD></TR></TBODY></TABLE><P>The serial dilution shows that the signal is decreasing for both cell lines. The Caco-2 Cells have a higher signal compared to A549.</P></DIV></FIELDSET><FIELDSET class="exp23"><LEGEND><A name="exp23.48">23.48 Purification of digested plasmid and fragments</A></LEGEND><DIV class="aim"><P>Aim: In order to improve the ligation results pure DNA samples are needed</P></DIV><DIV class="exp-content"><P>All digested samples have been purified via gel extraction, resulting in the following concentration:</P><UL class="list"><LI>pSB1C3= 19 ng/µl</LI><LI>Arc1p= 25 ng/µl</LI><LI>PheA= 23 ng/µl</LI></UL></DIV></FIELDSET><FIELDSET class="exp23"><LEGEND><A name="exp23.49">23.49 Ligation of digested plasmid and inserts</A></LEGEND><DIV class="aim"><P>Aim: Ligate plasmids for further transformation</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Ligation Mix</TH><TH scope="col">pSB11C3 + Arc1p (µl)</TH><TH scope="col">pSB11C3 + PheA (µl)</TH></TR><TR><TH scope="row">Water</TH><TD>0</TD><TD>4.3</TD></TR><TR><TH scope="row">HF Buffer 10x</TH><TD>2.0</TD><TD>2.0</TD></TR><TR><TH scope="row">T4 Ligase</TH><TD>1.0</TD><TD>1.0</TD></TR><TR><TH scope="row">Insert</TH><TD>7.6</TD><TD>9</TD></TR><TR><TH scope="row">Vector</TH><TD>9.4</TD><TD>3.7</TD></TR></TBODY></TABLE><P>The ligation was performed at 16°C for 3h.</P></DIV></FIELDSET><FIELDSET class="exp23"><LEGEND><A name="exp23.50">23.50 Transformation of competent <I>E. coli</I> XL-1-Blue with ligated plasmid</A></LEGEND><DIV class="aim"><P>Aim: Ligate plasmids for further transformation</P></DIV><DIV class="exp-content"><P>After the ligation the reaction was inactivated by incubating the reaction mix at 85%deg;C for 10 minutes. Afterwards chemically competent <I>E. coli</I> XL-1-blue were transformed with the whole reaction mix. Standard <I>E. coli</I> transformation protocol was used.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="10.10.2014">10.10.2014</A></H2><FIELDSET class="exp24"><LEGEND><A name="exp24.8">24.8 Test for solubility of StrepCup</A></LEGEND><DIV class="aim"><P>Aim: check if StrepCup is soluble or enclosed in inclusion bodies</P></DIV><DIV class="exp-content"><P>A second test expression was carried out in 30 ml LB-amp, which was inoculated with <I>E. coli</I> BL21(DE3) and incubated until they reached an OD<SUB>600</SUB> of 0.62. A 1.12 ml preinduction sample was taken and the cells were induced with 30 µl IPTG for 3 hours shaking at 37°C. After this time they reached an OD<SUB>600</SUB> of 1.4, so 500 µl were taken as an induction sample. The taken samples were pelleted by centrifugation, resuspended in 80 µl of water with addition of 20 µl 5x SDS loading dye. The cells were centrifuged at 4000 rpm for 20 minutes and lysed by the microfluidizer. The cell lysate was centrifuged at 20000 rpm for 20 minutes. The supernatant was withdrawn and 80 µl were taken as a sample, which was treated with 20 µl of 5x SDS loading dye. The pellet was resuspended in some buffer A from which 80 µl were taken and 20 µl of 5x SDS loading dye were added. The samples were boiled at 95°C for 10 minutes. Afterwards 10 µl of each sample was loaded onto a 12% SDS polyacrylamide gel.</P><P>It could clearly be seen that the StrepCup was in the unsoluble fraction of the lysate.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.115">13.115 Ligation of piGEM008/009 with constitutive promoter</A></LEGEND><DIV class="aim"><P>Aim: ligate the digested vector with the constitutive promoter</P></DIV><DIV class="exp-content"><P>Since the control PCR for piGEM008 with the constitutive promoter was negative and not even the colony PCR for piGEM009 with the constitutive promoter was positive, the ligation with these two plasmids was repeated.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Ligation I (µl)</TH><TH scope="col">Ligation II (µl)</TH></TR><TR><TD>piGEM008 (ca 196.7 ng/µl)</TD><TD>5.2</TD><TD>-</TD></TR><TR><TD>piGEM009 (ca 218.3 ng/µl)</TD><TD>-</TD><TD>3.05</TD></TR><TR><TD>constitutive promoter (ca 49.3 ng/µl)</TD><TD>2.3</TD><TD>4.45</TD></TR><TR><TD>T4 ligase</TD><TD>0.5</TD><TD>0.5</TD></TR><TR><TD>Ligation buffer 10x</TD><TD>2</TD><TD>2</TD></TR><TR><TD>Water</TD><TD>10</TD><TD>10</TD></TR><TR><TD>Total</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><P>The reaction was incubated at room temperature for over one hour and afterwards used to transform competent <I>E. coli</I> XLIBlue, which were plated out on LB-ampicillin and incubated at 37°C over night.</P><P>Additionally, clones were picked from the old ligation plates to inoculate cultures of LB-ampicillin for plasmid preps.</P></DIV></FIELDSET><FIELDSET class="exp23"><LEGEND><A name="exp23.51">23.51 Inoculation of presumably positive clones from the ligation plates</A></LEGEND><DIV class="aim"><P>Aim: Grow cultures for the isolation of plasmid psB1C3 with the two new Biobricks</P></DIV><DIV class="exp-content"><P>Two clones for every biobrick plasmid were inoculated in liquid LB-CM (10:30) for plasmid isolation.</P></DIV></FIELDSET><FIELDSET class="exp23"><LEGEND><A name="exp23.52a">23.52a Plasmid isolation and test digest</A></LEGEND><DIV class="aim"><P>Aim: Analyze the plasmids for the insertion of the fragments into the plasmid</P></DIV><DIV class="exp-content"><P>Plasmid isolation of the pSB1C3 plasmid from <I>E. coli</I> was performed with the Quiagen Mini-prep kit. Plasmid concentration was determined with a Nanodrop spectrophotometer (100-120 ng/µl).</P><P>Restriction: 4 µl plasmid in a total 10 µl restriction mix</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Restriction mix (µl)</TH></TR><TR><TH scope="col">Water</TH><TD>12.0</TD></TR><TR><TH scope="col">CutSmart 10x</TH><TD>2.0</TD></TR><TR><TH scope="col"><I>Xba</I>I</TH><TD>0.5</TD></TR><TR><TH scope="col"><I>Pst</I>I</TH><TD>0.5</TD></TR><TR><TH scope="col">Plasmid (600 ng/µl)</TH><TD>5.0</TD></TR></TBODY></TABLE><P>Gel analysis of the digested plasmids reveals that all clones are negative, new ligation will be performed with a higher amount of insert.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="11.10.2014">11.10.2014</A></H2><FIELDSET class="exp25"><LEGEND><A name="exp25.10">25.10 new Fluorometer assay with A549 and Caco-2</A></LEGEND><DIV class="aim"><P>Aim: Checking interaction of StrepDARPidin with EpCAM on A549 Lung carcinoma cells &amp; Caco-2 cells using a new serial dilution</P></DIV><DIV class="exp-content"><P>This time Caco-2 (F1-6( and A549 (G1-6) were used to coat row C of a black 96-well plate (100000 cells/ well). 100000 cells of 3T3WT fibroblasts were used as negative control and as well used to coat H2 &amp; H3.</P><P>The same protocol like in 22.5 was used for GeFi-purified StrepDARPidin.</P><P>Following concentrations of StrepDARPidin dissolved in PBS pH 7,4 +2,5 % glycerin were used:</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Well Col</TH><TH scope="col">Well Row</TH><TH scope="col">Blank corrected raw data (485, 520)</TH><TH scope="col">StrepDARPidin concentration (M)</TH></TR><TR><TD>Caco-2</TD><TD>1</TD><TD>F</TD><TD>382</TD><TD>25 µM</TD></TR><TR><TD>Caco-2</TD><TD>2</TD><TD>F</TD><TD>556</TD><TD>2,5 µM</TD></TR><TR><TD>Caco-2</TD><TD>3</TD><TD>F</TD><TD>550</TD><TD>250 nM</TD></TR><TR><TD>Caco-2</TD><TD>4</TD><TD>A</TD><TD>522</TD><TD>25 nM</TD></TR><TR><TD>Caco-2</TD><TD>5</TD><TD>F</TD><TD>541</TD><TD>2,5 nM</TD></TR><TR><TD>Caco-2</TD><TD>6</TD><TD>F</TD><TD>548</TD><TD>250 pM</TD></TR><TR><TD>A549</TD><TD>1</TD><TD>G</TD><TD>407</TD><TD>25 µM</TD></TR><TR><TD>A549</TD><TD>2</TD><TD>G</TD><TD>491</TD><TD>2,5 µM</TD></TR><TR><TD>A549</TD><TD>3</TD><TD>G</TD><TD>557</TD><TD>250 nM</TD></TR><TR><TD>A549</TD><TD>4</TD><TD>G</TD><TD>359</TD><TD>25 nM</TD></TR><TR><TD>A549</TD><TD>5</TD><TD>G</TD><TD>338</TD><TD>2,5 nM</TD></TR><TR><TD>A549</TD><TD>6</TD><TD>G</TD><TD>632</TD><TD>250 pM</TD></TR><TR><TD>negative control 3T3WT fibroblasts</TD><TD>2</TD><TD>H</TD><TD>409</TD><TD>25 µM</TD></TR><TR><TD>negative control 3T3WT fibroblasts</TD><TD>3</TD><TD>H</TD><TD>468</TD><TD>2,5 µM</TD></TR><TR><TD>Pos. control AB</TD><TD>4</TD><TD>H</TD><TD>3786</TD><TD>-</TD></TR></TBODY></TABLE><P>The serial dilution shows that the signal is decreasing for both cell lines. The Caco-2 Cells have a higher signal.</P><P>After the previous measurements the results were summarized and a graphic created:</P><P>The tendency which could be seen in the previous assays can be reproduced after summarizing the previous measurements as well.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.116">13.116 Making new competent <I>Bacillus subtilis</I> PY79 cells</A></LEGEND><DIV class="aim"><P>Aim: making competent cells to transform Bacillus subtilis PY79 with the Nose-plasmids</P></DIV><DIV class="exp-content"><P>Since the previous cells were not competent, a new try to make competent <I>Bacillus subtilis</I> PY79 was carried out. The procedure was performed according to the protocol using the SPC- and SPII-media. This time the cells were not centrifuged at 4°C but at room temperature, which should be gentler for the cells, since they do not need to induce cold stress responses that could affect the competence. The cells were tested for transformation efficiency by transforming them with each 25 µl and 50 µl of each nose plasmid with a metallosensitive promoter. After adding the plasmid DNA to 200 µl cells in a test tube, they were incubated shaking a 37°C for 30 minutes. Afterwards they were plated out on LB-chloramphenicol (5 µg/ml) were incubated at 37°C.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.117">13.117 control restriction of nose plasmids with the constitutive promoter</A></LEGEND><DIV class="aim"><P>Aim: check the plasmids for the correct insertion of the promoter</P></DIV><DIV class="exp-content"><P>The plasmids were isolated from the inoculated cultures (13.115; 2 clones piGEM002 + const, 6 clones piGEM007 + const, 5 clones piGEM008 + const and 4 clones piGEM009 + const) and digested with the enzymes <I>Nco</I>I-HF and <I>Eco</I>RI-HF. Negative plasmids should yield fragments in size of 476 and 6190 (for piGEM002) or 6235 bp (for piGEM007/008/009). The positive plasmids should result in fragments in the size of 711 and 6190/6235 bp.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Volumes (µl)</TH></TR><TR><TD>piGEM002/007/008/009 + const (ca 400 ng/µl)</TD><TD>5</TD></TR><TR><TD><I>Nco</I>I-HF</TD><TD>0.5</TD></TR><TR><TD><I>Eco</I>RI-HF</TD><TD>0.5</TD></TR><TR><TD>CutSmart 10x</TD><TD>2</TD></TR><TR><TD>Water</TD><TD>12</TD></TR><TR><TD>Total</TD><TD>20</TD></TR></TBODY></TABLE><P>The reactions were incubated at 37°C for over one hour, 3 µl 6x Purple Loading Dye were added and 7 µl of the restrictions were applied onto a 1% agarose gel.</P><P>The undigested piGEM002 was taken as a control. The isolated plasmids from the piGEM002 + const ligation plates did not yield in two fragments but look linearized. The clones 5 and 6 of piGEM007 + const, 4 and 5 of piGEM008 + const were positive. No plasmid of piGEM009 + const was positive.</P><P>The plasmids piGEM002 + const and piGEM007 + const were already confirmed as positive by the control PCR (13.113), now piGEM008 + const was positive as well, only piGEM009 + const was missing, so 10 clones from the old and new ligation plates were picked and used to inoculate more cultures LB-amp for plasmid isolation.</P></DIV></FIELDSET><FIELDSET class="exp23"><LEGEND><A name="exp23.53">23.53 New Ligation and transformation</A></LEGEND><DIV class="aim"><P>Aim: Analyze the plasmids for the insertion of the fragments into the plasmid</P></DIV><DIV class="exp-content"><P>In order to calculate the needed amount of insert and vector the iGEM ligation calculator was used. For this reaction a higher amount of insert was used (1:5)</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Ligation Mix</TH><TH scope="col">pSB11C3 + Arc1p (µl)</TH><TH scope="col">pSB11C3 + PheA (µl)</TH></TR><TR><TH scope="row">Water</TH><TD>0.6</TD><TD>5.5</TD></TR><TR><TH scope="row">HF Buffer 10x</TH><TD>2.0</TD><TD>2.0</TD></TR><TR><TH scope="row">T4 Ligase</TH><TD>1.0</TD><TD>1.0</TD></TR><TR><TH scope="row">Insert</TH><TD>9.0</TD><TD>9.0</TD></TR><TR><TH scope="row">Vector</TH><TD>7.4</TD><TD>2.5</TD></TR></TBODY></TABLE><P>The ligation was performed at 16°C for 3 hours.
	</P><P>After the ligation the reaction was inactivated by incubating the reaction mix at 85°C for 10 minutes. Afterwards chemically competent <I>E. coli</I> XL-1-blue were transformed with the whole reaction mix. Standard <I>E. coli</I> transformation protocol was used.
	</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="12.10.2014">12.10.2014</A></H2><FIELDSET class="exp13"><LEGEND><A name="exp13.118">13.118 control restriction of piGEM009 with the constitutive promoter</A></LEGEND><DIV class="aim"><P>Aim: check the plasmid for the correct insertion of the promoter</P></DIV><DIV class="exp-content"><P>The plasmids were isolated from the cultures (13.117) and digested again with the enzymes <I>Nco</I>I and <I>Eco</I>RI. The expected fragments were the same like in 13.117.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Volumes (µl)</TH></TR><TR><TD>piGEM009 + const (200-400 ng/µl)</TD><TD>5</TD></TR><TR><TD><I>Nco</I>I-HF</TD><TD>0.5</TD></TR><TR><TD><I>Eco</I>RI-HF</TD><TD>0.5</TD></TR><TR><TD>CutSmart 10x</TD><TD>2</TD></TR><TR><TD>Water</TD><TD>12</TD></TR><TR><TD>Total</TD><TD>20</TD></TR></TBODY></TABLE><P>The reactions were incubated at 37°C for over one hour, 3 µl 6x Purple Loading Dye were added and 7 µl of the restrictions were applied onto a 1% agarose gel.</P><P>The O‘ Gene Ruler 1 kb DNA ladder of Thermo Scientific was used as a marker. All cut fragments were in the size of ca. 500 bp (and the backbone at around 6000 bp), which meant, the plasmids did not contain the constitutive promoter.</P></DIV></FIELDSET><FIELDSET class="exp23"><LEGEND><A name="exp23.51b">23.51b Inoculation of presumably positive clones from the ligation plates</A></LEGEND><DIV class="aim"><P>Aim: Grow cultures for the isolation of plasmid psB1C3 with the two new Biobricks</P></DIV><DIV class="exp-content"><P>Two clones for every biobrick plasmid were inoculated in liquid LB-CM for plasmid isolation.</P></DIV></FIELDSET><FIELDSET class="exp23"><LEGEND><A name="exp23.52b">23.52b Plasmid isolation and test restriction</A></LEGEND><DIV class="aim"><P>Aim: Analyse the plasmids for the insertion of the fragments into the plasmid</P></DIV><DIV class="exp-content"><P>Plasmid isolation of the pSB1C3 plasmid from <I>E.coli</I> was performed with the Quiagen Mini-prep kit. Plasmid concentration was determined with a Nanodrop spectrophotometer (90-135 ng/µl).</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Restriction mix (µl)</TH></TR><TR><TH scope="col">Water</TH><TD>12.0</TD></TR><TR><TH scope="col">CutSmart 10x</TH><TD>2.0</TD></TR><TR><TH scope="col"><I>Xba</I>I</TH><TD>0.5</TD></TR><TR><TH scope="col"><I>Pst</I>I</TH><TD>0.5</TD></TR><TR><TH scope="col">Plasmid (600 ng/µl)</TH><TD>5.0</TD></TR></TBODY></TABLE><P>Both plasmids have been cloned successfully as visible in the two agarose gels. The expected inserts could be cut out of the iGEM plasmid pSB1C3.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="13.10.2014">13.10.2014</A></H2><FIELDSET class="exp18"><LEGEND><A name="exp18.74">18.74 Isolation of flagella from different <I>Bacillus subtilis</I> strains</A></LEGEND><DIV class="aim"><P>Aim: separation of the flagella from the cells</P></DIV><DIV class="exp-content"><P>Since the flagella of the <I>Bacillus subtilis</I> mutants Hag-D2-Strep or Hag-Strep were needed for affinity assays together with the StrepDARPidin and the cancer cell lines, the flagella should be isolated. Based on the pictures of the electron microscopy it was observed, that the Hag-D2-Cup strain formed flagella, which are shorter than these of the wildtype, which could mean that they break. However, the mutant flagellin should be expressed and so the Cup on the flagella should be able to fish for ions. In order to confirm that, the flagella of D2-Cup should be isolated as well. 1L LB each were inoculated with <I>Bacillus subtilis</I> 3610wt, Hag-D2-Strep, Hag-D2-Cup and Py79 Hag-Strep in 5 L Erlenmeyer flasks with baffles and incubated shaking at 37°C over night.</P></DIV></FIELDSET><FIELDSET class="exp24"><LEGEND><A name="exp24.9">24.9 Expression and purification of StrepCup</A></LEGEND><DIV class="aim"><P>Aim: expression of Streptavidin-Cup</P></DIV><DIV class="exp-content"><P>The Streptavidin-Cup construct sould also be overproduced, purified and tested for ion binding capacity or interaction with StrepTags of the flagella mutants. For this reason 2x 1 L LB-ampicillin was inoculated with <I>E. coli</I> BL21(DE3) pET16b_StrepCup. The medium also contained 50 ml 25 % (w/v; 12.5 g/50ml) lactose as an inducer of the expression. The cultures were incubated shaking at 30°C over night.</P></DIV></FIELDSET><FIELDSET class="exp23"><LEGEND><A name="exp23.53">23.53 Sequencing of BBa-K1329004 and BBa-K1329005</A></LEGEND><DIV class="aim"><P>Aim: check for the correct sequence of the biobricks</P></DIV><DIV class="exp-content"><P>Sequencing samples have been prepared in a concentration of 50 - 100 ng/µl and primers were added. Plasmids were then sequenced by eurofins mwg.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="14.10.2014">14.10.2014</A></H2><FIELDSET class="exp24"><LEGEND><A name="exp24.9">24.9 Expression and purification of StrepCup</A></LEGEND><DIV class="aim"><P>Aim: purification of StrepCup</P></DIV><DIV class="exp-content"><P>The two 1 L cultures were harvested by centrifugation at 4000 rpm for 20 min. The pellets were resuspended in 10 ml buffer A and centrifuged again in a falcon at 4000 rpm for 15 min. Again the pellet was resuspended in 25 ml buffer A and the cells were broken by a microfluidizer. The lysate was centrifuged with 20000rpm for 20 minutes at 4°C. Samples were taken from the pellet (P1) and the supernatant (S1), which was stored on ice. The pellet was resuspended in IB-wash buffer and centrifuged again at 27000 rpm for 10 minutes. This washing step was repeated. Samples were taken from the pellet (P2) and the supernatant (S2). The pellet was covered in 5 ml 6 M guanidine-hydrochloride and stirred at room temperature until it was completely resuspended, followed by another centrifugation with 20000 rpm at 4°C for 10 min. The pellet (P3) and the supernatant (S3) were stored separately in the refrigerator at 4°C over night after taking samples of both. It was observed that the proteins contained by the supernatant were precipitating instantly as they came in contact with water to dilute the sample and/or SDS loading dye. The precipitate was rich of protein, which resulted in an almost solid, sludgy consistence. After several dilution attempts the sample was diluted 1:1000 in water (protein precipitated by contact with water) the sample was vortexed until the precipitate was gone and 80 µl were taken together with 20 µl of 5x SDS loading dye and bolied at 95°C for 10 min together with the other samples, from which none precipitated as intense as the supernatant of the gua pellet. 10 µl of all samples were transferred onto a 12 % SDS-polyacrylamide gel.</P><P>The produced StrepCup was in the unsoluble phase like expected. During the process of isolating StrepCup from the inclusion bodies it could be seen, that StrepCup always remained in the pellet. It was not possible to see the protein in the soluble phase of the guanidinium-hydrochloride treated pellet, since the previous mentioned problems of instant precipitation in contact with water or SDS loading dye.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.74a">18.74a Isolation of flagella from different <I>Bacillus subtilis</I> strains</A></LEGEND><DIV class="aim"><P>Aim: separation of the flagella from the cells</P></DIV><DIV class="exp-content"><P>Strains: Wildtype 3610, PY-Strep, Hag-D2-Strep, Hag-D2-Cup</P><P>The different cultures were harvested by centrifugation at 4000 rpm for 20 minutes. The pellets were resuspended in 30 ml 0.1 M TRIS/HCl buffer pH 8.0 containing 0.5 % Brij-58. 330 µl of a 10 mg/ml lysozyme solution were added and the lysis was performed in a roller at 4°C until the samples were mostly clear.</P><P>20 µl of RNase free DNase was added and lysis continued for another 30 min. The bacterial debris was removed by centrifugation at 10.000 rpm for 10 min, followed by an ultracentrifugation step of the supernatant at 100.000 rpm for 90 min (4°C).</P><P>For the analysis of flagella via SDS-PAGE 1 ml with OD<SUB>600</SUB> 0.7 was prepared for every sample by centrifugation and subsequent heating to 95°C (10 min). Cell pellets were centrifuged and resuspended in 40  µL water and 10  µL SDS loading dye (5x).</P><P>Calculated size of flagellin is about 37 kDa, running slightly higher than the fourth band of the unstained protein marker (first line). Flagellin is present in the wildtype sample before and after cell lysis. Other proteins can be seen for PY-Strep before (higher than Hag) and after (lower than Hag) cell lysis. The two Hag constructs with the inserted D2 domain run a bit higher than wildtype flagellin and are present after cell lysis in a small amount.</P><P>Flagellin is present in the wildtype supernatant before and after cell lysis. Before lysis no flagellin can be seen in the supernatant of the other three strains. Differing proteins bands can be seen for these three strains in the supernatant after cell lysis.</P><P>A high amount of flagellin is present after cell lysis of wildtype <I>B. subtilis</I>. A thin band can be seen for Hag-D2-Cup before cell lysis and a thicker band for Hag-D2-Strep that does not fit the size of flagellin. The supernatant of the ultra-centrifugation step was transferred to a new falcon and the pellet was resuspended in 3 ml standard saline citrate buffer (pH 7.3) and both fractions were stored at 4°C overnight.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="15.10.2014">15.10.2014</A></H2><FIELDSET class="exp18"><LEGEND><A name="exp18.74b">18.74b Isolation of flagella from different <I>Bacillus subtilis</I> strains II</A></LEGEND><DIV class="exp-content"><P>20% ammonium sulphate were added to the pellet resulting from the ultracentrifugation resuspended in standard saline citrate buffer (pH 7.3). The obtained precipitant was analysed in an SDS PAGE gel.</P><P>A high amount of flagellin is present in the ultracentrifugation pellet of the wildtype cells after lysis. A mixture of proteins can be seen for Hag-D2-Strep after cell lysis. It can be assumed that flagellin hag-D2-Strep is partially unstable. Anyway, the analysis of presumably isolated flagella was continued.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="16.10.2014">16.10.2014</A></H2><FIELDSET class="exp24"><LEGEND><A name="exp23.54">23.54 Sequencing results</A></LEGEND><DIV class="exp-content"><P>Both plasmid have been sequenced and the correct inserts are in the iGEM backbone pSB1C3.</P></DIV></FIELDSET><FIELDSET class="exp23"><LEGEND><A name="exp23.55">23.55 Send Biobricks to registry</A></LEGEND><DIV class="exp-content"><P>Preparation of both bricks:</P><UL class="list"><LI>15 µl plasmid DNA (app. 200 ng/µl) in PCR cup</LI><LI>Tape with Biobrick number around the PCR cup</LI><LI>Parafilm wrapped around the whole Eppendorf cup</LI><LI>50 ml Falkon with Shipping number, team name and shipping date</LI></UL><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Part number</TH><TH scope="col">Name</TH></TR><TR><TH scope="col">Part:BBa_K1329004</TH><TD>synthetic catalyst part 1: tRNA scaffold</TD></TR><TR><TH scope="col">Part:BBa_K1329005</TH><TD>Synthetic catalyst part 2</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.75">18.75 Investigation the interaction between the isolated flagella and StrepDARPidin</A></LEGEND><DIV class="aim"><P>Aim: A possible interaction of the flagella with StrepDARPidin should be visualized by electron microscopy</P></DIV><DIV class="exp-content"><P>In order to visualize the interaction of the StrepDARPidin, the protein was incubated with the precipitated flagella and subsequently stained for electron microscopy.</P><P>Samples prepared:</P><UL class="list"><LI>1 = Hag-D2-Strep (1.2 mg/ml)</LI><LI>2 = Wildtype (2 mg/ml)</LI><LI>3 = PY-Strep (0.4 mg/ml)</LI><LI>4 = Hag-D2-Strep + StrepDARPidin (1:10)</LI><LI>5 = Wildtype+ StrepDARPidin (1:10)</LI><LI>6 = PY-Strep+ StrepDARPidin (1:10)</LI><LI>7 = StrepDARPidin (7 mg/ml)</LI></UL><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Number</TH><TH scope="col">Flagellin</TH><TH scope="col">StrepDARPidin</TH><TH scope="col">Buffer (Saline citrate)</TH></TR><TR><TD>1</TD><TD>50 µl</TD><TD>-</TD><TD>50 µl</TD></TR><TR><TD>2</TD><TD>200 µl</TD><TD>-</TD><TD>-</TD></TR><TR><TD>3</TD><TD>80 µl</TD><TD>-</TD><TD>20 µl</TD></TR><TR><TD>4</TD><TD>3.5 µl</TD><TD>10 µl</TD><TD>36.5 µl</TD></TR><TR><TD>5</TD><TD>17.5 µl</TD><TD>10 µl</TD><TD>50 µl</TD></TR><TR><TD>6</TD><TD>6 µl</TD><TD>10 µl</TD><TD>34 µl</TD></TR><TR><TD>7</TD><TD>-</TD><TD>10 µl</TD><TD>40 µl</TD></TR></TBODY></TABLE><P>Incubation on ice for 19 minutes before samples were stained for electron microscopy.</P></DIV></FIELDSET></DIV></FIELDSET></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Marburg:Project:Notebook:October">http://2014.igem.org/Team:Marburg:Project:Notebook:October</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Marburg:Project:Notebook:October" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Marburg:Project:Notebook:October" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Marburg:Project:Notebook:October&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Marburg:Project:Notebook:October&amp;oldid=347612" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>