<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Marburg_Project_Notebook_September"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Marburg:Project:Notebook:September">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Marburg:Project:Notebook:September&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Marburg:Project:Notebook:September&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Marburg:Project:Notebook:September&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Marburg:Project:Notebook:September" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Marburg:Project:Notebook:September</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="navbox"><NAV class="menu center first" aria-haspopup="true" aria-controls="p1"><UL id="p1" class="first"><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg" class="a">Home</A></LI><LI class="current"><A href="https://2014.igem.org/Team:Marburg:Project" class="b">Project</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Safety" class="c">Safety</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Team" class="d">Team</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Policy_Practices" class="e">Policy &amp; Practices</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Attributions" class="f">Attributions</A></LI></UL></NAV><NAV class="menu center second" aria-haspopup="true" aria-controls="p1"><UL id="p1" class="second"><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:Silver" class="d">SilverSURF</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:Tumor" class="c">CancerSURF</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:NRPS" class="e">RiboSURF</A></LI><LI class="current"><A href="https://2014.igem.org/Team:Marburg:Project:Notebook" class="f">Notebook</A></LI><LI class="last" aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:Parts" class="g">Parts</A></LI></UL></NAV></DIV><DIV class="mr_content"><H1>

Notebook: September

</H1><UL id="submenu"><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook">March</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:April">April</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:May">May</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:June">June</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:July">July</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:August">August</A></LI><LI class="current"><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:September">September</A></LI><LI><A href="#01.09.2014">01.09.2014</A></LI><LI><A href="#02.09.2014">02.09.2014</A></LI><LI><A href="#03.09.2014">03.09.2014</A></LI><LI><A href="#04.09.2014">04.09.2014</A></LI><LI><A href="#05.09.2014">05.09.2014</A></LI><LI><A href="#06.09.2014">06.09.2014</A></LI><LI><A href="#07.09.2014">07.09.2014</A></LI><LI><A href="#08.09.2014">08.09.2014</A></LI><LI><A href="#09.09.2014">09.09.2014</A></LI><LI><A href="#10.09.2014">10.09.2014</A></LI><LI><A href="#11.09.2014">11.09.2014</A></LI><LI><A href="#12.09.2014">12.09.2014</A></LI><LI><A href="#13.09.2014">13.09.2014</A></LI><LI><A href="#14.09.2014">14.09.2014</A></LI><LI><A href="#15.09.2014">15.09.2014</A></LI><LI><A href="#16.09.2014">16.09.2014</A></LI><LI><A href="#17.09.2014">17.09.2014</A></LI><LI><A href="#18.09.2014">18.09.2014</A></LI><LI><A href="#19.09.2014">19.09.2014</A></LI><LI><A href="#20.09.2014">20.09.2014</A></LI><LI><A href="#21.09.2014">21.09.2014</A></LI><LI><A href="#22.09.2014">22.09.2014</A></LI><LI><A href="#23.09.2014">23.09.2014</A></LI><LI><A href="#24.09.2014">24.09.2014</A></LI><LI><A href="#25.09.2014">25.09.2014</A></LI><LI><A href="#26.09.2014">26.09.2014</A></LI><LI><A href="#27.09.2014">27.09.2014</A></LI><LI><A href="#28.09.2014">28.09.2014</A></LI><LI><A href="#29.09.2014">29.09.2014</A></LI><LI><A href="#30.09.2014">30.09.2014</A></LI></UL><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:October">October</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:Methods">Methods</A></LI><DIV class="notebooky-entry"><H2 class="title"><A name="01.09.2014">01.09.2014</A></H2><FIELDSET class="exp23"><LEGEND><A name="exp23.2">23.2 Transformation of competent <I>E. coli</I> DH5α with pSB1C3 BB8 </A></LEGEND><DIV class="aim"><P>Aim: Isolate the plasmid in order to use the pSB1C3 backbone for the cloning of our biobricks.The contained brick will be digested out and the backbone will be used together with our inserts to clone the vectors.</P></DIV><DIV class="exp-content"><P>Standard <I>E. coli</I> transformation protocol was followed and cells were plated on LB-CM plates.</P></DIV></FIELDSET><FIELDSET class="exp"><LEGEND><A name="exp_20">Sequencing results of constructs from 29.08.2014</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Premix</TH><TH scope="col">Label-Nr.</TH><TH scope="col">Construct</TH><TH scope="col">Result</TH></TR><TR><TH scope="row">1</TH><TD>AGB0023<STRONG>512</STRONG></TD><TD>PCR Fragment StrepDARPidin I</TD><TD>fine</TD></TR><TR><TH scope="row">2</TH><TD>AGB0023<STRONG>513</STRONG></TD><TD>PCR Fragment StrepDARPidin II</TD><TD>fine</TD></TR><TR><TH scope="row">3</TH><TD>AGB0023<STRONG>514</STRONG></TD><TD>PCR Fragment D2-StrepII </TD><TD>fine</TD></TR><TR><TH scope="row">4</TH><TD>AGB0023<STRONG>515</STRONG></TD><TD>PCR Fragment StrepDARPidin I</TD><TD>fine</TD></TR><TR><TH scope="row">5</TH><TD>AGB0023<STRONG>516</STRONG></TD><TD>PCR Fragment StrepDARPidin II</TD><TD>fine</TD></TR><TR><TH scope="row">6</TH><TD>AGB0023<STRONG>517</STRONG></TD><TD>PCR Fragment D2-StrepII</TD><TD>fine</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.65">18.65 Test digest of isolated clones from 18.63</A></LEGEND><DIV class="aim"><P>Aim: Checking insertion of StrepDARPidin into pET16b </P></DIV><DIV class="exp-content"><P>The isolated plasmids from the positive clones were digested with <I>Nco</I>I/ <I>Xho</I>I in order to see if the 900 bp were flipping out of the pET16b backbone.
     </P><TABLE width="100%" border="1"><TBODY><TR><TH>Mastermix (µL)</TH><TH>Mix per samle (µL)</TH></TR><TR><TH>pET16b-Insert (approx. 50-80 ng/ µL)	</TH><TD>-</TD><TD>8</TD></TR><TR><TH scope="row">CutSmart  10x</TH><TD>10</TD><TD>-</TD></TR><TR><TH scope="row"><I>Nco</I>I</TH><TD>0.5</TD><TD>-</TD></TR><TR><TH scope="row"><I>Xho</I>I</TH><TD>0.5</TD><TD>-</TD></TR><TR><TH>Water</TH><TD>9</TD><TD>-</TD></TR><TR><TH scope="row">Mastermix</TH><TD>-</TD><TD>2</TD></TR><TR><TH scope="row">Total </TH><TD>20</TD><TD>10</TD></TR></TBODY></TABLE><P> Incubation at 37°C for 40 min.</P><P>The gel showed that all picked clones were positive. Clone 17 was sent was sequencing and transformed into <I>E. coli</I> BL21(DE3).</P></DIV></FIELDSET><FIELDSET class="exp"><LEGEND><A name="exp_21">Sequencing results of constructs from 01.09.2014</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Premix</TH><TH scope="col">Label-Nr.</TH><TH scope="col">Construct</TH><TH scope="col">Primer</TH></TR><TR><TH scope="row">1</TH><TD>AGB0023<STRONG>518</STRONG></TD><TD>piGEM-027 I1 (pMAD-D2-Strep)</TD><TD>Flo96 D2_3 fw</TD></TR><TR><TH scope="row">2</TH><TD>AGB0023<STRONG>519</STRONG></TD><TD>piGEM-027 I3 (pMAD-D2-Strep)</TD><TD>Flo96 D2_3 fw</TD></TR><TR><TH scope="row">3</TH><TD>AGB0023<STRONG>520</STRONG></TD><TD>piGEM-027 I6 (pMAD-D2-Strep)</TD><TD>Flo96 D2_3 fw</TD></TR><TR><TH scope="row">4</TH><TD>AGB0023<STRONG>521</STRONG></TD><TD>piGEM-027 II2 (pMAD-D2-Strep)</TD><TD>Flo96 D2_3 rv</TD></TR><TR><TH scope="row">5</TH><TD>AGB0023<STRONG>551</STRONG></TD><TD>piGEM-028 clone 7</TD><TD>T7</TD></TR><TR><TH scope="row">6</TH><TD>AGB0023<STRONG>552</STRONG></TD><TD>piGEM-028 clone 7</TD><TD>T7 term</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.66">18.66 Expression-Test with transformed <I>E. coli</I> BL21 (DE3)</A></LEGEND><DIV class="aim"><P>Aim: Checking the overexpression of pET24d-StrepDARPidin </P></DIV><DIV class="exp-content"><P>In order to check the expression of the StrepDARPidin a colony containing piGEM-028 (pET16b-StrepDARPidin) was used to inoculate 20 mL of LB-Amp. The culture incubated at 37°C until an OD of 0,7.</P><P>A approx. 1 mL preinduction probe (PI) was taken (0,7 (favoured OD) : 0,7 (actual OD) = 1 mL). After that the cultures were induced with 20 µL IPTG for 3h.</P><P>An induction probe (I) was taken (320 µL with an OD of 2,2; 350 µL with an OD of 2).</P><P>PI and I were centrifuged at 14000 rpm for 1 min, the supernatant was discarded and the pellet resuspended in 60 µL water and 40 µL SDS-Buffer.</P><P>The four probes were analysed on an SDS-PAGE gel with 10 µL volume per probe.</P><P>The gel showed a positive overexpression of a protein at approx. 30 kDa which fits the calculated molecular mass.</P></DIV></FIELDSET><FIELDSET class="exp22"><LEGEND><A name="exp22.5">22.5 Repeated amplification of killswitch module 2 and flanks</A></LEGEND><DIV class="aim"><P>Aim: cleaner amplification of KSII, since there always were many unspecific bands and the expected band was very weak. The same went for the flanks.</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix</TH><TH scope="col"><I><I>amyE</I></I> FlankI</TH><TH scope="col"><I><I>amyE</I></I> FlankII</TH><TH scope="col"><I><I>lacA</I></I> FlankI</TH><TH scope="col"><I><I>lacA</I></I> FlankII</TH><TH scope="col"><I>amyE</I>-<I>yvyD</I> FlankII</TH><TH scope="col">KSII</TH></TR><TR><TH scope="row">Template (Δ51 fac gDNA 1:10)</TH><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>-</TD></TR><TR><TH scope="row">Template (gBlock DNA KSII)</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>0.5</TD></TR><TR><TH scope="row">Primer iGEM-34</TH><TD>1</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-35</TH><TD>1</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-36 </TH><TD>-</TD><TD>1</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-37</TH><TD>-</TD><TD>1</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-40</TH><TD>-</TD><TD>-</TD><TD>1</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-41</TH><TD>-</TD><TD>-</TD><TD>1</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-42</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>1</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-43</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>1</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-48</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-37</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-44</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>1</TD></TR><TR><TH scope="row" style="height: 25px">Primer iGEM-45</TH><TD style="height: 25px">-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">dNTPs</TH><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">5x HF-buffer </TH><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD></TR><TR><TH scope="row">Phusion polymerase</TH><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>34</TD><TD>34</TD><TD>34</TD><TD>34</TD><TD>34</TD><TD>34</TD></TR><TR><TH scope="row">Total  (µl) </TH><TD>50</TD><TD><STRONG>50</STRONG></TD><TD><STRONG>50</STRONG></TD><TD><STRONG>50</STRONG></TD><TD><STRONG>50</STRONG></TD><TD><STRONG>50</STRONG></TD></TR></TBODY></TABLE><P>The amplification of KSII did not work: some unspecific bands were visible but not of the expected size. The amplification of flank 1 <I><I>amyE</I></I> fw, flank 3 <I>lacA</I> fw and flank 4 <I>lacA</I>  rev was successful but yielded in a low amount of PCR product.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.79">13.79 Transformation of <EM>Bacillus subtilis</EM> PY79 with the Nose-plasmids</A></LEGEND><DIV class="aim"><P>Aim: insertion of the Nose-plasmids into <EM>B. subtilis</EM> in order to test the promoters.</P></DIV><DIV class="exp-content"><P>The plasmids of 13.78 were isolated at the weekend. Frozen and competent 
	   <EM><I>B. subtilis</I></EM>cells were thawed without ice. 15 µl of each plasmid were given to the cells that were incubated at 37°C for 0.5 hours. After regeneration the cells were plated on LB-Cm plates (5 g/µl) and were incubated at 30°C for several days.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="02.09.2014">02.09.2014</A></H2><FIELDSET class="exp23"><LEGEND><A name="exp23.3">23.3 Inoculation of transformants</A></LEGEND><DIV class="aim"><P>Aim: Grow cultures for the isolation of plasmid psB1C3 BB8
transformants were inoculated in liquid LB-CM (10:30) for plasmid isolation.
</P></DIV></FIELDSET><FIELDSET class="exp23"><LEGEND><A name="exp23.4">23.4 Plasmid isolation and digest of pSB1C3, Hag-<I>Kpn</I>I and StrepDARPidin</A></LEGEND><DIV class="aim"><P>Aim: Gain digested plasmid and inserts for ligation of our first two biobricks</P></DIV><DIV class="exp-content"><P>Test digest with the isolated plasmid was performed with the corresponding enzymes (<I>Eco</I>RI and <I>Pst</I>I) over night. Besides 600 ng of the already existing PCR fragments Hag-<I>Kpn</I>I and StrepDARPidin were digested with the same enzymes.</P><P><U>Concentrations:</U></P><P>Hag-<I>Kpn</I>I 130 ng/µl</P><P>Strep-DARPidin 55 ng/µl after purification</P><P>psB1C3 130 ng/µl after plasmid isolation</P><P><U>Digest:(20µl reaction mix)</U></P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Digestion Mix</TH><TH scope="col">StrepDARPidin (600 ng) (µl)</TH><TH scope="col">Hag-<I>Kpn</I>I (600 ng) (µl)</TH><TH scope="col">pSB1C3 (2 µg) (µl)</TH></TR><TR><TH scope="row">Water</TH><TD>10</TD><TD>5</TD><TD>13</TD></TR><TR><TH scope="row">DNA</TH><TD>7</TD><TD>12</TD><TD>4</TD></TR><TR><TH scope="row">CutSmart  Buffer</TH><TD>2.0</TD><TD>2.0</TD><TD>2.0</TD></TR><TR><TH scope="row"><I>Eco</I>RI</TH><TD>0.5</TD><TD>0.5</TD><TD>0.5</TD></TR><TR><TH scope="row"><I>Pst</I>I</TH><TD>0.5</TD><TD>0.5</TD><TD>0.5</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp"><LEGEND>Sequencing Results of constructs from 01.09.2014</LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Premix</TH><TH scope="col">Label-Nr.</TH><TH scope="col">Construct</TH><TH scope="col">Result</TH></TR><TR><TH>1</TH><TD>AGB0023<STRONG>518</STRONG></TD><TD>piGEM-027 I1 (pMAD-D2-Strep)</TD><TD>fine</TD></TR><TR><TH>2</TH><TD>AGB0023<STRONG>519</STRONG></TD><TD>piGEM-027 I3 (pMAD-D2-Strep)</TD><TD>fine</TD></TR><TR><TH>3</TH><TD>AGB0023<STRONG>520</STRONG></TD><TD>piGEM-027 I6 (pMAD-D2-Strep)</TD><TD>fine</TD></TR><TR><TH scope="row">4</TH><TD>AGB0023<STRONG>521</STRONG></TD><TD>piGEM-027 II2 (pMAD-D2-Strep)</TD><TD>fine</TD></TR><TR><TH scope="row">5</TH><TD>AGB0023<STRONG>551</STRONG></TD><TD>piGEM-028 clone 7</TD><TD>fine</TD></TR><TR><TH scope="row">6</TH><TD>AGB0023<STRONG>552</STRONG></TD><TD>piGEM-028 clone 7</TD><TD>fine</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp22"><LEGEND><A name="exp22.6">22.6 </A>Repeated fusion-PCR of the Killswitch module I with the flanks <I><I>amyE</I></I>  I/II</LEGEND><DIV class="aim"><P>Aim: Fuse the <I><I>amyE</I></I>  flanks to killswitch module I.</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">KSI (µl)</TH></TR><TR><TH scope="row" style="height: 25px">Template: KSI (approx.5 ng/µl) </TH><TD style="height: 25px">1</TD></TR><TR><TH scope="row"><I><I>amyE</I></I>  flank I (5 ng/µl)</TH><TD>1</TD></TR><TR><TH scope="row"><I><I>amyE</I></I>  flank II (approx. 5 ng/µl)</TH><TD>1</TD></TR><TR><TH scope="row">Primer iGEM034 (1:40)</TH><TD>1</TD></TR><TR><TH scope="row">Primer piGEM037 (1:40)</TH><TD>1</TD></TR><TR><TH scope="row">HF Buffer 5x</TH><TD>10</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>1</TD></TR><TR><TH scope="row">dNTPs</TH><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>33</TD></TR><TR><TH scope="row">Total</TH><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Fusion PCR Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>20 sec</TD></TR><TR><TH scope="row">3</TH><TD>60</TD><TD>20 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>2 min </TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>35x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The bands were at a height of approx. 1500 bp which is too small for the expected fragment of 1847 bp.</P></DIV></FIELDSET><FIELDSET class="exp22"><LEGEND><A name="exp22.7">22.7 </A>Gibson Assembly of KS module I with the flanks <I><I>amyE</I></I>  I/II</LEGEND><DIV class="aim"><P>Aim: Fuse the <I><I>amyE</I></I>  flanks to killswitch module I.</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Gibson with vector (µL)</TH><TH scope="col">Gibson without vector (µl)</TH><TH scope="col">Control (µl)</TH></TR><TR><TH scope="row">pMAD</TH><TD>2</TD><TD>-</TD><TD>2</TD></TR><TR><TH scope="row">KS module I (approx. 5 ng/µl)</TH><TD>1</TD><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Flank <I><I>amyE</I></I>  I (approx. 5 ng/µl)</TH><TD>1</TD><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Flank <I><I>amyE</I></I>  II (approx. 5 ng/µl)</TH><TD>1</TD><TD>2</TD><TD>-</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O</TH><TD>-</TD><TD>1</TD><TD>3</TD></TR><TR><TH scope="row">Gibson mix</TH><TD>15</TD><TD>15</TD><TD>15</TD></TR><TR><TH scope="row">Total</TH><TD>20</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><P>The reactions were incubated at 50°C for an hour and used to transform <I>E. Coli</I> XL1-Blue, plated out on LB-Amp. An additional reaction was started without pMAD in a lower (1 µl) and a higher (2 µl) amount of <I>amyE</I>  flank I. These reactions were used to do a PCR reaction with the primers piGEM034 and piGEM037 to test if the assembled fragment is present in the reaction. The expected bands should have a size of 1847 bp but the fragment visible is only approx.1300 bp big. It seems like the flank I is missing but then the fragment should not have been amplified, since the primers were chosen to include both flanks.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="03.09.2014">03.09.2014</A></H2><FIELDSET class="exp23"><LEGEND><A name="exp23.5">23.5 Gelextraction of the digested plasmid and fragments in a 1% agarose gel</A></LEGEND><DIV class="aim"><P>Aim: The purified plasmids/inserts will afterwards be used for ligations.</P></DIV><DIV class="exp-content"><P>Information: iBB8 has a size of 1461 bp, vector 2000bp, Inserts Hag-<I>Kpn</I>I and StrepDARPidin 1000 bp</P><P>Image of the gelextraction, as a marker the 1kb gene ruler was used, 1 = StrepDARPidin, 2= Hag-<I>Kpn</I>I, 3 and 4 = pSB1C3</P></DIV></FIELDSET><FIELDSET class="exp23"><LEGEND><A name="exp23.6">23.6 Ligation of digested inserts StrepDARPidin and Hag-<I>Kpn</I>I into pSB1C3and transformation into chemically competent E. coli XL-1-blue</A></LEGEND><DIV class="aim"><P>Aim: Ligate the two inserts into pSB1C3 backbone in order to create the first two biobricks</P></DIV><DIV class="exp-content"><P><U>Concentrations:</U></P><P>pSB1C3= 18 ng/µl</P><P>StrepDARPidin= 5 ng/µl</P><P>Hag-<I>Kpn</I>I= 13 ng/µl</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Ligation Mix</TH><TH scope="col">StrepDARPidin </TH><TH scope="col">Hag-<I>Kpn</I>I </TH><TH scope="col">Control</TH></TR><TR><TH scope="row">T-4 Ligase</TH><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">T-4 ligase Buffer</TH><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR><TH scope="row">Water</TH><TD>-</TD><TD>-</TD><TD>14</TD></TR><TR><TH scope="row">Insert </TH><TD>14.9</TD><TD>12.5</TD><TD>0</TD></TR><TR><TH scope="row">Vector</TH><TD>2.1</TD><TD>4.5</TD><TD>3</TD></TR></TBODY></TABLE><P>Ligation at RT for 90 minutes</P><P>After the ligation the whole reaction mix was used to transform chemically competent <I>E. coli</I> XL-1-blue cells (standard protocol). Cells were plated on LB-Cm plates and incubated at 37°C over night.</P></DIV></FIELDSET><FIELDSET class="exp22"><LEGEND><A name="exp22.8">22.8 Repeated amplification of KSI and flank I</A></LEGEND><DIV class="aim"><P>Aim: amplification KSI and flank I to have correct fragments for Gibson / Fusion PCR</P></DIV><DIV class="exp-content"><P>Since the Gibson assembly of the single parts without the vector (22.7) 
	and the Fusion PCR yielded no positive results, the KSI should be amplified 
	once again to assure usage of the correct and pure fragments for the 
	FusionPCR or Gibson assembly. </P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">KS module I</TH><TH scope="col">Flank I</TH></TR><TR><TH scope="row">Template (Δ51 fac gDNA 1:10)</TH><TD>-</TD><TD>2</TD></TR><TR><TH scope="row">Template (gBlock DNA 1:10)</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-38 (1:50)</TH><TD>2.5</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-39 (1:50)</TH><TD>2.5</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-34 (1:50)</TH><TD>-</TD><TD>2.5</TD></TR><TR><TH scope="row">Primer iGEM-35 (1:50) </TH><TD>-</TD><TD>2.5</TD></TR><TR><TH scope="row">5x HF-Buffer </TH><TD>10 </TD><TD>10</TD></TR><TR><TH scope="row">dNTPs (40 mM)</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>32</TD><TD>31</TD></TR><TR><TH scope="row">Total</TH><TD>50</TD><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col"> PCR Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>20 sec</TD></TR><TR><TH scope="row">3</TH><TD>58.2, 61.3, 63.5</TD><TD>20 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>50 sec </TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The gel showed the expected band for the killswitch module 1 but only one band for flank 1 in the reaction with the annealing temperature of 58°C. The remaining reactions for KSI were pooled and purified with a Gel Ex kit. Since the only band for the flank 1 is visible at 58°C and not above another PCR was prepared with lower annealing temperatures but the same reaction mix as above.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col"> PCR Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>20 sec</TD></TR><TR><TH scope="row">3</TH><TD>58.2, 61.3, 63.5</TD><TD>20 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>50 sec </TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>infinite</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.80">13.80 repeated transformation of <I>B. subtilis</I> PY79 and <I>E. coli</I> DHα with the Nose-plasmids</A></LEGEND><DIV class="aim"><P>Aim: insertion of the Nose-plasmids into <I>B. subtilis</I> in order to test the promoters and into E. coli for multiplication of the plasmids </P></DIV><DIV class="exp-content"><P>All plasmids (from positive clones 13.78) were again used to transform 
	   <I>B. subtilis</I> PY79, since the former plates seemed to be contaminated by other organisms. <I>E. coli DH5µ</I> was transformed with the rest of the plasmids to multiply it for later purposes.</P></DIV></FIELDSET><FIELDSET class="exp22"><LEGEND><A name="exp22.9">22.9 colony PCR of clones of the Gibson assembly with KSI and flanksI and II</A></LEGEND><DIV class="aim"><P>Aim: check for positive clones</P></DIV><DIV class="exp-content"><P>Three colonies were grown on the plate with the Gibson assembly transformed XLI-Blue cells. The control also contained three colonies. However, the colonies of the Gibson plate were tested by colony PCR with the primer surrounding the flanks of the connected KSI module, since no primer were available which target regions in the vector pMAD.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix colony PCR</TH><TH scope="col">Master mix for 4 reactions (µl)</TH><TH scope="col">Single reaction </TH></TR><TR><TH scope="row">Template</TH><TD>-</TD><TD>small part of colony</TD></TR><TR><TH scope="row">Primer iGEM034 (1:50)</TH><TD>4</TD><TD>1</TD></TR><TR><TH scope="row">Primer iGEM037 (1:50) </TH><TD>4</TD><TD>1</TD></TR><TR><TH scope="row" style="height: 25px">5x HF-Buffer </TH><TD style="height: 25px">16</TD><TD style="height: 25px">16</TD></TR><TR><TH scope="row">dNTPs (40 mM)</TH><TD>2</TD><TD>0.5</TD></TR><TR><TH scope="row">Phusion DNA polymerase 1:10</TH><TD>2</TD><TD>0.5</TD></TR><TR><TH scope="row">Water</TH><TD>52</TD><TD>13</TD></TR><TR><TH scope="row">Total</TH><TD>80</TD><TD>20</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">colony PCR Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time (min/sec) KSI</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>20 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>20 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1:20 min </TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The Gibson assembly of KSI with flanks I and II did not work, but the flank I was amplified well.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="04.09.2014">04.09.2014</A></H2><FIELDSET class="exp23"><LEGEND><A name="exp23.7">23.7 Colony PCR on presumably positive transformants + inoculation for plasmid preparation</A></LEGEND><DIV class="aim"><P>Aim: Analyze the transformants for the insertion of the fragments into the plasmid</P></DIV><DIV class="exp-content"><P>Colonies could be detected on both transformation plates, whereas no colonies grew in the ligation control plate. Therefore a colony PCR will be performed in order to analyze the transformants for the insertion of the fragments into the plasmid. 5 clones from both plates were chosen for the colony PCR and inoculated in liquid LB-Cm for plasmid isolation in the evening (in case positive clones can be detected in the colony PCR).</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Colony PCR</TH><TH scope="col">StrepDARPidin MM 6x (µl)</TH><TH scope="col">Hag-<I>Kpn</I>I MM 6x (µl)</TH></TR><TR><TH scope="row">Water</TH><TD>84</TD><TD>84</TD></TR><TR><TH scope="row">PhusionBuffer (5x)</TH><TD>24</TD><TD>24</TD></TR><TR><TH scope="row">Phusion</TH><TD>3</TD><TD>3</TD></TR><TR><TH scope="row">Primer 1</TH><TD>3(51)</TD><TD>3(53)</TD></TR><TR><TH scope="row">Primer 2</TH><TD>3(53)</TD><TD>3(54)</TD></TR><TR><TH scope="row">dNTPs</TH><TD>3</TD><TD>3</TD></TR><TR><TH scope="row">Colony</TH><TD>1</TD><TD>1</TD></TR></TBODY></TABLE><P>Saved colony PCR program in the PCR cycler was used with 1 minute elongation time.</P><P>Colony PCR negative, therefore plasmids were isolated and digested with the <I>Eco</I>RI and <I>Pst</I>I.</P></DIV></FIELDSET><FIELDSET class="exp23"><LEGEND><A name="exp23.8">23.8 Test restriction of presumably positive plasmids after plasmid isolation</A></LEGEND><DIV class="aim"><P>Aim: Analyze the plasmids for the insertion of the fragments into the plasmid</P></DIV><DIV class="exp-content"><P>Restriction: 4 µl plasmid in a total 10 µl restriction mix</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Amount (µL)</TH></TR><TR><TH scope="row">Plasmid</TH><TD>4</TD></TR><TR><TH scope="row">Water</TH><TD>4</TD></TR><TR><TH scope="row">CutSmart  Buffer</TH><TD>1</TD></TR><TR><TH scope="row"><I>Eco</I>RI</TH><TD>0.5</TD></TR><TR><TH scope="row"><I>Pst</I>I</TH><TD>0.5</TD></TR></TBODY></TABLE><P>Restriction positive (2kb plasmid and 1kb insert), one clone each will be send for sequencing.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.67">18.67 StrepDARPidin expression in <I>E. coli</I> BL21 (DE3)</A></LEGEND><DIV class="aim"><P>Aim: Preculture for upcoming purification</P></DIV><DIV class="exp-content"><P>3 L LB medium were prepared with 1 mL ampicillin each and inoculated with some clones from the B21 transformation plate. The cultures were induced with 50 ml lactose solution (12,5g/ 50 mL) each and incubated at 30°C over night.</P><P>In order to check the expression of the StrepDARPidin a colony containing piGEM-028 (pET16b-StrepDARPidin) was used to inoculate 20 mL of LB-Amp.</P></DIV></FIELDSET><FIELDSET class="exp20"><LEGEND><A name="exp20.10">22.10 Gradient fusion PCR to amplify KSI with flanks</A></LEGEND><DIV class="aim"><P>Aim: fuse killswitch module I and flanks I <I>amyE</I> fw and II <I>amyE</I>  rev together</P></DIV><DIV class="exp-content"><P>The fragments were all amplified and purified. This time a gradient was performed with an initial step of 50°C for 0.5 hours (similar to a Gibson assembly)</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix fusion/gibson PCR</TH><TH scope="col">Master mix for 10 reactions (µl)</TH><TH scope="col">Single reaction (µl)</TH></TR><TR><TH scope="row">Killswitch module I (approx. 10 ng/µl)</TH><TD>10</TD><TD>1</TD></TR><TR><TH scope="row">Flank I <I>amyE</I>  for</TH><TD>10</TD><TD>1</TD></TR><TR><TH scope="row">Flank II <I>amyE</I>  rev </TH><TD>10</TD><TD>1</TD></TR><TR><TH scope="row">Primer iGEM034 (1:50)</TH><TD>62.5</TD><TD>6.25</TD></TR><TR><TH scope="row">Primer iGEM037 (1:50) </TH><TD>62.5</TD><TD>6.25</TD></TR><TR><TH scope="row">5x HF-Buffer </TH><TD>100</TD><TD>10</TD></TR><TR><TH scope="row">dNTPs (40 mM)</TH><TD>10</TD><TD>1</TD></TR><TR><TH scope="row">Phusion DNA polymerase 1:10</TH><TD>10</TD><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>225</TD><TD>22.5</TD></TR><TR><TH scope="row">Total</TH><TD>500</TD><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">colony PCR Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time (min/sec) KSI</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>20 sec</TD></TR><TR><TH scope="row">3</TH><TD>52-62</TD><TD>20 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>2:10 min </TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The fusion-PCR worked and the bands had the expected size of approx.1800 bp.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="05.09.2014">05.09.2014</A></H2><FIELDSET class="exp23"><LEGEND><A name="exp23.9">23.9Sequencing of pSB1C3 StrepDARPidin and pSB1C3 Hag-<I>Kpn</I>I</A></LEGEND><DIV class="exp-content"><P>iGEM Primer in the list of last year's team number 37 and 38 (Sample premix)</P><P>AGB0023-642= StrepDARPidinFw (Clone 1)</P><P>AGB0023-643=StrepDARPidinRv</P><P>AGB0023-644= Hag-<I>Kpn</I>I Fw (Clone 4)</P><P>AGB0023-645= Hag-<I>Kpn</I>I Rv</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.68">18.68 StrepDARPidin expression in <I>E. coli</I> BL21(DE3) and purification</A></LEGEND><DIV class="exp-content"><P>The 3 x 1 liter LB culture which were induced overnight with lactose were harvested and the pellets resuspended in 15 mL Buffer A. The cell suspension was cracked with the micro fluidizer. The lysate was centrifuged at 20000 rpm for 20 min and purified with a Ni-NTA Column.</P><P>The following steps were performed for the Ni-NTA Column:</P><UL class="list"><LI>equilibration with Buffer A 10 min</LI><LI>taking 40µL supernatant  (load)+ 10 µL SDS-Buffer - <EM>L-Probe</EM></LI><LI>50 mL load on column</LI><LI>taking 40 µL of flow through + 10 µL SDS-buffer - <EM>FT-Probe</EM></LI><LI>first washing with 25 ml Buffer A (half the load)</LI><LI>taking 40 µL of washing flow through + 10 µL SDS-Buffer - <EM>W-Probe</EM></LI><LI>equilibrate with Buffer B (output pipe off the column into glas, input into B)</LI><LI>hanging pipe on column, 20 ml Evolution-<EM> E</EM></LI><LI>taking 40 µL of Elution 1-6 + 10 µL SDS-Buffer<EM> E-probe</EM></LI><LI><EM>SDS-PAGE analysis</EM></LI></UL><P>regeneration of column:</P><UL class="list"><LI>10min water</LI><LI>10min EDTA</LI><LI>10min water</LI><LI>10min NiSO<SUB>4</SUB></LI><LI>10min water</LI></UL><P>Meanwhile the elution was transferred in a concentrator column which was centrifuged at 4000 rpm until the volume in the filter was 1.5 mL for injection into the gel filtration station.</P><P>The gel showed that there was not the desired StrepDARPidin loaded on the column. A SDS-PAGE probe from the pellet showed us that the pellet contained the protein.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.69">18.69 New StrepDARPidinexpression in <I>E. coli</I> BL21 (DE3) </A></LEGEND><DIV class="exp-content"><P>We considered that the StrepDARPidin might be in inclusion bodies because of the Streptavidin part and the codon optimization. So we tried expression overnight with 0.1 mM IPTG of an 100 mL culture at OD 0,7. We induced as well with lactose overnight as a control.</P><P>The 4 cultures were incubated at 20°C and 30°C in the shaker overnight.</P></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.12">19.12 Amplification of Hag-D2-Cup/-Strep</A></LEGEND><DIV class="aim"><P>Aim: amplification of the Hag-D2-Cup/ -Strep for cloning into pET24d in order to prepair crystallization.</P></DIV><DIV class="exp-content"><P>piGEM-026 and -027 were used as a PCR template to amplify Hag-D2 constructs for cloning into pET24d in order to express the flagellin modification for crystallization. Fragments the size of approx. 1450 for Hag-D2-Cup and 1350 for Hag-D2-Strep were expected.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix</TH><TH scope="col">PCR D2-Cup I (µL)</TH><TH scope="col">PCR D2-Cup II (µL)</TH><TH scope="col">PCR D2-Strep  (µL)</TH></TR><TR><TH scope="row">piGEM-026 (1:10 - 10 ng/µL) </TH><TD>1</TD><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">piGEM-027 (1:10- 10 ng/µL)</TH><TD>-</TD><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Flo96 D2_3 fw</TH><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Flo97 D2_3 rv)</TH><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Phusion DNA-polymerase </TH><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row"> HF-Buffer 5x</TH><TD>10</TD><TD>10</TD><TD>10</TD></TR><TR><TH scope="row">dNTPs </TH><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>35</TD><TD>35</TD><TD>35</TD></TR><TR><TH scope="row">Total</TH><TD>50</TD><TD>50</TD><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">PCR Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>90 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>31x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4 min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The PCR Probes were purified and pooled with the Omega Gel Extraction Kit due to the enzymatic reaction purification protocol. Final concentrations:</P><UL>Hag-D2-Cup: 41 ng/ µL</UL><UL>Hag-D2-Strep: 25 ng/ µL</UL></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.13">19.13 Restriction digest of PCR Fragments Hag-D2-Constructs</A></LEGEND><DIV class="aim"><P>Aim: digest with <I>Nco</I>I and Bam of the Hag-D2 PCR product for cloning into the expression vector pET24d <I>Nco</I>I/<I>Bam</I>HI cut</P></DIV><DIV class="exp-content"><P>The PCR products from 19.12 were digested with <I>Nco</I>I and <I><I>Bam</I>HI</I>HI to get the restriction sites for cloning into pET24d <I>Nco</I>I/<I>Bam</I>HI cut. </P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">PCR PCR D2-Cup (µL)</TH><TH scope="col">PCR D2-Strep (µL)</TH></TR><TR><TH scope="row">Hag-D2-Cup (41 ng/µL) </TH><TD>30</TD><TD>-</TD></TR><TR><TH scope="row">Hag-D2-Strep(25 ng/µL)</TH><TD>-</TD><TD>30</TD></TR><TR><TH scope="row">CutSmart   10x</TH><TD>3.4</TD><TD>3.4</TD></TR><TR><TH scope="row"><I>Nco</I>I</TH><TD>0.25</TD><TD>0.25</TD></TR><TR><TH scope="row"><I>Xho</I>I</TH><TD>0.25</TD><TD>0.25</TD></TR><TR><TH scope="row">Water</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Total</TH><TD>34</TD><TD>34</TD></TR></TBODY></TABLE><P>Incubation overnight at room temperature.</P></DIV></FIELDSET><FIELDSET class="exp22"><LEGEND><A name="exp22.11">22.11 Gibson assembly of pMAD <I>Nco</I>I/<I><I>Bam</I>HI</I>HI with the fused KSI + flanks <I>amyE</I> fw and rev</A></LEGEND><DIV class="aim"><P>Aim: clone the fragment into the vector pMAD for integration in Bacillus chromosome</P></DIV><DIV class="exp-content"><P>The fragment of the fusion PCR (22.10) was separated on a 1% agarose gel, digested out and purified with a Gel Ex kit. The already digested pMAD plasmid was used to perform a Gibson assembly.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Gibson with vector (µL)</TH><TH scope="col">Control (µL) (only digested pMAD)</TH></TR><TR><TH scope="row">pMAD <I>Nco</I>I/<I><I>Bam</I>HI</I>HI (47.6 ng/µL)</TH><TD>2.1</TD><TD>2.1</TD></TR><TR><TH scope="row">KS part I (88.1 ng/µL)</TH><TD>0.9</TD><TD>-</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O</TH><TD>2</TD><TD>2.9</TD></TR><TR><TH scope="row">Gibson mix</TH><TD>15</TD><TD>15</TD></TR><TR><TH scope="row">Total</TH><TD>20</TD><TD>20</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.81">13.81 Preparation of media and strains for another <I>B. subtilis</I> transformation</A></LEGEND><DIV class="aim"><P>Aim: Since the transformation of <I>B. subtilis</I> with the SPC/SPII method and the Nose-plasmids did not work the Low Salt/High Salt method should be tested.</P></DIV><DIV class="exp-content"><P>The protocol and the strains were provided by AG Bremer of the microbiology department of the Philipps Universität Marburg. All media were prepared as possible (methods) and the strains <I>B. subtilis</I> 168 and <I>B. subtilis</I> JH642 were streaked on LB-Agar and incubated at 37°C.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="06.09.2014">06.09.2014</A></H2><FIELDSET class="exp18"><LEGEND><A name="exp18.70">18.70 New StrepDARPidin expression in <I>E. coli</I> BL21 (DE3)</A></LEGEND><DIV class="exp-content"><P>The cultures from the day before were spinned down at 4000 rpm for 20 min. The pellets were resuspended in 10 ml Buffer A and cracked with the micro fluidizer. The lysate was centrifuged at 20000 rpm for 20 min and purified with a Ni-NTA Beads from Qiagen according to the protocol. Probes from the pellet, supernatant and Elution wereanalysed on an SDS-PAGE gel:</P><P>The gels showed that the protein remains in the pellet no matter which attempt.</P><P>For further experiments for the isolation of the protein in inclusion bodies new expression cultures from a freshly transformed BL21(DE3) strain with piGEM-028 were used to inoculate 3 L expression culture with lactose overnight.3 L LB were prepared with 1 mL ampicillin each. The cultures were induced with 50 ml lactose solution (12,5g/ 50 mL) each and incubated at 30°C overnight. </P></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.14">19.14 Ligation of digested Hag-D2-Cup/ -Strep constructs with pET24d <I>Nco</I>I <I>Bam</I>HIcut</A></LEGEND><DIV class="aim"><P>Aim: cloning of expression vector with Hag-modifications</P></DIV><DIV class="exp-content"><P>The <I>Nco</I>I/<I>Bam</I>HI digested PCR fragments from 19.13 were ligated into predigested pET24d. The 20 µL attempts were transformed into <I>E. coli</I> XL1-Blue after inactivation (10min at 65°C) and selected on LB- Amp plates incubated overnight at 37°C.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Ligation I: pet-Hag-D2-Cup(µL)</TH><TH scope="col">Ligation II: pet-Hag-D2-Strep (µL)</TH><TH scope="col">Control (µL)</TH></TR><TR><TH scope="row"><I>Nco</I>I<I>Bam</I>HI digested Hag-D2-Cup</TH><TD>1</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row"><I>Nco</I>I<I>Bam</I>HI digested Hag-D2-Strep</TH><TD>-</TD><TD>1.5</TD><TD>-</TD></TR><TR><TH scope="row">pET24d <I>Nco</I>I/<I>Bam</I>HI(6,5 ng/µL)</TH><TD>5</TD><TD>5</TD><TD>5</TD></TR><TR><TH scope="row">Ligation Buffer 10x</TH><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR><TH scope="row">Ligase</TH><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O</TH><TD>11</TD><TD>10.5</TD><TD>12</TD></TR><TR><TH scope="row">Total</TH><TD>20</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="07.09.2014">07.09.2014</A></H2><FIELDSET class="exp19"><LEGEND><A name="exp19.14">19.14 Ligation of digested Hag-D2-Cup/ -Strep constructs with pET24d <I>Nco</I>I/ <I>Bam</I>HI cut</A></LEGEND><DIV class="aim"><P>Aim: cloning of expression vector with Hag-modifications.</P></DIV><DIV class="exp-content"><P>The control plate showed no colonies in comparison the ligation plates I and II over 15 clones. 6 Clones per plate were picked for a inoculation of minipreps for a test digest next day.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.70">18.70 3rd StrepDARPidin expression in <EM>E. coli BL21 (DE3)</EM></A></LEGEND><DIV class="aim"><P>Aim: Preculture for upcoming purification tests</P></DIV><DIV class="exp-content"><P>The cultures were centrifuged at 4000 rpm. The pellets were resuspended in 10 ml Buffer A and centrifuged down at 4000 rpm again. The supernatant was discarded and the pellets were frozen at -80°C.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="08.09.2014">08.09.2014</A></H2><FIELDSET class="exp23"><LEGEND><A name="exp23.10">23.10 Results of thesequencing of pSB1C3 StrepDARPidin and pSB1C3 Hag-<I>Kpn</I>I</A></LEGEND><DIV class="exp-content"><P>Sequencing results -positive for both sent plasmids.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND>18.71 Purification of StrepDARPidin from inclusion bodies</LEGEND><DIV class="aim"><P>Aim: Purification of StrepDARPidin from inclusion bodies.</P></DIV><DIV class="exp-content"><P>In order to rescue the StrepDARPidin from the inclusion bodies we tried different protocols. The pellet from 07.09.2014 was warmed up and the cells were cracked with the microfluidizer after resuspension in 15 mL Buffer A (lysate)). </P><P>The suspension was centrifuged at 20.000 x g for 15 min. The supernatant (S1) was discarded  and the pellet (P1)resuspended in 5 mL 6 M guanidinium-HCL solution for solubilisation and unfolding of the StrepDARPidin.</P><P>After centrifugation at 4000 rpm for 10 min the supernatant (S2) was used for refolding. The membranous pellet (P2) was thrown away.</P><P>For refolding 100 mL Buffer A were filled into a Erlenmeyer-Flask on ice with a magnet stirrer inside on 600 rpm.</P><P>For refolding it was necessary to dilute the Guanidinium-HCL dilution very fast and softly. For that purpose rapid dilution was done by pipetting the supernatant drop by drop into the vortex of the Buffer A. After adding the whole Guanidinum-HCL/ Protein solution the suspension was observed and precipitation noticed. The suspension was centrifuged at 20.000 rpm (Pellet P3). The supernatant was loaded on a Ni-NTA Column from GE-Healthcare (L). Flow through (FT), Wash (W) in Buffer A and Elution (E) in Buffer B were kept for gel analysis. The probes lysat to Load were cooked for 5 min at 95 degrees.</P><P>From every fraction 40 µL were taken for a commassie SDS-PAGE and 10 µL 10x SDS Buffer added. The pellet probes were resuspended in 60 µL water and 40 µL SDS-Buffer.</P><P>The gel showed us that the StrepDARPidin is still in the pellet fraction but we were able to make some of it soluble as it can be seen in  S2 and the load. The StrepDARPidin was thought to build tetramers with approx. 120 kDa size. The complex seems to be stabil enough to be seen on the gel above the marker which fits to the size of approx. 120 kDa. Boiling up would break the tetramer.</P><P>In order to get enough protein for a gel filtration run 4 L of BL21 with piGEM-028 were induced with lactose overnight.</P></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND>19.15 Test digest <I>Nco</I>I/ <I>Bam</I>HI with pET24d-Hag-D2-Cup/ -Strep clones</LEGEND><DIV class="aim"><P>Aim: checking insertion of Hag-D2-Cup/ -Strep</P></DIV><DIV class="exp-content"><P>The plasmids from 6 clones per Ligation plate were isolated and approx. 350 ng DNA were digested with <I>Nco</I>I/<I>Bam</I>HI in order to check the right insertion of the 1350-1450 bp sized inserts.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">PCR D2-Cup (µL)</TH><TH scope="col"><I>Nco</I>I/ <I>Bam</I>HI Mastermix (14x)(µL)</TH></TR><TR><TH scope="row">Plasmid (60 ng/ µL)</TH><TD>5</TD><TD>-</TD></TR><TR><TH scope="row">CutSmart   10x</TH><TD>-</TD><TD>14</TD></TR><TR><TH scope="row"><I>Nco</I>I</TH><TD>-</TD><TD>0.5</TD></TR><TR><TH scope="row"><I>Bam</I>HI</TH><TD>-</TD><TD>0.5</TD></TR><TR><TH scope="row">Water</TH><TD>-</TD><TD>55</TD></TR><TR><TH scope="row">Master mix</TH><TD>5</TD><TD>-</TD></TR><TR><TH scope="row">Total</TH><TD>10</TD><TD>70</TD></TR></TBODY></TABLE><P>Digest at 37°C for half an hour.</P><P>Upper half of the gel shows hag-D2 cup, lower half shows hag-D2-Strepdarp; 1kb gene ruler ladder was used. 3 clones seem to be positive for cup, but as an mixture occurred, the cloning procedure will be repeated from the PCR step on.</P></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.16">19.16 PCR amplification of Hag-D2-Cup/ -Strep </A></LEGEND><DIV class="aim"><P>Aim: clone the expression vectors pET24d</P></DIV><DIV class="exp-content"><P>Template was diluted plasmid piGEM026/027</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">1x Mix (µL)</TH><TH scope="col">5x Mastermix (µL)</TH></TR><TR><TH scope="row">Buffer</TH><TD>10</TD><TD>50</TD></TR><TR><TH scope="row">Water</TH><TD>26.5</TD><TD>132.5</TD></TR><TR><TH scope="row">dNTPs</TH><TD>1</TD><TD>5</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>1</TD><TD>5</TD></TR><TR><TH scope="row">DNA (10-13 ng/µl)</TH><TD>1.5</TD><TD>1.5</TD></TR><TR><TH scope="row">Primer 54 Flo (1:50)</TH><TD>6.5</TD><TD>32.5 </TD></TR><TR><TH scope="row">Primer 56 Flo (1:50)</TH><TD>6.5 </TD><TD>32.5</TD></TR></TBODY></TABLE><P>After the PCR the fragments were purified via gel extraction for further cloning steps. In the gel the digested vector pET24d was also purified.</P><P>Concentrations after clean up:</P><P>pET24d = 20 ng/µl, D2-Cup = 208 ng/µl, D2-Strep = 180 ng/µl</P></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.17">19.17 Restriction of -Hag-D2-Cup/ -Strep </A></LEGEND><DIV class="exp-content"><P>After the clean up the two fragments they had to be digested with <I>Nco</I>I and <I>Bam</I>HI in order to clone them into the digested pET24d vecor. </P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">D2-Cup (µL)</TH><TH scope="col">D2-Strep (µL)</TH></TR><TR><TH scope="row">Water</TH><TD>12 </TD><TD>12</TD></TR><TR><TH scope="row">Buffer</TH><TD>2</TD><TD>2</TD></TR><TR><TH scope="row"><I>Nco</I>I</TH><TD>0.5</TD><TD>0.5</TD></TR><TR><TH scope="row"><I>Bam</I>HI</TH><TD>0.5</TD><TD>0.5</TD></TR><TR><TH scope="row">Insert</TH><TD>5</TD><TD>5</TD></TR></TBODY></TABLE><P>The restriction was performed for 1h at 37°C (heating block).</P></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND>19.18 Ligation of Hag-D2-Cup/ -Strep into digested pET24d</LEGEND><DIV class="exp-content"><P>After the plasmid and the two inserts have been digested, two ligations and a ligation control were performed in order to gain the final expression vectors.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">pET24d + D2-cup (µL)</TH><TH scope="col">pET24d + D2- Strep (µL)</TH></TR><TR><TH scope="row">Water</TH><TD>2 </TD><TD>2</TD></TR><TR><TH scope="row">Buffer (10x)</TH><TD>2</TD><TD>2</TD></TR><TR><TH scope="row">Ligase</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Vector</TH><TD>10</TD><TD>10</TD></TR><TR><TH scope="row">Insert</TH><TD>5</TD><TD>5</TD></TR></TBODY></TABLE><P>For the optimal ligation results the ligation calculator of the  iGEM team Dallas was used. Ligation was performed for 2h. Afterwards the ligase was inactivated at 65°C for 15 min in order to improve the ligation/transformation result.</P></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.19">19.19 Transformation of pET24d with Hag-D2-Cup/ -Strep </A></LEGEND><DIV class="exp-content"><P>After the ligase inactivation the whole reaction mix was used to transform <I>E. coli</I> XL-blue which seem to be the best cells for important transformations. Standard transformation protocol was used.</P></DIV></FIELDSET><FIELDSET class="exp22"><LEGEND><A name="exp22.12">22.12 colony PCR and restriction of isolated plasmids from Gibson assembly plate</A></LEGEND><DIV class="aim"><P>Aim: test the clones for the correct pMAD-KSI construct</P></DIV><DIV class="exp-content"><P>The Gibson assembly plate showed sixteen colonies. All colonies were picked, transferred onto a master plate and into a premixed colony PCR. The same colonies were used to inoculate a miniprep.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mixcolony PCR</TH><TH scope="col">Master mix for 18 reactions (µL)</TH><TH scope="col">Single reaction</TH></TR><TR><TH scope="row">Template</TH><TD>-</TD><TD>Small part of colony</TD></TR><TR><TH scope="row">Primer iGEM034 (1:50)</TH><TD>54 </TD><TD>3</TD></TR><TR><TH scope="row">Primer iGEM037 (1:50)</TH><TD>54</TD><TD>3</TD></TR><TR><TH scope="row">5x HF-Buffer </TH><TD>72</TD><TD>4</TD></TR><TR><TH scope="row">dNTPs (40 mM)</TH><TD>18</TD><TD>1</TD></TR><TR><TH scope="row">Phusion DNA polymerase 1:10</TH><TD>9</TD><TD>0.5</TD></TR><TR><TH scope="row">Water</TH><TD>153</TD><TD>8.5</TD></TR><TR><TH scope="row">Total</TH><TD>360 </TD><TD>20 </TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col"> colony PCR Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>10 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>20 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>20 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>2:10 min </TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>33x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The colony PCR was negative, even the positive control with the amplified KS part I with the flanks.</P><P>The plasmids were isolated and digested with <I>Kpn</I>I and <I>Sac</I>I to test the insertion of the KS part I into pMAD. The expected sizes were 5054 bp and 6402 bp with the insert and pMAD only would yield in 4440 bp, 170 bp and 5054 bp fragments.</P><P>The clones 1, 2, 5, 7, 8, 9, 10, 11, 12 and 14 showed the expected results. The rest of the isolated plasmids were discarded.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="09.09.2014">09.09.2014</A></H2><FIELDSET class="exp18"><LEGEND><A name="exp18.71a">18.71a Gelfiltration purification of StrepDARPidin tetramers</A></LEGEND><DIV class="aim"><P>Aim: Purification of StrepDARPidin from inclusion bodies</P></DIV><DIV class="exp-content"><P>The overnight cultures from 18.70 were harvested for 10 min at 4000 x g and the pellets were stored at -80°C until use. </P></DIV></FIELDSET><FIELDSET class="exp22"><LEGEND><A name="exp22.14">22.14 Start of the pMAD-transformation of <I>B. subtilis</I> WT3610 with pMAD-KSI</A></LEGEND><DIV class="aim"><P>Aim: integrate KS part I into the <I>amyE</I>  locus of the <I>B. subtilis</I> genome.</P></DIV><DIV class="exp-content"><P>A preculture of 10 ml LB was inoculated with <I>B. subtilis</I> WT3610 cells and incubated at 37°C over night.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.81">13.81 Midi prep of the nose-plasmids and the piGEM002-plasmids with the killswitch promoters</A></LEGEND><DIV class="aim"><P>Aim: Since no transformation of <EM><I>B. subtilis</I></EM> has worked yet, two new protocols should be tested, which require a huge amount of plasmid</P></DIV><DIV class="exp-content"><P>The midi prep of the plasmids piGEM002 with the silver and copper promoters and the killswitch promoters was carried out according to the Qiagenmidiprep kit instructions. The isolated plasmids were tested via restriction with <I>Bam</I>HI and <I>Spe</I>I.</P><P>Precultures of all four strains (JH642, PY79, 168 and WT3610) were inoculated in 3 ml HS-medium and 10 ml LB and were incubated over night at 37°C.</P></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.20">19.20 Test digest <I>Nco</I>I/<I>Bam</I>HI with pET24d-Hag-D2-Cup/ -Strep clones</A></LEGEND><DIV class="aim"><P>Aim: checking insertion of Hag-D2-Cup/ -Strep</P></DIV><DIV class="exp-content"><P>After the successful ligation (colonies in both plates), 5 clones per ligation have been inoculated in LB-Kan Medium for a later Mini-prep and test digest of the expression plasmids. Colonies with the numbers 6-10 were chosen as 1-5 did not grow over the day. 6-10 were incubated over night.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="10.09.2014">10.09.2014</A></H2><FIELDSET class="exp13"><LEGEND>13.82 Transformation of <EM><I>B. subtilis</I></EM> with the Nose-plasmids (Ag-/Cu-promoters, killswitch promoters) according to the HS/LS-protocol / Spizizen-medium protocol</LEGEND><DIV class="aim"><P>Aim: transform <EM>Bacillus</EM> with the plasmids</P></DIV><DIV class="exp-content"><P>The only preculture, which was grown in HS was the JH642. Every other strain did not grow in High Salt medium. 20 ml of prewarmed (30°C) Low Salt medium were inoculated with 1 ml of the HS culture and incubated in a shaking waterbath at 30°C with 120 rpm for 3 hours. 5 µg of the plasmids were preset in Eppendorf cups and 1 ml of the LS culture was transferred into these cups. piGEM002 was transformed as a control. These culture/plasmid mixes were incubated at 37°C shaking at 200 rpm. After 2 hours of incubation the cells were centrifuged at 8000 rpm for 1.5 minutes, most of the medium was withdrawn,the cells were resuspended in the rest of the medium and plated out on LB-Cm plates that were incubated at 37°C over night. Precultures of all strains were again inoculated, since many were not grown.</P><P>The precultures in LB medium were all grown. 10 ml Spizizens minimal medium were inoculated with an OD<SUB>600</SUB> of 0.1. The cultures were incubated at 37°C until they reached an OD<SUB>600</SUB> of 1.5 µg of DNA were mixed with 1 ml of the culture and incubated for further 2 hours. 200 and 500 µl of each transformation sample were plated out on separate LB-Cm plates, which were incubated at 37°C over night.</P></DIV></FIELDSET><FIELDSET class="exp22"><LEGEND>22.14 pMAD-transformation of <I><I>B. subtilis</I></I> WT3610 with pMAD-KSI</LEGEND><DIV class="aim"><P>Aim: transformation of <I><I>B. subtilis</I></I> with the vector pMAD-KSI</P></DIV><DIV class="exp-content"><P>10 ml of MNGE-medium were inoculated 1/100 with the preculture. This culture was grown at 37°C with an agitation of 200 rpm until it reached OD 1.4. 5 µg of he isolated plasmids pMAD-KSI from clone 1, 2, 10 and 11 (22.12) were used to perform four transformations. 400 µl of the cells  were added to the plasmids in a test tube and were incubated at 37°C 200 rpm for 1 hour. 100 µl expression mix were added and the cells were incubated for another hour. Dilutions up to 10<SUP>-6</SUP> were carried out and the 10<SUP>-5</SUP> and 10<SUP>-6</SUP> dilutions were plated out on LB-MLS-X-Gal plates that were incubated at 30°C for several days.</P></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND>19.21 test digest <I>Nco</I>I/<I>Bam</I>HI with pET24d-Hag-D2-Cup/ -Strep clones</LEGEND><DIV class="aim"><P>Aim: checking insertion of Hag-D2-Cup/ -Strep</P></DIV><DIV class="exp-content"><P>Plasmids were extracted from the cells and a test restriction was performed with 300 ng plasmid. Gelfoto was unfortunately not saved but the bands were at the expected size (appr. 1500 bp). Therefore one plasmid each was sent for sequencing. Cup Number 6!</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">1x Mix (µL)</TH><TH scope="col">11x Mastermix (µL)</TH></TR><TR><TH scope="row">Plasmid DNA </TH><TD>3 </TD><TD>- </TD></TR><TR><TH scope="row">Water</TH><TD>14</TD><TD>154</TD></TR><TR><TH scope="row">CutSmart   Buffer</TH><TD>2 </TD><TD>22 </TD></TR><TR><TH scope="row"><I>Nco</I>I</TH><TD>0.5 </TD><TD>5.5</TD></TR><TR><TH scope="row" style="height: 25px"><I>Bam</I>HI</TH><TD style="height: 25px">0.5 </TD><TD style="height: 25px">5.5 </TD></TR></TBODY></TABLE></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="11.09.2014">11.09.2014</A></H2><FIELDSET class="exp19"><LEGEND><A name="exp19.22">19.22 Sequencing results and further cloning plans</A></LEGEND><DIV class="exp-content"><P>Due to the sequencing results pET24d with Hag-D2-Cup is perfect (the plasmid map was compared to the sequence delivered by eurofins).</P><P>pET24d with Hag-D2-Strep cannot be used, since the strep tag seems to be lost. Therefore this plasmid will be cloned again from the beginning in order to exclude any mistakes at any step.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.83">13.83 Repeated Transformation of <EM><I>B. subtilis</I></EM> with the Nose-plasmids (Ag-/Cu-promoters, killswitch promoters) according to the HS/LS-protocol</A></LEGEND><DIV class="aim"><P>Aim: transform Bacillus with the plasmids</P></DIV><DIV class="exp-content"><P>The plates of the former transformations did not show any colonies, so the trafo was repeated with the precultures (13.82). Again the only strain which grew in HS-medium was JH642. The procedure was carried out like before with a higher agitation (200 rpm) in the waterbath.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="12.09.2014">12.09.2014</A></H2><FIELDSET class="exp19"><LEGEND><A name="exp19.23">19.23 PCR amplification of Hag-D2-Strep</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Materials</TH><TH scope="col">Amount (µL)</TH></TR><TR><TH scope="row">Plasmid piGEM027</TH><TD>1.0</TD></TR><TR><TH scope="row">Water</TH><TD>24.0 </TD></TR><TR><TH scope="row">Buffer (5x)</TH><TD>10.0</TD></TR><TR><TH scope="row">Phusion</TH><TD>1.0</TD></TR><TR><TH scope="row">dNTPs</TH><TD>1.0</TD></TR><TR><TH scope="row">Primer Flo 54 (1:50)</TH><TD>6.5 </TD></TR><TR><TH scope="row">Primer Flo 56 (1:50) </TH><TD>6.5 </TD></TR></TBODY></TABLE><P>The fragment with the size of approx. 1300 bp was successfully amplified.</P></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.24">19.24 Digestion of Hag-D2-Strep</A></LEGEND><DIV class="exp-content"><P>After the purification via gel extraction the fragments had the following concentrations:</P><P>Attempt I = 27 ng/µl Attempt II = 35 ng/µl</P><P>20 µl reactions were prepared:</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Materials</TH><TH scope="col">Attempt I (µL)</TH><TH scope="col">Attempt II (µL)</TH></TR><TR><TH scope="row">Buffer (10x)</TH><TD>2.0</TD><TD>2.0</TD></TR><TR><TH scope="row">Water</TH><TD>8.0</TD><TD>6.0</TD></TR><TR><TH scope="row">Fragment</TH><TD>9.0</TD><TD>11.0</TD></TR><TR><TH scope="row"><I>Nco</I>I</TH><TD>0.5</TD><TD>0.5</TD></TR><TR><TH scope="row"><I>Bam</I>HI</TH><TD>0.5</TD><TD>0.5</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.24">19.24 Ligation of Hag-D2-Strep into digested pET24d</A></LEGEND><DIV class="exp-content"><P>In order to gain the complete expression vector the digested insert had to be ligated into digested pET24d. 20 µl reaction mix was prepared, correct ratio of insert to vector was calculated with the iGEM ligation calculator. </P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Materials</TH><TH scope="col">Amount (µL)</TH></TR><TR><TH scope="row">Buffer </TH><TD>2.0</TD></TR><TR><TH scope="row">Water</TH><TD>4.0</TD></TR><TR><TH scope="row">Ligase</TH><TD>1.0</TD></TR><TR><TH scope="row">Vector</TH><TD>5.0</TD></TR><TR><TH scope="row">Insert </TH><TD>8.0</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.25">19.25 Transformation of pET24d with Hag-D2-Strep</A></LEGEND><DIV class="exp-content"><P>Whole ligation mix was transformed into <I>E. coli</I> XL-1-Blue with the standard transformation protocol.</P></DIV></FIELDSET><FIELDSET class="exp22"><LEGEND><A name="exp22.14">22.14 pMAD-transformation of <I><I>B. subtilis</I></I> WT3610 with pMAD-KSI</A></LEGEND><DIV class="aim"><P>Aim: transformation of <EM><I>B. subtilis</I></EM> with the vector pMAD-KSI</P></DIV><DIV class="exp-content"><P>5 ml LB-MLS were inoculated with a blue colony of the trafo plates and incubated over night at 37°C. Colonies of clone 1, 2, 10 and 11 were picked.</P></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.26">19.26 colony PCR of transformed <I><I>B. subtilis</I></I>  WT3610 with pMAD-D2-Strep/Cup</A></LEGEND><DIV class="aim"><P>Aim: check for positive clones </P></DIV><DIV class="exp-content"><P>The transformation of <EM><I>B. subtilis</I></EM> WT3610 with the plasmids 
	piGEM026 and piGEM027was carried out. In order to check for positive clones a colony 
	PCR was performed using the Hag primers.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mixcolony PCR</TH><TH scope="col">Single reaction</TH></TR><TR><TH scope="row">Colony in 50 µl PBS</TH><TD>2</TD></TR><TR><TH scope="row">Primer Flo 56 (1:50)</TH><TD>6.25</TD></TR><TR><TH scope="row">Primer Flo 54 (1:50)</TH><TD>6.25</TD></TR><TR><TH scope="row">5x HF-Buffer </TH><TD>4</TD></TR><TR><TH scope="row">dNTPs (40 mM)</TH><TD>0.5</TD></TR><TR><TH scope="row">DMSO</TH><TD>0.3</TD></TR><TR><TH scope="row">Phusion DNA polymerase 1:10</TH><TD>0.5</TD></TR><TR><TH scope="row">Water</TH><TD>0.2</TD></TR><TR><TH scope="row">Total</TH><TD>20</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Fusion PCR Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>20 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>20 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1:35 min </TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>infinite</TD></TR></TBODY></TABLE><P>No positive clone could be detected.</P></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="13.09.2014">13.09.2014</A></H2><FIELDSET class="exp23"><LEGEND><A name="exp23.1">23.11 Transformation of of pSB1C3 StrepDARPidin and pSB1C3 Hag-<I>Kpn</I>I in E. coli DH5α</A></LEGEND><DIV class="aim"><P>Aim: Gain more plasmid for further cloning procedures and sending the bricks to the registry. </P></DIV><DIV class="exp-content"><P>Standard <I>E. coli</I> transformation protocol was used. Cells were plated on LB-Cm, incubation over night at 37°C.</P></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.26">19.26 Test restriction of pET24d with Hag-D2-Strep</A></LEGEND><DIV class="exp-content"><P>8 Clones from the ligation plates were inoculated in 5ml LB-Kan and incubated over night. Plasmids were extracted with the quiagen kit and digested with <I>Nco</I>I and <I>Bam</I>HI in order to check if Hag-D2-strep is in pET24d. </P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Components</TH><TH scope="col">1x Mix</TH></TR><TR><TH scope="row">Plasmid</TH><TD>3.0</TD></TR><TR><TH scope="row">Buffer </TH><TD>2.0</TD></TR><TR><TH scope="row">Water</TH><TD>14.0</TD></TR><TR><TH scope="row"><I>Nco</I>I</TH><TD>0.5</TD></TR><TR><TH scope="row"><I>Bam</I>HI</TH><TD>0.5</TD></TR></TBODY></TABLE><P>Every picked  clone was positive so that clone 1 was sent for sequencing. The gel photo got lost unfortunately. pET24d-Hag-D2-Strep was named piGEM-030.</P></DIV></FIELDSET><FIELDSET class="exp22"><LEGEND><A name="exp22.14">22.14 First heatshock of pMAD-Trafo KSI</A></LEGEND><DIV class="aim"><P>Aim: integration of pMAD-KSI into the Bacillus genome</P></DIV><DIV class="exp-content"><P>4 ml of LB-MLS were inoculated with 100 µl of the precultures (in a test tube) and incubated at 30°C and 210 rpm for 2 hours before switching the temperature of the incubator to 42°C for 6 h. Dilutions up to 10<SUP>-6</SUP> were made and 10<SUP>-5</SUP> and 10<SUP>-6</SUP> were plated out on LB-MLS-XGal plates at 42°C.</P></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.27">19.27 repeated colony PCR of transformed <I>B. subtilis</I>  WT3610 with pMAD-D2-Strep/Cup</A></LEGEND><DIV class="aim"><P>Aim: check for positive clones </P></DIV><DIV class="exp-content"><P>The picked clones were boiled at 100°C in 1x PBS for 10 minutes and the the cell rests were centrifuged. The colony PCR was repeated as follows:</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mixcolony PCR</TH><TH scope="col">Single reaction (µL)</TH></TR><TR><TH scope="row">Colony in 100 µl PBS</TH><TD>5</TD></TR><TR><TH scope="row">Primer Flo 56 (1:50)</TH><TD>6.25</TD></TR><TR><TH scope="row">Primer Flo 54 (1:50)</TH><TD>6.25</TD></TR><TR><TH scope="row">5x HF-Buffer </TH><TD>10.0</TD></TR><TR><TH scope="row">dNTPs (40 mM)</TH><TD>1.0</TD></TR><TR><TH scope="row">Phusion DNA polymerase 1:10</TH><TD>1.0</TD></TR><TR><TH scope="row">Water</TH><TD>20.5</TD></TR><TR><TH scope="row">Total</TH><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">colony PCR Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time (min/sec) KSI</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1:40 min </TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The size of the expected bands was approx.1400 bp. The negative clones with the wildtypeflagellin (approx.1 kb) were not further used. Clones D2-Strep 1, 10 and D2-Cup 1 were picked and streaked out separately on LB plates.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="14.09.2014">14.09.2014</A></H2><FIELDSET class="exp23"><LEGEND><A name="exp23.12">23.12 Inoculation of colonies from the transformation plate (2x)</A></LEGEND><DIV class="aim"><P>Aim: Isolate more plasmid for the shipment to the registry</P></DIV><DIV class="exp-content"><P>Incubation over night shaking at 37°C</P></DIV></FIELDSET><FIELDSET class="exp22"><LEGEND><A name="exp22.14">22.14 Second heatshock of pMAD-Trafo KSI</A></LEGEND><DIV class="aim"><P>Aim: flip the pMAD-backbone out of the Bacillus genome</P></DIV><DIV class="exp-content"><P>4 ml of LB were inoculated (in a test tube) with a blue colony of the plates from the first heat shock. These cultures were incubated at 30°C at 210 rpm for 6 hours and afterwards the temperature was shifted to 42°C for 3 hours. The plating was performed as usual.</P></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.27">19.27 colony PCR on positive picked and separated <EM>Bacillus subtilis</EM> clones </A></LEGEND><DIV class="aim"><P>Aim: check for more pure clones with less wt flagellin and GelEx of positive fragment for sequencing</P></DIV><DIV class="exp-content"><P>The colony PCR was repeated as listed above (19.26) and the fragments of the right size were purified by Gel Ex.</P><P>Additionally the positive clones were used to inoculate 100 mL LB with WT3610 as a control. The cultures were raised to an OD<SUB>600</SUB> of 0,7 and 2 mL of cultures were centrifuged before dissolving them in 60 µL water and 40 µL SDS-PAGE buffer. The samples were cooked for 10 min at 95°C and analyzed via SDS-PAGE with coomassie stain.</P><P>It is possible to see that the D2-Strep clone expresses the modified Hag-D2-Strep. The size between the WT flagellin the D2-3-Hag fits perfectly. Hag-D2-Cup seems to be not expressed by the clone. The sequencing results had to be expected.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.84">13.84 colony PCR of<EM> Bacillus subtilis 168</EM> piGEM037</A></LEGEND><DIV class="aim"><P>Aim: check for positive clones with piGEM037</P></DIV><DIV class="exp-content"><P>The plates of the transformation with the Spizizens-medium were still stored at 37°C and none of the plates contained colonies except the plate of 
	<EM>Bacillus subtilis 168</EM>. These clones were picked, transferred onto a new plate and into 1x PBS medium boiled at 100°C or 10 minutes.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mixcolony PCR</TH><TH scope="col">Single reaction</TH></TR><TR><TH scope="row">Colony in 100 µl PBS</TH><TD>5.0</TD></TR><TR><TH scope="row">Primer iGEM006 (1:50)</TH><TD>6.25</TD></TR><TR><TH scope="row">Primer iGEM063 (1:50)</TH><TD>6.25</TD></TR><TR><TH scope="row">5x HF-Buffer </TH><TD>10.0</TD></TR><TR><TH scope="row">dNTPs (40 mM)</TH><TD>1.0</TD></TR><TR><TH scope="row">Phusion DNA polymerase 1:10</TH><TD>1.0</TD></TR><TR><TH scope="row">Water</TH><TD>20.5</TD></TR><TR><TH scope="row">Total</TH><TD>50.0</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">colony PCR Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1:40 min </TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The expected size of the bands was approx.750 bp, the negative control was genomic DNA of the wt3610, the positive control was the isolated plasmid piGEM037.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="15.09.2014">15.09.2014</A></H2><FIELDSET class="exp23"><LEGEND><A name="exp23.13">23.13 Plasmid isolation</A></LEGEND><DIV class="exp-content"><P>Plasmid isolation of the pSB1C3-Hag-<I>Kpn</I>I plasmid from <I>E. coli</I> was performed with the Quiagen Mini-prep kit. Plasmid concentration was determined with Nanodrop (96 ng/µl).</P></DIV></FIELDSET><FIELDSET class="exp23"><LEGEND><A name="exp23.14">23.14 Restriction of pSB1C3 Hag-<I>Kpn</I>I with <I>Kpn</I>I</A></LEGEND><DIV class="aim"><P>Aim: Linearize the plasmid for Gibson assembly </P></DIV><DIV class="exp-content"><P>Restriction of the plasmid with <I>Kpn</I>I was performed in a duplicate for the creation of further biobricks. </P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Digestion Mix</TH><TH scope="col">Amount (µL)</TH></TR><TR><TH scope="row">Water</TH><TD>6.0</TD></TR><TR><TH scope="row">CutSmart  Buffer (10x)</TH><TD>2.0</TD></TR><TR><TH scope="row">Plasmid (3µg)</TH><TD>11.0</TD></TR><TR><TH scope="row"><I>Kpn</I>I</TH><TD>1.0</TD></TR></TBODY></TABLE><P>The digest was incubated at 37°C for 210 minutes and subsequently analyzed on a 1% agarose gel. </P><P>The band of the digested plasmid pSB1C3-Hag-<I>Kpn</I>I was digested out and purified with the Qiuagen gel extraction kit. Concentration after purification was 100 ng/µl.</P></DIV></FIELDSET><FIELDSET class="exp23"><LEGEND><A name="exp23.14">23.14 Gibson assembly of linearized pSB1C3 with three different PCR fragments</A></LEGEND><DIV class="aim"><P>Aim: </P></DIV><DIV class="exp-content"><P>For the creation of further biobricks three PCR fragments that were already present from previous PCR reactions have been inserted into flagellin:</P><UL><LI>Cup 1-1 (amplified with primer iGEM24 and iGEM25) 16 ng/µl</LI><LI>D2-Cup (amplified  with primers  flo96 and flo97) 189 ng/µl</LI><LI>D2-Strep (mplified  with primers  flo96 and flo97) 233 ng/µl</LI></UL><P>For the Gibson assembly 4 Gibson assembly mixes (3 reactions and one control) have been thawed on ice. 2.5 µl of digested plasmid and the PCR fragment have been added respectively. The total mix of 20 µl was incubated at 50°C for one hour (PCR cycler). Afterwards the complete mix was used to transform competent <I>E. coli</I> XL-1-Blue. Cells were afterwards plated on LB-Cm plates. </P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.85">13.85 Cloning of the new Ag/Cu-promoters and the killswitch promoters into the nose plasmids with the ssrA-tags</A></LEGEND><DIV class="aim"><P>Aim: create plasmids with the promoters in front of a GFP with degradation tag</P></DIV><DIV class="exp-content"><P>Since we tried the Bacillus trafo with the ethanol free isolated nose plasmids (without degradation tags) as well and succeeded in nearly all cases except piGEM035, all strains are there for plate reader tests. Competent cells of PY79 were again transformed with piGEM035. The new promoters should also be tested with GFP containing degradation tags. In order to do that the plasmids piGEM007, piGEM008 and piGEM009 that contain the ssrA tags behind the GFP were digested with the digest enzymes <I>Nco</I>I and <I>Sac</I>I.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Components</TH><TH scope="col">1x Mix (µL)</TH></TR><TR><TH scope="row">piGEM007/8/9 (amount: approx. 2 µg)</TH><TD>5.0</TD></TR><TR><TH scope="row">10x CutSmart</TH><TD>2.0</TD></TR><TR><TH scope="row">Water</TH><TD>14.0 </TD></TR><TR><TH scope="row"><I>Nco</I>I</TH><TD>0.5</TD></TR><TR><TH scope="row"><I>Sac</I>I</TH><TD>0.5</TD></TR><TR><TH scope="row">Volume</TH><TD>20</TD></TR></TBODY></TABLE><P>The restriction mixes were incubated at 37°C for over 1 h, separated on an agarose gel and purified via Gel Ex.</P><P>To connect the linearized plasmids with the promoter fragments ligations and Gibson assemblies were performed similar to 13.76 and 13.77.</P><P>Ligation of piGEM007/8/9 with the killswitch promoters:</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Ligation I(µL)</TH><TH scope="col">Ligation II(µL)</TH><TH scope="col">Ligation III (µL)</TH></TR><TR><TH scope="row">piGEM-007/8/9 (approx.20ng/µL)</TH><TD>5</TD><TD>5</TD><TD>5</TD></TR><TR><TH scope="row">iGEM-59/60</TH><TD>0.8</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">iGEM-61/62</TH><TD>-</TD><TD>0.8</TD><TD>-</TD></TR><TR><TH scope="row">iGEM-63/64</TH><TD>-</TD><TD>-</TD><TD>0.8</TD></TR><TR><TH scope="row">Polynucleotide kinase</TH><TD>0.5</TD><TD>0.5</TD><TD>0.5</TD></TR><TR><TH scope="row">T4 Ligase</TH><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Ligation Buffer 10x</TH><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR><TH scope="row">Water</TH><TD>9.7</TD><TD>9.7</TD><TD>9.7</TD></TR><TR><TH scope="row">Total</TH><TD>20</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><P>The reaction was incubated at room temperature for over 1 hour and afterwards used to transform XLIBlue cells, which were plated out on LB-Amp plates.</P><P>Gibson assembly of linearized piGEM007/8/9 with the new silver and copper promoters:</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Gibson Cu I (µL)</TH><TH scope="col">GibsonAg II (µL)</TH><TH scope="col">Control (µL)</TH></TR><TR><TH scope="row">piGEM-002 (approx.20 ng/µL)</TH><TD>2.5</TD><TD>2.5</TD><TD>5</TD></TR><TR><TH scope="row">Cu promoter (4.2 ng/µl)</TH><TD>2.5</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Ag promoter (2 ng/µl)</TH><TD>-</TD><TD>2.5</TD><TD>-</TD></TR><TR><TH scope="row">Gibson mix</TH><TD>15</TD><TD>15</TD><TD>15</TD></TR><TR><TH scope="row">Total</TH><TD>20</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><P>The Gibson reaction was incubated at 50°C for 1 hour and the whole sample was used to transform XLIBlue cells that were plated out on LB-Amp.</P></DIV></FIELDSET><FIELDSET class="exp22"><LEGEND><A name="exp22.15">22.15 colony PCR of pMADtrafo wt3610 with piGEM031 (pMAD-KSI)</A></LEGEND><DIV class="aim"><P>Aim: check for positive insertion of KS part I into the Bacillus genome</P></DIV><DIV class="exp-content"><P>A colony PCR with the primers that bind to the ends of the <I>amyE</I>  flanks was carried out but it was negative. Since it was not sure whether the primer would be able to bind the PCR was repeated with the primers surrounding the killswitch module I iGEM038 and iGEM039:</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mixcolony PCR</TH><TH scope="col">Single reaction</TH></TR><TR><TH scope="row">Colony in 100 µl PBS</TH><TD>5.0</TD></TR><TR><TH scope="row">Primer iGEM006 (1:50)</TH><TD>6.25</TD></TR><TR><TH scope="row">Primer iGEM063 (1:50)</TH><TD>6.25</TD></TR><TR><TH scope="row">5x HF-Buffer </TH><TD>10.0</TD></TR><TR><TH scope="row">dNTPs (40 mM)</TH><TD>1.0</TD></TR><TR><TH scope="row">Phusion DNA polymerase 1:10</TH><TD>1.0</TD></TR><TR><TH scope="row">Water</TH><TD>20.5</TD></TR><TR><TH scope="row">Total</TH><TD>50.0</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">colony PCR Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time (min/sec)KSI</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1:40 min </TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The expected bandsize was about 360 bp, which was visible in nearly all the samples. The positive control for the PCR reaction was isolated piGEM 031 and the negative control was chromosomal DNA of the unmodified wt3610.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.71">18.71 Gelfiltration purification of StrepDARPidin tetramers</A></LEGEND><DIV class="aim"><P>Aim: Purification of StrepDARPidin from inclusion bodies</P></DIV><DIV class="exp-content"><P>In order to improve the rescue the StrepDARPidin from the inclusion bodies we tried an additional IB washing buffer. The pellet from last week at -80°C  was warmed up and the cells were cracked with the microfluidizer after resuspension in 15 mL Buffer A (lysate)). </P><P>The suspension was centrifuged at 20.000 x g for 15 min. The supernatant (S1) was discarded  and the pellet (P1) resuspended in 5 mL IB Wash Buffer. Then the suspension (SS1) was centrifuged at 27.000 x g for 10 min. The resulting pellet (P2) and the supernatant (S2) were used for taking probes for SDS-PAGE. The wash was repeated. 6M Guanidinium-HCL solution for solubilisation and unfolding of the StrepDARPidin was added.</P><P> After centrifugation at 4000 rpm for 10 min the supernatantwas used for refolding. The membranous pellet (P3) was thrown away.</P><P>For refolding 100 mL Buffer A were filled into a Erlenmeyer-Flask on ice with a magnet stirrer inside on 600 rpm.</P><P>For refolding it was necessary to dilute the Guanidinium-HCL dilution very fast and softly. For that purpose rapid dilution was done by pipetting the supernatant drop by drop into the vortex of the Buffer A. After adding the whole Guanidinum-HCL/ Protein solution the suspension was observed and precipitation noticed. The suspension was centrifuged at 20.000 rpm (Pellet P4) . The supernatant was loaded on a Ni-NTA Column from GE-Healthcare (L). Flow through (FT), Wash (20 mL W) in Buffer A and Elution (15 mL) (E) in Buffer B were kept for gel analysis. The samples lysate to Load were cooked for 5 min at 95°C.</P><P>From every fraction 40 µL were taken for a commassie SDS-PAGE and 10 µL 10x SDS Buffer added. The pellet probes were resuspended in 60 µL water and 40 µL SDS-Buffer.</P><P>The gel shows that the monomeric and tetramericStrepDARPidin can be found in Load, Flow through and elution. The forming of the tetramer tells us that the refolding had to be successful partially.  The elution was concentrated with an Amicon to an endvolume of 1,5 mL. 75 µL glycerol were added and the concentrate was frozen away in liquid nitrogen and stored at -80°C for further use.</P></DIV></FIELDSET><FIELDSET class="exp"><LEGEND>Sequencing of constructs from 15.09.2014</LEGEND><DIV class="exp-content"><P>Sequencing of plasmid piGEM030 and PCR products of B.subtilis colony PCR for insertion of D2-Cup and D2-Strep into Flagellin.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Label-Nr.</TH><TH scope="col">Construct</TH><TH scope="col">Primers</TH></TR><TR><TD>AGB000k-583</TD><TD>piGEM-030</TD><TD>T7</TD></TR><TR><TD>AGB000k-584</TD><TD>piGEM-030</TD><TD>T7 term</TD></TR><TR><TD>AGB000k-585</TD><TD>Hag-D2-Strep</TD><TD>Flo96 Hag-D2-fw</TD></TR><TR><TD>AGB000k-586</TD><TD>Hag-D2-Strep</TD><TD>Flo97 Hag-D2-rv</TD></TR><TR><TD>AGB000k-587</TD><TD>Hag-D2-Cup</TD><TD>Flo96 Hag-D2-fw</TD></TR><TR><TD>AGB000k-588</TD><TD>Hag-D2-Cup</TD><TD>Flo97 Hag-D2-rv</TD></TR></TBODY></TABLE></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="16.09.2014">16.09.2014</A></H2><FIELDSET class="exp23"><LEGEND><A name="exp23.15">23.15 Inoculation of presumably positive clones from all plates</A></LEGEND><DIV class="exp-content"><P>Colonies appeared on all plates. Control plate showed appr. 50 colonies whereas the 3 reactions showed uncountable colonies on the plates. 4 clones have been picked per reaction and incubated at 37°C over night for a test restriction of the correct insertion of the 3 inserts of interest.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.71a">18.71a Gelfiltration purification of StrepDARPidin tetramers</A></LEGEND><DIV class="aim"><P>Aim: Purification of StrepDARPidin from inclusion bodies</P></DIV><DIV class="exp-content"><P>The stored protein concentrate was thawn on ice and injected into the injection valve of the ÄtkaPurifier which was buffered with PBS+ 2,5% glycerin. A S200 SepharoseColumn was used for the gel filtration run.</P><P>40 µL of the Fraction C6. C10, C11(Peak) and D12 were picked for SDS-PAGE analysis. 10 µL SDS-Buffer were added and 20 µL loaded on the gel. Additionally the rest was cooked up and the same amount loaded onto the gel.</P><P>The gel shows us that the whole peak contains the tetramericStrepDARPidin. After cooking it up the tetramer dissolves into the monomeric StrepDARPidin.</P><P>The fractions covering the peak were concentrated with an amicon to an endvolume of 500 µL. 25 µL glycerin were added and the protein aliquot in 60 µL aliquods. </P></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.28">19.28 Purification of Hag-D2-Strep</A></LEGEND><DIV class="aim"><P>Aim: Purification of Hag-D2-Strep for analytical gel filtration was already purified StrepDARPidin</P></DIV><DIV class="exp-content"><P>piGEM-029 and piGEM-030 were transformed together with FliS into <EM>E. Coli BL21 (DE3)</EM> and plated out on LB-Amp/Can plates for overnight incubation at 37°C.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.86">13.86 Cloning of the Nose plasmids with the ssrA- degradation tags</A></LEGEND><DIV class="aim"><P>Aim: create plasmids with the promoters in front of a GFP with degradation tag</P></DIV><DIV class="exp-content"><P>The following entries of the nose plasmids will care all about the cloning, which involves creating the plasmid by Gibson/ligation checking by colony PCR, transformation of <I>E. coli</I> DH5α and 
	<I>Bacillus subtilis</I> PY79 with the plasmids. Not every step of transformation will be mentioned, since they all were carried out according to the standard protocols. After each plasmid was checked by colony PCR and/or control restriction DH5α and Bacillus were transformed with it. DH5α were transformed with the plasmids piGEM034, 035, 036 and 037.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="17.09.2014">17.09.2014</A></H2><FIELDSET class="exp23"><LEGEND><A name="exp23.16">23.16 Plasmid isolation and test digest</A></LEGEND><DIV class="exp-content"><P>Plasmid isolation of the 12 cultures was performed with the Quiagen Mini Prep Kit. Concentration afterwards was about 250 ng/µl in all samples. </P><P>Test restriction of the previously isolated plasmids with <I>Eco</I>RI and <I>Pst</I>I:</P><P>12 plasmids + control →	14x master mix for 20µl reaction mixes</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">1x Mix (µL)</TH><TH scope="col">14x Mix (µL]</TH></TR><TR><TH scope="row">Water</TH><TD>15.5</TD><TD>217.0</TD></TR><TR><TH scope="row">CutSmart   Buffer (10x)</TH><TD>2.0</TD><TD>28.0</TD></TR><TR><TH scope="row"><I>Eco</I>RI</TH><TD>0.5</TD><TD>7.0</TD></TR><TR><TH scope="row"><I>Pst</I>I</TH><TD>0.5</TD><TD>7.0</TD></TR><TR><TH scope="row">Plasmid DNA</TH><TD>1.5</TD><TD>1.5</TD></TR></TBODY></TABLE><P>Restriction was incubated at 37°C for 180 minutes and subsequently analyzed on a 1% agarose gel.</P><P>Result: All tested clones were negative, i.e. bands at the same size of the control occurred. </P><P>1500bp= Hag-<I>Kpn</I>I;	2000bp= 13C plasmid backbone</P></DIV></FIELDSET><FIELDSET class="exp23"><LEGEND><A name="exp23.17">23.17 Inoculation of 5 new clones from all plates</A></LEGEND><DIV class="exp-content"><P>Plan: inoculate 5 new clones each for plasmid isolation and restriction tomorrow in LB-Cm.</P></DIV><DIV class="aim"><P>Aim: Gain more plasmid in case the Gibson assembly has to be repeated Inoculation in 5 ml LB-Cm</P></DIV></FIELDSET><FIELDSET class="exp"><LEGEND><A name="exp_24">Sequencing results  of constructs from 15.09.2014</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Label-Nr.</TH><TH scope="col">Construct</TH><TH scope="col">results</TH></TR><TR><TD>AGB000k-583</TD><TD>piGEM-030</TD><TD>fine</TD></TR><TR><TD>AGB000k-584</TD><TD>piGEM-030</TD><TD>fine</TD></TR><TR><TD>AGB000k-585</TD><TD>Hag-D2-Strep</TD><TD>fine</TD></TR><TR><TD>AGB000k-586</TD><TD>Hag-D2-Strep</TD><TD>fine</TD></TR><TR><TD>AGB000k-587</TD><TD>Hag-D2-Cup</TD><TD>fine</TD></TR><TR><TD>AGB000k-588</TD><TD>Hag-D2-Cup</TD><TD>fine</TD></TR></TBODY></TABLE><P>The sequencing told us that Both Hag-D2-Strep &amp; -Cup are perfectly integrated into the chromosome of the mutants. Hag-D2-Cup seems not to be expressed. Anyways a swarming assay was planned.</P></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.28">19.28 Purification of Hag-D2-Strep</A></LEGEND><DIV class="aim"><P>Aim: Purification of Hag-D2-Strep for analytical gel filtration was already purified StrepDARPidin</P></DIV><DIV class="exp-content"><P>An overnight expression culture from piGEM-030/FliS was made by inoculating 2 x 1L LB-Amp/Can with clones from the transformation plates and inducing with 50 mL 25% lactose solution overnight at 30°C at 150 rpm.</P></DIV></FIELDSET><FIELDSET class="exp21"><LEGEND><A name="exp21.3">21.3 Swarming AssayHag-D2-Strep &amp; -Cup</A></LEGEND><DIV class="aim"><P>Aim: Check motility of Bacillus mutants Hag-D2-Cup &amp; -Strep</P></DIV><DIV class="exp-content"><P>Overnight cultures were inoculated of 5 mL LB in test tubes from WT3610 , Hag-D2-Cup &amp; -Strep strains. Incubation was done at 37°C overnight.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.87">13.87 Making Bacillus competent for transformation</A></LEGEND><DIV class="aim"><P>Aim: making competent PY 79 cells for the transformation with the new nose-plasmids</P></DIV><DIV class="exp-content"><P>The <I>Bacillus subtilis</I> PY79 cells were made competent according to the protocol using SPC and SPII medium. As a test the cells were transformed with piGEM038, which was transformed before.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.88">13.88 Screening for positive clones containing the new Nose-plasmids</A></LEGEND><DIV class="aim"><P>Aim: colony PCR to check colonies for the correct plasmids</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix colony PCR</TH><TH scope="col">Cu-promoter</TH><TH scope="col">Ag-promoter</TH><TH scope="col">lac-promoter</TH><TH scope="col">const. + tetO promoter</TH><TH scope="col">constitutive promoter</TH></TR><TR><TD>Template</TD><TD>Part of a colony</TD><TD>Part of a colony</TD><TD>Part of a colony</TD><TD>Part of a colony</TD><TD>Part of a colony</TD></TR><TR><TD>iGEM006 (1:50)</TD><TD>6.25</TD><TD>6.25</TD><TD>6.25</TD><TD>6.25</TD><TD>6.25</TD></TR><TR><TD>iGEM055 (1:50)</TD><TD>6.25</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TD>iGEM057 (1:50)</TD><TD>-</TD><TD>6.25</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TD>iGEM059 (1:50)</TD><TD>-</TD><TD>-</TD><TD>6.25</TD><TD>-</TD><TD>-</TD></TR><TR><TD>iGEM061 (1:50)</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>6.25</TD><TD>-</TD></TR><TR><TD>iGEM063 (1:50)</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>6.25</TD></TR><TR><TD>dNTPs (10 mM each)</TD><TD>0.5</TD><TD>0.5</TD><TD>0.5</TD><TD>0.5</TD><TD>0.5</TD></TR><TR><TD>5x HF-buffer</TD><TD>4</TD><TD>4</TD><TD>4</TD><TD>4</TD><TD>4</TD></TR><TR><TD>Phusion DNA-polymerase</TD><TD>0.5</TD><TD>0.5</TD><TD>0.5</TD><TD>0.5</TD><TD>0.5</TD></TR><TR><TD>Water</TD><TD>2.5</TD><TD>2.5</TD><TD>2.5</TD><TD>2.5</TD><TD>2.5</TD></TR><TR><TD>Total</TD><TD>20</TD><TD>20</TD><TD>20</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">colony PCR Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time (min/sec)KSI</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>10 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1 min </TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The size of the expected band was depending on the promoter between 700 and 850 bp. As a positive control cells containing the piGEM002 vector with the different promoters were used. The clones that seemed positive were given in culture for a miniprep. More clones were picked from the plates to test them via colony PCR (same reaction as above).</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="18.09.2014">18.09.2014 </A></H2><FIELDSET class="exp23"><LEGEND><A name="exp23.19">23.19 Plasmid isolation and test restriction of the 15 new clones</A></LEGEND><DIV class="exp-content"><P>Same procedure as yesterday (17.09)</P><P>Plasmid concentration after plasmid isolation was about 250 to 300 ng/µl. 2 µl of plasmid DNA were used for the restriction reaction.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Digestion Mix</TH><TH scope="col">1x Mix (µL)</TH><TH scope="col">16x Mix (µL]</TH></TR><TR><TH scope="row">Water</TH><TD>15.5</TD><TD>248.0</TD></TR><TR><TH scope="row">CutSmart   Buffer (10x)</TH><TD>2.0</TD><TD>32.0</TD></TR><TR><TH scope="row"><I>Eco</I>RI</TH><TD>0.25</TD><TD>4.0</TD></TR><TR><TH scope="row"><I>Pst</I>I</TH><TD>0.25</TD><TD>4.0</TD></TR><TR><TH scope="row">Plasmid DNA</TH><TD>2.0</TD><TD>xxx</TD></TR></TBODY></TABLE><P>Incubation for 240 min. digested plasmids were analyzed on a 1% agarose gel.</P></DIV></FIELDSET><FIELDSET class="exp23"><LEGEND><A name="exp23.20">23.20 pSB1C3 over night restriction</A></LEGEND><DIV class="exp-content"><P>All tested clones were negative. In order to repeat the Gibson assembly pSB1C3 with Hag-<I>Kpn</I>I will be digested with <I>Kpn</I>I over night (37°C).</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Digestion Mix</TH><TH scope="col">Amount (µL)</TH></TR><TR><TH scope="row">CutSmart Buffer (10x)</TH><TD>2.0</TD></TR><TR><TH scope="row">Plasmid </TH><TD>17.00</TD></TR><TR><TH scope="row"><I>Kpn</I>I</TH><TD>1.0</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp21"><LEGEND>21.3a Swarming Assay Hag-D2-Strep &amp; -Cup</LEGEND><DIV class="aim"><P>Aim: Check motility of Bacillus mutants Hag-D2-Cup &amp; -Strep</P></DIV><DIV class="exp-content"><P>The OD<SUB>600</SUB> of the overnight cultures was measured and was approx. 5. 100 µL of culture were centrifuged and the pellet resuspended in 50 µL medium to get an OD of 10.</P><P>10 µL of the cultures was dropped on Swimming and Swarming plates (0,3% and 0,7% LB Agar plates) and incubated at 37°C.</P><P>The cellular diffusion was observed after 30-60 min and the distanced measured starting from the cell circle after letting the drop dry.</P><TABLE border="1" cellspacing="0" cellpadding="0" width="100%"><TBODY><TR><TD width="33" valign="top"><P><STRONG>Inubation Time (min)</STRONG></P></TD><TD width="105" colspan="2" valign="top"><P><STRONG>WT3610 Distance (mm)</STRONG></P></TD><TD width="105" colspan="2" valign="top"><P><STRONG>Hag-D2-Strep Distance (mm)</STRONG></P></TD><TD width="105" colspan="2" valign="top"><P><STRONG>Hag-D2-Cup Distance (mm)</STRONG></P></TD></TR><TR><TD width="48" valign="top"><P>Swim</P></TD><TD width="57" valign="top"><P>Swarm</P></TD><TD width="48" valign="top"><P>Swim</P></TD><TD width="57" valign="top"><P>Swarm</P></TD><TD width="48" valign="top"><P>Swim</P></TD><TD width="57" valign="top"><P>Swarm</P></TD></TR><TR><TD width="33" valign="top"><P>30</P></TD><TD width="48" valign="top"><P>-</P></TD><TD width="57" valign="top"><P>-</P></TD><TD width="48" valign="top">-</TD><TD width="57" valign="top">-</TD><TD width="48" valign="top"><P>-</P></TD><TD width="57" valign="top">-</TD></TR><TR><TD width="33" valign="top"><P>60</P></TD><TD width="48" valign="top"><P>-</P></TD><TD width="57" valign="top"><P>-</P></TD><TD width="48" valign="top">-</TD><TD width="57" valign="top">-</TD><TD width="48" valign="top"><P>-</P></TD><TD width="57" valign="top">-</TD></TR><TR><TD width="33" valign="top">90</TD><TD width="48" valign="top">-</TD><TD width="57" valign="top">0.5</TD><TD width="48" valign="top">-</TD><TD width="57" valign="top">-</TD><TD width="48" valign="top">-</TD><TD width="57" valign="top">-</TD></TR><TR><TD width="33" valign="top">120</TD><TD width="48" valign="top">0.5</TD><TD width="57" valign="top">1</TD><TD width="48" valign="top">0.4</TD><TD width="57" valign="top">0.5</TD><TD width="48" valign="top">-</TD><TD width="57" valign="top">-</TD></TR><TR><TD width="33" valign="top">150</TD><TD width="48" valign="top">4.5</TD><TD width="57" valign="top">2</TD><TD width="48" valign="top">1</TD><TD width="57" valign="top">1.5</TD><TD width="48" valign="top">-</TD><TD width="57" valign="top">-</TD></TR><TR><TD width="33" valign="top">180</TD><TD width="48" valign="top">12</TD><TD width="57" valign="top">4</TD><TD width="48" valign="top">4</TD><TD width="57" valign="top">9</TD><TD width="48" valign="top">-</TD><TD width="57" valign="top">-</TD></TR><TR><TD width="33" valign="top">210</TD><TD width="48" valign="top">23</TD><TD width="57" valign="top">10</TD><TD width="48" valign="top">12</TD><TD width="57" valign="top">13</TD><TD width="48" valign="top">-</TD><TD width="57" valign="top">-</TD></TR><TR><TD width="33" valign="top">240</TD><TD width="48" valign="top">31</TD><TD width="57" valign="top">16</TD><TD width="48" valign="top">18</TD><TD width="57" valign="top">17</TD><TD width="48" valign="top">-</TD><TD width="57" valign="top">-</TD></TR><TR><TD width="33" valign="top">270</TD><TD width="48" valign="top">38</TD><TD width="57" valign="top">29</TD><TD width="48" valign="top">23</TD><TD width="57" valign="top">26</TD><TD width="48" valign="top">-</TD><TD width="57" valign="top">-</TD></TR><TR><TD width="33" valign="top">300</TD><TD width="48" valign="top">38</TD><TD width="57" valign="top">36</TD><TD width="48" valign="top">29</TD><TD width="57" valign="top">30</TD><TD width="48" valign="top">-</TD><TD width="57" valign="top">-</TD></TR><TR><TD width="33" valign="top">330</TD><TD width="48" valign="top">38</TD><TD width="57" valign="top">38</TD><TD width="48" valign="top">34</TD><TD width="57" valign="top">32</TD><TD width="48" valign="top"> </TD><TD width="57" valign="top"> </TD></TR><TR><TD width="33" valign="top"> </TD><TD width="48" valign="top"> </TD><TD width="57" valign="top"> </TD><TD width="48" valign="top">38</TD><TD width="57" valign="top">38</TD><TD width="48" valign="top">-</TD><TD width="57" valign="top">-</TD></TR></TBODY></TABLE><P>The measurements showed that Hag-D2-Strep is able to swim and swarm. In comparison to the wild type the strain is slower in swarming and swimming. Hag-D2-Strep did not swarm or swim as the commassie gel showing no flagellin suggested before. Electronmicroscopy and alternatives for Hag-D2-Strep were planned.</P><P>For reproduction of the results new precultures of Hag-D2-Strep &amp; -Cup were inoculated and incubated at 30°C.</P></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.28">19.28 Purification of Hag-D2-Strep</A></LEGEND><DIV class="aim"><P>Aim: Purification of Hag-D2-Strep for analytical gel filtration was already purified StrepDARPidin</P></DIV><DIV class="exp-content"><P>The cultures from yesterday were harvested at 4000 rpm at 4 °C and the pellets were resuspended in 10 mL Buffer A for microfluidizing. The lysate was centrifuged at 20000 rpm so that the clear lysat could be loaded on a 1 mL Ni-NTA column. Preinduction probe (PI), induction probe (I), Load (L), flow through (FT), wash (W) and elution with 15 mL Buffer B (E) were analysed on a SDS-PAGE gel commassie stained:</P><P>Meanwhile the elution was concentrated with an amicon at 4000 rpm until an endvolume of 1 mL. The ÄKTApurifier was buffered with GeFi Buffer and the concentrate was injected. Presets: 4 mL fraction size, 0,65 mPa pressure alarm, fractioning after 90 mL run, endvolme 400 mL. A Superdex S200 column was used.</P><P>FT and W contained much protein. The 1 mL Ni-NTA had not enough capacity to bind all the protein.</P><P>The Ni-NTA column seemed to be contaminated with left StrepDARPidin on the NI-NTA columns from previous use. We made the GeFi run to see if we culd purifier the proteins anyways.</P><P>The fractions building the peak were analyzed on a SDS-Gel to cover the peak. Then the fractions from  were concentrated with an amicon to an endvolume of 500 µL. Aliquods of 50 µL were frozen in liquid nitrogen and stored at -80°C.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="13.89">13.89 Control digestion of the plasmids with <I>Spe</I>I and <I>Bam</I>HI and control PCR</A></LEGEND><DIV class="aim"><P>Aim: Aim: check isolated plasmids for correct insertion of the promoter sequences</P></DIV><DIV class="exp-content"><P>To be sure that the promoter sequences were inserted correctly the isolated plasmids were digested with the enzymes <I>Spe</I>I and <I>Bam</I>HI. The difference between the negative and positive clones should be very few but be detectable on the gel: positive: 3426 and approx.3500 bp; negative: 3426 and 3276 bp.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix restriction </TH><TH scope="col">Single reaction (µl)</TH></TR><TR><TH scope="row">Plasmid</TH><TD>6.0</TD></TR><TR><TH scope="row">CutSmart  10x</TH><TD>2.0</TD></TR><TR><TH scope="row"><I>Spe</I>I</TH><TD>1.0</TD></TR><TR><TH scope="row"><I>Bam</I>HI</TH><TD>1.0</TD></TR><TR><TH scope="row">Water</TH><TD>10.0</TD></TR><TR><TH scope="row">Total</TH><TD>20.0</TD></TR></TBODY></TABLE><P>The restriction reaction was incubated at 37°C for over one hour and analyzed with an agarose gel.</P><P>Although the colony PCR for most of the plasmids was like expected, some plasmids were not digested at all. The bands around 3000 bp are double bands of the two fragments. A difference between negative and positive plasmids could not be discovered since the amount of DNA was too high and the bands were not separated well enough. It was striking that every undigested plasmid was a plasmid with the piGEM007 backbone (strong degradation tag). For a better check a control PCR was performed with the 007-plasmids as template:</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix colony PCR</TH><TH scope="col">Single reaction</TH></TR><TR><TH scope="row">Plamid (2 ng/µl)</TH><TD>2.5</TD></TR><TR><TH scope="row">Primer iGEM006 (1:50)</TH><TD>6.25</TD></TR><TR><TH scope="row">Primer fw 55/57/59/61 or 63 (1:50)</TH><TD>6.25</TD></TR><TR><TH scope="row">5x HF-Buffer </TH><TD>10.0</TD></TR><TR><TH scope="row">dNTPs (40 mM)</TH><TD>1.0</TD></TR><TR><TH scope="row">Phusion DNA polymerase 1:10</TH><TD>1.0</TD></TR><TR><TH scope="row">Water</TH><TD>24.5</TD></TR><TR><TH scope="row">Total</TH><TD>50.0</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">colony PCR Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time (min/sec)KSI</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1:00 min </TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>infinite</TD></TR></TBODY></TABLE><P>According to the control PCR on the plasmids the plasmids were ok and the promoters were inserted correctly. The copper sensitive promoter is slightly bigger than the used lac or constitutive promoters, which can be seen very well on the gel. The plasmids were assumed to be correct and were ready for further using (trafo of Bacillus and DH5α). However, a further Gibson assembly and ligation reaction (according to 13.85) was carried out with all promoters and plasmids that were negative (piGEM007 + Cu + Ag; 008 + Cu; 009 + Ag + Cu; 007 + const + tetO + lac; 008 + const; 009 + tetO + lac).</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="19.09.2014">19.09.2014</A></H2><FIELDSET class="exp25"><LEGEND><A name="exp25.2">25.2 Preparation of LLCs and A549</A></LEGEND><DIV class="aim"><P>Aim: Preparation of LLCs for assays</P></DIV><DIV class="exp-content"><P>Lewis Lung cell Carcinoma Cell line and A549 Adenocarcinoma cell line were kindly provided by Dr. Jürgen Adamkiewicz and Margitta Alt (ZTI Marburg) and splitted 1:3 at 37°C in DMEM + 10% FCS and L-Glutamine. The cells were cultivated for further use.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="19.09.2014">19.09.2014 </A></H2><FIELDSET class="exp23"><LEGEND><A name="exp23.1">23.21 Analysis of digested pSB1C3 </A></LEGEND><DIV class="exp-content"><P>Checked Test digestion on agarose gel, three bands were visible, which could not be. Only 2 bands were expected.</P><P>Check undigested plasmids on agarose gel. Result: Two bands</P><P>Retransformation of the already sequenced plasmid pSB1C3-Hag-<I>Kpn</I>I for further cloning processes.</P></DIV></FIELDSET><FIELDSET class="exp21"><LEGEND><A name="exp21.4">21.3b Swarming Assay Hag-D2-Strep &amp; -Cup</A></LEGEND><DIV class="aim"><P>Aim: Check motility of Bacillus mutants Hag-D2-Cup &amp; -Strep</P></DIV><DIV class="exp-content"><P>The OD of the overnight cultures was measured and was approx. OD 5. 100 µL of culture were centrifuged and the pellet resuspended in 50 µL medium to get an OD of 10.</P><P>10 µL of the cultures were spotted on Swimming and Swarming plates (0,3% and 0,7% LB Agar plates) and incubated at 37°C.</P><P>The cellular diffusion was observed after 30-60 min and the distanced measured starting from the cell circle after letting the drop dry.</P><TABLE border="1" cellspacing="0" cellpadding="0" width="100%"><TBODY><TR><TD width="33" valign="top"><P><STRONG>Inubation Time (min)</STRONG></P></TD><TD width="105" colspan="2" valign="top"><P><STRONG>WT3610 Distance (mm)</STRONG></P></TD><TD width="105" colspan="2" valign="top"><P><STRONG>Hag-D2-Strep Distance (mm)</STRONG></P></TD><TD width="105" colspan="2" valign="top"><P><STRONG>Hag-D2-Cup Distance (mm)</STRONG></P></TD></TR><TR><TD width="48" valign="top"><P>Swim</P></TD><TD width="57" valign="top"><P>Swarm</P></TD><TD width="48" valign="top"><P>Swim</P></TD><TD width="57" valign="top"><P>Swarm</P></TD><TD width="48" valign="top"><P>Swim</P></TD><TD width="57" valign="top"><P>Swarm</P></TD></TR><TR><TD width="33" valign="top"><P>30</P></TD><TD width="48" valign="top"><P>-</P></TD><TD width="57" valign="top"><P>-</P></TD><TD width="48" valign="top">-</TD><TD width="57" valign="top">-</TD><TD width="48" valign="top"><P>-</P></TD><TD width="57" valign="top">-</TD></TR><TR><TD width="33" valign="top"><P>60</P></TD><TD width="48" valign="top"><P>-</P></TD><TD width="57" valign="top"><P>-</P></TD><TD width="48" valign="top">-</TD><TD width="57" valign="top">-</TD><TD width="48" valign="top"><P>-</P></TD><TD width="57" valign="top">-</TD></TR><TR><TD width="33" valign="top">90</TD><TD width="48" valign="top">4</TD><TD width="57" valign="top">6</TD><TD width="48" valign="top">-</TD><TD width="57" valign="top">6</TD><TD width="48" valign="top">-</TD><TD width="57" valign="top">-</TD></TR><TR><TD width="33" valign="top">120</TD><TD width="48" valign="top">11</TD><TD width="57" valign="top">8</TD><TD width="48" valign="top">5</TD><TD width="57" valign="top">8</TD><TD width="48" valign="top">-</TD><TD width="57" valign="top">-</TD></TR><TR><TD width="33" valign="top">150</TD><TD width="48" valign="top">20</TD><TD width="57" valign="top">16</TD><TD width="48" valign="top">6</TD><TD width="57" valign="top">12</TD><TD width="48" valign="top">-</TD><TD width="57" valign="top">-</TD></TR><TR><TD width="33" valign="top">180</TD><TD width="48" valign="top">32</TD><TD width="57" valign="top">19</TD><TD width="48" valign="top">18</TD><TD width="57" valign="top">20</TD><TD width="48" valign="top">-</TD><TD width="57" valign="top">-</TD></TR><TR><TD width="33" valign="top">210</TD><TD width="48" valign="top">36</TD><TD width="57" valign="top">32</TD><TD width="48" valign="top">26</TD><TD width="57" valign="top">26</TD><TD width="48" valign="top">-</TD><TD width="57" valign="top">-</TD></TR><TR><TD width="33" valign="top">240</TD><TD width="48" valign="top">36</TD><TD width="57" valign="top">32</TD><TD width="48" valign="top">18</TD><TD width="57" valign="top">34</TD><TD width="48" valign="top">-</TD><TD width="57" valign="top">-</TD></TR><TR><TD width="33" valign="top">270</TD><TD width="48" valign="top">36</TD><TD width="57" valign="top">36</TD><TD width="48" valign="top">34</TD><TD width="57" valign="top">36</TD><TD width="48" valign="top">-</TD><TD width="57" valign="top">-</TD></TR><TR><TD width="33" valign="top">300</TD><TD width="48" valign="top">36</TD><TD width="57" valign="top">36</TD><TD width="48" valign="top">36</TD><TD width="57" valign="top">36</TD><TD width="48" valign="top">-</TD><TD width="57" valign="top">-</TD></TR><TR><TD width="33" valign="top">330</TD><TD width="48" valign="top">38</TD><TD width="57" valign="top">38</TD><TD width="48" valign="top">34</TD><TD width="57" valign="top">32</TD><TD width="48" valign="top"> </TD><TD width="57" valign="top"> </TD></TR></TBODY></TABLE><P>The Hag-D2-Strep strain swarms and swims slower than the WT but is motile in comparison to the Hag-D2-Cup strain.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.90"> 13.90 Control restriction of nose plasmids</A></LEGEND><DIV class="aim"><P>Aim: check isolated plasmids for correct insertion of the promoter sequences</P></DIV><DIV class="exp-content"><P>Plasmids were isolated from cultures inoculated yesterday. These plasmids were again digested with <I>Bam</I>HI and <I>Spe</I>I to check for the insertion of the promoters.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix restriction </TH><TH scope="col">Single reaction (µl)</TH></TR><TR><TH scope="row">Plasmid</TH><TD>2.0</TD></TR><TR><TH scope="row">CutSmart  10x</TH><TD>2.0</TD></TR><TR><TH scope="row"><I>Spe</I>I</TH><TD>1.0</TD></TR><TR><TH scope="row"><I>Bam</I>HI</TH><TD>1.0</TD></TR><TR><TH scope="row">Water</TH><TD>14.0</TD></TR><TR><TH scope="row">Total</TH><TD>20.0</TD></TR></TBODY></TABLE><P>The reactions were incubated at 37°C for over 1 hour.</P><P>All bands were in the expected height of approx. 3400 bp. The plasmids were used further for transformation of DH5α and Bacillus.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="20.09.2014">20.09.2014</A></H2><FIELDSET class="exp23"><LEGEND><A name="exp23.22">23.22 Inoculation of presumably positive clones from retransformation, plasmid isolation and over night restriction</A></LEGEND><DIV class="aim"><P>Aim: Gain more plasmid for Gibson assembly</P></DIV><DIV class="exp-content"><UL><LI>Inoculation of positive clones of pSB1C3-Hag-<I>Kpn</I>I for Mini-prep (3 clones) in the morning </LI><LI>Plasmid isolation in the evening</LI><LI>Overnight restriction of pSB1C3-Hag-<I>Kpn</I>I for Gibson assembly.</LI></UL><P>Check initial plasmids for purity: an additional band was visible, which could probably be coiled plasmid.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.90a">13.90a Screening for positive clones containing the new Nose-plasmids</A></LEGEND><DIV class="aim"><P>Aim: colony PCR to check colonies for the correct plasmids</P></DIV><DIV class="exp-content"><P>The clones on the plates of the new ligation and transformation were screened for positive constructs by colony PCR.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix colony PCR</TH><TH scope="col">Single reaction</TH></TR><TR><TH scope="row">Template</TH><TD>part of a colony</TD></TR><TR><TH scope="row">Primer iGEM006 (1:50)</TH><TD>3.125</TD></TR><TR><TH scope="row">Primer fw 55/57/59/61 or 63 (1:50)</TH><TD>3.125</TD></TR><TR><TH scope="row">5x HF-Buffer </TH><TD>4.0</TD></TR><TR><TH scope="row">dNTP mix (10 mM each)</TH><TD>1.0</TD></TR><TR><TH scope="row">Phusion DNA polymerase 1:10</TH><TD>0.5</TD></TR><TR><TH scope="row">Water</TH><TD>8.75</TD></TR><TR><TH scope="row">Total</TH><TD>20.0</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">colony PCR Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time (min/sec)KSI</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>10 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1:00 min </TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>infinite</TD></TR></TBODY></TABLE><P>As a positive control XLIBlue containing piGEM034 or piGEM035 were used, the negative control was an untransformed XLIBlue. None of the picked clones was positive for the lac and constitutive promoters + tetO. Some clones containing the plasmids piGEM009 + Ag promoter and piGEM009 + Cu promoter were positive. These positive clones were picked and used to inoculate a miniprep culture.</P></DIV></FIELDSET><FIELDSET class="exp21"><LEGEND><A name="exp21.4">21.4 Analytical gel filtration</A></LEGEND><DIV class="aim"><P>Aim: Checking interactions between Hag-D2-Strep and StrepDARPidin</P></DIV><DIV class="exp-content"><P>In order to check the interaction between the Strep-Tag in the D2-Flagellin and the Streptavidin in our StrepDARPidin we used an analytical Superdex S200 column. Probes from both proteins were injected into the ÄTKAPurifier and a separate run each done in order to see when the protein gets through the column.</P><P>In case of an interaction the complex of both proteins would come off early from the column because of its size. The column was buffered with PBS+2,5% glycerin.</P><P>Size and ratio were calculated with http://web.expasy.org/protparam/ and http://christoph-leidig.de/tprot.html for injecting adequate amounts of both proteins.</P><P>StrepDARPidin:</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Absorption at 280 nm</TH><TD>5.025</TD></TR><TR><TH scope="row">Extinction coefficient ε [l/mol*cm)]</TH><TD>57410</TD></TR><TR><TH scope="row">Cuvette length [cm]</TH><TD>1</TD></TR><TR><TH scope="row">Molecular weight [g/mol]</TH><TD>31809 </TD></TR><TR><TH scope="row">Protein concentration [µM]</TH><TD>88.57341926493642</TD></TR><TR><TH scope="row">Protein concentration [mg/ml]</TH><TD>2.8174318933983624</TD></TR></TBODY></TABLE><P>Hag-D2-Strep/FliS</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Absorption at 280 nm</TH><TD>7.015</TD></TR><TR><TH scope="row">Extinction coefficient ε [l/mol*cm)]</TH><TD>130786.5</TD></TR><TR><TH scope="row">Cuvette length [cm]</TH><TD>1</TD></TR><TR><TH scope="row">Molecular weight [g/mol]</TH><TD>130786.5</TD></TR><TR><TH scope="row">Protein concentration [µM]</TH><TD>122.19125587876675</TD></TR><TR><TH scope="row">Protein concentration [mg/ml]</TH><TD>15.980966686988328</TD></TR></TBODY></TABLE><P>88.57/ 122,19 = 0,7248</P><P>(1-0,7248)* 50 µL (StrepDARPidin) =13,76 µL(Hag-D2-Strep)</P><P>Analytical run StrepDARPidin 50 µL + 50 µL PBS+2,5% glycerin:</P><P>Analytical run Hag-D2-Strep/ FliS 13,7 µL+ 86,3 µL PBS+2,5% glycerin:</P><P>13,7 µL Hag-D2-Strep/FliS &amp; 50 µL StrepDARPidin + 36,3 µL PBS+2,5% glycerin incubated together for 1h at room temperature:</P><P>The peaks showed no shift to the front and ran into each other. They seemed to show no interaction. In order to be sure we tried to make a pulldown with Strep-Beads.</P></DIV></FIELDSET><FIELDSET class="exp21"><LEGEND><A name="exp21.5">21.5 Strep-Bead pulldown</A></LEGEND><DIV class="aim"><P>Aim: Checking interactions between Hag-D2-Strep and StrepDARPidin</P></DIV><DIV class="exp-content"><P>Strep-Beads from Novagene were used to check the interactions between the Strep-Tag in our Flagellin with the StrepDARPidin and the Strep-Beads themselves.</P><P>Theoretically the flagellin should bind to the beads and should be found in the elution in contrast to the StrepDARPidin which should be washed off. </P><P>In the third elution there should be less Flagellin or even nothing in case of an interaction with the StrepDARPidin because of the competitive situation. The flagellin should bind totally to the StrepDARPidin and for that reason not/ less to the beads. </P><P>3 columns were prepaired and filled with 500 µL GeFi Buffer. 20µL Strep-Beads were added and the columns were spinned down at 4000 rpm for 1 min. Then another 500 µL GeFi buffer was added.</P><P>The first (1) probe of 3 µL Hag-D2-Strep was pipetted into the buffer. 20 µL StrepDARPidin were added to column 2. Column 3 was used for loading a mixture of 20 µL StrepDARPidin and 3 µL Flagellin onto the column. The three columns were spinned for 20 min at room temperature in a spinning wheel. After that the columns were washed twice with 500 µL GeFi buffer. For eluting 40 µL Elution Buffer from Novagene (with Desthiobiotin) was used to resuspend the beads. After centrifugation at 13000 rpm the elution was mixed with 10 µL SDS-Loading buffer and analyzed on a SDS-PAGE gel with commassie stain.</P><P>As a control 40 µL of a 1:10 dilution of the flagellin concentrate was used on the gel. Unfortunately there was not enough StrepDARPidin left for a second control.</P><P>Gel-Analysis:</P><P>The gel shows that a very tiny amount of the flagellin binds to the Strep-Beads (line 1). The StrepDARPidin does not (line 2). Surprisingly the mixture shows both proteins in the elution (line 3). Unfortunately this cannot be seen on the scanned picture . The control (line 4) shows us that the flagellin purification was not pure enough and the flagellin concentrate was contaminated with StrepDARPidin from the Ni-NTA purification. </P><P>The Strep-Beads might have been not efficient working enough because of their age and usage. The binding of the flagellin might tell us that the Strep-Tag is able to interact with the Streptavidin.</P><P>In order to get a better and reliable result we decided to purify the flagellin and the StrepDARPidin newly and repeat the pulldown.</P><P>New expression cultures of 1L volume each were induced with lactose and incubated overnight at 30°C and 150 rpm.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="21.09.2014">21.09.2014</A></H2><FIELDSET class="exp23"><LEGEND><A name="exp23.23">23.23 PCR for Gibson assembly fragments Cup-1, D2-Cup and D2-Strep</A></LEGEND><DIV class="aim"><P>Aim: Start all fragments from the beginning to exclude mistakes in the early steps of the cloning procedure</P></DIV><DIV class="exp-content"><P>→ Templates piGEM021, piGEM 026 and piGEM 027.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Cup-1</TH><TH scope="col">D2-Cup</TH><TH scope="col">D2-Strep</TH></TR><TR><TH scope="row">Primer 1 (1:50)</TH><TD>24 → 6.5</TD><TD>96 → 6.5</TD><TD>96 → 6.5</TD></TR><TR><TH scope="row">Primer 2 (1:50)</TH><TD>25 →6.5</TD><TD>97 → 6.5</TD><TD>97 →; 6.5</TD></TR><TR><TH scope="row">Template</TH><TD>piGEM021 → 1</TD><TD>piGEM026 → 1</TD><TD>piGEM027 → 1</TD></TR><TR><TH scope="row">Phusion Buffer (5x)</TH><TD>10</TD><TD>10</TD><TD>10</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">dNTPs</TH><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>24</TD><TD>24</TD><TD>24</TD></TR></TBODY></TABLE><P>1:30 min elongation times was changed in standard PCR program. </P><P>The digested vector (over night) and the PCR products were purified via gel extraction for further use in Gibson assembly reactions. </P><P>All fragments have been amplified in the correct size with some side products (probably partially coming from the plasmids used as a backbone).</P><P>The digested plasmid yielded in a similar pattern like the last time (2 bands). Therefore the gel was run for additional 20 minutes and a third band could be seen. After additional 40 min it became clear that the middle thin band resembles the digested plasmid. It was therefore digested out and purified together with the PCR products.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.91">13.91 Ligation of piGEM007 and piGEM009 with the const. + tetO promoter</A></LEGEND><DIV class="aim"><P>Aim: insert the constitutive promoter with the tetO into the nose plasmids</P></DIV><DIV class="exp-content"><P>The ligation was carried out like in 13.85. The reaction was incubated at room temperature for over one hour and E. coli XLIBlue were transformed with the whole reaction mix.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="22.09.2014 ">22.09.2014 </A></H2><FIELDSET class="exp23"><LEGEND><A name="exp23.25">23.25 Plasmid isolation of pSB1C3-Hag-<I>Kpn</I>I for test restriction with <I>Kpn</I>I on the one hand and <I>Eco</I>RI/<I>Pst</I>I on the other hand</A></LEGEND><DIV class="aim"><P>Aim: Analyze the products to get information about purity of the plasmid</P></DIV><DIV class="exp-content"><P>Restriction results in the same pattern → idea cook DNA before restriction and shock on ice, restrict for appr. 3h afterwards</P></DIV></FIELDSET><FIELDSET class="exp23"><LEGEND><A name="exp23.26">23.26 Gibson assembly and transformation</A></LEGEND><DIV class="exp-content"><P>Digestion with <I>Eco</I>RI/<I>Pst</I>I showed that the plasmid is pure and not contaminated with other plasmids. Restriction with less DNA and cooking DNA before yielded in a thicker band for the hopefully digested plasmid at 3 kb. Band was digested out and purified via gel extraction (Omega Kit) for further Gibson assembly and transformation. </P></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.29">19.29 new Purification of Hag-D2-Strep</A></LEGEND><DIV class="aim"><P>Aim: Purification of Hag-D2-Strep for analytical gel filtration was already purified StrepDARPidin</P></DIV><DIV class="exp-content"><P>The cultures from yesterday were harvested at 4000 rpm at 4 °C and the pellets were resuspended in 10 mL Buffer A for microfluidizing. The lysate was centrifuged at 20000 rpm so that the clear lysate (Load/L) could be loaded on a 5 mL Ni-NTA column. Load (L), flow through (FT), wash (W) and elution with 15 mL Buffer B (E) were analysed on a SDS-PAGE gel coomassie stained:</P><P>FT and W contain much protein. The 1 mL Ni-NTA had not enough capacity to bind all the protein.</P><P>Meanwhile the elution was concentrated with an amicon at 4000 rpm until an endvolume of 1 mL. The ÄKTAPrime was buffered with GeFi Buffer and the concentrate was injected. Presets: 4 mL fraction size, 0,65 mPa pressure alarm, fractioning after 90 mL run, endvolme 400 mL. A Superdex S200 column was used.</P><P>The Ni-NTA column seemed to be not contaminated with StrepDARPidin. So the last contamination came form the left StrepDARPidin on the NI-NTA columns from previous use.</P><P>Unfortunately the S200 column seemed to be very dirty so that it was not possible to separate the proteins properly. </P><P>The fractions were collected and concentrated again. After that the concentrate was injected into ÄktaPrime again with a different S200 with the same settings as before. The run was done overnight.</P><P>Additionally new expression cultures with E. Coli BL21(DE3) containing piGEM-030 were induced with lactose for a new purification run.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.92">13.92 Colony PCR to check the ligation of piGEM0077 and piGEM009 with the const. + tetO promoter</A></LEGEND><DIV class="aim"><P>Aim: check the transformands for the correct plasmid</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix colony PCR</TH><TH scope="col">Single reaction</TH></TR><TR><TH scope="row">Template</TH><TD>part of a colony</TD></TR><TR><TH scope="row">Primer iGEM006 (1:50)</TH><TD>6.25</TD></TR><TR><TH scope="row">Primer iGEM063 (1:50)</TH><TD>6.25</TD></TR><TR><TH scope="row">5x HF-Buffer </TH><TD>10.0</TD></TR><TR><TH scope="row">dNTP mix (10 mM each)</TH><TD>0.7</TD></TR><TR><TH scope="row">Phusion DNA polymerase 1:10</TH><TD>1.0</TD></TR><TR><TH scope="row">Water</TH><TD>25.8</TD></TR><TR><TH scope="row">Total</TH><TD>50.0</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">colony PCR Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time (min/sec)KSI</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>10 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1:00 min </TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>infinite</TD></TR></TBODY></TABLE><P><I>E. coli</I> XLIBlue piGEM037 (002 + tetO) was used as a positive control. Clones 2 and 3 of piGEM007 + tetO were positive and cultures for a miniprep were inoculated with these clones.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="23.09.2014">23.09.2014</A></H2><FIELDSET class="exp23"><LEGEND><A name="exp23.1">23.27 Colony PCR (10 clones for all 3 constructs)</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">PCR Mix</TH><TH scope="col">1x Mix (µL)</TH><TH scope="col">11x Mix (µL]</TH></TR><TR><TH scope="row">Water</TH><TD>8.8</TD><TD>96.8</TD></TR><TR><TH scope="row">Buffer</TH><TD>4</TD><TD>44</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>1</TD><TD>11</TD></TR><TR><TH scope="row">Primer 1</TH><TD>2.6</TD><TD>28.6</TD></TR><TR><TH scope="row">Primer 2</TH><TD>2.6</TD><TD>28.6</TD></TR><TR><TH scope="row">dNTPs</TH><TD>1</TD><TD>11</TD></TR></TBODY></TABLE><P> Positive clones could only be detected for Cup-1. Clones 5 and 7 looked most promising and were inoculated in LB-Cm overnight for plasmid isolation and test-restriction. PCR for the other two modules has to be repeated or clones need to be inoculated for plasmid isolation and test-restriction. </P></DIV></FIELDSET><FIELDSET class="exp23"><LEGEND><A name="exp23.28">23.28 Digestion of pSB1C3-Hag-<I>Kpn</I>I</A></LEGEND><DIV class="aim"><P>Aim: Gain digested pSB1C3 plasmid for further cloning procedures</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Digestion Mix</TH><TH scope="col">Amount (µL)</TH></TR><TR><TH scope="row">Plasmid</TH><TD>5</TD></TR><TR><TH scope="row">Water</TH><TD>11</TD></TR><TR><TH scope="row">CutSmart  Buffer (10x)</TH><TD>2</TD></TR><TR><TH scope="row"><I>Eco</I>RI</TH><TD>1</TD></TR><TR><TH scope="row"><I>Pst</I>I</TH><TD>1</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.29a">19.29a new Purification of Hag-D2-Strep</A></LEGEND><DIV class="aim"><P>Aim: Purification of Hag-D2-Strep for analytical gel filtration was already purified StrepDARPidin</P></DIV><DIV class="exp-content"><P>The fractions building the peak were analyzed on a SDS-Gel to cover the peak. </P><P>The gel showed a degradation of our protein. For crystallization it is not adequate enough. A new purification with the expression cultures from yesterday was started.</P><P>The cultures from yesterday were harvested at 4000 rpm at 4 °C and the pellets were resuspended in 10 mL Buffer A for lysis by microfluidizer. The lysate was centrifuged at 20000 rpm so that the clear lysate (Load/L) could be loaded on a 5 mL Ni-NTA column. Load (L), flow through (FT), wash (W) and elution with 15 mL Buffer B (E) were analysed on a SDS-PAGE  coommassie stained:</P><P>The gel showed no contamination with StrepDARPidin as well. </P><P>The elution was concentrated with an amicon at 4000 rpm until an endvolume of 1,5 mL. The ÄKTAPrime was buffered with GeFi Buffer and the concentrate was injected. Presets: </P><P>4 mL fraction size, 0,65 mPa pressure alarm, fractioning after 90 mL run, endvolme 400 mL. A Superdex S200 column was used.</P><P>The fractions building the peak were analyzed on a SDS-Gel to cover the peak which looked fine.</P><P>Then the fractions were concentrated with an amicon to an endvolume of 300 µL. The new robot for setting drops was used for crystallization. Cores I-III were set in full concentration (A280=25) and half concentration diluted with GeFi-Buffer. Left hag-d2-Strep was aliquoted in 50 µL and stored at -80°C.</P><P>2 L expression culture were made for new purification attempts and incubated overnight at 30°C after lactose induction.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.93">13.93 Gibson assembly with piGEM008 and the Cu promoter and piGEM007/009 with the Ag promoter</A></LEGEND><DIV class="aim"><P>Aim: insert the Ag and Cu promoter into the Nose plasmids</P></DIV><DIV class="exp-content"><P>Since some plasmids with the silver and copper sensitive promoters were still missing a new Gibson assembly was carried out with the </P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Gibson I (µL)</TH><TH scope="col">Gibson II (µL)</TH><TH scope="col">Gibson III (µl)</TH><TH scope="col">Control  (µL)</TH></TR><TR><TH scope="row">piGEM-007/008/009 (approx.40 ng/µL)</TH><TD>4</TD><TD>2.5</TD><TD>1</TD><TD>2.5</TD></TR><TR><TH scope="row">Cu/Ag promoter (approx.4 ng/µl)</TH><TD>1</TD><TD>2.5</TD><TD>4</TD><TD>-</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>2.5</TD></TR><TR><TH scope="row">Gibson mix</TH><TD>15</TD><TD>15</TD><TD>15</TD><TD>15</TD></TR><TR><TH scope="row">Total</TH><TD>20</TD><TD>20</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><P>Nine reactions were carried out: piGEM 008 + Cu, piGEM009 + Ag and piGEM007 + Ag in three different variations in DNA concentration. The reactions were kept on room temperature for 30 seconds and incubated at 50°C for one hour. Afterwards the whole reaction mix was used to transform E. coli XLIBlue, which were incubated over night at 37°C.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="24.09.2014">24.09.2014</A></H2><FIELDSET class="exp23"><LEGEND><A name="exp23.1">23.29 Digest pSB1C3-Hag-<I>Kpn</I>I-Cup-1 <I>Eco</I>RI/<I>Pst</I>I and subsequent transformation</A></LEGEND><DIV class="exp-content"><P>Both prepared plasmids have been digested with <I>Eco</I>RI/<I>Pst</I>I and compared with the digested 13C-Hag-<I>Kpn</I>I. No significant difference could be detected. Therefore different Plasmids have been digested with <I>Eco</I>RI/<I>Pst</I>I and will be compared for the different inserts. </P><P>New transformation with sequenced plasmids → strepDARP clone 1 and Hag-<I>Kpn</I>I clone 4</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.72">18.72 Gelfiltration purification of StrepDARPidin tetramers</A></LEGEND><DIV class="aim"><P>Aim: Purification of StrepDARPidin from inclusion bodies</P></DIV><DIV class="exp-content"><P>10 L expression culture were inoculated with clones from a BL21(DE3) transformation plate and induced with lactose overnight at 30°C.</P></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.30">19.30 Purification of Hag-D2-Strep for crystallization</A></LEGEND><DIV class="aim"><P>Aim: Purification of Hag-D2-Strep for analytical gel filtration was already purified StrepDARPidin</P></DIV><DIV class="exp-content"><P>The cultures were harvested at 4000 rpm at 4°C and the pellet was frozen in liquid nitrogen and stored at -80°C for further use.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.93">13.93 Screening of transformands of Gibson assembly of piGEM008/009 and the Cu promoter and piGEM007 with the Ag promoter</A></LEGEND><DIV class="aim"><P>Aim: check the transformands for the correct plasmid</P></DIV><DIV class="exp-content"><P>A colony PCR was carried out to screen the transformands for the correct plasmids. No transformands could be detected on the plate with piGEM008 + Cu.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix colony PCR</TH><TH scope="col">Single reaction</TH></TR><TR><TH scope="row">Template</TH><TD>part of a colony</TD></TR><TR><TH scope="row">Primer iGEM006 (1:10)</TH><TD>1</TD></TR><TR><TH scope="row">Primer iGEM063 (1:10)</TH><TD>1</TD></TR><TR><TH scope="row">5x HF-Buffer </TH><TD>4</TD></TR><TR><TH scope="row">dNTP mix (10 mM each)</TH><TD>0.5</TD></TR><TR><TH scope="row">Phusion DNA polymerase 1:10</TH><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>12.5</TD></TR><TR><TH scope="row">Total</TH><TD>20</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">colony PCR Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time (min/sec)KSI</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>10 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1:00 min </TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>infinite</TD></TR></TBODY></TABLE><P>As a positive control E. coli XLIBlue piGEM034 (02 + Ag) was used. Clone 4 was positive. A LB culture was inoculated with clone 4 for a miniprep. It was grown shaking at 37°C over night.</P><P>It was noticed that the IPTG inducible Gram positive (described as lac) promoter, the strong constitutive for gram positive promoter and the tetO including variant were designed falsely. Only a part (30 bp) of the sequence was taken and cloned. That means that all nose plasmids containing the lac, const. and const. + tetO promoters were disposed. Only the plasmids with the metal ion sensitive promoters were used further.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.94">13.94 Production of competent <I>Bacillus subtilis</I> PY79 cells with test transformation</A></LEGEND><DIV class="aim"><P>Aim: Making PY 79 competent to take up foreign DNA</P></DIV><DIV class="exp-content"><P>Since the last PY79 cells were not competent a new try of producing competent cells was carried out according to the protocol in the methods section using SPC and SPII medium. After making the cells competent they were stored in a -80°C freezer without shock freezing in liquid nitrogen. The protocol does not including shock freezing of the cells. For a test transformation four aliquots of cells were thawed on room temperature and 100 µl each were transferred into sterile test tubes and mixed with 7 µl of the plasmids piGEM034, piGEM035, piGEM044 and piGEM050. The test tubes were incubated shaking at 37°C for 0.5 hours. The whole transformation samples were plated out on LB-chloramphenicol (5 µg/ml) plates and were incubated at 30°C for several days.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="25.09.2014">25.09.2014</A></H2><FIELDSET class="exp23"><LEGEND><A name="exp23.30">23.30 Inoculation of presumably positive clones from the tramsformation</A></LEGEND><DIV class="exp-content"><P>Transformation from yesterday: inoculate clones at 8 in order to isolate the plasmids, perform Gibson assembly and transformation. </P><P>Plasmid isolation → To exclude a that samples were exchanged by mistake a test-restriction of the plasmids has to be performed with <I>Eco</I>RI/<I>Kpn</I>I, since <I>Kpn</I>I does not digested in pSB1C3-Strep DARP. When cutting Hag-<I>Kpn</I>I a 600 bp fragment should occur in the gel whereas no fragment is expected when cutting pSB1C3-Strep DARP. According to the test restriction samples have not been switches, restrictions 1, 2 and 4 can be used for the Gibson assembly (and transformation). Restriction reaction 1 was used.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Digestion Mix</TH><TH scope="col">Amount (µL)</TH><TD rowspan="6">Expected:
	Strep Darp: only one fragment Hag-<I>Kpn</I>I: appr. 2330 + 670</TD></TR><TR><TH scope="row">Water</TH><TD>13.4</TD></TR><TR><TH scope="row">CutSmart  Buffer (10x)</TH><TD>2.0</TD></TR><TR><TH scope="row">Plamid (100 ng)</TH><TD>4.0</TD></TR><TR><TH scope="row"><I>Eco</I>RI</TH><TD>0.3</TD></TR><TR><TH scope="row"><I>Kpn</I>I</TH><TD>0.3</TD></TR></TBODY></TABLE><P>Restriction (25µl mix with complete plasmid)</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Digestion Mix</TH><TH scope="col">Amount (µL)</TH></TR><TR><TH scope="row">Water</TH><TD>1.5</TD></TR><TR><TH scope="row">CutSmart  Buffer (10x)</TH><TD>2.5</TD></TR><TR><TH scope="row">Plasmid (500 ng)</TH><TD>20.0</TD></TR><TR><TH scope="row"><I>Kpn</I>I</TH><TD>1.0</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.72">18.72 Gelfiltration purification of StrepDARPidin tetramers</A></LEGEND><DIV class="aim"><P>Aim: Purification of StrepDARPidin from inclusion bodies</P></DIV><DIV class="exp-content"><P>The cells from yesterday were harvested and cracked with the microfluidizer after resuspension in 40 mL Buffer A (lysate)). </P><P>The suspension was centrifuged at 20.000 x g for 15 min. The supernatant (S1) was discarded  and the pellet (P1) resuspended in 5 mL IB Wash Buffer. Then the suspension (SS1) was centrifuged at 27.000 x g for 10 min. The resulting pellet (P2) and the supernatant (S2) were used for taking probes for SDS-PAGE. The wash was repeated. 10 mL 6M Guanidinium-HCL solution for solubilisation and unfolding of the StrepDARPidin was added and the pellet resuspended. The suspension was incubated at room temperature permanently stirring with a magnet.</P><P>After centrifugation at 4000 rpm for 10 min the supernatant was used for refolding. The membranous pellet (P3) was thrown away.</P><P>For refolding 5 mL of the Guanidinium-HCL 100 mL Buffer A were filled into a Erlenmeyer-Flask on ice with a magnet stirrer inside on 600 rpm. The other 5 mL Guanidinium-HCL/ Pellet resuspension was stored in the fridge until further use.</P><P>For refolding it was necessary to dilute the Guanidinium-HCL dilution very fast and softly. For that purpose rapid dilution was done by pipetting the supernatant drop by drop into the vortex of the Buffer A. After adding the whole Guanidinum-HCL/ Protein solution the suspension was observed and precipitation noticed. The suspension was centrifuged at 20.000 rpm (Pellet P4) . The supernatant was loaded on a Ni-NTA Column from GE-Healthcare (L). Flow through (FT), Wash (20 mL W) in Buffer A and Elution (15 mL) (E) in Buffer B were kept for gel analysis. The samples lysate to Load were cooked for 5 min at 95 degrees.</P><P>From every fraction 40 µL were taken for a coommassie SDS-PAGE and 10 µL 10x SDS Buffer added. The pellet samples were resuspended in 60 µL water and 40 µL SDS-Buffer.</P><P>The gel shows that the monomeric and tetrameric StrepDARPidin can be found in Load, Flow through and elution. The forming of the tetramer tells us that the refolding had to be successful partially.  The elution was concentrated with an Amicon to an endvolume of 1,5 mL. 75 µL glycerol were added and the concentrate was frozen away in liquid nitrogen and stored at -80°C for further use.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.95">13.95 restriction of piGEM007/008 and 009 with <I>Nco</I>I and <I>Sac</I>I</A></LEGEND><DIV class="aim"><P>Aim: linearize the plasmids to for further cloning of the promoters</P></DIV><DIV class="exp-content"><P>The plasmids with the ssrA-degradation tags were digested with <I>Nco</I>I and <I>Sac</I>I to linearize them for further cloning of promoters in front of the GFP. Different concentrations of plasmid were digested with the enzymes to test the capacity of the restriction enzymes and to be sure that all plasmid was cut.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Concentration I (µL)</TH><TH scope="col">Concentration II (µL)</TH><TH scope="col">Concentration III (µL)</TH></TR><TR><TH scope="row">piGEM-007/008/009 (approx.400 ng/µL)</TH><TD>2</TD><TD>5</TD><TD>7</TD></TR><TR><TH scope="row"><I>Nco</I>I</TH><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row"><I>Sac</I>I</TH><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">CutSmart  Buffer 10x</TH><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR><TH scope="row">Water</TH><TD>14</TD><TD>11</TD><TD>9</TD></TR><TR><TH scope="row">Total</TH><TD>20</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><P>The reaction was incubated at 37°C for over one hour and afterwards separated on a gel. As a control the undigested piGEM008 was used.</P><P>No difference between the undigested and the digested plasmids could be detected, but a small band directly under the thick, visible one. The linearized plasmids should have a size of 6711 bp. It is possible that the undigested plasmids do rarely form supercoiled structures and mostly behave like the linear form, only a small part of it is supercoiled. Since the same plasmids were digested before with the same enzymes and looked the same on the gel but were linear (further cloning of promoter sequences otherwise not possible), the plasmids were assumed to be linearized. The three concentrations of each plasmid were pooled and were purified with an Omega Gel Ex kit.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="26.09.2014">26.09.2014</A></H2><FIELDSET class="exp23"><LEGEND><A name="exp23.31">23.31 Plasmid isolation and new Gibson assembly</A></LEGEND><DIV class="exp-content"><P>No colonies on plates. New Miniprep with much higher concentrations of plasmid → yesterday mostly only 20 ng/µl. Band before gel extraction was very thin. </P><P>New Gibson assembly and Transformation. Clones will be inoculated tomorrow; spin down and pellets will be frozen at -20°C.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.71">18.71 Gelfiltration purification of StrepDARPidin tetramers</A></LEGEND><DIV class="aim"><P>Aim: Purification of StrepDARPidin from inclusion bodies</P></DIV><DIV class="exp-content"><P>The stored protein concentrate was thawn up on ice and injected into the injection valve 
of the ÄktaPrime which was buffered with PBS+ 2,5% glycerin. A S200 Sepharose 
Column was used for the gel filtration run.</P><P>40 µL of the Fraction 31. 37, 38 and 44 were picked for SDS-PAGE analysis. 10 µL SDS-Buffer were added and 20 µL loaded on the gel. Additionally the rest was boiled and the same amount loaded onto the gel. </P><P>The gel shows us that the whole peak contains the tetrameric StrepDARPidin. After cooking it up the tetramer dissolves into the monomeric StrepDARPidin.</P><P>The fractions covering the peak were concentrated with an amicon to an endvolume of 500 µL. 25 µL glycerin were added and the protein aliquoted in 60 µL aliquods with an Absorption of 8 A<SUB>280</SUB> nm.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND>13.96 Gibson Assembly of piGEM007 +Ag promoter and piGEM008/009 with Cu promoter</LEGEND><DIV class="aim"><P>Aim: insert the promoter in the nose plasmids</P></DIV><DIV class="exp-content"><P>A new Gibson assembly was carried out to insert the missing promoters into the different nose plasmids. Every combination (007 + Ag, 008 + Cu, 009 + Cu) was carried out in different relations of vector to insert: </P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">equal amounts (µL)</TH><TH scope="col">condition II for 07 + Ag (µL)</TH><TH scope="col">condition II for 08 + Cu (µL)</TH><TH scope="col">condition II for 09 + Cu (µL)</TH></TR><TR><TH scope="row">piGEM-007/008/009 (approx.40 ng/µL)</TH><TD>2.5</TD><TD>2.5</TD><TD>2.5</TD><TD>2.5</TD></TR><TR><TH scope="row">Cu/Ag promoter (approx.5 ng/µl)</TH><TD>2.5</TD><TD>1.2</TD><TD>1.4</TD><TD>0.95</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O</TH><TD>-</TD><TD>1.3</TD><TD>1.1 </TD><TD>1.55</TD></TR><TR><TH scope="row">Gibson mix</TH><TD>15</TD><TD>15</TD><TD>15</TD><TD>15</TD></TR><TR><TH scope="row">Total</TH><TD>20</TD><TD>20</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><P>The reactions were kept on room temperature for 0.5 min and were incubated at 50°C for 1 h. Afterwards E. coli XLIBlue were transformed with the whole reaction mix, plated on LB-ampicillin and incubated over night at 37°C. </P><P>The existent nose plasmids (piGEM034/035/039/044/050) were previously transformed into DH5α cells, which were plated out new to inoculate cryo stocks of the cells.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.97">13.97 Transformation of <I>Bacillus subtilis</I> PY79 cel</A>ls</LEGEND><DIV class="aim"><P>Aim: comparison of competence of PY79 cells</P></DIV><DIV class="exp-content"><P>To compare the competence of our PY79 cells with cells that are competent for sure, one aliquot of competent PY79 were borrowed together with three LB-chloramphenicol plates. The 300 µl aliquot was divided into 100 µl portions. One negative control was mixed with 10 µl water, the other two 100 µl portions were treated with 5 µl and 10 µl piGEM034 (approx.400 ng/µl), which has been shown that it could be transformed. The cell/DNA mixture was incubated in a test tube, shaking for 30 minutes before plating them out on the mentioned medium plates. This time the plates were incubated at 37°C.</P></DIV></FIELDSET><FIELDSET class="exp22"><LEGEND><A name="exp22.16">22.16 amplification of KSII in single modules</A></LEGEND><DIV class="aim"><P>Aim: amplification of KSII</P></DIV><DIV class="exp-content"><P>KSII could not be amplified with the nesting primers which cover the whole KSII module, even with a variety of modified reaction conditions. Thus, primers were ordered to amplify the modules (holing, tetR, rplE (L5)) separately.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">holin (µL)</TH><TH scope="col">tetR (µL)</TH><TH scope="col">L5 (µL)</TH></TR><TR><TH scope="row">KSII gBlock DNA (1:10)</TH><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Primer iGEM065 (1:50)</TH><TD>6.25</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM069 (1:50)</TH><TD>-</TD><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM067 (1:50)</TH><TD>-</TD><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM068 (1:50)</TH><TD>-</TD><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Primer iGEM066 (1:50)</TH><TD>-</TD><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Primer iGEM070 (1:50)</TH><TD>-</TD><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">HF Buffer 5x</TH><TD>10</TD><TD>10</TD><TD>10</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">dNTPs</TH><TD>0.5</TD><TD>0.5</TD><TD>0.5</TD></TR><TR><TH scope="row">Water</TH><TD>25</TD><TD>25</TD><TD>25</TD></TR><TR><TH scope="row">Total</TH><TD>50</TD><TD>50</TD><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">colony PCR Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time (min/sec)KSI</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1:00 min </TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The sizes of the expected fragments were: holin: approx.700 bp, tetR: 684 bp and L5: 540 bp. The fragments of the expected size could be seen, it meets the eye that the unspecific bands that appeared in previous amplifications of KSII with the nesting primers were visible as well in the lane of tetR. However, tetR also showed an amplificate with the correct size.</P><P>Although the KSII parts were amplified successfully no further work will be done with KSII, because of the false promoter in front of these genes. Like mentioned before the IPTG inducible promoter for Gram positives and the strong constitutive promoter for Gram positives used in the killswitch and nose and taken from the iGEM registry were falsely designed. Only a short part of the real sequence was used.</P><P>For this reason the promoters were designed again correctly: the lac promoter (BBa_K090501; 107 bp) will be ordered in single stranded oligonucleotides from which one contains overhangs like it was digested with <I>Nco</I>I and <I>Sac</I>I. These oligos will be annealed and phosphorylated. A ligation with the nose plasmids and KSI follows. It was only enough time left to build KSI anew and not the other killswitch modules. The primer to connect the new lac promoter with the rest of KSI also introduces a RBS.</P><P>The constitutive promoter (BBa_K090504) will be amplified from gDNA of <I>Bacillus cereus</I>, which was kindly provided by members of the Zentralinstitut für Ernährungs- und Lebensmittelforschung ZIEL, Technische Universität München. The amplificate will be used for ligation with the nose plasmids.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="27.09.2014">27.09.2014</A></H2><FIELDSET class="exp13"><LEGEND><A name="exp18.73">18.73 Gelfiltration purification of StrepDARPidin tetramers</A></LEGEND><DIV class="aim"><P>Aim: Purification of StrepDARPidin from inclusion bodies</P></DIV><DIV class="exp-content"><P>The left 5 mL Gua-suspension in the fridge were refolded with the same protocol from 18.72.</P><P>P1 shows a very weak line at the expected size which comes from picking just a membranous part on top of of the pellet. The line would have been more prominent if we had taken more of the pellet itself. The gel shows that the monomeric and tetrameric StrepDARPidin can be found in Load, Flow through and elution. The forming of the tetramer tells us that the refolding had to be successful partially.  The elution was concentrated with an Amicon to an endvolume of 1,5 mL. 75 µL glycerol were added and the concentrate was frozen away in liquid nitrogen and stored at -80°C for further use.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="28.09.2014">28.09.2014</A></H2><FIELDSET class="exp23"><LEGEND><A name="exp23.32">23.32 Test digest</A></LEGEND><DIV class="exp-content"><P>Saturday clones could be observed on all plates (more than on the control plate). 5 clones have been inoculated in LB-Cm and were centrifuged and frozen in the evening. Plasmid isolation were performed in the evening and restriction was performed over night. </P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Digestion mix</TH><TH scope="col">1x Mix (µL)</TH><TH scope="col">18x Mix (µL]</TH></TR><TR><TH scope="row">Plasmid</TH><TD>10</TD><TD>x</TD></TR><TR><TH scope="row">Water</TH><TD>7.4</TD><TD>133.2</TD></TR><TR><TH scope="row">Buffer</TH><TD>2.0</TD><TD>36.0</TD></TR><TR><TH scope="row"><I>Eco</I>RI</TH><TD>0.3</TD><TD>5.4</TD></TR><TR><TH scope="row"><I>Pst</I>I</TH><TD>0.3</TD><TD>5.4</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp13.71">18.71 Gelfiltration purification of StrepDARPidin tetramers</A></LEGEND><DIV class="aim"><P>Aim: Purification of StrepDARPidin from inclusion bodies</P></DIV><DIV class="exp-content"><P>The stored protein concentrate was thawn up on ice and injected into the injection valve 
of the ÄktaPrime which was buffered with PBS+ 2,5% glycerin. A S200 Sepharose 
Column was used for the gel filtration run.</P><P>40 µL of the Fraction 32. 38, 39 and 45 were picked for SDS-PAGE analysis. 10 µL SDS-Buffer were added and 20 µL loaded on the gel. Additionally the rest was boiled and the same amount loaded onto the gel. </P><P>The gel shows us that the whole peak contains the tetrameric StrepDARPidin. After boiling it the tetramer dissolves into the monomeric StrepDARPidin.</P><P>The fractions covering the peak were concentrated with an amicon to an endvolume of 500 µL. 25 µL glycerin were added and the protein aliquoded in 60 µL aliquods with an Absorption of 7 A280 nm.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="29.09.2014">29.09.2014</A></H2><FIELDSET class="exp23"><LEGEND><A name="exp23.33">23.33 Analysis of the test digest</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Expected size</TH><TH scope="col">Cup-1</TH><TH scope="col">D2-Strep</TH><TH scope="col">D2-Cup</TH></TR><TR><TH scope="row">Plasmid</TH><TD>2 kb</TD><TD>2 kb</TD><TD>2 kb</TD></TR><TR><TH scope="row">Insert</TH><TD>1043 bp</TD><TD>941 bp</TD><TD>1463 bp</TD></TR><TR><TH scope="row">2<SUP>nd</SUP> insert</TH><TD>x</TD><TD>450</TD><TD>x</TD></TR></TBODY></TABLE><P>Presumably Cup-1 clones 3-5 are positive, all D2-Strep clones are positive but D2-Cup clones are negative. Therefore PCR reactions will be performed with the positive plasmids and colony PCR with 10 clones of D2-Cup will be added.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">PCR Mix</TH><TH scope="col">Cup-1</TH><TH scope="col">D2-Cup</TH><TH scope="col">D2-Strep</TH></TR><TR><TH scope="row">Expected size</TH><TD>746 bp</TD><TD>1103 bp</TD><TD>1166 bp</TD></TR><TR><TH scope="row">Water</TH><TD>33.2</TD><TD>102.3</TD><TD>49.8</TD></TR><TR><TH scope="row">Buffer (5x)</TH><TD>16</TD><TD>44</TD><TD>24</TD></TR><TR><TH scope="row">Primer 1 (1:50)</TH><TD>10.4 (iGEM 025)</TD><TD>28.6 (iGEM 053)</TD><TD>15.6 (iGEM 053)</TD></TR><TR><TH scope="row">Primer 2 (1:50)</TH><TD>10.4 (iGEM 053)</TD><TD>28.6 (iGEM 097)</TD><TD>15.6 (iGEM 097)</TD></TR><TR><TH scope="row">dNTPs</TH><TD>4</TD><TD>11</TD><TD>6</TD></TR><TR><TH scope="row">Phusion</TH><TD>4</TD><TD>11</TD><TD>6</TD></TR><TR><TH scope="row">Template</TH><TD>0.5</TD><TD>x</TD><TD>0.5</TD></TR></TBODY></TABLE><P>1:30 Elongation time.</P><P>Agarose gel: Colony PCR for Hag-<I>Kpn</I>I-D2-Cup is negative, but all other results from the test restriction have been confirmed.</P></DIV></FIELDSET><FIELDSET class="exp25"><LEGEND><A name="exp25.3">25.3 Preparation of LLCs</A></LEGEND><DIV class="aim"><P>Aim: Preparation of LLCs for assays</P></DIV><DIV class="exp-content"><P>We decided to cultivate the LLCs for Fluorometer assays in 12-Well plates.  The medium of the 25 mL culture flask was used to resuspend the cells carefully because of their sparsely adherent  trait.</P><P>The cells were transferred into a 50 mL the  culture was spinned down at 1500 rpm  for 5 min at 4°C. </P><P>The supernatant was discarded and resuspended in 10 mL DMEM. The cells were counted with a Neubauer-chamber.</P><P>10 µL of the cell suspension was mixed with 200 µL Trypan blue and 85-100 cells were counted in the grids of the chamber.</P><P>Calculation:</P><P>90 (cells total in grid) / 4 (number of quadrants) *20 (dilution factor) * 10000 (Neubauer factor) * 10 mL (volume of suspension)= approx. 40 mio. Cells/ 10 mL</P><P>We calculated with 50 mio. Cells in total.</P><P>The suspension was filled up to 50 mL to get 1 mio cells/ mL. 2 mL/ well were pipetted into each well of the 12-well plate and incubated overnight at 37°C.</P><P>A Black Fluotrac600 96-well plate was kindly received from the immunology. Row H 1-12 was coated with 100000 cells/ well overnight at 37°C.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.98">13.98 Screening of the transformands from the Gibson by colony PCR</A></LEGEND><DIV class="aim"><P>Aim: check the transformands for the correct plasmid</P></DIV><DIV class="exp-content"><P>A colony PCR was carried out to screen the transformands for the correct plasmids. No transformands could be detected on the plate with piGEM008 + Cu.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix colony PCR</TH><TH scope="col">Single reaction</TH></TR><TR><TH scope="row">Template</TH><TD>part of a colony</TD></TR><TR><TH scope="row">Primer iGEM006 (1:10)</TH><TD>1</TD></TR><TR><TH scope="row">Primer iGEM057 (1:10)</TH><TD>1</TD></TR><TR><TH scope="row">5x HF-Buffer </TH><TD>4</TD></TR><TR><TH scope="row">dNTP mix (10 mM each)</TH><TD>0.5</TD></TR><TR><TH scope="row">Phusion DNA polymerase 1:10</TH><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>12.5</TD></TR><TR><TH scope="row">Total (µL)</TH><TD>20</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">PCR Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time (min/sec)KSI</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>10 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1:00 min </TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>infinite</TD></TR></TBODY></TABLE><P>As a positive control <I>E. coli</I> XLIBlue piGEM034 and piGEM035 were used. No positive clones could be seen. Indeed there was a band at the right size of clone 2 07 + Ag but there was a bigger and a smaller band as well.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.99">13.99 Competent <I>Bacillus subtilis</I> PY79</A></LEGEND><DIV class="exp-content"><P>The transformation of the previus batch of cells with the plasmid piGEM034 worked only in case of the higher concentration. Two colonies could be observed. For the next competent cells fructose will be used instead of glucose. For making of new competent PY79 cells the cells were plated out again on a new LB plate and were incubated at 37°C over night.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="30.09.2014">30.09.2014</A></H2><FIELDSET class="exp23"><LEGEND><A name="exp23.34">23.34 Repeated agarose gel of the colony PCR</A></LEGEND><DIV class="exp-content"><P>Better image, bands occur exactly at expected size. Clone 3 from the Cup-1 clones will be sent for sequencing today. Clone 1 - 4 will be used for mutagenesis, when mutagenesis primer arrives. </P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Label number</TH><TH scope="col">Construct</TH><TH scope="col">Added Primer</TH></TR><TR><TD>AGB0023-<STRONG>557</STRONG></TD><TD>1C3-Hag <I>Kpn</I>I-Cup1</TD><TD>piGEM 053</TD></TR><TR><TD>AGB0023-<STRONG>558</STRONG></TD><TD>1C3-Hag <I>Kpn</I>I-Cup1</TD><TD>piGEM 054</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.72">18.72 Isolation of flagella from Hag-D2-Strep</A></LEGEND><DIV class="aim"><P>Aim: Isolation of filaments for assays with StrepDARPidin</P></DIV><DIV class="exp-content"><P>The flagella isolation was tried for the strain Hag-D2-Strep according to the protocol from ''Purification and Thermal Stability of Intact <I>Bacillus subtilis</I> Flagella''.</P></DIV></FIELDSET><FIELDSET class="exp25"><LEGEND><A name="exp25.3a">25.3a Preparation of LLCs and A549</A></LEGEND><DIV class="aim"><P>Aim: Preparation of LLCs and A549 for assays</P></DIV><DIV class="exp-content"><P>Because of the literature we found we decided to throw the LLCs for now into the waste. We did not find enough information about the EpCAM expression in that cell line.</P><P>Instead we prepared the A549 for a Fluophor-Assay tomorrow.  The 25 mL flask was splitted 1:3. The medium of the culture flask was discarded from the side opposite to the adherent cells and washed with 1X PBS from the opposite site as well.</P><P>The PBS was aspirated with a glass pipette until the supernatant was clear. After adding 4 mL 1x trypsin from the opposite of the adherent cells the flask was incubated for 5 min at 37°C depending on how fast the cells come off from the ground. Under the microscope the free floating cells were checked.</P><P>The cells were transferred into a 15 mL falcon and rinsed with 10 mL DMEM to collect all remaining cells in the flask. The 15 mL falcon was filled with DMEM and the culture spinned down at 1500 rpm  for 5 min at 4°C.</P><P>The supernatant was discarded and resuspended in 5 mL DMEM.</P><P>10 µL of the cell suspension was mixed with 200 µL Trypan blue and approx. 140 cells were counted in the grids of the chamber.</P><P>Calculation:</P><P>140 (cells total in grid) / 4 (number of quadrants) *20 (dilution factor) * 10000 (Neubauer factor) * 5 mL (volume of suspension)= approx. 35 mio. Cells/ 10 mL</P><P>The cells were splitted. 2 mL :1 mL : 1 mL of cell suspension was filled into 3 new 15 mL flasks and filled up with 10 mL DMEM+10%FCS and L-Glutamin. The 2mL flask was planned for microscopy on Friday with the StrepDARPidin constructs. The other flasks are for backups and assays. Incubation was performed at 37°C.</P><P>Additionally  row H1-12 of the Fluotrac600 was coated with 25 µL cell suspension and 100 µL DMEM +10%FCS+L-Glut. (incubation at 37°C).</P><P>As a backup Row F of a 6x8 was filled with 25 µL cell suspension as well and 300 µL DMEM +10%FCS+L-Glut. (incubation at 37°C).</P></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.30">19.30 Purification of Hag-D2-Strep for crystallization</A></LEGEND><DIV class="aim"><P>Aim: Purification of Hag-D2-Strep for crystallization</P></DIV><DIV class="exp-content"><P>The pellets from 23.09.14  were resuspended in 10 mL Buffer A for microfluidizing. The lysat was centrifuged at 20000 rpm so that the clear lysat could be loaded on a 5 mL Ni-NTA column. Load (L), flow through (FT), wash (W) and elution with 15 mL Buffer B (E) were analysed on a SDS-PAGE gel commassie stained:</P><P>Further purification was done. Meanwhile the elution was concentrated with an amicon at 4000 rpm until an endvolume of 1 mL. The ÄKTApurifier was buffered with GeFi Buffer and the concentrate was injected. Presets: 4 mL fraction size, 0,65 mPa pressure alarm, fractioning after 90 mL run, endvolume 400 mL. A Superdex S200 column was used.</P><P>Then the fractions building the peak were concentrated with an amicon to an endvolume of 500 µL which were used for setting dropes with core I and IV (QIAGEN cores). Aliquods of left flagellin of 50 µL were frozen in liquid nitrogen and stored at -80°C.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.100a">13.100a Competent<I>Bacillus subtilis</I> PY79</A></LEGEND><DIV class="aim"><P>Aim: making PY79 cells competent</P></DIV><DIV class="exp-content"><P>The SPC medium and the SPII medium were assembled. The media were prewarmed before the cells were transferred into the media. New LB-chloramphenicol plates were made. After making the cells competent their competence was tested by transformation of piGEM034.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.101">13.101 Restriction of piGEM007/008/009</A></LEGEND><DIV class="aim"><P>Aim: linearization of the plasmids and test of the restriction enzymes</P></DIV><DIV class="exp-content"><P>The yield of transformands after ligations or Gibson assemblies with the plasmids piGEM007, piGEM008 and piGEm009 was erratically fewer than with the piGEM002, which was digested a few weeks ago. Since the only difference between these plasmids are the ssrA-degradation tags on the gfp-gene, there should be no difference in their restriction behavior. This is aggravated by the fact that all four plasmids contain the restriction sites for <I>Nco</I>I and <I>Sac</I>I on the same positions. For this reason the plasmids were digested again, together with test plasmids to check the activity of the enzymes <I>Nco</I>I and <I>Sac</I>I. The test plasmids were pMAD that contains two <I>Sac</I>I sites, thus resulting in fragments of 4431 bp and 5235 bp size and piGEM002, digested by <I>Nco</I>I and <I>Eco</I>RI yielding in fragments of 476 bp and 6190 bp.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Restriction of 07/08/09 (µL)</TH><TH scope="col">Test I (µL)</TH><TH scope="col">Test I (µL)</TH></TR><TR><TH scope="row">piGEM-007/008/009 (200-300 ng/µL)</TH><TD>10</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">piGEM002 (approx.400 ng/µl)</TH><TD>-</TD><TD>2</TD><TD>-</TD></TR><TR><TH scope="row">pMAD (114 ng/µl)</TH><TD>-</TD><TD>-</TD><TD>2</TD></TR><TR><TH scope="row"><I>Nco</I>I</TH><TD>1</TD><TD>1</TD><TD>-</TD></TR><TR><TH scope="row"><I>Sac</I>I</TH><TD>1</TD><TD>-</TD><TD>1</TD></TR><TR><TH scope="row"><I>Eco</I>RI</TH><TD>-</TD><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">CutSmart  Buffer 10x</TH><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR><TH scope="row">Water</TH><TD>14</TD><TD>14</TD><TD>15</TD></TR><TR><TH scope="row">Total</TH><TD>20</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><P>The reactions were incubated at 37°C for over one hour.</P><P>According to the bands <I>Nco</I>I was functional, the expected bands could be seen after the restriction of piGEM002. However, the functionality of <I>Sac</I>I was not fully determined since there was no control with the undigested plasmid. A band in the height between 5000 and 6000 bp could be seen, another very slight band at approx.4 kb could also be noticed, what seems to be the other expected fragment. Yet it the bigger band was more intense than the smaller one. The digested plasmids piGEM007, 008 and 009 were in had the correct size and seemed to be digested. These bands were pooled and purified via a Gel Ex kit.</P></DIV></FIELDSET></DIV></FIELDSET></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Marburg:Project:Notebook:September">http://2014.igem.org/Team:Marburg:Project:Notebook:September</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Marburg:Project:Notebook:September" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Marburg:Project:Notebook:September" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Marburg:Project:Notebook:September&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Marburg:Project:Notebook:September&amp;oldid=381671" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>