"
Page
Discussion
View source
History
teams
Log in
 
Team:Marburg:Project:Notebook:September
From 2014.igem.org
Home
Project
Safety
Team
Policy & Practices
Attributions
SilverSURF
CancerSURF
RiboSURF
Notebook
Parts
Notebook: September
March
April
May
June
July
August
September
01.09.2014
02.09.2014
03.09.2014
04.09.2014
05.09.2014
06.09.2014
07.09.2014
08.09.2014
09.09.2014
10.09.2014
11.09.2014
12.09.2014
13.09.2014
14.09.2014
15.09.2014
16.09.2014
17.09.2014
18.09.2014
19.09.2014
20.09.2014
21.09.2014
22.09.2014
23.09.2014
24.09.2014
25.09.2014
26.09.2014
27.09.2014
28.09.2014
29.09.2014
30.09.2014
October
Methods
01.09.2014
23.2 Transformation of competent E. coli DH5α with pSB1C3 BB8
Aim: Isolate the plasmid in order to use the pSB1C3 backbone for the cloning of our biobricks.The contained brick will be digested out and the backbone will be used together with our inserts to clone the vectors.
Standard E. coli transformation protocol was followed and cells were plated on LB-CM plates.
Sequencing results of constructs from 29.08.2014
Premix
Label-Nr.
Construct
Result
1
AGB0023512
PCR Fragment StrepDARPidin I
fine
2
AGB0023513
PCR Fragment StrepDARPidin II
fine
3
AGB0023514
PCR Fragment D2-StrepII
fine
4
AGB0023515
PCR Fragment StrepDARPidin I
fine
5
AGB0023516
PCR Fragment StrepDARPidin II
fine
6
AGB0023517
PCR Fragment D2-StrepII
fine
18.65 Test digest of isolated clones from 18.63
Aim: Checking insertion of StrepDARPidin into pET16b
The isolated plasmids from the positive clones were digested with NcoI/ XhoI in order to see if the 900 bp were flipping out of the pET16b backbone.
Mastermix (µL)
Mix per samle (µL)
pET16b-Insert (approx. 50-80 ng/ µL)
-
8
CutSmart  10x
10
-
NcoI
0.5
-
XhoI
0.5
-
Water
9
-
Mastermix
-
2
Total
20
10
Incubation at 37°C for 40 min.
The gel showed that all picked clones were positive. Clone 17 was sent was sequencing and transformed into E. coli BL21(DE3).
Sequencing results of constructs from 01.09.2014
Premix
Label-Nr.
Construct
Primer
1
AGB0023518
piGEM-027 I1 (pMAD-D2-Strep)
Flo96 D2_3 fw
2
AGB0023519
piGEM-027 I3 (pMAD-D2-Strep)
Flo96 D2_3 fw
3
AGB0023520
piGEM-027 I6 (pMAD-D2-Strep)
Flo96 D2_3 fw
4
AGB0023521
piGEM-027 II2 (pMAD-D2-Strep)
Flo96 D2_3 rv
5
AGB0023551
piGEM-028 clone 7
T7
6
AGB0023552
piGEM-028 clone 7
T7 term
18.66 Expression-Test with transformed E. coli BL21 (DE3)
Aim: Checking the overexpression of pET24d-StrepDARPidin
In order to check the expression of the StrepDARPidin a colony containing piGEM-028 (pET16b-StrepDARPidin) was used to inoculate 20 mL of LB-Amp. The culture incubated at 37°C until an OD of 0,7.
A approx. 1 mL preinduction probe (PI) was taken (0,7 (favoured OD) : 0,7 (actual OD) = 1 mL). After that the cultures were induced with 20 µL IPTG for 3h.
An induction probe (I) was taken (320 µL with an OD of 2,2; 350 µL with an OD of 2).
PI and I were centrifuged at 14000 rpm for 1 min, the supernatant was discarded and the pellet resuspended in 60 µL water and 40 µL SDS-Buffer.
The four probes were analysed on an SDS-PAGE gel with 10 µL volume per probe.
The gel showed a positive overexpression of a protein at approx. 30 kDa which fits the calculated molecular mass.
22.5 Repeated amplification of killswitch module 2 and flanks
Aim: cleaner amplification of KSII, since there always were many unspecific bands and the expected band was very weak. The same went for the flanks.
Mix
amyE FlankI
amyE FlankII
lacA FlankI
lacA FlankII
amyE-yvyD FlankII
KSII
Template (Δ51 fac gDNA 1:10)
2
2
2
2
2
-
Template (gBlock DNA KSII)
-
-
-
-
-
0.5
Primer iGEM-34
1
-
-
-
-
-
Primer iGEM-35
1
-
-
-
-
-
Primer iGEM-36
-
1
-
-
-
-
Primer iGEM-37
-
1
-
-
-
-
Primer iGEM-40
-
-
1
-
-
-
Primer iGEM-41
-
-
1
-
-
-
Primer iGEM-42
-
-
-
1
-
-
Primer iGEM-43
-
-
-
1
-
-
Primer iGEM-48
-
-
-
-
1
-
Primer iGEM-37
-
-
-
-
1
-
Primer iGEM-44
-
-
-
-
-
1
Primer iGEM-45
-
-
-
-
-
1
dNTPs
1
1
1
1
1
1
5x HF-buffer
10
10
10
10
10
10
Phusion polymerase
1
1
1
1
1
1
Water
34
34
34
34
34
34
Total  (µl)
50
50
50
50
50
50
The amplification of KSII did not work: some unspecific bands were visible but not of the expected size. The amplification of flank 1 amyE fw, flank 3 lacA fw and flank 4 lacA rev was successful but yielded in a low amount of PCR product.
13.79 Transformation of Bacillus subtilis PY79 with the Nose-plasmids
Aim: insertion of the Nose-plasmids into B. subtilis in order to test the promoters.
The plasmids of 13.78 were isolated at the weekend. Frozen and competent B. subtilis cells were thawed without ice. 15 µl of each plasmid were given to the cells that were incubated at 37°C for 0.5 hours. After regeneration the cells were plated on LB-Cm plates (5 g/µl) and were incubated at 30°C for several days.
02.09.2014
23.3 Inoculation of transformants
Aim: Grow cultures for the isolation of plasmid psB1C3 BB8 transformants were inoculated in liquid LB-CM (10:30) for plasmid isolation.
23.4 Plasmid isolation and digest of pSB1C3, Hag-KpnI and StrepDARPidin
Aim: Gain digested plasmid and inserts for ligation of our first two biobricks
Test digest with the isolated plasmid was performed with the corresponding enzymes (EcoRI and PstI) over night. Besides 600 ng of the already existing PCR fragments Hag-KpnI and StrepDARPidin were digested with the same enzymes.
Concentrations:
Hag-KpnI 130 ng/µl
Strep-DARPidin 55 ng/µl after purification
psB1C3 130 ng/µl after plasmid isolation
Digest:(20µl reaction mix)
Digestion Mix
StrepDARPidin (600 ng) (µl)
Hag-KpnI (600 ng) (µl)
pSB1C3 (2 µg) (µl)
Water
10
5
13
DNA
7
12
4
CutSmart  Buffer
2.0
2.0
2.0
EcoRI
0.5
0.5
0.5
PstI
0.5
0.5
0.5
Sequencing Results of constructs from 01.09.2014
Premix
Label-Nr.
Construct
Result
1
AGB0023518
piGEM-027 I1 (pMAD-D2-Strep)
fine
2
AGB0023519
piGEM-027 I3 (pMAD-D2-Strep)
fine
3
AGB0023520
piGEM-027 I6 (pMAD-D2-Strep)
fine
4
AGB0023521
piGEM-027 II2 (pMAD-D2-Strep)
fine
5
AGB0023551
piGEM-028 clone 7
fine
6
AGB0023552
piGEM-028 clone 7
fine
22.6 Repeated fusion-PCR of the Killswitch module I with the flanks amyE I/II
Aim: Fuse the amyE flanks to killswitch module I.
Content
KSI (µl)
Template: KSI (approx.5 ng/µl)
1
amyE flank I (5 ng/µl)
1
amyE flank II (approx. 5 ng/µl)
1
Primer iGEM034 (1:40)
1
Primer piGEM037 (1:40)
1
HF Buffer 5x
10
Phusion DNA-polymerase
1
dNTPs
1
Water
33
Total
50
Fusion PCR Step
Temperature °C
Time
1
98
5 min
2
98
20 sec
3
60
20 sec
4
72
2 min
5
Go To 2
35x
6
72
5 min
7
4
infinite
The bands were at a height of approx. 1500 bp which is too small for the expected fragment of 1847 bp.
22.7 Gibson Assembly of KS module I with the flanks amyE I/II
Aim: Fuse the amyE flanks to killswitch module I.
Component
Gibson with vector (µL)
Gibson without vector (µl)
Control (µl)
pMAD
2
-
2
KS module I (approx. 5 ng/µl)
1
1
-
Flank amyE I (approx. 5 ng/µl)
1
1
-
Flank amyE II (approx. 5 ng/µl)
1
2
-
H2O
-
1
3
Gibson mix
15
15
15
Total
20
20
20
The reactions were incubated at 50°C for an hour and used to transform E. Coli XL1-Blue, plated out on LB-Amp. An additional reaction was started without pMAD in a lower (1 µl) and a higher (2 µl) amount of amyE flank I. These reactions were used to do a PCR reaction with the primers piGEM034 and piGEM037 to test if the assembled fragment is present in the reaction. The expected bands should have a size of 1847 bp but the fragment visible is only approx.1300 bp big. It seems like the flank I is missing but then the fragment should not have been amplified, since the primers were chosen to include both flanks.
03.09.2014
23.5 Gelextraction of the digested plasmid and fragments in a 1% agarose gel
Aim: The purified plasmids/inserts will afterwards be used for ligations.
Information: iBB8 has a size of 1461 bp, vector 2000bp, Inserts Hag-KpnI and StrepDARPidin 1000 bp
Image of the gelextraction, as a marker the 1kb gene ruler was used, 1 = StrepDARPidin, 2= Hag-KpnI, 3 and 4 = pSB1C3
23.6 Ligation of digested inserts StrepDARPidin and Hag-KpnI into pSB1C3and transformation into chemically competent E. coli XL-1-blue
Aim: Ligate the two inserts into pSB1C3 backbone in order to create the first two biobricks
Concentrations:
pSB1C3= 18 ng/µl
StrepDARPidin= 5 ng/µl
Hag-KpnI= 13 ng/µl
Ligation Mix
StrepDARPidin
Hag-KpnI
Control
T-4 Ligase
1
1
1
T-4 ligase Buffer
2
2
2
Water
-
-
14
Insert
14.9
12.5
0
Vector
2.1
4.5
3
Ligation at RT for 90 minutes
After the ligation the whole reaction mix was used to transform chemically competent E. coli XL-1-blue cells (standard protocol). Cells were plated on LB-Cm plates and incubated at 37°C over night.
22.8 Repeated amplification of KSI and flank I
Aim: amplification KSI and flank I to have correct fragments for Gibson / Fusion PCR
Since the Gibson assembly of the single parts without the vector (22.7)   and the Fusion PCR yielded no positive results, the KSI should be amplified   once again to assure usage of the correct and pure fragments for the   FusionPCR or Gibson assembly.
Component
KS module I
Flank I
Template (Δ51 fac gDNA 1:10)
-
2
Template (gBlock DNA 1:10)
1
-
Primer iGEM-38 (1:50)
2.5
-
Primer iGEM-39 (1:50)
2.5
-
Primer iGEM-34 (1:50)
-
2.5
Primer iGEM-35 (1:50)
-
2.5
5x HF-Buffer
10 
10
dNTPs (40 mM)
1
1
Phusion DNA-polymerase
1
1
Water
32
31
Total
50
50
PCR Step
Temperature °C
Time
1
98
5 min
2
98
20 sec
3
58.2, 61.3, 63.5
20 sec
4
72
50 sec
5
Go To 2
34x
6
72
5 min
7
4
infinite
The gel showed the expected band for the killswitch module 1 but only one band for flank 1 in the reaction with the annealing temperature of 58°C. The remaining reactions for KSI were pooled and purified with a Gel Ex kit. Since the only band for the flank 1 is visible at 58°C and not above another PCR was prepared with lower annealing temperatures but the same reaction mix as above.
PCR Step
Temperature °C
Time
1
98
5 min
2
98
20 sec
3
58.2, 61.3, 63.5
20 sec
4
72
50 sec
5
Go To 2
34x
6
72
5 min
7
4
infinite
13.80 repeated transformation of B. subtilis PY79 and E. coli DHα with the Nose-plasmids
Aim: insertion of the Nose-plasmids into B. subtilis in order to test the promoters and into E. coli for multiplication of the plasmids
All plasmids (from positive clones 13.78) were again used to transform B. subtilis PY79, since the former plates seemed to be contaminated by other organisms. E. coli DH5µ was transformed with the rest of the plasmids to multiply it for later purposes.
22.9 colony PCR of clones of the Gibson assembly with KSI and flanksI and II
Aim: check for positive clones
Three colonies were grown on the plate with the Gibson assembly transformed XLI-Blue cells. The control also contained three colonies. However, the colonies of the Gibson plate were tested by colony PCR with the primer surrounding the flanks of the connected KSI module, since no primer were available which target regions in the vector pMAD.
Mix colony PCR
Master mix for 4 reactions (µl)
Single reaction
Template
-
small part of colony
Primer iGEM034 (1:50)
4
1
Primer iGEM037 (1:50)
4
1
5x HF-Buffer
16
16
dNTPs (40 mM)
2
0.5
Phusion DNA polymerase 1:10
2
0.5
Water
52
13
Total
80
20
colony PCR Step
Temperature °C
Time (min/sec) KSI
1
98
5 min
2
98
20 sec
3
55
20 sec
4
72
1:20 min
5
Go To 2
34x
6
72
5 min
7
4
infinite
The Gibson assembly of KSI with flanks I and II did not work, but the flank I was amplified well.
04.09.2014
23.7 Colony PCR on presumably positive transformants + inoculation for plasmid preparation
Aim: Analyze the transformants for the insertion of the fragments into the plasmid
Colonies could be detected on both transformation plates, whereas no colonies grew in the ligation control plate. Therefore a colony PCR will be performed in order to analyze the transformants for the insertion of the fragments into the plasmid. 5 clones from both plates were chosen for the colony PCR and inoculated in liquid LB-Cm for plasmid isolation in the evening (in case positive clones can be detected in the colony PCR).
Colony PCR
StrepDARPidin MM 6x (µl)
Hag-KpnI MM 6x (µl)
Water
84
84
PhusionBuffer (5x)
24
24
Phusion
3
3
Primer 1
3(51)
3(53)
Primer 2
3(53)
3(54)
dNTPs
3
3
Colony
1
1
Saved colony PCR program in the PCR cycler was used with 1 minute elongation time.
Colony PCR negative, therefore plasmids were isolated and digested with the EcoRI and PstI.
23.8 Test restriction of presumably positive plasmids after plasmid isolation
Aim: Analyze the plasmids for the insertion of the fragments into the plasmid
Restriction: 4 µl plasmid in a total 10 µl restriction mix
Amount (µL)
Plasmid
4
Water
4
CutSmart  Buffer
1
EcoRI
0.5
PstI
0.5
Restriction positive (2kb plasmid and 1kb insert), one clone each will be send for sequencing.
18.67 StrepDARPidin expression in E. coli BL21 (DE3)
Aim: Preculture for upcoming purification
3 L LB medium were prepared with 1 mL ampicillin each and inoculated with some clones from the B21 transformation plate. The cultures were induced with 50 ml lactose solution (12,5g/ 50 mL) each and incubated at 30°C over night.
In order to check the expression of the StrepDARPidin a colony containing piGEM-028 (pET16b-StrepDARPidin) was used to inoculate 20 mL of LB-Amp.
22.10 Gradient fusion PCR to amplify KSI with flanks
Aim: fuse killswitch module I and flanks I amyE fw and II amyE rev together
The fragments were all amplified and purified. This time a gradient was performed with an initial step of 50°C for 0.5 hours (similar to a Gibson assembly)
Mix fusion/gibson PCR
Master mix for 10 reactions (µl)
Single reaction (µl)
Killswitch module I (approx. 10 ng/µl)
10
1
Flank I amyE for
10
1
Flank II amyE rev
10
1
Primer iGEM034 (1:50)
62.5
6.25
Primer iGEM037 (1:50)
62.5
6.25
5x HF-Buffer
100
10
dNTPs (40 mM)
10
1
Phusion DNA polymerase 1:10
10
1
Water
225
22.5
Total
500
50
colony PCR Step
Temperature °C
Time (min/sec) KSI
1
98
5 min
2
98
20 sec
3
52-62
20 sec
4
72
2:10 min
5
Go To 2
34x
6
72
5 min
7
4
infinite
The fusion-PCR worked and the bands had the expected size of approx.1800 bp.
05.09.2014
23.9Sequencing of pSB1C3 StrepDARPidin and pSB1C3 Hag-KpnI
iGEM Primer in the list of last year's team number 37 and 38 (Sample premix)
AGB0023-642= StrepDARPidinFw (Clone 1)
AGB0023-643=StrepDARPidinRv
AGB0023-644= Hag-KpnI Fw (Clone 4)
AGB0023-645= Hag-KpnI Rv
18.68 StrepDARPidin expression in E. coli BL21(DE3) and purification
The 3 x 1 liter LB culture which were induced overnight with lactose were harvested and the pellets resuspended in 15 mL Buffer A. The cell suspension was cracked with the micro fluidizer. The lysate was centrifuged at 20000 rpm for 20 min and purified with a Ni-NTA Column.
The following steps were performed for the Ni-NTA Column:
equilibration with Buffer A 10 min
taking 40µL supernatant  (load)+ 10 µL SDS-Buffer - L-Probe
50 mL load on column
taking 40 µL of flow through + 10 µL SDS-buffer - FT-Probe
first washing with 25 ml Buffer A (half the load)
taking 40 µL of washing flow through + 10 µL SDS-Buffer - W-Probe
equilibrate with Buffer B (output pipe off the column into glas, input into B)
hanging pipe on column, 20 ml Evolution- E
taking 40 µL of Elution 1-6 + 10 µL SDS-Buffer E-probe
SDS-PAGE analysis
regeneration of column:
10min water
10min EDTA
10min water
10min NiSO4
10min water
Meanwhile the elution was transferred in a concentrator column which was centrifuged at 4000 rpm until the volume in the filter was 1.5 mL for injection into the gel filtration station.
The gel showed that there was not the desired StrepDARPidin loaded on the column. A SDS-PAGE probe from the pellet showed us that the pellet contained the protein.
18.69 New StrepDARPidinexpression in E. coli BL21 (DE3)
We considered that the StrepDARPidin might be in inclusion bodies because of the Streptavidin part and the codon optimization. So we tried expression overnight with 0.1 mM IPTG of an 100 mL culture at OD 0,7. We induced as well with lactose overnight as a control.
The 4 cultures were incubated at 20°C and 30°C in the shaker overnight.
19.12 Amplification of Hag-D2-Cup/-Strep
Aim: amplification of the Hag-D2-Cup/ -Strep for cloning into pET24d in order to prepair crystallization.
piGEM-026 and -027 were used as a PCR template to amplify Hag-D2 constructs for cloning into pET24d in order to express the flagellin modification for crystallization. Fragments the size of approx. 1450 for Hag-D2-Cup and 1350 for Hag-D2-Strep were expected.
Mix
PCR D2-Cup I (µL)
PCR D2-Cup II (µL)
PCR D2-Strep  (µL)
piGEM-026 (1:10 - 10 ng/µL)
1
1
-
piGEM-027 (1:10- 10 ng/µL)
-
-
1
Flo96 D2_3 fw
1
1
1
Flo97 D2_3 rv)
1
1
1
Phusion DNA-polymerase
1
1
1
HF-Buffer 5x
10
10
10
dNTPs
1
1
1
Water
35
35
35
Total
50
50
50
PCR Step
Temperature °C
Time
1
98
3 min
2
98
30 sec
3
55
30 sec
4
72
90 sec
5
Go To 2
31x
6
72
4 min
7
4
infinite
The PCR Probes were purified and pooled with the Omega Gel Extraction Kit due to the enzymatic reaction purification protocol. Final concentrations:
Hag-D2-Cup: 41 ng/ µL
Hag-D2-Strep: 25 ng/ µL
19.13 Restriction digest of PCR Fragments Hag-D2-Constructs
Aim: digest with NcoI and Bam of the Hag-D2 PCR product for cloning into the expression vector pET24d NcoI/BamHI cut
The PCR products from 19.12 were digested with NcoI and BamHIHI to get the restriction sites for cloning into pET24d NcoI/BamHI cut.
Component
PCR PCR D2-Cup (µL)
PCR D2-Strep (µL)
Hag-D2-Cup (41 ng/µL)
30
-
Hag-D2-Strep(25 ng/µL)
-
30
CutSmart   10x
3.4
3.4
NcoI
0.25
0.25
XhoI
0.25
0.25
Water
1
1
Total
34
34
Incubation overnight at room temperature.
22.11 Gibson assembly of pMAD NcoI/BamHIHI with the fused KSI + flanks amyE fw and rev
Aim: clone the fragment into the vector pMAD for integration in Bacillus chromosome
The fragment of the fusion PCR (22.10) was separated on a 1% agarose gel, digested out and purified with a Gel Ex kit. The already digested pMAD plasmid was used to perform a Gibson assembly.
Component
Gibson with vector (µL)
Control (µL) (only digested pMAD)
pMAD NcoI/BamHIHI (47.6 ng/µL)
2.1
2.1
KS part I (88.1 ng/µL)
0.9
-
H2O
2
2.9
Gibson mix
15
15
Total
20
20
13.81 Preparation of media and strains for another B. subtilis transformation
Aim: Since the transformation of B. subtilis with the SPC/SPII method and the Nose-plasmids did not work the Low Salt/High Salt method should be tested.
The protocol and the strains were provided by AG Bremer of the microbiology department of the Philipps Universität Marburg. All media were prepared as possible (methods) and the strains B. subtilis 168 and B. subtilis JH642 were streaked on LB-Agar and incubated at 37°C.
06.09.2014
18.70 New StrepDARPidin expression in E. coli BL21 (DE3)
The cultures from the day before were spinned down at 4000 rpm for 20 min. The pellets were resuspended in 10 ml Buffer A and cracked with the micro fluidizer. The lysate was centrifuged at 20000 rpm for 20 min and purified with a Ni-NTA Beads from Qiagen according to the protocol. Probes from the pellet, supernatant and Elution wereanalysed on an SDS-PAGE gel:
The gels showed that the protein remains in the pellet no matter which attempt.
For further experiments for the isolation of the protein in inclusion bodies new expression cultures from a freshly transformed BL21(DE3) strain with piGEM-028 were used to inoculate 3 L expression culture with lactose overnight.3 L LB were prepared with 1 mL ampicillin each. The cultures were induced with 50 ml lactose solution (12,5g/ 50 mL) each and incubated at 30°C overnight.
19.14 Ligation of digested Hag-D2-Cup/ -Strep constructs with pET24d NcoI BamHIcut
Aim: cloning of expression vector with Hag-modifications
The NcoI/BamHI digested PCR fragments from 19.13 were ligated into predigested pET24d. The 20 µL attempts were transformed into E. coli XL1-Blue after inactivation (10min at 65°C) and selected on LB- Amp plates incubated overnight at 37°C.
Component
Ligation I: pet-Hag-D2-Cup(µL)
Ligation II: pet-Hag-D2-Strep (µL)
Control (µL)
NcoIBamHI digested Hag-D2-Cup
1
-
-
NcoIBamHI digested Hag-D2-Strep
-
1.5
-
pET24d NcoI/BamHI(6,5 ng/µL)
5
5
5
Ligation Buffer 10x
2
2
2
Ligase
1
1
1
H2O
11
10.5
12
Total
20
20
20
07.09.2014
19.14 Ligation of digested Hag-D2-Cup/ -Strep constructs with pET24d NcoI/ BamHI cut
Aim: cloning of expression vector with Hag-modifications.
The control plate showed no colonies in comparison the ligation plates I and II over 15 clones. 6 Clones per plate were picked for a inoculation of minipreps for a test digest next day.
18.70 3rd StrepDARPidin expression in E. coli BL21 (DE3)
Aim: Preculture for upcoming purification tests
The cultures were centrifuged at 4000 rpm. The pellets were resuspended in 10 ml Buffer A and centrifuged down at 4000 rpm again. The supernatant was discarded and the pellets were frozen at -80°C.
08.09.2014
23.10 Results of thesequencing of pSB1C3 StrepDARPidin and pSB1C3 Hag-KpnI
Sequencing results -positive for both sent plasmids.
18.71 Purification of StrepDARPidin from inclusion bodies
Aim: Purification of StrepDARPidin from inclusion bodies.
In order to rescue the StrepDARPidin from the inclusion bodies we tried different protocols. The pellet from 07.09.2014 was warmed up and the cells were cracked with the microfluidizer after resuspension in 15 mL Buffer A (lysate)).
The suspension was centrifuged at 20.000 x g for 15 min. The supernatant (S1) was discarded  and the pellet (P1)resuspended in 5 mL 6 M guanidinium-HCL solution for solubilisation and unfolding of the StrepDARPidin.
After centrifugation at 4000 rpm for 10 min the supernatant (S2) was used for refolding. The membranous pellet (P2) was thrown away.
For refolding 100 mL Buffer A were filled into a Erlenmeyer-Flask on ice with a magnet stirrer inside on 600 rpm.
For refolding it was necessary to dilute the Guanidinium-HCL dilution very fast and softly. For that purpose rapid dilution was done by pipetting the supernatant drop by drop into the vortex of the Buffer A. After adding the whole Guanidinum-HCL/ Protein solution the suspension was observed and precipitation noticed. The suspension was centrifuged at 20.000 rpm (Pellet P3). The supernatant was loaded on a Ni-NTA Column from GE-Healthcare (L). Flow through (FT), Wash (W) in Buffer A and Elution (E) in Buffer B were kept for gel analysis. The probes lysat to Load were cooked for 5 min at 95 degrees.
From every fraction 40 µL were taken for a commassie SDS-PAGE and 10 µL 10x SDS Buffer added. The pellet probes were resuspended in 60 µL water and 40 µL SDS-Buffer.
The gel showed us that the StrepDARPidin is still in the pellet fraction but we were able to make some of it soluble as it can be seen in  S2 and the load. The StrepDARPidin was thought to build tetramers with approx. 120 kDa size. The complex seems to be stabil enough to be seen on the gel above the marker which fits to the size of approx. 120 kDa. Boiling up would break the tetramer.
In order to get enough protein for a gel filtration run 4 L of BL21 with piGEM-028 were induced with lactose overnight.
19.15 Test digest NcoI/ BamHI with pET24d-Hag-D2-Cup/ -Strep clones
Aim: checking insertion of Hag-D2-Cup/ -Strep
The plasmids from 6 clones per Ligation plate were isolated and approx. 350 ng DNA were digested with NcoI/BamHI in order to check the right insertion of the 1350-1450 bp sized inserts.
Component
PCR D2-Cup (µL)
NcoI/ BamHI Mastermix (14x)(µL)
Plasmid (60 ng/ µL)
5
-
CutSmart   10x
-
14
NcoI
-
0.5
BamHI
-
0.5
Water
-
55
Master mix
5
-
Total
10
70
Digest at 37°C for half an hour.
Upper half of the gel shows hag-D2 cup, lower half shows hag-D2-Strepdarp; 1kb gene ruler ladder was used. 3 clones seem to be positive for cup, but as an mixture occurred, the cloning procedure will be repeated from the PCR step on.
19.16 PCR amplification of Hag-D2-Cup/ -Strep
Aim: clone the expression vectors pET24d
Template was diluted plasmid piGEM026/027
Component
1x Mix (µL)
5x Mastermix (µL)
Buffer
10
50
Water
26.5
132.5
dNTPs
1
5
Phusion DNA-polymerase
1
5
DNA (10-13 ng/µl)
1.5
1.5
Primer 54 Flo (1:50)
6.5
32.5 
Primer 56 Flo (1:50)
6.5 
32.5
After the PCR the fragments were purified via gel extraction for further cloning steps. In the gel the digested vector pET24d was also purified.
Concentrations after clean up:
pET24d = 20 ng/µl, D2-Cup = 208 ng/µl, D2-Strep = 180 ng/µl
19.17 Restriction of -Hag-D2-Cup/ -Strep
After the clean up the two fragments they had to be digested with NcoI and BamHI in order to clone them into the digested pET24d vecor.
Component
D2-Cup (µL)
D2-Strep (µL)
Water
12 
12
Buffer
2
2
NcoI
0.5
0.5
BamHI
0.5
0.5
Insert
5
5
The restriction was performed for 1h at 37°C (heating block).
19.18 Ligation of Hag-D2-Cup/ -Strep into digested pET24d
After the plasmid and the two inserts have been digested, two ligations and a ligation control were performed in order to gain the final expression vectors.
Component
pET24d + D2-cup (µL)
pET24d + D2- Strep (µL)
Water
2 
2
Buffer (10x)
2
2
Ligase
1
1
Vector
10
10
Insert
5
5
For the optimal ligation results the ligation calculator of the  iGEM team Dallas was used. Ligation was performed for 2h. Afterwards the ligase was inactivated at 65°C for 15 min in order to improve the ligation/transformation result.
19.19 Transformation of pET24d with Hag-D2-Cup/ -Strep
After the ligase inactivation the whole reaction mix was used to transform E. coli XL-blue which seem to be the best cells for important transformations. Standard transformation protocol was used.
22.12 colony PCR and restriction of isolated plasmids from Gibson assembly plate
Aim: test the clones for the correct pMAD-KSI construct
The Gibson assembly plate showed sixteen colonies. All colonies were picked, transferred onto a master plate and into a premixed colony PCR. The same colonies were used to inoculate a miniprep.
Mixcolony PCR
Master mix for 18 reactions (µL)
Single reaction
Template
-
Small part of colony
Primer iGEM034 (1:50)
54 
3
Primer iGEM037 (1:50)
54
3
5x HF-Buffer
72
4
dNTPs (40 mM)
18
1
Phusion DNA polymerase 1:10
9
0.5
Water
153
8.5
Total
360 
20 
colony PCR Step
Temperature °C
Time
1
98
10 min
2
98
20 sec
3
55
20 sec
4
72
2:10 min
5
Go To 2
33x
6
72
5 min
7
4
infinite
The colony PCR was negative, even the positive control with the amplified KS part I with the flanks.
The plasmids were isolated and digested with KpnI and SacI to test the insertion of the KS part I into pMAD. The expected sizes were 5054 bp and 6402 bp with the insert and pMAD only would yield in 4440 bp, 170 bp and 5054 bp fragments.
The clones 1, 2, 5, 7, 8, 9, 10, 11, 12 and 14 showed the expected results. The rest of the isolated plasmids were discarded.
09.09.2014
18.71a Gelfiltration purification of StrepDARPidin tetramers
Aim: Purification of StrepDARPidin from inclusion bodies
The overnight cultures from 18.70 were harvested for 10 min at 4000 x g and the pellets were stored at -80°C until use.
22.14 Start of the pMAD-transformation of B. subtilis WT3610 with pMAD-KSI
Aim: integrate KS part I into the amyE locus of the B. subtilis genome.
A preculture of 10 ml LB was inoculated with B. subtilis WT3610 cells and incubated at 37°C over night.
13.81 Midi prep of the nose-plasmids and the piGEM002-plasmids with the killswitch promoters
Aim: Since no transformation of B. subtilis has worked yet, two new protocols should be tested, which require a huge amount of plasmid
The midi prep of the plasmids piGEM002 with the silver and copper promoters and the killswitch promoters was carried out according to the Qiagenmidiprep kit instructions. The isolated plasmids were tested via restriction with BamHI and SpeI.
Precultures of all four strains (JH642, PY79, 168 and WT3610) were inoculated in 3 ml HS-medium and 10 ml LB and were incubated over night at 37°C.
19.20 Test digest NcoI/BamHI with pET24d-Hag-D2-Cup/ -Strep clones
Aim: checking insertion of Hag-D2-Cup/ -Strep
After the successful ligation (colonies in both plates), 5 clones per ligation have been inoculated in LB-Kan Medium for a later Mini-prep and test digest of the expression plasmids. Colonies with the numbers 6-10 were chosen as 1-5 did not grow over the day. 6-10 were incubated over night.
10.09.2014
13.82 Transformation of B. subtilis with the Nose-plasmids (Ag-/Cu-promoters, killswitch promoters) according to the HS/LS-protocol / Spizizen-medium protocol
Aim: transform Bacillus with the plasmids
The only preculture, which was grown in HS was the JH642. Every other strain did not grow in High Salt medium. 20 ml of prewarmed (30°C) Low Salt medium were inoculated with 1 ml of the HS culture and incubated in a shaking waterbath at 30°C with 120 rpm for 3 hours. 5 µg of the plasmids were preset in Eppendorf cups and 1 ml of the LS culture was transferred into these cups. piGEM002 was transformed as a control. These culture/plasmid mixes were incubated at 37°C shaking at 200 rpm. After 2 hours of incubation the cells were centrifuged at 8000 rpm for 1.5 minutes, most of the medium was withdrawn,the cells were resuspended in the rest of the medium and plated out on LB-Cm plates that were incubated at 37°C over night. Precultures of all strains were again inoculated, since many were not grown.
The precultures in LB medium were all grown. 10 ml Spizizens minimal medium were inoculated with an OD600 of 0.1. The cultures were incubated at 37°C until they reached an OD600 of 1.5 µg of DNA were mixed with 1 ml of the culture and incubated for further 2 hours. 200 and 500 µl of each transformation sample were plated out on separate LB-Cm plates, which were incubated at 37°C over night.
22.14 pMAD-transformation of B. subtilis WT3610 with pMAD-KSI
Aim: transformation of B. subtilis with the vector pMAD-KSI
10 ml of MNGE-medium were inoculated 1/100 with the preculture. This culture was grown at 37°C with an agitation of 200 rpm until it reached OD 1.4. 5 µg of he isolated plasmids pMAD-KSI from clone 1, 2, 10 and 11 (22.12) were used to perform four transformations. 400 µl of the cells  were added to the plasmids in a test tube and were incubated at 37°C 200 rpm for 1 hour. 100 µl expression mix were added and the cells were incubated for another hour. Dilutions up to 10-6 were carried out and the 10-5 and 10-6 dilutions were plated out on LB-MLS-X-Gal plates that were incubated at 30°C for several days.
19.21 test digest NcoI/BamHI with pET24d-Hag-D2-Cup/ -Strep clones
Aim: checking insertion of Hag-D2-Cup/ -Strep
Plasmids were extracted from the cells and a test restriction was performed with 300 ng plasmid. Gelfoto was unfortunately not saved but the bands were at the expected size (appr. 1500 bp). Therefore one plasmid each was sent for sequencing. Cup Number 6!
Component
1x Mix (µL)
11x Mastermix (µL)
Plasmid DNA
3 
- 
Water
14
154
CutSmart   Buffer
2 
22 
NcoI
0.5 
5.5
BamHI
0.5 
5.5 
11.09.2014
19.22 Sequencing results and further cloning plans
Due to the sequencing results pET24d with Hag-D2-Cup is perfect (the plasmid map was compared to the sequence delivered by eurofins).
pET24d with Hag-D2-Strep cannot be used, since the strep tag seems to be lost. Therefore this plasmid will be cloned again from the beginning in order to exclude any mistakes at any step.
13.83 Repeated Transformation of B. subtilis with the Nose-plasmids (Ag-/Cu-promoters, killswitch promoters) according to the HS/LS-protocol
Aim: transform Bacillus with the plasmids
The plates of the former transformations did not show any colonies, so the trafo was repeated with the precultures (13.82). Again the only strain which grew in HS-medium was JH642. The procedure was carried out like before with a higher agitation (200 rpm) in the waterbath.
12.09.2014
19.23 PCR amplification of Hag-D2-Strep
Materials
Amount (µL)
Plasmid piGEM027
1.0
Water
24.0 
Buffer (5x)
10.0
Phusion
1.0
dNTPs
1.0
Primer Flo 54 (1:50)
6.5 
Primer Flo 56 (1:50)
6.5 
The fragment with the size of approx. 1300 bp was successfully amplified.
19.24 Digestion of Hag-D2-Strep
After the purification via gel extraction the fragments had the following concentrations:
Attempt I = 27 ng/µl Attempt II = 35 ng/µl
20 µl reactions were prepared:
Materials
Attempt I (µL)
Attempt II (µL)
Buffer (10x)
2.0
2.0
Water
8.0
6.0
Fragment
9.0
11.0
NcoI
0.5
0.5
BamHI
0.5
0.5
19.24 Ligation of Hag-D2-Strep into digested pET24d
In order to gain the complete expression vector the digested insert had to be ligated into digested pET24d. 20 µl reaction mix was prepared, correct ratio of insert to vector was calculated with the iGEM ligation calculator.
Materials
Amount (µL)
Buffer
2.0
Water
4.0
Ligase
1.0
Vector
5.0
Insert
8.0
19.25 Transformation of pET24d with Hag-D2-Strep
Whole ligation mix was transformed into E. coli XL-1-Blue with the standard transformation protocol.
22.14 pMAD-transformation of B. subtilis WT3610 with pMAD-KSI
Aim: transformation of B. subtilis with the vector pMAD-KSI
5 ml LB-MLS were inoculated with a blue colony of the trafo plates and incubated over night at 37°C. Colonies of clone 1, 2, 10 and 11 were picked.
19.26 colony PCR of transformed B. subtilis WT3610 with pMAD-D2-Strep/Cup
Aim: check for positive clones
The transformation of B. subtilis WT3610 with the plasmids   piGEM026 and piGEM027was carried out. In order to check for positive clones a colony   PCR was performed using the Hag primers.
Mixcolony PCR
Single reaction
Colony in 50 µl PBS
2
Primer Flo 56 (1:50)
6.25
Primer Flo 54 (1:50)
6.25
5x HF-Buffer
4
dNTPs (40 mM)
0.5
DMSO
0.3
Phusion DNA polymerase 1:10
0.5
Water
0.2
Total
20
Fusion PCR Step
Temperature °C
Time
1
98
5 min
2
98
20 sec
3
55
20 sec
4
72
1:35 min
5
Go To 2
34x
6
72
5 min
7
4
infinite
No positive clone could be detected.
13.09.2014
23.11 Transformation of of pSB1C3 StrepDARPidin and pSB1C3 Hag-KpnI in E. coli DH5α
Aim: Gain more plasmid for further cloning procedures and sending the bricks to the registry.
Standard E. coli transformation protocol was used. Cells were plated on LB-Cm, incubation over night at 37°C.
19.26 Test restriction of pET24d with Hag-D2-Strep
8 Clones from the ligation plates were inoculated in 5ml LB-Kan and incubated over night. Plasmids were extracted with the quiagen kit and digested with NcoI and BamHI in order to check if Hag-D2-strep is in pET24d.
Components
1x Mix
Plasmid
3.0
Buffer
2.0
Water
14.0
NcoI
0.5
BamHI
0.5
Every picked  clone was positive so that clone 1 was sent for sequencing. The gel photo got lost unfortunately. pET24d-Hag-D2-Strep was named piGEM-030.
22.14 First heatshock of pMAD-Trafo KSI
Aim: integration of pMAD-KSI into the Bacillus genome
4 ml of LB-MLS were inoculated with 100 µl of the precultures (in a test tube) and incubated at 30°C and 210 rpm for 2 hours before switching the temperature of the incubator to 42°C for 6 h. Dilutions up to 10-6 were made and 10-5 and 10-6 were plated out on LB-MLS-XGal plates at 42°C.
19.27 repeated colony PCR of transformed B. subtilis WT3610 with pMAD-D2-Strep/Cup
Aim: check for positive clones
The picked clones were boiled at 100°C in 1x PBS for 10 minutes and the the cell rests were centrifuged. The colony PCR was repeated as follows:
Mixcolony PCR
Single reaction (µL)
Colony in 100 µl PBS
5
Primer Flo 56 (1:50)
6.25
Primer Flo 54 (1:50)
6.25
5x HF-Buffer
10.0
dNTPs (40 mM)
1.0
Phusion DNA polymerase 1:10
1.0
Water
20.5
Total
50
colony PCR Step
Temperature °C
Time (min/sec) KSI
1
98
5 min
2
98
30 sec
3
55
30 sec
4
72
1:40 min
5
Go To 2
34x
6
72
5 min
7
4
infinite
The size of the expected bands was approx.1400 bp. The negative clones with the wildtypeflagellin (approx.1 kb) were not further used. Clones D2-Strep 1, 10 and D2-Cup 1 were picked and streaked out separately on LB plates.
14.09.2014
23.12 Inoculation of colonies from the transformation plate (2x)
Aim: Isolate more plasmid for the shipment to the registry
Incubation over night shaking at 37°C
22.14 Second heatshock of pMAD-Trafo KSI
Aim: flip the pMAD-backbone out of the Bacillus genome
4 ml of LB were inoculated (in a test tube) with a blue colony of the plates from the first heat shock. These cultures were incubated at 30°C at 210 rpm for 6 hours and afterwards the temperature was shifted to 42°C for 3 hours. The plating was performed as usual.
19.27 colony PCR on positive picked and separated Bacillus subtilis clones
Aim: check for more pure clones with less wt flagellin and GelEx of positive fragment for sequencing
The colony PCR was repeated as listed above (19.26) and the fragments of the right size were purified by Gel Ex.
Additionally the positive clones were used to inoculate 100 mL LB with WT3610 as a control. The cultures were raised to an OD600 of 0,7 and 2 mL of cultures were centrifuged before dissolving them in 60 µL water and 40 µL SDS-PAGE buffer. The samples were cooked for 10 min at 95°C and analyzed via SDS-PAGE with coomassie stain.
It is possible to see that the D2-Strep clone expresses the modified Hag-D2-Strep. The size between the WT flagellin the D2-3-Hag fits perfectly. Hag-D2-Cup seems to be not expressed by the clone. The sequencing results had to be expected.
13.84 colony PCR of Bacillus subtilis 168 piGEM037
Aim: check for positive clones with piGEM037
The plates of the transformation with the Spizizens-medium were still stored at 37°C and none of the plates contained colonies except the plate of Bacillus subtilis 168. These clones were picked, transferred onto a new plate and into 1x PBS medium boiled at 100°C or 10 minutes.
Mixcolony PCR
Single reaction
Colony in 100 µl PBS
5.0
Primer iGEM006 (1:50)
6.25
Primer iGEM063 (1:50)
6.25
5x HF-Buffer
10.0
dNTPs (40 mM)
1.0
Phusion DNA polymerase 1:10
1.0
Water
20.5
Total
50.0
colony PCR Step
Temperature °C
Time
1
98
5 min
2
98
30 sec
3
55
30 sec
4
72
1:40 min
5
Go To 2
34x
6
72
5 min
7
4
infinite
The expected size of the bands was approx.750 bp, the negative control was genomic DNA of the wt3610, the positive control was the isolated plasmid piGEM037.
15.09.2014
23.13 Plasmid isolation
Plasmid isolation of the pSB1C3-Hag-KpnI plasmid from E. coli was performed with the Quiagen Mini-prep kit. Plasmid concentration was determined with Nanodrop (96 ng/µl).
23.14 Restriction of pSB1C3 Hag-KpnI with KpnI
Aim: Linearize the plasmid for Gibson assembly
Restriction of the plasmid with KpnI was performed in a duplicate for the creation of further biobricks.
Digestion Mix
Amount (µL)
Water
6.0
CutSmart  Buffer (10x)
2.0
Plasmid (3µg)
11.0
KpnI
1.0
The digest was incubated at 37°C for 210 minutes and subsequently analyzed on a 1% agarose gel.
The band of the digested plasmid pSB1C3-Hag-KpnI was digested out and purified with the Qiuagen gel extraction kit. Concentration after purification was 100 ng/µl.
23.14 Gibson assembly of linearized pSB1C3 with three different PCR fragments
Aim:
For the creation of further biobricks three PCR fragments that were already present from previous PCR reactions have been inserted into flagellin:
Cup 1-1 (amplified with primer iGEM24 and iGEM25) 16 ng/µl
D2-Cup (amplified  with primers  flo96 and flo97) 189 ng/µl
D2-Strep (mplified  with primers  flo96 and flo97) 233 ng/µl
For the Gibson assembly 4 Gibson assembly mixes (3 reactions and one control) have been thawed on ice. 2.5 µl of digested plasmid and the PCR fragment have been added respectively. The total mix of 20 µl was incubated at 50°C for one hour (PCR cycler). Afterwards the complete mix was used to transform competent E. coli XL-1-Blue. Cells were afterwards plated on LB-Cm plates.
13.85 Cloning of the new Ag/Cu-promoters and the killswitch promoters into the nose plasmids with the ssrA-tags
Aim: create plasmids with the promoters in front of a GFP with degradation tag
Since we tried the Bacillus trafo with the ethanol free isolated nose plasmids (without degradation tags) as well and succeeded in nearly all cases except piGEM035, all strains are there for plate reader tests. Competent cells of PY79 were again transformed with piGEM035. The new promoters should also be tested with GFP containing degradation tags. In order to do that the plasmids piGEM007, piGEM008 and piGEM009 that contain the ssrA tags behind the GFP were digested with the digest enzymes NcoI and SacI.
Components
1x Mix (µL)
piGEM007/8/9 (amount: approx. 2 µg)
5.0
10x CutSmart
2.0
Water
14.0 
NcoI
0.5
SacI
0.5
Volume
20
The restriction mixes were incubated at 37°C for over 1 h, separated on an agarose gel and purified via Gel Ex.
To connect the linearized plasmids with the promoter fragments ligations and Gibson assemblies were performed similar to 13.76 and 13.77.
Ligation of piGEM007/8/9 with the killswitch promoters:
Component
Ligation I(µL)
Ligation II(µL)
Ligation III (µL)
piGEM-007/8/9 (approx.20ng/µL)
5
5
5
iGEM-59/60
0.8
-
-
iGEM-61/62
-
0.8
-
iGEM-63/64
-
-
0.8
Polynucleotide kinase
0.5
0.5
0.5
T4 Ligase
1
1
1
Ligation Buffer 10x
2
2
2
Water
9.7
9.7
9.7
Total
20
20
20
The reaction was incubated at room temperature for over 1 hour and afterwards used to transform XLIBlue cells, which were plated out on LB-Amp plates.
Gibson assembly of linearized piGEM007/8/9 with the new silver and copper promoters:
Component
Gibson Cu I (µL)
GibsonAg II (µL)
Control (µL)
piGEM-002 (approx.20 ng/µL)
2.5
2.5
5
Cu promoter (4.2 ng/µl)
2.5
-
-
Ag promoter (2 ng/µl)
-
2.5
-
Gibson mix
15
15
15
Total
20
20
20
The Gibson reaction was incubated at 50°C for 1 hour and the whole sample was used to transform XLIBlue cells that were plated out on LB-Amp.
22.15 colony PCR of pMADtrafo wt3610 with piGEM031 (pMAD-KSI)
Aim: check for positive insertion of KS part I into the Bacillus genome
A colony PCR with the primers that bind to the ends of the amyE flanks was carried out but it was negative. Since it was not sure whether the primer would be able to bind the PCR was repeated with the primers surrounding the killswitch module I iGEM038 and iGEM039:
Mixcolony PCR
Single reaction
Colony in 100 µl PBS
5.0
Primer iGEM006 (1:50)
6.25
Primer iGEM063 (1:50)
6.25
5x HF-Buffer
10.0
dNTPs (40 mM)
1.0
Phusion DNA polymerase 1:10
1.0
Water
20.5
Total
50.0
colony PCR Step
Temperature °C
Time (min/sec)KSI
1
98
5 min
2
98
30 sec
3
55
30 sec
4
72
1:40 min
5
Go To 2
34x
6
72
5 min
7
4
infinite
The expected bandsize was about 360 bp, which was visible in nearly all the samples. The positive control for the PCR reaction was isolated piGEM 031 and the negative control was chromosomal DNA of the unmodified wt3610.
18.71 Gelfiltration purification of StrepDARPidin tetramers
Aim: Purification of StrepDARPidin from inclusion bodies
In order to improve the rescue the StrepDARPidin from the inclusion bodies we tried an additional IB washing buffer. The pellet from last week at -80°C  was warmed up and the cells were cracked with the microfluidizer after resuspension in 15 mL Buffer A (lysate)).
The suspension was centrifuged at 20.000 x g for 15 min. The supernatant (S1) was discarded  and the pellet (P1) resuspended in 5 mL IB Wash Buffer. Then the suspension (SS1) was centrifuged at 27.000 x g for 10 min. The resulting pellet (P2) and the supernatant (S2) were used for taking probes for SDS-PAGE. The wash was repeated. 6M Guanidinium-HCL solution for solubilisation and unfolding of the StrepDARPidin was added.
After centrifugation at 4000 rpm for 10 min the supernatantwas used for refolding. The membranous pellet (P3) was thrown away.
For refolding 100 mL Buffer A were filled into a Erlenmeyer-Flask on ice with a magnet stirrer inside on 600 rpm.
For refolding it was necessary to dilute the Guanidinium-HCL dilution very fast and softly. For that purpose rapid dilution was done by pipetting the supernatant drop by drop into the vortex of the Buffer A. After adding the whole Guanidinum-HCL/ Protein solution the suspension was observed and precipitation noticed. The suspension was centrifuged at 20.000 rpm (Pellet P4) . The supernatant was loaded on a Ni-NTA Column from GE-Healthcare (L). Flow through (FT), Wash (20 mL W) in Buffer A and Elution (15 mL) (E) in Buffer B were kept for gel analysis. The samples lysate to Load were cooked for 5 min at 95°C.
From every fraction 40 µL were taken for a commassie SDS-PAGE and 10 µL 10x SDS Buffer added. The pellet probes were resuspended in 60 µL water and 40 µL SDS-Buffer.
The gel shows that the monomeric and tetramericStrepDARPidin can be found in Load, Flow through and elution. The forming of the tetramer tells us that the refolding had to be successful partially.  The elution was concentrated with an Amicon to an endvolume of 1,5 mL. 75 µL glycerol were added and the concentrate was frozen away in liquid nitrogen and stored at -80°C for further use.
Sequencing of constructs from 15.09.2014
Sequencing of plasmid piGEM030 and PCR products of B.subtilis colony PCR for insertion of D2-Cup and D2-Strep into Flagellin.
Label-Nr.
Construct
Primers
AGB000k-583
piGEM-030
T7
AGB000k-584
piGEM-030
T7 term
AGB000k-585
Hag-D2-Strep
Flo96 Hag-D2-fw
AGB000k-586
Hag-D2-Strep
Flo97 Hag-D2-rv
AGB000k-587
Hag-D2-Cup
Flo96 Hag-D2-fw
AGB000k-588
Hag-D2-Cup
Flo97 Hag-D2-rv
16.09.2014
23.15 Inoculation of presumably positive clones from all plates
Colonies appeared on all plates. Control plate showed appr. 50 colonies whereas the 3 reactions showed uncountable colonies on the plates. 4 clones have been picked per reaction and incubated at 37°C over night for a test restriction of the correct insertion of the 3 inserts of interest.
18.71a Gelfiltration purification of StrepDARPidin tetramers
Aim: Purification of StrepDARPidin from inclusion bodies
The stored protein concentrate was thawn on ice and injected into the injection valve of the ÄtkaPurifier which was buffered with PBS+ 2,5% glycerin. A S200 SepharoseColumn was used for the gel filtration run.
40 µL of the Fraction C6. C10, C11(Peak) and D12 were picked for SDS-PAGE analysis. 10 µL SDS-Buffer were added and 20 µL loaded on the gel. Additionally the rest was cooked up and the same amount loaded onto the gel.
The gel shows us that the whole peak contains the tetramericStrepDARPidin. After cooking it up the tetramer dissolves into the monomeric StrepDARPidin.
The fractions covering the peak were concentrated with an amicon to an endvolume of 500 µL. 25 µL glycerin were added and the protein aliquot in 60 µL aliquods.
19.28 Purification of Hag-D2-Strep
Aim: Purification of Hag-D2-Strep for analytical gel filtration was already purified StrepDARPidin
piGEM-029 and piGEM-030 were transformed together with FliS into E. Coli BL21 (DE3) and plated out on LB-Amp/Can plates for overnight incubation at 37°C.
13.86 Cloning of the Nose plasmids with the ssrA- degradation tags
Aim: create plasmids with the promoters in front of a GFP with degradation tag
The following entries of the nose plasmids will care all about the cloning, which involves creating the plasmid by Gibson/ligation checking by colony PCR, transformation of E. coli DH5α and Bacillus subtilis PY79 with the plasmids. Not every step of transformation will be mentioned, since they all were carried out according to the standard protocols. After each plasmid was checked by colony PCR and/or control restriction DH5α and Bacillus were transformed with it. DH5α were transformed with the plasmids piGEM034, 035, 036 and 037.
17.09.2014
23.16 Plasmid isolation and test digest
Plasmid isolation of the 12 cultures was performed with the Quiagen Mini Prep Kit. Concentration afterwards was about 250 ng/µl in all samples.
Test restriction of the previously isolated plasmids with EcoRI and PstI:
12 plasmids + control → 14x master mix for 20µl reaction mixes
1x Mix (µL)
14x Mix (µL]
Water
15.5
217.0
CutSmart   Buffer (10x)
2.0
28.0
EcoRI
0.5
7.0
PstI
0.5
7.0
Plasmid DNA
1.5
1.5
Restriction was incubated at 37°C for 180 minutes and subsequently analyzed on a 1% agarose gel.
Result: All tested clones were negative, i.e. bands at the same size of the control occurred.
1500bp= Hag-KpnI; 2000bp= 13C plasmid backbone
23.17 Inoculation of 5 new clones from all plates
Plan: inoculate 5 new clones each for plasmid isolation and restriction tomorrow in LB-Cm.
Aim: Gain more plasmid in case the Gibson assembly has to be repeated Inoculation in 5 ml LB-Cm
Sequencing results  of constructs from 15.09.2014
Label-Nr.
Construct
results
AGB000k-583
piGEM-030
fine
AGB000k-584
piGEM-030
fine
AGB000k-585
Hag-D2-Strep
fine
AGB000k-586
Hag-D2-Strep
fine
AGB000k-587
Hag-D2-Cup
fine
AGB000k-588
Hag-D2-Cup
fine
The sequencing told us that Both Hag-D2-Strep & -Cup are perfectly integrated into the chromosome of the mutants. Hag-D2-Cup seems not to be expressed. Anyways a swarming assay was planned.
19.28 Purification of Hag-D2-Strep
Aim: Purification of Hag-D2-Strep for analytical gel filtration was already purified StrepDARPidin
An overnight expression culture from piGEM-030/FliS was made by inoculating 2 x 1L LB-Amp/Can with clones from the transformation plates and inducing with 50 mL 25% lactose solution overnight at 30°C at 150 rpm.
21.3 Swarming AssayHag-D2-Strep & -Cup
Aim: Check motility of Bacillus mutants Hag-D2-Cup & -Strep
Overnight cultures were inoculated of 5 mL LB in test tubes from WT3610 , Hag-D2-Cup & -Strep strains. Incubation was done at 37°C overnight.
13.87 Making Bacillus competent for transformation
Aim: making competent PY 79 cells for the transformation with the new nose-plasmids
The Bacillus subtilis PY79 cells were made competent according to the protocol using SPC and SPII medium. As a test the cells were transformed with piGEM038, which was transformed before.
13.88 Screening for positive clones containing the new Nose-plasmids
Aim: colony PCR to check colonies for the correct plasmids
Mix colony PCR
Cu-promoter
Ag-promoter
lac-promoter
const. + tetO promoter
constitutive promoter
Template
Part of a colony
Part of a colony
Part of a colony
Part of a colony
Part of a colony
iGEM006 (1:50)
6.25
6.25
6.25
6.25
6.25
iGEM055 (1:50)
6.25
-
-
-
-
iGEM057 (1:50)
-
6.25
-
-
-
iGEM059 (1:50)
-
-
6.25
-
-
iGEM061 (1:50)
-
-
-
6.25
-
iGEM063 (1:50)
-
-
-
-
6.25
dNTPs (10 mM each)
0.5
0.5
0.5
0.5
0.5
5x HF-buffer
4
4
4
4
4
Phusion DNA-polymerase
0.5
0.5
0.5
0.5
0.5
Water
2.5
2.5
2.5
2.5
2.5
Total
20
20
20
20
20
colony PCR Step
Temperature °C
Time (min/sec)KSI
1
98
10 min
2
98
30 sec
3
55
30 sec
4
72
1 min
5
Go To 2
34x
6
72
5 min
7
4
infinite
The size of the expected band was depending on the promoter between 700 and 850 bp. As a positive control cells containing the piGEM002 vector with the different promoters were used. The clones that seemed positive were given in culture for a miniprep. More clones were picked from the plates to test them via colony PCR (same reaction as above).
18.09.2014
23.19 Plasmid isolation and test restriction of the 15 new clones
Same procedure as yesterday (17.09)
Plasmid concentration after plasmid isolation was about 250 to 300 ng/µl. 2 µl of plasmid DNA were used for the restriction reaction.
Digestion Mix
1x Mix (µL)
16x Mix (µL]
Water
15.5
248.0
CutSmart   Buffer (10x)
2.0
32.0
EcoRI
0.25
4.0
PstI
0.25
4.0
Plasmid DNA
2.0
xxx
Incubation for 240 min. digested plasmids were analyzed on a 1% agarose gel.
23.20 pSB1C3 over night restriction
All tested clones were negative. In order to repeat the Gibson assembly pSB1C3 with Hag-KpnI will be digested with KpnI over night (37°C).
Digestion Mix
Amount (µL)
CutSmart Buffer (10x)
2.0
Plasmid
17.00
KpnI
1.0
21.3a Swarming Assay Hag-D2-Strep & -Cup
Aim: Check motility of Bacillus mutants Hag-D2-Cup & -Strep
The OD600 of the overnight cultures was measured and was approx. 5. 100 µL of culture were centrifuged and the pellet resuspended in 50 µL medium to get an OD of 10.
10 µL of the cultures was dropped on Swimming and Swarming plates (0,3% and 0,7% LB Agar plates) and incubated at 37°C.
The cellular diffusion was observed after 30-60 min and the distanced measured starting from the cell circle after letting the drop dry.
Inubation Time (min)
WT3610 Distance (mm)
Hag-D2-Strep Distance (mm)
Hag-D2-Cup Distance (mm)
Swim
Swarm
Swim
Swarm
Swim
Swarm
30
-
-
-
-
-
-
60
-
-
-
-
-
-
90
-
0.5
-
-
-
-
120
0.5
1
0.4
0.5
-
-
150
4.5
2
1
1.5
-
-
180
12
4
4
9
-
-
210
23
10
12
13
-
-
240
31
16
18
17
-
-
270
38
29
23
26
-
-
300
38
36
29
30
-
-
330
38
38
34
32
 
 
 
 
 
38
38
-
-
The measurements showed that Hag-D2-Strep is able to swim and swarm. In comparison to the wild type the strain is slower in swarming and swimming. Hag-D2-Strep did not swarm or swim as the commassie gel showing no flagellin suggested before. Electronmicroscopy and alternatives for Hag-D2-Strep were planned.
For reproduction of the results new precultures of Hag-D2-Strep & -Cup were inoculated and incubated at 30°C.
19.28 Purification of Hag-D2-Strep
Aim: Purification of Hag-D2-Strep for analytical gel filtration was already purified StrepDARPidin
The cultures from yesterday were harvested at 4000 rpm at 4 °C and the pellets were resuspended in 10 mL Buffer A for microfluidizing. The lysate was centrifuged at 20000 rpm so that the clear lysat could be loaded on a 1 mL Ni-NTA column. Preinduction probe (PI), induction probe (I), Load (L), flow through (FT), wash (W) and elution with 15 mL Buffer B (E) were analysed on a SDS-PAGE gel commassie stained:
Meanwhile the elution was concentrated with an amicon at 4000 rpm until an endvolume of 1 mL. The ÄKTApurifier was buffered with GeFi Buffer and the concentrate was injected. Presets: 4 mL fraction size, 0,65 mPa pressure alarm, fractioning after 90 mL run, endvolme 400 mL. A Superdex S200 column was used.
FT and W contained much protein. The 1 mL Ni-NTA had not enough capacity to bind all the protein.
The Ni-NTA column seemed to be contaminated with left StrepDARPidin on the NI-NTA columns from previous use. We made the GeFi run to see if we culd purifier the proteins anyways.
The fractions building the peak were analyzed on a SDS-Gel to cover the peak. Then the fractions from  were concentrated with an amicon to an endvolume of 500 µL. Aliquods of 50 µL were frozen in liquid nitrogen and stored at -80°C.
13.89 Control digestion of the plasmids with SpeI and BamHI and control PCR
Aim: Aim: check isolated plasmids for correct insertion of the promoter sequences
To be sure that the promoter sequences were inserted correctly the isolated plasmids were digested with the enzymes SpeI and BamHI. The difference between the negative and positive clones should be very few but be detectable on the gel: positive: 3426 and approx.3500 bp; negative: 3426 and 3276 bp.
Mix restriction
Single reaction (µl)
Plasmid
6.0
CutSmart  10x
2.0
SpeI
1.0
BamHI
1.0
Water
10.0
Total
20.0
The restriction reaction was incubated at 37°C for over one hour and analyzed with an agarose gel.
Although the colony PCR for most of the plasmids was like expected, some plasmids were not digested at all. The bands around 3000 bp are double bands of the two fragments. A difference between negative and positive plasmids could not be discovered since the amount of DNA was too high and the bands were not separated well enough. It was striking that every undigested plasmid was a plasmid with the piGEM007 backbone (strong degradation tag). For a better check a control PCR was performed with the 007-plasmids as template:
Mix colony PCR
Single reaction
Plamid (2 ng/µl)
2.5
Primer iGEM006 (1:50)
6.25
Primer fw 55/57/59/61 or 63 (1:50)
6.25
5x HF-Buffer
10.0
dNTPs (40 mM)
1.0
Phusion DNA polymerase 1:10
1.0
Water
24.5
Total
50.0
colony PCR Step
Temperature °C
Time (min/sec)KSI
1
98
5 min
2
98
30 sec
3
55
30 sec
4
72
1:00 min
5
Go To 2
34x
6
72
5 min
7
4
infinite
According to the control PCR on the plasmids the plasmids were ok and the promoters were inserted correctly. The copper sensitive promoter is slightly bigger than the used lac or constitutive promoters, which can be seen very well on the gel. The plasmids were assumed to be correct and were ready for further using (trafo of Bacillus and DH5α). However, a further Gibson assembly and ligation reaction (according to 13.85) was carried out with all promoters and plasmids that were negative (piGEM007 + Cu + Ag; 008 + Cu; 009 + Ag + Cu; 007 + const + tetO + lac; 008 + const; 009 + tetO + lac).
19.09.2014
25.2 Preparation of LLCs and A549
Aim: Preparation of LLCs for assays
Lewis Lung cell Carcinoma Cell line and A549 Adenocarcinoma cell line were kindly provided by Dr. Jürgen Adamkiewicz and Margitta Alt (ZTI Marburg) and splitted 1:3 at 37°C in DMEM + 10% FCS and L-Glutamine. The cells were cultivated for further use.
19.09.2014
23.21 Analysis of digested pSB1C3
Checked Test digestion on agarose gel, three bands were visible, which could not be. Only 2 bands were expected.
Check undigested plasmids on agarose gel. Result: Two bands
Retransformation of the already sequenced plasmid pSB1C3-Hag-KpnI for further cloning processes.
21.3b Swarming Assay Hag-D2-Strep & -Cup
Aim: Check motility of Bacillus mutants Hag-D2-Cup & -Strep
The OD of the overnight cultures was measured and was approx. OD 5. 100 µL of culture were centrifuged and the pellet resuspended in 50 µL medium to get an OD of 10.
10 µL of the cultures were spotted on Swimming and Swarming plates (0,3% and 0,7% LB Agar plates) and incubated at 37°C.
The cellular diffusion was observed after 30-60 min and the distanced measured starting from the cell circle after letting the drop dry.
Inubation Time (min)
WT3610 Distance (mm)
Hag-D2-Strep Distance (mm)
Hag-D2-Cup Distance (mm)
Swim
Swarm
Swim
Swarm
Swim
Swarm
30
-
-
-
-
-
-
60
-
-
-
-
-
-
90
4
6
-
6
-
-
120
11
8
5
8
-
-
150
20
16
6
12
-
-
180
32
19
18
20
-
-
210
36
32
26
26
-
-
240
36
32
18
34
-
-
270
36
36
34
36
-
-
300
36
36
36
36
-
-
330
38
38
34
32
 
 
The Hag-D2-Strep strain swarms and swims slower than the WT but is motile in comparison to the Hag-D2-Cup strain.
13.90 Control restriction of nose plasmids
Aim: check isolated plasmids for correct insertion of the promoter sequences
Plasmids were isolated from cultures inoculated yesterday. These plasmids were again digested with BamHI and SpeI to check for the insertion of the promoters.
Mix restriction
Single reaction (µl)
Plasmid
2.0
CutSmart  10x
2.0
SpeI
1.0
BamHI
1.0
Water
14.0
Total
20.0
The reactions were incubated at 37°C for over 1 hour.
All bands were in the expected height of approx. 3400 bp. The plasmids were used further for transformation of DH5α and Bacillus.
20.09.2014
23.22 Inoculation of presumably positive clones from retransformation, plasmid isolation and over night restriction
Aim: Gain more plasmid for Gibson assembly
Inoculation of positive clones of pSB1C3-Hag-KpnI for Mini-prep (3 clones) in the morning
Plasmid isolation in the evening
Overnight restriction of pSB1C3-Hag-KpnI for Gibson assembly.
Check initial plasmids for purity: an additional band was visible, which could probably be coiled plasmid.
13.90a Screening for positive clones containing the new Nose-plasmids
Aim: colony PCR to check colonies for the correct plasmids
The clones on the plates of the new ligation and transformation were screened for positive constructs by colony PCR.
Mix colony PCR
Single reaction
Template
part of a colony
Primer iGEM006 (1:50)
3.125
Primer fw 55/57/59/61 or 63 (1:50)
3.125
5x HF-Buffer
4.0
dNTP mix (10 mM each)
1.0
Phusion DNA polymerase 1:10
0.5
Water
8.75
Total
20.0
colony PCR Step
Temperature °C
Time (min/sec)KSI
1
98
10 min
2
98
30 sec
3
55
30 sec
4
72
1:00 min
5
Go To 2
34x
6
72
5 min
7
4
infinite
As a positive control XLIBlue containing piGEM034 or piGEM035 were used, the negative control was an untransformed XLIBlue. None of the picked clones was positive for the lac and constitutive promoters + tetO. Some clones containing the plasmids piGEM009 + Ag promoter and piGEM009 + Cu promoter were positive. These positive clones were picked and used to inoculate a miniprep culture.
21.4 Analytical gel filtration
Aim: Checking interactions between Hag-D2-Strep and StrepDARPidin
In order to check the interaction between the Strep-Tag in the D2-Flagellin and the Streptavidin in our StrepDARPidin we used an analytical Superdex S200 column. Probes from both proteins were injected into the ÄTKAPurifier and a separate run each done in order to see when the protein gets through the column.
In case of an interaction the complex of both proteins would come off early from the column because of its size. The column was buffered with PBS+2,5% glycerin.
Size and ratio were calculated with http://web.expasy.org/protparam/ and http://christoph-leidig.de/tprot.html for injecting adequate amounts of both proteins.
StrepDARPidin:
Absorption at 280 nm
5.025
Extinction coefficient ε [l/mol*cm)]
57410
Cuvette length [cm]
1
Molecular weight [g/mol]
31809
Protein concentration [µM]
88.57341926493642
Protein concentration [mg/ml]
2.8174318933983624
Hag-D2-Strep/FliS
Absorption at 280 nm
7.015
Extinction coefficient ε [l/mol*cm)]
130786.5
Cuvette length [cm]
1
Molecular weight [g/mol]
130786.5
Protein concentration [µM]
122.19125587876675
Protein concentration [mg/ml]
15.980966686988328
88.57/ 122,19 = 0,7248
(1-0,7248)* 50 µL (StrepDARPidin) =13,76 µL(Hag-D2-Strep)
Analytical run StrepDARPidin 50 µL + 50 µL PBS+2,5% glycerin:
Analytical run Hag-D2-Strep/ FliS 13,7 µL+ 86,3 µL PBS+2,5% glycerin:
13,7 µL Hag-D2-Strep/FliS & 50 µL StrepDARPidin + 36,3 µL PBS+2,5% glycerin incubated together for 1h at room temperature:
The peaks showed no shift to the front and ran into each other. They seemed to show no interaction. In order to be sure we tried to make a pulldown with Strep-Beads.
21.5 Strep-Bead pulldown
Aim: Checking interactions between Hag-D2-Strep and StrepDARPidin
Strep-Beads from Novagene were used to check the interactions between the Strep-Tag in our Flagellin with the StrepDARPidin and the Strep-Beads themselves.
Theoretically the flagellin should bind to the beads and should be found in the elution in contrast to the StrepDARPidin which should be washed off.
In the third elution there should be less Flagellin or even nothing in case of an interaction with the StrepDARPidin because of the competitive situation. The flagellin should bind totally to the StrepDARPidin and for that reason not/ less to the beads.
3 columns were prepaired and filled with 500 µL GeFi Buffer. 20µL Strep-Beads were added and the columns were spinned down at 4000 rpm for 1 min. Then another 500 µL GeFi buffer was added.
The first (1) probe of 3 µL Hag-D2-Strep was pipetted into the buffer. 20 µL StrepDARPidin were added to column 2. Column 3 was used for loading a mixture of 20 µL StrepDARPidin and 3 µL Flagellin onto the column. The three columns were spinned for 20 min at room temperature in a spinning wheel. After that the columns were washed twice with 500 µL GeFi buffer. For eluting 40 µL Elution Buffer from Novagene (with Desthiobiotin) was used to resuspend the beads. After centrifugation at 13000 rpm the elution was mixed with 10 µL SDS-Loading buffer and analyzed on a SDS-PAGE gel with commassie stain.
As a control 40 µL of a 1:10 dilution of the flagellin concentrate was used on the gel. Unfortunately there was not enough StrepDARPidin left for a second control.
Gel-Analysis:
The gel shows that a very tiny amount of the flagellin binds to the Strep-Beads (line 1). The StrepDARPidin does not (line 2). Surprisingly the mixture shows both proteins in the elution (line 3). Unfortunately this cannot be seen on the scanned picture . The control (line 4) shows us that the flagellin purification was not pure enough and the flagellin concentrate was contaminated with StrepDARPidin from the Ni-NTA purification.
The Strep-Beads might have been not efficient working enough because of their age and usage. The binding of the flagellin might tell us that the Strep-Tag is able to interact with the Streptavidin.
In order to get a better and reliable result we decided to purify the flagellin and the StrepDARPidin newly and repeat the pulldown.
New expression cultures of 1L volume each were induced with lactose and incubated overnight at 30°C and 150 rpm.
21.09.2014
23.23 PCR for Gibson assembly fragments Cup-1, D2-Cup and D2-Strep
Aim: Start all fragments from the beginning to exclude mistakes in the early steps of the cloning procedure
→ Templates piGEM021, piGEM 026 and piGEM 027.
Cup-1
D2-Cup
D2-Strep
Primer 1 (1:50)
24 → 6.5
96 → 6.5
96 → 6.5
Primer 2 (1:50)
25 →6.5
97 → 6.5
97 →; 6.5
Template
piGEM021 → 1
piGEM026 → 1
piGEM027 → 1
Phusion Buffer (5x)
10
10
10
Phusion DNA-polymerase
1
1
1
dNTPs
1
1
1
Water
24
24
24
1:30 min elongation times was changed in standard PCR program.
The digested vector (over night) and the PCR products were purified via gel extraction for further use in Gibson assembly reactions.
All fragments have been amplified in the correct size with some side products (probably partially coming from the plasmids used as a backbone).
The digested plasmid yielded in a similar pattern like the last time (2 bands). Therefore the gel was run for additional 20 minutes and a third band could be seen. After additional 40 min it became clear that the middle thin band resembles the digested plasmid. It was therefore digested out and purified together with the PCR products.
13.91 Ligation of piGEM007 and piGEM009 with the const. + tetO promoter
Aim: insert the constitutive promoter with the tetO into the nose plasmids
The ligation was carried out like in 13.85. The reaction was incubated at room temperature for over one hour and E. coli XLIBlue were transformed with the whole reaction mix.
22.09.2014
23.25 Plasmid isolation of pSB1C3-Hag-KpnI for test restriction with KpnI on the one hand and EcoRI/PstI on the other hand
Aim: Analyze the products to get information about purity of the plasmid
Restriction results in the same pattern → idea cook DNA before restriction and shock on ice, restrict for appr. 3h afterwards
23.26 Gibson assembly and transformation
Digestion with EcoRI/PstI showed that the plasmid is pure and not contaminated with other plasmids. Restriction with less DNA and cooking DNA before yielded in a thicker band for the hopefully digested plasmid at 3 kb. Band was digested out and purified via gel extraction (Omega Kit) for further Gibson assembly and transformation.
19.29 new Purification of Hag-D2-Strep
Aim: Purification of Hag-D2-Strep for analytical gel filtration was already purified StrepDARPidin
The cultures from yesterday were harvested at 4000 rpm at 4 °C and the pellets were resuspended in 10 mL Buffer A for microfluidizing. The lysate was centrifuged at 20000 rpm so that the clear lysate (Load/L) could be loaded on a 5 mL Ni-NTA column. Load (L), flow through (FT), wash (W) and elution with 15 mL Buffer B (E) were analysed on a SDS-PAGE gel coomassie stained:
FT and W contain much protein. The 1 mL Ni-NTA had not enough capacity to bind all the protein.
Meanwhile the elution was concentrated with an amicon at 4000 rpm until an endvolume of 1 mL. The ÄKTAPrime was buffered with GeFi Buffer and the concentrate was injected. Presets: 4 mL fraction size, 0,65 mPa pressure alarm, fractioning after 90 mL run, endvolme 400 mL. A Superdex S200 column was used.
The Ni-NTA column seemed to be not contaminated with StrepDARPidin. So the last contamination came form the left StrepDARPidin on the NI-NTA columns from previous use.
Unfortunately the S200 column seemed to be very dirty so that it was not possible to separate the proteins properly.
The fractions were collected and concentrated again. After that the concentrate was injected into ÄktaPrime again with a different S200 with the same settings as before. The run was done overnight.
Additionally new expression cultures with E. Coli BL21(DE3) containing piGEM-030 were induced with lactose for a new purification run.
13.92 Colony PCR to check the ligation of piGEM0077 and piGEM009 with the const. + tetO promoter
Aim: check the transformands for the correct plasmid
Mix colony PCR
Single reaction
Template
part of a colony
Primer iGEM006 (1:50)
6.25
Primer iGEM063 (1:50)
6.25
5x HF-Buffer
10.0
dNTP mix (10 mM each)
0.7
Phusion DNA polymerase 1:10
1.0
Water
25.8
Total
50.0
colony PCR Step
Temperature °C
Time (min/sec)KSI
1
98
10 min
2
98
30 sec
3
55
30 sec
4
72
1:00 min
5
Go To 2
34x
6
72
5 min
7
4
infinite
E. coli XLIBlue piGEM037 (002 + tetO) was used as a positive control. Clones 2 and 3 of piGEM007 + tetO were positive and cultures for a miniprep were inoculated with these clones.
23.09.2014
23.27 Colony PCR (10 clones for all 3 constructs)
PCR Mix
1x Mix (µL)
11x Mix (µL]
Water
8.8
96.8
Buffer
4
44
Phusion DNA-polymerase
1
11
Primer 1
2.6
28.6
Primer 2
2.6
28.6
dNTPs
1
11
Positive clones could only be detected for Cup-1. Clones 5 and 7 looked most promising and were inoculated in LB-Cm overnight for plasmid isolation and test-restriction. PCR for the other two modules has to be repeated or clones need to be inoculated for plasmid isolation and test-restriction.
23.28 Digestion of pSB1C3-Hag-KpnI
Aim: Gain digested pSB1C3 plasmid for further cloning procedures
Digestion Mix
Amount (µL)
Plasmid
5
Water
11
CutSmart  Buffer (10x)
2
EcoRI
1
PstI
1
19.29a new Purification of Hag-D2-Strep
Aim: Purification of Hag-D2-Strep for analytical gel filtration was already purified StrepDARPidin
The fractions building the peak were analyzed on a SDS-Gel to cover the peak.
The gel showed a degradation of our protein. For crystallization it is not adequate enough. A new purification with the expression cultures from yesterday was started.
The cultures from yesterday were harvested at 4000 rpm at 4 °C and the pellets were resuspended in 10 mL Buffer A for lysis by microfluidizer. The lysate was centrifuged at 20000 rpm so that the clear lysate (Load/L) could be loaded on a 5 mL Ni-NTA column. Load (L), flow through (FT), wash (W) and elution with 15 mL Buffer B (E) were analysed on a SDS-PAGE  coommassie stained:
The gel showed no contamination with StrepDARPidin as well.
The elution was concentrated with an amicon at 4000 rpm until an endvolume of 1,5 mL. The ÄKTAPrime was buffered with GeFi Buffer and the concentrate was injected. Presets:
4 mL fraction size, 0,65 mPa pressure alarm, fractioning after 90 mL run, endvolme 400 mL. A Superdex S200 column was used.
The fractions building the peak were analyzed on a SDS-Gel to cover the peak which looked fine.
Then the fractions were concentrated with an amicon to an endvolume of 300 µL. The new robot for setting drops was used for crystallization. Cores I-III were set in full concentration (A280=25) and half concentration diluted with GeFi-Buffer. Left hag-d2-Strep was aliquoted in 50 µL and stored at -80°C.
2 L expression culture were made for new purification attempts and incubated overnight at 30°C after lactose induction.
13.93 Gibson assembly with piGEM008 and the Cu promoter and piGEM007/009 with the Ag promoter
Aim: insert the Ag and Cu promoter into the Nose plasmids
Since some plasmids with the silver and copper sensitive promoters were still missing a new Gibson assembly was carried out with the
Component
Gibson I (µL)
Gibson II (µL)
Gibson III (µl)
Control  (µL)
piGEM-007/008/009 (approx.40 ng/µL)
4
2.5
1
2.5
Cu/Ag promoter (approx.4 ng/µl)
1
2.5
4
-
H2O
-
-
-
2.5
Gibson mix
15
15
15
15
Total
20
20
20
20
Nine reactions were carried out: piGEM 008 + Cu, piGEM009 + Ag and piGEM007 + Ag in three different variations in DNA concentration. The reactions were kept on room temperature for 30 seconds and incubated at 50°C for one hour. Afterwards the whole reaction mix was used to transform E. coli XLIBlue, which were incubated over night at 37°C.
24.09.2014
23.29 Digest pSB1C3-Hag-KpnI-Cup-1 EcoRI/PstI and subsequent transformation
Both prepared plasmids have been digested with EcoRI/PstI and compared with the digested 13C-Hag-KpnI. No significant difference could be detected. Therefore different Plasmids have been digested with EcoRI/PstI and will be compared for the different inserts.
New transformation with sequenced plasmids → strepDARP clone 1 and Hag-KpnI clone 4
18.72 Gelfiltration purification of StrepDARPidin tetramers
Aim: Purification of StrepDARPidin from inclusion bodies
10 L expression culture were inoculated with clones from a BL21(DE3) transformation plate and induced with lactose overnight at 30°C.
19.30 Purification of Hag-D2-Strep for crystallization
Aim: Purification of Hag-D2-Strep for analytical gel filtration was already purified StrepDARPidin
The cultures were harvested at 4000 rpm at 4°C and the pellet was frozen in liquid nitrogen and stored at -80°C for further use.
13.93 Screening of transformands of Gibson assembly of piGEM008/009 and the Cu promoter and piGEM007 with the Ag promoter
Aim: check the transformands for the correct plasmid
A colony PCR was carried out to screen the transformands for the correct plasmids. No transformands could be detected on the plate with piGEM008 + Cu.
Mix colony PCR
Single reaction
Template
part of a colony
Primer iGEM006 (1:10)
1
Primer iGEM063 (1:10)
1
5x HF-Buffer
4
dNTP mix (10 mM each)
0.5
Phusion DNA polymerase 1:10
1
Water
12.5
Total
20
colony PCR Step
Temperature °C
Time (min/sec)KSI
1
98
10 min
2
98
30 sec
3
55
30 sec
4
72
1:00 min
5
Go To 2
34x
6
72
5 min
7
4
infinite
As a positive control E. coli XLIBlue piGEM034 (02 + Ag) was used. Clone 4 was positive. A LB culture was inoculated with clone 4 for a miniprep. It was grown shaking at 37°C over night.
It was noticed that the IPTG inducible Gram positive (described as lac) promoter, the strong constitutive for gram positive promoter and the tetO including variant were designed falsely. Only a part (30 bp) of the sequence was taken and cloned. That means that all nose plasmids containing the lac, const. and const. + tetO promoters were disposed. Only the plasmids with the metal ion sensitive promoters were used further.
13.94 Production of competent Bacillus subtilis PY79 cells with test transformation
Aim: Making PY 79 competent to take up foreign DNA
Since the last PY79 cells were not competent a new try of producing competent cells was carried out according to the protocol in the methods section using SPC and SPII medium. After making the cells competent they were stored in a -80°C freezer without shock freezing in liquid nitrogen. The protocol does not including shock freezing of the cells. For a test transformation four aliquots of cells were thawed on room temperature and 100 µl each were transferred into sterile test tubes and mixed with 7 µl of the plasmids piGEM034, piGEM035, piGEM044 and piGEM050. The test tubes were incubated shaking at 37°C for 0.5 hours. The whole transformation samples were plated out on LB-chloramphenicol (5 µg/ml) plates and were incubated at 30°C for several days.
25.09.2014
23.30 Inoculation of presumably positive clones from the tramsformation
Transformation from yesterday: inoculate clones at 8 in order to isolate the plasmids, perform Gibson assembly and transformation.
Plasmid isolation → To exclude a that samples were exchanged by mistake a test-restriction of the plasmids has to be performed with EcoRI/KpnI, since KpnI does not digested in pSB1C3-Strep DARP. When cutting Hag-KpnI a 600 bp fragment should occur in the gel whereas no fragment is expected when cutting pSB1C3-Strep DARP. According to the test restriction samples have not been switches, restrictions 1, 2 and 4 can be used for the Gibson assembly (and transformation). Restriction reaction 1 was used.
Digestion Mix
Amount (µL)
Expected:  Strep Darp: only one fragment Hag-KpnI: appr. 2330 + 670
Water
13.4
CutSmart  Buffer (10x)
2.0
Plamid (100 ng)
4.0
EcoRI
0.3
KpnI
0.3
Restriction (25µl mix with complete plasmid)
Digestion Mix
Amount (µL)
Water
1.5
CutSmart  Buffer (10x)
2.5
Plasmid (500 ng)
20.0
KpnI
1.0
18.72 Gelfiltration purification of StrepDARPidin tetramers
Aim: Purification of StrepDARPidin from inclusion bodies
The cells from yesterday were harvested and cracked with the microfluidizer after resuspension in 40 mL Buffer A (lysate)).
The suspension was centrifuged at 20.000 x g for 15 min. The supernatant (S1) was discarded  and the pellet (P1) resuspended in 5 mL IB Wash Buffer. Then the suspension (SS1) was centrifuged at 27.000 x g for 10 min. The resulting pellet (P2) and the supernatant (S2) were used for taking probes for SDS-PAGE. The wash was repeated. 10 mL 6M Guanidinium-HCL solution for solubilisation and unfolding of the StrepDARPidin was added and the pellet resuspended. The suspension was incubated at room temperature permanently stirring with a magnet.
After centrifugation at 4000 rpm for 10 min the supernatant was used for refolding. The membranous pellet (P3) was thrown away.
For refolding 5 mL of the Guanidinium-HCL 100 mL Buffer A were filled into a Erlenmeyer-Flask on ice with a magnet stirrer inside on 600 rpm. The other 5 mL Guanidinium-HCL/ Pellet resuspension was stored in the fridge until further use.
For refolding it was necessary to dilute the Guanidinium-HCL dilution very fast and softly. For that purpose rapid dilution was done by pipetting the supernatant drop by drop into the vortex of the Buffer A. After adding the whole Guanidinum-HCL/ Protein solution the suspension was observed and precipitation noticed. The suspension was centrifuged at 20.000 rpm (Pellet P4) . The supernatant was loaded on a Ni-NTA Column from GE-Healthcare (L). Flow through (FT), Wash (20 mL W) in Buffer A and Elution (15 mL) (E) in Buffer B were kept for gel analysis. The samples lysate to Load were cooked for 5 min at 95 degrees.
From every fraction 40 µL were taken for a coommassie SDS-PAGE and 10 µL 10x SDS Buffer added. The pellet samples were resuspended in 60 µL water and 40 µL SDS-Buffer.
The gel shows that the monomeric and tetrameric StrepDARPidin can be found in Load, Flow through and elution. The forming of the tetramer tells us that the refolding had to be successful partially.  The elution was concentrated with an Amicon to an endvolume of 1,5 mL. 75 µL glycerol were added and the concentrate was frozen away in liquid nitrogen and stored at -80°C for further use.
13.95 restriction of piGEM007/008 and 009 with NcoI and SacI
Aim: linearize the plasmids to for further cloning of the promoters
The plasmids with the ssrA-degradation tags were digested with NcoI and SacI to linearize them for further cloning of promoters in front of the GFP. Different concentrations of plasmid were digested with the enzymes to test the capacity of the restriction enzymes and to be sure that all plasmid was cut.
Component
Concentration I (µL)
Concentration II (µL)
Concentration III (µL)
piGEM-007/008/009 (approx.400 ng/µL)
2
5
7
NcoI
1
1
1
SacI
1
1
1
CutSmart  Buffer 10x
2
2
2
Water
14
11
9
Total
20
20
20
The reaction was incubated at 37°C for over one hour and afterwards separated on a gel. As a control the undigested piGEM008 was used.
No difference between the undigested and the digested plasmids could be detected, but a small band directly under the thick, visible one. The linearized plasmids should have a size of 6711 bp. It is possible that the undigested plasmids do rarely form supercoiled structures and mostly behave like the linear form, only a small part of it is supercoiled. Since the same plasmids were digested before with the same enzymes and looked the same on the gel but were linear (further cloning of promoter sequences otherwise not possible), the plasmids were assumed to be linearized. The three concentrations of each plasmid were pooled and were purified with an Omega Gel Ex kit.
26.09.2014
23.31 Plasmid isolation and new Gibson assembly
No colonies on plates. New Miniprep with much higher concentrations of plasmid → yesterday mostly only 20 ng/µl. Band before gel extraction was very thin.
New Gibson assembly and Transformation. Clones will be inoculated tomorrow; spin down and pellets will be frozen at -20°C.
18.71 Gelfiltration purification of StrepDARPidin tetramers
Aim: Purification of StrepDARPidin from inclusion bodies
The stored protein concentrate was thawn up on ice and injected into the injection valve  of the ÄktaPrime which was buffered with PBS+ 2,5% glycerin. A S200 Sepharose  Column was used for the gel filtration run.
40 µL of the Fraction 31. 37, 38 and 44 were picked for SDS-PAGE analysis. 10 µL SDS-Buffer were added and 20 µL loaded on the gel. Additionally the rest was boiled and the same amount loaded onto the gel.
The gel shows us that the whole peak contains the tetrameric StrepDARPidin. After cooking it up the tetramer dissolves into the monomeric StrepDARPidin.
The fractions covering the peak were concentrated with an amicon to an endvolume of 500 µL. 25 µL glycerin were added and the protein aliquoted in 60 µL aliquods with an Absorption of 8 A280 nm.
13.96 Gibson Assembly of piGEM007 +Ag promoter and piGEM008/009 with Cu promoter
Aim: insert the promoter in the nose plasmids
A new Gibson assembly was carried out to insert the missing promoters into the different nose plasmids. Every combination (007 + Ag, 008 + Cu, 009 + Cu) was carried out in different relations of vector to insert:
Component
equal amounts (µL)
condition II for 07 + Ag (µL)
condition II for 08 + Cu (µL)
condition II for 09 + Cu (µL)
piGEM-007/008/009 (approx.40 ng/µL)
2.5
2.5
2.5
2.5
Cu/Ag promoter (approx.5 ng/µl)
2.5
1.2
1.4
0.95
H2O
-
1.3
1.1 
1.55
Gibson mix
15
15
15
15
Total
20
20
20
20
The reactions were kept on room temperature for 0.5 min and were incubated at 50°C for 1 h. Afterwards E. coli XLIBlue were transformed with the whole reaction mix, plated on LB-ampicillin and incubated over night at 37°C.
The existent nose plasmids (piGEM034/035/039/044/050) were previously transformed into DH5α cells, which were plated out new to inoculate cryo stocks of the cells.
13.97 Transformation of Bacillus subtilis PY79 cel ls
Aim: comparison of competence of PY79 cells
To compare the competence of our PY79 cells with cells that are competent for sure, one aliquot of competent PY79 were borrowed together with three LB-chloramphenicol plates. The 300 µl aliquot was divided into 100 µl portions. One negative control was mixed with 10 µl water, the other two 100 µl portions were treated with 5 µl and 10 µl piGEM034 (approx.400 ng/µl), which has been shown that it could be transformed. The cell/DNA mixture was incubated in a test tube, shaking for 30 minutes before plating them out on the mentioned medium plates. This time the plates were incubated at 37°C.
22.16 amplification of KSII in single modules
Aim: amplification of KSII
KSII could not be amplified with the nesting primers which cover the whole KSII module, even with a variety of modified reaction conditions. Thus, primers were ordered to amplify the modules (holing, tetR, rplE (L5)) separately.
Component
holin (µL)
tetR (µL)
L5 (µL)
KSII gBlock DNA (1:10)
1
1
1
Primer iGEM065 (1:50)
6.25
-
-
Primer iGEM069 (1:50)
-
1
-
Primer iGEM067 (1:50)
-
1
-
Primer iGEM068 (1:50)
-
-
1
Primer iGEM066 (1:50)
-
-
1
Primer iGEM070 (1:50)
-
-
1
HF Buffer 5x
10
10
10
Phusion DNA-polymerase
1
1
1
dNTPs
0.5
0.5
0.5
Water
25
25
25
Total
50
50
50
colony PCR Step
Temperature °C
Time (min/sec)KSI
1
98
5 min
2
98
30 sec
3
55
30 sec
4
72
1:00 min
5
Go To 2
34x
6
72
5 min
7
4
infinite
The sizes of the expected fragments were: holin: approx.700 bp, tetR: 684 bp and L5: 540 bp. The fragments of the expected size could be seen, it meets the eye that the unspecific bands that appeared in previous amplifications of KSII with the nesting primers were visible as well in the lane of tetR. However, tetR also showed an amplificate with the correct size.
Although the KSII parts were amplified successfully no further work will be done with KSII, because of the false promoter in front of these genes. Like mentioned before the IPTG inducible promoter for Gram positives and the strong constitutive promoter for Gram positives used in the killswitch and nose and taken from the iGEM registry were falsely designed. Only a short part of the real sequence was used.
For this reason the promoters were designed again correctly: the lac promoter (BBa_K090501; 107 bp) will be ordered in single stranded oligonucleotides from which one contains overhangs like it was digested with NcoI and SacI. These oligos will be annealed and phosphorylated. A ligation with the nose plasmids and KSI follows. It was only enough time left to build KSI anew and not the other killswitch modules. The primer to connect the new lac promoter with the rest of KSI also introduces a RBS.
The constitutive promoter (BBa_K090504) will be amplified from gDNA of Bacillus cereus, which was kindly provided by members of the Zentralinstitut für Ernährungs- und Lebensmittelforschung ZIEL, Technische Universität München. The amplificate will be used for ligation with the nose plasmids.
27.09.2014
18.73 Gelfiltration purification of StrepDARPidin tetramers
Aim: Purification of StrepDARPidin from inclusion bodies
The left 5 mL Gua-suspension in the fridge were refolded with the same protocol from 18.72.
P1 shows a very weak line at the expected size which comes from picking just a membranous part on top of of the pellet. The line would have been more prominent if we had taken more of the pellet itself. The gel shows that the monomeric and tetrameric StrepDARPidin can be found in Load, Flow through and elution. The forming of the tetramer tells us that the refolding had to be successful partially.  The elution was concentrated with an Amicon to an endvolume of 1,5 mL. 75 µL glycerol were added and the concentrate was frozen away in liquid nitrogen and stored at -80°C for further use.
28.09.2014
23.32 Test digest
Saturday clones could be observed on all plates (more than on the control plate). 5 clones have been inoculated in LB-Cm and were centrifuged and frozen in the evening. Plasmid isolation were performed in the evening and restriction was performed over night.
Digestion mix
1x Mix (µL)
18x Mix (µL]
Plasmid
10
x
Water
7.4
133.2
Buffer
2.0
36.0
EcoRI
0.3
5.4
PstI
0.3
5.4
18.71 Gelfiltration purification of StrepDARPidin tetramers
Aim: Purification of StrepDARPidin from inclusion bodies
The stored protein concentrate was thawn up on ice and injected into the injection valve  of the ÄktaPrime which was buffered with PBS+ 2,5% glycerin. A S200 Sepharose  Column was used for the gel filtration run.
40 µL of the Fraction 32. 38, 39 and 45 were picked for SDS-PAGE analysis. 10 µL SDS-Buffer were added and 20 µL loaded on the gel. Additionally the rest was boiled and the same amount loaded onto the gel.
The gel shows us that the whole peak contains the tetrameric StrepDARPidin. After boiling it the tetramer dissolves into the monomeric StrepDARPidin.
The fractions covering the peak were concentrated with an amicon to an endvolume of 500 µL. 25 µL glycerin were added and the protein aliquoded in 60 µL aliquods with an Absorption of 7 A280 nm.
29.09.2014
23.33 Analysis of the test digest
Expected size
Cup-1
D2-Strep
D2-Cup
Plasmid
2 kb
2 kb
2 kb
Insert
1043 bp
941 bp
1463 bp
2nd insert
x
450
x
Presumably Cup-1 clones 3-5 are positive, all D2-Strep clones are positive but D2-Cup clones are negative. Therefore PCR reactions will be performed with the positive plasmids and colony PCR with 10 clones of D2-Cup will be added.
PCR Mix
Cup-1
D2-Cup
D2-Strep
Expected size
746 bp
1103 bp
1166 bp
Water
33.2
102.3
49.8
Buffer (5x)
16
44
24
Primer 1 (1:50)
10.4 (iGEM 025)
28.6 (iGEM 053)
15.6 (iGEM 053)
Primer 2 (1:50)
10.4 (iGEM 053)
28.6 (iGEM 097)
15.6 (iGEM 097)
dNTPs
4
11
6
Phusion
4
11
6
Template
0.5
x
0.5
1:30 Elongation time.
Agarose gel: Colony PCR for Hag-KpnI-D2-Cup is negative, but all other results from the test restriction have been confirmed.
25.3 Preparation of LLCs
Aim: Preparation of LLCs for assays
We decided to cultivate the LLCs for Fluorometer assays in 12-Well plates.  The medium of the 25 mL culture flask was used to resuspend the cells carefully because of their sparsely adherent  trait.
The cells were transferred into a 50 mL the  culture was spinned down at 1500 rpm  for 5 min at 4°C.
The supernatant was discarded and resuspended in 10 mL DMEM. The cells were counted with a Neubauer-chamber.
10 µL of the cell suspension was mixed with 200 µL Trypan blue and 85-100 cells were counted in the grids of the chamber.
Calculation:
90 (cells total in grid) / 4 (number of quadrants) *20 (dilution factor) * 10000 (Neubauer factor) * 10 mL (volume of suspension)= approx. 40 mio. Cells/ 10 mL
We calculated with 50 mio. Cells in total.
The suspension was filled up to 50 mL to get 1 mio cells/ mL. 2 mL/ well were pipetted into each well of the 12-well plate and incubated overnight at 37°C.
A Black Fluotrac600 96-well plate was kindly received from the immunology. Row H 1-12 was coated with 100000 cells/ well overnight at 37°C.
13.98 Screening of the transformands from the Gibson by colony PCR
Aim: check the transformands for the correct plasmid
A colony PCR was carried out to screen the transformands for the correct plasmids. No transformands could be detected on the plate with piGEM008 + Cu.
Mix colony PCR
Single reaction
Template
part of a colony
Primer iGEM006 (1:10)
1
Primer iGEM057 (1:10)
1
5x HF-Buffer
4
dNTP mix (10 mM each)
0.5
Phusion DNA polymerase 1:10
1
Water
12.5
Total (µL)
20
PCR Step
Temperature °C
Time (min/sec)KSI
1
98
10 min
2
98
30 sec
3
55
30 sec
4
72
1:00 min
5
Go To 2
34x
6
72
5 min
7
4
infinite
As a positive control E. coli XLIBlue piGEM034 and piGEM035 were used. No positive clones could be seen. Indeed there was a band at the right size of clone 2 07 + Ag but there was a bigger and a smaller band as well.
13.99 Competent Bacillus subtilis PY79
The transformation of the previus batch of cells with the plasmid piGEM034 worked only in case of the higher concentration. Two colonies could be observed. For the next competent cells fructose will be used instead of glucose. For making of new competent PY79 cells the cells were plated out again on a new LB plate and were incubated at 37°C over night.
30.09.2014
23.34 Repeated agarose gel of the colony PCR
Better image, bands occur exactly at expected size. Clone 3 from the Cup-1 clones will be sent for sequencing today. Clone 1 - 4 will be used for mutagenesis, when mutagenesis primer arrives.
Label number
Construct
Added Primer
AGB0023-557
1C3-Hag KpnI-Cup1
piGEM 053
AGB0023-558
1C3-Hag KpnI-Cup1
piGEM 054
18.72 Isolation of flagella from Hag-D2-Strep
Aim: Isolation of filaments for assays with StrepDARPidin
The flagella isolation was tried for the strain Hag-D2-Strep according to the protocol from ''Purification and Thermal Stability of Intact Bacillus subtilis Flagella''.
25.3a Preparation of LLCs and A549
Aim: Preparation of LLCs and A549 for assays
Because of the literature we found we decided to throw the LLCs for now into the waste. We did not find enough information about the EpCAM expression in that cell line.
Instead we prepared the A549 for a Fluophor-Assay tomorrow.  The 25 mL flask was splitted 1:3. The medium of the culture flask was discarded from the side opposite to the adherent cells and washed with 1X PBS from the opposite site as well.
The PBS was aspirated with a glass pipette until the supernatant was clear. After adding 4 mL 1x trypsin from the opposite of the adherent cells the flask was incubated for 5 min at 37°C depending on how fast the cells come off from the ground. Under the microscope the free floating cells were checked.
The cells were transferred into a 15 mL falcon and rinsed with 10 mL DMEM to collect all remaining cells in the flask. The 15 mL falcon was filled with DMEM and the culture spinned down at 1500 rpm  for 5 min at 4°C.
The supernatant was discarded and resuspended in 5 mL DMEM.
10 µL of the cell suspension was mixed with 200 µL Trypan blue and approx. 140 cells were counted in the grids of the chamber.
Calculation:
140 (cells total in grid) / 4 (number of quadrants) *20 (dilution factor) * 10000 (Neubauer factor) * 5 mL (volume of suspension)= approx. 35 mio. Cells/ 10 mL
The cells were splitted. 2 mL :1 mL : 1 mL of cell suspension was filled into 3 new 15 mL flasks and filled up with 10 mL DMEM+10%FCS and L-Glutamin. The 2mL flask was planned for microscopy on Friday with the StrepDARPidin constructs. The other flasks are for backups and assays. Incubation was performed at 37°C.
Additionally  row H1-12 of the Fluotrac600 was coated with 25 µL cell suspension and 100 µL DMEM +10%FCS+L-Glut. (incubation at 37°C).
As a backup Row F of a 6x8 was filled with 25 µL cell suspension as well and 300 µL DMEM +10%FCS+L-Glut. (incubation at 37°C).
19.30 Purification of Hag-D2-Strep for crystallization
Aim: Purification of Hag-D2-Strep for crystallization
The pellets from 23.09.14  were resuspended in 10 mL Buffer A for microfluidizing. The lysat was centrifuged at 20000 rpm so that the clear lysat could be loaded on a 5 mL Ni-NTA column. Load (L), flow through (FT), wash (W) and elution with 15 mL Buffer B (E) were analysed on a SDS-PAGE gel commassie stained:
Further purification was done. Meanwhile the elution was concentrated with an amicon at 4000 rpm until an endvolume of 1 mL. The ÄKTApurifier was buffered with GeFi Buffer and the concentrate was injected. Presets: 4 mL fraction size, 0,65 mPa pressure alarm, fractioning after 90 mL run, endvolume 400 mL. A Superdex S200 column was used.
Then the fractions building the peak were concentrated with an amicon to an endvolume of 500 µL which were used for setting dropes with core I and IV (QIAGEN cores). Aliquods of left flagellin of 50 µL were frozen in liquid nitrogen and stored at -80°C.
13.100a Competent Bacillus subtilis PY79
Aim: making PY79 cells competent
The SPC medium and the SPII medium were assembled. The media were prewarmed before the cells were transferred into the media. New LB-chloramphenicol plates were made. After making the cells competent their competence was tested by transformation of piGEM034.
13.101 Restriction of piGEM007/008/009
Aim: linearization of the plasmids and test of the restriction enzymes
The yield of transformands after ligations or Gibson assemblies with the plasmids piGEM007, piGEM008 and piGEm009 was erratically fewer than with the piGEM002, which was digested a few weeks ago. Since the only difference between these plasmids are the ssrA-degradation tags on the gfp-gene, there should be no difference in their restriction behavior. This is aggravated by the fact that all four plasmids contain the restriction sites for NcoI and SacI on the same positions. For this reason the plasmids were digested again, together with test plasmids to check the activity of the enzymes NcoI and SacI. The test plasmids were pMAD that contains two SacI sites, thus resulting in fragments of 4431 bp and 5235 bp size and piGEM002, digested by NcoI and EcoRI yielding in fragments of 476 bp and 6190 bp.
Component
Restriction of 07/08/09 (µL)
Test I (µL)
Test I (µL)
piGEM-007/008/009 (200-300 ng/µL)
10
-
-
piGEM002 (approx.400 ng/µl)
-
2
-
pMAD (114 ng/µl)
-
-
2
NcoI
1
1
-
SacI
1
-
1
EcoRI
-
1
-
CutSmart  Buffer 10x
2
2
2
Water
14
14
15
Total
20
20
20
The reactions were incubated at 37°C for over one hour.
According to the bands NcoI was functional, the expected bands could be seen after the restriction of piGEM002. However, the functionality of SacI was not fully determined since there was no control with the undigested plasmid. A band in the height between 5000 and 6000 bp could be seen, another very slight band at approx.4 kb could also be noticed, what seems to be the other expected fragment. Yet it the bigger band was more intense than the smaller one. The digested plasmids piGEM007, 008 and 009 were in had the correct size and seemed to be digested. These bands were pooled and purified via a Gel Ex kit.
Retrieved from " http://2014.igem.org/Team:Marburg:Project:Notebook:September "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
