"
Page
Discussion
View source
History
teams
Log in
 
Team:Marburg:Project:Notebook:April
From 2014.igem.org
Home
Project
Safety
Team
Policy & Practices
Attributions
SilverSURF
CancerSURF
RiboSURF
Notebook
Parts
Notebook: April
March
April
02.04.2014
03.04.2014
04.04.2014
07.04.2014
08.04.2014
09.04.2014
10.04.2014
11.04.2014
12.04.2014
14.04.2014
16.04.2014
17.04.2014
22.04.2014
23.04.2014
24.04.2014
25.04.2014
26.04.2014
28.04.2014
29.04.2014
30.04.2014
May
June
July
August
September
October
Methods
02.04.2014
9.1 PciI: Restriction Digest
Digestion scheme:
[µL]
PSG1164 - uncut
PSG1164 - PciL (10 kU/mL)
PSG1164 - EcoRI-HF
DNA
261,3 ng/µL
ca. 1 µg → 3,5 µL
261,3 ng/µL
ca. 1 µg → 3,5 µL
261,3 ng/µL
ca. 1 µg → 3,5 µL
Enzyme 1
-
PciI
1,0 µL
-
Enzyme 2
-
-
EcoRI-HF
0,5 µL
Buffer
-
10x 3.1 Buffer
1,5 µL
10x Cutsmart
1,5 µL
H2O
11,5 µL
9,0 µL
9,5 µL
Total Volume
15 µL
15 µL
15 µL
Loading Dye (6x)
2,5 µL
2,5 µL
2,5 µL
Incubate samples for 60 min at 37 °C, then induce a heat shock for 2 min at 45 °C. 5 µL marker and 10 µL of the samples are loaded onto a 1% agarose gel.
Results:
03.04.2014
10.1 pMA12: Restriction Digest
Aim: linearizing the backbone for cloning of our Nose plasmid + destroying the lac promoter
Digestion scheme:
[µL]
pMA12 - uncut
pMA12 - HindIII + EcoRI-HF (20 kU/mL)
DNA
479,3 ng/µL
ca. 1 µg →2,1 µL
479,3 ng/µL
ca. 5 µg → 10,5 µL
Enzyme 1
-
HindIII
0,5 µL
Enzyme 2
-
EcoRI
0,5 µL
Buffer
-
10x -2.0 Buffer
1,5 µL
H2O
12,9 µL
2 µL
Total Volume
15 µL
15 µL
Loading Dye (6x)
2,5 µL
2,5 µL
Results:
The restriction was successful.
10.3 Gel Extraction (QIAquick Gel Extraction Kit)
Gel extraction was performed according to the quiagen kit protocol.
DNA-concentration: 12 ng/µL
10.4 Test Transformation of E.Coli DH5α with isolated Plasmid Digest
04.04.2014
13.1 Vector Constructs
Dilution of primers:
Stock: dilution of primers with aqua bidest. to 100 µM
Working stock: 1:10 (10 µL primer stock, 90 µL water)
Oligo Name
µL for 100 µM
iGEM-001
292 µL
iGEM-002
236 µL
iGEM-003
397 µL
iGEM-004
291 µL
iGEM-005
364 µL
iGEM-006
273 µL
Dilution of templates:
Bacillus gDNA: 1:50 → 10 µL gDNA + 490 µL water
pKH-Stp: 10 ng/µL
PSG1164: 232 ng/µL → 1:20 → 10 µL plasmid + 190 µL water
Reaction Mix
 
1&2 plasmid PSG1146 (856 bp)
3&4 Bacillus gDNA (543 bp)
5&6 plasmid KG-Stp (743 bp)
Fragment
chloramphenicol-resistance
amyE-gene
gfp
Template
1,5 µL
1,5 µL
1,5 µL
dNTPs
1 µL
1 µL
1 µL
Primer fwd
1 µL
1 µL
1 µL
Primer rev
1 µL
1 µL
1 µL
Phusion Buffer (5x)
10 µL
10 µL
10 µL
Phusion
1 µL
1 µL
1 µL
H2O
34,5 µL
34,5 µL
34,5 µL
Total Volume
50 µL
50 µL
50 µL
Step
Temperature °C
Time
1
98
3 min
2
98
30 sec
3
55
30 sec
4
72
1 min
5
Go To 2
30x
6
72
4 min
7
8
infinite
4 µL Mix + 1 µL Loading dye
5 µL Gene ruler
5 µL 2-log ladder
Results
13.5 Purification of PCR Products
Purification according to the "Purification of PCR Products (QIAquick Gel Extraction Kit) short protocol".
Samples:
1' chloramphenicol-resistance
2' amyE-Gene
3' gfp
4' (=2+4+6)-Pool, all fragments
13.6 Yeast Transformation
[µL]
1''-Control
2''-P+frag
3''-P*+pool
4''-P**+pool
PEG 3350(50% w/v)
260 µL
260 µL
260 µL
260 µL
LiAc 1 M
36 µL
36 µL
36 µL
36 µL
Salmonsperm(2 mg/mL)
50 µL
50 µL
50 µL
50 µL
H2O
14 µL
10 µL
2 µL
2 µL
Fragm. (GFP, Cat, AmyE)
-
4 µL
-
-
Pool (all fragments)
-
-
12 µL
12 µL
Total Volume
360 µL
360 µL
360 µL
360 µL
Salmonsperm for 10 min on 95 °C, then on ice
*pMA12 - digestion with HindIII + EcoRI
**pMA12 - parallel digestion with HindIII and EcoRI
resuspend yeast pellet in all samples
induce heat shock at 42 °C for 35 min
centrifuge for 30 sec - 1 min at 13.000 g
resuspend pellet in 200 µL sterile water
plate on selective medium
incubate at 30 °C for 3 days
07.04.2014
13.8 Miniprep of Yeast Culture
Miniprep of Yeast culture 2'', 3'' and 4''
flush cells from plate with 2 mL water
centrifuge for 1 min at 11.000 rpm
pellet cracking: add glass balls
→ miniprep accoding to the miniprep procotol (Omega)
13.9 Test PCR with Miniprep
Test PCR with Miniprep 2'', 3'' and 4''
Dilution of elution to 20 ng/µL:
2'': 72 ng/µL*X=10*20 ng/µL → 5,3:4,7
3'': 38 ng/µL*X=10*20 ng/µL → 3:7
4'': 66 ng/µL*X=10*20 ng/µL → 2,7:3,7
 
2''' 20 ng
3''' 20 ng
4''' 20 ng
Content
P out of fragments
P out of pool
P (DS) out of pool
Template
1 µL
1 µL
1 µL
dNTPs
1 µL
1 µL
1 µL
Primer001 fwd
1 µL
1 µL
1 µL
Primer006 rev
1 µL
1 µL
1 µL
Buffer (10x)
3 µL
3 µL
3 µL
Taq-Pol
1 µL
1 µL
1 µL
H2O
22,5 µL
22,5 µL
22,5 µL
Total Volume
30 µL
30 µL
30 µL
PCR for 3 h
The control PCR was not successful.
Results
13.11 Amplification in DH5α
Amplification of plasmid in E.coli DH5α.
transformation of 2'', 3'' and 4'' in DH5α
plating on each ampicillin and canamycin plates
13.12 Miniprep of E.coli pMA12: Preculture
Inoculation of preculture plates: 6 mL LB + 6 µL ampicillin + colony clone
08.04.2014
13.13 New PCR
[µL]
Master Mix (7x)
2''' (20 ng) x2
3''' (20 ng) x2
4''' (20 ng) x2
Template
Reaction mix
P out of fragments
P out of pool
P (DS) out of pool
dNTPs
-
1
1
1
Primer001 fwd
3,5
-
-
-
Primer006 rev
3,5
-
-
-
Buffer (10x) Thermo-Pol
17,5
-
-
-
Taq-Pol
0,7
-
-
-
H2O
139,3
-
-
-
Master Mix
-
24
24
24
Total Volume
168
25
25
25
Step
Temperature °C
Time
1
95
2 min
2
95
20 sec
3
58
20 sec
4
68
2,5 min
5
Go To 2
40x
6
68
4 min
7
8
Infinite
The new control PCR was not successful either.
13.14 Picking Clones for Sequencing
Inoculation of 4x6 mL LB with a clone (trafo 13.11.14) from plate 2,3 and 4 (12 falcons). Incubation overnight.
14.1 PCR of PheA-Arc1p-C-8x
Aim: Amplify PheA-Arc1p-C-8x fragment for Gibson Assembly in pET-28a
The PCR with TG_PheA-Arc1p-C-8x FP und TG_PheA-Arc1p-C-8x RP should generate the 2346 bp fragment for Gibson Assembly from the synthesized template.
The used Primer concentration was 1 µM, the concentration of the template was 1 ng/µL and we used a hot start to avoid unspecific annealing of primers.
Content
Volume [µL]
Water
29.5
5x GC Buffer
10
DMSO
2.5
TG_PheA-Arc1p-C-8x FP
2.5
TG_PheA-Arc1p-C-8x RP
2.5
synthesized template
1
dNTPs
1
Phusion-Polymerase
1
Total Volume
50
Step
Temperature [°C]
Time [min:sec]
1
98
2:00
2
add Polymerase
 
3
98
0:10
4
67.5
0:30
5
72
1:40
6
go to 3
32x
7
72
10:00
8
4
hold
The amplified fragment was purified on an agarose gel on which it showed the correct size. It was extracted from the gel for further use in a Gibson Assembly yielding a concentration of 258 ng/µL.
14.2 Digestion of pET-28a
Aim: Linearize pET-28a to prepare it for Gibson Assembly
The pET-28a vector was cut with XhoI and NdeI in order to create the ends needed for the integration of the prepared PheA-Arc1p-C-8x construct via Gibson Assembly
Content
Volume [µL]
Water
34.2
CutSmart Buffer
5
pET-28a (128 ng/µL)
7.81
NdeI
1.5
XhoI
1.5
Total Volume
50
The reaction mixture was incubated for 3 h at 37 °C and 350 rpm. The resulting linearized vector was purified using an agarose gel from which it was extracted for further use yielding a concentration of 24 ng/µL.
14.3 Gibson Assembly
Aim: Integrate PheA-Arc1p-C-8x into linearized pET-28a(XhoI,NdeI) via Gibson Assembly
Content
Volume [µL]
Water
1.46
pET-28a(XhoI,NdeI) (24 ng/µL)
4.17
PheA-Arc1p-C-8x (30 ng/µL)
4.37
2x Gibson Mastermix
10
Total Volume
20
The reaction mixture was incubated for 1 h at 50 °C and 350 rpm. E. coli Top10 electrocompetent cells were transformed with a 1:3 and a 1:6 dilution of the mixture, incubated for 1 h at 37 °C and plated out on LB-Kan50 plates that were incubated overnight at 37 °C.
13.12 Miniprep of E.coli pMA12
Miniprep according to the miniprep kit protocol (Omega) with overnight preculture.
DNA-concentration: 660 ng/µL
15.1 Cloning of Flagellin Construct
Dilution of Primers
dilution with aqua bidest to 100 µM
dilution 1:10 → 10 µL primers (100 µM) with 90 µL aqua bidest
Reaction Mix
[µL]
HagI (1145 bp)
HagII (859 bp)
Content
1
1
Template Flo Pet24-Hag
1
1
dNTPs
1
1
Primer iGEM-007 fwd (Hag-SpeI-F)
1
-
Primer iGEM-008 red (Hag-SpeI)
-
1
Primer 12 rev Flo
1
-
Primer 11 few Flo
-
1
Q5 Reaction Buffer 5x
5
5
Q5-Pol
1
1
H2O
40
40
Total Volume
50
50
Step
Temperature °C
Time
1
95
3 min
2
95
30 sec
3
55
30 sec
4
72
1 min
5
Go To 2
30x
6
72
5 min
7
4
Infinite
The gel shows that the PCR fragments were amplified successfully in the correct size (Size HagI=639 bp  HagII= 297 bp), The next step is a fusion PCR fusing the HagI/II constructs with homologous flanks for integration into the hag locus in B. subtilis genome.
09.04.2014
13.15 Miniprep
Miniprep of transformed E.coli DH5α was performed according to the miniprep kit protocol (Omega) with overnight preculture.
13.16/17 Restriction Digest with NcoI and Test-Plasmids
Restriction digest with NcoI and test-plasmids from the miniprep.
[µL]
Mastermix with NcoI - for 14 attempts
Attempt - NcoI (20 kU/mL) - 12 attempts
DNA
-
ca. 350 ng/µL → ca. 1 µg → 2,9 µL
NcoI
3,5 (0,25/attempt)
 
Master-Mix
-
12,1
Buffer 10x Tango
21
 
H2O
144,9
 
Total Volume
169,4
 
Loading Dye 6x
-
2,5 /attempt
Incubation for 60 min at 37 °C. Then a heat shock was induced for 2 min at 45 °C.
Results
Expectations: 4 bands (167 bp - 1033 bp - 1056 bp - 4410 bp)
13.18 Restriction Digest of positive Clones with NdeI
[µL]
Attempt 2.1
Attempt 2.3
Attempt 2.4
Attempt 4.2
DNA
628,8 ng/µL → ca. 1 µg → 1,6 µL
477,2 ng/µL → ca. 1 µg → 2,1 µL
694,9 ng/µL → ca. 1 µg → 1,4 µL
406,8 ng/µL → ca. 1 µg → 2,46 µL
NdeI
0,5
0,5
0,5
0,5
Cutsmart-Buffer 10x
1,5
1,5
1,5
1,5
H2O
11,4
10,9
11,6
10,5
Total Volume
15
15
15
15
Loading Dye 6x
2,5
2,5
2,5
2,5
Results
Expected bands: 464 bp - 2148 bp - 4054 bp
The expected bands were visible.
13.19 Restriction Digest of positive Clones with NcoI overnight
[µL]
Attempt 2.1
Attempt 2.3
Attempt 2.4
Attempt 4.2
DNA
628,8 ng/µL → ca. 5 µg → 8 µL
477,2 ng/µL → ca. 5 µg → 10,5 µL
694,9 ng/µL → ca. 5 µg → 7,2 µL
406,8 ng/µL → ca. 5 µg → 12,3 µL
NcoI
0,2
0,2
0,2
0,2
Tango Buffer 10x
1,5
1,5
1,5
1,5
H2O
5,3
10,9
11,6
10,5
Total Volume
15
15
15
15
14.4 Overnight culture
Aim: Amplify Plasmid in an overnight culture
Six clones of the transformed Top10 cells were picked and used to inoculate overnight cultures (6 x 5 mL).
10.04.2014
15.4 PCR Hag-Flanks
Dilution of primers
Hag I: 601,7 ng/µL
Hag II: 548,2 ng/µL
Flank I: 226 ng/µL
Flank II: 221 ng/µL
Hag I dilution: 35 ng/µL
Hag I: 601,7 ng/µL * X = 10*35 ng/µL → 0,58:9,42
Hag II: 548,2 ng/µL * X = 10*35 ng/µL → 0,64:9,36
Flank I: 226 ng/µL * X = 10*35 ng/µL → 1,55:8,45
Flank II: 221 ng/µL * X = 10*35 ng/µL → 1,58:8,42
PCR Hag-Flanks Mix
Mix
Template Mix [µL]
Flank-Hag construct [µL]
Hag I(35 ng/µL)
2
-
Hag II(35 ng/µL)
2
-
Flank I(35 ng/µL)
2
-
Flank II(35 ng/µL)
2
-
Template Mix
-
3
Primer pMAD-Flank I fwd
-
1
Primer pMAD-Flank I rev
-
1
Q5-Mastermix 2x
-
25
H2O
-
20
Total Volume
8
50
Step
Temperature °C
Time
1
95
3 min
2
95
30 sec
3
55
30 sec
4
72
2,5 min
5
Go To 2
35x
6
72
5 min
7
4
Infinite
Results
Expected bands: 2 kb band (whole Flank-Hag fragment)
The gel shows the successful amplification of the Hag-flank construct.
15.6 Gel Extraction
Gel extraction of 2,2 kb band:
27,6 ng/µL
13.20 Yeast Transformation with positive NcoI Digests and Mutagenesis Primer (iGEM 009-0011)
Attempts with 2 positive clones (2.1 NcoI cut & 2.3 NcoI cut)
Mix for trafo [µL]
p2.1 control
p2.3 control
p2.1
+oligos
+amyE
p2.3
+oligos
+amyE
p2.1
Anneal+oligo
+amyE
p2.3
Anneal+oligos
+amyE
p2.1
1
-
1
-
2
-
p2.3
-
1
-
1
-
2
iGEM-009
-
-
1
1
1
1
iGEM-010
-
-
1
1
1
1
iGEM-011
-
-
1
1
1
1
amyE-fragment
-
-
1
1
1
1
H2O
-
-
-
-
14
14
PEG 3350
(50% w/v)
260
260
260
260
260
260
LiAc 1 M
36
36
36
36
36
36
Salmonsperm
(2 mg/mL)
50
50
50
50
50
50
H2O
13
13
9
9
-
-
Total Volume
360
360
360
360
-
-
13.21 Yeast Transformation with positive Clones
Mix [µL]
2.1 control
2.3 control
3'' - P* + pool
4'' - P** (DS) + pool
PEG 3350 (50% w/v)
260
260
260
260
LiAc 1 M
36
36
36
36
Salmonsperm (2 mg/mL)
50
50
50
50
H2O
14
10
2
2
Fragm. (GFP, Cat, AmyE)
-
4
-
-
Pool (all 3 fragments)
-
-
12
12
Total Volume
360
360
360
360
Salmonsperm for 10 min on 95 °C, then on ice - all attempts on ice!
*pMA12 - digestion with HindIII + EcoRI
**pMA12 - parallel digestion of HindIII and EcoRI
13.22 Yeast Transformation
Resuspend yeast pellet in attempts
Heat shock at 42 °C for 35 min
Centrifugation for 30 sec - 1 min at 13.000 g
Resuspend pellet in 200 µL sterile water
Plate yeast on selective medium
Incubate at 30 °C till monday
14.5 Preparation of plasmids
Aim: Purify the amplified plasmids from overnight cultures
The plasmids were purified from the overnight cultures using Sigma GenElute Plasmid Miniprep Kit following the included protocol. The plasmids were eluted using 45 µL water.
14.6 Transformation of BL21 cells
Aim: Transform E. coli BL21(DE3) with PheA-Arc1p-C-8x-pET28a
40 µL E. coli BL21(DE3) electrocompetent cells were transformed with 1 ng of the previously purified plasmid PheA-Arc1p-C-8x-pET28a from two different clones, incubated for 1 h at 37 °C and plated out on LB-Kan50 plates that were incubated overnight at 37 °C.
11.04.2014
15.7 Test PCR with PCR created Fragment as Template Flagellin Constructs
Mix [µL]
Test PCR
Template (Hag-Flank construct)
2
Primer pMAD-Flank I fwd
1
Primer pMAD-Flank II rev
1
Q5-Mastermix 2x
5
H2O
1
Total Volume
10
Step
Temperature °C
Time
1
95
3 min
2
95
30 sec
3
55
30 sec
4
72
2,5 min
5
Go To 2
35x
6
72
5 min
7
4
Infinite
The extracted 2000 bp fragment was used as a PCR template for amplification of the Hag-flank construct for further cloning steps.
15.8 Digestion of Flagellin Fragment with NcoI/BamHI
Aim: Creating restriction sites for cloning into pMAD-vector.
Mix [µL]
Hag-Flank-Fragment digest NcoI/BamHI
DNA
9 ng/µL → 25 µL = 225 ng
27,6 ng/µL → 15 µL = 414 ng → 639 ng in 40 µL
Enzyme NcoI-HF
0,5
Enzyme BamHI-HF
0,5
Cutsmart Buffer (10x)
5
H2O
4
Total Volume
50
12.04.2014/13.04.2014
13.22 Miniprep of Yeast Culture Transformation
The plasmid isolation was carried out according to the miniprep kit protocol.
13.23 Transformation of E.coli DH5α with prepared plasmids
Plasmids prepared above have been transformed into E.coli.
Colonies on transformation plates have been inoculated in shaking culture (16 colonies).
14.04.2014
13.24 Miniprep of Shaking Cultures
Miniprep according to the miniprep kit protocol (Omega).
2.1 and 2.3 different positive colonies from     09.04.2014.
13.25 Digestion
Aim: Check for elimination of NcoI-restriction sites.
Mix
[µl]
DNA
1
Enzyme NcoI-HF
0,75
Cutsmart Buffer 10x
1,5
H2O
12,25
Total Volume
15
Results
One band was expected for a linearized plasmid, 4 bands occurred, the restriction sites have presumably not been removed. Expected sizes for plasmid with 4 NcoI-restriction sites: ~ 4400 bp, 2 x ~ 1000 bp, ~ 200 bp
13.26 Overnight Digestion
Aim: Check for elimination of NcoI-restriction sites
Overnight digestion of two samples with the highest DNA-concentration (2.1 2 and 3.1 A1
Mix [µL]
2.1 2
3.1 A1
DNA
10 µg/8 µL
10 µg/9 µL
Enzyme NcoI-HF
1
1
Cutsmart Buffer 10x
2
2
H2O
9
8
Total Volume
20
20
14.7 Overnight culture
Aim: Inoculate Overnight culture of PheA-Arc1p-C-8x-pET28a
5 mL of LB-Kan50 medium were inoculated with a clone from 14.6 bearing the PheA-Arc1p-C-8x-pET28a plasmid.
15.04.2014
13.27 Mutagenesis PCR
Aim: Mutate the NcoI-restriction sites, because earlier attempts were not successful
As template the restriction of the sample 3.1 A1 has been used. The mutagenesis primers have been chosen together with fitting reverse primers to create fragments without the NcoI-restriction sites. These fragments should further be transformed into yeast and assembled into the whole plasmid without the restriction sites for NcoI.
Mix [µL]
Fragment 1
Fragment 2
Fragment 3
Restriction mix 3.1 A1 (2,08 ng/µL)
1
1
1
iGEM-009 (10 mM)
1
-
-
Ag TWp7 (10 mM)
1
-
-
iGEM-010 (10 mM)
-
1
-
iGEM-003 (10 mM)
-
1
-
iGEM-011 (10 mM)
-
-
1
iGEM-006 (10 mM)
-
-
1
Q5-Mastermix 2x
25
25
25
H2O
22
22
22
Total Volume
50
50
50
Every PCR has been carried out in double and was further analyzed on a 1% gel.
Results
Mutagenesis PCR for different fragments;
1: 2-log-DNA-ladder,
2 & 3: fragment 1,
4 & 5: fragment 2,
6 & 7: fragment 3.
Lane 1 & 2 contain the expected Fragments (~ 600  bp). In Lane 3 there is a very thin fragment with the expected size (~ 1100 bp)  but smaller fragments can also be seen. Lane 5 does not contain any fragment at  all and in lane 6 there is the fragment of the expected size (~ 600 bp).      Both samples with fragment 1 were pooled, sample 3.1  was discarded and the whole PCR-product 2.1 has been separated in an agarosegel  and the fragment with 1100 bp has been cut out and purified with the Gel  Extraction Kit (c = 4,4 ng/µL). A second PCR for fragment 2 has been done in parallel.  All parameters remained the same but the annealing-temperature has been changed  to 62 °C.
Modified mutagenesis PCR for fragment 2; again there are two fragments with a too small size and the expected fragments are not high concentrated.
14.8 Test Expression
Aim: Test the expression of PheA-Arc1p-C-8x
Using the preculture from 14.7 60 mL of LB-Kan50 medium were inoculated with 600 µL of the preculture and incubated at 37 °C and 220 rpm until OD600 reached 0.5. The culture was cooled down to 18 °C and the protein production was induced with 12 µL of 500 mM IPTG. Over night the culture was incubated at 18 °C and 220 rpm. A glycerolstock of the strain containing the PheA-Arc1p-C-8x-pET28a plasmid was stored at -80 °C.
16.04.2014
13.27 Cloning of Promotor
To be sure to obtain just a fragment of the wanted size a PCR with the same Primers (iGEM-003 and iGEM-010) has been carried out with the thin fragment which was cut out of the gel yesterday.
PCR of the fragment 2;
1: 2-log-DNA-ladder,
2 & 3: fragment 2
Besides the expected fragment there are also other fragments which indicate that the primers partially bind unspecific. The rest of the PCR products has been separated with an agarose gel and the wanted fragment has been cut out and purified with the Gel Extraction Kit (c = 21 ng/µL).
13.28 Yeast Transformation with PCR fragments
Aim: Rebuild the plasmid via the yeast recombination system
The three obtained fragments together with the NcoI-digested plasmid pMA12 have been transformed into S. cerevisiae to get the whole plasmid again. The whole procedure has been done according to the protocol.
Mix for trafo [µL]
Volumes
NcoI-restriced pMA12 (2.1 1)
1
Fragment 1 (146 ng/µL)
1
Fragment 2 (21,7 ng/µL)
9
Fragment 3 (78,6 ng/µL)
3
PEG 3350(50% w/v)
260
LiAc 1 M
36
Salmonsperm (2 mg/mL)
50
H2O
-
Total Volume
360
The transformed yeast cells are incubated at RT over the easter holidays.
14.9 Purification of test expression
Aim: Purify PheA-Arc1p-C-8x on a small scale
The cell culture from 14.8 was centrifuged (17000 rpm, 20 min, 4 °C) and the pellet resuspended in 3 mL buffer A. Lysozyme was added to a final concentration of 1 mg/mL and incubated on ice for 30 min. Afterwards the cells were lysed using ultra sound (2x 18 pulses, 30 s on ice inbetween). To clear the suspension of cell debris it was centrifuged (13000 rpm, 15 min, 4 °C). The supernatant was purified using QIAGEN Ni-NTA Spin Columns. An SDS-PAGE gel showed that the protein is produced and can be purified using Ni affinity chromatography.
60 mL LB-Kan50 medium was inoculated from the glycerolstock prepared in 14.8 and incubated at 37 °C and 220 rpm over night.
17.04.2014
14.10 Expression of PheA-Arc1p-C-8x
Aim: Express PheA-Arc1p-C-8x for further use
10 x 500 mL LB-Kan50 medium were inoculated with 5 mL of the overnight culture prepared in 14.9 and incubated at 37 °C and 220 rpm until OD600 reached 0.5. The flasks were cooled to 18 °C and protein production was induced by adding 100 µL 500 mM IPTG. The cultures were incubated over night at 18 °C and 220 rpm. Afterwards the cells were collected by centrifugation (6000 rpm, 20 min, 4 °C), resuspended in buffer A and stored at -20 °C until further use.
22.04.2014
13.29 Miniprep of Transformed Yeast S.cerevisiae and Trafo of E.coli DH5Alpha
Aim: Increase the amount of plasmid DNA to perform restriction and further cloning steps.
Miniprep of Transformed Yeast S.cerevisiae from 21.04.2014 and Trafo of E.coli DH5Alpha.
Miniprep was performed according to the miniprep protocol (Omega).
Plasmid pMa12 with 3 inserts (cat, amyE and gfp) with removed NcoI restriction sites.
Concentration after plasmid prep= 7.5 ng/µL
Transformation of E.coli DH5α with 5 µL DNA (37.5 ng)
Over night incubation on LB-Amp plates.
14.11 Purification of PheA-Arc1p-C-8x
Aim: Purify PheA-Arc1p-C-8x in three steps
The resuspended cells from 14.10 were lysed using a french press. The lysate was incubated on ice with DNAseA and RNAseI for 10 min and centrifuged (27000 rpm, 45 min, 4 °C). The supernatant was filtered (0.45 µm) and PheA-Arc1p-C-8x purified from the cleared solution by NiNTA affinity chromatography. The combined elution fractions containing crude PheA-Arc1p-C were concentrated to a volume of 2 mL and further purified using an anion exchange column to get rid of possible tRNA contaminations. Finally the buffer was changed to dialysis buffer and the protein concentrated and stored at -80 °C in aliquods for further use.
23.04.2014
13.30 Transformation of pMA12-construct into E.coli DH5α
The transformation of the E.coli cells was not successful. So it has been repeated with the rest of the prepared plasmid.
15.9 PCR-Amplification of Flagellin constructs
Aim: PCR to amplify the product for ligation into pMAD
The concentration of the Hag-Flank-construct was too low (c = 5.8 ng/µL) to use it for the ligation into pMAD. A PCR has been run to increase the amount of the construct.
Mix [µL]
Test PCR
Template (Hag-Flank construct)
2 (12 ng)
Primer 89 from Florian
1
Primer 90 from Florian
1
Q5-Mastermix 2x
25
H2O
21
Total Volume
50
PCR Q5 elongation for 2 kb fragments
Step
Temperature °C
Time
1
95
3 min
2
95
30 sec
3
55
30 sec
4
72
2,5 min
5
Go To 2
35x
6
72
5 min
7
4
Infinite
The PCR-product has been checked on an agarose gel (3 µL + 1 µL 6x Loading-Dye) Expected band: 2000 bp is there, but also 2 additional bands A band with the expected size of ca 2000 bp is visible but there are also two additional DNA-fragments with a lower size. The rest of the PCR-product (47 µL) has been separated on a gel and the expected DNA-fragment has been excised and purified with the Gel Extraction Kit. The purification led to a concentration of 45 ng/µL.
Results
Figure: PCR of the Hag-Flank-construct
14.12 Creating PheA-Arc1p-C-4x
Aim: Using Round-the-horn mutagenesis to create PheA-Arc1p-C-4x-pET28a
The PCR with 5' phosphorylated TG_2xGSSG FP und TG_2xGSSG RP should generate the 7545 bp fragment for ligation from the methylated template PheA-Arc1p-C-8x-pET28a from 14.5.
The used Primer concentration was 1 µM, the concentration of the template was 1 ng/µL and we used a hot start to avoid unspecific annealing of primers.
Content
Volume [µL]
Water
29.5
5x GC Buffer
10
DMSO
2.5
TG_2xGSSG FP
2.5
TG_2xGSSG RP
2.5
PheA-Arc1p-C-8x-pET28a
1
dNTPs
1
Phusion-Polymerase
1
Total Volume
50
Step
Temperature [°C]
Time [min:sec]
1
98
2:00
2
add Polymerase
 
3
98
0:10
4
67.5
0:30
5
72
4:00
6
go to 3
32x
7
72
10:00
8
4
hold
The amplified fragment was purified on an agarose gel on which it showed the correct size. The extracted linear template was then digested to degrade the original unmutated plasmid using DpnI.
Content
Volume [µL]
10x CutSmart Buffer
3.6
Plasmid
30
DpnI
3
Total Volume
36.6
The reaction mixture was incubated at 37 °C and 300 rpm for 1 h and afterwards heated to 80 °C for 20 min
In order to create the circular plasmid 100 ng of the linear template were mixed with 1 µL 10x T4-Ligasebuffer,  1 µL T4-Ligase and water to give a total volume of 10 µL. This mixture was incubated for 9 h at 16 °C. The resulting plasmid was dialysed with water to decrease the salt concentration for the following transformation of E. coli Top10 electrocompetent cells with the plasmid.
24.04.2014
13.31 Inoculation of Miniprep of Transformed DH5α
Aim: Increase the amount of plasmid DNA to perform restriction and further cloning steps.
Seven small colonies grew on the plate. The plate was further incubated  until the afternoon. Each 6 mL LB-Amp have been inoculated with one of the  colonies and incubated over night at 37°C.
15.10 Digestion with NcoI/BamHI of PCR-amplified Flagellin construct
Aim: Create restriction sites for ligation the fragment into pMad.
The hag-flank-construct which was amplified yesterday has been digested with NcoI and BamHI to create sticky ends for the following ligation.
Mix [µL]
Hag-Flank-Fragment digest NcoI/BamHI
DNA
1350 ng in 30 µL
Enzyme NcoI-HF
1
Enzyme BamHI-HF
1
Cutsmart-Buffer 10x
5
H2O
13
Total Volume
50
The restriction was carried out at 37°C for 45 min and the digested fragment was purified with the Gel Extraction Kit (c = 24 ng/µL).
15.11 Ligation of pMAD and Flagellin Construct Digestion
Aim: Ligation of the flagellin-fragment with pMAD-vector
(20 ng of vector x kb size of insert/kb size of vector) x molar of ratio of (insert/vector)
0.6 µL of the pMAD-vector (30 ng) and 1 µL of the digested hag-flank-construct (18 ng) were used for the ligation.
Mix [µL]
Control atempt
2x attempt
Plasmid (pMAD)
0,6
0,6
Insert (construct)
-
1
T4 Ligase
1
1
T4 Ligase Buffer (10x)
2
2
H2O
16,4
15,4
Total Volume
20
20
The ligation-mix was incubated at roomtemperature for 1 h followed by a transformation of E. coli DH5α with 10 µL of the ligation-mix. The transformation was incubated on a LB-Amp-plate at 37 °C over night.
14.13 Overnight culture
Aim: Amplify Plasmid in an overnight culture
Six clones of the transformed Top10 cells from 14.12 were picked and used to inoculate overnight cultures (6 x 5 mL).
25.04.2014
13.32 Miniprep of Transformed DH5&alpha
The seven cultures were used to do a miniprep. The miniprep was performed according to the miniprep protocol (Omega).
Clone
Concentration (ng/µL)
1
420
2
298,9
3
330
4
392
5
435,3
6
560
7
485
13.33 Restriction Digestion with NcoI
Aim: Check for elimination of NcoI-restriction sites.
[µL]
Attempt 1-7
Mastermix
Plasmid
1,5
-
Enzyme NcoI
-
1
Tango Buffer (10x)
2
16
H2O
15,5
124
Total Volume
20
1
Results
The digest was further analyzed on a 1% agarose gel.
Restriction digest of the prepared pMa12-construct with NcoI; the star means in this lane was the pMa12-construct (before the installation of the mutated-fragments) digested with NcoI as a control.
The clones 2, 3, 4, 5 & 7 appeared to contain the plasmid with only one NcoI-restriction side. This clones were transferred into 6 mL LB-Amp and incubated at 37 &deg,C over night.
15.13 New Ligation of pMAD and Flagellin construct digest
Aim: Ligation of the flagellin-fragment with pMAD-vector.
The plates from 15.12 were empty which indicates an unsuccessful ligation. A new ligation was performed after calculation with a Ligation calculator: UT Dallas
Mix [µL]
Control attempt
2x attempt
Plasmid (pMAD)
1
1
Insert (construct)
-
1,9
T4 Ligase
1
1
T4 Ligase Buffer (10x)
2
2
H2O
16
14,4
Total Volume
20
20
The ligation was incubated at roomtemperature for 1.5 h. Afterwards it was stored at  -20 °C.
14.14 Preparation of plasmids
Aim: Purify the amplified plasmids from overnight cultures
The plasmids were purified from the overnight cultures created in 14.13 using Sigma GenElute Plasmid Miniprep Kit following the included protocol. The plasmids were eluted using 45 µL water.
28.04.2014
13.35 PCR with Primers iGEM-002 und iGEM-010
Aim: Create new mutation fragment for eliminating a NcoI-restriction site.
The plasmids prepared on Saturday seem to be smaller as expected. For this reason a new attempt to eliminate the restriction sites was carried out. The amplification of fragment 2 (with primers iGEM 003 and iGEM 010 on 15.04.2014) did not happen without any problems such as unspecific smaller bands in addition to the expected one (ca 1000 bp). So this time primer iGEM 003 was replaced by iGEM 002 which would result in a smaller fragment of approximately 600 bp. Later on the fragments 1 and 3 together with the new created one, the digested pMa12 construct and the amyE-fragment (to overlap the region between fragments) would be transformed into S. cerevisiae for recombination.
Mix [µL]
New Fragment 2
Restriction mix 3.1 A1 (2,08 ng/µL)
1
iGEM-010 (10 mM)
1
iGEM-002 (10 mM)
1
Q5-Mastermix 2x
25
H20
22
Total Volume
50
The PCR was performed according to the PCR protocol and the PCR-products were analyzed on a 1% agarose gel.
Step
Temperature °C
Time
1
95
3 min
2
95
30 sec
3
58
20 sec
4
72
1 min
5
Go To 2
35x
6
72
5 min
7
4
Infinite
Results
PCR with the primers iGEM-002 and iGEM-010 and the digested pMa12-vector The expected fragments with the sizes of ca 600 bp are visible.
13.35 Transformation of Yeast with the different Fragments
Aim: Recombination of the fragment to a whole plasmid in yeast.
Mix [µL]
Volumes
NcoI-digested pMA12 (2.1 1)
2
Fragment 1 (146 ng/µL)
2
New Fragment 2 (412,9 ng/µL)
2
Fragment 3 (78,6 ng/µL)
3
Fragment amyE (48,3 ng/µL)
5
PEG 3350(50% w/v)
260
LiAc 1 M
36
Salmonsperm(2mg/mL)
50
H2O
-
Total Volume
360
The transformed yeast cells were incubated at 30 °C.
14.15 Transformation of BL21 cells
Aim: Transform E. coli BL21(DE3) with PheA-Arc1p-C-4x-pET28a
40 µL E. coli BL21(DE3) electrocompetent cells were transformed with 1 ng of the previously purified plasmid PheA-Arc1p-C-4x-pET28a from two different clones, incubated for 1 h at 37 °C and plated out on LB-Kan50 plates that were incubated overnight at 37 °C.
29.04.2014
17.1 Making new competent E.coli DH5α
Aim: Making competent E.coli DH5α cells.
Some new competent E. coli DH5α cells were made because the last batch of cells have not yielded in good results in the last few transformations. The cells were made according to the protocol in the method section below. To test the cells they were plated on normal LB, LB-Amp, LB-Kan and a test-trafo was carried out with pMa12 which provides an ampicillin resistance.
15.14 New PCR-Amplification of different Attempts for the Hag-Flank-Construct
Aim: Amplification of construct after unsuccessful transformation for new digests.
The transformation of E. coli DH5α with the ligation of the pMAD-vector and the hag-flank-construct was not successful. Therefore the PCR for amplification of the construct was repeated by using the hag / flank-parts or the construct itself as template.
Mix [µL]
PCR 1
PCR 2
PCR 3
Template
2 (Template mix Hag I/II + Flanks 15,5)
1 (Flagellin-construct 15,6)
1 (Flagellin-construct digest (15,10)
Primer 89 from Florian
1
1
1
Primer 90 from Florian
1
1
1
Q5-Mastermix 2x
25
25
25
H2O
21
22
22
Total Volume
50
 
 
PCR was performed according to the "PCR Q5 elongation for 2kb fragments" protocol and the PCR-products were analyzed on a 1% agarose gel.
Step
Temperature °C
Time
1
95
3 min
2
95
30 sec
3
55
30 sec
4
72
2,5 min
5
Go To 2
35x
6
72
5 min
7
4
Infinite
Results
PCR with primers 89 and 90 from Florian and  different templates; lane 1: 2-log-ladder, lane 2: PCR1, lane 3: PCR2, lane 4:  PCR3
Although the marker is  barely visible the expected bands can detected at a relative height of 2000 in  addition to some other unspecific fragments.
14.16 Overnight culture
Aim: Inoculate Overnight culture of PheA-Arc1p-C-4x-pET28a
5 mL of LB-Kan50 medium were inoculated with a clone from 14.15 bearing the PheA-Arc1p-C-4x-pET28a plasmid.
30.04.2014
15.15 Restriction of the Hag-Flank-Construct and the pMAD-Vector
Aim: Create the fitting sticky ends for a ligation.
Since the last ligation attempts were not successful the plasmid pMAD is digested by us instead using the digested pMAD from Florian. The PCR-fragments of the hag-flank-construct were purified (PCR1: c = 62 ng/µL, PCR2: c = 55 ng/µL, PCR3: c = 46 ng/µL) and digested with the enzymes BamHI and NcoI.
[µL]
PCR 1
PCR 2
PCR 3
PCR 4
DNA
all
all
all
5 (300 ng/µL)
BamHI
1
1
1
1
NcoI
1
1
1
1
Cutsmart Buffer
4
4
4
2
H2O
4
4
4
2
Total Volume
40
40
40
20
The restriction was  incubated at 37 °C for 1 h and stored at -20 °C.
14.17 Test Expression
Aim: Test the expression of PheA-Arc1p-C-4x
Using the preculture from 14.16 60 mL of LB-Kan50 medium were inoculated with 600 µL of the preculture and incubated at 37 °C and 220 rpm until OD600 reached 0.5. The culture was cooled down to 18 °C and the protein production was induced with 12 µL of 500 mM IPTG. Over night the culture was incubated at 18 °C and 220 rpm. A glycerolstock of the strain containing the PheA-Arc1p-C-4x-pET28a plasmid was stored at -80 °C.
Retrieved from " http://2014.igem.org/Team:Marburg:Project:Notebook:April "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
