"
Page
Discussion
View source
History
teams
Log in
 
Team:Marburg:Project:Notebook:May
From 2014.igem.org
Home
Project
Safety
Team
Policy & Practices
Attributions
SilverSURF
CancerSURF
RiboSURF
Notebook
Parts
Notebook: May
March
April
May
01.05.2014
02.05.2014
05.05.2014
06.05.2014
07.05.2014
08.05.2014
09.05.2014
10.05.2014
11.05.2014
12.05.2014
13.05.2014
14.05.2014
15.05.2014
16.05.2014
19.05.2014
20.05.2014
21.05.2014
22.05.2014
23.05.2014
26.05.2014
27.05.2014
28.05.2014
29.05.2014
30.05.2014
June
July
August
September
October
Methods
01.05.2014
15.16 Purification of Restriction Digest from 30.04.14
Purifying the samples from the 30.04.2014 with gel extraction kit (QIAquick Gel Extraction Kit) according to the protocol.
Fragment
Concentration [ng/µl]
pMAD Digest 1
21
pMAD Digest 2
19
Hag-Flank Construct 1 Digest
16,9
Hag-Flank Construct 2 Digest
17,6
Hag-Flank Construct 3 Digest
11,3
15.17 Ligation of purified pMAD and Constructs
The ligation is carried out with the purified pMAD digest and the hag-flank-construct digest from 15.16 in order to transform the ligation mix into E.Coli DH5α. For the ligation was the following calculator used: UT Dallas
Mix [µl]
Control 1
Control 2
Construct 1.1
Construct 1.2
Construct 2.1
Construct 2.2
Construct 3.1
Construct 3.2
Plasmid (pMAD digest 1)
1
-
4,5
-
4,7
-
3
-
Plasmid (pMAD digest 2)
-
1
-
5
-
5
-
3,3
Insert (construct)
-
-
5
5
5
4,8
5
5
T Ligase
1
1
1
1
1
1
1
1
T4 Ligase Buffer 10x
2
2
2
2
2
2
2
2
H2O
16
16
7,5
7
7,3
7,2
9
8,7
Total Volume
20
20
20
20
20
20
20
20
Incubation for 1,5 h at room temperature.
15.18 Transformation of E.coli DH5α
Transformation of E.coli DH5α with the ligation attempt.
10µl of each the attempts were used to transform 50 µl of competent E. coli DH5α in order to test the success of the ligation and raising colonies for inoculating a mini prep.
15.19 PCR with primers
To have enough of the construct for further works, a new PCR (as described  before) was done overnight.
02.05.2014
13.36 Miniprep of pMA12 from Yeast
Aim: Isolate the plasmid from yeast.
The yeast was washed from the plate with 2 ml of A. dest. After centrifugation at 11000 rpm for 1 min the supernatant was discarded and 500 µl of Solution 1 of the Omega plasmid prep kit was added. The yeast was vortexed for 5 min after addition of a few glas balls. After that the miniprep was carried out according to the protocol (Omega). (c = 134.5 ng/µl)
13.377 Transformation of E.coli DH5α with the isolated pMA12
Aim: Transform E.coli with the isolated plasmid to increase the amount of plasmid.
E. coli DH5Alpha cells were transformed with 10µl of the isolated plasmid pMa12. They were incubated at room temperature over the weekend.
15.20 Purification of Fragments
Purification of the fragments from 01.05.14. The PCR-products were separated on a 1% agarose-gel.
Results
The fragment with the correct size was  excised from the gel and purified with the Gel Extraction Kit (QIAquick Gel Extraction Kit)(c = 6.6 ng/µl; c2  = 5.8 ng/µl).
05.05.2014
13.38 Inoculation of Miniprep Cultures
Aim: Pick clones to check for the correct plasmids.
15 colonies of the transformed E.coli DH5α were picked and incubated in 6 ml LB-Amp at 37°C overnight.
15.21 Ligation of the hag/flank and pET24d both cut with BamHI and NcoI followed by Transformation of E. coli
The ligation with pMAD was not successful so it was decided to clone the hag/flank-construct first into pET24d. The ligation was done after using the ligation calculator: UT Dallas
[µl]
Fragment 1
Fragment 2
Fragment 3
pET24d
2,9
3,1
2
Fragment
10
10
10
T4-Ligase-Buffer (10x)
2
2
2
T4 Ligase
1
1
1
H20
4,1
3,9
5
Total
20
20
20
The ligation was incubated at RT for about 2 hours followed by a transformation of E. coli DH5α. The transformed cells were plated out on LB-Kan and incubated at 37°C over night.
06.05.2014
13.39 Miniprep of the 15 Cultures and Restriction with NcoI
Aim: isolate the plasmid from the picked clones and check for restriction sites of NcoI.
The plasmids were isolated according to the protocol "Miniprep (Plasmid DNA Miniprep Kit II, Omega)".
Clone
Concentration (ng/µl)
1
570,2
2
735,5
3
671,9
4
642,6
5
660,9
6
688
7
982,6
8
637,4
9
635,4
10
1306,6
11
523,9
12
554,4
13
501,3
14
825
15
771,2
The isolated plasmids were digested with NcoI for 1h 20 min and further analyzed on a 1% agarose gel.
Results
Restriction of the isolated pMa12 with NcoI; the lanes contain the digested pMa12 in the order of the picked clones 1 - 15; clone 9 was negative; the lanes between the restrictions and the marker contain the isolated pMAD but it appears to be empty.
The most clones contained a plasmid that has only one NcoI restriction site. The expected size of the pMa12 should be around 6.6 kb. All fragments are of the expected size.
13.40 Restriction of pMa12 (isolated from Clone 10) with SacI and NcoI
Aim: create fitting ends to anneal the promotor oligos.
The isolated pMa12 of clone 10 was used for a restriction with SacI-HF and NcoI. The restriction leads to fitting ends for the designed oligos which are the promoters we are about to test: PargJ and PykuO. The restriction is incubated at RT over night.
Restriction
Volume [µl]
pMA12 from clone 10
2
NcoI
1
SacI-HF
1
CutSmart Buffer (10x)
2
H2O
14
Total
20
15.21 Inoculation of the DH5α and Miniprep of pET24d_hag/flank
Aim: amplifying and isolation of pET24d_hag/flank.
On the plates with the transformands of yesterday were only three colonies. Each 6 ml LB-Kan were inoculated with the colonies and incubated at 27°C for around 9 h. After growth of the bacteria the plasmids were isolated with the Omega Plasmid Prep Kit.
07.05.2014
15.22 Restriction of 3 isolated plasmids (6.5.14) with NcoI and BamHI
Aim: Test whether the isolated plasmid contains the 2 kb insert (hag-flank construct).
[µl]
Clone 1
Clone 2
Clone 3
Plasmid (500ng)
10
5
5
Water
8
12
12
CutSmart Buffer (10x)
2
2
3
NcoI
0,5
0,5
0,5
BamHI
0,5
0,5
0,5
Restriction for 45 min at 37°C.
Agarose-gel (1%) to compare the cut and uncut plasmid.
Agarose gel with control (uncut) and 3 cut clones of pEt24d containing the hag-flank construct, fragments are of expected size.
13.41 Purification of overnight digested plasmid pMa12 (3 inserts, only 1 NcoI restriction site) by gel-elution.
Aim: purify plasmid to clone annealed oligos for the promoter sequences into the plasmid.
Concentration: 3ng/µl
13.42 Annealing of Oligos
Aim: create promotor sequences in plasmid piGEM001
[µl]
Reaction 1
Reaction 2
Oligos
iGEM012/iGEM013
10µl/10µl
iGEM014/iGEM015
10µl/10µl
Water
80
80
Total
100
100
Heat up at 98°C for a few minutes. Let it cool down until room temperature
13.43 Ligation of the annealed oligos into digested piGEM001
Aim: Create plasmid with the corresponding promoter sequences for Ag/Cu
[µl]
Reaction (Ag)
Reaction (Cu)
Plasmid (3ng/µl)
5,5
5,5
Water
15
15
Buffer (Ligation Buffer 10x)
2,5
2,5
Oligo-Mix
12/13
14/15
T4-DNA-Ligase
1
1
10µl of the ligation reaction will be transformed into chemically competent E.coli DH5α as described before.
13.44 Over night restriction of pMA12 to obtain more digested plasmid (clone 10)
See 06.05.2014
Restriction
Volume [µl]
pMA12 from clone 10
2
NcoI
1
SacI-HF
1
CutSmart Buffer (10x)
2
H2O
14
Total
20
08.05.2014
13.43 Repeat Transformation for AG/Cu Promoters
No colonies on the plates, repeat ligation and transformation with new digested plasmid.
Concentration plasmid: 17 ng/µl
Ligation: 1.5µl of plasmid used
Tranformation: as usual
1. Sequencing
Label
Description
Primer
AFD001881
piGEM001
T7
AFD001882
piGEM001
T7 Turn
AFD001883
piGEM001
Hag revers (Flo)
AFD001884
piGEM002
TW 260 (Daniel)
Plasmids have to be prepared in a concentration between 50-100 ng/µl in a total volume of 10 to 15µl.
2µl Primers have been added to the premixes, Sample 881 and 882 will be sequenced with primers already present at the company (mwg eurofins).
09.05.2014
13.43 Repeat Transformation for AG/Cu Promotors
No colonies on the plates
Repeat annealing of oligos
Step 1:
Amount [µl]
 
1
Oligo 1
1
Oligo 2
5
Ligase Buffer
1
PNK
42
Water
Incubation at 37°C for 30 min
Step 2:
Heat up water in microwave until boiling, add the tubes with primer mix
let it cool down to room temperature
Step 3:
1:100 dilution
Step 4: Ligation
Amount [µl]
 
3
Plasmid (17,6ng/µl)
2
Annealed Oligos (1:100)
1
Ligase
2
Ligase Buffer
12
Water
Step 5: Transformation
Incubation at room temperature over two days.
Step 6: Results
Hag Flank construct digestion with SpeI was positive.
10.05.2014
13.43 Repeated Transformation for AG/Cu Promoters
No colonies
15.23 PCR for amplification of flagellin-constructs out of pet24-fla-Construct from 05.05.14
Aim: Amplification of the flagellin-constructs from the pET24-fla-plasmid as a template was done in order to make a restriction digest with SpeI. This showed, which clone contained the successfully synthesized construct with SpeI restriction site because of the Strep-Tag from sequencing.
Mix [µl]
Pet24fla1
Pet24fla2
Pet24fla3
Temp. - pet24fla cone 1 (lig1 05.05.14)
0,5
-
-
Temp. - pet24fla cone 2 (lig2 05.05.14)
-
0,5
-
Temp. - pet24fla cone 3 (lig3 05.05.14)
-
-
0,5
Primer  iGEM-89flo
1
1
1
Primer  iGEM-89flo
1
1
1
Q5-Buffer  5x
10
10
10
Phusion-Polymerase
0,5
0,5
0,5
Water
37
37
37
Total Volume
50
50
50
PCR protocol for "PCR Q5 elongation for 2kb fragments".
Step
Temperature °C
Time
1
95
3 min
2
95
30 sec
3
55
30 sec
4
72
2,5 min
5
Go To 2
35x
6
72
5min
7
4
Infinite
Result: PCR not successful - no band at 2,2kb
15.24 Transformation of E.coli DH5α with pET24-fla-Construct from 05.05.14
Aim: Amplification plasmid for further experiments.
3 aliquots competent E.Coli DH5α were transformed with 1µll of the purified plasmid  pet24-fla construct from 05.05.14 from 3 different clones (lig 1-3) each
After regeneration the cells were plated on LB-canamycin plates and incubated at room temperature until Monday
15.25 New PCR for amplification of flagellin-constructs out of pET24-fla-Construct from 05.05.14
Aim: Amplification of the fragment for further experiments with pet24-fla from 15.21 as template. The prior PCR was not successful.
Mix [µl]
Pet24fla1
Pet24fla2
Pet24fla3
Temp.  – pet24fla cone 1 (lig1 05.05.14)
0,5
-
-
Temp.  – pet24fla cone 2 (lig2 05.05.14)
-
0,5
-
Temp.  – pet24fla cone 3 (lig3 05.05.14)
-
-
0,5
Primer  iGEM-89flo
1
1
1
Primer  iGEM-89flo
1
1
1
Q5-Mastermix
25
25
25
Water
22,5
22,5
22,5
Total Volume
50
50
50
Q5 PCR
Step
Temperature °C
Time
1
95
3 min
2
95
30 sec
3
55
30 sec
4
72
1 min
5
Go To 2
35x
6
72
5min
7
4
Infinite
11.05.2014
13.43 Repeat Transformation for AG/Cu Promotors
No colonies
15.25 New PCR for ampl. of flagellin-constructs out of pet24-fla-Construct from 05.05.14
Results:
PCR result from the amplification of the fla-construct out of the pET24d. The gel shows that the three clones contain a construct in the pet24d vector. The fragment is about 2,2kb.
15.26 Gel Extraction of PCR Products
Aim: purification of constructs from gel as a preparation for the restriction digest with spe1. The digest will show which clone contains the construct with the replaced strap-tag.
Gel extraction was performed according to the gel extraction kit by Omega.
Extracted PCR product
Concentration [ng/µl]
clone 1
80,2
clone 2
88,2
clone 3
93,9
15.27 Restriction Digest of PCR Products from with SpeI
Aim: The digest with Spe1 will show, which clone contains the fla-construct with replaced StrepTag by a SpeI restriction site
[µl]
Mastermix with NcoI - for 4 attempts
Attempt 1-3 SpeI (20kU/ml)
DNA
-
Ca. 85ng/µl
ca. 1µg → 12µl
Spe1
1
-
Master-Mix
-
3
Cutsmart (10x)
6
-
H2O
5
-
Total Volume
12
15
Loading Dye 6x
-
2,5/attempt
Incubation for 60min at 37°C
Heat shock for 2min at 80°C
Results:
The digestion of the PCR amplified construct shows 3 bands. The 2,2kb band is the uncut construct whereby the smaller bands are the two fragments as a result of the SpeI cut. The constructs in the pET24d still contain the right SpeI restriction site.
15.28 Transformation of E.coli DH5α with pET24-fla-Construct from 05.05.14
Colonies were grown on every plate → successful transformation
15.29 Inoculation for Miniprep of Transformed E.Coli DH5α with pet24-fla-Construct from 10.05.14
Aim: Amplification of transformed plasmid for miniprep in order to receive more plasmid for further experiments
3x6ml LB + 6µl canamycin each were inoculated with 1 colony of plate 1-3 from 10.05.14. The plates were incubated over night at 37°C
12.05.2014
15.30 Miniprep of Transformed E.coli DH5α with pET24-fla-Construct from 11.05.14
Aim: Isolation of plasmid in order to obtain more plasmid for further experiments
15.31 PCR of Construct with pET24d as Template from Miniprep 11.05.14
Aim: Amplification of the plasmid from clone 2 for further experiments
Mix [µl]
Pet24fla2
Temp. - pet24fla clone 2 (lig2 11.05.14)
0,5
Primer  iGEM-89flo
1
Primer  iGEM-89flo
1
Q5-Mastermix
25
Water
22,5
Total Volume
50
The PCR was performed according to the "PCRQ5" protocol.
Step
Temperature °C
Time
1
95
3 min
2
95
30 sec
3
55
30 sec
4
72
1 min
5
Go To 2
35x
6
72
5min
7
4
Infinite
The PCR-products were separated on a 1 % agarose gel. The bands of 2.2 kb were cut out and a gel extraction was carried out (c = 100 ng/µl) according to the "Gel Extraction (QIAquick Gel Extraction Kit)" protocol.
15.32 Restriction of the PCR-Fragment with BamHI and NcoI
Aim: Creating corresponding ends for ligation into pMAD
Restriction
Volume [µl]
PCR-product  hag/flank (100 ng/µl)
5
NcoI-HF
1
BamHI-HF
1
CutSmart-Buffer (10x)
2
H2O
11
Total Volume
20
The restriction is incubated at RT overnight.
13.44 Phosphorylation and Annealing of Promoter-Oligos
Aim: phosphorylation of single stranded oligos with subsequent annealing for cloning into pMa12
For ligation to occur at least one of the DNA ends (insert or vector) should contain a 5' phosphate. Primers are usually supplied non-phosphorylated; therefore, the oligos or PCR product will not contain a 5' phosphate. The digestion of DNA with a restriction enzyme will always produce a 5´ phosphate. A DNA fragment can be phosphorylated by incubation with T4 Polynucleotide Kinase in T4-ligase-buffer which already contains the necessary ATP in an appropriate amount (1 mM):
Phosphorylation [µl]
P(argJ)
P(ykuO)
iGEM-012
1
-
iGEM-013
1
-
iGEM-014
-
1
iGEM-015
-
1
T4-ligase  buffer
5
5
T4 PNK
1
1
H2O
42
42
Total Volume
50
50
The reaction was done in a PCR cycler with a subsequent heat inactivation of the PNK and annealing reaction (protocol: "Annealing with PNK Inactivation")
14.24 Transformation of BL21 cells
Aim: Transform E. coli BL21(DE3) with PheA-Arc1p-C-2x-pET28a
40 µL E. coli BL21(DE3) electrocompetent cells were transformed with 1 ng of the previously purified plasmid PheA-Arc1p-C-2x-pET28a from two different clones, incubated for 1 h at 37 °C and plated out on LB-Kan50 plates that were incubated overnight at 37 °C.
13.05.2014
14.25 Overnight culture
Aim: Inoculate Overnight culture of PheA-Arc1p-C-2x-pET28a
5 mL of LB-Kan50 medium were inoculated with a clone from 14.24 bearing the PheA-Arc1p-C-2x-pET28a plasmid.
15.33 Miniprep and Restriction of pMAD with BamHI and NcoI
Aim: isolate pMAD and create linearized vector with corresponding ends for ligation with the construct
The plasmid pMAD was isolated and digested with the restriction enzymes BamHI and NcoI to create fitting ends for the ligation with the digested PCR-fragment.
Restriction
Volume [µl]
pMAD  (174.9 ng/µl)
2
NcoI-HF
1
BamHI-HF
1
CutSmart-Buffer (10x)
2
H2O
14
Total Volume
20
The restriction was incubated at 37°C for 1h.
15.34 Ligation of digested pMAD with the digested hag/flank-Construct and trafo
Aim: link the fragment with the vector and trafo of DH5α
The ligation of pMAD and the digested PCR-fragment was done according to the previous mentioned ligation calculator.
Ligation
Volume [µl]
pMAD
5
Hag/flank-construct
10
T4 Ligase Buffer (10x)
2
T4 DNA-Ligase
1
H2O
2
Total Volume
20
The ligation was incubated at RT for 1.5 h. After that E. coli DH5α were transformed with the ligation mix and plated out onto LB-Amp plates which were incubated at 37°C overnight.
13.45 Ligation of Annealed Oligos with digested pMa12-Construct and Trafo of E. coli DH5Alpha
Aim: clone the promoter sequences in front of the gfp
A ligation of the SacI and NcoI digested pMa12-construct with the annealed oligos was carried out. The annealed oligos (iGEM-012/013 c = 762.3 ng/µl and iGEM-014/015 c = 825.5 ng/µl) were diluted up to 10-2. 10 ng of the insert and 100 ng of the vector were used for the ligation.
Ligation [µl]
P(argJ)-pMa12-construct
P(ykuO)-pMa12-construct
iGEM-012/013  (7.6 ng/µl)
1,5
-
iGEM-014/015  (8.3 ng/µl)
-
1,5
T4-ligase-buffer  (10x)
2
2
T4-Ligase
1
1
H2O
15,5
15,5
Total Volume
20
20
The ligation was incubated at RT for 2h. The E. coli DH5Alpha were transformed afterwards with the whole ligation mix and plated on LB-Amp, which were incubated at 37°C overnight.
14.05.2014
14.26 Test Expression
Aim: Test the expression of PheA-Arc1p-C-2x
Using the preculture from 14.25 60 mL of LB-Kan50 medium were inoculated with 600 µL of the preculture and incubated at 37 °C and 220 rpm until OD600 reached 0.5. The culture was cooled down to 18 °C and the protein production was induced with 12 µL of 500 mM IPTG. Over night the culture was incubated at 18 °C and 220 rpm. A glycerolstock of the strain containing the PheA-Arc1p-C-2x-pET28a plasmid was stored at -80 °C.
15.35 Inoculation Miniprep and Restriction of pMAD with BamHI and NcoI
Aim: the ligation from 15.34 was not successful. Inoculation of pMAD for miniprep in order to have more plasmid for further experiments.
Additionally we create linearized vector with corresponding ends for ligation with the construct with rest from 13.05.14.
The plasmid pMAD from 13.05.14 (ca. 25µl) and digested with the restriction enzymes BamHI and NcoI to create fitting ends for the ligation with the digested PCR-fragment. The digest was performed in 2 Eppis with 12,5µl plasmid each.
Restriction [µl]
Mastermix
Volume
pMAD  (174.9 ng/µl)
-
12,5
Mastermix
-
7,5
NcoI-HF
1
-
BamHI-HF
1
-
CutSmart-Buffer (10x)
4
-
H2O
9
-
Total Volume
15
20
The restriction was incubated at 37°C for 2h.
15.36 Transformation of pET24d-clone2 into E.Coli DH5α
Aim: raising colonies for further minipreps of successful construct containing clone
The plasmid pET24d with the fla construct from clone 2 was transformed into E.Coli DH5α and plated on LB-canamycin. Incubation overnight at 37°C.
13.37 Inoculation for Miniprep of E.Coli DH5α with pET24d-fla Construct Clone2
Aim: raising colonies for further minipreps of successful construct containing clone2
One clone with plasmid pET24d with the fla construct was used for inoculation of LB-medium. Incubation overnight at 37°C
13.46 Transformation of Nose-Plasmid from Clone 10 into E.Coli DH5α
Aim: raising colonies for miniprep of new Nose-plasmid for further experiments
The plasmid pMA12+Nose-Construct from clone 10 was transformed into E.Coli DH5α and plated on LB-AMP. Incubation overnight at 37°C
13.37 Ligation of digested pMAD from 14.05.14 with the digested hag/flank-construct
Aim: link the fragment with the vector
The ligation of pMAD and the digested PCR-fragment was done according to the previous ligation calculator.
Ligation
Volume [µl]
pMAD
5
Hag/flank-construct
20
T4-Ligase-Buffer (10x)
3
T4-DNA-Ligase
1
H2O
1
Total Volume
30
Incubation overnight at room temperature.
13.47 Restriction and purification of piGEM002 (Nose) and following ligation and transformation
Aim: create more plasmid for further experiments, 2 digests were performed to get a high amount of plasmid (pooled during purification)
concentration= 44 ng/µl
Restriction
Volume [µl]
pMa12 from clone 10
1,5
NcoI
1
SacI-HF
1
CutSmart-Buffer (10x)
2
H2O
14,5
Total Volume
20
One ligation was performed for 1h and transformed into chemocompetent cells, the second ligation was performed over night in case no colonies grow on the trafo plates.
Ligation Mix
Volume [µl]
Buffer
2
Water
14,5
Plasmid
1,5
Primer Mix (1:100)
1
Ligase
1
Transformation:
thaw on ice: 5 min
add DNA, keep on ice: 5 min
heat shock in water bath at 37°C: 45 sec
put back on ice and add 1 ml LB
recovery for LB: 20 min
17.1 Inoculate Bacillus Strains
Aim: have bacillus strains to work with in MTA and make competent cells
Wildtype on LB plate and in LB liquid without an antibiotic.
Bacillus subtilis strain 186 on plates containing xylose (Incubation overnight).
15.05.2014
17.2 Make Competent Cells
Aim: making Bacillus subtilis cells competent for transformation
Inoculate Bacillus wildtype strain in SPC medium.
Time
OD570
13:30
5,7
14:00
6,2
15:45
6,6
16:30
7
17:00
7,2
 
→ new media
Incubation for 1,5h
Aliquot and thaw in liquid nitrogen. Store at -80°C.
14.27 Purification of test expression
Aim: Purify PheA-Arc1p-C-2x on a small scale
The cell culture from 14.26 was centrifuged (17000 rpm, 20 min, 4 °C) and the pellet resuspended in 3 mL buffer A. Lysozyme was added to a final concentration of 1 mg/mL and incubated on ice for 30 min. Afterwards the cells were lysed using ultra sound (2x 18 pulses, 30 s on ice inbetween). To clear the suspension of cell debris it was centrifuged (13000 rpm, 15 min, 4 °C). The supernatant was purified using QIAGEN Ni-NTA Spin Columns. An SDS-PAGE gel showed that the protein is produced and can be purified using Ni affinity chromatography.
60 mL LB-Kan50 medium was inoculated from the glycerolstock prepared in 14.26 and incubated at 37 °C and 220 rpm over night.
15.38 Transformation of ligated pMAD and the digested hag/flank-Construct
Aim: produce cells that contain the ligated plasmid
Chemically competent cells from Daniel were transformed with the 20µl ligation-mix, 20µl of the cells were transformed with 1µl pMad uncut (control).
Protocol: see yesterday
15.39 PCR for amplification of Hag-Flank-construct in pET24d and following restriction
After plasmid isolation a PCR was run to amplify the hag-Flank-construct, afterwards it will be digested with SpeI to find a clone with pure product (no tag but only SpeI restriction site).
PCR Mix
Volume [µl]
Q5-Mastermix
88
Water
110
Primer 89
5,5
Primer 90
5,5
Template
1
Gel foto after amplification of hag flank insert from 10 clones: all positive.
Whole insert has been digested for 1h with SpeI in a 20µl reaction mix
17.5µl insert, 2µl buffer and 1µl ligase
Gel foto after restriction of the 2.2 kb fragment with SpeI (expected sizes appear on the gel) no undigested bands= only fragments with the restriction site are in
13.48 Colony PCR after transformation
Aim: find out which colonies contain the right plasmid
Pick a colony from the trafo plate, streak it on a new plate and dip in the PCR Premix
[µl]
Amount Ag
Amount Cu
Q5-Mastermix
8
8
Water
10
10
TW 260
1
1
Primer 12
1
-
Primer 14
-
1
17.3 First Plate Reader Measurement with Bacillus subtilis wildtype and strain 186 containing a gfp
Aim: Test for differences between the two strains and first check if experiment works at all
Both strains grew best in LB medium like expected, a clearly better growth was shown by the wildtype in the minimal medium compared to the gfp bearing strain.
The fluorescence is shown in dependence of the culture growth in order to relativise the fluorescence signal. Like expected, the wt exhibited no fluorescence in both media. The fluorescence of the gfp strain was higher in the minimal medium with xylose, than without xylose.
16.05.2014
15.40 Colony PCR with colonies from 16.05.14
Aim: test the success of ligation from pMAD & Hag-Flank-Construct
A PCR ran with 5 picked clones from 16.05.14 and primers Flo89 and 90 for amplification of inserted construct
 
Attempt [µl]
Colony
1 colony
Q5-Mastermix
10
Water
8
Primer Flo89
1
Primer Flo90
1
Results
Test if insert in pMad has the right size of about 2000 bp → not precise repeat on Monday
15.41 Inoculation of colonies E. coli pMad plus hag-flank-Insert
Aim: grow cells that contain the mentioned plasmid and subsequently isolation of the plasmid for further work
Inoculate 5 clones in LB liquid and incubation over the weekend at room temperature
14.28 Expression of PheA-Arc1p-C-2x
Aim: Express PheA-Arc1p-C-2x for further use
10 x 500 mL LB-Kan50 medium were inoculated with 5 mL of the overnight culture prepared in 14.27 and incubated at 37 °C and 220 rpm until OD600 reached 0.5. The flasks were cooled to 18 °C and protein production was induced by adding 100 µL 500 mM IPTG. The cultures were incubated over night at 18 °C and 220 rpm. Afterwards the cells were collected by centrifugation (6000 rpm, 20 min, 4 °C), resuspended in buffer A and stored at -20 °C until further use.
19.05.2014
15.42 Test restriction of pMad-hag-flank and Test PCR
Aim: determine whether the plasmid contains the right insert
Restriction: concentration of plasmid after isolation between 20 and 30 ng/µl
 
Volume [µl]
Buffer / Water
27
Template
10
NcoI
0,5
BamHI
0,5
Results
Test restriction of pMad with hag flank insert did not show a clear result (last band is a different sample)
 
Amount [µl]
Q5-Mastermix
10
Primer 89
1
Primer 90
1
Template
1
Water
7
Results
The PCR showed the expected fragment of the hag-flank construct with a size of approx. 2 kb.
17.4 Bacillus Trafo with Nose Plasmids
5µl DNA were added to 100µl cells, 30 min incubation at 37°C and plated on LB-Cm.
14.29 Purification of PheA-Arc1p-C-2x
Aim: Purify PheA-Arc1p-C-2x in three steps
The resuspended cells from 14.28 were lysed using a french press. The lysate was incubated on ice with 5 µg/mL DNAseA and RNAseI for 10 min and centrifuged (27000 rpm, 45 min, 4 °C) to pellet cell debris. The supernatant was filtered (0.45 µm) and PheA-Arc1p-C-2x purified from the cleared solution by NiNTA affinity chromatography. The combined elution fractions containing crude PheA-Arc1p-C-2x were concentrated to a volume of 2 mL and further purified using an anion exchange column to get rid of possible tRNA contaminations. Finally the buffer was changed to dialysis buffer and the protein concentrated and stored at -80 °C in aliquods for further use.
20.05.2014
14.30 Creating PheA-Arc1p-C-1x
Aim: Using Round-the-horn mutagenesis to create PheA-Arc1p-C-1x-pET28a
The PCR with 5' phosphorylated TG_1xGSSG FP und TG_0xGSSG RP should generate the 7509 bp fragment for ligation from the methylated template PheA-Arc1p-C-8x-pET28a from 14.5.
The used Primer concentration was 1 µM, the concentration of the template was 1 ng/µL and we used a hot start to avoid unspecific annealing of primers.
Content
Volume [µL]
Water
29.5
5x GC Buffer
10
DMSO
2.5
TG_1xGSSG FP
2.5
TG_0xGSSG RP
2.5
PheA-Arc1p-C-8x-pET28a
1
dNTPs
1
Phusion-Polymerase
1
Total Volume
50
Step
Temperature [°C]
Time [min:sec]
1
98
2:00
2
add Polymerase
 
3
98
0:10
4
67.5
0:30
5
72
4:00
6
go to 3
32x
7
72
10:00
8
4
hold
The amplified fragment was purified on an agarose gel on which it showed the correct size. The extracted linear template was then digested to degrade the original unmutated plasmid using DpnI.
Content
Volume [µL]
10x CutSmart Buffer
3.6
Plasmid
30
DpnI
3
Total Volume
36.6
The reaction mixture was incubated at 37 °C and 300 rpm for 1 h and afterwards heated to 80 °C for 20 min.
In order to create the circular plasmid 100 ng of the linear template were mixed with 1 µL 10x T4-Ligasebuffer,  1 µL T4-Ligase and water to give a total volume of 10 µL. This mixture was incubated for 9 h at 16 °C. The resulting plasmid was dialysed with water to decrease the salt concentration for the following transformation of E. coli Top10 electrocompetent cells with the plasmid.
15.43 PCR to Amplify Cup 1 from Yeast gDNA
Aim: create the DNA fragment cup1 (metallothionein) for insertion into the flagellin (Gibson assembly with pMad by cutting the plasmid with SpeI at the created restriction site)
 
Amount [µl]
Q5-Mix
20
Primer 89
1
Primer 90
1
Template
1
Water
17
The Purple 2-log ladder was used as a marker. The amplified cup1 could be seen as a 204 bp fragment on the gel.
17.5 Bacillus Trafo with Nose Plasmids
No colonies → new trafo
New LB-chloramphenicol plates for Bacillus (only 5µg/ml instead of 25µg/ml)
21.05.2014
15.44 Restriction pMad with SpeI
Aim: Create a linearized plasmid for following Gibson assembly with cup1
 
Volume [µl]
Buffer
2,5
Plasmid
20
SpeI
1
Water
-
Concentration of plasmid after purification only 7 ng/µl
14.31 Overnight culture
Aim: Amplify Plasmid in an overnight culture
Six clones of the transformed E. coli Top10 cells from 14.30 were picked and used to inoculate overnight cultures (6 x 5 mL).
15.45 Gibson Assembly of digested pMad-fla and cup 1 and Transformation into E.Coli XL1-Blue
Aim: create a flagellin that contains the metallothinein cup1
[µl]
Control
With Insert
Gibson Mix
15
15
Plasmid (7ng/µl)
5
5
Insert
-
1
Water
1
-
A part of the aliquot was plated on LB-tet for inoculation of colonies for competent XL1-Blue.
22.05.2014
17.6 MTA Bacillus Nose Ag
Transformation from 20.5.14: one colony on the Ag Nose plate
Inoculation of shaking culture in the morning together with the wildtype and the gfp strain in LB liquid without antibiotics
Inoculate 96 well plate in the afternoon 15:00
Prepare Cu and Ag solutions to test different concentrations in MTA and see if the Ag nose responds to Ag and also if it is specific for Ag and does not respond to Cu
Ag and Cu in stock solution of 10 mM have been prepared
Added (Concentration): 11 mM, 100µM and 10µM
Cells: Ag Nose mutant, wildtype (AG GM) and gfp
The MTA showed no significant increase of fluorescence. The bacteria had difficulties in growth, thus it seemed that the metal ion concentrations were too high.
17.7 Digestion of piGEM002 and the nose plasmids for Ag/Cu
Aim: Linearize the plasmids with SpeI to built in the instability tags (25µl mix)
[µl]
piGem 002 clone 7
Nose plasmid 003/004
Amount
Template ~10µg
15
20
15
SpeI
1
1
1
CutSmart-Buffer
2,5
2,5
2,5
Water
6,5
1,5
6,5
Incubation at room temperature over night.
14.32 Preparation of plasmids
Aim: Purify the amplified plasmids from overnight cultures
The plasmids were purified from the overnight cultures created in 14.31 using Sigma GenElute Plasmid Miniprep Kit following the included protocol. The plasmids were eluted using 45 µL water.
23.05.2014
17.7 Create Instability Tags for gfp in the Nose Plasmids
Aim: GFP is a stable protein, which would accumulate and distort the results if not degraded, instability tags of different strength cause the degradation and therefore cause quantitative results
First step: Linearize the following plasmids: piGEM 002 without promoter as well as the plasmids containing the silver and copper sensitive promoters
Second step: PCR to anneal the oligos 20+21, 20+22 and 20+23
 
Volume [µl]
Primer A
5
Primer B
5
Phusion
0,25
dNTPs
0,25
Buffer
5
Water
34,5
Cycle
3
Afterwards clean up with kit (all three mixes were carried out in a duplicate, 1 each was frozen away at minus 20)
Third step: Yeast transformation
9 Reactions (3 plasmids and 3 instability (ssrA-)tags)
100 ng plasmid plus 1µl of primer mix were filled to 14µl and added to the yeast transformation mix and subsequently transformed into yeast.
26.05.2014
15.46 Restriction of pMAD-fla with SpeI, Gibson-Assembly with cup1-1 and transformation of E. coli DH5α
Aim: insert cup1-1 into the hag/flank-construct
To insert the metallothionein gene cup1-1 into the hag/flank-construct the pMAD-fla was digested with SpeI to linearize it.
Restriction
Volume [µl]
pMAD-fla  (330 ng/µl)
4
SpeI
1
CutSmart Buffer (10x)
1
H2O
6
Total Volume
10
The restriction was carried out at 37°C for 1h.
The restriction was cleaned with a DNA Gel Ex Kit (c = 40 ng/µl). 4µl (160 ng) of this digested plasmid were mixed with 1 µl of the cup1-1 fragment (9.6 ng/µl). These 5 µl were inserted into the prepared 15 µl Gibson-mix. The reaction was incubated at RT for 1 min and then at 50°C for 1 hour. An extra Gibson-reaction was carried out as a control with just the linearized pMAD without the cup1-1.
E. coli DH5α cells were transformed with the Gibson reactions and plated out on LB-Amp. They were incubated at 37°C over night.
15.47 Colony-PCR of E. coli XLI-Blue pMAD-fla + cup1-1 from 21.05.14 and 23.05.14
Aim: check for correct insertion of cup1-1 into the hag/flank-construct
Since a Gibson-Assembly with the same aim was carried out last week, the colonies were checked by a colony PCR with the primers for cup1-1 iGEM-018 and iGEM-019. One colony was picked from the plate from 23rd May and three were taken from the plate from 21st May. The latter was contaminated with Bacillus colonies. For this reason E. coli was picked and streaked out onto a new plate on Friday the 23rd May. The consequence is a plate with just one clone, what is the reason, that only one colony was picked from this plate.
Mix
Volume [µl]
E. coli XLI-Blue pMAD-fla+cup1-1
part of a colony
Primer  iGEM-018
0,5
Primer  iGEM-019
0,5
Q5-Mastermix
10
Water
9
Total Volume
20
A part of each colony was transferred onto a new LB-Amp plate, which was incubated at 37°C overnight.
Result
The expected fragment in size of the cup1 could be amplified from the template, which indicates a positive result.
14.33 Transformation of BL21 cells
Aim: Transform E. coli BL21(DE3) with PheA-Arc1p-C-1x-pET28a
40 µL E. coli BL21(DE3) electrocompetent cells were transformed with 1 ng of the previously purified plasmid PheA-Arc1p-C-1x-pET28a from two different clones, incubated for 1 h at 37 °C and plated out on LB-Kan50 plates that were incubated overnight at 37 °C.
13.49 PlasmidPprep from Yeast and Transformation of E. coli DH5Alpha
Aim: increase the amount of plasmid in E. coli for further experiments with the degradation tags
The gfp in piGEM002, piGEM002-Ag (and piGEM002-Cu was tagged with different degradation tags by yeast recombination. The plasmids have been isolated from yeast.
E. coli DH5α cells were transformed with 15 µl of each isolated plasmid and plated out on LB-Amp. They were incubated at 37°C over night.
Gly-Stocks - Making of -stocks for long time storage of bacteria strains
For long time freezer storage of bacteria, they have to be treated with glycerine to prevent the crystallization and damaging of the cells. For this reason 2 ml of LB with the respective antibiotic were inoculated with the bacteria, which were meant to be stored and were incubated at 37°C overnight.
Strains that are to be stored:
E. coli XLI-Blue pMAD-fla  (piGEM-005)
E. coli XLI-Blue pMa12-construct  (piGEM-002)
E. coli XLI-Blue pMa12-Cu  (piGEM-004)
E. coli XLI-Blue pMa12-Ag (piGEM-003)
E. coli BL21(DE3) pLysS
27.05.2014
14.34 Overnight culture
Aim: Inoculate Overnight culture of PheA-Arc1p-C-1x-pET28a
5 mL of LB-Kan50 medium were inoculated with a clone from 14.33 bearing the PheA-Arc1p-C-1x-pET28a plasmid.
15.48 Restriction of pET24d-fla with SpeI and Gibson-Assembly with cup1-1
Aim: linearize pET24d-fla and insert cup1-1 into pET24d-fla
Since the colony-PCR (26.05.14) was negative and the Gibson-Assembly of linearized pMAD-fla with cup1-1 showed no results the pET24d-fla construct was linearized with SpeI.
Restriction
Volume [µl]
pET24d-fla  (98.8 ng/µl)
8
SpeI
1
CutSmart-Buffer (10x)
2
H2O
9
Total Volume
20
The restriction was incubated at 37°C for 1 h.
The restriction mix was purified with the Gel Ex Kit. The yield was too low (c = 3.1 ng/µl) to use it for a Gibson-Assembly, so the rest of the pET24d-fla was digested with SpeI.
Restriction
Volume [µl]
pET24d-fla  (98.8 ng/µl)
18,5
SpeI
1
CutSmart-Buffer (10x)
2,5
H2O
3
Total Volume
25
The restriction was incubated at 37°C for 1h.
The restriction mix was purified with the Gel Ex Kit (c = 20.1 ng/µl), which was enough for the Gibson-Assembly. 4µl (80 ng) of this digested plasmid were mixed with 1 µl of the cup1-1 fragment (9.6 ng/µl). These 5 µl were inserted into the prepared 15µl Gibson-mix. The reaction was incubated at RT for 1 min and then at 50°C for 1 h. An extra Gibson-reaction was carried out as a control with just the linearized pET24d-fla without the cup1-1.
Competent E. coli XLI-Blue were transformed with the whole Gibson-reactions. They were plated on LB-Kan an incubated at 37°C overnight.
13.50 Colony-PCR of the E. colis Containing the Degradation tagged Constructs
Aim: check for correct tagging of gfp with the respective degradation tag
The transformation with the degradation tag-constructs were successful nearly in all cases except the pMa12-Ag-21. The transformation was repeated, the plate was incubated at 37°C over night.
Four colonies of every plate have been picked, transferred to new LB-Amp plates and were subsequently used for the colony-PCR.  The Master Mix was portioned in aliquots of 20 µl per PCR-cup and inoculated with the respective colony.
MasterMix for 33 reactions
Volume [µl]
E. coli DH5Alpha with construct
part of a colony
AG  TW 260
16
AG  TW 316
16
Q5-Mastermix
165
Water
133
Total Volume
330
Step
Temperature °C
Time
1
95
5 min
2
95
20 sec
3
55
20 sec
4
72
20 sec
5
Go To 2
35x
6
72
5min
7
4
Infinite
Colony-PCR of E. coli with the degradation-tag constructs
The bands slightly higher than 200 bp indicated a positive clone. Positive constructs are available: 002-21, Cu-23, Ag-22 and Cu-22.
Gly-Stocks - Making of glycerin-stocks for long time storage of bacteria strains
1 ml of the bacteria which were inoculated yesterday (26.05.2014) were transferred to new, sterile 1.5 ml Eppendorf-cups. 600 µl of 50% glycerin were added and the cells were frozen and stored at -80°C in the freezer.
18.1 Making competent E. coli XLI-Blue with RbCl: Inoculation of LB-Tet with E.Coli XL1-Blue as a preculture
9 ml LB-Tetracycline (1:1000) were inoculated with E. coli XLI-Blue cells as a preculture. They were incubated at 37°C overnight.
28.05.2014
18.2 Inoculating E.Coli XL1-Blue main culture
93 mL of LB-tet were added to 7 mL of preculture for inculation of main culture. The main culture was incubated at 37°C until an OD of 0,5.
18.3 Preparing Transformation Buffer TBF1 and TBF2
TBF1 recipe
TBF2 recipe
for 50ml
 
 
 
potassium acetate
0,15g
MnCl2 x 4H2O
 
1M RbCl
5 ml
1m CaCl2
0,5 ml
87% glycerine
8,6 ml
A.dest
35,9 ml
autoclave or sterile filtration
 
 
 
for 250 ml
 
 
 
1M NaMOPS (pH7)
2,5 ml
1M CaCl2 x 2H2O
8,75 ml
1M RbCl2
2,5 ml
87% glycerine
43,75 ml
A.dest
192,5 ml
adjust pH7
 
store at 4°C in the dark
 
sterile filtration
 
18.4 Preparing E.Coli XL1-Blue
Grow 100 ml E.coli culture to OD600=0,5
Put on  ice for 10 min
Centrifuge  2x 50 ml for 10 min at 3000 rpm and 4°C
Wash  pellets in 10 ml TFB1 (10 min; 3000 rpm; 4°C)
Resuspend  pellets in 2ml TFB2
Pipett  200µl aliquots in precooled 1,5 ml tubes
store  at  -80°C
14.35 Test Expression
Aim: Test the expression of PheA-Arc1p-C-1x
Using the preculture from 14.34 60 mL of LB-Kan50 medium were inoculated with 600 µL of the preculture and incubated at 37 °C and 220 rpm until OD600 reached 0.5. The culture was cooled down to 18 °C and the protein production was induced with 12 µL of 500 mM IPTG. Over night the culture was incubated at 18 °C and 220 rpm. A glycerolstock of the strain containing the PheA-Arc1p-C-1x-pET28a plasmid was stored at -80 °C.
15.49 Colony-PCR with transformed E. coli XLI-Blue including pET24d-fla-cup1-1
Aim: testing the success of insertion of cup1-1 into pET24d-fla
6 clones were picked from transformation plate and a colony PCR was ran with primers iGEM-018 & - 019. The result should be a band at 240 bp in case of a positive clone.
[µl]
Mastermix
Mix for PCR attempts
E. coli XLI-Blue pet24d-fla+cup1-1
 
part of a colony
Primer  iGEM-018
3,5
-
Primer  iGEM-019
3,5
-
Q5-Masterix
70
-
Water
63
-
Master-Mix
-
20
Total Volume
140
 
Step
Temperature °C
Time
1
95
1 min
2
95
20 sec
3
55
20 sec
4
72
20 sec
5
Go To 2
35x
6
72
5min
7
4
Infinite
The expected band of approx. 240 bp could be seen in every sample, more or less clearly
13.51 Colony-PCR of the E. coli containing the degradation tagged constructs
Aim: check for correct tagging of gfp with the respective degradation tag
The Colony-PCR was repeated with pMa12-Ag-21 after successful transformation.
Four colonies have been picked, transferred to new LB-Amp plates and were subsequently used for the colony-PCR.
The Master Mix was portioned in aliquots of 20 µl per PCR-cup and inoculated with the respective colony.
[µl]
Master-Mix for attempts
Mix for PCR attempt
E. coli DH5α with Ag21
-
part of a colony
AG  TW 260
2,5
-
AG  TW 316
2,5
-
Q5-Mastermix
50
-
Water
45
-
Master-Mix
-
20
Total Volume
100
20
Colony-PCR of E. coli containing the degradation-tag construct Ag-21
In an additional reaction the colony-PCRs of the negative clones (gel 28.05.14) were repeated with more picked clones per construct. This time 8 colonies of each negative construct were picked, transferred to another LB-Amp-plate and used for the PCR-reaction. Four times eight reactions were carried out, a master mix was prepared for 33 reactions.
[µl]
Master-Mix for 33  reactions
Mix for one PCR reaction
Part of a colony
-
1
AG  TW 260
8
0,25
AG  TW 316
8
0,25
Q5-Mastermix
330
10
Water
314
9,5
Master-Mix
-
20
Total Volume
660
20
Step
Temperature °C
Time
1
95
3 min
2
95
20 sec
3
55
20 sec
4
72
20 sec
5
Go To 2
35x
6
72
5min
7
4
Infinite
E. coli containing the degradation-tag constructs
Positive clones could be obtained with the Ag-23 and the 002-22 constructs (approx. 250 bp).
18.5 Transformation test with E.Coli XL1-Blue and pMAD (fla)
E.Coli XL1-Blue were transformed with 1µl pMAD (189 ng/µl). Another aliquod was transformed with pMAD-fla.
In comparison E.Coli DH5α were transformed the same way.
29.05.2014
18.5 Transformation Test with E.Coli XL1-Blue and pMAD (fla)
Every transformation was successful. The higher number of colonies on the pMAD-transformation plate shows a higher transformation efficiency compared with DH5α.
18.6 Inoculation for Miniprep and Glycerin stock of pMAD-fla
Aim: In order to save pMAD and pMAD-fla for further experiments and storage, minipreps and glycerine stocks have to be inoculated
For minipreps 2 x 6 ml of LB-Amp were inoculated with one colony from the pMAD XL1-Blue-transformation plate. Additionally 3 x 6 ml of LB-Amp were inoculated with one colony from the pMAD-fla XL1-Blue-Transformation plate.
Furthermore 2 x 2 mL LB-Amp were inoculated with a clone from pMAD-fla XL1-Blue-Transformation plate for glycerine stocks.
Every attempt was incubated at 37°C overnight.
13.52 Colony-PCR of the E. coli containing the degradation tagged constructs
Aim: check for correct tagging of gfp with the respective degradation tag
The Colony-PCR was repeated with 002-Cu21 and 002-23.
Four colonies have been picked, transferred to new LB-Amp plates and were subsequently used for the colony-PCR.
The Master Mix was portioned in aliquots of 20 µl per PCR-cup and inoculated with the respective colony.
[µl]
Master-Mix for 18  attempts
Mix for PCR attempt  002-23.1-8
Mix for PCR attempt 002-Cu21.1-8
E. coli DH5α with 002-23
-
part of a colony
-
E. coli DH5α with Cu21
-
-
part of a colony
AG  TW 260
9
-
-
AG  TW 316
9
-
-
Phusion DNA-polymerase
3,6
-
-
Q5-Buffer 5x
72
-
-
dNTP-Mix
5
-
-
Water
261,4
-
-
Master-Mix
-
20
20
Total Volume
360
20
20
Step
Temperature °C
Time
1
95
3 min
2
95
20 sec
3
55
20 sec
4
72
20 sec
5
Go To 2
35x
6
72
5min
7
4
Infinite
14.36 Purification of test expression
Aim: Purify PheA-Arc1p-C-1x on a small scale
The cell culture from 14.35 was centrifuged (17000 rpm, 20 min, 4 °C) and the pellet resuspended in 3 mL buffer A. Lysozyme was added to a final concentration of 1 mg/mL and incubated on ice for 30 min. Afterwards the cells were lysed using ultra sound (2x 18 pulses, 30 s on ice inbetween). To clear the suspension of cell debris it was centrifuged (13000 rpm, 15 min, 4 °C). The supernatant was purified using QIAGEN Ni-NTA Spin Columns. An SDS-PAGE gel showed that the protein is produced and can be purified using Ni affinity chromatography.
60 mL LB-Kan50 medium was inoculated from the glycerolstock prepared in 14.35 and incubated at 37 °C and 220 rpm over night.
15.53 Inoculation of Minipreps from Nose-Tag-Plasmids
Aim: isolate the plasmid to tansform Bacillus for further plate reader assays
6 mL LB-Amp were inoculated with different colonies cotaining the following plasmid-constructs:
002-21
002-22
002-Cu-23
002-Ag-23
002-Cu-22
002-Ag-22
Incubation over night at 37°C
30.05.2014
14.37 Expression of PheA-Arc1p-C-1x
Aim: Express PheA-Arc1p-C-1x for further use
10 x 500 mL LB-Kan50 medium were inoculated with 5 mL of the overnight culture prepared in 14.36 and incubated at 37 °C and 220 rpm until OD600 reached 0.5. The flasks were cooled to 18 °C and protein production was induced by adding 100 µL 500 mM IPTG. The cultures were incubated over night at 18 °C and 220 rpm. Afterwards the cells were collected by centrifugation (6000 rpm, 20 min, 4 °C), resuspended in buffer A and stored at -20 °C until further use.
18.7 Miniprep of E.Coli XL1-Blue with pMAD and pMAD-fla
Plasmid
Concentration  after prep (ng/µL)
pMAD
114 & 108
pMAD-fla (iGEM-005)
114,  128 & 142
Plasmids were frozen away at -20°C
13.54 Minipreps of Nose-Tag-plasmids
Plasmid
Concentration  after prep (ng/µl)
002-21
368
002-22
401
002-Cu-22
425
002-Cu-23
427
002-Ag-22
806
002-Cu-23
402
13.55 Inoculation of Minipreps with Ag-21
The positive clone 3 from the colony PCR (28.05.14) was used to inoculate 6 ml LB-Amp for a miniprep in order to receive the plasmid for transforming Bacillus subtilis.
Incubation at room temperature over two days.
15.49 Inoculation of pMAD-fla-cup1-1 for Miniprep
Aim: inoculation for miniprep in order to receive plasmid for bacillus-trafo
6 mL LB-Amp were inoculated with  the positive clone 1 from the Gibson assembly with cup1-1 of the transformed E. coli XL1-Blue from 26.05.14. After isolation of the plasmid Bacillus Subtilis. can be transformed.
Incubation at room temperature over two days.
13.54 Transformation of Bacillus with isolated plasmids
Aim: Creation of Bacillus strains with the following plasmids:
BS piGEM002 + DT22
BS piGEM 011 (002 AG + DT22)
BS piGEM 012 (002 AG + DT23)
BS piGEM 014 (002 Cu + DT22)
BS piGEM 002 (Nose)
BS piGEM 003 (Nose Ag)
BS piGEM 004 (Nose Cu)
13.55 Colony-PCR of the E. coli containing the degradation tagged constructs
Aim: check for correct tagging of gfp with the respective degradation tag
Phusion and Q5 Buffer did not work out so the PCR was repeated with Q5 Mastermix.
PCR performed with 002-Cu21 and 002-23
Positive clones for both tags (approx. 250 bp) have subsequently been inoculated in LB-Amp and incubated at room temperature over two days.
Retrieved from " http://2014.igem.org/Team:Marburg:Project:Notebook:May "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
