"
Page
Discussion
View source
History
teams
Log in
 
Team:Marburg:Project:Notebook:Methods
From 2014.igem.org
Home
Project
Safety
Team
Policy & Practices
Attributions
SilverSURF
CancerSURF
RiboSURF
Notebook
Parts
Notebook: Methods
March
April
May
June
July
August
September
October
Methods
Competent E. coli
Transformation
Expression Test
Competent B. subitilis
Miniprep
Gel Extraction
pMAD
High Salt/Low Salt
Transf. B. subitilis
Ni-NTA column
Caco-2-cells
PCR Purification
Annealing PNK
Medium
Buffer
Microtiter Assay
Competent Cells Escherichia coli
1. Peparation for preculture
50 µL Aliquod in 2x5ml LB  each
incubation Overnight at 37°C
2. Preparation for main culture
5ml of preculture in 250ml LB
incubation at 37°C until 0,5 OD (2h) →10min on ice
3. Making main culture competent
Pellets: 4x50ml in 4 Falcons
centrifugation at °C, 3500rpm, 15min
resuspending pellets in 4ml HTP* buffer for each Falcon (16ml total)
transfer new suspensions in 1 falcon → 10min on ice
centrifugation at 4°C, 3500rpm, 15min
resuspend pellet in 3ml HTP* buffer
add 225 µL DMSO
50 µ aliquods → freeze in N2 (l)
store at -80°C
Transformation and Cultivation
1. Transformation
transfer 1 ÂµL plasmid into aliquots for 10min on ice
45sec. Heat shock at 42Â°C
10 min on ice
2. Cultivation of transformed cells
add 700 ÂµL LB to transformed cells
incubation for 1,5h at 37Â°C
centrifugation for 3min at 13000rpm
get off 600 ÂµL per 1,5ml tube
resuspend pellet in left 100ÂµL LB
plating on agar plate
incubation overnight
Expression Test: Induction + SDS-Gel
1. Picking colonies
inoculate 20 ml LB medium (sterile) containing 20ÂµL antibiotics**  and incubate until  an OD of 0,6
45sec. Heat shock at 42Â°C
10 min on ice
2. Induction with IPTG/ Lactose
before induction:  0,7/ OD = sample  volume
pellet + 80 ÂµL water were resuspended in 20ÂµL loading buffer  (preinduction sample)
induction with 100 ÂµL IPTG/ lactose at an OD of 0,6
incubation for 1,5h    - 4 h max
after induction:  200 ÂµL sample
pellet  +80 ÂµL water + 20 ÂµL loading buffer  resuspended (induction sample)
big volume gap because of   decreased growth of bacteria
after induction
3.Expression test with induced culture
transfer in 2x50ml falcons
centrifugation at 4Â°C, 3500rpm, 15min
washing  pellets in 10ml buffer A
cracking  cells with micro fluidizer
thermo  centrifugation for 20 min â€“ 4Â°C
200 ÂµL Ni-NTA beats +  supernatant
5min 4000  rpm → pellet
resuspended  pellet in 500 ÂµL  Buffer A (low imidazole lv)
centrifugation  1 min â€“ 4000rpm
pellet  resuspended in buffer A
centrifugation  1 min â€“ 4000rpm
pellet  resuspended in 200 ÂµL Buffer B (elution)
centrifugation  1min â€“ 13000
80ÂµL supernatant (incl.  protein)+ 20ÂµL  loading buffer
4. Expression test with induced culture → SDS-Gel
ladder â€“ 5 ÂµL
PIâ€“ preinduction sample - 10ÂµL
I    - induction sample -10 ÂµL
E    -elution sample -10 ÂµL
Making Competent Bacillus subtilis
B. subtilis (PY79) is naturally competent (Albano et al., 1987). In order to increase the transformation rate, Bacillus can be grown in special media to enhance the competence. To achieve this, 20ml SPC-medium was inoculated with half a well grown LB-plate and incubated at 37Â°C until the cells reached the stationary phase. (Optical density OD550 nm does not change within a a timespan of 30 min) The cells were transfered into 100ml SPII-medium and incubated for 90min at 37Â°C. Afterwards, the cells were centrifuged (500 rpm, 15min, Rotor: BS4402/A Heraeus). The pellet was resuspended in 1ml glycerine (50% (v/v)). Samples were aliquoted and stored at -80Â°C.
For the transformation with plasmid-DNA 100Âµl cells and 5-7Âµl DNA (chromosomal DNA: 0,1-1Âµl DNA) were mixed andd incubated at 37Â°C for 30min. Afterwards, LB-plates (with antibiotics for selection) were inoculated with the culture and incubated at 30Â°C until colonies formed.
SPC-Medium:
T-Base (10x)
2 ml
D-Glucose Monohydrate
0,5% (w/v)
MgSO4
0,018% (w/v)
Caseinhydrolysate
0,025% (w/v)
Yeast-extract
0,2% (w/v)
ddH2O ad
20 ml
SPII-Medium
T-Base (10x)
 
10 ml
D-Glucose Monohydrat
50% (w/v)*
1 ml
MgSO4
1,2% (w/v)
7 ml
Caseinhydrolysate
1% (w/v)
1 ml
Yeast-extract
10% (w/v)
1 ml
CaCl2
0,1 M
500 Î¼l
ddH2O ad
 
100 ml
10x T-Base:
(NH4)2SO4
150 mM
K2HPO4*3 H2O
1 M
KH2PO4
440 mM
Tri- Natriumcitrat di- Hydrat
39 mM
ddH2O ad 1000 ml
*Glucose was substituted with D-fructose in case one of the bacteria strains has a gene inserted under the xylose-promoter, since glucose does inhibit the uptake of xylose
Miniprep (Plasmid DNA Miniprep Kit II, Omega)
centrifuge at 10.000g for 1min
add 250µl Resuspension Buffer P1 per tube (incl. RNase), vortex
transfer suspension into new 1,5 ml tube
add 250µl Lysis Buffer P2 per tube, invert until lysate is clear (2-3min)
add 350µl Neutralisation Buffer N3 per tube
invert immediately (precipitate forms)
centrifuge 10min at 13.000 rpm
transfer supernatant onto column
centrifuge for 1min
add 500µl Washing Buffer PE (HBC Buffer) per tube
centrifugation for 1min throw filrate away
add 700µl DNA Wash Buffer per tube
centrifugation for 1min throw filrate away
add 30µl milipore per tube â€“ let column sit for 2min
centrifuge for 2min
Gel Extraction (QIAquick Gel Extraction Kit)
band is cut out of the gel under UV light and transferred into a 1,5ml tube
weigh the gel with the tube
add QG (100mg gel → 100µl)
incubate for approx. 10min at 50-60Â°C, vortex to dissolve the gel
put spin colum into 2mlcollection tube
add dissolved gel onto column → centrifuge for 1min
discard flow-through
wash with 750Âµl Buffer PE → centrifuge for 1min
discard flow-through
place column in 1,5ml tube
elute with 50Âµl milipore water: incubate 5min at 37Â°C → centrifuge for 1min
pMAD-Transformation for Bacillus
1. Preculture for competent Bacillus subtilis
Aim:Preparation for mainculture the next day
5 mL of LB were inoculated with Bacillus WT3610 from an LB plate and  incubated at 37Â°C overnight.
2. Transformation of competent Bacillus subtilis
Aim: transformation of plasmid into Bacillus subtilis WT3610
100 ÂµL of overnight culture were added to 10  mL of MNGE-Medium and incubated till an OD of 1,1-1,3 at 37Â°C which could take  4-5h.4
After reaching OD of 1,1-1,3 400 ÂµL of the  culture were transformed with 1,5 Âµg plasmid. After 1h incubation at 37Â°C 100  ÂµL Expression mix were added and incubated for 1h as well.
In the end the 500 ÂµL attempt was plated out  on MLS-X-Gal plates and incubated at 30 Â°C overnight until colonies could be  seen.
3. Overnight culture of blue clones
Aim: transformation of plasmid into Bacillus subtilis WT3610
Colonies were grown on the plates with  transformed plasmid. The blue/ white screening showed positive transformed blue  clones.  3 clones of different morphology  per plate were picked and used for inoculation of LB-MLS (4 mL LB, 4 ÂµL  Lincomycin, 4 ÂµL Erythromycin). Incubation was carried out overnight at 30Â°C  with the cultures.
4. First temperature shift
Aim: integration of  pMAD-Insert  into Bacillus chromosome via flanks
The   overnight cultures were used to inoculate 10 mL LB MLS until  the culture obtained an OD of 0,1. The  cultures were incubated at 30Â°C for 2h.
Then the temperature was shifted to 42Â°C for  6h.
After the heat shock dilutions from 10-4  to 10-6 of each culture were plated out on MLS-X-Gal so that plates  could be incubated overnight at 42Â°C.
5. Second temperature shift
Aim: flip out of the pMAD backbone
One blue colony per diluted clone was used to  inoculate 4 mL LB. The cultures were incubated at 30Â°C for 6h and afterwards  for 3h at 42Â°C.
Dilutions from 10-4Â­ to 10-6  were plated out on  X-Gal plates WITHOUT  MLS selection. The positive clones should not contain the resistance inside the  backbone as well as the galactosidase. The plates were incubated at 42Â°C  overnight.
6.1 selection of positive clones
Aim: checking the correct flip out of the pMAD backbone
From the dilution plates was one WHITE clone  picked and transferred on a Master X-Gal Plate as well as on a MLS plate so  that clones were proven for the right integration of the insert although  flipping out the pMAD backbone.
The plates were incubated at 42Â°C overnight.
6.2 selection of positive clones
Aim: checking the correct flip out of the pMAD backbone
The white colonies grew on the X-Gal Master  plates but not on MLS plates so the transformation seemed to be successful. The  Backbone with the MLS resistance flipped out of the genome.
7. cPCR with checked clones
Aim: checking integration of constructs Hag-Spe and Hag-Spe-DARPin into  B. s. genome
No clone was growing on MLS plates so the  picked clones seemed to be positive. In order to check the correct integration  of the domain constructs a cPCR with the picked clones on the X-Gal plates was  done. The picked B.s. clones were cooked in 10 ÂµL PBS for 5 min at 95Â°C.
Making Competent Bacillus with the High Salt/Low Salt Methode
modified after Harwood and Cutting 1990
LB-plates were inoculated with Bacillus subtilies and incubated at 37Â°C overnight.
3ml HS- (high-salt-) medium were inoculated with a single colony and incubated at 37Â°C overnight (rolling).
20ml preheated LS- (low salt-) medium was inoculated with 1ml HS-culture and incubated at 30Â°C for 3h at 100rpm in a water bath.
10 x S-Base
(NH4)2SO4
2g
K2HPO4
14g
KH2PO4
6g
Natriumcitrat* 2 H2O
1g
aqua dest.      ad
100ml
After being autoclaved 0,1ml 1M MgSO4 (sterile filtered) were added
 
HS-Medium
aqua dest.
74,5 ml
10 x S-Base
10 ml
50% Glucose
1 ml
Tryptophan 5 mg/ml
1 ml
Phenylalanin 3 mg/ml
1,5 ml
Casein-Hydrolysat 2 %
1 ml
Yeast-extract 10 %
1 ml
Arginin 8 % + Histidin 0,4 %
10 ml
Solutions were sterile filtered and mixed under sterile conditions
 
LS-Medium
aqua dest.
17,2 ml
10 x S-Base
2 ml
50% Glucose
0,2 ml
Tryptophane 5 mg/ml
20 Âµl
Phenylalanine 3 mg/ml
30 Âµl
Casein-Hydrolysat 2 %
0,1 ml
Yeast-extract 10 %
0,2 ml
Spermin 50 mM
0,2 ml
MgCl2 1M
50 Âµl
Solutions were prepared directly before use and mixed under sterile conditions
 
Transformation
Prepare DNA-Solution (1-5Âµg plasmid DNA or 10-20Âµg chromosomal DNA) in eppendorf cup.
Add 1ml LS-culture, incubate the transformation samples at 37Â°C for 2h an a thermomixer/roller.
Centrifuge the transformation samples fpr 20sec, discard some of the supernatant and resuspend the cells.
Incubate selective plates.
(Harwood, C.R.,  and Cutting, S.M. (1990) Molecular  biological methods for Bacillus.  John Wiley and Sons Ltd., Chichester, England.)
Transformation of Bacillus subtilis with Spizizens minimal medium
10 ml LB  inoculated with single colony as preculture over night at 37Â°C
Inoculate  10 ml Spizizens minimal medium with the preculture to an OD600 of  0.1
Incubation  at 37Â°C until OD600 1 more or less
Addition  of 5 Âµg plasmid DNA/5-10 Âµg of chromosomal DNA to 1 ml of culture (in a test  tube)
Incubation  at 37Â°C for 2 h
Plate out 200 Âµl and 500 Âµl on different plates containing selecting  antibiotics and incubate the plates at 37Â°C overnight
Purification of Ni-NTA column
The following steps were performed for the Ni-NTA Column:
equilibration with Buffer A 10 min
taking 40 ÂµL supernatant  (load)+  10 ÂµL SDS-Buffer - L-sample
50 mL load on column
taking 40 ÂµL of flow through + 10 ÂµL SDS-buffer - FT-sample
first washing with 25ml Buffer A (half the  load)
taking 40 ÂµL of washing flow through + 10 ÂµL SDS-Buffer - W-sample
equilibrate with Buffer B (output pipe off the column into glas, input  into B)
hanging pipe on column, 20 ml Elution - E
taking 40 ÂµL of Elution 1-6 + 10 ÂµL SDS-Buffer E- sample
SDS-PAGE analysis
Regeneration of column
10min water
10 min EDTA
10min water
10min NiSO4
10min water
Splitting of Caco-2-cells
Caco-2-cells require  DMEM (Dulbeccoâ€™s Modified Eagle Medium) with 20% FCS and L-Glutamine.
A centrifuge was set  on 4Â°C.
The medium of the  culture flask was discarded from the side opposite to the adherent cells and  washed with 1X PBS from the opposite site as well.
The PBS was aspirated  with a glass pipette until the supernatant was clear. After adding 1-2 mL 1x  trypsin from the opposite of the adherent cells the flask was incubated for  5-10 min at 37Â°C depending on how fast the cells come of from the ground. Under  the microscope the free floating cells were checked.
The cells were  transferred into a 15 mL falcon and rinsed with 10 mL DMEM to collect all  remaining cells in the flask. The 15 mL falcon was filled with DMEM and the  culture spinned down at 1500 rpm  for 5  min at 4Â°C.
The supernatant was  discarded and resuspended in 2 mL DMEM. The cells were splitted 1:3 which means  that 666 ÂµL were taken from the suspension and transferred into a new culture  flask. After adding 20-25 mL DMEM the flask was incubated at 37Â°C and checked  on the 3rd / 4th day under the microscope.
Purification of PCR Products (QIAquick Gel Extraction Kit) short protocol
short protocol without gel extraction
add 3 volumes or QG Buffer to tubes  and 1 volume isopropanol
place QIAquick spin column in 2ml collection tube
centrifuge for 1min →discard flow-through
add 750Âµl Buffer PE
centrifuge for 1min →discard flow-through
place column in 1,5 ml tube
eluate with 50Âµl milipore water
incubate for 5min at 37Â°C, centrifuge for 1min
Annealing with PNK Inactivation
Step
Temperature °C
Time
1
37
1h
2
65
20 min
3
95
5 min
4
95-4
1 min/1°C
5
4
Infinite
Medium
LB-medium
Concentration
Final
Ampicilin
Ampicilin (1000x) 100Âµl:1000ml
Final: 50µg/ml
Canamycin
Canamycin (1000x) 100Âµl:100ml
Final: 50µg/ml
Chloramphenicol
Chloramphenicol (1000x) 100Âµl:100ml
Buffer
Buffer
Ingredients
-2.0 Buffer
2.0 (10x) + BSA (10mg/ml -10) → BSA end concentration 100µg/ml (0,1µg/µl)
HTP (for 500ml)
10mM Hepes
1,19g
15mM CaCl2 x2H20
1,1g
250mM KCl
9,32g
55mM MnCl2 x 4 H2O
5,44g
Buffer A 20mM HEPES
250mM NaCl
20mM KCl
20 mM MgCl
40 Imidazole
pH 8,0 Buffer B 20mM HEPES
250mM NaCl
20mM KCl
20 mM MgCl
500 mM Imidazole
pH 8,0 GeFi Buffer 2mM HEPES
20mM NaCl
2mM KCl
2 mM MgCl
pH 7,5 Inclusion Body Wash Buffer 100 mM NaCl
50 mM Tris
pH 8.0
0.5% Triton X-100
Microtiter-Assay
1. Grow overnight cultures of B. subtilis in LB (+ antibiotic) or minimal medium S750
2. Prepare medium, cells and Ag+ solutions depending on the assay
Cells: dilutions between 1:100 and 1:1000
Ag+ dilutions: 10 mM to 0.1 nM
3. Add 135 µl Medium in every well of a 96 well plate (Premix of minimal medium and Ag+ dilution)
4. Add 15 µl cells to the medium
5. Add 70 µl mineral oil on top
6.  Strains: As fluorescence is measured always include a positive and negative control
X = Medium (Control without cells)
A-H = Different media
1-9 = Strains
10 = Wildtype (negative control)
11 = GFP-Control
7. After the addition of media, cells and mineral oil it is important to check all wells for bubbles that have tob e removed with pipette tips.
8. Start Plate reader program [120 Cycles]
Shake 4 min
Measurement OD450
Shake 1 min
Measurement GFP
9. Save data
10. Evaluation: First step sort with R
11. Evaluation: Second step excel
Retrieved from " http://2014.igem.org/Team:Marburg:Project:Notebook:Methods "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
