<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Marburg_Project_Silver"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Marburg:Project:Silver">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Marburg:Project:Silver&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Marburg:Project:Silver&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Marburg:Project:Silver&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Marburg:Project:Silver" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Marburg:Project:Silver</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="navbox"><NAV class="menu center first" aria-haspopup="true" aria-controls="p1"><UL id="p1" class="first"><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg" class="a">Home</A></LI><LI class="current"><A href="https://2014.igem.org/Team:Marburg:Project" class="b">Project</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Safety" class="c">Safety</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Team" class="d">Team</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Policy_Practices" class="e">Policy &amp; Practices</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Attributions" class="f">Attributions</A></LI></UL></NAV><NAV class="menu center second" aria-haspopup="true" aria-controls="p1"><UL id="p1" class="second"><LI class="current"><A href="https://2014.igem.org/Team:Marburg:Project:Silver" class="d">SilverSURF</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:Tumor" class="c">CancerSURF</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:NRPS" class="e">RiboSURF</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:Notebook" class="f">Notebook</A></LI><LI class="last" aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:Parts" class="g">Parts</A></LI></UL></NAV></DIV><DIV class="mr_content"><H1>

SilverSURF

</H1><UL id="submenu" style="text-align:left;"><LI><A href="#intro">Introduction</A></LI><LI><A href="#silver">SilverSURF</A></LI><LI><A href="#motility">Motility assays and structural analysis</A></LI></UL><A name="intro"><H2>Introduction</H2></A><P>&quot;Surfing is a surface water sport in which the wave rider, referred to as a surfer, 
rides on the forward or deep face of a moving wave, which is usually carrying the 
surfer towards the shore.&quot; 
</P><P>
In modern science, at the limits of visualization and detection feasibility, the 
amplification of signals is a serious challenge. Scaffolding – building a high 
density of factors in close proximity – is the key to optimize chemical reactions or 
increasing signals for detection. Here we use the bacterial flagellum, a structure provided by nature itself,
as a toolbox for different applications. This huge formation consists 
of the basal body, the hook and the filament, which is composed of up to 20.000 copies of the protein Flagellin (Figure 1). Due to the rotating flagellum, bacteria are able to 
&quot;surf&quot; through their environment.
</P><DIV class="figure" style="width:90%;"><SPAN class="caption"><B>Figure 1: Composition of the bacterial flagellar filament.</B></SPAN></DIV><P>Among bacteria, Flagellin is a protein with highly conserved domains such as D0 and D1, whereas the sections D2 and D3 are extremely variable (Yonekura et al. 2003). 
By comparison of <I>Salmonella</I>, <I>Sphingomonas</I> and <I>Bacillus</I> 
Flagellin it is remarkable that <I>Bacillus</I> D2 and D3 is reduced to a small loop 
(Figure 2; Altegoer et al. 2014).
</P><DIV class="figure" style="width:40%;"><SPAN class="caption"><B>Figure 2: Conserved protein domains of Flagellin among different bacteria.</B></SPAN></DIV><P>This fact was the reason why we took a closer look at the Flagellin of <I>Bacillus subtilis</I>. 
Our vision is to use the bacterial flagellum as a scaffold for high density accumulation of proteins with valuable functions.
</P><P>“SURFing (Synthetic Units for Redirecting Functionalities) is the redirection of
natural occurring structures to a high density surface which can be modified for different 
purposes, towards solving important challenges for humanity e.g. reduction of
environmental pollution and improvement or development of new medical applications (Figure 3).”
</P><DIV class="figure" style="width:85%;"><SPAN class="caption"><B>Figure 3: SURFing application workflow.</B></SPAN></DIV><P>Utilizing the principle of SURF on the basis of bacterial flagella, we decided to contribute to the reduction of 
environmental pollution, concerning heavy metal ions, some of which expose a high toxicity and are harmful even in small doses. A well known representative of these kind of metals is silver (Ag). The toxic effect  is on the one hand industrially welcome, as silver is used as an additive to kill bacteria on the other hand it is a big problem due to its dissemination by industrial waste water. 
Thus the SilverSURF project was born.
</P><H2><A name="silver">SilverSURF</A></H2><P>The purpose of the SilverSURF project is to catch ions from soil or solutions and sequester 
them with a high affinity. How should this be realized? Our vision is to introduce an 
additional modified copy of Flagellin into the genome of <I>Bacillus subtilis</I>. This allele 
should be controlled by a silver ion sensitive promoter. In presence of silver ions 
the modified Flagellin is expressed and incorporated into the filament. This modified 
Flagellin is a chimeric protein with a new domain – the metal binding part of the metallothionein Cup1 from 
yeast which specifically binds silver and copper ions (Moore et al. 2005). This leads 
to engineered <I>B. subtilis</I> cells, that catch the surrounding ions from their environment (Figure 4).
</P><DIV class="figure" style="width:85%;"><SPAN class="caption"><B>Figure 4: SilverSURF vision.</B></SPAN></DIV><P>Many published data about transcriptional changes due to heavy metal ion stress in 
<I>B. subtilis</I>; we were able to find specific promoters, which are more or less specific f
or certain metal ions. To analyze different promoters we 
established a reporter system. Therefore we used the promoter of 
<I>argJ</I> or the <I>arg</I>-operon respectively for sensing silver ions and the promoter of 
<I>ykuO</I> or its operon for the detection of copper ions, by generating a highly 
modular <I>amyE</I> locus integration plasmid with a GFP reporter. By analyzing our reporter 
constructs via STED microscopy,we discovered that the chosen silver sensitive promoter does 
not show any differences by exposure to different silver concentrations compared to our negative 
control without silver ions (Figure 5). However, in principle the established 
promoter reporter system works though. 
</P><DIV class="figure" style="width:70%;"><SPAN class="caption"><B>Figure 5: SilverSURF promoter reporter system with silver concentrations 0, 0.1 µM and 10 µM.</B></SPAN></DIV><P>Nevertheless we focused on the Flagellin Cup1 chimera. We designed 
a modular system for chromosomal integration, where the <I>hag</I> gene, which codes for Flagellin, 
features a <I>Spe</I>I site for fast and modular insertion of domains via Gibson Assembly (pIGEM-0016). 
The Cup1 module was amplified from <I>Saccharomyces cerevisiae</I> genomic DNA and integrated into the pIGEM-0016 system. In the end the wild type
<I>hag</I> was successfully replaced with the chimeric <I>hag-cup1</I> 
construct <A href="http://parts.igem.org/Part:BBa_K1329002" target="_blank">(BBa_K1329002)</A>. 
pMAD allows high efficiency allelic replacement of the wild type <I>hag</I> via the 
pMAD system (Arnaud et al. 2004). 
</P><H2><A name="motility">Motility assays and structural analysis</A></H2><P>As an <I>in vivo</I> functional test of the engineered Flagellin, swimming and swarming 
assays were performed. Unfortunately the assays showed that the engineered Flagellin 
containing strains were strongly reduced in their motility compared to both the wild 
type strain WT3610 and the control strain producing our modularized Flagellin Hag-<I>Spe</I>I <A href="" target="_blank">(BBa_K1329003)</A> 
without the inserted <I>cup1</I> domain. 
</P><P>Anyway, to learned what might disrupt the flagellum functionality. The <A href="http://parts.igem.org/Part:BBa_K1329002" target="_blank"><I>hag-cup1</I></A> was 
overexpressed in <I>Escherichia coli</I>and sufficient amounts of protein could be purified, 
which were subjected to crystallization to solve the structure of the chimeric protein. 
Crystals could be obtained but the structure could only be solved partially, due to the disruption 
of the D1 domain by the <I>cup1</I> integration into <I>hag</I>.
</P><P>These results led us to rethink our approach and we concluded that the Flagellin might be functional if 
we replace the <I>Bacillus</I> loop with the D2 Domain of <I>Salmonella typhimurium</I> (Figure 2) and 
use it as a linker for the Cup1 (Figure 6). This construct is further on called 
<A href="http://parts.igem.org/Part:BBa_K1329002" target="_blank">Hag-D2-Cup1</A>.
</P><DIV class="figure" style="width:1%;"><SPAN class="caption"><B>Figure 6: New engineered synthetic <A href="http://parts.igem.org/Part:BBa_K1329002" target="_blank">Hag-D2-Cup1</A> Flagellin.</B></SPAN></DIV><P>At the same time we developed the idea to insert a StrepTag into the Hag-D2 construct which would provide an enormous
flexibility by coupling a multitude of functional proteins to streptavidin and further on 
build extracellular complexes of Flagellin with chimeric streptavidin proteins (for 
further information see the <A href="https://2014.igem.org/Team:Marburg:Project:Tumor">CancerSURF</A> project).
</P><P>The <A href="http://parts.igem.org/Part:BBa_K1329002" target="_blank">Hag-D2-Cup1</A> and Hag-D2-Strep 
<A href="http://parts.igem.org/Part:BBa_K1329003" target="_blank">(BBa_K1329003)</A> 
were also integrated into <I>B. subtilis</I> via homologous 
recombination by using our modular pMAD system. Again motility assays were 
performed, which showed comparable results for <A href="http://parts.igem.org/Part:BBa_K1329002" target="_blank">Hag-Cup1</A> 
and <A href="http://parts.igem.org/Part:BBa_K1329002" target="_blank">Hag-D2-Cup1</A> though the strain containing 
<A href="http://parts.igem.org/Part:BBa_K1329003" target="_blank">Hag-D2-Strep</A> behaves similar to the wild type in this assay (Figure 7).
</P><DIV class="figure" style="width:85%;"><SPAN class="caption"><B>Figure 7: Motility assays of the <A href="http://parts.igem.org/Part:BBa_K1329002" target="_blank">Hag-D2-Cup1</A> and the 
	<A href="http://parts.igem.org/Part:BBa_K1329003" target="_blank">Hag-D2-Strep</A> mutants.</B></SPAN></DIV><P>We asked ourselves how one construct can show a wild type like phenotype whereas the other doesn't.
Therefore, we visualized the flagella by using electron microscopy
with an uranyl acetate negative staining of our different <I>Bacillus</I> strains (Figure 8). To our surprise 
we could show that the <A href="http://parts.igem.org/Part:BBa_K1329002" target="_blank">Hag-D2-Cup1</A> strain is able to form filaments though they 
are much shorter and inflexible, compared to the wild type flagella, which might 
be the reason why we observed fragmented filaments. Astonishingly the <A href="http://parts.igem.org/Part:BBa_K1329003" target="_blank">Hag-D2-Strep</A> 
filaments looked like wild type filaments. Although we observed less filaments they seemed to be functional. 
What might be the difference between <A href="http://parts.igem.org/Part:BBa_K1329002" target="_blank">Hag-D2-Cup1</A> and 
<A href="http://parts.igem.org/Part:BBa_K1329003" target="_blank">Hag-D2-Strep</A> 
is part of further investigations. A major factor could be the smaller size of the StrepTag, which consists of only eight amino acids, in contrast to the 4.5-fold bigger part of the Cup1 (36 amino acids).
</P><DIV class="figure" style="width:90%;"><SPAN class="caption"><B>Figure 8: Uranylacetate negative stained electron microscopy of <I>B. subtilis</I> (A) and 
	<A href="http://parts.igem.org/Part:BBa_K1329002" target="_blank">Hag-D2-Cup1</A> strain (B).</B></SPAN></DIV><P>The successful engineering of the <A href="http://parts.igem.org/Part:BBa_K1329003" target="_blank">Hag-D2-Strep</A> 
Flagellin is the cornerstone for the success story of our 
CancerSURF project. In further investigations we plan to couple Cup1-Streptavidin, which was successfully produced recombinantly, to isolated 
<A href="http://parts.igem.org/Part:BBa_K1329003" target="_blank">Hag-D2-Strep</A> filaments 
for our SilverSURF – this would directly optimize our approach for pollution reduction. Beyond that, 
there is no need to use the GMO outside the laboratory. The isolated recombinant flagella could be used directly so no major safety concerns would emerge. Also the higher level of modularity allows us to couple every 
imaginable high value applicable protein to our filaments. There is no reason to use only one protein, we 
also plan to use different proteins coupled to streptavidin and bring them into close proximity by using 
the filaments as a scaffold for molecular crowding. There is a huge repertoire of valuable applications for 
this system to solve all kinds of challenges humanity has to face.
</P><P>Altegoer, Florian; Schuhmacher, Jan; Pausch, Patrick; Bange, Gert (2014): From molecular evolution to biobricks and synthetic modules: a lesson by the bacterial flagellum. In: Biotechnology and Genetic Engineering Reviews 30 (1), S. 49–64. DOI: 10.1080/02648725.2014.921500.
</P><P>Arnaud, Maryvonne; Chastanet, Arnaud; Débarbouillé, Michel (2004): New vector for efficient allelic replacement in naturally nontransformable, low-GC-content, gram-positive bacteria. In: Appl. Environ. Microbiol. 70 (11), S. 6887–6891. DOI: 10.1128/AEM.70.11.6887-6891.2004.
</P><P>Moore, Charles M.; Gaballa, Ahmed; Hui, Monica; Ye, Rick W.; Helmann, John D. (2005): Genetic and physiological responses of Bacillus subtilis to metal ion stress. In: Molecular Microbiology 57 (1), S. 27–40, zuletzt geprüft am 16.10.2014.
</P><P>Yonekura, Koji; Maki-Yonekura, Saori; Namba, Keiichi (2003): Complete atomic model of the bacterial flagellar filament by electron cryomicroscopy. In: Nature 424 (6949), S. 643–650. DOI: 10.1038/nature01830.

</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Marburg:Project:Silver">http://2014.igem.org/Team:Marburg:Project:Silver</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Marburg:Project:Silver" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Marburg:Project:Silver" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Marburg:Project:Silver&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Marburg:Project:Silver&amp;oldid=388332" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>