"
Page
Discussion
View source
History
teams
Log in
 
Team:Marburg:Project:Notebook:August
From 2014.igem.org
Home
Project
Safety
Team
Policy & Practices
Attributions
SilverSURF
CancerSURF
RiboSURF
Notebook
Parts
Notebook: August
March
April
May
June
July
August
01.08.2014
04.08.2014
05.08.2014
06.08.2014
11.08.2014
12.08.2014
13.08.2014
14.08.2014
18.08.2014
19.08.2014
20.08.2014
21.08.2014
22.08.2014
23.08.2014
24.08.2014
25.08.2014
26.08.2014
27.08.2014
28.08.2014
29.08.2014
30.08.2014
31.08.2014
September
October
Methods
01.08.2014
18.50 Design of a Strep-Tag/ StrepDARPidin system
Aim: Designing a new system for using the Flagellin-DARPin-Konstrukt
A new plan was made to obtain the flagellas multi-functionality, using an affinity tag. We want to integrate a Strep-Tag into the flagellin so that the DARPin domain can be fused to the flagellum via streptavidin.
The D2 domain of Salmonella typhimurium's flagellin was used as a linker between the hag of Bacillus subtilis and the Strep-Tag, furthermore GSGS linkers were used between the D2 and the StrepTag.
Additionally  the DARPin was designed with an N-terminal His-Tag and C-Terminal streptavidin, so a chimeric protein would be generated.
The designed  structures were synthesized by Integrated DNA Technologies.
15.61 Design of Hag-D2-Cup
Aim: Designing a new domain construct for expression of cup1-1-Hag
We decided to design the flagellin as well with the D2 domain and the cup1-1.
The designed structures were synthesized by Integrated  DNA Technologies
04.08.2014
14.68 Crystallization of Arc1p-C
Aim: Crystallize the Arc1p-C-single domain
For the experiments the sitting drop method in SWISSCI MRC 2 well crystallization plates with a drop size of 1 µL was used (1:1 mixture of protein and crystallization solution) with an reservoir volume of 50 µL. Crystals of Arc1p-C-single were obtained by room temperature and at 6 mg/mL protein concentration after 1 week in 0.1 M MES, pH 6.0, 10% (v/v) glycerol, 30% (w/v) PEG 600 and 5% (w/v) PEG 1000. To collect data the crystals were flash frozen in liquid nitrogen and measured at the European Synchrotron Radiation Facility (ESRF) in Grenoble, France at beamline ID23-1. The structure of Arc1p-C-single was determined by molecular replacement (MR) using the crystal structure of human Arc1p-C (PDB ID: 1FL0).
05.08.2014
18.51 Isolation of flagella from Bacillus subtilis WT3610
Aim: isolation of filaments of WT3610 as negative control for e.g. FACS analysis & microscopy
1 L LB was inoculated with WT 3610 and incubated over night at 37°C.
15.62 Restriction digest of piGEM-005 SpeI
Aim: linearizing vector for Gibson assembly with Hag-D2-Cup and -D2-Strep
Component
Volume (µl)
piGEM-005 (100 ng/µL
20
CutSmart 10x
2,5
SpeI
0,2
Water
2,3
Total Volume
25
Incubation at 37°C for 2 hours – Inactivation by heat shock at 85° for 10  min.
End concentration: 50 ng/µL
18.51 NcoI/BamHI & NcoI/XhoI digest of pET24d
Aim: purification of vector for ligation with StrepDARPidin gene
Component
Volume [µl]
Volume [µl]
pET24d (80 ng/µL)
30
30
CutSmart 10x
3,5
3,5
NcoI
0,25
0,25
BamHI
0,25
-
XhoI
-
0,25
Water
1
1
Total Volume
35
35
Incubation at 37°C for 2h .
The fragments were purified via gel extraction.
End concentration: 25 ng/µL pET24d-N/B & 15 ng/µL-N/X
06.08.2014
18.52 Isolation of flagella from Bacillus subtilis WT3610
Aim: isolation of filaments of WT3610 as negative control for e.g. FACS analysis & microscopy
The preculture  was centrifuged at 4000 rpm for 15 minutes and the pellet was resuspended in 33 mL TRIS/HCl buffer  (pH 8.0) with 0,5% Brij-58. A stock solution of lysozyme with 10 mg/mL was made.  330 µL were added to the TRIS/HCl buffer mix. The lysis was performed at 4°C while rotating the tubes until the pellet was completely solved which took approx. 2 - 3 hours. DNase (5 µg/ mL) in presence of 0,01 M magnesium chloride was added and incubated for  half an hour at 4°C in the rotator as well.
After that the  suspension was centrifuged at 10000x g for 10 min. 80 µL of the supernatant were  transferred into a 1,5 mL tube. The supernatant was then centrifuged at 87000xg  for 90min.
The pellet was  resuspended in 10 mL standard saline citrate (0,01M trisodium citrate & 0,1M sodium chloride, pH 7,3) on ice and 80 µL were taken as well. For fractioning, 2  g of ammonium sulfate (20%) were added slowly until precipitation could be seen.
After some  minutes of resting the suspension was centrifuged at 20000 rpm for 15 min. The  pellet was then resuspended in 2 mL of standard saline citrate buffer. After  taking a sample of 80 µL the 4 samples ware mixed with 20 µL SDS-loading buffer  each and analysed on an SDS-PAGE.
The purified  filaments were frozen in liquid nitrogen and then at -80°C.
The flagella purification was successful. Besides the flagellin the samples also contain the hook proteins, the smaller proteins were removed from the purified pellet.
14.69 tRNAPhe assay
Aim: Measure aminoacylation levels
To measure aminoacylation of tRNAPhe the components were mixed.
Content
Final concentration
Dialysis-buffer
ad 250 µL
DTT
1 mM
MgCl2
20 mM
KCl
10 mM
tRNAPhe
10 µM
L- or D-Phenylalanine
1 mM
catalytic fusion Protein
10 µM
After mixing, reaction was initiated by the addition of 2 mM ATP and then incubated for 1 to 60 min by 37 °C and 350 rpm. Assays were stopped by the addition of 25 µL 3 M sodium acetate solution. Afterwards tRNAs were precipitated with ethanol. Dissolved pellets were further purified using size-exclusion columns. The elution fraction was split into two aliquots. One samples was treated with 0.2 M NaOH (final concentration: 10 mM) and incubated at 50 °C for 10 min, while the other aliquot was kept on ice. The base treated sample was again acidified by the addition of 3 M sodium acetate (pH 5.2, final concentration: 300 mM). The volume of the untreated sample was adjusted using 2 mM sodium acetate (pH 5.2). Subsequently, both samples were subjected to LCMS analysis using 100 µL for injection.
11.08.2014
25. Cell culture assays
25.1 Splitting of Caco-2-cells
Aim: Splitting cell culture for growing
The Caco-2-cells were kindly provided by Dr. Hartmann Reifert (BMFZ). In order to let them grow, the cells had to be  splitted according to a specific protocol. Caco-2-cells require DMEM (Dulbecco’s  Modified Eagle Medium) with 20% FCS and L-Glutamine.
A centrifuge was precooled to 4°C.
The medium of the culture flask was discarded  from the side opposite to the adherent cells and washed with 1X PBS from the  opposite site as well.
The PBS was aspirated with a glass pipette  until the supernatant was clear. After adding 1 - 2 mL 1x trypsin from the  opposite of the adherent cells the flask was incubated for 5 - 10 min at 37°C  depending on how fast the cells unfastened of from the ground. Under the microscope  the free floating cells were checked.
The cells were transferred into a 15 mL falcon  and rinsed with 10 mL DMEM to collect all remaining cells in the flask. The 15  mL falcon was filled with DMEM and the culture centrifuged at 1500 rpm  for 5 min at 4°C.
The supernatant was discarded and resuspended  in 2 mL DMEM. The cells were splitted 1:3 which means that 666 µL were taken  from the suspension and transferred into a new culture flask. After adding  20-25 mL DMEM the flask was incubated at 37°C and checked on the 3rd  / 4th day under the microscope.
On the medium flask were the following notes:
Date, Name, Cell Line, Medium, Ratio of splitting, passage number
15.63 PCR Amplification of Hag-D2-Cup
Aim: Amplification of insert for Gibson assembly into piGEM-005
The synthesized fragments by Integrated DNA technologies arrived as a dried 200 ng DNA pellet. In order to use it as a PCR template the pellet was  centrifuged and resuspended in 20 µL Millipore water to get an end concentration  of 10 ng/µL as a Stock solution.
0,5 µL were used as a PCR  template.
Mix (µl)
Mastermix
D2-Cup
D2-Strep
Template
-
2
2
Primer  Flo96 D23-fw
3
-
-
Primer  Flo97 D23-rv
3
-
-
Q5-Master  mix
75
-
-
Water
63
-
-
Mastermix
-
48
48
Total Volume
144
50
20
Step
Temperature °C
Time
1
98
3 min
2
98
30 sec
3
55
30 sec
4
72
30 sec
5
Go To 2
31x
6
72
4min
7
8
infinite
The expected bands could be seen.
15.64 Gibson assembly with linearized piGEM-005 & PCR products from 15.63
Aim: Insertion of Hag-D2-Cup and Hag-D2-Strep into piGEM-005
Component
Hag-D2-Cup(µL)
Hag-D2-Strep (µL)
Gibson-mix
15
15
Insert I:
 (Hag-D2-Cup 431 ng/µL)
2,5
-
Insert II:
(Hag-D2-Strep 423 ng/µL)
-
2,5
2
piGEM-005 SpeI
 (50 ng/ µL)
2,5
2,5
Total
20
20
Incubation was done  at 50°C for  1 hour with following transformation into E. coli XL1-Blue plated out on LB-Amp plates and incubated overnight at 37°C.
12.08.2014
15.64 Screening clones from 15.63
Aim: Checking insertion of Hag-D2-Cup and Hag-D2-Strep into piGEM-005
In order to check the successful integration of D2-Cup and D2-Strep, clones were picked for cPCR and transferred onto a LB-Amp plate. The primers Flo96 (D23-fw) and Flo97 (D23-v) were used.
Mix (µl)
Mastermix
D2-Cup
D2-Strep
Template
-
Colony
Colony
Primer  Flo96 D23-fw
4,5
-
-
Primer  Flo97 D23-rv
4,5
-
-
dNTP
4,5
-
-
5x Phusion Buffer
36
-
-
Phusion DNA-polymerase
4,5
-
-
Water
126
-
-
Mastermix
-
20
20
Total Volume
180
20
20
Step
Temperature °C
Time
1
98
10 min
2
98
30 sec
3
55
30 sec
4
72
40 sec
5
Go To 2
33x
6
72
4min
7
8
infinite
The gel shows that clone 2 from the Gibson Hag-D2-Cup clones and clones  3 & 4 from the Gibson Hag-D2-Strep clones contain the integrated plasmid  constructs. All 3 clones were picked for a miniprep and used for the inoculation of  10 mL LB-Amp. Incubation was performed over night at 37°C.
The new  constructs were labelled with piGEM-026 (pMAD-Hag-D2-Cup) and piGEM-027  (pMAD-Hag-D2-Strep).
15.65 Overnight culture for WT3610
Aim:Preparation for mainculture the next day
5 mL of LB  were inoculated with Bacillus subtilis WT3610 from an LB plate and incubated at 37°C  over night.
13.08.2014
15.66 Transformation of competent Bacillus subtilis WT3610
Aim: transformation of plasmid into Bacillus subtilis WT3610
100 µL  preculture were added to 10 mL of MNGE-Medium and  incubated to an OD600 of 1,1-1,3 at 37°C which could take 4-5h.
After reaching an OD600 of 1,1-1,3 2 x 400 µL of the culture were transformed  with 1,5 µg plasmid (piGEM-026 & -027). After 1 hour of incubation at 37°C 100 µL  expression mix were added and incubated for 1 hour as well.
In the end the 500 µL attempt was plated out on MLS-X-Gal plates and  incubated at 30 °C overnight until colonies could be seen.
Sequencing
Premix
Label-Nr.
Construct
Primer
1
AGB0023458
piGEM-023
TW260
2
AGB0023459
piGEM-024
TW260
3
AGB0023460
piGEM-025
TW260
4
AGB0023461
piGEM-026 D2-Cup cl.2
Flo96-D2 3 for
5
AGB0023462
piGEM-026 D2-Cup cl.2
Flo56-Hag rv
6
AGB0023463
piGEM-027 D2-Strep cl.3
Flo96-D2 3 for
7
AGB0023464
piGEM-027 D2-Strep cl.3
Flo56-Hag rv
8
AGB0023465
piGEM-027 D2-Strep cl.4
Flo96-D2 3 for
9
AGB0023466
piGEM-027 D2-Strep cl.4
Flo56-Hag rv
13.72 Restriction of Nose plasmids with EcoRV
Aim: check if amyE and cat are still in plasmid, since B. subtilis could not successfully be transformed with the plasmids yet
All Nose-plasmids (iGEM002-004 and 007-015) were digested with EcoRV to check if amyE and cat are still present in these plasmids, since B. subtilis could not be transformed successfully.
Per sample 1 µl 2x CutSmart Buffer, 7.9 µl H2O,  0.1 µl EcoRV and 2 µl of plasmids  were added. The reaction was incubated over night at RT.
14.08.2014
13.72 Restriction of Nose plasmids with EcoRV - continuation
Aim: Analysis of the restriction on an agarose gel
The samples were mixed with 2 µl of 5x Loading dye and 8 µl of it was loaded on a 1% agarose gel.
All plasmids but piGEM004 and piGEM011 show the expected bands of ca 2000 and 4000 bp.
The plasmids were used to transform Bacillus  subtilis, which was incubated at 30°C.
18.08.2014
15.67 Transformation of competent WT3610
Aim: Bacillus subtilis pMad Trafo mit D2-Strep and D2-Cup
Next step: inoculate overnight culture from the X-gal plate with 3 ml LB-MLS.
As the cultures were  recognizable blue (thin or old plate) wildtype cultures were inoculated for a new  transformation. Therefore the plasmids piGEM-026 and -027 were isolated from frozen pellets.
19.08.2014
15.68 Transformation of competent WT3610
Bacillus subtilis pMad trafo mit D2-Strep and D2-Cup
For the experimental procedure see the Materials section. Anyway, two transformations were running currently that will be referred to as 'old' and 'new' transformation from now on.
For the old transformation the first temperature  shift was performed in LB-MLS medium. The culture was plated on LB-MLS-X-Gal plates  and incubated at 42°C over night.
The new transformation was started with a culture  grown in MNGE medium, inoculated with a wildtype overnight culture. The culture  was grown until an optical density of 1,2 that is normally reached after 4-5 h.  Strangely the culture reached this OD after almost 7 h today. Plasmids (1.5µg) piGEM026 and -027 were  transformed and the protocol was followed as described below.
20.08.2014
15.69 Transformation of competent WT3610
Bacillus subtilis pMad Trafo mit D2-Strep and D2-Cup
New trafo showed no colonies so far. It was further incubated.
For the old trafo 2nd temperature shift of positive colonies (only light blue).
21.08.2014
15.70 Transformation of competent WT3610
Bacillus subtilis pMad Trafo mit D2-Strep and D2-Cup
Old trafo: white colonies were re-streaked on x-Gal plates and on MLS plates to check for MLS sensitivity. Incubation at 42°C.
22.08.2014
15.70 Transformation of competent WT3610
Aim: integration of  pMAD-Insert into Bacillus chromosome via flanks
The blue/ white screening showed positive transformed blue clones from  the new transformation.  Three piGEM026 and -027 clones of different morphology  per plate were picked and used for inoculation of LB-MLS (4 mL LB, 4 µL  lincomycin, 4 µL erythromycin). Incubation was carried out over night at 30°C  with the cultures.
22. The Killswitch
22.1 Amplification of flanks
Aim: The flanks have to be amplified for a fusion PCR with the Killswitch modules in order to integrate them into the amyE and the lacA/ganA locus of Bacillus subtilis after cloning into pMAD
Primers arrived and have been dissolved in millipore water, dilution 1:10. Q5 Master Mix was used for the following PCRs to create the flanks of the killswitch parts I-III:
Fragment
Fw primer (iGEM-0xx)
Rv primer (iGEM-0xx)
AmyE flankI
iGEM-034
iGEM-035
AmyE flankII
iGEM-036
iGEM-037
lacA flankI
iGEM-040
iGEM-041
lacA flankII
iGEM-042
iGEM-043
KSIII AmyE-yvyd  FlankII
iGEM-048
-
Mix
Master Mix
amyE FlankI
amyE FlankII
lacA FlankI
lacA FlankII
amyE-yvyD FlankII
Template  (B. subtilis gDNA)
11
-
-
-
-
-
Primer  iGEM-34
-
1
-
-
-
-
Primer  iGEM-35
-
1
-
-
-
-
Primer  iGEM-36
-
-
1
-
-
-
Primer  iGEM-37
-
-
1
-
-
-
Primer  iGEM-40
-
-
-
1
-
-
Primer  iGEM-41
-
-
-
1
-
-
Primer  iGEM-42
-
-
-
-
1
-
Primer  iGEM-43
-
-
-
-
1
-
Primer  iGEM-48
-
-
-
-
-
1
Primer  iGEM-37
-
-
-
-
-
1
Q5-Master  mix
165
-
-
-
-
-
Water
132
-
-
-
-
-
Mastermix
-
28
28
28
28
28
Total Volume [µl]
308
30
30
30
30
30
Step
Temperature °C
Time
1
98
3 min
2
98
30 sec
3
55
30 sec
4
72
60 sec
5
Go To 2
30x
6
72
4min
7
8
infinite
All bands had the expected size (500 /1000 bp)
23.08.2014
15.71 Transformation of competent WT3610
Aim: First heat shock - integration of  pMAD-Insert into Bacillus chromosome via flanks
The  overnight cultures were used to  inoculate 10 mL LB MLS until  the culture  obtained an OD of 0,1. The cultures were incubated at 30°C for 2h.
Then the temperature was shifted to 42°C for 6h.
After the heat shock dilutions from 10-5 to 10-6 of  each culture were plated out on MLS-X-Gal so that plates could be incubated  overnight at 42°C.
24.08.2014
22.2 Amplification of Killswitch modules I-III
Aim: The modules had to be amplified via PCR in order to proceed with a fusion PCR linking the modules with the flanks for integration into the Bacillus Loci.
The synthesized fragments by Integrated DNA technologies arrived as a dried 200 ng DNA pellet. In order to use it as a PCR template the pellet was  centrifuged down and resuspended in 20 µL millipore water to get an end concentration of 10 ng/µL as a stock solution.
0,5  µL were used as a PCR  template. The PCR fragments should be ca. 350, 650 & 2000 bp long.
Mix
KS module I
KS module II
KS module III
Template (gBlock DNA)
0,5
0,5
0,5
Primer  iGEM-38
1
-
-
Primer  iGEM-39
1
-
-
Primer  iGEM-44
-
1
-
Primer  iGEM-45
-
1
-
Primer  iGEM-49
-
-
1
Primer  iGEM-50
-
-
1
Q5-Mastermix
25
25
25
Water
22,5
22,5
22,5
Total Volume (µl)
50
50
50
Step
Temperature °C
Time
KSI & III
Time
KSII
1
98
3 min
3 min
2
98
20 sec
30 sec
3
55
20 sec
20 sec
4
72
60 sec
120 sec
5
Go To 2
31x
31x
6
q
72
4min
4min
7
8
infinite
infinite
The modules I and III could be amplified without problems, but KSII showed no distinct band and many unspecific fragments.
15.71 Transformation of competent WT3610
Aim: Second heat shock -flip out of the pMAD backbone
One blue colony per diluted piGEM026  and -027 clone was used to inoculate 4 mL LB. The cultures were incubated at 30°C  for 6 hours and afterwards for 3 hours at 42°C.
Dilutions from 10-5 to 10-6 were plated out on  X-Gal plates without MLS selection. The  positive clones should not contain the resistance inside the backbone as well  as the galactosidase. The plates were incubated at 42°C overnight. Some clones  (piGEM027.1 and one of 27.2) did not grow and will be inoculated again. They  will be inoculated in LB-MLS for the first temperature shift tomorrow.
22.3 Transformation of competent DH5α with pMAD
Aim: Gain E.coli on plate for mini preps of pMAD
Standard heat shock transformation protocol was  followed, incubation over night at 37°C. pMAD from the plasmid box was used for  transformation.
22.4 Restriction of pMAD with NcoI and BamHI over night
Aim: Linearizing Vector for Gibson Assembly with KS modules
Component
Volume (µl)
pMAD (108 ng/µL)
9
CutSmart 10x
2
NcoI
0,2
BamHI
0,2
Water
8,6
Total Volume
20
Incubation over night at room temperature.
25.08.2014
15.72 Transformation of competent WT3610
Aim: checking the correct flip out of the pMAD backbone and MLS sensitivity
From the dilution plates was one white clone picked and transferred on a  X-Gal plate as well as on a MLS plate so that clones were proven for the  right integration of the insert although flipping out the pMAD backbone.
The plates were incubated at 42°C overnight.
22.5a Fusion PCR
Aim: Assembly of Killswitch modules I & II with amyE and lacA flanks
Segment
Concentraction  (ng/µL)
KS module I
401
KS module II
414
KS module III
369
amyE flank I
446
amyE flank II
403
lacA flank I
368
lacA flank II
369
amyE yvyD flank II
372
Mix
amyE KS I
lacA KS II
KS  module I (40,1 ng/µL)
1
-
KS  module II (41,4 ng/µL)
-
1
amyE flank I ( 44,6 ng/µL)
1
-
amyE flank II (40,3 ng/µL)
1
-
lacA flank I (36,8 ng/µL)
-
1
lacA flank II (39,6 ng/µL)
-
1
Primer  iGEM-34
1
-
Primer  iGEM-37
1
-
Primer  iGEM-40
-
1
Primer  iGEM-43
-
1
Phusion  Buffer 5x
10
10
Phusion DNA-polymerase
1
1
dNTPs
1
1
Water
33
33
Total Volume [µl]
50
50
Step
Temperature °C
Time
amyE KSI
Time
amyE KSII
1
98
3 min
3 min
2
98
20 sec
30 sec
3
55
20 sec
20 sec
4
72
120 sec
190 sec
5
Go To 2
31x
31x
6
q
72
4min
4min
7
8
infinite
infinite
No PCR amplificates could be seen on the gel.
18.53 PCR amplification of StrepDARPidin
Aim: amplification of the StrepDARPidin for cloning into the expression vector pET24d for protein purification and overproduction in E. coli BL21(DE3)
The synthesized fragments by Integrated DNA technologies arrived as a dried  200n g DNA pellet. In order to use it as a PCR template the pellet was  centrifuged and resuspended in 20 µL millipore water to get an end concentration  of 10 ng/µL as a stock solution.
0,5  µL were used as a PCR  template. The PCR fragment should be ca. 900 bp long.
Mix
PCR attempt
Template
0,5
Primer iGEM-33
1
Primer iGEM-34
1
Q5-Mastermix
25
Water
22,5
Mastermix
-
Total Volume
50
Step
Temperature °C
Time
1
98
3 min
2
98
30 sec
3
55
30 sec
4
72
60 sec
5
Go To 2
31x
6
72
4min
7
8
infinite
Both samples had a  concentration of 380 ng/µL.
18.54 Digestion of amplified of StrepDARPidin for cloning into pET24d
Aim: digest with NcoI and XhoI of the StrepDARPidin PCR product for cloning into the expression vector pet24d
The PCR product was cut with NcoI and XhoI to get the restriction sites for cloning into pET24d NcoI/XhoI cut.
Component
StrepDARPidin (µl)
PCR product (380 ng/µL)
6
CutSmart 10x
2
NcoI
0,25
XhoI
0,25
Water
11,5
Total Volume
20
Incubation for 1h at  37°C with heat shock at 85°C for 10 min.
18.55 Ligation of digested StrepDARPidin for cloning into pET24d
Aim: ligation of StrepDARPidin PCR product with the expression vector pET24d
The PCR product was ligated into digested pET24d NcoI/XhoI. Two samples were made for a transformation of E. coli XL1-Blue and BL21.
Component
Ligation I (µl)
Ligation II (µl)
NcoI and XhoI StrepDARPidin (64 ng/µL)
2
2
pET24d NcoI and XhoI (15  ng/µL)
5
5
Ligation Buffer 10x
2
2
Ligase
1
1
Water
10
10
Total Volume
20
20
Incubation for 1,5h at room temperature with heat shock at 85°C for 10 min. The 20 µL from ligation I were used to transform E. coli XLI-Blue, 10 µL from Ligation II were used to transform E. coli BL21(DE3) and the rest of ligation II into  Xl1-Blue as well and selected on  LB-Canamycin plates.
26.08.2014
15.73 Transformation of competent WT3610
Aim: checking the correct flip out of the pMAD backbone and MLS sensitivity
All clones on the master plate grew on x-gal  plates but were MLS sensitive and could be seen as positive. In order to check  the correct insertion of D2-Cup an D2-Strep into the Bacillus subtilis genome  the clones were cooked in 100 µL 1x PBS at 95°C for 10 min and used for colony  PCR. Because of the repetition of the temperature shifts with clones 27.1.1,  27.1.2, 27.1.3 yesterday and today and 27.2.3, 6 clones from 26.1 -26.3, 27.2.1 & 27.2.2 were picked for the cPCR (30 reactions).
Mix (µl)
Master-Mix for 30 attempts (µL)
Mix for PCR attempt (µL)
Colony PBS suspension
-
2
Primer Flo54 Hag-fw
15
-
Primer  Flo56 Hag-fw
15
-
Phusion DNA-polymerase
15
-
Phusion Buffer 5x
120
-
dNTP-mix
15
-
Water
410
-
DMSO
10
-
Mastermix
-
18
Total Volume
600
20
Step
Temperature °C
Time
1
98
3 min
2
98
30 sec
3
55
30 sec
4
72
1 min 30 sec
5
Go To 2
30x
6
72
4min
7
8
infinite
There were no bands visible on the gel.
18.56 cPCR with plates from ligation
Aim: checking the success of the ligation
The grown clones from the ligation the day before were inoculated for plasmid isolation and then checked via cPCR with primers iGEM-032 and -033.
Content (µl)
Master mix  for  20 clones (µL)
Clones from Ligation I (µL)
Clones from Ligation II (µL)
Clones from Ligation III (µL)
Template (colony)
-
1
1
1
Primer iGEM-32
10
-
-
-
Primer iGEM-32
10
-
-
-
Phusion Buffer 5x
80
-
-
-
Phusion DNA-polymerase
10
-
-
-
dNTP-mix
10
-
-
-
Water
280
-
-
-
Mastermix
-
19
19
19
Total Volume
400
20
20
20
Step
Temperature °C
Time
1
98
10 min
2
98
20 sec
3
55
20 sec
4
72
60 sec
5
Go To 2
33x
6
72
4min
7
8
infinite
The gel shows that the clones I  2-6, II 1-6 and III 2, 3, 4 and 6 are positive due to the band at approx. 900 bp.  The other bands might be plasmid which was used as template.
The clones I6, II3, III3 were used to transform E. coli BL21(DE3) and selected on LB-Can plates. III4 and III6  were already on a plate. The clones were inoculated for plasmid isolation and sent for sequencing.
22.5b Fusion PCR Attempt 3
Aim: Fusion of amyE and lacA flanks to the killswitch fragments
The Fusion PCR was repeated since  it did not work the first two times. The fragments of the killswitch I and II  were diluted 10fold. A gradient was carried out from 52°C to 62°C.
Mix (µl)
amyE KSI
lacA KS II
KS module I (40,1 ng/µL)
1
-
KS module II (41,4 ng/µL)
-
1
amyE flank I ( 44,6 ng/µL)
1
-
amyE flank II (40,3 ng/µL)
1
-
lacA flank I (36,8 ng/µL)
-
1
lacA flank II (39,6 ng/µL)
-
1
Primer iGEM-34
1
-
Primer iGEM-37
1
-
Primer iGEM-40
-
1
Primer iGEM-43
-
1
Phusion Buffer 5x
4
4
Phusion DNA-polymerase
0,5
0,5
dNTPs
1
1
Water
9,5
9,5
Total Volume
20
20
Step
Temperature °C
Time
1
95
5 min
2
95
30 sec
3
52-62
30 sec
4
72
200 sec
5
Go To 2
32x
6
72
5 min
7
4
infinite
Again, no fragments could be noticed on the gel.
13.73 Amplification of new Cu- and Ag-Promotor sequences
Aim: Amplification of Cu- and Ag promoter sequences from the chromosomal DNA of Bacillus subtilis.
Content
Master mix  (µL)
2x Attempt Cu (µL)
2x Attempt Ag (µL)
Template (gDNA B. subtilis)
4
-
-
Primer iGEM-55
-
-
1
Primer iGEM-56
-
-
1
Primer iGEM-57
-
1
-
Primer iGEM-58
-
1
-
Phusion Buffer 5x
40
-
-
Phusion DNA-polymerase
4
-
-
dNTPs
4
-
-
Water
144
-
-
Mastermix
-
48
48
Total Volume (µl)
196
50
50
Step
Temperature °C
Time
1
98
10 min
2
98
20 sec
3
55
20 sec
4
72
20 sec
5
Go To 2
30x
6
72
4 min
7
8
infinite
The gel shows positive bands at approx. 100 bp and 250 bp, which was like expected.
13.74 NcoI/SacI digest of piGEM-002
Aim: linearizing the plasmid backbone for Gibson Assembly and Ligation with Cu/Ag promotor sequences and IPTG/ Constitutive (/tetO) promotor sequences.
Component
Attempt I (µl)
piGEM-002 (476 ng/µL)
6,5
NcoI
0,25
SacI
0,25
CutSmart Buffer 10x
2
Water
11
Total Volume
20
Incubation at 37°C for  60 min and heat shocked at 85°C for 10 min.
13.75 Ligation of NcoI/Sac cut piGEM-002 with annealed oligos containing promoter sequences
Aim: insertion of a IPTG inducible/ constitutive and constitutive+tetO into piGEM-002
In order to test our nose system with new promoter sequences oligos were designed and annealed by heating up water and letting it cool down with the oligo pairs iGEM-59/60, -61/62, - 63/64 so that the individual fragments were able to anneal with overlaps to the NcoI/SacI sites.
Component
Ligation I (µl)
Ligation II (µl)
Ligation III (µl)
piGEM-002 (209 ng/µL)
5
5
5
iGEM-59/60
1
-
-
iGEM-61/62
-
1
-
iGEM-63/64
-
-
1
T4 Ligase
1
1
1
Ligation Buffer 10x
2
2
2
Water
11
11
11
Total Volume (µl]
20
20
20
The reactions were  incubated at room temperature for an hour and transformed into E. coli XL1-Blue, plated out on LB-Amp.
13.76 Gibson assembly of NcoI/SacI piGEM-002 & Cu/ Ag promotor
Aim: assembling of Nco/Sac cut piGEM-002 with Cu/ ag promotor sequences
In order to test new Cu/ Ag promotor sequences the PCR fragments from 13.73 were used for a Gibson assembly.
Component
Gibson Cu I (µL)
Gibson Ag II (µL)
Control  (µL)
piGEM-002 (209 ng/µL)
2,5
2,5
5
Cu promotor
2,5
-
-
Ag promotor
-
2,5
-
Gibson mix
15
15
15
Total Volume
20
20
20
The attempts were  incubated at 50°C for an hour and transformed into E. coli XL1-Blue, plated out on LB-Amp.
27.08.2014
22.3b Repeated Amplification of Killswitch modules I and II
Aim: Increase the amount of modules for further purification, since Fusion PCR did not work 'quick and dirty'
This time the Phusion polymerase was used. The template was the already amplified module.
Mix
KS module I
KS module II
Template (already amplified modules)
0,5
0,5
Primer iGEM-38
1
-
Primer iGEM-39
1
-
Primer iGEM-44
-
1
Primer iGEM-45
-
1
5x HF buffer
10
10
dNTPs
1
1
Water
36
36
Phusion DNA-polymerase
0,5
0,5
Total Volume (µl)
50
50
Step
Temperature °C
 KS I
KS II
1
98
5 min
5 min
2
98
30 sec
30 sec
3
55
30 sec
30 sec
4
72
120 sec
200 sec
5
Go To 2
34x
34x
6
72
5 min
5 min
7
8
infinite
infinite
The gel can be seen under the next topic
22.3c Repeated amplification of killswitch flanks
Aim: The flanks have to be amplified for a fusion PCR with the Killswitch modules in order to integrate them into the amyE and the lacA locus of Bacillus subtilis after cloning into pMAD, yield higher amount of fragments for purification and FusionPCR.
The Phusion polymerase was used.
Mix
Master Mix
amyE FlankI
amyE FlankII
lacA FlankI
lacA FlankII
amyE-yvyD FlankII
Template
 
0,5
-
-
-
-
Primer iGEM-34
-
1
-
-
-
-
Primer iGEM-35
-
1
-
-
-
-
Primer iGEM-36
-
-
1
-
-
-
Primer iGEM-37
-
-
1
-
-
-
Primer iGEM-40
-
-
-
1
-
-
Primer iGEM-41
-
-
-
1
-
-
Primer iGEM-42
-
-
-
-
1
-
Primer iGEM-43
-
-
-
-
1
-
Primer iGEM-48
-
-
-
-
-
1
Primer iGEM-37
-
-
-
-
-
1
dNTPs
11
1
-
-
-
-
DMSO
3,3
0,3
 
 
 
 
MgCl2
2,2
0,2
 
 
 
 
5x HF-buffer
110
10
 
 
 
 
Phusion DNA-polymerase
5,5
0,5
 
 
 
 
Water
390,5
35,5
 
 
 
 
Mastermix
-
47,5
47,5
47,5
47,5
47,5
Total Volume (µl)
522,5
50
50
50
50
50
With exception of KSI and StrepDARP the PCR was negative for all other samples. Even for the positive reactions the bands were very thin.
18.57 repeated PCR amplification of Hag-KpnI StrepDARPidin
Aim: amplification of the StrepDARPidin for cloning into the expression vector pET24d for protein purification and overproduction in E. coli BL21(DE3)
The synthesized fragments by Integrated DNA technologies arrived as a dried  200 ng DNA pellet. In order to use it as a PCR template the pellet was  centrifuged and resuspended in 20 µL millipore water to get an end concentration  of 10 ng/µL as a stock solution.
0,5  µL were used as a PCR  template. The PCR fragment should be approx. 900 bp long.
Content (µl)
2x Attempt I (µL)
2x Attempt II(µL)
Template 1: StrepDARPidin (38 ng/ µL)
-
1
Template 2: Hag-KpnI (10 ng/ µL)
1
-
Primer iGEM-51
-
1
Primer iGEM-52
-
1
Primer iGEM-53
1
-
Primer iGEM-54
1
-
Phusion Buffer 5x
10
10
Phusion DNA-polymerase
1
1
dNTPs
1
1
Water
35
35
Total Volume
50
50
Step
Temperature °C
Time
1
98
3 min
2
98
30 sec
3
55
30 sec
4
72
60 sec
5
Go To 2
31x
6
72
4min
7
8
infinite
The PCR was clearly positive for StrepDARP, Hag-KpnI could be seen as a very slight band at 900 bp, but more unspecific bands could be noticed.
18.56 Expression-Test with transformed E. coli BL21(DE3)
Aim: Checking the overexpression of pET24d-StrepDARPidin
In  order to check the expression of the proteins 5 clones (I6, II3, III3, III4,  III6) containing piGEM-028 (pet24d-StrepDARPidin) were used to inoculate 20 mL  of LB-Can. 5 cultures were incubated at 37°C until an OD of 0,7.
A ca. 1  mL preinduction sample (PI) was taken (0,7 (favoured OD) : 0,7 (actual OD) = 1  mL). After that the cultures were induced with 20 µL IPTG for 2h.
An  induction sample (I) was taken (320 µL with an OD of 2,2; 350 µL with an OD of  2).
PI and  I were centrifuged at 14000 rpm for 1 min, the supernatant was discarded and  the pellet resuspended in 60 µL water and 40 µL SDS-Buffer.
The  four samples were analysed on an SDS-PAGE gel with 10 µL volume per sample.
The gel  did not show any signs of over expression so that lactose was chosen as an alternative inducer to test  the expression. New test expression cultures (I6, II3, III3, III4, III6)  were made with 100 mL LB-Can and 5 mL lactose  (6,25 g/ 25 mL) per culture. Incubation was performed over night at 30°C.
28.08.2014
Sequencing of constructs from 28.08.2014
Premix
Label-Nr.
Construct
Primer
1
AGB0023467
piGEM-028 I6(pET24d-StrepDARPidin)
T7 turn
2
AGB0023468
piGEM-028 III3  (pET24d-StrepDARPidin)
T7 turn
3
AGB0023469
piGEM-026  (pMAD-D2-Cup)
Flo D2_3 fw
4
AGB0023470
piGEM-027.1  (pMAD-D2-Strep)
Flo D2_3 fw
5
AGB0023471
piGEM-027.2  (pMAD-D2-Strep)
Flo D2_3 fw
18.56 Expression-Test with transformed E. coli BL21(DE3)
Aim: Checking the overexpression of pET24d-StrepDARPidin
No sign of an overexpression could be noticed.
18.57 His-bead purification of pellet from E. coli BL21(DE3)
Aim: Checking the overexpression of pET24d-StrepDARPidin
The 100 mL culture from I6 & III3 were centrifuged at 4000 rpm for 10 min in 2x50 mL falcons. The pellets  were washed in 10 mL buffer A and cracked with the microfluidizer. After that  the suspension was centrifuged at 20000 rpm for 20 min/ 4°C. The supernatant  was transferred into a 50 mL falcon.
200 µL Ni-NTA beads were added to  the suspension, inverted and incubated on ice for half an hour.
After that the suspension was  centrifuged or 5 min at 4000 rpm so that the pellet could be resuspended in 500 µL Buffer A after discarding the supernatant. The 500 µL were transferred into a 1,5 mL tube and centrifuged at 4000 rpm for washing. After discarding the  supernatant the pellet was washed again with 500 µL Buffer A. After a second  centrifugation at 4000 rpm the supernatant was discarded, the pellet  resuspended in 200 µL Buffer A and centrifuged at 13000 rpm to destroy the  Ni-bead – protein interaction.
In the end the supernatant was  transferred into a new 1,5 mL tube. The Ni-beads were mixed with 100 µL 20%  ethanol.
40 µL of the supernatant from  clone I6 and III3 were added with 10 µL SDS-PAGE Buffer and analysed on the  SDS-Gel (visible at 18.56).
22.3d Repeated amplification of killswitch flanks
Aim: The flanks have to be amplified for a fusion PCR with the Killswitch modules in order to integrate them into the amyE and the lacA locus of Bacillus subtilis after cloning into pMAD, yield higher amount of fragments for purification and FusionPCR.
Mix [µl]
Master Mix 6x
amyE FlankI
amyE FlankII
lacA FlankI
lacA FlankII
amyE-yvyD FlankII
Template (B. subtilis gDNA)
6
-
-
-
-
-
Primer iGEM-34
-
1
-
-
-
-
Primer iGEM-35
-
1
-
-
-
-
Primer iGEM-36
-
-
1
-
-
-
Primer iGEM-37
-
-
1
-
-
-
Primer iGEM-40
-
-
-
1
-
-
Primer iGEM-41
-
-
-
1
-
-
Primer iGEM-42
-
-
-
-
1
-
Primer iGEM-43
-
-
-
-
1
-
Primer iGEM-48
-
-
-
-
-
1
Primer iGEM-37
-
-
-
-
-
1
Q5-Master mix
150
-
-
-
-
-
Water
132
-
-
-
-
-
Mastermix
-
48
48
48
48
48
Total Volume
288
50
50
50
50
50
Step
Temperature °C
Time
1
98
5 min
2
98
25 sec
3
55
25 sec
4
72
60 sec
5
Go To 2
31x
6
72
4 min
7
8
infinite
The gel is visible under 22.3e
22.3e Repeated Amplification of Killswitch modules I and II
Aim: Increase the amount of modules for further purification, since Fusion PCR did not work 'quick and dirty'. Q5 was used and gDNA since the previously amplified modules did not work as template yesterday.
Mix [µl]
KS module I
KS module II
KS module III
Template (gBlock DNA)
0,5
0,5
0,5
Primer iGEM-38
1
-
-
Primer iGEM-39
1
-
-
Primer iGEM-44
-
1
-
Primer iGEM-45
-
1
-
Primer iGEM-49
-
-
1
Primer iGEM-50
-
-
1
Q5-Master mix
25
25
25
Water
2,5
2,5
2,5
Total Volume
50
50
50
Step
Temperature °C
 KS I & III
KS II
1
98
5 min
5 min
2
98
25 sec
25 sec
3
55
25 sec
25 sec
4
72
60 sec
120 sec
5
Go To 2
31x
31x
6
72
4 min
4 min
7
8
infinite
infinite
The expected bands could be seen for the flank I of amyE and flank I and II of lacA. These bands were cut out and purified via a Gel Extraction kit, but the concentration was too low to work further.
13.75 NcoI/SacI digest of piGEM-002
Aim: linearizing the plasmid backbone for Gibson Assembly and Ligation with Cu/Ag promotor sequences and IPTG/ Constitutive (/tetO) promotor sequences.
The already digested piGEM002 of 26.08. was nearly empty. To have some digested plasmid as backup a new digestion was carried out. piGEM002 was isolated and used for the restriction.
Component
Attempt (µl)
piGEM-002 (476 ng/µL)
10
NcoI
0,25
SacI
0,25
CutSmart Buffer 10x
2
Water
7,5
Total Volume
20
An agarose gel of the previous restriction was done with unrestricted piGEM002 as reference. A band at ca between 6000 and 8000 bp could be seen, which indicates a successful restriction (expected size: ca 6600 bp).
13.76 Repeated Gibson assembly of NcoI/SacI piGEM-002 & Cu/ Ag promotor
Aim: assembling of NcoI/SacI cut piGEM-002 with Cu/Ag promoter sequences
In order to test new Cu/ Ag promoter sequences the PCR fragments from 13.73 were used for a Gibson assembly.
Component
Gibson Cu I (µL)
Gibson Ag II (µL)
Control  (µL)
piGEM-002 (ca.  40ng/µL)
2,5
2,5
2,5
Cu promotor (ca 4 ng/µl)
2,5
-
-
Ag promotor (ca 4 ng/µl)
-
2,5
-
H2O
-
-
-
Gibson Mix
15
15
15
Total Volume
20
20
20
The attempts were  incubated at 50°C for an hour and used to transform E. coli XL1-Blue, plated out on LB-Amp.
13.77 Repeated ligation of NcoI/SacI cut piGEM-002 with annealed oligos containing promoter sequences
Aim: insertion of an IPTG inducible/ constitutive and constitutive+tetO into piGEM-002
The ligation was prepared to contain ca 100 ng of the restricted piGEM002 with an appropriate amount of insert (much fewer).
Component
Ligation I (µL)
Ligation II (µL)
Ligation III (µL)
piGEM-002 (ca 40  ng/µL)
2,5
2,5
2,5
iGEM-59/60 (ca 4 ng/µl)
0,8
-
-
iGEM-61/62 (ca 4 ng/µl)
-
0,8
-
iGEM-63/64 (ca 4 ng/µl)
-
-
0,8
Polynucleotide kinase
1
1
1
T4 Ligase
1
1
1
Ligation Buffer 10x
2
2
2
Water
12,7
12,7
12,7
Total Volume
20
20
20
The reactions were  incubated at room temperature for an hour and transformed into E. coli XL1-Blue, plated out on LB-Amp.
29.08.2014
Sequencing results of constructs from 28.08.2014
Premix
Label-Nr.
Construct
Results
1
AGB0023467
piGEM-028  I6(pET24d-StrepDARPidin)
No results
2
AGB0023468
piGEM-028 III3  (pET24d-StrepDARPidin)
No alignment
3
AGB0023469
piGEM-026  (pMAD-D2-Cup)
Good
4
AGB0023470
piGEM-027.1  (pMAD-D2-Strep)
Point mutation
5
AGB0023471
piGEM-027.2  (pMAD-D2-Strep)
Point mutation
The sequencing showed that pET24d did not contain any known insert.
In  order to check the correct synthesis of the templates purified PCR products  were sent for sequencing again. In the meanwhile the constructs were cloned again.
Sequencing of constructs from 29.08.2014
Premix
Label-Nr.
Construct
Primer
1
AGB0023512
PCR Fragment  StrepDARPidin I
piGEM-032 fw
2
AGB0023513
PCR Fragment  StrepDARPidin II
piGEM-032 fw
3
AGB0023514
PCR Fragment D2-StrepII
Flo96 D2_3 fw
4
AGB0023515
PCR Fragment  StrepDARPidin I
piGEM-033 rv
5
AGB0023516
PCR Fragment  StrepDARPidin II
piGEM-033 rv
6
AGB0023517
PCR Fragment D2-StrepII
Flo97 D2_3 rv
18.58 New PCR amplification of StrepDARPidin and D2-Cup
Aim: amplification of the StrepDARPidin for cloning into the expression vector pET24d for protein purification and overproduction in E. coli BL21(DE3) & amplification of D2-StrepII for Gibson Assembly into piGEM-005
The synthesized fragments were supposed to be sequenced in order to  check the right synthesis.  10 ng/µL as a  Stock solution and a purified PCR Product were used as template for each  reaction.
0,5  µL were used as a PCR  template. The PCR fragment should be ca. 900 bp long for StrepDARPidin and ca.  450 bp for D2-StrepII.
Mix (µl)
PCR StrepDARPidin I
PCR StrepDARPidin II
PCR StrepDARPidin III
PCR D2-Strep I
PCR D2-Strep II
Pur. StrepDARP. (1:10- 10 ng/µL)
1
-
-
-
-
StrepDARP. Stock (20 ng/µL)
-
0,25
0,25
-
-
D2-StrepII Stock 20 ng/µL)
-
-
-
0,25
0,25
Primer iGEM-32
1
1
1
-
-
Primer iGEM-33
1
1
1
-
-
Flo96 D2_3 fw
-
-
-
1
1
Flo97 D2_3 rv
-
-
-
1
1
Q5-Master mix
25
25
25
25
25
Water
22
22
22
22
22
Total Volume (µl)
50
50
50
50
50
Step
Temperature °C
Time
1
98
5 min
2
98
30 sec
3
55
30 sec
4
72
60sec
5
Go To 2
31x
6
72
4min
7
8
infinite
The PCR samples were purified with the Omega Gel Extraction Kit according to the enzymatic reaction purification protocol.
18.59 Digestion of amplified of StrepDARPidin for cloning into pET16b
Aim: digest with NcoI and XhoI of the StrepDARPidin PCR product for cloning into the expression vector pET16b & digest of pET16b
The PCR products from 15.58 were cut with NcoI and XhoI to get the restriction sites for cloning into the NcoI/XhoI  digested pET16b. This vector was used because of its Amp resistance for fast transformation.
Component
PCR StrepDARPidin I (µL)
PCR StrepDARPidin II(µL)
PCR StrepDARPidin III (µL)
Pet16b(µL)
Mastermix
StrepDARP. I ( 45 ng/µL)
10
-
-
-
-
StrepDARP. II (70 ng/µL)
-
15
-
-
-
StrepDARP. III (45 ng/µL)
-
-
10
-
-
pET16b ( 58 ng/µL)
-
-
-
17
-
CutSmart 10x
-
-
-
-
8
NcoI
-
-
-
-
0,5
XhoI
-
-
-
-
0,5
Water
-
-
-
-
17
Mastermix
10
5
10
-
-
Total Volume
20
20
20
20
28
Incubation for 1h at  37°C with heat shock at 85°C for 10 min.
18.60 Ligation of digested StrepDARPidin for cloning into pET16b
Aim: ligation of StrepDARPidin PCR product with the expression vector pET16b
The PCR product was ligated into the NcoI/XhoI digested pET16b. Two attempts were made for a transformation into E. Coli  XL1-Blue and BL21.
Component
Ligation I (µL)
Ligation II (µL)
Ligation III (µL)
Control
NcoI/XhoI StrepDARPidin  (ca. 40  ng/ µL) I/II/III
1
1
1
-
pET16b NcoI/XhoI (45 ng/µL)
5
5
5
5
Ligation Buffer 10x
2
2
2
2
Ligase
1
1
1
1
Water
11
11
11
11
Total Volume
20
20
20
20
Incubation for 1,5 hours at  rom temperature with heat shock at 85°C for 10 min. The 20 µL from ligation I  were used to transform E. coli XLI-Blue, which was plated out on LB-Amp. Incubation was  done overnight at 37°C.
18.61 Gibson assembly of piGEM-005 & D2-StrepII
Aim: Insertion of Hag-D2-Strep into piGEM-005
Component
Hag-D2-Strep I(µL)
Hag-D2-Strep II (µL)
Gibson-mix
15
15
Insert I
(Hag-D2-Strep I 60  ng/µL)
2,5
-
Insert II
(Hag-D2-Strep II 70  ng/µL)
-
2,5
piGEM-005 SpeI (50 ng/ µL)
2,5
2,5
Total Volume
20
20
Incubation was done  at 50°C for  1 hour with following transformation into E. coli XL1-Blue, plated out on LB-Amp plates and incubated overnight at 37°C.
13.78 colony PCR of colonies of Gibson and ligation (13.76 and 13.77)
Aim: check for correct ligation/Gibson assembly of the oligos with the piGEM002 plasmid
Several clones were picked and transferred onto another plate, the same colony was used to inoculate a PCR reaction to check the insertion of the oligos.
Mix [µl]
Master mix for 45 reactions  (µl)
Const. promoter
Cont. + tetO
Mod. Lac promoter
Ag promoter new
Cu promoter new
Template
-
Small part
Small part
Small part
Small part
Small part
Primer iGEM006
45
-
-
-
-
-
Primer iGEM061
-
1
-
-
-
-
Primer iGEM063
-
-
1
-
-
-
Primer iGEM059
-
-
-
1
-
-
Primer iGEM057
-
-
-
-
1
-
Primer iGEM055
-
-
-
-
-
1
5x HF buffer
180
-
-
-
-
-
Phusion DNA polymerase 1:10
45
-
-
-
-
-
Water
585
-
-
-
-
-
Master Mix
-
19
19
19
19
19
Total Volume
855
20
20
20
20
20
Step
Temperature °C
Time
1
95
10 min
2
98
30 sec
3
55
30 sec
4
72
60 sec
5
Go To 2
34x
6
72
5 min
7
8
infinite
The positive clones (band at ca 1000 bp) were  transferred into LB-amp medium and were incubated at 37°C over night for a  miniprep. A new colony PCR was carried out with six colonies of the XLIBlue piGEM002+Ag-promoter.
22.4 Repeated amplification of KS module II
Aim: amplify module II for further purification and fusion PCR with flanks of lacA
The PCR for the KSII was repeated  with the Phusion DNA-polymerase. A gradient was carried out from 52°C to 62°C.
Content
Master mix
KSII (µl)
Template: (gBlock KSII)
4,5
0,5
Primer iGEM044
9
1
Primer iGEM045
9
1
HF Buffer 5x
90
10
Phusion DNA-polymerase
9
1
dNTPs
9
1
Water
319,5
35
Total Volume (µl)
450
50
Step
Temperature °C
Time
KS II
1
95
5 min
5 min
2
95
30 sec
20 sec
3
52-62
30 sec
20 sec
4
72
1 min 45 sec
120 sec
5
Go To 2
30x
33x
6
72
4min
4 min
7
8
infinite
infinite
The gel showed many unspecific bands of fewer basepairs than expected. The expected band of ca 2000 bp was very weak but got stronger the higher the annealing temperature was.
22.4 repeated fusion-PCR of the Killswitch modules with the flanks
Aim: fuse the modules with the flanks to integrate them into the B. subtilis genome
For this fusion-PCR the flanks of  22.08.14 and the amplified and purified modules I and III from 28.08.14 were  used. The module II from 24.08.14 was separated on an agarose gel, cut out and  purified as well.
Content
KS I (µl)
KS II (µl)
Template KSI (ca 5 ng/µl)
1
-
Template KSII (7 ng/µl)
-
1
amyE flank I (1.5 ng/µl)
3
-
amyE flank II (ca. 6.5 ng/µl)
1
-
lacA flank I (ca. 5.5 ng/µl)
-
1
lacA flank II (7 ng/µl)
-
1
Primer piGEM034
1
-
Primer piGEM037
1
-
Primer piGEM040
-
1
Primer pIGEM043
-
1
HF Buffer 5x
10
10
Phusion DNA-polymerase
1
1
dNTPs
1
1
Water
31
31
Total Volume
50
50
Step
Temperature °C
Time (min/sec) KSI
Time (min/sec) KSII
1
95
5 min
5 min
2
95
30 sec
30 sec
3
55
30 sec
30 sec
4
72
2 min
3 min 10 sec
5
Go To 2
34x
34x
6
72
5 min
5 min
7
8
infinite
infinite
The fusion PCR was negative, expected bands would be around 1881 bp for KSI and around 3000 for KSII.
30.08.2014
18.62 Gibson assembly of piGEM-005 & D2-StrepII/ Ligation StrepDARPidin-pET16b
The transformation plates were taken out of the incubator and put in the fridge at 4°C.
31.08.2014
18.63 cPCR with clones from Ligation pET16b & StrepDARPidin
Aim: checking the success of the ligation
The grown clones from the ligation from 18.60 were checked via cPCR with  primers iGEM-032 and -033. 40 clones from Ligation plates I-III were picked for  cPCR and transferred on a plate before.
Content
Master mix  for  40 clones (µL)
Clones from Ligation I (µL)
Clones from Ligation II(µL)
Clones from Ligation III(µL)
Template (colony)
-
1
1
1
Primer iGEM-32
20
-
-
-
Primer iGEM-33
20
-
-
-
Phusion Buffer 5x
160
-
-
-
Phusion DNA-polymerase
20
-
-
-
dNTPs
20
-
-
-
Water
560
-
-
-
Master mix
-
19
19
19
Total Volume
800
20
20
20
Step
Temperature °C
Time
1
98
10 min
2
98
20 sec
3
55
20 sec
4
72
60  sec
5
Go To 2
33x
6
72
4 min
7
8
infinite
The positive clones show a band at ca. 900 bp  and were used for inoculation minipreps so that a test digest could confirm the  insertion of StrepDARPidin.
18.64 cPCR with XLI-Blue clones containing piGEM005 Hag-D2-Strep or Hag-D2-Cup
Aim: Checking insertion of Hag-D2-Cup and Hag-D2-Strep into piGEM-005
In order to check the successful integration of D2-Cup and D2-Strep clones were picked for cPCR and transferred on a LB-Amp master plate. As primers Flo96 (D23-fw) and Flo97 (D23-v) were used. 12 clones were screened.
Mix
Master-Mix (14x)
D2-Strep
Template
-
Colony
Primer Flo96 D23-fw
7
-
Primer Flo96 D23-rv
7
-
dNTPs
7
-
5x Phusion Buffer
56
-
Phusion DNA-polymerase
7
-
Water
196
-
Mastermix
-
20
Total Volume (µl)
280
20
Step
Temperature °C
Time
1
98
10 min
2
98
30 sec
3
55
30 sec
4
72
40 sec
5
Go To 2
33x
6
72
4 min
7
8
infinite
The clones I1, I3, I6, II2 and II4 were picked for plasmid isolation and sent for sequencing the next day.
Retrieved from " http://2014.igem.org/Team:Marburg:Project:Notebook:August "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
