<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Marburg_Project_Tumor"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Marburg:Project:Tumor">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Marburg:Project:Tumor&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Marburg:Project:Tumor&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Marburg:Project:Tumor&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Marburg:Project:Tumor" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Marburg:Project:Tumor</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="navbox"><NAV class="menu center first" aria-haspopup="true" aria-controls="p1"><UL id="p1" class="first"><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg" class="a">Home</A></LI><LI class="current"><A href="https://2014.igem.org/Team:Marburg:Project" class="b">Project</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Safety" class="c">Safety</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Team" class="d">Team</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Policy_Practices" class="e">Policy &amp; Practices</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Attributions" class="f">Attributions</A></LI></UL></NAV><NAV class="menu center second" aria-haspopup="true" aria-controls="p1"><UL id="p1" class="second"><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:Silver" class="d">SilverSURF</A></LI><LI class="current"><A href="https://2014.igem.org/Team:Marburg:Project:Tumor" class="c">CancerSURF</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:NRPS" class="e">RiboSURF</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:Notebook" class="f">Notebook</A></LI><LI class="last" aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:Parts" class="g">Parts</A></LI></UL></NAV></DIV><DIV class="mr_content"><H1>

CancerSURF

</H1><UL id="submenu" style="text-align:left;"><LI><A href="#intro">Introduction</A></LI><LI><A href="#production">Production of a DARPin-Strep module</A></LI><LI><A href="#coupling">Coupling StrepDARPidin to engineered Hag-D2-Strep</A></LI><LI><A href="#perspectives">Future perspectives</A></LI><LI><A href="#improve">Improve a brick</A></LI></UL><A name="intro"><H2>Introduction</H2></A><P>DARPins (Designed Ankyrin Repeat Proteins) serve as high potential next generation protein drugs. They are derived from natural ankyrin repeat proteins that contain 
ankyrin repeats, the most common protein-protein interaction motif which are predominantly found in eukaryotic proteins (Al-Khodor et al. 2010). Also, the genomes of 
various pathogenic or symbiotic bacteria and eukaryotic viruses contain several genes encoding ankyrin repeat proteins. Typically, two to four library modules, each 
corresponding to one ankyrin repeat of 33 amino acids with seven variable positions, are encased by two caps (N- and C-; Figure 1A). Additionally to the resulting high 
diversity, these repetitive modules are extremely small compared to conventional IgG antibodies, thus preventing tissue penetration. They show no effector 
function and can also exert allosteric inhibition mechanisms. 
</P><DIV class="figure" style="width:80%;"><SPAN class="caption"><B>Figure 1: (A)</B> Scheme of the DARPin library design. 
			<B>(B)</B> Tetrameric <A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A>
			construct. Four DARPin Ec1 molecules were fused via linker peptides to a Streptavidin protein. At the N-terminus 
			the Ec1 DARPin is fused to a polyhistidine-Tag (His-Tag) and the C-terminus prevents the linkage to the Streptavidin monomer. 
			Adapted from Stumpp et al. 2008.
		</SPAN></DIV><P>CancerSURF combines the scaffold of a tetrameric Streptavidin to increase the specificity of a DARPin to detect a prominent tumor surface marker of many epithelial-derived 
cancer cells (like lung and colon cancer cells), called EpCAM (epithelial cell adhesion molecule; Went et al. 2006). EpCAM is expressed by every 
epithelial cell on the basolateral side in a specific ratio while being overexpressed on the surface irregularly by transformed tumor cells (Figure 2).  We planned 
to fuse an EpCAM-binding DARPin Ec1 (Stefan et al. 2011) with a tetrameric Streptavidin increasing the local concentration of the construct for targeting EpCAM-positive 
tumor cells. In the next step, we wanted to combine the high density of epitopes offered by the filament of the bacterial flagellum as a scaffold with the modular 
advantages of a Streptavidin-Strep-Tag-system. 
</P><DIV style="margin: 0 auto; width: 75%;"><SPAN class="caption" style="text-align:justify;width: 94%;"><B>Figure 2: Transformation of epithelial-derived cancer cells overexpressing EpCAM.</B></SPAN></DIV><H2><A name="production">Production of a DARPin-Strep module for specific recognition of EpCAM-positive epithelial-derived cancer cells</A></H2><P>To address the goal of creating a CancerSURF for efficient and rapid targeting of cancer cells, we first had to synthesize a Strep-DARPin fusion construct 
(<A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A>). 
Template DNA for the tetrameric <A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A> construct was commercially 
synthesized by IDT (Integrated DNA Technologies) and cloned into a pET-plasmide in order to 
overexpress the protein in <I>E.coli</I> BL21 (DE3). Since we discovered that <A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A> 
is not soluble when overexpressed and remains in inclusion bodies, we had to apply 
a refolding protocol to (re)-gain <A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A> 
in sufficient amounts (Howarth and Ting, 2008). After refolding, the protein was purified in a two-step process: 
Ni-NTA-affinity purification and size-exclusion chromatography (Figure 3). The calculated protein mass of the monomeric 
<A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A> 
is 31 kDa. The tetramer should 
have a size of 124 kDa and is visible above the highest marker line (116 kDa; Figure 3).
</P><DIV class="figure" style="width:90%;"><SPAN class="caption"><B>Figure 3:
			Purification of <A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A>.</B> Size-exclusion chromatogram and 
			coomassie stained SDS-PAGE of purified <A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A>. Upper SDS-PAGE: 
			Fractions taken from Ni-NTA purification. M: marker, L: lysate, FT: flow through, 
			W: wash, E: elution. Lower SDS-PAGE: Samples of peak fractions taken from size-exclusion 
			fractions. Samples on the right side have been cooked prior to loading showing the monomeric 
			<A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A>. 
		</SPAN></DIV><P>Protein containing fractions were analyzed on a coomassie stained SDS-PAGE and refolding was verified by incubating the eluate for 10 min at 95 °C (Figure 3). The peak 
fractions after size-exclusion were pooled and concentrated for further analysis. 
</P><P>In the next step, we designed an assay to test the functionality and binding specificity of the engineered 
<A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A> 
to EpCAM on lung and colon cancer cells. Colon 
cancer cells (Caco-2) and lung cancer cells (A549) which are known to highly express EpCAM (Maaser and Borlak, 2008) were treated with 
<A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A> 
in serial dilutions. 
After washing, they were incubated with anti-His-antibody-Alexa488 conjugate. Fibroblasts (3T3 wild type cells) were used as EpCAM-negative control. Fluorescence was 
quantified at 520 nm (Figure 4C) with a fluorometer. 
</P><DIV class="figure" style="width:90%;"><SPAN class="caption"><B>Figure 4: (A)</B> Immunofluorescent staining of Caco-2 and 3T3 cells incubated with 
			<A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A> protein.   
			Both cell lines were stained with phalloidin (red) conjugated with rhodamine (F-Actin) and Alexa488 (green) conjugated anti-His-antibody 
			(<A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A>) in presence of 25 µM 
			<A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A>. The nucleus was stained with DAPI (blue). Scale bar represents 5 µm. <B>(B)</B> 
			Immunofluorescent staining of A549 cells incubated with 
			<A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A> protein. Cells were stained with rhodamine (red) conjugated phalloidin 
			(F-Actin) and Alexa488 (green) conjugated anti-His antibody (<A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A>) 
			in presence of 25 µM <A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A>. The nucleus was stained with DAPI (blue). The control 
			was stained in 
			absence of <A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A>. Scale bar represents 5 µm. <B>(C)</B> Binding specificity of 
			<A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A> to epithelial-derived  cancer cell lines. Fluorescence of Alexa488-coupled 
			anti-His-antibody in A549 and Caco-2 (epithelial) cells was normalized to 3T3 (interstitial) cells. <B>(D)</B> Fluorescence intensities of Caco-2 and 3T3 cells in comparison.
		</SPAN></DIV><P>The assay confirmed the optimal concentration of <A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A> 
to be  25 µM for the treatment of EpCAM-positive cancer cell lines (Figure 4C). 
</P><P>Next we wanted to investigate if <A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A> 
can bind to epithelial-derived cancer cells. Therefore we used an immunofluorescence based approach. Caco-2 and A549 cells were grown on 
gelatin coated coverslips and incubated with 25 µM <A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A>. 
As a negative control fibroblast cell line 3T3 wild type was used. After fixation with formaldehyde, cells were permeabilized with Triton-X-100. 
The membrane-associated F-Actin was stained with phalloidin-rhodamine and the nucleus was stained with DAPI (Figure 4A, B). Anti-His-antibody-Alexa488 (green) 
conjugate was used for the staining of <A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A>. 
Phalloidin conjugated with rhodamine (red) was used for staining of the F-Actin cytoskeleton to follow cell morphology. In Caco-2 and A549 cells a co-localization of 
<A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A> with membrane-associated F-Actin could be shown (Figure 4A, B). Some unspecific 
His-staining was detected in the nucleus of all cell lines both in presence and absence of 
<A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A>. 
In presence of <A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A> 
both epithelial-derived cancer cell lines showed the highest fluorescence intensity at the cell membrane. In Caco-2 cells the total fluorescence 
intensity ratio membrane/cytoplasm was 2.5x higher compared to 3T3 cells (Figure  4D). 
</P><P>We were able to show that our tetrameric <A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A> 
is capable of binding different EpCAM-positive cancer cell lineages, which already is a remarkable result. Additionally, 
we wanted to increase the local concentration of <A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A> 
molecules and also integrate the activation of the innate immune system for enhancing our CancerSURF. Bacterial surface proteins, such as Flagellin, 
activate TLR5 (toll-like receptor 5) on dendritic cells, leading to a stimulation of T helper cells.  Therefore, employing bacterial flagellar filaments not only drastically 
increases the amount of <A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A> present on the cancer cell surface but also leads 
to a stimulation of the inflammatory immune response (Leigh et al. 2014) which 
creates an anti-tumor environment (Tarahomjoo et al. 2013) . 
</P><DIV class="figure" style="width:10%"><SPAN class="caption"><B>Figure 5: Cellular response after TLR-5 activation by Flagellin.</B></SPAN></DIV><H2><A name="coupling">Coupling StrepDARPidin to engineered Hag-D2-Strep</A></H2><P>As described previously, enhancing the local concentration of DARPin molecules is crucial to allow a rapid and efficient targeting of cancer cells. Therefore, 
B. subtilis Flagellin was engineered carrying an insertion in the D1-domain replacing the small native loop (see SilverSURF for detailed information) with the <I>S. 
typhimurium</I> D2-domain and a Strep-Tag. The Hag-D2-Strep <A href="http://parts.igem.org/Part:BBa_K1329003" target="_blank">(BBa_K1329003)</A> was used for 
the generation of modified <I>B. subtilis</I> strains as well as overexpression in <I>E. coli</I> BL21 
(DE3) cells (Figure 6A). As Flagellin tends to form polymers when overexpressed (a natural feature necessary to build up the filament), it was co-purified with its 
cognate chaperone FliS, forming a stable and stoichiometric complex on a size-exclusion column (Figure 6A). The fractions of the main peak were pooled and concentrated 
to a final volume of 500 µL for further analysis. 
</P><DIV class="figure" style="width:90%;"><SPAN class="caption"><B>Figure 6: Purification of <A href="http://parts.igem.org/Part:BBa_K1329003" target="_blank">Hag-D2-Strep</A> and binding 
			assays with <A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A>. (A)</B> Size-exclusion 
			chromatogram of <A href="http://parts.igem.org/Part:BBa_K1329003" target="_blank">Hag-D2-Strep</A> 
			in complex with its chaperone FliS. Upper SDS-PAGE: Ni-NTA purification. 
			M: marker, L: lysate, FT: flow through, W: wash, E: elution. Lower SDS-PAGE: Samples of peak fractions 
			taken from size-exclusion fractions. <B>(B)</B> Streptavidin- and GST-interaction assay using purified 
			<A href="http://parts.igem.org/Part:BBa_K1329003" target="_blank">Hag-D2-Strep/FliS</A> 
			and Hag-Strep/FliS. (1) Strep-beads without protein (contaminant protein at 30 kDa). (2) Strep-beads with Hag-Strep/FliS immobilized. (3) 
			Strep-beads with <A href="http://parts.igem.org/Part:BBa_K1329003" target="_blank">Hag-D2-Strep/FliS</A>. 
			(4) GST-beads with Hag-Strep/FliS. (5) GST-beads with <A href="http://parts.igem.org/Part:BBa_K1329003" target="_blank">Hag-D2-Strep/FliS</A>. 
			<B>(C)</B> Interaction of <A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A> and 
			<A href="http://parts.igem.org/Part:BBa_K1329003" target="_blank">Hag-D2-Strep/FliS</A> 
			visualized by competition assays. Soluble fractions: 
			(1) Hag-Strep/FliS. (2) <A href="http://parts.igem.org/Part:BBa_K1329003" target="_blank">Hag-D2-Strep/FliS</A>. 
			(3) Hag-Strep/FliS and <A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A>. (4) 
			<A href="http://parts.igem.org/Part:BBa_K1329003" target="_blank">Hag-D2-Strep/FliS</A> and 
			<A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A>. (5) 
			<A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A>. Insoluble fractions: (6) Hag-Strep/FliS and 
			<A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A> (1:5 dilution). (7) 
			<A href="http://parts.igem.org/Part:BBa_K1329003" target="_blank">Hag-D2-Strep/FliS</A> and 
			<A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A> (1:5 dilution).(8) Hag-Strep/FliS and 
			<A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A> (1:1 dilution). 
			(9) <A href="http://parts.igem.org/Part:BBa_K1329003" target="_blank">Hag-D2-Strep/FliS</A> 
			and <A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A> (1:1 dilution).
		</SPAN></DIV><P>To investigate whether the purified <A href="http://parts.igem.org/Part:BBa_K1329003" target="_blank">Hag-D2-Strep</A> 
is able to interact with Streptavidin <I>in vitro</I> prior to 
labeling intact filaments from <I>B. subtilis</I> flagella, we used Streptavidin-pulldown assays to confirm their functionality. For this we immobilized purified 
<A href="http://parts.igem.org/Part:BBa_K1329003" target="_blank">Hag-D2-Strep</A> and Hag-Strep on Streptavidin- and GST-beads (Figure 6B). Both engineered 
variants of Flagellin were able to bind to Streptavidin-beads but not GST and thus confirmed that the Strep-Tag is accessible and functional (Figure 5B, lanes 2 and 3). 
</P><P>Furthermore, we wanted to know if our functional 
<A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A> is capable of binding to the engineered Flagellin variants. 
This was the last and crucial step that had to be verified before the docking of 
<A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A> to the flagellar filaments could be performed. 
Purified proteins were mixed in different ratios and precipitation was observed. Samples from the supernatant (soluble fraction) and the 
pellet (insoluble fraction) were loaded on a SDS-PAGE showing that the pellet contained a huge amount of 
<A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A> and the Flagellin variants (Figure 5C). 
We assumed that the strong interaction of Streptavidin with the Strep-Tag leads to an aggregation of Hag-Strep variants followed by 
precipitation (Figure 5C, lanes 6-9). 
</P><P>The D2-Strep variant of Flagellin was also used for replacing the native Flagellin in <I>B. subtilis</I> as described before (see 
<A href="https://2014.igem.org/Team:Marburg:Project:Silver" target="_blank">SilverSURF</A> for detailed information) in order to generate a 
strain exposing the Strep-Tag on the surface of the flagellar filament. <I>In vitro</I> assays confirmed that the 
<A href="http://parts.igem.org/Part:BBa_K1329003" target="_blank">Hag-D2-Strep</A> was stable and functional but we had no information about the incorporation 
of the engineered Flagellin variant into the flagellar filament <I>in vivo</I>. Therefore, we inoculated swimming plates (containing 0.3 % agar) and swarming plates 
(containing 0.7 % agar) with <I>B. subtilis</I> wild type and D2-Strep cells. The mutant strain showed only slightly slower spreading on both plates, indicating that both 
single cellular and multi cellular movement is not strongly affected (Figure 6E, F). Additionally, we used negative staining transmission electron microscopy and were able 
to show that D2-Strep cells are as much flagellated as the wild type (Figure 6A, B). Finally, we can state that our engineered Flagellin variant D2-Strep is fully functional 
and incorporated into intact flagellar filaments. 
The last step, immobilizing <A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A> 
on purified filaments, is currently finalized in the laboratory. 
</P><DIV class="figure" style="width:90%;"><SPAN class="caption"><B>Figure 7: Electron micrographs of uranyl-acetate stained <I>B. subtilis</I> cells. (A)</B> Whole cell of wild type strain 
			3610 <B>(B)</B> Whole cell of <A href="http://parts.igem.org/Part:BBa_K1329003" target="_blank">Hag-D2-Strep</A> strain. Scale bar represents 1 μm. 
			<B>(C)</B> Magnification of flagellar filaments from 3610 wild type strain. Scale bar represents 100 nm <B>(D)</B> Magnification of flagellar filaments 
			from Hag-D2-Strepstrain. Scale bar represents 300 nm. <B>(E)</B> Swarming assay of both <I>B. subtilis</I> strains. 
			<B>(F)</B> Swimming assay of both <I>B. subtilis</I> strains.
		</SPAN></DIV><H2><A name="perspectives">Future perspectives in therapy and diagnostics </A></H2><P>The CancerSURF is a novel, modular and efficient method to target different cancer cell lineages and opens the gate for new applications in cancer treatment. 
The flagellar filament serves as a huge scaffold that can be loaded with different agents of ones own choice. 
By fusing the tetrameric Streptavidin to small molecules such as DARPins, a wide range of different applications is imaginable. Combining e.g. 
<A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A> 
and fluorescent proteins fused to Streptavidin, cancer cells could be visualized during clinical surgeries. 
Cytotoxic agents in combination with 
<A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A> could directly lead to cancer 
cell lysis without causing collateral damage to the surrounding tissues making <A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A> 
an imaginable antibody alternative for both future diagnostic and therapeutic perspectives.
</P><DIV class="figure" style="width:90%;"><SPAN class="caption"><B>Figure 8: Model for possible future applications in lung cancer treatment.</B></SPAN></DIV><H2>Improve a brick</H2><P>One of the strongest interactions established by biological molecules is the interaction 
between Streptavidin and biotin, a cofactor of carboxylases. The property of Streptavidin to 
tightly bind substrates in combination with engineered Strep-Tags (peptide-tag) has been 
utilized over the years for several approaches, e.g. biochemical and immunohistochemical 
methods for protein purification and immunofluorescent stainings.
</P><P>In 2006, the iGEM team from Harvard used Streptavidin to stain the outer membrane of <I>E. coli</I> cells. By placing 
the <A href="http://parts.igem.org/Part:BBa_J36848" target="_blank">Streptavidin</A> 
coding sequence downstream of a membrane targeting signal sequence and outer membrane protein 
under control of an IPTG inducible promoter, the heterologous production in <I>E. coli</I> was achieved 
to selectively stain the outer membrane by biotinylated fluorophores.
</P><P>Our aim was to use the extremely tight interaction of Streptavidin and the engineered Strep-TagII, 
which binds to Streptavidin in the low pico molar range, in combination with 
DARPin to selectively target epithelial-derived cancer cells and amplify the interaction by 
our Strep-tagged flagellar filament scaffold. Therefore, we designed a codon-optimized 
Streptavidin/DARPin chimeric gene (<A href="http://parts.igem.org/Part:BBa_K1329000" target="_blank">StrepDARPidin</A>) 
for the heterologous production in <I>E. coli</I>. This chimeric gene enables the efficient 
production of StrepDARPidin in <I>E. coli</I> and may also serve as a template for the codon optimized Streptavidin gene for future iGEM teams, 
thereby improving the existing 
Streptavidin biobrick gene in the registry. Additionally, we established a refolding protocol 
for the purification of StrepDARPidin and verified its functionality and selectivity by an 
immunofluorescent analysis of targeted cancer cells and Flagellin-strep binding assays, 
thereby improving the sole Streptavidin by adding the EpCAM binding property to the protein. 
</P><P>
Al-Khodor S, Price CT, Kalia A, Abu Kwaik Y (2010). Ankyrin-repeat containing proteins of microbes: a conserved structure with functional diversity.Trends Microbiol. 18, 132–139 10.1016
</P><P>Howarth M, TingAY (2008). Imaging proteins in live mammalian cells with biotin ligase and monovalent Streptavidin. Nature Protocols 3, 534 - 545 
</P><P>Leigh ND, Guanglin Bian, Xilai Ding, Hong Liu, Semra Aygun-Sunar, Lyudmila G. Burdelya, Andrei V. Gudkov, Xuefang Cao (2014). A Flagellin-Derived Toll-Like Receptor 5 Agonist Stimulates Cytotoxic Lymphocyte-Mediated Tumor Immunity. PLoS ONE 9(1): e85587
</P><P>Maaser K, Borlak J (2008). A genome-wide expression analysis identifies a network of EpCAM-induced cell cycle regulators.British Journal of Cancer 99,1635 – 1643
</P><P>Stefan N, Martin-Killias P, Wyss-Stoeckle (2011). DARPins Recognizing the Tumor-Associated Antigen EpCAM Selected by Phage and Ribosome Display and Engineered for Multivalency.
J. Mol. Biol. (2011) 413, 826–843
</P><P>Stumpp MT, Binz HK, Amstutz P. (2008). DARPins a new generation of protein therapeutics.Drug Discov. Today. Aug. 13(15-16):695-701.
</P><P>Tarahomjoo S (2013). Utilizing bacterial flagellins against infectious diseases and cancers.
Antonie van Leeuwenhoek 105:275–288
</P><P>Went P, Vasei M, Bubendorf L (2006). Frequent high-level expression of the Immunotherapeutic targetEp-CAM in colon, stomach, prostate and lung cancers. British Journal of Cancer 94, 128 – 135 
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Marburg:Project:Tumor">http://2014.igem.org/Team:Marburg:Project:Tumor</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Marburg:Project:Tumor" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Marburg:Project:Tumor" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Marburg:Project:Tumor&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Marburg:Project:Tumor&amp;oldid=398043" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>