"
Page
Discussion
View source
History
teams
Log in
 
Team:Marburg:Project:Tumor
From 2014.igem.org
Home
Project
Safety
Team
Policy & Practices
Attributions
SilverSURF
CancerSURF
RiboSURF
Notebook
Parts
CancerSURF
Introduction
Production of a DARPin-Strep module
Coupling StrepDARPidin to engineered Hag-D2-Strep
Future perspectives
Improve a brick
Introduction
DARPins (Designed Ankyrin Repeat Proteins) serve as high potential next generation protein drugs. They are derived from natural ankyrin repeat proteins that contain  ankyrin repeats, the most common protein-protein interaction motif which are predominantly found in eukaryotic proteins (Al-Khodor et al. 2010). Also, the genomes of  various pathogenic or symbiotic bacteria and eukaryotic viruses contain several genes encoding ankyrin repeat proteins. Typically, two to four library modules, each  corresponding to one ankyrin repeat of 33 amino acids with seven variable positions, are encased by two caps (N- and C-; Figure 1A). Additionally to the resulting high  diversity, these repetitive modules are extremely small compared to conventional IgG antibodies, thus preventing tissue penetration. They show no effector  function and can also exert allosteric inhibition mechanisms.
Figure 1: (A) Scheme of the DARPin library design. (B) Tetrameric StrepDARPidin construct. Four DARPin Ec1 molecules were fused via linker peptides to a Streptavidin protein. At the N-terminus     the Ec1 DARPin is fused to a polyhistidine-Tag (His-Tag) and the C-terminus prevents the linkage to the Streptavidin monomer.     Adapted from Stumpp et al. 2008.
CancerSURF combines the scaffold of a tetrameric Streptavidin to increase the specificity of a DARPin to detect a prominent tumor surface marker of many epithelial-derived  cancer cells (like lung and colon cancer cells), called EpCAM (epithelial cell adhesion molecule; Went et al. 2006). EpCAM is expressed by every  epithelial cell on the basolateral side in a specific ratio while being overexpressed on the surface irregularly by transformed tumor cells (Figure 2).  We planned  to fuse an EpCAM-binding DARPin Ec1 (Stefan et al. 2011) with a tetrameric Streptavidin increasing the local concentration of the construct for targeting EpCAM-positive  tumor cells. In the next step, we wanted to combine the high density of epitopes offered by the filament of the bacterial flagellum as a scaffold with the modular  advantages of a Streptavidin-Strep-Tag-system.
Figure 2: Transformation of epithelial-derived cancer cells overexpressing EpCAM.
Production of a DARPin-Strep module for specific recognition of EpCAM-positive epithelial-derived cancer cells
To address the goal of creating a CancerSURF for efficient and rapid targeting of cancer cells, we first had to synthesize a Strep-DARPin fusion construct  ( StrepDARPidin ).  Template DNA for the tetrameric StrepDARPidin construct was commercially  synthesized by IDT (Integrated DNA Technologies) and cloned into a pET-plasmide in order to  overexpress the protein in E.coli BL21 (DE3). Since we discovered that StrepDARPidin is not soluble when overexpressed and remains in inclusion bodies, we had to apply  a refolding protocol to (re)-gain StrepDARPidin in sufficient amounts (Howarth and Ting, 2008). After refolding, the protein was purified in a two-step process:  Ni-NTA-affinity purification and size-exclusion chromatography (Figure 3). The calculated protein mass of the monomeric StrepDARPidin is 31 kDa. The tetramer should  have a size of 124 kDa and is visible above the highest marker line (116 kDa; Figure 3).
Figure 3:    Purification of StrepDARPidin . Size-exclusion chromatogram and     coomassie stained SDS-PAGE of purified StrepDARPidin . Upper SDS-PAGE:     Fractions taken from Ni-NTA purification. M: marker, L: lysate, FT: flow through,     W: wash, E: elution. Lower SDS-PAGE: Samples of peak fractions taken from size-exclusion     fractions. Samples on the right side have been cooked prior to loading showing the monomeric StrepDARPidin .
Protein containing fractions were analyzed on a coomassie stained SDS-PAGE and refolding was verified by incubating the eluate for 10 min at 95 °C (Figure 3). The peak  fractions after size-exclusion were pooled and concentrated for further analysis.
In the next step, we designed an assay to test the functionality and binding specificity of the engineered StrepDARPidin to EpCAM on lung and colon cancer cells. Colon  cancer cells (Caco-2) and lung cancer cells (A549) which are known to highly express EpCAM (Maaser and Borlak, 2008) were treated with StrepDARPidin in serial dilutions.  After washing, they were incubated with anti-His-antibody-Alexa488 conjugate. Fibroblasts (3T3 wild type cells) were used as EpCAM-negative control. Fluorescence was  quantified at 520 nm (Figure 4C) with a fluorometer.
Figure 4: (A) Immunofluorescent staining of Caco-2 and 3T3 cells incubated with StrepDARPidin protein.       Both cell lines were stained with phalloidin (red) conjugated with rhodamine (F-Actin) and Alexa488 (green) conjugated anti-His-antibody     ( StrepDARPidin ) in presence of 25 µM StrepDARPidin . The nucleus was stained with DAPI (blue). Scale bar represents 5 µm. (B) Immunofluorescent staining of A549 cells incubated with StrepDARPidin protein. Cells were stained with rhodamine (red) conjugated phalloidin     (F-Actin) and Alexa488 (green) conjugated anti-His antibody ( StrepDARPidin )     in presence of 25 µM StrepDARPidin . The nucleus was stained with DAPI (blue). The control     was stained in     absence of StrepDARPidin . Scale bar represents 5 µm. (C) Binding specificity of StrepDARPidin to epithelial-derived  cancer cell lines. Fluorescence of Alexa488-coupled     anti-His-antibody in A549 and Caco-2 (epithelial) cells was normalized to 3T3 (interstitial) cells. (D) Fluorescence intensities of Caco-2 and 3T3 cells in comparison.
The assay confirmed the optimal concentration of StrepDARPidin to be  25 µM for the treatment of EpCAM-positive cancer cell lines (Figure 4C).
Next we wanted to investigate if StrepDARPidin can bind to epithelial-derived cancer cells. Therefore we used an immunofluorescence based approach. Caco-2 and A549 cells were grown on  gelatin coated coverslips and incubated with 25 µM StrepDARPidin .  As a negative control fibroblast cell line 3T3 wild type was used. After fixation with formaldehyde, cells were permeabilized with Triton-X-100.  The membrane-associated F-Actin was stained with phalloidin-rhodamine and the nucleus was stained with DAPI (Figure 4A, B). Anti-His-antibody-Alexa488 (green)  conjugate was used for the staining of StrepDARPidin .  Phalloidin conjugated with rhodamine (red) was used for staining of the F-Actin cytoskeleton to follow cell morphology. In Caco-2 and A549 cells a co-localization of StrepDARPidin with membrane-associated F-Actin could be shown (Figure 4A, B). Some unspecific  His-staining was detected in the nucleus of all cell lines both in presence and absence of StrepDARPidin .  In presence of StrepDARPidin both epithelial-derived cancer cell lines showed the highest fluorescence intensity at the cell membrane. In Caco-2 cells the total fluorescence  intensity ratio membrane/cytoplasm was 2.5x higher compared to 3T3 cells (Figure  4D).
We were able to show that our tetrameric StrepDARPidin is capable of binding different EpCAM-positive cancer cell lineages, which already is a remarkable result. Additionally,  we wanted to increase the local concentration of StrepDARPidin molecules and also integrate the activation of the innate immune system for enhancing our CancerSURF. Bacterial surface proteins, such as Flagellin,  activate TLR5 (toll-like receptor 5) on dendritic cells, leading to a stimulation of T helper cells.  Therefore, employing bacterial flagellar filaments not only drastically  increases the amount of StrepDARPidin present on the cancer cell surface but also leads  to a stimulation of the inflammatory immune response (Leigh et al. 2014) which  creates an anti-tumor environment (Tarahomjoo et al. 2013) .
Figure 5: Cellular response after TLR-5 activation by Flagellin.
Coupling StrepDARPidin to engineered Hag-D2-Strep
As described previously, enhancing the local concentration of DARPin molecules is crucial to allow a rapid and efficient targeting of cancer cells. Therefore,  B. subtilis Flagellin was engineered carrying an insertion in the D1-domain replacing the small native loop (see SilverSURF for detailed information) with the S.  typhimurium D2-domain and a Strep-Tag. The Hag-D2-Strep (BBa_K1329003) was used for  the generation of modified B. subtilis strains as well as overexpression in E. coli BL21  (DE3) cells (Figure 6A). As Flagellin tends to form polymers when overexpressed (a natural feature necessary to build up the filament), it was co-purified with its  cognate chaperone FliS, forming a stable and stoichiometric complex on a size-exclusion column (Figure 6A). The fractions of the main peak were pooled and concentrated  to a final volume of 500 µL for further analysis.
Figure 6: Purification of Hag-D2-Strep and binding     assays with StrepDARPidin . (A) Size-exclusion     chromatogram of Hag-D2-Strep in complex with its chaperone FliS. Upper SDS-PAGE: Ni-NTA purification.     M: marker, L: lysate, FT: flow through, W: wash, E: elution. Lower SDS-PAGE: Samples of peak fractions     taken from size-exclusion fractions. (B) Streptavidin- and GST-interaction assay using purified Hag-D2-Strep/FliS and Hag-Strep/FliS. (1) Strep-beads without protein (contaminant protein at 30 kDa). (2) Strep-beads with Hag-Strep/FliS immobilized. (3)     Strep-beads with Hag-D2-Strep/FliS .     (4) GST-beads with Hag-Strep/FliS. (5) GST-beads with Hag-D2-Strep/FliS . (C) Interaction of StrepDARPidin and Hag-D2-Strep/FliS visualized by competition assays. Soluble fractions:     (1) Hag-Strep/FliS. (2) Hag-D2-Strep/FliS .     (3) Hag-Strep/FliS and StrepDARPidin . (4) Hag-D2-Strep/FliS and StrepDARPidin . (5) StrepDARPidin . Insoluble fractions: (6) Hag-Strep/FliS and StrepDARPidin (1:5 dilution). (7) Hag-D2-Strep/FliS and StrepDARPidin (1:5 dilution).(8) Hag-Strep/FliS and StrepDARPidin (1:1 dilution).     (9) Hag-D2-Strep/FliS and StrepDARPidin (1:1 dilution).
To investigate whether the purified Hag-D2-Strep is able to interact with Streptavidin in vitro prior to  labeling intact filaments from B. subtilis flagella, we used Streptavidin-pulldown assays to confirm their functionality. For this we immobilized purified Hag-D2-Strep and Hag-Strep on Streptavidin- and GST-beads (Figure 6B). Both engineered  variants of Flagellin were able to bind to Streptavidin-beads but not GST and thus confirmed that the Strep-Tag is accessible and functional (Figure 5B, lanes 2 and 3).
Furthermore, we wanted to know if our functional StrepDARPidin is capable of binding to the engineered Flagellin variants.  This was the last and crucial step that had to be verified before the docking of StrepDARPidin to the flagellar filaments could be performed.  Purified proteins were mixed in different ratios and precipitation was observed. Samples from the supernatant (soluble fraction) and the  pellet (insoluble fraction) were loaded on a SDS-PAGE showing that the pellet contained a huge amount of StrepDARPidin and the Flagellin variants (Figure 5C).  We assumed that the strong interaction of Streptavidin with the Strep-Tag leads to an aggregation of Hag-Strep variants followed by  precipitation (Figure 5C, lanes 6-9).
The D2-Strep variant of Flagellin was also used for replacing the native Flagellin in B. subtilis as described before (see SilverSURF for detailed information) in order to generate a  strain exposing the Strep-Tag on the surface of the flagellar filament. In vitro assays confirmed that the Hag-D2-Strep was stable and functional but we had no information about the incorporation  of the engineered Flagellin variant into the flagellar filament in vivo. Therefore, we inoculated swimming plates (containing 0.3 % agar) and swarming plates  (containing 0.7 % agar) with B. subtilis wild type and D2-Strep cells. The mutant strain showed only slightly slower spreading on both plates, indicating that both  single cellular and multi cellular movement is not strongly affected (Figure 6E, F). Additionally, we used negative staining transmission electron microscopy and were able  to show that D2-Strep cells are as much flagellated as the wild type (Figure 6A, B). Finally, we can state that our engineered Flagellin variant D2-Strep is fully functional  and incorporated into intact flagellar filaments.  The last step, immobilizing StrepDARPidin on purified filaments, is currently finalized in the laboratory.
Figure 7: Electron micrographs of uranyl-acetate stained B. subtilis cells. (A) Whole cell of wild type strain     3610 (B) Whole cell of Hag-D2-Strep strain. Scale bar represents 1 μm. (C) Magnification of flagellar filaments from 3610 wild type strain. Scale bar represents 100 nm (D) Magnification of flagellar filaments     from Hag-D2-Strepstrain. Scale bar represents 300 nm. (E) Swarming assay of both B. subtilis strains. (F) Swimming assay of both B. subtilis strains.
Future perspectives in therapy and diagnostics
The CancerSURF is a novel, modular and efficient method to target different cancer cell lineages and opens the gate for new applications in cancer treatment.  The flagellar filament serves as a huge scaffold that can be loaded with different agents of ones own choice.  By fusing the tetrameric Streptavidin to small molecules such as DARPins, a wide range of different applications is imaginable. Combining e.g. StrepDARPidin and fluorescent proteins fused to Streptavidin, cancer cells could be visualized during clinical surgeries.  Cytotoxic agents in combination with StrepDARPidin could directly lead to cancer  cell lysis without causing collateral damage to the surrounding tissues making StrepDARPidin an imaginable antibody alternative for both future diagnostic and therapeutic perspectives.
Figure 8: Model for possible future applications in lung cancer treatment.
Improve a brick
One of the strongest interactions established by biological molecules is the interaction  between Streptavidin and biotin, a cofactor of carboxylases. The property of Streptavidin to  tightly bind substrates in combination with engineered Strep-Tags (peptide-tag) has been  utilized over the years for several approaches, e.g. biochemical and immunohistochemical  methods for protein purification and immunofluorescent stainings.
In 2006, the iGEM team from Harvard used Streptavidin to stain the outer membrane of E. coli cells. By placing  the Streptavidin coding sequence downstream of a membrane targeting signal sequence and outer membrane protein  under control of an IPTG inducible promoter, the heterologous production in E. coli was achieved  to selectively stain the outer membrane by biotinylated fluorophores.
Our aim was to use the extremely tight interaction of Streptavidin and the engineered Strep-TagII,  which binds to Streptavidin in the low pico molar range, in combination with  DARPin to selectively target epithelial-derived cancer cells and amplify the interaction by  our Strep-tagged flagellar filament scaffold. Therefore, we designed a codon-optimized  Streptavidin/DARPin chimeric gene ( StrepDARPidin )  for the heterologous production in E. coli. This chimeric gene enables the efficient  production of StrepDARPidin in E. coli and may also serve as a template for the codon optimized Streptavidin gene for future iGEM teams,  thereby improving the existing  Streptavidin biobrick gene in the registry. Additionally, we established a refolding protocol  for the purification of StrepDARPidin and verified its functionality and selectivity by an  immunofluorescent analysis of targeted cancer cells and Flagellin-strep binding assays,  thereby improving the sole Streptavidin by adding the EpCAM binding property to the protein.
Al-Khodor S, Price CT, Kalia A, Abu Kwaik Y (2010). Ankyrin-repeat containing proteins of microbes: a conserved structure with functional diversity.Trends Microbiol. 18, 132–139 10.1016
Howarth M, TingAY (2008). Imaging proteins in live mammalian cells with biotin ligase and monovalent Streptavidin. Nature Protocols 3, 534 - 545
Leigh ND, Guanglin Bian, Xilai Ding, Hong Liu, Semra Aygun-Sunar, Lyudmila G. Burdelya, Andrei V. Gudkov, Xuefang Cao (2014). A Flagellin-Derived Toll-Like Receptor 5 Agonist Stimulates Cytotoxic Lymphocyte-Mediated Tumor Immunity. PLoS ONE 9(1): e85587
Maaser K, Borlak J (2008). A genome-wide expression analysis identifies a network of EpCAM-induced cell cycle regulators.British Journal of Cancer 99,1635 – 1643
Stefan N, Martin-Killias P, Wyss-Stoeckle (2011). DARPins Recognizing the Tumor-Associated Antigen EpCAM Selected by Phage and Ribosome Display and Engineered for Multivalency. J. Mol. Biol. (2011) 413, 826–843
Stumpp MT, Binz HK, Amstutz P. (2008). DARPins a new generation of protein therapeutics.Drug Discov. Today. Aug. 13(15-16):695-701.
Tarahomjoo S (2013). Utilizing bacterial flagellins against infectious diseases and cancers. Antonie van Leeuwenhoek 105:275–288
Went P, Vasei M, Bubendorf L (2006). Frequent high-level expression of the Immunotherapeutic targetEp-CAM in colon, stomach, prostate and lung cancers. British Journal of Cancer 94, 128 – 135
Retrieved from " http://2014.igem.org/Team:Marburg:Project:Tumor "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
