<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Marburg_Project_Notebook_June"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Marburg:Project:Notebook:June">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Marburg:Project:Notebook:June&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Marburg:Project:Notebook:June&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Marburg:Project:Notebook:June&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Marburg:Project:Notebook:June" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Marburg:Project:Notebook:June</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="navbox"><NAV class="menu center first" aria-haspopup="true" aria-controls="p1"><UL id="p1" class="first"><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg" class="a">Home</A></LI><LI class="current"><A href="https://2014.igem.org/Team:Marburg:Project" class="b">Project</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Safety" class="c">Safety</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Team" class="d">Team</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Policy_Practices" class="e">Policy &amp; Practices</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Attributions" class="f">Attributions</A></LI></UL></NAV><NAV class="menu center second" aria-haspopup="true" aria-controls="p1"><UL id="p1" class="second"><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:Silver" class="d">SilverSURF</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:Tumor" class="c">CancerSURF</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:NRPS" class="e">RiboSURF</A></LI><LI class="current"><A href="https://2014.igem.org/Team:Marburg:Project:Notebook" class="f">Notebook</A></LI><LI class="last" aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:Parts" class="g">Parts</A></LI></UL></NAV></DIV><DIV class="mr_content"><H1>

Notebook: June

</H1><UL id="submenu"><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook">March</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:April">April</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:May">May</A></LI><LI class="current"><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:June">June</A></LI><LI><A href="#02.06.2014">02.06.2014</A></LI><LI><A href="#03.06.2014">03.06.2014</A></LI><LI><A href="#04.06.2014">04.06.2014</A></LI><LI><A href="#05.06.2014">05.06.2014</A></LI><LI><A href="#06.06.2014">06.06.2014</A></LI><LI><A href="#07.06.2014">07.06.2014</A></LI><LI><A href="#08.06.2014">08.06.2014</A></LI><LI><A href="#09.06.2014">09.06.2014</A></LI><LI><A href="#10.06.2014">10.06.2014</A></LI><LI><A href="#11.06.2014">11.06.2014</A></LI><LI><A href="#12.06.2014">12.06.2014</A></LI><LI><A href="#13.06.2014">13.06.2014</A></LI><LI><A href="#14.06.2014">14.06.2014</A></LI><LI><A href="#15.06.2014">15.06.2014</A></LI><LI><A href="#16.06.2014">16.06.2014</A></LI><LI><A href="#17.06.2014">17.06.2014</A></LI><LI><A href="#18.06.2014">18.06.2014</A></LI><LI><A href="#19.06.2014">19.06.2014</A></LI><LI><A href="#20.06.2014">20.06.2014</A></LI><LI><A href="#21.06.2014">21.06.2014</A></LI><LI><A href="#22.06.2014">22.06.2014</A></LI><LI><A href="#23.06.2014">23.06.2014</A></LI><LI><A href="#24.06.2014">24.06.2014</A></LI><LI><A href="#25.06.2014">25.06.2014</A></LI><LI><A href="#26.06.2014">26.06.2014</A></LI><LI><A href="#27.06.2014">27.06.2014</A></LI><LI><A href="#28.06.2014">28.06.2014</A></LI><LI><A href="#29.06.2014">29.06.2014</A></LI><LI><A href="#30.06.2014">30.06.2014</A></LI></UL><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:July">July</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:August">August</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:September">September</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:October">October</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:Methods">Methods</A></LI><DIV class="notebooky-entry"><H2 class="title"><A name="02.06.14">02.06.2014</A></H2><FIELDSET class="exp13"><LEGEND><A name="exp13.56">13.56 Making glycerine stocks of the following strains </A></LEGEND><DIV class="aim"><P>Aim: Storage of strains</P></DIV><DIV class="exp-contest"><UL class="list"><LI>Dh5α piGEM010 (p003 + Tag 21)</LI><LI>Dh5α piGEM015 (p004 + Tag 23)</LI><LI>Dh5α piGEM014 (p003 + Tag 22)</LI><LI>Dh5α piGEM008 (p002 + Tag 22)</LI><LI>Dh5α piGEM007 (p002 + Tag 21)</LI><LI>Dh5α piGEM011 (p003 + Tag 22)</LI><LI>Dh5α piGEM012 (p003 + Tag 23)</LI></UL></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.57">13.57 Mini Prep of  cultures inoculated on Friday 30.05.14 containing the nose plasmids</A></LEGEND><DIV class="exp-content"><UL class="list"><LI>piGEM010 = 22ng/µl</LI><LI>piGEM013 = 46 ng/µl</LI><LI>piGEM 009 = 13 ng/µl</LI></UL><P>low concentration: new transformation to obtain more plasmid DNA
		Strain: new competent <EM>E. coli</EM> XL1-blue</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.50">15.50 Miniprep of pMAD-fla-cup1-1  for miniprep</A></LEGEND><DIV class="exp-content"><P>The miniprep was performed according to the miniprep protocol.</P><P>Concentration 40 ng/µL
    low concentration: new transformation to gain more plasmid DNA should be done
    Strain: new competent <EM>E. coli</EM> XL1-blue</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.58">13.58 Bacillus Trafo with all Nose Plasmids containing one/no Degradation Tag</A></LEGEND><DIV class="aim"><P>Aim:  Create <I>Bacillus subtilis</I> strain 3610 containing the following plasmids: piGEM002 to 004 and piGEM007 to 016</P></DIV><DIV class="exp-content"><P>Add 5µl plasmid DNA and incubate for 30 min at 37°C, LB plate with 5 ng/µL Cm (prepared freshly)</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.59">13.59 Inoculation of <EM>Bacillus subtilis</EM> wildtype (3610), <I>gfp</I> and silver nose</A></LEGEND><DIV class="aim"><P>Aim: perform MTA to test the Ag concentrations where cells are still viable, since 1 mM until 10 µM were too high and the cells died</P></DIV><DIV class="exp-content"><P>Starting from 1 mM, eight serial dilutions have been prepared</P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.38">14.38 Purification of PheA-Arc1p-C-1x</A></LEGEND><DIV class="aim"><P>Aim: Purify PheA-Arc1p-C-1x in three steps</P></DIV><DIV class="exp-content"><P>The resuspended cells from 14.37 were lysed using a french press. The lysate was incubated on ice with 5 µg/mL DNAseA and RNAseI for 10 min and centrifuged (27000 rpm, 45 min, 4 °C) to pellet cell debris. The supernatant was filtered (0.45 µm) and PheA-Arc1p-C-1x purified from the cleared solution by NiNTA affinity chromatography. The combined elution fractions containing crude PheA-Arc1p-C-1x were concentrated to a volume of 2 mL and further purified using an anion exchange column to get rid of possible tRNA contaminations. Finally the buffer was changed to dialysis buffer and the protein concentrated and stored at -80 °C in aliquods for further use.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.1">18.1 Tumor Terminator - Dilution of Plasmid</A></LEGEND><DIV class="aim"><P>Aim: Creating a stock solution</P></DIV><DIV class="exp-content"><P>We received the synthetized DARPin-gene in a pEC-A2 vector with <I>Spe</I>I restriction sites at the 5' - and 3' -site with an amount of 2,7 µg plasmid per tube.</P><P>A stock solution was created by adding 27 µl water to the tube.</P><P>Concentration of stock solution = 100 ng/µl</P><P>A 1:10 dilution was created by using 1 µl of stock solution adding 9µL of water to an end concentration of 10ng/µl.</P><P>Concentration of stock dilution = 10 ng/µl.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.2">18.2 Transformation of <EM>E.coli</EM> XL1-Blue with pEX-A2-DARPin</A></LEGEND><DIV class="aim"><P>Aim: Transformation of <EM>E.coli</EM> XL1-Blue with pEX-A2-DARPin for amplification of the plasmid.</P></DIV><DIV class="exp-content"><P>1µl of the 1:10 dilution was used for transformation of <EM>E. Coli</EM> XL1-Blue</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="03.06.2014">03.06.2014</A></H2><FIELDSET class="exp18"><LEGEND><A name="exp18.3">18.3 Miniprep of pEX-A2-DARPin from <EM>E.coli</EM> XL1-Blue</A></LEGEND><DIV class="aim"><P>Aim: Receiving a higher amount of plasmid for further experiments.</P></DIV><DIV class="exp-content"><P>3 x 6ml LB-Amp were inoculated with clones from the successful transformed plate (18.2) and incubated at 37°C overnight.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.60">13.60 Microtiter plate assay with wildtype, gfp strain 186 and silver nose</A></LEGEND><DIV class="aim"><P>Aim: perform MTA to test the Ag concentrations where cells are still viable, since 1mM until 10 µM was too high and the cells died</P></DIV><DIV class="exp-content"><P>Starting from 1 mM 8 dilutions have been prepared.In the beginning Minimalmedium S750 has been prepared with 0.5‰ XyloseIn the wells the following mix was added to a total volume of 150 µl:135µl of MM (90‰) and Silver solution (10‰)15µl of cellsFinally 70 µl are added to avoid evaporation of the medium</P><P>Incubation for appr. 24 hours at 37°C shaking (Viktor AG Waldminghaus)</P><P>No positive GFP signal could be detected for the <I>B. subtilis</I> strain with the possible Ag sensitive promoter. The strain was restreaked onto a LB-Cm plate and he did not grow, which proved it to be a wrong strain, which has not been transformed successfully.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.61">13.61 Checking Bacillus Trafo with all Nose Plasmids containing one/no Degradation Tag</A></LEGEND><DIV class="exp-content"><P>Every transformation plate showed a huge growth of bacillus without single colonies. After one day of incubation this seems to be an unrealistic growth, which is why we planned to repeat the transformation after checking the growth of the <EM>Bacillus subtilis</EM> wildtype 3610 on LB-CM plate with 5 µg/ml chloramphenicol.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.62a">13.62 Checking Bacillus Trafo Wildtype 3610</A></LEGEND><DIV class="aim"><P>Aim: check if wildtype has a resistance against chloramphenicol or if the plates are faulty</P></DIV><DIV class="exp-content"><P><I>Bacillus subtilis</I> Wildtype 3610 was plated on LB-chloramphenicol plate with 5 µg/ml chloramphenicol and incubated overnight at 37°C.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.51">15.51 Making selection plates for clean deletions - integration of pMAD-construct into <I>Bacillus subtilis</I> chromosome</A></LEGEND><DIV class="aim"><P>Aim: For the integration of the constructs we cloned into pMAD many steps of a special selection are necessary.</P></DIV><DIV class="exp-content"><P>Antibiotics and chemicals:</P><UL class="list"><LI>Erythromycin 4 mg/ml</LI><LI>Lincomycin 25mg/ml</LI><LI>X-Gal 100mg/ml</LI></UL><P>For MLS selection plates erythromycin with 1 mg/ml were needed. Therefore the 4 mg/ml stock solution was diluted 1:4 by mixing 400 µl eryhtromycin with 1200 µl of 70% ethanol.</P><P>3 types of plates were made:</P><UL class="list"><LI>1.	MLS selectin: 800 mL LB agar + 800 µl eryhtromycin + 800 µl lincomycin</LI><LI>2.	MLS X-Gal: 800 mL LB agar + 800 µl eryhtromycin + 800 µl lincomycin + 800 µl X-Gal</LI><LI>3. X-Gal: 800 mL LB agar + 800 µL X-Gal
    </LI></UL></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="04.06.2014">04.06.2014</A></H2><FIELDSET class="exp18"><LEGEND><A name="exp18.4">18.4 Miniprep of pEX-A2-DARPin from <EM>E.coli</EM> XL1-Blue</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Plasmid pEX-A2 + DARPinEc1</TH><TH scope="col">Concentration after prep (ng/µl)</TH></TR><TR><TD>Clone 1</TD><TD>398,7</TD></TR><TR><TD>Clone 2</TD><TD>340</TD></TR><TR><TD>Clone 3</TD><TD>309</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.52">15.52 Sequencing of pMAD-fla-cup1-1</A></LEGEND><DIV class="aim"><P>Aim: Checking the correct insertion of the metallothionein-domain into the hag-flank construct</P></DIV><DIV class="exp-content"><P>Making a pre-mix for Eurofins MWG prepairing the sequencing. Primers from Florian Altegoer were used:</P><UL class="list"><LI>Flo83- Hag11_fw 1:10 with 1µl primer and 9µl water</LI><LI>	Flo86- Hag12_rv 1:10 with 1µl primer and 9µl water</LI></UL><TABLE width="100%" border="1"><TBODY><TR><TH scope="col"> </TH><TH scope="col">Pre-mix 1 (µL)</TH><TH scope="col">Pre-mix 2 (µL)</TH></TR><TR><TH scope="row">piGEM-016 pMAD-cup1-140ng/µL</TH><TD>10</TD><TD>10</TD></TR><TR><TH scope="row">Flo83- Hag11_fw</TH><TD>2</TD><TD>-</TD></TR><TR><TH scope="row">Flo86- Hag12_rv</TH><TD>-</TD><TD>2</TD></TR><TR><TH scope="row">Total Volume</TH><TD>12</TD><TD>12</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Label</TH><TH scope="col">Pre-mix 1</TH><TH scope="col">Primer</TH></TR><TR><TH scope="row">AGB0023375</TH><TD>Pre-mix 1</TD><TD>Flo83 – Hag11_fw</TD></TR><TR><TH scope="row">AGB0023374</TH><TD>Pre-mix 1</TD><TD>Flo86 - Hag12_rv</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.62b">13.62 Checking Bacillus Trafo Wildtype 3610</A></LEGEND><DIV class="aim"><P>Aim: check if wildtype has a resistance against chloramphenicol or if the plates are not effective/ made wrong</P></DIV><DIV class="results"><P>Result: <I>Bacillus subtilis</I> Wildtype 3610 grew on the chloramphenicol plate which shows us that there has to be a mistake in making the LB-chloramphenicol plates or the WT has a resistance.</P></DIV><DIV class="exp-content"><P>New plates were made with new chloramphenicol-stocks with 25 mg/ml and 5 mg/ml.</P><P>3 aliquots of competent <EM>B.s.</EM> WT3610 were plated on new LB-chloramphenicol plates and incubated at 37°C.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.5">18.5 <I>Spe</I>I restriction pEX-A2-DARPin &amp; pMAD-hag-flank (piGEM-005)</A></LEGEND><DIV class="aim"><P>Aim: Receiving iGEM-005 as linearized vector for cloning the DARPin gene with <I>Spe</I>I sites into pMAD-fla (iGEM-005) </P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Restriction</TH><TH scope="col"><STRONG>pEX-A2-DARPin  digest (µl)</STRONG></TH><TH scope="col"><STRONG>pMAD-Hag-Flank  digest (µl</STRONG>)</TH><TH scope="col"><STRONG>Spe1-Mastermix</STRONG></TH></TR><TR><TH scope="row">pEX-A2+DARPin (398 µg/µL)</TH><TD>15 = ca. 6µg</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">pMAD-Fla (114 µg/µL)</TH><TD>-</TD><TD>10= ca. 1,14 µg</TD><TD>-</TD></TR><TR><TH scope="row"><EM>Spe</EM>I</TH><TD>-</TD><TD>-</TD><TD>1,5</TD></TR><TR><TH scope="row">CutSmart-Buffer (10x)</TH><TD>-</TD><TD>-</TD><TD>2</TD></TR><TR><TH scope="row">H2O</TH><TD>-</TD><TD>-</TD><TD>16,5</TD></TR><TR><TH scope="row">Mastermix</TH><TD>5</TD><TD>10</TD><TD>-</TD></TR><TR><TH scope="row">Total</TH><TD>20</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><P>Incubation at 37°C for 2h and heat shock at 81°C for 5 min. After that the pMAD-hag-flank-construct was purified with the OMEGA gel extraction kit according to the protocol.</P><P>c(digested piGEM-005)= 4 ng/µl</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.62c">13.62 Checking Bacillus Trafo Wildtype 3610</A></LEGEND><DIV class="aim"><P>Aim: check if wildtype has a resistance against chloramphenicol or if the plates are not effective/ made wrong</P></DIV><DIV class="exp-content"><P>Result: <I>Bacillus subtilis</I> Wildtype 3610 grew on the chloramphenicol plate. We have already repeated making plates so it might be that the WT has got a resistance while making it competent. A new WT was plated on 2 chloramphenicol plates with 5 µg/mL (old and new one) and 2x LB plates from gly stocks.</P><P>Incubation at 37°C overnight.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="05.06.2014">05.06.2014</A></H2><FIELDSET class="exp14"><LEGEND><A name="exp14.39">14.39 Creating PheA-Arc1p-C-0x</A></LEGEND><DIV class="aim"><P>Aim: Using Round-the-horn mutagenesis to create PheA-Arc1p-C-0x-pET28a</P></DIV><DIV class="exp-content">
    The PCR with 5' phosphorylated TG_0xGSSG FP und TG_0xGSSG RP should generate the 7491 bp fragment for ligation from the methylated template PheA-Arc1p-C-8x-pET28a from 14.5.<P>The used Primer concentration was 1 
    µM, the concentration of the template was 1 ng/µL and we used a hot start to avoid unspecific annealing of primers.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Volume [µL]</TH></TR><TR><TH scope="row">Water</TH><TD>29.5</TD></TR><TR><TH scope="row">5x GC Buffer</TH><TD>10</TD></TR><TR><TH scope="row">DMSO</TH><TD>2.5</TD></TR><TR><TH scope="row">TG_0xGSSG FP</TH><TD>2.5</TD></TR><TR><TH scope="row">TG_0xGSSG RP</TH><TD>2.5</TD></TR><TR><TH scope="row">PheA-Arc1p-C-8x-pET28a</TH><TD>1</TD></TR><TR><TH scope="row">dNTPs</TH><TD>1</TD></TR><TR><TH scope="row">Phusion-Polymerase</TH><TD>1</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature [°C]</TH><TH scope="col">Time [min:sec]</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>2:00</TD></TR><TR><TH scope="row">2</TH><TD>add Polymerase</TD><TD> </TD></TR><TR><TH scope="row">3</TH><TD>98</TD><TD>0:10</TD></TR><TR><TH scope="row">4</TH><TD>67.5</TD><TD>0:30</TD></TR><TR><TH scope="row">5</TH><TD>72</TD><TD>4:00</TD></TR><TR><TH scope="row">6</TH><TD>go to 3</TD><TD>32x</TD></TR><TR><TH scope="row">7</TH><TD>72</TD><TD>10:00</TD></TR><TR><TH scope="row">8</TH><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P>The amplified fragment was purified on an agarose gel on which it showed the correct size. The extracted linear template was then digested to degrade the original unmutated plasmid using DpnI.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Volume [µL]</TH></TR><TR><TH scope="row">10x CutSmart Buffer</TH><TD>3.6</TD></TR><TR><TH scope="row">Plasmid</TH><TD>30</TD></TR><TR><TH scope="row">DpnI</TH><TD>3</TD></TR><TR><TH scope="row">Total Volume</TH><TD>36.6</TD></TR></TBODY></TABLE><P>The reaction mixture was incubated at 37 °C and 300 rpm for 1 h and afterwards heated to 80 °C for 20 min.</P><P>In order to create the circular plasmid 100 ng of the linear template were mixed with 1 µL 10x T4-Ligasebuffer,  1 µL T4-Ligase and water to give a total volume of 10 µL. This mixture was incubated for 9 h at 16 °C. The resulting plasmid was dialysed with water to decrease the salt concentration for the following transformation of <I>E. coli</I> Top10 electrocompetent cells with the plasmid.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="06.06.2014">06.06.2014</A></H2><FIELDSET class="exp19"><LEGEND><A name="exp19">19. Crystallization of flagellin</A></LEGEND><DIV class="concent"><P>In order to crystallize flagellin with a domain it is necessary to produce the protein by <I>E. coli</I> in a high concentration.  For that aim the Fla-construct will be amplified out of pET24d which includes a domain. For elimination of the flanks a PCR has to be performed which just amplifies hag with domain and creates <I>Nco</I>I and <I>Bam</I>HI restriction sites. After purification and digest with <I>Nco</I>/<I>Bam</I> the fragment can be ligated into digested pET24d . After transformation the production can be induced by IPTG.</P></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.1">19.1 Elimination of flanks from fla-construct including <I>cup</I>1-1 from pET24d</A></LEGEND><DIV class="aim"><P>Aim: Preparation for using pET24d as expression vector for flagellin</P></DIV><DIV class="exp-content"><P>The flagellin construct was  amplified with specifically designed primers (95 and 54 from Florian Altegoer), which allow the  amplification of the construct with <I>cup</I>1-1 without flanks (just Hag) including  <I>Nco</I>I &amp;<I>Bam</I>HI restriction sites.
      Template dilution 1:10:
      1 µL pET24d-<I>cup</I>1-1 was mixed with  9 µL to an 1:10 dilution of 16,3 ng/µL
      Primer dilution 1:10:
      The primers were diluted 1:10 by mixing 10 µL of primers with 90 µL millipore water  each.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix [µl]</TH><TH scope="col">reaction with phusion</TH><TH scope="col">reaction with Q5 mastermix</TH></TR><TR><TH scope="row">pET24d-<I>cup</I>1-11:10 dilution</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Primer 54Flo</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Primer 56Flo</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Phusion</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Phusion Buffer 5x</TH><TD>10</TD><TD>-</TD></TR><TR><TH scope="row">dNTP Mix</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Water</TH><TD>35</TD><TD>22</TD></TR><TR><TH scope="row">Q5 Mastermix</TH><TD>-</TD><TD>25</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD><TD>20</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>2 min</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>30x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The PCR was run over night.</P></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.2">19.2 Restriction digest of pET24d with <I>Nco</I>I and <I>Bam</I>HI</A></LEGEND><DIV class="aim"><P>Aim: Creating Nco/Bam restriction sites in pET24d for cloning PCR fragments amplified with primers flo56 and flo54 which contain these sites as well.</P></DIV><DIV class="exp-content"><P>Isolated pET24d was cut with <I>Nco</I>I and <I>Bam</I>HI in order to create restriction sites for ligating the Flagellin/ Hag-Domain fragments into the vector. The Hag-fragments which were amplified via PCR with primers flo54 and 56 contain <I>Nco</I>I/ <I>Bam</I>HI restriction sites for that purpose as well.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Volume [µl]</TH></TR><TR><TH scope="row">pET24d (148 ng/µL)</TH><TD>7</TD></TR><TR><TH scope="row">CutSmart 10x</TH><TD>2</TD></TR><TR><TH scope="row"><I>Nco</I>I</TH><TD>0,5</TD></TR><TR><TH scope="row"><I>Bam</I>HI</TH><TD>0,5</TD></TR><TR><TH scope="row">Water</TH><TD>10</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD></TR></TBODY></TABLE><P>Incubation for 1 hour at 37°C and heat shocked at 80°C.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.52">15.52 Miniprep of different plasmids</A></LEGEND><DIV class="aim"><P>Aim: Receiving more plasmid DNA for further experiments</P></DIV><DIV class="exp-content"><P>Colonies were picked from plates and 8ml LB with necessary antibiotics each inoculated, followed by incubation at 37°C overnight.</P><P>Clones were picked from plates with plasmids:</P><UL class="list"><LI>pET24d-Fla-Cup1-1 (piGEM-006)</LI><LI>pMAD-Hag-Flank (piGEM-005)</LI><LI>pMAD-Hag-Flank -Cup1-1 (piGEM-016)</LI><LI>pEX-A2+DARPin (piGEM-017)</LI></UL></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="07.062014">07.062014</A></H2><FIELDSET class="exp15"><LEGEND><A name="exp15.53">15.53 Sequencing pMAD-fla-cup1-1 </A></LEGEND><DIV class="exp-content"><P>The sequencing showed the correct insertion of the cup-1-1 gene into the hag-flank construct but contained a stop codon, which was not removed by the used reverse primer. Unfortunately we used primers for amplification of the whole cup1-1 gene and not just for the truncated gene coding  for the metallothionein domain only. Therefore new primers had to be designed an the Gibson assembly for integration of the metallothionein domain into pMAD and pET24d has to be repeated.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.6">18.6 gel extraction &amp; purification of DARPin gene </A></LEGEND><DIV class="aim"><P>Aim: Receiving DARPin gene with <I>Spe</I>I sites for ligation into linearized piGEM-005</P></DIV><DIV class="exp-content"><P>The with <I>Spe</I>I digested pEX-A2 was purified via electrophoresis with an 1% agarose gel in order to divide pEX-A2 backbone and the DARPin gene (489 bp). The digest attempt was splitted and loaded into two pockets to be sure not to overload the gel. The correct bands were cut out and the DNA fragment was rescued according to the protocol of our OMEGA gel extraction kit.</P><P>Expected bands:
      DARPin gene: 489 bp
      pEX-A2 backbone: 1961bp
      The concentration after gel  extraction: 9 ng/µl</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.7">18.7 Ligation of DARPin gene and linearized piGEM-005</A></LEGEND><DIV class="aim"><P>Aim: Ligation of DARPin- (from 18.6) into <I>Spe</I>1 digested piGEM-005 from 18.5 for integration of the DARPin-Flagellin construct into <I>Bacillus subtilis</I> chromosome</P></DIV><DIV class="exp-content"><P>The DARPin gene with <I>Spe</I>I sites was used as insert for the ligation into linearized pMAD (from 18.5) which contains the designed restriction site. The ligation attempt was done twofold and calculated by the Ligation Calculator from the iGEM-Team Texas in 2011:</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Volume [µl]</TH></TR><TR><TH scope="row">Insert (DARPin, c= 9 ng/µL)</TH><TD>2</TD></TR><TR><TH scope="row">Vector (piGEM-005, c=4ng/µL)</TH><TD>10</TD></TR><TR><TH scope="row">Ligase-Buffer 10x</TH><TD>2</TD></TR><TR><TH scope="row">T4 Ligase</TH><TD>1</TD></TR><TR><TH scope="row">Milipore water</TH><TD>5</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD></TR></TBODY></TABLE><P>Incubation was performed overnight at room temperature.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.63">13.63 checking Bacillus WT 3610  growth on 5µg/µL CM</A></LEGEND><DIV class="aim"><P>Aim: check if wildtype has a resistance against chloramphenicol (CM) </P></DIV><DIV class="exp-content"><P>Result: <EM>Bacillus subtilis</EM> Wildtype 3610 grew on the CM plate.</P><P>In order to make out if a selection is possible WT3610 was plated out  from an LB plate (13.62)  on 25 ng/ µL CM  to check were the threshold of CM for <I>B. subtilis</I> is.</P><P>Incubation at 30°C over two days.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="08.06.2014">08.06.2014</A></H2><FIELDSET class="exp18"><LEGEND><A name="exp18.8">18.8 new <I>Spe</I>I restriction of DARPin-gene in pEX-A2 &amp; pMAD-hag-flank (piGEM-005)</A></LEGEND><DIV class="aim"><P>Aim: Receiving new insert and vector in case of an unsuccessful ligation for further attempts. </P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Restriction</TH><TH scope="col">pEX-A2-DARPin digest (µl)</TH><TH scope="col">pMAD-Hag-Flank digest (µl)</TH><TH scope="col"><I>Spe</I>I-Mastermix</TH></TR><TR><TH scope="row">pEX-A2+DARPin (398 µg/µL)</TH><TD>10 = ca. 4µg</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">pMAD-Fla (150 µg/µL)</TH><TD>-</TD><TD>7= ca. 1 µg</TD><TD>-</TD></TR><TR><TH scope="row"><I>Spe</I>I</TH><TD>-</TD><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">CutSmart-Buffer (10x)</TH><TD>-</TD><TD>-</TD><TD>2,5</TD></TR><TR><TH scope="row">Water</TH><TD>-</TD><TD>-</TD><TD>21,5</TD></TR><TR><TH scope="row">Mastermix</TH><TD>10</TD><TD>13</TD><TD>-</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>20</TD><TD>25</TD></TR></TBODY></TABLE><P>Incubation was performed for 2 h at 37°C and heat inactivated at 80°C for  5 minutes.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.9">18.9 Transformation of ligation from 18.7</A></LEGEND><DIV class="exp-content"><P>The whole 20µL attempt from the ligations were used to transform <EM>E.Coli  XL1-Blue</EM>, which were plated on LB-Amp plates.   Additionally a negative control was created by transforming <EM>E.Coli XL1-Blue </EM> with 1 µL <I>Spe</I>I cut  piGEM-005 as  well.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.10">18.10 Gel extraction for isolation of DARPin gene out of pEX-A2</A></LEGEND><DIV class="aim"><P>Aim: Receiving new insert in case of an unsuccessful ligation for further attempts like in 18.6</P></DIV><DIV class="exp-content"><P>The <I>Spe</I>I digested pEX-A2 was purified via electrophoresis with an 1% agarose gel like in 18.6. The correct bands were cut out and the DNA fragment was purified according to the  protocol of our OMEGA gel extraction kit.</P><P>Expected bands:
      DARPin gene: 489 bp
      pEX-A2 backbone: 1961bp
      The concentration after gel extraction: 10 ng/µl</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="09.06.2014">09.06.2014</A></H2><FIELDSET class="exp18"><LEGEND><A name="exp18.12">18.12 Checking transformation of overnight ligation from 18.7 </A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Plate</TH><TH scope="col">Amount of colonies</TH></TR><TR><TH scope="row">Negative control</TH><TD>6</TD></TR><TR><TH scope="row">Ligation attempt 1</TH><TD>7</TD></TR><TR><TH scope="row">Ligation attempt 2</TH><TD>0</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.13">18.13 Checking success of ligation via cPCR</A></LEGEND><DIV class="aim"><P>Aim: A cPCR with the primers 54 and 56 shows the success of ligation by amplification of the Hag-construct.</P></DIV><DIV class="exp-content"><P>In order to check the positive  insertion of the DARPin gene into pMAD a PCR amplification with primers 54 and  56 for the flagellin should generate an ca. 1500 bp fragment on the gel.  Negative clones would just show a 1000 bp band because of the  amplification of the Hag-<I>Spe</I>I site construct. The picked  clones were transferred on a new LB-amp plate.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix</TH><TH scope="col">Master-Mix for 7 attempts (µL)</TH><TH scope="col">Mix for PCR attempt (µL)</TH></TR><TR><TH scope="row">Colony</TH><TD>-</TD><TD>part of a colony</TD></TR><TR><TH scope="row">Primer Flo 54</TH><TD>3,5</TD><TD>-</TD></TR><TR><TH scope="row">Primer Flo 56</TH><TD>3,5</TD><TD>-</TD></TR><TR><TH scope="row">Phusion</TH><TD>7</TD><TD>-</TD></TR><TR><TH scope="row">Phusion Buffer 5x</TH><TD>28</TD><TD>-</TD></TR><TR><TH scope="row">dNTP Mix</TH><TD>3,5</TD><TD>-</TD></TR><TR><TH scope="row">Water</TH><TD>94,5</TD><TD>-</TD></TR><TR><TH scope="row">Mastermix</TH><TD>-</TD><TD>20</TD></TR><TR><TH scope="row">Total Volume</TH><TD>140</TD><TD>20</TD></TR></TBODY></TABLE><P>Expected bands:
	  Hag-DARPin-construct: ca. 1500 bp</P><P>The gel shows bands at 1000 which means that the clones contain just relegated pMAD vector and are all negative.
The ligation had to be repeated.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.14">18.14 Dephosphorylation of linearized piGEM-005</A></LEGEND><DIV class="aim"><P>Aim: Dephosphorylation of 5'-end with antarctic phosphatase in order to decrease the religation of the linearized piGEM-005 from 18.8</P></DIV><DIV class="exp-content"><P>The cut <I>Spe</I>I cut piGEM-005 was dephosporylated at the 5'-end with antarctic phosphatase to prevent the vector from relegation.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Volume [µl]</TH></TR><TR><TH scope="row">Vector (piGEM-005, c=148  ng/µL)</TH><TD>20 (=ca. 2 µg)</TD></TR><TR><TH scope="row">Antarctic phos.-Buffer 10x</TH><TD>3</TD></TR><TR><TH scope="row">Antarctic phosphatase</TH><TD>1</TD></TR><TR><TH scope="row">Milipore Water</TH><TD>6</TD></TR><TR><TH scope="row">Total Volume</TH><TD>30</TD></TR></TBODY></TABLE><P>The sample was incubated for 1 h at 37°C at heat inactivated at 70°C for  10 minutes.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.15">18. new ligation of linearized/ dephosphorylized piGEM-005 &amp;  DARPin gene</A></LEGEND><DIV class="aim"><P>Aim: Dephosphorylation piGEM-005 was ligated with extracted DARPin gene again</P></DIV><DIV class="exp-content"><P>Because of the negative colony PCR from 18.13 a new ligation with dephosphorylized piGEM-005 from 18.14 and new extracted DARPin gene from 18.10 was performed in 2 attempts.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Volume [µl]</TH></TR><TR><TH scope="row">Insert (DARPin, c= 10  ng/µL)</TH><TD>12,4</TD></TR><TR><TH scope="row">Vector (piGEM-005, c= 148  ng/µL)</TH><TD>4,6</TD></TR><TR><TH scope="row">Ligase-Buffer 10x</TH><TD>2</TD></TR><TR><TH scope="row">T4 Ligase</TH><TD>1</TD></TR><TR><TH scope="row">Milipore Water</TH><TD>5</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD></TR></TBODY></TABLE><P>Incubation 3h at room temperature</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.16">18.16 Transformation of overnight ligation from 18.15</A></LEGEND><DIV class="exp-content"><P>The whole 20 µL reaction of the ligations (18.15) were used to transform <EM>E. coli</EM>  XL1-Blue and plated on LB-Amp plates.   Additionally a negative control was created by transforming <EM>E.Coli XL1-Blue </EM> with 1µL <I>Spe</I>I digested  piGEM-005 as  well.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.64">13.64 checking Bacillus WT 3610 growth on 5, 10 &amp; 15 µg/µL CM</A></LEGEND><DIV class="aim"><P>Aim: check if wildtype has a resistance against CM </P></DIV><DIV class="exp-content"><P>Result: <EM>Bacillus subtilis</EM> Wildtype 3610 did not grow on the 25 ng/µl CM plate</P><P>In order to check at which concentration CM Bacillus cannot survive he  was plated out from an LB plate on LB-CM plates with different CM  concentrations (5, 10, 15 ng/mL).</P><P>Incubation at 30°C over night.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="10.06.2014">10.06.2014</A></H2><FIELDSET class="exp18"><LEGEND><A name="exp18.17">18.17 Checking Transformation of ligation from 18.16</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Plate</TH><TH scope="col">Amount of colonies</TH></TR><TR><TH scope="row">Negative control</TH><TD>1</TD></TR><TR><TH scope="row">Ligation attempt 1</TH><TD>2</TD></TR><TR><TH scope="row">Ligation attempt 2</TH><TD>&gt;80</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.18">18.18 Checking success of ligation via cPCR</A></LEGEND><DIV class="aim"><P>Aim: A cPCR with the primers 54 and 56 shows the success of ligation by amplification of the Hag-construct.</P></DIV><DIV class="exp-content"><P>In order to check the positive  insertion of the DARPin gene into pMAD a PCR amplification with primers 54 and  56 for the flagellin should generate an ca. 1500 bp fragment on the gel.  Negative clones would show still just a 1000 bp band because of the  amplification of the Hag-Spe1 site construct. The picked  clones were transferred on a new LB-amp plate. 12 clones were picked.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix</TH><TH scope="col">Master-Mix for 14 attempts (µL)</TH><TH scope="col">Mix for PCR attempt (µL)</TH></TR><TR><TH scope="row">Colony</TH><TD>-</TD><TD>part of a colony</TD></TR><TR><TH scope="row">Primer Flo 54</TH><TD>7</TD><TD>-</TD></TR><TR><TH scope="row">Primer Flo 56</TH><TD>7</TD><TD>-</TD></TR><TR><TH scope="row">Phusion</TH><TD>14</TD><TD>-</TD></TR><TR><TH scope="row">Phusion Buffer 5x</TH><TD>56</TD><TD>-</TD></TR><TR><TH scope="row">dNTP Mix</TH><TD>7</TD><TD>-</TD></TR><TR><TH scope="row">Water</TH><TD>189</TD><TD>-</TD></TR><TR><TH scope="row">Mastermix</TH><TD>-</TD><TD>20</TD></TR><TR><TH scope="row">Total Volume</TH><TD>280</TD><TD>20</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>1min 45sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1 min</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>30x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>Expected bands:
      Hag-DARPin-construct: ca. 1500 bp</P><P>Clone 1, 2, 3, 4, 5, 7, 9, 11 and 12 were the positive clones at 1500 bp.</P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.40">14.40 Overnight culture</A></LEGEND><DIV class="aim"><P>Aim: Amplify Plasmid in an overnight culture</P></DIV><DIV class="exp-content"><P>Six clones of the transformed <I>E. coli</I> Top10 cells from 14.39 were picked and used to inoculate overnight cultures (6 x 5 mL).</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.9">18.19 Miniprep of positive clones</A></LEGEND><DIV class="aim"><P>Aim: In order to check the orientation of the insert in the vector of the positive clones minipreps were inoculated for further experiments</P></DIV><DIV class="exp-content"><P>Because of the ligation via two  <I>Spe</I>I sites, the chance is 50:50 that the clones contain the  pMAD-Hag-Flank-DARPin construct in the right orientation (Startcodon on the  Hag1-site). For test digests a higher amount of plasmid is needed, which is achieved by inoculation of minipreps.</P><P>The clones were used to inoculate  7 mL LB-Amp and incubated at 37°C over night.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.64">13.64 checking <I>Bacillus subtilis</I> WT 3610 growth on 25 &amp; 35 µg/µL CM</A></LEGEND><DIV class="aim"><P>Aim: check if wildtype has a resistance against CM </P></DIV><DIV class="exp-content"><P>Result: <I>Bacillus subtilis</I> Wildtype 3610 grew on the 5, 10 &amp; 15µg/µl LB-CM plate.</P><P>We saw that the Bacillus grew on almost every CM concentrations except 25  µg/µL. So we transformed competent WT3610 with Nose plasmid piGEM-003 as a  positive control and plated them out on LB-CM with 25 µg/µL and 35 µg/µL.</P><P>Additionally, competent wildtype  3610 cells were transformed with piGEM-004 and plated out on LB-CM with 5, 10 and 15 µg/mL CM  to check, if these cells can be selected better. </P><P>Incubation at 30°C for 2 days.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="11.06.2014">11.06.2014</A></H2><FIELDSET class="exp18"><LEGEND><A name="exp18.19">18.19 Miniprep of positive clones</A></LEGEND><DIV class="exp-content"><P>The concentration of the isolated plasmids was between 100 and 150 ng/µL.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.20">18.20 Test digest with <I>Eco</I>RV </A></LEGEND><DIV class="aim"><P>Aim: Checking the right orientation of the insert in the isolated plasmids</P></DIV><DIV classs="exp-content"><P>The plasmids from the positive clones were digested with <I>Eco</I>RV so that specific fragments emerge when cutting into the DARPin gene and the vector backbone.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Volume [µl]</TH><TH scope="col">Mastermix for 10 attempts [µl]</TH></TR><TR><TH scope="row">Plasmid of clone</TH><TD>2 (ca. 200ng)</TD><TD>-</TD></TR><TR><TH scope="row">Cutsmart 10x</TH><TD>-</TD><TD>8</TD></TR><TR><TH scope="row"><I>Eco</I>RV</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Millipore water</TH><TD>-</TD><TD>71</TD></TR><TR><TH scope="row">Mastermix</TH><TD>8</TD><TD>-</TD></TR><TR><TH scope="row">Total Volume</TH><TD>10</TD><TD>80</TD></TR></TBODY></TABLE><P>Right orientation:    7900 –<STRONG>1570</STRONG>– 2500 –<STRONG>55</STRONG>
      Wrong orientation: 7900 –<STRONG> 1170 </STRONG>– 2500 –<STRONG>460 </STRONG></P><P>Clone 1, 2, 3, 4, 7, 9 and 11 have the right orientation.</P></DIV></FIELDSET><FIELDSET class="exp20"><LEGEND><A name="exp20.1">20.1 inoculation of preculture from <I>Bacillus subtilis</I> WT 3610</A></LEGEND><DIV class="aim"><P>Aim: Preparation of for mainculture the next day</P></DIV><DIV class="exp-content"><P>2 x 6 mL of LB were inoculated with <I>Bacillus subtilis</I> WT3610 from an LB plate and incubated at 37°C overnight.</P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.41">14.41 Preparation of plasmids</A></LEGEND><DIV class="aim"><P>Aim: Purify the amplified plasmids from overnight cultures</P></DIV><DIV class="exp-content"><P>The plasmids were purified from the overnight cultures created in 14.40 using Sigma GenElute Plasmid Miniprep Kit following the included protocol. The plasmids were eluted using 45 µL water.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="12.06.2014">12.06.2014</A></H2><FIELDSET class="exp18"><LEGEND><A name="exp18.21">18.21 PCR with pMAD-Hag-Flank-DARPin (piGEM-019) and piGEM-005</A></LEGEND><DIV class="aim"><P>Aim: Amplification  of Hag-<I>Spe</I>I-construct and Hag-DARPin construct with <I>Nco</I>I/<I>Bam</I>HI sites for  cloning into pET24d → crystallization of domains.</P></DIV><DIV class="exp-content"><P>The  templates (piGEM-005, prepped pasmid from clone 7 &amp; 12 ) were diluted 1:10.  piGEM-005 should function as a negative control. Clone 12 was picked in case  the second orientation could be used for different experiments. 5 µL PCR  product + 1 µL loading dye were analysed on an 1% agarose gel.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix</TH><TH scope="col">Volume</TH></TR><TR><TH scope="row">Template 1:10</TH><TD>1</TD></TR><TR><TH scope="row">Primer Flo 54</TH><TD>1</TD></TR><TR><TH scope="row">Primer Flo 56</TH><TD>1</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>1</TD></TR><TR><TH scope="row">Phusion Buffer 5x</TH><TD>10</TD></TR><TR><TH scope="row">dNTP Mix</TH><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>35</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>1min 45sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>2 min</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>30x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE></DIV><P>Expected  bands: clone 7 &amp; 12 PCR product at ca. 1500bp –  piGEM-005 PCR product at 1kb</P></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.42">14.42 Transformation of BL21 cells</A></LEGEND><DIV class="aim"><P>Aim: Transform E. coli BL21(DE3) with PheA-Arc1p-C-0x-pET28a</P></DIV><DIV class="exp-content"><P>40 µL E. coli BL21(DE3) electrocompetent cells were transformed with 1 ng of the previously purified plasmid PheA-Arc1p-C-0x-pET28a from two different clones, incubated for 1 h at 37 °C and plated out on LB-Kan<SUP>50</SUP> plates that were incubated overnight at 37 °C.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.22">18.22 restriction digest of Hag PCR fragments from 18.21 with <I>Nco</I>I/<I>Bam</I>HI</A></LEGEND><DIV class="aim"><P>Aim: creating restriction sites for cloning into pET24d cut with <I>Nco</I>I/<I>Bam</I>HI</P></DIV><DIV class="exp-content"><P>The PCR  products from 18.21 were digested with <I>Nco</I>I and <I>Bam</I>HI to create restriction  sites in order to ligate the fragments into <I>Nco</I>I/<I>Bam</I>HI cut pET24d.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Volume [µl]</TH></TR><TR><TH scope="row">PCR products</TH><TD>45</TD></TR><TR><TH scope="row">Cutsmart 10x</TH><TD>5,2</TD></TR><TR><TH scope="row"><I>Nco</I>I</TH><TD>1</TD></TR><TR><TH scope="row"><I>Bam</I>HI</TH><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>2,8</TD></TR><TR><TH scope="row">Total Volume</TH><TD>52</TD></TR></TBODY></TABLE><P> Incubation for 2 hours at 37°C with heat shock at 80°C.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.23">18.23 Ligation of pET24d and Hag PCR fragments from 18.22 cut with <I>Nco</I>I/<I>Bam</I>HI</A></LEGEND><DIV class="aim"><P>Aim: cloning inserts into pET24d for crystallization</P></DIV><DIV class="exp-content"><P>The <I>Nco</I>I/<I>Bam</I>HI digested flagellin PCR products from 18.22 (from clone 7 and piGEM-005) were ligated with <I>Nco</I>I/<I>Bam</I>HI digested pET24d. </P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Pet-Hag attempt (µL)</TH><TH scope="col">Pet-Hag-DARPin attempt (µL)</TH></TR><TR><TH scope="row">Insert (PCR fragments , c= ca. 400 ng/µL)</TH><TD>2</TD><TD>2</TD></TR><TR><TH scope="row">Vector (pet24d, c= 50ng/µL)</TH><TD>5</TD><TD>5</TD></TR><TR><TH scope="row">Ligase-Buffer 10x</TH><TD>2</TD><TD>2</TD></TR><TR><TH scope="row">T4 Ligase</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Millipore Water</TH><TD>10</TD><TD>10</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><P>Incubation over night at room temperature.</P></DIV></FIELDSET><FIELDSET class="exp20"><LEGEND><A name="exp20.2">20.2 Transformation of competent <EM>Bacillus subtilis</EM></A></LEGEND><DIV class="aim"><P>Aim: transformation of <I>Bacillus subtilis</I> WT3610 with piGEM005 and piGEM018</P></DIV><DIV class="exp-content"><P>100 µL  of overnight culture were added to 10 mL of MNGE-Medium and incubated to an  OD<SUB>600</SUB> of 1,1-1,3 at 37°C which could take 4-5 hours.</P><P>After  reaching an OD of 1,1-1,3 2 x 400 µL of the culture were transformed with 1,5 µg  piGEM-005 and -018. After one hour incubation at 37°C 100 µL expression mix were  added and incubated for one hour as well.</P><P>In the  end the 500 µL sample was plated out on MLS-X-Gal plates and incubated at 30  °C until colonies could be seen. </P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="13.06.2014">13.06.2014</A></H2><FIELDSET class="exp18"><LEGEND><A name="exp18.24">18.24 Transformation of ligation</A></LEGEND><DIV class="aim"><P>Aim: checking success of ligation and amplification of ligated construct</P></DIV><DIV class="exp-content"><P>The  whole 20 µL attempt was used to transform <I>E. coli</I> X1-Blue, plated out on LB-Can  plates and incubated at 37°C overnight.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.25">18.25 Miniprep of pMAD-Hag-Flank-DARPin (piGEM-018)</A></LEGEND><DIV class="aim"><P>Aim: increase amount of plasmid for work and storage</P></DIV><DIV class="exp-content"><P>2 x 6 mL of LB-Amp were inoculated  with a colony from a piGEM-018 transformation plate and incubated at 37°C  overnight. </P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.26">18.26 Sequencing of constructs</A></LEGEND><DIV class="aim"><P>Aim: checking the right insertion into the plasmids</P></DIV><DIV class="exp-content"><P>pMAD-Hag-Flank-DARPin was sent in form of a premix with primers for sequencing. 10 µL with 50-100 ng/µL were mixed with 2 µL of primers for  sequencing according to the following scheme:</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Premix</TH><TH scope="col">Label-Nr.</TH><TH scope="col">Construct</TH><TH scope="col">Primer</TH></TR><TR><TH scope="row">1</TH><TD>391</TD><TD>pMAD-Hag-Flank-DARPin</TD><TD>Flo54 (Hag-F-Nco)</TD></TR><TR><TH scope="row">2</TH><TD>392</TD><TD>pMAD-Hag-Flank-DARPin</TD><TD>Flo56 (Hag-R-Bam)</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.65">13.65 Checking growth of <I>Bacillus subtilis</I> WT3610</A></LEGEND><DIV class="aim"><P>Aim: check which WT can be used for selection with Nose plasmids</P></DIV><DIV class="exp-content"><P>Competent 3610 were transformed with piGEM-002, -003  and -004 by incubating the aliquots with 5 µL plasmid for 30 min at 37°C. After  half an hour the whole sample was plated out on LB-Cm (5 ng/µL). </P><P>Incubation at 30°C for 2 days.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="14.06.2014">14.06.2014</A></H2><FIELDSET class="exp18"><LEGEND><A name="exp18.27">18.27 miniprep of plasmids from 18.25</A></LEGEND><DIV class="aim"><P>Aim: isolation of plasmids</P></DIV><DIV class="exp-content"><P>The miniprep of pMAD-Hag-Flank-DARPin was done according to the protocol of the OMEGA DNA kit. </P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.28">18.28 pET24d digest with Nco/Bam</A></LEGEND><DIV class="aim"><P>Aim: preparing vector for new ligation attempts</P></DIV><DIV class="exp-content"><P>Because of the unsuccessful ligation from 18.23 new vector (pET24d) was digested with <I>Nco</I>I and <I>Bam</I>HI according to the following scheme: </P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Volume [µl]</TH></TR><TR><TH scope="row">pET24d (148 ng/µL)</TH><TD>15</TD></TR><TR><TH scope="row">Cutsmart 10x</TH><TD>3</TD></TR><TR><TH scope="row"><I>Nco</I>I</TH><TD>0,5</TD></TR><TR><TH scope="row"><I>Bam</I>HI</TH><TD>0,5</TD></TR><TR><TH scope="row">Water</TH><TD>6</TD></TR><TR><TH scope="row">Total Volume</TH><TD>30</TD></TR></TBODY></TABLE><P>Incubation at 37°C for 2h.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.29">18.29 <I>Pst</I>I digestion of PCR fragments from piGEM-018 with Hag-primers</A></LEGEND><DIV class="aim"><P>Aim: check if the correct inserts are used for ligation </P></DIV><DIV class="exp-content"><P>The DARPin-Hag  constructs contain a <I>Pst</I>I restriction site which could be used to check if the  DARPin gene was inserted into the Hag-construct correctly again. For that  purpose PCR products based on piGEM-018 from clone 7 and 12 (right and wrong  orientation) as a template were digested with <I>Pst</I>I. </P><P>PCR  products from clone 7: 658 bp and 781 bp fragment.</P><P> PCR products from clone 12: 1000bp and 429 bp  fragments.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Volume [µl]</TH></TR><TR><TH scope="row">PCR fragment (350 ng/µL)</TH><TD>3</TD></TR><TR><TH scope="row">Cutsmart 10x</TH><TD>1</TD></TR><TR><TH scope="row"><I>Pst</I>I</TH><TD>0,5</TD></TR><TR><TH scope="row">Water</TH><TD>5,5</TD></TR><TR><TH scope="row">Total Volume</TH><TD>10</TD></TR></TBODY></TABLE><P>Incubation at 37°C for 2h.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.30">18.30 PCR amplification of DARPin-Hag and Hag fragment</A></LEGEND><DIV class="aim"><P>Aim: amplification of DARPin-Hag fragments for repeating the ligation </P></DIV><DIV class="exp-content"><P>piGEM-018 &amp; -005 were used as template for a PCR with the primers  54flo and 56flo. The products could be used for further cloning after <I>Nco</I>I/<I>Bam</I>HI digest.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix</TH><TH scope="col">Volume</TH></TR><TR><TH scope="row">Template (piGEM-005 &amp; -018)</TH><TD>1</TD></TR><TR><TH scope="row">Primer Flo 54</TH><TD>1</TD></TR><TR><TH scope="row">Primer Flo 56</TH><TD>1</TD></TR><TR><TH scope="row">Phusion</TH><TD>1</TD></TR><TR><TH scope="row">Phusion Buffer 5x</TH><TD>10</TD></TR><TR><TH scope="row">dNTP Mix</TH><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>35</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>1min 45sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>2 min</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>30x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The fragments were analysed on an 1% agarose gel.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.54">15.54 new PCR amplification of cup1-1</A></LEGEND><DIV class="aim"><P>Aim: amplification of cup1-1 domain for Gibson assembly with pMAD</P></DIV><DIV class="exp-content"><P>The cup1-1 domain was amplified via PCR with new primers iGEM-024 and -025. The fragment should be ca. 160 bp.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix</TH><TH scope="col">Volume</TH></TR><TR><TH scope="row">Template (cup1-1 gene)</TH><TD>1</TD></TR><TR><TH scope="row">Primer iGEM-024</TH><TD>1</TD></TR><TR><TH scope="row">Primer iGEM-025</TH><TD>1</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>1</TD></TR><TR><TH scope="row">Phusion Buffer 5x</TH><TD>10</TD></TR><TR><TH scope="row">dNTP Mix</TH><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>35</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>30 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>30x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>According to the gel analysis the amplification was successful.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="15.06.2014">15.06.2014</A></H2><FIELDSET class="ex20"><LEGEND><A name="exp20.3a">20.3 Overnight culture of positive clones</A></LEGEND><DIV class="aim"><P>Aim: transformation of <I>Bacillus subtilis</I> WT3610 with plasmid</P></DIV><DIV class="exp-content"><P>Colonies grew on the transformand plates (piGEM-005 &amp; - 018). The blue/  white screening showed positive transformed blue clones.  Three clones per plate were picked and used for inoculation of LB-MLS (4 mL LB, 4 µL lincomycin, 4 µL  erythromycin). Incubation was carried out over night at 30°C with the six cultures.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="16.06.2014">16.06.2014</A></H2><FIELDSET class="exp14"><LEGEND><A name="exp14.43">14.43 Overnight culture</A></LEGEND><DIV class="aim"><P>Aim: Inoculate Overnight culture of PheA-Arc1p-C-0x-pET28a</P></DIV><DIV class="exp-content"><P>5 mL of LB-Kan<SUP>50</SUP> medium were inoculated with a clone from 14.42 bearing the PheA-Arc1p-C-0x-pET28a plasmid.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="17.06.2014">17.06.2014</A></H2><FIELDSET class="exp20"><LEGEND><A name="exp20.3b">20.3 First temperature shift</A></LEGEND><DIV class="aim"><P>Aim: integration of  pMAD-Insert into Bacillus chromosome via flanks</P></DIV><DIV class="exp-content"><P>The 6  overnight cultures were used to inoculate 10 mL LB MLS, which were incubated until the culture obtained an OD of 0,1. The  cultures were incubated at 30°C for 2h.</P><P>Subsequently, the temperature was shifted to 42°C for 6h.</P><P>After  the heat shock dilutions from 10<SUP>-4</SUP> to 10<SUP>-6</SUP> of each culture  were plated out on MLS-X-Gal so that 18 plates could be incubated overnight at  42°C.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.54">15.54 Gibson-assembly with  PCR amplified <I>cup</I>1-1 and <I>Spe</I>I digested piGEM-005</A></LEGEND><DIV class="aim"><P>Aim: insertion of <I>cup</I>1-1 into piGEM-005</P></DIV><DIV class="exp-content"><P>The <I>cup</I>1-1 domain was amplified via PCR with new primers iGEM-024 and  -025. Vector and template were diluted to a concentration under 75 ng/µL. Spe1  cut piGEM-005 was used from the cloning experiments with DARPin.</P><P>PCR fragment <I>cup</I>1-1 (464 ng/µL) dilution 1:10</P><P><I>Spe</I>I digested pIGEM-005 (148 ng/µL) dilution 1:2</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Volume [µl]</TH></TR><TR><TH scope="row">Gibson-Mix</TH><TD>15</TD></TR><TR><TH scope="row">Insert (<I>cup</I>1-1 58 ng/µL)</TH><TD>1,3</TD></TR><TR><TH scope="row">piGEM-005 </TH><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>2,7</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD></TR></TBODY></TABLE><P>The mix was incubated 5 min at room temperature and then for 1h at 50°C.</P><P>In the end the whole mix was transformed into E.Coli XL1-Blue and plated  out on LB-Amp. Incubation was done overnight at 37°C.</P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.44">14.44 Test Expression</A></LEGEND><DIV class="aim"><P>Aim: Test the expression of PheA-Arc1p-C-0x</P></DIV><DIV class="exp-content"><P>Using the preculture from 14.43 60 mL of LB-Kan<SUP>50</SUP> medium were inoculated with 600 µL of the preculture and incubated at 37 °C and 220 rpm until OD<SUB>600</SUB> reached 0.5. The culture was cooled down to 18 °C and the protein production was induced with 12 µL of 500 mM IPTG. Over night the culture was incubated at 18 °C and 220 rpm. Afterwards the cell culture was centrifuged (17000 rpm, 20 min, 4 °C), the pellet resuspended in 3 mL buffer A and stored at -20 °C. A glycerolstock of the strain containing the PheA-Arc1p-C-0x-pET28a plasmid was stored at -80 °C.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="18.06.2014">18.06.2014</A></H2><FIELDSET class="exp20"><LEGEND><A name="exp20.4">20.4 Second temperature shift</A></LEGEND><DIV class="aim"><P>Aim: flip out of the pMAD backbone</P></DIV><DIV class="exp-content"><P>One  blue colony per diluted clone was used to inoculate 4 mL LB. The 6 cultures  were incubated at 30°C for 6h and afterwards for 3h at 42°C.</P><P>Dilutions  from 10<SUP>-4</SUP> to 10<SUP>-4</SUP> were plated out on 18 X-Gal plates  without MLS selection. The positive clones should not contain the resistance  inside the backbone as well as the galactosidase, resulting in normal colored colonies. The plates were incubated at  42°C overnight.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.54">15.54 Checking Gibson assembly plate piGEM-005-cup1-1</A></LEGEND><DIV class="aim"><P>Aim: insertion of cup1-1 into piGEM-005</P></DIV><DIV class="exp-content"><P>Unfortunately the plate was empty: no colonies. The vector was digested again in order to repeat the assembly.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.55">15.55 restriction digest of piGEM-005 with <I>Spe</I>I</A></LEGEND><DIV class="aim"><P>Aim: linearizing plasmid for repeating the Gibson assembly</P></DIV><DIV class="exp-content"><P>In order to repeat the Gibson assembly the piGEM-005 was cut with <I>Spe</I>I again.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Volume [µl]</TH></TR><TR><TH scope="row">piGEM-005 (108 ng/µL)</TH><TD>10</TD></TR><TR><TH scope="row">Cutsmart 10x</TH><TD>2</TD></TR><TR><TH scope="row"><I>Spe</I>I</TH><TD>0,5</TD></TR><TR><TH scope="row">Water</TH><TD>7,5</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD></TR></TBODY></TABLE><P>Incubation 1h  at 37°C with following heat shock at 80°C</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="19.06.2014">19.06.2014</A></H2><FIELDSET class="exp20"><LEGEND><A name="exp20.5">20.5 Selection of positive clones</A></LEGEND><DIV class="aim"><P>Aim: check the correct excision of the pMAD backbone</P></DIV><DIV class="exp-content"><P>One white clone was picked from the dilution plates and transferred on a X-Gal Plate as well as on a MLS plate so that 18 clones were tested for the  right integration of the insert although flipping out the pMAD backbone.</P><P>The  plates were incubated at 42°C over night.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.56">15.56 new Gibson-assembly with  cup1-1 and <I>Spe</I>I digested piGEM-005</A></LEGEND><DIV class="aim"><P>Aim: insertion of cup1-1 into piGEM-005</P></DIV><DIV class="exp-content"><P>As insert cup1-1 PCR product from 15.55 and <I>Spe</I>I digested piGEM-005 from  15.55 were used for a new Gibson assembly.</P><P>PCR fragment cup1-1 (464 ng/µL)</P><P><I>Spe</I>I digested pIGEM-005 (53 ng/µL) </P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Volume [µl]</TH></TR><TR><TH scope="row">Gibson-Mix</TH><TD>15</TD></TR><TR><TH scope="row">Insert (cup1-1 58 ng/µL)</TH><TD>2,5</TD></TR><TR><TH scope="row">piGEM-005 </TH><TD>2,5</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD></TR></TBODY></TABLE><P>The mix was incubated 5 min at room temperature and then for 1h at 50°C.</P><P>In the end the whole mix was used to transform  <I>E. coli</I> XL1-Blue and plated  out on LB-Amp. Incubation was done overnight at 37°C.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="20.06.2014">20.06.2014</A></H2><FIELDSET class="exp20"><LEGEND><A name="exp20.5">20.5 selection of positive clones</A></LEGEND><DIV class="aim"><P>Aim: checking the correct flip out of the pMAD backbone</P></DIV><DIV class="exp-content"><P>The white colonies  grew on the X-Gal Master plates but not on MLS plates so the transformation  seemed to be successful. The Backbone with the MLS resistance flipped out of  the genome.</P></DIV></FIELDSET><FIELDSET class="exp20"><LEGEND><A name="exp20.6a">20.6a cPCR with checked clones</A></LEGEND><DIV class="aim"><P>Aim: checking integration of constructs Hag-Spe and Hag-Spe-DARPin into B. s. genome</P></DIV><DIV class="exp-content"><P>No  clone was growing on MLS plates so the picked clones seemed to be positive. In  order to check the correct integration of the domain constructs, a cPCR with the picked clones on the X-Gal plates was done. The 18 picked <I>B.subtilis</I> clones were  cooked in 10 µL PBS for 5 min at 95°C.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix</TH><TH scope="col">Master-Mix for 18 attempts (µL)</TH><TH scope="col">Mix for PCR attempt (µL)</TH></TR><TR><TH scope="row">Colony (picked clones 1-9 from Flagellin/-DARPin B.s.  plates)</TH><TD>-</TD><TD>Part of a cooked colony</TD></TR><TR><TH scope="row">Primer Flo54 Hag-fw</TH><TD>9</TD><TD>-</TD></TR><TR><TH scope="row">Primer Flo56 Hag-rv</TH><TD>9</TD><TD>-</TD></TR><TR><TH scope="row">Phusion</TH><TD>18</TD><TD>-</TD></TR><TR><TH scope="row">Phusion Buffer 5x</TH><TD>72</TD><TD>-</TD></TR><TR><TH scope="row">dNTP Mix</TH><TD>9</TD><TD>-</TD></TR><TR><TH scope="row">Water</TH><TD>237</TD><TD>-</TD></TR><TR><TH scope="row">DMSO</TH><TD>6</TD><TD>-</TD></TR><TR><TH scope="row">Mastermix</TH><TD>-</TD><TD>20</TD></TR><TR><TH scope="row">Total Volume</TH><TD>100</TD><TD>360</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>1 min 45 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>2 min</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>30x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The gel showed no bands. The PCR had to be repeated.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.57">15.57 Miniprep of clones from Gibson assembly from 15.56</A></LEGEND><DIV class="aim"><P>Aim: increase the amount of plasmid for further experiments</P></DIV><DIV class="exp-content"><P>4 x 5 mL of LB-Amp was were  inoculated with a picked colony from a transformation plate with XL1-Blue  containing the Gibson-assembly product from 15.56.</P><P>Incubation at 37°C overnight.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="21.06.2014">21.06.2014</A></H2><FIELDSET class="exp15"><LEGEND><A name="exp15.58">15.58 cPCR with colonies from Gibson-assembly of piGEM-005 + cup1-1 plate</A></LEGEND><DIV class="aim"><P>Aim: checking correct insertion of cup1-1</P></DIV><DIV class="exp-content"><P>In order to check the correct  insertion of the cup1-1 domain into the Hag-Flank-construct in pMAD a cPCR with  primers iGEM-025 (Cup1-1 rv) and Flo89 (pMAD-flank1-<I>Bam</I>HI fw) was performed. The picked colonies were the same which were used to inoculate minipreps from 15.57. The fragment was supposed to have approx. 1250  bp. </P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix</TH><TH scope="col"><STRONG>Master-Mix for 5 attempts (µL)</STRONG></TH><TH scope="col"><STRONG>Mix for PCR attempt (µL)</STRONG></TH></TR><TR><TH scope="row">Colony (picked clones 1-4 for miniprep)</TH><TD>-</TD><TD>Part of a  colony</TD></TR><TR><TH scope="row">Primer Flo89 </TH><TD>2,5</TD><TD>-</TD></TR><TR><TH scope="row">Primer piGEM-025</TH><TD>2,5</TD><TD>-</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>2,5</TD><TD>-</TD></TR><TR><TH scope="row">Phusion Buffer 5x</TH><TD>20</TD><TD>-</TD></TR><TR><TH scope="row">dNTP Mix</TH><TD>2,5</TD><TD>-</TD></TR><TR><TH scope="row">Water</TH><TD>70</TD><TD>-</TD></TR><TR><TH scope="row">Mastermix</TH><TD>-</TD><TD>20</TD></TR><TR><TH scope="row">Total Volume</TH><TD>100</TD><TD>20</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1 min 30 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>30x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The analysis via 1% agarose gel eletrophoresis showed now bands. The plasmids of all four clones were isolated to use them as template.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.59">15.59 PCR with isolated plasmid from Gibson piGEM-005 + cup1-1 plate</A></LEGEND><DIV class="aim"><P>Aim: checking correct insertion of cup1-1</P></DIV><DIV class="exp-content"><P>This  time the isolated plasmid DNA from the picked six clones was used as a template  with different primer pairs. First of all the PCR reactions were performed with  primers Flo89 and Flo90 in order to receive the whole Hag-Flank-Cup1-1  construct with ca. 2100 bp length. As negative control also piGEM-005 was used  as template producing a 2000kb fragment. Because of the small difference, all six reactions were also performed with the primers Flo89 and iGEM-025 (cup1-1 rv) in  order to check the insertion of cup1-1 directly with a fragment of 1268 bp.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix</TH><TH scope="col"><STRONG>Master-Mix 1 for   5 reactions (µL)</STRONG></TH><TH scope="col"><STRONG>Master-Mix 2 for   5 reactions (µL)</STRONG></TH><TH scope="col"><STRONG>Mix 1 for PCR reactions (µL)</STRONG></TH><TH scope="col"><STRONG>Mix 2for PCR reactions (µL)</STRONG></TH></TR><TR><TH scope="row">Plasmid DNA from clone 1-4/ piGEM-005</TH><TD>-</TD><TD>-</TD><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer Flo89 </TH><TD>2,5</TD><TD>2,5</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer Flo90</TH><TD>2,5</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer piGEM-025</TH><TD>-</TD><TD>2,5</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>2,5</TD><TD>2,5</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Phusion Buffer 5x</TH><TD>20</TD><TD>20</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">dNTP-mix</TH><TD>2,5</TD><TD>2,5</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Water</TH><TD>70</TD><TD>70</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Master-Mix 1/ 2</TH><TD>-</TD><TD>-</TD><TD>20</TD><TD>20</TD></TR><TR><TH scope="row">Total Volume</TH><TD>100</TD><TD>100</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>20 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>30x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The gel analysis shows that all 4 clones seem to be positive. An amplification of the cup1-1 fragment with the plasmid DNA as template could proof that.</P></DIV></FIELDSET><FIELDSET class="exp20"><LEGEND><A name="exp20.6a">20.6b Repeated cPCR with checked clones</A></LEGEND><DIV class="aim"><P>Aim: checking integration of constructs Hag-<I>Spe</I>I and Hag-<I>Spe</I>I-DARPin into B. s. genome</P></DIV><DIV class="exp-content"><P>Because  of the unsuccessful cPCR before, the PCR was repeated. The 18 picked <I>B.subtilis</I> clones  were cooked in 30 µL PBS for 10 min at 95°C.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix</TH><TH scope="col">Master-Mix for 18 attempts (µL)</TH><TH scope="col">Mix for PCR attempt (µL)</TH></TR><TR><TH scope="row">Colony (picked clones 1-9 from Flagellin/-DARPin B.s.  plates)</TH><TD>-</TD><TD>Part of a cooked colony</TD></TR><TR><TH scope="row">Primer Flo54 Hag-fw</TH><TD>9</TD><TD>-</TD></TR><TR><TH scope="row">Primer Flo56 Hag-rv</TH><TD>9</TD><TD>-</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>18</TD><TD>-</TD></TR><TR><TH scope="row">Phusion Buffer 5x</TH><TD>72</TD><TD>-</TD></TR><TR><TH scope="row">dNTP Mix</TH><TD>9</TD><TD>-</TD></TR><TR><TH scope="row">Water</TH><TD>237</TD><TD>-</TD></TR><TR><TH scope="row">DMSO</TH><TD>6</TD><TD>-</TD></TR><TR><TH scope="row">Mastermix</TH><TD>-</TD><TD>20</TD></TR><TR><TH scope="row">Total Volume</TH><TD>100</TD><TD>360</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>1 min 45 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>2 min</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>30x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>All nine clones with piGEM-005 seem to be positive. Clone 1, 3, 7, 8 and  9 of transformed piGEM-018.1 might be positive as well as clone 2-6 from  piGEM-018.2. The sequencing of the PCR products should tell us if the insertion  was successful.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="22.06.2014">22.06.2014</A></H2><FIELDSET class="exp18"><LEGEND><A name="exp18.31">18.31 PCR amplification of DARPin-Hag and Hag fragment</A></LEGEND><DIV class="aim"><P>Aim: amplification of DARPin-Hag fragments for repeating the ligation </P></DIV><DIV class="exp-content"><P>piGEM-018  &amp; -005 were used as template for a PCR with the primers Flo 54 and Flo 56. The products could be used for further cloning after  <I>Nco</I>I/<I>Bam</I>HI digest. </P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Volume [µl]</TH></TR><TR><TH scope="row">Template (piGEM-005 &amp; -018)</TH><TD>1</TD></TR><TR><TH scope="row">Primer Flo 54 </TH><TD>1</TD></TR><TR><TH scope="row">Primer Flo 56</TH><TD>1</TD></TR><TR><TH scope="row">Phusion</TH><TD>1</TD></TR><TR><TH scope="row">Phusion Buffer 5x</TH><TD>10</TD></TR><TR><TH scope="row">dNTP Mix</TH><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>35</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1 min 45 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>30x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.32">18.32 cPCR with new clones</A></LEGEND><DIV class="aim"><P>Aim: checking picked clones for correct Hag/ Hag-DARPin insert in pET24d</P></DIV><DIV class="exp-content"><P>More clones from each  plate were picked to analyse them via cPCR with the primers flo54 and flo56  (Hag-fw and Hag-rv). The result should be a ca. 1kb fragment for correct  pET24d-Hag containing clones and a 1,5 kb fragment for pET24d-Hag-DARPin  containing clones.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Volume (µL)</TH><TH scope="col">MM for 14 attempts (µL</TH></TR><TR><TH scope="row"><P>Colony</P> (8-15 from pet-Hag,7-14 from pet-Hag-DARPin)</TH><TD>1 colony</TD><TD>-</TD></TR><TR><TH scope="row">Primer Flo 54 </TH><TD>-</TD><TD>7</TD></TR><TR><TH scope="row">Primer Flo 56</TH><TD>-</TD><TD>7</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>-</TD><TD>7</TD></TR><TR><TH scope="row">Phusion Buffer 5x</TH><TD>-</TD><TD>56</TD></TR><TR><TH scope="row">dNTP Mix</TH><TD>-</TD><TD>7</TD></TR><TR><TH scope="row">Water</TH><TD>-</TD><TD>196</TD></TR><TR><TH scope="row">Mastermix</TH><TD>20</TD><TD>-</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>280</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>10 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1 min 45 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>35x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>Clone 1 from each pET24d-Hag-<I>Spe</I>I and pET24d-Hag-DARPin clones was positive and were digested with <I>Nco</I>I/<I>Bam</I>I to check them again.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.33">18.33 pET24d digest with <I>Nco</I>I/<I>Bam</I>HI</A></LEGEND><DIV class="aim"><P>Aim: preparing vector for new ligation attempts</P></DIV><DIV class="exp-content"><P>Because  of the low amount of right clones, new vector (pET24d) was digested with <I>Nco</I>I and <I>Bam</I>HI according to the following scheme:</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col"><STRONG><U>Volume (µL)</U></STRONG></TH></TR><TR><TH scope="row">pET24d (50 ng/µL)</TH><TD>20</TD></TR><TR><TH scope="row">CutSmart 10x</TH><TD>3</TD></TR><TR><TH scope="row"><I>Nco</I>I</TH><TD>0,5</TD></TR><TR><TH scope="row"><I>Bam</I>HI</TH><TD>0,5</TD></TR><TR><TH scope="row">Water</TH><TD>6</TD></TR><TR><TH scope="row">Total Volume</TH><TD>30</TD></TR></TBODY></TABLE><P>Incubation at 37°C for 2 hours with heat shock in the end at 80°C</P><P>Endconcentration:  32 ng/µL</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.34">18.34 Restriction digest of Hag PCR fragments from 18.31 with <I>Nco</I>I/<I>Bam</I>HI</A></LEGEND><DIV class="aim"><P>Aim: creating restriction sites for cloning into pet24d cut with <I>Nco</I>I/<I>Bam</I>HI</P></DIV><DIV class="exp-content"><P>The PCR products from 18.31 were digested with <I>Nco</I>I and<I>Bam</I>HI to create restriction sites in order to ligate the fragments into <I>Nco</I>I/<I>Bam</I>HI digested pet24d in case of negative clones refering to the cPCR from 18.x</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">PCR Hag-fragment (µL)</TH><TH scope="col">PCR Hag-DARPin fragment  (µL)</TH></TR><TR><TH scope="row">PCR products</TH><TD>45</TD><TD>45</TD></TR><TR><TH scope="row">Cutsmart 10x</TH><TD>5,2</TD><TD>5,2</TD></TR><TR><TH scope="row"><I>Nco</I>I</TH><TD>0,5</TD><TD>0,5</TD></TR><TR><TH scope="row"><I>Bam</I>HI</TH><TD>0,5</TD><TD>0,5</TD></TR><TR><TH scope="row">Water</TH><TD>3,8</TD><TD>3,8</TD></TR><TR><TH scope="row">Total Volume</TH><TD>52</TD><TD>52</TD></TR></TBODY></TABLE><P>Incubation  for 2h at 37°C with heat shock at 80°C.</P><P>Endconcentration: 
	  digested Hag-fragment  260 ng/µL	
	  digested  Hag-fragment 268 ng/µL</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.60">15.60 PCR pMAD-Hag-Flank-cup1-1 (piGEM-016)</A></LEGEND><DIV class="aim"><P>Aim: checking success of Gibson assembly with pMAD-Hag-Flank and cup1-1</P></DIV><DIV class="exp-content"><P>Using  the plasmids as template the cup1-1 fragment at 160 bp should be visible on the  agarose gel. As positive control PCR products from the cup1-1 amplification  used for the Gibson Assembly into piGEM-005 were applied to the gel as well. </P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Volume (µL)</TH><TH scope="col">MM for 5 attempts (µL)</TH></TR><TR><TH scope="row">Template (Plasmid from clone 1-4)</TH><TD>0,5</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-024</TH><TD>-</TD><TD>2,5</TD></TR><TR><TH scope="row">Primer iGEM-025</TH><TD>-</TD><TD>2,5</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>-</TD><TD>2,5</TD></TR><TR><TH scope="row">Phusion Buffer 5x</TH><TD>-</TD><TD>20</TD></TR><TR><TH scope="row">dNTP Mix</TH><TD>-</TD><TD>2,5</TD></TR><TR><TH scope="row">Water</TH><TD>-</TD><TD>70</TD></TR><TR><TH scope="row">Mastermix</TH><TD>19,5</TD><TD>-</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>100</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>20 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>35x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The bands under the 200 bp mark indicate the positive insertion of  cup1-1 into piGEM-005. The plasmid has to be sequenced and is named iGEM-016.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.35">18.35 Test digest with <I>Nco</I>I/<I>Bam</I>HI - Checking insertion of Hag/-DARPin into pet24d</A></LEGEND><DIV class="aim"><P>Aim: proving insertion of Hag/-DARPin fragment into pET24d</P></DIV><DIV class="exp-content"><P>The  plasmids are digested with <I>Nco</I>I and <I>Bam</I>HI. In case of a correct insertion the  pET24d backbone and the insert should be seen on the gel at 5300 bp and 1000/  1500 bp. As control pET24d-Hag-Flank (piGEM-001) is used to check the  mastermix.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Volume (µl)</TH><TH scope="col">Mastermix</TH></TR><TR><TH scope="row">2x Plasmid from clone 6</TH><TD>5</TD><TD>-</TD></TR><TR><TH scope="row">CutSmart 10x</TH><TD>-</TD><TD>5</TD></TR><TR><TH scope="row"><I>Nco</I>I</TH><TD>-</TD><TD>0,5</TD></TR><TR><TH scope="row"><I>Bam</I>HI</TH><TD>-</TD><TD>0,5</TD></TR><TR><TH scope="row">Water</TH><TD>-</TD><TD>19</TD></TR><TR><TH scope="row"><STRONG>Total</STRONG> Volume</TH><TD>10</TD><TD>25</TD></TR></TBODY></TABLE><P>The used marker was the GeneRuler 1 kb DNA ladder. The lower fragment of the digested pET24d-Hag was too big (2 kb) and pET24d-Hag-DARPin seemed only to be linearized with the exception of two very thin bands below the linear plasmid, which were all of the wrong size.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.36">18.36 ligation of pET24d  and PCR Hag/ Hag-DARPin fragments cut <I>Nco</I>I/<I>Bam</I>HI</A></LEGEND><DIV class="aim"><P>Aim: ligation of construct with pET24d for overproduction of flagellin and crystallization</P></DIV><DIV class="exp-content"><P>The  <I>Nco</I>I/<I>Bam</I>HI digested flagellin PCR products from 18.22 (from clone 7 and piGEM-005) were  ligated with <I>Nco</I>I/<I>Bam</I>HI digested pET24d.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">pET24d-Hag reaction (µL) </TH><TH scope="col">pET24d-Hag-DARPin raction (µL)</TH></TR><TR><TH scope="row">Insert (PCR fragments) (ca. 260 ng/µL)</TH><TD>2</TD><TD>2</TD></TR><TR><TH scope="row">Vector (pET24d, c= 50ng/µL)</TH><TD>5</TD><TD>5</TD></TR><TR><TH scope="row">Ligase-Buffer 10x</TH><TD>2</TD><TD>2</TD></TR><TR><TH scope="row">T4 Ligase</TH><TD>1</TD><TD>10</TD></TR><TR><TH scope="row">Millipore Water</TH><TD>10</TD><TD>10</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><P>The reactions were prepared in double and were incubated at room  temperature. One reaction was incubated for 1 hour followed  by transformation of <EM>E. coli</EM> XL1-Blue and  the second one over night.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.37a">18.37 Repetition of PCR with 3 clones of pET24d-Hag and pET24d-Hag-DARPin ligation</A></LEGEND><DIV clacc="aim"><P>Aim: checking picked clones for correct Hag/ Hag-DARPin insert in pET24d</P></DIV><DIV class="exp-content"><P>3  clones from ligation plates with pET-Hag and pET-Hag-DARPin were resuspended in  30 µL PBS and cooked at 95 °C for 10 min. After that procedure 1 µL of this  suspension was used as PCR template.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Volume (µL)</TH><TH scope="col">MM for 7 reactions (µL)</TH></TR><TR><TH scope="row">Template ( 1µL cooked suspension) (8-10 from pET-Hag, 7-9 from pET-Hag-DARPin)</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer Flo 54 </TH><TD>-</TD><TD>3,5</TD></TR><TR><TH scope="row">Primer Flo 56</TH><TD>-</TD><TD>3,5</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>-</TD><TD>3,5</TD></TR><TR><TH scope="row">Phusion Buffer 5x</TH><TD>-</TD><TD>28</TD></TR><TR><TH scope="row">dNTP-mix</TH><TD>-</TD><TD>3,5</TD></TR><TR><TH scope="row">Water</TH><TD>-</TD><TD>98</TD></TR><TR><TH scope="row">Mastermix</TH><TD>19</TD><TD>-</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>140</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1 min 45 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>35x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>Run over night</P></DIV></FIELDSET><FIELDSET class="exp"><LEGEND><A name="exp_1">Transformation</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Plasmid</TH><TH scope="col">Concentration (ng/µL)</TH><TH scope="col">Antibiotic</TH><TH scope="col">Cells</TH><TH scope="col">Volume (µL)</TH></TR><TR><TH scope="row">pET24d</TH><TD>148</TD><TD>Can</TD><TD>DH5α</TD><TD>1</TD></TR><TR><TH scope="row">piGEM-018</TH><TD>160</TD><TD>Amp</TD><TD>DH5α</TD><TD>0,5</TD></TR><TR><TH scope="row">piGEM-014</TH><TD>350</TD><TD>Amp</TD><TD>DH5α</TD><TD>0,5</TD></TR><TR><TH scope="row">Gibson pMAD-cup1-1 clone 4</TH><TD>60</TD><TD>Amp</TD><TD>DH5α</TD><TD>1</TD></TR><TR><TH scope="row">pET-Hag attempt from 18.36</TH><TD>-</TD><TD>Can</TD><TD>XL1-Blue</TD><TD>20</TD></TR><TR><TH scope="row">pET-Hag-DARPin attempt from 18.36</TH><TD>-</TD><TD>Can</TD><TD>XL1-Blue</TD><TD>20</TD></TR><TR><TH scope="row">Control pET24d cut <I>Nco</I>I/<I>Bam</I>HI</TH><TD>32</TD><TD>Can</TD><TD>XL1-Blue</TD><TD>1</TD></TR></TBODY></TABLE></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="23.06.2014">23.06.2014</A></H2><FIELDSET class="exp14"><LEGEND><A name="exp14.45">14.45 Purification of test expression</A></LEGEND><DIV class="aim"><P>Aim: Purify PheA-Arc1p-C-0x on a small scale</P></DIV><DIV class="exp-content"><P>Lysozyme was added to the cell suspension from 14.44 to a final concentration of 1 mg/mL and incubated on ice for 30 min. Afterwards the cells were lysed using ultra sound (2x 18 pulses, 30 s on ice inbetween). To clear the suspension of cell debris it was centrifuged (13000 rpm, 15 min, 4 °C). The supernatant was purified using QIAGEN Ni-NTA Spin Columns. An SDS-PAGE gel showed that the protein is produced and can be purified using Ni affinity chromatography.</P><P>60 mL LB-Kan<SUP>50</SUP> medium was inoculated from the glycerolstock prepared in 14.44 and incubated at 37 °C and 220 rpm over night.</P></DIV></FIELDSET><FIELDSET class="exp"><LEGEND><A name="exp_2">Sequencing of constructs from 23.06.2014</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Premix</TH><TH scope="col">Label-Nr.</TH><TH scope="col">Construct</TH><TH scope="col">Primer</TH></TR><TR><TH scope="row">1</TH><TD>AGB0023<STRONG>402</STRONG></TD><TD>piGEM-018  (pMAD-DARPin)</TD><TD>Flo54 (Hag-F-Nco)</TD></TR><TR><TH scope="row">2</TH><TD>AGB0023<STRONG>403</STRONG></TD><TD>piGEM-018  (pMAD-DARPin)</TD><TD>Flo56 (Hag-R-Bam)</TD></TR><TR><TH scope="row">3</TH><TD>AGB0023<STRONG>404</STRONG></TD><TD>pMAD-Hag-Flank-cup1-1</TD><TD>Flo54 (Hag-F-Nco)</TD></TR><TR><TH scope="row">4</TH><TD>AGB0023<STRONG>405</STRONG></TD><TD>pMAD-Hag-Flank-cup1-1</TD><TD>Flo56 (Hag-R-Bam)</TD></TR><TR><TH scope="row">5</TH><TD>AGB0023<STRONG>406</STRONG></TD><TD>piGEM-002 (Nose  plasmid)</TD><TD>TW259</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.37b">18.37 Repetition of PCR with 3 clones of pET24d-Hag and pET24d-Hag-DARPin ligation</A></LEGEND><DIV class="aim"><P>Aim: checking picked clones for correct Hag/ Hag-DARPin insert in pET24d</P></DIV><DIV class="exp-content"><P>Gel electrophoresis:</P><P>The gel shows negative bands in case of the Hag-DARPin clones. The PCR for Hag 8 and 9 did not work although we cooked the clones in PBS. The transformation efficiency seemed to be very low, so clones from the new ligation attempts should be analysed.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.38">18.38 Ligation checking success of transformed ligation attempts</A></LEGEND><DIV class="exp-content"><P>On every plate grew a high amount of colonies. 6 clones of each plate were picked for inoculation of minipreps and for cPCR after cooking with PBS. The clones were transferred to another LB-Can plate.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.39">18.39 cPCR with picked clones from 18.38 ligation attempt transformation</A></LEGEND><DIV class="aim"><P>Aim: checking picked clones for correct Hag/ Hag-DARPin insert in pET24d</P></DIV><DIV class="exp-content"><P>For  checking the insertion of the Hag/-DARPin into pET24d the clones from the  transformation plate were transferred to another LB-Can plate  and cooked for 10 min in 30 µL PBS at 95°C. At  the same time the colonies were used to inoculate LB-Can for minipreps.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Volume (µL)</TH><TH scope="col">MM for 14 attempts (µL)</TH></TR><TR><TH scope="row">Template ( 1µL cooked suspension) (1-6 from pet-Hag, 1-6 from pet-Hag-DARPin)</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer Flo 54 </TH><TD>-</TD><TD>7</TD></TR><TR><TH scope="row">Primer Flo 56</TH><TD>-</TD><TD>7</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>-</TD><TD>7</TD></TR><TR><TH scope="row">Phusion Buffer 5x</TH><TD>-</TD><TD>198</TD></TR><TR><TH scope="row">dNTP-mix</TH><TD>-</TD><TD>7</TD></TR><TR><TH scope="row">Water</TH><TD>-</TD><TD>98</TD></TR><TR><TH scope="row">Mastermix</TH><TD>19</TD><TD>-</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>280</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1 min 45 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>35x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>Run over night</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="24.06.2014">24.06.2014</A></H2><FIELDSET class="exp14"><LEGEND><A name="exp14.46">14.46 Expression of PheA-Arc1p-C-0x</A></LEGEND><DIV class="aim"><P>Aim: Express PheA-Arc1p-C-0x for further use</P></DIV><DIV class="exp-content"><P>10 x 500 mL LB-Kan<SUP>50</SUP> medium were inoculated with 5 mL of the overnight culture prepared in 14.45 and incubated at 37 °C and 220 rpm until OD<SUB>600</SUB> reached 0.5. The flasks were cooled to 18 °C and protein production was induced by adding 100 µL 500 mM IPTG. The cultures were incubated over night at 18 °C and 220 rpm. Afterwards the cells were collected by centrifugation (6000 rpm, 20 min, 4 °C), resuspended in buffer A and stored at -20 °C until further use.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="25.06.2014">25.06.2014</A></H2><FIELDSET class="exp"><LEGEND><A name="exp_3">Sequencing  results of constructs from 23.06.2014</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Premix</TH><TH scope="col">Label-Nr.</TH><TH scope="col">Construct</TH><TH scope="col">Result</TH></TR><TR><TH scope="row">1</TH><TD>AGB0023<STRONG>402</STRONG></TD><TD>piGEM-018  (pMAD-DARPin)</TD><TD>Correct insertion of  DARPin</TD></TR><TR><TH scope="row">2</TH><TD>AGB0023<STRONG>403</STRONG></TD><TD>piGEM-018  (pMAD-DARPin)</TD><TD>Did not work</TD></TR><TR><TH scope="row">3</TH><TD>AGB0023<STRONG>404</STRONG></TD><TD>pMAD-Hag-Flank-cup1-1</TD><TD>Correct insertion,  point mutation refers to bad sequencing</TD></TR><TR><TH scope="row">4</TH><TD>AGB0023<STRONG>405</STRONG></TD><TD>pMAD-Hag-Flank-cup1-1</TD><TD>Did not work</TD></TR><TR><TH scope="row">5</TH><TD>AGB0023<STRONG>406</STRONG></TD><TD>piGEM-002 (Nose  plasmid)</TD><TD>Correct insertion of  the CM resistance</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.39">18.39 Gel cPCR with picked clones from 18.38 ligation attempt transformation</A></LEGEND><DIV class="aim"><P>Aim: checking picked clones for correct Hag/ Hag-DARPin insert in pET24d</P></DIV><DIV class="exp-content"><P>Gel electrophoresis: no bands except for the controls were seen. The PCR has to be repeated with more colony in the PCR tubes or the prepped plasmids</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.40">18.40 Test digest with <I>Nco</I>I/<I>Bam</I>HI of isolated plasmids from picked colonies</A></LEGEND><DIV class="aim"><P>Aim: checking picked clones for correct Hag/ Hag-DARPin insert in pET24d</P></DIV><DIV class="exp-content"><P>The picked clones from the ligation attempts 18.38 were isolated. In  order to check the insertion of the Hag/ Hag-DARPin domain the plasmids were  digested with <I>Nco</I>I/<I>Bam</I>HI so that the gel should show the insert of 1 or 1,5 bp  length and the pET24d backbone at 5300 bp. Because of an unavailable marker as  a control the PCR amplified inserts which were used for the ligation will  run  with the gel as well. piGEM-001 will  also be cut <I>Nco</I>I/<I>Bam</I>HI in order to receive a marker band for the pET24d backbone.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Volume (µL)</TH><TH scope="col">Mastermix for 8 attempts</TH></TR><TR><TH scope="row">Plasmid (ca. 45 ng/µL</TH><TD>5</TD><TD>-</TD></TR><TR><TH scope="row">Cutsmart 10x</TH><TD>-</TD><TD>4</TD></TR><TR><TH scope="row"><I>Nco</I>I</TH><TD>-</TD><TD>0,5</TD></TR><TR><TH scope="row"><I>Bam</I>HI</TH><TD>-</TD><TD>0,5</TD></TR><TR><TH scope="row">Water</TH><TD>-</TD><TD>35</TD></TR><TR><TH scope="row">Mastermix</TH><TD>5</TD><TD>-</TD></TR><TR><TH scope="row">Total Volume</TH><TD>10</TD><TD>40</TD></TR></TBODY></TABLE><P>The gel shows that the digested clones were all negative. The ligation had to be performed again.</P></DIV></FIELDSET><FIELDSET class="exp20"><LEGEND><A name="exp20.6">20.6 Purification of  cPCR samples with Gel extraction kit</A></LEGEND><DIV class="aim"><P>Aim: purification of  samples for sequencing</P></DIV><DIV class="exp-content"><P>Purified samples:
    clone 3 &amp; 5 from transformed piGEM-005clone 3 &amp; 9 from transformed piGEM-018
    </P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Clone</TH><TH scope="col">Concentration (ng/µl)</TH></TR><TR><TH scope="row">Hag 3</TH><TD>10</TD></TR><TR><TH scope="row">Hag 5</TH><TD>14</TD></TR><TR><TH scope="row">Hag-DARPin 3</TH><TD>9</TD></TR><TR><TH scope="row">Hag-DARPin 9</TH><TD>9</TD></TR></TBODY></TABLE></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="26.06.2014">26.06.2014</A></H2><FIELDSET class="exp20"><LEGEND><A name="exp20.7">20.7 PCR with purified cPCR samples from 20.6</A></LEGEND><DIV class="aim"><P>Aim: PCR amplification of purified cPCR samples</P></DIV><DIV class="exp-content"><P>The concentration of the cPCR samples after purification was too low for sequencing. Therefore the cPCR samples have to be amplified with PCR again.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Volume (µL)</TH><TH scope="col">MM for 6 attempts (µL)</TH></TR><TR><TH scope="row">Purified cPCR product from 20.6 (3 &amp; 5 from pet-Hag, 3 &amp; 9 from  pet-Hag-DARPin)</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer Flo 54 </TH><TD>-</TD><TD>6</TD></TR><TR><TH scope="row">Primer Flo 56</TH><TD>-</TD><TD>6</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>-</TD><TD>6</TD></TR><TR><TH scope="row">Phusion Buffer 5x</TH><TD>-</TD><TD>6</TD></TR><TR><TH scope="row">dNTP-mix</TH><TD>-</TD><TD>60</TD></TR><TR><TH scope="row">Water</TH><TD>-</TD><TD>216</TD></TR><TR><TH scope="row">Mastermix</TH><TD>19</TD><TD>-</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>300</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1 min 45 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>35x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp20"><LEGEND><A name="exp20.8">20.8 Sequencing  of purified cPCR samples from pMAD transformation </A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Premix</TH><TH scope="col">Label-Nr.</TH><TH scope="col">Construct</TH><TH scope="col">Primer</TH></TR><TR><TH scope="row">1</TH><TD>AGB0023<STRONG>407</STRONG></TD><TD>Hag-<I>Spe</I>I in <I>B. subtilis</I>  genome clone 3</TD><TD>Hag-Nco-fw</TD></TR><TR><TH scope="row">2</TH><TD>AGB0023<STRONG>408</STRONG></TD><TD>Hag-<I>Spe</I>I in <I>B. subtilis</I>  genome clone 5</TD><TD>Hag-Nco-fw</TD></TR><TR><TH scope="row">3</TH><TD>AGB0023<STRONG>409</STRONG></TD><TD>Hag-<I>Spe</I>I-DARPin in <I>B. subtilis</I>  genome clone3</TD><TD>Hag-Nco-fw</TD></TR><TR><TH scope="row">4</TH><TD>AGB0023<STRONG>410</STRONG></TD><TD>Hag-<I>Spe</I>I-DARPin in <I>B. subtilis</I>  genome clone 9</TD><TD>Hag-Nco-fw</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.41">18.41 New ligation of pET24d  and PCR Hag/ Hag-DARPin fragments cut <I>Nco</I>I/<I>Bam</I>HI transformation</A></LEGEND><DIV class="aim"><P>Aim: ligation of construct with pET24d for overproduction of flagellin and crystallization</P></DIV><DIV class="exp-content"><P>The cut  pET24d <I>Nco</I>I/<I>Bam</I>HI which was used for the previous ligations was not purified and  contained an 1 kb insert. For a new clean ligation the reaction was repeated  with digested pET24d. The <I>Nco</I>I/<I>Bam</I>HI digested flagellin PCR products (from  clone 7 and piGEM-005) were ligated with <I>Nco</I>I/<I>Bam</I>HI digested pET24d.</P><P>The  reactions were carried out in double and were incubated 1h  at room temperature. Finally they were used to transform  <I>E. coli</I> XL1-Blue after  inactivation at 65°C for 10 min. The attempts 1.1 and 2.1 were incubated  over night at room temperature. 1 µL of digested vector was used for a transformation as a negative  control in order to check the amount of religants.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">pET-Hag attempt (µL) – 1 &amp; 1.1</TH><TH scope="col">pET-Hag-DARPin attempt (µL) 2&amp; 2.1</TH></TR><TR><TH scope="row">Insert (PCR fragments) (ca. 260 ng/µL)</TH><TD>3</TD><TD>3</TD></TR><TR><TH scope="row">Vector (pET24d, c= 86ng/µL)</TH><TD>5</TD><TD>5</TD></TR><TR><TH scope="row">Ligase-Buffer 10x</TH><TD>2</TD><TD>2</TD></TR><TR><TH scope="row">T4 Ligase</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Millipore water</TH><TD>9</TD><TD>9</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>20</TD></TR></TBODY></TABLE></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="27.06.2014">27.06.2014</A></H2><FIELDSET class="exp18"><LEGEND><A name="exp18.42">18.42 Miniprep of ligation clones</A></LEGEND><DIV class="exp-content"><P>The  ligation between pET24d and the Hag-/Hag-DARPin fragments which were both cut  <I>Nco</I>I/<I>Bam</I>HI was successful. The ligation plates showed over 20 clones each. However, on the negative control  grew a few clones  as well, which were very small and sticking together. In order to analyse the insert of the clones, 5 clones per  ligation plate 1 and 2 were used to inoculate minipreps in 5 mL LB-Can. For the  Ligation 1 pET-Hag clones 1-5 were picked, as well as clones 6-10 from Plate  Ligation 2 pET-Hag-DARPin. </P><P>The  cultures were inoculated over night at 37°C.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="28.06.2014">28.06.2014</A></H2><FIELDSET class="exp18"><LEGEND><A name="exp18.42">18.42 Miniprep of ligation clones</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Construct</TH><TH scope="col">Clone</TH><TH scope="col">Concentration (ng/µL)</TH></TR><TR><TH scope="row">pET24d-Hag</TH><TD>1</TD><TD>50</TD></TR><TR><TH scope="row">pET24d-Hag</TH><TD>2</TD><TD>87</TD></TR><TR><TH scope="row">pET24d-Hag</TH><TD>3</TD><TD>65</TD></TR><TR><TH scope="row">pET24d-Hag-DARPin</TH><TD>6</TD><TD>83</TD></TR><TR><TH scope="row">pET24d-Hag-DARPin</TH><TD>7</TD><TD>59</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.43">18.43 cPCR of clones from 18.41</A></LEGEND><DIV class="aim"><P>Aim: checking picked clones for correct Hag/ Hag-DARPin insert in pET24d</P></DIV><DIV class="exp-content"><P>Plasmid  from isolated clones from each plate were analysed via cPCR with the primers flo54  and flo56 (Hag-fw and Hag-rv). The result should be a ca. 1kb fragment for  correct pET24d-Hag containing clones and a 1,5 kb fragment for  pET24d-Hag-DARPin containing clones.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Volume (µL)</TH><TH scope="col">MM for 6 attempts (µL)</TH></TR><TR><TH scope="row">Picked clones from 18.42(1-3 pET24d-Hag, 6-7 from pET24d-Hag-DARPin)</TH><TD>0,5</TD><TD>-</TD></TR><TR><TH scope="row">Primer Flo 54 </TH><TD>-</TD><TD>3</TD></TR><TR><TH scope="row">Primer Flo 56</TH><TD>-</TD><TD>3</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>-</TD><TD>3</TD></TR><TR><TH scope="row">Phusion Buffer 5x</TH><TD>-</TD><TD>6</TD></TR><TR><TH scope="row">dNTP-mix</TH><TD>-</TD><TD>24</TD></TR><TR><TH scope="row">Water</TH><TD>-</TD><TD>81</TD></TR><TR><TH scope="row">Mastermix</TH><TD>19,5</TD><TD>-</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>120</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1 min 45 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>30x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The samples can be seen on the gel under 18.44. The PCR for the Hag-fragment in pET24d seemed to be positive (PCR.H1 - H3), as well as for the Hag-DARPin fragment (PCR.D7) </P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.44">18.44 test digest with <I>Nco</I>I/<I>Bam</I>HI - Checking insertion of Hag/-DARPin into pET24d</A></LEGEND><DIV class="aim"><P>Aim: proving insertion of Hag/-DARPin fragment into pET24d</P></DIV><DIV class="exp-content"><P>The plasmids were digested with <I>Nco</I>I and <I>Bam</I>HI. In case of a correct insertion the pet24d backbone and the insert should be seen on the gel at 5300 bp and 1000/ 1500 bp. As control pET24d-Hag-Flank (piGEM-001) is used to check the mastermix.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Volume (µL)</TH><TH scope="col">Mastermix(6 attempts)</TH></TR><TR><TH scope="row">Plasmid from clone 
      ( pET24d-Hag 1-3, pET24d-Hag-DARPin 6 &amp; 7)</TH><TD>5</TD><TD>-</TD></TR><TR><TH scope="row">Cutsmart 10x</TH><TD>-</TD><TD>3</TD></TR><TR><TH scope="row"><I>Nco</I>I</TH><TD>-</TD><TD>0,5</TD></TR><TR><TH scope="row"><I>Bam</I>HI</TH><TD>-</TD><TD>0,5</TD></TR><TR><TH scope="row">Water</TH><TD>-</TD><TD>26</TD></TR><TR><TH scope="row">Total Volume</TH><TD>10</TD><TD>30</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp20"><LEGEND><A name="exp20.9">20.9 Gel Purification of amplified cPCR products</A></LEGEND><DIV class="aim"><P>Aim: Purification for more efficient sequencing</P></DIV><DIV class="exp-content"><P>The PCR product of the  amplification of cPCR products was not pure considering different bands on the  gel next to the right one. In order to prevent sequencing mistakes the PCR  attempts were purified via gel extraction of the 1,5 kb band in case of the  Hag-DARPin construct.</P></DIV></FIELDSET><FIELDSET class="exp20"><LEGEND><A name="exp20.10">20.10  PCR with purified Hag-DARPin cPCR products for sequencing</A></LEGEND><DIV class="aim"><P>Aim: amplification of purified Hag-DARPin fragment for sequencing</P></DIV><DIV class="exp-content"><P>The purified cPCR product from 20.9 has to be amplified again in order to get a higher concentration for sequencing. </P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Volume (µL)</TH><TH scope="col">MM for 2 attempts (µL)</TH></TR><TR><TH scope="row">Purified PCR product from 20.x(Hag-/ Hag-DARPin fragment)</TH><TD>1-2</TD><TD>-</TD></TR><TR><TH scope="row">Primer Flo 54 </TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Primer Flo 56</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Phusion Buffer 5x</TH><TD>-</TD><TD>2</TD></TR><TR><TH scope="row">dNTP-mix</TH><TD>-</TD><TD>20</TD></TR><TR><TH scope="row">Water</TH><TD>-</TD><TD>75</TD></TR><TR><TH scope="row">Mastermix</TH><TD>19</TD><TD>-</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>100</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1 min 45 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>35x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.45">18.45 <I>Spe</I>I  digest of pET24d-Hag (piGEM-019)</A></LEGEND><DIV class="aim"><P>Aim: linearizing pET24d-Hag with <I>Spe</I>I site for a Gibson assembly with Cup1-1</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Volume (µL)</TH></TR><TR><TH scope="row">piGEM-016 (ng/µL)</TH><TD>17</TD></TR><TR><TH scope="row">CutSmart 10x</TH><TD>2</TD></TR><TR><TH scope="row"><I>Spe</I>I</TH><TD>0,5</TD></TR><TR><TH scope="row">Water</TH><TD>0,5</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD></TR></TBODY></TABLE><P>Incubation 1 h  at 37°C with following heat inactivation at 80°C</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.46">18.46 Gibson assembly with linearized pET24d-Hag (piGEM-019) and cup1-1</A></LEGEND><DIV class="aim"><P>Aim: Insertion of cup1-1 domain into pET24d-Hag construct for crystallization</P></DIV><DIV class="exp-content"><P>As insert cup1-1 PCR product from 15.55 and <I>Spe</I>I digested piGEM-016 from 18.44 were used for a new Gibson assembly.
      PCR fragment cup1-1 (464 ng/µL)    <I>Spe</I>I digested pIGEM-005 ( ng/µL)    </P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Volume (µl)</TH></TR><TR><TH scope="row">Gibson-mix</TH><TD>15</TD></TR><TR><TH scope="row">Insert (cup1-1 464 ng/µL)</TH><TD>2,5</TD></TR><TR><TH scope="row">piGEM-019</TH><TD>2,5</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD></TR></TBODY></TABLE><P>The mix was incubated 5 min at room temperature and then for 1h at 50°C.</P><P>In the end the whole mix was used to transform <I>E. coli</I> XL1-Blue, which were plated  out on LB-Amp. Incubation was done over night at 37°C.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="29.06.2014">29.06.2014</A></H2><FIELDSET class="exp18"><LEGEND><A name="exp18.47">18.47 cPCR of Gibson assembly clones of pET24d-Hag (piGEM-019) and cup1-1</A></LEGEND><DIV class="aim"><P>Aim: screening clones for right insert</P></DIV><DIV class="exp-content"><P>In order to screen the clones for  the right insertion of cup1-1 four clones were picked from the LB-Can plate and  transferred onto a LB-Can plate. Additionally the clones were used for  inoculation of minipreps and also cooked in 50 µL dest. Water for 10 min at  95°C. The lysate was used as template.</P><P>The PCR with Flo54 and iGEM-025  should generate a 785 bp fragment and the one with iGEM-024 and -025 the 164bp cup domain fragment.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Volume (µL)</TH><TH scope="col">MM 1 for 5 attempts (µL)</TH><TH scope="col">MM 2 for 5 attempts (µL)</TH></TR><TR><TH scope="row"><P>Cooked lysate solution from clones 1-4</P></TH><TD>3</TD><TD>-</TD><TD> </TD></TR><TR><TH scope="row">Primer Flo 54 </TH><TD>-</TD><TD>2,5</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-024</TH><TD>-</TD><TD>-</TD><TD>2,5</TD></TR><TR><TH scope="row">Primer iGEM-025</TH><TD>-</TD><TD>-</TD><TD>2,5</TD></TR><TR><TH scope="row">dNTP-mix</TH><TD>-</TD><TD>2,5</TD><TD>2,5</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>-</TD><TD>5</TD><TD>5</TD></TR><TR><TH scope="row">Phusion Buffer 5x</TH><TD>-</TD><TD>20</TD><TD>20</TD></TR><TR><TH scope="row">Water</TH><TD>-</TD><TD>67,5</TD><TD>67,5</TD></TR><TR><TH scope="row">Mastermix</TH><TD>17</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>100</TD><TD>100</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>10 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1 min 45 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>35x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The PCR showed no bands. Nevertheless, colonies were used to inoculate cultures for minipreps so that the isolated plasmid can be further analysed for the insert.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="30.06.2014">30.06.2014</A></H2><FIELDSET class="exp"><LEGEND><A name="exp_5">Sequencing</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Premix</TH><TH scope="col">Label-Nr.</TH><TH scope="col">Construct</TH><TH scope="col">Primers</TH></TR><TR><TH scope="row">1</TH><TD>AGB0023<STRONG>411</STRONG></TD><TD>pET24d-Hag-<I>Spe</I>I  construct cl. 2</TD><TD>Hag-Nco-fw</TD></TR><TR><TH scope="row">2</TH><TD>AGB0023<STRONG>412</STRONG></TD><TD>pET24d-Hag-DARPin  construct cl. 7</TD><TD>Hag-Nco-fw</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.48a">18.48a Miniprep of plasmids from Gibson assembly clones with pET24d-Hag (piGEM-019) and cup1-1</A></LEGEND><DIV class="aim"><P>Aim: isolation of Plasmid DNA for further experiments</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Construct</TH><TH scope="col">Clone</TH><TH scope="col">Concentration (ng/µL)</TH></TR><TR><TH scope="row">pET24d-Hag-Cup1-1</TH><TD>1</TD><TD>122</TD></TR><TR><TH scope="row">pET24d-Hag-Cup1-1</TH><TD>2</TD><TD>85</TD></TR><TR><TH scope="row">pET24d-Hag-Cup1-1</TH><TD>3</TD><TD>172</TD></TR><TR><TH scope="row">pET24d-Hag-Cup1-1</TH><TD>4</TD><TD>116</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.48b">18.48b PCR of isolated plasmids from Gibson assembly clones (pET24d-Hag (piGEM-019) and cup1-1)</A></LEGEND><DIV class="aim"><P>Aim: screening clones for right insert</P></DIV><DIV class="exp-content"><P>The isolated plasmid 1:10 diluted from the picked clones 1-4 were used as PCR template.</P><P>The PCR with Flo54 and iGEM-025  should generate a 785 bp fragment and the one with iGEM-024 and -025 the 164bp  domain fragment.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Volume (µL)</TH><TH scope="col">MM 1 for 5 attempts (µL)</TH><TH scope="col">MM 2 for 5 attempts (µL)</TH></TR><TR><TH scope="row"><P>Cooked lysate solution from clones 1-4 (1:10)</P></TH><TD>3</TD><TD>-</TD><TD> </TD></TR><TR><TH scope="row">Primer Flo 54 </TH><TD>-</TD><TD>2,5</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-024</TH><TD>-</TD><TD>-</TD><TD>2,5</TD></TR><TR><TH scope="row">Primer iGEM-025</TH><TD>-</TD><TD>2,5</TD><TD>2,5</TD></TR><TR><TH scope="row">dNTP-mix</TH><TD>-</TD><TD>2,5</TD><TD>2,5</TD></TR><TR><TH scope="row">Phusion</TH><TD>-</TD><TD>5</TD><TD>5</TD></TR><TR><TH scope="row">Phusion Buffer 5x</TH><TD>-</TD><TD>20</TD><TD>20</TD></TR><TR><TH scope="row">Water</TH><TD>-</TD><TD>67,5</TD><TD>67,5</TD></TR><TR><TH scope="row">Mastermix</TH><TD>17</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>100</TD><TD>100</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1 min 45 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>35x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>No bands could be observed on the gel. The PCR was repeated with Q5 mastermix to exclude the disfunction of the polymerase.</P></DIV></FIELDSET><FIELDSET class="exp20"><LEGEND><A name="exp20.9">20.9 Inoculation of a preculture from <I>Bacillus subtilis</I> WT 3610</A></LEGEND><DIV class="aim"><P>Aim: Preparation for a mainculture the for the next day</P></DIV><DIV class="exp-content"><P>2 x 6 mL of LB were inoculated with  <I>Bacillus subtilis</I> WT3610 from an LB plate and incubated at 37°C over night.</P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.47">14.47 Purification of PheA-Arc1p-C-0x</A></LEGEND><DIV class="aim"><P>Aim: Purify PheA-Arc1p-C-0x in three steps</P></DIV><DIV class="exp-content"><P>The resuspended cells from 14.46 were lysed using a french press. The lysate was incubated on ice with 5 µg/mL DNAseA and RNAseI for 10 min and centrifuged (27000 rpm, 45 min, 4 °C) to pellet cell debris. The supernatant was filtered (0.45 µm) and PheA-Arc1p-C-0x purified from the cleared solution by NiNTA affinity chromatography. The combined elution fractions containing crude PheA-Arc1p-C-0x were concentrated to a volume of 2 mL and further purified using an anion exchange column to get rid of possible tRNA contaminations. Finally the buffer was changed to dialysis buffer and the protein concentrated and stored at -80 °C in aliquods for further use.</P></DIV></FIELDSET></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Marburg:Project:Notebook:June">http://2014.igem.org/Team:Marburg:Project:Notebook:June</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Marburg:Project:Notebook:June" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Marburg:Project:Notebook:June" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Marburg:Project:Notebook:June&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Marburg:Project:Notebook:June&amp;oldid=288963" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>