<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Marburg_Safety_Modelling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Marburg:Safety:Modelling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Marburg:Safety:Modelling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Marburg:Safety:Modelling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Marburg:Safety:Modelling&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Marburg:Safety:Modelling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Marburg:Safety:Modelling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="navbox"><NAV class="menu center first" aria-haspopup="true" aria-controls="p1"><UL id="p1" class="first"><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg" class="a">Home</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project" class="b">Project</A></LI><LI class="current"><A href="https://2014.igem.org/Team:Marburg:Safety" class="c">Safety</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Team" class="d">Team</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Policy_Practices" class="e">Policy &amp; Practices</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Attributions" class="f">Attributions</A></LI></UL></NAV><NAV class="menu center second" aria-haspopup="true" aria-controls="p1"><UL id="p1" class="second"><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Safety" class="b">Killswitch</A></LI><LI class="current"><A href="https://2014.igem.org/Team:Marburg:Safety:Modelling" class="c">Modelling</A></LI><LI class="last" aria-haspopup="false"><A href="https://igem.org/Safety/Safety_Form?team_id=1329" target="_blank" class="e">Safety form</A></LI></UL></NAV></DIV><DIV class="mr_content"><H1>

Modelling cellular behaviour and robustness of the SURFkiller

</H1><P>In order to predict the behaviour of our SURFkiller, we created a model of our system. For this we used MATLAB software environment. 
The aim was to simulate cellular protein synthesis in different situations, and based on this information predict the robustness of the SURFkiller. 
Modelling was based on the following equations using the parameters explained in Table 1.
</P><P><TABLE><CAPTION>
	Table 1: Parameters used in the equations modelling the SURFkiller.
	</CAPTION><THEAD><TR><TH>Parameter</TH><TH>Description</TH></TR></THEAD><TBODY><TR><TD class="t1">[m<SUB>x</SUB>]</TD><TD class="t2">mRNA concentration</TD></TR><TR><TD class="t1">α<SUB>0,x</SUB></TD><TD class="t2">Maximum transcription rate</TD></TR><TR><TD class="t1">β<SUB>x</SUB></TD><TD class="t2">Protein synthesis rate</TD></TR><TR><TD class="t1">δ<SUB><I>m</I>x</SUB></TD><TD class="t2">mRNA degradation rate</TD></TR><TR><TD class="t1">δ<SUB>p</SUB></TD><TD class="t2">Standard dilution rate of the protein due to cell division</TD></TR><TR><TD class="t1">δ<SUB>R</SUB></TD><TD class="t2">Degradation rate of TetR due to LVA tag</TD></TR><TR><TD class="t1">Κ<SUB>x</SUB></TD><TD class="t2">Dissociation (Equilibrium) constant</TD></TR><TR><TD class="t1">[Χ]</TD><TD class="t2">Protein concentration</TD></TR></TBODY></TABLE></P><P>The mathematical equations we based our model on are given as:
</P><P><B>a) Production of Antiholin</B></P><P><B>b) Production of Holin</B></P><P><B>c) Production of TetR</B></P><P><B>d) Production of the ribosome-hibernation factor YvyD</B></P><P><B>e) Production of the ribosomal protein RpL5</B></P><P>We assumed the transcription rate of both promoters to be <I>α<SUB>0</SUB></I>=10 nM/min representing a strong promoter. The translation rate is set to <I>β</I>=2.3 min<SUP>-1</SUP> to represent the average burst size of 10 proteins/mRNA for the assumed half-life time of five minutes resulting in the degradation rate <I>δ<SUB>mx</SUB></I>=log(2)/5 min<SUP>-1</SUP>. The proteins except for TetR are assumed to be stable and only diluted by cell division every 30 min resulting in the degradation rate <I>δ<SUB>P</SUB></I>=log(2)/30 min<SUP>-1</SUP>. The additional degradation of TetR due to the lva-tag is assumed to be <I>δ<SUB>P</SUB></I>=0.018 min<SUP>-1</SUP> (Andersen et al., 1998).
</P><P><B>Challenging different SURFkiller scenarios <I>in silico</I></B></P><P>To understand behaviour and robustness of SURFkiller in our model, we simulated several scenarios which would pose a challenge for the SURFkiller. The efficient and secure system would kill the cells under virtually any conditions as quickly as possible.
</P><P><B>Scenario 1: A SURFkiller-equipped GMO leaves the laboratory</B></P><P>The organism leaves the laboratory environment. Concentration of the IPTG anti-repressor quickly drops to minimal levels, as a consequence of the degradation and diffusion through the cell membrane. In this case the critical variable is the concentration of the RpL5 essential ribosomal protein. The results are shown in graphs below.
</P><DIV class="figure" style="width:90%;"><SPAN class="caption"><B>Figure 1:</B> mRNA concentration is plotted against time <B>(left)</B>. Concentration of RpL5 plotted against time <B>(right)</B>. 
The time of outbreak is t=0 (red line). The cell is killed after 63 min at latest (green line).
</SPAN></DIV><P>At the time point t=200 min, the mRNA concentration sharply drops, production of the RpL5 protein stops and its concentration in the cells starts to decrease. If we assume that a cell needs around 2000 functional ribosomes in order to survive the cell death occurs at latest when the level of the RpL5 protein drops below 2000 nM. A concentration of 2000 nM in the volume of a cell, which is about 1 fL, equals about 2000 molecules. From our simulation we can see that the current number of ribosomes is below the critical number of 2000 at latest 60 minutes after the cell leaves the laboratory environment even if we assume that every RpL5 molecule leads to the formation of a functional ribosome.
</P><P><B>Case 2: Mutations compromising modular functions of the SURFkiller</B></P><P>The efficiency of a killswitch can be compromised with mutation that may occur in one of the promoters used in the system. SURFkiller is designed to remain robust even in these situations, incorporating a secondary toxin-antitoxin system (Holin-Antiholin) that balances the promoter function in our system. In this case we simulated a situation where one of the LacI promoters gets constitutive. The essential RpL5 protein is always produced in this case, and does not lead to cell death. However, since the toxin is also under control of the same promoter it also gets produced.
</P><DIV class="figure" style="width:90%;"><SPAN class="caption"><B>Figure 2:</B> Plotted on the left is the concentration of Holin (red) and Antiholin (blue) against time. 
The right graph shows the difference in concentration between both. The cells die 32 min (green line) after the outbreak (red line).</SPAN></DIV><P>As the organism leaves the laboratory environment, the level of antitoxin production start to sink, widening the gap between the level of toxin and antitoxin in the organism. The critical concentration of toxin in the cell, in the case of T4-Holin, is around 1000-3000 molecules. Since the number of free Holin molecules dictates cell death, we concentrate on the difference between toxin and antitoxin levels in the cell. Even if we take the worst case, where 3000 free T4- Holin molecules are needed, our model shows that the cell lysis will occur at latest 30 minutes after it leaves the laboratory environment.
</P><P>A variation of this scenario could be when a mutation occurs after the bacteria leave the laboratory but before they die due to the lack of RpL5.
</P><DIV class="figure" style="width:40%;"><SPAN class="caption"><B>Figure 3: Concentration of RpL5 plotted against time showing that a mutation occurs before the concentration drops below a safe level.</B></SPAN></DIV><DIV class="figure" style="width:90%;"><SPAN class="caption"><B>Figure 4:</B> On the left concentration of Holin (red) and Antiholin (blue) is plotted against time. On the right graph the 
difference between both is shown. With the mutation occurring 50 min after the outbreak (red line) the cells are dead 31 min later (green line).
</SPAN></DIV><P>The mutation occurs at the T=250 min in the simulation depicted in graphs above, and we can see how the concentration of both the RpL5 protein and the T4-Holin rebounds after the initial drop caused by the bacteria leaving the laboratory. The Antiholin production is not affected by this mutation, as it is located on another module in the system. Regardless of the mutation, Holin-induced cell lysis still occurs at latest 30 minutes after the mutation. Otherwise, cell dies from the lack of the RpL5 protein, as in the first case.
</P><P><B>Scenario 3: RpL5 is lost from SURFkiller</B></P><P>For our third scenario we decided to test the unlikely case where the RpL5 encoding gene is removed from the operon controlled by the Lac-promoter, for instance by the means of homologous recombination. It is then possible that the levels of the RpL5 protein will be high enough in the cell while the difference between Holin and Antiholin is kept low, even if it escapes the laboratory. The third security layer of our SURFkiller is designed for this case, and activated if all other security measures get compromised. 
</P><DIV class="figure" style="width:40%;"><SPAN class="caption"><B>Figure 5: Concentration of TetR (red) and YvyD (blue) plotted against time.</B></SPAN></DIV><P>Outside of the laboratory environment the TetR production stops, and its concentration starts to sink, as shown in Figure 5. As its 
concentration sinks it is unable to repress the constitutive promoter and the<I>yvyD</I> transcription slowly starts resulting in the 
production of YvyD, shown in red. As the YvyD level rises, more and more ribosomes dimerize, resulting in cell death. The short period 
of YvyD production at the beginning is due to the fact that in the beginning of the simulation the concentration to TetR is assumed to 
be zero.
</P><P><B>Sensitivity of the simulation on the parameters used</B></P><P>In order to make sure that our simulation adequately describes real-world scenarios we ran a calculation on how our results depend on different choice of parameters, primarily transcription rates of mRNA and translation rates of proteins. Choice and calculation of these parameters is one of the most challenging tasks in modelling process, so this simulation sheds light on how our SURFkiller would function, even with non-optimal parameters.
</P><DIV class="figure" style="width:90%;"><SPAN class="caption"><B>Figure 6: Time till the cells die plotted against translation or transcription rate showing the values (green line) used in the calculation of the above scenarios.</B></SPAN></DIV><P>As we can see from the graphs above, if we vary the standard translation rate of 2.3 min <SUP>-1</SUP> there is comparatively small change in the time needed for the cell death to occur. If the translation rate is below 0.6 the cells cannot survive even under laboratory conditions because the steady state number of ribosomes is below 2000. Similar is applicable to variations in transcription rate. If the assumed transcription rate of 10 nM/min doubles, the expected survival time of the cell is around 25 minutes longer.
</P><P>The modelling of our SURFkiller showed that the system indeed works and leads to a quick decrease of RpL5 upon the deprivation of IPTG. Also the further layers of security against mutations that might occur have been shown to be working. What we learned though is that it might be better to use several copies of <I>yvyD</I> to increase the steady state protein concentration, so that the ribosome number decreases by dimerization quickly below an essential number for survival. Our analysis of the sensitivity of the system suggest that we established a robust circuit design concerning the translation rate, which means that changes of this rate lead only to a small change in time until the cells die. However the transcription rate should be increased for example by using a stronger promoter in order to make the system even more robust.
</P><P>Andersen, J. B., Sternberg, C., Poulsen, L. K., Bjorn, S. P., Givskov, M., &amp; Molin, S. (1998). New unstable variants of green fluorescent protein for studies of transient gene expression in bacteria. Applied and Environmental Microbiology, 64(6), 2240–6. Retrieved from <A href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=106306&amp;tool=pmcentrez&amp;rendertype=abstract" class="external free" rel="nofollow">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=106306&amp;tool=pmcentrez&amp;rendertype=abstract</A></P><P>Lauber, M.A., Running, W.E., and Reilly, J.P. (2009) B. subtilis ribosomal proteins: structural homology and post-translational modifications. J Proteome Res 8: 4193–206 <A href="http://www.ncbi.nlm.nih.gov/pubmed/19653700" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/19653700</A>. Accessed October 6, 2014.
</P><P>Savva, C.G., Dewey, J.S., Deaton, J., White, R.L., Struck, D.K., Holzenburg, A., and Young, R. (2008) The holin of bacteriophage lambda forms rings with large diameter. Mol Microbiol 69: 784–793 <A href="http://www.ncbi.nlm.nih.gov/pubmed/18788120" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/18788120</A>. Accessed October 14, 2014.
</P><P>Wang, I.N., Smith, D.L., and Young, R. (2000) Holins: the protein clocks of bacteriophage infections. Annu Rev Microbiol 54: 799–825 <A href="http://www.ncbi.nlm.nih.gov/pubmed/11018145" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/11018145</A>. Accessed October 13, 2014.
</P><P>Young, R., and Bläsi, U. (1995) Holins: form and function in bacteriophage lysis. FEMS Microbiol Rev 17: 191–205 <A href="http://www.ncbi.nlm.nih.gov/pubmed/7669346" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/7669346</A>. Accessed October 14, 2014.
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Marburg:Safety:Modelling">http://2014.igem.org/Team:Marburg:Safety:Modelling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Marburg:Safety:Modelling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Marburg:Safety:Modelling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Marburg:Safety:Modelling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Marburg:Safety:Modelling&amp;oldid=390338" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>