<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Marburg_Project_Notebook_May"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Marburg:Project:Notebook:May">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Marburg:Project:Notebook:May&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Marburg:Project:Notebook:May&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Marburg:Project:Notebook:May&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Marburg:Project:Notebook:May" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Marburg:Project:Notebook:May</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="navbox"><NAV class="menu center first" aria-haspopup="true" aria-controls="p1"><UL id="p1" class="first"><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg" class="a">Home</A></LI><LI class="current"><A href="https://2014.igem.org/Team:Marburg:Project" class="b">Project</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Safety" class="c">Safety</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Team" class="d">Team</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Policy_Practices" class="e">Policy &amp; Practices</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Attributions" class="f">Attributions</A></LI></UL></NAV><NAV class="menu center second" aria-haspopup="true" aria-controls="p1"><UL id="p1" class="second"><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:Silver" class="d">SilverSURF</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:Tumor" class="c">CancerSURF</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:NRPS" class="e">RiboSURF</A></LI><LI class="current"><A href="https://2014.igem.org/Team:Marburg:Project:Notebook" class="f">Notebook</A></LI><LI class="last" aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:Parts" class="g">Parts</A></LI></UL></NAV></DIV><DIV class="mr_content"><H1>

Notebook: May

</H1><UL id="submenu"><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook">March</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:April">April</A></LI><LI class="current"><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:May">May</A></LI><LI><A href="#01.05.2014">01.05.2014</A></LI><LI><A href="#02.05.2014">02.05.2014</A></LI><LI><A href="#05.05.2014">05.05.2014</A></LI><LI><A href="#06.05.2014">06.05.2014</A></LI><LI><A href="#07.05.2014">07.05.2014</A></LI><LI><A href="#08.05.2014">08.05.2014</A></LI><LI><A href="#09.05.2014">09.05.2014</A></LI><LI><A href="#10.05.2014">10.05.2014</A></LI><LI><A href="#11.05.2014">11.05.2014</A></LI><LI><A href="#12.05.2014">12.05.2014</A></LI><LI><A href="#13.05.2014">13.05.2014</A></LI><LI><A href="#14.05.2014">14.05.2014</A></LI><LI><A href="#15.05.2014">15.05.2014</A></LI><LI><A href="#16.05.2014">16.05.2014</A></LI><LI><A href="#19.05.2014">19.05.2014</A></LI><LI><A href="#20.05.2014">20.05.2014</A></LI><LI><A href="#21.05.2014">21.05.2014</A></LI><LI><A href="#22.05.2014">22.05.2014</A></LI><LI><A href="#23.05.2014">23.05.2014</A></LI><LI><A href="#26.05.2014">26.05.2014</A></LI><LI><A href="#27.05.2014">27.05.2014</A></LI><LI><A href="#28.05.2014">28.05.2014</A></LI><LI><A href="#29.05.2014">29.05.2014</A></LI><LI><A href="#30.05.2014">30.05.2014</A></LI></UL><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:June">June</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:July">July</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:August">August</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:September">September</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:October">October</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:Methods">Methods</A></LI><DIV class="notebooky-entry"><H2 class="title"><A name="01.05.2014">01.05.2014</A></H2><FIELDSET class="exp15"><LEGEND><A name="exp15.16">15.16 Purification of Restriction Digest from 30.04.14</A></LEGEND><DIV class="exp-content"><P>Purifying the samples from the 30.04.2014 with gel extraction kit (QIAquick Gel Extraction Kit) according to the protocol.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Fragment</TH><TH scope="col">Concentration [ng/µl]</TH></TR><TR><TH scope="row">pMAD Digest 1</TH><TD>21</TD></TR><TR><TH scope="row">pMAD Digest 2</TH><TD>19</TD></TR><TR><TH scope="row">Hag-Flank Construct 1 Digest</TH><TD>16,9</TD></TR><TR><TH scope="row">Hag-Flank Construct 2 Digest</TH><TD>17,6</TD></TR><TR><TH scope="row">Hag-Flank Construct 3 Digest</TH><TD>11,3</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.17">15.17 Ligation of purified pMAD and Constructs</A></LEGEND><DIV class="exp-content"><P>The ligation is carried out with the purified pMAD digest and the hag-flank-construct digest from 15.16 in order to transform the ligation mix into<EM> E.Coli</EM> DH5α.
For the ligation was the following calculator used: <A href="https://2011.igem.org/Team:UT_Dallas/ligation">UT Dallas</A></P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix [µl]</TH><TH scope="col">Control 1</TH><TH scope="col">Control 2</TH><TH scope="col">Construct 1.1</TH><TH scope="col">Construct 1.2</TH><TH scope="col">Construct 2.1</TH><TH scope="col">Construct 2.2</TH><TH scope="col">Construct 3.1</TH><TH scope="col">Construct 3.2</TH></TR><TR><TH scope="row">Plasmid (pMAD digest 1)</TH><TD>1</TD><TD>-</TD><TD>4,5</TD><TD>-</TD><TD>4,7</TD><TD>-</TD><TD>3</TD><TD>-</TD></TR><TR><TH scope="row">Plasmid (pMAD digest 2)</TH><TD>-</TD><TD>1</TD><TD>-</TD><TD>5</TD><TD>-</TD><TD>5</TD><TD>-</TD><TD>3,3</TD></TR><TR><TH scope="row">Insert (construct)</TH><TD>-</TD><TD>-</TD><TD>5</TD><TD>5</TD><TD>5</TD><TD>4,8</TD><TD>5</TD><TD>5</TD></TR><TR><TH scope="row">T Ligase</TH><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">T4 Ligase Buffer 10x</TH><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR><TH scope="row">H2O</TH><TD>16</TD><TD>16</TD><TD>7,5</TD><TD>7</TD><TD>7,3</TD><TD>7,2</TD><TD>9</TD><TD>8,7</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>20</TD><TD>20</TD><TD>20</TD><TD>20</TD><TD>20</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><P>Incubation for 1,5 h at room temperature.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.18">15.18 Transformation of <EM>E.coli</EM> DH5α</A></LEGEND><DIV class=""><P>Transformation of <EM>E.coli</EM> DH5α with the ligation attempt.
      10µl of each the attempts were used to transform 50 µl of competent<EM> E. coli </EM>DH5α in order to test the success of the ligation and raising colonies for inoculating a mini prep. </P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.19">15.19 PCR with primers</A></LEGEND><DIV class="exp-content"><P>To have enough of the construct for further works, a new PCR (as described  before) was done overnight.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="02.05.2014">02.05.2014</A></H2><FIELDSET class="exp13"><LEGEND><A name="exp13.36">13.36 Miniprep of pMA12 from Yeast</A></LEGEND><DIV class="aim"><P>Aim: Isolate the plasmid from yeast.</P></DIV><DIV class="exp-content"><P>The yeast was washed from the plate with 2 ml of A. dest. After centrifugation at 11000 rpm for 1 min the supernatant was discarded and 500 µl of Solution 1 of the Omega plasmid prep kit was added. The yeast was vortexed for 5 min after addition of a few glas balls. After that the miniprep was carried out according to the protocol (Omega). (c = 134.5 ng/µl)</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.37">13.377 Transformation of <EM>E.coli</EM> DH5α with the isolated pMA12</A></LEGEND><DIV class="aim"><P>Aim: Transform <EM>E.coli</EM> with the isolated plasmid to increase the amount of plasmid.</P></DIV><DIV class="exp-content"><P><EM>E. coli</EM> DH5Alpha cells were transformed with 10µl of the isolated plasmid pMa12. They were incubated at room temperature over the weekend.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.20">15.20 Purification of Fragments</A></LEGEND><DIV class="exp-content"><P>Purification of the fragments from 01.05.14. The PCR-products were separated on a 1% agarose-gel.</P></DIV><DIV class="results"><H3>Results</H3><P>The fragment with the correct size was  excised from the gel and purified with the Gel Extraction Kit (QIAquick Gel Extraction Kit)(c = 6.6 ng/µl; c2  = 5.8 ng/µl).</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="05.05.2014">05.05.2014</A></H2><FIELDSET class="exp13"><LEGEND><A name="exp13.38">13.38 Inoculation of Miniprep Cultures </A></LEGEND><DIV class="aim"><P>Aim: Pick clones to check for the correct plasmids.
    </P></DIV><DIV class="exp-content"><P>15 colonies of the transformed <EM>E.coli</EM> DH5α were picked and incubated in 6 ml LB-Amp at 37°C overnight.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.21">15.21 Ligation of the hag/flank and pET24d both cut with <I>Bam</I>HI and <I>Nco</I>I followed by Transformation of <EM>E. coli</EM></A></LEGEND><DIV class="exp-content"><P>The ligation with pMAD was not successful so it was decided to clone the hag/flank-construct first into pET24d. The ligation was done after using the ligation calculator: <A href="https://2011.igem.org/Team:UT_Dallas/ligation" target="_blank">UT Dallas</A></P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">[µl]</TH><TH scope="col">Fragment 1</TH><TH scope="col">Fragment 2</TH><TH scope="col">Fragment 3</TH></TR><TR><TH scope="row">pET24d</TH><TD>2,9</TD><TD>3,1</TD><TD>2</TD></TR><TR><TH scope="row">Fragment</TH><TD>10</TD><TD>10</TD><TD>10</TD></TR><TR><TH scope="row">T4-Ligase-Buffer (10x)</TH><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR><TH scope="row">T4 Ligase</TH><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">H20</TH><TD>4,1</TD><TD>3,9</TD><TD>5</TD></TR><TR><TH scope="row">Total</TH><TD>20</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><P>The ligation was incubated at RT for about 2 hours followed by a transformation of <EM>E. coli</EM> DH5α. The transformed cells were plated out on LB-Kan and incubated at 37°C over night.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="06.05.2014">06.05.2014</A></H2><FIELDSET class="exp13"><LEGEND><A name="exp13.39">13.39 Miniprep of the 15 Cultures and Restriction with <I>Nco</I>I</A></LEGEND><DIV class="aim"><P>Aim: isolate the plasmid from the picked clones and check for restriction sites of <I>Nco</I>I.</P></DIV><DIV class="exp-content"><P>The plasmids were isolated according to the protocol &quot;Miniprep (Plasmid DNA Miniprep Kit II, Omega)&quot;.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Clone</TH><TH scope="col">Concentration (ng/µl)</TH></TR><TR><TH scope="row">1</TH><TD>570,2</TD></TR><TR><TH scope="row">2</TH><TD>735,5</TD></TR><TR><TH scope="row">3</TH><TD>671,9</TD></TR><TR><TH scope="row">4</TH><TD>642,6</TD></TR><TR><TH scope="row">5</TH><TD>660,9</TD></TR><TR><TH scope="row">6</TH><TD>688</TD></TR><TR><TH scope="row">7</TH><TD>982,6</TD></TR><TR><TH scope="row">8</TH><TD>637,4</TD></TR><TR><TH scope="row">9</TH><TD>635,4</TD></TR><TR><TH scope="row">10</TH><TD>1306,6</TD></TR><TR><TH scope="row">11</TH><TD>523,9</TD></TR><TR><TH scope="row">12</TH><TD>554,4</TD></TR><TR><TH scope="row">13</TH><TD>501,3</TD></TR><TR><TH scope="row">14</TH><TD>825</TD></TR><TR><TH scope="row">15</TH><TD>771,2</TD></TR></TBODY></TABLE><P>The isolated plasmids were digested with <I>Nco</I>I for 1h 20 min and further analyzed on a 1% agarose gel.</P></DIV><DIV class="results"><H3>Results</H3><P>Restriction of the isolated pMa12 with <I>Nco</I>I; the lanes contain the digested pMa12 in the order of the picked clones 1 - 15; clone 9 was negative; the lanes between the restrictions and the marker contain the isolated pMAD but it appears to be empty.   The most clones contained a plasmid that has only one <I>Nco</I>I restriction site. The expected size of the pMa12 should be around 6.6 kb. All fragments are of the expected size.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.40">13.40 Restriction of pMa12 (isolated from Clone 10) with <I>Sac</I>I and <I>Nco</I>I</A></LEGEND><DIV class="aim"><P>Aim: create fitting ends to anneal the promotor oligos.</P></DIV><DIV class="exp-content"><P>The isolated pMa12 of clone 10 was used for a restriction with <I>Sac</I>I-HF and <I>Nco</I>I. The restriction leads to fitting ends for the designed oligos which are the promoters we are about to test: PargJ and PykuO. The restriction is incubated at RT over night.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Restriction</TH><TH scope="col">Volume [µl]</TH></TR><TR><TH scope="row">pMA12 from clone 10</TH><TD>2</TD></TR><TR><TH scope="row"><I>Nco</I>I</TH><TD>1</TD></TR><TR><TH scope="row"><I>Sac</I>I-HF</TH><TD>1</TD></TR><TR><TH scope="row">CutSmart Buffer (10x)</TH><TD>2</TD></TR><TR><TH scope="row">H2O</TH><TD>14</TD></TR><TR><TH scope="row">Total</TH><TD>20</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.21">15.21 Inoculation of the DH5α and Miniprep of pET24d_hag/flank</A></LEGEND><DIV class="aim"><P>Aim: amplifying and isolation of pET24d_hag/flank.</P></DIV><DIV class="exp-content"><P>On the plates with the transformands of yesterday were only three colonies. Each 6 ml LB-Kan were inoculated with the colonies and incubated at 27°C for around 9 h. After growth of the bacteria the plasmids were isolated with the Omega Plasmid Prep Kit. </P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="07.05.2014">07.05.2014</A></H2><FIELDSET class="exp15"><LEGEND><A name="exp15.22">15.22 Restriction of 3 isolated plasmids (6.5.14) with <I>Nco</I>I and <I>Bam</I>HI</A></LEGEND><DIV class="aim"><P>Aim: Test whether the isolated plasmid contains the 2 kb insert (hag-flank construct).</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">[µl]</TH><TH scope="col">Clone 1</TH><TH scope="col">Clone 2</TH><TH scope="col">Clone 3</TH></TR><TR><TH scope="row">Plasmid (500ng)</TH><TD>10</TD><TD>5</TD><TD>5</TD></TR><TR><TH scope="row">Water</TH><TD>8</TD><TD>12</TD><TD>12</TD></TR><TR><TH scope="row">CutSmart Buffer (10x)</TH><TD>2</TD><TD>2</TD><TD>3</TD></TR><TR><TH scope="row"><I>Nco</I>I</TH><TD>0,5</TD><TD>0,5</TD><TD>0,5</TD></TR><TR><TH scope="row"><I>Bam</I>HI</TH><TD>0,5</TD><TD>0,5</TD><TD>0,5</TD></TR></TBODY></TABLE><P>Restriction for 45 min at 37°C. 
	  Agarose-gel (1%) to compare the cut and uncut plasmid.</P></DIV><DIV class="results"><P>Agarose gel with control (uncut) and 3 cut clones of pEt24d containing the hag-flank construct, fragments are of expected size.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.41">13.41 Purification of overnight digested plasmid pMa12 (3 inserts, only 1 <I>Nco</I>I restriction site) by gel-elution.</A></LEGEND><DIV class="aim"><P>Aim: purify plasmid to clone annealed oligos for the promoter sequences into the plasmid.</P></DIV><DIV class="exp-content"><P>Concentration: 3ng/µl</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.42">13.42 Annealing of Oligos</A></LEGEND><DIV class="aim"><P>Aim: create promotor sequences in plasmid piGEM001</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">[µl]</TH><TH scope="col">Reaction 1</TH><TH scope="col">Reaction 2</TH></TR><TR><TH scope="row">Oligos</TH><TD>iGEM012/iGEM013
      10µl/10µl</TD><TD>iGEM014/iGEM015
10µl/10µl</TD></TR><TR><TH scope="row">Water</TH><TD>80</TD><TD>80</TD></TR><TR><TH scope="row">Total</TH><TD>100</TD><TD>100</TD></TR></TBODY></TABLE><P>Heat up at 98°C for a few minutes. Let it cool down until room temperature</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.43">13.43 Ligation of the annealed oligos into digested piGEM001</A></LEGEND><DIV class="aim"><P>Aim: Create plasmid with the corresponding promoter sequences for Ag/Cu</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">[µl]</TH><TH scope="col">Reaction (Ag)</TH><TH scope="col">Reaction (Cu)</TH></TR><TR><TH scope="row">Plasmid (3ng/µl)</TH><TD>5,5</TD><TD>5,5</TD></TR><TR><TH scope="row">Water</TH><TD>15</TD><TD>15</TD></TR><TR><TH scope="row">Buffer (Ligation Buffer 10x)</TH><TD>2,5</TD><TD>2,5</TD></TR><TR><TH scope="row">Oligo-Mix</TH><TD>12/13</TD><TD>14/15</TD></TR><TR><TH scope="row">T4-DNA-Ligase</TH><TD>1</TD><TD>1</TD></TR></TBODY></TABLE><P>10µl of the ligation reaction will be transformed into chemically competent <EM>E.coli</EM> DH5α as described before.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.44">13.44 Over night restriction of pMA12 to obtain more digested plasmid (clone 10)</A></LEGEND><DIV class="exp-content"><P>See 06.05.2014</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Restriction</TH><TH scope="col">Volume [µl]</TH></TR><TR><TH scope="row">pMA12 from clone 10</TH><TD>2</TD></TR><TR><TH scope="row"><I>Nco</I>I</TH><TD>1</TD></TR><TR><TH scope="row"><I>Sac</I>I-HF</TH><TD>1</TD></TR><TR><TH scope="row">CutSmart Buffer (10x)</TH><TD>2</TD></TR><TR><TH scope="row">H2O</TH><TD>14</TD></TR><TR><TH scope="row">Total</TH><TD>20</TD></TR></TBODY></TABLE></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="08.05.2014">08.05.2014</A></H2><FIELDSET class="exp13"><LEGEND><A name="exp13.43a">13.43 Repeat Transformation for AG/Cu Promoters</A></LEGEND><DIV class="exp-content"><P>No colonies on the plates, repeat ligation and transformation with new digested plasmid.
	  Concentration plasmid: 17 ng/µl
	Ligation: 1.5µl of plasmid used
	Tranformation: as usual</P><H3>1. Sequencing</H3><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Label</TH><TH scope="col">Description</TH><TH scope="col">Primer</TH></TR><TR><TH scope="row">AFD001881</TH><TD>piGEM001</TD><TD>T7</TD></TR><TR><TH scope="row">AFD001882</TH><TD>piGEM001</TD><TD>T7 Turn</TD></TR><TR><TH scope="row">AFD001883</TH><TD>piGEM001</TD><TD>Hag revers (Flo)</TD></TR><TR><TH scope="row">AFD001884</TH><TD>piGEM002</TD><TD>TW 260 (Daniel)</TD></TR></TBODY></TABLE><P>Plasmids have to be prepared in a concentration between 50-100 ng/µl in a total volume of 10 to 15µl.
2µl Primers have been added to the premixes, Sample 881 and 882 will be sequenced with primers already present at the company (mwg eurofins).</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="09.05.2014">09.05.2014</A></H2><FIELDSET class="exp13"><LEGEND><A name="exp13.43b">13.43 Repeat Transformation for AG/Cu Promotors</A></LEGEND><DIV class="exp-content"><P>No colonies on the plates
	Repeat annealing of oligos
	Step 1:</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Amount [µl]</TH><TH scope="col"> </TH></TR><TR><TH scope="row">1</TH><TD>Oligo 1</TD></TR><TR><TH scope="row">1</TH><TD>Oligo 2</TD></TR><TR><TH scope="row">5</TH><TD>Ligase Buffer</TD></TR><TR><TH scope="row">1</TH><TD>PNK</TD></TR><TR><TH scope="row">42</TH><TD>Water</TD></TR></TBODY></TABLE><P> Incubation at 37°C for 30 min
    Step 2:
    Heat up water in microwave until boiling, add the tubes with primer mix
    let it cool down to room temperature
    Step 3:
    1:100 dilution
    Step 4: Ligation
    </P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Amount [µl]</TH><TH scope="col"> </TH></TR><TR><TH scope="row">3</TH><TD>Plasmid (17,6ng/µl)</TD></TR><TR><TH scope="row">2</TH><TD>Annealed Oligos (1:100)</TD></TR><TR><TH scope="row">1</TH><TD>Ligase</TD></TR><TR><TH scope="row">2</TH><TD>Ligase Buffer</TD></TR><TR><TH scope="row">12</TH><TD>Water</TD></TR></TBODY></TABLE><P>Step 5: Transformation
	Incubation at room temperature over two days.
    Step 6: Results
    Hag Flank construct digestion with <I>Spe</I>I was positive.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="10.05.2014">10.05.2014</A></H2><FIELDSET class="exp13"><LEGEND><A name="exp13.43c">13.43 Repeated Transformation for AG/Cu Promoters</A></LEGEND><P>No colonies</P></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.23">15.23 PCR for amplification of flagellin-constructs out of pet24-fla-Construct from 05.05.14</A></LEGEND><DIV class="aim"><P>Aim: Amplification of the flagellin-constructs from the pET24-fla-plasmid as a template was done in order to make a restriction digest with <I>Spe</I>I. This showed, which clone contained the successfully synthesized construct with <I>Spe</I>I restriction site because of the Strep-Tag from sequencing.</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix [µl]</TH><TH scope="col"><STRONG>Pet24fla1</STRONG></TH><TH scope="col"><STRONG>Pet24fla2</STRONG></TH><TH scope="col"><STRONG>Pet24fla3</STRONG></TH></TR><TR><TH scope="row">Temp. - pet24fla cone 1 (lig1 05.05.14)</TH><TD>0,5</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Temp. - pet24fla cone 2 (lig2 05.05.14)</TH><TD>-</TD><TD>0,5</TD><TD>-</TD></TR><TR><TH scope="row">Temp. - pet24fla cone 3 (lig3 05.05.14)</TH><TD>-</TD><TD>-</TD><TD>0,5</TD></TR><TR><TH scope="row">Primer  iGEM-89flo</TH><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Primer  iGEM-89flo</TH><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Q5-Buffer  5x</TH><TD>10</TD><TD>10</TD><TD>10</TD></TR><TR><TH scope="row">Phusion-Polymerase</TH><TD>0,5</TD><TD>0,5</TD><TD>0,5</TD></TR><TR><TH scope="row">Water</TH><TD>37</TD><TD>37</TD><TD>37</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD><TD>50</TD><TD>50</TD></TR></TBODY></TABLE><P>PCR protocol for &quot;PCR Q5 elongation for 2kb fragments&quot;.</P><TABLE width="100%" border="1"><TBODY><TR><TH width="17%" scope="col">Step</TH><TH width="58%" scope="col">Temperature °C</TH><TH width="25%" scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>95</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>95</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>2,5 min</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>35x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>Infinite</TD></TR></TBODY></TABLE><P>Result: PCR not successful - no band at 2,2kb</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.24">15.24 Transformation of <EM>E.coli</EM> DH5α with pET24-fla-Construct from 05.05.14</A></LEGEND><DIV class="aim"><P>Aim: Amplification plasmid for further experiments.</P></DIV><DIV class="exp-content"><UL class="list"><LI>3 aliquots competent E.Coli DH5α were transformed with 1µll of the purified plasmid  pet24-fla construct from 05.05.14 from 3 different clones (lig 1-3) each</LI><LI>After regeneration the cells were plated on LB-canamycin plates and incubated at room temperature until Monday</LI></UL></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.25a">15.25 New PCR for amplification of flagellin-constructs out of pET24-fla-Construct from 05.05.14</A></LEGEND><DIV class="exp-content"><P>Aim: Amplification of the fragment for further experiments with pet24-fla from 15.21 as template. The prior PCR was not successful.</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix [µl]</TH><TH scope="col"><STRONG>Pet24fla1</STRONG></TH><TH scope="col"><STRONG>Pet24fla2</STRONG></TH><TH scope="col"><STRONG>Pet24fla3</STRONG></TH></TR><TR><TH scope="row">Temp.  – pet24fla cone 1 (lig1 05.05.14)</TH><TD>0,5</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Temp.  – pet24fla cone 2 (lig2 05.05.14)</TH><TD>-</TD><TD>0,5</TD><TD>-</TD></TR><TR><TH scope="row">Temp.  – pet24fla cone 3 (lig3 05.05.14)</TH><TD>-</TD><TD>-</TD><TD>0,5</TD></TR><TR><TH scope="row">Primer  iGEM-89flo</TH><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Primer  iGEM-89flo</TH><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Q5-Mastermix</TH><TD>25</TD><TD>25</TD><TD>25</TD></TR><TR><TH scope="row">Water</TH><TD>22,5</TD><TD>22,5</TD><TD>22,5</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD><TD>50</TD><TD>50</TD></TR></TBODY></TABLE><P>Q5 PCR</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>95</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>95</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1 min</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>35x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>Infinite</TD></TR></TBODY></TABLE></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="11.05.2014">11.05.2014</A></H2><FIELDSET class="exp13"><LEGEND><A name="exp13.43d">13.43 Repeat Transformation for AG/Cu Promotors</A></LEGEND><DIV class="exp-content"><P>No colonies</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.25b">15.25 New PCR for ampl. of flagellin-constructs out of pet24-fla-Construct from 05.05.14</A></LEGEND><DIV class="results"><H3>Results:</H3><P>PCR result from the amplification of the fla-construct out of the pET24d. The gel shows that the three clones contain a construct in the pet24d vector. The fragment is about 2,2kb.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.26">15.26 Gel Extraction of PCR Products </A></LEGEND><DIV class="aim"><P>Aim: purification of constructs from gel as a preparation for the restriction digest with spe1. The digest will show which clone contains the construct with the replaced strap-tag.</P></DIV><DIV classs="exp-content"><P>Gel extraction was performed according to the gel extraction kit by Omega.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Extracted PCR product</TH><TH scope="col">Concentration [ng/µl]</TH></TR><TR><TH scope="row">clone 1</TH><TD>80,2</TD></TR><TR><TH scope="row">clone 2</TH><TD>88,2</TD></TR><TR><TH scope="row">clone 3</TH><TD>93,9</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.27">15.27 Restriction Digest of PCR Products from with <I>Spe</I>I</A></LEGEND><DIV class="aim"><P>Aim: The digest with <I>Spe</I>1 will show, which clone contains the fla-construct with replaced StrepTag by a <I>Spe</I>I restriction site</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">[µl]</TH><TH scope="col">Mastermix with <I>Nco</I>I - for 4 attempts</TH><TH scope="col">Attempt 1-3 <I>Spe</I>I (20kU/ml)</TH></TR><TR><TH scope="row">DNA</TH><TD>-</TD><TD>Ca. 85ng/µl ca. 1µg → 12µl</TD></TR><TR><TH scope="row">Spe1</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Master-Mix</TH><TD>-</TD><TD>3</TD></TR><TR><TH scope="row">Cutsmart (10x)</TH><TD>6</TD><TD>-</TD></TR><TR><TH scope="row">H2O</TH><TD>5</TD><TD>-</TD></TR><TR><TH scope="row">Total Volume</TH><TD>12</TD><TD>15</TD></TR><TR><TH scope="row">Loading Dye 6x</TH><TD>-</TD><TD>2,5/attempt</TD></TR></TBODY></TABLE><P>Incubation for 60min at 37°CHeat shock for 2min at 80°C</P></DIV><DIV classs="results"><H3>Results:</H3><P>The digestion of the PCR amplified construct shows 3 bands. The 2,2kb band is the uncut construct whereby the smaller bands are the two fragments as a result of the <I>Spe</I>I cut. The constructs in the pET24d still contain the right <I>Spe</I>I restriction site.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.28">15.28 Transformation of <EM>E.coli</EM> DH5α with pET24-fla-Construct from 05.05.14</A></LEGEND><DIV class="exp-content"><P>Colonies were grown on every plate → successful transformation</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.29">15.29 Inoculation for Miniprep of Transformed <EM>E.Coli</EM> DH5α with pet24-fla-Construct from 10.05.14</A></LEGEND><DIV class="aim"><P>Aim: Amplification of transformed plasmid for miniprep in order to receive more plasmid for further experiments</P></DIV><DIV class="exp-content"><P>3x6ml LB + 6µl canamycin each were inoculated with 1 colony of plate 1-3 from 10.05.14. The plates were incubated over night at 37°C</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="12.05.2014">12.05.2014</A></H2><FIELDSET class="exp15"><LEGEND><A name="exp15.30">15.30 Miniprep of Transformed <EM>E.coli</EM> DH5α  with pET24-fla-Construct from 11.05.14</A></LEGEND><DIV classs="aim"><P>Aim: Isolation of plasmid in order to obtain more plasmid for further experiments</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.31">15.31 PCR of Construct with pET24d as Template from Miniprep 11.05.14</A></LEGEND><DIV class="aim"><P>Aim: Amplification of the plasmid from clone 2 for further experiments</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix [µl]</TH><TH scope="col"><STRONG>Pet24fla2</STRONG></TH></TR><TR><TH scope="row">Temp. - pet24fla clone 2 (lig2 11.05.14)</TH><TD>0,5</TD></TR><TR><TH scope="row">Primer  iGEM-89flo</TH><TD>1</TD></TR><TR><TH scope="row">Primer  iGEM-89flo</TH><TD>1</TD></TR><TR><TH scope="row">Q5-Mastermix</TH><TD>25</TD></TR><TR><TH scope="row">Water</TH><TD>22,5</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD></TR></TBODY></TABLE><P>The PCR was performed according to the &quot;PCRQ5&quot; protocol.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>95</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>95</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1 min</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>35x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>Infinite</TD></TR></TBODY></TABLE><P>The PCR-products were separated on a 1 % agarose gel. The bands of 2.2 kb were cut out and a gel extraction was carried out (c = 100 ng/µl) according to the &quot;Gel Extraction (QIAquick Gel Extraction Kit)&quot; protocol.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.32">15.32 Restriction of the PCR-Fragment with <I>Bam</I>HI and <I>Nco</I>I</A></LEGEND><DIV class="aim"><P>Aim: Creating corresponding ends for ligation into pMAD</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Restriction</TH><TH scope="col">Volume [µl]</TH></TR><TR><TH scope="row">PCR-product  hag/flank (100 ng/µl)</TH><TD>5</TD></TR><TR><TH scope="row"><I>Nco</I>I-HF</TH><TD>1</TD></TR><TR><TH scope="row"><I>Bam</I>HI-HF</TH><TD>1</TD></TR><TR><TH scope="row">CutSmart-Buffer (10x)</TH><TD>2</TD></TR><TR><TH scope="row">H2O</TH><TD>11</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD></TR></TBODY></TABLE><P>The restriction is incubated at RT overnight.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.44">13.44 Phosphorylation and Annealing of Promoter-Oligos</A></LEGEND><DIV class="aim"><P>Aim: phosphorylation of single stranded oligos with subsequent annealing for cloning into pMa12</P></DIV><DIV class="exp-content"><P>For ligation to occur at least one of the DNA ends (insert or vector) should contain a 5' phosphate. Primers are usually supplied non-phosphorylated; therefore, the oligos or PCR product will not contain a 5' phosphate. The digestion of DNA with a restriction enzyme will always produce a 5´ phosphate. A DNA fragment can be phosphorylated by incubation with T4 Polynucleotide Kinase in T4-ligase-buffer which already contains the necessary ATP in an appropriate amount (1 mM):</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col"><STRONG>Phosphorylation [µl]</STRONG></TH><TH scope="col"><STRONG>P(argJ)</STRONG></TH><TH scope="col"><STRONG>P(ykuO)</STRONG></TH></TR><TR><TH scope="row">iGEM-012</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">iGEM-013</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">iGEM-014</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">iGEM-015</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">T4-ligase  buffer</TH><TD>5</TD><TD>5</TD></TR><TR><TH scope="row">T4 PNK</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">H2O</TH><TD>42</TD><TD>42</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD><TD>50</TD></TR></TBODY></TABLE><P>The reaction was done in a PCR cycler with a subsequent heat inactivation of the PNK and annealing reaction (protocol: &quot;Annealing with PNK Inactivation&quot;)</P></DIV></FIELDSET></DIV><FIELDSET class="exp14"><LEGEND><A name="exp14.24">14.24 Transformation of BL21 cells</A></LEGEND><DIV class="aim"><P>Aim: Transform E. coli BL21(DE3) with PheA-Arc1p-C-2x-pET28a</P></DIV><DIV class="exp-content"><P>40 µL E. coli BL21(DE3) electrocompetent cells were transformed with 1 ng of the previously purified plasmid PheA-Arc1p-C-2x-pET28a from two different clones, incubated for 1 h at 37 °C and plated out on LB-Kan<SUP>50</SUP> plates that were incubated overnight at 37 °C.</P></DIV></FIELDSET><DIV class="notebooky-entry"><H2 class="title"><A name="13.05.2014">13.05.2014</A></H2><FIELDSET class="exp14"><LEGEND><A name="exp14.25">14.25 Overnight culture</A></LEGEND><DIV class="aim"><P>Aim: Inoculate Overnight culture of PheA-Arc1p-C-2x-pET28a</P></DIV><DIV class="exp-content"><P>5 mL of LB-Kan<SUP>50</SUP> medium were inoculated with a clone from 14.24 bearing the PheA-Arc1p-C-2x-pET28a plasmid.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.33">15.33 Miniprep and Restriction of pMAD with <I>Bam</I>HI and <I>Nco</I>I</A></LEGEND><DIV class="aim"><P>Aim: isolate pMAD and create linearized vector with corresponding ends for ligation with the construct</P></DIV><DIV class="exp-content"><P>The plasmid pMAD was isolated and digested with the restriction enzymes <I>Bam</I>HI and <I>Nco</I>I to create fitting ends for the ligation with the digested PCR-fragment.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Restriction</TH><TH scope="col">Volume [µl]</TH></TR><TR><TH scope="row"><P>pMAD  (174.9 ng/µl)</P></TH><TD>2</TD></TR><TR><TH scope="row"><I>Nco</I>I-HF</TH><TD>1</TD></TR><TR><TH scope="row"><I>Bam</I>HI-HF</TH><TD>1</TD></TR><TR><TH scope="row">CutSmart-Buffer (10x)</TH><TD>2</TD></TR><TR><TH scope="row">H2O</TH><TD>14</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD></TR></TBODY></TABLE><P>The restriction was incubated at 37°C for 1h.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.34">15.34 Ligation of digested pMAD with the digested hag/flank-Construct and trafo</A></LEGEND><DIV class="aim"><P>Aim: link the fragment with the vector and trafo of DH5α</P></DIV><DIV class="exp-content"><P>The ligation of pMAD and the digested PCR-fragment was done according to the previous mentioned ligation calculator.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Ligation</TH><TH scope="col">Volume [µl]</TH></TR><TR><TH scope="row">pMAD</TH><TD>5</TD></TR><TR><TH scope="row">Hag/flank-construct</TH><TD>10</TD></TR><TR><TH scope="row">T4 Ligase Buffer (10x)</TH><TD>2</TD></TR><TR><TH scope="row">T4 DNA-Ligase</TH><TD>1</TD></TR><TR><TH scope="row">H2O</TH><TD>2</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD></TR></TBODY></TABLE><P>The ligation was incubated at RT for 1.5 h. After that <EM>E. coli</EM> DH5α were transformed with the ligation mix and plated out onto LB-Amp plates which were incubated at 37°C overnight.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.45">13.45 Ligation of Annealed Oligos with digested pMa12-Construct and Trafo of <EM>E. coli</EM> DH5Alpha</A></LEGEND><DIV class="aim"><P>Aim: clone the promoter sequences in front of the gfp</P></DIV><DIV class="exp-content"><P>A ligation of the <I>Sac</I>I and <I>Nco</I>I digested pMa12-construct with the annealed oligos was carried out. The annealed oligos (iGEM-012/013 c = 762.3 ng/µl and iGEM-014/015 c = 825.5 ng/µl) were diluted up to 10-2. 10 ng of the insert and 100 ng of the vector were used for the ligation.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Ligation [µl]</TH><TH scope="col"><STRONG>P(argJ)-pMa12-construct</STRONG></TH><TH scope="col"><STRONG>P(ykuO)-pMa12-construct</STRONG></TH></TR><TR><TH scope="row">iGEM-012/013  (7.6 ng/µl)</TH><TD>1,5</TD><TD>-</TD></TR><TR><TH scope="row">iGEM-014/015  (8.3 ng/µl)</TH><TD>-</TD><TD>1,5</TD></TR><TR><TH scope="row">T4-ligase-buffer  (10x)</TH><TD>2</TD><TD>2</TD></TR><TR><TH scope="row">T4-Ligase</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">H2O</TH><TD>15,5</TD><TD>15,5</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><P>The ligation was incubated at RT for 2h. The <EM>E. coli</EM> DH5Alpha were transformed afterwards with the whole ligation mix and plated on LB-Amp, which were incubated at 37°C overnight.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="14.05.2014">14.05.2014</A></H2><FIELDSET class="exp14"><LEGEND><A name="exp14.26">14.26 Test Expression</A></LEGEND><DIV class="aim"><P>Aim: Test the expression of PheA-Arc1p-C-2x</P></DIV><DIV class="exp-content"><P>Using the preculture from 14.25 60 mL of LB-Kan<SUP>50</SUP> medium were inoculated with 600 µL of the preculture and incubated at 37 °C and 220 rpm until OD<SUB>600</SUB> reached 0.5. The culture was cooled down to 18 °C and the protein production was induced with 12 µL of 500 mM IPTG. Over night the culture was incubated at 18 °C and 220 rpm. A glycerolstock of the strain containing the PheA-Arc1p-C-2x-pET28a plasmid was stored at -80 °C.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.35">15.35 Inoculation Miniprep and Restriction of pMAD with <I>Bam</I>HI and <I>Nco</I>I</A></LEGEND><DIV class="aim"><P>Aim: the ligation from 15.34 was not successful. Inoculation of pMAD for miniprep in order to have more plasmid for further experiments. 
    Additionally we create linearized vector with corresponding ends for ligation with the construct with rest from 13.05.14.</P></DIV><DIV class="exp-content"><P>The plasmid pMAD from 13.05.14 (ca. 25µl) and digested with the restriction enzymes <I>Bam</I>HI and <I>Nco</I>I to create fitting ends for the ligation with the digested PCR-fragment. The digest was performed in 2 Eppis with 12,5µl plasmid each.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Restriction [µl]</TH><TH scope="col">Mastermix</TH><TH scope="col">Volume</TH></TR><TR><TH scope="row">pMAD  (174.9 ng/µl)</TH><TD>-</TD><TD>12,5</TD></TR><TR><TH scope="row">Mastermix</TH><TD>-</TD><TD>7,5</TD></TR><TR><TH scope="row">NcoI-HF</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">BamHI-HF</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">CutSmart-Buffer (10x)</TH><TD>4</TD><TD>-</TD></TR><TR><TH scope="row">H2O</TH><TD>9</TD><TD>-</TD></TR><TR><TH scope="row">Total Volume</TH><TD>15</TD><TD>20</TD></TR></TBODY></TABLE><P>The restriction was incubated at 37°C for 2h.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.36">15.36 Transformation of pET24d-clone2 into <EM>E.Coli</EM> DH5α</A></LEGEND><DIV class="aim"><P>Aim: raising colonies for further minipreps of successful construct containing clone</P></DIV><DIV class="exp-content"><P>The plasmid pET24d with the fla construct from clone 2 was transformed into <EM>E.Coli</EM> DH5α and plated on LB-canamycin. Incubation overnight at 37°C.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.37a">13.37 Inoculation for Miniprep of <EM>E.Coli</EM> DH5α with pET24d-fla Construct Clone2</A></LEGEND><DIV class="aim"><P>Aim: raising colonies for further minipreps of successful construct containing clone2</P></DIV><DIV class="exp-content"><P>One clone with plasmid pET24d with the fla construct was used for inoculation of LB-medium. Incubation overnight at 37°C</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.46">13.46 Transformation of Nose-Plasmid from Clone 10 into <EM>E.Coli</EM> DH5α</A></LEGEND><DIV class="aim"><P>Aim: raising colonies for miniprep of new Nose-plasmid for further experiments</P></DIV><DIV class="exp-content"><P>The plasmid pMA12+Nose-Construct from clone 10 was transformed into <EM>E.Coli</EM> DH5α and plated on LB-AMP. Incubation overnight at 37°C</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.37b">13.37 Ligation of digested pMAD from 14.05.14 with the digested hag/flank-construct</A></LEGEND><DIV class="aim"><P>Aim: link the fragment with the vector</P></DIV><DIV class="exp-content"><P>The ligation of pMAD and the digested PCR-fragment was done according to the previous ligation calculator.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Ligation</TH><TH scope="col">Volume [µl]</TH></TR><TR><TH scope="row">pMAD</TH><TD>5</TD></TR><TR><TH scope="row">Hag/flank-construct</TH><TD>20</TD></TR><TR><TH scope="row">T4-Ligase-Buffer (10x)</TH><TD>3</TD></TR><TR><TH scope="row">T4-DNA-Ligase</TH><TD>1</TD></TR><TR><TH scope="row">H2O</TH><TD>1</TD></TR><TR><TH scope="row">Total Volume</TH><TD>30</TD></TR></TBODY></TABLE><P>Incubation overnight at room temperature.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.47">13.47 Restriction and purification of piGEM002 (Nose) and following ligation and transformation</A></LEGEND><DIV class=""><P>Aim: create more plasmid for further experiments, 2 digests were performed to get a high amount of plasmid (pooled during purification)
concentration= 44 ng/µl</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Restriction</TH><TH scope="col">Volume [µl]</TH></TR><TR><TH scope="row">pMa12 from clone 10</TH><TD>1,5</TD></TR><TR><TH scope="row"><I>Nco</I>I</TH><TD>1</TD></TR><TR><TH scope="row"><I>Sac</I>I-HF</TH><TD>1</TD></TR><TR><TH scope="row">CutSmart-Buffer (10x)</TH><TD>2</TD></TR><TR><TH scope="row">H2O</TH><TD>14,5</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD></TR></TBODY></TABLE><P>One ligation was performed for 1h and transformed into chemocompetent cells, the second ligation was performed over night in case no colonies grow on the trafo plates.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Ligation Mix</TH><TH scope="col">Volume [µl]</TH></TR><TR><TH scope="row">Buffer</TH><TD>2</TD></TR><TR><TH scope="row">Water</TH><TD>14,5</TD></TR><TR><TH scope="row">Plasmid</TH><TD>1,5</TD></TR><TR><TH scope="row">Primer Mix (1:100)</TH><TD>1</TD></TR><TR><TH scope="row">Ligase</TH><TD>1</TD></TR></TBODY></TABLE></DIV><P>Transformation:
    thaw on ice: 5 min
    add DNA, keep on ice: 5 min
    heat shock in water bath at 37°C: 45 sec
    put back on ice and add 1 ml LB
    recovery for LB: 20 min</P></FIELDSET><FIELDSET class="exp17"><LEGEND><A name="exp17.1">17.1 Inoculate Bacillus Strains</A></LEGEND><DIV class="aim"><P>Aim: have bacillus strains to work with in MTA and make competent cells</P></DIV><DIV class="exp-content"><P>Wildtype on LB plate and in LB liquid without an antibiotic.<I>Bacillus subtilis</I> strain 186 on plates containing xylose (Incubation overnight).</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="15.05.2014">15.05.2014</A></H2><FIELDSET class="exp17"><LEGEND><A name="exp17.2">17.2 Make Competent Cells</A></LEGEND><DIV class="aim"><P>Aim: making <I>Bacillus subtilis</I> cells competent for transformation</P></DIV><DIV class="exp-content"><P>Inoculate Bacillus wildtype strain in SPC medium.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Time</TH><TH scope="col">OD570</TH></TR><TR><TH scope="row">13:30</TH><TD>5,7</TD></TR><TR><TH scope="row">14:00</TH><TD>6,2</TD></TR><TR><TH scope="row">15:45</TH><TD>6,6</TD></TR><TR><TH scope="row">16:30</TH><TD>7</TD></TR><TR><TH scope="row">17:00</TH><TD>7,2</TD></TR><TR><TH scope="row"> </TH><TD>→ new mediaIncubation for 1,5h</TD></TR></TBODY></TABLE><P>Aliquot and thaw in liquid nitrogen. Store at -80°C.
    </P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.27">14.27 Purification of test expression</A></LEGEND><DIV class="aim"><P>Aim: Purify PheA-Arc1p-C-2x on a small scale</P></DIV><DIV class="exp-content"><P>The cell culture from 14.26 was centrifuged (17000 rpm, 20 min, 4 °C) and the pellet resuspended in 3 mL buffer A. Lysozyme was added to a final concentration of 1 mg/mL and incubated on ice for 30 min. Afterwards the cells were lysed using ultra sound (2x 18 pulses, 30 s on ice inbetween). To clear the suspension of cell debris it was centrifuged (13000 rpm, 15 min, 4 °C). The supernatant was purified using QIAGEN Ni-NTA Spin Columns. An SDS-PAGE gel showed that the protein is produced and can be purified using Ni affinity chromatography.</P><P>60 mL LB-Kan<SUP>50</SUP> medium was inoculated from the glycerolstock prepared in 14.26 and incubated at 37 °C and 220 rpm over night.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.38">15.38 Transformation of ligated pMAD and the digested hag/flank-Construct</A></LEGEND><DIV class="aim"><P>Aim: produce cells that contain the ligated plasmid</P></DIV><DIV class="exp-content"><P>Chemically competent cells from Daniel were transformed with the 20µl ligation-mix, 20µl of the cells were transformed with 1µl pMad uncut (control).Protocol: see yesterday</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.39">15.39 PCR for amplification of Hag-Flank-construct in pET24d and following restriction</A></LEGEND><DIV class="aim"><P>After plasmid isolation a PCR was run to amplify the hag-Flank-construct, afterwards it will be digested with <I>Spe</I>I to find a clone with pure product (no tag but only <I>Spe</I>I restriction site).</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">PCR Mix</TH><TH scope="col">Volume [µl]</TH></TR><TR><TH scope="row">Q5-Mastermix</TH><TD>88</TD></TR><TR><TH scope="row">Water</TH><TD>110</TD></TR><TR><TH scope="row">Primer 89</TH><TD>5,5</TD></TR><TR><TH scope="row">Primer 90</TH><TD>5,5</TD></TR><TR><TH scope="row">Template</TH><TD>1</TD></TR></TBODY></TABLE></DIV><DIV class="results"><P>Gel foto after amplification of hag flank insert from 10 clones: all positive.
Whole insert has been digested for 1h with SpeI in a 20µl reaction mix
17.5µl insert, 2µl buffer and 1µl ligase</P><P>Gel foto after restriction of the 2.2 kb fragment with <I>Spe</I>I (expected sizes appear on the gel) no undigested bands= only fragments with the restriction site are in</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.48">13.48 Colony PCR after transformation</A></LEGEND><DIV class="aim"><P>Aim: find out which colonies contain the right plasmid</P></DIV><DIV class="exp-content"><P>Pick a colony from the trafo plate, streak it on a new plate and dip in the PCR Premix</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">[µl]</TH><TH scope="col">Amount Ag</TH><TH scope="col">Amount Cu</TH></TR><TR><TH scope="row">Q5-Mastermix</TH><TD>8</TD><TD>8</TD></TR><TR><TH scope="row">Water</TH><TD>10</TD><TD>10</TD></TR><TR><TH scope="row">TW 260</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Primer 12</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer 14</TH><TD>-</TD><TD>1</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp17"><LEGEND><A name="exp17.3">17.3 First Plate Reader Measurement with <I>Bacillus subtilis</I> wildtype and strain 186 containing a <I>gfp</I></A></LEGEND><DIV class="aim"><P>Aim: Test for differences between the two strains and first check if experiment works at all</P></DIV><P>Both strains grew best in LB medium like expected, a clearly better growth was shown by the wildtype in the minimal medium compared to the <I>gfp</I> bearing strain.</P><P>The fluorescence is shown in dependence of the culture growth in order to relativise the fluorescence signal. Like expected, the wt exhibited no fluorescence in both media. The fluorescence of the <I>gfp</I> strain was higher in the minimal medium with xylose, than without xylose.</P></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="16.05.2014">16.05.2014</A></H2><FIELDSET class="exp15"><LEGEND><A name="exp15.40">15.40 Colony PCR with colonies from 16.05.14</A></LEGEND><DIV class="aim"><P>Aim: test the success of ligation from pMAD &amp; Hag-Flank-Construct</P></DIV><DIV class="exp-content"><P>A PCR ran with 5 picked clones from 16.05.14 and primers Flo89 and 90 for amplification of inserted construct</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col"> </TH><TH scope="col">Attempt [µl]</TH></TR><TR><TH scope="row">Colony</TH><TD>1 colony</TD></TR><TR><TH scope="row">Q5-Mastermix</TH><TD>10</TD></TR><TR><TH scope="row">Water</TH><TD>8</TD></TR><TR><TH scope="row">Primer Flo89</TH><TD>1</TD></TR><TR><TH scope="row">Primer Flo90</TH><TD>1</TD></TR></TBODY></TABLE></DIV><DIV class="results"><H3>Results</H3><P>Test if insert in pMad has the right size of about 2000 bp → not precise repeat on Monday</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.41">15.41 Inoculation of colonies <I>E. coli</I> pMad plus hag-flank-Insert</A></LEGEND><DIV class="aim"><P>Aim: grow cells that contain the mentioned plasmid and subsequently isolation of the plasmid for further work</P></DIV><DIV class="exp-content"><P>Inoculate 5 clones in LB liquid and incubation over the weekend at room temperature</P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.28">14.28 Expression of PheA-Arc1p-C-2x</A></LEGEND><DIV class="aim"><P>Aim: Express PheA-Arc1p-C-2x for further use</P></DIV><DIV class="exp-content"><P>10 x 500 mL LB-Kan<SUP>50</SUP> medium were inoculated with 5 mL of the overnight culture prepared in 14.27 and incubated at 37 °C and 220 rpm until OD<SUB>600</SUB> reached 0.5. The flasks were cooled to 18 °C and protein production was induced by adding 100 µL 500 mM IPTG. The cultures were incubated over night at 18 °C and 220 rpm. Afterwards the cells were collected by centrifugation (6000 rpm, 20 min, 4 °C), resuspended in buffer A and stored at -20 °C until further use.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="19.05.2014">19.05.2014</A></H2><FIELDSET class="exp15"><LEGEND><A name="exp15.42">15.42 Test restriction of pMad-hag-flank and Test PCR</A></LEGEND><DIV class="aim"><P>Aim: determine whether the plasmid contains the right insert</P></DIV><DIV class="exp-content"><P>Restriction: concentration of plasmid after isolation between 20 and 30 ng/µl</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col"> </TH><TH scope="col">Volume [µl]</TH></TR><TR><TH scope="row">Buffer / Water</TH><TD>27</TD></TR><TR><TH scope="row">Template</TH><TD>10</TD></TR><TR><TH scope="row"><I>Nco</I>I</TH><TD>0,5</TD></TR><TR><TH scope="row"><I>Bam</I>HI</TH><TD>0,5</TD></TR></TBODY></TABLE></DIV><DIV class="results"><H3>Results</H3><P>Test restriction of pMad with hag flank insert did not show a clear result (last band is a different sample)</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col"> </TH><TH scope="col">Amount [µl]</TH></TR><TR><TH scope="row">Q5-Mastermix</TH><TD>10</TD></TR><TR><TH scope="row">Primer 89</TH><TD>1</TD></TR><TR><TH scope="row">Primer 90</TH><TD>1</TD></TR><TR><TH scope="row">Template</TH><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>7</TD></TR></TBODY></TABLE></DIV><DIV class="results"><H3>Results</H3><P>The PCR showed the expected fragment of the hag-flank construct with a size of approx. 2 kb.</P></DIV></FIELDSET><FIELDSET class="exp17"><LEGEND><A name="exp17.4">17.4 Bacillus Trafo with Nose Plasmids</A></LEGEND><DIV class="exp-content"><P>5µl DNA were added to 100µl cells, 30 min incubation at 37°C and plated on LB-Cm.</P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.29">14.29 Purification of PheA-Arc1p-C-2x</A></LEGEND><DIV class="aim"><P>Aim: Purify PheA-Arc1p-C-2x in three steps</P></DIV><DIV class="exp-content"><P>The resuspended cells from 14.28 were lysed using a french press. The lysate was incubated on ice with 5 µg/mL DNAseA and RNAseI for 10 min and centrifuged (27000 rpm, 45 min, 4 °C) to pellet cell debris. The supernatant was filtered (0.45 µm) and PheA-Arc1p-C-2x purified from the cleared solution by NiNTA affinity chromatography. The combined elution fractions containing crude PheA-Arc1p-C-2x were concentrated to a volume of 2 mL and further purified using an anion exchange column to get rid of possible tRNA contaminations. Finally the buffer was changed to dialysis buffer and the protein concentrated and stored at -80 °C in aliquods for further use.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="20.05.2014">20.05.2014</A></H2><FIELDSET class="exp14"><LEGEND><A name="exp14.30">14.30 Creating PheA-Arc1p-C-1x</A></LEGEND><DIV class="aim"><P>Aim: Using Round-the-horn mutagenesis to create PheA-Arc1p-C-1x-pET28a</P></DIV><DIV class="exp-content">
    The PCR with 5' phosphorylated TG_1xGSSG FP und TG_0xGSSG RP should generate the 7509 bp fragment for ligation from the methylated template PheA-Arc1p-C-8x-pET28a from 14.5.<P>The used Primer concentration was 1 
    µM, the concentration of the template was 1 ng/µL and we used a hot start to avoid unspecific annealing of primers.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Volume [µL]</TH></TR><TR><TH scope="row">Water</TH><TD>29.5</TD></TR><TR><TH scope="row">5x GC Buffer</TH><TD>10</TD></TR><TR><TH scope="row">DMSO</TH><TD>2.5</TD></TR><TR><TH scope="row">TG_1xGSSG FP</TH><TD>2.5</TD></TR><TR><TH scope="row">TG_0xGSSG RP</TH><TD>2.5</TD></TR><TR><TH scope="row">PheA-Arc1p-C-8x-pET28a</TH><TD>1</TD></TR><TR><TH scope="row">dNTPs</TH><TD>1</TD></TR><TR><TH scope="row">Phusion-Polymerase</TH><TD>1</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature [°C]</TH><TH scope="col">Time [min:sec]</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>2:00</TD></TR><TR><TH scope="row">2</TH><TD>add Polymerase</TD><TD> </TD></TR><TR><TH scope="row">3</TH><TD>98</TD><TD>0:10</TD></TR><TR><TH scope="row">4</TH><TD>67.5</TD><TD>0:30</TD></TR><TR><TH scope="row">5</TH><TD>72</TD><TD>4:00</TD></TR><TR><TH scope="row">6</TH><TD>go to 3</TD><TD>32x</TD></TR><TR><TH scope="row">7</TH><TD>72</TD><TD>10:00</TD></TR><TR><TH scope="row">8</TH><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P>The amplified fragment was purified on an agarose gel on which it showed the correct size. The extracted linear template was then digested to degrade the original unmutated plasmid using DpnI.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Volume [µL]</TH></TR><TR><TH scope="row">10x CutSmart Buffer</TH><TD>3.6</TD></TR><TR><TH scope="row">Plasmid</TH><TD>30</TD></TR><TR><TH scope="row">DpnI</TH><TD>3</TD></TR><TR><TH scope="row">Total Volume</TH><TD>36.6</TD></TR></TBODY></TABLE><P>The reaction mixture was incubated at 37 °C and 300 rpm for 1 h and afterwards heated to 80 °C for 20 min.</P><P>In order to create the circular plasmid 100 ng of the linear template were mixed with 1 µL 10x T4-Ligasebuffer,  1 µL T4-Ligase and water to give a total volume of 10 µL. This mixture was incubated for 9 h at 16 °C. The resulting plasmid was dialysed with water to decrease the salt concentration for the following transformation of E. coli Top10 electrocompetent cells with the plasmid.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.43">15.43 PCR to Amplify Cup 1 from Yeast gDNA</A></LEGEND><DIV class="aim"><P>Aim: create the DNA fragment <I>cup</I>1 (metallothionein) for insertion into the flagellin (Gibson assembly with pMad by cutting the plasmid with <I>Spe</I>I at the created restriction site)</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col"> </TH><TH scope="col">Amount [µl]</TH></TR><TR><TH scope="row">Q5-Mix</TH><TD>20</TD></TR><TR><TH scope="row">Primer 89</TH><TD>1</TD></TR><TR><TH scope="row">Primer 90</TH><TD>1</TD></TR><TR><TH scope="row">Template</TH><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>17</TD></TR></TBODY></TABLE></DIV><DIV class="results"><P>The Purple 2-log ladder was used as a marker. The amplified cup1 could be seen as a 204 bp fragment on the gel.</P></DIV></FIELDSET><FIELDSET class="exp17"><LEGEND><A name="exp17.5">17.5 Bacillus Trafo with Nose Plasmids</A></LEGEND><DIV class="exp-content"><P>No colonies → new trafoNew LB-chloramphenicol plates for Bacillus (only 5µg/ml instead of 25µg/ml)</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="21.05.2014">21.05.2014</A></H2><FIELDSET class="exp15"><LEGEND><A name="exp15.44">15.44 Restriction pMad with <I>Spe</I>I</A></LEGEND><DIV class="aim"><P>Aim: Create a linearized plasmid for following Gibson assembly with <I>cup</I>1</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col"> </TH><TH scope="col">Volume [µl]</TH></TR><TR><TH scope="row">Buffer</TH><TD>2,5</TD></TR><TR><TH scope="row">Plasmid</TH><TD>20</TD></TR><TR><TH scope="row">SpeI</TH><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>-</TD></TR></TBODY></TABLE><P>Concentration of plasmid after purification only 7 ng/µl</P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.31">14.31 Overnight culture</A></LEGEND><DIV class="aim"><P>Aim: Amplify Plasmid in an overnight culture</P></DIV><DIV class="exp-content"><P>Six clones of the transformed <I>E. coli</I> Top10 cells from 14.30 were picked and used to inoculate overnight cultures (6 x 5 mL).</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.45">15.45 Gibson Assembly of digested pMad-fla and cup 1 and Transformation into <EM>E.Coli</EM> XL1-Blue</A></LEGEND><DIV class="aim"><P>Aim: create a flagellin that contains the metallothinein cup1</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">[µl]</TH><TH scope="col">Control</TH><TH scope="col">With Insert</TH></TR><TR><TH scope="row">Gibson Mix</TH><TD>15</TD><TD>15</TD></TR><TR><TH scope="row">Plasmid (7ng/µl)</TH><TD>5</TD><TD>5</TD></TR><TR><TH scope="row">Insert</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>1</TD><TD>-</TD></TR></TBODY></TABLE><P>A part of the aliquot was plated on LB-tet for inoculation of colonies for competent XL1-Blue.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="22.05.2014">22.05.2014</A></H2><FIELDSET class="exp17"><LEGEND><A name="exp17.6">17.6 MTA Bacillus Nose Ag </A></LEGEND><DIV class="exp-content"><P>Transformation from 20.5.14: one colony on the Ag Nose plate
    Inoculation of shaking culture in the morning together with the wildtype and the <I>gfp</I> strain in LB liquid without antibiotics</P><P>Inoculate 96 well plate in the afternoon 15:00</P><P>Prepare Cu and Ag solutions to test different concentrations in MTA and see if the Ag nose responds to Ag and also if it is specific for Ag and does not respond to Cu</P><P>Ag and Cu in stock solution of 10 mM have been prepared
    Added (Concentration): 11 mM, 100µM and 10µM
    Cells: Ag Nose mutant, wildtype (AG GM) and <I>gfp</I></P><P>The MTA showed no significant increase of fluorescence. The bacteria had difficulties in growth, thus it seemed that the metal ion concentrations were too high.</P></DIV></FIELDSET><FIELDSET class="exp17"><LEGEND><A name="exp17.7a">17.7 Digestion of piGEM002 and the nose plasmids for Ag/Cu</A></LEGEND><DIV class="aim"><P>Aim: Linearize the plasmids with <I>Spe</I>I to built in the instability tags (25µl mix)</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">[µl]</TH><TH scope="col"><STRONG>piGem 002 clone 7</STRONG></TH><TH scope="col"><STRONG>Nose plasmid 003/004</STRONG></TH><TH scope="col">Amount</TH></TR><TR><TH scope="row"><STRONG>Template ~10µg</STRONG></TH><TD>15</TD><TD>20</TD><TD>15</TD></TR><TR><TH scope="row">SpeI</TH><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">CutSmart-Buffer</TH><TD>2,5</TD><TD>2,5</TD><TD>2,5</TD></TR><TR><TH scope="row">Water</TH><TD>6,5</TD><TD>1,5</TD><TD>6,5</TD></TR></TBODY></TABLE><P>Incubation at room temperature over night.</P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.32">14.32 Preparation of plasmids</A></LEGEND><DIV class="aim"><P>Aim: Purify the amplified plasmids from overnight cultures</P></DIV><DIV class="exp-content"><P>The plasmids were purified from the overnight cultures created in 14.31 using Sigma GenElute Plasmid Miniprep Kit following the included protocol. The plasmids were eluted using 45 µL water.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="23.05.2014">23.05.2014</A></H2><FIELDSET class="exp17"><LEGEND><A name="exp17.7b">17.7 Create Instability Tags for gfp in the Nose Plasmids</A></LEGEND><DIV class="aim"><P>Aim: GFP is a stable protein, which would accumulate and distort the results if not degraded, instability tags of different strength cause the degradation and therefore cause quantitative results</P></DIV><DIV class="exp-content"><P>First step: Linearize the following plasmids: piGEM 002 without promoter as well as the plasmids containing the silver and copper sensitive promoters
		Second step: PCR to anneal the oligos 20+21, 20+22 and 20+23</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col"> </TH><TH scope="col">Volume [µl]</TH></TR><TR><TH scope="row">Primer A</TH><TD>5</TD></TR><TR><TH scope="row">Primer B</TH><TD>5</TD></TR><TR><TH scope="row">Phusion</TH><TD>0,25</TD></TR><TR><TH scope="row">dNTPs</TH><TD>0,25</TD></TR><TR><TH scope="row">Buffer</TH><TD>5</TD></TR><TR><TH scope="row">Water</TH><TD>34,5</TD></TR><TR><TH scope="row">Cycle</TH><TD>3</TD></TR></TBODY></TABLE><P>Afterwards clean up with kit (all three mixes were carried out in a duplicate, 1 each was frozen away at minus 20)</P><P>Third step: Yeast transformation
	9 Reactions (3 plasmids and 3 instability (ssrA-)tags)</P>
	100 ng plasmid plus 1µl of primer mix were filled to 14µl and added to the yeast transformation mix and subsequently transformed into yeast.</DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="26.05.2014">26.05.2014</A></H2><FIELDSET class="exp15"><LEGEND><A name="exp15.46">15.46 Restriction of pMAD-fla with <I>Spe</I>I, Gibson-Assembly with <I>cup</I>1-1 and transformation of <EM>E. coli</EM> DH5α</A></LEGEND><DIV class="aim"><P>Aim: insert cup1-1 into the hag/flank-construct</P></DIV><DIV class="exp-content"><P>To insert the metallothionein gene <I>cup</I>1-1 into the hag/flank-construct the pMAD-fla was digested with <I>Spe</I>I to linearize it.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Restriction</TH><TH scope="col">Volume [µl]</TH></TR><TR><TH scope="row">pMAD-fla  (330 ng/µl)</TH><TD>4</TD></TR><TR><TH scope="row">SpeI</TH><TD>1</TD></TR><TR><TH scope="row">CutSmart Buffer (10x)</TH><TD>1</TD></TR><TR><TH scope="row">H2O</TH><TD>6</TD></TR><TR><TH scope="row">Total Volume</TH><TD>10</TD></TR></TBODY></TABLE><P>The restriction was carried out at 37°C for 1h.
    The restriction was cleaned with a DNA Gel Ex Kit (c = 40 ng/µl). 4µl (160 ng) of this digested plasmid were mixed with 1 µl of the cup1-1 fragment (9.6 ng/µl). These 5 µl were inserted into the prepared 15 µl Gibson-mix. The reaction was incubated at RT for 1 min and then at 50°C for 1 hour. An extra Gibson-reaction was carried out as a control with just the linearized pMAD without the <I>cup</I>1-1.<EM>E. coli</EM> DH5α cells were transformed with the Gibson reactions and plated out on LB-Amp. They were incubated at 37°C over night.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.47">15.47 Colony-PCR of <EM>E. coli</EM> XLI-Blue pMAD-fla + <I>cup</I>1-1 from 21.05.14 and 23.05.14</A></LEGEND><DIV class="aim"><P>Aim: check for correct insertion of <I>cup</I>1-1 into the hag/flank-construct</P></DIV><DIV class="exp-content"><P>Since a Gibson-Assembly with the same aim was carried out last week, the colonies were checked by a colony PCR with the primers for <I>cup</I>1-1 iGEM-018 and iGEM-019. One colony was picked from the plate from 23rd May and three were taken from the plate from 21st May. The latter was contaminated with Bacillus colonies. For this reason <EM>E. coli</EM> was picked and streaked out onto a new plate on Friday the 23rd May. The consequence is a plate with just one clone, what is the reason, that only one colony was picked from this plate.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix</TH><TH scope="col">Volume [µl]</TH></TR><TR><TH scope="row"><EM>E. coli</EM> XLI-Blue pMAD-fla+cup1-1</TH><TD>part of a colony</TD></TR><TR><TH scope="row">Primer  iGEM-018 </TH><TD>0,5</TD></TR><TR><TH scope="row">Primer  iGEM-019</TH><TD>0,5</TD></TR><TR><TH scope="row">Q5-Mastermix</TH><TD>10</TD></TR><TR><TH scope="row">Water</TH><TD>9</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD></TR></TBODY></TABLE><P>A part of each colony was transferred onto a new LB-Amp plate, which was incubated at 37°C overnight. </P></DIV><DIV class="result"><H3>Result</H3><P>The expected fragment in size of the <I>cup</I>1 could be amplified from the template, which indicates a positive result.</P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.33">14.33 Transformation of BL21 cells</A></LEGEND><DIV class="aim"><P>Aim: Transform E. coli BL21(DE3) with PheA-Arc1p-C-1x-pET28a</P></DIV><DIV class="exp-content"><P>40 µL E. coli BL21(DE3) electrocompetent cells were transformed with 1 ng of the previously purified plasmid PheA-Arc1p-C-1x-pET28a from two different clones, incubated for 1 h at 37 °C and plated out on LB-Kan<SUP>50</SUP> plates that were incubated overnight at 37 °C.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.49">13.49 PlasmidPprep from Yeast and Transformation of <EM>E. coli</EM> DH5Alpha</A></LEGEND><DIV class="aim"><P>Aim: increase the amount of plasmid in E. coli for further experiments with the degradation tags</P></DIV><DIV class="exp-content"><P>The <I>gfp</I> in piGEM002, piGEM002-Ag (and piGEM002-Cu was tagged with different degradation tags by yeast recombination. The plasmids have been isolated from yeast.</P><P><EM>E. coli</EM> DH5α cells were transformed with 15 µl of each isolated plasmid and plated out on LB-Amp. They were incubated at 37°C over night.</P><P>Gly-Stocks - Making of -stocks for long time storage of bacteria strains
    For long time freezer storage of bacteria, they have to be treated with glycerine to prevent the crystallization and damaging of the cells. For this reason 2 ml of LB with the respective antibiotic were inoculated with the bacteria, which were meant to be stored and were incubated at 37°C overnight.</P><P>Strains that are to be stored:
    <UL class="strains"><LI><EM>E. coli </EM>XLI-Blue pMAD-fla  (piGEM-005)</LI><LI><EM>E. coli </EM>XLI-Blue pMa12-construct  (piGEM-002)</LI><LI><EM>E. coli </EM>XLI-Blue pMa12-Cu  (piGEM-004)</LI><LI><EM>E. coli </EM>XLI-Blue pMa12-Ag (piGEM-003)</LI><LI><EM>E. coli </EM>BL21(DE3) pLysS</LI></UL></P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="27.05.2014">27.05.2014</A></H2><FIELDSET class="exp14"><LEGEND><A name="exp14.34">14.34 Overnight culture</A></LEGEND><DIV class="aim"><P>Aim: Inoculate Overnight culture of PheA-Arc1p-C-1x-pET28a</P></DIV><DIV class="exp-content"><P>5 mL of LB-Kan<SUP>50</SUP> medium were inoculated with a clone from 14.33 bearing the PheA-Arc1p-C-1x-pET28a plasmid.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.48">15.48 Restriction of pET24d-fla with <I>Spe</I>I and Gibson-Assembly with <I>cup</I>1-1</A></LEGEND><DIV class="aim"><P>Aim: linearize pET24d-fla and insert <I>cup</I>1-1 into pET24d-fla</P></DIV><DIV class="exp-content"><P>Since the colony-PCR (26.05.14) was negative and the Gibson-Assembly of linearized pMAD-fla with <I>cup</I>1-1 showed no results the pET24d-fla construct was linearized with <I>Spe</I>I.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Restriction</TH><TH scope="col">Volume [µl]</TH></TR><TR><TH scope="row">pET24d-fla  (98.8 ng/µl)</TH><TD>8</TD></TR><TR><TH scope="row">SpeI</TH><TD>1</TD></TR><TR><TH scope="row">CutSmart-Buffer (10x)</TH><TD>2</TD></TR><TR><TH scope="row">H2O</TH><TD>9</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD></TR></TBODY></TABLE><P>The restriction was incubated at 37°C for 1 h.</P><P>The restriction mix was purified with the Gel Ex Kit. The yield was too low (c = 3.1 ng/µl) to use it for a Gibson-Assembly, so the rest of the pET24d-fla was digested with <I>Spe</I>I.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Restriction</TH><TH scope="col">Volume [µl]</TH></TR><TR><TH scope="row">pET24d-fla  (98.8 ng/µl)</TH><TD>18,5</TD></TR><TR><TH scope="row">SpeI</TH><TD>1</TD></TR><TR><TH scope="row">CutSmart-Buffer (10x)</TH><TD>2,5</TD></TR><TR><TH scope="row">H2O</TH><TD>3</TD></TR><TR><TH scope="row">Total Volume</TH><TD>25</TD></TR></TBODY></TABLE><P>The restriction was incubated at 37°C for 1h.</P><P>The restriction mix was purified with the Gel Ex Kit (c = 20.1 ng/µl), which was enough for the Gibson-Assembly. 4µl (80 ng) of this digested plasmid were mixed with 1 µl of the cup1-1 fragment (9.6 ng/µl). These 5 µl were inserted into the prepared 15µl Gibson-mix. The reaction was incubated at RT for 1 min and then at 50°C for 1 h. An extra Gibson-reaction was carried out as a control with just the linearized pET24d-fla without the <I>cup</I>1-1.</P><P>Competent <EM>E. coli</EM> XLI-Blue were transformed with the whole Gibson-reactions. They were plated on LB-Kan an incubated at 37°C overnight.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.50">13.50 Colony-PCR of the <EM>E. colis</EM> Containing the Degradation tagged Constructs</A></LEGEND><DIV class="aim"><P>Aim: check for correct tagging of <I>gfp</I> with the respective degradation tag</P></DIV><DIV class="exp-content"><P>The transformation with the degradation tag-constructs were successful nearly in all cases except the pMa12-Ag-21. The transformation was repeated, the plate was incubated at 37°C over night. 
Four colonies of every plate have been picked, transferred to new LB-Amp plates and were subsequently used for the colony-PCR. 
The Master Mix was portioned in aliquots of 20 µl per PCR-cup and inoculated with the respective colony.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">MasterMix for 33 reactions</TH><TH scope="col">Volume [µl]</TH></TR><TR><TH scope="row"><EM>E. coli</EM> DH5Alpha with construct</TH><TD>part of a colony</TD></TR><TR><TH scope="row">AG  TW 260 </TH><TD>16</TD></TR><TR><TH scope="row">AG  TW 316 </TH><TD>16</TD></TR><TR><TH scope="row">Q5-Mastermix</TH><TD>165</TD></TR><TR><TH scope="row">Water</TH><TD>133</TD></TR><TR><TH scope="row">Total Volume</TH><TD>330</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>95</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>95</TD><TD>20 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>20 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>20 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>35x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>Infinite</TD></TR></TBODY></TABLE></DIV><DIV class="results"><P>Colony-PCR of E. coli with the degradation-tag constructs</P><P>The bands slightly higher than 200 bp indicated a positive clone. Positive constructs are available: 002-21, Cu-23, Ag-22 and Cu-22.</P></DIV><DIV class="exp-content"><P>Gly-Stocks - Making of glycerin-stocks for long time storage of bacteria strains</P><P>1 ml of the bacteria which were inoculated yesterday (26.05.2014) were transferred to new, sterile 1.5 ml Eppendorf-cups. 600 µl of 50% glycerin were added and the cells were frozen and stored at -80°C in the freezer.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.1">18.1 Making competent <EM>E. coli</EM> XLI-Blue with RbCl: Inoculation of LB-Tet with <EM>E.Coli</EM> XL1-Blue as a preculture</A></LEGEND><DIV class="exp-content"><P>9 ml LB-Tetracycline (1:1000) were inoculated with <EM>E. coli</EM> XLI-Blue cells as a preculture. They were incubated at 37°C overnight.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="28.05.2014">28.05.2014</A></H2><FIELDSET class="exp18"><LEGEND><A name="exp18.2">18.2 Inoculating <EM>E.Coli</EM> XL1-Blue main culture</A></LEGEND><DIV class="exp-content"><P>93 mL of LB-tet were added to 7 mL of preculture for inculation of main culture. The main culture was incubated at 37°C until an OD of 0,5.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.3">18.3 Preparing Transformation Buffer TBF1 and TBF2</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">TBF1 recipe</TH><TH scope="col">TBF2 recipe</TH></TR><TR><TD>for 50ml</TD><TD> </TD></TR><TR><TD> </TD><TD> </TD></TR><TR><TD>potassium acetate</TD><TD>0,15g</TD></TR><TR><TD>MnCl2 x 4H2O</TD><TD> </TD></TR><TR><TD>1M RbCl</TD><TD>5 ml</TD></TR><TR><TD>1m CaCl2</TD><TD>0,5 ml</TD></TR><TR><TD>87% glycerine</TD><TD>8,6 ml</TD></TR><TR><TD>A.dest</TD><TD>35,9 ml</TD></TR><TR><TD>autoclave or sterile filtration</TD><TD> </TD></TR><TR><TD> </TD><TD> </TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TD>for 250 ml</TD><TD> </TD></TR><TR><TD> </TD><TD> </TD></TR><TR><TD>1M NaMOPS (pH7)</TD><TD>2,5 ml</TD></TR><TR><TD>1M CaCl2 x 2H2O</TD><TD>8,75 ml</TD></TR><TR><TD>1M RbCl2</TD><TD>2,5 ml</TD></TR><TR><TD>87% glycerine</TD><TD>43,75 ml</TD></TR><TR><TD>A.dest</TD><TD>192,5 ml</TD></TR><TR><TD>adjust pH7</TD><TD> </TD></TR><TR><TD>store at 4°C in the dark</TD><TD> </TD></TR><TR><TD>sterile filtration</TD><TD> </TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.4">18.4 Preparing <EM>E.Coli</EM> XL1-Blue</A></LEGEND><DIV class="exp-content"><UL class="list"><LI>Grow 100 ml <EM>E.coli</EM> culture to OD600=0,5</LI><LI>Put on  ice for 10 min </LI><LI>Centrifuge  2x 50 ml for 10 min at 3000 rpm and 4°C </LI><LI>Wash  pellets in 10 ml TFB1 (10 min; 3000 rpm; 4°C)</LI><LI>Resuspend  pellets in 2ml TFB2 </LI><LI>Pipett  200µl aliquots in precooled 1,5 ml tubes</LI><LI>store  at  -80°C </LI></UL></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.35">14.35 Test Expression</A></LEGEND><DIV class="aim"><P>Aim: Test the expression of PheA-Arc1p-C-1x</P></DIV><DIV class="exp-content"><P>Using the preculture from 14.34 60 mL of LB-Kan<SUP>50</SUP> medium were inoculated with 600 µL of the preculture and incubated at 37 °C and 220 rpm until OD<SUB>600</SUB> reached 0.5. The culture was cooled down to 18 °C and the protein production was induced with 12 µL of 500 mM IPTG. Over night the culture was incubated at 18 °C and 220 rpm. A glycerolstock of the strain containing the PheA-Arc1p-C-1x-pET28a plasmid was stored at -80 °C.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.49a">15.49 Colony-PCR with transformed <EM>E. coli</EM> XLI-Blue including pET24d-fla-cup1-1</A></LEGEND><DIV class="aim"><P>Aim: testing the success of insertion of <I>cup</I>1-1 into pET24d-fla </P></DIV><DIV class="exp-content"><P>6 clones were picked from transformation plate and a colony PCR was ran with primers iGEM-018 &amp; - 019. The result should be a band at 240 bp in case of a positive clone.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">[µl]</TH><TH scope="col">Mastermix</TH><TH scope="col">Mix for PCR attempts</TH></TR><TR><TH scope="row"><EM>E. coli</EM> XLI-Blue pet24d-fla+cup1-1</TH><TD> </TD><TD>part of a colony</TD></TR><TR><TH scope="row">Primer  iGEM-018 </TH><TD>3,5</TD><TD>-</TD></TR><TR><TH scope="row">Primer  iGEM-019</TH><TD>3,5</TD><TD>-</TD></TR><TR><TH scope="row">Q5-Masterix</TH><TD>70</TD><TD>-</TD></TR><TR><TH scope="row">Water</TH><TD>63</TD><TD>-</TD></TR><TR><TH scope="row">Master-Mix</TH><TD>-</TD><TD>20</TD></TR><TR><TH scope="row">Total Volume</TH><TD>140</TD><TD> </TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>95</TD><TD>1 min</TD></TR><TR><TH scope="row">2</TH><TD>95</TD><TD>20 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>20 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>20 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>35x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>Infinite</TD></TR></TBODY></TABLE><P>The expected band of approx. 240 bp could be seen in every sample, more or less clearly</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.51">13.51 Colony-PCR of the <EM>E. coli</EM> containing the degradation tagged constructs</A></LEGEND><DIV class="aim"><P>Aim: check for correct tagging of <I>gfp</I> with the respective degradation tag</P></DIV><DIV class="exp-content"><P>The Colony-PCR was repeated with pMa12-Ag-21 after successful transformation.
Four colonies have been picked, transferred to new LB-Amp plates and were subsequently used for the colony-PCR.
The Master Mix was portioned in aliquots of 20 µl per PCR-cup and inoculated with the respective colony.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">[µl]</TH><TH scope="col">Master-Mix for attempts</TH><TH scope="col">Mix for PCR attempt</TH></TR><TR><TH scope="row"><EM>E. coli</EM> DH5α with Ag21</TH><TD>-</TD><TD>part of a colony</TD></TR><TR><TH scope="row">AG  TW 260 </TH><TD>2,5</TD><TD>-</TD></TR><TR><TH scope="row">AG  TW 316</TH><TD>2,5</TD><TD>-</TD></TR><TR><TH scope="row">Q5-Mastermix</TH><TD>50</TD><TD>-</TD></TR><TR><TH scope="row">Water</TH><TD>45</TD><TD>-</TD></TR><TR><TH scope="row">Master-Mix</TH><TD>-</TD><TD>20</TD></TR><TR><TH scope="row">Total Volume</TH><TD>100</TD><TD>20</TD></TR></TBODY></TABLE></DIV><DIV class="results"><P>Colony-PCR of <EM>E. coli</EM> containing the degradation-tag construct Ag-21</P></DIV><DIV class="exp-content"><P>In an additional reaction the colony-PCRs of the negative clones (gel 28.05.14) were repeated with more picked clones per construct. This time 8 colonies of each negative construct were picked, transferred to another LB-Amp-plate and used for the PCR-reaction. Four times eight reactions were carried out, a master mix was prepared for 33 reactions.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">[µl]</TH><TH scope="col"><STRONG>Master-Mix for 33  reactions</STRONG></TH><TH scope="col"><STRONG>Mix for one PCR reaction</STRONG></TH></TR><TR><TH scope="row">Part of a colony</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">AG  TW 260 </TH><TD>8</TD><TD>0,25</TD></TR><TR><TH scope="row">AG  TW 316</TH><TD>8</TD><TD>0,25</TD></TR><TR><TH scope="row">Q5-Mastermix</TH><TD>330</TD><TD>10</TD></TR><TR><TH scope="row">Water</TH><TD>314</TD><TD>9,5</TD></TR><TR><TH scope="row">Master-Mix</TH><TD>-</TD><TD>20</TD></TR><TR><TH scope="row">Total Volume</TH><TD>660</TD><TD>20</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>95</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>95</TD><TD>20 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>20 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>20 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>35x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>Infinite</TD></TR></TBODY></TABLE></DIV><DIV class="results"><P><COLONY-PCR with="">E. coli containing the degradation-tag constructs</COLONY-PCR></P><P>Positive clones could be obtained with the Ag-23 and the 002-22 constructs (approx. 250 bp).</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.5a">18.5 Transformation test with <EM>E.Coli</EM> XL1-Blue and pMAD (fla)</A></LEGEND><DIV class="exp-content"><P><EM>E.Coli</EM> XL1-Blue were transformed with 1µl pMAD (189 ng/µl). Another aliquod was transformed with pMAD-fla.
In comparison <EM>E.Coli</EM> DH5α were transformed the same way.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="29.05.2014">29.05.2014</A></H2><FIELDSET class="exp18"><LEGEND><A name="exp18.5b">18.5 Transformation Test with <EM>E.Coli</EM> XL1-Blue and pMAD (fla)</A></LEGEND><DIV class="exp-content"><P>Every transformation was successful. The higher number of colonies on the pMAD-transformation plate shows a higher transformation efficiency compared with DH5α.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.6">18.6 Inoculation for Miniprep and Glycerin stock of pMAD-fla</A></LEGEND><DIV class="aim"><P>Aim: In order to save pMAD and pMAD-fla for further experiments and storage, minipreps and glycerine stocks have to be inoculated</P></DIV><DIV class="exp-content"><P>For minipreps 2 x 6 ml of LB-Amp were inoculated with one colony from the pMAD XL1-Blue-transformation plate. Additionally 3 x 6 ml of LB-Amp were inoculated with one colony from the pMAD-fla XL1-Blue-Transformation plate. 
    Furthermore 2 x 2 mL LB-Amp were inoculated with a clone from pMAD-fla XL1-Blue-Transformation plate for glycerine stocks.
    Every attempt was incubated at 37°C overnight.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.52">13.52 Colony-PCR of the <I>E. coli</I> containing the degradation tagged constructs</A></LEGEND><DIV class="aim"><P>Aim: check for correct tagging of <I>gfp</I> with the respective degradation tag</P></DIV><DIV class="exp-content"><P>The Colony-PCR was repeated with 002-Cu21 and 002-23.
    Four colonies have been picked, transferred to new LB-Amp plates and were subsequently used for the colony-PCR.
    The Master Mix was portioned in aliquots of 20 µl per PCR-cup and inoculated with the respective colony.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">[µl]</TH><TH scope="col"><STRONG>Master-Mix for 18  attempts </STRONG></TH><TH scope="col"><STRONG>Mix for PCR attempt  002-23.1-8</STRONG></TH><TH scope="col"><STRONG>Mix for PCR attempt 002-Cu21.1-8</STRONG></TH></TR><TR><TH scope="row"><EM>E. coli</EM> DH5α with 002-23</TH><TD>-</TD><TD>part of a colony</TD><TD>-</TD></TR><TR><TH scope="row"><EM>E. coli</EM> DH5α with Cu21</TH><TD>-</TD><TD>-</TD><TD>part of a colony</TD></TR><TR><TH scope="row">AG  TW 260 </TH><TD>9</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">AG  TW 316</TH><TD>9</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>3,6</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Q5-Buffer 5x</TH><TD>72</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">dNTP-Mix</TH><TD>5</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Water</TH><TD>261,4</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Master-Mix</TH><TD>-</TD><TD>20</TD><TD>20</TD></TR><TR><TH scope="row">Total Volume</TH><TD>360</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>95</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>95</TD><TD>20 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>20 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>20 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>35x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>Infinite</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.36">14.36 Purification of test expression</A></LEGEND><DIV class="aim"><P>Aim: Purify PheA-Arc1p-C-1x on a small scale</P></DIV><DIV class="exp-content"><P>The cell culture from 14.35 was centrifuged (17000 rpm, 20 min, 4 °C) and the pellet resuspended in 3 mL buffer A. Lysozyme was added to a final concentration of 1 mg/mL and incubated on ice for 30 min. Afterwards the cells were lysed using ultra sound (2x 18 pulses, 30 s on ice inbetween). To clear the suspension of cell debris it was centrifuged (13000 rpm, 15 min, 4 °C). The supernatant was purified using QIAGEN Ni-NTA Spin Columns. An SDS-PAGE gel showed that the protein is produced and can be purified using Ni affinity chromatography.</P><P>60 mL LB-Kan<SUP>50</SUP> medium was inoculated from the glycerolstock prepared in 14.35 and incubated at 37 °C and 220 rpm over night.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.53">15.53 Inoculation of Minipreps from Nose-Tag-Plasmids</A></LEGEND><DIV class="aim"><P>Aim: isolate the plasmid to tansform Bacillus for further plate reader assays</P></DIV><DIV class="exp-content"><P>6 mL LB-Amp were inoculated with different colonies cotaining the following plasmid-constructs:</P><UL class="list"><LI>002-21</LI><LI>002-22</LI><LI>002-Cu-23</LI><LI>002-Ag-23</LI><LI>002-Cu-22</LI><LI>002-Ag-22</LI></UL><P>Incubation over night at 37°C</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="30.05.2014">30.05.2014</A></H2><FIELDSET class="exp14"><LEGEND><A name="exp14.37">14.37 Expression of PheA-Arc1p-C-1x</A></LEGEND><DIV class="aim"><P>Aim: Express PheA-Arc1p-C-1x for further use</P></DIV><DIV class="exp-content"><P>10 x 500 mL LB-Kan<SUP>50</SUP> medium were inoculated with 5 mL of the overnight culture prepared in 14.36 and incubated at 37 °C and 220 rpm until OD<SUB>600</SUB> reached 0.5. The flasks were cooled to 18 °C and protein production was induced by adding 100 µL 500 mM IPTG. The cultures were incubated over night at 18 °C and 220 rpm. Afterwards the cells were collected by centrifugation (6000 rpm, 20 min, 4 °C), resuspended in buffer A and stored at -20 °C until further use.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.7">18.7 Miniprep of <EM>E.Coli</EM> XL1-Blue with pMAD and pMAD-fla</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Plasmid</TH><TH scope="col">Concentration  after prep (ng/µL)</TH></TR><TR><TH scope="row">pMAD</TH><TD>114  &amp; 108</TD></TR><TR><TH scope="row">pMAD-fla (iGEM-005)</TH><TD>114,  128 &amp; 142</TD></TR></TBODY></TABLE><P>Plasmids were frozen away at -20°C</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.54">13.54 Minipreps of Nose-Tag-plasmids</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Plasmid</TH><TH scope="col">Concentration  after prep (ng/µl)</TH></TR><TR><TH scope="row">002-21</TH><TD>368</TD></TR><TR><TH scope="row">002-22</TH><TD>401</TD></TR><TR><TH scope="row">002-Cu-22</TH><TD>425</TD></TR><TR><TH scope="row">002-Cu-23</TH><TD>427</TD></TR><TR><TH scope="row">002-Ag-22</TH><TD>806</TD></TR><TR><TH scope="row">002-Cu-23</TH><TD>402</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.55">13.55 Inoculation of Minipreps with Ag-21</A></LEGEND><DIV class="exp-content"><P>The positive clone 3 from the colony PCR (28.05.14) was used to inoculate 6 ml LB-Amp for a miniprep in order to receive the plasmid for transforming <EM>Bacillus subtilis</EM>.
      Incubation at room temperature over two days.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.49b">15.49 Inoculation of pMAD-fla-cup1-1 for Miniprep</A></LEGEND><DIV class="aim"><P>Aim: inoculation for miniprep in order to receive plasmid for bacillus-trafo</P></DIV><DIV class="exp-content"><P>6 mL LB-Amp were inoculated with  the positive clone 1 from the Gibson assembly with <I>cup</I>1-1 of the transformed <I>E. coli</I> XL1-Blue from 26.05.14. After isolation of the plasmid <EM>Bacillus Subtilis.</EM> can be transformed.Incubation at room temperature over two days.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.54b">13.54 Transformation of Bacillus with isolated plasmids</A></LEGEND><DIV class="aim"><P>Aim: Creation of Bacillus strains with the following plasmids:</P></DIV><DIV class="exp-content"><UL class="list"><LI>BS piGEM002 + DT22</LI><LI>BS piGEM 011 (002 AG + DT22)</LI><LI>BS piGEM 012 (002 AG + DT23)</LI><LI>BS piGEM 014 (002 Cu + DT22)</LI><LI>BS piGEM 002 (Nose)</LI><LI>BS piGEM 003 (Nose Ag)</LI><LI>BS piGEM 004 (Nose Cu)</LI></UL></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.55b">13.55 Colony-PCR of the <EM>E. coli</EM> containing the degradation tagged constructs</A></LEGEND><DIV class="aim"><P>Aim: check for correct tagging of <I>gfp</I> with the respective degradation tag</P></DIV><DIV class="exp-content"><P>Phusion and Q5 Buffer did not work out so the PCR was repeated with Q5 Mastermix.
	PCR performed with 002-Cu21 and 002-23
	Positive clones for both tags (approx. 250 bp) have subsequently been inoculated in LB-Amp and incubated at room temperature over two days.</P></DIV></FIELDSET></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Marburg:Project:Notebook:May">http://2014.igem.org/Team:Marburg:Project:Notebook:May</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Marburg:Project:Notebook:May" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Marburg:Project:Notebook:May" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Marburg:Project:Notebook:May&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Marburg:Project:Notebook:May&amp;oldid=285272" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>