<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Marburg_Project_Notebook_July"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Marburg:Project:Notebook:July">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Marburg:Project:Notebook:July&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Marburg:Project:Notebook:July&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Marburg:Project:Notebook:July&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Marburg:Project:Notebook:July" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Marburg:Project:Notebook:July</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="navbox"><NAV class="menu center first" aria-haspopup="true" aria-controls="p1"><UL id="p1" class="first"><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg" class="a">Home</A></LI><LI class="current"><A href="https://2014.igem.org/Team:Marburg:Project" class="b">Project</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Safety" class="c">Safety</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Team" class="d">Team</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Policy_Practices" class="e">Policy &amp; Practices</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Attributions" class="f">Attributions</A></LI></UL></NAV><NAV class="menu center second" aria-haspopup="true" aria-controls="p1"><UL id="p1" class="second"><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:Silver" class="d">SilverSURF</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:Tumor" class="c">CancerSURF</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:NRPS" class="e">RiboSURF</A></LI><LI class="current"><A href="https://2014.igem.org/Team:Marburg:Project:Notebook" class="f">Notebook</A></LI><LI class="last" aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:Parts" class="g">Parts</A></LI></UL></NAV></DIV><DIV class="mr_content"><H1>

Notebook: July

</H1><UL id="submenu"><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook">March</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:April">April</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:May">May</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:June">June</A></LI><LI class="current"><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:July">July</A></LI><LI><A href="#01.07.2014">01.07.2014</A></LI><LI><A href="#02.07.2014">02.07.2014</A></LI><LI><A href="#03.07.2014">03.07.2014</A></LI><LI><A href="#04.07.2014">04.07.2014</A></LI><LI><A href="#05.07.2014">05.07.2014</A></LI><LI><A href="#06.07.2014">06.07.2014</A></LI><LI><A href="#07.07.2014">07.07.2014</A></LI><LI><A href="#08.07.2014">08.07.2014</A></LI><LI><A href="#09.07.2014">09.07.2014</A></LI><LI><A href="#10.07.2014">10.07.2014</A></LI><LI><A href="#11.07.2014">11.07.2014</A></LI><LI><A href="#14.07.2014">14.07.2014</A></LI><LI><A href="#15.07.2014">15.07.2014</A></LI><LI><A href="#16.07.2014">16.07.2014</A></LI><LI><A href="#17.07.2014">17.07.2014</A></LI><LI><A href="#18.07.2014">18.07.2014</A></LI><LI><A href="#19.07.2014">19.07.2014</A></LI><LI><A href="#20.07.2014">20.07.2014</A></LI><LI><A href="#21.07.2014">21.07.2014</A></LI><LI><A href="#22.07.2014">22.07.2014</A></LI><LI><A href="#23.07.2014">23.07.2014</A></LI><LI><A href="#24.07.2014">24.07.2014</A></LI><LI><A href="#25.07.2014">25.07.2014</A></LI><LI><A href="#28.07.2014">28.07.2014</A></LI><LI><A href="#31.07.2014">31.07.2014</A></LI></UL><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:August">August</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:September">September</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:October">October</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:Methods">Methods</A></LI><DIV class="notebooky-entry"><H2 class="title"><A name="01.07.2014">01.07.2014</A></H2><FIELDSET class="exp20"><LEGEND><A name="exp20.10">20.10 pMAD-Transformation of competent <I>Bacillus subtilis</I>  WT 3610</A></LEGEND><DIV class="aim"><P>Aim: Transformation of piGEM-016</P></DIV><DIV class="exp-content"><P>100 µL  of overnight culture were added to 10 mL of MNGE-Medium and incubated till an  OD of 0,7 at 37°C which took  7 hours.</P><P>After  reaching OD of  0,7  400 µL of the culture were transformed with  1,5 µg piGEM-021. After 1 hour incubation at 37°C 100 µL Expression mix was added  and incubated for 1h as well.</P><P>In the  end the 500 µL attempt were plated out on MLS-X-Gal plates and incubated at 30°C overnight until colonies could be seen. </P></DIV></FIELDSET><FIELDSET class="exp"><LEGEND><A name="exp_6">Sequencing  results of constructs from 30.06.2014</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Premix</TH><TH scope="col">Label-Nr.</TH><TH scope="col">Construct</TH><TH scope="col">Result</TH></TR><TR><TH scope="row">1</TH><TD>AGB0023<STRONG>411</STRONG></TD><TD>pET24d-Hag-<I>Spe</I>I  construct cl. 2</TD><TD>Correct insertion of  Hag-<I>Spe</I>I</TD></TR><TR><TH scope="row">2</TH><TD>AGB0023<STRONG>412</STRONG></TD><TD>pET24d-Hag-DARPin  construct cl. 7</TD><TD>Correct insertion of  Hag-DARPin</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.48">14.48 PCR of PheA-single</A></LEGEND><DIV class="aim"><P>Aim: Amplify PheA-single fragment for Gibson Assembly in pET-28a</P></DIV><DIV class="exp-content"><P>The PCR with TG_PheA-Arc1p-C-8x FP und TG-PheA-single-GA RP should generate the 1716 bp fragment for Gibson Assembly from the PheA-Arc1p-C-2x-pET28a construct (14.23).</P><P>The used Primer concentration was 1 
    µM, the concentration of the template was 1 ng/µL and we used a hot start to avoid unspecific annealing of primers.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Volume [µL]</TH></TR><TR><TH scope="row">Water</TH><TD>29.5</TD></TR><TR><TH scope="row">5x GC Buffer</TH><TD>10</TD></TR><TR><TH scope="row">DMSO</TH><TD>2.5</TD></TR><TR><TH scope="row">TG_PheA-Arc1p-C-8x FP</TH><TD>2.5</TD></TR><TR><TH scope="row">TG_PheA-Arc1p-C-8x RP</TH><TD>2.5</TD></TR><TR><TH scope="row">PheA-Arc1p-C-2x-pET28a</TH><TD>1</TD></TR><TR><TH scope="row">dNTPs</TH><TD>1</TD></TR><TR><TH scope="row">Phusion-Polymerase</TH><TD>1</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature [°C]</TH><TH scope="col">Time [min:sec]</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>2:00</TD></TR><TR><TH scope="row">2</TH><TD>add Polymerase</TD><TD> </TD></TR><TR><TH scope="row">3</TH><TD>98</TD><TD>0:10</TD></TR><TR><TH scope="row">4</TH><TD>67.5</TD><TD>0:30</TD></TR><TR><TH scope="row">5</TH><TD>72</TD><TD>1:15</TD></TR><TR><TH scope="row">6</TH><TD>go to 3</TD><TD>32x</TD></TR><TR><TH scope="row">7</TH><TD>72</TD><TD>10:00</TD></TR><TR><TH scope="row">8</TH><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P>The amplified fragment was purified on an agarose gel on which it showed the correct size. It was extracted from the gel for further use in a Gibson Assembly yielding a concentration of 198 ng/µL.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="02.07.2014">02.07.2014</A></H2><FIELDSET class="exp14"><LEGEND><A name="exp14.49">14.49 Gibson Assembly</A></LEGEND><DIV class="aim"><P>Aim: Integrate PheA-single into linearized pET-28a(<I>Xho</I>I,<I>Nde</I>I) via Gibson Assembly</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Volume [µL]</TH></TR><TR><TH scope="row">Water</TH><TD>5.34</TD></TR><TR><TH scope="row">pET-28a(XhoI,NdeI) (24 ng/µL)</TH><TD>4.2</TD></TR><TR><TH scope="row">PheA-single (198 ng/µL)</TH><TD>0.46</TD></TR><TR><TH scope="row">2x Gibson Mastermix</TH><TD>10</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD></TR></TBODY></TABLE><P>The reaction mixture was incubated for 1 h at 50 °C and 350 rpm. E. coli Top10 electrocompetent cells were transformed with a 1:3 and a 1:6 dilution of the mixture, incubated for 1 h at 37 °C and plated out on LB-Kan<SUP>50</SUP> plates that were incubated overnight at 37 °C.</P></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.3">19.3 Co-Transformation of piGEM-019 &amp; -020 into <I>E. coli</I> BL21(DE3) with FliS</A></LEGEND><DIV class="aim"><P>Aim: Transformation of plasmids with flagellin modifications and chaperone for Flagellin into protein producing strain</P></DIV><DIV class="exp-content"><P>piGEM-019  and -020 were used to transform <I>E. coli</I> BL21 (DE3) together with a plasmid  containing the flagellin chaperone FlaS in order to express both proteins at  the same time after induction so that the flagellin monomer accumulates inside  the cells.</P><P>The  cells were plated out on Can/Amp-LB plates and incubated overnight at 37°C.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="03.07.2014">03.07.2014</A></H2><FIELDSET class="exp14"><LEGEND><A name="exp14.50">14.50 Overnight culture</A></LEGEND><DIV class="aim"><P>Aim: Amplify Plasmid in an overnight culture</P></DIV><DIV class="exp-content"><P>Six clones of the transformed Top10 cells from 14.49 were picked and used to inoculate overnight cultures (6 x 5 mL).</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.49">18.49 new PCR of isolated Gibson assembly clones ( pET24d-Hag (piGEM-019) and cup1-1)</A></LEGEND><DIV class="aim"><P>Aim: screening clones for right insert</P></DIV><DIV class="exp-content"><P>The isolated plasmid 1:10 diluted from the picked clones 1-4 were used as PCR template.</P><P>The PCR with Flo54 and iGEM-025  should generate a 785 bp fragment and the one with iGEM-024 and -025 the 164bp  domain fragment.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Volume (µL)</TH><TH scope="col">MM 1 for 5 attempts (µL)</TH><TH scope="col">MM 2 for 5 attempts (µL)</TH></TR><TR><TH scope="row"><P>Plasmid from clones 1-4 (1:10)</P></TH><TD>1</TD><TD>-</TD><TD> </TD></TR><TR><TH scope="row">Primer Flo 54 </TH><TD>-</TD><TD>2,5</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-024</TH><TD>-</TD><TD>-</TD><TD>2,5</TD></TR><TR><TH scope="row">Primer iGEM-025</TH><TD>-</TD><TD>2,5</TD><TD>2,5</TD></TR><TR><TH scope="row">Q5 mastermix 2X</TH><TD>-</TD><TD>50</TD><TD>50</TD></TR><TR><TH scope="row">Water</TH><TD>-</TD><TD>45</TD><TD>45</TD></TR><TR><TH scope="row">Mastermix</TH><TD>19</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>100</TD><TD>100</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1 min 45 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>30x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The agarose gel showed bands at  ca. 750 bp which fits to the expected 780 bp fragment with primers piGEM-025  (cup rv) and Flo54 ( Hag fw).</P><P>The gel's second row shows the  amplified cup-fragment under the 200 bp ladder mark. The expected size is ca.  164 bp and fits to the gel result.</P><P>Plasmid from clone 4 was used to transform <EM>E.Coli</EM> DH5α and named piGEM-021.</P></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.4">19.4 Expression-Test with transformed <I>E. coli</I> BL21 (DE3)</A></LEGEND><DIV class="aim"><P>Aim: Checking the overexpression of Hag-<I>Spe</I>I and Hag-<I>Spe</I>I-DARPin</P></DIV><DIV class="exp-content"><P>In  order to check the expression of the proteins a single clone containing  piGEM-019/-20 was used to inoculate 20 mL of LB-Can/Amp. The culture was  incubated at 37°C until an OD of 0,7.</P><P>A 1 mL  preinduction sample (PI) was taken (0,7 (favoured OD) : 0,7 (actual OD) = 1 mL).  After that the cultures were induced with 20 µL IPTG for 2 hours.</P><P>An  induction sample (I) was taken (320 µL with an OD of 2,2; 350 µL with an OD of  2).</P><P>PI and  I were centrifuged at 14000 rpm for 1 min, the supernatant was discarded and  the pellet resuspended in 60 µL water and 40 µL SDS-Buffer.</P><P>The  four samples were analysed on an SDS-PAGE gel with 5 and 10 µL volume per sample.</P><P>The gel showed that the induction with IPTG was successful and the cells overproduce a protein after 2h incubation.</P></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.a5">19.5 Protein overexpression with transformed <I> E. coli</I> BL21 (DE3)</A></LEGEND><DIV class="aim"><P>Aim: Overexpression of flagellin modifications for crystallization</P></DIV><DIV class="exp-content"><P>After the positive expression test the protein expression was done on a higher level with an overnight lactose induction. </P><P>For  that purpose 2 x 1 L LB were inoculated with a solution of LB and every clone  on the cotransformation plate. All clones from the piGEM-019/ -020 + FlaS  cotransformation plates were picked and resuspended in 1 mL LB. 500 µL were  added t the 1 L culture flasks. Each flask was additionally mixed with 1 mL  Ampicillin (100 mg/ mL) and 1 mL Canamycin (50 mg/ mL).</P><P>The  expression was induced overnight with lactose (12,5 g/ L). 50 g lactose were  solved in 200 mL millipore water. Each culture was mixed with 50 mL of the  lactose/ water suspension.</P><P>The  cultures were incubated overnight at 30°C.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col"> </TH><TH scope="col">Lactose solution</TH><TH scope="col">BL21-piGEM-019 culture</TH><TH scope="col">BL21-piGEM-020 culture</TH></TR><TR><TH scope="row">LB</TH><TD>-</TD><TD>1 L</TD><TD>1 l</TD></TR><TR><TH scope="row">Ampicillin</TH><TD>-</TD><TD>1 ml</TD><TD>1 ml</TD></TR><TR><TH scope="row">Canamycin</TH><TD>-</TD><TD>1 ml</TD><TD>1 ml</TD></TR><TR><TH scope="row">Lactose</TH><TD>50g</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Millipore water</TH><TD>200ml</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Lactose solution</TH><TD>-</TD><TD>50 ml</TD><TD>50 ml</TD></TR><TR><TH scope="row">BL21-piGEM-019 clones</TH><TD>-</TD><TD>+</TD><TD>-</TD></TR><TR><TH scope="row">BL21-piGEM-020 clones</TH><TD>-</TD><TD>-</TD><TD>+</TD></TR></TBODY></TABLE></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="04.07.2014">04.07.2014</A></H2><FIELDSET class="exp"><LEGEND><A name="exp_7">Sequencing</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Premix</TH><TH scope="col">Label-Nr.</TH><TH scope="col">Construct</TH><TH scope="col">Primers</TH></TR><TR><TH scope="row">1</TH><TD>AGB0023<STRONG>413</STRONG></TD><TD>pET24d-Hag-<I>Spe</I>I-Cup  constr. Cl. 4</TD><TD>T7 Term</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.51">14.51 Preparation of plasmids</A></LEGEND><DIV class="aim"><P>Aim: Purify the amplified plasmids from overnight cultures</P></DIV><DIV class="exp-content"><P>The plasmids were purified from the overnight cultures created in 14.50 using Sigma GenElute Plasmid Miniprep Kit following the included protocol. The plasmids were eluted using 45 µL water.</P></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.5b">19.5 Protein overexpression with transformed <I>E. coli</I> BL21 (DE3)</A></LEGEND><DIV class="exp-content"><P>1 mL of  culture was spinned down, mixed with 60 µL water, 40 µL SDS-PAGE loading buffer  and analysed on a SDS-PAGE gel.</P><P>The gel shows a high  concentration of a specific protein.</P><P> The overnight induced cells were raised till  an OD of 2-3  and spinned down at 4000  rpm.  The pellets were frozen in liquid  nitrogen and stored at -80µC.</P></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.6a"> 19.6 Co-Transformation of piGEM-021 &amp; with FlaiS  <I>E. coli</I> BL21(DE3) </A></LEGEND><DIV class="aim"><P>Aim: Transformation of plasmids with flagellin-cup modification and chaperone for Flagellin into protein producing strain</P></DIV><DIV class="exp-content"><P>piGEM-021  and pFlaiS were used to transform into <EM>E.Coli</EM> BL21(DE3) and plated out  on LB-Canamycin plates. The plate was incubated at 37µC overnight. </P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="05.07.2014">05.07.2014</A></H2><FIELDSET class="exp19"><LEGEND><A name="exp19.7a">19.7 Expression-Test with transformed <I>E. coli</I> BL21 (DE3) with piGEM-021</A></LEGEND><DIV class="aim"><P>Aim: Checking the overexpression of Hag-<I>Spe</I>I-cup1-1</P></DIV><DIV class="exp-content"><P>In  order to check the expression of the proteins a single clone containing  piGEM-021 was used to inoculate 20 mL of LB-Can/Amp. The culture was incubated  at 37°C until an OD of 0,7.</P><P>A 1 mL  preinduction sample (PI) was taken (0,7 (favoured OD) : 0,7 (actual OD) = 1 mL).  After that the cultures were induced with 20 µL IPTG for 2h.</P><P>An  induction sample (I) was taken (520 µL with an OD of 1,34).</P><P>PI and  I were centrifuged at 14000 rpm for 1 min, the supernatant was discarded and  the pellet resuspended in 60 µL water and 40 µL SDS-Buffer.</P><P>The  samples were analysed on an SDS-PAGE gel with 5 and 10 µL volume per sample.  Additionally PI and I from the expression test with transformed piGEM-021 were  loaded on the gel as well as a comparison.</P><P>The gel shows a high concentration of a specific protein so that the test could be seen as successful.</P></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.6b">19.6 Protein overexpression with transformed <I>E. coli</I> BL21 (DE3) (piGEM-021)</A></LEGEND><DIV class="aim"><P>Aim: Overexpression of flagellin modifications for crystallization</P></DIV><DIV class="exp-content"><P>After  the positive expression test the protein expression was done on a higher level  with an overnight lactose induction. </P><P>For  that purpose 2 x 1 L LB were inoculated with a solution of LB and every clone  on the cotransformation plate. All clones from the piGEM-021 + FlaiS  cotransformation plates were picked and resuspended in 1 mL LB. 500 µL were  added t the 1 L culture flasks. Each flask was additionally mixed with 1 mL Ampicillin  (100 mg/ mL) and 1 mL Canamycin (50 mg/ mL).</P><P>The  expression was induced overnight with lactose (12,5 g/ L). 25 g lactose were  solved in 100 mL millipore water which was heated 1 min in the microwave for  solvation. Each culture was mixed with 50 mL of the lactose/ water suspension.</P><P>The  cultures were incubated overnight at 30°C.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="06.07.2014">06.07.2014</A></H2><FIELDSET class="exp20"><LEGEND><A name="exp20.11">20.11 Overnight culture of blue clones</A></LEGEND><DIV class="aim"><P>Aim: transformation of <I>Bacillus subtilis</I> WT3610 with plasmid</P></DIV><DIV class="exp-content"><P>Colonies  were grown on the plates with transformed piGEM-021. The blue/ white screening  showed positive transformed blue clones.   3 clones of different morphology per plate were picked and used for  inoculation of LB-MLS (4 mL LB, 4 µL lincomycin, 4 µL erythromycin). Incubation  was carried out overnight at 30°C with the 3 cultures.</P></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.6">19.6 Protein overexpression with transformed <I>E. coli</I> BL21 (DE3) (piGEM-021)</A></LEGEND><DIV class="exp-content"><P>The gel  shows a high concentration of a specific protein after overnight induction with  lactose referring to the Hag-Cup construct.</P><P>The  overnight induced cells were raised till an OD of 2-3  and spinned down at 4000 rpm.  The pellets were frozen in liquid nitrogen  and stored at -80°C.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="07.07.2014">07.07.2014</A></H2><FIELDSET class="exp20"><LEGEND><A name="exp20.12">20.12 First temperature shift</A></LEGEND><DIV class="aim"><P>Aim: integration of  pMAD-Insert into Bacillus chromosome via flanks</P></DIV><DIV class="exp-content"><P>The 3  overnight cultures were used to inoculate 10 mL LB MLS until  the culture obtained an OD of 0,1. The  cultures were incubated at 30°C for 2 hours.</P><P>Then  the temperature was shifted to 42°C for 6 hours. Unfortunately the temperature  decreased to 33°C because of a wrong incubator setting. New overnight cultures  were inoculated with 50 µL from those used in the morning. The new cultures  were incubated overnight at 30°C.</P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.52">14.52 Transformation of BL21 cells</A></LEGEND><DIV class="aim"><P>Aim: Transform <I>E. coli</I> BL21(DE3) with PheA-single-pET28a</P></DIV><DIV class="exp-content"><P>40 µL <I>E. coli</I> BL21(DE3) electrocompetent cells were transformed with 1 ng of the previously purified plasmid PheA-single-pET28a from two different clones, incubated for 1 h at 37 °C and plated out on LB-Kan<SUP>50</SUP> plates that were incubated overnight at 37 °C.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="08.07.2014">08.07.2014</A></H2><FIELDSET class="exp"><LEGEND><A name="exp_8">Sequencing  results of constructs from 04.07.2014</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Premix</TH><TH scope="col">Label-Nr.</TH><TH scope="col">Construct</TH><TH scope="col">Result</TH></TR><TR><TH scope="row">1</TH><TD>AGB0023<STRONG>413</STRONG></TD><TD>pET24d-Hag-<I>Spe</I>I-Cup  constr. Cl. 4</TD><TD>Correct insertion of  Cup1-1 into Hag</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.53">14.53 Overnight culture</A></LEGEND><DIV class="aim"><P>Aim: Inoculate Overnight culture of PheA-single-pET28a</P></DIV><DIV class="exp-content"><P>5 mL of LB-Kan<SUP>50</SUP> medium were inoculated with a clone from 14.52 bearing the PheA-single-pET28a plasmid.</P></DIV></FIELDSET><FIELDSET class="exp20"><LEGEND><A name="exp20.13">20.13 First temperature shift</A></LEGEND><DIV class="aim"><P>Aim: integration of  pMAD-Insert into Bacillus chromosome via flanks</P></DIV><DIV class="exp-content"><P>The 3  overnight cultures were used to inoculate 10 mL LB MLS until  the culture obtained an OD of 0,1. The  cultures were incubated at 30°C for 2h.</P><P>Then  the temperature was shifted to 42°C for 6h.</P><P>After  the heat shock 100 µL dilutions from 10<SUP>-4</SUP> to 10<SUP>-6</SUP> of each  culture were plated out on MLS-X-Gal so that nine plates could be incubated  overnight at 42°C.</P></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.7b">19.7 Purification of Flagellin-Hag-<I>Spe</I>I from frozen pellet</A></LEGEND><DIV class="exp-content"><P>The  pellets which were stored at -80°C were resuspended in 20 mL buffer A and  cracked with the microfluidizer. The cell lysate was centrifuged at 20000 rpm  for 20 min at 4&amp;degC. The supernatant was transferred to a new 50 mL falcon and  stored on ice.</P><P>The following steps were performed:</P><UL class="list"><LI>equilibration with Buffer A 10 min</LI><LI>taking 40 µL supernatant  (load)+ 10 µL SDS-Buffer - L-Sample</LI><LI>50 mL load on column</LI><LI>taking 40 µL of flow through + 10 µL SDS-buffer - FT-Sample</LI><LI>first washing with 25ml Buffer A (half the load)</LI><LI>taking 40 µL of washing flow through + 10 µL SDS-Buffer - W-Sample</LI><LI>equilibrate with Buffer B (output pipe off the column into glas, input into B)</LI><LI>hanging pipe on column, 6 x 2 ml Evolutions - E 1-6</LI><LI>taking 40 µL of Elution 1-6 + 10 µL SDS-Buffer E-sample 1-6</LI></UL><P>regeneration of column:</P><UL class="list"><LI>10min water</LI><LI>10min EDTA</LI><LI>10min water</LI><LI>10min NiSO4</LI><LI>10min water</LI></UL><P>Meanwhile  the elutions 1-6 were pooled (combined 12 mL) in a 50 mL falcon and transferred  in a concentrator column which was centrifuged until the volume in the filter  was 1,5 mL for injection into the gel filtration station. </P><P>Parallel  the L-, FL-, W- and E 1-6-samples were analysed via SDS-PAGE gel in order to  proof the purification in the elution fractions.</P><P>The gel  shows a huge protein concentration in E1 and E2 so that the purification could  be continued with gel filtration. For that purpose the filtrate  out of the concentrator was transferred into a new 2 mL tube resuspending very  well without air bubbles.</P><P>17 mL  of GeFi-buffer were injected into the loop. After that the protein filtrate was  injected as well. </P><P>The  GeFi was started according to the following settings:</P><UL class="list"><LI>Flow rate: 2500</LI><LI>max. pressure: 0,65 mPa</LI><LI>First endtimer at 90 mL volume before fractioning</LI></UL><P>After  observing a peak on the screen 40 µL of different fractions (C7 , C9, C11, D1,  D3, D5, D7) covering the peak were taken for analysis via SDS-PAGE gel.</P><P>The gel  shows that the elutions contain the protein concentrated. The Elutions C7-D3  were pooled and transferred to the concentrator until an end volume of 200 µL.</P><P>The  protein concentrate was used for crystallisation pure and in a 1:2 dilution  with GeFi buffer.</P><P>Core I  and II were used for setting drops.</P></DIV></FIELDSET><FIELDSET class="exp"><LEGEND><A name="exp_9">Sequencing  of purified cPCR samples from pMAD transformation</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Premix</TH><TH scope="col">Label-Nr.</TH><TH scope="col">Construct</TH><TH scope="col">Primers</TH></TR><TR><TH scope="row">1</TH><TD>AGB0023<STRONG>414</STRONG></TD><TD>Hag-Spe1 in B.s.  genome clone 1</TD><TD>Hag-Nco-fw</TD></TR><TR><TH scope="row">2</TH><TD>AGB0023<STRONG>415</STRONG></TD><TD>Hag-Spe1 in B.s.  genome clone 3</TD><TD>Hag-Nco-fw</TD></TR><TR><TH scope="row">3</TH><TD>AGB0023<STRONG>416</STRONG></TD><TD>Hag-Spe1-DARPin  in B.s. genome clone 1</TD><TD>Hag-Nco-fw</TD></TR><TR><TH scope="row">4</TH><TD>AGB0023<STRONG>417</STRONG></TD><TD>Hag-Spe1-DARPin  in B.s. genome clone 3</TD><TD>Hag-Nco-fw</TD></TR></TBODY></TABLE></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="09.07.2014">09.07.2014</A></H2><FIELDSET class="exp20"><LEGEND><A name="exp20.14">20.14 Second temperature shift</A></LEGEND><DIV class="aim"><P>Aim: flip out of the pMAD backbone</P></DIV><DIV class="exp-content"><P>One  blue colony per diluted clone was used to inoculate 4 mL LB. The 6 cultures  were incubated at 30°C for 6 hours and afterwards for 3 hours at 42°C.</P><P>Dilutions  from 10<SUP>-4</SUP> - to 10<SUP>-6</SUP> were plated out on 18 X-Gal plates  without MLS selection. The positive clones should not contain the resistance  inside the backbone as well as the galactosidase. The plates were incubated at  42°C overnight.</P></DIV></FIELDSET><FIELDSET class="exp"><LEGEND><A name="exp_10">Results of Sequencing  of cPCR samples from pMAD transformation from 08.07.14</A></LEGEND><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Premix</TH><TH scope="col">Label-Nr.</TH><TH scope="col">Construct</TH><TH scope="col">Result</TH></TR><TR><TH scope="row">1</TH><TD>AGB0023<STRONG>414</STRONG></TD><TD>Hag-<I>Spe</I>I in <I>B. subtilis</I> genome clone 1</TD><TD>No <I>Spe</I>I insertion</TD></TR><TR><TH scope="row">2</TH><TD>AGB0023<STRONG>415</STRONG></TD><TD>Hag-<I>Spe</I>I in <I>B. subtilis</I> genome clone 3</TD><TD>No <I>Spe</I>I insertion</TD></TR><TR><TH scope="row">3</TH><TD>AGB0023<STRONG>416</STRONG></TD><TD>Hag-<I>Spe</I>I-DARPin  in <I>B. subtilis</I> genome clone 1</TD><TD>Not sequenced through</TD></TR><TR><TH scope="row">4</TH><TD>AGB0023<STRONG>417</STRONG></TD><TD>Hag-<I>Spe</I>I-DARPin  in <I>B. subtilis</I> genome clone 3</TD><TD>Not sequenced through</TD></TR></TBODY></TABLE><P>The sequencing will be repeated with the reverse Primer instead of the  forward primer. In case of the Hag-<I>Spe</I>I clones, different clones have to be  taken for sequencing.</P></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.7">19.7 Purification of Flagellin-Hag-SpeI from frozen pellet</A></LEGEND><DIV class="exp-content"><P>After one day in well B1 Core I fine crystals could be seen.</P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.54">14.54 Test Expression</A></LEGEND><DIV class="aim"><P>Aim: Test the expression of PheA-single</P></DIV><DIV class="exp-content"><P>Using the preculture from 14.53 60 mL of LB-Kan<SUP>50</SUP> medium were inoculated with 600 µL of the preculture and incubated at 37 °C and 220 rpm until OD<SUB>600</SUB> reached 0.5. The culture was cooled down to 18 °C and the protein production was induced with 12 µL of 500 mM IPTG. Over night the culture was incubated at 18 °C and 220 rpm. Afterwards the cell culture was centrifuged (17000 rpm, 20 min, 4 °C), the pellet resuspended in 3 mL buffer A and stored at -20 °C. A glycerolstock of the strain containing the PheA-single-pET28a plasmid was stored at -80 °C.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="10.07.2014">10.07.2014</A></H2><FIELDSET class="exp"><LEGEND><A name="exp_11">Sequencing  of cPCR samples from pMAD transformation </A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Premix</TH><TH scope="col">Label-Nr.</TH><TH scope="col">Construct</TH><TH scope="col">Primer</TH></TR><TR><TH scope="row">1</TH><TD>AGB0023<STRONG>439</STRONG></TD><TD>Hag-<I>Spe</I>I in <I>B. subtilis</I>  genome clone 7</TD><TD>Hag-<I>Bam</I>HI-rv</TD></TR><TR><TH scope="row">2</TH><TD>AGB0023<STRONG>440</STRONG></TD><TD>Hag-<I>Spe</I>I-DARPin  in <I>B. subtilis</I> genome clone 1</TD><TD>Hag-<I>Bam</I>HI-rv</TD></TR><TR><TH scope="row">3</TH><TD>AGB0023<STRONG>441</STRONG></TD><TD>Hag-<I>Spe</I>I-DARPin  in <I>B. subtilis</I> genome clone 3</TD><TD>Hag-<I>Bam</I>HI-rv</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.8">19.8 Purification of Flagellin-Hag-Cup1-1 from frozen pellet</A></LEGEND><DIV class="exp-content"><P>The  pellets which were stored at -80°C were resuspended in 20 mL buffer A and  cracked with the microfluidizer. The cell parts were spinned down at 20000 rpm  for 20 min at 4°C. The supernatant was transferred to a new 50 mL falcon and  stored on ice.</P><P>The following steps were performed:</P><UL class="list"><LI>equilibration with Buffer A 10 min</LI><LI>taking 40 µL supernatant  (load)+ 10 µL SDS-Buffer - L-Sample</LI><LI>50 mL load on column</LI><LI>taking 40 µL of flow through + 10 µL SDS-buffer - FT-Sample</LI><LI>first washing with 25ml Buffer A (half the load)</LI><LI>taking 40 µL of washing flow through + 10 µL SDS-Buffer - W-Sample</LI><LI>equilibrate with Buffer B (output pipe off the column into glas, input into B)</LI><LI>hanging pipe on column, 6 x 2 ml Evolutions - E 1-6</LI><LI>taking 40 µL of Elution 1-6 + 10 µL SDS-Buffer E-sample 1-6</LI></UL><P>regeneration of column:</P><UL class="list"><LI>10min water</LI><LI>10min EDTA</LI><LI>10min water</LI><LI>10min NiSO4</LI><LI>10min water</LI></UL><P>Meanwhile  the Elutions 1-6 were pooled (combined 12 mL) in a 50 mL falcon and transferred  in a concentrator column which was centrifuged until the volume in the filter  was 1,5 mL for injection into the gel filtration station. </P><P>Parallel  the L-, FL-, W- and E 1-6-samples were analysed on a SDS-PAGE gel in order to  proof the purification in the elution fractions.</P><P>The gel  shows a huge protein concentration in E1 and E2 so that the purification could  be continued with the gel filtration. For that purpose the filtrate  out of the concentrator was transferred into a new 2 mL tube resuspending very  well without air bubbles. </P><P>17 mL  of GeFi-buffer werd injected into the loop. After that the protein filtrate was  injected as well. </P><P>The  GeFi was started according to the following settings:</P><UL class="list"><LI>Flow rate: 2500-3000  µL / min</LI><LI>max. pressure: 0,60 -  0,65 mPa</LI><LI>First  end timer at 80-90 mL volume before fractioning </LI><LI>Injectvalve</LI></UL><P>After  observing a peak on the screen 40 µL of different fractions (C7 , C9, C11, D1,  D3, D5, D7) covering the peak were taken for analysis via SDS-PAGE gel.</P><P>The gel  shows that the elutions contain the protein concentrated. The Elutions  C7-D7were pooled and transferred to the concentrator until an end volume of 200 µL.</P><P>The protein  concentrate (27,4 mg/mL A280) was used for crystallization pure and in a 1:2  dilution with GeFi buffer.</P><P>Core I  - IV were used for setting drops.</P></DIV></FIELDSET><FIELDSET class="exp20"><LEGEND><A name="exp20.15">20.15 selection of positive clones</A></LEGEND><DIV class="aim"><P>Aim: checking the correct flip out of the pMAD backbone</P></DIV><DIV class="exp-content"><P>From  the dilution plates was one WHITE clone picked and transferred on a Master  X-Gal Plate as well as on a MLS plate so that nine clones were proven for the right  integration of the insert although flipping out the pMAD backbone.</P><P>The  plates were incubated at 42°C overnight.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.66">13.66 Digest of piGEM-002 <I>Nco</I>I/<I>Spe</I>I</A></LEGEND><DIV class="aim"><P>Aim: digest for isolation of piGEM-002 without GFP, replacing it with mCherry out of pMA17</P></DIV><DIV class="exp-content"><P>piGEM-002 was digested with <I>Nco</I>I/<I>Spe</I>I in order to isolate the backbone without the GFP so that an mCherry could be integrated via Gibson assembly.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Volume (µl)</TH></TR><TR><TH scope="row">piGEM-002 (73 ng/µL)</TH><TD>14</TD></TR><TR><TH scope="row">CutSmart 10x</TH><TD>2</TD></TR><TR><TH scope="row"><I>Spe</I>I</TH><TD>0,2</TD></TR><TR><TH scope="row"><I>Nco</I>I</TH><TD>0,2</TD></TR><TR><TH scope="row">Water</TH><TD>3,6</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD></TR></TBODY></TABLE><P>The digested vector was isolated via gel extraction.</P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.55">14.55 Purification of test expression</A></LEGEND><DIV class="aim"><P>Aim: Purify PheA-single on a small scale</P></DIV><DIV class="exp-content"><P>Lysozyme was added to the cell suspension from 14.54 to a final concentration of 1 mg/mL and incubated on ice for 30 min. Afterwards the cells were lysed using ultra sound (2x 18 pulses, 30 s on ice inbetween). To clear the suspension of cell debris it was centrifuged (13000 rpm, 15 min, 4 °C). The supernatant was purified using QIAGEN Ni-NTA Spin Columns. An SDS-PAGE gel showed that the protein is produced and can be purified using Ni affinity chromatography.</P><P>60 mL LB-Kan<SUP>50</SUP> medium was inoculated from the glycerolstock prepared in 14.54 and incubated at 37 °C and 220 rpm over night.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.67">13.67 mCherry PCR amplification</A></LEGEND><DIV class="aim"><P>Aim: PCR for the amplification of mCherry</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix</TH><TH scope="col">Master-Mix I</TH><TH scope="col">Master-Mix II</TH></TR><TR><TH scope="row">Template: pMA17 1:10 (25 ng/ µL)</TH><TD>5</TD><TD>5</TD></TR><TR><TH scope="row">Primer iGEM-027</TH><TD>5</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-028</TH><TD>5</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-029</TH><TD>-</TD><TD>5</TD></TR><TR><TH scope="row">Primer iGEM-030</TH><TD>-</TD><TD>5</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>10</TD><TD>10</TD></TR><TR><TH scope="row">Phusion Buffer 5x</TH><TD>40</TD><TD>40</TD></TR><TR><TH scope="row">dNTP-mix</TH><TD>5</TD><TD>5</TD></TR><TR><TH scope="row">MgCl2</TH><TD>5</TD><TD>5</TD></TR><TR><TH scope="row">DMSO</TH><TD>5</TD><TD>5</TD></TR><TR><TH scope="row">Water</TH><TD>120</TD><TD>120</TD></TR><TR><TH scope="row">Total Volume</TH><TD>200</TD><TD>200</TD></TR></TBODY></TABLE><P>The PCR was ran as a gradient from 55-65°C annealing temperature overnight.</P><P>The gel shows the successful amplification of the two parts of mCherry.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="11.07.2014">11.07.2014</A></H2><FIELDSET class="exp"><LEGEND><A name="exp_12">Sequencing results  of cPCR samples from pMAD transformation </A></LEGEND><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Premix</TH><TH scope="col">Lebel-Nr.</TH><TH scope="col">Construct</TH><TH scope="col">Primer</TH></TR><TR><TH scope="row">1</TH><TD>AGB0023<STRONG>439</STRONG></TD><TD>Hag-<I>Spe</I>I in <I>B. subtilis</I>  genome clone 7</TD><TD>Positive, but HagII deletion/ frame shift</TD></TR><TR><TH scope="row">2</TH><TD>AGB0023<STRONG>440</STRONG></TD><TD>Hag-<I>Spe</I>I-DARPin  in <I>B. subtilis</I> genome clone 1</TD><TD>1 point mutations in domain, deletion in HagII/ Frameshift</TD></TR><TR><TH scope="row">3</TH><TD>AGB0023<STRONG>441</STRONG></TD><TD>Hag-<I>Spe</I>I-DARPin  in <I>B. subtilis</I> genome clone 3</TD><TD>2 point mutations in domain, deletion in HagII/ Frameshift</TD></TR></TBODY></TABLE></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp_13">Sequencing  of pMAD-constructs </A></LEGEND><DIV class="aim"><P>Aim: finding out if the mutations have been in the constructs or made by Bacillus</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Premix</TH><TH scope="col">Label-Nr.</TH><TH scope="col">Construct</TH><TH scope="col">Primer</TH></TR><TR><TH scope="row">1</TH><TD>AGB0023<STRONG>442</STRONG></TD><TD>piGEM-005 - Hag-<I>Spe</I>I</TD><TD>pMAD-seq-rv</TD></TR><TR><TH scope="row">2</TH><TD>AGB0023<STRONG>443</STRONG></TD><TD>piGEM-016 - Hag-Cup</TD><TD>pMAD-seq-rv</TD></TR><TR><TH scope="row">3</TH><TD>AGB0023<STRONG>444</STRONG></TD><TD>piGEM-018 - Hag-DARPin</TD><TD>pMAD-seq-rv</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.67">13.67 mCherry PCR amplification</A></LEGEND><DIV class="aim"><P>Aim: digest for isolation of piGEM-002 without GFP, replacing it with mCherry</P></DIV><DIV class="exp-contetn"><P>Gel analysis shows that the PCR was successful at every annealing temperature. MgCl<SUB>2</SUB> and DMSO were important for the function of the polymerase. </P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.56">14.56 Expression of PheA-single</A></LEGEND><DIV class="aim"><P>Aim: Express PheA-single for further use</P></DIV><DIV class="exp-content"><P>10 x 500 mL LB-Kan<SUP>50</SUP> medium were inoculated with 5 mL of the overnight culture prepared in 14.55 and incubated at 37 °C and 220 rpm until OD<SUB>600</SUB> reached 0.5. The flasks were cooled to 18 °C and protein production was induced by adding 100 µL 500 mM IPTG. The cultures were incubated over night at 18 °C and 220 rpm. Afterwards the cells were collected by centrifugation (6000 rpm, 20 min, 4 °C), resuspended in buffer A and stored at -20 °C until further use.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="14.07.2014">14.07.2014</A></H2><FIELDSET class="exp13"><LEGEND><A name="exp13.68">13.68 Gibson Assembly with mCherry I and II &amp; linearized piGEM-002</A></LEGEND><DIV class="aim"><P>Aim: recombination of piGEM-002 and mCherry</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Volume (µl)</TH></TR><TR><TH scope="row">Gibson-mix</TH><TD>15</TD></TR><TR><TH scope="row">Insert I (mCherry I 460 ng/µL)</TH><TD>1,25</TD></TR><TR><TH scope="row">Insert II (mCherry II 460 ng/µL)</TH><TD>1,25</TD></TR><TR><TH scope="row">piGEM-002 <I>Nco</I>I/<I>Spe</I>I (5,5 ng/µL)</TH><TD>2,5</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD></TR></TBODY></TABLE><P>The mix was incubated 1 hour at 50°C, and used to transform <I>E. coli</I> XL1-Blue that were plated out on LB-Amp plates. Incubation overnight at 37°C. </P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.57">14.57 Purification of PheA-single</A></LEGEND><DIV class="aim"><P>Aim: Purify PheA-single in three steps</P></DIV><DIV class="exp-content"><P>The resuspended cells from 14.56 were lysed using a french press. The lysate was incubated on ice with 5 µg/mL DNAseA and RNAseI for 10 min and centrifuged (27000 rpm, 45 min, 4 °C) to pellet cell debris. The supernatant was filtered (0.45 µm) and PheA-Arc1p-C-0x purified from the cleared solution by NiNTA affinity chromatography. The combined elution fractions containing crude PheA-Arc1p-C-0x were concentrated to a volume of 2 mL and further purified using an anion exchange column to get rid of possible tRNA contaminations. Finally the buffer was changed to dialysis buffer and the protein concentrated and stored at -80 °C in aliquods for further use.</P></DIV></FIELDSET><FIELDSET class="exp"><LEGEND><A name="exp_14">Sequencing results of pMAD-constructs </A></LEGEND><DIV class="aim"><P>Aim: finding out if the mutations have been in the constructs or made by Bacillus</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Premix</TH><TH scope="col">Label-Nr.</TH><TH scope="col">Construct</TH><TH scope="col">Primer</TH></TR><TR><TH scope="row">1</TH><TD>AGB0023<STRONG>442</STRONG></TD><TD>piGEM-005 - Hag-<I>Spe</I>I</TD><TD>Deletion in FlankII</TD></TR><TR><TH scope="row">2</TH><TD>AGB0023<STRONG>443</STRONG></TD><TD>piGEM-016 - Hag-Cup</TD><TD>Deletion in FlankII</TD></TR><TR><TH scope="row">3</TH><TD>AGB0023<STRONG>444</STRONG></TD><TD>piGEM-018 - Hag-DARPin</TD><TD>Deletion in FlankII</TD></TR></TBODY></TABLE><P>The Flank was checked by sequencing and showed no deletion in the flank.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="15.07.2014">15.07.2014</A></H2><FIELDSET class="exp13"><LEGEND><A name="exp13.68">13.68 Gibson Assembly with mCherry I and II &amp; linearized piGEM-002</A></LEGEND><DIV class="aim"><P>Aim: recombination of piGEM-002 and mCherry</P></DIV><DIV class="exp-content"><P>No colonies were on the plates. The Gibson assembly was repeated with higher concentrated vector.</P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.58">14.58 PCR of Arc1p-C-single</A></LEGEND><DIV class="aim"><P>Aim: Amplify Arc1p-C-single fragment for Gibson Assembly in pET-28a</P></DIV><DIV class="exp-content"><P>The PCR with TG_Arc1p-C FP and TG_PheA-Arc1p-C-8x RP should generate the 579 bp fragment for Gibson Assembly from the PheA-Arc1p-C-2x-pET28a construct (14.23).</P><P>The used primer concentration was 1 
    µM, the concentration of the template was 1 ng/µL and we used a hot start to avoid unspecific annealing of primers.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Volume [µL]</TH></TR><TR><TH scope="row">Water</TH><TD>29.5</TD></TR><TR><TH scope="row">5x GC Buffer</TH><TD>10</TD></TR><TR><TH scope="row">DMSO</TH><TD>2.5</TD></TR><TR><TH scope="row">TG_Arc1p-C FP</TH><TD>2.5</TD></TR><TR><TH scope="row">TG_PheA-Arc1p-C-8x RP</TH><TD>2.5</TD></TR><TR><TH scope="row">PheA-Arc1p-C-2x-pET28a</TH><TD>1</TD></TR><TR><TH scope="row">dNTPs</TH><TD>1</TD></TR><TR><TH scope="row">Phusion-Polymerase</TH><TD>1</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature [°C]</TH><TH scope="col">Time [min:sec]</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>2:00</TD></TR><TR><TH scope="row">2</TH><TD>add Polymerase</TD><TD> </TD></TR><TR><TH scope="row">3</TH><TD>98</TD><TD>0:10</TD></TR><TR><TH scope="row">4</TH><TD>67.5</TD><TD>0:30</TD></TR><TR><TH scope="row">5</TH><TD>72</TD><TD>1:15</TD></TR><TR><TH scope="row">6</TH><TD>go to 3</TD><TD>32x</TD></TR><TR><TH scope="row">7</TH><TD>72</TD><TD>10:00</TD></TR><TR><TH scope="row">8</TH><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P>The amplified fragment was purified on an agarose gel on which it showed the correct size. It was extracted from the gel for further use in a Gibson Assembly yielding a concentration of 122 ng/µL.</P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.59">14.59 Gibson Assembly</A></LEGEND><DIV class="aim"><P>Aim: Integrate Arc1p-C-single into linearized pET-28a(XhoI,NdeI) via Gibson Assembly</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Volume [µL]</TH></TR><TR><TH scope="row">Water</TH><TD>5.38</TD></TR><TR><TH scope="row">pET-28a(XhoI,NdeI) (24 ng/µL)</TH><TD>4.2</TD></TR><TR><TH scope="row">Arc1p-C-single (122 ng/µL)</TH><TD>0.42</TD></TR><TR><TH scope="row">2x Gibson Mastermix</TH><TD>10</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD></TR></TBODY></TABLE><P>The reaction mixture was incubated for 1 h at 50 °C and 350 rpm. E. coli Top10 electrocompetent cells were transformed with a 1:3 and a 1:6 dilution of the mixture, incubated for 1 h at 37 °C and plated out on LB-Kan<SUP>50</SUP> plates that were incubated overnight at 37 °C.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="16.07.2014">16.07.2014</A></H2><FIELDSET class="exp14"><LEGEND><A name="exp14.60">14.60 Overnight culture</A></LEGEND><DIV class="aim"><P>Aim: Amplify Plasmid in an overnight culture</P></DIV><DIV class="exp-content"><P>Six clones of the transformed Top10 cells from 14.59 were picked and used to inoculate overnight cultures (6 x 5 mL).</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="17.07.2014">17.07.2014</A></H2><FIELDSET class="exp13"><LEGEND><A name="exp13.69">13.69 new Gibson Assembly with mCherry I and II &amp; linearized piGEM-002</A></LEGEND><DIV class="aim"><P>Aim: recombination of piGEM-002 and mCherry</P></DIV><DIV class="exp-content"><P>In order to receive the construct more piGEM-002  a different ration of the used parts was used for a new Gibson assembly.</P><P>From Insert I was a higher amount taken becaue of the bigger fragment size. New attempts with 1:2 (vector:insert) and 1:3 ration:</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">1:2 (µl)</TH><TH scope="col">1:3 (µl)</TH></TR><TR><TH scope="row">Gibson-mix</TH><TD>15</TD><TD>15</TD></TR><TR><TH scope="row">Insert I (mCherry  I 460 ng/µL)</TH><TD>2,2</TD><TD>1,1</TD></TR><TR><TH scope="row">Insert II (mCherry  II 460 ng/µL)</TH><TD>2,5</TD><TD>1,25</TD></TR><TR><TH scope="row">piGEM-002 <I>Nco</I>I/<I>Spe</I>I (42 ng/µL)</TH><TD>2,7</TD><TD>1,25</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><P>Gibson reactions were used to transform <I>E. coli</I>  XL1-Blue after 1h incubation at 50°C and plated out on LB-Amp. Incubation was  proceeded at 37°C overnight.</P></DIV></FIELDSET><FIELDSET class="exp21"><LEGEND><A name="exp21.1a">21.1 Swarming Assays</A></LEGEND><DIV class="aim"><P>Aim: Check motility of Bacillus transformants</P></DIV><DIV class="exp-content"><P>For  testing the motility swarming agar (0,7% LB-Agar) and swimming agar (0,3%  LB-Agar) were made and given away for autoclaving.</P><P>Overnight  cultures of Bacillus transformants (DARPin clone 3 &amp; 8, Cup1-1 clone 2  &amp; 3, Hag-<I>Spe</I>I clone 7), PY79 and WT3610 were incubated at 37°C.</P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.61">14.61 Isolation of plasmids</A></LEGEND><DIV class="aim"><P>Aim: Purify the amplified plasmids from overnight cultures</P></DIV><DIV class="exp-content"><P>The plasmids were purified from the overnight cultures created in 14.60 using Sigma GenElute Plasmid Miniprep Kit following the included protocol. The plasmids were eluted using 45 µL water.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="18.07.2014">18.07.2014</A></H2><FIELDSET class="exp21"><LEGEND><A name="exp21.1b">21.1 Swarming Assay</A></LEGEND><DIV class="aim"><P>Aim: Check motility of Bacillus transformands</P></DIV><DIV class="exp-content"><P>The overnight cultures were set to an OD of 10. 10 µL of each culture were spotted on a swarming and a swimming plate. The distance of the front line was measured after periods of time.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col"> </TH><TH colspan="2" scope="col">DARPin cl. 3</TH><TH colspan="2" scope="col">DARPin cl. 8</TH><TH colspan="2" scope="col">Cup1-1 cl. 2</TH><TH colspan="2" scope="col">Cup1-1 cl. 3</TH><TH colspan="2" scope="col">Hag-<I>Spe</I>I cl. 7</TH><TH colspan="2" scope="col">WT3610</TH><TH colspan="2" scope="col">PY79</TH></TR><TR><TH scope="row">Time</TH><TD>Swim</TD><TD>Swarm</TD><TD>Swim</TD><TD>Swarm</TD><TD>Swim</TD><TD>Swarm</TD><TD>Swim</TD><TD>Swarm</TD><TD>Swim</TD><TD>Swarm</TD><TD>Swim</TD><TD>Swarm</TD><TD>Swim</TD><TD>Swarm</TD></TR><TR><TH scope="row">1h</TH><TD>-</TD><TD>-</TD><TD>0,5</TD><TD>2</TD><TD>-</TD><TD>1</TD><TD>-</TD><TD>1,5</TD><TD>3</TD><TD>6</TD><TD>2</TD><TD>3</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">2,5h</TH><TD>-</TD><TD>-</TD><TD>0,5</TD><TD>3,5</TD><TD>-</TD><TD>3</TD><TD>-</TD><TD>2,5</TD><TD>12</TD><TD>40&lt;</TD><TD>7</TD><TD>20</TD><TD>-</TD><TD>-</TD></TR></TBODY></TABLE></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="19.07.2014">19.07.2014</A></H2><FIELDSET class="exp21"><LEGEND><A name="exp21.2a">21.2 Swarming Assay</A></LEGEND><DIV class="aim"><P>Aim: Check motility of Bacillus transformands</P></DIV><DIV class="exp-content"><P>In  order to repeat the swarming assay new overnight cultures of  <I>Bacillus subtilis</I> WT3610, the mutants DARPin  clone 3 &amp; 8, Cup1-1 clone 2 , Hag-SpeI clone 7 in 10 mL LB and incubated at  37°C.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="20.07.2014">20.07.2014</A></H2><FIELDSET class="exp21"><LEGEND><A name="exp21.2">21.2 Swarming Assay</A></LEGEND><DIV class="aim"><P>Aim: Check motility of Bacillus transformants</P></DIV><DIV class="exp-content"><P>The  overnight cultures were set to an OD of 10. 10 µL of each culture were spotted  on a swarming and a swimming plate. The plates with the drops were dried for  half an hour on the bench. The distance of the front line was measured after  periods of time. The measurement at t=0 is the radius of the start spot. The  following measurements refer to the difference between the start spot and the  new front line.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col"> </TH><TH scope="col">Swarming</TH><TH scope="col">Swimming</TH></TR><TR><TH scope="row">WT 3610</TH><TD>+</TD><TD>+</TD></TR><TR><TH scope="row">Hag-<I>Spe</I>I cl. 7</TH><TD>+</TD><TD>+</TD></TR><TR><TH scope="row">DARPin cl. 3</TH><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">DARPin cl. 8</TH><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Cup1-1 cl. 3</TH><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Cup1-1 cl. 8</TH><TD>-</TD><TD>-</TD></TR></TBODY></TABLE></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="21.07.2014">21.07.2014</A></H2><FIELDSET class="exp13"><LEGEND><A name="exp13.70">13.70 Screening clones of Gibson plates</A></LEGEND><DIV class="aim"><P>Aim: checking success of Gibson assembly</P></DIV><DIV class="exp-content"><P>On the plate with 1:2 ratio have been 2 clones, on the 1:3 ratio plate have been 3 clones. All clones were picked for inoculation of minipreps (5 mL LB-Amp, incubation overnight).</P></DIV></FIELDSET><FIELDSET class="exp"><LEGEND><A name="exp_15">Sequencing cPCR Products from <I>B. subtilis</I> mutants with Flank fw/rv</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Premix</TH><TH scope="col">Label-Nr.</TH><TH scope="col">Construct</TH><TH scope="col">Primer</TH></TR><TR><TH scope="row">1</TH><TD>AGB00234<STRONG>52</STRONG></TD><TD><I>B. subtilis</I> - Hag-<I>Spe</I>I Cl.7</TD><TD>Flo89 Flank-fw</TD></TR><TR><TH scope="row">2</TH><TD>AGB00234<STRONG>53</STRONG></TD><TD><I>B. subtilis</I>  - Hag-<I>Spe</I>I Cl.7</TD><TD>Flo90 Flank-rv</TD></TR><TR><TH scope="row">3</TH><TD>AGB00234<STRONG>54</STRONG></TD><TD><I>B. subtilis</I> - Hag-Cup1-1 Cl.2</TD><TD>Flo89 Flank-fw</TD></TR><TR><TH scope="row">4</TH><TD>AGB00234<STRONG>55</STRONG></TD><TD><I>B. subtilis</I> - Hag-Cup1-1 Cl.2</TD><TD>Flo90 Flank-rv</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.62">14.62 Transformation of BL21 cells</A></LEGEND><DIV class="aim"><P>Aim: Transform <I>E. coli</I> BL21(DE3) with Arc1p-C-single-pET28a</P></DIV><DIV class="exp-content"><P>40 µL <I>E. coli</I> BL21(DE3) electrocompetent cells were transformed with 1 ng of the previously purified plasmid Arc1p-C-single-pET28a from two different clones, incubated for 1 h at 37 °C and plated out on LB-Kan<SUP>50</SUP> plates that were incubated overnight at 37 °C.</P></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.9">19.9 Protein overexpression with transformed <I>E. coli</I> BL21 (DE3) (piGEM-021)</A></LEGEND><DIV class="aim"><P>Aim: Overexpression of flagellin modifications for a second crystallization attempt</P></DIV><DIV class="exp-content"><P>After freshly packed GeFi Columns we decided to purify the Fla-Cup again for crystallization.</P><P>For that purpose 2 x 1 L LB were inoculated with a solution of LB and  every clone on the cotransformation plate. All clones from the piGEM-021 +  FlaiS cotransformation plates were picked and resuspended in 1 mL LB. 500 µL  were added to the 1 L culture flasks. Each flask was additionally mixed with 1  mL ampicillin (100 mg/ mL) and 1 mL canamycin (50 mg/ mL).</P><P>The expression was induced overnight with lactose (12,5 g/ L). 25 g  lactose were solved in 100 mL millipore water which was heated 1 min in the  microwave for solvation. Each culture was mixed with 50 mL of the lactose/  water suspension.</P><P>The cultures were incubated overnight at 30°C.</P></DIV></FIELDSET><FIELDSET class="exp21"><LEGEND><A name="exp21.3">21.3 Expression of Flagellin</A></LEGEND><DIV class="aim"><P>Aim: Check the expression of the modified flagella of Bacillus mutants</P></DIV><DIV class="exp-content"><P>In order to check the expression of  the flagellum in the Bacillus mutants cultures  in 100 mL LB medium were grown until an OD of 0,8 at 37°C. 10 mL culture were centrifuged  at 4000 rpm for 10 min to get the pellet. The pellet was resuspended in 60 µL  water and 40 µl SDS-loading buffer. After that the sample was analysed via  SDS-PAGE. Overexpressed <I>E. coli</I> flagellin was used as a positive control. The analysed samples should run on the same level as the ones produced in <I>E. coli</I> samples in case of a positive expression.</P><P>The gel showed that the Hag-<I>Spe</I>I mutant generates a band on the same level as the WT3610 which is a positive result because the modified flagellin with <I>Spe</I>I site has no remarkable change of molecule mass. The Bacillus transformation was successful though.</P><P>The mutants with the inserted Cup and DARPin  showed no significant expression of the flagellin. Either the export of the flagellin  is not possible for Bacillus because of the size/ folding of our constructs and  ends in degradation of our modifications or the expression is too low to be  analysed via SDS-PAGE. In that case it would not be efficient enough for further usage. The results of the SDS-PAGE fits to the  previously made Swarming assays. </P><P>A new strategy had to be planned for further  work.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="22.07.2014">22.07.2014</A></H2><FIELDSET class="exp"><LEGEND><A name="exp_16">Sequencing cPCR Products from <I>B. subtilis</I> mutants with Flank fw/rv</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Premix</TH><TH scope="col">Label-Nr.</TH><TH scope="col">Construct</TH><TH scope="col">Results</TH></TR><TR><TH scope="row">1</TH><TD>AGB00234<STRONG>52</STRONG></TD><TD><I>B. subtilis</I>  - Hag-<I>Spe</I>I Cl.7</TD><TD> </TD></TR><TR><TH scope="row">2</TH><TD>AGB00234<STRONG>53</STRONG></TD><TD><I>B. subtilis</I>  - Hag-<I>Spe</I>I Cl.7</TD><TD> </TD></TR><TR><TH scope="row">3</TH><TD>AGB00234<STRONG>54</STRONG></TD><TD><I>B. subtilis</I> - Hag-Cup1-1 Cl.2</TD><TD> </TD></TR><TR><TH scope="row">4</TH><TD>AGB00234<STRONG>55</STRONG></TD><TD><I>B. subtilis</I> - Hag-Cup1-1 Cl.2</TD><TD> </TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.9">19.9 Protein overexpression with transformed <I>E. coli</I> BL21 (DE3) (piGEM-021)</A></LEGEND><DIV class="aim"><P>Aim: Overexpression of flagellin modifications for a second crystallization try</P></DIV><DIV class="exp-content"><P>The  overnight cultures were grown till an OD of 2-3 and centrifuged. The pellets  were frozen away at -80°C.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.70">13.70 screening clones of Gibson plates</A></LEGEND><DIV class="aim"><P>Aim: checking success of Gibson assembly</P></DIV><DIV class="exp-content"><P>The isolated plasmids were used for PCR analysis. The primers iGEM-27 and -30 were used for a PCR to amplify the whole mCherry with a size of ca. 711 bp.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix</TH><TH scope="col">Mastermix</TH><TH scope="col">PCR attempt</TH></TR><TR><TH scope="row">Template: plasmid from clones 1:10 </TH><TD>-</TD><TD>0,5</TD></TR><TR><TH scope="row">Primer iGEM-027</TH><TD>3</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-030</TH><TD>3</TD><TD>-</TD></TR><TR><TH scope="row">dNTP-mix</TH><TD>3</TD><TD>-</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>3</TD><TD>-</TD></TR><TR><TH scope="row">Phusion Buffer 5x</TH><TD>36</TD><TD>-</TD></TR><TR><TH scope="row">MgCl2</TH><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">DMSO</TH><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Water</TH><TD>132</TD><TD>-</TD></TR><TR><TH scope="row">Mastermix</TH><TD>-</TD><TD>19,5</TD></TR><TR><TH scope="row">Total Volume</TH><TD>180</TD><TD>20</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1 min </TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>30x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The gel shows that clone 1:3.2 might be positive for containing the mCherry fragment. The clone has to be sequenced.</P></DIV></FIELDSET><FIELDSET class="exp21"><LEGEND><A name="exp21.2b">21.2 Swarming Assay</A></LEGEND><DIV class="aim"><P>Aim: Check motility of Bacillus transformands with D2-3-domain</P></DIV><DIV class="exp-content"><P>The mutants showed expression on the SDS-gel. Swarming and swimming were tested to  check the motility of his mutants for further planning of our domain  constructions. The hag-construct contains the D2-3 domain of Salmonella.</P><P>WT3610  and the D2-3-mutant were used to inoculate 50 mL LB-medium for overnight culture  incubated at 37&amp;degC.</P></DIV></FIELDSET><FIELDSET class="exp"><LEGEND><A name="exp_17">Sequencing cPCR Products from <I>B. subtilis</I> mutants with flank fw/rv</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Premix</TH><TH scope="col">Label-Nr.</TH><TH scope="col">Construct</TH><TH scope="col">Results</TH></TR><TR><TH scope="row">1</TH><TD>AGB00234<STRONG>52</STRONG></TD><TD><I>B. subtilis</I>  - Hag-<I>Spe</I>I Cl.7</TD><TD>Correct</TD></TR><TR><TH scope="row">2</TH><TD>AGB00234<STRONG>53</STRONG></TD><TD><I>B. subtilis</I>  - Hag-&gt;SpeI Cl.7</TD><TD>Correct</TD></TR><TR><TH scope="row">3</TH><TD>AGB00234<STRONG>54</STRONG></TD><TD><I>B. subtilis</I> - Hag-Cup1-1 Cl.2</TD><TD>Correct sequencing  till Hag</TD></TR><TR><TH scope="row">4</TH><TD>AGB00234<STRONG>55</STRONG></TD><TD><I>B. subtilis</I> - Hag-Cup1-1 Cl.2</TD><TD>Correct sequencing  till Hag</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.63">14.63 Overnight culture</A></LEGEND><DIV class="aim"><P>Aim: Inoculate Overnight culture of Arc1p-C-single-pET28a</P></DIV><DIV class="exp-content"><P>5 mL of LB-Kan<SUP>50</SUP> medium were inoculated with a clone from 14.62 bearing the Arc1p-C-single-pET28a plasmid.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="23.07.2014">23.07.2014</A></H2><FIELDSET class="exp14"><LEGEND><A name="exp14.64">14.64 Test Expression</A></LEGEND><DIV class="aim"><P>Aim: Test the expression of Arc1p-C-single</P></DIV><DIV class="exp-content"><P>Using the preculture from 14.63 60 mL of LB-Kan<SUP>50</SUP> medium were inoculated with 600 µL of the preculture and incubated at 37 °C and 220 rpm until OD<SUB>600</SUB> reached 0.5. The culture was cooled down to 18 °C and the protein production was induced with 12 µL of 500 mM IPTG. Over night the culture was incubated at 18 °C and 220 rpm. Afterwards the cell culture was centrifuged (17000 rpm, 20 min, 4 °C), the pellet resuspended in 3 mL buffer A and stored at -20 °C. A glycerolstock of the strain containing the Arc1p-C-single-pET28a plasmid was stored at -80 °C.</P></DIV></FIELDSET><FIELDSET class="exp19"><LEGEND><A name="exp19.10">19.10 Purification of flagellin-Hag-Cup1-1 from frozen pellet</A></LEGEND><DIV class="exp-content"><P>The  pellets which were stored at -80°C were resuspended in 20 mL buffer A and  cracked with the micro fluidizer. The cell parts were centrifuged at 20000 rpm  for 20 min at 4°C. The supernatant was transferred to a new 50 mL falcon and  stored on ice.</P><P>The following steps were performed:</P><UL class="list"><LI>equilibration with Buffer A 10 min</LI><LI>taking 40 µL supernatant  (load)+ 10 µL SDS-Buffer - L-Sample</LI><LI>50 mL load on column</LI><LI>taking 40 µL of flow through + 10 µL SDS-buffer - FT-Sample</LI><LI>first washing with 25ml Buffer A (half the load)</LI><LI>taking 40 µL of washing flow through + 10 µL SDS-Buffer - W-Sample</LI><LI>equilibrate with Buffer B (output pipe off the column into glas, input into B)</LI><LI>hanging pipe on column, 6 x 2 ml elutions - E 1-6</LI><LI>taking 40 µL of elution 1-6 + 10 µL SDS-Buffer E-sample 1-6</LI></UL><P>regeneration of column:</P><UL class="list"><LI>10min water</LI><LI>10min EDTA</LI><LI>10min water</LI><LI>10min NiSO4</LI><LI>10min water</LI></UL><P>Meanwhile  the Elutions 1 - 6 were pooled (combined 12 mL) in a 50 mL falcon and transferred  into a concentrator column which was centrifuged until the volume in the filter  was 1,5 mL for injection into the gel filtration station. </P><P>Parallel  the L-, FL-, W- and E 1 - 6 -samples were analyzed on a SDS-PAGE in order to  proof the purification in the elution fractions.</P><P>The gel  shows a huge protein concentration in E1 and E2 so that the purification could  be continued with gel filtration. For that purpose the filtrate  out of the concentrator was transferred into a new 2 mL tube resuspending very  well without air bubbles. </P><P>20 mL  of GeFi-buffer were injected into the 2 mL loop. After that the protein filtrate  was injected as well. </P><P>The  GeFi was started according to the following settings:</P><UL class="list"><LI>Flow rate: 2500-3000 µL / min</LI><LI>max. pressure: 0,60 - 0,65 mPa</LI><LI>First endtimer at 80-90 mL volume before fractioning</LI><LI>Injectvalve</LI></UL><P>After  observing a peak 40 µL of different fractions (C7 , C9, C11, D1,  D3, D5, D7, D9) covering the peak were taken for analysis via SDS-PAGE.</P><P>The gel  shows that the elutions contain the protein concentrated. The Elutions C7-D9  were pooled and transferred to a new concentrator until an end volume of 400 µL.</P><P>The  concentrated protein had a pink color. The concentrated protein (38 mg/mL A280)  was used for crystallization pure and in a 1:2 dilution with GeFi buffer.</P><P>Core I  - IV were used for setting drops.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="24.07.2014">24.07.2014</A></H2><FIELDSET class="exp"><LEGEND><A name="exp_18">Sequencing </A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Premix</TH><TH scope="col">Label-Nr.</TH><TH scope="col">Construct</TH><TH scope="col">Primer</TH></TR><TR><TH scope="row">1</TH><TD>AGB00234<STRONG>56</STRONG></TD><TD>piGEM-002 Nose +  mCherry 1:3 cl.2</TD><TD>TW260</TD></TR><TR><TH scope="row">2</TH><TD>AGB00234<STRONG>57</STRONG></TD><TD>Hag-Cup1-1 cl. 2</TD><TD>Flo90 Flank-rv</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.71">13.71 Insertion of degradation tags into Nose- mCherry (piGEM-022 in case of pos. seq.)</A></LEGEND><DIV class="aim"><P>Aim: insertion of degradation Tags</P></DIV><DIV class="exp-content"><P>For insertion of the DTs via gibson assembly the newly generated piGEM-022 had to be linearized by digestion with <I>Spe</I>I:</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Volume (µl)</TH></TR><TR><TH scope="row">piGEM-022 (500 ng/µL)</TH><TD>4</TD></TR><TR><TH scope="row">CutSmart 10x</TH><TD>2</TD></TR><TR><TH scope="row"><I>Spe</I>I</TH><TD>0,2</TD></TR><TR><TH scope="row">Water</TH><TD>13,8</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD></TR></TBODY></TABLE><P>Incubation at 37°C for 2 hours.</P><P>Meanwhile the DTs were generated by a PCR:</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix</TH><TH scope="col">Master Mix</TH><TH scope="col">PCR attempt DT 21</TH><TH scope="col">PCR attempt DT 22</TH><TH scope="col">PCR attempt DT 23</TH></TR><TR><TH scope="row">Template </TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-021</TH><TD>-</TD><TD>0,5</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-022</TH><TD>-</TD><TD>-</TD><TD>0,5</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-023</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>0,5</TD></TR><TR><TH scope="row">Primer iGEM-031</TH><TD>2</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">dNTP-mix</TH><TD>2</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>2</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Phusion Buffer 5x</TH><TD>16</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">MgCl2</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">DMSO</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Water</TH><TD>48</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Mastermix</TH><TD>-</TD><TD>19,5</TD><TD>19,5</TD><TD>19,5</TD></TR><TR><TH scope="row">Total Volume</TH><TD>80</TD><TD>20</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>20 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>30x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The DTs were integrated into the vector by Gibson assembly:</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Control</TH><TH scope="col">DT.21</TH><TH scope="col">DT.22</TH><TH scope="col">DT.23</TH></TR><TR><TH scope="row">Gibson-mix</TH><TD>15</TD><TD>15</TD><TD>15</TD><TD>15</TD></TR><TR><TH scope="row">Insert I (DT.x 600 ng/µL)</TH><TD>-</TD><TD>2,5</TD><TD>2,5</TD><TD>2,5</TD></TR><TR><TH scope="row">piGEM-002 <I>Nco</I>/<I>Spe</I>I (130 ng/µL)</TH><TD>2,5</TD><TD>2,5</TD><TD>2,5</TD><TD>2,5</TD></TR><TR><TH scope="row">Water</TH><TD>2,5</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Total Volume</TH><TD>17,5</TD><TD>20</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><P>The reactions  were incubated at 50°C for an hour and used to transform <I>E. coli</I> XLI-blue that were plated out on LB-Amp. Incubation was carried out over night.</P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.65">14.65 Purification of test expression</A></LEGEND><DIV class="aim"><P>Aim: Purify Arc1p-C-single on a small scale</P></DIV><DIV class="exp-content"><P>Lysozyme was added to the cell suspension from 14.64 to a final concentration of 1 mg/mL and incubated on ice for 30 min. Afterwards the cells were lysed using ultra sound (2x 18 pulses, 30 s on ice inbetween). To clear the suspension of cell debris it was centrifuged (13000 rpm, 15 min, 4 °C). The supernatant was purified using QIAGEN Ni-NTA Spin Columns. An SDS-PAGE gel showed that the protein is produced and can be purified using Ni affinity chromatography.</P><P>60 mL LB-Kan<SUP>50</SUP> medium was inoculated from the glycerolstock prepared in 14.64 and incubated at 37 °C and 220 rpm over night.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="25.07.2014">25.07.2014</A></H2><FIELDSET class="exp14"><LEGEND><A name="exp14.66">14.66 Expression of Arc1p-C-single</A></LEGEND><DIV class="aim"><P>Aim: Express Arc1p-C-single for further use</P></DIV><DIV class="exp-content"><P>10 x 500 mL LB-Kan<SUP>50</SUP> medium were inoculated with 5 mL of the overnight culture prepared in 14.65 and incubated at 37 °C and 220 rpm until OD<SUB>600</SUB> reached 0.5. The flasks were cooled to 18 °C and protein production was induced by adding 100 µL 500 mM IPTG. The cultures were incubated over night at 18 °C and 220 rpm. Afterwards the cells were collected by centrifugation (6000 rpm, 20 min, 4 °C), resuspended in buffer A and stored at -20 °C until further use.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.72">13.72 Checking plates of Nose-mCherry-Dt containing clones from Gibson assembly</A></LEGEND><DIV class="aim"><P>Aim: checking success of DT 21- 23 integration in Nose-mCherry (piGEM-022)</P></DIV></FIELDSET><FIELDSET class="exp"><LEGEND><A name="exp_19">Sequencing results </A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Premix</TH><TH scope="col">Laber-Nr.</TH><TH scope="col">Construct</TH><TH scope="col">Result</TH></TR><TR><TH scope="row">1</TH><TD>AGB00234<STRONG>56</STRONG></TD><TD>piGEM-002 Nose +  mCherry 1:3 cl.2</TD><TD>2 point mutations, 1  silent, Phe -&gt; Ile</TD></TR></TBODY></TABLE></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="28.07.2014">28.07.2014</A></H2><FIELDSET class="exp14"><LEGEND><A name="exp14.67">14.67 Purification of Arc1p-C-single</A></LEGEND><DIV class="aim"><P>Aim: Purify Arc1p-C-single in three steps</P></DIV><DIV class="exp-content"><P>The resuspended cells from 14.66 were lysed using a french press. The lysate was incubated on ice with 5 µg/mL DNAseA and RNAseI for 10 min and centrifuged (27000 rpm, 45 min, 4 °C) to pellet cell debris. The supernatant was filtered (0.45 µm) and PheA-Arc1p-C-0x purified from the cleared solution by NiNTA affinity chromatography. The combined elution fractions containing crude PheA-Arc1p-C-0x were concentrated to a volume of 2 mL and further purified using an anion exchange column to get rid of possible tRNA contaminations. Finally the buffer was changed to dialysis buffer and the protein concentrated and stored at -80 °C in aliquods for further use.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="31.07.2014">31.07.2014</A></H2><FIELDSET class="exp19"><LEGEND><A name="exp19.11">19.11 crystallization of Hag-Cup</A></LEGEND><DIV class="exp-content"><P>Protein crystals can be seen in the wells of the first Hag-Cup crystallization attempt from 19.9</P></DIV></FIELDSET></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Marburg:Project:Notebook:July">http://2014.igem.org/Team:Marburg:Project:Notebook:July</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Marburg:Project:Notebook:July" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Marburg:Project:Notebook:July" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Marburg:Project:Notebook:July&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Marburg:Project:Notebook:July&amp;oldid=316074" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>