"
Page
Discussion
View source
History
teams
Log in
 
Team:Marburg:Project:Notebook:June
From 2014.igem.org
Home
Project
Safety
Team
Policy & Practices
Attributions
SilverSURF
CancerSURF
RiboSURF
Notebook
Parts
Notebook: June
March
April
May
June
02.06.2014
03.06.2014
04.06.2014
05.06.2014
06.06.2014
07.06.2014
08.06.2014
09.06.2014
10.06.2014
11.06.2014
12.06.2014
13.06.2014
14.06.2014
15.06.2014
16.06.2014
17.06.2014
18.06.2014
19.06.2014
20.06.2014
21.06.2014
22.06.2014
23.06.2014
24.06.2014
25.06.2014
26.06.2014
27.06.2014
28.06.2014
29.06.2014
30.06.2014
July
August
September
October
Methods
02.06.2014
13.56 Making glycerine stocks of the following strains
Aim: Storage of strains
Dh5α piGEM010 (p003 + Tag 21)
Dh5α piGEM015 (p004 + Tag 23)
Dh5α piGEM014 (p003 + Tag 22)
Dh5α piGEM008 (p002 + Tag 22)
Dh5α piGEM007 (p002 + Tag 21)
Dh5α piGEM011 (p003 + Tag 22)
Dh5α piGEM012 (p003 + Tag 23)
13.57 Mini Prep of  cultures inoculated on Friday 30.05.14 containing the nose plasmids
piGEM010 = 22ng/µl
piGEM013 = 46 ng/µl
piGEM 009 = 13 ng/µl
low concentration: new transformation to obtain more plasmid DNA
Strain: new competent E. coli XL1-blue
15.50 Miniprep of pMAD-fla-cup1-1  for miniprep
The miniprep was performed according to the miniprep protocol.
Concentration 40 ng/µL
low concentration: new transformation to gain more plasmid DNA should be done
Strain: new competent E. coli XL1-blue
13.58 Bacillus Trafo with all Nose Plasmids containing one/no Degradation Tag
Aim:  Create Bacillus subtilis strain 3610 containing the following plasmids: piGEM002 to 004 and piGEM007 to 016
Add 5µl plasmid DNA and incubate for 30 min at 37°C, LB plate with 5 ng/µL Cm (prepared freshly)
13.59 Inoculation of Bacillus subtilis wildtype (3610), gfp and silver nose
Aim: perform MTA to test the Ag concentrations where cells are still viable, since 1 mM until 10 µM were too high and the cells died
Starting from 1 mM, eight serial dilutions have been prepared
14.38 Purification of PheA-Arc1p-C-1x
Aim: Purify PheA-Arc1p-C-1x in three steps
The resuspended cells from 14.37 were lysed using a french press. The lysate was incubated on ice with 5 µg/mL DNAseA and RNAseI for 10 min and centrifuged (27000 rpm, 45 min, 4 °C) to pellet cell debris. The supernatant was filtered (0.45 µm) and PheA-Arc1p-C-1x purified from the cleared solution by NiNTA affinity chromatography. The combined elution fractions containing crude PheA-Arc1p-C-1x were concentrated to a volume of 2 mL and further purified using an anion exchange column to get rid of possible tRNA contaminations. Finally the buffer was changed to dialysis buffer and the protein concentrated and stored at -80 °C in aliquods for further use.
18.1 Tumor Terminator - Dilution of Plasmid
Aim: Creating a stock solution
We received the synthetized DARPin-gene in a pEC-A2 vector with SpeI restriction sites at the 5' - and 3' -site with an amount of 2,7 µg plasmid per tube.
A stock solution was created by adding 27 µl water to the tube.
Concentration of stock solution = 100 ng/µl
A 1:10 dilution was created by using 1 µl of stock solution adding 9µL of water to an end concentration of 10ng/µl.
Concentration of stock dilution = 10 ng/µl.
18.2 Transformation of E.coli XL1-Blue with pEX-A2-DARPin
Aim: Transformation of E.coli XL1-Blue with pEX-A2-DARPin for amplification of the plasmid.
1µl of the 1:10 dilution was used for transformation of E. Coli XL1-Blue
03.06.2014
18.3 Miniprep of pEX-A2-DARPin from E.coli XL1-Blue
Aim: Receiving a higher amount of plasmid for further experiments.
3 x 6ml LB-Amp were inoculated with clones from the successful transformed plate (18.2) and incubated at 37°C overnight.
13.60 Microtiter plate assay with wildtype, gfp strain 186 and silver nose
Aim: perform MTA to test the Ag concentrations where cells are still viable, since 1mM until 10 µM was too high and the cells died
Starting from 1 mM 8 dilutions have been prepared.
In the beginning Minimalmedium S750 has been prepared with 0.5‰ Xylose
In the wells the following mix was added to a total volume of 150 µl:
135µl of MM (90‰) and Silver solution (10‰)
15µl of cells
Finally 70 µl are added to avoid evaporation of the medium
Incubation for appr. 24 hours at 37°C shaking (Viktor AG Waldminghaus)
No positive GFP signal could be detected for the B. subtilis strain with the possible Ag sensitive promoter. The strain was restreaked onto a LB-Cm plate and he did not grow, which proved it to be a wrong strain, which has not been transformed successfully.
13.61 Checking Bacillus Trafo with all Nose Plasmids containing one/no Degradation Tag
Every transformation plate showed a huge growth of bacillus without single colonies. After one day of incubation this seems to be an unrealistic growth, which is why we planned to repeat the transformation after checking the growth of the Bacillus subtilis wildtype 3610 on LB-CM plate with 5 µg/ml chloramphenicol.
13.62 Checking Bacillus Trafo Wildtype 3610
Aim: check if wildtype has a resistance against chloramphenicol or if the plates are faulty
Bacillus subtilis Wildtype 3610 was plated on LB-chloramphenicol plate with 5 µg/ml chloramphenicol and incubated overnight at 37°C.
15.51 Making selection plates for clean deletions - integration of pMAD-construct into Bacillus subtilis chromosome
Aim: For the integration of the constructs we cloned into pMAD many steps of a special selection are necessary.
Antibiotics and chemicals:
Erythromycin 4 mg/ml
Lincomycin 25mg/ml
X-Gal 100mg/ml
For MLS selection plates erythromycin with 1 mg/ml were needed. Therefore the 4 mg/ml stock solution was diluted 1:4 by mixing 400 µl eryhtromycin with 1200 µl of 70% ethanol.
3 types of plates were made:
1. MLS selectin: 800 mL LB agar + 800 µl eryhtromycin + 800 µl lincomycin
2. MLS X-Gal: 800 mL LB agar + 800 µl eryhtromycin + 800 µl lincomycin + 800 µl X-Gal
3. X-Gal: 800 mL LB agar + 800 µL X-Gal
04.06.2014
18.4 Miniprep of pEX-A2-DARPin from E.coli XL1-Blue
Plasmid pEX-A2 + DARPinEc1
Concentration after prep (ng/µl)
Clone 1
398,7
Clone 2
340
Clone 3
309
15.52 Sequencing of pMAD-fla-cup1-1
Aim: Checking the correct insertion of the metallothionein-domain into the hag-flank construct
Making a pre-mix for Eurofins MWG prepairing the sequencing. Primers from Florian Altegoer were used:
Flo83- Hag11_fw 1:10 with 1µl primer and 9µl water
Flo86- Hag12_rv 1:10 with 1µl primer and 9µl water
 
Pre-mix 1 (µL)
Pre-mix 2 (µL)
piGEM-016 pMAD-cup1-1
40ng/µL
10
10
Flo83- Hag11_fw
2
-
Flo86- Hag12_rv
-
2
Total Volume
12
12
Label
Pre-mix 1
Primer
AGB0023375
Pre-mix 1
Flo83 – Hag11_fw
AGB0023374
Pre-mix 1
Flo86 - Hag12_rv
13.62 Checking Bacillus Trafo Wildtype 3610
Aim: check if wildtype has a resistance against chloramphenicol or if the plates are not effective/ made wrong
Result: Bacillus subtilis Wildtype 3610 grew on the chloramphenicol plate which shows us that there has to be a mistake in making the LB-chloramphenicol plates or the WT has a resistance.
New plates were made with new chloramphenicol-stocks with 25 mg/ml and 5 mg/ml.
3 aliquots of competent B.s. WT3610 were plated on new LB-chloramphenicol plates and incubated at 37°C.
18.5 SpeI restriction pEX-A2-DARPin & pMAD-hag-flank (piGEM-005)
Aim: Receiving iGEM-005 as linearized vector for cloning the DARPin gene with SpeI sites into pMAD-fla (iGEM-005)
Restriction
pEX-A2-DARPin  digest (µl)
pMAD-Hag-Flank  digest (µl)
Spe1-Mastermix
pEX-A2+DARPin (398 µg/µL)
15 = ca. 6µg
-
-
pMAD-Fla (114 µg/µL)
-
10= ca. 1,14 µg
-
SpeI
-
-
1,5
CutSmart-Buffer (10x)
-
-
2
H2O
-
-
16,5
Mastermix
5
10
-
Total
20
20
20
Incubation at 37°C for 2h and heat shock at 81°C for 5 min. After that the pMAD-hag-flank-construct was purified with the OMEGA gel extraction kit according to the protocol.
c(digested piGEM-005)= 4 ng/µl
13.62 Checking Bacillus Trafo Wildtype 3610
Aim: check if wildtype has a resistance against chloramphenicol or if the plates are not effective/ made wrong
Result: Bacillus subtilis Wildtype 3610 grew on the chloramphenicol plate. We have already repeated making plates so it might be that the WT has got a resistance while making it competent. A new WT was plated on 2 chloramphenicol plates with 5 µg/mL (old and new one) and 2x LB plates from gly stocks.
Incubation at 37°C overnight.
05.06.2014
14.39 Creating PheA-Arc1p-C-0x
Aim: Using Round-the-horn mutagenesis to create PheA-Arc1p-C-0x-pET28a
The PCR with 5' phosphorylated TG_0xGSSG FP und TG_0xGSSG RP should generate the 7491 bp fragment for ligation from the methylated template PheA-Arc1p-C-8x-pET28a from 14.5.
The used Primer concentration was 1 µM, the concentration of the template was 1 ng/µL and we used a hot start to avoid unspecific annealing of primers.
Content
Volume [µL]
Water
29.5
5x GC Buffer
10
DMSO
2.5
TG_0xGSSG FP
2.5
TG_0xGSSG RP
2.5
PheA-Arc1p-C-8x-pET28a
1
dNTPs
1
Phusion-Polymerase
1
Total Volume
50
Step
Temperature [°C]
Time [min:sec]
1
98
2:00
2
add Polymerase
 
3
98
0:10
4
67.5
0:30
5
72
4:00
6
go to 3
32x
7
72
10:00
8
4
hold
The amplified fragment was purified on an agarose gel on which it showed the correct size. The extracted linear template was then digested to degrade the original unmutated plasmid using DpnI.
Content
Volume [µL]
10x CutSmart Buffer
3.6
Plasmid
30
DpnI
3
Total Volume
36.6
The reaction mixture was incubated at 37 °C and 300 rpm for 1 h and afterwards heated to 80 °C for 20 min.
In order to create the circular plasmid 100 ng of the linear template were mixed with 1 µL 10x T4-Ligasebuffer,  1 µL T4-Ligase and water to give a total volume of 10 µL. This mixture was incubated for 9 h at 16 °C. The resulting plasmid was dialysed with water to decrease the salt concentration for the following transformation of E. coli Top10 electrocompetent cells with the plasmid.
06.06.2014
19. Crystallization of flagellin
In order to crystallize flagellin with a domain it is necessary to produce the protein by E. coli in a high concentration.  For that aim the Fla-construct will be amplified out of pET24d which includes a domain. For elimination of the flanks a PCR has to be performed which just amplifies hag with domain and creates NcoI and BamHI restriction sites. After purification and digest with Nco/Bam the fragment can be ligated into digested pET24d . After transformation the production can be induced by IPTG.
19.1 Elimination of flanks from fla-construct including cup1-1 from pET24d
Aim: Preparation for using pET24d as expression vector for flagellin
The flagellin construct was  amplified with specifically designed primers (95 and 54 from Florian Altegoer), which allow the  amplification of the construct with cup1-1 without flanks (just Hag) including NcoI & BamHI restriction sites.
Template dilution 1:10:
1 µL pET24d-cup1-1 was mixed with  9 µL to an 1:10 dilution of 16,3 ng/µL
Primer dilution 1:10:
The primers were diluted 1:10 by mixing 10 µL of primers with 90 µL millipore water  each.
Mix [µl]
reaction with phusion
reaction with Q5 mastermix
pET24d-cup1-1
1:10 dilution
1
1
Primer 54Flo
1
1
Primer 56Flo
1
1
Phusion
1
-
Phusion Buffer 5x
10
-
dNTP Mix
1
-
Water
35
22
Q5 Mastermix
-
25
Total Volume
50
20
Step
Temperature °C
Time
1
98
3 min
2
98
30 sec
3
55
30 sec
4
72
2 min
5
Go To 2
30x
6
72
4min
7
8
infinite
The PCR was run over night.
19.2 Restriction digest of pET24d with NcoI and BamHI
Aim: Creating Nco/Bam restriction sites in pET24d for cloning PCR fragments amplified with primers flo56 and flo54 which contain these sites as well.
Isolated pET24d was cut with NcoI and BamHI in order to create restriction sites for ligating the Flagellin/ Hag-Domain fragments into the vector. The Hag-fragments which were amplified via PCR with primers flo54 and 56 contain NcoI/ BamHI restriction sites for that purpose as well.
Component
Volume [µl]
pET24d (148 ng/µL)
7
CutSmart 10x
2
NcoI
0,5
BamHI
0,5
Water
10
Total Volume
20
Incubation for 1 hour at 37°C and heat shocked at 80°C.
15.52 Miniprep of different plasmids
Aim: Receiving more plasmid DNA for further experiments
Colonies were picked from plates and 8ml LB with necessary antibiotics each inoculated, followed by incubation at 37°C overnight.
Clones were picked from plates with plasmids:
pET24d-Fla-Cup1-1 (piGEM-006)
pMAD-Hag-Flank (piGEM-005)
pMAD-Hag-Flank -Cup1-1 (piGEM-016)
pEX-A2+DARPin (piGEM-017)
07.062014
15.53 Sequencing pMAD-fla-cup1-1
The sequencing showed the correct insertion of the cup-1-1 gene into the hag-flank construct but contained a stop codon, which was not removed by the used reverse primer. Unfortunately we used primers for amplification of the whole cup1-1 gene and not just for the truncated gene coding  for the metallothionein domain only. Therefore new primers had to be designed an the Gibson assembly for integration of the metallothionein domain into pMAD and pET24d has to be repeated.
18.6 gel extraction & purification of DARPin gene
Aim: Receiving DARPin gene with SpeI sites for ligation into linearized piGEM-005
The with SpeI digested pEX-A2 was purified via electrophoresis with an 1% agarose gel in order to divide pEX-A2 backbone and the DARPin gene (489 bp). The digest attempt was splitted and loaded into two pockets to be sure not to overload the gel. The correct bands were cut out and the DNA fragment was rescued according to the protocol of our OMEGA gel extraction kit.
Expected bands:
DARPin gene: 489 bp
pEX-A2 backbone: 1961bp
The concentration after gel  extraction: 9 ng/µl
18.7 Ligation of DARPin gene and linearized piGEM-005
Aim: Ligation of DARPin- (from 18.6) into Spe1 digested piGEM-005 from 18.5 for integration of the DARPin-Flagellin construct into Bacillus subtilis chromosome
The DARPin gene with SpeI sites was used as insert for the ligation into linearized pMAD (from 18.5) which contains the designed restriction site. The ligation attempt was done twofold and calculated by the Ligation Calculator from the iGEM-Team Texas in 2011:
Component
Volume [µl]
Insert (DARPin, c= 9 ng/µL)
2
Vector (piGEM-005, c=4ng/µL)
10
Ligase-Buffer 10x
2
T4 Ligase
1
Milipore water
5
Total Volume
20
Incubation was performed overnight at room temperature.
13.63 checking Bacillus WT 3610  growth on 5µg/µL CM
Aim: check if wildtype has a resistance against chloramphenicol (CM)
Result: Bacillus subtilis Wildtype 3610 grew on the CM plate.
In order to make out if a selection is possible WT3610 was plated out  from an LB plate (13.62)  on 25 ng/ µL CM  to check were the threshold of CM for B. subtilis is.
Incubation at 30°C over two days.
08.06.2014
18.8 new SpeI restriction of DARPin-gene in pEX-A2 & pMAD-hag-flank (piGEM-005)
Aim: Receiving new insert and vector in case of an unsuccessful ligation for further attempts.
Restriction
pEX-A2-DARPin digest (µl)
pMAD-Hag-Flank digest (µl)
SpeI-Mastermix
pEX-A2+DARPin (398 µg/µL)
10 = ca. 4µg
-
-
pMAD-Fla (150 µg/µL)
-
7= ca. 1 µg
-
SpeI
-
-
1
CutSmart-Buffer (10x)
-
-
2,5
Water
-
-
21,5
Mastermix
10
13
-
Total Volume
20
20
25
Incubation was performed for 2 h at 37°C and heat inactivated at 80°C for  5 minutes.
18.9 Transformation of ligation from 18.7
The whole 20µL attempt from the ligations were used to transform E.Coli  XL1-Blue, which were plated on LB-Amp plates.  Additionally a negative control was created by transforming E.Coli XL1-Blue with 1 µL SpeI cut  piGEM-005 as  well.
18.10 Gel extraction for isolation of DARPin gene out of pEX-A2
Aim: Receiving new insert in case of an unsuccessful ligation for further attempts like in 18.6
The SpeI digested pEX-A2 was purified via electrophoresis with an 1% agarose gel like in 18.6. The correct bands were cut out and the DNA fragment was purified according to the  protocol of our OMEGA gel extraction kit.
Expected bands:
DARPin gene: 489 bp
pEX-A2 backbone: 1961bp
The concentration after gel extraction: 10 ng/µl
09.06.2014
18.12 Checking transformation of overnight ligation from 18.7
Plate
Amount of colonies
Negative control
6
Ligation attempt 1
7
Ligation attempt 2
0
18.13 Checking success of ligation via cPCR
Aim: A cPCR with the primers 54 and 56 shows the success of ligation by amplification of the Hag-construct.
In order to check the positive  insertion of the DARPin gene into pMAD a PCR amplification with primers 54 and  56 for the flagellin should generate an ca. 1500 bp fragment on the gel.  Negative clones would just show a 1000 bp band because of the  amplification of the Hag-SpeI site construct. The picked  clones were transferred on a new LB-amp plate.
Mix
Master-Mix for 7 attempts (µL)
Mix for PCR attempt (µL)
Colony
-
part of a colony
Primer Flo 54
3,5
-
Primer Flo 56
3,5
-
Phusion
7
-
Phusion Buffer 5x
28
-
dNTP Mix
3,5
-
Water
94,5
-
Mastermix
-
20
Total Volume
140
20
Expected bands:
Hag-DARPin-construct: ca. 1500 bp
The gel shows bands at 1000 which means that the clones contain just relegated pMAD vector and are all negative. The ligation had to be repeated.
18.14 Dephosphorylation of linearized piGEM-005
Aim: Dephosphorylation of 5'-end with antarctic phosphatase in order to decrease the religation of the linearized piGEM-005 from 18.8
The cut SpeI cut piGEM-005 was dephosporylated at the 5'-end with antarctic phosphatase to prevent the vector from relegation.
Component
Volume [µl]
Vector (piGEM-005, c=148  ng/µL)
20 (=ca. 2 µg)
Antarctic phos.-Buffer 10x
3
Antarctic phosphatase
1
Milipore Water
6
Total Volume
30
The sample was incubated for 1 h at 37°C at heat inactivated at 70°C for  10 minutes.
18. new ligation of linearized/ dephosphorylized piGEM-005 & DARPin gene
Aim: Dephosphorylation piGEM-005 was ligated with extracted DARPin gene again
Because of the negative colony PCR from 18.13 a new ligation with dephosphorylized piGEM-005 from 18.14 and new extracted DARPin gene from 18.10 was performed in 2 attempts.
Component
Volume [µl]
Insert (DARPin, c= 10  ng/µL)
12,4
Vector (piGEM-005, c= 148  ng/µL)
4,6
Ligase-Buffer 10x
2
T4 Ligase
1
Milipore Water
5
Total Volume
20
Incubation 3h at room temperature
18.16 Transformation of overnight ligation from 18.15
The whole 20 µL reaction of the ligations (18.15) were used to transform E. coli XL1-Blue and plated on LB-Amp plates.  Additionally a negative control was created by transforming E.Coli XL1-Blue with 1µL SpeI digested  piGEM-005 as  well.
13.64 checking Bacillus WT 3610 growth on 5, 10 & 15 µg/µL CM
Aim: check if wildtype has a resistance against CM
Result: Bacillus subtilis Wildtype 3610 did not grow on the 25 ng/µl CM plate
In order to check at which concentration CM Bacillus cannot survive he  was plated out from an LB plate on LB-CM plates with different CM  concentrations (5, 10, 15 ng/mL).
Incubation at 30°C over night.
10.06.2014
18.17 Checking Transformation of ligation from 18.16
Plate
Amount of colonies
Negative control
1
Ligation attempt 1
2
Ligation attempt 2
>80
18.18 Checking success of ligation via cPCR
Aim: A cPCR with the primers 54 and 56 shows the success of ligation by amplification of the Hag-construct.
In order to check the positive  insertion of the DARPin gene into pMAD a PCR amplification with primers 54 and  56 for the flagellin should generate an ca. 1500 bp fragment on the gel.  Negative clones would show still just a 1000 bp band because of the  amplification of the Hag-Spe1 site construct. The picked  clones were transferred on a new LB-amp plate. 12 clones were picked.
Mix
Master-Mix for 14 attempts (µL)
Mix for PCR attempt (µL)
Colony
-
part of a colony
Primer Flo 54
7
-
Primer Flo 56
7
-
Phusion
14
-
Phusion Buffer 5x
56
-
dNTP Mix
7
-
Water
189
-
Mastermix
-
20
Total Volume
280
20
Step
Temperature °C
Time
1
98
3 min
2
98
30 sec
3
55
1min 45sec
4
72
1 min
5
Go To 2
30x
6
72
4min
7
8
infinite
Expected bands:
Hag-DARPin-construct: ca. 1500 bp
Clone 1, 2, 3, 4, 5, 7, 9, 11 and 12 were the positive clones at 1500 bp.
14.40 Overnight culture
Aim: Amplify Plasmid in an overnight culture
Six clones of the transformed E. coli Top10 cells from 14.39 were picked and used to inoculate overnight cultures (6 x 5 mL).
18.19 Miniprep of positive clones
Aim: In order to check the orientation of the insert in the vector of the positive clones minipreps were inoculated for further experiments
Because of the ligation via two SpeI sites, the chance is 50:50 that the clones contain the  pMAD-Hag-Flank-DARPin construct in the right orientation (Startcodon on the  Hag1-site). For test digests a higher amount of plasmid is needed, which is achieved by inoculation of minipreps.
The clones were used to inoculate  7 mL LB-Amp and incubated at 37°C over night.
13.64 checking Bacillus subtilis WT 3610 growth on 25 & 35 µg/µL CM
Aim: check if wildtype has a resistance against CM
Result: Bacillus subtilis Wildtype 3610 grew on the 5, 10 & 15µg/µl LB-CM plate.
We saw that the Bacillus grew on almost every CM concentrations except 25 µg/µL. So we transformed competent WT3610 with Nose plasmid piGEM-003 as a  positive control and plated them out on LB-CM with 25 µg/µL and 35 µg/µL.
Additionally, competent wildtype  3610 cells were transformed with piGEM-004 and plated out on LB-CM with 5, 10 and 15 µg/mL CM  to check, if these cells can be selected better.
Incubation at 30°C for 2 days.
11.06.2014
18.19 Miniprep of positive clones
The concentration of the isolated plasmids was between 100 and 150 ng/µL.
18.20 Test digest with EcoRV
Aim: Checking the right orientation of the insert in the isolated plasmids
The plasmids from the positive clones were digested with EcoRV so that specific fragments emerge when cutting into the DARPin gene and the vector backbone.
Component
Volume [µl]
Mastermix for 10 attempts [µl]
Plasmid of clone
2 (ca. 200ng)
-
Cutsmart 10x
-
8
EcoRV
-
1
Millipore water
-
71
Mastermix
8
-
Total Volume
10
80
Right orientation:    7900 – 1570 – 2500 – 55
Wrong orientation: 7900 – 1170 – 2500 – 460
Clone 1, 2, 3, 4, 7, 9 and 11 have the right orientation.
20.1 inoculation of preculture from Bacillus subtilis WT 3610
Aim: Preparation of for mainculture the next day
2 x 6 mL of LB were inoculated with Bacillus subtilis WT3610 from an LB plate and incubated at 37°C overnight.
14.41 Preparation of plasmids
Aim: Purify the amplified plasmids from overnight cultures
The plasmids were purified from the overnight cultures created in 14.40 using Sigma GenElute Plasmid Miniprep Kit following the included protocol. The plasmids were eluted using 45 µL water.
12.06.2014
18.21 PCR with pMAD-Hag-Flank-DARPin (piGEM-019) and piGEM-005
Aim: Amplification  of Hag-SpeI-construct and Hag-DARPin construct with NcoI/BamHI sites for  cloning into pET24d → crystallization of domains.
The  templates (piGEM-005, prepped pasmid from clone 7 & 12 ) were diluted 1:10.  piGEM-005 should function as a negative control. Clone 12 was picked in case  the second orientation could be used for different experiments. 5 µL PCR  product + 1 µL loading dye were analysed on an 1% agarose gel.
Mix
Volume
Template 1:10
1
Primer Flo 54
1
Primer Flo 56
1
Phusion DNA-polymerase
1
Phusion Buffer 5x
10
dNTP Mix
1
Water
35
Total Volume
50
Step
Temperature °C
Time
1
98
3 min
2
98
30 sec
3
55
1min 45sec
4
72
2 min
5
Go To 2
30x
6
72
4min
7
8
infinite
Expected  bands:
 clone 7 & 12 PCR product at ca. 1500bp – piGEM-005 PCR product at 1kb
14.42 Transformation of BL21 cells
Aim: Transform E. coli BL21(DE3) with PheA-Arc1p-C-0x-pET28a
40 µL E. coli BL21(DE3) electrocompetent cells were transformed with 1 ng of the previously purified plasmid PheA-Arc1p-C-0x-pET28a from two different clones, incubated for 1 h at 37 °C and plated out on LB-Kan50 plates that were incubated overnight at 37 °C.
18.22 restriction digest of Hag PCR fragments from 18.21 with NcoI/BamHI
Aim: creating restriction sites for cloning into pET24d cut with NcoI/BamHI
The PCR  products from 18.21 were digested with NcoI and BamHI to create restriction  sites in order to ligate the fragments into NcoI/BamHI cut pET24d.
Component
Volume [µl]
PCR products
45
Cutsmart 10x
5,2
NcoI
1
BamHI
1
Water
2,8
Total Volume
52
 Incubation for 2 hours at 37°C with heat shock at 80°C.
18.23 Ligation of pET24d and Hag PCR fragments from 18.22 cut with NcoI/BamHI
Aim: cloning inserts into pET24d for crystallization
The NcoI/BamHI digested flagellin PCR products from 18.22 (from clone 7 and piGEM-005) were ligated with NcoI/BamHI digested pET24d.
Component
Pet-Hag attempt (µL)
Pet-Hag-DARPin attempt (µL)
Insert (PCR fragments , c= ca. 400 ng/µL)
2
2
Vector (pet24d, c= 50ng/µL)
5
5
Ligase-Buffer 10x
2
2
T4 Ligase
1
1
Millipore Water
10
10
Total Volume
20
20
Incubation over night at room temperature.
20.2 Transformation of competent Bacillus subtilis
Aim: transformation of Bacillus subtilis WT3610 with piGEM005 and piGEM018
100 µL  of overnight culture were added to 10 mL of MNGE-Medium and incubated to an  OD600 of 1,1-1,3 at 37°C which could take 4-5 hours.
After  reaching an OD of 1,1-1,3 2 x 400 µL of the culture were transformed with 1,5 µg  piGEM-005 and -018. After one hour incubation at 37°C 100 µL expression mix were  added and incubated for one hour as well.
In the  end the 500 µL sample was plated out on MLS-X-Gal plates and incubated at 30 °C until colonies could be seen.
13.06.2014
18.24 Transformation of ligation
Aim: checking success of ligation and amplification of ligated construct
The  whole 20 µL attempt was used to transform E. coli X1-Blue, plated out on LB-Can  plates and incubated at 37°C overnight.
18.25 Miniprep of pMAD-Hag-Flank-DARPin (piGEM-018)
Aim: increase amount of plasmid for work and storage
2 x 6 mL of LB-Amp were inoculated  with a colony from a piGEM-018 transformation plate and incubated at 37°C  overnight.
18.26 Sequencing of constructs
Aim: checking the right insertion into the plasmids
pMAD-Hag-Flank-DARPin was sent in form of a premix with primers for sequencing. 10 µL with 50-100 ng/µL were mixed with 2 µL of primers for  sequencing according to the following scheme:
Premix
Label-Nr.
Construct
Primer
1
391
pMAD-Hag-Flank-DARPin
Flo54 (Hag-F-Nco)
2
392
pMAD-Hag-Flank-DARPin
Flo56 (Hag-R-Bam)
13.65 Checking growth of Bacillus subtilis WT3610
Aim: check which WT can be used for selection with Nose plasmids
Competent 3610 were transformed with piGEM-002, -003  and -004 by incubating the aliquots with 5 µL plasmid for 30 min at 37°C. After  half an hour the whole sample was plated out on LB-Cm (5 ng/µL).
Incubation at 30°C for 2 days.
14.06.2014
18.27 miniprep of plasmids from 18.25
Aim: isolation of plasmids
The miniprep of pMAD-Hag-Flank-DARPin was done according to the protocol of the OMEGA DNA kit.
18.28 pET24d digest with Nco/Bam
Aim: preparing vector for new ligation attempts
Because of the unsuccessful ligation from 18.23 new vector (pET24d) was digested with NcoI and BamHI according to the following scheme:
Component
Volume [µl]
pET24d (148 ng/µL)
15
Cutsmart 10x
3
NcoI
0,5
BamHI
0,5
Water
6
Total Volume
30
Incubation at 37°C for 2h.
18.29 PstI digestion of PCR fragments from piGEM-018 with Hag-primers
Aim: check if the correct inserts are used for ligation
The DARPin-Hag  constructs contain a PstI restriction site which could be used to check if the  DARPin gene was inserted into the Hag-construct correctly again. For that  purpose PCR products based on piGEM-018 from clone 7 and 12 (right and wrong  orientation) as a template were digested with PstI.
PCR  products from clone 7: 658 bp and 781 bp fragment.
 PCR products from clone 12: 1000bp and 429 bp  fragments.
Component
Volume [µl]
PCR fragment (350 ng/µL)
3
Cutsmart 10x
1
PstI
0,5
Water
5,5
Total Volume
10
Incubation at 37°C for 2h.
18.30 PCR amplification of DARPin-Hag and Hag fragment
Aim: amplification of DARPin-Hag fragments for repeating the ligation
piGEM-018 & -005 were used as template for a PCR with the primers  54flo and 56flo. The products could be used for further cloning after NcoI/BamHI digest.
Mix
Volume
Template (piGEM-005 & -018)
1
Primer Flo 54
1
Primer Flo 56
1
Phusion
1
Phusion Buffer 5x
10
dNTP Mix
1
Water
35
Total Volume
50
Step
Temperature °C
Time
1
98
3 min
2
98
30 sec
3
55
1min 45sec
4
72
2 min
5
Go To 2
30x
6
72
4min
7
8
infinite
The fragments were analysed on an 1% agarose gel.
15.54 new PCR amplification of cup1-1
Aim: amplification of cup1-1 domain for Gibson assembly with pMAD
The cup1-1 domain was amplified via PCR with new primers iGEM-024 and -025. The fragment should be ca. 160 bp.
Mix
Volume
Template (cup1-1 gene)
1
Primer iGEM-024
1
Primer iGEM-025
1
Phusion DNA-polymerase
1
Phusion Buffer 5x
10
dNTP Mix
1
Water
35
Total Volume
50
Step
Temperature °C
Time
1
98
3 min
2
98
30 sec
3
55
30 sec
4
72
30 sec
5
Go To 2
30x
6
72
4min
7
8
infinite
According to the gel analysis the amplification was successful.
15.06.2014
20.3 Overnight culture of positive clones
Aim: transformation of Bacillus subtilis WT3610 with plasmid
Colonies grew on the transformand plates (piGEM-005 & - 018). The blue/  white screening showed positive transformed blue clones.  Three clones per plate were picked and used for inoculation of LB-MLS (4 mL LB, 4 µL lincomycin, 4 µL  erythromycin). Incubation was carried out over night at 30°C with the six cultures.
16.06.2014
14.43 Overnight culture
Aim: Inoculate Overnight culture of PheA-Arc1p-C-0x-pET28a
5 mL of LB-Kan50 medium were inoculated with a clone from 14.42 bearing the PheA-Arc1p-C-0x-pET28a plasmid.
17.06.2014
20.3 First temperature shift
Aim: integration of  pMAD-Insert into Bacillus chromosome via flanks
The 6  overnight cultures were used to inoculate 10 mL LB MLS, which were incubated until the culture obtained an OD of 0,1. The  cultures were incubated at 30°C for 2h.
Subsequently, the temperature was shifted to 42°C for 6h.
After  the heat shock dilutions from 10-4 to 10-6 of each culture  were plated out on MLS-X-Gal so that 18 plates could be incubated overnight at  42°C.
15.54 Gibson-assembly with  PCR amplified cup1-1 and SpeI digested piGEM-005
Aim: insertion of cup1-1 into piGEM-005
The cup1-1 domain was amplified via PCR with new primers iGEM-024 and  -025. Vector and template were diluted to a concentration under 75 ng/µL. Spe1  cut piGEM-005 was used from the cloning experiments with DARPin.
PCR fragment cup1-1 (464 ng/µL) dilution 1:10
SpeI digested pIGEM-005 (148 ng/µL) dilution 1:2
Component
Volume [µl]
Gibson-Mix
15
Insert (cup1-1 58 ng/µL)
1,3
piGEM-005
1
Water
2,7
Total Volume
20
The mix was incubated 5 min at room temperature and then for 1h at 50°C.
In the end the whole mix was transformed into E.Coli XL1-Blue and plated  out on LB-Amp. Incubation was done overnight at 37°C.
14.44 Test Expression
Aim: Test the expression of PheA-Arc1p-C-0x
Using the preculture from 14.43 60 mL of LB-Kan50 medium were inoculated with 600 µL of the preculture and incubated at 37 °C and 220 rpm until OD600 reached 0.5. The culture was cooled down to 18 °C and the protein production was induced with 12 µL of 500 mM IPTG. Over night the culture was incubated at 18 °C and 220 rpm. Afterwards the cell culture was centrifuged (17000 rpm, 20 min, 4 °C), the pellet resuspended in 3 mL buffer A and stored at -20 °C. A glycerolstock of the strain containing the PheA-Arc1p-C-0x-pET28a plasmid was stored at -80 °C.
18.06.2014
20.4 Second temperature shift
Aim: flip out of the pMAD backbone
One  blue colony per diluted clone was used to inoculate 4 mL LB. The 6 cultures  were incubated at 30°C for 6h and afterwards for 3h at 42°C.
Dilutions  from 10-4 to 10-4 were plated out on 18 X-Gal plates  without MLS selection. The positive clones should not contain the resistance  inside the backbone as well as the galactosidase, resulting in normal colored colonies. The plates were incubated at  42°C overnight.
15.54 Checking Gibson assembly plate piGEM-005-cup1-1
Aim: insertion of cup1-1 into piGEM-005
Unfortunately the plate was empty: no colonies. The vector was digested again in order to repeat the assembly.
15.55 restriction digest of piGEM-005 with SpeI
Aim: linearizing plasmid for repeating the Gibson assembly
In order to repeat the Gibson assembly the piGEM-005 was cut with SpeI again.
Component
Volume [µl]
piGEM-005 (108 ng/µL)
10
Cutsmart 10x
2
SpeI
0,5
Water
7,5
Total Volume
20
Incubation 1h  at 37°C with following heat shock at 80°C
19.06.2014
20.5 Selection of positive clones
Aim: check the correct excision of the pMAD backbone
One white clone was picked from the dilution plates and transferred on a X-Gal Plate as well as on a MLS plate so that 18 clones were tested for the  right integration of the insert although flipping out the pMAD backbone.
The  plates were incubated at 42°C over night.
15.56 new Gibson-assembly with  cup1-1 and SpeI digested piGEM-005
Aim: insertion of cup1-1 into piGEM-005
As insert cup1-1 PCR product from 15.55 and SpeI digested piGEM-005 from  15.55 were used for a new Gibson assembly.
PCR fragment cup1-1 (464 ng/µL)
SpeI digested pIGEM-005 (53 ng/µL)
Component
Volume [µl]
Gibson-Mix
15
Insert (cup1-1 58 ng/µL)
2,5
piGEM-005
2,5
Total Volume
20
The mix was incubated 5 min at room temperature and then for 1h at 50°C.
In the end the whole mix was used to transform E. coli XL1-Blue and plated  out on LB-Amp. Incubation was done overnight at 37°C.
20.06.2014
20.5 selection of positive clones
Aim: checking the correct flip out of the pMAD backbone
The white colonies  grew on the X-Gal Master plates but not on MLS plates so the transformation  seemed to be successful. The Backbone with the MLS resistance flipped out of  the genome.
20.6a cPCR with checked clones
Aim: checking integration of constructs Hag-Spe and Hag-Spe-DARPin into B. s. genome
No  clone was growing on MLS plates so the picked clones seemed to be positive. In  order to check the correct integration of the domain constructs, a cPCR with the picked clones on the X-Gal plates was done. The 18 picked B.subtilis clones were  cooked in 10 µL PBS for 5 min at 95°C.
Mix
Master-Mix for 18 attempts (µL)
Mix for PCR attempt (µL)
Colony (picked clones 1-9 from Flagellin/-DARPin B.s.  plates)
-
Part of a cooked colony
Primer Flo54 Hag-fw
9
-
Primer Flo56 Hag-rv
9
-
Phusion
18
-
Phusion Buffer 5x
72
-
dNTP Mix
9
-
Water
237
-
DMSO
6
-
Mastermix
-
20
Total Volume
100
360
Step
Temperature °C
Time
1
98
3 min
2
98
30 sec
3
55
1 min 45 sec
4
72
2 min
5
Go To 2
30x
6
72
4min
7
8
infinite
The gel showed no bands. The PCR had to be repeated.
15.57 Miniprep of clones from Gibson assembly from 15.56
Aim: increase the amount of plasmid for further experiments
4 x 5 mL of LB-Amp was were  inoculated with a picked colony from a transformation plate with XL1-Blue  containing the Gibson-assembly product from 15.56.
Incubation at 37°C overnight.
21.06.2014
15.58 cPCR with colonies from Gibson-assembly of piGEM-005 + cup1-1 plate
Aim: checking correct insertion of cup1-1
In order to check the correct  insertion of the cup1-1 domain into the Hag-Flank-construct in pMAD a cPCR with  primers iGEM-025 (Cup1-1 rv) and Flo89 (pMAD-flank1-BamHI fw) was performed. The picked colonies were the same which were used to inoculate minipreps from 15.57. The fragment was supposed to have approx. 1250  bp.
Mix
Master-Mix for 5 attempts (µL)
Mix for PCR attempt (µL)
Colony (picked clones 1-4 for miniprep)
-
Part of a  colony
Primer Flo89
2,5
-
Primer piGEM-025
2,5
-
Phusion DNA-polymerase
2,5
-
Phusion Buffer 5x
20
-
dNTP Mix
2,5
-
Water
70
-
Mastermix
-
20
Total Volume
100
20
Step
Temperature °C
Time
1
98
3 min
2
98
30 sec
3
55
30 sec
4
72
1 min 30 sec
5
Go To 2
30x
6
72
4min
7
8
infinite
The analysis via 1% agarose gel eletrophoresis showed now bands. The plasmids of all four clones were isolated to use them as template.
15.59 PCR with isolated plasmid from Gibson piGEM-005 + cup1-1 plate
Aim: checking correct insertion of cup1-1
This  time the isolated plasmid DNA from the picked six clones was used as a template  with different primer pairs. First of all the PCR reactions were performed with  primers Flo89 and Flo90 in order to receive the whole Hag-Flank-Cup1-1  construct with ca. 2100 bp length. As negative control also piGEM-005 was used  as template producing a 2000kb fragment. Because of the small difference, all six reactions were also performed with the primers Flo89 and iGEM-025 (cup1-1 rv) in  order to check the insertion of cup1-1 directly with a fragment of 1268 bp.
Mix
Master-Mix 1 for  5 reactions (µL)
Master-Mix 2 for  5 reactions (µL)
Mix 1 for PCR reactions (µL)
Mix 2for PCR reactions (µL)
Plasmid DNA from clone 1-4/ piGEM-005
-
-
1
-
Primer Flo89
2,5
2,5
-
-
Primer Flo90
2,5
-
-
-
Primer piGEM-025
-
2,5
-
-
Phusion DNA-polymerase
2,5
2,5
-
-
Phusion Buffer 5x
20
20
-
-
dNTP-mix
2,5
2,5
-
-
Water
70
70
-
-
Master-Mix 1/ 2
-
-
20
20
Total Volume
100
100
20
20
Step
Temperature °C
Time
1
98
3 min
2
98
30 sec
3
55
30 sec
4
72
20 sec
5
Go To 2
30x
6
72
4min
7
8
infinite
The gel analysis shows that all 4 clones seem to be positive. An amplification of the cup1-1 fragment with the plasmid DNA as template could proof that.
20.6b Repeated cPCR with checked clones
Aim: checking integration of constructs Hag-SpeI and Hag-SpeI-DARPin into B. s. genome
Because  of the unsuccessful cPCR before, the PCR was repeated. The 18 picked B.subtilis clones  were cooked in 30 µL PBS for 10 min at 95°C.
Mix
Master-Mix for 18 attempts (µL)
Mix for PCR attempt (µL)
Colony (picked clones 1-9 from Flagellin/-DARPin B.s.  plates)
-
Part of a cooked colony
Primer Flo54 Hag-fw
9
-
Primer Flo56 Hag-rv
9
-
Phusion DNA-polymerase
18
-
Phusion Buffer 5x
72
-
dNTP Mix
9
-
Water
237
-
DMSO
6
-
Mastermix
-
20
Total Volume
100
360
Step
Temperature °C
Time
1
98
3 min
2
98
30 sec
3
55
1 min 45 sec
4
72
2 min
5
Go To 2
30x
6
72
4min
7
8
infinite
All nine clones with piGEM-005 seem to be positive. Clone 1, 3, 7, 8 and  9 of transformed piGEM-018.1 might be positive as well as clone 2-6 from  piGEM-018.2. The sequencing of the PCR products should tell us if the insertion  was successful.
22.06.2014
18.31 PCR amplification of DARPin-Hag and Hag fragment
Aim: amplification of DARPin-Hag fragments for repeating the ligation
piGEM-018 & -005 were used as template for a PCR with the primers Flo 54 and Flo 56. The products could be used for further cloning after NcoI/BamHI digest.
Content
Volume [µl]
Template (piGEM-005 & -018)
1
Primer Flo 54
1
Primer Flo 56
1
Phusion
1
Phusion Buffer 5x
10
dNTP Mix
1
Water
35
Total Volume
50
Step
Temperature °C
Time
1
98
3 min
2
98
30 sec
3
55
30 sec
4
72
1 min 45 sec
5
Go To 2
30x
6
72
4min
7
8
infinite
18.32 cPCR with new clones
Aim: checking picked clones for correct Hag/ Hag-DARPin insert in pET24d
More clones from each  plate were picked to analyse them via cPCR with the primers flo54 and flo56  (Hag-fw and Hag-rv). The result should be a ca. 1kb fragment for correct  pET24d-Hag containing clones and a 1,5 kb fragment for pET24d-Hag-DARPin  containing clones.
Content
Volume (µL)
MM for 14 attempts (µL
Colony
 (8-15 from pet-Hag,7-14 from pet-Hag-DARPin)
1 colony
-
Primer Flo 54
-
7
Primer Flo 56
-
7
Phusion DNA-polymerase
-
7
Phusion Buffer 5x
-
56
dNTP Mix
-
7
Water
-
196
Mastermix
20
-
Total Volume
20
280
Step
Temperature °C
Time
1
98
10 min
2
98
30 sec
3
55
30 sec
4
72
1 min 45 sec
5
Go To 2
35x
6
72
4min
7
8
infinite
Clone 1 from each pET24d-Hag-SpeI and pET24d-Hag-DARPin clones was positive and were digested with NcoI/BamI to check them again.
18.33 pET24d digest with NcoI/BamHI
Aim: preparing vector for new ligation attempts
Because  of the low amount of right clones, new vector (pET24d) was digested with NcoI and BamHI according to the following scheme:
Component
Volume (µL)
pET24d (50 ng/µL)
20
CutSmart 10x
3
NcoI
0,5
BamHI
0,5
Water
6
Total Volume
30
Incubation at 37°C for 2 hours with heat shock in the end at 80°C
Endconcentration:  32 ng/µL
18.34 Restriction digest of Hag PCR fragments from 18.31 with NcoI/BamHI
Aim: creating restriction sites for cloning into pet24d cut with NcoI/BamHI
The PCR products from 18.31 were digested with NcoI andBamHI to create restriction sites in order to ligate the fragments into NcoI/BamHI digested pet24d in case of negative clones refering to the cPCR from 18.x
Component
PCR Hag-fragment (µL)
PCR Hag-DARPin fragment  (µL)
PCR products
45
45
Cutsmart 10x
5,2
5,2
NcoI
0,5
0,5
BamHI
0,5
0,5
Water
3,8
3,8
Total Volume
52
52
Incubation  for 2h at 37°C with heat shock at 80°C.
Endconcentration:
digested Hag-fragment  260 ng/µL
digested  Hag-fragment 268 ng/µL
15.60 PCR pMAD-Hag-Flank-cup1-1 (piGEM-016)
Aim: checking success of Gibson assembly with pMAD-Hag-Flank and cup1-1
Using  the plasmids as template the cup1-1 fragment at 160 bp should be visible on the  agarose gel. As positive control PCR products from the cup1-1 amplification  used for the Gibson Assembly into piGEM-005 were applied to the gel as well.
Content
Volume (µL)
MM for 5 attempts (µL)
Template (Plasmid from clone 1-4)
0,5
-
Primer iGEM-024
-
2,5
Primer iGEM-025
-
2,5
Phusion DNA-polymerase
-
2,5
Phusion Buffer 5x
-
20
dNTP Mix
-
2,5
Water
-
70
Mastermix
19,5
-
Total Volume
20
100
Step
Temperature °C
Time
1
98
3 min
2
98
30 sec
3
55
30 sec
4
72
20 sec
5
Go To 2
35x
6
72
4min
7
8
infinite
The bands under the 200 bp mark indicate the positive insertion of  cup1-1 into piGEM-005. The plasmid has to be sequenced and is named iGEM-016.
18.35 Test digest with NcoI/BamHI - Checking insertion of Hag/-DARPin into pet24d
Aim: proving insertion of Hag/-DARPin fragment into pET24d
The  plasmids are digested with NcoI and BamHI. In case of a correct insertion the  pET24d backbone and the insert should be seen on the gel at 5300 bp and 1000/  1500 bp. As control pET24d-Hag-Flank (piGEM-001) is used to check the  mastermix.
Component
Volume (µl)
Mastermix
2x Plasmid from clone 6
5
-
CutSmart 10x
-
5
NcoI
-
0,5
BamHI
-
0,5
Water
-
19
Total Volume
10
25
The used marker was the GeneRuler 1 kb DNA ladder. The lower fragment of the digested pET24d-Hag was too big (2 kb) and pET24d-Hag-DARPin seemed only to be linearized with the exception of two very thin bands below the linear plasmid, which were all of the wrong size.
18.36 ligation of pET24d  and PCR Hag/ Hag-DARPin fragments cut NcoI/BamHI
Aim: ligation of construct with pET24d for overproduction of flagellin and crystallization
The NcoI/BamHI digested flagellin PCR products from 18.22 (from clone 7 and piGEM-005) were  ligated with NcoI/BamHI digested pET24d.
Component
pET24d-Hag reaction (µL)
pET24d-Hag-DARPin raction (µL)
Insert (PCR fragments) (ca. 260 ng/µL)
2
2
Vector (pET24d, c= 50ng/µL)
5
5
Ligase-Buffer 10x
2
2
T4 Ligase
1
10
Millipore Water
10
10
Total Volume
20
20
The reactions were prepared in double and were incubated at room  temperature. One reaction was incubated for 1 hour followed  by transformation of E. coli XL1-Blue and  the second one over night.
18.37 Repetition of PCR with 3 clones of pET24d-Hag and pET24d-Hag-DARPin ligation
Aim: checking picked clones for correct Hag/ Hag-DARPin insert in pET24d
3  clones from ligation plates with pET-Hag and pET-Hag-DARPin were resuspended in  30 µL PBS and cooked at 95 °C for 10 min. After that procedure 1 µL of this  suspension was used as PCR template.
Content
Volume (µL)
MM for 7 reactions (µL)
Template ( 1µL cooked suspension)
 (8-10 from pET-Hag, 7-9 from pET-Hag-DARPin)
1
-
Primer Flo 54
-
3,5
Primer Flo 56
-
3,5
Phusion DNA-polymerase
-
3,5
Phusion Buffer 5x
-
28
dNTP-mix
-
3,5
Water
-
98
Mastermix
19
-
Total Volume
20
140
Step
Temperature °C
Time
1
98
3 min
2
98
30 sec
3
55
30 sec
4
72
1 min 45 sec
5
Go To 2
35x
6
72
4min
7
8
infinite
Run over night
Transformation
Plasmid
Concentration (ng/µL)
Antibiotic
Cells
Volume (µL)
pET24d
148
Can
DH5α
1
piGEM-018
160
Amp
DH5α
0,5
piGEM-014
350
Amp
DH5α
0,5
Gibson pMAD-cup1-1 clone 4
60
Amp
DH5α
1
pET-Hag attempt from 18.36
-
Can
XL1-Blue
20
pET-Hag-DARPin attempt from 18.36
-
Can
XL1-Blue
20
Control pET24d cut NcoI/BamHI
32
Can
XL1-Blue
1
23.06.2014
14.45 Purification of test expression
Aim: Purify PheA-Arc1p-C-0x on a small scale
Lysozyme was added to the cell suspension from 14.44 to a final concentration of 1 mg/mL and incubated on ice for 30 min. Afterwards the cells were lysed using ultra sound (2x 18 pulses, 30 s on ice inbetween). To clear the suspension of cell debris it was centrifuged (13000 rpm, 15 min, 4 °C). The supernatant was purified using QIAGEN Ni-NTA Spin Columns. An SDS-PAGE gel showed that the protein is produced and can be purified using Ni affinity chromatography.
60 mL LB-Kan50 medium was inoculated from the glycerolstock prepared in 14.44 and incubated at 37 °C and 220 rpm over night.
Sequencing of constructs from 23.06.2014
Premix
Label-Nr.
Construct
Primer
1
AGB0023402
piGEM-018  (pMAD-DARPin)
Flo54 (Hag-F-Nco)
2
AGB0023403
piGEM-018  (pMAD-DARPin)
Flo56 (Hag-R-Bam)
3
AGB0023404
pMAD-Hag-Flank-cup1-1
Flo54 (Hag-F-Nco)
4
AGB0023405
pMAD-Hag-Flank-cup1-1
Flo56 (Hag-R-Bam)
5
AGB0023406
piGEM-002 (Nose  plasmid)
TW259
18.37 Repetition of PCR with 3 clones of pET24d-Hag and pET24d-Hag-DARPin ligation
Aim: checking picked clones for correct Hag/ Hag-DARPin insert in pET24d
Gel electrophoresis:
The gel shows negative bands in case of the Hag-DARPin clones. The PCR for Hag 8 and 9 did not work although we cooked the clones in PBS. The transformation efficiency seemed to be very low, so clones from the new ligation attempts should be analysed.
18.38 Ligation checking success of transformed ligation attempts
On every plate grew a high amount of colonies. 6 clones of each plate were picked for inoculation of minipreps and for cPCR after cooking with PBS. The clones were transferred to another LB-Can plate.
18.39 cPCR with picked clones from 18.38 ligation attempt transformation
Aim: checking picked clones for correct Hag/ Hag-DARPin insert in pET24d
For  checking the insertion of the Hag/-DARPin into pET24d the clones from the  transformation plate were transferred to another LB-Can plate  and cooked for 10 min in 30 µL PBS at 95°C. At the same time the colonies were used to inoculate LB-Can for minipreps.
Content
Volume (µL)
MM for 14 attempts (µL)
Template ( 1µL cooked suspension)
 (1-6 from pet-Hag, 1-6 from pet-Hag-DARPin)
1
-
Primer Flo 54
-
7
Primer Flo 56
-
7
Phusion DNA-polymerase
-
7
Phusion Buffer 5x
-
198
dNTP-mix
-
7
Water
-
98
Mastermix
19
-
Total Volume
20
280
Step
Temperature °C
Time
1
98
5 min
2
98
30 sec
3
55
30 sec
4
72
1 min 45 sec
5
Go To 2
35x
6
72
4min
7
8
infinite
Run over night
24.06.2014
14.46 Expression of PheA-Arc1p-C-0x
Aim: Express PheA-Arc1p-C-0x for further use
10 x 500 mL LB-Kan50 medium were inoculated with 5 mL of the overnight culture prepared in 14.45 and incubated at 37 °C and 220 rpm until OD600 reached 0.5. The flasks were cooled to 18 °C and protein production was induced by adding 100 µL 500 mM IPTG. The cultures were incubated over night at 18 °C and 220 rpm. Afterwards the cells were collected by centrifugation (6000 rpm, 20 min, 4 °C), resuspended in buffer A and stored at -20 °C until further use.
25.06.2014
Sequencing  results of constructs from 23.06.2014
Premix
Label-Nr.
Construct
Result
1
AGB0023402
piGEM-018  (pMAD-DARPin)
Correct insertion of  DARPin
2
AGB0023403
piGEM-018  (pMAD-DARPin)
Did not work
3
AGB0023404
pMAD-Hag-Flank-cup1-1
Correct insertion,  point mutation refers to bad sequencing
4
AGB0023405
pMAD-Hag-Flank-cup1-1
Did not work
5
AGB0023406
piGEM-002 (Nose  plasmid)
Correct insertion of  the CM resistance
18.39 Gel cPCR with picked clones from 18.38 ligation attempt transformation
Aim: checking picked clones for correct Hag/ Hag-DARPin insert in pET24d
Gel electrophoresis: no bands except for the controls were seen. The PCR has to be repeated with more colony in the PCR tubes or the prepped plasmids
18.40 Test digest with NcoI/BamHI of isolated plasmids from picked colonies
Aim: checking picked clones for correct Hag/ Hag-DARPin insert in pET24d
The picked clones from the ligation attempts 18.38 were isolated. In  order to check the insertion of the Hag/ Hag-DARPin domain the plasmids were  digested with NcoI/BamHI so that the gel should show the insert of 1 or 1,5 bp  length and the pET24d backbone at 5300 bp. Because of an unavailable marker as  a control the PCR amplified inserts which were used for the ligation will  run  with the gel as well. piGEM-001 will  also be cut NcoI/BamHI in order to receive a marker band for the pET24d backbone.
Component
Volume (µL)
Mastermix for 8 attempts
Plasmid (ca. 45 ng/µL
5
-
Cutsmart 10x
-
4
NcoI
-
0,5
BamHI
-
0,5
Water
-
35
Mastermix
5
-
Total Volume
10
40
The gel shows that the digested clones were all negative. The ligation had to be performed again.
20.6 Purification of  cPCR samples with Gel extraction kit
Aim: purification of  samples for sequencing
Purified samples:
clone 3 & 5 from transformed piGEM-005
clone 3 & 9 from transformed piGEM-018
Clone
Concentration (ng/µl)
Hag 3
10
Hag 5
14
Hag-DARPin 3
9
Hag-DARPin 9
9
26.06.2014
20.7 PCR with purified cPCR samples from 20.6
Aim: PCR amplification of purified cPCR samples
The concentration of the cPCR samples after purification was too low for sequencing. Therefore the cPCR samples have to be amplified with PCR again.
Content
Volume (µL)
MM for 6 attempts (µL)
Purified cPCR product from 20.6
 (3 & 5 from pet-Hag, 3 & 9 from  pet-Hag-DARPin)
1
-
Primer Flo 54
-
6
Primer Flo 56
-
6
Phusion DNA-polymerase
-
6
Phusion Buffer 5x
-
6
dNTP-mix
-
60
Water
-
216
Mastermix
19
-
Total Volume
20
300
Step
Temperature °C
Time
1
98
3 min
2
98
30 sec
3
55
30 sec
4
72
1 min 45 sec
5
Go To 2
35x
6
72
4min
7
8
infinite
20.8 Sequencing  of purified cPCR samples from pMAD transformation
Premix
Label-Nr.
Construct
Primer
1
AGB0023407
Hag-SpeI in B. subtilis genome clone 3
Hag-Nco-fw
2
AGB0023408
Hag-SpeI in B. subtilis genome clone 5
Hag-Nco-fw
3
AGB0023409
Hag-SpeI-DARPin in B. subtilis genome clone3
Hag-Nco-fw
4
AGB0023410
Hag-SpeI-DARPin in B. subtilis genome clone 9
Hag-Nco-fw
18.41 New ligation of pET24d  and PCR Hag/ Hag-DARPin fragments cut NcoI/BamHI transformation
Aim: ligation of construct with pET24d for overproduction of flagellin and crystallization
The cut  pET24d NcoI/BamHI which was used for the previous ligations was not purified and  contained an 1 kb insert. For a new clean ligation the reaction was repeated  with digested pET24d. The NcoI/BamHI digested flagellin PCR products (from  clone 7 and piGEM-005) were ligated with NcoI/BamHI digested pET24d.
The  reactions were carried out in double and were incubated 1h  at room temperature. Finally they were used to transform E. coli XL1-Blue after  inactivation at 65°C for 10 min. The attempts 1.1 and 2.1 were incubated  over night at room temperature. 1 µL of digested vector was used for a transformation as a negative  control in order to check the amount of religants.
Component
pET-Hag attempt (µL) – 1 & 1.1
pET-Hag-DARPin attempt (µL) 2& 2.1
Insert (PCR fragments) (ca. 260 ng/µL)
3
3
Vector (pET24d, c= 86ng/µL)
5
5
Ligase-Buffer 10x
2
2
T4 Ligase
1
1
Millipore water
9
9
Total Volume
20
20
27.06.2014
18.42 Miniprep of ligation clones
The  ligation between pET24d and the Hag-/Hag-DARPin fragments which were both cut NcoI/BamHI was successful. The ligation plates showed over 20 clones each. However, on the negative control  grew a few clones  as well, which were very small and sticking together. In order to analyse the insert of the clones, 5 clones per  ligation plate 1 and 2 were used to inoculate minipreps in 5 mL LB-Can. For the  Ligation 1 pET-Hag clones 1-5 were picked, as well as clones 6-10 from Plate  Ligation 2 pET-Hag-DARPin.
The  cultures were inoculated over night at 37°C.
28.06.2014
18.42 Miniprep of ligation clones
Construct
Clone
Concentration (ng/µL)
pET24d-Hag
1
50
pET24d-Hag
2
87
pET24d-Hag
3
65
pET24d-Hag-DARPin
6
83
pET24d-Hag-DARPin
7
59
18.43 cPCR of clones from 18.41
Aim: checking picked clones for correct Hag/ Hag-DARPin insert in pET24d
Plasmid  from isolated clones from each plate were analysed via cPCR with the primers flo54  and flo56 (Hag-fw and Hag-rv). The result should be a ca. 1kb fragment for  correct pET24d-Hag containing clones and a 1,5 kb fragment for  pET24d-Hag-DARPin containing clones.
Content
Volume (µL)
MM for 6 attempts (µL)
Picked clones from 18.42
(1-3 pET24d-Hag, 6-7 from pET24d-Hag-DARPin)
0,5
-
Primer Flo 54
-
3
Primer Flo 56
-
3
Phusion DNA-polymerase
-
3
Phusion Buffer 5x
-
6
dNTP-mix
-
24
Water
-
81
Mastermix
19,5
-
Total Volume
20
120
Step
Temperature °C
Time
1
98
3 min
2
98
30 sec
3
55
30 sec
4
72
1 min 45 sec
5
Go To 2
30x
6
72
4min
7
8
infinite
The samples can be seen on the gel under 18.44. The PCR for the Hag-fragment in pET24d seemed to be positive (PCR.H1 - H3), as well as for the Hag-DARPin fragment (PCR.D7)
18.44 test digest with NcoI/BamHI - Checking insertion of Hag/-DARPin into pET24d
Aim: proving insertion of Hag/-DARPin fragment into pET24d
The plasmids were digested with NcoI and BamHI. In case of a correct insertion the pet24d backbone and the insert should be seen on the gel at 5300 bp and 1000/ 1500 bp. As control pET24d-Hag-Flank (piGEM-001) is used to check the mastermix.
Component
Volume (µL)
Mastermix(6 attempts)
Plasmid from clone
( pET24d-Hag 1-3, pET24d-Hag-DARPin 6 & 7)
5
-
Cutsmart 10x
-
3
NcoI
-
0,5
BamHI
-
0,5
Water
-
26
Total Volume
10
30
20.9 Gel Purification of amplified cPCR products
Aim: Purification for more efficient sequencing
The PCR product of the  amplification of cPCR products was not pure considering different bands on the  gel next to the right one. In order to prevent sequencing mistakes the PCR  attempts were purified via gel extraction of the 1,5 kb band in case of the  Hag-DARPin construct.
20.10  PCR with purified Hag-DARPin cPCR products for sequencing
Aim: amplification of purified Hag-DARPin fragment for sequencing
The purified cPCR product from 20.9 has to be amplified again in order to get a higher concentration for sequencing.
Content
Volume (µL)
MM for 2 attempts (µL)
Purified PCR product from 20.x
(Hag-/ Hag-DARPin fragment)
1-2
-
Primer Flo 54
-
1
Primer Flo 56
-
1
Phusion DNA-polymerase
-
1
Phusion Buffer 5x
-
2
dNTP-mix
-
20
Water
-
75
Mastermix
19
-
Total Volume
20
100
Step
Temperature °C
Time
1
98
5 min
2
98
30 sec
3
55
30 sec
4
72
1 min 45 sec
5
Go To 2
35x
6
72
4min
7
8
infinite
18.45 SpeI  digest of pET24d-Hag (piGEM-019)
Aim: linearizing pET24d-Hag with SpeI site for a Gibson assembly with Cup1-1
Component
Volume (µL)
piGEM-016 (ng/µL)
17
CutSmart 10x
2
SpeI
0,5
Water
0,5
Total Volume
20
Incubation 1 h  at 37°C with following heat inactivation at 80°C
18.46 Gibson assembly with linearized pET24d-Hag (piGEM-019) and cup1-1
Aim: Insertion of cup1-1 domain into pET24d-Hag construct for crystallization
As insert cup1-1 PCR product from 15.55 and SpeI digested piGEM-016 from 18.44 were used for a new Gibson assembly.
PCR fragment cup1-1 (464 ng/µL)
SpeI digested pIGEM-005 ( ng/µL)
Component
Volume (µl)
Gibson-mix
15
Insert (cup1-1 464 ng/µL)
2,5
piGEM-019
2,5
Total Volume
20
The mix was incubated 5 min at room temperature and then for 1h at 50°C.
In the end the whole mix was used to transform E. coli XL1-Blue, which were plated  out on LB-Amp. Incubation was done over night at 37°C.
29.06.2014
18.47 cPCR of Gibson assembly clones of pET24d-Hag (piGEM-019) and cup1-1
Aim: screening clones for right insert
In order to screen the clones for  the right insertion of cup1-1 four clones were picked from the LB-Can plate and  transferred onto a LB-Can plate. Additionally the clones were used for  inoculation of minipreps and also cooked in 50 µL dest. Water for 10 min at  95°C. The lysate was used as template.
The PCR with Flo54 and iGEM-025  should generate a 785 bp fragment and the one with iGEM-024 and -025 the 164bp cup domain fragment.
Content
Volume (µL)
MM 1 for 5 attempts (µL)
MM 2 for 5 attempts (µL)
Cooked lysate solution from clones 1-4
3
-
 
Primer Flo 54
-
2,5
-
Primer iGEM-024
-
-
2,5
Primer iGEM-025
-
-
2,5
dNTP-mix
-
2,5
2,5
Phusion DNA-polymerase
-
5
5
Phusion Buffer 5x
-
20
20
Water
-
67,5
67,5
Mastermix
17
-
-
Total Volume
20
100
100
Step
Temperature °C
Time
1
98
10 min
2
98
30 sec
3
55
30 sec
4
72
1 min 45 sec
5
Go To 2
35x
6
72
4min
7
8
infinite
The PCR showed no bands. Nevertheless, colonies were used to inoculate cultures for minipreps so that the isolated plasmid can be further analysed for the insert.
30.06.2014
Sequencing
Premix
Label-Nr.
Construct
Primers
1
AGB0023411
pET24d-Hag-SpeI  construct cl. 2
Hag-Nco-fw
2
AGB0023412
pET24d-Hag-DARPin  construct cl. 7
Hag-Nco-fw
18.48a Miniprep of plasmids from Gibson assembly clones with pET24d-Hag (piGEM-019) and cup1-1
Aim: isolation of Plasmid DNA for further experiments
Construct
Clone
Concentration (ng/µL)
pET24d-Hag-Cup1-1
1
122
pET24d-Hag-Cup1-1
2
85
pET24d-Hag-Cup1-1
3
172
pET24d-Hag-Cup1-1
4
116
18.48b PCR of isolated plasmids from Gibson assembly clones (pET24d-Hag (piGEM-019) and cup1-1)
Aim: screening clones for right insert
The isolated plasmid 1:10 diluted from the picked clones 1-4 were used as PCR template.
The PCR with Flo54 and iGEM-025  should generate a 785 bp fragment and the one with iGEM-024 and -025 the 164bp  domain fragment.
Content
Volume (µL)
MM 1 for 5 attempts (µL)
MM 2 for 5 attempts (µL)
Cooked lysate solution from clones 1-4 (1:10)
3
-
 
Primer Flo 54
-
2,5
-
Primer iGEM-024
-
-
2,5
Primer iGEM-025
-
2,5
2,5
dNTP-mix
-
2,5
2,5
Phusion
-
5
5
Phusion Buffer 5x
-
20
20
Water
-
67,5
67,5
Mastermix
17
-
-
Total Volume
20
100
100
Step
Temperature °C
Time
1
98
3 min
2
98
30 sec
3
55
30 sec
4
72
1 min 45 sec
5
Go To 2
35x
6
72
4min
7
8
infinite
No bands could be observed on the gel. The PCR was repeated with Q5 mastermix to exclude the disfunction of the polymerase.
20.9 Inoculation of a preculture from Bacillus subtilis WT 3610
Aim: Preparation for a mainculture the for the next day
2 x 6 mL of LB were inoculated with Bacillus subtilis WT3610 from an LB plate and incubated at 37°C over night.
14.47 Purification of PheA-Arc1p-C-0x
Aim: Purify PheA-Arc1p-C-0x in three steps
The resuspended cells from 14.46 were lysed using a french press. The lysate was incubated on ice with 5 µg/mL DNAseA and RNAseI for 10 min and centrifuged (27000 rpm, 45 min, 4 °C) to pellet cell debris. The supernatant was filtered (0.45 µm) and PheA-Arc1p-C-0x purified from the cleared solution by NiNTA affinity chromatography. The combined elution fractions containing crude PheA-Arc1p-C-0x were concentrated to a volume of 2 mL and further purified using an anion exchange column to get rid of possible tRNA contaminations. Finally the buffer was changed to dialysis buffer and the protein concentrated and stored at -80 °C in aliquods for further use.
Retrieved from " http://2014.igem.org/Team:Marburg:Project:Notebook:June "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
