"
Page
Discussion
View source
History
teams
Log in
 
Team:Marburg:Project:Notebook:July
From 2014.igem.org
Home
Project
Safety
Team
Policy & Practices
Attributions
SilverSURF
CancerSURF
RiboSURF
Notebook
Parts
Notebook: July
March
April
May
June
July
01.07.2014
02.07.2014
03.07.2014
04.07.2014
05.07.2014
06.07.2014
07.07.2014
08.07.2014
09.07.2014
10.07.2014
11.07.2014
14.07.2014
15.07.2014
16.07.2014
17.07.2014
18.07.2014
19.07.2014
20.07.2014
21.07.2014
22.07.2014
23.07.2014
24.07.2014
25.07.2014
28.07.2014
31.07.2014
August
September
October
Methods
01.07.2014
20.10 pMAD-Transformation of competent Bacillus subtilis WT 3610
Aim: Transformation of piGEM-016
100 µL  of overnight culture were added to 10 mL of MNGE-Medium and incubated till an  OD of 0,7 at 37°C which took  7 hours.
After  reaching OD of  0,7  400 µL of the culture were transformed with  1,5 µg piGEM-021. After 1 hour incubation at 37°C 100 µL Expression mix was added  and incubated for 1h as well.
In the  end the 500 µL attempt were plated out on MLS-X-Gal plates and incubated at 30°C overnight until colonies could be seen.
Sequencing  results of constructs from 30.06.2014
Premix
Label-Nr.
Construct
Result
1
AGB0023411
pET24d-Hag-SpeI  construct cl. 2
Correct insertion of  Hag-SpeI
2
AGB0023412
pET24d-Hag-DARPin  construct cl. 7
Correct insertion of  Hag-DARPin
14.48 PCR of PheA-single
Aim: Amplify PheA-single fragment for Gibson Assembly in pET-28a
The PCR with TG_PheA-Arc1p-C-8x FP und TG-PheA-single-GA RP should generate the 1716 bp fragment for Gibson Assembly from the PheA-Arc1p-C-2x-pET28a construct (14.23).
The used Primer concentration was 1 µM, the concentration of the template was 1 ng/µL and we used a hot start to avoid unspecific annealing of primers.
Content
Volume [µL]
Water
29.5
5x GC Buffer
10
DMSO
2.5
TG_PheA-Arc1p-C-8x FP
2.5
TG_PheA-Arc1p-C-8x RP
2.5
PheA-Arc1p-C-2x-pET28a
1
dNTPs
1
Phusion-Polymerase
1
Total Volume
50
Step
Temperature [°C]
Time [min:sec]
1
98
2:00
2
add Polymerase
 
3
98
0:10
4
67.5
0:30
5
72
1:15
6
go to 3
32x
7
72
10:00
8
4
hold
The amplified fragment was purified on an agarose gel on which it showed the correct size. It was extracted from the gel for further use in a Gibson Assembly yielding a concentration of 198 ng/µL.
02.07.2014
14.49 Gibson Assembly
Aim: Integrate PheA-single into linearized pET-28a(XhoI,NdeI) via Gibson Assembly
Content
Volume [µL]
Water
5.34
pET-28a(XhoI,NdeI) (24 ng/µL)
4.2
PheA-single (198 ng/µL)
0.46
2x Gibson Mastermix
10
Total Volume
20
The reaction mixture was incubated for 1 h at 50 °C and 350 rpm. E. coli Top10 electrocompetent cells were transformed with a 1:3 and a 1:6 dilution of the mixture, incubated for 1 h at 37 °C and plated out on LB-Kan50 plates that were incubated overnight at 37 °C.
19.3 Co-Transformation of piGEM-019 & -020 into E. coli BL21(DE3) with FliS
Aim: Transformation of plasmids with flagellin modifications and chaperone for Flagellin into protein producing strain
piGEM-019  and -020 were used to transform E. coli BL21 (DE3) together with a plasmid  containing the flagellin chaperone FlaS in order to express both proteins at  the same time after induction so that the flagellin monomer accumulates inside  the cells.
The  cells were plated out on Can/Amp-LB plates and incubated overnight at 37°C.
03.07.2014
14.50 Overnight culture
Aim: Amplify Plasmid in an overnight culture
Six clones of the transformed Top10 cells from 14.49 were picked and used to inoculate overnight cultures (6 x 5 mL).
18.49 new PCR of isolated Gibson assembly clones ( pET24d-Hag (piGEM-019) and cup1-1)
Aim: screening clones for right insert
The isolated plasmid 1:10 diluted from the picked clones 1-4 were used as PCR template.
The PCR with Flo54 and iGEM-025  should generate a 785 bp fragment and the one with iGEM-024 and -025 the 164bp  domain fragment.
Content
Volume (µL)
MM 1 for 5 attempts (µL)
MM 2 for 5 attempts (µL)
Plasmid from clones 1-4 (1:10)
1
-
 
Primer Flo 54
-
2,5
-
Primer iGEM-024
-
-
2,5
Primer iGEM-025
-
2,5
2,5
Q5 mastermix 2X
-
50
50
Water
-
45
45
Mastermix
19
-
-
Total Volume
20
100
100
Step
Temperature °C
Time
1
98
3 min
2
98
30 sec
3
55
30 sec
4
72
1 min 45 sec
5
Go To 2
30x
6
72
4min
7
8
infinite
The agarose gel showed bands at  ca. 750 bp which fits to the expected 780 bp fragment with primers piGEM-025  (cup rv) and Flo54 ( Hag fw).
The gel's second row shows the  amplified cup-fragment under the 200 bp ladder mark. The expected size is ca.  164 bp and fits to the gel result.
Plasmid from clone 4 was used to transform E.Coli DH5α and named piGEM-021.
19.4 Expression-Test with transformed E. coli BL21 (DE3)
Aim: Checking the overexpression of Hag-SpeI and Hag-SpeI-DARPin
In  order to check the expression of the proteins a single clone containing  piGEM-019/-20 was used to inoculate 20 mL of LB-Can/Amp. The culture was  incubated at 37°C until an OD of 0,7.
A 1 mL  preinduction sample (PI) was taken (0,7 (favoured OD) : 0,7 (actual OD) = 1 mL).  After that the cultures were induced with 20 µL IPTG for 2 hours.
An  induction sample (I) was taken (320 µL with an OD of 2,2; 350 µL with an OD of  2).
PI and  I were centrifuged at 14000 rpm for 1 min, the supernatant was discarded and  the pellet resuspended in 60 µL water and 40 µL SDS-Buffer.
The  four samples were analysed on an SDS-PAGE gel with 5 and 10 µL volume per sample.
The gel showed that the induction with IPTG was successful and the cells overproduce a protein after 2h incubation.
19.5 Protein overexpression with transformed E. coli BL21 (DE3)
Aim: Overexpression of flagellin modifications for crystallization
After the positive expression test the protein expression was done on a higher level with an overnight lactose induction.
For  that purpose 2 x 1 L LB were inoculated with a solution of LB and every clone  on the cotransformation plate. All clones from the piGEM-019/ -020 + FlaS  cotransformation plates were picked and resuspended in 1 mL LB. 500 µL were  added t the 1 L culture flasks. Each flask was additionally mixed with 1 mL  Ampicillin (100 mg/ mL) and 1 mL Canamycin (50 mg/ mL).
The  expression was induced overnight with lactose (12,5 g/ L). 50 g lactose were  solved in 200 mL millipore water. Each culture was mixed with 50 mL of the  lactose/ water suspension.
The  cultures were incubated overnight at 30°C.
 
Lactose solution
BL21-piGEM-019 culture
BL21-piGEM-020 culture
LB
-
1 L
1 l
Ampicillin
-
1 ml
1 ml
Canamycin
-
1 ml
1 ml
Lactose
50g
-
-
Millipore water
200ml
-
-
Lactose solution
-
50 ml
50 ml
BL21-piGEM-019 clones
-
+
-
BL21-piGEM-020 clones
-
-
+
04.07.2014
Sequencing
Premix
Label-Nr.
Construct
Primers
1
AGB0023413
pET24d-Hag-SpeI-Cup  constr. Cl. 4
T7 Term
14.51 Preparation of plasmids
Aim: Purify the amplified plasmids from overnight cultures
The plasmids were purified from the overnight cultures created in 14.50 using Sigma GenElute Plasmid Miniprep Kit following the included protocol. The plasmids were eluted using 45 µL water.
19.5 Protein overexpression with transformed E. coli BL21 (DE3)
1 mL of  culture was spinned down, mixed with 60 µL water, 40 µL SDS-PAGE loading buffer  and analysed on a SDS-PAGE gel.
The gel shows a high  concentration of a specific protein.
The overnight induced cells were raised till  an OD of 2-3  and spinned down at 4000  rpm.  The pellets were frozen in liquid  nitrogen and stored at -80µC.
19.6 Co-Transformation of piGEM-021 & with FlaiS E. coli BL21(DE3)
Aim: Transformation of plasmids with flagellin-cup modification and chaperone for Flagellin into protein producing strain
piGEM-021  and pFlaiS were used to transform into E.Coli BL21(DE3) and plated out  on LB-Canamycin plates. The plate was incubated at 37µC overnight.
05.07.2014
19.7 Expression-Test with transformed E. coli BL21 (DE3) with piGEM-021
Aim: Checking the overexpression of Hag-SpeI-cup1-1
In  order to check the expression of the proteins a single clone containing  piGEM-021 was used to inoculate 20 mL of LB-Can/Amp. The culture was incubated  at 37°C until an OD of 0,7.
A 1 mL  preinduction sample (PI) was taken (0,7 (favoured OD) : 0,7 (actual OD) = 1 mL).  After that the cultures were induced with 20 µL IPTG for 2h.
An  induction sample (I) was taken (520 µL with an OD of 1,34).
PI and  I were centrifuged at 14000 rpm for 1 min, the supernatant was discarded and  the pellet resuspended in 60 µL water and 40 µL SDS-Buffer.
The  samples were analysed on an SDS-PAGE gel with 5 and 10 µL volume per sample.  Additionally PI and I from the expression test with transformed piGEM-021 were  loaded on the gel as well as a comparison.
The gel shows a high concentration of a specific protein so that the test could be seen as successful.
19.6 Protein overexpression with transformed E. coli BL21 (DE3) (piGEM-021)
Aim: Overexpression of flagellin modifications for crystallization
After  the positive expression test the protein expression was done on a higher level  with an overnight lactose induction.
For  that purpose 2 x 1 L LB were inoculated with a solution of LB and every clone  on the cotransformation plate. All clones from the piGEM-021 + FlaiS  cotransformation plates were picked and resuspended in 1 mL LB. 500 µL were  added t the 1 L culture flasks. Each flask was additionally mixed with 1 mL Ampicillin  (100 mg/ mL) and 1 mL Canamycin (50 mg/ mL).
The  expression was induced overnight with lactose (12,5 g/ L). 25 g lactose were  solved in 100 mL millipore water which was heated 1 min in the microwave for  solvation. Each culture was mixed with 50 mL of the lactose/ water suspension.
The  cultures were incubated overnight at 30°C.
06.07.2014
20.11 Overnight culture of blue clones
Aim: transformation of Bacillus subtilis WT3610 with plasmid
Colonies  were grown on the plates with transformed piGEM-021. The blue/ white screening  showed positive transformed blue clones.   3 clones of different morphology per plate were picked and used for  inoculation of LB-MLS (4 mL LB, 4 µL lincomycin, 4 µL erythromycin). Incubation  was carried out overnight at 30°C with the 3 cultures.
19.6 Protein overexpression with transformed E. coli BL21 (DE3) (piGEM-021)
The gel  shows a high concentration of a specific protein after overnight induction with  lactose referring to the Hag-Cup construct.
The  overnight induced cells were raised till an OD of 2-3  and spinned down at 4000 rpm.  The pellets were frozen in liquid nitrogen  and stored at -80°C.
07.07.2014
20.12 First temperature shift
Aim: integration of  pMAD-Insert into Bacillus chromosome via flanks
The 3  overnight cultures were used to inoculate 10 mL LB MLS until  the culture obtained an OD of 0,1. The  cultures were incubated at 30°C for 2 hours.
Then  the temperature was shifted to 42°C for 6 hours. Unfortunately the temperature  decreased to 33°C because of a wrong incubator setting. New overnight cultures  were inoculated with 50 µL from those used in the morning. The new cultures  were incubated overnight at 30°C.
14.52 Transformation of BL21 cells
Aim: Transform E. coli BL21(DE3) with PheA-single-pET28a
40 µL E. coli BL21(DE3) electrocompetent cells were transformed with 1 ng of the previously purified plasmid PheA-single-pET28a from two different clones, incubated for 1 h at 37 °C and plated out on LB-Kan50 plates that were incubated overnight at 37 °C.
08.07.2014
Sequencing  results of constructs from 04.07.2014
Premix
Label-Nr.
Construct
Result
1
AGB0023413
pET24d-Hag-SpeI-Cup  constr. Cl. 4
Correct insertion of  Cup1-1 into Hag
14.53 Overnight culture
Aim: Inoculate Overnight culture of PheA-single-pET28a
5 mL of LB-Kan50 medium were inoculated with a clone from 14.52 bearing the PheA-single-pET28a plasmid.
20.13 First temperature shift
Aim: integration of  pMAD-Insert into Bacillus chromosome via flanks
The 3  overnight cultures were used to inoculate 10 mL LB MLS until  the culture obtained an OD of 0,1. The  cultures were incubated at 30°C for 2h.
Then  the temperature was shifted to 42°C for 6h.
After  the heat shock 100 µL dilutions from 10-4 to 10-6 of each  culture were plated out on MLS-X-Gal so that nine plates could be incubated  overnight at 42°C.
19.7 Purification of Flagellin-Hag-SpeI from frozen pellet
The  pellets which were stored at -80°C were resuspended in 20 mL buffer A and  cracked with the microfluidizer. The cell lysate was centrifuged at 20000 rpm  for 20 min at 4&degC. The supernatant was transferred to a new 50 mL falcon and  stored on ice.
The following steps were performed:
equilibration with Buffer A 10 min
taking 40 µL supernatant  (load)+ 10 µL SDS-Buffer - L-Sample
50 mL load on column
taking 40 µL of flow through + 10 µL SDS-buffer - FT-Sample
first washing with 25ml Buffer A (half the load)
taking 40 µL of washing flow through + 10 µL SDS-Buffer - W-Sample
equilibrate with Buffer B (output pipe off the column into glas, input into B)
hanging pipe on column, 6 x 2 ml Evolutions - E 1-6
taking 40 µL of Elution 1-6 + 10 µL SDS-Buffer E-sample 1-6
regeneration of column:
10min water
10min EDTA
10min water
10min NiSO4
10min water
Meanwhile  the elutions 1-6 were pooled (combined 12 mL) in a 50 mL falcon and transferred  in a concentrator column which was centrifuged until the volume in the filter  was 1,5 mL for injection into the gel filtration station.
Parallel  the L-, FL-, W- and E 1-6-samples were analysed via SDS-PAGE gel in order to  proof the purification in the elution fractions.
The gel  shows a huge protein concentration in E1 and E2 so that the purification could  be continued with gel filtration. For that purpose the filtrate  out of the concentrator was transferred into a new 2 mL tube resuspending very  well without air bubbles.
17 mL  of GeFi-buffer were injected into the loop. After that the protein filtrate was  injected as well.
The  GeFi was started according to the following settings:
Flow rate: 2500
max. pressure: 0,65 mPa
First endtimer at 90 mL volume before fractioning
After  observing a peak on the screen 40 µL of different fractions (C7 , C9, C11, D1,  D3, D5, D7) covering the peak were taken for analysis via SDS-PAGE gel.
The gel  shows that the elutions contain the protein concentrated. The Elutions C7-D3  were pooled and transferred to the concentrator until an end volume of 200 µL.
The  protein concentrate was used for crystallisation pure and in a 1:2 dilution  with GeFi buffer.
Core I  and II were used for setting drops.
Sequencing  of purified cPCR samples from pMAD transformation
Premix
Label-Nr.
Construct
Primers
1
AGB0023414
Hag-Spe1 in B.s.  genome clone 1
Hag-Nco-fw
2
AGB0023415
Hag-Spe1 in B.s.  genome clone 3
Hag-Nco-fw
3
AGB0023416
Hag-Spe1-DARPin  in B.s. genome clone 1
Hag-Nco-fw
4
AGB0023417
Hag-Spe1-DARPin  in B.s. genome clone 3
Hag-Nco-fw
09.07.2014
20.14 Second temperature shift
Aim: flip out of the pMAD backbone
One  blue colony per diluted clone was used to inoculate 4 mL LB. The 6 cultures  were incubated at 30°C for 6 hours and afterwards for 3 hours at 42°C.
Dilutions  from 10-4 - to 10-6 were plated out on 18 X-Gal plates  without MLS selection. The positive clones should not contain the resistance  inside the backbone as well as the galactosidase. The plates were incubated at  42°C overnight.
Results of Sequencing  of cPCR samples from pMAD transformation from 08.07.14
Premix
Label-Nr.
Construct
Result
1
AGB0023414
Hag-SpeI in B. subtilis genome clone 1
No SpeI insertion
2
AGB0023415
Hag-SpeI in B. subtilis genome clone 3
No SpeI insertion
3
AGB0023416
Hag-SpeI-DARPin  in B. subtilis genome clone 1
Not sequenced through
4
AGB0023417
Hag-SpeI-DARPin  in B. subtilis genome clone 3
Not sequenced through
The sequencing will be repeated with the reverse Primer instead of the  forward primer. In case of the Hag-SpeI clones, different clones have to be  taken for sequencing.
19.7 Purification of Flagellin-Hag-SpeI from frozen pellet
After one day in well B1 Core I fine crystals could be seen.
14.54 Test Expression
Aim: Test the expression of PheA-single
Using the preculture from 14.53 60 mL of LB-Kan50 medium were inoculated with 600 µL of the preculture and incubated at 37 °C and 220 rpm until OD600 reached 0.5. The culture was cooled down to 18 °C and the protein production was induced with 12 µL of 500 mM IPTG. Over night the culture was incubated at 18 °C and 220 rpm. Afterwards the cell culture was centrifuged (17000 rpm, 20 min, 4 °C), the pellet resuspended in 3 mL buffer A and stored at -20 °C. A glycerolstock of the strain containing the PheA-single-pET28a plasmid was stored at -80 °C.
10.07.2014
Sequencing  of cPCR samples from pMAD transformation
Premix
Label-Nr.
Construct
Primer
1
AGB0023439
Hag-SpeI in B. subtilis genome clone 7
Hag-BamHI-rv
2
AGB0023440
Hag-SpeI-DARPin  in B. subtilis genome clone 1
Hag-BamHI-rv
3
AGB0023441
Hag-SpeI-DARPin  in B. subtilis genome clone 3
Hag-BamHI-rv
19.8 Purification of Flagellin-Hag-Cup1-1 from frozen pellet
The  pellets which were stored at -80°C were resuspended in 20 mL buffer A and  cracked with the microfluidizer. The cell parts were spinned down at 20000 rpm  for 20 min at 4°C. The supernatant was transferred to a new 50 mL falcon and  stored on ice.
The following steps were performed:
equilibration with Buffer A 10 min
taking 40 µL supernatant  (load)+ 10 µL SDS-Buffer - L-Sample
50 mL load on column
taking 40 µL of flow through + 10 µL SDS-buffer - FT-Sample
first washing with 25ml Buffer A (half the load)
taking 40 µL of washing flow through + 10 µL SDS-Buffer - W-Sample
equilibrate with Buffer B (output pipe off the column into glas, input into B)
hanging pipe on column, 6 x 2 ml Evolutions - E 1-6
taking 40 µL of Elution 1-6 + 10 µL SDS-Buffer E-sample 1-6
regeneration of column:
10min water
10min EDTA
10min water
10min NiSO4
10min water
Meanwhile  the Elutions 1-6 were pooled (combined 12 mL) in a 50 mL falcon and transferred  in a concentrator column which was centrifuged until the volume in the filter  was 1,5 mL for injection into the gel filtration station.
Parallel  the L-, FL-, W- and E 1-6-samples were analysed on a SDS-PAGE gel in order to  proof the purification in the elution fractions.
The gel  shows a huge protein concentration in E1 and E2 so that the purification could  be continued with the gel filtration. For that purpose the filtrate  out of the concentrator was transferred into a new 2 mL tube resuspending very  well without air bubbles.
17 mL  of GeFi-buffer werd injected into the loop. After that the protein filtrate was  injected as well.
The  GeFi was started according to the following settings:
Flow rate: 2500-3000 µL / min
max. pressure: 0,60 -  0,65 mPa
First  end timer at 80-90 mL volume before fractioning
Injectvalve
After  observing a peak on the screen 40 µL of different fractions (C7 , C9, C11, D1,  D3, D5, D7) covering the peak were taken for analysis via SDS-PAGE gel.
The gel  shows that the elutions contain the protein concentrated. The Elutions  C7-D7were pooled and transferred to the concentrator until an end volume of 200 µL.
The protein  concentrate (27,4 mg/mL A280) was used for crystallization pure and in a 1:2  dilution with GeFi buffer.
Core I  - IV were used for setting drops.
20.15 selection of positive clones
Aim: checking the correct flip out of the pMAD backbone
From  the dilution plates was one WHITE clone picked and transferred on a Master  X-Gal Plate as well as on a MLS plate so that nine clones were proven for the right  integration of the insert although flipping out the pMAD backbone.
The  plates were incubated at 42°C overnight.
13.66 Digest of piGEM-002 NcoI/SpeI
Aim: digest for isolation of piGEM-002 without GFP, replacing it with mCherry out of pMA17
piGEM-002 was digested with NcoI/SpeI in order to isolate the backbone without the GFP so that an mCherry could be integrated via Gibson assembly.
Component
Volume (µl)
piGEM-002 (73 ng/µL)
14
CutSmart 10x
2
SpeI
0,2
NcoI
0,2
Water
3,6
Total Volume
20
The digested vector was isolated via gel extraction.
14.55 Purification of test expression
Aim: Purify PheA-single on a small scale
Lysozyme was added to the cell suspension from 14.54 to a final concentration of 1 mg/mL and incubated on ice for 30 min. Afterwards the cells were lysed using ultra sound (2x 18 pulses, 30 s on ice inbetween). To clear the suspension of cell debris it was centrifuged (13000 rpm, 15 min, 4 °C). The supernatant was purified using QIAGEN Ni-NTA Spin Columns. An SDS-PAGE gel showed that the protein is produced and can be purified using Ni affinity chromatography.
60 mL LB-Kan50 medium was inoculated from the glycerolstock prepared in 14.54 and incubated at 37 °C and 220 rpm over night.
13.67 mCherry PCR amplification
Aim: PCR for the amplification of mCherry
Mix
Master-Mix I
Master-Mix II
Template: pMA17 1:10 (25 ng/ µL)
5
5
Primer iGEM-027
5
-
Primer iGEM-028
5
-
Primer iGEM-029
-
5
Primer iGEM-030
-
5
Phusion DNA-polymerase
10
10
Phusion Buffer 5x
40
40
dNTP-mix
5
5
MgCl2
5
5
DMSO
5
5
Water
120
120
Total Volume
200
200
The PCR was ran as a gradient from 55-65°C annealing temperature overnight.
The gel shows the successful amplification of the two parts of mCherry.
11.07.2014
Sequencing results  of cPCR samples from pMAD transformation
Premix
Lebel-Nr.
Construct
Primer
1
AGB0023439
Hag-SpeI in B. subtilis genome clone 7
Positive, but HagII deletion/ frame shift
2
AGB0023440
Hag-SpeI-DARPin  in B. subtilis genome clone 1
1 point mutations in domain, deletion in HagII/ Frameshift
3
AGB0023441
Hag-SpeI-DARPin  in B. subtilis genome clone 3
2 point mutations in domain, deletion in HagII/ Frameshift
Sequencing  of pMAD-constructs
Aim: finding out if the mutations have been in the constructs or made by Bacillus
Premix
Label-Nr.
Construct
Primer
1
AGB0023442
piGEM-005 - Hag-SpeI
pMAD-seq-rv
2
AGB0023443
piGEM-016 - Hag-Cup
pMAD-seq-rv
3
AGB0023444
piGEM-018 - Hag-DARPin
pMAD-seq-rv
13.67 mCherry PCR amplification
Aim: digest for isolation of piGEM-002 without GFP, replacing it with mCherry
Gel analysis shows that the PCR was successful at every annealing temperature. MgCl2 and DMSO were important for the function of the polymerase.
14.56 Expression of PheA-single
Aim: Express PheA-single for further use
10 x 500 mL LB-Kan50 medium were inoculated with 5 mL of the overnight culture prepared in 14.55 and incubated at 37 °C and 220 rpm until OD600 reached 0.5. The flasks were cooled to 18 °C and protein production was induced by adding 100 µL 500 mM IPTG. The cultures were incubated over night at 18 °C and 220 rpm. Afterwards the cells were collected by centrifugation (6000 rpm, 20 min, 4 °C), resuspended in buffer A and stored at -20 °C until further use.
14.07.2014
13.68 Gibson Assembly with mCherry I and II & linearized piGEM-002
Aim: recombination of piGEM-002 and mCherry
Component
Volume (µl)
Gibson-mix
15
Insert I (mCherry I 460 ng/µL)
1,25
Insert II (mCherry II 460 ng/µL)
1,25
piGEM-002 NcoI/SpeI (5,5 ng/µL)
2,5
Total Volume
20
The mix was incubated 1 hour at 50°C, and used to transform E. coli XL1-Blue that were plated out on LB-Amp plates. Incubation overnight at 37°C.
14.57 Purification of PheA-single
Aim: Purify PheA-single in three steps
The resuspended cells from 14.56 were lysed using a french press. The lysate was incubated on ice with 5 µg/mL DNAseA and RNAseI for 10 min and centrifuged (27000 rpm, 45 min, 4 °C) to pellet cell debris. The supernatant was filtered (0.45 µm) and PheA-Arc1p-C-0x purified from the cleared solution by NiNTA affinity chromatography. The combined elution fractions containing crude PheA-Arc1p-C-0x were concentrated to a volume of 2 mL and further purified using an anion exchange column to get rid of possible tRNA contaminations. Finally the buffer was changed to dialysis buffer and the protein concentrated and stored at -80 °C in aliquods for further use.
Sequencing results of pMAD-constructs
Aim: finding out if the mutations have been in the constructs or made by Bacillus
Premix
Label-Nr.
Construct
Primer
1
AGB0023442
piGEM-005 - Hag-SpeI
Deletion in FlankII
2
AGB0023443
piGEM-016 - Hag-Cup
Deletion in FlankII
3
AGB0023444
piGEM-018 - Hag-DARPin
Deletion in FlankII
The Flank was checked by sequencing and showed no deletion in the flank.
15.07.2014
13.68 Gibson Assembly with mCherry I and II & linearized piGEM-002
Aim: recombination of piGEM-002 and mCherry
No colonies were on the plates. The Gibson assembly was repeated with higher concentrated vector.
14.58 PCR of Arc1p-C-single
Aim: Amplify Arc1p-C-single fragment for Gibson Assembly in pET-28a
The PCR with TG_Arc1p-C FP and TG_PheA-Arc1p-C-8x RP should generate the 579 bp fragment for Gibson Assembly from the PheA-Arc1p-C-2x-pET28a construct (14.23).
The used primer concentration was 1 µM, the concentration of the template was 1 ng/µL and we used a hot start to avoid unspecific annealing of primers.
Content
Volume [µL]
Water
29.5
5x GC Buffer
10
DMSO
2.5
TG_Arc1p-C FP
2.5
TG_PheA-Arc1p-C-8x RP
2.5
PheA-Arc1p-C-2x-pET28a
1
dNTPs
1
Phusion-Polymerase
1
Total Volume
50
Step
Temperature [°C]
Time [min:sec]
1
98
2:00
2
add Polymerase
 
3
98
0:10
4
67.5
0:30
5
72
1:15
6
go to 3
32x
7
72
10:00
8
4
hold
The amplified fragment was purified on an agarose gel on which it showed the correct size. It was extracted from the gel for further use in a Gibson Assembly yielding a concentration of 122 ng/µL.
14.59 Gibson Assembly
Aim: Integrate Arc1p-C-single into linearized pET-28a(XhoI,NdeI) via Gibson Assembly
Content
Volume [µL]
Water
5.38
pET-28a(XhoI,NdeI) (24 ng/µL)
4.2
Arc1p-C-single (122 ng/µL)
0.42
2x Gibson Mastermix
10
Total Volume
20
The reaction mixture was incubated for 1 h at 50 °C and 350 rpm. E. coli Top10 electrocompetent cells were transformed with a 1:3 and a 1:6 dilution of the mixture, incubated for 1 h at 37 °C and plated out on LB-Kan50 plates that were incubated overnight at 37 °C.
16.07.2014
14.60 Overnight culture
Aim: Amplify Plasmid in an overnight culture
Six clones of the transformed Top10 cells from 14.59 were picked and used to inoculate overnight cultures (6 x 5 mL).
17.07.2014
13.69 new Gibson Assembly with mCherry I and II & linearized piGEM-002
Aim: recombination of piGEM-002 and mCherry
In order to receive the construct more piGEM-002  a different ration of the used parts was used for a new Gibson assembly.
From Insert I was a higher amount taken becaue of the bigger fragment size. New attempts with 1:2 (vector:insert) and 1:3 ration:
Component
1:2 (µl)
1:3 (µl)
Gibson-mix
15
15
Insert I (mCherry  I 460 ng/µL)
2,2
1,1
Insert II (mCherry  II 460 ng/µL)
2,5
1,25
piGEM-002 NcoI/SpeI (42 ng/µL)
2,7
1,25
Total Volume
20
20
Gibson reactions were used to transform E. coli XL1-Blue after 1h incubation at 50°C and plated out on LB-Amp. Incubation was  proceeded at 37°C overnight.
21.1 Swarming Assays
Aim: Check motility of Bacillus transformants
For  testing the motility swarming agar (0,7% LB-Agar) and swimming agar (0,3%  LB-Agar) were made and given away for autoclaving.
Overnight  cultures of Bacillus transformants (DARPin clone 3 & 8, Cup1-1 clone 2 & 3, Hag-SpeI clone 7), PY79 and WT3610 were incubated at 37°C.
14.61 Isolation of plasmids
Aim: Purify the amplified plasmids from overnight cultures
The plasmids were purified from the overnight cultures created in 14.60 using Sigma GenElute Plasmid Miniprep Kit following the included protocol. The plasmids were eluted using 45 µL water.
18.07.2014
21.1 Swarming Assay
Aim: Check motility of Bacillus transformands
The overnight cultures were set to an OD of 10. 10 µL of each culture were spotted on a swarming and a swimming plate. The distance of the front line was measured after periods of time.
 
DARPin cl. 3
DARPin cl. 8
Cup1-1 cl. 2
Cup1-1 cl. 3
Hag-SpeI cl. 7
WT3610
PY79
Time
Swim
Swarm
Swim
Swarm
Swim
Swarm
Swim
Swarm
Swim
Swarm
Swim
Swarm
Swim
Swarm
1h
-
-
0,5
2
-
1
-
1,5
3
6
2
3
-
-
2,5h
-
-
0,5
3,5
-
3
-
2,5
12
40<
7
20
-
-
19.07.2014
21.2 Swarming Assay
Aim: Check motility of Bacillus transformands
In  order to repeat the swarming assay new overnight cultures of Bacillus subtilis WT3610, the mutants DARPin  clone 3 & 8, Cup1-1 clone 2 , Hag-SpeI clone 7 in 10 mL LB and incubated at  37°C.
20.07.2014
21.2 Swarming Assay
Aim: Check motility of Bacillus transformants
The  overnight cultures were set to an OD of 10. 10 µL of each culture were spotted  on a swarming and a swimming plate. The plates with the drops were dried for  half an hour on the bench. The distance of the front line was measured after  periods of time. The measurement at t=0 is the radius of the start spot. The  following measurements refer to the difference between the start spot and the  new front line.
 
Swarming
Swimming
WT 3610
+
+
Hag-SpeI cl. 7
+
+
DARPin cl. 3
-
-
DARPin cl. 8
-
-
Cup1-1 cl. 3
-
-
Cup1-1 cl. 8
-
-
21.07.2014
13.70 Screening clones of Gibson plates
Aim: checking success of Gibson assembly
On the plate with 1:2 ratio have been 2 clones, on the 1:3 ratio plate have been 3 clones. All clones were picked for inoculation of minipreps (5 mL LB-Amp, incubation overnight).
Sequencing cPCR Products from B. subtilis mutants with Flank fw/rv
Premix
Label-Nr.
Construct
Primer
1
AGB0023452
B. subtilis - Hag-SpeI Cl.7
Flo89 Flank-fw
2
AGB0023453
B. subtilis - Hag-SpeI Cl.7
Flo90 Flank-rv
3
AGB0023454
B. subtilis - Hag-Cup1-1 Cl.2
Flo89 Flank-fw
4
AGB0023455
B. subtilis - Hag-Cup1-1 Cl.2
Flo90 Flank-rv
14.62 Transformation of BL21 cells
Aim: Transform E. coli BL21(DE3) with Arc1p-C-single-pET28a
40 µL E. coli BL21(DE3) electrocompetent cells were transformed with 1 ng of the previously purified plasmid Arc1p-C-single-pET28a from two different clones, incubated for 1 h at 37 °C and plated out on LB-Kan50 plates that were incubated overnight at 37 °C.
19.9 Protein overexpression with transformed E. coli BL21 (DE3) (piGEM-021)
Aim: Overexpression of flagellin modifications for a second crystallization attempt
After freshly packed GeFi Columns we decided to purify the Fla-Cup again for crystallization.
For that purpose 2 x 1 L LB were inoculated with a solution of LB and  every clone on the cotransformation plate. All clones from the piGEM-021 +  FlaiS cotransformation plates were picked and resuspended in 1 mL LB. 500 µL  were added to the 1 L culture flasks. Each flask was additionally mixed with 1  mL ampicillin (100 mg/ mL) and 1 mL canamycin (50 mg/ mL).
The expression was induced overnight with lactose (12,5 g/ L). 25 g  lactose were solved in 100 mL millipore water which was heated 1 min in the  microwave for solvation. Each culture was mixed with 50 mL of the lactose/  water suspension.
The cultures were incubated overnight at 30°C.
21.3 Expression of Flagellin
Aim: Check the expression of the modified flagella of Bacillus mutants
In order to check the expression of  the flagellum in the Bacillus mutants cultures  in 100 mL LB medium were grown until an OD of 0,8 at 37°C. 10 mL culture were centrifuged  at 4000 rpm for 10 min to get the pellet. The pellet was resuspended in 60 µL  water and 40 µl SDS-loading buffer. After that the sample was analysed via  SDS-PAGE. Overexpressed E. coli flagellin was used as a positive control. The analysed samples should run on the same level as the ones produced in E. coli samples in case of a positive expression.
The gel showed that the Hag-SpeI mutant generates a band on the same level as the WT3610 which is a positive result because the modified flagellin with SpeI site has no remarkable change of molecule mass. The Bacillus transformation was successful though.
The mutants with the inserted Cup and DARPin  showed no significant expression of the flagellin. Either the export of the flagellin  is not possible for Bacillus because of the size/ folding of our constructs and  ends in degradation of our modifications or the expression is too low to be  analysed via SDS-PAGE. In that case it would not be efficient enough for further usage. The results of the SDS-PAGE fits to the  previously made Swarming assays.
A new strategy had to be planned for further  work.
22.07.2014
Sequencing cPCR Products from B. subtilis mutants with Flank fw/rv
Premix
Label-Nr.
Construct
Results
1
AGB0023452
B. subtilis - Hag-SpeI Cl.7
 
2
AGB0023453
B. subtilis - Hag-SpeI Cl.7
 
3
AGB0023454
B. subtilis - Hag-Cup1-1 Cl.2
 
4
AGB0023455
B. subtilis - Hag-Cup1-1 Cl.2
 
19.9 Protein overexpression with transformed E. coli BL21 (DE3) (piGEM-021)
Aim: Overexpression of flagellin modifications for a second crystallization try
The  overnight cultures were grown till an OD of 2-3 and centrifuged. The pellets  were frozen away at -80°C.
13.70 screening clones of Gibson plates
Aim: checking success of Gibson assembly
The isolated plasmids were used for PCR analysis. The primers iGEM-27 and -30 were used for a PCR to amplify the whole mCherry with a size of ca. 711 bp.
Mix
Mastermix
PCR attempt
Template: plasmid from clones 1:10
-
0,5
Primer iGEM-027
3
-
Primer iGEM-030
3
-
dNTP-mix
3
-
Phusion DNA-polymerase
3
-
Phusion Buffer 5x
36
-
MgCl2
-
-
DMSO
-
-
Water
132
-
Mastermix
-
19,5
Total Volume
180
20
Step
Temperature °C
Time
1
98
3 min
2
98
30 sec
3
55
30 sec
4
72
1 min
5
Go To 2
30x
6
72
4min
7
8
infinite
The gel shows that clone 1:3.2 might be positive for containing the mCherry fragment. The clone has to be sequenced.
21.2 Swarming Assay
Aim: Check motility of Bacillus transformands with D2-3-domain
The mutants showed expression on the SDS-gel. Swarming and swimming were tested to  check the motility of his mutants for further planning of our domain  constructions. The hag-construct contains the D2-3 domain of Salmonella.
WT3610  and the D2-3-mutant were used to inoculate 50 mL LB-medium for overnight culture  incubated at 37&degC.
Sequencing cPCR Products from B. subtilis mutants with flank fw/rv
Premix
Label-Nr.
Construct
Results
1
AGB0023452
B. subtilis - Hag-SpeI Cl.7
Correct
2
AGB0023453
B. subtilis - Hag->SpeI Cl.7
Correct
3
AGB0023454
B. subtilis - Hag-Cup1-1 Cl.2
Correct sequencing  till Hag
4
AGB0023455
B. subtilis - Hag-Cup1-1 Cl.2
Correct sequencing  till Hag
14.63 Overnight culture
Aim: Inoculate Overnight culture of Arc1p-C-single-pET28a
5 mL of LB-Kan50 medium were inoculated with a clone from 14.62 bearing the Arc1p-C-single-pET28a plasmid.
23.07.2014
14.64 Test Expression
Aim: Test the expression of Arc1p-C-single
Using the preculture from 14.63 60 mL of LB-Kan50 medium were inoculated with 600 µL of the preculture and incubated at 37 °C and 220 rpm until OD600 reached 0.5. The culture was cooled down to 18 °C and the protein production was induced with 12 µL of 500 mM IPTG. Over night the culture was incubated at 18 °C and 220 rpm. Afterwards the cell culture was centrifuged (17000 rpm, 20 min, 4 °C), the pellet resuspended in 3 mL buffer A and stored at -20 °C. A glycerolstock of the strain containing the Arc1p-C-single-pET28a plasmid was stored at -80 °C.
19.10 Purification of flagellin-Hag-Cup1-1 from frozen pellet
The  pellets which were stored at -80°C were resuspended in 20 mL buffer A and  cracked with the micro fluidizer. The cell parts were centrifuged at 20000 rpm  for 20 min at 4°C. The supernatant was transferred to a new 50 mL falcon and  stored on ice.
The following steps were performed:
equilibration with Buffer A 10 min
taking 40 µL supernatant  (load)+ 10 µL SDS-Buffer - L-Sample
50 mL load on column
taking 40 µL of flow through + 10 µL SDS-buffer - FT-Sample
first washing with 25ml Buffer A (half the load)
taking 40 µL of washing flow through + 10 µL SDS-Buffer - W-Sample
equilibrate with Buffer B (output pipe off the column into glas, input into B)
hanging pipe on column, 6 x 2 ml elutions - E 1-6
taking 40 µL of elution 1-6 + 10 µL SDS-Buffer E-sample 1-6
regeneration of column:
10min water
10min EDTA
10min water
10min NiSO4
10min water
Meanwhile  the Elutions 1 - 6 were pooled (combined 12 mL) in a 50 mL falcon and transferred  into a concentrator column which was centrifuged until the volume in the filter  was 1,5 mL for injection into the gel filtration station.
Parallel  the L-, FL-, W- and E 1 - 6 -samples were analyzed on a SDS-PAGE in order to  proof the purification in the elution fractions.
The gel  shows a huge protein concentration in E1 and E2 so that the purification could  be continued with gel filtration. For that purpose the filtrate  out of the concentrator was transferred into a new 2 mL tube resuspending very  well without air bubbles.
20 mL  of GeFi-buffer were injected into the 2 mL loop. After that the protein filtrate  was injected as well.
The  GeFi was started according to the following settings:
Flow rate: 2500-3000 µL / min
max. pressure: 0,60 - 0,65 mPa
First endtimer at 80-90 mL volume before fractioning
Injectvalve
After  observing a peak 40 µL of different fractions (C7 , C9, C11, D1,  D3, D5, D7, D9) covering the peak were taken for analysis via SDS-PAGE.
The gel  shows that the elutions contain the protein concentrated. The Elutions C7-D9  were pooled and transferred to a new concentrator until an end volume of 400 µL.
The  concentrated protein had a pink color. The concentrated protein (38 mg/mL A280)  was used for crystallization pure and in a 1:2 dilution with GeFi buffer.
Core I  - IV were used for setting drops.
24.07.2014
Sequencing
Premix
Label-Nr.
Construct
Primer
1
AGB0023456
piGEM-002 Nose +  mCherry 1:3 cl.2
TW260
2
AGB0023457
Hag-Cup1-1 cl. 2
Flo90 Flank-rv
13.71 Insertion of degradation tags into Nose- mCherry (piGEM-022 in case of pos. seq.)
Aim: insertion of degradation Tags
For insertion of the DTs via gibson assembly the newly generated piGEM-022 had to be linearized by digestion with SpeI:
Component
Volume (µl)
piGEM-022 (500 ng/µL)
4
CutSmart 10x
2
SpeI
0,2
Water
13,8
Total Volume
20
Incubation at 37°C for 2 hours.
Meanwhile the DTs were generated by a PCR:
Mix
Master Mix
PCR attempt DT 21
PCR attempt DT 22
PCR attempt DT 23
Template
-
-
-
-
Primer iGEM-021
-
0,5
-
-
Primer iGEM-022
-
-
0,5
-
Primer iGEM-023
-
-
-
0,5
Primer iGEM-031
2
-
-
-
dNTP-mix
2
-
-
-
Phusion DNA-polymerase
2
-
-
-
Phusion Buffer 5x
16
-
-
-
MgCl2
-
-
-
-
DMSO
-
-
-
-
Water
48
-
-
-
Mastermix
-
19,5
19,5
19,5
Total Volume
80
20
20
20
Step
Temperature °C
Time
1
98
5 min
2
98
30 sec
3
55
30 sec
4
72
20 sec
5
Go To 2
30x
6
72
4min
7
8
infinite
The DTs were integrated into the vector by Gibson assembly:
Component
Control
DT.21
DT.22
DT.23
Gibson-mix
15
15
15
15
Insert I (DT.x 600 ng/µL)
-
2,5
2,5
2,5
piGEM-002 Nco/SpeI (130 ng/µL)
2,5
2,5
2,5
2,5
Water
2,5
-
-
-
Total Volume
17,5
20
20
20
The reactions  were incubated at 50°C for an hour and used to transform E. coli XLI-blue that were plated out on LB-Amp. Incubation was carried out over night.
14.65 Purification of test expression
Aim: Purify Arc1p-C-single on a small scale
Lysozyme was added to the cell suspension from 14.64 to a final concentration of 1 mg/mL and incubated on ice for 30 min. Afterwards the cells were lysed using ultra sound (2x 18 pulses, 30 s on ice inbetween). To clear the suspension of cell debris it was centrifuged (13000 rpm, 15 min, 4 °C). The supernatant was purified using QIAGEN Ni-NTA Spin Columns. An SDS-PAGE gel showed that the protein is produced and can be purified using Ni affinity chromatography.
60 mL LB-Kan50 medium was inoculated from the glycerolstock prepared in 14.64 and incubated at 37 °C and 220 rpm over night.
25.07.2014
14.66 Expression of Arc1p-C-single
Aim: Express Arc1p-C-single for further use
10 x 500 mL LB-Kan50 medium were inoculated with 5 mL of the overnight culture prepared in 14.65 and incubated at 37 °C and 220 rpm until OD600 reached 0.5. The flasks were cooled to 18 °C and protein production was induced by adding 100 µL 500 mM IPTG. The cultures were incubated over night at 18 °C and 220 rpm. Afterwards the cells were collected by centrifugation (6000 rpm, 20 min, 4 °C), resuspended in buffer A and stored at -20 °C until further use.
13.72 Checking plates of Nose-mCherry-Dt containing clones from Gibson assembly
Aim: checking success of DT 21- 23 integration in Nose-mCherry (piGEM-022)
Sequencing results
Premix
Laber-Nr.
Construct
Result
1
AGB0023456
piGEM-002 Nose +  mCherry 1:3 cl.2
2 point mutations, 1  silent, Phe -> Ile
28.07.2014
14.67 Purification of Arc1p-C-single
Aim: Purify Arc1p-C-single in three steps
The resuspended cells from 14.66 were lysed using a french press. The lysate was incubated on ice with 5 µg/mL DNAseA and RNAseI for 10 min and centrifuged (27000 rpm, 45 min, 4 °C) to pellet cell debris. The supernatant was filtered (0.45 µm) and PheA-Arc1p-C-0x purified from the cleared solution by NiNTA affinity chromatography. The combined elution fractions containing crude PheA-Arc1p-C-0x were concentrated to a volume of 2 mL and further purified using an anion exchange column to get rid of possible tRNA contaminations. Finally the buffer was changed to dialysis buffer and the protein concentrated and stored at -80 °C in aliquods for further use.
31.07.2014
19.11 crystallization of Hag-Cup
Protein crystals can be seen in the wells of the first Hag-Cup crystallization attempt from 19.9
Retrieved from " http://2014.igem.org/Team:Marburg:Project:Notebook:July "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
