<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Marburg_Project_Notebook_Methods"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Marburg:Project:Notebook:Methods">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Marburg:Project:Notebook:Methods&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Marburg:Project:Notebook:Methods&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Marburg:Project:Notebook:Methods&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Marburg:Project:Notebook:Methods" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Marburg:Project:Notebook:Methods</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="navbox"><NAV class="menu center first" aria-haspopup="true" aria-controls="p1"><UL id="p1" class="first"><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg" class="a">Home</A></LI><LI class="current"><A href="https://2014.igem.org/Team:Marburg:Project" class="b">Project</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Safety" class="c">Safety</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Team" class="d">Team</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Policy_Practices" class="e">Policy &amp; Practices</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Attributions" class="f">Attributions</A></LI></UL></NAV><NAV class="menu center second" aria-haspopup="true" aria-controls="p1"><UL id="p1" class="second"><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:Silver" class="d">SilverSURF</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:Tumor" class="c">CancerSURF</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:NRPS" class="e">RiboSURF</A></LI><LI class="current"><A href="https://2014.igem.org/Team:Marburg:Project:Notebook" class="f">Notebook</A></LI><LI class="last" aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:Parts" class="g">Parts</A></LI></UL></NAV></DIV><DIV class="mr_content"><H1>

Notebook: Methods

</H1><UL id="submenu"><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook">March</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:April">April</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:May">May</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:June">June</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:July">July</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:August">August</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:September">September</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:October">October</A></LI><LI class="current"><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:Methods">Methods</A></LI><LI><A href="#competent_cells">Competent <I>E. coli</I></A></LI><LI><A href="#transformation_cultivation">Transformation</A></LI><LI><A href="#expression_text">Expression Test</A></LI><LI><A href="#comp_bac">Competent <I>B. subitilis</I></A></LI><LI><A href="#miniprep">Miniprep</A></LI><LI><A href="#gelex">Gel Extraction</A></LI><LI><A href="#pmad_trans">pMAD</A></LI><LI><A href="#comp_bac_salt">High Salt/Low Salt</A></LI><LI><A href="#spizizens">Transf. <I>B. subitilis</I></A></LI><LI><A href="#pur_ni-nta">Ni-NTA column</A></LI><LI><A href="#split_caco2">Caco-2-cells</A></LI><LI><A href="#pcr_purification">PCR Purification</A></LI><LI><A href="#annealing_pnk">Annealing PNK</A></LI><LI><A href="#med">Medium</A></LI><LI><A href="#buff">Buffer</A></LI><LI><A href="#microtiter">Microtiter Assay</A></LI></UL><DIV class="method"><FIELDSET class="competent_cells"><LEGEND><A name="competent_cells">Competent Cells <I>Escherichia coli</I></A></LEGEND><DIV class="exp-content"><P><STRONG>1. Peparation for preculture</STRONG></P><UL class="comp"><LI>50 µL Aliquod in 2x5ml LB  each</LI><LI>incubation Overnight at 37°C   </LI></UL><P><STRONG>2. Preparation for main culture</STRONG></P><UL class="comp"><LI>5ml of preculture in 250ml LB</LI><LI>incubation at 37°C until 0,5 OD (2h) →10min on ice</LI></UL><P><STRONG>3. Making main culture competent</STRONG></P><UL class="comp"><LI>Pellets: 4x50ml in 4 Falcons</LI><LI>centrifugation at °C, 3500rpm, 15min</LI><LI>resuspending pellets in 4ml HTP* buffer for each Falcon (16ml total)</LI><LI>transfer new suspensions in 1 falcon → 10min on ice</LI><LI>centrifugation at 4°C, 3500rpm, 15min</LI><LI>resuspend pellet in 3ml HTP* buffer</LI><LI>add 225 µL DMSO</LI><LI>50 µ aliquods → freeze in N2 (l)</LI><LI>store at -80°C</LI></UL></DIV></FIELDSET></DIV><DIV class="method"><FIELDSET class="transformation_cultivation"><LEGEND><A name="transformation_cultivation">Transformation and Cultivation</A></LEGEND><DIV class="exp-content"><P><STRONG>1. Transformation</STRONG></P><UL class="comp"><LI>transfer 1 ÂµL plasmid into aliquots for 10min on ice</LI><LI>45sec. Heat shock at 42Â°C</LI><LI>10 min on ice</LI></UL><P><STRONG>2. Cultivation of transformed cells</STRONG></P><UL class="comp"><LI>add 700 ÂµL LB to transformed cells                                     </LI><LI>incubation for 1,5h at 37Â°C</LI><LI>centrifugation for 3min at 13000rpm</LI><LI>get off 600 ÂµL per 1,5ml tube</LI><LI>resuspend pellet in left 100ÂµL LB</LI><LI>plating on agar plate</LI><LI>incubation overnight</LI></UL></DIV></FIELDSET></DIV><DIV class="method"><FIELDSET class="expression_text"><LEGEND><A name="expression_text">Expression Test: Induction + SDS-Gel </A></LEGEND><DIV class="exp-content"><P><STRONG>1. Picking colonies</STRONG></P><UL class="comp"><LI>inoculate 20 ml LB medium (sterile) containing 20ÂµL antibiotics**  and incubate until  an OD of 0,6</LI><LI>45sec. Heat shock at 42Â°C</LI><LI>10 min on ice</LI></UL><P><STRONG>2. Induction with IPTG/ Lactose</STRONG></P><UL class="comp"><LI>before induction:  0,7/ OD = sample  volume</LI><LI>pellet + 80 ÂµL water were resuspended in 20ÂµL loading buffer  (preinduction sample) </LI><LI>induction with 100 ÂµL IPTG/ lactose at an OD of 0,6</LI><LI>incubation for 1,5h    - 4 h max</LI><LI>after induction:  200 ÂµL sample    </LI><LI>pellet  +80 ÂµL water + 20 ÂµL loading buffer  resuspended (induction sample)</LI><LI>big volume gap because of   decreased growth of bacteria   
		    after induction</LI></UL><P><STRONG>3.Expression test with induced culture</STRONG></P><UL class="comp"><LI>transfer in 2x50ml falcons</LI><LI>centrifugation at 4Â°C, 3500rpm, 15min</LI><LI>washing  pellets in 10ml buffer A</LI><LI>cracking  cells with micro fluidizer</LI><LI>thermo  centrifugation for 20 min â€“ 4Â°C</LI><LI>200 ÂµL Ni-NTA beats +  supernatant</LI><LI>5min 4000  rpm → pellet</LI><LI>resuspended  pellet in 500 ÂµL  Buffer A (low imidazole lv)</LI><LI>centrifugation  1 min â€“ 4000rpm</LI><LI>pellet  resuspended in buffer A</LI><LI>centrifugation  1 min â€“ 4000rpm</LI><LI>pellet  resuspended in 200 ÂµL Buffer B (elution)</LI><LI>centrifugation  1min â€“ 13000</LI><LI>80ÂµL supernatant (incl.  protein)+ 20ÂµL  loading buffer</LI></UL><P><STRONG>4. Expression test with induced culture → SDS-Gel</STRONG></P><UL class="comp"><LI>ladder â€“ 5 ÂµL</LI><LI>PIâ€“ preinduction sample - 10ÂµL</LI><LI>I    - induction sample -10 ÂµL</LI><LI>E    -elution sample -10 ÂµL</LI></UL></DIV></FIELDSET></DIV><DIV class="method"><FIELDSET class="comp_bac"><LEGEND><A name="comp_bac">Making Competent <I>Bacillus subtilis</I></A></LEGEND><DIV class="exp-content"><P>B. subtilis (PY79) is naturally competent (Albano et al., 1987). In order to increase the transformation rate, Bacillus can be grown in special media to enhance the competence. To achieve this, 20ml SPC-medium was inoculated with half a well grown LB-plate and incubated at 37Â°C until the cells reached the stationary phase. (Optical density OD550 nm does not change within a a timespan of 30 min) The cells were transfered into 100ml SPII-medium and incubated for 90min at 37Â°C. Afterwards, the cells were centrifuged (500 rpm, 15min, Rotor: BS4402/A Heraeus). The pellet was resuspended in 1ml glycerine (50% (v/v)). Samples were aliquoted and stored at -80Â°C.</P><P>For the transformation with plasmid-DNA 100Âµl cells and 5-7Âµl DNA (chromosomal DNA: 0,1-1Âµl DNA) were mixed andd incubated at 37Â°C for 30min. Afterwards, LB-plates (with antibiotics for selection) were inoculated with the culture and incubated at 30Â°C until colonies formed.</P><P><STRONG>SPC-Medium: </STRONG></P><TABLE width="100%" border="1"><TBODY><TR><TD>T-Base (10x) </TD><TD>2 ml</TD></TR><TR><TD>D-Glucose Monohydrate</TD><TD> 0,5% (w/v)</TD></TR><TR><TD>MgSO4 </TD><TD>0,018% (w/v) </TD></TR><TR><TD>Caseinhydrolysate </TD><TD>0,025% (w/v)</TD></TR><TR><TD>Yeast-extract </TD><TD>0,2% (w/v)</TD></TR><TR><TD>ddH2O ad </TD><TD>20 ml</TD></TR></TBODY></TABLE><P><STRONG>SPII-Medium </STRONG></P><TABLE width="100%" border="1"><TBODY><TR><TD>T-Base (10x)</TD><TD> </TD><TD>10 ml</TD></TR><TR><TD>D-Glucose Monohydrat </TD><TD>50% (w/v)* </TD><TD>1 ml</TD></TR><TR><TD>MgSO4</TD><TD> 1,2% (w/v)</TD><TD>7 ml </TD></TR><TR><TD>Caseinhydrolysate </TD><TD>1% (w/v) </TD><TD>1 ml</TD></TR><TR><TD>Yeast-extract</TD><TD> 10% (w/v)</TD><TD>1 ml</TD></TR><TR><TD>CaCl2 </TD><TD>0,1 M </TD><TD>500 Î¼l </TD></TR><TR><TD>ddH2O ad </TD><TD> </TD><TD>100 ml </TD></TR></TBODY></TABLE><P><STRONG>10x T-Base: </STRONG></P><TABLE width="100%" border="1"><TBODY><TR><TD>(NH4)2SO4</TD><TD>150 mM</TD></TR><TR><TD>K2HPO4*3 H2O </TD><TD>1 M</TD></TR><TR><TD>KH2PO4</TD><TD>440 mM</TD></TR><TR><TD>Tri- Natriumcitrat di- Hydrat</TD><TD>39 mM</TD></TR></TBODY></TABLE><P>ddH2O ad 1000 ml</P><P>*Glucose was substituted with D-fructose in case one of the bacteria strains has a gene inserted under the xylose-promoter, since glucose does inhibit the uptake of xylose</P></DIV></FIELDSET></DIV><DIV class="method"><FIELDSET class="miniprep"><LEGEND><A name="miniprep">Miniprep (Plasmid DNA Miniprep Kit II, Omega)</A></LEGEND><DIV class="exp-content"><UL class="miniprep"><LI>centrifuge at 10.000g for 1min</LI><LI>add 250µl Resuspension Buffer P1 per tube (incl. RNase), vortex</LI><LI>transfer suspension into new 1,5 ml tube</LI><LI>add 250µl Lysis Buffer P2 per tube, invert until lysate is clear (2-3min)</LI><LI>add 350µl Neutralisation Buffer N3 per tube</LI><LI>invert immediately (precipitate forms)</LI><LI>centrifuge 10min at 13.000 rpm</LI><LI>transfer supernatant onto column</LI><LI>centrifuge for 1min
          </LI><LI>add 500µl Washing Buffer PE (HBC Buffer) per tube</LI><LI>centrifugation for 1min throw filrate away</LI><LI>add 700µl DNA Wash Buffer per tube</LI><LI>centrifugation for 1min throw filrate away</LI><LI>add 30µl milipore per tube â€“ let column sit for 2min</LI><LI>centrifuge for 2min</LI></UL></DIV></FIELDSET></DIV><DIV class="method"><FIELDSET class="gelex"><LEGEND><A name="gelex">Gel Extraction (QIAquick Gel Extraction Kit)</A></LEGEND><DIV class="exp-content"><UL class="gelex"><LI>band is cut out of the gel under UV light and transferred into a 1,5ml tube</LI><LI>weigh the gel with the tube</LI><LI>add QG (100mg gel → 100µl)</LI><LI>incubate for approx. 10min at 50-60Â°C, vortex to dissolve the gel</LI><LI>put spin colum into 2mlcollection tube</LI><LI>add dissolved gel onto column → centrifuge for 1min</LI><LI>discard flow-through</LI><LI>wash with 750Âµl Buffer PE →
centrifuge for 1min          </LI><LI>discard flow-through</LI><LI>place column in 1,5ml tube</LI><LI>elute with 50Âµl milipore water: incubate 5min at 37Â°C →
centrifuge for 1min</LI></UL></DIV></FIELDSET></DIV><DIV class="method"><FIELDSET class="pmad_trans"><LEGEND><A name="pmad_trans">pMAD-Transformation for Bacillus</A></LEGEND><DIV class="exp-content"><P><STRONG>1. Preculture for competent Bacillus subtilis</STRONG></P><P>Aim:Preparation for mainculture the next day</P><P> 5 mL of LB were inoculated with Bacillus WT3610 from an LB plate and  incubated at 37Â°C overnight.</P><P><STRONG>2. Transformation of competent Bacillus subtilis</STRONG></P><P>Aim: transformation of plasmid into Bacillus subtilis WT3610</P><P>100 ÂµL of overnight culture were added to 10  mL of MNGE-Medium and incubated till an OD of 1,1-1,3 at 37Â°C which could take  4-5h.4</P><P>After reaching OD of 1,1-1,3 400 ÂµL of the  culture were transformed with 1,5 Âµg plasmid. After 1h incubation at 37Â°C 100  ÂµL Expression mix were added and incubated for 1h as well.</P><P>In the end the 500 ÂµL attempt was plated out  on MLS-X-Gal plates and incubated at 30 Â°C overnight until colonies could be  seen. </P><P><STRONG>3. Overnight culture of blue clones</STRONG></P><P>Aim: transformation of plasmid into Bacillus subtilis WT3610</P><P>Colonies were grown on the plates with  transformed plasmid. The blue/ white screening showed positive transformed blue  clones.  3 clones of different morphology  per plate were picked and used for inoculation of LB-MLS (4 mL LB, 4 ÂµL  Lincomycin, 4 ÂµL Erythromycin). Incubation was carried out overnight at 30Â°C  with the cultures. </P><P><STRONG>4. First temperature shift</STRONG></P><P>Aim: integration of  pMAD-Insert  into Bacillus chromosome via flanks</P><P>The   overnight cultures were used to inoculate 10 mL LB MLS until  the culture obtained an OD of 0,1. The  cultures were incubated at 30Â°C for 2h.</P><P>Then the temperature was shifted to 42Â°C for  6h.</P><P>After the heat shock dilutions from 10-4  to 10-6 of each culture were plated out on MLS-X-Gal so that plates  could be incubated overnight at 42Â°C.</P><P><STRONG>5. Second temperature shift</STRONG></P><P>Aim: flip out of the pMAD backbone</P><P>One blue colony per diluted clone was used to  inoculate 4 mL LB. The cultures were incubated at 30Â°C for 6h and afterwards  for 3h at 42Â°C.</P><P>Dilutions from 10-4Â­ to 10-6  were plated out on  X-Gal plates WITHOUT  MLS selection. The positive clones should not contain the resistance inside the  backbone as well as the galactosidase. The plates were incubated at 42Â°C  overnight.</P><P><STRONG>6.1 selection of positive clones</STRONG></P><P>Aim: checking the correct flip out of the pMAD backbone</P><P>From the dilution plates was one WHITE clone  picked and transferred on a Master X-Gal Plate as well as on a MLS plate so  that clones were proven for the right integration of the insert although  flipping out the pMAD backbone.</P><P>The plates were incubated at 42Â°C overnight.</P><P><STRONG>6.2 selection of positive clones</STRONG></P><P>Aim: checking the correct flip out of the pMAD backbone</P><P>The white colonies grew on the X-Gal Master  plates but not on MLS plates so the transformation seemed to be successful. The  Backbone with the MLS resistance flipped out of the genome.</P><P><STRONG>7. cPCR with checked clones</STRONG></P><P>Aim: checking integration of constructs Hag-Spe and Hag-Spe-DARPin into  B. s. genome</P><P>No clone was growing on MLS plates so the  picked clones seemed to be positive. In order to check the correct integration  of the domain constructs a cPCR with the picked clones on the X-Gal plates was  done. The picked B.s. clones were cooked in 10 ÂµL PBS for 5 min at 95Â°C.</P></DIV></FIELDSET></DIV><DIV class="method"><FIELDSET class="comp_bac_salt"><LEGEND><A name="comp_bac_salt">Making Competent Bacillus with the High Salt/Low Salt Methode</A></LEGEND><DIV class="exp-content"><P><EM>modified after Harwood and Cutting 1990</EM></P><P>LB-plates were inoculated with Bacillus subtilies and incubated at 37Â°C overnight.</P><P>3ml HS- (high-salt-) medium were inoculated with a single colony and incubated at 37Â°C overnight (rolling).</P><P>20ml preheated LS- (low salt-) medium was inoculated with 1ml HS-culture and incubated at 30Â°C for 3h at 100rpm in a water bath.</P><P><STRONG>10 x S-Base</STRONG></P><TABLE width="100%" border="1"><TBODY><TR><TD>(NH4)2SO4</TD><TD>2g</TD></TR><TR><TD>K2HPO4</TD><TD>14g</TD></TR><TR><TD>KH2PO4</TD><TD>6g</TD></TR><TR><TD>Natriumcitrat* 2 H2O</TD><TD>1g</TD></TR><TR><TD>aqua dest.      ad </TD><TD>100ml</TD></TR><TR><TD>After being autoclaved 0,1ml 1M MgSO4 (sterile filtered) were added</TD><TD> </TD></TR></TBODY></TABLE><P><STRONG>HS-Medium</STRONG></P><TABLE width="100%" border="1"><TBODY><TR><TD>aqua dest. </TD><TD>74,5 ml</TD></TR><TR><TD>10 x S-Base </TD><TD>10 ml</TD></TR><TR><TD>50% Glucose </TD><TD>1 ml</TD></TR><TR><TD>Tryptophan 5 mg/ml    </TD><TD>1 ml</TD></TR><TR><TD>Phenylalanin 3 mg/ml </TD><TD>1,5 ml</TD></TR><TR><TD>Casein-Hydrolysat 2 %</TD><TD>1 ml</TD></TR><TR><TD>Yeast-extract 10 %</TD><TD>1 ml</TD></TR><TR><TD>Arginin 8 % + Histidin 0,4 %</TD><TD>10 ml</TD></TR><TR><TD>Solutions were sterile filtered and mixed under sterile conditions</TD><TD> </TD></TR></TBODY></TABLE><P><STRONG>LS-Medium</STRONG></P><TABLE width="100%" border="1"><TBODY><TR><TD>aqua dest. </TD><TD>17,2 ml</TD></TR><TR><TD>10 x S-Base </TD><TD>2 ml</TD></TR><TR><TD>50% Glucose </TD><TD>0,2 ml</TD></TR><TR><TD>Tryptophane 5 mg/ml </TD><TD>20 Âµl</TD></TR><TR><TD>Phenylalanine 3 mg/ml </TD><TD>30 Âµl</TD></TR><TR><TD>Casein-Hydrolysat 2 %</TD><TD>0,1 ml</TD></TR><TR><TD>Yeast-extract 10 %</TD><TD>0,2 ml</TD></TR><TR><TD>Spermin 50 mM</TD><TD>0,2 ml</TD></TR><TR><TD>MgCl2 1M</TD><TD>50 Âµl</TD></TR><TR><TD>Solutions were prepared directly before use and mixed under sterile conditions</TD><TD> </TD></TR></TBODY></TABLE><P><STRONG>Transformation</STRONG></P><P>Prepare DNA-Solution (1-5Âµg plasmid DNA or 10-20Âµg chromosomal DNA) in eppendorf cup.
      Add 1ml LS-culture, incubate the transformation samples at 37Â°C for 2h an a thermomixer/roller.
      Centrifuge the transformation samples fpr 20sec, discard some of the supernatant and resuspend the cells.
      Incubate selective plates.</P><P>(Harwood, C.R.,  and Cutting, S.M. (1990) Molecular  biological methods for Bacillus.  John Wiley and Sons Ltd., Chichester, England.)</P></DIV></FIELDSET></DIV><DIV class="method"><FIELDSET class="spizizens"><LEGEND><A name="spizizens">Transformation of Bacillus subtilis with Spizizens minimal medium </A></LEGEND><DIV class="exp-content"><UL><LI>10 ml LB  inoculated with single colony as preculture over night at 37Â°C</LI><LI>Inoculate  10 ml Spizizens minimal medium with the preculture to an OD600 of  0.1</LI><LI>Incubation  at 37Â°C until OD600 1 more or less</LI><LI>Addition  of 5 Âµg plasmid DNA/5-10 Âµg of chromosomal DNA to 1 ml of culture (in a test  tube)</LI><LI>Incubation  at 37Â°C for 2 h</LI><LI>Plate out 200 Âµl and 500 Âµl on different plates containing selecting  antibiotics and incubate the plates at 37Â°C overnight</LI></UL></DIV></FIELDSET></DIV><DIV class="method"><FIELDSET class="pur_ni-nta"><LEGEND><A name="pur_ni-nta">Purification of Ni-NTA column</A></LEGEND><DIV class="exp-content"><P>The following steps were performed for the Ni-NTA Column:</P><UL class="list"><LI>equilibration with Buffer A 10 min</LI><LI>taking 40 ÂµL supernatant  (load)+  10 ÂµL SDS-Buffer - L-sample</LI><LI>50 mL load on column</LI><LI>taking 40 ÂµL of flow through + 10 ÂµL SDS-buffer - FT-sample</LI><LI>first washing with 25ml Buffer A (half the  load)</LI><LI>taking 40 ÂµL of washing flow through + 10 ÂµL SDS-Buffer - W-sample</LI><LI>equilibrate with Buffer B (output pipe off the column into glas, input  into B)</LI><LI>hanging pipe on column, 20 ml Elution - E</LI><LI>taking 40 ÂµL of Elution 1-6 + 10 ÂµL SDS-Buffer E- sample </LI><LI>SDS-PAGE analysis</LI></UL><P>Regeneration of column</P><UL class="list"><LI>10min water</LI><LI>10 min EDTA</LI><LI>10min water</LI><LI>10min NiSO4</LI><LI>10min water</LI></UL></DIV></FIELDSET></DIV><DIV class="method"><FIELDSET class="split_caco2"><LEGEND><A name="split_caco2">Splitting of Caco-2-cells</A></LEGEND><DIV class="exp-content"><P>Caco-2-cells require  DMEM (Dulbeccoâ€™s Modified Eagle Medium) with 20% FCS and L-Glutamine.</P><P>A centrifuge was set  on 4Â°C.</P><P>The medium of the  culture flask was discarded from the side opposite to the adherent cells and  washed with 1X PBS from the opposite site as well.</P><P>The PBS was aspirated  with a glass pipette until the supernatant was clear. After adding 1-2 mL 1x  trypsin from the opposite of the adherent cells the flask was incubated for  5-10 min at 37Â°C depending on how fast the cells come of from the ground. Under  the microscope the free floating cells were checked.</P><P>The cells were  transferred into a 15 mL falcon and rinsed with 10 mL DMEM to collect all  remaining cells in the flask. The 15 mL falcon was filled with DMEM and the  culture spinned down at 1500 rpm  for 5  min at 4Â°C.</P><P>The supernatant was  discarded and resuspended in 2 mL DMEM. The cells were splitted 1:3 which means  that 666 ÂµL were taken from the suspension and transferred into a new culture  flask. After adding 20-25 mL DMEM the flask was incubated at 37Â°C and checked  on the 3rd / 4th day under the microscope.</P></DIV></FIELDSET></DIV><DIV class="method"><FIELDSET class="pcr_purification"><LEGEND><A name="pcr_purification">Purification of PCR Products (QIAquick Gel Extraction Kit) short protocol</A></LEGEND><DIV class="exp-content"><P>short protocol without gel extraction</P><UL class="gelex"><LI>add 3 volumes or QG Buffer to tubes  and 1 volume isopropanol</LI><LI>place QIAquick spin column in 2ml collection tube</LI><LI>centrifuge for 1min →discard flow-through</LI><LI>add 750Âµl Buffer PE</LI><LI>centrifuge for 1min  →discard flow-through</LI><LI>place column in 1,5 ml tube</LI><LI>eluate with 50Âµl milipore water</LI><LI>incubate for 5min at 37Â°C, centrifuge for 1min</LI></UL></DIV></FIELDSET></DIV><DIV class="method"><FIELDSET class="annealing"><LEGEND><A name="annealing_pnk">Annealing with PNK Inactivation</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH width="17%" scope="col">Step</TH><TH width="58%" scope="col">Temperature °C</TH><TH width="25%" scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>37</TD><TD>1h</TD></TR><TR><TH scope="row">2</TH><TD>65</TD><TD>20 min</TD></TR><TR><TH scope="row">3</TH><TD>95</TD><TD>5 min</TD></TR><TR><TH scope="row">4</TH><TD>95-4</TD><TD>1 min/1°C</TD></TR><TR><TH scope="row">5</TH><TD>4</TD><TD>Infinite</TD></TR></TBODY></TABLE></DIV></FIELDSET></DIV><DIV class="method"><FIELDSET class="medium"><LEGEND><A name="med">Medium</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">LB-medium</TH><TH scope="col">Concentration</TH><TH scope="col">Final</TH></TR><TR><TH scope="row">Ampicilin</TH><TD>Ampicilin (1000x) 100Âµl:1000ml </TD><TD>Final: 50µg/ml</TD></TR><TR><TH scope="row">Canamycin</TH><TD>Canamycin (1000x) 100Âµl:100ml</TD><TD>Final: 50µg/ml</TD></TR><TR><TH scope="row">Chloramphenicol</TH><TD>Chloramphenicol (1000x) 100Âµl:100ml</TD></TR></TBODY></TABLE></DIV></FIELDSET></DIV><DIV class="method"><FIELDSET class="bbuffer"><LEGEND><A name="buff">Buffer</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Buffer</TH><TH scope="col">Ingredients</TH></TR><TR><TH scope="row">-2.0 Buffer</TH><TD>2.0 (10x) + BSA (10mg/ml -10) → BSA end concentration 100µg/ml (0,1µg/µl)</TD></TR><TR><TH scope="row">HTP (for 500ml)</TH></TR><TR><TD>10mM Hepes                     </TD><TD>1,19g</TD></TR><TR><TD>15mM CaCl2 x2H20</TD><TD>1,1g</TD></TR><TR><TD>250mM KCl</TD><TD>9,32g</TD></TR><TR><TD>55mM MnCl2 x 4 H2O</TD><TD>5,44g</TD></TR></TBODY></TABLE>Buffer A20mM HEPES
      250mM NaCl
      20mM KCl
      20 mM MgCl
      40 Imidazole
      pH 8,0Buffer B20mM HEPES
      250mM NaCl
      20mM KCl
      20 mM MgCl
      500 mM Imidazole
      pH 8,0GeFi Buffer2mM HEPES
      20mM NaCl
      2mM KCl
      2 mM MgCl
      pH 7,5Inclusion Body Wash Buffer100 mM NaCl
      50 mM Tris
      pH 8.0
      0.5% Triton X-100</DIV></FIELDSET></DIV><DIV class="method"><FIELDSET class="microtiter"><LEGEND><A name="microtiter">Microtiter-Assay</A></LEGEND><DIV class="exp-content"><UL class="comp"><LI>1.	Grow overnight cultures of <I>B. subtilis</I> in LB (+ antibiotic) or minimal medium S750</LI><LI>2.	Prepare medium, cells and Ag+ solutions depending on the assay
				</LI><LI>Cells: dilutions between 1:100 and 1:1000</LI><LI>Ag+ dilutions: 10 mM to 0.1 nM</LI></UL><LI>3.	Add 135 µl Medium in every well of a 96 well plate (Premix of minimal medium and Ag+ dilution)</LI><LI>4.	Add 15 µl cells to the medium</LI><LI>5.	Add 70 µl mineral oil on top</LI><LI>6.  Strains: As fluorescence is measured always include a positive and negative control</LI><LI>X = Medium (Control without cells)</LI><LI>A-H = Different media</LI><LI>1-9 = Strains</LI><LI>10 = Wildtype (negative control)</LI><LI>11 = GFP-Control</LI><UL class="comp"><LI>7.	After the addition of media, cells and mineral oil it is important to check all wells for bubbles that have tob e removed with pipette tips. </LI><LI>8.	Start Plate reader program [120 Cycles]</LI><UL class="comp"><LI>Shake 4 min</LI><LI>Measurement OD<SUB>450</SUB></LI><LI>Shake 1 min</LI><LI>Measurement GFP</LI></UL><LI>9.	Save data</LI><LI>10.	Evaluation: First step sort with R</LI><LI>11.	Evaluation: Second step excel</LI></UL></DIV></FIELDSET></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Marburg:Project:Notebook:Methods">http://2014.igem.org/Team:Marburg:Project:Notebook:Methods</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Marburg:Project:Notebook:Methods" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Marburg:Project:Notebook:Methods" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Marburg:Project:Notebook:Methods&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Marburg:Project:Notebook:Methods&amp;oldid=384838" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>