<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Marburg_Project_Notebook_August"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Marburg:Project:Notebook:August">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Marburg:Project:Notebook:August&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Marburg:Project:Notebook:August&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Marburg:Project:Notebook:August&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Marburg:Project:Notebook:August" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Marburg:Project:Notebook:August</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="navbox"><NAV class="menu center first" aria-haspopup="true" aria-controls="p1"><UL id="p1" class="first"><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg" class="a">Home</A></LI><LI class="current"><A href="https://2014.igem.org/Team:Marburg:Project" class="b">Project</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Safety" class="c">Safety</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Team" class="d">Team</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Policy_Practices" class="e">Policy &amp; Practices</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Attributions" class="f">Attributions</A></LI></UL></NAV><NAV class="menu center second" aria-haspopup="true" aria-controls="p1"><UL id="p1" class="second"><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:Silver" class="d">SilverSURF</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:Tumor" class="c">CancerSURF</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:NRPS" class="e">RiboSURF</A></LI><LI class="current"><A href="https://2014.igem.org/Team:Marburg:Project:Notebook" class="f">Notebook</A></LI><LI class="last" aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:Parts" class="g">Parts</A></LI></UL></NAV></DIV><DIV class="mr_content"><H1>

Notebook: August

</H1><UL id="submenu"><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook">March</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:April">April</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:May">May</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:June">June</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:July">July</A></LI><LI class="current"><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:August">August</A></LI><LI><A href="#01.08.2014">01.08.2014</A></LI><LI><A href="#04.08.2014">04.08.2014</A></LI><LI><A href="#05.08.2014">05.08.2014</A></LI><LI><A href="#06.08.2014">06.08.2014</A></LI><LI><A href="#11.08.2014">11.08.2014</A></LI><LI><A href="#12.08.2014">12.08.2014</A></LI><LI><A href="#13.08.2014">13.08.2014</A></LI><LI><A href="#14.08.2014">14.08.2014</A></LI><LI><A href="#18.08.2014">18.08.2014</A></LI><LI><A href="#19.08.2014">19.08.2014</A></LI><LI><A href="#20.08.2014">20.08.2014</A></LI><LI><A href="#21.08.2014">21.08.2014</A></LI><LI><A href="#22.08.2014">22.08.2014</A></LI><LI><A href="#23.08.2014">23.08.2014</A></LI><LI><A href="#24.08.2014">24.08.2014</A></LI><LI><A href="#25.08.2014">25.08.2014</A></LI><LI><A href="#26.08.2014">26.08.2014</A></LI><LI><A href="#27.08.2014">27.08.2014</A></LI><LI><A href="#28.08.2014">28.08.2014</A></LI><LI><A href="#29.08.2014">29.08.2014</A></LI><LI><A href="#30.08.2014">30.08.2014</A></LI><LI><A href="#31.08.2014">31.08.2014</A></LI></UL><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:September">September</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:October">October</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:Methods">Methods</A></LI><DIV class="notebooky-entry"><H2 class="title"><A name="01.08.2014">01.08.2014</A></H2><FIELDSET class="exp18"><LEGEND><A name="exp18.50">18.50 Design of a Strep-Tag/ StrepDARPidin system</A></LEGEND><DIV class="aim"><P>Aim: Designing a new system for using the Flagellin-DARPin-Konstrukt</P></DIV><DIV class="exp-content"><P>A new plan was made to obtain the flagellas multi-functionality, using an affinity tag. We want to integrate a Strep-Tag into the flagellin so that the DARPin domain can be fused to the flagellum via streptavidin. </P><P>The D2 domain of <I>Salmonella typhimurium</I>'s flagellin was used as a linker between the hag of <I>Bacillus subtilis</I> and the Strep-Tag, furthermore GSGS linkers were used between the D2 and the StrepTag.</P><P>Additionally  the DARPin was designed with an N-terminal His-Tag and C-Terminal streptavidin, so a chimeric protein would be generated.</P><P>The designed  structures were synthesized by Integrated DNA Technologies.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.61">15.61 Design of Hag-D2-Cup</A></LEGEND><DIV class="aim"><P>Aim: Designing a new domain construct for expression of cup1-1-Hag</P></DIV><DIV class="exp-content"><P>We decided to design the flagellin as well with the D2 domain and the cup1-1.</P><P>The designed structures were synthesized by Integrated  DNA Technologies</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="04.08.2014">04.08.2014</A></H2><FIELDSET class="exp14"><LEGEND><A name="exp14.68">14.68 Crystallization of Arc1p-C</A></LEGEND><DIV class="aim"><P>Aim: Crystallize the Arc1p-C-single domain</P></DIV><DIV class="exp-content"><P>For the experiments the sitting drop method in SWISSCI MRC 2 well crystallization plates with a drop size of 1 µL was used (1:1 mixture of protein and crystallization solution) with an reservoir volume of 50 µL. Crystals of Arc1p-C-single were obtained
by room temperature and at 6 mg/mL protein concentration after 1 week in 0.1 M MES, pH 6.0, 10% (v/v) glycerol, 30% (w/v) PEG 600 and 5% (w/v) PEG 1000. To collect data the crystals
were flash frozen in liquid nitrogen and measured at the European Synchrotron Radiation Facility (ESRF) in Grenoble, France at beamline ID23-1. The structure of Arc1p-C-single
was determined by molecular replacement (MR) using the crystal structure of human Arc1p-C (PDB ID: 1FL0).</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="05.08.2014">05.08.2014</A></H2><FIELDSET class="exp18"><LEGEND><A name="exp18.51">18.51 Isolation of flagella from <I>Bacillus subtilis</I> WT3610</A></LEGEND><DIV class="aim"><P>Aim: isolation of filaments of WT3610 as negative control for e.g. FACS analysis &amp; microscopy</P></DIV><DIV class="exp-content"><P>1 L LB was inoculated with WT 3610 and incubated over night at 37°C.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.62">15.62 Restriction digest of piGEM-005 SpeI</A></LEGEND><DIV class="aim"><P>Aim: linearizing vector for Gibson assembly with Hag-D2-Cup and -D2-Strep</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Volume (µl)</TH></TR><TR><TH scope="row">piGEM-005 (100 ng/µL</TH><TD>20</TD></TR><TR><TH scope="row">CutSmart 10x</TH><TD>2,5</TD></TR><TR><TH scope="row"><I>Spe</I>I</TH><TD>0,2</TD></TR><TR><TH scope="row">Water</TH><TD>2,3</TD></TR><TR><TH scope="row">Total Volume</TH><TD>25</TD></TR></TBODY></TABLE><P>Incubation at 37°C for 2 hours – Inactivation by heat shock at 85° for 10  min.</P><P>End concentration: 50 ng/µL</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.51">18.51 <I>Nco</I>I/<I>Bam</I>HI &amp;<I>Nco</I>I/<I>Xho</I>I digest of pET24d</A></LEGEND><DIV class="aim"><P>Aim: purification of vector for ligation with StrepDARPidin gene</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Volume [µl]</TH><TH scope="col">Volume [µl]</TH></TR><TR><TH scope="row">pET24d (80 ng/µL)</TH><TD>30</TD><TD>30</TD></TR><TR><TH scope="row">CutSmart 10x</TH><TD>3,5</TD><TD>3,5</TD></TR><TR><TH scope="row"><I>Nco</I>I</TH><TD>0,25</TD><TD>0,25</TD></TR><TR><TH scope="row"><I>Bam</I>HI</TH><TD>0,25</TD><TD>-</TD></TR><TR><TH scope="row"><I>Xho</I>I</TH><TD>-</TD><TD>0,25</TD></TR><TR><TH scope="row">Water</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row"><STRONG>Total</STRONG> Volume</TH><TD>35</TD><TD>35</TD></TR></TBODY></TABLE><P>Incubation at 37°C for 2h .</P><P>The fragments were purified via gel extraction.</P><P>End concentration: 25 ng/µL pET24d-N/B &amp; 15 ng/µL-N/X</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="06.08.2014">06.08.2014</A></H2><FIELDSET class="exp18"><LEGEND><A name="exp18.52">18.52 Isolation of flagella from <I>Bacillus subtilis</I> WT3610</A></LEGEND><DIV class="aim"><P>Aim: isolation of filaments of WT3610 as negative control for e.g. FACS analysis &amp; microscopy</P></DIV><DIV class="exp-content"><P>The preculture  was centrifuged at 4000 rpm for 15 minutes and the pellet was resuspended in 33 mL TRIS/HCl buffer  (pH 8.0) with 0,5% Brij-58. A stock solution of lysozyme with 10 mg/mL was made.  330 µL were added to the TRIS/HCl buffer mix. The lysis was performed at 4°C while rotating the tubes until the pellet was completely solved which took approx. 2 - 3 hours. DNase (5 µg/ mL) in presence of 0,01 M magnesium chloride was added and incubated for  half an hour at 4°C in the rotator as well.</P><P>After that the  suspension was centrifuged at 10000x g for 10 min. 80 µL of the supernatant were  transferred into a 1,5 mL tube. The supernatant was then centrifuged at 87000xg  for 90min.</P><P>The pellet was  resuspended in 10 mL standard saline citrate (0,01M trisodium citrate &amp;  0,1M sodium chloride, pH 7,3) on ice and 80 µL were taken as well. For fractioning, 2  g of ammonium sulfate (20%) were added slowly until precipitation could be seen.</P><P>After some  minutes of resting the suspension was centrifuged at 20000 rpm for 15 min. The  pellet was then resuspended in 2 mL of standard saline citrate buffer. After  taking a sample of 80 µL the 4 samples ware mixed with 20 µL SDS-loading buffer  each and analysed on an SDS-PAGE.</P><P>The purified  filaments were frozen in liquid nitrogen and then at -80°C.</P><P>The flagella purification was successful. Besides the flagellin the samples also contain the hook proteins, the smaller proteins were removed from the purified pellet.</P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.69">14.69 tRNA<SUP>Phe</SUP> assay</A></LEGEND><DIV class="aim"><P>Aim: Measure aminoacylation levels</P></DIV><DIV class="exp-content"><P>To measure aminoacylation of tRNA<SUP>Phe</SUP> the components were mixed.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Final concentration</TH></TR><TR><TH scope="row">Dialysis-buffer</TH><TD>ad 250 µL</TD></TR><TR><TH scope="row">DTT</TH><TD>1 mM</TD></TR><TR><TH scope="row">MgCl<SUB>2</SUB></TH><TD>20 mM</TD></TR><TR><TH scope="row">KCl</TH><TD>10 mM</TD></TR><TR><TH scope="row">tRNA<SUP>Phe</SUP></TH><TD>10 µM</TD></TR><TR><TH scope="row">L- or D-Phenylalanine</TH><TD>1 mM</TD></TR><TR><TH scope="row">catalytic fusion Protein</TH><TD>10 µM</TD></TR></TBODY></TABLE><P>After mixing, reaction was initiated by the addition of 2 mM ATP and then incubated for 1 to 60 min by 37 °C and 350 rpm. Assays were stopped by the addition of 25 µL 3 M sodium acetate solution. Afterwards tRNAs were precipitated with ethanol. Dissolved pellets were further purified using size-exclusion columns. The elution fraction was split into two aliquots. One samples was treated with 0.2 M NaOH (final concentration: 10 mM) and incubated at 50 °C for 10 min, while the other aliquot was kept on ice. The base treated sample was again acidified by the addition of 3 M sodium acetate (pH 5.2, final concentration: 300 mM). The volume of the untreated sample was adjusted using 2 mM sodium acetate (pH 5.2). Subsequently, both samples were subjected to LCMS analysis using 100 µL for injection.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="11.08.2014">11.08.2014</A></H2><FIELDSET class="exp22"><LEGEND><A name="exp25.0">25. Cell culture assays</A></LEGEND></FIELDSET><FIELDSET class="exp22"><LEGEND><A name="exp25.1">25.1 Splitting of Caco-2-cells</A></LEGEND><DIV class="aim"><P>Aim: Splitting cell culture for growing</P></DIV><DIV class="exp-content"><P>The Caco-2-cells were kindly provided by Dr. Hartmann Reifert (BMFZ). In order to let them grow, the cells had to be  splitted according to a specific protocol. Caco-2-cells require DMEM (Dulbecco’s  Modified Eagle Medium) with 20% FCS and L-Glutamine.</P><P>A centrifuge was precooled to 4°C.</P><P>The medium of the culture flask was discarded  from the side opposite to the adherent cells and washed with 1X PBS from the  opposite site as well.</P><P>The PBS was aspirated with a glass pipette  until the supernatant was clear. After adding 1 - 2 mL 1x trypsin from the  opposite of the adherent cells the flask was incubated for 5 - 10 min at 37°C  depending on how fast the cells unfastened of from the ground. Under the microscope  the free floating cells were checked.</P><P>The cells were transferred into a 15 mL falcon  and rinsed with 10 mL DMEM to collect all remaining cells in the flask. The 15  mL falcon was filled with DMEM and the culture centrifuged at 1500 rpm  for 5 min at 4°C.</P><P>The supernatant was discarded and resuspended  in 2 mL DMEM. The cells were splitted 1:3 which means that 666 µL were taken  from the suspension and transferred into a new culture flask. After adding  20-25 mL DMEM the flask was incubated at 37°C and checked on the 3rd  / 4th day under the microscope.</P><P>On the medium flask were the following notes:
      Date, Name, Cell Line, Medium, Ratio of splitting, passage number</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.63">15.63 PCR Amplification of Hag-D2-Cup</A></LEGEND><DIV class="aim"><P>Aim: Amplification of insert for Gibson assembly into piGEM-005</P></DIV><DIV class="exp-content"><P>The synthesized fragments by Integrated DNA technologies arrived as a dried 200 ng DNA pellet. In order to use it as a PCR template the pellet was  centrifuged and resuspended in 20 µL Millipore water to get an end concentration  of 10 ng/µL as a Stock solution.</P><P>0,5 µL were used as a PCR  template.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix (µl)</TH><TH scope="col">Mastermix</TH><TH scope="col">D2-Cup</TH><TH scope="col">D2-Strep</TH></TR><TR><TH scope="row">Template</TH><TD>-</TD><TD>2</TD><TD>2</TD></TR><TR><TH scope="row">Primer  Flo96 D23-fw</TH><TD>3</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer  Flo97 D23-rv</TH><TD>3</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Q5-Master  mix</TH><TD>75</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Water</TH><TD>63</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Mastermix</TH><TD>-</TD><TD>48</TD><TD>48</TD></TR><TR><TH scope="row">Total Volume</TH><TD>144</TD><TD>50</TD><TD>20</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>30 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>31x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The expected bands could be seen.
	</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.64a">15.64 Gibson assembly with linearized piGEM-005 &amp; PCR products from 15.63</A></LEGEND><DIV class="aim"><P>Aim: Insertion of Hag-D2-Cup and Hag-D2-Strep into piGEM-005</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Hag-D2-Cup(µL)</TH><TH scope="col">Hag-D2-Strep (µL)</TH></TR><TR><TH scope="row">Gibson-mix</TH><TD>15</TD><TD>15</TD></TR><TR><TH scope="row">Insert I: (Hag-D2-Cup 431 ng/µL)</TH><TD>2,5</TD><TD>-</TD></TR><TR><TH scope="row">Insert II:
      (Hag-D2-Strep 423 ng/µL)</TH><TD>-</TD><TD>2,5</TD>2
  </TR><TR><TH scope="row">piGEM-005 <I>Spe</I>I (50 ng/  µL)</TH><TD>2,5</TD><TD>2,5</TD></TR><TR><TH scope="row"><STRONG>Total</STRONG></TH><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><P>Incubation was done  at 50°C for  1 hour with following transformation into <EM>E. coli</EM>  XL1-Blue plated out on LB-Amp plates and incubated overnight at 37°C.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="12.08.2014">12.08.2014</A></H2><FIELDSET class="exp15"><LEGEND><A name="exp15.64b">15.64 Screening clones from 15.63</A></LEGEND><DIV class="aim"><P>Aim: Checking insertion of Hag-D2-Cup and Hag-D2-Strep into piGEM-005</P></DIV><DIV class="exp-content"><P>In order to check the successful integration of D2-Cup and D2-Strep, clones were picked for cPCR and transferred onto a LB-Amp plate. The primers Flo96 (D23-fw) and Flo97 (D23-v) were used.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix (µl)</TH><TH scope="col">Mastermix</TH><TH scope="col">D2-Cup</TH><TH scope="col">D2-Strep</TH></TR><TR><TH scope="row">Template</TH><TD>-</TD><TD>Colony</TD><TD>Colony</TD></TR><TR><TH scope="row">Primer  Flo96 D23-fw</TH><TD>4,5</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer  Flo97 D23-rv</TH><TD>4,5</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">dNTP</TH><TD>4,5</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">5x Phusion Buffer</TH><TD>36</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>4,5</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Water</TH><TD>126</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Mastermix</TH><TD>-</TD><TD>20</TD><TD>20</TD></TR><TR><TH scope="row">Total Volume</TH><TD>180</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>10 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>40 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>33x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The gel shows that clone 2 from the Gibson Hag-D2-Cup clones and clones  3 &amp; 4 from the Gibson Hag-D2-Strep clones contain the integrated plasmid  constructs. All 3 clones were picked for a miniprep and used for the inoculation of  10 mL LB-Amp. Incubation was performed over night at 37°C. </P><P><STRONG>The new  constructs were labelled with piGEM-026 (pMAD-Hag-D2-Cup) and piGEM-027  (pMAD-Hag-D2-Strep).</STRONG></P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.65">15.65 Overnight culture for WT3610</A></LEGEND><DIV class="aim"><P>Aim:Preparation for mainculture the next day</P></DIV><DIV class="exp-content"><P>5 mL of LB  were inoculated with <I>Bacillus subtilis</I> WT3610 from an LB plate and incubated at 37°C  over night.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="13.08.2014">13.08.2014</A></H2><FIELDSET class="exp15"><LEGEND><A name="exp15.66">15.66 Transformation of competent <I>Bacillus subtilis</I> WT3610</A></LEGEND><DIV class="aim"><P>Aim: transformation of plasmid into <I>Bacillus subtilis</I> WT3610</P></DIV><DIV class="exp-content"><P>100 µL  preculture were added to 10 mL of MNGE-Medium and  incubated to an OD<SUB>600</SUB> of 1,1-1,3 at 37°C which could take 4-5h.</P><P>After reaching an OD<SUB>600</SUB> of 1,1-1,3 2 x 400 µL of the culture were transformed  with 1,5 µg plasmid (piGEM-026 &amp; -027). After 1 hour of incubation at 37°C 100 µL  expression mix were added and incubated for 1 hour as well.</P><P>In the end the 500 µL attempt was plated out on MLS-X-Gal plates and  incubated at 30 °C overnight until colonies could be seen. </P></DIV></FIELDSET><FIELDSET class="exp"><LEGEND><A name="exp_20">Sequencing </A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Premix</TH><TH scope="col">Label-Nr.</TH><TH scope="col">Construct</TH><TH scope="col">Primer</TH></TR><TR><TH scope="row">1</TH><TD>AGB0023458</TD><TD>piGEM-023</TD><TD>TW260</TD></TR><TR><TH scope="row">2</TH><TD>AGB0023459</TD><TD>piGEM-024</TD><TD>TW260</TD></TR><TR><TH scope="row">3</TH><TD>AGB0023460</TD><TD>piGEM-025</TD><TD>TW260</TD></TR><TR><TH scope="row">4</TH><TD>AGB0023461</TD><TD>piGEM-026 D2-Cup cl.2</TD><TD>Flo96-D2 3 for</TD></TR><TR><TH scope="row">5</TH><TD>AGB0023462</TD><TD>piGEM-026 D2-Cup cl.2</TD><TD>Flo56-Hag rv</TD></TR><TR><TH scope="row">6</TH><TD>AGB0023463</TD><TD>piGEM-027 D2-Strep cl.3</TD><TD>Flo96-D2 3 for</TD></TR><TR><TH scope="row">7</TH><TD>AGB0023464</TD><TD>piGEM-027 D2-Strep cl.3</TD><TD>Flo56-Hag rv</TD></TR><TR><TH scope="row">8</TH><TD>AGB0023465</TD><TD>piGEM-027 D2-Strep cl.4</TD><TD>Flo96-D2 3 for</TD></TR><TR><TH scope="row">9</TH><TD>AGB0023466</TD><TD>piGEM-027 D2-Strep cl.4</TD><TD>Flo56-Hag rv</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.72">13.72 Restriction of Nose plasmids with <I>Eco</I>RV</A></LEGEND><DIV class="aim"><P>Aim: check if <I>amyE</I> and <I>cat</I> are still in plasmid, since <I>B. subtilis</I> could not successfully be transformed with the plasmids yet</P></DIV><DIV class="exp-content"><P>All Nose-plasmids (iGEM002-004 and 007-015) were digested with <I>Eco</I>RV to check if <I>amyE</I> and <I>cat</I> are still present in these plasmids, since <I>B. subtilis</I> could not be transformed successfully.</P><P>Per sample 1 µl 2x CutSmart Buffer, 7.9 µl H2O,  0.1 µl <EM>EcoR</EM>V and 2 µl of plasmids  were added. The reaction was incubated over night at RT.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="14.08.2014">14.08.2014</A></H2><FIELDSET class="exp13"><LEGEND><A name="exp13.72">13.72 Restriction of Nose plasmids with <I>Eco</I>RV - continuation</A></LEGEND><DIV class="aim"><P>Aim: Analysis of the restriction on an agarose gel</P></DIV><DIV class="exp-content"><P>The samples were mixed with 2 µl of 5x Loading dye and 8 µl of it was loaded on a 1% agarose gel.</P><P>All plasmids but piGEM004 and piGEM011 show the expected bands of ca 2000 and 4000 bp.</P><P>The plasmids were used to transform  <I>Bacillus  subtilis</I>, which was incubated at 30°C.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="18.08.2014">18.08.2014</A></H2><FIELDSET class="exp15"><LEGEND><A name="exp15.67">15.67 Transformation of competent WT3610</A></LEGEND><DIV class="aim"><P>Aim: <I>Bacillus subtilis</I> pMad Trafo mit D2-Strep and D2-Cup</P></DIV><DIV class="exp-content"><P>Next step: inoculate overnight culture from the X-gal plate with 3 ml LB-MLS.</P><P>As the cultures were  recognizable blue (thin or old plate) wildtype cultures were inoculated for a new  transformation. Therefore the plasmids piGEM-026 and -027 were isolated from frozen pellets. </P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="19.08.2014">19.08.2014</A></H2><FIELDSET class="exp15"><LEGEND><A name="exp15.68">15.68 Transformation of competent WT3610</A></LEGEND><DIV class="exp"><P><I>Bacillus subtilis</I> pMad trafo mit D2-Strep and D2-Cup</P></DIV><DIV class="exp-content"><P>For the experimental procedure see the Materials section. Anyway, two transformations were running currently that will be referred to as 'old' and 'new' transformation from now on.</P><P>For the old transformation the first temperature  shift was performed in LB-MLS medium. The culture was plated on LB-MLS-X-Gal plates  and incubated at 42°C over night.</P><P>The new transformation was started with a culture  grown in MNGE medium, inoculated with a wildtype overnight culture. The culture  was grown until an optical density of 1,2 that is normally reached after 4-5 h.  Strangely the culture reached this OD after almost 7 h today. Plasmids (1.5µg) piGEM026 and -027 were  transformed and the protocol was followed as described below.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="20.08.2014">20.08.2014</A></H2><FIELDSET class="exp15"><LEGEND><A name="exp15.69">15.69 Transformation of competent WT3610</A></LEGEND><DIV class="exp"><P><I>Bacillus subtilis</I> pMad Trafo mit D2-Strep and D2-Cup</P></DIV><DIV class="exp-content"><P>New trafo showed no colonies so far. It was further incubated.</P><P>For the old trafo 2nd temperature shift of positive colonies (only light blue).</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="21.08.2014">21.08.2014</A></H2><FIELDSET class="exp15"><LEGEND><A name="exp15.70a">15.70 Transformation of competent WT3610</A></LEGEND><DIV class="exp"><P><I>Bacillus subtilis</I> pMad Trafo mit D2-Strep and D2-Cup</P></DIV><DIV class="exp-content"><P>Old trafo: white colonies were re-streaked on x-Gal plates and on MLS plates to check for MLS sensitivity. Incubation at 42°C.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="22.08.2014">22.08.2014</A></H2><FIELDSET class="exp15"><LEGEND><A name="exp15.70ab">15.70 Transformation of competent WT3610</A></LEGEND><DIV class="aim"><P>Aim: integration of  pMAD-Insert into Bacillus chromosome via flanks</P></DIV><DIV class="exp-content"><P>The blue/ white screening showed positive transformed blue clones from  the new transformation.  Three piGEM026 and -027 clones of different morphology  per plate were picked and used for inoculation of LB-MLS (4 mL LB, 4 µL  lincomycin, 4 µL erythromycin). Incubation was carried out over night at 30°C  with the cultures.</P></DIV></FIELDSET><FIELDSET class="exp22"><LEGEND><A name="exp22.0">22. The Killswitch</A></LEGEND></FIELDSET><FIELDSET class="exp22"><LEGEND><A name="exp22.1">22.1 Amplification of flanks</A></LEGEND><DIV class="aim"><P>Aim: The flanks have to be amplified for a fusion PCR with the Killswitch modules in order to integrate them into the <I>amyE</I> and the <I>lacA</I>/<I>ganA</I> locus of <I>Bacillus subtilis</I> after cloning into pMAD</P></DIV><DIV class="exp-content"><P>Primers arrived and have been dissolved in millipore water, dilution 1:10. Q5 Master Mix was used for the following PCRs to create the flanks of the killswitch parts I-III:</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Fragment</TH><TH scope="col">Fw primer (iGEM-0xx)</TH><TH scope="col">Rv primer (iGEM-0xx)</TH></TR><TR><TH scope="row">AmyE flankI</TH><TD>iGEM-034</TD><TD>iGEM-035</TD></TR><TR><TH scope="row">AmyE flankII</TH><TD>iGEM-036</TD><TD>iGEM-037</TD></TR><TR><TH scope="row">lacA flankI</TH><TD>iGEM-040</TD><TD>iGEM-041</TD></TR><TR><TH scope="row">lacA flankII</TH><TD>iGEM-042</TD><TD>iGEM-043</TD></TR><TR><TH scope="row">KSIII AmyE-yvyd  FlankII</TH><TD>iGEM-048</TD><TD>-</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix</TH><TH scope="col">Master Mix</TH><TH scope="col"><I>amyE</I> FlankI</TH><TH scope="col"><I>amyE</I> FlankII</TH><TH scope="col"><I>lacA</I> FlankI</TH><TH scope="col"><I>lacA</I> FlankII</TH><TH scope="col"><I>amyE</I>-<I>yvyD</I> FlankII</TH></TR><TR><TH scope="row">Template  (<I>B. subtilis</I>  gDNA)</TH><TD>11</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer  iGEM-34</TH><TD>-</TD><TD>1</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer  iGEM-35</TH><TD>-</TD><TD>1</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer  iGEM-36</TH><TD>-</TD><TD>-</TD><TD>1</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer  iGEM-37</TH><TD>-</TD><TD>-</TD><TD>1</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer  iGEM-40</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>1</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer  iGEM-41</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>1</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer  iGEM-42</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer  iGEM-43</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer  iGEM-48</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Primer  iGEM-37</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Q5-Master  mix</TH><TD>165</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Water</TH><TD>132</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Mastermix</TH><TD>-</TD><TD>28</TD><TD>28</TD><TD>28</TD><TD>28</TD><TD>28</TD></TR><TR><TH scope="row">Total Volume [µl]</TH><TD>308</TD><TD>30</TD><TD>30</TD><TD>30</TD><TD>30</TD><TD>30</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>60 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>30x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>All bands had the expected size (500 /1000 bp)</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="23.08.2014">23.08.2014</A></H2><FIELDSET class="exp15"><LEGEND><A name="exp15.71a">15.71 Transformation of competent WT3610</A></LEGEND><DIV class="aim"><P>Aim: First heat shock - integration of  pMAD-Insert into Bacillus chromosome via flanks</P></DIV><DIV classs="exp-content"><P>The  overnight cultures were used to  inoculate 10 mL LB MLS until  the culture  obtained an OD of 0,1. The cultures were incubated at 30°C for 2h.</P><P>Then the temperature was shifted to 42°C for 6h.</P><P>After the heat shock dilutions from 10-5 to 10-6 of  each culture were plated out on MLS-X-Gal so that plates could be incubated  overnight at 42°C.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="24.08.2014">24.08.2014</A></H2><FIELDSET class="exp22"><LEGEND><A name="exp22.2">22.2 Amplification of Killswitch modules I-III </A></LEGEND><DIV class="aim"><P>Aim: The modules had to be amplified via PCR in order to proceed with a fusion PCR linking the modules with the flanks for integration into the Bacillus Loci.</P></DIV><DIV class="exp-content"><P>The synthesized fragments by Integrated DNA technologies arrived as a dried 200 ng DNA pellet. In order to use it as a PCR template the pellet was  centrifuged down and resuspended in 20 µL millipore water to get an end concentration of 10 ng/µL as a stock solution.</P><P>0,5 µL were used as a PCR  template. The PCR fragments should be ca. 350, 650 &amp; 2000 bp long. </P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix</TH><TH scope="col">KS module I</TH><TH scope="col">KS module II</TH><TH scope="col">KS module III</TH></TR><TR><TH scope="row">Template (gBlock DNA)</TH><TD>0,5</TD><TD>0,5</TD><TD>0,5</TD></TR><TR><TH scope="row">Primer  iGEM-38</TH><TD>1</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer  iGEM-39</TH><TD>1</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer  iGEM-44</TH><TD>-</TD><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer  iGEM-45</TH><TD>-</TD><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer  iGEM-49</TH><TD>-</TD><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Primer  iGEM-50</TH><TD>-</TD><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Q5-Mastermix</TH><TD>25</TD><TD>25</TD><TD>25</TD></TR><TR><TH scope="row">Water</TH><TD>22,5</TD><TD>22,5</TD><TD>22,5</TD></TR><TR><TH scope="row">Total Volume (µl)</TH><TD>50</TD><TD>50</TD><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">TimeKSI &amp; III</TH><TH scope="col">Time
            KSII</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>3 min</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>20 sec</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>20 sec</TD><TD>20 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>60 sec</TD><TD>120 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>31x</TD><TD>31x</TD></TR><TR><TH scope="row">6</TH>q
            <TD>72</TD><TD>4min</TD><TD>4min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The modules I and III could be amplified without problems, but KSII showed no distinct band and many unspecific fragments.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="15.7b">15.71 Transformation of competent WT3610</A></LEGEND><DIV class="aim"><P>Aim: Second heat shock -flip out of the pMAD backbone</P></DIV><DIV class="exp-content"><P>One blue colony per diluted piGEM026  and -027 clone was used to inoculate 4 mL LB. The cultures were incubated at 30°C  for 6 hours and afterwards for 3 hours at 42°C.</P><P>Dilutions from 10<SUP>-5</SUP> to 10<SUP>-6</SUP> were plated out on  X-Gal plates without MLS selection. The  positive clones should not contain the resistance inside the backbone as well  as the galactosidase. The plates were incubated at 42°C overnight. Some clones  (piGEM027.1 and one of 27.2) did not grow and will be inoculated again. They  will be inoculated in LB-MLS for the first temperature shift tomorrow.</P></DIV></FIELDSET><FIELDSET class="exp22"><LEGEND><A name="exp22.3a">22.3 Transformation of competent DH5α with pMAD</A></LEGEND><DIV class="aim"><P>Aim: Gain <I>E.coli</I> on plate for mini preps of pMAD</P></DIV><DIV class="exp-content"><P>Standard heat shock transformation protocol was  followed, incubation over night at 37°C. pMAD from the plasmid box was used for  transformation.</P></DIV></FIELDSET><FIELDSET class="exp22"><LEGEND><A name="exp22.4">22.4 Restriction of pMAD with <I>Nco</I>I and <I>Bam</I>HI over night</A></LEGEND><DIV class="aim"><P>Aim: Linearizing Vector for Gibson Assembly with KS modules</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Volume (µl)</TH></TR><TR><TH scope="row">pMAD (108 ng/µL)</TH><TD>9</TD></TR><TR><TH scope="row">CutSmart 10x</TH><TD>2</TD></TR><TR><TH scope="row"><I>Nco</I>I</TH><TD>0,2</TD></TR><TR><TH scope="row"><I>Bam</I>HI</TH><TD>0,2</TD></TR><TR><TH scope="row">Water</TH><TD>8,6</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD></TR></TBODY></TABLE><P>Incubation over night at room temperature.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="25.08.2014">25.08.2014</A></H2><FIELDSET class="exp15"><LEGEND><A name="exp15.72">15.72 Transformation of competent WT3610</A></LEGEND><DIV class="aim"><P>Aim: checking the correct flip out of the pMAD backbone and MLS sensitivity</P></DIV><DIV class="exp-content"><P>From the dilution plates was one white clone picked and transferred on a  X-Gal plate as well as on a MLS plate so that clones were proven for the  right integration of the insert although flipping out the pMAD backbone.</P><P>The plates were incubated at 42°C overnight.</P></DIV></FIELDSET><FIELDSET class="exp22"><LEGEND><A name="exp22.5a">22.5a Fusion PCR </A></LEGEND><DIV class="aim"><P>Aim: Assembly of Killswitch modules I &amp; II with <I>amyE</I> and <I>lacA</I> flanks</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Segment</TH><TH scope="col"><STRONG>Concentraction  (ng/µL)</STRONG></TH></TR><TR><TH scope="row">KS module I</TH><TD>401</TD></TR><TR><TH scope="row">KS module II</TH><TD>414</TD></TR><TR><TH scope="row">KS module III</TH><TD>369</TD></TR><TR><TH scope="row"><I>amyE</I> flank I</TH><TD>446</TD></TR><TR><TH scope="row"><I>amyE</I> flank II</TH><TD>403</TD></TR><TR><TH scope="row"><I>lacA</I> flank I</TH><TD>368</TD></TR><TR><TH scope="row"><I>lacA</I> flank II</TH><TD>369</TD></TR><TR><TH scope="row"><I>amyE</I><I>yvyD</I> flank II</TH><TD>372</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix</TH><TH scope="col"><I>amyE</I> KS I</TH><TH scope="col"><I>lacA</I> KS II</TH></TR><TR><TH scope="row">KS  module I (40,1 ng/µL)</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">KS  module II (41,4 ng/µL)</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row"><I>amyE</I>  flank I ( 44,6 ng/µL)</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row"><I>amyE</I>  flank II (40,3 ng/µL)</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row"><I>lacA</I>  flank I (36,8 ng/µL)</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row"><I>lacA</I>  flank II (39,6 ng/µL)</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Primer  iGEM-34</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer  iGEM-37</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer  iGEM-40</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Primer  iGEM-43</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Phusion  Buffer 5x</TH><TD>10</TD><TD>10</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">dNTPs</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>33</TD><TD>33</TD></TR><TR><TH scope="row">Total Volume [µl]</TH><TD>50</TD><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time
            amyE KSI</TH><TH scope="col">Time
            amyE KSII</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>3 min</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>20 sec</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>20 sec</TD><TD>20 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>120 sec</TD><TD>190 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>31x</TD><TD>31x</TD></TR><TR><TH scope="row">6</TH>q
            <TD>72</TD><TD>4min</TD><TD>4min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD><TD>infinite</TD></TR></TBODY></TABLE><P>No PCR amplificates could be seen on the gel.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.53">18.53 PCR amplification of StrepDARPidin</A></LEGEND><DIV class="aim"><P>Aim: amplification of the StrepDARPidin for cloning into the expression vector pET24d for protein purification and overproduction in <I>E. coli</I> BL21(DE3)</P></DIV><DIV class="exp-content"><P>The synthesized fragments by Integrated DNA technologies arrived as a dried  200n g DNA pellet. In order to use it as a PCR template the pellet was  centrifuged and resuspended in 20 µL millipore water to get an end concentration  of 10 ng/µL as a stock solution.</P><P>0,5 µL were used as a PCR  template. The PCR fragment should be ca. 900 bp long. </P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix</TH><TH scope="col">PCR attempt</TH></TR><TR><TH scope="row">Template</TH><TD>0,5</TD></TR><TR><TH scope="row">Primer iGEM-33</TH><TD>1</TD></TR><TR><TH scope="row">Primer iGEM-34</TH><TD>1</TD></TR><TR><TH scope="row">Q5-Mastermix</TH><TD>25</TD></TR><TR><TH scope="row">Water</TH><TD>22,5</TD></TR><TR><TH scope="row">Mastermix</TH><TD>-</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>60 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>31x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>Both samples had a  concentration of 380 ng/µL.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.54">18.54 Digestion of amplified of StrepDARPidin for cloning into pET24d</A></LEGEND><DIV class="aim"><P>Aim: digest with <I>Nco</I>I and <I>Xho</I>I of the StrepDARPidin PCR product for cloning into the expression vector pet24d </P></DIV><DIV class="exp-content"><P>The PCR product was cut with <I>Nco</I>I and <I>Xho</I>I to get the restriction sites for cloning into pET24d <I>Nco</I>I/<I>Xho</I>I cut.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">StrepDARPidin (µl)</TH></TR><TR><TH scope="row">PCR product (380 ng/µL)</TH><TD>6</TD></TR><TR><TH scope="row">CutSmart 10x</TH><TD>2</TD></TR><TR><TH scope="row"><I>Nco</I>I</TH><TD>0,25</TD></TR><TR><TH scope="row"><I>Xho</I>I</TH><TD>0,25</TD></TR><TR><TH scope="row">Water</TH><TD>11,5</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD></TR></TBODY></TABLE><P>Incubation for 1h at  37°C with heat shock at 85°C for 10 min.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.55">18.55 Ligation of digested StrepDARPidin for cloning into pET24d</A></LEGEND><DIV class="aim"><P>Aim: ligation of StrepDARPidin PCR product with the expression vector pET24d </P></DIV><DIV class="exp-content"><P>The PCR product was ligated into digested pET24d <I>Nco</I>I/<I>Xho</I>I. Two samples were made for a transformation of <I>E. coli</I> XL1-Blue and BL21.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Ligation I (µl)</TH><TH scope="col">Ligation II (µl)</TH></TR><TR><TH scope="row"><I>Nco</I>I and <I>Xho</I>I StrepDARPidin (64 ng/µL)</TH><TD>2</TD><TD>2</TD></TR><TR><TH scope="row">pET24d <I>Nco</I>I and <I>Xho</I>I (15  ng/µL)</TH><TD>5</TD><TD>5</TD></TR><TR><TH scope="row">Ligation Buffer 10x</TH><TD>2</TD><TD>2</TD></TR><TR><TH scope="row">Ligase</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>10</TD><TD>10</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><P>Incubation for 1,5h at room temperature with heat shock at 85°C for 10 min. The 20 µL from ligation I were used to transform <I>E. coli</I> XLI-Blue, 10 µL from Ligation II were used to transform <EM>E. coli</EM> BL21(DE3) and the rest of ligation II into  Xl1-Blue as well and selected on  LB-Canamycin plates.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="26.08.2014">26.08.2014</A></H2><FIELDSET class="exp15"><LEGEND><A name="exp15.73">15.73 Transformation of competent WT3610</A></LEGEND><DIV class="aim"><P>Aim: checking the correct flip out of the pMAD backbone and MLS sensitivity </P></DIV><DIV class="exp-content"><P>All clones on the master plate grew on x-gal  plates but were MLS sensitive and could be seen as positive. In order to check  the correct insertion of D2-Cup an D2-Strep into the <I>Bacillus subtilis</I> genome  the clones were cooked in 100 µL 1x PBS at 95°C for 10 min and used for colony  PCR. Because of the repetition of the temperature shifts with clones 27.1.1,  27.1.2, 27.1.3 yesterday and today and 27.2.3, 6 clones from 26.1 -26.3, 27.2.1  &amp; 27.2.2 were picked for the cPCR (30 reactions).</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix (µl)</TH><TH scope="col">Master-Mix for 30 attempts (µL)</TH><TH scope="col">Mix for PCR attempt (µL)</TH></TR><TR><TH scope="row">Colony PBS suspension</TH><TD>-</TD><TD>2</TD></TR><TR><TH scope="row">Primer Flo54 Hag-fw</TH><TD>15</TD><TD>-</TD></TR><TR><TH scope="row">Primer  Flo56 Hag-fw</TH><TD>15</TD><TD>-</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>15</TD><TD>-</TD></TR><TR><TH scope="row">Phusion Buffer 5x</TH><TD>120</TD><TD>-</TD></TR><TR><TH scope="row">dNTP-mix</TH><TD>15</TD><TD>-</TD></TR><TR><TH scope="row">Water</TH><TD>410</TD><TD>-</TD></TR><TR><TH scope="row">DMSO</TH><TD>10</TD><TD>-</TD></TR><TR><TH scope="row">Mastermix</TH><TD>-</TD><TD>18</TD></TR><TR><TH scope="row">Total Volume</TH><TD>600</TD><TD>20</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1 min 30 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>30x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>There were no bands visible on the gel.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.56a">18.56 cPCR with plates from ligation</A></LEGEND><DIV class="aim"><P>Aim: checking the success of the ligation </P></DIV><DIV class="exp-content"><P>The grown clones from the ligation the day before were inoculated for plasmid isolation and then checked via cPCR with primers iGEM-032 and -033.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content (µl)</TH><TH scope="col">Master mix  for  20 clones (µL)</TH><TH scope="col">Clones from Ligation I (µL)</TH><TH scope="col">Clones from Ligation II (µL)</TH><TH scope="col">Clones from Ligation III (µL)</TH></TR><TR><TH scope="row">Template (colony)</TH><TD>-</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Primer iGEM-32</TH><TD>10</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-32</TH><TD>10</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Phusion Buffer 5x</TH><TD>80</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Phusion DNA-polymerase </TH><TD>10</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">dNTP-mix</TH><TD>10</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Water</TH><TD>280</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Mastermix</TH><TD>-</TD><TD>19</TD><TD>19</TD><TD>19</TD></TR><TR><TH scope="row">Total Volume</TH><TD>400</TD><TD>20</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>10 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>20 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>20 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>60 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>33x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The gel shows that the clones I  2-6, II 1-6 and III 2, 3, 4 and 6 are positive due to the band at approx. 900 bp.  The other bands might be plasmid which was used as template.</P><P>The clones I6, II3, III3 were used to transform <I> E. coli</I> BL21(DE3) and selected on LB-Can plates. III4 and III6  were already on a plate. The clones were inoculated for plasmid isolation and sent for sequencing.</P></DIV></FIELDSET><FIELDSET class="exp22"><LEGEND><A name="exp22.5b">22.5b Fusion PCR Attempt 3</A></LEGEND><DIV class="aim"><P>Aim: Fusion of <I>amyE</I> and <I>lacA</I> flanks to the killswitch fragments</P></DIV><DIV class="exp-content"><P>The Fusion PCR was repeated since  it did not work the first two times. The fragments of the killswitch I and II  were diluted 10fold. A gradient was carried out from 52°C to 62°C.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix (µl)</TH><TH scope="col">amyE KSI</TH><TH scope="col">lacA KS II</TH></TR><TR><TH scope="row">KS module I (40,1 ng/µL)</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">KS module II (41,4 ng/µL)</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row"><I>amyE</I> flank I ( 44,6 ng/µL)</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row"><I>amyE</I> flank II (40,3 ng/µL)</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row"><I>lacA</I> flank I (36,8 ng/µL)</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row"><I>lacA</I> flank II (39,6 ng/µL)</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Primer iGEM-34</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-37</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-40</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Primer iGEM-43</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Phusion Buffer 5x</TH><TD>4</TD><TD>4</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>0,5</TD><TD>0,5</TD></TR><TR><TH scope="row">dNTPs</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>9,5</TD><TD>9,5</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>95</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>95</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>52-62</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>200 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>32x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>infinite</TD></TR></TBODY></TABLE><P>Again, no fragments could be noticed on the gel.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.73">13.73 Amplification of new Cu- and Ag-Promotor sequences</A></LEGEND><DIV class="aim"><P>Aim: Amplification of Cu- and Ag promoter sequences from the chromosomal DNA of <I>Bacillus subtilis</I>.</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Master mix  (µL)</TH><TH scope="col">2x Attempt Cu (µL)</TH><TH scope="col">2x Attempt Ag (µL)</TH></TR><TR><TH scope="row">Template (gDNA <I>B. subtilis</I>)</TH><TD>4</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-55</TH><TD>-</TD><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Primer iGEM-56</TH><TD>-</TD><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Primer iGEM-57</TH><TD>-</TD><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-58</TH><TD>-</TD><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Phusion Buffer 5x</TH><TD>40</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>4</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">dNTPs</TH><TD>4</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Water</TH><TD>144</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Mastermix</TH><TD>-</TD><TD>48</TD><TD>48</TD></TR><TR><TH scope="row">Total Volume (µl)</TH><TD>196</TD><TD>50</TD><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>10 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>20 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>20 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>20 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>30x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4 min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The gel shows positive bands at approx. 100 bp and 250 bp, which was like expected.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.74">13.74 <I>Nco</I>I/<I>Sac</I>I digest of piGEM-002</A></LEGEND><DIV class="aim"><P>Aim: linearizing the plasmid backbone for Gibson Assembly and Ligation with Cu/Ag promotor sequences and IPTG/ Constitutive (/tetO) promotor sequences.</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Attempt I (µl)</TH></TR><TR><TH scope="row">piGEM-002 (476 ng/µL)</TH><TD>6,5</TD></TR><TR><TH scope="row"><I>Nco</I>I</TH><TD>0,25</TD></TR><TR><TH scope="row"><I>Sac</I>I</TH><TD>0,25</TD></TR><TR><TH scope="row">CutSmart Buffer 10x</TH><TD>2</TD></TR><TR><TH scope="row">Water</TH><TD>11</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD></TR></TBODY></TABLE><P>Incubation at 37°C for  60 min and heat shocked at 85°C for 10 min.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.75">13.75 Ligation of <I>Nco</I>I/<I>Sac</I> cut piGEM-002 with annealed oligos containing promoter sequences</A></LEGEND><DIV class="aim"><P>Aim: insertion of a IPTG inducible/ constitutive and constitutive+tetO into piGEM-002</P></DIV><DIV class="exp-content"><P>In order to test our nose system with new promoter sequences oligos were designed and annealed by heating up water and letting it cool down with the oligo pairs iGEM-59/60, -61/62, - 63/64 so that the individual fragments were able to anneal with overlaps to the <I>Nco</I>I/<I>Sac</I>I sites.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Ligation I (µl)</TH><TH scope="col">Ligation II (µl)</TH><TH scope="col">Ligation III (µl)</TH></TR><TR><TH scope="row">piGEM-002 (209 ng/µL)</TH><TD>5</TD><TD>5</TD><TD>5</TD></TR><TR><TH scope="row">iGEM-59/60</TH><TD>1</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">iGEM-61/62</TH><TD>-</TD><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">iGEM-63/64</TH><TD>-</TD><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">T4 Ligase</TH><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Ligation Buffer 10x</TH><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR><TH scope="row">Water</TH><TD>11</TD><TD>11</TD><TD>11</TD></TR><TR><TH scope="row">Total Volume (µl]</TH><TD>20</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><P>The reactions were  incubated at room temperature for an hour and transformed into <EM>E. coli</EM> XL1-Blue, plated out on LB-Amp.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.76">13.76 Gibson assembly of <I>Nco</I>I/<I>Sac</I>I piGEM-002 &amp; Cu/ Ag promotor</A></LEGEND><DIV class="aim"><P>Aim: assembling of Nco/Sac cut piGEM-002 with Cu/ ag promotor sequences</P></DIV><DIV class="exp-content"><P>In order to test new Cu/ Ag promotor sequences the PCR fragments from 13.73 were used for a Gibson assembly.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Gibson Cu I (µL)</TH><TH scope="col">Gibson Ag II (µL)</TH><TH scope="col">Control  (µL)</TH></TR><TR><TH scope="row">piGEM-002 (209 ng/µL)</TH><TD>2,5</TD><TD>2,5</TD><TD>5</TD></TR><TR><TH scope="row">Cu promotor</TH><TD>2,5</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Ag promotor</TH><TD>-</TD><TD>2,5</TD><TD>-</TD></TR><TR><TH scope="row">Gibson mix</TH><TD>15</TD><TD>15</TD><TD>15</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><P>The attempts were  incubated at 50°C for an hour and transformed into <EM>E. coli</EM> XL1-Blue, plated out on LB-Amp.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="27.08.2014">27.08.2014</A></H2><FIELDSET class="exp22"><LEGEND><A name="exp22.3b">22.3b Repeated Amplification of Killswitch modules I and II</A></LEGEND><DIV class="aim"><P>Aim: Increase the amount of modules for further purification, since Fusion PCR did not work 'quick and dirty'</P></DIV><DIV class="exp-content"><P>This time the Phusion polymerase was used. The template was the already amplified module.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix</TH><TH scope="col">KS module I</TH><TH scope="col">KS module II</TH></TR><TR><TH scope="row">Template (already amplified modules)</TH><TD>0,5</TD><TD>0,5</TD></TR><TR><TH scope="row">Primer iGEM-38</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-39</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-44</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Primer iGEM-45</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">5x HF buffer</TH><TD>10</TD><TD>10</TD></TR><TR><TH scope="row">dNTPs</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>36</TD><TD>36</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>0,5</TD><TD>0,5</TD></TR><TR><TH scope="row">Total Volume (µl)</TH><TD>50</TD><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col"> KS I</TH><TH scope="col">KS II</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>5 min</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>120 sec</TD><TD>200 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD> 5 min</TD><TD> 5 min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The gel can be seen under the next topic</P></DIV></FIELDSET><FIELDSET class="exp.."><LEGEND><A name="exp22.3c">22.3c Repeated amplification of killswitch flanks</A></LEGEND><DIV class="aim"><P>Aim: The flanks have to be amplified for a fusion PCR with the Killswitch modules in order to integrate them into the amyE and the lacA locus of Bacillus subtilis after cloning into pMAD, yield higher amount of fragments for purification and FusionPCR.</P></DIV><DIV class="exp-content"><P>The Phusion polymerase was used. </P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix</TH><TH scope="col">Master Mix</TH><TH scope="col"><I>amyE</I> FlankI</TH><TH scope="col"><I>amyE</I> FlankII</TH><TH scope="col"><I>lacA</I> FlankI</TH><TH scope="col"><I>lacA</I> FlankII</TH><TH scope="col"><I>amyE</I>-<I>yvyD</I> FlankII</TH></TR><TR><TH scope="row">Template </TH><TD> </TD><TD>0,5</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-34</TH><TD>-</TD><TD>1</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-35</TH><TD>-</TD><TD>1</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-36</TH><TD>-</TD><TD>-</TD><TD>1</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-37</TH><TD>-</TD><TD>-</TD><TD>1</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-40</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>1</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-41</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>1</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-42</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-43</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-48</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Primer iGEM-37</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">dNTPs</TH><TD>11</TD><TD>1</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">DMSO</TH><TD>3,3</TD><TD>0,3</TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TH scope="row">MgCl2</TH><TD>2,2</TD><TD>0,2</TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TH scope="row">5x HF-buffer</TH><TD>110</TD><TD>10</TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>5,5</TD><TD>0,5</TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TH scope="row">Water</TH><TD>390,5</TD><TD>35,5</TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TH scope="row">Mastermix</TH><TD>-</TD><TD>47,5</TD><TD>47,5</TD><TD>47,5</TD><TD>47,5</TD><TD>47,5</TD></TR><TR><TH scope="row">Total Volume (µl)</TH><TD>522,5</TD><TD>50</TD><TD>50</TD><TD>50</TD><TD>50</TD><TD>50</TD></TR></TBODY></TABLE><P>With exception of KSI and StrepDARP the PCR was negative for all other samples. Even for the positive reactions the bands were very thin.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.1">18.57 repeated PCR amplification of Hag-<I>Kpn</I>I StrepDARPidin</A></LEGEND><DIV class="aim"><P>Aim: amplification of the StrepDARPidin for cloning into the expression vector pET24d for protein purification and overproduction in <I>E. coli</I> BL21(DE3)</P></DIV><DIV class="exp-content"><P>The synthesized fragments by Integrated DNA technologies arrived as a dried  200 ng DNA pellet. In order to use it as a PCR template the pellet was  centrifuged and resuspended in 20 µL millipore water to get an end concentration  of 10 ng/µL as a stock solution.</P><P>0,5 µL were used as a PCR  template. The PCR fragment should be approx. 900 bp long. </P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content (µl)</TH><TH scope="col">2x Attempt I (µL)</TH><TH scope="col">2x Attempt II(µL)</TH></TR><TR><TH scope="row">Template 1: StrepDARPidin (38 ng/ µL)</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Template 2: Hag-<I>Kpn</I>I (10 ng/ µL)</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-51</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Primer iGEM-52</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Primer iGEM-53</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-54</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Phusion Buffer 5x</TH><TD>10</TD><TD>10</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">dNTPs</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>35</TD><TD>35</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>60 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>31x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The PCR was clearly positive for StrepDARP, Hag-<I>Kpn</I>I could be seen as a very slight band at 900 bp, but more unspecific bands could be noticed.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.56b">18.56 Expression-Test with transformed <I>E. coli</I> BL21(DE3)</A></LEGEND><DIV class="aim"><P>Aim: Checking the overexpression of pET24d-StrepDARPidin</P></DIV><DIV class="exp-content"><P>In  order to check the expression of the proteins 5 clones (I6, II3, III3, III4,  III6) containing piGEM-028 (pet24d-StrepDARPidin) were used to inoculate 20 mL  of LB-Can. 5 cultures were incubated at 37°C until an OD of 0,7.</P><P>A ca. 1  mL preinduction sample (PI) was taken (0,7 (favoured OD) : 0,7 (actual OD) = 1  mL). After that the cultures were induced with 20 µL IPTG for 2h.</P><P>An  induction sample (I) was taken (320 µL with an OD of 2,2; 350 µL with an OD of  2).</P><P>PI and  I were centrifuged at 14000 rpm for 1 min, the supernatant was discarded and  the pellet resuspended in 60 µL water and 40 µL SDS-Buffer.</P><P>The  four samples were analysed on an SDS-PAGE gel with 10 µL volume per sample.</P><P>The gel  did not show any signs of over expression so that lactose was chosen as an alternative inducer to test  the expression. New test expression cultures (I6, II3, III3, III4, III6)  were made with 100 mL LB-Can and 5 mL lactose  (6,25 g/ 25 mL) per culture. Incubation was performed over night at 30°C.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="28.08.2014">28.08.2014</A></H2><FIELDSET class="exp"><LEGEND><A name="exp_21">Sequencing of constructs from 28.08.2014</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Premix</TH><TH scope="col">Label-Nr.</TH><TH scope="col">Construct</TH><TH scope="col">Primer</TH></TR><TR><TH scope="row">1</TH><TD>AGB0023467</TD><TD>piGEM-028 I6(pET24d-StrepDARPidin)</TD><TD>T7 turn</TD></TR><TR><TH scope="row">2</TH><TD>AGB0023468</TD><TD>piGEM-028 III3  (pET24d-StrepDARPidin)</TD><TD>T7 turn</TD></TR><TR><TH scope="row">3</TH><TD>AGB0023469</TD><TD>piGEM-026  (pMAD-D2-Cup)</TD><TD>Flo D2_3 fw</TD></TR><TR><TH scope="row">4</TH><TD>AGB0023470</TD><TD>piGEM-027.1  (pMAD-D2-Strep)</TD><TD>Flo D2_3 fw</TD></TR><TR><TH scope="row">5</TH><TD>AGB0023471</TD><TD>piGEM-027.2  (pMAD-D2-Strep)</TD><TD>Flo D2_3 fw</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.561">18.56 Expression-Test with transformed <I>E. coli</I> BL21(DE3)</A></LEGEND><DIV class="aim"><P>Aim: Checking the overexpression of pET24d-StrepDARPidin</P></DIV><DIV class="exp-content"><P>No sign of an overexpression could be noticed.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.57">18.57 His-bead purification of pellet from <I>E. coli</I> BL21(DE3)</A></LEGEND><DIV class="aim"><P>Aim: Checking the overexpression of pET24d-StrepDARPidin</P></DIV><DIV class="exp-content"><P>The 100 mL culture from I6 &amp;  III3 were centrifuged at 4000 rpm for 10 min in 2x50 mL falcons. The pellets  were washed in 10 mL buffer A and cracked with the microfluidizer. After that  the suspension was centrifuged at 20000 rpm for 20 min/ 4°C. The supernatant  was transferred into a 50 mL falcon.</P><P>200 µL Ni-NTA beads were added to  the suspension, inverted and incubated on ice for half an hour.</P><P>After that the suspension was  centrifuged or 5 min at 4000 rpm so that the pellet could be resuspended in 500  µL Buffer A after discarding the supernatant. The 500 µL were transferred into a 1,5 mL tube and centrifuged at 4000 rpm for washing. After discarding the  supernatant the pellet was washed again with 500 µL Buffer A. After a second  centrifugation at 4000 rpm the supernatant was discarded, the pellet  resuspended in 200 µL Buffer A and centrifuged at 13000 rpm to destroy the  Ni-bead – protein interaction.</P><P>In the end the supernatant was  transferred into a new 1,5 mL tube. The Ni-beads were mixed with 100 µL 20%  ethanol.</P><P>40 µL of the supernatant from  clone I6 and III3 were added with 10 µL SDS-PAGE Buffer and analysed on the  SDS-Gel (visible at 18.56).</P></DIV></FIELDSET><FIELDSET class="exp22"><LEGEND><A name="exp22.3d">22.3d Repeated amplification of killswitch flanks</A></LEGEND><DIV class="aim"><P>Aim: The flanks have to be amplified for a fusion PCR with the Killswitch modules in order to integrate them into the <I>amyE</I> and the <I>lacA</I> locus of Bacillus subtilis after cloning into pMAD, yield higher amount of fragments for purification and FusionPCR.</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix [µl]</TH><TH scope="col">Master Mix 6x</TH><TH scope="col"><I>amyE</I> FlankI</TH><TH scope="col"><I>amyE</I> FlankII</TH><TH scope="col"><I>lacA</I> FlankI</TH><TH scope="col"><I>lacA</I> FlankII</TH><TH scope="col"><I>amyE</I>-<I>yvyD</I> FlankII</TH></TR><TR><TH scope="row">Template (<I>B. subtilis</I>  gDNA)</TH><TD>6</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-34</TH><TD>-</TD><TD>1</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-35</TH><TD>-</TD><TD>1</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-36</TH><TD>-</TD><TD>-</TD><TD>1</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-37</TH><TD>-</TD><TD>-</TD><TD>1</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-40</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>1</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-41</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>1</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-42</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-43</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-48</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Primer iGEM-37</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Q5-Master mix</TH><TD>150</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Water</TH><TD>132</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Mastermix</TH><TD>-</TD><TD>48</TD><TD>48</TD><TD>48</TD><TD>48</TD><TD>48</TD></TR><TR><TH scope="row">Total Volume</TH><TD>288</TD><TD>50</TD><TD>50</TD><TD>50</TD><TD>50</TD><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>25 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>25 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>60 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>31x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4 min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The gel is visible under 22.3e</P></DIV></FIELDSET><FIELDSET class="exp22"><LEGEND><A name="exp22.3e">22.3e Repeated Amplification of Killswitch modules I and II</A></LEGEND><DIV class="aim"><P>Aim: Increase the amount of modules for further purification, since Fusion PCR did not work 'quick and dirty'. Q5 was used and gDNA since the previously amplified modules did not work as template yesterday.</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix [µl]</TH><TH scope="col">KS module I</TH><TH scope="col">KS module II</TH><TH scope="col">KS module III</TH></TR><TR><TH scope="row">Template (gBlock DNA)</TH><TD>0,5</TD><TD>0,5</TD><TD>0,5</TD></TR><TR><TH scope="row">Primer iGEM-38</TH><TD>1</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-39</TH><TD>1</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-44</TH><TD>-</TD><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-45</TH><TD>-</TD><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-49</TH><TD>-</TD><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Primer iGEM-50</TH><TD>-</TD><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Q5-Master mix</TH><TD>25</TD><TD>25</TD><TD>25</TD></TR><TR><TH scope="row">Water</TH><TD>2,5</TD><TD>2,5</TD><TD>2,5</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD><TD>50</TD><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col"> KS I &amp; III</TH><TH scope="col">KS II</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>5 min</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>25 sec</TD><TD>25 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>25 sec</TD><TD>25 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>60 sec</TD><TD>120 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>31x</TD><TD>31x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD> 4 min</TD><TD> 4 min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The expected bands could be seen for the flank I of <I>amyE</I> and flank I and II of <I>lacA</I>. These bands were cut out and purified via a Gel Extraction kit, but the concentration was too low to work further.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.75">13.75 <I>Nco</I>I/<I>Sac</I>I digest of piGEM-002</A></LEGEND><DIV class="aim"><P>Aim: linearizing the plasmid backbone for Gibson Assembly and Ligation with Cu/Ag promotor sequences and IPTG/ Constitutive (/tetO) promotor sequences.</P></DIV><DIV class="exp-content"><P>The already digested piGEM002 of 26.08. was nearly empty. To have some digested plasmid as backup a new digestion was carried out. piGEM002 was isolated and used for the restriction.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Attempt (µl)</TH></TR><TR><TH scope="row">piGEM-002 (476 ng/µL)</TH><TD>10</TD></TR><TR><TH scope="row"><I>Nco</I>I</TH><TD>0,25</TD></TR><TR><TH scope="row"><I>Sac</I>I</TH><TD>0,25</TD></TR><TR><TH scope="row">CutSmart Buffer 10x</TH><TD>2</TD></TR><TR><TH scope="row">Water</TH><TD>7,5</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD></TR></TBODY></TABLE><P>An agarose gel of the previous restriction was done with unrestricted piGEM002 as reference. A band at ca between 6000 and 8000 bp could be seen, which indicates a successful restriction (expected size: ca 6600 bp).</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.76">13.76 Repeated Gibson assembly of <I>Nco</I>I/<I>Sac</I>I piGEM-002 &amp; Cu/ Ag promotor</A></LEGEND><DIV class="aim"><P>Aim: assembling of <I>Nco</I>I/<I>Sac</I>I cut piGEM-002 with Cu/Ag promoter sequences</P></DIV><DIV class="exp-content"><P>In order to test new Cu/ Ag promoter sequences the PCR fragments from 13.73 were used for a Gibson assembly.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Gibson Cu I (µL)</TH><TH scope="col">Gibson Ag II (µL)</TH><TH scope="col">Control  (µL)</TH></TR><TR><TH scope="row">piGEM-002 (ca.  40ng/µL)</TH><TD>2,5</TD><TD>2,5</TD><TD>2,5</TD></TR><TR><TH scope="row">Cu promotor (ca 4 ng/µl)</TH><TD>2,5</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Ag promotor (ca 4 ng/µl)</TH><TD>-</TD><TD>2,5</TD><TD>-</TD></TR><TR><TH scope="row">H2O</TH><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Gibson Mix</TH><TD>15</TD><TD>15</TD><TD>15</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><P>The attempts were  incubated at 50°C for an hour and used to transform <EM>E. coli</EM> XL1-Blue, plated out on LB-Amp.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.77">13.77 Repeated ligation of <I>Nco</I>I/<I>Sac</I>I cut piGEM-002 with annealed oligos containing promoter sequences</A></LEGEND><DIV class="aim"><P>Aim: insertion of an IPTG inducible/ constitutive and constitutive+tetO into piGEM-002</P></DIV><DIV class="exp-content"><P>The ligation was prepared to contain ca 100 ng of the restricted piGEM002 with an appropriate amount of insert (much fewer).</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Ligation I (µL)</TH><TH scope="col">Ligation II (µL)</TH><TH scope="col">Ligation III (µL)</TH></TR><TR><TH scope="row">piGEM-002 (ca 40  ng/µL)</TH><TD>2,5</TD><TD>2,5</TD><TD>2,5</TD></TR><TR><TH scope="row">iGEM-59/60 (ca 4 ng/µl)</TH><TD>0,8</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">iGEM-61/62 (ca 4 ng/µl)</TH><TD>-</TD><TD>0,8</TD><TD>-</TD></TR><TR><TH scope="row">iGEM-63/64 (ca 4 ng/µl)</TH><TD>-</TD><TD>-</TD><TD>0,8</TD></TR><TR><TH scope="row">Polynucleotide kinase</TH><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">T4 Ligase</TH><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Ligation Buffer 10x</TH><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR><TH scope="row">Water</TH><TD>12,7</TD><TD>12,7</TD><TD>12,7</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><P>The reactions were  incubated at room temperature for an hour and transformed into <EM>E. coli</EM> XL1-Blue, plated out on LB-Amp.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="29.08.2014">29.08.2014</A></H2><FIELDSET class="exp"><LEGEND><A name="exp_22">Sequencing results of constructs from 28.08.2014</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Premix</TH><TH scope="col">Label-Nr.</TH><TH scope="col">Construct</TH><TH scope="col">Results</TH></TR><TR><TH scope="row">1</TH><TD>AGB0023467</TD><TD>piGEM-028  I6(pET24d-StrepDARPidin)</TD><TD>No results</TD></TR><TR><TH scope="row">2</TH><TD>AGB0023468</TD><TD>piGEM-028 III3  (pET24d-StrepDARPidin)</TD><TD>No alignment</TD></TR><TR><TH scope="row">3</TH><TD>AGB0023469</TD><TD>piGEM-026  (pMAD-D2-Cup)</TD><TD>Good</TD></TR><TR><TH scope="row">4</TH><TD>AGB0023470</TD><TD>piGEM-027.1  (pMAD-D2-Strep)</TD><TD>Point mutation</TD></TR><TR><TH scope="row">5</TH><TD>AGB0023471</TD><TD>piGEM-027.2  (pMAD-D2-Strep)</TD><TD>Point mutation</TD></TR></TBODY></TABLE><P>The sequencing showed that pET24d did not contain any known insert.</P><P>In  order to check the correct synthesis of the templates purified PCR products  were sent for sequencing again. In the meanwhile the constructs were cloned again.</P></DIV></FIELDSET><FIELDSET class="exp"><LEGEND><A name="exp_23">Sequencing of constructs from 29.08.2014</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Premix</TH><TH scope="col">Label-Nr.</TH><TH scope="col">Construct</TH><TH scope="col">Primer</TH></TR><TR><TH scope="row">1</TH><TD>AGB0023512</TD><TD>PCR Fragment  StrepDARPidin I</TD><TD>piGEM-032 fw</TD></TR><TR><TH scope="row">2</TH><TD>AGB0023513</TD><TD>PCR Fragment  StrepDARPidin II</TD><TD>piGEM-032 fw</TD></TR><TR><TH scope="row">3</TH><TD>AGB0023514</TD><TD>PCR Fragment D2-StrepII </TD><TD>Flo96 D2_3 fw</TD></TR><TR><TH scope="row">4</TH><TD>AGB0023515</TD><TD>PCR Fragment  StrepDARPidin I</TD><TD>piGEM-033 rv</TD></TR><TR><TH scope="row">5</TH><TD>AGB0023516</TD><TD>PCR Fragment  StrepDARPidin II</TD><TD>piGEM-033 rv</TD></TR><TR><TH scope="row">6</TH><TD>AGB0023517</TD><TD>PCR Fragment D2-StrepII</TD><TD>Flo97 D2_3 rv</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.58">18.58 New PCR amplification of StrepDARPidin and D2-Cup</A></LEGEND><DIV class="aim"><P>Aim: amplification of the StrepDARPidin for cloning into the expression vector pET24d for protein purification and overproduction in <I>E. coli</I> BL21(DE3) &amp; amplification of D2-StrepII for Gibson Assembly into piGEM-005</P></DIV><DIV class="exp-content"><P>The synthesized fragments were supposed to be sequenced in order to  check the right synthesis.  10 ng/µL as a  Stock solution and a purified PCR Product were used as template for each  reaction.</P><P>0,5 µL were used as a PCR  template. The PCR fragment should be ca. 900 bp long for StrepDARPidin and ca.  450 bp for D2-StrepII.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix (µl)</TH><TH scope="col">PCR StrepDARPidin I</TH><TH scope="col">PCR StrepDARPidin II</TH><TH scope="col">PCR StrepDARPidin III</TH><TH scope="col">PCR D2-Strep I</TH><TH scope="col">PCR D2-Strep II</TH></TR><TR><TH scope="row">Pur. StrepDARP. (1:10- 10 ng/µL)</TH><TD>1</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">StrepDARP. Stock (20 ng/µL)</TH><TD>-</TD><TD>0,25</TD><TD>0,25</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">D2-StrepII Stock 20 ng/µL)</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>0,25</TD><TD>0,25</TD></TR><TR><TH scope="row">Primer iGEM-32</TH><TD>1</TD><TD>1</TD><TD>1</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-33</TH><TD>1</TD><TD>1</TD><TD>1</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Flo96 D2_3 fw</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Flo97 D2_3 rv</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Q5-Master mix</TH><TD>25</TD><TD>25</TD><TD>25</TD><TD>25</TD><TD>25</TD></TR><TR><TH scope="row">Water</TH><TD>22</TD><TD>22</TD><TD>22</TD><TD>22</TD><TD>22</TD></TR><TR><TH scope="row">Total Volume (µl)</TH><TD>50</TD><TD>50</TD><TD>50</TD><TD>50</TD><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>60sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>31x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The PCR samples were purified with the Omega Gel Extraction Kit according to the enzymatic reaction purification protocol.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.59">18.59 Digestion of amplified of StrepDARPidin for cloning into pET16b</A></LEGEND><DIV class="aim"><P>Aim: digest with <I>Nco</I>I and <I>Xho</I>I of the StrepDARPidin PCR product for cloning into the expression vector pET16b &amp; digest of pET16b</P></DIV><DIV class="exp-content"><P>The PCR products from 15.58 were cut with <I>Nco</I>I and <I>Xho</I>I to get the restriction sites for cloning into the <I>Nco</I>I/<I>Xho</I>I  digested pET16b. This vector was used because of its Amp resistance for fast transformation.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">PCR StrepDARPidin I (µL)</TH><TH scope="col">PCR StrepDARPidin II(µL)</TH><TH scope="col">PCR StrepDARPidin III (µL)</TH><TH scope="col">Pet16b(µL) </TH><TH scope="col">Mastermix</TH></TR><TR><TH scope="row">StrepDARP. I ( 45 ng/µL)</TH><TD>10</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">StrepDARP. II (70 ng/µL)</TH><TD>-</TD><TD>15</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">StrepDARP. III (45 ng/µL)</TH><TD>-</TD><TD>-</TD><TD>10</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">pET16b ( 58 ng/µL)</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>17</TD><TD>-</TD></TR><TR><TH scope="row">CutSmart 10x</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>8</TD></TR><TR><TH scope="row"><I>Nco</I>I</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>0,5</TD></TR><TR><TH scope="row"><I>Xho</I>I</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>0,5</TD></TR><TR><TH scope="row">Water</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>17</TD></TR><TR><TH scope="row">Mastermix</TH><TD>10</TD><TD>5</TD><TD>10</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>20</TD><TD>20</TD><TD>20</TD><TD>28</TD></TR></TBODY></TABLE><P>Incubation for 1h at  37°C with heat shock at 85°C for 10 min.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.60">18.60 Ligation of digested StrepDARPidin for cloning into pET16b</A></LEGEND><DIV class="aim"><P>Aim: ligation of StrepDARPidin PCR product with the expression vector pET16b</P></DIV><DIV class="exp-content"><P>The PCR product was ligated into the <I>Nco</I>I/<I>Xho</I>I digested pET16b. Two attempts were made for a transformation into E. Coli  XL1-Blue and BL21. </P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Ligation I (µL)</TH><TH scope="col">Ligation II (µL)</TH><TH scope="col">Ligation III (µL)</TH><TH scope="col">Control</TH></TR><TR><TH scope="row"><I>Nco</I>I/<I>Xho</I>I StrepDARPidin  (ca. 40  ng/ µL) I/II/III</TH><TD>1</TD><TD>1</TD><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">pET16b <I>Nco</I>I/<I>Xho</I>I (45 ng/µL)</TH><TD>5</TD><TD>5</TD><TD>5</TD><TD>5</TD></TR><TR><TH scope="row">Ligation Buffer 10x</TH><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR><TH scope="row">Ligase</TH><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>11</TD><TD>11</TD><TD>11</TD><TD>11</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>20</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><P>Incubation for 1,5 hours at  rom temperature with heat shock at 85°C for 10 min. The 20 µL from ligation I  were used to transform <I>E. coli</I> XLI-Blue, which was plated out on LB-Amp. Incubation was  done overnight at 37°C.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.61">18.61 Gibson assembly of piGEM-005 &amp; D2-StrepII</A></LEGEND><DIV class="aim"><P>Aim: Insertion of Hag-D2-Strep into piGEM-005</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Component</TH><TH scope="col">Hag-D2-Strep I(µL)</TH><TH scope="col">Hag-D2-Strep II (µL)</TH></TR><TR><TH scope="row">Gibson-mix</TH><TD>15</TD><TD>15</TD></TR><TR><TH scope="row">Insert I
      (Hag-D2-Strep I 60  ng/µL)</TH><TD>2,5</TD><TD>-</TD></TR><TR><TH scope="row">Insert II
      (Hag-D2-Strep II 70  ng/µL)</TH><TD>-</TD><TD>2,5</TD></TR><TR><TH scope="row">piGEM-005 SpeI (50 ng/ µL)</TH><TD>2,5</TD><TD>2,5</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><P>Incubation was done  at 50°C for  1 hour with following transformation into <EM>E. coli</EM>  XL1-Blue, plated out on LB-Amp plates and incubated overnight at 37°C.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.78">13.78 colony PCR of colonies of Gibson and ligation (13.76 and 13.77)</A></LEGEND><DIV class="aim"><P>Aim: check for correct ligation/Gibson assembly of the oligos with the piGEM002 plasmid</P></DIV><DIV class="exp-content"><P>Several clones were picked and transferred onto another plate, the same colony was used to inoculate a PCR reaction to check the insertion of the oligos.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix [µl]</TH><TH scope="col">Master mix for 45 reactions  (µl)</TH><TH scope="col">Const. promoter</TH><TH scope="col">Cont. + tetO</TH><TH scope="col">Mod. Lac promoter</TH><TH scope="col">Ag promoter new</TH><TH scope="col">Cu promoter new</TH></TR><TR><TH scope="row">Template</TH><TD>-</TD><TD>Small part</TD><TD>Small part</TD><TD>Small part</TD><TD>Small part</TD><TD>Small part</TD></TR><TR><TH scope="row">Primer iGEM006</TH><TD>45</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM061</TH><TD>-</TD><TD>1</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM063</TH><TD>-</TD><TD>-</TD><TD>1</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM059</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>1</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM057</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM055</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">5x HF buffer</TH><TD>180</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Phusion DNA polymerase 1:10</TH><TD>45</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Water</TH><TD>585</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Master Mix</TH><TD>-</TD><TD>19</TD><TD>19</TD><TD>19</TD><TD>19</TD><TD>19</TD></TR><TR><TH scope="row">Total Volume</TH><TD>855</TD><TD>20</TD><TD>20</TD><TD>20</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>95</TD><TD>10 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>60 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The positive clones (band at ca 1000 bp) were  transferred into LB-amp medium and were incubated at 37°C over night for a  miniprep. A new colony PCR was carried out with six colonies of the XLIBlue piGEM002+Ag-promoter.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp22.4">22.4 Repeated amplification of KS module II</A></LEGEND><DIV class="aim"><P>Aim: amplify module II for further purification and fusion PCR with flanks of <I>lacA</I></P></DIV><DIV class="exp-content"><P>The PCR for the KSII was repeated  with the Phusion DNA-polymerase. A gradient was carried out from 52°C to 62°C.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Master mix</TH><TH scope="col">KSII (µl)</TH></TR><TR><TH scope="row">Template: (gBlock KSII)</TH><TD>4,5</TD><TD>0,5</TD></TR><TR><TH scope="row">Primer iGEM044</TH><TD>9</TD><TD>1</TD></TR><TR><TH scope="row">Primer iGEM045</TH><TD>9</TD><TD>1</TD></TR><TR><TH scope="row">HF Buffer 5x</TH><TD>90</TD><TD>10</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>9</TD><TD>1</TD></TR><TR><TH scope="row">dNTPs</TH><TD>9</TD><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>319,5</TD><TD>35</TD></TR><TR><TH scope="row">Total Volume (µl)</TH><TD>450</TD><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH><TH scope="col">KS II</TH></TR><TR><TH scope="row">1</TH><TD>95</TD><TD>5 min</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>95</TD><TD>30 sec</TD><TD>20 sec</TD></TR><TR><TH scope="row">3</TH><TD>52-62</TD><TD>30 sec</TD><TD>20 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1 min 45 sec</TD><TD>120 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>30x</TD><TD>33x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4min</TD><TD>4 min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The gel showed many unspecific bands of fewer basepairs than expected. The expected band of ca 2000 bp was very weak but got stronger the higher the annealing temperature was. </P></DIV></FIELDSET><FIELDSET class="exp22"><LEGEND><A name="exp22.4">22.4 repeated fusion-PCR of the Killswitch modules with the flanks</A></LEGEND><DIV class="aim"><P>Aim: fuse the modules with the flanks to integrate them into the <I>B. subtilis</I> genome</P></DIV><DIV class="exp-content"><P>For this fusion-PCR the flanks of  22.08.14 and the amplified and purified modules I and III from 28.08.14 were  used. The module II from 24.08.14 was separated on an agarose gel, cut out and  purified as well.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">KS I (µl)</TH><TH scope="col">KS II (µl)</TH></TR><TR><TH scope="row">Template KSI (ca 5 ng/µl)</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Template KSII (7 ng/µl)</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row"><I>amyE</I> flank I (1.5 ng/µl)</TH><TD>3</TD><TD>-</TD></TR><TR><TH scope="row"><I>amyE</I>  flank II (ca. 6.5 ng/µl)</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row"><I>lacA</I> flank I (ca. 5.5 ng/µl)</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row"><I>lacA</I> flank II (7 ng/µl)</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Primer piGEM034</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer piGEM037</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer piGEM040</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Primer pIGEM043</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">HF Buffer 5x</TH><TD>10</TD><TD>10</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">dNTPs</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Water</TH><TD>31</TD><TD>31</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time (min/sec) KSI</TH><TH scope="col">Time (min/sec) KSII</TH></TR><TR><TH scope="row">1</TH><TD>95</TD><TD>5 min</TD><TD>5 min</TD></TR><TR><TH scope="row">2</TH><TD>95</TD><TD>30 sec</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>2 min</TD><TD>3 min 10 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>34x</TD><TD>34x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The fusion PCR was negative, expected bands would be around 1881 bp for KSI and around 3000 for KSII.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="30.08.2014">30.08.2014</A></H2><FIELDSET class="exp18"><LEGEND><A name="exp18.62">18.62 Gibson assembly of piGEM-005 &amp; D2-StrepII/ Ligation StrepDARPidin-pET16b</A></LEGEND><DIV class="exp-content"><P>The transformation plates were taken out of the incubator and put in the fridge at 4°C.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="31.08.2014">31.08.2014</A></H2><FIELDSET class="exp18"><LEGEND><A name="exp18.63">18.63 cPCR with clones from Ligation pET16b &amp; StrepDARPidin</A></LEGEND><DIV class="aim"><P>Aim: checking the success of the ligation </P></DIV><DIV class="exp-content"><P>The grown clones from the ligation from 18.60 were checked via cPCR with  primers iGEM-032 and -033. 40 clones from Ligation plates I-III were picked for  cPCR and transferred on a plate before.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Master mix  for  40 clones (µL)</TH><TH scope="col">Clones from Ligation I (µL)</TH><TH scope="col">Clones from Ligation II(µL)</TH><TH scope="col">Clones from Ligation III(µL)</TH></TR><TR><TH scope="row">Template (colony)</TH><TD>-</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Primer iGEM-32</TH><TD>20</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-33</TH><TD>20</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Phusion Buffer 5x</TH><TD>160</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>20</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">dNTPs</TH><TD>20</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Water</TH><TD>560</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Master mix</TH><TD>-</TD><TD>19</TD><TD>19</TD><TD>19</TD></TR><TR><TH scope="row">Total Volume</TH><TD>800</TD><TD>20</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>10 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>20 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>20 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>60  sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>33x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4 min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The positive clones show a band at ca. 900 bp  and were used for inoculation minipreps so that a test digest could confirm the  insertion of StrepDARPidin.</P></DIV></FIELDSET><FIELDSET class="exp18"><LEGEND><A name="exp18.64">18.64 cPCR with XLI-Blue clones containing piGEM005 Hag-D2-Strep or Hag-D2-Cup</A></LEGEND><DIV class="aim"><P>Aim: Checking insertion of Hag-D2-Cup and Hag-D2-Strep into piGEM-005</P></DIV><DIV class="exp-content"><P>In order to check the successful integration of D2-Cup and D2-Strep clones were picked for cPCR and transferred on a LB-Amp master plate. As primers Flo96 (D23-fw) and Flo97 (D23-v) were used. 12 clones were screened.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix</TH><TH scope="col">Master-Mix (14x) </TH><TH scope="col">D2-Strep</TH></TR><TR><TH scope="row">Template</TH><TD>-</TD><TD>Colony</TD></TR><TR><TH scope="row">Primer Flo96 D23-fw</TH><TD>7</TD><TD>-</TD></TR><TR><TH scope="row">Primer Flo96 D23-rv</TH><TD>7</TD><TD>-</TD></TR><TR><TH scope="row">dNTPs</TH><TD>7</TD><TD>-</TD></TR><TR><TH scope="row">5x Phusion Buffer</TH><TD>56</TD><TD>-</TD></TR><TR><TH scope="row">Phusion DNA-polymerase</TH><TD>7</TD><TD>-</TD></TR><TR><TH scope="row">Water</TH><TD>196</TD><TD>-</TD></TR><TR><TH scope="row">Mastermix</TH><TD>-</TD><TD>20</TD></TR><TR><TH scope="row">Total Volume (µl)</TH><TD>280</TD><TD>20</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>10 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>40 sec</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>33x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4 min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><P>The clones I1, I3, I6, II2 and II4 were picked for plasmid isolation and sent for sequencing the next day.</P></DIV></FIELDSET></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Marburg:Project:Notebook:August">http://2014.igem.org/Team:Marburg:Project:Notebook:August</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Marburg:Project:Notebook:August" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Marburg:Project:Notebook:August" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Marburg:Project:Notebook:August&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Marburg:Project:Notebook:August&amp;oldid=329119" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>