<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Marburg_Project_Notebook_April"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Marburg:Project:Notebook:April">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Marburg:Project:Notebook:April&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Marburg:Project:Notebook:April&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Marburg:Project:Notebook:April&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Marburg:Project:Notebook:April" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Marburg:Project:Notebook:April</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="navbox"><NAV class="menu center first" aria-haspopup="true" aria-controls="p1"><UL id="p1" class="first"><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg" class="a">Home</A></LI><LI class="current"><A href="https://2014.igem.org/Team:Marburg:Project" class="b">Project</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Safety" class="c">Safety</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Team" class="d">Team</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Policy_Practices" class="e">Policy &amp; Practices</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Attributions" class="f">Attributions</A></LI></UL></NAV><NAV class="menu center second" aria-haspopup="true" aria-controls="p1"><UL id="p1" class="second"><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:Silver" class="d">SilverSURF</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:Tumor" class="c">CancerSURF</A></LI><LI aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:NRPS" class="e">RiboSURF</A></LI><LI class="current"><A href="https://2014.igem.org/Team:Marburg:Project:Notebook" class="f">Notebook</A></LI><LI class="last" aria-haspopup="false"><A href="https://2014.igem.org/Team:Marburg:Project:Parts" class="g">Parts</A></LI></UL></NAV></DIV><DIV class="mr_content"><H1>

Notebook: April

</H1><UL id="submenu"><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook">March</A></LI><LI class="current"><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:April">April</A></LI><LI><A href="#02.04.2014">02.04.2014</A></LI><LI><A href="#03.04.2014">03.04.2014</A></LI><LI><A href="#04.04.2014">04.04.2014</A></LI><LI><A href="#07.04.2014">07.04.2014</A></LI><LI><A href="#08.04.2014">08.04.2014</A></LI><LI><A href="#09.04.2014">09.04.2014</A></LI><LI><A href="#10.04.2014">10.04.2014</A></LI><LI><A href="#11.04.2014">11.04.2014</A></LI><LI><A href="#12.04.2014">12.04.2014</A></LI><LI><A href="#14.04.2014">14.04.2014</A></LI><LI><A href="#16.04.2014">16.04.2014</A></LI><LI><A href="#17.04.2014">17.04.2014</A></LI><LI><A href="#22.04.2014">22.04.2014</A></LI><LI><A href="#23.04.2014">23.04.2014</A></LI><LI><A href="#24.04.2014">24.04.2014</A></LI><LI><A href="#25.04.2014">25.04.2014</A></LI><LI><A href="#26.04.2014">26.04.2014</A></LI><LI><A href="#28.04.2014">28.04.2014</A></LI><LI><A href="#29.04.2014">29.04.2014</A></LI><LI><A href="#30.04.2014">30.04.2014</A></LI></UL><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:May">May</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:June">June</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:July">July</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:August">August</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:September">September</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:October">October</A></LI><LI><A href="https://2014.igem.org/Team:Marburg:Project:Notebook:Methods">Methods</A></LI><DIV class="notebooky-entry"><H2 class="title"><A name="02.04.2014">02.04.2014</A></H2><FIELDSET class="exp9"><LEGEND><A name="exp9.1">9.1 PciI: Restriction Digest</A></LEGEND><DIV class="exp-content"><H3>Digestion scheme:</H3><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">[µL]</TH><TH scope="col">PSG1164 - uncut</TH><TH scope="col">PSG1164 - PciL (10 kU/mL)</TH><TH scope="col">PSG1164 - EcoRI-HF</TH></TR><TR><TH scope="row">DNA</TH><TD>261,3 ng/µL
          ca. 1 µg → 3,5 µL</TD><TD>261,3 ng/µL
    ca. 1 µg → 3,5 µL</TD><TD>261,3 ng/µL
    ca. 1 µg → 3,5 µL</TD></TR><TR><TH scope="row">Enzyme 1</TH><TD>-</TD><TD>PciI
          1,0 µL</TD><TD>-</TD></TR><TR><TH scope="row">Enzyme 2</TH><TD>-</TD><TD>-</TD><TD>EcoRI-HF
          0,5 µL</TD></TR><TR><TH scope="row">Buffer</TH><TD>-</TD><TD>10x 3.1 Buffer
          1,5 µL</TD><TD>10x Cutsmart
          1,5 µL</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O </TH><TD>11,5 µL</TD><TD>9,0 µL</TD><TD>9,5 µL</TD></TR><TR><TH scope="row">Total Volume</TH><TD>15 µL</TD><TD>15 µL</TD><TD>15 µL</TD></TR><TR><TH scope="row">Loading Dye (6x)</TH><TD>2,5 µL</TD><TD>2,5 µL</TD><TD>2,5 µL</TD></TR></TBODY></TABLE><P>Incubate samples for 60 min at 37 °C, then induce a heat shock for 2 min at 45 °C. 5 µL marker and 10 µL of the samples are loaded onto a 1% agarose gel.</P></DIV><DIV class="exp-results"><H3>Results:</H3></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="03.04.2014">03.04.2014</A></H2><FIELDSET class="exp10"><LEGEND><A name="exp10.1">10.1 pMA12: Restriction Digest</A></LEGEND><DIV class="aim"><P>Aim: linearizing the backbone for cloning of our Nose plasmid + destroying the lac promoter </P></DIV><DIV class="exp-content"><H3>Digestion scheme:</H3><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">[µL]</TH><TH scope="col">pMA12 - uncut</TH><TH scope="col">pMA12 - HindIII + EcoRI-HF (20 kU/mL)</TH></TR><TR><TH scope="row">DNA</TH><TD>479,3 ng/µL
          ca. 1 µg →2,1 µL</TD><TD>479,3 ng/µL
          ca. 5 µg → 10,5 µL</TD></TR><TR><TH scope="row">Enzyme 1</TH><TD>-</TD><TD>HindIII
          0,5 µL</TD></TR><TR><TH scope="row">Enzyme 2</TH><TD>-</TD><TD>EcoRI
          0,5 µL</TD></TR><TR><TH scope="row">Buffer</TH><TD>-</TD><TD>10x -2.0 Buffer
          1,5 µL</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O</TH><TD>12,9 µL</TD><TD>2 µL</TD></TR><TR><TH scope="row">Total Volume</TH><TD>15 µL</TD><TD>15 µL</TD></TR><TR><TH scope="row">Loading Dye (6x)</TH><TD>2,5 µL</TD><TD>2,5 µL</TD></TR></TBODY></TABLE></DIV><DIV class="exp-results"><H3>Results:</H3><P>The restriction was successful.</P></DIV></FIELDSET><FIELDSET class="exp10"><LEGEND><A name="exp10.3">10.3 Gel Extraction (QIAquick Gel Extraction Kit)</A></LEGEND><DIV class="exp-results"><P>Gel extraction was performed according to the quiagen kit protocol.</P><P>DNA-concentration: 12 ng/µL</P></DIV></FIELDSET><FIELDSET class="exp10"><LEGEND><A name="exp10.4">10.4 Test Transformation of <I>E.Coli</I> DH5α with isolated Plasmid Digest</A></LEGEND></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="04.04.2014">04.04.2014</A></H2><FIELDSET class="exp13"><LEGEND><A name="exp13.1">13.1 Vector Constructs</A></LEGEND><DIV class="exp-content"><H3>Dilution of primers:</H3><P>Stock: dilution of primers with aqua bidest. to 100 µM
      Working stock: 1:10 (10 µL primer stock, 90 µL water)
    </P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Oligo Name</TH><TH scope="col">µL for 100 µM</TH></TR><TR><TH scope="row">iGEM-001</TH><TD>292 µL</TD></TR><TR><TH scope="row">iGEM-002</TH><TD>236 µL</TD></TR><TR><TH scope="row">iGEM-003</TH><TD>397 µL</TD></TR><TR><TH scope="row">iGEM-004</TH><TD>291 µL</TD></TR><TR><TH scope="row">iGEM-005</TH><TD>364 µL</TD></TR><TR><TH scope="row">iGEM-006</TH><TD>273 µL</TD></TR></TBODY></TABLE></DIV><DIV class="exp-content"><H3>Dilution of templates:</H3><UL class="templates"><LI><I>Bacillus</I> gDNA: 1:50 → 10 µL gDNA + 490 µL water</LI><LI>pKH-Stp: 10 ng/µL</LI><LI>PSG1164: 232 ng/µL → 1:20 → 10 µL plasmid + 190 µL water
        </LI></UL></DIV><DIV class="exp-content"><H3>Reaction Mix</H3><TABLE width="100%" border="1"><TBODY><TR><TH scope="col"> </TH><TH scope="col">1&amp;2 plasmid PSG1146 (856 bp)</TH><TH scope="col">3&amp;4 <I>Bacillus</I> gDNA (543 bp)</TH><TH scope="col">5&amp;6 plasmid KG-Stp (743 bp)</TH></TR><TR><TH scope="row">Fragment</TH><TD>chloramphenicol-resistance</TD><TD><I>amyE</I>-gene</TD><TD><I>gfp</I></TD></TR><TR><TH scope="row">Template</TH><TD>1,5 µL</TD><TD>1,5 µL</TD><TD>1,5 µL</TD></TR><TR><TH scope="row">dNTPs</TH><TD>1 µL</TD><TD>1 µL</TD><TD>1 µL</TD></TR><TR><TH scope="row">Primer fwd</TH><TD>1 µL</TD><TD>1 µL</TD><TD>1 µL</TD></TR><TR><TH scope="row">Primer rev</TH><TD>1 µL</TD><TD>1 µL</TD><TD>1 µL</TD></TR><TR><TH scope="row">Phusion Buffer (5x) </TH><TD>10 µL</TD><TD>10 µL</TD><TD>10 µL</TD></TR><TR><TH scope="row">Phusion</TH><TD>1 µL</TD><TD>1 µL</TD><TD>1 µL</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O</TH><TD>34,5 µL</TD><TD>34,5 µL</TD><TD>34,5 µL</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50 µL</TD><TD>50 µL</TD><TD>50 µL</TD></TR></TBODY></TABLE></DIV><DIV class="results"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>98</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1 min</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>30x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>4 min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>infinite</TD></TR></TBODY></TABLE><UL><LI>4 µL Mix + 1 µL Loading dye</LI><LI>5 µL Gene ruler</LI><LI>5 µL 2-log ladder</LI></UL></DIV><DIV class="results"><H3>Results</H3></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.5">13.5 Purification of PCR Products</A></LEGEND><DIV class="exp-content"><P>Purification according to the &quot;Purification of PCR Products (QIAquick Gel Extraction Kit) short protocol&quot;.</P><P>Samples:</P><UL class="samples"><LI>1' chloramphenicol-resistance</LI><LI>2' <I>amyE</I>-Gene</LI><LI>3' <I>gfp</I></LI><LI>4' (=2+4+6)-Pool, all fragments</LI></UL></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.6">13.6 Yeast Transformation</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">[µL]</TH><TH scope="col">1''-Control</TH><TH scope="col">2''-P+frag</TH><TH scope="col">3''-P*+pool</TH><TH scope="col">4''-P**+pool</TH></TR><TR><TH scope="row">PEG 3350(50% w/v)</TH><TD>260 µL</TD><TD>260 µL</TD><TD>260 µL</TD><TD>260 µL</TD></TR><TR><TH scope="row">LiAc 1 M</TH><TD>36 µL</TD><TD>36 µL</TD><TD>36 µL</TD><TD>36 µL</TD></TR><TR><TH scope="row">Salmonsperm(2 mg/mL)</TH><TD>50 µL</TD><TD>50 µL</TD><TD>50 µL</TD><TD>50 µL</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O</TH><TD>14 µL</TD><TD>10 µL</TD><TD>2 µL</TD><TD>2 µL</TD></TR><TR><TH scope="row">Fragm. (GFP, Cat, AmyE)</TH><TD>-</TD><TD>4 µL</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Pool (all fragments)</TH><TD>-</TD><TD>-</TD><TD>12 µL</TD><TD>12 µL</TD></TR><TR><TH scope="row">Total Volume</TH><TD>360 µL</TD><TD>360 µL</TD><TD>360 µL</TD><TD>360 µL</TD></TR></TBODY></TABLE><P>Salmonsperm for 10 min on 95 °C, then on ice
	    *pMA12 - digestion with HindIII + EcoRI
	    **pMA12 - parallel digestion with HindIII and EcoRI</P><UL class="yeast"><LI>resuspend yeast pellet in all samples</LI><LI>induce heat shock at 42 °C for 35 min</LI><LI>centrifuge for 30 sec - 1 min at 13.000 g</LI><LI>resuspend pellet in 200 µL sterile water</LI><LI>plate on selective medium</LI><LI>incubate at 30 °C for 3 days</LI></UL></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="07.04.2014">07.04.2014</A></H2><FIELDSET class="exp13"><LEGEND><A name="exp13.8">13.8 Miniprep of Yeast Culture</A></LEGEND><DIV class="exp-content"><P>Miniprep of Yeast culture 2'', 3'' and 4''</P><UL class="yeast"><LI>flush cells from plate with 2 mL water</LI><LI>centrifuge for 1 min at 11.000 rpm</LI><LI>pellet cracking: add glass balls</LI><LI>→ miniprep accoding to the miniprep procotol (Omega)</LI></UL></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.9">13.9 Test PCR with Miniprep</A></LEGEND><DIV class="exp-content"><P>Test PCR with Miniprep 2'', 3'' and 4''</P><P>Dilution of elution to 20 ng/µL:</P><UL class="yeast"><LI>2'': 72 ng/µL*X=10*20 ng/µL → 5,3:4,7</LI><LI>3'': 38 ng/µL*X=10*20 ng/µL → 3:7</LI><LI>4'': 66 ng/µL*X=10*20 ng/µL → 2,7:3,7</LI></UL><TABLE width="100%" border="1"><TBODY><TR><TH scope="col"> </TH><TH scope="col">2''' 20 ng</TH><TH scope="col">3''' 20 ng</TH><TH scope="col">4''' 20 ng</TH></TR><TR><TH scope="row">Content</TH><TD>P out of fragments</TD><TD>P out of pool</TD><TD>P (DS) out of pool</TD></TR><TR><TH scope="row">Template</TH><TD>1 µL</TD><TD>1 µL</TD><TD>1 µL</TD></TR><TR><TH scope="row">dNTPs</TH><TD>1 µL</TD><TD>1 µL</TD><TD>1 µL</TD></TR><TR><TH scope="row">Primer001 fwd</TH><TD>1 µL</TD><TD>1 µL</TD><TD>1 µL</TD></TR><TR><TH scope="row">Primer006 rev</TH><TD>1 µL</TD><TD>1 µL</TD><TD>1 µL</TD></TR><TR><TH scope="row">Buffer (10x)</TH><TD>3 µL</TD><TD>3 µL</TD><TD>3 µL</TD></TR><TR><TH scope="row">Taq-Pol</TH><TD>1 µL</TD><TD>1 µL</TD><TD>1 µL</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O</TH><TD>22,5 µL</TD><TD>22,5 µL</TD><TD>22,5 µL</TD></TR><TR><TH scope="row">Total Volume</TH><TD>30 µL</TD><TD>30 µL</TD><TD>30 µL</TD></TR></TBODY></TABLE><P>PCR for 3 h</P></DIV><DIV class="results"><P>The control PCR was not successful.</P></DIV><H3>Results</H3></FIELDSET></DIV><FIELDSET class="exp13"><LEGEND><A name="exp13.11">13.11 Amplification in DH5α</A></LEGEND><DIV class="exp-content"><P>Amplification of plasmid in <I>E.coli</I> DH5α.</P><UL class="amplification"><LI>transformation of 2'', 3'' and 4'' in DH5α</LI><LI>plating on each ampicillin and canamycin plates</LI></UL></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.12">13.12 Miniprep of <I>E.coli</I> pMA12: Preculture</A></LEGEND><DIV class="exp-content"><P>Inoculation of preculture plates: 6 mL LB + 6 µL ampicillin + colony clone</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="08.04.2014">08.04.2014</A></H2><FIELDSET class="exp13"><LEGEND><A name="exp13.13">13.13 New PCR</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">[µL]</TH><TH scope="col">Master Mix (7x)</TH><TH scope="col">2''' (20 ng) x2</TH><TH scope="col">3''' (20 ng) x2</TH><TH scope="col">4''' (20 ng) x2</TH></TR><TR><TH scope="row">Template</TH><TD>Reaction mix</TD><TD>P out of fragments</TD><TD>P out of pool</TD><TD>P (DS) out of pool</TD></TR><TR><TH scope="row">dNTPs</TH><TD>-</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Primer001 fwd</TH><TD>3,5</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Primer006 rev</TH><TD>3,5</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Buffer (10x) Thermo-Pol</TH><TD>17,5</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Taq-Pol</TH><TD>0,7</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O</TH><TD>139,3</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Master Mix</TH><TD>-</TD><TD>24</TD><TD>24</TD><TD>24</TD></TR><TR><TH scope="row">Total Volume</TH><TD>168</TD><TD>25</TD><TD>25</TD><TD>25</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>95</TD><TD>2 min</TD></TR><TR><TH scope="row">2</TH><TD>95</TD><TD>20 sec</TD></TR><TR><TH scope="row">3</TH><TD>58</TD><TD>20 sec</TD></TR><TR><TH scope="row">4</TH><TD>68</TD><TD>2,5 min</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>40x</TD></TR><TR><TH scope="row">6</TH><TD>68</TD><TD>4 min</TD></TR><TR><TH scope="row">7</TH><TD>8</TD><TD>Infinite</TD></TR></TBODY></TABLE><P>The new control PCR was not successful either.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.14">13.14 Picking Clones for Sequencing</A></LEGEND><DIV class="exp-content">
    Inoculation of 4x6 mL LB with a clone (trafo 13.11.14) from plate 2,3 and 4 (12 falcons). Incubation overnight.
    </DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.1">14.1 PCR of PheA-Arc1p-C-8x</A></LEGEND><DIV class="aim"><P>Aim: Amplify PheA-Arc1p-C-8x fragment for Gibson Assembly in pET-28a</P></DIV><DIV class="exp-content"><P>The PCR with TG_PheA-Arc1p-C-8x FP und TG_PheA-Arc1p-C-8x RP should generate the 2346 bp fragment for Gibson Assembly from the synthesized template.</P><P>The used Primer concentration was 1 
    µM, the concentration of the template was 1 ng/µL and we used a hot start to avoid unspecific annealing of primers.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Volume [µL]</TH></TR><TR><TH scope="row">Water</TH><TD>29.5</TD></TR><TR><TH scope="row">5x GC Buffer</TH><TD>10</TD></TR><TR><TH scope="row">DMSO</TH><TD>2.5</TD></TR><TR><TH scope="row">TG_PheA-Arc1p-C-8x FP</TH><TD>2.5</TD></TR><TR><TH scope="row">TG_PheA-Arc1p-C-8x RP</TH><TD>2.5</TD></TR><TR><TH scope="row">synthesized template</TH><TD>1</TD></TR><TR><TH scope="row">dNTPs</TH><TD>1</TD></TR><TR><TH scope="row">Phusion-Polymerase</TH><TD>1</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature [°C]</TH><TH scope="col">Time [min:sec]</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>2:00</TD></TR><TR><TH scope="row">2</TH><TD>add Polymerase</TD><TD> </TD></TR><TR><TH scope="row">3</TH><TD>98</TD><TD>0:10</TD></TR><TR><TH scope="row">4</TH><TD>67.5</TD><TD>0:30</TD></TR><TR><TH scope="row">5</TH><TD>72</TD><TD>1:40</TD></TR><TR><TH scope="row">6</TH><TD>go to 3</TD><TD>32x</TD></TR><TR><TH scope="row">7</TH><TD>72</TD><TD>10:00</TD></TR><TR><TH scope="row">8</TH><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P>The amplified fragment was purified on an agarose gel on which it showed the correct size. It was extracted from the gel for further use in a Gibson Assembly yielding a concentration of 258 ng/µL.</P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.2">14.2 Digestion of pET-28a</A></LEGEND><DIV class="aim"><P>Aim: Linearize pET-28a to prepare it for Gibson Assembly</P></DIV><DIV class="exp-content"><P>The pET-28a vector was cut with XhoI and NdeI in order to create the ends needed for the integration of the prepared PheA-Arc1p-C-8x construct via Gibson Assembly</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Volume [µL]</TH></TR><TR><TH scope="row">Water</TH><TD>34.2</TD></TR><TR><TH scope="row">CutSmart Buffer</TH><TD>5</TD></TR><TR><TH scope="row">pET-28a (128 ng/µL)</TH><TD>7.81</TD></TR><TR><TH scope="row">NdeI</TH><TD>1.5</TD></TR><TR><TH scope="row">XhoI</TH><TD>1.5</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD></TR></TBODY></TABLE><P>The reaction mixture was incubated for 3 h at 37 °C and 350 rpm. The resulting linearized vector was purified using an agarose gel from which it was extracted for further use yielding a concentration of 24 ng/µL.</P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.3">14.3 Gibson Assembly</A></LEGEND><DIV class="aim"><P>Aim: Integrate PheA-Arc1p-C-8x into linearized pET-28a(XhoI,NdeI) via Gibson Assembly</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Volume [µL]</TH></TR><TR><TH scope="row">Water</TH><TD>1.46</TD></TR><TR><TH scope="row">pET-28a(XhoI,NdeI) (24 ng/µL)</TH><TD>4.17</TD></TR><TR><TH scope="row">PheA-Arc1p-C-8x (30 ng/µL)</TH><TD>4.37</TD></TR><TR><TH scope="row">2x Gibson Mastermix</TH><TD>10</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD></TR></TBODY></TABLE><P>The reaction mixture was incubated for 1 h at 50 °C and 350 rpm. E. coli Top10 electrocompetent cells were transformed with a 1:3 and a 1:6 dilution of the mixture, incubated for 1 h at 37 °C and plated out on LB-Kan<SUP>50</SUP> plates that were incubated overnight at 37 °C.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.12">13.12 Miniprep of <I>E.coli</I> pMA12</A></LEGEND><DIV class="exp-content"><P>Miniprep according to the miniprep kit protocol (Omega) with overnight preculture.</P><P>DNA-concentration: 660 ng/µL</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.1">15.1 Cloning of Flagellin Construct</A></LEGEND><DIV class="exp-content"><H3>Dilution of Primers</H3><UL class="dilution"><LI>dilution with aqua bidest to 100 µM</LI><LI>dilution 1:10 → 10 µL primers (100 µM) with 90 µL aqua bidest</LI></UL></DIV><DIV class="exp-content"><H3>Reaction Mix</H3><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">[µL]</TH><TH scope="col">HagI (1145 bp)</TH><TH scope="col">HagII (859 bp)</TH></TR><TR><TH scope="row">Content</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Template Flo Pet24-Hag</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">dNTPs</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Primer iGEM-007 fwd (Hag-SpeI-F)</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer iGEM-008 red (Hag-SpeI)</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Primer 12 rev Flo</TH><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">Primer 11 few Flo</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Q5 Reaction Buffer 5x</TH><TD>5</TD><TD>5</TD></TR><TR><TH scope="row">Q5-Pol</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O</TH><TD>40</TD><TD>40</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>95</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>95</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1 min</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>30x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>Infinite</TD></TR></TBODY></TABLE></DIV><DIV class="results"><P>The gel shows that the PCR fragments were amplified successfully in the correct size (Size HagI=639 bp  HagII= 297 bp), The next step is a fusion PCR fusing the HagI/II constructs with homologous flanks for integration into the hag locus in <I>B. subtilis</I> genome.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="09.04.2014">09.04.2014</A></H2><FIELDSET class="exp13"><LEGEND><A name="exp13.15">13.15 Miniprep</A></LEGEND><DIV class="exp-content">Miniprep of transformed <I>E.coli</I> DH5α was performed according to the miniprep kit protocol (Omega) with overnight preculture.</DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.16/17">13.16/17 Restriction Digest with NcoI and Test-Plasmids</A></LEGEND><DIV class="exp-content">Restriction digest with NcoI and test-plasmids from the miniprep.</DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">[µL]</TH><TH scope="col">Mastermix with NcoI - for 14 attempts</TH><TH scope="col">Attempt - NcoI (20 kU/mL) - 12 attempts</TH></TR><TR><TH scope="row">DNA</TH><TD>-</TD><TD>ca. 350 ng/µL → ca. 1 µg → 2,9 µL</TD></TR><TR><TH scope="row">NcoI</TH><TD>3,5 (0,25/attempt)</TD><TD> </TD></TR><TR><TH scope="row">Master-Mix</TH><TD>-</TD><TD>12,1</TD></TR><TR><TH scope="row">Buffer 10x Tango</TH><TD>21</TD><TD> </TD></TR><TR><TH scope="row">H<SUB>2</SUB>O</TH><TD>144,9</TD><TD> </TD></TR><TR><TH scope="row">Total Volume</TH><TD>169,4</TD><TD> </TD></TR><TR><TH scope="row">Loading Dye 6x</TH><TD>-</TD><TD>2,5 /attempt</TD></TR></TBODY></TABLE></DIV><DIV class="exp-content"><P>Incubation for 60 min at 37 °C. Then a heat shock was induced for 2 min at 45 °C.</P></DIV><DIV class="results"><H3>Results</H3><P>Expectations: 4 bands (167 bp - 1033 bp - 1056 bp - 4410 bp)</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.18">13.18 Restriction Digest of positive Clones with NdeI</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">[µL]</TH><TH scope="col">Attempt 2.1</TH><TH scope="col">Attempt 2.3</TH><TH scope="col">Attempt 2.4</TH><TH scope="col">Attempt 4.2</TH></TR><TR><TH scope="row">DNA</TH><TD>628,8 ng/µL → ca. 1 µg → 1,6 µL</TD><TD>477,2 ng/µL → ca. 1 µg → 2,1 µL</TD><TD>694,9 ng/µL → ca. 1 µg → 1,4 µL</TD><TD>406,8 ng/µL → ca. 1 µg → 2,46 µL</TD></TR><TR><TH scope="row">NdeI</TH><TD>0,5</TD><TD>0,5</TD><TD>0,5</TD><TD>0,5</TD></TR><TR><TH scope="row">Cutsmart-Buffer 10x</TH><TD>1,5</TD><TD>1,5</TD><TD>1,5</TD><TD>1,5</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O</TH><TD>11,4</TD><TD>10,9</TD><TD>11,6</TD><TD>10,5</TD></TR><TR><TH scope="row">Total Volume</TH><TD>15</TD><TD>15</TD><TD>15</TD><TD>15</TD></TR><TR><TH scope="row">Loading Dye 6x</TH><TD>2,5</TD><TD>2,5</TD><TD>2,5</TD><TD>2,5</TD></TR></TBODY></TABLE></DIV><DIV class="results"><H3>Results</H3><P>Expected bands: 464 bp - 2148 bp - 4054 bp</P><P>The expected bands were visible.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.19">13.19 Restriction Digest of positive Clones with NcoI overnight</A></LEGEND><DIV class="exp-results"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">[µL]</TH><TH scope="col">Attempt 2.1</TH><TH scope="col">Attempt 2.3</TH><TH scope="col">Attempt 2.4</TH><TH scope="col">Attempt 4.2</TH></TR><TR><TH scope="row">DNA </TH><TD>628,8 ng/µL → ca. 5 µg → 8 µL</TD><TD>477,2 ng/µL → ca. 5 µg → 10,5 µL</TD><TD>694,9 ng/µL → ca. 5 µg → 7,2 µL</TD><TD>406,8 ng/µL → ca. 5 µg → 12,3 µL</TD></TR><TR><TH scope="row">NcoI</TH><TD>0,2</TD><TD>0,2</TD><TD>0,2</TD><TD>0,2</TD></TR><TR><TH scope="row">Tango Buffer 10x</TH><TD>1,5</TD><TD>1,5</TD><TD>1,5</TD><TD>1,5</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O</TH><TD>5,3</TD><TD>10,9</TD><TD>11,6</TD><TD>10,5</TD></TR><TR><TH scope="row">Total Volume</TH><TD>15</TD><TD>15</TD><TD>15</TD><TD>15</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.4">14.4 Overnight culture</A></LEGEND><DIV class="aim"><P>Aim: Amplify Plasmid in an overnight culture</P></DIV><DIV class="exp-content"><P>Six clones of the transformed Top10 cells were picked and used to inoculate overnight cultures (6 x 5 mL).</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="10.04.2014">10.04.2014</A></H2><FIELDSET class="exp15"><LEGEND><A name="exp15.4">15.4 PCR Hag-Flanks</A></LEGEND><DIV class="exp-content"><H3>Dilution of primers</H3><UL class="dilution"><LI>Hag I: 601,7 ng/µL</LI><LI>Hag II: 548,2 ng/µL</LI><LI>Flank I: 226 ng/µL</LI><LI>Flank II: 221 ng/µL</LI></UL></DIV><DIV class="results"><P>Hag I dilution: 35 ng/µL</P></DIV><DIV class="ex-content"><UL class="dilution"><LI>Hag I: 601,7 ng/µL * X = 10*35 ng/µL → 0,58:9,42</LI><LI>Hag II: 548,2 ng/µL * X = 10*35 ng/µL → 0,64:9,36</LI><LI>Flank I: 226 ng/µL * X = 10*35 ng/µL → 1,55:8,45</LI><LI>Flank II: 221 ng/µL * X = 10*35 ng/µL → 1,58:8,42</LI></UL></DIV><DIV class="exp-content"><H3>PCR Hag-Flanks Mix</H3><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix</TH><TH scope="col">Template Mix [µL]</TH><TH scope="col">Flank-Hag construct [µL]</TH></TR><TR><TH scope="row">Hag I(35 ng/µL)</TH><TD>2</TD><TD>-</TD></TR><TR><TH scope="row">Hag II(35 ng/µL)</TH><TD>2</TD><TD>-</TD></TR><TR><TH scope="row">Flank I(35 ng/µL)</TH><TD>2</TD><TD>-</TD></TR><TR><TH scope="row">Flank II(35 ng/µL)</TH><TD>2</TD><TD>-</TD></TR><TR><TH scope="row">Template Mix</TH><TD>-</TD><TD>3</TD></TR><TR><TH scope="row">Primer pMAD-Flank I fwd</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Primer pMAD-Flank I rev</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Q5-Mastermix 2x</TH><TD>-</TD><TD>25</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O</TH><TD>-</TD><TD>20</TD></TR><TR><TH scope="row">Total Volume</TH><TD>8</TD><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH width="17%" scope="col">Step</TH><TH width="58%" scope="col">Temperature °C</TH><TH width="25%" scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>95</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>95</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>2,5 min</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>35x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>Infinite</TD></TR></TBODY></TABLE></DIV><DIV class="exp-content"><H3>Results</H3><P>Expected bands: 2 kb band (whole Flank-Hag fragment)</P><P>The gel shows the successful amplification of the Hag-flank construct.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.6">15.6 Gel Extraction</A></LEGEND><DIV class="exp-content"><P>Gel extraction of 2,2 kb band:
      27,6 ng/µL</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.20">13.20 Yeast Transformation with positive NcoI Digests and Mutagenesis Primer (iGEM 009-0011)</A></LEGEND><DIV class="exp-content"><P>Attempts with 2 positive clones (2.1 NcoI cut &amp; 2.3 NcoI cut)</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix for trafo [µL]</TH><TH scope="col">p2.1 control</TH><TH scope="col">p2.3 control</TH><TH scope="col">p2.1
      +oligos
      +<I>amyE</I></TH><TH scope="col">p2.3
      +oligos
      +<I>amyE</I></TH><TH scope="col">p2.1
      Anneal+oligo
      +<I>amyE</I></TH><TH scope="col">p2.3
      Anneal+oligos
      +<I>amyE</I></TH></TR><TR><TH scope="row">p2.1</TH><TD>1</TD><TD>-</TD><TD>1</TD><TD>-</TD><TD>2</TD><TD>-</TD></TR><TR><TH scope="row">p2.3</TH><TD>-</TD><TD>1</TD><TD>-</TD><TD>1</TD><TD>-</TD><TD>2</TD></TR><TR><TH scope="row">iGEM-009</TH><TD>-</TD><TD>-</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">iGEM-010</TH><TD>-</TD><TD>-</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">iGEM-011</TH><TD>-</TD><TD>-</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row"><I>amyE</I>-fragment</TH><TD>-</TD><TD>-</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O</TH><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>14</TD><TD>14</TD></TR><TR><TH scope="row">PEG 3350
      (50% w/v)</TH><TD>260</TD><TD>260</TD><TD>260</TD><TD>260</TD><TD>260</TD><TD>260</TD></TR><TR><TH scope="row">LiAc 1 M</TH><TD>36</TD><TD>36</TD><TD>36</TD><TD>36</TD><TD>36</TD><TD>36</TD></TR><TR><TH scope="row">Salmonsperm
      (2 mg/mL)</TH><TD>50</TD><TD>50</TD><TD>50</TD><TD>50</TD><TD>50</TD><TD>50</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O</TH><TD>13</TD><TD>13</TD><TD>9</TD><TD>9</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Total Volume</TH><TD>360</TD><TD>360</TD><TD>360</TD><TD>360</TD><TD>-</TD><TD>-</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.21">13.21 Yeast Transformation with positive Clones</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix [µL]</TH><TH scope="col">2.1 control</TH><TH scope="col">2.3 control</TH><TH scope="col">3'' - P* + pool</TH><TH scope="col">4'' - P** (DS) + pool</TH></TR><TR><TH scope="row">PEG 3350 <EM>(50% w/v)</EM></TH><TD>260</TD><TD>260</TD><TD>260</TD><TD>260</TD></TR><TR><TH scope="row">LiAc 1 M</TH><TD>36</TD><TD>36</TD><TD>36</TD><TD>36</TD></TR><TR><TH scope="row">Salmonsperm <EM>(2 mg/mL)</EM></TH><TD>50</TD><TD>50</TD><TD>50</TD><TD>50</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O</TH><TD>14</TD><TD>10</TD><TD>2</TD><TD>2</TD></TR><TR><TH scope="row">Fragm. <EM>(GFP, Cat, AmyE)</EM></TH><TD>-</TD><TD>4</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Pool (all 3 fragments)</TH><TD>-</TD><TD>-</TD><TD>12</TD><TD>12</TD></TR><TR><TH scope="row">Total Volume</TH><TD>360</TD><TD>360</TD><TD>360</TD><TD>360</TD></TR></TBODY></TABLE><P>Salmonsperm for 10 min on 95 °C, then on ice - all attempts on ice!<EM>*pMA12 - digestion with HindIII + EcoRI
	**pMA12 - parallel digestion of HindIII and EcoRI</EM></P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.22">13.22 Yeast Transformation</A></LEGEND><DIV class="exp-content"><UL class="cloning"><LI>Resuspend yeast pellet in attempts</LI><LI>Heat shock at 42 °C for 35 min</LI><LI>Centrifugation for 30 sec - 1 min at 13.000 g</LI><LI>Resuspend pellet in 200 µL sterile water</LI><LI>Plate yeast on selective medium</LI><LI>Incubate at 30 °C till monday</LI></UL></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.5">14.5 Preparation of plasmids</A></LEGEND><DIV class="aim"><P>Aim: Purify the amplified plasmids from overnight cultures</P></DIV><DIV class="exp-content"><P>The plasmids were purified from the overnight cultures using Sigma GenElute Plasmid Miniprep Kit following the included protocol. The plasmids were eluted using 45 µL water.</P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.6">14.6 Transformation of BL21 cells</A></LEGEND><DIV class="aim"><P>Aim: Transform E. coli BL21(DE3) with PheA-Arc1p-C-8x-pET28a</P></DIV><DIV class="exp-content"><P>40 µL E. coli BL21(DE3) electrocompetent cells were transformed with 1 ng of the previously purified plasmid PheA-Arc1p-C-8x-pET28a from two different clones, incubated for 1 h at 37 °C and plated out on LB-Kan<SUP>50</SUP> plates that were incubated overnight at 37 °C.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="11.04.2014">11.04.2014</A></H2><FIELDSET class="exp15"><LEGEND><A name="exp15.7">15.7 Test PCR with PCR created Fragment as Template Flagellin Constructs</A></LEGEND><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix [µL]</TH><TH scope="col">Test PCR</TH></TR><TR><TH scope="row">Template <EM>(Hag-Flank construct)</EM></TH><TD>2</TD></TR><TR><TH scope="row">Primer pMAD-Flank I fwd</TH><TD>1</TD></TR><TR><TH scope="row">Primer pMAD-Flank II rev</TH><TD>1</TD></TR><TR><TH scope="row">Q5-Mastermix 2x</TH><TD>5</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O</TH><TD>1</TD></TR><TR><TH scope="row">Total Volume</TH><TD>10</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH width="17%" scope="col">Step</TH><TH width="58%" scope="col">Temperature °C</TH><TH width="25%" scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>95</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>95</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>2,5 min</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>35x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>Infinite</TD></TR></TBODY></TABLE><P>The extracted 2000 bp fragment was used as a PCR template for amplification of the Hag-flank construct for further cloning steps.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.8">15.8 Digestion of Flagellin Fragment with <I>Nco</I>I/<I>Bam</I>HI</A></LEGEND><DIV class="aim"><P>Aim: Creating restriction sites for cloning into pMAD-vector.</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix [µL]</TH><TH scope="col">Hag-Flank-Fragment digest <I>Nco</I>I/<I>Bam</I>HI</TH></TR><TR><TH scope="row">DNA</TH><TD>9 ng/µL → 25 µL = 225 ng
      27,6 ng/µL → 15 µL = 414 ng → 639 ng in 40 µL</TD></TR><TR><TH scope="row">Enzyme <I>NcoI</I>-HF</TH><TD>0,5</TD></TR><TR><TH scope="row">Enzyme <I>BamHI</I>-HF</TH><TD>0,5</TD></TR><TR><TH scope="row">Cutsmart Buffer (10x)</TH><TD>5</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O</TH><TD>4</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD></TR></TBODY></TABLE></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="12.04.2014">12.04.2014/13.04.2014</A></H2><FIELDSET class="exp13"><LEGEND><A name="exp13.22">13.22 Miniprep of Yeast Culture Transformation</A></LEGEND><DIV class="exp-content"><P>The plasmid isolation was carried out according to the miniprep kit protocol.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.23">13.23 Transformation of <I>E.coli</I> DH5<FONT face="arial">α</FONT> with prepared plasmids</A></LEGEND><DIV class="exp-content"><P>Plasmids prepared above have been transformed into <I>E.coli</I>.
      Colonies on transformation plates have been inoculated in shaking culture (16 colonies).
    </P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="14.04.2014">14.04.2014</A></H2><FIELDSET class="exp13"><LEGEND><A name="exp13.24">13.24 Miniprep of Shaking Cultures</A></LEGEND><DIV class="exp-content"><P>Miniprep according to the miniprep kit protocol (Omega).
      2.1 and 2.3 different positive colonies from
    09.04.2014.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.25">13.25 Digestion</A></LEGEND><DIV class="aim"><P>Aim: Check for elimination of <I>Nco</I>I-restriction sites.</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix</TH><TH scope="col">[µl]</TH></TR><TR><TH scope="row">DNA</TH><TD>1</TD></TR><TR><TH scope="row">Enzyme <I>Nco</I>I-HF</TH><TD>0,75</TD></TR><TR><TH scope="row">Cutsmart Buffer 10x</TH><TD>1,5</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O</TH><TD>12,25</TD></TR><TR><TH scope="row">Total Volume</TH><TD>15</TD></TR></TBODY></TABLE></DIV><DIV class="results"><H3>Results</H3><P>One band was expected for a linearized plasmid, 4 bands occurred, the restriction sites have presumably not been removed. Expected sizes for plasmid with 4 NcoI-restriction sites: ~ 4400 bp, 2 x ~ 1000 bp, ~ 200 bp</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.26">13.26 Overnight Digestion</A></LEGEND><DIV class="aim"><P>Aim: Check for elimination of <I>Nco</I>I-restriction sites</P></DIV><DIV class="exp-content"><P>Overnight digestion of two samples with the highest DNA-concentration (2.1 2 and 3.1 A1</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix [µL]</TH><TH scope="col">2.1 2</TH><TH scope="col">3.1 A1</TH></TR><TR><TH scope="row">DNA</TH><TD>10 µg/8 µL</TD><TD>10 µg/9 µL</TD></TR><TR><TH scope="row">Enzyme <I>NcoI</I>-HF</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Cutsmart Buffer 10x</TH><TD>2</TD><TD>2</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O</TH><TD>9</TD><TD>8</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>20</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.7">14.7 Overnight culture</A></LEGEND><DIV class="aim"><P>Aim: Inoculate Overnight culture of PheA-Arc1p-C-8x-pET28a</P></DIV><DIV class="exp-content"><P>5 mL of LB-Kan<SUP>50</SUP> medium were inoculated with a clone from 14.6 bearing the PheA-Arc1p-C-8x-pET28a plasmid.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="13.04.2014">15.04.2014</A></H2><FIELDSET class="exp13"><LEGEND><A name="exp13.27a">13.27 Mutagenesis PCR</A></LEGEND><DIV class="aim"><P>Aim: Mutate the NcoI-restriction sites, because earlier attempts were not successful</P></DIV><DIV class="exp-content"><P>As template the restriction of the sample 3.1 A1 has been used. The mutagenesis primers have been chosen together with fitting reverse primers to create fragments without the NcoI-restriction sites. These fragments should further be transformed into yeast and assembled into the whole plasmid without the restriction sites for NcoI.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix [µL]</TH><TH scope="col">Fragment 1</TH><TH scope="col">Fragment 2</TH><TH scope="col">Fragment 3</TH></TR><TR><TH scope="row">Restriction mix 3.1 A1<EM> (2,08 ng/µL)</EM></TH><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">iGEM-009 (10 mM)</TH><TD>1</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">Ag TWp7 (10 mM)</TH><TD>1</TD><TD>-</TD><TD>-</TD></TR><TR><TH scope="row">iGEM-010 (10 mM)</TH><TD>-</TD><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">iGEM-003 (10 mM)</TH><TD>-</TD><TD>1</TD><TD>-</TD></TR><TR><TH scope="row">iGEM-011 (10 mM)</TH><TD>-</TD><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">iGEM-006 (10 mM)</TH><TD>-</TD><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Q5-Mastermix 2x</TH><TD>25</TD><TD>25</TD><TD>25</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O</TH><TD>22</TD><TD>22</TD><TD>22</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD><TD>50</TD><TD>50</TD></TR></TBODY></TABLE><P>Every PCR has been carried out in double and was further analyzed on a 1% gel.</P></DIV><DIV class="results"><H3>Results</H3><P><STRONG>Mutagenesis PCR for different fragments; 
      1: 2-log-DNA-ladder, 
      2  &amp; 3: fragment 1, 
      4 &amp; 5: fragment 2, 
      6 &amp; 7: fragment 3.
      </STRONG>
      Lane 1 &amp; 2 contain the expected Fragments (~ 600  bp). In Lane 3 there is a very thin fragment with the expected size (~ 1100 bp)  but smaller fragments can also be seen. Lane 5 does not contain any fragment at  all and in lane 6 there is the fragment of the expected size (~ 600 bp).      Both samples with fragment 1 were pooled, sample 3.1  was discarded and the whole PCR-product 2.1 has been separated in an agarosegel  and the fragment with 1100 bp has been cut out and purified with the Gel  Extraction Kit (c = 4,4 ng/µL). A second PCR for fragment 2 has been done in parallel.  All parameters remained the same but the annealing-temperature has been changed  to 62 °C.</P><P><STRONG>Modified mutagenesis PCR for fragment 2; again there are two fragments with a too small size and the expected fragments are not high concentrated.</STRONG></P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.8">14.8 Test Expression</A></LEGEND><DIV class="aim"><P>Aim: Test the expression of PheA-Arc1p-C-8x</P></DIV><DIV class="exp-content"><P>Using the preculture from 14.7 60 mL of LB-Kan<SUP>50</SUP> medium were inoculated with 600 µL of the preculture and incubated at 37 °C and 220 rpm until OD<SUB>600</SUB> reached 0.5. The culture was cooled down to 18 °C and the protein production was induced with 12 µL of 500 mM IPTG. Over night the culture was incubated at 18 °C and 220 rpm. A glycerolstock of the strain containing the PheA-Arc1p-C-8x-pET28a plasmid was stored at -80 °C.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="16.04.2014">16.04.2014</A></H2><FIELDSET class="13"><LEGEND><A name="exp13.27b">13.27 Cloning of Promotor</A></LEGEND><DIV class="exp-content"><P>To be sure to obtain just a fragment of the wanted size a PCR with the same Primers (iGEM-003 and iGEM-010) has been carried out with the thin fragment which was cut out of the gel yesterday.</P><P><STRONG>PCR of the fragment 2; 
      1: 2-log-DNA-ladder, 
      2 &amp; 3: fragment 2
      </STRONG>
      Besides the expected fragment there are also other fragments which indicate that the primers partially bind unspecific. The rest of the PCR products has been separated with an agarose gel and the wanted fragment has been cut out and purified with the Gel Extraction Kit (c = 21 ng/µL).</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.28">13.28 Yeast Transformation with PCR fragments</A></LEGEND><DIV class="aim"><P>Aim: Rebuild the plasmid via the yeast recombination system</P></DIV><DIV class="exp-content"><P>The three obtained fragments together with the <I>Nco</I>I-digested plasmid pMA12 have been transformed into <EM>S. cerevisia</EM>e to get the whole plasmid again. The whole procedure has been done according to the protocol.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix for trafo [µL]</TH><TH scope="col">Volumes</TH></TR><TR><TH scope="row"><I>Nco</I>I-restriced pMA12 (2.1 1)</TH><TD>1</TD></TR><TR><TH scope="row">Fragment 1 (146 ng/µL)</TH><TD>1</TD></TR><TR><TH scope="row">Fragment 2 (21,7 ng/µL)</TH><TD>9</TD></TR><TR><TH scope="row">Fragment 3 (78,6 ng/µL)</TH><TD>3</TD></TR><TR><TH scope="row">PEG 3350(50% w/v)</TH><TD>260</TD></TR><TR><TH scope="row">LiAc 1 M</TH><TD>36</TD></TR><TR><TH scope="row">Salmonsperm <EM>(2 mg/mL)</EM></TH><TD>50</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O</TH><TD>-</TD></TR><TR><TH scope="row">Total Volume</TH><TD>360</TD></TR></TBODY></TABLE><P>The transformed yeast cells are incubated at RT over the easter holidays.</P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.9">14.9 Purification of test expression</A></LEGEND><DIV class="aim"><P>Aim: Purify PheA-Arc1p-C-8x on a small scale</P></DIV><DIV class="exp-content"><P>The cell culture from 14.8 was centrifuged (17000 rpm, 20 min, 4 °C) and the pellet resuspended in 3 mL buffer A. Lysozyme was added to a final concentration of 1 mg/mL and incubated on ice for 30 min. Afterwards the cells were lysed using ultra sound (2x 18 pulses, 30 s on ice inbetween). To clear the suspension of cell debris it was centrifuged (13000 rpm, 15 min, 4 °C). The supernatant was purified using QIAGEN Ni-NTA Spin Columns. An SDS-PAGE gel showed that the protein is produced and can be purified using Ni affinity chromatography.</P><P>60 mL LB-Kan<SUP>50</SUP> medium was inoculated from the glycerolstock prepared in 14.8 and incubated at 37 °C and 220 rpm over night.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="17.04.2014">17.04.2014</A></H2><FIELDSET class="exp14"><LEGEND><A name="exp14.10">14.10 Expression of PheA-Arc1p-C-8x</A></LEGEND><DIV class="aim"><P>Aim: Express PheA-Arc1p-C-8x for further use</P></DIV><DIV class="exp-content"><P>10 x 500 mL LB-Kan<SUP>50</SUP> medium were inoculated with 5 mL of the overnight culture prepared in 14.9 and incubated at 37 °C and 220 rpm until OD<SUB>600</SUB> reached 0.5. The flasks were cooled to 18 °C and protein production was induced by adding 100 µL 500 mM IPTG. The cultures were incubated over night at 18 °C and 220 rpm. Afterwards the cells were collected by centrifugation (6000 rpm, 20 min, 4 °C), resuspended in buffer A and stored at -20 °C until further use.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="22.04.2014">22.04.2014</A></H2><FIELDSET class="exp13"><LEGEND><A name="exp13.29">13.29 Miniprep of Transformed Yeast <EM>S.cerevisiae </EM>and Trafo of <EM>E.coli</EM> DH5Alpha</A></LEGEND><DIV class="Aim"><P>Aim: Increase the amount of plasmid DNA to perform restriction and further cloning steps.</P></DIV><DIV class="exp-content"><P>Miniprep of Transformed Yeast <EM>S.cerevisiae</EM> from 21.04.2014 and Trafo of <EM>E.coli</EM> DH5Alpha.
			  Miniprep was performed according to the miniprep protocol (Omega).
			</P><P>Plasmid pMa12 with 3 inserts (cat, amyE and gfp) with removed NcoI restriction sites.
			  Concentration after plasmid prep= 7.5 ng/µL
			  Transformation of <I>E.coli</I> DH5α with 5 µL DNA (37.5 ng)
			  Over night incubation on LB-Amp plates.</P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.11">14.11 Purification of PheA-Arc1p-C-8x</A></LEGEND><DIV class="aim"><P>Aim: Purify PheA-Arc1p-C-8x in three steps</P></DIV><DIV class="exp-content"><P>The resuspended cells from 14.10 were lysed using a french press. The lysate was incubated on ice with DNAseA and RNAseI for 10 min and centrifuged (27000 rpm, 45 min, 4 °C). The supernatant was filtered (0.45 µm) and PheA-Arc1p-C-8x purified from the cleared solution by NiNTA affinity chromatography. The combined elution fractions containing crude PheA-Arc1p-C were concentrated to a volume of 2 mL and further purified using an anion exchange column to get rid of possible tRNA contaminations. Finally the buffer was changed to dialysis buffer and the protein concentrated and stored at -80 °C in aliquods for further use.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="23.04.2014">23.04.2014</A></H2><FIELDSET class="exp13"><LEGEND><A name="exp13.30">13.30 Transformation of pMA12-construct into <EM>E.coli</EM> DH5α</A></LEGEND><DIV class="exp-content"><P>The transformation of the <EM>E.coli</EM> cells was not successful. So it has been repeated with the rest of the prepared plasmid.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.9">15.9 PCR-Amplification of Flagellin constructs</A></LEGEND><DIV class="aim"><P>Aim: PCR to amplify the product for ligation into pMAD</P></DIV><DIV class="exp-content"><P>The concentration of the Hag-Flank-construct was too low (c = 5.8 ng/µL) to use it for the ligation into pMAD. A PCR has been run to increase the amount of the construct.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix [µL]</TH><TH scope="col">Test PCR</TH></TR><TR><TH scope="row">Template (Hag-Flank construct)</TH><TD>2 (12 ng)</TD></TR><TR><TH scope="row">Primer 89 from Florian</TH><TD>1</TD></TR><TR><TH scope="row">Primer 90 from Florian</TH><TD>1</TD></TR><TR><TH scope="row">Q5-Mastermix 2x</TH><TD>25</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O</TH><TD>21</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD></TR></TBODY></TABLE><P>PCR Q5 elongation for 2 kb fragments</P><TABLE width="100%" border="1"><TBODY><TR><TH width="17%" scope="col">Step</TH><TH width="58%" scope="col">Temperature °C</TH><TH width="25%" scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>95</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>95</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>2,5 min</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>35x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>Infinite</TD></TR></TBODY></TABLE><P>The PCR-product has been checked on an agarose gel (3 µL + 1 µL 6x Loading-Dye)
Expected band: 2000 bp is there, but also 2 additional bands
A band with the expected size of ca 2000 bp is visible but there are also two additional DNA-fragments with a lower size. The rest of the PCR-product (47 µL) has been separated on a gel and the expected DNA-fragment has been excised and purified with the Gel Extraction Kit. The purification led to a concentration of 45 ng/µL.</P></DIV><DIV class="results"><H3>Results</H3><P>Figure: PCR of the Hag-Flank-construct</P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.12">14.12 Creating PheA-Arc1p-C-4x</A></LEGEND><DIV class="aim"><P>Aim: Using Round-the-horn mutagenesis to create PheA-Arc1p-C-4x-pET28a</P></DIV><DIV class="exp-content">
    The PCR with 5' phosphorylated TG_2xGSSG FP und TG_2xGSSG RP should generate the 7545 bp fragment for ligation from the methylated template PheA-Arc1p-C-8x-pET28a from 14.5.<P>The used Primer concentration was 1 
    µM, the concentration of the template was 1 ng/µL and we used a hot start to avoid unspecific annealing of primers.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Volume [µL]</TH></TR><TR><TH scope="row">Water</TH><TD>29.5</TD></TR><TR><TH scope="row">5x GC Buffer</TH><TD>10</TD></TR><TR><TH scope="row">DMSO</TH><TD>2.5</TD></TR><TR><TH scope="row">TG_2xGSSG FP</TH><TD>2.5</TD></TR><TR><TH scope="row">TG_2xGSSG RP</TH><TD>2.5</TD></TR><TR><TH scope="row">PheA-Arc1p-C-8x-pET28a</TH><TD>1</TD></TR><TR><TH scope="row">dNTPs</TH><TD>1</TD></TR><TR><TH scope="row">Phusion-Polymerase</TH><TD>1</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD></TR></TBODY></TABLE><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature [°C]</TH><TH scope="col">Time [min:sec]</TH></TR><TR><TH scope="row">1</TH><TD>98</TD><TD>2:00</TD></TR><TR><TH scope="row">2</TH><TD>add Polymerase</TD><TD> </TD></TR><TR><TH scope="row">3</TH><TD>98</TD><TD>0:10</TD></TR><TR><TH scope="row">4</TH><TD>67.5</TD><TD>0:30</TD></TR><TR><TH scope="row">5</TH><TD>72</TD><TD>4:00</TD></TR><TR><TH scope="row">6</TH><TD>go to 3</TD><TD>32x</TD></TR><TR><TH scope="row">7</TH><TD>72</TD><TD>10:00</TD></TR><TR><TH scope="row">8</TH><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P>The amplified fragment was purified on an agarose gel on which it showed the correct size. The extracted linear template was then digested to degrade the original unmutated plasmid using DpnI.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Content</TH><TH scope="col">Volume [µL]</TH></TR><TR><TH scope="row">10x CutSmart Buffer</TH><TD>3.6</TD></TR><TR><TH scope="row">Plasmid</TH><TD>30</TD></TR><TR><TH scope="row">DpnI</TH><TD>3</TD></TR><TR><TH scope="row">Total Volume</TH><TD>36.6</TD></TR></TBODY></TABLE><P>The reaction mixture was incubated at 37 °C and 300 rpm for 1 h and afterwards heated to 80 °C for 20 min</P><P>In order to create the circular plasmid 100 ng of the linear template were mixed with 1 µL 10x T4-Ligasebuffer,  1 µL T4-Ligase and water to give a total volume of 10 µL. This mixture was incubated for 9 h at 16 °C. The resulting plasmid was dialysed with water to decrease the salt concentration for the following transformation of E. coli Top10 electrocompetent cells with the plasmid.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="24.04.2014">24.04.2014</A></H2><FIELDSET class="exp13"><LEGEND><A name="exp13.31">13.31 Inoculation of Miniprep of Transformed DH5α</A></LEGEND><DIV class="aim"><P>Aim: Increase the amount of plasmid DNA to perform restriction and further cloning steps.</P></DIV><DIV class="exp-content"><P>Seven small colonies grew on the plate. The plate was further incubated  until the afternoon. Each 6 mL LB-Amp have been inoculated with one of the  colonies and incubated over night at 37°C.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.10">15.10 Digestion with <I>Nco</I>I/<I>Bam</I>HI of PCR-amplified Flagellin construct</A></LEGEND><DIV class="aim"><P>Aim: Create restriction sites for ligation the fragment into pMad.</P></DIV><DIV class="exp-content"><P>The hag-flank-construct which was amplified yesterday has been digested with <I>Nco</I>I and <I>Bam</I>HI to create sticky ends for the following ligation.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix [µL]</TH><TH scope="col">Hag-Flank-Fragment digest <I>Nco</I>I/<I>Bam</I>HI</TH></TR><TR><TH scope="row">DNA</TH><TD>1350 ng in 30 µL</TD></TR><TR><TH scope="row">Enzyme <I>Nco</I>I-HF</TH><TD>1</TD></TR><TR><TH scope="row">Enzyme <I>Bam</I>HI-HF</TH><TD>1</TD></TR><TR><TH scope="row">Cutsmart-Buffer 10x</TH><TD>5</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O</TH><TD>13</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD></TR></TBODY></TABLE><P>The restriction was carried out at 37°C for 45 min and the digested fragment was purified with the Gel Extraction Kit (c = 24 ng/µL).</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.11">15.11 Ligation of pMAD and Flagellin Construct Digestion</A></LEGEND><DIV class="aim"><P>Aim: Ligation of the flagellin-fragment with pMAD-vector</P></DIV><DIV class="exp-content"><P>(20 ng of vector x kb size of insert/kb size of vector) x molar of ratio of (insert/vector)</P><P>0.6 µL of the pMAD-vector (30 ng) and 1 µL of the digested hag-flank-construct (18 ng) were used for the ligation.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix [µL]</TH><TH scope="col">Control atempt </TH><TH scope="col">2x attempt</TH></TR><TR><TH scope="row">Plasmid (pMAD)</TH><TD>0,6</TD><TD>0,6</TD></TR><TR><TH scope="row">Insert (construct)</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">T4 Ligase</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">T4 Ligase Buffer (10x)</TH><TD>2</TD><TD>2</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O</TH><TD>16,4</TD><TD>15,4</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><P>The ligation-mix was incubated at roomtemperature for 1 h followed by a transformation of <I>E. coli</I> DH5α with 10 µL of the ligation-mix. The transformation was incubated on a LB-Amp-plate at 37 °C over night.</P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.13">14.13 Overnight culture</A></LEGEND><DIV class="aim"><P>Aim: Amplify Plasmid in an overnight culture</P></DIV><DIV class="exp-content"><P>Six clones of the transformed Top10 cells from 14.12 were picked and used to inoculate overnight cultures (6 x 5 mL).</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="25.04.2014">25.04.2014</A></H2><FIELDSET class="exp13"><LEGEND><A name="exp13.32">13.32 Miniprep of Transformed DH5&amp;alpha</A></LEGEND><DIV class="aim"><P>The seven cultures were used to do a miniprep. The miniprep was performed according to the miniprep protocol (Omega).</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Clone</TH><TH scope="col">Concentration (ng/µL)</TH></TR><TR><TH scope="row">1</TH><TD>420</TD></TR><TR><TH scope="row">2</TH><TD>298,9</TD></TR><TR><TH scope="row">3</TH><TD>330</TD></TR><TR><TH scope="row">4</TH><TD>392</TD></TR><TR><TH scope="row">5</TH><TD>435,3</TD></TR><TR><TH scope="row">6</TH><TD>560</TD></TR><TR><TH scope="row">7</TH><TD>485</TD></TR></TBODY></TABLE></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.33">13.33 Restriction Digestion with NcoI</A></LEGEND><DIV class="aim"><P>Aim: Check for elimination of <I>Nco</I>I-restriction sites.</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">[µL]</TH><TH scope="col">Attempt 1-7 </TH><TH scope="col">Mastermix</TH></TR><TR><TH scope="row">Plasmid</TH><TD>1,5</TD><TD>-</TD></TR><TR><TH scope="row">Enzyme <I>Nco</I>I</TH><TD>-</TD><TD>1</TD></TR><TR><TH scope="row">Tango Buffer (10x)</TH><TD>2</TD><TD>16</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O</TH><TD>15,5</TD><TD>124</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>1</TD></TR></TBODY></TABLE></DIV><DIV class="results"><H3>Results</H3><P>The digest was further analyzed on a 1% agarose gel.</P><P><STRONG>Restriction digest of the prepared pMa12-construct with NcoI; the star means in this lane was the pMa12-construct (before the installation of the mutated-fragments) digested with NcoI as a control.</STRONG>
      The clones 2, 3, 4, 5 &amp; 7 appeared to contain the plasmid with only one NcoI-restriction side. This clones were transferred into 6 mL LB-Amp and incubated at 37 &amp;deg,C over night.
</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.13">15.13 New Ligation of pMAD and Flagellin construct digest</A></LEGEND><DIV class="aim"><P>Aim: Ligation of the flagellin-fragment with pMAD-vector.</P></DIV><DIV class="exp-content"><P>The plates from 15.12 were empty which indicates an unsuccessful ligation. A new ligation was performed after calculation with a Ligation calculator: <A href="https://2011.igem.org/Team:UT_Dallas/ligation" target="_blank">UT Dallas</A></P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix [µL]</TH><TH scope="col">Control attempt </TH><TH scope="col">2x attempt</TH></TR><TR><TH scope="row">Plasmid (pMAD)</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Insert (construct)</TH><TD>-</TD><TD>1,9</TD></TR><TR><TH scope="row">T4 Ligase</TH><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">T4 Ligase Buffer (10x)</TH><TD>2</TD><TD>2</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O</TH><TD>16</TD><TD>14,4</TD></TR><TR><TH scope="row">Total Volume</TH><TD>20</TD><TD>20</TD></TR></TBODY></TABLE><P>The ligation was incubated at roomtemperature for 1.5 h. Afterwards it was stored at 
-20 °C.
</P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.14">14.14 Preparation of plasmids</A></LEGEND><DIV class="aim"><P>Aim: Purify the amplified plasmids from overnight cultures</P></DIV><DIV class="exp-content"><P>The plasmids were purified from the overnight cultures created in 14.13 using Sigma GenElute Plasmid Miniprep Kit following the included protocol. The plasmids were eluted using 45 µL water.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="28.04.2014">28.04.2014</A></H2><FIELDSET class="exp13"><LEGEND><A name="exp13.35a">13.35 PCR with Primers iGEM-002 und iGEM-010</A></LEGEND><DIV class="aim"><P>Aim: Create new mutation fragment for eliminating a NcoI-restriction site.</P></DIV><DIV class="exp-content"><P>The plasmids prepared on Saturday seem to be smaller as expected. For this reason a new attempt to eliminate the restriction sites was carried out. The amplification of fragment 2 (with primers iGEM 003 and iGEM 010 on 15.04.2014) did not happen without any problems such as unspecific smaller bands in addition to the expected one (ca 1000 bp). So this time primer iGEM 003 was replaced by iGEM 002 which would result in a smaller fragment of approximately 600 bp. Later on the fragments 1 and 3 together with the new created one, the digested pMa12 construct and the amyE-fragment (to overlap the region between fragments) would be transformed into <EM>S. cerevisiae </EM>for recombination.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix [µL]</TH><TH scope="col">New Fragment 2</TH></TR><TR><TH scope="row">Restriction mix 3.1 A1 (2,08 ng/µL)</TH><TD>1</TD></TR><TR><TH scope="row">iGEM-010 (10 mM)</TH><TD>1</TD></TR><TR><TH scope="row">iGEM-002 (10 mM)</TH><TD>1</TD></TR><TR><TH scope="row">Q5-Mastermix 2x</TH><TD>25</TD></TR><TR><TH scope="row">H<SUB>2</SUB>0</TH><TD>22</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD></TR></TBODY></TABLE><P>The PCR was performed according to the PCR protocol and the PCR-products were analyzed on a 1% agarose gel.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Step</TH><TH scope="col">Temperature °C</TH><TH scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>95</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>95</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>58</TD><TD>20 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>1 min</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>35x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>Infinite</TD></TR></TBODY></TABLE></DIV><DIV class="results"><H3>Results</H3><P>PCR with the primers iGEM-002 and iGEM-010 and the digested pMa12-vector
The expected fragments with the sizes of ca 600 bp are visible.</P></DIV></FIELDSET><FIELDSET class="exp13"><LEGEND><A name="exp13.35b">13.35 Transformation of Yeast with the different Fragments</A></LEGEND><DIV class="aim"><P>Aim: Recombination of the fragment to a whole plasmid in yeast.</P></DIV><DIV class="exp-content"><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix [µL]</TH><TH scope="col">Volumes</TH></TR><TR><TH scope="row"><I>Nco</I>I-digested pMA12 (2.1 1)</TH><TD>2</TD></TR><TR><TH scope="row">Fragment 1 (146 ng/µL)</TH><TD>2</TD></TR><TR><TH scope="row">New Fragment 2 (412,9 ng/µL)</TH><TD>2</TD></TR><TR><TH scope="row">Fragment 3 (78,6 ng/µL)</TH><TD>3</TD></TR><TR><TH scope="row">Fragment amyE (48,3 ng/µL)</TH><TD>5</TD></TR><TR><TH scope="row">PEG 3350(50% w/v)</TH><TD>260</TD></TR><TR><TH scope="row">LiAc 1 M</TH><TD>36</TD></TR><TR><TH scope="row">Salmonsperm(2mg/mL)</TH><TD>50</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O</TH><TD>-</TD></TR><TR><TH scope="row">Total Volume</TH><TD>360</TD></TR></TBODY></TABLE><P>The transformed yeast cells were incubated at 30 °C.</P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.15">14.15 Transformation of BL21 cells</A></LEGEND><DIV class="aim"><P>Aim: Transform E. coli BL21(DE3) with PheA-Arc1p-C-4x-pET28a</P></DIV><DIV class="exp-content"><P>40 µL E. coli BL21(DE3) electrocompetent cells were transformed with 1 ng of the previously purified plasmid PheA-Arc1p-C-4x-pET28a from two different clones, incubated for 1 h at 37 °C and plated out on LB-Kan<SUP>50</SUP> plates that were incubated overnight at 37 °C.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="29.04.2014">29.04.2014</A></H2><FIELDSET class="exp17"><LEGEND><A name="exp17.1">17.1 Making new competent <EM>E.coli</EM> DH5α</A></LEGEND><DIV class="aim"><P>Aim: Making competent <EM>E.coli</EM> DH5α cells.</P></DIV><DIV class="exp-content"><P>Some new competent <EM>E. coli</EM> DH5α cells were made because the last batch of cells have not yielded in good results in the last few transformations. The cells were made according to the protocol in the method section below. To test the cells they were plated on normal LB, LB-Amp, LB-Kan and a test-trafo was carried out with pMa12 which provides an ampicillin resistance.</P></DIV></FIELDSET><FIELDSET class="exp15"><LEGEND><A name="exp15.14">15.14 New PCR-Amplification of different Attempts for the Hag-Flank-Construct</A></LEGEND><DIV class="aim"><P>Aim: Amplification of construct after unsuccessful transformation for new digests.</P></DIV><DIV class="exp-content"><P>The transformation of <EM>E. coli</EM> DH5α with the ligation of the pMAD-vector and the hag-flank-construct was not successful. Therefore the PCR for amplification of the construct was repeated by using the hag / flank-parts or the construct itself as template.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">Mix [µL]</TH><TH scope="col">PCR 1</TH><TH scope="col">PCR 2</TH><TH scope="col">PCR 3</TH></TR><TR><TH scope="row">Template</TH><TD>2 (Template mix Hag I/II + Flanks 15,5)</TD><TD>1 (Flagellin-construct 15,6)</TD><TD>1 (Flagellin-construct digest (15,10)</TD></TR><TR><TH scope="row">Primer 89 from Florian</TH><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Primer 90 from Florian</TH><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Q5-Mastermix 2x</TH><TD>25</TD><TD>25</TD><TD>25</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O</TH><TD>21</TD><TD>22</TD><TD>22</TD></TR><TR><TH scope="row">Total Volume</TH><TD>50</TD><TD> </TD><TD> </TD></TR></TBODY></TABLE><P>PCR was performed according to the &quot;PCR Q5 elongation for 2kb fragments&quot; protocol and the PCR-products were analyzed on a 1% agarose gel.</P><TABLE width="100%" border="1"><TBODY><TR><TH width="17%" scope="col">Step</TH><TH width="58%" scope="col">Temperature °C</TH><TH width="25%" scope="col">Time</TH></TR><TR><TH scope="row">1</TH><TD>95</TD><TD>3 min</TD></TR><TR><TH scope="row">2</TH><TD>95</TD><TD>30 sec</TD></TR><TR><TH scope="row">3</TH><TD>55</TD><TD>30 sec</TD></TR><TR><TH scope="row">4</TH><TD>72</TD><TD>2,5 min</TD></TR><TR><TH scope="row">5</TH><TD>Go To 2</TD><TD>35x</TD></TR><TR><TH scope="row">6</TH><TD>72</TD><TD>5 min</TD></TR><TR><TH scope="row">7</TH><TD>4</TD><TD>Infinite</TD></TR></TBODY></TABLE></DIV><DIV class="results"><H3>Results</H3><P>PCR with primers 89 and 90 from Florian and  different templates; lane 1: 2-log-ladder, lane 2: PCR1, lane 3: PCR2, lane 4:  PCR3</P><P>Although the marker is  barely visible the expected bands can detected at a relative height of 2000 in  addition to some other unspecific fragments.</P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.16">14.16 Overnight culture</A></LEGEND><DIV class="aim"><P>Aim: Inoculate Overnight culture of PheA-Arc1p-C-4x-pET28a</P></DIV><DIV class="exp-content"><P>5 mL of LB-Kan<SUP>50</SUP> medium were inoculated with a clone from 14.15 bearing the PheA-Arc1p-C-4x-pET28a plasmid.</P></DIV></FIELDSET></DIV><DIV class="notebooky-entry"><H2 class="title"><A name="30.04.2014">30.04.2014</A></H2><FIELDSET class="exp15"><LEGEND><A name="exp15.15">15.15 Restriction of the Hag-Flank-Construct and the pMAD-Vector</A></LEGEND><DIV class="aim"><P>Aim: Create the fitting sticky ends for a ligation.</P></DIV><DIV class="exp-content"><P>Since the last ligation attempts were not successful the plasmid pMAD is digested by us instead using the digested pMAD from Florian. The PCR-fragments of the hag-flank-construct were purified (PCR1: c = 62 ng/µL, PCR2: c = 55 ng/µL, PCR3: c = 46 ng/µL) and digested with the enzymes <I>Bam</I>HI and <I>Nco</I>I.</P><TABLE width="100%" border="1"><TBODY><TR><TH scope="col">[µL]</TH><TH scope="col">PCR 1</TH><TH scope="col">PCR 2</TH><TH scope="col">PCR 3</TH><TH scope="col">PCR 4</TH></TR><TR><TH scope="row">DNA</TH><TD>all</TD><TD>all</TD><TD>all</TD><TD>5 (300 ng/µL)</TD></TR><TR><TH scope="row"><I>Bam</I>HI</TH><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row"><I>NcoI</I></TH><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH scope="row">Cutsmart Buffer</TH><TD>4</TD><TD>4</TD><TD>4</TD><TD>2</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O</TH><TD>4</TD><TD>4</TD><TD>4</TD><TD>2</TD></TR><TR><TH scope="row">Total Volume</TH><TD>40</TD><TD>40</TD><TD>40</TD><TD>20</TD></TR></TBODY></TABLE><P>The restriction was  incubated at 37 °C for 1 h and stored at -20 °C.</P></DIV></FIELDSET><FIELDSET class="exp14"><LEGEND><A name="exp14.17">14.17 Test Expression</A></LEGEND><DIV class="aim"><P>Aim: Test the expression of PheA-Arc1p-C-4x</P></DIV><DIV class="exp-content"><P>Using the preculture from 14.16 60 mL of LB-Kan<SUP>50</SUP> medium were inoculated with 600 µL of the preculture and incubated at 37 °C and 220 rpm until OD<SUB>600</SUB> reached 0.5. The culture was cooled down to 18 °C and the protein production was induced with 12 µL of 500 mM IPTG. Over night the culture was incubated at 18 °C and 220 rpm. A glycerolstock of the strain containing the PheA-Arc1p-C-4x-pET28a plasmid was stored at -80 °C.</P></DIV></FIELDSET></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Marburg:Project:Notebook:April">http://2014.igem.org/Team:Marburg:Project:Notebook:April</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Marburg:Project:Notebook:April" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Marburg:Project:Notebook:April" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Marburg:Project:Notebook:April&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Marburg:Project:Notebook:April&amp;oldid=254981" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>