Team:WPI Worcester/Background
ICEBERG
MENU
HIDE
Home
Project Plans
Description
Background
Design
Safety
Results
All Results
iGEM Parts
Improvements
Demonstration
Hardware
Plant Synthetic Biology
Lab Work
Protocols
Notebooks
InterLab
People
Human Practices
Public Engagement
Collaborations
Attributions
About Us
Background Information
Our biofilm experiments entailed the usage of different biologics, each possessing their own mechanism of production in vivo and chemistry. Here we summarize information from the scientific literature necessary for our experimental setups and provide a context in which our results may lie.
Foodborne Illness
Foodborne illness is an umbrella term for a plethora of infections of multiple of pathogens. A common bacterium implicated in its cause is E. Coli O157:H7, which can make someone ill in as little as 24 hours and last for over a week ("Center"). A viral source is hepatitis A which can take a couple of weeks to onset and cause several months of infection ("Center").  The hallmark symptoms are vomiting, fever, cramps, lowered appetite, and nausea. Depending on the strain one is infected with, the onset time and treatment options may vary ("Center"). Over 250 infection types currently are known ("Food").
Antifreeze Proteins
Antifreeze proteins exist in several shapes and arrangements. Found in many different organisms subjected to freezing weather, these proteins appear to share a water-binding amino acid sequence commonly comprised of threonine, or of another similarly polar/charged amino acid ("Exploring"). The protein achieves its ice-fighting ability from apparently interrupting the networking process of hydrogen bonding mandated for ice formation (Sinha, 2016). Four broad variants exist, with a fifth, called the AFGP, possessing a sugar moiety (Harding, 2003):
Curcumin
Antibiotic and Chemical Nature
Curcumin, the active ingredient of the spice turmeric, originates from the rhizome Curcuma longa. The compound possesses a yellow hue, and has been used in alternative medicine as an anti-inflammatory and antioxidant, and has recently been explored for potentially possessing an antibacterial ability (Tyagi, 2015).
Curcumin apparently achieves its antibacterial nature from lysing the cell membranes of both Gram-positive and Gram-negative bacteria. Though the composition of bacterial types differ, this compound perforates bacterial membranes to allow the contents of each cell type to eject into the extracellular space, thereby killing the cells in the process. Specifically, by virtue of possessing an amphipathic chemistry, curcumin can selectively insert into lipid bilayers and disrupt the native structure (Tyagi, 2015).
As biofilms develop through quorum sensing mechanisms (the communication pathways between bacteria that selectively activates gene transcription via signaling molecules), the breakdown of cellular membranes does away the entryways of such molecules (Solano,2014). Therefore, if bacteria survive initial membrane perforation, they lose the ability to accurately secrete and absorb signaling molecules to further biofilm growth (“What is QS?”).
In terms of chemical structure, curcumin consists of a heptane backbone with phenyl groups at its terminal points. Moreover, two carbonyl groups, two methoxy groups, and two hydroxyl confer mild polarity to the molecule. The larger proportion of the aliphatic components make the substance mostly soluble in hydrophobic substances like oils and DMSO and amphyllic solvents like ethanol, and hardly soluble in water (“Curcumin”). Here is curcumin’s structure (“Curcumin”):
Synthetic Biology Approach
Some studies have successfully subcloned bacteria to produce curcumin and other related flavonoids in vitro. Current literature mainly describes the genetic modification of bacterial models, however, a logical extension would be to subclone genes encoding enzymes comprising the curcumin production pathway into plant genomes. Two exemplar methods of curcumin production were found in our research, utilizing E. coli as a model. Both ways utilize restriction digestion for gene insertion.  Briefly, the first method, devised by Katsuyama et al., involved the combined/staggered usage of three enzymes in vitro, all activated by a T7 promoter with IPTG on the plasmids containing the enzymatic genes: phenylalanine ammonia-lyase (PAL), 4-coumarate:CoA ligase (4CL),  acetyl-CoA carboxylase (AC), and curcuminoid synthase (CUS), with reactions based on components from the larger and common phenylpropanoid pathways (Katsuyama, 2008). Each of these aforementioned enzymes can accept multiple chemical species to produce several flavonoids, but to attain curcumin, ferulic acid must be presented to the bacteria expressing these enzymes (Katsuyama, 2008). While tyrosine or phenylalanine could initiate the reactions, carboxylic acid (here, ferulic acid) subverts the need for the inevitable first step of amino acid-conversion by PAL (Katsuyama, 2008). The CL enzyme then takes ferulic acid and supplants a hydroxyl group for a CoA group to form feruloyl-CoA, which two molecules of it are then reacted with malonyl-CoA by CUS to make curumin with carbon dioxide as waste (Katsuyama, 2008). The plasmids used in this study are as follows (Katsuyama, 2008):
The second method, also by the Katsuyama group, is described in a later paper that serves as a refinement of the above procedure. The basis of the paper was to explore other CUS variants, with the CUS mentioned originating in rice (Katsuyama, 2009). To summarize, the paper outlines the discernment of two similar enzymes,  diketide-CoA synthase (DCS) and curcumin synthase 1 (CURS1), both in the curcumina longa plant,  that belong to an enzyme class labeled type III polyketide synthases (to which CUS is a member) (Katsuyama, 2009). The authors found that similar enzymes in the plant, CURS2 and CURS3, also produced curcumin in the plant’s roots (Katsuyama, 2009). To describe their methodology briefly, the authors used a pUC19 plasmid to subclone the cDNA genes for CURS2 and CURS3 with  pET16b plasmid as the receiving vector for the translation of the genes with His tags for analysis with affinity chromatography using nickel ions (Katsuyama, 2009). The overall pathway from the second paper is shown below with DCS and CURS responsible for steps (3) and step (4), respectively (Machado, 2014):
Overall chemical reactions describing the second method are as follows (Machado, 2014):
References
Center for Food Safety and Applied Nutrition. (n.d.). Foodborne Illnesses: What You Need to Know. Retrieved from https://www.fda.gov/food/foodborneillnesscontaminants/foodborneillnessesneedtoknow/default.htm
Curcumin. (n.d.). Retrieved from https://pubchem.ncbi.nlm.nih.gov/compound/curcumin#section=Mass-Spectrometry
Exploring antifreeze proteins. (n.d.). Retrieved from https://bigpictureeducation.com/exploring-antifreeze-proteins
Food Safety. (2018, February 16). Retrieved from https://www.cdc.gov/foodsafety/foodborne-germs.html
Katsuyama, Y., Matsuzawa, M., Funa, N., & Horinouchi, S. (2008). Production of curcuminoids by Escherichia coli carrying an artificial biosynthesis pathway. Microbiology,154(9), 2620-2628. doi:10.1099/mic.0.2008/018721-0
Katsuyama, Y., Kita, T., & Horinouchi, S. (2009). Identification and characterization of multiple curcumin synthases from the herb Curcuma longa. FEBS Letters,583(17), 2799-2803. doi:10.1016/j.febslet.2009.07.029
Harding, M. M., Anderberg, P. I., & Haymet, A. D. (2003). Antifreeze glycoproteins from polar fish. European Journal of Biochemistry, 270(7), 1381-1392. doi:10.1046/j.1432-1033.2003.03488.x
Machado, D., Rodrigues, L. R., & Rocha, I. (2014). A kinetic model for curcumin production in Escherichia coli. Biosystems,125, 16-21. doi:10.1016/j.biosystems.2014.09.001
Sinha, P., Muralidharan, S., Sengupta, S., & Veerappapillai, S. (2016). A Brief Review on Antifreeze Proteins: Structure, Function and Applications. [Abstract]. Research Journal of Pharmaceutical, Biological and Chemical Sciences, 7(3), 914-919. Retrieved October 16, 2018, from https://www.researchgate.net/publication/304935678_A_brief_review_on_antifreeze_proteins_Structure_function_and_applications.
Solano, C., Echeverz, M., & Lasa, I. (2014). Biofilm dispersion and quorum sensing. Current Opinion in Microbiology,18, 96-104. doi:10.1016/j.mib.2014.02.008
Tyagi, P., Singh, M., Kumari, H., Kumari, A., & Mukhopadhyay, K. (2015). Bactericidal Activity of Curcumin I Is Associated with Damaging of Bacterial Membrane. Plos One,10(3). doi:10.1371/journal.pone.0121313
What is QS? (n.d.). Retrieved October 3, 2018, from https://www.nottingham.ac.uk/quorum/what.html
