<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Toronto_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Toronto/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="row navigation bg-dark-cyan"><DIV class="row" id="navigation"><UL><LI><SPAN class="text-white">• </SPAN><A href="https://2017.igem.org/Team:Toronto" id="grey-hover" class="text-white">Home</A></LI><LI><SPAN class="text-grey">• </SPAN><A href="https://2017.igem.org/Team:Toronto/Team" id="grey-hover" class="text-white">Team</A></LI><LI><SPAN class="text-red">• </SPAN><A href="https://2017.igem.org/Team:Toronto/Wetlab" id="red-hover" class="text-white">Wetlab</A></LI><LI><SPAN class="text-yellow">• </SPAN><A href="https://2017.igem.org/Team:Toronto/Drylab" id="yellow-hover" class="text-white">Drylab</A></LI><LI><SPAN class="text-cyan">• </SPAN><A href="https://2017.igem.org/Team:Toronto/Human_Practices" id="cyan-hover" class="text-white">Human Practices</A></LI></UL></DIV></DIV><DIV class="section"><DIV class="container header" id="dark-red"><H1>Notebook</H1></DIV></DIV><DIV class="section bg-white"><DIV class="container content-page row"><DIV class="block content"><DIV id="prep" class="subsection"><H2 class="text-red">Preparing stocks, plates, cell cultures, etc.</H2><H3>Week 1, day 1 (26/06)</H3><UL class="bullets"><LI>Prepared wash bottles of 70% Ethanol</LI><LI>Made 4x500mL LB media
</LI><LI>2 with agar for plating</LI><LI>2 without agar for preparing overnight cultures</LI></UL><LI>Made chloramphenicol (CAM) stocks</LI><LI>Plated out LB + agar onto sterile plates with CAM</LI><H3>Week 1, day 2 (27/06)</H3><UL class="bullets"><LI>Made 10mL 0.2M acetic acid/1M NaOH</LI><LI>Prepared 50mL of RF1/RF2 buffer (to make RbCl competent cells)</LI><LI>Streaked 4 plates (stored in 4°C):
</LI><LI>MG1655ΔLacI onto 2 LB and 1 LB+CAM plates</LI><LI>DH10β onto 1 LB plate</LI></UL><H3>Week 1, day 3 (28/06)</H3><P>Streaked 2 LB plates: MG1655ΔLacI &amp; DH10β to replace Tuesday’s plates (because they over-grew)</P><UL class="bullets"><LI>New plates put into 37°C incubator overnight, backup plates placed into 4°C fridge</LI></UL><P>Prepared 5mL LB cultures and placed in overnight shaker of:</P><UL class="bullets"><LI>MG1655ΔLacI</LI><LI>DH10β</LI></UL><H3>Week 1, day 4 (29/06)</H3><UL class="bullets"><LI>Prepared SOC medium</LI><LI>Cultured DH10β cells in baffled flasks until OD reached 0.4-0.55</LI></UL><H3>Week 1, day 5 (30/06)</H3><UL class="bullets"><LI>Plated DH10β cells and incubated at 37°C overnight</LI></UL><H3>Week 1, day 6 (31/06)</H3><UL class="bullets"><LI>Checked on plated cells from 30/06:</LI></UL><H3>Week 3, day 1 (10/07)</H3><UL class="bullets"><LI>Made and autoclaved LB and LB+CAM liquid.</LI><LI>Poured 13 LB plates.
</LI><LI>LB+CAM plates were unsuccessful since incorrect amount of CAM was added (12.5g instead of 12.5mg).</LI></UL><H3>Week 3, day 2 (11/07)</H3><UL class="bullets"><LI>Made LB+CAM plates</LI><LI>Plated and spread 50µL and 100µL DH5α cells onto LB agar plates using sterile loops.
</LI><LI>Placed in overnight incubator</LI></UL><H3>Week 3, day 3 (12/07)</H3><P>-Checked on DH5α plates from the incubator.</P><H3>Week 3, day 4 (13/07)</H3><UL class="bullets"><LI>KAN and CAM antibiotic stocks</LI><LI>500mL each of LB+CAM and LB+KAN</LI><LI>500mL 1X TAE buffer from 50X TAE buffer and MilliQ water</LI><LI>Prepared overnight culture of one DH5α colony in 5mL LB at 37°C.</LI></UL><H3>Week 3, day 5 (14/07)</H3><UL class="bullets"><LI>Checked on DH5α culture
</LI><LI>Initial OD600 was 0.011, so waited 110min for optimal OD600</LI><LI>Moved culture to ice once OD600 reached 0.466</LI><LI>Made DH5α stocks stored in -80°C, competence to be tested on Monday using iGEM competence cell test kit.</LI></UL><H3>Week 4, day 1 (17/07)</H3><UL class="bullets"><LI>Plated transformed cells.
</LI><LI>Spread the DH5α cells onto 4 LB+CAM plates, each containing 100µL of the 100pg/µL, 50pg/µL, 10pg/µL, and 0pg/µL cells.</LI><LI>Spread the DH10β cells onto two LB+KAN plates, both containing 100µL of the control and either 100µL or 50µL of the pKDL071 cells</LI></UL><LI>Incubated the plates at 37°C overnight.</LI><H3>Week 4, day 2 (18/07)</H3><UL class="bullets"><LI>Made LB media</LI><LI>Made LB+CAM plates</LI><LI>Checked on 17/07 transformed cells and moved them to the 4°C fridge.
</LI><LI>Confirmed DH5α cells were competent</LI><LI>Note: LB+CAM plates made on 13/07 may have been made with too much CAM (1250µL CAM to 500mL LB). This may explain why there were so few colonies from 17/07 transformation of DH5α + RFP/pSB1C3.</LI></UL><UL class="bullets"><LI>Obtained single colonies from transformed DH5α and DH10β plates from 17/07 and made overnight 37°C colonies.
</LI><LI>The DH5α was placed in the 5mL of LB+CAM (5µL of stock CAM added to 5mL LB).</LI><LI>The DH10β was placed in 5mL of LB+KAN (25µL of stock KAN added to 5mL of LB).</LI></UL><H3>Week 4, day 3 (19/07)</H3><UL class="bullets"><LI>Examined plates from 18/07
</LI><LI>Most plates had no cells likely due to the high CAM concentration (2.5x working concentration).</LI></UL><UL class="bullets"><LI>Made glycerol stocks of DH10β with pKDL071</LI></UL><H3>Week 4, day 5 (21/07)</H3><UL class="bullets"><LI>Made LB+AMP plates</LI></UL><H3>Week 5, day 1 (24/07)</H3><UL class="bullets"><LI>Made LB+CAM plates</LI></UL><H3>Week 5, day 3 (26/07)</H3><UL class="bullets"><LI>Made LB+CAM and LB+KAN solutions for overnight cultures</LI></UL><H3>Week 5, day 4 (26/07)</H3><UL class="bullets"><LI>Made 17 LB+KAN plates</LI></UL><H3>Week 5, day 5 (28/07)</H3><UL class="bullets"><LI>Prepared 10X PBS buffer</LI></UL><H3>Week 6, day 1 (31/07)</H3><UL class="bullets"><LI>Made 50mL non-sterilized 1X PBS Buffer</LI></UL><H3>Week 6, day 3 (02/08)</H3><UL class="bullets"><LI>Made LB+CAM plates</LI></UL><H3>Week 6, day 5 (04/08)</H3><UL class="bullets"><LI>Made LB liquid media</LI></UL><H3>Week 7, day 4 (10/08)</H3><UL class="bullets"><LI>Made 17 LB+Kan plates</LI></UL><H3>Week 9, day 2 (22/08)</H3><UL class="bullets"><LI>Made LB+KAN plates</LI></UL><H3>Week 9, day 3 (23/08)</H3><UL class="bullets"><LI>Made LB+AMP plates</LI></UL><H3>Week 9, day 4 (24/08)</H3><UL class="bullets"><LI>Made 5mL LB+Kan in falcon tube</LI></UL><H3>Week 10, day 4 (01/09)</H3><UL class="bullets"><LI>Made LB+CAM plates</LI></UL><H3>Week 16, day 5 (06/10)</H3><UL class="bullets"><LI>Made LB+AMP plates</LI></UL><H3>Week 18, day 2 (17/10)</H3><UL class="bullets"><LI>Made LB+KAN and LB+CAM plates</LI></UL><H3>Week 19, day 7 (29/10)</H3><UL class="bullets"><LI>Made 20% stock D-glucose solution by adding 20g of glucose to 100ml MilliQ water then autoclaved</LI><LI>Made Modified M9 Media (+0.2% glucose + 0.2% arabinose)</LI></UL></DIV><DIV id="interlab_152" class="subsection"><H2 class="text-red">Interlab</H2><H3>Week 3, day 4 (13/07)</H3><UL class="bullets"><LI>Prepared overnight culture of one DH5α colony in 5mL LB at 37°C</LI></UL><H3>Week 3, day 5 (14/07)</H3><UL class="bullets"><LI>Checked on DH5α culture
</LI><LI>Initial OD600 was 0.011, so waited 110min for optimal OD600</LI><LI>Moved culture to ice once OD600 reached 0.466</LI><LI>Made DH5α stocks stored in -80°C, competence to be tested on Monday using iGEM competence cell test kit.</LI></UL><H3>Week 4, day 1 (17/07)</H3><UL class="bullets"><LI>Transformed DH5α cells using iGEM competence cell test kit with RFP/pSB1C3 at concentrations 100pg/µL, 50pg/µL, 10pg/µL, and 0pg/µL (control)</LI><LI>Plated the cells onto 4 LB+CAM plates, each containing 100µL of the 100pg/µL, 50pg/µL, 10pg/µL, and 0pg/µL cells.</LI><LI>Incubated the plates at 37°C overnight.</LI></UL><H3>Week 4, day 2 (18/07)</H3><UL class="bullets"><LI>Checked on transformed cells from previous day and moved them to the 4°C fridge.
</LI><LI>Confirmed DH5α cells were competent</LI><LI>Note: LB+CAM plates made on 13/07 may have been made with too much CAM (1250µL CAM to 500mL LB). This may explain why there were so few colonies from 17/07 transformation of DH5α + RFP/pSB1C3.</LI></UL><UL class="bullets"><LI>Resuspended the iGEM kit parts containing the DNA required for the Interlab study (DNA was obtained from kit plate #6, which was then moved to the 4°C fridge for storage) and transformed the chemically competent DH5α cells (from 14/07) with resuspended DNA
</LI><LI>Made 9 tubes: 8 GFP samples and one no plasmid control (labelled 20B, 20D, 20F, 20H, 20J,  20L, 20N, and 20P, respectively).</LI><LI>Originally, the goal was to make 17 tubes for two transformations (8 samples x 2 + 1 no plasmid control), however, there were not enough cells to to make a second set (missing second set of 20L and 20B).</LI><LI>Spread 100µL of transformed bacteria on LB+CAM plates to incubate and dry overnight at 37°C.</LI></UL><H3>Week 4, day 3 (19/07)</H3><UL class="bullets"><LI>Examined plates from yesterday
</LI><LI>Most plates had no cells likely due to the high CAM concentration (2.5x working concentration).</LI></UL><P>07-19 plates</P><UL class="bullets"><LI>Re-plated DH5α cells transformed with Interlab study plasmids from transformation tubes done on Tuesday and incubated overnight in 37°C shaker.</LI></UL><H3>Week 4, day 4 (20/07)</H3><UL class="bullets"><LI>Obtained 19/07 GFP plates from the 37°C shaker.</LI><LI>Picked a single colony from each plate and incubated in 1.5mL LB media overnight in 37°C shaker and left to grow in the 37 degree C shaker.</LI><LI>In total, there were 8 sample tubes containing the GFP interlab samples and one control tube of just LB.</LI><LI>Made 5mL cultures of the DH5α cells transformed with GFP plasmids and placed them in the overnight shaker to grow.</LI></UL><H3>Week 4, day 5 (21/07)</H3><UL class="bullets"><LI>Plated transfromed DH5a</LI></UL><H3>Week 4, day 6 (22/07)</H3><UL class="bullets"><LI>Checked plates of DH5a cells transformed with Interlab plasmids (from 21/07).</LI><LI>Colonies grew into a lawn of bacteria and no single colonies could be isolated.</LI></UL><H3>Week 5, day 2 (25/07)</H3><UL class="bullets"><LI>Obtained single colonies from the GFP sample plates (21/07) and plated them on LB+CAM plates. The plates were left at 37°C to grow overnight.</LI></UL><H3>Week 5, day 3 (26/07)</H3><UL class="bullets"><LI>Examined LB+CAM plates and saw single colonies</LI></UL><UL class="bullets"><LI>Inoculated single colonies from plates into LB+KAN and LB+CAM and put in incubator</LI></UL><H3>Week 5, day 4 (26/07)</H3><UL class="bullets"><LI>Made 80x1mL glycerol stocks of 26/07 cultures and stored in -80°C freezer</LI></UL><H3>Week 5, day 5 (28/07)</H3><UL class="bullets"><LI>Ran LUDOX-S40 calibration</LI></UL><P>Table 1: LUDOX Calibration</P><TABLE class="table table-striped table-bordered"><THEAD><TR><TH>100% LUDOX</TH><TH>H&lt;sub&gt;2&lt;/sub&gt;O</TH></TR></THEAD><TBODY><TR><TD>0.0508</TD><TD>0.0402</TD></TR><TR><TD>0.0561</TD><TD>0.0415</TD></TR><TR><TD>0.0525</TD><TD>0.0419</TD></TR><TR><TD>0.0527</TD><TD>0.0435</TD></TR></TBODY></TABLE><H3>Week 6, day 1 (31/07)</H3><UL class="bullets"><LI>Transferred samples 20J, 20L, 20N, and 20P from glycerol aliquots (from 26/07) to a 96-well plate and measured maximum emission and exciation wavelengths of GFP and YFP.
</LI><LI>GFP: Literature said the maximum exictation wavelength is 395 nm, the maximum emission wavelength is 509 nm
</LI><LI>We measured maximum exication: 390 nm, maximum emission: 400-750 nm.</LI></UL><LI>YFP: Literature said the maximum excitation wavelength is 512 nm, the maximum emission wavelength is 529 nm.</LI><LI><EM>Note: No negative control was measured so data can’t be normalized.</EM></LI><H3>Week 6, day 2 (01/08)</H3><UL class="bullets"><LI>Measured the emission spectra of GFP samples 20B, 20D, and 20F with the plate reader</LI><LI>The interlab GFP samples use the mut3b GFP, which has a shifted excitation and emission spectra
</LI><LI>Excitation range of 488 nm - 501 nm</LI><LI>Emission maximum at 511 nm</LI><LI>At a 395 nm excitation wavelength, this GFP mutant emits 0.2x the intensity of wild type GFP. However, at a excitation wavelength of 488 nm, the emission intensity is 21x that of wild type GFP.</LI></UL><LI>The emission spectra was tested using samples 20B (positive control), 20D (negative control), and 20F from glycerol aliquots.
</LI><LI>In total, three different excitation spectra were tested: 488 nm, 508 nm, and 495 nm.</LI><H3>Week 6, day 3 (02/08)</H3><UL class="bullets"><LI>Washed glycerol stocks and then suspended in LB+CAM to ensure consistent measurements from the plate reader
</LI><LI>All subsequent interlab samples will be measured in LB+CAM</LI></UL><LI>Restreaked samples to obtain biological replicates</LI><LI>Measured excitation spectra using plate reader in WB 316 of GFP samples 20B, 20D, and 20F.</LI><H3>Week 6, day 5 (04/08)</H3><UL class="bullets"><LI>Restreaked 20D transfomed cell</LI></UL><UL class="bullets"><LI>Took plate reader measurements according to Interlab protocol
</LI><LI>Used LUDOX 100% and H2O to set the standard point</LI><LI>Found fluorescein fluorescence standard curve</LI><LI>Took measurements of all cell samples</LI></UL></DIV><DIV id="LacILOV" class="subsection"><H2 class="text-red">LacILOV</H2><H3>Week 1, day 5 (30/06)</H3><UL class="bullets"><LI>Bacterial transformation of competent cells with RFP plasmid</LI></UL><H3>Week 2, day 5 (07/07)</H3><UL class="bullets"><LI>Electroporation of MG1655ΔLacI electrocompetent cells.</LI><LI>Modified electroporation protocol to incubate transformed cells at 300rpm, 37°C after SOC recovery step.</LI><LI>Bacterial transformation of electroporated cells with RFP plasmid (BBa_j04450 in pSB1C3) to check transformation efficiency at 0, 10, 50 and 100pg/ul.</LI><LI>MiniPrepped transformed DH10β (BBa_j04450 in pSB1C3) to purify plasmid (sample 1/2 stored in -20°C)</LI><LI>Performed RE digestion, nicked with EcoRI-HF (sample 1/2 stored in -20°C)</LI></UL><H3>Week 3, day 1 (10/07)</H3><UL class="bullets"><LI>Miniprepped RFP plasmids on the pSB1C3 backbone from chemically competent DH10β cells from 07/07.</LI><LI>Conducted gel electrophoresis of RE digested and non RE digested dupicate samples from 07/07.
</LI><LI>10µL of each ladder was used, but in the future we should experiment with adding less ladder (conserve reagents!).</LI><LI>Each 10µg sample was mixed with 2µL loading dye and loaded on the gel along with two ladders. The sequences are as follows:
</LI><LI>EcoRI-HF Digested Sample 1 (D1)</LI><LI>Undigested Sample 1 (UD1)</LI><LI>1kb DNA ladder for measurement</LI><LI>EcoRI-HF Digested Sample 2 (D2)</LI><LI>Undigested Sample 2 (UD2)</LI><LI>1kb DNA ladder for measurement</LI></UL><LI>Demonstration: Nanodropped each sample to show the protocol to general members.</LI><H3>Week 3, day 2 (11/07)</H3><P>Calculated optimization for Q5 (High Fidelity) PCR of UNS3/2 into ends of  pSB1C3 backbone</P><UL class="bullets"><LI>Optimized annealing step because our primer Tms are high (85°C), so using different additives/temperature gradients to optimize yield</LI></UL><P>“07-11 Tm Calculator.png”</P><P>Table 1: PCR Optimization</P><TABLE class="table table-striped table-bordered"><THEAD><TR><TH>Component</TH><TH>No treatment (µL)</TH><TH>3% DMSO (µL)</TH><TH>6% DMSO (µL)</TH><TH>1X GC Buffer (µL)</TH><TH>No DNA control (µL)</TH></TR></THEAD><TBODY><TR><TD>5X Q5 Reaction Buffer</TD><TD>5</TD><TD>5</TD><TD>5</TD><TD>5</TD><TD>5</TD></TR><TR><TD>10mM dNTPs</TD><TD>0.5</TD><TD>0.5</TD><TD>0.5</TD><TD>0.5</TD><TD>0.5</TD></TR><TR><TD>10µM Forward primers</TD><TD>1.25</TD><TD>1.25</TD><TD>1.25</TD><TD>1.25</TD><TD>1.25</TD></TR><TR><TD>10µM Reverse primers</TD><TD>1.25</TD><TD>1.25</TD><TD>1.25</TD><TD>1.25</TD><TD>1.25</TD></TR><TR><TD>1ng/µL Template DNA (prediluted)</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>0</TD></TR><TR><TD>Q5 HF DNA Polymerase</TD><TD>0.25</TD><TD>0.25</TD><TD>0.25</TD><TD>0.25</TD><TD>0.25</TD></TR><TR><TD>1X GC Buffer</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>5</TD><TD>5</TD></TR><TR><TD>DMSO</TD><TD>0</TD><TD>0.75</TD><TD>1.5</TD><TD>0</TD><TD>0</TD></TR><TR><TD>Nuclease-free water</TD><TD>15.75</TD><TD>15</TD><TD>14.25</TD><TD>10.75</TD><TD>11.75</TD></TR><TR><TD>Total volume</TD><TD>25</TD><TD>25</TD><TD>25</TD><TD>25</TD><TD>25</TD></TR></TBODY></TABLE><P>* Multiply five for 5 reactions (5 different annealing temperatures: 72, 71, 70, 69, and 68°C).</P><P>Table 2: Master Mixes</P><TABLE class="table table-striped table-bordered"><THEAD><TR><TH>Master Mix</TH><TH>No treatment (µL)</TH><TH>3% DMSO (µL)</TH><TH>6% DMSO (µL)</TH><TH>1X GC Buffer (µL)</TH><TH>No DNA control (µL)</TH></TR></THEAD><TBODY><TR><TD>GC Buffer</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>40</TD><TD>40</TD></TR><TR><TD>DMSO</TD><TD>0</TD><TD>6</TD><TD>12</TD><TD>0</TD><TD>0</TD></TR><TR><TD>Nuclease-free water</TD><TD>126</TD><TD>120</TD><TD>114</TD><TD>86</TD><TD>94</TD></TR><TR><TD>Sum</TD><TD>126</TD><TD>126</TD><TD>126</TD><TD>126</TD><TD>134</TD></TR></TBODY></TABLE><H3>Week 3, day 3 (12/07)</H3><P>Performed Q5 HF PCR (discussed 11/07) with modifications:</P><UL class="bullets"><LI>Revised PCR calculations based on Christian’s suggestions: 7 Q5 HF PCR 50µL reactions with 3% DMSO in each sample (one control tube with no DNA template).</LI></UL><P>Table 1: Revised PCR Calculations</P><TABLE class="table table-striped table-bordered"><THEAD><TR><TH>Component</TH><TH>Reaction (×7)</TH><TH>Control</TH></TR></THEAD><TBODY><TR><TD>5X Q5 Reaction Buffer</TD><TD>10</TD><TD>10</TD></TR><TR><TD>10mM dNTPs</TD><TD>1</TD><TD>1</TD></TR><TR><TD>10µM Forward primers</TD><TD>2.5</TD><TD>2.5</TD></TR><TR><TD>10µM Reverse primers</TD><TD>2.5</TD><TD>2.5</TD></TR><TR><TD>1ng/µL Template DNA (prediluted)</TD><TD>1</TD><TD>0</TD></TR><TR><TD>Q5 HF DNA Polymerase</TD><TD>0.5</TD><TD>0.5</TD></TR><TR><TD>1X GC Buffer</TD><TD>10</TD><TD>10</TD></TR><TR><TD>100% DMSO</TD><TD>1.5</TD><TD>1.5</TD></TR><TR><TD>Nuclease-free water</TD><TD>31</TD><TD>32</TD></TR></TBODY></TABLE><UL class="bullets"><LI>Prepared master mix for Q5 HF PCR reactions (with 50µL 3% DMSO). Template DNA is included in 6 tubes with 1 control without template DNA.
</LI><LI>One template DNA tube only contains 25 uL, possibly due to pippetting error
</LI><LI>Note: prepare a master mix of 8 reactions of 50 μL, instead of 7.1 reactions, to accommodate pippeting errors).</LI></UL><LI>Ran PCR. Tubes were meant to be optimized through a temperature gradient, but were placed incorrectly in the PCR machine, so they were run all at the same temperature.</LI><LI>Ran a gel electrophoresis of psB1C3 backbone PCR. The gel was not successful. (Life: What a cruel mistress)
</LI><LI>Note: Gel was made with 15 wells - too small for 10µl samples (15 wells should use ~6-8µl).</LI><H3>Week 3, day 4 (13/07)</H3><P>Performed Q5 HF PCR to amplify pSB1C3 backbone</P><UL class="bullets"><LI>Prepared same master master mixes as calculated 12/07 for 6 samples with template DNA and one control without template DNA</LI><LI>Ran the PCR of samples 1-6 at different temperatures, one each at 72.0, 71.4, 70.1, 68.3, 66, and 64.3°C respectively
</LI><LI>Control was tested at 70.1°C</LI></UL><LI>Performed gel electrophoresis of Q5 PCR samples, along with EcoRI digested pSB1C3 with RFP control</LI><UL class="bullets"><LI>Advisor suggested results may be due to primer dimerization, possibly because of high primer Tm</LI><LI>Planned to redesign primers to reduce this problem by removing base pairs from the 3’ binding up to ~15bp homology between primers and backbone
</LI><LI>Aiming for Tm of ~68°C</LI></UL><H3>Week 3, day 5 (14/07)</H3><P>Redesigned primers based on suggestions from 13/07 to have lower Tm</P><UL class="bullets"><LI>New reverse primer for pKDL071 backbone</LI><LI>New forward and reverse primers for pSB1C3 backbone</LI></UL><P>The Q5 polymerase used in previous PCRs is very bubbly which could have been a source of error, so instead we performed 2-step NEB Phusion HF PCR of pSB1C3 with RFP construct</P><UL class="bullets"><LI>Used primers from 2016 and 2017 to compare results</LI><LI>Tubes were labelled 16.1-16.5 (for 2016 primers) and 17.1-17.5 (for 2017 primers)</LI><LI>Controls without DNA template were labelled NTC (no template control)</LI><LI>Ran the samples on a temperature gradient to determine the optimal PCR settings</LI></UL><P>Table 1: Phusion PCR Reaction Component Table (for total volume of 50µL per tube)</P><P>| Component | 16.NTC | 16.1 | 16.2 | 16.3 | 16.4 | 16.5 | 17.NTC | 17.1 | 17.2 | 17.3 | 17.4 | 17.5 |
| — | — | — | — | — | — | — | — | — | — | — | — |
| 5X Phusion HF or GC Buffer | 10µL | 10µL | 10µL | 10µL | 10µL | 10µL | 10µL | 10µL | 10µL | 10µL | 10µL | 10µL |
| 10mM dNTPs | 1µL | 1µL | 1µL | 1µL | 1µL | 1µL | 1µL | 1µL | 1µL | 1µL | 1µL | 1µL |
| 10µM 2016 Forward Primer | 2.5µL | 2.5µL | 2.5µL | 2.5µL | 2.5µL | 2.5µL | 0µL | 0µL | 0µL | 0µL | 0µL | 0µL |
| 10µM 2016 Reverse Primer | 2.5µL | 2.5µL | 2.5µL | 2.5µL | 2.5µL | 2.5µL | 0µL | 0µL | 0µL | 0µL | 0µL | 0µL |
| 10µM 2017 Forward Primer | 0µL | 0µL | 0µL | 0µL | 0µL | 0µL | 2.5µL | 2.5µL | 2.5µL | 2.5µL | 2.5µL | 2.5µL |
| 10µM 2017 Reverse Primer | 0µL | 0µL | 0µL | 0µL | 0µL | 0µL | 2.5µL | 2.5µL | 2.5µL | 2.5µL | 2.5µL | 2.5µL |
| Template DNA | 0µL | 1µL | 1µL | 1µL | 1µL | 1µL | 0µL | 1µL | 1µL | 1µL | 1µL | 1µL |
| DMSO | 1.5µL | 1.5µL | 1.5µL | 1.5µL | 1.5µL | 1.5µL | 1.5µL | 1.5µL | 1.5µL | 1.5µL | 1.5µL | 1.5µL |
| Phusion DNA Polymerase | 0.5µL | 0.5µL | 0.5µL | 0.5µL | 0.5µL | 0.5µL | 0.5µL | 0.5µL | 0.5µL | 0.5µL | 0.5µL | 0.5µL |
| Nuclease-free water | 32µL | 31µL | 31µL | 31µL | 31µL | 31µL | 32µL | 31µL | 31µL | 31µL | 31µL | 31µL |</P><P>Table 2: Phusion PCR Temperature Gradient</P><TABLE class="table table-striped table-bordered"><THEAD><TR><TH>Temperature</TH><TH>Tubes</TH></TR></THEAD><TBODY><TR><TD>62°C</TD><TD>16.1, 17.1</TD></TR><TR><TD>63.7°C</TD><TD>16.2, 17.2</TD></TR><TR><TD>68°C</TD><TD>16.3, 16.NTC, 17.3, 17.NTC</TD></TR><TR><TD>70.5°C</TD><TD>16.4, 17.4</TD></TR><TR><TD>72°C</TD><TD>16.5, 17.5</TD></TR></TBODY></TABLE><UL class="bullets"><LI>Performed gel electrophoresis of Phusion PCR samples</LI></UL><P>“07-14 gel of Phusion PCR”</P><H3>Week 4, day 1 (17/07)</H3><UL class="bullets"><LI>Performed bacterial transformation
</LI><LI>Transformed DH5α cells using iGEM competence cell test kit with pSB1C3+RFP at concentrations 100pg/µL, 50pg/µL, 10pg/µL, and 0pg/µL (control)</LI><LI>Transformed DH10β cells with pKDL071 or no plasmid control</LI><LI>Plated transformed cells.
</LI><LI>Spread the DH5α cells onto 4 LB+CAM plates, each containing 100µL of the 100pg/µL, 50pg/µL, 10pg/µL, and 0pg/µL cells.</LI><LI>Spread the DH10β cells onto two LB+KAN plates, both containing 100µL of the control and either 100µL or 50µL of the pKDL071 cells</LI></UL><LI>Incubated the plates at 37°C overnight.</LI><H3>Week 4, day 2 (18/07)</H3><UL class="bullets"><LI>Checked on 17/07 transformed cells and moved them to the 4°C fridge.
</LI><LI>Confirmed DH5α cells were competent</LI><LI>Note: LB+CAM plates made on 13/07 may have been made with too much CAM (1250µL CAM to 500mL LB). This may explain why there were so few colonies from 17/07 transformation of DH5α + RFP/pSB1C3.</LI></UL><UL class="bullets"><LI>Obtained single colonies from DH5α and DH10β plates from 17/07 and made overnight 37°C colonies.
</LI><LI>The DH5α was placed in the 5mL of LB+CAM (5µL of stock CAM added to 5mL LB).</LI><LI>The DH10β was placed in 5mL of LB+KAN (25µL of stock KAN added to 5mL of LB).</LI></UL><H3>Week 4, day 3 (19/07)</H3><UL class="bullets"><LI>Examined plates from 18/07
</LI><LI>Most plates had no cells likely due to the high CAM concentration (2.5x working concentration).</LI></UL><UL class="bullets"><LI>Miniprepped DH10β and DH5α to extract plasmides (pKDL071 and pSB1C3 with RFP respectively) and did Nanodrop on the results</LI></UL><P>Table 1: Nanodrop results from Miniprep</P><TABLE class="table table-striped table-bordered"><THEAD><TR><TH>Plasmid</TH><TH>260/280</TH><TH>260/230</TH><TH>ng/µL</TH></TR></THEAD><TBODY><TR><TD>pSB1C3 #1</TD><TD>1.82</TD><TD>1.42</TD><TD>86.6</TD></TR><TR><TD>pSB1C3 #2</TD><TD>1.85</TD><TD>1.52</TD><TD>87.9</TD></TR><TR><TD>psB1C3 #3</TD><TD>1.79</TD><TD>0.99</TD><TD>136.6</TD></TR><TR><TD>pKDL071 #1</TD><TD>1.89</TD><TD>1.55</TD><TD>60.0</TD></TR><TR><TD>pKDL071 #2</TD><TD>1.78</TD><TD>1.25</TD><TD>66.0</TD></TR><TR><TD>pKDL071 #3</TD><TD>1.95</TD><TD>1.90</TD><TD>48.7</TD></TR></TBODY></TABLE><UL class="bullets"><LI>Performed Phusion PCR of pSB1C3 with RFP construct using psB1C3 #2 and the pKDL071 FWD/RVS primers
</LI><LI>Made 1ng/uL of pDKL071 #3 for PCR via following steps:</LI><UL class="bullets"><LI>2μL of pDKL071 #3 Miniprep was mixed with 95.4 μL of nuclease free water</LI><LI>Made 100μL aliquots of 100μM from pKDL071 FWD UNS3 primer (shipped 05/07) resuspended in 785μL nuclease free water</LI><LI>Made 100μL aliquots of 100μM from pKDL071 REV UNS2 primer (shipped 05/07) resuspended in 761μL nuclease free water</LI></UL><OL start="2"><LI>Made a Master mix without DNA template containing:</LI></OL><UL class="bullets"><LI>17.5μL of 10 uM FWD_UNS3 primer</LI><LI>17.5μL of 10 uM REV_UNS2 primer</LI><LI>70uL of 5x phusion buffer</LI><LI>7uL of 10mM dNTPS</LI><LI>10.5uL 100% DMSO ( to reach a final concentration of 3%)</LI><LI>3.5uL Phusion polymerase</LI></UL><OL start="3"><LI>Took 18uL was taken from this master mix added 32 uL of Nuclease free water for a final volume of 50 uL as the No Template Control (NTC)</LI><LI>Added DNA template to the Master Mix:</LI></OL><UL class="bullets"><LI>6 uL 1ng/uL Miniprepped pKDL071 #3</LI><LI>186 uL nuclease free water</LI></UL><OL start="5"><LI>Added 50uL of the Master Mix to 5 PCR tubes</LI><LI>Ran Phusion PCR on temperature gradient with samples 1-5 at 62, 63.7, 68, 71.3, and 72°C respectively</LI></OL><UL class="bullets"><LI>NTC was at 72°C</LI></UL><LI>Ran gel on 12μL samples of Phusion PCR results for 1 minute then left for 1 hour then run for 1 hour due to equipment issues.
</LI><LI>Timing issues due might be why the ladder is not as intense.</LI><LI>Sample 3 appears to be our optimal temperature @ 68°C</LI></UL><H3>Week 4, day 4 (20/07)</H3><P>Ran Phusion PCR of the pKDL071 plasmid with RFP using the pKDL071 plasmid from Miniprep #3 (19/07).</P><UL class="bullets"><LI>Used the pKDL071 FWD/RVS primers shipped on 05/07 (the first set of primers designed in 2017)</LI><LI>Made Master Mix without DNA template or primers (for 8 reactions)
</LI><LI>20 μL of 10 uM FWD_UNS3 primer</LI><LI>20 μL of 10 uM REV_UNS2 primer</LI><LI>80 uL of 5x phusion buffer</LI><LI>8 uL of 10mM dNTPS</LI><LI>12 uL 100% DMSO ( to reach a final concentration of 3%)</LI><LI>4 uL Phusion polymerase</LI></UL><LI>18 uL was taken from the master mix as the No Template Control (NTC) and mixed with 32 uL of Nuclease free water.</LI><LI>Added DNA template to Master Mix (for 7 reactions)
</LI><LI>7 uL of 1ng/uL miniprep#3 pKDL071</LI><LI>217 uL of nuclease free water</LI><LI>50 uL of the master mix were added to 5 PCR tubes</LI><LI>6 samples and one NTC run at 68°C, the optimal temperature determined on 17/07.</LI><P>Optimization Phusion PCRs of pSB1C3 and pKDL071 plasmids with new UNS primers</P><UL class="bullets"><LI>Made 5 samples and one NTC for PCR of the pSB1C3 plasmid using pSB1C3 miniprep#2 and the new forward and reverse UNS primers.
</LI><LI>Annealing temperature was 64°C so samples 1-5 at 72, 70.1, 66, 64.3, and 62°C respectively with NTC at 64.3°C</LI></UL><LI>Made 5 samples and one NTC for PCR of the pKDL071 plasmid using pKDL071 miniprep#3, the old forward, and new reverse UNS primers
</LI><LI>Annealing temperature was 62°C so samples 1-5 at 67, 65.1, 63.3, 61, and 57°C respectively with NTC at 63.3°C</LI><LI>Made Master mixes with DNA
</LI><LI>OLD_FWD and NEW_REV pDKL071 (7 reactions)
</LI><LI>17.5 μL of 10 uM FWD primer</LI><LI>17.5 μL of 10 uM REV primer</LI><LI>NEW_FWD_USN3 and NEW_REV_UNS2 pSB1C3  (7 reactions)
</LI><LI>17.5 μL of 10 uM FWD_UNS3 primer</LI><LI>17.5 μL of 10 uM REV_UNS2 primer</LI><LI>Made Master mix without DNA template or primers (for 15 reactions)
</LI><LI>150 uL of 5x phusion buffer</LI><LI>15 uL of 10mM dNTPS</LI><LI>22.5 uL 100% DMSO ( to reach a final concentration of 3%)</LI><LI>7.5 uL Phusion polymerase</LI><LI>Added 91 uL of Master mix without DNA to the pSB1C3 &amp; pDKL071 primer master mixes</LI><LI>18 uL was taken from each primer master mix as the No Template Control (NTC) and added to 32 uL of Nuclease free water to reach a final volume of 50 uL.</LI><LI>Added DNA template to primer mixes:
</LI><LI>pSB1C3 master with DNA template (for 6 reactions):
</LI><LI>5.5 uL of 1ng/uL miniprep pSB1C3</LI><LI>170.5 uL of nuclease free water</LI><LI>pDKL071 master with DNA template (for 6 reactions)
</LI><LI>5.5 uL of 1ng/uL miniprep pDKL071</LI><LI>170.5 uL of nuclease free water</LI><LI>50 uL of each master mix were added to 5 PCR tubes (10 tubes total + 2 NTCs)</LI><P>Performed gel electrophoresis all PCR results</P><UL class="bullets"><LI>8 wells for Phusion PCR of the pKDL071 plasmid</LI></UL><UL class="bullets"><LI>15 wells for optimization PCR of pSB1C3 and pKDL071 plasmids
</LI><LI>Wells 1-6 contained pSB1C3 samples 1-5 and NTC</LI><LI>Well 7 contained the 10kb ladder</LI><LI>Wells 8-13 contained the pKDL071 samples 1-5 and NTC</LI></UL><P>Made 5mL cultures of the DH5 alpha cells transformed with GFP plasmids and placed them in the overnight shaker to grow.</P><H3>Week 4, day 5 (21/07)</H3><P>Nanodrop PCR products of pKDL071 from 20/07</P><UL class="bullets"><LI>Note: NOT purified</LI></UL><P>Table 1: Nanodrop of pKDL071-UNS with old primers (at 68°C)</P><TABLE class="table table-striped table-bordered"><THEAD><TR><TH>Sample</TH><TH>ng/uL</TH><TH>260/280</TH><TH>260/230</TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>200.7</TD><TD>1.34</TD><TD>0.47</TD></TR><TR><TD>2</TD><TD>379.0</TD><TD>1.32</TD><TD>0.62</TD></TR><TR><TD>3</TD><TD>357.5</TD><TD>1.37</TD><TD>0.62</TD></TR><TR><TD>4</TD><TD>346.1</TD><TD>1.33</TD><TD>0.64</TD></TR><TR><TD>5</TD><TD>363.9</TD><TD>1.33</TD><TD>0.59</TD></TR><TR><TD>6</TD><TD>353.6</TD><TD>1.37</TD><TD>0.61</TD></TR><TR><TD>NTC</TD><TD>339.3</TD><TD>1.34</TD><TD>0.57</TD></TR></TBODY></TABLE><P>Table 2: Nanodrop of pKDL071-UNS optimization with new REV primer (at 62-72°C)</P><TABLE class="table table-striped table-bordered"><THEAD><TR><TH>Sample</TH><TH>ng/uL</TH><TH>260/280</TH><TH>260/230</TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>436.9</TD><TD>1.42</TD><TD>0.74</TD></TR><TR><TD>2</TD><TD>437.1</TD><TD>1.46</TD><TD>0.72</TD></TR><TR><TD>3</TD><TD>443.0</TD><TD>1.41</TD><TD>0.72</TD></TR><TR><TD>4</TD><TD>446.7</TD><TD>1.42</TD><TD>0.73</TD></TR><TR><TD>5</TD><TD>434.9</TD><TD>1.43</TD><TD>0.71</TD></TR><TR><TD>NTC</TD><TD>436.3</TD><TD>1.43</TD><TD>0.71</TD></TR></TBODY></TABLE><P>Dpn1 digest and purification of pKDL071 PCR product with new REV primer: 40uL digestion reaction each tube.</P><UL class="bullets"><LI>6 tubes in total, including 4 samples (D1-D4), 1 NTC and 1 positive control (miniprepped pKDL071)</LI><LI>Master Mix contained:
</LI><LI>32uL of pKDL071</LI><LI>4uL 10X CutSmart</LI><LI>4uL Dpn1</LI></UL><LI>Nanodropped the results</LI><P>Table 3: Nanodrop of the digested and purified pKDL071 PCR product with new REV primer</P><TABLE class="table table-striped table-bordered"><THEAD><TR><TH>Sample</TH><TH>ng/ul</TH><TH>260/280</TH><TH>260/230</TH></TR></THEAD><TBODY><TR><TD>D1</TD><TD>28.1</TD><TD>1.82</TD><TD>1.69</TD></TR><TR><TD>D2</TD><TD>27.9</TD><TD>1.84</TD><TD>1.84</TD></TR><TR><TD>D3</TD><TD>31.6</TD><TD>1.80</TD><TD>1.73</TD></TR><TR><TD>D4</TD><TD>30.5</TD><TD>1.94</TD><TD>1.84</TD></TR><TR><TD>NTC</TD><TD>18.3</TD><TD>1.79</TD><TD>1.43</TD></TR><TR><TD>Positive Control (minipreped pKDL071)</TD><TD>22.3</TD><TD>1.81</TD><TD>1.70</TD></TR></TBODY></TABLE><P>Performed Phusion PCR of pSB1C3 with new FWD UNS &amp; REV UNS with temperature gradient</P><UL class="bullets"><LI>Made Primer Master mix (MM1) new FWD UNS3 &amp; New REV UNS2 pSB1C3 (for 7 reactions) containing:
</LI><LI>17.5 μL of 10 uM NEW FWD_UNS3 primer</LI><LI>17.5 μL of 10 uM NEW REV_UNS2 primer</LI></UL><LI>Made Master mix (MM2) without DNA template or primers:
</LI><LI>70 uL of 5x Phusion buffer</LI><LI>7 uL of 10mM dNTPS</LI><LI>10.5 uL 100% DMSO ( to reach a final concentration of 3%)</LI><LI>3.5 uL Phusion polymerase</LI><LI>Added MM2 to MM1 to make a Primer + Ploymerase mix</LI><LI>Took 18uL from polymerase + primer mix as NTC and added 32 uL of nuclease free water for final volume of 50uL</LI><LI>Added the following to make a Primer + ploymerase + DNA Mix (for 6 reactions)
</LI><LI>6 uL of 1 ng/uL miniprep pSB1C3 (RFP)</LI><LI>186 uL of Nuclease free water</LI><LI>Divided final mix into 6 tubes, 50 uL each</LI><LI>Ran PCR with S1-6 in 62.3, 63.7, 66.3, 69.5, 71.3, and 62°C respectively, with NTC in 69.5°C</LI><LI>Ran a gel of the results</LI><H3>Week 5, day 1 (24/07)</H3><UL class="bullets"><LI>Dpn1 digest of pSB1C3+UNS PCR from 21/07
</LI><LI>4 sample tubes and 1 control without Dpn1</LI></UL><LI>Gel electrophoresis of the results</LI><UL class="bullets"><LI>Note: The absence of bands could be due to multiple reasons; the product was lost during digestion and purification or there was not enough product initially.</LI><LI>Nanodrop of the purified samples was preformed to test concentration of DNA purified.</LI></UL><P>Table 1: Nanodrop results of purified Dpn1 digestion of pSB1C3+UNS</P><TABLE class="table table-striped table-bordered"><THEAD><TR><TH>Sample</TH><TH>260/280</TH><TH>260/230</TH><TH>ng/uL</TH></TR></THEAD><TBODY><TR><TD>D1</TD><TD>1.88</TD><TD>1.43</TD><TD>10.6</TD></TR><TR><TD>D2</TD><TD>1.74</TD><TD>1.34</TD><TD>11.8</TD></TR><TR><TD>D3</TD><TD>1.91</TD><TD>0.75</TD><TD>8.5</TD></TR><TR><TD>D4</TD><TD>1.87</TD><TD>1.64</TD><TD>13.6</TD></TR><TR><TD>Undigested pSB1C3</TD><TD>1.33</TD><TD>1.16</TD><TD>6.2</TD></TR></TBODY></TABLE><H3>Week 5, day 2 (25/07)</H3><P>Assembled reporter and mCherry constructs</P><UL class="bullets"><LI>Pooled the purified pKDL071+UNS product from 21/07 purification to obtain a greater amount of DNA necessary for assembly.
</LI><LI>The resulting concentration of product, measured via Nanodrop, was 38.2 ng/uL (Contamination values: 260/280 = 1.93; 260/230 = 1.77).</LI></UL><LI>Created assembly according to the HiFi DNA assembly protocol. The reporter and mCherry master mix values are listed in the table below.</LI><P>Table 1: HiFi DNA Assembly Calculations</P><TABLE class="table table-striped table-bordered"><THEAD><TR><TH>Sample</TH><TH>Reporter</TH><TH>mCherry</TH></TR></THEAD><TBODY><TR><TD>pKDL071+UNS</TD><TD>1.67uL</TD><TD>1.67uL</TD></TR><TR><TD>Reporter mCherry</TD><TD>10.81uL</TD><TD>N/A</TD></TR><TR><TD>Reporter YFP</TD><TD>9.7uL</TD><TD>N/A</TD></TR><TR><TD>mCherry</TD><TD>N/A</TD><TD>10uL</TD></TR><TR><TD>Total fragment volume</TD><TD>22.18uL</TD><TD>11.67uL</TD></TR><TR><TD>Buffer (HiFi MM)</TD><TD>22.18uL</TD><TD>11.67uL</TD></TR><TR><TD>Total Reaction Volume</TD><TD>44.36uL</TD><TD>23.34uL</TD></TR></TBODY></TABLE><UL class="bullets"><LI>Transformed the assembly parts into the NEB 5α competent cells (came with the HiFi DNA assembly kit).</LI></UL><P>Plated NEB 5α cells to grow overnight in the 37°C shaker.</P><UL class="bullets"><LI>12 LB+KAN plates(6 mCherry, 6 reporter)</LI><LI>4 LB+AMP plates (2 for the positive control transformation (pUC19), 2 for the positive assembly control)</LI></UL><H3>Week 5, day 3 (26/07)</H3><UL class="bullets"><LI>Examined NEB 5α plates from 25/07 and saw single colonies</LI></UL><UL class="bullets"><LI>Inoculated single colonies from pKDL071 mCherry and Reporter plates into LB+KAN and LB+CAM and put in incubator</LI></UL><P>Ran Phusion PCR of pSB1C3 (minprep #2) with new primers</P><UL class="bullets"><LI>Made 5 samples and 1 NTC</LI><LI>Loaded samples qPCR machine at 68°C</LI></UL><H3>Week 5, day 4 (26/07)</H3><UL class="bullets"><LI>Gel Electrophoresis of pSB1C3 Phusion PCR from 26/07
</LI><LI>Primer dimers &amp; no amplified product</LI><LI>NEB’s 2-Log-DNA Ladders were used</LI></UL><UL class="bullets"><LI>Miniprepped mCherry and reporter (YFP) constructs from our pKDL071 plasmid backbone from DH5a transformed bacteria (from 25/07) with subsequent nanodrop to measure concentration of DNA:
</LI><LI>9 minipreps in total for mCherry construct:
</LI><LI>6 from inoculation culture of colony A and B</LI><LI>3 from inoculation culture of colony C</LI></UL><LI>6 minipreps in total for Reporter construct:
</LI><LI>3 minipreps from inoculation culture of colony A</LI><LI>3 minipreps from inoculation culture of colony B</LI><LI>Nuclease free water was added to resuspend DNA after vacufuging the miniprepped product (to eliminate elution buffer)</LI><P>Table 1: Nanodrop results of mCherry and Reporter pKDL071 plasmids</P><TABLE class="table table-striped table-bordered"><THEAD><TR><TH>Sample</TH><TH>DNA Concentration (ng/uL)</TH><TH>260/280</TH><TH>260/230</TH><TH>Nuclease-free water (uL)</TH></TR></THEAD><TBODY><TR><TD>mCherry 1 (A)</TD><TD>20.0</TD><TD>1.59</TD><TD>1.30</TD><TD>22.8571428571</TD></TR><TR><TD>mCherry 2 (A)</TD><TD>36.6</TD><TD>1.78</TD><TD>1.74</TD><TD>41.8285714286</TD></TR><TR><TD>mCherry 3 (A)</TD><TD>38.7</TD><TD>1.60</TD><TD>0.90</TD><TD>44.2285714286</TD></TR><TR><TD>mCherry 1 (B)</TD><TD>74.5</TD><TD>1.54</TD><TD>0.74</TD><TD>85.1428571429</TD></TR><TR><TD>mCherry 2 (B)</TD><TD>25.0</TD><TD>1.69</TD><TD>1.54</TD><TD>28.5714285714</TD></TR><TR><TD>mCherry 3 (B)</TD><TD>27.2</TD><TD>1.65</TD><TD>1.35</TD><TD>31.0857142857</TD></TR><TR><TD>mCherry 1 ©</TD><TD>25.3</TD><TD>1.89</TD><TD>2.46</TD><TD>28.9142857143</TD></TR><TR><TD>mCherry 2 ©</TD><TD>24.1</TD><TD>2.08</TD><TD>2.18</TD><TD>27.5428571429</TD></TR><TR><TD>mCherry 3 ©</TD><TD>24.0</TD><TD>2.05</TD><TD>2.54</TD><TD>27.4285714286</TD></TR><TR><TD>Reporter 1 (A)</TD><TD>18.1</TD><TD>1.57</TD><TD>1.42</TD><TD>20.6857142857</TD></TR><TR><TD>Reporter 2 (A)</TD><TD>18.8</TD><TD>1.55</TD><TD>1.36</TD><TD>21.4857142857</TD></TR><TR><TD>Reporter 3 (A)</TD><TD>21.1</TD><TD>1.55</TD><TD>1.33</TD><TD>24.1142857143</TD></TR><TR><TD>Reporter 1 (B)</TD><TD>21.6</TD><TD>2.05</TD><TD>2.31</TD><TD>24.6857142857</TD></TR><TR><TD>Reporter 2 (B)</TD><TD>20.7</TD><TD>2.09</TD><TD>2.17</TD><TD>23.6571428571</TD></TR><TR><TD>Reporter 3 (B)</TD><TD>21.8</TD><TD>2.06</TD><TD>2.63</TD><TD>24.9142857143</TD></TR></TBODY></TABLE><UL class="bullets"><LI>Set up and re-ran Phusion PCR of pSB1C3 with same settings as 26/07</LI><LI>Streaked out 5 plates of pKDL071_mCherry (from colonies a, b, c) and pKDL071_Reporter (from colonies a and b) and put in 37°C incubator overnight</LI></UL><H3>Week 5, day 5 (28/07)</H3><UL class="bullets"><LI>Checked the plates with pKDL017_mCherry and pKDL017_Reporter from 27/07</LI><LI>Ran the gel for 2nd PCR of PSB1C3 with new primers from 27/07</LI></UL><UL class="bullets"><LI>Note: The exposure time might be too little (only 0.02), needs to be at least 0.35s.</LI></UL><H3>Week 6, day 1 (31/07)</H3><UL class="bullets"><LI>Ran PCR for pSB1C3 + new UNS primers (Tm Calculator indicated annealing temperature was 67°C)
</LI><LI>6 samples at 62°C, 62.9°C, 64.9°C, 68°C, 70.5°C, and 72°C</LI><LI>One NTC at 68°C</LI></UL><LI>Ran gel of the results</LI><H3>Week 6, day 3 (02/08)</H3><UL class="bullets"><LI>Made electrocompetent MG 1655ΔLacI cells according to the electrocompetent cell protocol.</LI><LI>Made 80 glycerol stocks of the electrocompetent MG1655ΔLacI cells. Stored cells in -80°C freezer.</LI></UL><H3>Week 6, day 4 (03/08)</H3><UL class="bullets"><LI>Vacufuged pKDL071 plasmids (w/ mCherry and Reporter) to concentrate DNA</LI><LI>Transformed electrocompetent MG1655ΔLacI cells (from 02/08) with RFP competence test kit and placed in 37°C incubator overnight</LI><LI>Resuspended vacufuged DNA samples in 7uL of nuclease free water to reach a concentration of 35ng/uL</LI></UL><H3>Week 6, day 5 (04/08)</H3><UL class="bullets"><LI>Checked competence of MG1655ΔLacI cells (from 03/08) - the results were good.</LI><LI>Nanodropped mCherry and Reporter DNA samples from 03/08</LI></UL><P>Table 1: Nanodrop results of vacufuged mCherry and Reporter DNA</P><TABLE class="table table-striped table-bordered"><THEAD><TR><TH>Sample</TH><TH>ng/uL</TH><TH>260/280</TH><TH>260/230</TH></TR></THEAD><TBODY><TR><TD>mCherry A1</TD><TD>67.3</TD><TD>2.74</TD><TD>1.00</TD></TR><TR><TD>mCherry A2</TD><TD>14.6</TD><TD>3.6</TD><TD>1.14</TD></TR><TR><TD>mCherry A3</TD><TD>17.2</TD><TD>5.23</TD><TD>0.83</TD></TR><TR><TD>mCherry B1</TD><TD>26.4</TD><TD>2.86</TD><TD>1.06</TD></TR><TR><TD>mCherry B2</TD><TD>38.6</TD><TD>2.76</TD><TD>1.16</TD></TR><TR><TD>mCherry B3</TD><TD>46.2</TD><TD>2.58</TD><TD>1.23</TD></TR><TR><TD>mCherry C1</TD><TD>56.2</TD><TD>2.61</TD><TD>1.23</TD></TR><TR><TD>mCherry C2</TD><TD>60.8</TD><TD>2.37</TD><TD>1.17</TD></TR><TR><TD>mCherry C3</TD><TD>48.4</TD><TD>2.57</TD><TD>1.23</TD></TR><TR><TD>Reporter A1</TD><TD>45</TD><TD>2.86</TD><TD>1.13</TD></TR><TR><TD>Reporter A2</TD><TD>47.5</TD><TD>2.69</TD><TD>1.06</TD></TR><TR><TD>Reporter A3</TD><TD>57.5</TD><TD>2.76</TD><TD>1.07</TD></TR><TR><TD>Reporter B1</TD><TD>81.9</TD><TD>2.69</TD><TD>1.1</TD></TR><TR><TD>Reporter B2</TD><TD>54.2</TD><TD>2.8</TD><TD>1.07</TD></TR><TR><TD>Reporter B3</TD><TD>54.3</TD><TD>2.63</TD><TD>1.11</TD></TR></TBODY></TABLE><UL class="bullets"><LI>Performed Colony PCR of mCherry + Reporter construct</LI><LI>Ran gel of colony PCR results</LI></UL><H3>Week 7, day 2 (08/08)</H3><UL class="bullets"><LI>Ran 12-well gel of 04/08 PCR products: pKDL071 plasmid in wells 2-8; pSB1C3 plasmid in wells 9-12
</LI><LI>DNA ladder 10kb biobasic</LI><LI>Reporter A2</LI><LI>Reporter B3</LI><LI>mCherry A1</LI><LI>mCherry B2</LI><LI>mCherry C2</LI><LI>Positive control</LI><LI>Negative control</LI><LI>Sample 1</LI><LI>Sample 2</LI><LI>Sample 3</LI><LI>Negative control.</LI></UL><UL class="bullets"><LI>Repeated colony PCR of mCherry and reporter constructs in the pKDL071 plasmid
</LI><LI>PCR was run with samples mCherry A1, B3, and C1, and Reporter A1 and B3.</LI><LI>Samples were first diluted to concentrations of 1 ng/uL based on nanodrop results from 04/08.</LI></UL><LI>Gel electrophoresis of colony PCR of mCherry and reporter constructs.</LI><H3>Week 7, day 3 (09/08)</H3><UL class="bullets"><LI>Q5 PCR with Temperature Gradient (62-72°C) of PKDL071 with mCherry sample A1
</LI><LI>Samples 1-5 at 72.0°C, 70.0°C, 68.3°C, 64.3°C, 62.0°C, and 66.0°C</LI><LI>NTC at 66.0°C</LI></UL><LI>NEB OneTaq Hot Start PCR with temperature gradient of pSB1C3 to add UNSs with new primers. Tm Calculator indicated annealing temperature of 54°C.
</LI><LI>Samples 1-6 at 61.0°C, 59.1°C, 57.3°C, 53.5°C, 51.9°C, 51.0°C, 57.3°C</LI><LI>NTC at 57.3°C</LI><LI>Transformation of MG1655ΔLacI with Reporter construct and mCherry construct respectively, both in pKDL071 bakcbone.</LI><LI>Gel eletrophoresis of both PCRs</LI><H3>Week 7, day 4 (10/08)</H3><UL class="bullets"><LI>Examined transformed MG1655ΔLacI with pKDL071 plasmid with insert from 09/08 (NC, Rep A, B, mCherry A, B, C)</LI></UL><UL class="bullets"><LI>NEB OneTaq Hot Start PCR with temperature gradient of Christian’s inserts (using colony PCR primers to test if they are working)
</LI><LI>Made Master Mix (8 reactions) (50uL)
</LI><LI>80uL GC Reaction Buffer</LI><LI>8uL of 10uM dNTP</LI><LI>8uL of 10uM FWD</LI><LI>8uL of 10uM REV</LI><LI>4uL of OneTaq Polymerase</LI><LI>12uL of 100% DMSO (to make final concentration of 3%)</LI></UL><LI>Made NTC with 15uL of MM and 35uL nuclease free water</LI><LI>Added DNA to MM:
</LI><LI>7uL of DNA Template</LI><LI>238 uL nuclease free water</LI><LI>Added 50uL final mix to 6 tubes and ran the PCR</LI><LI>Gel electrophoresis of PCR products showed colony PCR primers weren’t working</LI><H3>Week 7, day 5 (11/08)</H3><UL class="bullets"><LI>Designed new pKDL071 colony PCR primers
</LI><LI>BLASTED against pKDL071 w/ UNSs, also BLASTED against <EM>E.coli</EM> K-12</LI><LI>Highest off target E-value was 0.46 for both FWD and RVS.</LI><LI>When analyzed with IDT OligoAnalyzer Tool the highest Delta G: -6.75 kcal/mole (Base Pairs3) was found for hetero-dimerization between fwd and rvs primers.)</LI></UL><P>“08-11 primer design.png”</P><UL class="bullets"><LI>Notes on how to interpret IDT OligoAnalyzer Output:
</LI><LI>For self-dimer analysis, click on ‘Self-Dimer’ to bring up a new window with each possible self-dimer your oligo can form. For each diagram you will be able to see the calculated delta G value for this secondary structure. If you have a strong delta G (-9kcal/mol or more negative) this oligo could be problematic.</LI><LI>Heterodimer analysis and hairpin formation is same as above</LI></UL><P>“08-11 analysis.png”</P><H3>Week 8, day 4 (17/08)</H3><UL class="bullets"><LI>Received new pSB1C3 and pKDL071 w/ UNS2/UNS3 FWD/REV primers</LI><LI>Phusion PCR with temperature gradient (62°C to 72°C) to put UNS primers onto pKDL071 and pSB1C3 backbone
</LI><LI>6 pKDL071 samples + NTC and 6 pSB1C3 samples + NTC</LI></UL><LI>Gel Electrophoresis of PCR products to determine size</LI><UL class="bullets"><LI>Redid Phusion PCR for pSB1C3 with new primers with increased template; also tested 2016 pSB1C3 primers with increased template
</LI><LI>Made Master Mix (MM1): (these were done for BOTH old (2016) and new (2.0) primers) for 8 reactions (50 μL each)
</LI><LI>80uL of 5x Phusion HF Buffer</LI><LI>8uL of 10mM dNTPs</LI><LI>20uL FWD primer</LI><LI>20uL REV primer</LI><LI>12uL of 100% DMSO</LI><LI>4uL Phusion polymerase</LI></UL><LI>Made NTC with 18uL of MM1 and 32uL of nuclease free water</LI><LI>Added DNA to MM1 to make MM2
</LI><LI>50uL (1ng/uL) template</LI><LI>174uL nuclease free H2O</LI><LI>Made 6 50uL PCR tubes from MM2</LI><LI>Gel electrophoresis of PCR products</LI><UL class="bullets"><LI>Dpn1 digest of pKDL071 to get rid of template</LI><LI>PCR purification and Nanodrop of digested pKDL071</LI></UL><P>Table 1: Nanodrop results of purified Dpn1 digested pKDL071</P><P>| Sample    | ng/uL | 260/280   | 260/230 |
| — | — | — |
| 4 | 12.2  | 2.55  | 0.91 |
| 6 | 11.6  | 2.80  | 0.75 |
*Note: samples 3/4 and 5/6 were combined into sample 4 and 6, receptively.</P><UL class="bullets"><LI>Performed Gibson HiFi DNA assembly, adding 0.1 pmol of fragments and 0.05pmol of inserts</LI></UL><P>Table 2: Reaction Component Table for Gibson HiFi DNA Assembly</P><TABLE class="table table-striped table-bordered"><THEAD><TR><TH>Component</TH><TH>Reporter</TH><TH>mCherry</TH></TR></THEAD><TBODY><TR><TD>pKDL071 + UNS BB</TD><TD>5.23 uL</TD><TD>5.23 uL</TD></TR><TR><TD>Reporter_mCherry</TD><TD>10.81 uL</TD><TD>-</TD></TR><TR><TD>Reporter_YFP</TD><TD>9.7 uL</TD><TD>-</TD></TR><TR><TD>LacILOV_mCherry</TD><TD>-</TD><TD>10 uL</TD></TR></TBODY></TABLE><H3>Week 8, day 5 (18/08)</H3><UL class="bullets"><LI>Gibson plates from 17/08 (as well as the positive control supplied by Gibson kit) showed no colonies</LI><LI>PCR of pSB1C3 UNS with 2.0 UNS primers using a linearized pSB1C3 backbone to control for supercoil interference. We also ran a temperature gradient: 50-71°C. Phusion HF polymerase was used as well as 3% DMSO.</LI><LI>Gel electrophoresis of PCR products:</LI></UL><P>“08-18 pSB1C3 gel.png”</P><UL class="bullets"><LI><P>Result: Primer Dimers and what appears to be a slightly larger product (we hypothesize that these could be the palindromic ends of our UNS primers (backbone complement) annealing and amplifying.</P></LI><LI><P>Redid Gibson assembly based on new Nanodrop of samples</P></LI></UL><P>Table 1: Nanodrop results of purified Dpn1 digested pKDL071</P><P>| Sample    | ng/uL | 260/280   | 260/230 |
| — | — | — |
| 4 | 16.8  | 1.63  | 1.23 |
| 6 | 16.1  | 1.63  | 0.72 |</P><P>Table 2: Reaction Component Table for Gibson HiFi DNA Assembly</P><TABLE class="table table-striped table-bordered"><THEAD><TR><TH>Component</TH><TH>Reporter</TH><TH>mCherry</TH></TR></THEAD><TBODY><TR><TD>pKDL071 + UNS BB</TD><TD>4 uL</TD><TD>4 uL</TD></TR><TR><TD>Reporter_mCherry</TD><TD>10.87 uL</TD><TD>-</TD></TR><TR><TD>Reporter_YFP</TD><TD>9.7 uL</TD><TD>-</TD></TR><TR><TD>LacILOV_mCherry</TD><TD>-</TD><TD>10 uL</TD></TR></TBODY></TABLE><H3>Week 9, day 1 (21/08)</H3><UL class="bullets"><LI>OneTaq PCR of Christian’s miniprepped pKDL071 plasmid with backbone specific PCR colony primers to test them out with temperature gradient from 45-55°C</LI><LI>Gel electrophoresis of PCR product:</LI></UL><UL class="bullets"><LI>Gel electrophoresis for assembled pKDL071 Reporter ® and mCherry (M) plasmids from 25/07 (1), 17/08 (2), and 18/08 (3)</LI></UL><UL class="bullets"><LI>Made 5mL overnight LB+KAN cultures of pKDL071 mCherry colonies 1, 2, and 3 in preparation for miniprep the next day</LI></UL><H3>Week 9, day 2 (22/08)</H3><UL class="bullets"><LI>Miniprepped Gibson assembly colonies A through C</LI><LI>Performed EZ-10 spin column PCR products purification kit of all PCR reactions using 2.0 primers on pSB1C3 backbone (samples 9-14 from 17/08 and samples 1-11 from 17/08)
</LI><LI>Purpose: to find out if the backbone + UNS was actually amplified during PCR, and was not visible due to low yield.</LI><LI>The purified products would then be combined to obtain enough product to run a gel extraction.</LI></UL><LI>Nanodropped PCR purified pKDL071+UNS samples 4 and 6 from 17/08 and miniprepped plasmid with mCherry in pKDL071 (colonies A,B and C)</LI><P>Table 1: Nanodrop results of PCR purified pKDL071+UNS and mCherry+pKDL071 colonies</P><TABLE class="table table-striped table-bordered"><THEAD><TR><TH>Sample</TH><TH>ng/uL</TH><TH>260/280</TH><TH>260/230</TH></TR></THEAD><TBODY><TR><TD>4</TD><TD>1.85</TD><TD>1.02</TD><TD>40.7</TD></TR><TR><TD>6</TD><TD>2.11</TD><TD>0.91</TD><TD>25.4</TD></TR><TR><TD>A</TD><TD>1.85</TD><TD>1.46</TD><TD>68.6</TD></TR><TR><TD>B</TD><TD>1.76</TD><TD>1.17</TD><TD>74.9</TD></TR><TR><TD>C</TD><TD>1.90</TD><TD>1.45</TD><TD>77.2</TD></TR></TBODY></TABLE><UL class="bullets"><LI>OneTaq PCR of purified plasmid: 6 samples and 1 NTC</LI><LI>Gel electrophoresis of PCR purified pSB1C3</LI></UL><P>“08-22 PCR purified pSC1C”</P><UL class="bullets"><LI>Gel electrophoresis of LacILOV mCherry from colonies and colony PCR products</LI></UL><P>“08-22 OneTaq HotStart pKDL071 mCherry with 2.0 Colony PCR primers”</P><UL class="bullets"><LI>Gibson assembly of the reporter constructs</LI></UL><H3>Week 9, day 3 (23/08)</H3><UL class="bullets"><LI>Vacufuged assembly from 22/08 and took Nanodrop each time</LI><LI>Ran a gel of miniprepped reporter</LI></UL><H3>Week 9, day 4 (24/08)</H3><UL class="bullets"><LI>Gibson Assembly of mCherry_LacILOV insert and YFP_Reporter insert
</LI><LI>Note: we put the tubes in the PCR at 50°C instead of using thermocycler</LI></UL><LI>Constructed the light array</LI><LI>Transformed DH5α cells with Gibson assembled mCherry_LacILOV_YFP_reporter insert</LI><H3>Week 9, day 5 (25/08)</H3><UL class="bullets"><LI>Checked plates from yesterday and got 1 colony</LI><LI>Colony PCR of reporter insert (resuspended colony in water)</LI><LI>RE Digestion (pstI)</LI><LI>Gel electrophoresis in order: 2-log NEB ladder, Christian’s insert, Reporter insert, NTC</LI></UL><H3>Week 10, day 1 (29/08)</H3><UL class="bullets"><LI>Phusion PCR with colony PCR primers on our Gibson Assembly product</LI></UL><UL class="bullets"><LI>PCR of LacILOV_mCherry insert WITH restriction enzyme primers</LI></UL><H3>Week 10, day 2 (30/08)</H3><UL class="bullets"><LI>Ran the gel for PCR purified Reporter and LacILOV.</LI><LI>Gibson Assembly of mCherry and YFP in pKDL071</LI><LI>RE Digest of reporter and pSB1C3</LI><LI>NEB Transformation protocol (E2621) for HIFI Gibson
</LI><LI>3 transformations: reporter, transformation, and assembly</LI></UL><LI>T4 DNA ligation of pSB1C3+reporter</LI><LI>Nanodropped LacILOV mCherry gel</LI><H3>Week 10, day 3 (31/08)</H3><UL class="bullets"><LI>RE Digest (XbaI, PstI) of LacILOV &amp; mCherry from gel</LI><LI>Transformation of rep_pSB1C3 into DH10B</LI><LI>Gibson assembly of reporter + pJDK071 with 1:1 ratio insert:BB</LI></UL><H3>Week 10, day 4 (01/09)</H3><UL class="bullets"><LI>Colony PCR :
</LI><LI>Transformed DH10B cells with ligated pSB1C3 &amp; Reporter construct (3:1, reporter: Backbone) using OneTaq Hot Start polymerase.</LI><LI>Transformed reporter + pKDL071 HiFi DNA assembly with 1:1:1 of Reporter_YFP: Reporter_mCherry: pKDL071 w UNS, using OneTaq Hotstart</LI></UL><LI>Streaked reporter_pSB1C3 and reporter_pKDL071 colonies onto new LB+CAM plates</LI><LI>Transformed ligated reporter + pSB1C3 (1:3, reporter: Backbone) and ligated LacILOV_mCherry+ pSB1C3(3:1, insert:Backbone) into DH10B cells</LI><H3>Week 11, day 2 (05/09)</H3><UL class="bullets"><LI>OneTaq Hot Start Colony PCR on colonies from 01/09 (pSB1C3_reporter &amp; pSB1C3_mCherry_LacILOV (transformed into DH10Beta))</LI><LI>Phusion PCR: pSB1C3 backbone amplification using 2016 primers</LI><LI>Overnight culture of LacILOV_mCherry insert in pSB1C3 plasmid in DH5beta</LI></UL><H3>Week 11, day 3 (06/09)</H3><UL class="bullets"><LI>Redid Colony PCR w/ LacILOV_mCherry &amp; Reporter constructs and Phusion PCR for amplification of pSB1C3 backbone (with 2016 primers) from 05/09</LI><LI>Miniprepped our overnight culture of DH10Beta with LacILOV construct in pSB1C3</LI><LI>Phusion PCR of Gibson Mix with new UNS 2 REV and UNS 3 FWD primers</LI></UL><H3>Week 11, day 4 (07/09)</H3><UL class="bullets"><LI>Nanodrop results of pooled PCR purified samples from 06/09</LI><LI>Redid Phusion PCR from 06/09</LI><LI>RE Digest for PCR with RE primers of pSB1C3</LI><LI>PCR purification of digested pSB1C3</LI></UL><H3>Week 11, day 5 (08/09)</H3><UL class="bullets"><LI>Phusion PCR of LacILOV_mCherry insert from plasmid (inserted into both BBs) + directly from IDT stock to test UNS primers</LI><LI>Ran gel for PCR purified reporter construct with RE sites from 29/08
</LI><LI>Gel extract of single band on gel, following protocol from NEB Gel extraction kit</LI><LI>Nanodropped results</LI><LI>Phusion PCR of gel extracted reporter</LI></UL><LI>PCR purified PCR product from phusion PCR of reporter construct with RE sites using UNS primers (07/09)
</LI><LI>Nanodropped product</LI><LI>Gel of purified product</LI><LI>Gibson Assembly of pKDL071 + reporter with vector: insert (2:1 ratio) from 22/08
</LI><LI>Transformed Gibson products following NEB Gel extraction protocol</LI><LI>Ligated dephosphorylated + digested pSB1C3 BB from 07/09 and gel extraction product from today</LI><LI>LacILOV_mCherry colony and completely resuspended in microcentrifuge tube with LB+Kan</LI><H3>Week 11, day 6 (09/09)</H3><UL class="bullets"><LI>We got colonies! From HiFi assembly of reporter+pKDL071 in 2:1 vector:insert ratio from 08/09
</LI><LI>Colony PCR to test</LI></UL><LI>Phusion PCR with UNS primers for (A) Gibson mix HiFi DNA assembly from 08/09, and B) IDT LacILOV_mCherry insert</LI><LI>Phusion PCR with RE primers for Gibson mix (HiFi assembly) from 08/09 with temperature gradient</LI><H3>Week 12, day 1 (11/09)</H3><UL class="bullets"><LI>Colony PCR of reporter construct assembled 08/09 using UNS primers to eliminate non-specific binding possibility of colony PCR primers</LI><LI>Crossover PCR of reporter subpart YFP &amp; reporter subpart mCherry to assemble reporter construct</LI><LI>Inoculated 5ml overnight cultures with all reporter colonies</LI><LI>Performed LacILOV_mCherry end point measurement and start new assay</LI></UL><H3>Week 12, day 2 (12/09)</H3><UL class="bullets"><LI>Temperature gradient PCR with UNS 3 FWD and UNS 2 REV primers with LacILOV_mCherry in pKDL071 and pSB1C3</LI><LI>LacILOV_mCherry pSB1C3 miniprep</LI><LI>Overlap PCR of reporter inserts</LI></UL><H3>Week 12, day 3 (13/09)</H3><UL class="bullets"><LI>Miniprepping reporter construct in pKDL071</LI><LI>New UNS 4 fwd and rev primers</LI><LI>PCR purified reporter constructs from PCR amplification</LI></UL><H3>Week 12, day 4 (14/09)</H3><UL class="bullets"><LI>Phusion PCR of miniprep’d “reporter” plasmid in pKDL071 using colony PCR primers</LI></UL><H3>Week 12, day 5 (15/09)</H3><UL class="bullets"><LI>RE Digest of minipreped “reporter” plasmids R1, R2, and RA (used 10ul of plasmid in each rxn) from gibson assemblies with PKDL071 back bone.</LI><LI>Ran gel of PCR purified reporter inserts
</LI><LI>DNA Gel Extraction and Nanodropped results</LI></UL><LI>Phusion PCR temp gradient of reporter mcherry with UNS primers (65-55C)</LI><H3>Week 13, day 1 (18/09)</H3><UL class="bullets"><LI>Redid reporter mCherry temperature gradient using PCR purified mCherry sample from 14/09</LI></UL><H3>Week 13, day 2 (19/09)</H3><UL class="bullets"><LI>Phusion PCR of LacILOV_mCherry in pKDL071 using UNS primers to test if PCR reagents are good</LI></UL><H3>Week 13, day 3 (20/09)</H3><UL class="bullets"><LI>PCR amplification of YFP reporter &amp; mCherry reporter from PCR purified product and gel extracted product.
</LI><LI>Used 1:10 diluted PCR purified YFP and mCherry and Gel extracted YFP and mCherry will as template for PCR</LI></UL><LI>Nanodropped Reporter Parts</LI><LI>Amplification of reporter insert from ALL previous Gibson Assemblies using UNS 3 FWD and UNS 2 REV</LI><LI>Final PCR: PCR of LacILOV_mCherry purified (1rxn + NTC + buffer)</LI><H3>Week 13, day 4 (21/09)</H3><UL class="bullets"><LI>RE (BglII) digest of LacILOV_mCherry in pKDL071</LI></UL><H3>Week 13, day 5 (22/09)</H3><P>Note: From PCR results done on 21/09, PCRing after PCR purification is not usually successful due to incomplete amplicons from the first PCR</P><UL class="bullets"><LI>RE Digest results:
</LI><LI>Bgl II digest of LacILOV mcherry yielded unexpected results and digestion will be repeated today for a longer incubation period (~2.5h)</LI><LI>DpnI digest of PCR puified pKDL071; was not digested beforePCR purification, PCR purify after digest.</LI></UL><LI>Bgl II digest of LacILOV mcherry PKDL071, LacILOVmcherry pSB1C3, and Christan’s pKDL071 (digested for 2.5h)</LI><H3>Week 14, day 1 (25/09)</H3><UL class="bullets"><LI>Ran temperature gradient PCR with colony PCR primers of LacILOV mCherry with Phusion HF to test if OneTaq gave artifacts</LI><LI>Started overnight culture of LacILOV_mCherry for assays</LI><LI>Designed primers to add RE sites (xbai and psti) to LacILOV, reason: be able to clone LacILOV into pSB1C3 for parts submission</LI></UL><H3>Week 14, day 2 (26/09)</H3><UL class="bullets"><LI>PCR (25ul rxn, 6 rxns) using colony PCR primers to check if successfully cloned reporter insert into pKDL071</LI><LI>Phusion PCR to amplify this to insert reporter_mCherry</LI><LI>DpnI digest of pKDL071 amplified backbone with UNS 2 at 5’ end and UNS 4 at 3’ end
</LI><LI>combined PCR samples 1&amp;2, 3&amp;4</LI></UL><LI>PCR purification and Nanodrop of dpnI digested pKDL071</LI><H3>Week 15, day 3 (27/09)</H3><UL class="bullets"><LI>Gibson Assembly of Reporter_mCherry and pKDL071 BB with UNS 2 + 4
</LI><LI>Reporter_mCherry will be used from IDT stock</LI><LI>pKDL071 BB with UNS 2 and 4 will be used from sample 1 prepared 26/09</LI></UL><LI>Phusion PCR using UNS 4 FWD and UNS 3 REV to amplify pKDL071</LI><LI>Transformed of DH5a cells with our mCherry reporter in pKDL071
</LI><LI>Spreaded LB+Kan plates and placed in incubator overnight @ 37oC</LI><H3>Week 15, day 4 (28/09)</H3><UL class="bullets"><LI>Got 28 colonies from 27/09!</LI><LI>Colony PCR with OneTaq and colony PCR primers</LI><LI>DpnI digest of pKDL071 BB using UNS4 and UNS3 primers</LI><LI>PCR purification of DpnI digested pKDL071</LI><LI>Nanodrop of dpnI digested + PCR purified pKDL071 BB</LI></UL><H3>Week 15, day 5 (29/09)</H3><UL class="bullets"><LI>Started an overnight culture of colonies 1,4,5,6,14,18,26,29 (streaked onto new plates) in LB+KAN</LI><LI>Colony PCR of colony 29 from 28/09</LI></UL><H3>Week 15, day 6 (30/09)</H3><UL class="bullets"><LI>Miniprepped reporter_mCherry_pKDL071 colonies 1,4,5.6.14.18.26 &amp;29</LI><LI>PCR amplified those plasmids with UNS4 FWD &amp; UNS2 REV primers</LI></UL><H3>Week 16, day 1 (02/10)</H3><UL class="bullets"><LI>Phusion PCR to add UNS 3 to reporter mCherry in pKDL071 - will make it easier to add YFP reporter (Gibson)</LI></UL><H3>Week 16, day 2 (03/10)</H3><UL class="bullets"><LI>Received LacILOV Fwd and rev primers adding cut sites XbaI and PstI and YFP reporter stock from IDT</LI><LI>Phusion PCR for LacILOV PCR with RE sites</LI><LI>PCR amplification of reporter_mCherry colonies with UNS 4 FWD and pKDL071 UNS 3 rev to linearized plasmid and add UNS 3 for Gibson assembly of reporter_YFP</LI></UL><H3>Week 16, day 3 (04/10)</H3><UL class="bullets"><LI>Prepared samples for sequencing:
</LI><LI>Reporter_mCherry in pKDL071</LI><LI>LacILOV_mCherry in pSB1C3</LI></UL><H3>Week 16, day 4 (05/10)</H3><UL class="bullets"><LI>DpnI digest of LacILOV AND digestion of LacILOV using XbaI/PstI</LI><LI>PCR purified digested LacILOV_with_RE sites</LI></UL><H3>Week 16, day 5 (06/10)</H3><UL class="bullets"><LI>Transformed ligated LacILOV in pSB1C3 to DH5B cells</LI><LI>Gibson assembly of reporter YFP with reporter_mCherry in pKDL071 from colony 1</LI><LI>Transformed into NEB Competent cells DH5a, plated on LB+AMP and incubated overnight</LI></UL><H3>Week 16, day 6 (07/10)</H3><UL class="bullets"><LI>Colony PCR</LI></UL><H3>Week 17, day 2 (10/10)</H3><UL class="bullets"><LI>Received mCherry reporter construct from IDT, dCas9 on CamR/TetR controlled plasmid from AddGene</LI><LI>DH5a-pKDL071-reporter colonies 13,20,21,22,23 were restreaked on LB+Kan and overnight cultures of these colonies were also made</LI><LI>dCas9 containing cultures were streaked onto an LB+Cam plate and put in 37°C incubator</LI></UL><H3>Week 17, day 3 (11/10)</H3><UL class="bullets"><LI>Miniprepped pKDL071_reporter plasmids from colonies 13, 20, 21</LI><LI>Made overnight culture of pdCas9 cells to miniprep the next day</LI><LI>Phusion PCR of reporter colonies with UNS 3 FWD and UNS 2 REV primers (7rxns, 25ul each)</LI><LI>Prepared sequencing primers for sequencing reporter construct in pKDL071
</LI><LI>seq 1,2,3,4,5,6</LI><LI>To amplify pSB1C3 out of LacILOV mCherry
</LI><LI>primers for UNS3 REV and UNS 2 FWD</LI><LI>UNS 5 REV - sgRNA + antiCRISPR</LI><LI>add UNS 3 beginning and UNS 2 REV for antiCRISPR</LI></UL><LI>Transform MG1655ΔLacI cells with reporter pKDL071 colony 21 and 22
</LI><LI>Reporter mCherry pKDL071 was also transformed into MG1655ΔLacI as a +ve control for mCherry expression in LacILOV assays</LI><LI>RFP_pSB1C3 (10pg/ul) was used as +ve transformation control</LI><LI>Miniprepped pdCas9 from overnight culture (1.5ml into 2 tubes)</LI><H3>Week 17, day 5 (13/10)</H3><UL class="bullets"><LI>Phusion PCR for amplifying pSB1C3 backbone out of LacILOV with UNS 2 and UNS 3</LI></UL><H3>Week 17, day 6 (14/10)</H3><UL class="bullets"><LI>DpnI digest of pSB1C3 with UNS 2 and 3 PCR product</LI><LI>PCR purification of DpnI digested pSB1C3 UNS2 and 3 PCR product</LI></UL><H3>Week 18, day 2 (17/10)</H3><UL class="bullets"><LI>Phusion PCR of reporter_mCherry in pKDL071 and of AcrIIA4 and LacILOV with new UNS primers</LI><LI>DpnI digest of LacILOV and pKDL071 PCR product</LI><LI>PCR purification and Nanodrop of LacILOV, reporter_mCherry_pKDL071 with UNS 5, AcrIIA4, and reporter construct</LI><LI>Calculated Gibson Assembly of AcrIIA4, LacILOV, and sgRNA</LI></UL><H3>Week 18, day 3 (18/10)</H3><UL class="bullets"><LI>Transformed into DH5a cells and incubated at 37oC overnight (3plates each):
</LI><LI>AcrIIA4 in pSB1C3</LI><LI>sgRNA_reporter mCherry in pKDL071</LI><LI>LacILOV in pSB1C3</LI></UL><H3>Week 18, day 4 (19/10)</H3><UL class="bullets"><LI>Colony PCR of 18/10 plates
</LI><LI>AcrIIA4 using colony rev/fwd primers</LI><LI>LacILOV using colony rev/fwd primers</LI><LI>sgRNA_reporter mCherry</LI></UL><H3>Week 18, day 5 (20/10)</H3><UL class="bullets"><LI>Measurement of LacILOV_mCherry after diluting and growing overnight:
</LI><LI>First measurement was 12hrs after dilution</LI><LI>Note: LacILOV_pSB1C3 colony 9 was incubated from colony PCR sample after 5hrs from preparation as colony labeling on the plate was not clear. LacILOV_pSB1C3 colony 9 overnight culture didn’t grow and was not miniprep’d</LI></UL><LI>Miniprepped overnight cultures from colonies obtained after assembly</LI><H3>Week 18, day 7 (22/10)</H3><UL class="bullets"><LI>Phusion PCR of :
</LI><LI>Reporter_mCherry_sgRNA_pKDL071 with UNS 4 FWD and pKDL071 UNS 3 REV (3rxn + NTC + extra) to assemble full construct</LI><LI>Reporter_mCherry_sgRNA_pKDL071 with UNS 5 REV and pKDL071 UNS 6 FWD (3rxn + ntc + extra) to assemble AcrIIA4 for antiCRISPR assays</LI><LI>Reporter_pKDL071 colony 21 &amp; 22 with UNS 3 FWD and UNS 4 REV (2rxn)</LI><LI>Reporter_pKDL071_colony 21 and 22 with UNS 4 FWD and UNS 2 REV (2rxn)</LI></UL><LI>DpnI digest of sgRNA_reporter mCherry for full construct and sgRNA_reporter mCherry_pKDL071 for AcrIIA4 assembly
</LI><LI>PCR purification of S1 and AS1</LI><LI>Nanodropped results</LI><LI>Redid PCR of:
</LI><LI>Reporter_pKDL071 (colony 21 &amp; 22) with UNS 3 FWD and UNS 4 REV</LI><LI>Reporter_pKDL071 colony 21 and 22 with UNS 4 FWD and UNS 2 REV. (reporter_mCherry_pKDL071 will be used as +ve control) (3rxns)</LI><LI>Gibson assembly:
</LI><LI>AcrIIA4_YFP subparts into sgRNA reporter_mCherry_pKDL071 (assembly of full construct)</LI><LI>AcrIIA4 alone into sgRNA_reporter_mCherry_pKDL071 with UNS 5 and 6</LI><LI>Transformed MG1655ΔLacI cells with:
</LI><LI>sgRNA_rep_mCherry_pKDL071 colony 1 and pdCas9 (1ul of each plasmid will be used)</LI><LI>BBa_K523013 pSB1C3 to be used as +ve control for YFP found in plate 1 well 8H</LI><LI>Colony PCR of:
</LI><LI>Full construct (AcrIIA4_YFP_sgRNA_repmCherry)</LI><LI>sgRNA rep_mCherry in pKDL071 with UNS 5 and 6 and AcrIIA4</LI><LI>Streaked MG1655ΔLacI pdCas9 + sgRNA_pKDL071 on a new plate to get single colonies
</LI><LI>Started overnight culture of MG1655ΔLacI pdCas9 + sgRNA_pKDL071 for assays</LI><LI>Started overnight culture of MG1655ΔLacI YFP to trouble shoot YFP measurements</LI><H3>Week 19, day 2 (24/10)</H3><UL class="bullets"><LI>Miniprep and Nanodrop colonies F3 and F7, S2 and S5</LI><LI>Phusion PCR to take:
</LI><LI>Reporter construct out of pKDL071 (reporter colony #21)</LI><LI>Full construct out of miniprep’d pKDL071 (to Gibson into pSB1C3 tomorrow)</LI></UL><LI>DpnI digest of reporter construct [50ul rxn]</LI><LI>Gibson of reporter construct with pSB1C3…UNS 2 and 3</LI><H3>Week 19, day 3 (25/10)</H3><UL class="bullets"><LI>Phusion PCR of pSB1C3 out of LacILOV pSB1C3 to add UNS 5 and extend UNS 3 for Gibson of full construct in pSB1C3</LI><LI>PCR purification of full construct (F3 and F7)</LI><LI>Colony PCR of reporter in pSB1C3 (twice)</LI><LI>Redid Phusion PCR from 24/10
</LI><LI>Full construct PCR out of pKDL071 (2rxns) using UNS primers</LI><LI>Full construct PCR out of pKDL071 (F3+F7) using phusion and pKDL071 colony PCR primers</LI></UL><H3>Week 19, day 4 (26/10)</H3><UL class="bullets"><LI>Sent samples for sequencing:
</LI><LI>LacILOV_pSB1C3 (L3 + L4)</LI><LI>AcrIIA4_pSB1C3 (A1+A3+A7)</LI><LI>sgRNA_repmCherry_pKDL071 (S1)</LI><LI>AcrIIA4_sgRNA_repmCherry (S2+S5)</LI></UL><LI>Colony PCR of reporter_pSB1C3 Colonies 7-20
</LI><LI>Colonies 10,11,15 to be streaked onto new plates and miniprep’d</LI><H3>Week 19, day 5 (27/10)</H3><UL class="bullets"><LI>Miniprep of reporter_pSB1C3 colonies 10,11,15</LI><LI>XbaI and PstI digest of :
</LI><LI>LacILOV_pSB1C3 (L3 and L4)</LI><LI>AcrIIA4, pSB1C3 (A1, A3 and A7)</LI><LI>Reporter_pSB1C3</LI></UL><LI>Colony PCR (10ul/rxn, 14 rxns) + 1 buffer</LI><H3>Week 19, day 7 (29/10)</H3><UL class="bullets"><LI>Minipreped full construct from overnight culture</LI><LI>Transformed full construct plasmids + pdCas9 in MG1655 delta LacI</LI><LI>Started overnight cultures:
</LI><LI>reporter_mCherry_pKDL071 + pdCas9 MG1655ΔLacI in minimum media + 0.2% glucose</LI><LI>reporter_mCherry_sgRNA_pKDL071 + pdCas9 MG1655ΔLacI in minimum media + 0.2% glucose</LI><LI>reporter_mCherry_sgRNA_AcrIIA4_pKDL071 + pdCas9 MG1655ΔLacI in minimum media + 0.2% glucose</LI></UL><LI>Streaked LacILOV_mCherry_pKDL071 MG1655ΔLacI on 4 LB+Kan plates to test LOKI</LI><LI>Grew :
</LI><LI>MG1655ΔLacI + YFP +ve to measure YFP</LI><LI>MG165ΔLacI repmCherry</LI><LI>pdCas9 cells</LI><H3>Week 20, day 1 (30/10)</H3><UL class="bullets"><LI>Restreaked 9 colonies (labeled) of MG1655ΔLacI_sgRNA_Acr_reportermCherry in pKDL071 backbone onto LB + CAM + KAN plates</LI></UL></DIV></DIV><DIV class="block sidebar"><DIV id="sidebar-box"><H4>Contents</H4><H4>Related Pages</H4><DIV class="sidebar-minibox"><UL><LI><A href="https://2017.igem.org/Team:Toronto/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/InterLab">Interlab</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Results">Results</A></LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="section bg-black"><DIV class="container footer row"><DIV class="block content"><H3 class="white">General Pages</H3><UL><LI><A href="https://2017.igem.org/Team:Toronto">Home</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Safety">Safety</A></LI></UL></DIV><DIV class="block content"><H3 class="white">Wet Lab</H3><UL><LI><A href="https://2017.igem.org/Team:Toronto/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Hardware">Hardware</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Demonstrate">Demonstrate</A></LI></UL></DIV><DIV class="block content"><H3 class="white">Dry Lab</H3><UL><LI><A href="https://2017.igem.org/Team:Toronto/ODE">ODE</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Analysis">Analysis</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Protein-Modelling">Protein Modelling</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Model">Model</A></LI></UL></DIV><DIV class="block content"><H3 class="white">Human Practices</H3><UL><LI><A href="https://2017.igem.org/Team:Toronto/Interviews">Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Genetic_Code_Workshop">Genetic Code Workshop</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Podcast">Podcast</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Icon-a-thon">Icon-a-thon</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Guide">Guide</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/HP/Silver">HP/Silver</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/HP/Gold_Integrated">HP/Gold Integrated</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Engagement">Engagement</A></LI></UL></DIV></DIV><DIV class="container footer-bottom row"><DIV class="block content"><P>iGEM Toronto</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>