<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Toronto_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Toronto/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="row navigation bg-dark-cyan"><DIV class="row" id="navigation"><UL><LI><SPAN class="text-white">• </SPAN><A href="https://2017.igem.org/Team:Toronto" id="grey-hover" class="text-white">Home</A></LI><LI><SPAN class="text-grey">• </SPAN><A href="https://2017.igem.org/Team:Toronto/Team" id="grey-hover" class="text-white">Team</A></LI><LI><SPAN class="text-red">• </SPAN><A href="https://2017.igem.org/Team:Toronto/Wetlab" id="red-hover" class="text-white">Wetlab</A></LI><LI><SPAN class="text-yellow">• </SPAN><A href="https://2017.igem.org/Team:Toronto/Drylab" id="yellow-hover" class="text-white">Drylab</A></LI><LI><SPAN class="text-cyan">• </SPAN><A href="https://2017.igem.org/Team:Toronto/Human_Practices" id="cyan-hover" class="text-white">Human Practices</A></LI></UL></DIV></DIV><DIV class="section"><DIV class="container header" id="red"><H1>Experiments</H1></DIV></DIV><DIV class="section bg-white"><DIV class="container content-page row"><DIV class="block content"><DIV id="starter_pack_0" class="subsection"><H2>Starter pack</H2><H3 class="text-red">70% Ethanol Protocol (starter pack)</H3><H4>Introduction</H4><P>Ethanol is a commonly used antiseptic in the lab environment. It has the highest effective concentration at 70% compared to a stronger solution. This occurs due to alcohol’s ability to coagulate protein on contact. A higher concentration of ethanol inflicts a very rapid coagulation of protein in the cell wall or membrane of target organism, effectively blocking further penetration of the organism and its subsequent neutralization. Because of this, a more diluted 70% concentration is used for optimal penetration.</P><H4>Safety precautions</H4><P><STRONG>SDS (safety data sheet)</STRONG>: Refer to the SDS sheets for all listed materials before entering the lab. Be prepared to answer any questions regarding th3e information on these sheets.</P><P><STRONG>PPE (Personal protective equipment)</STRONG>: Proper lab attire should be worn throughout the experiment: This means that upon entering the lab you should be wearing long pants and close-toed shoes. Contact lenses should not be worn. Furthermore, a lab coat, goggles, and gloves should be worn at all times, and long hair should be tied back.</P><P><STRONG>Hazards</STRONG>: Highly flammable, irritant</P><H4>Materials</H4><H5>Reagents</H5><UL class="bullets"><LI>95% ethanol</LI><LI>Distilled water</LI></UL><H5>Equipment</H5><UL class="bullets"><LI>Wash bottle</LI><LI>Graduated cylinder</LI></UL><H4>Procedure</H4><P>This protocol results in 200mL of stock solution.</P><OL><LI>Obtain a graduated cylinder with 42.1mL dH<SUB>2</SUB>O (distilled water).</LI><LI>Obtain a graduated cylinder with 157.9mL ethanol.</LI><LI>Transfer 42.1mL dH<SUB>2</SUB>O into a wash bottle.</LI><LI>Transfer 157.9mL ethanol into the wash bottle, minimizing any splashing of the solution.</LI><LI>Close the wash bottle and swirl it in circular motion to mix</LI></OL><H4>Leaving the lab</H4><UL class="bullets"><LI>Clean dirty glassware, or at least set aside the glassware to be cleaned by a designated individual.</LI><LI>Wipe down your workspace.</LI><LI>Ensure that all materials have been returned to their places, and that the plates have been properly stored in the fridge.</LI><LI>Dispose of all disposable tubes and pipette tips used in biohazard containers.</LI><LI>Make sure your plates are labelled and put in a place they can be found.</LI></UL><H4>Acknowledgements</H4><P>Made by Katariina Jaenes (iGEM Toronto 2015).</P><P>Protocols of previous iGEM teams were used to make this guideline.</P><H3 class="text-red">Antibacterial Stock Preparation (starter pack)</H3><H4>Introduction</H4><P>The preparation of antibiotic stock is a relatively simple series of mixing and dilutions. The antibiotic stocks are typically made from four antibiotics: Ampicillin, Kanamycin, Tetracycline and Chloramphenicol. With the exception of chloramphenicol, the other antibiotics are light sensitive and once stock is prepared they are wrapped in foil to reduce light exposure.</P><P>An antibiotic comes in a powder form as the basic stock and is mixed with milli-Q water to form a stock solution. Antibiotic stock preparation follows the same basic process for each antibiotic, varying only in concentration which is dependent on the type of antibiotic used.</P><P>Antibiotic stocks are prepared for use either in liquid or solid media. In the case of liquid media, the antibiotic stock is simply added and mixed in. For solid media, usually agar and LB, the media is sent for autoclaving, and while the media is still hot the antibiotic is added and mixed. This is to ensure that the antibiotic can be mixed evenly amongst the media before it solidifies into a solid state.</P><P>Antibiotics are used for the purpose of selection. In terms of synthetic biology, plasmids confer selective antibiotic resistance when successfully transformed into their target bacteria. Using liquid or solid growth media that has been treated with antibiotics provides a selection control for those bacteria that have successfully incorporated the plasmid.</P><H4>Basic Terminology and Concepts</H4><P>There are some terms which will be commonly used to describe the preparation of antibiotic stocks. Familiarity and understanding of these terms is key to comprehend the protocol.</P><UL class="bullets"><LI><STRONG>Milli-Q</STRONG>: Water that has been purified through successive steps of filtration and deionization. The standard used in our lab is typically 18.2MΩ·cm at 25°C, measured in resistance due to the lack of ions. The filters used are 0.22μm in size to ensure a high level of purity. This water is used for preparing our antibacterial stock to ensure purity.</LI><LI><STRONG>LB (Lysogeny Broth)</STRONG>: a very standard and simple media to create due to its recipe consisting of 3 components - tryptone, yeast extract, and sodium chloride. This mix of anhydrous ingredients is added to water and then autoclaved, producing liquid media. The production of solid LB for plates is done by adding agar, a protein isolated from certain species of seaweed which coagulates the liquid into a gel-like form when cooled. This is done prior to autoclaving and is poured into the plates while still hot, where it will cool into the plate shape.</LI><LI><STRONG>Aliquot</STRONG>: An aliquot is a term to denote a certain quantity of something. In this case an aliquot of antibiotic stock would denote either 1mL or 0.5mL, depending on the antibiotic.</LI><LI><STRONG>Antibiotic</STRONG>: Compounds which inhibit bacterial growth. They act either bacteriostatically by preventing reproduction of the bacteria, or bacteriocidally where they directly kill the bacteria. Generally bacteriocides work by interfering with the synthesis of peptidoglycan in the bacteria’s cell walls. Tetracycline is an example of a bacteriostatic, where it acts by binding to the ribosomes of prokaryotic bacteria and inhibits translation.</LI><LI><STRONG>Autoclave</STRONG>: A piece of equipment used for sterilization. The autoclave performs much like a pressure cooker: it subjects the contents inside it to a high temperature and high pressure steam bath. Usually the temperature is 121°C and at 15lbs/in<SUP>2</SUP>, 20 minutes is enough to kill most microorganisms and render equipment sterile. When adding antibiotics to media, it is done after autoclaving so that the heat does not destroy the antibiotic activity.</LI></UL><H4>Safety precautions</H4><P><STRONG>SDS (safety data sheet)</STRONG>: Refer to the SDS sheets for all listed materials before entering the lab. Be prepared to answer any questions regarding th3e information on these sheets.</P><P><STRONG>PPE (Personal protective equipment)</STRONG>: Proper lab attire should be worn throughout the experiment: This means that upon entering the lab you should be wearing long pants and close-toed shoes. Contact lenses should not be worn. Furthermore, a lab coat, goggles, and gloves should be worn at all times, and long hair should be tied back.</P><P><STRONG>Hazards</STRONG>: Carcinogen (D2A), mutagen (D2B), Acute oral toxicity</P><H4>Materials</H4><H5>Reagents</H5><UL class="bullets"><LI>MilliQ water</LI><LI>4g Ampicillin</LI><LI>800mg Kanamycin</LI><LI>400mg Tetracycline</LI><LI>2.72g Chloramphenicol</LI><LI>80mL 70% EtOH</LI><LI>80mL 100% EtOH</LI></UL><H5>Equipment</H5><UL class="bullets"><LI>Antistatic weighing boat</LI><LI>100mL Pyrex bottle</LI><LI>Magnetic stir bar</LI><LI>Magnetic stirrer</LI><LI>Analytical balance</LI><LI>Milli-Q water dispenser</LI><LI>50mL Falcon tubes</LI><LI>Aluminum foil</LI><LI>20mL syringe</LI><LI>0.22μm Filter</LI><LI>1mL microcentrifuge tubes</LI></UL><H4>Ampicillin</H4><H5>Notes</H5><P>Stocks and Usage:</P><UL class="bullets"><LI>Stock concentration 50mg/mL in milliQ water</LI><LI>Aliquots: 500μL (use a P1000 set to 0500)</LI><LI>Working Concentration: 50μg/mL preparation of 80mL stock solution</LI></UL><P>Ampicillin is kept in the 4°C fridge, and is light sensitive. To ensure your stock solution is not degraded, cover all microcentrifuge tubes used for storing the solution with foil (you could use a falcon tube and wrap it with foil as well).</P><H5>Procedure</H5><OL><LI>Weigh 4g ampicillin onto an antistatic weighing boat.</LI><LI>Add 80mL milliQ water to a 100mL Pyrex bottle.</LI><LI>Add the ampicillin to the milliQ water.</LI><LI>Place a small magnetic stir bar into the solution and place the Pyrex bottle on the stirrer. Set at 300-600rpm and stir until dissolved.</LI><LI>Filter sterilise the solution into 50mL Falcon tubes using a 20mL syringe outfitted with a 0.22μm filter.</LI><LI>Aliquot into the appropriate microcentrifuge tubes, labelled with an “A” on top, and store in the Nalgene racks found in the 20°C fridge</LI></OL><H4>Kanamycin (KAN)</H4><H5>Notes</H5><P>Stocks and Usage:</P><UL class="bullets"><LI>Stock Concentration 10mg/mL in milliQ water</LI><LI>Aliquots: 1mL (use a P1000 set to 1000)</LI><LI>Working Concentration: 50μg/mL preparation of 80mL stock solution</LI><LI>Add 5mL of stock per litre of LB</LI></UL><P>Kanamycin is kept in the 4°C fridge, and is light sensitive. To ensure your stock solution is not degraded, cover all microcentrifuge tubes used for storing the solution with foil.</P><H5>Procedure</H5><OL><LI>Weigh 800mg kanamycin onto an antistatic weighing boat.</LI><LI>Add 80mL milliQ water to a 100mL Pyrex bottle.</LI><LI>Add the kanamycin to the milliQ.</LI><LI>Place a small magnetic stir bar into the solution and place the Pyrex bottle on the stirrer. Set at 300-600rpm and stir until dissolved.</LI><LI>Filter sterilise the solution into 50mL Falcon tubes using a 20mL syringe outfitted with a 0.22μm filter</LI><LI>Aliquot (1mL) into the appropriate microcentrifuge tubes, labelled with a “K” on top. Store in the Nalgene racks found in the 20°C fridge. (Make sure microcentrifuge tubes are covered with foil)</LI></OL><H4>Tetracycline</H4><H5>Notes</H5><P>Stocks and Usage:</P><UL class="bullets"><LI>Stock concentration 5mg/mL in 70% EtOH (N.B. <STRONG>70%</STRONG>, not 100% EtOH!)</LI><LI>Aliquots: 1mL (use a P1000 set to 1000)</LI><LI>Working Concentration: 20μg/mL preparation of 80mL stock solution</LI></UL><P>Tetracycline is kept in the 4°C fridge, and is light sensitive. To ensure your stock solution is not degraded, cover all microcentrifuge tubes used for storing the solution with foil.</P><H5>Procedure</H5><OL><LI>Weigh 400mg tetracycline onto an antistatic weighing boat.</LI><LI>Add 80mL 70% EtOH to a 100 mL Pyrex bottle.</LI><LI>Add the tetracycline to the 70% EtOH.</LI><LI>Place a small magnetic stir bar into the solution and place the Pyrex bottle on the stirrer. Set at 300-600rpm and stir until dissolved.</LI><LI>Recommended, but optional, because of storage in 70% EtOH: Filter sterilise the solution into 50mL Falcon tubes using a 20mL syringe outfitted with a 0.2μm filter.</LI><LI>Aliquot into the appropriate microcentrifuge tubes, labelled with a “T” on top. Store in the Nalgene racks found in the 20°C fridge.</LI></OL><H4>Chloramphenicol (CAM)</H4><H5>Notes</H5><P>Stocks and Usage:</P><UL class="bullets"><LI>Stock Concentration 34mg/mL in 100% EtOH (N.B. 100%, not 70%!)</LI><LI>Aliquots: 1mL (P1000 set to 1000)</LI><LI>Working Concentration: 25μL/mL preparation of 80mL stock solution</LI></UL><P>Chloramphenicol is kept with the general chemicals, and is not light sensitive. The microcentrifuge tubes do not need to be covered with foil to store chloramphenicol.</P><H5>Procedure</H5><OL><LI>Weigh 2.72g chloramphenicol onto an antistatic weighing boat.</LI><LI>Add 80mL 100% EtOH to a 100mL Pyrex bottle.</LI><LI>Add the chloramphenicol to the 100% EtOH.</LI><LI>Place a small magnetic stir bar into the solution and place the Pyrex bottle on the stirrer. Set at 300-600rpm and stir until dissolved.</LI><LI>Aliquot (1mL) into the appropriate microcentrifuge tubes, labelled with a “C” on top. Store in the Nalgene racks found in the 20°C fridge.</LI></OL><P><EM>Note: No filter sterilization is needed because it is stored in 100% EtOH.</EM></P><H4>Leaving the Lab</H4><P>Prior to leaving the lab, you should:</P><UL class="bullets"><LI>Clean dirty glassware, or at least set aside the glassware to be cleaned by a designated individual.</LI><LI>Wipe down your workspace.</LI><LI>Ensure that all materials have been returned to their places, and that the plates have been properly stored in the fridge.</LI></UL><H4>Acknowledgements</H4><P>Protocols of previous iGEM teams were used to make this guideline.</P><H3 class="text-red">Kayla’s M9 Media Preparation (starter pack)</H3><H4>Introduction</H4><P>Modified M9 media is used for Escherichia coli cultures. It consists of an additional nitrogen source that seems to help with enzyme expression of heterologous pathways. The following recipe uses 3% of final glucose concentration but this can be modified accordingly.</P><H4>Materials</H4><P>The following components need to be autoclaved/filter sterilized separately. When sterilizing components by filtration, ensure that you filter into a sterile (autoclaved) bottle or container.</P><UL class="bullets"><LI>5X Modified M9 Salts (per Litre):
			</LI><LI>64g Na<SUB>2</SUB>HPO<SUB>4</SUB>•7H<SUB>2</SUB>O (or 33.89g Na<SUB>2</SUB>HPO<SUB>4</SUB> Anhydrous)</LI><LI>15g KH<SUB>2</SUB>PO<SUB>4</SUB></LI><LI>2.5g NaCl</LI><LI>10.0g NH<SUB>4</SUB>Cl</LI><LI>5.0g (NH<SUB>4</SUB>)<SUB>2</SUB>SO<SUB>4</SUB></LI></UL><LI>100mL of 1M MgSO<SUB>4</SUB>:
			</LI><LI>24.65g MgSO<SUB>4</SUB>•7H<SUB>2</SUB>O</LI><LI>Water</LI><LI>100mL of 1M CaCl<SUB>2</SUB>:
			</LI><LI>14.70g CaCl<SUB>2</SUB>•2H<SUB>2</SUB>O</LI><LI>Water</LI><LI>1L of (10X) 1M MOPS:
			</LI><LI>209.3g MOPS</LI><LI>Deionized water</LI><LI>NaOH</LI><LI>Trace Metals (1000X) (per litre of stock):
			</LI><LI>1.6g FeCl<SUB>3</SUB></LI><LI>0.2g CoCl<SUB>2</SUB>•6H<SUB>2</SUB>O</LI><LI>0.1g CuCl<SUB>2</SUB></LI><LI>0.2g ZnCl<SUB>2</SUB>•4H<SUB>2</SUB>O</LI><LI>0.2g NaMoO<SUB>2</SUB></LI><LI>0.05g H<SUB>3</SUB>BO<SUB>3</SUB></LI><LI>50% Glucose:
			</LI><LI>500g D-glucose</LI><LI>Deionized water</LI><H4>Procedure</H4><H5>5X Modified M9 Salts</H5><OL><LI>Combine 64g Na<SUB>2</SUB>HPO<SUB>4</SUB>•7H<SUB>2</SUB>O (or 33.89g Na<SUB>2</SUB>HPO<SUB>4</SUB> Anhydrous), 15g KH<SUB>2</SUB>PO<SUB>4</SUB>, 2.5g NaCl, 10.0g NH<SUB>4</SUB>Cl, and 5.0g (NH<SUB>4</SUB>)<SUB>2</SUB>SO<SUB>4</SUB> per litre of stock</LI><LI>Autoclave or filter sterilize</LI></OL><H5>100mL of 1M MgSO<SUB>4</SUB></H5><OL><LI>Add 24.65 g of MgSO<SUB>4</SUB>•7H<SUB>2</SUB>O to 87mL water.</LI><LI>Make up volume to 100mL.</LI><LI>Autoclave.</LI></OL><H5>100mL of 1M CaCl<SUB>2</SUB></H5><OL><LI>Add 14.70g CaCl<SUB>2</SUB>•2H<SUB>2</SUB>O to 94.5mL water.</LI><LI>Make up volume to 100mL.</LI><LI>Autoclave.</LI></OL><H5>1L of (10X) 1M MOPS</H5><OL><LI>Add 209.3g of MOPS (free acid) to 800mL deionized water.</LI><LI>Adjust to pH 7 with NaOH.</LI><LI>When dissolved make up volume to 1L.</LI><LI>Filter through 0.22uM filter or can be autoclaved.</LI></OL><H5>Trace Metals (1000X)</H5><OL><LI>Prepare trace metal stock in 0.1 M HCl</LI><LI>Add 1.6g FeCl<SUB>3</SUB>, 0.2g CoCl<SUB>2</SUB>•6H<SUB>2</SUB>O, 0.1g CuCl<SUB>2</SUB>, 0.2g ZnCl<SUB>2</SUB>•4H<SUB>2</SUB>O, 0.2g NaMoO<SUB>2</SUB>, and 0.05g H<SUB>3</SUB>BO<SUB>3</SUB> per litre of stock desired</LI></OL><H5>50% Glucose</H5><OL><LI>Add 500g D-glucose to 500mL deionized water.</LI></OL><UL class="bullets"><LI>NOTE: You have to add glucose slowly to the water while mixing and heating, otherwise glucose will not fully dissolve.</LI></UL><OL start="2"><LI>Autoclave to sterilize.</LI></OL><H5>Preparation of final modified M9 media</H5><P><EM>Note that in our experiment, we will be making M9 supplemented with Arabinose, and thus will be making a 10% stock which we will dilute further to 0.2% for all assays.</EM></P><DIV class="figures"><TABLE class="table table-striped table-bordered"><THEAD><TR><TH>Component (stock concentration)</TH><TH>Amount to add per 1L stock (mL)</TH><TH>Amount to add per 500mL stock (mL)</TH><TH>Amount to add per 250mL stock (mL)</TH><TH>Amount to add per 200mL stock (mL)</TH><TH>Amount to add per 100mL stock (mL)</TH></TR></THEAD><TBODY class="table-hover"><TR><TD>5X Modified M9 salts</TD><TD>200</TD><TD>100</TD><TD>50</TD><TD>40</TD><TD>20</TD></TR><TR><TD>Glucose (50%)</TD><TD>60</TD><TD>30</TD><TD>15</TD><TD>12</TD><TD>6</TD></TR><TR><TD>MgSO<SUB>4</SUB> (1M)</TD><TD>1</TD><TD>0.5</TD><TD>0.25</TD><TD>0.2</TD><TD>0.1</TD></TR><TR><TD>CaCl<SUB>2</SUB> (1M)</TD><TD>0.1</TD><TD>0.05</TD><TD>0.025</TD><TD>0.02</TD><TD>0.01</TD></TR><TR><TD>Thiamine (0.05mg/mL)</TD><TD>1</TD><TD>0.5</TD><TD>0.25</TD><TD>0.2</TD><TD>0.1</TD></TR><TR><TD>MOPS (1M)</TD><TD>100</TD><TD>50</TD><TD>25</TD><TD>20</TD><TD>10</TD></TR><TR><TD>Trace Metals (1000X)</TD><TD>1</TD><TD>0.5</TD><TD>0.25</TD><TD>0.2</TD><TD>0.1</TD></TR><TR><TD>Deionized water (with MOPS/<EM>without MOPS</EM>)</TD><TD>636.9/<EM>736.9</EM></TD><TD>318.5/<EM>368.5</EM></TD><TD>159.3/<EM>184.3</EM></TD><TD>127.4/<EM>147.4</EM></TD><TD>63.7/<EM>73.7</EM></TD></TR></TBODY></TABLE></DIV><OL><LI>Add the components listed in the table above in the approrpiate amounts.</LI><LI>Thiamine is light and temperature sensitive; therefore, store the stock at 4°C. Trace metals are light sensitive; therefore, store the stock in an amber bottle. Once thiamine and trace metals are added to the media, store the media at 4°C.</LI></OL><P><EM>Note: Supplement with required antibiotic before use.</EM></P><H4>Acknowledgements</H4><P>1000X Trace Metals: Refer to Causey et al. (2003) PNAS</P><H3 class="text-red">LB media preperation (starter pack)</H3><H4>Introduction</H4><P>Note: This protocol makes 500mL of broth or ~25 plates.</P><P>In order for bacteria to be successfully cultured, they must be grown in the appropriate media. LB, also known as Lysogeny broth, is a nutrient rich broth that is a standard for culturing Escherichia coli, as it allows for quick growth and high yields. Therefore, the proper preparation of LB will be crucial to maintaining our bacterial stock throughout the summer. Furthermore, addition of agar to LB broth creates a gel for bacteria to grow upon, and is therefore used for plating bacterial cultures on petri dishes.</P><P>** WATCH THESE VIDEOS BEFORE ATTEMPTING THIS PROTOCOL**</P><UL class="bullets"><LI>Making LB agar: <A href="https://www.youtube.com/watch?v=74Ndr9AVowM">https://www.youtube.com/watch?v=74Ndr9AVowM</A></LI><LI>Pouring plates: <A href="https://www.youtube.com/watch?v=X4L16Vu5MSk">https://www.youtube.com/watch?v=X4L16Vu5MSk</A></LI></UL><H4>Basic Terminology and Concepts</H4><UL class="bullets"><LI>Agar vs. Agarose: Agar is used for making petri plates to culture organisms, while agarose is used for making gels, in the likes of SDS-PAGE and gel electrophoresis</LI></UL><H4>Safety Precautions</H4><P><STRONG>SDS (safety data sheet)</STRONG>: Refer to the SDS sheets for all listed materials before entering the lab. Be prepared to answer any questions regarding the information on these sheets.<STRONG>PPE (Personal protective equipment)</STRONG>: Proper lab attire should be worn throughout the experiment. This means that upon entering the lab you should be wearing long pants and close-toed shoes. Contact lenses should not be worn. Furthermore, a lab coat, goggles, and gloves should be worn at all times, and long hair should be tied back.<STRONG>Autoclave</STRONG>: The autoclave should only be handled by leads and managers. Note that any autoclaved materials may still be hot and should therefore be handled with caution. Be careful not to burn yourself.</P><H4>Materials</H4><H5>Reagents</H5><UL class="bullets"><LI>5g Bacto-tryptone</LI><LI>2.5g yeast extract</LI><LI>5 gNaCl</LI><LI>7.5g agar (Only necessary if making LB agar plates)</LI><LI>500mL of dH<SUB>2</SUB>O (distilled water)</LI></UL><H5>Equipment</H5><UL class="bullets"><LI>1L Pyrex bottle</LI><LI>1L graduated cylinder</LI><LI>Filter paper and scoopula</LI><LI>Stack of sterile plates (this protocol makes approximately 25)</LI><LI>Bunsen burner/ethanol burner</LI><LI>70% EtOH wash bottle</LI><LI>Paper towels/wipes</LI></UL><H4>Procedure</H4><H5>Part 1: Making the LB broth</H5><P>This part can be carried out at a regular lab bench.</P><OL><LI>Obtain a clean 1L pyrex bottle</LI><LI>Obtain a graduated cylinder with 500mL of dH<SUB>2</SUB>O and add to the bottle. Record the amount added.</LI><LI>Using filter paper, separately measure out 5g of NaCl, 5g of Tryptone, and 2.5g of yeast extract on a scale and add them to the bottle. Swirl the bottle in a circular motion to mix. Remember to recalibrate your scales in between measurements.</LI><LI>If you are making LB agar plates, weigh and add 7.5g of agar and swirl to mix. Record the amount added.</LI></OL><P>Note the contents do not necessarily need to be completely in solution before autoclaving.</P><H5>Part 2: Autoclaving</H5><OL><LI>Lightly seal the top of the beaker with aluminium foil, and label the beaker with autoclave tape stating LB (agar)–[your name]–[date]–[media number]–iGEM.</LI></OL><P><STRONG>Unless you have been trained to use the autoclave, you will not be conducting the following steps in this part</STRONG></P><OL start="2"><LI>Use appropriate transportation protocols to bring the LB bottle into the autoclave room. Remember to store the beaker in an autoclavable basin, in case of spills.</LI><LI>Check the water level on the autoclave, if necessary. Autoclave on the liquid setting for approximately 20 min.</LI></OL><UL class="bullets"><LI>The contents of the beaker will be hot after autoclaving, therefore take the necessary measures to prevent burns.</LI></UL><OL start="4"><LI>After autoclaving, allow the LB media to cool to 55°C before handling.
			</LI><LI>Use laser thermometer to check the temperature of the glass.</LI><LI>The LB broth can be stored in sterile conditions at room temperature, and should be good for 3-4 months. Flame the lip of the bottle each time the LB is used. If the LB contains antibiotics, store in a -4°C freezer.
			</LI><LI>However, it is not recommended to store LB with antibiotics as the antibiotics will degrade over time</LI></OL><H5>Part 3: Pouring the plates (for LB agar)</H5><P>While pouring the plates, it is crucial to maintain a sterile environment. This should be done in room WB 303, with a sterile environment provided by a lit Bunsen burner.</P><P><EM>Note: steps 1-3, in addition to the clean up from Part 1, can be done while waiting for autoclave.</EM></P><OL><LI>Sterilize the workspace with 70% EtOH before depositing your materials. Light the Bunsen burner.</LI><LI>Obtain a stack/roll of empty plates. The plates should still be in their plastic sleeve/wrapping, as they should be sterile. Don’t throw out the wrapping as it can be used to store the plates. It is essential that you minimize any chance of contaminating the plates. Make sure that you open the package at the top and expose the plates as minimally as possible.
			</LI><LI><EM>Note that this protocol makes ~25 plates.</EM></LI><LI>Once you take the plates out, store them upside down on your lab bench. Label the plates with [your name]–iGEM 2017–[date]–[media number]–[antibiotic stripe]. Once labelled, you may stack the plates to free up workspace.
			</LI><LI>One stripe along the sides corresponds to CAM, two stripes corresponds to AMP</LI><LI>Allow the LB media to cool before pouring. The LB will start to settle at ~30°C.</LI><LI>If you are preparing selective media, add antibiotic to the mixture. Swirl the flask in a circular motion to mix. If you don’t know whether or not you are preparing selective media, ASK.
			</LI><LI>Use concentrated liquid stocks for the antibiotics.</LI></OL><UL class="bullets"><LI>Recommenced antibiotic concentrations:
			</LI><LI>Chloramphenicol (CAM): 25μg/mL</LI><LI>Ampicillin (AMP): 100μg/mL</LI></UL><OL start="6"><LI>Take an empty plate and open it slightly. You do not need to open it all the way to pour the agar.</LI><LI>Pour agar until 2/3 of the plate has been covered, or approximately half of the plate has been filled when viewed from the side. Pour the agar slowly to prevent the formation of bubbles. Swirl the plate in a circular motion to distribute the media evenly on the plate.</LI></OL><UL class="bullets"><LI>If you pour too much LB, you will not be able to produce 25 plates. If you don’t pour enough media, it may minimize bacterial growth.</LI></UL><OL start="8"><LI>After pouring, set the plates to cool in stacks of 4-5 to save space and flip the plates to prevent condensation forming on the agar. Don’t stack plates too high - we want to minimize the risk of spills. Allow the plates to cool for at least 20 minutes until the agar has solidified.</LI><LI>Rinse the Pyrex bottle with water before the remnants solidify and become hard to remove.</LI><LI>The plates can then be stacked and stored in plastic bags (ideally, re-use the plastic bags that the plates came in.)</LI><LI>Store LB agar plates in a 4°C freezer. They should be good for 1-2 months.</LI></OL><H4>Leaving the lab</H4><P>Prior to leaving the lab, you should:</P><UL class="bullets"><LI>Clean dirty glassware, or at least set aside the glassware to be cleaned by a designated individual.</LI><LI>Wipe down your workspace.</LI><LI>Ensure that all materials have been returned to their places, and that the plates have been properly stored in the fridge.</LI></UL><H4>Acknowledgements</H4><UL class="bullets"><LI>Addgene: Protocol – Making LB Agar Plates for Bacteria. From: <A href="https://www.addgene.org/plasmid-protocols/bacterial-plates">https://www.addgene.org/plasmid-protocols/bacterial-plates</A></LI><LI>Department of Ecology and Evolutionary Biology, UCLA. Making LB Agar Plates. From: <A href="https://www.eeb.ucla.edu/Faculty/Barber/Web%20Protocols/LB%20Agar%20Plates.pdf">https://www.eeb.ucla.edu/Faculty/Barber/Web Protocols/LB Agar Plates.pdf</A></LI><LI>iGEM Toronto Luria Broth (LB) protocol</LI><LI>Sezonov G, Joseleau-Petit D, D’Ari R. Escherichia coli Physiology in Luria-Bertani Broth. Journal of Bacteriology. 2007;189(23):8746-8749. doi:10.1128/JB.01368-07.</LI></UL><H3 class="text-red">SOC Medium Protocol (starter pack)</H3><H4>Introduction</H4><P>SOC is a variant of the rich media SOB (super optimal broth) with catabolite repression. This means that glucose is supplemented in the media, allowing for optimal metabolic conditions for the bacteria. SOC increases the transformation efficiency of cells, as it provides ample nutrients to cells that have recently undergone stress as result of having been made competent. Accordingly, it will be used in bacterial transformation to stabilize the cells and to increase transformation yields. Since SOC is high in nutrients, it is more easily contaminated than LB or King’s B media.</P><H4>Safety Precautions</H4><P><STRONG>SDS (safety data sheet)</STRONG>: Refer to the SDS sheets for all listed materials before entering the lab. Be prepared to answer any questions regarding the information on these sheets.</P><P><STRONG>PPE (Personal protective equipment)</STRONG>: Proper lab attire should be worn throughout the experiment: This means that upon entering the lab you should be wearing long pants and close-toed shoes. Contact lenses should not be worn. Furthermore, a lab coat, goggles, and gloves should be worn at all times, and long hair should be tied back.</P><P><STRONG>Autoclave</STRONG>: The autoclave should only be handled by execs. <STRONG>Note that any autoclaved materials may still be hot and should therefore be handled with caution. Be careful not to burn yourself.</STRONG></P><H4>Materials</H4><H5>Reagents</H5><UL class="bullets"><LI>1.802g glucose</LI><LI>10g tryptone</LI><LI>2.5g Yeast extract</LI><LI>0.584g NaCl</LI><LI>0.093g KCl</LI><LI>1.016g MgCl<SUB>2</SUB> (anhydrous)</LI><LI>1.234g MgSO<SUB>4</SUB>•7H<SUB>2</SUB>O</LI><LI>500mL dH<SUB>2</SUB>O (distilled water)</LI></UL><H5>Equipment</H5><UL class="bullets"><LI>2×1L pyrex bottle (must have cap)
			</LI><LI>Note: A smaller pyrex bottle may be used to accommodate the glucose solution</LI></UL><LI>1L graduated cylinder</LI><LI>Weighing boats and scoopula</LI><LI>70% EtOH wash bottle</LI><LI>Paper towels/wipes</LI><H4>Procedure</H4><H5>Part 1: Making the SOC broth</H5><P>This step can be carried out at a regular lab bench.</P><OL><LI>Obtain two 1L pyrex bottles. Ensure that the bottles can be sealed with a cap - this will help prevent contamination and enable long term storage.</LI><LI>Obtain a graduated cylinder with 500mL dH<SUB>2</SUB>O (distilled water).</LI><LI>Using filter paper, separately measure out 10g tryptone, 2.5g yeast extract, 0.292g NaCl, 0.093g KCl, 1.016g MgCl<SUB>2</SUB> anhydrate, and 1.234g MgSO<SUB>4</SUB>•7H<SUB>2</SUB>O on a scale and add them to the 1L bottle.
			</LI><LI>Remember to recalibrate your scales in between measurements.</LI><LI>Add 400mL dH<SUB>2</SUB>O. Swirl the flask in a circular motion to mix.</LI><LI>In the separate bottle, and measure out 1.802g glucose on a scale and add the rest of the 100 mL dH<SUB>2</SUB>0. Recalibrate your scales in between measurements.</LI></OL><P><EM>Note: The 1 L bottles are autoclaved separately, as the contents will react if autoclaved together.</EM></P><H5>Autoclaving</H5><P><STRONG>ONLY DONE BY EXECS</STRONG></P><OL><LI>Lightly seal the top of the bottles with aluminium foil, or unscrew the caps. Label both with autoclave tape. Include a label with SOC – [your name] – [date] - iGEM 2017.</LI></OL><UL class="bullets"><LI>**Unless you have been trained to use the autoclave, you will not be conducting the following steps 2-4. **</LI></UL><OL start="2"><LI>Use appropriate transportation protocols to bring the bottles into the autoclave room. Remember to store the beaker in an autoclavable basin, in case of spills.</LI><LI>Check the water level on the autoclave, if necessary. Autoclave on the liquid setting for approximately 20 min.</LI><LI><STRONG>The contents of the bottles will be hot after autoclaving, so take necessary measures to prevent burns.</STRONG> After autoclaving, allow the media too cool to 55°C before handling.</LI></OL><H5>Part 2: Making the SOC broth</H5><P>This part should be done in room WB303, in sterile conditions close to a Bunsen burner.</P><OL><LI>In a sterile environment, slowly add the autoclaved 1.802g glucose and dH<SUB>2</SUB>O to the beaker containing the autoclaved 1.016g MgCl<SUB>2</SUB> and 1.234 g MgSO<SUB>4</SUB>, 10g tryptone, 2.5g yeast extract, 0.292g NaCl, 0.093g KCl, and dH<SUB>2</SUB>0.</LI></OL><UL class="bullets"><LI>Flame the lip of the bottle before transferring the contents. Swirl to mix, and seal tightly to prevent contamination. Flame the cap before sealing.</LI></UL><OL start="2"><LI>The SOC broth can be stored in sterile conditions at room temperature, and should be good for a 3-4 months. Flame the lip of the bottle each time the SOC is used. SOC should be handled carefully, as it is especially prone to contamination.</LI></OL><H4>Before leaving the lab</H4><UL class="bullets"><LI>Clean dirty glassware, or at least set aside the glassware to be cleaned by a designated individual.</LI><LI>Wipe down your workspace.</LI><LI>Ensure that all materials have been returned to their places, and that the plates have been properly stored in the fridge.</LI></UL><H4>Acknowledgements</H4><UL class="bullets"><LI>Cold Spring Harbour Protocols. SOC Medium for E.coli. From: <A href="http://cshprotocols.cshlp.org/content/2012/6/pdb.rec069732.short">http://cshprotocols.cshlp.org/content/2012/6/pdb.rec069732.short</A></LI><LI>Sun QY et al. Culture of Escherichia coli in SOC medium improves the cloning efficiency of toxic protein genes. Anal Biochem. 2009: From <A href="https://www.ncbi.nlm.nih.gov/pubmed/19622338">http://www.ncbi.nlm.nih.gov/pubmed/19622338</A></LI><LI>Qiagen. Recipe for SOC Medium. From: <A href="https://www.qiagen.com/ca/resources/faq?id=d3be05d0-ec02-4ced-aabe-dcd2a1061920&amp;lang=en">https://www.qiagen.com/ca/resources/faq?id=d3be05d0-ec02-4ced-aabe-dcd2a1061920&amp;lang=en</A></LI><LI>Protocol by Katariina Jaenes</LI></UL></DIV><DIV id="PCR_412" class="subsection"><H2>PCR</H2><H3 class="text-red">(M0530) NEB Phusion HF PCR (PCR)</H3><H4>Safety Precautions</H4><P><STRONG>SDS (safety data sheet)</STRONG>: Refer to the SDS sheets for all listed materials before entering the lab. Be prepared to answer any questions regarding the information on these sheets.</P><P><STRONG>PPE (Personal protective equipment)</STRONG>: Proper lab attire should be worn throughout the experiment. This means that upon entering the lab you should be wearing long pants and close-toed shoes. Contact lenses should not be worn. Furthermore, a lab coat, goggles, and gloves should be worn at all times, and long hair should be tied back.</P><H4>Materials</H4><H5>Reaction Component Table</H5><DIV class="figures"><TABLE class="table table-striped table-bordered"><THEAD><TR><TH>Component</TH><TH>20µL Reaction</TH><TH>50µL Reaction</TH><TH>Final Concentration</TH></TR></THEAD><TBODY class="table-hover"><TR><TD>Nuclease-free water</TD><TD>to 20µL</TD><TD>to 50µL</TD><TD>-</TD></TR><TR><TD>5X Phusion HF or GC Buffer</TD><TD>4µL</TD><TD>10µL</TD><TD>1X</TD></TR><TR><TD>10mM dNTPs</TD><TD>0.4µL</TD><TD>1µL</TD><TD>200 µM</TD></TR><TR><TD>10µM Forward Primer</TD><TD>1µL</TD><TD>2.5µL</TD><TD>0.5µM</TD></TR><TR><TD>10µM Reverse Primer</TD><TD>1µL</TD><TD>2.5µL</TD><TD>0.5µM</TD></TR><TR><TD>Template DNA</TD><TD>Variable</TD><TD>Variable</TD><TD>&lt; 250 ng</TD></TR><TR><TD>DMSO (optional)</TD><TD>0.6µL</TD><TD>1.5µL</TD><TD>3%</TD></TR><TR><TD>Phusion DNA Polymerase</TD><TD>0.2µL</TD><TD>0.5µL</TD><TD>1.0 units/50µL PCR</TD></TR></TBODY></TABLE></DIV><H5>Templates</H5><P>Use of high quality, purified DNA templates greatly enhances the success of PCR. Recommended amounts of DNA template for a 50μL reaction are as follows:</P><UL class="bullets"><LI>Genomic DNA: 50-250ng</LI><LI>Plasmid or viral DNA: 1pg-10ng</LI></UL><H5>Equipment</H5><UL class="bullets"><LI>PCR Thermocycler</LI><LI>Ice</LI><LI>Centrifuge</LI><LI>PCR tubes</LI></UL><H4>Protocol</H4><P><EM>Please note that protocols with Phusion DNA Polymerase may differ from protocols with other standard polymerases. As such, conditions recommended below should be used for optimal performance.</EM></P><OL><LI>Preheat a thermocycler to the denaturation temperature (98°C)</LI><LI>Mix and centrifuge all components prior to use.</LI></OL><UL class="bullets"><LI>Note: Phusion DNA Polymerase may be diluted in 1X HF or GC Buffer just prior to use in order to reduce pipetting errors.</LI></UL><OL start="3"><LI>Assemble all reaction components on ice. Gently mix the reaction, collecting all liquid to the bottom of the tube by a quick spin if necessary.</LI></OL><UL class="bullets"><LI>Note: It is important to add Phusion DNA Polymerase last in order to prevent any primer degradation caused by the 3´→ 5´ exonuclease activity.</LI><LI>Note: Overlay the sample with mineral oil if using a PCR machine without a heated lid.</LI></UL><OL start="3"><LI>Quickly transfer PCR tubes from ice to PCR machine previously preheated to 98°C and begin thermocycling with the following conditions:</LI></OL><H5>Thermocycling conditions:</H5><DIV class="figures"><TABLE class="table table-striped table-bordered"><THEAD><TR><TH>Step</TH><TH>Temperature</TH><TH>Time</TH></TR></THEAD><TBODY class="table-hover"><TR><TD>Initial Denaturation</TD><TD>98°C</TD><TD>30 seconds</TD></TR><TR><TD>25-35 cycles</TD><TD>Denaturation</TD><TD>98°C</TD><TD>5-10 seconds</TD></TR><TR><TD>Annealing</TD><TD>45-72°C</TD><TD>10-30 seconds</TD></TR><TR><TD>Extension</TD><TD>72°C</TD><TD>15-30 seconds/kb</TD></TR><TR><TD>Final Extension</TD><TD>72°C</TD><TD>5-10 minutes</TD></TR><TR><TD>Hold</TD><TD>4-10°C</TD><TD>-</TD></TR></TBODY></TABLE></DIV><H4>Leaving the lab</H4><P>Prior to leaving the lab, you should:</P><UL class="bullets"><LI>Clean dirty glassware, or at least set aside the glassware to be cleaned by a designated individual.</LI><LI>Wipe down your workspace.</LI><LI>Ensure that all materials have been returned to their places, and that the plates have been properly stored in the fridge.</LI></UL><H4>Acknowledgements</H4><P>This protocol was sourced from NEB.</P><H3 class="text-red">NEB OneTaq Hot Start PCR (M0481) (PCR)</H3><H4>Introduction</H4><P><STRONG>Note: When using OneTaq for colony PCR, an excess of cells will inhibit the reaction. Take a sterile loop and <EM>touch</EM> it to a colony, then mix this colony in ~1mL nuclease-free water and from this dilution take 1µl for your PCR reaction.</STRONG></P><H4>Safety Precautions</H4><P><STRONG>SDS (safety data sheet)</STRONG>: Refer to the SDS sheets for all listed materials before entering the lab. Be prepared to answer any questions regarding the information on these sheets.</P><P><STRONG>PPE (Personal protective equipment)</STRONG>: Proper lab attire should be worn throughout the experiment. This means that upon entering the lab you should be wearing long pants and close-toed shoes. Contact lenses should not be worn. Furthermore, a lab coat, goggles, and gloves should be worn at all times, and long hair should be tied back.</P><H4>Materials</H4><H5>Reaction Component Table</H5><DIV class="figures"><TABLE class="table table-striped table-bordered"><THEAD><TR><TH>Component</TH><TH>25μL reaction</TH><TH>50μL reaction</TH><TH>Final Concentration</TH></TR></THEAD><TBODY class="table-hover"><TR><TD>5X OneTaq Standard Reaction Buffer*</TD><TD>5µl</TD><TD>10μL</TD><TD>1X</TD></TR><TR><TD>10mM dNTPs (#N0447)</TD><TD>0.5µL</TD><TD>1µL</TD><TD>200µM</TD></TR><TR><TD>10µM Forward Primer</TD><TD>0.5µL</TD><TD>1µL</TD><TD>0.2µM</TD></TR><TR><TD>10µM Reverse Primer</TD><TD>0.5µL</TD><TD>1µL</TD><TD>0.2µM</TD></TR><TR><TD>OneTaq Hot Start DNA Polymerase</TD><TD>0.125µL</TD><TD>0.25µL</TD><TD>1.25 units/50µL PCR**</TD></TR><TR><TD>Template DNA</TD><TD>Variable</TD><TD>Variable</TD><TD>&lt; 1000ng</TD></TR><TR><TD>Nuclease-free water</TD><TD>To 25µL</TD><TD>To 50µL</TD><TD>-</TD></TR></TBODY></TABLE></DIV><P>*OneTaq GC Reaction Buffer and High GC Enhancer can be used for difficult amplicons.
			**For amplicons between 3–6kb, use 2.5–5 units/50µl reaction</P><H5>Templates</H5><P>Use of high quality, purified DNA templates greatly enhances the success of PCR. Recommended amounts of DNA template for a 50µl reaction are as follows:</P><UL class="bullets"><LI>Genomic DNA: 1ng-1µg</LI><LI>Plasmid or viral DNA: 1 pg–1 ng</LI></UL><H5>Equipment</H5><UL class="bullets"><LI>Thermocycler</LI><LI>PCR tubes</LI></UL><H4>Procedure</H4><H5>Reaction Setup:</H5><P><EM>Due to the presence of the inhibitor, reactions can be assembled on the bench at room temperature and transferred to a thermocycler. No separate activation step is required to release the inhibitor from the enzyme.</EM></P><OL><LI>Preheat a thermocycler to the denaturation temperature (94°C)</LI><LI>Mix and centrifuge all components prior to use.</LI><LI>Assemble all reaction components. Gently mix the reaction, collecting all liquid to the bottom of the tube by a quick spin if necessary.</LI></OL><UL class="bullets"><LI>Note: Overlay the sample with mineral oil if using a PCR machine without a heated lid.</LI></UL><OL start="3"><LI>Quickly transfer PCR tubes from ice to PCR machine previously preheated to 98°C and begin thermocycling.</LI></OL><H5>Thermocycling conditions:</H5><DIV class="figures"><TABLE class="table table-striped table-bordered"><THEAD><TR><TH>Step</TH><TH>Temperature</TH><TH>Time</TH></TR></THEAD><TBODY class="table-hover"><TR><TD>Initial Denaturation</TD><TD>94°C</TD><TD>30 seconds</TD></TR><TR><TD>30 cycles</TD><TD>Denaturation</TD><TD>94°C</TD><TD>15-20 seconds</TD></TR><TR><TD>Annealing</TD><TD>45-68°C</TD><TD>15-60 seconds</TD></TR><TR><TD>Extension</TD><TD>68°C</TD><TD>1 min/kb</TD></TR><TR><TD>Final Extension</TD><TD>68°C</TD><TD>5 min</TD></TR><TR><TD>Hold</TD><TD>4-10°C</TD></TR></TBODY></TABLE></DIV><H4>Leaving the lab</H4><P>Prior to leaving the lab, you should:</P><UL class="bullets"><LI>Clean dirty glassware, or at least set aside the glassware to be cleaned by a designated individual.</LI><LI>Wipe down your workspace.</LI><LI>Ensure that all materials have been returned to their places, and that the plates have been properly stored in the fridge.</LI></UL><H4>Acknowledgements</H4><P>This protocol was sourced from NEB.</P><H3 class="text-red">Q5 (High Fidelity) PCR (PCR)</H3><H4>Introduction</H4><H5>General Guidelines</H5><P><STRONG>Primers</STRONG>: Oligonucleotide primers are generally 20–40 nucleotides in length and ideally have a GC content of 40–60%. Computer programs such as <A href="http://bioinfo.ut.ee/primer3/">Primer3</A> can be used to design or analyze primers. The best results are typically seen when using each primer at a final concentration of 0.5 µM in the reaction.</P><P><STRONG>Mg<SUP>2+</SUP> and additives</STRONG>: Mg<SUP>2+</SUP> concentration of 2.0mM is optimal for most PCR products generated with Q5 High-Fidelity DNA Polymerase. When used at a final concentration of 1X, the Q5 Reaction Buffer provides the optimal Mg<SUP>2+</SUP> concentration.</P><P><STRONG>Deoxynucleotides</STRONG>: The final concentration of dNTPs is typically 200μM of each deoxynucleotide. Q5 High-Fidelity DNA Polymerase cannot incorporate dUTP and is not recommended for use with uracil-containing primers or templates.</P><P><STRONG>Q5 High-Fidelity DNA Polymerase concentration</STRONG>: We generally recommend using Q5 High-Fidelity DNA Polymerase at a final concentration of 20units/mL (1.0 units/50μL reaction). However, the optimal concentration of Q5 High-Fidelity DNA Polymerase may vary from 10–40 units/mL (0.5–2 units/50 μL reaction) depending on amplicon length and difficulty. Do not exceed 2 units/50 μL reaction, especially for amplicons longer than 5 kb.</P><P><STRONG>Buffers</STRONG>: The 5X Q5 Reaction Buffer provided with the enzyme is recommended as the first-choice buffer for robust, high-fidelity amplification. The 5X Q5 Reaction Buffer is detergent-free and contains 2.0 mM Mg++ at the final (1X) concentration.</P><UL class="bullets"><LI>Amplification of some difficult targets, like GC-rich sequences or secondary structures, may be improved by the addition of 1X Q5 High GC Enhancer. The Q5 High GC Enhancer is not a buffer and should not be used alone. It should be added only to reactions with the Q5 Reaction Buffer when other conditions have failed.</LI></UL><P><STRONG>Denaturation</STRONG>: An initial denaturation of 30 seconds at 98°C is sufficient for most amplicons from pure DNA templates. Longer denaturation times can be used (up to 3 minutes) for templates that require it.</P><UL class="bullets"><LI>During thermocycling, the denaturation step should be kept to a minimum. Typically, a 5–10 second denaturation at 98°C is recommended for most templates.</LI></UL><P><STRONG>Annealing</STRONG>: Optimal annealing temperatures for Q5 High-Fidelity DNA Polymerase tend to be higher than for other PCR polymerases. The <A href="https://www.neb.com/tools-and-resources/interactive-tools/tm-calculator_">NEB T<SUB>m</SUB> Calculator</A> should be used to determine the annealing temperature when using this enzyme. Typically, use a 10–30 second annealing step at 3°C above the T<SUB>m</SUB> of the lower T<SUB>m</SUB> primer. A temperature gradient can also be used to optimize the annealing temperature for each primer pair.</P><UL class="bullets"><LI>For high Tm primer pairs, two-step cycling without a separate annealing step can be used.</LI></UL><P><STRONG>Extension</STRONG>: The recommended extension temperature is 72°C. Extension times are generally 20–30 seconds per kb for complex, genomic samples, but can be reduced to 10 seconds per kb for simple templates (plasmid, <EM>E. coli</EM>, etc.) or complex templates &lt; 1 kb. Extension time can be increased to 40 seconds per kb for cDNA or long, complex templates, if necessary.</P><UL class="bullets"><LI>A final extension of 2 minutes at 72°C is recommended.</LI></UL><P><STRONG>Cycle number</STRONG>: Generally, 25–35 cycles yield sufficient product. For genomic amplicons, 30-35 cycles are recommended.</P><P><STRONG>2-step PCR</STRONG>: When primers with annealing temperatures ≥ 72°C are used, a 2-step thermocycling protocol (combining annealing and extension into one step) is possible.</P><P><STRONG>Amplification of long products</STRONG>: When amplifying products &gt; 6 kb, it is often helpful to increase the extension time to 40–50 seconds/kb.</P><P><STRONG>PCR product</STRONG>: The PCR products generated using Q5 High-Fidelity DNA Polymerase have blunt ends. If cloning is the next step, then blunt-end cloning is recommended. If T/A-cloning is preferred, the DNA should be purified prior to A-addition, as Q5 High-Fidelity DNA Polymerase will degrade any overhangs generated.</P><H5>Additives:</H5><P><STRONG>DMSO</STRONG>: Dimethyl sulfoxide (DMSO) is an organosulfur compound with a high polarity and high dielectric constant used in PCR to disrupt secondary structure formation in the DNA template. DMSO is believed to hydrogen bond to the major and minor grooves of template DNA, and as a result destabilizes the double helix structure. This is particularly useful in templates with high GC content because the increased hydrogen bond strength increases the difficulty of denaturing the template and causes intermolecular secondary structures to form more readily, which can compete with primer annealing. Thus, the addition of DMSO can greatly improve yields and specificities of PCR priming reactions, test a variety of DMSO concentrations between 2-10%</P><P><STRONG>Glycerol</STRONG>: Similarly reduces secondary structure. Use 5-10%.</P><P><STRONG>Formamide</STRONG>: Formamide is a widely used organic PCR additive. Formamide is thought to work by binding in the major and minor grooves of DNA, destabilizing the template double-helix and lower melting temperature. Formamide is usually used at 1-5%.</P><H4>Safety Precautions</H4><P><STRONG>SDS (safety data sheet)</STRONG>: Refer to the SDS sheets for all listed materials before entering the lab. Be prepared to answer any questions regarding the information on these sheets.</P><P><STRONG>PPE (Personal protective equipment)</STRONG>: Proper lab attire should be worn throughout the experiment. This means that upon entering the lab you should be wearing long pants and close-toed shoes. Contact lenses should not be worn. Furthermore, a lab coat, goggles, and gloves should be worn at all times, and long hair should be tied back.</P><H4>Materials</H4><H5>Reaction Component Table</H5><DIV class="figures"><TABLE class="table table-striped table-bordered"><THEAD><TR><TH>Component</TH><TH>25µL Reaction</TH><TH>50µL Reaction</TH><TH>Final Concentration</TH></TR></THEAD><TBODY class="table-hover"><TR><TD>5X Q5 Reaction Buffer</TD><TD>5µL</TD><TD>10µL</TD><TD>1X</TD></TR><TR><TD>10mM dNTPs</TD><TD>0.5µL</TD><TD>1µL</TD><TD>200µM</TD></TR><TR><TD>10µM Forward Primer</TD><TD>1.25µL</TD><TD>2.5µL</TD><TD>0.5µM</TD></TR><TR><TD>10µM Reverse Primer</TD><TD>1.25µL</TD><TD>2.5µL</TD><TD>0.5µM</TD></TR><TR><TD>Template DNA</TD><TD>Variable</TD><TD>Variable</TD><TD>&lt; 1000ng</TD></TR><TR><TD>Q5 High-Fidelity DNA Polymerase</TD><TD>0.25µL</TD><TD>0.5µL</TD><TD>0.02 units/µL</TD></TR><TR><TD>5X Q5 High GC Enhancer (optional)</TD><TD>5µL</TD><TD>10µL</TD><TD>1X</TD></TR><TR><TD>Nuclease-free water</TD><TD>To 25µL</TD><TD>To 50µL</TD><TD>-</TD></TR></TBODY></TABLE></DIV><H5>Templates</H5><P>Use of high quality, purified DNA templates greatly enhances the success of PCR. Recommended amounts of DNA template for a 50µl reaction are as follows:</P><UL class="bullets"><LI>Genomic DNA: 1ng-1µg</LI><LI>Plasmid or viral DNA: 1pg-1ng</LI></UL><H5>Equipment</H5><UL class="bullets"><LI>Thermocycler</LI><LI>PCR tubes</LI></UL><H4>Procedure</H4><OL><LI>Preheat a thermocycler to the denaturation temperature (98°C)</LI><LI>All components should be mixed prior to use.</LI></OL><UL class="bullets"><LI>Q5 High-Fidelity DNA Polymerase may be diluted in 1X Q5 Reaction Buffer just prior to use in order to reduce pipetting errors.</LI></UL><OL start="2"><LI>Assemble all reaction components in PCR tubes on ice. Gently mix the reaction.</LI></OL><UL class="bullets"><LI>Collect all liquid to the bottom of the tube by a quick spin if necessary.</LI><LI>Overlay the sample with mineral oil if using a PCR machine without a heated lid. (BioZone PCR machine has heated lid)</LI></UL><OL start="3"><LI>Quickly transfer the PCR tubes to preheated PCR machine and begin thermocycling.</LI></OL><H5>Thermocycling conditions:</H5><DIV class="figures"><TABLE class="table table-striped table-bordered"><THEAD><TR><TH>Step</TH><TH>Temperature</TH><TH>Duration</TH></TR></THEAD><TBODY class="table-hover"><TR><TD>Initial Denaturation</TD><TD>98°C</TD><TD>30 seconds</TD></TR><TR><TD>25-35 cycles</TD><TD>Denaturation</TD><TD>98°C</TD><TD>5-10 seconds</TD></TR><TR><TD>Annealing</TD><TD>50-72°C*</TD><TD>10-30 seconds</TD></TR><TR><TD>Extension</TD><TD>72°C</TD><TD>20-30 seconds/kb</TD></TR><TR><TD>Final Extension</TD><TD>72°C</TD><TD>2 min</TD></TR><TR><TD>Hold</TD><TD>4-10°C</TD></TR></TBODY></TABLE></DIV><P>*Use of the <A href="https://www.neb.com/tools-and-resources/interactive-tools/tm-calculator_">NEB T<SUB>m</SUB> Calculator</A> is highly recommended.</P><H4>Leaving the lab</H4><P>Prior to leaving the lab, you should:</P><UL class="bullets"><LI>Clean dirty glassware, or at least set aside the glassware to be cleaned by a designated individual.</LI><LI>Wipe down your workspace.</LI><LI>Ensure that all materials have been returned to their places, and that the plates have been properly stored in the fridge.</LI></UL><H4>Acknowledgements</H4><P>This protocol was sourced from NEB: <A href="https://www.neb.com/protocols/2013/12/13/pcr-using-q5-high-fidelity-dna-polymerase-m0491">https://www.neb.com/protocols/2013/12/13/pcr-using-q5-high-fidelity-dna-polymerase-m0491</A></P></DIV><DIV id="Purification_633" class="subsection"><H2>Purification</H2><H3 class="text-red">Dpn1 Digestion (Purification)</H3><H4>Introduction</H4><P>When higher amounts of plasmid template must be used in the PCR reaction, it is recommended that the PCR product be digested with Dpn1 (NEB #R0176) in order to destroy the plasmid template before setting up the assembly reaction. Dpn1 cleaves only <EM>E. coli</EM> Dam methylase-methylated plasmid DNA, but does not cleave the PCR product, since it is not methylated.</P><H4>Basic Terminology and Concepts</H4><H4>Safety Precautions</H4><P><STRONG>SDS (safety data sheet)</STRONG>: Refer to the SDS sheets for all listed materials before entering the lab. Be prepared to answer any questions regarding the information on these sheets.</P><P><STRONG>PPE (Personal protective equipment)</STRONG>: Proper lab attire should be worn throughout the experiment. This means that upon entering the lab you should be wearing long pants and close-toed shoes. Contact lenses should not be worn. Furthermore, a lab coat, goggles, and gloves should be worn at all times, and long hair should be tied back.</P><P><STRONG>Autoclave</STRONG>: The autoclave should only be handled by leads and managers. Note that any autoclaved materials may still be hot and should therefore be handled with caution. Be careful not to burn yourself.</P><H4>Materials</H4><H5>Reagents</H5><UL class="bullets"><LI>PCR product</LI><LI>10X CutSmart™ Buffer</LI><LI>Dpn1</LI></UL><H5>Equipment</H5><UL class="bullets"><LI>Incubator</LI></UL><H4>Procedure</H4><OL><LI>In a 10μL reaction, mix 5–8μL of PCR product with 1μL of 10X CutSmart™ Buffer and 1μL (20 units) of Dpn1.</LI><LI>Incubate at 37°C for 30 minutes.</LI><LI>Heat-inactivate Dpn1 by incubating at 80°C for 20 minutes.</LI><LI>Proceed with the NEBuilder HiFi DNA Assembly Protocol.</LI></OL><H4>Leaving the lab</H4><P>Prior to leaving the lab, you should:</P><UL class="bullets"><LI>Clean dirty glassware, or at least set aside the glassware to be cleaned by a designated individual.</LI><LI>Wipe down your workspace.</LI><LI>Ensure that all materials have been returned to their places, and that the plates have been properly stored in the fridge.</LI></UL><H4>Acknowledgements</H4><P>This protocol was sourced from NEBuilder® HiFi DNA Assembly Master Mix/NEBuilder HiFi DNA Assembly Cloning Kit.</P><H3 class="text-red">EZ-10 Spin Column PCR Products Purification Kit (Purification)</H3><H4>Introduction</H4><P>EZ-10 spin column purification kits use a silica-gel membrane that selectively absorbs up to 10μg of DNA fragments in the presence of specialized binding buffers. Nucleotides, oligos (&lt; 40-mer), enzymes, mineral oil, and other impurities do not bind to the membrane and are washed away. The DNA fragments can then be eluted off the column in small volume and used in downstream applications without further processing.</P><P>Note: If the PCR reaction mixture contains seriously non-specific amplified DNA fragments, use of the DNA Gel Extraction Kit is recommended.</P><P>This kit can not remove the template and primers with chain length longer than 40-mer.</P><H4>Safety Precautions</H4><P><STRONG>SDS (safety data sheet)</STRONG>: Refer to the SDS sheets for all listed materials before entering the lab. Be prepared to answer any questions regarding the information on these sheets.</P><P><STRONG>PPE (Personal protective equipment)</STRONG>: Proper lab attire should be worn throughout the experiment. This means that upon entering the lab you should be wearing long pants and close-toed shoes. Contact lenses should not be worn. Furthermore, a lab coat, goggles, and gloves should be worn at all times, and long hair should be tied back.</P><H4>Materials</H4><H5>Reagents</H5><P>For BS363 50 Preps:</P><UL class="bullets"><LI>24mL Buffer B3 with isopropanol added</LI><LI>20mL Wash Solution with ethanol added</LI><LI>Elution Buffer</LI></UL><H5>Equipment</H5><UL class="bullets"><LI>50 EZ-10 Column</LI><LI>1.5mL microfuge tube</LI></UL><H4>Procedure</H4><OL><LI>Transfer PCR reaction mixture to a 1.5mL microfuge tube and add 5 volumes of Buffer B3.</LI></OL><UL class="bullets"><LI>Note: please ensure isopropanol has been added to Buffer B3 prior to use.</LI></UL><OL start="2"><LI>Transfer the above mixture solution to the EZ-10 column and let it stand at room temperature for 2 minutes. Centrifuge at 10,000 rpm for 2 minutes.</LI><LI>Remove the flow-through in the tube. Add 750μL of Wash Solution to the column and centrifuge at 10,000 rpm for 2 minutes.</LI><LI>Repeat washing procedure in step 3. Spin at 10,000 rpm for an additional minute to remove any residual Wash Solution.</LI><LI>Transfer the column into a clean 1.5mL microfuge tube and add 30-50μL Elution Buffer to the <STRONG>center</STRONG> of the column. Incubate at room temperature for 2 minutes. Centrifuge at 10,000 rpm for 2 minutes to elute the DNA.</LI></OL><UL class="bullets"><LI>Incubating the column with the Elution Buffer at a higher temperature (37°C to 50°C) may slightly increase the yield, especially of large (&gt; 10,000 bp) DNA plasmids.</LI><LI>Prewarming the Elution Buffer at 55°C to 80°C may also slightly increase elution efficiency.</LI></UL><OL start="6"><LI>Store purified DNA at -20°C.</LI></OL><H4>Leaving the lab</H4><P>Prior to leaving the lab, you should:</P><UL class="bullets"><LI>Clean dirty glassware, or at least set aside the glassware to be cleaned by a designated individual.</LI><LI>Wipe down your workspace.</LI><LI>Ensure that all materials have been returned to their places, and that the plates have been properly stored in the fridge.</LI></UL><H4>Acknowledgements</H4><P>All information from <A href="https://store.biobasic.com/resources/productinfo/Kit%20Brochure%20Complete%20-v20.pdf">https://store.biobasic.com/resources/productinfo/Kit Brochure Complete -v20.pdf</A> (page 11-13)</P><H3 class="text-red">NEB Monarch Miniprep (Purification)</H3><H4>Safety Precautions</H4><P><STRONG>SDS (safety data sheet)</STRONG>: Refer to the SDS sheets for all listed materials before entering the lab. Be prepared to answer any questions regarding the information on these sheets.</P><P><STRONG>PPE (Personal protective equipment)</STRONG>: Proper lab attire should be worn throughout the experiment. This means that upon entering the lab you should be wearing long pants and close-toed shoes. Contact lenses should not be worn. Furthermore, a lab coat, goggles, and gloves should be worn at all times, and long hair should be tied back.</P><H4>Materials</H4><H5>Reagents</H5><UL class="bullets"><LI>NEB Monarch Miniprep Kit</LI><LI>Make sure that you have enough spin columns and that ethanol has been added to your wash buffer.</LI><LI>Store Plasmid Neutralization Buffer (B3) at 4°C after opening.</LI></UL><H5>Equipment</H5><UL class="bullets"><LI>Autoclaved Microcentrifuge tubes</LI><LI>p1000, p200, and p2 for subsequent nanodrop afterwards</LI><LI>50mL Falcon tubes for convenient waste disposal in between steps</LI></UL><H4>Procedure</H4><OL><LI>Pellet 1–5mL bacterial culture by centrifugation for 30 seconds. Discard supernatant.</LI></OL><UL class="bullets"><LI>Note: For a standard miniprep to prepare DNA for restriction digestion or PCR, we recommend 1.5mL of culture.</LI></UL><OL start="2"><LI>Resuspend pellet in 200μL Plasmid Resuspension Buffer (B1) (pink). Vortex or pipet to ensure cells are completely resuspended. There should be no visible clumps.</LI><LI>Lyse cells by adding 200μL Plasmid Lysis Buffer (B2) (blue/green). Invert tube immediately and gently 5–6 times until color changes to dark pink and the solution is clear and viscous. <STRONG>Do not vortex!</STRONG> Incubate for one minute.</LI></OL><UL class="bullets"><LI>Note: Care should be taken not to handle the sample roughly and risk shearing chromosomal DNA, which will co-purify as a contaminant. Avoid incubating longer than one minute to prevent irreversible plasmid denaturation.</LI></UL><OL start="4"><LI>Neutralize the lysate by adding 400μL of Plasmid Neutralization Buffer (B3) (yellow). Gently invert tube until color is uniformly yellow and a precipitate forms. <STRONG>Do not vortex!</STRONG> Incubate for 2 minutes.</LI></OL><UL class="bullets"><LI>Note: Be careful not to shear chromosomal DNA by vortexing or vigorous shaking. Firmly inverting the tube promotes good mixing, important for full neutralization.</LI></UL><OL start="5"><LI>Clarify the lysate by spinning for 2–5 minutes at 16,000 g.</LI></OL><UL class="bullets"><LI>Note: Spin time should not be less than 2 minutes. Careful handling of the tube will ensure no debris is transferred and the 2 minute recommended spin can be successfully employed to save valuable time.</LI><LI>For culture volumes &gt; 1mL, we recommend a 5 minute spin to ensure efficient RNA removal by RNase A. Also, longer spin times will result in a more compact pellet that lowers the risk of clogging the column.</LI></UL><OL start="6"><LI>Carefully transfer supernatant to the spin column and centrifuge for 1 minute. Discard flow-through.</LI><LI>Re-insert column in the collection tube and add 200μL of Plasmid Wash Buffer 1. <EM>If the DNA will be used in transfection, incubate 5 minutes</EM>. Centrifuge for 1 minute. Discarding the flow-through is optional.</LI></OL><UL class="bullets"><LI>Plasmid Wash Buffer 1 removes RNA, protein, and endotoxin.</LI><LI>Note: The collection tube is designed to hold 800μL of flow-through fluid and still allow the tip of the column to be safely above the top of the liquid. Empty the tube whenever necessary to ensure the column tip and flow-though do not make contact.</LI></UL><OL start="8"><LI>Add 400μL of Plasmid Wash Buffer 2 and centrifuge for 1 minute.</LI><LI>Transfer column to a clean 1.5mL microfuge tube. Use care to ensure that the tip of the column has not come into contact with the flow-through. If there is any doubt, re-spin the column for 1 minute before inserting it into the clean microfuge tube.</LI><LI>Add ≥ 30 μL DNA Elution Buffer to the <STRONG>center</STRONG> of the matrix. Wait for 1 minute, then spin for 1 minute to elute DNA.</LI></OL><UL class="bullets"><LI>Note: Nuclease-free water (pH 7–8.5) can also be used to elute the DNA.</LI><LI>Delivery of the Monarch DNA Elution Buffer should be made directly to the center of the column to ensure the matrix is completely covered for maximal efficiency of elution.</LI><LI>Yield may slightly increase if a larger volume of DNA Elution Buffer is used, but the DNA will be less concentrated as a result of dilution.</LI><LI>For larger plasmids (≥ 10 kb), heating the DNA Elution Buffer to 50°C prior to eluting and extending the incubation time after buffer addition to 5 minutes can improve yield.</LI></UL><OL start="11"><LI>Proceed to Nanodrop for DNA quantification. Be sure to blank with the appropriate elution substance (ie. Elution Buffer OR Nuclease-free water).</LI></OL><H4>Leaving the lab</H4><P>Prior to leaving the lab, you should:</P><UL class="bullets"><LI>Clean dirty glassware, or at least set aside the glassware to be cleaned by a designated individual.</LI><LI>Wipe down your workspace.</LI><LI>Ensure that all materials have been returned to their places, and that the plates have been properly stored in the fridge.</LI></UL></DIV><DIV id="Competence_and_Transformation_763" class="subsection"><H2>Competence and Transformation</H2><H3 class="text-red">Bacterial Transformation Protocol (Competence and Transformation)</H3><H4>Introduction</H4><P>In this protocol, you will be using chemically competent <EM>E. coli</EM> that have been placed in a calcium chloride solution prior to freezing. The solution acts to neutralize the cells so that they don’t repel each other. The bacteria and plasmid mixture will be chilled on ice for 30 minutes. Placing the mixture in a 42°C water bath for 30 seconds will heat shock the mixture, which will cause the transformation. Once the LB is added and mixed with the transformed bacteria, it can be plated on the LB plate with an antibiotic. Since much of the bacteria will not be transformed after the heat shock, plating <EM>E. coli</EM> with an antibiotic will ensure that only transformed <EM>E. coli</EM> survive.</P><P>Before doing this protocol, please watch this video: <A href="https://vimeo.com/25201947">https://vimeo.com/25201947</A>. Note that we do not have a control for our experiment, and we are not using a thermomixer.</P><P>Separate reagents from materials.</P><H4>Basic Terminology and Concepts</H4><UL class="bullets"><LI><STRONG>Transformation</STRONG>: The process in which the genetic makeup of a cell is changed by introduction of DNA from the surrounding environment.</LI><LI><STRONG>Competent <EM>E. coli</EM></STRONG>: <EM>E. coli</EM> which can allow the uptake of DNA.</LI><LI><STRONG>Heat Shock</STRONG>: A sudden increase in temperature used to propel a plasmid into a bacterium.</LI></UL><H4>Safety precautions</H4><P>This is a benign lab protocol, but don’t forget that PPE is necessary at all times. The <EM>E. coli</EM> you will be working with will be non-pathogenic but it should still be handled properly, which means it should not come into contact with your skin or gloves. If you are removing materials from the -80°C freezer, do not use your bare hands or regular lab gloves. Use gloves designated specifically for the freezer.</P><H4>Materials</H4><H5>Reagents</H5><UL class="bullets"><LI>Competent <EM>E. coli</EM> cells</LI><LI>250μL SOC</LI><LI>LB plate supplemented with antibiotics</LI><LI>Plasmid</LI></UL><H4>Equipment</H4><UL class="bullets"><LI>1 set of micropipettes and pipette tips</LI><LI>5✕1.5mL microcentrifuge tubes</LI><LI>Ice bath</LI><LI>Thermomixer</LI><LI>Streaker</LI></UL><H4>Procedure<OL><LI>Thaw competent <EM>E. coli</EM> on ice.
			</LI><LI>Competent cells will be in the -80°C freezer.</LI><LI>Do <STRONG>not</STRONG> thaw cells by hand. If they are warmed by hand they will no longer be competent.</LI><LI>Mix the cells gently with a pipette tip (Do <STRONG>not</STRONG> pipette up and down).</LI><LI>Aliquot 50μL of bacteria into a pre-chilled 1.5mL microcentrifuge tube.</LI><LI>Add 1-2μL of the plasmid to the bacterial sample, and mix gently with the pipette tip (Do <STRONG>not</STRONG> pipette up and down).</LI><LI>Incubate the tube on ice for 30 minutes.</LI><LI>While the bacteria are on ice, ensure the water bath is set to 42°C.</LI><LI>Put 250μL LB in a microcentrifuge tube and place it in the 37°C incubator to warm up during this time.</LI><LI>After the 30 minute incubation period, place the tube in the 42°C water bath (mixing OFF) for exactly 30 seconds (45 seconds for <EM>E. Coli</EM> DH10β).</LI><LI>Place the tube on ice for 2 minutes.</LI><LI>Add 250μL pre-warmed SOC.</LI><LI>Shake in 37°C incubator at 300 rpm for 1-2 hours
			</LI><LI>Note that DH5α grows faster than DH10β, so adjust shake time accordingly</LI><LI>Spread 100μL of the transformed bacteria on an LB plate supplemented with the appropriate antibiotic.</LI><LI>Label plate as <STRONG>iGEM 2017 - [Date] - Transformed [<EM>E. Coli</EM> strain] - [Your Name] - [Media Number]</STRONG></LI><LI>Allow the plates to dry inverted, then place in the 37°C incubator overnight.</LI></OL></H4><H4>Leaving the lab<UL class="bullets"><LI>Clean dirty glassware, or at least set aside the glassware to be cleaned by a designated individual.</LI><LI>Wipe down your workspace.</LI><LI>Ensure that all materials have been returned to their places, and that the plates have been properly stored in the fridge.</LI><LI>Dispose of all disposable tubes and pipette tips used in biohazard containers.</LI><LI>Make sure your plates are labelled and put in a place they can be found.</LI></UL></H4><H4>Acknowledgements<P>Adapted from iGEM Toronto 2015.</P></H4><H3>Electrocompetent Cell Protocol (Competence and Transformation)</H3><H4>Introduction</H4><P>This protocol can be used to prepare electrocompetent cells for electroporation.</P><H4>Materials</H4><H5>Reagents</H5><UL class="bullets"><LI>Cells to be made competent</LI><LI>LB Media</LI><LI>Sterile 10% glycerol (can be autoclaved) is needed for the washes.
			</LI><LI>The volume of 10% glycerol needed is 2X the culture volume (for example, a 500 mL culture requires 1 L of 10% glycerol).</LI></UL><H5>Equipment</H5><UL class="bullets"><LI>15mL and 50mL falcon tubes</LI><LI>1L baffled flask</LI><LI>37°C shaker</LI><LI>Centrifuge</LI></UL><H4>Procedure</H4><H5>Day 1</H5><OL><LI>Inoculate 1 colony from a fresh plate of the strain to be made electrocompetent into 5m: of LB in a 15mL falcon tube.</LI><LI>Incubate for 16-18 hours at 37°C and 250 rpm</LI></OL><UL class="bullets"><LI>Keep in the 37°C shaker overnight, make sure cap is on loosely and tape is securing cap to tube - this allows aeration of the cultures</LI></UL><H5>Day 2</H5><OL><LI>Have a 1L baffled flask containing 99mL LB ready. Add 1mL of the overnight culture.</LI><LI>Shake at 37°C and 250 rpm until the cultures reach an OD600 of 0.4-0.55.</LI></OL><UL class="bullets"><LI>Be sure to turn on centrifuge and cool rotor to 4°C well in advance of harvesting cells.</LI><LI>Be sure to place 1L 10% glycerol on ice well in advance of harvesting cells</LI></UL><OL start="3"><LI>Pour culture into 2 chilled 50mL falcon tubes.</LI><LI>Centrifuge at 4000 rpm for 10 min. Pour off the supernatant and aspirate any residual broth.</LI><LI>Add glycerol to the 50mL mark on each of the falcon tubes and completely suspend the cells by pipetting up and down.</LI><LI>Place cultures on ice for 6-10 minutes. From this point on the cultures must be kept ice cold.</LI><LI>Centrifuge at 4000 rpm for 10 minutes. Pour off the supernatant and aspirate any residual broth.</LI><LI>Add glycerol to the 25mL mark on each of the falcon tubes and completely suspend the cells by pipetting up and down.</LI><LI>Centrifuge at 4000 rpm for 10 min.</LI><LI>Pour off the supernatant and suspend the cells in the residual glycerol by pipetting up and down (there should be ~1-2mL per tube).</LI><LI>At this point you can electroporate or freeze the cells away.</LI></OL><H5>Freezing cells</H5><OL><LI>Add 100μL of the culture to microcentrifuge tubes on ice.</LI><LI>Once you have used all of the culture, transfer the tubes to dry ice for 10 minutes.</LI><LI>Once the cultures are frozen, transfer them to a -80°C freezer. The cultures should be good for 3 weeks.</LI></OL><H4>Leaving the lab</H4><UL class="bullets"><LI>Clean dirty glassware, or at least set aside the glassware to be cleaned by a designated individual.</LI><LI>Wipe down your workspace.</LI><LI>Ensure that all materials have been returned to their places, and that the plates have been properly stored in the fridge.</LI><LI>Dispose of all disposable tubes and pipette tips used in biohazard containers.</LI><LI>Make sure your plates are labelled and put in a place they can be found.</LI></UL><H4>Acknowledgements</H4><P>Edited from NEB by Carla H.</P><H3 class="text-red">Electroporation Protocol (Competence and Transformation)</H3><H4>Introduction</H4><P><A href="http://raizadalab.weebly.com/uploads/4/2/4/8/4248579/12b._biorad_micropulser_manual.pdf">Link</A> to lab’s BioRad electroporator user’s manual</P><H4>Safety precautions</H4><P><STRONG>SDS (safety data sheet)</STRONG>: Refer to the SDS sheets for all listed materials before entering the lab. Be prepared to answer any questions regarding the information on these sheets.</P><P><STRONG>PPE (Personal protective equipment)</STRONG>: Proper lab attire should be worn throughout the experiment. This means that upon entering the lab you should be wearing long pants and close-toed shoes. Contact lenses should not be worn. Furthermore, a lab coat, goggles, and gloves should be worn at all times, and long hair should be tied back.</P><H4>Materials</H4><H5>Reagents</H5><UL class="bullets"><LI>Electrocompetent <EM>E. coli</EM></LI><LI>Plasmid DNA</LI><LI>SOC media</LI><LI>LB media</LI></UL><H5>Equipment</H5><UL class="bullets"><LI>Electroporation cuvette</LI><LI>Electroporator</LI><LI>Serological pipette</LI><LI>Falcon tube</LI><LI>Incubator</LI></UL><H4>Procedure</H4><OL><LI>Thaw frozen tube of competent cells on ice.</LI><LI>Transfer 70µL of electrocompetent <EM>E. coli</EM> into the cuvette, ensuring that the liquid does not go over the metal bar.</LI><LI>Add 1µL of the desired plasmid DNA to the thawed bacteria in the cuvette.</LI><LI>Gently tap/shake the cuvette to ensure that the cell suspension covers the cuvette bottom</LI><LI><EM>(edit: Victoria recomends skipping this step)</EM> Incubate the cuvette containing DNA and bacteria on ice for 10 minutes.</LI><LI>Thoroughly dry the outside of the electroporation cuvette and place in the cuvette holder.</LI><LI>Set electroporator to Ec1: <EM>E.coli</EM> in 1mm cuvette (1.8kV) or Ec2: <EM>E.coli</EM> in 2mm cuvette (2.5kV) and pulse each cuvette once.</LI><LI>Immediately (within 10 seconds) add 1mL SOC and use a serological pipette to transfer the mix into a falcon tube.</LI><LI>Incubate the bacteria at 37°C for 30 to 60 minutes with gentle rotation or shaking.</LI><LI>Plate 10µL on LB and antibiotic plates (such as CAM).</LI><LI>Incubate overnight at 37°C.</LI></OL><H4>Leaving the lab</H4><UL class="bullets"><LI>Clean dirty glassware, or at least set aside the glassware to be cleaned by a designated individual.</LI><LI>Wipe down your workspace.</LI><LI>Ensure that all materials have been returned to their places, and that the plates have been properly stored in the fridge.</LI><LI>Dispose of all disposable tubes and pipette tips used in biohazard containers.</LI><LI>Make sure your plates are labelled and put in a place they can be found.</LI></UL><H4>Acknowledgements</H4><P>Adapted from iGEM Toronto 2015.</P><H3 class="text-red">Rubidium Chloride Competent Cells (Competence and Transformation)</H3><H4>Safety Precautions</H4><P><STRONG>SDS (safety data sheet)</STRONG>: Refer to the SDS sheets for all listed materials before entering the lab. Be prepared to answer any questions regarding the information on these sheets.</P><P><STRONG>PPE (Personal protective equipment)</STRONG>: Proper lab attire should be worn throughout the experiment. This means that upon entering the lab you should be wearing long pants and close-toed shoes. Contact lenses should not be worn. Furthermore, a lab coat, goggles, and gloves should be worn at all times, and long hair should be tied back.</P><P><STRONG>Autoclave</STRONG>: The autoclave should only be handled by leads and managers. Note that any autoclaved materials may still be hot and should therefore be handled with caution. Be careful not to burn yourself.</P><H4>Materials</H4><H5>Reagents</H5><UL class="bullets"><LI>Filter sterilized 50mL chilled RF1 (33mL would be used):
			</LI><LI>0.605g 100mM RbCl</LI><LI>0.495g 50mM MnCl<SUB>2</SUB>•4H<SUB>2</SUB>O</LI><LI>0.147g 30mM Potassium acetate</LI><LI>0.074g 10mM CaCl<SUB>2</SUB>•2H<SUB>2</SUB>O</LI><LI>7.5g or 6mL 15% m/v glycerol</LI></UL><LI>Filter sterilized 50mL chilled RF2 (4mL would be used)
			</LI><LI>0.105g 10mM MOPS</LI><LI>0.06g 10mM RbCl</LI><LI>0.55g 75mM CaCl<SUB>2</SUB>•2H<SUB>2</SUB>O</LI><LI>7.5 g or 6 mL 15% m/v glycerol</LI><LI>Sterilized 100mL LB</LI><LI>MilliQ water</LI><LI>0.2M Acetic acid</LI><H5>Equipment</H5><UL class="bullets"><LI>Sterilized 250mL centrifuge bottles</LI><LI>Sterilized 1.5mL microfuge tubes (at least 50)</LI><LI>250mL flask</LI><LI>Syringe filters</LI></UL><H4>Procedure</H4><H5>How to use Syringe Filters</H5><P>Always work in a sterile environment (near a Bunsen burner).</P><OL><LI>Screw filter onto end of syringe</LI><LI>Pull out stopper (plunger) and fill with reagent (RF1 or RF2)</LI><LI>Place plunger back in and slowly push down to expel liquid into a new labelled sterile container</LI><LI>Single use only</LI></OL><H5>Making RF1 Solution</H5><P>This protocol amkes 50mL RF1 solution.</P><OL><LI>Mix  0.605g 100mM RbCl, 0.495g 50mM MnCl<SUB>2</SUB>•4H<SUB>2</SUB>O, 0.147g 30mM Potassium acetate, 0.074g 10mM CaCl<SUB>2</SUB>•2H<SUB>2</SUB>O, and 7.5g or 6mL 15% m/v glycerol.</LI><LI>Add sterile MilliQ up to ~45mL.</LI><LI>Adjust final pH to 5.8 using 0.2M acetic acid (~400μL for 33mL).</LI><LI>Add additional sterile MiliQ to reach final volume of 50mL.</LI><LI>Filter sterilize.</LI></OL><P>Note: use glacial acetic acid [1.049g/cm<SUP>3</SUP> / 60.05g/mol = 17.47M] to make 0.2M acetic acid.</P><H5>Making RF2 Solution</H5><OL><LI>Mix 0.105g 10mM MOPS, 0.06g 10mM RbCl, 0.55g 75mM CaCl<SUB>2</SUB>•2H<SUB>2</SUB>O, and 7.5 g or 6 mL 15% m/v glycerol.</LI><LI>Add sterile MilliQ up to ~45mL.</LI><LI>Adjust final pH to 6.8 using 1M NaOH (~200μL for 30mL).</LI><LI>Add additional sterile MilliQ to reach final volume of 50mL.</LI><LI>Filter sterilize.</LI></OL><H5>Day 1</H5><OL><LI>Streak DH5α from frozen glycerol stock on the LB plate.</LI><LI>Incubate at 37 °C over night.</LI><LI>Prepare sterilized LB.</LI></OL><H5>Day 2</H5><OL><LI>Pick up a single colony from the LB plate.</LI><LI>Inoculate to 3mL sterilized LB.</LI><LI>Incubate at 37°C overnight.</LI><LI>Put RF1, RF2, centrifuge tube, and Eppendorf tubes into the 4°C refrigerator.</LI></OL><H5>Day 3</H5><OL><LI>Put RF1, RF2, centrifuge tube, and Eppendorf tubes on ice.</LI><LI>Inoculate 1mL of overnight culture to 100mL of LB in 250mL flask.</LI><LI>Monitor OD600 from initial until 0.2-0.6. (0.4-0.55 optimum).</LI><LI>Transfer culture to centrifuge bottle and chill on ice 10-15 min.</LI><LI>Pellet cells by centrifugation at 2700 g (4200 rpm in an F14 6x250y rotor) for 10 min at 4°C.</LI><LI>Decant liquid and stand the bottle in an inverted position for &lt; 1 min.</LI><LI>Resuspend in 1/3 original volume (33mL) chilled RF1 buffer gently (do not vortex).</LI><LI>Optimally, resuspend using a 25mL disposable pipette (RbCl will permanently stain glass pipets).</LI><LI>Continue mixing until cells are evenly resuspended and no clumps are visible.</LI><LI>Incubate on ice for 15 min.</LI><LI>Pellet cells by centrifugation at 580 g (1950 rpm in an F14 6x250y rotor) for 15 min at 4°C.</LI><LI>Decant liquid and gently resuspend in 1/25 original volume (4mL) chilled RF2 buffer.</LI><LI>Incubate on ice for 15 min.</LI><LI>Aliquot 100µL into each chilled 1.5mL Eppendorf tube and freeze on dry ice (or ice).</LI><LI>Store at -80 °C.</LI></OL><H4>Acknowledgements<P>Protocol adapted from <A href="http://openwetware.org/wiki/RbCl_competent_cell">http://openwetware.org/wiki/RbCl_competent_cell</A></P></H4><H2>Cloning Protocols</H2><H3 class="text-red">Gibson Assembly (Cloning)</H3><H4>Introduction</H4><H5>Calculating Optimal pmols of Each Insert</H5><P>Equation: pmols = (1000×weight in ng) / (length in bp×650 Da)</P><UL class="bullets"><LI>50ng of 5000bp dsDNA is about 0.015pmols</LI><LI>Online Tools: <A href="https://www.neb.com/external-links/nebiocalculator">NEBioCalculator</A></LI></UL><P>NEB recommendation:</P><UL class="bullets"><LI>0.02–0.5 pmols when 1-2 inserts are being assembled into a vector (or 2-3 fragments are being assembled)</LI><LI>0.2–1.0 pmoles when 3–5 inserts are being assembled into a vector (or 4-6 fragments are being assembled)</LI></UL><H5>Notes:</H5><UL class="bullets"><LI>Efficiency of assembly decreases as the number or length of inserts increases</LI><LI>The mass of each insert can be measured using the NanoDrop instrument, with absorbance at 260nm or estimated from agarose gel electrophoresis followed by  ethidium bromide staining.</LI><LI>Yields will be best when the the different inserts are present in equimolar concentrations.</LI><LI>Inserts to be assembled should not have stable single stranded DNA secondary structure, such as a hairpin or a stem loop, or repeated sequences, as this would directly compete with the required single-stranded annealing/priming of neighboring assembly fragments.</LI><LI>The Gibson Cloning Master Mix consists of three different enzymes within a single buffer. Each enzyme has a specific and unique function for the reaction:
			</LI><LI>T5 Exonuclease: creates single-strand DNA 3’ overhangs by chewing back from the DNA 5’ end. Complementary DNA fragments can subsequently anneal to each other.</LI><LI>Phusion DNA Polymerase: incorporates nucleotides to “fill in” the gaps in the annealed DNA fragments.</LI><LI>Taq DNA Ligase: covalently joins the annealed complementary DNA fragments, removing any nicks and creating a contiguous DNA fragment.</LI></UL><P><A href="https://www.neb.com/tools-and-resources/search?relatedProducts=%7B0AA961B2-94E4-44AF-BEDD-5C4A904C76E6%7D&amp;ProductFilter=Gibson&amp;type=FAQ&amp;num=26">Troubleshooting help</A></P><H4>Safety Precautions</H4><P><STRONG>SDS (safety data sheet)</STRONG>: Refer to the SDS sheets for all listed materials before entering the lab. Be prepared to answer any questions regarding the information on these sheets.</P><P><STRONG>PPE (Personal protective equipment)</STRONG>: Proper lab attire should be worn throughout the experiment. This means that upon entering the lab you should be wearing long pants and close-toed shoes. Contact lenses should not be worn. Furthermore, a lab coat, goggles, and gloves should be worn at all times, and long hair should be tied back.</P><H4>Materials</H4><H5>Reagents</H5><UL class="bullets"><LI>Milli-Q Water (deionized H<SUB>2</SUB>0)</LI><LI>DNA fragments (gBlocks/inserts)
			</LI><LI>Store at -20°C in TE for up to 24 months or nuclease-free H<SUB>2</SUB>O for up to 1 month</LI></UL><LI>Gibson Assembly Master Mix (2x)
			</LI><LI>Store at -20°C</LI><LI>Vectors (plasmids)</LI><LI>Ice</LI><H5>Reaction Component Table</H5><DIV class="figures"><TABLE class="table table-striped table-bordered"><THEAD><TR><TH>Assembly type</TH><TH>2-3 Fragment Assembly (vector + 1-2 inserts)</TH><TH>4-6 Fragment Assembly (vector + 3-5 inserts)</TH><TH>Positive Control**</TH></TR></THEAD><TBODY class="table-hover"><TR><TD>Total Amount of Fragments</TD><TD>0.02–0.5 pmols* (x μL)</TD><TD>0.2–1 pmols* (x μL)</TD><TD>10 μL</TD></TR><TR><TD>2X Gibson Assembly Master Mix</TD><TD>10 μL</TD><TD>10 μL</TD><TD>10 μL</TD></TR><TR><TD>Milli-Q water</TD><TD>(10-x) μL</TD><TD>(10-x) μL</TD><TD>0</TD></TR><TR><TD>Total Volume</TD><TD>20 μL***</TD><TD>20 μL***</TD><TD>20 μL</TD></TR></TBODY></TABLE></DIV><P>*Optimized cloning efficiency is 50–100 ng of vectors with 2–3 fold of excess inserts. Use 5 times more of inserts if size is less than 200 bps. Total volume of unpurified PCR fragments in Gibson Assembly reaction should not exceed 20%. 
			**Makes enough positive control reagents for five experiments. 
			***If greater numbers of fragments are assembled, additional Gibson Assembly Master Mix may be required.</P><H5>Equipment</H5><UL class="bullets"><LI>Thermocycler</LI><LI>Ice Bucket</LI><LI>-20°C Fridge</LI></UL><H4>Procedure</H4><OL><LI>Set up the reaction on ice using the reaction component table above.</LI><LI>Incubate samples at 50°C in a thermocycler for 15 minutes for 2-3 fragment assembly or 60 minutes for 4-6 fragment assembly.</LI></OL><UL class="bullets"><LI>Note: Reaction times less than 15 minutes are generally not recommended. Extended incubation times (up to 4 hours) have been shown to improve assembly efficiencies in some cases. Do not incubate the reaction overnight.</LI></UL><OL start="3"><LI>Following incubation, store samples on ice or at –20°C for subsequent transformation.</LI><LI>Transform competent <EM>E. coli</EM> cells with 2μL of the assembly reaction, following the transformation protocol. To ensure you’ve made the correct plasmid, screen by restriction digest.</LI></OL><UL class="bullets"><LI>Note: NEB recommends using NEB 5-α Competent <EM>E. coli</EM> (High Efficiency, NEB #C2987). If the assembled products are larger than 10 kb, NEB recommends using NEB 10-β Competent <EM>E. coli</EM> (High Efficiency, NEB #C3019) or NEB 10-β Electrocompetent <EM>E. coli</EM> (NEB #C3020). If the assembled genes contain repetitive sequences, NEB Stable Competent <EM>E. coli</EM> (NEB #C3040) should be used.</LI></UL><H4>Leaving the lab</H4><P>Prior to leaving the lab, you should:</P><UL class="bullets"><LI>Clean dirty glassware, or at least set aside the glassware to be cleaned by a designated individual.</LI><LI>Wipe down your workspace.</LI><LI>Ensure that all materials have been returned to their places, and that the plates have been properly stored in the fridge.</LI></UL><H4>Acknowledgements</H4><P>Adapted from New England BioLabs Inc. (NEB): <A href="https://www.neb.com/protocols/2012/12/11/gibson-assembly-protocol-e5510">https://www.neb.com/protocols/2012/12/11/gibson-assembly-protocol-e5510</A></P><H3 class="text-red">T4 Ligase Protocol (Cloning)</H3><H4>Introduction</H4><P>Note: We did not Dpn1 digest our components before T4 ligation. The protocol was still successful, but typically Dpn1 digest is recommended.</P><H4>Safety Precautions</H4><P><STRONG>SDS (safety data sheet)</STRONG>: Refer to the SDS sheets for all listed materials before entering the lab. Be prepared to answer any questions regarding the information on these sheets.</P><P><STRONG>PPE (Personal protective equipment)</STRONG>: Proper lab attire should be worn throughout the experiment. This means that upon entering the lab you should be wearing long pants and close-toed shoes. Contact lenses should not be worn. Furthermore, a lab coat, goggles, and gloves should be worn at all times, and long hair should be tied back.</P><H4>Materials</H4><P>For a 20μL reaction:</P><UL class="bullets"><LI>2μL 10X T4 DNA ligase buffer*</LI><LI>50ng 0.020 pmol vector DNA</LI><LI>37.5ng 0.060 pmol insert DNA</LI><LI>1μL T4 DNA Ligase</LI><LI>Nuclease-free water</LI></UL><P>*Note: The T4 DNA ligase buffer should be thawed and resuspended at room temperature.</P><H5>Equipment</H5><UL class="bullets"><LI>Microcentrifuge tubes</LI><LI>Ice</LI><LI>Incubator</LI><LI>Microfuge</LI></UL><H4>Procedure</H4><OL><LI>Set up the reaction by adding the components listed above in a microcentrifuge tube on ice.</LI></OL><UL class="bullets"><LI><EM>T4 DNA Ligase should be added last. Note that the table shows a ligation using a molar ratio of 1:3 vector to insert for the indicated DNA sizes.</EM></LI><LI>Use <A href="http://nebiocalculator.neb.com/#!/">NEBioCalculator</A> to calculate molar ratios.</LI></UL><OL start="2"><LI>Gently mix the reaction by pipetting up and down and microfuge briefly.</LI><LI>For blunt ends, single base overhangs, or cohesive (sticky) ends, incubate at 16°C overnight.</LI><LI>Heat inactivate at 65°C for 10 minutes.</LI><LI>Chill on ice and transform 1-5μL of the reaction mix into 50μL competent cells.</LI></OL><H4>Leaving the lab</H4><P>Prior to leaving the lab, you should:</P><UL class="bullets"><LI>Clean dirty glassware, or at least set aside the glassware to be cleaned by a designated individual.</LI><LI>Wipe down your workspace.</LI><LI>Ensure that all materials have been returned to their places, and that the plates have been properly stored in the fridge.</LI></UL><H4>Acknowledgements</H4><P>The following protocol was sourced from NEB: <EM>Ligation Protocol with T4 DNA Ligase (M0202)</EM>.</P></DIV><DIV id="Confirmation_Protocols_1125" class="subsection"><H2>Confirmation Protocls</H2><H3 class="text-red">Basic RE Digest (Confirmation)</H3><H4>Introduction</H4><P>For relevant Buffer performance, incubation temperature, and inactivation temperature please consult the RE’s specific details at (<A href="https://www.neb.com/tools-and-resources/usage-guidelines/heat-inactivation">https://www.neb.com/tools-and-resources/usage-guidelines/heat-inactivation</A>). Also see the <A href="https://www.neb.com/protocols/2012/12/07/optimizing-restriction-endonuclease-reactions">general optimization tips</A>.</P><H4>Safety Precautions</H4><P><STRONG>SDS (safety data sheet)</STRONG>: Refer to the SDS sheets for all listed materials before entering the lab. Be prepared to answer any questions regarding the information on these sheets.</P><P><STRONG>PPE (Personal protective equipment)</STRONG>: Proper lab attire should be worn throughout the experiment. This means that upon entering the lab you should be wearing long pants and close-toed shoes. Contact lenses should not be worn. Furthermore, a lab coat, goggles, and gloves should be worn at all times, and long hair should be tied back.</P><H4>Materials</H4><H5>Reaction Component Table</H5><P>| Component | 50µL reaction |
			| Restriction enzyme | 1µL |
			| DNA | 1µg |
			| 10X Buffer | 5µL |
			| Nuclease-free water | To 50µL |</P><H5>Equipment</H5><UL class="bullets"><LI>Microcentrifuge tubes</LI><LI>Microcentrifuge</LI><LI>Incubator</LI></UL><H4>Procedure</H4><OL><LI>Set up the reaction.</LI><LI>Mix components by pipetting the reaction mixture up and down, or by “flicking” the reaction tube.</LI><LI>Quick (“touch”) spin-down in a microcentrifuge. Do not vortex the reaction.</LI><LI>Incubate for 1 hour at the enzyme-specific appropriate temperature.</LI><LI>Inactivation method depends on RE type. Typically this will be heat inactivation.</LI></OL><UL class="bullets"><LI>ex: XbaI requires a 20 min heat inactivation at 65°C.</LI></UL><H4>Leaving the lab</H4><P>Prior to leaving the lab, you should:</P><UL class="bullets"><LI>Clean dirty glassware, or at least set aside the glassware to be cleaned by a designated individual.</LI><LI>Wipe down your workspace.</LI><LI>Ensure that all materials have been returned to their places, and that the plates have been properly stored in the fridge.</LI></UL><H3 class="text-red">Gel Electrophoresis (Confirmation)</H3><H4>Safety Precautions</H4><P><STRONG>SDS (safety data sheet)</STRONG>: Refer to the SDS sheets for all listed materials before entering the lab. Be prepared to answer any questions regarding the information on these sheets.</P><P><STRONG>PPE (Personal protective equipment)</STRONG>: Proper lab attire should be worn throughout the experiment. This means that upon entering the lab you should be wearing long pants and close-toed shoes. Contact lenses should not be worn. Furthermore, a lab coat, goggles, and gloves should be worn at all times, and long hair should be tied back.</P><H4>Materials</H4><H5>Reagents</H5><UL class="bullets"><LI>0.5g agarose</LI><LI>1X TAE Buffer</LI><LI>3µL 10,000X SYBR Safe per gel</LI><LI>Sample DNA</LI><LI>Loading dye</LI><LI>1kb ladder</LI></UL><H5>Equipment</H5><UL class="bullets"><LI>Microwave</LI><LI>Erlenmeyer flask</LI><LI>Heat gloves</LI><LI>Nanodrop</LI><LI>Parafilm</LI><LI>PCR tubes</LI><LI>Gel electrophoresis machine</LI></UL><H4>Procedure</H4><H5>Part 1: Casting the gel (optimized for large gels)</H5><OL><LI>Add 0.5g agarose and 50mL of 1X TAE buffer to an Erlenmeyer flask.</LI></OL><UL class="bullets"><LI>Note that the TAE buffer in the stock has 50X concentration and needs to be diluted with autoclaved Milli-Q water.</LI></UL><OL start="2"><LI>Place the flask into the microwave. <STRONG>After this step flask will be hot. Use heat gloves</STRONG>.</LI><LI>Set the timer to 2 min. Start the microwave oven but stop every 30 sec to swirl the contents of the flask. This helps suspend the undissolved agarose. Continue until agarose particles are dissolved.</LI><LI>While hot, add 3µl of 10,000X SYBR Safe into the agarose solution and mix throughly.</LI><LI>Set the flask aside for cooling. Occasionally swirl to make the cooling even.</LI><LI>Set up the gel tray and make sure that there is no leak by testing it with dH<SUB>2</SUB>0 and KimWipes.</LI><LI>When the flask is cool enough for you to handle easily, pour the solution into the tray.</LI><LI>Wait until the gel solidifies.</LI></OL><H5>Part 2: Loading the gel</H5><OL><LI>Make 400mL of 1XTAE buffer per gel.</LI></OL><UL class="bullets"><LI>Note that the TAE buffer in the stock has 50X concentration and needs to be diluted with autoclaved Milli-Q water.</LI></UL><OL start="2"><LI>Make a note of what is going into each well and put that in your lab notebook.</LI><LI>Determine the DNA concentrations via Nanodrop. (Approximately 100ng DNA per band is required.)</LI><LI>Cut a piece of parafilm to use as a mixing surface. Alternatively, you can use PCR tubes.</LI><LI>Mix DNA with loading dye in a 1:1 ratio directly in the PCR tubes using a pipette tip and gently aspirating.</LI></OL><UL class="bullets"><LI>i.e. 10µL of DNA with 2µL of 6X loading dye.</LI></UL><OL start="6"><LI>Mix by pipetting up and down. Avoid bubbles.</LI><LI>Carefully remove the comb of the gel.</LI><LI>Place the gel into the gel electrophoresis apparatus with its tray.</LI><LI>Add more 1XTAE buffer to cover the gel completely. Buffer should be 1-2mm above the gel. Make sure that all the wells are submerged.</LI><LI>Load 10µL of DNA ladder to the first well. Add 10µL of your sample DNA to corresponding wells. (Remember record which wells you load)</LI><LI>Put the lid on. The negative (-/black) electrode should be closer to the DNA wells as DNA migrates towards the positive (+/red) electrode.</LI><LI>Set the rig to 100V &amp; 60 min. Hit start.</LI><LI>Make sure that your DNA does not run off the gel. The loading buffer enables you to track the DNA.</LI><LI>The gel apparatus will stop on its own when the time is up. Proceed to image the gel. It can sit in the rig overnight if need be.</LI></OL><H4>Leaving the lab</H4><P>Prior to leaving the lab, you should:</P><UL class="bullets"><LI>Clean dirty glassware, or at least set aside the glassware to be cleaned by a designated individual.</LI><LI>Wipe down your workspace.</LI><LI>Ensure that all materials have been returned to their places, and that the plates have been properly stored in the fridge.</LI></UL><H4>Acknowledgements</H4><P>Created by Seray Cicek from:</P><UL class="bullets"><LI>iGEM Wiki:<A href="http://local.biochemistry.utoronto.ca/igem/index.php/Analytical_Gel_Electrophoresis">http://local.biochemistry.utoronto.ca/igem/index.php/Analytical_Gel_Electrophoresis</A></LI><LI>Jove: <A href="http://www.jove.com/video/3923/agarose-gel-electrophoresis-for-the-separation-of-dna-fragments">http://www.jove.com/video/3923/agarose-gel-electrophoresis-for-the-separation-of-dna-fragments</A></LI><LI>Open Wetware: <A href="http://openwetware.org/wiki/Agarose_gel_electrophoresis">http://openwetware.org/wiki/Agarose_gel_electrophoresis</A></LI><LI>Life Technologies: <A href="https://www.lifetechnologies.com/ca/en/home/life-science/pcr/elevate-pcr-research/agarose-content-with-tips-and-tricks.html">https://www.lifetechnologies.com/ca/en/home/life-science/pcr/elevate-pcr-research/agarose-content-with-tips-and-tricks.html</A></LI><LI>Syber green catalog: <A href="http://www.lifetechnologies.com/order/catalog/product/S33102">http://www.lifetechnologies.com/order/catalog/product/S33102</A></LI><LI>Syber green staining protocols: <A href="http://www.lumiprobe.com/protocols/sybr-green-gel-staining">http://www.lumiprobe.com/protocols/sybr-green-gel-staining</A></LI></UL><H3 class="text-red">Transformation Efficiency (Confirmation)</H3><H4>Introduction</H4><P>Transformation efficiency is the efficiency by which bacterial cells take up foreign DNA and express the genes encoded by it. Since in reality only a small portion of competent cells are transformed, a reasonable representation of transformation efficiency is calculated by dividing successful transformants by the amount of DNA used during transformation procedure, in CFU/μg of DNA. By verifying the transformation efficiency value against that of standard competent <EM>E.coli</EM> cells (1.5x10<SUP>8</SUP> to 6x10<SUP>8</SUP> cfu/μg DNA), competency of electroporated sample can be validated. The transformation efficiency kit from iGEM includes five vials of purified DNA from BBa_J04450 (RFP construct) in plasmid backbone pSB1C3. Each vial contains DNA at a different concentration: 50pg/ul, 20pg/ul, 10pg/ul, 5pg/ul, 0.5pg/ul. The transformation efficiency test is performed with each concentration of DNA in parallel.</P><H4>Safety Precautions</H4><P><STRONG>SDS (safety data sheet)</STRONG>: Refer to the SDS sheets for all listed materials before entering the lab. Be prepared to answer any questions regarding the information on these sheets.</P><P><STRONG>PPE (Personal protective equipment)</STRONG>: Proper lab attire should be worn throughout the experiment. This means that upon entering the lab you should be wearing long pants and close-toed shoes. Contact lenses should not be worn. Furthermore, a lab coat, goggles, and gloves should be worn at all times, and long hair should be tied back.</P><P>Disinfect lab bench with 70% ethanol prior to use.</P><H4>Materials</H4><H5>Reagents</H5><UL class="bullets"><LI>70% ethanol</LI><LI>Ice</LI><LI>Competent cell aliquot(s)</LI><LI>SOC media</LI></UL><H5>Equipment</H5><UL class="bullets"><LI>Paper towels</LI><LI>5x 2.0mL microcentrifuge tube</LI><LI>Transformation Efficiency Kit</LI><LI>Agar plates with chloramphenicol</LI><LI>Water bath with thermometer</LI><LI>Incubator</LI><LI>Spreader</LI><LI>Pipettor P10, P100, P1000 and pipette tips</LI><LI>Lab marker</LI><LI>Ice bucket</LI></UL><H4>Procedure</H4><OL><LI>Spin down the DNA tubes from the Transformation Efficiency Kit to collect all of the DNA into the bottom of each tube prior to use. A quick spin of 20-30 seconds at 8,000-10,000 rpm will be sufficient.</LI></OL><UL class="bullets"><LI>Note: There should be 50μL of DNA in each tube sent in the Kit.</LI></UL><OL start="2"><LI>Thaw competent cells on ice. Label one 2.0mL microcentrifuge tube for each concentration and then pre-chill by placing the tubes on ice.</LI><LI>Pipette 1μL of DNA into each microcentrifuge tube. There will be 5 microcentrifuges that contain 1μL of DNA of different concentration.</LI><LI>Pipette 50μL of competent cells into each tube. Flick the tube gently with your finger to mix. Incubate on ice for 30 minutes. Pre-heat waterbath so that thermometer reads 42°C.</LI><LI>Heat-shock the cells by placing into the waterbath for 1 minute. Be careful to keep the lids of the tubes above the water level, and keep the ice close by.</LI><LI>Immediately transfer the tubes back to ice, and incubate on ice for 5 minutes. This helps the cells recover.</LI><LI>Add 200μL of SOC media per tube, and incubate at 37°C for 2 hours. Prepare the agar plates during this time, also label them with concentration of DNA used.</LI><LI>Pipette 20μL from each tube onto the appropriate plate, and spread the mixture evenly across the plate with spreader. Incubate at 37°C overnight or approximately 16 hours. Position the plates so the agar side is facing up, and the lid is facing down.</LI><LI>Count the number of colonies on a light field or a dark background, such as a lab bench. Use the following equation to calculate your competent cell efficiency.</LI></OL><UL class="bullets"><LI>Transformation efficiency = (# of colonies on one plate) / (DNA plated in ng × 1000ng/μg)</LI><LI>DNA plated in ng = 1μL × (DNA concentration - refer to vial) × (volume plated (20 μL) / total reaction volume(251 μL))
			</LI><LI>Note: This refers to how much DNA was plated onto <EM>each</EM> agar plate, not the total amount of DNA used per transformation.</LI></UL><H5>Evaluation</H5><DIV class="figures"><TABLE class="table table-striped table-bordered"><THEAD><TR><TH>DNA concentration</TH><TH>0.5pg/μL</TH><TH>5pg/μL</TH><TH>10pg/μL</TH><TH>20pg/μL</TH><TH>50pg/μL</TH></TR></THEAD><TBODY class="table-hover"><TR><TD># of colonies</TD><TD>10-20</TD><TD>120-170</TD><TD>280-360</TD><TD>480-802</TD><TD>500-1000+</TD></TR></TBODY></TABLE></DIV><P>Competent cells should have an efficiency of 1.5x10<SUP>8</SUP>-6x10<SUP>8</SUP> cfu/μg DNA, where “cfu” means “colony-forming unit” and is a measurement of cells.</P><H4>Leaving the lab</H4><P>Prior to leaving the lab, you should:</P><UL class="bullets"><LI>Clean dirty glassware, or at least set aside the glassware to be cleaned by a designated individual.</LI><LI>Wipe down your workspace.</LI><LI>Ensure that all materials have been returned to their places, and that the plates have been properly stored in the fridge.</LI></UL><H4>Acknowledgements</H4><P>Made by Mark Wang (iGEM Toronto 2015).</P><P>Protocol taken with minor modification from iGEM guidelines.</P></DIV><DIV id="Characterization_Assays_1295" class="subsection"><H2>Characterization Assays</H2><H3 class="text-red">Interlab 2017 (Characterization Assays)</H3><P>For this year’s Interlab study, our team followed the plate reader protocol. Please refer to the links below for relevant procedural details:</P><P><A href="https://2017.igem.org/Competition/InterLab_Study/Plate_Reader">https://2017.igem.org/Competition/InterLab_Study/Plate_Reader</A></P><P><A href="https://static.igem.org/mediawiki/2017/8/85/InterLab_2017_Plate_Reader_Protocol.pdf">https://static.igem.org/mediawiki/2017/8/85/InterLab_2017_Plate_Reader_Protocol.pdf</A></P><H3 class="text-red">Phosphate Buffered Saline (PBS) (Characterization Assays)</H3><H4>Introduction</H4><P>Phosphate buffered saline (abbreviated as PBS) is a buffer solution commonly used in biological research. It is a salty solution buffer that helps to maintain a constant pH. The osmolarity and ion concentrations of the solution usually match those of the human body (isotonic).</P><H4>Safety Precautions</H4><P><STRONG>SDS (safety data sheet)</STRONG>: Refer to the SDS sheets for all listed materials before entering the lab. Be prepared to answer any questions regarding the information on these sheets.</P><P><STRONG>PPE (Personal protective equipment)</STRONG>: Proper lab attire should be worn throughout the experiment. This means that upon entering the lab you should be wearing long pants and close-toed shoes. Contact lenses should not be worn. Furthermore, a lab coat, goggles, and gloves should be worn at all times, and long hair should be tied back.</P><H4>Materials</H4><H5>Reagents</H5><P>For 10X PBS:</P><UL class="bullets"><LI>Distilled Water</LI><LI>80g of NaCl</LI><LI>2g of KCl</LI><LI>14.4g of Na<SUB>2</SUB>HPO<SUB>4</SUB></LI><LI>2.4g of KH<SUB>2</SUB>PO<SUB>4</SUB></LI><LI>HCl</LI></UL><H5>Equipment</H5><UL class="bullets"><LI>1L Pyrex Bottle</LI><LI>1L Graduated Cylinder</LI></UL><H4>Procedure</H4><P>This protocol makes 1L 10X solution.</P><OL><LI>Start with 800 mL of distilled water and add NaCl, KCl, Na<SUB>2</SUB>HPO<SUB>4</SUB>, and KH<SUB>2</SUB>PO<SUB>4</SUB></LI><LI>Adjust the pH to 7.4 with HCl.</LI></OL><UL class="bullets"><LI><EM>Note: The pH for 10X PBS should be 6.8, since after dilution to 1X, the pH will automatically change to 7.4.</EM></LI></UL><OL start="3"><LI>Add distilled water to a total volume of 1 liter.</LI><LI>Dispense the solution into aliquots and sterilize by autoclaving (20 min, 121°C, liquid cycle).</LI><LI>Store at room temperature.</LI></OL><H4>Leaving the lab</H4><P>Prior to leaving the lab, you should:</P><UL class="bullets"><LI>Clean dirty glassware, or at least set aside the glassware to be cleaned by a designated individual.</LI><LI>Wipe down your workspace.</LI><LI>Ensure that all materials have been returned to their places, and that the plates have been properly stored in the fridge.</LI></UL><H3 class="text-red">AraC Growth Profile Assay (Charcterization Assays)</H3><H4>Introduction</H4><P><STRONG>Aim</STRONG>: To assess the efficacy of our CRISPRi system (sgRNA) by shifting cells to arabinose media that necessitates expression of the araC gene.</P><P><STRONG>Hypothesis</STRONG>: If our sgRNA targeting of araC is sufficient, then a progressive decrease in optical density with regard to the control nonsense sgRNA will be observed due to the inability for the cells to utilize arabinose as a carbon source for metabolism.</P><UL class="bullets"><LI>6 technical replicates per strain are required to account for variation in biological data and ensure reproducibility</LI><LI>aTc is the inducer for the  dCas9 system – need to test across a large range of induction conditions</LI></UL><H4>Safety Precautions</H4><P><STRONG>SDS (safety data sheet)</STRONG>: Refer to the SDS sheets for all listed materials before entering the lab. Be prepared to answer any questions regarding the information on these sheets.</P><P><STRONG>PPE (Personal protective equipment)</STRONG>: Proper lab attire should be worn throughout the experiment. This means that upon entering the lab you should be wearing long pants and close-toed shoes. Contact lenses should not be worn. Furthermore, a lab coat, goggles, and gloves should be worn at all times, and long hair should be tied back.</P><H4>Materials</H4><H5>Reagents</H5><UL class="bullets"><LI>MG1655-pdCas9-pX cells</LI><LI>M9-Cm-Kan</LI><LI>aTc stocks</LI></UL><H5>Equipment</H5><UL class="bullets"><LI>250mL Erlenmeyer flask</LI><LI>96-well plate</LI><LI>Tinfoil</LI><LI>Centrifuge</LI><LI>Multichannel pipettor and pipette tips</LI></UL><H4>Procedure</H4><OL><LI>Set up an overnight culture of MG1655-pdCas9-pX cells in 4mL M9-Cm-Kan from a streaked-out plate.</LI><LI>Subculture 500μL cells in a 250mL Erlenmeyer flask of 50mL M9-Cm-Kan.</LI></OL><UL class="bullets"><LI>This returns cells to normal metabolism after being in late log phase overnight.</LI></UL><OL start="3"><LI>Meanwhile, set up a 96 well assay plate with serial-diluted aTc stocks prepared the night before. Stock concentrations should be (in μM): 0.4, 0.2, 0.1, 0.05, 0.025, 0.0125, 0.00625, 0.003125, and 0 (control is water). Add 10μl of each dilution to the plate as depicted below. Wrap plate in tin foil once complete.</LI><LI>At a corrected OD~0.5, dilute back to a corrected OD~0.1 in a 24 well block of M9-Cm-Kan.</LI><LI>Dilute back in 3mL of media.</LI><LI>Mix the plate at 300 rpm for ~1 minute to ensure the cultures are well-mixed.</LI><LI>Using the first two tips of a multichannel pipettor, fill the columns of the aTc plate with 190μl of diluted culture. Try to work as quickly as possible to reduce variation within and between columns.</LI><LI>Measure OD at 595nm for timepoint 0. Cover the plate with a breathable seal and incubate at 37°C with shaking at 200 rpm. Measure OD every hour for 7-8 hours.</LI><LI><STRONG>Analysis</STRONG>: plot OD vs. time for each aTc concentration in R or Excel. Compare growth curves between the control and test sgRNA to approximate the effectivity of your CRISPR system.</LI></OL><H4>Leaving the lab</H4><P>Prior to leaving the lab, you should:</P><UL class="bullets"><LI>Clean dirty glassware, or at least set aside the glassware to be cleaned by a designated individual.</LI><LI>Wipe down your workspace.</LI><LI>Ensure that all materials have been returned to their places, and that the plates have been properly stored in the fridge.</LI></UL><H4>Acknowledgements</H4><P>Developed by Dr. Soumaya Zlitni.</P></DIV></DIV><DIV class="block sidebar"><DIV id="sidebar-box"><H3>Contents</H3><H3>Related Pages</H3><DIV class="sidebar-minibox"><UL><LI><A href="https://2017.igem.org/Team:Toronto/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Design">Design</A></LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="section bg-black"><DIV class="container footer row"><DIV class="block content"><H3 class="white">General Pages</H3><UL><LI><A href="https://2017.igem.org/Team:Toronto">Home</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Safety">Safety</A></LI></UL></DIV><DIV class="block content"><H3 class="white">Wet Lab</H3><UL><LI><A href="https://2017.igem.org/Team:Toronto/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Hardware">Hardware</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Demonstrate">Demonstrate</A></LI></UL></DIV><DIV class="block content"><H3 class="white">Dry Lab</H3><UL><LI><A href="https://2017.igem.org/Team:Toronto/ODE">ODE</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Analysis">Analysis</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Protein-Modelling">Protein Modelling</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Model">Model</A></LI></UL></DIV><DIV class="block content"><H3 class="white">Human Practices</H3><UL><LI><A href="https://2017.igem.org/Team:Toronto/Interviews">Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Genetic_Code_Workshop">Genetic Code Workshop</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Podcast">Podcast</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Icon-a-thon">Icon-a-thon</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Guide">Guide</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/HP/Silver">HP/Silver</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/HP/Gold_Integrated">HP/Gold Integrated</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Engagement">Engagement</A></LI></UL></DIV></DIV><DIV class="container footer-bottom row"><DIV class="block content"><P>iGEM Toronto</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>