<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Toronto_Genetic_Code_Workshop skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Toronto/Genetic Code Workshop</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="row navigation bg-dark-cyan"><DIV class="row" id="navigation"><UL><LI><SPAN class="text-white">• </SPAN><A href="https://2017.igem.org/Team:Toronto" id="grey-hover" class="text-white">Home</A></LI><LI><SPAN class="text-grey">• </SPAN><A href="https://2017.igem.org/Team:Toronto/Team" id="grey-hover" class="text-white">Team</A></LI><LI><SPAN class="text-red">• </SPAN><A href="https://2017.igem.org/Team:Toronto/Wetlab" id="red-hover" class="text-white">Wetlab</A></LI><LI><SPAN class="text-yellow">• </SPAN><A href="https://2017.igem.org/Team:Toronto/Drylab" id="yellow-hover" class="text-white">Drylab</A></LI><LI><SPAN class="text-cyan">• </SPAN><A href="https://2017.igem.org/Team:Toronto/Human_Practices" id="cyan-hover" class="text-white">Human Practices</A></LI></UL></DIV></DIV><DIV class="section"><DIV class="container header" id="cyan"><H1>The Genetic Code Workshop</H1></DIV></DIV><DIV class="section bg-white"><DIV class="container content-page row"><DIV class="block content"><DIV id="subsection-coding" class="subsection"><H2 class="text-cyan">Coding Workshop</H2><P>
					The first half of the workshop emphasized on the foundation of computational biology and bioinformatics tools development through programming in Python. In the second half, participants were introduced to the basics of PyMol, a molecular visualization software used extensively in biochemical research.
				</P><P>
					Participants were challenged to develop new ways of problem-solving and to enter a computer scientist's mindset to come up with solutions to problems in the field of biology. In order to accommodate the best learning experience for students with varying levels of experience in computer science, the mentors provided one-on-one guidance and organized small group activities, for the best understanding of concepts for everyone. By the end of it, participants learned how to transcribe, translate and mutate DNA and RNA sequences by creating simple Python Scripts. Individuals that were already experienced in programming were encouraged to test themselves by attempting more difficult Rosalind problems, a set of problems developed by a community of esteemed computational biologists.
				</P><FIGURE><FIGCAPTION>Students observing hemoglobin model in pyMOL.</FIGCAPTION></FIGURE><P>
					With PyMol, the participants were given the opportunity to explore the essential principle of molecular biology, Form Follows Function. Students dived deep into the molecular biology of sickle cell anemia by observing the structure of hemoglobin in both healthy individuals and patients. Each student was then responsible for solving a puzzle on their own to find the point mutation that leads to the coagulation of hemoglobins and consequent the sickling of the red blood cells. The problem required students to hypothesize the amino acid location of the point mutation based on some preliminary information given about sickle cell anemia during the workshop.
				</P><P>
					After the technical workshop, we arranged a two-hour long lunch and social holding where many participants were able to get answers to career-related questions and concerns.
				</P></DIV><DIV id="subsection-ethics" class="subsection"><H2 class="text-cyan">Ethics Workshop</H2><P>After the coding portion of the day, students participated in 5 team activities designed to give students an understanding of our CRISPR/Cas9 project, the technical and ethical challenges of gene editing, and various synthetic biology topics.</P><FIGURE><FIGCAPTION>Students deep in discussion.</FIGCAPTION></FIGURE><P>The Python station reinforced concepts taught earlier on, while an RNA folding puzzle game was a platform for introducing basic mechanisms behind CRISPR and the idea of the “open laboratory,” or collaboration with the public as a method to tackle difficult challenges such as CRISPR off-target effects. In addition, a hands-on genetic circuit activity illustrated the mechanisms behind gene expression and feedback loops, which was followed by ethical case studies that provided an opportunity to discuss different perspectives and understand the societal effects of advancements in gene editing technology.</P><P> Mentors were available at each station to provide guidance, but students had to work together to solve each challenge in order to gain points. In particular, students performed well in the two case studies regarding gene drives and gene editing in Down’s syndrome patients, generating thoughtful answers in response to our discussion questions about the environmental, moral, and social impact of these potential applications of CRISPR technology. The winning team received 3D printed commemorative plaques to celebrate their success and enthusiasm in synthetic biology.</P></DIV><DIV id="subsection-engagement" class="subsection"><H2 class="text-cyan">Engagement</H2><P>Upon students’ reflections on their experiences at the workshop, many reported that they had learned new information regarding bioinformatics, genetics, the ethical considerations of gene editing, as well as university programs. In addition, a student remarked that one of their key takeaways from the workshop was “learning about the real world applications of topics they had learned in school.”</P><FIGURE><FIGCAPTION>Mentor helping students understand python code.</FIGCAPTION></FIGURE><P>Students also commented that they found the introduction to bioinformatics and group activities engaging and informative for their future career choices. It was rewarding to see a high level of engagement in our students, and we hope to have inspired the next generation of synthetic biologists, scientists and engineers to see the world in a different way.</P></DIV></DIV><DIV class="block sidebar"><DIV id="sidebar-box"><H3>Contents</H3><H3>Related Pages</H3><DIV class="sidebar-minibox"><UL><LI><A href="https://2017.igem.org/Team:Toronto/Icon-a-thon">Icon-a-thon</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Engagement">Engagement</A></LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="section bg-black"><DIV class="container footer row"><DIV class="block content"><H3 class="white">General Pages</H3><UL><LI><A href="https://2017.igem.org/Team:Toronto">Home</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Safety">Safety</A></LI></UL></DIV><DIV class="block content"><H3 class="white">Wet Lab</H3><UL><LI><A href="https://2017.igem.org/Team:Toronto/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Hardware">Hardware</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Demonstrate">Demonstrate</A></LI></UL></DIV><DIV class="block content"><H3 class="white">Dry Lab</H3><UL><LI><A href="https://2017.igem.org/Team:Toronto/ODE">ODE</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Analysis">Analysis</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Protein-Modelling">Protein Modelling</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Model">Model</A></LI></UL></DIV><DIV class="block content"><H3 class="white">Human Practices</H3><UL><LI><A href="https://2017.igem.org/Team:Toronto/Interviews">Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Genetic_Code_Workshop">Genetic Code Workshop</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Podcast">Podcast</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Icon-a-thon">Icon-a-thon</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Guide">Guide</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/HP/Silver">HP/Silver</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/HP/Gold_Integrated">HP/Gold Integrated</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Engagement">Engagement</A></LI></UL></DIV></DIV><DIV class="container footer-bottom row"><DIV class="block content"><P>iGEM Toronto</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>