Team:Toronto/Notebook
•  Home
•  Team
•  Wetlab
•  Drylab
•  Human Practices
Notebook
Preparing stocks, plates, cell cultures, etc.
Week 1, day 1 (26/06)
Prepared wash bottles of 70% Ethanol
Made 4x500mL LB media
2 with agar for plating
2 without agar for preparing overnight cultures
Made chloramphenicol (CAM) stocks
Plated out LB + agar onto sterile plates with CAM
Week 1, day 2 (27/06)
Made 10mL 0.2M acetic acid/1M NaOH
Prepared 50mL of RF1/RF2 buffer (to make RbCl competent cells)
Streaked 4 plates (stored in 4°C):
MG1655ΔLacI onto 2 LB and 1 LB+CAM plates
DH10β onto 1 LB plate
Week 1, day 3 (28/06)
Streaked 2 LB plates: MG1655ΔLacI & DH10β to replace Tuesday’s plates (because they over-grew)
New plates put into 37°C incubator overnight, backup plates placed into 4°C fridge
Prepared 5mL LB cultures and placed in overnight shaker of:
MG1655ΔLacI
DH10β
Week 1, day 4 (29/06)
Prepared SOC medium
Cultured DH10β cells in baffled flasks until OD reached 0.4-0.55
Week 1, day 5 (30/06)
Plated DH10β cells and incubated at 37°C overnight
Week 1, day 6 (31/06)
Checked on plated cells from 30/06:
Week 3, day 1 (10/07)
Made and autoclaved LB and LB+CAM liquid.
Poured 13 LB plates.
LB+CAM plates were unsuccessful since incorrect amount of CAM was added (12.5g instead of 12.5mg).
Week 3, day 2 (11/07)
Made LB+CAM plates
Plated and spread 50µL and 100µL DH5α cells onto LB agar plates using sterile loops.
Placed in overnight incubator
Week 3, day 3 (12/07)
-Checked on DH5α plates from the incubator.
Week 3, day 4 (13/07)
KAN and CAM antibiotic stocks
500mL each of LB+CAM and LB+KAN
500mL 1X TAE buffer from 50X TAE buffer and MilliQ water
Prepared overnight culture of one DH5α colony in 5mL LB at 37°C.
Week 3, day 5 (14/07)
Checked on DH5α culture
Initial OD600 was 0.011, so waited 110min for optimal OD600
Moved culture to ice once OD600 reached 0.466
Made DH5α stocks stored in -80°C, competence to be tested on Monday using iGEM competence cell test kit.
Week 4, day 1 (17/07)
Plated transformed cells.
Spread the DH5α cells onto 4 LB+CAM plates, each containing 100µL of the 100pg/µL, 50pg/µL, 10pg/µL, and 0pg/µL cells.
Spread the DH10β cells onto two LB+KAN plates, both containing 100µL of the control and either 100µL or 50µL of the pKDL071 cells
Incubated the plates at 37°C overnight.
Week 4, day 2 (18/07)
Made LB media
Made LB+CAM plates
Checked on 17/07 transformed cells and moved them to the 4°C fridge.
Confirmed DH5α cells were competent
Note: LB+CAM plates made on 13/07 may have been made with too much CAM (1250µL CAM to 500mL LB). This may explain why there were so few colonies from 17/07 transformation of DH5α + RFP/pSB1C3.
Obtained single colonies from transformed DH5α and DH10β plates from 17/07 and made overnight 37°C colonies.
The DH5α was placed in the 5mL of LB+CAM (5µL of stock CAM added to 5mL LB).
The DH10β was placed in 5mL of LB+KAN (25µL of stock KAN added to 5mL of LB).
Week 4, day 3 (19/07)
Examined plates from 18/07
Most plates had no cells likely due to the high CAM concentration (2.5x working concentration).
Made glycerol stocks of DH10β with pKDL071
Week 4, day 5 (21/07)
Made LB+AMP plates
Week 5, day 1 (24/07)
Made LB+CAM plates
Week 5, day 3 (26/07)
Made LB+CAM and LB+KAN solutions for overnight cultures
Week 5, day 4 (26/07)
Made 17 LB+KAN plates
Week 5, day 5 (28/07)
Prepared 10X PBS buffer
Week 6, day 1 (31/07)
Made 50mL non-sterilized 1X PBS Buffer
Week 6, day 3 (02/08)
Made LB+CAM plates
Week 6, day 5 (04/08)
Made LB liquid media
Week 7, day 4 (10/08)
Made 17 LB+Kan plates
Week 9, day 2 (22/08)
Made LB+KAN plates
Week 9, day 3 (23/08)
Made LB+AMP plates
Week 9, day 4 (24/08)
Made 5mL LB+Kan in falcon tube
Week 10, day 4 (01/09)
Made LB+CAM plates
Week 16, day 5 (06/10)
Made LB+AMP plates
Week 18, day 2 (17/10)
Made LB+KAN and LB+CAM plates
Week 19, day 7 (29/10)
Made 20% stock D-glucose solution by adding 20g of glucose to 100ml MilliQ water then autoclaved
Made Modified M9 Media (+0.2% glucose + 0.2% arabinose)
Interlab
Week 3, day 4 (13/07)
Prepared overnight culture of one DH5α colony in 5mL LB at 37°C
Week 3, day 5 (14/07)
Checked on DH5α culture
Initial OD600 was 0.011, so waited 110min for optimal OD600
Moved culture to ice once OD600 reached 0.466
Made DH5α stocks stored in -80°C, competence to be tested on Monday using iGEM competence cell test kit.
Week 4, day 1 (17/07)
Transformed DH5α cells using iGEM competence cell test kit with RFP/pSB1C3 at concentrations 100pg/µL, 50pg/µL, 10pg/µL, and 0pg/µL (control)
Plated the cells onto 4 LB+CAM plates, each containing 100µL of the 100pg/µL, 50pg/µL, 10pg/µL, and 0pg/µL cells.
Incubated the plates at 37°C overnight.
Week 4, day 2 (18/07)
Checked on transformed cells from previous day and moved them to the 4°C fridge.
Confirmed DH5α cells were competent
Note: LB+CAM plates made on 13/07 may have been made with too much CAM (1250µL CAM to 500mL LB). This may explain why there were so few colonies from 17/07 transformation of DH5α + RFP/pSB1C3.
Resuspended the iGEM kit parts containing the DNA required for the Interlab study (DNA was obtained from kit plate #6, which was then moved to the 4°C fridge for storage) and transformed the chemically competent DH5α cells (from 14/07) with resuspended DNA
Made 9 tubes: 8 GFP samples and one no plasmid control (labelled 20B, 20D, 20F, 20H, 20J,  20L, 20N, and 20P, respectively).
Originally, the goal was to make 17 tubes for two transformations (8 samples x 2 + 1 no plasmid control), however, there were not enough cells to to make a second set (missing second set of 20L and 20B).
Spread 100µL of transformed bacteria on LB+CAM plates to incubate and dry overnight at 37°C.
Week 4, day 3 (19/07)
Examined plates from yesterday
Most plates had no cells likely due to the high CAM concentration (2.5x working concentration).
07-19 plates
Re-plated DH5α cells transformed with Interlab study plasmids from transformation tubes done on Tuesday and incubated overnight in 37°C shaker.
Week 4, day 4 (20/07)
Obtained 19/07 GFP plates from the 37°C shaker.
Picked a single colony from each plate and incubated in 1.5mL LB media overnight in 37°C shaker and left to grow in the 37 degree C shaker.
In total, there were 8 sample tubes containing the GFP interlab samples and one control tube of just LB.
Made 5mL cultures of the DH5α cells transformed with GFP plasmids and placed them in the overnight shaker to grow.
Week 4, day 5 (21/07)
Plated transfromed DH5a
Week 4, day 6 (22/07)
Checked plates of DH5a cells transformed with Interlab plasmids (from 21/07).
Colonies grew into a lawn of bacteria and no single colonies could be isolated.
Week 5, day 2 (25/07)
Obtained single colonies from the GFP sample plates (21/07) and plated them on LB+CAM plates. The plates were left at 37°C to grow overnight.
Week 5, day 3 (26/07)
Examined LB+CAM plates and saw single colonies
Inoculated single colonies from plates into LB+KAN and LB+CAM and put in incubator
Week 5, day 4 (26/07)
Made 80x1mL glycerol stocks of 26/07 cultures and stored in -80°C freezer
Week 5, day 5 (28/07)
Ran LUDOX-S40 calibration
Table 1: LUDOX Calibration
100% LUDOX
H<sub>2</sub>O
0.0508
0.0402
0.0561
0.0415
0.0525
0.0419
0.0527
0.0435
Week 6, day 1 (31/07)
Transferred samples 20J, 20L, 20N, and 20P from glycerol aliquots (from 26/07) to a 96-well plate and measured maximum emission and exciation wavelengths of GFP and YFP.
GFP: Literature said the maximum exictation wavelength is 395 nm, the maximum emission wavelength is 509 nm
We measured maximum exication: 390 nm, maximum emission: 400-750 nm.
YFP: Literature said the maximum excitation wavelength is 512 nm, the maximum emission wavelength is 529 nm.
Note: No negative control was measured so data can’t be normalized.
Week 6, day 2 (01/08)
Measured the emission spectra of GFP samples 20B, 20D, and 20F with the plate reader
The interlab GFP samples use the mut3b GFP, which has a shifted excitation and emission spectra
Excitation range of 488 nm - 501 nm
Emission maximum at 511 nm
At a 395 nm excitation wavelength, this GFP mutant emits 0.2x the intensity of wild type GFP. However, at a excitation wavelength of 488 nm, the emission intensity is 21x that of wild type GFP.
The emission spectra was tested using samples 20B (positive control), 20D (negative control), and 20F from glycerol aliquots.
In total, three different excitation spectra were tested: 488 nm, 508 nm, and 495 nm.
Week 6, day 3 (02/08)
Washed glycerol stocks and then suspended in LB+CAM to ensure consistent measurements from the plate reader
All subsequent interlab samples will be measured in LB+CAM
Restreaked samples to obtain biological replicates
Measured excitation spectra using plate reader in WB 316 of GFP samples 20B, 20D, and 20F.
Week 6, day 5 (04/08)
Restreaked 20D transfomed cell
Took plate reader measurements according to Interlab protocol
Used LUDOX 100% and H2O to set the standard point
Found fluorescein fluorescence standard curve
Took measurements of all cell samples
LacILOV
Week 1, day 5 (30/06)
Bacterial transformation of competent cells with RFP plasmid
Week 2, day 5 (07/07)
Electroporation of MG1655ΔLacI electrocompetent cells.
Modified electroporation protocol to incubate transformed cells at 300rpm, 37°C after SOC recovery step.
Bacterial transformation of electroporated cells with RFP plasmid (BBa_j04450 in pSB1C3) to check transformation efficiency at 0, 10, 50 and 100pg/ul.
MiniPrepped transformed DH10β (BBa_j04450 in pSB1C3) to purify plasmid (sample 1/2 stored in -20°C)
Performed RE digestion, nicked with EcoRI-HF (sample 1/2 stored in -20°C)
Week 3, day 1 (10/07)
Miniprepped RFP plasmids on the pSB1C3 backbone from chemically competent DH10β cells from 07/07.
Conducted gel electrophoresis of RE digested and non RE digested dupicate samples from 07/07.
10µL of each ladder was used, but in the future we should experiment with adding less ladder (conserve reagents!).
Each 10µg sample was mixed with 2µL loading dye and loaded on the gel along with two ladders. The sequences are as follows:
EcoRI-HF Digested Sample 1 (D1)
Undigested Sample 1 (UD1)
1kb DNA ladder for measurement
EcoRI-HF Digested Sample 2 (D2)
Undigested Sample 2 (UD2)
1kb DNA ladder for measurement
Demonstration: Nanodropped each sample to show the protocol to general members.
Week 3, day 2 (11/07)
Calculated optimization for Q5 (High Fidelity) PCR of UNS3/2 into ends of  pSB1C3 backbone
Optimized annealing step because our primer Tms are high (85°C), so using different additives/temperature gradients to optimize yield
“07-11 Tm Calculator.png”
Table 1: PCR Optimization
Component
No treatment (µL)
3% DMSO (µL)
6% DMSO (µL)
1X GC Buffer (µL)
No DNA control (µL)
5X Q5 Reaction Buffer
5
5
5
5
5
10mM dNTPs
0.5
0.5
0.5
0.5
0.5
10µM Forward primers
1.25
1.25
1.25
1.25
1.25
10µM Reverse primers
1.25
1.25
1.25
1.25
1.25
1ng/µL Template DNA (prediluted)
1
1
1
1
0
Q5 HF DNA Polymerase
0.25
0.25
0.25
0.25
0.25
1X GC Buffer
0
0
0
5
5
DMSO
0
0.75
1.5
0
0
Nuclease-free water
15.75
15
14.25
10.75
11.75
Total volume
25
25
25
25
25
* Multiply five for 5 reactions (5 different annealing temperatures: 72, 71, 70, 69, and 68°C).
Table 2: Master Mixes
Master Mix
No treatment (µL)
3% DMSO (µL)
6% DMSO (µL)
1X GC Buffer (µL)
No DNA control (µL)
GC Buffer
0
0
0
40
40
DMSO
0
6
12
0
0
Nuclease-free water
126
120
114
86
94
Sum
126
126
126
126
134
Week 3, day 3 (12/07)
Performed Q5 HF PCR (discussed 11/07) with modifications:
Revised PCR calculations based on Christian’s suggestions: 7 Q5 HF PCR 50µL reactions with 3% DMSO in each sample (one control tube with no DNA template).
Table 1: Revised PCR Calculations
Component
Reaction (×7)
Control
5X Q5 Reaction Buffer
10
10
10mM dNTPs
1
1
10µM Forward primers
2.5
2.5
10µM Reverse primers
2.5
2.5
1ng/µL Template DNA (prediluted)
1
0
Q5 HF DNA Polymerase
0.5
0.5
1X GC Buffer
10
10
100% DMSO
1.5
1.5
Nuclease-free water
31
32
Prepared master mix for Q5 HF PCR reactions (with 50µL 3% DMSO). Template DNA is included in 6 tubes with 1 control without template DNA.
One template DNA tube only contains 25 uL, possibly due to pippetting error
Note: prepare a master mix of 8 reactions of 50 μL, instead of 7.1 reactions, to accommodate pippeting errors).
Ran PCR. Tubes were meant to be optimized through a temperature gradient, but were placed incorrectly in the PCR machine, so they were run all at the same temperature.
Ran a gel electrophoresis of psB1C3 backbone PCR. The gel was not successful. (Life: What a cruel mistress)
Note: Gel was made with 15 wells - too small for 10µl samples (15 wells should use ~6-8µl).
Week 3, day 4 (13/07)
Performed Q5 HF PCR to amplify pSB1C3 backbone
Prepared same master master mixes as calculated 12/07 for 6 samples with template DNA and one control without template DNA
Ran the PCR of samples 1-6 at different temperatures, one each at 72.0, 71.4, 70.1, 68.3, 66, and 64.3°C respectively
Control was tested at 70.1°C
Performed gel electrophoresis of Q5 PCR samples, along with EcoRI digested pSB1C3 with RFP control
Advisor suggested results may be due to primer dimerization, possibly because of high primer Tm
Planned to redesign primers to reduce this problem by removing base pairs from the 3’ binding up to ~15bp homology between primers and backbone
Aiming for Tm of ~68°C
Week 3, day 5 (14/07)
Redesigned primers based on suggestions from 13/07 to have lower Tm
New reverse primer for pKDL071 backbone
New forward and reverse primers for pSB1C3 backbone
The Q5 polymerase used in previous PCRs is very bubbly which could have been a source of error, so instead we performed 2-step NEB Phusion HF PCR of pSB1C3 with RFP construct
Used primers from 2016 and 2017 to compare results
Tubes were labelled 16.1-16.5 (for 2016 primers) and 17.1-17.5 (for 2017 primers)
Controls without DNA template were labelled NTC (no template control)
Ran the samples on a temperature gradient to determine the optimal PCR settings
Table 1: Phusion PCR Reaction Component Table (for total volume of 50µL per tube)
| Component | 16.NTC | 16.1 | 16.2 | 16.3 | 16.4 | 16.5 | 17.NTC | 17.1 | 17.2 | 17.3 | 17.4 | 17.5 |
| — | — | — | — | — | — | — | — | — | — | — | — |
| 5X Phusion HF or GC Buffer | 10µL | 10µL | 10µL | 10µL | 10µL | 10µL | 10µL | 10µL | 10µL | 10µL | 10µL | 10µL |
| 10mM dNTPs | 1µL | 1µL | 1µL | 1µL | 1µL | 1µL | 1µL | 1µL | 1µL | 1µL | 1µL | 1µL |
| 10µM 2016 Forward Primer | 2.5µL | 2.5µL | 2.5µL | 2.5µL | 2.5µL | 2.5µL | 0µL | 0µL | 0µL | 0µL | 0µL | 0µL |
| 10µM 2016 Reverse Primer | 2.5µL | 2.5µL | 2.5µL | 2.5µL | 2.5µL | 2.5µL | 0µL | 0µL | 0µL | 0µL | 0µL | 0µL |
| 10µM 2017 Forward Primer | 0µL | 0µL | 0µL | 0µL | 0µL | 0µL | 2.5µL | 2.5µL | 2.5µL | 2.5µL | 2.5µL | 2.5µL |
| 10µM 2017 Reverse Primer | 0µL | 0µL | 0µL | 0µL | 0µL | 0µL | 2.5µL | 2.5µL | 2.5µL | 2.5µL | 2.5µL | 2.5µL |
| Template DNA | 0µL | 1µL | 1µL | 1µL | 1µL | 1µL | 0µL | 1µL | 1µL | 1µL | 1µL | 1µL |
| DMSO | 1.5µL | 1.5µL | 1.5µL | 1.5µL | 1.5µL | 1.5µL | 1.5µL | 1.5µL | 1.5µL | 1.5µL | 1.5µL | 1.5µL |
| Phusion DNA Polymerase | 0.5µL | 0.5µL | 0.5µL | 0.5µL | 0.5µL | 0.5µL | 0.5µL | 0.5µL | 0.5µL | 0.5µL | 0.5µL | 0.5µL |
| Nuclease-free water | 32µL | 31µL | 31µL | 31µL | 31µL | 31µL | 32µL | 31µL | 31µL | 31µL | 31µL | 31µL |
Table 2: Phusion PCR Temperature Gradient
Temperature
Tubes
62°C
16.1, 17.1
63.7°C
16.2, 17.2
68°C
16.3, 16.NTC, 17.3, 17.NTC
70.5°C
16.4, 17.4
72°C
16.5, 17.5
Performed gel electrophoresis of Phusion PCR samples
“07-14 gel of Phusion PCR”
Week 4, day 1 (17/07)
Performed bacterial transformation
Transformed DH5α cells using iGEM competence cell test kit with pSB1C3+RFP at concentrations 100pg/µL, 50pg/µL, 10pg/µL, and 0pg/µL (control)
Transformed DH10β cells with pKDL071 or no plasmid control
Plated transformed cells.
Spread the DH5α cells onto 4 LB+CAM plates, each containing 100µL of the 100pg/µL, 50pg/µL, 10pg/µL, and 0pg/µL cells.
Spread the DH10β cells onto two LB+KAN plates, both containing 100µL of the control and either 100µL or 50µL of the pKDL071 cells
Incubated the plates at 37°C overnight.
Week 4, day 2 (18/07)
Checked on 17/07 transformed cells and moved them to the 4°C fridge.
Confirmed DH5α cells were competent
Note: LB+CAM plates made on 13/07 may have been made with too much CAM (1250µL CAM to 500mL LB). This may explain why there were so few colonies from 17/07 transformation of DH5α + RFP/pSB1C3.
Obtained single colonies from DH5α and DH10β plates from 17/07 and made overnight 37°C colonies.
The DH5α was placed in the 5mL of LB+CAM (5µL of stock CAM added to 5mL LB).
The DH10β was placed in 5mL of LB+KAN (25µL of stock KAN added to 5mL of LB).
Week 4, day 3 (19/07)
Examined plates from 18/07
Most plates had no cells likely due to the high CAM concentration (2.5x working concentration).
Miniprepped DH10β and DH5α to extract plasmides (pKDL071 and pSB1C3 with RFP respectively) and did Nanodrop on the results
Table 1: Nanodrop results from Miniprep
Plasmid
260/280
260/230
ng/µL
pSB1C3 #1
1.82
1.42
86.6
pSB1C3 #2
1.85
1.52
87.9
psB1C3 #3
1.79
0.99
136.6
pKDL071 #1
1.89
1.55
60.0
pKDL071 #2
1.78
1.25
66.0
pKDL071 #3
1.95
1.90
48.7
Performed Phusion PCR of pSB1C3 with RFP construct using psB1C3 #2 and the pKDL071 FWD/RVS primers
Made 1ng/uL of pDKL071 #3 for PCR via following steps:
2μL of pDKL071 #3 Miniprep was mixed with 95.4 μL of nuclease free water
Made 100μL aliquots of 100μM from pKDL071 FWD UNS3 primer (shipped 05/07) resuspended in 785μL nuclease free water
Made 100μL aliquots of 100μM from pKDL071 REV UNS2 primer (shipped 05/07) resuspended in 761μL nuclease free water
Made a Master mix without DNA template containing:
17.5μL of 10 uM FWD_UNS3 primer
17.5μL of 10 uM REV_UNS2 primer
70uL of 5x phusion buffer
7uL of 10mM dNTPS
10.5uL 100% DMSO ( to reach a final concentration of 3%)
3.5uL Phusion polymerase
Took 18uL was taken from this master mix added 32 uL of Nuclease free water for a final volume of 50 uL as the No Template Control (NTC)
Added DNA template to the Master Mix:
6 uL 1ng/uL Miniprepped pKDL071 #3
186 uL nuclease free water
Added 50uL of the Master Mix to 5 PCR tubes
Ran Phusion PCR on temperature gradient with samples 1-5 at 62, 63.7, 68, 71.3, and 72°C respectively
NTC was at 72°C
Ran gel on 12μL samples of Phusion PCR results for 1 minute then left for 1 hour then run for 1 hour due to equipment issues.
Timing issues due might be why the ladder is not as intense.
Sample 3 appears to be our optimal temperature @ 68°C
Week 4, day 4 (20/07)
Ran Phusion PCR of the pKDL071 plasmid with RFP using the pKDL071 plasmid from Miniprep #3 (19/07).
Used the pKDL071 FWD/RVS primers shipped on 05/07 (the first set of primers designed in 2017)
Made Master Mix without DNA template or primers (for 8 reactions)
20 μL of 10 uM FWD_UNS3 primer
20 μL of 10 uM REV_UNS2 primer
80 uL of 5x phusion buffer
8 uL of 10mM dNTPS
12 uL 100% DMSO ( to reach a final concentration of 3%)
4 uL Phusion polymerase
18 uL was taken from the master mix as the No Template Control (NTC) and mixed with 32 uL of Nuclease free water.
Added DNA template to Master Mix (for 7 reactions)
7 uL of 1ng/uL miniprep#3 pKDL071
217 uL of nuclease free water
50 uL of the master mix were added to 5 PCR tubes
6 samples and one NTC run at 68°C, the optimal temperature determined on 17/07.
Optimization Phusion PCRs of pSB1C3 and pKDL071 plasmids with new UNS primers
Made 5 samples and one NTC for PCR of the pSB1C3 plasmid using pSB1C3 miniprep#2 and the new forward and reverse UNS primers.
Annealing temperature was 64°C so samples 1-5 at 72, 70.1, 66, 64.3, and 62°C respectively with NTC at 64.3°C
Made 5 samples and one NTC for PCR of the pKDL071 plasmid using pKDL071 miniprep#3, the old forward, and new reverse UNS primers
Annealing temperature was 62°C so samples 1-5 at 67, 65.1, 63.3, 61, and 57°C respectively with NTC at 63.3°C
Made Master mixes with DNA
OLD_FWD and NEW_REV pDKL071 (7 reactions)
17.5 μL of 10 uM FWD primer
17.5 μL of 10 uM REV primer
NEW_FWD_USN3 and NEW_REV_UNS2 pSB1C3  (7 reactions)
17.5 μL of 10 uM FWD_UNS3 primer
17.5 μL of 10 uM REV_UNS2 primer
Made Master mix without DNA template or primers (for 15 reactions)
150 uL of 5x phusion buffer
15 uL of 10mM dNTPS
22.5 uL 100% DMSO ( to reach a final concentration of 3%)
7.5 uL Phusion polymerase
Added 91 uL of Master mix without DNA to the pSB1C3 & pDKL071 primer master mixes
18 uL was taken from each primer master mix as the No Template Control (NTC) and added to 32 uL of Nuclease free water to reach a final volume of 50 uL.
Added DNA template to primer mixes:
pSB1C3 master with DNA template (for 6 reactions):
5.5 uL of 1ng/uL miniprep pSB1C3
170.5 uL of nuclease free water
pDKL071 master with DNA template (for 6 reactions)
5.5 uL of 1ng/uL miniprep pDKL071
170.5 uL of nuclease free water
50 uL of each master mix were added to 5 PCR tubes (10 tubes total + 2 NTCs)
Performed gel electrophoresis all PCR results
8 wells for Phusion PCR of the pKDL071 plasmid
15 wells for optimization PCR of pSB1C3 and pKDL071 plasmids
Wells 1-6 contained pSB1C3 samples 1-5 and NTC
Well 7 contained the 10kb ladder
Wells 8-13 contained the pKDL071 samples 1-5 and NTC
Made 5mL cultures of the DH5 alpha cells transformed with GFP plasmids and placed them in the overnight shaker to grow.
Week 4, day 5 (21/07)
Nanodrop PCR products of pKDL071 from 20/07
Note: NOT purified
Table 1: Nanodrop of pKDL071-UNS with old primers (at 68°C)
Sample
ng/uL
260/280
260/230
1
200.7
1.34
0.47
2
379.0
1.32
0.62
3
357.5
1.37
0.62
4
346.1
1.33
0.64
5
363.9
1.33
0.59
6
353.6
1.37
0.61
NTC
339.3
1.34
0.57
Table 2: Nanodrop of pKDL071-UNS optimization with new REV primer (at 62-72°C)
Sample
ng/uL
260/280
260/230
1
436.9
1.42
0.74
2
437.1
1.46
0.72
3
443.0
1.41
0.72
4
446.7
1.42
0.73
5
434.9
1.43
0.71
NTC
436.3
1.43
0.71
Dpn1 digest and purification of pKDL071 PCR product with new REV primer: 40uL digestion reaction each tube.
6 tubes in total, including 4 samples (D1-D4), 1 NTC and 1 positive control (miniprepped pKDL071)
Master Mix contained:
32uL of pKDL071
4uL 10X CutSmart
4uL Dpn1
Nanodropped the results
Table 3: Nanodrop of the digested and purified pKDL071 PCR product with new REV primer
Sample
ng/ul
260/280
260/230
D1
28.1
1.82
1.69
D2
27.9
1.84
1.84
D3
31.6
1.80
1.73
D4
30.5
1.94
1.84
NTC
18.3
1.79
1.43
Positive Control (minipreped pKDL071)
22.3
1.81
1.70
Performed Phusion PCR of pSB1C3 with new FWD UNS & REV UNS with temperature gradient
Made Primer Master mix (MM1) new FWD UNS3 & New REV UNS2 pSB1C3 (for 7 reactions) containing:
17.5 μL of 10 uM NEW FWD_UNS3 primer
17.5 μL of 10 uM NEW REV_UNS2 primer
Made Master mix (MM2) without DNA template or primers:
70 uL of 5x Phusion buffer
7 uL of 10mM dNTPS
10.5 uL 100% DMSO ( to reach a final concentration of 3%)
3.5 uL Phusion polymerase
Added MM2 to MM1 to make a Primer + Ploymerase mix
Took 18uL from polymerase + primer mix as NTC and added 32 uL of nuclease free water for final volume of 50uL
Added the following to make a Primer + ploymerase + DNA Mix (for 6 reactions)
6 uL of 1 ng/uL miniprep pSB1C3 (RFP)
186 uL of Nuclease free water
Divided final mix into 6 tubes, 50 uL each
Ran PCR with S1-6 in 62.3, 63.7, 66.3, 69.5, 71.3, and 62°C respectively, with NTC in 69.5°C
Ran a gel of the results
Week 5, day 1 (24/07)
Dpn1 digest of pSB1C3+UNS PCR from 21/07
4 sample tubes and 1 control without Dpn1
Gel electrophoresis of the results
Note: The absence of bands could be due to multiple reasons; the product was lost during digestion and purification or there was not enough product initially.
Nanodrop of the purified samples was preformed to test concentration of DNA purified.
Table 1: Nanodrop results of purified Dpn1 digestion of pSB1C3+UNS
Sample
260/280
260/230
ng/uL
D1
1.88
1.43
10.6
D2
1.74
1.34
11.8
D3
1.91
0.75
8.5
D4
1.87
1.64
13.6
Undigested pSB1C3
1.33
1.16
6.2
Week 5, day 2 (25/07)
Assembled reporter and mCherry constructs
Pooled the purified pKDL071+UNS product from 21/07 purification to obtain a greater amount of DNA necessary for assembly.
The resulting concentration of product, measured via Nanodrop, was 38.2 ng/uL (Contamination values: 260/280 = 1.93; 260/230 = 1.77).
Created assembly according to the HiFi DNA assembly protocol. The reporter and mCherry master mix values are listed in the table below.
Table 1: HiFi DNA Assembly Calculations
Sample
Reporter
mCherry
pKDL071+UNS
1.67uL
1.67uL
Reporter mCherry
10.81uL
N/A
Reporter YFP
9.7uL
N/A
mCherry
N/A
10uL
Total fragment volume
22.18uL
11.67uL
Buffer (HiFi MM)
22.18uL
11.67uL
Total Reaction Volume
44.36uL
23.34uL
Transformed the assembly parts into the NEB 5α competent cells (came with the HiFi DNA assembly kit).
Plated NEB 5α cells to grow overnight in the 37°C shaker.
12 LB+KAN plates(6 mCherry, 6 reporter)
4 LB+AMP plates (2 for the positive control transformation (pUC19), 2 for the positive assembly control)
Week 5, day 3 (26/07)
Examined NEB 5α plates from 25/07 and saw single colonies
Inoculated single colonies from pKDL071 mCherry and Reporter plates into LB+KAN and LB+CAM and put in incubator
Ran Phusion PCR of pSB1C3 (minprep #2) with new primers
Made 5 samples and 1 NTC
Loaded samples qPCR machine at 68°C
Week 5, day 4 (26/07)
Gel Electrophoresis of pSB1C3 Phusion PCR from 26/07
Primer dimers & no amplified product
NEB’s 2-Log-DNA Ladders were used
Miniprepped mCherry and reporter (YFP) constructs from our pKDL071 plasmid backbone from DH5a transformed bacteria (from 25/07) with subsequent nanodrop to measure concentration of DNA:
9 minipreps in total for mCherry construct:
6 from inoculation culture of colony A and B
3 from inoculation culture of colony C
6 minipreps in total for Reporter construct:
3 minipreps from inoculation culture of colony A
3 minipreps from inoculation culture of colony B
Nuclease free water was added to resuspend DNA after vacufuging the miniprepped product (to eliminate elution buffer)
Table 1: Nanodrop results of mCherry and Reporter pKDL071 plasmids
Sample
DNA Concentration (ng/uL)
260/280
260/230
Nuclease-free water (uL)
mCherry 1 (A)
20.0
1.59
1.30
22.8571428571
mCherry 2 (A)
36.6
1.78
1.74
41.8285714286
mCherry 3 (A)
38.7
1.60
0.90
44.2285714286
mCherry 1 (B)
74.5
1.54
0.74
85.1428571429
mCherry 2 (B)
25.0
1.69
1.54
28.5714285714
mCherry 3 (B)
27.2
1.65
1.35
31.0857142857
mCherry 1 ©
25.3
1.89
2.46
28.9142857143
mCherry 2 ©
24.1
2.08
2.18
27.5428571429
mCherry 3 ©
24.0
2.05
2.54
27.4285714286
Reporter 1 (A)
18.1
1.57
1.42
20.6857142857
Reporter 2 (A)
18.8
1.55
1.36
21.4857142857
Reporter 3 (A)
21.1
1.55
1.33
24.1142857143
Reporter 1 (B)
21.6
2.05
2.31
24.6857142857
Reporter 2 (B)
20.7
2.09
2.17
23.6571428571
Reporter 3 (B)
21.8
2.06
2.63
24.9142857143
Set up and re-ran Phusion PCR of pSB1C3 with same settings as 26/07
Streaked out 5 plates of pKDL071_mCherry (from colonies a, b, c) and pKDL071_Reporter (from colonies a and b) and put in 37°C incubator overnight
Week 5, day 5 (28/07)
Checked the plates with pKDL017_mCherry and pKDL017_Reporter from 27/07
Ran the gel for 2nd PCR of PSB1C3 with new primers from 27/07
Note: The exposure time might be too little (only 0.02), needs to be at least 0.35s.
Week 6, day 1 (31/07)
Ran PCR for pSB1C3 + new UNS primers (Tm Calculator indicated annealing temperature was 67°C)
6 samples at 62°C, 62.9°C, 64.9°C, 68°C, 70.5°C, and 72°C
One NTC at 68°C
Ran gel of the results
Week 6, day 3 (02/08)
Made electrocompetent MG 1655ΔLacI cells according to the electrocompetent cell protocol.
Made 80 glycerol stocks of the electrocompetent MG1655ΔLacI cells. Stored cells in -80°C freezer.
Week 6, day 4 (03/08)
Vacufuged pKDL071 plasmids (w/ mCherry and Reporter) to concentrate DNA
Transformed electrocompetent MG1655ΔLacI cells (from 02/08) with RFP competence test kit and placed in 37°C incubator overnight
Resuspended vacufuged DNA samples in 7uL of nuclease free water to reach a concentration of 35ng/uL
Week 6, day 5 (04/08)
Checked competence of MG1655ΔLacI cells (from 03/08) - the results were good.
Nanodropped mCherry and Reporter DNA samples from 03/08
Table 1: Nanodrop results of vacufuged mCherry and Reporter DNA
Sample
ng/uL
260/280
260/230
mCherry A1
67.3
2.74
1.00
mCherry A2
14.6
3.6
1.14
mCherry A3
17.2
5.23
0.83
mCherry B1
26.4
2.86
1.06
mCherry B2
38.6
2.76
1.16
mCherry B3
46.2
2.58
1.23
mCherry C1
56.2
2.61
1.23
mCherry C2
60.8
2.37
1.17
mCherry C3
48.4
2.57
1.23
Reporter A1
45
2.86
1.13
Reporter A2
47.5
2.69
1.06
Reporter A3
57.5
2.76
1.07
Reporter B1
81.9
2.69
1.1
Reporter B2
54.2
2.8
1.07
Reporter B3
54.3
2.63
1.11
Performed Colony PCR of mCherry + Reporter construct
Ran gel of colony PCR results
Week 7, day 2 (08/08)
Ran 12-well gel of 04/08 PCR products: pKDL071 plasmid in wells 2-8; pSB1C3 plasmid in wells 9-12
DNA ladder 10kb biobasic
Reporter A2
Reporter B3
mCherry A1
mCherry B2
mCherry C2
Positive control
Negative control
Sample 1
Sample 2
Sample 3
Negative control.
Repeated colony PCR of mCherry and reporter constructs in the pKDL071 plasmid
PCR was run with samples mCherry A1, B3, and C1, and Reporter A1 and B3.
Samples were first diluted to concentrations of 1 ng/uL based on nanodrop results from 04/08.
Gel electrophoresis of colony PCR of mCherry and reporter constructs.
Week 7, day 3 (09/08)
Q5 PCR with Temperature Gradient (62-72°C) of PKDL071 with mCherry sample A1
Samples 1-5 at 72.0°C, 70.0°C, 68.3°C, 64.3°C, 62.0°C, and 66.0°C
NTC at 66.0°C
NEB OneTaq Hot Start PCR with temperature gradient of pSB1C3 to add UNSs with new primers. Tm Calculator indicated annealing temperature of 54°C.
Samples 1-6 at 61.0°C, 59.1°C, 57.3°C, 53.5°C, 51.9°C, 51.0°C, 57.3°C
NTC at 57.3°C
Transformation of MG1655ΔLacI with Reporter construct and mCherry construct respectively, both in pKDL071 bakcbone.
Gel eletrophoresis of both PCRs
Week 7, day 4 (10/08)
Examined transformed MG1655ΔLacI with pKDL071 plasmid with insert from 09/08 (NC, Rep A, B, mCherry A, B, C)
NEB OneTaq Hot Start PCR with temperature gradient of Christian’s inserts (using colony PCR primers to test if they are working)
Made Master Mix (8 reactions) (50uL)
80uL GC Reaction Buffer
8uL of 10uM dNTP
8uL of 10uM FWD
8uL of 10uM REV
4uL of OneTaq Polymerase
12uL of 100% DMSO (to make final concentration of 3%)
Made NTC with 15uL of MM and 35uL nuclease free water
Added DNA to MM:
7uL of DNA Template
238 uL nuclease free water
Added 50uL final mix to 6 tubes and ran the PCR
Gel electrophoresis of PCR products showed colony PCR primers weren’t working
Week 7, day 5 (11/08)
Designed new pKDL071 colony PCR primers
BLASTED against pKDL071 w/ UNSs, also BLASTED against E.coli K-12
Highest off target E-value was 0.46 for both FWD and RVS.
When analyzed with IDT OligoAnalyzer Tool the highest Delta G: -6.75 kcal/mole (Base Pairs3) was found for hetero-dimerization between fwd and rvs primers.)
“08-11 primer design.png”
Notes on how to interpret IDT OligoAnalyzer Output:
For self-dimer analysis, click on ‘Self-Dimer’ to bring up a new window with each possible self-dimer your oligo can form. For each diagram you will be able to see the calculated delta G value for this secondary structure. If you have a strong delta G (-9kcal/mol or more negative) this oligo could be problematic.
Heterodimer analysis and hairpin formation is same as above
“08-11 analysis.png”
Week 8, day 4 (17/08)
Received new pSB1C3 and pKDL071 w/ UNS2/UNS3 FWD/REV primers
Phusion PCR with temperature gradient (62°C to 72°C) to put UNS primers onto pKDL071 and pSB1C3 backbone
6 pKDL071 samples + NTC and 6 pSB1C3 samples + NTC
Gel Electrophoresis of PCR products to determine size
Redid Phusion PCR for pSB1C3 with new primers with increased template; also tested 2016 pSB1C3 primers with increased template
Made Master Mix (MM1): (these were done for BOTH old (2016) and new (2.0) primers) for 8 reactions (50 μL each)
80uL of 5x Phusion HF Buffer
8uL of 10mM dNTPs
20uL FWD primer
20uL REV primer
12uL of 100% DMSO
4uL Phusion polymerase
Made NTC with 18uL of MM1 and 32uL of nuclease free water
Added DNA to MM1 to make MM2
50uL (1ng/uL) template
174uL nuclease free H2O
Made 6 50uL PCR tubes from MM2
Gel electrophoresis of PCR products
Dpn1 digest of pKDL071 to get rid of template
PCR purification and Nanodrop of digested pKDL071
Table 1: Nanodrop results of purified Dpn1 digested pKDL071
| Sample    | ng/uL | 260/280   | 260/230 |
| — | — | — |
| 4 | 12.2  | 2.55  | 0.91 |
| 6 | 11.6  | 2.80  | 0.75 |
*Note: samples 3/4 and 5/6 were combined into sample 4 and 6, receptively.
Performed Gibson HiFi DNA assembly, adding 0.1 pmol of fragments and 0.05pmol of inserts
Table 2: Reaction Component Table for Gibson HiFi DNA Assembly
Component
Reporter
mCherry
pKDL071 + UNS BB
5.23 uL
5.23 uL
Reporter_mCherry
10.81 uL
-
Reporter_YFP
9.7 uL
-
LacILOV_mCherry
-
10 uL
Week 8, day 5 (18/08)
Gibson plates from 17/08 (as well as the positive control supplied by Gibson kit) showed no colonies
PCR of pSB1C3 UNS with 2.0 UNS primers using a linearized pSB1C3 backbone to control for supercoil interference. We also ran a temperature gradient: 50-71°C. Phusion HF polymerase was used as well as 3% DMSO.
Gel electrophoresis of PCR products:
“08-18 pSB1C3 gel.png”
Result: Primer Dimers and what appears to be a slightly larger product (we hypothesize that these could be the palindromic ends of our UNS primers (backbone complement) annealing and amplifying.
Redid Gibson assembly based on new Nanodrop of samples
Table 1: Nanodrop results of purified Dpn1 digested pKDL071
| Sample    | ng/uL | 260/280   | 260/230 |
| — | — | — |
| 4 | 16.8  | 1.63  | 1.23 |
| 6 | 16.1  | 1.63  | 0.72 |
Table 2: Reaction Component Table for Gibson HiFi DNA Assembly
Component
Reporter
mCherry
pKDL071 + UNS BB
4 uL
4 uL
Reporter_mCherry
10.87 uL
-
Reporter_YFP
9.7 uL
-
LacILOV_mCherry
-
10 uL
Week 9, day 1 (21/08)
OneTaq PCR of Christian’s miniprepped pKDL071 plasmid with backbone specific PCR colony primers to test them out with temperature gradient from 45-55°C
Gel electrophoresis of PCR product:
Gel electrophoresis for assembled pKDL071 Reporter ® and mCherry (M) plasmids from 25/07 (1), 17/08 (2), and 18/08 (3)
Made 5mL overnight LB+KAN cultures of pKDL071 mCherry colonies 1, 2, and 3 in preparation for miniprep the next day
Week 9, day 2 (22/08)
Miniprepped Gibson assembly colonies A through C
Performed EZ-10 spin column PCR products purification kit of all PCR reactions using 2.0 primers on pSB1C3 backbone (samples 9-14 from 17/08 and samples 1-11 from 17/08)
Purpose: to find out if the backbone + UNS was actually amplified during PCR, and was not visible due to low yield.
The purified products would then be combined to obtain enough product to run a gel extraction.
Nanodropped PCR purified pKDL071+UNS samples 4 and 6 from 17/08 and miniprepped plasmid with mCherry in pKDL071 (colonies A,B and C)
Table 1: Nanodrop results of PCR purified pKDL071+UNS and mCherry+pKDL071 colonies
Sample
ng/uL
260/280
260/230
4
1.85
1.02
40.7
6
2.11
0.91
25.4
A
1.85
1.46
68.6
B
1.76
1.17
74.9
C
1.90
1.45
77.2
OneTaq PCR of purified plasmid: 6 samples and 1 NTC
Gel electrophoresis of PCR purified pSB1C3
“08-22 PCR purified pSC1C”
Gel electrophoresis of LacILOV mCherry from colonies and colony PCR products
“08-22 OneTaq HotStart pKDL071 mCherry with 2.0 Colony PCR primers”
Gibson assembly of the reporter constructs
Week 9, day 3 (23/08)
Vacufuged assembly from 22/08 and took Nanodrop each time
Ran a gel of miniprepped reporter
Week 9, day 4 (24/08)
Gibson Assembly of mCherry_LacILOV insert and YFP_Reporter insert
Note: we put the tubes in the PCR at 50°C instead of using thermocycler
Constructed the light array
Transformed DH5α cells with Gibson assembled mCherry_LacILOV_YFP_reporter insert
Week 9, day 5 (25/08)
Checked plates from yesterday and got 1 colony
Colony PCR of reporter insert (resuspended colony in water)
RE Digestion (pstI)
Gel electrophoresis in order: 2-log NEB ladder, Christian’s insert, Reporter insert, NTC
Week 10, day 1 (29/08)
Phusion PCR with colony PCR primers on our Gibson Assembly product
PCR of LacILOV_mCherry insert WITH restriction enzyme primers
Week 10, day 2 (30/08)
Ran the gel for PCR purified Reporter and LacILOV.
Gibson Assembly of mCherry and YFP in pKDL071
RE Digest of reporter and pSB1C3
NEB Transformation protocol (E2621) for HIFI Gibson
3 transformations: reporter, transformation, and assembly
T4 DNA ligation of pSB1C3+reporter
Nanodropped LacILOV mCherry gel
Week 10, day 3 (31/08)
RE Digest (XbaI, PstI) of LacILOV & mCherry from gel
Transformation of rep_pSB1C3 into DH10B
Gibson assembly of reporter + pJDK071 with 1:1 ratio insert:BB
Week 10, day 4 (01/09)
Colony PCR :
Transformed DH10B cells with ligated pSB1C3 & Reporter construct (3:1, reporter: Backbone) using OneTaq Hot Start polymerase.
Transformed reporter + pKDL071 HiFi DNA assembly with 1:1:1 of Reporter_YFP: Reporter_mCherry: pKDL071 w UNS, using OneTaq Hotstart
Streaked reporter_pSB1C3 and reporter_pKDL071 colonies onto new LB+CAM plates
Transformed ligated reporter + pSB1C3 (1:3, reporter: Backbone) and ligated LacILOV_mCherry+ pSB1C3(3:1, insert:Backbone) into DH10B cells
Week 11, day 2 (05/09)
OneTaq Hot Start Colony PCR on colonies from 01/09 (pSB1C3_reporter & pSB1C3_mCherry_LacILOV (transformed into DH10Beta))
Phusion PCR: pSB1C3 backbone amplification using 2016 primers
Overnight culture of LacILOV_mCherry insert in pSB1C3 plasmid in DH5beta
Week 11, day 3 (06/09)
Redid Colony PCR w/ LacILOV_mCherry & Reporter constructs and Phusion PCR for amplification of pSB1C3 backbone (with 2016 primers) from 05/09
Miniprepped our overnight culture of DH10Beta with LacILOV construct in pSB1C3
Phusion PCR of Gibson Mix with new UNS 2 REV and UNS 3 FWD primers
Week 11, day 4 (07/09)
Nanodrop results of pooled PCR purified samples from 06/09
Redid Phusion PCR from 06/09
RE Digest for PCR with RE primers of pSB1C3
PCR purification of digested pSB1C3
Week 11, day 5 (08/09)
Phusion PCR of LacILOV_mCherry insert from plasmid (inserted into both BBs) + directly from IDT stock to test UNS primers
Ran gel for PCR purified reporter construct with RE sites from 29/08
Gel extract of single band on gel, following protocol from NEB Gel extraction kit
Nanodropped results
Phusion PCR of gel extracted reporter
PCR purified PCR product from phusion PCR of reporter construct with RE sites using UNS primers (07/09)
Nanodropped product
Gel of purified product
Gibson Assembly of pKDL071 + reporter with vector: insert (2:1 ratio) from 22/08
Transformed Gibson products following NEB Gel extraction protocol
Ligated dephosphorylated + digested pSB1C3 BB from 07/09 and gel extraction product from today
LacILOV_mCherry colony and completely resuspended in microcentrifuge tube with LB+Kan
Week 11, day 6 (09/09)
We got colonies! From HiFi assembly of reporter+pKDL071 in 2:1 vector:insert ratio from 08/09
Colony PCR to test
Phusion PCR with UNS primers for (A) Gibson mix HiFi DNA assembly from 08/09, and B) IDT LacILOV_mCherry insert
Phusion PCR with RE primers for Gibson mix (HiFi assembly) from 08/09 with temperature gradient
Week 12, day 1 (11/09)
Colony PCR of reporter construct assembled 08/09 using UNS primers to eliminate non-specific binding possibility of colony PCR primers
Crossover PCR of reporter subpart YFP & reporter subpart mCherry to assemble reporter construct
Inoculated 5ml overnight cultures with all reporter colonies
Performed LacILOV_mCherry end point measurement and start new assay
Week 12, day 2 (12/09)
Temperature gradient PCR with UNS 3 FWD and UNS 2 REV primers with LacILOV_mCherry in pKDL071 and pSB1C3
LacILOV_mCherry pSB1C3 miniprep
Overlap PCR of reporter inserts
Week 12, day 3 (13/09)
Miniprepping reporter construct in pKDL071
New UNS 4 fwd and rev primers
PCR purified reporter constructs from PCR amplification
Week 12, day 4 (14/09)
Phusion PCR of miniprep’d “reporter” plasmid in pKDL071 using colony PCR primers
Week 12, day 5 (15/09)
RE Digest of minipreped “reporter” plasmids R1, R2, and RA (used 10ul of plasmid in each rxn) from gibson assemblies with PKDL071 back bone.
Ran gel of PCR purified reporter inserts
DNA Gel Extraction and Nanodropped results
Phusion PCR temp gradient of reporter mcherry with UNS primers (65-55C)
Week 13, day 1 (18/09)
Redid reporter mCherry temperature gradient using PCR purified mCherry sample from 14/09
Week 13, day 2 (19/09)
Phusion PCR of LacILOV_mCherry in pKDL071 using UNS primers to test if PCR reagents are good
Week 13, day 3 (20/09)
PCR amplification of YFP reporter & mCherry reporter from PCR purified product and gel extracted product.
Used 1:10 diluted PCR purified YFP and mCherry and Gel extracted YFP and mCherry will as template for PCR
Nanodropped Reporter Parts
Amplification of reporter insert from ALL previous Gibson Assemblies using UNS 3 FWD and UNS 2 REV
Final PCR: PCR of LacILOV_mCherry purified (1rxn + NTC + buffer)
Week 13, day 4 (21/09)
RE (BglII) digest of LacILOV_mCherry in pKDL071
Week 13, day 5 (22/09)
Note: From PCR results done on 21/09, PCRing after PCR purification is not usually successful due to incomplete amplicons from the first PCR
RE Digest results:
Bgl II digest of LacILOV mcherry yielded unexpected results and digestion will be repeated today for a longer incubation period (~2.5h)
DpnI digest of PCR puified pKDL071; was not digested beforePCR purification, PCR purify after digest.
Bgl II digest of LacILOV mcherry PKDL071, LacILOVmcherry pSB1C3, and Christan’s pKDL071 (digested for 2.5h)
Week 14, day 1 (25/09)
Ran temperature gradient PCR with colony PCR primers of LacILOV mCherry with Phusion HF to test if OneTaq gave artifacts
Started overnight culture of LacILOV_mCherry for assays
Designed primers to add RE sites (xbai and psti) to LacILOV, reason: be able to clone LacILOV into pSB1C3 for parts submission
Week 14, day 2 (26/09)
PCR (25ul rxn, 6 rxns) using colony PCR primers to check if successfully cloned reporter insert into pKDL071
Phusion PCR to amplify this to insert reporter_mCherry
DpnI digest of pKDL071 amplified backbone with UNS 2 at 5’ end and UNS 4 at 3’ end
combined PCR samples 1&2, 3&4
PCR purification and Nanodrop of dpnI digested pKDL071
Week 15, day 3 (27/09)
Gibson Assembly of Reporter_mCherry and pKDL071 BB with UNS 2 + 4
Reporter_mCherry will be used from IDT stock
pKDL071 BB with UNS 2 and 4 will be used from sample 1 prepared 26/09
Phusion PCR using UNS 4 FWD and UNS 3 REV to amplify pKDL071
Transformed of DH5a cells with our mCherry reporter in pKDL071
Spreaded LB+Kan plates and placed in incubator overnight @ 37oC
Week 15, day 4 (28/09)
Got 28 colonies from 27/09!
Colony PCR with OneTaq and colony PCR primers
DpnI digest of pKDL071 BB using UNS4 and UNS3 primers
PCR purification of DpnI digested pKDL071
Nanodrop of dpnI digested + PCR purified pKDL071 BB
Week 15, day 5 (29/09)
Started an overnight culture of colonies 1,4,5,6,14,18,26,29 (streaked onto new plates) in LB+KAN
Colony PCR of colony 29 from 28/09
Week 15, day 6 (30/09)
Miniprepped reporter_mCherry_pKDL071 colonies 1,4,5.6.14.18.26 &29
PCR amplified those plasmids with UNS4 FWD & UNS2 REV primers
Week 16, day 1 (02/10)
Phusion PCR to add UNS 3 to reporter mCherry in pKDL071 - will make it easier to add YFP reporter (Gibson)
Week 16, day 2 (03/10)
Received LacILOV Fwd and rev primers adding cut sites XbaI and PstI and YFP reporter stock from IDT
Phusion PCR for LacILOV PCR with RE sites
PCR amplification of reporter_mCherry colonies with UNS 4 FWD and pKDL071 UNS 3 rev to linearized plasmid and add UNS 3 for Gibson assembly of reporter_YFP
Week 16, day 3 (04/10)
Prepared samples for sequencing:
Reporter_mCherry in pKDL071
LacILOV_mCherry in pSB1C3
Week 16, day 4 (05/10)
DpnI digest of LacILOV AND digestion of LacILOV using XbaI/PstI
PCR purified digested LacILOV_with_RE sites
Week 16, day 5 (06/10)
Transformed ligated LacILOV in pSB1C3 to DH5B cells
Gibson assembly of reporter YFP with reporter_mCherry in pKDL071 from colony 1
Transformed into NEB Competent cells DH5a, plated on LB+AMP and incubated overnight
Week 16, day 6 (07/10)
Colony PCR
Week 17, day 2 (10/10)
Received mCherry reporter construct from IDT, dCas9 on CamR/TetR controlled plasmid from AddGene
DH5a-pKDL071-reporter colonies 13,20,21,22,23 were restreaked on LB+Kan and overnight cultures of these colonies were also made
dCas9 containing cultures were streaked onto an LB+Cam plate and put in 37°C incubator
Week 17, day 3 (11/10)
Miniprepped pKDL071_reporter plasmids from colonies 13, 20, 21
Made overnight culture of pdCas9 cells to miniprep the next day
Phusion PCR of reporter colonies with UNS 3 FWD and UNS 2 REV primers (7rxns, 25ul each)
Prepared sequencing primers for sequencing reporter construct in pKDL071
seq 1,2,3,4,5,6
To amplify pSB1C3 out of LacILOV mCherry
primers for UNS3 REV and UNS 2 FWD
UNS 5 REV - sgRNA + antiCRISPR
add UNS 3 beginning and UNS 2 REV for antiCRISPR
Transform MG1655ΔLacI cells with reporter pKDL071 colony 21 and 22
Reporter mCherry pKDL071 was also transformed into MG1655ΔLacI as a +ve control for mCherry expression in LacILOV assays
RFP_pSB1C3 (10pg/ul) was used as +ve transformation control
Miniprepped pdCas9 from overnight culture (1.5ml into 2 tubes)
Week 17, day 5 (13/10)
Phusion PCR for amplifying pSB1C3 backbone out of LacILOV with UNS 2 and UNS 3
Week 17, day 6 (14/10)
DpnI digest of pSB1C3 with UNS 2 and 3 PCR product
PCR purification of DpnI digested pSB1C3 UNS2 and 3 PCR product
Week 18, day 2 (17/10)
Phusion PCR of reporter_mCherry in pKDL071 and of AcrIIA4 and LacILOV with new UNS primers
DpnI digest of LacILOV and pKDL071 PCR product
PCR purification and Nanodrop of LacILOV, reporter_mCherry_pKDL071 with UNS 5, AcrIIA4, and reporter construct
Calculated Gibson Assembly of AcrIIA4, LacILOV, and sgRNA
Week 18, day 3 (18/10)
Transformed into DH5a cells and incubated at 37oC overnight (3plates each):
AcrIIA4 in pSB1C3
sgRNA_reporter mCherry in pKDL071
LacILOV in pSB1C3
Week 18, day 4 (19/10)
Colony PCR of 18/10 plates
AcrIIA4 using colony rev/fwd primers
LacILOV using colony rev/fwd primers
sgRNA_reporter mCherry
Week 18, day 5 (20/10)
Measurement of LacILOV_mCherry after diluting and growing overnight:
First measurement was 12hrs after dilution
Note: LacILOV_pSB1C3 colony 9 was incubated from colony PCR sample after 5hrs from preparation as colony labeling on the plate was not clear. LacILOV_pSB1C3 colony 9 overnight culture didn’t grow and was not miniprep’d
Miniprepped overnight cultures from colonies obtained after assembly
Week 18, day 7 (22/10)
Phusion PCR of :
Reporter_mCherry_sgRNA_pKDL071 with UNS 4 FWD and pKDL071 UNS 3 REV (3rxn + NTC + extra) to assemble full construct
Reporter_mCherry_sgRNA_pKDL071 with UNS 5 REV and pKDL071 UNS 6 FWD (3rxn + ntc + extra) to assemble AcrIIA4 for antiCRISPR assays
Reporter_pKDL071 colony 21 & 22 with UNS 3 FWD and UNS 4 REV (2rxn)
Reporter_pKDL071_colony 21 and 22 with UNS 4 FWD and UNS 2 REV (2rxn)
DpnI digest of sgRNA_reporter mCherry for full construct and sgRNA_reporter mCherry_pKDL071 for AcrIIA4 assembly
PCR purification of S1 and AS1
Nanodropped results
Redid PCR of:
Reporter_pKDL071 (colony 21 & 22) with UNS 3 FWD and UNS 4 REV
Reporter_pKDL071 colony 21 and 22 with UNS 4 FWD and UNS 2 REV. (reporter_mCherry_pKDL071 will be used as +ve control) (3rxns)
Gibson assembly:
AcrIIA4_YFP subparts into sgRNA reporter_mCherry_pKDL071 (assembly of full construct)
AcrIIA4 alone into sgRNA_reporter_mCherry_pKDL071 with UNS 5 and 6
Transformed MG1655ΔLacI cells with:
sgRNA_rep_mCherry_pKDL071 colony 1 and pdCas9 (1ul of each plasmid will be used)
BBa_K523013 pSB1C3 to be used as +ve control for YFP found in plate 1 well 8H
Colony PCR of:
Full construct (AcrIIA4_YFP_sgRNA_repmCherry)
sgRNA rep_mCherry in pKDL071 with UNS 5 and 6 and AcrIIA4
Streaked MG1655ΔLacI pdCas9 + sgRNA_pKDL071 on a new plate to get single colonies
Started overnight culture of MG1655ΔLacI pdCas9 + sgRNA_pKDL071 for assays
Started overnight culture of MG1655ΔLacI YFP to trouble shoot YFP measurements
Week 19, day 2 (24/10)
Miniprep and Nanodrop colonies F3 and F7, S2 and S5
Phusion PCR to take:
Reporter construct out of pKDL071 (reporter colony #21)
Full construct out of miniprep’d pKDL071 (to Gibson into pSB1C3 tomorrow)
DpnI digest of reporter construct [50ul rxn]
Gibson of reporter construct with pSB1C3…UNS 2 and 3
Week 19, day 3 (25/10)
Phusion PCR of pSB1C3 out of LacILOV pSB1C3 to add UNS 5 and extend UNS 3 for Gibson of full construct in pSB1C3
PCR purification of full construct (F3 and F7)
Colony PCR of reporter in pSB1C3 (twice)
Redid Phusion PCR from 24/10
Full construct PCR out of pKDL071 (2rxns) using UNS primers
Full construct PCR out of pKDL071 (F3+F7) using phusion and pKDL071 colony PCR primers
Week 19, day 4 (26/10)
Sent samples for sequencing:
LacILOV_pSB1C3 (L3 + L4)
AcrIIA4_pSB1C3 (A1+A3+A7)
sgRNA_repmCherry_pKDL071 (S1)
AcrIIA4_sgRNA_repmCherry (S2+S5)
Colony PCR of reporter_pSB1C3 Colonies 7-20
Colonies 10,11,15 to be streaked onto new plates and miniprep’d
Week 19, day 5 (27/10)
Miniprep of reporter_pSB1C3 colonies 10,11,15
XbaI and PstI digest of :
LacILOV_pSB1C3 (L3 and L4)
AcrIIA4, pSB1C3 (A1, A3 and A7)
Reporter_pSB1C3
Colony PCR (10ul/rxn, 14 rxns) + 1 buffer
Week 19, day 7 (29/10)
Minipreped full construct from overnight culture
Transformed full construct plasmids + pdCas9 in MG1655 delta LacI
Started overnight cultures:
reporter_mCherry_pKDL071 + pdCas9 MG1655ΔLacI in minimum media + 0.2% glucose
reporter_mCherry_sgRNA_pKDL071 + pdCas9 MG1655ΔLacI in minimum media + 0.2% glucose
reporter_mCherry_sgRNA_AcrIIA4_pKDL071 + pdCas9 MG1655ΔLacI in minimum media + 0.2% glucose
Streaked LacILOV_mCherry_pKDL071 MG1655ΔLacI on 4 LB+Kan plates to test LOKI
Grew :
MG1655ΔLacI + YFP +ve to measure YFP
MG165ΔLacI repmCherry
pdCas9 cells
Week 20, day 1 (30/10)
Restreaked 9 colonies (labeled) of MG1655ΔLacI_sgRNA_Acr_reportermCherry in pKDL071 backbone onto LB + CAM + KAN plates
Contents
Related Pages
Experiments
Interlab
Results
General Pages
Home
Description
Team
Attributions
Collaborations
Safety
Wet Lab
Design
Parts
Hardware
Experiments
Notebook
InterLab
Results
Demonstrate
Dry Lab
ODE
Analysis
Protein Modelling
Model
Human Practices
Interviews
Genetic Code Workshop
Podcast
Icon-a-thon
Guide
HP/Silver
HP/Gold Integrated
Engagement
iGEM Toronto
