<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Toronto_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Toronto/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="row navigation bg-dark-cyan"><DIV class="row" id="navigation"><UL><LI><SPAN class="text-white">• </SPAN><A href="https://2017.igem.org/Team:Toronto" id="grey-hover" class="text-white">Home</A></LI><LI><SPAN class="text-grey">• </SPAN><A href="https://2017.igem.org/Team:Toronto/Team" id="grey-hover" class="text-white">Team</A></LI><LI><SPAN class="text-red">• </SPAN><A href="https://2017.igem.org/Team:Toronto/Wetlab" id="red-hover" class="text-white">Wetlab</A></LI><LI><SPAN class="text-yellow">• </SPAN><A href="https://2017.igem.org/Team:Toronto/Drylab" id="yellow-hover" class="text-white">Drylab</A></LI><LI><SPAN class="text-cyan">• </SPAN><A href="https://2017.igem.org/Team:Toronto/Human_Practices" id="cyan-hover" class="text-white">Human Practices</A></LI></UL></DIV></DIV><DIV class="section"><DIV class="container header" id="red"><H1>Design</H1></DIV></DIV><DIV class="section bg-white"><DIV class="container content-page row"><DIV class="block content"><DIV id="subsection-PartI" class="subsection"><H2 class="text-red">Part I: All you need is LOV</H2><H3>Expanding the synthetic biology toolbox using LacILOV</H3><P>LacILOV is a novel, photoactive fusion protein of the LacI repressor DNA binding domain from the lactose operon of E. coli and the Light Oxygen Voltage (LOV)-Sensing domain from Avena sativa (oats)<SUP><A href="#ref1">[1]</A></SUP>. The LOV domain is derived from asLOV2 and activated in the presence of blue light through a flavin mononucleotide (FMN). FMN transfers energy to a catalytic cysteine and induces a conformational change in the protein. In the absence of light, it is hypothesized that LacILOV behaves much like the lac repressor, dimerizing and binding to the lacO promoter sequence, Ptrc-2. However, in the presence of light, specifically of wavelength 465 nm <SUP><A href="#ref2">[2]</A></SUP>, the repressor monomerizes and no longer blocks the promoter site allowing RNA polymerase to start transcription.</P><H3>Experiments/Testing:</H3><P>In order to characterize the kinetics of LacILOV, we designed a simple composite part by placing mCherry, a fluorescent reporter, under the control of the LacILOV-controlled Ptrc-2 promoter. The part mCherry (BBa_K2469003) was selected as a reporter because it is chromoprotein, i.e. a fluorescent protein with an emitted wavelength visible to the human eye.</P><FIGURE><FIGCAPTION>Figure 1. LacILOV mCherry part BBa_K2469003</FIGCAPTION></FIGURE><P>Given the sensitivity of the LOV domain from asLOV2 to blue light, specialized equipment was designed to exclude ambient light and deliver precise wavelengths and intensities of light to our cells. https://2017.igem.org/Team:Toronto/Hardware. Design, assembly and programming of the light delivery systems was documented and improved upon based on feedback.</P><FIGURE><FIGCAPTION>Figure 2. Fluorescence assay for LacILOV mCherry and Reporter Switch Design</FIGCAPTION></FIGURE><P>To characterize LacILOV, cells containing LacILOV-mCherry construct were grown under a consistent light intensity and the fluorescent output was measured at different time points. To account for the rapid growth rate of cells in the exponential phase, the fluorescence date was divided by the optical density. A statistically significant difference was observed between cells grown in light and in the dark. However, since the activation time was slow, we identified an opportunity to computationally optimize the structure of LacILOV. Using ITASSER, the protein structure of LacILOV was predicted and compared to the PDB files of LacI and LOV. In doing so, an unexpected stabilizing structure in the linker region was identified. Since it could potentially contribute to the slow de-repression of LacILOV, Foldit standalone, a predictive software, was used to iteratively select mutations to de-stabilize the structure of the linker.</P><FIGURE><FIGCAPTION>Figure 3. Our predicted LacILOV structure</FIGCAPTION></FIGURE><H3>Basic Switch Design:</H3><P>With LacILOV as a part in our toolbox, we were inspired by the bistable switch design created by Gardner et al. 2000 <SUP><A href="#ref1">[3]</A></SUP> to develop a light-sensitive switch by coupling two different repressible promoters - LacILOV-repressed Ptrc-2, and BBa_R0051, modulated by the viral repressor, cI. Since cI is a stringent repressor of the switch between the lytic and lysogenic phase in viruses, the reversibility of the switch was a concern. We therefore decided to utilize a version of cI with a LVA degradation tag to improve its turnover. Our cI:LVA is similar to the part BBa_K327018, save the exclusion of a stop codon. In order to assay for the activity of our switch, we positioned fluorescent reporters downstream of our repressible promoters:</P><UL class="bullets"><LI>YFP was added downstream of BBa_R0051. If cI:LVA is induced and expressed, there will be no transcription of YFP and subsequently no fluorescence emission will be measured at  528nm.
mCherry was added downstream of Ptrc-2: LacILOV-mCherry was submitted as BBa_K2469003. In the absence of blue light, LacILOV (which is constitutively expressed) will bind to the Ptrc-2 promoter and inhibit transcription. No fluorescence at 610nm will be observed.
</LI><LI>YFP was added downstream of BBa_R0051. If cI:LVA is induced and expressed, there will be no transcription of YFP and subsequently no fluorescence emission will be measured at  528nm.
mCherry was added downstream of Ptrc-2: LacILOV-mCherry was submitted as BBa_K2469003. In the absence of blue light, LacILOV (which is constitutively expressed) will bind to the Ptrc-2 promoter and inhibit transcription. No fluorescence at 610nm will be observed.
</LI></UL><FIGURE><FIGCAPTION>Figure 4. YFP/mCherry Switch part  BBa_K2469004</FIGCAPTION></FIGURE><P>To test the activity of the basic switch mechanism, samples were grown in either dark or light conditions using our hardware devices, with the following expectations:</P><P>In the dark: LacILOV is bound to the Ptrc-2 promoters. Since mCherry and cI:LVA are both controlled by Ptrc-2, no mCherry or cI:LVA is transcribed. Therefore no fluorescence is measured at 610nm for mCherry. Additionally, since no cI:LVA is produced, YFP is transcribed and fluorescence is measured at 528 nm.</P><FIGURE><FIGCAPTION>Figure 5. Animated model for the switch in the dark.</FIGCAPTION></FIGURE><P>In the light: LacILOV unbinds from the Ptrc-2 promoter and mCherry and cI:LVA are both expressed. mCherry reporter fluorescence is measured at 610nm. Since cI:LVA is expressed, the BBa_R0051 promoter is repressed and prevents downstream expression of YFP. </P><FIGURE><FIGCAPTION>Figure 6. Animated model for the switch in the light.</FIGCAPTION></FIGURE><P>In order to optimize the design of our switch, we modelled the interactions of each component using ordinary differential equations (ODEs). Through Quasi steady-state assumption, we combined and simplified the model down to four equations. We then simulated the model through the MATLAB tool, Simbiology. In order to validate the accuracy of our models, we compared our predicted results for LacILOV-mCherry (BBa_K2469003) to the data from our aforementioned characterization assay of LacILOV and calculated the fit.</P><FIGURE><FIGCAPTION>Figure 7. Fitting the mCherry reporter assay results from wet lab to our predicted ODE model</FIGCAPTION></FIGURE><P>Similar to the activity of the LacILOV mCherry construct, the assays for assessing reporter construct use the same hardware devices for delivery of light and dark conditions. The fluorescence output of mCherry and YFP were measured. It is expected that mCherry expression would increase and YFP would decrease in light conditions.</P><P>We believe that scientific communication is an integral part of scientific research, and that engaging with stakeholders should be a continuous process. Considering that the core tool of our project is rooted in foundational synthetic biology principles, we decided that we would focus on different disciplines as they intersect within the field of synthetic biology. Our first initiative was a podcast series, Synversations, where each episode was intended to address a discipline (ex. Arts, business, ethics, engineering)  that could make significant contributions to the field synthetic biology. Furthermore, we organised an Iconathon workshop in order to engage both artists and researchers in a collaborative initiative to meet the need for better scientific icons. Additionally, we designed a trivalent workshop for high school students, encompassing bioinformatics, bioethics, and genetics. The aim to not only inspire the them to engage with synthetic biology in an academic setting, but to also educate them as members of the public about the diversity of opportunities and possibilities that the field can offer.
				</P></DIV><DIV id="subsection-partII" class="subsection"><H2 class="text-red">Part II: The Future of Gene Editing is Lit</H2><H3>Employing LacILOV &amp; Anti-CRISPR to Regulate Gene Editing</H3><P>The CRISPR-Cas9 technology holds tremendous promise for human gene therapy. The technology has already been applied to various human diseases, from Duchenne muscular dystrophy to the congenital heart disorder that results from a mutation in MYBPC3. However, despite the potential of these initial studies, off-targeting editing remains a concern, particularly during different phases of the cell cycle. This is often exacerbated by the relative efficiency of the error-prone Non-Homologous End-Joining (NHEJ) pathway over Homology-Directed Repair (HDR), which are both activated upon Cas9’s creation of a double-stranded break in DNA. If the technology is to be employed in the clinic for human gene therapy in the future, a dramatic improvement in our control of Cas9 is desperately needed. To address this problem, we designed a switch for the sgRNA of the CRISPR-Cas9 system using one of the recently discovered anti-CRISPR proteins, AcrIIA4.</P><P>This anti-CRISPR protein is encoded by the Listeria monocytogenes phage that inhibits Streptococcus pyogenes Cas9 by binding to the Cas9 endonuclease:sgRNA complex. This prevents Cas9 from binding DNA. The design of the CRISPR switch is similar to the design of our basic switch, with a few additions:</P><UL><LI>An sgRNA expressed from the Ptrc-2 promoter, therefore placed under the regulation of LacILOV</LI><LI>The AcrIIA4 expressed from the BBa_R0051 promoter, therefore placed under the regulation of cI:LVA.</LI></UL><FIGURE><FIGCAPTION>Figure 8. The layout of our CRISPR/anti-CRISPR switch is depicted above</FIGCAPTION></FIGURE><P>The basic states of our switch are as follows:</P><P>In the dark: LacILOV is bound to the Ptrc-2 promoter, therefore all genes under control of Ptrc-2 are not expressed. This means that neither the sgRNA nor mCherry should be expressed, and therefore no fluorescence should be measured at 610nm. Since cI:LVA is downstream of Ptrc-2, no cI:LVA is produced and therefore both YFP, and AcrIIA4 are transcribed. AcrIIA4 binds to any dCas9-sgRNA complexes in order to block the PAM site, rendering the CRISPRi system inactive. Produced YFP fluorescence is measured at 528nm.</P><P>In the light: LacILOV unbinds from the Ptrc-2 promoter and cI:LVA, the sgRNA, and mCherry are transcribed, therefore fluorescence is measured at 610nm. cI:LVA binds to the BBa_R0051 promoter and prevents transcription and expression of AcrIIA4 and YFP. Consequently, the CRISPRi system is active. </P><P>The genetic circuit should be co-transformed with a pdCas9 plasmid, for CRISPRi, as cleavage assays are not possible in E.coli due to lack of HDR repair mechanisms. The pdCas9 plasmid that used was aTc inducible (addgene - plasmid#44249). Additionally, due to the fact that our pdCas9 had a chloramphenicol resistance marker, we employed a pKDL071 (kanamycin resistant) backbone for the assays where pdCas9 was to be co-transformed with our assays.</P><P>For the purpose of testing our AcrIIA4, we decided to design our sgRNA such that it would target the Escherichia coli K12 MG1655 AraC gene in the arabinose metabolism pathway. We collaborated with iGEM British Columbia, who used their sgRNA modeling program to generate sgRNAs with higher partition functions than the previous sgRNAs that we had generated. Ultimately, we selected TGGGCGTTAAACGAGTATCC targeting the non-tamplate strand (with the PAM 33nt from the start).</P><FIGURE><FIGCAPTION>Figure 9. outline of the constructs that were used to test our AcrIIA4 part BBa_K2469002. A) Construct constitutively expressing cI:LVA and mCherry B) Construct constitutively expressing cI:LVA, mCherry and sgRNA and C) Construct constitutively expressing cI:LVA,  mCherry, sgRNA, and HIS tagged AcrIIA4.</FIGCAPTION></FIGURE><P>In order to test the activity of our sgRNA and our HIS:AcrIIA4 we designed a quantitative growth assay, as shown below.</P><FIGURE><FIGCAPTION>Figure 10. Quantitative Growth Assay for parts in figure 9</FIGCAPTION></FIGURE></DIV><DIV id="content-yellow" class="subsection"><H2 class="text-cyan">References</H2><OL><LI id="ref1">LOV (light, oxygen, or voltage) domains of the blue-light photoreceptor phototropin (nph1): Binding sites for the chromophore flavin mononucleotide (Christie et al., 1998)</LI><LI id="ref2">AN ACTION SPECTRUM IN THE BLUE and ULTRAVIOLET FOR PHOTOTROPISM IN ALFALFA (Baskin and Iino, 1987)</LI><LI id="ref3">Gardner, T. S., Cantor, C. R. &amp; Collins, J. J. Construction of a genetic toggle switch in Escherichia coli. Nature 403, 339–342 (2000).</LI><LI id="ref4">Mendel, J. R. and Rodino-Klapac, L. R. (2016). Duchenne-Muscular Dystrophy: CRISPR/Cas9 Treatment. Cell Research. 26: 513-514. doi: 10.1038/cr.2016.28</LI><LI id="ref5">Ma, H. et al. (2017). Correction of a pathogenic gene mutation in human embryos. Nature. 548: 413-419. doi: 10.1038/nature23305</LI><LI id="ref6">Schaefer, K. A. et al. (2017). Unexpected mutations after CRISPR-Cas9 gene editing in vivo. Nature Methods. 14: 547-548. doi: 10.1038/nmeth.4293.</LI><LI id="ref7">Dong, D. et al. (2017). Structural Basis of CRISPR-SpyCas9 Inhibition by an Anti-CRISPR Protein. Nature. 546: 436-439. Doi: 10.1038/nature22377.</LI></OL></DIV></DIV><DIV class="block sidebar"><DIV id="sidebar-box"><H3>Contents</H3><H3>Related Pages</H3><DIV class="sidebar-minibox"><UL><LI><A href="https://2017.igem.org/Team:Toronto/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Improve">Improve</A></LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="section bg-black"><DIV class="container footer row"><DIV class="block content"><H3 class="white">General Pages</H3><UL><LI><A href="https://2017.igem.org/Team:Toronto">Home</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Safety">Safety</A></LI></UL></DIV><DIV class="block content"><H3 class="white">Wet Lab</H3><UL><LI><A href="https://2017.igem.org/Team:Toronto/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Hardware">Hardware</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Demonstrate">Demonstrate</A></LI></UL></DIV><DIV class="block content"><H3 class="white">Dry Lab</H3><UL><LI><A href="https://2017.igem.org/Team:Toronto/ODE">ODE</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Analysis">Analysis</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Protein-Modelling">Protein Modelling</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Model">Model</A></LI></UL></DIV><DIV class="block content"><H3 class="white">Human Practices</H3><UL><LI><A href="https://2017.igem.org/Team:Toronto/Interviews">Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Genetic_Code_Workshop">Genetic Code Workshop</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Podcast">Podcast</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Icon-a-thon">Icon-a-thon</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Guide">Guide</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/HP/Silver">HP/Silver</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/HP/Gold_Integrated">HP/Gold Integrated</A></LI><LI><A href="https://2017.igem.org/Team:Toronto/Engagement">Engagement</A></LI></UL></DIV></DIV><DIV class="container footer-bottom row"><DIV class="block content"><P>iGEM Toronto</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>