Team:ZJU-China/Notebook
Home
Project
Description
Design
Experiments
Proof of concept
Result
Notebook
Demonstrate
Parts
Parts
Composite Parts
Part collection
Modeling
Overview
Light Control
Infiltration
Oscillation
Hardware
Safety
Human Practice
HP summary
Silver
Gold
Engagement
Team
Team
Collaborations
Attributions
ZJU-China
Home
Project
Description
Design
Experiments
Proof of concept
Result
Notebook
Demonstrate
Parts
Parts
Composite Parts
Part collection
Modeling
Overview
Light Control
Infiltration
Oscillation
Hardware
Safety
Human Practice
HP summary
Silver
Gold
Engagement
Team
Team
Collaborations
Attributions
Back to top
Notebook
June
6.15
        Participant: Zhanyu Wang, Han Wan         1.Get all the basic parts from Kit plate         supD-tRNA ,T7RNAP &PT7
Transformation
Gene of interest
Vector
Ice
42°C
Ice
37°C+LB
Plate
BBa_K228001
pSB1C3
30min
90s
5min
1h/1ml
LBCM
BBa_K228000
pSB1C3
30min
90s
5min
1h/1ml
LBCM
BBa_K1321338
pSB1C3
30min
90s
5min
1h/1ml
LBCM
        2.Done the transformation
6.30
        Transformation         Participant： Shisheng Li
Transformation
Gene of interest
Vector
Ice
42°C
Ice
37°C+LB
Plate
BBa_I15008
pSB1C3
30min
90s
5min
1h/1ml
LBCM
BBa_I15009
pSB1C3
30min
90s
5min
1h/1ml
LBCM
July
    
7.1
        Liquid Culture         Participant: Shisheng Li
Liquid Culture
Name
Medium
Volume/ml
BBa_I15008(pSB1C3) colony_1
LB
10
BBa_I15008(pSB1C3) colony_2
LB
10
BBa_I15008(pSB1C3) colony_3
LB
10
BBa_I15009(pSB1C3) colony_1
LB
10
BBa_I15009(pSB1C3) colony_2
LB
10
BBa_I15009(pSB1C3) colony_3
LB
10
7.2
        miniprep         Participant: Shisheng Li
Miniprep
Name
260/280
C ng/μl
BBa_I15008(pSB1C3) colony_1
1.90
34.6
BBa_I15008(pSB1C3) colony_2
1.87
33.2
BBa_I15008(pSB1C3) colony_3
1.88
58.3
BBa_I15009(pSB1C3) colony_1
1.75
33.0
BBa_I15009(pSB1C3) colony_2
1.85
45.1
BBa_I15009(pSB1C3) colony_3
1.93
49.0
        PCR         Participant： Shisheng Li
PCR
V/μL
Steps
Temperature/μL
Time/s
Cycle
taq polymerase
0.5
Pre Denature
94
180
/
tempalte
0.5
Denature
95
30
30
10X Taq buffer
5
Annealing
55
30
30
primer-forward(10μM)
2
Extension
72
45/45
30
primer-reverse(10μM)
2
Final Extension
72
300
/
dNTP(2.5mM)
4
storage
4
/
/
ddH2O
36
/
/
/
/
    Result:
7.5
        Construct of double oscillation plasmid Ptd103aiiA-luxS         Participant：Jinshi Ran, Hongya Zhu         •  We got luxS gene from 15M, plate 6, distribution kit 2016.         •  We amplified luxS with PCR and added sequences homo with pTD103aiiA to it.         •  We amplified the backbone pTD103aiiA, and added sequence homo with luxS.         •  Using one-step cloning, we assembled them together.         •  We transformed the new constructed plasmid into DH5alpha and cultured it overnight.         •  We screened the positive colony with colony PCR. After conformed, we inoculated it in 5ml LB and cultured it overnight to harvest the pTD103aiiA-luxS.
7.8
        Transformation         Participant：Shisheng Li
Transformation
Gene of interest
Vector
Ice
42°C
Ice
37°C+LB
Plate
cph8
pSB1C3
30min
90s
5min
1h/1ml
LBCM
double terminator
pSB1C3
30min
90s
5min
1h/1ml
LBCM
PompC
pSB1C3
30min
90s
5min
1h/1ml
LBCM
7.9
        Liquid Culture         Participant: Shisheng Li
Liquid Culture
Name
Medium
Volume/ml
Cph8 colony_1
LB
10
Cph8 colony_2
LB
10
double terminator colony_1
LB
10
double terminator colony_2
LB
10
PompC colony_1
LB
10
PompC colony_2
LB
10
7.10
        Miniprep         Participant: Shisheng Li
Miniprep
Name
260/280
C ng/μl
Cph8 colony_1
1.90
53.3
Cph8 colony_2
1.87
74.1
double terminator colony_1
1.84
44.2
double terminator colony_2
1.61
50.8
PompC colony_1
1.87
43.8
PompC colony_2
2.12
37.6
        PCR         Participant: Shisheng Li
PCR
V/μL
Steps
Temperature/μL
Time/s
Cycle
taq polymerase
0.5
Pre Denature
94
180
/
tempalte
0.5
Denature
95
30
30
10X Taq buffer
5
Annealing
55
30
30
primer-forward(10μM)
2
Extension
72
180/45/45
30
primer-reverse(10μM)
2
Final Extension
72
300
/
dNTP(2.5mM)
4
storage
4
/
/
ddH2O
36
/
/
/
/
        Result:         The result of gel electrophoresis showed that we managed to get the pcr product of Cph8 and PompC but failed in getting double terminator. After we changed the annealing temperature, we finally made it. When sorting out data, we accidently deleted the picture of the gel.
7.16
        1.Measuring and modeling of promoter pluxR         Participant: Jinshi Ran, Hongya Zhu         For better description of single oscillation system, we measured the response intensity of pluxR to AHL. First, we mutated pTD103luxI-sfGFP with knocking gene luxI out with PCR. We named the new plasmid pTD103sfGFP. We transformed MG1655 strain of E•coil with pTD103sfGFP, inoculated positive colony in 5ml LB with antibiotic 100μg/ml kanamycin and cultured it over night. We inoculated over night culture with a 1:1000 dilution in 1ml LB and added AHL according a concentration range:10-6~10-11M when the bacteria solution reached an A600nm 0.1. Cultured it in 37℃ for 8h. We spun it down and concentrated in PBS (pH=7.2), then measured the fluorescence intensity.         2.pick colonies and shake         Participant：Zhanyu Wang, Han Wan
Liquid Culture
Name
Medium
Volume/ml
BBa_K28001(pSB1C3) colony_1
LB
10
BBa_K28001(pSB1C3) colony_2
LB
10
BBa_K28001(pSB1C3) colony_3
LB
10
BBa_K28000(pSB1C3) colony_1
LB
10
BBa_K28000(pSB1C3) colony_2
LB
10
BBa_K1321338(pSB1C3) colony_1
LB
10
BBa_K1321338(pSB1C3) colony_2
LB
10
BBa_K1321338(pSB1C3) colony_3
LB
10
7.26
        Find the T7ptag mutation site and design primers for point mutation         Participant：Zhanyu Wang, Han Wan
7.27
        Point mutation pcr         Participant：Zhanyu Wang, Han Wan
August
8.2
        Confirm other parts need in logic gate         Participant：Wang Zhanyu, Wan Han
Transformation
Gene of interest
Vector
Ice
42°C
Ice
37°C+LB
Plate
BBa_R0010
pSB1C3
30min
90s
5min
1h/1ml
LBCM
BBa_I13453
pSB1C3
30min
90s
5min
1h/1ml
LBCM
BBa_895006
pSB1C3
30min
90s
5min
1h/1ml
LBCM
8.3
        pick colonies and shake         Participant：Participant：Zhanyu Wang, Han Wan
Liquid Culture
Name
Medium
Volume/ml
BBa_R0010(pSB1C3) colony_1
LB
10
BBa_R0010(pSB1C3) colony_2
LB
10
BBa_R0010(pSB1C3) colony_3
LB
10
BBa_I13453(pSB1C3) colony_1
LB
10
BBa_I13453(pSB1C3) colony_2
LB
10
BBa_K895006(pSB1C3) colony_1
LB
10
BBa_K895006(pSB1C3) colony_2
LB
10
BBa_K895006(pSB1C3) colony_3
LB
10
8.4
        Participant：Zhanyu Wang, Han Wan         1. Extract the plasmid         2. PCR
PCR
V/μL
Steps
Temperature/μL
Time/s
Cycle
taq polymerase
0.5
Pre Denature
94
180
/
tempalte
0.5
Denature
95
30
30
10X Taq buffer
5
Annealing
55
30
30
primer-forward(10μM)
2
Extension
72
45/45
30
primer-reverse(10μM)
2
Final Extension
72
300
/
dNTP(2.5mM)
4
storage
4
/
/
ddH2O
36
/
/
/
/
        3.Conduct electrophoresis in 1% agarose gel         After sequencing, the other parts are all confirmed.
8.5
        Doing overlap PCR, construct the logic gate plasmid.         Participant：Zhanyu Wang, Han Wan
8.7-8.9
        Finish the construction of logic gate’s input plasmid(~3500bp,marked with an arrow)         Participant： Zhanyu Wang, Han Wan/br>
8.9-8.11
        Finish the construction of logic gate’s output plasmid(~1100bp,marked with  arrows)         Participant： Zhanyu Wang, Han Wan
8.11-8.14
        Using infusion cloning to integrate the input plasmid and output plasmid together. (~4500bp,marked with an arrow)         Participant： Zhanyu Wang, Han Wan
        Thus , we finally get our logic gate plasmid with input and output together.
8.15
        Validate the logic gate plasmid. For details, see our protocol in the experiment part.         Participant： Zhanyu Wang, Han Wan
8.18
        Validate the logic gate's response concentration.         Participant： Zhanyu Wang, Han Wan         we add 0.5ml of the fresh bacteria culture and appropriate volume of inducer solution to prepare induction system with the concentration gradient of 10^-9, 10^-8, 10^-7, 10^-6, 10^-5, 10^-4, 10^-3.         For details, see our protocol in the experiment part.
8.20
        Co-transformation         Participant: Shisheng Li
Co-Transformation
Gene of interest
Vector
Ice
42°C
Ice
37°C+LB
Plate
Two green-TCS plasmids
/
30min
90s
5min
1h/1ml
LBCM
Two red-TCS plasmids
/
30min
90s
5min
1h/1ml
LBCM
8.21
        Liquid Culture         Participant: Zhanyu Wang, Han Wan
Liquid Culture
Name
Medium
Volume/ml
Green TCS_1
LB
5
Green TCS_2
LB
5
Green TCS_3
LB
5
Green TCS_4
LB
5
Green TCS_5
LB
5
Green TCS_6
LB
5
Green TCS_7
LB
5
Green TCS_8
LB
5
Red TCS_1
LB
5
Red TCS_2
LB
5
Red TCS_3
LB
5
Red TCS_4
LB
5
Red TCS_5
LB
5
Red TCS_6
LB
5
Red TCS_7
LB
5
Red TCS_8
LB
5
8.22
        Fluorescence intensity measuring         Participant: Shisheng Li
Green TCS
RFU
Control
Dark/th>
green light
red light
1
9074
31813
39406
32336
2
9022
31512
39105
32491
3
9063
31980
38535
32000
4
9072
32396
39816
32261
5
9130
32203
39201
33136
6
9063
32463
39829
33519
7
9329
32370
39991
33775
8
9192
32724
39886
33168
Red light TCS
RFU
Control
Dark/th>
red light
1
369
3716
3241
2
248
3719
3332
3
384
3743
3263
4
404
3761
3449
5
408
3663
3179
6
409
3761
3416
7
350
3761
3416
8
357
3584
3178
8.23
        Verifying the single oscillation circuit in microfluidics device         Participant：Jinshi Ran, Hongya Zhu         • We co-transformed the MG1655 strain with pTD103aiiA and pTD103luxI-sfGFP         • We screened the positive colony and inoculated in 50ml LB with antibiotic 100μg/ml ampicillin (Amp) and 50 mgml21 kanamycin (Kan).         • We spun it down and concentrated it in 5ml of fresh media with surfactant concentration of0.075% Tween20 when the bacteria solution reach an A600nm of 0.05~0.1.         • We loaded the sample from the cell port while keeping the media port at sufficiently higher pressure than the waste port below to prevent contamination. We manually applied pressure pulses to line to induce a momentary flow change. The flow was then reversed and allow for cell to receive fresh media with 0.075% Tween20 which prevented cells from adhering to the main channels and waste ports.         • We took pictures of microfluidics in fluorescence microscope every 5 mins and analyzed the fluorescent density with imageJ.
8.25
        Liquid Culture         Participant: Shisheng Li
Liquid Culture
Name
Medium
Volume/ml
Green TCS_1
LB
5
Green TCS_2
LB
5
Green TCS_3
LB
5
Green TCS_4
LB
5
Green TCS_5
LB
5
Green TCS_6
LB
5
Green TCS_7
LB
5
Green TCS_8
LB
5
Red TCS_1
LB
5
Red TCS_2
LB
5
Red TCS_3
LB
5
Red TCS_4
LB
5
Red TCS_5
LB
5
Red TCS_6
LB
5
Red TCS_7
LB
5
Red TCS_8
LB
5
8.26
        Fluorescence intensity measuring         Participant： Li Shisheng
Delay of Red TCS
RFU
0h
1h
2h
3h
4h
5h
1
411
431
439
489
739
1681
2
438
489
403
471
667
1690
3
431
503
541
599
820
1756
4
384
489
456
419
833
1681
5
506
434
434
530
815
1553
6
330
364
426
481
684
1501
7
361
391
482
552
739
1482
8
395
460
426
574
775
1641
Green TCS
RFU
0h
1h
2h
3h
4h
5h
1
411
516
736
917
1680
4200
2
438
554
715
907
1752
4190
3
431
534
723
923
1712
4210
4
384
702
805
839
1692
4148
5
506
590
687
878
1563
4045
6
330
577
639
891
1580
3968
7
361
588
617
860
1559
3881
8
395
569
692
992
1541
3849
8.27
        PCR         Participant: Shisheng Li
PCR
V/μL
Steps
Temperature/μL
Time/s
Cycle
super fidelity DNA polymerase
1
Pre Denature
95
180
/
tempalte
0.5
Denature
95
15
30
2X buffer
25
Annealing
55
15
30
primer-forward(10μM)
2
Extension
72
80
30
primer-reverse(10μM)
2
Final Extension
72
300
/
dNTP(10mM)
1
storage
4
/
/
ddH2O
18.5
/
/
/
/
        Template: broken Ptet-cph8 (BBa_K1886006)
8.28
    Validate the logic gate's response time. For details, see our protocol in the experiment part.         Participant：Zhanyu Wang, Han Wan
8.30
        Verifying the double oscillation circuit         Participant：Jinshi Ran ,Hongya Zhu         • We co-transform the MG1655 strain with pTD103 aiiA-luxS、pTD103 luxI-sfGFP and pSB1C3 plsr-YFP.         • We screened the positive colony and inoculated it in 50ml LB with antibiotic 100μg/ml ampicillin (Amp) and 50 mg/ml kanamycin (Kan)and Chloromycetin(C). We spun it down and concentrated it in 5ml LB when the bacteria solution reached an A600nm of 0.05~0.1.         • We inoculated it with a 1:1000 dilution in 100ml fresh media. When the bacteria solution reached A600nm 0.2~0.3, we sampled in every 10 min and measured the fluorescence intensity.
September
9.2
        Transformation         Participant: Shisheng Li
PCR
V/μL
Steps
Temperature/μL
Time/s
Cycle
super fidelity DNA polymerase
1
Pre Denature
95
180
/
tempalte
0.5
Denature
95
15
30
2X buffer
25
Annealing
55
15
30
primer-forward(10μM)
2
Extension
72
80
30
primer-reverse(10μM)
2
Final Extension
72
300
/
dNTP(10mM)
1
storage
4
/
/
ddH2O
18.5
/
/
/
/
        Template: promoter-ccas
9.4
        Transformation         Participant: Shisheng Li
PCR
V/μL
Steps
Temperature/μL
Time/s
Cycle
super fidelity DNA polymerase
1
Pre Denature
95
180
/
tempalte
0.5
Denature
95
15
30
2X buffer
25
Annealing
55
15
30
primer-forward(10μM)
2
Extension
72
80
30
primer-reverse(10μM)
2
Final Extension
72
300
/
dNTP(10mM)
1
storage
4
/
/
ddH2O
18.5
/
/
/
/
        p-sfGFP-ccaR(BBa_K1886003)
9.7
        Transformation         Participant： Shisheng Li
PCR
V/μL
Steps
Temperature/μL
Time/s
Cycle
super fidelity DNA polymerase
1
Pre Denature
95
180
/
tempalte
0.5
Denature
95
15
30
2X buffer
25
Annealing
55
15
30
primer-forward(10μM)
2
Extension
72
80
30
primer-reverse(10μM)
2
Final Extension
72
300
/
dNTP(10mM)
1
storage
4
/
/
ddH2O
18.5
/
/
/
/
        Template: p-HO1-PCYA(BBa_K1886005)
9.9
        Validate the plasmid combining logic gate and red light-induced system: PompC+Plac (3600bp)         Participant: Zhanyu Wang, Han Wan
9.17
        Validate the plasmid combining logic gate and green light-induced system: PcpcG2+Plac (3700bp)         Participant：Zhanyu Wang, Han Wan
October
10.10
        PCR         Participant: Shisheng Li
PCR
V/μL
Steps
Temperature/μL
Time/s
Cycle
super fidelity DNA polymerase
1
Pre Denature
95
180
/
tempalte
0.5
Denature
95
15
30
2X buffer
25
Annealing
55
15
30
primer-forward(10μM)
2
Extension
72
80
30
primer-reverse(10μM)
2
Final Extension
72
300
/
dNTP(10mM)
1
storage
4
/
/
ddH2O
18.5
/
/
/
/
        Pompc_CL_p-lamda_sfGFP(BBa_K1886018)         Ho1_pcyA_Pompc_CL_Plamda_sfGFP(BBa_K1886019)
Notebook for dry lab
5.19
        Started to design the circuit board for the “cipher machine” used in human practice. (Kang)
5.22
        Received the PCB from the factory, started to welding. (Ma)         Decided to make a computer game that relate to our project.
5.27
        The PCB has some problems, we were working on with through it.
5.30
        Finished the computer game, it looks great! (Li)
6.7
        Started to design the microfluidic device. (Fu) The electric cipher machine failed to work, Kang made a new PCB.
6.13
        Sent the blueprint of our microfluidic device to the factory. Started to build our modeling.
6.22
        New PCB arrived, and it worked well. (Ma)
7.13
        Started to edit our wiki.
7.29
        The model for our light control system was finished, the curve is pretty good but not perfect, and we were trying to update the parameters.
8.4
        We compared the results between the modeling and the experiment. The curve is a little bit flat than we expected.
8.29
        The microfluidic device has arrived. The wet lab used it for their experiment.
9.3
        Started to build oscillation model. (Fu)
9.17
        The wiki style has been settled.
9.28
        We finished the oscillation model, but the result didn’t converge.
10.7
        After we changed some of the parameters, the result of our oscillation model finally became reasonable.
10.14
        Most of our content are finished, started to fulfill the wiki.
Contact Us Room 413,Biology lab center, Zijingang Campus Zhejiang University, YuHangTang Road NO.866 Hangzhou, China
iGEM ZJU-China 2016 Team
igem_zjuchina_2016@outlook.com
