<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BIT-China_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BIT-China/Protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><UL id="left-nav"><LI><A>PROJECT</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Background">Background</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Design">Idea &amp; Design</A></LI></UL><LI><A>EXPERIMENTS</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/ExperimentsRegulator">Regulator</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/ExperimentsFeedback">Feedback</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/ExperimentsOutput">Output</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Results">Results</A></LI><LI><A>MODELING</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Model">Overview</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/FluorescentProbesModel">Fluorescent Probe Model </A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/H2O2DecompositionModel">H<SUB>2</SUB>O<SUB>2</SUB>
                        Decomposition Model</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/roGFP2-Orp1MichaelisEquationModel">roGFP2-Orp1
                        Michaelis equation Model</A></LI><LI><A>HUMAN PRACTICES</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/HPOverview">Overview</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Human_Practices">Integrated Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Public_Engagement">Education &amp; Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Collaborations">Collaborations</A></LI><LI><A>NOTEBOOK</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Notebook">Lab Book</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Protocols">Methodology / Protocols</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Equipment">Material &amp; Equipment</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/InterLab">Measurement / InterLab</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Safety">Safety</A></LI><LI><A>ACHIEVEMENTS</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/JudgingForm">Judging Form</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Applied_Design">Applied Design</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Demonstrate">Demonstrate</A></LI><LI><A>TEAM</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Team">Members</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Gallery">Gallery</A></LI></P><DIV id="ME-content-all" class="ME-container" style="margin-top:calc(25vh - 30px);"><DIV class="MD-title"><DIV class="ME-title-content"><A style="text-decoration: none;color:#131313;">PROTOCOLS</A></DIV></DIV><DIV id="ME1" class="cd-section ME-margin-toTitle"><DIV class="ME-title-1"><A style="text-decoration: none;color:#131313;">Basic Protocol</A></DIV><DIV class="ME-content-all ME-margin-toTitle1 "><DIV class="ME-title-2"><A style="text-decoration: none;color:#131313;">1.Protocol for PCR</A></DIV><DIV id="E1" class="collapseDiv"><H3 id="a-nav-dot-box-E1" class="ME-title-content-a">
                        a. Protocol for PCR with PrimeSTAR (Premix)
                    </H3><DIV id="collapse_body_E1" class="collapse_body"><DIV class="collapse_content"><P class="ME-content-p">
                                1) Prepare the reaction mix in PCR tube:
                            </P><DIV class="table-body"><TABLE><TBODY style="font-size: 16px;"><TR><TD>Template DNA</TD><TD>1µl</TD></TR><TR><TD>Forward primer</TD><TD>2µl</TD></TR><TR><TD>Reverse primer</TD><TD>2µl</TD></TR><TR><TD>PrimeSTAR (Premix)</TD><TD>25µl</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>20µl</TD></TR><TR><TD>Total volume</TD><TD>50µl</TD></TR></TBODY></TABLE></DIV><P class="ME-content-p">
                                When we ligate two fragments by OE-PCR, we usually add two kinds of fragment DNA 1µl
                                respectively and add ddH<SUB>2</SUB>0 up to 50µl.
                            </P><P class="ME-content-p">
                                2) Set the program of thermal cycler:
                            </P><DIV class="table-body" style="width:80%;"><TABLE><TBODY style="font-size: 16px;"><TR><TD>1X</TD><TD>98℃</TD><TD>2min</TD></TR><TR><TD>30X</TD><TD>98℃</TD><TD>10s</TD></TR><TR><TD>30X</TD><TD>55℃</TD><TD>5s</TD></TR><TR><TD>30X</TD><TD>72℃</TD><TD>1000bp/min</TD></TR><TR><TD>1X</TD><TD>72℃</TD><TD>5min</TD></TR><TR><TD>1X</TD><TD>16℃</TD><TD>∞</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><DIV id="E2" class="collapseDiv"><H3 id="a-nav-dot-box-E1" class="ME-title-content-a">
                        b. Protocol for PCR with Primer Star HS
                    </H3><DIV id="collapse_body_E1" class="collapse_body"><DIV class="collapse_content"><P class="ME-content-p">
                                1) Prepare the reaction mix in PCR tube:
                            </P><DIV class="table-body"><TABLE><TBODY style="font-size: 16px;"><TR><TD>Template DNA</TD><TD>3µL</TD></TR><TR><TD>Forward primer</TD><TD>3µL</TD></TR><TR><TD>Reverse primer</TD><TD>3µL</TD></TR><TR><TD>Primer Star HS DNA Polymerase</TD><TD>0.5µL</TD></TR><TR><TD>dNTP Mixture</TD><TD>4µL</TD></TR><TR><TD>2 X GC buffer</TD><TD>25µL</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>11.5µL</TD></TR><TR><TD>Total volume</TD><TD>50µL</TD></TR></TBODY></TABLE></DIV><P class="ME-content-p">
                                When we ligate two fragments by OE-PCR, we usually add two kinds of fragment DNA 1µl
                                respectively and add ddH<SUB>2</SUB>0 up to 50µl.
                            </P><P class="ME-content-p">
                                2) Set the program of thermal cycler:
                            </P><DIV class="table-body" style="width:80%;"><TABLE><TBODY style="font-size: 16px;"><TR><TD>1X</TD><TD>98℃</TD><TD>2min</TD></TR><TR><TD>30X</TD><TD>98℃</TD><TD>10s</TD></TR><TR><TD>30X</TD><TD>55℃</TD><TD>5s</TD></TR><TR><TD>30X</TD><TD>72℃</TD><TD>1000bp/min</TD></TR><TR><TD>1X</TD><TD>72℃</TD><TD>5min</TD></TR><TR><TD>1X</TD><TD>16℃</TD><TD>∞</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><DIV id="E3" class="collapseDiv"><H3 id="a-nav-dot-box-E1" class="ME-title-content-a">
                        c. Fast pfu fly
                    </H3><DIV id="collapse_body_E1" class="collapse_body"><DIV class="collapse_content"><P class="ME-content-p">
                                1) Prepare the reaction mix in PCR tube:
                            </P><DIV class="table-body" style="width:80%;"><TABLE><TBODY style="font-size: 16px;"><TR><TD>Template DNA</TD><TD>1µl</TD></TR><TR><TD>Forward primer</TD><TD>2µl</TD></TR><TR><TD>Reverse primer</TD><TD>2µl</TD></TR><TR><TD>dNTP</TD><TD>2µl</TD></TR><TR><TD>5X FastPfu Fly buffer</TD><TD>5µl</TD></TR><TR><TD>FastPfu Fly DNA polymerase</TD><TD>0.5µl</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>12.5µl</TD></TR><TR><TD>Total volume</TD><TD>25µl</TD></TR></TBODY></TABLE></DIV><P class="ME-content-p">
                                When we ligate two fragments by OE-PCR, we usually add two kinds of fragment DNA 1µl
                                respectively and add ddH<SUB>2</SUB>O up to 25µl.
                            </P><P class="ME-content-p">
                                2) Set the program of thermal cycler:
                            </P><DIV class="table-body" style="width:80%;"><TABLE><TBODY style="font-size: 16px;"><TR><TD>1X</TD><TD>94℃</TD><TD>5min</TD></TR><TR><TD>30X</TD><TD>94℃</TD><TD>30s</TD></TR><TR><TD>30X</TD><TD>55℃</TD><TD>30s</TD></TR><TR><TD>30X</TD><TD>72℃</TD><TD>1500bp/min</TD></TR><TR><TD>1X</TD><TD>72℃</TD><TD>10min</TD></TR><TR><TD>1X</TD><TD>16℃</TD><TD>∞</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><DIV id="E4" class="collapseDiv"><H3 id="a-nav-dot-box-E1" class="ME-title-content-a">
                        d. Taq mix
                    </H3><DIV id="collapse_body_E1" class="collapse_body"><DIV class="collapse_content"><P class="ME-content-p">
                                1) Prepare the reaction mix in PCR tube:
                            </P><DIV class="table-body" style="width:80%;"><TABLE><TBODY style="font-size: 16px;"><TR><TD>ddH<SUB>2</SUB>O</TD><TD>6.5µl</TD></TR><TR><TD>Forward primer</TD><TD>0.5µl</TD></TR><TR><TD>Reverse primer</TD><TD>0.5µl</TD></TR><TR><TD>2×Taq PCR Master Mix</TD><TD>7.5µl</TD></TR><TR><TD>Total volume</TD><TD>15µl</TD></TR></TBODY></TABLE></DIV><P class="ME-content-p">
                                2) Select positive clones and put them in the PCR tubes.
                            </P><P class="ME-content-p">
                                3) Make sure the bacteria are in the cPCR tubes.
                            </P><P class="ME-content-p">
                                4) Set the program of thermal cycler:
                            </P><DIV class="table-body"><TABLE><TBODY style="font-size: 16px;"><TR><TD>1X</TD><TD>94℃</TD><TD>10min</TD></TR><TR><TD>25X</TD><TD>94℃</TD><TD>30s</TD></TR><TR><TD>25X</TD><TD>55℃</TD><TD>30s</TD></TR><TR><TD>25X</TD><TD>72℃</TD><TD>1000bp/min</TD></TR><TR><TD>1X</TD><TD>72℃</TD><TD>10min</TD></TR><TR><TD>1X</TD><TD>16℃</TD><TD>∞</TD></TR></TBODY></TABLE></DIV><DIV class="ME-title-p"><A style="text-decoration: none;color:#131313;">2. Agarose gel electrophoresis </A></DIV><P class="ME-content-p">
                                1) Measure TAE buffer.
                            </P><P class="ME-content-p">
                                2) Weigh agarose powder and add agarose powder to the buffer.
                   </P><P class="ME-content-p">
                                3) Melt the mixture in a microwave until the solution becomes clear.
                            </P><P class="ME-content-p">
                                4) Let the solution cool to about 40-50℃ and add nucleic acid dye.
                            </P><P class="ME-content-p">
                                5) Pour the solution into the gel tray with corresponding comb.
                            </P><P class="ME-content-p">
                                6) Let the gel cool until it becomes solid.
                            </P><P class="ME-content-p">
                                7) Carefully pull out the comb and place the gel in the electrophoresis chamber.
                            </P><P class="ME-content-p">
                                8) Add enough TAE Buffer.
                            </P><P class="ME-content-p">
                                Pipette DNA samples mixed with appropriate amount of loading buffer and dye (Gene
                                Finder) into wells on the gel.
                            </P><P class="ME-content-p">
                                10) Run the gel at 60V for 20 minutes.
                            </P><DIV class="ME-title-p"><A>3. Gel purification</A></DIV><P class="ME-content-p">
                                Protocol for Gel Purification is based on the introduction of Tiangen Gel Extraction
                                Kit.
                            </P><DIV class="ME-title-p"><A style="text-decoration: none;color:#131313;">4. Digestion</A></DIV><P class="ME-content-p">
                                1) Prepare reaction mix according to the following table
                            </P><DIV class="table-head"><TABLE><TBODY><TR><TH>Enzyme digestion system (20μL)</TH></TR></TBODY></TABLE></DIV><DIV class="table-body"><TABLE><TBODY style="font-size: 16px;"><TR><TD>Fragment or plasmid need to digest</TD><TD>16μL</TD></TR><TR><TD>corresponding enzymes of upstream restriction Enzyme cutting site</TD><TD>1μL</TD></TR><TR><TD>corresponding enzymes of downstream restriction Enzyme cutting site</TD><TD>1μL</TD></TR><TR><TD>Enzyme buffer (Buffer)</TD><TD>2μL</TD></TR></TBODY></TABLE></DIV><P class="ME-content-p">
                                2) incubate at recommended temperature (37℃) for at least 1 hour.
                            </P><P class="ME-content-p">
                                3) Purify the digestion product by inactivating at 80°C.
                            </P><DIV class="ME-title-p"><A style="text-decoration: none;color:#131313;">5.Ligation</A></DIV><P class="ME-content-p">
                                1) Prepare reaction mix according to the following table
                            </P><DIV class="table-head"><TABLE><TBODY><TR><TH>T4 DNA Ligase system (10μL)</TH></TR></TBODY></TABLE></DIV><DIV class="table-body"><TABLE><TBODY style="font-size: 16px;"><TR><TD>digested fragment/vector</TD><TD>8.5μL</TD></TR><TR><TD>T4 DNA ligase</TD><TD>0.5μL</TD></TR><TR><TD>10x T4 DNA ligase buffer</TD><TD>1μL</TD></TR></TBODY></TABLE></DIV><P class="ME-content-p">
                                2) Ligate the mix overnight at 16℃.
                            </P><P class="ME-content-p">
                                Note: when 3A assembly is performed, we always adjust the dose of each fragment
                                according to their concentration to make sure the amounts of them are same to each
                                other. Vector can be added less than fragment but no less than 1.5μL.
                            </P><DIV class="ME-title-p"><A style="text-decoration: none;color:#131313;">6. Transformation</A></DIV><DIV class="ME-title-p2"><A style="text-decoration: none;color:#131313;">a. Yeast</A></DIV><P class="ME-content-p">
                                1) The cells are added into 2 to 5 ml of liquid YPD medium and incubate overnight at
                                30°C with a shaker.
                            </P><P class="ME-content-p">
                                2) The cultured cells are seeded in fresh YPD medium at a volume of 10% by volume.
                            </P><P class="ME-content-p">
                                3) Cultured at 30°C and 200 rpm for 4-5 hours. Note: To make the cells complete at
                                least twice mitosis, the conversion efficiency will remain unchanged after 3-4 mitosis.
                            </P><P class="ME-content-p">
                                4) Add the cell culture solution to a 1.5 ml centrifuge tube, centrifuge at 4000 rpm
                                for 5 minutes or 5000 rpm for 2mins.
                            </P><P class="ME-content-p">
                                5) Discard the supernatant and resuspend the cells with 1 ml of sterile water and
                                centrifuge again.
                            </P><P class="ME-content-p">
                                6) The supernatant is discarded and the cells are resuspended with 1 ml of 100mM LiAc
                                and allowed to stand for five minutes.
                            </P><P class="ME-content-p">
                                7) 4000 rpm for three minutes, discard the supernatant.
                            </P><P class="ME-content-p">
                                8) The SS-DNA is boiled for five minutes and rapidly cooled on ice.
                            </P><P class="ME-content-p">
                                9) Add the conversion mixture in sequence: 240ul PEG(50% w/v), 36ul 1.0M LiAc; 10ul
                                SS-DNA（10.0mg/ml); x ul DNA（1~5ul); (74-x) ul Sterile water; Total:360ul;
                            </P><P class="ME-content-p">
                                10) Centrifuge the tube for one minute to allow the ingredients to mix well.
                            </P><P class="ME-content-p">
                                11) Incubated in a 30 degree incubator for 30 minutes; 20 to 30 minutes in a 42 degree
                                water bath.
                            </P><P class="ME-content-p">
                                Note: The time of thermal shock and the type of bacteria related to the specific
                                circumstances can be optimized to improve the conversion efficiency.
                            </P><P class="ME-content-p">           12) 4000 rpm for three minutes, and the supernatant is removed with a micropipette.
                            </P><P class="ME-content-p">
                                13) 4000 rpm for five minutes, discard the medium and wash twice with sterile water.
                            </P><P class="ME-content-p">
                                14) 100ul of sterile water to resuspend the cells (action to be gentle), coated plate,
                                and then cultured 2 to 4 days.
                            </P><DIV class="ME-title-p2"><A style="text-decoration: none;color:#131313;">b. <I>E.coli</I></A></DIV><P class="ME-content-p">
                                1) Take the competent cells (BMTOP10/DH5α) out of the Ultra-low temperature freezer.
                            </P><P class="ME-content-p">
                                2) Put the competent cells in the ice for 5 minutes.
                            </P><P class="ME-content-p">
                                3) Add 10ul of the ligation product to the competent cells and put them in the ice for
                                30 minutes.
                            </P><P class="ME-content-p">
                                4) Next put the competent cells in water about 42℃ for 90 seconds. Then put the
                                competent cells in ice for 5 minutes immediately.
                            </P><P class="ME-content-p">
                                5) Add 600ul LB-media to the competent cells and put them into the Incubator shaker for
                                1 hour in 37℃.
                            </P><P class="ME-content-p">
                                6) Take 100ul of the competent cells into the LB plate containing corresponding
                                antibiotics and spread them on the plate evenly.
                            </P><P class="ME-content-p">
                                7) Put the LB plate to the Incubator for 18 hours.
                            </P><DIV class="ME-title-p"><A>7. Protocol for mini-prep</A></DIV><P class="ME-content-p">
                                Protocol for mini-prep is based on the introduction of Tiangen Mini-prep Kit8.
                            </P><DIV class="ME-title-p"><A style="text-decoration: none;color:#131313;">8. Yeast Genome extraction</A></DIV><P class="ME-content-p">
                                Protocol for Yeast Genome extraction is based on the introduction of TIANamp Yeast DNA
                                Kit.
                            </P></DIV></DIV></DIV></DIV></DIV><DIV id="ME2" class="cd-section"><DIV class="ME-title-1"><A style="text-decoration: none;color:#131313;">Special protocol</A></DIV><DIV class="ME-content-all ME-margin-toTitle1 "><DIV id="M1" class="collapseDiv"><H3 id="a-nav-dot-box-M1" class="ME-title-content-a">
                        1. fluorescence measurement
                    </H3><DIV id="collapse_body_M1" class="collapse_body"><DIV class="collapse_content"><P class="ME-content-p">
                                Protocol for fluorescence measurement is based on the introduction of Reactive oxygen
                                species (ROS) testing Kit.
                            </P><DIV class="ME-title-p"><A style="text-decoration: none;color:#131313;">a. Yeast induced culture</A></DIV><P class="ME-content-p">
                                1) add the yeast into 2 to 5 mL of liquid medium and incubated overnight in a shaker.
                            </P><P class="ME-content-p">
                                2) The cultured cells are seeded in fresh medium at a volume of 10% by volume.
                            </P><DIV class="ME-title-p"><A style="text-decoration: none;color:#131313;">b. ROS detection by DCFH-DA staining</A></DIV><P class="ME-content-p">
                                For the detection of accumulated ROS, incubate DCFH-DA with yeast cells for 30 min
                                (37℃), then wash the cells with PBS twice and analyzed by flow cytometry and
                                Fluorescent microplate reader.
                            </P></DIV></DIV></DIV><DIV id="M2" class="collapseDiv"><H3 id="a-nav-dot-box-M2" class="ME-title-content-a">
                        2. Yeast Survival Rate Test
                    </H3><DIV id="collapse_body_M2" class="collapse_body"><DIV class="collapse_content"><P class="ME-content-p">
                                Spotting assays and viability tests
                            </P><P class="ME-content-p">
                                For spotting assays, yeast strains recovered from −80°C vials were shaken in SGR-URA
                                liquid medium overnight for revitalization. Then they were adjusted to the identical
                                OD600 of ∼0.5 and serially diluted to 10−1, 10−2, and 10−3. A 3-μl amount of each
                                diluted yeast culture was spotted on SD-URA or other plates. SGR-URA plates served as
                                the spotting control. Generally, pictures were taken after 2 or 3 d, depending on the
                                growth.
                            </P><P class="ME-content-p">
                                For survival tests, yeast cells were shaken in test tubes and grown until they reached
                                the exponential phase, and then different apoptotic inducers were added to induce
                                apoptosis. After the cells were treated by Mn for 12 h or H<SUB>2</SUB>O<SUB>2</SUB>
                                for 100 min, the number
                                of surviving colonies was determined by plating a small aliquot of the treated cultures
                                on yeast extract/peptone/dextrose plates or selective plates (SD or SGR dropout
                                plates).
                            </P></DIV></DIV></DIV><DIV id="M3" class="collapseDiv"><H3 id="a-nav-dot-box-M3" class="ME-title-content-a">
                        3. Yeast Growth Measurement
                    </H3><DIV id="collapse_body_M3" class="collapse_body"><DIV class="collapse_content"><P class="ME-content-p">             1) The cells are added into 2 to 5 ml of liquid YPD medium and incubate overnight at
                                30°C with a shaker.
                            </P><P class="ME-content-p">
                                2) The cultured cells are seeded in fresh YPD medium at a volume of 10% by volume.
                            </P><P class="ME-content-p">
                                3) When it grow to the right time, take some of it to do the test through
                                Fluorescent microplate reader.
                            </P></DIV></DIV></DIV><DIV id="M4" class="collapseDiv"><H3 id="a-nav-dot-box-M4" class="ME-title-content-a">
                        4. dCas9
                    </H3><DIV id="collapse_body_M4" class="collapse_body"><DIV class="collapse_content"><P class="ME-content-p">
                                1) PCR of a plasmid containing dCas9 to obtain a gene fragment of dCas9 protein.
                            </P><P class="ME-content-p">
                                2) Synthesis of g RNA by chemical synthesis.
                            </P><P class="ME-content-p">
                                3) Construct the skeleton of the plasmid by OE-PCR.
                            </P><P class="ME-content-p">
                                4) Using dCas9 plasmid as template, PCR was used to recover the fragment and the vector
                                backbone was added to a 6.5 μl mix system at a molar ratio of 2:1, and then 1 μl of taq
                                Ligase was added, 50 degree reaction for 1 h, PCR instrument Temperature control, after
                                the reaction is completed, the large intestine transformation is coated with the
                                ampicillin plate.
                            </P><P class="ME-content-p">
                                5) Colony PCR, get the target plasmid
                            </P></DIV></DIV></DIV><DIV id="M5" class="collapseDiv"><H3 id="a-nav-dot-box-M5" class="ME-title-content-a">
                        5. qPCR
                    </H3><DIV id="collapse_body_M5" class="collapse_body"><DIV class="collapse_content"><P class="ME-content-p">
                                1) RNA extraction
                            </P><P class="ME-content-p">
                                1. Take 2mL of cells cultured to log phase for centrifugation at 1000g for 5 minutes
                                at 4°C
                            </P><P class="ME-content-p">
                                2. Discard the supernatant, add 2mL BufferSE/LyticaseMixture, and keep it at 30 degrees
                                for half an hour.
                            </P><P class="ME-content-p">
                                3. Centrifuge at 3400g for 5 minutes, discard the supernatant, add 350u
                                LBufferYRL/2-mercaptoethanol (20uL/mL) and 50g glass beads, beat at maximum speed for 5
                                minutes, centrifuge at 13000g for 3 minutes, transfer the supernatant to a new
                                centrifuge tube.
                            </P><P class="ME-content-p">
                                4. Add an equal amount of 70% ethanol, transfer the sample to the adsorption column,
                                centrifuge at 13000g for 1 minute, discard the supernatant.
                            </P><P class="ME-content-p">
                                5. Add 700uL WashBufferI, centrifuge for 1 minute, discard the supernatant
                            </P><P class="ME-content-p">
                                6. Add 500uLWashBufferII wash, centrifuge at 13000g for 1 minute, discard the
                                supernatant, repeat once, centrifuge again for 2 minutes.
                            </P><P class="ME-content-p">
                                7. Pack the adsorption column with a clean 1.5mL centrifuge tube, collect the RNA with
                                30-50uLDEP-treated water, and centrifuge for 1 minute to get the RNA solution.
                            </P><P class="ME-content-p">
                                2) RT-PCR
                            </P><P class="ME-content-p">
                                System
                            </P><DIV class="table-head"><TABLE><TBODY><TR><TH>ingredient</TH><TH>volume（μL) </TH></TR></TBODY></TABLE></DIV><DIV class="table-body"><TABLE><TBODY style="font-size: 16px;"><TR><TD>ddH<SUB>2</SUB>O</TD><TD>6</TD></TR><TR><TD>5×Primrscript RT Master Mix</TD><TD>2</TD></TR><TR><TD>TotalRNA</TD><TD>2</TD></TR><TR><TD>Total</TD><TD>10</TD></TR></TBODY></TABLE></DIV><P class="ME-content-p">
                                Program
                            </P><DIV class="table-body"><TABLE><TBODY style="font-size: 16px;"><TR><TD>37℃</TD><TD>15min</TD></TR><TR><TD>85℃</TD><TD>5sec</TD></TR></TBODY></TABLE></DIV><P class="ME-content-p">
                                3) qPCRr
                            </P><P class="ME-content-p">
                                System
                            </P><DIV class="table-head"><TABLE><TBODY><TR><TH>ingredient</TH><TH>volume（μL) </TH></TR></TBODY></TABLE></DIV><DIV class="table-body"><TABLE><TBODY style="font-size: 16px;"><TR><TD>2×SYBR Green qPCR MasterMix</TD><TD>10</TD></TR><TR><TD>T</TD><TD>1</TD></TR><TR><TD>F</TD><TD>1</TD></TR><TR><TD>R</TD><TD>1</TD></TR><TR><TD>ROX Reference Dye</TD><TD>1</TD></TR><TR><TD>DdH<SUB>2</SUB>O</TD><TD>6</TD></TR><TR><TD>Total</TD><TD>20</TD></TR></TBODY></TABLE></DIV><P class="ME-content-p">
                                Program
                            </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>