<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BIT-China_ExperimentsRegulator skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BIT-China/ExperimentsRegulator</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><UL id="left-nav"><LI><A>PROJECT</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Background">Background</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Design">Idea &amp; Design</A></LI></UL><LI><A>EXPERIMENTS</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/ExperimentsRegulator">Regulator</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/ExperimentsFeedback">Feedback</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/ExperimentsOutput">Output</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Results">Results</A></LI><LI><A>MODELING</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Model">Overview</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/FluorescentProbesModel">Fluorescent Probe Model </A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/H2O2DecompositionModel">H<SUB>2</SUB>O<SUB>2</SUB>
                        Decomposition Model</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/roGFP2-Orp1MichaelisEquationModel">roGFP2-Orp1
                        Michaelis equation Model</A></LI><LI><A>HUMAN PRACTICES</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/HPOverview">Overview</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Human_Practices">Integrated Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Public_Engagement">Education &amp; Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Collaborations">Collaborations</A></LI><LI><A>NOTEBOOK</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Notebook">Lab Book</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Protocols">Methodology / Protocols</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Equipment">Material &amp; Equipment</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/InterLab">Measurement / InterLab</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Safety">Safety</A></LI><LI><A>ACHIEVEMENTS</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/JudgingForm">Judging Form</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Applied_Design">Applied Design</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Demonstrate">Demonstrate</A></LI><LI><A>TEAM</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Team">Members</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:BIT-China/Gallery">Gallery</A></LI></P><DIV class="EXP-title"><A class="EXP-title-1" style="z-index:4;border-bottom-style: solid;text-decoration: none;color: #131313;">REGULATOR</A></DIV><DIV id="REG" class="EXP-content-container" style="z-index: 1;margin-top:calc(25vh - 30px);"><DIV id="REG0" class="cd-section"><DIV id="REG-1" class="cd-section"><DIV class="EXP-title-2 EXP-margin-Title2Up"><A style="text-decoration: none;color: #131313;">Overview</A></DIV><DIV class="EXP-content-all"><DIV class="EXP-content"><P class="EXP-content-p">
                            Previously we showed how we design the whole system, now we are going to tell you how we
                            achieve them steps by steps:
                        </P></DIV></DIV></DIV><DIV id="REG-2" class="cd-section "><DIV class="EXP-title-2 EXP-margin-Title2Up"><A style="text-decoration: none;color: #131313;">Regulator</A></DIV><DIV class="EXP-content-all"><DIV class="EXP-content"><P class="EXP-content-p">
                            To increase the accumulation of endogenous ROS in <I>Saccharomyces cerevisiae</I> and
                            to improve the tolerance of yeast to high levels of ROS, we optimized our yeast, which will
                            be shown as follow:
                        </P></DIV><DIV class="EXP-title-3 EXP-new-margin-title4UP"><A style="text-decoration: none;color: #131313;">Increase the accumulation of endogenous ROS</A></DIV><DIV class="EXP-title-4 EXP-new-margin-title4UP"><A style="text-decoration: none;color: #131313;">Overview:</A></DIV><P class="EXP-content-p">
                        After literature searching, we selected two genes, <I>yno1</I> and <I>ndi1</I>, as candidates
                        for overexpression to increase the accumulation of endogenous ROS. According to our data, both
                        of these two genes can effectively increase the accumulation of endogenous ROS.
                    </P><P class="EXP-content-p">
                        To determine whether overexpression of <I>ndi1</I> or <I>yno1</I> can increase the endogenous
                        ROS accumulation in yeast, we constructed an expression plasmid, based on pESC-Leu, in which
                        the cloned <I>ndi1</I> or <I>yno1</I> is driven by GAL1 promoter. Thus the target gene can be
                        induced by galactose and repressed by glucose.
                    </P><FIGURE class="EXP-Fig EXP-margin-toContentP"><FIGCAPTION>Fig. 1,2 Transformed the plasmid into yeast</FIGCAPTION></FIGURE><P class="EXP-content-p">
                        After transformed the plasmid into <I>Saccharomyces cerevisiae</I>, we first measured the ROS
                        production of yeast cells cultured in non-screening or screening media by Fluorescent
                        microplate reader. We mixed yeast cells with DCFH-DA, which can be oxidized by ROS to become a
                        strong green fluorescent substance DCF (dichlorofluorescein) that cannot penetrate the cell
                        membrane.
                    </P><FIGURE class="EXP-Fig EXP-margin-toContentP"><FIGCAPTION>Fig. 3,4 results of ROS accumulation through overexpress <I>yno1</I> or <I>ndi1</I></FIGCAPTION></FIGURE><P class="EXP-content-p">
                        Fluorescent microplate reader analysis showed that overexpression of either <I>ndi1</I> or <I>yno1</I>
                        may cause significant ROS production as revealed by green fluorescent substance DCF, and the
                        fluorescence intensity in <I>ndi1</I>-overexpressing cells was a liitle bit higher than tat in
                        <I>yno1</I>-overexpressing cells.
                    </P><P class="EXP-content-p">
                        we performed three times in YPD medium and three times in SD-Leu-deficient medium. A time
                        period of 0h-48h was selected and detected once every four hours. After excluding some
                        experimental operational errors, we obtained the results in the figure.
                    </P><P class="EXP-content-p">
                        It has been shown that external NADH dehydrogenases Ndi1 and Yno1 are involved
                        in the generation of intracellular oxidative stress, however, the results measured by the
                        fluorescence microplate reader cannot exclude the interference of dead cells on OD600, we
                        therefore examined the ROS accumulation through flow cytometry.
                    </P><FIGURE class="EXP-Fig EXP-margin-toContentP"><FIGCAPTION>Fig.5 Results of ROS accumulation detection through flow cytometry</FIGCAPTION></FIGURE><DIV class="EXP-title-4 EXP-new-margin-title4UP"><A style="text-decoration: none;color: #131313;">Replace the
                            promoter of <I>yno1/ndi1</I> in yeast genome</A></DIV><P class="EXP-content-p">
                        In order to reduce the pressure on our engineered strains from the addition of plasmid and to
                        prevent plasmid loss, we decided to introduce the gene circuit which increases the level of
                        endogenous ROS into the yeast genome. We chose to replace the <I>ndi1/yno1</I> promoter with
                        the galactose-inducible promoter GAL1p we used. Because the promoter of the endogenous <I>ndi1/yno1</I>
                        gene in yeast is a bidirectional promoter, we chose to insert the gal1 promoter upstream of the
                        <I>ndi1/yno1</I> gene by OE-PCR.
                    </P><FIGURE class="EXP-Fig EXP-margin-toContentP"><FIGCAPTION>Fig.6 Replace the promoter of <I>yno1/ndi1</I></FIGCAPTION></FIGURE><P class="EXP-content-p">
                        After successfully replacing the promoter of the yeast endogenous <I>ndi1/yno1</I> gene,
                        to obtain the overexpression effect, we tested the mRNA quantity. The results are as follows.
                    </P><FIGURE class="EXP-Fig EXP-margin-toContentP"><FIGCAPTION>Fig.7 qPCR result in original strain (C, in blue below), and in strain with <I>yno1</I>
                            overexpressed and <I>yca1</I> knockout (YY, in red below).</FIGCAPTION></FIGURE><FIGURE class="EXP-Fig EXP-margin-toContentP"><FIGCAPTION>Fig.8 qPCR result in original strain (C, in blue below), and in strain with <I>yno1</I>
                            overexpressed and <I>ndi1</I> knockout (YN, in red below).</FIGCAPTION></FIGURE><P class="EXP-content-p">
                        According to the result, we could find out that strain with <I>yno1</I> overexpressed and <I>yca1</I>
                        knockout had an obvious increase of mRNAs. However, we didn't get the same result in ndi1
                        overexpressed strain. We believed this result showed that the overexpression of <I>ndi1</I> was
                        failed while the <I>yno1</I> one succeeded.
                    </P><DIV class="EXP-title-3 EXP-new-margin-title4UP"><A style="text-decoration: none;color: #131313;">Improve the tolerance to high levels of ROS</A></DIV><DIV class="EXP-title-4 EXP-new-margin-title4UP"><A style="text-decoration: none;color: #131313;">Overview:</A></DIV><P class="EXP-content-p">
                        To block the response of yeast to ROS, we knocked out the yeast-derived yca1 gene. Because Yca1
                        is the only known yeast metacaspase, to demonstrate genetically that Ndi1 functions
                        independently of Yca1, we overexpressed Ndi1 in yca1 mutant background. Consistently,Ndi1
                        overexpression exacerbated cell death in the absence of Yca1
                    </P><DIV class="EXP-title-4 EXP-new-margin-title4UP"><A style="text-decoration: none;color: #131313;">knocking out gene <I>yca1</I></A></DIV><FIGURE class="EXP-Fig EXP-margin-toContentP"><FIGCAPTION>Fig.9 knocking out gene <I>yca1</I></FIGCAPTION></FIGURE><FIGURE class="EXP-Fig EXP-margin-toContentP"><FIGCAPTION>Fig.10 Verify whether the gene is successfully knocked out</FIGCAPTION></FIGURE><UL class="EXP-content-p"><LI>
                            1. Left homologous arm of <I>Δyca1</I>（500bp）
                        </LI><LI>
                            2: Right homologous arm of <I>Δyca1</I>（500bp）
                        </LI><LI>
                            3: wide-type
                        </LI><LI>
                            4: wide-type
                        </LI></UL><P class="EXP-content-p">
                        Based on the result, we determined that there was no mistake in the process of knocking out <I>yca1</I>
                        gene and that the <I>yca1</I> gene was knocked out indeed.
                    </P><DIV class="EXP-title-4 EXP-new-margin-title4UP"><A style="text-decoration: none;color: #131313;">verify whether knocking out gene <I>yca1</I>
                            can increase yeast tolerance to H<SUB>2</SUB>O<SUB>2</SUB></A></DIV><P class="EXP-content-p">
                        If the growth of <I>Δyca1</I> and <I>yno1-Δyca1</I> was better than that of WT and yno1, it
                        can be concluded that knocking out <I>yca1</I> gene can improve yeast tolerance to H<SUB>2</SUB>O<SUB>2</SUB>.
                        We diluted seed liquid to OD600 at 2, which was Saccharomyces Cerevisiae of overexpressing yno1
                        gene and lacking the <I>yca1</I> gene <I> (yno1-Δyca1) </I> and strains of overexpressing yno1
                        gene <I> (yno1) </I> as control group. And then adjust the concentration of H<SUB>2</SUB>O<SUB>2</SUB>
                        in the seed
                        liquid to 0 mM, 1 mM, and 2 mM respectively. The yeasts were cultured for 16h. Take samples
                        every 2-4h. Its OD600 was measured by the ultraviolet spectrophotometer. OD600 reflected a
                        growth of yeasts which can judge whether <I>yca1</I> knockout can improve yeast tolerance to
                        H<SUB>2</SUB>O<SUB>2</SUB>. <B>The results are shown as follow:</B></P><FIGURE class="EXP-Fig EXP-margin-toContentP"><FIGCAPTION>Fig.11 Strain Growth Curve under 0mM H<SUB>2</SUB>O<SUB>2</SUB> Stress</FIGCAPTION></FIGURE><FIGURE class="EXP-Fig EXP-margin-toContentP"><FIGCAPTION>Fig.12 Strain Growth Curve under 1mM H<SUB>2</SUB>O<SUB>2 Stress</SUB></FIGCAPTION></FIGURE><FIGURE class="EXP-Fig EXP-margin-toContentP"><FIGCAPTION>Fig.13 Strain Growth Curve under 2mM H<SUB>2</SUB>O<SUB>2 Stress</SUB></FIGCAPTION></FIGURE><FIGURE class="EXP-Fig EXP-margin-toContentP"><FIGCAPTION>Figure.14 The OD<SUB>600</SUB> of strains at 12h under 0mM, 1mM, and 2mM H<SUB>2</SUB>O<SUB>2</SUB>
                            Stress stress</FIGCAPTION></FIGURE><P class="EXP-content-p">
                        As Figure.11~ Figure.14 shown, comparing the growth curves of <I>yno1</I> and <I>yno1</I>-∆<I>yca1</I>
                        under H<SUB>2</SUB>O<SUB>2</SUB> stress with 0mM, 1mM and 2mM, we can see that, the growth of
                        strain knocking out <I>yca1</I>
                        is always better than the strain without knocking out <I>yca1</I> after overexpressing <I>yno1</I>.
                    </P><P class="EXP-content-p">
                        It can be concluded that knocking out <I>yca1</I> can improve our engineering strain tolerance
                        to H<SUB>2</SUB>O<SUB>2</SUB>.
                    </P><DIV class="EXP-title-3 EXP-new-margin-title4UP"><A style="text-decoration: none;color: #131313;">Conclusions:</A></DIV><P class="EXP-content-p">
                        Knock out <I>yca1</I> gene can increase the yeast tolerance to hydrogen peroxide.
                    </P><DIV class="EXP-title-4 EXP-new-margin-title4UP"><A style="text-decoration: none;color: #131313;">Summary of regulator part</A></DIV><P class="EXP-content-p">
                        To summarize briefly, overexpress <I>ndi1/yno1</I> can accumulate ROS successfully, also,
                        knocking out gene <I>yca1</I> can improve the tolerance to high level of ROS content.
                    </P></DIV></DIV></DIV></DIV><DIV class="footer-all" style="margin-top:65px;z-index: 1;"><DIV class="footer-right"><P style="font-family:'kg_second_chances_solidRg';font-size:12px;margin: 0;padding: 0 0 3px;margin-top: 6px;">Contact</P><P style="font-size:15px;margin: 0;padding: 0;margin-top: 4px;">Institute of
                Biotransformation and synthetic biosystem School of Chemistry and Chemical Engineering.
            </P><P style="font-size:15px;margin: 0;padding: 0;margin-top: 4px;">
                Beijing Institute of Technology
            </P><P style="font-size:15px;margin: 0;padding: 0;margin-top: 4px;">
                100081, Beijing
            </P><P style="font-size:15px;margin: 0;padding: 0;margin-top: 4px;">
                Email:lichun@bit.edu.cn
            </P><P style="font-size:15px;margin: 0;padding: 0;margin-top: 4px;color:  rgb(161, 161, 161);">
                Copyright © 2018 BIT-China
            </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>