<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NEU-China_Progress skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NEU-China/Progress</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Progress - NEU-China</TITLE></P><DIV class="container"><DIV class="navbar-header"><A class="navbar-brand" href="https://2016.igem.org/Team:NEU-China"><SPAN style="font-size: 26px"><SPAN style="color: #F00;">I</SPAN><SPAN style="color: #0F0;">T</SPAN><SPAN style="color: lightblue;">S</SPAN> COLOR</SPAN></A></DIV><DIV class="collapse navbar-collapse navbar-right"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:NEU-China">Home</A></LI><LI><A href="Background">Background</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Project </A></LI><LI><A href="Description">Description</A></LI><LI><A href="Design">Design</A></LI><LI><A href="ExperimentResults">Experiment &amp; Results</A></LI><LI><A href="Proof">Proof of concept</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Lab </A></LI><LI><A href="Team">Team</A></LI><LI><A href="Attributions">Attributions</A></LI><LI><A href="LabLife">Lab Life</A></LI><LI class="dropdown active"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Notebook </A></LI><LI><A href="Progress">Progress</A></LI><LI><A href="Protocol">Protocol</A></LI><LI><A href="Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Parts </A></LI><LI><A href="OurParts">Our Parts</A></LI><LI><A href="BiobricksForMedal">Biobricks for medal</A></LI><LI><A href="Basic_Part">Basic Parts</A></LI><LI><A href="Part_Collection">Part Collection</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Human Practices </A></LI><LI><A href="Human_Practices">Human Practices</A></LI><LI><A href="Silver">Silver</A></LI><LI><A href="Gold">Gold</A></LI><LI><A href="Collaborations">Collaborations</A></LI></DIV></DIV><SECTION id="blog" class="container"><DIV class="row"><DIV class="col-md-10 col-md-offset-1"><DIV class="title-img"><H2 style="left: 38%; top: 5%; margin-left: -30px;">Progress</H2></DIV></DIV></DIV><DIV class="blog"><DIV class="row"><DIV class="col-md-10 col-md-offset-1"><DIV class="blog-item"><DIV class="row"><DIV class="col-xs-12 blog-content"><OL><LI><P>    Extraction of the total RNA of <I>Arabidopsis thaliana</I>, then reverse transcription PCR to obtain the cDNA.Amplification of the CIB1 gene and CRY2 gene from the cDNA. Nest PCR to verify the length and to amplify the PCR product.
                                    </P></LI><LI>
                                            Borrowed a plant of Arabidopsis thaliana from Shenyang Normal University and got ready to start the experiment.
                                        </LI><LI>
                                           Extracted of the total RNA of <I>Arabidopsis thaliana</I> twice times. The RNA we extracted for the first time showed a little degraded. The second time, we changed the extraction kit and improved the grinding method and the gel testing was here. It was apparent that there existed 3 bands which might mean 28S、18S、5S RNA. 
                                             <P class="text-center"><STRONG>Figure 1.1: Used Plant RNA Kit to extract the total RNA.</STRONG></P><DIV class="text-center"><STRONG>Figure 1.2: Extracted the total RNA again in an improved method.</STRONG></DIV><P>
                                        There was nothing to do with the band of DNA marker. We just wanted to see how did the total RNA degrade and then we performed RT-PCR with the OligdT primer rather than random primers immediately to obtain the cDNA of mRNA.
                                    </P></LI><LI> Amplified the CIB1 gene and CRY2 gene with designed primers from the cDNA we extracted before. Both of them showed pretty weak bands which could be hardly see. A nest PCR was performed to increase concentration and to enhance specificity. </LI><LI> We extracted and purified the correct length of those two gene fragments as shown below: </LI><P class="text-center"><STRONG>Figure 1.3: A weak band in the left picture, and a clear band in the right after NEST PCR. </STRONG></P><P class="text-center"><STRONG>Figure 1.4: CRY2 gene was successfully extracted from cDNA.</STRONG></P><I> Abstract We extracted the total RNA in a normal way according the kit instruction at the first time. The second time we improved our experiment by changing grinding tools and exposing the plant under blue light overnight. Finally, We have successfully obtained the fragments of the correct length as so far. </I><LI>
                                    PCR amplification of the tCas9 gene from the commercial pHS-CR042 plasmid. Overlap PCR the tCas9 and CIB1 gene after gel purification respectively. Restriction digest of the overlap PCR product and the expression vector pBD024. Ligation of the digested products and then transformed the ligation product into <I>DH5alpha</I> bacteria cells. Patch and miniprep to obtain our expression plasmid that produces tCas9-CIBN fusion protein.
                                    </LI><LI>Amplified of the tCas9 gene from the commercial pHS-CR042 plasmid and modified at 5’ end. Overlap pcr was applied to fuse CIBN with tCas9 after purification. </LI><P class="text-center"><STRONG>Figure 2.1: The top strip was the overlapping product tCas9-CIBN. The other bands were by-products.</STRONG></P><LI>
                                            Gel extracted of the fusion gene and then ligated with pSB024 vector. 
                                        </LI><P class="text-center"><STRONG> Figure 2.2: Picked out SC(single colony)-3 and SC-4 containing tCas-CIB1 to culture overnight. Extracted the plasmid.</STRONG></P><P class="text-center"><STRONG>Figure 2.3: Double enzyme (Kpn I, Xba I ) digestion test showed the correct band of tCac9-CIB1(5Kb)</STRONG></P><LI>
                                            After gene sequencing, we found that the vector we constructed appeared the frameshift mutation. We consulted the literature again, adjusted the structure of the CIB1 gene, and cloned the partial sequence of the N terminal before the mutation, named CIBN.
                                        </LI><P class="text-center"><STRONG>Figure 2.4: Modified the CIB1 gene. Amplified CIBN geng by specific primers. </STRONG></P><LI>
                                            Next, the same procedure was carried out: PCR with specific primers we booked, double enzyme digestion, phenol purification, ligation, transformation, colony PCR, path ,and restriction digest testing at last. the following is our digestion test results, sequencing results show completely consistent with the expected return sequence. 
                                        </LI><P class="text-center"><STRONG>Figure 2.5: SC-2, SC-6, SC-9, SC-11, SC-13, SC-15 were verified correct through colony PCR.</STRONG></P><P class="text-center"><STRONG>Figure 2.5: Enzyme (BamH I, XbaI) cut and the length of band were correct.</STRONG></P><I>
                                        Abstract: Unexpected mutation might occur when you amplify a specific gene through PCR method. What’s more, the efficiency of restricted digestion and ligation would also effect the result of constructing a functional plasmid. We have improved our protocal according to the experiments we performed before: PCR annealing temperature was reset at 55 degrees Celsius up and down; After removal from the PCR instrument, the PCR product was rapidly cooled; Enzyme digestion for the night to ensure its efficiency; ligation and transformation were both performed on the ice with low temperature and so on. In conclusion, we have constructed an expression vector for fusion protein expression pBD024-tCas9-CIBN. The corresponding improvements were made about the the mutation occurred at experiments.
                                    </I><LI>
                                    PCR amplification of the synthetic VP64. Cloning the VP64 into SK plasmid which is used in our own laboratory. Sending the plasmid to sequence after digest testing. Overlap PCR of the CRY2 and VP64 gene after gel purification respectively. Restriction digest of the overlap products and a commercial plasmid pBD021 and then ligated. Transformed into DH5alpha and patched. Plasmid preparation of the plasmid after colony PCR testing. 
                                    </LI><LI> 
                                            Vp64, the transcriptional activator we purchased from the company turned out to be wrong to a certain extent. the DNA didn’t match the sequence of NCBI after BLAST. Therefore, we tried to synthesis of that gene entirely. 
                                            <P class="text-center"><STRONG>Figure 3.1: Fused the wrong VP64 with CRY2. The bands of these two gene were so close.</STRONG></P></LI><LI>
                                            Amplified of the synthetic VP64 and cloned it into SK plasmid. Send this plasmid to sequencing and the feedback showed that the VP64 synthesized completely was correct, It meant that we could continue to use it in the later experiment. The following picture showed the correct result of double enzyme digestion test.
                                            <P class="text-center"><STRONG>Figure3.3: we used 10Kb marker this time for the lacking of 2000Kb marker. VP64 strips were verified to be correct in the later experiment.</STRONG></P></LI><LI>
                                            Amplified of the CRY2 gene we extracted at the first step. Overlap PCR to connect the CRY2 and PV64 together after gel purification respectively. 
                                        </LI><LI>
                                            Gel extracted of the fusion gene and then ligated with pSB024 vector. After sending the constructed plasmid to sequencing, time had passed for nearly a week. 
                                            <P class="text-center"><STRONG>Figure 3.4: Colony PCR result showed that there were six positive colonies.</STRONG></P></LI><LI>
                                            A bad news made us mad after analyzing the feedback gave by sequencing company that there existed 3 intron fragments in CRY2 gene due to incompletely removal of genome DNA. We had no method but to clone the 4 extrons respectively and to fuse them together by the means of redesigning overlapping primers.
                                            <P class="text-center"><STRONG>Figure 3.5: Did Overlap PCR three times in order to fuse four fragments together.</STRONG></P></LI><LI>
                                            Fortunately, the progress of removing the introns went smoothly. You could see from the picture below that we purified the CRY2 without introns successfully and the sequence analyzing proved it correctly at the same time.
                                            <P class="text-center"><STRONG>Figure 3.6: Successfully extracted the plasmid containing the CRY2 without introns.</STRONG></P></LI><LI>
                                            Next, the same procedure was carried out: PCR with specific primers we booked, double enzyme digestion, phenol purification, ligation, transformation, colony PCR, path, and restriction digest testing at last. the following is our digestion test results, sequencing results showed that we had successfully constructed the pBD021-CRY2-VP64 expression vector.
                                            <P class="text-center"><STRONG>Figure 3.7: PCR amplification of CRY2-VP64</STRONG></P><P class="text-center"><STRONG>Figure 3.8: Colony PCR: CRY2-VP64 (free of intron). </STRONG></P></LI><I>
                                        Abstract: prepare a backup in case of need. Overall, we successfully constructed the pBD021-CRY2-VP64 plasmid even though the process is tortuous.
                                    </I><LI>
                                    Transformed the pBD024-tCas9-CIBN and pBD021-CRY2-VP64 expression plasmid into <I>BL21 E.coli</I> strain respectively. Cultured in LBK(LB culture with kanamycin antibiotics) overnight. Protein expression induced by arabinose. Protein extraction from the <I>E.coli</I> culture. western blot or Silver Staining after SDS-PAGE Gel electrophoresis to detect if the protein is expressed.
                                </LI><UL><LI> 
                                        Designed experiments to check if the fusion protein would be expressed in prokaryote cell such as <I>E.coli</I>. Transformed the pbD024-tCas9-CIBN and pBD021-CRY2-VP64 vectors into <I>BL21</I> strain respectively. 
                                    </LI><LI>
                                        Colony PCR was done to select positive colonies after transformation. pBD024 plasmid consisted of inducible promoter (pBad/araC). Protein expression started within the existence of arabinose. 
                                    </LI><LI>
                                        Coomassie blue staining and silver staining was performed at first, but the effects were not satisfactory. The purpose band was not distinguished clearly from negative control group. Under that condition, we planed to do western blot to verify the existence of tCas9-CIBN protein’s HA tag. 
                                    </LI><LI>
                                        We explored the appropriate induce condition via pre-experiment. Data proved the protein was truly expressed. We found that 4mM arabinose, 30 Celsius culture temperature, and 16h culture time were the best choice to induce the expression of protein tCas9-CIBN. 
                                    </LI><LI>
                                        Protein CRY2-VP64 was also detected by western bolt rather than coomassie blue staining and silver staining. The best culture condition concluded above was applied into that experiment but did not use arabinose because of the constitutive promoter of pBD021. The results are illustrated as bellows:
                                    </LI><P class="text-center"><STRONG>Figure 4.1: Colony PCR of transformed <I>E.coli</I> Picked out the positive transformed colonies to culture. Protein was induced to expressed after a night.</STRONG></P><P class="text-center"><STRONG>Figure 4.2：Western blot analysis of CRY2-VP64 protein levels.</STRONG>
                                1:Control groups transfected with empty plasmid;      2 3 4:transfected with CRY2-VP64 plasmid (J23114 promoter) from different single colonies, ~78.5 kDa. Stationary cultures of <I>BL21 </I>was subcultured into fresh media and growth for 4 hours. 30ng of protein with total volume of 30ul (protein sample + dissociation buffer).
                            </P><P class="text-center"><STRONG>Figure 4.3：Western blot analysis of tCas9-CIBN protein levels.</STRONG>
                                     1 3 5 7 9 11:Control groups transfected with empty plasmid; 2 4 6 8 10 12: transfected with tCas9-CIBN plasmid (pBAD promoter), ~180 kDa
. Stationary cultures of <I>BL21 </I>was subcultured into fresh media and growth for 4 hours or 16 hours using different concentrations L-arabinose. 30ng of protein with total volume of 30ul (protein sample + dissociation buffer).
                            </P><P class="text-center"><STRONG>Figure 4.4: Western blot analysis of tCas9-CIBN protein levels.</STRONG>
                                     1 3 5 7 9 11:Control groups transfected with empty plasmid; 2 4 6 8 10 12: transfected with tCas9-CIBN plasmid (pBAD promoter), ~180 kDa
. Stationary cultures of <I>BL21 </I>was subcultured into fresh media and growth for 4 hours or 16 hours using different concentrations L-arabinose. 30ng of protein with total volume of 30ul (protein sample + dissociation buffer).
                            </P></UL><I>
                                    Abstract: the result of western blot showed that tCas9-CIBN protein and CRY2-VP64 protein were detected. Next we would test the function of LACE system by means of co-transformation of these device vector.
                                </I><LI>
                                    gRNA designed and booked the specific primers. Cloned the gRNA fragment into the expression plasmid containing CRY2-VP64. Transformed all of the pBD024-tCas9-CIBN, pBD021-CRY2-VP64-gRNA and pCold-GFP (containing the gRNA targeted reporter gene) into <I>BL21 E.coli</I> strain. Test the function of the light-inducible system in prokaryote cell after selection.
                                </LI><P>
                                With different target loci have been tested by the usage of a GFP reporter plasmid (pCold-1) with a CSPA promotor. The target sites can be determined by directing the gRNA consisting of 20 bp length against the desired sequence of interest. R1 with target sites at different distances to the promotor regions proved successfully as potential activation sites (see Table 1 and Figure 1).
                            </P><TABLE class="table table-striped table-bordered table-responsive text-center"><TBODY><TR><TD><B>Name</B></TD><TD><B>Binding Site</B></TD><TD><B>Distance to promoter</B></TD><TD><B>Sequence</B></TD><TD><B>Position</B></TD></TR><TR><TD>F1</TD><TD>CSPA promoter</TD><TD>15</TD><TD>TGCATCACCCGCCAATGCG</TD><TD>sense sequences</TD></TR><TR><TD>F2</TD><TD>non-coding</TD><TD>68</TD><TD>GCCGCCGCAAGGAATGGTG</TD><TD>sense sequences</TD></TR><TR><TD>R1</TD><TD>CSPA promoter</TD><TD>43</TD><TD>ATTAATCATAAATATGAAA</TD><TD>antisense sequences</TD></TR><TR><TD>R2</TD><TD>non-coding</TD><TD>94</TD><TD>CATCATCCAACTCCGGCAAC</TD><TD>antisense sequences</TD></TR></TBODY></TABLE><P class="text-center"><B>Table 1: Overview of the tested gRNAs with different binding sites on the GFP pCold-1 plasmid.</B></P><P class="text-center"><STRONG>Figure 5.1: Position of the target loci on the GFP pCold-1 plasmid.</STRONG><LI>
                                    Used the methods above to construct the eukaryotic expression vector. Cloned the right gene into PESC and P426 eukaryotic expression vector respectively. Send the plasmid to sequencing.
                                    </LI><LI>
                                            Cloned tCas9-CIBN, CRY2-VP64 into pESC vector, this plasmid consisted of eukaryotic promoter and we added <I>yeast</I> RBS at the N terminal of the gene. After send the constructed vector to sequencing, we designed the next step to transform it into <I>yeast</I> by some different ways.
                                        </LI><LI>
Cloned tCas9-CIBN, CRY2-VP64 into p426 vector, this plasmid consisted of eukaryotic promoter and we added yeast RBS at the N terminal of the gene. That plasmid contained constitutive promoter. What’s more, the transformant bacteria which is nutrient deficiency type before could survive at SC plate without Ura amino acid.</LI></P><P class="text-center"><STRONG>Figure 6: 4 plasmids were cut by Xba I and EcoR I respectively. All of these plasmids had 2 restriction digest Xba I sites and 1 restriction digest EcoR I sites.</STRONG></P><P>Three lanes in each group were added in such sequence: Xba I (double cut), EcoR I (single cut), original plasmid. Expected results were lists as below:</P><TABLE class="table table-striped table-bordered table-responsive text-center"><TBODY><TR><TD><B>Name</B></TD><TD><B>Length</B></TD><TD><B>Cut by EcoR I</B></TD><TD><B>Cut by Xba I right result</B></TD><TD><B>Cut by Xba I wrong result</B></TD></TR><TR><TD>pESC-tCas9-VP64</TD><TD>11068</TD><TD>11068</TD><TD>7112, 3956</TD><TD>2827, 3956</TD></TR><TR><TD>pESC-tCas9-CIBN</TD><TD>11413</TD><TD>11413</TD><TD>7457, 3956</TD><TD>2715, 3956</TD></TR><TR><TD>p426-tCas9-CIBN</TD><TD>11881</TD><TD>11881</TD><TD>7462, 4419</TD><TD>2726, 4419</TD></TR><TR><TD>pESC-CRY2-VP64</TD><TD>8800</TD><TD>8800</TD><TD>4844, 3956</TD><TD>2727, 3956</TD></TR></TBODY></TABLE><P>We could conclude by comparing the gel test photo: p426-tCas9-CIBN, pESC-CRY2-VP64 were correct but the bacteria we picked out were not single clone. The recombinant plasmid we extracted contained empty plasmid. They need extra purification.</P><UL><LI>Constructed the recombinant pESC-tCas9-CIBN again. The picture below was the enzyme digestion result, the length of the tCas9-CIBN was around 4700bp.</LI></UL><LI>
                                    Overlap of the tCas9 and VP64 gene and cloned it to different empty expression plasmids, which acted as control.
                                </LI><UL><LI>
                                        We designed a control experiment to verify the validity of the LACE system. However, we met across some trouble of creating the tCas9-VP64 fusion gene.
                                        <P class="text-center"><STRONG>Figure 7.1: PCR amplification of VP64.</STRONG></P><P class="text-center"><STRONG>Figure 7.2: PCR amplification of tCas9.</STRONG></P></LI><LI>
                                        At first, we followed the regular progress to perform overlap PCR. A unexpected phenomenon came up that gel electrophorisis showed nothing, even a weak band, which never happened before. We guessed that it might have something to do with the length of that two genes, tCas9 is about 4Kb and VP64 240bp. So, we planed to change the experimental method to create that fusion gene.
                                    </LI><LI>
                                        Changed the method of purification before overlap PCR. We purified the PCR products through gel extraction. But this time, we tried phenol purification instead. The result was not better in that way. 
                                    </LI><LI>
                                        Changed the origin of the VP64 amplification. We used double enzymes to digest the SK-VP64 plasmid constructed before to see if the plasmid was degraded. Finally, unclear strips let us down and we had to turn eyes to other plasmid which contained VP64. We were so upset to accept the fact that we were failed again. Colony PCR demonstrated that there existed something wrong during the previous experiments.
                                        <P class="text-center"><STRONG>Figure 7.3: the right strip was tCas9-VP64 while the left tCas9 as a contrast.</STRONG></P></LI><LI>
                                        Then we doubted the annealing temperature might influenced the result. The optimal annealing temperature for that reaction was proposed by gradient PCR (51 Celsius ~59 Celsius). We could conclude from the picture that every temperature between 51 Celsius to 59 Celsius was okay to overlap contrast to another gene’s obvious changes of gradient on the left. We were confronted with a series weak bands full of confusion. What we chose to do next was to continue base on the unsatisfactory result. 
                                        </LI><LI>
                                        When we cloned the fusion gene into eukaryote vector pESC, we found there existed a correct strip via colony PCR. Then, we extracted the plasmid from the right colony. After enzyme digestion test and sequencing test, we finally attained the right control vector.
                                        </LI><LI>
                                        Amplified the prokaryote fragment of tCas9-VP64 from the eukaryote vector we constructed above. 
                                        </LI><P>Abstract: The progress of fusing tCas9 with VP64 is tortuous. That might have something to do with the sharp difference of the length. In addition, we were confused with the efficiency of transformation during the experiment. When we did colony PCR before, for example: pBD024-dCas9-CIBN and pBD021-CRY2-VP64, more than 80 percent colonies were verified correct. However, when it came to pBD024-tCas9-VP64, less than 10 percent were right. We speculated that expression of tCas9-VP64 may affect the growth of <I>E.coli</I>. Then, we replaced pBD024 plasmid into pSB1C3, because the RFP in the pSB1C3 could reflect the transformation efficiency through the changing of color. The colonies which didn’t possess the tCas9-VP64 showed red color while the colonies possessing the object gene showed white. The result was that the efficiency of the tCas9-VP64 transformation was extremely low, around 0.4 percent colonies was white. It is a regret that we forget to take a picture of the transformation plate to conserve the record. Fortunately, we selected a right colony and extracted its plasmid. Digestion testing and sequence analyzing results showed it correct.</P></UL><LI>
                                    Parts preparation. All parts for the submission are amplified using PCR that added the biobrick restriction sites. After the restriction digest, these fragments were cloned into the PSB1C3 vectors. Finally, the parts were verified using colony PCR and send to sequencing.
                                    </LI><LI>
                                            At first, we extracted the pSB1C3 backbone from the kit. Then selected the red colonies which expressed RFP to culture and extracted the plasmid. 
                                        </LI><LI>
                                            Amplifed the fragments that would be submitted: tCas9, tCas9-CIBN, CIBN, CRY2, VP64, CRY2-VP64. All of these parts consisted of prokaryotic RBS.
                                        </LI><P class="text-center"><STRONG>Figure 8.1: amplification of CRY2, CRY2-VP64</STRONG></P><P class="text-center"><STRONG>Figure 8.4: Restriction enzyme test results.</STRONG></P><P class="text-center"><STRONG>Figure 8.5: Restriction enzyme test results.</STRONG></P><P>Abstract: 
We had constructed these parts as below:
<A href="http://parts.igem.org/Part:BBa_K1982000">BBa_K1982000</A><A href="http://parts.igem.org/Part:BBa_K1982001">BBa_K1982001</A><A href="http://parts.igem.org/Part:BBa_K1982002">BBa_K1982002</A><A href="http://parts.igem.org/Part:BBa_K1982003">BBa_K1982003</A><A href="http://parts.igem.org/Part:BBa_K1982005">BBa_K1982005</A><A href="http://parts.igem.org/Part:BBa_K1982006">BBa_K1982006</A><A href="http://parts.igem.org/Part:BBa_K1982007">BBa_K1982007</A><A href="http://parts.igem.org/Part:BBa_K1982008">BBa_K1982008</A><A href="http://parts.igem.org/Part:BBa_K1982009">BBa_K1982009</A><A href="http://parts.igem.org/Part:BBa_K19820010">BBa_K1982010</A><A href="http://parts.igem.org/Part:BBa_K1982011">BBa_K1982011</A><A href="http://parts.igem.org/Part:BBa_K1982012">BBa_K1982012</A>
You can get the better understanding of the information by clicking the links.
</P><P>
We had did some experiments to help NEFU-China to find out the best induce condition and to observe the total protein we extracted. The details also can be found at <A href="https://2016.igem.org/Team:NEU-China/Progress">progress part</A> we received a tube of plasmid from NEFU-China. According to their description, we designed experiments below to achieve the test goals:
</P><OL><LI>Transformed the plasmid into  <I>DH5alpla </I> strain to amplify.</LI><LI>Then miniprep the right vector after culturing.</LI><LI>Transformed the obtained plasmid into  <I>Bl21 </I> strain in 3 50ml centrifuge tubes respectively.</LI><LI> Cultured at LB liquid medium with ampicillin antibiotic. Induced time was set as 4h, 8h, 16h.</LI><LI>Ultrasonication in the protein lysis buffer.</LI><LI>Reserved the supernatant after centrifuging.</LI><LI>SDS-page electrophoresis.</LI><LI>Coomassie blue staining after times of washing</LI></OL><P>
We could see from the below picture (Figure 1) that the quantity of the expression protein was increasing with the time increasing. Concerning the obvious death of bacteria beyond 16h culturing, we recommend NEFU-China to induce the protein around 16h.
</P><DIV class="text-center"><P class="text-center"><STRONG>Figure 9.1</STRONG></P><P>
In order to compare with the control apparently, we contrasted the 16h induced protein and not induced protein. The result we got was illustrated as below (Figure 2) . We put these results together and send to NEFU-China, hoping what we did would helped them.
</P><DIV class="text-center"><P class="text-center"><STRONG>Figure 9.2</STRONG></P></DIV><P>
We also asked NEFU-China to help us detect the function of a plasmid which could express GFP. That really saved us amount of time. The final effect was pretty good, we had build a closely cooperation relationship! Both of each sides were looking forward another collaboration!
</P><H3 class="text-center">Summary of experimental records</H3><P>1.Extract total RNA from Arabidopsis thaliana. Gel electrophoto showed 2 bands. Reverse transcription to prepare cDNA. Label samples as R1, R2. 
2.Extract total RNA from <I>Arabidopsis thaliana</I>. Gel electrophoto showed 3 bands. Reverse transcription to prepare cDNA. Label samples as R3, R4. 
3.PCR amplification: (the number below are Laboratory primer codes and commercial disignation of plasmid)   
  tCas9: template 1619; primer 2086, 2087 
  VP64: template 1619; primer 2092, 2104
  CIB1: template cDNA R3; primer 2088, 2103
  CRY2: template cDNA R3; primer 2090, 2091
4. Gel electrophoresis: tCas9 right; CIB1, CRY2, VP64 wrong. 
(04/18/2016~04/22/2016)
1. PCR amplification:  
  CIBN: template cDNA R4; primer 2088, 2103 
  CRY2: template cDNA R4; primer 2090,2091  
2. Gel electrophoresis: CIB1 right weak; CRY2 wrong.  
3. Nest PCR: CIB1 54 Celsius, 34x. Result: bright right.  
(04/27/2016~04/30/2016) 
1. Gel purification: tCas9 (87.2 ng/ul), CIB1 (60.7ng/ul). 
2. Overlap PCR: template: tCas9 CIB1; primer 2086, 2103. Three strips: 1Kb 4Kb 5Kb. 
Gel purification: tCas9-CIB1
3.Enzyme digestion: Kpn I, Xba I; tCas9-VP64 and pBD024; overnight 
4. Colony PCR: 8 tubes, Patches, pick out the correct colonies to culture overnight.
5. Miniprep of plasmid. 
6. ecut (Enzyme cut) test and run gel electrophoresis. 
pBD024-tCas9-CIB1 SC-3, SC-4 are right.
7. PCR amplification:
  CRY2: template cDNA 4; primer 2180, 2091. 
Electrophoresis: right bright.
(05/02/2016~05/06/2016)
1.phenol purification of CRY2 (86.3 ng/ul) and VP64 (81.5 ng/ul)
2.Electrophoresis: both are correct
3.Overlap PCR
4.Ecut (Enzyme cut) overnight: Xba I, Sal I; CRY2-VP64, pBD021.
Phenol purification, Ligation, Transformation: <I>DH5alpha</I> (resistance Kan) ,Centrifuge, spread the transformed bacteria on the LB solid medium, culture overnight.
5.Colony PCR: 16 tubes, pathches, pick out the right colonies to culture overnight: SC-1, 2, 7, 9 (primer dimer in the front)
6.Miniprep of plasmid.
7.Ecut test and run gel electrophoresis.
pBD021-CRY2-VP64 SC-1, 2, 7, 9 are correct.
(05/07/2016~05/11/2016)
1. PCR amplification: tCas9-VP64
Template: 1619; primer 2086, 2091
2. gel purification
3. Ecut overnight : Kpn I, Spe I; tCas9-VP64, pBD024
4. Phenol purification, Ligation, Transformation: DH5alpha (resistance Kan) ,Centrifuge, spread the transformed bacteria on the LB solid medium, culture overnight.
5. Colony PCR: 16 tubes, pathches, pick out the right colonies to culture overnight: SC-2, 3, 5, 7
6. Miniprep of plasmid.
7.Ecut test and run gel electrophoresis.
pBD024-tCas9-VP64 SC-3, 5, 7 are correct.
(05/12/2016~05/17/2016)
1. Plasmid amplification of pBD021, pBD024
2. Remove introns of CRY2, amplication of fragment:
F1 (319bp), F2 (225bp), F3 (713bp), F4 (792bp)
3. Clone the gene tCas9-CIBN (4725bp) from the pBD024-tCas9-CIB1 plasmid.
4. Overlap PCR: F1, F2, F3; weak band, need a nest PCR.
Nest PCR of F1~F3, successful.
Overlap PCR: F1~F3, F4; tree band 1200bp, 800bp, 2000bp. 
Gel purification of F1~F4 (CRY2 free of intron). 
5. Ecut overnight: tCas9-CIBN, pBD024 (Kpn I, Xba I); CRY2, pBD021 (Xba I, Sal)
Phenol purification, Ligation, Transformation: <I>DH5alpha</I> (resistance Kan), Centrifuge, spread the transformed bacteria on the LB solid medium, culture overnight.
6. Colony PCR: pBD021-CRY2 SC-3, 5, 7, 8. tCas9-VP64 none.
7. Colony PCR of tCas9-VP64 again SC-6, 11, 13.
(05/27~05/31)
1. Design and book the primer within biobrick digestion site.
2. Point mutation to change the Pst I restriction enzyme site of tCas9-CIBN.
PCR amplification of tCas9-CIBN, phosphorylation, ligation, transformed into DH5alpha strain. Centrifuge, spread the transformed bacteria on the LB solid medium, culture overnight.
3. Miniprep of the plasmid. 
Point mutation again to change the Pst I restriction enzyme site of tCas9-CIBN, phosphorylation, ligation, transformed into <I>DH5alpha</I> strain. Centrifuge, spread the transformed bacteria on the LB solid medium, culture overnight.
4. Colony PCR to pick out the colonies possessing the recombinant plasmid.
5. Miniprep of the plasmid in which no Pst I restriction enzyme site exist.
(06/05-06/12)
1.design the experiment of characterization of fusion protein.
2.transform the pBD024-tCas9-CIBN into BL21 strain in order to induce the expression of fusion protein.
3.culture in LB liquid medium after colony PCR.
4.use L-arabinose to induce the protein expression and control groups are set at the same time.
5.centrifuge and discard the supernatant, extracted the total protein.
6.SDS-PAGE electrophoresis, the target protein is around 180KDa.
7.silver staining, no obvious bands are observed.
(06/14~06/19)
1.transform the pBD024-tCas9-CIBN into BL21 strain in order to induce the expression of fusion protein.
2.culture in LB liquid medium after colony PCR.
3.use L-arabinose to induce the protein expression and control groups are set at the same time.
Induce condition: 4mM 30Celsius; 10mM 30Celsius; 15mM 30Celsius; 4mM 37Celsius; 10mM 37Celsius; 15mM 37Celsius.
4.centrifuge and discard the supernatant, extracted the total protein, determine the concentration by BCA method.
5.SDS-PAGE electrophoresis , the target protein is around 180KDa.
6.prepare western bolt reagents.
7.conduct transmembrane, add first antibody (HA), put it on the shaking table overnight.
8.add the second antibody after washing.
9.luminescence detection.
(06/20~06/30)
1.amplification of CRY2: primer 2538, 2091; VP64: primer 2495, 2539. (eukaryotic RBS, biobrick prefix and suffix)
2.Gel extraction:
CRY2-1 (74 ng/ul), CRY2-2 (68.9 ng/ul), VP64-1 (19.1 ng/ul), VP64-2 (22.7 ng/ul)
3.overlap PCR of CRY2-1 with Vp64-2.
4.Restriction enzyme digestion (EcoR I, Sal I)
5.Ligation and transformation.
6.Colony PCR: SC-7, 8, 9, 13.
7.Minipreped of the plasmid:
SC-7 506.5 ng/ul, SC-8 312.8 ng/ul, SC-9 262.4 ng/ul, SC-13 319.1 ng/ul.
8.send to sequencing.
(07/02~07/10 )
1.amplification of CRY2-VP64 ( prokaryotic RBS, biorick prefix and suffix).
2.Look for parts in DNA Kit:
pSB1A3: 2016 plate 4 2H.
pBad/araC Promoter: 2016 plate 5 17N.
pSB1C3 (BBa_J04450) 2016 plate 4 4B.
Extracte and amplify of these plasmid.
3.amplification of tCas9-CIBN. 
4.Miniprep of the plasmid:
pSB1A3 153.0 ng/ul.
pSB1C3 137.7 ng/ul.
pBad/araC Promoter 125.8 ng/ul.
5.Ecut:  pSB1A3 EcoR I, Pst I; pBad/araC Promoter EcoR I, Spe I; pSB1C3 Xba I, Pst I.
6.Ligation and transformation.
7.Colony PCR: SC-2, 6, 15, 16.
8.Miniprep of the plasmid (ng/ul): SC-2 258.5, SC-6 245.6, SC-15 260.4, SC-16 276.7
9.Ecut testing.
(07/10~07/16)
1.transform the pBD021-CRY2-VP64 into BL21 strain in order to induce the expression of fusion protein.
2.culture in LB liquid medium (possesses Kan antibiotics) after colony PCR.
3.set control groups at the same time.(constitutive promoter)
culture condition: 15mM 30Celsius 6h.
Adjust the cell absorbance value up to 1, start the time.
4.centrifuge and discard the supernatant, extracted the total protein, determine the concentration by BCA method.
5.SDS-PAGE electrophoresis , the target protein is around 180KDa.
6.prepare western bolt reagents.
7.conduct transmembrane, add first antibody (FLAG), put it on the shaking table overnight.
8.add the anti-rabbit after washing.
9.luminescence detection.
10.repeat the experiments above 3 times to avoid the bad effect brought about by faults and to ensure the accuracy of the results.
(07/15~08/04)
1.construct the eukaryotic plasmid pESC-tCas9-CIBN, PESC-tCas9-VP64.
2.PCR amplify of.
tCas9: template pBD024-tCas9-CIBN; primer 2542, 2087
VP64: template Sk-VP64; primer 2496, 2573
3.gel purification: tCas9 5.2ng/ul; VP64 22.0ng/ul
4.pSB1A3-[pBad/araC Promoter]-[CRY2-VP64] plasmid is supposed to be purified and sent to sequencing.
5.gel test result showed that there existed no bands. PCR amplification of tCas9 and VP64 need to be performed again.
6.PCR amplification of tCas9 VP64 and CRY2-VP64, the upstream primer is modified with prokaryotic RBS.
7.Phenol purification after PCR and prepare the plasmid pESC and pSB1C3.
8.Overlap tCas9 and VP64, weak bands is observed.
9.Ecut plasmid and gene fragments mentioned above overnight after purification (Xba I Pst I).
Phenol purification, Ligation, Transformation: DH5alpha ,Centrifuge, spread the transformed bacteria on the LB solid medium, culture overnight.
10.Miniprep of the pESC-tCas9-CIBN pESC-tCas9-VP64 and pSB1C3-CRY2-VP64.
11.Gel testing and send to sequencing.
(08/04~08/12) 
1.construct the eukaryotic plasmid p426-CRY2-VP64.
2.Amplification of the CRY2-VP64 gene by the specific primer with eukaryotic RBS.
3.Ecut plasmid and gene fragments mentioned above overnight after purification (Xba I Pst I).
Phenol purification, Ligation, Transformation: DH5alpha ,Centrifuge, spread the transformed bacteria on the LB solid medium, culture overnight.
3.Miniprep of the plasmid.
4.Yeast transformation:
Competent yeast cell preparation: <I>BY4741</I> strain.
Transform according to the protocal. 
Finally spreed the yeast cell on the SC plate without His Amino acid.
Wait for 2~3days to get the transfected cell.
5.there is only one colony among 5 plates and the colony is polluted by bacteria under a microscope.
(08/13~08/18)

1.yeast transformation experiment:
Results: P426 plasmid (can survive at SC plate without Ura) pESC plasmid (can survive at SC plate without His)
The first plan without ssDNA
  1 P426 empty plasmid, spreed on the SC plate without Ura, colony morphology: sparse, big, around 50 
  2 pESC empty plasmid, spreed on the SC plate without His, colony morphology: dense, big, hundreds
  3 PESC-tCas9-CIBN, spreed on the SC plate without His, colony morphology: sparse, big, around 50
  4 PESC-tCas9-CIBN, spreed on the SC plate without Ura, colony morphology: none
  5 P426-tCa9-CIBN, spreed on the SC plate without His, colony morphology: none
  6 P426-tCa9-CIBN, spreed on the SC plate without Ura, colony morphology: sparse, big, around 20
  7 P426-tCa9-VP64, spreed on the SC plate without Ura, colony morphology: too dense
The second plan by using ssDNA
  1 P426 empty plasmid, spreed on the SC plate without Ura, colony morphology: big, 4 colonies
  2 PESC-tCas9-CIBN, spreed on the SC plate without His, colony morphology: none
  3 PESC-tCas9-CIBN, spreed on the SC plate without Ura, colony morphology: none
  4 P426-tCa9-CIBN, spreed on the SC plate without His, colony morphology: none
  5 P426-tCa9-CIBN, spreed on the SC plate without Ura, colony morphology: small, 2 colonies
  6 P426-tCas9-VP64, spreed on the SC plate without Ura, colony morphology: big, one colony
2.Extract the protein of selected colonies to western bolt
3.PAGE electrophoresis 
4.Transmembrane
5.Add antibody in turn
6.Luminescence detection
(08/19~08/25)

1.part preparation. Design and book the primers with BioBrick prefix and suffix.
2.amplification of pSB1C3.
3.amplification of the gene tCa9, CIBN, tCas9-CIBN, CRY2, VP64, CRY2-VP64, tCas9-VP64 (all of these genes possess prokaryotic RBS); tCas9, tCas9-CIBN, tCas9-VP64, CRY2-VP64 (all of these genes possess eukaryotic RBS).
4.Ecut overnight: Xba I, Pst I.
Phenol purification, Ligation, Transformation: <I>DH5alpha</I> (resistance Kan) ,Centrifuge, spread the transformed bacteria on the LB solid medium, culture overnight.
5.. Colony PCR: 8 tubes, Patches, pick out the correct colonies to culture overnight.
6. Miniprep of plasmid. 
7. ecut (Enzyme cut) test and run gel electrophoresis. The cut test result can be found at progress above.
8.send to sequencing.
(08/24~09/10)
1.amplified of the gene: CRY2-VP64-gRNA (F1, F2, R1, R2 ).
2.Prepare the plasmid pCold-GFP.
3.Ecut overnight after purification.
4.Phenol purification, ligation (3h), transformation, spread the bacterial fluid, culture overnight.
5.Colony PCR, run gel electrophoresis.
6.Cotransformation experiment: pBD024-tCas9-CIBN, pCold-CRY2-VP64-gRNA, pBD024-tCas9-VP64.</P><TABLE class="table table-striped table-bordered table-responsive text-center"><TBODY><TR><TD><B>Inserting gene</B></TD><TD><B>Right</B></TD></TR><TR><TD><B>F1 (1-5)</B></TD><TD><B>SC-3</B></TD></TR><TR><TD><B>F1 (2-5)</B></TD><TD><B>none</B></TD></TR><TR><TD><B>F2 (1-3)</B></TD><TD><B>none</B></TD></TR><TR><TD><B>F2 (2-5)</B></TD><TD><B>none</B></TD></TR><TR><TD><B>R1 (2-1)</B></TD><TD><B>none</B></TD></TR><TR><TD><B>R2 (2-5)</B></TD><TD><B>SC-2</B></TD></TR><TR><TD><B>R2 (1-3)</B></TD><TD><B>SC-2, 7</B></TD></TR><TR><TD><B>F2 (1-3)</B></TD><TD><B> none </B></TD></TR><TR><TD><B>F2 (1-5)</B></TD><TD><B>SC-1</B></TD></TR><TR><TD><B>F2 (2-1)</B></TD><TD><B>SC-2</B></TD></TR><TR><TD><B>F2 (2-5)</B></TD><TD><B>none</B></TD></TR></TBODY></TABLE><P>7. Culture the selected colonies to OD 0.2~0.5, add 100ul 1.5mM arabinose into each tube to induce the expression of protein. Culture for 7h and 20h respectively.
Colonies:</P><TABLE class="table table-striped table-bordered table-responsive text-center"><TBODY><TR><TD><B>pCold-GFP-gRNA F1</B></TD><TD><B>pCold-GFP-gRNA F2</B></TD><TD><B>pCold-GFP-gRNA R1</B></TD><TD><B>pCold-GFP-gRNA R2</B></TD><TD><B>pCold-GFP</B></TD></TR><TR><TD><B>pBD024-tCas9-CIBN + pCold-GFP-gRNA F1</B></TD><TD><B>pBD024-tCas9-CIBN + pCold-GFP-gRNA F2</B></TD><TD><B>pBD024-tCas9-CIBN + pCold-GFP-gRNA R1</B></TD><TD><B>pBD024-tCas9-CIBN + pCold-GFP-gRNA R2</B></TD><TD><B>pBD024-tCas9-CIBN + pCold-GFP</B></TD></TR><TR><TD><B>pBD024-tCas9-CIBN + pCold-GFP-CRY2-VP64-gRNA F1</B></TD><TD><B>pBD024-tCas9-CIBN + pCold-GFP-CRY2-VP64-gRNA F2</B></TD><TD><B>pBD024-tCas9-CIBN + pCold-GFP-CRY2-VP64-gRNA R1</B></TD><TD><B>pBD024-tCas9-CIBN + pCold-GFP-CRY2-VP64-gRNA R2</B></TD><TD><B>pBD024-tCas9-CIBN + pCold-GFP-CRY2-VP64</B></TD><TD><B>Light</B></TD></TR><TR><TD><B>pBD024-tCas9-CIBN + pCold-GFP-CRY2-VP64-gRNA F1</B></TD><TD><B>pBD024-tCas9-CIBN + pCold-GFP-CRY2-VP64-gRNA F2</B></TD><TD><B>pBD024-tCas9-CIBN + pCold-GFP-CRY2-VP64-gRNA R1</B></TD><TD><B>pBD024-tCas9-CIBN + pCold-GFP-CRY2-VP64-gRNA R2</B></TD><TD><B>pBD024-tCas9-CIBN + pCold-GFP-CRY2-VP64</B></TD><TD><B>Dark</B></TD></TR></TBODY></TABLE><P>8.extract the RNA after culture; culture condition: pick bacteria into 10ml LB medium within antibiotics (Kan at first), then make competent cells using that transformed cells, add arabinose to 15mM of final concentration when cell density is up to OD=0.2, culture to OD=1.
RNA concentration ( ng/ul ):</P><TABLE class="table table-striped table-bordered table-responsive text-center"><TBODY><TR><TD><B>Single trans</B></TD><TD><B>F1</B></TD><TD><B>F2</B></TD><TD><B>R1</B></TD><TD><B>R2</B></TD><TD><B>GFP</B></TD></TR><TR><TD><B>Parallel 1</B></TD><TD><B>127.4</B></TD><TD><B>556.7</B></TD><TD><B>141.2</B></TD><TD><B>228.5</B></TD><TD><B>76.2</B></TD></TR><TR><TD><B>Parallel 2</B></TD><TD><B>118.9</B></TD><TD><B>112.8</B></TD><TD><B>484</B></TD><TD><B>426.9</B></TD></TR><TR><TD><B>Parallel 3</B></TD><TD><B>193.7</B></TD><TD><B>139.3</B></TD><TD><B>220.8</B></TD><TD><B>17</B></TD></TR><TR><TD><B>Double trans (dark)
</B></TD><TD><B>F1</B></TD><TD><B>F2</B></TD><TD><B>R1</B></TD><TD><B>R2</B></TD><TD><B>GFP</B></TD></TR><TR><TD><B>Parallel 1</B></TD><TD><B>9.7</B></TD><TD><B>91.9</B></TD><TD><B>25.3</B></TD><TD><B>200.1</B></TD><TD><B>156.6</B></TD></TR><TR><TD><B>Parallel 2</B></TD><TD><B>193</B></TD><TD><B>9.3</B></TD><TD><B>115.1</B></TD><TD><B>98.1</B></TD><TD><B>161.9</B></TD></TR><TR><TD><B>Parallel 3</B></TD><TD><B>142.9</B></TD><TD><B>230.4</B></TD><TD><B>85.4</B></TD><TD><B>98.6</B></TD></TR></TBODY></TABLE><P>9.The result of protein extraction ( ug/ml ): 
induce 7 hours:</P><TABLE class="table table-striped table-bordered table-responsive text-center"><TBODY><TR><TD><B>Insert gene</B></TD><TD><B>F1</B></TD><TD><B>F2</B></TD><TD><B>R1</B></TD><TD><B>R2</B></TD><TD><B>GFP</B></TD></TR><TR><TD><B>Single-trans</B></TD><TD><B>4.85</B></TD><TD><B>6.16</B></TD><TD><B>8.44</B></TD><TD><B>6.41</B></TD><TD><B>10.5</B></TD></TR><TR><TD><B>Double-trans Dark
</B></TD><TD><B>7.97</B></TD><TD><B>7.264</B></TD><TD><B>10.18</B></TD><TD><B>9.69</B></TD><TD><B>10.422</B></TD></TR><TR><TD><B>Double-trans light
</B></TD><TD><B>7.29</B></TD><TD><B>13.18</B></TD><TD><B>10.34</B></TD><TD><B>8.6</B></TD><TD><B>10.18</B></TD></TR></TBODY></TABLE><P>10.Induce 20 hours:</P><TABLE class="table table-striped table-bordered table-responsive text-center"><TBODY><TR><TD><B>Insert gene</B></TD><TD><B>F1</B></TD><TD><B>F2</B></TD><TD><B>R1</B></TD><TD><B>R2</B></TD><TD><B>GFP</B></TD></TR><TR><TD><B>Single-trans</B></TD><TD><B>8.726</B></TD><TD><B>11.4</B></TD><TD><B>6.85</B></TD><TD><B>6.35</B></TD><TD><B>7.165</B></TD></TR><TR><TD><B>Double-trans Dark
</B></TD><TD><B>5.97</B></TD><TD><B>3.73</B></TD><TD><B>4.14</B></TD><TD><B>2</B></TD><TD><B>4.14</B></TD></TR><TR><TD><B>Double-trans light
</B></TD><TD><B>2.7</B></TD><TD><B>9</B></TD><TD><B>6.77</B></TD><TD><B>4.268</B></TD><TD><B>4.29</B></TD></TR></TBODY></TABLE><P>11.a strange phenomenon occurred that there are no colonies grow when we transformed the pBD024-tCas9-VP64 and pCold-GFP-CRY2-VP64-gRNA together into <I>BL21</I> strain. A deeper analysis need to be done!
(09/03~09/18)</P><H3>Data of fluorescence intensity</H3><P>1.single transformation (only pCold-GFP-CRY2-VP64-gRNA)</P><TABLE class="table table-striped table-bordered table-responsive text-center"><TBODY><TR><TD><B>F1</B></TD><TD><B>F2</B></TD><TD><B>R1</B></TD><TD><B>R2</B></TD><TD><B>GFP</B></TD></TR><TR><TD><B>4514.174</B></TD><TD><B>4466.732</B></TD><TD><B>3995.316</B></TD><TD><B>260.39</B></TD><TD><B>3051.696</B></TD></TR><TR><TD><B>4585.992</B></TD><TD><B>4539.22</B></TD><TD><B>4091.156</B></TD><TD><B>263.135</B></TD><TD><B>3117.996</B></TD></TR><TR><TD><B>3585.179</B></TD><TD><B>3738.119</B></TD><TD><B>3389.38</B></TD><TD><B>224.196</B></TD><TD><B>2715.204</B></TD></TR></TBODY></TABLE><P>2.double transformation (pCold-GFP-gRNA and pBD024-tCas9-CIBN)</P><TABLE class="table table-striped table-bordered table-responsive text-center"><TBODY><TR><TD><B>F1</B></TD><TD><B>F2</B></TD><TD><B>R1</B></TD><TD><B>R2</B></TD><TD><B>GFP</B></TD></TR><TR><TD><B>547.648</B></TD><TD><B>428.32</B></TD><TD><B>1948.543</B></TD><TD><B>271.859</B></TD><TD><B>5155.953</B></TD></TR><TR><TD><B>556.357</B></TD><TD><B>445.635</B></TD><TD><B>1968.589</B></TD><TD><B>272.803</B></TD><TD><B>5273.771</B></TD></TR><TR><TD><B>428.077</B></TD><TD><B>364.831</B></TD><TD><B>1579.659</B></TD><TD><B>230.373</B></TD><TD><B>4458.959</B></TD></TR></TBODY></TABLE><P>2. LACE detection: select F2 gRNA</P><TABLE class="table table-striped table-bordered table-responsive text-center"><TBODY><TR><TD><B>F2-LACE Night</B></TD><TD><B>F2-LACE Dark</B></TD><TD><B>GFP Repoter Only</B></TD></TR><TR><TD><B>654.632</B></TD><TD><B>327.749</B></TD><TD><B>324.668</B></TD></TR><TR><TD><B>642.65</B></TD><TD><B>324.943</B></TD><TD><B>328.759</B></TD></TR><TR><TD><B>539.614</B></TD><TD><B>273.303</B></TD><TD><B>292.301</B></TD></TR></TBODY></TABLE><P>(09/10~09/17)</P></DIV></OL></DIV></DIV></DIV></DIV></DIV></DIV></SECTION><FOOTER id="footer" class="midnight-blue"><DIV class="container"><DIV class="row"><DIV class="col-sm-6">© 2016 NEU-China Team.
            </DIV><DIV class="col-sm-6"><UL class="pull-right"><LI><A href="https://2016.igem.org/Team:NEU-China">Home</A></LI><LI><A href="Team">Our Team</A></LI></UL></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>