<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_MSU-Michigan_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:MSU-Michigan/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="topbar" style:=""><H1 style="color: black; position: relative;"><B> Michigan State University iGEM 2016 </B></H1><DIV class="nav"><UL><LI class="main"><A href="https://2016.igem.org/Team:MSU-Michigan">Home</A></LI><LI class="main">Team
      </LI><LI><A href="https://2016.igem.org/Team:MSU-Michigan/Team">Team Members</A></LI><LI><A href="https://2016.igem.org/Team:MSU-Michigan/Attributions">Attributions</A></LI></UL><LI class="main">Project
      </LI><LI><A href="https://2016.igem.org/Team:MSU-Michigan/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:MSU-Michigan/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:MSU-Michigan/Demonstration">Demonstration</A></LI><LI class="active main"><A href="https://2016.igem.org/Team:MSU-Michigan/Notebook">Notebook</A></LI><LI class="main"><A href="https://2016.igem.org/Team:MSU-Michigan/Parts">Parts</A></LI><LI class="main"><A href="https://2016.igem.org/Team:MSU-Michigan/Human_Practices">Human Practices</A></LI><LI class="main">More
      </LI><LI><A href="https://2016.igem.org/Team:MSU-Michigan/Hardware">Hardware</A></LI><LI><A href="https://2016.igem.org/Team:MSU-Michigan/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:MSU-Michigan/Awards">Awards</A></LI><LI><A href="https://2016.igem.org/Team:MSU-Michigan/Sponsors">Sponsors</A></LI></DIV></DIV><DIV class="column full_size" style="padding:20px; background-image:url('https://static.igem.org/mediawiki/2016/d/d8/T--MSU-Michigan--Notebook-img.png'); background-size: 100vw auto; background-repeat: no-repeat;"><H1 style="color:black; padding: 20px 40px 20px 20px; text-align:center; font-size:-webkit-xx-large; font-family: castellar,Helvetica,serif;">Experimental Documentation</H1><H1 style="text-align: center; padding: 0px 0px 10px 50px">July</H1><P style="text-align:right; padding: 10px 50px 0px 0px; color: black; font-size: 14px">(07/18/16)</P><H3 style="text-align: center">PCR Amplification (SFR2, tSFR2, desA) with Phusion High Fidelity Polymerase</H3><P style="padding: 10px 50px 10px 50px; color: black; font-size: 14px">
1. Mix the following reagents for PCR amplification, each reaction of 50 µl and we did one negative control where we did not put the template DNA. We also use 2 different buffers because we figured that our gene sequence has a high GC%.
Master-mix reaction mixture (4X):</P><P style="padding: 0px 0px 0px 100px; color: black; font-size: 14px">• Nuclease free H2O 134 µl• 5X HF/GC buffer 40 µl• DMSO 2 µl• dNTPs 4 µl• Forward primer (10 µM) 5 µl• Reverse primer (10 µM) 5 µl• Template DNA 6 ng (2 ng/reaction) • Phusion Polymerase 2 µl</P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">2. Run the PCR reaction with touch up program, where we run 7 cycles with the temperature of the primer without the overhang and another 30 cycles with the temperature of the primer full sequence. </P><P style="padding: 0px 0px 0px 100px; color: black; font-size: 14px">• SFR2 Touch-up PCR Program</P><TABLE style="margin-left: auto; margin-right: auto;width: 100%;"><TBODY><TR><TH>Step 1</TH><TH>Step 2</TH><TH>Step 3</TH></TR><TR><TD rowspan="2" style="vertical-align: middle;">98°C45 secs(1 cycle)</TD><TD>98°C, 61.7°C, 72°C7 secs, 20 secs, 80 secs(7 cycles)</TD><TD rowspan="2" style="vertical-align: middle;">72°C7 mins(1 cycle)</TD></TR><TR><TD>95°C, 71.3°C, 72°C7 secs, 20 secs, 80 secs(30 cycles)</TD></TR></TBODY></TABLE><P style="padding: 0px 0px 0px 100px; color: black; font-size: 14px">• tSFR2 Touch-up PCR Program</P><TABLE style="margin-left: auto; margin-right: auto;width: 100%;"><TBODY><TR><TH>Step 1</TH><TH>Step 2</TH><TH>Step 3</TH></TR><TR><TD rowspan="2" style="vertical-align: middle;">98°C45 secs(1 cycle)</TD><TD>98°C, 61.7°C, 72°C7 secs, 20 secs, 80 secs(7 cycles)</TD><TD rowspan="2" style="vertical-align: middle;">72°C7 mins(1 cycle)</TD></TR><TR><TD>95°C, 72°C, 72°C7 secs, 20 secs, 80 secs(30 cycles)</TD></TR></TBODY></TABLE><P style="padding: 0px 0px 0px 100px; color: black; font-size: 14px">• desA Touch-up PCR Program</P><TABLE style="margin-left: auto; margin-right: auto; width: 100%;"><TBODY><TR><TH>Step 1</TH><TH>Step 2</TH><TH>Step 3</TH></TR><TR><TD rowspan="2" style="vertical-align: middle;">98°C45 secs(1 cycle)</TD><TD>98°C, 62.9°C, 72°C7 secs, 20 secs, 60 secs(7 cycles)</TD><TD rowspan="2" style="vertical-align: middle;">72°C7 mins(1 cycle)</TD></TR><TR><TD>95°C, 72°C, 72°C7 secs, 20 secs, 60 secs(30 cycles)</TD></TR></TBODY></TABLE><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">3. Run Gel Electrophoresis with 0.8% agarose at 150 V for 30 minutes. 1 KB plus ladder is used.</P><DIV style="position:relative; padding: 0px;"><P style="color: black;">• A-C: SFR2 with HF buffer • D    : SFR2 negative control • E-G: SFR2 with GC buffer• H      : SFR2 negative control • I-K: tSFR2 with HF buffer • L  : tSFR2 negative control • M-O: tSFR2 with GC buffer • P  : SFR2 negative control</P></DIV><P style="color: black; position: relative;">• A-C: desA with HF buffer • D-F: desA with GC buffer</P><P style="padding: 0px 50px 10px 50px; color: black; font-size: 14px;">4. Gel Extraction was done with Wizard SV Gel and PCR clean up kit by Promega. Then, DNA concentrations were measured with nano drop.</P><P style="text-align:right; padding: 10px 50px 0px 0px; color: black; font-size: 14px">(07/19/16)</P><H3 style="text-align: center">Restriction Digest of Riboswitch Plasmid with EcoRI and XbaI </H3><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">
1. Restriction digest for the riboswitch plasmid (3745 bp) with Fast Digest from NEB in total reaction of 20 ul. Set up the following reaction:</P><P style="padding: 0px 50px 0px 100px; color: black; font-size: 14px">• Fast Digest buffer 2 µl• EcoRI restriction enzyme 1 µl• XbaI restriction enzyme 1 µl• Plasmid 1 µg• Nuclease free H2O until 20 µl</P><P style="padding: 0px 50px 10px 50px; color: black; font-size: 14px">2. Run Gel Electrophoresis with 0.8% agarose at 150 V for 30 minutes. Gel extraction and concentration is measured with nano drop. Plasmid Concentration: 8.1 ng/µl</P><H3 style="text-align: center">Gibson Assembly </H3><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">1. Calculate the number of pmols needed for each fragment if we use 30 ng of the vectors. </P><P style="padding: 0px 0px 0px 100px; color: black; font-size: 14px">(30ng X 1000)/(3745bp X 650)= 0.012 pmols </P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">2. We used 3 fold of excess insert, so the number of pmols for plasmid will be 3 times the amount of insert we need (0.036 pmols).</P><P style="padding: 0px 0px 0px 100px; color: black; font-size: 14px"> Amount of SFR2 needed: (x X 1000)/(1869bp X 650)= 0.036 pmols x= 44.91 ng</P><P style="padding: 0px 0px 0px 100px; color: black; font-size: 14px"> Amount of tSFR2 needed: (x X 1000)/(1788bp X 650)= 0.036 pmols x= 41.84 ng</P><P style="padding: 0px 0px 0px 100px; color: black; font-size: 14px"> Amount of desA needed: (x X 1000)/(1095bp X 650)= 0.036 pmols x= 25.62 ng</P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">3. Gibson Assembly master mix from NEB is used, so all we need to add is the plasmid backbone and the inserts according to the amount we calculated above in total reaction of 20 µl. We also run negative control where we just put the plasmid backbone without insert.</P><P style="padding: 0px 50px 10px 50px; color: black; font-size: 14px">4. Set up the reaction in 50°C for 30 minutes </P><H3 style="text-align: center">Transformation of the Recombinant Plasmid into DH5α Competent Cells </H3><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">1. Thaw competent cells on ice for around 5-10 minutes. </P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">2. Use 2 different volumes of Gibson product to mix with the competent cells. Add each tube of competent cells (50 µl) with 2.5 µl and 5 µl to 50 µl. </P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">3. Incubate on ice for 20 minutes. Heat shock cells at 42°C for 45 seconds.</P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">4.	Put it back on ice for another 2 minutes and add 900 µl of LB into the tube. Incubate in the 37°C shaker for an hour. </P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">5.	Spread plate 50 µl and 200 µl of them onto LB plates with spectinomycin antibiotic as the selective marker. </P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">6.	Incubate at 37°C incubator overnight </P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px"> Result: No colony grew in SFR2 and tSFR2 plates, but there were several colonies grew in the desA plate.</P><P style="text-align:right; padding: 10px 50px 0px 0px; color: black; font-size: 14px">(07/20/16)</P><H3 style="text-align: center">Prepare Bacterial Culture for Mini Prep</H3><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">1.	Add 50 ul of spectinomycin antiobiotic into 50 ml LB</P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">2.	In culture tubes, take 4 ml of LB + spectinomycin media. Pick isolated colonies formed on the desA transformation plate and mix well with the media (pipette up and down). </P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">3. Incubate overnight at 37°C shaker. </P><P style="text-align:right; padding: 10px 50px 0px 0px; color: black; font-size: 14px">(07/21/16)</P><H3 style="text-align: center">Plasmid Purification (Miniprep)</H3><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">1.	Spin down overnight culture in microcentrifuge tubes and discard the supernatant.</P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">2.	Re-suspend each pellet in 215 µl buffer P1 (cell resuspension solution).</P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">3.	Add 215 µl buffer P2 (cell lysis solution) and invert the tube 8 times to mix.</P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">4.	Add 330µl buffer P3 and invert 8 times to mix.</P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">5.	Spin down at 16,000 g for 5 minutes.</P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">6.	Apply 730 µl of supernatant to DNA column.</P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">7.	Spin down at 16,000 g for 1 minute and discard flow through. </P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">8.	Wash column with 700 µl wash buffer and spin down at 16,000 g for another minute. Discard flow through. Wash one more time with 500 µl wash buffer and discard flow through.</P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">9.	Dry spin column for 2 minutes to evaporate all the ethanol.</P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">10.Elute into new microcentrifuge tube with 37 µl nuclease free H<SUB>2</SUB>O. </P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">11. Measure concentration with nano drop. 
</P><P style="padding: 0px 50px 0px 100px; color: black; font-size: 14px">• SFR2 recombinant plasmid: 330.3 ng/ul • tSFR2 recombinant plasmid: 368.8 ng/ul • desA recombinant plasmid: 453.3 ng/ul</P><P style="text-align:right; padding: 10px 50px 0px 0px; color: black; font-size: 14px">(07/22/16)</P><H3 style="text-align: center">Recombinant Plasmid Verification with Restriction Digest</H3><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">1.	Run virtual digest of desA recombinant plasmid with KpnI restriction enzyme and SFR2, tSFR2 recombinant plasmid with SalI. </P><DIV style="position:relative; padding: 0px;"><P style="color: black;">1 KB plus ladder Lane 1: desA recombinant plasmid Lane 2: plasmid only Lane 3: SFR2 recombinant plasmid Lane 4: tSFR2 recombinant plasmid  Lane 5: plasmid only</P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">2.	Make the restriction reaction mixture in a total volume of 20 µl. Set up the following reaction:</P><P style="padding: 0px 50px 0px 100px; color: black; font-size: 14px">• 1 ug plasmid DNA• 2 ul cutsmart buffer• 1 ul restriction enzyme• Nuclease free H2O until 20 ul</P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">3.	Run gel electrophoresis in 0.8% agarose at 150 V for 30 minutes.</P><DIV style="position:relative; padding: 0px;"><P style="color: black;">We got positive result, where the total of 3 bands showed up at the correct size for desA recombinant plasmid and 2 bands for the control (plasmid only).</P></DIV><DIV style="position:relative; padding: 0px;"><P style="color: black;">We got negative results, it did not seems like the one in virtual digest. All the bands showed up almost at the same position as the negative control (plasmid only).</P></DIV><P style="text-align:right; padding: 10px 50px 0px 0px; color: black; font-size: 14px">(07/28/16)</P><H3 style="text-align: center">SFR2 &amp; tSFR2 Plasmid Verification with Restriction Digest(2<SUP>nd</SUP> attempt)</H3><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">1.	Run virtual digest of SFR2 and tSFR2 recombinant plasmid with XhoI and KpnI.</P><DIV style="position:relative; padding: 0px;"><P style="color: black;">1 KB plus ladder Lane 1: SFR2 recombinant plasmid Lane 2: tSFR2 recombinant plasmid  Lane 3: plasmid only</P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">2.	Make the restriction reaction mixture in a total volume of 20 µl. Set up the following reaction:</P><P style="padding: 0px 50px 0px 100px; color: black; font-size: 14px">• 1 ug plasmid DNA• 2 µl cutsmart buffer• 1 µl XhoI• 1 µl KpnI• Nuclease free H2O until 20 ul</P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">3.	Run gel electrophoresis in 0.8% agarose at 150 V for 30 minutes.</P><P style="color: black;">We got negative results. All the bands still showed up exactly like the negative control (plasmid only), where the top band showed up around 2 kb instead of 4 kb.</P></DIV><P style="text-align:right; padding: 10px 50px 0px 0px; color: black; font-size: 14px">(07/29/16)</P><H3 style="text-align: center">Send desA recombinant plasmid to sequencing facility</H3><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">1.	Dilute recombinant plasmid to 50 ng/µl.</P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">2.	In 1.5 ml centrifuge tube, take 5 µl of diluted recombinant plasmid, total of 250 ng, and 5 µl of forward <U>or</U> reverse sequencing primer, total of 25 µM.</P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">3. Plasmid is ready to be sequenced!</P><H1 style="text-align: center; padding: 10px 0px 10px 50px">August</H1><P style="text-align:right; padding: 10px 50px 0px 0px; color: black; font-size: 14px">(08/03/16)</P><H3 style="text-align: center">Analyze the sequencing result of SFR2, tSFR2, and desA</H3><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">1.	Open Benchling software and clone the SFR2, tSFR2 and desA to the riboswitch plasmid to prepare for sequence alignment.</P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">2.	Open the sequencing file (.ab1) in Snapgene Viewer and copy the whole sequence. </P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">3.	In Benchling, open the recombinant plasmid sequence, choose alignment button on the right side of the screen and paste the sequence we copied from Snapgene viewer into the sequence box.</P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">4.	Analyze how the gene sequence align with the expected recombinant plasmid. </P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">5.	Our desA plasmid sequence looks really well-aligned with the expected recombinant plasmid, but we lost a big chunk of sequence in our SFR2 and tSFR2 gene sequence.</P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px"> Next step: While we re-clone our SFR2 and tSFR2 gene, we were going to transform our desA gene into wild type cyanobacteria and wait a couple days to let them grow in BG11 plate. </P><P style="text-align:right; padding: 10px 50px 0px 0px; color: black; font-size: 14px">(08/04/16)</P><H3 style="text-align: center"> Repeat PCR for SFR2 and tSFR2 </H3><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">1. Set up the PCR reaction (refer to 08.17.16) </P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">2. Run the gel and extract the gel. </P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">3. Measure concentration with nano drop. SFR2 concentration: 8.1 ng/µl. </P><H3 style="text-align: center"> Clone SFR2 and tSFR2 PCR product into pJET plasmid</H3><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">Goal: Since the sequencing result of our SFR2 and tSFR2 gene did not come out good, so we decided to clone them to pJET plasmid to see if we got the actual sequence of our genes , after PCR-ing with the Gibson overhang (~20 bp sequence from the riboswitch plasmid).</P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">1.	Set up the ligation reaction on ice:</P><P style="padding: 0px 50px 0px 100px; color: black; font-size: 14px">•	5 ul 2X Reaction buffer• 90 ng PCR product• 0.5 µl pJET cloning vector (50 ng/µl)• 0.5 µl T4 DNA Ligase• Nuclease free H2O until 10 µl</P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">2.	Incubate the ligation mixture at room temperature for 10 minutes. Use ligation product directly for transformation.</P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">3. Thaw the competent cells on ice. </P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">4.	Add &lt;2.5µl ligation reaction to 25 µl cells. Mix gently by flicking the tubes and incubate on ice for 30 minutes. </P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">5.	Heat shock at 42°C for 30 seconds. Place back on ice for 2 minutes. </P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">6.	Add 250 µl room temperature SOC medium. Shake tubes for 60 minutes at 37°C. </P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">7.	Plate 50 µl of the transformation reactions on LB agar plates containing carbenicillin using sterile glass beads. Incubate at 37°C incubator overnight.</P><P style="text-align:right; padding: 10px 50px 0px 0px; color: black; font-size: 14px">(08/05/16)</P><H3 style="text-align: center"> Transformation of desA into WT Cyanobacteria </H3><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">1.	Take 2 ml of cyanobacteria culture that has been inoculated and grown to an optical density (OD<SUB>750</SUB>) of 0.5 to 0.8 under continuous illumination and permanent agitation. </P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">2.	Centrifuged cell culture and cell pellet was re-suspended in 500 µl BG11 medium.</P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">3.	100 ng of plasmid DNA was added and cells were gently mix by tapping. Incubated in darkness at 30°C for 4 hours. </P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">4.	While waiting for incubation, prepare BG11 plate with appropriate antibiotic (we used spectinomycin as the marker). </P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">5.	Following incubation, plate 50 µl and 200 µl of cells onto the plate with spread plate technique. </P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">6.	Incubate at 30°C with constant illumination for at least 5 days. </P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">Results: No colony grow even after 5 days. </P><P style="text-align:right; padding: 10px 50px 0px 0px; color: black; font-size: 14px">(08/08/16)</P><H3 style="text-align: center"> Colony PCR after transformation on 08.04.16</H3><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">1.	Pick 4 colonies from each plate (SFR2 &amp; tSFR2) and mix it well with 20 µl of nuclease free H2O.</P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">2.	Set up PCR reaction mixture:</P><P style="padding: 0px 50px 0px 100px; color: black; font-size: 14px">•	5 µl 2X Hot-Start Taq mix• 1 µl pJET 1.2 Forward (5 µM)• 1 µl pJET 1.2 Reverse (5 µM)• 3 µl colony suspension</P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">3.	Mix and briefly centrifuge. </P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">4.	Thermocycling condition for colony PCR:</P><TABLE style="margin-left: auto; margin-right: auto;width: 100%;">
  <TBODY><TR>
    <TH>Step 1</TH>
    <TH>Step 2</TH>
    <TH>Step 3</TH>
  </TR><TR><TD>95°C3 mins(1 cycle)</TD><TD>95°C, 55°C, 68°C10 secs, 20 secs, 90 secs(30 cycles)</TD><TD>68°C5 mins(1 cycle)</TD></TR></TBODY></TABLE><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">5.	Run gel electrophoresis (0.8% agarose) to check the size of the fragment.</P><DIV style="position:relative; padding: 0px;"><P style="color: black;">• 1 KB plus ladder • Lane 1-4: SFR2 colony PCR • Lane 5-8: tSFR2 colony PCR</P><P style="padding: 0px 50px 0px 50px; color: black; font-size: 14px">6.	Culture colony suspension in 4 ml of LB with carbenicillin. Incubate in 37°C shaker to miniprep the next day. </P><P style="text-align:right; padding: 10px 50px 0px 0px; color: black; font-size: 14px">(08/09/16)</P><H3 style="text-align: center"> Miniprep from bacterial culture</H3><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">1. Centrifuge 3 ml of bacterial culture (13,000 rpm) to get bacterial pellet and transfer to microcentrifuge tube. </P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">2. Re-suspend in 200 µl of resuspension buffer and vortex well. Add 200 µl of lyse buffer and mix gently by inverting the tube 5 times.</P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">3. Incubate the tube for 2 minutes. Add 300 µl of precipitation buffer and mix immediately until the solution is homogenous (do not vortex).</P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">4. Centrifuged for 5 minutes (13,000 rpm) at room temperature. </P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">5. Load the supernatant from step 5 to spin column. Centrifuge for 1 minute and discard flow through. </P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">6. Add 400 µl of wash buffer into spin column. Centrifuge for 1 minute and discard flow through. </P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">7. Centrifuge 1 more minute to remove remaining ethanol from wash buffer.</P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">8. Transfer the column to a new microcentrifuge tube. Elute DNA with 35 µl of elution buffer. Let it stand for 1 minute. </P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">9. Centrifuge for 1 minute at 13,000 rpm. Measure concentration with nano drop. </P><P style="text-align:left; padding: 0px 50px 0px 100px; color: black; font-size: 14px">• SFR2 recombinant plasmid concentration: 315 ng/ul• SFR2 recombinant plasmid concentration: 372 ng/ul</P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">10. Set up reaction mixture consists of 250 ng plasmid DNA and 5 ul of 5 µM primer, to send it to the sequencing facility. </P><P style="text-align:right; padding: 10px 50px 0px 0px; color: black; font-size: 14px">(08/10/16)</P><H3 style="text-align: center">Repeat PCR amplification for SFR2 and tSFR2</H3><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">1. Run the same experiment as before (refer to the protocol in 07/18/16), except that we tried to double the template DNA. Instead of using 2 ng, we used 4 ng of SFR2 or tSFR2 gBlock.</P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">2. Run the gel electrophoresis (0.8% agarose) at 150 V for 50 minutes.</P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">3. Extract the gel in lane G-M with Wizard SV Gel Clean-Up system from Promega.</P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">4. Measure the concentration with nanodrop.</P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">• SFR2 concentration: 40.6 ng/µl • tSFR2 concentration: 48.3 ng/µl</P><H3 style="text-align: center">Retransform desA plasmid into WT Cyanobacteria</H3><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">(Refer to the protocol on 08/05/16)</P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">Result: no colony grow even after 10 days of incubation and constant light illumination. </P><P style="text-align:right; padding: 10px 50px 0px 0px; color: black; font-size: 14px">(08/11/16)</P><H3 style="text-align: center">Gibson Assembly followed by Transformation into Competent Cells</H3><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">Refer to the protocol on 19/07/16</P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">Result: No colony grow on any of the BGII plates. </P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">Fill in the rest of august with chemostat construction.</P><H1 style="text-align: center; padding: 10px 0px 10px 50px">September</H1><P style="text-align:right; padding: 10px 50px 0px 0px; color: black; font-size: 14px">(09/15/16)</P><H3 style="text-align: center">Retransform desA plasmid into WT Cyanobacteria</H3><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">(Refer to the protocol on 08/05/16)</P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">Results: Green colonies are start to appear after 4 days of incubation with constant light illumination.</P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">1. In 50 ml eppendorf tube, take 50 ml of BGII fresh media and 100 µl of spectinomycin. Mix well.</P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">2. Aliquot 200 µl of the prepared media + antibiotic in each well of 96 well plate. </P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">3. Pick single colonies from the plates under the microscope and inoculate in 96 well plates. Parafilm well plate to avoid media evaporation. Incubate at 30°C incubator with constant light illumination.</P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">Repeat these steps everyday and tried to pick only a single colony to grow in the media.</P><P style="text-align:right; padding: 10px 50px 0px 0px; color: black; font-size: 14px">(09/21/16)</P><H3 style="text-align: center">Colony PCR of desA plasmid in Cyanobacteria</H3><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">1. Set up the following PCR mixture in PCR tubes, 23 reactions @ 50 ul and 1 control where we replace the cyanobacterial culture (template) with nuclease free H<SUB>2</SUB>O:</P><P style="text-align:left; padding: 0px 50px 0px 100px; color: black; font-size: 14px">• 33.5 ul of nuclease free H<SUB>2</SUB>O • 10 ul of HF buffer• 0.5 ul of DMSO• 1 ul of dNTPs• 1 ul of each forward and reverse desA primer • 2 ng of template DNA (cyanobacteria culture from 96 well plates)• 0.5 ul of phusion polymerase 
</P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">2. Run the PCR reaction in thermocycler with touch up program.</P><TABLE style="margin-left: auto; margin-right: auto; width: 100%;"><TBODY><TR><TH>Step 1</TH><TH>Step 2</TH><TH>Step 3</TH></TR><TR><TD rowspan="2" style="vertical-align: middle;">98°C45 secs(1 cycle)</TD><TD>98°C, 62.9°C, 72°C7 secs, 20 secs, 60 secs(7 cycles)</TD><TD rowspan="2" style="vertical-align: middle;">72°C7 mins(1 cycle)</TD></TR><TR><TD>95°C, 72°C, 72°C7 secs, 20 secs, 60 secs(30 cycles)</TD></TR></TBODY></TABLE><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">3. Run gel electrophoresis at 155 V for 30 minutes. Check the bands under UV light.</P><DIV style="position:relative; padding: 0px;"><P style="color: black;">• All lanes are colony PCR from different well plates, except for the last lane on the bottom right corner, which is negative control. Conclusion: we got our desA plasmidinto our cyanobacteria. Next step: grow them in batch culture and continuously in our chemostat before we test them in different temperature conditions.</P><H1 style="text-align: center; padding: 10px 0px 10px 50px">October</H1><P style="text-align:right; padding: 10px 50px 0px 0px; color: black; font-size: 14px">(10/03/16)</P><H3 style="text-align: center">PCR amplification of desA with pSB1C3 overhang primers (including the prefix and suffix of the iGEM standard plasmid)</H3>
Goal: Cloning of desA gene into pSB1C3 for parts Submission to iGEM Registry
<P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">1. Set up the following PCR mixture in PCR tubes, 3 reactions @ 50 ul and 1 control where we replace the template DNA with nuclease free H2O:</P><P style="text-align:left; padding: 0px 50px 0px 100px; color: black; font-size: 14px">• 33.5 ul of nuclease free H2O• 10 ul of HF buffer• 0.5 ul of DMSO• 1 ul of dNTPs• 1 ul of each forward and reverse primer• 2 ng of template DNA (desA recombinant spectinomycin plasmid)• 0.5 ul of phusion polymerase</P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">2. Run the PCR reaction in thermocycler with touch up program</P><TABLE style="margin-left: auto; margin-right: auto; width: 100%;"><TBODY><TR><TH>Step 1</TH><TH>Step 2</TH><TH>Step 3</TH></TR><TR><TD rowspan="2" style="vertical-align: middle;">98°C45 secs(1 cycle)</TD><TD>98°C, 58.4°C, 72°C7 secs, 20 secs, 50 secs(7 cycles)</TD><TD rowspan="2" style="vertical-align: middle;">72°C7 mins(1 cycle)</TD></TR><TR><TD>95°C, 72°C, 72°C7 secs, 20 secs, 50 secs(30 cycles)</TD></TR></TBODY></TABLE><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">3. Run gel electrophoresis at 155 V for 30 minutes. Check the bands under UV light.</P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">4. Cut the gel to extract it. We used Gel Clean Up kit from Promega to accomplish this step.</P><P style="text-align:right; padding: 10px 50px 0px 0px; color: black; font-size: 14px">(10/06/16)</P><H3 style="text-align: center">Restriction Digest of pSB1C3 with EcoR, PstI, DpnI</H3><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">1. Dilute lyophilized plasmid, pSB1C3 sent by iGEM, in 50 µl of nuclease free H<SUB>2</SUB>O.</P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">2. Restriction digest of pSb1C3 plasmid (2070 bp) with DpnI, PstI, EcoRI restriction enzyme from Fast Digest NEB, in a total reaction of 25 ul. Set up the following reaction in PCR tube:</P><P style="text-align:left; padding: 0px 50px 0px 100px; color: black; font-size: 14px">• Fast Digest buffer 2 µl
• EcoRI restriction enzyme 1 µl• PstI restriction enzyme 1 µl• KpnI restriction enzyme 1 µl• Plasmid 500 ng• Nuclease free H2O until 25 µl</P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">3. Since we only use 500 ng of DNA, we intended not to run the gel and rather proceed to PCR clean up with Wizard SV PCR clean up kit from Promega. Otherwise, the yield will be really low after gel extraction.</P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">4. Measure DNA concentration with nano drop. Plasmid pSB1C3 concentration: 16.5 ng/µl </P><P style="text-align:right; padding: 10px 50px 0px 0px; color: black; font-size: 14px">(10/11/16)</P><H3 style="text-align: center">Gibson Assembly of desA into pSB1C3</H3><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">1. Calculate the number of pmols needed for each fragment if we use 50 ng and 100 ng of the vectors.(50ng X 1000)/(2070bp X 650)= 0.037 pmols (100ng X 1000)/(2070bp X 650)= 0.074 pmols</P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">2. We used 3 fold of excess insert, so the number of pmols for plasmid will be 3 times the amount of insert we need (0.111 pmols for 50 ng of vectors and 0.222 pmols for 100 ng of vectors)Amount of desA needed:(x X 1000)/(1095 bp X 650) = 0.111 pmolsx = 79 ng(x X 1000)/(1095 bp X 650) = 0.222 pmolsx = 158 ng</P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">3. Gibson Assembly master mix from NEB is used with a total volume of 15 µl, so all we need to add is the plasmid backbone and the inserts according to the amount we calculated above in total reaction of 20 ul. We also run negative control where we just put the plasmid backbone without insert. So, there are 3 gibson reactions in total.</P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">4. Set up the reaction in 50°C for 30 minutes.</P><P style="text-align:right; padding: 10px 50px 0px 0px; color: black; font-size: 14px">(10/13/16)</P><H3 style="text-align: center">Transformation of Gibson Product into DH5α Competent Cells</H3><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">1. Thaw competent cells on ice for around 5-10 minutes.</P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">2. Use 2 different volumes of Gibson product to mix with the competent cells. Add 2.5 ul and 5 ul to 50 ul competent cells. </P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">3. Incubate on ice for 20 minutes. Heat shock cells at 42°C for 45 seconds.</P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">4. Put it back on ice for another 2 minutes and add 900 µl of LB into the tube.</P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">5. Incubate in the 37°C shaker for about an hour to recover the cells.</P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">6. Spread plate 50 µl and 200 µl of them onto LB plates with chloramphenicol antibiotic as the selective marker. </P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">7. Incubate at 37°C incubator overnight.</P><P style="text-align:right; padding: 10px 50px 0px 0px; color: black; font-size: 14px">(10/17/16)</P><H3 style="text-align: center">Culture single colonies of desA in pSB1C3 and Miniprep</H3><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">1. In culture tubes, take 5 ml of LB media and 5 µl of 1000X chloramphenicol antibiotic</P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">2. Incubate in 37°C degree shaker incubator.</P><P style="text-align:right; padding: 10px 50px 0px 0px; color: black; font-size: 14px">(10/18/16)</P><H3 style="text-align: center">Plasmid Purification (Miniprep) of desA</H3><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">1. Plasmid purification/miniprep (protocol refer to 07/21/16)</P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">2. Verify plasmid by restriction digest with XhoI and PstI. Virtual digest of the plasmid shown below:</P><DIV style="position:relative; padding: 0px;"><P style="color: black;">1 KB plus ladder Lane 1: desA in pSB1C3 plasmid backbone Lane 2: plasmid (pSB1C3) only</P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">3. Set up the following restriction reaction mixture in PCR tubes:</P><P style="text-align:left; padding: 0px 50px 0px 100px; color: black; font-size: 14px">• 2 µl Fast Digest Buffer• 1 µl of each restriction enzyme (KpnI and XbaI)• 1 µg of plasmid DNA • dH<SUB>2</SUB>O until a total volume of 20µl</P><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">4. Run gel electrophoresis with 0.8% agarose at 150 V for 30 minutes. </P><DIV style="position:relative; padding: 0px;"><P style="color: black;">• 1-8: digested plasmid DNA which miniprepped from  different bacterial culture</P><P style="color: black;"> Result: Lane 1 bands did not show up at the correct size, but all the the other lanes are positive (bands showed up at the same size as shown in the virtual digest).</P></DIV><P style="text-align:left; padding: 0px 50px 0px 50px; color: black; font-size: 14px">5. desA gene in pSB1C3 plasmid backbone is further diluted to make 25 ng/µl, aliquot in 96 well plates and ready to be shipped to iGEM Registry! </P></DIV><DIV class="footer"><P style="text-align:center;">msuigem@gmail.com</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>