Team:MSU-Michigan/Notebook
Michigan State University iGEM 2016
Home
Team
Team Members
Attributions
Project
Description
Design
Demonstration
Notebook
Parts
Human Practices
More
Hardware
Safety
Awards
Sponsors
Experimental Documentation
July
(07/18/16)
PCR Amplification (SFR2, tSFR2, desA) with Phusion High Fidelity Polymerase
1. Mix the following reagents for PCR amplification, each reaction of 50 µl and we did one negative control where we did not put the template DNA. We also use 2 different buffers because we figured that our gene sequence has a high GC%. Master-mix reaction mixture (4X):
• Nuclease free H2O 134 µl
• 5X HF/GC buffer 40 µl
• DMSO 2 µl
• dNTPs 4 µl
• Forward primer (10 µM) 5 µl
• Reverse primer (10 µM) 5 µl
• Template DNA 6 ng (2 ng/reaction)
• Phusion Polymerase 2 µl
2. Run the PCR reaction with touch up program, where we run 7 cycles with the temperature of the primer without the overhang and another 30 cycles with the temperature of the primer full sequence.
• SFR2 Touch-up PCR Program
Step 1
Step 2
Step 3
98°C
45 secs
(1 cycle)
98°C, 61.7°C, 72°C
7 secs, 20 secs, 80 secs
(7 cycles)
72°C
7 mins
(1 cycle)
95°C, 71.3°C, 72°C
7 secs, 20 secs, 80 secs
(30 cycles)
• tSFR2 Touch-up PCR Program
Step 1
Step 2
Step 3
98°C
45 secs
(1 cycle)
98°C, 61.7°C, 72°C
7 secs, 20 secs, 80 secs
(7 cycles)
72°C
7 mins
(1 cycle)
95°C, 72°C, 72°C
7 secs, 20 secs, 80 secs
(30 cycles)
• desA Touch-up PCR Program
Step 1
Step 2
Step 3
98°C
45 secs
(1 cycle)
98°C, 62.9°C, 72°C
7 secs, 20 secs, 60 secs
(7 cycles)
72°C
7 mins
(1 cycle)
95°C, 72°C, 72°C
7 secs, 20 secs, 60 secs
(30 cycles)
3. Run Gel Electrophoresis with 0.8% agarose at 150 V for 30 minutes. 1 KB plus ladder is used.
• A-C: SFR2 with HF buffer
• D    : SFR2 negative control
• E-G: SFR2 with GC buffer
• H      : SFR2 negative control
• I-K: tSFR2 with HF buffer
• L  : tSFR2 negative control
• M-O: tSFR2 with GC buffer
• P  : SFR2 negative control
• A-C: desA with HF buffer
• D-F: desA with GC buffer
4. Gel Extraction was done with Wizard SV Gel and PCR clean up kit by Promega. Then, DNA concentrations were measured with nano drop.
(07/19/16)
Restriction Digest of Riboswitch Plasmid with EcoRI and XbaI
1. Restriction digest for the riboswitch plasmid (3745 bp) with Fast Digest from NEB in total reaction of 20 ul. Set up the following reaction:
• Fast Digest buffer 2 µl
• EcoRI restriction enzyme 1 µl
• XbaI restriction enzyme 1 µl
• Plasmid 1 µg
• Nuclease free H2O until 20 µl
2. Run Gel Electrophoresis with 0.8% agarose at 150 V for 30 minutes. Gel extraction and concentration is measured with nano drop. Plasmid Concentration: 8.1 ng/µl
Gibson Assembly
1. Calculate the number of pmols needed for each fragment if we use 30 ng of the vectors.
(30ng X 1000)/(3745bp X 650)= 0.012 pmols
2. We used 3 fold of excess insert, so the number of pmols for plasmid will be 3 times the amount of insert we need (0.036 pmols).
Amount of SFR2 needed:
(x X 1000)/(1869bp X 650)= 0.036 pmols
x= 44.91 ng
Amount of tSFR2 needed:
(x X 1000)/(1788bp X 650)= 0.036 pmols
x= 41.84 ng
Amount of desA needed:
(x X 1000)/(1095bp X 650)= 0.036 pmols
x= 25.62 ng
3. Gibson Assembly master mix from NEB is used, so all we need to add is the plasmid backbone and the inserts according to the amount we calculated above in total reaction of 20 µl. We also run negative control where we just put the plasmid backbone without insert.
4. Set up the reaction in 50°C for 30 minutes
Transformation of the Recombinant Plasmid into DH5α Competent Cells
1. Thaw competent cells on ice for around 5-10 minutes.
2. Use 2 different volumes of Gibson product to mix with the competent cells. Add each tube of competent cells (50 µl) with 2.5 µl and 5 µl to 50 µl.
3. Incubate on ice for 20 minutes. Heat shock cells at 42°C for 45 seconds.
4. Put it back on ice for another 2 minutes and add 900 µl of LB into the tube. Incubate in the 37°C shaker for an hour.
5. Spread plate 50 µl and 200 µl of them onto LB plates with spectinomycin antibiotic as the selective marker.
6. Incubate at 37°C incubator overnight
Result: No colony grew in SFR2 and tSFR2 plates, but there were several colonies grew in the desA plate.
(07/20/16)
Prepare Bacterial Culture for Mini Prep
1. Add 50 ul of spectinomycin antiobiotic into 50 ml LB
2. In culture tubes, take 4 ml of LB + spectinomycin media. Pick isolated colonies formed on the desA transformation plate and mix well with the media (pipette up and down).
3. Incubate overnight at 37°C shaker.
(07/21/16)
Plasmid Purification (Miniprep)
1. Spin down overnight culture in microcentrifuge tubes and discard the supernatant.
2. Re-suspend each pellet in 215 µl buffer P1 (cell resuspension solution).
3. Add 215 µl buffer P2 (cell lysis solution) and invert the tube 8 times to mix.
4. Add 330µl buffer P3 and invert 8 times to mix.
5. Spin down at 16,000 g for 5 minutes.
6. Apply 730 µl of supernatant to DNA column.
7. Spin down at 16,000 g for 1 minute and discard flow through.
8. Wash column with 700 µl wash buffer and spin down at 16,000 g for another minute. Discard flow through. Wash one more time with 500 µl wash buffer and discard flow through.
9. Dry spin column for 2 minutes to evaporate all the ethanol.
10.Elute into new microcentrifuge tube with 37 µl nuclease free H2O.
11. Measure concentration with nano drop.
• SFR2 recombinant plasmid: 330.3 ng/ul
• tSFR2 recombinant plasmid: 368.8 ng/ul
• desA recombinant plasmid: 453.3 ng/ul
(07/22/16)
Recombinant Plasmid Verification with Restriction Digest
1. Run virtual digest of desA recombinant plasmid with KpnI restriction enzyme and SFR2, tSFR2 recombinant plasmid with SalI.
1 KB plus ladder
Lane 1: desA recombinant plasmid
Lane 2: plasmid only
Lane 3: SFR2 recombinant plasmid
Lane 4: tSFR2 recombinant plasmid
Lane 5: plasmid only
2. Make the restriction reaction mixture in a total volume of 20 µl. Set up the following reaction:
• 1 ug plasmid DNA
• 2 ul cutsmart buffer
• 1 ul restriction enzyme
• Nuclease free H2O until 20 ul
3. Run gel electrophoresis in 0.8% agarose at 150 V for 30 minutes.
We got positive result, where the total of 3 bands showed up at the correct size
for desA recombinant plasmid and 2 bands for the control (plasmid only).
We got negative results, it did not seems like the one in virtual digest.
All the bands showed up almost at the same position as the negative control (plasmid only).
(07/28/16)
SFR2 & tSFR2 Plasmid Verification with Restriction Digest
(2nd attempt)
1. Run virtual digest of SFR2 and tSFR2 recombinant plasmid with XhoI and KpnI.
1 KB plus ladder
Lane 1: SFR2 recombinant plasmid
Lane 2: tSFR2 recombinant plasmid
Lane 3: plasmid only
2. Make the restriction reaction mixture in a total volume of 20 µl. Set up the following reaction:
• 1 ug plasmid DNA
• 2 µl cutsmart buffer
• 1 µl XhoI
• 1 µl KpnI
• Nuclease free H2O until 20 ul
3. Run gel electrophoresis in 0.8% agarose at 150 V for 30 minutes.
We got negative results. All the bands still showed up exactly like the negative control
(plasmid only), where the top band showed up around 2 kb instead of 4 kb.
(07/29/16)
Send desA recombinant plasmid to sequencing facility
1. Dilute recombinant plasmid to 50 ng/µl.
2. In 1.5 ml centrifuge tube, take 5 µl of diluted recombinant plasmid, total of 250 ng, and 5 µl of forward or reverse sequencing primer, total of 25 µM.
3. Plasmid is ready to be sequenced!
August
(08/03/16)
Analyze the sequencing result of SFR2, tSFR2, and desA
1. Open Benchling software and clone the SFR2, tSFR2 and desA to the riboswitch plasmid to prepare for sequence alignment.
2. Open the sequencing file (.ab1) in Snapgene Viewer and copy the whole sequence.
3. In Benchling, open the recombinant plasmid sequence, choose alignment button on the right side of the screen and paste the sequence we copied from Snapgene viewer into the sequence box.
4. Analyze how the gene sequence align with the expected recombinant plasmid.
5. Our desA plasmid sequence looks really well-aligned with the expected recombinant plasmid, but we lost a big chunk of sequence in our SFR2 and tSFR2 gene sequence.
Next step: While we re-clone our SFR2 and tSFR2 gene, we were going to transform our desA gene into wild type cyanobacteria and wait a couple days to let them grow in BG11 plate.
(08/04/16)
Repeat PCR for SFR2 and tSFR2
1. Set up the PCR reaction (refer to 08.17.16)
2. Run the gel and extract the gel.
3. Measure concentration with nano drop. SFR2 concentration: 8.1 ng/µl.
Clone SFR2 and tSFR2 PCR product into pJET plasmid
Goal: Since the sequencing result of our SFR2 and tSFR2 gene did not come out good, so we decided to clone them to pJET plasmid to see if we got the actual sequence of our genes , after PCR-ing with the Gibson overhang (~20 bp sequence from the riboswitch plasmid).
1. Set up the ligation reaction on ice:
• 5 ul 2X Reaction buffer
• 90 ng PCR product
• 0.5 µl pJET cloning vector (50 ng/µl)
• 0.5 µl T4 DNA Ligase
• Nuclease free H2O until 10 µl
2. Incubate the ligation mixture at room temperature for 10 minutes. Use ligation product directly for transformation.
3. Thaw the competent cells on ice.
4. Add <2.5 µl ligation reaction to 25 µl cells. Mix gently by flicking the tubes and incubate on ice for 30 minutes.
5. Heat shock at 42°C for 30 seconds. Place back on ice for 2 minutes.
6. Add 250 µl room temperature SOC medium. Shake tubes for 60 minutes at 37°C.
7. Plate 50 µl of the transformation reactions on LB agar plates containing carbenicillin using sterile glass beads. Incubate at 37°C incubator overnight.
(08/05/16)
Transformation of desA into WT Cyanobacteria
1. Take 2 ml of cyanobacteria culture that has been inoculated and grown to an optical density (OD750) of 0.5 to 0.8 under continuous illumination and permanent agitation.
2. Centrifuged cell culture and cell pellet was re-suspended in 500 µl BG11 medium.
3. 100 ng of plasmid DNA was added and cells were gently mix by tapping. Incubated in darkness at 30°C for 4 hours.
4. While waiting for incubation, prepare BG11 plate with appropriate antibiotic (we used spectinomycin as the marker).
5. Following incubation, plate 50 µl and 200 µl of cells onto the plate with spread plate technique.
6. Incubate at 30°C with constant illumination for at least 5 days.
Results: No colony grow even after 5 days.
(08/08/16)
Colony PCR after transformation on 08.04.16
1. Pick 4 colonies from each plate (SFR2 & tSFR2) and mix it well with 20 µl of nuclease free H2O.
2. Set up PCR reaction mixture:
• 5 µl 2X Hot-Start Taq mix
• 1 µl pJET 1.2 Forward (5 µM)
• 1 µl pJET 1.2 Reverse (5 µM)
• 3 µl colony suspension
3. Mix and briefly centrifuge.
4. Thermocycling condition for colony PCR:
 
   
Step 1
   
Step 2
   
Step 3
 
95°C
3 mins
(1 cycle)
95°C, 55°C, 68°C
10 secs, 20 secs, 90 secs
(30 cycles)
68°C
5 mins
(1 cycle)
5. Run gel electrophoresis (0.8% agarose) to check the size of the fragment.
• 1 KB plus ladder
• Lane 1-4: SFR2 colony PCR
• Lane 5-8: tSFR2 colony PCR
6. Culture colony suspension in 4 ml of LB with carbenicillin. Incubate in 37°C shaker to miniprep the next day.
(08/09/16)
Miniprep from bacterial culture
1. Centrifuge 3 ml of bacterial culture (13,000 rpm) to get bacterial pellet and transfer to microcentrifuge tube.
2. Re-suspend in 200 µl of resuspension buffer and vortex well. Add 200 µl of lyse buffer and mix gently by inverting the tube 5 times.
3. Incubate the tube for 2 minutes. Add 300 µl of precipitation buffer and mix immediately until the solution is homogenous (do not vortex).
4. Centrifuged for 5 minutes (13,000 rpm) at room temperature.
5. Load the supernatant from step 5 to spin column. Centrifuge for 1 minute and discard flow through.
6. Add 400 µl of wash buffer into spin column. Centrifuge for 1 minute and discard flow through.
7. Centrifuge 1 more minute to remove remaining ethanol from wash buffer.
8. Transfer the column to a new microcentrifuge tube. Elute DNA with 35 µl of elution buffer. Let it stand for 1 minute.
9. Centrifuge for 1 minute at 13,000 rpm. Measure concentration with nano drop.
• SFR2 recombinant plasmid concentration: 315 ng/ul
• SFR2 recombinant plasmid concentration: 372 ng/ul
10. Set up reaction mixture consists of 250 ng plasmid DNA and 5 ul of 5 µM primer, to send it to the sequencing facility.
(08/10/16)
Repeat PCR amplification for SFR2 and tSFR2
1. Run the same experiment as before (refer to the protocol in 07/18/16), except that we tried to double the template DNA. Instead of using 2 ng, we used 4 ng of SFR2 or tSFR2 gBlock.
2. Run the gel electrophoresis (0.8% agarose) at 150 V for 50 minutes.
3. Extract the gel in lane G-M with Wizard SV Gel Clean-Up system from Promega.
4. Measure the concentration with nanodrop.
• SFR2 concentration: 40.6 ng/µl
• tSFR2 concentration: 48.3 ng/µl
Retransform desA plasmid into WT Cyanobacteria
(Refer to the protocol on 08/05/16)
Result: no colony grow even after 10 days of incubation and constant light illumination.
(08/11/16)
Gibson Assembly followed by Transformation into Competent Cells
Refer to the protocol on 19/07/16
Result: No colony grow on any of the BGII plates.
Fill in the rest of august with chemostat construction.
September
(09/15/16)
Retransform desA plasmid into WT Cyanobacteria
(Refer to the protocol on 08/05/16)
Results: Green colonies are start to appear after 4 days of incubation with constant light illumination.
1. In 50 ml eppendorf tube, take 50 ml of BGII fresh media and 100 µl of spectinomycin. Mix well.
2. Aliquot 200 µl of the prepared media + antibiotic in each well of 96 well plate.
3. Pick single colonies from the plates under the microscope and inoculate in 96 well plates. Parafilm well plate to avoid media evaporation. Incubate at 30°C incubator with constant light illumination.
Repeat these steps everyday and tried to pick only a single colony to grow in the media.
(09/21/16)
Colony PCR of desA plasmid in Cyanobacteria
1. Set up the following PCR mixture in PCR tubes, 23 reactions @ 50 ul and 1 control where we replace the cyanobacterial culture (template) with nuclease free H2O:
• 33.5 ul of nuclease free H2O
• 10 ul of HF buffer
• 0.5 ul of DMSO
• 1 ul of dNTPs
• 1 ul of each forward and reverse desA primer
• 2 ng of template DNA (cyanobacteria culture from 96 well plates)
• 0.5 ul of phusion polymerase
2. Run the PCR reaction in thermocycler with touch up program.
Step 1
Step 2
Step 3
98°C
45 secs
(1 cycle)
98°C, 62.9°C, 72°C
7 secs, 20 secs, 60 secs
(7 cycles)
72°C
7 mins
(1 cycle)
95°C, 72°C, 72°C
7 secs, 20 secs, 60 secs
(30 cycles)
3. Run gel electrophoresis at 155 V for 30 minutes. Check the bands under UV light.
• All lanes are colony PCR from different well plates, except for the last lane
on the bottom right corner, which is negative control. Conclusion: we got our desA plasmid
into our cyanobacteria. Next step: grow them in batch culture and continuously in our chemostat
before we test them in different temperature conditions.
October
(10/03/16)
PCR amplification of desA with pSB1C3 overhang primers
(including the prefix and suffix of the iGEM standard plasmid)
Goal: Cloning of desA gene into pSB1C3 for parts Submission to iGEM Registry
1. Set up the following PCR mixture in PCR tubes, 3 reactions @ 50 ul and 1 control where we replace the template DNA with nuclease free H2O:
• 33.5 ul of nuclease free H2O
• 10 ul of HF buffer
• 0.5 ul of DMSO
• 1 ul of dNTPs
• 1 ul of each forward and reverse primer
• 2 ng of template DNA (desA recombinant spectinomycin plasmid)
• 0.5 ul of phusion polymerase
2. Run the PCR reaction in thermocycler with touch up program
Step 1
Step 2
Step 3
98°C
45 secs
(1 cycle)
98°C, 58.4°C, 72°C
7 secs, 20 secs, 50 secs
(7 cycles)
72°C
7 mins
(1 cycle)
95°C, 72°C, 72°C
7 secs, 20 secs, 50 secs
(30 cycles)
3. Run gel electrophoresis at 155 V for 30 minutes. Check the bands under UV light.
4. Cut the gel to extract it. We used Gel Clean Up kit from Promega to accomplish this step.
(10/06/16)
Restriction Digest of pSB1C3 with EcoR, PstI, DpnI
1. Dilute lyophilized plasmid, pSB1C3 sent by iGEM, in 50 µl of nuclease free H2O.
2. Restriction digest of pSb1C3 plasmid (2070 bp) with DpnI, PstI, EcoRI restriction enzyme from Fast Digest NEB, in a total reaction of 25 ul. Set up the following reaction in PCR tube:
• Fast Digest buffer 2 µl
• EcoRI restriction enzyme 1 µl
• PstI restriction enzyme 1 µl
• KpnI restriction enzyme 1 µl
• Plasmid 500 ng
• Nuclease free H2O until 25 µl
3. Since we only use 500 ng of DNA, we intended not to run the gel and rather proceed to PCR clean up with Wizard SV PCR clean up kit from Promega. Otherwise, the yield will be really low after gel extraction.
4. Measure DNA concentration with nano drop.
Plasmid pSB1C3 concentration: 16.5 ng/µl
(10/11/16)
Gibson Assembly of desA into pSB1C3
1. Calculate the number of pmols needed for each fragment if we use 50 ng and 100 ng of the vectors.
(50ng X 1000)/(2070bp X 650)= 0.037 pmols
(100ng X 1000)/(2070bp X 650)= 0.074 pmols
2. We used 3 fold of excess insert, so the number of pmols for plasmid will be 3 times the amount of insert we need (0.111 pmols for 50 ng of vectors and 0.222 pmols for 100 ng of vectors)
Amount of desA needed:
(x X 1000)/(1095 bp X 650) = 0.111 pmols
x = 79 ng
(x X 1000)/(1095 bp X 650) = 0.222 pmols
x = 158 ng
3. Gibson Assembly master mix from NEB is used with a total volume of 15 µl, so all we need to add is the plasmid backbone and the inserts according to the amount we calculated above in total reaction of 20 ul. We also run negative control where we just put the plasmid backbone without insert. So, there are 3 gibson reactions in total.
4. Set up the reaction in 50°C for 30 minutes.
(10/13/16)
Transformation of Gibson Product into DH5α Competent Cells
1. Thaw competent cells on ice for around 5-10 minutes.
2. Use 2 different volumes of Gibson product to mix with the competent cells. Add 2.5 ul and 5 ul to 50 ul competent cells.
3. Incubate on ice for 20 minutes. Heat shock cells at 42°C for 45 seconds.
4. Put it back on ice for another 2 minutes and add 900 µl of LB into the tube.
5. Incubate in the 37°C shaker for about an hour to recover the cells.
6. Spread plate 50 µl and 200 µl of them onto LB plates with chloramphenicol antibiotic as the selective marker.
7. Incubate at 37°C incubator overnight.
(10/17/16)
Culture single colonies of desA in pSB1C3 and Miniprep
1. In culture tubes, take 5 ml of LB media and 5 µl of 1000X chloramphenicol antibiotic
2. Incubate in 37°C degree shaker incubator.
(10/18/16)
Plasmid Purification (Miniprep) of desA
1. Plasmid purification/miniprep (protocol refer to 07/21/16)
2. Verify plasmid by restriction digest with XhoI and PstI. Virtual digest of the plasmid shown below:
1 KB plus ladder
Lane 1: desA in pSB1C3 plasmid backbone
Lane 2: plasmid (pSB1C3) only
3. Set up the following restriction reaction mixture in PCR tubes:
• 2 µl Fast Digest Buffer
• 1 µl of each restriction enzyme (KpnI and XbaI)
• 1 µg of plasmid DNA
• dH2O until a total volume of 20µl
4. Run gel electrophoresis with 0.8% agarose at 150 V for 30 minutes.
• 1-8: digested plasmid DNA which miniprepped from
different bacterial culture
Result: Lane 1 bands did not show up at the correct size, but all the the other lanes are positive
(bands showed up at the same size as shown in the virtual digest).
5. desA gene in pSB1C3 plasmid backbone is further diluted to make 25 ng/µl, aliquot in 96 well plates and ready to be shipped to iGEM Registry!
msuigem@gmail.com
