Team:JMU Wuerzburg/Experiments
Home
Team
Team Members
Collaborations
Project
Description
Design
Introduction to Malaria
Experiments
Notebook
InterLab
Results
Attributions
Sponsors
Parts
Safety
Human Practices
Workshops
Education
Establishing an iGEM-Team
Collaborations for our Laboratory Work
Judging Form
Dry Lab
The general Plasmodium primer-probe pair was created as described below.
To distinguish suitable sequences for primer/probe design we created a                  FASTA file with all available mitochondrial nucleotide sequences of                  Plasmodium falciparum, P. malariae, P. knowlesi, P. ovale and P. vivax from the NCBI database 5 .                  These 2080 Sequences were aligned with ClustalO 1 . For this step we used ClustalO,                  running on a computing cluster and the following bash command:
nohup clustalo --outfmt=msf -o outputname.msf -i inputname.fasta & > nohup.log
Afterwards, we created a frequency matrix from the resulting MSF file. The frequency matrix was created with Emboss Prophecy 2 . We used the following parameters:
EMBOSS data file: EDNAFULL
Threshold reporting percentage: 75
Gap opening penalty: 3.0
Gap extension penalty: 0.3
Subsequently, we calculated a new matrix based on the frequency matrix. Table 1 shows how we calculated this matrix.
Table 1: We calculated a matrix                      with eight columns and as many rows as positions in the frequency matrix. In the columns,                      B - F are the nucleotides Adenin (A), Cytosin (C), Guanin (G) and Thymin (T).                      The first row and column are only shown for a better orientation in the matrix.
In this matrix, we searched for positions with a high value in column I                  (near or equal 1) and a nucleotide summation as high as possible.                  The sequence of Plasmodium is often Thymin/Adenin rich,                  so we had also to look for a high Guanin/Cytosin-content.                  If a suitable sequence is long enough for a primer and probe design,                  we designed primers and a probe with OligoArchitectTM 3 .                  Afterwards, we calculated the secondary structure of the primers and the probe with RNAstructure 4 .                  Therefore we used the following parameters:
Temperature [K]: 310.15
Maximum Loop Size: 30
Maximum % Energy Difference (MFE, MEA): 10
Maximum Number of Structures (MFE, MEA): 20
Window Size (MFE, MEA): 3
Gamma (MEA): 1
Iterations (Pseudoknot Prediction): 1
Minimum Helix Length (Pseudoknot Prediction): 3
If the calculated secondary structure is moderate or low, we started testing the primers and the probe in the wet lab.
Our primer/probe pair for P. falciparum was designed as written below.                 We created five alignments with ClustalO 1 .                  One alignment is for P. falciparum with 1012                  mitochondrial sequences of P. falciparum from NCBI                  and four alignments are for P. malariae, P. knowlesi,                      P. ovale and P. vivax 5 .                  These four alignments were calculated with mitochondrial nucleotide                  sequences from NCBI of each Plasmodium species 5 .                  The alignments were calculated with ClustalO with the following bash command:
nohup clustalo --outfmt=msf -o outputname.msf -i inputname.fasta & > nohup.log
We used the resulting consensus sequences and aligned these                  five sequences with ClustalO.                  Subsequently, we searched in the resulting MSF                  file after differences between the consensus sequence of P. falciparum and the other four consensus sequences.                  Afterwards, we created a frequency matrix as described                  in the part for Plasmodium in general and checked if our regions                 of interest are highly conserved as described in the part of Plasmodium in general.                  The primer and probe design were also carried out as described                  in the upper part for Plasmodium.
Furthermore, we aligned over 40 000 nucleotide sequences from                  NCBI 5 , DDBJ 6 and Embl 7 of Norovirus with ClustalO.                  We searched in the resulting alignment for highly conserved                  regions, as described in the part for Plasmodium in general                  and designed a primer/probe pair for Norovirus.
List of References
1 Sievers F, Wilm A, Dineen DG, Gibson TJ, Karplus K, Li W, Lopez R, McWilliam H, Remmert M, Söding J, Thompson JD, Higgins DG (2011). Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega. Molecular Systems Biology 7:539 doi:10.1038/msb.2011.75
2 Rice P., Longden I. and Bleasby A. EMBOSS: The European Molecular Biology Open Software Suite. Trends in Genetics. 2000 16(6):276-277
3 https://www.sigmaaldrich.com/technical-documents/articles/biology/oligoarchitect-online.html
4 J.S. Reuter and D.H. Mathews. "RNAstructure: software for RNA secondary structure prediction and analysis." BMC Bioinformatics, 11:129. (2010).
5 https://www.ncbi.nlm.nih.gov/
6 https://www.ddbj.nig.ac.jp/index-e.html
7 https://www.ebi.ac.uk/ena
Hardware
Having a qPCR test system is fine. But it is only useful, if you have the specific hardware device for it. Our cycler is constructed very simply: a drawer for the test itself has two clamps to fixate the tube, so it won’t move. Underneath the drawer-slot are the operators for our test system. These operators are heat-elements, thermo sensors and relapsing magnets. To get the blood from one chamber of the tube to the next, we use a pump-lever. This pump-lever is montaged above the test-tube and runs by a small stepper-motor. The lever drives on a gear shaft that rotates by the stepper motor. At the end of this lever there is a roller, which pulls the liquid forward. At the end of the device there are LED-diodes and photocells wired to an ARDUINO computer. The diodes radiate light with 495 nm wavelength at the fluorescent probe. The photocells measure the emitted light from the probe and give the data to the ARDUINO computer. This data makes it possible to derive the presence of Plasmodium types.
Scheme of our hardware device.
Oligonucleotides
We designed the following oligonucleotieds as either Primer, Probe or our BioBrick:
Protocols
We used the following kits and protocols to conduct our experiments:
(RT)qPCR Probe-Kis
Biozym Probe qPCR Kit (Biozym) 1
One Step RT qPCR Probe ROX L Kit (highQu) 2
Quanti Nova Probe PCR Kit (Quiagen) 3
LightCycler ® 480 RNA Master Hydrolysis Probes (Roche) 4
(RT)qPCR Sybr Green Kits
LightCycler® FastStart DNA Master SYBR Green I (Roche) 5
LightCycler® EvoScript RNA SYBR® Green I Master (Roche) 6
LightCycler® 480 SYBR Green I Master (Roche) 7
DNA/Plasmid Extaction Kit
innuPREP Plasmid Mini Kit (Analytic Jena) 8
innuPREP Gel Extraction Kit (Analytic Jena) 9
GenElute Plasmid DNA Midiprep Kit (Analytic Jena) 10
E.Z.N.A. Total RNA Kit I (Omega) 11
Agarose gel
1,8 % (0,9 g Agarose, 50 ml TAE buffer), 2 µl Ethidiumbromid
Voltage: 100 V
NEB® PCR Cloning Kit (NEB) 12
cDNA Synthese with M-MuLV RT Quick Protocol (NEB) 13
Template RNA 4 µl, Primer 1 µl, 10x M-MulV buffer 2 µl, M-MulVRT 1 µl, 10 mM dNTP 1 µl, H2O 11 µl                 1h incubation at 42 °C
Digest with EcoRI-HF and PSTI-HF 14 15
DNA 1 µg, CutSmart 10 x buffer 10 µl, EcoRI-HF 2 µl, PSTI-HF 2 µl, H2O to 100 µl
Ligation with T4 DNA ligase 16
Agarplates with chloramphenicol resistance
10 g tryptone, 5 g yeast extract, 5 g NaCl, 10 g Agar, H2O to 1 l; autoclave; 10 mg/ml Chloramphenicol
Transformation (Changed from NEB) 17
Thaw chemical competent cells on ice
Put 15 ng of the ligation mixture to 50 µl of competent cells
Place the mixture 10 minutes on ice.
Heat shock at 42°C for 45 seconds
Add 1 ml LB-Medium
Place tube at 37°C for 60 minutes and shake 300 rpm.
Centrifuge 2 min at 11000 rpm
Take 850 µl supernatant und resuspend the 200 µl.
Spread it onto the plates.
Incubate overnight at 37°C
Sequencing 18
1 µl DNA, 1 µl BigDye 1.1, 1 µl Primer (10 µM), 2 µl H2O
Thermocycler program:      1. denaturation  96°C  1min     2. denaturation  96°C  20 s     3. annealing            50°C 15 s     4. polymerisation  60°C  4 min     5. cooling   4°C ∞
Seed cells
1. Aspirate medium from cell-culture (two 75 cm2)
2. Trypsinize cells with 2 ml ATV
3. Check cell-detachment under microscope
4. Add 2 ml FCS-medium to stop trypsinization
5. Resuspend cells by pipetting up and down
6. Transfer 3 ml to fresh falcon tube
7. Add 9 ml medium to the flask
8. Transfer 10 µl to the Neubauer chamber,
   Count cells in a four to four square (Count 16 mini squares);
   we counted 182 cells, so we had 1,82 * 10 ^6 cells per ml;
   seed 1,5 * 10^5 cells per well in a 6-well plate, 1,5 * 10^5 / 1,82 * 10^6 = 82,4 µl
9. Dilute cells;
   We had 6 ml in a falcon, so we add 6 ml medium
10. Prepare 2* 917,5 µl = 1,835 ml of cell-culture medium per well in a 6-well plate
11. Add 165 µl cell suspension to each well
12. Shake 6-well plate (3 pieces)
Transfection with PEI
For each well of a 6-well plate
1. 0,3 - 0,5 µg GFP
2. 2,5 - 3,5 µg PUC
3. 100µl DMEM
4. Mix
5. 6 - 9 µl PEI
5. Mix
6. Incubate 15 min at room temperature (RT)
7. Drop briefly at the cells
List of References
1 https://www.biozym.com/desktopmodules/webshop/downloads/331455_170220_Biozym-Probe-qPCR-Kit.pdf
2 https://www.highqu.com/1step-rt-qpcr-probe-rox-l-kit.html
3 https://www.qiagen.com/us/resources/resourcedetail?id=5167d782-9fef-4202-bc79-95f358be7d8c&lang=en
4 https://lifescience.roche.com/en_de/products/lightcycler14301-480-rna-master-hydrolysis-probes.html#documents
5 https://lifescience.roche.com/en_de/products/lightcycler-faststart-dna-master-sybr-green-i.html#documents
6 https://www.lifescience.roche.com/en_de/products/lightcycler-evoscript-rna-sybr-green-i-master.html#documents
7 https://biochimie.umontreal.ca/wp-content/uploads/sites/37/2015/11/LC480SYBRMasterguide.pdf
8 https://www.analytik-jena.de/fileadmin/content/pdf_life_science/Manual/Manual_innuPREP_Plasmid_Mini_Kit_2.0.pdf
9 https://www.analytik-jena.de/fileadmin/content/pdf_life_science/Manual/Manual_innuPREP_Gel_Extraction_Kit.pdf
10 https://www.sigmaaldrich.com/catalog/product/sigma/pld35?lang=de&region=DE
11 http://omegabiotek.com/store/wp-content/uploads/2013/04/R6834-Total-RNA-Mini-Kit-I-Combo-Online.pdf
12 https://international.neb.com/-/media/catalog/datacards-or-manuals/manuale1202.pdf
13 https://international.neb.com/protocols/2016/04/26/first-strand-cdna-synthesis-quick-protocol-neb-m0253
14 https://international.neb.com/protocols/2012/12/07/optimizing-restriction-endonuclease-reactions
15 https://nebcloner.neb.com/#!/redigest
16 https://international.neb.com/Protocols/0001/01/01/dna-ligation-with-t4-dna-ligase-m0202
17 https://international.neb.com/protocols/2012/05/21/transformation-protocol
18 http://www3.appliedbiosystems.com/cms/groups/mcb_support/documents/generaldocuments/cms_041330.pdf
