<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_JMU_Wuerzburg_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:JMU Wuerzburg/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>Home</P><LI class="menu"><P>Team</P></LI><LI><A href="https://2018.igem.org/Team:JMU_Wuerzburg/Team">Team Members</A></LI><LI><A href="https://2018.igem.org/Team:JMU_Wuerzburg/Collaborations">Collaborations</A></LI><LI class="menu"><P>Project</P></LI><LI><A href="https://2018.igem.org/Team:JMU_Wuerzburg/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:JMU_Wuerzburg/Applied_Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:JMU_Wuerzburg/Introduction_Malaria">Introduction to Malaria</A></LI><LI><A href="https://2018.igem.org/Team:JMU_Wuerzburg/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:JMU_Wuerzburg/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:JMU_Wuerzburg/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:JMU_Wuerzburg/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:JMU_Wuerzburg/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:JMU_Wuerzburg/Sponsors">Sponsors</A></LI><LI class="menu"><P>Parts</P></LI><LI class="menu"><P>Safety</P></LI><LI class="menu"><P>Human Practices</P></LI><LI><A href="https://2018.igem.org/Team:JMU_Wuerzburg/Human_Practices#workshops">Workshops</A></LI><LI><A href="https://2018.igem.org/Team:JMU_Wuerzburg/Human_Practices#education">Education</A></LI><LI><A href="https://2018.igem.org/Team:JMU_Wuerzburg/Human_Practices#establishing">Establishing an iGEM-Team</A></LI><LI><A href="https://2018.igem.org/Team:JMU_Wuerzburg/Human_Practices#collabs">Collaborations for our Laboratory Work</A></LI><LI class="menu"><P>Judging Form</P></LI><DIV class="content"><H3>Dry Lab</H3><DIV>
                The general<I> Plasmodium </I>primer-probe pair was created as described below.
                
                To distinguish suitable sequences for primer/probe design we created a 
                FASTA file with all available mitochondrial nucleotide sequences of 
                Plasmodium falciparum, P. malariae, P. knowlesi, P. ovale and P. vivax from the NCBI database<SUP><A href="#drylab_refs">5</A></SUP>. 
                These 2080 Sequences were aligned with ClustalO<SUP><A href="#drylab_refs">1</A></SUP>. For this step we used ClustalO, 
                running on a computing cluster and the following bash command:
                <CODE>nohup clustalo --outfmt=msf -o outputname.msf -i inputname.fasta &amp; &gt; nohup.log</CODE>
                Afterwards, we created a frequency matrix from the resulting MSF file. The frequency matrix was created with Emboss Prophecy 
                <SUP><A href="#drylab_refs">2</A></SUP>. We used the following parameters:
                <CODE>EMBOSS data file: EDNAFULL</CODE><CODE>Threshold reporting percentage: 75</CODE><CODE>Gap opening penalty: 3.0</CODE><CODE>Gap extension penalty: 0.3</CODE>
                Subsequently, we calculated a new matrix based on the frequency matrix. Table 1 shows how we calculated this matrix.

                <DIV style="border-style: solid; border-color: #CCC; border-width: 1px; margin: 2em auto; width: 60%;"><SPAN style="text-decoration: underline;">Table 1:  </SPAN> We calculated a matrix 
                    with eight columns and as many rows as positions in the frequency matrix. In the columns, 
                    B - F are the nucleotides Adenin (A), Cytosin (C), Guanin (G) and Thymin (T). 
                    The first row and column are only shown for a better orientation in the matrix.
                </DIV>

                In this matrix, we searched for positions with a high value in column I 
                (near or equal 1) and a nucleotide summation as high as possible. 
                The sequence of Plasmodium is often Thymin/Adenin rich, 
                so we had also to look for a high Guanin/Cytosin-content. 
                If a suitable sequence is long enough for a primer and probe design, 
                we designed primers and a probe with OligoArchitectTM<SUP><A href="#drylab_refs">3</A></SUP>. 
                Afterwards, we calculated the secondary structure of the primers and the probe with RNAstructure<SUP><A href="#drylab_refs">4</A></SUP>. 
                Therefore we used the following parameters:

                <CODE>Temperature [K]: 310.15</CODE><CODE>Maximum Loop Size: 30</CODE><CODE>Maximum % Energy Difference (MFE, MEA): 10</CODE><CODE>Maximum Number of Structures (MFE, MEA): 20</CODE><CODE>Window Size (MFE, MEA): 3</CODE><CODE>Gamma (MEA): 1</CODE><CODE>Iterations (Pseudoknot Prediction): 1</CODE><CODE>Minimum Helix Length (Pseudoknot Prediction): 3</CODE>

                If the calculated secondary structure is moderate or low, we started testing the primers and the probe in the wet lab.

                

                Our primer/probe pair for <I>P. falciparum</I> was designed as written below.
                We created five alignments with ClustalO<SUP><A href="#drylab_refs">1</A></SUP>. 
                One alignment is for P. falciparum with 1012 
                mitochondrial sequences of <I>P. falciparum</I> from NCBI 
                and four alignments are for <I> P. malariae, P. knowlesi, 
                    P. ovale</I> and <I>P. vivax</I><SUP><A href="#drylab_refs">5</A></SUP>. 
                These four alignments were calculated with mitochondrial nucleotide 
                sequences from NCBI of each <I>Plasmodium</I> species<SUP><A href="#drylab_refs">5</A></SUP>. 
                The alignments were calculated with ClustalO with the following bash command:

                <CODE> nohup clustalo --outfmt=msf -o outputname.msf -i inputname.fasta &amp; &gt; nohup.log</CODE>

                We used the resulting consensus sequences and aligned these 
                five sequences with ClustalO. 
                Subsequently, we searched in the resulting MSF 
                file after differences between the consensus sequence of 
                <I>P. falciparum</I> and the other four consensus sequences. 
                Afterwards, we created a frequency matrix as described 
                in the part for <I>Plasmodium</I> in general and checked if our regions
                of interest are highly conserved as described in the part of <I>Plasmodium</I> in general. 
                The primer and probe design were also carried out as described 
                in the upper part for <I>Plasmodium</I>.

                
                
                Furthermore, we aligned over 40 000 nucleotide sequences from 
                NCBI<SUP><A href="#drylab_refs">5</A></SUP>, DDBJ<SUP><A href="#drylab_refs">6</A></SUP>
                and Embl<SUP><A href="#drylab_refs">7</A></SUP> of Norovirus with ClustalO. 
                We searched in the resulting alignment for highly conserved 
                regions, as described in the part for Plasmodium in general 
                and designed a primer/probe pair for Norovirus.
                
                <H5 id="drylab_refs">List of References</H5><SUP>1</SUP> Sievers F, Wilm A, Dineen DG, Gibson TJ, Karplus K, Li W, Lopez R, McWilliam H, Remmert M, Söding J, Thompson JD, Higgins DG (2011). Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega. Molecular Systems Biology 7:539 doi:10.1038/msb.2011.75
                <SUP>2</SUP> Rice P., Longden I. and Bleasby A. EMBOSS: The European Molecular Biology Open Software Suite. Trends in Genetics. 2000 16(6):276-277
                <SUP>3</SUP><A href="https://www.sigmaaldrich.com/technical-documents/articles/biology/oligoarchitect-online.html">https://www.sigmaaldrich.com/technical-documents/articles/biology/oligoarchitect-online.html</A><SUP>4</SUP> J.S. Reuter and D.H. Mathews. &quot;RNAstructure: software for RNA secondary structure prediction and analysis.&quot; BMC Bioinformatics, 11:129. (2010).
                <SUP>5</SUP><A href="https://www.ncbi.nlm.nih.gov/">https://www.ncbi.nlm.nih.gov/</A><SUP>6</SUP><A href="https://www.ddbj.nig.ac.jp/index-e.html">https://www.ddbj.nig.ac.jp/index-e.html</A><SUP>7</SUP><A href="https://www.ebi.ac.uk/ena">https://www.ebi.ac.uk/ena</A></DIV><H3>Hardware</H3><DIV>
                Having a qPCR test system is fine. But it is only useful, if you have the specific hardware device for it. Our cycler is constructed very simply: a drawer for the test itself has two clamps to fixate the tube, so it won’t move. Underneath the drawer-slot are the operators for our test system. These operators are heat-elements, thermo sensors and relapsing magnets. To get the blood from one chamber of the tube to the next, we use a pump-lever. This pump-lever is montaged above the test-tube and runs by a small stepper-motor. The lever drives on a gear shaft that rotates by the stepper motor. At the end of this lever there is a roller, which pulls the liquid forward. At the end of the device there are LED-diodes and photocells wired to an ARDUINO computer. The diodes radiate light with 495 nm wavelength at the fluorescent probe. The photocells measure the emitted light from the probe and give the data to the ARDUINO computer. This data makes it possible to derive the presence of<I> Plasmodium </I>types.

                <DIV style="border-style: solid; border-color: #CCC; border-width: 1px; margin: 2em auto; width: 80%;"> Scheme of our hardware device.
                </DIV></DIV><H3>Oligonucleotides</H3><DIV>
                We designed the following oligonucleotieds as either Primer, Probe or our BioBrick:

                </DIV><H3>Protocols</H3><DIV>
                We used the following kits and protocols to conduct our experiments: 
                <H4>(RT)qPCR Probe-Kis</H4><I>Biozym Probe qPCR Kit</I> (Biozym)<SUP><A href="#experiment_refs">1</A></SUP><I>One Step RT qPCR Probe ROX L Kit</I> (highQu)<SUP><A href="#experiment_refs">2</A></SUP><I>Quanti Nova Probe PCR Kit</I> (Quiagen)<SUP><A href="#experiment_refs">3</A></SUP><I>LightCycler ® 480 RNA Master Hydrolysis Probes</I> (Roche)<SUP><A href="#experiment_refs">4</A></SUP><H4>(RT)qPCR Sybr Green Kits</H4><I>LightCycler® FastStart DNA Master SYBR Green I</I> (Roche)<SUP><A href="#experiment_refs">5</A></SUP><I>LightCycler® EvoScript RNA SYBR® Green I Master</I> (Roche)<SUP><A href="#experiment_refs">6</A></SUP><I>LightCycler® 480 SYBR Green I Master</I> (Roche)<SUP><A href="#experiment_refs">7</A></SUP><H4>DNA/Plasmid Extaction Kit</H4><I>innuPREP Plasmid Mini Kit</I> (Analytic Jena)<SUP><A href="#experiment_refs">8</A></SUP><I>innuPREP Gel Extraction Kit</I> (Analytic Jena)<SUP><A href="#experiment_refs">9</A></SUP><I>GenElute Plasmid DNA Midiprep Kit</I> (Analytic Jena)<SUP><A href="#experiment_refs">10</A></SUP>E.Z.N.A. Total RNA Kit I (Omega)<SUP><A href="#experiment_refs">11</A></SUP><H4>Agarose gel</H4>
                1,8 % (0,9 g Agarose, 50 ml TAE buffer), 2 µl Ethidiumbromid
                Voltage: 100 V

                <H4><I>NEB® PCR Cloning Kit</I> (NEB)<SUP><A href="#experiment_refs">12</A></SUP></H4><H4>cDNA Synthese with M-MuLV RT Quick Protocol (NEB)<SUP><A href="#experiment_refs">13</A></SUP></H4>
                Template RNA 4 µl, Primer 1 µl, 10x M-MulV buffer 2 µl, M-MulVRT 1 µl, 10 mM dNTP 1 µl, H2O 11 µl
                1h incubation at 42 °C

                <H4>Digest with EcoRI-HF and PSTI-HF<SUP><A href="#experiment_refs">14</A></SUP><SUP><A href="#experiment_refs">15</A></SUP></H4>
                DNA 1 µg, CutSmart 10 x buffer 10 µl, EcoRI-HF 2 µl, PSTI-HF 2 µl, H2O to 100 µl

                <H4>Ligation with T4 DNA ligase<SUP><A href="#experiment_refs">16</A></SUP></H4><H4>Agarplates with chloramphenicol resistance</H4>
                10 g tryptone, 5 g yeast extract, 5 g NaCl, 10 g Agar, H2O to 1 l; autoclave; 10 mg/ml Chloramphenicol

                <H4>Transformation (Changed from NEB)<SUP><A href="#experiment_refs">17</A></SUP></H4>
                Thaw chemical competent cells on ice
                Put 15 ng of the ligation mixture to 50 µl of competent cells
                Place the mixture 10 minutes on ice.
                Heat shock at 42°C for 45 seconds
                Add 1 ml LB-Medium
                Place tube at 37°C for 60 minutes and shake 300 rpm.
                Centrifuge 2 min at 11000 rpm
                Take 850 µl supernatant und resuspend the 200 µl.
                Spread it onto the plates.
                Incubate overnight at 37°C

                <H4>Sequencing<SUP><A href="#experiment_refs">18</A></SUP></H4>
                1 µl DNA, 1 µl BigDye 1.1, 1 µl Primer (10 µM), 2 µl H2O
                <PRE>
    Thermocycler program: 
    1. denaturation 	96°C 	1min
    2. denaturation 	96°C 	20 s
    3. annealing            50°C	15 s
    4. polymerisation 	60°C 	4 min
    5. cooling 		4°C	∞
                </PRE><H4>Seed cells</H4>
                1. Aspirate medium from cell-culture (two 75 cm2) 
                2. Trypsinize cells with 2 ml ATV 
                3. Check cell-detachment under microscope 
                4. Add 2 ml FCS-medium to stop trypsinization 
                5. Resuspend cells by pipetting up and down 
                6. Transfer 3 ml to fresh falcon tube 
                7. Add 9 ml medium to the flask 
                8. Transfer 10 µl to the Neubauer chamber,
                   Count cells in a four to four square (Count 16 mini squares);
                   <I>we counted 182 cells, so we had 1,82 * 10 ^6 cells per ml; </I>   <I>seed 1,5 * 10^5 cells per well in a 6-well plate, 1,5 * 10^5 / 1,82 * 10^6 = 82,4 µl </I>9. Dilute cells; 
                   <I>We had 6 ml in a falcon, so we add 6 ml medium</I>10. Prepare 2* 917,5 µl = 1,835 ml of cell-culture medium per well in a 6-well plate 
                11. Add 165 µl cell suspension to each well 
                12. Shake 6-well plate (3 pieces)

                <H4>Transfection with PEI</H4>
                For each well of a 6-well plate
                1. 0,3 - 0,5 µg GFP
                2. 2,5 - 3,5 µg PUC
                3. 100µl DMEM
                4. Mix
                5. 6 - 9 µl PEI
                5. Mix
                6. Incubate 15 min at room temperature (RT) 
                7. Drop briefly at the cells
            </DIV><H5 id="experiments_refs">List of References</H5><SUP>1</SUP><A href="https://www.biozym.com/desktopmodules/webshop/downloads/331455_170220_Biozym-Probe-qPCR-Kit.pdf">https://www.biozym.com/desktopmodules/webshop/downloads/331455_170220_Biozym-Probe-qPCR-Kit.pdf</A><SUP>2</SUP><A href="https://www.highqu.com/1step-rt-qpcr-probe-rox-l-kit.html">https://www.highqu.com/1step-rt-qpcr-probe-rox-l-kit.html</A><SUP>3</SUP><A href="https://www.qiagen.com/us/resources/resourcedetail?id=5167d782-9fef-4202-bc79-95f358be7d8c&amp;lang=en">https://www.qiagen.com/us/resources/resourcedetail?id=5167d782-9fef-4202-bc79-95f358be7d8c&amp;lang=en</A><SUP>4</SUP><A href="https://lifescience.roche.com/en_de/products/lightcycler14301-480-rna-master-hydrolysis-probes.html#documents">https://lifescience.roche.com/en_de/products/lightcycler14301-480-rna-master-hydrolysis-probes.html#documents</A><SUP>5</SUP><A href="https://lifescience.roche.com/en_de/products/lightcycler-faststart-dna-master-sybr-green-i.html#documents">https://lifescience.roche.com/en_de/products/lightcycler-faststart-dna-master-sybr-green-i.html#documents</A><SUP>6</SUP><A href="https://www.lifescience.roche.com/en_de/products/lightcycler-evoscript-rna-sybr-green-i-master.html#documents">https://www.lifescience.roche.com/en_de/products/lightcycler-evoscript-rna-sybr-green-i-master.html#documents</A><SUP>7</SUP><A href="https://biochimie.umontreal.ca/wp-content/uploads/sites/37/2015/11/LC480SYBRMasterguide.pdf">https://biochimie.umontreal.ca/wp-content/uploads/sites/37/2015/11/LC480SYBRMasterguide.pdf</A><SUP>8</SUP><A href="https://www.analytik-jena.de/fileadmin/content/pdf_life_science/Manual/Manual_innuPREP_Plasmid_Mini_Kit_2.0.pdf">https://www.analytik-jena.de/fileadmin/content/pdf_life_science/Manual/Manual_innuPREP_Plasmid_Mini_Kit_2.0.pdf</A><SUP>9</SUP><A href="https://www.analytik-jena.de/fileadmin/content/pdf_life_science/Manual/Manual_innuPREP_Gel_Extraction_Kit.pdf">https://www.analytik-jena.de/fileadmin/content/pdf_life_science/Manual/Manual_innuPREP_Gel_Extraction_Kit.pdf</A><SUP>10</SUP><A href="https://www.sigmaaldrich.com/catalog/product/sigma/pld35?lang=de&amp;region=DE">https://www.sigmaaldrich.com/catalog/product/sigma/pld35?lang=de&amp;region=DE</A><SUP>11</SUP><A href="http://omegabiotek.com/store/wp-content/uploads/2013/04/R6834-Total-RNA-Mini-Kit-I-Combo-Online.pdf">http://omegabiotek.com/store/wp-content/uploads/2013/04/R6834-Total-RNA-Mini-Kit-I-Combo-Online.pdf</A><SUP>12</SUP><A href="https://international.neb.com/-/media/catalog/datacards-or-manuals/manuale1202.pdf">https://international.neb.com/-/media/catalog/datacards-or-manuals/manuale1202.pdf</A><SUP>13</SUP><A href="https://international.neb.com/protocols/2016/04/26/first-strand-cdna-synthesis-quick-protocol-neb-m0253">https://international.neb.com/protocols/2016/04/26/first-strand-cdna-synthesis-quick-protocol-neb-m0253</A><SUP>14</SUP><A href="https://international.neb.com/protocols/2012/12/07/optimizing-restriction-endonuclease-reactions">https://international.neb.com/protocols/2012/12/07/optimizing-restriction-endonuclease-reactions</A><SUP>15</SUP><A href="https://nebcloner.neb.com/#!/redigest">https://nebcloner.neb.com/#!/redigest</A><SUP>16</SUP><A href="https://international.neb.com/Protocols/0001/01/01/dna-ligation-with-t4-dna-ligase-m0202">https://international.neb.com/Protocols/0001/01/01/dna-ligation-with-t4-dna-ligase-m0202</A><SUP>17</SUP><A href="https://international.neb.com/protocols/2012/05/21/transformation-protocol">https://international.neb.com/protocols/2012/05/21/transformation-protocol</A><SUP>18</SUP><A href="http://www3.appliedbiosystems.com/cms/groups/mcb_support/documents/generaldocuments/cms_041330.pdf">http://www3.appliedbiosystems.com/cms/groups/mcb_support/documents/generaldocuments/cms_041330.pdf</A></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>