<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_JMU_Wuerzburg_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:JMU Wuerzburg/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>Home</P><LI class="menu"><P>Team</P></LI><LI><A href="https://2018.igem.org/Team:JMU_Wuerzburg/Team">Team Members</A></LI><LI><A href="https://2018.igem.org/Team:JMU_Wuerzburg/Collaborations">Collaborations</A></LI><LI class="menu"><P>Project</P></LI><LI><A href="https://2018.igem.org/Team:JMU_Wuerzburg/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:JMU_Wuerzburg/Applied_Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:JMU_Wuerzburg/Introduction_Malaria">Introduction to Malaria</A></LI><LI><A href="https://2018.igem.org/Team:JMU_Wuerzburg/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:JMU_Wuerzburg/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:JMU_Wuerzburg/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:JMU_Wuerzburg/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:JMU_Wuerzburg/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:JMU_Wuerzburg/Sponsors">Sponsors</A></LI><LI class="menu"><P>Parts</P></LI><LI class="menu"><P>Safety</P></LI><LI class="menu"><P>Human Practices</P></LI><LI><A href="https://2018.igem.org/Team:JMU_Wuerzburg/Human_Practices#workshops">Workshops</A></LI><LI><A href="https://2018.igem.org/Team:JMU_Wuerzburg/Human_Practices#education">Education</A></LI><LI><A href="https://2018.igem.org/Team:JMU_Wuerzburg/Human_Practices#establishing">Establishing an iGEM-Team</A></LI><LI><A href="https://2018.igem.org/Team:JMU_Wuerzburg/Human_Practices#collabs">Collaborations for our Laboratory Work</A></LI><LI class="menu"><P>Judging Form</P></LI><DIV class="content"><H3>Bioinformatics</H3><DIV>
                To detect <I>Plasmodium</I> in general we aligned 2080 sequences from NCBI. 
                Our primers and the probe has a consensus of more than 98% and in one case 94,9%. 
                The oligos are located in the mitochondria in a regulatory region. 
                
                In the case of <I>Plasmodium falciparum</I>, we aligned 1012 sequences of 
                <I>Plasmodium falciparum</I> and our primer set and the probe has a higher 
                consensus than 97,5%. The location of the oligos is in a regulatory region of the mitochondria.
            </DIV><H3>Wet Lab</H3><DIV>
                In general, we can detect <I>Plasmodium</I> in all five cultured <I>Plasmodium</I> strains of <I>Plasmodium falciparum</I> 
                (FCR3, Mali2K, HB3, Dd2 and 3D7 from Tuebingen and Heidelberg). 
                In addition, we can detect our Biobrick (BBa_K2614000) 
                that includes a synthetic template as a positive control for the 
                qPCR that detects <I>Plasmodium</I> in general (figure 1). 
                We can detect less than 1000 copies of our synthetic template with a probe (FAM) 
                and less than 50 copies with SybrGreen.

                <DIV style="border-style: solid; border-color: #CCC; border-width: 1px; margin: 2em auto; width: 80%;"><SPAN style="text-decoration: underline;">Figure 1:</SPAN> qPCR with cultured <I>Plasmodium</I> stems and our BioBrick as a control for <I>Plasmodium</I> general
                </DIV>

                Moreover, we are able to detect <I>P. falciparum</I> in all five cultured <I>Plasmodium</I> strains. 
                Our BioBrick is a negative control for our qPCR that detects <I>P. falciparum</I>. 
                With the probe (HEX) we detect less than ten copies of our synthetic template. (figure 2)

                <DIV style="border-style: solid; border-color: #CCC; border-width: 1px; margin: 2em auto; width: 80%;"><SPAN style="text-decoration: underline;">Figure 2:</SPAN> qPCR with cultured <I>Plasmodium stems</I> and our BioBrick as a control for <I>P. falciparum</I>.
                </DIV>

                In a multiplex system the two probes (FAM and HEX) also 
                detect the five cultured <I>Plasmodium</I> strains. 
                They detect them with nearly the same cq-mean which means that 
                both qPCRs can detect <I>Plasmodium</I>. With this result we showed that 
                multiplexing is also possible with the two primer/probe pairs (figure 3).

                <DIV style="border-style: solid; border-color: #CCC; border-width: 1px; margin: 2em auto; width: 80%;"><SPAN style="text-decoration: underline;">Figure 3:</SPAN> Multiplex qPCR with cultured <I>Plasmodium</I> stems and our BioBrick as a control for <I>Plasmodium</I> general and specific for <I>P. falciparum</I></DIV></DIV><H3>Human practices</H3><DIV>
                We organised different workshops from scientists about 
                malaria or primer/probe design which were open for 
                all interested students. There was also a pipetting test where 
                more experienced students could give their knowledge to pipetting beginners. 
                At our “cake and science” events we presented our project to the public.
            </DIV><H3>Interlab</H3><DIV>
                We participated successfully at the
                <I>fifth International InterLaboratory Measurement Study</I> 
                in synthetic biology. At the iGEM 2018 Vibrigens InterLab Study 
                from Marburg we also take place.
            </DIV><H3>Collaboration</H3><DIV>
                We were able to establish a very pleasant and close 
                collaboration with team Munich. They have mentored us with great 
                care and reliability and answered all our questions. 
                So, it was much easier to start a new iGEM team in Wuerzburg. 
                We also received advice from different members of team Tuebingen 
                who were so kind to exchange some valuable information and experience with us.
            </DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>