<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU_Delft_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU Delft/Description</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mobile-navigation"><UL><LI><A href="https://2015.igem.org/Team:TU_Delft">Home</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Achievements">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Achievements#prizes">Prizes and Nominations</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Achievements#jamboree">Giant Jamboree</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Achievements#medal">Medal Requirements</A></LI></UL><LI><A href="https://2015.igem.org/Team:TU_Delft/Description">Project</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Description#description">Description</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Description#strains">Strains</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Description#cells">Linking Cells</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Description#tags">Affinity Tags</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Description#printing">Printing Biofilm Layers</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Description#references">References</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Parts#overview">Overview</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Parts#table">Parts</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Parts#collection">Part Collection</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Notebook#work">Work Space</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Notebook#safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Notebook#notes">Day Notes</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Notebook#protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Design">Hardware</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Design#overview">Overview</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Design#3dbiofilms">Making 3D Biofilms</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Design#biolinker">The Biolinker</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Design#integration">Design and Integration</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Design#manuals">DIY Manuals</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Design#protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Modeling#overview">Overview</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Modeling#gene">Kinetic Modeling</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Modeling#curli">Curli Formation</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Modeling#biofilm">Biofilm Strength</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Modeling#p&amp;p">Time for Printing</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices">Policy&amp;Practice</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices#overview">Overview</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices#regulations">Regulations</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices#ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices#social">Experts' Opinions</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices#community">iGEM Community</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices#tool" class="sub-nav-item">Biolink P&amp;P Tool</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices#business">Business Plan</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices#outreach">Outreach</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Collaborations">Collaboration</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Collaborations#overview">Overview</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Collaborations#testing">Testing Biolinker</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Collaborations#course">Dutch iGEM Teams Meeting</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Collaborations#cooperations">Cooperations</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Attributions#members">Team Members</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Attributions#advisors">Advisors</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Attributions#support">Team Work Attributions</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Attributions#media">Media</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Attributions#sponsors">Sponsors</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Attributions#acknowledgements">Acknowledgements</A></LI></DIV><DIV id="headerWrapper" class="hidden-xs hidden-sm"><DIV id="header"><DIV id="navigation"><UL><LI><A href="https://2015.igem.org/Team:TU_Delft/Achievements" class="achievements nav-item" id="achievements">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Description" class="project nav-item" id="project">Project</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Parts" class="results nav-item" id="parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Notebook" class="notebook nav-item" id="notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Design" class="hardware nav-item" id="hardware">Hardware</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Modeling" class="modeling nav-item" id="modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices" class="practices nav-item" id="practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Collaborations" class="collaborations nav-item" id="collaborations">Collaboration</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Attributions" class="attributions nav-item" id="attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Measurement" class="interlab nav-item" id="interlab">InterLab</A></LI></UL></DIV></DIV><DIV id="sub-navigation-wrapper"><DIV id="sub-navigation"><UL id="notebook-sub"><LI><A href="https://2015.igem.org/Team:TU_Delft/Notebook#work" class="sub-nav-item">Work Space</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Notebook#safety" class="sub-nav-item">Safety</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Notebook#notes" class="sub-nav-item">Day Notes</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Notebook#protocols" class="sub-nav-item">Protocols</A></LI></UL><UL id="achievements-sub"><LI><A href="https://2015.igem.org/Team:TU_Delft/Achievements#prizes" class="sub-nav-item">Prizes and Nominations</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Achievements#jamboree" class="sub-nav-item">Giant Jamboree</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Achievements#medal" class="sub-nav-item">Medal Requirements</A></LI></UL><UL id="attributions-sub"><LI><A href="https://2015.igem.org/Team:TU_Delft/Attributions#members" class="sub-nav-item">Team Members</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Attributions#advisors" class="sub-nav-item">Advisors</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Attributions#support" class="sub-nav-item">Team Work Attributions</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Attributions#media" class="sub-nav-item">Media</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Attributions#sponsors" class="sub-nav-item">Sponsors</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Attributions#acknowledgements" class="sub-nav-item">Acknowledgements</A></LI></UL><UL id="project-sub"><LI><A href="https://2015.igem.org/Team:TU_Delft/Description#description" class="sub-nav-item">Description</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Description#strains" class="sub-nav-item">Strains</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Description#cells" class="sub-nav-item">Linking Cells</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Description#tags" class="sub-nav-item">Affinity Tags</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Description#printing" class="sub-nav-item">Printing Biofilm Layers</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Description#references" class="sub-nav-item">References</A></LI></UL><UL id="parts-sub"><LI><A href="https://2015.igem.org/Team:TU_Delft/Parts#overview" class="sub-nav-item">Overview</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Parts#table" class="sub-nav-item">Parts</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Parts#collection" class="sub-nav-item">Part Collection</A></LI></UL><UL id="hardware-sub"><LI><A href="https://2015.igem.org/Team:TU_Delft/Design#overview" class="sub-nav-item">Overview</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Design#3dbiofilms" class="sub-nav-item">Making 3D Biofilms</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Design#biolinker" class="sub-nav-item">The Biolinker</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Design#integration" class="sub-nav-item">Design and Integration</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Design#manuals" class="sub-nav-item">DIY Manuals</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Design#protocols" class="sub-nav-item">Protocols</A></LI></UL><UL id="collaborations-sub"><LI><A href="https://2015.igem.org/Team:TU_Delft/Collaborations#overview" class="sub-nav-item">Overview</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Collaborations#testing" class="sub-nav-item">Testing Biolinker</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Collaborations#course" class="sub-nav-item">Dutch iGEM Teams Meeting</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Collaborations#cooperations" class="sub-nav-item">Cooperations</A></LI></UL><UL id="practices-sub"><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices#overview" class="sub-nav-item">Overview</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices#regulations" class="sub-nav-item">Regulations</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices#ethics" class="sub-nav-item">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices#social" class="sub-nav-item">Experts' Opinions</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices#community" class="sub-nav-item">iGEM Community</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices#tool" class="sub-nav-item">Biolink P&amp;P Tool</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices#business" class="sub-nav-item">Business Plan</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices#outreach" class="sub-nav-item">Outreach</A></LI></UL><UL id="interlab-sub"><LI><A href="https://2015.igem.org/Team:TU_Delft/Measurement#results" class="sub-nav-item">Results</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Measurement#protocols" class="sub-nav-item">Protocols</A></LI></UL><UL id="modeling-sub"><LI><A href="https://2015.igem.org/Team:TU_Delft/Modeling#overview" class="sub-nav-item">Overview</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Modeling#gene" class="sub-nav-item">Kinetic Modeling</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Modeling#curli" class="sub-nav-item">Curli Formation</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Modeling#biofilm" class="sub-nav-item">Biofilm Strength</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Modeling#p&amp;p" class="sub-nav-item">Time for Printing</A></LI></UL></DIV></DIV></DIV><DIV class="jumbotron" id="cover"><H1>Project Biolink</H1><P class="lead">3D printing of bacterial biofilms, linked together through nanowires</P></DIV><DIV id="white"><DIV class="jumbotron"><DIV class="container"><H1>Project Description</H1><P class="lead text-muted">In this section we describe the Problem and Solution, our Biolink project as well as an overview of the synthetic biology and printing process in the project.</P></DIV></DIV><DIV class="container"><H2>Problem and Solution</H2><P class="lead">Biofilms are communities of bacteria connected by protein nanowires and surrounded by an extracellular matrix. In this form, they are more resistant and can severely affect human health, industrial productivity and the environment. More precisely, biofilms can cause infections in the human body, affect water quality, and damage industrial installations and equipment. Research and industry have been working to find various solutions for preventing and removing this threat. Potential solutions, such as health products, drugs and industrial removal products, are tested on artificially formed biofilms.</P><FIGURE><FIGCAPTION>Figure 1. Fields and industries affected by detrimental biofilms (4Inno.com)</FIGCAPTION></FIGURE><P class="lead">The problem with biofilms formed artificially is that they are time consuming, difficult to control and to reproduce. This means that artificial biofilms do not reflect natural biofilm characteristics, making product testing unreliable. Therefore, biofilm-removal products may have a different effect when used in natural settings, with unforeseen negative side-effects and reduced efficiency. </P><DIV class="row"><DIV class="col-md-5"><FIGURE><FIGCAPTION>Figure 2. Design of the prototype of our DIY 3D printer</FIGCAPTION></FIGURE></DIV><DIV class="col-md-7"><P class="lead">Our project is entitled Biolink and provides an alternative to current biofilm formation technologies. We use a 3D printer, which we call <I>The Biolinker</I>, to form layers of a designed bioink made of bacteria that can bind together into a desired structure.</P><P class="lead">Biolink helps biofilm-related industries in several ways. First, it brings reproducibility and control to how bacterial biofilms can be artificially formed. Second, biofilm printing adds automation and scalability, making biofilm formation processes more efficient, and thus, cheaper. Hence, Biolink can help to design safer and more effective anti-biofilm solutions by increasing biofilm testing process efficiency and resemblance to reality. </P></DIV></DIV><H2>The Biolink Project</H2><P class="lead">Our printing system, called Biolink, can be summarized in the following sentence: biofilm producing bacteria are printed with the help of a flexible scaffold hydrogel. First of all, our homemade bacteria (modified to make biofilms) are mixed with a solution of sodium alginate and subsequently with calcium chloride. There, the Ca2+ molecules keep the structure fixed creating a stable gel. This hydrogel-bacteria mixture is then induced with rhamnose, a sugar specific for our promoter,  which makes them synthesize CsgA, the linking molecule. CsgA proteins polymerize to an amyloid structure surrounding the cells and connecting them to each other through the scaffold. Once the cells are all attached in the structure defined by the gel scaffold, it is no longer necessary. Consequently, the hydrogel is dissolved with sodium citrate. But the cells are still connected due to the curli amyloid! So, we obtain a perfectly defined 3D structure made of bacteria.</P><DIV id="covercar" class="carousel slide" data-ride="carousel" data-interval="20000" data-pause="hover"><A class="left carousel-control" href="#covercar" role="button" data-slide="prev"><SPAN class="sr-only">Previous</SPAN></A><A class="right carousel-control" href="#covercar" role="button" data-slide="next"><SPAN class="sr-only">Next</SPAN></A></DIV><P class="lead">The Biolink project promotes the open source and educational spirit of iGEM. Our 3D printer, the Biolinker, is made of K’NEX construction toys, a DIY solution that is both easy to build and efficient in doing its job. Through policy and practice we try to position our project within the synthetic biology industry and academia, as well as observe socio-economic perception and feedback. We accomplish this by analyzing and interviewing stakeholders, treating ethical and regulatory issues, and building a business plan around our project. </P><H2>Synthetic Biology in our Project</H2><P class="lead">During biofilm formation, bacteria produce an extracellular matrix made of amyloid structures. These amyloid structures are curli fimbriae, composed of intertwined filaments with a thickness of approximately 4-7 nm (Nguyen, 2014). Therefore, curli production helps bacteria bind to each other in natural biofilms (Taylor et. al. 2012). There are two distinctive operons involved in this highly regulated pathway; csgBA and csgDEFG. The csgBA operon encodes for two proteins: CsgA and CsgB. The csgDEFG operon encodes for the proteins required for the transport of CsgA and CsgB to the cell surface (Dueholm et al, 2011).</P><P class="lead">On one hand, CsgA is an amyloid protein that acts as monomer for curli formation. On the other hand, the protein CsgB in an integral membrane protein, which binds CsgA to the cell; CsgB acts as an anchor for curli formation. CsgA is transported as an unfolded protein to the extracellular matrix. Once outside the cell, it aggregates with CsgB and the self-assembly of these aggregates form the amyloid fibrils. When CsgA comes in contact with CsgB, the fibrils bind to another cell and the process is repeated again until an entire network has been created (Barnhart et al, 2006).</P><P class="lead">In our project, we designed an inducible system for synthesizing CsgA in biofilm-making deficient cells. Besides to that, we aimed to create a customized biofilm. To do so, we designed different biobricks that contain a peptide tail attached to the sequence of the biofilm protein CsgA that provides a specific surface affinity. In the end, we planned to use our engineered cells (which also express a fluorescent reporter) for printing in different layers; printing biofilms in 3 dimensions.</P><H2>The Bioink and Alginate as Supporting Scaffold</H2><P class="lead">In initial experiments printing with bacterial cells dispersed in LB media it quickly became obvious, that a supporting scaffold would be required: due to surface interactions the liquid spread out creating a final thickness of the printed line of almost one centimeter.  In tissue engineering sodium alginate is commonly used as a synthetic extracellular matrix material (Rowley et al., 1999). Inspired by this, we took a look at sodium alginate to use as a scaffold material. Sodium alginate is a carbohydrate polymer which can be fairly well dissolved in water, but in contact with calcium ions (or other divalent cations) the polymers are connected via electrostatic interactions forming a hydrogel. Made for instance from LB, this hydrogel could provide bacterial cells with everything they need for weeks and keep them alive. Furthermore, jellification is a reversible process by complexing the calcium ions with citrate and replacing it again with sodium ions (Rowley et al., 1999). Thus, we had found a substance ideally meeting our purpose of being initially liquid, capable of rapidly turning into a viscous gel and reversing this process again.</P></DIV></DIV><DIV id="grey"><DIV class="jumbotron"><DIV class="container"><H1>Strains</H1><P class="lead text-muted">The strains used for our project are described here</P></DIV></DIV><DIV class="container"><H2>Escherichia coli K-12 MG1655 PRO ΔcsgA ompR234</H2><DIV class="row"><DIV class="col-md-5"><P class="lead">This strain of Escherichia coli, used in previous studies of amyloid fiber production in bacteria, is characteristic for having knocked-out the CsgA gene. However, it has all the other genes required for formation of the curli structure.</P><P class="lead">We have used this strain for our project as the main organism for the printing process. As the bacteria cannot express csgA, we transformed our strains with a plasmid containing this gene under the control of an inducible promoter. Consequently, we can modulate where and when the amyloid fiber will be formed! (Chen, A.Y., et al. 2014)</P></DIV><DIV class="col-md-7"><FIGURE><FIGCAPTION>Figure 3. Photograph of some cells of the strain Escherichia coli K-12 MG1655 PRO ΔcsgA ompR234 visualized with TEM, after they were transformed with a plasmid containing the genes necessaries for producing CsgA.</FIGCAPTION></FIGURE></DIV></DIV><H2>Escherichia coli Top10</H2><P class="lead">This strain was used exclusively for highly efficient transformations. We used this organism for cloning experiments, plasmid isolations and other basic steps of our project.</P><FIGURE><FIGCAPTION>Figure 4. Picture of a successful transformation carried out during the project. Top10 cells were used for highly efficient transformations.</FIGCAPTION></FIGURE></DIV></DIV><DIV id="white"><DIV class="jumbotron"><DIV class="container"><H1>Linking Cells</H1><P class="lead text-muted">The possibility of printing bacteria in structured layers is supported by the biological property of making biofilms. In this part of the project we studied the proteins that allow cells to produce connections, and how we can modulate their production</P></DIV></DIV><DIV class="container"><H2>Introduction and Motivation</H2><P class="lead">Printing a biofilm requires the bacteria to produce several components that allow them to make a connection between them. Furthermore, they require to attach to the surface where they are printed on. Finally, they need to make the biofilm only when it is desired.</P><P class="lead">In order to study the connections that exist between bacteria, we focused on the bacterial amyloid curli structure. These bacterial structures contribute to biofilm performances where cells interact with other cells and even surfaces. The curli consists of proteins bound together and to the cell membrane. CsgA, the main subunit, polymerizes in the extracellular space creating an amyloid nanowire; and CsgB anchors this nanowire to the membrane, creating connections between different cells (Joshi et al., 2014).</P><P class="lead">Consequently, we designed different versions of biobricks (see Parts) coding for the CsgA gene to create a biofilm in a controlled manner from CsgA deficient cells. To prevent premature biofilm formation and clogging of our printer, we designed the expression of CsgA to be  inducible by rhamnose so that the biofilm is only created after the cells are printed. This also functions as a safety feature in case of escape. The experiments (and their results) that we used to analyse the expression of CsgA and its promoter are explained in this section.</P><DIV class="panel-group" id="accordionproject" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="heading1"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordionproject" href="#collapse1" aria-expanded="false" aria-controls="collapse1">
 Fluorescence assay for kinetic measurements regarding induction of csgA expression
          </A></DIV><DIV id="collapse1" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading1"><DIV class="panel-body"><H3>Background</H3><P class="lead">In the kinetic experiment, the fluorescence signal of the proteins CsgA and CsgA-GFPmut3 (I13504) proteins expressed by our constructs was recorded in time. Besides fluorescence, we measured the OD600 of our cultures in order to normalize the fluorescence signal per cell. The main goal of this experiment was to see, if different induction levels with increasing rhamnose concentrations would lead to a higher production of CsgA. If this was true, the quantity of produced CsgA present in the medium could be controlled. This degree of control is key to achieving our main goal; making a biofilm with reproducible strength.</P><P class="lead">The obtained results can be used to calculate the promoter strength at different induction levels of rhamnose in our model. In order to use the mathematical model previously constructed in the modeling section, it has to be fitted to GFPmut3 units/cell/second. Therefore, we set up a calibration curve with exactly the same settings as the fluorescent experiment in order to correlate fluorescence signal to units of GFPmut3 per cell.  </P><H3>Methodology</H3><P class="lead">In this experiment, the strains ∆csgA – csgA- I13504 and ∆csgA – csgA strain were both induced with rhamnose at different concentrations. The different induction levels can be found in Table 1.</P><P class="lead">All conditions (ID 1 – 6) were carried out in triplicates for accurate statistical analysis of the data. The different cultures were grown and induced in a 96-well plate. The OD600 and the fluorescence signal was recorded with a plate reader during an 18-hour period at 30°C.  </P><H3>Results and discussion</H3><P class="lead">In Figure 5, the fluorescent signal was normalized with the number of cells and plotted as a function of time. The red bars denote the error within each ID.</P><FIGURE><FIGCAPTION>Figure 5: Fluorescence signal normalized by the number of cells for 0% (w/v), 0.2% (w/v) and 0.5% (w/v) rhamnose with the csgA and csgA-GFPmut3 construct. The error bars are included for all experiments.</FIGCAPTION></FIGURE><P class="lead">As observed in Figure 5, only the strains carrying the csgA-GFPmut3 construct induced with 0.2% (w/v) and 0.5% (w/v) showed a clear increase in fluorescence signal over time. The rest of the cultures, didn’t show significant fluorescence over time. </P><P class="lead">Furthermore, we have showed that increasing concentrations of rhamnose lead to increasing amounts of produced csgA-GFPmut3 and thus fluorescence. Finally, as the fluorescence signal is normalized by the cell density, one can make statements about the activity of the rhamnose promoter. The promoter seems to not be active directly after induction, but activity is observed after a time period of 3 to 4 hours.  This is in accordance with data from literature (Wegerer et. Al, 2008), in which low fluorescence levels were observed after 2 hours of induction of the rhamnose promoter.</P><P class="lead">The calibration line of fluorescence versus mass amount GFPmut3 is given in Figure 6.</P><FIGURE><FIGCAPTION>Figure 6. Calibration line of the fluorescence signal of GFPmut3 versus its amount in nanograms.</FIGCAPTION></FIGURE><P class="lead">The corresponding function of the GFPmut3 calibration line is:</P><P class="lead">With mass<SUB>GFP</SUB> in ng.</P><P class="lead">In the modelling, the fluorescent data in Figure 5 will be further converted to molecules GFPmut3/cell and the promoter activity will be calculated for both the 0.2% (w/v) and 0.5% (w/v) level of rhamnose induction. With this kinetic experiment, we have proven that our csgA-GFPmut3 construct is able to produce different levels of GFPmut3 by varying the rhamnose concentration.</P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="heading2"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordionproject" href="#collapse2" aria-expanded="false" aria-controls="collapse2">
 Transmission Electron Microscopy (TEM) for visualization of bacterial amyloid curli
          </A></DIV><DIV id="collapse2" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading2"><DIV class="panel-body"><H3>Background</H3><P class="lead">With our biofilm formation model we aimed to estimate how many times the nanowires overlap in order to predict the biofilm strength. One of the variables required for this model is the persistence length. In order to determine it, we have to visualize the nanowires produced by our engineered bacteria. Moreover we wanted to visualize the difference between induced and uninduced cultures containing the csgA_pSB1C3 plasmid with a rhamnose inducible promoter. These experiments  required using an electron microscope.</P><P class="lead">Electron microscopes use electrons to visualize nano-objects, such as the curli nanowires. In the large “gun” the electrons get enough space to accelerate before they illuminate the sample. As shown in Figure 7 after the electrons are fired by the gun, an electric field of several thousand volts is applied to the electron beam in order to point this to the sample. In the case of an TEM, the electrons pass through the sample where the detector can detect the sample (Boysen &amp; Muir, 2011).</P><FIGURE><FIGCAPTION>Figure 7: Transmission electron microscopy. The picture on the left represents a model of the TEM and on right the different elements and positions within the microscope (Boysen &amp; Muir, 2011).</FIGCAPTION></FIGURE><H3>Methodology</H3><P class="lead">At first we measured an OD600 of 3.00 of an overnight culture of ∆csgA_csgA_pSB1C3 cells. We diluted the sample down to an OD600 of approximately 0.05, with LB medium + CAM (antibiotic, chloramphenicol). To achieve an end volume of 4 mL we took 67 µl of the overnight culture and 3933 µl of LB + CAM. We let the cells in the new sample  grow for 1.5 hour at 37°C to reach the exponential phase (OD between 0.35-0.60). After the cells reached the exponential phase, we induced 1 mL of the culture with 1% rhamnose (250 µl from our 5% rhamnose stock) and also took 1 mL sample of the uninduced culture. We incubated both samples overnight at room temperature to allow the production of nanowires. Finally, the samples were directly added to the grid and the following TEM pictures were taken.</P><H3>Results</H3><FIGURE><FIGCAPTION>Figure 8: Pictures taken with TEM (magnification 7300 x). The left picture shows the situation without induction (0% rhamnose) and the right one shows the situation with induction (1% rhamnose).</FIGCAPTION></FIGURE><P class="lead">In Figure 8, both the situation, pictured with TEM, without induction (the left one) and with induction (the right one) are shown with a magnification of 7300 x. The situation in which there was no induction with rhamnose, the bacteria did not show any curli formation. The opposite was true for the case in which the cells were induced with rhamnose. As the TEM picture clearly shows, nanowires are present after induction.</P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="heading3"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordionproject" href="#collapse3" aria-expanded="false" aria-controls="collapse3">
            Microtiter Plate Assay for Ring-Shaped, curli dependent biofilm detection
          </A></DIV><DIV id="collapse3" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading3"><DIV class="panel-body"><H3>Background</H3><P class="lead">The bacteria that we engineered for the project are capable of producing curli proteins after induction with L-rhamnose. However, the goal of our project is the printing of biofilms and not the sole overexpression of these proteins. In order to assess whether our CsgA-producing bacteria can make a biofilm or just remains planktonic, our team adapted the protocol from Zhou et al. (2013) that employs crystal violet (methyl violet 10B) for dying the biofilm-making bacteria that attach to surfaces. </P><P class="lead">According to the source article (Zhou et al., 2013), a deficient mutant of Escherichia coli is unable to produce a biofilm. By performing this experiment, we want to go further in characterizing our curli proteins; we want to answer this question: are we really making biofilms?</P><H3>Methodology</H3><P class="lead">In the experiment, our CsgA-producing strain of E.coli is induced at a high (0.5% w/v), low (0.2% w/v) and no (0% w/v) concentration of L-rhamnose. Furthermore, csgA deficient bacteria transformed with an empty plasmid (pSB1C3) are also tested and used as a control (Table 2.). All the experiments were carried out in triplicates.</P><FIGURE><FIGCAPTION>Table 2. Strains of Escherichia coli used for assess the biofilm formation using crystal violet</FIGCAPTION></FIGURE><P class="lead">The cultures of both strains were grown and induced in a 96-well plate. 40 hours after the induction, the samples were washed with water (in order to disregard non-attached, planktonic cells) and dyed with crystal violet (figure 9.). </P><FIGURE><FIGCAPTION>Figure 9. 96-well plate used for test the biofilm-making capability of our designed constructs. The first two rows (A-B) contained cultures with 0% w/v of rhamnose. The columns D-E and G-H were inducted with l-rhamnose to a final concentration of 0.2%w/v and 0.5%w/v respectively. The green boxes show the strains carrying BBa_K1583000, and the red ones the control strain with pSB1C3.</FIGCAPTION></FIGURE><H3>Results and discussion</H3><P class="lead">In the end, the wells were diluted with ethanol so all the content can dissolve in the liquid phase. We measured the absorbance at 590 nm of wavelength for all the samples, obtaining the following results (figure 10.):</P><FIGURE><FIGCAPTION>Figure 10. Microtiter Plate Assay results for testing biofilm formation. CTRL0, CTRL2 and CTRL5 are E. coli K-12 MG1655 PRO ΔcsgA ompR234 cells with pSB1C3, used as a control. CSGA0, CSGA2 and CSGA5 are E. coli K-12 MG1655 PRO ΔcsgA ompR234 cells with BBa_K1583000. The termination 0, 2 and 5 denote the induction with no rhamnose (0), 0.2% w/v (2) and 0.5% w/v (5).</FIGCAPTION></FIGURE><P class="lead">The cells induced with l-rhamnose showed an increased absorbance at 590 (the peak of the dye) when compared to the empty-plasmid controls, and also with the non-induced sample. That results clearly demonstrate how our engineered cells with BBa_K1583000 successfully make biofilms, regarding that the empty plasmid controls and the analysed samples have significantly (table 3.) higher crystal violet retention. On the other hand, a higher concentration of rhamnose is not leading to a higher expression. The cells induced with a 0.2% w/v of rhamnose seem to create better a biofilm structure. However, this could be a consequence of different growth patterns; the cultures induced with 0.5% w/v of rhamnose could have stopped duplicating earlier, so the cell concentration could have decreased. So, the dyed area could be consequently smaller.</P><FIGURE><FIGCAPTION>Table 3. Significance analysis of the samples analysed, with a significance α value of 10%. All the samples (CSGA) but the 0% induction control display a significant difference when compared to the empty plasmid homolog sample (CTRL).</FIGCAPTION></FIGURE></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="heading4"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordionproject" href="#collapse4" aria-expanded="false" aria-controls="collapse4">
            Quantification of the internal CsgA concentration using Western Blot
          </A></DIV><DIV id="collapse4" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading4"><DIV class="panel-body"><H3>Background</H3><P class="lead">For the <A href="https://2015.igem.org/Team:TU_Delft/Modeling#gene">kinetic model</A>, we needed to determine the internal concentration of csgA. We cloned a CsgA gene, with a C-terminus His tag, in the construct in a CsgA deficient strain. The His-tag would give us the advantage to use multiple techniques, for measuring the amount of CsgA proteins. Based on the kinetic experiment with GFPmut3, we obtained an internal amount of CsgA in the order of 104 molecules/cell. To validate whether this value was plausible we decided to use a sensitive, and an accurate method: the Western Blot.</P><H3>Methodology</H3><P class="lead"><B>Experiment 1: Qualitative analysis of internal CsgA_His protein</B></P><P class="lead">We performed the Western Blot according to protocol 24. The following samples (Table 4) were used for performing the Western Blotting experiment.</P><FIGURE><FIGCAPTION>Table 4. Samples used for the first Western Blot experiment.</FIGCAPTION></FIGURE><P class="lead">Alternative steps were done during step 5 of the protocol. After incubation we removed the medium, by centrifugation at 4000 rpm for 15 minutes at 4000 rpm. The supernatant was discarded, and the cells were resuspended in sample buffer to obtain an OD600 of 10. The cells were then heated for 10 minutes at 95°C, followed by centrifugation for 5 minutes at 32000 rpm at 4°C. This was then continued with step 6. For step 6, we added 10 µl per slot in the SDS page gel. </P><P class="lead"><B>Experiment 2: Quantitative analysis of the internal CsgA_His protein.</B></P><P class="lead">We executed a second experiment to quantify the amount of CsgA_His present in the ΔcsgA_csgA strain. We compared this amount by using a the T-SSRA madrid proteïn solution.  T-SSRA madrid proteïn solution has a set amount of proteins with the His tag on it. Therefore by using the Lambert-Beer law, the concentration of proteïn was calculated to be 200µM. We used a dilution series of 1/10, 1/25,1/50, 1/100, 1/500, 1/1000 (Table 5) in order to make a calibration curve.</P><FIGURE><FIGCAPTION>Table 5.Samples used for the second Western Blot experiment.</FIGCAPTION></FIGURE><P class="lead">The Western Blot protocol (nr 24) was followed, as executed in experiment 1. </P><H3>Results and discussion</H3><P class="lead"><B>Experiment 1: Qualitative analysis of internal CsgA_His protein</B></P><P class="lead">In Figure 11A, the SDS page gel is shown. The gel is too condensed to visualize a difference between the non-induced and induced samples, because the gel was overloaded.</P><FIGURE><FIGCAPTION>Figure 11: A. SDS page gel, experiment 1 samples as described in Table 1.B. Membrane after blotting, of the samples of experiment 1 (Left to right).</FIGCAPTION></FIGURE><P class="lead">We saw in figure 11B a band in lane 4 and 8, with a size of circa. 17 kDA. The lanes correspond to the strain ΔcsgA_csgA_his induced rhamnose, after 5 hours and 24 hours of induction. The antibodies bind specifically to the His-tag of proteins, which did not bind to the ΔcsgA_csgA samples, line 1, 2, 5 and 6. Therefore it can be concluded that the strain ΔcsgA_csgA_his produced his-tagged CsgA proteins. The effect of rhamnose was visible when compared to the samples with ΔcsgA_csgA_his without rhamnose induction, lane 3 and 7. There are no bands visible, therefore rhamnose induces the production of CsgA_His. We can conclude with experiment 1 that the antibodies bind specifically to CsgA_His proteins. The CsgA_His proteins are only present when induced with 0.5% rhamnose.</P><P class="lead"><B>Experiment 2: Quantitative analysis of the internal CsgA_His protein.</B></P><P class="lead">With experiment 1 we proved that the antibodies bind to CsgA_His proteins. The CsgA_His strains produced CsgA_His after induction with 0.5% rhamnose. We executed experiment 2 to determine the internal concentration of CsgA_His proteins, by comparing the bands with the bands of known concentrations of CsgA_His proteins.</P><FIGURE><FIGCAPTION>Figure 12: Membrane after western blotting; samples of experiment 1 (Right to left).</FIGCAPTION></FIGURE><P class="lead">As seen in figure 12, the bands of the calibration curve were visible. The bands of the ΔcsgA_csgA_His were not visible. We still wanted to know the correlation between the thickness of the band, and the concentration of internal CsgA_His. We solved this by comparing the band in lane 8 from figure 11B with the bands seen in figure 11A, to acquire an approximation of the total concentration. The bands in lane 4, and 5 (right to left) were too saturated to measure with the Typhoon. The band in figure 2 was thinner than the two highest concentrations, therefore we removed the last two points from the calibration line.</P><P class="lead">The trend line of the concentration (in µM) versus intensity has a formula of:</P><P class="lead text-center">Intensity = 509067 · [concentration] + 792496</P><P class="lead">We calculated from figure 11B that the average intensity of our samples 2.3*10^6 was. We used the calibration curve (figure 13) we made with the dilution series to determine the average CsgA-His amount per well to be 2.75 μM. The amount of sample we loaded on the gel was 10 µl, therefore the amount of CsgA_His protein in the lane 2.7510-5µmol.</P><FIGURE><FIGCAPTION>Figure 13: Calibration curve of csgA concentration (µM) versus intensity.</FIGCAPTION></FIGURE><P class="lead">The cell concentration was set to an OD600 of 10 at the beginning of the experiment, which corresponds to 8109 cells/ml. 10 µl of this suspension was loaded per gel slot. With the number of cells and the amount of CsgA per slot known we were able to calculate the number of CsgA proteïns per cell: 2.06105 molecules/cell.</P></DIV></DIV></DIV></DIV><H2>Conclusions and Future Directions</H2><P class="lead">With the experiments described previously we achieved biofilm formation with our harmless lab Escherichia coli strains. Also, we managed to limit this expression only when rhamnose (a metabolizable inductor) is supplied. In overall, these results show a promising genetic construct to be used for bio-printing purposes. However, the experiments conducted in that part of the project could not really determine the exact effect of the induction level (i.e. concentration of rhamnose) in the biofilm formation. So, we would like to advise future researchers in this field to look for more detailed and complex experiments that could give information about the biofilm structure differences in time. </P><P class="lead">Next to the linking experiments, a step further in biofilm design would be to endow our strains with surface-attachment capability. This property will be analysed in the next part of our project, Affinity Tags.</P></DIV></DIV><DIV id="grey"><DIV class="jumbotron"><DIV class="container"><H1>Affinity Tags</H1><P class="lead text-muted">In this section we worked in the characterization of strains that display special affinity to certain surfaces. </P></DIV></DIV><DIV class="container"><H2>Introduction and Motivation</H2><P class="lead">In our aim to develop a fully customizable biofilm formation process, we wanted to improve their adhesive properties and make them stick to any possible surface. To achieve this goal, we modified the CsgA protein (monomer of the curli nanowire) and added affinity tags to it. First of all, we thought about biofilms that naturally grow in our mouths. These biofilms are a major issue in testing antibacterial products like toothpastes, so we thought we could contribute to that process by adding an affinity tag for the dental surface.  Moreover we also designed other affinity tags of interest. The complete list is shown in Table 6:</P><FIGURE><FIGCAPTION>Table 6. Summary of the affinity tags added to the CsgA molecule in order to give a specific surface attachment</FIGCAPTION></FIGURE><P class="lead">In the following section, we have analyzed the biofilm-making ability of the strains carrying the plasmids containing the modified csgA sequences. Also, for the HA tag (hydroxyapatite), we have designed an experiment for evaluating the attachment specificity of the modified amyloid to the dental surface.</P><DIV class="panel-group" id="accordionproject2" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="heading12"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordionproject2" href="#collapse12" aria-expanded="false" aria-controls="collapse12">
Microtiter Plate Assay for ring-shaped, curli dependent biofilm formation using differently tagged CsgA proteins
          </A></DIV><DIV id="collapse12" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading12"><DIV class="panel-body"><H3>Background</H3><P class="lead">The bacteria that we engineered for the project are capable of producing curli proteins after induction with L-rhamnose. We assessed whether our CsgA-producing bacteria can make a ring-shaped biofilm by adapting the protocol from Zhou et al. (2013) using crystal violet for dying the biofilm-making bacteria. However, now we would like to know if the cells that we engineered to have a special affinity for different surfaces can also make biofilms. For that reason, we repeated the assay with crystal violet for the different strains that produce CsgA but contained in a different plasmid.</P><H3>Methodology</H3><P class="lead">In the experiment, our CsgA-producing strain of E. coli is induced at a high (0.5% w/v), low (0.2% w/v) and no (0% w/v) concentration of L-rhamnose. Furthermore, csgA-deficient bacteria transformed with an empty plasmid (pSB1C3) are also tested and used as a control (Table 7). All the experiments were carried out in triplicates. A summary of the samples used for the experiment is also shown in Table 7:</P><FIGURE><FIGCAPTION>Table 7. Summary of the samples analysed after the cells grew on them</FIGCAPTION></FIGURE><P class="lead">The cultures of all the transformed strains were grown and induced in a 96-well plate. 40 hours after the induction, the samples were washed with water (in order to disregard non-attached, planktonic cells) and dyed with crystal violet. Finally, the wells were filled with ethanol so all the content can dissolve in the liquid phase and be measured in the plate reader in triplicates.</P><H3>Results and discussion</H3><P class="lead">The absorbance at 590 nm of wavelength is measured for all the samples, obtaining the following results displayed as bar plots (figure 14, 15, 16, 17, and 18):</P><FIGURE><FIGCAPTION>Figure 14. Microtiter Plate Assay results for testing biofilm formation. All the measurements were conducted in triplicates.  CTRL0, CTRL2 and CTRL5 are E. coli K-12 MG1655 PRO ΔcsgA ompR234 cells with pSB1C3, used as a control. CSGARFP0, CSGARFP2 and CSGARFP5 are E. coli K-12 MG1655 PRO ΔcsgA ompR234 cells expressing a plasmid that contains csgA under an inducible promoter and RFP under a constitutive promoter. The termination of 0, 2 and 5 denotes the induction with no rhamnose (0), 0.2% w/v (2) and 0.5% w/v (5).</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 15. Microtiter Plate Assay results for testing biofilm formation. All the measurements were conducted in triplicates.  CTRL0, CTRL2 and CTRL5 are E. coli K-12 MG1655 PRO ΔcsgA ompR234 cells with pSB1C3, used as a control. HA0, HA2 and HA5 are E. coli K-12 MG1655 PRO ΔcsgA ompR234 cells expressing a plasmid that contains csgA attached to the hydroxyapatite tag under an inducible promoter. The termination 0, 2 and 5 denote the induction with no rhamnose (0), 0.2% w/v (2) and 0.5% w/v (5).</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 16. Microtiter Plate Assay results for testing biofilm formation. All the measurements were conducted in triplicates.  CTRL0, CTRL2 and CTRL5 are E. coli K-12 MG1655 PRO ΔcsgA ompR234 cells with pSB1C3, used as a control. HIST0, HIST2 and HIST5 are E. coli K-12 MG1655 PRO ΔcsgA ompR234 cells expressing a plasmid that contains csgA attached to a 6xHis tag under an inducible promoter. The termination 0, 2 and 5 denote the induction with no rhamnose (0), 0.2% w/v (2) and 0.5% w/v (5).</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 17. Microtiter Plate Assay results for testing biofilm formation. All the measurements were conducted in triplicates.  CTRL0, CTRL2 and CTRL5 are E. coli K-12 MG1655 PRO ΔcsgA ompR234 cells with pSB1C3, used as a control. MFP30, MFP32 and MFP35 are E. coli K-12 MG1655 PRO ΔcsgA ompR234 cells expressing a plasmid that contains csgA attached to the mfp3 peptide under an inducible promoter. The termination 0, 2 and 5 denote the induction with no rhamnose (0), 0.2% w/v (2) and 0.5% w/v (5).</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 18. Microtiter Plate Assay results for testing biofilm formation. All the measurements were conducted in triplicates. CTRL0, CTRL2 and CTRL5 are E. coli K-12 MG1655 PRO ΔcsgA ompR234 cells with pSB1C3, used as a control. MFP50, MFP52 and MFP55 are E. coli K-12 MG1655 PRO ΔcsgA ompR234 cells expressing a plasmid that contains csgA attached to the mfp5 peptide under an inducible promoter. The termination 0, 2 and 5 denote the induction with no rhamnose (0), 0.2% w/v (2) and 0.5% w/v (5).</FIGCAPTION></FIGURE><P class="lead">As we also observed in the crystal violet assay for the plasmid containing the csgA gene, the sample that can also express RFP as a reporter gene is also able to make a biofilm. That confirms that this strain can be used for the experiments of characterization of the 3D printing, as the reporter gene expression is not interfering in the biofilm-making ability. </P><P class="lead">On the other hand, the tagged CsgA proteins seem to have biofilm-making capability despite having a peptidic modification in the structure. To confirm that there is a real change between the analysed samples and the empty plasmid control, a significance analysis was performed for α=0.05 (Table 8.).</P><FIGURE><FIGCAPTION>Table 8. Significance analysis of the samples analysed, with a significance α value of 5%. All the samples but the 0.5% induction of CSGARFP5 display a significant difference when compared to the empty plasmid homolog sample (CTRL). </FIGCAPTION></FIGURE><P class="lead">The significance analysis shows how all the samples with a modified CsgA can efficiently create a biofilm, when compared with an empty plasmid control (i.e. without csgA expression). The sample with CsgA and RFP as reporter gene, when induced with 0.5% w/v of rhamnose, shows no significant difference with an alpha value of 0.05. However, if we look at the p-value (0.0503) we can see that it is actually really close to a 5% of significance. So, the variation could be due to the manipulation.</P><P class="lead">We could conclude that the tags added to the CsgA molecule do not have a negative effect on biofilm formation, but also that the rhamnose concentration used for induction do not have a remarkable effect for such a long induction. Now that the biofilm capability of these strains is proved, proper function tests for the affinity tags are required. With that, we could confirm that we have generated biofilm with special affinity for certain surfaces.</P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="heading22"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordionproject2" href="#collapse22" aria-expanded="false" aria-controls="collapse22">
 Attachment assay for testing biofilm adhesion strength on hydroxyapatite using Bos taurus dental parts
          </A></DIV><DIV id="collapse22" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading22"><DIV class="panel-body"><H3>Background</H3><P class="lead">In our research for a highly representative biofilm that could be used for testing products, we thought about the main surfaces where a non-desirable biofilm can be attached. The first structure that came into our minds was the dental cavity; there, biofilms can attach to the tooth cover and create strong and resistant biofilms (Kidd, E.A., et al, 2004). </P><P class="lead">In our research about the dental surface we found that the external part that protects the inner tooth fragments is the enamel. Hydroxyapatite (Ca10(PO4)6(OH)2), also known as bone mineral, is a mineral compound that represents the major inorganic part of the dental enamel and dentin (Staines,M. et al, 1981). Thus, we looked forward to a way to make our biofilm attach better into a hydroxyapatite-covered surface.</P><P class="lead">In the end, we found that our homemade biofilm could be improved with surface-specificity just by adding a particular tag attached to our amyloid protein, CsgA (Roy,M. et al, 2008). This tag, successfully added to our csgA construct and cloned in our vehicle strain, does successfully make biofilms when induced with rhamnose. However, we thought that the best way to prove that it can resemble a real mouth-attached biofilm is testing it with teeth!</P><FIGURE><FIGCAPTION>Figure 19. Teeth from a cow (Bos taurus) used in the experiment for testing the hydroxyapatite peptide tag</FIGCAPTION></FIGURE><H3>Methodology</H3><P class="lead">For the experiment aiming to prove that our biofilm can be strongly linked to the enamel surface, we managed to get some teeth from a cow (Bos taurus) (figure 19.).  We cut them in small pieces of similar size (~1cm3) and place them in a 96-well plate. Afterwards the well was filled with 5 mL of cell culture, induced and incubated for 40 hours at room temperature, without shaking. The strains used for the investigation are summarized in table 9:</P><FIGURE><FIGCAPTION>Table 9.  Summary table of the strains used for the experiment of hydroxyapatite attachment.</FIGCAPTION></FIGURE><P class="lead">Once the dental pieces had enough time to be covered with specific curli-generating cells, we designed two different protocols for measuring the efficiency of the affinity tag. In both cases, the cells were expressing RFP (red fluorescent protein) under a constitutive promoter. Thus, the number of cells that remained attached to the surface could be calculated out of the fluorescence intensity.</P><P class="lead">We tried to measure the remaining cells attached to the dental surface after submerging it in water for 2 seconds. The first approach included a direct measurement technique: measuring RFP presence using a Typhoon ® Fluorescence Scanner (figure 20).</P><FIGURE><FIGCAPTION>Figure 20. Typhoon ® Fluorescence reader, used for measuring the remaining RFP signal after rinsing the tooth samples with water. This way, the differences in the attachment can be measured.</FIGCAPTION></FIGURE><P class="lead">On the other hand, we also measured the cells attached to the tooth by putting the samples in an ethanol (70%) solution for 5 minutes. Consequently, the cells will detach and float in the supernatant. When we measure the fluorescence intensity of this ethanol supernatant, we will be able to discriminate if there is actually a difference in the attachment properties of tagged and untagged cells. Detailed information of the nomenclature used in the figures of this section can be found in table 10.</P><FIGURE><FIGCAPTION>Table 10. Summary of the samples analysed after the cells grew on them</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 21.  Dental samples used in the experiment of hydroxyapatite affinity. From left to right, the samples are HA+, HA-, CSGA+, CSGA-, PSB+, PSB-. All the tooth fractions are approximately of 1 cm3. </FIGCAPTION></FIGURE><H3>Results and discussion</H3><P class="lead">We tested the RFP presence in the tooth samples (figure 21) with a fluorescence scanner, obtaining the following results (figure 22):</P><FIGURE><FIGCAPTION>Figure 22. Fluorescence scanning of the teeth samples with an excitation wavelength of 532 nm and emission detection at 610 nm. The samples analysed were HA+ (top left), HA- (top centre), CSGA+ (top right), CSGA- (bottom left), PSB+ (bottom centre) and PSB- (bottom right). </FIGCAPTION></FIGURE><P class="lead">The results of this experiment showed a strong background from the samples, meaning that probably the teeth display some kind of self-fluorescence. Consequently, the results of this test cannot be considered for proving the affinity to hydroxyapatite of our biofilms.</P><P class="lead">Then we performed the second measurement assay, using a 96-well plate and the plate reader. The results are displayed in figure 23, and the parameters were confirmed using the supernatant of the CsgA producing cells with no ethanol treatment. </P><FIGURE><FIGCAPTION>Figure 23. Fluorescence read of the ethanol supernatant where the dental samples where incubated. The liquids were excited with 560 nm of wavelength and the emission length detected was 607, for RFP fluorescence detection. All the samples are measured in triplicates, and the error bars show their standard deviation.</FIGCAPTION></FIGURE><P class="lead">From the results shown above, we could conclude that our affinity tag for hydroxyapatite gives an actual difference when compared with the control samples, for significance level of 5%. Also, the statistical analysis confirms that there is a significant change between cells with the affinity tag and without it (Table 11).</P><FIGURE><FIGCAPTION>Table 11. Significance analysis of the samples analysed. All the samples but the empty plasmid control display a significant difference when compared to the blank. Thus, both CsgA and CsgA+HA tagged cells attach better to a dental surface than cells that do not express the amyloid curli. Furthermore, the difference of fluorescence between the cells expressing the tagged protein and the untagged is also significantly different, confirming the efficiency of the hydroxyapatite peptide tag.</FIGCAPTION></FIGURE></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="heading32"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordionproject2" href="#collapse32" aria-expanded="false" aria-controls="collapse32">
            SpyTag-SpyCatcher
          </A></DIV><DIV id="collapse32" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading32"><DIV class="panel-body"><P class="lead">Our initial plan was to, in addition to the His- and Hydroxyapatite-tag, also attach a SpyTag to the CsgA proteins. This idea was based on an article of the Joshi Lab on programmable biofilm-based materials (Nguyen, Botyanszki, Tay, &amp; Joshi, 2014). This SpyTag can spontaneously form covalent bonds with a protein called SpyCatcher. The SpyCatcher protein can be attached to other proteins to form a fusion protein. This way the fusion protein can be immobilized on the SpyTag. By attaching the SpyTag to the CsgA protein, the SpyCatcher can indirectly bind to the nanowires formed by CsgA. This way fusion proteins can be immobilized on the nanowires and a functionalized biofilm can be created. (Nguyen et al., 2014)</P><P class="lead">To prove that this mechanism works, we envisioned to make a fusion protein of SpyCatcher and amylase, a protein that catalyses the hydrolysis of starch. Amylase is actively transported out of the cell and thereby a biofilm could be created that produces both the nanowires and the enzymes that will be immobilized on these nanowires. However, shortly after coming up with this idea, a paper was published on exactly this system. (Botyanszki, Tay, Nguyen, Nussbaumer, &amp; Joshi, 2015). This showed us that using the SpyTag/SpyCatcher system to immobilize enzymes on the nanowires was indeed deemed interesting.</P><P class="lead">To at least prove that this idea could work with our biobricks, we created a fusion protein of Blue Fluorescent Protein (BFP) with SpyCatcher (BBa_K1583113). The BFP could be easily assessed on its fluorescent properties to see whether it could still properly function and fold when attached to SpyCatcher. However, we experienced difficulties with the construction of both the CsgA_SpyTag biobrick and the BFP_SpyCatcher_His biobricks. Unfortunately CsgA_SpyTag contained a frameshift, that proved to be hard to remove and the arabinose induction of BFP_SpyCatcher_His also did not work as expected. Still, since we do very much like the idea of creating functionalized biofilms in this manner, we still wanted to explain this idea here and hope that future iGEM teams can successfully conduct this experiment.</P></DIV></DIV></DIV></DIV><H2>Conclusions and Future Directions</H2><P class="lead">These experiments represent a big step in biofilm customization, allowing researchers to create biofilm structures that bind specifically to certain surfaces. Nevertheless, this could be only used as an extra for 3D biofilm printing, as linking cells between them seems to be the limiting step for printing with our hardware prototype. </P><P class="lead">We have tested that the strains producing CsgA with the different affinity tags can still form biofilms despite having a peptidic modification in the CsgA structure. However, the affinity test has been performed for the hydroxyapatite (HA) tag and the polyhistidine (His) tag, meaning that the binding capabilities of the mfp3 and mfp5 tags still have to be proven.</P><P class="lead">Our initial plan was to, in addition to the His- and Hydroxyapatite-tag, also attach a SpyTag to the CsgA protein. This idea was based on an article of the Joshi Lab on programmable biofilm-based materials (Nguyen, Botyanszki, Tay, &amp; Joshi, 2014). The SpyTag can spontaneously form a covalent isopeptide bond with a protein called SpyCatcher. The SpyCatcher protein can be attached to other proteins to form a fusion protein. This way the fusion protein can be immobilized on the nanowire by means of the SpyTag. By attaching the SpyTag to the CsgA protein, the SpyCatcher can indirectly bind to the nanowires formed by the CsgA. This way fusion proteins can be immobilized on the nanowires and a functionalized biofilm can be created.(Nguyen et al., 2014)</P></DIV></DIV><DIV id="white"><DIV class="jumbotron"><DIV class="container"><H1>Printing Biofilm Layers Using Fluorescent Cells</H1><P class="lead text-muted">Our final goal in the project was the ordered printing of biofilm layers. So, in the following part we describe how we did that and how we achieved 3D printing of biofilms</P></DIV></DIV><DIV class="container"><H2>Introduction</H2><P class="lead">One of the goals of our project is to print biofilms with a defined structure. This could be applied in e.g. industrial processes in which the production pathway is divided over several organisms (Zhou et al. 2015). An example of such a process is given below (Figure 24). The orange substrate is converted by the blue bacterium to the purple intermediate. This purple intermediate is converted by the red bacterium to the green final product. This conversion can be done in two separate reactors, in which case purification steps are required in between the two reactors. </P><P class="lead">Another option is to do the conversion in one reaction. In this case the outflow will contain not only the end product, but also substrate and intermediates. When you would be able to optimally structure the two bacteria, the produced intermediates would be directly converted by the red bacteria, thereby minimizing the amount of intermediates in your outflow and thereby increasing your yield.</P><P class="lead">Also in the case that the purple intermediates inhibit their own production (feedback inhibition), structuring the bacteria and thereby minimizing the intermediate concentration could have a beneficial effect on the yield (Kiely, Regan, and Logan 2011).</P><FIGURE><FIGCAPTION>Figure 24: Industrial process with metabolic pathway divided among a microbial consortium. In this process the orange substrate is converted by the blue bacterium to the purple intermediate. This intermediate is converted to the green end product by the red bacterium. In the upper process, the bacteria are not structured and the outflow contains also substrate and intermediates next to the end product. In the bottom example the cells are structured to improve the conversion of the intermediate to the end product.</FIGCAPTION></FIGURE><P class="lead">To be able to make structured biofilms of different (types) of bacteria, it is important that we prove that we are able to structure the bacteria and that these bacteria stay in place.</P><P class="lead">To prove this we used cells producing the fluorescent proteins GFP and RFP, to resemble different strains in an industrial process. To to show that we can structure these cells and that they do not mix, we wanted to assess the distribution of the cells after printing with confocal microscopy (Figure 2).</P><FIGURE><FIGCAPTION>Figure 25: Layer formation experiment setup. To test whether we can produce structured layers of bacteria that do not mix, the above experiment will be conducted. Layers of cell producing GFP and RFP will be printed on top of each other and with confocal microscopy the spatial distribution of the cells will be assessed.</FIGCAPTION></FIGURE><DIV class="panel-group" id="accordionproject3" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="heading13"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordionproject3" href="#collapse13" aria-expanded="false" aria-controls="collapse13">
 Methodology
          </A></DIV><DIV id="collapse13" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading13"><DIV class="panel-body"><P class="lead">We have transformed TOP10 cells with the plasmids I27020 (GFP) and I13521 (RFP) (Table 12.). We grew cultures of these cells overnight in 5 mL LB with antibiotics. To obtain 500 μL of bio-ink, 1 mL of cells was spun down at 4000 rpm for 3 minutes and the supernatant discarded. We added 100 μL of sterile LB, resuspended the cells and then added 400 μL of 1% w/v sodium alginate. This mixture was vortexed to ensure that the cells are homogeneously distributed. The formation of the hydrogel by alginate and CaCl2 is discussed in more detail in the hardware section.</P><FIGURE><FIGCAPTION>Table 12. Strains used for the production of bio-ink</FIGCAPTION></FIGURE><P class="lead">To be able to visualize the cells, the bio-ink needed to be printed on a coverslip suitable for the spinning disk microscope instead of the usual agar plates (See hardware). We used plasma-cleaned coverslips, since the normal coverslips are very hydrophobic and cause liquids to form a bubble on top of the surface. Plasma cleaning removes the impurities and contaminants on the coverslip and negatively charges the surface, thereby rendering it hydrophilic. This way contact between the coverslip and the bio-ink is improved.</P><P class="lead">For printing the layers of bio-ink, we first made a line of the alginate with RFP cells on the coverslip (Figure 3). This was done in the first experiment (Layer formation) by extruding the bio-ink at a flow rate of 100 uL/min with the syringe pump. In the second experiment (3D-stack), we used a 1-10 μL pipet. After printing of the first layer, CaCl2 was added on top of the alginate to start the formation of the hydrogel. We discarded the excess of CaCl2 by tilting the coverslip and removing the liquid with a tissue. This makes the layers of alginate a bit thinner, which is beneficial for the image quality with the spinning disk microscope. The image quality improves because the shorter distance to the objective leads to a lower background signal.  The layers on top of this first layer we applied in the same manner.</P><FIGURE><FIGCAPTION>Figure 26. Plasma-cleaned coverslip containing three lines of bio-ink. Each bio-ink line consists of a layer of cells producing RFP and a layer of cell producing GFP on top. The irregular formation of the lines is due to the incomplete removal of CaCl2 after formation of the hydrogel of the first layer. As a result, the second layer partly diffuses away from the printed line before hydrogel formation.</FIGCAPTION></FIGURE><P class="lead">Normally, the bio-ink is printed on an agar plate containing CaCl2. When the bio-ink gets into contact with the agar, the CaCl2 reacts with the alginate to form the hydrogel. To be able to better represent the printing with the DIY printer, we also tried to first apply a layer of CaCl2 on the coverslip and print the layers of bio-ink on top of this. However, the results was that the alginate would float around in the CaCl2, and it was not possible to make several layers of bio-ink on top of each other in a structured way.</P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="heading23"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordionproject3" href="#collapse23" aria-expanded="false" aria-controls="collapse23">
Results
          </A></DIV><DIV id="collapse23" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading23"><DIV class="panel-body"><P class="lead"><B>Layer formation</B></P><P class="lead">We used the syringe pump to make four layers of the bio-ink, with cells containing RFP on the bottom and layers of cells containing GFP, RFP and GFP, respectively, on top of that.  This sample was imaged with a spinning disk microscope at 488 nm and 561 nm, for the excitation of respectively GFP and RFP (Figure 27).</P><FIGURE><FIGCAPTION>Figure 27. Spatial distribution in z-direction of cells expressing GFP (green) and RFP (red). Layers of hydrogel containing GFP (layer 2 &amp; 4) or RFP (1 &amp; 3) cells were created on a plasma-cleaned coverslip.  The first and third layer do indeed contain RFP cells (1.2 &amp; 3.2), while not containing GFP cells (1.1 &amp; 3.1). The second and fourth layer contain GFP cells (2.1 &amp; 4.1), while not containing RFP cells (2.2 &amp; 4.2). Reduced visibility of layer 3 and 4 is due to increased distance from objective, which results in a higher background signal. Imaging was performed using spinning disk confocal microscope with a 100x objective. Images were captured by excitation at 488 nm for GFP and 561 nm for RFP, both at an intensity of 9.7% and using an exposure time of respectively 200-ms and 50-ms.</FIGCAPTION></FIGURE><P class="lead">As can be seen in figure 27, the first layer does contain red cells (1.2), but does not contain green cells (1.1). The same holds for layer three (3.1 &amp; 3.2). Layer two and four both contain green cells (2.1 &amp; 4.1), while they both do not contain red cells (2.2 &amp; 4.2). The visibility of the cells in layer three and four is reduced due to their increased distance from objective, which results in a higher background signal. This spatial distribution of the cells proves that we can make distinct layers of cells.</P><P class="lead"><B>3D-stack</B></P><P class="lead">We proved that we are able to make distinct layers of cells, but we did not know yet whether there was a mixture of the different cells at the interface between two layers, or whether they are clearly separated. Also we did not know yet how thick the layers were. To test this, we made a new sample with two layers and took images with the spinning disk microscope in 74 planes over a distance of 146 μm at the same two wavelengths. Of these planes, 34 were used to make a 3D stack of the sample (Figure 28ab). As a control to show that the hydrogel formation assists in layer formation, layers of red and green cells were printed without alginate and CaCl2. Images of 76 planes over a distance of 37.5 μm with excitation at 488 nm and 561 nm were taken and of these 36 were used to make a 3D stack (Figure 28cd).</P><FIGURE><FIGCAPTION>Figure 5.	Layer formation of GFP-expressing cells (green) and RFP-expressing cells (red).
a. Layers of hydrogel containing RFP and GFP cells were created on a plasma-cleaned coverslip. The z-stack consists of 34 planes and was created by stacking and interpolation of these planes. The bottom layer of around 29 μm in height contains cells producing RFP and the layer on top of around 34 μm in height contains cells producing GFP. b. Same sample as a, but seen from below. The red cells are in front, while the green cells are further in the background. Imaging of a. &amp; b. was performed using spinning disk confocal microscope with a 100x objective. Images were captured by excitation at 488 nm for GFP at 9.7% intensity and 561 nm for RFP at 3.2% intensity using an exposure time of respectively 200-ms and 50-ms.
c. Layers of LB containing RFP and GFP cells were created on a plasma-cleaned coverslip. The z-stack consists of 36 planes and was created by stacking and interpolation of these planes. d. Same sample as c, but seen from below. Imaging of c. &amp; d. was performed using spinning disk confocal microscope with a 100x objective. Images were captured by excitation at 488 nm for GFP at 16.1% intensity and 561 nm for RFP at 9.7% intensity using an exposure time of respectively 5-ms and 1-ms.</FIGCAPTION></FIGURE><P class="lead">This image shows that two distinct layers are formed when bio-ink was used and mixing between the layers is almost negligible. The height of the red layer is around 29 μm and the height of the green layer is around 34 μm depending on the position in the layer. The sample without alginate shows that without the use of bio-ink, the cells are not structured and a mixture of red and green cells is formed. This proves that the hydrogel assists in layer formation and that we can use this system for printing the biofilm; the cells can be printed in a structured way and no mixing occurs, even at the interfaces between layers.</P></DIV></DIV></DIV></DIV><H2>Conclusions and Future Directions 
<P class="lead">We started these experiments with the question whether we can print biofilms in a defined structure. To assess whether we can do this, we wanted to show that we can structure the cells and that they do not mix. The first experiment, layer formation, showed that we can indeed structure the cells in layers. In the second experiment, 3D-stack, we additionally proved that these layers are separated from each other and do not mix. Therefore we conclude that we can print bacteria in a defined structure.</P><P class="lead">The iGEM team Groningen already set the foundation to the future, where our developed procedure could be tested on different species of bacteria, to assess whether they can also be printed in layers. Supported by interest from industry, further experiments with an applied focus could prove very interesting.</P></H2><H1>References</H1></DIV></DIV><DIV class="container"><P>Kiely, Patrick D, John M Regan, and Bruce E Logan. 2011. “The Electric Picnic: Synergistic Requirements for Exoelectrogenic Microbial Communities.” Current opinion in biotechnology 22(3): 378–85. http://www.ncbi.nlm.nih.gov/pubmed/21441020 (September 17, 2015).
</P><P>Zhou, Kang, Kangjian Qiao, Steven Edgar, and Gregory Stephanopoulos. 2015. “Distributing a Metabolic Pathway among a Microbial Consortium Enhances Production of Natural Products.” Nature Biotechnology 33(4): 377–83. http://www.nature.com/doifinder/10.1038/nbt.3095 (January 5, 2015).</P><P>Kidd, E. A. M., &amp; Fejerskov, O. (2004). What constitutes dental caries? Histopathology of carious enamel and dentin related to the action of cariogenic biofilms. Journal of dental research, 83(suppl 1), C35-C38.</P><P>M. Staines, W. H. Robinson and J. A. A. Hood (1981). &quot;Spherical indentation of tooth enamel&quot;. Journal of Materials Science 16 (9): 2551–2556.</P><P>Roy, M. D., Stanley, S. K., Amis, E. J., &amp; Becker, M. L. (2008). Identification of a Highly Specific Hydroxyapatite‐binding Peptide using Phage Display. Advanced Materials, 20(10), 1830-1836.</P><P>Zhong, C., Gurry, T., Cheng, A. A., Downey, J., Deng, Z., Stultz, C. M., &amp; Lu, T. K. (2014). Strong underwater adhesives made by self-assembling multi-protein nanofibres. Nature nanotechnology.</P><P>Chen, A. Y., Deng, Z., Billings, A. N., Seker, U. O., Lu, M. Y., Citorik, R. J., ... &amp; Lu, T. K. (2014). Synthesis and patterning of tunable multiscale materials with engineered cells. Nature materials, 13(5), 515-523.</P><P>Barnhart, M. M., &amp; Chapman, M. R. (2006). Curli biogenesis and function. Annual review of microbiology, 60, 131.</P><P>Wegerer, A., Sun, T., &amp; Altenbuchner, J. (2008). Optimization of an E. coli L-rhamnose-inducible expression vector: test of various genetic module combinations. BMC biotechnology, 8(1), 2.</P><P>Earl Boysen and Nancy C. Muir (2015) See at the Nano Level with Electron Microscopy, Available at: http://www.dummies.com/how-to/content/see-at-the-nano-level-with-electron-microscopy.html (Accessed: September 2015).</P><P>Rowley, J. A., Madlambayan, G., &amp; Mooney, D. J. (1999). Alginate hydrogels as synthetic extracellular matrix materials. Biomaterials, 20(1), 45-53.</P><P>Nguyen, P.Q., Botyanszki, Z., Tay, P.K., Joshi, N.S.. (2014). Programmable biofilm-based materials from engineered curli nanofibres. Nat Commun, 17(5), 4945.</P><P>Taylor JD, Zhou Y, Salgado PS, Patwardhan A, McGuffie M, Pape T, Grabe G, Ashman E, Constable SC, Simpson PJ, Lee WC, Cota E, Chapman MR, Matthews SJ. (2011). Atomic resolution insights into curli fiber biogenesis. Structure. 19(9):1307-16.</P><P>Dueholm MS1, Nielsen SB, Hein KL, Nissen P, Chapman M, Christiansen G, Nielsen PH, Otzen DE. (2011). Fibrillation of the major curli subunit CsgA under a wide range of conditions implies a robust design of aggregation. Biochemistry. 4;50(39):8281-90.</P></DIV></DIV><DIV class="container"><DIV class="footer-left"><DIV class="row"><H2 style="text-align:center">Contact Details</H2></DIV><DIV class="row"><DIV class="col-xs-6 col-sm-5 col-md-4 col-lg-4"><P class="lead thick"> Address: </P></DIV><DIV class="col-xs-6 col-sm-7 col-md-8 col-lg-8"><P class="lead"><SPAN class="text-muted">Faculty of Applied Sciences</SPAN></P><P class="lead"><SPAN class="text-muted">Room F109</SPAN></P><P class="lead"><SPAN class="text-muted">TU Delft</SPAN></P><P class="lead"><SPAN class="text-muted">Lorentzweg 1</SPAN></P><P class="lead"><SPAN class="text-muted">2628 CJ Delft</SPAN></P><P class="lead"><SPAN class="text-muted">The Netherlands</SPAN></P></DIV></DIV><DIV class="row"><DIV class="col-xs-6 col-sm-5 col-md-4 col-lg-4"><P class="lead thick"> Phone: </P></DIV><DIV class="col-xs-6 col-sm-7 col-md-8 col-lg-8"><P class="lead"><SPAN class="text-muted">+31 15 278 18 62</SPAN></P></DIV></DIV><DIV class="row"><DIV class="col-xs-6 col-sm-5 col-md-4 col-lg-4"><P class="lead thick"> Email: </P></DIV><DIV class="col-xs-6 col-sm-7 col-md-8 col-lg-8"><P class="lead"><SPAN class="text-muted">tudelft.igem@gmail.com</SPAN></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>