Team:TU Delft/Notebook
Home
Achievements
Prizes and Nominations
Giant Jamboree
Medal Requirements
Project
Description
Strains
Linking Cells
Affinity Tags
Printing Biofilm Layers
References
Parts
Overview
Parts
Part Collection
Notebook
Work Space
Safety
Day Notes
Protocols
Hardware
Overview
Making 3D Biofilms
The Biolinker
Design and Integration
DIY Manuals
Protocols
Modeling
Overview
Kinetic Modeling
Curli Formation
Biofilm Strength
Time for Printing
Policy&Practice
Overview
Regulations
Ethics
Experts' Opinions
iGEM Community
Biolink P&P Tool
Business Plan
Outreach
Collaboration
Overview
Testing Biolinker
Dutch iGEM Teams Meeting
Cooperations
Attributions
Team Members
Advisors
Team Work Attributions
Media
Sponsors
Acknowledgements
Achievements
Project
Parts
Notebook
Hardware
Modeling
Practices
Collaboration
Attributions
InterLab
Work Space
Safety
Day Notes
Protocols
Prizes and Nominations
Giant Jamboree
Medal Requirements
Team Members
Advisors
Team Work Attributions
Media
Sponsors
Acknowledgements
Description
Strains
Linking Cells
Affinity Tags
Printing Biofilm Layers
References
Overview
Parts
Part Collection
Overview
Making 3D Biofilms
The Biolinker
Design and Integration
DIY Manuals
Protocols
Overview
Testing Biolinker
Dutch iGEM Teams Meeting
Cooperations
Overview
Regulations
Ethics
Experts' Opinions
iGEM Community
Biolink P&P Tool
Business Plan
Outreach
Results
Protocols
Overview
Kinetic Modeling
Curli Formation
Biofilm Strength
Time for Printing
Notebook
Work Space
Our lab and office. Our new home for the summer.
About our lab
The iGEM lab, inside the Applied Sciences building of the TU Delft, has been the place where our science took place. It has the certification for ML-1 experiments, which was enough for our experiments.  Here we learned and improved our skills in cloning and manipulating biological materials. We also tried to apply our ideas from paper to the real world. All in all, we discovered the world of synthetic biology from the inside.
About our office
Our office has been our meeting point and our dry-lab zone during the project. Here, amazing talks about biology, engineering and philosophy took place. And here we debated about science and lab topics. To sum up, the office that you can see in the picture in the left has been our home this summer: the place where we built the project, but also where we built our team spirit.
Safety
Lab and Project Safety
Safety in our Lab
Our lab is classified as Level 1 of biosafety. This is the lowest safety level, meaning that our experiments involve low or no risk. The biological materials used for our experiments are handled in an open bench All the members of the team have received safety training, including:
Introduction to sterile working
RNA handling
Laser safety
Microscopy training
ML-1 safety test completion
Chemical safety training
General safety information, regarding contact persons and locations
Computer infrastructure
The safety of our experiments was supervised by Susanne Hage (Safety Manager of the TNW faculty) and Marinka Almering (Safety officer of the TU Delft). The research has been conducted with respect to the regulations of biosafety for The Netherlands, that can be found here
Safety in our Project
Our harmless bacteria produces curli subunits in order to make an inducible biofilm. The curli is natively produced by several safe strains, like the TOP10 used for transformations. Our strain contains a CsgA deletion and can only produce the curli when transformed with the designed plasmid and induced. This part of the project is based on sufficient scientific publications and previous iGEM information and parts. Also, one assay is designed to test the efficiency of the biofilm created carrying a specific affinity tag. For the assay teeth pieces were used, taking care of all the existing regulations and safety recommendations.
The chassis organism used for our project is a modified strain of the lab organism Escherichia coli K-12. It is called E. coli K-12 MG1655 PRO ΔcsgA ompR234.
In addition to our chassis organism, Escherichia coli Top 10 cells were used for cloning procedures
A pig tooth is used for testing our affinity application to hydroxyapatite. This will be arranged following the lab regulations, and the tooth will be provided following all the existing regulations. We will handle the part in the ML1 lab, and using gloves while working with the part. Furthermore, the teeth are cleaned properly before the use. The use of that samples is regulated by the Ministry of Economic affairs, Agriculture and Innovation of the Government of The Netherlands. The waste of all animal byproduct is designated as Specific Animal Waste (following Euralcode 180102). Later on, it is transported to the ZAVIN in Dordrecht for its destruction. For further information, you could contact the Residual Management Department of TU Delft (ReststoffenBeheer@tudelft.nl)
One of the problems to be faced with this project is the production of csgA as biofilm-maker unit. It has been reported previously as a virulence factor from harmful strains of E.coli, because the amyloid make the attachment to surfaces easier. However, we implemented two safety points for minimizing the risk of plasmid transmission to pathogen strains. First of all, the gene which codifies for csgA is preceded by a Rhamnose promoter. That prevents curli (csgA+csgB polymer) to be produces when no inductor is applied to the system. Furthermore, as rhamnose is a metabolizable inductor the gene expression could be avoided just by stopping the rhamnose supply. On the other hand, we aim our project for product testing or industrial production. So, the good laboratory practices and good manufacturing practices in these two fields should avoid any safety issue.
If our project could be fully developed into a real product, it would aim the following fields:
Factories
Lab applications
Consumer products
Test products to be used in the human body
Test products on food
We would like to encourage every interested iGEM team or researcher in general to contact us for further details about the safety in our lab and project!
Day Notes
InterLab Study – Measurement Award
June
19/06/2015
We obtained the plasmids for the InterLab study (containing 3 promoters and a GFP marker) from the iGEM distribution kit. 10 µL Milli-Q was added to dissolve the plasmids. After 5 minutes at room temperature, the samples were placed on ice until the transformation took place.
After collecting the genes from the distribution kit, Top10 competent cells were taken from the storage at -80 ºC and the transformation started using the heat-shock protocol (2). The result was plated in different petri dishes as explained below:
For the transformation, two types of cells were tested; diluted (black dotted) and non-diluted (no dot). After the incubation, we have plated the cells. However, we did not have any control strains (only pUC19). Petri dishes were left at room temperature (21ºC) during the weekend.
22/06/2015
We checked the plates made on 19/06/2015. The growth on the pUC19 control plate confirms the efficiency of the transformation protocol. Furthermore, it was clear that the cells are transformation-competent.
The lack of colonies in the LB+CAM plates is a consequence of a mismatch between the screening antibiotic used (CAM) and the actual antibiotic resistance present in the plasmids J23101, J23106 and J23117. The antibiotic resistance on the plasmids is AMP not CAM. Consequently, we have carried out new experiments using ampicillin (AMP) plates later.
In the plate with LB and CAM, using the cells transformed with the GFP marker plasmid (I13504), we couldn’t see any colonies early in the morning (probably because we kept the plates at room temperature). However, after incubation at 37ºC during the day, some colonies grew enough to be noticed, meaning a good transformation with the mentioned plasmid.
23/06/2015
As we described on 22/06/2015, we performed another transformation (2) for J523101, J523106, J523117 and I13522. We also picked colonies from the I13504 plate, which we inoculated in 5mL of LB and put at 37ºC and 200 rpm
24/06/2015
We performed plasmid isolation of the inoculated I13504 colonies. In order to do this, the Promega miniprep unit was used. The procedure of the kit can be found online (4) We stored the obtained plasmid solution at - 20°C (30 µl).
PCR of I13504 primer, PCR protocol (5) was used for this procedure. To check the PCR reaction, an agarose gel (6) was made. The PCR product should be at the level of 2.7 KB.
(4): Protocol related to Promega miniprep unit for plasmid isolation.
(5): Protocol for doing a PCR reaction.
(6): Protocol for running an agarose gel.
25/06/2015
Check the plates made two days ago. The result is shown in the table below:
From the I13522 – CAM, we picked from both plates a colony. Also the J23117 gave colonies which has been used for further experiments. J23101 and J23106 we have decided to redo, because the transformation failed. The picked colonies were incubated in 8 mL LB; I13522 in 1:100 AMP and J23117 in 1:1000 CAM.
Transformation (2) of J23101 (1 µl, 0.5 mL of LB instead of 1 mL) and J23106 (1 µl, 0.5 mL of LB instead of 1 mL). After transformation they were plated on LB+AMP petri dishes. It might be that there has been sample confusion during incubation on ice for 2 minutes occurs (J23106 with J23101), should be confirmed later with sequencing.
26/06/2015
Results of the above transformation: J23101 3 purple colonies; J23106 6 purple colonies with small colonies around them. The plates were stored in the fridge.
29/06/2015
Plasmids were isolated for I13522 (16.6 ng/µL), J23117 (16.9 ng/µl). Cell cultures were created for the 100 µL plates from 25/06/2015 in 8mL LB+ 80µL 100 mg/mL AMP solution. No growth was observed indicating problems with the transformation.
July
01/07/2015
Restriction and ligation Interlab promoter (J23117) and GFP (I3504). Transformation of
- J23117/GFP product – plated on LB-AMP;
- J23101 – plated on LB-AMP;
- J23106 – plated on LB-AMP;
I3504 was restricted with EcorI/XbaI (30µl). In this case the same situation, only a concentration of 7.57 ng/µl. Actually, more plasmid was required, but there is only 30 µl of volume after the plasmid isolation. The concentration was 16.6 ng/µl. J23117 was restricted with EcorI/SpaI (30µl). In this case the same situation as above. Concentration was equal to 16.9 ng/µl. The reaction mixtures for restriction were as below and it was incubated at 37°C:
Selected colonies from - I13504 LB-CAM and J23117 LB-AMP. The selected colonies were incubated in 8 mL of LB containing CAM or AMP and they were grown at 37°C at 210 rpm. A gel was made, the running was performed at V is 100 mL and for 60 minutes.
02/07/2015
Plasmid isolation (Promega, protocol (4)) of I13505 and J23117. Different in this protocol were the facts that we do not use elution buffer but Milli-Q at 50°C (even if we work with small plasmids). The following concentration were obtained with the nanodrop machine.  I13504 – 63.19 ng/ µl;  J23117- 143.98 ng/ µl.
Therefore, the following restriction scheme, incubated at 37°C for 2 hours.
Transformation was performed with Top10 cells prepared by Ilja. The normal protocol of transformation was followed (2) for J23106, J23101 and pUC19 (as a control). For the transformation 1 µl of DNA was added. Another gel was performed.
03/07/2015
A gel purification was performed (Promega (7)) I13504 0,44g → 440 µl.
DNA content: I13522: 5,47 ng/ µl; Vector DNAL 20-100 ng; insert DNA; Ligase buffer (2 µl); Ligase (1 µl); H2O up to 20 µl.
Transformation (2) of - Interlab J23117 (1  µl); - Control TOP10 strain
Cells were plated duplicated and also one control was made.
06/07/2015
Results of the transformation (J23117) 1. Empty plasmid control (also growth, 100 colonies); 2. TOP10 + J23117 (growth, 100 colonies, small);
After the checking the plates, a small culture of 5 mL is made using a single colony from the obtained plates, the following cultures were made:
- TOP10 + J23101;
- TOP10 + J23106;
- TOP10 + J23117;
The antibiotics were added in a ratio of 1:1000 (thus in 5 mL, we have added 5µl). After cloning, we putted the 50 mL tubes at 37°C at 200 rpm.
FACS (250µL of K1583000) did not result in a relevant fluorescence signal in the construct. Therefore, we decided to do the transformation again. Thereby, we found out that we did not use the correct backbones for the interlab. I13504_pSB1A2; J23101_pSB1C3; J23106_pSB1C3; J23117_pSB1C3.
The backbone for our own construct is pSB4K5. For the transformation, we did use the protocol (2). Then we have grown the cells on the plates: I13504 and pUC19 on AMP, J23101, J23106 and J23117 at CAM and pSB4K5 at KAN.
07/07/2015
The plates of 06/07/2015 were checked.
Colonies were picked from each plate and added to 5 mL of LB and 5 µl of AMP, CAM, KAN (for pSB4K5) and furthermore the antibiotics that belong to the rest. The antibiotic added was added in a 1:1000 way (like shown with the example of KAN). Cultures were then putted at 37°C at 200 rpm for growing.
In order to let the colonies grow overnight, the 5 ml cultures were refilled with fresh LB to 20 mL (with 20µl of antibiotics). Afterwards, the cultures were putted at 37°C overnight at 180 rpm.
08/07/2015
Cultures were taken from the 37°C shaker and we proceed to isolate the plasmids and store the cultures at -80°C. For this, the protocol for making -80°C stocks was followed (10). For this a cell pellet was used (pellet obtained after 10 minutes centrifugation at 2000 rpm), resolved in 0.5 mL LB and 0.5 Glycerol (80% solution).
Plasmid isolation was performed and the plasmid J23101, J23106 , J23117, I13504. After the protocol, the plasmids were stored at -20°C. However, the utilization of 50°C Milli-Q water was not performed during the plasmid isolation, which could be a reason for the failure during the process which is shown later on.
With the nanodrop measurement of the isolated plasmids we determined the following concentrations of plasmids:
I13504: 13.85 ng/ µl;
J23101: 13.64 ng/ µl;
J23106: 31.60 ng/ µl;
J23117: 14.50 ng/ µl;
These yields are all very low. Therefore, we decided to start new cell cultures in order to perform the plasmid isolation again the day after. For this, other colonies only for the IL-study were taken.
I13504, the cells for this cell culture were taken from -80°C stock; J23101, J23106, J23117, the cells for these cell cultures were taken from the plate; These cells were grown overnight in 5 mL culture with 1:1000 solution of antibiotics (from 15:10h) at 37°C and 200 rpm.  (10) -80°C stock solution preparation.
09/07/2015
Plasmid isolation
Small changes were applied to the original protocol, according to Essengüls advice.  - 1.5 mL of all culture is centrifuged before the first step, in order to get rid of the LB media. Water is added (600 µl afterwards); - MQ water at >37°C is used instead of Elution buffer;
Minus 80°C stock
Ligation was done with I13504 (2) (I13504.2) because the concentration was the highest. The DNA ratio of vector: insert used was 1:1. It was calculated by using the Promega Math app that 25ng of vector was required. The insert size is smaller than the plasmid itself so the weight of insert used is less.
10/07/2015 (Friday)
13/07/2015 (Monday)
Plate results
Transformation
The plasmid I20270 is the positive control which was present in the distribution kit 2015/3/8P. 10 µL of Milli-Q was added, left for 5 min before extracting. It was transformed in Top10 component cells and plated on a LB-KAN plate. Top10 Component cells were used as a negative control and plated on a LB plate. The plates had 100 and 400 µL of the cultures, and were left overnight at 37°C. One 50 mL tube with 5mL of pSB1C3_j23101 culture was also left overnight in at 37°C at 200 rpm.
14/07/2015 (Tuesday)
Plasmid isolation of J23101_pSBIC3 was done for IDT genes backbones. This was done by using 4,5mL overnight culture for isolation of plasmids, to increase the yield. The yield was determined by using the nanodrop, and the concentration was 75,13 ng/ µL.
A restriction was done of pSB4K5 & J23101_pSBIC3 with EcoRI-HF & PstI-HF. The volume was 9x normal to create large stock with a total volume of 450µL. 90 µL 6x loading dye was added to bring the final volume to 540 µL. The restriction was carried out for 3 hours at 37°C.  The restrictions were loaded on a large gel. Slots 12-20 were filled with J23101_pSBIC3 with 60µL in each slot.
The results of the plates of 13/07/2015 were as followed. The cells without a plasmid grew significantly, but the 100 plate with plasmid I20270 had only 8 small colonies that were left overnight, at 37°C, again to grow larger.
Fluorescence  was checked of the plates with J23101_I13504.2, J23106_I13504.2, J23117_I13504.2. The conclusion was that J23101 and J23106 were fluorescent in UV light, and J23117 was not. Therefore the ligation of J23117 with I13504 was repeated.
15/07/2015 (Wednesday)
Gel extraction was done on the gel with restricted pSB45K5 and pSB1C3.
Plasmid isolation (4) of the plasmids 101+504 and 106+504 has been performed with 1,5 mL of each culture. The following concentration were measured with the nanodrop measurement: 101 + 504: 109 ng/µl. The A260/280 was equal to 1.91 and 106 + 504: 163.85 ng/µl. The A260/280 was equal to 2.30.
In order to check the sequences, sequencing was performed with the isolated plasmids (J23101+I13504 and J23106+I13504). In order to prepare the samples for sequencing, the following steps were conducted.
16/07/2015
Transformation of J23117_I135K4 gave more than 100 colonies for the concentrated one and around 70 colonies for the normal one. And the control gave around 40 colonies for the concentrated sample and 8 colonies for normal one.
Then inoculation was performed in 5 mL, in 15 mL tubes @ 37°C @ 200 rpm overnight. The following colonies were selected:
J23117_I13504 (5 colonies). We did this one again, because the sample did not give any GFP signal;
∆csgB;
I13522.
17/07/2015
We performed plasmid isolations for the 7 colonies selected on 16/07/2015. The plasmid concentrations obtained were measured by using the nanodrop method. The following results were obtained.
After the plasmid isolation we did the restriction again for J23117_I13504 with EcoRI and PstI. The following scheme was used in which the mixture for each of the 5 reactions is described. The total volume of each reaction was equal to 20 µl.
The restriction of the ∆csgB was performed with the restriction enzymes EcoRI and XbaI and the restriction of I13522 was performed with EcoRI and SpeI. In this case a total volume of 50 µl was used.
The restriction products were putted on a Agarose gel 1% for 30 minutes, 110V and after that for 15 minutes, 125V. The description of the gel is as above.*These are not for the Interlab study but for our own constructs (module 1).
21/07/2015
The interlab cultures were inoculated in 5 mL LB + antibiotics. The J23101_I13504, J23106_I13504 and J23117_I13504 were inoculated in triplicate with CAM resistance. J23101; J23106 and J23117 with CAM only one each. I20270 was used as a positive control both in CAM and KAN, due to backbone uncertainty.
Back to Top
Labwork for Curlis
June
15/06/2015
We started preparing the materials for the lab. All the devices are set up, and the disposables ready to sterilize. At the end of the day, both plastic ware and liquids were autoclaved (121ºC, 15-30 min). Here the materials prepared are summarized:
LB medium, 6 bottles of 200 ml*
LB-agar mixture, 6 bottles of 400 ml*
CaCl2 100mM, 1 bottle of 200 ml*
Ethanol 70%, 1 bottle of 400 ml*
CaCl2 100mM + 40% Glycerol, 1 bottle of 200 ml*
*All the materials were prepared using standard methods
16/06/2015
The first stock of competent cells was made, so the first cloning experiments were conducted. could. The Top10 strain was chosen due to its availability and reliability in the results. 100 tubes of working solution containing 50 µL of Top10 competent cells (1) were made, and stored at -80ºC afterwards.
(1) Protocol for Top10 competent cells
19/06/2015
The cloning experiments started today. The experiments for InterLab study were performed initially (see Interlab labjournal).  First, LB-Agar plates were made so they were solid and available for growing the cells later:
*The antibiotics were taken from a stock solution containing:
1:1000 ampicillin
1:1000 chloramphenicol
1:1000 kanamycin
23/06/2015
The following cell cultures were made in 9 mL LB  ∆csgA strain, K1316015 and I13504 in and grown overnight at 30°C. I13522 (TETP-GFP) has been taken from the iGEM distribution kit (2015, plate 3, location I6).
Primers were dissolved in sterile Milli-Q water according to the amount specified in the shipping information (3) and shaken at 400 rpm for 20 minutes at room temperature to procedure stock solution for iGEM15 1 and iGEM15 2.
(3): Shipping information for primers.
24/06/2015
Plasmids were isolated for K1316015 using the Promega miniprep unit and the related protocol you can find online (4). Plasmids were stored at minus 20°C (30 µl);
AM35α has been made competent (1) and grown overnight. In order to see whether the density was good enough, the OD was measured.
To check the PCR reaction, an agarose gel (6) was made. The PCR product should be at the level of 2.7 KB.
The construct K1316015∆CsgB was isolated from the gel by using the Promega DNA purification protocol (7).
Finally, the ligation was performed with K1316015∆Csg (0.3 µl, based on a concentration of 153.94 ng/µl).
(4): Protocol related to Promega miniprep unit for plasmid isolation.
(5): Protocol for doing a PCR reaction.
(6): Protocol for running an agarose gel.
(7): Protocol for Promega DNA purification.
25/06/2015
Transformation (2) of ligation product  K1316015∆CsgB (top10 cells, 46 ng). After transformation, the cells were plated on LB + CAM.
Plated the transformation products K1316015∆CsgB on LB - Cam plates;
Preparation of competent cells for ∆csgA AM35α. 1 mL of the overnight culture was inoculated in 100 mL of LB medium.
Isolation of genomic csgC & csgEFG with primers Primers selected: iGEM15 15;16;17;18. The primers were dissolved in Milli-Q water (sterile) according to the amount specified in the delivery note (8) and shaken for 20 minutes at 400 rpm at room temperature.  Produced primer stock solutions from which the working solutions were prepared (100 µM). By a 1:10 dilution to give a working solution of 10 µM or 10 pmol/µl. Working solutions were produced for a volume of 100 µL.
The competent cells (AM35α) had an OD600 of 0.561 in a volume of 100 mL. Cells were harvested in 2 50mL tubes at 4°C for 5 minutes at 4000 rpm. 10 ml of ice cold 0.1 M CaCl2 was added to each tube to resuspend the cells and incubate them on ice for 20 minutes. They were centrifuged and resuspended in 5 mL ice cold CaCl2 each. Incubation for 1h on ice. The cells were pelleted and resuspended in 5 mL 50mM CaCl2 + 40% glycerol each. They were divided in portions of 50 µL and frozen in liquid N2.
(8): Delivery note of primers.
29/06/2015
Isolation of E.coli U12 DH5α genomic DNA. Cells were grown overnight in 6mL LB and stored in the fridge over the weekend. The genomic DNA was isolated using Promega Wizard genomic DNA purification kit following the producer’s protocol (9). The DNA pellet was rehydrated at 4°C overnight.
Cell cultures were created for the 100 µL plates from 25/06/2015 in 8mL LB+ 80µL 100x AMP (actually, 100 mg/mL). No growth was observed indicating problems with the transformation.
(9): Promega Wizard genomic DNA purification kit.
30/06/2015
Measurements of dsDNA indicates failure of isolation of genomic DNA. The concentration is within the machine’s fluctuation 1-2 ng/µl with fluctuations of 1 ng/µl in measurement.
July
01/07/2015
Transformation of GFP/∆csgB product – plated on LB-CAM; So, plates that should be prepared: 4 plates LB; 2 plates LB-CAM; 6 plates LB-AMP;
Restriction reaction was performed as followed, - ∆csgB was restricted with EcoRI/XbaI (16 µl) The reaction mixtures for restriction were as below and it was incubated at 37°C.
Selected colonies from
∆csgB LB-CAM
I13522 LB-CAM
The selected colonies were incubated in 8 mL of LB containing CAM or AMP and they were grown at 37°C at 210 rpm.
An agarose gel was made, the running was performed at V is 100 mL and for 60 minutes.
02/07/2015
Plasmid isolation (Promega, protocol (4)) of ∆csgB, and I13522. Different in this protocol were the facts that we did not use elution buffer but Milli-Q at 50°C (even if we work with small plasmids). The plasmid concentration was determined with the nanodrop machine. A restriction of these four plasmids was performed at 37°C for 2 hours. To check the restriction products, an agarose gel was performed.
Transformation was performed with Top10 cells prepared by Ilja. The normal protocol of transformation was followed (2) for transforming and pUC19 (as a control). For the transformation 1 µl of DNA was added.
03/07/2015
A gel purification(Promega (7)) and a ligation of ∆csgB and I13522 (to obtain the csgA_GFP construct) were performed. Then, transformation of Ligation product csgA-GFP; Control TOP10 strain; was performed. Cells were plated in duplo and also one control.
06/07/2015
Results of the transformation (csgA-GFP)
1. Empty plasmid control (also growth, 100 colonies);
2. TOP10 + CsgA+GFP (growth, 43 colonies, big).
After the checking the plates, a small culture of 5 mL was made using a single colony from the obtained plates, the following cultures were made:  TOP10 + csgA-GFP.
The antibiotics were added in a ratio of 1:1000 (thus in 5 mL, we have added 5µl). After cloning, we put the 50 mL tubes at 37°C at 200 rpm.
Besides that we made LB and LB-agar.
- 2 x 200 mL LB (4 g LB powder and 200 mL MQ-water);
- 1 x 400 mL LB (8 g LB powder and 400 mL MQ-water);
- 2 x 400 mL LB agar, 14.24 LB agar powder and 400 mL MQ-water);
- 1x 900 mL LB agar, 32.05 LB agar powder and 900 mL MQ-water);
Then the materials were autoclaved in the kitchen.
The backbone for our own construct is pSB4K5. For the transformation, we did use the protocol (2). Then we have grown the cells on the plates, I13504 and pUC19 on AMP, J23101, J23106 and J23117 at CAM and pSB4K5 at KAN.
07/07/2015
For IDT transformation we already decided which plates we should use
- 16 plates LB-KAN;
- 16 plates BL-CAM;
08/07/2015
We prepared our own antibiotic stock solutions (although we had to redo it anyway)
Amp: 100mg/ml = 0.1 g/ml (dissolved in water)
Kan: 50 mg/ml = 0.05 mg/ml (dissolved in water)
Cam: 35 mg/ml = 0.035 mg/ml. (dissolved in 100% ethanol).
After preparing the antibiotic stocks in only 1 mL, the supervisors (Essengül) told us to make larger stocks and also filter them in order to sterilize them (since we cannot autoclave them).
13/07/2015
Overnight three tubes with 5mL cultures were grown at 37°C shaken at 200 rpm. These were:
K1316015ΔCsgB
I13522
pSB4K5
14/07/2015
Plasmid isolation of pSB4K5 was done for IDT genes backbones. This was done by using 4,5mL overnight culture for isolation of plasmids, to increase the yield. The yield was determined by using the nanodrop, and the concentration was 62,7 ng/ µL.
Glycerol stock was made from the 0,5mL culture left according to protocol from
I13522, →  B1 (F1, block 2, column 1, row 6, box 1)
K1316015ΔCsgB→  B2
15/07/2015
For the gBlocks® we performed the restriction. The standard mixture used was followed.The cutsafe and H2O were mixed in the ratio of 14µl cutsafe and 98µl H2O. The DNA was subscribed as followed (gBlocks®):
Mfp5_CsgA;
CsgA_Mfp3;
CsgA_His;
CsgA;
CsgA_HA;
CsgA_Spytag;
CsgC_low;
CsgC_medium;
CsgC_high;
CsgBC_low;
CsgBC_medium;
CsgBC_high;
CsgEFG;
The restriction was left overnight, the temperature schedule was as follows:
The first 3 hours at 37°C;
20 minutes at 65°C;
The rest of the night at 4°C.
16/07/2015
The restricted IDT genes of 15/07/2015 were ligated according a ratio 3:1 for insert:vector. The first 3h the samples were placed at 16°C and the other 20 minutes it was placed at 4°C. After the ligation, 2µl of each sample was transformed into 50µl TOP10 cells. After 1h at 37°C, 35µl of the sample was plated. For the concentrated sample, we centrifuged the samples for 3 minutes at 4000 rpm. Discarded the supernatant partly and resuspended pellet. Plate also 35µl of concentrated cells.
20/07/2015
Column purification was done on CsgAΔCsgB to remove the small restricted pieces. This was done following protocol of the Wizard column purification.
Colonies from plate #2-13 were grown in triplicate in LB - CAM. In total 36 tubes were used, because triplicates were performed due to possibility of self ligation of the backbones pSB1C3 and pSB4K5. Plate #1 was placed in stove again to grow overnight, for larger colonies, at 37°C.
21/07/2015
CsgEFG was placed in a pSB4K5 plasmid which contains an Kanamycin (KAN) resistance gene. This plasmid was attempted to be grown in Chloramphenicol (CAM), but no growth was obtained. The transformation will be repeated in LB-KAN, with the same colonies used first.
Plasmid isolation was done for cultures #1.1-12.3 (36 total). Analytical restriction was performed to confirm insert size for all samples with EcoRI-HF & PstI-HF (See lab journal, page 38 for pipetted volumes). Only 24 out of 36 had the insert, which will be continued with. The correct plasmids are marked green in Table 1
Analytical Restrictions of 7.2 – 12.3, the ladder is a Benchtop 1kb DNA ladder (Left)
Table 1. The successful plasmids (green) will have to be restricted again to continue with ligation. Constructs 3 - 6 will be cut with EcoRI-HF & XbaI-HF. Constructs 7-12 will be cut with EcoRI-HF & SpeI-HF. One Glycerol stock was made per successful construct.
22/07/2015
Construct 13 has been grown again in LB-KAN. Plasmid isolation was done. These plasmids had to be analytically restricted to make sure that the plasmid contained an insert. To speed up the progress, the analytical restriction (EcoRI-HF & PstI-HF) was done at the same time as the “real” restriction (EcoRI-HF & SpeI-HF).
Restriction of constructs 3-6 were done with EcoRI-HF & XbaI-HF. This should cut out a DNA fragment of around ~10 basepairs, which will be removed using column purification. Restriction of constructs 7-13 were done with EcoRI-HF & SpeI-HF. The restriction products were added to electrophoresis, proceeded for 30 minutes at 120V.
Transformation of I13521 [RFP] into top10 cells. Of the I13521 plasmid, 2µl was added to 50µl of cells, incubated for 30 minutes on ice, heat stocked, punt on ice for 2 minutes and incubated at 37°C in 1 ml of pre-warmed LB for 1 hour. Plated on CAM plates in both low and high concentration. Empty cells were used as control to check the quality of the plates.
Another step we performed was the ligation of CsgA∆CsgB with I13522 (GFP). The concentrations of both plasmids after isolation was equal to CsgA∆CsgB: 3.2 ng/µl and I13522: 5.9 ng/µl. Construct:backbone ratio was equal to 1:3.7.
The reagents were mixed and incubated at room temperature for one hour. After that, the mixture was inactivated by heat shock for ten minutes at 65°C. Half of the ligation mixture (25µl) was used for the transformation. The concentration of the ligation mixture was determined to be 600 ng/µl but this result is rather unlikely and most likely due to the ligation buffer and ligase components.
The restriction described above gave the following results.  The cloning of CsgEFG did not work for the 3 colonies chosen; 13.1 and 13.3 were self-ligated plasmids and 13.2 was an empty plasmid. 8.1, 8.2, 8.3, 9.2 and 9.3 were not digested by the restriction enzymes; the only band observed on the gel was the one corresponding to the plasmids.
The restrictions of 10.1, 10.2, 10.3, 11.1, 11.2 and 12.1 were successful. Unfortunately, it was impossible to see the bands under the UV. Therefore, the experiment has to be repeated.
23/07/2015
Material preparation for LB agar. Four new bottles of 400 mL were prepared. 14.24 grams of LB agar mix + 400 mL Milli-Q water were mixed while heating to boiling point. After preparation, the bottles were autoclaved.
Another material preparation was performed, 3 bottles of 200 mL LB and 1 bottle of 400 mL LB were prepared. For the 200 mL bottles, 2 grams of LB powder was weighed and added to 200 mL Milli-Q water and for the other one 4 grams of LB powder was weighted and added to 400 mL Milli-Q water.
Chloramphenicol preparation: 35 mg chloramphenicol was dissolved in 1 mL EtOH 100% to give 1000x concentrated stock.
Besides material preparation, we performed plasmid isolation and after that restriction.
The restricted fragments were put on a 1% Agarose gel. The fragment size of 7 to 9 is equal to 441 bp. However, due to the gel used, these fragments run off the gel. The empty cut plasmid was observed at 2070 bp, but previously analysis showed the insert was there. Both 10 and 12 were observed at the correct fragment size.
Clones for 13 still showed only plasmid self-ligation at 6000 bp. Therefore, the ligation/transformation will be repeated for this one.
Transformation results for the I13521 & CsgA_GFP (I13522)
The stock solutions for sequencing primers made by dissolving the DNA in 10µl Milli-Q H2O per 1ng to obtain a concentration of 100µM. 20µl of the stock solution was added to 180µl of Milli-Q to afford 10 µM working solution.
24/07/2015
Dissolving gel slices and extracting DNA for 10, 11 and 12. After extraction, the following weights for the three types of constructs were obtained.
10: 370 ng;
11: 550 ng;
12: 740 ng.
Membrane binding solution was added in 1:1 and the slice was dissolved at 55°C.
Thus, the samples were loaded on a 2% agarose gel to achieve higher concentrations. For the transformations CsgA_GFP and RFP (I135521) a colony PCR was done, to see if the plasmids contained an insert. We did this by using a sterile tip to pick off a piece of a colony into the PCR reaction mixture. The PCR reaction mixture contained:
We repeated the purification with gel of the restricted plasmid DNA. The gel slices weighted each: 7,2: 900 mg;  8: 600 mg;  9: 420 mg;  10: 500mg;  11: 500 mg;  12: 500 mg;
We dissolved the gel slices in 1ug:1uL membrane binding solution and followed the purification from gel protocol. The final concentrations were: 7.2: 1 ng/µL;  8: 2,5 ng/µL ; 9: 3,2 ng/µL; 10: 2,05 ng/µL; 11: 3,6 ng/µL; 12: 2,5 ng/µL;
The purified DNA fragments were stored in the -20°C.
27/07/2015
When looking on the gel 13 (CsgEFG) did not work. Therefore, the ligation and restriction should be  repeated. This is done by restricting the plasmid pSB4K5 (14-07-2015) and CsgEFG (23-07-2015) with EcoRI-HF and PstI-HF, and ligating the products afterwards.
The pSB4K5 was then run on a gel at 110 V for 30 minutes, together with a DNA ladder (Benchtop 1kb DNA ladder) and uncut pSB4K5. In order to make the bands visible on the gel, Blue/orange loading dye was added to the uncut and cut pSB4K5 samples. We looked at the gel using a UV lamp, and cut out the band of the cut pSB4K5 at approximately 1700 kb. Gel purification was then done with a yield of 2.08 ng/µl.
The restricted CsgEFG was column purified using the Wizard SV, gel and PCR clean-up system following protocol with a yield of 1.32 ng/µl.
28/07/2015
Plasmid isolation for CsgA_GFP and I13521 (RFP) in triplicate with the goal to restrict RFP for ligation and confirm the insert for CsgA_GFP. 5 mL of the bacterial cultures were used.
The restrictions of I13521 and CsgA_GFP have been performed EcoRI-HF and SpeI-HF.The restriction has been incubated for 1 hour at 37°C in PCR with final heat shock.
The CsgA has been put on the gel on the left side, RFP on the right. A size of 2400 bp indicates failure of ligation of CsgA with GFP. RFP was confirmed and cut out of the gel.
The DNA constructs made on 22/07/2015 (3.1-6.3) were purified out of the gel. The next step is ligation in a ratio of 3:1 insert:vector. During the ligation, different samples are ligated. The CsgEFG_pSB4K5 and the constructs described in the table above.
Ligation has been incubated for 3 hours at 16°C and the final heat shock, the samples were hold for 6 minutes at 62°C. The improvement in the protocol is the fact that molar ratios are used instead of mass ratios!
Another step performed was the transformation into CaCl2 competent cells. Before the protocol could be followed, the cells were thawed for 20 minutes on ice. After the thawing, 8 µl of the ligation mixture was added to cells and incubated for 30 minutes. Running out of ‘sticks’, some of the results of the transformations were plated the next day.
30/07/2015
The restricted RFP was isolated from the gel, which resulted in concentrations of 17, 14 and 12 ng/µl in volumes of 50µl each. After that, ligation of I13521 (RFP) with CsgA & CsgA_His/HA/SpyTag. The ligation scheme is calculated in such a way that a vector DNA concentration of 50 ng is achieved.
Another part of today was checking the plates made previously (24/07). No colonies were shown there. Consequently, the plates one tested with top 10 cells without plasmid in order to check the KAN antibiotic. So for the plating we used top 10 cells on LB + KAN plates.
Moreover, we made overnight cultures in 5 mL LB + antibiotics (although mentioned otherwise).
31/07/2015
Today we started with plasmid isolation of the overnight cultures 1-20 inoculated at 30/07/2015. For the plasmid isolations, 4.5 mL (3 x 1.5 mL) was used from each overnight cultures, except for 7.1 & 7.3 (3 mL → 2x 1,5 mL) due to pipetting mistake. In the table below, the concentrations we found after the plasmid isolations are shown.
From the obtained plasmids, number 1 to 12 and the Interlab construct J23117_GFP were sent for sequencing. All in a total volume of 10 µl, a total amount of plasmid of 500 ng and a total amount of primer of 25 pmol. The primer working concentration would be 10 pmol/µl.
After plasmid isolation, we performed a restriction reaction. For this, the constructs Mfp5_csgA, CsgA_Mfp3, CsgA_his, CsgA_HA, CsgA and CsgA_Spy (1 to 6) were cutted with the restriction enzymes EcoRI-HF and XbaI-HF. pSB4K5 (14) and pSB1C3 were cut with EcoRI-HF and PstI-HF. Finally, the constructs CsgC_low #1, CsgC_low #2, CsgC_low #3 (7.1, 7.2, 7.3), I13522 #1, I13522 #2, I13522 #3, I13521 #1, I13521 #2, I13521 #3 were cut with EcoRI-HF and SpeI-HF.
The restricted samples were put on a gel to check whether the restriction worked.
- EcoRI/XbaI = small fragment → Column purification for #1-6 (Mfp5_csgA, CsgA_Mfp3, CsgA_his, CsgA_HA, CsgA and CsgA_Spy);
- EcoRI/SpeI = bad of ~700/400bp → Gel purification #9-17 (CsgC_low #1, CsgC_low #2, CsgC_low #3 (7.1, 7.2, 7.3), I13522 #1, I13522 #2, I13522 #3, I13521 #1, I13521 #2, I13521 #3);
- EcoRI/PstI = band of ~3000bp → Gel purification #7-8 (pSB4K5 (14) and pSB1C3).
Column purification was performed according to the protocol. Gel samples #7-17 were loaded with 5µl Blue loading buffer, with 25 µl sample loaded per slot.
The transformation of (30/07) CsgA_His/Spy/””/HA and RFP were grown overnight to confirm the insert, and make a cryostock.
Plasmid isolation was performed for the RFP + CsgA + tags. The samples were lost in the centrifuge so this must be repeated.
Ligation Csg_EFG with pSB1C3 & pSB4K5, CsgA&tags with GFP, Mfp5 & Mfp3 with GFP/RFP.
August
03/08/2015
The gel purification of 7 - 17 was performed according to the promega protocol. We obtained the following concentrations.
The CsgEFG had a concentration of 46.73 ng/µl.
After that, the ligation of CsgEFG into pSB4K5 and pSB1C3 has been performed. In the mixture a 1:1 ratio of construct:vector was used, which resulted in 4.3 µl pSB4K5 to 1.1 µl insert and 6.8 µl pSB1C3 to 1.8 µl insert. The ligation has been incubated for 2 hours at 16 degrees.
Also the CsgA, CsgA_His, CsgA_spy, CsgA_HA, Mfp5_CsgA, CsgA_Mfp3 has been ligated with GFP and Mfp5_CsgA and CsgA_Mfp3 with RFP. All of these were put together to obtain a 3:1 ratio.
The next step was the transformation of the ligated samples into TOP10 cells. For this, samples of 2 µl of CsgA_GFP, CsgA_His_GFP, CsgA_Spy_GFP, CsgA_HA_GFP, Mfp5_CsgA_GFP, CsgA_Mfp3_GFP, Mfp5_CsgA_RFP, CsgA_Mfp3_RFP, CsgEFG_pSB4K5 and CsgEFG_pSB1C3 were used. After transformation all cells were plated in the LB-agar + KAN plates.
Cells are regrown from cryostock for CsgA_RFP and the tags in triplicate; I13522 & I13521 for the hardware track.
04/08/2015
The first part we have done was the isolation of plasmids of the RFP clones with CsgA and the three tags. For this, the His/HA/Spytag were done in triplicate. Restriction was carried out for 1 hour at 37°C and heat shocked for 10 minutes at 80°C to confirm the insert with a total of 500 ng of DNA. For all these constructs, the restriction enzymes EcoRI-HF and PstI-HF were used. The added volumes were based on the concentrations after the plasmid isolations, determined with the nanodrop.
After restriction the samples were loaded on the gel. 3 µl of loading dye was added to the samples. The bands were observed at the expected size of 1600 bp for CsgA_tag_RFP and the empty pSB1C3 plasmid at 2100 bp for all samples. The sample preparation of sequencing was done following the sequencing protocol. Sequencing was done by adding 25pmol forward and reverse primer and 500ng plasmid DNA in 10µL. The CsgA_RFP, CsgA_His_RFP, CsgA_HA_RFP, and CsgA_Spytag_RFP.
CsgA and I13504 were grown from cyrostock in 5ml LB with 5 µL Cam overnight at 37°C.
05/08/2015
We isolated plasmid the overnight cultures CsgA and I13504, with a final concentration of 120ng/µL and 141 ng/µL respectively.
The plasmids and Spycatcher were then restricted by using the following:
Spycatcher:  EcoRI / PstI
CsgA:  SpeI / PstI
I13504:  XbaI / PstI
With a final volume of 50 µL. We left the restriction for 1 hour.
We then did a Colony PCR of the transformations made 03/08/2015 which were the following:pSB4K5_CsgA_GFP, , pSB4K5_CsgA_His_GFP, pSB4K5_CsgA_Spy_GFP,pSB4K5_CsgA_HA_GFP, pSB4K5_CsgA_Mfp5_GFP, pSB4K5_CsgA_Mfp3_GFP, pSB4K5_CsgEFG, pSB1C3_CsgEFGThis was done by using the PCR mastermix (Qiagen), FW2 & Rv primers, dNTP’s and Milli-Q. Reaction volumes were 50µL per sample, with 8 colonies per transformation analysed.
From the transformations CsgA_His_GFP and Csg_HA_GFP both two colonies had correct inserts and were grown in LB+ CAM overnight.
07/08/2015
For the cloning of CsgEFG a alternative protocol was used called the “TA cloning protocol”. We used this protocol to ligate the CsgEFG into a backbone, since previous attempts hadn’t worked.
10/08/2015
Cryostocks were made from CsgA_His_GFP, CsgA_HA_GFP, CsgA_Mfp3_GFP, CsgA_Mfp5_GFP. The same genes were sent for sequencing.
The CsgA_”tag”_RFP was made in ΔCsgA strains.
CsgEFG 4 was successful and we continued with the gene by ligating it in the plasmids pSB4K5 and pSB1C3. We also ligated Spycatcher_His with pSB1C3.
11/08/2015
We did a plasmid isolation on CsgC_low/med/high, CsgBC_low/med/high, and twice of I13504. The plasmids CsgC and CsgBC were restricted with EcoRI-HF & SpeI-HF. The I13504 were restricted with XbaI-HF & PstI-HF. The restrictions were than loaded on a 2% agarose gel, after running the bands were cut out and purified from the gel following protocol.
For the congo red assay’s plates had to be made which include LB, Congo Red  (CR; 50µg/ml), antibiotic, Brilliant blue (BB; 1µg/ml), and rhamnose in increasing concentrations (0.1%, 0.2%, and 1%)
We made 6 different types of plates. All plates contained LB, CR, BB, and Chloramphenicol. Four plates contained different concentrations of rhamnose.  The 5th plate contained IPTG as inducer instead of rhamnose, and the last plate was the control with no additional components.
For the CR assay we transformed pSB4K5_CsgA , I3504 (- control), and CsgA_IPTG (+ control)  into CsgA deficient strains (ΔCsgA).
12/08/2015
We checked the  restriction/ligation/transformation of Top10-pSB1C3_CsgEFG (11/08/2015) by  colony PCR’ed. For the colony PCR, we picked 9 colonies by first dipping the colony in 50µL LB, which was to be grown at 37°C, and than in the PCR mix. The PCR was done using altered conditions:
13/08/2015
Since the colony PCR of Wednesday(12/08) hadn’t worked, we retried to see if results could be obtained today. The PCR parameters were changed to the following:
14/08/2015
We made cryo stocks of the following overnight cultures:
pSB4K5_CsgBC_L/M/H_EFG (Top 10 cells)
pSB1C3_CsgEFG  (Top 10 cells)
pSB1C3_CsgA_His/””  (ΔCsgA)
We repeated the colony PCR of 13/08 for CsgC_L/M/H_EFG to see if clear bands were present. The results were inconclusive since the difference between the in- and correct inserts is too small to see. We decided to send the plasmids for sequencing.
CsgA_I13504  and Spycatcher_His were successfully inserted.
For the CR assay we used cells transformed (11/08), which had been grown overnight, and then placed in the fridge (4°C). The strains were pSB4K5_CsgA , I3504 (- control), and CsgA_IPTG (+ control)  in CsgA deficient strains (ΔCsgA).
The bacteria were streaked out on the plates made (11/08). The pSB4K5_CsgA was plated on the plates with different concentrations of rhamnose (0, 0.1, 0.2, 0.5, and 1%) in total 5 plates. The – control was plated on a plate with no rhamnose. The + control was plated on plate with IPTG. The plates were left in the stove at 30°C for 60 hours.
17/08/2015
The plates were checked after growth. All the pSB4K5_CsgA colonies on the CR plates were red. The + control was red as well. The positive control colonies plated on the 0% rhamnose were white. The negative control was reddish with the negative strain.
The bands of CsgBC_high of 13/08 were too high. The Colony PCR pSB4K5_CsgBCEFG was repeated on 16 colonies of the plate 12/08. The colonies were also dipped in 50µL LB with KAN. After running the samples on a agarose gel, when viewed above a UV-lamp the bands were still located too high.
18/08/2015
We dissolved BFP_Spycatcher_His in 100 µL Milli-Q, to obtain a concentration of 10ng/µL. We then proceeded to restrict 10µL with EcoRI-HF&PstI-HF, to prepare for ligation with pSB1C3. The construct would be pSB1C3_ BFP_Spycatcher_His.
19/08/2015
We transformed the following plasmids into Top10 cells, the transformations were grown overnight at 37°C:
The pSB4K5_CsgBCEFG colonies grown in 50µL LB (17/08) were transferred to 1mL fresh LB and grown overnight at 37°C.
20/08/2015
The transformation worked for the #1, 2, and 8. The transformations #3-7 had no visible colonies.
The pSB4K5_CsgBCEFG colonies grown overnight (19/08) were put in a colony PCR. The results showed again that the insert was too large.
21/08/2015
The ligation of pSB1C3 with BFP_Spycatcher_His was not successful. The BFP_Spycatcher_His (18/08) was restricted with EcoRI-HF & PstI-HF. The restricted construct was then ligated with pSB1C3 from 03/08 and 15/07, and pSB4K5 to improve the chance of a successful ligation.
The CsgC_L/M/H constructs were repeated by ligating to with EFG. The BC_High was also repeated by ligating to EFG.
There were in total 6 ligations, as shown above. These were then all transformed into Top10 cells, and grown overnight.
A double transformation was done with ΔCsgA with pSB1C3_CsgA The plasmids made 14/08 were transformed into, since the sequencing results hadn’t been sent back yet.
24/08/2015
25/08/2015
Top10_pSB1C3 was grown overnight in ~10 mL LB+CAM. 5.7 mL of overnight culture was used for plasmid isolation. The 60µl of isolated plasmid was then transformed into ∆CsgA strain using the heat shock protocol. Cells were plated on a LB+CAM plate, and grown in 37°C overnight.
Both strains were diluted till an OD600 of 0.0556 was reached. The wells were filled with Rhamnose, water and LB + cells. The following experiments were performed.
26/08/2015
∆CsgA_pSB1C3_CsgA and ∆CsgA_pSB1C3_CsgA_His were grown overnight (25/08/2015) and then diluted 1/60 in 10 mL LB+Cam until an OD of 0.6. 1 mL of 5% Rhamnose was added to both cultures and grown at 37°C for 2 hours. Then the cultures were centrifuged at 4000 rpm @ 4°C for 15 minutes. Supernatant was discarded and pellets were placed in -20°C.
Also a restriction was performed for CEFG_low, CEFG_med, CEFG_high, BCEFG_low, BCEFG_med and BCEFG_high.
27/08/2015
OD600 was measured for ∆csgA_csgA (1.77) and ∆csgA_GFP (1.90). Based on this result, 500 µl of each culture was added to 5 mL M9.
6 tubes of ∆csgA_pSB1C3_CsgA, ∆csgA_pSB1C3, ∆csgA_pSB1C3_csgA_His were grown to an OD between 0,4 and 0,5. The cultures were then induced with 0, 0.1, 0.2, 0.3, 0.4, 0.5% rhamnose. The cultures were left overnight at room temperature.
The pellets of 26/08 were resuspended in 1mL lysis buffer with lysozyme, and 1μL DNAse. The lysis reaction was left on ice for 30 minutes, followed by centrifugation at 12000 g for 30 min at 4°C. The supernatant was collected and run through 1mL Ni-NTA spin column.
28/08/2015
The restrictions of 26/08 were run on a 1% and 2% agarose gel. 1% agarose gel Top Ladder, #1-6, Ladder Bottom: Ladder, #7-18 (Under) 2% agarose gel: Ladder, #1-18, Ladder, CsgA= 400 bp ca. CsgEFG= 1700 bp ca. CsgBC= 600bp. The Congo red liquid assay started 27/08, 6 tubes of ∆csgA_pSB1C3_CsgA, ∆csgA_pSB1C3, ∆csgA_pSB1C3_csgA_His being induced with different concentrations of rhamnose (0, 0.1, 0.2, 0.3, 0.4, 0.5%) were grown to an OD between 0.4 and 0.5. Congo Red was then added to a concentration of 20µL/mL.
pSB4K5_CsgEFG colonies were taken from Amp TA cloning (07/08) and grown overnight in LB. 3x1,5mL of culture was used for the plasmid isolation, following protocol. After obtaining plasmids, restriction was done with Eco-RI & PstI-HF. When ran on a 1% agarose gel no bands were visible.
OD600 was measured for ∆csgA_csgA (0.99), and ∆csgA_GFP (0.79). ∆csgA_csgA was diluted 4x, while ∆csgA_GFP was diluted 3x, both dilutions were done with m9 media. Two cultures were then diluted another 6x, in a volume of 6mL and 3mL respectively.
September
01/09/2015
A Congo red liquid assay was done with ∆csgA_pSB1C3_CsgA, ∆csgA_pSB1C3, ∆csgA_pSB1C3_csgA_His. The cultures were grown to an OD600 of 0.4-0.6, when induced with different concentrations of rhamnose (0, 0.1, 0.5%).
A crystal violet (CV) assay was done to measure the amount of biofilm being formed. In a 96-well plate the following volumes were pipetted by using the cultures: A∆csgA_pSB1C3_CsgA, ∆csgA_pSB1C3, ∆csgA_pSB1C3_csgA_His
02/09/2015
For the Congo red assay the cultures (01/09) that had been induced for 21 hours were measured for at the absorbances of 600nm, A280nm (protein concentration), 480nm (Curli formation), and 500nm. The OD600 was measured before congo red was added at a final concentration for  20µL/mL. The samples were centrifuged at 14000 rpm for 5 minutes.
03/09/2015
Top10_pSB1C3_BFP_Spycatcher_His was sent for sequencing.
In the sequence results of pSB1C3_CsgA_Spytag and pSB1C3_CsgA_Spytag_RFP a SNP mutation had occurred. We used PCR with primers with the correct base pair to remove this mutation.
04/09/2015
07/09/2015
Due to no colonies forming on the plates of the transformation, it was repeated with electroporation ΔCsgA cells.
The electroporation was done following protocol, only the cells were grown for 2 hours instead of 1 hour. The cells were also centrifuged 3 minutes at 4000 rpm, resuspended in LB and then plated on CAM plates (pUC19 was plated on AMP).
For the congo red liquid assay, to determine the amount of curli’s being produced; ΔCsgA_pSB1C3_CsgA and  ΔCsgA_pSB1C3 were grown overnight at 37°C, then induced with 0, 0.1, 0.2, 0.3, and 0.5% rhamnose, in duplicate, at room temperature for 24 hours.
The mutagenesis PCR with pSB1C3_CsgA_Spytag and pSB1C3_CsgA_Spytag_RFP was repeated. This was done with an overnight culture, followed by plasmid isolation, then PCR, restricted with DpnI-HF. After 3 hours the plasmids were transformed in Top10 and ΔCsgA cells.
08/09/2015
Congo red liquid assay: After induction of 24 hours at room temperature the following results were acquired. The strains were ΔCsgA_pSB1C3_CsgA and  ΔCsgA_pSB1C3 induced with 0, 0.1, 0.2, 0.3% rhamnose, in duplicate.
11/09/2015
The plasmids pSB1C3_CsgA_Spytag_RFP-mut and pSB1C3_BC_M were isolated from overnight cultures. They were then sent for sequencing.
Western blot was done to see if quantification of amount of CsgA_His could be reached.
Cultures were grown 09/09 overnight of ΔCsgA_pSB1C3_CsgA and  ΔCsgA_pSB1C3_CsgA_His in 2mL LB+CAM. The cultures were then diluted to an OD600 of 0,05 on the 10th of september. The cultures were then grown for 1.5 hours at 37°C. The cultures were then induced with 0, or 5% rhamnose, and induced for 5 hours and overnight at 30°C. After incubation the cells were spun down for 20 minutes at 4000 rpm, at 4°C. The pellets were stored in the freezer at -20°C. They were resuspended (11/09) in 4mL LB. The OD600 of the cells were measured and centrifuged 15 minutes at 4000 rpm. The supernatant was discarded, and the cells were resuspended in sample buffer to obtain an OD600 of 10. The cells were then heated for lysis, at 95°C for 10 minutes, followed by centrifugation for 5 minutes at 32000 rpm at 4°C. The supernatant and pellet were stored separately at -20°C.
The teeth experiment were analyzed, after 44 hours of induction, with Typhoon. The teeth were separated from the attached bacteria by vortexing with ethanol. The amount of attached bacteria were checked using the plate reader.  The experiment was repeated twice.
Back to Top
Lab journal of the Hardware Experiments
June
17/06/2015
Preparation of 2% (w/v) sodium alginate stock solution using food grade alginate (obtained from Helena Shomar Monges). 2.05 grams were dissolved in 100 mL of Milli-Q water with continued stirring at 50°C. The next day (18/06/2015), a homogeneous solution was obtained. The mixture used was a blend of sodium alginate, CaSO4 and sodium pyrophosphate obtained from Minerals Water (UK).
Experiment 1: Continuous extrusion of sodium alginate and CaCl2 (aq) from separate tubes in close proximity.
Syringe pump settings:  Inner diameter = 4.61 millimetre for 1 mL Luer-Lock syringe (BD);
1. Extrusion without movement into Falcon Tube. Conclusions of the experiment:
- Rate = 0.50 mL/min;
- Total V=0.20 mL;
- Rate was too high, next run should be at a rate of 0.25 mL/min;
- Initial formation of single drops, later continuous strut which become solid after 210 seconds.
2. Two times tried with a volume of 0.30 mL at a rate of 0.25 mL/min. Both tries were only droplets, not a strut;
3. Settings were a volume 0.15 mL at a rate of 0.35 mL/min. It is extremely important to have the two nozzles as close as possible to each other to prevent droplet formations and favour struts. It appears there is too much CaCl2 (aq) compared to alginate.
4. Settings were a volume of 0.15 mL at a rate of 0.1 mL/min. In this case no droplets were formed, only a strut. So, we need a slow extrusion rate for a smooth line.
Change in the experimental setup: 3-way valves make the recharging of the syringe more practical. Therefore, in the experimental setup two 3-way valves and tubes were added.
23/06/2015
In this case movement is into petri dish.
Extrusion of alginate in CaCl2 solution draws the strut behind the needle and tends to form beads around the tip at low extrusion rates. The extrusion of both substances from the two needles seems more controllable in this way. During the formation, the alginate beads swim, but then they sink to the bottom.
Strut extrusion is disfavoured by the surface of the petri dish. It rather sticks to the nozzle than the plastic, thus it slides over the print surface instead of sticking to it. Extrusion on a paper tissue improved this, thus follow up experiments will be conducted on agar plated petri dishes.
25/06/2015
Alginate beads (circa 1 mL in total volume) . The supernatant was removed and 10 mL 0.1M NaCl was added to dissolve the hydrogel without mixing (12.30 pm).
29/06/2015
The hydrogel did not dissolve in 0.1M NaCl over the weekend. To discover the reasons for this, the literature should be checked again. 0.1M sodium citrate could actually be up to 0.3M in Na+ concentration.
July
06/07/2015
Experiment: dissolving hydrogel by sodium citrate (2% (w/v)). In the ratio 500 µl alginate and 500 µl CaCl2 (0.1 M), hydrogel formation occurs within seconds.
To the formed hydrogel, 500 µl Na3C6H5O7 was added in order to dissolve the hydrogel. The following observations were done:
1. There was no direct response in the first few minutes;
2. Even after a vortex period of 2 minutes, the hydrogel was still untouched;
3. Even at 09/07 it was still untouched.
09/07/2015
Since Na3C6H5 was sufficient to dissolve the hydrogel, another strategy has been tried. In this setup, we tried to dissolve the hydrogel by LB medium. Two different conditions were chosen:
1. 250 µl alginate, 250 µl CaCl2 (0.1M) and 250 µl LB;
2. 250 µl alginate, 250 µl CaCl2 (0.1M) and 1 mL sodium citrate.
First both the alginate and CaCl2 were added, then it was quickly vortexed. After hydrogel formation, the LB or sodium citrate was added. Both did not have a direct effect.
After around 4 hours reaction mixture 2 (with the sodium citrate) was partly dissolved. The first mixture (with LB) did not have any changes in the structure of the hydrogel.
10/07/2015
The reaction mixture (2) of 09/07 has completely been dissolved after one day. Reaction mixture (1) did not lead to any changes in the structure of the hydrogel (still intact).
Two new mixtures were prepared:
3. 250 µl alginate, 250 µl CaCl2 (0.1M) and 2 mL sodium citrate;
4. 250 µl alginate, 250 µl CaCl2 (0.1M) and 2 mL LB;
With reaction mixture 3, there was a direct effect. The gel almost dissolved after twisting. 10 seconds of vortexing did not have any effect on the hydrogel. Reaction mixture 4, no changes in the hydrogel structure were observed. Then, to mixture 3 two mL of sodium citrate and to 4 two mL of LB was added. For both cases, no change was observed (even no change on 13/07).
13/07/2015
Again, two reaction mixture were prepared:  (5) 250 µl alginate, 250 µl CaCl2 (0.1M) and 5 mL sodium citrate; and (6) 250 µl alginate, 250 µl CaCl2 (0.1M) and 5 mL NaCl.
Up/down turning of mixture (5) resulted in a partly dissolved hydrogel, whereas mixture (6) did not change anything in the structure of the hydrogel. The 10 seconds vortex of both mixtures did not affect the hydrogel.
Alginate/CaCl2 mixtures
Since a layer of CaCl2 was present in the prepared hydrogels, different mixtures of alginate and CaCl2 (0.1M) were prepared and tested (see Table).
In all the cases presented in the Table, there is still a layer of CaCl2 on top of the hydrogel. Although there is even a layer in the 9:1 case, this layer is really small. By performing 10 seconds of centrifugation, the layer did not increase (so, it is kind of stable).
K’NEX printer
First experiment
The printer reached a speed of 1 rpm, the syringe pump had a speed of 0.3 mL/min and the volume was equal to 0.3 mL. Strut formation was achieved, although it was not interconnected (0.4-1 cm in length and 0.2-0.5 cm in height).
Second experiment
The printer still had the same speed of 1 rpm. The rate of the syringe pump was equal to 0.5 mL/min and a volume of 0.5 mL. This results in the formation of an interconnected strut, that was much larger than droplet (0.3-5 cm in length, 0.2-0.5 cm in height).
Preparation of new CaCl2 (0.1M) solution
CaCl2 has a molecular weight of 110.984 g/mole. So, for the preparation of 50 mL, 0.555 grams of CaCl2 was required.
Recommendation: try to put CaCl2 in a petri dish and print the alginate on top. Problem with this: the K’NEX hole is too big, which results in large droplets. Solution for this could be to put the needle closer to the petri dish.
August
03/08/2015
IT turns out that the tubes are just as useful to extrude alginate as needles, but more flexible when actually touching the surface. The thinnest tubing does not pose too much resistance and can still extrude alginate. Paper soaked with CaCl2 seems to be a practical surface, as the alginate sticks to it rather than the needle and thus a continuous line rather than a blob is formed. The extrusion so far has been performed by hand, but the relative rate was kept rather low.
Experiments in planning
Identify good flow rate for the new tubing system on agar plates covered with a thin layer of CaCl2.
An agar plate was soaked with 1 mL of 0.1 M CaCl2 and it was nicely spread using a plating hockey stick.
- 0.2 mL/min is of good order of magnitude for extrusion
- gelatinization was found to be sufficient to create a stable gel
- a second layer was printed and formed a gel, a bit wider than the first layer however
- Four layers still formed one stable gel without any further addition of CaCl2 than the initial one.
04/08/2015
2g sodium alginate was stirred in 100 mL sterile LB and subsequently autoclaved. This seems to improve dissolving the alginate, the solution is much less viscous. Chloramphenicol (100μL) was added to keep it sterile. The cells, GFP (I13522) and RFP (I13521) are both in the pSB1C3 backbone. Dropping LB alginate into CaCl2 leads to gel formation, but it seemed to take a little bit longer since bigger droplets were created, nevertheless stable ones. Patterns were drawn to print the cells the next day.
10/08/2015
It turns out, autoclaving breaks down sodium alginate. Since a hydrogel is still formed, 4% w/v sodium alginate is dissolved in 50 mL LB and autoclaved to check for improvement of gelatinization. 2g alginate was dissolved in 50 mL LB and autoclaved.
LB agar is tested containing CaCl2 at a concentration of 0.1M for hydrogel formation.
10 mL was plated and tested for gelatinization with both 2% w/v sodium alginate dissolved in water and in LB.
0.1M CaCl2 in the agar results in efficient gelatinization of both normal 2% w/v sodium alginate in water and in LB. 4% w/v sodium alginate is less homogeneous than 2% and does not show improved gelatinization compared to 2%. 2% is fine to use.
11/08/2015
Patterns (logo, fb, twitter, cells) were printed at 0.1 mL/min on LB agar CAM + 0.1 M CaCl2 plates. Both line thickness and precision were improved like this. They were kept for 2h in the fridge before imaging.
Cell viability after dissolution of sodium alginate matrix in sodium citrate: I13521 1,5 mL was pelleted and redissolved; In 1 mL LB agar and in 1 mL LB
The alginate sample was dropped into CaCl2 (0.1M) to form alginate beads. These beads were then incubated in sodium citrate to be redissolved. (2% w/v). 300 μL LB alginate in 5 mL 0.1M CaCl2 for 5 minutes. Decant and resuspend in 15 mL 2% w/v sodium citrate.
LB alginate is viscous enough to print even without CaCl2 directly on the plate. Curing is carried out afterwards with CaCl2. However, the alginate then loses contact to the petri dish surface and starts floating around. Printing on LB agar + CaCl2 proves to be the best method.
13/08/15
Cells were still alive after 48h in 2% w/v 15 mL sodium citrate; all samples grew after inoculation with 5 mL LB CAM.
Cryostocks were plated for I20270 (interlab positive control), I13504_J23101 (interlab high promoter), I13522 and I13521. Best samples (high fluorescence and contrast to LB agar auto fluorescence) was I13521 (RFP) and I20270 (GFP). These should be used for bio-ink.
Bio-ink
- Grow cells in 5 mL LB overnight
- Spin down at 13’500 rpm for 3 min.
- Resuspend in 50 μL LB
- Add 200 μL 2% w/v sodium alginate
- Mix by pipette, then vortex
While the bio-ink is quite colourful under UV light, distinction between red & green is difficult. Increased fluorescence in seen compared to alginate only.
A printed RFP alginate pattern was inoculated with 50 mL 2% w/v sodium citrate to dissolve the gel.
18/08/2015
Sodium alginate was extruded at a rate of 0.11 mL/min determined to be the minimum rate of extrusion for a circle of 4,5 cm in diameter. First one layer was printed and subsequently eight more on top of it.
Accurate measures
- 4.6 cm diameter circle
- 1.5 mm height single line
- 3 mm for 8 layers
Circles of one layer thickness are printed for 0.1%, 0.5%, 1% and 2% sodium alginate to study dissolution properties,  printing properties of the bio-ink and cell viability. FRP in H20 was printed on LB CAM for direct comparison with performance of using alginate. Performance of using alginate:
After 30 minutes the water fraction of 0.2% and 0.5% is absorbed by the agar and a thin film remains on the plate of defined structure and thinness. A second layer could be printed on this.
Plates with 0.2%, 0.5%, 1% and 2% were incubated with 25 mL 2% w/v sodium citrate to study dissolution properties.
18/08/2015
After 24h none of the samples were completely dissolved. However 0.2% alginate which dried into the agar floated the least and thus remained its initial shape the most of all four samples. The sodium citrate supernatant was removed. The plate was rinsed with 1 mL 5x CAM. Incubation was done at 37 oC overnight. No growth was observed.
The patterns with GFP/RFP increased significantly in colour over a time period of 1 week in the fridge. The samples were placed in the 37 oC stove for further growth  and testing of alginate response to higher temperatures. After 24h, the plates started to dry out; agar becomes dehydrated and shrinks.
24/08/2015
Sterile stocks were made for CaCl2 (0.1M, 200 mL) and sodium citrate (2% w/v). Sterile filtration of 1% w/v sodium alginate using a 45 μm frit does not work at all. Resistance is far too high, not a single drop was obtained from the filter. Addition of antibiotic to the alginate and dissolution in sterile water.
Sodium alginate was pasteurized by boiling for 15s in the microwave and subsequently placed in cold water to cool down again. Antibiotic was added to the cold alginate to further improve and maintain lower contamination levels. Alginate is kept in the fridge. gelatinization seems to be unaffected by the pasteurization.
28/08/2015
Preparation bio-ink
- 500 mL RFP & GFP cells
- Spin down at 4’000 rpm for 3 min
- Discard supernatant
- Resuspend pellet in 100 μL LB and vortex
- Add 400 μL 1% alginate and vortex
Bio-ink was used to practice line formation.
31/08/2015
class="lead">All stored at 37 oC and checked on 01/09/2015. In all cases hydrogel was still present. * Excess CaCl2 removed, to assess impact
September
01/09/2015
Spinning disk
Several samples were created in the same manner as 28/08/2015 for imaging with the spinning disk microscope.
Additionally a plasma cleaned coverslip was prepared with two lines containing RFP, CaCl2 (100 μL), GFP, CaCl2 (100 μL). This was stored in the freezer to be measured on 02/09/2015.
07/09/2015
A comparison experiment of alginate from Groningen vs Delft was started. Observation: their powder is brown and dissolves to a clear solution. No further info available. Our powder is white and forms a milky emulsion. No solids are dispersed. 1% w/v solutions were used.
150 μL alginate was added from bottom to top into 1 mL 0.1M CaCl2. For visualisation purposes, 4 μL of brilliant blue was added to 400 μL alginate. Both formed a gel strut instantly. The CaCl2 was removed by pipet and 1 mL of 0.1M sodium citrate (monobasic) was added. A second sample was added  mL of LB. Samples were incubated at 37 oC.
08/09/2015
No sample is completely dissolved. Temperature is raised to 50 oC and shaking at 300 rpm is started.
09/09/2015
Samples are still not dissolved.
Samples for spinning disk are prepared.
0.5 w/v Rhamnose was used. Samples 2 % 4 are useless, since the GFP is only expressed after rhamnose induction, so only RFP will be visible.
Survival study
BBa_I13521 was used to make bio-ink (RFP). The hydrogel (100 μL) was made in 1 mL CaCl2 (0.1M) and then incubated in 1 mL 0.1M sodium citrate + 1 μl CAM. Incubation was carried out at 37 oC with shaking at 300 rpm.
After 36 hours, the samples were centrifuged for 3 minutes at 4000 rpm to remove the supernatant sodium citrate. It was replaced with 1 mL of LB and the sample incubated again at 37 oC with shaking at 300 rpm.
After 24 hours the sample was centrifuged for 3 minutes at 13’500 rpm. A small, red pellet was observed indicating survival of the bacterial cells under conditions used to dissolve the hydrogel.
Back to Top
Protocols
During our lab work we used the following protocols. You can also download them here. Protocols
Competent cells
Plate Top10 cells and incubate at 37ºC overnight
Pick one colony, inoculate in LB media and incubate overnight while shaking at 37ºC
Dilute the culture in fresh medium, and continue the incubation until the OD600= 0.4-0.6
Centrifugation 5 min. at 4000 rpm.
Pellet cells and resuspend in 100mM of CaCl2 solution
Incubate on ice for 20 min
Centrifugation 5 min at 3000 rpm
Pellet cells and suspend again in 100mM CaCl2 solution
Incubate on ice for 60 min
Centrifugation 5 min at 3000 rpm
Add a solution of 100mM CaCl2 + 40% glycerol
Store immediately at -80ºC
Electrocompetent Cells
Grow single Escherichia coli colony in LB overnight, at 37ºC and 200 rpm
Dilute the overnight culture 1:100 in fresh LB medium and grow at 37ºC and 200 rpm, until OD600 is between 0.4 and 0.6
Place the culture on ice for 15 minutes
Harvest the cells by centrifugation (5 minutes at 4000 rpm at 4ºC) and resuspend in ice cold Mili-Q water, to the volume of the original culture
Repeat the last washing step with Mili-Q water 2 times
Resuspend cells in equal volume as cell pellet ice cold 10% glycerol
Make aliquots of 40 µL and store at -80ºC
Transformation
Take the competent cells from the storage at -80ºC and leave them on ice for 10-15 min
Add 1-2 µL of plasmid solution to the 50 µL cell tube
Incubate on ice for 30 min
Heat-shock the cells at 42ºC for 45s
Incubate on ice for 2 min
Add 500 µL of LB media and incubate 60 min at 37ºC
Plate the cultures on agar plate
Electroporation
Use 40 µL electrocompetent cells for electroporation
Add 1-10 ng of plasmid DNA or 1 µL of ligation product to the cells
Transfer cells + DNA to 2mm electroporation cuvette
Electroporate at 2500 V, with a pulse of 5-6 ms
Cell Cultures
Pick a single colony from a plate or cryostock
Put the colony in a 50 mL sterile tube and add 5-10 mL of LB fresh medium
Put the tube to incubate for at least 16 h, at 37ºC and 200 rpm
DNA Electrophoresis
Gel making
Prepare 200 mL of TAE buffer
Mix the TAE solution with 2g of agarose (for 1%)
Heat the solution to boiling, and then cool it to 50ºC aprox.
Add 5 µL of Ethidium bromide to the solution
Pour the solution in the electrophoresis vessel. Apply the combs.
Let it polymerize, and then cover it with TAE
Gel running
Add 1/6 of total volume of Loading buffer to every DNA sample.
Remove the combs from the gel, and pipette DNA samples and DNA ladder
Run at 100-130V for 30-60 min (depends on the fragments)
Ligation
Add 20-100 ng of vector DNA (can be calculated from the DNA concentration in the sample)
Add X ng of insert DNA. X is calculated using the length of both vector and insert and the molar ratio desired.
Add 2µL of ligation buffer
Add MQ water to set the final volume to 15-20
Add 1 µL of T4 ligase (always at the end to keep the enzyme in optimal conditions)
Incubate for at least 3 hours at 16ºC
Cell Cryostock
Take 1.5 mL from a freshly grown culture and put it in a 1.5 mL tube
Spin the tube for 10 min at 2000 rpm
Decant the supernatant without disturbing the pellet
Add 0.5 mL of LB media and 0.5 mL of glycerol 80% solution
Mix by vortexing
Save in the -80ºC freezer
Restriction (NEB enzymes)
Add 1 µg of DNA (can be calculated from concentration in the sample)
Add 5 µL of NEB buffer
Add 1 µL of restriction enzyme 1
Add 1 µL of restriction enzyme 2
Add MQ water to set the final volume at 50 µL
Mix the solution by flicking the tube
Spin-down in a microcentrifuge for 15 s
Incubate at 37ºC for 1-2 hours
Plasmid isolation (from Promega PureYield ® Kit)
Add 1.5 mL of bacterial culture in LB medium to a 1.5 mL micro-centrifuge tube. Centrifuge that tube at max speed for 3 min
Remove the supernatant, and add 600 µL of MQ water to the pellet
Add 100 µL of Cell Lysis Buffer, and mix by inverting 6 times. The color change to blue indicates complete lysis
Add 350 µL of cold (4-8ºC) Neutralization Buffer, and mix by inverting the tube. The color change to yellow indicates total neutralization
Centrifugate at maximum speed for 3 minutes, and transfer the supernatant to a PureYield Minicolumn
Place the minicolumn into a PureYield Collection Tube and centrifuge at maximum speed for 15 seconds
Discard the flowthrough and place the minicolumn again into the same PureYield Collection tube
Add 200 µL of Endotoxin Removal Wash to the minicolumn. Centrifuge at maximum speed for 15 seconds. Do not empty the Collection Tube now
Add 400 µL of Column Wash Solution to the minicolumn, and centrifuge at maximum speed for 30 seconds
Transfer the minicolumn to a clean 1.5 mL tube, and 30 µL of hot (50ºC, pre-warmed) MQ water directly to the minicolumn matrix. Let stand for 5 minutes at room temperature
Centrifuge at maximum speed in a microcentrifuge for 15 seconds to elute plasmidic DNA. Cap the tube, and store the DNA solution at -20 ºC (or use it directly for cloning experiments)
Gel isolation (from Promega Wizard ® Kit)
Weigh a 1.5 mL microcentrifuge tube for each DNA fragment to be isolated, and record the weight
Visualize the DNA in the agarose gel using a long-wavelength UV lamp and an intercalating dye (Ethidium bromide). Irradiate the gel the minimum possible time to reduce nicking
Excise the DNA fragment of interest in a minimal volume of agarose using a clean scalpel or razor blade. Transfer the gel slice to a weighted 1.5 mL tube and record the weight, again. Subtract the previously measured tube weight to obtain               the weight of the gel slice containing the DNA fragment
Add Membrane Binding Solution at a ratio of 10 µL of solution per 10 mg of agarose gel slice
Vortex the mixture and incubate at 50-65ºC for 10 minutes, or until the gel slice is completely dissolve in the liquid. You can vortex the tube every few minutes to increase the rate of agarose melting
Centrifuge the tube briefly at room temperature to ensure the contents are at the bottom of the tube. Once the agarose gel is melted, the gel will not re-solidify at room temperature
Place one SV Minicolumn in a Collection Tube for each dissolved gel slice
Transfer the dissolved gel mixture to the SV minicolumn assembly and incubate for 1 minute at room temperature
Centrifuge the SV Minicolumn assembly in a microcentrifuge at max speed for 1 minute. Remove the SV Minicolumn from the Spin Column assembly and discard the liquid in the Collection Tube. Return the SV Minicolumn to the Collection Tube afterwards
Wash the column by adding 700 µL of Membrane Wash Solution, previously diluted with 95% ethanol to the SV Minicolumn. Centrifuge the SV Minicolumn assembly for 1 minute at maximum speed
Empty the Collection Tube as before, and place the SV Minicolumn back in the Collection Tube. Repeat the wash with 500 µL of Membrane Wash Solution, and centrifuge the SV Minicolumn assembly for 5 minutes at maximum speed
Remove the SV Minicolumn assembly from the centrifuge (not wetting the bottom of the column with the supernatant). Empty the Collection Tube and centrifuge the assembly for 1 minute with the microcentrifuge lid open (or off) to allow ethanol               evaporation
Carefully transfer the SV Minicolumn to a clean 1.5 mL tube. Apply 50 µL of Nuclease-Free Water (at 50ºC) directly to the center of the column, without touching the membrane with the pipette. Incubate at room temperature for 5 minutes
Centrifuge for 1 minute at 14000 rpm. Discard the SV Minicolumn, and store the tube containing the eluted DNA at 4ºC or -20ºC
Restriction product purification (from Promega Wizard ® Kit)
Add an equal volume of Membrane Binding Solution to the restriction product tube
Place one SV Minicolumn in a Collection Tube for each restriction product solution
Transfer the mixture to the SV Minicolumn assembly and incubate for 1 minute at room temperature
Centrifuge the SV Minicolumn assembly in a microcentrifuge at max speed for 1 minute. Remove the SV Minicolumn from the Spin Column assembly and discard the liquid in the Collection Tube. Return the SV Minicolumn to the Collection Tube afterwards
Wash the column by adding 700 µL of Membrane Wash Solution, previously diluted with 95% ethanol to the SV Minicolumn. Centrifuge the SV Minicolumn assembly for 1 minute at maximum speed
Empty the Collection Tube as before, and place the SV Minicolumn back in the Collection Tube. Repeat the wash with 500 µL of Membrane Wash Solution, and centrifuge the SV Minicolumn assembly for 5 minutes at maximum speed
Remove the SV Minicolumn assembly from the centrifuge (not wetting the bottom of the column with the supernatant). Empty the Collection Tube and centrifuge the assembly for 1 minute with the microcentrifuge lid open (or off) to allow ethanol               evaporation
Carefully transfer the SV Minicolumn to a clean 1.5 mL tube. Apply 50 µL of Nuclease-Free Water (at 50ºC) directly to the center of the column, without touching the membrane with the pipette. Incubate at room temperature for 5 minutes
Centrifuge for 1 minute at 14000 rpm. Discard the SV Minicolumn, and store the tube containing the eluted DNA at 4ºC or -20ºC
IDT GBlocks ® working solutions
Linearize 1 µg vector by restriction digest
Remove the 5’ phosphates from the vector with alkaline phosphatase
Purify the linearized vector using an agarose gel
Resuspend the GBlocks gene fragments to a final concentration of 10 ng/µL of water or TE buffer. The amount of DNA in each tube can be checked in the delivery document
Prepare ends of GBlocks fragments by restriction of 10 µL
Heat inactivate the enzyme, or column purify the restriction product (depending on the restriction protocol) keeping the digested insert as concentrated as possible
Ligate 50 ng of vector with a 3 times molar excess of GBlocks in fresh T4 DNA ligase buffer and 400 u of T4 DNA ligase, setting the final volume to 20 µL. Incubate the solution for 2 hours at 16º
Transform the ligation protocol into competent cells using the Transformation protocol
PCR (Phusion ®)
Colony PCR (GoTaq ®)
Make the MasterMix using the materials specified below
Pick a colony and dip in this mixture and then in 50 µL LB, so the colonies that contain the correct insert can be grown
Run the PCR using the program specified below
TOPO TA ® Cloning procedure
Materials
Salt solution, containing 1.2 M NaCl and 0.06 M MgCl2
Taq polymerase
dATP solution
10x buffer, containing 100 mM Tris-HCl, pH 8.3 (at 42°C), 500 mM KCl, 25 mM MgCl2 and 0.01% gelatin
DNA for transform into the cells
Competent CaCl2 cells
SOC medium, containing 2% Tryptone, 0.5% Yeast Extract, 10 mM NaCl, 2.5 mM KCl, 10 mM MgCl2, 10 mM MgSO4, 20 mM glucose
LB-Agar petri dishes
Methodology
Get adenine nucleotide at 3’, by adding
6 uL of DNA; 1 uL of Taq Polymerase; 1 uL of dATP; 1 uL of 10x buffer
Use 3 uL of this solution and put them in a different tube.
Add 1 uL of Salt Solution, 2 uL of miliQ water
Incubate for 10 minutes at room temperature
2 uL are added into thawed competent cells, and the mixture is incubated for 15 minutes at room temperature
Heat shock the cells at 42ºC for 30 seconds
Place the tubes on ice, and add 250 uL of SOC medium to the mixture
Grow the cells at 37ºC for 1 hour
Plate the cultures on agar plate, using the necessary antibiotic
http://www.google.nl/url?sa=t&rct=j&q=&esrc=s&source=web&cd=2&ved=0CDEQFjABahUKEwixiM67zcHHAhVEOxQKHQnADyY&url=http%3A%2F%2Ftools.thermofisher.com%2Fcontent%2Fsfs%2Fmanuals%2Ftopota_man.pdf&ei=mv7aVbHTKMT2UImAv7AC&usg=AFQjCNFl0jognsFPzru6PEToemB5eauPWg&sig2=R4P1618Ds-W-meECeGP-sg&cad=rja
Microtiter Plate Assay for Ring-Shaped (curli dependent) biofilm2
LB medium
Lb+rhamnose medium, at different concentrations for induction
Autoclaved, flat bottomed, transparent 96-well microtiter plates with lids
Crystal violet solution at 0.1% (w/v) in water
95% (v/v) ethanol
Platform shaker
Plate reader
Methodology
Make an overnight culture of the bacteria of interest in LB+antibiotic at 37°C
Dilute each overnight culture by 1:100 into 200 μL of LB-rhamnose in wells of a cell culture treated, flat bottom, transparent 96-well microtiter plates with lids. Cover the plate
Incubate the plate for ~3 days, leaving enough time for the induction to occur
Remove the planktonic bacteria from each well by pipetting or shaking out the liquid
Wash the wells with water to remove the remaining planktonic cells. Submerge the plate in a tray of water and shake out the liquid. One can also add in water and remove the liquid with a multichannel pipet. Repeat this step twice
Add 210 μL of crystal violet solution into each well and stain for 10 min at RT
Discard the crystal violet solution, washing the wells three times in a tray or by pipetting (for removing non-specific interactions of the dye)
Invert the microtiter plate and vigorously tap the plate on a paper tower to remove the remaining water in the well. Air-dry the plate
Add 200 μL of 95% ethanol into each stained well. Cover the plate and incubate it on a platform shaker at RT for 15 min
Transfer the 100 μL of liquid onto a non-sterile, flat bottom, transparent 96-well microtiter plate. Measure the OD at 600 nm on a plate reader
Zhou, Y., Smith, D. R., Hufnagel, D. A., & Chapman, M. R. (2013). Experimental manipulation of the microbial functional amyloid called curli. In Bacterial cell surfaces (pp. 53-75). Humana Press.
Isolation using Ni-column purification assay (QIAGEN kit) for extracellular protein tagged with 6xHis3
Wash buffer NPI-20
Elution Buffer NPI-500
Lysis buffer NPI-10
Methodology
Inoculate 10 mL of LB medium containing the appropriate antibiotics with a fresh bacterial colony harboring the expression plasmid. Grow at 37ºC overnight
Dilute the non-induced overnight culture 1:60 with fresh LB medium containing the appropriate antibiotics. Grow at 37ºC while shaking until the OD600 reaches 0.6
Add Rhamnose (induction) to the desired final concentration for induction, and grow at 37ºC for the appropriate induction time
Harvest the cells by centrifugation at 4000 g for 15 min
Transfer the supernatant to another tube and discard the pellet. Keep a part of the supernatant for SDS-PAGE analysis, if needed
Equilibrate the Ni-NTA spin column with 600 µL Buffer NPI-10. Centrifuge for 2 min at 2900 rpm (890 g)
Make the supernatant solution, by setting the final solution to contain 200mM NaCl, 1mM PMSF, and 20mM imidazole. The pH is adjusted to 7.8, using 1M K2HPO4 solution
Load up to 600 µL of the supernatant solution containing the 6xHis-tagged protein onto the pre-equilibrated Ni-NTA spin column. Centrifuge for 5 min at 1600 rpm (270 g) and collect the flow-through
Wash the Ni-NTA spin column twice with 600 µL Buffer NPI-20. Centrifuge for 2 min at 2900 rpm (890 g)
Elute the protein twice with 300 µL of Buffer NPI-500. Centrifuge for 2 min at 2900 rpm (890 g) and collect the eluate
Isolation using Ni-column purification assay (QIAGEN kit) for intracellular protein tagged with 6xHis3
Wash buffer NPI-20
Elution Buffer NPI-500
Lysis buffer NPI-10
Lysozyme stock solution 10 mg/mL in water
Benzonase ® Endonuclease 25 U/µL
Methodology
Inoculate 10 mL of LB medium containing the appropriate antibiotics with a fresh bacterial colony harboring the expression plasmid. Grow at 37ºC overnight
Dilute the non-induced overnight culture 1:60 with fresh LB medium containing the appropriate antibiotics. Grow at 37ºC while shaking until the OD600 reaches 0.6
Add Rhamnose (induction) to the desired final concentration for induction, and grow at 37ºC for the appropriate induction time
Harvest the cells by centrifugation at 4000 g for 15 min
Discard the supernatant and keep the pellet. Then resuspend the pellet in 630 µL of Lysis buffer (NPI10). Add 70 µL of Lysozyme stock solution (10 mg/mL) and add 3 units/mL of Benzonase®
Incubate on ice for 15-30 min
Centrifuge the lysate at 12000 g for 15-30 min at 4ªC. Collect the supernatant
Equilibrate the Ni-NTA spin column with 600 µL Buffer NPI-10. Centrifuge for 2 min at 2900 rpm (890 g)
Load up to 600 µL of the cleared lysate solution containing the 6xHis-tagged protein onto the pre-equilibrated Ni-NTA spin column. Centrifuge for 5 min at 1600 rpm (270 g) and collect the flow-through
Wash the Ni-NTA spin column twice with 600 µL Buffer NPI-20. Centrifuge for 2 min at 2900 rpm (890 g)
Elute the protein twice with 300 µL of Buffer NPI-500. Centrifuge for 2 min at 2900 rpm (890 g) and collect the eluate
Congo Red (CR) Assay to quantify curli generation0
LB agar plates
LB liquid medium
Congo Red (CR) stock: Dissolve 1 g of Congo Red in 100 mL of water and sterilize by filtering. Store at 4ºC
Chloramphenicol stock solution (35 mg/mL)
Rhamnose (0.1-1% w/v)
IPTG (4mM)
Methodology
Streak out the cells from a -80ºC cryostock onto LB+agar plate. Pick grown colonies and grow them overnight in liquid LB + antibiotic
Transfer 1 mL of the overnight culture in 30 mL LB+antibiotic, and keep it growing while shaking
Once the OD600 of the culture is around 0.4, induce the cultures with the desired amount of inductor (Rhamnose or IPTG). Wait 2 days after the induction
Take 1 mL of every sample and measure the OD600, using LB+CAM as blank
Add Congo Red to a final concentration of 20 µg/mL, mix and incubate for 5 min at room temperature
Centrifuge at 14000 rpm for 5 minutes
Take the supernatant and measure it at 480 nm (or the peak in the spectrophotometric absorption curve), using LB+CAM+Congo Red as a reference
Zhou, Y., Smith, D. R., Hufnagel, D. A., & Chapman, M. R. (2013). Experimental manipulation of the microbial functional amyloid called curli. In Bacterial cell surfaces (pp. 53-75). Humana Press.
Plate reader Assay to quantify GFP intensity for testing plasmid performance and kinetic parameters
Grow a ∆csgA - csgA and ∆csgA BBa_I13504(csgA) strain at 37°C overnight in LB medium.
Measure the OD600. As the OD600 will probably be high (>2.5), make a ~50x dilution and grow the liquid cultures at 37°C for ~2h.
Measure the OD600 again. If the OD600 is between 0.4 and 0.6 (exponential phase), continue to step 4.
Write down the OD600 that was obtained. Make a dilution to an OD600 of 0.05.
Fill the wells with different amounts of rhamnose (from 50 g / L stock) and sterile water.
Fill the wells with 180 ul LB + cells, or solely 180 ul LB.
When all the wells are filled, measure the fluorescence and OD600 in a plate reader in time (at a temperature of 30°C).
Attachment assay for testing biofilm adhesion strength on hydroxyapatite using Bos Taurus dental parts
Characterization of the surface of the samples
A tooth sample is cleaned and disinfected by ethanol bath overnight
The tooth is then carefully broken down into smaller parts
Three of these pieces are then selected based on the size and shape, taking the three cubical pieces with more resemblance
Keep the samples in ethanol, so they stay sterile before their use
Biofilm formation on the surface
Cells csgA+Hydroxyapatite+RFP affinity tag positive after induction are taken from a cryostock/plate and cultured overnight in LB media. It is important to also grow two cultures that can be used as a control, making the total number of samples               3
The cultures are then dropped in a sterile, empty 12-wells plate. After that, the dental samples are placed in the middle of one well
Next to the dental sample, Rhamnose solution is added to the dish to a final concentration of 0.5%. After this step, the plate is left for induction for 20-40 hours
Hydroxyapatite-tag strength test
The fluorescence of the samples is then measured Typhoon fluorescence reader, after two immersions in water
The samples are then placed in a tube and rinsed with 1 ml of ethanol. The mixture is left 5 minutes so all the cells can detach from the tooth
Then, the ethanol solutions are taken from the tube and placed in a 96-well plate, and the RFP intensity is measured using a fluorescence plate reader
Western Blotting
Grow a ∆csgA - csgA and ∆csgA – csgA-His strain at 37°C overnight in LB medium.
Measure the OD600. Make a ~50x dilution and grow the liquid cultures at 37°C for 1.5h.
Measure the OD600 again. If the OD600 is between 0.35 and 0.6 (exponential phase) , the induction can be made.
Induce the cells with Rhamnose (from 50 g / L stock) and sterile water. You need to have a final concentration of 0.5 wt% Rhamnose. The induction has to be performed at 30°C.
After 5 hours of induction, measure the OD600 and disrupt the cells.
Run SDS-PAGE gel with protein marker and samples (use pre-stained protein marker)
Transfer protein from gel to membrane:
Soak 2 x filter thick paper and the gel in transfer buffer.
Soak 1 x PVDF membrane in 100% methanol, then water, then transfer buffer or use cellulose membranes soaked only in transfer buffer.
Make sandwich: filter bottom paper, membrane, gel (upside down), filter top paper.
Roll out bubbles with glass pipette.
Attach top electrode plate.
Transfer 40 min at 15V.
Development blot with Supersignal West pico kit (Pierce):
Block blot membrane in 5% milk or BSA in TBS-T, on orbital shaker 1 hour (RT) to overnight (cold room).
Add anti-His-HRP antibody (dilution 1:5000) in blocking solution and incubate on orbital shaker for 1h at room temperature.
Wash 4 x 10min with TBS-T on orbital shaker, RT.
Detection (Supersignal West pico kit)
Put blot membrane on Saran wrap
Mix 1.5 ml detection reagent 1 and 1.5 ml detection reagent just before use
Pipette 3 mL of mix detection reagent on blot (make sure there is even distribution on the membrane)
Incubate 5 minutes at RT in dark
detect chemiluminescence with CCD camera (Biorad Imager in Biobrick)
Spinning Disk Sample preparation for visualization of bacterial biofilm layers
Materials
1 % w/v Sodium alginate
0.1M CaCl2
0.1M sodium citrate (monobasic)
5 % w/v L-Rhamnose
Preparation bio-ink
Grow cells overnight in ~ 5mL LB + CAM.
Spin down cells at 4000 rpm for 3 min
Discard supernatant
Resuspend pellet in 100 μL LB and transfer to 1.5 mL tube
Add 55.6 μL 5% w/v rhamnose to get a 0.5% rhamnose concentration
Add 400 μL alginate and vortex
Printing
Make line with pipet (~10-20 μL each) on plasma cleaned cover slip
Place cover slip onto a tissue
Add CaCl2 by dripping it on the alginate lines with pipet
Place the cover slip in vertical position on the tissue to remove most of the CaCl2
Retake above steps for the second line
Store
Put a tissue in a petri dish and make it wet
Put a piece of parafilm on top and place the sample on top of the parafilm
Store at room temperature
After induction with rhamnose wait at least 24h before dissolving the hydro gel
Dissolving gel
Place cover slip onto a tissue
Add sodium citrate by dripping it on the alginate lines with pipet
Place the cover slip in vertical position on the tissue to remove most of the sodium citrate
Store sample again in the petri dish
Transmission Electron Microscopy (TEM) Sample preparation for visualization of bacterial amyloid curli
Make an overnight culture of the cells containing the plasmid with csgA, using LB+CAM, at 37ºC and 220 RPM
Dilute the cultures to a final OD600 of 0.05, by taking a small part of the overnight culture and adding LB+CAM
Let the cultures grow at 37ºC and 220 RPM until the OD600 reaches a value between 0.4-0.6
Induce the cultures with different concentrations of L-rhamnose, from 0% to 1%
Add a small volume (1%) of the sample directly on the grid of the TEM
Take pictures of the visualized bacteria
Blue Fluorescent Protein (BFP) assay
Gel making
Grow BFP_SpyCatcher_His (TOP10) and empty TOP10 strain at 37°C overnight in LB medium.
Measure the OD600. As the OD600 will probably be high (>2.5), make a ~50x dilution and grow the liquid cultures at 37°C for ~2h.
Measure the OD600 again. If the OD600 is between 0.4 and 0.6 (exponential phase), continue to step 4.
Fill the wells with different amounts of arabinose (from 100 g / L stock) and sterile water.
Fill the wells with 180 ul LB + cells, or solely 180 ul LB.
When all the wells are filled, measure the fluorescence and OD600 in a plate reader in time (at a temperature of 30°C).
Contact Details
Address:
Faculty of Applied Sciences
Room F109
TU Delft
Lorentzweg 1
2628 CJ Delft
The Netherlands
Phone:
+31 15 278 18 62
Email:
tudelft.igem@gmail.com
