<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU_Delft_Measurement skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU Delft/Measurement</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mobile-navigation"><UL><LI><A href="https://2015.igem.org/Team:TU_Delft">Home</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Achievements">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Achievements#prizes">Prizes and Nominations</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Achievements#jamboree">Giant Jamboree</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Achievements#medal">Medal Requirements</A></LI></UL><LI><A href="https://2015.igem.org/Team:TU_Delft/Description">Project</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Description#description">Description</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Description#strains">Strains</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Description#cells">Linking Cells</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Description#tags">Affinity Tags</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Description#printing">Printing Biofilm Layers</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Description#references">References</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Parts#overview">Overview</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Parts#table">Parts</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Parts#collection">Part Collection</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Notebook#work">Work Space</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Notebook#safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Notebook#notes">Day Notes</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Notebook#protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Design">Hardware</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Design#overview">Overview</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Design#3dbiofilms">Making 3D Biofilms</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Design#biolinker">The Biolinker</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Design#integration">Design and Integration</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Design#manuals">DIY Manuals</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Design#protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Modeling#overview">Overview</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Modeling#gene">Kinetic Modeling</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Modeling#curli">Curli Formation</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Modeling#biofilm">Biofilm Strength</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Modeling#p&amp;p">Time for Printing</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices">Policy&amp;Practice</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices#overview">Overview</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices#regulations">Regulations</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices#ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices#social">Experts' Opinions</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices#community">iGEM Community</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices#tool" class="sub-nav-item">Biolink P&amp;P Tool</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices#business">Business Plan</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices#outreach">Outreach</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Collaborations">Collaboration</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Collaborations#overview">Overview</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Collaborations#testing">Testing Biolinker</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Collaborations#course">Dutch iGEM Teams Meeting</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Collaborations#cooperations">Cooperations</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Attributions#members">Team Members</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Attributions#advisors">Advisors</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Attributions#support">Team Work Attributions</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Attributions#media">Media</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Attributions#sponsors">Sponsors</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Attributions#acknowledgements">Acknowledgements</A></LI></DIV><DIV id="headerWrapper" class="hidden-xs hidden-sm"><DIV id="header"><DIV id="navigation"><UL><LI><A href="https://2015.igem.org/Team:TU_Delft/Achievements" class="achievements nav-item" id="achievements">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Description" class="project nav-item" id="project">Project</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Parts" class="results nav-item" id="parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Notebook" class="notebook nav-item" id="notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Design" class="hardware nav-item" id="hardware">Hardware</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Modeling" class="modeling nav-item" id="modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices" class="practices nav-item" id="practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Collaborations" class="collaborations nav-item" id="collaborations">Collaboration</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Attributions" class="attributions nav-item" id="attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Measurement" class="interlab nav-item" id="interlab">InterLab</A></LI></UL></DIV></DIV><DIV id="sub-navigation-wrapper"><DIV id="sub-navigation"><UL id="notebook-sub"><LI><A href="https://2015.igem.org/Team:TU_Delft/Notebook#work" class="sub-nav-item">Work Space</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Notebook#safety" class="sub-nav-item">Safety</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Notebook#notes" class="sub-nav-item">Day Notes</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Notebook#protocols" class="sub-nav-item">Protocols</A></LI></UL><UL id="achievements-sub"><LI><A href="https://2015.igem.org/Team:TU_Delft/Achievements#prizes" class="sub-nav-item">Prizes and Nominations</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Achievements#jamboree" class="sub-nav-item">Giant Jamboree</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Achievements#medal" class="sub-nav-item">Medal Requirements</A></LI></UL><UL id="attributions-sub"><LI><A href="https://2015.igem.org/Team:TU_Delft/Attributions#members" class="sub-nav-item">Team Members</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Attributions#advisors" class="sub-nav-item">Advisors</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Attributions#support" class="sub-nav-item">Team Work Attributions</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Attributions#media" class="sub-nav-item">Media</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Attributions#sponsors" class="sub-nav-item">Sponsors</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Attributions#acknowledgements" class="sub-nav-item">Acknowledgements</A></LI></UL><UL id="project-sub"><LI><A href="https://2015.igem.org/Team:TU_Delft/Description#description" class="sub-nav-item">Description</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Description#strains" class="sub-nav-item">Strains</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Description#cells" class="sub-nav-item">Linking Cells</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Description#tags" class="sub-nav-item">Affinity Tags</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Description#printing" class="sub-nav-item">Printing Biofilm Layers</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Description#references" class="sub-nav-item">References</A></LI></UL><UL id="parts-sub"><LI><A href="https://2015.igem.org/Team:TU_Delft/Parts#overview" class="sub-nav-item">Overview</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Parts#table" class="sub-nav-item">Parts</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Parts#collection" class="sub-nav-item">Part Collection</A></LI></UL><UL id="hardware-sub"><LI><A href="https://2015.igem.org/Team:TU_Delft/Design#overview" class="sub-nav-item">Overview</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Design#3dbiofilms" class="sub-nav-item">Making 3D Biofilms</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Design#biolinker" class="sub-nav-item">The Biolinker</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Design#integration" class="sub-nav-item">Design and Integration</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Design#manuals" class="sub-nav-item">DIY Manuals</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Design#protocols" class="sub-nav-item">Protocols</A></LI></UL><UL id="collaborations-sub"><LI><A href="https://2015.igem.org/Team:TU_Delft/Collaborations#overview" class="sub-nav-item">Overview</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Collaborations#testing" class="sub-nav-item">Testing Biolinker</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Collaborations#course" class="sub-nav-item">Dutch iGEM Teams Meeting</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Collaborations#cooperations" class="sub-nav-item">Cooperations</A></LI></UL><UL id="practices-sub"><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices#overview" class="sub-nav-item">Overview</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices#regulations" class="sub-nav-item">Regulations</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices#ethics" class="sub-nav-item">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices#social" class="sub-nav-item">Experts' Opinions</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices#community" class="sub-nav-item">iGEM Community</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices#tool" class="sub-nav-item">Biolink P&amp;P Tool</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices#business" class="sub-nav-item">Business Plan</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Practices#outreach" class="sub-nav-item">Outreach</A></LI></UL><UL id="interlab-sub"><LI><A href="https://2015.igem.org/Team:TU_Delft/Measurement#results" class="sub-nav-item">Results</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Measurement#protocols" class="sub-nav-item">Protocols</A></LI></UL><UL id="modeling-sub"><LI><A href="https://2015.igem.org/Team:TU_Delft/Modeling#overview" class="sub-nav-item">Overview</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Modeling#gene" class="sub-nav-item">Kinetic Modeling</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Modeling#curli" class="sub-nav-item">Curli Formation</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Modeling#biofilm" class="sub-nav-item">Biofilm Strength</A></LI><LI><A href="https://2015.igem.org/Team:TU_Delft/Modeling#p&amp;p" class="sub-nav-item">Time for Printing</A></LI></UL></DIV></DIV></DIV><DIV class="jumbotron" id="cover"><H1>InterLab Study</H1><P class="lead">The Second International InterLab Measurement Study in Synthetic Biology.</P><P class="lead">Among 81 other teams from all over the world, our team decided to participate in this year’s iGEM Interlab Study of 2015.</P></DIV><DIV id="white"><DIV class="jumbotron"><DIV class="container"><H1>Results</H1></DIV></DIV><DIV class="container"><P class="lead">For this study, three devices needed to be constructed consisting of a constitutive Anderson promoter with low (J23117), medium (J23106) and high (J23101) promoter strengths expressing GFP (I13504) as reporter. Using the Biobrick distribution, the
      required promoters and GFP were cloned in E.coli top10 cells, restricted and ligated to afford the final devices. Fluorescence was measured from overnight cultures using FACS (fluorescence-activated cell sorting) to compare the individual promoter
      strengths in relative units.
    </P><P class="lead">
      To measure fluorescence, single cells were excited at a wavelength of 488 nm and emitted light was measured at 530 ± 30 nm and a total of 30’000 events were recorded for each sample. The data is presented in relative units of fluorescence on the x axis and fluorescent side scattering (cell size, a constant parameter) on the y axis. A shift to the right side thus corresponds to an increase in fluorescence. 

    </P><P class="lead">
      In order to compare the three different promoters described above, a histogram showing the fluorescence intensity at 530 nm versus the number of cells is made (figure 1.). In the figure, it is clear how J23101 leads to a better GFP expression in average,
      while J23106 and J23117 show a lower performance in that order.
    </P><A href="#" data-toggle="modal" data-target=".bs-example-modal-lg1"><FIGURE><FIGCAPTION>Figure 1. Histogram of fluorescence intensity at 530 nm per number of cells, comparing the reporter GFP intensity after J23101, J23106 and J23117 promoters.
        </FIGCAPTION></FIGURE></A><DIV class="modal fade bs-example-modal-lg1" tabindex="-1" role="dialog" aria-labelledby="myLargeModalLabel1"><DIV class="modal-dialog modal-lg" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="Close" style="background:transparent"><SPAN aria-hidden="true">×</SPAN></BUTTON></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion1" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="headingOne"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion1" href="#collapseOne" aria-expanded="false" aria-controls="collapseOne">
			  Negative and positive control
					  </A></DIV><DIV id="collapseOne" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne"><DIV class="panel-body"><DIV class="col-md-5"><P class="lead">
                Plasmids containing only the promoter without GFP were used as negative controls to calibrate the FACS machine. The Biobrick I20270 (containing J23151 as constitutive promoter, obtained from the distribution) was used as a positive control. x label: fluorescence
                (relative units); y label: FSC fluorescence side scattering (cell size)
              </P></DIV><DIV class="col-md-7"><DIV class="modal fade bs-example-modal-lg2" tabindex="-1" role="dialog" aria-labelledby="myLargeModalLabel2"><DIV class="modal-dialog modal-lg" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="Close" style="background:transparent"><SPAN aria-hidden="true">×</SPAN></BUTTON></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion2" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="headingTwo"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion2" href="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
			  High promoter strength
					</A></DIV><DIV id="collapseTwo" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingTwo"><DIV class="panel-body"><DIV class="col-md-5"><P class="lead">      J23101_I13504 was grown in biological triplicate (hi clone 1,2,3) and was measured in technical triplicate (a,b,c). x label: fluorescence (relative units); y label: FSC fluorescence side scattering (cell size)
              </P></DIV><DIV class="col-md-7"><DIV class="modal fade bs-example-modal-lg3" tabindex="-1" role="dialog" aria-labelledby="myLargeModalLabel3"><DIV class="modal-dialog modal-lg" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="Close" style="background:transparent"><SPAN aria-hidden="true">×</SPAN></BUTTON></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion3" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="headingThree"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion3" href="#collapseThree" aria-expanded="false" aria-controls="collapseThree">
			  Medium promoter strength
					  </A></DIV><DIV id="collapseThree" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingThree"><DIV class="panel-body"><DIV class="col-md-5"><P class="lead">
                J23106_I13504 was grown in biological triplicate (med clone 1,2,3) and was measured in technical triplicate (a,b,c). x label: fluorescence (relative units); y label: FSC fluorescence side scattering (cell size)
              </P></DIV><DIV class="col-md-7"><DIV class="modal fade bs-example-modal-lg4" tabindex="-1" role="dialog" aria-labelledby="myLargeModalLabel4"><DIV class="modal-dialog modal-lg" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="Close" style="background:transparent"><SPAN aria-hidden="true">×</SPAN></BUTTON></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion4" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="headingFour"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion4" href="#collapseFour" aria-expanded="false" aria-controls="collapseFour">
			Low promoter strength
					</A></DIV><DIV id="collapseFour" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingFour"><DIV class="panel-body"><DIV class="col-md-5"><P class="lead">
                J23106_I13504 was grown in biological triplicate (low clone 1,2,3) and was measured in technical triplicate (a,b,c). x label: fluorescence; y label: FSC fluorescence side scattering (cell size)
              </P></DIV><DIV class="col-md-7"><DIV class="modal fade bs-example-modal-lg5" tabindex="-1" role="dialog" aria-labelledby="myLargeModalLabel5"><DIV class="modal-dialog modal-lg" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="Close" style="background:transparent"><SPAN aria-hidden="true">×</SPAN></BUTTON></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><P class="lead">
     
To process our raw data, the relative fluorescence of the corresponding negative controls, which were cells containing the same promoter without GFP, was subtracted from the measured samples expressing GFP to obtain our final data in relative units. 

    </P><DIV class="modal fade bs-example-modal-lg6" tabindex="-1" role="dialog" aria-labelledby="myLargeModalLabel6"><DIV class="modal-dialog modal-lg" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="Close" style="background:transparent"><SPAN aria-hidden="true">×</SPAN></BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade bs-example-modal-lg7" tabindex="-1" role="dialog" aria-labelledby="myLargeModalLabel7"><DIV class="modal-dialog modal-lg" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="Close" style="background:transparent"><SPAN aria-hidden="true">×</SPAN></BUTTON></DIV></DIV></DIV></DIV><P class="lead">
      As expected, high, medium and low levels of fluorescence were measured in accordance with the promoter strengths used in our constructs. The weak promoter (J23117) however was lower than expected with only approximately 15 relative units surpassing the negative controls. In contrast, the medium promoter (J23106) is approximately 70 times stronger than the weak promoter (J23117) and the high promoter (J23101) is approximately 170 times stronger than the weak one (J23117). Compared, the high promoter (J23101) is approximately 2.3 times stronger than the medium promoter (J23106). 

    </P><P class="lead">
     Using FACS, we measured 30’000 single cells and recorded the fluorescence level in triplicate for each sample. Surprisingly, these data sets are still prone to large errors with a standard deviation of roughly 10%. 
    </P></DIV></DIV><DIV id="grey"><DIV class="jumbotron"><DIV class="container"><H1>Protocols</H1><P class="lead text-muted">Protocols used during the Interlab Study and the Sequencing Files</P></DIV></DIV><DIV class="container"><DIV class="panel-group" id="accordionprotocols" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="heading1"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordionprotocols" href="#collapse1" aria-expanded="false" aria-controls="collapse1">
 Competent cells
          </A></DIV><DIV id="collapse1" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading1"><DIV class="panel-body"><OL style="list-style: decimal inside; font-size:19px;"><LI><P class="lead">Plate Top10 cells and incubate at 37ºC overnight</P></LI><LI><P class="lead">Pick one colony, inoculate in LB media and incubate overnight while shaking at 37ºC</P></LI><LI><P class="lead">Dilute the culture in fresh medium, and continue the incubation until the OD<SUB>600</SUB>= 0.4-0.6</P></LI><LI><P class="lead">Centrifugation 5 min. at 4000 rpm.</P></LI><LI><P class="lead">Pellet cells and resuspend in 100mM of CaCl<SUB>2</SUB> solution</P></LI><LI><P class="lead">Incubate on ice for 20 min</P></LI><LI><P class="lead">Centrifugation 5 min at 3000 rpm</P></LI><LI><P class="lead">Pellet cells and suspend again in 100mM CaCl<SUB>2</SUB> solution</P></LI><LI><P class="lead">Incubate on ice for 60 min</P></LI><LI><P class="lead">Centrifugation 5 min at 3000 rpm</P></LI><LI><P class="lead">Add a solution of 100mM CaCl<SUB>2</SUB> + 40% glycerol</P></LI><LI><P class="lead">Store immediately at -80ºC</P></LI></OL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="heading2"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordionprotocols" href="#collapse2" aria-expanded="false" aria-controls="collapse2">
           Transformation
          </A></DIV><DIV id="collapse2" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading2"><DIV class="panel-body"><OL style="list-style: decimal inside; font-size:19px;"><LI><P class="lead">Take the competent cells from the storage at -80ºC and leave them on ice for 10-15 min</P></LI><LI><P class="lead">Add 1-2 µL of plasmid solution to the 50 µL cell tube</P></LI><LI><P class="lead">Incubate on ice for 30 min</P></LI><LI><P class="lead">Heat-shock the cells at 42ºC for 45s</P></LI><LI><P class="lead">Incubate on ice for 2 min</P></LI><LI><P class="lead">Add 500 µL of LB media and incubate 60 min at 37ºC</P></LI><LI><P class="lead">Plate the cultures on agar plate</P></LI></OL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="heading3"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordionprotocols" href="#collapse3" aria-expanded="false" aria-controls="collapse3">
            Cell Cultures
     </A></DIV><DIV id="collapse3" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading3"><DIV class="panel-body"><OL style="list-style: decimal inside; font-size:19px;"><LI><P class="lead">Pick a single colony from a plate or cryostock</P></LI><LI><P class="lead">Put the colony in a 50 mL sterile tube and add 5-10 mL of LB fresh medium</P></LI><LI><P class="lead">Put the tube to incubate for at least 16 h, at 37ºC and 200 rpm</P></LI></OL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="heading4"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordionprotocols" href="#collapse4" aria-expanded="false" aria-controls="collapse4">
            DNA Electrophoresis
          </A></DIV><DIV id="collapse4" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading4"><DIV class="panel-body"><H3>Gel making</H3><OL style="list-style: decimal inside; font-size:19px;"><LI><P class="lead">Prepare 200 mL of TAE buffer</P></LI><LI><P class="lead">Mix the TAE solution with 2g of agarose (for 1%)</P></LI><LI><P class="lead">Heat the solution to boiling, and then cool it to 50ºC aprox.</P></LI><LI><P class="lead">Add 5 µL of Ethidium bromide to the solution</P></LI><LI><P class="lead">Pour the solution in the electrophoresis vessel. Apply the combs.</P></LI><LI><P class="lead">Let it polymerize, and then cover it with TAE</P></LI></OL><H3>Gel running</H3><OL style="list-style: decimal inside; font-size:19px;"><LI><P class="lead">Add 1/6 of total volume of Loading buffer to every DNA sample.
                  </P></LI><LI><P class="lead">Remove the combs from the gel, and pipette DNA samples and DNA ladder
                      </P></LI><LI><P class="lead">Run at 100-130V for 30-60 min (depends on the fragments)</P></LI></OL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="heading5"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordionprotocols" href="#collapse5" aria-expanded="false" aria-controls="collapse5">
Ligation
          </A></DIV><DIV id="collapse5" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading5"><DIV class="panel-body"><OL style="list-style: decimal inside; font-size:19px;"><LI><P class="lead">Add 20-100 ng of vector DNA (can be calculated from the DNA concentration in the sample)</P></LI><LI><P class="lead">Add X ng of insert DNA. X is calculated using the length of both vector and insert and the molar ratio desired.</P></LI><LI><P class="lead">Add 2µL of ligation buffer</P></LI><LI><P class="lead">Add MQ water to set the final volume to 15-20</P></LI><LI><P class="lead"> Add 1 µL of T4 ligase (always at the end to keep the enzyme in optimal conditions)</P></LI><LI><P class="lead">Incubate for at least 3 hours at 16ºC</P></LI></OL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="heading6"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordionprotocols" href="#collapse6" aria-expanded="false" aria-controls="collapse6">
           Cell Cryostock
          </A></DIV><DIV id="collapse6" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading6"><DIV class="panel-body"><OL style="list-style: decimal inside; font-size:19px;"><LI><P class="lead">Take 1.5 mL from a freshly grown culture and put it in a 1.5 mL tube</P></LI><LI><P class="lead">Spin the tube for 10 min at 2000 rpm</P></LI><LI><P class="lead"> Decant the supernatant without disturbing the pellet</P></LI><LI><P class="lead">Add 0.5 mL of LB media and 0.5 mL of glycerol 80% solution</P></LI><LI><P class="lead">Mix by vortexing</P></LI><LI><P class="lead">Save in the -80ºC freezer</P></LI></OL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="heading7"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordionprotocols" href="#collapse7" aria-expanded="false" aria-controls="collapse7">
            Restriction (NEB enzymes)
          </A></DIV><DIV id="collapse7" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading7"><DIV class="panel-body"><OL style="list-style: decimal inside; font-size:19px;"><LI><P class="lead">Add 1 µg of DNA (can be calculated from concentration in the sample)</P></LI><LI><P class="lead">Add 5 µL of NEB buffer</P></LI><LI><P class="lead">Add 1 µL of restriction enzyme 1</P></LI><LI><P class="lead">Add 1 µL of restriction enzyme 2</P></LI><LI><P class="lead">Add MQ water to set the final volume at 50 µL </P></LI><LI><P class="lead">Mix the solution by flicking the tube</P></LI><LI><P class="lead">Spin-down in a microcentrifuge for 15 s</P></LI><LI><P class="lead">Incubate at 37ºC for 1-2 hours</P></LI></OL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="heading8"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordionprotocols" href="#collapse8" aria-expanded="false" aria-controls="collapse8">
            Plasmid isolation (from Promega PureYield ® Kit)
          </A></DIV><DIV id="collapse8" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading8"><DIV class="panel-body"><OL style="list-style: decimal inside; font-size:19px;"><LI><P class="lead">Add 1.5 mL of bacterial culture in LB medium to a 1.5 mL micro-centrifuge tube. Centrifuge that tube at max speed for 3 min</P></LI><LI><P class="lead">Remove the supernatant, and add 600 µL of MQ water to the pellet</P></LI><LI><P class="lead">Add 100 µL of Cell Lysis Buffer, and mix by inverting 6 times. The color change to blue indicates complete lysis</P></LI><LI><P class="lead">Add 350 µL of cold (4-8ºC) Neutralization Buffer, and mix by inverting the tube. The color change to yellow indicates total neutralization</P></LI><LI><P class="lead">Centrifugate at maximum speed for 3 minutes, and transfer the supernatant to a PureYield Minicolumn

                </P></LI><LI><P class="lead">Place the minicolumn into a PureYield Collection Tube and centrifuge at maximum speed for 15 seconds

                </P></LI><LI><P class="lead">Discard the flowthrough and place the minicolumn again into the same PureYield Collection tube</P></LI><LI><P class="lead">Add 200 µL of Endotoxin Removal Wash to the minicolumn. Centrifuge at maximum speed for 15 seconds. Do not empty the Collection Tube now</P></LI><LI><P class="lead"> Add 400 µL of Column Wash Solution to the minicolumn, and centrifuge at maximum speed for 30 seconds

                </P></LI><LI><P class="lead">Transfer the minicolumn to a clean 1.5 mL tube, and 30 µL of hot (50ºC, pre-warmed) MQ water directly to the minicolumn matrix. Let stand for 5 minutes at room temperature</P></LI><LI><P class="lead">Centrifuge at maximum speed in a microcentrifuge for 15 seconds to elute plasmidic DNA. Cap the tube, and store the DNA solution at -20 ºC (or use it directly for cloning experiments)</P></LI></OL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="heading9"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordionprotocols" href="#collapse9" aria-expanded="false" aria-controls="collapse9">
          Gel isolation (from Promega Wizard ® Kit)
          </A></DIV><DIV id="collapse9" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading9"><DIV class="panel-body"><OL style="list-style: decimal inside; font-size:19px;"><LI><P class="lead">Weigh a 1.5 mL microcentrifuge tube for each DNA fragment to be isolated, and record the weight</P></LI><LI><P class="lead">Visualize the DNA in the agarose gel using a long-wavelength UV lamp and an intercalating dye (Ethidium bromide). Irradiate the gel the minimum possible time to reduce nicking</P></LI><LI><P class="lead">Excise the DNA fragment of interest in a minimal volume of agarose using a clean scalpel or razor blade. Transfer the gel slice to a weighted 1.5 mL tube and record the weight, again. Subtract the previously measured tube weight to obtain
                  the weight of the gel slice containing the DNA fragment</P></LI><LI><P class="lead">Add Membrane Binding Solution at a ratio of 10 µL of solution per 10 mg of agarose gel slice</P></LI><LI><P class="lead">Vortex the mixture and incubate at 50-65ºC for 10 minutes, or until the gel slice is completely dissolve in the liquid. You can vortex the tube every few minutes to increase the rate of agarose melting

                </P></LI><LI><P class="lead">Centrifuge the tube briefly at room temperature to ensure the contents are at the bottom of the tube. Once the agarose gel is melted, the gel will not re-solidify at room temperature</P></LI><LI><P class="lead">Place one SV Minicolumn in a Collection Tube for each dissolved gel slice</P></LI><LI><P class="lead">Transfer the dissolved gel mixture to the SV minicolumn assembly and incubate for 1 minute at room temperature</P></LI><LI><P class="lead">Centrifuge the SV Minicolumn assembly in a microcentrifuge at max speed for 1 minute. Remove the SV Minicolumn from the Spin Column assembly and discard the liquid in the Collection Tube. Return the SV Minicolumn to the Collection Tube
                  afterwards</P></LI><LI><P class="lead">Wash the column by adding 700 µL of Membrane Wash Solution, previously diluted with 95% ethanol to the SV Minicolumn. Centrifuge the SV Minicolumn assembly for 1 minute at maximum speed

                </P></LI><LI><P class="lead">Empty the Collection Tube as before, and place the SV Minicolumn back in the Collection Tube. Repeat the wash with 500 µL of Membrane Wash Solution, and centrifuge the SV Minicolumn assembly for 5 minutes at maximum speed</P></LI><LI><P class="lead"> Remove the SV Minicolumn assembly from the centrifuge (not wetting the bottom of the column with the supernatant). Empty the Collection Tube and centrifuge the assembly for 1 minute with the microcentrifuge lid open (or off) to allow ethanol
                  evaporation</P></LI><LI><P class="lead">Carefully transfer the SV Minicolumn to a clean 1.5 mL tube. Apply 50 µL of Nuclease-Free Water (at 50ºC) directly to the center of the column, without touching the membrane with the pipette. Incubate at room temperature for 5 minutes</P></LI><LI><P class="lead">Centrifuge for 1 minute at 14000 rpm. Discard the SV Minicolumn, and store the tube containing the eluted DNA at 4ºC or -20ºC</P></LI></OL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="heading10"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordionprotocols" href="#collapse10" aria-expanded="false" aria-controls="collapse10">
            Restriction product purification (from Promega Wizard ® Kit)
          </A></DIV><DIV id="collapse10" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading10"><DIV class="panel-body"><OL style="list-style: decimal inside; font-size:19px;"><LI><P class="lead">Add an equal volume of Membrane Binding Solution to the restriction product tube</P></LI><LI><P class="lead">Place one SV Minicolumn in a Collection Tube for each restriction product solution</P></LI><LI><P class="lead">Transfer the mixture to the SV Minicolumn assembly and incubate for 1 minute at room temperature

                </P></LI><LI><P class="lead">Centrifuge the SV Minicolumn assembly in a microcentrifuge at max speed for 1 minute. Remove the SV Minicolumn from the Spin Column assembly and discard the liquid in the Collection Tube. Return the SV Minicolumn to the Collection Tube
                  afterwards</P></LI><LI><P class="lead"> Wash the column by adding 700 µL of Membrane Wash Solution, previously diluted with 95% ethanol to the SV Minicolumn. Centrifuge the SV Minicolumn assembly for 1 minute at maximum speed

                </P></LI><LI><P class="lead">Empty the Collection Tube as before, and place the SV Minicolumn back in the Collection Tube. Repeat the wash with 500 µL of Membrane Wash Solution, and centrifuge the SV Minicolumn assembly for 5 minutes at maximum speed</P></LI><LI><P class="lead">Remove the SV Minicolumn assembly from the centrifuge (not wetting the bottom of the column with the supernatant). Empty the Collection Tube and centrifuge the assembly for 1 minute with the microcentrifuge lid open (or off) to allow ethanol
                  evaporation</P></LI><LI><P class="lead">Carefully transfer the SV Minicolumn to a clean 1.5 mL tube. Apply 50 µL of Nuclease-Free Water (at 50ºC) directly to the center of the column, without touching the membrane with the pipette. Incubate at room temperature for 5 minutes</P></LI><LI><P class="lead">Centrifuge for 1 minute at 14000 rpm. Discard the SV Minicolumn, and store the tube containing the eluted DNA at 4ºC or -20ºC</P></LI></OL></DIV></DIV></DIV><P class="lead text-center" style="margin-top:30px">Here you can download an archive of our sequencing files. Click the download button</P><BUTTON type="button" class="btn btn-default btn-sm"><A href="https://static.igem.org/mediawiki/2015/8/89/TU_Delft_Sequencing_Files.zip"> Download
        </A></BUTTON></DIV></DIV><DIV class="container"><DIV class="footer-left"><DIV class="row"><H2 style="text-align:center">Contact Details</H2></DIV><DIV class="row"><DIV class="col-xs-6 col-sm-5 col-md-4 col-lg-4"><P class="lead thick"> Address: </P></DIV><DIV class="col-xs-6 col-sm-7 col-md-8 col-lg-8"><P class="lead"><SPAN class="text-muted">Faculty of Applied Sciences</SPAN></P><P class="lead"><SPAN class="text-muted">Room F109</SPAN></P><P class="lead"><SPAN class="text-muted">TU Delft</SPAN></P><P class="lead"><SPAN class="text-muted">Lorentzweg 1</SPAN></P><P class="lead"><SPAN class="text-muted">2628 CJ Delft</SPAN></P><P class="lead"><SPAN class="text-muted">The Netherlands</SPAN></P></DIV></DIV><DIV class="row"><DIV class="col-xs-6 col-sm-5 col-md-4 col-lg-4"><P class="lead thick"> Phone: </P></DIV><DIV class="col-xs-6 col-sm-7 col-md-8 col-lg-8"><P class="lead"><SPAN class="text-muted">+31 15 278 18 62</SPAN></P></DIV></DIV><DIV class="row"><DIV class="col-xs-6 col-sm-5 col-md-4 col-lg-4"><P class="lead thick"> Email: </P></DIV><DIV class="col-xs-6 col-sm-7 col-md-8 col-lg-8"><P class="lead"><SPAN class="text-muted">tudelft.igem@gmail.com</SPAN></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>