"
Page
Discussion
View source
History
teams
Log in
 
Team:WashU/Yeast/Background
From 2010.igem.org
Home
Team
Undergraduates
Mentors
Advisors
Pictures
Yeast
Background
Project Goals
Results
Splicing
Background
Project Goals
Results
Parts
Notebook
Molecular Biology Group
Yeast Group
Microscopy Group
Biobrick Group
Safety
Yeast Vectors
Currently there exist two different methods for insertion of synthetic DNA into yeast cells, plasmid vectors or chromosomal integration. Plasmid vectors contain the components necessary to replicate autonomously after entering the yeast cells.  Chromosomal integration inserts DNA directly into the yeast chromosome and contains no origin of replication.
Yeast Plasmids
There exist two main types of plasmids used to insert and maintain DNA in yeast cells, Yeast Episomal plasmids (YEp) and Yeast Centromer containing plasmids (YCp).  Natively yeast contain a plasmid referred to as the 2 μm plasmid, YEp vectors use an origin of replication derived from this plasmid and maintain copy numbers of 50-100. The YCp contain an Autonomous Replicating Sequence (ARS) and a yeast Centromer sequence (CEN) and maintain a copy number of 1-3 per cell. (Keshav)
Commonly these plasmids will also contain an E. coli origin of replication and selectable marker, this allows for DNA manipulation in E. coli, and then the transfer of the plasmid directly into yeast, earning this type of plasmid the name "Shuttle Vectors".
However, the use of these plasmids is not without drawbacks.  Both types of origins face significance loss of plasmid on the order of 10-2.  When using the 2 μm replication origin and grown in selective media 60% to 95% of yeast cells may not contain the plasmid (Sherman).  Additionally the variability in copy number can introduces noise when quantify the amount of protein expressed in cells.
Chromosomal Integration
An alternative to the use of plasmids in yeast is direct integration into the chromosome of yeast cells.  Yeast cells natively have high levels of homologous recombination activity, consequently when a DNA sequence is introduced into a yeast cell containing sequences homologous to those present on the yeast chromosome, it can be swapped into the yeast chromosome.  This method is commonly used to create knock out strains excising genes such as Ura3, His3, Trp1, His3 and Lys2.  It can also be used to insert synthetic DNA constructs into the genome.  Chromosomally integrated DNA is lost at a much lower rate, approximately 10^-3 to 10^-4, than plasmid discharge (Sherman).  However transformation efficiency is lower than with plasmids, on an order of 10^3 transformants/μg opposed to efficiency of 10^5 transformants/μg when plasmids are used. Yeast Integrating plasmids (YIp) contain homologous sequences to the yeast chromosome and cause integration into the yeast chromosome.  To increase transformation efficiency by 10^2 these plasmids are linearized before transformation (Keshav)
Yeast Selection
Works Cited
S. Keshav, J. Heinemann. Yeast Plasmids, 1997 v.62, 113-130
An Introduction to the Genetics and Molecular Biology of the Yeast Saccharomyces cerevisiae. F. Sherman, 1998. http://dbb.urmc.rochester.edu/labs/sherman_f/yeast/Index.html
Yeast Protocols Handbook. Clontech, 1990
Retrieved from " http://2010.igem.org/Team:WashU/Yeast/Background "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
