"
Page
Discussion
View source
History
teams
Log in
 
Team:WashU/Splicing/Background
From 2010.igem.org
Home
Team
Undergraduates
Mentors
Advisors
Pictures
Yeast
Background
Project Goals
Results
Splicing
Background
Project Goals
Results
Parts
Notebook
Molecular Biology Group
Yeast Group
Microscopy Group
Biobrick Group
Safety
Splicing
Splicing
One of the multitudes of ways in which eukaryotes process RNA transcripts into mRNA is through the splicing of segments of the RNA.  Splicing has a multitude of uses and is integral in organisms all the way from yeast to humans. The process involves the excision of a RNA segment called the intron and the joining of the two “exons” which flank the intron.
Alternative Splicing
One of the more interesting aspects of splicing is a process called alternative splicing.  In alternative splicing, a single RNA transcript can be spliced in alternative ways depending on the cellular environment, resulting in different mRNAs that can form different isoforms of the same protein. Thus, a single gene may code for a multitude of proteins.
There are several modes of alternative splicing. This project involves what is called Mutually Exclusive exons because the intron contains a stop codon. If the stop codon containing intron is expressed translation will cease before the second exon, preventing translation of sequences downstream of intron.
Splicing Machinery
Splicing is mediated by a large complex within the nucleus called the spliceosome.  The spliceosome is composed of many different proteins and five uradine rich snRNPs (small nuclear ribonucleic proteins). Spliceosome formation is directed by several highly conserved sequences on the pre-mRNA.  The three major recognition sequences in S. cerevisiae are highly conserved and are the 5' splice site (N/GTATGT), the 3' splice site (TAG/N), and the branchpoint sequence (TACTAAC), where '/' represents an exon-intron boundary and the 'A' represents the catalytic Adenosine at the branchpoint.
Spliceosome Mechanism
To accomplish splicing the spliceosome performs two precise, sequential transesterification reactions which cause the excision of the intron and joining of the two flanking exons.  The first reaction involves the nucleophilic attack of the active adenosine of the branch point on the phosphodiester bond of the 5' splice site.  This leads to cleavage of the 5' splice site and formation of a lariate intermediate. The 3' OH of the of the 5' exon is then free to attack the 3' splice site.  This causes the two exons to join and the intron to be released.
Splicing in S. cerevisiae
S. cerevisiae was chosen as a model organism due to its widespread use in synthetic biology and its low amount of native splicing activity. The lack of abundant alternative splicing mechanisms in S. cerevisiae has prohibited the use of splicing as a synthetic biology tool.  Additionally the small amount of naturally occurring splicing activity helps to minimize interference with our synthetic mechanism.  
When compared to other Eukaryotes splicing in S. cerevisiae is greatly diminished.  Genome wide analysis has shown only 3.8% of S. cerevisiae genes contain introns (Lopez et. al., 1999) and only one example of alternative splicing has been shown (Juneau et. al., 2009).  This differs noticably from humans where the average gene contains 7.8 introns and 35-65% of genes are alternatively spliced (Ast, 2004).
Despite the greatly reduced splicing activity in S. cerevisiae it still contains splicing machinery greatly conserved all the way to human systems (Shen et. al., 2006).  The high conservation of splicing machinery is what allows the use of a Drosophila regulatory protein, SxL, to be used in S. cerevisiae.  Furthermore it makes the designed system highly portable from one organism to the next allowing for widespread application.
Sex-Lethal
Sex Lethal (Sxl) is a gene that is important both for sexual differentiation and dosage compensation in Drosophila melanogaster. As an mRNA binding protein, one of its functions is regulation of mRNA splicing events. (Bell et. al., 1988, Kelley et. al. 1995) The Sxl product can prevent splicing by the splicosome at either the 3’ or 5’ splice sites. This is through competitive binding with U2 Auxiliary Factor (U2AF) to the poly(Y) track within the intron. Normally the U2AF splicing factor binds to the poly(y) tract and recruits the U2 snRNP, a necessary step in spliceosome formation. Having a similar active binding site to U2AF allows Sxl to bind to the same recognition site and block U2AF activity. (Merendino et. al., 1999) When this occurs, two things can happen. The first, as seen in the msl-2 gene of D. melanogaster, is that Sxl binding prevents the excision of an intron.  When translated, the intron will code for a premature stop codon and result in a truncated protein. (Merendino et. al., 1999)  Another outcome of Sxl binding can be seen in the tra-1 gene of D. melanogaster, where suppressed splicing at one site causes the splicosome to prefer a secondary downstream splice site. This causes a larger intron to be excised, and a different protein product. (Bell et. al., 1988) In both of these cases, the different gene products function in sex determination. Furthermore Sxl is an autoregulatory gene, affecting its own pre-mRNA so that only female cells will express the active protein. (Bell et. al., 1988) In the two examples given, default splicing will result in male, and regulated splicing due to Sxl will result in female phenotype/differentiation.
For our project we have chosen to work with the second type of Sxl regulation, alternative 3' splice site selection. To do this we have included the Sxl binding domain, which is (U)8 or A(U)7, in our construct. (Merendino et. al., 1999) This site is recognized by U2AF in Saccharomyces cerevisiae whose binding is required prior to U2 snRNP binding and spliceosome assembly (Zamore et. al., 1989). The expected outcome is that by introducing Sxl and the Sxl binding site to yeast we will be able to regulate native splicing mechanisms to regulate alternative splicing of our construct.
Works Cited
Lopez et. al. 1999, Genomic-scale quantitative analysis of yeast pre-mRNA splicing: implications for splice-site recognition, RNA, v. 5, 1135-1137
Juneau et. al. 2009, Alternative splicing of PTC7 in Saccharomyces cerevisiae determines protein localization, Genetics, v.183, 185-195
Ast 2004, How did alternative splicing evolve?, Nature reviews. Genetics, v. 5, 773-782
Shen, Green, 2006, RS domains contact splicing signals and promote splicing by a common mechanism in yeast through humans, Genes and Development, v. 20, 1755-1765
Bell et. al. 1988, Sex-lethal, A Drosophila Sex Determination Switch Gene, Exhibits Sex-Specific RNA Splicing and Sequence Similarity to RNA Binding Proteins, Cell, v. 55, 1037-1046.
Kelley et. al. 1995, Expression of Msl-2 Causes Assembly of Dosage Compensation Regulators on the X Chromosomes and Female Lethality in Drosophila, Cell,, v. 81, 867-877
Merendino et. al.1999, Inhibition of msl-2 splicing by Sex lethal reveals interaction between U2AF35 and the 39 splice site AG, Letters to Nature, v.202 p.838-841
Zamore et. al. 1989, Identification, purification, and biochemical characterization of U2 small nuclear ribonucleoprotein auxiliary factor,Proc. Nati. Acad. Sci. USA, v. 86, 9243-9246
Retrieved from " http://2010.igem.org/Team:WashU/Splicing/Background "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
