<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_WashU_Notebook_MolecularBiology"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:WashU/Notebook/MolecularBiology">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:WashU/Notebook/MolecularBiology&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:WashU/Notebook/MolecularBiology&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:WashU/Notebook/MolecularBiology&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:WashU/Notebook/MolecularBiology" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:WashU/Notebook/MolecularBiology</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="topmenu"><UL><LI><A href="https://2010.igem.org/Team:WashU">Home</A></LI><LI><A href="https://2010.igem.org/Team:WashU/Team" id="current">Team</A></LI><LI><A href="https://2010.igem.org/Team:WashU/Team#Undergraduates">Undergraduates</A></LI><LI><A href="https://2010.igem.org/Team:WashU/Team#Mentors">Mentors</A></LI><LI><A href="https://2010.igem.org/Team:WashU/Team#Advisors">Advisors</A></LI><LI><A href="https://2010.igem.org/Team:WashU/Team#Pictures_of_Us_and_our_Lab">Pictures</A></LI></UL><LI><A href="https://2010.igem.org/Team:WashU/Yeast" id="current">Yeast</A></LI><LI><A href="https://2010.igem.org/Team:WashU/Yeast#Background">Background</A></LI><LI><A href="https://2010.igem.org/Team:WashU/Yeast#Project_Goals">Project Goals</A></LI><LI><A href="https://2010.igem.org/Team:WashU/Yeast#Results">Results</A></LI><LI><A href="https://2010.igem.org/Team:WashU/Splicing" id="current">Splicing</A></LI><LI><A href="https://2010.igem.org/Team:WashU/Splicing#Background">Background</A></LI><LI><A href="https://2010.igem.org/Team:WashU/Splicing#Splicing:_The_New_Alternative_-_Project_Goals">Project Goals</A></LI><LI><A href="https://2010.igem.org/Team:WashU/Splicing#Results">Results</A></LI><LI><A href="https://2010.igem.org/Team:WashU/Parts">Parts</A></LI><LI><A href="https://2010.igem.org/Team:WashU/Notebook">Notebook</A></LI><LI><A href="https://2010.igem.org/Team:WashU/Notebook/MolecularBiology">Molecular Biology Group</A></LI><LI><A href="https://2010.igem.org/Team:WashU/Notebook/Yeast">Yeast Group</A></LI><LI><A href="https://2010.igem.org/Team:WashU/Notebook/Microscropy">Microscopy Group</A></LI><LI><A href="https://2010.igem.org/Team:WashU/Notebook/Biobricks">Biobrick Group</A></LI><LI><A href="https://2010.igem.org/Team:WashU/Safety">Safety</A></LI></DIV><H2><SPAN class="mw-headline" id="2010.2F07.2F01">2010/07/01</SPAN></H2><UL><LI>Plated The Megax BH10B strain and Strain 8 from the cohen lab onto an amp plate to check that the Megax BH10B strain is killed
</LI></UL><H2><SPAN class="mw-headline" id="2010.2F07.2F13">2010/07/13</SPAN></H2><UL><LI>Minipreped all the cultures and nanodropped them
</LI><LI>Ran 3 PCR reactions (@ 58,60, and 62 C) with primers p1 and p2 in order to attach the kozak onto the YFP
</LI><LI>Enzyme digested Nat (bglII and EcoRI), Kan (BamHI and EcoRI), and Promoter (EcoRI).  Did 2 reactions each with ~500 ng DNA
</LI></UL><H2><SPAN class="mw-headline" id="2010.2F07.2F14">2010/07/14</SPAN></H2><UL><LI>Gel from July 13th
</LI></UL><UL><LI>Lanes from top to bottom:
</LI></UL><PRE>1)   Ladder
2)   5uL of 58 degree YFP + 1uL of loading dye
3)   5uL of 60 degree YFP + 1uL of loading dye
4)   5uL of 62 degree YFP + 1uL of loading dye
5)   5uL of KanMx4 1 + 1uL of loading dye
6)   5uL of KanMx4 2+ 1uL of loading dye
7)   5uL of NatMx4 1 + 1uL of loading dye
8)   5uL of NatMx4 2+ 1uL of loading dye
9)   5uL of Promoter 1 + 1uL of loading dye
10) 5uL of Promoter 2 + 1uL of loading dye
</PRE><H2><SPAN class="mw-headline" id="2010.2F07.2F15">2010/07/15</SPAN></H2><UL><LI>Gel reads from left to right as follows: 
</LI></UL><PRE>Dna Ladder; 
NatMX4-Promoter construct 2x; 
KanMX4 plasmid backbone 2x.
</PRE><UL><LI>DNA ladder  information is here at <A href="http://www.neb.com/nebecomm/products/productN3232.asp" class="external free" rel="nofollow">http://www.neb.com/nebecomm/products/productN3232.asp</A></LI></UL><H2><SPAN class="mw-headline" id="2010.2F07.2F16">2010/07/16</SPAN></H2><UL><LI>Transformed EColi with the Kan plasmid containing the NatMX4 Promoter sequence.
</LI><LI>Bacteria Transformation July 16, 2010 
</LI></UL><PRE>1)	Thaw competent cells on ice right before transformation.
2)	Make 3 100uL tubes of competent cells
3)	Add 4 ul of plasmid to cells, flick gently.
4)	Incubate on ice 30 min.
5)	Heat-shock cells for 30 seconds at 42oC w/o shaking
6)	Transfer to ice, incubate 2 min
7)	add 250 ul room temp SOC Media
8)	Shake @ 37o for 1 hr
9)	Spread 150 ul on a pre-warmed ampicilin selector plates, allow sample to dry on plate
10)	Incubate o/n at 37oC upside down
</PRE><H2><SPAN class="mw-headline" id="2010.2F07.2F19">2010/07/19</SPAN></H2><UL><LI>Transformation of NatMX4/Promoter vector failed. 
</LI><LI>Analysis of gel purification showed that the separate pieces which were ligated together on Thursday July 15th were of the correct length. 
</LI><LI>Conclusion: the ligation failed; most likely due to denatured ligase. The ligase icebath had melted prematurely and heat inactivated the enzyme. 
</LI><LI>Yeast group obtained several replacement ligases and necessary buffers from the Cohen Laboratory in the 4444 building of the WashU Med School.
</LI></UL><UL><LI>Transformation of C2 and SxL constructs into E. coli (AH)
</LI><LI>Reconstituted 5ug of each construct (in plasmid) in 10uL dH20. Final conc. of this stock solution is 500ng/uL. 10uL of working stock of each solution was made by diluting stock solution 1:10. Final conc. of working stock is 50ng/uL.
</LI><LI>Thaw 300uL tube of DH5alpha competent cells and divide among 5 tubes (60uL each). Amounts of 50ng/uL working stock were added to tubes labeled A-E in the following way:
</LI></UL><PRE>A: 1uL C2 
B: 5uL C2
C: 1uL SxL
D: 5uL SxL
E: Control=no DNA
</PRE><UL><LI>Incubate on ice 30 min, heat shock for 30s at 42C, rest on ice for 2 min.
</LI><LI>Add 250uL of LB to each tube, Shake at 37C for 1hr
</LI><LI>Plate 150uL on warm LB + Amp plates o/n at 37C
</LI></UL><H2><SPAN class="mw-headline" id="2010.2F07.2F20">2010/07/20</SPAN></H2><UL><LI>Digested 4 sequences:
</LI></UL><PRE>1) NatMx4 with BglII and EcoRI: in microliters [5 buffer 3(NEB); 2.5 DNA at 200 ng/uL; 0.5 of BSA; 
   1 of each enzyme; 40 of H2O.]
2) Constitutive Promoter with EcoRI and SpeI: in uL [5 buffer 4(NEB); 2.5 DNA at 200 ng/uL; 0.5 of BSA; 
   1 of each enzyme; 40 of H2O.]
3) Construct 2 with BglII and XbaI: in uL [5 buffer 2(NEB); 1 DNA at 500 ng/uL; 0.5 of BSA; 1 of each enzyme; 
   41.5 of H2O.]
4) SxL with BamHI: in uL [5 buffer 3(NEB); 1 DNA at 500 ng/uL; 0.5 of BSA; 1 of BamHI; 41.5 of H2O.]
</PRE><UL><LI>all were incubated for 4 hours followed 20 minutes of heat inactivation.
</LI></UL><H2><SPAN class="mw-headline" id="2010.2F07.2F21">2010/07/21</SPAN></H2><UL><LI>Digest SxL construct with EcoRI: added 1uL EcoRI to already done mix from 20 July digestion.
</LI><LI>Gel purify NatMx4, Promoter, Construct 2, and KanMx4 digest results from 20 July 2010
</LI></UL><PRE>      Note: KanMx4 bands were not observed but approximate region of gel was removed for ligation KSB.
</PRE><UL><LI>Ligate all three pieces together: NatMx4, Promoter, and Construct 2
</LI><LI>2 reactions:
</LI></UL><PRE>       1) NCPA = NatMx4+Promoter+Construct 2
                       3.5uL Nat + 3.5uL Pro + 1.5uL C2
       2) NCPB = NatMx4&amp;Promoter+Construct 2     
         [NatMx4&amp;Promoter was pre-existing ligation product from 15/16 Jul]
                       7uL NP + 1.5uL C2
</PRE><UL><LI>Ligate KanMx4 with Sxl
</LI><LI>2 reactions:
</LI></UL><PRE>       1) KSA = KanMx4 Digest 2 + SxL
                       7uL KanDigest2 + 1.5uL SxL
       2) KSB = KanMx4 (purified) + SxL
                       7uL GelPureKanMx4 + 1.5uL SxL

</PRE><UL><LI>Freezer stocks of C2 and SxL
</LI><LI>1:1 ratio of LB was mixed with 100% glycerol to make a 50% glycerol/LB solution.
</LI><LI>750uL of glycerol/LB solution was added to 750uL of each C2 and SxL culture from transformation on 2010/07/19. 
</LI><LI>Two tubes of each were placed at -80C.
</LI></UL><H2><SPAN class="mw-headline" id="2010.2F07.2F22">2010/07/22</SPAN></H2><UL><LI>Transformed the ligation DNA from reactions NCPA, NCPB, KSA, and KSB into <I>E. Coli</I> using the following protocol.
</LI><LI>Bacteria Transformation 
</LI></UL><PRE>1.      Thaw competent cells on ice right before transformation.
2.      Add 4ul of plasmid to cells, flick gently.
3.      Incubate on ice 30 min.
4.      Heat-shock cells for 30 seconds at 42oC w/o shaking
5.      Transfer to ice, incubate 2 min
6.      add 125 ul room temp LB Media
7.      Shake @ 37o for 1 hr +
8.      Spread 100 ul on a pre-warmed plate w/ correct antibiotic selector, allow sample to dry on plate
9.      Incubate o/n at 37oC upside down
</PRE><H2><SPAN class="mw-headline" id="2010.2F07.2F26">2010/07/26</SPAN></H2><UL><LI>Digest and gel purify to confirm the results of the SxL construct transformants created on July 22. 
</LI><LI>Gel image is attached:
</LI></UL><UL><LI>Reactions 4, 5, 6, and 7 of the Yeast Group's gel are from July 21 ligation, July 22 transformation, and the July 24 miniprep of the July 23 cultures. [The leftmost lane is a ladder; numbering begins at the second lane with Number 1.]
</LI></UL><UL><LI>Sequencing the YFP to confirm the Kozak Sequence:
</LI><LI>iGEM1 YFP 62 degrees C primer P9
</LI><LI>iGEM2 YFP 62 degrees C primer P2
</LI><LI>iGEM3 YFP 62 degrees C primer P9
</LI><LI>iGEM4 YFP 62 degrees C primer P2
</LI></UL><H2><SPAN class="mw-headline" id="2010.2F07.2F27">2010/07/27</SPAN></H2><UL><LI>Digestion of Terminator and YFP pieces (YFP is currently being sequenced to confirm Kozak Sequence)
</LI><LI>Terminator
</LI></UL><PRE>â¢	37.5 uL of H20
â¢	5 uL of NEB buffer 3
â¢	5 uL of Terminator DNA
â¢	1 uL of Pst1 enzyme
â¢	1 uL of Xba1 enzyme
â¢	0.5 uL of BSA
</PRE><UL><LI>YFP
</LI></UL><PRE>â¢	37.5 uL of H20
â¢	5 uL of NEB buffer 2
â¢	5 uL of YFP DNA
â¢	1 uL of Pst1 enzyme
â¢	1 uL of Spe1 enzyme
â¢	0.5 uL of BSA
</PRE><UL><LI>Gel Purification and Confirmation of NPCA, NPCB, Terminator, and YFP
</LI></UL><UL><LI>Gel Lanes: 
</LI></UL><PRE>1) Ladder
2) NPCA 1
3) NPCA 2
4) NPCB 1
5) NPCB 2
6) Terminator
7) YFP
8) YFP
</PRE><H2><SPAN class="mw-headline" id="2010.2F07.2F28">2010/07/28</SPAN></H2><UL><LI>Double Digests of KSA2 (confirmed); NPCA 1 &amp; 2; and NPCB 1 &amp; 2.
</LI></UL><PRE>â¢	32.5 uL of H2O
â¢	10 uL of DNA
â¢	5 uL of NEB buffer 4
â¢	1 uL of XhoI
â¢	1 uL of AvrII
â¢	0.5 uL of BSA
</PRE><UL><LI>Began Colony PCR of NPCA and NPCB plates: 14 colonies taken from each.
</LI></UL><H2><SPAN class="mw-headline" id="2010.2F07.2F29">2010/07/29</SPAN></H2><UL><LI>Colony PCR:
</LI><LI>Primers p1 and p5 are defective Colony PCRs will not yield proper results
</LI><LI>The back up cultures still are intact and can be reused when the proper primers arrive
</LI></UL><UL><LI>Digestions:
</LI></UL><UL><LI>YFP Biobrick:
</LI></UL><PRE>â¢	32.5 uL of H2O
â¢	5 uL of DNA
â¢	5 uL NEB buffer 3
â¢	5 ul CIP
â¢	1 uL PstI
â¢	1 uL XbaI
â¢	0.5 uL BSA
</PRE><UL><LI>YFP kozak [Mix is defective due to Primer 1 error; DNA does not contain XbaI site]
</LI></UL><PRE>â¢	30 uL kYFP DNA
â¢	12.5 uL of H2O
â¢	5 uL NEB buffer 4
â¢	1 uL XbaI
â¢	1 uL SpeI
â¢	0.5 uL BSA
</PRE><UL><LI>C2 control
</LI></UL><PRE>â¢	37.5 uL H2O
â¢	5 uL DNA
â¢	5 uL NEB buffer 4
â¢	1 uL XhoI
â¢	1 uL AvrII
â¢	0.5 uL BSA
</PRE><UL><LI>Gel:
</LI></UL><PRE>â¢	Lane 1: Ladder 1kb
â¢	Lane 2: C2 control
â¢	Lane 3: NPCA1         3.7 and 2.3kb
â¢	Lane 4: NPCA2         3.7 and 2.3kb
â¢	Lane 5: NPCb1         3.7 and 2.3kb
â¢	Lane 6: NPCb2         3.7 and 2.3k
</PRE><UL><LI>Conclusion NP was not present within the construct 2 plasmid
</LI></UL><UL><LI>Gel 2:
</LI></UL><UL><LI>YFP backbone--- not observed Gel tossed out
</LI></UL><UL><LI>Ligations:
</LI></UL><UL><LI>NPCc: ligate C2// BglII &amp; XbaI with CIP A with NatMx4 and Promoter
</LI></UL><PRE>â¢	1uL Y4 ligase buffer
â¢	0.5 uL ligase
â¢	3 uL NatMx4
â¢	3 uL Promoter 
â¢	2.5 uL C2
</PRE><UL><LI>NPb: NatMX4 ligates to Promoter
</LI></UL><PRE>â¢	17 uL NatMx4
â¢	17 uL Promoter
â¢	4 uL T4 ligase buffer
â¢	2 uL ligase
</PRE><UL><LI>Extra Digestion:
</LI></UL><UL><LI>C2 // bglII &amp; XbaI CIP B
</LI></UL><PRE>â¢	27.5 uL H2O
â¢	10 uL DNA
â¢	5 uL CIP
â¢	5 uL NEB buffer 3
â¢	1 uL BglII
â¢	1 uL XbaI
â¢	0.5 uL BSA
</PRE><H2><SPAN class="mw-headline" id="2010.2F07.2F30">2010/07/30</SPAN></H2><UL><LI>Gel purify 30 uL NPb and 30 uL C2b
</LI><LI>Make NPCd: Ligate NPb &amp; C2b (non purified) ~ 10 uL of each
</LI></UL><PRE>â¢	5.5 uL NPb
â¢	3 uL C2b
â¢	0.5 ligase
â¢	1 uL buffer
</PRE><UL><LI>Make NPCe: Ligate NPb &amp; C2b (purified) 
</LI></UL><PRE>â¢	24 uL of NPb DNA
â¢	1.5 uL of C2b DNA
â¢	3 uL buffer
â¢	1.5 uL ligase
</PRE><UL><LI>Re-attempt ligation of NatMx4 and Promoter
</LI></UL><PRE>â¢	25.5 uL mix of NatMx4 and Promoter
â¢	3 uL buffer
â¢	1.5 uL ligase
</PRE><H2><SPAN class="mw-headline" id="2010.2F08.2F02">2010/08/02</SPAN></H2><UL><LI>Transformations: NPCc NPCd NPCe and KSA2
</LI></UL><PRE>1.	Thaw competent E.Coli cells on ice right before transformation.
2.	Make 4 tubes with 70uL of competent cells each
3.	Add 3 ul of NPCc, d and e plasmid to 3 tubes and 1 uL of KSA2 to the last tube, flick gently.
4.	Incubate on ice 30 min.
5.	Heat-shock cells for 30 seconds at 42oC w/o shaking
6.	Transfer to ice, incubate 2 min
7.	add 140 ul room temp LB Media
8.	Shake @ 37o for 2 hr [to ensure recovery from shock]
9.	Spread 100 ul on a pre-warmed ampicilin selector plates, allow sample to dry on plate
10.	Incubate o/n at 37oC upside down
</PRE><UL><LI>PCR reactions:
</LI></UL><UL><LI>Reaction 1st round
</LI></UL><PRE>kYFPa: 94ng of YFP at 59 degrees C using p10 &amp; p2   
kYFPb: 94ng of YFP at 61 degrees C using p10 &amp; p2
</PRE><UL><LI>Reaction 2nd Round
</LI></UL><PRE>kYFPa1: 1 uL of kYFPa at 59 degrees C using p11 &amp; p2 
kYFPb1: 1 uL of kYFPb at 59 degrees C using p11 &amp; p2
kYFPa2: 1 uL of kYFPa at 61 degrees C using p11 &amp; p2
kYFPb2: 1 uL of kYFPb at 61 degrees C using p11 &amp; p2
</PRE><H2><SPAN class="mw-headline" id="2010.2F08.2F03">2010/08/03</SPAN></H2><UL><LI>Cultures:
</LI><LI>Created liquid cultures from the transformation results of Ksa2, NPCc, NPCd, NPCe, and C2.
</LI></UL><PRE>               [C2 transformation was performed by Amanda when the C2 construct arrived]
</PRE><UL><LI>Digests: kYFPa1, a2, b1, b2 and YFPBiobrick on backbone pSB1C3
</LI><LI>For ALL:
</LI></UL><PRE>â¢	27.5 uL of H2O
â¢	15 uL of DNA of the respective construct
â¢	5 uL of NEB buffer 3
â¢	1 uL of PstI enzyme
â¢	1 uL of XbaI enzyme
â¢	0.5 uL of BSA
</PRE><UL><LI>[resulting products were column purified and eluted to 50 uL]
</LI></UL><UL><LI>Ligation of kYFP's and the backbone
</LI></UL><PRE>â¢	21 uL of kYFPa1, a2, b1, b2 in 4 respective tubes
â¢	4.5 uL of BB into all four tubes
â¢	3 uL buffer 
â¢	1.5 uL T4 ligase
</PRE><UL><LI>Of NatMx4 and Promoter
</LI></UL><UL><LI>NatMx4
</LI></UL><PRE>â¢	35 uL of H2O
â¢	7.5 uL of DNA
â¢	5 uL of NEB buffer 3
â¢	1 uL of bglII
â¢	1 uL of EcoRI
â¢	0.5 uL BSA
</PRE><UL><LI>Promoter
</LI></UL><PRE>â¢	35 uL of H2O
â¢	7.5 uL of DNA
â¢	5 uL of NEB buffer 4 
â¢	1 uL of EcoRI
â¢	1 ul of SpeI
â¢	0.5 uL BSA
</PRE><H2><SPAN class="mw-headline" id="2010.2F08.2F04">2010/08/04</SPAN></H2><UL><LI>Gel purification of NatMX4 and Promoter
</LI><LI>4 lanes Nat and 4 lanes Pro
</LI><LI>Successfully extracted the lower of 2 visible lines from all lanes [top line was vector; bottom was Nat and Pro inserts]
</LI><LI>Miniprep: Kan-SxL construct and Construct 2
</LI><LI>Transformation: kYFPa1, a2, b1, b2 on the Biobrick Backbone.
</LI></UL><PRE>1.	Thaw competent cells on ice right before transformation.
2.	Make 4 75uL tubes of competent cells
3.	Add 4 ul of plasmid to cells, flick gently.
4.	Incubate on ice 30 min.
5.	Heat-shock cells for 30 seconds at 42oC w/o shaking
6.	Transfer to ice, incubate 2 min
7.	add 150 ul room temp LB Media
8.	Shake @ 37o for 1 hr
9.	Spread 100 ul on a pre-warmed ampicilin selector plates, allow sample to dry on plate
10.	Incubate o/n at 37oC upside down
</PRE><UL><LI>Ligation: NatMx4 + Promoter
</LI></UL><PRE>â¢	30 uL of DNA (mix of both parts; they were extracted together)
â¢	4 uL of Buffer
â¢	4 uL of H2O
â¢	2 uL of T4 ligase
</PRE><H2><SPAN class="mw-headline" id="2010.2F08.2F06">2010/08/06</SPAN></H2><UL><LI>Transformation: kYFP on the Biobrick Backbone.
</LI></UL><PRE>1.	Thaw competent cells on ice right before transformation.
2.	Make 1 100 uL tubes of competent cells
3.	Add 4 ul of plasmid to cells, flick gently.
4.	Incubate on ice 30 min. (10:45 to 11:15)
5.	Heat-shock cells for 30 seconds at 42oC w/o shaking
6.	Transfer to ice, incubate 2 min
7.	add  ul room temp SOC Media
8.	Shake @ 37o for 1 hr 
9.	Spread 100 ul on a pre-warmed ampicilin selector plates, allow sample to dry on plate
10.	Incubate o/n at 37oC upside down
</PRE><H2><SPAN class="mw-headline" id="2010.2F08.2F09">2010/08/09</SPAN></H2><UL><LI>Make Tetracycline plates to select for kYFP transformants
</LI><LI>Transform kYFP into E.Coli
</LI></UL><PRE>1.	Thaw competent cells on ice right before transformation.
2.	Make a 75 uL tube of competent cells
3.	Add 2 ul of plasmid to cells, flick gently.
4.	Incubate on ice 30 min. (12:10 to 12:40)
5.	Heat-shock cells for 30 seconds at 42oC w/o shaking
6.	Transfer to ice, incubate 2 min
7.	add 125 ul room temp SOC Media
8.	Shake @ 37o for 1 hr
9.	Spread 100 ul on a pre-warmed tetracycline selector plates, allow sample to dry on plate
10.	Incubate o/n at 37oC upside down
</PRE><H2><SPAN class="mw-headline" id="2010.2F08.2F10">2010/08/10</SPAN></H2><UL><LI>Made a culture from possible colonies on tetracycline plate
</LI><LI>Double Digest of pSB1A3
</LI></UL><PRE>â¢	H2O
â¢	DNA
â¢	NEB buffer 3
â¢	XbaI
â¢	PstI
â¢	BSA
</PRE><UL><LI>Gel purification of digest products
</LI><LI>1kb ladder, 3 lanes of pSB1A3, and 3 lanes of YFP-BB
</LI><LI>results: all 3 bands observed (one for the pSB1A3 digest and 2 for the YFPbb) [an extra band from the biobrick was seen above the 2 desired YFP bands possibly single or uncut plasmid]
</LI></UL><LI>Gel extraction of digest products
</LI><LI>1 extraction for each of the 3 desired bands
</LI><UL><LI>Ligations
</LI></UL><PRE>â¢	kYFP to pSB1A3
â¢	kYFP to Biobrick backbone
â¢	YFP to pSB1A3
â¢	YFP to Biobrick backbone
</PRE><UL><LI>Transformation of the ligation products into <I>E. Coli</I></LI></UL><PRE>1.	Thaw competent cells on ice right before transformation.
2.	Make 4, 70 uL tube of competent cells
3.	Add 2 ï­l of plasmid to cells, flick gently.
4.	Incubate on ice 30 min. (3:45 to 4:15)
5.	Heat-shock cells for 30 seconds at 42oC w/o shaking
6.	Transfer to ice, incubate 2 min
7.	add 125 ï­l room temp SOC Media
8.	Shake @ 37o for 1 hr
9.	Spread 100 ï­l on a pre-warmed ampicilin selector plates, allow sample to dry on plate
10.	Incubate o/n at 37oC upside down
</PRE><H2><SPAN class="mw-headline" id="2010.2F08.2F11">2010/08/11</SPAN></H2><UL><LI>Colony PCR
</LI></UL><PRE>â¢	4 possible transformant colonies PCR'd and back-up culture made
</PRE><UL><LI>PCR
</LI></UL><PRE>â¢	Re-PCRing the kYFP at 59, 60, 62 degrees C [product was named kYFP2]
</PRE><UL><LI>Transformation of kYFP with o/n ligation results
</LI></UL><PRE>â¢	50uL of cells
â¢	2 uL of kYFP on Biobrick backbone
â¢	2 uL of kYFP on pSB1A3
â¢	150uL of SOC media
â¢	plated all of the incubated/shaken culture
</PRE><UL><LI>Gel purifications
</LI><LI>100b ladder; 4 lanes of colony PCR; 59*C PCR of new kYFP; 60*C PCR of new kYFP; 62*C PCR of new kYFP.
</LI><LI>results:
</LI></UL><UL><LI>Digests of PCR Products and the Terminator
</LI><LI>Digesting PCR kYFP with XbaI and SpeI
</LI><LI>Digesting terminator [in vector] with XbaI and CIP
</LI><LI>Ligation
</LI></UL><H2><SPAN class="mw-headline" id="2010.2F08.2F12">2010/08/12</SPAN></H2><UL><LI>Miniprepped the cultures made on the 11th [#4 had no growth or pellet]
</LI><LI>Created cultures of the 4 transformant colonies from the 11th's transformation
</LI><LI>Redesigned the kYFP reverse primer
</LI><LI>Ligate Terminator and kYFP2 (from the 11th's digestion) 
</LI><LI>[30min room temp ligation]
</LI><LI>PCR at 60*C the Miniprep results, a YFP control and the Ligation product kYFP2term with primers p2 and p12&amp; p9 and p12 reactions with the YFP control and the kYPF2term. [7 rxns total as #4 yielded no pellet for miniprepping]
</LI><LI>[p2 was the questionable reverse kYFP primer]
</LI></UL><UL><LI>Gel run: 100bp ladder, culture 1 DNA; cult. 2 DNA; cult. 3 DNA; YFP 2-12 PCR; YFP 9-12 PCR; kYFP2 2-12 PCR; kYFP2 9-12 PCR
</LI><LI>Results
</LI></UL><CENTER><P>]
</P></CENTER><UL><LI>Conclusion: Colonies were not desired kYFP transformants [match the pattern of the YFP (2-12) control]
</LI></UL><UL><LI>Transformation of E.Coli with the kYFP2term plasmid (ligation product)
</LI></UL><PRE>   3uL in 50uL of cells
   150uL SOC media
   75 ul plated onto amp selector plates
</PRE><H2><SPAN class="mw-headline" id="2010.2F08.2F13">2010/08/13</SPAN></H2><UL><LI>Transformation yielded a large number of colonies which may contain the kYFP2term construct
</LI><LI>Pour new ampicilin plates
</LI><LI>Beginning screening processes today:
</LI><LI>Grow 16 cultures of several colonies from transformants
</LI><LI>Future screening steps:
</LI><LI>Miniprep the cultures
</LI><LI>Digest DNA with XbaI &amp; PstI
</LI><LI>Digest C2 with XbaI &amp; PstI
</LI><LI>Gel12 purify C2 and the correct kYFP2term from the improper kYFP2term [X will cut improper into 3 pieces, correct into 2]
</LI><LI>30min room temp ligate correct kYFP2term with C2
</LI><LI>Transform the ligated construct, C2kYFP2term, into <I>E. Coli</I></LI><LI>Grow cultures from transformation 
</LI><LI>Miniprep the cultures
</LI><LI>Submit DNA for sequencing
</LI><LI>Digest with bglII &amp; XbaI &amp; CIP
</LI><LI>Gel purify
</LI><LI>30 min room temp ligation of C2kYFP2term with the NatMX4-Promoter construct
</LI><LI>Transform ligation product into E.Coli
</LI></UL><H2><SPAN class="mw-headline" id="2010.2F08.2F14">2010/08/14</SPAN></H2><UL><LI>Miniprepped the cultures (16) made from kYFP2term transformants
</LI><LI>Digests:
</LI><LI>Digested C2 with SpeI and PstI and CIP in NEBuffer 2. [27.5 H2O; 14 DNA 1 CIP; 1 SpeI; 1 PstI; 5 buffer; 0.5 BSA] 
</LI><LI>6 Digests of Miniprepped DNA with XbaI and PstI in NEBuffer 2 [35.5 H2O; 7 DNA; 5 buffer; 1 XbaI; 1 PstI, 0.5 BSA]
</LI></UL><PRE>               Note: rxns 12,13,14,15, and 16 used low concentration XbaI
</PRE><UL><LI>Gels:
</LI><LI>12 lane gel contained: 1kb ladder; culture samples 1-11
</LI><LI>8 lane gel contained: 1 kb ladder; culture samples 12-16
</LI></UL><LI>Expected results
</LI><PRE>1. Successful construction: 2 kb band and .993kb band
2. Backwards kYFP2: 3 kb band
3. No kYFP2: 2kb band and .25 kb band
</PRE><UL><LI>Results
</LI></UL><UL><LI>Conclusion: #3 results for all samples; no kYFP2 ligated into the terminator
</LI></UL><H2><SPAN class="mw-headline" id="2010.2F08.2F19">2010/08/19</SPAN></H2><UL><LI>Got the new primers in today.  
</LI><LI>Ran a PCR to make the new KYFP inserts. 
</LI><LI>Used 1 ul of YFP biobrick plasmid as the template.  
</LI><LI>p15 and p 16 were used to make the KYFP3.  
</LI><LI>p10 and p17 were used to make the preKYFP4.  
</LI><LI>Three PCR reactions were set up for each and were then run at 58, 60, and 64 C with 1 min extension time. 
</LI><LI>The PCR products were then run out on a gel.  
</LI><LI>The bands coresponding to ~ 800 bp were cut out and gel purified.
</LI></UL><UL><LI>Lanes:
</LI></UL><PRE>1        100 bp ladder
2,3      KYFP3 @58
4,5      KYFP3 @60
6,7      KYFP3 @64
</PRE><UL><LI>Lanes:
</LI></UL><PRE>1        100 bp ladder
2,3      preKYFP4 @58
4,5      preKYFP4 @60
6,7      preKYFP4 @64
8        100 bp ladder
</PRE><UL><LI>The preKYFP4 products were cleaner as was to be expected.  
</LI><LI>The KYFP3 @60 has anomalous lower bands ~200 bp and the KYFP3's in general have more minor banding patterns
</LI></UL><UL><LI>Running a PCR over night. 
</LI></UL><PRE>â¢	1 ul of KFYP3 64+p13+p16=KYFP5
o	running 2 reactions: one at 60 and the other at 62 C
â¢	1 ul of preKYFP4 64+p11+p16=KYFP4
o	running 2 reactions: one at 60 and the other at 62 C
</PRE><H2><SPAN class="mw-headline" id="2010.2F08.2F20">2010/08/20</SPAN></H2><UL><LI>Ran the PCR products out on a gel.
</LI></UL><UL><LI>Lanes:
</LI></UL><PRE>1        100 bp ladder
2        KYFP5 @60
3,4     KYFP4 @60
5,6     KYFP5 @62
7,8     KYFP4 @62
</PRE><UL><LI>The bands that are ~800 bp were cut out and purified.  
</LI><LI>Then I set up a five hour enzyme digest @ 37 C.  
</LI><LI>I cut KYFP3 @64 w/ XbaI and PstI, KYFP4 @62 w/ XbaI and PstI, KYFP5 @62 w/ XbaI and PstI, YFP backbone w/ XbaI, PstI, and CIP, and Terminator with XbaI and CIP.  
</LI><LI>The digested results were run out on a gel and then the appropriate bands were excised and the DNA was purified out.  
</LI><LI>An over night RT ligation was set up with 1 ul of backbone: 7.5 ul insert.  
</LI><LI>Cut KYFP3, KYFP4, and KYFP5 were ligated into the cut YFP back bone.  
</LI><LI>A negative control (only cut YFP back) and a positive control (YFP insert back into YFP back bone) were also set up.
</LI></UL><H2><SPAN class="mw-headline" id="2010.2F08.2F26">2010/08/26</SPAN></H2><UL><LI>Results from transformation on 8/25                    
</LI></UL><PRE>Number	Plasmid 	Plate 	Observations
1	pSB1AT3 	Tet 	 Lawn of colonies with sprinkled red transformants
2	None 		Tet 	 Lawn of colonies
3	pSB1AT3 	Amp 	 ~500 colonies, all small, mostly red transformations but some
4	None 		Amp 	 ~100 colonies, some very large in size
5	pSB1K3 		Kan 	 ~80 colonies observed all red transformants
6	None 		Kan 	 No colonies observed
</PRE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:WashU/Notebook/MolecularBiology">http://2010.igem.org/Team:WashU/Notebook/MolecularBiology</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:WashU/Notebook/MolecularBiology" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:WashU/Notebook/MolecularBiology" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:WashU/Notebook/MolecularBiology&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:WashU/Notebook/MolecularBiology&amp;oldid=175558" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>