"
Page
Discussion
View source
History
teams
Log in
 
Team:WashU/Yeast
From 2010.igem.org
Home
Team
Undergraduates
Mentors
Advisors
Pictures
Yeast
Background
Project Goals
Results
Splicing
Background
Project Goals
Results
Parts
Notebook
Molecular Biology Group
Yeast Group
Microscopy Group
Biobrick Group
Safety
Yeast: A Simpler Transformation
S. cerevisiae has long served as the model organism of the ekaryotic domain, however the biobrick registry is lacking in terms of the diversity of parts necessary to provide comprehensive support when working with this organism.  In order to rectify this problem, the 2010 iGEM team is desiging a set of biobricks that will help simplify the transformation process in S. cerevisiae.  Drug resistant genes will be submitted as biobricks allowing for positive selection to be preformed on wildtype yeast strains instead of the more basic supplementation of auxotrophic strains.  Furthermore, new plasmid backbrones are to be designed which will allow DNA manipulation to be conducted in E. coli and then direct chromosomal intergration into S. cerevisiae.
Background
Yeast Vectors
Currently there exist two different methods for insertion of synthetic DNA into yeast cells, plasmid vectors or chromosomal integration. Plasmid vectors contain the components necessary to replicate autonomously after entering the yeast cell.  Chromosomal integration inserts DNA directly into the yeast chromosome from a linear construct.
Yeast Plasmids
There exist two main types of plasmids used to insert and maintain DNA in yeast cells, Yeast Episomal plasmids (YEp) and Yeast Centromer containing plasmids (YCp).  Natively yeast contain a plasmid referred to as the 2 μm plasmid, YEp vectors use an origin of replication derived from this plasmid and maintain copy numbers of 50-100. YCp vectors contain an Autonomous Replicating Sequence (ARS) and a yeast Centromer sequence (CEN) and maintain a copy number of 1-3 per cell (Keshav).
Commonly these plasmids will also contain an E. coli origin of replication and selectable marker, this allows for DNA manipulation in E. coli, and then the transfer of the plasmid directly into yeast, earning these plasmids the name "Shuttle Vectors".
However, the use of these plasmids is not without drawbacks.  Both types of replication origins have significant loss of plasmid on the order of 10-2.  When using the 2 μm replication origin and grown in selective media 60% to 95% of yeast cells may not contain the plasmid (Sherman).  Additionally the variability in copy number introduces noise when quantifying the amount of protein expressed in cells.
Chromosomal Integration
An alternative to the use of plasmids in yeast is direct integration of DNA constructs into the chromosome of yeast cells.  Yeast cells natively have high levels of homologous recombination activity; consequently when a DNA sequence is introduced into a yeast cell containing sequences homologous to those present on the yeast chromosome, it can be swapped into the yeast chromosome.  This method is commonly used to create knock out strains excising genes such as Ura3, His3, Trp1, His3 and Lys2.  It can also be used to insert synthetic DNA constructs into the genome.  Chromosomally integrated DNA is lost at a much lower rate, approximately 10-3 to 10-4, than plasmid discharge (Sherman).  However transformation efficiency is lower than with plasmids, on an order of 103 transformants/μg opposed to efficiency of 105 transformants/μg when plasmids are used. Yeast Integrating plasmids (YIp) contain homologous sequences to the yeast chromosome and cause integration into the yeast chromosome.  To increase transformation efficiency by 102 these plasmids are linearized before transformation (Keshav).
Project Goals
Yeast Vectors
Two new plasmid backbones are to be created and submitted to the registry.  Similar in structure the two vectors will be derived from BioBrick pSB1AT3 which contains a high copy E. coli replication origin and both ampicillin and tetracycline  resistance.  PCR will be used to modify the site upstream of the prefix and downstream of the suffix in vector.  The plasmid will take the following design:
Plasmid-BbsI-5' Homologous Region-BioBrick Prefix-BioBrick Suffix-3' Homologous Region-Reverse Bbs1 Cut Site-Plasmid
This allows the insertion of yeast BioBrick parts (including positive selection markers) into the vector using E. coli.  Once the construct is fully assembled the vector is linearized with BbsI and transformed into yeast.  BbsI was chosen since it cuts downstream of its recognition site, and will create non-overlapping ends preventing circularization of the linear products.  The two new vectors will contain sequences homologous to the Ura3 and His3 loci in the yeast genome.  When employed either the Ura3 or His3 loci will be replaced with the construct, creating an auxotrophic strain of yeast.  When coupled with a positive selection marker this will allow for both positive and negative selection of transfomants.
Results
Primers were synthesized for use in production of the designed plasmids
Forward Ura3-1 GCACAGAACAATAACCTGCTGGAAACGAAGATAAATCgaagacGATTACTTCGCGTTATGCAGGC
Reverse Ura3-1 GCATCTTCTCAAATATGCTTCCCAGCCTGCTTATCcttctgAAATTCTGCCTCGTGATACGCC
Forward Ura3-2 tactagtagcggccgctgcagTCTTAACCCAACTGCACAGAACAATAACCTGCTGGAAACG
Reverse Ura3-2 ctctagaagcggccgcgaattcTTAGTATTGCTGGCCGCATCTTCTCAAATATGCTTCCCAGCC
Forward His3-1 GAACAGGCCACACAATCGCAAGTGATTAACgaagacGATTACTTCGCGTTATGCAGGC
Reverse His3-1 CCTTGAACGCACTCTCACTACGGTGATGATCActtctgAAATTCTGCCTCGTGATACGCC
Forward His3-2 tactagtagcggccgctgcagAGAGGGAGAAGCAGTAGCAGAACAGGCCACACAATCGCAAG
Reverse His3-2 ctctagaagcggccgcgaattcTTATGGCAACCGCAAGAGCCTTGAACGCACTCTCACTACGG
Two rounds of successive PCR was attempted on pSB1AT3. This was initially unsuccessful and time constraints did no allow successful trouble shooting. Sequences of the designed constructs were entered into the parts registry as BBa_K394001 and BBa_K394002 .
A positive selection protein coding region was synthesized using PCR from the KanMX4 cassette (Wach et al.).  It was entered in the resgistry as BBa_K394000 in the parts registry. PCR was attempted on the pSB1AT3
Detailed information on our experimentation can be found in the teams laboratory notebook .
Works Cited
A. Wach, A. Brachat ,R. Poehlmann & P. Philippsen New heterologous modules for classical or PCR-based gene disruptions in Saccharomyces cerevisiae, Yeast, 1994 v.10, 1793-1808.
F. ShermanAn. Introduction to the Genetics and Molecular Biology of the Yeast Saccharomyces cerevisiae, 1998. http://dbb.urmc.rochester.edu/labs/sherman_f/yeast/Index.html
S. Keshav, J. Heinemann. Yeast Plasmids, 1997 v.62, 113-130
Yeast Protocols Handbook. Clontech, 1990
Retrieved from " http://2010.igem.org/Team:WashU/Yeast "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
