<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Rutgers_Full_Adder"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Rutgers/Full_Adder">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Rutgers/Full_Adder&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Rutgers/Full_Adder&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Rutgers/Full_Adder&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Rutgers/Full_Adder" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Rutgers/Full Adder</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV class="WordSection1"><P class="MsoNormalCxSpFirst" style="line-height:normal"><B style="mso-bidi-font-weight:
normal"><U>Abstract</U></B></P><P class="MsoNormalCxSpMiddle" style="line-height:normal"><SPAN style="mso-tab-count:
1">                </SPAN>The Full Adder project seeks to create bacteria that
can mimic a digital full adder. Since many teams have difficulty creating even
something small like a XOR gate, this project would seem nearly impossible.
However, we have found that the problem can be greatly simplified if we use a
certain simple “encoding” on the outputs of the full adder. Subsequently
designing a circuit to “decode” our circuit’s output will hopefully prove
significantly easier than creating a complete full adder outright. Our insights
may prove useful to any genetic engineer or synthetic biologist working on
highly complex systems.</P><P class="MsoNormalCxSpMiddle" style="line-height:normal"><O:P> </O:P></P><P class="MsoNormalCxSpMiddle" style="line-height:normal"><B style="mso-bidi-font-weight:
normal"><U>Overview </U></B></P><P class="MsoNormalCxSpMiddle" style="line-height:normal"><I style="mso-bidi-font-style:
normal">Engineering Problems</I></P><P class="MsoNormalCxSpMiddle" style="line-height:normal"><SPAN style="mso-tab-count:
1">                </SPAN>A bacterial full adder faces two immediate problems.
First, an electrical full adder requires at least 5 logic gates. However, a
single AND gate in bacteria may require as many as 11 biobricks (as in Peking
2009). Chaining together tens of biobricks is a lot of work and may result in
plasmids that are, at best, inconvenient to transform. </P><P class="MsoNormalCxSpMiddle" style="text-indent:.5in;line-height:normal">The
second problem is that, for the most part, the current “logic gate” biobricks
are not easily connected to each other.<SPAN style="mso-spacerun:yes"> 
</SPAN>We cannot easily connect the ETH 2006 XOR gate to the Davidson-Missouri
Western (DMW) 2008 XOR gate since the inputs and outputs are not the same; ETH
has PoPS ins and out, but DMW uses various chemicals. Simply using PoPS only
parts may be an attractive solution, but that may exacerbate the first problem
as parts with PoPS ins and outs tend to be more complex. Furthermore, there is
a significant possibility of some PoPS parts interfering with the operation of
other PoPS parts as these are huge, complex circuits, many of which share some
parts.</P><P class="MsoNormalCxSpMiddle" style="line-height:normal"><O:P> </O:P></P><P class="MsoNormalCxSpMiddle" style="line-height:normal"><I style="mso-bidi-font-style:
normal">Insights</I></P><P class="MsoNormalCxSpMiddle" style="line-height:normal"><SPAN style="mso-tab-count:
1">                </SPAN>We had two major insights into the Full Adder
problems. First, we found a way to easily create biological AND gates. Second,
we found that, if we change the output of the full adder from 2 outputs to
three outputs (our so-called encoding), we can create the circuit with 4 AND
gates.</P><P class="MsoNormalCxSpMiddle" style="line-height:normal"><SPAN style="mso-tab-count:
1">                </SPAN>Of course, this is no longer a faithful rendition of
a full adder. However, the methods we used to create the AND gates should be
helpful in the creation of a whole new generation of fusion proteins, and it
should be easier to create another circuit that converts our three output adder
to a standard two output adder than creating a single monolithic full adder
circuit.</P><P class="MsoNormalCxSpMiddle" style="line-height:normal"><O:P> </O:P></P><P class="MsoNormalCxSpMiddle" style="line-height:normal"><B style="mso-bidi-font-weight:
normal"><U>Peking 2009 AND Gate</U></B></P><P class="MsoNormalCxSpMiddle" style="line-height:normal"><SPAN style="mso-tab-count:
1">                </SPAN>One AND gate that we decided to use was created
previously by the Peking 2009 iGEM team. It relies on T7ptag, T7 polymerase
with two early amber mutations (stop codons) such that normal translation will
result in a truncated non-functional protein, and SupD, a tRNA which allows for
translation through amber stop codons (an amber suppressor). So, if just T7ptag
or just SupD is expressed, there will not be any functional T7 polymerase; and,
if both T7ptag and SupD are expressed, then functional T7 polymerase will be
created. To complete the gate, we place T7ptag on one inducible promoter (our
first input), SupD on a different inducible promoter (our second input), and
our output on a T7 promoter. Note that this system relies on the fact that
basal transcription from the T7 promoter is extremely low, but, once functional
T7 polymerase is present, transcription is very high. </P><P class="MsoNormalCxSpMiddle" style="line-height:normal"><SPAN style="mso-spacerun:yes"> </SPAN></P><P class="MsoNormalCxSpMiddle" style="line-height:normal"><O:P> </O:P></P><P class="MsoNormalCxSpMiddle" style="line-height:normal"><B style="mso-bidi-font-weight:
normal"><I style="mso-bidi-font-style:normal"><U>Split Fluorescent Proteins</U></I></B></P><P class="MsoNormalCxSpMiddle" style="line-height:normal"><I style="mso-bidi-font-style:
normal">Overview</I></P><P class="MsoNormalCxSpMiddle" style="line-height:normal"><SPAN style="mso-tab-count:
1">                </SPAN>In our search for AND gates, we sought AND gates that
functioned similarly to Peking’s AND gate (and for that matter, most other AND
gates); two things that are individually non-functional, but when both are
expressed together function is restored. Ghosh et al. created exactly that with
their split GFP. They cut GFP, which usually displays green fluorescence, into
two parts, each of which did not fluoresce. Then they fused leucine zipper
domains to each part which allowed them to reconstitute functional GFP when
both were expressed together. They dubbed the two parts nzGFP and czGFP. </P><P class="MsoNormalCxSpMiddle" style="line-height:normal"><SPAN style="mso-tab-count:
1">                </SPAN>Split fluorescent proteins are particularly
attractive in C. <I style="mso-bidi-font-style:normal">elegans</I> as they
allow for combinatorial marking of cells. As C. <I style="mso-bidi-font-style:
normal">elegans </I>are transparent, a common method to study particular cells
is to put fluorescent proteins under the transcription of promoters specific to
those cells. Thus, only the interesting cells light up under a fluorescent
microscope. However, some groups of cells are difficult to target with just a
single promoter. With split fluorescent proteins, a scientist can light up
cells which have high transcription at two different promoters, allowing for
much a finer selection of cells to study. Thus, further research on split
fluorescent proteins was done in C. <I style="mso-bidi-font-style:normal">elegans
</I>(Ghosh et al.’s study was in E. <I style="mso-bidi-font-style:normal">coli</I>).
Zhang et al. created similar split CFP and YFP (cyan and yellow fluorescent
proteins). </P><P class="MsoNormalCxSpMiddle" style="line-height:normal"><SPAN style="mso-tab-count:
1">                </SPAN>All of these split fluorescent proteins function
essentially as AND gates. The inputs are whatever promoter we place the split
fluorescent proteins under and the output is fluorescence. The primary
advantage of this system is its simplicity. It just requires two proteins and
two coding sequences. Furthermore, the coding sequences do not code for enzymes
which can have all sorts of messy interactions with other parts. The primary
disadvantage of this system is that the output is limited to fluorescence.</P><P class="MsoNormalCxSpMiddle" style="line-height:normal"><O:P> </O:P></P><P class="MsoNormalCxSpMiddle" style="line-height:normal"><I style="mso-bidi-font-style:
normal">Construction</I></P><P class="MsoNormalCxSpMiddle" style="line-height:normal"><SPAN style="mso-tab-count:
1">                </SPAN>As mentioned previously, a significant amount of the
research on these split fluorescent proteins has been done in C. <I style="mso-bidi-font-style:normal">elegans</I>. Unfortunately, C. <I style="mso-bidi-font-style:normal">elegans</I> require introns for the
expression of certain genes, and so much of the easily accessible DNA with
split fluorescent proteins contains artificial introns, which would mess up
expression in E. <I style="mso-bidi-font-style:normal">coli</I>. We acquired
such DNA for split GFP, CFP, and YFP. However, we decided that it was not such
a bad thing. We aimed to clone the zipper portions for use in future split
protein construction, and, as a demonstration of their effectiveness, we would
recreate the split GFP, CFP, and YFP using fresh GFP, CFP, and YFP from the
parts registry.</P><P class="MsoNormalCxSpMiddle" style="line-height:normal"><SPAN style="mso-tab-count:
1">                </SPAN>We decided to use RFC25 to generate our fusion parts.
We did not have any particular rationale. All of the fusion standards seem
reasonable.</P><P class="MsoNormalCxSpMiddle" style="line-height:normal"><SPAN style="mso-tab-count:
1">                </SPAN>First, we cloned the zipper domains from nzGFP and
czGFP with the RFC25 prefix and suffix. Then we mutagenized GFP (BBa_E0040) so
that it contained the necessary restriction sites in the correct positions in
order to create nzGFP and czGFP. Finally, we cut and ligated the parts.</P><P class="MsoNormalCxSpMiddle" style="line-height:normal"><O:P> </O:P></P><P class="MsoNormalCxSpMiddle" style="line-height:normal"><B style="mso-bidi-font-weight:
normal"><U>Circuit</U></B></P><P class="MsoNormalCxSpLast" style="line-height:normal"><SPAN style="mso-tab-count:
1">                </SPAN>Nice image and description of how it works here.</P></DIV><TABLE style="color:#1b2c8a;background-color:#0c6;" cellpadding="3" cellspacing="1" border="1" width="62%" align="center"><TBODY><TR><TH align="center"><A href="/Team:Rutgers" title="Team:Rutgers">Home</A></TH><TH align="center"><A href="/Team:Rutgers/Team" title="Team:Rutgers/Team">Team</A></TH><TH align="center"><A href="https://igem.org/Team.cgi?year=2011&amp;team_name=Rutgers" class="external text" rel="nofollow">Official Team Profile</A></TH><TH align="center"><A href="/Team:Rutgers/Etch_a_Sketch" title="Team:Rutgers/Etch a Sketch">Etch-a-Sketch</A></TH><TH align="center"><STRONG class="selflink">Full Adder</STRONG></TH><TH align="center"><A href="/Team:Rutgers/MYSIS" title="Team:Rutgers/MYSIS">MYS!S</A></TH><TH align="center"><A href="/Team:Rutgers/Parts" title="Team:Rutgers/Parts">Parts Submitted to the Registry</A></TH><TH align="center"><A href="/Team:Rutgers/Modeling" title="Team:Rutgers/Modeling">Modeling</A></TH><TH align="center"><A href="/Team:Rutgers/Notebook" title="Team:Rutgers/Notebook">Notebook</A></TH><TH align="center"><A href="/Team:Rutgers/Safety" title="Team:Rutgers/Safety">Safety</A></TH><TH align="center"><A href="/Team:Rutgers/Acknowledgements" title="Team:Rutgers/Acknowledgements">Acknowledgements</A></TH></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Rutgers/Full_Adder">http://2011.igem.org/Team:Rutgers/Full_Adder</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Rutgers/Full_Adder" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Rutgers/Full_Adder" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Rutgers/Full_Adder&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Rutgers/Full_Adder&amp;oldid=182269" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>