"
Page
Discussion
View source
History
teams
Log in
 
Team:Rutgers/MYS!S
From 2011.igem.org
Rutgers 2011 iGEM Team: Complex Circuits in Synthetic Biology
Home
"Welcome!"
Hi and welcome here! This is the Rutgers iGEM Team of 2011's Official Wiki.
Enjoy!
Projects
RUiGEM 2011: Complex is Simple
Complex Circuits
Bacterial ETCH-A-SKETCH
Bacterial FULL ADDER
MYS!S BioBrick Optimizer
Modeling
The Etch-a-Sketch project aims to create a lawn of bacteria that can be drawn on with a laser pointer. Read more...
The Full Adder project seeks to create bacteria that can mimic a digital full adder. Read more...
Rutgers iGEM software team strives to provide a tool to improve the standard parts registry by checking, and if need be modifying, the BioBrick parts. Read more...
Notebook
RUiGEM 2011: Complex is Simple
Complex Circuits
Our Lab Notebook
Our References
Openwetware/E. coli Protocols
Team
RUiGEM 2011: The team and our appreciation to those that helped us.
Complex Circuits
Our Team
Our Acknowledgements
Safety
How can synthetic biology be better understood?
Complex Circuits
BioSafety
Human Practices
iGEM
The International Genetically Engineered Machine (iGEM)
2011 iGEM
Wikipedia/iGEM
2010 Wikis
2009 Wikis
Rutgers iGEM
Rutgers
Rutgers 2011 iGEM Team: Complex Circuits in Synthetic Biology
RUTGERS iGEM TEAM WIKI
Menu >> The Bacterial Etch-a-Sketch >> Goals
the MYS!S Project
Abstract
MYS!S is a stand alone software tool designed to assist an “iGEMer” when designing a new BioBrick part or modifying an existing one.
MYS!S handles several problems that a synthetic biologist might encounter in the lab that can be solved with site directed mutagenesis.  The utilities that MYS!S supports to modify DNA sequences on paper, can be “actualized” in the lab using site directed mutagenesis to change physical DNA “stored” in a backbone.
MYS!S will determine the most efficient modified sequence by optimizing for the least number of rounds of site directed mutagenesis. Once the changes are finalized primers are designed.  In addition to primer design, MYS!S provides lab protocols for both site directed mutagenesis and transformation into competent cells.  Currently the lab protocols are supplied as a BioCoder compliant C++ file, that can then be compiled with the BioCoder library.
 
Operations
I. DNA Optimization
First, it performs DNA codon optimization based on the target organism by removing rare codons and replacing them with more prevalent codons.
(We noticed that many teams decided to utilize sequences from a variety of organisms such as firefly luciferase to ultimately be transformed into an experimentally “stable” organism.)
 
II. Processing
Secondly, the program checks the coding sequence for any restriction sites that are not allowed by the format specified by the user.  After checking the sequence, MYS!S removes the restriction sites while ensuring that the modified sequence is still codon optimized.
In addition, it was pointed out that frequently a synthetic biologist would like to change specific amino acids in the DNA coding sequence.  MYS!S decides what difference in nucleotide sequence are required to best make the amino acid change.
 
III. Restriction site analysis
Alternatively, an iGEMer might also want to add restriction sites to a DNA sequence in order to remove a particularly useful sequence.  The program can handle this by either allowing the user to specify where the restriction site should be or by choosing what restriction site is desired and how sites need to be added to the sequence.
 
 
How Mys!S Does it
The MYS!S Environment
One of the main ideas behind MYS!S was to make synthetic biology more accessible to a wider “spectrum” of people.  We want the program to provide an environment that enables the user to understand and explore the changes being made to the DNA in a more intuitive manner.
Custimization
In addition, MYS!S is designed to be highly customizable by providing the user with the ability to add and modify components such as an organism’s codon usage table and new assembly standards.
The ultimate goal is to create a program that can be customized by the user for their specific work and the lab protocols they are comfortable with.
 
MYS!S: An Introduction
Walkthrough
The MYS!S walkthrough is located here!
https://2011.igem.org/Team:Rutgers/MYS!S_WT
 
Features
I. Protocols
In the protocol tab is a BioCoder compatible C++ file that contains the lab procedures for mutating the original Lovtap DNA into a standard safe form.  The C++ file can be compiled with the BioCoder software available here.  http://research.microsoft.com/en-us/um/india/projects/biocoder/
 
II. Rna Structure Analysis
In the RNA structure tab there are images of the unmodified and modified RNA structure.  We hope that this will help the user decide whether the changes are structurally advantageous.  Hopefully in the future more advanced RNA structure modeling algorithms can be implemented to help the user make an informed decision.
 
 
Future of MYS!S
beta
Unfortunately, we’re talking about the capabilities of the future MYS!S v.10, for now all we have is the beta edition.  So for the time being, these are the imminent improvements we would like to make for the second version of MYS!S.
Better algorithms for modifying DNA
Currently, when determining how to modify DNA MYS!S does not take into account the eventual RNA structure and whether the changes will inhibit protein production.  We would like to incorporate algorithms that make changes to DNA in a way that will increase the amount of protein formed by translation. On the same note, it might also be helpful for a synthetic biologist not just to increase protein production but maybe to limit it.
 
Not just support site directed mutagenesis
Right now MYS!S for a codon optimization creates a large number of primers for a sequence of say 700bp.  We’re talking about upwards of 50 primers making site directed mutagenesis realistically impossible.  We would like MYS!S to support other methods of manipulating physical DNA.
 
Better visualization methods for RNA structure
We want the user to be able to visually check whether the RNA structure is acceptable.  If it is not acceptable the user should be able to manually modify the DNA sequence to improve the RNA structure.           Preference for lab protocols : Not all labs do things the same, MYS!S should be able to customize lab protocols to how the user’s lab gets things done.
 
 
Where can I get MYS!S?
Github
MYS!S is currently available as an Eclipse download.  It will very soon be available as a java application.
The downloads can be obtained from our GitHub site.
https://github.com/RutgersGEARS/iGEM-Rutgers-Software
MYS!S is still a work in progress.  Please feel free to report any bugs or crashes that occur as issues on our github page.
https://github.com/RutgersGEARS/iGEM-Rutgers-Software/issues
 
 
 
 
 
 
 
 
 
 
 
 
 
Retrieved from " http://2011.igem.org/Team:Rutgers/MYS!S "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
