"
Page
Discussion
View source
History
teams
Log in
 
Team:Rutgers/Full Adder
From 2011.igem.org
Abstract
                The Full Adder project seeks to create bacteria that can mimic a digital full adder. Since many teams have difficulty creating even something small like a XOR gate, this project would seem nearly impossible. However, we have found that the problem can be greatly simplified if we use a certain simple “encoding” on the outputs of the full adder. Subsequently designing a circuit to “decode” our circuit’s output will hopefully prove significantly easier than creating a complete full adder outright. Our insights may prove useful to any genetic engineer or synthetic biologist working on highly complex systems.
 
Overview
Engineering Problems
                A bacterial full adder faces two immediate problems. First, an electrical full adder requires at least 5 logic gates. However, a single AND gate in bacteria may require as many as 11 biobricks (as in Peking 2009). Chaining together tens of biobricks is a lot of work and may result in plasmids that are, at best, inconvenient to transform.
The second problem is that, for the most part, the current “logic gate” biobricks are not easily connected to each other.  We cannot easily connect the ETH 2006 XOR gate to the Davidson-Missouri Western (DMW) 2008 XOR gate since the inputs and outputs are not the same; ETH has PoPS ins and out, but DMW uses various chemicals. Simply using PoPS only parts may be an attractive solution, but that may exacerbate the first problem as parts with PoPS ins and outs tend to be more complex. Furthermore, there is a significant possibility of some PoPS parts interfering with the operation of other PoPS parts as these are huge, complex circuits, many of which share some parts.
 
Insights
                We had two major insights into the Full Adder problems. First, we found a way to easily create biological AND gates. Second, we found that, if we change the output of the full adder from 2 outputs to three outputs (our so-called encoding), we can create the circuit with 4 AND gates.
                Of course, this is no longer a faithful rendition of a full adder. However, the methods we used to create the AND gates should be helpful in the creation of a whole new generation of fusion proteins, and it should be easier to create another circuit that converts our three output adder to a standard two output adder than creating a single monolithic full adder circuit.
 
Peking 2009 AND Gate
                One AND gate that we decided to use was created previously by the Peking 2009 iGEM team. It relies on T7ptag, T7 polymerase with two early amber mutations (stop codons) such that normal translation will result in a truncated non-functional protein, and SupD, a tRNA which allows for translation through amber stop codons (an amber suppressor). So, if just T7ptag or just SupD is expressed, there will not be any functional T7 polymerase; and, if both T7ptag and SupD are expressed, then functional T7 polymerase will be created. To complete the gate, we place T7ptag on one inducible promoter (our first input), SupD on a different inducible promoter (our second input), and our output on a T7 promoter. Note that this system relies on the fact that basal transcription from the T7 promoter is extremely low, but, once functional T7 polymerase is present, transcription is very high.
 
 
Split Fluorescent Proteins
Overview
                In our search for AND gates, we sought AND gates that functioned similarly to Peking’s AND gate (and for that matter, most other AND gates); two things that are individually non-functional, but when both are expressed together function is restored. Ghosh et al. created exactly that with their split GFP. They cut GFP, which usually displays green fluorescence, into two parts, each of which did not fluoresce. Then they fused leucine zipper domains to each part which allowed them to reconstitute functional GFP when both were expressed together. They dubbed the two parts nzGFP and czGFP.
                Split fluorescent proteins are particularly attractive in C. elegans as they allow for combinatorial marking of cells. As C. elegans are transparent, a common method to study particular cells is to put fluorescent proteins under the transcription of promoters specific to those cells. Thus, only the interesting cells light up under a fluorescent microscope. However, some groups of cells are difficult to target with just a single promoter. With split fluorescent proteins, a scientist can light up cells which have high transcription at two different promoters, allowing for much a finer selection of cells to study. Thus, further research on split fluorescent proteins was done in C. elegans (Ghosh et al.’s study was in E. coli). Zhang et al. created similar split CFP and YFP (cyan and yellow fluorescent proteins).
                All of these split fluorescent proteins function essentially as AND gates. The inputs are whatever promoter we place the split fluorescent proteins under and the output is fluorescence. The primary advantage of this system is its simplicity. It just requires two proteins and two coding sequences. Furthermore, the coding sequences do not code for enzymes which can have all sorts of messy interactions with other parts. The primary disadvantage of this system is that the output is limited to fluorescence.
 
Construction
                As mentioned previously, a significant amount of the research on these split fluorescent proteins has been done in C. elegans. Unfortunately, C. elegans require introns for the expression of certain genes, and so much of the easily accessible DNA with split fluorescent proteins contains artificial introns, which would mess up expression in E. coli. We acquired such DNA for split GFP, CFP, and YFP. However, we decided that it was not such a bad thing. We aimed to clone the zipper portions for use in future split protein construction, and, as a demonstration of their effectiveness, we would recreate the split GFP, CFP, and YFP using fresh GFP, CFP, and YFP from the parts registry.
                We decided to use RFC25 to generate our fusion parts. We did not have any particular rationale. All of the fusion standards seem reasonable.
                First, we cloned the zipper domains from nzGFP and czGFP with the RFC25 prefix and suffix. Then we mutagenized GFP (BBa_E0040) so that it contained the necessary restriction sites in the correct positions in order to create nzGFP and czGFP. Finally, we cut and ligated the parts.
 
Circuit
                Nice image and description of how it works here.
Home
Team
Official Team Profile
Etch-a-Sketch
Full Adder
MYS!S
Parts Submitted to the Registry
Modeling
Notebook
Safety
Acknowledgements
Retrieved from " http://2011.igem.org/Team:Rutgers/Full_Adder "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
