<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Rutgers_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Rutgers/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Rutgers/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Rutgers/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Rutgers/Project&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Rutgers/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Rutgers/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>Rutgers project</TITLE></P><DIV class="container"><HEADER class="clearfix"><NAV id="menu" class="nav"><UL><LI><A href="https://2013.igem.org/Team:Rutgers/Project" class="active"><SPAN>project</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:Rutgers/People"><SPAN>people</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:Rutgers/Modelling"><SPAN>modelling</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:Rutgers/Safety"><SPAN>safety</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:Rutgers/Notebook"><SPAN>notebook</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:Rutgers/Practices"><SPAN>practices</SPAN></A></LI></UL></NAV></HEADER><DIV id="sidebar"><UL><LI><STRONG><A href="#Abstract">Abstract</A></STRONG></LI><LI><STRONG><A href="#PhaseI">Phase I</A></STRONG></LI><LI><CODE><A href="#lux"> -lux</A></CODE></LI><LI><A href="#qsda"> -qsdA</A></LI><LI><A href="#linker"> -linker</A></LI><LI><A href="#assayI"> -assay</A></LI><LI><STRONG><A href="#PhaseII">Phase II</A></STRONG></LI><LI><CODE><A href="#goal"> -goal</A></CODE></LI><LI><A href="#design"> -design</A></LI><LI><A href="#assayII"> -assay</A></LI><LI><A href="#future"> -future</A></LI></UL></DIV><DIV class="main" style="color:black"><DIV class="subjects"><H1 id="Abstract">ABSTRACT</H1><P>Our project aims at developing a self-regulatory system for the degradation of virulent factors. We plan to integrate quorum sensing and qsdA to create this system of self-regulation. Quorum sensing will be used to detect the presence of high cell densities and to activate qsdA upon detecting high cell density. qsdA, acting as a repressor, will degrade the signal molecules needed for quorum sensing and thus turn off quorum sensing as well as itself. It will reactivate upon the reactivation of quorum sensing and this cycle of self-regulation will persist. This prototype system is meant to test the application of this system but the main goal is to introduce this system to pseudomonas aeruginosa as a means to prevent the harm that it causes.</P><P>The quorum sensing regulatory gene circuit is meant to target pathogens such as pseudomonas aeruginosa that use quorum sensing as a means for regulation of virulent factors. However, our hope is that by creating a new paradigm for fighting virulence in pathogenic cells will provide insight for future works and endeavors.  This system is meant to not be only restricted to pseudomonas aeruginosa but to be altered to fight virulence in other pathogens.</P><H1 id="PhaseI">PHASE I</H1><H2 id="lux">LUX OPERON</H2><P>AHL or Acylated Homoserine Lactones belong to a class of autoinducers. These autoinducers act as chemical signaling molecules that gram-negative and gram-positive bacteria use as a way of communicating with one another. Additionally, this form of communication is used to regulate various physiological activities and more specifically virulence.</P><P>The detection of these autoinducers involves diffusion into the cells and binding to receptors. However, the binding autoinducers and receptors does not usually occur until a threshold concentration of autoinducers is reached. High concentrations of gram-negative and grampositive bacteria is often accompanied by a high concentration of autoinducers and so detection of these autoinducers is achieved in the presence of a high concentration of autoinducer producing bacteria [1]. This cell-cell communication in bacteria is a process called quorum sensing, which allows for bacteria to respond to fluctuations in the number of species present by altering physiological activities [1].</P><P>Acylated Homoserine Lactones are neutral lipid molecules composed of a homoserine lactone ring with an acyl chain. AHLs produced by different species of Gram-negative bacteria vary in length and composition of the acyl chain which contain 4 to 18 carbon atoms, making most quorum-sensing systems within bacterium species or group specific. AHLs are synthesized by AHL synthases and diffuse in and out of the cell by active or passive transport [1].</P><P><STRONG>LuxR:</STRONG> LuxR is a gene that codes for the receptor protein that binds to the autoinducer. As the receptor binds to the autoinducer, the complex binds a specific sequence of DNA in front of the right operon and activates the expression of lux structural genes. The LuxR protein is at first inactive but when the autoinducer binds the receptor protein becomes activated. The LuxR-autoinducer complex binds to a 20 bp palindromic sequence, called the LuxBox [2]. </P><P><STRONG>LuxI:</STRONG> LuxI is a gene that synthesizes the autoinducer.</P><P><STRONG>LuxBox:</STRONG> The LuxBox is a 20 bp palindromic promoter sequence. The binding of the luxR-autoinducer complex to this sequence increases RNA polymerase's affinity towards the promoter region of the LuxI and structural genes downstream of LuxI region [2]. </P><P>[1] Taga, Michiko E., and Bonnie L. Bassler. &quot;Chemical Communication among Bacteria.&quot;(Sackler NAS Colloquium) Chemical Communication in a Post-Genomic World. The National Academies, 25 Nov. 203. Web. 30 July 2013.</P><P>[2] Bluth, Brian J., Sarah E. Frew, and Brian McNally. &quot;Cell-Cell Communication and the Lux Operon in Vibrio Fischeri.&quot; Bioluminescence and the Lux Operon in Vibrio Fischeri. Carnegie Mellon University, 31 July 2013. Web. 31 July 2013. <A href="https://www.bio.cmu.edu/courses/03441/TermPapers/97TermPapers/lux/default.html">https://www.bio.cmu.edu/courses/03441/TermPapers/97TermPapers/lux/default.html</A>.</P><H2 id="qsda">qsdA</H2><P><STRONG>HSL Degradation Protein</STRONG> - Quorum quenching has remarkable properties that give it a pathway to become a major therapeutically relevant process. Quorum quenching primary focus is to turn off virulence expression in pathogenic bacteria when certain requirements are met.  The main requirement is that cell density of the bacteria needs to high enough that enough N-acyl-homoserine lactone (AHL) is present to activate quorum sensing. This would mean that quorum quenching can stop the expression of virulent factors at a critical point as well as allow the immune system to deal with the problem by itself before interfering. This method of treatment is different from conventional methods of treatment in that it does not interfere with DNA, RNA or protein synthesis.</P><P>Quorum quenching can be achieved through the use of qsdA, a protein found in Rhodococcus erythropolis W2. qsdA belongs to phosphotriesterase family that harbors phosphotriesterase, lactonase or amidohydrolase activities. qsdA has the ability to inactivate AHLs with a carbon chain ranging from six carbons to fourteen. It has also been shown that bacteria that have the qsdA gene interfere efficiently with quorum sensing functions. It should be noted that qsdA has a low sequence similarity with that of other lactonases; for example, AiiA is a well-studied lactonase that shows significant similarities with other lactonases, except for qsdA. This prompted us to examine qsdA in detail and try to find and add to the known array of information about qsdA and its functions in quorum quenching.</P><P>Chen, Fang, Yuxin Gao, Xiaoyi Chen, Zhimin Yu, and Zianzhen Li. &quot;Quorum Quenching Enzymes and Their Application in Degrading Signal Molecules to Block Quorum Sensing-Dependent Infection.&quot; International Journal of Molecular Sciences12.1422-0067 (2013): 17477-7500</P><H2 id="linker">GS LINKER</H2><P>The fusion of two proteins and its ability to still function correctly is facilitated by the glycine serine linker. The fusion of two proteins, and therefore the expression, requires that the folding and attributes of one protein not affected by the other. This is usually not the case, an extreme example being when the last amino acid of a particular protein is near some sort of active site. If the other protein was to begin its formation from this point then the active site and the functionality of the first protein would be at risk. In order to eliminate such risk there are linkers used. Linkers usually contain amino acids that are not very active and would not harm the functionality of the protein, glycine and serine are two very common amino acids used for linkers. The linker would be composed so that there would be an alternating trend of glycine then serene. The amount of reiterations is variable and depends on the location on the protein where the last amino acid is located. Normally, there is one or two reiterations of the glycine and serine. Upon the proper insertion of a linker, the expression of two proteins will be joined with no decrease in activity or functionality.</P><P>We are using a glycine serine linker in our project to join the expression of qsdA and GFP. The last amino acid of the qsdA protein lies on the surface of the protein and is far away from any important locations. Therefore, we are implementing a 2 repeats of glycine serine to make sure no factors harm either protein’s function. The reason we are implementing this fusion is to have a clear indication when qsdA in the cell has become active, GFP will become active and create green pigmentation, and to visually see the self-regulation that the lux operon and qsdA create.  This, amongst other things, will allow us to do multiple assays about qsdA and its function.</P><H2 id="assayI">ASSAY</H2><P><STRONG>Flourescence Assay</STRONG> - Our project submission BBa_6120600, is designed with a glycine serine amino acid link between the degradation enzyme qsda-1 and GFP reporter.  Because of this, when the enzyme begins degrading the autoinducer AHL in the surrounding environment, our plasmid will fluoresce green.  For this assay, we used a Tecan plate reader to measure relative fluorescence given off by our modified E. Coli cell when AHL is prominent in the cell’s environment.</P><P>To be sure we are getting correct results, we decided to measure the relative fluorescence given off by our project cell when it is uninduced, our project cell when it is induced by the autoinducer AHL, and cells of the green fluorescence protein that will be constitutively be expressing fluorescence.  The hypothesized results before the assay was preformed were that the GFP would show the most relative fluorescence.  That the uninduced project cells will show little to no fluorescence, and the induced project cells will have a relative fluorescence in between the two.</P><P>Before we could run the assay, we had to look up the emission wavelength and the excitation wavelength from the iGEM parts registry data base for the specific green fluorescence protein we chose from the iGEM biobrick.  These values were 395 nm for emission and 515 nm for excitation wavelength.  These values were submitted into the Tecan plate reader computer program for accurate readings.  When we ran our plate reader, the average fluorescence for each were determined, and the results fit in with our hypothesis.  The GFP cells fluoresced the most, the uniduced cells fluoresced the least, and the induced project cells relative fluorescence was in the midde.  The average relative fluorescence of the three are as follows and are measured as a numerical value of relative fluorescence given off.</P><TABLE style="margin-bottom:30px"><TBODY><TR><TD>Green Flourescence proteins</TD><TD>42,205</TD></TR><TR><TD>Induced E. Coli cell</TD><TD>31,248</TD></TR><TR><TD>Uninduced E. Coli cell</TD><TD>24,025</TD></TR></TBODY></TABLE><H1 id="PhaseII">PHASE II</H1><P>One of the developing problems in healthcare is how to treat bacterial infections. Since the discovery of penicillin, antibiotics have been the answer. Antibiotics work by selectively targeting bacteria and killing them before they are able to reproduce. Unfortunately, when bacteria are treated with antibiotics, some will develop a resistance to particular strains of antibiotics due to random mutations. Antibiotic pressure is the term used to describe antibiotic promotion of resistance development by pathogens1. The evolution of bacteria, stimulated by antibiotics, is easily observed because of how quickly new bacterial generations are produced. If bacteria become resistant to multiple antibiotics, they can be termed “superbugs”.</P><P>As one could imagine, this problem is magnified in hospital settings. Hospitals have a high usage of antibiotics and therefore are exposing strains of bacteria to many antibiotics2. Some examples of resistant bacteria pathogens are methicillin resistant Staphylococcus aureus, Enterococci, Pseudomonas, and Acinetobacter1.</P><P>New methods to eliminate bacteria to circumvent traditional antibiotics are being investigated. One of these methods is quenching autoinducing peptides of quorum sensing bacteria. Quorum sensing is the ability of microorganisms to coordinate gene expression in relation to their population density3. When certain bacteria are in high- density situations, small autoinducers or peptides are released to neighboring bacteria.</P><P>After binding to other bacteria, the peptide initiates a change in the expression of certain genes. This could result in bioluminescence in some strains of bacteria such as Vibrio fischeri, or the release of virulent factors in other strains such as Staphylococcus aureus5. S. aureus infections can be devastating or relatively minor. It can result in lesions, blood poisoning, and septic arthritis amongst other conditions.</P><P>The peptides are only effective when numerous bacterial organisms are present. If only a few are present, the effect will not be large enough to succeed. In principle, it should be possible to eliminate bacterial infections by intercepting this communication system. In fact, Park et al. demonstrated this potential. But, first, a little more information regarding S. aureus is necessary. The two main virulent factors in the organism are protein A and alpha-Hemolysin with both proteins being regulated by quorum sensing3. They showed that interrupting the signaling communication on blood agar plates diminished expression of alpha-Hemolysin. Additionally, it was shown that staphylococci infection in mice was reduced with the administration of an antibody to bind the autoinducing peptide3. This research offers promise that quenching of autoinducing peptides can be a viable treatment. Utilizing quorum quenching treatments over traditional antibiotics is advantageous because it does not provide selective pressure against the bacteria3. The expression of virulent factors is suppressed, but the essential metabolic activities of the individual bacteria are not disrupted and bacterial resistance developed as easily.</P><H2 id="goal">GOAL</H2><P>We hope to take the research described above a step further by designing an enzyme to disrupt the quorum sensing system by degrading the autoinducers. While antibodies can bind to autoinducers, they cannot hydrolyze them. Additionally, antibodies are bulky structures, which can pose problems for drug delivery applications. The particular peptide precursor for MRSA has a thioester linkage created from a bond between a cysteine residue and the terminal carboxyl group. The thioester linkage will be the target for the enzyme to hydrolyze the peptide precursor and disrupt the quorum sensing system of the bacteria.</P><FIGURE><FIGCAPTION>Different Structures of AIP<SUP>3</SUP></FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Computational Model of AIP-4</FIGCAPTION></FIGURE><H2 id="design">DESIGN</H2><P>Rosetta is an essential software tool for protein redesign. The software optimizes the residues around the transition state to maximize the catalytic efficiency. This is achieved by first providing the software with a model of the transition state surrounded by the side chain and backbone of a protein in the best theoretical positions. The enzyme model can then be called a theozyme. Rosetta then Score. Rosetta Scores correlate with Gibbs Free Energy, but do not measure entropy directly, only indirectly. The design process utilizing Rosetta3 can be broken down into four parts: choice of catalytic mechanism and corresponding theozyme, matching identifying sites in the scaffold library where the theozyme can be placed, designing the found sites, and evaluating and ranking the resulting designed sequences. This process allows one to alter the substrate specificity of an enzyme<SUP>4</SUP>.</P><P>Foldit is a program that provides a visualization of all of the interactions a protein makes. It is particularly helpful in viewing hydrophobic, aromatic stacking, and hydrogen bonding interactions. Foldit also contains a scoring function that useful for understanding the effects the potential side chain mutations can make. We made the following mutations, listed below, to enhance binding.</P><TABLE><TBODY><TR><TD><P>Original Score: <CODE>-983.35</CODE></P></TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><I>Mutation</I></TH><TH><I>Original</I></TH><TH><I>New Score</I></TH></TR><TR><TD>Aspartate-54</TD><TD>Alanine</TD><TD><CODE>-990.54</CODE></TD></TR><TR><TD>Asparagine-103</TD><TD>Threonine</TD><TD><CODE>-988.43</CODE></TD></TR><TR><TD>Methionine-105</TD><TD>Asparagine</TD><TD><CODE>-991.94</CODE></TD></TR><TR><TD>Asparagine-101</TD><TD>Threonine</TD><TD><CODE>-988.35</CODE></TD></TR></TBODY></TABLE><FIGURE><FIGCAPTION>Foldit representation of designed protein bound to AIP-4 showing the numerous intramolecular forces</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Designed protein active site with AIP-4 and mutations colored</FIGCAPTION></FIGURE><H2 id="assayII">ASSAY</H2><P>If the designed protein is successful, it will hydrolyze the autoinducer at the thioester linkage and produce free cysteine with exposed thiol groups. To determine the protein’s activity, we utilized Ellman’s Reagent. This reagent enables us to measure the concentration of sulfur in solution with UV absorbance. The reaction that Ellman’s Reagent undergoes in the presence of sulfur can be seen below.</P><P>Initial results are promising. As the figure shows, when the thiol absorbance of AIP and the protein are measured individually, the combined absorbance is 0.326. However when the two are measured simultaneously (interacting with each other), the absorbance rises to 0.367. This appears to indicate that the protein hydrolyzed the peptide and increased the sulfur concentration.</P><H2 id="future">FUTURE</H2><P>The designed protein contains a total of seven cysteine residues. While these should be accounted for with the background absorbance measurements, it would be ideal to remove this absorbance by mutating the residues to another amino acid with similar properties, such as serine. Additionally, further mutations to residues can be used to enhance the binding of the protein with the various types of autoinducers to generate specificity for different strains of bacteria.</P><TABLE style="margin-bottom:40px"><TBODY><TR><TH><I>Original</I></TH><TH><I>Potential Mutation</I></TH><TH><I>Amino Acid number</I></TH></TR><TR><TD>Threonine</TD><TD>Glutamine</TD><TD><CODE>101</CODE></TD></TR><TR><TD>Threonine</TD><TD>Serine</TD><TD><CODE>101</CODE></TD></TR><TR><TD>Alanine</TD><TD>Serine</TD><TD><CODE>54</CODE></TD></TR><TR><TD>Alanine</TD><TD>Glutamate</TD><TD><CODE>54</CODE></TD></TR><TR><TD>Alanine</TD><TD>Serine</TD><TD><CODE>56</CODE></TD></TR><TR><TD>Aspartate</TD><TD>Methionine</TD><TD><CODE>175</CODE></TD></TR><TR><TD>Glutamate</TD><TD>Glutamine</TD><TD><CODE>60</CODE></TD></TR><TR><TD>Glutamate</TD><TD>Aspartate</TD><TD><CODE>60</CODE></TD></TR><TR><TD>Glutamine</TD><TD>Alanine</TD><TD><CODE>233</CODE></TD></TR><TR><TD>Glutamine</TD><TD>Methionine</TD><TD><CODE>233</CODE></TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Rutgers/Project">http://2013.igem.org/Team:Rutgers/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Rutgers/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Rutgers/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Rutgers/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Rutgers/Project&amp;oldid=241682" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>