<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Frankfurt_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Frankfurt/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Frankfurt/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Frankfurt/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Frankfurt/Notebook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Frankfurt/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Frankfurt/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE> Team: iGEM Frankfurt - 2012.igem.org </TITLE></P><DIV id="wrap"><DIV id="left"><UL id="menu"><LI><A href="https://2012.igem.org/Team:Frankfurt">Home</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/Team">Team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2012&amp;team_name=Frankfurt">Official Profile</A></LI></UL><LI><A href="https://2012.igem.org/Team:Frankfurt/Project">Project</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/Organisms">Organisms</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/New_Yeast_RFC">Yeast RFC</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/Parts">Parts</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/Notebook">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/Notebook">Labwork</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/Protocol">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/Safety">Safety</A></LI><LI><A>Human practices</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/Ethics">Ethics</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/Events">Events</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/BioSecurity">BioSecurity</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/Attributions">Attributions</A></LI><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Labwork"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Labwork</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#May_and_June_2012"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">May and June 2012</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#July_2012"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">July 2012</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#August_2012"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">August 2012</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#September_2012"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">September 2012</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Labwork">Labwork</SPAN></H1><H2><SPAN class="mw-headline" id="May_and_June_2012">May and June 2012</SPAN></H2><P>1. Arrangements for labwork </P><UL><LI> preparation of competent cells (<I>E.coli</I>, <I>S.cerevisiae</I>), agarose plates (LB, YEPD, SCD-ura,…), medium for <I>E.coli</I> and <I>S.cerevisiae</I></LI></UL><P>2. Purchasing of the equipment (reaction tubes, glass bottles, pipette tips,..)
3. Primer design
</P><H2><SPAN class="mw-headline" id="July_2012">July 2012</SPAN></H2><P>1. Plasmid isolation of p426, p423, pUD8e, pUD22e from <I>E.coli</I>
2. Isolation of chromosomal DNA of CEN.PK2-1C
3. Trials to get the genes, promoters and terminators via PCR</P><H2><SPAN class="mw-headline" id="August_2012">August 2012</SPAN></H2><P>1. PCR of the genes, promoters and terminators</P><UL><LI> all genes (without <I>KO</I> and <I>KAH</I>), promoters and terminators could be amplified
</LI></UL><P>2. Linearization of p426 and p423 with <I>SpeI</I> and <I>XhoI</I></P><TABLE class="wikitable zebra"><TBODY><TR><TH>Templates</TH><TH>Amplified DNA Fragments
</TH></TR><TR><TD>synthesized sequence of <I>HMG-CoA</I></TD><TD><I>HMG-CoA</I></TD></TR><TR><TD>synthesized sequence of <I>GGPPS</I></TD><TD><I>GGPPS</I></TD></TR><TR><TD>synthesized sequence of <I>Cps/Ks</I></TD><TD><I>CPS/KS</I></TD></TR><TR><TD>chromosomal DNA of CEN.PK2-1C</TD><TD><I>ERG20</I></TD></TR></TBODY></TABLE><TABLE width="100%" align="center"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Gel 1: PCR of the promoters and terminators.</B>There are shown the DNA fragments of the PCR with appropriate annealing temperatures. All promoters and terminators could be amplified. As templates were used the plasmids pUD8e(<I>tHXT7</I>, <I>pPFK1</I>, <I>tTAL1</I>, <I>pTPI1</I>) and pUD22e (<I>tPFK2</I>, <I>tPDC1</I>, <I>pPGI1</I>).</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:417px;"><DIV class="thumbcaption"><B>Gel 2: Linearization of p426, p423, pSB1C3 and PCR of the genes <I>HMG-CoA</I>, <I>GGPPS</I>, <I>ERG20</I>, <I>CPS/KS</I>.</B> As a control it is also the circular p426 and p423 shown. The linear p426 and p423 are about 6300 bp long. pSB1C3-linear means the restriction of the plasmid with <I>EcoRI</I> and <I>PstI</I> in this case (it is 2050 bp long). As PCR templates for <I>HMG-CoA</I>, <I>GGPP</I>S and <I>CPS/KS</I> were used the synthesized DNA fragments and for <I>ERG20</I> chromosomal DNA of CEN.PK2-1C.</DIV></DIV></DIV></TABLE><P>3. Biobrick production of the genes <I>HMG-CoA</I>, <I>ERG20</I>, <I>CPS/KS</I></P><UL><LI> restriction of 3 µg of the genes with <I>EcoRI</I> and <I>PstI</I></LI><LI> ligation of biobrick genes with linear pSB1C3</LI><LI> transformation of the ligation in <I>E.coli</I></LI><LI> plasmid isolation of <I>E.coli</I> clones</LI><LI> control restriction of biobrick plasmids with <I>EcoRI</I> and <I>SpeI</I></LI></UL><TABLE width="100%" align="center"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:372px;"><DIV class="thumbcaption"><B>Gel 3: Control restriction of pSB1C3-<I>HMG-CoA</I>, pSB1C3-<I>ERG20</I> and PCR of <I>GGPPS</I>.</B> The biobrick plasmids were cut with <I>EcoRI</I> and <I>SpeI</I>. The correct sizes of pSB1C3-<I>HMG-</I> CoA<I> from two clones could be observed (2050 bp + 1605 bp), as well as the correct sizes of</I> pSB1C3-<I>ERG20</I> (2050 bp + 1080 bp). However the fragment of 1080 bp is very hard to see on this picture. With the PCR of <I>GGPPS</I> could not only amplified the correct fragment (870 bp) but also a fragment of 3000 bp.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:372px;"><DIV class="thumbcaption"><B>Gel 5: Control restriction of pSB1C3-<I>CPS/KS</I>.</B> The biobrick plasmids were cut with <I>EcoRI</I> and <I>SpeI</I>. The correct sizes of pSB1C3-<I>CPS/</I> KS<I> from two clones could be observed (2050 bp + 2900 bp). The plasmid of clone 3 is the</I> pSB1C3 without the insert.</DIV></DIV></DIV></TABLE><P>4. Formation of the mevalonate overexpression plasmid via gap repair
</P><UL><LI> first and second yeast transformation with equimolar quantities of DNA fragments for mevalonate overexpression (p426 with 7 inserts): only very small colonies could grow after the first and second transformation
</LI><LI> using pure <I>GGPPS</I> (purification of a preparative gel) for the third yeast transformation: normal size of the colonies
</LI><LI> inoculation of several clones of the third yeast transformation
</LI><LI> plasmid preparation of the clones
</LI><LI> transformation of the plasmids in <I>E.coli</I></LI></UL><P>5. Amplifying pSB1C3 for biobrick production
</P><UL><LI> trials to amplify pSB1C3, whose blunt ends were ligated and transformed in <I>E.coli</I></LI><LI> pSB1C3 should be linearized by <I>EcoRI</I> and <I>PstI</I> : did not work (two fragments instead of one)</LI></UL><TABLE width="100%" align="center"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Gel 4: Purified <I>GGPPS</I> and linearization of pSB1C3.</B>The 870 bp fragment of the <I>GGPPS</I> PCR (see gel 3) were cut out from a preparative gel and then purified with a gel extraction kit (The purified <I>GGPPS</I> could be used for transformation, but not for constructing a biobrick. Therefore another PCR was made with a very short synthesis time. So only the 870 bp fragment was obtained.)Because of lack of pSB1C3 the remained linear pSB1C3 was ligated and transformed in <I>E.coli</I>. After isolation of the plasmid it was restricted with <I>EcoRI</I> and <I>PstI</I>. But there was not only the linear fragment of 2050 bp but also another fragment about 4000 bp.</DIV></DIV></DIV></TABLE><UL><LI> preparative gel of the linear fragment: very low concentration of linear pSB1C3 (was not sufficient for ligation)
</LI></UL><P>6. GC analysis
</P><UL><LI> GC analysis of the wild type CEN.PK2-1C (standard GGOH): as expected no GGOH could be observed
</LI></UL><P>7. Assembly of the <I>KO</I> and the <I>KAH</I> fragments
</P><UL><LI> amplification of the fragments via PCR (there are four fragments of each gene with an overhang to the fragment beside of 30 bp)
</LI><LI> Gibson assembly of the fragments of <I>KO</I> and <I>KAH</I>: did not work</LI></UL><TABLE width="100%" align="center"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Gel 6: PCR of <I>KO</I> and <I>KAH</I> fragments</B> Each gene consists of four fragments, which have a 30 bp overhang to the fragments beside. In order to do the Gibson Assembly, the fragments were amplified via PCR.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:335px;"><DIV class="thumbcaption"><B>Gel 7: Gibson Assembly of <I>KO</I> and <I>KAH</I></B> The assembly did not work, because there are only the 500 bp of the single fragments.</DIV></DIV></DIV></TABLE><H2><SPAN class="mw-headline" id="September_2012">September 2012</SPAN></H2><P>1. Formation of the mevalonate overexpression plasmid via gap repair
</P><UL><LI> plasmid isolation (p426 with 7 inserts) from <I>E.coli</I></LI><LI> control restriction of the plasmids with <I>EcoRI</I> and <I>SpeI</I>: one clone out of 10 got the right sizes
</LI></UL><TABLE width="100%" align="center"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:272px;"><DIV class="thumbcaption"><B>Gel 11: Possible mevalonate overexpression plasmids from <I>E.coli</I>.</B>After yeast transformation with p426 and 7 inserts, isolation of the plasmids, transformation in <I>E.coli</I> and isolation of them, they were loaded on a gel with a negative control (p426) in order to see which plasmid runs slowest. In this case it is the plasmid of clone 10.</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:272px;"><DIV class="thumbcaption"><B>Gel 12: Control restriction of possible mevalonate overexpression plasmids.</B>The plasmids of clone 1, 5 and 10 were cut with EcoRI. The correct sizes of the mevalonate overexpression plasmids are 7000 bp, 2400 bp and 1900 bp. The plasmid of clone 10 seems to be the correct one.</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:272px;"><DIV class="thumbcaption"><B>Gel 13: Control restriction of possible mevalonate overexpression plasmids.</B>The plasmids of clone 2, 3 and 10 were cut with <I>SpeI</I>. The correct sizes of the mevalonate overexpression plasmids are 7800 bp, 1900 bp and 1610 bp. The plasmid of clone 10 seems to be the correct one. To have a positive control p426 was also cut with <I>SpeI</I> (6300 bp).</DIV></DIV></DIV></TABLE><P>2. Ergosterol experiment
</P><UL><LI> idea: maybe the clones of the first and the second yeast transformation grow better after ergosterol addition (0,02 g/l)): wildtype with and without ergosterol and one of the clones with and without ergosterol (no significant difference in growth could be observed)
</LI></UL><P>3. <I>GGPPS</I>-PCR
</P><UL><LI> PCR of <I>GGPPS</I> with shorter synthesis time in order to get only the correct fragment
</LI></UL><P>4. Amplification of pSB1C3 for biobrick production
</P><UL><LI> transformation of pSB1C3-RFP in <I>E.coli</I></LI><LI> isolation of the plasmid from <I>E.coli</I></LI><LI> linearization with <I>EcoRI</I> and <I>PstI</I>: it worked (only the correct fragment was observed)
</LI></UL><P>5. PCR of biobrick-promoters and -terminators
</P><UL><LI> PCR of biobrick promoters and terminators that were used to build the mevalonate overexpression plasmid
</LI></UL><TABLE width="100%" align="center"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Gel 10: PCR for biobrick-promoters and –terminators.</B> There are shown the DNA fragments of the PCR with appropriate annealing temperatures. All promoters and terminators could be amplified, except <I>pHXT7</I>. As templates were used the plasmids pUD8e (<I>tHXT7</I>, <I>pPFK1</I>), pUD22e (<I>tPFK2</I>, <I>pPGK1</I>) and p426 (<I>pHXT7</I>, <I>tCYC1</I>).</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Gel 14: PCR for biobrick-<I>pHXT7</I> and –<I>pPGK1</I>.</B> There are shown the DNA fragments of the PCR with appropriate annealing temperatures. As templates were used the plasmids pUD22e (<I>pPGK1</I>) and p426 (<I>pHXT7</I>).</DIV></DIV></DIV></TABLE><P>6. Assembly of the <I>KO</I> and the <I>KAH</I> fragments
</P><UL><LI> trials to assembly the <I>KO</I> and the <I>KAH</I> fragments via PCR (the overhangs were used as primer): <I>KO</I> worked, <I>KAH</I> not
</LI></UL><TABLE width="100%" align="center"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Gel 8: Assembly of <I>KO1/2</I>, <I>KO3/4</I>, <I>KAH1/2</I> and <I>KAH3/4</I> via PCR.</B> Because the Gibson Assembly did not work the assembly was done via PCR. Therefore the 30 bp overhangs were used as primer and the appropriate annealing temperature was chosen.<I>KO1/2</I>: 500 bp + 500 bp = 1000 bp<I>KO3/4</I>: 500 bp + 240 bp = 740 bp<I>KAH1/2</I>: 500 bp + 500 bp = 1000 bp<I>KAH3/4</I>: 500 bp + 260 bp = 760 bp</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><B>Gel 9: Assembly of <I>KO</I> and <I>KAH</I> via PCR.</B>The assembly of <I>KAH</I> did not work. For the PCR of <I>KO</I> were used the assembled <I>KO1/2</I> and <I>KO3/4</I> with an appropriate annealing temperature (the 30 bp overhang was used as primer). The <I>KO</I> fragment (1600 bp) was then amplified via a PCR with normal primers.</DIV></DIV></DIV></TABLE><UL><LI> amplification of <I>KO</I></LI></UL><P>7. Biobrick production of the genes <I>GGPPS</I>, <I>KO</I> and the promoters and terminators
</P><UL><LI> restriction of 3 µg of the DNA fragments with <I>EcoRI</I> and <I>PstI</I></LI><LI> ligation of the promoters, terminators, <I>KO</I> and the <I>GGPPS</I> with pSB1C3
</LI><LI> transformation of the ligation in <I>E.coli</I></LI><LI> plasmid isolation of <I>E.coli</I> clones
</LI><LI> control restriction of biobrick plasmids with <I>EcoRI</I> and <I>SpeI</I></LI></UL><P>8. Midi-preparation of plasmids for sequencing
</P><UL><LI> biobrick plasmids
</LI><LI> mevalonate overexpression plasmid
</LI></UL><P>9. GC
</P><UL><LI> GC analysis of the wild type CEN.PK2-1C with p426 and the wild type with the mevalonate overexpression plasmid in SCD-ura
</LI><LI> standard GGOH
</LI></UL><P>10. Formation of the plasmid for steviol synthesis
</P><UL><LI> yeast transformation with equimolar quantities of DNA fragments for steviol synthesis (p423 with 7 inserts)
</LI><LI> problem: could not assemble KAH, assembled <I>KAH1/2</I> and <I>KAH3/4</I> were used (<I>KAH1/2</I> and <I>KAH3/4</I> only got a 30 bp overhang instead of a 45 bp overhang): gap repair did not work
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Frankfurt/Notebook">http://2012.igem.org/Team:Frankfurt/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Frankfurt/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Frankfurt/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Frankfurt/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Frankfurt/Notebook&amp;oldid=220167" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>