"
Page
Discussion
View source
History
teams
Log in
 
Team:Frankfurt/Notebook
From 2012.igem.org
Team: iGEM Frankfurt - 2012.igem.org
Home
Team
Official Profile
Project
Organisms
Yeast RFC
Parts
Notebook
Labwork
Protocols
Safety
Human practices
Ethics
Events
BioSecurity
Attributions
Contents
1Labwork
1.1May and June 2012
1.2July 2012
1.3August 2012
1.4September 2012
Labwork
May and June 2012
1. Arrangements for labwork
preparation of competent cells (E.coli, S.cerevisiae), agarose plates (LB, YEPD, SCD-ura,…), medium for E.coli and S.cerevisiae
2. Purchasing of the equipment (reaction tubes, glass bottles, pipette tips,..) 3. Primer design
July 2012
1. Plasmid isolation of p426, p423, pUD8e, pUD22e from E.coli
2. Isolation of chromosomal DNA of CEN.PK2-1C
3. Trials to get the genes, promoters and terminators via PCR
August 2012
1. PCR of the genes, promoters and terminators
all genes (without KO and KAH), promoters and terminators could be amplified
2. Linearization of p426 and p423 with SpeI and XhoI
Templates
Amplified DNA Fragments
synthesized sequence of HMG-CoA
HMG-CoA
synthesized sequence of GGPPS
GGPPS
synthesized sequence of Cps/Ks
CPS/KS
chromosomal DNA of CEN.PK2-1C
ERG20
Gel 1: PCR of the promoters and terminators.
There are shown the DNA fragments of the PCR with appropriate annealing temperatures. All promoters and terminators could be amplified. As templates were used the plasmids pUD8e(tHXT7, pPFK1, tTAL1, pTPI1) and pUD22e (tPFK2, tPDC1, pPGI1).
Gel 2: Linearization of p426, p423, pSB1C3 and PCR of the genes HMG-CoA, GGPPS, ERG20, CPS/KS.
As a control it is also the circular p426 and p423 shown. The linear p426 and p423 are about 6300 bp long. pSB1C3-linear means the restriction of the plasmid with EcoRI and PstI in this case (it is 2050 bp long). As PCR templates for HMG-CoA, GGPPS and CPS/KS were used the synthesized DNA fragments and for ERG20 chromosomal DNA of CEN.PK2-1C.
3. Biobrick production of the genes HMG-CoA, ERG20, CPS/KS
restriction of 3 µg of the genes with EcoRI and PstI
ligation of biobrick genes with linear pSB1C3
transformation of the ligation in E.coli
plasmid isolation of E.coli clones
control restriction of biobrick plasmids with EcoRI and SpeI
Gel 3: Control restriction of pSB1C3-HMG-CoA, pSB1C3-ERG20 and PCR of GGPPS.
The biobrick plasmids were cut with EcoRI and SpeI. The correct sizes of pSB1C3-HMG- CoA from two clones could be observed (2050 bp + 1605 bp), as well as the correct sizes of pSB1C3-ERG20 (2050 bp + 1080 bp). However the fragment of 1080 bp is very hard to see on this picture. With the PCR of GGPPS could not only amplified the correct fragment (870 bp) but also a fragment of 3000 bp.
Gel 5: Control restriction of pSB1C3-CPS/KS.
The biobrick plasmids were cut with EcoRI and SpeI. The correct sizes of pSB1C3-CPS/ KS from two clones could be observed (2050 bp + 2900 bp). The plasmid of clone 3 is the pSB1C3 without the insert.
4. Formation of the mevalonate overexpression plasmid via gap repair
first and second yeast transformation with equimolar quantities of DNA fragments for mevalonate overexpression (p426 with 7 inserts): only very small colonies could grow after the first and second transformation
using pure GGPPS (purification of a preparative gel) for the third yeast transformation: normal size of the colonies
inoculation of several clones of the third yeast transformation
plasmid preparation of the clones
transformation of the plasmids in E.coli
5. Amplifying pSB1C3 for biobrick production
trials to amplify pSB1C3, whose blunt ends were ligated and transformed in E.coli
pSB1C3 should be linearized by EcoRI and PstI : did not work (two fragments instead of one)
Gel 4: Purified GGPPS and linearization of pSB1C3.
The 870 bp fragment of the GGPPS PCR (see gel 3) were cut out from a preparative gel and then purified with a gel extraction kit (The purified GGPPS could be used for transformation, but not for constructing a biobrick. Therefore another PCR was made with a very short synthesis time. So only the 870 bp fragment was obtained.)
Because of lack of pSB1C3 the remained linear pSB1C3 was ligated and transformed in E.coli. After isolation of the plasmid it was restricted with EcoRI and PstI. But there was not only the linear fragment of 2050 bp but also another fragment about 4000 bp.
preparative gel of the linear fragment: very low concentration of linear pSB1C3 (was not sufficient for ligation)
6. GC analysis
GC analysis of the wild type CEN.PK2-1C (standard GGOH): as expected no GGOH could be observed
7. Assembly of the KO and the KAH fragments
amplification of the fragments via PCR (there are four fragments of each gene with an overhang to the fragment beside of 30 bp)
Gibson assembly of the fragments of KO and KAH: did not work
Gel 6: PCR of KO and KAH fragments
Each gene consists of four fragments, which have a 30 bp overhang to the fragments beside. In order to do the Gibson Assembly, the fragments were amplified via PCR.
Gel 7: Gibson Assembly of KO and KAH
The assembly did not work, because there are only the 500 bp of the single fragments.
September 2012
1. Formation of the mevalonate overexpression plasmid via gap repair
plasmid isolation (p426 with 7 inserts) from E.coli
control restriction of the plasmids with EcoRI and SpeI: one clone out of 10 got the right sizes
Gel 11: Possible mevalonate overexpression plasmids from E.coli.
After yeast transformation with p426 and 7 inserts, isolation of the plasmids, transformation in E.coli and isolation of them, they were loaded on a gel with a negative control (p426) in order to see which plasmid runs slowest. In this case it is the plasmid of clone 10.
Gel 12: Control restriction of possible mevalonate overexpression plasmids.
The plasmids of clone 1, 5 and 10 were cut with EcoRI. The correct sizes of the mevalonate overexpression plasmids are 7000 bp, 2400 bp and 1900 bp. The plasmid of clone 10 seems to be the correct one.
Gel 13: Control restriction of possible mevalonate overexpression plasmids.
The plasmids of clone 2, 3 and 10 were cut with SpeI. The correct sizes of the mevalonate overexpression plasmids are 7800 bp, 1900 bp and 1610 bp. The plasmid of clone 10 seems to be the correct one. To have a positive control p426 was also cut with SpeI (6300 bp).
2. Ergosterol experiment
idea: maybe the clones of the first and the second yeast transformation grow better after ergosterol addition (0,02 g/l)): wildtype with and without ergosterol and one of the clones with and without ergosterol (no significant difference in growth could be observed)
3. GGPPS-PCR
PCR of GGPPS with shorter synthesis time in order to get only the correct fragment
4. Amplification of pSB1C3 for biobrick production
transformation of pSB1C3-RFP in E.coli
isolation of the plasmid from E.coli
linearization with EcoRI and PstI: it worked (only the correct fragment was observed)
5. PCR of biobrick-promoters and -terminators
PCR of biobrick promoters and terminators that were used to build the mevalonate overexpression plasmid
Gel 10: PCR for biobrick-promoters and –terminators.
There are shown the DNA fragments of the PCR with appropriate annealing temperatures. All promoters and terminators could be amplified, except pHXT7. As templates were used the plasmids pUD8e (tHXT7, pPFK1), pUD22e (tPFK2, pPGK1) and p426 (pHXT7, tCYC1).
Gel 14: PCR for biobrick-pHXT7 and –pPGK1.
There are shown the DNA fragments of the PCR with appropriate annealing temperatures. As templates were used the plasmids pUD22e (pPGK1) and p426 (pHXT7).
6. Assembly of the KO and the KAH fragments
trials to assembly the KO and the KAH fragments via PCR (the overhangs were used as primer): KO worked, KAH not
Gel 8: Assembly of KO1/2, KO3/4, KAH1/2 and KAH3/4 via PCR.
Because the Gibson Assembly did not work the assembly was done via PCR. Therefore the 30 bp overhangs were used as primer and the appropriate annealing temperature was chosen.
KO1/2: 500 bp + 500 bp = 1000 bp
KO3/4: 500 bp + 240 bp = 740 bp
KAH1/2: 500 bp + 500 bp = 1000 bp
KAH3/4: 500 bp + 260 bp = 760 bp
Gel 9: Assembly of KO and KAH via PCR.
The assembly of KAH did not work. For the PCR of KO were used the assembled KO1/2 and KO3/4 with an appropriate annealing temperature (the 30 bp overhang was used as primer). The KO fragment (1600 bp) was then amplified via a PCR with normal primers.
amplification of KO
7. Biobrick production of the genes GGPPS, KO and the promoters and terminators
restriction of 3 µg of the DNA fragments with EcoRI and PstI
ligation of the promoters, terminators, KO and the GGPPS with pSB1C3
transformation of the ligation in E.coli
plasmid isolation of E.coli clones
control restriction of biobrick plasmids with EcoRI and SpeI
8. Midi-preparation of plasmids for sequencing
biobrick plasmids
mevalonate overexpression plasmid
9. GC
GC analysis of the wild type CEN.PK2-1C with p426 and the wild type with the mevalonate overexpression plasmid in SCD-ura
standard GGOH
10. Formation of the plasmid for steviol synthesis
yeast transformation with equimolar quantities of DNA fragments for steviol synthesis (p423 with 7 inserts)
problem: could not assemble KAH, assembled KAH1/2 and KAH3/4 were used (KAH1/2 and KAH3/4 only got a 30 bp overhang instead of a 45 bp overhang): gap repair did not work
Retrieved from " http://2012.igem.org/Team:Frankfurt/Notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
