<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Frankfurt_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Frankfurt/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Frankfurt/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Frankfurt/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Frankfurt/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Frankfurt/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Frankfurt/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE> Team: iGEM Frankfurt - 2012.igem.org </TITLE></P><DIV id="wrap"><DIV id="left"><UL id="menu"><LI><A href="https://2012.igem.org/Team:Frankfurt">Home</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/Team">Team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2012&amp;team_name=Frankfurt">Official Profile</A></LI></UL><LI><A href="https://2012.igem.org/Team:Frankfurt/Project">Project</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/Organisms">Organisms</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/New_Yeast_RFC">Yeast RFC</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/Parts">Parts</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/Notebook">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/Notebook">Labwork</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/Protocol">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/Safety">Safety</A></LI><LI><A>Human practices</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/Ethics">Ethics</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/Events">Events</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/BioSecurity">BioSecurity</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/Attributions">Attributions</A></LI><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Our_project_aims_for_this_years_competition"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Our project aims for this years competition</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#The_steviol_precursor_metabolism_in_yeast"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">The steviol precursor metabolism in yeast</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#The_early_gibberelline_pathway_for_the_production_of_steviol"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">The early gibberelline pathway for the production of steviol</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-4"><A href="#DNA_constructs_for_realisation_of_our_idea"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">DNA constructs for realisation of our idea</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Mevalonate_Pathway_Overexpression"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Mevalonate Pathway Overexpression</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Steviol_Production"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Steviol Production</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#DNA_sources"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">DNA sources</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Sequence_Editing"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Sequence Editing</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Gas_Chromatographic_Analysis_for_Examination_of_the_Product_content_of_the_cells"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Gas Chromatographic Analysis for Examination of the Product content of the cells</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Gap_repair_Cloning_for_Vector_Assembly"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">Gap repair Cloning for Vector Assembly</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Our_project_aims_for_this_years_competition"> Our project aims for this years competition </SPAN></H1><P>The sweetening agents produced by <I>Stevia rebaudiana</I> are all derived from the common precursor steviol which belongs to the diverse group of isoprenoids (also called terpenes). The term isoprenoid indicates that these compounts can theoretically be decomposed into units of the compound isoprene (2-methylbuta-1,3-dien). A diterpene like steviol is composed of four isopren units. All diterpenes in cell metabolism are derived from geranylgeranyl pyrophosphate (which is a diterpene itself). The synthesis of steviol is the aim for our participation in the iGEM 2012 competition. At least we want to introduce the necessary genes into our host organism <I>Saccharomyces cerevisiae</I>. To reach this goal one part of the project is the increase of the mevalonate pathway activity.
</P><H2><SPAN class="mw-headline" id="The_steviol_precursor_metabolism_in_yeast">The steviol precursor metabolism in yeast</SPAN></H2><P>It is known from literature that yeast produces some mono-, sesqui- and diterpenes in very small amounts and so possesses a functional isoprenoid pathway. There are two known pathways for the production of isoprenyl pyrophosphate in nature. Isoprenyl pyrophosphate is converted into all the diverse isoprenoid compounts by condensing several units together. In bacteria and plastides isoprenyl pyrophosphate is produced via the methylerytritol pyrophosphate way (MEP) while it is produced via the mevalonate pathway (MVA) in eukaryotes. Many plants including stevia have crosslinks between the plastidic and the cytosolic pathway and produce the majority of isoprenoid compounds via the MEP pathway though they are eukaryotic organisms.
</P><P>In yeast only the mevalonate pathway is present. The compound chiefly produced by yeast (from the isoprenoid family) is squalene. This is a triterpene from which the sterols (like cholesterol and ergosterol) are derived. Those compounds are essential for propagation because they control the membrane fluidity. Yeast with a corrupted sterol production is not able to grow unless ergosterol is present in the medium. Geranylgeranyl pyrophosphate is only produced in small amounts since yeast only uses it for very special purposes like the modification of proteins. We suppose that a proper function of the steviol producing pathway in yeast will only be possible when the flow of the mevalonate pathway is redirected to geranylgeranyl pyrophosphate without disturbing the production of squalene.
</P><H2><SPAN class="mw-headline" id="The_early_gibberelline_pathway_for_the_production_of_steviol">The early gibberelline pathway for the production of steviol</SPAN></H2><P>The generation of steviol from geranylgeranyl pyrophosphate takes place via four reactions.
</P><P>The first two steps imply the double cyclization of geranylgeranyl pyrophosphate. In the first reaction <I>ent</I>-copalyl pyrophosphate is formed (induced by the protonation of a double bond, for that reason called proton-induced cyclization). The second reaction step leads to <I>ent</I>-kaurene (induced by dephosphorylation, which leaves a positive charge that acts as electrophile and attacks another double bond).
</P><P>The following two reactions imply the oxidation of <I>ent</I>-kaurene. The third reaction implies the oxidation of <I>ent</I>-kaurene to form <I>ent</I>-kaurenoic acid. It is divided into three steps (<I>ent</I>-kaurenol, <I>ent</I>-kaurenal, <I>ent</I>-kaurenoic acid). The last reaction leads to a hydroxygroup via oxidation at position 13 to form steviol. It is the key reaction of the steviol glycoside pathway which seperates it from the parallel gibberelline pathway. As far as it is known it only occurs in plants of the genus <I>Stevia</I> and in the blackberry species <I>Rubus suavissimus</I>.
</P><H1><SPAN class="mw-headline" id="DNA_constructs_for_realisation_of_our_idea">DNA constructs for realisation of our idea </SPAN></H1><H2><SPAN class="mw-headline" id="Mevalonate_Pathway_Overexpression">Mevalonate Pathway Overexpression</SPAN></H2><P>The vector for improving the MVA pathway flow contains three genes: A trunctated version of the HMG-CoA-Reductase, the native gene of the Farnesylpyrophosphate Synthase (ERG20) and the GGPP Synthase gene from <I>Sulfolobius acileratius</I>. The sources of the physical DNA were as following: </P><UL><LI><BIG> HMG-CoA-Reductase: </BIG> The sequence was obtained from the Saccharomyces Genome Database and modified. This version was send to a company for <I>de novo</I> synthesis.
</LI></UL><UL><LI><BIG> FPP-Synthase: </BIG> The gene was obtained from genomic DNA of the strain CEN.PK2. By using primers which contained overlaps with the pre- and suffix sequences it was amplified by PCR.
</LI></UL><TABLE width="100%" align="centre"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">FPP Synthase - Thanks to iGEM Darmstadt</DIV></DIV></DIV></TABLE><UL><LI><BIG> GGPP-Synthase </BIG>: The sequence was obtained from the Pubmed Database and modified. This version was send to a company for <I>de novo</I> synthesis.
</LI></UL><P>construction of the insert:
pHXT7 <I>HMG-CoA</I> tHXT7  pPFK1 <I>ERG20</I> tPFK2 pPGK1 <I>GGPPS</I> tCYC1</P><P>plasmid backbone with <I>URA3</I></P><H2><SPAN class="mw-headline" id="Steviol_Production">Steviol Production</SPAN></H2><P>The vector for steviol production also contains three genes: The bifunctional cyclase from <I>Gibberella fujikuroi</I> (CPS/KS), the <I>ent</I>-kaurene oxidase from <I>Gibberella fujikuroi</I> (KO) and the <I>ent</I>-kaurenoic acid hydroxylase from <I>Stevia rebaudiana</I> (KAH). The sources of the physical DNA were as following: </P><UL><LI><BIG> bifunctional cyclase: </BIG> The sequence was obtained from the Pubmed Database and modified. This version was send to a company for <I>de novo</I> synthesis.
</LI></UL><UL><LI><BIG><I>ent</I>-kaurene oxidase: </BIG> The sequence was obtained from the Pubmed Database and modified. This version was send to a company for <I>de novo</I> synthesis (4 x 500 bp for Gibson Assembly).
</LI></UL><UL><LI><BIG><I>ent</I>-kaurenoic acid hydroxylase: </BIG> The sequence was obtained from the Pubmed Database and modified. This version was send to a company for <I>de novo</I> synthesis (4 x 500 bp for Gibson Assembly).
</LI></UL><P>construction of the insert:
pHXT7 <I>CPS/KS</I> tTAL1  pTPI1 <I>KO</I> tPDC1 pPGI1 <I>KAH</I> tCYC1</P><P>plasmid backbone with <I>HIS3</I></P><H2><SPAN class="mw-headline" id="DNA_sources">DNA sources</SPAN></H2><TABLE class="wikitable zebra"><TBODY><TR><TH>Templates</TH><TH>Amplified DNA Fragments
</TH></TR><TR><TD>synthesized sequence of <I>HMG-CoA</I></TD><TD><I>HMG-CoA</I></TD></TR><TR><TD>synthesized sequence of <I>GGPPS</I></TD><TD><I>GGPPS</I></TD></TR><TR><TD>chromosomal DNA of CEN.PK2-1C</TD><TD><I>ERG20</I></TD></TR><TR><TD>synthesized sequence of <I>Cps/Ks</I></TD><TD><I>CPS/KS</I></TD></TR><TR><TD>synthesized sequence of <I>KO</I></TD><TD><I>ent</I>-kaurene oxidase
</TD></TR><TR><TD>synthesized sequence of <I>KAH</I></TD><TD><I>ent</I>-kaurenoic acid hydroxylase
</TD></TR><TR><TD>pUD8e (yeast shuttle plasmid)</TD><TD>pPFK1, pTPI1, tHXT7, tTAL1
</TD></TR><TR><TD>pUD22e (yeast shuttle plasmid)</TD><TD>pPGK1, pPGI1, tPFK2, tPDC1
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Sequence_Editing">Sequence Editing</SPAN></H2><P>The most of our genes were synthesized, because they contained several biobrick restriction sites. The relevant codons were replaced by different codons which encode for the same amino acid. Only <I>ERG20</I> has no biobrick restriction sites.</P><UL><LI><I>HMG-CoA</I></LI></UL><P>synthesized sequence 
5`: 45 bp overlap to pHXT7 and the prefix
3`: the suffix and 33 bp overlap to tHXT7</P><UL><LI><I>ERG20</I></LI></UL><P>gene from chromosomal DNA of CEN.PK2-1C
5`: 16 bp overlap to pPFK1 and the prefix (added via primer)
3`: the suffix and 14 bp overlap to tPFK2 (added via primer)</P><UL><LI><I>GGPPS</I></LI></UL><P>synthesized sequence
5`: 18 bp overlap to pPGK1 and the prefix
3`: the suffix and 45 bp overlap to tCYC1</P><UL><LI><I>CPS/KS</I></LI></UL><P>synthesized sequence
5': 45 bp overlap to PHXT7 and the prefix
3': the suffix and 17 bp overlap to tTAL1</P><UL><LI><I>KO</I></LI></UL><P>synthesized sequence (4 x 500 bp for Gibson Assembly)
5': 15 bp overlap to pTPI1 and the prefix
3': the suffix and 16 bp overlap to tPDC1</P><UL><LI><I>KAH</I></LI></UL><P>synthesized sequence (4 x 500 bp for Gibson Assembly)
5': 15 bp overlap to pPGI1 and the prefix
3': the suffix and 45 bp overlap to tCYC1</P><UL><LI>promoters and terminators</LI></UL><P>They were amplified from plasmids via PCR with designed primers. 
The primer contain about 20 to 30 bp of the promoter or terminator sequence for PCR and about 30 bp overlap to the neighbouring sequence for homologue recombination.
</P><H2><SPAN class="mw-headline" id="Gas_Chromatographic_Analysis_for_Examination_of_the_Product_content_of_the_cells"> Gas Chromatographic Analysis for Examination of the Product content of the cells</SPAN></H2><P>The metabolites of our modified pathways in particular GGPP and Steviol should be analyzed via Gas Chromatography - Mass Spectrometry. We used overnight cultures of yeast cells at OD600=2. The cells were harvested via centrifugation and washed with physiological NaCl solution. The washed cells were homogenized by addition of glass beads and disruption for 20 minutes. Afterwards hexane was added to the disrupted cells. The Analytes were extracted by shaking for 15 minutes. Afterwards the suspension was centrifuges untill a clear organic phase had established itself on the surface. This phase was pipetted into a reaction tube and concentrated to a volume of 80Âµl using a Speed-Vac. In a GC vial the samples were treated with a derivation agent (Trimethylchlorsilan, TMS) for better seperation of Alcohols on non-polar columns. We used a simple temperature program for the GC runs: The oven temperature was increased constantly from 150Â°C to 300Â°C with 70Â°C/min.
</P><P>Unfortunatly we were not able to proof our first intermediate GGPP. GGPP can not be analyzed directly (via GC), only the dephosphorylation product Geranylgeraniol can be analyzed. We abdicate on an active enzymatical digest of GGPP in our first trials. When we conduct further biochemical experiments we will try to proof GGPP both in wildtype and in transformant cells to compare the amounts.
</P><P>We poofed the general method to be functional because other diterpene compounts such as Farnesol and Squalene which are closely related to our product could be analyzed.
</P><H2><SPAN class="mw-headline" id="Gap_repair_Cloning_for_Vector_Assembly">Gap repair Cloning for Vector Assembly</SPAN></H2><P>Gap repair means the homologue recombination in yeast in order to assemble desired DNA fragments. It is easier and more effective than cloning via restriction and ligation. We used it to assemble the mevalonate overexpression plasmid and the steviol synthesis plasmid. All genes, promoters and terminators could be amplified except of the fragments for Gibson Assembly of <I>KAH</I>. Thus the mevalonate overexpression plasmid was assembled and transformed into yeast. To verify the correct assembly of the plasmid we did two control restrictions (see &quot;Notebook&quot;). Because we were not able to assemble <I>KAH</I> the gap repair of the steviol synthesis plasmid did not work. We also tried the gap repair with the four 500 bp fragments of <I>KAH</I> but the overlapping sequences are only 30 bp and not 40 bp which is needed for homologue recombination.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Frankfurt/Project">http://2012.igem.org/Team:Frankfurt/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Frankfurt/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Frankfurt/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Frankfurt/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Frankfurt/Project&amp;oldid=238930" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>