<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Frankfurt_New_Yeast_RFC"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Frankfurt/New_Yeast_RFC">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Frankfurt/New_Yeast_RFC&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Frankfurt/New_Yeast_RFC&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Frankfurt/New_Yeast_RFC&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Frankfurt/New_Yeast_RFC" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Frankfurt/New Yeast RFC</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE> Team: iGEM Frankfurt - 2012.igem.org </TITLE></P><DIV id="wrap"><DIV id="left"><UL id="menu"><LI><A href="https://2012.igem.org/Team:Frankfurt">Home</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/Team">Team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2012&amp;team_name=Frankfurt">Official Profile</A></LI></UL><LI><A href="https://2012.igem.org/Team:Frankfurt/Project">Project</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/Organisms">Organisms</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/New_Yeast_RFC">Yeast RFC</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/Parts">Parts</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/Notebook">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/Notebook">Labwork</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/Protocol">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/Safety">Safety</A></LI><LI><A>Human practices</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/Ethics">Ethics</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/Events">Events</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/BioSecurity">BioSecurity</A></LI><LI><A href="https://2012.igem.org/Team:Frankfurt/Attributions">Attributions</A></LI><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#The_benefit_of_vector_assembly_in_yeast"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">The benefit of vector assembly in yeast</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Introduction"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Homologue_Recombination_System_of_Yeast"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Homologue Recombination System of Yeast</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Design_of_DNA_fragments_for_gap_repair_cloning"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Design of DNA fragments for gap repair cloning</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Example_from_our_project"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Example from our project</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Yeast_BioBrick_Assembly_-_Gap_repair_cloning_for_iGEM"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Yeast BioBrick Assembly - Gap repair cloning for iGEM</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Assembly_of_Yeast_Expression_Vector_Using_YBA_Standard"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Assembly of Yeast Expression Vector Using YBA Standard</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Nomenclature_and_Design_of_Standardized_YBA_Primers"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Nomenclature and Design of Standardized YBA Primers</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Methods"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">Methods</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Assembly_of_E.coli_Vectors"><SPAN class="tocnumber">1.9</SPAN><SPAN class="toctext">Assembly of E.coli Vectors</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Full_Version_of_YBA_standard"><SPAN class="tocnumber">1.10</SPAN><SPAN class="toctext">Full Version of YBA standard</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="The_benefit_of_vector_assembly_in_yeast">The benefit of vector assembly in yeast</SPAN></H1><H2><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H2><P>iGEM Team Frankfurt 2012 successfully used a relatively new methode for vector assembly in yeast called <I>Gap Repair Cloning</I>. It is a more and more established methode for efficient, fast and error-free construction of plasmids based on the homologous recombination system of <I>Saccharomyces cerevisiae</I> (common yeast). Naturally yeast uses this process to repair DNA double strand breaks which are one of the most dangerous and life-threatening damages of the DNA for a cell. Therefor this eucaryotic microorganism has developed a few enzymes which have the ability to repair a broken DNA double strand by pairing it with a very similiar DNA region (typically on the homologous chromosome).
Using gap repair cloning a series of linear, successive DNA fragments with homologous overlaps to the respectively following fragment can be transformed in only one step into a yeast cell. After that the micoorganism recombines all fragments in the predetermined, specific order to the final targeting vector. The advantage is that up to eighteen and more successive DNA fragments can be assembled in a single transformation. Also only one restriction enzyme for linearization of the plasmid is needed. 
For these reasons iGEM Team Frankfurt thought that gap repair cloning is usefull tool for next iGEM generations. Therefore we developed a standardized methode that describes a new way of assembling BioBrick devices in a desired order to a targeting plasmid using homologue recombination system of yeast. It is called <I>Yeast BioBrick Assembly (YBA)</I>. YBA standard only needs one restriction enzyme and a standardized selection of primers and promotors/termintors. It is a continuation of the BioBrick standard and compartible with all BBF <A href="http://tools.ietf.org/html/rfc10" class="external mw-magiclink-rfc">RFC 10</A> parts. Additionally it can be adapted by specific primer design to all other BioBrick standards. In the following we focuse on assembly of yeast expression vectors by using YBA methode. However it also can be used for E.coli vector design or assembly.
</P><H2><SPAN class="mw-headline" id="Homologue_Recombination_System_of_Yeast">Homologue Recombination System of Yeast</SPAN></H2><P>There are many endogenous and exogenous factors (for example reactive oxygen-species, ionizing radiation, chemicals and failing of DNA binding enzymes (e.g. collapsed replication forks)) which causes DNA double strand breaks. For the cell this is the most dangerous DNA damage because even if it occurs in rather unimportant regions the cell will not survive the next cell cycle. That's the reason why yeast possesses highly active enzymes which have the ability to repair a broken double strand by pairing it with a very similiar DNA region (typically on the homologous chromosome). This process is called homologous recombination. Using the gap repair method this natural process can be exploited for the construction of large cloning vectors in yeast.
</P><H2><SPAN class="mw-headline" id="Design_of_DNA_fragments_for_gap_repair_cloning">Design of DNA fragments for gap repair cloning</SPAN></H2>The idea of the method is to transform a series of linear, successive DNA fragments into one yeast cell. The linear fragments have open blunt ends like they occur after a double strand break. If a homologous sequence is available it will be treated like a genomic double strand break and homologous recombination takes place. When the successive DNA fragments are designed in a specific way which includes large sequence overlaps (overall app. 40 bp) to the respectively following fragment yeast will recombinate them together. <P>For the formation of a cloning vector the first fragment is a yeast-<I>E.coli</I> shuttle plasmid which is linearized by an appropriate restriction digest. A shuttle plasmid is a plasmid which is stable both in yeast and in <I>Escherichia coli</I>. The first fragment of the insert has to possess an homologous overlap to both the wished insertion site on the plasmid and to the beginning of the second fragment. The end of the second fragment has to possess an overlap to the beginning of the third one and so on. At least the end of the last fragment of the insert again has to possess an overlap homologous to the second insertion site on the plasmid. 
At the lab of our instructor up to eighteen single fragments were assembled in a single transformation. Another advantage of the method is that no scars are left between the inserted fragments. Assembly of fragments to joint genes is possible. Restriction enzymes only have to be used once for linearization of the shuttle plasmid.
</P><H2><SPAN class="mw-headline" id="Example_from_our_project">Example from our project</SPAN></H2><TABLE width="100%" align="center"><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">The overlap sequences at the beginning and the end of each fragment are 20 bp long and homologous to the respectively previous and subsequent fragment. The overlaps to promoter and terminator on the plasmid are longer (40 bp) because the plasmid does not contain any overlaps to the first and last insert fragment. Thus the homologous region between every fragmentpair is always 40 pb long.</DIV></DIV></DIV><P>In our project we have constructed a plasmid for overexpression of three genes of the <A href="/Team:Frankfurt/Project" title="Team:Frankfurt/Project">Mevalonate</A> pathway. Therefore we used synthesized fragments of the mentioned three genes, two promoters, two terminators (both from yeast) and a yeast expression plasmid which already contained one promoter, one terminator and a gene for uracil synthesis. Moreover we used a mutant strain where this uracil gene is deleted. So we needed the uracil gene on the plasmid as a marker for selection of the transformants. Via PCR we assembled appropriate homologous overlaps to our fragments using primers containing these overlaps. By that we were able to arrange our fragments in the order shown in the picture on the right. 
The DNA of all fragments is transformed together into competent yeast cells. The transformants were selected on synthetic medium lacking from uracil (only transformants should be able to grow). After preparing the plasmid from the transfomed yeast clones it can be transformed into <I>Escherichia coli</I> and gained from there in large amounts for further use.
The DNA sequence of the complete expression vectore is shown in the following file:
<A href="/wiki/images/a/ae/DNA_Sequence_of_Plasmid_for_Overexpression_of_Mevalonat_Pathway.txt" class="internal" title="DNA Sequence of Plasmid for Overexpression of Mevalonat Pathway.txt">Media:DNA_Sequence_of_Plasmid_for_Overexpression_of_Mevalonat_Pathway.txtâ</A>.
</P></TABLE><H2><SPAN class="mw-headline" id="Yeast_BioBrick_Assembly_-_Gap_repair_cloning_for_iGEM">Yeast BioBrick Assembly - Gap repair cloning for iGEM</SPAN></H2><TABLE width="100%" align="center"><DIV class="thumb tright"><DIV class="thumbinner" style="width:462px;"><DIV class="thumbcaption">Example of BioBrick Assembly via gap repair cloning with YBA standard. First primers are assembled to the respective gene via PCR. The primer overlap to suffix or prefix is about 20 bp. Now the assembled genes have homologous overlaps to the respective promoters and terminators of about 40 bp length. In a yeast transformation the shuffle plasmid, a ligated terminator-promoter part and the assembled genes are put into the yeast cells. They put all parts together via homologous recombination to form the complete vector.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Variable assembly of terminator-promoter parts via gap repair cloning.</DIV></DIV></DIV><P>The BioBrick cloning standards used in the Parts Registry and in the iGEM competition are based on restriction digest and ligation. One of the main advantages of the gap repair method is to avoid this. Moreover it leaves no scars between the assembled fragments like restriction digest and religation does. Another advantage of gap repair cloning is itÂ´s heightened time effiency when a large amount of fragments shall be assembled. Furthermore the expensive use of restriction and ligation enzymes can be reduced significantly. For these reasons gap repair cloning promises to be a useful tool for future iGEM teams. The problem is that the common Biobrick standards are useless by now with regard to gap repair cloning. The idea of YBA is now to design a new standard for assembly of yeast vectors based on standardized PCR primers. YBA is compartible to common BioBrick standards and allows gap repair cloning.

Although the restriction sites of the Biobrick pre- and suffix are unimportant in our context, the biobricking of genes leads to possiblity to amplify all Biobricks with the same prefix and suffix type (in our case we focus on BBF <A href="http://tools.ietf.org/html/rfc10" class="external mw-magiclink-rfc">RFC 10</A>) with suitable PCR primers. There are already some PCR primers in the Parts Registry which anneal at the pre- or suffix sequence. By adding an specific non annealing sequence to the primers a desired overlap to other fragments can be produced on every Biobrick device. These primers which consist of an annealing sequenz to either prefix or suffix and an specific overlap are the stardardized PCR primers for YBA method. 

The further idea would be to create DNA fragments suitable for gap repair cloning. It would be great if the assembly and cloning of genes in yeast become simpler than the cloning by restriction and ligation. In the following we want to outline the basic idea. Any yeast expression plasmid has a similiar ordered insert region. Depending on the number of genes on the final plasmid the insert region consists of several repeats of the general scheme promoter-gene-terminator. Gap repair suitable DNA can simply be created by choosing promoters and terminators of yeast and the desired genes from the parts registry. Now every gen has to be amplified with YBA standard primers which contain homologue DNA overlaps to the end of a chosen promoter and the beginning of a proper terminator. With such PCR products it is possible to do the homologue recombination in yeast like it is shown on the first picture on the right. 

Since any Biobrick (from the same standard) can be amplified with the same primers it is sufficient to construct one standard primer for every promoter, overlapping with the end of the promoter, and one for every terminator, overlapping with the beginning of the terminator. We designed one standard primer compartible to BioBrick <A href="http://tools.ietf.org/html/rfc10" class="external mw-magiclink-rfc">RFC 10</A> for any of the yeast promoters and terminators we used in our project. Of course the annealing sequence of every standard primer can be adapted to all other BioBrick RFCs.

The connecting terminator-promoter fragments between the BioBrick devices shall not have BioBrick prefix or suffix at their ends because this could lead to incorrect assembly of targeting vector and reduces the efficiency of gap repair cloning.

After all there are two ways of connecting the terminator of one gene with the promoter of the following gene. The first one is to design specific combinations of ligated terminator-promoter parts (shown on the first picture). At the vector assembly you just have to transform the linearized plasmid, the amplified genes with homologue overlaps and the terminator-promoter parts to get the targeting vector. The advantage is that you need a lower number of DNA fragments which increases the efficiency of gap repair cloning. We designed a few of this terminator-promoter parts for the registry. The disadvantages are first that you are not able to choose all possible combinations because they are too many. Secondly you have to amplify the parts out of the BioBrick plasmid anyway. Therefore you must use a forward primer annealing behind the prefix and a reverse primer anneling in front of the suffix. This leads to the second way.

The second way allows much more possibilities of combining terminators and promoters (shown on the second picture). Here you can choose BioBricks of single terminators and promotors from the parts registry. First you amplify the terminator with two specific primers annealing behind the prefix and previous to the suffix. The next step is amplification of a suitable promoter which comes after this terminator. There you take one primer annealing at the prefix. This primer has an additional homologue overlap to the previous terminator to connect both at gap repair cloning. The second primer is annealing in front of the suffix. 

In the end you transform all amplified genes, promotors and terminators and the linearized plasmid into yeast cells to get the targeting plasmid.

Whenever you want to assemble a high number of BioBrick devices in a specific order to a targeting vector you can use standardized YBA methode for an easy, effective and cheap assembly. 
</P></TABLE><H2><SPAN class="mw-headline" id="Assembly_of_Yeast_Expression_Vector_Using_YBA_Standard">Assembly of Yeast Expression Vector Using YBA Standard</SPAN></H2><TABLE width="100%" align="center">
The targeting yeast expression vector consists of a shuffle plasmid (containing selection markers for yeast and E.coli, one yeast promoter, one yeast terminator) and a specific number of genes, each possessing one promoter and one terminator. To assemble the final vector we use YBA method. 

First you have to choose a suitable shuffle plasmid for your project. After that you need one specific restriction enzyme to linearize the shuffle plasmid. It has to cut specifically at the MCS (Multiple Cloning Site). 

Now you have to look for BioBrick devices of the genes you want to express in the parts registry and arrange them in the desired order. Afterwards choose an appropriate number of yeast promoters an terminators for every gene. Fit the respective promoters and terminators between the genes. Pay attention that you already have one promotor and terminator on your shuffle plasmid. At the end the insert region of your vector construct should consist of suitable numbers of the general scheme promoter-gene-terminator. You can decide if you want to have BioBricks of specific combinations of already ligated terminator-promoter parts which you can transform into the yeast cell as one fragment or if you want to amplify your own combination of terminator-promoter parts. Than you have to take single terminator and promoter BioBricks. 

To use YBA method you have to amplify the genes, promotors and terminators with suitable primers. First you have to look for the right BioBrick standard of your primers. The most common BioBrick assembly standard which is used by most iGEM teams and most BioBricks in the parts registry is BBF <A href="http://tools.ietf.org/html/rfc10" class="external mw-magiclink-rfc">RFC 10</A>. The BioBrick standard of your genes, promoters and terminators has to be suitable to your choosen primer standard. 

For every gene amplification you have to choose the appendant forward primer with a homologue overlap to the promoter in front of the gene and the suitable reverse primer with a homologue overlap to the terminator at the end of the gene. 

If you want to amplify your own combination of terminator-promoter parts you also have to look for the suitable primers to the respective promoters or terminators. The terminator has to be ampified with primers annealing behind the prefix and in front of the suffix of the BioBrick. For promoter ampification you need one primer annealing to the prefix and having a homologue overlap to the suitable terminator which is in front of this promoter. The second primer has to anneal in front of the suffix of this promoter.

If you have choosen BioBricks of already ligated terminator-promoter parts you need to amplify them with primers annealing at the beginning of the terminator and the end of the promoter. This amplification is just to get the terminator-promoter part out of the BioBrick plasmid and to increase the amount of terminator-promoter fragments.

After you have choosen the right primers (pay attention that you need two primers (forward- and reverse-primer) for every fragment) for your fragments you have to amplify all of them (genes, promoters, terminators) in a PCR reaction. 

Then you transform all PCR products and the linearized plasmid into yeast cells and select the transformants on LB medium lacking of one specific substance which is your yeast selection marker on the shuffle plasmid. 

With positive clones you are able to inoculate yeast cultures. After plasmid preparation you are able to transform the isolated targeting vector into E.coli to increase amount of it. You should control if your targeting vector possesses the complete insert via control restriction digestion and agarose gel electrophoresis.
</TABLE><H2><SPAN class="mw-headline" id="Nomenclature_and_Design_of_Standardized_YBA_Primers">Nomenclature and Design of Standardized YBA Primers</SPAN></H2><TABLE width="100%" align="center">
All parts are named by a specific nomenclature to indentify the right parts for the respective amplification. The following legend shows how the parts are named. For every primer is said what kind of amplification is done with it, where it is annealing and what kind of homologue overlap it has.<TABLE class="wikitable zebra"><TBODY><TR><TH>Abbreviation</TH><TH>Meaning
</TH></TR><TR><TD>p</TD><TD>Primer
</TD></TR><TR><TD>PG</TD><TD>Primer for Gene Amplification
</TD></TR><TR><TD>PTP</TD><TD>Primer for Terminator-Promoter Part Amplification
</TD></TR><TR><TD>PP</TD><TD>Primer for Promoter Amplification
</TD></TR><TR><TD>PT</TD><TD>Primer for Terminator Amplification
</TD></TR><TR><TD>fw</TD><TD>forward Primer
</TD></TR><TR><TD>rev</TD><TD>reverse Primer
</TD></TR><TR><TD>p</TD><TD>Promoter
</TD></TR><TR><TD>t</TD><TD>Terminator
</TD></TR><TR><TD>Pre10</TD><TD>Prefix BioBrick <A href="http://tools.ietf.org/html/rfc10" class="external mw-magiclink-rfc">RFC 10</A></TD></TR><TR><TD>Suf10</TD><TD>Suffix BioBrick <A href="http://tools.ietf.org/html/rfc10" class="external mw-magiclink-rfc">RFC 10</A></TD></TR><TR><TD>a</TD><TD>annealing at
</TD></TR><TR><TD>h</TD><TD>homologue to
</TD></TR></TBODY></TABLE><P>Example: fw_PG_aPre10_hpXXX is the name of a forward primer for gene amplification annealing at the BioBrick prefix (BioBrick standard <A href="http://tools.ietf.org/html/rfc10" class="external mw-magiclink-rfc">RFC 10</A>) having a homologue overlap to a specific promoter.

Of course it is possible to design new YBA standard primers for other promoters/terminators of yeast but also for E.coli promoters/terminators. Note that all of these primers must have a specific annealing temperatur in the field of 58 +/- 1Â°C to combine different primers for every possible PCR amplification.
</P><P>All primers for gene amplification must anneal at prefix or suffix of the BioBrick device. The annealing sequence must be compartible to at least BBF <A href="http://tools.ietf.org/html/rfc10" class="external mw-magiclink-rfc">RFC 10</A> standard. Additionally it can be also compartible to other BBF RFC standards. The homologue overlap to the new promoters/terminators must be not less than 40 bp. 
</P><P>Primers for amplification of singel promoters or terminators as well as for ligated terminator-promoter parts must anneal behind the prefix or in front of the suffix, except of the forward primer of single promoter parts which is intended to be the only one having an homologue overlap of 40 bp to previous terminators. The homologue overlap of this primer is essential for possibility of combining own terminator-promoter parts. 
</P><P>Pay attention that you must design two primers (forward and reverse) for every PCR reaction. Each primer must have a free 3'-ending in direction of the DNA fragment so the DNA-Polymerase can synthesise the new strand in 5' to 3' direction. The consequence of this is that the forward primer must anneal to the 3'-5' singel strand DNA and the reverse primer must anneal to the 5'-3' single strand DNA of the template.
</P><P>The standardization of primers from YBA standard is only depending on the annealing sequence which must be homologue to the respective BioBrick standard prefix or suffix. The fact that it is insignificant what kind of homologue overlaps the primers have offers the option to choose also other fragments (not only promoters/terminators) incorporating between the BioBrick devices. For assembly of such fragments the suitable homologue overlaps have to be designed onto the primers.
</P></TABLE><H2><SPAN class="mw-headline" id="Methods">Methods</SPAN></H2><P>The main methods you have to use for YBA standard are: 
â¢ PCR
â¢ Restriction Digestion for Linearization of a Plasmid 
â¢ Control Restriction Digestion of Plasmid with Insert 
â¢ Yeast Transformation 
â¢ <I>E.coli</I> Transformation </P><H2><SPAN class="mw-headline" id="Assembly_of_E.coli_Vectors">Assembly of <I>E.coli</I> Vectors</SPAN></H2><P>YBA method can also be used for assembly of <I>E.coli</I> vectors. Therefore you need a special shuffle plasmid which has an <I>E.coli</I> promoter and terminator next to the MCS (multiple cloning site) and is also stable in yeast. After that you can design specific primers annealing at the prefix and suffix of your BioBrick device to amplify homologue overlaps to promoters and terminators like it is shown above for yeast expression vectors. Even though <I>E.coli</I> as a bacteria possesses polycistronic mRNA you have the possibility to regulate every gene by it's own if you assemble additional promoters and terminators between the genes. But you are also able to incorporate other parts, for example other regulatory fragments, between the genes by designing suitable homologue overlaps to these fragments at the primers. 

Another possibility is to create primers for one BioBrick device with prefix/suffix annealing sequence and an additional homologue overlap to the coding sequence of a successive BioBrick device. It is important that every second BioBrick device of the insert region must be amplified with specific primers in such a way that only the coding sequence remains. So you can assemble a vector with only one promoter and terminator and a few aligned BioBricks as insert.

After amplifying all fragments in an PCR reaction you transform them into yeast cells. Then you just have to isolate the complete targeting plasmid from yeast, transform it into E.coli and check after another plasmid preparation and a control restriction digestion if it possesses the complete insert. 

Up to this point all mentioned primers are designed as standard parts of YBA standard annealing at prefix or suffix of a BioBrick device. More interesting for E.coli vector assembly are the possiblities you have if you are independent of any standard. Then gap repair cloning and homologue recombination system of yeast gets a usefull tool for creating new biobrick devices by combining parts. For example you have the possibility to amplify a RBS (ribosom binding site) to a BioBrick device containing a pure gene. Therefore you have to design a suitable primer paar of respectively app. 20 bp length which must anneale behind of the prefix and in front of the suffix to amplify the pure gene without any BioBrick endings. After that you creat a second primer pair. The forward primer has to anneal at the beginning of the gene having an overlap containing the Shine-Dalgarno-Sequenz/RBS and maybe additionally an homologue overlap to a previous part (e.g. another gene). It is important that you have the pure gene because RBS must be app. 6-7 bp in front of the beginning of the gene. The reverse primer must anneal at the end of the gene and could also have a homologue overlap to a following part. With homologue recombination you can assemble all of your amplified fragments to a completely new BioBrick device by only one yeast transformation. 

Moreover at all times you want to assemble a higher number of fragments efficient, fast and with the lowest possible costs homologue recombination in yeast is the better methode compared to restriction digestion and ligation. 
</P><H2><SPAN class="mw-headline" id="Full_Version_of_YBA_standard">Full Version of YBA standard</SPAN></H2><P>In the following file the YBA standard is described completely. It also contains sequences of yeast promoters/terminators and suitable YBA standard primers. Additionally all methodes are explained in detail.
</P><P><A href="/wiki/images/3/32/RFC_final.pdf" class="internal" title="RFC final.pdf">Media:RFC_final.pdf</A></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Frankfurt/New_Yeast_RFC">http://2012.igem.org/Team:Frankfurt/New_Yeast_RFC</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Frankfurt/New_Yeast_RFC" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Frankfurt/New_Yeast_RFC" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Frankfurt/New_Yeast_RFC&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Frankfurt/New_Yeast_RFC&amp;oldid=224126" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>