<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NYMU-Taipei_Project_Guanidine"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:NYMU-Taipei/Project/Guanidine">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NYMU-Taipei/Project/Guanidine&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NYMU-Taipei/Project/Guanidine&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NYMU-Taipei/Project/Guanidine&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NYMU-Taipei/Project/Guanidine" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NYMU-Taipei/Project/Guanidine</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><TABLE id="nav" style="border:1px solid gray;width:965px"><TBODY><TR><TD class="right"><A href="/Team:NYMU-Taipei" title="Team:NYMU-Taipei"> Home</A></TD><TD class="left" style="width:120px;"><A href="/Team:NYMU-Taipei/Project" title="Team:NYMU-Taipei/Project"> Project Overview:</A></TD><TD class="left"><A href="/Team:NYMU-Taipei/Project/pH_Sensor" title="Team:NYMU-Taipei/Project/pH Sensor"> pH Sensor</A></TD><TD class="left"><A href="/Team:NYMU-Taipei/Project/Attachment" title="Team:NYMU-Taipei/Project/Attachment"> Attachment</A></TD><TD class="left" style="width:140px;"><A href="/Team:NYMU-Taipei/Project/Time_Regulation" title="Team:NYMU-Taipei/Project/Time Regulation"> Time Regulation</A></TD><TD class="both" style="width:120px;"><A href="/Team:NYMU-Taipei/Project/Waste_Removal" title="Team:NYMU-Taipei/Project/Waste Removal"> Waste Removal</A></TD><TD class="both" style="width:180px;"><A href="/Team:NYMU-Taipei/Experiments" title="Team:NYMU-Taipei/Experiments"> Experiments and Parts</A></TD><TD class="left"><A href="/Team:NYMU-Taipei/Team" title="Team:NYMU-Taipei/Team"> About Us</A></TD></TR></TBODY></TABLE><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Motivation"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Motivation</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Literature"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Literature</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Background"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Background</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Strategy"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Strategy</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Experiment"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Experiment</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#CLONING"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">CLONING</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#HPLC"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">HPLC</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Motivation"> Motivation </SPAN></H2><P>Renal failure can make one unable to regulate water, electrolyte balances, and metabolic waste. If not treated, lethargy, weakness, shortness of breath, and generalized swelling may occur.
</P><H2><SPAN class="mw-headline" id="Literature"> Literature </SPAN></H2><P><A href="http://www.clinchem.org/cgi/reprint/25/7/1343.pdf" class="external text" rel="nofollow">Proposed Mechanism for Urea Nitrogen Re-Utilization: Relationship between Urea and proposed Guanidine cycles</A></P><P>
1. Ramesh Kekuda, Puttur D. Prasad, Xiang Wu, Haiping Wang, You-Jun Fei, Frederick H. Leibach, and Vadivel Ganapathy; Cloning and Functional Characterization of a Potential-   sensitive, Polyspecific Organic Cation Transporter (OCT3) Most Abundantly Expressed in Placenta; THE JOURNAL OF BIOLOGICAL CHEMISTRY (1998).
</P><P>2. Kazumasa Aoyagi; Significance of reactive oxygen in kidney disease elucidated by uremic toxins; J Artif Organs (2001).
</P><P>3. Samuel Natelson, Hsiu-Yu Tseng, and John E. Sherwin; On the Biosynthesis of Guanidinosuccinate; CLIN. CHEM (1978).
</P><P>4. Jinshu Qiu, Hans Lee, Chengfeng Zhou; Analysis of guanidine in high salt and protein matrices by cation-exchange chromatography and UV detection; Journal of Chromatography A (2005).
</P><P>5. Toshihiko Hanai; Separation of polar compounds using carbon columns; Journal of Chromatography A (2003).
</P><H2><SPAN class="mw-headline" id="Background"> Background </SPAN></H2><P>Methylguanidine, guanidinosuccinic acid (GSA) and guanidinoacetic acid are the three compounds that provide alternative pathways for the excretion of nitrogen in patients with renal failure (J. Sawynok et al., 2007).
GSA was first detected in the urine of uremic patients. Its excretion in the urine is accompanied by a marked decrease in the production guanidinoacetate. This is because the formation of GSA and guanidinoacetate are alternative metabolites of aspartate, which condenses with ureidohomoserine to form canavaninosuccinate in a transamidination cycle (Samuel Natelson et al., 1978).</P><PRE>J. Sawynok et al., 2007   </PRE><P>Toxicity: GSA (N-amidino-Laspartate) has been identified as one of the major causes of bleeding tendency, a Na-K ATPase inhibitor, and the cause of convulsions in end-stage renal disease. GSA was found to be an activator of the N-methyl-D-aspartate (NMDA) receptor that generates NO and plays an important role in nerve development and the death of neurons. In peripheral nerves, it functions in pain and itching sensations.
</P><H2><SPAN class="mw-headline" id="Strategy"> Strategy </SPAN></H2><P>In the renal proximal tubular epithelial cells, the cationic xenobiotics enter the cell across the basolateral membrane and exit the cell across the brush border membrane.
A concerted action of the potential-sensitive transport system in the basolateral membrane and the H1-dependent transport system in the brush border membrane effectively
mediates the elimination of several xenobiotics by the kidney.
These transporters belong to the organic cation transporter (OCT) family, which contains OCT1, OCT2, OCT3, NKT, NLT, RST, and OCTN1. Compared with other transpoters, OCT3 is not only a transmembrane protein but also functions as an channel for anti-transporting cations, such as GSA and sodium ions.
</P><P>Initiated by pNhaA and promoted by urea sensors, the OCT3 homolog transporter from <I>E. coli</I> K-12 is aimed to remove guanidino compounds when the environmental pH is allowed and the urea concentration reaches the optimal state.
<A href="http://www.vitrax.com/products/catalog.php" class="external free" rel="nofollow">http://www.vitrax.com/products/catalog.php</A></P><H2><SPAN class="mw-headline" id="Experiment"> Experiment </SPAN></H2><H4><SPAN class="mw-headline" id="CLONING"> CLONING </SPAN></H4><TABLE border="1"><TBODY><TR><TD><UL><LI>Primers are designed by Jesse, Blent, and Yuan-Hao (sequences not shown)
</LI><LI>Human OCT3 gene contains 2 PstI restriction enzyme cutting sites.
</LI><LI>This figure shows how to mutate the 2 sites by PCR; the inner primers were designed to do point mutation in nucleic acid level without changing the amino acid sequence.
</LI></UL></TD></TR><TR><TD><UL><LI>09/10, Yuan-Hao &amp; Ming-Han
</LI><LI>Materials and methods: yaaU in K-12, human OCT3, primers and enzymes.
</LI><LI>(Gene yaaU is the homologue of human OCT3, which has the potential to transport GSA into the membrane.)    
</LI><LI><I>PCR</I></LI><LI>Through this gel picture, we can differentiate each sample with 100 bp markers. Unfortunately, only yaaU has been amplified. This gradient PCR was done by setting temperatures from 53 Celsius degrees to 59 Celsius degrees.
</LI></UL></TD></TR><TR><TD><UL><LI>09/11, Yuan-Hao
</LI><LI>This is a gel loaded with yaaU and marker; yaaU could be extracted. 
</LI></UL></TD></TR><TR><TD><UL><LI>09/12, Yuan-Hao 
</LI><LI>Digested pLac-ECFP was analyzed with control (the non-digested) by agarose gel. 
</LI></UL></TD></TR><TR><TD><UL><LI> 09/12, Ming-Han &amp; Yuan-Hao
</LI><LI> Hu_OCT3(right seg.)and digested pLac-ECFP were electrophoresized in 1.5% gel and extracted. 
</LI></UL></TD></TR><TR><TD><UL><LI> 09/12, Yuan-Hao, <I>et al.</I></LI><LI> PCR using the former PCR products (09/10) as templates. Since middle and right segments were not shown, the preciseness of the applied primers should be concerned. 
</LI></UL></TD></TR><TR><TD><UL><LI> 09/12, Ming-Han
</LI><LI> Ligation and/or transformation of yaaU + pLac-ECFP failed
</LI><LI> Discussion see <A href="/Team:NYMU-Taipei/Project/Urea" title="Team:NYMU-Taipei/Project/Urea">Team:NYMU-Taipei/Project/Urea</A>/Experiments/0912 
</LI></UL></TD></TR><TR><TD><UL><LI> 09/18, a failure with transformation of pLac-ECFP-yaaU and confirmation of the clone OCT3. 
</LI><LI> Being tested with PstI to validate two restriction enzyme sites of human OCT3.
</LI><LI> Although the PstI sites existed, the number of them are still unsure (The leftmost lane: 100 bp marker, the middle left: non-digested OCT3, the middle right: digested OCT3, the righmost: 1 kb marker). 
</LI></UL></TD></TR><TR><TD><UL><LI> 09/19, trying to perform touchdown PCR from 55 Celsius degrees to 42 Celsius degrees with the primer combinations of (F_out+ R_out), (F_out+ R_in2) and (F_in1+ R_out).  
</LI><LI> 09/23, confirmed with gel electrophoresis, PCR products and 100 bp marker were loaded in the sequence from the left to the right: 100 marker, four supposed 1500 bp PCR products from (F_out+ R_out), four supposed 1200 bp PCR products from (F_out+ R_in2), four supposed 300 bp PCR products from(F_in1+ R_out) and 100 marker.    
</LI></UL></TD></TR><TR><TD><UL><LI> 09/24, trying to perform gradient PCR in a small temperature range from 50 Celsius degrees to 51 Celsius degrees and the temperature 60 Celsius degrees with yaaU primers.
</LI><LI> 09/25, confirmed with the gel elecrophoresis, PCR products and 1 kb marker were loaded in the sequence from the left to the right: 1 kb marker, five Taq polymerized products, the other 1 kb marker and five pfu polymerized PCR products.  
</LI></UL></TD></TR><TR><TD><UL><LI> 10/1, using 1% agarose gel to validate the colony PCR products. From the very left to the righmost lane: 1 kb Marker, v1, v2, y1, y2, y3, y4, y5, y6, y7, y8, y9, and y10 (v= plasmid R0100, y= supposed yaaU construct). 
</LI><LI> 10/2, the construct (pLac-yaaU, which contains {1500+200+8+238} base pairs) was invalid due to the wrong sizes of the products.
</LI></UL></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="HPLC"> HPLC </SPAN></H4><UL><LI><B>Materials and methods</B></LI><LI>UV (215 nm) and fluorescence (ex: 305nm, em: 395 nm)
</LI><LI>Sodium citrate, LB, TSKgel column G3000PW and TSKgel column ODS-120T 
</LI><LI>GSA (stock concentration= 10 mg/ml) serial dilution factor: 2
</LI></UL><UL><LI><B>Background standard curve</B></LI><LI>Total volumes are 1000 ml, concentrations are set to 5, 2.5, 1.25, 0.625, 0.3125 mg/ml (the supposed minimum detectable conc.= 10 mg/L)
</LI><LI>Prepare different GSA contained LB, and quantify each of them after incubation with &quot;natural&quot; <I>E. coli</I> by HPLC equipment.
</LI></UL><UL><LI><B>OCT-3 contained standard curve</B></LI><LI>Choose a suitable GSA concentration to test with engineered <I>E. coli</I>.
</LI><LI>Be sure to add GSA in different time points (O.D.600=0.5). The presumed incubation times are 0, 5, 10, 30, 60 and 120 min.
</LI></UL><DIV id="bottomlinkbox"><A id="bottomlink" href="https://2008.igem.org/Team:NYMU-Taipei">NYMU-Taipei iGEM 2008 Team</A></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:NYMU-Taipei/Project/Guanidine">http://2008.igem.org/Team:NYMU-Taipei/Project/Guanidine</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NYMU-Taipei/Project/Guanidine" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NYMU-Taipei/Project/Guanidine" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NYMU-Taipei/Project/Guanidine&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NYMU-Taipei/Project/Guanidine&amp;oldid=92171" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>