<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NYMU-Taipei_Project_Attachment"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:NYMU-Taipei/Project/Attachment">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NYMU-Taipei/Project/Attachment&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NYMU-Taipei/Project/Attachment&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NYMU-Taipei/Project/Attachment&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NYMU-Taipei/Project/Attachment" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NYMU-Taipei/Project/Attachment</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><TABLE id="nav" style="border:1px solid gray;width:965px"><TBODY><TR><TD class="right"><A href="/Team:NYMU-Taipei" title="Team:NYMU-Taipei"> Home</A></TD><TD class="left" style="width:120px;"><A href="/Team:NYMU-Taipei/Project" title="Team:NYMU-Taipei/Project"> Project Overview:</A></TD><TD class="left"><A href="/Team:NYMU-Taipei/Project/pH_Sensor" title="Team:NYMU-Taipei/Project/pH Sensor"> pH Sensor</A></TD><TD class="left"><STRONG class="selflink"> Attachment</STRONG></TD><TD class="left" style="width:140px;"><A href="/Team:NYMU-Taipei/Project/Time_Regulation" title="Team:NYMU-Taipei/Project/Time Regulation"> Time Regulation</A></TD><TD class="both" style="width:120px;"><A href="/Team:NYMU-Taipei/Project/Waste_Removal" title="Team:NYMU-Taipei/Project/Waste Removal"> Waste Removal</A></TD><TD class="both" style="width:180px;"><A href="/Team:NYMU-Taipei/Experiments" title="Team:NYMU-Taipei/Experiments"> Experiments and Parts</A></TD><TD class="left"><A href="/Team:NYMU-Taipei/Team" title="Team:NYMU-Taipei/Team"> About Us</A></TD></TR></TBODY></TABLE><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Motivation"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Motivation</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Goals"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Goals</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#What_to_Use_to_Attach_to_the_Small_Intestine"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">What to Use to Attach to the Small Intestine</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#FimH"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">FimH</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#FliC"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">FliC</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Intimin"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Intimin</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-7"><A href="#How_to_Attach_in_Intestine"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">How to Attach in Intestine</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#How_to_Detach_in_Intestine"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">How to Detach in Intestine</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Circuit_Design_and_Methods"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Circuit Design and Methods</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Construct"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Construct</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Attachment_Circuit_Design"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">Attachment Circuit Design</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Detachment_Circuit_Design"><SPAN class="tocnumber">6.3</SPAN><SPAN class="toctext">Detachment Circuit Design</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Experiment_Result"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Experiment Result</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#References"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Motivation"> Motivation </SPAN></H2><UL><LI>  Using gene-transformed bacteria in medical way is currently a hot biology and medicine topic, so we wonder about one thing: is there any way to boost the medical ability of bacteria? It's well known that many species of bacteria have the ability to attach to human tissue or epithelial cells. If we can control the attaching/detaching mechanism of these gene-transformed bacteria, we can enhance the bacteria ability by making a more direct interaction with a more specific targets, thereby make the whole system work more efficiently.
</LI></UL><H2><SPAN class="mw-headline" id="Goals"> Goals </SPAN></H2><UL><LI>Design a genetic <I>E.coli</I> to remove waste through the small intestine epithelium. 
</LI><LI>When our genetic <I>E.coli</I> senses the alternation of pH in the small intestine, the adhesion mechanism will be turned on in order to <B><FONT color="red">enhance</FONT></B> the ability of attachment.
</LI><LI>The enhanced adhesion will give our genetic <I>E.coli</I> more time to remove the wastes removed by the <A href="/Team:NYMU-Taipei/Project/Waste_Removal" title="Team:NYMU-Taipei/Project/Waste Removal">Waste Removal</A> subgroups.
</LI><LI>To cooperate with the &quot;<A href="/Team:NYMU-Taipei/Project/Time_Regulation" title="Team:NYMU-Taipei/Project/Time Regulation">Time Regulation</A>&quot; subgroup to allow our genetic <I>E.coli</I> to detach from intestinal epithelia cells after a specified amount of time.
</LI></UL><H2><SPAN class="mw-headline" id="What_to_Use_to_Attach_to_the_Small_Intestine">What to Use to Attach to the Small Intestine</SPAN></H2><UL><LI>We begin with three candidates: <B><FONT color="red">FimH</FONT></B>, <B><FONT color="red">FliC</FONT></B> and <B><FONT color="red">intimin</FONT></B>, and compare them below.
</LI></UL><H4><SPAN class="mw-headline" id="FimH"><FONT color="blue">FimH</FONT></SPAN></H4><UL><LI>The FimH protein is the receptor recognizing element of type 1 fimbriae
</LI></UL><TABLE><TBODY><TR><TD><UL><LI>Fimbriae
</LI><LI>Function: attachment of bacteria to its host as in the case of bacteria <I>Escherichia coli</I>, <I>salmonella typhimurium</I>, <I>Nisseria gonorrhoea</I>, and <I>bordella pertussis</I>.
</LI><LI>Size and number: Found many in number (up to 1000 per cell), each having a diameter of between 3 to 25 nm, and each being 0.5 to 20 micrometers in length.
</LI></UL><LI><B>Type 1 fimbriae</B> are surface organelles of <I>Escherichia coli</I> which mediate <B>D-mannose-sensitive binding</B> to different host surfaces.
</LI><LI> A single type 1 fimbria is a 7-nm-wide and »1-um-long surface polymer, the bulk of the structure is made up of »1000 copies of the major subunit protein, FimA, polymerized into a right-handed helical structure.
</LI><LI> Small quantities of the minor components (FimF, FimG, and FimH) are also present.
</LI><LI> The <A href="/Team:NYMU-Taipei/Project/Attachment/FimH" title="Team:NYMU-Taipei/Project/Attachment/FimH"><B>FimH</B></A>  protein is the <B>receptor recognizing element</B> of type 1 fimbriae.
</LI><LI> Studies on <I>Salmonella Typhimurium</I> revealed that FimH adhesin is responsible for bacteria binding to HeLa, HEp-2 and mouse intestinal epithelial cells.
</LI><LI>Research indicats that bacteria with FimH-FocH hybrids still can act as normal bacteria, while the bacteria with <I>fimH</I> mutation loses its ability to attach to intestine.     
</LI></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="FliC"><FONT color="blue">FliC</FONT></SPAN></H4><TABLE><TBODY><TR><TD><UL><LI>FliC is an antigen of <B><I>E.coli</I> flagellum</B>.
</LI><LI>It binds to <B>mucin</B> and ECM.
</LI><LI>We didn't choose it to use because there are four PstI cuttting sites on K12 <I>fliC</I> gene.
</LI><LI>However, it still can be one of our models.
</LI></UL></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Intimin"><FONT color="blue">Intimin</FONT></SPAN></H4><TABLE><TBODY><TR><TD><UL><LI>Intimin is an adhesin which includes in a protein collection <B>Esps(EPEC secreted proteins</B>).
</LI><LI>It is coded in <I>eae</I> gene, which is a part of pathogenicity island termed <B>the locus of enterocyte effacement</B><B>(LEE).</B></LI><LI>Intimin can binds to host cell either through <A href="http://en.wikipedia.org/wiki/Type_three_secretion_system#Type_III_secretion_system_.28T3SS.29|" class="external text" rel="nofollow"><B>type III secretion system</B></A>, or via cell receptor, <B>integrin</B>.
</LI><LI>Studies of intimin reaval that even intimin domain could cause unsual cytoskeleton rearrange ment, which could lead to A/E leison; Therefore, we didn't use it as model.   
</LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="How_to_Attach_in_Intestine"> How to Attach in Intestine </SPAN></H2><UL><LI>We enhance the attachment in intestine by enhancing the expression of the membrane form of <B><FONT color="red">FimH</FONT></B>.
</LI></UL><UL><LI><FONT color="red">We can use the biobrick part!</FONT></LI><LI>Lac-inducible generator of Lpp-OmpA(46-66)-Streptavidin wild-type+ His6 tag(<B>Part:BBa_J36848</B>)<A href="http://partsregistry.org/Part:BBa_J36848" class="external autonumber" rel="nofollow">[1]</A></LI><LI>This device contains a lac promoter and strong ribosome binding site for lac-inducible expression of the fusion protein of Lpp signal peptide, OmpA aa46-66, and streptavidin wild-type + His6 tag. This expression should display streptavidin on the cell surface of <I>E. coli</I>.
</LI><LI>By displace the lac promoter and streptavidin part with <B>pH promoter</B> and <B>FimH binding domain</B>, we can induce the expression of FimH in an optimal pH, thus enhance the adhesion of genetic <I>E.coli</I> in intestine.
</LI></UL><H2><SPAN class="mw-headline" id="How_to_Detach_in_Intestine">How to Detach in Intestine </SPAN></H2><UL><LI>We <B>design a cutting site between FimH and trans-membrane domain</B>, than we tfansform a <B>protease</B> gene into our <I>E.coli</I>.
</LI><LI>The protease gene can be regulated by the promoter of <B>time regulation</B>, giving the expression of a secreted protease, which can cut the cutting site we design.
</LI></UL><H2><SPAN class="mw-headline" id="Circuit_Design_and_Methods">Circuit Design and Methods</SPAN></H2><H4><SPAN class="mw-headline" id="Construct">Construct</SPAN></H4><UL><LI>We begin our work at gaining FimH and LRLO.
</LI><LI>FimH is from E.coli K12. We uses its genome as template and use PCR to gain FimH.
</LI><LI> As mentioned in &quot;<B>How to Attach in Intestine</B>&quot;, there are five  units in partJ36848 : a lacZ promoter, a ribosome binding site , Lpp aa 20, OmpA aa46-66 and  streptavidin + His6 tag. Because we only need the first fout parts, so we  use PCR to produce the sequence we need, LRLO or partC.(J36848 without streptavidin)
</LI><LI><FONT color="blue">The reverse primer of LRLO is designed complemented to the front primer of muFimH.</FONT></LI><LI>Because there is a PstI cutting site in FimH, so we use a PCR method to mutate the cutting site. We desigh two set of primers, primerA(<B>pA</B>) anr primerB(<B>pB</B>).
</LI></UL><UL><LI>The front primer of pA and the reverse primer of pB is actually the primers we use to gain FimH, While pA reverse primer and pB front primer are designed to aneal to the two strand of  PstI cutting site, while one of the base pair has been replaced.
</LI><LI>We use pA and pB to gained the front part of FimH as well as back part seperately. Because the reverse pA and front pB are  complemented, so the two parts can be linked together by PCR, and since one of the basepair has been replaceed, so the cutting site will be mutated, thus we gain mutant FimH(muFimH, we also call it partD).
</LI></UL><LI>After we gained muFimH and LRLO , we use PCR to link the two parts as the primer are complemented. Finally we get LRLO-muFimH sequence(partE), the main constructed sequence in our project.
</LI><LI>Before combining with other group's design, we need to test ours. Therefore we do two binding assay:
</LI><LI>One use GFP-partE to observe wether the ability of LRLO-FimH hybrids can attach to intestine.
</LI><LI>The other use pH sensor to replace lacZ promoter to see wether the protein expression could be regulate by pH.
</LI><H4><SPAN class="mw-headline" id="Attachment_Circuit_Design">Attachment Circuit Design</SPAN></H4><UL><LI><B>WORKING PART</B></LI><LI><B>TESTING PART</B></LI></UL><H4><SPAN class="mw-headline" id="Detachment_Circuit_Design">Detachment Circuit Design</SPAN></H4><UL><LI><B>PROTEASE PART</B></LI><LI>Design protease cleavage site between OmpA and FimH
</LI></UL><H2><SPAN class="mw-headline" id="Experiment_Result"> Experiment Result </SPAN></H2><UL><LI><B>Protocal and Method Check out</B></LI><LI> Methods
</LI><LI><B>Figure A. </B>DH5a transformed with E0840 (pTET) reporter plasmid can be detected by multi-detection microplate reader. 
</LI><LI><B>Figure B. </B> HT29 cells incubated with DH5a transformed with E0840 (pTET) <B>GFP</B> reporter plasmid.
</LI></UL><H2><SPAN class="mw-headline" id="References"> References </SPAN></H2><UL><LI><FONT color="blue">Functional characterization of the FimH adhesin from Salmonella enterica serovar Enteritidis</FONT>(D. Kisiela etc,2006)<A href="http://mic.sgmjournals.org/cgi/content/full/152/5/1337?view=long&amp;pmid=16622051" class="external autonumber" rel="nofollow">[2]</A></LI><LI><FONT color="blue">Combining sites of bacterial fimbriae</FONT>(H. D. Grevea, L. Wynsa, J. Bouckaerta,2007) <A href="http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B6VS6-4PGPKS8-2&amp;_user=1576506&amp;_rdoc=1&amp;_fmt=&amp;_orig=search&amp;_sort=d&amp;view=c&amp;_acct=C000053839&amp;_version=1&amp;_urlVersion=0&amp;_userid=1576506&amp;md5=33dfc6974f43b7ece79e0e7b90044ee7" class="external autonumber" rel="nofollow">[3]</A></LI><LI>Probing the receptor recognition site of the FimH adhesin by fimbriae- displayed FimH-FocH hybrids<A href="http://mic.sgmjournals.org/cgi/reprint/144/7/1919?view=reprint&amp;pmid=9695925" class="external autonumber" rel="nofollow">[4]</A></LI><LI><FONT color="blue">Host Protein Binding and Adhesive Properties of H6 and H7 Flagella of Attaching and Effacing Escherichia coli</FONT>(A. L. Erdem etc,2007)<A href="http://jb.asm.org/cgi/content/full/189/20/7426" class="external autonumber" rel="nofollow">[5]</A></LI><LI><FONT color="blue">Sequence Diversity of the Escherichia coli H7 fliC Genes:Implication for a DNA-Based Typing Scheme forE. coli O157:H7</FONT>(L. WANG etc,2007)<A href="http://jcm.asm.org/cgi/reprint/38/5/1786.pdf" class="external autonumber" rel="nofollow">[6]</A></LI><LI><FONT color="blue">Exploitation of host cells by enteropathogenic Escherichia coli</FONT>(B. A. Vallance and B. B. Finlay,2000)<A href="http://cmgm.stanford.edu/micro/MI209/EPEC.pdf" class="external autonumber" rel="nofollow">[7]</A></LI><LI><FONT color="blue">The Cell-binding Domain of Intimin from Enteropathogenic Escherichia coli Binds to b1 Integrins</FONT>(G. Dougan etc,1996)<A href="http://www.jbc.org/cgi/reprint/271/34/20359" class="external autonumber" rel="nofollow">[8]</A></LI><LI><FONT color="blue">Intimin from enteropathogenic Escherichia coli mediates remodelling of the eukaryotic cell surface</FONT>(A.D.Phillips etc,2000)<A href="http://mic.sgmjournals.org/cgi/content/full/146/6/1333" class="external autonumber" rel="nofollow">[9]</A></LI><LI><FONT color="blue">Intervening with Urinary Tract Infections Using Anti-Adhesives Based on the Crystal Structure of the FimH–Oligomannose-3 Complex</FONT>(A. Wellens etc,2008)<A href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2323111" class="external autonumber" rel="nofollow">[10]</A></LI></UL><DIV id="bottomlinkbox"><A id="bottomlink" href="https://2008.igem.org/Team:NYMU-Taipei">NYMU-Taipei iGEM 2008 Team</A></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:NYMU-Taipei/Project/Attachment">http://2008.igem.org/Team:NYMU-Taipei/Project/Attachment</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NYMU-Taipei/Project/Attachment" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NYMU-Taipei/Project/Attachment" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NYMU-Taipei/Project/Attachment&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NYMU-Taipei/Project/Attachment&amp;oldid=94252" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>