"
Page
Discussion
View source
History
teams
Log in
 
Team:NYMU-Taipei/Project/Time Regulation/Reloxilator
From 2008.igem.org
Home
Project Overview:
pH Sensor
Attachment
Time Regulation
Waste Removal
Experiments and Parts
About Us
Our Reloxilator (Relaxation Oscillator): A four-part system consisting of an oscillator, a synchronizer, a tuner, and a reporter.
Contents
1 Brief Description
2 System Description
2.1 The Oscillator
2.1.1 Components
2.1.2 Related Biobricks
2.1.3 Problems
2.1.4 References
2.2 The Synchronizer
2.2.1 Components
2.2.1.1 Component positions on Vibrio Fischeri
2.2.2 Related Biobricks
2.2.3 Problems
2.2.4 References
2.3 The Tuner
2.3.1 Components
2.3.2 Problems
2.3.3 References
3 Sequences
3.1 Promoters
3.1.1 pRE
3.1.2 pLUX
3.1.3 pLac
3.2 Genes
3.2.1 CII
3.2.2 CIIIC
3.2.3 HtlB
3.2.4 LuxI
3.2.5 LuxR
4 Testing
4.1 Promoter Testing
4.2 Component Testing
5 Modelling
5.1 Oscillator Modelling 1
5.1.1 Variables Used
5.1.2 Formulas used
5.1.3 Conclusion
6 Experimental Results
7 References
Brief Description
The oscillator is a two-component relaxation oscillator . The first component consists of the PRE promoter and the gene CII in a positive feedback loop, and both originating from from the λ phage. The second component uses the HtlB gene from E. coli attached to the same promoter as the first component. The HtlB gene produces HtlB proteins that degrade CII proteins.
The synchronizer allows near-full synchronization between different cells after about two time periods of oscillation 1 . The main parts of the synchronizer is made up of the PLux promoter, and the LuxI and LuxR genes, all coming from the Vibrio Fischeri organism. PRE and the CII gene are the same as the parts used in the oscillator, tying the oscillator and synchronizer together.
The Tuner is a way to control the time period of the system by inhibiting the rate of degradation HtlB has on CII in the oscillator. The protein used is the CIIICd, a DNA synthesised version of the CIIIC protein from the study done by Halder et al 2 .
The Reporter is the promoter PRE (the same part used in the oscillator) attached to a green flurorescent protein gene. It is used in various tests to check many of the components in the system.
System Description
This section houses the description of our system and the we use.
The Oscillator
A two-component oscillator consisting of two genes: one that induces both genes, while the second gene represses the first one. This oscillation is much like the oscillation of a capacitor charging and discharging .
Components
pRE
CII
HtlB
Related Biobricks
BBa_R0053 works but comes from a different phage (P22). Therefore using it means no proof of oscillation with HtlB [citation required]. Just realised it doesn't mean we can't try it.
BBa_I7108 uses BBa_R0053 and produces little GFP output. But that may be good for us?
BBa_R1053 is a standardised version of it.
BBa_C0053 , the related coding region that activates BBa_R0053.
Problems
HtlB already exists in the strain of E. coli we're going to use (Escherichia coli K12 strain MG1655 (tax-id:511145).
Go ahead with the experiment, and then find out whether the amount that already exists would significantly impact our results or not by checking with a prediction.
Due to HtlB already existing in E. coli, if the initial concentration of CII isn't enough, would the oscillator never start?
References
McMillen, D. et al (2002) " Synchronizing genetic relaxation oscillators by intercell signalling ", Proc Natl Acad Sci USA, Vol. 22, No. 3, pp. 679-684.
Where the oscillator idea came from.
The Synchronizer
The synchronizer integrates the use of cell quorum sensing to introduce synchronization of intercell oscillations.
Components
pRE
LuxI
LuxR
pLUX
CII
Component positions on Vibrio Fischeri
We performed a bunch of sequence alignments between the sequences from the Ecocyc database and the Biobrick parts to determine position of those components on Vibrio Fischeri. Orange colour means transcription/translation in reverse direction.
norm means normalised (norm=x-1050153+1)
Component
length
Start
End
norm start
norm end
Source
LuxI
573
1050153
1050725
1
573
Biocyc
BBa_C0161
585
1050150
1050734
-2
582
BBa_c0161 and sequence alignment[2] (87.3% consequence.)
BBa_R0062 ( pLUX-right )
55
1050749
1050803
597
651
BBa_R0062 and sequence alignment[2] (72.7% consequence).
55->57
1050784
1050840
632
688
sequence alignment[1] (56% consensus and added 2bp). Looks wrong.
BBa_R0061
30->28
1050781
1050808
627
656
BBa_R0061 and sequence alignment[1] (75% consensus, but they deleted the first 2bp).
30->43
1050775
1050817
623
665
sequence alignment[2] (76.5% consensus. This looks wrong.)
BBa_R0063 (pLUX-left)
151->150
1050784
1050933
632
781
BBa_R0063 and sequence alignment[1] (64.5% consensus and removed 1bp).
BBa_c0062
756->753
1050941
1051693
789
1541
BBa_c0062 and sequence alignment[1] (78.9% consensus and removed 3bp)
LuxR
753
1050941
1051693
789
1541
Biocyc
[1]: Sequence Alignment with Biocyc Vibrio Fischeri (1050153-1051693) (end of LuxI to end of LuxR).
[2]: Sequence Alignment with the reverse compliment of [1].
Related Biobricks
BBa_R0062 or BBa_R1062 in conjunction with or instead of new construct (pRE).
BBa_C0062 in conjunction with or instead of LuxR .
BBa_C0061 or BBa_C0161 in conjunction with or instead of LuxI .
Other interesting parts that may come in use (all working):
BBa_F2621
BBa_F2622
BBa_T9002
Other interesting parts we may or may not use.
BBa_J37019 titled AHL induced LuxR generator.
BBa_J37032 if we want to test if BBa_R0062 works.
BBa_J37033 looks pretty useless.
BBa_I731012 looks interesting, but we're probably not going to do anything about it.
Problems
Aligning BBa_R0062 and BBa_R1062 gives 92.2% consensus.
Aligning pLUX-left and pLUX-right produced a 64% consensus.
Why are so many nucleotides different?
Why do the two promoters overlap so much (exactly according to the sequence alignment), yet the design doesn't say so?
Are they using a difference Vibrio Fischeri strain for both?
A sequence similar to LuxR exists in the E. coli strain we're using. Don't know if it'll affect anything.
References
McMillen, D. et al (2002) " Synchronizing genetic relaxation oscillators by intercell signalling ", Proc Natl Acad Sci USA, Vol. 22, No. 3, pp. 679-684.
Same paper as the main paper from the oscillator section.
The Tuner
A way to control the oscillator's time period.
Components
pLAC
CIIIC
Problems
The CIIIC protein does not appear in any other papers (nor does it appear on pdb for that matter, but then again CIII doesn't either). So we have to believe that one paper.
By synthesizing the CIIIC protein from DNA, we have to add the start codon ATG, resulting in an extra Methionine at the start. This may affect our experiments.
References
Hadler, S. et al (2007) Probing the Anitprotease Activity of lCIII, an Inhibitor of the Escherichia coli Metalloprotease HflB (FtsH) ", Journal of Bacteriology, Vol. 189, No. 22, pp. 8130-8138.
The only paper about the experimental results of CIIIC. Also has a lot of experimental values (such as ratios) that can be used.
Herman, C. et al (1997) " The HflB Protease of Escherichia coli Degrades Its Inhibitor λcIII ", Journal of Bacteriology, Vol. 179, No. 2, pp. 358-363.
Also has a lot of experimental values that could be used.
Sequences
This section lists the sequences of promoters and genes.
Promoters
The list of promoters used.
pRE
few hundred bp upstream from the CI gene.
40 to 60bp long.
Appropriate Literature
Interactions among CII protein, RNA polymerase adn λ pRE promoter: contacts between RNA polymerase and the -35 region of pRE are identical in the presence and absence of CII protein
Mutations that improve the pRE promoter of coliphage lambda
Cross-specificities between cII-like proteins and pRE-like promoters of lambdoid bacteriophages
complement(38391->38340) on lambda phage. AGAGCCTCGTTGCGTTTGTTTGCACGAACCATATGTAAGTATTTCCTT|AGAT (52bp)                                                 +----  transcription start site ----> so what we want: AGAGCCTCGTTGCGTTTGTTTGCACGAACCATATGTAAGTATTTCCTT (48bp)
pRE Overlaps with CII (but in the other direction)                        ATGGTTCGTGCAAACAAACGCAACGAGGCTCTACGAATCGAGAGTGCGTTG--> CII cont' <--ATCTAAGGAAATACTTACATATGGTTCGTGCAAACAAACGCAACGAGGCTCT (<--pRE)
For reference, CI : 37940-37227
Primers: AGAGCCTCGTTGCGTTT (len:17, tm:55deg, GC%:53) AAGGAAATACTTACATATGGTTCGTG (len:26, tm:55deg, GC%:35)  Primer used regions: AGAGCCTCGTTGCGTTTGTTTGCACGAACCATATGTAAGTATTTCCTT
There is only 5bp not included in the primer. Feels kinda weird.
pLUX
Part:BBa_R0062-luxR & HSL regulated  lux pR
acctgtaggatcgtacaggtttacgcaagaaaatggtttgttatagtcgaataaa(55 bp)
pLac
BBa_R0010 (200bp)
Genes
The list of genes used.
CII
A λ phage gene.
NCBI protein : 97aa
NCBI nucleotide : 294bp
>gi|9626243:38360-38653 Enterobacteria phage lambda, complete genome ATGGTTCGTGCAAACAAACGCAACGAGGCTCTACGAATCGAGAGTGCGTTGCTTAACAAAATCGCAATGC TTGGAACTGAGAAGACAGCGGAAGCTGTGGGCGTTGATAAGTCGCAGATCAGCAGGTGGAAGAGGGACTG GATTCCAAAGTTCTCAATGCTGCTTGCTGTTCTTGAATGGGGGGTCGTTGACGACGACATGGCTCGATTG GCGCGACAAGTTGCTGCGATTCTCACCAATAAAAAACGCCCGGCGGCAACCGAGCGTTCTGAACAAATCC AGATGGAGTTCTGA
Primers: ATGGTTCGTGCAAACAAACG (56deg) tcagaactccatctggatttgt
CIIIC
CIII (A λ phage gene) after residues 1-13 and 42-54 are cleaved (so only residues 14-41) are left.
Inhibites the degradation rate HtlB has on CII by binding to HtlB.
HtlB does not degrade CIIIC (unlike CIII).
NCBI (CIII) protein :54aa
NCBI (CIII) gene :165bp
CIII gene:
>gi|9626243:c33463-33299 Enterobacteria phage lambda, complete genome ATGCAATATGCCATTGCAGGGTGGCCTGTTGCTGGCTGCCCTTCCGAATCTTTACTTGAACGAATCACCC GTAAATTACGTGACGGATGGAAACGCCTTATCGACATACTTAATCAGCCAGGAGTCCCAAAGAATGGATC AAACACTTATGGCTATCCAGACTAA
Start with 165bp (54 residues + stop codon) then cleave off residues 2-13 (39bp) and 42-54 (39bp), leaving residues 1,14-41,55 (90bp)
CIIIC (with added ATG at the start):
ATGCAATATGCCATTGCAGGGTGGCCTGTTGCTGGCTGCCCTTCCGAATCTTTACTTGAACGAATCACCC GTAAATTACGTGACGGATGGAAACGCCTTATCGACATACTTAATCAGCCAGGAGTCCCAAAGAATGGATC AAACACTTATGGCTATCCAGACTAA
Changes to:
ATGCCTTCCGAATCTTTACTTGAACGAATCACCC GTAAATTACGTGACGGATGGAAACGCCTTATCGACATACTTAATCAGCCAGGA TAA
The resulting primers are:
ATGCCTTCCGAATCTTTACTTG ttatcctggctgattaagtatgtcg (56deg)
HtlB
An E. coli gene.
HtlB degrades CII.
Ecocyc : 1935bp.
No extra EcoRI,PstI,SpeI,XbaI in gene below.
>EG11506 hflB (complement(3324957..3323023)) E. coli atgGCGAAAA ACCTAATACT CTGGCTGGTC ATTGCCGTTG TGCTGATGTC AGTATTCCAG AGCTTTGGGC CCAGCGAGTC TAATGGCCGT AAGGTGGATT ACTCTACCTT CCTACAAGAG GTCAATAACG ACCAGGTTCG TGAAGCGCGT ATCAACGGAC GTGAAATCAA CGTTACCAAG AAAGATAGTA ACCGTTATAC CACTTACATT CCGGTTCAGG ATCCGAAATT ACTGGATAAC CTGTTGACCA AGAACGTCAA GGTTGTCGGT GAACCGCCTG AAGAACCAAG CCTGCTGGCT TCTATCTTCA TCTCCTGGTT CCCGATGCTG TTGCTGATTG GTGTCTGGAT CTTCTTCATG CGTCAAATGC AGGGCGGCGG TGGCAAAGGT GCCATGTCGT TTGGTAAGAG CAAAGCGCGC ATGCTGACGG AAGATCAGAT CAAAACGACC TTTGCTGACG TTGCGGGCTG CGACGAAGCA AAAGAAGAAG TTGCTGAACT GGTTGAGTAT CTGCGCGAGC CGAGCCGCTT CCAGAAACTC GGCGGTAAGA TCCCGAAAGG CGTCTTGATG GTCGGTCCTC CGGGTACCGG TAAAACGCTG CTGGCGAAAG CGATTGCAGG CGAAGCGAAA GTTCCGTTCT TTACTATCTC CGGTTCTGAC TTCGTAGAAA TGTTCGTCGG TGTGGGTGCA TCCCGTGTTC GTGACATGTT CGAACAGGCG AAGAAAGCGG CACCGTGCAT CATCTTTATC GATGAAATCG ACGCCGTAGG CCGCCAGCGT GGCGCTGGTC TGGGCGGTGG TCACGATGAA CGTGAACAGA CTCTGAACCA GATGCTGGTT GAGATGGATG GCTTCGAAGG TAACGAAGGT ATCATCGTTA TCGCCGCGAC TAACCGTCCG GACGTTCTCG ACCCGGCCCT GCTGCGTCCT GGCCGTTTCG ACCGTCAGGT TGTGGTCGGC TTGCCAGATG TTCGCGGTCG TGAGCAGATC CTGAAAGTTC ACATGCGTCG CGTACCATTG GCACCCGATA TCGACGCGGC AATCATTGCC CGTGGTACTC CTGGTTTCTC CGGTGCTGAC CTGGCGAACC TGGTGAACGA AGCGGCACTG TTCGCTGCTC GTGGCAACAA ACGCGTTGTG TCGATGGTTG AGTTCGAGAA AGCGAAAGAC AAAATCATGA TGGGTGCGGA ACGTCGCTCC ATGGTGATGA CGGAAGCGCA GAAAGAATCG ACGGCTTACC ACGAAGCGGG TCATGCGATT ATCGGTCGCC TGGTGCCGGA ACACGATCCG GTGCACAAAG TGACGATTAT CCCACGCGGT CGTGCGCTGG GTGTGACTTT CTTCTTGCCT GAGGGCGACG CAATCAGCGC CAGCCGTCAG AAACTGGAAA GCCAGATTTC TACGCTGTAC GGTGGTCGTC TGGCAGAAGA GATCATCTAC GGGCCGGAAC ATGTATCTAC CGGTGCGTCC AACGATATTA AAGTTGCGAC CAACCTGGCA CGTAACATGG TGACTCAGTG GGGCTTCTCT GAGAAATTGG GTCCACTGCT GTACGCGGAA GAAGAAGGTG AAGTGTTCCT CGGCCGTAGC GTAGCGAAAG CGAAACATAT GTCCGATGAA ACTGCACGTA TCATCGACCA GGAAGTGAAA GCACTGATTG AGCGTAACTA TAATCGTGCG CGTCAGCTTC TGACCGACAA TATGGATATT CTGCATGCGA TGAAAGATGC TCTCATGAAA TATGAGACTA TCGACGCACC GCAGATTGAT GACCTGATGG CACGTCGCGA TGTACGTCCG CCAGCGGGCT GGGAAGAACC AGGCGCTTCT AACAATTCTG GCGACAATGG TAGTCCAAAG GCTCCTCGTC CGGTTGATGA ACCGCGTACG CCGAACCCGG GTAACACCAT GTCAGAGCAG TTAGGCGACA AGTAA
HtlB primers: atgGCGAAAAACCTAATACTCTG ttacttgtcgcctaactgct
LuxI
A Vibrio Fischeri gene.
Biocyc : 573bp
Biobrick available
BBa_J37034 : LuxI + GFP. We could use this if we can extract the RBS+LuxI part.
>VFA0924 VFA0924 (complement(1050725..1050153)) V. fischeri chromosome es114 ii ATGataaaaa aatcggactt tttgggcatt ccatcagagg agtatagagg tattcttagt cttcgttatc aggtatttaa acgaagactg gagtgggact tggtaagtga ggataatctt gaatcagatg aatatgataa ctcaaatgca gaatatattt atgcttgtga tgatgcggaa gaggtaaatg gctgttggcg tttgttacct acaacgggtg attacatgtt aaaaactgtt tttcctgaat tgctcggaga tcaagtagcc ccaagagatc caaatatagt cgaattaagc cgttttgctg tgggaaaaaa tagctcaaaa ataaataact ctgctagtga aataacaatg aaattgtttc aagctatata taaacacgca gttagtcaag gtattacaga atatgtaaca gtaacatcaa tagcaataga gcgatttctg aaacgtatta aagttccttg tcatcgcatt ggtgataagg agattcattt attaggtaat actagatctg ttgtattgtc tatgcctatt aatgatcagt ttagaaaagc tgtatcaaat taa
LuxI primers: ATGataaaaaaatcggactttttggg ttaatttgatacagcttttctaaactgatc (length=30 too long?)
LuxR
A Vibrio Fischeri gene.
NCBI : 753bp.
Biobrick available
BBa_J37019 : Promoter + LuxR. 862bp
>VFA0925 VFA0925 1050941..1051693 V. fischeri chromosome es114 ii ATGaacatta aaaatataaa tgctaatgag aagataattg ataaaattaa aacttgtaat aataataaag atattaatca atgtttatct gaaatagcaa agataataca ttgtgaatat tacctattcg ctattatcta tcctcactca ataattaaac ctgatgtttc aattatagat aattaccctg aaaaatggcg taaatattat gatgatgccg gactactaga atatgaccct gtagtcgatt actctaagtc ccatcattca ccaattaatt ggaacgtatt cgaaaaaaaa acaataaaaa aagagtctcc gaatgtaata aaagaagcac aggaatcggg actcattact ggatttagct ttccaattca tactgcaagt aatggttttg gaatgctcag ttttgctcat tcagataaag atatttatac tgacagttta tttttacacg ctagtacaaa tgtaccatta atgcttcctt ctttagtcga taattatcaa aaaataaata cgacacgtaa aaagtcagat tctattttaa caaaaagaga aaaagaatgc ttagcgtggg cgagtgaagg aaaaagtaca tgggatattt caaaaatact tggctgcagt gagcgtactg tcacttttca tttaaccaat actcaaatga aactcaatac aactaaccgc tgccaaagta tttctaaagc aattttaact ggcgccatta attgtccata ccttaaaaat taa
LuxR primers: ATGaacattaaaaatataaatgctaatgagaaga (length=34 too long?) ttaatttttaaggtatggacaattaatggc (length=30 too long?)
Testing
How we plan to test our parts when we've finished constructed them.
Promoter Testing
We are attempting to establish a "inducible promoter standard," much like the repressible promoter standard, where instead we define the IPTG inducible "standard" pLac promoter as having a value of 1, while all other inducible promoters will be measured in multiples of this standard. Then the value of IPTG can also be varied to test the ratios between the promoters as to whether the ratio is linear, sinusoidal, polynomial, etc.
We can test pRE promoter via this standard by connecting the |CII gene behind pLac, and connecting a reporter behind pRE.
This standard can also be used to test promoters that require a complex to be induced, such as pLux. pLux requires the R+AHL complex to be induced. To test the the strength of pLux, first let all parts of the complex save one to be always existing, while the last one would be controlled by pLac. In this example, LuxR is always expressed and the amount of AHL is determined by pLac.
Component Testing
This is the part where we test to see if our component works. The diagram above shows the pathway between the various inputs, and the output (GFP).
Modelling
Oscillator Modelling 1
HtlB concentration
CII concentration.
20sec intervals.
Variables Used
pRE Stength: 0.4 (1 CII protein produces 0.4 CII and HtlB proteins).
H-t1/2: 8mins (random value I picked, its probably not the real value.)
Half life of HtlB.
Changing this value just makes the oscllation period longer.
C0: 8mins.
CII t1/2 with no HtlB present. Based on Shotland2000 )
C1: 1min.
Also based on the same paper.
CII t1/2 with an equal amount of HtlB is what I assume because they don't say.
Δt: 20sec
Time segment used for the calculation (can't be too close to the t1/2's).
Formulas used
The HtlB/CII ratio used to calculate the t1/2 at a specific time point:
R = [HtlB] / [CII] * [CII] = [HtlB] ----- (1)
The last *[CII] has something to do with the [HtlB][CII] --> [CII] equation.
Calculation of CII's t1/2 based on the ratio above:
C-t1/2 = C0 x (C1/C0)R ----- (2)
The calculation of amount of substrates remaining w.r.t the half lives:
[HtlB] = [HtlB] x 0.5(Δt / H-t1/2) ----- (3) [CII] = [CII] x 0.5(Δt / C-t1/2) ----- (4)
Conclusion
Fiddling around a bit yields:
Variable
Affects
Initial concentration of:
HtlB (higher)
(longer) startup time
CII (higher)
Natural degradation rate of
HtlB (longer)
(longer) time period
CII (longer)
pRE strength (higher)
(shorter) time period
HtlB to CII degradation rate (higher)
(lower) global maximum concentration (not counting startup)
Is a promoter that can be activated by HtlB? because then our calculations would be even more accurate.
Experimental Results
This is a picture of all the constructs, and green constructs are the ones that were successfully created.
References
McMillen, D. et al (2002) " Synchronizing genetic relaxation oscillators by intercell signalling ", Proc Natl Acad Sci USA, Vol. 22, No. 3, pp. 679-684.
Hadler, S. et al (2007) Probing the Anitprotease Activity of λCIII, an Inhibitor of the Escherichia coli Metalloprotease HflB (FtsH) ", Journal of Bacteriology, Vol. 189, No. 22, pp. 8130-8138.
NYMU-Taipei iGEM 2008 Team
Retrieved from " http://2008.igem.org/Team:NYMU-Taipei/Project/Time_Regulation/Reloxilator "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
