<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NYMU-Taipei_Project_Time_Regulation_Reloxilator"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:NYMU-Taipei/Project/Time_Regulation/Reloxilator">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NYMU-Taipei/Project/Time_Regulation/Reloxilator&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NYMU-Taipei/Project/Time_Regulation/Reloxilator&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NYMU-Taipei/Project/Time_Regulation/Reloxilator&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NYMU-Taipei/Project/Time_Regulation/Reloxilator" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NYMU-Taipei/Project/Time Regulation/Reloxilator</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><TABLE id="nav" style="border:1px solid gray;width:965px"><TBODY><TR><TD class="right"><A href="/Team:NYMU-Taipei" title="Team:NYMU-Taipei"> Home</A></TD><TD class="left" style="width:120px;"><A href="/Team:NYMU-Taipei/Project" title="Team:NYMU-Taipei/Project"> Project Overview:</A></TD><TD class="left"><A href="/Team:NYMU-Taipei/Project/pH_Sensor" title="Team:NYMU-Taipei/Project/pH Sensor"> pH Sensor</A></TD><TD class="left"><A href="/Team:NYMU-Taipei/Project/Attachment" title="Team:NYMU-Taipei/Project/Attachment"> Attachment</A></TD><TD class="left" style="width:140px;"><A href="/Team:NYMU-Taipei/Project/Time_Regulation" title="Team:NYMU-Taipei/Project/Time Regulation"> Time Regulation</A></TD><TD class="both" style="width:120px;"><A href="/Team:NYMU-Taipei/Project/Waste_Removal" title="Team:NYMU-Taipei/Project/Waste Removal"> Waste Removal</A></TD><TD class="both" style="width:180px;"><A href="/Team:NYMU-Taipei/Experiments" title="Team:NYMU-Taipei/Experiments"> Experiments and Parts</A></TD><TD class="left"><A href="/Team:NYMU-Taipei/Team" title="Team:NYMU-Taipei/Team"> About Us</A></TD></TR></TBODY></TABLE><P>Our <B>Reloxilator</B> (Relaxation Oscillator): A four-part system consisting of an oscillator, a synchronizer, a tuner, and a reporter.
</P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Brief_Description"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Brief Description</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#System_Description"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">System Description</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#The_Oscillator"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">The Oscillator</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Components"><SPAN class="tocnumber">2.1.1</SPAN><SPAN class="toctext">Components</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Related_Biobricks"><SPAN class="tocnumber">2.1.2</SPAN><SPAN class="toctext">Related Biobricks</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Problems"><SPAN class="tocnumber">2.1.3</SPAN><SPAN class="toctext">Problems</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#References"><SPAN class="tocnumber">2.1.4</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-8"><A href="#The_Synchronizer"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">The Synchronizer</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Components_2"><SPAN class="tocnumber">2.2.1</SPAN><SPAN class="toctext">Components</SPAN></A></LI><LI class="toclevel-4 tocsection-10"><A href="#Component_positions_on_Vibrio_Fischeri"><SPAN class="tocnumber">2.2.1.1</SPAN><SPAN class="toctext">Component positions on Vibrio Fischeri</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Related_Biobricks_2"><SPAN class="tocnumber">2.2.2</SPAN><SPAN class="toctext">Related Biobricks</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Problems_2"><SPAN class="tocnumber">2.2.3</SPAN><SPAN class="toctext">Problems</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#References_2"><SPAN class="tocnumber">2.2.4</SPAN><SPAN class="toctext">References</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#The_Tuner"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">The Tuner</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Components_3"><SPAN class="tocnumber">2.3.1</SPAN><SPAN class="toctext">Components</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#Problems_3"><SPAN class="tocnumber">2.3.2</SPAN><SPAN class="toctext">Problems</SPAN></A></LI><LI class="toclevel-3 tocsection-17"><A href="#References_3"><SPAN class="tocnumber">2.3.3</SPAN><SPAN class="toctext">References</SPAN></A></LI><LI class="toclevel-1 tocsection-18"><A href="#Sequences"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Sequences</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#Promoters"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Promoters</SPAN></A></LI><LI class="toclevel-3 tocsection-20"><A href="#pRE"><SPAN class="tocnumber">3.1.1</SPAN><SPAN class="toctext">pRE</SPAN></A></LI><LI class="toclevel-3 tocsection-21"><A href="#pLUX"><SPAN class="tocnumber">3.1.2</SPAN><SPAN class="toctext">pLUX</SPAN></A></LI><LI class="toclevel-3 tocsection-22"><A href="#pLac"><SPAN class="tocnumber">3.1.3</SPAN><SPAN class="toctext">pLac</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#Genes"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Genes</SPAN></A></LI><LI class="toclevel-3 tocsection-24"><A href="#CII"><SPAN class="tocnumber">3.2.1</SPAN><SPAN class="toctext">CII</SPAN></A></LI><LI class="toclevel-3 tocsection-25"><A href="#CIIIC"><SPAN class="tocnumber">3.2.2</SPAN><SPAN class="toctext">CIIIC</SPAN></A></LI><LI class="toclevel-3 tocsection-26"><A href="#HtlB"><SPAN class="tocnumber">3.2.3</SPAN><SPAN class="toctext">HtlB</SPAN></A></LI><LI class="toclevel-3 tocsection-27"><A href="#LuxI"><SPAN class="tocnumber">3.2.4</SPAN><SPAN class="toctext">LuxI</SPAN></A></LI><LI class="toclevel-3 tocsection-28"><A href="#LuxR"><SPAN class="tocnumber">3.2.5</SPAN><SPAN class="toctext">LuxR</SPAN></A></LI><LI class="toclevel-1 tocsection-29"><A href="#Testing"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Testing</SPAN></A></LI><LI class="toclevel-2 tocsection-30"><A href="#Promoter_Testing"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Promoter Testing</SPAN></A></LI><LI class="toclevel-2 tocsection-31"><A href="#Component_Testing"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Component Testing</SPAN></A></LI><LI class="toclevel-1 tocsection-32"><A href="#Modelling"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Modelling</SPAN></A></LI><LI class="toclevel-2 tocsection-33"><A href="#Oscillator_Modelling_1"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Oscillator Modelling 1</SPAN></A></LI><LI class="toclevel-3 tocsection-34"><A href="#Variables_Used"><SPAN class="tocnumber">5.1.1</SPAN><SPAN class="toctext">Variables Used</SPAN></A></LI><LI class="toclevel-3 tocsection-35"><A href="#Formulas_used"><SPAN class="tocnumber">5.1.2</SPAN><SPAN class="toctext">Formulas used</SPAN></A></LI><LI class="toclevel-3 tocsection-36"><A href="#Conclusion"><SPAN class="tocnumber">5.1.3</SPAN><SPAN class="toctext">Conclusion</SPAN></A></LI><LI class="toclevel-1 tocsection-37"><A href="#Experimental_Results"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Experimental Results</SPAN></A></LI><LI class="toclevel-1 tocsection-38"><A href="#References_4"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Brief_Description"> Brief Description </SPAN></H2><P>The oscillator is a two-component <A href="http://en.wikipedia.org/wiki/Relaxation_Oscillator" class="external text" rel="nofollow">relaxation oscillator</A>. The first component consists of the P<SUB>RE</SUB> promoter and the gene CII in a positive feedback loop, and both originating from from the λ phage. The second component uses the HtlB gene from <I>E. coli</I> attached to the same promoter as the first component. The HtlB gene produces HtlB proteins that degrade CII proteins.
</P><P>The synchronizer allows near-full synchronization between different cells after about two time periods of oscillation<A href="#References"><SUP>1</SUP></A>.
The main parts of the synchronizer is made up of the P<SUB>Lux</SUB> promoter, and the LuxI and LuxR genes, all coming from the <I>Vibrio Fischeri</I> organism. P<SUB>RE</SUB> and the CII gene are the same as the parts used in the oscillator, tying the oscillator and synchronizer together.
</P><P>The Tuner is a way to control the time period of the system by inhibiting the rate of degradation HtlB has on CII in the oscillator. The protein used is the CIIICd, a DNA synthesised version of the CIIIC protein from the study done by Halder <I>et al</I><A href="#References"><SUP>2</SUP></A>.
</P><P>The Reporter is the promoter P<SUB>RE</SUB> (the same part used in the oscillator) attached to a green flurorescent protein gene. It is used in various tests to check many of the components in the system.
</P><H2><SPAN class="mw-headline" id="System_Description"> System Description </SPAN></H2><P>This section houses the description of our system and the we use.
</P><H3><SPAN class="mw-headline" id="The_Oscillator"> The Oscillator </SPAN></H3><P>A two-component oscillator consisting of two genes: one that induces both genes, while the second gene represses the first one. This oscillation is much like the oscillation of a <A href="http://en.wikipedia.org/wiki/Relaxation_Oscillator" class="external text" rel="nofollow">capacitor charging and discharging</A>.
</P><H4><SPAN class="mw-headline" id="Components"> Components </SPAN></H4><UL><LI><A href="#pRE">pRE</A></LI><LI><A href="#CII">CII</A></LI><LI><A href="#HtlB">HtlB</A></LI></UL><H4><SPAN class="mw-headline" id="Related_Biobricks"> Related Biobricks </SPAN></H4><UL><LI><A href="http://partsregistry.org/Part:BBa_R0053" class="external text" rel="nofollow">BBa_R0053</A> works but comes from a different phage (P22). Therefore using it means no proof of oscillation with HtlB [<FONT color="blue">citation required</FONT>]. Just realised it doesn't mean we can't try it.
</LI><LI><A href="http://partsregistry.org/Part:BBa_I7108" class="external text" rel="nofollow">BBa_I7108</A> uses BBa_R0053 and produces little GFP output. But that may be good for us?
</LI><LI><A href="http://partsregistry.org/Part:BBa_R1053" class="external text" rel="nofollow">BBa_R1053</A> is a standardised version of it.
</LI><LI><A href="http://partsregistry.org/Part:BBa_C0053" class="external text" rel="nofollow">BBa_C0053</A>, the related coding region that activates BBa_R0053.
</LI></UL><H4><SPAN class="mw-headline" id="Problems"> Problems </SPAN></H4><UL><LI><A href="http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?db=protein&amp;val=16131068" class="external text" rel="nofollow">HtlB</A> already exists in the strain of <I>E. coli</I> we're going to use (<I>Escherichia coli K12 strain MG1655 (tax-id:511145</I>).
</LI><LI> Go ahead with the experiment, and then find out whether the amount that already exists would significantly impact our results or not by checking with a prediction.
</LI><LI> Due to HtlB already existing in <I>E. coli</I>, if the initial concentration of CII isn't enough, would the oscillator never start?
</LI></UL><H4><SPAN class="mw-headline" id="References"> References </SPAN></H4><UL><LI> McMillen, D. et al (2002) &quot;<A href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=117365" class="external text" rel="nofollow">Synchronizing genetic relaxation oscillators by intercell signalling</A>&quot;, Proc Natl Acad Sci USA, Vol. 22, No. 3, pp. 679-684.
</LI><LI> Where the oscillator idea came from.
</LI></UL><H3><SPAN class="mw-headline" id="The_Synchronizer"> The Synchronizer </SPAN></H3><P>The synchronizer integrates the use of cell quorum sensing to introduce synchronization of intercell oscillations.
</P><H4><SPAN class="mw-headline" id="Components_2"> Components </SPAN></H4><UL><LI><A href="#pRE">pRE</A></LI><LI><A href="#LuxI">LuxI</A></LI><LI><A href="#LuxR">LuxR</A></LI><LI><A href="#pLUX">pLUX</A></LI><LI><A href="#CII">CII</A></LI></UL><H5><SPAN class="mw-headline" id="Component_positions_on_Vibrio_Fischeri"> Component positions on <I>Vibrio Fischeri</I></SPAN></H5><P>We performed a bunch of sequence alignments between the sequences from the Ecocyc database and the Biobrick parts to determine position of those components on <I>Vibrio Fischeri</I>.
<FONT color="orange">Orange colour means transcription/translation in reverse direction.</FONT>norm means normalised (norm=x-1050153+1)
</P><TABLE border="1"><TBODY><TR><TH> Component </TH><TH> length </TH><TH> Start </TH><TH> End </TH><TH> norm start </TH><TH> norm end </TH><TH> Source
</TH></TR><TR><TD class="ob"><A href="#LuxI">LuxI</A></TD><TD> 573 </TD><TD> 1050153 </TD><TD> 1050725 </TD><TD> 1 </TD><TD> 573 </TD><TD><A href="http://biocyc.org/VFIS312309/NEW-IMAGE?type=GENE&amp;object=VFA0924" class="external text" rel="nofollow">Biocyc</A></TD></TR><TR><TD class="ob"> BBa_C0161 </TD><TD> 585 </TD><TD> 1050150 </TD><TD> 1050734 </TD><TD> -2 </TD><TD> 582 </TD><TD><A href="http://partsregistry.org/Part:BBa_C0161" class="external text" rel="nofollow">BBa_c0161</A> and sequence alignment[2] (87.3% consequence.)
</TD></TR><TR><TD class="ob" rowspan="2"> BBa_R0062 (<A href="#pLUX">pLUX-right</A>) </TD><TD> 55 </TD><TD> 1050749 </TD><TD> 1050803 </TD><TD> 597 </TD><TD> 651 </TD><TD><A href="http://partsregistry.org/Part:BBa_R0062:Design" class="external text" rel="nofollow">BBa_R0062</A> and sequence alignment[2] (72.7% consequence).
</TD></TR><TR><TD> 55-&gt;57 </TD><TD> 1050784 </TD><TD> 1050840 </TD><TD> 632 </TD><TD> 688 </TD><TD> sequence alignment[1] (56% consensus and added 2bp). Looks wrong.
</TD></TR><TR><TD rowspan="2"> BBa_R0061 </TD><TD> 30-&gt;28 </TD><TD> 1050781 </TD><TD> 1050808 </TD><TD> 627 </TD><TD> 656 </TD><TD><A href="http://partsregistry.org/wiki/index.php/Part:BBa_R0061" class="external text" rel="nofollow">BBa_R0061</A> and sequence alignment[1] (75% consensus, but they deleted the first 2bp).
</TD></TR><TR><TD> 30-&gt;43 </TD><TD> 1050775 </TD><TD> 1050817 </TD><TD> 623 </TD><TD> 665 </TD><TD> sequence alignment[2] (76.5% consensus. This looks wrong.)
</TD></TR><TR><TD> BBa_R0063 (pLUX-left) </TD><TD> 151-&gt;150 </TD><TD> 1050784 </TD><TD> 1050933 </TD><TD> 632 </TD><TD> 781 </TD><TD><A href="http://partsregistry.org/Part:BBa_R0063:Design" class="external text" rel="nofollow">BBa_R0063</A> and sequence alignment[1] (64.5% consensus and removed 1bp).
</TD></TR><TR><TD> BBa_c0062 </TD><TD> 756-&gt;753 </TD><TD> 1050941 </TD><TD> 1051693 </TD><TD> 789 </TD><TD> 1541 </TD><TD><A href="http://partsregistry.org/Part:BBa_C0062" class="external text" rel="nofollow">BBa_c0062</A> and sequence alignment[1] (78.9% consensus and removed 3bp)
</TD></TR><TR><TD><A href="#LuxR">LuxR</A></TD><TD> 753 </TD><TD> 1050941 </TD><TD>1051693 </TD><TD> 789 </TD><TD> 1541 </TD><TD><A href="http://biocyc.org/VFIS312309/NEW-IMAGE?type=GENE-IN-MAP&amp;object=VFA0925" class="external text" rel="nofollow">Biocyc</A></TD></TR></TBODY></TABLE><P>[1]: Sequence Alignment with <A href="http://biocyc.org/VFIS312309/sequence-delimited?chromosome=VF-CHROM-ES114-II&amp;object=VFA0924&amp;left=1050153&amp;right=1051693" class="external text" rel="nofollow">Biocyc <I>Vibrio Fischeri</I> (1050153-1051693)</A> (end of LuxI to end of LuxR).
[2]: Sequence Alignment with the reverse compliment of [1].
</P><H4><SPAN class="mw-headline" id="Related_Biobricks_2"> Related Biobricks </SPAN></H4><UL><LI><A href="http://partsregistry.org/Part:BBa_R0062" class="external text" rel="nofollow">BBa_R0062</A> or <A href="http://partsregistry.org/Part:BBa_R1062" class="external text" rel="nofollow">BBa_R1062</A> in conjunction with or instead of new construct (pRE).
</LI><LI><A href="http://partsregistry.org/Part:BBa_C0062" class="external text" rel="nofollow">BBa_C0062</A> in conjunction with or instead of <A href="#LuxR">LuxR</A>.
</LI><LI><A href="http://partsregistry.org/Part:BBa_C0061" class="external text" rel="nofollow">BBa_C0061</A> or <A href="http://partsregistry.org/Part:BBa_C0161" class="external text" rel="nofollow">BBa_C0161</A> in conjunction with or instead of <A href="#LuxI">LuxI</A>.
</LI></UL><P>Other interesting parts that may come in use (all working):
</P><UL><LI><A href="http://partsregistry.org/Part:BBa_F2621" class="external text" rel="nofollow">BBa_F2621</A></LI><LI><A href="http://partsregistry.org/Part:BBa_F2622" class="external text" rel="nofollow">BBa_F2622</A></LI><LI><A href="http://partsregistry.org/Part:BBa_T9002" class="external text" rel="nofollow">BBa_T9002</A></LI></UL><P>Other interesting parts we may or may not use.
</P><UL><LI><A href="http://partsregistry.org/Part:BBa_J37019" class="external text" rel="nofollow">BBa_J37019</A> titled <I>AHL induced LuxR generator</I>.
</LI><LI><A href="http://partsregistry.org/Part:BBa_J37032" class="external text" rel="nofollow">BBa_J37032</A> if we want to test if <A href="http://partsregistry.org/Part:BBa_R0062" class="external text" rel="nofollow">BBa_R0062</A> works.
</LI><LI><A href="http://partsregistry.org/Part:BBa_J37033" class="external text" rel="nofollow">BBa_J37033</A> looks pretty useless.
</LI><LI><A href="http://partsregistry.org/Part:BBa_I731012" class="external text" rel="nofollow">BBa_I731012</A> looks interesting, but we're probably not going to do anything about it.
</LI></UL><H4><SPAN class="mw-headline" id="Problems_2"> Problems </SPAN></H4><OL><LI>Aligning <A href="http://partsregistry.org/Part:BBa_R0062" class="external text" rel="nofollow">BBa_R0062</A> and <A href="http://partsregistry.org/Part:BBa_R1062" class="external text" rel="nofollow">BBa_R1062</A> gives 92.2% consensus.
</LI><LI>Aligning <A href="http://partsregistry.org/Part:BBa_R0063" class="external text" rel="nofollow">pLUX-left</A> and <A href="http://partsregistry.org/Part:BBa_R0062" class="external text" rel="nofollow">pLUX-right</A> produced a 64% consensus.
</LI><LI> Why are so many nucleotides different?
</LI><LI> Why do the two promoters overlap so much (exactly according to the sequence alignment), yet the <A href="http://partsregistry.org/Part:BBa_R0062:Design" class="external text" rel="nofollow">design</A> doesn't say so?
</LI><LI> Are they using a difference <I>Vibrio Fischeri</I> strain for both?
</LI><LI>A sequence <A href="http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?tool=portal&amp;db=protein&amp;term=&amp;query_key=1&amp;dopt=gp&amp;dispmax=20&amp;page=1&amp;qty=1&amp;WebEnv=0NZu2wco5pgt4AFlKGBZSBuHz1xTMr4TRAfhLUXMT0Mg3JomrvvDqnSCvcYGDYXw1omCOSTlDgeb-UU%40264C254084F93770_0012SID&amp;WebEnvRq=1" class="external text" rel="nofollow">similar to LuxR</A> exists in the <I>E. coli</I> strain we're using. Don't know if it'll affect anything.
</LI></OL><H4><SPAN class="mw-headline" id="References_2"> References </SPAN></H4><UL><LI> McMillen, D. et al (2002) &quot;<A href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=117365" class="external text" rel="nofollow">Synchronizing genetic relaxation oscillators by intercell signalling</A>&quot;, Proc Natl Acad Sci USA, Vol. 22, No. 3, pp. 679-684.
</LI><LI> Same paper as the main paper from the oscillator section.
</LI></UL><H3><SPAN class="mw-headline" id="The_Tuner"> The Tuner </SPAN></H3><P>A way to control the oscillator's time period.
</P><H4><SPAN class="mw-headline" id="Components_3"> Components </SPAN></H4><UL><LI><A href="#pLAC">pLAC</A></LI><LI><A href="#CIIIC">CIIIC</A></LI></UL><H4><SPAN class="mw-headline" id="Problems_3"> Problems  </SPAN></H4><OL><LI> The CIIIC protein does not appear in any other papers (nor does it appear on pdb for that matter, but then again CIII doesn't either). So we have to believe that one paper.
</LI><LI> By synthesizing the CIIIC protein from DNA, we have to add the start codon ATG, resulting in an extra Methionine at the start. This may affect our experiments.
</LI></OL><H4><SPAN class="mw-headline" id="References_3"> References </SPAN></H4><UL><LI> Hadler, S. et al (2007) <A href="http://www.ncbi.nlm.nih.gov/pubmed/17890311" class="external text" rel="nofollow">Probing the Anitprotease Activity of lCIII, an Inhibitor of the Escherichia coli Metalloprotease HflB (FtsH)</A>&quot;, Journal of Bacteriology, Vol. 189, No. 22, pp. 8130-8138.
</LI><LI> The only paper about the experimental results of CIIIC. Also has a lot of experimental values (such as ratios) that can be used.
</LI></UL><LI> Herman, C. et al (1997) &quot;<A href="http://jb.asm.org/cgi/content/abstract/179/2/358" class="external text" rel="nofollow">The HflB Protease of Escherichia coli Degrades Its Inhibitor λcIII</A>&quot;, Journal of Bacteriology, Vol. 179, No. 2, pp. 358-363.
</LI><LI> Also has a lot of experimental values that could be used.
</LI><H2><SPAN class="mw-headline" id="Sequences"> Sequences </SPAN></H2><P>This section lists the sequences of promoters and genes.
</P><H3><SPAN class="mw-headline" id="Promoters"> Promoters </SPAN></H3><P>The list of promoters used.
</P><H4><SPAN class="mw-headline" id="pRE"> pRE </SPAN></H4><UL><LI> few hundred bp upstream from the CI gene.
</LI><LI> 40 to 60bp long.
</LI><LI> Appropriate Literature
</LI><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/14872063?ordinalpos=3&amp;itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_RVDocSum" class="external text" rel="nofollow">Interactions among CII protein, RNA polymerase adn λ pRE promoter: contacts between RNA polymerase and the -35 region of pRE are identical in the presence and absence of CII protein</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/2953648?ordinalpos=13&amp;itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_RVDocSum" class="external text" rel="nofollow">Mutations that improve the pRE promoter of coliphage lambda</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/2953649?ordinalpos=12&amp;itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_RVDocSum" class="external text" rel="nofollow">Cross-specificities between cII-like proteins and pRE-like promoters of lambdoid bacteriophages</A></LI></UL><PRE>complement(38391-&gt;38340) on lambda phage.
AGAGCCTCGTTGCGTTTGTTTGCACGAACCATATGTAAGTATTTCCTT|AGAT (52bp)
                                                +----  transcription start site ----&gt;
so what we want:
AGAGCCTCGTTGCGTTTGTTTGCACGAACCATATGTAAGTATTTCCTT (48bp)
</PRE><PRE>pRE Overlaps with CII (but in the other direction)
                       ATGGTTCGTGCAAACAAACGCAACGAGGCTCTACGAATCGAGAGTGCGTTG--&gt; CII cont'
&lt;--ATCTAAGGAAATACTTACATATGGTTCGTGCAAACAAACGCAACGAGGCTCT (&lt;--pRE)
</PRE><PRE>For reference, <A href="http://www.ncbi.nlm.nih.gov/sites/entrez?db=gene&amp;cmd=Retrieve&amp;dopt=full_report&amp;list_uids=3827059" class="external text" rel="nofollow">CI</A>: 37940-37227
</PRE><PRE>Primers:
AGAGCCTCGTTGCGTTT (len:17, tm:55deg, GC%:53)
AAGGAAATACTTACATATGGTTCGTG (len:26, tm:55deg, GC%:35)

Primer used regions:
<FONT color="red">AGAGCCTCGTTGCGTTT</FONT>GTTTG<FONT color="blue">CACGAACCATATGTAAGTATTTCCTT</FONT></PRE><P>There is only 5bp not included in the primer. Feels kinda weird.
</P><H4><SPAN class="mw-headline" id="pLUX"> pLUX </SPAN></H4><UL><LI><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_R0062" class="external text" rel="nofollow">Part:BBa_R0062-luxR &amp; HSL regulated  lux pR</A></LI></UL><PRE>
acctgtaggatcgtacaggtttacgcaagaaaatggtttgttatagtcgaataaa(55 bp)
</PRE><H4><SPAN class="mw-headline" id="pLac"> pLac </SPAN></H4><P><A href="http://partsregistry.org/Part:BBa_R0010" class="external text" rel="nofollow">BBa_R0010</A> (200bp)
</P><H3><SPAN class="mw-headline" id="Genes"> Genes </SPAN></H3><P>The list of genes used.
</P><H4><SPAN class="mw-headline" id="CII"> CII </SPAN></H4><UL><LI> A λ phage gene.
</LI><LI><A href="http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?val=NP_040630.1" class="external text" rel="nofollow">NCBI protein</A>: 97aa
</LI><LI><A href="http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?WebEnv=0_IrKu54rUXySHgWXB0EaSRAhFNRuWQc8t_9Pvcu9deRGYI7MtHyfyf2B79C0DiodB6zisjQl5K8vOD%40264C254084F93770_0012SID&amp;db=nucleotide&amp;qty=1&amp;c_start=1&amp;list_uids=NC_001416.1&amp;uids=&amp;dopt=fasta&amp;dispmax=5&amp;sendto=&amp;fmt_mask=0&amp;from=38360&amp;to=38653&amp;extrafeatpresent=1&amp;ef_CDD=8&amp;ef_MGC=16&amp;ef_HPRD=32&amp;ef_STS=64&amp;ef_tRNA=128&amp;ef_microRNA=256&amp;ef_Exon=512" class="external text" rel="nofollow">NCBI nucleotide</A>: 294bp
</LI></UL><PRE><FONT size="4">&gt;gi|9626243:38360-38653 Enterobacteria phage lambda, complete genome
ATGGTTCGTGCAAACAAACGCAACGAGGCTCTACGAATCGAGAGTGCGTTGCTTAACAAAATCGCAATGC
TTGGAACTGAGAAGACAGCGGAAGCTGTGGGCGTTGATAAGTCGCAGATCAGCAGGTGGAAGAGGGACTG
GATTCCAAAGTTCTCAATGCTGCTTGCTGTTCTTGAATGGGGGGTCGTTGACGACGACATGGCTCGATTG
GCGCGACAAGTTGCTGCGATTCTCACCAATAAAAAACGCCCGGCGGCAACCGAGCGTTCTGAACAAATCC
AGATGGAGTTCTGA</FONT></PRE><PRE>
Primers:
ATGGTTCGTGCAAACAAACG (56deg)
tcagaactccatctggatttgt
</PRE><H4><SPAN class="mw-headline" id="CIIIC"> CIIIC </SPAN></H4><UL><LI> CIII (A λ phage gene) after residues 1-13 and 42-54 are cleaved (so only residues 14-41) are left.
</LI><LI> Inhibites the degradation rate HtlB has on CII by binding to HtlB.
</LI><LI> HtlB <B>does not</B> degrade CIIIC (unlike CIII).
</LI><LI><A href="http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?db=protein&amp;id=9626284" class="external text" rel="nofollow">NCBI (CIII) protein</A>:54aa
</LI><LI><A href="http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?WebEnv=0cMBABYtiOTu-KvSSXngm5VVsgrz7RFiY24GdDY0xzrnBMiRgRDlrJjhu6Ybt2Q7bn2aiqpN4WI4qWl%40264C254084F93770_0012SID&amp;db=nucleotide&amp;qty=1&amp;c_start=1&amp;list_uids=NC_001416.1&amp;uids=&amp;dopt=fasta&amp;dispmax=5&amp;sendto=&amp;from=33299&amp;to=33463&amp;strand=on" class="external text" rel="nofollow">NCBI (CIII) gene</A>:165bp
</LI></UL><P>CIII gene:
</P><PRE>&gt;gi|9626243:c33463-33299 Enterobacteria phage lambda, complete genome
ATGCAATATGCCATTGCAGGGTGGCCTGTTGCTGGCTGCCCTTCCGAATCTTTACTTGAACGAATCACCC
GTAAATTACGTGACGGATGGAAACGCCTTATCGACATACTTAATCAGCCAGGAGTCCCAAAGAATGGATC
AAACACTTATGGCTATCCAGACTAA
</PRE><P>Start with 165bp (54 residues + stop codon) then cleave off residues 2-13 (39bp) and 42-54 (39bp), leaving residues 1,14-41,55 (90bp)
</P><P>CIIIC (with added ATG at the start):
</P><PRE>ATG<FONT color="silver">CAATATGCCATTGCAGGGTGGCCTGTTGCTGGCTGC</FONT>CCTTCCGAATCTTTACTTGAACGAATCACCC
GTAAATTACGTGACGGATGGAAACGCCTTATCGACATACTTAATCAGCCAGGAG<FONT color="silver">TCCCAAAGAATGGATC
AAACACTTATGGCTATCCAGAC</FONT>TAA
</PRE><P>Changes to:
</P><PRE>
ATGCCTTCCGAATCTTTACTTGAACGAATCACCC
GTAAATTACGTGACGGATGGAAACGCCTTATCGACATACTTAATCAGCCAGGA
TAA
</PRE><P>The resulting primers are:
</P><PRE>
ATGCCTTCCGAATCTTTACTTG
ttatcctggctgattaagtatgtcg (56deg)
</PRE><H4><SPAN class="mw-headline" id="HtlB"> HtlB </SPAN></H4><UL><LI> An <I>E. coli</I> gene.
</LI><LI> HtlB degrades CII.
</LI><LI><A href="http://biocyc.org/ECOLI/NEW-IMAGE?type=GENE&amp;object=EG11506" class="external text" rel="nofollow">Ecocyc</A>: 1935bp.
</LI><LI> No extra EcoRI,PstI,SpeI,XbaI in gene below.
</LI></UL><PRE>&gt;EG11506 hflB (complement(3324957..3323023)) E. coli
atgGCGAAAA ACCTAATACT CTGGCTGGTC ATTGCCGTTG TGCTGATGTC AGTATTCCAG
AGCTTTGGGC CCAGCGAGTC TAATGGCCGT AAGGTGGATT ACTCTACCTT CCTACAAGAG
GTCAATAACG ACCAGGTTCG TGAAGCGCGT ATCAACGGAC GTGAAATCAA CGTTACCAAG
AAAGATAGTA ACCGTTATAC CACTTACATT CCGGTTCAGG ATCCGAAATT ACTGGATAAC
CTGTTGACCA AGAACGTCAA GGTTGTCGGT GAACCGCCTG AAGAACCAAG CCTGCTGGCT
TCTATCTTCA TCTCCTGGTT CCCGATGCTG TTGCTGATTG GTGTCTGGAT CTTCTTCATG
CGTCAAATGC AGGGCGGCGG TGGCAAAGGT GCCATGTCGT TTGGTAAGAG CAAAGCGCGC
ATGCTGACGG AAGATCAGAT CAAAACGACC TTTGCTGACG TTGCGGGCTG CGACGAAGCA
AAAGAAGAAG TTGCTGAACT GGTTGAGTAT CTGCGCGAGC CGAGCCGCTT CCAGAAACTC
GGCGGTAAGA TCCCGAAAGG CGTCTTGATG GTCGGTCCTC CGGGTACCGG TAAAACGCTG
CTGGCGAAAG CGATTGCAGG CGAAGCGAAA GTTCCGTTCT TTACTATCTC CGGTTCTGAC
TTCGTAGAAA TGTTCGTCGG TGTGGGTGCA TCCCGTGTTC GTGACATGTT CGAACAGGCG
AAGAAAGCGG CACCGTGCAT CATCTTTATC GATGAAATCG ACGCCGTAGG CCGCCAGCGT
GGCGCTGGTC TGGGCGGTGG TCACGATGAA CGTGAACAGA CTCTGAACCA GATGCTGGTT
GAGATGGATG GCTTCGAAGG TAACGAAGGT ATCATCGTTA TCGCCGCGAC TAACCGTCCG
GACGTTCTCG ACCCGGCCCT GCTGCGTCCT GGCCGTTTCG ACCGTCAGGT TGTGGTCGGC
TTGCCAGATG TTCGCGGTCG TGAGCAGATC CTGAAAGTTC ACATGCGTCG CGTACCATTG
GCACCCGATA TCGACGCGGC AATCATTGCC CGTGGTACTC CTGGTTTCTC CGGTGCTGAC
CTGGCGAACC TGGTGAACGA AGCGGCACTG TTCGCTGCTC GTGGCAACAA ACGCGTTGTG
TCGATGGTTG AGTTCGAGAA AGCGAAAGAC AAAATCATGA TGGGTGCGGA ACGTCGCTCC
ATGGTGATGA CGGAAGCGCA GAAAGAATCG ACGGCTTACC ACGAAGCGGG TCATGCGATT
ATCGGTCGCC TGGTGCCGGA ACACGATCCG GTGCACAAAG TGACGATTAT CCCACGCGGT
CGTGCGCTGG GTGTGACTTT CTTCTTGCCT GAGGGCGACG CAATCAGCGC CAGCCGTCAG
AAACTGGAAA GCCAGATTTC TACGCTGTAC GGTGGTCGTC TGGCAGAAGA GATCATCTAC
GGGCCGGAAC ATGTATCTAC CGGTGCGTCC AACGATATTA AAGTTGCGAC CAACCTGGCA
CGTAACATGG TGACTCAGTG GGGCTTCTCT GAGAAATTGG GTCCACTGCT GTACGCGGAA
GAAGAAGGTG AAGTGTTCCT CGGCCGTAGC GTAGCGAAAG CGAAACATAT GTCCGATGAA
ACTGCACGTA TCATCGACCA GGAAGTGAAA GCACTGATTG AGCGTAACTA TAATCGTGCG
CGTCAGCTTC TGACCGACAA TATGGATATT CTGCATGCGA TGAAAGATGC TCTCATGAAA
TATGAGACTA TCGACGCACC GCAGATTGAT GACCTGATGG CACGTCGCGA TGTACGTCCG
CCAGCGGGCT GGGAAGAACC AGGCGCTTCT AACAATTCTG GCGACAATGG TAGTCCAAAG
GCTCCTCGTC CGGTTGATGA ACCGCGTACG CCGAACCCGG GTAACACCAT GTCAGAGCAG
TTAGGCGACA AGTAA
</PRE><PRE>
HtlB primers:
atgGCGAAAAACCTAATACTCTG
ttacttgtcgcctaactgct
</PRE><H4><SPAN class="mw-headline" id="LuxI"> LuxI </SPAN></H4><UL><LI> A <I>Vibrio Fischeri</I> gene.
</LI><LI><A href="http://biocyc.org/VFIS312309/NEW-IMAGE?type=GENE&amp;object=VFA0924" class="external text" rel="nofollow">Biocyc</A>: 573bp
</LI><LI> Biobrick available
</LI><LI><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J37034" class="external text" rel="nofollow">BBa_J37034</A>: LuxI + GFP. We could use this if we can extract the RBS+LuxI part.
</LI></UL><PRE>&gt;VFA0924 VFA0924 (complement(1050725..1050153)) V. fischeri chromosome es114 ii
ATGataaaaa aatcggactt tttgggcatt ccatcagagg agtatagagg tattcttagt
cttcgttatc aggtatttaa acgaagactg gagtgggact tggtaagtga ggataatctt
gaatcagatg aatatgataa ctcaaatgca gaatatattt atgcttgtga tgatgcggaa
gaggtaaatg gctgttggcg tttgttacct acaacgggtg attacatgtt aaaaactgtt
tttcctgaat tgctcggaga tcaagtagcc ccaagagatc caaatatagt cgaattaagc
cgttttgctg tgggaaaaaa tagctcaaaa ataaataact ctgctagtga aataacaatg
aaattgtttc aagctatata taaacacgca gttagtcaag gtattacaga atatgtaaca
gtaacatcaa tagcaataga gcgatttctg aaacgtatta aagttccttg tcatcgcatt
ggtgataagg agattcattt attaggtaat actagatctg ttgtattgtc tatgcctatt
aatgatcagt ttagaaaagc tgtatcaaat taa
</PRE><PRE>
LuxI primers:
ATGataaaaaaatcggactttttggg
ttaatttgatacagcttttctaaactgatc (length=30 too long?)
</PRE><H4><SPAN class="mw-headline" id="LuxR"> LuxR </SPAN></H4><UL><LI> A <I>Vibrio Fischeri</I> gene.
</LI><LI><A href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=gene&amp;Cmd=ShowDetailView&amp;TermToSearch=3280300&amp;ordinalpos=1&amp;itool=EntrezSystem2.PEntrez.Gene.Gene_ResultsPanel.Gene_RVDocSum" class="external text" rel="nofollow">NCBI</A>: 753bp.
</LI><LI> Biobrick available
</LI><LI><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J37019" class="external text" rel="nofollow">BBa_J37019</A>: Promoter + LuxR. 862bp
</LI></UL><PRE>&gt;VFA0925 VFA0925 1050941..1051693 V. fischeri chromosome es114 ii
ATGaacatta aaaatataaa tgctaatgag aagataattg ataaaattaa aacttgtaat
aataataaag atattaatca atgtttatct gaaatagcaa agataataca ttgtgaatat
tacctattcg ctattatcta tcctcactca ataattaaac ctgatgtttc aattatagat
aattaccctg aaaaatggcg taaatattat gatgatgccg gactactaga atatgaccct
gtagtcgatt actctaagtc ccatcattca ccaattaatt ggaacgtatt cgaaaaaaaa
acaataaaaa aagagtctcc gaatgtaata aaagaagcac aggaatcggg actcattact
ggatttagct ttccaattca tactgcaagt aatggttttg gaatgctcag ttttgctcat
tcagataaag atatttatac tgacagttta tttttacacg ctagtacaaa tgtaccatta
atgcttcctt ctttagtcga taattatcaa aaaataaata cgacacgtaa aaagtcagat
tctattttaa caaaaagaga aaaagaatgc ttagcgtggg cgagtgaagg aaaaagtaca
tgggatattt caaaaatact tggctgcagt gagcgtactg tcacttttca tttaaccaat
actcaaatga aactcaatac aactaaccgc tgccaaagta tttctaaagc aattttaact
ggcgccatta attgtccata ccttaaaaat taa
</PRE><PRE>
LuxR primers:
ATGaacattaaaaatataaatgctaatgagaaga (length=34 too long?)
ttaatttttaaggtatggacaattaatggc (length=30 too long?)
</PRE><H2><SPAN class="mw-headline" id="Testing"> Testing </SPAN></H2><P>How we plan to test our parts when we've finished constructed them.
</P><H3><SPAN class="mw-headline" id="Promoter_Testing"> Promoter Testing </SPAN></H3><P>We are attempting to establish a &quot;inducible promoter standard,&quot; much like the repressible promoter standard, where instead we define the IPTG inducible &quot;standard&quot; pLac promoter as having a value of 1, while all other inducible promoters will be measured in multiples of this standard. Then the value of IPTG can also be varied to test the ratios between the promoters as to whether the ratio is linear, sinusoidal, polynomial, etc.
</P><P>We can test <A href="#pRE">pRE</A> promoter via this standard by connecting the <A href="#CII">|CII</A> gene behind pLac, and connecting a reporter behind pRE.
</P><P>This standard can also be used to test promoters that require a complex to be induced, such as pLux. pLux requires the R+AHL complex to be induced. To test the the strength of pLux, first let all parts of the complex save one to be always existing, while the last one would be controlled by pLac. In this example, LuxR is always expressed and the amount of AHL is determined by pLac.
</P><H3><SPAN class="mw-headline" id="Component_Testing"> Component Testing </SPAN></H3><P>This is the part where we test to see if our component works. The diagram above shows the pathway between the various inputs, and the output (GFP).
</P><H2><SPAN class="mw-headline" id="Modelling"> Modelling </SPAN></H2><H3><SPAN class="mw-headline" id="Oscillator_Modelling_1"> Oscillator Modelling 1 </SPAN></H3><UL><LI><FONT color="red"><B>HtlB</B></FONT> concentration
</LI><LI><FONT color="darkblue"><B>CII</B></FONT> concentration.
</LI><LI> 20sec intervals.
</LI></UL><H4><SPAN class="mw-headline" id="Variables_Used"> Variables Used </SPAN></H4><UL><LI><B>pRE Stength</B>: 0.4 (1 CII protein produces 0.4 CII and HtlB proteins).
</LI><LI><B>H-t<SUB>1/2</SUB></B>: 8mins (random value I picked, its probably not the real value.)
</LI><LI> Half life of HtlB.
</LI><LI> Changing this value just makes the oscllation period longer.
</LI></UL><LI><B>C<SUB>0</SUB></B>: 8mins.
</LI><LI> CII t<SUB>1/2</SUB> with no HtlB present. Based on <A href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=94496" class="external text" rel="nofollow">Shotland2000</A>)
</LI><LI><B>C<SUB>1</SUB></B>: 1min.
</LI><LI> Also based on the same paper.
</LI><LI>CII t<SUB>1/2</SUB> with an equal amount of HtlB is what I assume because they don't say.
</LI><LI><B>Δt</B>: 20sec
</LI><LI> Time segment used for the calculation (can't be too close to the t<SUB>1/2</SUB>'s).
</LI><H4><SPAN class="mw-headline" id="Formulas_used"> Formulas used </SPAN></H4><P>
The HtlB/CII ratio used to calculate the t<SUB>1/2</SUB> at a specific time point:
</P><PRE><SPAN class="bigger">R = [HtlB] / [CII] * [CII] = [HtlB]</SPAN>     ----- (1)

</PRE><P>The last *[CII] has something to do with the [HtlB][CII] --&gt; [CII] equation.
</P><P>Calculation of CII's t<SUB>1/2</SUB> based on the ratio above:
</P><PRE><SPAN class="bigger">C-t<SUB>1/2</SUB> = C<SUB>0</SUB> x (C<SUB>1</SUB>/C<SUB>0</SUB>)<SUP>R</SUP></SPAN>     ----- (2)

</PRE><P>The calculation of amount of substrates remaining w.r.t the half lives:
</P><PRE><SPAN class="bigger">[HtlB] = [HtlB] x 0.5<SUP>(Δt / H-t<SUB>1/2</SUB>)</SUP></SPAN>     ----- (3)

<SPAN class="bigger">[CII] = [CII] x 0.5<SUP>(Δt / C-t<SUB>1/2</SUB>)</SUP></SPAN>     ----- (4)
</PRE><H4><SPAN class="mw-headline" id="Conclusion"> Conclusion </SPAN></H4><P>Fiddling around a bit yields:
</P><TABLE border="1" style="width:90%;border:1px solid gray;"><TBODY><TR><TH colspan="2"> Variable </TH><TH> Affects
</TH></TR><TR><TD rowspan="2"><B>Initial concentration</B> of: </TD><TD> HtlB (higher) </TD><TD rowspan="2"> (longer) startup time
</TD></TR><TR><TD> CII (higher)
</TD></TR><TR><TD rowspan="2"><B>Natural degradation rate</B> of </TD><TD> HtlB (longer) </TD><TD rowspan="2"> (longer) time period
</TD></TR><TR><TD> CII (longer)
</TD></TR><TR><TD colspan="2"><B>pRE strength</B> (higher) </TD><TD> (shorter) time period
</TD></TR><TR><TD colspan="2"><B>HtlB to CII degradation rate</B> (higher) </TD><TD> (lower) global maximum concentration (not counting startup)
</TD></TR></TBODY></TABLE><P>Is a promoter that can be activated by HtlB? because then our calculations would be even more accurate.
</P><H2><SPAN class="mw-headline" id="Experimental_Results"> Experimental Results </SPAN></H2><P>This is a picture of all the constructs, and green constructs are the ones that were successfully created.
</P><H2><SPAN class="mw-headline" id="References_4"> References </SPAN></H2><OL><LI> McMillen, D. <I>et al</I> (2002) &quot;<A href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=117365" class="external text" rel="nofollow">Synchronizing genetic relaxation oscillators by intercell signalling</A>&quot;, Proc Natl Acad Sci USA, Vol. 22, No. 3, pp. 679-684. 
</LI><LI> Hadler, S. <I>et al</I> (2007) <A href="http://www.ncbi.nlm.nih.gov/pubmed/17890311" class="external text" rel="nofollow">Probing the Anitprotease Activity of λCIII, an Inhibitor of the Escherichia coli Metalloprotease HflB (FtsH)</A>&quot;, Journal of Bacteriology, Vol. 189, No. 22, pp. 8130-8138.
</LI></OL><DIV id="bottomlinkbox"><A id="bottomlink" href="https://2008.igem.org/Team:NYMU-Taipei">NYMU-Taipei iGEM 2008 Team</A></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:NYMU-Taipei/Project/Time_Regulation/Reloxilator">http://2008.igem.org/Team:NYMU-Taipei/Project/Time_Regulation/Reloxilator</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NYMU-Taipei/Project/Time_Regulation/Reloxilator" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NYMU-Taipei/Project/Time_Regulation/Reloxilator" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NYMU-Taipei/Project/Time_Regulation/Reloxilator&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NYMU-Taipei/Project/Time_Regulation/Reloxilator&amp;oldid=100136" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>