<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BIT-China_project_Basic_circuits_html skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BIT-China/project Basic circuits.html</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="bannerContainer"><H2> Add a banner to your wiki! </H2><P>You can make the image 980px  by  200px</P><P> Remember to call the file: &quot;<I>Team_BIT-China_banner.jpg</I>&quot; </P></DIV><DIV id="menuContainer"><DIV id="top_nav"><UL><LI><A href="#"><DIV><P>Project</P></DIV></A><SPAN id="nav_second_level"><A href="https://2015.igem.org/Team:BIT-China/Description">Description</A><A href="https://2015.igem.org/Team:BIT-China/Design">Design</A><A href="https://2015.igem.org/Team:BIT-China/project_Basic_circuits.html">Basic circuits</A><A href="https://2015.igem.org/Team:BIT-China/project_Fine-regulation_circuits.html">Fine-regulation circuits</A></SPAN></LI><LI><A href="#"><DIV><P>Modeling</P></DIV></A><SPAN id="nav_second_level"><A href="https://2015.igem.org/Team:BIT-China/modeling_Basic_circuits.html">Basic circuits</A><A href="https://2015.igem.org/Team:BIT-China/modeling_Fine-regulation_circuits.html">Fine-regulation circuits</A></SPAN></LI><LI><A href="#" id="active"><DIV><P>Notebook</P></DIV></A><SPAN id="nav_second_level"><A href="https://2015.igem.org/Team:BIT-China/Notebook">Laboratory Notes</A><A href="https://2015.igem.org/Team:BIT-China/Experiments">Protocols</A></SPAN></LI><LI><A href="#"><DIV><P>Judging</P></DIV></A><SPAN id="nav_second_level"><A href="https://2015.igem.org/Team:BIT-China/Parts">Parts Submission</A><A href="https://2015.igem.org/Team:BIT-China/Achievements">Achievements</A></SPAN></LI><LI><A href=""><DIV id="active"><P>Team</P></DIV></A><SPAN id="nav_second_level"><A href="https://2015.igem.org/Team:BIT-China/Team">Team</A><A href="https://2015.igem.org/Team:BIT-China/gallery">Gallery</A></SPAN></LI><LI><A href=""><DIV><P>Practice</P></DIV></A><SPAN id="nav_second_level"><A href="https://2015.igem.org/Team:BIT-China/Practices">Human Practice</A><A href="https://2015.igem.org/Team:BIT-China/Collaborations">Collaboration </A></SPAN></LI><LI><A href=""><DIV><P>Safety</P></DIV></A><SPAN id="nav_second_level"><A href="https://2015.igem.org/Team:BIT-China/Safety">Safety</A><A href="https://2015.igem.org/Team:BIT-China/Questions_and_answers">Q/A</A></SPAN></LI><LI><A href="#"><DIV><P>Attributions</P></DIV></A><SPAN id="nav_second_level" style="margin-left:-100px;"><A href="https://2015.igem.org/Team:BIT-China/Attributions">Attributions</A><A href="https://2015.igem.org/Team:BIT-China/Sponsors">Sponsors</A></SPAN></LI></UL></DIV></DIV><DIV id="contentContainer"><DIV class="container" id="container_Resistance_System"><DIV id="Resistance System" class="project"><H2>○ Introduction</H2><P>
				Normally, bacteria cannot survive in extremely acidic and alkaline environment. So before producing acid or alkali, the bacteria should be able to tolerate acidic or alkaline environment. Two devices are applied to achieve it.
				
				The first is alkali-resistance device, consisting of Nha protein family.
				
				Nha is a family containing a number of bacterial sodium-proton anti-porters. These are integral membrane proteins that catalyze the exchange of proton for sodium in a manner highly dependent on the pH. To resist different pH, we select two genes, <I>NhaA</I> and <I>NhaB</I>, from this family.
				<I>NhaA</I> is the archetypal Na<SUP>+</SUP>/H<SUP>+</SUP> antiporter and the only member of the family that is required by <I>E.coli</I> for survival under alkaline stress<SUP>[1,2]</SUP>. It is a membrane protein consisting of 388 residues<SUP>[3]</SUP>. <I>NhaA</I> excretes Na<SUP>+</SUP> from the cytoplasm using the energy from the cotransport of protons down their electrochemical gradient into the cell, with a characteristic electrogenic stoichiometry of two protons to one Na<SUP>+</SUP> (Fig.1)<SUP>[4,5]</SUP>. Like many other Na<SUP>+</SUP>/H<SUP>+</SUP> antiporters, NhaA  is regulated by pH. It is essentially inactive below pH 7.0, and the Na<SUP>+</SUP> efflux rate increases by three orders of magnitude between pH 6.5 and 8.5<SUP>[6]</SUP>.</P><DIV style="width:100%; heigth:auto"><DIV style="float:left; width:45%;margin-left:5% "><P style="text-align:center; margin-left:-65px;">Fig.1 The mechanism of NhaA</P></DIV><DIV style="float:left;width:45%;"><P style="text-align:center; margin-left:-65px;">Fig.2 The mechanism of NhaB</P></DIV></DIV><P style="clear:left;">
				The <I>NhaB</I> gene encodes Na<SUP>+</SUP>/H<SUP>+</SUP> antiporter catalyzing the exchange of 3 H<SUP>+</SUP> per 2 Na<SUP>+</SUP>(Fig.2)<SUP>[7]</SUP>. NhaB has a high affinity for sodium. In the absence of <I>NhaA</I>, <I>NhaB</I> confers a certain tolerance to Na<SUP>+</SUP> which decreases with increasing pH. Essential for regulation of intracellular pH under alkaline conditions, NhaB is crucial when the level of NhaA activity is limiting, when NhaA is not sufficiently induced, and/or when NhaA is not activated<SUP>[8]</SUP>. Unlike NhaA, the activity of NhaB shows no dependence on pH in the range 6.4~8.3. The affinity of NhaB to Na<SUP>+</SUP> (Km = 40-70μM) is higher than that of NhaA<SUP>[9]</SUP>.
			</P><H2>○ Design</H2><P>
				The gene circuits of the alkali-resistance device is shown in Fig.3. This device consists of two genes and two promoters. The strong constitutive promoter J23119 is used in our gene circuits. In order to resist alkaline stress, we chose two of Nha sodium ion-proton antiporter proteins referred before to construct device one. NhaA could make sure <I>E.coli</I>’s survival under alkaline pH. NhaB regulates intracellular pH when NhaA is limited. But NhaB is weakly pH-dependent, so a alkali-induced promoter J23119 is necessary. The construction has been finished .
			</P><P style="text-align:center;">Fig.3 The gene circuits of alkali resistence device.</P><H2>○ Results</H2><P>
				Firstly, we cloned the genes NhaA and NhaB from the genome of <I>Escherichia coli str. K-12.</I> Then, the standard part <I>NhaA</I> (BBa_K1675000) and <I>NhaB</I> (BBa_K1675001) were assembled together. The agarose gel electrophoresis analysis of <I>NhaA</I> and <I>NhaB</I> is shown in Fig.4. The construction results of JBA(J23119+B0034+<I>NhaA</I>) and JBB(J23119+B0034+<I>NhaB</I>) are shown in Fig.5 and Fig.6.
				Finally, the standard part JBA+JBB was successfully constructed through 3A assembly. The agarose gel electrophoresis analysis is shown in Fig.7.
			</P><P>
				 After finishing the construction of the standard parts, we measured the OD<SUB>600</SUB> and the intensity of proteins expressed to test the efficacy of our functional genes. The genes <I>NhaA</I> and <I>NhaB</I> encode membrane proteins, we inserted these two genes into plasmid pET28a separately and transformed them into <I>BL21(DE3)</I>. The results weren’t satisfying. To solve this, we transformed the recombinant plasmid into <I>E.coli C41</I> and <I>C43</I><SUP>[10]</SUP>. Luckily, the overexpression was successful this time. The result of SDS-PAGE is shown in Fig.8.</P><P style="text-align:center;">Fig.8 The SDS-PAGE of <I>NhaA</I> and <I>NhaB</I> express in <I>E.coli C41</I> and <I>C43</I></P><P style="text-align:center;">(1: <I>NhaA</I> in <I>C41</I> 2: <I>NhaA</I> in <I>C43</I> 3: <I>NhaB</I> in <I>C41</I> 4: <I>NhaB</I> in <I>C43</I>
5: control (sediment) 6: control (superaantant) 7: control (Whole cells))
			</P><H2>○ Introduction</H2><P>
			Naturally, the bacteria have developed several kinds of acid resistance system to ensure the survival under extremely acidic environment <SUP>[11]</SUP>. <I>GlsA</I> is a newly characterized functional gene in <I>E.coli</I> which encodes an acid-activated glutaminase <SUP>[12]</SUP>.
			This kind of glutaminase is able to catalyze a reaction in which the glutamine (Gln) and water will be transformed into glutamate (Glu) and gaseous ammonia <SUP>[13]</SUP>. The free gaseous ammonia will consume the proton (H<SUP>+</SUP>) and increase the intracellular pH (Fig.9). The robust glutaminase activity only exists at pH 6.0 or lower. The highest activity is obtained at pH 4.0, followed by pH 5.0 and 6.0. In contrast, GlsA is not activated at pH 7.0 and 8.0 <SUP>[14]</SUP>. The dependence on pH makes it an effective tool to respond when necessary.
		</P><P style="text-align:center;">Fig.9 The mechanism of GlsA</P><P>With the presence of extracellular glutamine (Gln), the membrane amino acid antiporter GadC will transport Gln into the cytoplasm, subsequently the glutaminase encoded by <I>GlsA</I> will transform Gln to Glu (glutamate) and release gaseous ammonia. Besides, the production glutamate is able to anticipate in a decarboxylation process led by decarboxylases <A href="#gadA">GadA</A>, and the latter process will also consume H<SUP>+</SUP>. More details about <I>GadA</I> wil be discussed in  <A href="project_Fine-regulation_circuits.html">fine-regulation circuits</A>.</P><H2>○ Design</H2><P>
			The functional gene <I>GlsA</I> is used for acid resistance in our project and is able to regulate the intracellular pH when the environment turns to acidic. It has been reported to protect <I>E.coli</I> from extremely acidic environment so we came up with the gene circuit of acid resistance part (Fig.10). A strong constitutive promoter J23119 controls the transcription of <I>GlsA</I>. The function of glsA will be activated under acidic environment and it will play a critical role in ensuring bacteria’s survival with the anticipation of glutamine (Gln). The activity of GlsA will be repressed when the environmental pH is below 6.0. The dependence on pH makes it a suitable tool to respond necessarily.
		</P><P style="text-align:center;">Fig.10 The gene circuits of acid resistance device</P><H2>○ Experiment Results </H2><P> Firstly, we cloned the gene <I>GlsA</I> through PCR from the genome of <I>E.coli str.</I><I>K-12</I>, and constructed it on the vector pSB1C3 as a standard part(Fig.11).</P><P style="text-align:center;">Fig.11 The construction result of <I>GlsA</I></P><P>
			Adding sequences through designing primers, the promoter J23119, strong RBS B0034 and our functional gene <I>GlsA</I> are combined together. Then we transformed it into <I>E.coli BMTOP10</I>, and selected positive strain from them (Fig.12). We have measured the sequence of No.10 and it turns out to be completely right.
		</P><P style="text-align:center;">Fig.12 The construction result of JBG (J23119+B0034+<I>GlsA</I>)</P><P>
			After the construction of acid-resistance circuit, we measured the OD<SUB>600</SUB> under different pH to test the efficacy of acid resistance. The growth curve is shown in Fig.13
		</P><P style="text-align:center;">Fig.13 The growth curve of experimental group(JBG) and control</P><P>
			The difference between the testing group and the control group is not evident. We employed a strong promoter T7 to check whether the protein has been expressed. The functional gene <I>GlsA</I> was constructed on plasmid pET28a and the plasmid was transformed into <I>BL21(DE3)</I>. 0.5% IPTG was added to induce the expression of the protein. The following is the picture of SDS-PAGE (Fig.14). It shows that our target protein has been expressed successfully.
		</P><P style="width:40%;margin-left:30%;">
			Fig.14 The SDS-PAGE of GlsA and pET28a
		</P><H2 style="float:left;margin-left:10px;clear:right;">Asr promoter</H2><P style="clear:left;">
				Asr promoter belongs to <I>asr</I> gene encoding acid-shock RNA. It is a pH-responsive promoter native to <I>E.coli</I> which induces transcription in acidic environment from pH 4.0 ~ 5.0 , and shows low activity at a neutral pH<SUP>[15]</SUP>. This year, we chose asr promoter as the base of pH regulation system. 
			</P><H2 style="float:left;margin-left:10px;clear:right;"><I>gadA</I></H2><H2 style="clear:left;">○ Introduction</H2><P>
				Neutral environment is the optimal condition for most microorganisms. For the sake of regulating the acidic environment, we need to construct an alkali production circuit. Our project concentrates on the gad system in <I>Escherichia coli str. K-12.</I><I>GadA</I>, one of a functional genes for producing alkali, is employed in our circuit to convert glutamate to GABA, which is a certain kind of alkaline substance. GadC (glutamate γ-amino butyric acid anti-porter) exchanges extracellular Glutamate and intracellular GABA, whereas gadA and gadB converts glutamate to GABA by removing the α-carboxylate group of glutamate. 
			</P><H2>○ Design</H2><P>
				Therefore, we are able to regulate the external acidic environment through the secretion of GABA. The following picture(Fig.15) shows the entire process of gad system.
				This year, our system relies on the intracellular activity of two isoforms of glutamate decarboxylase, gadA and gadB, which catalyze the proton-consuming conversion of glutamate to γ-amino butyric acid (GABA)<SUP>[16]</SUP>. The constitutive promoter J23119, the ribosome binding site B0034 and the functional gene <I>gadA</I> will be used in our circuit (Fig.16).
			</P><DIV style="width:100%; heigth:auto;"><DIV style="float:left;width:45%;margin-left:5%;"><P style="text-align:center;">Fig.15 The mechanism of gadA</P></DIV><DIV style="float:left; width:35%;"><P style="text-align:center;">Fig.16 Alkali synthesis circuit of <I>gadA</I></P></DIV></DIV><H2 style="clear:left;">○ Results</H2><P>
				 Firstly, we cloned the gene <I>GadA</I> from the genome of <I>Escherichia coli str. K-12.</I>. It is found that there are two restriction sites (<I>EcoRI</I> and <I>PstI</I>) in the sequence of <I>GadA</I>. Thus, one-day step mutation was applied to move the restriction sites. Then, the standard part <I>GadA</I> was constructed successfully.
  				 In order to test the function of <I>GadA</I>, we inserted it into plasmid pET28a. The recombinant plasmid was transformed into <I>BL21(DE3)</I>. The agarose gel electrophoresis analysis below (Fig.17) shows the positive clones of <I>GadA</I>.
			</P><P style="text-align:center;">Fig.17 The positive clones of <I>GadA</I> connected with pET28a</P><P>
				After the construction of the alkali producing system, we measured the changes of pH in bacteria solution as well as the intensity of proteins expressed through SDS-PAGE. So long as OD<SUB>600</SUB> of the bacterium solution reaches 0.6, the bacteria was induced by IPTG under 16℃ for 16h. The following table (Table.1) shows the different pH values between the testing group and the control group. It shows that the gene <I>GadA</I> does work as we expected.
			</P><DIV id="program_table_container"><TABLE id="program_table" cellspacing="0"><TBODY><TR class="program_table_firstline"><TH colspan="3">Table.1 pH value of experimental group and control group</TH></TR><TR class="program_table_firstline"><TH>Bacteria</TH><TH>Ternary parallel pH</TH><TH>pH(average)</TH></TR><TR><TH rowspan="3">pET-28a(IPTG)</TH><TD>8.19</TD><TD rowspan="3">8.21</TD></TR><TR><TD>8.18</TD></TR><TR><TD>8.25</TD></TR><TR><TH rowspan="3">pET-28a+Glu(IPTG)</TH><TD>8.20</TD><TD rowspan="3">8.17</TD></TR><TR><TD>8.18</TD></TR><TR><TD>8.14</TD></TR><TR><TH rowspan="3">GadA(IPTG)</TH><TD>8.39</TD><TD rowspan="3">8.35</TD></TR><TR><TD>8.33</TD></TR><TR><TD>8.38</TD></TR><TR><TH rowspan="3">GadA+Glu(IPTG)</TH><TD>8.24</TD><TD rowspan="3">8.31</TD></TR><TR><TD>8.29</TD></TR><TR><TD>8.39</TD></TR></TBODY></TABLE></DIV><H2 style="float:left;margin-left:10px;clear:right;">P-atp2</H2><P style="clear:left;"> 
			P-atp2 (atpB) is an alkali-induced promoter in <I>Corynebacterium glutamicum</I> located in F<SUB>0</SUB>F<SUB>1</SUB> ATPase operon (Fig.18). Each microorganism in nature has an optimal pH and drastic changes in extracellular pH values trigger a stress response that results in overexpression of certain genes and suppression of others<SUP>[17]</SUP>. The P-atp2 promoter responds to the pH changes from pH 7.0 to pH 9.0, especially at alkaline pH. When the pH value increasing, the expression increased modestly<SUP>[19]</SUP>. It is activated by the alternative sigma factor of the RNA polymerase, whose synthesis would be activated when the bacteria are growing at alkaline pH<SUP>[17][18]</SUP>. In the <I>C.glutamicum</I> genome, there are several kinds of sigma factors, among them, expression of Sigma H and Sigma W respond to the alkaline shock(Fig.19)<SUP>[18][20][21]</SUP>.</P><P style="text-align:center;">Fig.18 F<SUB>0</SUB>F<SUB>1</SUB> operon</P><P style="text-align:center;">Fig.19 SigH responds to alkaline stress</P><P>
			P-atp2, as a kind of alkali-induced promoter, is believed to control the expression of functional genes depending on the extracellular pH. Because of this, we anticipate that the P-atp2 will control the functional gene to expression when the environment is alkaline. Combing these two parts, it would allow adaptive acid to neutralize alkaline. To achieve this goal of a regulation device that can be used to specific conditions, our main aim was to construct P-atp2, then testing and verifying it.

			The promoter P-atp2 is placed in <I>Corynebacterium glutamicum</I>, and we got it by standard methods and the agarose gel electrophoresis was used to analyse this gene. In order to verify the function of P-atp2, we attached it to <I>Lac Z alpha</I>. In a pXMJ19 plasmid, it was used to transfer this sequence. As this has been thoroughly tested in our lab, we designed to put X-gal into four fermenters equipped with P-atp2-B0034-<I>Lac Z alpha</I>, maintained at ±1 units in different pH from 6.0 to 9.0. The pH was controlled by addition of HCl or NaOH. Compared with empty plasmid, experimental group was bluer than the corresponding control group. However, it became bluer and bluer along with the increasing extracellular pH.

			So far, the promoter P-atp2 induced at a range of alkaline pH has been verified. To achieve precise control of the acid production, error-prone was utilized. In this step, we optimized the function of P-atp2, to mutate and filtrate a number of accurate promoters with activity. Despite using EP-PCR, the function of P-atp didn’t change, so we chose those promoters to build a database.


			Overall, we have achieved important success in establishing P-atp2 promoters with different efficiency. And we can use it in plenty of specific alkaline situations.</P><UL class="reference_container"><LI><H2>References<SPAN>[Expand]</SPAN></H2><P>
			    [1]  E. Padan, M. Venturi, Y. Gerchman, N. Dover, Biochim. Biophys. Acta 1505, 144 (2001).
				[2]  E. Padan <I>et al</I>., Biochim. Biophys. Acta 1658, 2 (2004).
				[3]  A. Rothman, E. Padan, S. Schuldiner, J. Biol. Chem. 271, 32288 (1996).
				[4]  D. Taglicht, E. Padan, S. Schuldiner, J. Biol. Chem. 268, 5382 (1993).
				[5]  S. Schuldiner, H. Fishkes, Biochemistry 17, 706 (1978).
				[6]  Dudu A, Abraham R, Maral B, <I>et al</I>. <I>NhaA</I> Na<SUP>+</SUP>/H<SUP>+</SUP> Antiporter Mutants That Hardly React to the Membrane Potential[J]. Plos One, 2014, 9(4):e93200.
				[7]  Pinner, E., Padan, E. and Schuldiner, S. (1994) J. Biol. Chem. 269, 26274-26279.
				[8]  Pinner E ,, Kotler Y ,, Padan E ,, <I>et al</I>. Physiological role of nhaB, a specific Na<SUP>+</SUP>/H<SUP>+</SUP> antiporter in Escherichia coli.[J]. J.biol.chem, 1993, 268(1):1729-1734.
				[9]  Schuldiner, S., and Padan, E. (1992) in Alkali Cation Transport Systems in Procaryotes (Bakker, E., ed) CRC Press, Boca Raton, FL, in press.
				
                                [10]Miroux B, Je. W. Over-production of Proteins in Escherichia coli : Mutant Hosts that Allow Synthesis of some Membrane Proteins and Globular Proteins at High Levels[J]. Journal of Molecular Biology, 1996, 260(3):289-298(10).

				[11] Standish C. Hartman, Glutaminase of Escherichia coli. The Journal of Biological Chemistry,1967.
				[12]  Brown G, Singer A, Proudfoot M, Skarina T, Kim Y, Chang C, Dementieva I, Kuznetsova E, Gonzalez CF, Joachimiak A, Savchenko A, Yakunin AF, Functional and structural characterization of four glutaminases from Escherichia coli and Bacillus subtilis. Biochemistry, 2008, 47:5724-35.
   				[13]  Hersh BM, Farooq FT, Barstad DN, Blankenhorn DL, Slonczewski JL, A glutamate-dependent acid resistance gene in Escherichia coli. J Bacteriol, 1996, 178:3978-81.
				[14]  Lu P, Ma D, Chen Y, Guo Y, Chen GQ, Deng H, Shi Y, L-glutamine provides acid resistance for Escherichia coli through enzymatic release of ammonia. Cell Res, 2013, 23:635-44.
				
				[15]Vaida Sˇeputiene, Domantas Motieju nas, Kestutis Suzˇiede˙lis, Henrik Tomenius, Staffan Normark, O¨ jar Melefors, and Edita Suzˇiede˙liene, Molecular Characterization of the Acid-Inducible asr Gene of Escherichia coli and Its Role in Acid Stress Response, 2003. 

				[16]Angela, Tramonti, Michele, De Canio, Isabel, Delany, <I>et al</I>. Mechanisms of Transcription Activation Exerted by GadX and GadW at the gadA and gadBC Gene Promoters of the Glutamate-Based Acid Resistance System in Escherichia coli[J]. Journal of Bacteriology, 2008, 188(23):8118-8127.

				[17] Barriuso-Iglesias M, Barreiro C, Flechoso F, <I>et al</I>. Transcriptional analysis of the F0F1 ATPase operon of Corynebacterium glutamicum ATCC 13032 reveals strong induction by alkaline pH[J]. Microbiology, 2006, 152(1): 11-21.
[18] Barriuso‐Iglesias M, Barreiro C, Sola‐Landa A, <I>et al</I>. Transcriptional control of the F0F1‐ATP synthase operon of Corynebacterium glutamicum: SigmaH factor binds to its promoter and regulates its expression at different pH values[J]. Microbial biotechnology, 2013, 6(2): 178-188.
[19]Kasimoglu E, Park S J, Malek JTseng C P, <I>et al</I>. TRANSCRIPTIONAL REGULATION OF THE PROTON-TRANSLOCATING ATPASE (ATPIBEFHAGDC) OPERON OF ESCHERICHIA COLI - CONTROL BY CELL GROWTH RATE[J]. Journal of Bacteriology, 1996, 178(19):5563-7.

[20]Barriuso-Iglesias M, Schluesener D, Barreiro C, <I>et al</I>. Response Of The Cytoplasmic And Membrane Proteome Of Corynebacterium Glutamicum Atcc 13032 To Ph Changes[J]. Bmc Microbiology, 2008, 8(24):4643-4652.

[21]Wiegert T ,, Homuth G ,, Versteeg S ,, <I>et al</I>. Alkaline shock induces the Bacillus subtilis sigma(W) regulon.[J]. Molecular Microbiology, 2001, 41(1):59-71.</P></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>