<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BIT_Biology skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BIT/Biology</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGem_BIT</TITLE></P><DIV id="content"><DIV id="bodyContent"><HEADER class="header-frontend"><DIV class="navbar navbar-default navbar-static-top"><DIV class="container"><DIV class="navbar-header"><A class="navbar-brand" href="index.html">Disease <SPAN>Alarm</SPAN></A></DIV><DIV class="navbar-collapse collapse "><UL class="nav navbar-nav"><LI><A href="https://2015.igem.org/Team:BIT">Home</A></LI><LI class="active" class="dropdown "><A href="#" class="dropdown-toggle " data-toggle="dropdown" data-hover="dropdown" data-delay="0" data-close-others="false">Projects ≡</A></LI><LI class="active"><A href="https://2015.igem.org/Team:BIT/Biology">Biology Part</A></LI><LI><A href="https://2015.igem.org/Team:BIT/Hardware">Hardware Part</A></LI></UL><LI><A href="https://2015.igem.org/Team:BIT/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:BIT/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:BIT/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:BIT/Description">Description</A></LI><LI class="dropdown "><A href="#" class="dropdown-toggle " data-toggle="dropdown" data-hover="dropdown" data-delay="0" data-close-others="false">Judging ≡</A></LI><LI><A href="https://2015.igem.org/Team:BIT/Judging_Form">Judging_Form</A></LI><LI><A href="https://2015.igem.org/Team:BIT/Interlab_Study">Interlab Study</A></LI><LI><A href="https://2015.igem.org/Team:BIT/Safety_Form">Safety Form</A></LI><LI><A href="https://2015.igem.org/Team:BIT/Parts_Submission">Parts Submission</A></LI><LI class="dropdown "><A href="#" class="dropdown-toggle " data-toggle="dropdown" data-hover="dropdown" data-delay="0" data-close-others="false">Team ≡</A></LI><LI><A href="https://2015.igem.org/Team:BIT/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:BIT/Acknowledgement">Acknowledgement</A></LI><LI><A href="https://2015.igem.org/Team:BIT/Gallery">Gallery</A></LI></DIV></DIV></DIV></HEADER><SECTION class="bio-bg"><DIV class="container"><DIV class="row"><DIV class="col-lg-4 col-sm-4"><H1>Biology Part</H1></DIV><DIV class="col-lg-8 col-sm-8"><OL class="breadcrumb pull-right"><LI><A href="#">Home</A></LI><LI class="active">Biology Part</LI></OL></DIV></DIV></DIV></SECTION><DIV class="container"><DIV class="row"><DIV class="col-lg-9 "><DIV class="blog-item"><DIV class="row"><DIV class="content"><H1>BACKGROUND</H1><P><SPAN class="dropcap">T</SPAN>here are varieties of methods of disease detection. However, the detection of disease can hardly meet the expectation with the features of high accuracy, short-term, low-cost and user-friendliness. Our project is aiming at making disease detection more instant, convenient and accurate.
<SPAN class="dropcap">S</SPAN>o after thorough investigation, we found that the human body is bound to cause a certain or some large molecular abnormalities in the human body before most of the disease occurs. So we can detect the differences in the molecular level, thus we can predict the possible occurrence of the disease, which becomes a good way to overcome the difficulties above. With social and economic development, people are in the pursuit of material enjoyment at the same time they pay more attention to the health, so the disease has become the primary factor that will threat to human life. But a lot of diseases are not immediate, there will be a certain period of time in the incubation period in general, we cannot know whether we have been sick in the incubation period. When in the incubation period, the contents of some large molecules in the body will be abnormal, and the more close to the onset, the more obvious. So it is feasible and necessary to predict the occurrence of the disease by detecting the contents of some macromolecules in human body.
<SPAN class="dropcap">I</SPAN>n twenty-first Century, the most dangerous diseases are heart disease, cancer and diabetes. The number of People’s death of those three diseases is 26,400,000 (heart diseases’ number is 15,000,000, cancers’ number is 7,600,000, diabetes’ number is 3,800,000). After research, we found that these three kinds of diseases can be adopted to detect the specific molecular content of disease in order to predict early, so that patients can take the corresponding medical measures to prevent, reduce or eliminate the pain. It is very helpful to the health of human beings.
<SPAN class="dropcap">T</SPAN>he detection process of the existing medical equipment is complicated, the time is long, and the cost is high and is not suitable for all of the people. But the product of our project is not limit to time and the venue. The detection time is short and the process is simple that patients can operate by themselves. It can quickly and easily predict the occurrence of diseases, so that it can save more people's lives.
<SPAN class="dropcap">T</SPAN>o sum up, it’s necessary to predict disease early and the applicability of the method used in this project.</P><H1>Design</H1><P><SPAN class="dropcap">I</SPAN>n biological part, we use synthetic biological engineering bacteria, connected with the aptamers, to make engineering bacteria able to detect the macromolecules.


The mechanism is as follows: First, we take the amino acid sequence of the binding site of target macromolecule and the aptamers as the template, and some of which are replaced by some oligopeptides of lower appetency with aptamers, while modifying arabinose. When detecting, we combine the artificial oligopeptides with the aptamers. As the macromolecules get into the measurement system, they will take places of glycopeptide and leave them as dissociated molecules. Finally, these free glycopeptide will enter the transformed engineering bacteria. Inside the upriver bacteria, certain enzyme will digest the oligopeptides that is from glycopeptide so that the arabinose will be released and induce the engineering bacteria expressing LasI protein. The protein can catalyze matrix producing a large number of small molecules PAI1. When PAI1 free diffuse into the downstream bacteria, it will combine with the constitutive expressed LasR protein and form a transcription factor complex. The combination between complex and the pLas promoter accelerate the process of transcription and translation to generate the Rhl1 enzyme, which can catalyze producing the autoinducer PAI2 from system RhIIR. The complex of PAI2 and RhIR protein that is mediated-expressed by double promoter in bacteria can act on pRhl promoter and induce the engineering bacteria to express GFP. At present, given that the research about aptamer of thrombin is relatively mature, the project chooses the thrombin as the target macromolecules.

The sequence of nucleic acid aptamer: H2N-ctatc-AGTCCGTGGTAGGGCAGGTTGGGGTGACT-3'

The sequence of binding site of aptamer and thrombin: IGKHSRTRYERNIEKI

The sequence of artificial designed oligopeptides: TRYERNIEKI

Because the peptide can’t be an inductor to trigger a sensor, we bind a sugar (arabinose) to the peptide. Then the glycopeptide can trigger the arabinose operon, which can be used to construct a fluorescence detector. Binding an arabinose is preferred because it doesn’t have a background level in E.coli, so the detector can be more sensitive and steady. After consulting some papers we know that the combination site of thrombin and its aptamer is from 425 to 438 of the thrombin sequence (IGKHSRTRYERNIEKI). Then we designed two peptides, Tmb1 (TRYERNIEKI) and Tmb2 (IGKHSRTRYERNI). We detected the affinity of Tmb1 and Tmb2 with aptamer respectively. Tmb1 showed a better affinity and we finally choose Tmb1 as the substrate of glycopeptide synthesis.


The experiment of glycopeptide incubation adopts the method of reductive amination. The amino terminal of oligopeptides reacts with the aldehyde group of arabinose and builds carbon-nitrogen double bond, namely imine (also known as Schiff Base). Then we can get the glycopeptide resulting from the reduction of sodium borohydride.

The competitive inhibition experiments are done in order to prove that thrombin can inhibit the combination of the glycopeptide and aptamer by competitive effect. The experiment was divided into three parts: the combination of the thrombin and the aptamer, the combination of the glycopeptide and the aptamer, and the competitive inhibition reaction when the thrombin is added into the complex which is combined with glycopeptide and aptamer. The experimental results are all demonstrated by capillary electrophoresis.

The Combination between Thrombin and Aptamers: According to the research of the existing literature, the research about the combination of thrombin and thrombin is very mature. The thrombin can combine with aptamer specifically, and the binding ability is strong.
The Combination between Glycopeptide and Aptamers: Referring to the amino acid sequence of the binding site between the thrombin and the aptamer, we compound the peptide. When we modify the arabinose into the peptide, because of the different binding site, the new complex we call glycopeptide can still combine with aptamer well.

Competitive Inhibition: Because of the different spatial structure between the peptide and the thrombin, the binding ability of the peptide and the aptamer is weaker than that between the thrombin and the aptamer. So when we add the thrombin into the system where exists the complex combined with glycopeptide and aptamer, the former can inhibit the combination of the glycopeptide and the aptamer, so that it can combine with the aptamer by the competitive effect, and make the glycopeptide become dissociative to be detected well.
The purpose of fluorescence signal amplifier system is to improve the accuracy. The experiment designs two kinds of bacteria that have different functions but in connecting link realizing the function of amplifying the signal of detecting macromolecule. The brief principle is: small molecule inducer PAI1 that is produced by upriver bacteria can activate biological amplifying circuit of downstream bacteria so that the system can realize amplifying fluorescence intensity of trace signal.

The Upriver PAI1 Follower: Function units of the first bacteria derive from simply transformed plasmid from iGEM Kit Plate. The plasmid only includes inducible promoter of arabinose and gene sequence of fluorescent protein (BBa_I746908). We use molecular cloning technique to replace fluorescent protein sequence for the LasI protein sequence. LasI protein can catalyze matrix to produce a large number of small molecules PAI1 and they are the very key substance that can trigger the cascade amplification mechanism inside downstream bacteria.


Downstream Cascade Amplification Unit: When the plenty of PAI1 that come from the upriver bacteria free diffuse into the downstream bacteria, it will combine with the constitutive expressed LasR protein and form a transcription factor complex. The combination between complex and the pLas promoter accelerate the process of transcription and translation to generate the Rhl1 enzyme, which can catalyze producing the autoinducer PAI2 from system RhIIR. The complex of PAI2 and RhIR protein that is mediated-expressed by double promoter in bacteria can act on pRhl promoter and induce the engineering bacteria to express GFP.


The small molecules free transmit between bacteria and the system includes enzymatic reaction for two times and gene transcription and induction for many times during the whole process. Besides, due to the existence of population density, the ultimate detected signal can amplify again. Finally, we can get the result that is multiply amplified.</P><H1>Results</H1><H3>Mass Spectrometry</H3><H5>Figure A Mass chromatogram of CG (with no arabinose added, only peptide). The peak area counts of peptide(441 to 441.5) is 1.2198e6 and that of internal standard(523.5 to 524) is 1.5069e6.</H5><H5>Figure B Mass chromatogram of EG (arabinose:peptide=5:1). The peak area counts of peptide(441 to 441.5) is 1.9649e7 and that of internal standard(523.5 to 524) is 3.4014e7.</H5><P>After peptide and arabinose reacted at 100℃ for 1h, the product was detected by TOF-MS. An internal standard was used to help us measure the concentration of peptide. The calculated reaction rate was 14.7%. But there’s a problem that we didn’t find an obvious peak of target product. We are improving our method and we believe the glycopeptide will be successfully synthesized in the near future. </P><H3>CE</H3><P>We put peptide and aptamer into a sample tube, and cultured for 45min under normal temperature. The peptide can combine with aptamer. </P>

From the chart, the peak value of aptamer appeared at last and the peak value of peptide appeared at about four min. And the peak value of complex appeared at the time different from the aptamer and peptide, so we can conclude that the peptide can combine with aptamer.
We find the dissociation constant of the combination of aptamer and thrombin through literature, so it is the same as that of aptamer and peptide.
Kd =
[P]0---The initial concentration of peptide
[DNA]--- The initial concentration of aptamer
A1---Peak area of aptamer uncombined
A2---Peak area of complex

From the chart, we can get A1=4235, A2=367.7 and [P]0=0.5μM, [DNA]=5Μ. So we can get Kd through calculation. It’s 1.158μM.
we put the thrombin into the complex which is combined with aptamer and peptide, it can replace the peptide, become the complex   which is combined with thrombin and aptamer, and the dissociative peptide. 


From the chart, we should see the peak value of peptide appear at the same time, but because of some problems, we can’t see the peak value of peptide in the complex clearly. But we can sure the peptide has been replaced.
BBa_I746908
Firstly, we tested how the plasmid that contains BBa_I746908(“pBAD-sfGFP”) will influence the growth condition of the bacterium. 
Since the “pBAD-sfGFP” and “pBAD-LasI” has the same promoter, it is justified that we can assess how “pBAD” will promote “sfGFP” by studying the growth curve of the bacterium containing BBa_I746908. What’s more, the sfGFP could be regarded as the report protein to make the study of the growth curve easier.
To gain the growth curve, we use bacterium containing plasmid (BBa_I746908). As the Chloramphenicol concentration in the liquid medium is 102μg/mL, we set the inoculum concentration of the bacterium at 4% along with the 200mL of LB volume.
Next, the liquid medium would be placed in the thermostatic shaker with the temperature set at 37℃. When the study begins, it would be supposed that the OD600 will be tested every 1 hour. In particular, the value is tested every 30 minutes at the first 4 hours to optimize the induction work afterwards. The following graph shows the OD600 changes over the time.




Figure C Growth Curve of BBa_I74698 within 12 hours.

As can be concluded from the chart, in 7 hours after inoculation, the OD600 reaches 0.6, which means the inducing operation could be performed as our next step.

To make data more informative, four different concentrations of arabinose are set to start the inducing test. The concentrations are 0mM, 0.01mM, 0.05mM, 0.1mM separately.When the culture time without arabinose gets to 7h, the inducing plan was carried out as mentioned above. 
In 5 hours after adding the arabinose, it starts to show significant difference in fluorescence between the four sets. The following is about how the fluorescence and OD600 changes over time.

<H5>Figure D Gradirnt Induction for four groups of BBa_I746908 (Flurescence).</H5><H5>Figure E Gradient Induction for four groups of BBa_I746908 (OD600).</H5><H5>Figure F Gradient Induction for four groups of BBa_I746908 (FLU per OD600).</H5>
If we only consider the relationship between the fluorescence at “7+5 hours” and concentration of arabinose, it is evident that there is a linear relationship between the two values as the flowing picture. 

<H5>Figure G Linear regression of FLU with different gradient (BBa_I746908).</H5>
The method can be concluded curtly as: 7 hours culturing, 5 hours inducing with different concentration of the arabinose. It exactly is linear relationship that inspire us to replace “sfGFP” by using “LasI” to go with the amplifier.
<H3>Amplifier</H3>
As to harvest the growth curve of the bacterium containing amplifier, the same protocol is applied. The amplifier is the core part of our biology detection machine. Inspired by the natural group effect existing in the P. Aeruginosa, the circuit now applied is production of optimization and simplifying.
In order to make it clear to us at which point the amplifier growth rate reaches a maximum, we use 200mL liquid medium with 1% inoculum concentration as well as 102μg/mL of Chloramphenicol concentration.
During the whole measurement process, OD600 is tested every 30 minutes and we finally achieve the graph below. 


As can be seen from the picture, when the culture time (without inducer) comes to 7 hours, the OD600 reaches about 0.6, which is a portable value for the maximum growth rate.
We decided to use the bacterium cultured for 7 hours adding with different inducer (PAI1) to make a more detailed study about the induce result.
Three series of gradient have been tried during the study about the amplifier as the graph below shows. The first graph contains four gradients while the latter contains six.


As a result, when the reducing time comes to 5 hours, the fluorescence value of each gradient begins to show significant differences. By a more detailed analysis, we could also find that at the same time point “induce 5 hours”, Fluorescence value of each gradient constitute linear relationship showed in the following graph meanwhile.


<H5>Figure H Linear regression of FLU with different concentration.</H5><H1>Model</H1>

The thrombin can specifically bind with the aptamer. The glycopeptide can also combine with the aptamer, but the combining capacity is weaker than the former. So when we put the thrombin into the complex that is combined with aptamer and glycopeptide, it can replace the glycopeptide, become the complex combined with thrombin and aptamer, and the dissociative glycopeptide. We can detect the concentration of the glycopeptide and get the concentration of the thrombin indirectly.


<H3>Modeling Building</H3>
In reaction chemical formula (1-1), we assume the concentration of AG is known and larger than that of T. When reaction achieves a balance, the concentration of TA and G is the same. So we can use value of the concentration about G to replace value of the concentration about TA. In this way, we can get some new transforms:


In expression (2-4), the concentration of [AG] is known, the concentration of [G] can be detected by the detector. K1 is known by literature, K2 can be detected by the CE experiment. Thus we can get the result of the concentration of T. But one important point is that the concentration of G is not larger than that of AG, so there is a limiting condition.

<H3>Result</H3>

This diagram illustrates the relationship between the macromolecular material and target detection
<H1>Discussion</H1><P><SPAN class="dropcap">T</SPAN>he biological experiment uses thrombin as the test object to test the principle of the experiment. The project will focus more on the detection of more biochemical macromolecules in the future, and focus on the macromolecules associated with human diseases. For now, the research on the aptamer has a history of more than ten years. By literature survey, we have obtained the following small database, which contains 15 kinds of names and sequences of aptamer, as well as the diseases related to the macromolecules.
</P>
In principle, any large molecules have aptamers that have a strong binding capacity, so in the future, we will try hard in the project to achieve the completion of the construction of the aptamer database and the experiments that match the database, thus developing detection products for a variety of specific diseases, finally to fulfil the goal of early detection of the disease set in the initial period of project.
<H3>Reference</H3>
I.Tombelli S, Minunni M, Luzi E, et al. Aptamer-based biosensors for the detection of HIV-1 Tat protein[J]. Bioelectrochemistry, 2005, 67(2): 135-141.

II.Liu Z, Duan J H, Song Y M, et al. Novel HER2 aptamer selectively delivers cytotoxic drug to HER2-positive breast cancer cells in vitro[J]. J Transl Med, 2012, 10(1): 148.
III.Ng E W M, Shima D T, Calias P, et al. Pegaptanib, a targeted anti-VEGF aptamer for ocular vascular disease[J]. Nature reviews drug discovery, 2006, 5(2): 123-132.
IV.G.J. Tong, S.C. Hsiao, Z.M. Carrico, M.B. Francis, Viral capsid DNA aptamer con- jugates as multivalent cell-targeting vehicles, J. Am. Chem. Soc. 131 (2009) 11174–11178. 
V.V. Bagalkot, O.C. Farokhzad, R. Langer, S. Jon, An aptamer–doxorubicin physical conjugate as a novel targeted drug-delivery platform, Angew. Chem. Int. Ed. Engl. 45 (2006) 8149–8152. 
VI.C.S. Ferreira, M.C. Cheung, S. Missailidis, S. Bisland, J. Gariepy, Phototoxic aptamers selectively enter and kill epithelial cancer cells, Nucleic Acids Res. 37 (2009) 866–876. 
VII.T.C. Chu, J.W. Marks 3rd, L.A. Lavery, S. Faulkner, M.G. Rosenblum, A.D. Ellington, M. Levy, Aptamer: toxin conjugates that specifically target prostate tumor cells, Cancer Res. 66 (2006) 5989–5992. 
VIII.Yasun E, Kang H, Erdal H, et al. Cancer cell sensing and therapy using affinity tag-conjugated gold nanorods[J]. Interface focus, 2013, 3(3): 20130006.
IX.Cao Z, Tong R, Mishra A, et al. Reversible Cell‐Specific Drug Delivery with Aptamer‐Functionalized Liposomes[J]. Angewandte Chemie International Edition, 2009, 48(35): 6494-6498.
X.Hicke B J, Marion C, Chang Y F, et al. Tenascin-C aptamers are generated using tumor cells and purified protein[J]. Journal of Biological Chemistry, 2001, 276(52): 48644-48654.


<H1>Notebook</H1><H3>Protocol</H3><H3>Experiment of Glycopeptide Incubation</H3>1. Dissolve 0.66mg peptide in 1mL deionized water.	
2. Compound 6μmol/mL arabinose solution and lactose solution. Adding 0.5mL each solution into two portions of peptide solution. 
3. Adjust the pH of two portions of solution to 5. 
4. Add 2mg sodium cyanoborohydride to each solution. 
5. Magnetic stirring for 20 hours.
<H3>Experiment of Competitive Inhibition</H3><H3>1. Detection of the glycopeptide</H3>

Add 1μL of the glycopeptide solution (the concentration is 0.33mmol/L) to a centrifugal tube with its capacity 200μL and then add 99μL of buffer solution (dilute PB solution whose PH is 7.2 by 10 times) to it. The diluted glycopeptide solution’s concentration is 3.3μmol/L and the tube is labeled “Tube 3”.

Pick two centrifugal tubes as same as the tube above, then add 100μL of the above buffer solution to them and label them as “Tube 1” and “Tube2”.

In addition, pick another centrifugal tube without any solution as a waste-liquid-tube to collect waste liquid.

The four tubes are placed in the capillary electrophoresis apparatus, and the detection parameters are set .In the meanwhile, the condition of peak is detected at the wavelength of 280nm.
<H3>2. Detection of the conjugation of glycopeptide and aptamer</H3>

The concentration of aptamer is 100μM/L. Add 10μL of it to a centrifugal tube with its capacity 200μL, then add 90μL of the buffer solution to the tube to dilute the solution to 10μM/L; after taking 50μL of the solution, it is added with another 50μL of the solution of glycopeptide which has already been diluted to a centrifuge tube the same as the above to incubate at room temperature, and the time for incubation is 45 min; and the tube is labeled as “Tube 3”.

Pick two of the same centrifugal tube with 100 μL of the above buffer solution, and label them as “Tube 1” and “Tube 2”.

In addition, pick another centrifugal tube without any solution as a waste-liquid-tube to collect waste liquid.

The four tubes are placed in the capillary electrophoresis apparatus, and the detection parameters are set .In the meanwhile, the condition of peak is detected at the wavelength of 280nm.

According to the peak area of the peak chart, the concentration of the solution and the formula, the dissociation constant of glycopeptide and aptamer, or K2, is calculated.
<H3>3. Add into thrombin:</H3>

The thrombin solution’s concentration is 0.1mg/ml. Add 50μL of it to the solution of the conjugation of glycopeptide and aptamer. The solution is incubated at the same condition for 45min.

In addition, pick another centrifugal tube without any solution as a waste-liquid-tube to collect waste liquid.

In addition, a same centrifugal tube is not added to the waste liquid of the waste pipe.

The four tubes are placed in the capillary electrophoresis apparatus, and the detection parameters are set .In the meanwhile, the condition of peak is detected at the wavelength of 280nm.
<H3>4. Data calculation:</H3>
By the replacement in Step 4, the concentration of the replaced and dissociative glycopeptide is detected.

We can get the competitive-inhibition-reaction’s equilibrium constant, or K. K is equal to K1/K2 (K=K1/K2). K1 is the dissociation constant of thrombin and ligand. And K2 is the dissociation constant of glycopeptide and aptamer.
<H3>The Experiment of Fluorescence Signal Amplifier System</H3>

1. Transformation Add 5μL plasmid solution (1-16H, 3-21P, 5-17P) into 50μL DH5α solution in a tube. Then put the solution in ice bath for 30 min, then in 42℃ for 45s, then in ice bath for 2 min. Then add 500μL LB liquid medium in the tube. Cultured in 37℃ for 1h. Centrifuged at 6000rmp for 1min. Then concentrated as three times concentration as former. Then inoculate the bacteria on the LB solid medium for 12h to be selected.

2. Bacteria culturing, plasmid miniprep, sequencing After 12h culturing, select a colony on LB solid medium, take a point of the colony into a 5μL LB liquid medium. Culturing in 37℃ for 12h. Then take 4μL solution to do the plasmid miniprep, electrophoresis test and sequencing.

3. Enzyme digestion, ligating &amp; Transformation Enzyme digestion: Use four kinds of restriction endonucleases, which are E.coliI, XbaI, SpeI and PstI , to digest plasmid for 2h in 37℃. SpeI and Pstl were used to digest the carrier of promoter, and the target gene was digested with PstI and XbaI. Subsequent agarose gel electrophoresis was performed to screen the required fragments to recover DNA fragments. The enzyme digestion system includes: 10μL plasmid, 7μL sterile water, 2μL buffer2.1, 2*0.5μL enzyme.

Ligating: let the DNA fragments recovered by gel recovery react with T4 ligase for 30min at 16℃. The ligating system includes: 7.5 microliters of the target gene, 1.5 microliters of carrier, 1 microliters of T4 buffer, 0.4 microliters of T4 ligase.

Transformation: Take 5μL solution to undertake the first step of the transformation.

4. Induction &amp; Validation The bacteria, which were successfully transformed and had the 12-hour culture, received the induction of sugar and were marked by enzyme to undertake the fluorescence detection. The radiated wavelength is 420nm and the received wavelength is 560nm. Different fluorescent curves are obtained by different concentrations and time.
<H1>Lab Note</H1><H3>Experiment of Glycopeptide Incubation May Plan：</H3>

May Plan: look up papers to find a good method to synthesize glycopeptide Sum up: We finally find three methods: 1.Maillard reaction 2. Reductive amination 3.NBS-ACN method

June Plan: Tried Maillard reaction Sum up: We can’t get the peak of predicted product in LC-TOF MS.

July Plan: Tried Reductive amination Sum up: We can get the product peak in LC, but when we induce it to E.coli, it can’t trigger the sensor.
<H3>Experiment of Competitive Inhibition</H3><H3>Apr.2015</H3>

1. The activation of the capillary;

2. Detecting the peak of the thrombin;
<H3>May.2015</H3>

In the initial literature search, we searched for the most suitable aptamer for thrombin, and finally identified three kinds of suitable, two of which were 29 base pairs, which we called Aptamer1 and Aptamer1, and Aptamer1 was made up 5 ‘Biotin, and Aptamer2 was made up 5’NH2C6, another aptamer was 15 base pairs, we called it Aptamer3. Aptamer1 was added into thrombin and incubated for 45min with the same concentrations and the peak was detected at the wavelength of 280nm, then Aptamer2 and Aptamer3 were added into the thrombin and incubated for the same time with the same way. The peak was detected at the wavelength of 280nm. After verification, the three kinds of suitable aptamers can be well combined with thrombin.
<H3>Jun.2015</H3>

According to the sequence of the binding site of thrombin and aptamer, we have changed the sequence of one or two one or two amino acids, and get two kinds of peptides, which are Tmb1 and Tmb2, and we also have detected the two kinds of peptides’ ability of combining with aptamer. We got six arrangements. They were Aptamer2+Tmb1, Aptamer1+Tmb2, Aptamer2+Tmb2, Aptamer3+Tmb1, Aptamer3+Tmb2 and Aptamer1+Tmb1. The final result shows that the effect of Aptamer2+Tmb1 is the best.
<H3>Jul.2015</H3>

Determine the final suitable aptamer and peptide selection and modification of arabinose of oligopeptide, glycopeptides and aptamers were incubated, incubation time still for 45min, the incubation of conjugates were detected, still at a wavelength of 280 nm observed peak, and with separate aptamer, separate glycopeptide peaks of the comparison chart that conjugates are indeed glycopeptides and aptamer conjugates.
<H3>Aug.2015</H3>

We added the thrombin into the complex that is combined with glycopeptide and aptamer, (thrombin concentrations below the peptide aptamer and combined with concentration) and the replacement of detection of glycopeptide concentration. Setting different gradient of thrombin and conducting thrombin displacement experiment of each gradient.
<H3>The Experiment of Fluorescence Signal Amplifier System</H3><H3>May.2015</H3>

We decided arabinose and lactose as our testing factors and then we transformed BBa_K52401 and BBa_1746908 into bacteria, and cultivated them after screening them.
<H3>Jun.2015</H3>

With the experiment going on, some problem we didn’t notice at the beginning appeared---- the detection of offline is so high. So we concentrated our attention to refer to the solution. We looked through the wiki of iGEM team in the past several years and other passages. Finally, we decided to try cascade system.
<H3>Jul.2015</H3>

This month we began to construct the systems we decided in the past month “Promter-lasR-Ter-Plas-RhlI-Ter” (part1) and “Promter-RuR-Ter-PRhl-GFP-Ter” (part2). Finally, part1 was transformed into bacteria, protein lasR and RhlI expressed successfully.
<H3>Aug.2015</H3>

Now we focus on constructing part 2 of cascade system. Because of the deficiency of the RuR protein terminator, so we choose a new system, now this part is being verification.</DIV></DIV></DIV></DIV></DIV></DIV><FOOTER class="footer"><DIV class="container"><DIV class="row"><DIV class="col-lg-3 col-sm-3"><H1>contact info</H1><ADDRESS><P>Address: Beijing Institute of Technology, No. 5 South Zhong Guan Cun Street, 
Haidian Beijing 100081, P. R. China </P><P>Twitter : @igem_BIT</P><P>Sina Weibo : @igem_BIT</P><P>Website : <A href="javascript:;">http://www.bit.edu.cn</A></P></ADDRESS></DIV><DIV class="col-lg-3 col-sm-3 col-lg-offset-1"><H1>stay connected</H1></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>