<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BIT-China_Project_Resistance_Subsystem skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BIT-China/Project Resistance Subsystem</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="bannerContainer"><H2> Add a banner to your wiki! </H2><P>You can make the image 980px  by  200px</P><P> Remember to call the file: &quot;<I>Team_BIT-China_banner.jpg</I>&quot; </P></DIV><DIV id="menuContainer"><DIV id="top_nav"><UL><LI><A href="#"><DIV><P>Project</P></DIV></A><SPAN id="nav_second_level"><A href="https://2015.igem.org/Team:BIT-China/Description">Description</A><A href="https://2015.igem.org/Team:BIT-China/Design">Design</A><A href="https://2015.igem.org/Team:BIT-China/project_Basic_circuits.html">Basic circuits</A><A href="https://2015.igem.org/Team:BIT-China/project_Fine-regulation_circuits.html">Fine-regulation circuits</A></SPAN></LI><LI><A href="#"><DIV><P>Modeling</P></DIV></A><SPAN id="nav_second_level"><A href="https://2015.igem.org/Team:BIT-China/modeling_Basic_circuits.html">Basic circuits</A><A href="https://2015.igem.org/Team:BIT-China/modeling_Fine-regulation_circuits.html">Fine-regulation circuits</A></SPAN></LI><LI><A href="#" id="active"><DIV><P>Notebook</P></DIV></A><SPAN id="nav_second_level"><A href="https://2015.igem.org/Team:BIT-China/Notebook">Laboratory Notes</A><A href="https://2015.igem.org/Team:BIT-China/Experiments">Protocols</A></SPAN></LI><LI><A href="#"><DIV><P>Judging</P></DIV></A><SPAN id="nav_second_level"><A href="https://2015.igem.org/Team:BIT-China/Parts">Parts Submission</A><A href="https://2015.igem.org/Team:BIT-China/Achievements">Achievements</A></SPAN></LI><LI><A href=""><DIV id="active"><P>Team</P></DIV></A><SPAN id="nav_second_level"><A href="https://2015.igem.org/Team:BIT-China/Team">Team</A><A href="https://2015.igem.org/Team:BIT-China/gallery">Gallery</A></SPAN></LI><LI><A href=""><DIV><P>Practice</P></DIV></A><SPAN id="nav_second_level"><A href="https://2015.igem.org/Team:BIT-China/Practices">Human Practice</A><A href="https://2015.igem.org/Team:BIT-China/Collaborations">Collaboration </A></SPAN></LI><LI><A href=""><DIV><P>Safety</P></DIV></A><SPAN id="nav_second_level"><A href="https://2015.igem.org/Team:BIT-China/Safety">Safety</A><A href="https://2015.igem.org/Team:BIT-China/Questions_and_answers">Q/A</A></SPAN></LI><LI><A href="#"><DIV><P>Attributions</P></DIV></A><SPAN id="nav_second_level" style="margin-left:-100px;"><A href="https://2015.igem.org/Team:BIT-China/Attributions">Attributions</A><A href="https://2015.igem.org/Team:BIT-China/Sponsors">Sponsors</A></SPAN></LI></UL></DIV></DIV><DIV id="contentContainer"><DIV class="container" id="container_Resistance_System"><DIV id="Resistance System" class="project"><H2>Background</H2><P>
				Normally, bacteria cannot survive in extremely acid and alkali. So before producing acid or alkali, the bacteria should be able to tolerate acidic or alkaline environment. Two devices are applied to achieve it.
				
				The first is alkali-resistance device, consisting of Nha protein family.
				
				Nha is a family containing a number of bacterial sodium-proton anti-porters. These are integral membrane proteins that catalyze the exchange of proton for sodium in a manner highly dependent on the pH. To resist different pH, we select two genes, <I>NhaA</I> and <I>Nha</I>, from this family.
				<I>NhaA</I> is the archetypal Na<SUP>+</SUP>/H<SUP>+</SUP> antiporter and the only member of the family that is required by <I>E.coli</I> for survival under alkaline stress<SUP>[1,2]</SUP>. It is a membrane protein consisting of 388 residues<SUP>[3]</SUP>. <SPAN style="color:red;"><I>NhaA</I> excretes Na<SUP>+</SUP> from the cytoplasm using the energy from the cotransport of protons down their electrochemical gradient into the cell, with a characteristic electrogenic stoichiometry of two protons to one Na<SUP>+</SUP></SPAN><SUP>[4,5]</SUP>(<I>Fig.1</I>). Like many other Na<SUP>+</SUP>/H<SUP>+</SUP> antiporters, <I>NhaA</I>  is regulated by pH. It is essentially inactive below pH 7, and the Na<SUP>+</SUP> efflux rate increases by three orders of magnitude between pH 6.5 and 8.5<SUP>[6]</SUP>.</P><DIV style="width:100%; heigth:auto"><DIV style="float:left; width:45%;margin-left:5% "><P style="text-align:center;">Fig.1</P></DIV><DIV style="float:left;width:45%;"><P style="text-align:center;">Fig.2</P></DIV></DIV><P style="clear:left;">
				The <I>NhaB</I> gene encodes Na<SUP>+</SUP>/H<SUP>+</SUP> antiporter catalyzing the exchange of 3 H<SUP>+</SUP> per 2 Na<SUP>+</SUP>(<I>Fig.2</I>). <SUP>[7]</SUP>NhaB has a high affinity for sodium. In the absence of <I>NhaA</I>, <I>NhaB</I> confers a certain tolerance to Na<SUP>+</SUP> which decreases with increasing pH. Essential for regulation of intracellular pH under alkaline conditions, <I>NhaB</I> is crucial when the level of <I>NhaA</I> activity is limiting, when <I>NhaA</I> is not sufficiently induced, and/or when <I>NhaA</I> is not activated<SUP>[8]</SUP>. Unlike <I>NhaA</I>, the activity of <I>NhaB</I> shows no dependence on pH in the range 6.4-8.3. The affinity of <I>NhaB</I> to Na<SUP>+</SUP> ions (Km = 40-70μM) is higher than that of <I>NhaA</I>.<SUP>[9]</SUP></P><H2>Design</H2><P>
				The gene circuits of the alkali-resistance device is shown in Fig.3. This device consists of two genes and two promoters. The strong constitutive promoter J23119 is used in our gene circuit. In order to resist alkaline stress, we chose two of Nha sodium-proton anti-porter proteins referred before to construct device one. <I>NhaA</I> could make sure <I>E.coli</I>’s survival under alkaline pH. <I>NhaB</I> regulates intracellular pH when <I>NhaA</I> is limited. But <I>NhaB</I> is weakly pH-dependent, so a alkali-induced promoter J23119 is necessary. The construction has been finished .
			</P><P style="text-align:center;">Fig.3 gene circuit.</P><H2>Result</H2><P>
				Firstly, we obtained the genes, <I>NhaA</I> (BBa_K1) and <I>NhaB</I> (BBa_K2), through PCR. The agarose gel electrophoresis analysis of gene <I>NhaA</I> and <I>NhaB</I> is shown in Fig.4.
				Secondly, we assembled the standard part J23119+B0034+<I>NhaA</I>(BBa_K3, JBA) and J23119+B0034+<I>NhaB</I>(BBa_K4, JBB). The agarose gel electrophoresis analysis of JBA and JBB is shown in Fig.5 and Fig.6.</P><DIV style="width:100%; heigth:auto"><DIV style="float:left;width:45%; margin-left:5%;"><P style="text-align:center;">Fig.4</P></DIV><DIV style="float:left;width:45%; margin-left:6px;"><P style="text-align:center;">Fig.5</P></DIV></DIV><P style="clear:left;">After finishing the construction of the device, we verified it through testing the growth of bacteria. We draw the growth curve via measure the OD<SUB>600</SUB> value, and the results are shown in Fig.7.</P><P style="text-align:center;">Fig.6 gene circuit.</P><H2>Conclusion</H2><H2>Introduction</H2><P><I>glsA</I> is a kind of glutaminase gene in <I>E.coli</I> functioning at acidic pH ranges<SUP>[12]</SUP>. <I>glsA</I> gene encodes glutaminase A(GlsA)<SUP>[13]</SUP> which can transform glutamine(Gln) and water into glutamate(Glu) and ammonium<SUP>[14]</SUP>. GlsA is activated in acidic environment, and exhibits robust glutaminase activity only at pH 6.0 or lower. The highest activity of GlsA is obtained at pH 4.0, followed by pH 5.0 and 6.0. In contrast, at the pH 7.0 and 8.0, GlsA is not activated<SUP>[15]</SUP>. With the presence of extracellular Gln, membrane amino acid antiporter GadC<SUP>[16]</SUP> could transport Gln into the cell, and glutaminase A can transform Gln and water into Glu and ammonium(Figure 1.). Gaseous ammonium will consume H<SUP>+</SUP> proton and increase intracellular pH, while Glu can anticipate in decarboxylation process led by decarboxylases GadA(链接), and the latter process will also consume H<SUP>+</SUP> proton. More details about GadA wil be discussed in fine-regulation subsystem(做链接).
		</P><H2>Design</H2><P>
			Our project chose <I>glsA</I> as function gene for acid-resistance device which can regulate intracellular environment and protect <I>E.coli</I> from extreme acidic environment(Figure 2.). J23119 is a constitutive promoter and control transcription of <I>glsA</I>. When environment become acidic, GlsA will be activated and function with the anticipation of glutamine(Gln). When environmental pH return neutral, activity of GlsA will be repressed<SUP>[18]</SUP>.
		</P><H2>Experiment Results</H2><H2>Construction of standard part</H2><P>
			We constructed J23119 promoter, B0034 with function gene <I>glsA</I> standard part on plasmid PSB1C3. J23119 promoter and B0034 sequences were gained from iGEM Registry. 
			We firstly applied PCR to clone <I>glsA</I> gene from genome of K12 strain. As J23119 promoter and B0034 are short(small) fragments, we straightly designed these sequences on upstream primer. And we designed SpeI and PstI restriction enzyme cutting sites on downstream primer for further experiments. Then, because EcoRI and Xba restriction sites are too long for upstream primer, we needed to perform a second time PCR. This time, EcoRI and Xba restriction sites were designed on upstream primer. Eventually, J23119+B0034+<I>glsA</I> with four restriction sites were inserted into PSB1C3 plasmid.
		</P><P>
			cPCR was performed to verify whether these sequences were successfully inserted into plasmid. Figure 2. shows that No.10 and No.13 were ideal results. We cultivated these two bacterial colony, preserved these bacterium and extracted plasmid. The final step is to record the base sequence and compare with standard sequences. 
		</P><UL class="reference_container"><LI><H2>References<SPAN>[Expand]</SPAN></H2><P>
			        [1]  E. Padan, M. Venturi, Y. Gerchman, N. Dover, Biochim. Biophys. Acta 1505, 144 (2001).
				[2]  E. Padan et al., Biochim. Biophys. Acta 1658, 2 (2004).			[3]  A. Rothman, E. Padan, S. Schuldiner, J. Biol. Chem. 271, 32288 (1996).
				[4]  D. Taglicht, E. Padan, S. Schuldiner, J. Biol. Chem. 268, 5382 (1993).
				[5]  S. Schuldiner, H. Fishkes, Biochemistry 17, 706 (1978).</P><P>
				[6]  Dudu A, Abraham R, Maral B, et al. <I>NhaA</I> Na<SUP>+</SUP>/H<SUP>+</SUP> Antiporter Mutants That Hardly React to the Membrane Potential[J]. Plos One, 2014, 9(4):e93200.
				[7]  Pinner, E., Padan, E. and Schuldiner, S. (1994) J. Biol. Chem. 269, 26274-26279.
				[8]  Pinner E ,, Kotler Y ,, Padan E ,, et al. Physiological role of nhaB, a specific Na<SUP>+</SUP>/H<SUP>+</SUP> antiporter in Escherichia coli.[J]. J.biol.chem, 1993, 268(1):1729-1734.
				[9]  Schuldiner, S., and Padan, E. (1992) in Alkali Cation Transport Systems in Procaryotes (Bakker, E., ed) CRC Press, Boca Raton, FL, in press.
				[10]  Kuroda T, Shimamoto T, Inaba K, et al. Properties and sequence of the NhaA Na<SUP>+</SUP>/H<SUP>+</SUP> antiporter of Vibrio parahaemolyticus.[J]. Journal of Biochemistry, 1994, 116(5):1030-8.
				[11]  Nozaki K, Kuroda T, Mizushima T, et al. A new Na<SUP>+</SUP>/H<SUP>+</SUP> antiporter, NhaD, of Vibrio parahaemolyticus.[J]. Biochimica et biophysica acta, 1998, 1369(2):213–220.


				[12],[18]  Standish C. Hartman, Glutaminase of Escherichia coli. The Journal of Biological Chemistry,1967.
				[13]  Brown G, Singer A, Proudfoot M, Skarina T, Kim Y, Chang C, Dementieva I, Kuznetsova E, Gonzalez CF, Joachimiak A, Savchenko A, Yakunin AF, Functional and structural characterization of four glutaminases from Escherichia coli and Bacillus subtilis. Biochemistry, 2008, 47:5724-35.
   				[14]  Hersh BM, Farooq FT, Barstad DN, Blankenhorn DL, Slonczewski JL, A glutamate-dependent acid resistance gene in Escherichia coli. J Bacteriol, 1996, 178:3978-81.
				[15]  Lu P, Ma D, Chen Y, Guo Y, Chen GQ, Deng H, Shi Y, L-glutamine provides acid resistance for Escherichia coli through enzymatic release of ammonia. Cell Res, 2013, 23:635-44.
				[16]  Dan Ma, Peilong Lu, Chuangye Yan, Chao Fan, Ping Yin, Jiawei Wang, Yigong Shi(2012).Structure and mechanism of a glutamate-GABA antiporter. Nature 483:632-6.</P></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>