Team:BIT-China/Project Resistance Subsystem
Add a banner to your wiki!
You can make the image 980px  by  200px
Remember to call the file: "Team_BIT-China_banner.jpg"
Project
Description Design Basic circuits Fine-regulation circuits
Modeling
Basic circuits Fine-regulation circuits
Notebook
Laboratory Notes Protocols
Judging
Parts Submission Achievements
Team
Team Gallery
Practice
Human Practice Collaboration
Safety
Safety Q/A
Attributions
Attributions Sponsors
Background
Normally, bacteria cannot survive in extremely acid and alkali. So before producing acid or alkali, the bacteria should be able to tolerate acidic or alkaline environment. Two devices are applied to achieve it.
The first is alkali-resistance device, consisting of Nha protein family.
Nha is a family containing a number of bacterial sodium-proton anti-porters. These are integral membrane proteins that catalyze the exchange of proton for sodium in a manner highly dependent on the pH. To resist different pH, we select two genes, NhaA and Nha, from this family.
NhaA is the archetypal Na+/H+ antiporter and the only member of the family that is required by E.coli for survival under alkaline stress[1,2]. It is a membrane protein consisting of 388 residues[3]. NhaA excretes Na+ from the cytoplasm using the energy from the cotransport of protons down their electrochemical gradient into the cell, with a characteristic electrogenic stoichiometry of two protons to one Na+[4,5](Fig.1). Like many other Na+/H+ antiporters, NhaA is regulated by pH. It is essentially inactive below pH 7, and the Na+ efflux rate increases by three orders of magnitude between pH 6.5 and 8.5[6].
Fig.1
Fig.2
The NhaB gene encodes Na+/H+ antiporter catalyzing the exchange of 3 H+ per 2 Na+(Fig.2). [7]NhaB has a high affinity for sodium. In the absence of NhaA, NhaB confers a certain tolerance to Na+ which decreases with increasing pH. Essential for regulation of intracellular pH under alkaline conditions, NhaB is crucial when the level of NhaA activity is limiting, when NhaA is not sufficiently induced, and/or when NhaA is not activated[8]. Unlike NhaA, the activity of NhaB shows no dependence on pH in the range 6.4-8.3. The affinity of NhaB to Na+ ions (Km = 40-70μM) is higher than that of NhaA.[9]
Design
The gene circuits of the alkali-resistance device is shown in Fig.3. This device consists of two genes and two promoters. The strong constitutive promoter J23119 is used in our gene circuit. In order to resist alkaline stress, we chose two of Nha sodium-proton anti-porter proteins referred before to construct device one. NhaA could make sure E.coli’s survival under alkaline pH. NhaB regulates intracellular pH when NhaA is limited. But NhaB is weakly pH-dependent, so a alkali-induced promoter J23119 is necessary. The construction has been finished .
Fig.3 gene circuit.
Result
Firstly, we obtained the genes, NhaA (BBa_K1) and NhaB (BBa_K2), through PCR. The agarose gel electrophoresis analysis of gene NhaA and NhaB is shown in Fig.4.
Secondly, we assembled the standard part J23119+B0034+NhaA(BBa_K3, JBA) and J23119+B0034+NhaB(BBa_K4, JBB). The agarose gel electrophoresis analysis of JBA and JBB is shown in Fig.5 and Fig.6.
Fig.4
Fig.5
After finishing the construction of the device, we verified it through testing the growth of bacteria. We draw the growth curve via measure the OD600 value, and the results are shown in Fig.7.
Fig.6 gene circuit.
Conclusion
Introduction
glsA is a kind of glutaminase gene in E.coli functioning at acidic pH ranges[12]. glsA gene encodes glutaminase A(GlsA)[13] which can transform glutamine(Gln) and water into glutamate(Glu) and ammonium[14]. GlsA is activated in acidic environment, and exhibits robust glutaminase activity only at pH 6.0 or lower. The highest activity of GlsA is obtained at pH 4.0, followed by pH 5.0 and 6.0. In contrast, at the pH 7.0 and 8.0, GlsA is not activated[15]. With the presence of extracellular Gln, membrane amino acid antiporter GadC[16] could transport Gln into the cell, and glutaminase A can transform Gln and water into Glu and ammonium(Figure 1.). Gaseous ammonium will consume H+ proton and increase intracellular pH, while Glu can anticipate in decarboxylation process led by decarboxylases GadA(链接), and the latter process will also consume H+ proton. More details about GadA wil be discussed in fine-regulation subsystem(做链接).
Design
Our project chose glsA as function gene for acid-resistance device which can regulate intracellular environment and protect E.coli from extreme acidic environment(Figure 2.). J23119 is a constitutive promoter and control transcription of glsA. When environment become acidic, GlsA will be activated and function with the anticipation of glutamine(Gln). When environmental pH return neutral, activity of GlsA will be repressed[18].
Experiment Results
Construction of standard part
We constructed J23119 promoter, B0034 with function gene glsA standard part on plasmid PSB1C3. J23119 promoter and B0034 sequences were gained from iGEM Registry.
We firstly applied PCR to clone glsA gene from genome of K12 strain. As J23119 promoter and B0034 are short(small) fragments, we straightly designed these sequences on upstream primer. And we designed SpeI and PstI restriction enzyme cutting sites on downstream primer for further experiments. Then, because EcoRI and Xba restriction sites are too long for upstream primer, we needed to perform a second time PCR. This time, EcoRI and Xba restriction sites were designed on upstream primer. Eventually, J23119+B0034+glsA with four restriction sites were inserted into PSB1C3 plasmid.
cPCR was performed to verify whether these sequences were successfully inserted into plasmid. Figure 2. shows that No.10 and No.13 were ideal results. We cultivated these two bacterial colony, preserved these bacterium and extracted plasmid. The final step is to record the base sequence and compare with standard sequences.
References[Expand]
[1]  E. Padan, M. Venturi, Y. Gerchman, N. Dover, Biochim. Biophys. Acta 1505, 144 (2001).
[2]  E. Padan et al., Biochim. Biophys. Acta 1658, 2 (2004).
[3]  A. Rothman, E. Padan, S. Schuldiner, J. Biol. Chem. 271, 32288 (1996).
[4]  D. Taglicht, E. Padan, S. Schuldiner, J. Biol. Chem. 268, 5382 (1993).
[5]  S. Schuldiner, H. Fishkes, Biochemistry 17, 706 (1978).
[6]  Dudu A, Abraham R, Maral B, et al. NhaA Na+/H+ Antiporter Mutants That Hardly React to the
Membrane Potential[J]. Plos One, 2014, 9(4):e93200.
[7]  Pinner, E., Padan, E. and Schuldiner, S. (1994) J. Biol. Chem. 269, 26274-26279.
[8]  Pinner E ,, Kotler Y ,, Padan E ,, et al. Physiological role of nhaB, a specific Na+/H+ antiporter in Escherichia coli.[J]. J.biol.chem, 1993, 268(1):1729-1734.
[9]  Schuldiner, S., and Padan, E. (1992) in Alkali Cation Transport Systems in Procaryotes (Bakker, E., ed) CRC Press, Boca Raton, FL, in press.
[10]  Kuroda T, Shimamoto T, Inaba K, et al. Properties and sequence of the NhaA Na+/H+ antiporter of Vibrio parahaemolyticus.[J]. Journal of Biochemistry, 1994, 116(5):1030-8.
[11]  Nozaki K, Kuroda T, Mizushima T, et al. A new Na+/H+ antiporter, NhaD, of Vibrio parahaemolyticus.[J]. Biochimica et biophysica acta, 1998, 1369(2):213–220.
[12],[18]  Standish C. Hartman, Glutaminase of Escherichia coli. The Journal of Biological Chemistry,1967.
[13]  Brown G, Singer A, Proudfoot M, Skarina T, Kim Y, Chang C, Dementieva I, Kuznetsova E, Gonzalez CF, Joachimiak A, Savchenko A, Yakunin AF, Functional and structural characterization of four glutaminases from Escherichia coli and Bacillus subtilis. Biochemistry, 2008, 47:5724-35.
[14]  Hersh BM, Farooq FT, Barstad DN, Blankenhorn DL, Slonczewski JL, A glutamate-dependent acid resistance gene in Escherichia coli. J Bacteriol, 1996, 178:3978-81.
[15]  Lu P, Ma D, Chen Y, Guo Y, Chen GQ, Deng H, Shi Y, L-glutamine provides acid resistance for Escherichia coli through enzymatic release of ammonia. Cell Res, 2013, 23:635-44.
[16]  Dan Ma, Peilong Lu, Chuangye Yan, Chao Fan, Ping Yin, Jiawei Wang, Yigong Shi(2012).Structure and mechanism of a glutamate-GABA antiporter. Nature 483:632-6.
