<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cooper_Union_Notebook_TdT_July"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Cooper_Union/Notebook/TdT_July">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cooper_Union/Notebook/TdT_July&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cooper_Union/Notebook/TdT_July&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cooper_Union/Notebook/TdT_July&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cooper_Union/Notebook/TdT_July" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cooper Union/Notebook/TdT July</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Cooper Union 2014 iGEM</TITLE></P><DIV class="parent-container"><DIV class="menu-parent"><UL class="menu"><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union"> Home </A></LI><LI class="menu-project"> Project
		</LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/TdT_project">De Novo Synthesis </A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Telomere_project">Programmable Lifespan Timer</A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Biohack_project">Biohacker Kit </A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Hardware">OpenSource Hardware </A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Parts">BioBrick Parts </A></LI></UL><LI class="menu-safety"> Social
		</LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Safety">Safety</A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Ethics">Ethics</A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Outreach">Outreach </A></LI><LI class="menu-notebook">   Notebook 
		</LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Protocols">Commonly Used Protocols </A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Notebook"> Labnotes </A></LI><LI class="menu-team"> Team
		</LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/biofanfic">Members </A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Attributions"> Attributions </A></LI><LI class="menu"><A class="menu" href="https://igem.org/Team.cgi?id=1354"> Official Team Profile </A></LI></DIV><DIV class="text"><DIV class="notebook-header"><DIV class="left-side"><H1>De Novo Synthesis</H1></DIV><DIV class="notebook-nav-container"><DIV class="month-links"><A href="https://2014.igem.org/Team:Cooper_Union/Notebook/TdT_June">JUN</A><A href="https://2014.igem.org/Team:Cooper_Union/Notebook/TdT_July" class="selected">JUL</A><A href="https://2014.igem.org/Team:Cooper_Union/Notebook/TdT_August">AUG</A><A href="https://2014.igem.org/Team:Cooper_Union/Notebook/TdT_September">SEP</A></DIV></DIV></DIV><H3>7/1/14</H3>

We first ran the 6% gel from yesterday in the order of ladder, +A, +G, -, 1A, 1G, 2A, and 2G.

Then we phosphatase treated the double digested pET28b+ (11.1 ng/μL). 17μL of pET28b+ was mixed with 2μL buffer and 1μL antartic phosphatase, incubated for an hour at 37°C, and denatured at 70°C for 5mins. The new concentration calculated is 9. 44 ng/μL. 

20μL of TE buffer was added to Gblock and spun down, chilled for 10min ice, and double digested with NdeI and Bam HI-HF. Phosphotase-treated pET28b+ was then ligated with digested Gblock. 10μL of Gblock and 10μL of vector (instead of 5μL) were put, and the total volume was 25μL instead of 20μL. Negative control with vector without insert but with ligase should be set up tomorrow. 

PCR was prepared with 1:10 diluted tdt primers prepared on the 18th, but because the machine was not working, we will run PCR tomorrow. Transformation should be also done tomorrow according to NEB 10β E. coli protocols. 

<H3>7/2/14</H3>

Checked the 6% gel from yesterday, not worked. We barely see anything, and it was really faint in general. Should troubleshoot and go over the protocols. 

We ran the PCR reaction from yesterday, with (95°C 30sec, 55°C 30sec, and 72°C 2mim)*30 cycles. After PCR reactions were done, we ran the gel with the kill switch team. 

We set up the negative control with 10μL vector without inserts but keeping everything the same. Then the transformation was done with the pET28b+ Tdt ligation sample from yesterday. Used NEB10β E. coli cells and we followed the exactly same protocol that NEB provided. 5μL of plasmid DNA was added to each tube, 200μL of mix were added each to the Kan plate (2 for negative control and 2 for pET28b+ Tdt ligation), and incubated at 37°C overnight.

<H3>7/3/14</H3>

Today we re-ran the PCR reaction and checked it using gel-electrophoresis. 1% gel and 1kb ladder was used. Gel did not show anything, so we are planning to run gradient PCR next week. 

We also re-ran the ligation reaction, however, this time we are allowing it to run overnight at 16°C. Professor Medvedik will come in tomorrow morning to stop the reaction. we used 10ng of vector and 30ng of insert instead of 50 and 75. 

<H3>7/7/14</H3>

Did the transformation of a ligated sample from yesterday. Because negative control was not set up (although we were supposed to set one when we are doing the experiment...), last experiment's negative control was used from the stock. Same protocol was used, and was stored in 37°C incubator. 

There might be a problem in the primer design itself, since the IDT site's program says the primers might be binding to each other. To minimize that, PCR was re-ran with increased cycles (40 instead of 30 cycles), and annealing temperature was lowered to 53°C, but time was increased to 45s instead of 30s. DMSO was added to prevent dimers. Sample 1 is 50pg DNA with primers, 2 is 50pg DNA with primers with 1% DMSO, and 3 is with 5% DMSO. Along with variation of DMSO concentrations, two different PCR reactions were prepared, one with PCR beads and the other one with Taq solution. 

pSB1C3(minipreped on 6/25, sample1, nanodropped and concentration was 71.1ng/μL) and Tdt (diluted: 20pg/μL) were double-digested with EcoRI-HF and PstI. The sample was incubated for an hour and ligated with T4 ligase. The ligation sample was then transformed to NEB10β competent cells. As a control and glycerol stock, pET28b+ was also transformed to DH5α cell on a Chlo plate. Same protocol was followed. 

<H3>7/8/14</H3>

We first ran a gel with PCR samples from yesterday. It seems like primers went through to the end, and we did not see any amplified templates. But we did see primers bound to each other. 

Transformation yesterday did not work, but pSB1C3+tdt ligation sample transformed to NEB10β cells showed several colonies (though it seems most of them are red). Colony PCR was set up with the colonies on the plate. 6 colonies from each plate were pick and resuspended to 5μL ddH<SUB>2</SUB>O. 2.5μL was used for plate, and with the other 2.5μL PCR reaction was set up with the cycle of 95°C 6min, 30*(95°C 6min, 55°C 30sec, 70°C 30sec), 70°C 5min, held for 4°C, using primer VF2(10μM, 0.5μL) and VR(10μM, 0.5μL)

Double digestion of Tdt and pET28b+ was set up. Original Gblock sample was digested with NdeI &amp; BamHI-HF and with EcoRI-HF &amp; PstI. pET28b+ was digested with NdeI and BamHI-HF. incubated for an hour, and ligated. 

pSB1C3 + TdT ligation 1 and 2 respectively



 

<H3>7/9/14</H3>

Today we transformed the ligation of pSB1C3+TdT and pET28B+_TdT and spread it on plates and left it overnight. We also made a 1:1000 dilution of the Gblock so it has a concentration of 44pg/μL and used it to run the 3 PCR beads and 3 PCR Taq solutions. We increased the annealing temperature from 53° to 54°, the rest remained the same. 

On the pET28B+_TdT ligation plate a colony grew and this colony was placed in broth and incubated overnight. Additionally, two minipreps of pET28B+ were done.

<H3>7/10/14</H3>

Today we minipreped the pET28B+_TdT ligation that was left to incubate overnight. We also made two 0.8% gels and 2-1% gels. We also ran a gel of the 6 PCR reactions that ran yesterday and found very faint bands at about 1.6kb, which indicated that our TdT was amplified but very inefficiently. It also appeared that the PCR bead with the 5% DMSO showed the heaviest band. In the near future, we will run more experiments on this PCR using 5% DMSO to optimize the reaction. 

When we analyzed the plates from yesterday, we found that the pET28b+_TdT ligation yielded many white colonies and the pSB1C3 + TdT ligation yielded many red colonies, which indicate that the TdT gene was not incorporated, with a few white colonies, which could be either the desired product or an unknown bacteria that is also resistant to chloramphenicol. Because of these results, we decided to take two white colonies from the two pSB1C3 + TdT ligation plates and inoculate them overnight, giving a total of 4 inoculations. We also decided to run a colony PCR on the pET28b+_TdT ligation colonies. We took 4 colonies from each plate and resuspended them in 5μL each, giving a total of 8 resuspended colonies. We then took 2.5mu;L of each and spotted them on a gel to incubate overnight. We then took the other 2.5mu;L and added 10x Buffer, first forward primer, reverse primer, dNTPs, ddH<SUB>2</SUB>O, and Taq polymerase to each. These were then run in the PCR machine overnight along with a PCR for the TdT G-Block and a PCR for the first truncated version of TdT. The reaction was set using the program 333, however a few changes were made to the program. The annealing temperature was increased to 57°C and the amount of cycles was increased from 30 to 40.

<H3>7/11/14</H3>

Today, when we came in the eight pET28b+_TdT ligation samples spotted on the same plate all grew. Because none of us will be in the lab tomorrow, we parafilmed the plate and placed it in the refrigerator so that it could be inoculated on Monday. The four samples of pSB1C3 + TdT that were inoculated overnight were used to four create glycerol stocks and then minipreped.

The PCR reactions that were run overnight were run on two 0.8% gels and it was found that the positive control and colonies 1A, 1B, and 2B all showed bands at about 1.6kb, indicating that the TdT gene was in fact inserted into the pET28b+ vector in these colonies. Those three out of the eight colonies will be inoculated overnight on Monday.
<EM>From left to right: ladder, colony 1A, colony 1B, colony 1C, colony 1D, TdT, truncated TdT, ladder</EM><EM>From left to right: ladder, colony 2A, colony 2B, colony 2C, colony 2D, negative control, blank, ladder</EM>

We then double digested some of the minipreped pSB1C3 + TdT ligation using EcoRI and PstI. The results of this digest were ran on a 0.8% gel.

<H3>7/13/14</H3>

1A, 1B, and 2A colonies on pET28b+ and Tdt ligation colony PCR plate were inoculated overnight. 

<H3>7/14/14</H3>

ligation samples from yesterday were miniprepped. Concentrations were low (7~12ng/μL) so we decided to try midiprep tomorrow. Colonies are prepared for overnight culture. 

<H3>7/15/14</H3>

Glycerol stocks were made from yesterday's overnight culture (pET28b+_Tdt ligation colony PCR on 7/10, 1A, 1B, 2A)

50mL LB media was added to the rest of the overnight culture and was incubated for another 4 hours. 

Midiprep was done until DNA elution step, and samples were stored in 4°C fridge. The samples were prepped for PCR Reaction along with miniprep samples. PCR reactions were done overnight (40 cycles, 57°C annealing for 2 mins).

We will run gels tomorrow.

<H3>7/16/14</H3>

Today we completed the midi prep and ran a PCR reaction on all 6 of the midi and mini preps of pET28b+_TdT vectors. Because the PCR machine was in use for the beginning of the day, we could not run our PCR until 3:30 and so we will have to wait until tomorrow to run the gels

<H3>7/17/14</H3>

Today we ran the gels on the four samples taken from the midi prep along with the PCR mini and midi prep. Three gels were run with each gel containing a different sample. The gels are shown below.<EM>Left to right: Ladder, 1A sample #1, 1A sample #2, 1A sample #3, 1A sample #4, 1A PCR midi prep, 1A PCR mini prep, ladder</EM><EM>Left to right: Ladder, 1B sample #1, 1B sample #2, 1B sample #3, 1B sample #4, 1B PCR midi prep, 1B PCR mini prep, ladder</EM><EM>Left to right: Ladder, 2B sample #1, 2B sample #2, 2B sample #3, 2B sample #4, 2B PCR midi prep, 2B PCR mini prep, ladder</EM><H3>7/18/14</H3>

We phosphotase-treated double digested pSB1C3 vector today. We also set up another Tdt CleanAmp experiment and ran it on a polyacrylamide gel. Nothing appeared on the gel, but the bands of the ladder were very crisp, indicating that there might be problems in Tdt itself. We will call the company to check. 

We also sent out pET28b+_Tdt ligation vectors for sequencing. The primers were T7 Terminal (Fwd), T7 (Rev).

<H3>7/21/14</H3>

Transformation was done to Rosetta cells and stored overnight at 37°C Kan plate (1A, 1B, 2B)

1:1000 diluted Tdt Gblock was double digested with EcoRI-HF and PstI, then ligated to pSB1C3 phosphotase-treated vector. The sample was incubated at 16°C overnight.

<H3>7/22/14</H3>

The plates from yesterday were re-streaked because they were overly competent. The original plates were stored in the fridge, and the streaked ones were stored in the incubator overnight at 37°C. 

The transformation of pSB1C3+Tdt ligation to NEB10β cells was also done and left to incubate overnight at 37°C.

We also received the sequencing results of the pET28b+_TdT ligations and compared the results to what was expected. It was found that 1A was in fact TdT ligated into the pET28b+ vector. Unfortunately, most of the sequencing results were poor so it can not be determined at this time whether or not the other vectors (1B and 2B) do have TdT integrated into pET28b+.

<H3>7/23/14</H3>

We re-streaked the streaked plate from yesterday.

Yesterday's transformation was not showing anything, so we did transformation again with 7/21's ligation sample. This time we used 20μL of sample and NEB10β cells and tried to maximize the amount of DNA. 

We also decided to try other backbone, and used pSB1K3 to ligate Tdt. pSB1K3 #2 (digested with EcoRI and PstI) was phoshotase treated and ligated with double digested Tdt. For Tdt double digestion, 1:10 dilution sample was used (4.4 ng/μL) The ligation sample was stored in ligation machine overnight at 16°C. Transformation should be done tomorrow. 

<H3>7/24/14</H3>

We did transformation of pSB1K3+Tdt ligation from yesterday. Kan plate was used, and the plate was incubated overnight at 37°C.

We also innoculated from pET28b+_Tdt plates restreaked (twice) and left it overnight. 

<H3>7/29/14</H3>

Transformation did not worked, so we redesigned the forward primer for Tdt full length.

Also searched for protein expression protocols based on the paper. Because we do not have negative control for pET28b+_Tdt transformation with Rosetta cells, we set up a transformation of empty pET28b+ uncut and left in the incubator overnight. Tomorrow we should be inoculating the glycerol stocks and transformed cell for protein expression experiments.  

<H3>7/30/14</H3>

We first inoculated the transformed control and pET28b+_Tdt Glycerol stock (1A sample#1). The inoculated sample was incubated at 30°C, 255rpm overnight from 10:25~. 

Buffers for protein expression experiment was also prepared (Buffer B, C, and E). 

Another set of Tdt experiment was performed to make sure that dNTPs are properly working. Using A and without denaturation, we tried various substrates. 24mer, PolyA, 33mer, and 52mer was used. We had to re-nanodrop everything since we concluded the concentrations written on the tubes were not based on ssDNA. New concentrations were written on the tubes. The samples were incubated for 30minutes at 37°C, and reaction was stopped at 70°C, 10min. We used polyacrylamide gels and ran for about 50 min, at 200V.

<H3>7/31/14</H3>

The Growth of Expression Cultures Protocol in Qiagen's <A href="http://www.qiagen.com/resources/download.aspx?id=3fc8c76d-6d21-4887-9bf8-f35f78fcc2f2&amp;lang=en">Ni-NTA Spin Kit Handbook</A> was followed with a few exceptions. The overnight cultured Rosetta cells (a pET28b+_TdT sample and two control samples of pEt28b+) were diluted with fresh LB medium but instead of following the 1:60 overnight culture to LB ratio stated, a 1:12 was used for a shorter incubation time. The samples were incubated and the OD<SUB>600</SUB> were regularly checked. When the OD<SUB>600</SUB> exceeded 0.6, the samples were diluted with additional LB medium until they an OD<SUB>600</SUB> of 0.5-0.6 were obtained. The cells were then IPTG induced with ten times the stated concentration (10nM final concentration). After 4 hours of incubation, the samples were pelleted, washed once with 1x PBS, pelleted into smaller tubes and stored in the -20°C freezer. 

The TdT G-Block was PCR-amplified with the new forward primer. The PCR products were ran on a 1.0% gel. As the gel shows, the PCR reaction failed to amplify the TdT G-block.

</DIV></DIV><DIV class="cu-footer"><DIV class="bottom-center"><A href="https://2014.igem.org/Team:Cooper_Union/Project" class="black"><SPAN>Projects</SPAN></A><A href="https://2014.igem.org/Team:Cooper_Union/Social" class="red"><SPAN>Social</SPAN></A><A href="https://2014.igem.org/Team:Cooper_Union/Notebook" class="green"><SPAN>Notebook</SPAN></A><A href="https://2014.igem.org/Team:Cooper_Union/biofanfic" class="blue"><SPAN>Team</SPAN></A></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Cooper_Union/Notebook/TdT_July">http://2014.igem.org/Team:Cooper_Union/Notebook/TdT_July</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cooper_Union/Notebook/TdT_July" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cooper_Union/Notebook/TdT_July" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cooper_Union/Notebook/TdT_July&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cooper_Union/Notebook/TdT_July&amp;oldid=377336" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>