<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cooper_Union_TdT_project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Cooper_Union/TdT_project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cooper_Union/TdT_project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cooper_Union/TdT_project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cooper_Union/TdT_project&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cooper_Union/TdT_project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cooper Union/TdT project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Cooper Union 2014 iGEM</TITLE></P><DIV class="parent-container"><DIV class="menu-parent"><UL class="menu"><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union"> Home </A></LI><LI class="menu-project"> Project
		</LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/TdT_project">De Novo Synthesis </A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Telomere_project">Programmable Lifespan Timer</A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Biohack_project">Biohacker Kit </A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Hardware">OpenSource Hardware </A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Parts">BioBrick Parts </A></LI></UL><LI class="menu-safety"> Social
		</LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Safety">Safety</A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Ethics">Ethics</A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Outreach">Outreach </A></LI><LI class="menu-notebook">   Notebook 
		</LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Protocols">Commonly Used Protocols </A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Notebook"> Labnotes </A></LI><LI class="menu-team"> Team
		</LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/biofanfic">Members </A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Attributions"> Attributions </A></LI><LI class="menu"><A class="menu" href="https://igem.org/Team.cgi?id=1354"> Official Team Profile </A></LI></DIV><DIV class="text"><DIV class="center"><H1>De novo Enzyme Mediated Oligonucleotide Synthesizer</H1></DIV>


De novo synthesis is a way of creating DNA oligonucleotides without the need of a template strand. Since the conventional method is expensive, time consuming, and inefficient, our team has focused on minimizing the time and money required for DNA synthesis while allowing labs to produce oligonucleotides easily without ordering. Here we introduce the De novo Enzyme Mediated Oligonucleotide Synthesizer (DEMOS); a programmable enzyme based, template-free, synthesizer for nucleic acid polymers. Our eventual goal will be to build a microfluidic De novo synthesizer that will allow laboratories, from academia to DIYBio community labs to rapidly and economically synthesize any strand of DNA for their projects. We hope that this system will become the key platform that bridges the <EM>in silico</EM> to <EM>in vitro</EM> gap in the design-test-build cycle of DNA synthesis and experimentation.


Our system is fundamentally based on two key technologies, one practical and the other theoretical:
<OL><LI>The development of deoxynucleotide triphosphate (dNTP) substrates with 3' reversible protective groups for &quot;sequencing by synthesis&quot; and &quot;hot start&quot; PCR technologies</LI><LI>The published theoretical model of directed template-free synthesis of DNA using the enzyme terminal deoxynucleotidyl transferase (TdT). This enzyme that has the ability to add nucleotides to the 3' ends of DNA, preferable 3' overhangs, in a template-free manner. </LI></OL>
In the general scheme shown below in Figure 1, an incoming 3'-RPdNTP (reverse protective group dNTP) is added to a free 3'OH of a growing DNA chain by the enzyme TdT at 37° Celsius. This is followed by a wash step where unincorporated 3'RPdNTPs and PPi (pyrophosphate) are removed. For heat labile 3'-RPdNTP’s deprotection is achieved by raising the temperature to 95° Celsius, while photolabile 3'-RPdNTP’s are deprotected by pulses of ultra-violet light. This is followed by another wash step where the decoupled protective groups are removed, thereby resetting the system to begin another cycle of addition. 
<DIV class="center"><SPAN>Figure 1. General DNA De novo Synthesis Scheme</SPAN></DIV>
The deprotection reaction of the 3'-RPdNTP’s is shown in Figure 2a. The structures of two heat labile 3' protective group dNTPs that are commercially available from TriLink Biotechnologies (www.trilinkbiotech.com) are shown in Figure 2b . For our proof of concept experiments, 3'-TBE-dNTP’s were used, as it has a faster rate of deprotection at 95° Celsius; 3'-TBE-dNTP t 1/2 + 5 minutes, 3'-THF-dNTP t 1/2= 90 minutes) 
<DIV class="center"><SPAN>Figure 2 a. General Deprotection of Protected Nucleotide b. Thermolabile Protective Groups</SPAN></DIV><DIV class="center"><H2>Advantages of Our System vs. Phosphoramidite Mediated Oligonucleotide Synthesis</H2><SPAN>Figure 3. General Phosphoramidite Synthesis Cycle Scheme</SPAN></DIV>

Overall, there are several notable advantages of our system over the Phosphoramidite system. Our system is substantially more eco-friendly, as some of the reagents used in the Phosphoramidite system are considered toxic. For instance, Palladium (Pd) complex is used to remove the protecting group from the dNTPs. Pd is considered to have low toxicity because the absorbency rate in the body is relatively low. However when the Pd complex is used as a reagent, it is possible for its toxicity to increase. There have been cases where it has caused acute toxic effects in lab mice, including mutagenesis to their isolated hearts. Another toxic reagent used is Dichloroacetic acid (DCA). DCA has been listed by the Environmental Protection Agency as a cancer-causing agent in humans and is also known to cause infertility in men. Additionally, long term use is associated  with increased risk of liver cancer. Thus the use of these reagents is not cost effective since many precautions must be taken for their use. Permits must be obtained for their use, proper lab and safety equipment must be installed,  safety protocols must be followed, and the reagents must be disposed of properly. Another advantage our system is that it is faster overall. As shown above in Figure 3, one cycle for the Phosphoramadite system is comprised of four main steps, while ours has only two. This greatly increases the kinetics of the system since the time it takes to complete a full synthesis is equivalent to the time that it takes to complete one cycle multiplied by the number of base pairs that will be synthesized.  <DIV class="center"><H2>Advantages of Our System vs. pH Regulated De novo Enzyme Mediated DNA Synthesis</H2></DIV>
In the controllable system proposed by Ud-Dean, the reversibility of a 3' acetyl protective group was achieved by lowering pH during each cycle and thereby activating a deacetylase enzyme that is active at lower pH. During this step, TdT is also inactivated by the shift to lower pH, thereby preventing the addition of extra nucleotides.
<DIV class="center"><SPAN>Figure 4. General pH Regulated De novo Enzyme Mediated DNA Synthesis Cycle Scheme</SPAN></DIV>
Our system simplifies this novel approach by using thermolabile reversible 3' protective groups. Since our system uses heat to deprotect the dNTPs, only one step is needed instead of two, decreasing the time that is needed to add one base pair. This greatly increases the kinetics of the system since the time it takes to complete a full synthesis is equivalent to the time that it takes to complete one cycle multiplied by the number of base pairs that will be synthesized. 
<DIV class="center"><H2>Results</H2><SPAN>Figure 5. TdT Enzyme Functional Assay</SPAN></DIV>As seen from the polyacrylamide gel shown above in Figure 5, the TdT enzyme expressed in the Rosetta cells using our genetically engineered plasmid works as intended; base pairs are appended to oligos by the enzyme. However, there is a difference between the activity of our enzyme compared to the commercially obtained TdT. The majority of the oligos have only a few base pairs added by our expressed TdT while the lane of the commercial TdT is a smear of oligos, suggesting the commercial TdT  has a greater reaction rate. But it can also be attributed to the fact that the 0.5 U/μL concentration of our expressed TdT is an overestimation. The TdT concentration of the reaction with our expressed TdT is less than that of the commercial TdT. 

There is also a large gap between the oligos with only a few bases added and those with 30+ bases added. This suggests that our expressed TdT has a higher processivity than that of commercially obtained TdT. This might be due to subtle sequence differences such as the His tag on our TdT sequence.

<DIV class="center"><H2>Potential Errors in the Synthesis System</H2></DIV>

During each cycle there are independent sources of error that can affect the overall efficiency of the reaction. The most obvious errors include: 
<OL><LI>Incomplete enzymatic addition of 3' reversibly protected dNTPs by TdT during the nucleotide incorporation step.</LI><LI>Incomplete hydrolysis of 3' protective group during heating or other treatment.</LI><LI>Incomplete removal of unincorporated 3' reversibly protected nucleotides after each incorporation step.</LI><LI>Spontaneous hydrolysis of 3' protective groups prior to intended deprotection. Each error, if not sufficiently addressed would compound the likelihood of misincorporated nucleotides and a reduction of overall yield. </LI></OL>

For example, during error scenario 1, oligonucletide chains that have not had a 3' reversibly protective group nucleotide added during an incorporation step, will, during the subsequent step after deprotection, be potential substrates for the addition of the next incoming nucleotide. This would lead to the base deletion. During error scenario 2, the failure to remove a protective group would lead to a population of oligonucleotides that would fail to add a nucleotide during the next subsequent addition of incoming nucleotides, leading to a base deletion at this step. For error scenario 3, the presence of unremoved, unreacted nucletides would lead to misincorporation and hence base substitutions during each subsequent stage of oligonucleotide elongation. During Scenario 4, the presence of dNTPs without 3' protective groups would lead to the incorporation of multiple dNTPs, resulting in homopolymeric stretches of nucleotides in the growing oligonucleotide chain. This would lead to insertion mutations. For a truly reliable system to be built, these and potentially other sources of error will need to be adequately countered.
<DIV class="center"><H2>Future Improvements to the De novo Oligonucleotide Synthesizer</H2></DIV>
In our present configuration, full length recombinant bovine TdT was used in the addition of thermolabile dNTPs to single stranded oligos. This particular enzyme denatures under elevated temperatures. Therefore, after the blocking group is removed from the dNTP at 95°C, it is necessary to add more of the enzyme. As a result, it would be useful to either isolate a naturally occurring variant of TdT or a similar enzyme that is resistant to denaturation under high temperature conditions. This type of enzyme might either be found in a thermophilic organism or synthetically engineered in a laboratory. Another potential improvement would be to construct truncated and other variants of engineered TdT that would incorporate modified dNTPs at a faster rate. Lastly, our current system could be improved by switching from commercially available thermolabile nucleotides to photolabile nucleotides. This should improve the system since temperature ramp up and down cycles are avoided, creating a faster reaction cycle and eliminating the need to re-add the TdT enzyme at the beginning of each cycle. We are presently exploring these improved features. 
<B>References</B><OL><LI>Minhaz Ud-Dean, S.M. (2008) A Theoretical Model for Template-Free Synthesis of Long DNA Sequence. Syst. Synth. Biol. 2:67-73</LI><LI>Koukhareva, I. and Lebedev, A. (2009) 3'-Protected 2'-Deoxynucleoside 5'-Triphosphates as a Novel Tool for Heat-Triggered Activation of PCR. Anal Chem. 81(12):4955-4962</LI><LI>Kuan, W.L., Joy, J. Mee, N.F., Perlyn, K.Z., Wen, T.S., Nguen, T., James, J., Chai, E., Flotow, H., Crasta, S., Chua, K., Peng, N.S. and Hill, J. (2010) Generation of Active Bovine Terminal Deoxynucleotidyl Transferase (TdT) in E. coli. Biochemistry Insights. 3: 41-46.</LI><LI>Boule, J.B., Rougeon, F.Papanicolaou, C. (2001) Terminal Deoxynucleotidyl Transferase Indiscriminately Incorporates Ribonucleotides and Deoxyribonucleotides. J. Biol. Chem. 276, 33: 31388-31393.</LI><LI>Motea, E.A. and Berdis, A.J. (2010) Terminal Deoxynucleotidyl Transferase: The Story of a Misguided DNA Polymerase. Biochim. Biophys. Acta. 1804, 5: 1151-1166.</LI><LI>Gardner, A.F., Wang, J., Wu, W., Karouby, J., Li, H., Stupi, B.P., Jack, W.E., Hersh, M.N. and Metzker, M.L. (2012) Rapid Incorporation Kinetics and Improved Fidelity of a Novel Class of 3'-OH Unblocked Reversible Terminators. Nucleic Acids Research. 40, 15: 7404-7415.</LI><LI>Kruszewski, Marcin, Elzbieta Bouzyk, Tomasz Oldak, Krystyna Samochocka, Leon Fuks, Wlodzimierz Lewandowski, Izabela Fokt, and Waldemar Priebe. &quot;Differential Toxic Effect of Cis-platinum(II) and Palladium(II) Chlorides Complexed with Methyl 3,4-diamine-2,3,4,6-tetradeoxy-α-L-lyxo-hexopyranoside in Mouse Lymphoma Cell Lines Differing in DSB and NER Repair Ability.&quot; Teratogenesis, Carcinogenesis, and Mutagenesis 23.S1 (2003): 1-11. Web. 17 Oct. 2014.</LI><LI>Hayakawa, Yoshihiro. &quot;ChemInform Abstract: Toward an Ideal Synthesis of Oligonucleotides: Development of a Novel Phosphoramidite Method with High Capability.&quot; ChemInform 33.2 (2002)</LI><LI>Palladium - Pd.&quot; Palladium (Pd). N.p., n.d. Web. 17 Oct. 2014.</LI><LI>&quot;Palladium and Its Compounds Toxicity Reports, Review - Hazard Potential, Risk.&quot; N.p., n.d. Web. 17 Oct. 2014.</LI><DIV class="cu-footer"><DIV class="bottom-center"><A href="https://2014.igem.org/Team:Cooper_Union/Project" class="black"><SPAN>Projects</SPAN></A><A href="https://2014.igem.org/Team:Cooper_Union/Social" class="red"><SPAN>Social</SPAN></A><A href="https://2014.igem.org/Team:Cooper_Union/Notebook" class="green"><SPAN>Notebook</SPAN></A><A href="https://2014.igem.org/Team:Cooper_Union/biofanfic" class="blue"><SPAN>Team</SPAN></A></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Cooper_Union/TdT_project">http://2014.igem.org/Team:Cooper_Union/TdT_project</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cooper_Union/TdT_project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cooper_Union/TdT_project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cooper_Union/TdT_project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cooper_Union/TdT_project&amp;oldid=401348" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></OL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>