<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cooper_Union_Notebook_Biohack_July"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Cooper_Union/Notebook/Biohack_July">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cooper_Union/Notebook/Biohack_July&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cooper_Union/Notebook/Biohack_July&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cooper_Union/Notebook/Biohack_July&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cooper_Union/Notebook/Biohack_July" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cooper Union/Notebook/Biohack July</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Cooper Union 2014 iGEM</TITLE></P><DIV class="parent-container"><DIV class="menu-parent"><UL class="menu"><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union"> Home </A></LI><LI class="menu-project"> Project
		</LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/TdT_project">De Novo Synthesis </A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Telomere_project">Programmable Lifespan Timer</A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Biohack_project">Biohacker Kit </A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Hardware">OpenSource Hardware </A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Parts">BioBrick Parts </A></LI></UL><LI class="menu-safety"> Social
		</LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Safety">Safety</A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Ethics">Ethics</A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Outreach">Outreach </A></LI><LI class="menu-notebook">   Notebook 
		</LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Protocols">Commonly Used Protocols </A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Notebook"> Labnotes </A></LI><LI class="menu-team"> Team
		</LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/biofanfic">Members </A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Attributions"> Attributions </A></LI><LI class="menu"><A class="menu" href="https://igem.org/Team.cgi?id=1354"> Official Team Profile </A></LI></DIV><DIV class="text"><DIV class="notebook-header"><DIV class="left-side"><H1>Biohacker Kit</H1></DIV><DIV class="notebook-nav-container"><DIV class="month-links"><A href="https://2014.igem.org/Team:Cooper_Union/Notebook/Biohack_April">PRE MAY</A><A href="https://2014.igem.org/Team:Cooper_Union/Notebook/Biohack_May">MAY</A><A href="https://2014.igem.org/Team:Cooper_Union/Notebook/Biohack_June">JUN</A><A href="https://2014.igem.org/Team:Cooper_Union/Notebook/Biohack_July" class="selected">JUL</A></DIV></DIV></DIV><H3>7/1/14</H3>


Working with the input and output backbones that had negative concentrations on the nanodrop yesterday, we added 5μL of 3.0M potassium ethoxide and 100μL of ethanol. This was frozen overnight. In the morning, we spun it in the centrifuge at max speed for 15 minutes. Then the liquid was removed and more ethanol was added. It was again centrifuged at max speed for 10 minutes, and then the liquid was removed again. It was resuspended in 20μL of water. On the nanodrop, the input backbone had a concentration of -0.8ng/μL and the output backbone had a concentration of 3.2ng/μL.

Of the four inoculations set up last night, only the UV+pACYC184 and GFP+pBR322 looked cloudy this morning. Since we don't have any more plungers, the midiprep couldn't be done. The 50mL conical tubes were spun down for 15minutes at 3900rpm. The liquid was poured out, and they were each resuspended in 25mL of PBS. This was vortexed. Then, for each one, 5mL was poured into each of four 25mL conical tubes. This were then used for mini preps. They were spun down for 15 minutes at 3900rpm, the liquid decanted, and then protocol &quot;Plasmid DNA Purification Using the QIAprep Spin Miniprep Kit and a Microcentrifuge&quot; was used. The P1 was added, and then they were all moved into centrifuges. The protocol was followed and the four tubes of each were combined after eluting. Each was eluted with 50μL, so there is a total volume around 190μL. They were then nanodropped. The results were: UV+pACYC184: 87.6ng/μL and GFP+pBR322: 109.2ng/μL.

More of the old UV+pACYC184 and GFP+pBR322 was digested with KpnI and XhoI. (GFP+pBR322: 19.9 μL of plasmid which is 150.7 ng/μL, 3 μL buffer, 1 μL XhoI, 1 μL KpnI, 5.1 μL ddH2O. UV+pACYC184: 26 μL plasmid which is 113.7 ng/μL, 3 μL buffer, 1 μL XhoI, 1 μL KpnI.) (NOTE: The amounts of each used were switched than in previous times, because we had mislabeled the tubes. So this is really what we had been doing. GFP is more concentrated that UV.)

The HSP+pACYC184 colonies 1&amp;3 were inoculated again in the same beakers as yesterday and left to shake overnight at 37°C.
<H3>7/2/14</H3>

The digested UV+pACYC184 and GFP+pBR322 with KpnI and XhoI (input and output backbones) were run on a gel. They were then cut out and purified using the Qiagen gel extraction kit following the &quot;QIAquick Gel Extraction Kit Protocol&quot; protocol. However, we didn't have PB, so W9 from the Invitrogen kit was used instead. Each sample was split into two because the agarose gel pieces were too large for one. So, each of the four was eluted using 30μL EB buffer, for a total volume of each backbone a little less than 60μL. They were nanodropped, and the results were: Input backbone: 11.5ng/μL and Output backbone: 9.3ng/μL. Although these were MUCH lower than expected, we're just happy there's anything there.

The inoculated HSP+pACYC184 colonies 1&amp;3 were mini prepped as was done yesterday. They were each poured into 50mL conical tubes, spun down at 3900rpm for 15 minutes, discarded the supernatant, resuspended the pellet in 25mL PBS, split into four 25mL conical tubes with approximately 5mL in each. Then the protocol &quot;Plasmid DNA Purification Using the QIAprep Spin Miniprep Kit and a Microcentrifuge&quot; was followed for all of them. Each set of four epindorfs were combined when the elution was complete. They were each eluted with 50μL for a total volume of each around 200μL. They were nanodropped and the results were: HSP+pACYC184 colony #1: 109.3ng/μL and HSP+pACYC184 colony #3: 95.9ng/μL.

We received our pBad and Vio operon primers that were designed to use for colony PCR and that add KpnI and XhoI sites to the BioBrick parts. Nolana made 1 in 10 dilutions of all four (forward and reverse for each). 

The PCR machine was run on setting #130 (&quot;PCR settings 6/20/14&quot;) for the pBad primers. We couldn't run it with Vio Operon also, because Vio Operon is around 7Kb so it needs a longer extension time. The PCR was set up by poking one colony on the plate: 2014P3:W19O (pBad), vortexing that in 200μL of water. 20μL of this was put into a PCR tube along with 2.5μL each of the forward and reverse 1 in 10 dilution pBad primers.

A ligation was set up overnight, left in the PCR machine at 16°C. We ligated the HSP cut with KpnI and XhoI (conc: 16.4ng/μL) into the input backbone (UV+phage activator, conc: 11.5ng/μL). We used 0.65μL of HSP, 4.3μL of input backbone, 12μL water, 2μL ligase buffer, and 1μL T4 ligase. 
<H3>7/3/14</H3>

The pBad promoter was purified. I eluted with 30μL. Then it was nanodropped, and the results were 33.8ng/μL. The purified piece was run on a 1.5% gel which only had one band right around 150bp as expected (2μL of PCR product was run.)
The pBad promoter was digested with KpnI and XhoI for one hour. 3μL of buffer, 1μL of each enzyme, and 25μL of pBad (845ng). 

The pBad promoter was ligated with the input backbone (pACYC184+phage activator.) 0.2μL pBad was used, 4.3μL input backbone, 2μL buffer, 1μL ligase, and 12.5μL water for a total of 20μL.

HSP that was ligated into the input backbone was transformed with NEB 5&amp;α; compotent cells onto two chlor 50μg/mL plates. Their protocol was used, which is the same as &quot;General Transformation Protocol&quot; excpet it heat shocks for 30 seconds, uses 950μL of SOC, and when it incubates at 37°C for one hour, it is done on a shaker platform. The same protocol was used to co-transform HSP+pACYC184 colonies 1 and 3 with GFP+pBR322 onto chlor(50μg/mL) amp plates.

 

<H3>7/7/14</H3>

The inoculated UV+pSB1C3 colony #2 was mini prepped. It was poured into a 50mL conical tube, spun down at 3900rpm for 15 minutes, discarded the supernatant, resuspended the pellet in 25mL PBS, split into four 25mL conical tubes with approximately 5mL in each. Then the protocol &quot;Plasmid DNA Purification Using the QIAprep Spin Miniprep Kit and a Microcentrifuge&quot; was followed for all of them. However, they were accidentally eluted into the tubes with the white pellet that was garbage and shouldn't be used. So, the liquid was taken out. 1mL of P1, 1mL of P2, and 1.4mL of N3 was combined in a separate tube.700μL of this was poured into the four tubes of liquid taken out from the white stuff. Then the procedure was followed again, centrifuging and washing...Each was eluted with 50μL and nanodropped. The results were very poor. So, we inoculated more of it overnight to mini-prep again tomorrow.

The vio operon PCR products were purified and eluted with 30μL. It was nanodropped and the result was 5.5ng/μL with a really high (2.2) 260/280 probably because the PCR wasn't very clean and efficient because the vio operon is so large.

The two colonies of HSP/GFP+pACYC184/pBR322 that were co-transformed with DH5α only had one colony, so they were co-transformed again using the NEB 5α compotent cells onto two amp/chlor 50μg/mL plates. Their protocol was used. Also, the pBAD ligated into the input backbone (pACYC184) was transformed the same way and put onto chlor50μg/mL plates.

 

<H3>7/8/14</H3>

The inoculated UV+pSB1C3 colony #2 was mini prepped. It was poured into a 50mL conical tube, spun down at 3900 for 15 minutes, discarded the supernatant, resuspended the pellet in 25mL PBS, split into four 25mL conical tubes with approximately 5mL in each. Then the protocol &quot;Plasmid DNA Purification Using the QIAprep Spin Miniprep Kit and a Microcentrifuge&quot; was followed for all of them. .Each was eluted with 50μL and nanodropped. The result was 5.1ng/μL. So we inoculated UV+pSB1C3 #6 in 50mL of SOC and chlor. 

10 colonies from the HSP+input backbone (pSB1C3) were colony PCRed. A poke of one colony was mixed in with 200μL of water, vortexed and 20μL of this was used for PCR with 2.5μL each of the forward and reverse 10X sequencing primers. 

Two 1.2% gels were poured.

UV and GFP g-blocks were digested with KpnI and XhoI. 3μL of each (around 600ng) were digested with 1μL of each enzyme, 3μL buffer, and water up to a total volume of 30μL. This was digested for 1 hour. Then heat inactivated for 5 minutes. Professor Medvedik purified it. 

<H3>7/9/14</H3>

We used the 2 1.2% gels to run the 10 HSP with the insert backbone colonies that we chose. It seems that there were at least 6 colonies that had the insert so we chose colonies 1 and 2 and inoculated overnight them to be mini prepped tomorrow. 

We also mini prepped the UV+pSB1C3 colony 6. There was one tube that had a crack so we had to get rid of it and then some of the white chunky stuff fell into one of the spin column so there was assumed to be a lower yield then there should have been. We also realized that the PE buffer that we had been using never had the ethanol added to it, which could be why all of our other mini preps were not going well. The plasmids were combined and it was nanodropped, showing 43.8 ng/μL of plasmid and we have 150 μL of it (because we had 3 spin columns and eluted with 50 μL).

We also sent out the HSP+pACYC184 colonies 1 and 3 to be sequenced. Because we are sending them out as plasmids we took 5 μL of plasmid (both were ~100 ng/μL) + 5 μL water + 5 μL 5X primer. There were 4 PCR tubes, with SLO1 containing HSP 1 with forward, SLO2 containing HSP 1 with reverse, SLO3 containing HSP 3 with forward and SLO$ containing HSP 3 with reverse. 


 

<H3>7/10/14</H3>

We mini prepped the HSP+input backbone colonies 1 and 2, from our 50mL inoculation. It was poured into a 50mL conical tube, spun down at 3900 for 15 minutes, discarded the supernatant, resuspended the pellet in 25mL PBS, split into four 25mL conical tubes with approximately 5mL in each. Then the protocol &quot;Plasmid DNA Purification Using the QIAprep Spin Miniprep Kit and a Microcentrifuge&quot; was followed for all of them. .Each was eluted with 50μL and nanodropped. The results were, colony 1: 3.2ng/μL or 7.5ng/μL, and for colony 2: 8.2ng/μL or 12.6ng/μL. We nanodropped twice, the first time before vortexing and the second time after vortexin. Dionne noticed that the 260/280 was very high so this may mean we had too much DNA so that could explain the low yields. 

We put tips into boxes.

We cleaned some Qiagen mini prep columns. 

We spun down all of the primers we got for 10 mins on max speed, then we resuspended them with 1 X TE buffer. We then made 10X dilutions of that by diluting 10 μL of plasmid in 90 μL of water.

 

<H3>7/11/14</H3>

Wanted to run a co-transformation with HSP+input backbone 1&amp;2 but prof. will not be in until late saturday so it doesn't make sense. We shall run it on monday at some point. (Maybe also transform the vio operon??)

Ran a PCR with the new primers we got for the input and output backbones. Because we are PCRing something that is in the 4-5 Kb range, we are not going to use the PCR beads and we are going to use the Q5 master mix as the taq polymerase/buffer/dNTP combo for the PCR. I took 12.5 μL of the Q5 master mix and put in 2.5 μL of the forward primer, 2.5 μL of the reverse , 5 μL of a 1000X diluted plasmid (used a GFP+pBR322 and UV+pACYC184 both at ~100-150 ng/μL), and 2.5 μL of water to get it to a 25 μL total volume. 

The settings for the PCR machine were edited a little. The settings were changed to:
<DL><DD>init denat: 95°C  300s</DD><DD>denaturation: 95°C  30s</DD><DD>annealing: 55°C  30s</DD><DD>extension: 72°C  200s</DD><DD>final extension: 72°C  350s</DD><DD>cycles: 30</DD></DL>
Also re-streaked 2 plates of the 1 colony that was found on the HSP pACYC184 +GFP pBR322 co-transformed plates and put it in the incubator. 

We got the sequences back from genewiz for colinies 1 and 3 of th eHSP+pACYC184. The HSP 1 forward was 98% accurate and the reverse was 99% accurate and for HSP colony 3 the forward was 97% accurate and the backward was 98% accurate. When looking at the sequences there seem to be a few point mutations in it so we are hoping that it will not affect the functionality of the system. 

 

<H3>7/14/14</H3>

Shoshana looked at the sequences again and also checked the free repeats that we got. I think that all of the weird point mutations except for one were correct on the other. So we are hoping that everything in the HSP+pACYC184 colony 1 and 3 department are good. 

We also ran a gel but then someone switched the leads or didn't check and everything ran off. So we ran another gel .
 It had the input and output backbones that we PCR'ed on friday on it to check it. The gel only showed a band for the output backbon. Therefore, there might be an annealing temp. problem. Also, we are going to look at the primers that were made to make sure they are good and it's not a primer problem. 

We also did a co-transformation of the mini-prepped HSP+input backbone with the GFP+pBR322 into the ultra competent cells. We followed the transformation protocol. We added 1 μL of each plasmid to 50 μL of the ultra competent cells and had them sit on ice for a half hour then heat shocked for 30 seconds in 42°CC water bath and added 900 μL of SOC to each tube and let them incubate on a shaker platform for an hour and then plated 200 μL of each on a chlor-amp plate and put it in the incubator overnight.
 

<H3>7/15/14</H3>

Today we checked on our co-transformations and we had a bunch of colonies! ! ! Yay.

We then colony PCR'ed colonies 2-5 on the plate of pBad promoter that we had along with a GFP, UV, and HSP. This was put into the new OpenPCR machine, and the settings that were used are below:
<DL><DD>initial step: 95°C  30s</DD><DD>anneal: 56°C  30</DD><DD>extention: 72°C  30</DD><DD>final hold: 4°C</DD><DD>final step: 72°C  420s</DD></DL>

*****THIS PCR RAN GREAT!!! THE SETTING HERE WERE GOOD FOR PBAD-GFP SIZES!******

We ran a gel to check the PCR'ed colonies mentioned above. All of the PCR'ed products came out clean without a band. One gel had the UV, HSP of the T-team's one colony on their plate, and two GFP's. The other had the last two GFP's and the last 5 wells were PBad, with the first of the pBad wells being a negative control (because we put in the wrong primers). 
<H3>7/17/14</H3>

Ran PCR of HSP+pACYC184 and GFP+pBR322 to amplify the input and output backbones. Trying HSP because UV didn't work, and making more GFP. We used 5μL 1000x dilution of plasmid, 2.5μL water, 2.5μL forward primer, 2.5μL reverse primer, and 12.5μL Q5 McMaster Mix.
<DL><DD>init denat: 95°C  300s</DD><DD>denaturation: 95°C  30s</DD><DD>annealing: 54°C  30s</DD><DD>extension: 72°C  270s</DD><DD>final extension: 72°C 350s</DD><DD>cycles: 30</DD></DL>

Also, for three of the four HSPs (used HSP+pACYC184 #1), I added Taq polymerase (1,2,3) instead of the Q5 McMaster mix.

Also we inoculated some of the HSP+pACYC184 #3 in 5 mL of LB media with 7.35 μL (34 mg/mL) of chlor to get a conc. of 50 μL/mL overnight so that we can do more testing 

Also Devora accidentally diluted the stock of HSP+pACYC184 #3 and now it is 17.5ng/μL concentration. Then, we made a 1 in 1000 dilution of this, so it's around 17pg/μL. We also diluted HSP+pACYC184#1, with a 1 in 1000 dilution, so it's around 100pg/μL now.

 

<H3>7/18/14</H3>

We ran a PCR of HSP+pACYC184 both colonies 1 and 3 so we could send them out for sequencing. We already sent them out as plasmids, but the reads weren't very good, so we're trying again and sending out PCR products.
We transformed the kids BioBricks that didn't transform for them using compotent DH5α cells, all onto chlor 50μg/mL plates.

We ran the four input backbone and four output backbone PCRs from yesterday on a gel to see if they actually were amplified. The results can be seen in the gels file for today. The inputs still didn't PCR but the outputs did.

We also took some of the inoculated 5 mL of HSP+pACYC184 and struck it out on a plate so we have some. 

Then we took the 5 mL that we inoculated to do testing and did a mini prep on it because we realized we didn't inoculate anything else so we couldn't run anything. We followed the protocol for the quiagen mini prep. We then nanodrpped it and it was 12.3 ng/μL. 



<H3>7/21/14</H3>

We PCRed more HSP+pACYC184 and UV+pACYC184 with the input backbone primers to try to successfully amplify the backbone. We used 5μL 1000x dilution of plasmid, 2.5μL water, 2.5μL forward primer, 2.5μL reverse primer, and 12.5μL Q5 McMaster Mix. The settings we used were:
<DL><DD>init denat: 95°C  300s</DD><DD>denaturation: 95°C;  30s</DD><DD>annealing: 50°C  30s</DD><DD>extension: 72°C  300s</DD><DD>final extension: 72°C  350</DD><DD>cycles: 30</DD></DL>
We'll see what happened tomorrow.

We also used DpnI on the five output backbones that we had previously PCRed successfully to chop up the template before we purify it. We used the 25μL of PCR, 19μL of water, 5μL Cut Smart buffer, and 1μL DpnI. This went in 37degree waterbath for 1 hour. 

We also ran the HSP experiment. We had inoculated in 5mL LB+antibiotic four samples: DH5α, HSP+pACYC184, GFP+pBR322, and HSP/GFP+pACYC184/pBR322. We made 1/10 and 1/20 dilutions of all four. For the 1/10 we used 4.5mL LB, the correct antibiotic, and 500μL of sample. For the 1/20 dilutions, we used 4.75mL LB, the correct antibiotic, and 250μL of sample. We took OD600 readings and found that the 1/20 dilutions were closer to one. So we split these into 2 tubes, 2.5mL in each. These were put in the shaker for forty minutes and OD600s were taken again. This was repeated, checking OD600s, until the concentration had doubled for all 8 samples. (The combo ones took the longest to rise). Then, we put the four heat shock ones into a 45degree water bath for 15minutes. The other four (controls) sat in 37°C (withOUT shaking) while they were in the water bath. Then they all went in the shaker (shaking) for 20 minutes, and then OD<SUB>600</SUB> and GFP readings were taken.

A preliminary GFP reading was taken while we were getting the concentration up to double, to check for background fluorescence, and found that the GFP wasn't that high yet. Actual numerical results are in an Excel sheet in &quot;HSP Experiment-7/21/14&quot;

 

<H3>7/22/14</H3>

We poured 2 1.5% gels to use tomorrow

We purified the PCR of the output backbone that had Dpn1 added to it and the nanodropped. We have 47.7 ng/μL and 50 μL. 

 

<H3>7/28/14</H3>

We ordered new primers for the input backbone. So we ran a PCR with them on UV and HSP#1 + pACYC184. We used 5μL 1000x dilution of plasmid, 2.5μL water, 2.5μL forward primer, 2.5μL reverse primer, and 12.5μL Q5 McMaster Mix. The settings were:
<DL><DD>init denat: 95°C  300s</DD><DD>denaturation: 95°C  30s</DD><DD>annealing: 50°C  30s</DD><DD>extension: 72°C  300s</DD><DD>final extension: 72°C  350s</DD><DD>cycles: 30</DD></DL><H3>7/29/14</H3>

The new primers were run on a gel and it looks like HSP#3+pACYC184 had a band indicating that the input backbone had been amplified. 

 We used an annealing temperature of 50°C so there were some other bands, but we believe it was amplified correctly as well. So we digested that with DpnI (added 21μL water, 5μL CutSmart, and 1μL DpnI directly to the PCR tube, 37°C for 40minutes and 80 for 20 minutes) and then split that into two tubes to digest with KpnI and XhoI (25μL PCR stuff, 3μL CutSmart, 1μL KpnI, 1μL XhoI).  We also digested with KpnI and XhoI the output backbone that we had previously PCR amplified and treated with DpnI. These were then purified, eluted with 50μL and nanodropped. The results were: input backbone: 26.2ng/μL and output backbone: 25.3ng/μL.

Then we phosphatase treated both input and output backbones, splitting each into two tubes using 25μL backbone, 3μL phosphatase buffer, and 1μL phosphatase. The two of each were combined, purified, eluted with 50μL and nanodropped. The results were: input backbone: 12.2ng/μL and output backbone: 12.2ng/μL.


</DIV></DIV><DIV class="cu-footer"><DIV class="bottom-center"><A href="https://2014.igem.org/Team:Cooper_Union/Project" class="black"><SPAN>Projects</SPAN></A><A href="https://2014.igem.org/Team:Cooper_Union/Social" class="red"><SPAN>Social</SPAN></A><A href="https://2014.igem.org/Team:Cooper_Union/Notebook" class="green"><SPAN>Notebook</SPAN></A><A href="https://2014.igem.org/Team:Cooper_Union/biofanfic" class="blue"><SPAN>Team</SPAN></A></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Cooper_Union/Notebook/Biohack_July">http://2014.igem.org/Team:Cooper_Union/Notebook/Biohack_July</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cooper_Union/Notebook/Biohack_July" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cooper_Union/Notebook/Biohack_July" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cooper_Union/Notebook/Biohack_July&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cooper_Union/Notebook/Biohack_July&amp;oldid=377262" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>