<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cooper_Union_Notebook_TdT"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Cooper_Union/Notebook/TdT">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cooper_Union/Notebook/TdT&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cooper_Union/Notebook/TdT&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cooper_Union/Notebook/TdT&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cooper_Union/Notebook/TdT" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cooper Union/Notebook/TdT</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Cooper Union 2014 iGEM</TITLE></P><DIV class="parent-container"><DIV class="menu-parent"><UL class="menu"><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union"> Home </A></LI><LI class="menu-project"> Project
		</LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/TdT_project">De Novo Synthesis </A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Telomere_project">Programmable Lifespan Timer</A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Biohack_project">Biohacker Kit </A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Hardware">OpenSource Hardware </A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Parts">BioBrick Parts </A></LI></UL><LI class="menu-safety"> Social
		</LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Safety">Safety</A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Ethics">Ethics</A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Outreach">Outreach </A></LI><LI class="menu-notebook">   Notebook 
		</LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Protocols">Commonly Used Protocols </A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Notebook"> Labnotes </A></LI><LI class="menu-team"> Team
		</LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/biofanfic">Members </A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Attributions"> Attributions </A></LI><LI class="menu"><A class="menu" href="https://igem.org/Team.cgi?id=1354"> Official Team Profile </A></LI></DIV><DIV class="text"><DIV class="notebook-header"><DIV class="left-side"><H1>De Novo Synthesis Lab Notebook</H1></DIV><DIV class="notebook-nav-container"><DIV class="month-links"><A href="#" class="selected">JUN</A><A href="#">JUL</A><A href="#">AUG</A><A href="#">SEP</A></DIV></DIV></DIV><H3>6/3/14</H3>

Transformed DH5alpha with pET28b (+) vector on two plates, a Kan and an Amp. They were then left to grow overnight.

<H3>6/4/14</H3>



The plates were checked first thing in the morning and no colonies were found. Therefore we went through the procedure of transforming DH5alpha with new C media and two different dilutions of the pET28b (+) vector as well as two control strains(pET26 (+) in DH5alpha, pEt28 + CDS7) that were known to grow on Kan plates. They were then left to grow overnight. This time we did not use the vortex for mixing and were much more gentle with the cells.

We also transformed DH5alpha with 6 different BioBrick plasmids (pSB1A3, pSB1C3, pSB1K3, pSB1T3, pSB2K3), two BioBrick RBS (B0030, B0034), and a BioBrick double terminator (B0015). They were then left to grown overnight.

Three 1% agarose gels were then prepared, with one containing about 100mL of mixture and the other two containing about 50mL of mixture.

Tomorrow, we need to check for cell growth on all plates and further grow any colonies that have formed.

<H3>6/5/14</H3>



First thing in the morning, we checked all plates and found that colonies had grown on all of them. Colonies from three of the plates (DH5alpha, DH5alpha and pET28b+, and DH5alpha and pSB1C3) were extracted and allowed to incubate overnight. Parafilm was used to secure the other plates so that they can be incubated at a later time.

<H3>6/10/14</H3>

Yesterday, Wilfrido and Joe took the three minipreps of pET28b and three minipreps of pSB1C3 and cut them using the enzymes Xho1 and Xba1 respectively to create three new samples each.




Today, we created a 0.8% agarose gel for the pET28b samples and a 1.5% agarose gel for the pSB1C3 samples. Once prepared, the gels were run. The gels could be read from left to right as: ladder, uncut 1, uncut 2, uncut 3, cut 1, cut 2, cut 3. There was an extra band at the top of all of the samples which we suspect is due to the fact that we used 100x BSA instead of 10x BSA. All of the samples appeared normal except for the third uncut sample of pET28b, which was missing a band. As a result, we discarded this sample. 

<H3>6/11/14</H3>

A couple of 1.0% Gels were made. 

<H3>6/12/14</H3>

Today, we realized that the cells that we transformed with  the pSB1C3 and pSB1T3 plasmids were not done properly due to the fact that they did not express the red protein. We attributed this fault to errors with the BioBrick plates. As a result, we retransformed DH5alpha with these same plasmids but from other wells and left them to incubate overnight. We also made six tetracycline plates. 

<H3>6/13/14</H3>

The cultures of the pSB1C3 and pSB1T3 once again appear to not have been successful. The pSB1C3 grew many small colonies none of which expressed the red protein and the pSB1T3 did not grow any colonies. We believe that the wells have defective plasmids. The wells are 2014, plate 1, well 23O; 2011, plate 1, well 7A. 

<H3>6/16/14</H3>

We took two cultures (pSB1A3 and pSB1K3) that showed colonies that expressed the red protein and are growing them overnight in LB broth with their respective antibiotics.

Double restriction digested pSB1C3 and pSB1T3 which are both linearized with enzymes EcoR1 and Pst1. The concentrations are 10ng/ul. The total volume for each was 20 microliters.

<H3>6/17/14</H3>

The cells grown overnight were checked and were confirmed to have the correct plasmids, pSB1A3 and pSB1K3, due to the red color of the LB solution. A 1mL miniculture of equal part cells and a 60% glycerol stock were prepared for each sample and stored in the -80C freezer.      

The rest of the overnight cultures, three samples of DH5alpha with pSB1A3 and three samples of DH5alpha with pSB1K3, were minipreped following the Miniprep Protocol with some exceptions. The 5mL of overnight culture was centrifuged at 3900rpm for 10 minutes for the first step of preparing the cell lysate. For the second step of eluting the DNA, 50ul of preheated TE buffer was added to the column instead of the stated 75ul.   

After the plasmids were isolated, the concentration was measured using the nanodrop. Samples 1, 2, and 3 of pSB1A3 was measured to be 142, 211, and 197 ng/ul, respectively and samples 1, 2, and 3 of pSB1K3 was measured to be 139, 161, and 152 ng/ul. The plasmids were stored at -20°C.   

Double restriction digest reactions were prepared for each sample of pSB1A3 and pSB1K3 with the restriction enzymes EcoR1 and Pst1. An error was made in the procedure in which 100X BSA was added instead of a diluted 10X BSA. The solution concentrations are 15ng/ul with a total volume of 20 ul.  

<H3>6/18/14</H3>



 

We ran the gel to verify that the isolated plasmids are pSB1A3 and pSB1K3 due to their base pair lengths and that the digestion was successful. We then ligated both plasmids with J04450. After we transformed the plasmids into DH5alpha and left it to grow on plates overnight.  

<H3>6/18/14</H3>

The cells left to grow overnight grew colonies successfully except the red protein was not expressed. The plates were parafilmed and left in the fridge to allow additional time for the cells to produce the red protein. Due to the colonies having similar traits to the first colonies that were transformed with pSB1C3 and pSB1K3, large amount of colonies and small colony sizes that failed to express the red protein, it is suspected that the competent cells used were contaminated with another type of  bacterial. 

DH5alpha competent cells from new stocks were re-transformed with the plasmids as well as using a 100:1 LB/glucose solution in place of the 1X SOC and were left to grow overnight. Two controls were also prepared: cells from the old competent cell stock were prepared in the LB/glucose solution and grown on a chloramphenicol plate and cells from the new competent cell stock were prepared in the old SOC media grown on a chloramphenicol plate.    

A double digestion reaction was prepared for pET28b+ with the restriction enzymes BamHI and NdeI. The cut plasmid was stored in the -20C freezer.

Three samples of DH5alpha colonies transformed with the pET28b+ plasmid that were previously grown on a plate were minicultured and left to grow overnight on the shaker. 

<H3>6/20/14</H3>



Despite being in the fridge overnight, the colonies transformed with the ligated pSB1C3 and pSB1T3 on plates did not start expressing the red protein marker. This suggests that the colonies were in fact not DH5alpha but a bacteria resistant to various antibiotics.   




Only one of the four plates incubated overnight grew colonies. The cells from the old competent cell stock prepared with a LB/glucose solution grew on a chloramphenicol plate while the cells from the new competent cell stock prepared in the old SOC media did not grow any colonies. The colonies that grew has similar colony sizes and numbers with the unknown strain of bacteria which suggests that the old competent cells are indeed contaminated. 



The DH5alpha from the new competent cell stock without any plasmids, as expected, did not grow on the chloramphenicol plate. However, the cells transformed with pSB1C3 also did not grow despite being plated on a less concentrated antibiotic plated (50ug/ml instead of 150ug/ml). The cells transformed with pSB1T3 on a tetracycline did not work as well. The plan for the future is to retry growing cells transformed with the plasmids with new competent cells while also transforming cells with the plasmids in the transformation efficiency kit to test if the transformation protocol is correct.     




Two more restriction digest reactions were prepared with 100ng of DNA each; a single digest of the pET28b+ with BamHI and another single digest with NdeI. Two 0.8% gels were ran, each with each sample of the plasmid with an uncut plasmid, double digested plasmid and the two single digested plasmid. The greater intensity of the double digested DNA bands are due to the fact that the concentration of the double digested DNA is several folds greater than that of the uncut and single digested DNA. As seen from the ladder, the plasmid is a bit greater than 5.3kbp which matches the known length of pET28b+, confirming the identity of the plasmid. The similar lengths of the single digested plasmid suggest that both enzymes are probably working however, from the large amount of uncut DNA in the double digested DNA lane, the enzymes are probably as active as it should be. For future digestions with BamHI and NdeI, fewer DNA will be used as well as longer incubation times to assure the digestion runs to a completion. 

After the confirmation of the plasmid, the three minicultures prepared were minipreped. The isolated pET28b+ were stored in the -20C freezer.

<H3>6/23/14</H3>

Prepared Ampicillin plates and two 0.8% gels. Additionally a 6% gel was made in preparation for the CleanAmp TdT experiment which will be run tomorrow. We were going to run the experiment today but it still needs to be tweaked in order to get usable and distinguishable results on the gel. 

(JWP: double digested pET28b+ with Nde1 and BamH1-HF for 3hrs, and stored in the freezer. Will run gel with other samples tomorrow)

<H3>6/24/14</H3>

Today we performed four experiments using the commercially purchased TdT, dNTPs, and Clean Amp heat labile dNTPs. The protocol &quot;Clean Amp Experiment Protocol&quot; was followed for two of these experiments while the other two were used as control. For the positive control, TdT was mixed with the dNTPs in TdT buffer along with CoCl2 and ddH2O and allowed to incubate in a 37 C water bath for 30 minutes. The reaction was stopped by placing the mixture in a 70 C water bath for 10 minutes. For the negative control, the same procedure as for the positive control was followed without the addition of dNTPs. All of the experimental products were stored at -20 C. We plan on running the products through a 6% gel tomorrow morning so that it can run all day long.

The double digested pET28b vectors from yesterday were run on a 0.8% gel along with the original uncut pET28b vector and the single digested pET28b vectors.

The double digested pEt28b vectors that were cut out of the agarose gel on Friday were gel extracted and placed in the -20C freezer.



<H3>6/25/14</H3>

Had Gel electrophoresis of CleanAmp tdt experiment from 6/24, with 6% gel, 30V, for 5 hrs and 22 mins.
- As a ladder 0.34 uL of 33 Mer, 2.5 uL 52 Mer, 0.43 uL of 42 Mer, 0.59 uL of 24 Mer were added with 21.14 uL of ddH2O, and 4.2 uL of 6x dye was added.
- To see the results, stained with the solution of 500 uL of Ethidium Bromide(10ug/ml) in 400 mL water. 400 uL was added first with 400 mL of water and stained for 10 minutes, but 100 uL was added later and stained for another hour and fifteen minutes.
- Because of the low voltage, the shape of the wire was influencing the overall shape.

Built truncated tdt primers with the reference of 2004 Rephasky paper. The four sequences are uploaded to Google Drive under Tdt Info.

Miniprepped PSB1C3 (1&amp;2) according to Invitrogen protocol with centrifugation and  were stored in the box. 

Additionally a 6% gel was made in preparation for additional cleanAmp tdt experiment

<H3>6/26/14</H3>

Checked the picture of the 6% gel from yesterday, and repeated the same CleanAmp Tdt experiment with newly arrived DNTPs. The procedures were the same, but we divided the experiment into two different DNTP's, one with A and one with G. This time we made sure to add Tdt to the controls. Ladders were also prepared same as last time, but dye has to be added before running the gel (which will be 4.2 uL).

<H3>6/30/14</H3>
Because of a mistake the gel was run in the opposite direction, this did not leave us with enough to re-run the gel so the CleanAmp TdT experiment was repeated exactly as it was done on 6/26. The plan is to run these results on the same gel first thing tomorrow morning. Additionally a preliminary outline of the procedure for the actual TdT experiment was made but still needs significant revisions and more research before it is feasible to do.

<H3>7/1/14</H3>

We first ran the 6% gel from yesterday in the order of ladder, +A, +G, -, 1A, 1G, 2A, and 2G.

Then we phosphatase treated the double digested pET28b+ (11.1 ng/uL). 17 uL of pET28b+ was mixed with 2uL buffer and 1uL antartic phosphatase, incubated for an hour at 37'C, and denatured at 70'C for 5mins. The new concentration calculated is 9. 44 ng/uL. 

20uL of TE buffer was added to Gblock and spinned down, chilled for 10min ice, and double digested with Nde1 and Bam H1-HF. Phos-treated pET28b+ was then ligated with digested Gblock. 10uL of Gblock and 10uL of vector (instead of 5uL) were put, and the total volume was 25uL instead of 20uL. Negative control with vector without insert but with ligase should be set up tomorrow. 

PCR was prepared with 1:10 diluted tdt primers prepared on the 18th, but because the machine was not working, we will run PCR tomorrow. Transformation should be also done tmrw according to NEB 10beta Ecoli protocols. 

<H3>7/2/14</H3>

Checked the 6% gel from yesterday, not worked. We bearly see anything, and it was really faint in general. Should troubleshoot and go over the protocols. 

We ran the PCR reaction from yesterday, with (95'C 30&quot;, 55'C 30&quot;, and 72'C 2')*30 cycles. After PCR reactions were done, we ran the gel with the killswitch team. 

We set up the negative control with 10uL vector without inserts but keeping everything the same. Then the transformation was done with the pET28b Tdt ligation sample from yesterday. Used NEB 10 Beta Ecoli cells and we followed the exactly same protocol that NEB provided. 5 uL of plasmid DNA was added to each tube, 200 uL of mix were added each to the Kan plate (2 for negative control and 2 for pET28b+ Tdt ligation), and incubated at 37'C overnight.

<H3>7/3/14</H3>

Today we re-ran the PCR reaction and checked it using gel-electrophoresis. 1% gel and 1kb ladder was used. Gel did not show anything, so we are planning to run gradient PCR next week. 

We also re-ran the ligation reaction, however, this time we are allowing it to run overnight at 16'C. Professor Medvedik will come in tomorrow morning to stop the reaction. we used 10ng of vector and 30ng of insert instead of 50 and 75. 

<H3>7/7/14</H3>

Did the transformation of a ligated sample from yesterday. Because negative control was not set up (although we were supposed to set one when we are doing the experiment...), last experiment's negative control was used from the stock. Same protocol was used, and was stored in 37'C incubator. 

There might be a problem in the primer design itself, since the IIDT site's program says the primers might be binding to each other. To minimize that, PCR was re-ran with increased cycles (40 instead of 30 cycles), and annealing temperature was lowered to 53'C, but time was increased to 45&quot; instead of 30&quot;. DMSO was added to prevent dimers. Sample 1 is 50pg DNA with primers, 2 is 50pg DNA with primers with 1% DMSO, and 3 is with 5% DMSO. Along with variation of DMSO concentrations, two different PCR reactions were prepared, one with PCR beads and the other one with Taq solution. 

pSB1C3(minipreped on 6/25, sample1, nanodropped and concentration was 71.1ng/uL) and Tdt(diluted: 20pg/uL) were double-digested with EcoR1-HF and Pst1. The sample was incubated for an hour and ligated with T4 ligase. The ligation sample was then transformed to NEB 10 Beta competent cells. As a control and glycerol stock, pET28b+ was also transformed to DH5a cell on a Chlo plate. Same protocol was followed. 

<H3>7/8/14</H3>

We first ran a gel with PCR samples from yesterday. It seems like primers went through to the end, and we did not see any amplified templates. But we did see primers bound to each other. 

Transformation yesterday did not work, but pSB1C3+tdt ligation sample transformed to NEB10beta cells showed several colonies (though it seems most of them are red). Colony PCR was set up with the colonies on the plate. 6 colonies from each plate were pick and resuspended to 5uL ddH2O. 2.5uL was used for plate, and with the other 2.5uL PCR reaction was set up with the cycle of 95'C 6min, 30*(95'C 6min, 55'C 30sec, 70'C 30sec), 70'C 5min, held for 4'C, using primer VF2(10uM, 0.5uL) and VR(10uM, 0.5uL)

Double digestion of Tdt and pET28b+ was set up. Orginal Gblock sample was digested with Nde1 &amp; BamH1-HF and with EcoR1-HF &amp; Pst1. pET28b+ was digested with Nde1 and BamH1-HF. incubated for an hour, and ligated. 

pSB1C3 + TdT ligation 1 and 2 respectively



 



<H3>7/9/14</H3>

Today we transformed the ligation of pSB1C3+TdT and pET28B+ + TdT and spread it on plates and left it overnight. We also made a 1:1000 dilution of the Gblock so it has a concentration of 44pg/uL and used it to run the 3 PCR beads and 3 PCR Taq solutions. We increased the annealing temperature from 53 to 54, the rest remained the same. 

On the pET28B+ + TdT ligation plate a colony grew and this colony was placed in broth and incubated overnight. Additionally, two minipreps of pET28B+ were done.

<H3>7/10/14</H3>

Today we minipreped the pET28B+ +TdT ligation that was left to incubate overnight. We also made two 0.8% gels and 2 1% gels. We also ran a gel of the 6 PCR reactions that ran yesterday and found very faint bands at about 1.6 kb, which indicated that our TdT was amplified but very inefficiently. It also appeared that the PCR bead with the 5% DMSO showed the heaviest band. In the near future, we will run more experiments on this PCR using 5% DMSO to optimize the reaction. 

When we analyzed the plates from yesterday, we found that the pET28b+ +TdT ligation yielded many white colonies and the pSB1C3 + TdT ligation yielded many red colonies, which indicate that the TdT gene was not incorporated, with a few white colonies, which could be either the desired product or an unknown bacteria that is also resistant to chloramphenicol. Because of these results, we decided to take two white colonies from the two pSB1C3 + TdT ligation plates and inoculate them overnight, giving a total of 4 inoculations. We also decided to run a colony PCR on the pET28b+ +TdT ligation colonies. We took 4 colonies from each plate and resuspended them in 5uL each, giving a total of 8 resuspended colonies. We then took 2.5 uL of each and spotted them on a gel to incubate overnight. We then took the other 2.5 uL and added 10x Buffer, first forward primer, reverse primer, dNTPs, ddH2O, and Taq polymerase to each. These were then run in the PCR machine overnight along with a PCR for the TdT G-Block and a PCR for the first truncated version of TdT. The reaction was set using the program 333, however a few changes were made to the program. The annealing temperature was increased to 57'C and the amount of cycles was increased from 30 to 40.

<H3>7/11/14</H3>

Today, when we came in the eight pET28b+ + TdT ligation samples spotted on the same plate all grew. Because none of us will be in the lab tomorrow, we parafilmed the plate and placed it in the refrigerator so that it could be innoculated on Monday. The four samples of pSB1C3 + TdT that were innoculated overnight were used to four create glycerol stocks and then minipreped.

The PCR reactions that were run overnight were run on two 0.8% gels and it was found that the positive control and colonies 1A, 1B, and 2B all showed bands at about 1.6 kb, indicating that the TdT gene was in fact inserted into the pEt28b+ vector in these colonies. Those three out of the eight colonies will be inoculated overnight on Monday.

from bottom to top: ladder, colony 1A, colony 1B, colony 1C, colony 1D, TdT, truncated TdT, ladder



from left to right: ladder, colony 2A, colony 2B, colony 2C, colony 2D, negative control, blank, ladder



W then double digested some of the minipreped pSB1C3 + TdT ligation using EcoR1 and Pst. The results of this digest were ran on a 0.8% gel.

<H3>7/13/14</H3>

1A, 1B, and 2A colonies on pET28b+ and Tdt ligation colony PCR plate were inoculated overnight. 

<H3>7/14/14</H3>

ligation samples from yesterday were miniprepped. Concentrations were low (7~12ng/uL) so we decided to try midiprep tomorrow. Colonies are prepared for overnight culture. 

<H3>7/15/14</H3>

Glycerol stocks were made from yesterday's overnight culture (pET28b+ Tdt ligation colony PCR on 7/10, 1A, 1B, 2A)

50mL LB media was added to the rest of the overnight culture and was incubated for another 4 hours. 

Midiprep was done until DNA elution step, and samples were stored in 4'C fridge. The samples were prepped for PCR Reaction along with miniprep samples. PCR reactions were done overnight (40 cycles, 57'C annealing for 2 mins).

We will run gels tomorrow.

<H3>7/16/14</H3>

Today we completed the midi prep and ran a PCR reaction on all 6 of the midi and mini preps of pET28b+ TdT vectors. Because the PCR machine was in use for the begining of the day, we could not run our PCR until 3:30 and so we will have to wait until tomorrow to run the gels

<H3>7/17/14</H3>

Today we ran the gels on the four samples taken from the midi prep along with the PCR mini and midi prep. Three gels were run with each gel containing a different sample. The gels are shown below.

Wells from left to right: Ladder, 1A sample #1, 1A sample #2, 1A sample #3, 1A sample #4, 1A PCR midi prep, 1A PCR mini prep, ladder



Wells from left to right: Ladder, 1B sample #1, 1B sample #2, 1B sample #3, 1B sample #4, 1B PCR midi prep, 1B PCR mini prep, ladder



Wells from left to right: Ladder, 2B sample #1, 2B sample #2, 2B sample #3, 2B sample #4, 2B PCR midi prep, 2B PCR mini prep, ladder



<H3>7/18/14</H3>

We phospho-treated double digested pSB1C3 vector today. We also set up another Tdt CleanAmp experiment and ran it on a polyacrylamide gel. Nothing appeared on the gel, but the bands of the ladder were very crisp, indicating that there might be problems in Tdt itself. We will call the company to check. 

We also sent out pET28b+ Tdt ligation vectors for sequencing. The primers were T7 Terminal (Fwd), T7 (Rev).

<H3>7/21/14</H3>

Transformation was done to Rosetta cells and stored overnight at 37'C Kan plate (1A, 1B, 2B)

1:1000 diluted Tdt Gblock was double digested with EcoR1-HF and Pst1, then ligated to pSB1C3 phos.treated vector. The sample was incubated at 16'C overnight.

<H3>7/22/14</H3>

The plates from yesterday were re-streaked because they were overly competent. The original plates were stored in the fridge, and the streaked ones were stored in the incubator overnight at 37'C. 

The transformation of pSB1C3 Tdt ligation to NEB10beta cells was also done and left to incubate overnight at 37'C.

We also received the sequencing results of the pET28b+ TdT ligations and compared the results to what was expected. It was found that 1A was in fact TdT ligated into the pET28b+ vector. Unfortunately, most of the sequencing results were poor so it can not be determined at this time whether or not the other vectors (1B and 2B) do have TdT integrated into pET28b+.

<H3>7/23/14</H3>

We re-streaked the streaked plate from yesterday.

Yesterday's transformation was not showing anything, so we did transformation again with 7/21's ligation sample. This time we used 20uL of sample and NEB10beta cell and tried to maximize the amount of DNA. 

We also decided to try other backbone, and used pSB1K3 to ligate Tdt. pSB1K3 #2 (digested with EcoR1 and Pst1) was phoshotase treated and ligated with double digested Tdt. For Tdt double digestion, 1:10 dilution sample was used (4.4 ng/uL) The ligation sample was stored in ligation machine overnight at 16'C. Transformation should be done tomorrow. 

<H3>7/24/14</H3>

We did transformation of pSB1K3 Tdt ligation from yesterday. Kan plate was used, and the plate was incubated overnight at 37'C.

We also innoculated from pET28b+ Tdt plates restreaked (twice) and left it overnight. 

<H3>7/29/14</H3>

Transformation did not worked, so we redesigned the forward primer for Tdt full length.

Also searched for protein expression protocols based on the paper. Because we do not have negative control for pET28b+ Tdt transformation with rosetta cells, we set up a transformation of empty pET28b+ uncut and left in the incubator overnight. Tomorrow we should be innoculating the glycerol stocks and transformed cell for protein expression experiments.  

<H3>7/30/14</H3>

We first innoculated the transformed control and pET28b+_Tdt Glycerol stock (1A sample#1). The innoculated sample was incubated at 30'C, 255rpm overnight from 10:25~. 

Buffers for protein expression experiment was also prepared (Buffer B, C, and E). 

Another set of Tdt experiment was performed to make sure that dNTPs are properly working. Using A and without denaturation, we tried various substrates. 24mer, PolyA, 33mer, and 52mer was used. We had to re-nanodrop everything since we concluded the concentrations written on the tubes were not based on ssDNA. New concentrations were written on the tubes. The samples were incubated for 30minutes at 37'C, and reaction was stopped at 70'C, 10min. We used polyacrylamide gels and ran for about 50 min, at 200V.

<H3>7/31/14</H3>

The Growth of Expression Cultures Protocol in the Ni-NTA Spin Kit Handbook was followed with a few exceptions. The overnight cultured Rosetta cells (a pET28b+_TdT sample and two control samples of pEt28b+) were diluted with fresh LB medium but instead of following the 1:60 overnight culture to LB ratio stated, a 1:12 was used for a shorter incubation time. The samples were incubated and the OD600s were regularly checked. When the OD600s exceeded 0.6, the samples were diluted with additional LB medium until they an OD600s of 0.5-0.6 were obtained. The cells were then IPTG induced with ten times the stated concentration (10nM final concentration). After 4 hours of incubation, the samples were pelleted, washed once with 1x PBS, pelleted into smaller tubes and stored in the -20'C freezer. 

EN-Ni-NTA-Spin-Kit-Handbook (1).pdf

The TdT G-Block was PCR-amplified with the new forward primer. The PCR products were ran on a 1.0% gel. As the gel shows, the PCR reaction failed to amplify the TdT G-block.

 

<H3>8/1/14</H3>

The PCR reaction to amplify the TdT G-block with the new forward primer was repeated with multiple changes. The annealing temperature was decreased from 56'C to 55'C. The contents of yesterday's PCR reaction was reproduced twice except the amount of Taq was doubled from 0.25uL to 0.5uL. Two other reactions were ran with double the amount of Taq as well as double the amount of TdT G-block (100pg instead of 50pg). One other reaction was ran was double the Taq, TdT G-block as well as 1uL of DMSO.  

After 40 cycles, the PCR products were ran on a 1.0% gel. The gel showed that the amplification of G-block failed for all PCR reactions tested. Only the primers were amplified except for the reaction that included DMSO.  



<H3>8/4/14</H3>

We single-digested 500ng of pET28b+_Tdt midiprep sample on 7/16 (1A, 43.5ng/uL) with Nde1 and used 1uL of that sample to do PCR reactions. 

First PCR reaction was done with 1st truncated forward and regular reverse primer. In addition to digested (linearlized) pET28b+_Tdt, circular DNA was also used. PCR was done with 95'C 30sec, 56'C 30sec, and 72'C 90sec (35 cycles). Then, another round of PCR was done with the same condition but 54'C annealing temperature. 1uL samples from each tube was taken from the first PCR and amplified again with new fwd (delta EcoR1), 2nd, 3rd, 4th truncated forward primer. And left overnight. We will run gels tomorrow (along with the first product of linearlized DNA, to make sure that first round of PCR went okay.) 

<H3>8/5/14</H3>

We ran the gels to see if PCR went well, and concluded that 2nd round of PCR had so much background residues. 1st round of PCR went okay, so gel extraction was performed. 

<H3>8/6/14</H3>

In the morning STEM presentation was done and enzyme activities were done along with presentation. 

We set up another set of PCR with the same protocol from yesterday, but used gel extracted sample from the 1st round of PCR and increased annealing temperature to 55'C. Along with those samples, 1st round of PCR was repeated with single digested pET28b+_Tdt. 

(picture: ladder | circular | linear | 1 | 2 | 3 | 4 | neg.control | ladder)



<H3>8/7/14</H3>

Today we did a protein purification and extraction according to bugbuster kit (Novagen protocol) and Ni-NTA spin kit. We collected the lysate as well as the flow-throughs at step 5,6,7,8. 

We loaded the gel, but because the power supply did not work we could not run the gel and lost the lysate sample.

We also gel purified the bands from the gel that was run yesterday.

<H3>8/8/14</H3>

Today we were able to fix the power supply and run the gel on a polyacrylamide gel without the lysate sample. We then did a Kumasi Stain on the gel to highlight the proteins. Because the step 5 flow through had so many proteins, it was unable to stay in tis lane and expanded into the elution flow through lane, so the results of this gel was inconclusive. The gel is shown below:

 Ladder | Commercial Tdt | Flowthrough 5 with insert | 6 with insert | 7 with insert | 8 with insert | flowthrough 5 w/o insert | 6 w/o insert | 7 w/o insert | 8 w/o insert | Ladder



<H3>8/11/14</H3>

Today we innoculated pET28b+_Tdt in Rosetta cells from the glycerol stock and used pET28b+ in Rosetta cells transformation and put in the shaker overnight (37'C, 250rpm). 

Tdt experiments were done again, one with the protocol we used before and the other one with the protocol that company suggested. 

<H3>8/12/14</H3>

Glycerol stock for pET28b+ in Rosetta cell is made and stored. 

4hr incubation in 20mL LB broth was done: samples were Tdt/IPTG treated, Tdt/neg, without insert/IPTG treated, without insert/neg.

With the gel extraction samples yesterday, double digestion was done with Xba1 and Pst1. Backbones (pSB1C3 and pSB1K3) were also digested, 50ng of vector and 75ng of insert were ligated and stored overnight at 16'C. 

Gel electrophoresis with Tdt experiment was done with TBE/Urea precast gels. 10uL of samples were boiled with 10uL of TBE/Urea buffer for 10minutes, and was run with 200V, 40 mins. Gel was kept loaded without running for about 25 minutes because of the power supply problem. 

<H3>8/13/14</H3>

we continued protein purification experiment, and lysed the cell with bugbuster mastermix (followed Novagen protocol). Lysates were then purified with Ni-NTA spin kit and gel electrophoresis with protein precast gels were done. Buffers for Ni-NTA spin kit were re-made with new urea. 

<H3>8/14/14</H3>

attaching secondary antibody was done in the morning according to thermoscientific protocols. PBST solution was made, stored, and used for wash steps. 

Transformation samples (pSB1C3_TdT and pSB1K3_TdT) were checked and pSB1C3 seems working. Will do colony PCR on monday, parafilmed and stored in the fridge.

<H3>8/15/14</H3>

We ran the gel electrophoresis with TBE/Urea Precast gels with TdT CleanAmp Experiment Samples.

Ladder | dATP | dATP 95'C | dCTP | dCTP 95'C | dGTP | dGTP 95'C | no TdT(neg) | 5min | 60min | Ladder



<H3>8/18/14</H3>

Colony PCR was done based on the transformation last week (pSB1C3_TdT and pSB1K3_TdT). Like last time, 2.5uL of samples were put into PCR reactions, and the other 2.5uL samples were applied to the plates. We will run gels tomorrow.

<H3>8/19/14</H3>

Colony PCR samples were run with 0.8% gel. We did see bands with TdT sizes, but there were too much background noises that we cannot send for sequencing. Also, transformation samples were re-streaked like colony PCR, since the colony PCR done yesterday turned out to be RFP. If the plates just have red colonies tomorrow, we have to re-ligate and transform the samples with phos-treatments. 

Another set of TdT experiments were done with the variation of time. CleanAmp dATP and regular dATPs are incubated for 5, 10, 30, and 60 minutes. Samples are stored and we will run them when the gels are ready. 

<H3>8/20/14</H3>

Checked the plates, and saw several white colonies. Should run PCR tomorrow since the other team was using the machine. 

Redid the double digestion of pSB1K3 and pSB1C3 for biobrick. Used Gel purified and PCRed TdT (9.4ng/uL) and double digested with Xba and Pst1. This time backbones are phosphotase treated, to make sure we don't get religated colonies. 

<H3>8/21/14</H3>

Set up another colony PCR reaction, with the sample protocol last week. Used one colony from pSB1C3 plate, and 9 from pSB1K3 plates (plates were marked and stored in the fridge).

We also ran a gel with TdT experiment samples,

Samples were: Ladder | CleanAmp 5min incubation | CleanAmp 10min incubation | CleanAmp 30min incubation | CleanAmp 60min incubation | Regular dATP 5min incubation | Regular dATP 10min incubation | Regular dATP 30min incubation | Regular dATP 60min incubation



<H3>8/25/14</H3>

Gel purified PCR samples were extracted with QIAquick Gel Extraction Kit Protocol. Gene was eluted in EB buffer and pSB1C3_TdT, pSB1K3_TdT 3, pSB1K3_TdT 6, pSB1K3_TdT 7 was extracted, using yellow new spin column. Concentrations were written on the tube, and the samples were sent out for sequencing. 40ng of DNA in 10uL ddH2O and 5uL of 5uM primer (VF2 and VR for each sample) was mixed and ordered. 

Transformation of phospotase treated ligation samples of pSB1C3_TdT and pSB1K3_TdT in Jam109 cells was done. Because everything was same except for phos. treatment, no controls were made and kept overnight in the incubator. Also, colonies from colony PCR were innoculated (pSB1C3_TdT 1, pSB1K3_TdT 1~9) and incubated overnight. 

<H3>8/26/14</H3>

We re-innoculated minicultures from yesterday, but this time only the ones that we sent out for sequencing was re-innoculated (pSB1C3_TdT, pSB1K3_TdT 3, 6, 7) and kept overnight. (37'C, 300rpm). Glycerol stocks were also made with all minicultures and stored in -80'C.

<H3>8/27/14</H3>

Midipreps for pSB1C3_TdT and pSB1K3_TdT 3, 6, 7 were done according to QIAGEN protocol. Airdried for overnight. 

<H3>8/28/14</H3>

Airdried samples were resuspended in 20uL TE buffer. should crank up the concentration. 

TdT experiments for enzyme denaturing activities were done. 

<H3>8/29/14</H3>

YFP cells should have been innoculated, but wasn't so we will put miniprep off till the first day of the school.

<H3>9/2/14</H3>

pET28YCL4 colonies were innoculated, and incubated overnight (37'C, 300rpm, 9:35~).

<H3>9/3/14</H3>

A 1mL glycerol stock of bacteria containing pET28YCLY4 of equal part cells and a 60% glycerol stock was prepared from sample 1 of the overnight culture and stored in the -80C freezer. 

The two pET28YCLY4 samples were minipreped following the Miniprep Protocol with some exceptions. The overnight culture was centrifuged at 3900rpm for 10 minutes for the first step of preparing the cell lysate. For the second step of eluting the DNA, 50ul of preheated TE buffer was added to the column instead of the stated 75ul.    

After the plasmids were isolated, the concentration was measured using the nanodrop. Samples 1 and 2 of pET28CLY4 was measured to be 62.5 and 47.4 ng/ul, respectively. The plasmids were stored at -20°C. 

<H3>9/4/14</H3>

Rosetta cells were transformed with the pET28CLY4 plasmid following the &quot;General Transformation using Chemically Competent E coli&quot; protocol and plated. One sample was made for each of the pET28CLY4 sampled minipreped as well as a negative control. 50ul of cells, 100ul of SOC media and 2ul of plasmid (or dH2O for the negative control) were used for each sample. 

<H3>9/5/14</H3>

The plated cultures were checked and are shown below.




Each plate had small and large colonies which is unusual for dissimilar sized colonies to grow. Because colonies of the negative control grew, the transformations were redone with the same protocol. 

<H3>9/6/14</H3>

The plates were checked for colonies. (Photos saved in camera.) There were a few colonies growing in the negative plate but since the number of colonies was much less than than those of the pET28CLY4 plates, it was attributed to degradation of the antibiotic over time. A colony from a pET28CLY4 plate was restreaked and incubated overnight. 


</DIV></DIV><DIV class="cu-footer"><DIV class="bottom-center"><A href="https://2014.igem.org/Team:Cooper_Union/Project" class="black"><SPAN>Projects</SPAN></A><A href="https://2014.igem.org/Team:Cooper_Union/Social" class="red"><SPAN>Social</SPAN></A><A href="https://2014.igem.org/Team:Cooper_Union/Notebook" class="green"><SPAN>Notebook</SPAN></A><A href="https://2014.igem.org/Team:Cooper_Union/biofanfic" class="blue"><SPAN>Team</SPAN></A></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Cooper_Union/Notebook/TdT">http://2014.igem.org/Team:Cooper_Union/Notebook/TdT</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cooper_Union/Notebook/TdT" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cooper_Union/Notebook/TdT" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cooper_Union/Notebook/TdT&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cooper_Union/Notebook/TdT&amp;oldid=375145" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>