"
Page
Discussion
View source
History
teams
Log in
 
Team:Cooper Union/Notebook/Biohack
From 2014.igem.org
Cooper Union 2014 iGEM
Home
Project
De Novo Synthesis
Programmable Lifespan Timer
Biohacker Kit
OpenSource Hardware
BioBrick Parts
Social
Safety
Ethics
Outreach
Notebook
Commonly Used Protocols
Labnotes
Team
Members
Attributions
Official Team Profile
Biohacker Kit
JUN JUL
5/28/14
6/2/14
6/3/14
PCR amplified 80uL each of GFP, UV, and HSP.
Purified these with the ones from 6/2/14 as well.
Rinsed and recharged silica wash columns.
On 6/4 and 6/5 while Devora and Shoshana are out, Prof. Medvedik is starting to make the compotent cells, to be completed on 6/6.
6/6/14
Poured plates-2 bottles of kan (20ug/mL) and 1 bottle of amp (100ug/mL).
We used 250 ul of 100 mg/ml stock concentration of Ampicilin and we used 143 ul of 35 mg/mL stock concentration of Kanamycin.
Chlor plate (150ug/mL) that we had tried to transform P3-3H from 2013 BioBrick registry didn't work.
No colonies were seen. A lower chlor concentration (50ug/mL) will be used to see if colonies grow.
Put plates in incubator to be used to transform our ligations.
1 chlor (50ug/mL), 1 chlor (150ug/mL), and 2 amp (100ug/mL).
Put 1 chlor (50ug/mL) plate in incubator to be used to transform the plasmid from the 2013 BioBrick plates P3-3H.
Poured two 0.8% agarose gels.
Inoculated 2.5 mL  (1/2 of the 5 mL we had) of DH5α cells in 125 mL of LB.
We separated it by adding 25 mL of the cell/LB mixture into 5-50 mL conical tubes so that it would fit in the shaker.
We started it in the shaker at 37°C at 10:30 AM when it had an optical density=.078 and let it shake until 12:20 when it had an optical density=.45.
We used the cuvette from the nanodrop to check the optical densities.
From then we used the protocol from Dionne to finish making competent cells to use for transformations.
Transformations
We used 40 uL of DH5-Alpha for all of the transformations.
(old ligations)
5 uL of HSP+pBR322 (1) on amp plate (amp conc.=100 ug/mL)
5uL of UV+pBR322 (2) on amp plate (amp conc.=100 ug/mL)
~1 uL of GFP+pBR322 (3) on amp plate (amp conc.=100 ug/mL)
5 uL of HSP+GFP+pBR322 (4) on amp plate (amp conc.=100 ug/mL)
5 uL of UV+GFP+pBR322 (5) on amp plate (amp conc.=100 ug/mL)
(new ligations)
5 uL of HSP+pBR322  on amp plate (amp conc.=100 ug/mL)
5uL of UV+pBR322  on amp plate (amp conc.=100 ug/mL)
2 tubes of 5 uL of GFP+pACYC184 on chlor plate (one chlor plate will be 150 ug/mL and the other will be 50 ug/mL)
~1-2 uL of P3-3H from the 2013 BioBrick registry on chlor plate (chlor plate=50 ug/mL) (only transforming it on the lower conc. plate because we already transformed it on the higher conc. and it did not grow).
6/9/14
Poured plates-3 bottles of amp (100ug/mL). We used 250 uL of 100 mg/mL stock concentration of Ampicillin.
Poured two 0.8% agarose gels, both about 60 mL in each.
Looked at all the plates that we transformed our plasmids on.
Successful transformations
UV+pBR322 (8colonies)
old UV+pBR322 (2 colonies)
HSP+pBR322 (2 colonies)
old HSP+pBR322 (2 colonies)
old GFP+pBR322 (1 colony)
Unsuccessful transformations
GFP+pACYC184 (both concentrations)
old UV+GFP+pBR322<
old HSP+GFP+pBR322
P3-3H
The fifteen successful colonies were PCR amplified using the input/output promoters.
Each colony was scraped into 200uL of water.
20uL of this along with 2.5uL of the forward and 2.5uL of the reverse promoter was used.
PCR program #130 was used.
These PCR products were run on gels using 1uL of construct, 2uL of loading dye, and 10uL of water, with two ladders on either side.
5uL each of the 1Kb ladder and the Versa ladder were used.
See "Gels of PCRed colonies" for results.
Summary: only old GFP contained a g-Block.
The fifteen successful colonies were also restreaked onto new plates so they could grow!
We digested 2ug of UV, HSP, and GFP with HindIII and XbaI.
Used 30uL total volume and 9.4uL of UV, 8.6uL of HSP, and 9.5uL of GFP.
Let digest for about 2 hours.
Purified these.
The concentrations were as follows:
UV 90.4ng/uL
HSP 99.5ng/uL
GFP 117.3ng/uL.
Ligated these three cut and purified g-Blocks with the X1 pACYC184 cut with HindIII and XbaI overnight.
50 ng of pACYC184 was used and the inserts were used in 1:5 ratios, calculated on the ligation calculator.
20uL total volume was used for the ligations.
6/10/2014
Took ligation out of PCR machine and put in freezer.
Found primers to amplify the successful GFP in pBR322 so that it can be amplified and sent out for sequencing.
Transformed the three ligations from yesterday.
Used 5uL of each ligation (pACYC184+UV/HSP/GFP), DH5α compotent cells, and 700uL of SOC.
Plated 200uL of each on three different plates: chlor(50ug/mL), chlor(150ug/mL), and LB agar.
Poured two bottles of chlor(50ug/mL) and one bottle of chlor(150ug/mL).
6/11/14
Found primers to amplify pACYC184. Ordered these and the ones for pBR322. The reverse promoter was the same for both pACYC184 and pBR322.
Two 0.8% agarose gels were made.
Checked transformations from yesterday: 4 colonies grew on the UV+pACYC184 on the low concentration chlor plates (50ug/mL) and 2 colonies grew on the UV+pACYC184 on the high concentration chlor plates (150ug/mL). These 6 colonies were PCR amplified diluting them with 200uL of water, and then taking 20uL out from here and combining that with 2.5uL of the forward and reverse promoters for the inputs. These were PCR amplified with two controls, DH5alpha with input promoters, and UV with input promoters. We also PCR amplified some HSP and GFP just to get more g-blocks.  The 6 PCR products along with the two controls were run on a gel. It appeared that H2 (The second colony from the high concentration chlor plates) had a faint band. Since only 1uL of each was used for the gel, it was repeated for the 6 PCR products, without controls, using 5uL each. Again, H2 had a band. YAY!  More 10X primers were made from the 100X stock. 10uL of primer was added to the existing 10X epindorf along with 90uL of water.  The UV, HSP and GFP from today's PCR and Monday's PCR were purified. However, we maybe mixed up some UV and HSP so after purification, 2uL of each of these purified samples were run on a gel to see if there are two bands that appear in either one. There weren't!!!! YAY!  We NanoDropped the three PCR'ed g-blocks, after there were purified. The concentrations were UV: 105.2ng/uL, HSP: 64.1ng/uL, and GFP: 77.6ng/uL.  We inoculated our GFP+pBR322 (that the gel showed it actually had the insert inside of it) and left it to grow overnight. We inoculated at 5:30PM.  The UV+pACYC184 that the gel showed actually had the insert in it was restreaked and left to incubate and grow overnight. Incubated around 5PM.  Devora made more LB media.
6/12/14: (Devora typed today up)  Took photos of the pACYC184 plates that didn't grow then threw them out.  Took some of the pACYC184+UV and put it in 5 mL of LB media + 30uL of 25 mg/mL chloramphenicol and let it incubate in the shaker starting at about 11:45 AM. Took it out at 5:00 PM.   We set up the PCR of the pACYC184 and pBR322 by adding 20 mL of pACYC184+UV and pBR322+GFP mixed with 200 uL of water along with 2.5 uL of each of our forward and backward primers. We put it on #130 and will leave it overnight because it's late and we won't be here to take it out.  Then we inoculated our pACYC184 and pBR322 with constructs by doing a 1 in 10 dilution of the already inoculated cultures. Basically, we split up the 2 cultures into two different 50 mL conical tubes each. Each conical tube got 25 mL of LB media with the required antibiotic concentration (150 uL of chloramphenicol for pACYC184 and 25 uL of ampicilin for pBR322) and added 250 uL of the already inoculated plasmid into each. We put them in the shaker-incubater at 37 C overnight atarting at 5:00 PM.   We made a stock concentration of LB ampicilin media using 250 ul of amp into 250 mL of LB media.   Re-suspended the primers that came by adding 10X the amount of TE buffer in uL as there was ng of primers in the container.   Then a working concentration was made by making a 1 in 10 dilution the re-suspended primers in water.      6/13/14: (devora)  Ran a gel with the PCR'ed pBR322+GFP and pACYC184+UV by using 5 uL of the PCR'ed product along with 2 uL of loading dye and 7 uL of water.   Made 4 gels that are 0.8% agrose (3 thick, one not as thick).  Did a midi-prep on the inoculated pBR322+GFP and pACYC184+UV using the Qaigen midi prep kit plus using the midi prep kit protocol. Eluted with 100 uL of EB buffer and then nanodropped and found that there was 150.7 ng/uL in the pACYC184+UV and 113.7 ng/uL for the pBR322+GFP.  We purified the PCR'ed pBR322+GFP and pACYC184+UV that was left overnight and then nanodropped. we found that there was 31.7 ng/uL in the pACYC184+UV and 54.8 ng/uL for the pBR322+GFP.   We poured Chlor/Amp plates (made 3 bottles of 250 mL LB media and  250 uL of Amp and 1.5 mL of Chlor)  We made more chlor stock by adding 10 mL of 100% ethanol to 0.34g of chlor powder to get a 34 mg/mL concentration stock.   We made a 5MM stock of our sequencing primers. We took 20uL of the 1 in 10 dilution of the primers and added 20uL of water to this. This was done for all three primers  We prepared the parts to be sent out for sequencing. Using the purified PCR fragments formed by using the sequence primers, we prepared four PCR tubes to be sent out. For GFP, we used 9.5uL water, 0.5uL sample, and 5uL of 5MM sequencing primer. This was done twice, once using the forward primer and once using the reverse primer. For UV, we used 9.0uL water, 1.0uL sample, and 5uL of 5MM sequencing primer. This was done twice, once using the forward primer and once using the reverse primer. The different amounts of sample was because GFP had a higher concentration than UV.   We co-transformed the UV+pACYC184 and GFP+pBR322 following the transformation protocol. 1uL of each plasmid was used. Also, we heat shocked for 50 seconds instead of 40.
6/16/14:  Four colonies grew on the co-transformed plate. These four colonies were each restreaked on two plates on Saturday. One plate was used as a control and the other one was UV shocked.   Today, we took some of each colony and mixed it with 1mL of PBS. This was vortexed and then centrifuged for 1 minute at top speed (15000). The liquid was removed and 500uL of PBS was added. This was then vortexed. 200uL of each of the 8 samples (four controls and four UV), as well as a blank of plain PBS, were put on the plate reader. The plate reader was used to find the cell concentration as well as the fluorescence. Using these two values, the normalized fluorescence was calculated and plotted in Excel. (See "GFP Fluorescence Reading on the co-transformed plates" for all of this data.) Using the concentrations, each of the samples were diluted to the lowest concentrations, and the concentration and fluorescence values were measured again on the plate reader. Colonies 1, 2, and 3 showed significantly higher fluorescence values that the controls. Colony 4 did not.  We inoculated the UV+pACYC184 and the three colonies that showed higher fluorescence values so that we can make glycerol stocks tomorrow. We used 5mL of LB along with the appropriate antibiotics (Amp and chlor for the 3 combo colonies and chlor for the UV+pACYC184.)  We ligated some more HSP and pACYC184. Used 50ng (3uL of X1:16.5ng/uL) of pACYC184 and 36ng (0.4uL of HSP:99.5ng/uL) of HSP. 2uL ligase buffer, 1uL ligase, and 13.6uL water to get 20uL total volume. Started ligation at 3:30PM.  We restreaked the three successful colonies of UV+pACYC184/GFP+pBR322 on five different plates. One is a control which was wrapped in tinfoil so no UV light would reach it. The other four were UV shocked for 1 second, 5 seconds, 30 seconds, and 60 seconds. Tomorrow we'll test their fluorescence.
6/17/14:  We made glycerol stocks of the UV+pACYC184 and the three colonies with higher fluorescence of UV/GFP+pACYC184/pBR322. 500uL of glycerol and 500uL of inoculated sample.  We transformed out HSP+pACYC184 ligation with DH5alpha cells. We used 5uL of ligated plsamid and one tube of DH5alpha. We did this twice, once with the DH5alpha that we made (I) and once with the ones Dionne made (II). In order to test the two batches of DH5alpha compotent cells, we also used a control of PUC19 plasmid, so that we would be certain something would grow. The "transformation" protocol was followed, except that they were heat shocked for 50 seconds, instead of 40.   We then looked at the five plates that were streaked yesterday: control, UV shocked for 1 second, UV shocked for 5 seconds, UV shocked for 30 seconds, and UV shocked for 60 seconds. Looking on the UV light, it appeared that the 5 second one was significantly greener than the control. We took 500uL of PBS and added some cells for each of the three colonies for each of the five plates. This was vortexed, centrifuged, liquid was taken out, 500uL PBS added, and vortexed. Then, we put 200uL in the plate reader and measured fluorescence and concentration. Normalized fluorescence was calculated. See "GFP Fluorescence Reading on the co-transformed plates #2" for results.  We researched BioBrick/iGEM compatible plasmids so we can send these g-blocks in to the competition as our parts. See "Biobrick Conversion Info for G-Blocks" to see the plasmids that we can use instead of pBR322 and pACYC184, which can't be used for iGEM.  We also set up more test plates by re-streaking the UV+pACYC184 (chlor 150), GFP+pBR322 (amp 100), UV+pACYC184 and GFP+pBR322 (amp 100/chlor 150), and DH5Alpha (LB). For each we made 2 plates; a control plate that was not hit by UV (and covered in tin foil to assure it does not get UV exposure from the surrounding) and a plate that was hit by UV for 5 seconds. They were left in the incubator at 37 C overnight.      6/18/14: (Devora)  We started by looking at the HSP+pACYC184 plates and the pUC19 control plates that we made using the DH5Alpha I and DH5Alpha II cells. There were weird colonies on the DH5Alpha I plate that had the HSP+pACYC184 and 4 colonies on the plate with pUC19 and DH5Alpha II. The other two had no colonies and were left in the incubator.  We also re-ligated the HSP+pACYC184 with Dionne's method of combining all of the usual ingredients in the usual amount but instead of ligating overnight at 16 C we ligated for an hour at room temperature (left it on the bench).  We transformed that ligation following the transformation protocol except that we heat shocked for 50 sec as opposed to 40 sec and put on on a chlor 150 plate and in the incubator.   We then looked at the eight plates that were streaked yesterday: DH5alpha control, DH5alpha UV shocked, UV+pACYC184 control, UV+pACYC184 UV shocked, GFP+pBR322 control, GFP+pBR322 UV shocked, UV/GFP+pACYC184/pBR322 control (3 colonies), UV/GFP+pACYC184/pBR322 UV shocked (3 colonies). We took 500uL of PBS and added some cells from each plate/colony to the PBS. This was vortexed, centrifuged, liquid was taken out, 500uL PBS added, and vortexed. Then, we put 200uL in the plate reader and measured fluorescence and concentration. Normalized fluorescence was calculated. See "GFP Fluorescence Reading on the co-transformed plates #3" for results. BASICALLY, WE ROCK!!!!  As another way of checking that everything was working as we planned in our system, we had sent our primered G-blocks from within the plasmids that we ligated them into to Genewiz to get in sequenced. The sequences came back today!! !! !! From comparing the sequences that Genewiz gave is to the sequences that we had from originally ordering our G-block, it was seen through putting them in the Blast 2 sequences aligner (https://blast.ncbi.nlm.nih.gov/Blast.cgiPAGE_TYPE=BlastSearch&BLAST_SPEC=blast2seq&LINK_LOC=align2seq) that GFP's sequences was 98% accurate just from the forward primer reading and that UV's sequence was 100% accurate from the forward primer reading and 99% accurat from the reverse primer reading. We were not able to open the traces of the sequences because Java was being annoying and wouldn't download on this computer properly, but it is safe to say that what we ligated into our plasmids was our G-blocks, and from the transformations it is clear that those G-block worked to create the 2 plasmid interaction that we intended it to!!! YAYATAYYY HAPPY DAYS!!
6/19/14:   Made more 1X TAE  We checked the colonies from yesterday and did not see any colonies. Therefore we decided to re-transform on a Chlor 50 ug/uL LB plate.  We transformed HSP+pACYC184 ligation with -80 C DH5Alpha cells and also using C-medium and T solution protocol on page 41 of the lab notebook (see attached photo?) except that after that protocol was followed we added 50 uL of T solution and cells to 5 uL of the ligation. Then we added 500 uL of 1X SOC and put it in a water bath at 37 C for an hour. Then we plated on the chlor plates.   We found 3 promoter and 3 reporter genes for the high school kids (Promoters and Reporters doc).   we took the 4 2014 parts from the above list of 6 that we got from the IGEM registry by adding 10 uL of ddH2O to the proper wells on the IGEM trays and then we transfered that to ependorfs that we put into our box. Then we followed the transformation from page 41 again with the same added steps as above with the expection that we used 2 uL of the biobrick instead of the 5 uL of ligation that we usually use.   We then streaked out those transformations and incubated overnight.      6/20/14 (Devora):  We looked at our plates from yesterday and saw 4 colonies on one of our HSP+pAYA184 plates! It was the one made from the frozen DH5Alpha cells. So in order to see if it has our construct or not we colony PCR'ed the 4 colonies on #130. We also used a positive control of HSP G-block to make sure that the PCR would work because someone deleted our protocol in the machine and so we had to re-do it and hope for the best. These five PCR products were run on a 1.2% gel. For the four colonies, we took 5uL PCR, 2uL dye, and 7uL water. For the HSP, we took 1uL PCR, 2uL dye, and 11uL water.   We also struck out the 4 colonies on a plate and put them in the incubator.   We made 2 gels that were about 0.8% agrose.   Did a digest with UV G-block cutting it with PstI and EcoRI (1 uL of G-block which is about 105 ng, 1 uL each enzyme, 3 uL cutsmart, 24 ddH2O)  We phosphetase treated the cut pSB1C3 plasmid that was cut with PstI and EcoRI that we got from the Denovo group (1uL anarctic phosphetase, 2 uL buffer, 10 uL plasmid that was 10 ng/uL, 7 ddH2O).  The UV g-block and pSB1C3 were then purified and eluted using 10uL. Assuming only 8uL came through, we nanodropped 1uL each. The UV was 4.5ng/uL and the pSB1C3 was 7.7ng/uL.  We ligated these two pieces together. Based on their sizes, and using a 1:3 ratio of insert to backbone, we needed to use all of the uL of both. This gave us approximately 50ng of backbone and 30ng of insert. So we used 7uL of each, 2uL ligase buffer, 1uL ligase, and 3uL water.  Then the new order of stuff came in so we digested the UV+pACYC184 and the GFP+pBR322 and HSP G-block with XhoI and KpnI so that we can swap promoter and reporter genes into our system.  (UV+pACYC184: 19.9 uL of plasmid which is 150.7 ng/uL, 3 uL buffer, 1 uL XhoI, 1 uL KpnI, 5.1 uL ddH2O. GFP+pBR322: 26 uL plasmid which is 113.7 ng/uL, 3 uL buffer, 1 uL XhoI, 1 uL KpnI.  HSP G-block: 8.5 uL HSP G-block which is ~60 ng/uL, 3 uL buffer, 1 uL XhoI, 1 uL KpnI, 16.5 ddH2O.)  We ran the two plasmids on a gel so that we could gel purify the plasmid backbones. The results showed that we didn't let it digest long enough, so we cut the pieces out and will digest again next week. The results also appeared to show that the two plasmids had been mixed up. What we thought was UV+pACYC184 seemed to be GFP+pBR322 and vice versa.   We purified the HSP that we cut with KpnI and XhoI and eluted with 20uL. It's concentration was 16.4ng/uL. We also purified our PCR HSP g-blcok control and eluted with 50uL.It's concentration was 36.5ng/uL.  The four BioBricks parts that we transformed yesterday on 50ug/mL chlor plates all had colonies. They were parafilmed and put in the fridge. We need to design primers to get the piece out that we want and put it into our two plasmid system.
6/23/14:  I purified the pieces cut from the gel that we ran on Friday of the semi-digested UV+pACYC184 and GFP+pBR322 with KpnI/XhoI. I followed the "PureLink Quick Gel Extraction Protocol." Then, I nanodropped it and got 10.2ng/uL and 12.6ng/uL which is MUCH lower than it should have been. So I started over with new plasmids and let them digest (UV+pACYC184: 19.9 uL of plasmid which is 150.7 ng/uL, 3 uL buffer, 1 uL XhoI, 1 uL KpnI, 5.1 uL ddH2O. GFP+pBR322: 26 uL plasmid which is 113.7 ng/uL, 3 uL buffer, 1 uL XhoI, 1 uL KpnI.) for 3 hours in the 37 water bath, then denatured the enzyme in 70 degree water bath for 7 minutes.  Professor Medvedik had transformed our UV+pSB1C3 ligations over the weekend on chlor 50ug/mL and 150ug/mL plates. They both grew colonies. I picked six colonies from the 150ug/mL plate and restreaked them onto another plate and let them grow overnight.   I also took those six colonies and put them in 200uL of water, vortexed that, and then took out 20uL of each into PCR tubes. 2.5uL of iGem VF2 and 2.5uL of iGem VR 10mM primers were used. As a control, one of the pSB1A3 plasmids with an approximately 1Kb insert that the De Novo team made was also used with these primers. The PCR was run using PCR program #500. 5uL of each of these were run on a gel with a Versa ladder. See "Gel of UV+pSB1C3 using sequencing primers"  I inoculated colonies 1 and 3 of HSP+pACYC184 in 5mL LB so we can do a midi prep on them tomorrow. 7.4uL of 34mg/mL stock concentration chlor was used to give a 50ug/mL concentration. It was put in the shaker at 3PM and left overnight.     6/26/14:  Glycerol stocks of 500uL 60% glycerol and 500uL of the HSP+pACYC184 colony (one for colony #1 and one for colony #3) were made and put in the -80 freezer.  A gel was run of the digest from Monday of GFP+pBR322 and UV+pACYC184 cut with XhoI and KpnI. The results again appeared to show that the two plasmids had been mixed up. What we thought was UV+pACYC184 seemed to be GFP+pBR322 and vice versa. Since this was now the second time that this happened, we are thinking that maybe the two epindorfs were mislabled. In order to check this, we will take some of each with our amplification primers and see which ones give PCR products. This hasn't been done yet, but will be done next week.  The plasmid backbones were cut from the gel and frozen in epindorfs, to be purified and phosphatase treated next week.   A colony PCR was done on the UV+pSB1C3 coloines 2, 4, and 6 using VF2 and VR primers and program 500. these were then purified and nanodropped. Colony 2: 63.9ng/uL, colony 4: 74.9ng/uL, and colony 6: 64.4ng/uL. A 1 in 30 dilution of each was made, 10uL of that taken, along with 2.5uL water, and 2.5uL 10uM primer to send out for sequencing by GeneWiz.
6/30/14:  Designed primers for various promoters and reporters found in the 2014 iGem plates.   I took 1uL of UV+pACYC184 and 1uL GFP+pBR322 and put each into 200uL of water. This was vortexed and then 20uL of it used for PCR. Each one was set up in two PCR tubes, one which got the pBR322 sequencing primers and one which got the pACYC184 sequencing primers. This was to check if the two epindorfs had been mislabeled. PCR program #130 was run, and then 5uL of product was run on a gel which showed that they had indeed been mislabeled. They were put into new epindorfs with the correct labels.  With that PCR program, I also did two colony PCRs of HSP+pACYC184 colonies 1 and 3. This was to send out for sequencing. We ran 5uL of the product on the gel, because last time we PCRed them, we got a lot of different bands, so we wanted to run it on a gel and gel purify the band that contained our desired piece. This time, again we saw a lot of bands, but I threw out the gel accidentally. So, instead of sending PCR products to genewiz for sequencing, we're going to send the whole plasmid once we midi prep it from the colonies. (See "Gel 6/30/14")  I purified the gel fragments that had the input and output backbones that had been cut with KpnI and XhoI, following "PureLink Quick Gel Extraction Protocol". I then nanodropped the four purified products (each one was split into two tubes, because there was too much volume to fit into one spin column). The results were very low or negative again. We are trying to figure out why this happened, but aren't sure yet. We ran the four products on a gel, and all showed that there was indeed DNA in the products, so we don't know why the nanodrop isn't giving good values. (See "Gel 6/30/14") Professor Medvedik found that the L3 buffer used to dissolve the gel fragments causes issues with the nanodrop, so they have to be washed in a certain way. When we do this, I will type up that protocol.  I autoclaved 3 bottles of LB agar and poured two of them as 50ug/mL chlor plates and the other one as chlor/amp (50ug/mL and 100ug/mL) plates.   I inoculated 50mL each of UV+pACYC184, GFP+pBR322, HSP+pACYC184 colony 1, and HSP+pACYC184 colony 3, so I could do midi preps on all of them tomorrow.
7/1/14:  Working with the input and output backbones that had negative concentrations on the nanodrop yesterday, we added 5uL of 3.0M potassium ethoxide and 100uL of ethanol. This was frozen overnight. In the morning, we spun it in the centrifuge at max speed for 15 minutes. Then the liquid was removed and more ethanol was added. It was again centrifuged at max speed for 10 minutes, and then the liquid was removed again. It was resuspended in 20uL of water. On the nanodrop, the input backbone had a concentration of -0.8ng/uL and the output backbone had a concentration of 3.2ng/uL.  Of the four inoculations set up last night, only the UV+pACYC184 and GFP+pBR322 looked cloudy this morning. Since we don't have any more plungers, the midiprep couldn't be done. The 50mL conical tubes were spun down for 15minutes at 3900rpm. The liquid was poured out, and they were each resuspended in 25mL of PBS. This was vortexed. Then, for each one, 5mL was poured into each of four 25mL conical tubes. This were then used for mini preps. They were spun down for 15 minutes at 3900rpm, the liquid decanted, and then protocol "Plasmid DNA Purification Using the QIAprep Spin Miniprep Kit and a Microcentrifuge" was used. The P1 was added, and then they were all moved into centrifuges. The protocol was followed and the four tubes of each were combined after eluting. Each was eluted with 50uL, so there is a total volume around 190uL. They were then nanodropped. The results were: UV+pACYC184: 87.6ng/uL and GFP+pBR322: 109.2ng/uL.  More of the old UV+pACYC184 and GFP+pBR322 was digested with KpnI and XhoI. (GFP+pBR322: 19.9 uL of plasmid which is 150.7 ng/uL, 3 uL buffer, 1 uL XhoI, 1 uL KpnI, 5.1 uL ddH2O. UV+pACYC184: 26 uL plasmid which is 113.7 ng/uL, 3 uL buffer, 1 uL XhoI, 1 uL KpnI.) (NOTE: The amounts of each used were switched than in previous times, because we had mislabeled the tubes. So this is really what we had been doing. GFP is more concentrated that UV.)  The HSP+pACYC184 colonies 1&3 were inoculated again in the same beakers as yesterday and left to shake overnight at 37 degrees.     7/2/14:  The digested UV+pACYC184 and GFP+pBR322 with KpnI and XhoI (input and output backbones) were run on a gel. They were then cut out and purified using the Qiagen gel extraction kit following the "QIAquick Gel Extraction Kit Protocol" protocol. However, we didn't have PB, so W9 from the Invitrogen kit was used instead. Each sample was split into two because the agarose gel pieces were too large for one. So, each of the four was eluted using 30uL EB buffer, for a total volume of each backbone a little less than 60uL. They were nanodropped, and the results were: Input backbone: 11.5ng/uL and Output backbone: 9.3ng/uL. Although these were MUCH lower than expected, we're just happy there's anything there.  The inoculated HSP+pACYC184 colonies 1&3 were mini prepped as was done yesterday. They were each poured into 50mL conical tubes, spun down at 3900 for 15 minutes, discarded the supernatant, resuspended the pellet in 25mL PBS, split into four 25mL conical tubes with approximately 5mL in each. Then the protocol "Plasmid DNA Purification Using the QIAprep Spin Miniprep Kit and a Microcentrifuge" was followed for all of them. Each set of four epindorfs were combined when the elution was complete. They were each eluted with 50uL for a total volume of each around 200uL. They were nanodropped and the results were: HSP+pACYC184 colony #1: 109.3ng/uL and HSP+pACYC184 colony #3: 95.9ng/uL.  We received our pBad and Vio operon primers that were designed to use for colony PCR and that add KpnI and XhoI sites to the BioBrick parts. Nolana made 1 in 10 dilutions of all four (forward and reverse for each).   The PCR machine was run on setting #130 ("PCR settings 6/20/14") for the pBad primers. We couldn't run it with Vio Operon also, because Vio Operon is around 7Kb so it needs a longer extension time. The PCR was set up by poking one colony on the plate: 2014P3:W19O (pBad), vortexing that in 200uL of water. 20uL of this was put into a PCR tube along with 2.5uL each of the forward and reverse 1 in 10 dilution pBad primers.  A ligation was set up overnight, left in the PCR machine at 16 degrees celcius. We ligated the HSP cut with KpnI and XhoI (conc: 16.4ng/uL) into the input backbone (UV+phage activator, conc: 11.5ng/uL). We used 0.65uL of HSP, 4.3uL of input backbone, 12uL water, 2uL ligase buffer, and 1uL T4 ligase.      7/3/14:  The pBad promoter was purified. I eluted with 30uL. Then it was nanodropped, and the results were 33.8ng/uL. The purified piece was run on a 1.5% gel which only had one band right around 150bp as expected (2uL of PCR product was run.)  The pBad promoter was digested with KpnI and XhoI for one hour. 3uL of buffer, 1uL of each enzyme, and 25uL of pBad (845ng).   The pBad promoter was ligated with the input backbone (pACYC184+phage activator.) 0.2uL pBad was used, 4.3uL input backbone, 2uL buffer, 1uL ligase, and 12.5uL water for a total of 20uL.  HSP that was ligated into the input backbone was transformed with NEB 5alpha compotent cells onto two chlor 50ug/mL plates. Their protocol was used, which is the same as "General Transformation Protocol" excpet it heat shocks for 30 seconds, uses 950uL of SOC, and when it incubates at 37 for one hour, it is done on a shaker platform. The same protocol was used to co-transform HSP+pACYC184 colonies 1 and 3 with GFP+pBR322 onto chlor(50ug/mL) amp plates.
7/7/14:  The inoculated UV+pSB1C3 colony #2 was mini prepped. It was poured into a 50mL conical tube, spun down at 3900 for 15 minutes, discarded the supernatant, resuspended the pellet in 25mL PBS, split into four 25mL conical tubes with approximately 5mL in each. Then the protocol "Plasmid DNA Purification Using the QIAprep Spin Miniprep Kit and a Microcentrifuge" was followed for all of them. However, they were accidentally eluted into the tubes with the white pellet that was garbage and shouldn't be used. So, the liquid was taken out. 1mL of P1, 1mL of P2, and 1.4mL of N3 was combined in a separate tube.700uL of this was poured into the four tubes of liquid taken out from the white stuff. Then the procedure was followed again, centrifuging and washing...Each was eluted with 50uL and nanodropped. The results were very poor. So, we inoculated more of it overnight to mini-prep again tomorrow.  The vio operon PCR products were purified and eluted with 30uL. It was nanodropped and the result was 5.5ng/uL with a really high (2.2) 260/280 probably because the PCR wasn't very clean and efficient because the vio operon is so large.  The two colonies of HSP/GFP+pACYC184/pBR322 that were co-transformed with DH5alpha only had one colony, so they were co-transformed again using the NEB 5alpha compotent cells onto two amp/chlor 50ug/mL plates. Their protocol was used. Also, the pBAD ligated into the input backbone (pACYC184) was transformed the same way and put onto chlor50ug/mL plates.     7/8/14:  The inoculated UV+pSB1C3 colony #2 was mini prepped. It was poured into a 50mL conical tube, spun down at 3900 for 15 minutes, discarded the supernatant, resuspended the pellet in 25mL PBS, split into four 25mL conical tubes with approximately 5mL in each. Then the protocol "Plasmid DNA Purification Using the QIAprep Spin Miniprep Kit and a Microcentrifuge" was followed for all of them. .Each was eluted with 50uL and nanodropped. The result was 5.1ng/uL. So we inoculated UV+pSB1C3 #6 in 50mL of SOC and chlor.   10 colonies from the HSP+input backbone (pSB1C3) were colony PCRed. A poke of one colony was mixed in with 200uL of water, vortexed and 20uL of this was used for PCR with 2.5uL each of the forward and reverse 10X sequencing primers.   Two 1.2% gels were poured.  UV and GFP g-blocks were digested with KpnI and XhoI. 3uL of each (around 600ng) were digested with 1uL of each enzyme, 3uL buffer, and water up to a total volume of 30uL. This was digested for 1 hour. Then heat inactivated for 5 minutes. Professor Medvedik purified it.   7/9/14: (Devora)  We used the 2 1.2% gels to run the 10 HSP with the insert backbone colonies that we chose. The annotated gels are posted in an experiment. It seems that there were at least 6 colonies that had the insert so we chose colonies 1 and 2 and inoculated overnight them to be mini prepped tomorrow.   We also mini prepped the UV+pSB1C3 colony 6. There was one tube that had a crack so we had to get rid of it and then some of the white chunky stuff fell into one of the spin column so there was assumed to be a lower yield then there should have been. We also realized that the PE buffer that we had been using never had the ethanol added to it, which could be why all of our other mini preps were not going well. The plasmids were combined and it was nanodropped, showing 43.8 ng/uL of plasmid and we have 150 uL of it (because we had 3 spin columns and eluted with 50 uL).  We also sent out the HSP+pACYC184 colonies 1 and 3 to be sequenced. Because we are sending them out as plasmids we took 5 uL of plasmid (both were ~100 ng/uL) + 5 uL water + 5 uL 5X primer. There were 4 PCR tubes, with SLO1 containing HSP 1 with forward, SLO2 containing HSP 1 with reverse, SLO3 containing HSP 3 with forward and SLO$ containing HSP 3 with reverse.
7/10/14:  We mini prepped the HSP+input backbone colonies 1 and 2, from our 50mL inoculation. It was poured into a 50mL conical tube, spun down at 3900 for 15 minutes, discarded the supernatant, resuspended the pellet in 25mL PBS, split into four 25mL conical tubes with approximately 5mL in each. Then the protocol "Plasmid DNA Purification Using the QIAprep Spin Miniprep Kit and a Microcentrifuge" was followed for all of them. .Each was eluted with 50uL and nanodropped. The results were, colony 1: 3.2ng/uL or 7.5ng/uL, and for colony 2: 8.2ng/uL or 12.6ng/uL. We nanodropped twice, the first time before vortexing and the second time after vortexin. Dionne noticed that the 260/280 was very high so this may mean we had too much DNA so that could explain the low yields.   We put tips into boxes.  We cleaned some Qiagen mini prep columns.   We spun down all of the primers we got for 10 mins on max speed, then we resuspended them with 1 X TE buffer. We then made 10X dilutions of that by diluting 10 uL of plasmid in 90 uL of water.     7/11/14: (Devora)  Wanted to run a co-transformation with HSP+input backbone 1&2 but prof. will not be in until late saturday so it doesn't make sense. We shall run it on monday at some point. (Maybe also transform the vio operon??)  Ran a PCR with the new primers we got for the input and output backbones. Because we are PCRing something that is in the 4-5 Kb range, we are not going to use the PCR beads and we are going to use the Q5 master mix as the taq polymerase/buffer/dNTP combo for the PCR. I took 12.5 uL of the Q5 master mix and put in 2.5 uL of the forward primer, 2.5 uL of the reverse , 5 uL of a 1000X diluted plasmid (used a GFP+pBR322 and UV+pACYC184 both at ~100-150 ng/uL), and 2.5 uL of water to get it to a 25 uL total volume.   The settings for the PCR machine were edited a little. The settings were changed to:  init denat: 95  300  denaturation: 95  30  annealing: 55  30  extension: 72  200  final extension: 72  350  cycles: 30  Also re-streaked 2 plates of the 1 colony that was found on the HSP pACYC184 +GFP pBR322 co-transformed plates and put it in the incubator.   We got the sequences back from genewiz for colinies 1 and 3 of th eHSP+pACYC184. The HSP 1 forward was 98% accurate and the reverse was 99% accurate and for HSP colony 3 the forward was 97% accurate and the backward was 98% accurate. When looking at the sequences there seem to be a few point mutations in it so we are hoping that it will not affect the functionality of the system.      7/14/14: (Devora)  Shoshana looked at the sequences again and also checked the free repeats that we got. I think that all of the weird point mutations except for one were correct on the other. So we are hoping that everything in the HSP+pACYC184 colony 1 and 3 department are good.   We also ran a gel but then someone switched the leads or didn't check and everything ran off. So we ran another gel . It had the input and output backbones that we PCR'ed on friday on it to check it. The gel only showed a band for the output backbone (see the gel picture in the experiment). Therefore, there might be an annealing temp. problem. Also, we are going to look at the primers that were made to make sure they are good and it's not a primer problem.   We also did a co-transformation of the mini-prepped HSP+input backbone with the GFP+pBR322 into the ultra competent cells. We followed the transformation protocol. We added 1 uL of each plasmid to 50 uL of the ultra competent cells and had them sit on ice for a half hour then heat shocked for 30 seconds in 42 C water bath and added 900 uL of SOC to each tube and let them incubate on a shaker platform for an hour and then plated 200 uL of each on a chlor-amp plate and put it in the incubator overnight.
7/15/14: (Devora)   Today we checked on our co-transformations and we had a bunch of colonies! ! ! Yay.  We then colony PCR'ed colonies 2-5 on the plate of pBad promoter that we had along with a GFP, UV, and HSP. This was put into the new OpenPCR machine, and the settings that were used are below:                 temp time   initial step: 95  30  anneal: 56  30  extention: 72  30  final hold: 4  final step: 72  420  *****THIS PCR RAN GREAT!!! THE SETTING HERE WERE GOOD FOR PBAD-GFP SIZES!******  We ran a gel to check the PCR'ed colonies mentioned above. All of the PCR'ed products came out clean without a band. One gel had the UV, HSP of the T-team's one colony on their plate, and two GFP's. The other had the last two GFP's and the last 5 wells were PBad, with the first of the pBad wells being a negative control (because we put in the wrong primers).
7/17/14:  Ran PCR of HSP+pACYC184 and GFP+pBR322 to amplify the input and output backbones. Trying HSP because UV didn't work, and making more GFP. We used 5uL 1000x dilution of plasmid, 2.5uL water, 2.5uL forward primer, 2.5uL reverse primer, and 12.5uL Q5 McMaster Mix.  init denat: 95  300  denaturation: 95  30  annealing: 54  30  extension: 72  270  final extension: 72  350  cycles: 30  Also, for three of the four HSPs (used HSP+pACYC184 #1), I added Taq polymerase (1,2,3) instead of the Q5 McMaster mix.  Also we inoculated some of the HSP+pACYC184 #3 in 5 mL of LB media with 7.35 uL (34 mg/mL) of chlor to get a conc. of 50 uL/mL overnight so that we can do more testing   Also Devora accidentally diluted the stock of HSP+pACYC184 #3 and now it is 17.5ng/uL concentration. Then, we made a 1 in 1000 dilution of this, so it's around 17pg/uL. We also diluted HSP+pACYC184#1, with a 1 in 1000 dilution, so it's around 100pg/uL now.     7/18/14:  We ran a PCR of HSP+pACYC184 both colonies 1 and 3 so we could send them out for sequencing. We already sent them out as plasmids, but the reads weren't very good, so we're trying again and sending out PCR products.  We transformed the kids BioBricks that didn't transform for them using compotent DH5alpha cells, all onto chlor 50ug/mL plates.  We ran the four input backbone and four output backbone PCRs from yesterday on a gel to see if they actually were amplified. The results can be seen in the gels file for today. The inputs still didn't PCR but the outputs did.  We also took some of the inoculated 5 mL of HSP+pACYC184 and struck it out on a plate so we have some.   Then we took the 5 mL that we inoculated to do testing and did a mini prep on it because we realized we didn't inoculate anything else so we couldn't run anything. We followed the protocol for the quiagen mini prep. We then nanodrpped it and it was 12.3 ng/uL.
7/21/14:  We PCRed more HSP+pACYC184 and UV+pACYC184 with the input backbone primers to try to successfully amplify the backbone. We used 5uL 1000x dilution of plasmid, 2.5uL water, 2.5uL forward primer, 2.5uL reverse primer, and 12.5uL Q5 McMaster Mix. The settings we used were:  init denat: 95  300  denaturation: 95  30  annealing: 50  30  extension: 72  300  final extension: 72  350  cycles: 30  We'll see what happened tomorrow.  We also used DpnI on the five output backbones that we had previously PCRed successfully to chop up the template before we purify it. We used the 25uL of PCR, 19uL of water, 5uL Cut Smart buffer, and 1uL DpnI. This went in 37degree waterbath for 1 hour.   We also ran the HSP experiment. We had inoculated in 5mL LB+antibiotic four samples: DH5alpha, HSP+pACYC184, GFP+pBR322, and HSP/GFP+pACYC184/pBR322. We made 1/10 and 1/20 dilutions of all four. For the 1/10 we used 4.5mL LB, the correct antibiotic, and 500uL of sample. For the 1/20 dilutions, we used 4.75mL LB, the correct antibiotic, and 250uL of sample. We took OD600 readings and found that the 1/20 dilutions were closer to one. So we split these into 2 tubes, 2.5mL in each. These were put in the shaker for forty minutes and OD600s were taken again. This was repeated, checking OD600s, until the concentration had doubled for all 8 samples. (The combo ones took the longest to rise). Then, we put the four heat shock ones into a 45degree water bath for 15minutes. The other four (controls) sat in 37 (withOUT shaking) while they were in the water bath. Then they all went in the shaker (shaking) for 20 minutes, and then OD600 and GFP readings were taken.  A preliminary GFP reading was taken while we were getting the concentration up to double, to check for background fluorescence, and found that the GFP wasn't that high yet. Actual numerical results are in an Excel sheet in "HSP Experiment-7/21/14"     7/22/14: (Devora)  We poured 2 1.5% gels to use tomorrow  We purified the PCR of the output backbone that had Dpn1 added to it and the nanodropped. We have 47.7 ng/uL and 50 uL.      7/28/14:  We ordered new primers for the input backbone. So we ran a PCR with them on UV and HSP#1 + pACYC184. We used 5uL 1000x dilution of plasmid, 2.5uL water, 2.5uL forward primer, 2.5uL reverse primer, and 12.5uL Q5 McMaster Mix. The settings were:  init denat: 95  300  denaturation: 95  30  annealing: 50  30  extension: 72  300  final extension: 72  350  cycles: 30     7/29/14:  The new primers were run on a gel and it looks like HSP#3+pACYC184 had a band indicating that the input backbone had been amplified. See "New input PCR gel 7/29/14" for the photo. We used an annealing temperature of 50 so there were some other bands, but we believe it was amplified correctly as well. So we digested that with DpnI (added 21uL water, 5uL CutSmart, and 1uL DpnI directly to the PCR tube, 37 for 40minutes and 80 for 20 minutes) and then split that into two tubes to digest with KpnI and XhoI (25uL PCR stuff, 3uL CutSmart, 1uL KpnI, 1uL XhoI).  We also digested with KpnI and XhoI the output backbone that we had previously PCR amplified and treated with DpnI. These were then purified, eluted with 50uL and nanodropped. The results were: input backbone: 26.2ng/uL and output backbone: 25.3ng/uL.  Then we phosphatase treated both input and output backbones, splitting each into two tubes using 25uL backbone, 3uL phosphatase buffer, and 1uL phosphatase. The two of each were combined, purified, eluted with 50uL and nanodropped. The results were: input backbone: 12.2ng/uL and output backbone: 12.2ng/uL.
Projects
Social
Notebook
Team
Retrieved from " http://2014.igem.org/Team:Cooper_Union/Notebook/Biohack "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
