"
Page
Discussion
View source
History
teams
Log in
 
Team:Cooper Union/Protocols
From 2014.igem.org
Cooper Union 2014 iGEM
Home
Project
De Novo Synthesis
Programmable Lifespan Timer
Biohacker Kit
OpenSource Hardware
BioBrick Parts
Social
Safety
Ethics
Outreach
Notebook
Commonly Used Protocols
Labnotes
Team
Members
Attributions
Official Team Profile
Common Laboratory Protocols
Bacterial Transformation
Gel Extraction
Ligation
Miniprep
PCR Purification
PCR Reaction
Restriction Enzyme Digest
TdT Heat Inactivationn
TdT Nucleotide Addition
Yeast Gene Knockout
Yeast Sporulation/Tetrad Dissection
Yeast Transformation
Yeast Zygote Selection
Bacterial Transformation
Materials
plasmid DNA (up to 100ng, no more than 5μL)
Competent bacteria aliquot (stored -80°C)
SOC (stored 4°C)
Procedure
Prewarm 2 water baths to 37°C and 42°C
Aliquot 100μL SOC into microcentrifuge tube. Prewarm in 37°C.
Prewarm LB plate + resistance to 37°C.
Add DNA to compontent cells. Incubate on ice 15-20 min.
Heat shock by quickly transferring tubes from ice to 42°C water bath for EXACTLY 45 seconds.
Recuperate for 2 minutes on ice.
Transfer prewarmed SOC to bacteria. Incubate 30 min 37°C.
Transfer all the solution to LB plate. Spread and wait 15 min on benchtop.
Invert and incubate 37°C overnight.
Top of Page
Gel Extraction
Qiagen protocol generally used
Materials
DNA sample ran through agarose gel
sharp edge, like scalpel or gel cutter
Qiagen Buffers QG, PE, EB
Isopropanol
Procedure
Prewarm waterbath to 50°C
Excise the DNA fragment from the agarose gel with a clean, sharp scalpel.
Weigh the gel slice in a colorless tube. Add 3 volumes of Buffer QG to 1 volume of gel (100 mg ~ 100 µl)
Incubate at 50°C for 10 min (or until the gel slice has completely dissolved). To help dissolve gel, mix by vortexing the tube every 2–3 min during the incubation.
After the gel slice has dissolved completely, check that the color of the mixture is yellow (similar to Buffer QG without dissolved agarose).
Add 1 gel volume of isopropanol to the sample and mix.
Place a QIAquick spin column in a provided 2 ml collection tube.
To bind DNA, apply the sample to the QIAquick column, and centrifuge for 1 min.
Discard flow-through and place QIAquick column back in the same collection tube.
(Optional): Add 0.5 ml of Buffer QG to QIAquick column and centrifuge for 1 min.
To wash, add 0.75 ml of Buffer PE to QIAquick column and centrifuge for 1 min.
Discard the flow-through and centrifuge the QIAquick column for an additional 1 min at ≥10,000 x g (~13,000 rpm).
Place QIAquick column into a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 50 µl of Buffer EB (10 mM Tris·Cl, pH 8.5) or H2O to the center of the QIAquick membrane and centrifuge the column for 1 min at maximum speed.
Top of Page
Miniprep
Qiagen protocol generally used
Materials
5mL of overnight bacteria culture
Qiagen Buffers P1, P2, N3, PE, EB
Procedure
Resuspend pelleted bacterial cells in 250 µl Buffer P1 and transfer to a microcentrifuge tube.
Add 250 µl Buffer P2 and gently invert the tube 4–6 times to mix
Add 350 µl Buffer N3 and invert the tube immediately but gently 4–6 times.
Centrifuge for 10 min at 13,000 rpm (~17,900 x g) in a table-top microcentrifuge.
Apply the supernatants from step 4 to the QIAprep spin column by decanting or pipetting.
Centrifuge for 30–60 s. Discard the flow-through.
(Optional): Wash the QIAprep spin column by adding 0.5 ml Buffer PB and centrifuging for 30–60 s. Discard the flow-through.
Wash QIAprep spin column by adding 0.75 ml Buffer PE and centrifuging for 30–60 s.
Discard the flow-through, and centrifuge for an additional 1 min to remove residual wash buffer.
Place the QIAprep column in a clean 1.5 ml microcentrifuge tube. To elute DNA, add 50 µl Buffer EB (10 mM Tris·Cl, pH 8.5) or water to the center of each QIAprep spin column, let stand for 1 min, and centrifuge for 1 min.
Top of Page
PCR Purification
Materials
PCR Reaction to be purified
Qiagen Buffers PB, PE, EB
Procedure
Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix.
If pH indicator has been added to Buffer PB, check that the color of the mixture is yellow.
Place a QIAquick spin column in a provided 2 ml collection tube.
To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through. Place the QIAquick column back into the same tube.
To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through and place the QIAquick column back in the same tube.
Centrifuge the column for an additional 1 min.
Place QIAquick column in a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 50 µl Buffer EB (10 mM Tris·Cl, pH 8.5) or water (pH 7.0–8.5) to the center of the QIAquick membrane and centrifuge the column for 1 min.
Top of Page
PCR Reaction
If PCR beads used, add template, primers and water to a final volume of 25μL
Materials
10x Buffer (Compatible to the enzyme)
dNTP Mix
Enzyme
Forward and Reverse Primers
ddH2O
DNA template
Procedure
Thaw buffers - ensure homogeneous resuspension. Keep all reagents on ice
Set up reactions based on table below.
Reagent
Volume
Example
Reaction Buffer
1/10 total volume
2μL
dNTP's
0.2mM of each
2μL
Primers
0.1-1μM of each
5μL
DNA
10 pg – 1 µg
3.7μL
ddH2O
volume up to (total volume - 1μL)
6.8μL
Enzyme
1 U
0.5μL
Total Volume
20μL
Program thermocycler with appropriate parameters
Step
Temp
Time
# Cycles
Initial Denaturation
95°C
3-5m
1
Denaturation
95°C
30s
30-40
Annealing
Tm-5°C
30s
Extension
72°C
1 min/kb
Final Extension
72°C
10m
1
Hold 4°C
Top of Page
Restriction Enzyme Digest
Materials
10x Buffer (Compatible to the enzyme)
Enzyme
BSA - diluted to 10x
ddH2O
DNA - up to 1μg per reaction.
Procedure Verify all times and temperatures for specific enzyme prior to use. This is general protocol.
Prewarm 2 water baths to 37°C and 70°C
Thaw buffers, diluted BSA. Keep all reagents on ice
Set up reactions based on table below.
Reagent
Volume
Example
Reaction Buffer
1/10 total volume
2μL
Diluted BSA (10x)
1/10 total volume
2μL
DNA
calculate for target ng
3.7μL
ddH2O
volume up to (total volume - 1μL)
11.3μL
Enzyme
generally 1μL
1μL
Total Volume
20μL
Incubate 60min 37°C.
Heat inactive 70°C for 15 min.
Store in -20°C or use immediately.
Top of Page
TdT Heat Inactivation
Materials
DNA source - commercially made oligo's
Terminal Transferase (NEB - M0315S )
Procedure
Pre-heat water baths to 37°C, 75°C, and 95°C
Mix 8 samples on ice:
1.0 μL 10X TdT Buffer
1.0 μL 2.5 mM CoCl2
500 ng 24-mer oligo (0.9 μL of 568.8 ng/μL)
1.0 μL 10mM dNTP
10 Units TdT (0.5 μL of 20,000 U/mL)
dH2O to final volume of 10 μL
Incubate at 95°C for 0, 0.5, 1, 2, 5, 10, 15, and 20 minutes.
Incubate at 37°C for 10 minutes.
Heat inactivate at 75° for 20 minutes.
Store at -20°C.
Top of Page
TdT Nucleotide Addition
Materials
DNA source - commercially made oligo's
Terminal Transferase (NEB - M0315S )
CleanAmp dNTP's (TriLink - N9507)
Procedure
repeat for each nucleotide
Pre-heat water baths to 37°C and 95°C
Mix:
5.0 μL 10X TdT Buffer
5.0 μL 2.5 mM CoCl2 solution
1μg 43 mer oligo (1.3μL of 813 ng/μL)
0.4 μL 50mM CleanAmp dATP
20 Units TdT (1 μL of 20,000 U/μL)
dH2O to final volume of 50 μL
Incubate at 37°C for 30 minutes.
Prepare 2 equal volume aliquots of the mixture (25 μL each).
Incubate 1 aliquot at 95°C for 15 minutes.
Add additional 10 U TdT (0.5 μL of 20,000 U/μL) to both aliqouts.
Cool 95°C water bath to 70°C.
Incubate at 37°C for 60 minutes.
Heat inactivate at 70°C for 10 minutes.
Store at -20°C.
Controls
Negative (No TdT)
Pre-heat water baths to 37°C and 70°C
Mix:
5.0 μL 10X TdT Buffer
5.0 μL 2.5 mM CoCl2 solution
500 ng 24-mer oligo (0.9 μL of 568.8 ng/μL)
1.0 μL 10mM dATP
dH2O to final volume of 50 μL
Incubate at 37°C for 60 minutes
Heat inactivate at 70°C for 10 minutes.
Store at -20°C.
Five Minute Incubation Time
Pre-heat water baths to 37°C and 70°C
Mix:
5.0 μL 10X TdT Buffer
5.0 μL 2.5 mM CoCl2 solution
500 ng 24-mer oligo (0.9 μL of 568.8 ng/μL)
1.0 μL 10mM dATP
10 Units TdT (0.5 μL of 20,000 U/μL)
dH2O to final volume of 50 μL
Incubate at 37°C for 5 minutes
Heat inactivate at 70°C for 10 minutes.
Store at -20°C.
Sixty Minute Incubation Time
Pre-heat water baths to 37°C and 70°C
Mix:
5.0 μL 10X TdT Buffer
5.0 μL 2.5 mM CoCl2 solution
500 ng 24-mer oligo (0.9 μL of 568.8 ng/μL)
1.0 μL 10mM dATP
10 Units TdT (0.5 μL of 20,000 U/μL)
dH2O to final volume of 50 μL
Incubate at 37°C for 60 minutes
Heat inactivate at 70°C for 10 minutes.
Store at -20°C.
Top of Page
Yeast Gene Knockout
To make a deletion of a gene with a drug resistant cassette, the cassette is amplified from a vector by PCR. Oligos for cassette amplification are designed in such a way that they contain 2 parts: the first contains 45 bp of sequence just upstream and downstream from the ORF (including the ATG start codon and the stop codon) of a gene of interest, the second contains common sequence homologous to the sequence of the cassette
Procedure
Design primers:
For upstream oligo
45bp of directly upstream sequence of the yeast open reading frame (including the ATG)+CGTACGCTGCAGGTCGAC
For the downstream oligo
45bp of directly downstream sequence of the yeast open reading frame (including the stop codon)+ATCGATGAATTCGAGCTCG
Amplify a cassette from a plasmid by PCR
standard PCR reaction conditions - add 5% DMSO for GC-rich template
Top of Page
Yeast Sporulation/Tetrad Dissection
sourced from Botstein Lab - Princeton
Sporulation Procedure
Grow a 5 mL overnight culture; dilute 1:50 in the morning and grow for 4 hours at 30°C (to log phase).
Pellet cells, wash in 1 mL of 1% potassium acetate, and resuspend in 3-4 mL of 1% potassium acetate.
Incubate at room temperature on a roller wheel for 3 day (can incubate longer, if desired).
Tetrad Dissection
Make a solution of zymolyase (0.25-2.5 mg/mL) in 1M sorbitol (can leave at 4°C for at least a lower concentration is slower but may help to avoid overdigestion, especially with very freshly sporulated cultures.
Examine culture cell density and remove 200 - 400 µL of sporulated cells (OD600 = 2-4).
Spin down cells, resuspend in 200 - 400 µL of zymolyase solution.
Incubate at room temperature for 5 minutes. Check cells under microscope to determine zymolyase efficiency.
Repeat #4 until asci are digested to the correct level (with fresh zymolyase, 5-10 minutes is usually good).
After tetrads are appropriately digested, GENTLY add and GENTLY (by inversion) mix 600µL – 1000 µL of ddH2O (stops zymolyase by dilution).
Spread cells on a plate in a line across the center of the plate (either with a loop, or by adding 50 µL and tilting the plate to form a line of cells).
Remove the four spores of a tetrad away from the line, and drop one at a time using the micromanipulator guides (preset clicks will separate spores by 5 mm)
Dissect approximately 20 tetrads per plate (10 on each side of the line)
Incubate at 30°C for 2-3 days until spore-derived colonies are about medium size.
Replica plate sporulation plate to desired selection media and/or mating type tester plates.
Top of Page
Yeast Transformation
Small-scale LiAc Yeast Transformation
Procedure
Inoculate 5mL of YPD with several colonies of yeast, 2-3mm in diameter
For host strains previously transformed with another autonomously replicating plasmid, use 10mL instead of the appropriate SD selection medium to maintain the plasmid
Vortex vigorously for 5 min to disperse any clumps
Incubate at 30°C for 16-18hr with shaking at 200 rpm to stationary phase (OD660>1.5)
Using the hemacytometer, count the overnight culture and inoculate 50mL of YPD to a cell density of 5x106 cells/ml.
Incubate the culture at 30°C for 3-5hrs with shaking (200rpm) until the cell density is at 2x107 cells/mL. Use the hemacytometer and also record the OD660 for future reference
It is important to allow the cells to complete at least two divisions. Transformation efficincy remains constant for three to four divisions.
Place cells in sterile 50-mL conical tubes and centrifuge at 3,0000g (~2500rpm) for 5 min at room temperature.
Pour off the medium and thoroughly resuspend the cell pellets in sterile distilled H2O
Centrifuge again at 3,000g for 5 min at room temperature
Pour off the dH2O, resuspend the cells in 1.0mL of 100mM lithium acetate (liAc), and transfer the suspension to a sterile 1.5mL microcentrifuge tube.
Pellet the cells at top speed (12,000 rpm) for 5 seconds and remove the LiAc carefully with a micropipettor.
Resuspend the cells to a final volume of 500 μ (2x109 cells/mL), which is approximately 200μL of LiAc.
If the cell titer of the culture is greater than 2x107 cells/mL, the volume of the LiAc should be increased to maintain the titer of this suspension at 2x109 cells/mL. If the titer of the culture is less than 2x107 cells/mL, decrease the amount of LiAc.
Boil a 1.0mL sample of single-stranded carrier DNA for 5 minutes and quickly chill in ice water.
It is not necessary or desirable to boil carrier DNA every time. Aliquot after boiling, or reboil after three or four freeze thaw cycles. Maintain on ice until ready to use
Vortex the cell suspension and pipet 50μL samples into labeled microfuge tubes. Pellet the cells and remove the LiAc with a micropipette. Each 50μL aliquot is enough cells for one transformation.
The basic "transformation mix" consists of the following components; carefully add them in the exact order listed
240μL PEG (50% w/v, MW 3350)
36μL 1.0M LiAc
5μL of boiled and chilled ssDNA (10mg/mL stock)
70μL of ddH2O and plasmid DNA (0.5μg each)
The order is important. The PEG, which shields the cells from the detrimental effects of the high concentration of LiAc, should go in first
Vortex each tube vigorously until the cell pellet has been completely mixed. This usually takes at least 1 minute.
Incubate for 30 minutes at 30°C
Heat shock for 25 minutes in a water bath set to 42°C
Optimum time may vary for different yeast strains. Test time if you need high efficiency transformations
Microfuge at 8000rpm for 15 seconds and remove the transformation mix with a pipette, leaving the pelleted cells behind.
Pipette 1mL of sterile ddH2O into each tube and resuspend the pellet by pipetting gently up and down. Try not to create air bubbles.
Be as gently (slow) as possible if high efficiency is important
Plate a 200μL aliquot of the resuspended transformed cells onto the appropriate selective plates. Let sit on bench for at least 10 min to dry.
Incubate plates, up-side-down, ato 30°C until colonies appear (generally 2-4 days)
Pick the largest colonies and restreak them on the same selection medium for master plates. Seal plates with Parafilm and store 4°C for 3-4 weeks.
Top of Page
Yeast Zygote Selection
Materials
Mated Yeast Culture
Procedure
Use sterile toothpicks to place equal amounts of the two parent cells (about the size of a 1-2mm sphere) adjacent to one another on the surface of a YPD plate, then mix them well, creating a circle of about 5 mm in diameter.
Incubate the plate for 3 hours at 30°C
Use a sterile toothpick to streak a sample of the mating mixture across the top of a fresh plate, such that individual cells are distinguishable when the plate is observed using a tetrad-dissecting microscope
Zygotes form through the fusion of shmoos, the elongated haploid cells that have arrested in response to mating pheromones.
These zygotes have a dumbbell shape that can be confused with large budding haploid cells in the mating mixture. Zygotes that have formed a medial bud have a distinctive three-lobed shape that allows them to be readily identified in the mating mixture
Use appropriate tetrad dissection techniques to isolate these zygotes to a clean area of the plate using a microneedle designed for tetrad dissection
Incubate the plate for 1-2 days at 30°C
To confirm that the colonies arising from your isolated cells are indeed diploid, replica-plate them to media that will confirm the predicted phenotypes of your diploid strain.
If the two haploid parents have no distinctive genetic markers, use mating test lawns to confirm that your strain is a non-mater
Top of Page
Projects
Social
Notebook
Team
Retrieved from " http://2014.igem.org/Team:Cooper_Union/Protocols "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
