<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cooper_Union_Protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Cooper_Union/Protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cooper_Union/Protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cooper_Union/Protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cooper_Union/Protocols&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cooper_Union/Protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cooper Union/Protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Cooper Union 2014 iGEM</TITLE></P><DIV class="parent-container"><DIV class="menu-parent"><UL class="menu"><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union"> Home </A></LI><LI class="menu-project"> Project
		</LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/TdT_project">De Novo Synthesis </A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Telomere_project">Programmable Lifespan Timer</A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Biohack_project">Biohacker Kit </A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Hardware">OpenSource Hardware </A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Parts">BioBrick Parts </A></LI></UL><LI class="menu-safety"> Social
		</LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Safety">Safety</A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Ethics">Ethics</A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Outreach">Outreach </A></LI><LI class="menu-notebook">   Notebook 
		</LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Protocols">Commonly Used Protocols </A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Notebook"> Labnotes </A></LI><LI class="menu-team"> Team
		</LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/biofanfic">Members </A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Attributions"> Attributions </A></LI><LI class="menu"><A class="menu" href="https://igem.org/Team.cgi?id=1354"> Official Team Profile </A></LI></DIV><DIV class="text"><H1>Common Laboratory Protocols</H1><UL><LI><A href="#btrans">Bacterial Transformation</A></LI><LI><A href="#gel">Gel Extraction</A></LI><LI><A href="http://openwetware.org/wiki/DNA_ligation" target="_blank">Ligation</A></LI><LI><A href="#mini">Miniprep</A></LI><LI><A href="#pcrpure">PCR Purification</A></LI><LI><A href="#pcr">PCR Reaction</A></LI><LI><A href="#digest">Restriction Enzyme Digest</A></LI><LI><A href="#tdtheat">TdT Heat Inactivationn</A></LI><LI><A href="#tdtbase">TdT Nucleotide Addition</A></LI><LI><A href="#gene">Yeast Gene Knockout</A></LI><LI><A href="#spore">Yeast Sporulation/Tetrad Dissection</A></LI><LI><A href="#ytrans">Yeast Transformation</A></LI><LI><A href="#zygote">Yeast Zygote Selection</A></LI></UL><H2>Bacterial Transformation</H2><DL><DT>Materials</DT><DD>plasmid DNA (up to 100ng, no more than 5μL)</DD><DD>Competent bacteria aliquot (stored -80°C)</DD><DD>SOC (stored 4°C)</DD></DL><B>Procedure</B><OL><LI>Prewarm 2 water baths to 37°C and 42°C</LI><LI>Aliquot 100μL SOC into microcentrifuge tube. Prewarm in 37°C.</LI><LI>Prewarm LB plate + resistance to 37°C.</LI><LI>Add DNA to compontent cells. Incubate on ice 15-20 min.</LI><LI>Heat shock by quickly transferring tubes from ice to 42°C water bath for EXACTLY 45 seconds.</LI><LI>Recuperate for 2 minutes on ice.</LI><LI>Transfer prewarmed SOC to bacteria. Incubate 30 min 37°C.</LI><LI>Transfer all the solution to LB plate. Spread and wait 15 min on benchtop.</LI><LI>Invert and incubate 37°C overnight.</LI></OL><A href="#top">Top of Page</A><H2>Gel Extraction</H2><EM>Qiagen protocol generally used</EM><DL><DT>Materials</DT><DD>DNA sample ran through agarose gel</DD><DD>sharp edge, like scalpel or gel cutter</DD><DD>Qiagen Buffers QG, PE, EB</DD><DD>Isopropanol</DD></DL><B>Procedure</B><OL><LI>Prewarm waterbath to 50°C</LI><LI>Excise the DNA fragment from the agarose gel with a clean, sharp scalpel.</LI><LI>Weigh the gel slice in a colorless tube. Add 3 volumes of Buffer QG to 1 volume of gel (100 mg ~ 100 µl)</LI><LI>Incubate at 50°C for 10 min (or until the gel slice has completely dissolved). To help dissolve gel, mix by vortexing the tube every 2–3 min during the incubation. </LI><LI>After the gel slice has dissolved completely, check that the color of the mixture is yellow (similar to Buffer QG without dissolved agarose). </LI><LI>Add 1 gel volume of isopropanol to the sample and mix.</LI><LI>Place a QIAquick spin column in a provided 2 ml collection tube.</LI><LI>To bind DNA, apply the sample to the QIAquick column, and centrifuge for 1 min.</LI><LI>Discard flow-through and place QIAquick column back in the same collection tube.</LI><LI>(Optional): Add 0.5 ml of Buffer QG to QIAquick column and centrifuge for 1 min.</LI><LI>To wash, add 0.75 ml of Buffer PE to QIAquick column and centrifuge for 1 min.</LI><LI>Discard the flow-through and centrifuge the QIAquick column for an additional 1 min at ≥10,000 x g (~13,000 rpm).</LI><LI>Place QIAquick column into a clean 1.5 ml microcentrifuge tube.</LI><LI>To elute DNA, add 50 µl of Buffer EB (10 mM Tris·Cl, pH 8.5) or H2O to the center of the QIAquick membrane and centrifuge the column for 1 min at maximum speed.</LI></OL><A href="#top">Top of Page</A><H2>Miniprep</H2><EM>Qiagen protocol generally used</EM><DL><DT>Materials</DT><DD>5mL of overnight bacteria culture</DD><DD>Qiagen Buffers P1, P2, N3, PE, EB </DD></DL><B>Procedure</B><OL><LI>Resuspend pelleted bacterial cells in 250 µl Buffer P1 and transfer to a microcentrifuge tube.</LI><LI>Add 250 µl Buffer P2 and gently invert the tube 4–6 times to mix</LI><LI>Add 350 µl Buffer N3 and invert the tube immediately but gently 4–6 times.</LI><LI>Centrifuge for 10 min at 13,000 rpm (~17,900 x g) in a table-top microcentrifuge.</LI><LI>Apply the supernatants from step 4 to the QIAprep spin column by decanting or pipetting.</LI><LI>Centrifuge for 30–60 s. Discard the flow-through.</LI><LI>(Optional): Wash the QIAprep spin column by adding 0.5 ml Buffer PB and centrifuging for 30–60 s. Discard the flow-through.</LI><LI>Wash QIAprep spin column by adding 0.75 ml Buffer PE and centrifuging for 30–60 s.</LI><LI>Discard the flow-through, and centrifuge for an additional 1 min to remove residual wash buffer.</LI><LI>Place the QIAprep column in a clean 1.5 ml microcentrifuge tube. To elute DNA, add 50 µl Buffer EB (10 mM Tris·Cl, pH 8.5) or water to the center of each QIAprep spin column, let stand for 1 min, and centrifuge for 1 min.</LI></OL><A href="#top">Top of Page</A><H2>PCR Purification</H2><DL><DT>Materials</DT><DD>PCR Reaction to be purified</DD><DD>Qiagen Buffers PB, PE, EB</DD></DL><B>Procedure</B><OL><LI>Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix. </LI><LI>If pH indicator has been added to Buffer PB, check that the color of the mixture is yellow.</LI><LI>Place a QIAquick spin column in a provided 2 ml collection tube.</LI><LI>To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s.</LI><LI>Discard flow-through. Place the QIAquick column back into the same tube.</LI><LI>To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s.</LI><LI>Discard flow-through and place the QIAquick column back in the same tube.</LI><LI>Centrifuge the column for an additional 1 min. </LI><LI>Place QIAquick column in a clean 1.5 ml microcentrifuge tube.</LI><LI>To elute DNA, add 50 µl Buffer EB (10 mM Tris·Cl, pH 8.5) or water (pH 7.0–8.5) to the center of the QIAquick membrane and centrifuge the column for 1 min.</LI></OL><A href="#top">Top of Page</A><H2>PCR Reaction</H2><EM>If PCR beads used, add template, primers and water to a final volume of 25μL</EM><DL><DT>Materials</DT><DD>10x Buffer (Compatible to the enzyme)</DD><DD>dNTP Mix</DD><DD>Enzyme</DD><DD>Forward and Reverse Primers</DD><DD>ddH2O</DD><DD>DNA template</DD></DL><B>Procedure</B><OL><LI>Thaw buffers - ensure homogeneous resuspension. Keep all reagents on ice</LI><LI>Set up reactions based on table below.       
<TABLE><TBODY><TR><TD>Reagent</TD><TD>Volume</TD><TD>Example</TD></TR><TR><TD>Reaction Buffer</TD><TD>1/10 total volume</TD><TD align="right">2μL</TD></TR><TR><TD>dNTP's</TD><TD>0.2mM of each</TD><TD align="right">2μL</TD></TR><TR><TD>Primers</TD><TD>0.1-1μM of each</TD><TD align="right">5μL</TD></TR><TR><TD>DNA</TD><TD>10 pg – 1 µg</TD><TD align="right">3.7μL</TD></TR><TR><TD>ddH2O</TD><TD>volume up to (total volume - 1μL)</TD><TD align="right">6.8μL</TD></TR><TR><TD>Enzyme</TD><TD>1 U</TD><TD align="right">0.5μL</TD></TR><TR><TD>Total Volume</TD><TD colspan="2" align="right">20μL</TD></TR></TBODY></TABLE></LI><LI>Program thermocycler with appropriate parameters
<TABLE><TBODY><TR><TH>Step</TH><TH>Temp</TH><TH>Time</TH><TH># Cycles</TH></TR><TR><TD>Initial Denaturation</TD><TD>95°C</TD><TD>3-5m</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>95°C</TD><TD>30s</TD><TD rowspan="3">30-40</TD></TR><TR><TD>Annealing</TD><TD>T<SUB>m</SUB>-5°C</TD><TD>30s</TD></TR><TR><TD>Extension</TD><TD>72°C</TD><TD>1 min/kb</TD></TR><TR><TD>Final Extension</TD><TD>72°C</TD><TD>10m</TD><TD>1</TD></TR><TR><TD colspan="4" align="right">Hold 4°C</TD></TR></TBODY></TABLE></LI></OL><A href="#top">Top of Page</A><H2>Restriction Enzyme Digest</H2><DL><DT>Materials</DT><DD>10x Buffer (Compatible to the enzyme)</DD><DD>Enzyme</DD><DD>BSA - diluted to 10x</DD><DD>ddH2O</DD><DD>DNA - up to 1μg per reaction.</DD></DL><B>Procedure</B><EM>Verify all times and temperatures for specific enzyme prior to use. This is general protocol.</EM><OL><LI>Prewarm 2 water baths to 37°C and 70°C</LI><LI>Thaw buffers, diluted BSA. Keep all reagents on ice</LI><LI>Set up reactions based on table below.       
<TABLE><TBODY><TR><TD>Reagent</TD><TD>Volume</TD><TD>Example</TD></TR><TR><TD>Reaction Buffer</TD><TD>1/10 total volume</TD><TD align="right">2μL</TD></TR><TR><TD>Diluted BSA (10x)</TD><TD>1/10 total volume</TD><TD align="right">2μL</TD></TR><TR><TD>DNA</TD><TD>calculate for target ng</TD><TD align="right">3.7μL</TD></TR><TR><TD>ddH2O</TD><TD>volume up to (total volume - 1μL)</TD><TD align="right">11.3μL</TD></TR><TR><TD>Enzyme</TD><TD>generally 1μL</TD><TD align="right">1μL</TD></TR><TR><TD>Total Volume</TD><TD align="right">20μL</TD></TR></TBODY></TABLE></LI><LI>Incubate 60min 37°C.</LI><LI>Heat inactive 70°C for 15 min.</LI><LI>Store in -20°C or use immediately.</LI></OL><A href="#top">Top of Page</A><H2>TdT Heat Inactivation</H2><B>Materials</B><DL><DD>DNA source - commercially made oligo's</DD><DD>Terminal Transferase (NEB - <A href="https://www.neb.com/products/m0315-terminal-transferase" target="_blank">M0315S</A>)</DD></DL><B>Procedure</B><OL><LI>Pre-heat water baths to 37°C, 75°C, and 95°C</LI><LI>Mix 8 samples on ice:
<DL><DD>1.0 μL 10X TdT Buffer</DD><DD>1.0 μL 2.5 mM CoCl<SUB>2</SUB></DD><DD>500 ng 24-mer oligo (0.9 μL of 568.8 ng/μL)</DD><DD>1.0 μL 10mM dNTP</DD><DD>10 Units TdT (0.5 μL of 20,000 U/mL)</DD><DD>dH<SUB>2</SUB>O to final volume of 10 μL</DD></DL></LI><LI>Incubate at 95°C for 0, 0.5, 1, 2, 5, 10, 15, and 20 minutes.</LI><LI>Incubate at 37°C for 10 minutes.</LI><LI>Heat inactivate at 75° for 20 minutes.</LI><LI>Store at -20°C.</LI></OL><A href="#top">Top of Page</A><H2>TdT Nucleotide Addition</H2><DL><DT>Materials</DT><DD>DNA source - commercially made oligo's</DD><DD>Terminal Transferase (NEB - <A href="https://www.neb.com/products/m0315-terminal-transferase" target="_blank">M0315S</A>)</DD><DD>CleanAmp dNTP's (TriLink - <A href="http://www.trilinkbiotech.com/cart/scripts/prodView.asp?idproduct=4665" target="_blank">N9507)</A></DD></DL><B>Procedure</B><EM>repeat for each nucleotide</EM><OL><LI>Pre-heat water baths to 37°C and 95°C</LI><LI>Mix:  
<DL><DD>5.0 μL 10X TdT Buffer</DD><DD>5.0 μL 2.5 mM CoCl<SUB>2</SUB> solution</DD><DD>1μg 43 mer oligo (1.3μL of 813 ng/μL)</DD><DD>0.4 μL 50mM CleanAmp dATP</DD><DD>20 Units TdT (1 μL of 20,000 U/μL)</DD><DD>dH<SUB>2</SUB>O to final volume of 50 μL</DD></DL></LI><LI>Incubate at 37°C for 30 minutes.</LI><LI>Prepare 2 equal volume aliquots of the mixture (25 μL each).</LI><LI>Incubate 1 aliquot at 95°C for 15 minutes.</LI><LI>Add additional 10 U TdT (0.5 μL of 20,000 U/μL) to both aliqouts.</LI><LI>Cool 95°C water bath to 70°C.</LI><LI>Incubate at 37°C for 60 minutes.</LI><LI>Heat inactivate at 70°C for 10 minutes.</LI><LI>Store at -20°C.</LI></OL><U>Controls</U><OL type="I"><LI>Negative (No TdT)
</LI><LI>Pre-heat water baths to 37°C and 70°C</LI><LI>Mix:  
<DL><DD> 5.0 μL 10X TdT Buffer</DD><DD>5.0 μL 2.5 mM CoCl<SUB>2</SUB> solution</DD><DD>500 ng 24-mer oligo (0.9 μL of 568.8 ng/μL)</DD><DD>1.0 μL 10mM dATP</DD><DD>dH<SUB>2</SUB>O to final volume of 50 μL</DD></DL></LI><LI>Incubate at 37°C for 60 minutes</LI><LI>Heat inactivate at 70°C for 10 minutes.</LI><LI>Store at -20°C.</LI></OL><LI>Five Minute Incubation Time
</LI><LI>Pre-heat water baths to 37°C and 70°C</LI><LI>Mix:  
<DL><DD> 5.0 μL 10X TdT Buffer</DD><DD>5.0 μL 2.5 mM CoCl<SUB>2</SUB> solution</DD><DD>500 ng 24-mer oligo (0.9 μL of 568.8 ng/μL)</DD><DD>1.0 μL 10mM dATP</DD><DD>10 Units TdT (0.5 μL of 20,000 U/μL)</DD><DD>dH<SUB>2</SUB>O to final volume of 50 μL</DD></DL></LI><LI>Incubate at 37°C for 5 minutes</LI><LI>Heat inactivate at 70°C for 10 minutes.</LI><LI>Store at -20°C.</LI><LI>Sixty Minute Incubation Time
</LI><LI>Pre-heat water baths to 37°C and 70°C</LI><LI>Mix:  
<DL><DD> 5.0 μL 10X TdT Buffer</DD><DD>5.0 μL 2.5 mM CoCl<SUB>2</SUB> solution</DD><DD>500 ng 24-mer oligo (0.9 μL of 568.8 ng/μL)</DD><DD>1.0 μL 10mM dATP</DD><DD>10 Units TdT (0.5 μL of 20,000 U/μL)</DD><DD>dH<SUB>2</SUB>O to final volume of 50 μL</DD></DL></LI><LI>Incubate at 37°C for 60 minutes</LI><LI>Heat inactivate at 70°C for 10 minutes.</LI><LI>Store at -20°C.</LI><A href="#top">Top of Page</A><H2>Yeast Gene Knockout</H2><EM>To make a deletion of a gene with a drug resistant cassette, the cassette is amplified from a vector by PCR. Oligos for cassette amplification are designed in such a way that they contain 2 parts: the first contains 45 bp of sequence just upstream and downstream from the ORF (including the ATG start codon and the stop codon) of a gene of interest, the second contains common sequence homologous to the sequence of the cassette</EM><B>Procedure</B><OL><LI>Design primers:
<DL><DT>For upstream oligo</DT><DD>45bp of directly upstream sequence of the yeast open reading frame (including the ATG)+CGTACGCTGCAGGTCGAC</DD><DT>For the downstream oligo</DT><DD>45bp of directly downstream sequence of the yeast open reading frame (including the stop codon)+ATCGATGAATTCGAGCTCG</DD></DL></LI><LI>Amplify a cassette from a plasmid by PCR<EM>standard PCR reaction conditions - add 5% DMSO for GC-rich template</EM></LI></OL><A href="#top">Top of Page</A><H2>Yeast Sporulation/Tetrad Dissection</H2><EM>sourced from <A href="http://www.princeton.edu/genomics/botstein/protocols/">Botstein Lab - Princeton</A></EM><DL><DT>Sporulation Procedure</DT><DD><OL><LI>Grow a 5 mL overnight culture; dilute 1:50 in the morning and grow for 4 hours at 30°C (to log phase).</LI><LI>Pellet cells, wash in 1 mL of 1% potassium acetate, and resuspend in 3-4 mL of 1% potassium acetate.</LI><LI>Incubate at room temperature on a roller wheel for 3 day (can incubate longer, if desired).</LI></OL></DD><DT>Tetrad Dissection</DT><OL><LI>Make a solution of zymolyase (0.25-2.5 mg/mL) in 1M sorbitol (can leave at 4°C for at least a lower concentration is slower but may help to avoid overdigestion, especially with very freshly sporulated cultures.</LI><LI>Examine culture cell density and remove 200 - 400 µL of sporulated cells (OD<SUB>600</SUB> = 2-4).</LI><LI>Spin down cells, resuspend in 200 - 400 µL of zymolyase solution.</LI><LI>Incubate at room temperature for 5 minutes. Check cells under microscope to determine zymolyase efficiency.</LI><LI>Repeat #4 until asci are digested to the correct level (with fresh zymolyase, 5-10 minutes is usually good).</LI><LI>After tetrads are appropriately digested, GENTLY add and GENTLY (by inversion) mix 600µL – 1000 µL of ddH<SUB>2</SUB>O (stops zymolyase by dilution). </LI><LI>Spread cells on a plate in a line across the center of the plate (either with a loop, or by adding 50 µL and tilting the plate to form a line of cells).</LI><LI>Remove the four spores of a tetrad away from the line, and drop one at a time using the micromanipulator guides (preset clicks will separate spores by 5 mm)</LI><LI>Dissect approximately 20 tetrads per plate (10 on each side of the line)</LI><LI>Incubate at 30°C for 2-3 days until spore-derived colonies are about medium size. </LI><LI>Replica plate sporulation plate to desired selection media and/or mating type tester plates.</LI></OL><A href="#top">Top of Page</A><H2>Yeast Transformation</H2><EM>Small-scale LiAc Yeast Transformation</EM><B>Procedure</B><OL><LI>Inoculate 5mL of YPD with several colonies of yeast, 2-3mm in diameter<EM>For host strains previously transformed with another autonomously replicating plasmid, use 10mL instead of the appropriate SD selection medium to maintain the plasmid</EM></LI><LI>Vortex vigorously for 5 min to disperse any clumps</LI><LI>Incubate at 30°C for 16-18hr with shaking at 200 rpm to stationary phase (OD<SUB>660</SUB>&gt;1.5)</LI><LI>Using the hemacytometer, count the overnight culture and inoculate 50mL of YPD to a cell density of 5x10<SUP>6</SUP> cells/ml.</LI><LI>Incubate the culture at 30°C for 3-5hrs with shaking (200rpm) until the cell density is at 2x10<SUP>7</SUP> cells/mL. Use the hemacytometer and also record the OD<SUB>660</SUB> for future reference<EM>It is important to allow the cells to complete at least two divisions. Transformation efficincy remains constant for three to four divisions.</EM></LI><LI>Place cells in sterile 50-mL conical tubes and centrifuge at 3,0000g (~2500rpm) for 5 min at room temperature.</LI><LI>Pour off the medium and thoroughly resuspend the cell pellets in sterile distilled H<SUB>2</SUB>O</LI><LI>Centrifuge again at 3,000g for 5 min at room temperature</LI><LI>Pour off the dH<SUB>2</SUB>O, resuspend the cells in 1.0mL of 100mM lithium acetate (liAc), and transfer the suspension to a sterile 1.5mL microcentrifuge tube.</LI><LI>Pellet the cells at top speed (12,000 rpm) for 5 seconds and remove the LiAc carefully with a micropipettor.</LI><LI>Resuspend the cells to a final volume of 500 μ (2x10<SUP>9</SUP> cells/mL), which is approximately 200μL of LiAc.<EM>If the cell titer of the culture is greater than 2x10<SUP>7</SUP> cells/mL, the volume of the LiAc should be increased to maintain the titer of this suspension at 2x10<SUP>9</SUP> cells/mL. If the titer of the culture is less than 2x10<SUP>7</SUP> cells/mL, decrease the amount of LiAc.</EM></LI><LI>Boil a 1.0mL sample of single-stranded carrier DNA for 5 minutes and quickly chill in ice water.<EM>It is not necessary or desirable to boil carrier DNA every time. Aliquot after boiling, or reboil after three or four freeze thaw cycles. Maintain on ice until ready to use</EM></LI><LI>Vortex the cell suspension and pipet 50μL samples into labeled microfuge tubes. Pellet the cells and remove the LiAc with a micropipette. Each 50μL aliquot is enough cells for one transformation.</LI><LI>The basic &quot;transformation mix&quot; consists of the following components; carefully add them in the <B>exact order listed</B><DL><DD>240μL PEG (50% w/v, MW 3350)</DD><DD>36μL 1.0M LiAc</DD><DD>5μL of boiled and chilled ssDNA (10mg/mL stock)</DD><DD>70μL of ddH<SUB>2</SUB>O and plasmid DNA (0.5μg each)</DD><DD><EM>The order is important. The PEG, which shields the cells from the detrimental effects of the high concentration of LiAc, should go in first</EM></DD></DL></LI><LI>Vortex each tube vigorously until the cell pellet has been completely mixed. This usually takes at least 1 minute.</LI><LI>Incubate for 30 minutes at 30°C</LI><LI>Heat shock for 25 minutes in a water bath set to 42°C<EM>Optimum time may vary for different yeast strains. Test time if you need high efficiency transformations</EM></LI><LI>Microfuge at 8000rpm for 15 seconds and remove the transformation mix with a pipette, leaving the pelleted cells behind.</LI><LI>Pipette 1mL of sterile ddH<SUB>2</SUB>O into each tube and resuspend the pellet by pipetting gently up and down. Try not to create air bubbles.<EM>Be as gently (slow) as possible if high efficiency is important</EM></LI><LI>Plate a 200μL aliquot of the resuspended transformed cells onto the appropriate selective plates. Let sit on bench for at least 10 min to dry.</LI><LI>Incubate plates, up-side-down, ato 30°C until colonies appear (generally 2-4 days)</LI><LI>Pick the largest colonies and restreak them on the same selection medium for master plates. Seal plates with Parafilm and store 4°C for 3-4 weeks.</LI></OL><A href="#top">Top of Page</A><H2>Yeast Zygote Selection</H2><B>Materials</B><DL><DD>Mated Yeast Culture</DD></DL><B>Procedure</B><OL><LI>Use sterile toothpicks to place equal amounts of the two parent cells (about the size of a 1-2mm sphere) adjacent to one another on the surface of a YPD plate, then mix them well, creating a circle of about 5 mm in diameter.</LI><LI>Incubate the plate for 3 hours at 30°C</LI><LI>Use a sterile toothpick to streak a sample of the mating mixture across the top of a fresh plate, such that individual cells are distinguishable when the plate is observed using a tetrad-dissecting microscope</LI><LI>Zygotes form through the fusion of shmoos, the elongated haploid cells that have arrested in response to mating pheromones.<EM>These zygotes have a dumbbell shape that can be confused with large budding haploid cells in the mating mixture. Zygotes that have formed a medial bud have a distinctive three-lobed shape that allows them to be readily identified in the mating mixture</EM></LI><LI>Use appropriate tetrad dissection techniques to isolate these zygotes to a clean area of the plate using a microneedle designed for tetrad dissection</LI><LI>Incubate the plate for 1-2 days at 30°C</LI><LI>To confirm that the colonies arising from your isolated cells are indeed diploid, replica-plate them to media that will confirm the predicted phenotypes of your diploid strain.<EM>If the two haploid parents have no distinctive genetic markers, use mating test lawns to confirm that your strain is a non-mater</EM></LI></OL><A href="#top">Top of Page</A><DIV class="cu-footer"><DIV class="bottom-center"><A href="https://2014.igem.org/Team:Cooper_Union/Project" class="black"><SPAN>Projects</SPAN></A><A href="https://2014.igem.org/Team:Cooper_Union/Social" class="red"><SPAN>Social</SPAN></A><A href="https://2014.igem.org/Team:Cooper_Union/Notebook" class="green"><SPAN>Notebook</SPAN></A><A href="https://2014.igem.org/Team:Cooper_Union/biofanfic" class="blue"><SPAN>Team</SPAN></A></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Cooper_Union/Protocols">http://2014.igem.org/Team:Cooper_Union/Protocols</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cooper_Union/Protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cooper_Union/Protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cooper_Union/Protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cooper_Union/Protocols&amp;oldid=374808" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>