<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cooper_Union_Notebook_Biohack_June"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Cooper_Union/Notebook/Biohack_June">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cooper_Union/Notebook/Biohack_June&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cooper_Union/Notebook/Biohack_June&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cooper_Union/Notebook/Biohack_June&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cooper_Union/Notebook/Biohack_June" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cooper Union/Notebook/Biohack June</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Cooper Union 2014 iGEM</TITLE></P><DIV class="parent-container"><DIV class="menu-parent"><UL class="menu"><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union"> Home </A></LI><LI class="menu-project"> Project
		</LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/TdT_project">De Novo Synthesis </A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Telomere_project">Programmable Lifespan Timer</A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Biohack_project">Biohacker Kit </A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Hardware">OpenSource Hardware </A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Parts">BioBrick Parts </A></LI></UL><LI class="menu-safety"> Social
		</LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Safety">Safety</A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Ethics">Ethics</A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Outreach">Outreach </A></LI><LI class="menu-notebook">   Notebook 
		</LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Protocols">Commonly Used Protocols </A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Notebook"> Labnotes </A></LI><LI class="menu-team"> Team
		</LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/biofanfic">Members </A></LI><LI class="menu"><A class="menu" href="https://2014.igem.org/Team:Cooper_Union/Attributions"> Attributions </A></LI><LI class="menu"><A class="menu" href="https://igem.org/Team.cgi?id=1354"> Official Team Profile </A></LI></DIV><DIV class="text"><DIV class="notebook-header"><DIV class="left-side"><H1>Biohacker Kit</H1></DIV><DIV class="notebook-nav-container"><DIV class="month-links"><A href="https://2014.igem.org/Team:Cooper_Union/Notebook/Biohack_April">PRE MAY</A><A href="https://2014.igem.org/Team:Cooper_Union/Notebook/Biohack_May">MAY</A><A href="https://2014.igem.org/Team:Cooper_Union/Notebook/Biohack_June" class="selected">JUN</A><A href="https://2014.igem.org/Team:Cooper_Union/Notebook/Biohack_July">JUL</A></DIV></DIV></DIV><H3>6/2/14</H3><H3>6/3/14</H3><UL><LI>PCR amplified 80μL each of GFP, UV, and HSP. </LI><LI>Purified these with the ones from 6/2/14 as well.</LI><LI>Rinsed and recharged silica wash columns.</LI></UL>
On 6/4 and 6/5 while Devora and Shoshana are out, Prof. Medvedik is starting to make the compotent cells, to be completed on 6/6.

<H3>6/6/14</H3><UL><LI>
Poured plates-2 bottles of kan (20μg/mL) and 1 bottle of amp (100μg/mL).<DL><DD> We used 250μl of 100 mg/ml stock concentration of Ampicilin and we used 143μl of 35 mg/mL stock concentration of Kanamycin.</DD></DL></LI><LI>Chlor plate (150μg/mL) that we had tried to transform P3-3H from 2013 BioBrick registry didn't work. <DL><DD>No colonies were seen. A lower chlor concentration (50μg/mL) will be used to see if colonies grow.</DD></DL></LI><LI>Put plates in incubator to be used to transform our ligations. <DL><DD>1 chlor (50μg/mL), 1 chlor (150ug/mL), and 2 amp (100μg/mL).</DD></DL></LI><LI>Put 1 chlor (50ug/mL) plate in incubator to be used to transform the plasmid from the 2013 BioBrick plates P3-3H.</LI><LI>Poured two 0.8% agarose gels. </LI><LI>Inoculated 2.5 mL  (1/2 of the 5 mL we had) of DH5α cells in 125 mL of LB. <DL><DD>We separated it by adding 25 mL of the cell/LB mixture into 5-50 mL conical tubes so that it would fit in the shaker. </DD><DD> We started it in the shaker at 37 C at 10:30 AM when it had an optical density=.078 and let it shake until 12:20 when it had an optical density=.45.</DD><DD> We used the cuvette from the nanodrop to check the optical densities.</DD><DD> From then we used the protocol from Dionne to finish making competent cells to use for transformations. </DD></DL></LI><LI><DL><DT>Transformations</DT><DD>We used 40 μL of DH5α for all of the transformations.</DD><DD><B>(old ligations)</B></DD></DL></LI><LI>5 μL of HSP+pBR322 (1) on amp plate (amp conc.=100 ug/mL)</LI><LI>5μL of UV+pBR322 (2) on amp plate (amp conc.=100 ug/mL)</LI><LI>~1 μL of GFP+pBR322 (3) on amp plate (amp conc.=100 ug/mL)</LI><LI>5 μL of HSP+GFP+pBR322 (4) on amp plate (amp conc.=100 ug/mL)</LI><LI>5 μL of UV+GFP+pBR322 (5) on amp plate (amp conc.=100 ug/mL)</LI><DL><DD><B>(new ligations)</B><OL><LI> 5 μL of HSP+pBR322  on amp plate (amp conc.=100 ug/mL)</LI><LI> 5μL of UV+pBR322  on amp plate (amp conc.=100 ug/mL)</LI><LI>2 tubes of 5 μL of GFP+pACYC184 on chlor plate (one chlor plate will be 150 ug/mL and the other will be 50 ug/mL)</LI><LI>~1-2 μL of P3-3H from the 2013 BioBrick registry on chlor plate (chlor plate=50 ug/mL) (only transforming it on the lower conc. plate because we already transformed it on the higher conc. and it did not grow).</LI></OL></DD></DL></UL><H3>6/9/14</H3><UL><LI>Poured plates-3 bottles of amp (100ug/mL). We used 250 μL of 100 mg/mL stock concentration of Ampicillin.</LI><LI>Poured two 0.8% agarose gels, both about 60 mL in each.</LI><LI>Looked at all the plates that we transformed our plasmids on.
<DL><DT>Successful transformations</DT><DD>UV+pBR322 (8colonies)</DD><DD> old UV+pBR322 (2 colonies)</DD><DD> HSP+pBR322 (2 colonies)</DD><DD> old HSP+pBR322 (2 colonies)</DD><DD>old GFP+pBR322 (1 colony)</DD></DL><DL><DT>Unsuccessful transformations</DT><DD>GFP+pACYC184 (both concentrations)</DD><DD> old UV+GFP+pBR322&lt;</DD><DD>old HSP+GFP+pBR322 </DD><DD>P3-3H </DD></DL></LI><LI>The fifteen successful colonies were PCR amplified using the input/output promoters.<DL><DD> Each colony was scraped into 200μL of water. </DD><DD>20μL of this along with 2.5μL of the forward and 2.5μL of the reverse promoter was used.</DD><DD> PCR program #130 was used.</DD></DL></LI><LI>These PCR products were run on gels using 1μL of construct, 2μL of loading dye, and 10μL of water, with two ladders on either side.<DL><DD> 5μL each of the 1Kb ladder and the Versa ladder were used.</DD><DD> See &quot;Gels of PCRed colonies&quot; for results.</DD><DD><B> Summary</B>: only old GFP contained a g-Block.</DD></DL></LI><LI>The fifteen successfμL colonies were also restreaked onto new plates so they could grow!</LI><LI>We digested 2ug of UV, HSP, and GFP with HindIII and XbaI.<DL><DD> Used 30μL total volume and 9.4μL of UV, 8.6μL of HSP, and 9.5μL of GFP. </DD><DD>Let digest for about 2 hours.</DD><DD> Purified these.</DD><DD> The concentrations were as follows:</DD></DL></LI><LI> UV 90.4ng/μL</LI><LI>HSP 99.5ng/μL </LI><LI> GFP 117.3ng/μL.</LI></UL><LI>Ligated these three cut and purified g-Blocks with the X1 pACYC184 cut with HindIII and XbaI overnight.<DL><DD> 50 ng of pACYC184 was used and the inserts were used in 1:5 ratios, calculated on the ligation calculator.</DD><DD> 20μL total volume was used for the ligations.</DD></DL></LI><H3>6/10/2014</H3><UL><LI>Took ligation out of PCR machine and put in freezer. </LI><LI>Found primers to amplify the successful GFP in pBR322 so that it can be amplified and sent out for sequencing.</LI><LI>Transformed the three ligations from yesterday.<DL><DD> Used 5μL of each ligation (pACYC184+UV/HSP/GFP), DH5α compotent cells, and 700μL of SOC.</DD><DD> Plated 200μL of each on three different plates: chlor(50ug/mL), chlor(150ug/mL), and LB agar. </DD></DL></LI><LI>Poured two bottles of chlor(50ug/mL) and one bottle of chlor(150ug/mL).</LI></UL><H3>6/11/14</H3><UL><LI>Found primers to amplify pACYC184. Ordered these and the ones for pBR322. The reverse promoter was the same for both pACYC184 and pBR322. </LI><LI>Two 0.8% agarose gels were made.</LI><LI>Checked transformations from yesterday: 4 colonies grew on the UV+pACYC184 on the low concentration chlor plates (50ug/mL) and 2 colonies grew on the UV+pACYC184 on the high concentration chlor plates (150ug/mL). These 6 colonies were PCR amplified diluting them with 200μL of water, and then taking 20μL out from here and combining that with 2.5μL of the forward and reverse promoters for the inputs. These were PCR amplified with two controls, DH5α with input promoters, and UV with input promoters. We also PCR amplified some HSP and GFP just to get more g-blocks.</LI><LI>The 6 PCR products along with the two controls were run on a gel. 

It appeared that H2 (The second colony from the high concentration chlor plates) had a faint band. Since only 1μL of each was used for the gel, it was repeated for the 6 PCR products, without controls, using 5μL each. Again, H2 had a band. YAY!</LI><LI>More 10X primers were made from the 100X stock. 10μL of primer was added to the existing 10X microfuge tube along with 90μL of water.</LI><LI>The UV, HSP and GFP from today's PCR and Monday's PCR were purified. However, we maybe mixed up some UV and HSP so after purification, 2μL of each of these purified samples were run on a gel to see if there are two bands that appear in either one. There weren't!!!! YAY!</LI><LI>We NanoDropped the three PCR'ed g-blocks, after there were purified. The concentrations were UV: 105.2ng/μL, HSP: 64.1ng/μL, and GFP: 77.6ng/μL.</LI><LI>We inoculated our GFP+pBR322 (that the gel showed it actually had the insert inside of it) and left it to grow overnight. We inoculated at 5:30PM.</LI><LI>The UV+pACYC184 that the gel showed actually had the insert in it was restreaked and left to incubate and grow overnight. Incubated around 5PM.</LI><LI>Devora made more LB media.</LI></UL><H3>6/12/14</H3><UL><LI>Took photos of the pACYC184 plates that didn't grow then threw them out.</LI><LI>Took some of the pACYC184+UV and put it in 5 mL of LB media + 30μL of 25 mg/mL chloramphenicol and let it incubate in the shaker starting at about 11:45 AM. Took it out at 5:00 PM. </LI><LI>We set up the PCR of the pACYC184 and pBR322 by adding 20 mL of pACYC184+UV and pBR322+GFP mixed with 200 μL of water along with 2.5 μL of each of our forward and backward primers. We put it on #130 and will leave it overnight because it's late and we won't be here to take it out.</LI><LI>Then we inoculated our pACYC184 and pBR322 with constructs by doing a 1 in 10 dilution of the already inoculated cultures. Basically, we split up the 2 cultures into two different 50 mL conical tubes each. Each conical tube got 25 mL of LB media with the required antibiotic concentration (150 μL of chloramphenicol for pACYC184 and 25 μL of ampicilin for pBR322) and added 250 μL of the already inoculated plasmid into each. We put them in the shaker-incubater at 37 C overnight atarting at 5:00 PM. </LI><LI>We made a stock concentration of LB ampicilin media using 250 μL of amp into 250 mL of LB media. </LI><LI>Re-suspended the primers that came by adding 10X the amount of TE buffer in μL as there was ng of primers in the container. </LI><LI>Then a working concentration was made by making a 1 in 10 dilution the re-suspended primers in water. </LI></UL><H3>6/13/14</H3><UL><LI>Ran a gel with the PCR'ed pBR322+GFP and pACYC184+UV by using 5 μL of the PCR'ed product along with 2 μL of loading dye and 7 μL of water. </LI><LI>Made 4 gels that are 0.8% agrose (3 thick, one not as thick).</LI><LI>Did a midi-prep on the inoculated pBR322+GFP and pACYC184+UV using the Qaigen midi prep kit plus using the midi prep kit protocol. Eluted with 100 μL of EB buffer and then nanodropped and found that there was 150.7 ng/μL in the pACYC184+UV and 113.7 ng/μL for the pBR322+GFP.</LI><LI>We purified the PCR'ed pBR322+GFP and pACYC184+UV that was left overnight and then nanodropped. we found that there was 31.7 ng/μL in the pACYC184+UV and 54.8 ng/μL for the pBR322+GFP. </LI><LI>We poured Chlor/Amp plates (made 3 bottles of 250 mL LB media and  250 μL of Amp and 1.5 mL of Chlor)</LI><LI>We made more chlor stock by adding 10 mL of 100% ethanol to 0.34g of chlor powder to get a 34 mg/mL concentration stock. </LI><LI>We made a 5mM stock of our sequencing primers. We took 20μL of the 1 in 10 dilution of the primers and added 20μL of water to this. This was done for all three primers</LI><LI>We prepared the parts to be sent out for sequencing. Using the purified PCR fragments formed by using the sequence primers, we prepared four PCR tubes to be sent out. For GFP, we used 9.5μL water, 0.5μL sample, and 5μL of 5MM sequencing primer. This was done twice, once using the forward primer and once using the reverse primer. For UV, we used 9.0μL water, 1.0μL sample, and 5μL of 5MM sequencing primer. This was done twice, once using the forward primer and once using the reverse primer. The different amounts of sample was because GFP had a higher concentration than UV. </LI><LI>We co-transformed the UV+pACYC184 and GFP+pBR322 following the transformation protocol. 1μL of each plasmid was used. Also, we heat shocked for 50 seconds instead of 40.</LI></UL><H3>6/16/14</H3><LI>Four colonies grew on the co-transformed plate. These four colonies were each restreaked on two plates on Saturday. One plate was used as a control and the other one was UV shocked.</LI><LI>Today, we took some of each colony and mixed it with 1mL of PBS. This was vortexed and then centrifuged for 1 minute at top speed (15000). The liquid was removed and 500μL of PBS was added. This was then vortexed. 200μL of each of the 8 samples (four controls and four UV), as well as a blank of plain PBS, were put on the plate reader. The plate reader was used to find the cell concentration as well as the fluorescence. Using these two values, the normalized fluorescence was calculated and plotted in Excel. (See &quot;GFP Fluorescence Reading on the co-transformed plates&quot; for all of this data.) Using the concentrations, each of the samples were diluted to the lowest concentrations, and the concentration and fluorescence values were measured again on the plate reader. Colonies 1, 2, and 3 showed significantly higher fluorescence values that the controls. Colony 4 did not.</LI><LI>We inoculated the UV+pACYC184 and the three colonies that showed higher fluorescence values so that we can make glycerol stocks tomorrow. We used 5mL of LB along with the appropriate antibiotics (Amp and chlor for the 3 combo colonies and chlor for the UV+pACYC184.)</LI><LI>We ligated some more HSP and pACYC184. Used 50ng (3μL of X1:16.5ng/μL) of pACYC184 and 36ng (0.4μL of HSP:99.5ng/μL) of HSP. 2μL ligase buffer, 1μL ligase, and 13.6μL water to get 20μL total volume. Started ligation at 3:30PM.</LI><LI>We restreaked the three successful colonies of UV+pACYC184/GFP+pBR322 on five different plates. One is a control which was wrapped in tinfoil so no UV light would reach it. The other four were UV shocked for 1 second, 5 seconds, 30 seconds, and 60 seconds. Tomorrow we'll test their fluorescence.</LI><H3>6/17/14</H3><UL><LI>We made glycerol stocks of the UV+pACYC184 and the three colonies with higher fluorescence of UV/GFP+pACYC184/pBR322. 500μL of glycerol and 500μL of inoculated sample.</LI><LI>We transformed out HSP+pACYC184 ligation with DH5α cells. We used 5μL of ligated plsamid and one tube of DH5α. We did this twice, once with the DH5α that we made (I) and once with the ones Dionne made (II). In order to test the two batches of DH5α compotent cells, we also used a control of PUC19 plasmid, so that we would be certain something would grow. The &quot;transformation&quot; protocol was followed, except that they were heat shocked for 50 seconds, instead of 40. </LI><LI>We then looked at the five plates that were streaked yesterday: control, UV shocked for 1 second, UV shocked for 5 seconds, UV shocked for 30 seconds, and UV shocked for 60 seconds. Looking on the UV light, it appeared that the 5 second one was significantly greener than the control. We took 500μL of PBS and added some cells for each of the three colonies for each of the five plates. This was vortexed, centrifuged, liquid was taken out, 500μL PBS added, and vortexed. Then, we put 200μL in the plate reader and measured fluorescence and concentration. Normalized fluorescence was calculated. See &quot;GFP Fluorescence Reading on the co-transformed plates #2&quot; for results.</LI><LI>We researched BioBrick/iGEM compatible plasmids so we can send these g-blocks in to the competition as our parts. See &quot;Biobrick Conversion Info for G-Blocks&quot; to see the plasmids that we can use instead of pBR322 and pACYC184, which can't be used for iGEM.</LI><LI>We also set up more test plates by re-streaking the UV+pACYC184 (chlor 150), GFP+pBR322 (amp 100), UV+pACYC184 and GFP+pBR322 (amp 100/chlor 150), and DH5α (LB). For each we made 2 plates; a control plate that was not hit by UV (and covered in tin foil to assure it does not get UV exposure from the surrounding) and a plate that was hit by UV for 5 seconds. They were left in the incubator at 37 C overnight. </LI></UL><H3>6/18/14</H3><UL><LI>We started by looking at the HSP+pACYC184 plates and the pUC19 control plates that we made using the DH5α I and DH5α II cells. There were weird colonies on the DH5α I plate that had the HSP+pACYC184 and 4 colonies on the plate with pUC19 and DH5α II. The other two had no colonies and were left in the incubator.</LI><LI>We also re-ligated the HSP+pACYC184 with Dionne's method of combining all of the usual ingredients in the usual amount but instead of ligating overnight at 16°C we ligated for an hour at room temperature (left it on the bench).</LI><LI>We transformed that ligation following the transformation protocol except that we heat shocked for 50 sec as opposed to 40 sec and put on on a chlor 150 plate and in the incubator. </LI><LI>We then looked at the eight plates that were streaked yesterday: DH5α control, DH5α UV shocked, UV+pACYC184 control, UV+pACYC184 UV shocked, GFP+pBR322 control, GFP+pBR322 UV shocked, UV/GFP+pACYC184/pBR322 control (3 colonies), UV/GFP+pACYC184/pBR322 UV shocked (3 colonies). We took 500μL of PBS and added some cells from each plate/colony to the PBS. This was vortexed, centrifuged, liquid was taken out, 500μL PBS added, and vortexed. Then, we put 200μL in the plate reader and measured fluorescence and concentration. Normalized fluorescence was calculated. BASICALLY, WE ROCK!!!!</LI><LI>As another way of checking that everything was working as we planned in our system, we had sent our primered G-blocks from within the plasmids that we ligated them into to Genewiz to get in sequenced. The sequences came back today!! !! !! From comparing the sequences that Genewiz gave is to the sequences that we had from originally ordering our G-block, it was seen through putting them in the Blast 2 sequences aligner (https://blast.ncbi.nlm.nih.gov/Blast.cgiPAGE_TYPE=BlastSearch&amp;BLAST_SPEC=blast2seq&amp;LINK_LOC=align2seq) that GFP's sequences was 98% accurate just from the forward primer reading and that UV's sequence was 100% accurate from the forward primer reading and 99% accurat from the reverse primer reading. We were not able to open the traces of the sequences because Java was being annoying and wouldn't download on this computer properly, but it is safe to say that what we ligated into our plasmids was our G-blocks, and from the transformations it is clear that those G-block worked to create the 2 plasmid interaction that we intended it to!!! YAYATAYYY HAPPY DAYS!!</LI></UL><H3>6/19/14</H3><UL><LI>Made more 1X TAE</LI><LI>We checked the colonies from yesterday and did not see any colonies. Therefore we decided to re-transform on a Chlor 50 ug/μL LB plate.</LI><LI>We transformed HSP+pACYC184 ligation with -80 C DH5α cells and also using C-medium and T solution protocol on page 41 of the lab notebook (see attached photo?) except that after that protocol was followed we added 50 μL of T solution and cells to 5 μL of the ligation. Then we added 500 μL of 1X SOC and put it in a water bath at 37 C for an hour. Then we plated on the chlor plates. </LI><LI>We found 3 promoter and 3 reporter genes for the high school kids (Promoters and Reporters doc). </LI><LI>we took the 4 2014 parts from the above list of 6 that we got from the IGEM registry by adding 10 μL of ddH2O to the proper wells on the IGEM trays and then we transfered that to ependorfs that we put into our box. Then we followed the transformation from page 41 again with the same added steps as above with the expection that we used 2 μL of the biobrick instead of the 5 μL of ligation that we usually use. </LI><LI>We then streaked out those transformations and incubated overnight. </LI></UL><H3>6/20/14</H3><UL><LI>We looked at our plates from yesterday and saw 4 colonies on one of our HSP+pAYA184 plates! It was the one made from the frozen DH5α cells. So in order to see if it has our construct or not we colony PCR'ed the 4 colonies on #130. We also used a positive control of HSP G-block to make sure that the PCR would work because someone deleted our protocol in the machine and so we had to re-do it and hope for the best. These five PCR products were run on a 1.2% gel. For the four colonies, we took 5μL PCR, 2μL dye, and 7μL water. For the HSP, we took 1μL PCR, 2μL dye, and 11μL water.</LI><LI>We also struck out the 4 colonies on a plate and put them in the incubator. </LI><LI>We made 2 gels that were about 0.8% agrose. </LI><LI>Did a digest with UV G-block cutting it with PstI and EcoRI (1 μL of G-block which is about 105 ng, 1 μL each enzyme, 3 μL cutsmart, 24 ddH2O)</LI><LI>We phosphetase treated the cut pSB1C3 plasmid that was cut with PstI and EcoRI that we got from the Denovo group (1μL anarctic phosphetase, 2 μL buffer, 10 μL plasmid that was 10 ng/μL, 7 ddH2O).</LI><LI>The UV g-block and pSB1C3 were then purified and eluted using 10μL. Assuming only 8μL came through, we nanodropped 1μL each. The UV was 4.5ng/μL and the pSB1C3 was 7.7ng/μL.</LI><LI>We ligated these two pieces together. Based on their sizes, and using a 1:3 ratio of insert to backbone, we needed to use all of the μL of both. This gave us approximately 50ng of backbone and 30ng of insert. So we used 7μL of each, 2μL ligase buffer, 1μL ligase, and 3μL water.</LI><LI>Then the new order of stuff came in so we digested the UV+pACYC184 and the GFP+pBR322 and HSP G-block with XhoI and KpnI so that we can swap promoter and reporter genes into our system.</LI><LI>UV+pACYC184: 19.9 μL of plasmid which is 150.7 ng/μL, 3 μL buffer, 1 μL XhoI, 1 μL KpnI, 5.1 μL ddH2O. GFP+pBR322: 26 μL plasmid which is 113.7 ng/μL, 3 μL buffer, 1 μL XhoI, 1 μL KpnI.  HSP G-block: 8.5 μL HSP G-block which is ~60 ng/μL, 3 μL buffer, 1 μL XhoI, 1 μL KpnI, 16.5 ddH2O.)</LI><LI>We ran the two plasmids on a gel so that we could gel purify the plasmid backbones. The results showed that we didn't let it digest long enough, so we cut the pieces out and will digest again next week. The results also appeared to show that the two plasmids had been mixed up. What we thought was UV+pACYC184 seemed to be GFP+pBR322 and vice versa. </LI><LI>We purified the HSP that we cut with KpnI and XhoI and eluted with 20μL. It's concentration was 16.4ng/μL. We also purified our PCR HSP g-blcok control and eluted with 50μL.It's concentration was 36.5ng/μL.</LI><LI>The four BioBricks parts that we transformed yesterday on 50ug/mL chlor plates all had colonies. They were parafilmed and put in the fridge. We need to design primers to get the piece out that we want and put it into our two plasmid system.</LI></UL><H3>6/23/14</H3><UL><LI>I purified the pieces cut from the gel that we ran on Friday of the semi-digested UV+pACYC184 and GFP+pBR322 with KpnI/XhoI. I followed the &quot;PureLink Quick Gel Extraction Protocol.&quot; Then, I nanodropped it and got 10.2ng/μL and 12.6ng/μL which is MUCH lower than it should have been. So I started over with new plasmids and let them digest (UV+pACYC184: 19.9 μL of plasmid which is 150.7 ng/μL, 3 μL buffer, 1 μL XhoI, 1 μL KpnI, 5.1 μL ddH2O. GFP+pBR322: 26 μL plasmid which is 113.7 ng/μL, 3 μL buffer, 1 μL XhoI, 1 μL KpnI.) for 3 hours in the 37 water bath, then denatured the enzyme in 70 degree water bath for 7 minutes.</LI><LI>Professor Medvedik had transformed our UV+pSB1C3 ligations over the weekend on chlor 50ug/mL and 150ug/mL plates. They both grew colonies. I picked six colonies from the 150ug/mL plate and restreaked them onto another plate and let them grow overnight. </LI><LI>I also took those six colonies and put them in 200μL of water, vortexed that, and then took out 20μL of each into PCR tubes. 2.5μL of iGem VF2 and 2.5μL of iGem VR 10mM primers were used. As a control, one of the pSB1A3 plasmids with an approximately 1Kb insert that the De Novo team made was also used with these primers. The PCR was run using PCR program #500. 5μL of each of these were run on a gel with a Versa ladder. See &quot;Gel of UV+pSB1C3 using sequencing primers&quot;</LI><LI>I inoculated colonies 1 and 3 of HSP+pACYC184 in 5mL LB so we can do a midi prep on them tomorrow. 7.4μL of 34mg/mL stock concentration chlor was used to give a 50ug/mL concentration. It was put in the shaker at 3PM and left overnight.</LI></UL><H3>6/26/14</H3><UL><LI>Glycerol stocks of 500μL 60% glycerol and 500μL of the HSP+pACYC184 colony (one for colony #1 and one for colony #3) were made and put in the -80 freezer.</LI><LI>A gel was run of the digest from Monday of GFP+pBR322 and UV+pACYC184 cut with XhoI and KpnI. The results again appeared to show that the two plasmids had been mixed up. What we thought was UV+pACYC184 seemed to be GFP+pBR322 and vice versa. Since this was now the second time that this happened, we are thinking that maybe the two epindorfs were mislabled. In order to check this, we will take some of each with our amplification primers and see which ones give PCR products. This hasn't been done yet, but will be done next week.</LI><LI>The plasmid backbones were cut from the gel and frozen in epindorfs, to be purified and phosphatase treated next week. </LI><LI>A colony PCR was done on the UV+pSB1C3 coloines 2, 4, and 6 using VF2 and VR primers and program 500. these were then purified and nanodropped. Colony 2: 63.9ng/μL, colony 4: 74.9ng/μL, and colony 6: 64.4ng/μL. A 1 in 30 dilution of each was made, 10μL of that taken, along with 2.5μL water, and 2.5μL 10uM primer to send out for sequencing by GeneWiz.</LI></UL><H3>6/30/14</H3><UL><LI>Designed primers for various promoters and reporters found in the 2014 iGem plates. </LI><LI>I took 1μL of UV+pACYC184 and 1μL GFP+pBR322 and put each into 200μL of water. This was vortexed and then 20μL of it used for PCR. Each one was set up in two PCR tubes, one which got the pBR322 sequencing primers and one which got the pACYC184 sequencing primers. This was to check if the two epindorfs had been mislabeled. PCR program #130 was run, and then 5μL of product was run on a gel which showed that they had indeed been mislabeled. They were put into new epindorfs with the correct labels.</LI><LI>With that PCR program, I also did two colony PCRs of HSP+pACYC184 colonies 1 and 3. This was to send out for sequencing. We ran 5μL of the product on the gel, because last time we PCRed them, we got a lot of different bands, so we wanted to run it on a gel and gel purify the band that contained our desired piece. This time, again we saw a lot of bands, but I threw out the gel accidentally. So, instead of sending PCR products to genewiz for sequencing, we're going to send the whole plasmid once we midi prep it from the colonies. </LI><LI>I purified the gel fragments that had the input and output backbones that had been cut with KpnI and XhoI, following &quot;PureLink Quick Gel Extraction Protocol&quot;. I then nanodropped the four purified products (each one was split into two tubes, because there was too much volume to fit into one spin column). The results were very low or negative again. We are trying to figure out why this happened, but aren't sure yet. We ran the four products on a gel, and all showed that there was indeed DNA in the products, so we don't know why the nanodrop isn't giving good values. (See &quot;Gel 6/30/14&quot;) Professor Medvedik found that the L3 buffer used to dissolve the gel fragments causes issues with the nanodrop, so they have to be washed in a certain way. When we do this, I will type up that protocol.</LI><LI>I autoclaved 3 bottles of LB agar and poured two of them as 50ug/mL chlor plates and the other one as chlor/amp (50ug/mL and 100ug/mL) plates. </LI><LI>I inoculated 50mL each of UV+pACYC184, GFP+pBR322, HSP+pACYC184 colony 1, and HSP+pACYC184 colony 3, so I could do midi preps on all of them tomorrow.</LI></UL></DIV></DIV><DIV class="cu-footer"><DIV class="bottom-center"><A href="https://2014.igem.org/Team:Cooper_Union/Project" class="black"><SPAN>Projects</SPAN></A><A href="https://2014.igem.org/Team:Cooper_Union/Social" class="red"><SPAN>Social</SPAN></A><A href="https://2014.igem.org/Team:Cooper_Union/Notebook" class="green"><SPAN>Notebook</SPAN></A><A href="https://2014.igem.org/Team:Cooper_Union/biofanfic" class="blue"><SPAN>Team</SPAN></A></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Cooper_Union/Notebook/Biohack_June">http://2014.igem.org/Team:Cooper_Union/Notebook/Biohack_June</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cooper_Union/Notebook/Biohack_June" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cooper_Union/Notebook/Biohack_June" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cooper_Union/Notebook/Biohack_June&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cooper_Union/Notebook/Biohack_June&amp;oldid=377215" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>