"
Page
Discussion
View source
History
teams
Log in
 
Team:Cooper Union/Notebook/Telomere June
From 2014.igem.org
Cooper Union 2014 iGEM
Home
Project
De Novo Synthesis
Programmable Lifespan Timer
Biohacker Kit
OpenSource Hardware
BioBrick Parts
Social
Safety
Ethics
Outreach
Notebook
Commonly Used Protocols
Labnotes
Team
Members
Attributions
Official Team Profile
Programmable Lifespan Timer
JUN JUL AUG SEP
6/3/14
(Nolana)
Researched genes that affect telomere length (lengthening or shortening).
Possible genes to be used for knockout: HSP104 (long), MAK31 (long), VPS75 (short), SMI1 (short).
Relevant lit: PNAS_Askree
For tomorrow: see if yeast plasmids in lab can be used for programmable kill switch project.  Design primers for gene deletion protocol.
6/4/14
(Nolana)
Designed primers for yeast genes (TLC1, RAD52, HSP104, VPS75, MAK31, SMI1).
Checked plasmids to be ordered for knockout cassettes (pFA6a-kanMX6, pFA6a-TRP1, pAG25).
Transformed DH5α with 6 different BioBrick plasmids (pSB1A3, pSB1C3, pSB1K3, pSB1T3, pSB2K3), two BioBrick RBS (B0030, B0034), and a BioBrick double terminator (B0015); left all 8 to grow overnight.
For tomorrow:  check for cell growth on plates and further grow any colonies that form.
6/5/14
(Nolana)
Plates were checked; colonies were growing on all of them.
Picked 4 genes to order primer oligos for (RAD52, EST4, MAK31, VPS75).
Streaked plates with old yeast strains.
For Monday (I won't be here tomorrow): check yeast plates for colonies
6/9/14
(Nolana)
Checked yeast plates -- all plates grew colonies.
Since EST4 knocked out in conjunction with RAD52 kills the yeast, we decided to use EST1 instead of EST4 in the gene deletion for the kill switch project.
Poured plates -- 3 bottles of amp (100μg/mL). We used 250 μL of 100 mg/mL stock concentration of Ampicillin.
Poured two 0.8% agarose gels, both about 60 mL in each.
Helped set up Restriction Digest Reaction with De novo group (used General Restriction Enzyme Digest protocol):  3 μL buffer, 3 μL BSA, DNA, H2O, and XhoI/XbaI enzyme.
For tomorrow: design check primers for knockout cassettes (programmable kill switch project)
6/10/14
(Nolana)
Designed check primers to be used to verify gene deletion -- used oligo JM42 (3' tag sequence of sense orientation deletion primer with homology to MX4 cassette) for reverse primer, and 500 bases upstream of each gene (RAD52, EST1, MAK31, VPS75) for forward primer.
For tomorrow: Design amplifying primers for kill switch project.
6/11/14
(Nolana)
Worked with Devora to find the primer sequences for amplifying the VPS75, RAD52, MAK31, EST1; added BioBrick prefixes and suffixes.
Checked all of the different primers and sequences that we ordered to check for forbidden restriction sites: there is one restriction site in KanMX, one in LEU2, and 3 forbidden restriction sites in TRP1; these will need to undergo PCR mutagenesis in order to  remove the restriction sites.
made YPD plates: (meant to make three 250mL bottles of media, but ended up making one 500mL bottle)
bacto-yeast extract (1%)
5 g
bacto-peptone (2%)
10 g
There were no results.  Need to check if primer sequences are correct.
Nanodropped plasmids from miniprep:
pAG25 (clonat):  563.7 ng/μL
pFA6a-TRP1:  455.7 ng/μL
pFA6a-LEU2MX6:  799.7 ng/μL
Checked why PCR gel didn't work: we looked at the primer sequences but there was no problem.
We made a 1:1000 dilution of each plasmid (pAG25, pFA6a-TRP1, pFA6a-LEU2MX6), then ran another PCR, this time with the purified plasmids from the miniprep.
5μL plasmid
15μL H2O
2.5μL each of the EST1 deletion primers (forward and reverse)
PCR conditions were the same as last time, program 333.
For tomorrow:  check PCR and run a gel; check on yeast selective plates.
6/26/14
(Jae Ho)
PCR Check Done, Gel run.
2-3 kbp results for EST1+LEU2, no results on other. Possibly contaminated template of Clonet.
Digest and run gel of clonet to check if it is contaminated etc
Yeast cultures are struck and photos taken.
For later: Digest/Gel Clonet, make more YPD plates (20 more)
(Nolana)
made YPD plates with Lily
looked up enzymes and buffer for clonat double digest: we will use the enzyme sites SacI and HindIII, and the NEB Cutsmart buffer
For Monday: run digest
6/30/14
(Nolana)
Looked at yeast plates -- not very conclusive
Ran clonat double digest:
Cutsmart buffer
2 uL
10x BSA
2 uL
pAG25
1 uL
ddH2O
13 uL
SacI
1 uL
BamHI-HF
1 uL
Total Volume: 20 uL
Designed BioBrick primer for EST2 (for yeast strains Professor Medvedik brought back from Boston)
Streaked 5 YPD plates with yeast strains and put them in 30 degree incubator: W303α, 1300, 1294, 1296, 1297
Helped Wilfrido make two 0.8% gels, and two 1% gels
Ran a 0.8% gel.
Wells:  1. Versa Ladder ; 2. digested clonat ; 3. undigested clonat ; 4-7. Shoshana's samples
Gel results: the SacI cut at 43bp, and the BamHI-HF at 1280, which gave two pieces of 1237bp and 2467bp, confirmed by the gel.  The undigested clonat result also looked good.  This confirmed that the plasmid we have is what we want, the pAG25 (clonat).  Now, the question is why didn't it work with the deletion primers?
Made yeast cultures of W303a, 1880, 1882 for yeast transformation tomorrow
For tomorrow: run PCR of MAK31 and VPS75; yeast transformation
Projects
Social
Notebook
Team
Retrieved from " http://2014.igem.org/Team:Cooper_Union/Notebook/Telomere_June "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
