"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Copenhagen/Results
From 2011.igem.org
Home Team Project Safety Ethics Sponsors
Contents
1 Results
1.1 A2 the igem way
1.2 B1 the DTU way
1.3 A1 the iGEM way
1.4 Killing fungus
1.5 The human Cyps
Results
A2 the igem way
We mutated the CYP79A2 to remove a restrictionsite. Afterwards we inserted the CYP in the shipping plasmid. We confirmed both the mutation and the insertion by sequencing. We tried to insert the CYP79A2 in the ekspressionvector BBa_J04500 and although it looked like the gene was inserted when we got it sequenced it wasn't a perfect fit. We picked the best fitting and expressed them in BL21 cells. A SDS gel didn't show any increase in protein amount for the band-size corresponding to the CYP. A membrane preperation induced with CYPs substrate phenylalanine showed no production of oxime.  Our original A2 template have been reported to work (1) and we therefore suspect the mutations and/or the prefix suffix to have influenced the activity. To explore this, we plan to analyse the device BBa_ K527001 - the CYPA2 inserted in an expressionvector by the DTU_Denmark2 user assembly method which doesn't contain prefix/suffix but does contain the mutation
References:
(1) Wittstock U, Halkier BA. Cytochrome P450 CYP79A2 from Arabidopsis thaliana L. Catalyzes the conversion of L-phenylalanine to phenylacetaldoxime in the biosynthesis of benzylglucosinolate. J Biol Chem. 2000 May 12;275(19):14659-66.
B1 the DTU way
We mutated the CYPB1 to remove an illigal restrictionsite and confirmed this by sequencing. We tried to insert the CYP in the expression vector as well as the shipping plasmid. But we was unable to get good results of a colony PCR reaction because our primers didn't bind. (An ekstra set of primers didn't either). Due to time pressure we turned to another assembly methods. Our good friends at DTU (The powerpuff team) have designed an assembly method using the User enzyme and they got our mutated CYPB1 and made user-friendly assembly tales on it, and on the promoter, RBS and terminator which we had used from the kit. We thereafter made a very easy, quick and succesful assembly. Afterwards we expressed the CYP and confirmed that a membrane preperation of BL21 cells were able to confirm trypthophane to an oxime with a TLC
TLC
cB1: B1 with illegal restriction sites.
User B1: B1 assembled with USER assembly method
Tryptophane: Substrate for B1.
Oxime: Reference oxime.
As is observed from the TLC, the bond of the reference oxime has moved slightly shorter than the oximes produced by the cytochromes. The reference oxime was the oxime produced by CYP79A1 and would probably migrate a little different, but insignificant.
We believe that the difference in migration is due to the fact that the container used, caused a smiling effect on the TLC.  Because of limited time, we did not have time to reproduce data in another container, but we are convinced that if we used an other container, the reference oxime would verify the production of oximes produced by B1.
CO-measurement
A specific characteristic of p450's, is their absorbance of light at 450 nm in their active state. As a part of our characterization, we have performed CO-measurements on the purified p450 membrane fractions, which supposedly should contain a enriched amount of p450 enzyme.
CO (carbon monoxide) is added to a membrane sample, by blowing bubbles at approximately 1 bubble/sec in 60 seconds.  The reason for doing this, is that the CO binds the heme group of cytochrome p450, which holds the p450 in its active conformation and shifts the absorbance from 420 to 450 nm. For the CO to bind, it is necessary to reduce the heme group with a strong reducing agent as dithionite.
Unfortunately, we did not observe a peak at 450 nm. We did although repeatedly observe peaks at 420. This is most probably something endogenous substance in E.coli. We did not perform a negative control, which could have indicated whether the 420 was only a result of endogenous compounds or the oxidized form of p450.
Although all active p450's theoretically should absorb at 450 nm in their active state, it is in some cases possible to have an active enzyme without observing the peak at 450 nm.
As the CO-measurement shows, there is no peak at 450 nm for the User assembled B1, but we did observe an oxime, indicating that the p450 in fact was active.
We have included a spectrum of an active p450 below. This is what we would have liked to observe. The different lines, are measurements at different times. It takes a little while for the dithionite to reduce the cytochrome. This is why there is an increase in the absorbance at 450 nm. Accordingly there is a decrease at 420 nm, as the cytochrome shifts to absorbing at 450 nm. Also, there is an increase and decrease at 420 and 450 nm, respectively, at the latest measurement. This is due to the enzyme being oxidized and returning to the inactive state.
A1 the iGEM way
After a lot of hard work we lost our A1. The sequencing supposed to confirm the insertion of A1 in the shipping plasmid showed that the gene was not alone. Part of our host bacterias own genome had been inserted as well (or we supposed that that is what the random sequence disturbing our A1 was). Due to limited time we were forced to give up on A1.
Killing fungus
We have prevented fungus growth with oximes.
To qualitatively test whether oximes could prevent fungi from growing, we applied purified oxime to a fungi growing in a culture dish. Water was also applied as a negative control.
The result is shown below, the spot indicating where oxime was applied.
The human Cyps
After extensive research on the topic we were finally able to uptain physical human CYP DNA that was kindly sent to us from Dr. Guengerich from Vanderbilt University in Nashville. Unfortunently we never recieved the sequence information. A few rescue mission was performed: we attempted to sequence the CYPs ourselves with information found on ncbi.com and we then attempted to assemble the device with help from the DTU-2 team But we were under time pressure and even though we actually got the device for human CYP1A1 and CYP2C9 assembled, we have not had the time to produce a prober BioBrick and test it's function (with an A-YES-Assay). We still believe in the project and it has been heartbreaking to accept that we couldn't get it done in time.
Retrieved from " http://2011.igem.org/Team:Copenhagen/Results "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
