<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Copenhagen_Protocol"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Copenhagen/Protocol">Page               </A></LI><LI><A href="/Talk:Team:Copenhagen/Protocol">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Copenhagen/Protocol&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Copenhagen/Protocol&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Copenhagen/Protocol" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Copenhagen/Protocol</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="navbar-container"><DIV id="navbar"><A href="https://2011.igem.org/Team:Copenhagen ">Home</A><A href="https://2011.igem.org/Team:Copenhagen/Team">Team</A><A href="https://2011.igem.org/Team:Copenhagen/Project">Project</A><A href="https://2011.igem.org/Team:Copenhagen/Safety">Safety</A><A href="https://2011.igem.org/Team:Copenhagen/Ethics">Ethics</A><A href="https://2011.igem.org/Team:Copenhagen/Sponsors">Sponsors</A></DIV></DIV></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Protocols"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Protocols</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#First_Week_22-24_of_June"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">First Week 22-24 of June</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Mutations"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Mutations</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Competent_Cells"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Competent Cells</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#LBamp_Plates"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">LBamp Plates</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Transformations"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Transformations</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-7"><A href="#Second_Week"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Second Week</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Mini_prep"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Mini prep</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Restrictionsite_analysis"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Restrictionsite analysis</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Third_Week"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Third Week</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Prefix_and_suffix_PCR_Reaction"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Prefix and suffix PCR Reaction</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#PCR_purification"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">PCR purification</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Double_Digest"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Double Digest</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#Fourth_Week"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Fourth Week</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Gel_Extraction_Protocol"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Gel Extraction Protocol</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Digestion_of_Linearized_Plasmid_Backbones"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Digestion of Linearized Plasmid Backbones</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Ligation_Protocol"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">Ligation Protocol</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Transformation_Protocol_for_pSB1C3"><SPAN class="tocnumber">5.4</SPAN><SPAN class="toctext">Transformation Protocol for pSB1C3</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#PCR_of_BioBrick_in_pSB1C3"><SPAN class="tocnumber">5.5</SPAN><SPAN class="toctext">PCR of BioBrick in pSB1C3</SPAN></A></LI><LI class="toclevel-1 tocsection-20"><A href="#Fifth_Week"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Fifth  Week</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#Standard_Assembly"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Standard Assembly</SPAN></A></LI><LI class="toclevel-1 tocsection-22"><A href="#Eighth_Week"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Eighth Week</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#Expression_.26_Purification_in_BL21"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">Expression &amp; Purification in BL21</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#SDS_PAGE"><SPAN class="tocnumber">7.2</SPAN><SPAN class="toctext">SDS PAGE</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#TLC"><SPAN class="tocnumber">7.3</SPAN><SPAN class="toctext">TLC</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#USER-cloning"><SPAN class="tocnumber">7.4</SPAN><SPAN class="toctext">USER-cloning</SPAN></A></LI><LI class="toclevel-1 tocsection-27"><A href="#Tenth_week"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Tenth week</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#Western_Blot"><SPAN class="tocnumber">8.1</SPAN><SPAN class="toctext">Western Blot</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Protocols"><B>Protocols</B></SPAN></H2><H2><SPAN class="mw-headline" id="First_Week_22-24_of_June">First Week 22-24 of June</SPAN></H2><H3><SPAN class="mw-headline" id="Mutations">Mutations</SPAN></H3><P><B>Ingredients</B></P><P>Site directed mutagenesis of 3 plant CYP450 (79A1, 79A2, 79B1).
</P><P>We aim to destroy restrictionenzymes recognitionsites.
</P><P>Primers was supplied by IDT
</P><P>
10 μl of 5× reaction buffer
</P><P>X μl (50 ng) of dsDNA template
</P><P>X μl (125 ng) of oligonucleotide primer #1
</P><P>X μl (125 ng) of oligonucleotide primer #2
</P><P>1 μl of dNTP mix
</P><P>ddH2O to a final volume of 50 μl
</P><P>Then add
</P><P>1 μl of X7 fusion DNA polymerase
</P><P><B>Poly Chain Reaction</B></P><P>We ran a PCR to syntesise and amplify our mutated CYP's.
</P><P>
Cycling Parameters for the QuikChange Site-Directed Mutagenesis Method
</P><P>Cycles 12 
</P><P>Temperature 98°C Time 30 seconds
</P><P>Temperature 55°C Time 1 minute
</P><P>Temperature 72°C Time 30 seconds/kb of plasmid length 
</P><P><B>Digestion</B></P><P>We aim to remove the parentel CYP.
</P><P>We take advantage of the fact that this CYP is methylated on cytosines. Dpn is a restriction enzyme that cuts DNA which is methylated - therefore our new mutated CYPs remain untouched.  
</P><P>1. Add 1 μl of the Dpn I restriction enzyme (10 U/μl) directly to each amplification reaction.
</P><P>2. Gently and thoroughly mix each reaction mixture by pipetting the solution up and down
several times. Spin down the reaction mixtures in a microcentrifuge for 1 minute and
immediately incubate each reaction at 37°C (in a heater with lid or in a 37°C room) for 1 hour to digest the parental (i.e., the
nonmutated) supercoiled dsDNA.
</P><P><B>Source</B></P><P>Adapted from
QuikChange™ Site-Directed
Mutagenesis Kit
INSTRUCTION MANUAL
</P><H3><SPAN class="mw-headline" id="Competent_Cells">Competent Cells</SPAN></H3><P><B>E. coli Calcium Chloride competent cell protocol</B></P><P>1. Inoculate a single colony into 5mL Lb in 15mL falcon tube. Grow
O/N at 37°C.
</P><P>2. Use 1mL to inoculate 100mL of LB in 250mL bottle the next morning.
</P><P>3. Shake at 37°C for 1.5-3hrs until OD600 = 0.4-0.8
</P><P>Then….
1. Put the cells on ice for 10 mins (keep cold form now on).
</P><P>2. Collect the cells by centrifugation in the big centrifugue for 10 mins
at 6krpm
</P><P>3. Decant supernatant and gently resuspend on 10 mL cold 0.1M
CaCl (cells are susceptible to mechanical disruption, so treat them
nicely).
</P><P>4. Incubate on ice x 20 mins
</P><P>5. Centrifuge as in 2
</P><P>6. Discard supernatant and gently resuspend on 5mL cold
0.1MCaCl/15%Glycerol (from a 85% stock)
</P><P>7. Dispense in microtubes (300μL/tube). Freeze in -80°C.
</P><P><B>Source:</B></P><P>Adapted from 
<A href="http://www.med.nyu.edu/medicine/labs/blaserlab/Protocols/E-coli_competent_cells_protocol_&amp;_transformation.pdf" class="external free" rel="nofollow">http://www.med.nyu.edu/medicine/labs/blaserlab/Protocols/E-coli_competent_cells_protocol_&amp;_transformation.pdf</A></P><H3><SPAN class="mw-headline" id="LBamp_Plates">LBamp Plates</SPAN></H3><P>1. 500 ml LB agar and 500 μL amphicilin/Or other antibiotic (If IPTG is needed ad it to a final concentration of 1mM, here 500 μL, which it is in the LBcam plates) 
</P><P>2. Pour on plates
</P><P>3. Leave with the lid half on for 30 minutes at room temperature
</P><P>4. Put in refrigerator until needed.
</P><H3><SPAN class="mw-headline" id="Transformations">Transformations</SPAN></H3><P><B>Transformation of Ca++ competent cells</B></P><P>1. Put ~50μL of competent cells to prechilled microtubes. Wait 1 minute. Add 10μL (1μL if it is step 26)  of circular plasmid (c. 50 ng) or all of a ligation reaction of plasmid DNA.
</P><P>2. Incubate for 15 mins on ice.
</P><P>3. Heat shock for 30 seconds at 42°C. Put back on ice.
</P><P>4. Add 70 μL LB
</P><PRE>If the resistence is cam, the sample has to incubate in an hour at 37°C while shaking.
</PRE><P>5. Plate the whole lot in LBamp/LBcam-IPTG plates
</P><P>6. Leave the plates at 37°C O/N
</P><PRE>If the transformation on LBcam-IPTG has succeded, the colonies are supposed to stay white. 
If they are red, the cells have not recieved the insert
</PRE><H2><SPAN class="mw-headline" id="Second_Week">Second Week</SPAN></H2><H3><SPAN class="mw-headline" id="Mini_prep">Mini prep</SPAN></H3><P><B>Ingredients</B></P><P>QIAprep® Spin Miniprep Kit
</P><UL><LI> Buffer P1 (with LyseBlue)
</LI></UL><UL><LI> Buffer P2 - Lysis Buffer
</LI></UL><UL><LI> Buffer N3 - Neutralisation buffer
</LI></UL><UL><LI> Buffer PB - Binding buffer
</LI></UL><UL><LI> Buffer PE - Wash Buffer
</LI></UL><UL><LI> Buffer EB - Elution Buffer
</LI></UL><UL><LI> 5 ml bacterial overnight culture transformed with our BioBrick
</LI></UL><P><B>Miniprep</B></P><P>1. Pellet 5 ml bacterial overnight culture by centrifugation at 4500 rpm for 10 min at room temperature (20°C).
</P><P>2. Resuspend pelleted bacterial cells in 500 μl Buffer P1 and transfer and divide in two
microcentrifuge tubes.
</P><P>3. Add 250 μl Buffer P2 to each tube and mix thoroughly by inverting the tube
4–6 times until the solution becomes blue. Do not allow the lysis reaction to
proceed for more than 5 min. 
</P><P>4. Add to each 350 μl Buffer N3 and mix immediately and thoroughly by inverting the tube
4–6 times. With LyseBlue reagent, the solution will turn colorless.
</P><P>5. Centrifuge for 10 min at 13,000 rpm (~17,900 x g) in a table-top
microcentrifuge.
</P><P>6. Apply the supernatant from step 5 to the QIAprep spin column by decanting or
pipetting. Centrifuge for 60 s and discard the flow-through.
</P><P>7. Wash the QIAprep spin column by adding 0.5 ml Buffer PB.
Centrifuge for 60 s and discard the flow-through
</P><P>8. Wash the QIAprep spin column by adding 0.75 ml Buffer PE.
Centrifuge for 60 s and discard the flow-through,
Transfer the QIAprep spin column to the collection tube.
</P><P>9. Centrifuge for 1 min to remove residual wash buffer.
</P><P>10. Place the QIAprep column in a clean 1.5 ml microcentrifuge tube. To elute DNA,
add 50 μl Buffer EB (10 mM Tris·Cl, pH 8.5) to the center of the
QIAprep spin column, let stand for 1 min, and centrifuge for 1 min.
</P><P><B>Adapted from</B></P><P>Quick-StartProtocol QIAprep® Spin Miniprep Kit October 2010
<A href="http://www.qiagen.com/literature/render.aspx?id=201081" class="external autonumber" rel="nofollow">[1]</A></P><H3><SPAN class="mw-headline" id="Restrictionsite_analysis">Restrictionsite analysis</SPAN></H3><P>To confirm that our mutations worked we analyses the purified plasmids by cutting them with the restrictionenzyme whose recognitionsite we aim to remove. 
</P><P>Ingrediens
</P><P>1μl BSA (10x) diluted
ca. 500ng Plasmid
1μl Restrictionenzyme
1μl NEBuffer
Water up to 10μl
</P><P>Cut at 37 degress for 1 hour. 
</P><P>Add 2,5μl loading buffer to the digestion
</P><P>Run it on a gel
</P><P>Visualize it and hope to see just one band. If two bands are present the mutation has not worked. (The original plasmid conatined a restrictionsite in the vector as well as in the CYP)
</P><H2><SPAN class="mw-headline" id="Third_Week">Third Week</SPAN></H2><H3><SPAN class="mw-headline" id="Prefix_and_suffix_PCR_Reaction">Prefix and suffix PCR Reaction</SPAN></H3><P>We add prefix and suffix to our newly made BioBricks with PCR
</P><P><B>Ingredients</B></P><P>
Water to a final volume of 50μl
</P><P>10 μl 5x Pfusion HF buffer
</P><P>1 μl 10mM dNTP
</P><P>0,5μM Primer A
</P><P>0,5μM Primer A
</P><P>0,5 μl template (ca. 150 ng Template)
</P><P>0,50 μl Pfusion X7 Polymerase
</P><P><B>PCR</B></P><P>
Initial denaturation 98°C
</P><P>
Cycles 25 
</P><P>Denaturation Temperature 98°C Time 10 seconds
</P><P>Anneal Temperature 55°C Time 30 seconds (The annealing temperature is determined on the basis of the sequence of the primer. Use <A href="http://www.finnzymes.fi/tm_determination.html" class="external text" rel="nofollow">the calculator</A>) 
</P><P>Extension Temperature 72°C Time 15 seconds/kb of plasmid length 
</P><P>
Final Extension 72°C Time 10 Minuts
</P><P><B>Adapted from</B></P><P>Finnzymes Phusion® High-Fidelity DNA Polymerase instruction guide
</P><H3><SPAN class="mw-headline" id="PCR_purification">PCR purification</SPAN></H3><P>When you have added the prefix and suffix you need to purify you BioBrick. You do this by using PCR purification.
</P><P><B>Ingredients</B></P><UL><LI> PE buffer
</LI></UL><UL><LI> PB buffer
</LI></UL><UL><LI> EB buffer
</LI></UL><UL><LI> PCR DNA
</LI></UL><P><B>Procedure</B></P><P>1. Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix. 
</P><P>2. If pH indicator I has beein added to Buffer PB, check that the color of the mixture is
yellow.
</P><P>3. Place a QIAquick spin column in a provided 2 ml collection tube.
</P><P>4. To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s.
</P><P>5. Discard flow-through. Place the QIAquick column back into the same tube.
</P><P>6. To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s.
</P><P>7. Discard flow-through and place the QIAquick column back in the same tube.
Centrifuge the column for an additional 1 min.
</P><P>8. Place QIAquick column in a clean 1.5 ml microcentrifuge tube.
</P><P>9. To elute DNA, add 50 μl Buffer EB (10 mM Tris•Cl, pH 8.5) to
the center of the QIAquick membrane wait 1 min and centrifuge the column for 1 min.
</P><H3><SPAN class="mw-headline" id="Double_Digest">Double Digest</SPAN></H3><P>Of BioBrick with prefix and suffux in order to ligate it with a plasmid.  
We have used two different approaches of the double digest. The differences are specified below as iGEM or Kenneth.
</P><P><B>Ingredients</B></P><P>iGEM: 500 ng DNA / Kenneth: 700ng DNA
</P><P>Water until 42,5 μl (43 ul if BSA is excluded)
</P><P>5 μl NEB buffer (Decide which buffer: <A href="http://www.neb.com/nebecomm/DoubleDigestCalculator.asp" class="external autonumber" rel="nofollow">[2]</A>)
</P><P>0,5 μl BSA (Not necessary if the enzymes are HF)
</P><P>1 μl Restrictionsenzyme A (Decide which restriction enzymes: <A href="http://ginkgobioworks.com/support/BioBrick_Assembly_Manual.pdf" class="external autonumber" rel="nofollow">[3]</A>)
</P><P>1 μl Restrictionsenzyme B
</P><P>Mix by flicking
</P><P>Incubate for 30 min in 37°C (BioBrick)or O/N in 37°C (Plasmid).
</P><P>iGEM: Deactivate the restriction enzymes by incubating at 80 degrees in 20 min.
</P><P>Kenneth: The restriction enzymes are deactivated and excluded when you purify the samples by gel electroforesis followed by <A href="/Team:Copenhagen/Protocol#Gel_Extraction_Protocol" title="Team:Copenhagen/Protocol">gel extraction</A>. This approach exclude the restrictionenzymes and other contaminations, but has aswell a down side - you loose some of the DNA. 
</P><P>
Freeze until you need them. The BioBrick is now ready for ligation.
</P><P><B>Adapted from</B></P><P>The BioBrick™ Assembly Manual from NEB and Ginkgo BioWorks
<A href="http://ginkgobioworks.com/support/BioBrick_Assembly_Manual.pdf" class="external free" rel="nofollow">http://ginkgobioworks.com/support/BioBrick_Assembly_Manual.pdf</A></P><H2><SPAN class="mw-headline" id="Fourth_Week">Fourth Week</SPAN></H2><H3><SPAN class="mw-headline" id="Gel_Extraction_Protocol"> Gel Extraction Protocol</SPAN></H3><UL><LI>After gelelectroforesis cut out the DNA bands (approx.. 1600 bp). Visualize by ultraviolet light (this does not damage the DNA as the Imaganizer does).
</LI></UL><UL><LI>Weigh the gelpieces.
</LI></UL><P><B>Gel Extraction Protocol</B></P><UL><LI>Use 600 uL QG buffer pr. Gelpiece.  
</LI></UL><UL><LI>Incubate at 50°C until the pieces are meltet (approx. 10-15 min). Speed up the melting process by vortexing.
</LI></UL><UL><LI>Add 1 gel volume (100mg ~100 uL) of Isopropanol (2-propanol) and mix.
</LI></UL><UL><LI>Transfer the samples to QIAquick spin columns (max. 800 uL) and centrifuge for 1 min. For samples volumes of more than 800 uL simply load and spin again.
</LI></UL><UL><LI>Discard flow-through.
</LI></UL><UL><LI>Add 500 uL QG buffer and centrifuge for 1 min.
</LI></UL><UL><LI>Discard the flow-through.
</LI></UL><UL><LI>Add 750 uL PE buffer and centrifuge for 1 min. IF you are to use the DNA for blunt-end ligation or direct sequencing let the column stand for 2-5 min. before centrifugation.
</LI></UL><UL><LI>Discard the flow-through
</LI></UL><UL><LI>Spin again for 1 min and discard the flow-through.
</LI></UL><UL><LI>Place the QIAquick column into an Eppendorff tube.
</LI></UL><UL><LI>Eluate the DNA by adding 50 uL EB buffer. Remember to load the EB buffer in the center of the membrane. Let the column stand for 1 min. Centrifuge for 1 min.
</LI></UL><UL><LI>Freeze the samples for later use.
</LI></UL><P>You should make use of the calendar feature on the wiki and start a lab notebook.  This may be looked at by the judges to see how your work progressed throughout the summer.  It is a very useful organizational tool as well.
</P><H3><SPAN class="mw-headline" id="Digestion_of_Linearized_Plasmid_Backbones">Digestion of Linearized Plasmid Backbones</SPAN></H3><P>This protocol is intended only for the preparation of the plasmid backbones (and NOT the expression vectors), before you can ligate it with your BioBrick.
</P><P><B>25 uL Enzyme Master Mix</B></P><P>5 uL NEB Buffer 2
</P><P>0,5 uL BSA
</P><P>0,5 uL EcoRI
</P><P>0,5 uL PstI
</P><P>0,5 uL DpnI
</P><P>18 uL H2O
</P><UL><LI> Add 4 uL linearized plasmid backbone (100 ng in total)
</LI></UL><UL><LI> Add 4 uL of the Master Mix
</LI></UL><UL><LI> Digest at 37 degrees 30 min/<B>O/N at 4 degrees</B>.
</LI></UL><UL><LI> We used two different approaches: 
</LI></UL><P>iGEM: Deactivate the restriction enzymes by incubating at 80 degrees in 20 min. You now have a plasmid concentration of 12,5 ng/uL 
Kenneth: Purify the plasmids by gel electrophoresis followed by <A href="/Team:Copenhagen/Protocol#Gel_Extraction_Protocol" title="Team:Copenhagen/Protocol">gel extraction</A>. 
</P><UL><LI> Freeze until you need them. The plasmids are now ready for ligation.
</LI></UL><H3><SPAN class="mw-headline" id="Ligation_Protocol">Ligation Protocol</SPAN></H3><P>How to ligate a BioBrick into a plasmid backbone: Use 3 times insert (BioBrick) to 1 time plasmid (3:1) in molar amounts. In calculating the amounts of DNA consider the length of the BioBrick and the plasmid, see example below. Use 25-30 ng plasmid. 
</P><P>Note: iGEM are ligating eqimolar amounts of insert and plasmid as the only one in the world.
</P><P>Following the two different approaches of digestion, where you obtain different concentrations of the DNA, use either iGEM or Kenneths procedure for ligation. See below.
</P><P>Use eppendorf tubes for the ligation, so you can add the competent cells directly into the tubes. 
</P><P><I>Example:</I></P><P><I>pSB1C3: 3000 bp</I></P><P><I>CYP79B1: 1600 bp</I></P><P><I>3000/1600 ≈ 2 (molar ratio)</I></P><P><I>25 ng/2= 12,5 ng (25 ng and 12,5 ng are equivalent molar amounts in grams)</I></P><P><I>In this procedure we use 3 times excess of insert (B1):</I></P><P><I>12,5 ng *3= 37,5 ng B1</I></P><P><B>iGEM</B></P><P>2 uL digested plasmid (12,5 ng/uL)
</P><P>x uL digested BioBrick (10 ng/uL)
</P><P>1 uL T4 Ligase buffer
</P><P>0,5 uL T4 DNA Ligase
</P><P>Add water to a final volume of 10 uL
</P><P><B>Our iGEM way</B></P><P>x ul digested plasmid (25 ng)
</P><P>x ul digested BioBrick (~40 ng if the insert is half the size than the plasmid)
</P><P>1 ul T4 Ligase buffer
</P><P>0,5 ul T4 DNA ligase
</P><P>Add water to a final volume of 10 ul
</P><P><B>Ligation O/N at 4 degrees</B></P><P><B>Kenneth</B></P><P>If you wish to use Kenneths digestion approach, you have to determine the concentrations of the BioBrick and the plasmid by NanoDrop.
</P><P>
x uL gel extracted plasmid (25-30 ng)
</P><P>x uL gel extracted BioBrick 
</P><P>2 uL T4 ligase buffer
</P><P>0,5 uL T4 DNA ligase
</P><P>Add water to a final volume of 20 uL
</P><P>
Ligate at room temperature for 10 min/<B>Ligation O/N at 4 degrees</B></P><P>Keep on ice until transformation.
</P><P>
To ensure that the ligation has succeded make at least two different control saples:
</P><P><B>C1</B></P><P>1 uL T4 ligase buffer
</P><P>0,5 uL digested plasmid
</P><P>8,5 uL H2O
</P><P><B>C2</B></P><P>1 uL T4 ligase buffer
</P><P>0,5 uL digested plasmid
</P><P>0,5 uL T4 ligase
</P><P>8 uL H2O
</P><P>
Transform the controls as if they were a real transformation containing a BioBrick.
</P><H3><SPAN class="mw-headline" id="Transformation_Protocol_for_pSB1C3">Transformation Protocol for pSB1C3</SPAN></H3><P>The only exeption to the procedure is an incubation step after adding the LB medium. Incubation for 1 hr. at 37 degrees while shaking.
</P><P>Plate it on agar plates with chloramphenicol.
</P><H3><SPAN class="mw-headline" id="PCR_of_BioBrick_in_pSB1C3">PCR of BioBrick in pSB1C3</SPAN></H3><P>4 uL HF buffer
</P><P>0,4 uL 10 mM dNTP
</P><P>1 uL 10 pmol/uL primer A
</P><P>1 uL 10 pmol/uL primer B
</P><P>0,2 uL x7 polymerase
</P><P>Add water uptil 20 uL
</P><P>Use a pipette tip to collect cells from the plate, and transfer it to the PCR mix. End by scrabing the tip against the inner surface of the PCR tube.
</P><P><B>PCR Program</B></P><P>10 sec - 98°C
</P><P>30 sec - 68°C (The annealing temperature must never be higher than the extension temperature)
</P><P>
30 sec - 72°C
</P><P>∞      - 12°C
</P><P>29 cycles
</P><P>Note: The annealing temperature is determined on the basis of the sequence of the primer. Use <A href="https://www.finnzymes.fi/tm_determination.html" class="external text" rel="nofollow">the calculator</A>. Remember that the BioBrick has been shortened in the double digest, so it misses 8 bases. The primer still has theese 8 bases that don't anneal, and they should be excluded in the calculation. The annealing temperature must never be higher than the extension temperature
(because of the length of the primers the annealing temperature calculated will probably be higher than the extension temperature, and should therefore be lowered)
</P><P>Note: The extension time should be 30 sek/kb
</P><H2><SPAN class="mw-headline" id="Fifth_Week">Fifth  Week</SPAN></H2><H3><SPAN class="mw-headline" id="Standard_Assembly">Standard Assembly</SPAN></H3><UL><LI> Double Digestion
</LI></UL><UL><LI> Ligation
</LI></UL><UL><LI> Transformation
</LI></UL><H2><SPAN class="mw-headline" id="Eighth_Week"> Eighth Week</SPAN></H2><H3><SPAN class="mw-headline" id="Expression_.26_Purification_in_BL21">Expression &amp; Purification in BL21</SPAN></H3><P><B>Growing the cells</B></P><P>Day 1:
</P><UL><LI> Book the centrifuges for day 3.
</LI></UL><UL><LI> Prepare starter culture in 5 mL LB + 5 uL antibiotics (amp) and grow O/N at 37 degrees.
</LI></UL><UL><LI> Prepare TB medium for use the next day
</LI></UL><PRE><B>TB formula - for final 1L medium (incl. 100 mL buffer)</B> 
24,00 g Yeast Extract
12,00 g Tryptone 
4 mL 99% Glycetol
H2O uptil 900 mL
</PRE><PRE><B> Buffer (200 mL)</B> 34 mL 1M KH2PO4
144 mL 1M K2HPO4
Add water uptil 200 mL
The pH should end up around 7,2-7,4
</PRE><UL><LI> Autoclave both the medium and the buffer.
</LI></UL><UL><LI> Prepare 1 M 5-aminolevulinic acid hydrochloride.
</LI></UL><P>
Day 2:
</P><UL><LI> Inoculate 200 mL TB (180 mL medium + 20 mL buffer) with the 5 mL starter culture from yesterday. Grow the cells at 37°C while shaking until reaching ~OD600=0,5 (2-4 h.)
</LI></UL><UL><LI> Take a sample of 1 mL for the SDS-PAGE assay.
</LI></UL><PRE><B>Sample preparation for the SDS-PAGE assay:</B>
* Dilute the samples (with water) until reaching ~OD600=0,5. 
* Spin down the cells for 5 min at 20000 G.
* Discard the supernatant and resuspend the cells in 50 uL SDS buffer. 
<B>The SDS buffer is very toxic - wear glows and work in the fume hood</B></PRE><UL><LI> Move the shaking flask to the 28°C shaker and induse the cells with <B>steril</B> 1 mM IPTG (inducing the promoter) and 1 mM 5-aminolevulinic acid hydrochloride (precursor of the HEM-group in p450).
</LI></UL><UL><LI>Take further samples of 1 mL for the SDS-PAGE assay at 1, 2 &amp; 4 hours. 
</LI></UL><UL><LI>The flask has to remain at 28°C O/N  
</LI></UL><UL><LI>Prepare Standard Buffer
</LI></UL><PRE><B>Standard Buffer</B>
50mM Tricine pH 7,9
100mM NaCl
</PRE><PRE>Add 2mM DTT and 5mM EDTA on the day of use
</PRE><P>Day 3:
</P><UL><LI>Take out the last sample for the SDS-PAGE assay.
</LI></UL><P><B>Membrane preparation</B></P><UL><LI> Harvest and sediment cells by centrifugation in 1L tubes at 1000G for 15 min at room temperature.
</LI></UL><UL><LI> Discard the supernatant and wash the cell pellet with sterile H2O and cetrifuge again at 1000G for 15 min.
</LI></UL><UL><LI> Discard the supernatant and resuspend the cells in Standard Buffer (as little volume as possible ~50mL).
</LI></UL><P>FROM HERE, KEEP EVERYTHING ON ICE!
</P><UL><LI> Apply the suspended cells to the French Press. (You migth want to grab a supervisor for the French Press).
</LI></UL><UL><LI> Add 200 ug DNase and 2mM (final concentration) MgCl2 and incubate at 4°C (on ice) on stirring board for ~1h until the DNA has been degraded.
</LI></UL><UL><LI> Sediment the cell debris and unbroken cells by centrifugation at 15000G for 15 min. <B>at 4°C</B>.
</LI></UL><UL><LI> Apply the supernatant (with both the membran- and cytosolic proteins) to ultracetrifuge tubes and adjust the tubes on the scale with a max. deviation 0,05 g. Centrifuge for 1 h at 46000 rpm at 4°C. 
</LI></UL><UL><LI> Poor away the supernatant and wash pellet twice with Standars Buffer
</LI></UL><UL><LI> Add 1 mL Standard Buffer (0,5 mL at a time) to loosen the pellet using a brush and transfer the pellet to a 5 mL Potter Elvehjem to homogenize the membranes and continue until EVERYTHING has been collected. Turn the mortar around while pulling it up and down. 
</LI></UL><UL><LI> Freeze the membranes and store them at -80°C.
</LI></UL><H3><SPAN class="mw-headline" id="SDS_PAGE">SDS PAGE</SPAN></H3><P>Run the 4 samples from day 2 and 3 on a gel for 1h at 200V followed by staining (a day) and destaining (2h).
</P><P>Use the MOBS buffer
</P><H3><SPAN class="mw-headline" id="TLC">TLC</SPAN></H3><UL><LI> Mix: 
</LI></UL><P>1uL 1M Tricine pH 7,5
</P><P>2uL 10mg/mL DLPC
</P><P>2uL 61,6mM NADPH
</P><P>10mL p450 microsomes
</P><P>3uL SbCPR 3.14 01-04-07
</P><P>2uL [3-14C] tyrosine/phenylalanin/Tryptophan
</P><UL><LI>Incubate for 1h at 30°C while shaking.
</LI></UL><UL><LI>Apply the samples (5 uL at a time), the substrates (0,5uL dissolved in 5 uL H2O) and oximes on a TLC plate.
</LI></UL><UL><LI> When the TLC plate is dry, pour ~ 1cm buffer in the tank and place the TLC plate in the buffer. Leave it for 20-30 min.
</LI></UL><UL><LI> Take the TLC plate up for drying (~1h).
</LI></UL><UL><LI> Cut the important part of the TLC plate with scissor and put it in a plastic pocket. Close the sides of the plastic pocket with tape
</LI></UL><UL><LI> Transfer the plate to the cassette wtih the white side up
</LI></UL><UL><LI> Wait on day to see the result
</LI></UL><H3><SPAN class="mw-headline" id="USER-cloning">USER-cloning</SPAN></H3><P>Delivery from DTU-team-2 in pcr-tubes:
</P><P>C1: Ampicilin marker cassette
</P><P>C2: pIPTG
</P><P>C4: CYP79A2
</P><P>C5: CYP79B2
</P><P>C6: terminator
</P><P>30: Plasmid_Mam
</P><P><B>Procedure</B></P><UL><LI> 1 uL USER enzym
</LI><LI> 0.5 uL NEB4 (af 10x)
</LI><LI> 0.5 uL BSA (af 10x)
</LI><LI> 1uL vector
</LI><LI> 8 uL PCR product equally distributed between the fragments 
</LI></UL><P>(reaction 1: C2 + C4 + C6 + C1 + 30; reaction 2: C2 + C5 + C6 + C1 + 30) 
</P><P>Incubate 40 min at 37 C and 30 min at 25 C for optimal efficiency.
</P><H2><SPAN class="mw-headline" id="Tenth_week">Tenth week</SPAN></H2><H3><SPAN class="mw-headline" id="Western_Blot">Western Blot</SPAN></H3><P><B>Procedure</B></P><P><B>Day1</B></P><UL><LI> Cut nitrocellulose membrane - same size as your gel. Cut two pieces of paper (the thick)- size bigger than your gel.
</LI></UL><UL><LI> Prepare: holder, casette, roll and two pieces of fabric
</LI></UL><UL><LI> Pour transferbuffer intro the holder
</LI></UL><UL><LI> Make your sandwich in the casette: 
</LI></UL><PRE>      1. Piece of fabric
</PRE><PRE>      2. paper
</PRE><PRE>      3. gel
</PRE><PRE>      4. membrane
</PRE><PRE>      5. paper
</PRE><PRE>      6. piece of fabric
</PRE><P>Everything has to be soaked in transferbuffer before and rolled to prevent air bubbles.
</P><UL><LI>  Put the casette in the holder, remember a block of ice and a magnet. Start the machine. Instructions: 40 min at 100 V.
</LI></UL><UL><LI> Prepare skimmed milk (5 % dilution  = 2,5 g skim milk powder in 50 ml PSB TWEEN)
</LI></UL><UL><LI> After 40 min, disassembly the cassette, put the membrane in a holder with the skimmed milk. Remember to have the gelside up. Leave it there for an hour.
</LI></UL><UL><LI>  Pour in the primary antibody (1:2000 = 25 uL)
</LI></UL><P><B>Day 2</B></P><UL><LI> Discard the skimmed milk with the primary antibody and wash the membran quikly with PBS TWEEN.
</LI></UL><UL><LI> Wash the membran 3 times 5 min with PBS TWEEN.
</LI></UL><UL><LI> Prepare 25 mL 5% skimmed milk (1,25 g skim milk powder)
</LI></UL><UL><LI> Discard the PBS TWEEN after the last wash and add the skimmed milk.
</LI></UL><UL><LI> Add the secondary antibody (1:2000 = 12,5 uL) and incubate for 1 h.
</LI></UL><UL><LI> Discard the skimmed milk and wash 6 times 5 min with PBS TWEEN.
</LI></UL><UL><LI> Add Horseradish peroxidase and take some pictures.
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Copenhagen/Protocol">http://2011.igem.org/Team:Copenhagen/Protocol</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Copenhagen/Protocol" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Copenhagen/Protocol" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Copenhagen/Protocol&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Copenhagen/Protocol&amp;oldid=89389" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>