<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NTNU_Trondheim_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NTNU Trondheim/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="nav-wrapper"><UL class="ntnu_navbar"><LI class="ntnu_menu_option ntnu_dropdown no-icon"><A href="javascript:void(0)" class="ntnu_dropbtn" onclick="void(0)" id="menu-team">Team</A><DIV class="ntnu_dropdown_content"><A href="https://2018.igem.org/Team:NTNU_Trondheim/Team">Team Members</A><A href="https://2018.igem.org/Team:NTNU_Trondheim/Collaborations">Collaborations</A></DIV></LI><LI class="ntnu_menu_option ntnu_dropdown no-icon"><A href="javascript:void(0)" class="ntnu_dropbtn" id="menu-project">Project</A><DIV class="ntnu_dropdown_content"><A href="https://2018.igem.org/Team:NTNU_Trondheim/Description">Description</A><A href="https://2018.igem.org/Team:NTNU_Trondheim/Design">Design</A><A href="https://2018.igem.org/Team:NTNU_Trondheim/Model">Model</A><A href="https://2018.igem.org/Team:NTNU_Trondheim/Results">Results</A><A href="https://2018.igem.org/Team:NTNU_Trondheim/Demonstrate">Demonstrate</A></DIV></LI><LI class="ntnu_menu_option ntnu_dropdown no-icon"><A href="javascript:void(0)" class="ntnu_dropbtn" id="menu-lab">Lab</A><DIV class="ntnu_dropdown_content"><A href="https://2018.igem.org/Team:NTNU_Trondheim/Experiments">Lab Journal</A><A href="https://2018.igem.org/Team:NTNU_Trondheim/Protocols">Protocols</A><A href="https://2018.igem.org/Team:NTNU_Trondheim/Measurement">Measurement</A><A href="https://2018.igem.org/Team:NTNU_Trondheim/Safety">Safety</A></DIV></LI><LI class="ntnu_menu_option ntnu_dropdown no-icon"><A href="javascript:void(0)" class="ntnu_dropbtn" id="menu-parts">Parts</A><DIV class="ntnu_dropdown_content"><A href="https://2018.igem.org/Team:NTNU_Trondheim/Parts">Parts Overview</A><A href="https://2018.igem.org/Team:NTNU_Trondheim/Improve">New Part</A></DIV></LI><LI class="ntnu_menu_option ntnu_dropdown no-icon"><A class="ntnu_dropbtn" href="https://2018.igem.org/Team:NTNU_Trondheim/InterLab" id="menu-interlabs">InterLab</A></LI><LI class="ntnu_menu_option ntnu_dropdown no-icon"><A href="javascript:void(0)" class="ntnu_dropbtn" id="menu-hp">Human Practices</A><DIV class="ntnu_dropdown_content"><A href="https://2018.igem.org/Team:NTNU_Trondheim/Human_Practices">Integrated Human Practices</A><A href="https://2018.igem.org/Team:NTNU_Trondheim/Public_Engagement">Education and Engangement</A></DIV></LI><LI class="ntnu_menu_option ntnu_dropdown no-icon"><A href="javascript:void(0)" class="ntnu_dropbtn" id="menu-acknowledge">Acknowledgements</A><DIV class="ntnu_dropdown_content" style="min-width: 170px"><A href="https://2018.igem.org/Team:NTNU_Trondheim/Sponsors">Sponsors</A><A href="https://2018.igem.org/Team:NTNU_Trondheim/Attributions">Attributions</A></DIV></LI><LI class="ntnu_menu_option ntnu_dropdown no-icon"><A class="ntnu_dropbtn" href="https://2018.igem.org/Team:NTNU_Trondheim/Awards" id="menu-jamboree">Awards</A></LI></UL></DIV><DIV class="container"><DIV class="headertext" style="padding-left: 50px; padding-right: 50px"><P style="text-align: center">Here is an overview of all the results we have gathered from all biofilm assays. At the end of the page is a summary of the results. For a thorough discussion and analysis of the results, go to our <A href="https://2018.igem.org/Team:NTNU_Trondheim/Demonstrate">Demonstrate</A> page.</P></DIV><H2 class="project-color">Main results</H2><UL class="collapsible"><LI><DIV class="collapsible-header protocol-title">Isolation of pdCas9 and pgRNA from  <EM>E. coli</EM>  Top10 cells</DIV><DIV class="collapsible-body"><SPAN><P>The plasmids were ordered from Addgene and delivered in <I>E. coli</I> Top10 cells. We started by plating the cells on agar plates containing the selective antibiotics for the specific plasmids – Chloramphenicol for selection of pdCas9 and ampicillin for pgRNA. The plates were incubated overnight, and single colonies were picked and inoculated in LB medium with the respective antibiotics. We isolated the plasmids and checked the quantity and quality using the NanoDrop. The results from the NanoDrop are given in <B>Table 1</B>. Since the 260/230-ratio is less than 1.80, it may indicate the presence of contaminants which absorbs strongly at 230 nm. Nevertheless, both the concentrations and the 260/280-ratios indicate that there was enough DNA to continue the process for both replicates of pdCas9 and pgRNA.</P><CENTER><TABLE style="width:100%"><CAPTION><B>Table 1:</B> Nucleic acid concentration, 260/280- and 260/230-ratios of plasmids (pgRNA and pdCas9) isolated from cell cultures of <I>E. coli</I> Top10 cells ordered from Addgene. Two replicates were made from separate cell cultures.
</CAPTION><TBODY><TR><TH><B>Sample (replicate)</B></TH><TH><B>Concentration [ng/µL]</B></TH><TH><B>260/280-ratio</B></TH><TH><B>260/230-ratio</B></TH></TR><TR><TD>pgRNA (1)</TD><TD>62.3</TD><TD>1.86</TD><TD>1.41</TD></TR><TR><TD>pgRNA (2)</TD><TD>68.1</TD><TD>1.84</TD><TD>1.26</TD></TR><TR><TD>pgCas9 (1)</TD><TD>20.1</TD><TD>1.96</TD><TD>1.41</TD></TR><TR><TD>pgCas9 (2)</TD><TD>52.1</TD><TD>1.79</TD><TD>1.05</TD></TR></TBODY></TABLE></CENTER><P>The isolated plasmids were digested with the restriction enzyme BspHI and the plasmid fragments were run on an agarose gel alongside GeneRuler 1 kbp ladder. A digestion tool in the software Benchling was used to predict the fragment lengths and gel patterns for each of the plasmids after cutting with BspHI. The pdCas9 was expected to be cut into two fragments with lengths of 2741 and 3964 bp, and the pgRNA was expected to be cut into three fragments with lengths of 105, 1008 and 1451 bp. As seen in <B>Figure 1</B> and <B>Figure 2</B>, both replicates for pgRNA seem to have bands with the expected lengths. The band at approximately 100 bp seemed to be only vaguely visible, possibly due to the fragment’s short length, which makes it bind a low amount of dye. For pdCas9 only replicate 2 had the correct bands.</P><CENTER><FIGURE><FIGCAPTION><B>Figure 1:</B> The cut sites for BspHI in pgRNA and pdCas9, showing the origin of the expected fragments and their lengths. </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><B>Figure 2:</B> Gel imaging of two replicates of both pgRNA (abbr. pR1 and pR2) and pdCas9 (abbr. pC1 and pC2) isolated from <I>E. coli</I> Top10 cells after digestion with the restriction enzyme BspHI. The samples are run alongside GeneRuler 1 kbp ladder (λ). </FIGCAPTION></FIGURE></CENTER></SPAN></DIV></LI><LI><DIV class="collapsible-header protocol-title">Plasmid transformation into competent  <EM>E. coli</EM>  DH5α cells</DIV><DIV class="collapsible-body"><SPAN><P>The isolated pdCas9 and pgRNA from the <I>E. coli</I> Top10 cells were separately transformed into competent <I>E. coli</I> DH5α cells. The transformed cells were streak plated and single colonies were picked and inoculated the next day. The OD<SUB>600</SUB> values of the overnight cultures were approximately 1 (pdCas9: 1.23; pgRNA: 0.88). The cell cultures were miniprepped and nanodropped. The NanoDrop results, given in <B>Table 2</B>, implied that both pgRNA replicates and pdCas9 replicate 2 might have been RNA contaminated.</P><TABLE style="width:100%"><CAPTION><B>Table 2:</B> Nucleic acid concentration, 260/280- and 260/230-ratios of plasmids (pgRNA and pdCas9) isolated from <I>E. coli</I> DH5α cells transformants. Two replicates were prepared from separate cell cultures.
</CAPTION><TBODY><TR><TH><B>Sample (replicate)</B></TH><TH><B>Concentration [ng/µL]</B></TH><TH><B>260/280-ratio</B></TH><TH><B>260/230-ratio</B></TH></TR><TR><TD>pgRNA (1)</TD><TD>60.2</TD><TD>2.09</TD><TD>2.26</TD></TR><TR><TD>pgRNA (2)</TD><TD>44.2</TD><TD>2.18</TD><TD>2.84</TD></TR><TR><TD>pgCas9 (1)</TD><TD>194.1</TD><TD>1.56</TD><TD>0.73</TD></TR><TR><TD>pgCas9 (2)</TD><TD>53.4</TD><TD>2.25</TD><TD>3.53</TD></TR></TBODY></TABLE><P>Test digest was performed on the isolated plasmids using the restriction enzyme BspHI. The gel imaging is shown in <B>Figure 3</B>. Both plasmids gave the expected number of bands and fragment lengths (pgRNA: 105, 1008 and 1451 bp; pdCas9: 2741 and 3964 bp). This observation verified that the transformation had been a success. Interestingly, replicate 1 of dCas9 seemed to give the right restriction pattern this time (for comparison see Figure 1). Nevertheless, we still decided to only continue using replicate 2 of this plasmid since the reason for the dissimilarities was uncertain.</P><CENTER><FIGURE><FIGCAPTION><B>Figure 3:</B> Gel imaging of two replicates of both pgRNA (pR1 and pR2) and pdCas9 (pC1 and pC2) isolated from transformed <I>E. coli</I> DH5α cells after digestion with the restriction enzyme BspHI. The samples were run alongside GeneRuler 1 kbp ladder (λ). </FIGCAPTION></FIGURE></CENTER></SPAN></DIV></LI><LI><DIV class="collapsible-header protocol-title">Insertion of anti-<EM>luxS</EM>  into pgRNA by Overhang Extension PCR</DIV><DIV class="collapsible-body"><SPAN><DIV class="row"><CENTER><FIGCAPTION style="position:bottom"><B>Figure 4:</B> Gel separation of PCR products after performing overhang extension PCR to insert the anti-luxS sequence into pgRNA (pR2). The samples were run alongside GeneRuler 1 kbp ladder (λ). Marked with a yellow dotted box is the fragment excised from the gel. </FIGCAPTION></CENTER></DIV><P>Overhang Extension PCR was performed to insert the 20 bp anti-<I>luxS</I> tracker sequence into the pgRNA. This sequence is complementary to a specific location near the TBS in the <I>luxS</I> gene, which is a gene involved in the production of the quorum sensing molecule Autoinducer-2. Primers with the anti-luxS overhang were designed using the software Benchling (see <A href="https://2018.igem.org/Team:NTNU_Trondheim/Design">Design</A> for more details about Overhang Extension PCR). The PCR products were run on an agarose gel to separate the linearized plasmid with the inserted of anti-<I>luxS</I> from the original circular pgRNA without anti-<I>luxS</I>. <B>Figure 4</B> shows the gel separation of the PCR products. The strong band at 2500 bp corresponds to the linearized plasmid. The circular configuration of the original pgRNA lacking the anti-<I>luxS</I>, makes it coil up and migrate further in the gel than the linearized plasmid due to less resistance. The original pgRNA can be seen at approximately 1500 bp.</P><P>The linearized plasmid at 2500 bp was excised from the gel, purified and nanodropped. The concentration was 52.8 ng/µL and the 260/280- and the 260/230-ratio was 1.90 and 0.27, respectively.  All though the DNA concentration and 260/280nm-ratio were acceptable, the low 260/230-ratio indicated that the recovered plasmids probably were contaminated. The contamination might be remnants of substances in the PCR-mix, agarose gel and/or chemicals used during the purification step.</P><P>The linearized pgRNA with the inserted anti-<I>luxS</I> sequence was transformed into <I>E. coli</I> DH5α cells. Prior to performing test digest, the OD<SUB>600</SUB> of the cell cultures were measured and the plasmids were miniprepped and the isolates were nanodropped. The results are listed in <B>Table 3</B>. Both replicates seem to have some impurities. Replicate 1 seems to have some RNA impurities, while replicate 2 seems to have contaminants that absorb at 230 nm. Still, both replicates were deemed good for further use.</P><TABLE style="width:100%"><CAPTION><B>Table 3:</B> OD<SUB>600</SUB> values, nucleic acid concentration, 260/280- and 260/230-ratios of two replicates of pgRNA (pR2) isolated from <I>E. coli</I> DH5α cells. </CAPTION><TBODY><TR><TH><B>Sample (replicate)</B></TH><TH><B>OD<SUB>600</SUB></B></TH><TH><B>Concentration [ng/µL]</B></TH><TH><B>260/280-ratio</B></TH><TH><B>260/230-ratio</B></TH></TR><TR><TD>pR2 (1)</TD><TD>0.95</TD><TD>37.2</TD><TD>2.06</TD><TD>4.29</TD></TR><TR><TD>pR2 (2)</TD><TD>1.00</TD><TD>123.2</TD><TD>1.77</TD><TD>1.01</TD></TR></TBODY></TABLE><P>The transformation was verified by test digest with PstI. PstI was chosen because it has a restriction cut site in the anti-<I>luxS</I> sequence. The band pattern obtained after gel electrophoresis corresponded with the expected lengths of the plasmid fragments of 490 bp and 2094 bp (see <B>Figure 5</B>). The transformation with pgRNA and insertion of the anti-<I>luxS</I> sequence was considered successful.</P><CENTER><FIGURE><FIGCAPTION><B>Figure 5:</B> (Left) The cut sites for PstI in pgRNA with the anti-luxS sequence, indicating the expected fragment lengths. (Right) Gel imaging of two replicates of pgRNA (pR2) isolated from DH5α E. coli cells after digestion with the restriction enzyme PstI. The samples were run alongside GeneRuler 1 kbp ladder (λ). </FIGCAPTION></FIGURE></CENTER></SPAN></DIV></LI><LI><DIV class="collapsible-header protocol-title">Double transformation of  <EM>E. coli </EM>   DH5α and TG1 cells</DIV><DIV class="collapsible-body"><SPAN><P>The pdCas9 and the pgRNA with anti-<I>luxS</I> were double transformed into competent <I>E. coli</I> DH5α and TG1 cells and plated on agar plates with both chloramphenicol and ampicillin. These cell strains were selected due to their different capacities to produce biofilm – DH5α being a poor biofilm producer, whereas TG1 is known to produce an abundancy of biofilm components if the conditions allow. Cultures of each of the transformed strains were miniprepped and the isolated plasmids were nanodropped and digested with PstI and BamHI-HF.</P><P>The NanoDrop results for the isolated plasmids from DH5α and TG1 are given in <B>Table 4</B>. For DH5α the 260/280-ratio was higher than 2.00 for both replicates, which may indicate that our isolates might have contained RNA impurities. The 260/230-ratio for the first replicate seemed acceptable, but the ratio for replicate 2 seemed oddly high. This gives reasons to believe that this sample was contaminated or was highly diluted. In addition, bobbles were detected when measuring this replicate. This might have interfered with the result giving a higher ratio than the actual value. The plasmids isolated from TG1 were all deemed good for further use, based on their near optimal ratios.</P><TABLE style="width:100%"><CAPTION><B>Table 4:</B> Nucleic acid concentration, 260/280- and 260/230-ratios of pgRNA with the anti-luxS (pR2L) and pdCas9 (pC2) isolated from double transformed E. coli DH5α and TG1 cells. Two replicates were prepared from separate DH5α cell cultures and five replicates were prepared from separate TG1 cell cultures.
  </CAPTION><TBODY><CAPTION><TR><TH><B>Sample (replicate)</B></TH><TH><B>Concentration [ng/µL]</B></TH><TH><B>260/280-ratio</B></TH><TH><B>260/230-ratio</B></TH></TR><TR><TD>DH5α, pR2L + pC2 (1)</TD><TD>219.9</TD><TD>2.01</TD><TD>1.82</TD></TR><TR><TD>DH5α, pR2L + pC2 (2)</TD><TD>35.3</TD><TD>2.03</TD><TD>3.86</TD></TR><TR><TD>TG1, pR2L + pC2 (1)</TD><TD>24.7</TD><TD>1.88</TD><TD>2.34</TD></TR><TR><TD>TG1, pR2L + pC2 (2)</TD><TD>23.0</TD><TD>1.87</TD><TD>2.18</TD></TR><TR><TD>TG1, pR2L + pC2 (3)</TD><TD>28.3</TD><TD>1.86</TD><TD>2.34</TD></TR><TR><TD>TG1, pR2L + pC2 (4)</TD><TD>29.4</TD><TD>1.76</TD><TD>1.34</TD></TR><TR><TD>TG1, pR2L + pC2 (5)</TD><TD>26.9</TD><TD>1.74</TD><TD>2.19</TD></TR></CAPTION></TBODY></TABLE><P>The fragments of the digested pgRNA with anti-luxS and pdCas9 were separated by gel electrophoresis. According to the digestion tool in Benchling, the expected fragment lengths were 6705bp, 2094bp, 465bp and 25bp when cutting both plasmids with PstI and BamHI-HF. As shown in <B>Figure 6</B>, bands were visible at approximately 7000, 2000 and 500 bp for both DH5α and TG1. We did not expect to see the 25 bp on the gel due to the fragment’s short length and consequently low dyeability. Since bands were visible at the expected lengths, we concluded that both pdCas9 and pgRNA with anti-luxS had successfully been transformed into both E. coli strains. </P><CENTER><FIGURE><FIGCAPTION><B>Figure 6:</B> Gel imaging of two replicates of PstI- and BamHI-digested pgRNA with anti-<I>luxS</I> (pR2L) and pdCas9 (pC2) isolated from double transformed (left) <I>E. coli</I> DH5α and (right) <I> E. coli</I> TG1 cells. The samples were run alongside GeneRuler 1 kbp ladder (λ).</FIGCAPTION></FIGURE></CENTER></SPAN></DIV></LI><LI><DIV class="collapsible-header protocol-title">Testing of biofilm promoting media</DIV><DIV class="collapsible-body"><SPAN><P>We had been informed by Kåre Bergh and Sven Even Borgos (see <A href="https://2018.igem.org/Team:NTNU_Trondheim/Human_Practices">Human Practices</A> for more information about these interviews) that the bacteria’s ability to form biofilms is highly dependent on their growth conditions, and that they are especially sensitive to glucose concentrations and pH levels. We wanted to test our CRISPRi system in a medium that promotes biofilm formation because it would make it easier to see a noticeable effect if the system works as expected.</P><P>In order to investigate which medium gives the highest degree of biofilm formation, we made nine variants of LB and M63B1 medium varying in pH values (4.5, 7.2 and 9.2) and glucose concentrations (0%, 0.4% and 0.8%). <I>E. coli</I> TG1 cells were inoculated in each medium and incubated. <B>Table 5</B> shows the measured OD<SUB>600</SUB> values for the overnight TG1 cultures. We observed a higher planktonic growth in LB than in M63B1. This is consistent with the fact that M63B1 is a minimal medium with a limited amount of nutrition, mimicking natural conditions. LB on the other hand, is a nutrition rich medium which provides good conditions for a rapid and numerous cell growth. Generally, the highest growth in both LB and M63B1 were observed in media with a pH value of 7.2 supplemented with glucose, either 0.4% and 0.8%, with only slight differences between the two.</P><TABLE style="width:100%"><CAPTION><B>Table 5:</B> Measured OD<SUB>600</SUB> values in overnight <I>E. coli</I> TG1 cell cultures grown in LB and M63B1 media with different pH values (4.5, 7.2 and 9.2) and supplemented with different glucose concentrations (0%, 0.4% and 0.8%). 
  </CAPTION><TBODY><CAPTION><TR><TH><B>pH, glucose</B></TH><TH><B>OD<SUB>600<SUB> LB</SUB></SUB></B></TH><TH><B>OD<SUB>600<SUB> M63B1</SUB></SUB></B></TH></TR><TR><TD>4.5, 0%</TD><TD>1.790</TD><TD>0.000</TD></TR><TR><TD>4.5, 0.4%</TD><TD>0.023</TD><TD>0.002</TD></TR><TR><TD>4.5, 0.8%</TD><TD>0.600</TD><TD>0.001</TD></TR><TR><TD>7.2, 0%</TD><TD>1.697</TD><TD>0.002</TD></TR><TR><TD>7.2, 0.4%</TD><TD>1.200</TD><TD>0.201</TD></TR><TR><TD>7.2, 0.8%</TD><TD>1.255</TD><TD>0.228</TD></TR><TR><TD>9.2, 0%</TD><TD>0.006</TD><TD>0.000</TD></TR><TR><TD>9.2, 0.4%</TD><TD>0.005</TD><TD>0.002</TD></TR><TR><TD>9.2, 0.8%</TD><TD>1.366</TD><TD>0.002</TD></TR></CAPTION></TBODY></TABLE><P>To measure the biofilm formation capabilities of the TG1 cells in the different media, we plated each culture on microtiter plates and incubated them for 24 hours. The amount of biofilm adhered to the walls of the wells were then measured by a crystal violet assay. The amount of produced biofilm in each medium is illustrated in <B>Figure 7</B>. Even though M63B1 gave a low degree of planktonic growth compared with LB (see Table 5), the M63B1 seemed to have a strong biofilm promoting effect in media with pH 7.2 and glucose concentrations of 0.4% and 0.8%. The biofilm formation in LB followed the same trend as the planktonic growth. We concluded that the best biofilm conditions were in M63B1 with a pH value of 7.2 and a glucose concentration of either 0.4% or 0.8%. For comparison purposes we also decided to continue doing measurements in LB medium having the equivalent pH value and glucose concentrations as the optimal biofilm promoting M63B1 media. All the other media were omitted.</P><CENTER><FIGURE><FIGCAPTION><B>Figure 7:</B> Testing of biofilm promoting media by crystal violet staining of biofilm adhered to the walls of the well of a microtiter plate. Prior to the assay, cultures of <I>E. coli</I> TG1 cells had been incubated in the wells at 37 °C for 24 hours in LB and M63B1 media differing in pH value (4.5, 7.2 and 9.2) and glucose concentration (0%, 0.4% and 0.8%). Eight replicates were made for each treatment. The absorbance was measured at 590 nm.</FIGCAPTION></FIGURE></CENTER></SPAN></DIV></LI><LI><DIV class="collapsible-header protocol-title"> Crystal violet assay for quantitative biofilm measurements</DIV><DIV class="collapsible-body"><SPAN><P>The next step was to test if we were able to reduce biofilm formation in bacteria using our CRISPRi system targeting the <I>luxS</I> gene in <I>E. coli</I>. We inoculated the chosen LB and M63B1 (pH 7.2 and glucose concentrations of 0.4% and 0.8%) with <I>E. coli</I> DH5α and TG1 cells transformed with pdCas9 and pgRNA with the anti-<I>luxS</I> sequence. After an overnight incubation, the cultures were diluted and plated on microtiter plates. Tetracycline was added to the media of our test groups, since it is the inducer for the transcription of dCas9. Our control groups lacked this addition.
While doing the crystal violet assay, we observed that there was some variation between replicates. We therefore decided to run it several times to refine the protocol to measure biofilm formation using crystal violet. In total we ran 4 assays. For each run we did some slight changes to the protocol. The following outlines the modifications made to the original protocol and the results we obtained for each run. For more information about the final experimental setup for our biofilm measurements, please visit <A href="https://2018.igem.org/Team:NTNU_Trondheim/Measurement">Measurement</A>.</P><P> As an extra feature to the biofilm assay, we did a cell viability test to check the condition of our cells when expressing the CRISPRi system to knock out the biofilm production. This test allowed us to confirm that our cells showed reduced biofilm production and were healthy at the same time. We used the <B> &quot;Cell Proliferation Kit II (XXT)&quot;</B> to investigate the health of the cells after 0h and 8h incubation with activated CRISPRi sytem. </P></SPAN></DIV></LI><LI><B>Crystal Violet assay after 24 hours of incubation</B><P>The procedure we followed during the first crystal violet assay was essentially the same as described in “Comparison of multiple methods for quantification of microbial biofilms grown in microtiter plates” [1]. This procedure was suggested to us by Sven Even Borgos from SINTEF. The protocol included a wash step after 4 hours of adhesion and a single absorbance measurement after 24 hours of incubation. The quantification of biofilm formation in the different media are shown in <B>Figure 8</B>. The biofilm formation seemed to be slightly reduced in M63B1 media for both <I>E. coli</I> strains. In the LB media, the biofilm formation of the strains did not seem to be affected by the CRISPRi system. This was seen by the relatively similar absorbance values between the test groups and the control groups. Surprisingly, the measured absorbance values were high for both strains in the LB media. This was not expected since TG1 theoretically should produce a higher amount of biofilm than DH5α cells. Overall, the standard deviations were within an interval that made it difficult to draw any clear conclusions from these results.</P><CENTER><FIGURE><FIGCAPTION><B>Figure 8:</B> Testing of biofilm-promoting media by crystal violet staining of biofilm adhered to the walls of the well of a microtiter plate. Prior to the assay, cultures of <I>E. coli</I> TG1 cells had been incubated in the wells at 37 °C for 24 hours in LB and M63B1 media differing in pH value (4.5, 7.2 and 9.2) and glucose concentration (0%, 0.4% and 0.8%). pH of 7.2 and glucose concentration of 0.4% and 0.8% showed the most biofilm formation and were selected as biofilm-promoting media <A href="https://2018.igem.org/Team:NTNU_Trondheim/Measurement"> (Test results)</A>. Eight replicates were made for each treatment. The absorbance was measured at 590 nm.</FIGCAPTION></FIGURE></CENTER></LI><LI><B>Crystal violet assay after 24, 48 and 72 hours of incubation time</B><P>For the second crystal violet assay we intended to include measurements after 48 and 72 hours of incubations. The results did not show any clear trend. After the 24 hours measurement it was observed that the control wells with only medium had too high absorbance values, which might indicate that the media were contaminated. Therefore, we decided to only do measurements after 48 hours of incubation time and not after 72 hours since this measurement probably would not give any valuable results.
During the incubation period we discovered that the media evaporated from the wells, leaving only a small volume left for the cells to continue their growth. The evaporation might lead to increased cell death. If so, these dead cell could precipitate and possibly stick to the deposited biofilm in the wells. Since crystal violet binds to all material attached to the walls of the wells, this potentially high cell death due to evaporation might have given higher absorbance values than expected. Therefore, it would be valuable to also do measurements of the cell viability in addition to the crystal violet assay.</P></LI><LI><B>Crystal violet assay after 3, 5, 8, 24 and 30 hours of incubation. Removing wash step after 4 hours of adhesion and sealing of microtiter plates with parafilm.</B><P>After the previous crystal violet assay, we decided to eliminate the wash step after 4 hours of adhesion to reduce the chance of getting the samples contaminated. To account for the observed evaporation of media from the wells, we sealed the microtiter plates with parafilm. Input from our model showing the development of biofilm formation over time made us also include measurements after 3, 5 and 8 hours of incubation (see <A href="https://2018.igem.org/Team:NTNU_Trondheim/Model">Model</A> for more information). In addition to this, we did measurements after 24 and 30 hours of incubation. The results after the crystal violet staining of biofilm can be seen in <B>Figure 9</B>.  TG1 cells containing the CRISPRi system incubated in M63B1 media seemed to have a significantly reduced biofilm formation compared with the control groups. The same was also true for the DH5a cells incubated in M63B1 media, but the reduction compared with the control groups were not as remarkable. For both strains incubated in LB media it was not observed any particularly lowered biofilm formation. The reason for this might be due to LB medium being a poor biofilm promoting medium in general.</P><CENTER><FIGURE><FIGCAPTION><B>Figure 9:</B> Quantitative measurements of the biofilm formation over time for <I>E. coli</I> TG1 and DH5α cells grown in either M63B1 or LB medium (pH 7.2, glucose concentration 0.4% or 0.8%) with an activated CRISPRi system (light and dark blue lines and points) and an inactivated CRISPRi system (yellow and red lines and points). Four replicates were made for each combination of cell type and medium. Crystal violet staining and absorbance measurements at 590 nm were conducted after 3, 5, 8, 24 and 30 hours of incubation. The standard deviations for each treatment are shown with error bars. The standard deviations for each treatment is shown with error bars.</FIGCAPTION></FIGURE></CENTER></LI><LI><B>Crystal violet assay conducted after 3, 5, 8, 24 and 30 hours of incubation (repetition of previous protocol)</B><P>Based on the somewhat promising results from the previous crystal violet assay we decided to repeat the same procedure to see if the results were repeatable. As illustrated in <B>Figure 10</B> the biofilm formation was lowered once again for the TG1 cells incubated in M63B1 media. For the other samples the biofilm formation seemed the be higher than what was observed previously. The variability between the replicates were also slightly higher than during the previous run, which might indicate that these measurements were poorer than before. Further refinements to our protocol for biofilm formation might be needed.</P><CENTER><FIGURE><FIGCAPTION><B>Figure 10:</B> Repetition of previous protocol. Quantitative measurements of the biofilm formation over time for <I>E. coli</I> TG1 and DH5α cells grown in either M63B1 or LB medium (pH 7.2; glucose concentration 0.4% or 0.8%) with an activated CRISPRi system (light and dark blue lines and points) and an inactivated CRISPRi system (yellow and red lines and points). Four replicates were made for each combination of cell type and medium. Crystal violet staining and absorbance measurements at 590 nm were conducted after 3, 5, 8, 24 and 30 hours of incubation. The standard deviations for each treatment are shown with error bars.</FIGCAPTION></FIGURE></CENTER></LI>

Source:
<UL><LI> [1] Comparison of multiple methods for quantification of microbial biofilms grown in microtiter plates.
J Microbiol Methods. 2008 Feb;72(2):157-65. Epub 2007 Nov 21. </LI><LI><DIV class="collapsible-header protocol-title">Our biobrick: Mutation of the dCas9 sequence and assembly into pSB1C3</DIV><DIV class="collapsible-body"><SPAN><P>The plasmid of dCas9 ordered from Addgene contained three illegal restriction sites according to the criteria for BioBrick RFC10. Therefore, it was necessary to remove these restriction sites via site directed mutagenesis (SDM) and Gibson assembly <U><B> without changing the amino acid sequence of dCas9 </B></U>. In addition, standard prefix and suffix sequences were also inserted into pdCas9 for ligation with iGEM backbone.
The original pdCas9 (bought from Addgene) contained two restriction sites recognized by EcoRI and one by XbaI, which do not fulfil the criteria for BioBrick RFC10. In order to remove all of the unwanted restriction sites, from the original pdCas9, specific primers were designed for introduction of SDM in pdCas9, and amplification of fragments of pdCas9 for Gibson assembly. First, one of the restriction sites, recognized by EcoRI, was removed by using PCR to introduce SDM in the plasmid. After the required PCR product was obtained and transformed into competent cells, the mutated pdCas9 was tested with restriction enzyme EcoRI for verification of a successful mutation in the plasmid. Afterwards, the fragments of the successful SDM pdCas9 was amplified via PCR with designed primers for removal of the last two restriction sites, recognized by EcoRI and XbaI, and insertion of standard prefix and suffix. The required fragments from PCR were isolated from gel electrophoresis and assembled together via Gibson assembly. The Gibson assembled pdCas9 was verified by using restriction enzymes. <B>Figure 11</B> shows the results after digestion with the enzymes EcoRI and XbaI, while <B>Figure 12</B> shows the fragments obtained after digestion with EcoRI and SpeI. The mutated pdCas9 with prefix and suffix, was assembled with pSB1C3. Correctly assembled plasmid with mutated dCas9 was verified via PCR colony screening <B>(Figure 13)</B> (for a more detailed description of the procedure, please read our lab journal – <A href="https://2018.igem.org/Team:NTNU_Trondheim/Experiments"> (Link)</A>.

</P><DIV><CENTER><B>Figure 11:</B> Gel electrophoresis of 1 kb ladder (λ), and three replicates of mutated pdCas9 with prefix and suffix (PC1, PC2, PC3) after digestion with EcoRI and XbaI.</CENTER></DIV><FIGURE>&quot;
<DIV><CENTER><B>Figure 12:</B> Gel electrophoresis of ladder (λ) and mutated pdCas9 (PC1) after digestion with EcoRI and SpeI.</CENTER></DIV></FIGURE><FIGURE><DIV><CENTER><B>Figure 13:</B> Gel electrophoresis of 1 kb ladder (λ), and the products from PCR colony screening for correct assemble of mutated dCas9 and pSB1C3 (G1.2).  </CENTER></DIV></FIGURE><B>Conclusion</B><P>If all of the three unwanted restriction sites in pdCas9 are removed, and the mutated plasmid are digested with the enzymes EcoRI and XbaI, the length of the predicted fragments should be 5744 bp, 989 bp and 15 bp, respectively. On the other hand, the length of the expected fragments of a successfully mutated pdCas9 digested with EcoRI and SpeI should be 5014 bp, 919 bp and 815 bp, respectively. By comparing the results obtained from gel electrophoresis of the mutated pdCas9 after digestion with the enzyme pairs, EcoRI and XbaI <B>(Figure 11)</B>, and EcoRI and SpeI <B>(Figure 12)</B>, with the fragments predicted by Benchling, the three unwanted restriction sites in pdCas9 were successfully removed and the standard prefix and suffix were inserted into the plasmid. Colonies that potentially carried pSB1C3 with mutated dCas9 were confirmed from PCR screening by obtaining a DNA fragment with a length approximately 5000 bp, which corresponds to the length of the mutated dCas9.
</P></SPAN></DIV></LI></UL><H2 class="project-color">Summary</H2><P>The results tell us that we successfully managed to isolate the pdCas9 and pgRNA, inserted the anti-<I>luxS</I> gene to the plasmids and transformed them into compotent <I>E. coli</I> DH5α cells. We have also tested the CRISPRi system, and the results show that the biofilm production from <I>E.coli</I> TG1 cells was greatly reduced. Furthermore, the unwanted restriction sites were successfully removed, and the mutated dCas9 was assembled into pSB1C3 plasmid backbone and the biobrick was submitted to the iGEM HQ.</P></UL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>