<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Duesseldorf_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Duesseldorf/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="sidebar"><UL><LI><A href="https://2018.igem.org/Team:Duesseldorf">Home</A></LI><UL class="sub"><A href="#!">Team</A><LI><A href="https://2018.igem.org/Team:Duesseldorf/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Ontour">On Tour</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Attributions">Attributions</A></LI></UL><UL class="sub"><A href="#!">Project</A><LI><A href="https://2018.igem.org/Team:Duesseldorf/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Improve">Improve</A></LI></UL><UL class="sub"><A href="#!">Parts</A><LI><A href="https://2018.igem.org/Team:Duesseldorf/Parts">Parts Overview</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Composite_Part">Composite Part</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Part_Collection">Part Collection</A></LI></UL><LI><A href="https://2018.igem.org/Team:Duesseldorf/Safety">Safety</A></LI><UL class="sub"><A href="#!">Human Practices</A><LI><A href="https://2018.igem.org/Team:Duesseldorf/Silver">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/HP/Gold_Integrated">Integrated Practices</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Newsletter">Newsletter</A></LI></UL><UL class="sub"><A href="#!">Awards</A><LI><A href="https://2018.igem.org/Team:Duesseldorf/Medals">Medal Criteria</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Plant">Plant Synthetic Biology</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Model">Model</A></LI></UL><LI><A href="https://igem.org/2018_Judging_Form?team=Duesseldorf">Judging Form</A></LI></UL><UL class="sub"><A href="#!">Advice</A><LI><A href="https://2018.igem.org/Team:Duesseldorf/MoClo">Modular Cloning</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/CyanoQandA">Cyano Q&amp;A</A></LI></UL></DIV><NAV><UL class="menu"><LI class="menu-item-has-children"><A href="#!">Team</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Ontour">On Tour</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Attributions">Attributions</A></LI></UL><LI class="menu-item-has-children"><A href="#!">Project</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Improve">Improve</A></LI><LI class="menu-item-has-children"><A href="#!">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Parts">Parts overview</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Composite_Part">Composite Part</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Part_Collection">Part Collection</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Safety">Safety</A></LI><LI class="menu-item-has-children"><A href="#!">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Silver">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/HP/Gold_Integrated">Integrated Practices</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Newsletter">Newsletter</A></LI><LI class="menu-item-has-children"><A href="#!">Awards</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Medals">Medal Criteria</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Plant">Plant Synthetic Biology</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Model">Model</A></LI><LI class="menu-item-has-children"><A href="#!">Advice</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/MoClo">Modular Cloning</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/CyanoQandA">Cyano Q&amp;A</A></LI><LI><A href="https://igem.org/2018_Judging_Form?team=Duesseldorf">Judging Form</A></LI></NAV><H1>Results</H1><P class="details">During the course of our entire project we were able to develop regulation mechanisms for single organisms, for two way co-cultures and for more complex systems of three organisms. The scientific results of our efforts are presented below.</P><BUTTON class="accordion"><H2>System 1: Self Regulating Microorganisms</H2></BUTTON><DIV class="panel"><P class="details">In this part of the project, the aim was to regulate cell growth of <I>Escherichia coli</I> cells. We wanted to accomplish this by expressing a lysis gene under the control of the quorum sensing promoter P<SUB><I>lux</I></SUB>. To test whether the cell growth was impaired by the presence of a lysis protein, different experiments were conducted. <NOBR><I>E. Coli</I></NOBR> BL21(DE3)C43 (<NOBR><I>E. Coli</I></NOBR> hereafter), cells harboring different plasmids - the lysis plasmid as well as different controls, were incubated overnight at 37 °C, 220 rpm. On the following day, the cells were diluted to a final OD<SUB>600</SUB> of 0.1. <NOBR>200 µl</NOBR> of these dilutions were loaded onto a 96-well plate for plate reader measurements in technical triplicates. In a final approach, cells harboring the <NOBR><I>luxI</I>-<I>luxR</I>-P<SUB><I>lux</I></SUB>-<I>phiX174E</I></NOBR> lysis construct were incubated in LB-medium until an OD<SUB>600</SUB> of 2 was reached. Culture supernatant was harvested, sterile filtered and then mixed 1:1 with fresh LB medium.</P><H4>Results</H4><P class="details">To check the functionality of the P<SUB><I>lux</I></SUB> promoter upon induction by the quorum sensing molecule acyl homoserine lactone (AHL) and the respective transcriptional activator protein LuxR, a test was performed, based on the expression of <I>gfp</I> under the control of the respective quorum sensing promoter (shown in Figure 1). It can be seen that fluorescence of GFP increases over time, while the untreated cells show small or no fluorescence at all. This observation leads to the conclusion that the promoter, P<SUB><I>lux</I></SUB> is induced upon synthesis of AHL by the LuxI synthase and subsequent binding to the LuxR regulator. Therefore, this promoter is characterized as functional for further use within our purposes, as well as characterized for use within the iGEM registry (<A href="http://parts.igem.org/Part:BBa_K2587027">BBa_K2587027</A>). 
In order to exclude that the lack of fluorescence of the progenitor cells is due to their scarce growth, a measurement of the optical density was done as well (Figure 1). As expected, the cells grow similarly therefore it can be concluded that the lack of fluorescence of the progenitoris not due to a smaller population size. </P><FIGURE><FIGCAPTION><STRONG>Figure 1: Ratio of fluorescence intensity and OD<SUB>600</SUB> under the control of the quorum sensing system (<I>luxI</I>-<I>luxR</I>-P<SUB><I>lux</I></SUB>-<I>gfp</I>).</STRONG> Samples were excited at a wavelength of 485 nm and emission was measured at 520 nm. Mean values of triplicates subtracted from the respective blank are plotted.</FIGCAPTION></FIGURE><P class="details">The performance of the P<SUB><I>lux</I></SUB> promoter was further confirmed by fluorescence microscopy (Figure 2). Here it can be seen that fluorescence is present in case of activation of the P<SUB><I>lux</I></SUB> promoter by the quorum sensing system (Figure 2A), while no fluorescence is present in the corresponding untreated cells (Figure 2B). As a control, GFP production under the control of a constitutive promoter <NOBR>was checked (Figure 2C)</NOBR>. When comparing the expression of <I>gfp</I> under the control of different promoters (P<SUB><I>lux</I></SUB> and constitutive promoter J23102) a higher expression could be visualized in the latter case.</P><FIGURE><FIGCAPTION><STRONG>Figure 2: Confocal fluorescence microscopy</STRONG>  of A: <NOBR><I>E. Coli</I></NOBR> cells with <I>luxI</I>-<I>luxR</I>-P<SUB><I>lux</I></SUB>-<I>gfp</I> construct; B: <NOBR><I>E. Coli</I></NOBR> cells with constitutive production of GFP and C: <NOBR><I>E. Coli</I></NOBR> BL21(DE3)C43 cells. </FIGCAPTION></FIGURE><P class="details">Next, the efficiency of the lysis plasmid containing the lysis gene <I>E</I> from bacteriophage phiX174 was tested (Figure 3). This was done by measuring the optical density of the cells containing the lysis plasmid (<A href="”http://parts.igem.org/Part:BBa_K2587024”">BBa_K2587024</A>). Progenitor <NOBR><NOBR><I>E. Coli</I></NOBR></NOBR> cells and cells harboring an empty vector control were included as negative controls. The latter was done to rule out a higher cell density of the progenitor strain than the other cells due to the lack of antibiotics in the culture medium. It is noticeable that growth of the cells is influenced by the presence of the lysis plasmid. This is especially observable during the stationary phase, starting after 10 hours, as the cells do not reach the same maximum density as the progenitor  cells or the cells harboring the uninduced plasmid (<I>luxR</I>-P<SUB><I>lux</I></SUB>-<I>gfp</I>).</P><FIGURE><FIGCAPTION><STRONG>Figure 3: Effect of lysis plasmid in <NOBR><I>E. Coli</I></NOBR> cells compared to progenitor <NOBR><I>E. Coli</I></NOBR> cells.</STRONG> As control, once the wild type strain (<NOBR><I>E. Coli</I></NOBR> BL21(DE3)C43) as well as another strain carrying the <I>luxR</I>-P<SUB><I>lux</I></SUB>-<I>gfp</I> construct was used. The different colors represent the <NOBR><I>E. Coli</I></NOBR> cells with the different plasmids and controls. Measurements were performed over a time period of 24 h. Mean values of triplicates subtracted from the respective blank are plotted. </FIGCAPTION></FIGURE><P class="details">Finally, to check whether additional induction with AHL would influence cell growth, we induced the cells harboring the lysis plasmid with 0.05 mM <NOBR>N-(3-oxohexanoyl)-l-homoserine</NOBR> lactone as compared to Scott and colleagues<A href="https://www.nature.com/articles/nmicrobiol201783"><SUP>1</SUP></A> and incubated them under similar conditions as the reference cells.
 As seen in Figure 4 as well, a slight decrease in optical density during the stationary phase and especially at the end of the stationary phase is observable, compared to the progenitore <NOBR><I>E. Coli</I></NOBR> cells as well as the uninduced cells with the lysis construct. However, with the amount of AHL added, a stronger effect on the cells was expected.
</P><FIGURE><FIGCAPTION><STRONG>Figure 4: Induction of E. coli cells with the lysis construct</STRONG> (<I>luxI</I>-<I>luxR</I>-P<SUB><I>lux</I></SUB>-<I>phiX174E</I>) with 0.05 mM N-(3-oxohexanoyl)l-homoserine lactone. As control the induced wild type cells as well as the uninduced cells with the lysis construct are shown. The different colors represent the <NOBR><I>E. Coli</I></NOBR> cells with the different plasmids and controls. Mean values of triplicates subtracted from the respective blank are plotted.</FIGCAPTION></FIGURE><P class="details">Also observable here is that the maximum growth of the <NOBR><I>E. Coli</I></NOBR> cells with the lysis plasmid compared to the progenitor cells is lower. <NOBR><I>E. Coli</I></NOBR> cells with the lysis construct reach a maximum OD<SUB>600</SUB> of 5.5, whilst the wild type cells reach an OD<SUB>600</SUB> of up to 8.1 after <NOBR>24 hours</NOBR>. This result further reinforces the impact of the lysis protein E from bacteriophage phiX174 on the <NOBR><NOBR><I>E. Coli</I></NOBR></NOBR> cells.
Finally, we wanted to know if the lysis protein was able to induce cell lysis in other cells (Figure 5). Here the experimental results have to be considered with care, since the presence of the antibiotic in the original medium might have influenced the growth of these cells and therefore the outcome.
</P><FIGURE><FIGCAPTION><STRONG>Figure 5: Impact on growth in medium containing the lysis protein.</STRONG> The colors represent the cells harbouring the different plasmids. Mean values of triplicates subtracted from the respective blank are plotted.</FIGCAPTION></FIGURE><H4>Conclusion</H4><P class="details">
The aim of this sub project was to construct a strain that grows slower than its progenitor counterpart. We consider this step to be essential in establishing the first step of our co-culture. Our priority here lies on us being able to make the co-culture populations achieve balance. We therefore applied the quorum sensing system, which is a reliable and well characterized tool of synthetic biology<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5603278/"><SUP>2</SUP></A> and wanted to accomplish this by choosing a lysis gene able to inhibit cell wall synthesis. First, the promoter P<SUB><I>lux</I></SUB> is characterized, showing efficient function by expression of <I>gfp</I> (Figure 1/2A). As compared to the constitutive expression of <I>gfp</I>, a lower fluorescence intensity is visible (Figure 2A/C). This might be due to the small amounts of AHL synthetized by LuxI, which might lead to the limited activation of the P<SUB><I>lux</I></SUB> or possibly different promoter strengths. In this case, lysis protein E is used which is able to interact with the host's SlyD (Sensitive to lysis D protein) and therefore enables protein E to be protected from proteolysis. This way protein E is able to interact with MraY translocase at the host membrane,blocking MraY, which is necessary for <NOBR>lipid I</NOBR> catalysis. The latter is an essential component for host cell wall synthesis<A href="https://www.ncbi.nlm.nih.gov/pubmed/11078734"><SUP>3</SUP></A>.
As suggested by our integrated human practice expert <I>Dr. Spencer Scott</I>, we allowed the cells to grow for only <NOBR>24 hours</NOBR>. We showed efficacy of our system by demonstrating that growth, based on OD<SUB>600</SUB> measurements, is decreased in cells harboring the lysis plasmid. Based on literature, lysis occurs because the lysis protein E impairs peptidoglycan synthesis<A href="https://www.uniprot.org/uniprot/P03639"><SUP>4</SUP></A>. This is consistent with our finding, since cell lysis results in less cells, shown by a lower OD<SUB>600</SUB>  measurement. Longer measurements resulted in lower OD<SUB>600</SUB> in the control strains as well, likely due to cell death from starvation, which can occur in older cultures. Moreover, as has been described in literature, lysis occurs only when the medium reaches a certain concentration threshold of the quorum sensing molecule AHL, probably during the stationary phase and induces a lysis event<A href="https://www.nature.com/articles/nmicrobiol201783"><SUP>1</SUP></A>. Also, similar to what Scott and colleagues discovered, population reduction is observable at around ten hours of growth<A href="https://www.nature.com/articles/nmicrobiol201783"><SUP>1</SUP></A>.
In Figure 4 we expected a much higher degree of lysis due to the addition of 0.5 mM AHL since Scott and coworkers achieved visual results with a concentration of 1 nM AHL<A href="https://www.nature.com/articles/nmicrobiol201783"><SUP>1</SUP></A>. Here, first improvements could be researched to find a method that achieves better results. With this discovery we have opened up new ways to control the growth of a population. In science it is a challenge to grow species with different characteristics together. Our approach therefore does not only benefit our co-culture system, but in the future might also benefit other branches of synthetic biology by opening up possibilities to control cell density.</P><H3>Outlook for the Quorum Sensing System</H3><P class="details">To optimize the system of the self lysing <NOBR><I>E. Coli</I></NOBR>, we assume that more controllable lysis can be achieved by modulating the strength of the promoters used for <I>LuxI</I> and <I>LuxR</I>. If more time had been available, we would have performed different experiments using varying concentrations of the quorum sensing molecule AHL. With this we would have found the concentration necessary to achieve the desired degree of lysis by selecting a suitable promoter for AHL synthase production. Finally we would have also tried implementing the same approach using other quorum sensing systems, such as the Rpa-system from <I>Rhodopseudomonas palustris</I> and the Las-system from <I>Pseudomonas aeruginosa</I>. Testing those systems would allow for comparison of lysis efficiency between different systems or combinations thereof.

Moreover, since <I>Saccharomyces cerevisiae</I> BY4742 (<NOBR><I>S. cerevisiae</I></NOBR> hereafter) is the second fastest growing organism in our co-culture, we also thought about a way to diminish its growth using a system native to <NOBR><I>S. cerevisiae</I></NOBR>.
We propose to regulate <NOBR><I>S. cerevisiae</I></NOBR> cell density by using the yeast mating type (MAT) pheromones. In a native context, these peptides serve as pheromones in yeast sexual differentiation. We use the beneficial side effect of the cell cycle arrest in the G<A href="https://www.nature.com/articles/nmicrobiol201783"><SUB>1</SUB></A> phase upon contact with the respective mating type. Yeast cells differ in their mating type: “α” type yeast cells produce the “α” pheromone but only respond to the “a” pheromone. Thus, a heterologous “α” producing “a” yeast strain should induce it’s own cell cycle arrest<A href="https://www.ncbi.nlm.nih.gov/pubmed/23656437"><SUP>5</SUP></A>. Our construct has a promoter which is pheromone inducible. The pheromone itself is encoded as a part of our designed plasmid. Activation of the promoter by the pheromone leads to the expression of a fluorescent gene<A href="https://pubs.acs.org/doi/abs/10.1021/sb500366v"><SUP>6</SUP></A>. This allows us to determine strength and functionality of the respective promoter. Using this approach we can distinguish which promoter is most suitable for our purposes. After successful assembly of our plasmids, we intensely monitor growth behavior using photometric methods. Since we want to establish a co-culture using different organisms, it is also important to check growth in different media. The growth behavior of the cells is then analyzed. In the end we aim to prepare an optimal medium composition, which allows optimal growth of the <NOBR>co-culture</NOBR>.</P><FIGURE><FIGCAPTION><STRONG>Figure 6: Regulation of the yeast cell density by the use of the yeast mating type (Mat) oligopeptides in <NOBR><I>S. cerevisiae</I></NOBR>.</STRONG> The yeast cells differ in their mating type: “α” type yeast cells produce the α pheromone but only respond to the “a” pheromone. Thus, a heterologous “a” producing “α” yeast strain should induce it’s own cell cycle arrest.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><STRONG>Figure 7: Construct with designed pMatα for S. cerevisiae.</STRONG> The construct with the ScMF(alpha)2 promoter and mVenus for following analysis. <I>mVenus</I> expression occurs only upon induction of this promoter by the respective pheromone.</FIGCAPTION></FIGURE><H3>Synthetic promoter in <NOBR><NOBR><I>S. cerevisiae</I></NOBR></NOBR></H3><P class="details">We also want to create and implement a novel synthetic promoter for yeast that responds to bacterial quorum sensing molecules by triggering cell lysis genes. Thus, bacteria and yeast could regulate their growth interspecific by using a quorum sensing device from a different class of organism. To regulate cell density in eukaryotes, the use of synthetically designed promoters to induce expression of a desired gene is necessary if possible.
Firstly, we will describe the layout of this promoter to start expression of <I>mTurquoise</I> as proof of concept. However, the aim is to replace the fluorescent protein with a lysis gene in the end in order to induce cell lysis here as well.
In eukaryotes, expression of genes can be accomplished by the construction of artificial transcription factors that are based on prokaryotic transcription factors<A href="https://www.frontiersin.org/articles/10.3389/fbioe.2017.00063/full"><SUP>7</SUP></A>. These engineered DNA binding proteins are able to bind to a specific sequence and start transcribing a target gene<A href="https://www.frontiersin.org/articles/10.3389/fbioe.2017.00063/full"><SUP>7</SUP></A>. We designed a synthetic promoter (a DNA component regulating gene expression)  for <NOBR><I>S. cerevisiae</I></NOBR> that is also based on the quorum sensing system. Therefore, we constructed an eukaryotic <I>trans</I>-activator consisting of the ligand binding domain and DNA binding domain of the quorum sensing transcription factor LuxR, fused to a yeast <I>trans</I>activation domain<A href="http://embor.embopress.org/content/4/2/159.long"><SUP>8</SUP></A>. We also included a nuclear localization sequence (NLS) fused to the Gal4 transcription factor, an eukaryotic positive regulator of gene expression. To show the functionality of the synthetic system, we included the reporter gene <I>mTurquoise</I> as proof of concept, which will be exchanged with a suitable lysis gene in the end.</P><FIGURE><FIGCAPTION><STRONG>Figure 8: Construct with designed synthetic promoter for <NOBR><I>S. cerevisiae</I></NOBR>.</STRONG> Based on the quorum sensing system concept this plasmid is supposed to induce the expression of <I>mTurquoise</I> upon activating the chimeric protein. For selection in <NOBR><I>S. cerevisiae</I></NOBR> the synthesis of leucine allows picking positive colonies on leucine auxotrophic medium. Since this system is based on the quorum sensing mechanism, LuxI will synthetize the AHL, which will bind to the LuxR regulator and finally to the LuxBox to induce synthesis of the fluorescent protein.</FIGCAPTION></FIGURE><P class="details">In addition to this, we tried to establish the same system with the more commonly used vectors pGBK und pGAD, which are mainly used in yeast two hybrid cloning. pGAD contains LuxR and the GAL-4 <I>trans</I>activation site to create the fusion protein<A href="https://www.sciencedirect.com/science/article/pii/S0968000400891197"><SUP>9</SUP></A>. We also added an NLS so the protein can be targeted to the nucleus for further interactions. The pGBK vector contains the LuxBox for recognition and the coding sequence of mTurquoise as proof of concept. Both constructs were generated by restriction ligation cloning<A href="http://embor.embopress.org/content/4/2/159.long"><SUP>8</SUP></A>.
</P><FIGURE><FIGCAPTION><STRONG> Figure 9: Complete constructs for the synthetic promoter.</STRONG> pGAD with LuxR and the GAL4-AD and pGBK with the LuxBox, a minimal promoter and <I>mTurquoise</I> for following analysis.</FIGCAPTION></FIGURE></DIV><BUTTON class="accordion"><H2>System 2: Two-way Co-culture based on Auxotrophies</H2></BUTTON><DIV class="panel"><H3>Experimental design</H3><P class="details">In our auxotrophy system, the aim was to establish an auxotrophic dependency of <NOBR><NOBR><I>S. cerevisiae</I></NOBR></NOBR> BY4742 (<NOBR><NOBR><I>S. cerevisiae</I></NOBR></NOBR> hereafter) on <NOBR><I>E. Coli</I></NOBR> BL21(DE3)C43, harboring <I>lysC</I> for lysine overproduction (<NOBR><NOBR><I>E. Coli</I></NOBR>_<I>lysC</I></NOBR> in the following). In order to analyze the growth of <NOBR><I>E. Coli</I></NOBR> and <NOBR><I>S. cerevisiae</I></NOBR> in mono- and co-culture, we performed a growth experiment over 36 hours. For all experiments, M2 medium supplemented with 1.5% glucose, was used. To demonstrate the functionality of our approach, <NOBR><I>E. Coli</I></NOBR> expressing a codon optimized and feedback resistant version of LysC under the control of a constitutively active promoter (<A href="http://parts.igem.org/Part:BBa_J23100">Bba_J23100</A>), was cultivated together with <NOBR><NOBR><I>S. cerevisiae</I></NOBR></NOBR> which is auxotrophic for uracil, histidine, leucine and lysine.
Both strains were cultivated in monocultures to observe their growth with these four additional amino acids as a control. In a second approach, both were cultivated in a co-culture without lysine. Furthermore the medium of the <NOBR><I>E. Coli</I></NOBR>_<I>lysC</I> strain was sterilized and reused to cultivate <NOBR><I>S. cerevisiae</I></NOBR>, to observe growth behavior in medium containing <NOBR><I>E. Coli</I></NOBR> produced lysine. In the final approach, three amino acids, but no lysine, were added.. For the experiments, <NOBR><I>E. Coli</I></NOBR> and <NOBR><NOBR><I>S. cerevisiae</I></NOBR></NOBR> were inoculated at a similar cell number to allow them to start out with similar conditions.</P><P class="details"><TABLE><CAPTION><STRONG>Table 1: Overview of the experimental design.</STRONG> Controls of both organisms in mono- and co-culture (Samples 1 - 4, 6) were observed to make the observations comparable. A co-culture was established to observe the behaviour of the organisms, including the dependency, without additional lysine (5).  A monoculture of <NOBR><NOBR><I>S. cerevisiae</I></NOBR></NOBR> in recycled M2 medium was used to check the lysine production by <NOBR><I>E. Coli</I></NOBR> (7).</CAPTION><THEAD><TR><TH>Number</TH><TH>Organims</TH><TH>Medium and conditions</TH><TH>Purpose</TH></TR></THEAD><TBODY><TR><TD>1</TD><TD><NOBR><I>E. Coli</I></NOBR> BL21 (DE3)C43</TD><TD>M2 with 1.5% glucose, uracil, histidine, leucine, lysine</TD><TD>Positive control / monoculture</TD></TR><TR><TD>2</TD><TD><NOBR><I>E. Coli</I></NOBR> BL21(DE3)C43 with lysine overproduction</TD><TD>M2 with 1.5% glucose, uracil, histidine, leucine</TD><TD>Proof of principle / monoculture</TD></TR><TR><TD>3</TD><TD><NOBR><I>S. cerevisiae</I></NOBR> BY4742</TD><TD>M2 with 1.5% glucose, uracil, histidine, leucine, lysine</TD><TD>Positive control / monoculture</TD></TR><TR><TD>4</TD><TD><NOBR><I>S. cerevisiae</I></NOBR> BY4742</TD><TD>M2 with 1.5% glucose, uracil, histidine, leucine</TD><TD>Negative control / monoculture</TD></TR><TR><TD>5</TD><TD><NOBR><I>E. Coli</I></NOBR> BL21(DE3)C43 with lysine overproduction +
<NOBR><I>S. cerevisiae</I></NOBR> BY4742</TD><TD>M2 with 1.5% glucose, uracil, histidine, leucine</TD><TD>Co-culture experiment</TD></TR><TR><TD>6</TD><TD><NOBR><I>E. Coli</I></NOBR> BL21(DE3)C43 + <NOBR><I>S. cerevisiae</I></NOBR> BY4742</TD><TD>M2 with 1.5% glucose, uracil, histidine, leucine, lysine</TD><TD>Negative control / Co-culture</TD></TR><TR><TD>7</TD><TD><NOBR><I>S. cerevisiae</I></NOBR> BY4742</TD><TD>Reused lysine enriched medium + 1.5% glucose, uracil, histidine, leucine</TD><TD>Proof of principle experiment</TD></TR></TBODY></TABLE></P><H4>Results</H4><P class="details">As seen in a preliminary experiment, the control samples of <NOBR><I>E. Coli</I></NOBR> grew at the same rate. For the following experiments it was decided to use the sample with number 2 (Table 1) as the positive control for the growth of <NOBR><I>E. Coli</I></NOBR> in monoculture. In the medium of sample 2, no additional lysine was added. It can also be shown that our transformed <NOBR><I>E. Coli</I></NOBR> is not inhibited by the overproduction of LysC. In the following, M2 media supplemented with lysine,leucine, histidine and uracil is referred to as M2 +lys. M2 media which is only supplemented with leucine, histidine and uracil is referred to M2 -lys hereafter. </P><FIGURE><FIGCAPTION><STRONG>Figure 1: Growth of the positive controls for <I>E. coli_lysC</I> and <NOBR><I>S. cerevisiae</I></NOBR> over 36 hours.</STRONG> Graph A shows the growth of both cultures at once while graph B only presents the growth of <NOBR><I>S. cerevisiae</I></NOBR>. The y-axis shows the total cell counts in 100 µl. The M2 medium contained for <NOBR><I>E. Coli</I></NOBR> additional uracil, histidine and leucine. In addition lysine was added for the positive control of <NOBR><I>S. cerevisiae</I></NOBR>. In each culture 1.5% glucose was the carbon source.</FIGCAPTION></FIGURE><P class="details">In Figure 1 the growth of the positive controls for both organisms in monoculture can be observed. The results show that <NOBR><I>E. Coli</I></NOBR>_<I>lysC</I> started growing at a much higher rate than <NOBR><NOBR><I>S. cerevisiae</I></NOBR></NOBR> (Figure 1A). Graph 1B shows only the growth of <NOBR><I>S. cerevisiae</I></NOBR> for better visualization. This indicates that the cells grew at first but died after around <NOBR>16 hours</NOBR> which isn’t visible in 1A. By comparing the growth of <NOBR><NOBR><I>E. Coli</I></NOBR>_<I>lysC</I></NOBR> (Figure 1A) and <NOBR><I>S. cerevisiae</I></NOBR> (in Figure 1B) the trend of the graphs show an analogous progress. In both cultures the cells grew to a certain point in time and decreased afterwards. After <NOBR>36 hours</NOBR> the cells in both cultures decreased in number.</P><FIGURE><FIGCAPTION><STRONG>Figure 2: Growth of the organisms in co-culture over 36 hours.</STRONG> Graph A shows the growth of <NOBR><I>E. Coli</I></NOBR>_<I>lysC</I> and <NOBR><I>S. cerevisiae</I></NOBR> as a co-culture in M2 with three additional amino acids. No lysine was added into the medium for part A. Graph B shows the growth of <NOBR><I>E. Coli</I></NOBR> and <NOBR><I>S. cerevisiae</I></NOBR> in M2 with additional four amino acids. Both cultures were placed under the same conditions containing 1.5% glucose. The y-axis shows the total amount of cells in 100 µl.</FIGCAPTION></FIGURE><P class="details">By comparing the co-cultures, it can be shown that in both cultures, <NOBR><I>E. Coli</I></NOBR> overgrew <NOBR><I>S. cerevisiae</I></NOBR>. This happened in the culture in +lys medium as well as in the culture with lysine producing <NOBR><NOBR><I>E. Coli</I></NOBR></NOBR> in the -lys medium. Our  results show that it was impossible to create a stable and working co-culture with only a lysine overproducing <NOBR><I>E. Coli</I></NOBR>. It can also be seen that the growth curves of <NOBR><I>E. Coli</I></NOBR> and <NOBR><I>E. Coli</I></NOBR>_<I>lysC</I> showed the same trend by comparing Figure 2A and 2B . First, the cells grew until the cell count started to decrease. Over time, the cells recovered and started growing again to another optimum but the density then decreased again until the end of the measurement. The results indicate that there was no set growth phase of <NOBR><I>S. cerevisiae</I></NOBR> in both cultures. Also, the positive control with four additional amino acids (Figure 2B) showed neither certain nor consistent growth phase of <NOBR><I>S. cerevisiae</I></NOBR> even though all four required amino acids were added to the media.</P><FIGURE><FIGCAPTION><STRONG>Figure 3: Growth of <NOBR><I>S. cerevisiae</I></NOBR> in reused M2 cultivation medium of <NOBR><I>E. Coli</I></NOBR>_<I>lysC</I>.</STRONG> Comparison of <NOBR><I>S. cerevisiae</I></NOBR> growth rates in the positive control with four additional amino acids and reused M2 cultivation medium of <NOBR><I>E. Coli</I></NOBR>_<I>lysC</I> over 36 h. In the culture with reused medium (lysC medium), no lysine was added. The y-axis shows the total number of cells in 100 µl.</FIGCAPTION></FIGURE><P class="details">Figure 3 shows the growth of <NOBR><I>S. cerevisiae</I></NOBR> monocultures. Here, growth of the positive control and the experiment with reused medium, described in the experimental design, are compared. The results show that it was possible to observe growth of <NOBR><I>S. cerevisiae</I></NOBR> in the medium which was lysine enriched by <NOBR><NOBR><I>E. Coli</I></NOBR>_<I>lysC</I></NOBR>. After around 4 hours, the cells in both cultures began to grow. After 12 hours, cell density reached a maximum in both cultures while the total density was higher in the case of the reused medium. After the highest density was reached, the cell density decreased faster in the medium experiment compared to the control. After 36 hours, the cells reached a similar final cell density. These results show that the lysine produced by <NOBR><I>E. Coli</I></NOBR>_<I>lysC</I>, is able to allow <NOBR><I>S. cerevisiae</I></NOBR> to grow as well as it would in lysine containing medium. This can also be seen as a proof of principle for synthetic dependencies based on amino acid auxotrophies in general.</P><H4>Conclusion</H4><P class="details">The experiments showed that we were not able to create a stable and working co-culture at this point. However, the experiment using the reused medium enriched with lysine by <NOBR><I>E. Coli</I></NOBR>_<I>lysC</I> showed that our approach with the auxotrophy system can work. Auxotrophy based dependencies between organisms are also used in common research. It would be interesting to improve our system by trying to cultivate the cultures under different conditions. It may make a difference if the cells in the co-cultures were inoculated with different cell densities or delayed addition of <NOBR><I>E. Coli</I></NOBR> cells after a specific time of pre-incubating <NOBR><I>S. cerevisiae</I></NOBR>. This could prevent <NOBR><I>E. Coli</I></NOBR> from overgrowing the slower growing organisms. One other approach would be the growth inhibition to the <NOBR><I>E. Coli</I></NOBR> cells using, for example, our self regulating system. 
But we were able to show that the lysine produced by <NOBR><I>E. Coli</I></NOBR>_<I>lysC</I> was sufficient for <NOBR><I>S. cerevisiae</I></NOBR> growth in the lysine enriched medium. Importantly, the cell density was higher in this experiment than in the positive control (Figure 3). At the end of the measurement the cell density reached a very low level in both cultures. It would be interesting to observe if longer pre-incubation of <NOBR><I>E. Coli</I></NOBR>_<I>lysC</I> in medium would lead to a higher growth rate or to a slower decrease in cell density of <NOBR><I>S. cerevisiae</I></NOBR>. Another possibility would be to let <NOBR><I>E. Coli</I></NOBR>_<I>lysC</I> produce a lysine exporter at the same time. This may cause further metabolic burden and could lead to a lower growth rate of the cells and possibly to a higher amount of lysine in the medium. 
The exact amount of lysine in the medium could be measured using HPLC analysis or even <I>in vivo</I> using biosensors. With this data it would be possible to compare the lysine production of <NOBR><I>E. Coli</I></NOBR>, <NOBR><I>E. Coli</I></NOBR>_<I>lysC</I> and further engineered <NOBR><NOBR><I>E. Coli</I></NOBR></NOBR> cells.</P><H3>Outlook for the Auxotrophy System</H3><P class="details">In our auxotrophy-based system <NOBR><I>S. cerevisiae</I></NOBR> BY4742<A href="https://onlinelibrary.wiley.com/doi/full/10.1002/%28SICI%291097-0061%2819980130%2914%3A2%3C115%3A%3AAID-YEA204%3E3.0.CO%3B2-2"><SUP>10</SUP></A> (<NOBR><I>S. cerevisiae</I></NOBR> hereafter) is auxotrophic for uracil, histidine, leucine and lysine. In order to create a dependency we created a lysine overproducing <NOBR><I>E. Coli</I></NOBR> (see <A href="https://2018.igem.org/Team:Duesseldorf/Demonstrate">Demonstrate page</A>).
In return, we planned to ectopically overexpress <I>LEU2</I> in a lysine auxotrophic strain of <NOBR><I>S. cerevisiae</I></NOBR><A href="https://onlinelibrary.wiley.com/doi/full/10.1002/%28SICI%291097-0061%2819980130%2914%3A2%3C115%3A%3AAID-YEA204%3E3.0.CO%3B2-2"><SUP>10</SUP></A>. This expression would complement the leucine production to make sure that every organism in our <NOBR>co-culture</NOBR> was provided with leucine. Lastly, a CRISPR-based knockout of <I>leuB</I> in <NOBR><I>E. Coli</I></NOBR> was planned. As a result, the production of leucine in <NOBR><I>E. Coli</I></NOBR> would be knocked out<A href="https://www.sciencedirect.com/science/article/pii/S0167483896001574"><SUP>11</SUP></A>. With this system, the auxotrophic dependencies would complement each other, creating a truly co-dependent co-culture.

In order to expand the auxotrophic system, <I>Synechococcus elongatus</I> PCC 7942 (<NOBR><I>S. elongatus</I></NOBR> in the following) which is included in our nutrient system, could be added. After discussing with Dr. Daniel Ducat, he gave us the following advice:<Q> The one that we played with is tryptophan auxotroph. [...] We have done some unpublished work with cyanobacteria that are tryptophan auxotroph [...].</Q><SUP>12</SUP>
The knockout in the tryptophan pathway would lead to a lack of tryptophan within <NOBR><I>S. elongatus</I></NOBR>. As an example the gene <I>trpA</I> could be a target for a knockout in this case. This gene codes for the tryptophan synthase alpha strain which is important for the formation of tryptophan. By knocking out this gene in <NOBR><NOBR><I>S. elongatus</I></NOBR></NOBR> it would be unable to synthesize tryptophan<A href="https://www.uniprot.org/uniprot/Q31R76"><SUP>13</SUP></A>.
Metabolic engineering of <NOBR><I>E. Coli</I></NOBR> could lead to a simultaneous overproduction of tryptophan and lysine. With this approach, the fastest growing organism would have to provide two substances at the same time, leading to a higher metabolic burden and to a possibly longer doubling time. 
A further possibility to improve our co-culture and the auxotrophy system could be to combine the different systems. Especially the auxotrophy and the nutrient system would likely be compatible with each other. The different dependencies of these systems would also be interesting to study beside each other. If the longer doubling time, which might be caused by the metabolic burden, would not occur, it would be the next step to also include the self regulating system. Since cell lysis would lead to intracellular nutrients and amino acids being released to the medium, it could be necessary to use genes which lead to a growth inhibition instead of cell lysis. To include the slowest growing organism in a better way it would also be interesting to inhibit the growth of the faster growing organisms. This could happen by adding the self regulating system or by reducing the metabolic burden of <NOBR><I>S. elongatus</I></NOBR>. By hypothesizing to combine our different systems, the toolbox character of our project is also set in place.</P></DIV><BUTTON class="accordion"><H2>System 3: Three-way Co-culture based on Nutrient Exchange</H2></BUTTON><DIV class="panel"><H3>Phosphite Measurement</H3><H4>Experimental Design</H4><P class="details">In our three-way co-culture, we want to use phosphite as a non-metabolizable phosphorus source. Only our engineered <NOBR><I>S. cerevisiae</I></NOBR> strain is able to convert it to phosphate for itself, as well as providing it to the other organisms.
To test if our construct with the codon optimized <I>ptxD</I> gene (<I>ptxD</I>_opt)<A href="https://www.sciencedirect.com/science/article/pii/S016816561400193X"><SUP>14</SUP></A> works, we performed a plate reader experiment over the course of 52 hours with different M2 media characteristics. <NOBR><I>S. cerevisiae</I></NOBR> and <NOBR><I>E. Coli</I></NOBR>, both used as negative controls, were cultivated in standard M2 medium<A href="https://www.sciencedirect.com/science/article/pii/S1096717617301763"><SUP>15</SUP></A> with 1.5% glucose, 0.5% ammonium sulfate, histidine (1.56 mg/l), leucine (380 mg/l), lysine (1.52 mg/l) and uracil (18 mg/l). For some experiments, M2 medium was modified to contain phosphite (also known as phosphonic acid) instead of the originally used phosphoric acid. The same supplements were used to ensure that the media only differed in the phosphorus source. Medium lacking uracil was used in the samples containing our construct in order to selection pressure. Five different constitutive promoters were tested. All samples were measured every 30 minutes in replicates of five; he sample size was 200 µl. At each measurement point, the OD<SUB>600</SUB> and temperature were measured. The experiment was performed at room temperature, while the plates were shaken vigorously. </P><FIGURE><FIGCAPTION><STRONG>Table 2: Loading scheme</STRONG> of the 96 well plate for the OD measurement of different cultures, different colors represent different media compositions.Yellow (11 A-E): M2 + phosphite + lysine + leucine + histidine + uracil + ammonium sulfate + glucose
Green (12 A-E): M2 + phosphite + lysine + leucine + histidine + ammonium sulfate + glucose
Orange/Red (H1-5): M2 + lysine + leucine + histidine + uracil + ammonium sulfate + glucose
Blue (H6-10): M2 + lysine + leucine + histidine + ammonium  sulfate + glucose
</FIGCAPTION></FIGURE><H4>Results</H4><P class="details">First it had to be tested whether other organisms in the co-culture were able to use phosphite as a phosphorus source. To test this, we compared growth of <NOBR><I>E. Coli</I></NOBR> and <NOBR><I>S. cerevisiae</I></NOBR> in normal M2 medium with M2 medium where phosphite is the only phosphorus source.</P><FIGURE><FIGCAPTION><STRONG>Figure 1: Growth of <NOBR><I>E. Coli</I></NOBR> in different media over 52 h in M2 medium</STRONG> with 1.5% glucose, 0.5% ammonium sulfate, histidine (1.56 mg/l), leucine (380 mg/l), lysine (1.52 mg/l) and uracil (18 mg/l) (blue) and in M2 medium with the same composition but with phosphite instead of phosphate (yellow). The cell density was measured at 600 nm.</FIGCAPTION></FIGURE><P class="details">As shown in Figure 1 in standard M2 medium <NOBR><I>E. Coli</I></NOBR> shows the common growth curve with a lag phase of 10 hours and a log phase over 10 hours. After 20 hours, <NOBR><I>E. Coli</I></NOBR> reaches stationary phase with an OD<SUB>600</SUB> of nearly 0.4. In M2 medium with phosphite, the bacteria stay in the lag phase and only reach an OD<SUB>600</SUB> of less than 0.1. At the end of the measurement, a slow decrease of the population is visible.</P><FIGURE><FIGCAPTION><STRONG>Figure 2: Growth of <NOBR><I>S. cerevisiae</I></NOBR> over 52 h in M2 medium</STRONG> with 1.5% glucose, 0.5% ammonium sulfate, histidine (1.56 mg/l), leucine (380 mg/l), lysine (1.52 mg/l) and uracil (18 mg/l) (blue) and in M2 medium with the same composition but containing phosphite instead of phosphate (yellow), measured at 600 nm.</FIGCAPTION></FIGURE><P class="details"><NOBR><I>S. cerevisiae</I></NOBR> shows the common growth curve similar to <NOBR><I>E. Coli</I></NOBR>, with a lag phase until 10 hours and a stationary phase after 20 hours (Figure 2). Cells incubated in M2 medium, supplemented with phosphite as the sole phosphorus source, no growth is detectable.

To figure out the strongest one, different promoters from the YTK toolbox<A href="https://pubs.acs.org/doi/abs/10.1021/sb500366v"><SUP>6</SUP></A> were tested: TDH3 (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K124002">BBa_K124002</A>), CCW12, PGK1 (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K122000">BBa_K122000</A>), HHF2, TEF1 controlling the <I>ptxD</I>_opt gene in <NOBR><I>S. cerevisiae</I></NOBR>. All previously described experiments were also performed using M2 medium with phosphite as phosphorus source. </P><FIGURE><FIGCAPTION><STRONG>Figure 3:  Growth of five <NOBR><I>S. cerevisiae</I></NOBR> strains with <I>ptxD</I>_opt under different promoter over 52 h in M2 medium</STRONG> with 1.5% glucose, 0.5% ammonium sulfate, histidine (1.56 mg/l), leucine (380 mg/l) and lysine (1.52 mg/l) with phosphite. Strain with promoter TDH3 (blue), CCW12 (red), PGK1 (green), HHF2 (gray), TEF1 (yellow), measured 600 nm.</FIGCAPTION></FIGURE><P class="details">All five promoter constructs show constant, but different growth in M2 medium with phosphite (Figure 3). With the strongest promoter TDH3, <NOBR><I>S. cerevisiae</I></NOBR> achieves an OD<SUB>600</SUB> of nearly 0.025, which is more than 100% more growth than the weakest promoter of the five: CCW12. The strains with the promoters PGK1 or TEF1 reach an OD<SUB>600</SUB> of a little bit over 0.015 and have medium strengths.
The final question is whether our modified <NOBR><I>S. cerevisiae</I></NOBR> strains shows better growth on phosphite than the progenitor <NOBR><I>S. cerevisiae</I></NOBR> strain. Therefore, the growth of the strain with <I>ptxD</I>_opt combined with the strongest promoter TDH3 were compared to the progenitor strain.</P><FIGURE><FIGCAPTION><STRONG>Figure 4: Growth of <NOBR><I>S. cerevisiae</I></NOBR> (blue) and <NOBR><I>S. cerevisiae</I></NOBR> with <I>ptxD_opt</I> and TDH3 (yellow) on M2 medium</STRONG> with 1.5% glucose, 0.5% ammonium sulfate, histidine (1.56 mg/l), leucine (380 mg/l) and lysine (1.52 mg/l) with phosphite over the course of 52 h, measured at 600 nm. For the <NOBR><I>S. cerevisiae</I></NOBR> BY4742 background strain, uracil (18 mg/l) was added.</FIGCAPTION></FIGURE><P class="details">In Figure 4 the graph demonstrates an initial growth of the progenitor strain <NOBR><I>S. cerevisiae</I></NOBR>, but the growth stops at an OD<SUB>600</SUB> of less than 0.03 after 10 hours. After 10 hours <NOBR><I>S. cerevisiae</I></NOBR> starts to decrease, in the beginning slowly, but in the end after over 50 hours a faster decrease is recognizable. At the end of the experiment the progenitor strain has an OD<SUB>600</SUB> of less than 0.02.
The modified <NOBR><I>S. cerevisiae</I></NOBR> strain with <I>ptxD</I>_opt and TDH3 starts nearly at the same OD<SUB>600</SUB> as the progenitor strain but after a lag phase of around 5 hours, the strain grows, slowly at the beginning and slightly faster towards the end, finally reaching an OD<SUB>600</SUB> of over 0.02.
Both strains start at the same OD<SUB>600</SUB> level. The non-modified strain initially grows faster than the modified strain, but then decreases more and more, while the modified strain needs some time but then shows a slowly increase in growth.</P><H4>Conclusion</H4><P class="details">Stable dependencies between organisms are often based on nutrient exchange. Phosphorus is a macro element essential for microorganisms. Creating a dependency based on the ability of <NOBR><I>S. cerevisiae</I></NOBR> to utilize an otherwise unusable phosphorus source like phosphite by expressing <I>ptxD</I> and thereby making it metabolically available for <NOBR><I>E. Coli</I></NOBR> is therefore a very promising approach.
In order to build a stable dependency, the other organisms in this system have to lack the enzymes for the catalysis from phosphite to phosphorus. The experiment with <NOBR><I>E. Coli</I></NOBR> (Figure 1) and <NOBR><I>S. cerevisiae</I></NOBR> (Figure 2) demonstrates that neither <NOBR><I>E. Coli</I></NOBR> nor <NOBR><I>S. cerevisiae</I></NOBR> show any growth in M2 medium with phosphite instead of phosphate. Figure 3 shows the growth of engineered <NOBR><I>S. cerevisiae</I></NOBR> with different promoter_<I>ptxD</I>_opt constructs. All show growth regardless of what promoter was used but the promoter TDH3 showed the strongest growth rates. This indicates that our approach can work.
When comparing the growth of both <NOBR><I>S. cerevisiae</I></NOBR> strains (Figure 4), some differences can be detected. The non-modified <NOBR><NOBR><I>S. cerevisiae</I></NOBR></NOBR> grows much faster in the beginning but begins to die soon after. The modified strain, however, shows slow, but constant growth. As a consequence, we assume that the modified strain would outcompete contaminating microorganisms and possibly the other members of the co-culture. But for that reason, the end goal is to implement multiple additional dependencies in the co-culture, such as carbon and nitrogen.
In general the growth was minimal, so for further experiments, higher concentrations of nutrients and phosphite might lead to better results. The slow growth rate of the modified strain is as we expected, because it is comparable to what the literature shows. There, a <I>ptxD</I> construct with the TEF1 promoter in <NOBR><I>S. cerevisiae</I></NOBR> was used and growth was monitored over 40 hours<A href="http://science.sciencemag.org/content/353/6299/583"><SUP>16</SUP></A>. A constant temperature of 30 °C also might increase growth. Moreover, a longer measurement time would also show the behavior of the culture over a longer time. This would be interesting because we would like to create a stable culture which can be maintained for as long as possible. In addition, co-culture experiments would likely lead to other results than monocultures. In this case it would be interesting to perform them as well with the same experimental design. For a future application in the co-culture, we suggest to use the phosphate exporter XPR1 from <I>Homo sapiens</I><A href="https://www.sciencedirect.com/science/article/pii/S2211124713002684"><SUP>17,</SUP></A><A href="”https://www.nature.com/articles/ng.3289”"><SUP>18</SUP></A>. It may help to feed the other organisms, due to the secretion of the produced phosphate. </P><H3>Biochemical analysis of Sucrose Content in <NOBR><I>S. elongatus</I></NOBR> PCC7942 <I>cscB</I>:::NS3  Samples</H3><P class="details">To detect the amount of sucrose production and secretion of our carbon source provider <NOBR><I>S. elongatus</I></NOBR> in the co-culture, we performed a biochemical assay which uses the YSI 2950 Biochemistry analyzer<A href="https://www.ysi.com/ysi-2950-biochemistry-analyzer"><SUP>19</SUP></A>. This method relies on the detection of sucrose mediated by a voltage change by immobilized enzymes on a specific membrane.
For this purpose we inoculated the <NOBR><I>S. elongatus</I></NOBR> PCC7942 <I>cscB</I>:::NS3 strain at an OD<SUB>750</SUB> of 0.3 and incubated it at standard conditions for two days. After two days, the culture was induced with 1 mM IPTG and 150 mM NaCl. The first samples were taken before induction, then after two days, four days and six days. As a control, wild type <NOBR><I>S. elongatus</I></NOBR> strain was treated in the same manner and samples were taken at the same time points, respectively.
To measure sucrose content, each sample was centrifuged at 4000 rpm for three minutes and the supernatant was sterile filtered for analysis.
A sucrose standard curve ranging from 1 mg/mL to 0.0625 mg/mL (1:2 dilutions each) was prepared
in order to determine the final sucrose concentration in the tested samples.
It can be seen that sucrose amounts rise after induction. The media of wild type <NOBR><I>S. elongatus</I></NOBR> before induction does not show any presence of sucrose, while a small amount of sucrose is detectable before the induction of <NOBR><I>S. elongatus</I></NOBR> PCC7942 <I>cscB</I>:::NS3.</P><FIGURE><FIGCAPTION><STRONG>Figure 5: Sucrose content in medium supernatant of <NOBR><I>S. elongatus</I></NOBR> and PCC 7942 <I>cscB</I>:::NS3 strain in days (d) post induction as well as pre-induction with 150 mM NaCl and 1 mM IPTG.</STRONG> Values are calculated from triplicates and measured based on a sucrose standard curve. (b. d. = below detection) </FIGCAPTION></FIGURE><H4>Conclusion</H4><P class="details">We could clearly demonstrate that upon induction with NaCl and IPTG the sucrose content is higher than in the uninduced control of the <NOBR><I>S. elongatus</I></NOBR> PCC 7942 <I>cscB</I>:::NS3 strain. Moreover, it could be shown that the wild type, as expected, does not show any presence of sucrose in the corresponding medium. This phenomenon can be explained by the heterologous sucrose transporter CscB in the <NOBR><I>S. elongatus</I></NOBR> PCC 7942 <I>cscB</I>:::NS3 strain<A href="https://academic.oup.com/pcp/article/57/12/2451/2528228"><SUP>20</SUP></A>. Upon induction, sucrose export is stimulated in the cells harboring this transporter. Wild type cells which lack the transporter cannot secrete sucrose in this manner. As a consequence, no sucrose is detectable in the medium. The amounts of sucrose secreted are comparable to literature values and even though the CscB transporter is inducible, low amounts of sucrose are exported when not induced by NaCl and IPTG, which is consistent with our data<A href="https://academic.oup.com/pcp/article/57/12/2451/2528228"><SUP>20</SUP></A>. </P><H3>Cultivation of <NOBR><I>E. Coli</I></NOBR> and <NOBR><I>S. cerevisiae</I></NOBR> in M2 Medium Enriched with Sucrose Secreted by <NOBR><I>S. elongatus</I></NOBR></H3><P class="details">After detecting the secretion of sucrose by <NOBR><I>S. elongatus</I></NOBR> sp PCC 7942 <I>cscB</I>:::NS3 via the YSI system, we wanted to test if the released amount of sucrose would be enough to sustain the growth of <NOBR><I>E. Coli</I></NOBR> and <NOBR><I>S. cerevisiae</I></NOBR> in a co-culture.
To avoid interactions between the species that could interfere with the growth of both microorganisms, only the cyanobacterial medium was used and transferred to the monocultures.</P><H4>Materials and Methods</H4><P class="details">During our experiments, 3 ml M2 medium of induced <I>S. elongatus_cscB</I> cultures were sampled two, four and six days after induction. The culture was centrifuged down, the supernatant was sterile filtered and used for new cultures of <NOBR><I>E. Coli</I></NOBR> and S. cerevisiae.  
As negative control, the medium of a wild type <NOBR><I>S. elongatus</I></NOBR>, which also included NaCl and IPTG normally used for the induction of the strain, was sampled and tested at a regular interval. 

Previous preliminary cultures of <NOBR><I>E. Coli</I></NOBR> or <NOBR><I>S. cerevisiae</I></NOBR> were centrifuged and washed with M2 medium to rid them of any additional sugar sources. After that the cultures were added to the sterile filtered medium from <NOBR><I>S. elongatus</I></NOBR> and adjusted to an OD<SUB>600</SUB> of 0.1. Afterwards a 96 well plate well was filled with 200 µl per well of the two cultures in technical triplicates.

The growth of <NOBR><I>E. Coli</I></NOBR> and <NOBR><I>S. cerevisiae</I></NOBR> cultures in the cyano medium was measured in a plate reader that incubated the cultures at 30 °C and agitated them 999 seconds between each measurement. The measurements of the optical density (OD) at 600 nm were taken every 30 min over a time frame of 24 hours.</P><H4>Results</H4><FIGURE><FIGCAPTION><STRONG>Figure 6: Regulation of the yeast cell density by the use of the yeast mating type (Mat) oligopeptides in <NOBR><I>S. cerevisiae</I></NOBR>.</STRONG> The yeast cells differ in their mating type: “α” type yeast cells produce the α pheromone but only respond to the “a” pheromone. Thus, a heterologous “a” producing “α” yeast strain should induce its own cell cycle arrest.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><STRONG>Figure 7: Construct with designed P<SUB>Matα</SUB> for <NOBR><I>S. cerevisiae</I></NOBR>. </STRONG> Schematic representation of the plasmid construct with the ScMF(α)2 promoter controlling mVenus. <I>mVenus</I> expression occurs only upon induction of this promoter by the respective pheromone.</FIGCAPTION></FIGURE><P class="details">During the first measurement two days after the induction of <NOBR><I>S. elongatus</I></NOBR>, the OD<SUB>600</SUB> of <NOBR><I>S. cerevisiae</I></NOBR> in the medium of the wild type cyanobacteria is constantly decreasing (Figure 6). While, initially, this happened in the <I>cscB</I> mutant medium as well, the OD<SUB>600</SUB> rose after around 18 hours. The OD<SUB>600</SUB> of <NOBR><I>E. Coli</I></NOBR> in both media obtained from the wt and <I>cscB</I> mutant is constantly decreasing (Figure 7).</P><H4>Discussion</H4><P class="details">Our results show that the engineered <NOBR><I>S. elongatus</I></NOBR> sp PCC 7942 <I>cscB</I>:::NS3 strain is able to support the growth of <NOBR><I>S. cerevisiae</I></NOBR>. While the negative controls in the WT media show a decreasing OD<SUB>600</SUB>- hence die off during the experiment, the OD<SUB>600</SUB> increases in all <I>cscB</I> mutant medium samples after 20 hours, regardless of whether <NOBR><I>S. elongatus</I></NOBR> was induced two, four or six days ago. We assume that the delay in growth is due to the change of culture medium.
However, <NOBR><I>E. Coli</I></NOBR> dies in the medium two days after induction and only shows a slightly elevated OD<SUB>600</SUB> in the end of the second and third measurement. We suspect that <NOBR><I>E. Coli</I></NOBR> BL21(DE3)C43 is not able to convert sucrose into a metabolizable carbon source<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3553775/"><SUP>21</SUP></A>. We assume that <NOBR><I>E. Coli</I></NOBR>s carbon source in the three-way co-culture could be obtained from <NOBR><I>S. cerevisiae</I></NOBR>. It is able to convert sucrose into glucose and fructose extracellularly with secreted invertases<A href="https://academic.oup.com/femsyr/article/16/1/fov107/2467713"><SUP>22</SUP></A>.</P></DIV><BUTTON class="accordion"><H2>Quantification of our Co-Culture Experiments</H2></BUTTON><DIV class="panel"><P class="details">Standard cultivation methods are usually optimized for monocultures and most quantification methods are also designed for this circumstance. Whether using optical density (OD) or dry cell mass measurements, many of these methods only provide reliable information about the cell quantity in monocultures. But as soon as we want to take advantage of a co-culture, we have to choose another quantitative method determination which takes into account the heterologous composition thereof.
One method is to use fluorescent measurements by introducing genes encoding fluorescent proteins into the organism beforehand. Because fluorescence can be measured quickly and automatically with suitable instruments such as plate readers, it is quite convenient. Thus, it is possible to recognize quickly whether a culture grows or dies under certain conditions. Since OD measurements are still the method of choice today, it is of interest to find ways to make OD and fluorescence comparable under certain conditions. In this part of our project, we tried exactly that.</P><P class="details">As <NOBR><I>S. elongatus</I></NOBR> already has an endogenous fluorescence due to its chlorophyll pigments, we decided to provide the other organisms with fluorescence proteins as well. This enabled us to measure different fluorescence emissions simultaneously in a plate reader and thus to determine the cell density of our organisms in a co-culture. 
At first, calibration curves had to be established. For this purpose, we cultivated the organisms as as monocultures in the co-culture medium M2 or SD. Subsequently, we loaded the plates for the plate reader in triplicates with the respective preliminary cultures.

In the <NOBR><I>S. elongatus</I></NOBR> experiment, we measured every 12 hours for 3.5 days in total. Between the measuring points, the cultures were incubated at 30 °C and 200 rpm in the light cabinet with 75 µmol m<SUP>-2</SUP>s<SUP>-1</SUP> intensity. As can be seen in Figure 8, both the fluorescence intensity and the OD<SUB>750</SUB> increase, resulting in the calibration standard shown. The OD<SUB>750</SUB> can then be calculated from the fluorescence intensity and used for our co-culture measurements.
We decided that the fluorescence is useful for measuring cell density at low concentrations but we do not plan on using it for higher cell densities since the light may be a limiting factor for chlorophyll production. The amount of chlorophyll in a cell can vary depending on various factors and is therefore not necessarily proportional to the number of cells. Because of that, we used the fluorescence intensity just for a qualitative statement if the cell density raises or lowers and not for an exact determination of cell density.</P><FIGURE><FIGCAPTION><STRONG>Figure 8: Calibration of optical density and fluorescence for <NOBR><I>S. elongatus</I></NOBR> sp PCC 7942 <I>cscB</I>:::NS3, M2 and BG11 medium.</STRONG> The pre-cultures were adjusted to an OD<SUB>750</SUB> of 0.1. <I>S elongatus</I> was measured for 3.5 days. Autofluorescence was measured at 680 nm after being excited at 620 nm.</FIGCAPTION></FIGURE><P class="details">For the measurement of <NOBR><I>E. Coli</I></NOBR>, we chose the fluorescent protein mVenus (excitation: 515 nm, emission: 527 nm)<A href="https://www.fpbase.org/protein/mvenus/"><SUP>23</SUP></A>. As in the <NOBR><I>S. elongatus</I></NOBR> experiment, we also loaded a plate for the plate reader to test the optical density and the fluorescence intensity. We also used dilutions to see which cell densities represent the best measurable data. This time we measured every two hours and incubated the cultures directly in the plate reader. The cells were incubated at 37 °C and 200 rpm. Shaking was interrupted during measurement steps. As seen in Figure 2 the OD and the fluorescence correlate in a constant way and the calibration curve can be used in further experiments.
</P><FIGURE><FIGCAPTION><STRONG>Figure 9: Calibration of optical density and fluorescence for <NOBR><I>E. Coli</I></NOBR> BL21(DE3)C43 in LB, M2 and BG11 medium.</STRONG> The green graph shows the growth of <NOBR><I>E. Coli</I></NOBR> in LB medium, blue in minimal media M2 and BG11 (typical culture medium for cyanobacteria) in yellow. The pre-cultures were adjusted to 0.1 OD and diluted 1:2 with in their medium. <NOBR><I>E. Coli</I></NOBR> growth was monitored for 24 hours.</FIGCAPTION></FIGURE><P class="details">As shown in Figure 9, a linear correlation from up to OD<SUB>600</SUB> 0.2 is recognizable for the growth in all three media. To get a closer look on this range, Figure 10 shows only the linear section of the three graphs.</P><FIGURE><FIGCAPTION><STRONG> Figure 10: Range of linear correlation of optical density and fluorescence for <NOBR><I>E. Coli</I></NOBR> BL21(DE3)C43 in LB, M2 and BG11 medium.</STRONG> The green graph shows the growth of <NOBR><I>E. Coli</I></NOBR> in lysogeny broth (LB) medium and can be described with y = 253.12x + 275.08, blue in minimal media M2 characterized by y = 296.21x + 200.15 and BG11 (typical culture medium for cyanobacteria) in yellow with feature y = 351.82x + 160.3. The pre-cultures were adjusted to 0.1 OD and diluted 1:2 in their respective medium. <NOBR><I>E. Coli</I></NOBR> was measured for 24 hours.</FIGCAPTION></FIGURE><P class="details">Under the given circumstances here and up to OD 0.2, it is possible to compare OD<SUB>600</SUB> to mVenus fluorescence in <NOBR><NOBR><I>E. Coli</I></NOBR> BL21(DE3)C43</NOBR>.

Lastly, the calibration curve for <NOBR><I>S. cerevisiae</I></NOBR> needed to be done. For this, we used the fluorescent protein mTurquoise (excitation: 424 nm, emission: 474 nm)<A href="https://www.fpbase.org/protein/mturquoise/"><SUP>24</SUP></A>. As the name and its Ex/Em values imply, this protein emits light with a bluish tone so we assume that none of the emission spectra will interact with or influence the other fluorescence measurements in our co-culture. In the experimental design of this experiment, we used the same technique as we used in the previous <NOBR><I>E. Coli</I></NOBR> experiment. We simply changed the temperature to 30 °C and used SD dropout medium to preserve the selection pressure for the yeast in order to force it to keep the fluorescence gene. As shown in Figure 11, the fluorescence intensity decreases constantly instead of increasing. This is likely due to the fact that mTurquoise is only expressed in the exponential growth phase, which is why the intensity decreases afterwards. We decided not to use this calibration curve for further quantification.</P><FIGURE><FIGCAPTION><STRONG> Figure 11: Measurement of fluorescence intensity in RFU compared to OD<SUB>600</SUB> of <NOBR><I>S. cerevisiae</I></NOBR>.</STRONG> The created calibration curve decreases because of the declining fluorescence intensity while the OD<SUB>600</SUB> is still increasing. The fluorescence intensity was measured at 474 nm after excitation at 424 nm.</FIGCAPTION></FIGURE><P class="details">The calibration curve experiments gave us a good overview of the application of fluorescence for cell density determination, but also showed us its limits. Fluorescence is not the most reliable or most accurate quantification method due to the fluctuating chlorophyll content in cyanobacteria, as well as the different folding and expression times for fluorescent proteins. After these experiments we decided to use the autofluorescence of <NOBR><I>S. elongatus</I></NOBR> in low cell density areas because of good correlations observed in our data. The other fluorescent measurements could have been useful but require a lot more research and fine tuning for our purposes. To get more reliable results we also plated and counted our co-culture cells.</P><H3>Three-way Co-culture Growth Curves</H3><P class="details">Our experiments regarding our three-way co-culture include <NOBR><I>S. elongatus</I></NOBR> sp PCC 7942 <I>cscB</I>:::NS3 (<NOBR><NOBR><I>S. elongatus</I></NOBR></NOBR> hereafter) which has been previously engineered to secrete sucrose. Next to this, the strains <NOBR><I>S. cerevisiae</I></NOBR> BY4742 (<NOBR><I>S. cerevisiae</I></NOBR> hereafter) and <NOBR><I>E. Coli</I></NOBR> BL21(DE3)C43 (<NOBR><I>E. Coli</I></NOBR> hereafter) have been tested as well.
In order to provide the same environmental conditions for each culture, the cultures were incubated at 30 °C 200 rpm in 300 ml flasks with 37.5 ml in a lighted incubator.<NOBR><I>S. elongatus</I></NOBR> preliminary cultures were incubated to an OD<SUB>750</SUB> of 0.5 and induced with up to 1 mM IPTG and up to 150 mM NaCl in final culture two days before the experiment.
Right before the measurement preliminary cultures of <NOBR><I>E. Coli</I></NOBR> and yeast were centrifuged and washed with M2 to get rid of the traces of the full medium. Then, they were either added to the previously induced <NOBR><I>S. elongatus</I></NOBR> monocultures or inoculated in M2 medium with additional sucrose and amino acids. <NOBR><I>S. cerevisiae</I></NOBR> cultures was adjusted to an OD<SUB>600</SUB> of 0.4, whereas <NOBR><I>E. Coli</I></NOBR> was adjusted to an OD<SUB>600</SUB> of 0.1.
In order to compare the interactions with either all or one specific strain, different culture combinations were incubated. 
Monocultures of each strain were necessary to compare to the growth in co-cultures.
In addition to that <NOBR><I>E. Coli</I></NOBR> and <NOBR><I>S. cerevisiae</I></NOBR> were co-cultured, as well as a three-way co-culture of <NOBR><I>E. Coli</I></NOBR>, <NOBR><I>S. cerevisiae</I></NOBR> and <NOBR><I>S. elongatus</I></NOBR> inoculated. 
The measurement intervals took place every 4 hours over an entire time frame of 48 hours.
During each measurement, the (co-)cultures containing <NOBR><I>E. Coli</I></NOBR> were diluted 1:100000 and plated onto LB plates, <NOBR><I>S. cerevisiae</I></NOBR> cultures were 1:1000 diluted and streaked on YPD plates. Since it would normally take a few weeks for <NOBR><NOBR><I>S. elongatus</I></NOBR></NOBR> to grow colonies on a plate under ambient conditions, the amount of cyanobacteria in the cultures was quantified using fluorescence measurements in a plate reader instead. These measurements were later compared to a calibration curve.</P><H2>Results</H2><FIGURE><FIGCAPTION><STRONG>Figure 12: Growth curve of <NOBR><I>S. cerevisiae</I></NOBR> in different culture conditions.</STRONG><NOBR><I>S. cerevisiae</I></NOBR> was cultured in M2 with 1.5% sucrose, the amino acids histidine, leucine and uracil as well as ammonium sulfate (5 g/l) (dark blue), <NOBR><I>S. cerevisiae</I></NOBR> and <NOBR><I>E. Coli</I></NOBR> were co-cultured in medium with lysine (orange) or without lysine (blue) in M2 with 1.5% sucrose, the amino acids histidine, leucine and uracil as well as ammonium sulfate (5 g/l). <NOBR><I>S. cerevisiae</I></NOBR> was co-cultured with <I>S. elongatus cscB</I>:::NS3 in M2 with lysine (gray). <NOBR><I>S. cerevisiae</I></NOBR> was co-cultured with <NOBR><I>E. Coli</I></NOBR> and <I>S. elongatus cscB</I>:::NS3 in M2 with the amino acids histidine, leucine and uracil as well as ammonium sulfate (5 g/l) with lysine (yellow) and without lysine (green). The measurement was conducted over 48 hours and every 4 hours 100 µL of the cells were plated on YPD plates with a dilution of 1:1000. <NOBR><I>S. cerevisiae</I></NOBR> was measured in colony forming units (CFU).</FIGCAPTION></FIGURE><P class="details">The graph shown in Figure 12 indicates that <NOBR><I>S. cerevisiae</I></NOBR> does grow in sucrose enriched media (1.5 %). The growth is slightly reduced when co-cultured with <NOBR><I>E. Coli</I></NOBR>, but still increases over time. The growth of <NOBR><I>S. cerevisiae</I></NOBR> in co-culture with <NOBR><I>S. elongatus</I></NOBR> without any additional sucrose in the medium decreases.</P><FIGURE><FIGCAPTION><STRONG>Figure 13: Growth curve of <NOBR><I>E. Coli</I></NOBR> in different culture conditions.</STRONG><NOBR><I>E. Coli</I></NOBR> was mono-cultured in M2 with 1.5% sucrose, the amino acids histidine, leucine and uracil (dark blue),<NOBR><I>E. Coli</I></NOBR> was co-cultured together with <NOBR><I>S. cerevisiae</I></NOBR> with lysine (orange) and without lysine (yellow) in M2 with 1.5% sucrose, the amino acids histidine, leucine and uracil. <NOBR><I>E. Coli</I></NOBR> was co-cultured together with <NOBR><I>S. cerevisiae</I></NOBR> and <I>S. elongatus cscB</I>:::NS3 in M2 with the amino acids histidine, leucine and uracil with lysine (gray) and without lysine (light blue). The measurement was conducted over 36 hours and every 4 hours the cells were spread out on LB plates with a dilution of 1:100000. <NOBR><I>E. Coli</I></NOBR> was quantified in colony forming units (CFU).</FIGCAPTION></FIGURE><P class="details">The total amount of <NOBR><I>E. Coli</I></NOBR> cells which grew in co-culture with <NOBR><I>S. cerevisiae</I></NOBR> in sucrose enriched M2 media is greater than that in monoculture as shown in Figure 13. Both curves are decreasing, whereas the monoculture curve remains stationary. Similar to the growth of the monoculture is the growth of the <NOBR><I>E. Coli</I></NOBR> cells in co-culture with <NOBR><I>S. cerevisiae</I></NOBR> and <NOBR><I>S. elongatus</I></NOBR>.</P><FIGURE><FIGCAPTION><STRONG>Figure 14: Fluorescence measurement of <NOBR><I>S. elongatus</I></NOBR><I>cscB</I>:::NS3 in different culture conditions over time.</STRONG><NOBR><I>S. elongatus</I></NOBR> was monocultured in M2 (dark blue). <NOBR><I>S. elongatus</I></NOBR> was co-cultured together with <NOBR><I>S. cerevisiae</I></NOBR> with lysine (green) in M2 with the amino acids histidine, leucine, lysine and uracil as well as ammonium sulfate (5 g/l). <NOBR><I>S. elongatus</I></NOBR> was co-cultured together with <NOBR><I>S. cerevisiae</I></NOBR> and <NOBR><I>E. Coli</I></NOBR> in M2 with the amino acids histidine, leucine, lysine and uracil as well as ammonium sulfate (5 g/l) with additional lysine (gray) and without lysine (yellow). The measurements were conducted over 48 hours and every 4 hours the fluorescence was measured at 680 nm with the organisms being excited at 620 nm. The value was plotted as Relative fluorescence units (RFU).</FIGCAPTION></FIGURE><P class="details">All <NOBR><I>S. elongatus</I></NOBR> cultures are constantly increasing regardless of the cultivation method (as shown in Figure 14), whether it is in a monoculture or co-culture.
In a two-organism co-culture the growth of <NOBR><I>S. cerevisiae</I></NOBR> is increasing, the growth of <NOBR><I>E. Coli</I></NOBR> is decreasing.</P><FIGURE><FIGCAPTION><STRONG>Figure 15: CFU of a co-culture of <NOBR><I>E. Coli</I></NOBR> (yellow) and <NOBR><I>S. cerevisiae</I></NOBR> (blue).</STRONG> Lysine was added to the M2 medium in addition to 1.5% sucrose, the amino acids histidine, leucine and uracil as well as ammonium sulfate. The measurement were taken over a time frame of 48 hours (36h for <NOBR><I>E. Coli</I></NOBR>) and every 4 h the cells were spreaded out in 100µl on LB plates with a dilution of 1:100000 for <NOBR><I>E. Coli</I></NOBR> or on YPD plates with a dilution of 1:1000 for on YPD plates with a dilution of 1:1000. <NOBR><I>E. Coli</I></NOBR> and <NOBR><I>S. cerevisiae</I></NOBR> were measured in colony forming unit (CFU).</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><STRONG>Figure 16: CFU of a co-culture of <NOBR><I>E. Coli</I></NOBR> (orange) and <NOBR><I>S. cerevisiae</I></NOBR> in </STRONG> M2 medium with 1.5% sucrose, the amino acids histidine, leucine and uracil as well as ammonium sulfate (5 g/l) (blue). The measurement was conducted over a time period of 48 hours (36 h for <NOBR><I>E. Coli</I></NOBR>) and every 4 h the cells were spreaded out in 100 µl on LB plates with a dilution of 1:100000 for <NOBR><I>E. Coli</I></NOBR> or on YPD plates with a dilution of 1:1000. <NOBR><I>E. Coli</I></NOBR> and <NOBR><I>S. cerevisiae</I></NOBR> were measured in colony forming unit (CFU).</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><STRONG>Figure 17: Co-culture growth curve of <NOBR><I>S. cerevisiae</I></NOBR> (blue) and <NOBR><I>S. elongatus</I></NOBR> (orange).</STRONG> While the growth of <NOBR><I>S. cerevisiae</I></NOBR> was measured in colony forming unit (CFU), <NOBR><I>S. elongatus</I></NOBR> was measured in RFU. Fluorescence was measured at 680 nm with it being excited at 620 nm. The measurement was conducted over a time period of 48 hours. </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><STRONG> Figure 18: Co-culture of <NOBR><I>E. Coli</I></NOBR> (orange), <NOBR><I>S. cerevisiae</I></NOBR> (blue) and <NOBR><I>S. elongatus</I></NOBR> (gray).</STRONG> The M2 medium was supplemented with lysine, histidine, leucine and uracil as well as ammonium sulfate (5 g/l). Growth measurements were conducted over a time period of 48 hours. Growth of <NOBR><I>S. elongatus</I></NOBR> was monitored via fluorescence emission at 680 nm (excitation at 620 nm). <NOBR><I>E. Coli</I></NOBR> and <NOBR><I>S. cerevisiae</I></NOBR> were measured in colony forming units (CFU). In order to compare <NOBR><I>E. Coli</I></NOBR> and <NOBR><I>S. cerevisiae</I></NOBR>, the y-axis is logarithmically scaled.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><STRONG> Figure 19: Co-culture of <NOBR><I>E. Coli</I></NOBR> (orange), <NOBR><I>S. cerevisiae</I></NOBR> (blue) and <NOBR><I>S. elongatus</I></NOBR> (gray).</STRONG> The M2 medium was supplemented with histidine, leucine and uracil as well as ammonium sulfate (5 g/l). Growth measurements were conducted over a time period of 48 h. While the growth of <NOBR><I>S. elongatus</I></NOBR> was measured via fluorescence emission at 680 nm with it being excited at 620 nm. <NOBR><I>E. Coli</I></NOBR> and <NOBR><I>S. cerevisiae</I></NOBR> were measured in colony forming unit (CFU).</FIGCAPTION></FIGURE><P class="details">In both three-way co-culture measurements (Figure 18,19) with all three organisms, the fluorescence of <NOBR><I>S. elongatus</I></NOBR> increases regardless of whether lysine is added or not. The CFU number of <NOBR><I>S. cerevisiae</I></NOBR> is decreasing, whereas it constantly varies for <NOBR><NOBR><I>E. Coli</I></NOBR></NOBR>.</P><H4>Discussion</H4><P class="details">Our results show that <NOBR><I>E. Coli</I></NOBR>  is unable to grow in M2 with sucrose on its own. Plating for <NOBR><I>E. Coli</I></NOBR> started 12 hours after the start of the incubation. Therefore, we cannot say that <NOBR><I>E. Coli</I></NOBR> is growing in sucrose enriched media with <NOBR><I>S. cerevisiae</I></NOBR>, since the growth is decreasing. However, the cell number is much higher in both co-cultures with <NOBR><I>S. cerevisiae</I></NOBR> compared to the growth rates of the monoculture. Hence, the possibility remains that we missed the exponential growth rate of <NOBR><I>E. Coli</I></NOBR> and just measured the stationary and dying phase of this strain, as we do not assume that we inoculated the strains differently in the flasks.

The growth of <NOBR><I>S. cerevisiae</I></NOBR> is slightly affected when co-cultured with <NOBR><I>E. Coli</I></NOBR> compared to the monoculture. While <NOBR><I>S. cerevisiae</I></NOBR> is co-cultured with <NOBR><I>E. Coli</I></NOBR>, its growth is visible, but more reduced than in monoculture. It may be due to the fact that <NOBR><I>E. Coli</I></NOBR> does consume a certain amount of glucose, which is extracellularly converted from sucrose by <NOBR><I>S. cerevisiae</I></NOBR>, or the added amino acids in the media.
However, when co-cultured with <NOBR><I>S. elongatus</I></NOBR>, the colony count quickly decreases, suggesting a dying population. We assume that <NOBR><I>S. elongatus</I></NOBR> is not able to provide enough sucrose for both <NOBR><I>E. Coli</I></NOBR> and <NOBR><NOBR><I>S. cerevisiae</I></NOBR></NOBR>. 

The growth of <NOBR><I>S. elongatus</I></NOBR> is unaffected by the addition of <NOBR><I>E. Coli</I></NOBR> and <NOBR><I>S. cerevisiae</I></NOBR> in the culture medium when compared to the growth in the monoculture.
For future measurements the experiment conditions should be adjusted to get clearer results.
Determination of the amount of CFU may be a very well established quantification method, but can be difficult to achieve accurate results since it is vital to work with correct dilutions, especially at very high cell densities. In addition, it can be difficult to predict cell densities at the point of measurement. Another way to quantify the cells without having to estimate  correct dilutions is with fluorescent measurements. For this co-culture combination, two different fluorescent proteins would be needed for <NOBR><I>E. Coli</I></NOBR> and <NOBR><I>S. cerevisiae</I></NOBR>. Here it is important to use fluorescent proteins with different emission and excitation spectrums which do not overlap.
The result shows that it might be better if the heterotrophic organisms were inoculated at a lower OD<SUB>600</SUB> while co-cultivated with <NOBR><I>S. elongatus</I></NOBR>. Modeling should be used to determine the optimal OD starting point for every organism.</P></DIV><BUTTON class="accordion"><H2>Outlook</H2></BUTTON><DIV class="panel"><H3>FRET based Biosensors</H3><P class="details">Towards the aim to evaluate amino acid and nutrient production of different strains, different quantification methods are required. A well established method currently is the high performance liquid chromatography (HPLC). Even though HPLC measurements are accurate and precise, the method has several disadvantages. It not only requires special equipment that might be too expensive for some laboratories and limits it's available, but measurements are complex and time consuming. Therefore, the auxotrophy approach is rather inconvenient for laboratories that don’t possess these capacities.
Because of the reasons mentioned above, an optical readout via biosensor construct is being suggested to take over the quantification of the produced amino acids. Fluorescent biosensor measurements are quick, simple, inexpensive and offer online analysis which will be helpful for high throughput productions.
Fluorescent Biosensors are already being used in a wide variety of experiments from observing cellular events<A href="https://dx.doi.org/10.1016%2Fj.chembiol.2013.12.012"><SUP>25</SUP></A> to tracking and detecting molecules<A href="https://doi.org/10.1016/j.snb.2005.08.032"><SUP>26</SUP></A>.

Fluorescence is defined as the emission of light by chemical substances - such as fluorophores - upon absorption of light waves at a specific wavelength. Single fluorophore biosensors can, for example, be used in detecting and determining a concentration of different molecules. However, a trend in biosensor engineering is to utilize dual fluorescent proteins. The attachment of two fluorophores for the purpose of observing Förster Resonanz Energy Transfer (FRET) offers the opportunity to detect and observe protein-protein interaction, protein folding or conformational changes of proteins.
FRET describes the nonradiative energy transfer between an excited donor fluorophore and an acceptor fluorophore in close proximity of about 2-10 nm<A href="https://www.nature.com/articles/nprot.2012.147#auth-1"><SUP>27</SUP></A>. The donor is transitioned into an excited stage by the absorption of light. Due to an energy transfer, provided that the emission spectrum of the donor overlaps with the excitation spectrum of the acceptor, the acceptor is excited, thus resulting in the emission of detectable light. The efficiency of the energy transfer depends on the correct orientation of donor and acceptor and the distance between them, which has to be below 10 nm<A href="https://www.nature.com/articles/nprot.2012.147#auth-1"><SUP>27</SUP></A>.</P><P class="details">For detection of substances via biosensors, FRET can be utilized in combination with periplasmic binding proteins.
These binding proteins share a common three-dimensional structure of two domains. Ligands are bound in the region between the two domains. They tend to have a high affinity for a particular substrate and will adopt a different conformation if bound to it - the so called ligand-bound closed form in contrast to the ligand-free open form<A href="https://doi.org/10.1016/j.sbi.2004.07.004"><SUP>28</SUP></A>.
The conformational changes are the key factor for detecting FRET signal changes when fluorophores are attached to the protein. If the conformational change induced by ligand binding brings donor and acceptor in closer proximity, a change in FRET will be measured<A href="https://www.nature.com/articles/nprot.2012.147#auth-1"><SUP>27</SUP></A>.
An example of a FRET biosensor, which the proposed biosensor is based on, is the construction of a lysine-/arginine-/ornithin (LAO) biosensor.
It consists of an enhanced cyan fluorescent protein (CFP) and a variant of enhanced yellow fluorescent protein (eYFP) fused to the LAO-binding protein from <I>E. coli</I> via different linker combinations. Those linker combinations - either rigid or flexible - alter the sensitivity and affinity of the biosensor<A href="https://doi.org/10.3390/s16101604"><SUP>29</SUP></A>.
For a more precise measurement, the binding protein was circularly permutated based on the work of Okada and colleagues<A href="https://doi.org/10.1002/pro.266"><SUP>30</SUP></A>. The FRET signal change of the native form was deemed to be not significant enough since the native C and N termini - and therefore the connected fluorophores - were located in the same domain. 
A hinge loop deletion and the connection of the native C and N termini created new C- and N- termini at different domains. Due to the attachment of the fluorescent proteins to the new termini, larger dynamic ranges were achieved that improved the measurement<A href="https://doi.org/10.1002/pro.266"><SUP>30</SUP></A>. </P><FIGURE><FIGCAPTION><STRONG>Figure 1: Construction of a FRET-biosensor based on Okada and colleagues.</STRONG>
A) Construct of Flip (=Fluorescent indicator protein) before (FLIP-HisJ) and after circular permutation (FLIP-cpHisJ194).
B) Structural model before circular permutation. The red line shows the hinges.  
C) Structural model after circular permutation. The dotted string represents the linker peptide between the two domains<A href="https://onlinelibrary.wiley.com/doi/full/10.1002/pro.266"><SUP>30</SUP></A>.
It’s clearly visible, that the fluorophores (blue and yellow) are reallocated through the circular permutation to different protein domains.</FIGCAPTION></FIGURE><P class="details">Besides amino acids, sugar detecting biosensors have been built as well. FLII12Pglu-700μδ6, derived from the FLII12Pglu-600μ biosensor<A href="https://onlinelibrary.wiley.com/doi/abs/10.1110/ps.051508105"><SUP>31</SUP></A>, was established by Takanaga, and is a biosensor for glucose<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2315637/"><SUP>32</SUP></A>. The glucose/galactose binding domain consisting of a Mglb binding protein is surrounded by eYFP and eCFP. When expressed in a cell, the FRET signal will give information about the current well-being of the organism, whether or not they are suffering a shortage in glucose - the main catabolized carbon source of microorganisms. 

Another concept of dual photometer biosensors relies on using a second fluorophore as an internal reference rather than for the observation of FRET.
Incorrect measurements can easily falsify the results as for instance fluorophores are prone to bleaching and measurement methods susceptible to changes in focus or variations in laser intensity. Therefore an internal reference fluorophore is used for comparison as to correct those artifacts<A href="https://www.nature.com/articles/s41467-017-00400-2"><SUP>33</SUP></A>.
Hence, the Matryoshka concept has been described by Ast and colleagues. The ratiometric Matryoshka-biosensor consists of a fluorophore inserted between the N and C terminus of another fluorophore inside the binding protein. Next to the advantage of having an internal reference fluorophore, the reporter and reference fluorophore can easily be cloned into one cassette<A href="https://www.nature.com/articles/s41467-017-00400-2"><SUP>33</SUP></A>.

As a proof of concept the MatryoshCaMP6s has been developed. Being an improved version of the CaMP6s sensor it carries cpEGFP and LSSmOrange between a calcium-binding calmodulin (CaM) domain and a CaM interacting M13 peptide<A href="https://www.nature.com/articles/s41467-017-00400-2"><SUP>33,</SUP></A><A href="https://www.nature.com/articles/nature12354"><SUP>34</SUP></A>.</P><FIGURE><FIGCAPTION><STRONG>Figure 2: Different constructs of the Matryoshka converted CaMP6s biosensor with LLSmOrange and GFP-variants as fluorophores.</STRONG> The LSSmOrange is nestled in between the GFP sequence, which is located between the binding protein<A href="https://www.nature.com/articles/s41467-017-00400-2"><SUP>33</SUP></A>.</FIGCAPTION></FIGURE><P class="details">In order to easily design dual fluorophore-biosensors based on the Matryoshka concept, a GO Matryoshka cassette was generated. It includes a green fluorescent protein (GFP) and an orange fluorescent protein with a Large Stoke Shift (OFPlss). Stoke Shift describes the difference between excitation and emission maxima which, in this case, is bigger than the average<A href="https://www.nature.com/articles/s41467-017-00400-2"><SUP>33</SUP></A>.

The proposed biosensor is going to be a combination of both FRET biosensor based on the results of Pohl and the reference concept developed by Ast and colleagues.
The LAO-binding protein used in the work of Pohl will be connected to two halves of a split GFP which make - if the conformational change takes place - one GFP. On one of the half mCherry as the reference fluorophore is attached.</P><FIGURE><FIGCAPTION><STRONG>Figure 3: Proposed biosensor consisting of a binding protein carrying a split GFP and mCherry as a reference.</STRONG> After binding to the analyte, the binding protein switches conformations and the GFP halfs reunite, resulting in fluorescence.
(Photo credit: Hau B. Nguyen et al, 2013, modified Researchgate.net, modified)</FIGCAPTION></FIGURE><H4>Materials and Methods</H4><P class="details">For cloning in <I>E. coli</I>, the CIDAR toolbox, established by Iverson and colleagues will be used<A href="https://doi.org/10.1021/acssynbio.5b00124"><SUP>35</SUP></A>.
The split GFP will consist of GFP<SUB>1-10</SUB> and GFP<SUB>11</SUB>. The GFP<SUB>1-10</SUB> sequence as used in “Versatile protein tagging in cells with split fluorescent protein” is: </P><P class="details">ATGTCCAAAGGAGAAGAACTGTTTACCGGTGTTGTGCCAATTTTGGTTGAACTCGATGG TGATGTCAACGGACATAAGTTCTCAGTGAGAGGCGAAGGAGAAGGTGACGCCACCATTG GAAAATTGACTCTTAAATTCATCTGTACTACTGGTAAACTTCCTGTACCATGGCCGACT CTCGTAACAACGCTTACGTACGGAGTTCAGTGCTTTTCGAGATACCCAGACCATATGAA AAGACATGACTTTTTTAAGTCGGCTATGCCTGAAGGTTACGTGCAAGAAAGAACAATTT CGTTCAAAGATGATGGAAAATATAAAACTAGAGCAGTTGTTAAATTTGAAGGAGATACT TTGGTTAACCGCATTGAACTGAAAGGAACAGATTTTAAAGAAGATGGTAATATTCTTGG ACACAAACTCGAATACAATTTTAATAGTCATAACGTATACATCACTGCTGATAAGCAAA AGAACGGAATTAAAGCGAATTTCACAGTACGCCATAATGTAGAAGATGGCAGTGTTCAA CTTGCCGACCATTACCAACAAAACACCCCTATTGGAGACGGTCCGGTACTTCTTCCTGA TAATCACTACCTCTCAACACAAACAGTCCTGAGCAAAGATCCAAATGAAAAA</P>

The GFP<SUB>11</SUB> sequence with a 5 amino acid linker is:<Q>ATGCGTGACCACATGGTCCTTCATGAGTATGTAAATGCTGCTGGGATTACAGGTGGCGGCAAATTC</Q>

The LAO binding protein sequence was characterized by Steffen and colleagues and is available at GenBank (No: CP001665.1, 1424860 to 1425642). Based on the work of Okada and colleagues, it is permutated with the deleted base pairs being GSSVKDKKYFGD, implemented in <I>Corynebacterium glutamicum</I><A href="https://doi.org/10.3390/s16101604"><SUP>29,</SUP></A><A href="https://doi.org/10.1002/pro.266"><SUP>30</SUP></A>.<FIGURE><FIGCAPTION><STRONG>Figure 4: Cloning construct of the proposed biosensor. </STRONG> GFP is split in half. Promoter, RBS and Terminator will be used from the CIDAR toolbox.</FIGCAPTION></FIGURE><P class="details">The sugar sensor FLII12Pglu-700µδ6 was established by Takanaga and colleagues and can be obtained via Addgene (Plasmid #28002).

Both<I>in vivo</I> and <I>ex vivo</I> measurements would provide interesting insights. A purified biosensor in the media would inform about ligand content in the media and secretion. The translation inside an organisms would show the content of measured compound in the cells and the uptake. </P><H4>Possible Adjustments</H4><P class="details">Other than FRET - and Matryoshka based biosensors - there are a lot more interesting possibilities for sensing amino acids via fluorophores.
Instead, transcriptional regulated reporter cells can be utilized. Through the activation of promoters or deactivation of repressors by the to be sensed substance, transcription and translation of thereafter encoded fluorescent proteins can take place<A href="https://www.ncbi.nlm.nih.gov/pubmed/26521244"><SUP>36</SUP></A>.
However, it is important to note that this construct will not respond as quickly, as it is based on transcription activities and only measures the uptake of the product instead of the concentration in the media. 
Another opportunity to implement the reference fluorophore instead of just linking it to the biosensor construct could be the establishment of viral 2A-peptide linkers between the receptor and reference fluorophores. Those linker-peptides, when translated into amino acids, break due to instability. As a result, the reference protein is not fused to the sensing machine and does not hinder the binding protein<A href="https://doi.org/10.1002/dvg.20338"><SUP>37</SUP></A>.</P><H3>Using our co-culture for production</H3><P class="details">Using monocultures for production of various compounds was a miracle of the previous century. Unfortunately the possibilities are limited. Nowadays researchers and industries are confronted with more and more difficulties, such as synthesizing products that can not be produced by only one organism or by the need to reduce high production costs. Furthermore, contamination or nutrient deficiencies often decrease the yield. 
Co-cultures not only offer new insights into microbial consortia, but also opportunities for diverse applications, including an increased production and the synthesis of novel compounds for the use in different fields. 
A higher yield for the production of different substances in co-cultures compared to monocultures has already been established for several biosynthesis<A href="http://www.pnas.org/content/early/2015/06/24/1506781112"><SUP>38,</SUP></A><A href="https://kundoc.com/pdf-increased-hydrogen-production-in-co-culture-of-chlamydomonas-reinhardtii-and-bra.html"><SUP>39,</SUP></A><A href="https://www.tandfonline.com/doi/abs/10.1080/21622515.2015.1068381"><SUP>40,</SUP></A><SUP><SUP><A href="https://www.sciencedirect.com/science/article/pii/S1096717617302057?via%3Dihub"><SUP>41,</SUP></A><A href="https://www.tandfonline.com/doi/full/10.1080/07388551.2017.1390728"><SUP>42</SUP></A>. As an instance, the synthesis of hydrogen has been significantly improved when microorganisms were incubated side by side<A href="https://kundoc.com/pdf-increased-hydrogen-production-in-co-culture-of-chlamydomonas-reinhardtii-and-bra.html"><SUP>39,</SUP></A><A href="https://www.tandfonline.com/doi/abs/10.1080/21622515.2015.1068381"><SUP>40,</SUP></A><SUP><SUP>. 
When splitting complex biosynthetic pathways into different organisms, the division of labor increases the yield as well, in addition to providing a more cost effective production method<A href="https://www.sciencedirect.com/science/article/pii/S1096717617302057?via%3Dihub"><SUP>41,</SUP></A>.
Microbial consortia can also induce the production of previously unknown substances. Secondary metabolites that are only produced in the presence of other organisms may be used in the medical field as new antibiotics<A href="https://www.mdpi.com/1660-3397/12/5/3046"><SUP>43,</SUP></A><A href="https://pubs.acs.org/doi/abs/10.1021/acschembio.7b00688"><SUP>44,</SUP></A><A href="https://www.frontiersin.org/articles/10.3389/fmicb.2017.00649/full"><SUP>45</SUP></A>.The genes responsible for those compounds are usually not transcribed in monocultures.
An additional advantage that are offered by co-cultures is the lower risk for contamination<A href="https://www.tandfonline.com/doi/full/10.1080/07388551.2017.1390728"><SUP>42,</SUP></A><A href="https://www.frontiersin.org/articles/10.3389/fmicb.2017.01125/full"><SUP>46</SUP></A> and the robustness<A href="https://www.tandfonline.com/doi/abs/10.1080/21622515.2015.1068381"><SUP>40</SUP></A> compared to conventional cultivation conditions, especially during environmental disturbances that are buffered by induced cooperativity<A href="https://www.biorxiv.org/content/early/2018/09/25/426171"><SUP>47</SUP></A>.
Because of our aim to establish a toolbox for several applications, our <I>E. coli</I> contains a GFP under the control of the quorum sensing promoter  as a proof of principle. The <I>gfp</I> gene can later be exchanged with a specific sequence for the desired product and production can be induced via AHL. 
Furthermore, our three-way co-culture has been engineered to use a nutrient marker as a dependency which has the additional benefit of reducing the contamination risk in place of antibiotic pressure and preventing antibiotic resistances in the future<A href="http://science.sciencemag.org/content/353/6299/583"><SUP>16</SUP></A>.
Future scientist just have to choose the system that fits best to their application, can add our and their constructs to obtain a complete co-culture, which produces the desired product including all advantages mentioned above, without having to generate a whole new co-culture.</SUP></SUP></SUP></SUP></P></DIV><BUTTON class="accordion"><H2>References</H2></BUTTON><DIV class="panel"><OL type="1"><LI>
Scott, Spencer R., et al. &quot;A stabilized microbial ecosystem of self-limiting bacteria using synthetic quorum-regulated lysis.&quot; <I>Nature microbiology</I> 2.8 (2017): 17083.
</LI><LI>
Scott, Spencer R., and Jeff Hasty. &quot;Quorum sensing communication modules for microbial consortia.&quot; <I>ACS synthetic biology</I> 5.9 (2016): 969-977.
</LI><LI>
Bernhardt, Thomas G., Douglas K. Struck, and R. Y. Young. &quot;The lysis protein E of φX174 is a specific inhibitor of the MraY-catalyzed step in peptidoglycan synthesis.&quot; <I>Journal of Biological Chemistry</I> 276.9 (2001): 6093-6097.
</LI><LI>
UniProt “ Lysis Protein E”. UniProt Consortium (2018).
</LI><LI>
Williams, Thomas C., Lars K. Nielsen, and Claudia E. Vickers. &quot;Engineered quorum sensing using pheromone-mediated cell-to-cell communication in Saccharomyces cerevisiae.&quot; <I>ACS synthetic biology</I> 2.3 (2013): 136-149.
</LI><LI>
Lee, Michael E., et al. &quot;A highly characterized yeast toolkit for modular, multipart assembly.&quot; <I>ACS synthetic biology</I> 4.9 (2015): 975-986.
</LI><LI>
Machens, Fabian, et al. &quot;Synthetic Promoters and Transcription Factors for heterologous Protein expression in <I>Saccharomyces cerevisiae</I>.&quot; <I>Frontiers in bioengineering and biotechnology</I> 5 (2017): 63.
</LI><LI>
Neddermann, Petra, et al. &quot;A novel, inducible, eukaryotic gene expression system based on the quorum‐sensing transcription factor TraR (EMBO reports, February 2003)(Erratum).&quot; <I>EMBO reports</I> 4.4 (2003): 439-439.
</LI><LI>
Allen, James B., et al. &quot;Finding prospective partners in the library: the two-hybrid system and phage display find a match.&quot; <I>Trends in biochemical sciences</I> 20.12 (1995): 511-516.
</LI><LI>
Brachmann, Carrie Baker, et al. &quot;Designer deletion strains derived from <I>Saccharomyces cerevisiae</I> S288C: a useful set of strains and plasmids for PCR‐mediated gene disruption and other applications.&quot; <I>Yeast</I> 14.2 (1998): 115-132.
</LI><LI>
Wallon, Gerlind, et al. &quot;Purification, catalytic properties and thermostability of 3-isopropylmalate dehydrogenase from <I>Escherichia coli</I>.&quot; <I>Biochimica et Biophysica Acta (BBA)-Protein Structure and Molecular Enzymology</I> 1337.1 (1997): 105-112.
</LI><LI>
Interview with Dr. Ducat (extract):
    <Q>Sarah: “Do you have any idea what we could use for S. elongatus as auxotrophy? Right now we’re depending on sucrose secretion. But we want another dependency for our other two organisms.”
 
Danny: “The one that we played with is tryptophan auxotroph. [...] If you produce it inside the cell it can slip right through the membrane without a transporter. We have done some unpublished work with cyanobacteria that are tryptophan auxotroph [...].</Q></LI><LI><A href="”https://www.uniprot.org/uniprot/Q31R76”">https://www.uniprot.org/uniprot/Q31R76</A></LI><LI>
Kanda, Keisuke, et al. &quot;Application of a phosphite dehydrogenase gene as a novel dominant selection marker for yeasts.&quot; <I>Journal of biotechnology</I> 182 (2014): 68-73.
</LI><LI>
Weiss, Taylor L., Eric J. Young, and Daniel C. Ducat. &quot;A synthetic, light-driven consortium of cyanobacteria and heterotrophic bacteria enables stable polyhydroxybutyrate production.&quot; <I>Metabolic engineering</I> 44 (2017): 236-245.
</LI><LI>
Shaw, A. Joe, et al. &quot;Metabolic engineering of microbial competitive advantage for industrial fermentation processes.&quot; <I>Science</I> 353.6299 (2016): 583-586.
</LI><LI>
Giovannini, Donatella, et al. &quot;Inorganic phosphate export by the retrovirus receptor XPR1 in metazoans.&quot; <I>Cell reports</I> 3.6 (2013): 1866-1873.
</LI><LI>
Legati, Andrea, et al. &quot;Mutations in XPR1 cause primary familial brain calcification associated with altered phosphate export.&quot; <I>Nature genetics</I> 47.6 (2015): 579.
</LI><LI>
YSI Inc.”2950D Biochemistry Analyzer” (2018).
</LI><LI>
Abramson, Bradley W., et al. &quot;Increased photochemical efficiency in cyanobacteria via an engineered sucrose sink.&quot; <I>Plant and Cell Physiology</I> 57.12 (2016): 2451-2460.  
</LI><LI>
Sabri, Suriana, Lars K. Nielsen, and Claudia E. Vickers. &quot;Molecular control of sucrose utilization in Escherichia coli W, an efficient sucrose-utilizing strain.&quot; <I>Applied and environmental microbiology</I> (2012): AEM-02544.
</LI><LI>
Marques, Wesley Leoricy, et al. &quot;Sucrose and Saccharomyces cerevisiae: a relationship most sweet.&quot; <I>FEMS Yeast research</I> 16.1 (2016).
</LI><LI>
Kremers, Gert-Jan, et al. &quot;Cyan and yellow super fluorescent proteins with improved brightness, protein folding, and FRET Förster radius.&quot; <I>Biochemistry</I> 45.21 (2006): 6570-6580.  
</LI><LI>
Goedhart, Joachim, et al. &quot;Bright cyan fluorescent protein variants identified by fluorescence lifetime screening.&quot; <I>Nature methods</I> 7.2 (2010): 137.
</LI><LI>
Oldach, Laurel, and Jin Zhang. &quot;Genetically encoded fluorescent biosensors for live-cell visualization of protein phosphorylation.&quot; <I>Chemistry &amp; biology</I> 21.2 (2014): 186-197.
</LI><LI>
Viveros, L., et al. &quot;A fluorescence-based biosensor for the detection of organophosphate pesticides and chemical warfare agents.&quot; <I>Sensors and Actuators B: Chemical</I> 115.1 (2006): 150-157.  
</LI><LI>
Broussard, Joshua A., et al. &quot;Fluorescence resonance energy transfer microscopy as demonstrated by measuring the activation of the serine/threonine kinase Akt.&quot; <I>Nature protocols</I> 8.2 (2013): 265.
</LI><LI>
Dwyer, Mary A., and Homme W. Hellinga. &quot;Periplasmic binding proteins: a versatile superfamily for protein engineering.&quot; <I>Current opinion in structural biology</I> 14.4 (2004): 495-504.
</LI><LI>
Steffen, Victoria, et al. &quot;A toolbox of genetically encoded FRET-based biosensors for rapid l-lysine analysis.&quot; <I>Sensors</I> 16.10 (2016): 1604.
</LI><LI>
Okada, Satoshi, Kazuhisa Ota, and Takashi Ito. &quot;Circular permutation of ligand‐binding module improves dynamic range of genetically encoded FRET‐based nanosensor.&quot; <I>Protein Science</I> 18.12 (2009): 2518-2527.
</LI><LI>
Deuschle, Karen, et al. &quot;Construction and optimization of a family of genetically encoded metabolite sensors by semirational protein engineering.&quot; <I>Protein Science</I> 14.9 (2005): 2304-2314.
</LI><LI>
Takanaga, Hitomi, Bhavna Chaudhuri, and Wolf B. Frommer. &quot;GLUT1 and GLUT9 as major contributors to glucose influx in HepG2 cells identified by a high sensitivity intramolecular FRET glucose sensor.&quot; <I>Biochimica et Biophysica Acta (BBA)-Biomembranes</I> 1778.4 (2008): 1091-1099.
</LI><LI>
Ast, Cindy, et al. &quot;Ratiometric Matryoshka biosensors from a nested cassette of green-and orange-emitting fluorescent proteins.&quot; <I>Nature communications</I> 8.1 (2017): 431.
</LI><LI>
Chen, Tsai-Wen, et al. &quot;Ultrasensitive fluorescent proteins for imaging neuronal activity.&quot; <I>Nature</I> 499.7458 (2013): 295.
</LI><LI>
Iverson, Sonya V., et al. &quot;CIDAR MoClo: improved MoClo assembly standard and new E. coli part library enable rapid combinatorial design for synthetic and traditional biology.&quot; <I>ACS synthetic biology</I> 5.1 (2015): 99-103.
</LI><LI>
Mahr, Regina, and Julia Frunzke. &quot;Transcription factor-based biosensors in biotechnology: current state and future prospects.&quot; <I>Applied microbiology and biotechnology</I> 100.1 (2016): 79-90.
</LI><LI>
Provost, Elayne, Jerry Rhee, and Steven D. Leach. &quot;Viral 2A peptides allow expression of multiple proteins from a single ORF in transgenic zebrafish embryos.&quot; <I>Genesis</I> 45.10 (2007): 625-629.
</LI><LI>
Zhang, Haoran, et al. &quot;Engineering Escherichia coli coculture systems for the production of biochemical products.&quot; <I>Proceedings of the National Academy of Sciences</I> (2015): 201506781.
</LI><LI>
Wu, Shuangxiu, et al. &quot;Increased hydrogen production in co-culture of Chlamydomonas reinhardtii and Bradyrhizobium japonicum.&quot; <I>Bioresource technology</I> 123 (2012): 184-188.
</LI><LI>
Pachapur, Vinayak Laxman, et al. &quot;Biological hydrogen production using co-culture versus mono-culture system.&quot; <I>Environmental Technology Reviews</I> 4.1 (2015): 55-70.
</LI><LI>
Liu, Yiqi, et al. &quot;Engineered monoculture and co-culture of methylotrophic yeast for de novo production of monacolin J and lovastatin from methanol.&quot; <I>Metabolic engineering</I> 45 (2018): 189-199.
</LI><LI>
Padmaperuma, Gloria, et al. &quot;Microbial consortia: a critical look at microalgae co-cultures for enhanced biomanufacturing.&quot; <I>Critical reviews in biotechnology</I> 38.5 (2018): 690-703.
</LI><LI>
Dashti, Yousef, et al. &quot;Production of induced secondary metabolites by a co-culture of sponge-associated actinomycetes, Actinokineospora sp. EG49 and Nocardiopsis sp. RV163.&quot; <I>Marine drugs</I> 12.5 (2014): 3046-3059.
</LI><LI>
Adnani, Navid, et al. &quot;Coculture of Marine Invertebrate-Associated Bacteria and Interdisciplinary Technologies Enable Biosynthesis and Discovery of a New Antibiotic, Keyicin.&quot; <I>ACS chemical biology</I> 12.12 (2017): 3093-3102.
</LI><LI>
Serrano, Rachel, et al. &quot;Co-culturing of fungal strains against botrytis cinerea as a model for the induction of chemical diversity and therapeutic agents.&quot; <I>Frontiers in microbiology</I> 8 (2017): 649.
</LI><LI>
Ben Said, Sami, and Dani Or. &quot;Synthetic microbial ecology: Engineering habitats for modular consortia.&quot; <I>Frontiers in microbiology</I> 8 (2017): 1125.
</LI><LI>
Ziesack, Marika, et al. &quot;Inducible Cooperation in a Synthetic Gut Bacterial Consortium Introduces Population Balance and Stability.&quot; <I>bioRxiv</I> (2018): 426171.

</LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>