<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Duesseldorf_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Duesseldorf/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="sidebar"><UL><LI><A href="https://2018.igem.org/Team:Duesseldorf">Home</A></LI><UL class="sub"><A href="#!">Team</A><LI><A href="https://2018.igem.org/Team:Duesseldorf/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Ontour">On Tour</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Attributions">Attributions</A></LI></UL><UL class="sub"><A href="#!">Project</A><LI><A href="https://2018.igem.org/Team:Duesseldorf/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Improve">Improve</A></LI></UL><UL class="sub"><A href="#!">Parts</A><LI><A href="https://2018.igem.org/Team:Duesseldorf/Parts">Parts Overview</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Composite_Part">Composite Part</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Part_Collection">Part Collection</A></LI></UL><LI><A href="https://2018.igem.org/Team:Duesseldorf/Safety">Safety</A></LI><UL class="sub"><A href="#!">Human Practices</A><LI><A href="https://2018.igem.org/Team:Duesseldorf/Silver">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/HP/Gold_Integrated">Integrated Practices</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Newsletter">Newsletter</A></LI></UL><UL class="sub"><A href="#!">Awards</A><LI><A href="https://2018.igem.org/Team:Duesseldorf/Medals">Medal Criteria</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Plant">Plant Synthetic Biology</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Model">Model</A></LI></UL><LI><A href="https://igem.org/2018_Judging_Form?team=Duesseldorf">Judging Form</A></LI></UL><UL class="sub"><A href="#!">Advice</A><LI><A href="https://2018.igem.org/Team:Duesseldorf/MoClo">Modular Cloning</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/CyanoQandA">Cyano Q&amp;A</A></LI></UL></DIV><NAV><UL class="menu"><LI class="menu-item-has-children"><A href="#!">Team</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Ontour">On Tour</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Attributions">Attributions</A></LI></UL><LI class="menu-item-has-children"><A href="#!">Project</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Improve">Improve</A></LI><LI class="menu-item-has-children"><A href="#!">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Parts">Parts overview</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Composite_Part">Composite Part</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Part_Collection">Part Collection</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Safety">Safety</A></LI><LI class="menu-item-has-children"><A href="#!">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Silver">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/HP/Gold_Integrated">Integrated Practices</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Newsletter">Newsletter</A></LI><LI class="menu-item-has-children"><A href="#!">Awards</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Medals">Medal Criteria</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Plant">Plant Synthetic Biology</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Model">Model</A></LI><LI class="menu-item-has-children"><A href="#!">Advice</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/MoClo">Modular Cloning</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/CyanoQandA">Cyano Q&amp;A</A></LI><LI><A href="https://igem.org/2018_Judging_Form?team=Duesseldorf">Judging Form</A></LI></NAV><H1>Demonstrate</H1><DIV class="callout"><H2>Self regulating <I>Escherichia coli</I></H2><P>In this part of the project, the aim was to regulate cell growth of <I>E. coli</I> BL21(DE3)C43 cells. We wanted to accomplish this by expressing a lysis gene under the control of the quorum sensing promoter P<SUB><I>lux</I></SUB>. To test whether the cell growth was impaired by the presence of a lysis protein, different experiments were conducted. <I>E. coli</I> cells harboring different plasmids, the lysis plasmid and the different controls, were incubated overnight at 37 °C, <NOBR>220 rpm</NOBR>. On the following day, the cells were diluted to a final OD<SUB>600</SUB> of 0.1. 200 µl of these dilutions were distributed on a 96-well plate for plate reader measurements. Each sample was measured in technical triplicates during all experiments. In a final approach, cells harboring the <NOBR><I>luxI</I>-<I>luxR</I>-P<SUB><I>lux</I></SUB> -<I>phiX174E</I></NOBR> lysis construct were grown in LB-medium until an OD<SUB>600</SUB> of two. Culture supernatant was harvested, sterile filtered and then mixed 1:1 with pure LB medium.</P><H4>Results</H4><P>
To check the functionality of the P<SUB><I>lux</I></SUB> promoter upon induction by the quorum sensing molecule acyl homoserine lactone (AHL) and the respective transcriptional activator protein LuxR, an experiment based on the expression of <I>gfp</I> under the control of the respective quorum sensing promoter  was performed (Figure 1). It can be seen that GFP fluorescence increases over time, while the untreated cells show low or no fluorescence at all. This observation led us to the conclusion that the promoter P<SUB><I>lux</I></SUB> is induced upon synthesis of AHL by the LuxI synthase and subsequent binding to the LuxR regulator. Therefore, this promoter is characterized as functional for further use within our project, as well as within the <A href="http://parts.igem.org/Part:BBa_K2587027"> iGEM registry</A>. 
In order to exclude that the lack of fluorescence of the progenitor cells is due to poor growth, a measurement of the optical density was included (Figure 1). As expected, the cells grow similar, therefore it can be concluded that the lack of fluorescence of the progenitor cells is not due to a smaller population size. 
The performance of the P<SUB><I>lux</I></SUB> promoter is further confirmed by fluorescence microscopy (Figure 1). Here it can be seen that fluorescence is present in the case of activation of the P<SUB><I>lux</I></SUB> promoter by the quorum sensing system (Figure 1A), while no fluorescence is present in the corresponding untreated cells (Figure 1C). As a positive control, GFP production under the control of a constitutive promoter was checked (Figure 1B). When comparing the expression of <I>gfp</I> under the control of different promoters (P<SUB><I>lux</I></SUB> and <A href="http://parts.igem.org/Part:BBa_J23102">constitutive promoter J23102</A>), higher fluorescence could be visualized in the latter case.  <FIGURE><FIGCAPTION><STRONG>Figure 1: Confocal fluorescence microscopy</STRONG>  of A: <I>E. coli</I> cells with <I>luxI</I>-<I>luxR</I>-P<SUB><I>lux</I></SUB>-<I>gfp</I> construct; B: <I>E. coli</I> cells with constitutive expression of gfp  and C: <I>E. coli</I> BL21(DE3)C43 cells.</FIGCAPTION></FIGURE></P><P>
Next, the efficiency of the lysis plasmid containing the lysis gene <I>E</I> from bacteriophage phiX174 was tested (Figure 2). This was done by measuring the optical density of the cells containing the <A href="http://parts.igem.org/Part:BBa_K2587024">lysis plasmid</A>. Progenitor <I>E. coli</I> cells and cells harboring an empty vector control  were included as negative controls. The latter was done to rule out a higher cell density of the progenitor strain than the other cells due to the lack of antibiotics in the culture medium. It is noticeable that growth of the cells is influenced by the presence of the lysis plasmid. This is especially observable during the stationary phase, starting after 10 hours, as the cells do not reach the same maximum density as the wild type cells or the cells harboring the uninduced plasmid (<I>luxR</I>-P<SUB><I>lux</I></SUB>-<I>gfp</I>).
<FIGURE><FIGCAPTION><STRONG>Figure 2: Effect of lysis plasmid in E. coli cells compared to progenitor E. coli cells.</STRONG> As a control, the progenitor strain (<I>E. coli</I> BL21(DE3)C43) as well as another strain carrying the <I>luxR</I>-P<SUB><I>lux</I></SUB>-<I>gfp</I> construct was used. The different colors represent different <I>E. coli</I> strains. Measurements were performed over a time period of 24 h. Mean values of triplicates subtracted from the respective blank are plotted.  </FIGCAPTION></FIGURE></P><H4>Conclusion</H4><P>
The aim of this subproject was to construct a strain that grows slower than its progenitor counterpart. We consider this step to be essential in establishing the first part of our co-culture. Our priority here lies on us being able to make the co-culture populations achieve balance. We therefore applied the quorum sensing system, which is a reliable and well characterized tool of synthetic biology<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5603278/"><SUP>2</SUP></A> and wanted to accomplish this by choosing a lysis gene able to inhibit cell wall synthesis. As compared to the constitutive expression of <I>gfp</I>, a lower fluorescence intensity is visible (Figure 1A/B). This might be due to the smaller amounts of AHL synthetized by LuxI, which might lead to more limited activation of the P<SUB><I>lux</I></SUB> or, more likely, different promoter strengths. In this case, lysis protein E is used which is able to interact with the host's SlyD and therefore enables protein E to be protected from proteolysis. This way, protein E is able to interact with MraY (an essential membrane enzyme for bacterial cell wall synthesis) at the host membrane, which blocks MraY translocase, necessary for lipid I catalysis. The latter is an essential component for host cell wall synthesis<A href="https://www.ncbi.nlm.nih.gov/pubmed/11078734"><SUP>3</SUP></A>.
As suggested by Dr. Spencer Scott, an expert for quorum sensing regulation, whom we asked for advice as <A href="https://2018.igem.org/Team:Duesseldorf/HP/Gold_Integrated">Integrated Human Practices</A>, we let the cells grow for only <NOBR>24 hours</NOBR>. We showed efficacy of our system by demonstrating that growth, based on OD<SUB>600</SUB> measurements, is decreased in cells harbouring the lysis plasmid. Based on literature, lysis occurs because the lysis protein E impairs peptidoglycan synthesis<A href="https://www.uniprot.org/uniprot/P03639"><SUP>4</SUP></A>. This is consistent with our finding, since cell lysis results in less cells, shown by a lower OD<SUB>600</SUB>  measurement. Longer measurements resulted in lower OD in the control strains as well, likely due to cell death from starvation, which often happens in older cultures. Moreover, as was described in literature, lysis occurs only when the medium reaches a certain concentration threshold of the quorum sensing molecule AHL, probably during the stationary phase and induces a lysis event<A href="https://www.nature.com/articles/nmicrobiol201783."><SUP>1</SUP></A>. Also, similar to what Scott and colleagues discovered, population reduction is observable at around ten hours of growth<A href="https://www.nature.com/articles/nmicrobiol201783."><SUP>1</SUP></A>.</P></DIV><DIV class="callout"><H2>Lysine Supply by <I>E. coli</I></H2><P>In our auxotrophy system the aim was to establish an auxotrophic dependency of <NOBR><I>S. cerevisiae</I></NOBR> BY4742 (<NOBR><I>S. cerevisiae</I></NOBR> in the following) and <I>E. coli</I> BL21(DE3)C43 harboring <I>lysC</I> for lysine overproduction (<NOBR><I>E. coli</I>_<I>lysC</I></NOBR> in the following). In order to analyze the growth of <NOBR><I>E. coli</I></NOBR> and <I>S. cerevisiae</I> in mono- and co-culture, we performed a growth experiment over 36 hours. For all experiments, M2 medium, supplemented with 1.5% glucose, was used. To show the functionality of our approach, <NOBR><I>E. coli</I></NOBR> expressing a codon optimized and feedback resistant version of <I>lysC</I> under the control of a constitutively active promoter <A href="http://parts.igem.org/Part:BBa_J23100">Bba_J23100</A>, was cultivated together with <I>S. cerevisiae</I> which is auxotrophic for uracil, histidine, leucine and lysine.
Both strains were cultivated in monocultures to observe their growth with the four additional amino acids as a control. In our second approach, both were cultivated in a co-culture without lysine. Furthermore, the medium of the <I>E. coli</I>_<I>lysC</I> strain was sterilized and reused to cultivate <I>S. cerevisiae</I> to observe growth behavior in medium containing lysine produced by <I>E. coli</I> . In this approach, three amino acids, but no lysine, were added.<FIGURE><FIGCAPTION><STRONG>Figure 1: Growth of S. cerevisiae in reused M2 cultivation medium of E. coli_lysC.</STRONG> Comparison of <I>S. cerevisiae</I> growth rates in the positive control with four additional amino acids (Standard Medium) and reused M2 cultivation medium of <I>E. coli</I>_<I>lysC</I> over 36 h. In the other culture the supernatant of E. coli_lysC cultures was used and no lysine was added (Supernatant of E. coli). The y-axis shows the CFU in 100 µl.</FIGCAPTION></FIGURE></P><P>
Figure 1 shows the growth of <I>S. cerevisiae</I> monocultures. Here, the growth of the positive control and the experiment with the reused medium, described in the experimental design, are compared. The results show that it was possible to observe growth of <NOBR><I>S. cerevisiae</I><NOBR> in the medium which was lysine-enriched by <NOBR><I>E. coli</I>_<I>lysC</I></NOBR>. After around 4 hours, the cells in both cultures began to grow. After 12 hours, cell density reached its maximum in both cultures while the total density was higher in the experiment with the reused medium. After the highest amount of cells was reached, the cell density decreased faster in the medium experiment compared to the control. After 36 hours the cells reached a similar cell density. These results show that the lysine produced by <I>E. coli</I>_<I>lysC</I>, is able to allow <NOBR><I>S. cerevisiae</I></NOBR> to grow as well as it would in lysine containing medium. This can also be seen as a proof of principle for synthetic dependencies based on amino acid auxotrophies in general.</NOBR></NOBR></P><H4>Conclusion</H4><P>
With the experiment using the reused medium enriched with lysine by <NOBR><I>E. coli</I>_<I>lysC</I></NOBR>, we could show that our approach with the auxotrophy system is very promising. Auxotrophy based dependencies between organisms are also used in recent research<A href="https://www.sciencedirect.com/science/article/pii/S136952741630025X"><SUP>5</SUP></A>. It would be interesting to improve our system by trying to cultivate the cultures under different conditions like inoculating the different organisms at lower cell density. One other approach and the next step for our desired, working co-culture would be the growth inhibition of <I>E. coli</I> using our self regulating system. This could prevent the <I>E. coli</I> cells from overgrowing the slower growing organisms. 
We were able to show that the lysine produced by <NOBR><I>E. coli</I>_<I>lysC</I></NOBR> was sufficient to let <I>S. cerevisiae</I> grow in the lysine enriched medium. At the end of the measurement the cell density was at the same low level in both cultures. It would be interesting to observe whether M2 medium pre-incubated with <I>E. coli</I>_<I>lysC</I> for a longer period of time would lead to a higher growth rate or to a slower decrease of the cell density of <I>S. cerevisiae</I>.

</P></DIV><DIV class="callout"><H2>Sucrose Production of <NOBR><I>S. elongatus</I></NOBR></H2><P>To detect the amount of sucrose production and secretion of our carbon source provider <NOBR><I>S. elongatus</I></NOBR> in the co-culture, we performed a biochemical assay using the YSI 2950 Biochemistry analyzer<A href="https://www.ysi.com/ysi-2950-biochemistry-analyzer"><SUP>6</SUP></A>. This method relies on the detection of sucrose mediated by a voltage change by immobilized enzymes on a specific membrane. 
For this purpose, we inoculated the <I>S. elongatus</I> PCC 7942 <I>cscB</I>:::NS3 strain at an OD<SUB>750</SUB> of 0.3 and incubated it under standard conditions for two days. After two days, the culture was induced with 1 mM IPTG and <NOBR>150 mM</NOBR> NaCl. The first samples were taken before induction, then after two days, four days and six days. As a control, wild type <I>S. elongatus</I>  was treated in the same manner and samples were taken at the same time points, respectively.
To measure sucrose content, each sample was centrifuged at 4000 rpm for three minutes and the supernatant was sterile filtered for analysis. 
A sucrose standard curve ranging from 1 mg/mL to 0.0625 mg/mL (1:2 dilutions each) was prepared
in order to determine the final sucrose concentration in the tested samples.
It can be seen that the amount of sucrose increases after induction. The media of wild type <NOBR><I>S. elongatus</I></NOBR> before induction does not show any presence of sucrose, while small amounts of sucrose are detectable before the induction of <NOBR><I>S. elongatus</I> PCC 7942 <I>cscB</I>:::NS3</NOBR>.  
<FIGURE><FIGCAPTION><STRONG>Figure 1: Sucrose content in medium supernatant of <I>S. elongatus</I> and PCC 7942 <I><I>cscB</I></I>:::NS3 strain in days (d) post induction as well as pre induction with 150 mM NaCl and 1 mM IPTG.</STRONG> Values are calculated from triplicates and measured based on a sucrose standard curve.</FIGCAPTION></FIGURE></P><H4>Conclusion</H4><P>
We could clearly demonstrate that upon induction with NaCl and IPTG, the sucrose content is higher than in the uninduced variant of the <I>S. elongatus</I> PCC 7942 <I>cscB</I>:::NS3 strain. Moreover, it could be shown that the wild type, as expected, does not show any presence of sucrose in the corresponding medium. This phenomenon can be explained by the heterologous sucrose transporter CscB in the <I>S. elongatus</I> PCC 7942 <I>cscB</I>:::NS3 strain<A href="https://academic.oup.com/pcp/article/57/12/2451/2528228"><SUP>7</SUP></A>. Upon induction, sucrose export is stimulated in the cells harboring this transporter. Wild type cells, which lack the transporter, cannot secrete sucrose in this manner. As a consequence, no sucrose is detectable in the medium. The amounts of sucrose secreted are comparable to literature values and even though the CscB transporter is inducible, low amounts of sucrose are exported when not induced by NaCl and IPTG due to promoter leakiness, which is consistent with our data<A href="https://academic.oup.com/pcp/article/57/12/2451/2528228"><SUP>7</SUP></A>.
</P></DIV><DIV class="callout"><H2>Growth of <NOBR><I>S. cerevisiae</I></NOBR> on Phosphite Media</H2><P>In our three-way co-culture, we want to use phosphite as a non-metabolizable phosphorus source where only our engineered <NOBR><I>S. cerevisiae</I></NOBR> strain is able to convert it to phosphate for itself, as well as the other organisms.
To test if our construct with the codon optimized <I>ptxD</I> gene (<I>ptxD</I>_opt)<A href="https://www.sciencedirect.com/science/article/pii/S016816561400193X"><SUP>8</SUP></A> works, we performed a plate reader experiment over 52 hours with different M2 media characteristics. <NOBR><I>S. cerevisiae</I></NOBR> and <I>E. coli</I> , both used as negative control, were cultivated in standard M2 medium<A href="https://www.sciencedirect.com/science/article/pii/S1096717617301763"><SUP>10</SUP></A> with 1.5% glucose, 0.5% ammonium sulfate, histidine <NOBR>(1.56 mg/l)</NOBR>, leucine (380 mg/l), lysine (1.52 mg/l) and uracil (18 mg/l). For some experiments, M2 medium was modified to contain phosphite (also known as phosphonic acid) instead of the originally used phosphoric acid. The same supplements were used to ensure that the media only differed in the phosphorus source. Medium lacking uracil was used in the samples containing our construct in order to maintain selection pressure. Five different constitutive promoters were tested. All samples were measured every <NOBR>30 minutes</NOBR> in replicates of five. The sample size was 200 µl. At each measurement point, the OD<SUB>600</SUB> and temperature were measured. The experiment was performed at room temperature, while the plates were shaken vigorously. </P><FIGURE><FIGCAPTION><STRONG>Figure 1: Growth of S. cerevisiae BY4742 (blue) and S. cerevisiae BY4742 with the ptxD_opt and TDH3 (yellow)</STRONG> on M2 with 1.5% glucose, 0.5% ammonium sulfate, histidine (1.56 mg/l), leucine (380 mg/l) and  lysine (1.52 mg/l) and phosphite instead of phosphate phosphite over 52 h, measured with OD<SUB>600</SUB>. For the not modified S. cerevisiae BY4742 uracil (18 mg/l) were added. As expected and the proof of the function of the ptxD_opt gene the modified strain of S. cerevisiae BY4742 grow on the media, while S. cerevisiae BY4742 as progenitor first grow but than the population decrease.</FIGCAPTION></FIGURE><P>
In Figure 1 the graph demonstrates an initial growth of the progenitor strain <I>S. cerevisiae</I>, but the growth stops at an OD<SUB>600</SUB> of less than 0.03 after <NOBR>10 hours</NOBR>. After 10 hours <I>S. cerevisiae</I> starts to decrease, in the beginning slowly, but in the end after over 50 hours a faster decrease is recognisable. At the end of the experiment the progenitor strain has an OD<SUB>600</SUB> of less than 0.02.
The modified <I>S. cerevisiae</I> strain with <I>ptxD</I>_opt and TDH3 starts nearly at the same ODy<SUB>600</SUB> as the progenitor strain but after a short lag phase of around 5 hours, the strain grows, slowly in the beginning and a bit faster in the end, where it reaches an OD<SUB>600</SUB> of over 0.02.
Both strains start at the same OD<SUB>600</SUB> level. The not modified strain grows initially faster than the modified strain, but then decreases more and more, while the modified strain needs some time but then shows a slowly rise of growth.



</P><H4>Conclusions</H4><P>
Stable dependencies between organisms are often based on nutrient exchange. Phosphorus is a macro element essential for microorganisms. Creating a dependency based on the ability of <NOBR><I>S. cerevisiae</I></NOBR> to utilize an otherwise unusable phosphorus source like phosphite by expressing <I>ptxD</I>, thereby making it metabolically available for <NOBR><I>E. coli</I></NOBR> is therefore a very promising approach.

In general the growth was minimal, so for further experiments, higher concentrations of nutrients and phosphite might lead to better results. The slow growth rate of the modified strain is as expected, because it is comparable to what the literature shows. There, a <I>ptxD</I>construct with the TEF1 promoter in <I>S. cerevisiae</I> was used and growth was monitored over 40 hours<A href="http://science.sciencemag.org/content/353/6299/583 "><SUP>11</SUP></A>. Moreover, a longer measurement time would also show the behavior of the culture over a longer time. This would be interesting because we would like to create a stable culture which can be maintained as long as possible. In addition, co-culture experiments could lead to other results than monocultures. In this case it would be interesting to perform them as well with the same experimental design. For a future application in the co-culture, we suggest to use the phosphate exporter XPR1 from <I>Homo sapiens</I><A href="https://www.sciencedirect.com/science/article/pii/S2211124713002684"><SUP>10</SUP></A>,<A href="https://www.nature.com/articles/ng.3289"><SUP>12</SUP></A>. It may help to supply the other organisms, due to the secretion of the produced phosphate.


            </P></DIV><BUTTON class="accordion"><H2>References</H2></BUTTON><DIV class="panel"><OL type="1"><LI>Scott, Spencer R., et al. &quot;A stabilized microbial ecosystem of self-limiting bacteria using synthetic quorum-regulated lysis.&quot; <I>Nature microbiology</I> 2.8 (2017): 17083.</LI><LI>Scott, Spencer R., and Jeff Hasty. &quot;Quorum sensing communication modules for microbial consortia.&quot; <I>ACS synthetic biology</I> 5.9 (2016): 969-977
                </LI><LI>Bernhardt, Thomas G., Douglas K. Struck, and R. Y. Young. &quot;The lysis protein E of φX174 is a specific inhibitor of the MraY-catalyzed step in peptidoglycan synthesis.&quot; <I>Journal of Biological Chemistry</I> 276.9 (2001): 6093-6097.
                </LI><LI>UniProt “ Lysis Protein E”. UniProt Consortium (2018).</LI><LI>Johns, Nathan I., et al. &quot;Principles for designing synthetic microbial communities.&quot; <I>Current opinion in microbiology</I> 31 (2016): 146-153.
</LI><LI>YSI Inc.”2950D Biochemistry Analyzer” (2018).</LI><LI>Abramson, Bradley W., et al. &quot;Increased photochemical efficiency in cyanobacteria via an engineered sucrose sink.&quot; <I>Plant and Cell Physiology</I> 57.12 (2016): 2451-2460.
                </LI><LI>Kanda, Keisuke, et al. &quot;Application of a phosphite dehydrogenase gene as a novel dominant selection marker for yeasts.&quot; <I>Journal of biotechnology</I> 182 (2014): 68-73.
                </LI><LI>Weiss, Taylor L., Eric J. Young, and Daniel C. Ducat. &quot;A synthetic, light-driven consortium of cyanobacteria and heterotrophic bacteria enables stable polyhydroxybutyrate production.&quot; <I>Metabolic engineering</I> 44 (2017): 236-245.
                </LI><LI>Shaw, A. Joe, et al. &quot;Metabolic engineering of microbial competitive advantage for industrial fermentation processes.&quot; <I>Science</I> 353.6299 (2016): 583-586.
                </LI><LI>Giovannini, Donatella, et al. &quot;Inorganic phosphate export by the retrovirus receptor XPR1 in metazoans.&quot; <I>Cell reports</I> 3.6 (2013): 1866-1873.
                </LI><LI>Legati, Andrea, et al. &quot;Mutations in XPR1 cause primary familial brain calcification associated with altered phosphate export.&quot; <I>Nature genetics</I> 47.6 (2015): 579.
                </LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>