<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Duesseldorf_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Duesseldorf/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="sidebar"><UL><LI><A href="https://2018.igem.org/Team:Duesseldorf">Home</A></LI><UL class="sub"><A href="#!">Team</A><LI><A href="https://2018.igem.org/Team:Duesseldorf/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Ontour">On Tour</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Attributions">Attributions</A></LI></UL><UL class="sub"><A href="#!">Project</A><LI><A href="https://2018.igem.org/Team:Duesseldorf/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Improve">Improve</A></LI></UL><UL class="sub"><A href="#!">Parts</A><LI><A href="https://2018.igem.org/Team:Duesseldorf/Parts">Parts Overview</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Composite_Part">Composite Part</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Part_Collection">Part Collection</A></LI></UL><LI><A href="https://2018.igem.org/Team:Duesseldorf/Safety">Safety</A></LI><UL class="sub"><A href="#!">Human Practices</A><LI><A href="https://2018.igem.org/Team:Duesseldorf/Silver">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/HP/Gold_Integrated">Integrated Practices</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Newsletter">Newsletter</A></LI></UL><UL class="sub"><A href="#!">Awards</A><LI><A href="https://2018.igem.org/Team:Duesseldorf/Medals">Medal Criteria</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Plant">Plant Synthetic Biology</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Model">Model</A></LI></UL><LI><A href="https://igem.org/2018_Judging_Form?team=Duesseldorf">Judging Form</A></LI></UL><UL class="sub"><A href="#!">Advice</A><LI><A href="https://2018.igem.org/Team:Duesseldorf/MoClo">Modular Cloning</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/CyanoQandA">Cyano Q&amp;A</A></LI></UL></DIV><NAV><UL class="menu"><LI class="menu-item-has-children"><A href="#!">Team</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Ontour">On Tour</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Attributions">Attributions</A></LI></UL><LI class="menu-item-has-children"><A href="#!">Project</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Improve">Improve</A></LI><LI class="menu-item-has-children"><A href="#!">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Parts">Parts overview</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Composite_Part">Composite Part</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Part_Collection">Part Collection</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Safety">Safety</A></LI><LI class="menu-item-has-children"><A href="#!">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Silver">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/HP/Gold_Integrated">Integrated Practices</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Newsletter">Newsletter</A></LI><LI class="menu-item-has-children"><A href="#!">Awards</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Medals">Medal Criteria</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Plant">Plant Synthetic Biology</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Model">Model</A></LI><LI class="menu-item-has-children"><A href="#!">Advice</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/MoClo">Modular Cloning</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/CyanoQandA">Cyano Q&amp;A</A></LI><LI><A href="https://igem.org/2018_Judging_Form?team=Duesseldorf">Judging Form</A></LI></NAV><H1>PROJECT DESCRIPTION</H1><H2>The Idea</H2><P class="details"> Scientists from all over the world and different fields are using microorganisms for their research - that is nothing new. But most of them are using monocultures and therefore miss hundreds and more possibilities which arise in co-cultures. It is estimated that less than 1% of all microorganisms have been successfully cultivated<A href="https://www.ncbi.nlm.nih.gov/pubmed/26586404"><SUP>1</SUP></A>. A reason for this might be the conditions in the laboratory. In nature no organism is completely isolated, but lives in very complex systems. Scientist have only just begun to investigate the complex interaction between different microorganisms but often fail at the cultivation. Easily obtainable and stable co-cultures would allow research of yet uninvestigated species and might give further insight into cell-cell-interactions between microorganisms<A href="https://www.ncbi.nlm.nih.gov/pubmed/24829281"><SUP>2</SUP></A>. 
One problem of isolated cultures is the regular use of antibiotics as selection marker and to prevent contaminations which leads to multiple resistances. Here as well, a co-culture with nutrient dependencies instead of higher antibiotic concentration is a better alternative.
Pharmacotherapeutic companies spend a lot of money on the development of new antibiotics, while in co-culturing conditions some microorganisms show differential gene expression which might lead to the production of new antibiotics<A href="https://www.ncbi.nlm.nih.gov/pubmed/29121465"><SUP>3</SUP></A>. In general, co-cultures might provide a great opportunity as a new method for cultivation and the production of new antibiotics or industrially interesting products. There is a lot of potential for new co-cultures and we - team iGEM 2018 HHU - want to tackle these challenges. 
       </P><H2>The Reason</H2><P class="details">  Besides all the scientific advantages, why do we need co-cultures?
A rising field in medical research is the correlation between gut microbiota and several diseases. Some of the most prominent examples are irritable bowel syndrome (IBS), Crohn’s disease and even colon cancer. But proving such a correlation is often only the first step. Afterwards doctors have to search for ways to use this knowledge to find new treatment options for these diseases.
A possible way to treat them is the use of antibiotics. But antibiotic treatment does not only affect the `healthy` microbiome but can also lead to unwanted resistances against antibiotics in some pathogens. One alternative is the use of probiotics to influence bacterial populations. Medical probiotics are living bacterial cultures that are taken orally in a stomach acid resistant capsule. As of now it is common to use monocultures, which are then separately and sterilely prepared and mixed in defined ratios. This makes it a complex and time-consuming process. Using our toolbox to design a stable co-culture, companies could save a lot of money and time by preparing their product in one culture. Since the fermenters don’t have to be opened that often and the organisms depend on each other, the contamination risk is lower. Furthermore, doctors can choose the organisms to design the co-culture as needed by the patient. This is important as it follows the rising trend of personalized healthcare.  
But our co-culture could not only have a great effect on the application of microorganisms in medicine, but also in basic research or the biotechnological production of new substances.
       </P><H2>The Challenges</H2><P class="details"> Before starting research, safety must be ensured. But this problem is solved through the strict rules of S1 laboratory work. At every work step we ensure that no organisms are set free and spread into the environment. Our experiments include a detailed control of the microorganisms, the cell density, their behavior and environmental conditions. For industrial usage the production must be scalable and high throughput has to be guaranteed as well. Furthermore, general technologies for a normed use for research projects that are also easily applicable for large scale production must be achieved<A href="https://www.ncbi.nlm.nih.gov/pubmed/29233009"><SUP>4</SUP></A>. One goal could be the reduction and simplification of processing steps and easy-to-use protocols for stable co-cultures to make cultivation and production faster and cheaper. Other requirements for making co-cultures a profitable alternative to monocultures are greater product diversities and efficient isolation of them. For scientists the possibility to build complex systems and methods for big data collection are desirable and greatly demanded.
</P><H2>The Project: &quot;Trinity&quot;</H2><P class="details"> Since our co-cultures should be universally usable by laboratories all over the world, we plan to design it as user-friendly and modular as possible. Therefore, the Golden Gate modular cloning standard techniques (MoClo) were used besides Gibson Assembly for our cloning steps, which allows exchange of all parts to adjust the co-culture to every possible condition<A href="https://www.ncbi.nlm.nih.gov/pubmed/25871405"><SUP>5</SUP></A><SUP>,</SUP><A href="https://www.ncbi.nlm.nih.gov/pubmed/26479688"><SUP>6</SUP></A>.  
We envision a standardised system, a three-way co-culture with a selection of organisms and regulations. `Trinity` is going to incorporate different dependencies between the organisms in order to control their growth and enable them to adjust each other’s behaviour. 
Project Trinity is divided into three systems which use different approaches to regulate cell density.
Our first system is utilizing the quorum sensing mechanism to regulate cell density in the population by using bacterial communication molecules. <I>E. coli</I> regulates its own cell density by expressing a lysis gene after induction by AHL1 quorum sensing molecules<A href="https://www.ncbi.nlm.nih.gov/pubmed/11459062"><SUP>11</SUP></A>. For <I>S. cerevisiae</I> the system depends on the design of a synthetic promoter that activates another lysis gene after the recognition of AHL2 molecules<A href="https://www.nature.com/articles/nmicrobiol201783"><SUP>12</SUP></A>.
The aim of our second approach is to achieve the dependency of <I>E. coli</I> and <I>S. cerevisiae</I>, through the exploitation of auxotrophies. Here <I>S. cerevisiae</I> is auxotrophic for lysine, which is produced by <I>E. coli</I><A href="https://www.ncbi.nlm.nih.gov/pubmed/1906065"><SUP>10</SUP></A>. At the same time, <I>E. coli</I> is auxotrophic for leucine, which is provided by <I>S. cerevisiae</I>. This way <I>E. coli</I> and <I>S. cerevisiae</I> are dependent upon each other. 
Our third system, also called “Nutrient System”,   deals with the dependence of the three organisms <I>Escherichia coli</I>, <I>Saccharomyces cerevisiae</I> and <I>Synechococcus elongatus</I> sp. PCC 7942 based on the availability of the essential nutrients nitrogen<A href="https://www.ncbi.nlm.nih.gov/pubmed/27493184"><SUP>7</SUP></A>, phosphate<A href="https://www.ncbi.nlm.nih.gov/pubmed/24786825"><SUP>8</SUP></A>and carbon<A href="https://www.ncbi.nlm.nih.gov/pubmed/20363793"><SUP>9</SUP></A>.
</P><BUTTON class="accordion"><H3>System 1</H3><P>
The first approach is utilizing quorum sensing mechanisms to regulate cell density in the population by using bacterial communication molecules.
</P></BUTTON><DIV class="panel"><H3>Introduction</H3><P><I>E. coli</I>:
For cell-density control, <I>E. coli</I> harbors the gene <I>luxI</I> encoding for LuxI, producing AHL constantly and the gene <I>luxR</I> which continuously produces the LuxR protein. This protein recognizes the AHL molecule and then activates the P<SUB><I>lux</I></SUB> promoter, which expresses the lysis gene <I>phiX174E</I>. Thus the population reduces itself after a certain threshold is crossed. <I>S. cerevisiae:</I>
For <I>S. cerevisiae</I> we will create a synthetic promoter, which makes the yeast compatible to the bacterial quorum sensing system. The synthetic promoter is activated by LuxR after binding the AHL2 molecule and then activates the lysis gene.

<FIGURE><FIGCAPTION><STRONG>Visual representation of the lysis gene induced by synthesis of the quorum sensing molecule acyl homoserine lactone (AHL)</STRONG> This way <I>E. coli</I> cells are supposed to regulate their own cell density</FIGCAPTION></FIGURE></P></DIV><BUTTON class="accordion"><H3>System 2</H3><P>
The aim of our second approach is the dependency of <I>E. coli</I> and <I>S. cerevisiae</I> on each other, achieved through the exploitation of auxotrophies.
</P></BUTTON><DIV class="panel"><H3>Introduction</H3><P><I>E. coli</I>:
Leucine auxotrophic <I>E. coli</I> is engineered to produce lysine for <I>S. cerevisiae</I>. For optimization of lysine production a non-feedback inhibiting gene <I>lysC</I> from <I>C. glutamicum</I> and the gene <I>ddh</I> are introduced to attain the highest possible lysine yield. This enables the auxotrophic character as a dependency as well as it being a selection marker. <I>S. cerevisiae:</I>
Is auxotroph for lysine, thus dependent on the production of lysine by <I>E. coli</I>, but on the other hand produces leucine for <I>E. coli</I>. Therefore <I>LEU2</I> is overexpressed to reach a sufficient concentration of leucine for <I>E. coli</I> to grow.

<FIGURE><FIGCAPTION>The exchange of lysine and leucine of <I>S. cerevisiae</I> and <I>E. coli</I></FIGCAPTION></FIGURE></P></DIV><BUTTON class="accordion"><H3>System 3</H3><P>
Our main system, called “Nutrient System”, deals with the dependence of three organisms on the availability of essential nutrients.
</P></BUTTON><DIV class="panel"><H3>Introduction</H3><P><I>E. coli</I>:
A synthetic cluster of 6 genes from different organisms (<I>Acidovorax avenae</I>, <I>E. coli</I>, <I>Pseudomonas sp.</I>, <I>Rhodococcus sp.</I>, S. <I>cerevisiae</I>) is used for the cleavage of melamin to ammonia and carbon dioxide.  <I>S. cerevisiae:</I>
With the gene <I>ptxD</I> from <I>P. putida</I>, <I>S. cerevisiae</I> is able to convert phosphite, which is unusable for most organisms, into phosphate. <I>S. elongatus</I>
The cyanobacterium converts sucrose, which is naturally produced by photosynthesis, into fructose and glucose by an invertase encoded by the gene <I>invA</I>. Glucose will be secreted into the media with the help of an exporter, encoded by <I>glf</I>. These genes are taken from the <I>Z. mobilis</I> genome.

<FIGURE><FIGCAPTION>All three organisms are dependend on each other by providing essentiell nutrients to the other both.</FIGCAPTION></FIGURE></P></DIV><BUTTON class="accordion"><H2>References</H2></BUTTON><DIV class="panel"><OL type="1"><LI>Katz, Micah, Bradley M. Hover, and Sean F. Brady. &quot;Culture-independent
   				 discovery of natural products from soil metagenomes.&quot; <I>Journal of industrial
   				 microbiology &amp; biotechnology</I> 43.2-3 (2016): 129-141.
   			 </LI><LI>Goers, Lisa, Paul Freemont, and Karen M. Polizzi. &quot;Co-culture systems and
   				 technologies: taking synthetic biology to the next level.&quot; <I>Journal of The
   				 Royal Society Interface</I> 11.96 (2014): 20140065.
   			 </LI><LI>Adnani, Navid, et al. &quot;Coculture of Marine Invertebrate-Associated
   				 Bacteria and Interdisciplinary Technologies Enable Biosynthesis and
   				 Discovery of a New Antibiotic, Keyicin.&quot; <I>ACS chemical biology</I> 12.12
   				 (2017): 3093-3102.</LI><LI>Padmaperuma, Gloria, et al. &quot;Microbial consortia: a critical look at
   				 microalgae co-cultures for enhanced biomanufacturing.&quot; <I>Critical reviews in
   				 biotechnology</I> 38.5 (2018): 690-703.
   			 </LI><LI>Lee, Michael E., et al. &quot;A highly characterized yeast toolkit for modular,
   				 multipart assembly.&quot; <I>ACS synthetic biology</I> 4.9 (2015): 975-986.
   			 </LI><LI>Iverson, Sonya V., et al. &quot;CIDAR MoClo: improved MoClo assembly standard
   				 and new <I>E. coli</I> part library enable rapid combinatorial design for
   				 synthetic and traditional biology.&quot; <I>ACS synthetic biology</I> 5.1 (2015):
   				 99-103.
   			 </LI><LI>Shaw, A. Joe, et al. &quot;Metabolic engineering of microbial competitive
   				 advantage for industrial fermentation processes.&quot; <I>Science</I> 353.6299
   				 (2016): 583-586.
   			 </LI><LI>Kanda, Keisuke, et al. &quot;Application of a phosphite dehydrogenase gene
   				 as a novel dominant selection marker for yeasts.&quot; <I>Journal of
   				 biotechnology</I> 182 (2014): 68-73.
   			 </LI><LI>Niederholtmeyer, Henrike, et al. &quot;Engineering cyanobacteria to synthesize
   				 and export hydrophilic products.&quot; <I>Applied and environmental microbiology</I>
   				 76.11 (2010): 3462-3466.
   			 </LI><LI>Schrumpf, Barbel, et al. &quot;A functionally split pathway for lysine
   				 synthesis in <I>Corynebacterium glutamicium</I>.&quot; <I>Journal of bacteriology</I> 173.14
   				 (1991): 4510-4516.
   			 </LI><LI>Dong, Yi-Hu, et al. &quot;Quenching quorum-sensing-dependent bacterial
   				 infection by an N-acyl homoserine lactonase.&quot; <I>Nature</I> 411.6839 (2001): 813.
   			 </LI><LI>Scott, Spencer R., et al. &quot;A stabilized microbial ecosystem of
   				 self-limiting bacteria using synthetic quorum-regulated lysis.&quot;
   				 <I>Nature microbiology</I> 2.8 (2017): 17083.
   			 </LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>