<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Duesseldorf_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Duesseldorf/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="sidebar"><UL><LI><A href="https://2018.igem.org/Team:Duesseldorf">Home</A></LI><UL class="sub"><A href="#!">Team</A><LI><A href="https://2018.igem.org/Team:Duesseldorf/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Ontour">On Tour</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Attributions">Attributions</A></LI></UL><UL class="sub"><A href="#!">Project</A><LI><A href="https://2018.igem.org/Team:Duesseldorf/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Improve">Improve</A></LI></UL><UL class="sub"><A href="#!">Parts</A><LI><A href="https://2018.igem.org/Team:Duesseldorf/Parts">Parts Overview</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Composite_Part">Composite Part</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Part_Collection">Part Collection</A></LI></UL><LI><A href="https://2018.igem.org/Team:Duesseldorf/Safety">Safety</A></LI><UL class="sub"><A href="#!">Human Practices</A><LI><A href="https://2018.igem.org/Team:Duesseldorf/Silver">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/HP/Gold_Integrated">Integrated Practices</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Newsletter">Newsletter</A></LI></UL><UL class="sub"><A href="#!">Awards</A><LI><A href="https://2018.igem.org/Team:Duesseldorf/Medals">Medal Criteria</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Plant">Plant Synthetic Biology</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Model">Model</A></LI></UL><LI><A href="https://igem.org/2018_Judging_Form?team=Duesseldorf">Judging Form</A></LI></UL><UL class="sub"><A href="#!">Advice</A><LI><A href="https://2018.igem.org/Team:Duesseldorf/MoClo">Modular Cloning</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/CyanoQandA">Cyano Q&amp;A</A></LI></UL></DIV><NAV><UL class="menu"><LI class="menu-item-has-children"><A href="#!">Team</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Ontour">On Tour</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Attributions">Attributions</A></LI></UL><LI class="menu-item-has-children"><A href="#!">Project</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Improve">Improve</A></LI><LI class="menu-item-has-children"><A href="#!">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Parts">Parts overview</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Composite_Part">Composite Part</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Part_Collection">Part Collection</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Safety">Safety</A></LI><LI class="menu-item-has-children"><A href="#!">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Silver">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/HP/Gold_Integrated">Integrated Practices</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Newsletter">Newsletter</A></LI><LI class="menu-item-has-children"><A href="#!">Awards</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Medals">Medal Criteria</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Plant">Plant Synthetic Biology</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Model">Model</A></LI><LI class="menu-item-has-children"><A href="#!">Advice</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/MoClo">Modular Cloning</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/CyanoQandA">Cyano Q&amp;A</A></LI><LI><A href="https://igem.org/2018_Judging_Form?team=Duesseldorf">Judging Form</A></LI></NAV><H1>Design</H1><P>
The world is shaped by interactions between living things. Although these relationships represent one of the foundations for diversity and complexity of life, they remain largely unexplored. To investigate these networks, researchers are trying to mimic these interactions with co-cultivation.
But we do not want to mimic, we want to understand.
We are convinced that we can only get a deeper understanding of these associations if we are able to build them from bottom-up. Our modular co-cultivation toolbox is intended to contribute to this. Every scientist should be able to design and use co-cultures with different strains or organisms for diverse purposes with the help of our toolbox. But what is the basic science behind it and how does this work?
</P><H2>Scientific Background</H2><H3>The Root Problem</H3><P>
Only 1% of microorganisms have been successfully cultivated so far<A href="https://doi.org/10.1007/s10295-015-1706-6 "><SUP>1</SUP></A>, whereas the remaining 99% require conditions that are not yet reproducible in monocultures or with standard cultivation methods. 
Microorganisms in nature are mostly, if not always, found in microbial consortia such as in the human gut or in the soil. Some microorganisms may struggle to grow in monocultures due to the lack of essential benefits arising through the coexistence with other microbes that can include cross-feeding and signaling molecules, among others<A href="https://doi.org/10.1111/j.1574-6968.2010.02000.x"><SUP>2</SUP></A>.

A problem for science is the complexity of networks between microorganisms in their natural environment. Studying these naturally occurring communities is very difficult since the establishment of such a co-culture is a big challenge by itself. For every microorganism, different needs need to be met such as media composition, pH or temperature and survival of all organisms in the co-culture needs to be ensured.
 
Thus, a lot of discoveries (e.g. production of biofuels and medical applications) are still neglected. This includes research about the gut microbiome, which has an overall effect on the human health. And while their cooperative interactions are not fully understood until now, the effect of drugs on every human being with differing gut bacteria cannot be calculated yet. Measures for keeping a healthy gut microbiome often remain ineffective since they are tested on monocultures.
</P><H3>Our Approach</H3><P>
The co-cultivation of microorganisms has multiple useful applications from basic research for cell-cell interactions to medical treatments for bacterial infections or industrial production of bioproducts. A simplified modular approach to co-culturing offers potential benefits for every aspect of our society.
 
In recent years, synthetic biology picked up the problem to co-cultivate microorganisms through synthetically created dependencies. We tried to tackle this challenge by creating not only three different dependencies, but also for three different organisms. The aim of our project was to create a modular toolbox for co-culturing of <I>Escherichia coli</I>, <I>Saccharomyces cerevisiae</I> and <I>Synechococcus elongatus</I> as a big advancement towards possible co-culturing between all known organisms.
 
With these organisms our team engineered a system based on nutrient exchange. Here phosphate is provided through oxidation of phosphite, nitrogen source produced by melamine breakdown, whilst a carbon source is provided by <NOBR><I>S. elongatus</I></NOBR> through photosynthesis. Two additional independent approaches were also added to the toolbox. The first includes regulation via cross-feeding by amino acid auxotrophies and production: lysine by <I>E. coli</I> and leucine by <NOBR><I>S. cerevisiae</I></NOBR>. The other utilizes regulated self-lysis via quorum sensing molecules, to control cell density by a phage lysis gene.
</P><BUTTON class="accordion"><H2>Cloning Strategies via Toolboxes</H2></BUTTON><DIV class="panel"><P>
To make our co-culture toolbox as modular as possible, we used Golden Gate cloning<A href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0005553"><SUP>3</SUP></A><SUP>,</SUP><A href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0003647"><SUP>4</SUP></A> to design constructs with exchangeable parts in order to easily customize them to the needed co-culture. Therefore, we decided to build our systems with already established toolboxes: the YTK toolbox<A href="https://pubs.acs.org/doi/abs/10.1021/sb500366v"><SUP>5</SUP></A> established in <A href="http://dueberlab.berkeley.edu/">John Dueber's laboratory</A> for <I><NOBR>S. cerevisiae</NOBR></I> and the CIDAR toolbox<A href="https://pubs.acs.org/doi/abs/10.1021/acssynbio.5b00124"><SUP>6</SUP></A> for <I>E. coli</I>.
 
The advantage of Golden Gate cloning is the usage of type II S restriction enzymes that do not cleave directly at the recognition site, but at a specific number of base pairs up - or downstream. This allows the design of scarless constructs and specific overhangs. As a result, different parts are assembled only in the correct order. In consequence, one-pot reactions with several parts are possible.
 
The YTK (Yeast toolkit) established by the Duber laboratory<A href="https://pubs.acs.org/doi/abs/10.1021/sb500366v"><SUP>5</SUP></A> is built up in different levels.
</P><FIGURE><FIGCAPTION><STRONG>Figure 1: </STRONG>General workflow using the YTK toolbox to create a plasmid with one or more transcription units<A href="https://pubs.acs.org/doi/abs/10.1021/sb500366v"><SUP>5</SUP></A>.</FIGCAPTION></FIGURE><P>
All backbones are cassettes that contain different antibiotic resistances, origin of replication and drop-outs. The backbones are chosen that best fit the cloning.
The level 0 backbone serves as saving and transporting the inserted gene. The gene of interest is cloned into the so called entry vector pYTK001 using the enzyme BsmBI. For this to work, recognition sites of BsmBI have to flank the gene and the overhangs must be designed to fit to overhangs of the restriction sites of the entry vector. The entry plasmid contains a chloramphenicol resistance and an origin of replication but no promoter or terminator so there is no expression of the insert.
 
The level 1 vector follows for cloning an expression plasmid for yeast. It always includes an appropriate promoter and terminator for the gene to be ready to be cloned into <I>S. cerevisiae</I>.  Therefore, a promoter, a gene of interest and a terminator are chosen from the toolbox, where over 20 promoters - constitutive and inducible - and different terminators are selectable. In contrast to the level 0 plasmid, the level 1 vectors contains an ampicillin resistance and recognition sites for the restriction enzyme BsaI.
 
With a GFP dropout system the cloning is facilitated. A <I>gfp</I> gene is located between the restriction sites in the backbone and will be replaced if the gene of interest is correctly inserted. In case of non-recombinant bacteria due to wrong ligation or failed cloning, the colonies show a green color.
 
In case it is required to express more than one gene in a yeast strain, a level 2 plasmid is available as well which can contain up to three transcriptional units.

For <I>E. coli</I>, the CIDAR toolbox of Iverson and colleagues is used for modular cloning with Golden Gate<A href="https://pubs.acs.org/doi/abs/10.1021/acssynbio.5b00124"><SUP>6</SUP></A>. The concept is analogous to the YTK.
</P><FIGURE><FIGCAPTION><STRONG>Figure 2: </STRONG>General workflow using the CIDAR toolbox to create a plasmid with one or more TUs<A href="https://pubs.acs.org/doi/abs/10.1021/acssynbio.5b00124"><SUP>6</SUP></A>.</FIGCAPTION></FIGURE><P>
To generate a level 0 plasmid, the gene of interest is flanked by BbsI recognition sites with fitting overhangs and is cloned into an entry vector with ampicillin resistance.
The level 1 plasmid is a functional <I>E. coli</I> expression plasmid. For this vector, ten different constitutive or controllable promoters, six different ribosome binding sites (RBS) and a terminator can be found in the toolbox. The plasmid contains a kanamycin resistance and recognition sites for the enzyme BsaI. 
In the CIDAR toolbox a multi-gene assembly plasmid with a maximum of four transcriptional units is possible as well in the level 2 plasmid. Just like the level 1 plasmid, the encoding resistance is ampicillin and the enzyme BbsI is used for restriction.

For all cloning steps, blue white screening can be used for the identification of recombinant bacteria. Therefore, the backbone contains <I>lacZ</I> which encodes ß-galactosidase with an IPTG inducible promoter at the insert point to help find positive colonies. ß-galactosidase hydrolyzes X-Gal into a molecule which spontaneously oxidizes into a blue pigment. Thus, agar plates with IPTG and X-Gal are used<A href="https://www.intechopen.com/books/molecular-cloning-selected-applications-in-medicine-and-biology/screening-of-bacterial-recombinants-strategies-and-preventing-false-positives "><SUP>7</SUP></A>.
After incubation, the colonies show either a white or a blue color. Blue colonies only contain the backbone whereas white colonies carry the gene of interest instead of the functional <I>lacZ</I> gene. 
</P></DIV><H2>Design of our Sub-projects</H2><BUTTON class="accordion"><H2>System 1: Self Regulating Microorganisms</H2><P>Before establishing a co-culture, we started with a bottom-up approach which regulates the growth of an organism by its own cell population. 
</P></BUTTON><DIV class="panel"><H3>Self-lysing Microorganisms</H3><P>
Growing cultures containing more than one organism without one or more of the populations dying out is a very complex task. Since our aim is not only to grow a co-culture of two microorganisms but of three different subjects, we decided to develop a tool to downregulate an organism's growth in a controlled way as a first step towards a stable co-culture. This part is essential because it permits the co-existence of at least two organisms at the same time, without erasing one another due to growth disadvantages. 
For this purpose we designed two mechanisms that are supposed to downregulate the growth of the two fastest-growing organisms in our final co-culture: <I>E. coli</I> and <I>S. cerevisiae</I>. 

Our models for the two different organisms are based on the same principle: Quorum sensing. This mechanism is used by many organisms to regulate gene expression depending on cell density fluctuation. This machinery is often referred to as the communication system of bacteria and is based on the activity of so-called autoinducers, such as acyl homoserine lactones (AHLs)<A href="https://www.ncbi.nlm.nih.gov/pubmed/11544353"><SUP>8</SUP></A>. The most commonly known quorum sensing regulatory mechanism is from the bacterium <I>Vibrio fischeri</I>. Depending on the population density, the amount of autoinducers fluctuates and, when elevated, can alter gene expression on a global level - in the case of <I>V. fischeri</I>, by activating bioluminescence<A href="https://onlinelibrary.wiley.com/doi/full/10.1046/j.1365-2958.1999.01261.x"><SUP>9</SUP></A>. To apply this mechanism, we inserted a lysis gene under the control of the AHL-stimulated Lux promoter, meaning that when the quorum sensing molecule, acyl homoserine lactone synthase is produced by the <I>luxI</I> gene and then binds to the regulator LuxR, expression of the lysis gene is started. At high cell densities, this promoter is activated and the cell population is thereby simply regulated by itself.
<B>Experimental Design</B>
All our constructs are virtually cloned and adapted to the <A href="https://2018.igem.org/Team:Duesseldorf/MoClo">CIDAR and YTK toolboxes</A> that we are using. For assembly with the toolbox, each part has to be cloned first into so called level 1 plasmids. Those are then brought together to the final functional system. The use of those toolboxes does not only allow the diverse assembly of various parts, but also the exchange of different promoters, RBSs and terminators, with individual strength and preferences. 
As an example, all level 0 virtual constructs, as well as all level 1 virtual constructs of our Lux quorum sensing system are shown in Fig. 1. Essentially, the level 0 constructs represent the basic part in one of the toolbox backbones.
</P><FIGURE><FIGCAPTION><STRONG>Figure 1: Level 0 plasmids for the quorum sensing system.</STRONG>The level 0 plasmids are the first step in modular cloning with the toolboxes, as they are required for multi-part assembly. In this case the level 0 plasmids that will later be assembled with one another are presented. For the Quorum sensing system, the promoter P<SUB><I>lux</I></SUB>, as well as the acyl homoserine lactone synthase, LuxI, and the respective regulator LuxR are required. To control cell density, the lysis gene <I>E</I> from bacteriophage phiX174 is used, which is designated here as phiX174E.
        </FIGCAPTION><P>
The level 1 assemblies should be functional units, comprising promoter, RBS, gene of interest and terminator.
</P><FIGURE><FIGCAPTION><STRONG>Figure 2: Level 1 plasmids of the quorum sensing system.</STRONG>The level 1 plasmids are fully functional units, comprising promoter (green), RBS (red), gene of interest (blue) and terminator (yellow).</FIGCAPTION></FIGURE><P>
In our case, only the level 2 constructs are functional, since the quorum sensing system works only if all parts are assembled together.
</P><FIGURE><FIGCAPTION><STRONG>Visual representation of the lysis gene induced by synthesis of the quorum sensing molecule acyl homoserine lactone (AHL).</STRONG> In this way <I>E. coli</I> cells are supposed to regulate their own cell density.</FIGCAPTION></FIGURE><P>
We codon optimized the Lux System for both organisms and put the lysis gene <I>E</I> from bacteriophage phiX174 under the control of P<SUB><I>lux</I></SUB> in <I>E. coli</I><A href="https://www.nature.com/articles/nmicrobiol201783
"><SUP>10</SUP></A>. This lysis protein induces host cell lysis by inhibiting the host cell’s translocase MraY activity, which is required for the synthesis of an important cell wall component. Lack of function of this protein prevents the bacterial cell from growing properly and as a result leads to final lysis. We constructed a plasmid, containing all genes necessary for the activation of this promoter: <I>luxI</I> and <I>luxR</I>. To measure the cell population using fluorescence our construct contains a <I>gfp</I> reporter as well, therefore the activation of the lysis gene leads to the co-activation of the reporter to keep track of the cells.
</P><FIGURE><FIGCAPTION><STRONG>Figure 4:  Level 2 construct for the quorum sensing system</STRONG> containing <I>luxI</I> - coding the acyl homoserine lactone synthase - which will bind to P<SUB><I>lux</I></SUB> together with the protein synthesized by <I>luxR</I> and induces the synthesis of the <I>E</I> lysis gene. A reporter, in this case GFP, is used to track the cell population.</FIGCAPTION></FIGURE><P>
Besides this final construct, we also created different controls. This includes a plasmid containing all the genes except the <I>luxI</I>, which is the acyl homoserine lactone synthase. It was created to show that the promoter is activated and lysis or reporter expression is functional only upon artificial induction with AHL. Moreover, we also designed a plasmid which is supposed to show functionality of the P<SUB><I>lux</I></SUB> promoter, hence exchanging the lysis gene <I>E</I> with <I>gfp</I>. 
</P><FIGURE><FIGCAPTION><STRONG>Figure 5: Level 2 plasmids as controls for the quorum sensing system.</STRONG></FIGCAPTION></FIGURE></FIGURE></DIV><BUTTON class="accordion"><H2>System 2: Auxotrophy based Two-way Co-culture</H2><P> Here the modulation is based on synthesis of amino acids by auxotrophic strains and their cross feed.
</P></BUTTON><DIV class="panel"><H3>Overproduction of Leucine by <NOBR><I>S. cerevisiae</I></NOBR></H3><P>
For yeast we implement a leucine producing, but lysine-auxotrophic yeast strain.

<FIGURE><FIGCAPTION><STRONG>Figure 1: Structural formula of L-leucine</STRONG> Hydroxy- and keto goups are labeled in red, amino groups in blue and methyl groups are shown in grey.  </FIGCAPTION></FIGURE></P><P> Our initial strain is <I>S. cerevisiae</I> BY4742 <I>(MATα his3Δ1 leu2Δ0 lys2Δ0 ura3Δ0)</I>. It is a deletion strain with auxotrophies for leucine, lysine, histidine and uracil. The strain is normally used for work with numerous non-antibiotic selection markers.
Important for our use is the null mutation <I>leu2Δ0</I>. <I>LEU2</I> is an essential gene in the leucine biosynthesis and is coding for the beta-isopropylmalate dehydrogenase (IMDH). IMDH catalyzes the third step in the leucine biosynthesis pathway.</P><FIGURE><FIGCAPTION><STRONG>Figure 2: Leucine biosynthesis in <I>S. cerevisiae.</I></STRONG> Important for our approach is <I>LEU2</I> which is an essential gene in the pathway and codes for the beta-isopropylmalate dehydrogenase (IMDH). IMDH oxidates 3-isopropylmalate into ketoisocaproate through dissociation of hydrogen anions. Our origin strain had a null mutation in <I>LEU2</I> which we rescued and improved for our two-way co-culture. </FIGCAPTION></FIGURE><P><B>Experimental Design</B>
We are not only restoring its ability to produce leucine but are improving its production: To repair the leucine auxotrophy, we cloned <I>LEU2</I> with a strong constitutive promoter into the yeast cells. This way, we added the missing gene which caused the auxotrophy. With the addition of the mentioned promoter, we are overexpressing <I>LEU2</I>.
Through its originally derived auxotrophies, the yeast is lysine-auxotrophic and fits perfectly for our design.

<FIGURE><FIGCAPTION><STRONG>Figure 3: <I>LEU2</I> cloned into the YTK level 0 backbone.</STRONG> The level 0 plasmid is not used for expression. </FIGCAPTION></FIGURE><FIGCAPTION><STRONG>Figure 4: <I>LEU2</I> cloned into an YTK level 1 plasmid.</STRONG> It includes a fitting promoter and terminator for the gene to be prepared to be cloned into <I>S. cerevisiae</I>. In order to enhance the production of LEU2  we chose the strong promoter TDH3 and the standard ENO1 terminator.</FIGCAPTION></P><H3>Overproduction of Lysine in <NOBR><I>E. coli</I></NOBR></H3><P>To create an artificial dependency between <I>E. coli</I> and <I>S. cerevisiae</I> based on amino acid auxotrophies we want <I>E. coli</I> to produce and secrete lysine for the lysine auxotrophic yeast.

<FIGURE><FIGCAPTION><STRONG>Figure 5: Structural formula of L-lysine.</STRONG> Hydroxy- and keto goups are labeled in red and amino groups are shown in blue. </FIGCAPTION></FIGURE></P><P>
At the same time <I>E. coli</I> is auxotroph for the amino acid leucine. To achieve overproduction of lysine we want to make use of the protein meso-diaminopimelate D-dehydrogenase (Ddh) encoded by the gene <I>ddh</I><A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC208115/"><SUP>11</SUP></A> and a non feedback inhibited version of the enzyme aspartokinase (LysC) encoded by the gene <I>lysC</I><A href="https://link.springer.com/article/10.1007%2FBF00240726"><SUP>12</SUP></A>. The native organism of both genes is <I>Corynebacterium glutamicum</I>.
<I>ddh</I> is a multistep enzyme which catalyses the reductive amination from L-A'-piperideine-2,6-dicarboxylate to D,L-2,6-diaminopimelate. This leads to an alternative pathway for the lysine biosynthesis in <I>C. glutamicum</I> which we want to implement in <I>E. coli</I>.
Aspartokinase taken from the wild type <I>E. coli</I> is feedback inhibited by lysine. An amino acid switch from serine to phenylalanine at position 381 results in a non feedback inhibited version of LysC which leads to an improved lysine production.

<FIGURE><FIGCAPTION><STRONG>Figure 6: Lysine pathway in prokaryotes.</STRONG> To improve the lysine biosynthesis, we integrated the gene <I>lysC</I> and <I>ddh</I>, both genes originally taken from Corynebacterium glutamicum. <I>lysC</I> variant we are using is not feedback inhibited by lysine and codon optimized for<I> E. coli</I>. <I>ddh</I> is a multi-step enzyme which is able to skip five enzymatic steps in the lysine synthesis pathway from <I>E. coli</I>, as shown here. Arrows in green shows our approach with integrating <I>ddh</I> in <I>E. coli</I>. Arrows in blue show the skipped steps by the Meso-diaminopimelate D-dehydrogenase expressed by <I>ddh</I>. </FIGCAPTION></FIGURE></P><P><B>Experimental Design</B>


The gene for <I>lysC</I> was ordered codon-optimized as a gBlock, whereas the gene <I>ddh</I> was isolated from <I>C. glutamicum</I> using the Quick-DNA Miniprep Plus Kit.
The cloning was performed according to the CIDAR MoClo toolbox<A href="https://pubs.acs.org/doi/10.1021/acssynbio.5b00124 "><SUP>12</SUP></A>.
Since the gene <I>ddh</I> contains recognition sites for the two restriction enzymes BsaI and BbsI, that we use in the CIDAR MoClo toolbox cloning, we had to perform two site-directed mutagenesis. Therefore, the gene <I>ddh</I> was first stored in the TOPO cloning vector using TA cloning and afterwards mutagenized two times.
<FIGURE><FIGCAPTION><STRONG>Figure 7: Plasmid of the gene <I>ddh</I> stored in pCR2.1-Topo vector in order to perform two mutagenizations.</STRONG> AmpR and KanR are used as selection markers. </FIGCAPTION></FIGURE></P><P>
We designed six different expression plasmids for each of both genes. They differ only in the constitutive promoters we used in order decide which promoter strength is needed for the amino acid overproduction needed for the auxotrophic <NOBR><I>S. cerevisiae</I></NOBR> to survive. The used promoters are <A href="http://parts.igem.org/Part:BBa_J23100">BBa_J23100</A>, being the strongest promoter, <A href="http://parts.igem.org/Part:BBa_J23102">BBa_J23102</A>, <A href="http://parts.igem.org/Part:BBa_J23103">BBa_J23103</A>, <A href="http://parts.igem.org/Part:BBa_J23106">BBa_J23106</A>, <A href="http://parts.igem.org/Part:BBa_J23107">BBa_J23107</A> and <A href="http://parts.igem.org/Part:BBa_J23116">BBa_J23116</A>, which is the weakest promoter. The promoters are from a small combinatorial library of constitutive promoters designed by John Anderson<A href="http://parts.igem.org/Part:BBa_J23117"><SUP>13</SUP></A>. Additionally our constructs contain the ribosome binding site B0032m and the terminator B0015.
The level 0 plasmids only contain our gene of interest without the other expression factors.

<FIGURE><FIGCAPTION><STRONG>Figure 8: Level 0 plasmid for the two-way co-culture system.</STRONG> Level 0 plasmid of the CIDAR MoClo toolbox containing the gene <I>ddh</I>. AmpR is used as selection marker.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><STRONG>Figure 9: Level 0 plasmid for the two-way co-culture system.</STRONG> Level 0 plasmid of the CIDAR MoClo toolbox with the gene <I>lysC</I>. AmpR is used as selection marker./figcaption&gt;
        </FIGCAPTION></FIGURE></P><P>
To create the expression plasmids, the level 1 backbones, the level 0 plasmid with the gene of interest as well as promoter, ribosome binding site and the terminator are cut with BbsI and afterwards ligated with the level 1 backbone vector.

<FIGURE><FIGCAPTION><STRONG>Figure 10: Level 1 plasmid for the Two-way co-culture system in <I>E. coli</I>.</STRONG> The gene of interest <I>ddh</I> with promoter J23100, the ribosome binding site B0032 and the terminator B0015 in the level 1 plasmid of the CIDAR MoClo toolbox. KanR is used as selection marker.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><STRONG>Figure 11: Level 1 plasmid for the Two-way co-culture system in <I>E. coli</I>.</STRONG> The gene of interest <I>lysC</I> with promoter J23100, the ribosome binding site B0032 and the terminator B0015 in the level 1 plasmid of the CIDAR MoClo toolbox. KanR is used as selection marker.</FIGCAPTION></FIGURE></P></DIV><BUTTON class="accordion"><H2>System 3: Nutrient based Three-way Co-culture</H2><P> Co-culture with three organisms is established by reciprocal supply of vital nutrients.
</P></BUTTON><DIV class="panel"><H3>Overproduction of Sucrose by <NOBR><I>S. elongatus</I></NOBR></H3><P>
A carbon source is needed in our co-culture system for <I>E. coli</I> and <I>S. cerevisiae</I> in order to stabilize the culture and to create an autonomous system. This is why we included the cyanobacterium <I>Synechococcus elongatus</I> PCC 7942 in our three-way co-culture since it is a photosynthetically active cyanobacterium. By engineering <I>S. elongatus</I> to overproduce and secrete sucrose, we aim to create a stable co-culture that is not further dependent on carbon in the media. <I>S. cerevisiae</I> is able to convert sucrose into fructose and glucose extracellularly which can then be used as a carbon source by the <I>S. cerevisiae</I> itself<A href="https://www.sciencedirect.com/science/article/pii/037810979390598V"><SUP>13</SUP></A> or the <I>E. coli</I>. 

Many cyanobacteria are able to overproduce substances like sugars in order to prevent the cell from osmotic stress<A href="https://aem.asm.org/content/84/2/e02023-17.long"><SUP>14</SUP></A>.
In addition it could be shown that upon induction of the symporter CscB, <I>S. elongatus</I> is able to produce higher amounts of sugar. <I>S. elongatus</I> produces sucrose in several steps from Glucose-1-P. The first rate limiting step in the pathway for the sucrose production is catalyzed via sucrose-phosphate synthase (Sps) and is induced by salt stress<A href="https://aem.asm.org/content/84/2/e02023-17.long"><SUP>15</SUP></A>.
We obtained a specific strain published by Dr. Daniel Ducat et al. which is able to secrete sucrose with the help of the CscB sucrose permease. The <I>cscB</I> gene encodes a symporter which is inducible by IPTG<A href="https://www.sciencedirect.com/science/article/pii/037810979390598V "><SUP>13</SUP></A>. In this case we plan to induce sucrose production with NaCl and IPTG. Furthermore, advice for cultivation media for the co-culturing experiments were communicated by Daniel Ducat. This is also the foundation for the Integrated Human Practice.

Our goal is to establish a system which allows the production and the secretion of sucrose via the induction with only IPTG instead of NaCl. With this approach we want to avoid salt stress for the culture.
<B>Experimental Design</B>
We are cloning an expression plasmid which contains an iGEM provided IPTG inducible promoter (BBa_R0010, termed P<I>lac</I> here) and <I>sps</I> (sucrose phosphate synthase) as the gene of interest. The origin species of <I>sps</I> we used is <I>Synechocystis</I> PCC 6803<A href="https://www.sciencedirect.com/science/article/pii/037810979390598V "><SUP>13</SUP></A>. This variant of sucrose phosphate synthase needs less NaCl in order to be activated, compared to the variant already found in <I>S. elongatus</I><A href="https://aem.asm.org/content/84/2/e02023-17.long"><SUP>14</SUP></A>.
We want to insert the plasmid into our strain of <I>S. elongatus</I> by conjugation. We hypothesize that the induction with IPTG is enough to induce the production and the secretion of sucrose. 
<FIGURE><FIGCAPTION><STRONG>Figure 1: Schematic overview of the experimental design for the production of sucrose.</STRONG><I>S. elongatus</I> PCC7942 <I>cscB</I>:::NS3 with pSHDY_<I>sps</I>.</FIGCAPTION></FIGURE></P><P>
For the cloning strategy of our expression system, Gibson Assembly was chosen. The used backbone is called pSHDY, which was created and provided by an advisor of our team. pSHDY is a conjugative shuttle vector with a broad host range which is commonly used as a plasmid-based expression system in cyanobacteria. Selection markers are antibiotic resistances - in this case kanamycin and spectinomycin. The strain we are working with provides a genomically integrated chloramphenicol resistance which also allows the specific selection of positively conjugated clones. For the uptake of plasmid DNA, the method triparental mating<A href="https://www.protocols.io/view/triparental-mating-of-synechocystis-ftpbnmn"><SUP>17</SUP></A> was used.</P><FIGURE><FIGCAPTION><STRONG>Figure 2 : Plasmid for conjugation.</STRONG> It is a pSHDY backbone containing <I>sps</I> and P<SUB>lac</SUB>.</FIGCAPTION></FIGURE><H3>Production of Phosphate from Phosphite by <NOBR><I>S. cerevisiae</I></NOBR></H3><P>
For our nutrient based dependency system we engineered <I>S. cerevisiae</I> to provide phosphate as a phosphorus source for the other two organisms.
A recent approach by Kanda et al<A href="https://www.sciencedirect.com/science/article/pii/S016816561400193X?via%3Dihub"><SUP>18</SUP></A> for a new selective marker in yeast is based on the oxidation of phosphite (P<SUB>T</SUB>) to metabolically useable phosphate (P<SUB>I</SUB>). The used phosphonate dehydrogenase encoded by <I>ptxD</I> caught our interest in using phosphite as a phosphorus source for our co-culture.

<FIGURE><FIGCAPTION><STRONG>Figure 3 : Oxidation reaction of phosphite to phosphate catalysed by a phosphonate dehydrogenase encoded by <I>ptxD</I></STRONG> The reaction needs NAD<SUP>+</SUP> and H<SUB>2</SUB>O and creates NADH as a byproduct. Phosphorus is labeled in orange, oxygen and hydroxyl groups are red-labeled.</FIGCAPTION></FIGURE></P><P><B>Experimental Design</B>
Based on the work of Kanda and colleagues a codon-optimized version of ptxD for <I>S. cerevisiae</I> was ordered from IDT. Additionally Golden Gate assembly overhangs based on the design by Lee et al<A href="https://pubs.acs.org/doi/10.1021/sb500366v"><SUP>19</SUP></A> in the pYTK MoClo Golden Gate assembly toolbox a type 3 CDS were added as well as biobrick assembly prefix and suffix at the 3` and 5` end of the sequence.
Concordant to the pYTK cloning workflow the optimized <I>ptxD</I> (<I>ptxD</I>_opt) was inserted into a level 0 entry vector


<FIGURE><FIGCAPTION><STRONG>Figure 4 : Level 0 entry vector with codon optimized <I>ptxD</I> (blue) inserted</STRONG> a ColE1 ori and CamR as selective marker.</FIGCAPTION></FIGURE></P><P>
These level 0 vectors were then used to create yeast expression vectors. 

<FIGURE><FIGCAPTION><STRONG>Figure 5 : Expression plasmid with pTDH3 promoter region, <I>ptxD</I>_opt CDS and tPGK1 terminator.</STRONG> The plasmid carries the yeast 2micron ori and the <I>E. coli</I> ColE1 ori. As a selective marker AmpR (<I>bla</I>) in bacteria and URA3 in yeast were added.</FIGCAPTION></FIGURE></P><P>
The expression plasmids always carried the <I>ptxD</I>_opt and the <I>tPGK1</I> terminator sequence as a fixed part of the newly assembled transcription units. For the promoter sequence five different constitutive yeast promoters were used to create 5 different expression plasmids respectively. The five used promoter regions are <I>pTDH3, pCCW12, pPGK1, pHHF2</I> and <I>pTEF1</I>. 
</P><H3>Production of Ammonia from Melamine by <NOBR><I>E. coli</I></NOBR></H3><P>
In order to create an artificial dependency for our three-way co-culture based on cross-feeding of essential nutrients, we developed the idea to engineer <I>E. coli</I> to supply the other two organisms with ammonia. 
While researching for papers and taking a look at several already established ammonia pathways we came across an idea <A href="”http://science.sciencemag.org/content/353/6299/583.long”">published by Shaw and colleagues  in 2016</A>  who suggested a pathway cluster made of six enzymes from different organisms for the conversion of melamin into ammonia<A href="http://science.sciencemag.org/content/353/6299/583.long "><SUP>20</SUP></A>. We implemented this melamine pathway cluster in <I>E. coli</I> according to Shaw and colleagues in order to reduce one mol melamine to six mol of ammonia in only six conversion steps.
The cluster contains three enzymes performing hydrolase reactions following another three deaminase reaction enzymes. 
It consists of a melamine deaminase (triA) from <I>Acidovorax avenae</I>,  an ammeline deaminase (<I>guaD</I>) from <I>E. coli</I> and an ammelide deaminase (<I>trzC</I>) from <I>A. avenae</I>. The hydrolase reactions are executed by a cyanuric acid hydrolase (<I>atzD</I>) from <I>Pseudomonas sp.</I>, a biuret hydrolase (<I>trzE</I>) from <I>Rhodococcus sp.</I> and an allophanate hydrolase (<I>DUR1,2</I>) as found in <NOBR><I>S. cerevisiae</I></NOBR>.

<FIGURE><FIGCAPTION><STRONG>Figure 6: Melamine pathway cluster reactions.</STRONG> The deaminase reactions are carried out by the melamine deaminase (TriA), ammeline deaminase (GuaD) and ammelide deaminase (TrzC) by cleaving a NH3 each from melamine. The cyanuric acid hydrolase (AtzD) converts cyanuric acid into biuret by cleaving carbon dioxide. The biuret hydrolase (TrzE) and allophanate hydrolase (DUR1,2) cleave another three NH<SUB>3</SUB>, which leaves 6 NH<SUB>3</SUB> in total for each melamin.</FIGCAPTION></FIGURE></P><P>
Since ammonia diffuses through the membrane of <I>E. coli</I> on its own, the design of an exporter was not needed. 


Instead of cloning the genes from the different, native organisms, the codon-optimized genes were ordered as gBlocks - except for <I>guaD</I> which was directly amplified from the genome of <I>E. coli</I> DH5α.
Due to the staggering amount of genes, the inserts were divided on to two plasmid vectors. The genes of the first four reaction enzymes (<I>guaD, trzC, atzD, triA</I>) as well as promoters, ribosome binding sites and terminators were equipped with different overhangs depending on their position in the final plasmid and were cloned separately into the level 0 vectors of the CIDAR toolbox. 

<FIGURE><FIGCAPTION><STRONG>Figure 7: Level 0 plasmids for the melamine pathway cluster.</STRONG> The level 0 plasmids contain promoter, RBS, <I>guaD</I> and terminator as single parts. Here examples for level 0 plasmids are shown.</FIGCAPTION></FIGURE></P><P>
In the following step, each level 0 plasmid was cut with the enzyme BsaI. The gene of interest was inserted into the level 1 plasmids with its corresponding promoter, RBS and terminator. Due to the different overhangs of each part they assembled correctly.

<FIGURE><FIGCAPTION><STRONG>Figure 8: Level 1 plasmids for the melamine pathway cluster.</STRONG> Genes of interest <I>atzD</I>, <I>guaD</I>, <I>trzC</I> and <I>triA</I> in their own level 1 plasmid between promoter, ribosome binding site and terminator.</FIGCAPTION></FIGURE></P><P>
Next, the level 1 plasmids of the four genes were cut with the enzyme BbsI to retrieve the genes of interest with corresponding promoter, RBS and terminator. Due to the varying overhangs on the promoter and terminator sequence they bind to one another, resulting in the final level 2 plasmid for transformation in <NOBR><I>E. coli</I></NOBR>.


<FIGURE><FIGCAPTION><STRONG>Figure 9: Level 2 expression plasmid for the melamine pathway cluster.</STRONG> It contains the four first genes required for the melamine pathway cluster with individual promoter, RBS and terminator.</FIGCAPTION></FIGURE></P><P>
The other two genes <I>trzE</I> and <I>DUR1,2</I> were cloned into a second plasmid via Gibson Assembly to evaluate a different cloning method. It is a molecular cloning method that allows the assembly of of multiple DNA fragments in a single reaction with the help of overlapping overhangs and exonucleases that cleave DNA at the 5’ end.
Since the <I>DUR1,2</I> gene was too long for one synthesis, it had to be ordered in two parts. Following an overlap PCR of <I>trzE</I> and the first part of <I>DUR1,2</I> the two inserts were assembled into pKIKO, a plasmid for genomic integration in <I>E. coli</I>.

<FIGURE><FIGCAPTION><STRONG>Figure 10.: Second final expression plasmid for the melamine pathway cluster.</STRONG>  It contains the genes<I>trzE</I> and <I>DUR1,2</I> for the final conversion steps.</FIGCAPTION></FIGURE></P></DIV><BUTTON class="accordion"><H2>Growth Curve Measurements</H2></BUTTON><DIV class="panel"><P>
For our project, an understanding of the growth of the three different organisms <NOBR><I>Escherichia coli</I></NOBR>, <NOBR><I>Saccharomyces cerevisiae</I></NOBR> and <NOBR><I>Synechococcus elongatus</I></NOBR> had to be established. In addition to this, consistent growth patterns under the same conditions (media, temperature, CO<SUB>2</SUB>, O<SUB>2</SUB>) had to be ensured in a co-culture. Growth rates of the three different organisms under different culturing conditions had to be analyzed and thoroughly evaluated. 
Therefore, we observed the growth behavior of these three organisms in different media compositions. </P><H3>Material and Methods</H3><P>During the experiments, the flasks - containing the monocultures in 20% of total flask volume (e.g. <NOBR>20 mL</NOBR> in a <NOBR>100 mL</NOBR> flask) -  were incubated at <NOBR>30 °C</NOBR> and <NOBR>200 rpm</NOBR>. Since cyanobacteria need light for photosynthesis, <I>S. elongatus</I> cultures were incubated under continuous illumination of <NOBR>75 µmol m<SUP>-2</SUP> s<SUP>-1</SUP></NOBR>. All flasks were sealed with cellulose plugs that allowed for gas exchange in order to provide <I>S. cerevisiae</I> and <I>S. elongatus</I> cultures with enough oxygen and carbon dioxide. 

To keep track of growth, the optical density (OD) of each culture was determined with a spectrophotometer. <NOBR><I>E. coli</I></NOBR> and <NOBR><I>S. cerevisiae</I></NOBR> cultures were measured at a wavelength of 600 nm. Since cyanobacteria contain a variety of photosynthetic pigments, such as phycocyanin and chlorophyll, which absorb at 600 nm, the growth of <I>S. elongatus</I> was measured at <NOBR>750 nm</NOBR><A href="http://www.jbc.org/content/253/22/8303.short"><SUP>20</SUP></A>.

Because of differing division rates among the three organisms, the measurements were performed over a varying period of time with individual measurement intervals adapted to each organisms generation time.  <I>S. cerevisiae</I> and <I>E. coli</I> were observed over a timeframe of 36 hours, whereas the growth of <I>S. elongatus</I> was observed over the course of a week. The OD of <I>S. cerevisiae</I> was measured every 2 hours, <I>E. coli</I> every 0.5 hours and <I>S. elongatus</I> every 12 hours. 

For our experiments the organisms were inoculated in different media to test various cultivation conditions: <A href="https://www.protocols.io/view/Lysogeny-Broth-LB-fcrbiv6">Lysogeny broth (LB)</A>  is the standard cultivation media for <I>E. coli</I> and contains different minerals, essential amino acids, organic compounds, vitamins and certain trace elements. <A href="http://cshprotocols.cshlp.org/content/2017/8/pdb.rec090563.full?text_only=true">Yeast extract peptone dextrose (YPD)</A>  is a complete media for <I>S. cerevisiae</I> and includes yeast extract, glucose and amino acids.<A href="https://www.protocols.io/view/recipe-for-standard-bg-11-media-mmvc466">BG11</A> - also called “blue green medium” - is the standard cultivation media for blue green algae such as cyanobacteria. It does not include a carbon source since the photoautotrophic organisms produce it themselves. <A href="https://www.protocols.io/view/m2-medium-uquevww">M2 medium</A> is a buffered derivate of BG11 with a higher amount of Mg<SUP>2+</SUP>, additional phosphate and less nitrate<A href="https://www.sciencedirect.com/science/article/pii/S1096717617301763"><SUP>21</SUP></A>. It is our three-way co-culture medium suggested by <A href="https://2018.igem.org/Team:Duesseldorf/HP/Gold_Integrated">Daniel Ducat</A>. 
<A href="http://cshprotocols.cshlp.org/content/2015/2/pdb.rec085639.short">
Synthetic defined media (SD)</A> is a minimal medium for <I>S. cerevisiae</I>. It can be used with synthetic dropout supplements for auxotrophic yeast strains. </P><H3>Results</H3><P><B>Growth of <I>E. coli</I></B><I>E. coli</I> has a relatively high growth rate (0.5274 h<SUP>-1</SUP>) in the first four hours in its commonly used cultivation medium LB. It reaches a maximum OD<SUB>600</SUB> of around 6 after 10 hours and stays in the stationary phase for the rest of the measurement. In comparison, growth was fairly reduced in M2 medium, lacking essential carbon sources. The optical density increased very slowly, but constantly with a growth rate of 0.0326 h<SUP>-1</SUP>, never reaching an OD<SUB>600</SUB> of 1.</P><FIGURE><FIGCAPTION><STRONG>Figure 1: <I>E. coli</I> C43 strain in M2 medium (grey) and and LB (blue) media. </STRONG> The optical density at 600 nm was observed over 36 h. </FIGCAPTION></FIGURE><P><B>Growth of <I>E. coli</I> in BG11 with additional Sugar and Salt stress</B>

As our three-way co-culture requires salt in order to induce the sucrose secretion of <NOBR><I>S. elongatus</I></NOBR>, the growth of an <NOBR><I>E. coli</I></NOBR> strain was tested under these conditions as well. Since BG11 is a derivative of our M2 co-culture medium, we assume that the behavior of the cells is similar in the two media and also added an additional control without NaCl. While there’s only a minimal increment in OD<SUB>600</SUB> in all <I>E. coli</I> cultures, the cultures in BG11 medium supplemented with sucrose show a rise in OD<SUB>600</SUB> after around 20 hours.</P><FIGURE><FIGCAPTION><STRONG>Figure 2: <I>E. Coli</I> Rosetta2 pLys 791 growth curves in BG11 (yellow), BG11 with 2% sucrose (green) and BG11 supplemented with 2% sucrose and <NOBR>150 mM</NOBR> NaCl (blue).</STRONG> The optical density at 600 nm was observed over 36 h. </FIGCAPTION></FIGURE><P><B>Growth of <I>S. cerevisiae</I> BY4742</B><I>S. cerevisiae</I> was observed in all media. It grows well in YPD (0.692 h<SUP>-1</SUP>) and SD media including amino acids (0.6719 h<SUP>-1</SUP>) with an OD<SUB>600</SUB> of around 10 after 36 hours. It also shows growth in LB (0.518 h<SUP>-1</SUP>). It doesn’t grow as well in the sugar-deprived M2 medium with the OD<SUB>600</SUB> decreasing after an initial growth spurt (0.4904 h<SUP>-1</SUP>), and barely shows any growth in BG11 with a growth rate of 0.0165 h<SUP>-1</SUP>.</P><FIGURE><FIGCAPTION><STRONG>Figure 3: <I>S. cerevisiae</I> BY 4742 growth curve in SD (purple), LB (yellow), M2 medium (grey), BG11 (green) and YPD (blue)</STRONG> Measurements of OD<SUB>600</SUB> were taken for 36 h.  </FIGCAPTION></FIGURE><P><B>Growth of <I>S. elongatus</I></B><I>S. elongatus</I> grows relatively slow compared to the other two organisms with a generation time of 12 hours.
There is a slight difference between the growth of <I>S. elongatus</I> in M2 medium and BG11 with growth rates around <NOBR>0.4 h<SUP>-1</SUP></NOBR> (M2: 0.3927 h<SUP>-1</SUP>; BG11: 0.4123 h<SUP>-1</SUP>) (Figure 4). 
However, no growth occurred in YPD. Instead, the monocultures show a decreasing OD<SUB>600</SUB> over time, which indicates that the cells were dying. Therefore, the measurements were terminated after 4.5 days.</P><FIGURE><FIGCAPTION><STRONG>Figure 4: <I>S. elongatus</I> sp PCC 7492 wt growth in BG11 (yellow), M2 medium (blue) and YPD (green). </STRONG>The measurements of the cultures in BG11 and M2 medium were taken over 7 days, whereas the measurements of the growth in YPD was terminated after 4.5 days. Growth in BG11 and M2 medium is constant, whereas the OD<SUB>600</SUB> constantly decreases in YPD. </FIGCAPTION></FIGURE><P>For our three-way co-culture, we contemplated using the <I><NOBR>S. elongatus</NOBR></I><NOBR>sp PCC 7942 <I>cscB</I>:::NS3</NOBR> strain provided by Danny Ducat. Hence, a growth curve for this strain was created. It shows a constant increase (Figure 5). </P><FIGURE><FIGCAPTION><STRONG>Figure 5: <I>S. elongatus</I><NOBR>sp PCC 7942 <I>cscB</I>:::N53</NOBR> in BG11 medium. </STRONG>The OD at 750 nm was measured over 6.5 days.</FIGCAPTION></FIGURE><H3>Discussion</H3><P>With our growth curve measurements we were able to make sure that consistent growth patterns exist for all of our three organisms. 


We predicted that <I>E. coli</I> would overgrow the co-culture over time since its generation time is usually around 0.5 hours. It is is a lot shorter compared to the other two organisms in the planned three way co-culture, which would complicate the establishment of a co-culture with <I>E. coli</I>. This can be observed during the growth development in LB. After a short exponential phase of 4 hours with a growth rate of 0.5274 h<SUP>-1</SUP>, <I>E. coli</I> cultures enter the stationary phase, having reached the maximum OD<SUB>600</SUB> of around 6 (Figure 1). 
However, OD<SUB>600</SUB> in the M2 medium increases at a slower rate, but constantly, thus making the medium optimal for a co-culture including <I>E. coli</I>. <I>S. cerevisiae</I> grows very well in YPD, SD and LB media and shows growth in M2 medium to some extend before OD<SUB>600</SUB> is decreasing. There is almost no visible growth in BG11 (Figure 3). We assume that the initial growth spurt of <NOBR><I>S. cerevisiae</I></NOBR> in M2 medium, despite it lacking a carbon source, is due to leftover ressources from the previous medium that were metabolized. After those were depleted, growth stopped and the the cells died.

In addition to that, we also proved that <I>S. elongatus</I><NOBR>sp PCC 4792 <I>cscB</I>:::NS3</NOBR> is a good cyanobacterial representative in our co-culture. Based on our observations, the <I>S. elongatus cscB</I> mutant strain shows similar growth to the wild type strain under uninduced conditions with a growth rate of 0.4446 h<SUP>-1</SUP> (Figure 5) compared to the wt growth rate of 0.4123 h<SUP>-1</SUP> (Figure 4). We therefore concluded that it would be a viable candidate for our co-culture experiments.
Based on the information we gained with the experiments performed on the wild type strain, we assume similar behavior for the <I>cscB</I> strain and therefore did not conduct further growth measurements in different media with the latter strain. Moreover, this medium has been used in previous, published experiments, which emphasizes our rational use of this media<A href="https://www.sciencedirect.com/science/article/pii/S1096717617301763"><SUP>21</SUP></A>.

Besides, we were able to observe that the conditions needed for inducing the <I>cscB</I> strain would not interfere with the growth of <I>E. coli</I> (Figure 2). BG11 as well as M2 medium do not contain a carbon source, which is why <I>E. coli</I> without any additional sucrose stops growing after one division. The growth rate is around 0.0142 h<SUP>-1</SUP>. We hypothesize that residual minimal growth at later time points might result from nutrients provided by dead cells in the media.
This is different with the sugar enriched media. While the growth is still reduced in the beginning, the cultures enter an exponential phase after around 20 hours. The log phase with sugar and salt is 0.1961 h<SUP>-1</SUP>, without salt it is 0.1645 h<SUP>-1</SUP>. The delay could be due to the preferred carbon source, which is glucose. It is possible that <NOBR><I>E. coli</I></NOBR> had to adjust to the given conditions in order to metabolize sucrose.

Furthermore, we showed that M2 medium promotes the growth of<NOBR><I>S. elongatus</I></NOBR> - the slowest reproducing organisms in our three-way co-culture - but restricts it for <NOBR><I>E. coli</I></NOBR> and <NOBR><I>S. cerevisiae</I></NOBR>. Thus, it is a very good choice for a co-cultivation medium. 

An improved experiment would be obtained with an addition of technical replicates and more biological replicates in order to reduce the standard deviation errors. In addition to that, the choice of only one incubator for the entire experiments and the usage of same sized flasks would ensure more similar conditions for every monoculture. 
Furthermore, washing the cells before inoculating them in their tested media could prevent falsifying the results during the first hours of the measurement, as an induction of growth due to transferred essential nutrients from the pre culture medium is possible. </P></DIV><BUTTON class="accordion"><H2>References</H2></BUTTON><DIV class="panel"><P><OL type="1"><LI>Katz, Micah, Bradley M. Hover, and Sean F. Brady. &quot;Culture-independent discovery of natural products from soil metagenomes.&quot; <I>Journal of industrial microbiology &amp; biotechnology</I> 43.2-3 (2016): 129-141
                </LI><LI>Sonia R. Vartoukian, Richard M. Palmer, William G. Wade “Strategies for culture of ‘unculturable’ bacteria” <I>FEMS Microbiology Letters</I>, Volume 309, Issue 1, 1 August 2010, 1–7 
                </LI><LI>Engler, Carola, et al. &quot;Golden gate shuffling: a one-pot DNA shuffling method based on type IIs restriction enzymes.&quot; <I>PloS one</I> 4.5 (2009): e5553
                </LI><LI>Engler, Carola, Romy Kandzia, and Sylvestre Marillonnet. &quot;A one pot, one step, precision cloning method with high throughput capability.&quot; <I>PloS one</I> 3.11 (2008): e3647.
                </LI><LI>Lee, Michael E., et al. &quot;A highly characterized yeast toolkit for modular, multipart assembly.&quot; <I>ACS synthetic biology</I> 4.9 (2015): 975-986
                </LI><LI>Iverson, Sonya V., et al. &quot;CIDAR MoClo: improved MoClo assembly standard and new <I>E. coli</I> part library enable rapid combinatorial design for synthetic and traditional biology.&quot; <I>ACS synthetic biology</I> 5.1 (2015): 99-103
                </LI><LI>Padmanabhan, Sriram, Sampali Banerjee, and Naganath Mandi. &quot;Screening of bacterial recombinants: strategies and preventing false positives.&quot; <I>Molecular Cloning-Selected Applications in Medicine and Biology</I>. InTech, 2011.
                </LI><LI>Miller, Melissa B., and Bonnie L. Bassler. &quot;Quorum sensing in bacteria.&quot; <I>Annual Reviews in Microbiology</I> 55.1 (2001): 165-199.
                </LI><LI>Egland, KA.; Greenberg,E.P. “Quorum sensing in <I>Vibrio fischeri</I>: elements of the luxI promoter”. <I>Molecular microbiology</I>. 1999; Volume 21, Issue 4, 1197-1204
                </LI><LI>Scott, Spencer R., et al. &quot;A stabilized microbial ecosystem of self-limiting bacteria using synthetic quorum-regulated lysis.&quot; <I>Nature microbiology</I> 2.8 (2017): 17083.
                </LI><LI>Schrumpf, Barbel, et al. &quot;A functionally split pathway for lysine synthesis in <I>Corynebacterium glutamicium</I>.&quot; <I>Journal of bacteriology</I> 173.14 (1991): 4510-4516. 
                </LI><LI>Bärbel Schrumpf, Lothar Eggeling, and Hermann Sahm
 “Isolation and prominent characteristics of an L-lysine hyperproducing strain of <I>Corynebacterium glutamicum</I>”
<I>Microbiol Biotechnol</I> (1992) 37:566-571
                </LI><LI>Lagunas, Rosario. &quot;Sugar transport in Saccharomyces cerevisiae.&quot; <I>FEMS Microbiology Letters</I> 104.3-4 (1993): 229-242
                </LI><LI>Klähn, Stephan, and Martin Hagemann. &quot;Compatible solute biosynthesis in cyanobacteria.&quot; <I>Environmental microbiology</I> 13.3 (2011): 551-562
                </LI><LI> Qiao, Cuncun, et al. &quot;Effects of reduced and enhanced glycogen pools on salt-induced sucrose production in a sucrose-secreting strain of Synechococcus elongatus PCC 7942.&quot; <I>Applied and environmental microbiology</I> 84.2 (2018): e02023-17.
                </LI><LI> Zinchenko, V. V., et al. &quot;Vectors for the complementation analysis of cyanobacterial mutants.&quot; <I>RUSSIAN JOURNAL OF GENETICS C/C OF GENETIKA</I> 35 (1999): 228-232
                </LI><LI> Kanda, Keisuke, et al. &quot;Application of a phosphite dehydrogenase gene as a novel dominant selection marker for yeasts.&quot; <I>Journal of biotechnology</I> 182 (2014): 68-73
                </LI><LI> Lee, M. E., DeLoache, W. C., Cervantes, B., &amp; Dueber, J. E. (2015). A highly characterized yeast toolkit for modular, multipart assembly. <I>ACS synthetic biology</I>, 4(9), 975-986
                </LI><LI> A. Joe Shaw,* Felix H. Lam  et al.  “Metabolic engineering of microbial competitive advantage for industrial fermentation processes. “ <I>Science</I> 05 Aug 2016 : 583-586
                </LI><LI>Yamanaka, G., A. N. Glazer, and R. C. Williams. &quot;Cyanobacterial phycobilisomes. Characterization of the phycobilisomes of <I>Synechococcus</I> sp. 6301.&quot; <I>Journal of Biological Chemistry</I> 253.22 (1978): 8303-8310.
                </LI><LI>Weiss, Taylor L., Eric J. Young, and Daniel C. Ducat. &quot;A synthetic, light-driven consortium of cyanobacteria and heterotrophic bacteria enables stable polyhydroxybutyrate production.&quot; <I>Metabolic engineering</I> 44 (2017): 236-245. 
              </LI></OL></P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>