<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Duesseldorf_Plant skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Duesseldorf/Plant</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="sidebar"><UL><LI><A href="https://2018.igem.org/Team:Duesseldorf">Home</A></LI><UL class="sub"><A href="#!">Team</A><LI><A href="https://2018.igem.org/Team:Duesseldorf/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Ontour">On Tour</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Attributions">Attributions</A></LI></UL><UL class="sub"><A href="#!">Project</A><LI><A href="https://2018.igem.org/Team:Duesseldorf/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Improve">Improve</A></LI></UL><UL class="sub"><A href="#!">Parts</A><LI><A href="https://2018.igem.org/Team:Duesseldorf/Parts">Parts Overview</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Composite_Part">Composite Part</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Part_Collection">Part Collection</A></LI></UL><LI><A href="https://2018.igem.org/Team:Duesseldorf/Safety">Safety</A></LI><UL class="sub"><A href="#!">Human Practices</A><LI><A href="https://2018.igem.org/Team:Duesseldorf/Silver">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/HP/Gold_Integrated">Integrated Practices</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Newsletter">Newsletter</A></LI></UL><UL class="sub"><A href="#!">Awards</A><LI><A href="https://2018.igem.org/Team:Duesseldorf/Medals">Medal Criteria</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Plant">Plant Synthetic Biology</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Model">Model</A></LI></UL><LI><A href="https://igem.org/2018_Judging_Form?team=Duesseldorf">Judging Form</A></LI></UL><UL class="sub"><A href="#!">Advice</A><LI><A href="https://2018.igem.org/Team:Duesseldorf/MoClo">Modular Cloning</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/CyanoQandA">Cyano Q&amp;A</A></LI></UL></DIV><NAV><UL class="menu"><LI class="menu-item-has-children"><A href="#!">Team</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Ontour">On Tour</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Attributions">Attributions</A></LI></UL><LI class="menu-item-has-children"><A href="#!">Project</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Improve">Improve</A></LI><LI class="menu-item-has-children"><A href="#!">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Parts">Parts overview</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Composite_Part">Composite Part</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Part_Collection">Part Collection</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Safety">Safety</A></LI><LI class="menu-item-has-children"><A href="#!">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Silver">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/HP/Gold_Integrated">Integrated Practices</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Newsletter">Newsletter</A></LI><LI class="menu-item-has-children"><A href="#!">Awards</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Medals">Medal Criteria</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Plant">Plant Synthetic Biology</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/Model">Model</A></LI><LI class="menu-item-has-children"><A href="#!">Advice</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/MoClo">Modular Cloning</A></LI><LI><A href="https://2018.igem.org/Team:Duesseldorf/CyanoQandA">Cyano Q&amp;A</A></LI><LI><A href="https://igem.org/2018_Judging_Form?team=Duesseldorf">Judging Form</A></LI></NAV><H1>Plant Synthetic Biology</H1><P>
This year´s iGEM team Duesseldorf proposed the use of a photosynthetic organism, <NOBR><I>Synechococcus elongatus</I> PCC 7942</NOBR> as part of its three way co-culture. We embedded this species in the focus of our system to show the advantages of such a microorganism in a stable culture mix. Indeed, in our setup<NOBR><I> S. elongatus</I></NOBR> is supplying the other two co-culture participants with a carbon source, in this case sucrose. Sucrose production is induced by addition of NaCl and IPTG, which allows us to control our system as desired. In this way we are exploiting the ability of this microorganism to synthesize a vital component for the other constituents of the culture by the natural presence of sunlight. This is not only convenient for the establishment of our co-culture, since the system will be able to be sustained by a carbon source, but also for future applications in synthetic biology. 

</P><H2>Overproduction of Sucrose by <NOBR><I>S. elongatus</I></NOBR></H2><P>
A carbon source is needed in our co-culture system for <NOBR><I>E. coli</I></NOBR> and <NOBR><I>S. cerevisiae</I></NOBR> in order to stabilize the culture and to create an autonomous system. This is why we included the cyanobacterium <NOBR><I>S. elongatus</I></NOBR> PCC 7942 in our three-way co-culture since it is a photosynthetically active cyanobacterium. By engineering <NOBR><I>S. elongatus</I></NOBR> to overproduce and secrete sucrose, we aim to create a stable co-culture that is not further dependent on carbon in the media. <NOBR><I>S. cerevisiae</I></NOBR> is able to convert sucrose into fructose and glucose extracellularly which can then be used as a carbon source by <NOBR><I>S. cerevisiae</I></NOBR> itself<A href="https://www.sciencedirect.com/science/article/pii/037810979390598V "><SUP>1</SUP></A> and <I>E. coli</I> or other microorganisms. 

Many cyanobacteria are able to overproduce substances like sugars in order to prevent the cell from osmotic stress<A href="https://aem.asm.org/content/84/2/e02023-17.long"><SUP>2</SUP></A>.
In addition it has been shown that upon induction of the symporter CscB <NOBR><I>S. elongatus</I></NOBR> is able to produce higher amounts of sugar. <NOBR><I>S. elongatus</I></NOBR> produces sucrose in several steps from Glucose-1-P. The first rate limiting step in the pathway for the sucrose production is catalyzed via sucrose-phosphate synthase (Sps) and is induced by salt stress<A href="https://aem.asm.org/content/84/2/e02023-17.long"><SUP>3</SUP></A>.
We obtained a specific strain published by Daniel C. Ducat et al. which is able to secrete sucrose with the help of the CscB sucrose permease. The <I>cscB</I> gene encodes a symporter which is inducible by IPTG<A href="https://www.sciencedirect.com/science/article/pii/037810979390598V "><SUP>1</SUP></A>. In this case we plan to induce sucrose production with NaCl and IPTG. Furthermore, advice for cultivation media for the co-culturing experiments was provided to us by Dr. Ducat. This is also one big part of our <A href="https://2018.igem.org/Team:Duesseldorf/HP/Gold_Integrated">Integrated Human Practices</A> .

Our goal is to establish a system which allows the production and the secretion of sucrose via induction with only IPTG instead of both NaCl and IPTG. With this approach we want to avoid salt stress, which could negatively influence our co-culture.
</P><H2>Experimental Design</H2><P>
We are cloning an expression plasmid which contains an iGEM provided IPTG inducible promoter (<A href="http://parts.igem.org/Part:BBa_R0010">BBa_R0010</A>, termed P<SUB><I>lac</I></SUB> here) and <I>sps</I> (sucrose phosphate synthase) as the gene of interest. The origin species of <I>sps</I> we used is <I>Synechocystis</I> PCC 6803<A href="https://www.sciencedirect.com/science/article/pii/037810979390598V "><SUP>1</SUP></A>. This variant of sucrose phosphate synthase needs less NaCl in order to be activated, compared to the variant already found in <NOBR><I>S. elongatus</I></NOBR><A href="https://aem.asm.org/content/84/2/e02023-17.long"><SUP>2</SUP></A>.
We want to insert the plasmid into our strain of <NOBR><I>S. elongatus</I></NOBR> by conjugation. We hypothesize that the induction with IPTG is enough to induce the overproduction and the secretion of sucrose. 
<FIGURE><FIGCAPTION><STRONG>Figure 1: Schematic overview of the experimental design for the production of sucrose</STRONG> S. elongatus PCC 7942 cscB:::NS3 with pSHDY_sps</FIGCAPTION></FIGURE></P><P>
For the cloning strategy of our expression system, Gibson Assembly was chosen. The used plasmid backbone is called pSHDY, which was created and provided by an advisor of our team. pSHDY is a conjugative shuttle vector with a broad host range which is commonly used as a plasmid-based expression system in cyanobacteria. Selection markers are antibiotic resistances - in this case kanamycin and spectinomycin. The strain we are working with provides an additional genomically integrated chloramphenicol resistance which also allows the specific selection of positively conjugated clones. For the uptake of plasmid DNA, the method triparental mating<A href="https://www.protocols.io/view/triparental-mating-of-synechocystis-ftpbnmn"><SUP>4</SUP></A> was used.
<FIGURE><FIGCAPTION><STRONG>Figure 2 : Plasmid for conjugation</STRONG> It is a pSHDY backbone containing sps and P<SUB>lac</SUB></FIGCAPTION></FIGURE></P><H2>Results</H2><H3>Biochemical analysis of Sucrose Content in <I>S. elongatus</I> PCC7942 <I>cscB</I>:::NS3  Samples</H3><P class="details">To detect the amount of sucrose production and secretion of our carbon source provider <NOBR><I>S. elongatus</I></NOBR> in the co-culture, we performed a biochemical assay which is using the YSI 2950 Biochemistry analyzer. This method relies on the detection of sucrose mediated by a voltage change by immobilized enzymes on a specific membrane.
For this purpose we inoculated the <NOBR><I>S. elongatus</I></NOBR> PCC7942 <I>cscB</I>:::NS3 strain at an OD<SUB>750</SUB> of 0.3 and incubated it at standard conditions for two days. After two days the culture was induced with 1 mM IPTG and 150 mM NaCl. The first samples were taken before induction, then after two days, four days and six days. As a control, wild type <NOBR><I>S. elongatus</I></NOBR> strain was treated in the same manner and samples were taken at the same time points, respectively.
To measure sucrose content, each sample was centrifuged at 4000 rpm for three minutes and the supernatant was sterile filtered for analysis.
A sucrose standard curve ranging from 1 mg/mL to 0.0625 mg/mL (1:2 dilutions each) was prepared
in order to determine the final sucrose concentration in the tested samples.
It can be seen that sucrose amounts rise after induction. The media of wild type <I>S. elongatus</I> before induction does not show a presence of sucrose at all, whilst a small amount of sucrose is detectable before the induction of <I>S. elongatus</I> PCC7942 <I>cscB</I>:::NS3.</P><FIGURE><FIGCAPTION><STRONG>Figure 3: Sucrose content in medium supernatant of <I>S. elongatus</I> and PCC 7942 cscB:::NS3 strain in days (d) post induction as well as pre induction with 150 mM NaCl and 1 mM IPTG.</STRONG> Values are calculated from triplicates and measured based on a sucrose standard curve.</FIGCAPTION></FIGURE><H4>Discussion</H4><P class="details">We could clearly demonstrate that upon induction with NaCl and IPTG the sucrose content is higher than in the uninduced variant of the <I>S. elongatus</I> PCC 7942 <I>cscB</I>:::NS3 strain. Moreover, it could be shown that the wild type, as expected, does not show any presence of sucrose in the corresponding medium. This phenomenon can be explained by heterologous sucrose transporter CscB in the <I>S. elongatus</I> PCC 7942 <I>cscB</I>:::NS3 strain<A href="https://www.ncbi.nlm.nih.gov/pubmed/27742883"><SUP>5</SUP></A> . Upon induction, sucrose export is stimulated in the cells harbouring this transporter. Wild type cells, which lack the transporter, cannot secrete sucrose in this manner. As a consequence, no sucrose is detectable in the medium. The amounts of sucrose secreted are comparable to literature values and even though the cscB transporter is inducible, low amounts of sucrose are exported when not induced by NaCl and IPTG, which is consistent with our data<A href="https://www.ncbi.nlm.nih.gov/pubmed/27742883"><SUP>5</SUP></A> . </P><H3>Cultivation of <I>E. coli</I> and <I>S. cerevisiae</I> in M2 Medium Enriched with Sucrose Secreted by <I>S. elongatus</I></H3><P class="details">After detecting the secretion of sucrose by <I>S. elongatus</I> sp PCC 7942 <I>cscB</I>:::NS3 via the YSI system, we wanted to test if the released amount of sucrose would be enough to sustain the growth of <I>E. coli</I> and <I>S. cerevisiae</I> in a co-culture.
To avoid interactions between the species that could interfere with the growth of both microorganisms, only the cyanobacterial medium was used and transferred to the monocultures.</P><H4>Materials and Methods</H4><P class="details">During our experiments, 3 ml M2 medium of induced <I>S. elongatus_cscB</I> cultures were sampled two, four and six days after induction. The culture was centrifuged down, the supernatant was sterile filtered and used for new cultures of <I>E. coli</I> and <I>S. cerevisiae</I>.  
As negative control, the medium of a wild type <I>S. elongatus</I>, which also included NaCL and IPTG normally used for the induction of the strain, was sampled and tested at a regular interval. 

Previous preliminary cultures of <I>E. coli</I> or <I>S. cerevisiae</I> were centrifuged and washed with M2 medium to rid them of any additional sugar sources. After that the cultures were added to the sterile filtered medium from <I>S. elongatus</I> and adjusted to an OD<SUB>600</SUB> of 0.1. Afterwards a 96 well plate well was filled with 200 µl per well of the two cultures in technical triplicates.

The growth of <I>E. coli</I> and <I>S. cerevisiae</I> cultures in the cyano medium was measured in a plate reader that incubated the cultures at 30°C and agitated them 999 seconds between each measurement. The measurements of the optical density (OD) at 600 nm were taken every 30 min over a time frame of 24 hours.</P><H4>Results</H4><FIGURE><FIGCAPTION><STRONG>Figure 4: Growth of <I>S. cerevisiae</I> in <I>S. elongatus</I> wt (blue) and <I>cscB</I> mutant (green) medium two days after induction with IPTG (1 mM) and NaCL (150 mM).</STRONG> The media was enriched with the amino acids histidine, leucine, lysine and uracil as well as ammonium sulfate (5 g/l). The measurement was conducted over a time frame of 36 hours.</FIGCAPTION></FIGURE><P><FIGURE><FIGCAPTION><STRONG>Figure 5: Growth of <I>E. coli</I> in <I>S. elongatus</I> wt (blue) and <I>cscB</I> mutant (green) medium two days after induction with IPTG (1 mM) and NaCL (150 mM).</STRONG> The media was enriched with the amino acids histidine, leucine, lysine and uracil as well as ammonium sulfate (5 g/l). The measurement was conducted over a time frame of 36 hours.</FIGCAPTION></FIGURE></P><P class="details">During the first measurement two days after the induction of <I>S. elongatus</I>, the OD<SUB>600</SUB> of <I>S. cerevisiae</I>  in the medium of the wild type cyanobacteria is constantly decreasing (Figure 4). While this happened in the <I>cscB</I> mutant medium as well, the OD<SUB>600</SUB> rose after around 18 hours. The OD<SUB>600</SUB> of <I>E. coli</I> in both media obtained from the wt and <I>cscB</I> mutant is constantly decreasing (Figure 5).</P><H4>Discussion</H4><P class="details">Our results show that the engineered <I>S. elongatus</I> sp PCC 7942 <I>cscB</I>:::NS3 strain is able to support the growth of <I>S. cerevisiae</I>. While the negative controls in the WT media show a decreasing OD<SUB>600</SUB>- hence die off during the experiment, the OD<SUB>600</SUB> is increasing in all <I>cscB</I> mutant medium samples after 20 hours, regardless of whether <I>S. elongatus</I> was induced two, four or six days ago. We assume that the delay in growth is due to the change of culture medium.
However, <I>E. coli</I> dies in the medium two days after induction and only shows a slightly elevated OD<SUB>600</SUB> in the end of the second and third measurement. We suspect that <I>E. coli</I> BL21(DE3)C43 is not able to convert sucrose into a metabolizable carbon source<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3553775/"><SUP>6</SUP></A>. We assume that <I>E. coli</I>s carbon source in the three way co-culture could be obtained from <I>S. cerevisiae</I>. It is able to convert sucrose into glucose and fructose extracellularly with secreted invertases<A href="https://academic.oup.com/femsyr/article/16/1/fov107/2467713"><SUP>7</SUP></A>.</P><BUTTON class="accordion"><H2>References</H2></BUTTON><DIV class="panel"><P><OL type="1"><LI>Lagunas, Rosario. &quot;Sugar transport in Saccharomyces cerevisiae.&quot; <I>FEMS Microbiology Letters</I> 104.3-4 (1993): 229-242
                </LI><LI>Klähn, Stephan, and Martin Hagemann. &quot;Compatible solute biosynthesis in cyanobacteria.&quot; <I>Environmental microbiology</I> 13.3 (2011): 551-562
                </LI><LI> Qiao, Cuncun, et al. &quot;Effects of reduced and enhanced glycogen pools on salt-induced sucrose production in a sucrose-secreting strain of <I>Synechococcus elongatu</I>s PCC 7942.&quot; <I>Applied and environmental microbiology</I> 84.2 (2018): e02023-17.
</LI><LI> Zinchenko, V. V., et al. &quot;Vectors for the complementation analysis of cyanobacterial mutants.&quot; <I>RUSSIAN JOURNAL OF GENETICS C/C OF GENETIKA</I> 35 (1999): 228-232
                </LI><LI> Abramson, Bradley W., et al. &quot;Increased photochemical efficiency in cyanobacteria via an engineered sucrose sink.&quot; <I>Plant and Cell Physiology</I> 57.12 (2016): 2451-2460.
                </LI><LI> Sabri, Suriana, Lars K. Nielsen, and Claudia E. Vickers. &quot;Molecular control of sucrose utilization in Escherichia coli W, an efficient sucrose-utilizing strain.&quot; <I>Applied and environmental microbiology</I> (2012): AEM-02544.
                </LI><LI>Marques, Wesley Leoricy, et al. &quot;Sucrose and <I>Saccharomyces cerevisiae</I>: a relationship most sweet.&quot; <I>FEMS Yeast research</I> 16.1 (2016).
                </LI></OL></P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>