Team:Duesseldorf/Description
Home
Team
Team
Collaborations
On Tour
Attributions
Project
Description
Design
Experiments
Notebook
InterLab
Model
Results
Demonstrate
Improve
Parts
Parts Overview
Basic Parts
Composite Part
Part Collection
Safety
Human Practices
Human Practices
Integrated Practices
Public Engagement
Newsletter
Awards
Medal Criteria
Plant Synthetic Biology
Model
Judging Form
Advice
Modular Cloning
Cyano Q&A
Team
Team
Collaborations
On Tour
Attributions
Project
Description
Design
Experiments
Notebook
InterLab
Model
Results
Demonstrate
Improve
Parts
Parts overview
Basic Parts
Composite Part
Part Collection
Safety
Human Practices
Human Practices
Integrated Practices
Public Engagement
Newsletter
Awards
Medal Criteria
Plant Synthetic Biology
Model
Advice
Modular Cloning
Cyano Q&A
Judging Form
PROJECT DESCRIPTION
The Idea
Scientists from all over the world and different fields are using microorganisms for their research - that is nothing new. But most of them are using monocultures and therefore miss hundreds and more possibilities which arise in co-cultures. It is estimated that less than 1% of all microorganisms have been successfully cultivated 1 . A reason for this might be the conditions in the laboratory. In nature no organism is completely isolated, but lives in very complex systems. Scientist have only just begun to investigate the complex interaction between different microorganisms but often fail at the cultivation. Easily obtainable and stable co-cultures would allow research of yet uninvestigated species and might give further insight into cell-cell-interactions between microorganisms 2 .
One problem of isolated cultures is the regular use of antibiotics as selection marker and to prevent contaminations which leads to multiple resistances. Here as well, a co-culture with nutrient dependencies instead of higher antibiotic concentration is a better alternative.
Pharmacotherapeutic companies spend a lot of money on the development of new antibiotics, while in co-culturing conditions some microorganisms show differential gene expression which might lead to the production of new antibiotics 3 . In general, co-cultures might provide a great opportunity as a new method for cultivation and the production of new antibiotics or industrially interesting products. There is a lot of potential for new co-cultures and we - team iGEM 2018 HHU - want to tackle these challenges.
The Reason
Besides all the scientific advantages, why do we need co-cultures? A rising field in medical research is the correlation between gut microbiota and several diseases. Some of the most prominent examples are irritable bowel syndrome (IBS), Crohn’s disease and even colon cancer. But proving such a correlation is often only the first step. Afterwards doctors have to search for ways to use this knowledge to find new treatment options for these diseases.
A possible way to treat them is the use of antibiotics. But antibiotic treatment does not only affect the `healthy` microbiome but can also lead to unwanted resistances against antibiotics in some pathogens. One alternative is the use of probiotics to influence bacterial populations. Medical probiotics are living bacterial cultures that are taken orally in a stomach acid resistant capsule. As of now it is common to use monocultures, which are then separately and sterilely prepared and mixed in defined ratios. This makes it a complex and time-consuming process. Using our toolbox to design a stable co-culture, companies could save a lot of money and time by preparing their product in one culture. Since the fermenters don’t have to be opened that often and the organisms depend on each other, the contamination risk is lower. Furthermore, doctors can choose the organisms to design the co-culture as needed by the patient. This is important as it follows the rising trend of personalized healthcare.   But our co-culture could not only have a great effect on the application of microorganisms in medicine, but also in basic research or the biotechnological production of new substances.
The Challenges
Before starting research, safety must be ensured. But this problem is solved through the strict rules of S1 laboratory work. At every work step we ensure that no organisms are set free and spread into the environment. Our experiments include a detailed control of the microorganisms, the cell density, their behavior and environmental conditions. For industrial usage the production must be scalable and high throughput has to be guaranteed as well. Furthermore, general technologies for a normed use for research projects that are also easily applicable for large scale production must be achieved 4 . One goal could be the reduction and simplification of processing steps and easy-to-use protocols for stable co-cultures to make cultivation and production faster and cheaper. Other requirements for making co-cultures a profitable alternative to monocultures are greater product diversities and efficient isolation of them. For scientists the possibility to build complex systems and methods for big data collection are desirable and greatly demanded.
The Project: "Trinity"
Since our co-cultures should be universally usable by laboratories all over the world, we plan to design it as user-friendly and modular as possible. Therefore, the Golden Gate modular cloning standard techniques (MoClo) were used besides Gibson Assembly for our cloning steps, which allows exchange of all parts to adjust the co-culture to every possible condition 5 , 6 .
We envision a standardised system, a three-way co-culture with a selection of organisms and regulations. `Trinity` is going to incorporate different dependencies between the organisms in order to control their growth and enable them to adjust each other’s behaviour.
Project Trinity is divided into three systems which use different approaches to regulate cell density. Our first system is utilizing the quorum sensing mechanism to regulate cell density in the population by using bacterial communication molecules. E. coli regulates its own cell density by expressing a lysis gene after induction by AHL1 quorum sensing molecules 11 . For S. cerevisiae the system depends on the design of a synthetic promoter that activates another lysis gene after the recognition of AHL2 molecules 12 . The aim of our second approach is to achieve the dependency of E. coli and S. cerevisiae, through the exploitation of auxotrophies. Here S. cerevisiae is auxotrophic for lysine, which is produced by E. coli 10 . At the same time, E. coli is auxotrophic for leucine, which is provided by S. cerevisiae. This way E. coli and S. cerevisiae are dependent upon each other.
Our third system, also called “Nutrient System”,   deals with the dependence of the three organisms Escherichia coli, Saccharomyces cerevisiae and Synechococcus elongatus sp. PCC 7942 based on the availability of the essential nutrients nitrogen 7 , phosphate 8 and carbon 9 .
System 1
The first approach is utilizing quorum sensing mechanisms to regulate cell density in the population by using bacterial communication molecules.
Introduction
E. coli:
For cell-density control, E. coli harbors the gene luxI encoding for LuxI, producing AHL constantly and the gene luxR which continuously produces the LuxR protein. This protein recognizes the AHL molecule and then activates the Plux promoter, which expresses the lysis gene phiX174E. Thus the population reduces itself after a certain threshold is crossed.
S. cerevisiae:
For S. cerevisiae we will create a synthetic promoter, which makes the yeast compatible to the bacterial quorum sensing system. The synthetic promoter is activated by LuxR after binding the AHL2 molecule and then activates the lysis gene.
Visual representation of the lysis gene induced by synthesis of the quorum sensing molecule acyl homoserine lactone (AHL) This way E. coli cells are supposed to regulate their own cell density
System 2
The aim of our second approach is the dependency of E. coli and S. cerevisiae on each other, achieved through the exploitation of auxotrophies.
Introduction
E. coli:
Leucine auxotrophic E. coli is engineered to produce lysine for S. cerevisiae. For optimization of lysine production a non-feedback inhibiting gene lysC from C. glutamicum and the gene ddh are introduced to attain the highest possible lysine yield. This enables the auxotrophic character as a dependency as well as it being a selection marker.
S. cerevisiae:
Is auxotroph for lysine, thus dependent on the production of lysine by E. coli, but on the other hand produces leucine for E. coli. Therefore LEU2 is overexpressed to reach a sufficient concentration of leucine for E. coli to grow.
The exchange of lysine and leucine of S. cerevisiae and E. coli
System 3
Our main system, called “Nutrient System”, deals with the dependence of three organisms on the availability of essential nutrients.
Introduction
E. coli:
A synthetic cluster of 6 genes from different organisms (Acidovorax avenae, E. coli, Pseudomonas sp., Rhodococcus sp., S. cerevisiae) is used for the cleavage of melamin to ammonia and carbon dioxide.
S. cerevisiae:
With the gene ptxD from P. putida, S. cerevisiae is able to convert phosphite, which is unusable for most organisms, into phosphate.
S. elongatus
The cyanobacterium converts sucrose, which is naturally produced by photosynthesis, into fructose and glucose by an invertase encoded by the gene invA. Glucose will be secreted into the media with the help of an exporter, encoded by glf. These genes are taken from the Z. mobilis genome.
All three organisms are dependend on each other by providing essentiell nutrients to the other both.
References
Katz, Micah, Bradley M. Hover, and Sean F. Brady. "Culture-independent         discovery of natural products from soil metagenomes." Journal of industrial         microbiology & biotechnology 43.2-3 (2016): 129-141.
Goers, Lisa, Paul Freemont, and Karen M. Polizzi. "Co-culture systems and         technologies: taking synthetic biology to the next level." Journal of The         Royal Society Interface 11.96 (2014): 20140065.
Adnani, Navid, et al. "Coculture of Marine Invertebrate-Associated         Bacteria and Interdisciplinary Technologies Enable Biosynthesis and         Discovery of a New Antibiotic, Keyicin." ACS chemical biology 12.12         (2017): 3093-3102.
Padmaperuma, Gloria, et al. "Microbial consortia: a critical look at         microalgae co-cultures for enhanced biomanufacturing." Critical reviews in         biotechnology 38.5 (2018): 690-703.
Lee, Michael E., et al. "A highly characterized yeast toolkit for modular,         multipart assembly." ACS synthetic biology 4.9 (2015): 975-986.
Iverson, Sonya V., et al. "CIDAR MoClo: improved MoClo assembly standard         and new E. coli part library enable rapid combinatorial design for         synthetic and traditional biology." ACS synthetic biology 5.1 (2015):         99-103.
Shaw, A. Joe, et al. "Metabolic engineering of microbial competitive         advantage for industrial fermentation processes." Science 353.6299         (2016): 583-586.
Kanda, Keisuke, et al. "Application of a phosphite dehydrogenase gene         as a novel dominant selection marker for yeasts." Journal of         biotechnology 182 (2014): 68-73.
Niederholtmeyer, Henrike, et al. "Engineering cyanobacteria to synthesize         and export hydrophilic products." Applied and environmental microbiology 76.11 (2010): 3462-3466.
Schrumpf, Barbel, et al. "A functionally split pathway for lysine         synthesis in Corynebacterium glutamicium." Journal of bacteriology 173.14         (1991): 4510-4516.
Dong, Yi-Hu, et al. "Quenching quorum-sensing-dependent bacterial         infection by an N-acyl homoserine lactonase." Nature 411.6839 (2001): 813.
Scott, Spencer R., et al. "A stabilized microbial ecosystem of         self-limiting bacteria using synthetic quorum-regulated lysis." Nature microbiology 2.8 (2017): 17083.
