<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Nevada_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Nevada/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Nevada/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Nevada/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Nevada/Notebook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Nevada/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Nevada/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="headerlinks"><UL id="nav"><LI><A class="bannertoplinks" href="https://2010.igem.org/Team:Nevada">Home</A></LI><LI><A class="bannertoplinks" href="https://2010.igem.org/Team:Nevada/Team">Team profile</A></LI><LI><A class="bannertoplinks" href="#">Project</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Nevada/plants_as_remote_sensors">Remote Sensors</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Nevada/long_term_goals">Long Term Goals</A></LI></UL><LI><A class="bannertoplinks" href="#">Parts</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Nevada/promoters">promoters</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Nevada/plant_compatible_reporters">reporters</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Nevada/plasmid_w_terminator_sequences">plasmid + term.</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Nevada/Composite">composite</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Nevada/ccdB">ccdB</A></LI><LI><A class="bannertoplinks" href="https://2010.igem.org/Team:Nevada/registry_submissions">Registry Submissions</A></LI><LI><A class="bannertoplinks" href="https://2010.igem.org/Team:Nevada/Transformations">NT Cell Transformations</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Nevada/Results" tabindex="2">Results</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Nevada/BY-2 (NT1)Transformation Protocol" tabindex="1">Protocol</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Nevada/Agrobacterium Transformations" tabindex="3">Agro. Trans.</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Nevada/Transgenic Plants" tabindex="3">Transgenic Plants</A></LI><LI><A class="bannertoplinks" href="#">Other Stuff</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Nevada/Modeling">Modeling</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Nevada/Safety">Safety</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Nevada/Notebook">Notebook</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Nevada/fundraising_sponsorships">Fundraising &amp; Sponsorships</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Nevada/Team Nevada: Plant Summit">Plant Summit</A></LI><LI><A class="bannerlinks" href="http://biotechniques.com/news/iGEM-competitors-gear-up-for-2010-challenge/biotechniques-304538.html">Media</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Cuckoo_Clock">Cuckoo Clock</A></LI></DIV><DIV style="padding: 10px 10px 30px 10px;"><P> </P><P> </P><P style="text-align:left;"><SPAN style="font-family:Arial;font-size:16pt;font-style:italic;font-weight:none;text-decoration:italic;color:#15317E;background-color:ffffff;">Team Nevada Notebook</SPAN></P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#APRIL"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">APRIL</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#MAY"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">MAY</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#JUNE"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">JUNE</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#JULY"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">JULY</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#AUGUST"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">AUGUST</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#SEPTEMBER"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">SEPTEMBER</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#OCTOBER"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">OCTOBER</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="APRIL">APRIL</SPAN></H2><P><B>Week of April 11-17:</B></P><UL><LI><I>Christian</I></LI><LI>Transformed pBIB into Top 10 Cells
</LI></UL><UL><LI><I>Bryson, Michael, Senny, Tyler</I></LI><LI> Made tobacco cell (NT-1) media in Dr. Shintani's lab
</LI></UL><P><B>Week of April 18-24:</B></P><UL><LI><I>Bryson, Christian</I></LI><LI> EcoRI digest of pBIB
</LI><LI> Made Na acetate buffer
</LI></UL><LI><I>Christian</I></LI><LI> Ran gel of EcoR1 Digested pBIB
</LI><H2><SPAN class="mw-headline" id="MAY">MAY</SPAN></H2><P><B>Week of April 25-May 1:</B></P><UL><LI><I>Bryson</I></LI><LI> Ran agarose gel of EcoRI digest
</LI></UL><UL><LI><I>Matthew</I></LI><LI> Team collaborated on pBIB assignment. Too many chefs in the kitchen.
</LI><LI> We decide to each tackle the pBIB problem in parallel
</LI></UL><P><B>Week of May 2-8:</B></P><UL><LI><I>Elaine</I></LI><LI> Ran 0.8% agarose gel of EcoRI digest
</LI><LI> Made LB/KAN plates
</LI><LI> Spread for colonies
</LI><LI> Did miniprep for pBIB liquid cultures
</LI></UL><UL><LI><I>Matthew</I></LI><LI> Miniprepped pBIB
</LI><LI> Ran EcoRI digest and Klenow
</LI><LI> Ethanol precipitated and ligated
</LI></UL><P><B>Week of May 9-15:</B></P><UL><LI><I>Christian</I></LI><LI> EcoR1/Xba1 Digest of pBIB
</LI><LI> Did Mini prep on 4 cultures
</LI></UL><UL><LI><I>Bryson</I></LI><LI> Ran 0.8% agarose gel of pBIB post-phenol:chloroform cleanup.
</LI><LI> Did minipreps on additional pBIB liquid cultures.
</LI></UL><UL><LI><I>Elaine</I></LI><LI> Did XbaI and EcoRI digest of pBIB
</LI><LI> Ran 2 0.8% gels of each digest
</LI><LI> Did a miniprep and a Phenol:chloroform clean-up
</LI><LI> Ran 0.8% gel of the XbaI and EcoRI digest with the uncut pBIB
</LI></UL><UL><LI><I>Matthew</I></LI><LI> Ligated and transformed
</LI><LI> Screened colonies, miniprepped, and digested, ran on gel
</LI><LI> No candidates had EcoRI eliminated
</LI></UL><P><B>Week of May 16-22:</B></P><UL><LI><I>Christian</I></LI><LI> Generated glycerol stock of pBIB transformed Top 10 cells
</LI><LI> Inoculated 5 cultures of pBIB transformed Top 10 cells grown on Kan resistant plates
</LI><LI> Ran samples on gel after digest with EcoR1
</LI></UL><UL><LI><I>Bryson</I></LI><LI> Klenow reactions of EcoRI digests
</LI><LI> Phenol:chloroform cleanup of pBIB prior to ligation
</LI><LI> Blunt-end ligation of klenowed pBIB
</LI></UL><UL><LI><I>Randy Pares and Vidim Gladwell</I></LI><LI> Designed primers pBIB-RB-F, pBIB-RB-R, NOS 3'-Foward Nos3' reverse for sequencing the pBIB plant plasmid.
</LI></UL><UL><LI><I>Elaine</I></LI><LI> Made LB/KAN plates
</LI><LI> Made 50mg/ml stock of KAN
</LI><LI> Made 1X TAE buffer
</LI></UL><UL><LI><I>Matthew</I></LI><LI> Digested and tested more colonies, none worked
</LI><LI> Started over with pBIB and Klenow
</LI><LI> Ethanol precipitated and ligated
</LI><LI> Transformed
</LI><LI> Screened colonies, miniprepped, and digested, ran on gel
</LI><LI> No candidates had EcoRI eliminated
</LI></UL><P><B>Week of May 23-29:</B></P><UL><LI><I>Christian</I></LI><LI> Miniprep on pBIB transformed Top 10 cells using &quot;low copy #&quot; modification
</LI><LI> Digested with EcoR1 and HindIII
</LI></UL><UL><LI><I>Bryson</I></LI><LI> Incubated 50 mL liquid culture of <I>E. coli</I> with pBIB (samples 3).
</LI><LI> Miniprepped sample 3, using modified protocol for large/low-copy plasmids.
</LI><LI> EcoRI digest of uncut sample 3
</LI><LI> Prepared 5 mM dNTP stock
</LI></UL><UL><LI><I>Elaine</I></LI><LI> Incubated 50 mL liquid culture of <I>E. coli</I> with pBIB (sample 4 &amp; sample 5)
</LI><LI> Qiaprep miniprep of sample 4 &amp; sample 5 according to manufacturer's modified protocol for large/low-copy plasmids
</LI><LI> Nanodrop of DNA recovery of miniprepped sample 4 &amp; sample 5
</LI></UL><UL><LI><I>Randy and Vadim</I></LI><LI> Sent pBIB vector to Nevada Genomics Center for sequencing with primers designed during the week of May 16-22
</LI><LI> Designed forward and reverse minimal ccdB primers for PCR and sequencing of ccdB gene
</LI><LI> Maxiprepped pBIB vector using QIAGEN QIAfilter Plasmid Maxi Kit
</LI></UL><UL><LI><I>Matthew</I></LI><LI> Investigated alternative ways of eliminating EcoRI site. 
</LI><LI> Want to try hexameric linkers
</LI></UL><H2><SPAN class="mw-headline" id="JUNE">JUNE</SPAN></H2><P><B>Week of May 30-June 5:</B></P><UL><LI><I>Christian</I></LI><LI> Miniprep Top 10 Cells transformed with pBIB
</LI></UL><UL><LI><I>Bryson</I></LI><LI> HinD3 digest of uncut sample 3
</LI><LI> Ran 0.8% gels of samples 1-5 to verify complete digestion by EcoRI
</LI><LI> Pooled uncut samples 3, 4 and 5 (pBIB-pool)
</LI><LI> Recieved 5 µg of pBIB from Randy and Vadim's maxi-prep (pBIB-maxi)
</LI><LI> EcoRI digest of pBIB-pool and pBIB-maxi
</LI><LI> Ran 0.8% agarose gel of EcoRI digests
</LI><LI> Klenow reactions of pBIB-pool and pBIB-maxi
</LI><LI> Made glycerol stocks of pBIB samples 1-5
</LI></UL><UL><LI><I>Elaine</I></LI><LI> EcoRI digest of uncut pBIB sample 4 and 5
</LI><LI> HinD3 digest of uncut sample 4 &amp; sample 5 as a positive control
</LI><LI> Ran 0.8% gels of samples 1-5 to see if EcoRI digest was successful
</LI><LI> Ethanol precipitation of the EcoRI digests of sample 4 &amp; sample 5
</LI><LI> Nanodrop resulted to a low DNA content
</LI><LI> Worked with Bryson for the EcoRI digest of the 5µg of pBIB maxi-prep
</LI><LI> Ran 0.8% agarose gel of the pBIB maxi-prep EcoRI digest 
</LI><LI> Modified all protocols of the Binary vector 
</LI></UL><UL><LI><I>Randy and Vadim</I></LI><LI> Calculated amount of reagent needed for Deep Vent DNA polymerase reaction (New England Biolabs)
</LI><LI> Programmed thermal cycler for PCR of ccdB gene
</LI><LI> Ran PCR for ccdB
</LI><LI> Prepared LB/KAN Broth
</LI><LI> Gel analyzed resultant ccdB PCR reaction with 1.2% agarose gel
</LI><LI> Reaction was unsuccessful
</LI></UL><UL><LI><I>Matthew</I></LI><LI> Miniprepped pBIB
</LI><LI> Ran EcoRI digests
</LI><LI> Ran ethanol precipitation
</LI><LI> Ligated with several concentrations of hexameric linkers designed to destroy site
</LI><LI> Ran transformation, no colonies
</LI><LI> Believed procedures/conditions ran on ligation and transformation not ideal
</LI></UL><P><B>Week of June 6-12:</B></P><UL><LI><I>Christian</I></LI><LI> Klenow reaction on EcoR1 digested pBIB
</LI><LI> T4 ligation on Klenow products
</LI></UL><UL><LI><I>Bryson</I></LI><LI> Ligation reactions for pBIB-pool and pBIB-maxi
</LI><LI> Digested pBIB-pool and pBIB-maxi again with EcoRI to linearize any unmodified pBIB
</LI><LI> Transformed Top-10 Cells with modified pBIB (designated pBIB#)
</LI><LI> Obtained two colonies after overnight incubation
</LI></UL><LI> Line-streaked the two colonies (pBIB# 1 and pBIB# 2) on an LB-Kan plate 
</LI><UL><LI><I>Randy and Vadim</I></LI><LI> Uploaded pBI101, pBIN19, pBIB-KAN, and ccdB gene to Vector NTI
</LI><LI> Ordered second set of pBIB primers: pBIB-RB-F2, pBIB-RB-F3, pBIB-RB-R2, and pBIB-RB-R3 for sequencing of pBIB
</LI><LI> Modified thermal cycler conditions for PCR of ccdB gene
</LI><LI> Ran PCR for ccdB using HiFi DNA polymerase (Invitrogen)
</LI><LI> Gel analyzed resultant ccdB PCR reaction with 1.2% agarose gel and successfully amplified ccdB gene
</LI></UL><UL><LI> Maxiprepped pBIB vector using QIAGEN QIAfilter Plasmid Maxi Kit
</LI></UL><UL><LI><I>Matthew</I></LI><LI> Miniprepped pBIB
</LI><LI> Ran Digestions and ethanol precipitation
</LI><LI> Ligated with hexameric linkers
</LI><LI> Transformed, had some colonies
</LI></UL><P><B>Week of June 13-19:</B></P><UL><LI><I>Christian</I></LI><LI> EcoR1 digest of ligation product
</LI><LI> Transformed Top 10 Cells with pBIB from ligation and plated
</LI><LI> Inoculated 30 colonies
</LI><LI> UNR mini prep on all 30 colonies
</LI><LI> Digested Samples with HindIII and EcoR1
</LI><LI> QIAGEN mini prep on samples 11,12 &amp;13
</LI><LI> Digested Samples with HindIII and EcoR1
</LI></UL><UL><LI><I>Bryson</I></LI><LI> Prepared liquid cultures of pBIB# 1 and pBIB# 2
</LI><LI> Miniprepped liquid cultures of pBIB#
</LI><LI> EcoRI and HinDIII digests of pBIB#
</LI><LI> Ran agarose gel of pBIB cut and uncut, pBIB# uncut, pBIB# cut with EcoRI and pBIB# cut with HinDIII to confirm the absence of EcoRI site in pBIB#
</LI><LI> Single-colony streaked pBIB# 2 on a fresh LB-Kan plate
</LI></UL><UL><LI><I>Elaine</I></LI><LI> Worked with Chris to incubate 30 liquid cell cultures
</LI><LI> Ran 0.8% gels of all 30 samples
</LI></UL><UL><LI><I>Randy and Vadim</I></LI><LI> Topocloned ccdB gene into TOPO PCR Blunt II vector
</LI><LI> Determined concentration of pBIB maxipreps using PicoGreen analysis
</LI><LI> Single-colony isolated nine colonies of TOPO-cloned ccdB gene
</LI><LI> Miniprepped cultures of ccdB gene in TOPO-clone
</LI><LI> Digested minipreps using EcoRI
</LI><LI> Ran 1% gel for digested minipreps
</LI><LI> Single colony streaked four cell lines of ccdB gene in TOPO-clone: line 3, line 7, line 8, line 9
</LI><LI> Ordered primers for Left Border Repeat of pBIB: pBIB-LB-F and pBIB-LB-R for sequencing of pBIB vector
</LI></UL><UL><LI><I>Matthew</I></LI><LI> Miniprepped candidates and digested them
</LI><LI> Ran on gel, some had strange bands, I want to retest
</LI><LI> Colonies appear to not have eliminated EcoRI
</LI></UL><P><B>Week of June 20-26:</B></P><UL><LI><I>Christian</I></LI><LI> Ran gel of Sample 11
</LI></UL><UL><LI><I>Bryson</I></LI><LI> Selected 3 colonies from pBIB# 2 (2-1, 2-2, 2-3) and streaked on a fresh LB-Kan plate
</LI><LI> Started 20 mL liquid cultures of 2-1, 2-2, and 2-3
</LI><LI> Miniprepped samples
</LI><LI> EcoRI and XbaI digests of samples to confirm that the EcoRI was gone and 2-1, 2-2, 2-3 arose from the same colony
</LI><LI> 0.8% agarose gel of digests
</LI></UL><UL><LI><I>Randy and Vadim</I></LI><LI> Miniprepped isolated cultures of ccdB gene using QIAGEN QIAprep Spin Miniprep Kit
</LI><LI> Nanodropped minipreps
</LI><LI> Digested minipreps using EcoRI
</LI><LI> Ran 1% gel for digested minipreps
</LI><LI> Sent cell lines 7-2 and 8-2 for sequencing to Nevada Genomics Center with ccdB M13 Forward and Reverse primers added
</LI><LI> Analyzed ccdB samples using Vector NTI (Invitrogen)
</LI><LI> Sent cell lines 7-2, 8-2, and 9-1 for sequencing/resequencing to Nevada Genomics Center with ccdB M13 Forward and Reverse primers added
</LI></UL><UL><LI><I>Elaine</I></LI><LI> Primer Design of RD29A
</LI></UL><UL><LI><I>Matthew</I></LI><LI> Attempted to test more candidates from hexameric ligation, no candidates had eliminated EcoRI
</LI><LI> Team reevaluated its standing, given a month of failed attempts to modify pBIB
</LI><LI> We are each assigned new tasks. I will be put in charge of making EYFP and mCherry with plant Kozak
</LI><LI> I design primers to extract parts from iGEM vectors with Kozak sequences
</LI></UL><H2><SPAN class="mw-headline" id="JULY">JULY</SPAN></H2><P><B>Week of June 27-July 3:</B></P><UL><LI><I>Randy and Vadim</I></LI><LI> Prepared ccdB for MaxiPrep using QIAGEN QIAfilter Plasmid Maxi Kit
</LI><LI> Prepared thermal cycler protocol for pBIB vector
</LI><LI> Performed multiple PCR on pBIB vector
</LI><LI> Transformed ccdB into NEB cells unsuccessfully
</LI><LI> Made chlorophenocol resistant agar plates
</LI><LI> Agarose gel analyzed PCR products
</LI></UL><UL><LI><I>Matthew</I></LI><LI> Transformed colonies with iGEM parts, miniprepped.
</LI><LI> Attempted PCR on EYFP, Failed. Reevaluated protocol.
</LI></UL><P><B>Week of July 4-10:</B></P><UL><LI><I>Randy and Vadim</I></LI><LI> Agarose gel analyzed PCR products
</LI><LI> Transformed ccdB into NEB, OmniMax, and DEB cells
</LI><LI> Made Kanamycin resistant LB plates
</LI></UL><UL><LI><I>Elaine</I></LI><LI> Transformed ccdB into NEB, OmniMax, and DEB cells
</LI><LI> Made LB/KAN plates
</LI><LI> Took picture of ccdB Sensitivity Experiment 1 Results
</LI><LI> GFP Primer Design
</LI></UL><P><B>Week of July 11-17:</B></P><UL><LI><I>Elaine</I></LI><LI> Made LB/Amp plates
</LI></UL><UL><LI><I>Randy and Vadim</I></LI><LI> TOPOcloned pBIB fragment
</LI><LI> Attempted to ligate ccdB into iGEM vector pSB1C3
</LI><LI> Single colony streaked ccdB transformation
</LI><LI> Made Chloramphenicol broth
</LI><LI> Performed PCR on pBIB cell lines
</LI><LI> Digested ccdB in iGEM vector pSB1C3 with PstI and EcoRI
</LI><LI> Miniprepped ccdB colonies
</LI><LI> Gel analyzed pBIB PCR product
</LI><LI> Transformed pSB1C3+RFP vector onto Chloramphenicol resistant plates
</LI><LI> Made 50x TAE buffer
</LI><LI> Miniprepped pSB1C3+RFP vector
</LI></UL><UL><LI><I>Matthew</I></LI><LI> Processed Miniprepped iGEM colonies for mCherry and EYFP
</LI><LI> Set up PCR for mCherry, EYFP, and NOS
</LI></UL><P><B>Week of July 18-24:</B></P><UL><LI><I>Elaine</I></LI><LI> PCR of GFP
</LI><LI> Ran agarose gel analysis of GFP PCR product
</LI></UL><UL><LI><I>Randy and Vadim</I></LI><LI> Amplified pBIB fragment using PCR
</LI><LI> Gel analyzed pBIB PCR product
</LI><LI> Attempted ligation of ccdB into iGEM vector pSB1C3
</LI><LI> Transformed ligation into NEB cells
</LI><LI> Selected supposed colonies with ccdB+pSB1C3
</LI></UL><UL><LI><I>Matthew</I></LI><LI> Analyzed PCR fragments on gels for mCherry, EYFP, and NOS
</LI></UL><P> </P><P> </P><P> </P><P>PCR results: worked for eyfp and mcherry with bands around 700bp. 
</P><P>From Left: Lane 1 - Lamda Hind III Mlc. Marker.</P><P>Lane 2 - Uncut iGem vector</P><P>Lane 3 - EYFP</P><P>Lane 4 - EYFP</P><P>Lane 5 - EYFP</P><P>Lane 6 - mCherry</P><P>Lane 7 - mCherry</P><P>Lane 8 - mCherry</P><P>Lane 9 - 100bp ladder</P><P> </P><P> </P><P> </P><P> </P><UL><LI> NOS PCR did not appear to work.
</LI><LI> Performed transformation protocol with TOPO vector
</LI><LI> Performed screening on Kan and miniprepped
</LI></UL><P><B>Week of July 25-31:</B></P><UL><LI><I>Christian</I></LI><LI> Transformed Top 10 cells with pBIB and pMA (containing rd29A)
</LI></UL><UL><LI><I>Elaine</I></LI><LI> GFP Transformation
</LI><LI> Cell Culture of GFP 
</LI><LI> Miniprep of GFP
</LI><LI> EcoR 1 &amp; Pst 1 Digest of GFP 
</LI><LI> Ran 1.2% agarose gel analysis of the GFP digest 
</LI></UL><UL><LI> The &quot;GFP E0040&quot; gel analysis was successful for the expected bands of GFP showed at ~720 bp when digested with EcoR 1 &amp; Pst 1 (labeled Cut #1, Cut #2, Cut#3, Cut#4).
</LI><LI> PCR of GFP
</LI></UL><UL><LI><I>Randy and Vadim</I></LI><LI> Miniprepped ccdB+pSB1C3 colonies
</LI><LI> Digested minipreps with EcoRI and PstI
</LI><LI> Performed colony PCR on pSB1C3 colonies
</LI></UL><UL><LI><I>Matthew</I></LI><LI> Ran Digests and gels for candidates of mCherry, EYFP, and NOS in TOPO
</LI><LI> 1 possible candidate each for mChery, EYFP, and NOS.
</LI><LI> Nos bands did not show but sent one for sequencing anyway
</LI></UL><H2><SPAN class="mw-headline" id="AUGUST">AUGUST</SPAN></H2><P><B>Week of August 1-7:</B></P><UL><LI><I>Christian</I></LI><LI> Designed and ordered primers for wt Arabidopsis isolation of DREB1C promoter region
</LI><LI> Mini prep on pBIB and pMA inoculations
</LI></UL><UL><LI><I>Elaine</I></LI><LI> Ran 1.2% Agarose Gel analysis of GFP PCR product
</LI></UL><UL><LI> The &quot;GFP PCR&quot; gel analysis was successful for the expected bands of GFP + Primers showed at ~ 804bp for all PCR products (labeled 1 ng, 5ng, 10ng, &amp; ~826ng)
</LI><LI> Topocloned GFP PCR product 
</LI><LI> Cell Culture of GFP Topocloned colonies
</LI><LI> Streaked colony plate
</LI><LI> Miniprep of GFP Topocloned colonies
</LI><LI> EcoR I digestion of GFP Topocloned 
</LI><LI> Ligation Test: GFP original vector digested with EcoR I 
</LI></UL><UL><LI><I>Samantha and Christian</I></LI><LI> pMA and pBIB liquid cultures and mini preps
</LI></UL><UL><LI><I>Bryson</I></LI><LI> Designed and ordered primers for the 35S promoter
</LI><LI> Made spectymycin plates
</LI><LI> Transformed DB3.1 cells with pK7FWG2
</LI><LI> Liquid cultures and minipreps to isolate pK7FWG2
</LI></UL><UL><LI><I>Randy and Vadim</I></LI><LI> Grew ccdB 8-2 on Kan. resistant plates
</LI><LI> Gel analyzed colony PCR product from July 29
</LI><LI> Made additional LB broth
</LI><LI> Gel analyzed ccdB+pSB1C3 ligation using EcoRI and Pst
</LI><LI> Ligation was unsuccessful
</LI></UL><LI> Performed PCR cleanup on ccdB
</LI><P><B>Week of August 8-14:</B></P><UL><LI><I>Elaine</I></LI><LI> Ran 1.2% Agarose Gel analysis of GFP Topocloned EcoR I digest
</LI></UL><UL><LI> The &quot;GFP in Topo Vector&quot; gel analysis was successful for the EcoR1 digested (cut samples labeled C1, C2, C3, C4, C5, &amp; C6) samples shows bands at ~804bp (GFP + primers) and the Topo bands at ~ 3,519bp. 
</LI><LI> Ligation Test: Ligated EcoR I digest GFP original vector
</LI><LI> Ligation Test: Transformed Ligation and EcoR I digest of GFP into NEB cells 
</LI><LI> Cell culture and streaked colonies of RFP in PSB1C3 vector 
</LI><LI> Miniprepped RFP in PSB1C3 vector
</LI><LI> Digested both RFP in PSB1C3 vector &amp; GFP Topo Vector together with EcoRI and PstI
</LI><LI> Ethanol Precipitation of Digested RFP &amp; GFP in PSB1C3 &amp; Topo Vectors
</LI><LI> Ligated the RFP &amp; GFP in either PSB1C3 &amp; Topo Vectors
</LI><LI> Transformed RFP &amp; GFP in either PSB1C3 &amp; Topo Vectors by growing in NEB Cells in KAN &amp; Chloramphenicol plates
</LI></UL><UL><LI><I>Samantha and Christian</I></LI><LI> Triple digest of pMA and pBIB with Hind III, Mfe I, and Eco RI
</LI><LI> Ligation of triple digest products
</LI></UL><UL><LI><I>Bryson</I></LI><LI> PCR of pK7FWG2
</LI></UL><UL><LI><I>Randy and Vadim</I></LI><LI> Made and aliquoted ligase buffer, ATP, and DTT for use for ligation
</LI><LI> Ligated ccdB+pSB1C3, did not use EtOH precipitation
</LI><LI> Miniprepped ccdB+pSB1C3 ligation
</LI><LI> Digested and gel analyzed ligation using EcoRI and Pst
</LI><LI> Ligation was unsuccessful
</LI></UL><LI> Ligated ccdB+pSB1C3 using EtOH precipitation
</LI><P><B>Week of August 15-21:</B></P><UL><LI><I>Elaine</I></LI><LI> Selected 20 colonies for cell culture and streaked into KAN &amp; Chloramphenicol plates
</LI><LI> Miniprepped 11 cell cultures that didn't grow in KAN plates which means it has to be GFP in PSB1C3 vectors
</LI><LI> Digested GFP in PSB1C3 vectors with EcoR1 &amp; PstI
</LI><LI> Ran 1.2% Agarose Gels analysis of GFP in PSB1C3 vectors digested with EcoR1 &amp; PstI
</LI></UL><UL><LI> The &quot;GFP in pSB1C3 iGEM Vector&quot; gel analysis was successful for it showed at colony #17 (Digested/Cut sample labeled C17) the GFP (720) + primers [which includes the KpnI (GGT ACC = 6bp), Kozak sequence (AAA AAA AAA ACA = 12bp), &amp; Translationsal Stop codon (TAATAA = 6bp)] band is at ~804bp and the pSB1C3 bands at ~2,072bp. 
</LI></UL><UL><LI><I>Bryson</I></LI><LI> 1.2% gel to confirm presence of amplicon
</LI><LI> Topocloning of amplicon and transformation of Topo vector into DB3.1 cells
</LI></UL><UL><LI><I>Randy and Vadim</I></LI><LI> Miniprepped ligation from Aug. 13
</LI><LI> Digested and analyzed ccdB+pSB1C3 ligation using EcoRI and Pst
</LI><LI> Ligation was unsuccessful
</LI></UL><LI> Ligated ccdB+pSB1C3 using EtOH precipitation
</LI><LI> Made Chlor. resistant plates
</LI><LI> Autoclaved labware
</LI><P><B>Week of August 22-28:</B></P><UL><LI><I>Christian</I></LI><LI> Ran PCR on genomic <I>Arabidopsis thaliana</I> DNA using custom primers
</LI><LI> Ran gel of PCR product, no bands corresponding to DREB1C
</LI></UL><UL><LI><I>Elaine</I></LI><LI> Sequenced colony #11 &amp; #17 of GFP in the PSB1C3 vectors (submitted 4: 11F, 11R, 17F, 17R)
</LI><LI> Made chloramphenicol plates and cell cultures from colony #17 &amp; streaked more of colony #17
</LI><LI> Miniprepped 8 cell cultures and nanodrop
</LI><LI> Ran 1.2% Agarose Gel to check for DNA (GFP in PSB1C3 vector)
</LI></UL><UL><LI><I>Bryson</I></LI><LI> Isolated and streaked 6 colonies
</LI><LI> Liquid cultures and minipreps of samples
</LI><LI> Digest of samples with EcoRI and PstI to confirm presence of insert
</LI></UL><UL><LI><I>Randy and Vadim</I></LI><LI> Miniprepped colonies from ligation from Aug. 20
</LI><LI> Digested and gel-analyzed ccdB+pSB1C3 ligation
</LI><LI> Ligation was unsuccessful
</LI></UL><LI> Ligated ccdB+pSB1C3 using EtOH precipitation
</LI><LI> Made Chlor. resistant plates
</LI><UL><LI><I>Richard and Nick</I></LI><LI>Designed BioBrick-compatible primers to amplify CD2+-Promoter <I>AtMRP3</I> from <I>A. thaliana</I> genomic DNA
</LI></UL><H2><SPAN class="mw-headline" id="SEPTEMBER">SEPTEMBER</SPAN></H2><P><B>Week of August 29-September 4:</B></P><UL><LI><I>Elaine</I></LI><LI> Ran 1.2% Agarose Gel to check for DNA (GFP in PSB1C3 vector)
</LI><LI> Analysis of sequence of GFP (PSB1C3)
</LI><LI> Sequencing analysis of colony #11 &amp; #17
</LI></UL><UL><LI><I>Bryson</I></LI><LI> Grew 20 mL liquid cultures of pSB1C3 and Topo vector to prepare for ligation
</LI><LI> Miniprepped cultures and eluted in 300 microliters
</LI><LI> 50 microliter digests done
</LI><LI> 20 microliters were run on 1.0% agarose gel to confirm complete digestion of both plasmids
</LI><LI> Digested plasmids were added in a roughly 1:1 ratio and ligated in a 40 microliter ligation reaction
</LI></UL><UL><LI><I>Randy and Vadim</I></LI><LI> Digested and gel-analyzed ccdB+pSB1C3 ligation from Aug. 27
</LI><LI> Ligation was unsuccessful
</LI></UL><LI> Performed PCR on ccdB
</LI><LI> Ligated ccdB+pSB1C3 using Phenol:Chloroform cleanup techniques and EtOH precipitation
</LI><UL><LI><I>Matthew</I></LI><LI> Sequence results from first attempt at mCherry,EYFP, and NOS failed. NOS abandoned.
</LI><LI> Started over with new colonies, miniprepped, digested, ethanol precipitated, and ligated for Topo
</LI><LI> Screened colonies on Kan and Amp, selected several to miniprep
</LI><LI> Digested and ran on gels, identified several more candidates for sequencing for mCherry and EYFP
</LI></UL><P><B>Week of September 5-11:</B></P><UL><LI><I>Elaine</I></LI><LI> Sequenced 4 tubes of #17 of GFP in PSB1C3 
</LI><LI> Digested rd29A with Hind III &amp; MfeI-HF (for Chris' Project)
</LI></UL><UL><LI><I>Bryson</I></LI><LI> Transformation of ligation reaction
</LI><LI> Plated on chloramphenicol
</LI><LI> Selected four white colonies and single-colony streaked them
</LI><LI> 4 mL liquid cultures of colonies
</LI><LI> Miniprepped samples and eluted in 50 microliters
</LI><LI> 20 microliter digests with EcoRI and PstI done to ensure insert was present in pSB1C3  
</LI></UL><UL><LI><I>Randy and Vadim</I></LI><LI> Transformed ccdB+pSB1C3 ligation from Sept. 3 into DB3 cells
</LI><LI> Transformation unsuccessful
</LI></UL><LI> Made Chlor. resistant plates
</LI><LI> Made new 3M NaOH stock
</LI><UL><LI><I>Matthew</I></LI><LI> Miniprepped candidates for mCherry and EYFP in Topo vector
</LI><LI> Ran digests and gels on topo candidates and submitted them for sequencing
</LI><LI> Sequencing Results came back positive for mCherry and EYFP in Topo
</LI><LI> mCherry showed a point mutation but it should not have an effect on translation
</LI></UL><UL><LI><I>Richard and Nick</I></LI><LI> Re-amplified CD2+ promoter from A. thaliana genomic DNA with ExTaq polymerase; several unspecific bands were seen on the gel; could be due to high sequence homology in AtMRP family
</LI><LI>Ran PCR product on a long gel and performed a gel extraction to isolate fragments that could be the CD2+ promoter
</LI><LI>Performed diagnostic digest of fragment extracted; fragment did not yield predicted digestion patterm; designed new primers
</LI></UL><P><B>Week of September 12-18:</B></P><UL><LI><I>Christian</I></LI><LI> PCR of DREB1C with custom primers using extaq instead of platinum taq using <I>Arabidopsis thaliana</I> columbian
</LI><LI> Ran gel of PCR product
</LI></UL><UL><LI> Bands showed at ~500 bp for all samples
</LI><LI> Sample 2 selected for Topo cloning
</LI></UL><UL><LI><I>Elaine</I></LI><LI> rd29A(pMA) with RFP(PSB1C3) EcoR1 &amp; Pst I Digest
</LI><LI> Ethanol Precipitation of digested synthetic RD29A + RFP(pMA) with RFP(PSB1C3) 
</LI><LI> Ligation of digested synthetic RD29A + RFP(pMA) with RFP(PSB1C3) 
</LI><LI> Transformation of ligated synthetic RD29A + RFP(pMA) with RFP(PSB1C3)
</LI></UL><UL><LI><I>Randy and Vadim</I></LI><LI> Miniprepped ccdB with Mfe site in TOPO vector
</LI><LI> Miniprepped RFP in pSB1C3
</LI><LI> Digested and gel analyzed ccdB with Mfe samples using EcoRI and Pst
</LI><LI> Digested ccdB in TOPO vector and RFP in pSB1C3 using EcoRI and Pst
</LI><LI> Gel purified ccdB and pSB1C3 fragments from digested samples
</LI></UL><UL><LI> Samantha
</LI><LI> Gel extraction of RD29A 
</LI></UL><UL><LI><I>Matthew</I></LI><LI> Miniprepped, digested, and ran on gels candidates for mCherry and EYFP with Kozak
</LI><LI> Several candidates possible for sequencing for EYFP and 3 candidates for mCherry
</LI></UL><P> </P><P> </P><P>EYFP Results - although it's hard to gel w/o mlc. marker. The bands are the right size around 700bp and iGEM pSB1c3 backbone at 2kb
</P><P>From Left: Lane 1 - 100 bp ladder. Did not show</P><P>Lane 2 - Uncut EYFP candidate 1</P><P>Lane 3 - EYFP candidate 1</P><P>Lane 4 - Uncut EYFP candidate 2</P><P>Lane 5 - EYFP candidate 2</P><P>Lane 6 - Uncut EYFP candidate 3</P><P>Lane 7 - EYFP candidate 3</P><P>Lane 8 - Uncut EYFP candidate 4</P><P>Lane 9 - EYFP candidate 4</P><P> </P><P> </P><P> </P><P> </P><P> </P><P> </P><P> </P><P>mCherry Results - although it's hard to gel w/o mlc. marker. The band in Lane 5 the right size around 700bp with the iGEM pSB1C3 backbone at 2kb
</P><P>From Left: Lane 1 - 100 bp ladder. Did not show</P><P>Lane 2 - Uncut mCherry candidate 1</P><P>Lane 3 - mCherry  candidate 1</P><P>Lane 4 - Uncut mCherry  candidate 2</P><P>Lane 5 - mCherry  candidate 2</P><P>Lane 6 - Uncut mCherry candidate 3</P><P>Lane 7 - mCherry  candidate 3</P><P>Lane 8 - Uncut mCherry candidate 4</P><P>Lane 9 - mCherry  candidate 4</P><P> </P><P> </P><P> </P><P> </P><P><B>Week of September 19-25:</B></P><UL><LI><I>Elaine</I></LI><LI> 09/17/10 Submitted GFP with Kozak sequence (PSB1C3) into registry 
</LI><LI> 09/20/10 Mailed GFP with Kozak sequence (PSB1C3) to MIT.
</LI><LI> Streaked single colonies [rd29A + RFP (PSB1C3)] and did Cell cultures
</LI><LI> Miniprepped [rd29A + RFP (PSB1C3)]and digested with EcoR I and Pst I
</LI><LI> Ran uncut and digested [RD29A + RFP (PSB1C3)] into 1.2 Agarose Gel 
</LI></UL><UL><LI> The &quot;rd29A promoter + RFP in pSB1C3 iGEM Vector&quot; gel analysis was successful for it showed the rd29A + RFP bands at ~1,529 bp and the pSB1C3 bands at ~2,072bp. (Digested/Cut samples with EcoR1 and Pst 1 = labeled C1-2, C1-7, C1-9).
</LI><LI> Submitted [rd29A + RFP (PSB1C3)] for sequencing 
</LI></UL><UL><LI><I>Randy and Vadim</I></LI><LI> Concentrated gel purification sample from Sept. 16
</LI><LI> Ligated ccdB gene into pSB1C3 vector
</LI><LI> Transformed ccdB+pSB1C3 ligation into DB3 cells
</LI><LI> Miniprepped ccdB+pSB1C3 ligation
</LI></UL><UL><LI> Samantha
</LI><LI> PCR RD29A from pMA plasmid
</LI><LI> Confirm PCR reaction via agarose gel
</LI></UL><UL><LI><I>Matthew</I></LI><LI> Miniprepped and submitted candidates for sequencing for mCherry and EYFP with Kozak
</LI><LI> Sequences came back positive for EYFP and mCherry 
</LI><LI> Point mutation in the mCherry but shouldn't affect the translation
</LI></UL><H2><SPAN class="mw-headline" id="OCTOBER">OCTOBER</SPAN></H2><P><B>Week of September 26-October 2:</B></P><UL><LI><I>Christian</I></LI><LI> Topocloned DREB1C PCR product and spread on plates using Kan/Amp counter selection
</LI><LI> 5 colonies selected and inoculated overnight at 37C
</LI></UL><UL><LI><I>Elaine</I></LI><LI> Miniprepped some more [RD29A + RFP (PSB1C3)] samples for submission
</LI></UL><UL><LI><I>Randy and Vadim</I></LI><LI> Digested and gel-analyzed ccdB-pSB1C3 ligation with EcoRI and Pst
</LI><LI> Digest yielded successful ligation
</LI></UL><UL><LI> Made Kan and Amp resistant plates
</LI><LI> Made additional LB broth
</LI><LI> Autoclaved labware and equipment
</LI><LI> Miniprepped ligations from Sept. 23
</LI><LI> Submitted ligation for sequencing at the Nevada Genomics Center
</LI><LI> Made glycerol stock of ligations from Sept. 23
</LI><LI> Transformed ligations from Sept. 23 into NEB cells
</LI><LI> Plates yielded no growth-ccdB ligation successful
</LI></UL><UL><LI> Samantha
</LI><LI> Gel purification of RD29A PCR product
</LI></UL><UL><LI><I>Matthew</I></LI><LI> Submitted Kozak-mCherry and Kozak-EYFP to iGEM
</LI><LI> Cultured and miniprepped some of Elaine's GFP iGEM vector
</LI><LI> Miniprepped, digested, ethanol precipitated, and began ligation on GFP iGEM with 35S in Topo
</LI></UL><P><B>Week of October 3-9:</B></P><UL><LI><I>Christian</I></LI><LI> Mini prep of 5 (DREB1C_1, DREB1C_2 etc) DREB1C Topo construct colonies
</LI><LI> Digested results with EcoR1 and Pst1
</LI><LI> Ran products on 0.8% agarose gel
</LI></UL><UL><LI> Expected bands show at ~550 bp
</LI><LI> Sent samples for sequencing analysis
</LI><LI> Digested DREB1C_2 and RFP_pSB1C3 with EcoR1 and Pst1
</LI><LI> EtOH ppt of completed digest
</LI><LI> T4 ligation (using custom buffer)
</LI><LI> Digested Topocloned DREB1C_2 with EcoR1 and Spe1
</LI><LI> Digested YFP_pSB1C3 with EcoR1 and Xba1
</LI><LI> EtOH ppt of DREB1C/YFP digest products
</LI><LI> T4 ligation (using custom buffer)
</LI><LI> Transformed DREB1C/YFP ligation product into TOP 10 Cells and streaked using Kan/Chlor counterselection
</LI></UL><UL><LI><I>Elaine</I></LI><LI> Analyzed rd29A PCR product ran on an Agarose Gel by Samantha Lee.
</LI></UL><UL><LI> The &quot;rd29A PCR product&quot; gel analysis was successful for the rd29A PCR products of both #1 &amp; #2 showed bands of the rd29A at ~836bp. 
</LI><LI> Topocloned Samantha Lee's PCR product #1 of rd29A
</LI><LI> Streaked single colonies of rd29A (Topo Vector)
</LI><LI> Cell cultured single colonies of rd29A (Topo Vector)
</LI><LI> Minipreped rd29A (Topo Vector) and digested with EcoR 1 and Pst 1. 
</LI></UL><UL><LI><I>Matthew</I></LI><LI> Completed ligation and began transformation for 35S-GFP candidates
</LI><LI> Screened 35S-GFP candidates on chloramphenicol and kanamycin 
</LI><LI> Cultured,miniprepped,digested, and ran on gel 20 candidates
</LI><LI> One candidate possible for sequencing
</LI></UL><P> </P><P>35S-GFP Composite Results - With careful inspection of my gels, I believed Lane 3 was a good candidate. I was looking for 2 bands, the composite at about 1.8kb and the pSB1C3 backbone at 2kb. Although, the image is not clean, I believed I saw a second band emerging in between the molecular markers which were 2kb and 1.6kb, making that emerging band at 1.8kb. Sequencing would prove me right
</P><P>From Left: Lane 1 - 1 kb ladder.</P><P>Lane 2 - Uncut 35S-GFP candidate 1</P><P>Lane 3 - 35S-GFP   candidate 1</P><P>Lane 4 - Uncut 35S-GFP candidate 2</P><P>Lane 5 - 35S-GFP candidate 2</P><P>Lane 6 - Uncut 35S-GFP candidate 3</P><P>Lane 7 - 35S-GFP candidate 3</P><P>Lane 8 - Uncut 35S-GFP candidate 4</P><P>Lane 9 - 35S-GFP candidate 4</P><P>Lane 10 - 35S-GFP candidate 5</P><P> </P><P> </P><UL><LI> Submitted for sequencing 1 35S-GFP candidate
</LI></UL><P><B>Week of October 10-16:</B></P><UL><LI><I>Elaine</I></LI><LI> Ran rd29A (Topo Vector) gel.
</LI></UL><UL><LI> The &quot;rd29A in Topo Vector&quot; gel analysis was successful for it showed the rd29A bands at ~836 bp and the Topo bands at ~3,519 bp. (Digested/Cut samples with EcoR1 and Pst 1 = labeled C5, C6, C7, &amp; C8).
</LI><LI> Digested rd29A (Topo Vector) and RFP (pSB1C3)with EcoR1 and Pst 1.
</LI><LI> Ethanol Precipitation of the rd29A (Topo Vector) and RFP (pSB1C3).
</LI><LI> Ligation of the rd29A (Topo Vector) and RFP (pSB1C3).
</LI><LI> Transformation of the rd29A (Topo Vector) and RFP (pSB1C3) to transform to what we want: rd29A (psB1C3 Vector).
</LI></UL><UL><LI><I>Christian</I></LI><LI> Transformed ligation product into Top 10 cells using Chloramphenicol and Kan selection
</LI><LI> 5 colonies selected and inoculated overnight at 37C
</LI><LI> Miniprep 5 cultures and digested with EcoR1 and Pst1
</LI><LI> Ran digests on 0.8% agarose gel
</LI></UL><UL><LI> Expected bands appeared at ~500 bp for sample 3 and sample 5
</LI><LI> Samples DREB1C_2.3 and DREB1C_2.5 selected and sent for sequencing
</LI><LI> Sequencing data for DREB1C_2.3 had 0 mismatches and was selected
</LI></UL><UL><LI><I>Matthew</I></LI><LI> Sequencing results came back positive for 35S-GFP
</LI><LI> Miniprepped 35S-GFP and made glycerol stock
</LI></UL><UL><LI><I>Richard and Nick</I></LI><LI> Amplified CD2+ promoter from A. thaliana genomic DNA with ExTaq polymerase; several unspecific bands were seen again on the gel; performed gel extraction on fragments of interest
</LI></UL><P><B>Week of October 17-23:</B></P><UL><LI><I>Elaine</I></LI><LI> 30 Cell cultures of rd29A (pSB1C3)
</LI><LI> Minipreped 30 cell cultures expected to have the rd29A (pSB1C3). Concentrations were too low and therefore only 4 colonies were digested. 
</LI><LI> Digested 4 colonies (labeled as seen on the gel below: #1, #3, #13, #15) suspected to have the rd29A in the iGEM(pSB1C3) Vector
</LI><LI> These 4 colonies were ran on a 1.2% Agarose gel for analysis - &quot;rd29A in iGEM (pSB1C3) Vector&quot;
</LI></UL><UL><LI> The &quot;rd29A in pSB1C3 iGEM Vector&quot; gel analysis was NOT successful for the expected bands of the rd29A at ~836bp didn't show up on the digested (cut with EcoR1 &amp; Pst 1) samples labeled C1, C3, C13, &amp; C15.
</LI><LI> Due to the 30 single colonies that was unsuccessful in having the rd29A in pSB1C3 iGEM Vector, I proceeded to test 45 single colonies by using the colony PCR Technique and made sure I streaked the 45 single colonies on LB-Chloramphenicol plates.  
</LI><LI> Ran 45 Colony PCR samples expected to have the rd29A in the iGEM(pSB1C3) Vector on 1.2% Agarose gels
</LI></UL><UL><LI> The &quot;Colony PCR # 21 of rd29A in pSB1C3 iGEM Vector&quot; gel analysis was successful for 1 (colony PCR #21) out of 45 colonies showed an rd29A band at ~836bp.(Band is shown in the Lane labeled #21).
</LI><LI> Minipreped 4 samples of the Colony PCR #21 (labeled 21-1, 21-2, 21-3, 21-4) and streaked them on LB-Chloramphenicol plate. 
</LI><LI> Saved glycerol stocks for 21-3 &amp; 21-4 of the rd29A in pSB1C3 &amp; placed in -80C freezer. 
</LI><LI> Sequenced Colony PCR #21 at the Nevada Genomics Center.
</LI><LI> Minipreped 1 sample of the 1-9 rd29A + RFP in pSB1C3.
</LI><LI> Saved 2 glycerol stocks for the 1-9 rd29A + RFP in pSB1C3. 
</LI></UL><UL><LI><I>Christian</I></LI><LI> Sequencing of DREB1C_3 in pSB1C3 had 0 mismatches and was submitted as part BBaK414007
</LI><LI> 24 colonies selected from DREB1C/YFP transformation and run in colony PCR using custom DREB1C primers
</LI><LI> Products of PCR reaction run on a 0.8% agarose gel
</LI></UL><UL><LI> Band at ~500 bp for colony 24
</LI><LI> Colony 24 was the only positive result
</LI><LI> Colony 24 DREB1C/YFP construct was sent for sequencing
</LI></UL><UL><LI><I>Matthew</I></LI><LI> Submitted 35S-GFP to iGEM
</LI></UL><UL><LI><I>Richard and Nick</I></LI><LI> Performed diagnostic digests on fragments that could potentially be Cd2+ promoter; no fragments yielded positive digestion patter...back to the drawing board
</LI></UL><P><B>Week of October 24-30:</B></P><UL><LI><I>Elaine</I></LI><LI> Added rd29A (pSB1C3) into the registry
</LI><LI> Submitted rd29A (pSB1C3) through the mail.
</LI></UL><P> </P><P> </P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Nevada/Notebook">http://2010.igem.org/Team:Nevada/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Nevada/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Nevada/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Nevada/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Nevada/Notebook&amp;oldid=197058" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>