"
Page
Discussion
View source
History
teams
Log in
 
Team:Nevada/Modeling
From 2010.igem.org
Home
Team profile
Project
Remote Sensors
Long Term Goals
Parts
promoters
reporters
plasmid + term.
composite
ccdB
Registry Submissions
NT Cell Transformations
Results
Protocol
Agro. Trans.
Transgenic Plants
Other Stuff
Modeling
Safety
Notebook
Fundraising & Sponsorships
Plant Summit
Media
Cuckoo Clock
 
Modeling
Introduction
Plant stress responses are often cascades involving hundreds of genes and gene products.  The possible interactions in these cascades are astronomical.  Therefore, the 2010 Nevada iGEM team worked with Bioinformatics Professor, Karen Schlauch, to use a computational method that could quickly analyze possible transcriptional regulation pathways, using either microarray data or data from continuous fluorometry experiments.  In conjunction with the use of tobacco BY-2 (NT1) cells, this method could allow for even greater time efficiency in identifying important aspects of gene networks in plants.  The method was intended to allow for easier identification of promoters useful to the team’s objective of creating remote plant biosensors.  An Arabidopsis cold-stress pathway involving dehydration response element binding proteins (DREB proteins) was the network of choice, as the 2010 Nevada iGEM team had produced BioBrick standards for 2 of the promoters in the network (DREB1C and RD29A).
The method used a Boolean network approach to examine the gene network and its possible regulatory system.  We first viewed transcripts as “on” when above a threshold value and “off” for lesser values.  All Boolean networks that could generate our dataset were generated and evaluated.  In this manner, we were able to look at all possible interactions between genes based on the Boolean approach.
Data
When it became clear that the team would not have sufficient time to perform fluorometry experiments and analyze data by November, it was decided that microarray experiments published to internet databases would have to do.  All data was originally obtained from a 24-hour time course microarray experiment performed by Jian-Kang Zhu, et al, and published on the Gene Expression Omnibus database (Zhu, et al. 2005).  This allowed for a proof-of-concept to see if the Boolean network would support what was known about the DREB1 pathway from published literature.
Click here to see the initial data set
Because this data consisted of only four time points, all eight genes had similar Boolean values at each time point.  Therefore, the Boolean functions for each were essentially the same and numbered in the billions.  This provided little data for interpretation.  Several methods were used to tease out differences in expression, so that the time courses would be sufficiently different.  First, the threshold value for "on" was raised to 2^3.5 rather than 4.  Second, The data was interpolated to estimate extra time points within the 24-hour time course.  Finally, the number of inputs for each gene was limited to four, as it was deemed unlikely that any gene in this network was receiving input from 5 or more.
Click here to see the interpolated data set and associated Boolean functions
Results
The data from the Boolean functions was used with knowledge from literature concerning cold-stress in Arabidopsis.  The literature showed that DREB1A, DREB1B and DREB1C were all known to have a role in cold-stress, that they acted on RD29A, and that they possibly acted on RAP2.1, RAP2.6 and ZAT6 (Yamaguchi-Shinozaki, K. 2006).  Literature also showed that ICE1 seemed to have a significant role in regulating DREB1A, but may not interact strongly with DREB1B or DREB1C (Zhu J-K. 2005).  From this information, a rough idea of what the network might look like was made.
As mentioned elsewhere, the Boolean functions for the initial data set were too numerous to work with.  Curves were fit to the few time points available from the initial data set.  The Boolean functions from this new data set did provide some useful information.  It does support the hypothesis that all three of the DREB1 genes in the network are feeding into RD29A, with the Boolean function: 4 ( 2 ) & ( 3 ) & ( 4 ) & ( 6 ), where 2, 3, and 4 are DREB1s A, B and C respectively and 6 is ZAT6.  The functions also showed that DREB1A was not receiving input from any gene in this network, indicating that DREB1A is probably not regulated by DREB1B or DREB1C.  It was assumed that no input into DREB1A from ICE1 was observed because of difficulty in determining when ICE1 was active or inactive.  The Boolean functions, as well as the expression data, indicate that RAP2.6 is not connected to this network.  However, it is possible that RAP2.6 is primarily regulated post-transcriptionally, which would not be picked up by the Boolean network.  Or, it may be an extremely late-response element only becoming active after 24 hours of freezing, and its increased expression at times after 24 hours may not have been adequately described by extrapolation.
It must be noted that the data used in the Boolean networking was not ideal.  More time points would have greatly increased the efficacy, stressing the desirability of continuous fluorometry experiments.  With the interpolated data, there may have been gross inaccuracies produced, which may have caused functions of the real network to be eliminated.  On the upside, this provides much room for refining the method, should any future teams attempt to use a Boolean approach in computations.  The Nevada team’s advice would be:  use as many genes from a network as are feasible, run microarray/fluorometry experiments at least in triplicate, collect data at many time points or preferably continuously, and before running the computations, minimize the number of possible connections by ruling out relationships between genes that are known to be false.
Acknowledgments
The 2010 Nevada iGEM team would like to thank Karen Schlauch for all of her hard work, performing computational analysis, explaining the concepts of Boolean networking, and working with the team to find biological meaning in the Boolean models.
References
Yamaguchi-Shinozaki, K., Nakashima, K. (2006) Regulons involved in osmotic stress-responsive and stress-responsive gene expression in plants. Physiologia Plantarum, 126: 62-71.
Zhu, J-K., Lee, B., Henderson, D. (2005) The Arabidopsis Cold-Responsive Transcriptome and Its Regulation by ICE1. Plant Cell. Vol. 17, Issue 11, p3155-3175.
Retrieved from " http://2010.igem.org/Team:Nevada/Modeling "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
