<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Duke_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Duke/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_menu displaying_menu"><A href="https://2018.igem.org/Team:Duke"><DIV class="menu_item direct_link">
				HOME
			</DIV></A><DIV class="menu_item">
			TEAM
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Duke/Team"><DIV class="submenu_item">
				 Team Members 
				</DIV></A><A href="https://2018.igem.org/Team:Duke/Advisors"><DIV class="submenu_item">
				 Team Advisors
				</DIV></A><A href="https://2018.igem.org/Team:Duke/Collaborations"><DIV class="submenu_item">
					Collaborations
				</DIV></A></DIV><DIV class="menu_item">
			PROJECT
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Duke/Description"><DIV class="submenu_item">
					Description
				</DIV></A><A href="https://2018.igem.org/Team:Duke/Design"><DIV class="submenu_item">
					Design
				</DIV></A><A href="https://2018.igem.org/Team:Duke/Experiments"><DIV class="submenu_item">
					Experiments
				</DIV></A><A href="https://2018.igem.org/Team:Duke/Notebook"><DIV class="submenu_item">
					Notebook
				</DIV></A><A href="https://2018.igem.org/Team:Duke/Model"><DIV class="submenu_item">
					Model
				</DIV></A><A href="https://2018.igem.org/Team:Duke/Results"><DIV class="submenu_item">
					Results
				</DIV></A><A href="https://2018.igem.org/Team:Duke/Attributions"><DIV class="submenu_item">
					Attributions 
				</DIV></A></DIV><DIV class="menu_item">
			PARTS
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Duke/Parts"><DIV class="submenu_item">
					Parts Overview
				</DIV></A><A href="https://2018.igem.org/Team:Duke/Basic_Part"><DIV class="submenu_item">
					Basic Parts
				</DIV></A></DIV><A href="https://2018.igem.org/Team:Duke/Safety"><DIV class="menu_item direct_link">
				SAFETY
			</DIV></A><A href="https://2018.igem.org/Team:Duke/Human_Practices"><DIV class="menu_item direct_link">
				      Human Practices 
			       </DIV></A><DIV class="menu_item"> 
			AWARDS
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Duke/Public_Engagement"><DIV class="submenu_item">
					Education &amp; Engagement

				</DIV></A><A href="https://2018.igem.org/Team:Duke/Model"><DIV class="submenu_item">
					Model
				</DIV></A></DIV><A href="https://igem.org/2018_Judging_Form?team=Duke"><DIV class="menu_item direct_link">
				JUDGING FORM ⇗ 
			</DIV></A></DIV><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><DIV class="column full_size"><H1> Homology Modeling Overview</H1><P> As mentioned in the project description, our goal is to link five genes from the taxol biosynthesis pathway. In order to better understand the behavior of the proteins that we have isolated from the taxol biosynthesis pathway, we are using homology modeling to learn about active site architecture and catalytic functions. Homology modeling is based on the observation that related protein structures tend to have similar 3-D structures and functions. During homology modeling, 1 or more template proteins are used to identify structurally conserved regions, and to predict structurally variable regions that often include mutations from an already known structure. Through homology modeling, we can learn a lot about details about the protein such as active site architecture, ligand binding, and etc. Usually, when the target sequence is 30-50% similar (30%-50% identical amino acids) to the template sequence, they will share 80%+ shared 3-D structures. During the modeling process, we will be looking for template sequences with 30%+ sequence identity as good models for the target sequence. </P></DIV><DIV class="column full_size"><H3> Tools</H3><P>
PyMol: a software modeling application that allows users to view the 3-d structure of any proteins, including secondary, tertiary, and quaternary structures and the molecular interactions between side chains. It's the main tool that we use to visualize active site, and terminals of proteins. 
</P><P>
ModWeb: a web-based database with all protein templates present and target sequences are matched with reliable models of template proteins.
</P><P>
Literature: previously published scholarly articles on research of proteins that include reliable template sequences for our specific model. Many mutations and active site construction has already been discovered in these articles.
</P></DIV><DIV class="column two_thirds_size"><H3>BAPT/DBAT</H3><P>
Template proteins: 5KJV/5KJS 
Organism: <I>A. thaliana</I>
Belongs to BAHD family of acyltransferases (Members can be identified by sequence homology &amp; universally conserved HXXXD &amp; DFGWG motifs) </P><P><BIG><B>General features</B></BIG>
2 quasi-symmetric N-terminal (1-171 &amp; 374-394) &amp; C-terminal (223-373 &amp; 395-431) domains, connected by a long loop (172-222)
<UL><LI>Each domain has beta sheet core flanked by alpha helices with similar spatial arrangement</LI><LI>The active site is at the interface of the domains</LI></UL></P><P><BIG><B>Structural features</B></BIG>
Tau-nitrogen of His153, H-bond with shikimate 5-O of rho-coumaroylshikimate
<UL><LI><SMALL>A base that deprotonates 5-hydroxyl of the acyl acceptor substrate</SMALL></LI></UL><BIG>Indolic N of Trp371 within H-bond distance of the carbonyl oxygen of rho-coumaroyl CoA </BIG><UL><LI><SMALL>An oxyanion hole that stabilizes negative charge on the tetrahedral intermediate as a leaving group to produce ester product rho-coumaroylshikimate</SMALL></LI></UL><BIG>Thr 369, H-bond with 3-hydroxyl of shikimate moiety</BIG><BIG>Arg 356, salt-bridge with carboxyl of shikimate moiety</BIG></P><P><BIG><B>Distinctive active site conformational states in HCTs</B></BIG>
Apo &amp; Holo AtHCT structures have active site conformational changes 
Catalytic His153 (switchlike conformational shift) <UL><LI><SMALL>Imidazole side chain stabilized in a 180o rotation relative to apo conformation</SMALL></LI><LI>In rho-coumaryol-CoA-bound AtHCT, His153 adopts side chain rotation ~90o to apo state</LI></UL><BIG>Arg 356</BIG><UL><LI><SMALL>In presence of rho-coumaroylshikimate, side chain stabilized inside active site</SMALL></LI><LI>Likely mediated by electrostatic attraction between positively charged guanidinium side chain of arginine side chain &amp; negatively charged carboxyl group of shikimate</LI></UL><BIG>3 loops (L1 (31-34), L2 (357-365), L3 (392-398) )</BIG><UL><LI><SMALL>Shift inward upon binding of various ligands, causing active site to shrink </SMALL></LI></UL></P><P><BIG><B>Universally Conserved Residues</B></BIG>
An arginine handle dictates acyl acceptor specificity in HCT <UL><LI><SMALL>Universal conservation of Arg356 &amp; surrounding residues (354-362)</SMALL></LI></UL><BIG>Other universally conserved residues: Thr369 &amp; Trp371 </BIG></P><P><BIG><B>More roles of Arg 356</B></BIG>
Positively charged Arg356 side chain &amp; negatively charged shikimate electrostatic attraction facilitates binding of acyl acceptor substrate to active site 
Orients substrate’s functional groups properly relative to enzyme’s catalytic machinery 
Contributes to catalytic mechanism <UL><LI><SMALL>Salt bridge formation between arginine side chain and shikimate carboxyl confers acyl acceptor substrate binding affinity</SMALL></LI><LI>Arg356 handle orients shikimate in a catalytically productive conformation in HCT’s active site, increase fraction of productive encounters</LI><LI>Mutating Arg356 resulted in a total loss of native enzymatic activity, mutating Arg356 to negatively charged residues increased specificity toward certain positive non-native substrates</LI></UL></P></DIV><DIV class="column full_size"><H3> badA</H3><P>
Template protein: 4RLQ 
Organism: Rhodopseudomonas palustris (bacteria) 
Class: PFAM00501 (ATP-dependent adenylation enzymes) 
Plants and bacteria employ aroyl CoA thioesters for biosynthesis of specialized metabolites 
Mechanism: ATP-dependent CoA ligases use 2 half-reactions to catalyze thioesterification </P><P><BIG><B>General Features </B></BIG>
BadA has C-terminal/N-terminal domains joined by a flexible hinge <UL><LI><SMALL>badA always in conformation primed for thioesterification</SMALL></LI><LI>Nonconserved active site Lys427, present in badA active site in thiolation conformation, not required for adenylation but necessary for thioesterification</LI></UL><BIG>Our protein of interest 2-Me-BzO has characteristic N-/C-terminal domains of other ATP-dependent adenylases </BIG><UL><LI><SMALL>N-terminal (1-434), C-terminal (435-522) domains fold analogously to the benzoate CoA ligase BCLM (2v7b)</SMALL></LI><LI>N-terminal contains benzoate binding site</LI><LI>C-terminal domain contacts other edge of benzoate binding pocket, positions benzoate into active site through charged interaction between carboxylate of benzoate and Lys427</LI></UL></P><P><BIG><B>Features of active site</B></BIG>
Largely hydrophobic 
Para- carbon of BzO neighbors carbonyl of Leu332 along peptide backbone 
2 meta- carbons of BzO point toward His333 &amp; Ala227 
Si- &amp; re- faces of BzO positioned between backbone amide bonds, comprising Gly327, Ser328, Thr329 on 1 face, Tyr on other face 
When Bz-AMP forms, Lys427 moves from interaction with carboxylate of substrate to several polar contacts with Bz-AMP &amp; peptide backbone 
Indicates Lys427 is needed for thiolation reaction </P><P><BIG><B>Rational mutation of the BadA active site</B></BIG>
4 residues (Ala227, Leu332, His 333, Ile334) surround the phenyl ring of BzO in the active site 
The residue positioned near a given carbon of BzO depends on whether BzO is in carboxylate-bound orientation or rotated Bz-AMP-bound orientation 
Targeted mutations of Ala227Gly, Leu332Ala, His333Ala, Ile334Ala should show relative importance of 2 orientations </P><P><BIG><B>BadA structures &amp; Homology</B></BIG>
Enzymes in this family typically fold in larger N-terminal domain (400-550 residues) &amp; a smaller C-terminal domain (~110 residues), active site lies in between 
N-terminal domain contains residues that bind carboxylate substrate 
C-terminal residues coordinate ribose &amp; phosphate groups 
Phe226 residue in badA offset by ~72o away from BzO, opening CoA binding channel </P><P><BIG><B>Catalytically Important Lysine Residues</B></BIG>
Uses C-terminal lysine (Lys427/512 for BadA) to orient BzO substrate 
Lys427 coordinates BzO in the active site &amp; Lys512 is far from active site and solvent-exposed in C-domain 
Lys427 of BadA makes 4 polar contacts with Bz-AMP, 1 with benzoyl oxygen, 3 with AMP moiety, and 2 to Gly303 &amp; Gly430 (these contacts anchor Bz-AMP) 
Predict Lys427Ala-badA mutant would show the second thioesterification of badA
<UL><LI><SMALL>When badA assumes the adenylation conformation, Lys572 enters the active site</SMALL></LI></UL></P></DIV><DIV class="column two_thirds_size"><H3>Tax10</H3><P>
Template protein:4KE4 
Organism: <I>Sorghum bicolor</I>
hydroxycinnamoyltransferase(HCT) participates in early step of phenylpropanoid pathway
</P><P><BIG><B>Structure of apo-form SbHCT</B></BIG>
domains (I &amp; II) with 16 beta-strands &amp; 17 alpha-helices <UL><LI><SMALL>Domain I (1-193, 389-409)</SMALL></LI><LI>Domain II (194-388, 410-448)</LI><LI>Both domains have mixed beta-sheet flanked by alpha-helices on both sides of the sheet</LI></UL><BIG>
Beta-sheets in core of domain I arranged in 1-6-7-8-3-14 
Beta-sheets in core of domain II arranged in 9-16-15-13-10-11-12 
Domain I has another anti-parallel beta-sheet (2-5-4) perpendicular to core beta-sheets 
Residues of 2 core beta-sheets were hydrophobic surface residues 
5 helices connecting beta12 &amp; beta 13 were interrupted by Gly &amp; Pro residues 
2 regions of high temperature factors in SbHCT </BIG><UL><LI><SMALL>182-236 linked domains I &amp; II </SMALL></LI><LI>C-terminal end of alpha 10 &amp; 257-267 (contained 3 Arg residues) </LI></UL></P><P><BIG><B>Substrate-binding Site</B></BIG>
Visualize position of p-coumaroyl-CoA &amp; shikimate <UL><LI><SMALL>Both located between domains I &amp; II </SMALL></LI></UL><BIG>Binding pocket is a tunnel throughout the enzyme </BIG><UL><LI><SMALL>One side formed by amino end of alpha7 &amp; the carboxy ends of beta10 &amp; beta13</SMALL></LI><LI>Other side formed by amino ends of beta3, alpha15, alpha16, and the loop connecting beta14 and beta15 </LI><LI>Residues lining the walls were hydrophobic</LI><LI>Tunnel had a preference for molecules capable of adopting a linear chain formation</LI></UL><BIG>Pantothenate unit located between beta10 &amp; beta13 of domain II</BIG><UL><LI><SMALL>Carbonyl oxygen of pantothenate interacted with backbone amide of Asp-300</SMALL></LI><LI>Guanidinium side chains of Arg-252 &amp; Arg-304 weakly interacted with phosphate groups of HS-CoA </LI></UL><BIG>Phenolic moiety of p-coumaroyl shikimate anchored to both the hydroxyl side chains of Ser38 &amp; Tyr-40 by an ordered solvent molecule</BIG><BIG>Shikimate moiety also coordinated by the neighboring residues </BIG><UL><LI><SMALL>Hydroxyl group on C3 atom forms H-bond with hydroxyl side chains of Thr-384 </SMALL></LI><LI>Carbonyl group forms salt bridge with guanide group of Arg-371 </LI></UL></P><P><BIG><B>Characterization of steady-state kinetics for SbHCT and its mutant forms</B></BIG>
6 mutants (T36A, S38A, Y40A, H162A, R371A, T384A) 
They contain a similar amount of secondary structures </P><P><BIG><B>Identification of related BAHD family members via distance matrix alignment and BLAST sequences</B></BIG>
All contain HXXXD motif (catalytic His site), some also share DFGWG motif (distal to active site) 
CoA dependent enzymes had 2 mixed beta-sheets at core 
162HHVAD166 located on alpha 6, at interface between 2 domains <UL><LI><SMALL>His 162 at middle of tunnel</SMALL></LI><LI>Carboxyl side chain of Asp-166 (salt bridge with Arg302) </LI></UL><BIG>395DFGWG399 motif located between beta13 and beta14 </BIG><UL><LI><SMALL>Core of domain I &amp; II respectively</SMALL></LI><LI>Likely involved in stabilizing 2-domain structure</LI></UL></P><P><BIG><B>Substrate binding &amp; catalytic mechanism</B></BIG>
Likely p-coumaroyl-CoA &amp; shikimate enter active site on opposite faces of enzymes 
Likely p-coumaroyl-CoA preconditions binding of shikimate 
Binding pocket for shikimate hydrophobic <UL><LI><SMALL>Val31, Pro32, Ala 298, Ile318, Phe376, Leu414, Phe416, Leu418</SMALL></LI><LI>These residues not conserved</LI><LI>2 polar residues Arg371, Thr384 at locations for binding a shikimate molecule </LI><LI>Proposed mechanism with H162 &amp; T36 </LI></UL></P><P><BIG><B>Substrate specificity of SbHCT and other members of BAHD family</B></BIG>
Residue activity in S38A &amp; Y40A is consistent with the subtle role of these residues in substrate binding 
T36A (6%), T384A(9%), S38A(40%), Y40A(62%) reduced activity </P></DIV><DIV class="column two_thirds_size"><H3>TycA</H3><P>
Template proteins: 2VSQ, 5N81, 2XHG, 5ISW 
Organism: Multiple </P><P><BIG><B>Overview</B></BIG>
L-Phe-specific TycA initiation module (TycAf herein) from tyrocidine biosynthesis <UL><LI><SMALL>Adenylation (A), thiolation (T), epimerization (E) domain </SMALL></LI><LI>Preference for native substrate L-Phe </LI><LI>T-domains were subsequently primed with the ppant arm using Sfp 4’-phosphopantetheinyl transferase and coenzyme A (CoASH)</LI></UL></P><P><BIG><B>Reprogramming TycAF for (S)-beta-Phe</B></BIG>
Simultaneous randomization of several positions in the tycA active site 
Share conserved residues Asp235 &amp; Lys517 <UL><LI><SMALL>Asp235 bind backbone amino groups of the substrate</SMALL></LI><LI>Lys517 bind carboxylate group of the substrate</LI></UL></P><P><BIG><B>Catalytic Functions</B></BIG>
Replaced T328-S329-I330-C331 in beta13beta14 loop of TycApgamma with fully randomized tripeptide <UL><LI><SMALL>A236 randomized for structural adjustments </SMALL></LI></UL><BIG>Fully randomized beta13beta14 loop converged to a Xaa-Leu-Val motif </BIG><UL><LI><SMALL>Xaa=Ala, Thr, Cys, Val, Leu</SMALL></LI></UL><BIG>Chose variant TycAbetapgamma with A236V mutation and a Cys-Leu-Valbeta13beta14 loop sequence and auxillary W239S substitution 
These changes inverted the substrate preference in favor of beta-amino acid </BIG><UL><LI><SMALL>Reversion of W239S mutation afforded variant TycAbetaF, 220:1 preference for (S)-Phe over L-Phe </SMALL></LI><LI>This was changed while maintain high catalytic efficiency </LI></UL><BIG>
Structural analysis of alpha/beta switch 
C-terminally truncated A domain of TycAbetapgamma &amp; TycAbetaF 
Invariant residue Asp235 and auxiliary position Trp/Ser239, used to enlarge the binding pocket 
TycAbetaF-AN &amp; A domain of GrsA (a close homologue of TycAf) </BIG><BIG><UL><LI><SMALL>Engineering of position 236 and loop connecting beta-sheets 13&amp; 14 resulted in a reshaped substrate binding pocket that accommodates rotated aryl group </SMALL></LI></UL><SMALL><BIG>Beta-aminoacyl-AMP analogues </BIG><UL><LI><SMALL>AMP moiety in conserved A-domain nucleotide pocket</SMALL></LI><LI>The appended beta-amino acid docked in remodeled substrate recognition site</LI></UL></SMALL><BIG>
W239S mutation enlarges recognition site, providing space for the propargyl substituent used for screening 
Beta-amine of (S)-beta-Phe conserved salt bridge with Asp235, which orients it syn-periplanar to sulfamate NH </BIG></BIG></P><P><BIG><B>Downstream processing of beta-amino acids</B></BIG>
Recombinantly produce tetramodular synthetase GrsB to test TycAbetaF 
Engineered NRPS consisting of TycAbetaf &amp; GrsB reconstituted in E. Coli HM0079 </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>