<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Duke_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Duke/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_menu displaying_menu"><A href="https://2018.igem.org/Team:Duke"><DIV class="menu_item direct_link">
				HOME
			</DIV></A><DIV class="menu_item">
			TEAM
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Duke/Team"><DIV class="submenu_item">
				 Team Members 
				</DIV></A><A href="https://2018.igem.org/Team:Duke/Advisors"><DIV class="submenu_item">
				 Team Advisors
				</DIV></A><A href="https://2018.igem.org/Team:Duke/Collaborations"><DIV class="submenu_item">
					Collaborations
				</DIV></A></DIV><DIV class="menu_item">
			PROJECT
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Duke/Description"><DIV class="submenu_item">
					Description
				</DIV></A><A href="https://2018.igem.org/Team:Duke/Design"><DIV class="submenu_item">
					Design
				</DIV></A><A href="https://2018.igem.org/Team:Duke/Experiments"><DIV class="submenu_item">
					Experiments
				</DIV></A><A href="https://2018.igem.org/Team:Duke/Notebook"><DIV class="submenu_item">
					Notebook
				</DIV></A><A href="https://2018.igem.org/Team:Duke/Model"><DIV class="submenu_item">
					Model
				</DIV></A><A href="https://2018.igem.org/Team:Duke/Results"><DIV class="submenu_item">
					Results
				</DIV></A><A href="https://2018.igem.org/Team:Duke/Attributions"><DIV class="submenu_item">
					Attributions 
				</DIV></A></DIV><DIV class="menu_item">
			PARTS
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Duke/Parts"><DIV class="submenu_item">
					Parts Overview
				</DIV></A><A href="https://2018.igem.org/Team:Duke/Basic_Part"><DIV class="submenu_item">
					Basic Parts
				</DIV></A></DIV><A href="https://2018.igem.org/Team:Duke/Safety"><DIV class="menu_item direct_link">
				SAFETY
			</DIV></A><A href="https://2018.igem.org/Team:Duke/Human_Practices"><DIV class="menu_item direct_link">
				      Human Practices 
			       </DIV></A><DIV class="menu_item"> 
			AWARDS
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Duke/Public_Engagement"><DIV class="submenu_item">
					Education &amp; Engagement

				</DIV></A><A href="https://2018.igem.org/Team:Duke/Model"><DIV class="submenu_item">
					Model
				</DIV></A></DIV><A href="https://igem.org/2018_Judging_Form?team=Duke"><DIV class="menu_item direct_link">
				JUDGING FORM â‡— 
			</DIV></A></DIV><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><DIV class="column full_size"><CENTER><H1>Design</H1></CENTER><H2>BAPT </H2><P>In 2016, the Duke iGEM team ordered a gBlock from IDT containing the dna code for the BAPT gene as well as 20 base pair overhangs with the Lucigen vector, pSMART-HC-Amp Plasmid. Linearized pSMART and gBlock underwent Gibson Assembly to create a circular plasmid </P><H4><CENTER>pSMART-HC-Amp-BAPT Plasmid Map</CENTER></H4><P>This year we amplified the BAPT gene from the above plasmid using primers SL1 and SR2. The PCR was then digested with DPN1 to remove the pSMART vector and digested with XbaI and SpeI. Likewise, PSB1C3-RFP was digested with XbaI, SpeI, and CIP to prevent autoligation. The digested BAPT and PSB1C3 were then ligated with a 6:1 molar ratio to obtain PSB1C3-BAPT. White colonies were picked from the transformation and confirmed with colony PCR and later sequencing.

</P><H4><CENTER>PSB1C3-BAPT Plasmid Map</CENTER></H4><H2>TAX10 </H2><P>In 2016, the Duke iGEM team ordered a gBlock from IDT containing the dna code for the TAX10 gene as well as 20 base pair overhangs with the Lucigen vector, pSMART-LC-Kan Plasmid. Linearized pSMART and gBlock underwent Gibson Assembly to create a circular plasmid </P><H4><CENTER>pSMART-LC-Kan-TAX10 Plasmid Map</CENTER></H4><P>This year we amplified the TAX10 gene from the above plasmid using primers SL1 and SR2. The PCR was then digested with DPN1 to remove the pSMART vector and digested with XbaI and SpeI. Likewise, PSB1C3-RFP was digested with XbaI, SpeI, and CIP to prevent autoligation. The digested Tax10 and PSB1C3 were then ligated with a 6:1 molar ratio to obtain PSB1C3-TAX10. White colonies were picked from the transformation and confirmed with colony PCR and later sequencing.

</P><H4><CENTER>PSB1C3-TAX10 Plasmid Map</CENTER></H4><H2>Modular Design</H2><P>
The end goal of the project was to create a linear stretch of DNA containing all five genes in the pathway that could  be recombineered into the E. coli genome.  In creating this pathway, each gene was viewed as a separate &quot;building block&quot; that would be linked with the other blocks to create a complete pathway.  Within one gene's &quot;building block,&quot; there were several components: the 5' linker arm, the T7 promoter, the coding sequence of the gene, and the 3' linker arm.  The genes were originally designed with 15 base pair linker regions for assembly.  These 15 base pair linker regions would be digested by the nicking endonuclease Nb.BsrDI, creating unique overlaps to link the 5 building blocks together in the proper order (based on <A href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0057943">this</A> protocol.  However, the protocol for nicking endonuclease assembly proved trickier than intended, and the design was revised to include longer linker regions that would allow for assembly in yeast.  This linking was not experimentally implemented.
</P><H2>Promoter Optimization Experimental Design </H2><P>The design of the following elements of the project was laid out, but not experimentally implemented.  
Each of the 5 genes in the pathway was designed to be fitted with a different T7 bacteriophage promoter, from a library of 64 variants of the T7 promoter.  The design of these variants builds upon the work of a previous iGEM team; their work can be found <A href="https://2011.igem.org/Team:EPF-Lausanne/Our_Project/T7_promoter_variants/t7prom">here.</A>
To test the expression strength of the promoter variants, the mCherry gene was to be placed under the control of a T7 promoter in a pSMART vector.  Mutagenic PCR with a random assortment of the 64 promoter variants would generate a library of linear DNA that could be recombineered into the E. coli genome exactly as planned for the entire construct.  The mCherry gene's expression level is quantifiable as fluorescence, so each of the 64 promoters could then be ranked into a library of promoter options. After ranking promoters, 5 promoters of various strengths would be chosen from the library; these promoters would then be used in random combinations (generated by mutagenic PCR on the pathway genes using only 5 mutant primers) on the pathway genes.  The resulting constructs could be recombineered into E. coli and tested for Taxol production levels.  This approach randomizes promoter selection to optimize the metabolic pathway producing Taxol.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>