<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SZU-China_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SZU-China/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV style="height: 56px;background-color: black;opacity: 0.7;padding: 0 1rem;position:relative;z-index:1000;"><A class="navbar-brand" href="https://2018.igem.org/Team:SZU-China"><SPAN class="d-inline-block ml-2">SZU-China</SPAN></A><DIV style="height: 56px;width: 60%;float: right;text-align: right;padding-top: 14px"><A class="nav-1-item" style="color: white;" href="https://2018.igem.org/Team:SZU-China/Applet">Applet</A><A class="nav-1-item father" id="nav1_5" data-id="5" href="https://2018.igem.org/Team:SZU-China/Human_Practices">Human Practices</A><A class="nav-1-item father" id="nav1_4" data-id="4">Experiment</A><A class="nav-1-item father" id="nav1_3" data-id="3">Project</A><A class="nav-1-item father" id="nav1_2" data-id="2">Team</A><A class="nav-1-item father" id="nav1_1" data-id="1">Awards</A></DIV><DIV class="sub_menu son" id="sub1" style="display: none;left: 21%;"><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Achievement">Achievement</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Applied_Design">Applied Design</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Human_Practices">Human Practices</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Public_Engagement">Public Engagement</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Model">Model</A></DIV></DIV><DIV class="sub_menu son" id="sub2" style="display: none;left: 28.6%"><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Team">Team Members</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Collaborations">Collaborations</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Attributions">Attributions</A></DIV></DIV><DIV class="sub_menu son" id="sub3" style="display: none;left: 35.6%"><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Description">Description</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Demonstrate">Demonstrate</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Model">Model</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Improve">Improve</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Parts">Parts</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Safety">Safety</A></DIV></DIV><DIV class="sub_menu son" id="sub4" style="display: none;left: 44.3%"><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Design">Design</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Procedure">Procedure</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Results">Results</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/InterLab">InterLab</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Notebook">Notebook</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Protocol">Protocol</A></DIV></DIV><DIV class="sub_menu son" id="sub5" style="display: none;left: 44.3%"><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Human_Practices">Integrated HP</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Applet">Applet</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/MeetUp">Meetup</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Investigation">Investigation</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Simulation">Simulation</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Public_Engagement">Education &amp; Public engagement</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Prospect">Prospect</A></DIV></DIV></DIV><DIV class="header-nav" style="text-align: center;display: none"><DIV style="width: 100%;height: 45px;text-align: center;" id="header_list"><A class="nav-item1" id="nav0" style="color: white;" href="https://2018.igem.org/Team:SZU-China">Home</A><A class="nav-item1 nav-father" style="display:inline-block;height:45px" id="nav1" data-id="1">Awards</A><A class="nav-item1 nav-father" style="display:inline-block;height:45px" id="nav2" data-id="2">Team</A><A class="nav-item1 nav-father" style="display:inline-block;height:45px" id="nav3" data-id="3">Project</A><A class="nav-item1 nav-father" style="display:inline-block;height:45px" id="nav4" data-id="4">Experiment</A><A class="nav-item1 nav-father" style="display:inline-block;height:45px;color: white;" id="nav5" data-id="5" href="https://2018.igem.org/Team:SZU-China/Human_Practices">Human Practices</A><A class="nav-item1" style="color: white;" href="https://2018.igem.org/Team:SZU-China/Applet">Applet</A></DIV><DIV class="sub_menu nav_son" id="header_nav1" style="display: none;left: 21%;"><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Achievement">Achievement</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Applied_Design">Applied Design</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Human_Practices">Human Practices</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Public_Engagement">Public Engagement</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Model">Model</A></DIV></DIV><DIV class="sub_menu nav_son" id="header_nav2" style="display: none;left: 28.6%"><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Team">Team Members</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Collaborations">Collaborations</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Attributions">Attributions</A></DIV></DIV><DIV class="sub_menu nav_son" id="header_nav3" style="display: none;left: 35.6%"><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Description">Description</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Demonstrate">Demonstrate</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Model">Model</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Improve">Improve</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Parts">Parts</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Safety">Safety</A></DIV></DIV><DIV class="sub_menu nav_son" id="header_nav4" style="display: none;left: 44.3%"><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Design">Design</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Procedure">Procedure</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Results">Results</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/InterLab">InterLab</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Notebook">Notebook</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Protocol">Protocol</A></DIV></DIV><DIV class="sub_menu nav_son" id="header_nav5" style="display: none;left: 44.3%"><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Human_Practices">Integrated HP</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Applet">Applet</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/MeetUp">Meetup</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Investigation">Investigation</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Simulation">Simulation</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Public_Engagement">Education &amp; Public engagement</A></DIV><DIV class="nav-sub-item"><A href="https://2018.igem.org/Team:SZU-China/Prospect">Prospect</A></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row"><DIV class="col-9 offset-1 shadow"><DIV class="text-center"><H1>Results</H1></DIV><DIV class="indent"><P>
							This year, in our project, we constructed a new expression vector working effectively in the fungus and transformed into <I>Metarhizium anisopliae</I> 128 to increase its adhesion, penetration and immune-avoidance capacity. Therefore, we performed experiment on three aspects and the experimental characterizations of our parts are shown as follows.
						</P></DIV><DIV id="Adhesion" class="indent"><H2>Adhesion：HsbA</H2><P>
							In this part, a strong promoter, PgpdA allows the HsbA protein to be expressed without induction.
						</P><P>The HsbA(
							<A href="http://parts.igem.org/Part:BBa_K2788000">BBa_K2788000</A> ) from <I>Beauveria bassiana</I> encodes a kind of membrane surface hydrophobic protein which helps our spores adhere to the wax on the cockroach body surface<SUP>[1]</SUP>. With the overexpression of HsbA, our spores can adhere to the cockroach more effectively . Then it will be followed as spore  germination, formation of germinal tube and appressorium.
						</P><P>This part was insert into the expression vector by restriction sites EcoRI and PstI (Fig.1), and the correct construction of this recombinant plasmid was confirmed by PCR identification and sequencing of the PCR products.
						</P><DIV class="col-6 offset-3"><B>Fig.1</B> Construction of expression vector HsbA-pBC. PgpdA and TtrpC come from parts of 2016_NYMU-Taipei:
								<A href="http://parts.igem.org/Part:BBa_K2040101">BBa_K2040101</A>
								and
								<A href="http://parts.igem.org/Part:BBa_K2040102">BBa_K2040102</A> , and HsbA comes from <I><I>Beauveria bassiana</I></I> ARSEF 2860. The PgpdA-HsbA-TtrpC part is connected to the pBC plasmid through the BioBrick site.</DIV><P>We transformed the expression vectors into <I>Metarhizium anisopliae</I> 128 by  CaCl ₂ -PEG induction method, and the positive clone was screened by G418 and colony PCR.
						</P><DIV class="col-8 offset-2"><B>Fig.2</B> 0.8% Agarose Gel Electrophoresis of DNA extracted from the positive clones and validated by PCR. The product of PCR showed two signal bands at 335 bp and 741bp respectively, which correspond to the length of M.a primer PCR product and HsbA primer PCR product. Lane 1: M.a primer PCR product; Lane 2: HsbA primer PCR product; Lane M: DL marker.
							</DIV><P>The transformed strain <I>Metarhizium anisopliae</I> 128 was grown in 1/4 SDAY liquid medium, and obtain total protein by FastPrep and ultrasonic crushing. The lysate was then centrifuged and the supernate were electrophoresed on a sodium dodecyl sulfate(SDS)-12% (wt/vol) polyacrylamide gel, followed by Coomassie blue staining.(Fig.3)
						</P><DIV class="col-8 offset-2"><B>Fig.3</B> SDS-PAGE analysis of total protein of wild-type <I>Metarhizium anisopliae</I> 128 and modified <I>Metarhizium anisopliae</I> 128. Lane M: Marker Ladder；Lane 1：<I>Metarhizium anisopliae</I> 128；Lane 2 and 3: recombinant strain <I>Metarhizium anisopliae</I> 128. Lane HsbA1 and HsbA2 showed the same band(in the red box) corresponded with the molecular weight of HsbA(24kDa). </DIV><DIV class="row"><P>
								Besides, by using the method of comparing four areas of each wing in the scanning electron microscope before and after treatment which is ‘put the petri dishes on WD-9405B horizontal shaking table and shaked at the lowest speed for 10 minutes (to ensure the spores on the wings impact by the water flow equivalently)’ in the
								<B>HsbA macro verification protocol</B>, we can finally compared whether there was any change of the position and number of spores in the observing area. (illustrated with Fig.4 and Fig.5)
							</P></DIV><DIV class="col-8 offset-2"><B>Fig.4</B> One of the wild-type <I>Metarhizium anisopliae</I> 128 groups: P1 is the picture before treatment and P2 is the picture after treatment. The quantity of spores is 40 in P1 and 102 in P2. And there are only two left that have not been washed away and moved. The adherence rate is 2/40=5.0%

							</DIV><P>
							It’s obvious that it changes a lot（circle in red）of the position and number of spores in the observing area of wild-type <I>Metarhizium anisopliae</I> 128 experimental groups.

						</P><DIV class="col-8 offset-2"><B>Fig.5</B> One of the <I>Metarhizium anisopliae</I> HsbA transformant groups: P3 is the picture before treated and P4 is the picture after treated. The spores quantity is 42 in P3 and 52 in P4. And there are 41 spores left that have not been washed away and moved. The adherence rate is 41/42=97.6%
							</DIV><P>
							It’s obvious that there is almost no change except for the place (circled in red) in the position and number of spores in the observing area of <I>Metarhizium anisopliae</I> HsbA transformant experimental groups.
						</P><P>
							Finally, the following chart (Fig.6) can be obtained by statistical data of four areas in all experimental groups.
						</P><DIV class="col-6 offset-3"><B>Fig.6</B> M.a128 is the wild-type <I>Metarhizium anisopliae</I> 128 groups, and the adherence rate in average is 26.7%. HsbA is the <I>Metarhizium anisopliae</I> 128 HsbA transformant groups, and the adherence rate in average is 97.7%.
							</DIV><P>
							In conclusion, this result confirmed that <I>Metarhizium anisopliae</I> HsbA transformant certainly enhanced the capacity of adhesion.

						</P></DIV><DIV id="Penetration" class="indent"><H2>Penetration: Bbchit</H2><P>In order to make <I>Metarhizium anisopliae</I> penetrate the cockroach's body wall more efficiently, we transferred Bbchit(
							<A href="http://parts.igem.org/Part:BBa_K2788001">BBa_K2788001</A> ), a gene from <I><I>Beauveria bassiana</I></I> ARSEF 2860, which encodes chitinase that can be secreted extracellularly<SUP>[2]</SUP>. Chitin decomposes, thereby cockroach body wall destroys, finally fungus enters the hemolymph.(Fig.7)
						</P><DIV class="col-6 offset-3"><B>Fig.7</B> Construction of expression vector Bbchit-pBC. PgpdA and TtrpC come from parts of 2016_NYMU-Taipei:
								<A href="http://parts.igem.org/Part:BBa_K2040101">BBa_K2040101</A> and
								<A href="http://parts.igem.org/Part:BBa_K2040102">BBa_K2040102</A>, and Bbchit comes from the <I>Beauveria bassiana</I> ARSEF 2860. The PgpdA-Bbchit-TtrpC part is connected to the pBC plasmid through the BioBrick site.
							</DIV><P> We constructed a shuttle vector to transform Bbchit and the positive clones were screened by G418 and colony PCR.(Fig.8)
						</P><DIV class="col-8 offset-2"><B>Fig.8</B> 0.8% Agarose Gel Electrophoresis of DNA extracted from the positive clones and validated by PCR. The product of PCR showed two signal bands at 335bp and 1044bp respectively, which correspond to the length of M.a primer PCR product and Bbchit primer PCR product. Lane 1: M.a primer PCR product; Lane 2: Bbchit primer PCR product; Lane M: DL marker.
								</DIV><P>The crude enzyme solution was obtained by cell disruption using ultrasonic,centrifuged to gain supernatant, followed by SDS-PAGE protein electrophoresis and Coomassie blue staining.(Fig.9)
						</P><DIV class="col-8 offset-2"><B>Fig.9</B> SDS-PAGE analysis of total protein of wild-type <I>Metarhizium anisopliae</I> 128 and modified <I>Metarhizium anisopliae</I> 128. Lane M: Marker Ladder；Lane 1：<I>Metarhizium anisopliae</I> 128；Lane 2: recombinant strain <I>Metarhizium anisopliae</I> 128. Lane 2 showed the band(in the red box) corresponded with the molecular weight of Bbchit(38kDa).</DIV><P>
							To determine the activity of chitinase, we improved a method based on the DNS colorimetric method(Kan Zhuo<SUP>[3]</SUP>, Xiaozhen Shi<SUP>[4]</SUP>). First, the standard curve was drawn with different concentration gradients of glucose solution.Second, took 0.5 ml culture solution of the wild-type and transformant grown for 1, 3, 5, 7, 9 and 12 days for enzyme activity test respectively,obtained the crude enzyme solution by filtering the culture solution. Then mixed the crude enzyme solution with 1% colloidal chitin, incubated at 37℃ for 60 minutes, and then added 0.5 mL DNS.Set in boiling water bath for 10 minutes. The absorbance of sample was measured by spectrophotometer and the enzyme activity was calculated according to the standard curve. There were three replicates of each group,perform three parallel experiments in each replicate, and the final data were averaged.
						</P><P>We calculated activity based on the standard curve formula: U=（A540+0.03279）/2.202 (Fig.10), a summary of the data at different times is made into a line chart as follows.We can see that after 12 days the transformant’s enzyme activity is still growing while the wild-type is falling.(Fig. 11)
						</P><DIV class="row"><DIV class="col-5 offset-1"><B>Fig.10</B> Glucose standard curve</DIV><DIV class="col-5 offset-1"><B>Fig.11</B> changes in chitinase activity over time.</DIV></DIV><P>In order to verify the function of Bbchit at the macroscopical level, we improved Kan Zhuo's chitin transparent circle method for verification<SUP>[3]</SUP>. We stained the czapek solid medium without chitin colloids in red with 0.1% Congo red dye solution and then inoculated wild-type and transformed Metarhizium.Then compared the size of the colony and the transparent circle between wild type and transformant. The size measured is the diameter of the chitin transparent ring (R2) and colony(R1). The ratio of the diameters showed as R2/R1.The conclusion that the chitinase activity of <I>Metarhizium anisopliae</I> is enhanced.
						</P><DIV class="col-8 offset-2"><B>Fig.12</B> A: wild-type <I>Metarhizium anisopliae</I> 128 produced transparent zone on Czapek chitin - induced medium; R1: 8mm, R2: 9mm; R2/R1=9/8 B: <I>Metarhizium anisopliae</I> HsbA transformant produced transparent zone on Czapek chitin - induced medium; R1’: 8mm, R2’: 12mm; R2’/R1’=12/8=3/2

							</DIV><P>Therefore, these results confirmed that the chitinase activity of <I>Metarhizium anisopliae</I> The enzyme activity of Bbchit transformant is about 1.3 times higher than wild-type's <I>Metarhizium anisopliae</I> 128. Our modified fungus certainly enhanced the ability of penetration.
						</P></DIV><DIV id="Immune_avoidance" class="indent"><H2>Immune avoidance: MCL1</H2><P>
							During evolution, insects have developed a very strong immune system against entomopathogenic fungi<SUP>[5]</SUP> In hemolymph, hemocytes, and plasma play important roles in eliminating the fungi that are present in the hemolymph,<SUP>[6]</SUP> But arm race between parasitic fungi and their insect hosts never stops. The fungus has mechanisms to overcome the immune systems of insects.,for example, the lower level of β-1,3-glucan in the cell surface does not stimulate the host immune response<SUP>[7]</SUP>. We extract a gene named MCL1 from <I>Metarhizium robertsiiARSEF</I> 23, which encodes collagen-like protein to combine and block β-1,3,-glucan.<SUP>[3]</SUP>This gene works like putting an “invisible cloak ”on the fungus so that immune avoidance can happen.

						</P><P>
							In this part, PgpdA is a strong promoter that allows MCL1 to be expressed without induction. of hemolymph.
						</P><DIV class="row"><DIV class="col-8 offset-2"><B>Fig.13</B> Construction of expression vector MCL1-pBC. PgpdA and TtrpC come from parts of 2016_NYMU-Taipei: BBa_K2040101 and BBa_K2040102, and MCL1 come from the Metarhizium robertsii ARSEF 23. The PgpdA-Bbchit-TtrpC part is connected to the pBC plasmid through the BioBrick site.

								</DIV></DIV><P>
							We transferred the expression vector MCL1-pBC by CaCl ₂ -PEG induction method, then screen transformant by G418 resistance genes and colony PCR.PCR product was identified by agarose gel electrophoresis (Fig.14)

						</P><DIV class="row"><DIV class="col-8 offset-2"><B>Fig.14</B> 0.8%Agarose Gel Electrophoresis of colony PCR product of the positive clones. The PCR product showed two signal bands at 335 bp and 1817bp respectively, which correspond to the length of M.a primer PCR product and MCL1 primer PCR product. Lane 1: M.a primer PCR product; Lane 2: MCL1 primer PCR product; Lane M: DL marker.
								</DIV></DIV><P>
							The transformed strain <I>Metarhizium anisopliae</I> 128 was grown in 1/4 SDAY liquid medium, and obtain total RNA by using RNAiso Plus(TAKARA), reverse transcription by using TAKARA PrimeScript™ RT reagent Kit, then perform quantitative PCR. (Fig.15)

						</P><DIV class="row"><DIV class="col-8 offset-2"><B>Fig.15</B> quantitative PCR analysis demonstrating WT <I>M.anasopliae</I> and genetically enhanced M.anasopiae mRNA levels, which was increased significantly in transformant. (p
									&lt;0.05) </DIV></DIV><P>
							We gain the total protein by FastPrep and ultrasonic crushing. The lysate was then centrifuged and the supernate was electrophoresed on a sodium dodecyl sulfate(SDS)-12% (wt/vol) polyacrylamide gel, followed by Coomassie blue staining.

						</P><DIV class="row"><DIV class="col-8 offset-2"><B>Fig.16</B> SDS-PAGE analysis of total protein of wild-type <I>Metarhizium anisopliae</I> 128 and genetically enhanced <I>Metarhizium anisopliae</I> 
									128. Lane 1: Marker Ladder；Lane 2：<I>Metarhizium anisopliae</I> 128；Lane 3: recombinant strain 
									<I>Metarhizium anisopliae</I> 128. Lane 2 showed the band corresponded with the molecular weight of collagen-like protein(60.4kDa).

								</DIV></DIV><P>
							In order to verify the ability of immune-avoidance of <I>M.anasopliae</I>,we inject hyphae homogenate into cockroaches as experimental groups and normal saline as control groups, then extract hemolymph in 0.5h, 1h , 2h, 4h, 8h, 12h and 24h. Meanwhile extract hemolymph from cockroaches that were not injected as group 0h. Count the nodules formed of hemocytes(Criteria: more than 10 hemocytes assemble closely)(Fig.17) and observe hemolymph smear under phase contrast microscope(Fig.18)
							&lt;4&gt;

						</P><DIV class="row"><DIV class="col-8 offset-2"><B>Fig.17</B> Change of nodules formed of hemocytes in cockroach hemolymph after injecting hyphae homogenate. At the time point of 0.5h,1h and 8h, the nodules caused by WT <I>M.anasopliae</I> is significantly higher than transformant which means immune-avoidance occurs in genetically enhanced <I>M.anasopliae</I>.(p
									&lt;0.05) </DIV></DIV><DIV class="row"><DIV class="accordion" style="width: 100%;" id="accordionExample"><DIV class="card" style="width: 100%;"><DIV class="card-header" id="headingOne"><H5 class="mb-0"><BUTTON class="btn btn-link" style="width: 100%;" type="button" data-toggle="collapse" data-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
          Fig.18 </BUTTON></H5></DIV><DIV id="collapseOne" class="collapse" aria-labelledby="headingOne" data-parent="#accordionExample"><DIV class="card-body"><DIV class="col-12"><P>
													Fig.18. Immune response of cockroaches’hemocytes to <I>Metarhizium anisopliae</I> spores under phase 
													contrast microscope.A.B.C. hemolymph without injection. The number of hemocytes is relatively large 
													and the shape is normal. Almost no nodule formation; D,G,J,M,P,S,V:control,almost no nodules formed;
													E:0.5h of WT injection,blood cells aggregated to form nodules,which means immune response on hypha;F:
													0.5h of transformant injection,almost no nodule formed;H:1h of WT injection,more nodules was formed;I
													:1h of transformant injection, nodules started to form but was less than WT group;K:2h of WT injection
													, nodules were becoming larger;L:2h of transformant injection, nodules were smaller than WT group;N:4h
													 of WT injection, nodules were less and smaller than 2h;O:4h of transformant injection, almost no 
													 nodules formed;Q:8h of WT injection, nodules were Most disintegrating.R:8h of transformant injection,
													  Metarhizium anisopliae was growing in the shape of yeast;T:12h of WT injection, the number of 
													  hemocytes and nodules decreased; U:12h of transformant injection, blood cells were     
													  destroyed and cell debris can be seen;W:24h of WT injection, blood cells’ structure were destroyed, 
													  a large amount of cell debris cen be seen; X:24h of transformant injection, Metarhizium aerated in 
													  large numbers and few blood cells left. Wild-type M.anasoplise triggers more intense immune response 
													  than MCL1 transformant, while hemocytes degraded more in transformant groups. 
												</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><P>
								In conclusion, MCL1 could trigger host immune response less and promote immune-avoidance.
							</P></DIV></DIV><DIV id="switch" class="indent"><H2>Suicide switch</H2><P>
							In order to confirm the limited concentration of Tryptophan, we did a macro experiment. We put Metarhizium in an L-Tryptophan concentration gradient Petrie dishes from 0.05% to 0.14%. with solid and liquid czapek. After six days of cultivation, it can be seen in the

						</P><DIV class="row"><DIV class="accordion" style="width: 100%;" id="accordion"><DIV class="card" style="width: 100%;"><DIV class="card-header" id="heading1"><H5 class="mb-0"><BUTTON class="btn btn-link" style="width: 100%;" type="button" data-toggle="collapse" data-target="#collapse1" aria-expanded="true" aria-controls="collapse1">
          following results: </BUTTON></H5></DIV></DIV></DIV></DIV><P>
							We can see the ten photos that, in the solid czapek, the Metarhizium can stay alive at the L-Trp concentration of 0.09% or higher while it could not grow well or die at a lower concentration. We can also see that Metarhizium could not survive without L-Tryptophan.

						</P><P>
							We put the Metarhizium in 100mL liquid czapek with different concentration L-Tryptophan, after 6 days of cultivation, here is the result:

						</P><DIV class="row"><DIV class="col-10 offset-1"><DIV class="card text-center">
										Chart 1. Dry Weight of Metarhizium in different concentration of L-Tryptophan
									</DIV></DIV></DIV><DIV class="row"><DIV class="col-8 offset-2"><B>Fig.18</B> Dry Weight of Metarhizium in different concentration of L-Tryptophan
								</DIV></DIV><P>
							We can see the chart and graph that, in the liquid czapek, the Dry Weight of Metarhizium stay in a stable level from 0% of [Trp] to 0.08% of [Trp], because it didn’t grow at this low concentration. At the L-Trp concentration of 0.09% or higher, its dry weight grow as the concentration of L-Tryptophan grow.

						</P><P>
							From our experiment we can see that, the limited Tryptophan concentration for Metarhizium to survive is 0.9% of Tryptophan.
						</P></DIV><DIV id="References" class="indent"><H2>References</H2><P>
                                                        [1] Ye Zhang, Zhongren Lei, Haihong Wang, Jiqing Zhan. Prokaryotic expression and immunolocalization Beauveria bassiana HsbA protein [J] Chinese Agricultural Sciences, 2013,46 (21): 4534-4541.
                                                </P><P>
                                                        [2] Xiao G, Ying S-H, Zheng P, et al. Genomic perspectives on the evolution of fungal entomopathogenicity in Beauveria bassiana. Scientific Reports. 2012;2:483. doi:10.1038/srep00483.
                                                </P><P>
                                                        [3] Kan Zhuo, Li Yu, Jinling Liao. Paecilomyces lilacinus MD1 chitinase activity assay [J] Agricultural Sciences, 2009 (05): 167-170.
                                                </P><P>
                                                        [4] Xiaozhen Shi, Min Wang, Huaping Huang, Lijia Guo, Wei Song, Junsheng Huang. Analysis of Chitinase Activity of Metarhizium anisopliae and Its Correlation with Toxicity of Coleoptera[J]. Guangxi Agricultural Sciences,2008(03):313-316.
                                                </P><P>
							[5] Gottar M, Gobert V, Matskevich AA, Reichhart JM, Wang CS, Butt TM, et al. Dual detection of fungal infections in Drosophila through recognition of microbial structures and sensing of virulence factors. Cell. 2007; 127(7): 1425–1437.

						</P><P>
							[6] Glupov VV. Mechanisms of resistance of insects, in: Glupov V. V. (ed.) Insect Pathogens: structural and functional aspects, Moscow, Kruglyi god, 2001, pp 475–557

						</P><P>
							[7] Wang C, St Leger RJ,A collagenous protective coat enables <I>Metarhizium anisopliae</I> to evade insect immune responses.Proc Natl Acad Sci U S A. 2006 Apr 25; 103(17):6647-52.
						</P></DIV></DIV><DIV class="col-2"><NAV id="myScrollspy" class="navbar position-fixed"><NAV class="nav nav-pills flex-column"><A class="nav-link" href="#Adhesion">Adhesion: HsbA</A><A class="nav-link" href="#Penetration">Penetration: Bbchit</A><A class="nav-link" href="#Immune_avoidance">Immune avoidance: MCL1</A><A class="nav-link" href="#switch">Suicide switch</A><A class="nav-link" href="#References">References</A></NAV></NAV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>