<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tuebingen_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tuebingen/Design</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV class="menu" id="menu" style="left:8.7vw"><DIV class="menuitem" style="width:10vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen';">Home</DIV><DIV class="menuitem" style="width:10vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Team';">Team</DIV><DIV class="menuitem" style="width:10vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Description';">Project</DIV><DIV class="menuitem" style="width:10vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Modeling';">Modeling</DIV><DIV class="menuitem" style="width:18vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Practices';">Human Practices</DIV><DIV class="menuitem" style="width:3.8vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Art_and_Design';">Art</DIV><DIV class="menuitem" style="width:18vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Attributions';">Attributions</DIV></DIV><DIV id="submenuProject" class="submenu" style="left:9.6vw;"><DIV class="menuitem" style="width:14vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Experiments';">Experiments</DIV><DIV class="menuitem" style="width:10vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Results';">Results</DIV><DIV class="menuitem" style="width:10vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Design';">Design</DIV><DIV class="menuitem" style="width:10vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Parts';">Parts</DIV><DIV class="menuitem" style="width:10vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Safety';">Safety</DIV><DIV class="menuitem" style="width:14vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Measurement';">Interlabstudy</DIV><DIV class="menuitem" style="width:10vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Notebook';">Notebook</DIV></DIV><DIV id="submenuAttributions" class="submenu" style="left:34.6vw;"><DIV class="menuitem" style="width:15vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Collaborations';">Collaboration</DIV><DIV class="menuitem" style="width:15vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/References';">References</DIV></DIV><DIV id="submenuPractices" class="submenu" style="left:27.4vw;"><DIV class="menuitem" style="width:12vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/synbio';">SynBio-Day</DIV><DIV class="menuitem" style="width:16vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/schoolclassatlab';">SchoolClass@lab</DIV><DIV class="menuitem" style="width:16vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/freiburg';">Theatre Freiburg</DIV></DIV><DIV id="contentWrapper"><DIV id="arrowLeft" class="arrow" style="left:-7vw;" onclick="diashow(-1);">&lt;</DIV><DIV id="arrowRight" class="arrow" style="right:-7vw;" onclick="diashow(1);">&gt;</DIV><DIV id="contentContainer"><DIV id="dia0" class="dia" style="text-align:block;"><H2>Introduction</H2><P>As we planned to build a very sensitive measurement system, we looked for a set of plasmids that allow a continuous and stable expression of our proteins, and offer a choice of several yeast selectable markers. In the iGEM registry we found the pTUM100-104 vector series, which was established by Team TU Munich in 2012, and the pRS315 vector with different promoters and terminators from Team UCSF in 2008.</P><P>We decided to use the pTUM vector series to characterize our simple parts since these plasmids are RFC25 compatible, replicate to high-copy numbers in E.coli and yeast and contain built-in terminators and different promoters. Unfortunately, the pTUM plasmid series only has the URA3 selection marker. We therefore cannot use the pTUM vectors to characterize our composite parts (Caged-Cre and Stop-Cassette), which need to be cotransfected. For our complex parts we decided to use the pRS plasmid system. </P><H2>The pRS vector series</H2><P>These vectors encode an ampicillin resistance gene and replicate to high-copy numbers in E.Coli. Furthermore, the CEN6/ARS4 (Chromosome VI centromere/Autonomously Replicating Sequence 4) cassette ensures that the plasmid stays at a low copy number in yeast, because they are treated as pseudo chromosomes (1-2 plasmids/cell). pRS315 is part of a vector series containing four different plasmids (pRS313-316). Each contains a different auxotrophy marker HIS3, TRP1, LEU2 and URA3 respectively,  (compare figure 1). Since the pRS315 and the other pRS plasmids are not compatible with the BioBrick standard, we decided to alter the pRS313, 315 and 316 to make them RFC10 compatible. <A href="https://2015.igem.org/Team:Tuebingen/References">[Sikorski 1989]</A></P><FONT size="2">Figure 1: Wild type pRS plasmid maps. Important features are shown and the important RFC10 restriction sites are highlighted.</FONT><H2>Neccessary changes to the vectors</H2><P>First, mutagenesis of the RFC10 restriction sites and the ApaI restriction site in the backbone of the pRS plasmids (compare figure: 2) was necessary for the execution of the following steps. </P><FONT size="2">Figure 2: Mutagenized pRS plasmid maps. Important features are shown and the important restriction sites are highlighted.</FONT><P>After digestion of the vector using the ApaI and the SacI restriction sites and the insertion of short annealed oligos with the RFC10 restriction sites in the correct order (compare figure 3) is possible.</P><FONT size="2">Figure 3: Inserted annealed primer containing RFC10 multiple cloning site with adjacent restriction sites.</FONT><P>After inserting the new RFC10 MCS and thereby biobricking the vector, it is possible to digest the vector with PstI and SacI in order to integrate the alcohol dehydrogenase terminator (tADH) in the pRS vectors. The restriction sites can be added to the ADH terminator by overhang PCR amplification (compare figure 4).</P><FONT size="2">Figure 4: pRS plasmid maps with integrated RFC10 MCS. Important features are shown and the important restriction sites are highlighted.</FONT><P>To increase versatility, future approaches will be to implement different promoters using the ApaI and EcoRI restriction sites. </P></DIV><DIV id="dia1" class="dia" style="text-align:block;"><H2> Introduction </H2><P>In our sensor memory system a light activatable Cre recombinase is used to save the state of the sensor in the DNA. The main challenge we had to overcome for this part of our project, was to find a way to control the activity of the Cre recombinase using light. Luckily, we found that by fusing two copies of the protein Dronpa to the N- and C-terminus of the Cre recombinase, the protein should become light activatable.</P><H2> Dronpa </H2><P>
Dronpa is a monomeric fluorescent protein with high similarity to GFP that is derived from the multimeric wild type protein of the Pectiniidae, which is a species of coral <A href="https://2015.igem.org/Team:Tuebingen/References"> [Ando 2004]</A>. Compared to other fluorescent proteins Dronpa is of special interest because it shows a photoswitchable behaviour. Illumination of the protein at around 488nm leads to bleaching of the protein inducing a ‘dark state’. Furthermore this transition is reversible, because illumination of the dark state protein at around 405nm recovers almost the full fluorescence potential <A href="https://2015.igem.org/Team:Tuebingen/References"> [Ando 2004]</A>.</P><P>Recently a mutant version of Dronpa was engineered that did not only show photoswitchable fluorescence but also oligomerization behaviour <A href="https://2015.igem.org/Team:Tuebingen/References"> [Zhou 2012] </A>. This mutation is lysine-145 to asparagine and it is located within the beta strand 7 of Dronpa, that is part of the oligomerization interface <A href="https://2015.igem.org/Team:Tuebingen/References"> [Mizuno 2008] </A>. Because a change of the fluorophore state within Dronpa also affects beta strand 7 <A href="https://2015.igem.org/Team:Tuebingen/References"> [Mizuno 2010] </A>, the oligomerization behaviour of 145N Dronpa can be controlled by illumination <A href="https://2015.igem.org/Team:Tuebingen/References"> [Zhou 2012] </A>. 
This control over quaternary protein structure can be used to control the activity of engineered proteins. Fusion constructs of an enzymatic or effector protein that carry a Dronpa protein at both their N- and C-terminus were shown to be inhibited as long as the Dronpa copies were fluorescent and olgimerising, but regained their activity upon turning off the Dronpa fluorescence <A href="https://2015.igem.org/Team:Tuebingen/References"> [Zhou 2012] </A>.
</P><H2>Cre</H2><P>The DNA recombinase Cre is a very useful tool for researchers who want to change the genomic makeup of living tools. We choose the Cre recombinase for its ability to cut a DNA region flanked by the loxp recognition sites, which allows us to use this permanent change in the DNA for our memory system.
While this application of the Cre recombinase is rather novel, the proteins has been used for years in engineering of genomes especially for the generation of knock out mice <A href="https://2015.igem.org/Team:Tuebingen/References"> [Nagy 2000] </A>. In eukaryotic cells the activity of the Cre protein can be inhibited by preventing it from entering the nucleus up to a certain time point. This can be achieved by fusing the Cre protein to a tamoxifen inducible receptor domain like ERT2 <A href="https://2015.igem.org/Team:Tuebingen/References"> [Graumann 2000] </A>. These variants of the Cre recombinase are very useful for the generation of conditional knock out lines that offer many more options to researchers.</P><P>Our goal is to apply the caging mechanism with Dronpa as described earlier to the Cre recombinase (compare figure 1). We needed a light activatable protein for our project, because the system should be able to take the momentary state of the sensor and immediately write it into the DNA. This is only possible through light induced activation of the writing process.</P><P>Further on a light inducible Cre recombinase would be very useful for researchers working with the Cre recombinase system, e.g. in mouse knockout models. While inducible tissue specific knockouts are already possible <THROUGH the="" use="" specific="" promoters="" and="" tamoxifen="" inducible="" cre="" recombinase="">, up to now no systems have been described that allow a light inducible or locally limited activation generation of gene knockouts be Cre recombinase activation. We hope that our work will be able to allow the generation of such systems that could be used to study the effects of genes within a cell population or between neighbouring cells.
</THROUGH></P><P>For the generation of the Dronpa caged Cre recombinase we will use biobricks containing the 145N Dronpa, wild type Cre recombinase, Delta1-19 Cre recombinase and serine-glycine linkers of different length. After discussions with experts on protein structure prediction (Birte Höcker, MPI für Entwicklungsbiologie) we decided to try out different linkers instead of trying to predict a possibly working solution. During these discussions we noticed that the N-terminal part of the Cre recombinase (AA1-19) seemed to be rather flexible, because it was not part of the crystal structure we looked (<A href="http://www.rcsb.org/pdb/explore.do?structureId=3MGV">pdb: 3GMV</A>). Therefore we decided to also try the caging of a truncated version of the protein, in case the fusion of Dronpa to the wild type N-terminus allows to much movement freedom of the protein domains.
</P><FONT size="2">Figure 1: Hypothesized caging mechanism of Dronpa proteins. A and B show the caged Cre recombinase (green) with the N- and C-terminal Dronpa fusions (blue and lightblue). Linkers between proteins are shown in red. C and D illustrate the inactive cage. The Cre recombinase is in complex with its target DNA (pdb accession code: 3mgv, 2iov).</FONT></DIV><DIV id="dia2" class="dia" style="text-align:block;"><H2> Introduction</H2><P>For our memory system we needed to create a reporter device that will be switched on by our activated writer protein, the Cre recombinase, and permanently retain this status. Because the Cre recombinase is able to cut out DNA sequences that are flanked by loxp sites, we decided to build a reporter cassette that would express only RFP without activation and switch to luciferase expression after activation of the Cre recombinase. We choose the Nanoluc(R) luciferase (designed by the <A href="http://www.promega.de/products/pm/nanoluc-luciferase-redefining-reporter-assays/">Promega corporation</A>) as final reporter for our system, because luciferase allows an easy readout that is at the same time very accurate. The expression of RFP in the non-activated cells is supposed to ease testing of the reporter device.</P><H2>General Cre reporters</H2><P>The design of our reporter cassette is very similar to other Cre reporters used in research: a single promotor is followed by to protein coding sequences, of which the first one flanked with loxp sites, whereby it can be removed from the DNA <A href="https://2015.igem.org/Team:Tuebingen/References">[Nagy 2000]</A>.</P><P>Our reporter cassette contains a constitutive promotor (the ADH promotor, <A href="http://parts.igem.org/Part:BBa_J63005">BBa_J63005</A>), followed by a RFP gene and the ADH terminator (<A href="http://parts.igem.org/Part:BBa_E1010">BBa_E1010</A> and <A href="http://parts.igem.org/Part:BBa_K392003">BBa_K392003</A>) which are flanked by two loxp sites with the same orientation. Behind the second loxp site is the gene encoding for the luciferase (see Figure 1). This setup leads to expression of luciferase only after the Cre recombinase has cut out the RFP-tADH region.</P><FONT size="2">Figure1: Processing of the Cre reporter cassette.</FONT><P>Our reporter is able to permanently save information, because the Cre induced switch to luciferase expression is irreversible. Without deactivation of the Cre recombinase this process will occur in every reporter within a given cell.
In order to relate the luciferase signal of a population of biosensor memory cells, it is therefore necessary that the reporter is only activated within a fraction of the cells. This can be achieved by activating the Dronpa caged Cre construct only for a small amount of time, when taking the ‘snapshot’. This leads to a statistically determined reporter activation in some of the cells, because the concentration of of the Cre construct stands in relation to the activation state of the sensor.</P><P>On the one side the time frame that is necessary for activation of the reporter, in such a way that an intermediate number of cells is activated, has to be determined empirically for every combination of sensor and promotor, which can be time consuming. On the other hand all this work done to calibrate the CREllumination memory system can also be used in other beneficial ways.
</P><P>If the interplay of sensor, promotor strength and Cre activation time are well characterised, any one of this factors can be controlled quite easily. For example assume a sensor measuring a very weak signal which can not easily be determined: using a longer activation time of CREllumination system, the strength of the luciferase readout can easily be amplified.</P></DIV><DIV id="dia3" class="dia" style="text-align:block;"><H2>Introduction</H2><P>The first step to establish a memory system was to characterise different promoters and check whether they were suitable to be used for expression of our construct.</P>
The iGEM Team Tuebingen 2013 already tried to establish a sensor and we decided to reuse their constructs for our purposes, because all the parts were readily available to us. We used the basic parts for the receptor and promoter (see below)  and rebuilt the construct in yeast using luciferase as a readout.<P>We also tested several other promoters, to look at alternatives. Suitable promoters should be easily inducible, and also produce reliable results. Also, the sensor should activate the promoter in a straight-forward way while not activating many other pathways to ensure a clear readout. 
</P><H2>2013 Sensor</H2><P>In 2013, we wanted to measure progestin concentrations <A href="https://2015.igem.org/Team:Tuebingen/References"> [Kolodziej et al. 2003]</A> that have an effect on fish/amphibians but usually do not harm humans (European Agency for the Evaluation of Medicinal Products, 1999). We decided to use progesterone membrane receptors (mPR) that mediate nongenomic progesterone responses and belong to the progestin and AdipoQ-Receptor (PAQR) family <A href="https://2015.igem.org/Team:Tuebingen/References"> [Tang et al. 2005] </A>. We used one mPR from teleost fishes (like Danio rerio) and one from <I>Xenopus laevis</I> (clawed frog).</P><P><A href="https://2015.igem.org/Team:Tuebingen/References"> Smith et al. (2008) </A> were able to show that human mPRs can sense progesterone and structurally similar compounds at physiologically relevant concentrations by affecting the FET3 promoter. In their study <A href="https://2015.igem.org/Team:Tuebingen/References"> Liu and Patiño (1993) </A> found such a correlation between structural similarity and binding affinity on female <I>Xenopus laevis</I>.
We chose <I>Saccharomyces cerevisiae</I> as chassis for our system due to the fact that yeast possesses endogenous receptors of the PAQR family <A href="https://2015.igem.org/Team:Tuebingen/References"> [Lyons et al. 2004] </A>, <I>S. cerevisiae</I> does not use progesterone in its own metabolism and does not possess any progesterone receptors. Furthermore <I>S. cerevisiae</I> is a well understood eukaryotic organism and offers acceptable reproduction rates. 
For this year’s project, we wanted to re-use both mPRs and the pFET3 part.</P><H2>Promotors</H2><P>We wanted to use five different promoters for our system, which are all easy to induce. To characterise the promoter activity we will measure  luciferase activity or RFP fluorescence by  expression of the respective protein under each promoter.</P><H2>pFET3</H2><P>FET3 is an essential element of <I>S. cerevisiae</I>’s iron metabolism. It is a multicopper oxidase which is responsible for iron (Fe(II)) uptake in yeast <A href="https://2015.igem.org/Team:Tuebingen/References"> [Askwith et al 1994] </A>. It is only expressed when intracellular metal levels, such as zinc and iron, are low, and repressed if metal levels are high <A href="https://2015.igem.org/Team:Tuebingen/References"> [Lyons et al 2004] </A>. High iron levels can thereby repress the pFET3-promoter. pFET3 can also be repressed by activation of the PAQR receptor family after binding of its ligand. Introduction of mPRs and addition of progesterone or progesterone-like molecules should induce the PAQR pathway and therefore repress the pFET3 promotor, resulting in a decrease luciferase signal. </P><H2>pENO2</H2><P>pENO2 is the promoter of the ENO2 gene encoding  yeast enolase (<A href="http://parts.igem.org/Part:BBa_K392001">BBa_K392001</A>). Enolase II plays an important role in glycolysis <A href="https://2015.igem.org/Team:Tuebingen/References"> [Entian KD et al. 1987] </A>. Being a  phosphopyruvate hydratase, it catalyses the conversion of 2-phosphoglycerate to phosphoenolpyruvate, as well as its opposite reaction during gluconeogenesis. The enolase promoter pENO2 in yeast is therefore glucose-inducible and addition of glucose should increase signal readout.</P><H2>pGal1</H2><P>The GAL genes encode for proteins which enable the growth on galactose by converting galactose into glucose. GAL1 codes for the galactokinase which traps galactose inside the cell by phosphorylating C1. The GAL1 promoter is induced in the presence of galactose in the cell and repressed if glucose is present (<A href="http://parts.igem.org/Part:BBa_J63006">BBa_J63006</A>).  </P><H2>pADH</H2><P>The promoter of the ADH gene, coding for the alcohol dehydrogenase <A href="https://2015.igem.org/Team:Tuebingen/References"> [Bennetzen JL and Hall BD 1982] </A>, is a constitutively active promoter, because of which it is often used as a positive control for gene expression (<A href="http://parts.igem.org/Part:BBa_J63005">BBa_J63005</A>).</P><H2>pSUC2</H2><P>The SUC2 gene codes for a sucrose hydrolysing enzyme. The pSUC2 (<A href="http://parts.igem.org/Part:BBa_K950003">BBa_K950003</A>) promoter is heavily repressed by high levels of glucose <A href="https://2015.igem.org/Team:Tuebingen/References">[Lutfiyya and Johnston 1996]</A>. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>