<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tuebingen_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tuebingen/Results</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV class="menu" id="menu" style="left:8.7vw"><DIV class="menuitem" style="width:10vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen';">Home</DIV><DIV class="menuitem" style="width:10vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Team';">Team</DIV><DIV class="menuitem" style="width:10vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Description';">Project</DIV><DIV class="menuitem" style="width:10vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Modeling';">Modeling</DIV><DIV class="menuitem" style="width:18vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Practices';">Human Practices</DIV><DIV class="menuitem" style="width:3.8vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Art_and_Design';">Art</DIV><DIV class="menuitem" style="width:18vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Attributions';">Attributions</DIV></DIV><DIV id="submenuProject" class="submenu" style="left:9.6vw;"><DIV class="menuitem" style="width:14vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Experiments';">Experiments</DIV><DIV class="menuitem" style="width:10vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Results';">Results</DIV><DIV class="menuitem" style="width:10vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Design';">Design</DIV><DIV class="menuitem" style="width:10vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Parts';">Parts</DIV><DIV class="menuitem" style="width:10vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Safety';">Safety</DIV><DIV class="menuitem" style="width:14vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Measurement';">Interlabstudy</DIV><DIV class="menuitem" style="width:10vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Notebook';">Notebook</DIV></DIV><DIV id="submenuAttributions" class="submenu" style="left:34.6vw;"><DIV class="menuitem" style="width:15vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Collaborations';">Collaboration</DIV><DIV class="menuitem" style="width:15vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/References';">References</DIV></DIV><DIV id="submenuPractices" class="submenu" style="left:27.4vw;"><DIV class="menuitem" style="width:12vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/synbio';">SynBio-Day</DIV><DIV class="menuitem" style="width:16vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/schoolclassatlab';">SchoolClass@lab</DIV><DIV class="menuitem" style="width:16vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/freiburg';">Theatre Freiburg</DIV></DIV><DIV id="contentWrapper"><DIV id="arrowLeft" class="arrow" style="left:-7vw;" onclick="diashow(-1);">&lt;</DIV><DIV id="arrowRight" class="arrow" style="right:-7vw;" onclick="diashow(1);">&gt;</DIV><DIV id="contentContainer"><DIV id="dia0" class="dia"><H4>Generation of RFC10 compatible pRS vectors</H4><P>Mutagenesis of the RFC10 restriction sites and the ApaI restriction site in the backbone of the pRS plasmids were 
performed by mutagenesis PCR. Positive clones were screened by restriction analysis with the respective restriction 
enzymes (<A href="https://2015.igem.org/Team:Tuebingen/Design?slide=0">compare plasmid maps</A>).</P><P>Then, the old multiple cloning site (MCS) was replaced by restriction digest (ApaI, SacI) and subsequent ligation of annealed oligonucleotides containing matching overhangs. Verification of successful replacement of the old MCS  was performed by using a restriction site that was only present in the initial pRS MCS (SalI).</P><FONT size="2">Figure 1: Schematic agarose gel showing the migration pattern of various pRS plasmids incubated with different restriction enzymes.</FONT><CENTER><FONT size="2">Figure 2: Restriction analysis of various pRS plasmid constructs.</FONT></CENTER><P>In figure 1, a schematic agarose gel showing the migration pattern of the three pRS vectors when digested with 

different restriction enzymes is depicted. Figure 2 shows the restriction analysis of various pRS plasmid constructs. The used pRS vectors show the expected migration behaviour. Lane 3 in figure 2c shows an additional fragment at around 5 kb which indicates an incomplete digestion of the pRS316 plasmid. In summary, every RFC10 restriction site in the pRS plasmid backbones was removed and the MCS was replaced. Unfortunately, due to time constraints we were not able to insert a terminator.</P></DIV><DIV id="dia1" class="dia"><H4>Analysis of the Dronpa protein</H4><P>First, Dronpa and NLS-Dronpa were cloned under the control of the pGAL1 promoter and yeast cells were transformed with these constructs. Cells were grown in galactose-containing media and fluorescence levels were determined in the plate reader. Different media were used to resuspend the cells to determine whether this has an effect on the fluorescence levels.</P><CENTER><FONT size="2">Figure 1: Relative fluorescence units normalised to OD<SUB>600</SUB>. Data were obtained from 3 independent biological replicates.</FONT></CENTER><P>In figure 1 it can be seen that the Dronpa construct-containing cells show a significantly higher fluorescence signal than the wild type cells. In addition to that, cells expressing NLS-Dronpa show a higher fluorescence intensity than the NLS-Dronpa expressing cells. Possibly,  the Dronpa proteins are localized to the nucleus and cannot be degraded as efficiently as the Dronpa proteins located in the cytoplasm. Furthermore, one can see that the resuspension media does not have an effect.</P><P>In order to characterise Dronpa we tried to express it in <I>E. coli</I> and afterwards purify it. Therefore, we cloned the coding sequence of Dronpa in the pETue, previously known as <A href="https://2014.igem.org/Team:Tuebingen/Project/Plasmids">pETblue</A> vector and transformed the vector in BL21(DE3) <I>E. coli</I>. The pETue vector introduces an N-terminal 6xHis-tag followed by a thrombin cleavage site to the inserted DNA fragment.</P><P>A liquid culture was inoculated with BL21(DE) bacteria containing the pETue-Dronpa construct. The culture was grown to an OD<SUB>600 </SUB> of 0.8 and the cells were then induced by adding 0.5 mM IPTG and shifted to 25°C overnight.</P><P>Induced and uninduced cells were pelleted, lysed and loaded on a 10% acrylamide gel (see figure 2).</P><CENTER><FONT size="2">Figure 2: Test expression of <I>E. coli</I> containing the pETue-Dronpa construct.</FONT></CENTER><P>The induced cells show an additional band at around 25 kDa, which cannot be seen in the non-induced cells.</P><P>We then tried to purify 6xHis-Dronpa in a larger scale experiment. We used the same expression conditions as before. Then, the cells were pelleted and lysis was performed using the sonicator. After that, the lysate was centrifuged to remove the insoluble parts and incubated was incubated with Ni-NTA beads at 4°C. We eluted the proteins from the beads by using buffer supplemented with 200 mM imidazole (compare figure 3). </P><CENTER><FONT size="2">Figure 3: Purification of 6xHis tagged Dronpa.</FONT></CENTER><P>It can be observed  that the His-tagged Dronpa protein is clearly visible in the soluble, flow through and wash fractions. Unfortunately, almost all protein is lost during the washing procedure. Milder washing conditions could probably improve the protein yield. Due to time constraints, the purification could not be repeated. </P><H4>Pulldown of Dronpa with the GFP-trap</H4><P>As the GFP-trap offers an easy purification method for all GFP-tagged proteins, we wondered whether it would also bind to Dronpa, as it has a high structural similarity to GFP. We also tested the RFP-trap to see whether this might bind if the GFP-trap did not. Yeast cells were lysed and the lysate was added to the GFP-trap, as well as an RFP-trap and blocked beads as a negative control using the <A href="http://www.chromotek.com/fileadmin/Redaktion/Products/Nano-Traps/Protocols/GFP-Trap_A__manual_13-12-10.pdf">protocol provided by ChromoTek GmbH</A>.</P><P>Figure 4 shows the resulting SDS-PAGE gel, stained with coomassie, of the positive control, Sec61-GFP. As we could not see any bound GFP after elution of all bound proteins from the beads, the lysis of the yeast cells may not have been efficient enough to retain the structure of the GFP or since the SEC61 is located in the ER it might be that the fusion protein is hyperglycosylated and cannot be recognized by the beads. As we did not have enough time to optimize the lysis protocol, we could not do any further experiments. </P><CENTER><FONT size="2">Figure 4: SDS-PAGE gel after GFP-trap purification.</FONT></CENTER><H4>Fluorescence microscopy</H4><P>In order to assess Dronpa fluorescence we expressed Dronpa and NLS-Dronpa in <I>Saccharomyces cerevisiae</I> in the pTUM104 plasmid (<A href="http://parts.igem.org/Part:BBa_K801004">BBa_K801004</A>). Expression of the normal Dronpa protein and an NLS-Dronpa fusion showed different distribution of the protein throughout the cell (Figure 5). The circular arrangement of the fluorescence signal in the cells expressing NLS-Dronpa indicates that the protein accumulates in the nucleus and that the nuclear location sequence (BBa_K1680004) works as intended.</P><B>pTUM104-Dronpa</B><B>pTUM104-NLS-Dronpa</B><B>Wild type cells</B><FONT size="2">Figure 5: Fluorescence and brightfield pictures of cells expressing Dronpa, NLS-dronpa and wild type cells. The left pictures shows the Dronpa fluorescence channel, the middle picture the brightfield channel and the right picture the overlay of both channels.</FONT><H4>Dronpa photoswitching</H4><P>In addition to simple pictures of Dronpa expressing cells, we were also able to show photoswitching of Dronpa under  the microscope. Figure 6 shows both switching off and switching on of NLS-Dronpa. Off switching was facilitated by illuminating the cells with a 488 nm laser during each frame. Respectively on switching was facilitated by illuminating cells with a 405 nm laser during each frame. The starting and end points of both photoswitching transitions are also shown in Figure 7.</P><FONT size="2">Figure 6: Animated graphic showing consecutive off and on photoswitching of Dronpa. Each frame of the video corresponds to 286 ms. Off switching of dronpa was achieved by illumination with 488 nm in 2.5 min. Follow-up switching on by illumination with 405 nm yielded immediate recovery of ground state fluorescence.</FONT><FONT size="2">Figure 7: Overlay pictures of Dronpa fluorescence and brightfield channel. The scale bar (left picture) corresponds to a size of 3 µm. From left to right: Dronpa before illumination, Dronpa after 1 min 488 nm illumination, Dronpa after 2 min (total) 488 nm illumination, Dronpa after 0.3 sec (follow up) 405 nm illumination.</FONT><P>In addition to the Dronpa photoswitching at the microscope we also illuminated cells expressing Dronpa or NLS-dronpa directly in a 96 well plate. For this setup (Figure 8 and figure 9) we used 50W halogen lamp powered with 16V and 4.8A as source for white light and used a monochromator to single out a specific wavelength.</P><CENTER><FONT size="2">Figure 8: Schematic of the setup we used to illuminate Dronpa expressing cells directly in a 96 well plate.</FONT></CENTER><CENTER><FONT size="2">Figure 9: Photograph of our Dronpa illumination setup for 96 well plates.</FONT></CENTER><P>With the monochromator set to 488 nm one Dronpa and one NLS-Dronpa expressing sample were illuminated for 10 min. Afterwards Dronpa fluorescence of the illuminated samples, non illuminated samples grown in the same batch and wild type cells were measured in a Tecan plate reader (Figure 10). Unfortunately, we could not detect any effect on the Dronpa fluorescence using our own illumination setup. Probably, the intensity of the light source was not high enough to activate photoswitching. </P><FONT size="2">Figure 10: Relative fluorescence measured in wild type cells, Dronpa and NLS-Dronpa expressing cells. One sample of each cell type was measured before and after illumination with 488 nm light from the monochromator setup for 10 min.</FONT></DIV><DIV id="dia2" class="dia"><H4>RFP test measurements</H4><P>To check whether the RFP (<A href="http://parts.igem.org/Part:BBa_E1010">BBa_E1010</A>) we wanted to implement in our Cre reporter cassette works as expected, the RFP was cloned into the pTUM104 vector under the control of the pGAL1 promoter. Yeast cells were transformed with this construct and grown in medium with 2% galactose. The RFP fluorescence intensity was then determined in the plate reader. We used different media to resuspend the cells after centrifugation (as indicated in figure 1).</P><FONT size="2">Figure 1: Relative fluorescence units normalised to OD<SUB>600</SUB>. Data were obtained from 3 independent biological replicates.</FONT><P>In figure 1 one can see that the RFP-construct containing cells show a significantly higher RFP fluorescence than the wild type cells. In addition to that, it shows that the media for resuspension does not strongly influence the  fluorescence.</P><P>In order to check fluorescence of the Cre reporter cassette we expressed the complete construct (<A href="http://parts.igem.org/Part:BBa_K1680025">BBa_K1680025</A>) in <I>Saccharomyces cerevisiae</I> in the pTUM100 plasmid (<A href="http://parts.igem.org/Part:BBa_K801000">BBa_K801000</A>). Microscopy images show that RFP is expressed in these cells (see figure 2).</P><FONT size="2">Figure 2: Fluorescence and brightfield pictures of cells expressing the Cre reporter stop cassette. The left picture shows the bright field channel, the middle picture the RFP channel and the right picture the overlay of both channels.</FONT><H4>Analysis of the Cre reporter cassette</H4><P>In order to assess the function of our Cre reporter regarding recombination, a plasmid containing the stop cassette was digested with EcoRI and PstI to get linearised backbone DNA and the stop cassette DNA. Then, purified Cre recombinase was added and incubated at 37°C. At certain time points samples were taken, heat inactivated and loaded on an agarose gel.</P><P>We expected that the backbone DNA (around 5 kb) stays unaffected (except possible DNAse contaminations in the purified Cre recombinase) in all samples, while the stop cassette (around 3 kb) should show a shift of approximately 1 kb (DNA size between the two loxP sites). In addition to that, a 1kb circularized DNA fragment should be visible. Figure 3 shows the experimental results. </P><FONT size="2">Figure 3: Agarose gel showing the Cre reporter cassette incubated with Cre recombinase.</FONT><P>It can be seen that the backbone DNA fragment stays unaffected during the experiment, while the stop cassette DNA fragment intensity slightly decreases. Furthermore, after 60 and 90 minutes an additional fragment can be seen at around 2 kb, which represents the stop cassette after recombination. Unfortunately, the small circularized fragment is not visible. Therefore, we conclude that the recombination of our stop cassette by Cre recombinase works as expected.</P><P>To check for leaking NanoLuc expression from the Cre reporter cassette, we measured the luciferase activity in overnight cultures of <I>S. cerevisiae</I> carrying the cre reporter construct but no additional plasmid with a Cre recombinase. As shown in figure 3, the luciferase within the cre reporter cassette does not show significant activity compared to wild-type and positive control (pADH-nanoluc).</P><FONT size="2">Figure 4: Cells with Cre reporter cassette do not show significant luciferase activity. RLU=relative luminescence units, positive control = pADH-nanoluc.</FONT><P>In conclusion, we managed to design and transform a working Cre reporter cassette which would be suitable to work with a co-transformed Cre recombinase to form the memory unit of the sensor system.</P></DIV><DIV id="dia3" class="dia"><H4>Promoter activity determination</H4><P><I>S. cerevisiae</I> expressing pGAL1-nanoluc or pSUC2-nanoluc were cultured overnight with different concentrations of galactose or glucose, respectively. Raffinose was added to ensure overall sugar concentrations were 2% for all cultures. Nanoluc activity was measured after addition of luciferin in a plate reader. OD<SUB>600</SUB> was determined for normalization on cell density.</P><FONT size="2">Figure 1: pGAL1 and pSUC2 respond to different sugar concentrations. Shown are two technical replicates from three biological replicates, normalised to OD<SUB>600</SUB> A) Induction of the pGAL promoter by galactose: Luciferase activity is increased between 0.5 and 1.5% of galactose. B) Repression of pSUC promoter by glucose: Luciferase activity is decreased with increasing glucose concentrations.</FONT><P>Figure 1A shows the luciferase activity after incubation of pGAL-nanoluc cells with different concentrations of galactose. As expected, luminescence is increased with the addition of galactose. No difference can be observed between 0.5, 1 and 1.5% galactose, which is due to the long incubation time during which the promoter was activated. Surprisingly, the levels for 2% are on the same level as without galactose. Possibly, the cells are not growing efficiently on galactose, or there is a contamination in the galactose that only affects the cells at higher concentrations, causing reduced growth.</P><P>The results for pSUC2-nanoluc cells is depicted in fig. 1B. As expected, the signal decreases with increasing glucose concentrations. This indicates lower expression of the nanoluc after glucose addition. Glucose is known to repress the pSUC2 promoter, so it can be concluded that the promoter construct is reliable.</P><P>The values for the pSUC2 promoter were also needed for modelling. As the plate reader measures relative units instead of total units, the values between the measurements vary strongly. To correct for this, the values were z-normalised and then fitted (see figure 2). This was then used for the <A href="”https://2015.igem.org/Team:Tuebingen/Modeling”">modelling</A>.</P><FONT size="2"><CENTER>Figure 2: Fitting of the pSUC2 repression, measured by luciferase activity.</CENTER></FONT><H4>pGAL induction over time</H4><P><I>S. cerevisiae</I> cells expressing the NanoLuc luciferase under the pGAL1 promoter were grown overnight in SC-URA medium supplemented with 2% raffinose. The cells were harvested by centrifugation and resuspended in SC-URA with 2% galactose or 2% glucose. Samples were taken immediately after resuspension and after every 30 minutes for the first 2.5 hours and then every hour up to 7.5 hours. Luminescence and optical density (OD<SUB>600</SUB>) were measured at each time point.</P><P>Figure 3 shows the readout of the luminescence normalised to the measure OD<SUB>600</SUB> to ensure comparability.</P><FONT size="2">Figure 3: Galactose induces pGAL-dependent expression. Cells expressing pGAL-nanoluc were cultured with the indicated sugar and luminescence  was measured at the given timepoints. Luminescence is normalised to OD.</FONT><P>Cells cultured in raffinose medium should show a low, but not repressed, expression of the pGAL1 promoter. As expected, after resuspension in glucose medium, the cells show a repression, which normalises to a stable level over time, as indicated by a decrease in luciferase activity. Cell cultured in galactose medium show an increase in luciferase activity, which is due to induction of the pGAL1 promoter and increased expression of the nanoluc.</P><P>The decrease in luciferase activity after 30 minutes is probably due to the harvesting and resuspension. After 2.5 hours, induction of the GAL1 promoter is visible and therefore reflects in the protein level of the nanoluc. Considerably higher expression levels are only achieved after 7.5 hours incubation time. It can be concluded that the GAL1 promoter is integrated correctly and can be induced by galactose.</P><H4>Microscopy of RFP expressed from different promotors</H4><P>Additional to the luciferase assays, we also expressed RFP under control of the various promoters and imaged them under the laser scanning microscope.</P><P>Figure 4 shows the results for the pENO2-RFP construct. Surprisingly, the RFP is localized only at the membrane, even though RFP without membrane target sequence was used. We therefore sequenced the whole construct and saw that the sequence of the promoter did not correspond to the sequence given in the registry.</P><FONT size="2">Figure 4: Fluorescence pictures of cells expressing pENO2-RFP. The RFP is localising to the plasma membrane on the cells.</FONT><P>pFET3 is usually activated by low-iron medium. We cultured the cells in SC medium with 2% glucose before imaging the cells. It could be observed that cells either expressed RFP strongly and evenly spread throughout the cell, or not at all (see fig. 5).</P><FONT size="2">Figure 5: Fluorescence pictures of cells expressing pFET3-RFP. RFP expressing cells show an even distribution of the protein.</FONT><P>Figure 6A to 6C show cells expressing pSUC-RFP cultured in galactose, glucose or raffinose, respectively. Cells cultured in galactose or raffinose medium show evenly spreaded RFP fluorescence, while the repression of the pSUC promoter with glucose results in diminished fluorescence.</P><FONT size="2">Figure 6A:RFP fluorescence under pSUC promotor in galactose culture.</FONT><FONT size="2">Figure 6B:RFP fluorescence under pSUC promotor in glucose culture.</FONT><FONT size="2">Figure 6C:RFP fluorescence under pSUC promotor in raffinose culture.</FONT><H4>Progesteron receptor characterisation</H4><P>We cloned both mPRs (from Xenopus laevis and Danio rerio) in the pYX122 vector under the control of a constitutive promoter and the pFET3-nLuc in the pTUM100. Then, both plasmids were co-transfected in yeast. Liquid cultures were inoculated with the transformed yeast and different hormone concentration were added. The hormones were extracted from different contraceptive pill containing either Ethinyl estradiol (EG) or Ulipristal acetate (U). Hormones were added at nanomolar level and luciferase assays were performed after 6 and 9 hours. In addition yeast cells containing only the pFET3-nLuc construct were treated with the hormones, aswell (see figure 7 and 8).</P><FONT size="2">Figure 7: Relative luminescence levels normalized to OD600 after 6 hours.</FONT><FONT size="2">Figure 8: Relative luminescence levels normalized to OD600 after 9 hours.</FONT><P>In general one can see that the cells containing the pFET-nLuc construct show a higher luminescence signal than the cells that contain in addition one of the two mPR constructs. The negative control (no hormones added) show a comparable decrease in the luminescence levels than the cells that were treated with the different hormones. We expected to see a drop in the luminescence levels for the hormone treated cells, while the untreated cells show a stable luminescence signal. A possible explanation for this observation is that the used hormones do not bind to the receptors, since these hormones are no Progesterones or the presence of the mPR receptors impair growth of the cells.</P></DIV><DIV id="dia4" class="dia"><H4>Background and Goal</H4><P>The team we sent into the race in 2014 sadly was not able to characterize all of their parts. We did not want to let this stand, seeing how we put so much effort into the project last year. So we set out to characterise one of the parts from 2014. As a part to characterize we chose the Ssp GyrB split intein <A href="”http://parts.igem.org/Part:BBa_K1483003”">(K1483003)</A>.</P><H4>A look back at the theory</H4><P>Inteins are self-splicing proteins. The part that is spliced out is referred to as intein, the parts which are joined are called exteins in analogy to introns and exons. In split inteins the intein is divided into two parts to be seperately expressed or synthesized. These parts are only active when combined.</P><P>In the case of the Ssp GyrB split intein, there is a larger protein (150 amino acids) and a small peptide (&gt;6 amino acids). Both parts can for example be expressed in fusion proteins, however, the small peptide can easily be synthesized in vitro and coupled to matrices or fluorophores. Upon reaction, the intein (both parts of the split intein) is spliced out, fusing their respective N- and C-terminal parts. If the reaction conditions are not met, a side reaction can take place. This reaction is termed N-cleavage and is illustrated in the graphic below.</P><CENTER><FONT size="2">Figure 1: Schemtic of the reactions the intein can undergo.</FONT></CENTER><P>The larger Intein part we expressed coupled to another part which was used in last year’s project: NAGA, α-N-acetylgalactosaminidase <A href="http://parts.igem.org/Part:BBa_K1483000">(BBa_K1483000)</A>. The expression vector we used was the pETue <A href="http://parts.igem.org/Part:BBa_K1680026">(BBa_K1680026)</A>, which we created last year based on the pETBlue-1 vector. This had the added benefit that any protein would be expressed with a 6xHis-Tag for purification.</P><H4>Results</H4><CENTER><FONT size="2">Figure 2: Schemtic of the experiment procedure.</FONT></CENTER><P>To make sure the Intein-part we sent in last year actually worked, we devised a simple assay, which could be read out in an SDS-PAGE. We acquired the <A href="https://2014.igem.org/Team:Tuebingen/Project/SyntheticPeptides">synthetic peptide</A> from last years team. This peptide was labeled with fluorescein in it’s extein part. </P><CENTER><FONT size="2">Figure 3: Molecular structure of the synthetic intein.</FONT></CENTER><H4>Gel I: Expression/Purification</H4><CENTER><FONT size="2">Figure 4: Coomassie blue stained SDS-PAGE of NAGA-Intein testexpression.</FONT></CENTER><CENTER><FONT size="2">Figure 5: Coomassie blue stained SDS-PAGE of different purification steps of the NAGA-intein.</FONT></CENTER><P>As we can see, before purification we could not determine whether or not the expression had worked as intended. However, after purification using the 6xHis-tag, the SDS-polyacrylamide gel clearly showed a band at 70 kDa, which is the exact height at which we expected our 69.4 kDa protein. Since 6xHis-tag purification unfortunately also enriches untagged proteins, we needed to make sure that the promising band was the NAGA-Intein construct. To this end, we cut the band from the gel, digested it with trypsin and analysed it using a MALDI-TOF mass spectrometer.</P><H4>MALDI 1: Expression/Purification Test</H4><P>This experiment was conducted according to the protocol outlined <A href="https://2015.igem.org/Team:Tuebingen/Experiments"> here </A>. We used the two bands around 70 kDa in the Eluate fraction. The lower one is not present in the Flow Through and Wash fractions. Additionally, it’s weight is closer to the expected protein, so this was the more likely candidate.</P><P>The upper band was identified (score = 384) to be ArnA a bifunctional polymyxin resistance protein. This protein weighs 74.289 kDa. When trying to match the spectrum to the expected NAGA-Intein, we achieved a sequence coverage of 8.3% and an intensity coverage of 8.1%.</P><P>For the lower band the best hit (score = 459) was glmS, a Glutamine-fructose-6-phosphate aminotransferase. This protein weighs 66.908 kDa. Trying to match the spectrum to the expected NAGA-Intein yielded a 23.3% sequence coverage, but only 10.1% intensity coverage (see figure).</P><CENTER><FONT size="2">Figure 6: Alignemnt of the NAGA-Intein sequence with the MALDI results.</FONT></CENTER><P>While the bands seemed very distinct and promising, the results from the MALDI experiment suggested that neither was in fact not the desired protein. However, since the lower band was a better fit than the top one, we do speculate, that the lower band does in fact also represent the desired NAGA-Intein, but we cut the band too broadly and the glmS, being only 2.5 kDa lighter, shows very similar migration behavior. This would have the effect, that NAGA-Intein’s peptides would be “overshadowed” by other peptides, like the ones generated from the glmS, a protein commonly purified during IMAC <A href="https://2015.igem.org/Team:Tuebingen/References">[Bolanos-Garcia 2006]</A> Further purification, for example with strong cation exchange chromatography, followed by MALDI analysis, might have confirmed or disconfirmed our suspicion. We were not able to conduct this follow-up experiment.</P><H4>Gel2: Peptide coupling/functional assay</H4><CENTER><FONT size="2">Figure 7: A) Coomassie blue stained SDS-PAGE of the reaction between the labeled paptide and the NAGA-intein. b) Fluorescence picture of the same SDS-PAGE.</FONT></CENTER><P>After incubation with the labeled peptide, we expected to see the NAGA-Intein construct to be 17.3 kDa lighter and thus cause a different band. This band should be labeled with the fluorescent dye from the peptide. Furthermore we expected the excised intein (17.3 kDa) to show up on the gel, unlabeled since the splicing mechanism should transfer the fluorescent dye unto the protein.</P><P>After 24 hours, we did instead encounter a brightly fluorescing band weighing roughly 20 kDa and one with less fluorescence weighing roughly 27 kDa. The SDS-PAGE showed a band around 60 kDa, which was absent in the control. We suspected this to be the product NAGA (52.2 kDa) without the excised intein (data not shown). </P><P>After 48 hours, we encountered the same pattern of bands, but the band weighing 60 kDa was much more pronounced, even though there was still no fluorescence visible from this band.</P><P>This pattern can be explained with the side reaction, which Volkmann et al. termed N-cleage. Upon formation of the intein-peptide complex, the part of the splicing reaction N-terminal of the intein complex takes place, while the reaction on its C-terminus does not. This way the generated fragments are different. While we expect: NAGA-Extein-Fluorescein (53 kDa), the excised Intein (17.3 kDa) and the 6 amino acid peptide to be generated, we get NAGA (52.2 kDa), Intein (17.3 kDa) and Peptide-Fluorescein (1.5 kDa). Since the part of the fluorescent peptide, which gets transferred to the NAGA is very small (0.8 kDa) this side reaction would barely show on an SDS-gel, aside from the lack of fluorescence.</P><P>We were unable to explain the actually fluorescing bands. However, since they were quite bright, a band only exhibiting little fluorescence might be not picked up. So to find if our intein was active but the resulting fluorescence overshadowed, we attempted another analysis using a MALDI-TOF mass spectrometer. At the same time, we tried to find out what caused the fluorescing bands.</P><H4>Maldi II: Peptide coupling/functional assay</H4><P>We cut out the bands indicated in the graphic below and treated them as explained <A href="”https://2015.igem.org/Team:Tuebingen/Experiments”"> here </A>.

Trying to fit the peaks to the expected protein (NAGA) yielded a sequence coverage of 18.2% and an intensity coverage of 2.9%. This result cannot be considered significant.</P><P>The two other gel pieces we analyzed turned out with high probability to be ribosomal proteins.

However, an additional SDS-PAGE revealed that loading the peptide without the purified protein resulted in the same pattern of fluorescing bands (data not shown). While neither the MALDI experiment, nor the fluorescence assay were able to confirm the activity of our intein, the SDS-PAGE does suggest that the intein is active at least insofar that N-Cleavage can occur. </P><P>For further experiments the used concentration of peptide for the coupling assay would need to be adjusted to allow the lower fluorescence of the product to be picked up by the instrument. Further, the buffer conditions may be optimized to match the conditions described by <A href="https://2015.igem.org/Team:Tuebingen/References">[Appleby 2009]</A>. Further MS experiments would have to be conducted on more thoroughly purified samples, with more tightly cut bands and on a more powerful instrument.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>