<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tuebingen_Parts skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tuebingen/Parts</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV class="menu" id="menu" style="left:8.7vw"><DIV class="menuitem" style="width:10vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen';">Home</DIV><DIV class="menuitem" style="width:10vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Team';">Team</DIV><DIV class="menuitem" style="width:10vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Description';">Project</DIV><DIV class="menuitem" style="width:10vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Modeling';">Modeling</DIV><DIV class="menuitem" style="width:18vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Practices';">Human Practices</DIV><DIV class="menuitem" style="width:3.8vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Art_and_Design';">Art</DIV><DIV class="menuitem" style="width:18vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Attributions';">Attributions</DIV></DIV><DIV id="submenuProject" class="submenu" style="left:9.6vw;"><DIV class="menuitem" style="width:14vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Experiments';">Experiments</DIV><DIV class="menuitem" style="width:10vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Results';">Results</DIV><DIV class="menuitem" style="width:10vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Design';">Design</DIV><DIV class="menuitem" style="width:10vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Parts';">Parts</DIV><DIV class="menuitem" style="width:10vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Safety';">Safety</DIV><DIV class="menuitem" style="width:14vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Measurement';">Interlabstudy</DIV><DIV class="menuitem" style="width:10vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Notebook';">Notebook</DIV></DIV><DIV id="submenuAttributions" class="submenu" style="left:34.6vw;"><DIV class="menuitem" style="width:15vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/Collaborations';">Collaboration</DIV><DIV class="menuitem" style="width:15vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/References';">References</DIV></DIV><DIV id="submenuPractices" class="submenu" style="left:27.4vw;"><DIV class="menuitem" style="width:12vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/synbio';">SynBio-Day</DIV><DIV class="menuitem" style="width:16vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/schoolclassatlab';">SchoolClass@lab</DIV><DIV class="menuitem" style="width:16vw;" onclick="window.location.href='https://2015.igem.org/Team:Tuebingen/freiburg';">Theatre Freiburg</DIV></DIV><DIV id="contentWrapper"><DIV id="arrowLeft" class="arrow" style="left:-7vw;" onclick="diashow(-1);">&lt;</DIV><DIV id="arrowRight" class="arrow" style="right:-7vw;" onclick="diashow(1);">&gt;</DIV><DIV id="contentContainer"><DIV id="dia0" class="dia"><P>For the generation of our memory system we needed to create different parts. Because the Dronpa-caged Cre constructs are fusion proteins, we used the RFC25 standard for the assembly of these parts from the simple parts we generated from gBlocks synthesized by IDT.</P><P>All simple parts were cloned into the shipping vector pSB1C3 for further use and submission to the registry. </P><H2>Linkers for protein fusion</H2><P>These parts contain the protein coding region for the generation of 6/9/12/15 amino acid linkers. The linker will be composed of 4 amino acids from the scars from using RFC25 standard and serine/glycine residues. The parts are supplied in RFC25 standard.
Characterisation of these parts was not possible, because they do not fulfill a direct function.
</P><LABEL class="collapse" for="_1"><H4><A href="http://parts.igem.org/Part:BBa_K1680000"> BBa_K1680000:</A> Protein linker, 6 amino acids</H4></LABEL><DIV>
Part only sequence: TCTTCT
        </DIV><LABEL class="collapse" for="_2"><H4><A href="http://parts.igem.org/Part:BBa_K1680001"> BBa_K1680001: </A> Protein linker, 9 amino acids</H4></LABEL><DIV>
Part only sequence: TCTGGTTCTGGTTCT
        </DIV><LABEL class="collapse" for="_3"><H4><A href="http://parts.igem.org/Part:BBa_K1680002"> BBa_K1680002: </A> Protein linker, 12 amino acids</H4></LABEL><DIV>
Part only sequence: TCTGGTTCTGGTGGTTCTGGTTCT 
        </DIV><LABEL class="collapse" for="_4"><H4><A href="http://parts.igem.org/Part:BBa_K1680003"> BBa_K1680003: </A> Protein linker, 15 amino acids</H4></LABEL><DIV>
Part only sequence: TCTGGTTCTGGTTCTGGTTCTGGTTCTGGTTCT
        </DIV><H2>SV40 nuclear location sequence</H2><P>
This part contains the protein coding sequence for the nuclear localisation signal of SV40 large T antigen (amino acids: PKKKRKV). The part is supplied in RFC25 standard.
This part was characterised as a fusion protein with Dronpa (BBa_K1680006). Fluorescence microscopy showed nuclear localisation of the fusion construct (<A href="https://2015.igem.org/Team:Tuebingen/Results?slide=1">see Results</A>).
</P><LABEL class="collapse" for="_5"><H4><A href="http://parts.igem.org/Part:BBa_K1680004"> BBa_K1680004: </A> SV40 NLS</H4></LABEL><DIV>
Part only sequence: CCACCAAAGAAAAAGAGAAAAGTT
        </DIV><H2>loxP site</H2><P>
This part contains the WT loxP site which is recognised by Cre recombinase (<A href="http://parts.igem.org/Part:BBa_K1680007">BBa_K1680007</A>), in forward orientation. The part is supplied in RFC10.
The Cre reporter part we assembled (<A href="http://parts.igem.org/Part:BBa_K1680025">BBa_K1680025</A>) used this part and in vitro experiments showed an effective turnover by a purified Cre recombinase (<A href="https://2015.igem.org/Team:Tuebingen/Results?slide=2">see Results</A>).
</P><LABEL class="collapse" for="_6"><H4><A href="http://parts.igem.org/Part:BBa_K1680005">BBa_K1680005</A>: loxP site</H4></LABEL><DIV>
Part only sequence: ATAACTTCGTATAGCATACATTATACGAAGTTAT 
        </DIV><H2>Fluorescent protein Dronpa</H2><P>
This part contains the protein coding sequence for a mutant (K145N) of the fluorescent protein Dronpa and is codon optimised for yeast. Dronpa is a photoswitchable protein. Fluorescence can be turned off by illumination with 488 nm light and switched back on by illumination with 405 nm. The K145N mutation leads to photoswitchable monomerisation/multimerisation behaviour (as described by Zhou et al. 2012). The part is supplied in RFC25.
Fluorescence microscopy showed that the expression of this part works and photoswitching can be observed as expected (<A href="https://2015.igem.org/Team:Tuebingen/Results?slide=1">see Results</A>). 
</P><LABEL class="collapse" for="_7"><H4><A href="http://parts.igem.org/Part:BBa_K1680006">BBa_K1680006</A>: Fluorescent protein Dronpa</H4></LABEL><DIV style="word-wrap:break-word;width:100%;">
TCTGTTATCAAGCCAGACATGAAGATCAAGTTGAGAATGGAAGGTGCTGTTAACGGTCACCCATTCGCTATCGAAGGTGTTGGTTTGGGTAAGCCATTCGAAGGTAAGCAATCTATGGACTTGAAGGTTAAGGAAGGTGGTCCATTGCCATTCGCTTACGACATCTTGACTACTGTTTTCTGTTACGGTAACAGAGTTTTCGCTAAGTACCCAGAAAACATCGTTGACTACTTCAAGCAATCTTTCCCAGAAGGTTACTCTTGGGAAAGATCTATGAACTACGAAGACGGTGGTATCTGTAACGCTACTAACGACATCACTTTGGACGGTGACTGTTACATCTACGAAATCAGATTCGACGGTGTTAACTTCCCAGCTAACGGTCCAGTTATGCAAAAGAGAACTGTTAAGTGGGAACCATCTACTGAAAATTTGTACGTTAGAGACGGTGTTTTGAAGGGTGACGTTAACATGGCTTTGTCTTTGGAAGGTGGTGGTCACTACAGATGTGACTTCAAGACTACTTACAAGGCTAAGAAGGTTGTTCAATTGCCAGACTACCACTTCGTTGACCACCACATCGAAATCAAGTCTCACGACAAGGACTACTCTAACGTTAACTTGCACGAACACGCTGAAGCTCACTCTGAATTGCCAAGACAAGCTAAG
        </DIV><H2>Cre recombinase</H2><P>
This part contains the protein coding sequence  for the DNA recombinase Cre and is codon optimized for yeast. Cre recombinases recognize two loxp sites (<A href="http://parts.igem.org/Part:BBa_K1680005">BBa_K1680005</A>) and depending on their orientation cuts out or reverses the DNA sequence between them. The part is supplied in RFC25.</P><P>
Due to time constraints and failed attempts at cloning our Cre reporter into a vector for cotransfection with this plasmids, we were not able to characterise this part.
</P><LABEL class="collapse" for="_8"><H4><A href="http://parts.igem.org/Part:BBa_K1680007">BBa_K1680007</A>: Cre recombinase</H4></LABEL><DIV style="word-wrap:break-word;width:100%;">
ATGTCTAATCTGCTTACTGTTCATCAAAATCTTCCTGCATTACCTGTTGATGCTACCTCAGACGAAGTTAGGAAAAACTTAATGGATATGTTTAGAGATAGACAAGCTTTCTCTGAACATACCTGGAAAATGTTATTAAGCGTTTGTCGTTCTTGGGCTGCATGGTGTAAATTAAATAATAGGAAGTGGTTCCCAGCAGAGCCTGAAGATGTGAGAGATTATTTGTTGTACTTACAGGCCAGAGGTTTAGCAGTAAAAACAATTCAACAACATTTAGGTCAATTGAATATGCTACACAGAAGATCCGGTTTGCCCAGACCTAGCGATTCTAATGCAGTGTCTTTAGTTATGAGGAGAATAAGAAAAGAAAATGTTGACGCCGGTGAAAGAGCCAAACAGGCATTAGCTTTTGAAAGGACAGATTTCGATCAAGTAAGGTCTCTTATGGAAAATTCAGATAGGTGCCAGGACATCAGAAATTTGGCATTTTTAGGCATAGCATATAATACGTTGCTGAGAATCGCAGAAATCGCAAGAATCAGAGTTAAAGACATTTCCAGAACAGATGGCGGCAGAATGTTGATACATATTGGCAGAACCAAAACTTTAGTTTCCACAGCAGGTGTCGAGAAAGCCTTGTCTTTAGGTGTTACGAAGCTTGTGGAAAGATGGATTTCTGTGTCTGGAGTGGCGGATGACCCAAATAACTATTTATTTTGCAGGGTGAGGAAAAATGGGGTAGCCGCTCCATCTGCTACTTCACAATTGTCAACGAGAGCCCTAGAGGGCATCTTCGAGGCTACACACAGGTTGATATATGGCGCCAAAGATGACTCTGGTCAAAGATACTTGGCCTGGTCCGGGCATTCTGCTAGAGTGGGGGCAGCAAGGGATATGGCAAGGGCGGGTGTCTCTATCCCAGAAATTATGCAAGCTGGTGGTTGGACCAATGTCAATATCGTTATGAACTATATAAGAAATCTTGACAGCGAGACTGGAGCAATGGTTAGACTTTTAGAAGACGGTGAC 
        </DIV><H2>Δ1-19 Cre recombinase</H2><P>This part contains the protein coding region for a mutant (Deletion of AA1-19) of DNA recombinase Cre and is codon optimised for yeast. Cre recombinases recognize two loxP sites (<A href="http://parts.igem.org/Part:BBa_K1680005">BBa_K1680005</A>) and depending on their orientation cuts out or reverses the DNA sequence between them. The first 19 amino acids were removed because these amino acids are most likely very flexible, because they could not be crystallized when the structure was determined (RCSB: <A href="http://www.rcsb.org/pdb/explore.do?structureId=3MGV">3MGV</A>). The part is supplied in RFC25.
</P><P>
Due to time constraints and failed attempts at cloning our Cre reporter into a vector for cotransfection with this part, we were not able to characterise this part.
</P><LABEL class="collapse" for="_9"><H4><A href="http://parts.igem.org/Part:BBa_K1680008">BBa_K1680008</A>: Δ1-19 Cre recombinase</H4></LABEL><DIV style="word-wrap:break-word;width:100%;">
Part only sequence: TCTGATGAAGTTAGAAAGAACCTGATGGACATGTTTAGAGATAGACAGGCTTTCAGTGAACATACCTGGAAGATGCTATTGTCCGTTTGCAGATCCTGGGCGGCGTGGTGTAAGCTTAATAATAGAAAATGGTTTCCTGCCGAACCTGAAGATGTTAGGGACTATCTACTGTATTTGCAAGCCAGAGGATTAGCCGTAAAAACAATTCAGCAGCACTTGGGTCAACTAAATATGTTACATAGACGTTCTGGTTTGCCGAGACCATCTGATTCCAATGCAGTCTCCTTAGTTATGAGAAGAATTAGAAAAGAAAATGTAGATGCCGGTGAAAGAGCAAAGCAGGCCTTAGCGTTCGAAAGAACTGACTTCGATCAAGTTAGGTCTTTAATGGAAAACAGTGACAGGTGTCAAGACATTAGAAATTTGGCTTTCTTGGGAATTGCCTATAACACGTTGTTAAGAATTGCAGAGATCGCCAGAATAAGGGTAAAGGATATATCTCGTACAGACGGTGGTAGAATGTTGATACACATCGGAAGAACTAAAACTTTGGTTTCAACAGCTGGGGTTGAGAAGGCCTTGAGTTTGGGTGTAACGAAGCTAGTCGAAAGATGGATTAGCGTTTCTGGTGTTGCCGACGACCCTAACAATTACCTATTTTGTAGAGTCAGGAAGAATGGCGTCGCTGCACCGTCAGCAACATCCCAGCTTTCAACTAGAGCGTTAGAGGGAATTTTTGAAGCAACACATAGACTGATTTATGGAGCCAAGGACGATAGTGGACAGCGTTATCTTGCTTGGTCTGGTCATAGTGCGAGAGTGGGAGCTGCAAGAGATATGGCGAGGGCTGGCGTCTCAATTCCCGAAATTATGCAAGCTGGGGGTTGGACAAACGTCAACATAGTGATGAACTACATCAGGAACTTGGACTCTGAGACAGGTGCGATGGTAAGGTTATTGGAGGATGGTGAC
        </DIV><H2>NanoLuc</H2><P>This part contains the protein coding region for the NanoLuc(tm) luciferase, which is designed by  Promega Corporation and free for research use, and is codon optimised for yeast. The Nanoluc is smaller than other luciferase proteins and yields equal or stronger luminescence readings. The part is supplied in RFC10.
</P><P>
We used this part successfully under the control of many promoters to determine promoter strength (<A href="https://2015.igem.org/Team:Tuebingen/Results?slide=3">see Results</A>).
</P><LABEL class="collapse" for="_10"><H4><A href="http://parts.igem.org/Part:BBa_K1680009">BBa_K1680009:</A> NanoLuc</H4></LABEL><DIV style="word-wrap:break-word;width:100%;">
ATGGTCTTCACCTTAGAAGACTTTGTTGGTGACTGGAGGCAAACTGCCGGTTATAATCTTGACCAAGTTTTGGAACAAGGTGGTGTATCATCACTTTTCCAGAATCTAGGAGTTAGCGTCACACCCATTCAAAGAATCGTCCTATCTGGCGAAAATGGTTTAAAAATTGATATTCATGTTATTATTCCCTATGAAGGTTTATCAGGAGATCAAATGGGTCAGATCGAGAAGATTTTTAAGGTGGTATATCCTGTCGATGACCACCACTTCAAAGTCATTTTGCATTATGGTACTTTGGTTATTGACGGCGTAACCCCTAACATGATTGATTACTTCGGTAGACCTTATGAGGGGATAGCAGTTTTTGATGGTAAAAAAATTACAGTCACAGGGACTTTATGGAACGGCAATAAAATCATCGACGAACGTCTAATCAATCCAGATGGATCATTGTTATTCAGGGTTACCATTAATGGAGTTACTGGTTGGAGATTGTGTGAGCGTATTTTGGCTTAA
        </DIV><H2>Firefly Luciferase</H2><P>
This part contains the protein coding region for the firefly (Photinus pyralis) luciferase and is codon optimised for yeast. The part is supplied in RFC10.
</P><P>
We tested this part with the Promega ONE-Glo Luciferase Assay System using both live cells and cell lysates without positive results.
</P><LABEL class="collapse" for="_11"><H4><A href="http://parts.igem.org/Part:BBa_K1680010">BBa_K1680010:</A> Firefly Luciferase</H4></LABEL><DIV style="word-wrap:break-word;width:100%;">
ATGGAGGACGCCAAGAATATCAAAAAGGGCCCCGCTCCATTCTATCCCTTGGAGGATGGAACCGCCGGGGAGCAGCTACATAAAGCTATGAAAAGATATGCTCTAGTACCTGGCACCATCGCCTTTACGGATGCCCATATCGAAGTTGACATAACATATGCAGAGTACTTTGAGATGTCTGTTAGATTGGCTGAGGCTATGAAGAGATATGGATTAAACACAAATCATCGTATCGTAGTTTGTTCTGAAAACTCCCTGCAATTTTTTATGCCAGTACTAGGTGCACTTTTCATTGGTGTTGCAGTAGCACCAGCCAATGACATATATAATGAAAGGGAATTGTTGAACAGTATGGGAATCTCACAACCTACCGTTGTCTTCGTTAGCAAAAAGGGCTTACAGAAGATATTGAACGTCCAAAAAAAATTGCCAATAATCCAAAAAATTATTATCATGGATTCAAAAACGGACTACCAAGGCTTTCAAAGCATGTACACGTTCGTTACGAGTCATCTTCCTCCAGGATTCAATGAATATGATTTTGTTCCTGAGTCTTTTGATCGTGATAAAACTATAGCTTTGATTATGAATAGCAGCGGTAGCACGGGCCTACCAAAAGGAGTAGCTCTGCCTCATAGGACAGCCTGTGTTAGATTTTCCCATGCAAGAGATCCCATTTTTGGTAATCAGATTATTCCTGACACGGCAATTCTATCAGTTGTGCCATTTCATCACGGTTTTGGAATGTTTACAACTCTAGGCTATCTAATATGTGGGTTTAGGGTAGTTTTGATGTATAGATTTGAAGAGGAATTGTTTTTAAGATCATTACAAGACTACAAGATTCAATCCGCTCTTCTTGTGCCTACCTTGTTCTCATTCTTTGCAAAATCTACTTTAATCGATAAATACGATTTGAGCAATTTACATGAAATTGCAAGTGGTGGCGCCCCGTTATCTAAGGAAGTGGGTGAGGCGGTTGCTAAAAGGTTTCATTTGCCTGGCATCAGGCAGGGTTATGGTCTAACTGAAACTACTTCAGCTATTCTGATCACTCCGGAGGGTGATGATAAACCTGGTGCAGTGGGAAAGGTTGTGCCATTTTTTGAAGCCAAAGTTGTAGATCTGGACACTGGTAAAACCTTAGGGGTGAATCAAAGAGGTGAATTGTGTGTGAGAGGACCCATGATCATGAGCGGTTACGTAAACAACCCAGAAGCTACTAATGCCTTAATCGATAAGGATGGCTGGCTGCATAGCGGGGATATCGCATATTGGGATGAAGATGAGCACTTCTTTATTGTAGATAGATTGAAATCATTGATTAAATATAAAGGTTACCAAGTGGCCCCTGCTGAATTGGAGTCAATACTGCTTCAGCATCCAAATATCTTCGATGCGGGTGTGGCTGGATTGCCAGATGATGATGCAGGTGAACTTCCCGCAGCAGTAGTTGTTTTAGAACATGGCAAAACGATGACTGAAAAAGAGATTGTTGATTACGTAGCATCTCAAGTTACTACAGCCAAAAAGCTTAGAGGTGGAGTGGTATTCGTTGATGAGGTACCAAAAGGTCTAACAGGTAAGTTAGACGCAAGAAAGATACGTGAAATATTAATTAAAGCCAAGAAAGGTGGTAAAATCGCTGTCTGA
        </DIV><H2>HCV protease with linkers</H2><P>This part contains the protein coding region for the Hepatitis C virus (HCV) NS3-4A protease with N- &amp; C-terminal linkers and is codon optimised for yeast. It is intended for combination with 145N-Dronpa (<A href="http://parts.igem.org/Part:BBa_K1680006">BBa_K1680006</A>) to a light- dependent caged construct as described by <A href="https://2015.igem.org/Team:Tuebingen/References">[Zhou 2012]</A>. The part is supplied in RFC25.</P><P>This part was intended as a backup for our project, due to time restraints we did not use it.</P><LABEL class="collapse" for="_27"><H4><A href="http://parts.igem.org/Part:BBa_K1680011">BBa_K1680011:</A> HCV protease with linkers</H4></LABEL><DIV style="word-wrap:break-word;width:100%;">Part only sequence: AAGGGTTCTATGGGTTCTTCTGGTACTGGTTGTGTTGTTATCGTTGGTAGAATCGTTTTGTCTGGTTCTGGTACTTCTGCTCCAATCACTGCTTACGCTCAACAAACTAGAGGTTTGTTGGGTTGTATCATCACTTCTTTGACTGGTAGAGACAAGAACCAAGTTGAAGGTGAAGTTCAAATCGTTTCTACTGCTACTCAAACTTTCTTGGCTACTTGTATCAACGGTGTTTGTTGGGCTGTTTACCACGGTGCTGGTACTAGAACTATCGCTTCTCCAAAGGGTCCAGTTATCCAAATGTACACTAACGTTGACCAAGACTTGGTTGGTTGGCCAGCTCCACAAGGTTCTCGTTCTTTGACTCCATGTACTTGTGGTTCTTCTGACTTGTACTTGGTTACTAGACACGCTGACGTTATCCCAGTTAGAAGAAGAGGTGACTCTCGTGGTTCTTTGTTGTCTCCAAGACCAATCTCTTACTTGAAGGGTTCTTCTGGTGGTCCATTGTTGTGTCCAGCTGGTCACGCTGTTGGTTTGTTCAGAGCTGCTGTTTGTACTAGAGGTGTTGCTAAGGCTGTTGACTTCATCCCAGTTGAAAACTTGGAAACTACTATGAGAGCTTCTGGTTCTTCTGGTTCTTCTGAATTT
 </DIV><H2>Cleavage site for HCV protease</H2><P>This part contains the protein coding region for the cleavage site of the HCV NS3-4A protease (<A href="http://parts.igem.org/Part:BBa_K1680011">BBa_K1680011</A>). The part is supplied in RFC25.
This part was intended as a backup for our project, due to time restraints we did not use it.</P><LABEL class="collapse" for="_12"><H4><A href="http://parts.igem.org/Part:BBa_K1680012">BBa_K1680012:</A> Cleavage site for HCV protease</H4></LABEL><DIV>
GATGAAATGGAAGAGTGCAGTCAACATTATCCTTAT
        </DIV><H2>NDegron</H2><P>This part contains the protein coding region for an NDegron as described by Taxis et al. 2009. The NDegron enhances protein turnover by destabilising the protein it is fused to upon cleavage by TEV protease.</P><P>The part is supplied in RFC25.
This part was intended as a backup for our project, due to time restraints we did not use it.</P><LABEL class="collapse" for="_13"><H4><A href="http://parts.igem.org/Part:BBa_K1680013">BBa_K1680013:</A>NDegron</H4></LABEL><DIV style="word-wrap:break-word;width:100%;">
ATGGCCGGCCCCCCTAAGAAAAAGAGAAAAGTCAGCATTACAAGTTTGTACAAAAAAGCCGGTTCTGAGAATCTGTATTTCCAGTTCCACAAGAGCGGCGCTTGGAAGCTACCAGTTTCCCTGGTCAAGTTGGGTCTTGATAAGTTAGATTATAAGACCGGTTAA
        </DIV></DIV><DIV id="dia1" class="dia"><P>
The complex parts were assembled into the pTUM vector series (Team TU Munich 2012, e.g. <A href="http://parts.igem.org/Part:BBa_K801000">BBa_K801000</A>). Due to time constraints we were not able to clone these parts into the pSB backbone prior to part submission.
Additionally, we submitted the biobricked pRS vectors as well as the pETue vector.
</P><H2>Yeast shuttle vectors with Biobrick MCS</H2><P>These parts are biobricked shuttle vectors for yeast. The vectors were made compatible with the RFC10 standard and also contain an RFC10 cloning site. They all encode an ampicillin resistance gene and replicate to high-copy numbers in E.Coli. Furthermore, the CEN6/ARS4 (Chromosome VI centromere/Autonomously Replicating Sequence 4) cassette ensures that the plasmid stays at a low copy number in yeast, because they are treated as pseudo chromosomes. Each vector contains a different auxotrophy marker:</P><UL><LI>pRS313 - HIS3</LI><LI>pRS315 - LEU2</LI><LI>pRS316 - URA3</LI></UL><LABEL class="collapse" for="_14"><H4><A href="http://parts.igem.org/Part:BBa_K1680014">BBa_K1680014</A>: pRS313-MCS</H4></LABEL><LABEL class="collapse" for="_15"><H4><A href="http://parts.igem.org/Part:BBa_K1680015">BBa_K1680015</A>: pRS315-MCS</H4></LABEL><LABEL class="collapse" for="_16"><H4><A href="http://parts.igem.org/Part:BBa_K1680016">BBa_K1680016</A>: pRS316-MCS</H4></LABEL><H2>pETue</H2><P>This part is a E. coli expression plasmid that is compatible with RFC10. The vector can be linearized with PstI/XmaI, so that parts cut with PstI/NgoMIV can be cloned into it. Overexpression of genes cloned into this site is IPTG inducible. It also encodes for a 6xHis-tag with a Thrombin cleavage site between coding region and 6xHis-tag, so that the His-tag can easily be cleaved off. The vector replicates with a high copy number in E. coli, and has an ampicillin resistance.</P><LABEL class="collapse" for="_17"><H4><A href="http://parts.igem.org/Part:BBa_K1680026">BBa_K1680026:</A> pETue</H4></LABEL><H2>Cre-Dronpa Fusions</H2><P>These parts contain the protein coding region for the fusion constructs of Cre Recombinase (BBa_K1680007) and 145N Dronpa (<A href="http://parts.igem.org/Part:BBa_K1680006">BBa_K1680006</A>) with intermediate linkers (BBa_K1680000 - BBa_K1680003). Oligomerisation of Dronpa should stop Cre from accessing DNA in the nucleus thereby inhibiting it until light deactivation of Dronpa.</P><P>These parts are</P><LABEL class="collapse" for="_18"><H4><A href="http://parts.igem.org/Part:BBa_K1680017">BBa_K1680017:</A> L6-Cre-L6-Dronpa</H4></LABEL><LABEL class="collapse" for="_19"><H4><A href="http://parts.igem.org/Part:BBa_K1680018">BBa_K1680018:</A> L9-Cre-L9-Dronpa</H4></LABEL><LABEL class="collapse" for="_20"><H4><A href="http://parts.igem.org/Part:BBa_K1680019">BBa_K1680019:</A> L12-Cre-L12-Dronpa</H4></LABEL><LABEL class="collapse" for="_21"><H4><A href="http://parts.igem.org/Part:BBa_K1680020">BBa_K1680020:</A> L15-Cre-L15-Dronpa</H4></LABEL><H2>Dronpa caged Cre with NLS</H2><P>These parts contains the protein coding region for a fusion protein of Cre recombinase (<A href="http://parts.igem.org/Part:BBa_K1680007">BBa_K1680007</A>) that is C- &amp; N-terminal flanked with 145N Dronpa (<A href="http://parts.igem.org/Part:BBa_K1680006">BBa_K1680006</A>) using a 6/9/12/15 AA linker as spacer (BBa_K1680000 - BBa_K16800003). The construct has an N-terminal NLS sequence (<A href="http://parts.igem.org/Part:BBa_K1680004">BBa_K1680004</A>). In the expressed construct the two Dronpa domains should form multimers leading to an inactive Cre recombinase that is reversibly activatable through Dronpa illumination (see Zhou et al 2012).
</P><LABEL class="collapse" for="_22"><H4><A href="http://parts.igem.org/Part:BBa_K1680021">BBa_K1680021:</A> NLS-DN-L6-Cre-L6-DN</H4></LABEL><LABEL class="collapse" for="_23"><H4><A href="http://parts.igem.org/Part:BBa_K1680022">BBa_K1680022:</A> NLS-DN-L9-Cre-L9-DN</H4></LABEL><LABEL class="collapse" for="_24"><H4><A href="http://parts.igem.org/Part:BBa_K1680023">BBa_K1680023:</A> NLS-DN-L12-Cre-L12-DN</H4></LABEL><LABEL class="collapse" for="_25"><H4><A href="http://parts.igem.org/Part:BBa_K1680024">BBa_K1680024:</A> NLS-DN-L15-Cre-L15-DN</H4></LABEL><H2>RFP-Luciferase Cre reporter </H2><P>This part encodes a reporter cassette for the Cre Recombinase (e.g. BBa_K1680007) in Yeast. An active Cre protein will switch expression from RFP to NanoLuc luciferase. It is composed of pADH-loxp-RFP-tADH-loxp-Nluc.</P><LABEL class="collapse" for="_26"><H4><A href="http://parts.igem.org/Part:BBa_K1680025">BBa_K1680025:</A> RFP-Luciferase Cre cassette</H4></LABEL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>