<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_EPF_Lausanne_Overview"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:EPF_Lausanne/Overview">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:EPF_Lausanne/Overview&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:EPF_Lausanne/Overview&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:EPF_Lausanne/Overview&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:EPF_Lausanne/Overview" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:EPF Lausanne/Overview</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><DIV class="nav-collapse"><UL class="nav"><LI><A href="https://2014.igem.org/Team:EPF_Lausanne">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle active" data-toggle="dropdown">Project </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Overview" class="active">Overview</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Envelope_stress_responsive_bacteria">Stress Responsive Bacteria</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Yeast">Osmo Responsive Yeast</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Microfluidics">Microfluidics</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Hardware">Hardware</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Applications">Applications</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Achievements </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Data">Data</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Judging">Judging</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Parts">Parts</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Policy &amp; Practices </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/HumanPractice">Human Practices</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Safety">Bio Safety</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/PolicyPractice">Metafluidics</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Notebook </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Notebook/Bacteria">Bacteria</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Notebook/Yeast">Yeast</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Notebook/Microfluidics">Microfluidics</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Protocol">Protocols</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Notebook">Timeline</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Team">Meet us!</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Acknowledgments">Acknowledgments</A></LI></DIV></UL></DIV></DIV><DIV class="container"><DIV class="box" id="boxbread"><OL class="breadcrumb breadcrumb-arrow"><LI><A href="https://2014.igem.org/Team:EPF_Lausanne"> Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Project </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Envelope_stress_responsive_bacteria">Stress Responsive Bacteria</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Yeast">Osmo Responsive Yeast</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Microfluidics">Microfluidics</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Hardware">Hardware</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Applications">Applications</A></LI><LI class="active"><SPAN> Overview</SPAN></LI></OL></DIV><DIV class="row"><DIV class="col col-md-9"><DIV class="whitebg box"><DIV class="align-justify"><H1 class="cntr">Overview</H1><H2 class="section-heading" id="title_intro">Introduction</H2><P class="lead">

Biological responses can be quick and precise! This is the message that our team wants to convey. With this in mind, our team brought forward a novel idea: 
</P><P style="border-style:solid; border-color:#120A2A;padding: 25px; background-color:#A5D2DA">
Combining Protein Complementation techniques with biosensors and microfluidics allows fast spatiotemporal analysis of bacterial/yeast responses to stimuli.
</P><P class="lead">
Our team explored this hypothesis by engineering two stress related pathways in <I>E. coli</I> and <I>S. cerevisiae</I> with in mind the development of a BioPad: a biological touchscreen consisting of a microfluidic chip, touch-responsive bacteria/yeast, and a signal detector. Learn more about <A href="#howitworks">how the BioPad works !</A></P><P class="lead">
The pathway engineered in <I>E. coli</I>, the Cpx Pathway, is a two-component regulatory system responsive to envelope stress. A full description of the pathway is available <A target="_blank" href="https://2014.igem.org/Team:EPF_Lausanne/Envelope_stress_responsive_bacteria#cpx">here.</A>
In <I>S. cerevisiae</I> we modified the HOG Pathway - a MAPKK pathway responsive to osmotic stress. For more information concerning the HOG Pathway click <A target="_blank" href="https://2014.igem.org/Team:EPF_Lausanne/Yeast">here.</A></P><DIV class="pull-left img-left"><FIGCAPTION class="cntr">EPFL microfluidic chips</FIGCAPTION></DIV><P class="lead">

Our project also includes an extensive <B>microfluidics section</B>. Our self designed chips helped us improve precision, safety and quantification methods used throughout the project. To learn more about the microfluidic components of our project check out <A target="_blank" href="https://2014.igem.org/Team:EPF_Lausanne/Microfluidics">this link.</A></P><DIV class="pull-right img-right"><FIGCAPTION class="cntr">Raspberry Pi</FIGCAPTION></DIV><P class="lead">
Last but not least, we designed a <B>novel signal detector</B>! To make signal detection more practical we developed an automatised cheap tracking system made of a mini-computer (Raspberry Pi) and a mini-HD camera. More details concerning the BioPad detector can be found <A target="_blank" href="https://2014.igem.org/Team:EPF_Lausanne/Hardware">here.</A></P><H2 class="section-heading" id="howitworks">How the BioPad works - <I>E. coli</I></H2><DIV class="pull-right img-right"><FIGCAPTION class="cntr">A potential BioPad</FIGCAPTION></DIV><P class="lead">
Our BioPad is a self-designed PDMS microfluidic chip, made of thousands of compartments representing “pixels”. Each 300μm x 300μm x 50μm compartment contains a few layers of <I>E. coli</I>. When the surface of the chip is touched, a deformation on the chip - and thus of the compartments – leads to cellular membrane shear stress and protein aggregation/misfolding in the periplasm.</P><P class="lead">

The histidine kinase sensor CpxA auto-phosphorylates and transfers its phosphate to its corresponding relay protein, CpxR, resulting in its dimerization. We engineered the pathway, by fusing split reporter protein fragments to the CpxR (IFP1.4). This allows the two fragments to remain inactive until physical interaction of CpxR (stimulated by envelope stress) leads to the proper folding of IFP1.4  and reconstitution of its fluorescent properties. As the reconstitution of the split fragments of IFP1.4 are reversible, the system can be shutdown upon stress removal (CpxA changes conformation to become a phosphatase and induces CpxR’s dissociation). 

The BioPad also includes a signal detector. This detector is composed of an inexpensive credit card-sized single-board computer called Raspberry Pi, a highly sensitive digital camera with appropriate light filters, and a light-emitting source. It identifies and processes the position of the light/fluorescence emitted by the BioPad. The information about the position of the light relative to chip is then used to control the associated electronic device. </P><H2 id="howweengineered">Engineering the HOG pathway in <I>S. cerevisiae</I></H2><P class="lead">
The <A target="_blank" href="https://2014.igem.org/Team:EPF_Lausanne/Yeast">HOG pathway</A> controls the response to osmotic stress in yeast cells.
Like bacteria, our modified <I>S. cerevisiae</I> cells can be loaded into the PDMS microfluidic chips we have designed. Thus cells are confined in small compartments made of a soft material. When the surface of the chip is touched, it leads to a deformation of the chip and its chambers. Since the HOG pathway is reactive to turgor pressure, the mechanical pressure applied activates it. Upon induction of the pathway, which is a classical MAP kinase pathway, PBS2 – a MAPKK – is phosphorylated and binds HOG1 – a MAPK – and in turn phosphorylates it.
</P><DIV class="pull-left img-left" style="width:55%"><FIGCAPTION class="cntr">Figure of Hog1/Pbs2 split sfGFP and split rLuc</FIGCAPTION></DIV><P class="lead">
These two kinases were fused to split fluorescent and luminescent proteins, via a 13-amino acid flexible linker, by homologous recombination. This allowed us to detect the phosphorylation of Hog1 by Pbs2 in response to osmotic pressure or touch. We used split sfGFP and split Renilla luciferase tags on the C-terminals of both proteins.
</P><P class="lead">
The split sfGFP is irreversible and aimed to show the interaction between our two Pbs2 and Hog1 while we used the reversible split luciferase tags to assess the activation and inactivation of the pathway. In fact, when stress is removed, the signal should decline. The BioPad Detector is programmed to identify and process the light position and can transmit the information to a computer.
</P><H2 class="section-heading" id="thebiopad">The BioPad Detector</H2><P class="lead">

The signals induced by the BioPad are then processed by our self designed detection system: the BioPad Detector. The BioPad Detector is mainly made of a cheap computer (Raspberry Pi), a highly sensitive digital camera with appropriate light filters, and a light emitting source. The BioPad Detector locates signals from various sources (infrared fluorescence, green fluorescence and luminescence), processes them and sends back the relative positions of the signals with respect to the BioPad. Thanks to this position, we are able to extract information such as giving a computer operating system that the position represents the position of the mouse on a screen, that the well at the given position is a suitable antibiotic candidate, or that a gene of interest has been activated. We therefore effectively control a computer or any other electronic device through a living interface: the BioPad.

</P><DIV class="cntr"><FIGCAPTION class="cntr">the Raspberry Pi with a microfluidic chip on the lens</FIGCAPTION></DIV></DIV></DIV></DIV><DIV class="col col-md-3"><NAV id="affix-nav" class="sidebar hidden-sm hidden-xs"><UL class="nav sidenav box" data-spy="affix" data-offset-top="200" data-offset-bottom="600"><LI class="active"><A href="#title_intro">Introduction</A></LI><LI><A href="#howitworks">How the BioPad works - <I>E. coli</I></A></LI><LI><A href="#howweengineered">Engineering the HOG pathway in <I>S. cerevisiae</I></A></LI><LI><A href="#thebiopad">The BioPad Detector</A></LI></UL></NAV></DIV></DIV></DIV><DIV id="contactSection"><DIV><H1 class="cntr">Sponsors</H1></DIV><DIV class="footerSection container"><DIV class="copyright"><P>Copyright © iGEM EPFL 2014. All Rights Reserved</P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:EPF_Lausanne/Overview">http://2014.igem.org/Team:EPF_Lausanne/Overview</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:EPF_Lausanne/Overview" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:EPF_Lausanne/Overview" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:EPF_Lausanne/Overview&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:EPF_Lausanne/Overview&amp;oldid=396223" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>