<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_EPF_Lausanne_Yeast"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:EPF_Lausanne/Yeast">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:EPF_Lausanne/Yeast&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:EPF_Lausanne/Yeast&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:EPF_Lausanne/Yeast&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:EPF_Lausanne/Yeast" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:EPF Lausanne/Yeast</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><DIV class="nav-collapse"><UL class="nav"><LI><A href="https://2014.igem.org/Team:EPF_Lausanne">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle active" data-toggle="dropdown">Project </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Overview">Overview</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Envelope_stress_responsive_bacteria">Stress Responsive Bacteria</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Yeast" class="active">Osmo Responsive Yeast</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Microfluidics">Microfluidics</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Hardware">Hardware</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Applications">Applications</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Achievements </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Data">Data</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Judging">Judging</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Parts">Parts</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Policy &amp; Practice </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/HumanPractice">Human Practice</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Safety">Bio Safety</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/PolicyPractice">Metafluidics</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Notebook </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Notebook/Bacteria">Bacteria</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Notebook/Yeast">Yeast</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Notebook/Microfluidics">Microfluidics</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Protocol">Protocols</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Notebook">Timeline</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Team">Meet us!</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Acknowledgments">Acknowledgments</A></LI></DIV></UL></DIV></DIV><DIV class="container"><DIV class="box" id="boxbread"><OL class="breadcrumb breadcrumb-arrow"><LI><A href="https://2014.igem.org/Team:EPF_Lausanne"> Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Project </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Overview">Overview</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Envelope_stress_responsive_bacteria">Stress Responsive Bacteria</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Microfluidics">Microfluidics</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Hardware">Hardware</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Applications">Applications</A></LI><LI class="active"><SPAN> Osmo Responsive Yeast</SPAN></LI></OL></DIV><DIV class="row"><DIV class="col col-md-9"><DIV class="whitebg box"><CENTER><H1>Osmo Responsive Yeast Cells</H1></CENTER><H2 id="motivation">Where it started...</H2><DIV class="pull-right img-right" style="width: 60%"><FIGCAPTION class="cntr">The HOG pathway activation</FIGCAPTION></DIV><P> Upon a suggestion by Professor Maerkl, we decided to take a look at the High Osmolarity Glycerol(HOG) pathway in <I>S. cerevisiae</I> and see whether we could expand our project into yeast cells. Similarly to the CpxR pathway, the HOG pathway is not only responsive to hyper-osmotic stress but to also heat shock, nitrogen stress and oxidative stress.<SUP><A href="#ref1">1</A></SUP>. Yeast cells use this MAPK (Mitogen Activated Protein Kinase) pathway to coordinate intracellular activities and to optimise survival and proliferation. It is activated mainly by two membrane proteins Sho1 and Sln1, of which Sho1's role is thought to be monitoring turgor during cell growth<SUP><A href="#ref2">2</A></SUP>. Ste11, a MAPKKK, then phosphorylates Pbs2 which in turn phosphorylates the MAPK Hog1. The two proteins' copy number is well known:  2160 and 6780 molecules/cell<SUP><A href="#ref3">3</A></SUP> respectively for Pbs2 and Hog1. 

</P><DIV class="pull-left img-left" style="width: 55%;"><FIGCAPTION class="cntr">Schema of Hog1/Pbs2 Split sfGFP and Split rLuc</FIGCAPTION></DIV><P>
From this we hypothesized that by tagging downstream proteins of the pathway, Pbs2 and Hog1, with split constructs we would be able to create a touch responsive system and chose to use split sfGFP and split Renilla Luciferase tags. The split sfGFP is irreversible so will serve as a positive control for the interaction of the two proteins. Meanwhile the split Luciferase is reversible so it can act as a on-off switch triggered by osmotic stress, and hopefully mechanical pressure.</P><P>
If you want to learn more on how the split complementation assay works, visit this <A target="_blank" href="https://2014.igem.org/Team:EPF_Lausanne/Envelope_stress_responsive_bacteria#split_reporter">page</A>.</P><H2 id="advantages">Choosing to work with yeast</H2><P> In <A target="_blank" href="http://en.wikipedia.org/wiki/Synthetic_biology">synthetic biology</A>, the most used organism is undoubtedly <I>E. coli</I>. The easy nature of its manipulation makes it ideal especially when the time for the project is limited to a few months. However, studies in bacteria are not always applicable to all cells, and an extension to a eukaryotic species adds another dimension to our project. Especially in the case of the HOG pathway, there is a very similar pair of proteins in mammalian cells, MEK5 and ERK5, meaning that the project could be extended even further<SUP><A href="#ref4">4</A></SUP>. Also, thinking of the safety of the 'biopad', a leak of modified yeast cells will carry less risk than bacterial cells due to the different nature of the cells and transformation methods.</P><DIV class="pull-left img-left" style="width: 35%;"><FIGCAPTION class="cntr">Parallel with the mammalian MEK5 pathway</FIGCAPTION></DIV><P> Using <I>S. cerevisiae</I> and <A target="_blank" href="http://en.wikipedia.org/wiki/Homologous_recombination">homologous recombination</A>, we could also address other problems. In bacteria, inserting plasmids can disrupt the equilibrium of the pathway and activate it or deactivate it constitutively because the copy number will be changed.
Also, the bacterial project consisted in attaching a split construct to a homodimerising protein there would be many unspecific interactions which would not produce the desired signal.
</P><P>
 However, there weren't only advantages in choosing to work with yeast. First, we had no idea how to work with yeast cells and neither did our assistants; we had to do a lot of research on basic protocols. Moreover, we could not find a study which had already tested the interaction between two proteins in the pathway using split constructs so we were making a small gamble. And last but not least, working with yeasts takes a lot longer due to the slower doubling time.
</P><H2 id="hogpbs">Why did we choose Hog1 and Pbs2?</H2><DIV class="pull-right img-right" style="width: 28%;"><FIGCAPTION class="cntr">HOG1 3D structure</FIGCAPTION></DIV><P>As mentioned above, we were unable to find a set of proteins which had already been assayed by split complementation. This meant that we would not be sure whether the split constructs would be able to interact once fused to the proteins and which orientation to use. We chose PBS2 because this is where the stress signal converges in the pathway from different membrane sensory proteins. And we chose HOG1 as the other protein because it is phosphorylated directly by PBS2 while there is a scaffold for the upstream interactions. For the orientation, there was no doubt that we had to use the C-terminal for PBS2 because the kinase domain interacting with HOG1 is on the C-terminal and the N-terminus is physically very far. We would have liked to test both orientations on HOG1 but due to limitation of time and people to invest on this part, we decided on the C-terminus. This choice was backed by the fact that kinases have their regulation tail on the N-terminus and we did not want to risk disrupting its activation.

</P><H2 id="challenge">Learning how to handle yeast: a challenge</H2><P>Having had several courses where yeast had been mentioned, we initially thought that we would not have too much trouble understanding how to take on the project. Little did we realise how much we were wrong. It took a whole month to fully understand what we had to do in terms of the design of the gene insert and order the plasmids we needed while we prepared everything we needed. Not knowing that the number of base pairs needed for homologous recombination varies between different species of yeast confused us, and then we were confused whether we should transform with a plasmid or linear fragments. In the end, we found plasmids specifically designed for GFP-tagging in yeast on AddGene and we consulted the <A target="_blank" href="http://simanis-lab.epfl.ch/">professor Simanis</A> and his assistant Andrea Krapp who helped us confirm our design.</P><H2 id="dnaconstruct">How did we make our DNA constructs and transform our cells?</H2><P>After the long discussions, the plasmids we found were the plasmids pFA6a-link-yoSuperfolderGFP-Kan (44901) and pFA6a-link-yoSuperfolderGFP-Ura (44873)<SUP><A href="#ref5">5</A></SUP> which are for GFP tagging via homologous recombination in yeast. They differ by the yeast selection marker they encode: Kan or Ura. We did all the constructs linked to Pbs2 with the kan resistance gene while Hog1 related constructs were coupled to the Ura3 selection marker. </P><P>We replaced the GFP tag by using overlap PCR and created split Renilla Luciferase, split GFP and whole Renilla luciferase tags to finally produce linear fragments of the form below. The figure below illustrates how the modified genome would look like and what proteins it would express (example shown: GFP tagged Hog1).</P><DIV class="cntr"><FIGCAPTION class="cntr">An example of construct we inserted into <I>S. cerevisiae</I> genome: GFP tagged Hog1</FIGCAPTION></DIV><P>It took us many PCRs to produce the 8 different linear fragments we wished to use and after a failed transformation, we hypothesized that it was due to the bad quality of the products and took our time to optimize the reactions. We preferred to do sequential transformations but, due to a lack of time, we also ended up trying the co-transformation while waiting for the verification of the first successful transformations.</P><H2 id="strains">Our strains and DNA constructs</H2><P>We use the S.cerevisiae strain YM4271. This strain has several inactivated genes, the main ones being Ura3, Leu2, His3 and Trp1. To select transformant colonies, one must include a selection marker. For example, we incorporated the gene Ura3 in our integration vectors, to allow the transformed cells to grow on uracil lacking medium. This strain also has a mutation in the Ade2 gene, so it tends to accumulate red fluorescent metabolites if grown in medium lacking adenine sulfate. </P><P>
Below is a summarizing table for the eight DNA constructs and six yeast strains we obtained.
</P><TABLE id="yeasts strains" class="table table-striped table-hover"><THEAD><TR><TH> DNA sequence(s) to insert by homologous recombination </TH><TH> Transformation?</TH><TH> Characterisation?  </TH></TR></THEAD><TBODY><TR><TD> HOG1-sfGFP </TD><TD>Done</TD><TD>Not completed</TD></TR><TR><TD> HOG1-RLuc </TD><TD>Done</TD><TD>Not completed</TD></TR><TR><TD> PBS2-sfGFP </TD><TD>Done</TD><TD>Characterised</TD></TR><TR><TD> PBS2-RLuc </TD><TD>Done</TD><TD>Characterised</TD></TR><TR><TD> Split sfGFP </TD><TD>Done</TD><TD>Characterised</TD></TR><TR><TD> Split RLuc </TD><TD>Done</TD><TD>Not completed</TD></TR></TBODY></TABLE><P>
We were able to get all our transformants in time however we were not able to characterize all the constructs before the wiki freeze. Tagging Pbs2 with both the yeast sfGFP and the Renilla luciferase was successful. </P><DIV class="img-right pull-right" style="width: 60%;"><FIGCAPTION class="cntr">split GFP expressing cells stress with Acetic Acid</FIGCAPTION></DIV><P>
          
Moreover we demonstrated that the split sfGFP strain designed worked as expected: in response to an osmotic stress, Pbs2 links and phosphorylates Hog1, upon which the full sfGFP re-assembles and fluorescence is emitted.

 This means the BioPad can potentially be implemented with yeast cells and fulfill the various functions we have been thinking of - especially if the split luciferase strain is reactive in the same way.

</P><P>Find out more about what we managed to do <A target="_blank" href="https://2014.igem.org/Team:EPF_Lausanne/Results#Yeast_experiments">here!</A></P><H3 id="references"> References </H3><P><OL><LI id="ref1"><A target="_blank" href="http://www.researchgate.net/profile/Christoph_Schueller/publication/15676218_The_HOG_pathway_controls_osmotic_regulation_of_transcription_via_the_stress_response_element_%28STRE%29_of_the_Saccharomyces_cerevisiae_CTT1_gene/links/0deec528cd051ae8bc000000?ev=pub_ext_doc_dl&amp;origin=publication_detail&amp;inViewer=true">Schüller, C., Brewster, J. L., Alexander, M. R., Gustin, M. C., &amp; Ruis, H. (1994). The HOG pathway controls osmotic regulation of transcription via the stress response element (STRE) of the Saccharomyces cerevisiae CTT1 gene. The EMBO Journal, 13(18), 4382. </A></LI><LI id="ref2"><A target="_blank" href="http://mmbr.asm.org/content/66/2/300.full">Hohmann, S. (2002). Osmotic stress signaling and osmoadaptation in yeasts. Microbiology and Molecular Biology Reviews, 66(2), 300-372.</A></LI><LI id="ref3"><A target="_blank" href="http://www.nature.com/nature/journal/v425/n6959/pdf/nature02046.pdf">Ghaemmaghami, S., Huh, W. K., Bower, K., Howson, R. W., Belle, A., Dephoure, N., ... &amp; Weissman, J. S. (2003). Global analysis of protein expression in yeast. Nature, 425(6959), 737-741.</A></LI><LI id="ref4"><A target="_blank" href="http://www.nature.com/onc/journal/v26/n22/full/1210411a.html">Dhanasekaran, D. N., Kashef, K., Lee, C. M., Xu, H., &amp; Reddy, E. P. (2007). Scaffold proteins of MAP-kinase modules. Oncogene, 26(22), 3185-3202.</A></LI><LI id="ref5"><A target="_blank" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0067902">Lee, S., Lim, W. A., &amp; Thorn, K. S. (2013). Improved blue, green, and red fluorescent protein tagging vectors for <I>S. cerevisiae</I>. PloS one, 8(7), e67902.</A></LI><LI><A target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2885218/">Lou, Y., &amp; Shanklin, J. (2010). Evidence that the yeast desaturase Ole1p exists as a dimer in vivo. Journal of Biological Chemistry, 285(25), 19384-19390.</A></LI><LI><A target="_blank" href="http://onlinelibrary.wiley.com/doi/10.1002/yea.320080602/abstract">Romanos, M. A., Scorer, C. A., &amp; Clare, J. J. (1992). Foreign gene expression in yeast: a review. Yeast, 8(6), 423-488.</A></LI><LI><A target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2674829/">Boysen, J. H., Fanning, S., Newberg, J., Murphy, R. F., &amp; Mitchell, A. P. (2009). Detection of Protein–Protein Interactions Through Vesicle Targeting. Genetics, 182(1), 33-39.</A></LI><LI><A target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1635360/pdf/zmk4400.pdf">Thorsen, M., Di, Y., Tängemo, C., Morillas, M., Ahmadpour, D., Van der Does, C., ... &amp; Tamás, M. J. (2006). The MAPK Hog1p modulates Fps1p-dependent arsenite uptake and tolerance in yeast. Molecular biology of the cell, 17(10), 4400-4410.</A></LI><LI><A target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2172993/pdf/200301099.pdf">Reiser, V., Raitt, D. C., &amp; Saito, H. (2003). Yeast osmosensor Sln1 and plant cytokinin receptor Cre1 respond to changes in turgor pressure. The Journal of cell biology, 161(6), 1035-1040.</A></LI><LI><A target="_blank" href="http://www.jbc.org/content/279/23/23961.full">Dihazi, H., Kessler, R., &amp; Eschrich, K. (2004). High osmolarity glycerol (HOG) pathway-induced phosphorylation and activation of 6-phosphofructo-2-kinase are essential for glycerol accumulation and yeast cell proliferation under hyperosmotic stress. Journal of Biological Chemistry, 279(23), 23961-23968.</A></LI><LI><A target="_blank" href="http://ac.els-cdn.com/S0014579300014459/1-s2.0-S0014579300014459-main.pdf?_tid=8706315a-5650-11e4-b18f-00000aacb35d&amp;acdnat=1413586639_1a2033d4d9b3e96b4c052ac8be7bd329"> Tamás, M. J., Rep, M., Thevelein, J. M., &amp; Hohmann, S. (2000). Stimulation of the yeast high osmolarity glycerol (HOG) pathway: evidence for a signal generated by a change in turgor rather than by water stress. FEBS letters, 472(1), 159-165   </A></LI></OL></P></DIV></DIV><DIV class="col col-md-3"><NAV id="affix-nav" class="sidebar hidden-sm hidden-xs"><UL class="nav sidenav box" data-spy="affix" data-offset-top="200" data-offset-bottom="600"><LI class="active"><A href="#motivation">Motivation</A></LI><LI><A href="#advantages">Advantages</A></LI><LI><A href="#hogpbs">Hog1 and Pbs2</A></LI><LI><A href="#challenge">Working with Yeast : a challenge</A></LI><LI><A href="#dnaconstruct">DNA constructs</A></LI><LI><A href="#strains">Our strains</A></LI><LI><A href="#references">References</A></LI></UL></NAV></DIV></DIV></DIV><DIV id="contactSection"><DIV><H1 class="cntr">Sponsors</H1></DIV><DIV class="footerSection container"><DIV class="copyright"><P>Copyright © iGEM EPFL 2014. All Rights Reserved</P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:EPF_Lausanne/Yeast">http://2014.igem.org/Team:EPF_Lausanne/Yeast</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:EPF_Lausanne/Yeast" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:EPF_Lausanne/Yeast" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:EPF_Lausanne/Yeast&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:EPF_Lausanne/Yeast&amp;oldid=401094" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>