"
Page
Discussion
View source
History
teams
Log in
 
Team:EPF Lausanne/Notebook/Yeast
From 2014.igem.org
Toggle navigation
Home
Project
Overview
Stress Responsive Bacteria
Osmo Responsive Yeast
Microfluidics
Hardware
Applications
Achievements
Results
Data
Judging
Parts
Policy & Practices
Human Practices
Bio Safety
Metafluidics
Notebook
Bacteria
Yeast
Microfluidics
Protocols
Team
Timeline
Meet us!
Attributions
Acknowledgments
Home
Notebook
Bacteria
Microfluidics
Protocols
Yeast
Final constructs and primers design
08-01-2014
15.07.14-01.08.14
 
Final constructs and primers design
 
After several weeks of brainstorming, doubts and research, we finally have all our primers for the PCRs required to assemble our different constructs. 
Our design is summarized in the table below. You can also download our primers sequences here .
 
 
 
 
 
Extraction of Renilla Luciferase and Split Renilla Luciferase from Waldor's plasmids
08-07-2014
07.08.14
 
PCR of - RLuc + addition of hog1 flanking sequence H1- RLuc + addition of pbs2 flanking sequence P1 - split RLucN + pbs2 flanking sequence P2 - split RLucC + hog1 flanking sequence H2
 
 
1 Experiment 1: First try
 
1.1Material and methods
Master Mix
5x phusion GC buffer 85 μL
10mM dNTPs 8,5 μL
Phusion polymerase 4,25 μL
 
Reaction:
 
H1
H2
P1
P2
Master Mix
11,5
11,5
11,5
11,5
Forward primer 10mM
2.5
2,5
2,5
2,5
Reverse primer
10mM
2,5
2,5
2,5
2,5
Template pRLuc ou pRLucNRLucC
0.21
0,2
0,21
0,2
Nuclease Free Water
33.32
33,33
33,33
33,33
 
PCR programs: H1, H2 HLuc1, P1, P2 PLuc1
 
1.2Results
1.2.1Data
 
 
 
 
1.2.2Interpretation
It didn't work so we tried it a second time.
 
 
2 Experiment 2: Second try
 
2.1Material and methods
Master Mix
5x phusion GC buffer 85 μL
10mM dNTPs 8,5 μL
Phusion polymerase 4,25 μL
 
Remark: We added more template to the reaction: 3 μL (60ng/μL solution).
 
 
H1
H2
P1
P2
Master Mix
11,5
11,5
11,5
11,5
Forward primer 10mM
2.5
2,5
2,5
2,5
Reverse primer
10mM
2,5
2,5
2,5
2,5
Template pRLuc ou pRLucNRLucC
3
3
3
3
Nuclease Free Water
30,75
30,75
30,75
30,75
 
1.2Results
1.2.1Data
 
2.2.2Interpretation
Only one PCR worked: the C terminal part of the split renilla luciferase was successfully amplified.
Expected amplicons size: 695 bp
Nanodrop after purification: 89,2 ng/μL
Note: three tubes exploded
 
What we have: RLuc + hog1 flanking sequences
https://static.igem.org/mediawiki/2014/2/25/Hog.JPG
 
 
 
 
 
 
 
Gradient PCR of problematic sequences
08-08-2014
08.08.14
 
 
Gradient PCR of - RLuc + addition of hog1 flanking sequence H1    - RLuc + addition of pbs2 flanking sequence P1   - split RLucN + pbs2 flanking sequence P2
  
1.Material and methods
 
 
Since we had some trouble with our PCRs and our primers are quite long we decided to try a gradient PCR.
Master Mix
GC buffer 250μL
dNTPs 25μL
Phusion 12,5μL
 
Reaction
 
H1
P1
P2
Master Mix
11,5
11,5
11,5
Forward primers 10mM
2,5
2,5
2,5
Reverse primers 10mM
2,5
2,5
2,5
Template pRLuc
0,6
0,6
0,6
Water
33
33
33
 
Temperature gradient:
H1 64à72°C
P1, P2 55à63°C
 
We also increase the extension time: 30secondesà1min30seconds!
We used the Invitrogen 1kb Plus DNA Ladder. In each well of a 1% agarose gel, we loaded 8.8ul Nuclease free Water, 2ul of PCR products or Ladder and 1.2ul 10X Loading Blue Dye. The electrophoresis ran 40 minutes at 120V.
 
 
2.Results
H1
We purified two PCR samples for each experiment and measured their concentration and purity with the Nanodrop.
 
 
 
 
 
 
 
Extraction of yeast resistance genes from psfGFPkan and psfGFPUra plasmids
08-12-2014
 12.08.14
 
 
1. Experiment 1:
PCR of - sfGFP+hog1 flanking sequence H4 - split sfGFPC+hog1 flanking sequence H5 - URA3+ rLuc overlap H3 - sfGFP+pbs2 flanking sequence P3 - split sfGFPN+pbs2 flanking sequence P4 - Kan+ rLuc overlap P5 - Kan+ nLuc overlap P6 - Kan+ sfGFPN overlap P7
(First try)
 
  
 
1.1.Material and methods
 
Template psfGFPURA3 or psfGFPKan
0.6
Forward primer 10mM
2,5
Reverse primer 10mM
2,5
Master Mix (with GC buffer)
11,5
Water
33
 
1.2Results
Results visualized by gel electrophoresis (1% agar, 120V, 40min)
Constructs we should have now:
 
 
2. Experiment 2:
PCR of - sfGFP+hog1 flanking sequence H4 - split sfGFPC+hog1 flanking sequence H5 - URA3+ rLuc overlap H3 - sfGFP+pbs2 flanking sequence P3 - split sfGFPN+pbs2 flanking sequence P4 - Kan+ rLuc overlap P5 - Kan+ nLuc overlap P6 - Kan+ sfGFPN overlap P7
(Second try)
H3, H4, H5 second try: Gradient PCR
Temperature gradient 58°Cà66°C
 
25μL reaction (instead of 50μL)
 
For all reactions:
Template psfGFPURA3
0,6
Forward primer 10mM
1,25
Reverse primer 10mM
1,25
Master Mix (with GC buffer)
5,75
DMSO
0,75
Water
15,4
 
2.2Results
 
 
 
 
Overlap Extension PCRs: test with only one construct
08-14-2014
14.08.14
 
Overlap extension PCR of- rLuc-Hog1+Ura3 H6 PCR ofsplit sfGFPC+hog1 flanking sequence H5
 
Overlap extension PCR
25μL reaction
Template: 1μL (60ng) H1
                  1,7μL (100ng) H3
 
15 cycles with 62°C of annealing temperature
+15 cycles with 62°C of annealing temperature after addition of primers H7 and H1_rv
 
PCR H5
25μL reaction with good primers this time !
 
Results
 
Our first overlap PCR worked!!! :)
 
Overlap Extension PCRs: Final constructs assembly
08-15-2014
15.08.14
 
 
Overlap extension PCR of- Spit LucC-Hog1+Ura3 H7 -split GFP-N+hog1+Kan P8 -rLuc-pbs2+Kan P9 -split rLucN-pbs2+Kan P10 
Extension PCR of Split GFP-C H55 (repeat previous experiment)
 
 
1.Material and methods
 
30μL reactions
Equimolar template ratios: we used 100 ng of the longest template, and estimated the necessary amount of the shortest template with its size.
In μL
Master Mix
Template 1
Template 2 (100ng)
Water
Primers*
P8: P4+P7
5.75
P4: 0.83
P7: 2
16.4
2.5 each (p12/p2)
P9: P1+P5
5.75
P1: 0.53
P5: 1
17.7
2.5 each (p12/p2)
P10: P2+P6
5.75
P2: 0.4
P6: 2
16.9
2.5 each (p12/p2)
H7: H3+H2
5.75
H1: 1
H3: 1.7
15.4
2.5 each (h7/h1.Rv)
 
*Add primers after the first program (after fusion of the templates).
 
Thermocycler programs :
P8,9,10 : 15 cycles at 60°C for overlap, add primers, then 15 cycles at 72°C annealing. Always 2min30 of extension time at 72°C.
 H7: 15 cycles at 62°C for overlap, add primers, then 15 cycles at 62°C annealing. Always 2min30 of extension time at 72°C.
 
2.Results
 
 
Overlap Extension PCRs: Final constructs assembly 2
08-18-2014
18.08.14
 
 
 
1.Experiment: Overlap extension PCR of-split GFP-N+hog1+Kan P8 -rLuc-pbs2+Kan P9 -split rLucN-pbs2+Kan P10
 
 
1.1Material and methods
 
25μL reactions
Equimolar template ratios: we used 200 ng of the longest template, and estimated the necessary amount of the shortest template with its size.
 
In μL
Master Mix (GC buffer)
Template 1
Template 2 (100ng)
Water
Primers*
P8: P4+P7
5.75
P4: 1.6
P7: 4
8.75
2.5 each (p12/p2)
P9: P1+P5
5.75
P1: 1.1
P5: 2
11.2
2.5 each (p12/p2)
P10: P2+P6
5.75
P2: 0.8
P6: 4
9.5
2.5 each (p12/p2)
Duplicates were made
 
*Add primers after the first program (after fusion of the templates).
 
Thermocycler programs : P8,9,10 : 15 cycles at 63°C for overlap, add primers, then 15 cycles at 70°C annealing. Always 2min30 of extension time at 72°C.
 
1.2Results:
 
 
The fusion of all the fragments seems to have proceeded (thanks to 63°C annealing temperature, the primers of the previous PCR that were not washed away could not interfere), but the 70°C temperature chosen for the amplification doesn’t work for P8 and P10. Next time: DMSO duplicates and annealing of 72°C.
It isn’t impossible that 200ng of the largest template is way too high for a 25ul reaction: next time, only 50ng. 
 
 
2.Experiment: Overlap extension PCR of  -split GFP-N+hog1+Kan P8  -split rLucN-pbs2+Kan P10
 
 
2.1Material and methods
 
25μL reactions
Equimolar template ratios: we used 50 ng of the longest template, and estimated the necessary amount of the shortest template with its size.
In μL
Master Mix (GC buffer)
Template 1
Template 2 (100ng)
Water
Primers*
P8: P4+P7
5.75
P4: 2.76
P7: 2
9.49
2.5 each (p12/p2)
P10: P2+P6
5.75
P2: 0.4
P6: 2
11.1
2.5 each (p12/p2)
Duplicates with DMSO were made -> Add 0.75ul DMSO and 0.75ul of water
 
*Add primers after the first program (after fusion of the templates).
 
Thermocycler programs :P8,10 : 15 cycles at 63°C for overlap, add primers, then 15 cycles at 72°C annealing. Always 2min30 of extension time at 72°C.
 
1.2Results:
 
First transformation of yeast cells
08-27-2014
 
 
27.08.14
First yeast transformation
 
1. Material and methods
 
Yeast transformtation and homologous recombination procedure from clontech yeast Handbook (see Protocols)
 
We used a 300mL culture with an OD of 0.588.
We plated all the transformation reaction tubes on selective agar plate.
Expected transformants
 
DNA template
- hogLucUra H10
- pbsLucKan P13
- hogSplitLucUra H11
- psbSplitLucKan P14
- hogGFPUra H4
- pbsGFPKan P11
- hogSplitGFPUra H9
- pbsSplitGFPKan P12
 
Note: for each transformation we use between 1 and 2 microgramms of DNA template (we put the 30microliters of purified PCR products so the exact amount depends on the concentration 50-90ng/μL)
 
 
2.Results
We only obtained plates with pbsGFPKan transformed yeast cells.
Troubleshooting
- not enough DNA template --> we’ll try again with more than 1ng of DNA
- poor quality and purity of our templates : we had some unexpected bands when we loaded our PCRs products and sometimes even smears
- mistakes during the transformation ? It was the first time we tried this protocol
 
 
Digestion+ligation of final constructs into pSB1C3 (Second try )
09-04-2014
 
04.09.14
 
1.Restriction enzyme digestion of linearized pSB1C3 and our constructs (second Try)
 
1.1 Material and methods
iGEM Restriction Digest Protocol  http://parts.igem.org/Help:Protocols/Restriction_Digest
 
We prepare the following reaction according to this protocol:
 
Note: digested template amount X2 compare to first try
Note: We let the reaction proceed overnight instead of only 30 minutes (one of our teaching assistant advised us to do so).
 
 
2.Ligation of our constructs into linearized pSB1C3
 
2.1 Material and methods
 iGEM Ligation Protocol  http://parts.igem.org/Help:Protocols/Ligation
 
Reaction Mix
 
Note: digested template 3uL (no equimolar amounts between digested plasmid and digested construct)
 
05.09.14
 
3.Transformation of DH5α with our ligation products
 
3.1 Material and methods
 
iGEMTransformation Protocol  http://parts.igem.org/Help:Protocols/Transformation
 
 
Results
We plated the cells and we did obtain many colonies but it was only the background.
We should have made a PCR purification before the ligation it would have reduce the background and we could have isolated the transformants.
 
 
 
Second transformation of yeast cells with our 8 final constructs
09-08-2014
 
 
 
08.09.14-12.09.14
Yeasts Transformation and homologous recombination (second Try)
 
 
 
1.Material and methods
 
Yeast tranformation and homologous recombination procedure from clontech Yeast Handbook (see protocols)
 
We follow the same protocol as the first time but different DNA templates: we improved the PCR conditions and now have our fragments in higher concentrations and they seem quite pure.
 
 
2.Results
 
2.1Data
 
We obtained colonies on all plates corresponding to pbs2 +… constructs!
We inoculated them in selective medium and they grew again.
 
 
2.2Interpretation
 
 We still need to sequence the genomic DNA to be sure that they integrated the right sequences and that no mutation are present but we observed the PBS2GFP cells under the microscope and they do express the PBS2 gen fused to GFP as we can see in previous photo (figure1).
 
Strains we might have:
- Pbs2 GFP Kan
- Pbs2 Luc Kan
- Pbs2 splitGFP Kan
- Pbs2 splitLuc kan
 
Cotransformation of N and C terminal parts of the split GFP and Renilla Luciferase
09-25-2014
 
25.09.14
 
Cotransformation  - split GFP  -split Renilla Luciferase
 
1.Aim
Since we’re starting to run out of time, we’ve decided to try a cotransformation in parallel of the sequential transformations we did previously, one the two methods should give us the expected strains!
 
2.Material and methods
 
Yeast transformation and homolgous recombination procedure from clontech Yeast Handbook (see Protocols)
 
DNA Templates:
For the split GFP strain= Pbs2-split_GFP_N-Kan construct + hog1-split_GFP_C-Ura construct
For the split RLuc strain= Pbs2-split_RLuc_N-Kan construct + hog1-split_RLuc_C-Ura construct
Between 1 and 2μg (depending on the purified PCR products concentration, typically 60-100ng/μL) are used to ensure a good efficiency.
 
 
3.Results
 
We plated the transformants on SD Ura- Kan+ plates (our 2 selection markers), here are the results!
We can see several colonies on each of these plates whereas no colony grew on the negative control plate! The transformation seems to have worked well , we now  need to confirm it by testing the splits with plate reader experiments.
 
 
Split GFP stress test
10-07-2014
1 Experiment 1:  split GFP stress test
 
1.1 Material and methods
 
Procedure Generale Plate reader:
Culture overnight cells (not necessarily in selective medium) that you will need in the plate reader. Including the positive and negative The OD should be between 0.7 and 1.
Centrifuge 5 mL of our cells cultures, throw the medium and resuspend the in approximatly 800μL of PBS
Fill the wells with 150 μ If you want to put coelenterazine, add 135μL cell culture resuspended in PBS and 15 μL of the coelenterazine dilluated our expected concentration. If you don’t want to add coelenterazine, just add 150μL of cell culture resuspended in PBS.This you want to stress the cells, add 120μL of cell culture resuspended in PBS, 15μL of coelenterazine and 15μL of stress.
 
 
 
 
 
1-3:
No cells
4-6:
Non transformed
7-9:
PBS2
-GFP
10-12:
split GFP
A: No stress
 
 
 
 
B: Glucose 4%
 
 
 
 
C: NaCl 0.07M
 
 
 
 
Plate Layout ( 135μl of cells in PBS, 15μl of stress tested)
 
Non-transformed cells were grown in YPD, PBS2-GFP cells were grown in YPD supplemented with kanamycin, and split GFP strains were grown in SD-Ura dropout supplemented with kanamycin.
Non-transformed cells and PBS2-GFP cells were autofluorescent, which turned out to be a problem in th medium and a mutation in the ADE genes in the initial strain. Fluorescence normalized by OD600 seemed to drop with time contrarily to split GFP tests.
 
1.2Results
1.2.1Data
figure 1:  Fluorescence of the split GFP strain with different stresses in fonction of the time.
figure 2:  Average fluorescence of the split GFP strain stress by different stresses.
 
 
1.2.2Interpretation
First graph (figure1) shows the fluorescence of the split GFP strain with different stresses, Glucose 2% and NaCl 0.07M. The second graph (figure2) gives the average of the three wells. Fluorescence seems to increase but a Fisher test comparing 3 samples to 3 samples indicate that the difference isn't significant at 95%
 
split Luciferase under stress Test
10-10-2014
1.Experiment 1:  split Luciferase under stress test _ Plate Reader 1
 
1.1 Material and methods
Procedure Generale Plate reader:
Culture overnight cells (not necessarily in selective medium) that you will need in the plate reader. Including the positive and negative The OD should be between 0.7 and 1.
Centrifuge 5 mL of our cells cultures, throw the medium and resuspend the in approximatly 800μL of PBS
Fill the wells with 150 μ If you want to put coelenterazine, add 135μL cell culture resuspended in PBS and 15 μL of the coelenterazine dilluated our expected concentration. If you don’t want to add coelenterazine, just add 150μL of cell culture resuspended in PBS.This you want to stress the cells, add 120μL of cell culture resuspended in PBS, 15μL of coelenterazine and 15μL of stress.
 
 
Cells were all grown in YPD.
 
 
1-3:
NoCells
4-6:
Non-tranformed
7-9:
PBS2-GFP
10-12:
split GFP
A: No stress
 
 
 
 
B: Glucose 8%
 
 
 
 
C: Glucose 6%
 
 
 
 
D: Glucose 4%
 
 
 
 
E: NaCl 0.28M
 
 
 
 
F: NaCl 0.21M
 
 
 
 
G: NaCl 0.1M
 
 
 
 
H: NaCl 07M
 
 
 
 
 Plate Layout (120μl of cells, 30μl of stress)
 
1.2Results
 
All cells were autofluorescent due to mutation in ADE gene of initial strain and medium.
There were no significant measurements with the different stresses, 3 different concentrations of glucose and 4 different concentrations of NaCl. However, this is not surprising since the signal from our split GFP tag would be covered by the noise.
 
2.Experiment 2:  split Luciferase under stress test _ Plate reader 2
 
2.1 Material and methods
Procedure Generale Plate reader:
Culture overnight cells (not necessarily in selective medium) that you will need in the plate reader. Including the positive and negative The OD should be between 0.7 and 1.
Centrifuge 5 mL of our cells cultures, throw the medium and resuspend the in approximatly 800μL of PBS
Fill the wells with 150 μ If you want to put coelenterazine, add 135μL cell culture resuspended in PBS and 15 μL of the coelenterazine dilluated our expected concentration. If you don’t want to add coelenterazine, just add 150μL of cell culture resuspended in PBS.This you want to stress the cells, add 120μL of cell culture resuspended in PBS, 15μL of coelenterazine and 15μL of stress.
 
Cells were all grown in SD medium with all amino acids
 
 
1-3:
NoCells
4-6:
Non-tranformed
7-9:
PBS2-GFP
10-12:
split GFP
A: No stress
 
 
 
 
B: Glucose 8%
 
 
 
 
C: Glucose 6%
 
 
 
 
D: Glucose 4%
 
 
 
 
E: NaCl 0.28M
 
 
 
 
F: NaCl 0.21M
 
 
 
 
G: NaCl 0.1M
 
 
 
 
H: NaCl 07M
 
 
 
 
 
Plate Layout (120μl of cells, 30μl of stress)
 
2.2Results
Two plate reader experiments were conducted but the cells still contained a small amount of residual auto fluorescence. However, the PBS2-GFP strain seemed to be more fluorescent than the non-transformed cells. The average in different conditions was always higher but significant in a Fisher test of three and three samples in 5 out of 8 conditions.
 
3.Experiment 3:  split Luciferase under stress test _  plate Reder 3
 
3.1 Material and methods
PLATE READER PROTOCOL
 
Cells were all grown in SD medium with all amino acids
 
 
1-3:
No cells
4-6:
Non-transformed
7-9:
PBS2-GFP
10-12:
split GFP
A: No stress
 
 
 
 
B: Glucoise 4%
 
 
 
 
C: NaCl 0.07M
 
 
 
 
Plate Layout ( 135μl of cells in PBS, 15μl of stress tested) 
3.2Results
3.2.1Data
Figure 1: Luminescence in function of time for cells in presence of coelenterazine 1 μM
Signal-to-noise ratio : 4
 
Figure 2: Luminescence in function of time for cells in presence of coelenterazine 5 μM
Signal-to-noise ratio : 10
Figure 3: Variantion of coelenterazine concentration
 
3.2.2Interpretation
 
Hog-rLuc strain died within the plate. Pbs2-Luc strain showed positive results which correlate with documented values.
Indeed we observed that cells in presence of  coelenterazine have a much higher concentration than cell without coelenterazine. And  the higher the concentration in coelenterazine is the higher the fluorescent signal is.
August
08-01-2014
08-07-2014
08-08-2014
08-12-2014
08-14-2014
08-15-2014
08-18-2014
08-27-2014
September
09-04-2014
09-08-2014
09-25-2014
October
10-07-2014
10-10-2014
Sponsors
Copyright © iGEM EPFL 2014. All Rights Reserved
Retrieved from " http://2014.igem.org/Team:EPF_Lausanne/Notebook/Yeast "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
