"
Page
Discussion
View source
History
teams
Log in
 
Team:EPF Lausanne/Overview
From 2014.igem.org
Toggle navigation
Home
Project
Overview
Stress Responsive Bacteria
Osmo Responsive Yeast
Microfluidics
Hardware
Applications
Achievements
Results
Data
Judging
Parts
Policy & Practices
Human Practices
Bio Safety
Metafluidics
Notebook
Bacteria
Yeast
Microfluidics
Protocols
Team
Timeline
Meet us!
Attributions
Acknowledgments
Home
Project
Stress Responsive Bacteria
Osmo Responsive Yeast
Microfluidics
Hardware
Applications
Overview
Overview
Introduction
Biological responses can be quick and precise! This is the message that our team wants to convey. With this in mind, our team brought forward a novel idea:
Combining Protein Complementation techniques with biosensors and microfluidics allows fast spatiotemporal analysis of bacterial/yeast responses to stimuli.
Our team explored this hypothesis by engineering two stress related pathways in E. coli and S. cerevisiae with in mind the development of a BioPad: a biological touchscreen consisting of a microfluidic chip, touch-responsive bacteria/yeast, and a signal detector. Learn more about how the BioPad works !
The pathway engineered in E. coli, the Cpx Pathway, is a two-component regulatory system responsive to envelope stress. A full description of the pathway is available here.
In S. cerevisiae we modified the HOG Pathway - a MAPKK pathway responsive to osmotic stress. For more information concerning the HOG Pathway click here.
EPFL microfluidic chips
Our project also includes an extensive microfluidics section. Our self designed chips helped us improve precision, safety and quantification methods used throughout the project. To learn more about the microfluidic components of our project check out this link.
Raspberry Pi
Last but not least, we designed a novel signal detector! To make signal detection more practical we developed an automatised cheap tracking system made of a mini-computer (Raspberry Pi) and a mini-HD camera. More details concerning the BioPad detector can be found here.
How the BioPad works - E. coli
A potential BioPad
Our BioPad is a self-designed PDMS microfluidic chip, made of thousands of compartments representing “pixels”. Each 300μm x 300μm x 50μm compartment contains a few layers of E. coli. When the surface of the chip is touched, a deformation on the chip - and thus of the compartments – leads to cellular membrane shear stress and protein aggregation/misfolding in the periplasm.
The histidine kinase sensor CpxA auto-phosphorylates and transfers its phosphate to its corresponding relay protein, CpxR, resulting in its dimerization. We engineered the pathway, by fusing split reporter protein fragments to the CpxR (IFP1.4). This allows the two fragments to remain inactive until physical interaction of CpxR (stimulated by envelope stress) leads to the proper folding of IFP1.4  and reconstitution of its fluorescent properties. As the reconstitution of the split fragments of IFP1.4 are reversible, the system can be shutdown upon stress removal (CpxA changes conformation to become a phosphatase and induces CpxR’s dissociation).
The BioPad also includes a signal detector. This detector is composed of an inexpensive credit card-sized single-board computer called Raspberry Pi, a highly sensitive digital camera with appropriate light filters, and a light-emitting source. It identifies and processes the position of the light/fluorescence emitted by the BioPad. The information about the position of the light relative to chip is then used to control the associated electronic device.
Engineering the HOG pathway in S. cerevisiae
The HOG pathway controls the response to osmotic stress in yeast cells. Like bacteria, our modified S. cerevisiae cells can be loaded into the PDMS microfluidic chips we have designed. Thus cells are confined in small compartments made of a soft material. When the surface of the chip is touched, it leads to a deformation of the chip and its chambers. Since the HOG pathway is reactive to turgor pressure, the mechanical pressure applied activates it. Upon induction of the pathway, which is a classical MAP kinase pathway, PBS2 – a MAPKK – is phosphorylated and binds HOG1 – a MAPK – and in turn phosphorylates it.
Figure of Hog1/Pbs2 split sfGFP and split rLuc
These two kinases were fused to split fluorescent and luminescent proteins, via a 13-amino acid flexible linker, by homologous recombination. This allowed us to detect the phosphorylation of Hog1 by Pbs2 in response to osmotic pressure or touch. We used split sfGFP and split Renilla luciferase tags on the C-terminals of both proteins.
The split sfGFP is irreversible and aimed to show the interaction between our two Pbs2 and Hog1 while we used the reversible split luciferase tags to assess the activation and inactivation of the pathway. In fact, when stress is removed, the signal should decline. The BioPad Detector is programmed to identify and process the light position and can transmit the information to a computer.
The BioPad Detector
The signals induced by the BioPad are then processed by our self designed detection system: the BioPad Detector. The BioPad Detector is mainly made of a cheap computer (Raspberry Pi), a highly sensitive digital camera with appropriate light filters, and a light emitting source. The BioPad Detector locates signals from various sources (infrared fluorescence, green fluorescence and luminescence), processes them and sends back the relative positions of the signals with respect to the BioPad. Thanks to this position, we are able to extract information such as giving a computer operating system that the position represents the position of the mouse on a screen, that the well at the given position is a suitable antibiotic candidate, or that a gene of interest has been activated. We therefore effectively control a computer or any other electronic device through a living interface: the BioPad.
the Raspberry Pi with a microfluidic chip on the lens
Introduction
How the BioPad works - E. coli
Engineering the HOG pathway in S. cerevisiae
The BioPad Detector
Sponsors
Copyright © iGEM EPFL 2014. All Rights Reserved
Retrieved from " http://2014.igem.org/Team:EPF_Lausanne/Overview "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
