<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_EPF_Lausanne_Notebook_Bacteria"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:EPF_Lausanne/Notebook/Bacteria">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:EPF_Lausanne/Notebook/Bacteria&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:EPF_Lausanne/Notebook/Bacteria&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:EPF_Lausanne/Notebook/Bacteria&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:EPF_Lausanne/Notebook/Bacteria" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:EPF Lausanne/Notebook/Bacteria</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><DIV class="nav-collapse"><UL class="nav"><LI><A href="https://2014.igem.org/Team:EPF_Lausanne">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Project </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Overview">Overview</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Envelope_stress_responsive_bacteria">Stress Responsive Bacteria</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Yeast">Osmo Responsive Yeast</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Microfluidics">Microfluidics</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Hardware">Hardware</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Applications">Applications</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Achievements </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Data">Data</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Judging">Judging</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Parts">Parts</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Policy &amp; Practices </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/HumanPractice">Human Practices</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Safety">Bio Safety</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/PolicyPractice">Metafluidics</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle active" data-toggle="dropdown">Notebook </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Notebook/Bacteria" class="active">Bacteria</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Notebook/Yeast">Yeast</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Notebook/Microfluidics">Microfluidics</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Protocol">Protocols</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Notebook">Timeline</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Team">Meet us!</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Acknowledgments">Acknowledgments</A></LI></DIV></UL></DIV></DIV><DIV class="container"><DIV class="box" id="boxbread"><OL class="breadcrumb breadcrumb-arrow"><LI><A href="https://2014.igem.org/Team:EPF_Lausanne"> Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Notebook </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Notebook/Yeast">Yeast</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Notebook/Microfluidics">Microfluidics</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Protocol">Protocols</A></LI><LI class="active"><SPAN> Bacteria</SPAN></LI></OL></DIV><DIV class="row"><DIV class="col col-md-9"><DIV class="whitebg box"><H3 id="date-07-07-2014">Split IFP &amp; GFP  - Week 1</H3><SPAN>07-07-2014</SPAN><DIV class="notebook-content"><H1>Split IFP &amp; GFP  - Week 1</H1><H2>1        CPXR extraction</H2><P>Extraction of CPXR from genome of E. Coli strain K-12 MG1655</P><P> </P><H3>1.1       Material and methods</H3><P>NEB Phusion HF PCR Protocol (see protocol), with corresponding primers (cf primers datasheet) for amplification. </P><H3>1.2       Results</H3><H3>1.2.1     Data</H3><P> </P><H3>1.2.2     Interpretation</H3><P>CPXR expected size : 699 bp</P><P>Amplification of CPXR worked as expected.</P><H2>2     PCR for amplification of iGEM plasmid pSB1C3 and for addition of overlapps to CPXR</H2><P>Amplification of pSB1C1 backbone from Cambridge biobrick K325219. This backbone contains the RBS, an arabinose-inducible pBAD promoter, iGEM prefix (containing ECORI and xbaI restriction sites) and suffix (contaning SpeI and PstI restriction sites), a chloramphenicol resistance gene, and an origin of replicaiton. Additon of overlapp to CPXR in order to make a Gisbon assembly of the backbone and the CPXR</P><H3>2.1       Material and methods</H3><P>NEB Phusion PCR and suitable primers.</P><H3>2.2       Results</H3><H4>2.2.1     Data</H4><H4> </H4><P> </P><H3>2.2.2     Interpretation</H3><P>We obtained the expected amplicon, and are now able to make a Gibson assembly to fuse the backbone amplicon and the CPXR.</P></DIV></DIV><DIV class="whitebg box"><H3 id="date-07-13-2014">Split IFP &amp; GFP - Week 2</H3><SPAN>07-13-2014</SPAN><DIV class="notebook-content"><P> </P><H1>1    Gibson assembly and colony PCR of pCPXR</H1><P>Fusion of pSB1C3 backbone and CPXR</P><P> </P><H2>1.1       Material and methods</H2><P>Gibson assembly kit (see protocol) for the Gibson assembly, and Taq polymerase for colony PCR (see protocol)</P><P> </P><H2>1.2       Results</H2><P> </P><H3>1.2.1     Data</H3><P>Colonies were obtained and analyzed by colony PCR, using VF2 and VR primers from iGEM</P><P> </P><P> </P><H3>1.2.2     Interpretation</H3><P>The amplicons have the expected size, meaning the Gibson assmbly worked and that we very likely obtained pCPXR (plasmid with pSB1C3 backbone and CPXR). Sequencing will reveal if the plasmid contain any mutation.</P><H1> </H1><H1>2     Sequencing of pCPXR</H1><P>Confirmation of pCPXR sequence with Microsynth sequencing services.</P><H2>2.1       Material and methods</H2><P>We sent the plasmid with a concentration of 80 ng/μl and with premixed primers VR and VF2</P><H2>2.2       Results</H2><P> </P><H3>2.2.1     Data</H3><P>We obtained the right sequence, excepted one deletion of a cytosine in the ara promoter at the position -41. This deletion appeared to be originally in the backbone pSB1C1 that we amplified from Cambridge biobrick K325219, without consequence on the transcription of the coding sequence.</P><P> </P><H1>3    Amplification of IFP1, IFP2 and pCPXR with suitable overlaps to fuse the split at the C or N terminal of CPXR</H1><P>PCR amplification of fragments to fuse IFP1 and IFP2 at the C or N terminal of CPXR in pCPXR plasmid</P><H2>3.1       Material and methods</H2><P>NEB Phusion kit (see protocol) and suitable primers (see list). Template come from Michnick lab IFP PCA.</P><H2>3.2       Results</H2><H3>3.2.1     Data</H3><H3>3.2.2     Interpretation</H3><P>IFP1 C didn't work but the other ones have the expected size. We are going to try again IFP1 C in a gradient PCR</P><P> </P><H1>4     PCR gradient to obtain IFP1 C fragment</H1><P>Amplification of IFP1 with overlap in order to insert it in pCPXR at the C terminal of CPXR</P><H2>4.1       Material and methods</H2><P>Gradient temperature was performed as the PCR didn’t work the first time. NEB Phusion kit, same procedure as last time</P><H2>4.2       Results</H2><H3>4.2.1     Data</H3><H3>4.2.2     Interpretation</H3><P>We finally obtain the amplicon, for each temperature.</P><P> </P></DIV></DIV><DIV class="whitebg box"><H3>CheY CheZ - Week 2</H3><SPAN>07-13-2014</SPAN><DIV class="notebook-content"><H1 class="western" lang="en-US">Experiment 1: Transformation</H1><P class="western" lang="en-US">Transform competent cells with plasmids recovered from Michnick’s and Waldor’s labs</P><P lang="en-US"> </P><H2 class="western" lang="en-US">1.1 Material and methods</H2><P lang="fr-FR"><FONT face="Arial, sans-serif"><SPAN lang="en-GB"><U>14/07/14</U></SPAN></FONT></P><P lang="en-GB"> </P><P lang="en-GB"><FONT face="Arial, sans-serif">Preparation of SOC: 3 x (20µl of glucose 20mM in 1ml SOB)</FONT></P><P lang="en-GB"> </P><P lang="en-GB"><FONT face="Arial, sans-serif">Transformation Protocol from iGEM HQ with 1 uL DNA in each Ependorf</FONT></P><P lang="en-GB"> </P><TABLE class="table table-striped"><TBODY><TR valign="top"><TD width="197"><P lang="fr-FR"><FONT face="Arial, sans-serif"><STRONG>Ependorf number</STRONG></FONT></P></TD><TD width="199"><P lang="fr-FR"><FONT face="Arial, sans-serif"><STRONG>Content</STRONG></FONT></P></TD><TD width="198"><P lang="fr-FR"><FONT face="Arial, sans-serif"><STRONG>Comments</STRONG></FONT></P></TD></TR><TR valign="top"><TD width="197"><P lang="fr-FR"><FONT face="Arial, sans-serif">1, 2</FONT></P></TD><TD width="199"><P lang="fr-FR"><FONT face="Arial, sans-serif">pYNZC (34.8 ng/uL)</FONT></P></TD><TD width="198"><P lang="fr-FR"><FONT face="Arial, sans-serif">Waldor, CamR</FONT></P></TD></TR><TR valign="top"><TD width="197"><P lang="fr-FR"><FONT face="Arial, sans-serif">3, 4</FONT></P></TD><TD width="199"><P lang="fr-FR"><FONT face="Arial, sans-serif">prLucNrLucC (33 ng/uL)</FONT></P></TD><TD width="198"><P lang="fr-FR"><FONT face="Arial, sans-serif">Waldor, CamR</FONT></P></TD></TR><TR valign="top"><TD width="197"><P lang="fr-FR"><FONT face="Arial, sans-serif">5, 6</FONT></P></TD><TD width="199"><P lang="fr-FR"><FONT face="Arial, sans-serif">prLuc (30 ng/uL)</FONT></P></TD><TD width="198"><P lang="fr-FR"><FONT face="Arial, sans-serif">Waldor, CamR</FONT></P></TD></TR><TR valign="top"><TD width="197"><P lang="fr-FR"><FONT face="Arial, sans-serif">7, 8</FONT></P></TD><TD width="199"><P lang="fr-FR"><FONT face="Arial, sans-serif">pAG25-L-IFP [1]</FONT></P></TD><TD width="198"><P lang="en-GB"><FONT face="Arial, sans-serif">Michnick, AmpR</FONT></P></TD></TR><TR valign="top"><TD width="197"><P lang="en-GB"><FONT face="Arial, sans-serif">9, 10</FONT></P></TD><TD width="199"><P lang="en-GB"><FONT face="Arial, sans-serif">pAG32-L-IFP [2]</FONT></P></TD><TD width="198"><P lang="en-GB"><FONT face="Arial, sans-serif">Michnick, AmpR</FONT></P></TD></TR><TR valign="top"><TD width="197"><P lang="en-GB"><FONT face="Arial, sans-serif">11, 12</FONT></P></TD><TD width="199"><P lang="en-GB"><FONT color="#282828"><FONT face="Arial, sans-serif">10pg/ul RFP Control (pSB1A3 w/ BBa_J04450)</FONT></FONT></P></TD><TD width="198"><P lang="en-GB"><FONT face="Arial, sans-serif">AmpR</FONT></P></TD></TR></TBODY></TABLE><P lang="en-GB"> </P><P lang="en-GB"><FONT face="Arial, sans-serif">Incubation times: 2h shaking at 37°C / 180 rpm and 2h resting on ice.</FONT></P><P lang="en-GB"><FONT face="Arial, sans-serif">Plates set for incubation overnight at 37°C</FONT></P><P lang="en-GB"> </P><P lang="fr-FR"><FONT face="Arial, sans-serif"><SPAN lang="en-GB"><U>15/07/14</U></SPAN></FONT></P><P lang="en-GB"> </P><P lang="en-GB"><FONT face="Arial, sans-serif">Inoculated cultures in LB Chloramphenicol or Ampicillin (3ml) </FONT></P><P lang="en-GB"><FONT face="Arial, sans-serif">Tubes left in the fridge all day</FONT></P><P lang="en-GB"><FONT face="Arial, sans-serif">Incubation overnight at 37°C / 180 rpm</FONT></P><P lang="en-GB"> </P><P lang="en-GB"><FONT face="Arial, sans-serif"><U>16/07/14</U></FONT></P><P lang="en-GB"> </P><P lang="en-GB"><FONT face="Arial, sans-serif">OD measurement</FONT></P><H2 class="western" lang="en-US">1.2 Results</H2><H3 class="western" lang="en-US">1.2.1 Data</H3><P lang="en-US"><FONT face="Arial, sans-serif">OD: 1. pYZNC A: 2.046</FONT></P><P lang="en-US"><FONT face="Arial, sans-serif"> 2. pYZNC B: 2.033</FONT></P><P lang="en-US"><FONT face="Arial, sans-serif">3. prLucNrLucC A: failed (overcrowded plate)</FONT></P><P lang="en-US"><FONT face="Arial, sans-serif">4. prLucNrLucC B: 2.072</FONT></P><P lang="en-US"><FONT face="Arial, sans-serif">5. prLuc A: 2.028</FONT></P><P lang="en-US"><FONT face="Arial, sans-serif">6. prLuc B: 2.055</FONT></P><P lang="en-US"><FONT face="Arial, sans-serif">7. pIFP 1 A: 1.950</FONT></P><P lang="en-US"><FONT face="Arial, sans-serif">8. pIFP 1 B: 1.953</FONT></P><P lang="en-US"><FONT face="Arial, sans-serif">9. pIFP 2 A: 1.954</FONT></P><P lang="en-US"><FONT face="Arial, sans-serif">10. pIFP 2 B :1.948</FONT></P><H3 class="western" lang="en-US">1.2.2 Interpretation</H3><P lang="en-US"><FONT face="Arial, sans-serif">Tube 3 discarded, others used for miniprep and glycerol stock preparation</FONT></P><H1 class="western" lang="en-US">Experiment 2: Bioluminescence assay</H1><P class="western" lang="en-US">Make a bioluminescence assay with the previously prepared cultures</P><H2 class="western" lang="en-US">2.1Material and methods</H2><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">17.07.14:</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">Overnight cultures (3ml LB + chloramphenicol) inoculated with cells from a frozen glycerol stock from previously transformed competent cells (pYNZC A &amp; B, prLucNrLucC B, prLuc A &amp; B) were incubated at 37°C with shaking at 180 rpm). OD was measured the following day.</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">18.07.14:</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">The cultures were diluted 1:100 in 5mL fresh LB-Chloramphenicol containing 670ul of 100mM L-arabinose and grown for 3h on a rotary shaker to an OD 0.8-0.9. The cultures (500ul of each) were washed twice with phosphate buffered saline (PBS; pH 7), adjusted to an OD 0.4-0.5 (final volume of culture should be 1ml) and aliquoted into a white 96-well plates in triplicates.</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">Aliquots of native coelenterazine (clz) dissolved in ethanol (10mM) were stored at 80°C. Solution of glucose (30mM in bi-distilled water) were prepared and used at a concentration of 1mM. Prior to the assay, an aliquot of clz was thawed, diluted in PBS to a final concentration of 250uM, and incubated in the dark at room temperature (RT) for 1h. Clz was added to a final concentration of 7.5uM to the PBS-washed cells, which were subsequently incubated in the dark for 30 minutes at RT. The total luminescence of each well was then measured every two minute for a total of 10 min with an integration time of 1 sec using (WHICH MACHINE??). Immediatly thereafter, 10 ul of glucose solution was manually injected into each well and a second luminescence read was initiated as before.</FONT></FONT></P><P lang="en-US"> </P><H2 class="western" lang="en-US">2.2Results</H2><H3 class="western" lang="en-US">2.2.1Data</H3><P> </P><P> </P><H3 class="western" lang="en-US"><FONT face="Calibri">2.2.2Interpretation</FONT></H3><P lang="en-US"><FONT face="Calibri">The measurements were taken too late. The substrate (clz) already started to be depleted. The positive control gave a very strong signal at the beginning, while the negative control gave a small or no signal. The construct itself gave a signal higher than the negative control but much lower than the negative control, meaning that the construct works but isn't optimal. The effect of the addition of glucose after 10 minutes can't be determined precisely. As there was no more clz in the wells after 6-7 minutes, the shut down of the luminescence observed in the construct wasn't necessarily due to glucose addition, as exprected. We cannot conclude on this experiment. We should start again and take measurements faster.</FONT></P><P> </P></DIV></DIV><DIV class="whitebg box"><H3 id="date-07-20-2014">Competent cells</H3><SPAN>07-20-2014</SPAN><DIV class="notebook-content"><P align="RIGHT">07.07.14-13.07.14</P><P> </P><P align="CENTER"><FONT size="4"><STRONG>Competent Cells </STRONG></FONT></P><P align="CENTER"> </P><P>Protocol : Thermocompetent cells protocol</P><P>First try – Not much care, all resulting tubes were thrown away</P><P>Second try – Lesser volume, more care to cold, poured liquid nitrogen right into box for freezing.</P><P> </P><P>Results : Tests seem to indicate that our method for transformation was simply not adapted to our competent cell protocol. We do not have the appropriate sterile cold room to produce high efficiency competent cells. Therefore, for biobricks and gibson assemblies, we need to use commercial competent cells. For transformations with plasmid constructs where the concentration is in the scale of 1 ng/ul or more our competent cells are usable.</P><P> </P></DIV></DIV><DIV class="whitebg box"><H3>CheY CheZ - Week 3</H3><SPAN>07-20-2014</SPAN><DIV class="notebook-content"><P class="western" lang="en-US" align="left"> </P><H1 class="western" lang="en-US">1 Experiment 1: Bioluminescence assay</H1><P class="western" lang="en-US">Make a bioluminescence assay with the cultures transformed with plasmids from Waldor's lab</P><H2 class="western" lang="en-US">1.1 Material and methods</H2><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">22.07.14: Overnight cultures (3ml LB + chloramphenicol) inoculated with cells from a frozen glycerol stock from previously transformed competent cells (pYNZC, prLucNrLucC, prLuc) were incubated at 37°C with shaking at 180 rpm). OD was measured the following day.</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">23.07.14: The cultures were diluted 1:100 in 5mL fresh LB-Chloramphenicol containing 670ul of 100mM L-arabinose and grown for 3h on a rotary shaker to an OD 0.6-0.7. The cultures (1ml of each) were washed twice with phosphate buffered saline (PBS; pH 7), (final volume of culture should be 1ml) and aliquoted into a white 96-well plates in triplicates.</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">Aliquots of native coelenterazine (clz) dissolved in ethanol (10mM) were stored at 80°C. Prior to the assay, an aliquot of clz was thawed, diluted in PBS to a final concentration of 250uM, and incubated in the dark at room temperature (RT) for 1h. Clz was added to the PBS-washed cells, at different concentrations: 7.5uM and 75uM for pYNZC and rLucNrLucC, 7.5uM, 15uM, 75uM and 150uM for rLuc, for a total volume of 300ul in each well (see table below). Cells were subsequently incubated in the dark for 5 minutes at RT. The total luminescence of each well was then measured every three minutes for a total of 30 min with an integration time of 1 sec.</FONT></FONT></P><P lang="en-US"> </P><TABLE class="table table-striped"><TBODY><TR valign="top"><TD width="8%"><P class="western" lang="fr-FR"><FONT size="2">PYNZC</FONT></P><P class="western" lang="fr-FR"><FONT size="2">7.5uM cz</FONT></P></TD><TD width="8%"><P class="western" lang="fr-FR"><FONT size="2">PYNZC</FONT></P><P class="western" lang="fr-FR"><FONT size="2">7.5uM cz</FONT></P></TD><TD width="8%"><P class="western" lang="fr-FR"><FONT size="2">PYNZC</FONT></P><P class="western" lang="fr-FR"><FONT size="2">7.5uM cz</FONT></P></TD><TD width="8%"><P class="western" lang="fr-FR"><FONT size="2">PYNZC</FONT></P><P class="western" lang="fr-FR"><FONT size="2">75uM cz</FONT></P></TD><TD width="8%"><P class="western" lang="fr-FR"><FONT size="2">PYNZC</FONT></P><P class="western" lang="fr-FR"><FONT size="2">75uM cz</FONT></P></TD><TD width="8%"><P class="western" lang="fr-FR"><FONT size="2">PYNZC</FONT></P><P class="western" lang="fr-FR"><FONT size="2">75uM cz</FONT></P></TD><TD width="8%"><P class="western" lang="fr-FR"> </P></TD><TD width="8%"><P class="western" lang="fr-FR"> </P></TD><TD width="8%"><P class="western" lang="fr-FR"> </P></TD><TD width="9%"><P class="western" lang="fr-FR"> </P></TD><TD width="8%"><P class="western" lang="fr-FR"> </P></TD><TD width="9%"><P class="western" lang="fr-FR"> </P></TD></TR><TR valign="top"><TD width="8%"><P class="western" lang="fr-FR"><FONT size="2">PrLNrLC</FONT></P><P class="western" lang="fr-FR"><FONT size="2">7.5uM cz</FONT></P></TD><TD width="8%"><P class="western" lang="fr-FR"><FONT size="2">PrLNrLC</FONT></P><P class="western" lang="fr-FR"><FONT size="2">7.5uM cz</FONT></P></TD><TD width="8%"><P class="western" lang="fr-FR"><FONT size="2">PrLNrLC</FONT></P><P class="western" lang="fr-FR"><FONT size="2">7.5uM cz</FONT></P></TD><TD width="8%"><P class="western" lang="fr-FR"><FONT size="2">PrLNrLC</FONT></P><P class="western" lang="fr-FR"><FONT size="2">75uM cz</FONT></P></TD><TD width="8%"><P class="western" lang="fr-FR"><FONT size="2">PrLNrLC</FONT></P><P class="western" lang="fr-FR"><FONT size="2">75uM cz</FONT></P></TD><TD width="8%"><P class="western" lang="fr-FR"><FONT size="2">PrLNrLC</FONT></P><P class="western" lang="fr-FR"><FONT size="2">75uM cz</FONT></P></TD><TD width="8%"><P class="western" lang="fr-FR"> </P></TD><TD width="8%"><P class="western" lang="fr-FR"> </P></TD><TD width="8%"><P class="western" lang="fr-FR"> </P></TD><TD width="9%"><P class="western" lang="fr-FR"> </P></TD><TD width="8%"><P class="western" lang="fr-FR"> </P></TD><TD width="9%"><P class="western" lang="fr-FR"> </P></TD></TR><TR valign="top"><TD width="8%"><P class="western" lang="fr-FR"><FONT size="2">PrLuc</FONT></P><P class="western" lang="fr-FR"><FONT size="2">7.5uM cz</FONT></P></TD><TD width="8%"><P class="western" lang="fr-FR"><FONT size="2">PrLuc</FONT></P><P class="western" lang="fr-FR"><FONT size="2">7.5uM cz</FONT></P></TD><TD width="8%"><P class="western" lang="fr-FR"><FONT size="2">PrLuc</FONT></P><P class="western" lang="fr-FR"><FONT size="2">7.5uM cz</FONT></P></TD><TD width="8%"><P class="western" lang="fr-FR"><FONT size="2">PrLuc</FONT></P><P class="western" lang="fr-FR"><FONT size="2">15uM cz</FONT></P></TD><TD width="8%"><P class="western" lang="fr-FR"><FONT size="2">PrLuc</FONT></P><P class="western" lang="fr-FR"><FONT size="2">15uM cz</FONT></P></TD><TD width="8%"><P class="western" lang="fr-FR"><FONT size="2">PrLuc</FONT></P><P class="western" lang="fr-FR"><FONT size="2">15uM cz</FONT></P></TD><TD width="8%"><P class="western" lang="fr-FR"><FONT size="2">PrLuc</FONT></P><P class="western" lang="fr-FR"><FONT size="2">75uM cz</FONT></P></TD><TD width="8%"><P class="western" lang="fr-FR"><FONT size="2">PrLuc</FONT></P><P class="western" lang="fr-FR"><FONT size="2">75uM cz</FONT></P></TD><TD width="8%"><P class="western" lang="fr-FR"><FONT size="2">PrLuc</FONT></P><P class="western" lang="fr-FR"><FONT size="2">75uM cz</FONT></P></TD><TD width="9%"><P class="western" lang="fr-FR"><FONT size="2">PrLuc</FONT></P><P class="western" lang="fr-FR"><FONT size="2">150uM cz</FONT></P></TD><TD width="8%"><P class="western" lang="fr-FR"><FONT size="2">PrLuc</FONT></P><P class="western" lang="fr-FR"><FONT size="2">150uM cz</FONT></P></TD><TD width="9%"><P class="western" lang="fr-FR"><FONT size="2">PrLuc</FONT></P><P class="western" lang="fr-FR"><FONT size="2">150uM cz</FONT></P></TD></TR><TR valign="top"><TD width="8%"><P class="western" lang="fr-FR"><FONT size="2">PBS </FONT></P><P class="western" lang="fr-FR"><FONT size="2">7.5uM cz</FONT></P></TD><TD width="8%"><P class="western" lang="fr-FR"><FONT size="2">PBS </FONT></P><P class="western" lang="fr-FR"><FONT size="2">7.5uM cz</FONT></P></TD><TD width="8%"><P class="western" lang="fr-FR"><FONT size="2">PBS </FONT></P><P class="western" lang="fr-FR"><FONT size="2">7.5uM cz</FONT></P></TD><TD width="8%"><P class="western" lang="fr-FR"><FONT size="2">PBS </FONT></P><P class="western" lang="fr-FR"><FONT size="2">75uM cz</FONT></P></TD><TD width="8%"><P class="western" lang="fr-FR"><FONT size="2">PBS </FONT></P><P class="western" lang="fr-FR"><FONT size="2">75uM cz</FONT></P></TD><TD width="8%"><P class="western" lang="fr-FR"><FONT size="2">PBS </FONT></P><P class="western" lang="fr-FR"><FONT size="2">75uM cz</FONT></P></TD><TD width="8%"><P class="western" lang="fr-FR"> </P></TD><TD width="8%"><P class="western" lang="fr-FR"> </P></TD><TD width="8%"><P class="western" lang="fr-FR"> </P></TD><TD width="9%"><P class="western" lang="fr-FR"> </P></TD><TD width="8%"><P class="western" lang="fr-FR"> </P></TD><TD width="9%"><P class="western" lang="fr-FR"> </P></TD></TR></TBODY></TABLE><P lang="en-US"> </P><H2 class="western" lang="en-US">1.2Results</H2><H3 class="western" lang="en-US">1.2.1Data</H3><P lang="en-US"> </P><H3 class="western" lang="en-US">1.2.2Interpretation</H3><P lang="en-US"><FONT face="Calibri">After addition of coelenterazine to the cells in the plate, wells were full and the cultures may have spread between the wells. Our results aren't as expected, as the signal in the negative control is higher than in the pYNZC construct.</FONT></P><P lang="en-US"><FONT face="Calibri">The measurements should be taken in an opaque 96-well plate, from the bottom of the plate. The wells shouldn't be filled completely to avoid spreading.</FONT></P><P lang="en-US"><FONT face="Calibri">The concentration of bacteria should be taken in account to make the gradient of concentrations.</FONT></P><P lang="en-US"> </P><P lang="en-US"> </P><H1 class="western" lang="en-US">2 Experiment 2: PCR extraction of fragments of interest</H1><P> </P><P><FONT face="Calibri, sans-serif">Extraction of sequences of interest from plasmids to proceed to Gibson assembly of the final construct: CheYrLucNCheZrLucC.</FONT></P><P> </P><H2 class="western" lang="en-US">2.1 Material and methods</H2><P><FONT face="Calibri, sans-serif">PCR kit (NEB), see protocol from kit</FONT></P><P><FONT face="Calibri, sans-serif">Minipreps from following plasmids: CheY (Bba_K569017), CheZ (Bba_K629003), rLucN (YNZC, Waldor), rLucC (YNZC, Waldor)</FONT></P><P> </P><P><FONT face="Calibri, sans-serif">Primers (TABLE)</FONT></P><P> </P><P><FONT face="Calibri, sans-serif">Preparation of the tubes (in duplicates) for 50uL reaction:</FONT></P><TABLE class="table table-striped"><TBODY><TR valign="top"><TD width="98"><P align="right"><FONT face="Calibri, sans-serif">[uL]</FONT></P></TD><TD width="89"><P><FONT face="Calibri, sans-serif">MasterMix*</FONT></P></TD><TD width="83"><P><FONT face="Calibri, sans-serif">For_primer</FONT></P></TD><TD width="90"><P><FONT face="Calibri, sans-serif">Rev_primer </FONT></P></TD><TD width="102"><P><FONT face="Calibri, sans-serif">Template DNA</FONT></P></TD><TD width="131"><P><FONT face="Calibri, sans-serif">Nuclease free Water</FONT></P></TD></TR><TR valign="top"><TD width="98"><P><FONT face="Calibri, sans-serif">CheY</FONT></P></TD><TD width="89"><P><FONT face="Calibri, sans-serif">11.5 </FONT></P></TD><TD width="83"><P><FONT face="Calibri, sans-serif">2.5</FONT></P></TD><TD width="90"><P><FONT face="Calibri, sans-serif">2.5</FONT></P></TD><TD width="102"><P><FONT face="Calibri, sans-serif">11.7</FONT></P></TD><TD width="131"><P><FONT face="Calibri, sans-serif">21.8</FONT></P></TD></TR><TR valign="top"><TD width="98"><P><FONT face="Calibri, sans-serif">CheZ</FONT></P></TD><TD width="89"><P><FONT face="Calibri, sans-serif">11.5</FONT></P></TD><TD width="83"><P><FONT face="Calibri, sans-serif">2.5</FONT></P></TD><TD width="90"><P><FONT face="Calibri, sans-serif">2.5</FONT></P></TD><TD width="102"><P><FONT face="Calibri, sans-serif">3.4</FONT></P></TD><TD width="131"><P><FONT face="Calibri, sans-serif">30.1</FONT></P></TD></TR><TR valign="top"><TD width="98"><P><FONT face="Calibri, sans-serif">RlucN </FONT></P></TD><TD width="89"><P><FONT face="Calibri, sans-serif">11.5</FONT></P></TD><TD width="83"><P><FONT face="Calibri, sans-serif">2.5</FONT></P></TD><TD width="90"><P><FONT face="Calibri, sans-serif">2.5</FONT></P></TD><TD width="102"><P><FONT face="Calibri, sans-serif">4.1</FONT></P></TD><TD width="131"><P><FONT face="Calibri, sans-serif">29.4</FONT></P></TD></TR><TR valign="top"><TD width="98"><P><FONT face="Calibri, sans-serif">RLucC</FONT></P></TD><TD width="89"><P><FONT face="Calibri, sans-serif">11.5</FONT></P></TD><TD width="83"><P><FONT face="Calibri, sans-serif">2.5</FONT></P></TD><TD width="90"><P><FONT face="Calibri, sans-serif">2.5</FONT></P></TD><TD width="102"><P><FONT face="Calibri, sans-serif">4.1</FONT></P></TD><TD width="131"><P><FONT face="Calibri, sans-serif">29.4</FONT></P></TD></TR></TBODY></TABLE><P> </P><P><FONT face="Calibri, sans-serif">*MasterMix: for one tube: 9uL HF buffer, 0.9uL dNTP, 0.5uL polymerase</FONT></P><P> </P><P><FONT face="Calibri, sans-serif">Control tubes:</FONT></P><TABLE class="table table-striped"><TBODY><TR valign="top"><TD width="98"><P align="right"><FONT face="Calibri, sans-serif">[uL]</FONT></P></TD><TD width="89"><P><FONT face="Calibri, sans-serif">MasterMix*</FONT></P></TD><TD width="83"><P><FONT face="Calibri, sans-serif">For_primer</FONT></P></TD><TD width="90"><P><FONT face="Calibri, sans-serif">Rev_primer </FONT></P></TD><TD width="102"><P><FONT face="Calibri, sans-serif">Template DNA</FONT></P></TD><TD width="131"><P><FONT face="Calibri, sans-serif">Nuclease free Water</FONT></P></TD></TR><TR valign="top"><TD width="98"><P><FONT face="Calibri, sans-serif">CheY</FONT></P></TD><TD width="89"><P><FONT face="Calibri, sans-serif">11</FONT></P></TD><TD width="83"><P><FONT face="Calibri, sans-serif">2.5</FONT></P></TD><TD width="90"><P><FONT face="Calibri, sans-serif">2.5</FONT></P></TD><TD width="102"><P><FONT face="Calibri, sans-serif">11.7</FONT></P></TD><TD width="131"><P><FONT face="Calibri, sans-serif">22.3</FONT></P></TD></TR><TR valign="top"><TD width="98"><P><FONT face="Calibri, sans-serif">CheZ</FONT></P></TD><TD width="89"><P><FONT face="Calibri, sans-serif">11</FONT></P></TD><TD width="83"><P><FONT face="Calibri, sans-serif">2.5</FONT></P></TD><TD width="90"><P><FONT face="Calibri, sans-serif">2.5</FONT></P></TD><TD width="102"><P><FONT face="Calibri, sans-serif">3.4</FONT></P></TD><TD width="131"><P><FONT face="Calibri, sans-serif">30.6</FONT></P></TD></TR><TR valign="top"><TD width="98"><P><FONT face="Calibri, sans-serif">RlucN </FONT></P></TD><TD width="89"><P><FONT face="Calibri, sans-serif">11</FONT></P></TD><TD width="83"><P><FONT face="Calibri, sans-serif">2.5</FONT></P></TD><TD width="90"><P><FONT face="Calibri, sans-serif">2.5</FONT></P></TD><TD width="102"><P><FONT face="Calibri, sans-serif">4.1</FONT></P></TD><TD width="131"><P><FONT face="Calibri, sans-serif">29.9</FONT></P></TD></TR><TR valign="top"><TD width="98"><P><FONT face="Calibri, sans-serif">RLucC</FONT></P></TD><TD width="89"><P><FONT face="Calibri, sans-serif">11</FONT></P></TD><TD width="83"><P><FONT face="Calibri, sans-serif">2.5</FONT></P></TD><TD width="90"><P><FONT face="Calibri, sans-serif">2.5</FONT></P></TD><TD width="102"><P><FONT face="Calibri, sans-serif">4.1</FONT></P></TD><TD width="131"><P><FONT face="Calibri, sans-serif">29.9</FONT></P></TD></TR></TBODY></TABLE><P><FONT face="Calibri, sans-serif">*Control MasterMix: for one tube: 9uL HF buffer, 0.9uL dNTP</FONT></P><P> </P><P><FONT face="Calibri, sans-serif">PCR program according to Phusion Polymerase protocol</FONT></P><P> </P><H2 class="western" lang="en-US">2.2 Material and methods</H2><P> </P><P><FONT face="Calibri, sans-serif"><FONT size="3"><STRONG>1.2.1 Data</STRONG></FONT></FONT></P><P> </P></DIV></DIV><DIV class="whitebg box"><H3 id="date-07-21-2014">Split IFP &amp; GFP - Week 3</H3><SPAN>07-21-2014</SPAN><DIV class="notebook-content"><P> </P><H1>1  Gibson assembly and colony PCR for <STRONG>IFP2-pCPXR</STRONG> (at the C and N terminal)</H1><P> </P><P>Assembly of IFP2 at the C ter or N ter of CPXR in pCPXR by Gibson assembly. Colony PCR is useful to check if competent cells were transformed by our expected Gibson Assembly product and not by the empty plasmid pCPXR.</P><P>We did the GA with PCR product of pCPXR opened with corresponding primers and amplified insert, but we didn’t get rid of the pCPXR DNA template so competent cells may have been transformed with empty vectors thus acquiring chloramphenicol resistance. This is why we obtained so many negative colonies. DPN1 digestion of our PCR product would have removed the plasmids template.</P><H2>1.1       Material and methods</H2><P>We used VR ans VF2 sequencing primers.</P><OL><LI>pick a colony and pouf pouf in 10ul tubes of water. Directly put the tip in 3ml of LB with chloramphenicol in culture tubes and put at 37°.</LI><LI>Boil the 10ul tubes of water containing the picked bacteria at 95° for 10 minutes. It will be your « DNA template ».</LI><LI>Prepare the PCR mix for Taq polymerase, with 1ul of DNA template (NEB Taq protocol)</LI></OL><P> </P><P> </P><P> </P><H2>1.2       Results</H2><H3>1.2.1     Data</H3><P> </P><P> </P><P>   </P><P> </P><P>We also loaded PCR product of pCPXR &quot;opened&quot; as negative control, in order to compare the size of our GA product and pCPXR and see if it contains the IFP2. Colonies that are thought to contain the insert IFP2 are:</P><UL><LI>IFP2 C terminal: 1, 4, 5, 6, 7, 9, 10, 11, 12</LI><LI>IFP2 N terminal: 1, 2, 3, 4, 6, 9, 12</LI></UL><H3>1.2.2     Interpretation</H3><P>By observing the difference of size between the negative control (PCR of pCPXR that doesn’t contain the insert) and the colony PCR fragment, we see that we obtained some colonies containing empty vectors and some that contains very likely IFP2 insert. Further anlyisis and sequencing will confirm it.</P><H1>2        PCR amplification of IFP2 in the plasmid thought to contain IFP2</H1><P>Amplification of IFP2 from IFP2-pCPXR plasmid, to confirm that our GA product contains the insert IFP2.</P><P> </P><H2>2.1       Material and methods</H2><P>We used primers previously used to amplify and to add an overlap to IFP2 Cter and Nter respectively. As the « tail » of the primers were now annealing the plasmid, one could think that changing the Annealing temp was necessary but you can keep the same that you used the first time. Unfortunately some of the PCR tubes opend in the PCR machine so we couldn’t analyzed all of the plasmid.</P><H2>2.2       Results</H2><H3>2.2.1     Data</H3><P> </P><P> </P><H3>2.2.2     Interpretation</H3><P>All of the plasmids analysed contain the insert IFP2 and the size is correct (600 bp). Our GA worked, at least for the colonies that were analyzed on this gel. We are now ready for sequencing</P><H1>3        Sequencing IFP2-pCPXR plasmid</H1><P>To check if our GA products, pCPXR fused to IFP2 at its Cter or Nter don’t contain any mutation. We sent sample 9 for IFP2 Nter and sample 10 for C terminal.</P><H2>3.1       Material and methods</H2><P>Microsynth Barcode and a plasmid concentartion of 80 ng/ul</P><H2>3.2       Results</H2><H3>3.2.1     Data</H3><P>No mutation for IFP2 at the C terminal. One mutation in the RBS of IFP2 at the Nterminal, position -18 a A instead of a T. It shouldn't matter. </P><P> </P><H1>4       </H1><H1>Gibson assembly and colony PCR for IFP1-pCPXR (at the C and N terminal)</H1><P>Assembly of IFP1 at the C ter or N ter of CPXR in pCPXR by Gibson assembly. Colony PCR to check if competent cells were transformed by our expected Gibson Assembly product and not by the empty plasmid pCPXR.</P><H2>4.1       Material and methods</H2><P>(See above for protocol)</P><P><STRONG>WARNING </STRONG>: We should always do a negative control for the colony PCR with <STRONG>Bacteria</STRONG> transformed with the empty vectors, not a PCR of an <STRONG>empty</STRONG><STRONG>plasmid.</STRONG> Otherwise you cannot make any conclusion if the colony PCR fails or if the Gibson Assembly fails. You can compare the colony PCR quality of an expected product (pCPXR) and colony PCR of your new Gibson Assembly.</P><P> </P><H2>4.2       Results</H2><H3>4.2.1     Data</H3><P> </P><P> </P><P> </P><P> </P><P>In blue: IFP1 at the C terminal. In red: IFP2 at the N terminal</P><P> </P><H3>4.2.2     Interpretation</H3><P>Colonies that are thought to contain IFP1:</P><UL><LI>IFP1 at the C terminal: all</LI><LI>IFP1 at the N terminal: 3, 4, 6, 7, 8, 9, 10</LI></UL><P>  </P><H1>5        PCR amplification of IFP1 in IFP1-pCPXR plasmid (after mini-prep)</H1><P> </P><P>Colony PCR was not really conculsive so we decided to min-prep all the colonies and analyse the obtained plasmids.</P><P> </P><H2>5.1       Material and methods</H2><P>IFP1 amplification of the obtained plasmid was done with primers previously used to add to IFP1 the overlapp (IFP1_Cter and IFP1_Nter). We used Taq polymerase.</P><H2>5.2       Results</H2><H3>5.2.1     Data</H3><UL><LI>IFP1 amplification from colonies supposed to have the plasmid pCPXR with IFP1 at its Nterminal:</LI></UL><P> </P><P> </P><UL><LI>IFP1 amplification from colonies supposed to have the plasmid pCPXR with IFP1 at its Cterminal.</LI></UL><P> </P><P> </P><P>Expected size: 484 bp which correspond to our results.</P><P> </P><H3>5.2.2     Interpretation</H3><P>Amplicon expected size is 463 bp, which correspond to our results. Colonie 5 (N terminal) apparently doesn’t have IFP1, the other ones do contain the expected amplicon.  “+” means that we amplified IFP1 from the plasmid known to contain this sequence. We also obtained a band at 850 bp for all the colonies that have the IFP1 fragment. Interestingly we also obtained this band in the positive control.</P><P>We very likely contain pIFP1-CPXR and pCPXR-IFP1 and further sequencing will confirm it.</P></DIV></DIV><DIV class="whitebg box"><H3 id="date-07-27-2014">Split IFP &amp; GFP - Week 4</H3><SPAN>07-27-2014</SPAN><DIV class="notebook-content"><P> </P><H1>1        Restriction Analysis of IFP1 Cter and Nter</H1><P> </P><P>To check if the plasmid contains IFP1 in our Gibson Assembly product, by digesting the plasmid with NheI and SpeI and see if we obtain the expected sizes.</P><P>  </P><H2>1.1       Material and methods</H2><UL><LI>500 ng of DNA (plasmid)</LI><LI>0.5 ul of SpeI and 0.5 ul NheI (should be added at the end)</LI><LI>5 ul of cut smart buffer</LI><LI>Add water to 50 ul</LI></UL><P> </P><P>Incubate for one hour at 37°</P><P>Load on gel</P><P> </P><H2>1.2       Results</H2><H3>1.2.1     Data</H3><P> </P><P>Plasmid that have been digested in 2 fragments of size 1169 bp and 2773 bp contains a fragment, which is likely IFP1 (expected size). First gel reveals that all the plasmid supposed to contain IFP1 at the Cterminal of CPXR likely have IFP1. The second gel reveals that all colonies exept the 1<SUP>st</SUP> one likely contains IFP1 at the Nterminal. Further sequencing will confirm it.</P><H1>2        IFP1 Cter and Nter sequencing</H1><P>To check if the inserts IFP1 Cter and Nter, IFP2 Cter and Nter are present in the Gibson Assembly products (pCPXR fused by the insert below)</P><H2>2.1       Material and methods</H2><P>We will use the primer pBad For provided by mycrosynth (standard primer list) which will allow the sequencing from the end of the arabinose promoter. We will use also the VR primer to sequence Reverse from the suffix</P><P> </P><UL><LI>IFP1 Cter : <STRONG>10 </STRONG> (90 ng/ul)</LI><LI>IFP1 Nter : <STRONG>9 (</STRONG>94 ng/ul)</LI></UL><H2>2.2       Results</H2><H3>2.2.1     Data</H3><UL><LI> <STRONG>9  </STRONG><STRONG>IFP1 Nter </STRONG>: No deletion </LI><LI><STRONG>10  </STRONG><STRONG>IFP1 Cter : </STRONG>one deletion in the linker, doesn’t change the reading frame of IFP1</LI></UL><P> </P><P> </P><P><STRONG>Expected</STRONG><STRONG>sequence</STRONG>                   <STRONG>Our</STRONG><STRONG>sequence</STRONG></P><UL><LI><P>GGG TCC TCC                                     GGA ---- TCC</P><P>Glycine Ser Ser                                  Gylcine----Serine</P></LI></UL><P> </P><P> </P><H3>2.2.2     Interpretation</H3><P>The deletion in the linker doesn't shift the reading frame. One Serine is missing, meaning the linker will certainly not be as flexible as it should be between the split IFP and CPXR. We decided to keep working on this construct, because it shouldn't alterate too much the construct. </P><H1> </H1><P> </P></DIV></DIV><DIV class="whitebg box"><H3>Split IFP &amp; GFP - Week 5</H3><SPAN>07-27-2014</SPAN><DIV class="notebook-content"><H1>1        PCR of pCPXR-IFP1 and pCPXR-IFP2 to put both fragment in the same vector</H1><P> </P><P>To do a new plasmid containing our two fragments of interest, CPXR-IFP1 and CPXR-IFP2 into each configuration (IFP1 at the C or N terminal of CPXR and IFP2 at the C or Nterminal of CPXR) to avoid co-transformation during further experiment.</P><H2>1.1       Material and methods</H2><P>Phusion PCR with a temperature gradient (as our two first trial failed completely). Gradient temperature was achieved from 60° to 70°, the theorical temperature being around 70°. pCPXR-IFP2 will be entirely amplified and IFP1-CPXR fragment will be amplified by PCR with suitable overlapping ends for further Gibson assembly reaction.</P><H2>1.2       Results</H2><H3>1.2.1     Data</H3><H3> </H3><H3> </H3><P>Every template has been amplified 7 times, each with a different annealing temperature, respectively from the first amplicons : 70°, 69.5°, 68.4°, 66.4°, 64°, 62°, 60°</P><H3>1.2.2     Interpretation</H3><P>Not all the annealing temperature worked but we obtained one amplicon of each IFP1-CPXR, CPXR-IFP1, pCPXR-IFP2 opened, pIFP2-CPXR opened.</P><H1>2        Gibson assembly to make 4 different constructs containing both IFP1-CPXR and IFP2-CPXR in different configuration and colony PCR</H1><P>Fusion of the insert IFP1-CPXR (IFP1 at the C or N terminal of CPXR) in the plasmid containing IFP2-CPXR (IFP2 at the C or N terminal of CPXR), to have our four constructs:</P><P> </P><P>GA1: IFP1 at the <STRONG>Cterminal</STRONG> of CPXR and IFP2 at the <STRONG>Cterminal</STRONG> of CPXR</P><P>GA2: IFP1 at the <STRONG>Cterminal</STRONG> of CPXR and IFP2 at the <STRONG>Nterminal</STRONG> of CPXR</P><P>GA3: IFP1 at the <STRONG>Nterminal</STRONG> of CPXR and IFP2 at the <STRONG>Cterminal</STRONG> of CPXR</P><P>GA4: IFP1 at the <STRONG>Nterminal</STRONG> of CPXR and IFP2 at the <STRONG>Nterminal</STRONG> of CPXR</P><P> </P><H2>2.1       Material and methods</H2><P>NEB Gibson assembly kit and NEB Taq polymerase for the colony PCR with primers VR and VF2</P><H2>2.2       Results</H2><P>We didn’t obtain any colonies for GA1 and GA3, and only a few for GA2 and GA4. We picked 13 colonies of GA4 and 10 colonies for GA.</P><H3>2.2.1     Data</H3><UL><LI>Colony PCR of <STRONG>GA4</STRONG> with VF2 and VR primers. </LI></UL><P> </P><UL><LI> </LI></UL><P> </P><UL><LI>Colony PCR of <STRONG>GA2</STRONG> with VF2 and VR primers. Expectations are the same.</LI></UL><P> </P><P> </P><P> </P><H3>2.2.2     Interpretation</H3><P>Expectation is about 4050 for plasmid containing both IFP1-CPXR and IFP2-CPXR. Expectation is about 3000 for a plasmid containing only one fragment.</P><P>We can’t conclude anything with this ladder, and the difference between our wanted product and one that doesn’t contain both fragment of interest is of 1000 bases so we can’t reaaly see it on the gel. Mini-prep has been done and restriciton analysis will allow us to conclude if the colonies contained the expected product.</P><P> </P><H1>3        Restriction enzyme analysis of GA2 and GA4</H1><P>Digestion of our constructs GA2 (containing IFP1 at the Cter of CPXR and IFP2 at the Nter) and GA4 (containing IFP1 at the Nter and IFP2 at the Nter) with enzymes SpeI and NHeI to see if they contain the expected fragments.</P><H2>3.1       Material and methods</H2><P>NEB protocol for restriction enzyme digestion was applied and enzyme SpeI and NheI were used.</P><H2>3.2       Results</H2><H3>3.2.1     Data</H3><UL><LI>Restriction analysis of <STRONG>GA4</STRONG></LI></UL><P> </P><UL><LI>Restriction analysis of <STRONG>GA2</STRONG></LI></UL><H3>3.2.2     Interpretation</H3><P>Expected fragment with both inserts: 3273 bp + 2539 bp</P><P>With only CPXR-IFP1: 1200 bp + 3273 bp</P><P>With only CPXR-IFP2: 1400 bp + 3273 bp</P><P><STRONG>GA4: </STRONG>plasmids 2, 4, 6, 7, 10 correspond to our expectation</P><P><STRONG>GA2</STRONG>: plasmids 5, 8 correspond to our expectation </P><H1> </H1><P> </P></DIV></DIV><DIV class="whitebg box"><H3>CheY CheZ - Week 4</H3><SPAN>07-27-2014</SPAN><DIV class="notebook-content"><P class="western" lang="en-US" align="left"> </P><H1 class="western" lang="en-US">1 Experiment 1: PCR for fLucN/C extraction</H1><P class="western" lang="en-US">Purpous: extract N and C temini of firefly Luciferase from EPIC_firefly biobrick (iGEM)</P><H2 class="western" lang="en-US">1.1Material and methods</H2><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2"><STRONG>Material:</STRONG></FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">Phusion PCR kit (NEB)</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">EPIC_firefly Miniprep (Bba_K325108)</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2"><STRONG>Method</STRONG></FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">PCR tubes preparation:</FONT></FONT></P><TABLE class="table table-striped"><TBODY><TR valign="top"><TD width="10"><P class="western" lang="fr-FR"> </P></TD><TD width="73"><P class="western" lang="fr-FR"><STRONG>Fragment</STRONG></P></TD><TD width="85"><P class="western" lang="fr-FR"><STRONG>Mastermix (ul)</STRONG></P></TD><TD width="47"><P class="western" lang="fr-FR"><STRONG>Fp (ul)</STRONG></P></TD><TD width="46"><P class="western" lang="fr-FR"><STRONG>Rp(ul)</STRONG></P></TD><TD width="67"><P class="western" lang="fr-FR"><STRONG>DNA (ul)</STRONG></P></TD><TD width="75"><P class="western" lang="fr-FR"><STRONG>Water (ul)</STRONG></P></TD><TD width="76"><P class="western" lang="fr-FR"><STRONG>Total (ul)</STRONG></P></TD><TD width="87"><P class="western" lang="fr-FR"><STRONG>PCR program</STRONG></P></TD></TR><TR valign="top"><TD width="10"><P class="western" lang="fr-FR"><STRONG>1</STRONG></P></TD><TD width="73"><P class="western" lang="fr-FR">fLucN_A</P></TD><TD width="85"><P class="western" lang="fr-FR">11.5</P></TD><TD width="47"><P class="western" lang="fr-FR">2.5</P></TD><TD width="46"><P class="western" lang="fr-FR">2.5</P></TD><TD width="67"><P class="western" lang="fr-FR">4.1</P></TD><TD width="75"><P class="western" lang="fr-FR">29.4</P></TD><TD width="76"><P class="western" lang="fr-FR">50</P></TD><TD width="87"><P class="western" lang="fr-FR">2</P></TD></TR><TR valign="top"><TD width="10"><P class="western" lang="fr-FR"><STRONG>2</STRONG></P></TD><TD width="73"><P class="western" lang="fr-FR">fLucN_B</P></TD><TD width="85"><P class="western" lang="fr-FR">11.5</P></TD><TD width="47"><P class="western" lang="fr-FR">2.5</P></TD><TD width="46"><P class="western" lang="fr-FR">2.5</P></TD><TD width="67"><P class="western" lang="fr-FR">4.1</P></TD><TD width="75"><P class="western" lang="fr-FR">29.4</P></TD><TD width="76"><P class="western" lang="fr-FR">50</P></TD><TD width="87"><P class="western" lang="fr-FR">1</P></TD></TR><TR valign="top"><TD width="10"><P class="western" lang="fr-FR"><STRONG>3</STRONG></P></TD><TD width="73"><P class="western" lang="fr-FR">fLucC_A</P></TD><TD width="85"><P class="western" lang="fr-FR">11.5</P></TD><TD width="47"><P class="western" lang="fr-FR">2.5</P></TD><TD width="46"><P class="western" lang="fr-FR">2.5</P></TD><TD width="67"><P class="western" lang="fr-FR">4.1</P></TD><TD width="75"><P class="western" lang="fr-FR">29.4</P></TD><TD width="76"><P class="western" lang="fr-FR">50</P></TD><TD width="87"><P class="western" lang="fr-FR">2</P></TD></TR><TR valign="top"><TD width="10"><P class="western" lang="fr-FR"><STRONG>4</STRONG></P></TD><TD width="73"><P class="western" lang="fr-FR">fLucC_B</P></TD><TD width="85"><P class="western" lang="fr-FR">11.5</P></TD><TD width="47"><P class="western" lang="fr-FR">2.5</P></TD><TD width="46"><P class="western" lang="fr-FR">2.5</P></TD><TD width="67"><P class="western" lang="fr-FR">4.1</P></TD><TD width="75"><P class="western" lang="fr-FR">29.4</P></TD><TD width="76"><P class="western" lang="fr-FR">50</P></TD><TD width="87"><P class="western" lang="fr-FR">1</P></TD></TR><TR valign="top"><TD width="10"><P class="western" lang="fr-FR"><STRONG>5</STRONG></P></TD><TD width="73"><P class="western" lang="fr-FR">FlucN control</P></TD><TD width="85"><P class="western" lang="fr-FR">11*</P></TD><TD width="47"><P class="western" lang="fr-FR">2.5</P></TD><TD width="46"><P class="western" lang="fr-FR">2.5</P></TD><TD width="67"><P class="western" lang="fr-FR">4.1</P></TD><TD width="75"><P class="western" lang="fr-FR">29.9</P></TD><TD width="76"><P class="western" lang="fr-FR">50</P></TD><TD width="87"><P class="western" lang="fr-FR">2</P></TD></TR><TR valign="top"><TD width="10"><P class="western" lang="fr-FR"><STRONG>6</STRONG></P></TD><TD width="73"><P class="western" lang="fr-FR">FlucN control</P></TD><TD width="85"><P class="western" lang="fr-FR">11*</P></TD><TD width="47"><P class="western" lang="fr-FR">2.5</P></TD><TD width="46"><P class="western" lang="fr-FR">2.5</P></TD><TD width="67"><P class="western" lang="fr-FR">4.1</P></TD><TD width="75"><P class="western" lang="fr-FR">29.9</P></TD><TD width="76"><P class="western" lang="fr-FR">50</P></TD><TD width="87"><P class="western" lang="fr-FR">1</P></TD></TR></TBODY></TABLE><P lang="en-US"> * For the controls, there was no phusion polymerase in the MasterMix</P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">Primers: [TABLE]</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">PCR programs according to Phusion polymerase protocol</FONT></FONT></P><P lang="en-US"> </P><H2 class="western" lang="en-US">1.2Results</H2><P> </P><H3 class="western" lang="en-US">1.2.1Data</H3><P lang="en-US"> </P><H3 class="western" lang="en-US">1.2.2Interpretation</H3><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">The expected amplicon size for fLucN is 1248 bp. As the bands are between 1000 and 1500 bases, the amplicon seems to be correct. The expected amplicon size for fLucC is 477 bp. As the band is a bit lower than 500 bp, the amplicon seems to be correct. The negative controls without polymerase are also OK.</FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"> </P><H1 class="western" lang="en-US">2Experiment 2: PCR for Gibson overlaps</H1><P class="western" lang="en-US">Purpose of the experiment: add overlapping sequences to prepare Gibson assemblies</P><H2 class="western" lang="en-US">2.1Material and methods</H2><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2"><STRONG>Material:</STRONG></FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">Phusion PCR kit (NEB)</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">EPIC_firefly Miniprep (Bba_K325108)</FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2"><STRONG>Method</STRONG></FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">PCR tubes preparation:</FONT></FONT></P><TABLE class="table table-striped"><TBODY><TR valign="top"><TD width="13"><P class="western" lang="fr-FR"> </P></TD><TD width="87"><P class="western" lang="fr-FR"><STRONG>Fragment</STRONG></P></TD><TD width="101"><P class="western" lang="fr-FR"><STRONG>Mastermix (ul)</STRONG></P></TD><TD width="43"><P class="western" lang="fr-FR"><STRONG>Fp (ul)</STRONG></P></TD><TD width="39"><P class="western" lang="fr-FR"><STRONG>Rp(ul)</STRONG></P></TD><TD width="69"><P class="western" lang="fr-FR"><STRONG>DNA (ul)</STRONG></P></TD><TD width="65"><P class="western" lang="fr-FR"><STRONG>Water (ul)</STRONG></P></TD><TD width="65"><P class="western" lang="fr-FR"><STRONG>Total (ul)</STRONG></P></TD><TD width="86"><P class="western" lang="fr-FR"><STRONG>PCR program</STRONG></P></TD></TR><TR valign="top"><TD width="13"><P class="western" lang="fr-FR"><STRONG>1</STRONG></P></TD><TD width="87"><P class="western" lang="fr-FR">CheY_A</P></TD><TD width="101"><P class="western" lang="fr-FR">11.5</P></TD><TD width="43"><P class="western" lang="fr-FR">2.5</P></TD><TD width="39"><P class="western" lang="fr-FR">2.5</P></TD><TD width="69"><P class="western" lang="fr-FR">12.7</P></TD><TD width="65"><P class="western" lang="fr-FR">20.8</P></TD><TD width="65"><P class="western" lang="fr-FR">50</P></TD><TD width="86"><P class="western" lang="fr-FR">1</P></TD></TR><TR valign="top"><TD width="13"><P class="western" lang="fr-FR"><STRONG>2</STRONG></P></TD><TD width="87"><P class="western" lang="fr-FR">CheY_B</P></TD><TD width="101"><P class="western" lang="fr-FR">11.5</P></TD><TD width="43"><P class="western" lang="fr-FR">2.5</P></TD><TD width="39"><P class="western" lang="fr-FR">2.5</P></TD><TD width="69"><P class="western" lang="fr-FR">NO DNA</P></TD><TD width="65"><P class="western" lang="fr-FR">33.5</P></TD><TD width="65"><P class="western" lang="fr-FR">50</P></TD><TD width="86"><P class="western" lang="fr-FR">1</P></TD></TR><TR valign="top"><TD width="13"><P class="western" lang="fr-FR"><STRONG>3</STRONG></P></TD><TD width="87"><P class="western" lang="fr-FR">CheZ_A</P></TD><TD width="101"><P class="western" lang="fr-FR">11.5</P></TD><TD width="43"><P class="western" lang="fr-FR">2.5</P></TD><TD width="39"><P class="western" lang="fr-FR">2.5</P></TD><TD width="69"><P class="western" lang="fr-FR">5.5</P></TD><TD width="65"><P class="western" lang="fr-FR">28</P></TD><TD width="65"><P class="western" lang="fr-FR">50</P></TD><TD width="86"><P class="western" lang="fr-FR">1</P></TD></TR><TR valign="top"><TD width="13"><P class="western" lang="fr-FR"><STRONG>4</STRONG></P></TD><TD width="87"><P class="western" lang="fr-FR">CheZ_B</P></TD><TD width="101"><P class="western" lang="fr-FR">11.5</P></TD><TD width="43"><P class="western" lang="fr-FR">2.5</P></TD><TD width="39"><P class="western" lang="fr-FR">2.5</P></TD><TD width="69"><P class="western" lang="fr-FR">5.5</P></TD><TD width="65"><P class="western" lang="fr-FR">28</P></TD><TD width="65"><P class="western" lang="fr-FR">50</P></TD><TD width="86"><P class="western" lang="fr-FR">1</P></TD></TR><TR valign="top"><TD width="13"><P class="western" lang="fr-FR"><STRONG>5</STRONG></P></TD><TD width="87"><P class="western" lang="fr-FR">CheY ctrl</P></TD><TD width="101"><P class="western" lang="fr-FR">11 (no phusion)</P></TD><TD width="43"><P class="western" lang="fr-FR">2.5</P></TD><TD width="39"><P class="western" lang="fr-FR">2.5</P></TD><TD width="69"><P class="western" lang="fr-FR">12.7</P></TD><TD width="65"><P class="western" lang="fr-FR">21.3</P></TD><TD width="65"><P class="western" lang="fr-FR">50</P></TD><TD width="86"><P class="western" lang="fr-FR">1</P></TD></TR><TR valign="top"><TD width="13"><P class="western" lang="fr-FR"><STRONG>6</STRONG></P></TD><TD width="87"><P class="western" lang="fr-FR">CheZ ctrl</P></TD><TD width="101"><P class="western" lang="fr-FR">11 (no phusion)</P></TD><TD width="43"><P class="western" lang="fr-FR">2.5</P></TD><TD width="39"><P class="western" lang="fr-FR">2.5</P></TD><TD width="69"><P class="western" lang="fr-FR">5.5</P></TD><TD width="65"><P class="western" lang="fr-FR">28.5</P></TD><TD width="65"><P class="western" lang="fr-FR">50</P></TD><TD width="86"><P class="western" lang="fr-FR">1</P></TD></TR><TR valign="top"><TD width="13"><P class="western" lang="fr-FR"><STRONG>7</STRONG></P></TD><TD width="87"><P class="western" lang="fr-FR">rLucN_A</P></TD><TD width="101"><P class="western" lang="fr-FR">11.5</P></TD><TD width="43"><P class="western" lang="fr-FR">2.5</P></TD><TD width="39"><P class="western" lang="fr-FR">2.5</P></TD><TD width="69"><P class="western" lang="fr-FR">9.1</P></TD><TD width="65"><P class="western" lang="fr-FR">24.4</P></TD><TD width="65"><P class="western" lang="fr-FR">50</P></TD><TD width="86"><P class="western" lang="fr-FR">3</P></TD></TR><TR valign="top"><TD width="13"><P class="western" lang="fr-FR"><STRONG>8</STRONG></P></TD><TD width="87"><P class="western" lang="fr-FR">rLucN_B</P></TD><TD width="101"><P class="western" lang="fr-FR">11.5</P></TD><TD width="43"><P class="western" lang="fr-FR">2.5</P></TD><TD width="39"><P class="western" lang="fr-FR">2.5</P></TD><TD width="69"><P class="western" lang="fr-FR">9.1</P></TD><TD width="65"><P class="western" lang="fr-FR">24.4</P></TD><TD width="65"><P class="western" lang="fr-FR">50</P></TD><TD width="86"><P class="western" lang="fr-FR">3</P></TD></TR><TR valign="top"><TD width="13"><P class="western" lang="fr-FR"><STRONG>9</STRONG></P></TD><TD width="87"><P class="western" lang="fr-FR">rLucC_A</P></TD><TD width="101"><P class="western" lang="fr-FR">11.5</P></TD><TD width="43"><P class="western" lang="fr-FR">2.5</P></TD><TD width="39"><P class="western" lang="fr-FR">2.5</P></TD><TD width="69"><P class="western" lang="fr-FR">4.6</P></TD><TD width="65"><P class="western" lang="fr-FR">28.9</P></TD><TD width="65"><P class="western" lang="fr-FR">50</P></TD><TD width="86"><P class="western" lang="fr-FR">1</P></TD></TR><TR valign="top"><TD width="13"><P class="western" lang="fr-FR"><STRONG>10</STRONG></P></TD><TD width="87"><P class="western" lang="fr-FR">rLucC_B</P></TD><TD width="101"><P class="western" lang="fr-FR">11.5</P></TD><TD width="43"><P class="western" lang="fr-FR">2.5</P></TD><TD width="39"><P class="western" lang="fr-FR">2.5</P></TD><TD width="69"><P class="western" lang="fr-FR">4.6</P></TD><TD width="65"><P class="western" lang="fr-FR">33.5</P></TD><TD width="65"><P class="western" lang="fr-FR">50</P></TD><TD width="86"><P class="western" lang="fr-FR">1</P></TD></TR><TR valign="top"><TD width="13"><P class="western" lang="fr-FR"><STRONG>11</STRONG></P></TD><TD width="87"><P class="western" lang="fr-FR">rlucN ctrl</P></TD><TD width="101"><P class="western" lang="fr-FR">11 (no phusion)</P></TD><TD width="43"><P class="western" lang="fr-FR">2.5</P></TD><TD width="39"><P class="western" lang="fr-FR">2.5</P></TD><TD width="69"><P class="western" lang="fr-FR">9.1</P></TD><TD width="65"><P class="western" lang="fr-FR">24.9</P></TD><TD width="65"><P class="western" lang="fr-FR">50</P></TD><TD width="86"><P class="western" lang="fr-FR">1</P></TD></TR><TR valign="top"><TD width="13"><P class="western" lang="fr-FR"><STRONG>12</STRONG></P></TD><TD width="87"><P class="western" lang="fr-FR">rlucC ctrl</P></TD><TD width="101"><P class="western" lang="fr-FR">11 (no phusion)</P></TD><TD width="43"><P class="western" lang="fr-FR">2.5</P></TD><TD width="39"><P class="western" lang="fr-FR">2.5</P></TD><TD width="69"><P class="western" lang="fr-FR">4.6</P></TD><TD width="65"><P class="western" lang="fr-FR">29.4</P></TD><TD width="65"><P class="western" lang="fr-FR">50</P></TD><TD width="86"><P class="western" lang="fr-FR">1</P></TD></TR><TR valign="top"><TD width="13"><P class="western" lang="fr-FR"><STRONG>13</STRONG></P></TD><TD width="87"><P class="western" lang="fr-FR">rlucC neg_A</P></TD><TD width="101"><P class="western" lang="fr-FR">11.5</P></TD><TD width="43"><P class="western" lang="fr-FR">2.5</P></TD><TD width="39"><P class="western" lang="fr-FR">2.5</P></TD><TD width="69"><P class="western" lang="fr-FR">4.6</P></TD><TD width="65"><P class="western" lang="fr-FR">28.9</P></TD><TD width="65"><P class="western" lang="fr-FR">50</P></TD><TD width="86"><P class="western" lang="fr-FR">1</P></TD></TR><TR valign="top"><TD width="13"><P class="western" lang="fr-FR"><STRONG>14</STRONG></P></TD><TD width="87"><P class="western" lang="fr-FR">rlucC neg_B</P></TD><TD width="101"><P class="western" lang="fr-FR">11.5</P></TD><TD width="43"><P class="western" lang="fr-FR">2.5</P></TD><TD width="39"><P class="western" lang="fr-FR">2.5</P></TD><TD width="69"><P class="western" lang="fr-FR">4.6</P></TD><TD width="65"><P class="western" lang="fr-FR">28.9</P></TD><TD width="65"><P class="western" lang="fr-FR">50</P></TD><TD width="86"><P class="western" lang="fr-FR">1</P></TD></TR><TR valign="top"><TD width="13"><P class="western" lang="fr-FR"><STRONG>15</STRONG></P></TD><TD width="87"><P class="western" lang="fr-FR">rlucN neg_A</P></TD><TD width="101"><P class="western" lang="fr-FR">11.5</P></TD><TD width="43"><P class="western" lang="fr-FR">2.5</P></TD><TD width="39"><P class="western" lang="fr-FR">2.5</P></TD><TD width="69"><P class="western" lang="fr-FR">9.1</P></TD><TD width="65"><P class="western" lang="fr-FR">24.4</P></TD><TD width="65"><P class="western" lang="fr-FR">50</P></TD><TD width="86"><P class="western" lang="fr-FR">1</P></TD></TR><TR valign="top"><TD width="13"><P class="western" lang="fr-FR"><STRONG>16</STRONG></P></TD><TD width="87"><P class="western" lang="fr-FR">rLucN_neg_B</P></TD><TD width="101"><P class="western" lang="fr-FR">11.5</P></TD><TD width="43"><P class="western" lang="fr-FR">2.5</P></TD><TD width="39"><P class="western" lang="fr-FR">2.5</P></TD><TD width="69"><P class="western" lang="fr-FR">NO DNA</P></TD><TD width="65"><P class="western" lang="fr-FR">33.5</P></TD><TD width="65"><P class="western" lang="fr-FR">50</P></TD><TD width="86"><P class="western" lang="fr-FR">1</P></TD></TR><TR valign="top"><TD width="13"><P class="western" lang="fr-FR"><STRONG>17</STRONG></P></TD><TD width="87"><P class="western" lang="fr-FR">rlucC neg ctrl</P></TD><TD width="101"><P class="western" lang="fr-FR">11 (no phusion)</P></TD><TD width="43"><P class="western" lang="fr-FR">2.5</P></TD><TD width="39"><P class="western" lang="fr-FR">2.5</P></TD><TD width="69"><P class="western" lang="fr-FR">4.6</P></TD><TD width="65"><P class="western" lang="fr-FR">29.9</P></TD><TD width="65"><P class="western" lang="fr-FR">50</P></TD><TD width="86"><P class="western" lang="fr-FR">1</P></TD></TR><TR valign="top"><TD width="13"><P class="western" lang="fr-FR"><STRONG>18</STRONG></P></TD><TD width="87"><P class="western" lang="fr-FR">rlucN neg ctrl</P></TD><TD width="101"><P class="western" lang="fr-FR">11 (no phusion)</P></TD><TD width="43"><P class="western" lang="fr-FR">2.5</P></TD><TD width="39"><P class="western" lang="fr-FR">2.5</P></TD><TD width="69"><P class="western" lang="fr-FR">9.1</P></TD><TD width="65"><P class="western" lang="fr-FR">29.4</P></TD><TD width="65"><P class="western" lang="fr-FR">50</P></TD><TD width="86"><P class="western" lang="fr-FR">1</P></TD></TR><TR valign="top"><TD width="13"><P class="western" lang="fr-FR"><STRONG>19</STRONG></P></TD><TD width="87"><P class="western" lang="fr-FR">flucN_A</P></TD><TD width="101"><P class="western" lang="fr-FR">11.5</P></TD><TD width="43"><P class="western" lang="fr-FR">2.5</P></TD><TD width="39"><P class="western" lang="fr-FR">2.5</P></TD><TD width="69"><P class="western" lang="fr-FR">4.3</P></TD><TD width="65"><P class="western" lang="fr-FR">29.2</P></TD><TD width="65"><P class="western" lang="fr-FR">50</P></TD><TD width="86"><P class="western" lang="fr-FR">2</P></TD></TR><TR valign="top"><TD width="13"><P class="western" lang="fr-FR"><STRONG>20</STRONG></P></TD><TD width="87"><P class="western" lang="fr-FR">flucN_B</P></TD><TD width="101"><P class="western" lang="fr-FR">11.5</P></TD><TD width="43"><P class="western" lang="fr-FR">2.5</P></TD><TD width="39"><P class="western" lang="fr-FR">2.5</P></TD><TD width="69"><P class="western" lang="fr-FR">4.3</P></TD><TD width="65"><P class="western" lang="fr-FR">29.2</P></TD><TD width="65"><P class="western" lang="fr-FR">50</P></TD><TD width="86"><P class="western" lang="fr-FR">2</P></TD></TR><TR valign="top"><TD width="13"><P class="western" lang="fr-FR"><STRONG>21</STRONG></P></TD><TD width="87"><P class="western" lang="fr-FR">fLucC_A</P></TD><TD width="101"><P class="western" lang="fr-FR">11.5</P></TD><TD width="43"><P class="western" lang="fr-FR">2.5</P></TD><TD width="39"><P class="western" lang="fr-FR">2.5</P></TD><TD width="69"><P class="western" lang="fr-FR">4.6</P></TD><TD width="65"><P class="western" lang="fr-FR">28.9</P></TD><TD width="65"><P class="western" lang="fr-FR">50</P></TD><TD width="86"><P class="western" lang="fr-FR">1</P></TD></TR><TR valign="top"><TD width="13"><P class="western" lang="fr-FR"><STRONG>22</STRONG></P></TD><TD width="87"><P class="western" lang="fr-FR">fLucC_B</P></TD><TD width="101"><P class="western" lang="fr-FR">11.5</P></TD><TD width="43"><P class="western" lang="fr-FR">2.5</P></TD><TD width="39"><P class="western" lang="fr-FR">2.5</P></TD><TD width="69"><P class="western" lang="fr-FR">4.6</P></TD><TD width="65"><P class="western" lang="fr-FR">28.9</P></TD><TD width="65"><P class="western" lang="fr-FR">50</P></TD><TD width="86"><P class="western" lang="fr-FR">1</P></TD></TR><TR valign="top"><TD width="13"><P class="western" lang="fr-FR"><STRONG>23</STRONG></P></TD><TD width="87"><P class="western" lang="fr-FR">fLucN_neg_A</P></TD><TD width="101"><P class="western" lang="fr-FR">11.5</P></TD><TD width="43"><P class="western" lang="fr-FR">2.5</P></TD><TD width="39"><P class="western" lang="fr-FR">2.5</P></TD><TD width="69"><P class="western" lang="fr-FR">4.3</P></TD><TD width="65"><P class="western" lang="fr-FR">29.2</P></TD><TD width="65"><P class="western" lang="fr-FR">50</P></TD><TD width="86"><P class="western" lang="fr-FR">2</P></TD></TR><TR valign="top"><TD width="13"><P class="western" lang="fr-FR"><STRONG>24</STRONG></P></TD><TD width="87"><P class="western" lang="fr-FR">fLucN_neg_B</P></TD><TD width="101"><P class="western" lang="fr-FR">11.5</P></TD><TD width="43"><P class="western" lang="fr-FR">2.5</P></TD><TD width="39"><P class="western" lang="fr-FR">2.5</P></TD><TD width="69"><P class="western" lang="fr-FR">4.3</P></TD><TD width="65"><P class="western" lang="fr-FR">29.2</P></TD><TD width="65"><P class="western" lang="fr-FR">50</P></TD><TD width="86"><P class="western" lang="fr-FR">2</P></TD></TR><TR valign="top"><TD width="13"><P class="western" lang="fr-FR"><STRONG>25</STRONG></P></TD><TD width="87"><P class="western" lang="fr-FR">fLucC_neg_A</P></TD><TD width="101"><P class="western" lang="fr-FR">11.5</P></TD><TD width="43"><P class="western" lang="fr-FR">2.5</P></TD><TD width="39"><P class="western" lang="fr-FR">2.5</P></TD><TD width="69"><P class="western" lang="fr-FR">4.6</P></TD><TD width="65"><P class="western" lang="fr-FR">28.9</P></TD><TD width="65"><P class="western" lang="fr-FR">50</P></TD><TD width="86"><P class="western" lang="fr-FR">1</P></TD></TR><TR valign="top"><TD width="13"><P class="western" lang="fr-FR"><STRONG>26</STRONG></P></TD><TD width="87"><P class="western" lang="fr-FR">fLucC_neg_B</P></TD><TD width="101"><P class="western" lang="fr-FR">11.5</P></TD><TD width="43"><P class="western" lang="fr-FR">2.5</P></TD><TD width="39"><P class="western" lang="fr-FR">2.5</P></TD><TD width="69"><P class="western" lang="fr-FR">4.6</P></TD><TD width="65"><P class="western" lang="fr-FR">28.9</P></TD><TD width="65"><P class="western" lang="fr-FR">50</P></TD><TD width="86"><P class="western" lang="fr-FR">1</P></TD></TR><TR valign="top"><TD width="13"><P class="western" lang="fr-FR"><STRONG>27</STRONG></P></TD><TD width="87"><P class="western" lang="fr-FR">plasmid_A</P></TD><TD width="101"><P class="western" lang="fr-FR">11.5</P></TD><TD width="43"><P class="western" lang="fr-FR">2.5</P></TD><TD width="39"><P class="western" lang="fr-FR">2.5</P></TD><TD width="69"><P class="western" lang="fr-FR">4.1</P></TD><TD width="65"><P class="western" lang="fr-FR">29.4</P></TD><TD width="65"><P class="western" lang="fr-FR">50</P></TD><TD width="86"><P class="western" lang="fr-FR">4</P></TD></TR><TR valign="top"><TD width="13"><P class="western" lang="fr-FR"><STRONG>28</STRONG></P></TD><TD width="87"><P class="western" lang="fr-FR">plasmid_B</P></TD><TD width="101"><P class="western" lang="fr-FR">11.5</P></TD><TD width="43"><P class="western" lang="fr-FR">2.5</P></TD><TD width="39"><P class="western" lang="fr-FR">2.5</P></TD><TD width="69"><P class="western" lang="fr-FR">4.1</P></TD><TD width="65"><P class="western" lang="fr-FR">29.4</P></TD><TD width="65"><P class="western" lang="fr-FR">50</P></TD><TD width="86"><P class="western" lang="fr-FR">4</P></TD></TR></TBODY></TABLE><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">Primers[TABLE]</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">For the backbone: primers opener 1 and opener 2 were used.</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">PCR Programs accorgind to Phusion PCR protocol</FONT></FONT></P><H2 class="western" lang="en-US">2.2Results</H2><H3 class="western" lang="en-US">2.2.1Data</H3><P lang="en-US"> </P><H3 class="western" lang="en-US">2.2.2Interpretation</H3><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">The expected amplicon size for CheY (lane 1) is 486 bp. As the band is near 500 bases, the amplicon seems good. However, we detect a weak band at approximately 1000 bases that shouldn't appear. This could be an sequence that we obtained after the first PCR for CheY extraction (week 3) and that we didn't see on the gel from last week. However, this shouldn't be a problem. As the band is weak, it has not been amplified this time and it will not react in our future Gibson assembly. The negative control without polymerase (lane 5) shows the previous PCR product that has not been amplified, which is OK.</FONT></FONT></P><P lang="fr-FR"><FONT face="Calibri, sans-serif"><FONT size="2"><SPAN lang="en-US">The expected amplicon size for CheZ (lanes 3&amp;4) is 703 bp. We see that the band is appoximately at 700 bases which confirms that the amplicon is OK. The negative control without polymerase (lane 6)</SPAN><SPAN lang="en-US"> shows the previous PCR product that has not been amplified, which is OK.</SPAN></FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">The expected amplicon size for rLucN (lanes 7&amp;8) is 388 bp. As the band is detected between 300 and 500 bases, the amplicon seems to be OK.The negative control without polymerase (lane 11) shows the previous PCR product that has not been amplified, which is OK.</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">The expected amplicon size for rLucC (lanes 9&amp;10) is 669 bp. We see a band a little bit lower than 700 bases, so the amplicon seems right. The negative control without polymerase (lane 12) shows the previous PCR product that has not been amplified, which is OK.</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">The expected amplicon size for rLucC (future negative control, lanes 13&amp;14) is 662 bp. The band is a bit lower than 700 bases, which is as expected. The negative control without polymerase (lane 17) shows the previous PCR product that has not been amplified, which is OK.</FONT></FONT></P><P lang="fr-FR"><FONT face="Calibri, sans-serif"><FONT size="2"><SPAN lang="en-US">The expected amplicon size for rLucN (future negative control, lane 15) is 420 bp. As the band is between 300 and 500 bases, which is right. The negative control without polymerase (lane 18)</SPAN><SPAN lang="en-US"> shows the previous PCR product that has not been amplified, which is OK.</SPAN></FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">The expected amplicon size for fLucN (lanes 19&amp;20) is 1306 bp. The band appears a little bit lower than 1500 bases, which is OK. The expected amplicon size for fLucC (lanes 21&amp;22) is 538 bases. We can see a band at approximately 500 bases, which is as expected.</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">The expected amplicon size for fLucN control (lanes 25&amp;26) is 1306 bp and the band appears a little bit lower than 1500 bases. The expected amplicon size gor fLucC control (lanes 27&amp;28) is 531 bp, and the band appears at approximately 500 bases. An error of pipetting happened for lane 25: no PCR product was loaded.</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">The expected amplicon size for the backbone was 3311 bp. The band appears a bit higher than 2500 bases, which is as expected. </FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">Our products all seem to be ready for the following Gibson assemblies:</FONT></FONT></P><UL><LI><FONT face="Calibri, sans-serif"><FONT size="2">CheY-rLucN-CheZ-rLuccC into pSB1C3 (5 fragments)</FONT></FONT></LI><LI lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">rLucN-rLucC into pSB1C3 (negative control) (3 fragments)</FONT></FONT></LI><LI lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">CheY-fLucN-CheZ-fLucC into pSB1C3 (5 fragments)</FONT></FONT></LI><LI lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">fLucN-fLucC into pSB1C3 (negative control) (3 fragments)</FONT></FONT></LI></UL><P> </P><P class="western" lang="en-US" align="left"> </P><H1 class="western" lang="en-US">3 Experiment 3: Gibson assembly</H1><P class="western" lang="en-US">Purpous: to obtain the following final constructs:</P><UL><LI lang="en-US"><FONT face="Calibri, sans-serif">CheY-fLucN-CheZ-fLucC into pSB1C3 (5 fragments) → fYNZC</FONT></LI><LI lang="en-US"><FONT face="Calibri, sans-serif">CheY-rLucN-CheZ-rLuccC into pSB1C3 (5 fragments) → rYNZC</FONT></LI><LI lang="en-US"><FONT face="Calibri, sans-serif">fLucN-fLucC into pSB1C3 (negative control) (3 fragments) → fLucNfLucC</FONT></LI><LI lang="en-US"><FONT face="Calibri, sans-serif">rLucN-rLucC into pSB1C3 (negative control) (3 fragments) → rLucNrLucC</FONT></LI></UL><H2 class="western" lang="en-US">3.1Material and methods</H2><P lang="en-US"><FONT face="Calibri, sans-serif">We followed the protocol “Gibson Assembly Master Mix” from BioLabs, for 4-6 fragments. (Total amount of fragments should be 0.2-1pmols)</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">The tubes were prepared according to the following table:</FONT></P><P lang="en-US"> </P><TABLE class="table table-striped"><TBODY><TR valign="top"><TD width="20%"><P class="western" lang="fr-FR"><STRONG>uL</STRONG></P></TD><TD width="16%"><P class="western" lang="fr-FR"><STRONG>1</STRONG></P></TD><TD width="16%"><P class="western" lang="fr-FR"><STRONG>2</STRONG></P></TD><TD width="16%"><P class="western" lang="fr-FR"><STRONG>3</STRONG></P></TD><TD width="17%"><P class="western" lang="fr-FR"><STRONG>4</STRONG></P></TD><TD width="16%"><P class="western" lang="fr-FR"><STRONG>5 (neg control)</STRONG></P></TD></TR><TR valign="top"><TD width="20%"><P class="western" lang="fr-FR"><STRONG>Fragment 1</STRONG></P></TD><TD width="16%"><P class="western" lang="fr-FR">0.44 (CheY)</P></TD><TD width="16%"><P class="western" lang="fr-FR">0.44 (CheY)</P></TD><TD width="16%"><P class="western" lang="fr-FR">0.89 (fLucNneg)</P></TD><TD width="17%"><P class="western" lang="fr-FR">0.96 (rLucNneg)</P></TD><TD bgcolor="#b3b3b3" width="16%"><P class="western" lang="fr-FR"> </P></TD></TR><TR valign="top"><TD width="20%"><P class="western" lang="fr-FR"><STRONG>Fragment 2</STRONG></P></TD><TD width="16%"><P class="western" lang="fr-FR">0.81 (fLucN)</P></TD><TD width="16%"><P class="western" lang="fr-FR">0.39 (rLucN)</P></TD><TD width="16%"><P class="western" lang="fr-FR">0.45 (fLucCneg)</P></TD><TD width="17%"><P class="western" lang="fr-FR">0.68 (rLucCneg)</P></TD><TD bgcolor="#b3b3b3" width="16%"><P class="western" lang="fr-FR"> </P></TD></TR><TR valign="top"><TD width="20%"><P class="western" lang="fr-FR"><STRONG>Fragment 3</STRONG></P></TD><TD width="16%"><P class="western" lang="fr-FR">0.58 (CheZ)</P></TD><TD width="16%"><P class="western" lang="fr-FR">0.58 (CheZ)</P></TD><TD bgcolor="#b3b3b3" width="16%"><P class="western" lang="fr-FR"> </P></TD><TD bgcolor="#b3b3b3" width="17%"><P class="western" lang="fr-FR"> </P></TD><TD bgcolor="#b3b3b3" width="16%"><P class="western" lang="fr-FR"> </P></TD></TR><TR valign="top"><TD width="20%"><P class="western" lang="fr-FR"><STRONG>Fragment 4</STRONG></P></TD><TD width="16%"><P class="western" lang="fr-FR">0.37 (fLucC)</P></TD><TD width="16%"><P class="western" lang="fr-FR">0.48 (rLucC)</P></TD><TD bgcolor="#b3b3b3" width="16%"><P class="western" lang="fr-FR"> </P></TD><TD bgcolor="#b3b3b3" width="17%"><P class="western" lang="fr-FR"> </P></TD><TD bgcolor="#b3b3b3" width="16%"><P class="western" lang="fr-FR"> </P></TD></TR><TR valign="top"><TD width="20%"><P class="western" lang="fr-FR"><STRONG>pSB1C3 (opened)</STRONG></P></TD><TD width="16%"><P class="western" lang="fr-FR">1.27</P></TD><TD width="16%"><P class="western" lang="fr-FR">1.27</P></TD><TD width="16%"><P class="western" lang="fr-FR">1.27</P></TD><TD width="17%"><P class="western" lang="fr-FR">1.27</P></TD><TD width="16%"><P class="western" lang="fr-FR">1.27</P></TD></TR><TR valign="top"><TD width="20%"><P class="western" lang="fr-FR"><STRONG>MasterMix</STRONG></P></TD><TD width="16%"><P class="western" lang="fr-FR">10</P></TD><TD width="16%"><P class="western" lang="fr-FR">10</P></TD><TD width="16%"><P class="western" lang="fr-FR">10</P></TD><TD width="17%"><P class="western" lang="fr-FR">10</P></TD><TD width="16%"><P class="western" lang="fr-FR">10</P></TD></TR><TR valign="top"><TD width="20%"><P class="western" lang="fr-FR"><STRONG>Deionized Water</STRONG></P></TD><TD width="16%"><P class="western" lang="fr-FR">6.53</P></TD><TD width="16%"><P class="western" lang="fr-FR">6.84</P></TD><TD width="16%"><P class="western" lang="fr-FR">7.39</P></TD><TD width="17%"><P class="western" lang="fr-FR">7.09</P></TD><TD width="16%"><P class="western" lang="fr-FR">18.73</P></TD></TR></TBODY></TABLE><P lang="en-US"><FONT face="Calibri, sans-serif">The tubes were then incubated in a thermocycler according to the protocol.</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">Competent cells were transformed with the products, plated (2 plates for each construct) and incubated overnight.</FONT></P><H2 class="western" lang="en-US">3.2Results</H2><H3 class="western" lang="en-US">3.2.1Data</H3><H3 class="western" lang="en-US">3.2.2Interpretation</H3><P lang="en-US"><FONT face="Calibri, sans-serif">All the plates had a lot of colonies, except for the negative control, which had a lot less, as expected, as it was transformed with linear fragments. The remaining colonies can be due to some vectors that have been used for the PCR amplification and that stayed circular.</FONT></P><P lang="en-US"> </P><H1 class="western" lang="en-US">4 Experiment 4: Gibson inoculation</H1><P class="western" lang="en-US">Purpose of the experiment: Pick up transformed single colonies in order to obtain the plasmids of our final constructs.</P><H2 class="western" lang="en-US">4.1Material and methods</H2><P lang="en-US"><FONT face="Calibri, sans-serif">Procedure:</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">Pick one colony with a tip (best = small colonies)</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">Agitate the tip in 1uL nuclease free water, then heat-shock the tubes for 10 min at 95° (→ colony PCR, see experiment 3)</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">Drop the tip in 3mL LB chloramphenicol and incubate overnight at 37°C, 180rpm (→ Minipreps + Glycerol stocks, see exp 4)</FONT></P><P lang="en-US"><SPAN style="font-family: Calibri, sans-serif;">Plate 1A → tubes 1-5 Plate 1B → tubes 6-10</SPAN></P><P lang="en-US"><FONT face="Calibri, sans-serif"> Plate 2A → tubes 11-15 Plate 2B → tubes 16-20</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"> Plate 3A → tubes 21-25 Plate 3B → tubes 26-30</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"> Plate 4A → tubes 31-35 Plate 4B → tubes 36-40</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"> Plate Neg → tubes C1-4</FONT></P><P lang="en-US"> </P><H2 class="western" lang="en-US">4.2Results</H2><P lang="en-US"><FONT face="Calibri, sans-serif">Results for the colony PCR will be discussed in the third part.</FONT></P><P lang="en-US"> </P><P lang="en-US"> </P><H1 class="western" lang="en-US">5 Experiment 5: Colony PCR</H1><P class="western" lang="en-US">Purpose of the experiment: We want to verify that the transformed cell contain the construct of interest.</P><H2 class="western" lang="en-US">5.1Material and methods</H2><P lang="en-US"><FONT face="Calibri, sans-serif">Protocol for 25uL reaction (with TAQ Polymerase)</FONT></P><UL><LI lang="en-US"><FONT face="Calibri, sans-serif">2.5uL 10x ThermoPol buffer</FONT></LI><LI lang="en-US"><FONT face="Calibri, sans-serif">0.5uL 10mM dNTPs</FONT></LI><LI lang="en-US"><FONT face="Calibri, sans-serif">0.125uL Taq DNA Polymerase</FONT></LI><LI lang="en-US"><FONT face="Calibri, sans-serif">1uL template DNA (from heat-shocked tubes)</FONT></LI><LI lang="en-US"><FONT face="Calibri, sans-serif">0.5uL forward primer</FONT></LI><LI lang="en-US"><FONT face="Calibri, sans-serif">0.5uL reverse primer</FONT></LI><LI lang="en-US"><FONT face="Calibri, sans-serif">19.875uL nuclease free water</FONT></LI></UL><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif">PCR program:</FONT></P><UL><LI lang="en-US"><FONT face="Calibri, sans-serif">Initial denaturation: 30 sec at 95°C</FONT></LI><LI lang="fr-FR"><FONT face="Calibri, sans-serif"><SPAN lang="en-US">30 cycles: 10-</SPAN><SPAN lang="en-US"><U><STRONG>30</STRONG></U></SPAN><SPAN lang="en-US"> sec at 95°C</SPAN></FONT></LI><LI lang="fr-FR"><FONT face="Calibri, sans-serif"><SPAN lang="en-US">15-</SPAN><SPAN lang="en-US"><U><STRONG>60</STRONG></U></SPAN><SPAN lang="en-US"> sec at 45-68°C (annealing temperature (A.T.))</SPAN></FONT></LI><LI lang="en-US"><FONT face="Calibri, sans-serif">1 min/kb at 68°C</FONT></LI><LI lang="en-US"><FONT face="Calibri, sans-serif">Final extension: 5 min at 68°C</FONT></LI><LI lang="en-US"><FONT face="Calibri, sans-serif">Pause at 4°C</FONT></LI></UL><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif">Program used:</FONT></P><UL><LI lang="en-US"><FONT face="Calibri, sans-serif">CHETAQON (A.T. 54°C, ext. time 3 min) → tubes 1 to 10 + C1, primers CheY for / fLucC rev</FONT></LI><LI lang="en-US"><FONT face="Calibri, sans-serif">CHETAQTW (A.T. 51°C, ext. time 2 min) → tubes 11 to 20 + C2, primers CheY For / rLucC rev</FONT></LI><LI lang="en-US"><FONT face="Calibri, sans-serif">CHETAQTH (A.T. 52°C, ext. time 2 min) → tubes 21 to 30 + C3, primers fLucN for / fLucC rev</FONT></LI><LI lang="en-US"><FONT face="Calibri, sans-serif">CHETAQFO (A.T. 51°C, ext time 1 min) → tubes 31 to 40 + C4, primers rLucN for / fLucC rev</FONT></LI></UL><P lang="en-US"><FONT face="Calibri, sans-serif">Rem: control tubes were prepared the same way (1uL from heat shocked tubes + 0.5 from each primers)</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">The PCR products were analyzed by gel electrophoresis (1.2% agarose gels. 2uL PCR reaction + 8 uL water + 2uL 6X loading dye. The ladder that was used was 1kb ladder)</FONT></P><P lang="en-US"> </P><P lang="en-US"> </P><H2 class="western" lang="en-US">5.2Results</H2><H3 class="western" lang="en-US">5.2.1Data</H3><P> </P><P lang="en-US"> </P><H3 class="western" lang="en-US">3.2.2Interpretation</H3><P lang="en-US"><FONT face="Calibri, sans-serif">For the construct with firefly luciferase, (fYNZC)none of the colonies had the right insert* with the expected size of 2887 bp (firefly). For the construct with renilla, lane 11 revealed a good colony with the right insert of the expected size of 2101 bp (renilla).</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">For the split luciferases, tubes 21, 24, 25, 26, 27, 28, 29 and 30 (split fLuc) have the good insert (1750 bp), as well as 31, 32, 33, 34, 35, 36, 38, 39 and 40 (split rLuc) (964 bp). </FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">We will send samples 26 and 33 for sequencing. We will pick other colonies in order to obtain a colony with the construct.</FONT></P><H1 class="western" lang="en-US">6 Experiment 6: Minipreps of negative controls</H1><P class="western" lang="en-US">Purpose of the experiment: Make minipreps of the negative controls (split alone)</P><H2 class="western" lang="en-US">6.1Material and methods</H2><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">Cf. Miniprep kit from Qiagen</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">Done on 5mL pelleted cultures from tubes 11, 21, 24, 26, 27, 28, 29, 31, 32, 33 34, 35, 36, 38, 39, 40 (in big centrifuge)</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">DNA concentration was determined using the Nanodrop Spectrophotometer.</FONT></FONT></P><H2 class="western" lang="en-US">6.2Results</H2><H3 class="western" lang="en-US">6.2.1Data</H3><TABLE class="table table-striped"><TBODY><TR valign="top"><TD width="26%"><P class="western" lang="fr-FR"><STRONG>Sample</STRONG></P></TD><TD width="28%"><P class="western" lang="fr-FR">Concentration [ng/uL]</P></TD><TD width="22%"><P class="western" lang="fr-FR">260/280</P></TD><TD width="24%"><P class="western" lang="fr-FR">260/230</P></TD></TR><TR valign="top"><TD width="26%"><P class="western" lang="fr-FR"><STRONG>11</STRONG></P></TD><TD width="28%"><P class="western" lang="fr-FR">320.3</P></TD><TD width="22%"><P class="western" lang="fr-FR">1.94</P></TD><TD width="24%"><P class="western" lang="fr-FR">2.12</P></TD></TR><TR valign="top"><TD width="26%"><P class="western" lang="fr-FR"><STRONG>21</STRONG></P></TD><TD width="28%"><P class="western" lang="fr-FR">390.1</P></TD><TD width="22%"><P class="western" lang="fr-FR">1.95</P></TD><TD width="24%"><P class="western" lang="fr-FR">2.14</P></TD></TR><TR valign="top"><TD width="26%"><P class="western" lang="fr-FR"><STRONG>24</STRONG></P></TD><TD width="28%"><P class="western" lang="fr-FR">482.4</P></TD><TD width="22%"><P class="western" lang="fr-FR">1.94</P></TD><TD width="24%"><P class="western" lang="fr-FR">2.23</P></TD></TR><TR valign="top"><TD width="26%"><P class="western" lang="fr-FR"><STRONG>26</STRONG></P></TD><TD width="28%"><P class="western" lang="fr-FR">494.1</P></TD><TD width="22%"><P class="western" lang="fr-FR">1.94</P></TD><TD width="24%"><P class="western" lang="fr-FR">2.27</P></TD></TR><TR valign="top"><TD width="26%"><P class="western" lang="fr-FR"><STRONG>27</STRONG></P></TD><TD width="28%"><P class="western" lang="fr-FR">795.4</P></TD><TD width="22%"><P class="western" lang="fr-FR">1.95</P></TD><TD width="24%"><P class="western" lang="fr-FR">2.3</P></TD></TR><TR valign="top"><TD width="26%"><P class="western" lang="fr-FR"><STRONG>28</STRONG></P></TD><TD width="28%"><P class="western" lang="fr-FR">578.1</P></TD><TD width="22%"><P class="western" lang="fr-FR">1.94</P></TD><TD width="24%"><P class="western" lang="fr-FR">2.23</P></TD></TR><TR valign="top"><TD width="26%"><P class="western" lang="fr-FR"><STRONG>29</STRONG></P></TD><TD width="28%"><P class="western" lang="fr-FR">381.9</P></TD><TD width="22%"><P class="western" lang="fr-FR">1.92</P></TD><TD width="24%"><P class="western" lang="fr-FR">2.16</P></TD></TR><TR valign="top"><TD width="26%"><P class="western" lang="fr-FR"><STRONG>31</STRONG></P></TD><TD width="28%"><P class="western" lang="fr-FR">543.8</P></TD><TD width="22%"><P class="western" lang="fr-FR">1.93</P></TD><TD width="24%"><P class="western" lang="fr-FR">2.13</P></TD></TR><TR valign="top"><TD width="26%"><P class="western" lang="fr-FR"><STRONG>32</STRONG></P></TD><TD width="28%"><P class="western" lang="fr-FR">401.6</P></TD><TD width="22%"><P class="western" lang="fr-FR">1.94</P></TD><TD width="24%"><P class="western" lang="fr-FR">2.27</P></TD></TR><TR valign="top"><TD width="26%"><P class="western" lang="fr-FR"><STRONG>33</STRONG></P></TD><TD width="28%"><P class="western" lang="fr-FR">442.3</P></TD><TD width="22%"><P class="western" lang="fr-FR">1.94</P></TD><TD width="24%"><P class="western" lang="fr-FR">2.3</P></TD></TR><TR valign="top"><TD width="26%"><P class="western" lang="fr-FR"><STRONG>34</STRONG></P></TD><TD width="28%"><P class="western" lang="fr-FR">429.2</P></TD><TD width="22%"><P class="western" lang="fr-FR">1.94</P></TD><TD width="24%"><P class="western" lang="fr-FR">2.31</P></TD></TR><TR valign="top"><TD width="26%"><P class="western" lang="fr-FR"><STRONG>35</STRONG></P></TD><TD width="28%"><P class="western" lang="fr-FR">436.1</P></TD><TD width="22%"><P class="western" lang="fr-FR">1.92</P></TD><TD width="24%"><P class="western" lang="fr-FR">2.17</P></TD></TR><TR valign="top"><TD width="26%"><P class="western" lang="fr-FR"><STRONG>36</STRONG></P></TD><TD width="28%"><P class="western" lang="fr-FR">753.2</P></TD><TD width="22%"><P class="western" lang="fr-FR">1.92</P></TD><TD width="24%"><P class="western" lang="fr-FR">2.24</P></TD></TR><TR valign="top"><TD width="26%"><P class="western" lang="fr-FR"><STRONG>38</STRONG></P></TD><TD width="28%"><P class="western" lang="fr-FR">230</P></TD><TD width="22%"><P class="western" lang="fr-FR">1.84</P></TD><TD width="24%"><P class="western" lang="fr-FR">1.96</P></TD></TR><TR valign="top"><TD width="26%"><P class="western" lang="fr-FR"><STRONG>39</STRONG></P></TD><TD width="28%"><P class="western" lang="fr-FR">525.7</P></TD><TD width="22%"><P class="western" lang="fr-FR">1.94</P></TD><TD width="24%"><P class="western" lang="fr-FR">2.19</P></TD></TR><TR valign="top"><TD width="26%"><P class="western" lang="fr-FR"><STRONG>40</STRONG></P></TD><TD width="28%"><P class="western" lang="fr-FR">271</P></TD><TD width="22%"><P class="western" lang="fr-FR">1.92</P></TD><TD width="24%"><P class="western" lang="fr-FR">1.89</P></TD></TR></TBODY></TABLE><P lang="en-US"> </P><H3 class="western" lang="en-US">6.2.2Interpretation</H3><P lang="en-US"><FONT face="Calibri, sans-serif">Concentration were very good.</FONT></P></DIV></DIV><DIV class="whitebg box"><H3 id="date-07-28-2014">Cpxr Overexpression</H3><SPAN>07-28-2014</SPAN><DIV class="notebook-content"><H1>1 Experiment 1: Gibson Assembly (GA) and PCR Colonies</H1><DIV class="notebook-content"><P>Assembly of GFP at the C ter or N ter of the Bba_1496001 cpxr under arabinose promoter biobrick by Gibson assembly.</P><P>The purpose of this colony PCR is to check if the insert is present and so determine if the cloning worked.</P><H2 class="western" lang="en-US">1.1 Material and methods</H2>
We did the GA with Bba_1496001 cpxr under arabinose promoter biobrick opened with corresponding primers and amplified insert. Our plasmid have a chloramphenicol resistance.</DIV><DIV class="notebook-content">We used the Gibson assembly kit (see protocol).</DIV><DIV class="notebook-content"> </DIV><DIV class="notebook-content">PCR colony, Taq polymerase 25µl reaction (see protocol) with the  primers VR and RF2 ( TM= 49°C)</DIV><DIV class="notebook-content"> </DIV><DIV class="notebook-content"><P align="left"><FONT face="Calibri, sans-serif"><FONT size="2"><U><FONT color="#000000">Thermocycling:</FONT></U><FONT color="#000000"> </FONT></FONT></FONT></P><TABLE class="table table-striped"><TBODY><TR valign="top"><TD width="33%"><P align="center"><FONT face="Calibri, sans-serif"><FONT size="2">Initial denaturation</FONT></FONT></P></TD><TD width="33%"><P align="center"><FONT face="Calibri, sans-serif"><FONT size="2">95°C</FONT></FONT></P></TD><TD width="33%"><P align="center"><FONT face="Calibri, sans-serif"><FONT size="2">30 seconds</FONT></FONT></P></TD></TR></TBODY><TBODY><TR valign="top"><TD width="33%"><P align="left"><FONT face="Calibri, sans-serif"><FONT size="2">30 cycles</FONT></FONT></P></TD><TD width="33%"><P align="center"><FONT face="Calibri, sans-serif"><FONT size="2">95°C</FONT></FONT></P></TD><TD width="33%"><P align="center"><FONT face="Calibri, sans-serif"><FONT size="2">15-30 seconds</FONT></FONT></P></TD></TR><TR valign="top"><TD width="33%"><P align="left"> </P></TD><TD width="33%"><P align="center"><FONT face="Calibri, sans-serif"><FONT size="2">45-68°C</FONT></FONT></P></TD><TD width="33%"><P align="center"><FONT face="Calibri, sans-serif"><FONT size="2">15-60 seconds </FONT></FONT></P></TD></TR><TR valign="top"><TD width="33%"><P align="left"> </P></TD><TD width="33%"><P align="center"><FONT face="Calibri, sans-serif"><FONT size="2">68°C</FONT></FONT></P></TD><TD width="33%"><P align="center"><FONT face="Calibri, sans-serif"><FONT size="2">1 min/kb</FONT></FONT></P></TD></TR></TBODY><TBODY><TR valign="top"><TD width="33%"><P align="center"><FONT face="Calibri, sans-serif"><FONT size="2">Final Extension</FONT></FONT></P></TD><TD width="33%"><P align="center"><FONT face="Calibri, sans-serif"><FONT size="2">68°C</FONT></FONT></P></TD><TD width="33%"><P align="center"><FONT face="Calibri, sans-serif"><FONT size="2">5 min</FONT></FONT></P></TD></TR></TBODY><TBODY><TR valign="top"><TD width="33%"><P align="center"><FONT face="Calibri, sans-serif"><FONT size="2">stop</FONT></FONT></P></TD><TD width="33%"><P align="center"><FONT face="Calibri, sans-serif"><FONT size="2">4°C</FONT></FONT></P></TD><TD width="33%"><P align="center"> </P></TD></TR></TBODY></TABLE><P align="left"> </P></DIV><DIV class="notebook-content"><H2 class="western" lang="en-US">1.2Results</H2><H3 class="western" lang="en-US">1.2.1Data</H3><P align="center"><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2"><U><STRONG>Gel of the PCR of the plasmid CpxR CGFP with VR and VF2 primers (gels 1 and 2)</STRONG></U></FONT></FONT></FONT></P><P> </P><P> </P><P><SPAN style="text-decoration: underline;"><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">Upper Gel :</FONT></FONT></FONT></SPAN></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">lane 1 : ladder</FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">lane 2 : plasmid of CpxR CGFP f rom the colonie 1</FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">lane 3 :plasmid of CpxR CGFP f rom the colonie 2</FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">lane 4 :plasmid of CpxR CGFP f rom the colonie 3</FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">lane 5 :plasmid of CpxR CGFP f rom the colonie 4</FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">lane 6 :plasmid of CpxR CGFP f rom the colonie 5</FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">lane 7 :plasmid containing only CpxR</FONT></FONT></FONT></P><P> </P><P><SPAN style="text-decoration: underline;"><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">Down Gel :</FONT></FONT></FONT></SPAN></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">lane 1 :ladder</FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">lane 2 :plasmid of CpxR CGFP f rom the colonie 6</FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">lane 3 :plasmid of CpxR CGFP f rom the colonie 7</FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">lane 4 :plasmid of CpxR CGFP f rom the colonie 8</FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">lane 5 :plasmid of CpxR CGFP f rom the colonie 9</FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">lane 6 :plasmid of CpxR CGFP f rom the colonie 10</FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">lane 7: plasmid containing only CpxR (negatif control)</FONT></FONT></FONT></P><P> </P><P> </P><P> </P><P> </P><P> </P><P align="center"><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2"><U><STRONG>Gel of the PCR of the plasmid CpxR NGFP with VR and VF2 primers ( gels 3 and 4)</STRONG></U></FONT></FONT></FONT></P><P> </P><P> </P><P><SPAN style="text-decoration: underline;"><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">Upper Gel :</FONT></FONT></FONT></SPAN></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">lane 1 : ladder</FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">lane 2 : plasmid of CpxR NGFP f rom the colonie 1</FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">lane 3 :plasmid of CpxR NGFP f rom the colonie 2</FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">lane 4 :plasmid of CpxR NGFP f rom the colonie 3</FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">lane 5 :plasmid of CpxR NGFP f rom the colonie 4</FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">lane 6 :plasmid of CpxR NGFP f rom the colonie 5</FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">lane 7 :plasmid containing only CpxR</FONT></FONT></FONT></P><P> </P><P><SPAN style="text-decoration: underline;"><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">Down Gel :</FONT></FONT></FONT></SPAN></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">lane 1 :ladder</FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">lane 2 :plasmid of CpxR NGFP f rom the colonie 6</FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">lane 3 :plasmid of CpxR NGFP f rom the colonie 7</FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">lane 4 :plasmid of CpxR NGFP f rom the colonie 8</FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">lane 5 :plasmid of CpxR NGFP f rom the colonie 9</FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">lane 6 :plasmid of CpxR NGFP f rom the colonie 10</FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">lane 7: plasmid containing only CpxR (negatif control)</FONT></FONT></FONT></P><P> </P><P> </P><P> </P><P><SPAN style="text-decoration: underline;"><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">Remarks:</FONT></FONT></FONT></SPAN></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">Gel 1, ladder 2 log</FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">Gel 2, 3, 4 1 kb ladder</FONT></FONT></FONT></P><P> </P><P lang="en-US"> </P><H3 class="western" lang="en-US">1.2.2Interpretation</H3><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">It seems to work for all the insert of the GFP in Cterminal</FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">but only for one (the plasmid from the colonie 10) for the insert of the GFP in Nterminal</FONT></FONT></FONT></P><P> </P><H1 class="western" lang="en-US">2 Experiment 2: Minprep the plamids containing the insert </H1><H1 class="western" lang="en-US"> </H1><H2 class="western" lang="en-US">2.1 Material and methods</H2><P>Minipreps protocol (see protocol) </P><P> </P><H2 class="western" lang="en-US">2.2Results</H2><H3 class="western" lang="en-US">2.2.1Data</H3><H1 class="western" lang="en-US"> </H1><P style="text-align: center;"><SPAN style="text-decoration: underline;"><STRONG>Nanodrops of the miniprep</STRONG></SPAN></P><P> </P><H3 class="western" lang="en-US">1.2.2Interpretation</H3><P> </P><P>Our Minipreps have a good concentration, we can now send them to sequencing.</P><P> </P><H1 class="western" lang="en-US">2 Experiment 3: Sequencing of CGFP and NGFP plamids</H1><H1 class="western" lang="en-US"> </H1><H2 class="western" lang="en-US">3.1 Material and methods</H2><P> </P><P>We sent to sequencing the sample: gfpC_3 ans gfpN_10.</P><P> </P><P><U>Primers used :</U></P><P> </P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">Nterm : </FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2"> pBad for </FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2"> cpxR for gen</FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2"> cpxR rv gen</FONT></FONT></FONT></P><P> </P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">Cterm :</FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2"> pBad for </FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2"> CGFP control fw </FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2"> VR</FONT></FONT></FONT></P><P> </P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">plasmid concentration: <STRONG>80 ng/µl</STRONG> ()</FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">primers concentration : <STRONG>2µM </STRONG>(diluer 10X les plasmids et prendre 0,6 µl)</FONT></FONT></FONT></P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">total volume : <STRONG>15µl</STRONG> (complete with  nuclease free water)</FONT></FONT></FONT></P><P> </P><H2 class="western" lang="en-US">3.2Results</H2><H3 class="western" lang="en-US">3.2.1Data</H3><P>The sequencing confimed the GA worked, the insert have the correct sequence without any mutation.</P><P> </P><H3 class="western" lang="en-US">3.2.2Interpretation</H3><P> </P><P>We wanted to confirm the expression of the GFP with a wet bench experiment :</P><P> </P><P><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">incubation overnight with 10^3 µM of Arabinose </FONT></FONT></FONT><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">(30 µL in </FONT></FONT></FONT>3mL of LB)</P><P>Conclusion: GFP is express.</P><P> </P></DIV></DIV></DIV><DIV class="whitebg box"><H3 id="date-08-03-2014">CheY CheZ - Week 5</H3><SPAN>08-03-2014</SPAN><DIV class="notebook-content"><P class="western" lang="en-US" align="left"> </P><H1 class="western" lang="en-US">1 Experiment 1: Colony PCR (cont.)</H1><P class="western" lang="en-US">Purpous: Retry a colony PCR to obtain the constructs of interest.</P><H2 class="western" lang="en-US">1.1Material and methods</H2><P lang="en-US"><FONT face="Calibri, sans-serif">We should pick colonies from the most efficient plate.</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"> 20 colonies were picked from plate 1B → tubes 1-20 </FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"> 10 colonies were picked from plate 2B → tubes 21-30</FONT></P><P> </P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif">Same protocol as the week before for Colony PCR</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">CHETAQON → tubes 1 to 20 + C1 (primers = CheY for/fLucC rev)</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">→ tubes +1 to +4 (primers = openers 1 &amp; 2)</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">CHETAQTW → tubes 21 to 30 + C3 (primers = CheY for/rLucC rev)</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">→ tubes +5, +6 (primers = openers 1 &amp; 2)</FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif">1.2% Agarose gels were used for analysing the samples. The ladder used was 1kb plus ladder →</FONT></P><H2 class="western" lang="en-US">1.2Results</H2><H3 class="western" lang="en-US">1.2.1Data</H3><P lang="en-US"> </P><H3 class="western" lang="en-US">1.2.2Interpretation</H3><P lang="en-US"><FONT face="Calibri, sans-serif">For the construct with Renilla (gel on the right), it seems that lane 26 is a sample containing the expected insert (2101 bp). Miniprep will be done on this tube.</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">We have to pick other colonies for the construct with Firefly luciferase, as none of the picked colonies seemed to contain the right insert.</FONT></P><H1 class="western" lang="en-US">2 Experiment 2: Sequencing</H1><P class="western" lang="en-US">Purpose of the experiment: To verify that the inserts don't have any mutations.</P><H2 class="western" lang="en-US">2.1Material and methods</H2><P lang="en-US"><FONT face="Calibri, sans-serif">Sequencing was done on tubes 26 for fLucNfLucC and 33 for rLucNrLucC</FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif">FlucNFlucC:</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">Primer pBAD for → 736 bp</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">SEQUENCING_Fluc_Middle → 619 bp</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">FlucC_pSB1C3_Forward_PCR_SN → 456 bp</FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif">RlucNRlucC:</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">primer pBAD for → 458 bp</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">rLucC_pBAD33_forward_PCR_SN → 583 bp</FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif">Sequencing on tube 26 for rYNZC</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">Primer pBAD For → 562 bp</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">rLucN_pBAD33_forward_PCR_SN → 327 bp</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">Ec_CheZ_pSB1C3_forward_PCR_SN → 658 bp</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">rLucC_pBAD33_forward_PCR_SN → 583 bp</FONT></P><H2 class="western" lang="en-US">2.2Results</H2><H3 class="western" lang="en-US">2.2.1Data</H3><P>[SEQUENCING RESULTS]</P><H3 class="western" lang="en-US">2.2.2Interpretation</H3><P lang="en-US"> </P><P lang="en-US"> </P><H1 class="western" lang="en-US">3Experiment 3: Colony PCR (cont.)</H1><P class="western" lang="en-US">Purpose of the experiment: Find a colony that has the right insert for Firefly construct</P><H2 class="western" lang="en-US">3.1Material and methods</H2><P lang="en-US"><FONT face="Calibri, sans-serif">PCR on firefly construct</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">20 colonies were picked from plate with firefly construct → tubes 1 to 20</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">Same protocol as the week before for Colony PCR</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"> primers: CheY_for with FlucN_rev </FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">CHETAQA , annealing temperature 52°C</FONT></P><H2 class="western" lang="en-US">3.2Results</H2><P lang="en-US"> </P><H3 class="western" lang="en-US">3.2.1Data</H3><P> </P><H3 class="western" lang="en-US">3.2.2Interpretation</H3><P lang="en-US"><FONT face="Calibri"><FONT size="2">The expected size of 2887 seems to appear on lane 17: this sample should contain the right insert with firefly luciferase construct. The other ones are not valid.</FONT></FONT></P></DIV></DIV><DIV class="whitebg box"><H3>Split IFP &amp; GFP - Week 6</H3><SPAN>08-03-2014</SPAN><DIV class="notebook-content"><H1>1        Gibson assembly and Colony PCR of GA3 (IFP1 at the N terminal of CPXR and IFP2 at the C terminal of CPXR)</H1><P>Last time we tried we obtained only colonies containing either only IFP1-CPXR either IFP2-CPXR.</P><H2>1.1       Material and methods</H2><P>Gibson assembly kit and Taq polymerase (see protocols)</P><H2>1.2       Results</H2><H3>1.2.1     Data</H3><H3>1.2.2     Interpretation</H3><P>The ladder doesn’t allow us to see precisely the size of the plasmid. Moreover, the colony PCR didn’t work very well as we obtained a lot of unspecific bands. Mini-prep will be necessary to analyse our Gibson assembly product.</P><P> </P><H1>2     Restriction analysis of GA3</H1><P>Digestion of our Gibson assembly product GA3 (IPF1 at the N terminal of CPXR and IFP2 at the C terminal of CPXR,) to see which plasmids contain the expected insert.</P><H2>2.1       Material and methods</H2><P>Restriction digestion with NheI and SpeI (see protocols)</P><H2>2.2       Results</H2><H3>2.2.1     Data</H3><H3>2.2.2     Interpretation</H3><P>Expectation with both fragments: 3273bp + 2535 bp: Colonies 9 and 10</P><P>Expectation with only IFP1-CPXR: 3273 bp + 1110 bp: Colonies 4 and 8</P><P>Expectation with only IFP2-CPXR: 3273 bp + 1394 bp: Colonies 1, 2, 3, 5, 6, 7</P><P> </P><P>We obtained two plasmids that seem to contain IFP1-CPXR (N terminal) and IFP2-CPXR (C terminal)</P><H1>3     GA2, GA3 and GA4 sequencing</H1><P>Restriction analyses of these 3 plasmids are thought to contain the expected fragments, but sequencing is necessary to see if they contain any mutations</P><H2>3.1       Material and methods</H2><P>Microsynth barcodes</P><H2>3.2       Results</H2><H3>3.2.1     Data</H3><P><STRONG>GA2: </STRONG>no mutation</P><P><STRONG>GA3: </STRONG>no mutation</P><P><STRONG>GA4: </STRONG>no mutation (one deletion in the promoter, not important)</P><H3>3.2.2    Interpretation </H3><P>We obtained our expected products for GA2, GA3, GA4</P><P> </P><P> </P></DIV></DIV><DIV class="whitebg box"><H3 id="date-08-10-2014">CheY CheZ - Week 6</H3><SPAN>08-10-2014</SPAN><DIV class="notebook-content"><P class="western" lang="en-US" align="left"> </P><H1 class="western" lang="en-US">1Experiment 1: Verification PCR</H1><P class="western" lang="en-US">Purpous: Verification in the constructs fYNZC and rYNZC to verify that the right inserts were present</P><H2 class="western" lang="en-US">1.1Material and methods</H2><P lang="en-US"><FONT face="Calibri, sans-serif">Done on minipreps of tubes 17 (fYNZC) and 26 (rYNZC)</FONT></P><H2 class="western" lang="en-US">1.2Results</H2><P> </P><H3 class="western" lang="en-US">1.2.1Data</H3><P> </P><H3 class="western" lang="en-US">1.2.2Interpretation</H3><P lang="en-US"><FONT face="Calibri, sans-serif">The expected fragments were obtained for rYNZC construct. As the first gel wasn't clear for fYNZC, we did an other one, which gave fragments that are all explained and expected.</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">We see different bands. The top one in all lanes is the whole plasmid. For lane 7, fLucN_rev primer also binds in fLucC → 2000bp instead of 1245. Lane 8: fLucC_for primer also binds to the end of fLucN → 1252 bp instead of the expected 474. Lane 4: we only obtain the “opened” primers in addition to the initial primer.</FONT></P><P lang="en-US"> </P><H1 class="western" lang="en-US">2Experiment 2: Restriction analysis</H1><P class="western" lang="en-US">Purpose of the experiment: Restriction enzyme digestion of rYNZC and fYNZC in order to confirm the presence of our constructs in each plasmid, using enzymes NheI and SpeI</P><H2 class="western" lang="en-US">2.1Material and methods</H2><P lang="en-US"><FONT face="Calibri, sans-serif">Restriction digest:</FONT></P><TABLE class="table table-striped"><TBODY><TR valign="top"><TD width="153"><P class="western" lang="fr-FR" style="text-align: center;" align="right"><EM>uL</EM></P></TD><TD width="110"><P class="western" lang="fr-FR">FYNZC – sample 17</P></TD><TD width="108"><P class="western" lang="fr-FR">RYNZC – sample 26</P></TD></TR><TR valign="top"><TD width="153"><P class="western" lang="fr-FR">Template DNA (500 ng)</P></TD><TD width="110"><P class="western" lang="fr-FR" align="right">1.47</P></TD><TD width="108"><P class="western" lang="fr-FR" align="right">1.59</P></TD></TR><TR valign="top"><TD width="153"><P class="western" lang="fr-FR">NheI</P></TD><TD width="110"><P class="western" lang="fr-FR" align="right">0.5</P></TD><TD width="108"><P class="western" lang="fr-FR" align="right">0.5</P></TD></TR><TR valign="top"><TD width="153"><P class="western" lang="fr-FR">SpeI</P></TD><TD width="110"><P class="western" lang="fr-FR" align="right">0.5</P></TD><TD width="108"><P class="western" lang="fr-FR" align="right">0.5</P></TD></TR><TR valign="top"><TD width="153"><P class="western" lang="fr-FR">Smart Cut Buffer</P></TD><TD width="110"><P class="western" lang="fr-FR" align="right">5</P></TD><TD width="108"><P class="western" lang="fr-FR" align="right">5</P></TD></TR><TR valign="top"><TD width="153"><P class="western" lang="fr-FR">Water</P></TD><TD width="110"><P class="western" lang="fr-FR" align="right">42.53</P></TD><TD width="108"><P class="western" lang="fr-FR" align="right">42.41</P></TD></TR><TR valign="top"><TD width="153"><P class="western" lang="fr-FR">Total reaction volume</P></TD><TD width="110"><P class="western" lang="fr-FR" align="right">50</P></TD><TD width="108"><P class="western" lang="fr-FR" align="right">50</P></TD></TR></TBODY></TABLE><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif">Samples were heated for 60 minutes at 37°C before being run on a gel.</FONT></P><H2 class="western" lang="en-US">2.2Results</H2><P> </P><H3 class="western" lang="en-US">2.2.1Data</H3><P> </P><P lang="en-US"> </P><H3 class="western" lang="en-US">2.2.2Interpretation</H3><P lang="en-US"><FONT face="Arial, sans-serif"><FONT face="Calibri, sans-serif">The expected sizes of fragments are observed. </FONT></FONT></P><H1 class="western" lang="en-US">Conclusion</H1><P lang="en-US"><FONT face="Calibri">The plasmids can be sent for sequencing.</FONT></P><P lang="en-US"> </P><P class="western" lang="en-US" align="left"> </P><H1 class="western" lang="en-US">3 Experiment 3: Bioluminescence assay 3</H1><P class="western" lang="en-US">Purpous: We want to check if our constructs work.</P><H2 class="western" lang="en-US">3.1Material and methods</H2><P lang="en-US"><FONT face="Calibri, sans-serif">Overnight cultures were prepared from glycerol stocks of fYNZC (tube 17), rYNZC (tube 26), fLucNfLucC (tube 26-), rLucNrLucC (tube 31-), fLuc (EPIC Firefly) and rLuc (rLuc from Waldor's lab) into 5mL LB-Chloramphenicol.</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">50mL of overnight cultures were transferred into 5mL fresh chloramphenicol containing 1mM arabinose. These sample were incubated for 4 hours at 37°C, 180rpm shaking, to an OD ~1.</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">Cultures were centrifuged, medium was discarded, cells were washed once with PBS.</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">The substrate (coelenterazine (clz) for renilla Luciferase and d-luciferin (d-luc) for firefly Luciferase) were prepared in aliquots of 100uM and were added at 4 different concentrations (7.5uM, 15uM, 22,5uM, 30uM).</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">A white opaque 96-Well plate was loaded as following. Samples were loaded in triplicate, in a total volume of 200uL contaning 140uL of culture, ajusting the OD to 0.7:</FONT></P><P lang="en-US"> </P><TABLE class="table table-striped"><TBODY><TR valign="top"><TD width="20%"><P class="western" lang="fr-FR">uL</P></TD><TD width="20%"><P class="western" lang="fr-FR">Clz</P></TD><TD width="20%"><P class="western" lang="fr-FR">D-luc</P></TD><TD width="20%"><P class="western" lang="fr-FR">Culture (OD 1)</P></TD><TD width="20%"><P class="western" lang="fr-FR">PBS</P></TD></TR><TR valign="top"><TD width="20%"><P class="western" lang="fr-FR">A/C/E 1-3</P></TD><TD width="20%"><P class="western" lang="fr-FR" align="right">15</P></TD><TD bgcolor="#cccccc" width="20%"><P class="western" lang="fr-FR" align="right"> </P></TD><TD width="20%"><P class="western" lang="fr-FR" align="right">140</P></TD><TD width="20%"><P class="western" lang="fr-FR" align="right">45</P></TD></TR><TR valign="top"><TD width="20%"><P class="western" lang="fr-FR">A/C/E 4-6</P></TD><TD width="20%"><P class="western" lang="fr-FR" align="right">30</P></TD><TD bgcolor="#cccccc" width="20%"><P class="western" lang="fr-FR" align="right"> </P></TD><TD width="20%"><P class="western" lang="fr-FR" align="right">140</P></TD><TD width="20%"><P class="western" lang="fr-FR" align="right">30</P></TD></TR><TR valign="top"><TD width="20%"><P class="western" lang="fr-FR">A/C/E 7-9</P></TD><TD bgcolor="#cccccc" width="20%"><P class="western" lang="fr-FR" align="right"> </P></TD><TD width="20%"><P class="western" lang="fr-FR" align="right">15</P></TD><TD width="20%"><P class="western" lang="fr-FR" align="right">140</P></TD><TD width="20%"><P class="western" lang="fr-FR" align="right">45</P></TD></TR><TR valign="top"><TD width="20%"><P class="western" lang="fr-FR">A/C/E 10-12</P></TD><TD bgcolor="#cccccc" width="20%"><P class="western" lang="fr-FR" align="right"> </P></TD><TD width="20%"><P class="western" lang="fr-FR" align="right">30</P></TD><TD width="20%"><P class="western" lang="fr-FR" align="right">140</P></TD><TD width="20%"><P class="western" lang="fr-FR" align="right">30</P></TD></TR><TR valign="top"><TD width="20%"><P class="western" lang="fr-FR">B/D/F 1-3</P></TD><TD width="20%"><P class="western" lang="fr-FR" align="right">45</P></TD><TD bgcolor="#cccccc" width="20%"><P class="western" lang="fr-FR" align="right"> </P></TD><TD width="20%"><P class="western" lang="fr-FR" align="right">140</P></TD><TD width="20%"><P class="western" lang="fr-FR" align="right">15</P></TD></TR><TR valign="top"><TD width="20%"><P class="western" lang="fr-FR">B/D/F 4-6</P></TD><TD width="20%"><P class="western" lang="fr-FR" align="right">60</P></TD><TD bgcolor="#cccccc" width="20%"><P class="western" lang="fr-FR" align="right"> </P></TD><TD width="20%"><P class="western" lang="fr-FR" align="right">140</P></TD><TD width="20%"><P class="western" lang="fr-FR" align="right">0</P></TD></TR><TR valign="top"><TD width="20%"><P class="western" lang="fr-FR">B/D/F 7-9</P></TD><TD bgcolor="#cccccc" width="20%"><P class="western" lang="fr-FR" align="right"> </P></TD><TD width="20%"><P class="western" lang="fr-FR" align="right">45</P></TD><TD width="20%"><P class="western" lang="fr-FR" align="right">140</P></TD><TD width="20%"><P class="western" lang="fr-FR" align="right">15</P></TD></TR><TR valign="top"><TD width="20%"><P class="western" lang="fr-FR">B/D/F 10-12</P></TD><TD bgcolor="#cccccc" width="20%"><P class="western" lang="fr-FR" align="right"> </P></TD><TD width="20%"><P class="western" lang="fr-FR" align="right">60</P></TD><TD width="20%"><P class="western" lang="fr-FR" align="right">140</P></TD><TD width="20%"><P class="western" lang="fr-FR" align="right">0</P></TD></TR></TBODY></TABLE><P lang="en-US"><FONT face="Calibri, sans-serif">Substrates for luciferins were added at the end.</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">After ~10 minutes RT incubation, the plate was read by a plate reader (1CPS) every 4 minutes.</FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif">Lanes A &amp; B: constructs</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">Lanes C &amp; E: negative controls (split luciferases alone)</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">Lanes D &amp; F: positive controls (luciferases)</FONT></P><P lang="en-US"> </P><H2 class="western" lang="en-US">3.2Results</H2><H3 class="western" lang="en-US">3.2.1Data</H3><P lang="en-US"><FONT face="Arial, sans-serif">Figure 1: Graph of the Luminescence of the constructs with Renilla Luciferase and of the corresponding negative controls (each with 4 different substrate concentration) in function of the time. RYNZC stands for the construct EcCheY_rLucN_EcCheZ_rLucC, and RlucNrLucC stands for the split luciferase alone. “clz” is for coelenterazine.</FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif">Figure 2: Graph of the Luminescence of the constructs with Firefly Luciferase and of the corresponding negative controls (each with 4 different substrate concentration) in function of the time. FYNZC stands for the construct EcCheY_fLucN_EcCheZ_fLucC, and flucNfLucC stands for the split luciferase alone. “luc” is for luciferin</FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif">Figure 3: Graph of the luminescence of the positive and negative controls for Renilla Luciferase (each with 4 different substrate concentration) in function of the time. </FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif">Figure 4: Graph of the luminescence of the positive and negative controls for Firefly Luciferase (each with 4 different substrate concentration) in function of the time. </FONT></P><P lang="en-US"><FONT face="Calibri"> </FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">Figure 5: Graph of the Luminescence of the constructs with Renilla Luciferase (rYNZC) and Firefly Luciferase (fYNZC) in function of the different concentrations of substrate (coelenterazine for rLuc and luciferin for fLuc). </FONT></P><P lang="en-US"><FONT face="Calibri"> </FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">Figure 6: Graph of the Luminescence of the positive and negative controls with Renilla Luciferase and Firefly Luciferase in function of the different concentrations of substrate (coelenterazine for rLuc and luciferin for fLuc). </FONT></P><P lang="en-US"> </P><H3 class="western" lang="en-US">3.2.2Interpretation</H3><P lang="en-US"><FONT face="Calibri, sans-serif">As the signal detected in the plates containing our constructs was lower than the negative control, we conclude that our construct didn't work. </FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">The problem might come from the fact that as the dimerization of CheY and CheZ should only happen in the absence of chemoattractants, it has not worked because of the presence of an attractant. This could have happened, as arabinose was used to induce the gene, and as the cells were washed only once with PBS. The next experiment should be done after more wash steps, to get rid of the arabinose.</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">However, the positive and negative controls of our experiment gave good results. The gap between the positive and negative controls is quite big, and it seems that the substrate concentration affects the activity of both luciferases.</FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif">We see that firefly luciferase seems to be better than renilla, because the gap between + and – controls is bigger and because it seems to be more sensitive to a substrate concentration gradient. We should consider these results for the choice of the Luciferase we will use for the rest of the project.</FONT></P><P lang="en-US"> </P><P class="western" lang="en-US" align="left"> </P><H1 class="western" lang="en-US">4 Experiment 4: Bioluminescence assay 4</H1><P class="western" lang="en-US">Purpous: To test our constructs by measuring their relative emitted bioluminescence.</P><H2 class="western" lang="en-US">4.1Material and methods</H2><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">Overnight cultures were prepared from glycerol stocks of unstransformed DH5 alpha, fYNZC (tube 17), rYNZC (tube 26), fLucNfLucC (tube 26-), rLucNrLucC (tube 31-), fLuc (EPIC Firefly) and rLuc (rLuc from Waldor's lab) into 5mL LB-Chloramphenicol.</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">50mL of overnight cultures were transferred into 5mL fresh chloramphenicol containing 1mM arabinose. These sample were incubated for 4 hours at 37°C, 180rpm shaking, to an OD ~1.</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">Cultures were centrifuged, medium was discarded, cells were washed three times with PBS.</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">The substrate (coelenterazine (clz) for renilla Luciferase and d-luciferin (d-luc) for firefly Luciferase) were prepared in aliquots of 100uM and added at 30 uM (d-luc) and 22.5uM (clz).</FONT></FONT></P><P lang="en-US"><FONT face="Calibri"> </FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">A white opaque 96-Well plate was loaded as following. Samples were loaded in triplicate, in a total volume of 180uL contaning 120uL of culture, ajusting the OD to 0.7:</FONT></FONT></P><P lang="en-US"> </P><H2 class="western" lang="en-US">4.2Results</H2><P lang="en-US"><FONT face="Calibri"> </FONT></P><H3 class="western" lang="en-US">4.2.1Data</H3><P lang="en-US"><FONT face="Calibri"> </FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2"><STRONG>Figure 1</STRONG>: (A) Graph of bioluminescence of different cultures with 22.5uM d-luciferin. F-PBS = PBS, DH5-alpha = untransformed competent cells, split fLuc = split Firefly Luciferase, fLuc = Firefly Luciferase, fYNZC = construct with CheY-fLucN-CheZ-fLucC. (B) Graph of bioluminescence of different cultures with 30mM coelenterazine. R-PBS = PBS, DH5-alpha = untransformed competent cells, split rLuc = split Renilla Luciferase, rLuc = Renilla Luciferase, fYNZC = construct with CheY-fLucN-CheZ-fLucC.</FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">Figure 2: (A) Graph of bioluminescence of different cultures, with 22.5uM d-luciferin, without positive control. Same legend as 1A. (B) Graph of bioluminescence of different cultures, with 30uM coelenterazine, without positive control. Same legend as 1B.</FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">Figure 3: Comparison of bioluminescence of the two constructs, with Firefly Luciferase (with 22.5uM d-luciferin) and Renilla Luciferase (with 30uM coelenterazine).</FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">Figure 4: Comparison of bioluminescence of the positive and negative controls with Firefly Luciferase and Renilla Luciferase </FONT></FONT></P><P lang="en-US"> </P><H3 class="western" lang="en-US">4.2.2Interpretation</H3><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">The obtained values are as expected for the controls, compared to the blanks with PBS and unstransformed competent cells. The values of our constructs being a bit lower than the negative controls, these results are not as good as expected. However, there is a good hope that our constructs can work, as their signals are higher than the blanks.</FONT></FONT></P><H1 class="western" lang="en-US">Conclusion</H1><P lang="en-US"><FONT face="Calibri"><FONT size="2">We have to try again with Waldor construct (pYNZC, with proteins from Vibrio Cholerae). We will do more steps of wash with PBS.</FONT></FONT></P></DIV></DIV><DIV class="whitebg box"><H3>Split IFP &amp; GFP - Week 7</H3><SPAN>08-10-2014</SPAN><DIV class="notebook-content"><H1>1        Gibson assembly and Colony PCR of GA1</H1><P>To fuse CPXR-IFP1 and CPXR.IFP2 and analyse the products by colony PCR</P><H2>1.1       Material and methods</H2><P>Gibson assembly kit and Taq polymerase (see protocol)</P><H2>1.2       Results</H2><H3>1.2.1     Data</H3><H3>1.2.2     Interpretation</H3><P>None of our colony contains our expected product. Colonies 1, 3, 5, 6, 8 only contain IFP2-CPXR, and the other ones only contain IFP1-CPXR, which are our PCR templates. We should have done our PCR with a lower amount of DNA template. It results that our Gibson assembly product contains a lot of DNA template, decreasing the probability to have bacteria transformed by our expected product. We will do the PCR again with a lower amount of DNA template. (Gibson assembly kit recommends a DNA template of 0.1-0.5 ng and we used 10 ng)</P><H1>2     Gibson assembly and of GA1: 2<SUP>nd</SUP> try</H1><P>We tried again the Gibson assembly once we did the PCR again with a lower amount of DNA template.</P><H2>2.1       Material and methods</H2><P>Gibson assembly kit and Taq polymerase (see protocol)</P><H2>2.2       Results</H2><H3>2.2.1     Data</H3><H3>2.2.2     Interpretation</H3><P>It seems that all our colonies contain the expected fragement. Restriction analysis will confirm it</P><H1>3     Restriction analysis of GA1</H1><P>To see if our GA1 product contains the expected fragment</P><H2>3.1       Material and methods</H2><P>NheI and SpeI restriction enzymes. (See protocol)</P><H2>3.2       Results</H2><H3>3.2.1     Data</H3><H3>3.2.2     Interpretation</H3><P> </P><P> We finally obtain our GA1 product (IFP1 and IFP2 at the C terminal of CPXR) in all the colonies (except the 11th). Sequencing will confirm it.</P><H1>4     GA1 sequencing</H1><P>To see if our GA1 product contains any mutation</P><H2>4.1       Material and methods</H2><P>Microsynth barcodes</P><H2>4.2       Results</H2><H3>4.2.1     Data</H3><P>No mutation occured.</P></DIV></DIV><DIV class="whitebg box"><H3 id="date-08-12-2014">Arabinose promoter characterization</H3><SPAN>08-12-2014</SPAN><DIV class="notebook-content"><H1>Experiment 1: Arabinose promoter characterization</H1><H2>1.1       Material and methods</H2><P>Cells with sfGFP CpxR &amp; CpxR sfGFP constructs were cultured in different concentrations of arabinose to check the functionality of the promoter. The concentration of arabinose per sample is indicated in Table 1.</P><TABLE class="table table-striped"><TBODY><TR><TD width="53"><P> Sample</P></TD><TD width="54"><P>uM Ara</P></TD><TD width="54"><P>uL Ara</P></TD><TD width="54"><P>ml LB-chl</P></TD><TD colspan="2" width="53"><P>tube 96-w       plate</P></TD></TR><TR><TD width="53"><P>A</P></TD><TD width="54"><P>0</P></TD><TD width="54"><P>0</P></TD><TD width="54"><P>5</P></TD><TD width="26"><P>A</P></TD><TD width="26"><P>1</P></TD></TR><TR><TD width="53"><P>B</P></TD><TD width="54"><P>5</P></TD><TD width="54"><P>0.025</P></TD><TD width="54"><P>5</P></TD><TD width="26"><P>B</P></TD><TD width="26"><P>2</P></TD></TR><TR><TD width="53"><P>C</P></TD><TD width="54"><P>10</P></TD><TD width="54"><P>0.05</P></TD><TD width="54"><P>5</P></TD><TD width="26"><P>C</P></TD><TD width="26"><P>3</P></TD></TR><TR><TD width="53"><P>D</P></TD><TD width="54"><P>50</P></TD><TD width="54"><P>0.25</P></TD><TD width="54"><P>4.9998</P></TD><TD width="26"><P>D</P></TD><TD width="26"><P>4</P></TD></TR><TR><TD width="53"><P>E</P></TD><TD width="54"><P>100</P></TD><TD width="54"><P>0.5</P></TD><TD width="54"><P>4.9995</P></TD><TD width="26"><P>E</P></TD><TD width="26"><P>5</P></TD></TR><TR><TD width="53"><P>F</P></TD><TD width="54"><P>500</P></TD><TD width="54"><P>2.5</P></TD><TD width="54"><P>4.9975</P></TD><TD width="26"><P>F</P></TD><TD width="26"><P>6</P></TD></TR><TR><TD width="53"><P>G</P></TD><TD width="54"><P>1000</P></TD><TD width="54"><P>5</P></TD><TD width="54"><P>4.995</P></TD><TD width="26"><P>G</P></TD><TD width="26"><P>7</P></TD></TR><TR><TD width="53"><P>H</P></TD><TD width="54"><P>5000</P></TD><TD width="54"><P>25</P></TD><TD width="54"><P>4.975</P></TD><TD width="26"><P>H</P></TD><TD width="26"><P>8</P></TD></TR><TR><TD width="53"><P>I</P></TD><TD width="54"><P>10 000</P></TD><TD width="54"><P>50</P></TD><TD width="54"><P>4.95</P></TD><TD width="26"><P>I</P></TD><TD width="26"><P>9</P></TD></TR><TR><TD width="53"><P>J</P></TD><TD width="54"><P>50 000</P></TD><TD width="54"><P>250</P></TD><TD width="54"><P>4.75</P></TD><TD width="26"><P>J</P></TD><TD width="26"><P>10</P></TD></TR><TR><TD width="53"><P>K</P></TD><TD width="54"><P>100 000</P></TD><TD width="54"><P>500</P></TD><TD width="54"><P>4.5</P></TD><TD width="26"><P>K</P></TD><TD width="26"><P>11</P></TD></TR><TR><TD width="53"><P>L</P></TD><TD width="54"><P>500 000</P></TD><TD width="54"><P>2500</P></TD><TD width="54"><P>2.5</P></TD><TD width="26"><P>L</P></TD><TD width="26"><P>12</P></TD></TR><TR><TD width="53"><P>+</P></TD><TD width="54"><P>0</P></TD><TD width="54"><P>0</P></TD><TD width="54"><P>5</P></TD><TD width="26"><P>+</P></TD><TD width="26"><P> </P></TD></TR><TR><TD width="53"><P>-</P></TD><TD width="54"><P>5000</P></TD><TD width="54"><P>25</P></TD><TD width="54"><P>4.975</P></TD><TD width="26"><P>-</P></TD><TD width="26"><P> </P></TD></TR></TBODY></TABLE><P>Table 1 - This table shows the concentration of Arabinose in each sample, as well as the quantity of medium and of Arabinose needed to reach it.</P><P>The negative control was the medium without cells, and the positive control was a sample of cells expressing sfGFP contitutively.</P><P> </P><P><STRONG>Procedure</STRONG></P><P>Cells from glycerol stocks were grown in 3 mL LB selective medium (chloramphenicol) during 3h (37 ºC &amp; shaking).</P><P>50 uL of cells were resuspended in a total volume of 5 mL of LB-chloramphenicol and Arabinose and incubated 5h at 37 ºC with shaking until an OD of ~1 was reached.The samples were then washed twice with PBS to a final OD of 0.4 - 0.5.</P><P> </P><P>Table 2 shows the quantity of PBS added to each sample (depending on the OD), as well as the OD after both PBS washes. This information is used to normalize the plate reader results.</P><TABLE class="table table-striped"><TBODY><TR><TD colspan="2" width="53"><P>tube 96-w        plate</P></TD><TD width="58"><P>OD N ter</P></TD><TD width="58"><P>PBS N</P></TD><TD width="58"><P>OD PBS</P></TD><TD width="58"><P>OD C ter</P></TD><TD width="58"><P>PBS C</P></TD><TD width="58"><P>OD PBS</P></TD><TD colspan="2" width="51"><P>tube 96-w        plate</P></TD></TR><TR><TD width="27"><P>A</P></TD><TD width="27"><P>1</P></TD><TD width="58"><P>1.136</P></TD><TD width="58"><P>10.5</P></TD><TD width="58"><P>0.281</P></TD><TD width="58"><P>1.224</P></TD><TD width="58"><P>11</P></TD><TD width="58"><P>0.369</P></TD><TD width="25"><P>A</P></TD><TD width="26"><P>1</P></TD></TR><TR><TD width="27"><P>B</P></TD><TD width="27"><P>2</P></TD><TD width="58"><P>1.071</P></TD><TD width="58"><P>10</P></TD><TD width="58"><P>0.402</P></TD><TD width="58"><P>1.061</P></TD><TD width="58"><P>10</P></TD><TD width="58"><P>0.384</P></TD><TD width="25"><P>B</P></TD><TD width="26"><P>2</P></TD></TR><TR><TD width="27"><P>C</P></TD><TD width="27"><P>3</P></TD><TD width="58"><P>1.014</P></TD><TD width="58"><P>10</P></TD><TD width="58"><P>0.360</P></TD><TD width="58"><P>1.082</P></TD><TD width="58"><P>10</P></TD><TD width="58"><P>0.332</P></TD><TD width="25"><P>C</P></TD><TD width="26"><P>3</P></TD></TR><TR><TD width="27"><P>D</P></TD><TD width="27"><P>4</P></TD><TD width="58"><P>1.098</P></TD><TD width="58"><P>10</P></TD><TD width="58"><P>0.342</P></TD><TD width="58"><P>1.068</P></TD><TD width="58"><P>10</P></TD><TD width="58"><P>0.393</P></TD><TD width="25"><P>D</P></TD><TD width="26"><P>4</P></TD></TR><TR><TD width="27"><P>E</P></TD><TD width="27"><P>5</P></TD><TD width="58"><P>1.016</P></TD><TD width="58"><P>10</P></TD><TD width="58"><P>0.346</P></TD><TD width="58"><P>1.07</P></TD><TD width="58"><P>10</P></TD><TD width="58"><P>0.338</P></TD><TD width="25"><P>E</P></TD><TD width="26"><P>5</P></TD></TR><TR><TD width="27"><P>F</P></TD><TD width="27"><P>6</P></TD><TD width="58"><P>1.038</P></TD><TD width="58"><P>10</P></TD><TD width="58"><P>0.349</P></TD><TD width="58"><P>1.06</P></TD><TD width="58"><P>10</P></TD><TD width="58"><P>0.351</P></TD><TD width="25"><P>F</P></TD><TD width="26"><P>6</P></TD></TR><TR><TD width="27"><P>G</P></TD><TD width="27"><P>7</P></TD><TD width="58"><P>0.912</P></TD><TD width="58"><P>9.5</P></TD><TD width="58"><P>0.336</P></TD><TD width="58"><P>1.06</P></TD><TD width="58"><P>10</P></TD><TD width="58"><P>0.314</P></TD><TD width="25"><P>G</P></TD><TD width="26"><P>7</P></TD></TR><TR><TD width="27"><P>H</P></TD><TD width="27"><P>8</P></TD><TD width="58"><P>0.686</P></TD><TD width="58"><P>7</P></TD><TD width="58"><P>0.316</P></TD><TD width="58"><P>1.028</P></TD><TD width="58"><P>10</P></TD><TD width="58"><P>0.307</P></TD><TD width="25"><P>H</P></TD><TD width="26"><P>8</P></TD></TR><TR><TD width="27"><P>I</P></TD><TD width="27"><P>9</P></TD><TD width="58"><P>0.645</P></TD><TD width="58"><P>7</P></TD><TD width="58"><P>0.314</P></TD><TD width="58"><P>0.999</P></TD><TD width="58"><P>10</P></TD><TD width="58"><P>0.305</P></TD><TD width="25"><P>I</P></TD><TD width="26"><P>9</P></TD></TR><TR><TD width="27"><P>J</P></TD><TD width="27"><P>10</P></TD><TD width="58"><P>0.52</P></TD><TD width="58"><P>6</P></TD><TD width="58"><P>0.305</P></TD><TD width="58"><P>0.918</P></TD><TD width="58"><P>9.5</P></TD><TD width="58"><P>0.281</P></TD><TD width="25"><P>J</P></TD><TD width="26"><P>10</P></TD></TR><TR><TD width="27"><P>K</P></TD><TD width="27"><P>11</P></TD><TD width="58"><P>0.449</P></TD><TD width="58"><P>5</P></TD><TD width="58"><P>0.281</P></TD><TD width="58"><P>0.822</P></TD><TD width="58"><P>9</P></TD><TD width="58"><P>0.285</P></TD><TD width="25"><P>K</P></TD><TD width="26"><P>11</P></TD></TR><TR><TD width="27"><P>L</P></TD><TD width="27"><P>12</P></TD><TD width="58"><P>0.315</P></TD><TD width="58"><P>4</P></TD><TD width="58"><P>0.296</P></TD><TD width="58"><P>0.505</P></TD><TD width="58"><P>6</P></TD><TD width="58"><P>0.248</P></TD><TD width="25"><P>L</P></TD><TD width="26"><P>12</P></TD></TR><TR><TD width="27"><P>+</P></TD><TD width="27"><P> </P></TD><TD width="58"><P>1.117</P></TD><TD width="58"><P>10</P></TD><TD width="58"><P> 0.283</P></TD><TD width="58"><P>-</P></TD><TD width="58"><P>-</P></TD><TD width="58"><P>-</P></TD><TD width="25"><P>+</P></TD><TD width="26"><P> </P></TD></TR><TR><TD width="27"><P>-</P></TD><TD width="27"><P> </P></TD><TD width="58"><P>0.005</P></TD><TD width="58"><P>10.5</P></TD><TD width="58"><P> </P></TD><TD width="58"><P>-</P></TD><TD width="58"><P>-</P></TD><TD width="58"><P>-</P></TD><TD width="25"><P>-</P></TD><TD width="26"><P> </P></TD></TR></TBODY></TABLE><P> Table 2 - OD measurments of the samples can be seen, as well as the OD after the PBS washes.</P><H2>Results</H2><H3>1.4.1     Data</H3><P><A>https://static.igem.org/mediawiki/2014/0/06/Sfgfpcpxr_gradient.jpg</A></P><P>Figure 1 - Photo of the pelleted samples between PBS wash steps 1 and 2.</P><P>As can be seen in the image, there is a gradient of fluorescence intensity.The results of the plate reader are depicted in Figure 2.</P><P><A href="https://static.igem.org/mediawiki/2014/f/f5/Twotermscpxrgfp.jpg">https://static.igem.org/mediawiki/2014/f/f5/Twotermscpxrgfp.jpg</A></P><P>Figure 2 - Graphs of the intensity curves of the samples. Cterm corresponds to CpxR sfGFP (sfGFP attached to CpxR's C temrinus) and Nterm corresponds to sfGFP CpxR.</P><H3>1.4.2     Interpretation</H3><P>As can be seen in the graph, the the results of the sfGFP CpxR sample was much better (closer to the expected results) than the one on the CpxR sfGFP, although in general, the curves grow.</P><P>It could be seen that adding too much arabinose decreases viability, so we considered that a concentration of 5 mM was the optimum arabinose concentration.</P></DIV></DIV><DIV class="whitebg box"><H3 id="date-08-17-2014">CheY CheZ - Week 7</H3><SPAN>08-17-2014</SPAN><DIV class="notebook-content"><P class="western" lang="fr-FR" align="left"> </P><H1 class="western" lang="en-US">1 Experiment 1: Bioluminescence assay 5</H1><P class="western" lang="en-US">Purpous: We want to test our constructs by measuring the emitted bioluminescence with a plate reader</P><H2 class="western" lang="en-US">1.1 Material and methods</H2><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">Overnight cultures were prepared from glycerol stocks of unstransformed DH5 alpha, fYNZC (tube 17), rYNZC (tube 26), pYNZC (received from Waldor's lab) fLucNfLucC (tube 26-), rLucNrLucC (tube 31-), fLuc (EPIC Firefly) and rLuc (rLuc from Waldor's lab) into 5mL LB-Chloramphenicol.</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">50mL of overnight cultures were transferred into 5mL fresh LB (+ chloramphenicol, except for untransformed DH5 alpha) containing 1mM arabinose. These sample were incubated for 3 hours at 37°C, 180rpm shaking, to an OD ~1.</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">Cultures were centrifuged, medium was discarded, cells were washed five times with PBS.</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">The substrate (coelenterazine (clz) for renilla Luciferase and d-luciferin (d-luc) for firefly Luciferase) were prepared in aliquots of 100uM and added at 30 uM (d-luc) and 22.5uM (clz).</FONT></FONT></P><P> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">A white opaque 96-Well plate was loaded as following. Samples were loaded in triplicate, in a total volume of 180uL contaning 120uL of culture, ajusting the OD to 0.7:</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">Measures were taken for 1h every 2 minutes with the plate reader Tekan Infinite.</FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"> </P><H2 class="western" lang="en-US">1.2 Results</H2><P>As shown in the graphs below (fig.1A and 1B), we couldn't really observe a high signal for our complementation assay. However, the signal being higher than the blanks, it is an encouraging sign that the splits luciferase can be used for other experiments of this kind. A possible explanation for these results is that arabinose being a chemoattractant, we might need to do more wash steps with PBS to get rid of the arabinose before taking the measurements. Moreover, CheY and CheZ being endogenously expressed in bacteria, the edogenous proteins could interfere with our fusion proteins and weaken our signal. This complementation assay should be tested with CheY/CheZ knock out strains, as it was done in Waldor Laboratory.</P><P>We also could determine which of the luciferases would best suit our following experiments. As shown in fig. 2, for the same concentration of substrate, we see that firefly luciferase has a more stable and higher signal. Moreover, the difference between the background noise (negative control, non fused split luciferase) and the full luciferase is bigger for Firefly luciferase, which is also preferable.</P><DIV class="container"> </DIV><P>$</P><H3 class="western" lang="en-US">1.2.2 Interpretation</H3><P><SPAN style="font-family: 'PT Sans', sans-serif; font-size: 14px; line-height: 21px;"> </SPAN></P></DIV></DIV><DIV class="whitebg box"><H3 id="date-08-24-2014">Leucine zippers - week 8</H3><SPAN>08-24-2014</SPAN><DIV class="notebook-content"><P lang="en-US" align="center"> </P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="5"><STRONG>1. Experiment 1: amplification of fragments of interest</STRONG></FONT></FONT></P><P lang="en-US">Purpous: We want to amplify the fragments of interest from different vectors in order to make the following constructs:</P><P lang="en-US" style="text-align: center;"><STRONG><FONT face="Calibri, sans-serif"><FONT size="2">Fig1A: Vector of zipper-rLucN + zipper-rLucC Fig1B: Vector of zipper-fLucN + zipper-fLucC</FONT></FONT></STRONG></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">The sequences will be amplified with overlapping fragments in order to further do a fusion PCR (exp. 2).</FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="4"><STRONG>1.1 Material and methods</STRONG></FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="3">Material:</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">Phusion PCR kit</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">The following primers: zip01, zip03, zip06, zip04, zip07, zip08, zip09, zip11, zip12, zip14, zip15, zip17, zip18 (see Sequences/Primers on Google drive)</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">Vectors: zip_IFP1m rlucNrLucC (from Waldor's lab), EPIC Firefly</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">PCR purification kit (Qiagen)</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">Nanodrop spectrophotometer</FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="3">Procedure:</FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2"><STRONG>Reaction set up:</STRONG></FONT></FONT></P><P lang="en-US" align="left"> </P><TABLE style="height: 206px;" width="595"><TBODY><TR><TD>PCR</TD><TD>Fragment</TD><TD>Template</TD><TD>Primer for</TD><TD>PRimer rev</TD><TD>Amp. size (bp)</TD><TD>A. temp (°C)</TD><TD>Programme</TD></TR><TR><TD>1</TD><TD>Zip + fusion OL1 for Ren</TD><TD>zip_IFP1</TD><TD>zip01</TD><TD>zip03</TD><TD>200</TD><TD>71</TD><TD>1</TD></TR><TR><TD>2</TD><TD>Zip + fusion OL2 for Ren </TD><TD>zip_IFP1</TD><TD>zip01</TD><TD>zip06</TD><TD>200</TD><TD>71</TD><TD>1</TD></TR><TR><TD>3</TD><TD>Zip + fusion OL1 for Fir</TD><TD>zip_IFP1</TD><TD>zip01</TD><TD>zip04</TD><TD>200</TD><TD>71</TD><TD>1</TD></TR><TR><TD>4</TD><TD>Zip + fusion OL2 for Fir</TD><TD>zip_IFP1</TD><TD>zip01</TD><TD>zip07</TD><TD>200</TD><TD>71</TD><TD>1</TD></TR><TR><TD>5</TD><TD>RLucN + fusion OL1</TD><TD>rlucNrlucC (Waldor)</TD><TD>zip08</TD><TD>zip09
<P lang="en-US" align="left"><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2"><STRONG> 1:</STRONG></FONT></FONT></FONT></P></TD><TD>340</TD><TD>72</TD><TD>1</TD></TR><TR><TD>6</TD><TD>RLucC + fusion OL2</TD><TD>rlucNrlucC (Waldor)</TD><TD>zip11</TD><TD>zip12</TD><TD>621</TD><TD>71</TD><TD>1</TD></TR><TR><TD>7</TD><TD>FlucN + fusion OL1</TD><TD>EPIC Firefly</TD><TD>zip14</TD><TD>zip15</TD><TD>1263</TD><TD>69</TD><TD>2</TD></TR><TR><TD>8</TD><TD>FlucC + OL2</TD><TD>EPIC Firefly</TD><TD>zip17</TD><TD>zip18</TD><TD>498</TD><TD>69</TD><TD>3</TD></TR></TBODY></TABLE><P lang="en-US" align="left"> </P><P lang="en-US" align="left"> </P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2">Each sample was done for 50uL of reaction. See protocol for phusion PCR for detailed quantities.</FONT></FONT></P><P lang="en-US" align="left"> </P><P lang="en-US" align="left"> </P><P lang="en-US" align="left"><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2"><STRONG>Thermocycling programs:</STRONG></FONT></FONT></FONT></P><P lang="en-US" align="left"> </P><P lang="en-US" align="left"> </P><TABLE class="table table-striped"><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">Initial denaturation</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">98°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">30 seconds</FONT></FONT></P></TD></TR></TBODY><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2">30 cycles</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">98°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">10 seconds</FONT></FONT></P></TD></TR><TR valign="top"><TD width="33%"><P lang="en-US" align="left"> </P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">71°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">30 seconds </FONT></FONT></P></TD></TR><TR valign="top"><TD width="33%"><P lang="en-US" align="left"> </P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">72°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">30 seconds</FONT></FONT></P></TD></TR></TBODY><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">Final Extension</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">72°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">10 min</FONT></FONT></P></TD></TR></TBODY><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">stop</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">4°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"> </P></TD></TR></TBODY></TABLE><P lang="en-US" align="left"> </P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2"><STRONG> 2:</STRONG></FONT></FONT></P><P lang="en-US" align="left"> </P><TABLE class="table table-striped"><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">Initial denaturation</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">98°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">30 seconds</FONT></FONT></P></TD></TR></TBODY><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2">30 cycles</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">98°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">10 seconds</FONT></FONT></P></TD></TR><TR valign="top"><TD width="33%"><P lang="en-US" align="left"> </P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">69°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">30 seconds </FONT></FONT></P></TD></TR><TR valign="top"><TD width="33%"><P lang="en-US" align="left"> </P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">72°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">1 minute</FONT></FONT></P></TD></TR></TBODY><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">Final Extension</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">72°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">10 min</FONT></FONT></P></TD></TR></TBODY><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">stop</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">4°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"> </P></TD></TR></TBODY></TABLE><P lang="en-US" align="left"> </P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2"><STRONG>3:</STRONG></FONT></FONT></P><P lang="en-US" align="left"> </P><TABLE class="table table-striped"><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">Initial denaturation</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">98°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">30 seconds</FONT></FONT></P></TD></TR></TBODY><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2">30 cycles</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">98°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">10 seconds</FONT></FONT></P></TD></TR><TR valign="top"><TD width="33%"><P lang="en-US" align="left"> </P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">69°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">30 seconds </FONT></FONT></P></TD></TR><TR valign="top"><TD width="33%"><P lang="en-US" align="left"> </P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">72°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">30 seconds</FONT></FONT></P></TD></TR></TBODY><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">Final Extension</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">72°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">10 min</FONT></FONT></P></TD></TR></TBODY><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">stop</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">4°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"> </P></TD></TR></TBODY></TABLE><P lang="en-US" align="left"> </P><P lang="en-US" align="left"> </P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2">After PCR, the products were purified according to the Qiagen PCR purification kit. Samples were analyzed by agarose gel electrophoresis</FONT></FONT></P><P lang="en-US" align="left"> </P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="4"><STRONG>1.2 Results</STRONG></FONT></FONT></P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="3"><STRONG>1.2.1 Data:</STRONG></FONT></FONT></P><P style="text-align: center;" align="left"><STRONG><FONT face="Calibri, sans-serif"><FONT size="2">Fig2A: Gel of the PCR for the amplification of the overlap fragments for fusion PCR</FONT></FONT></STRONG></P><P align="left"> </P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2"><STRONG>Caption: (lanes)</STRONG></FONT></FONT></P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2">1: Quick-load DNA marker</FONT></FONT></P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2">2: <FONT face="LiberationSans, sans-serif"><FONT size="2">Zip + fusion OL1 for Ren</FONT></FONT></FONT></FONT></P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2">3: <FONT face="LiberationSans, sans-serif"><FONT size="2">Zip + fusion OL2 for Ren</FONT></FONT></FONT></FONT></P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2">4: <FONT face="LiberationSans, sans-serif"><FONT size="2">Zip + fusion OL1 for Fir</FONT></FONT></FONT></FONT></P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2">5: <FONT face="LiberationSans, sans-serif"><FONT size="2">Zip + fusion OL2 for Fir</FONT></FONT></FONT></FONT></P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2">6: <FONT face="LiberationSans, sans-serif"><FONT size="2">RLucN + fusion OL1</FONT></FONT></FONT></FONT></P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2">7: <FONT face="LiberationSans, sans-serif"><FONT size="2">RLucC + fusion OL2</FONT></FONT></FONT></FONT></P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2">8: <FONT face="LiberationSans, sans-serif"><FONT size="2">FlucN + fusion OL1</FONT></FONT></FONT></FONT></P><P lang="en-US" align="left"><FONT face="LiberationSans, sans-serif"><FONT size="2">9: FlucC + fusion OL</FONT></FONT></P><P lang="en-US" align="center"> </P><P lang="en-US" align="center"> </P><P style="text-align: center;" align="left"><STRONG>Fig2B: Gel of the PCR for the amplification of rLucN + fusion OL1 (second trial, with all 4 samples of split renilla luciferase we received from Waldor's lab)</STRONG></P><P lang="en-US" align="left"> </P><P lang="en-US" align="left"> </P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2"><STRONG>Caption: (Lanes)</STRONG></FONT></FONT></P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2">2: Quick-load DNA marker</FONT></FONT></P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2">3: <FONT face="LiberationSans, sans-serif"><FONT size="2">RLucN + fusion OL1</FONT></FONT><FONT color="#000000"><FONT face="LiberationSans, sans-serif"><FONT size="2">(1)</FONT></FONT></FONT></FONT></FONT></P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2"><FONT color="#000000">4: </FONT><FONT color="#000000"><FONT face="LiberationSans, sans-serif"><FONT size="2">RLucN + fusion OL1 </FONT></FONT></FONT><FONT color="#000000"><FONT face="LiberationSans, sans-serif"><FONT size="2">(2)</FONT></FONT></FONT></FONT></FONT></P><P lang="en-US" align="left"><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">5: <FONT face="LiberationSans, sans-serif"><FONT size="2">RLucN + fusion OL1 </FONT></FONT><FONT face="LiberationSans, sans-serif"><FONT size="2">(3)</FONT></FONT></FONT></FONT></FONT></P><P lang="en-US" align="left"><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2"><FONT face="LiberationSans, sans-serif"><FONT size="2">6: RLucN + fusion OL1 </FONT></FONT><FONT face="LiberationSans, sans-serif"><FONT size="2">(4)</FONT></FONT></FONT></FONT></FONT></P><P lang="en-US" align="center"> </P><P lang="en-US" align="center"><STRONG><FONT face="Calibri, sans-serif"><FONT size="2"><FONT color="#000000"><FONT face="LiberationSans, sans-serif"><FONT size="2">Fig</FONT></FONT></FONT><FONT color="#000000"><FONT face="LiberationSans, sans-serif"><FONT size="2">3A</FONT></FONT></FONT><FONT color="#000000"><FONT face="LiberationSans, sans-serif"><FONT size="2">: Nanodrops of purified PCR products</FONT></FONT></FONT></FONT></FONT></STRONG></P><P lang="en-US" align="left"> </P><P lang="en-US" style="text-align: center;" align="left"><STRONG><FONT color="#000000"><FONT face="LiberationSans, sans-serif"><FONT size="2">Fig</FONT></FONT></FONT><FONT color="#000000"><FONT face="LiberationSans, sans-serif"><FONT size="2">3B</FONT></FONT></FONT><FONT color="#000000"><FONT face="LiberationSans, sans-serif"><FONT size="2">: Nanodrops of purified PCR products </FONT></FONT></FONT><FONT color="#000000"><FONT face="LiberationSans, sans-serif"><FONT size="2">(</FONT></FONT></FONT><FONT color="#000000">rLucN + fusion OL1 only, second trial)</FONT></STRONG></P><P lang="en-US" align="center"> </P><P lang="en-US" align="center"> </P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="3"><STRONG>1.2.2 Interpretation:</STRONG></FONT></FONT></P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2">There is nothing in the lane 5 in fig 2A, the PCR for RlucN+fusion OL1 didn't work. We tried it again with all the samples of split rLuc we had (which gave the gel in fig 2B) and obtained the right fragment in all 4 samples.</FONT></FONT></P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2">All the other obtained fragments have the expected size, so we can continue our experiments to obtain the 2 vectors we want.</FONT></FONT></P><P lang="en-US" align="left"> </P><P lang="en-US" align="left"> </P><P lang="en-US" align="left"> </P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="4"><STRONG><FONT size="5">2</FONT><FONT size="5">. Experiment </FONT><FONT size="5">2: fusion PCR</FONT></STRONG></FONT></FONT></P><P lang="en-US"><FONT size="2">Purpous: We want to </FONT><FONT size="2">perform a fusion PCR to obtain the following products</FONT><FONT size="2">: </FONT><FONT size="2">zipper + rLucN, zipper + rLucC, zipper + fLucN </FONT><FONT size="2">and </FONT><FONT size="2">zipper + fLucC</FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="4"><STRONG>2.1 Material and methods</STRONG></FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="3">Material:</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">Phusion PCR kit</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">Primers: zip02, zip10, zip05, zip13, zip16, zip19 (see Sequences/Primers on Google drive)</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">PCR products from previous experiment</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">PCR purification kit (Qiagen)</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">Nanodrop spectrophotometer</FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="3">Procedure:</FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">Each sample was done for 50uL total volume reaction. See protocol for phusion PCR for detailed quantities to add. The first PCR (part A) is done without primers, for 15 cycles. Primers are add before part B</FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2"><STRONG>Reaction set up:</STRONG></FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"> </P><TABLE class="table table-striped"><TBODY><TR valign="top"><TD width="10%"><P> </P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">1</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">2</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">3</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">4</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">5</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">6</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">7</FONT></P></TD><TD width="8%"><P><FONT face="Calibri, sans-serif">8</FONT></P></TD><TD width="9%"><P><FONT face="Calibri, sans-serif">Water</FONT></P></TD></TR><TR valign="top"><TD width="10%"><P><FONT face="Calibri, sans-serif">Tube 1</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">0.32µl</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">---</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">---</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">---</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">1.86µl</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">---</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">---</FONT></P></TD><TD width="8%"><P><FONT face="Calibri, sans-serif">---</FONT></P></TD><TD width="9%"><P><FONT face="Calibri, sans-serif">32.68µl</FONT></P></TD></TR><TR valign="top"><TD width="10%"><P><FONT face="Calibri, sans-serif">Tube 2</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">---</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">0.2µl</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">---</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">---</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">---</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">1.72µl</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">---</FONT></P></TD><TD width="8%"><P><FONT face="Calibri, sans-serif">---</FONT></P></TD><TD width="9%"><P><FONT face="Calibri, sans-serif">32.77µl</FONT></P></TD></TR><TR valign="top"><TD width="10%"><P><FONT face="Calibri, sans-serif">Tube 3</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">---</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">---</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">0.21µl</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">---</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">---</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">---</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">0.61µl</FONT></P></TD><TD width="8%"><P><FONT face="Calibri, sans-serif">---</FONT></P></TD><TD width="9%"><P><FONT face="Calibri, sans-serif">31.32µl</FONT></P></TD></TR><TR valign="top"><TD width="10%"><P><FONT face="Calibri, sans-serif">Tube 4</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">---</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">---</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">---</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">0.28µl</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">---</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">---</FONT></P></TD><TD width="10%"><P><FONT face="Calibri, sans-serif">---</FONT></P></TD><TD width="8%"><P><FONT face="Calibri, sans-serif">0.45µl</FONT></P></TD><TD width="9%"><P><FONT face="Calibri, sans-serif">31.58µl</FONT></P></TD></TR></TBODY></TABLE><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif">Master Mix: (for 5 samples): 50µl 5X Buffer, 5µl dNTPs, 2.5 µl polymerase</FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2"><STRONG>Thermocycling programs:</STRONG></FONT></FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT color="#000000"><FONT face="Calibri, sans-serif"><STRONG>4A </STRONG></FONT></FONT></P><TABLE class="table table-striped"><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">Initial denaturation</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">98°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">30 seconds</FONT></FONT></P></TD></TR></TBODY><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2">15 cycles</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">98°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">10 seconds</FONT></FONT></P></TD></TR><TR valign="top"><TD width="33%"><P lang="en-US" align="left"> </P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">60°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">30 seconds </FONT></FONT></P></TD></TR><TR valign="top"><TD width="33%"><P lang="en-US" align="left"> </P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">72°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">45 seconds</FONT></FONT></P></TD></TR></TBODY><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">Final Extension</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">72°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">10 min</FONT></FONT></P></TD></TR></TBODY><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">stop</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">4°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"> </P></TD></TR></TBODY></TABLE><P lang="en-US" align="left"> </P><P lang="en-US"><FONT color="#000000"><FONT face="Calibri, sans-serif"><STRONG>5A</STRONG></FONT></FONT></P><TABLE class="table table-striped"><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">Initial denaturation</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">98°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">30 seconds</FONT></FONT></P></TD></TR></TBODY><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2">15 cycles</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">98°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">10 secon</FONT></FONT></P></TD></TR><TR valign="top"><TD width="33%"><P lang="en-US" align="left"> </P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">60°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">30 seconds </FONT></FONT></P></TD></TR><TR valign="top"><TD width="33%"><P lang="en-US" align="left"> </P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">72°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">9 seconds</FONT></FONT></P></TD></TR></TBODY><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">Final Extension</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">72°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">10 min</FONT></FONT></P></TD></TR></TBODY><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">stop</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">4°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"> </P></TD></TR></TBODY></TABLE><P lang="en-US" align="left"> </P><P lang="en-US"><FONT color="#000000"><FONT face="Calibri, sans-serif"><STRONG>4B</STRONG></FONT></FONT></P><TABLE class="table table-striped"><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">Initial denaturation</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">98°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">30 seconds</FONT></FONT></P></TD></TR></TBODY><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2">15 cycles</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">98°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">10 seconds</FONT></FONT></P></TD></TR><TR valign="top"><TD width="33%"><P lang="en-US" align="left"> </P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">72°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">30 seconds </FONT></FONT></P></TD></TR><TR valign="top"><TD width="33%"><P lang="en-US" align="left"> </P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">72°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">45 seconds</FONT></FONT></P></TD></TR></TBODY><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">Final Extension</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">72°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">10 min</FONT></FONT></P></TD></TR></TBODY><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">stop</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">4°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"> </P></TD></TR></TBODY></TABLE><P lang="en-US"> </P><P lang="en-US"><FONT color="#000000"><FONT face="Calibri, sans-serif"><STRONG>5B</STRONG></FONT></FONT></P><TABLE class="table table-striped"><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">Initial denaturation</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">98°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">30 seconds</FONT></FONT></P></TD></TR></TBODY><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2">15 cycles</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">98°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">10 seconds</FONT></FONT></P></TD></TR><TR valign="top"><TD width="33%"><P lang="en-US" align="left"> </P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">72°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">30 seconds </FONT></FONT></P></TD></TR><TR valign="top"><TD width="33%"><P lang="en-US" align="left"> </P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">72°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">90 seconds</FONT></FONT></P></TD></TR></TBODY><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">Final Extension</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">72°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">10 min</FONT></FONT></P></TD></TR></TBODY><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">stop</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">4°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"> </P></TD></TR></TBODY></TABLE><P lang="en-US"> </P><P lang="en-US"><FONT color="#000000"><FONT face="Calibri, sans-serif"><FONT size="2">After the PCR, the products were purified. The concentration of DNA was determined with the nanodrop spectrophotometer. The fragments were verified by agarose gel electrophoresis</FONT></FONT></FONT></P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2">Changes in PCR program for sample 4 (as it didn't work as expected the first time)</FONT></FONT></P><P lang="en-US" align="left"> </P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2"><STRONG>6A</STRONG></FONT></FONT></P><TABLE class="table table-striped"><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">Initial denaturation</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">98°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">30 seconds</FONT></FONT></P></TD></TR></TBODY><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2">15 cycles</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">98°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">10 seconds</FONT></FONT></P></TD></TR><TR valign="top"><TD width="33%"><P lang="en-US" align="left"> </P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">59°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">30 seconds </FONT></FONT></P></TD></TR><TR valign="top"><TD width="33%"><P lang="en-US" align="left"> </P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">72°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">45 seconds</FONT></FONT></P></TD></TR></TBODY><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">Final Extension</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">72°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">10 min</FONT></FONT></P></TD></TR></TBODY><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">stop</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">4°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"> </P></TD></TR></TBODY></TABLE><P lang="en-US" align="center"> </P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2"><STRONG>6B</STRONG></FONT></FONT></P><TABLE class="table table-striped"><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">Initial denaturation</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">98°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">30 seconds</FONT></FONT></P></TD></TR></TBODY><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2">15 cycles</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">98°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">10 seconds</FONT></FONT></P></TD></TR><TR valign="top"><TD width="33%"><P lang="en-US" align="left"> </P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">73°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">30 seconds </FONT></FONT></P></TD></TR><TR valign="top"><TD width="33%"><P lang="en-US" align="left"> </P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">72°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">45 seconds</FONT></FONT></P></TD></TR></TBODY><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">Final Extension</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">72°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">10 min</FONT></FONT></P></TD></TR></TBODY><TBODY><TR valign="top"><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">stop</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2">4°C</FONT></FONT></P></TD><TD width="33%"><P lang="en-US" align="center"> </P></TD></TR></TBODY></TABLE><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="4"><STRONG>2.2 Results</STRONG></FONT></FONT></P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="3"><STRONG>2.2.1 Data:</STRONG></FONT></FONT></P><P style="text-align: center;" align="left"><STRONG><FONT face="Calibri, sans-serif"><FONT size="2">Fig 4A: Gel of the fusion PCR </FONT></FONT></STRONG></P><P lang="en-US" align="left"> </P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2"><STRONG>Caption</STRONG></FONT></FONT></P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2">Lane 2: ladder quick-load DNA marker</FONT></FONT></P><P align="left"><FONT face="Calibri, sans-serif"><FONT size="2"><SPAN lang="en-US">Lane 3:</SPAN>Zip-rLucN</FONT></FONT></P><P align="left"><FONT face="Calibri, sans-serif"><FONT size="2"><SPAN lang="en-US">Lane 4:</SPAN>Zip-rLucC</FONT></FONT></P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2">Lane 5: Zip-fLucN</FONT></FONT></P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2">Lane 6: Zip-fLucC</FONT></FONT></P><P lang="en-US" align="center"> </P><P lang="en-US" align="left"> </P><P lang="en-US" align="center"> </P><P lang="en-US" align="center"> </P><P lang="en-US" style="text-align: center;" align="left"> <STRONG>Fig 4B: Gel of the fusion PCR for sample 4 only (second trial)</STRONG></P><P lang="en-US" style="text-align: center;" align="left"> </P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2"><STRONG>Caption</STRONG></FONT></FONT></P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2">Lane 3: ladder quick-load DNA marker</FONT></FONT></P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="2">Lane 4: Zip-fLucC</FONT></FONT></P><P lang="en-US" align="left"> </P><P lang="en-US" align="left"> </P><P lang="en-US" align="left"> </P><P lang="en-US" align="left"> </P><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2"><FONT color="#000000"><FONT face="LiberationSans, sans-serif"><FONT size="2">Fig</FONT></FONT></FONT><FONT color="#000000"><FONT face="LiberationSans, sans-serif"><FONT size="2">5</FONT></FONT></FONT><FONT color="#000000"><FONT face="LiberationSans, sans-serif"><FONT size="2">A</FONT></FONT></FONT><FONT color="#000000"><FONT face="LiberationSans, sans-serif"><FONT size="2">: Nanodrops of purified PCR products</FONT></FONT></FONT></FONT></FONT></P><P lang="en-US" align="center"> </P><P lang="en-US" align="center"><FONT face="Calibri, sans-serif"><FONT size="2"><FONT color="#000000"><FONT face="LiberationSans, sans-serif"><FONT size="2">Fig</FONT></FONT></FONT><FONT color="#000000"><FONT face="LiberationSans, sans-serif"><FONT size="2">5</FONT></FONT></FONT><FONT color="#000000"><FONT face="LiberationSans, sans-serif"><FONT size="2">B</FONT></FONT></FONT><FONT color="#000000"><FONT face="LiberationSans, sans-serif"><FONT size="2">: Nanodrop of purified PCR product </FONT></FONT></FONT><FONT color="#000000"><FONT face="LiberationSans, sans-serif"><FONT size="2">(sample 4 only)</FONT></FONT></FONT></FONT></FONT></P><P lang="en-US" align="center"> </P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="3"><STRONG>2.2.2 Interpretation:</STRONG></FONT></FONT></P><P lang="en-US" align="left"><FONT face="Calibri, sans-serif"><FONT size="3">The first gel shows that the first three fusion PCR have been successful, as we obtain the expected sizes. We retried a PCR (changing the program) for sample 4, which was then successful too. The fragments are now ready for a Gibson assembly</FONT></FONT></P><P lang="en-US" align="center"> </P><P lang="en-US" align="center"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="4"><STRONG><FONT size="5">3</FONT><FONT size="5">. Experiment </FONT><FONT size="5">3</FONT><FONT size="5">: </FONT><FONT size="5">Gibson assembly</FONT></STRONG></FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT size="2">Purpous: </FONT><FONT size="2">We want to perform a Gibson assembly with the previous prepared fragments, in order to obtain the vectors from fig 1A and 1B</FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="4"><STRONG>2.1 Material and methods</STRONG></FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="3">Material:</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">PCR products from previous experiment</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">Vector (opened) BB3 (69.6 [ng/uL])</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">Competent cells</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">Gibson assembly master mix</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="2">Taq polymerase kit</FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="3">Procedure:</FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="3">GA1: zip-rLucN + zip-rLucC</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="3">GA2: zip fLucN + zip-fLucC</FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="3">We put 50 ng of vector (BB3, 69.6ng/uL) and 150ng of insert.</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="3">Reaction set up :</FONT></FONT></P><TABLE class="table table-striped"><TBODY><TR valign="top"><TD width="9%"><P><FONT face="Calibri, sans-serif"><FONT size="2">[uL]</FONT></FONT></P></TD><TD width="13%"><P><FONT face="Calibri, sans-serif"><FONT size="2">Vector (BB3)</FONT></FONT></P></TD><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="2">Zip-rLucN</FONT></FONT></P></TD><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="2">Zip-rLucC</FONT></FONT></P></TD><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="2">Zip-fLucN</FONT></FONT></P></TD><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="2">Zip-fLucC</FONT></FONT></P></TD><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="2">MasterMix</FONT></FONT></P></TD><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="2">Water</FONT></FONT></P></TD><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="2">Total</FONT></FONT></P></TD></TR><TR valign="top"><TD width="9%"><P><FONT face="Calibri, sans-serif"><FONT size="2">GA1</FONT></FONT></P></TD><TD width="13%"><P><FONT face="Calibri, sans-serif"><FONT size="2">0.72</FONT></FONT></P></TD><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="2">1.16</FONT></FONT></P></TD><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="2">0.97</FONT></FONT></P></TD><TD bgcolor="#e6e6ff" width="11%"><P> </P></TD><TD bgcolor="#e6e6ff" width="11%"><P> </P></TD><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="2">10</FONT></FONT></P></TD><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="2">7.15</FONT></FONT></P></TD><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="2">20</FONT></FONT></P></TD></TR><TR valign="top"><TD width="9%"><P><FONT face="Calibri, sans-serif"><FONT size="2">GA2</FONT></FONT></P></TD><TD width="13%"><P><FONT face="Calibri, sans-serif"><FONT size="2">0.72</FONT></FONT></P></TD><TD bgcolor="#e6e6ff" width="11%"><P> </P></TD><TD bgcolor="#e6e6ff" width="11%"><P> </P></TD><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="2">1.05</FONT></FONT></P></TD><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="2">1.45</FONT></FONT></P></TD><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="2">10</FONT></FONT></P></TD><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="2">6.78</FONT></FONT></P></TD><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="2">20</FONT></FONT></P></TD></TR></TBODY></TABLE><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="3">We transformed the competent cells according to the protocol from <A href="https://www.neb.com/protocols/2013/10/29/chemically-competent-cells-transformation-protocol-e2611">Neb</A></FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="3">Cells were plated onto 2 plates per Gibson Assembly reaction (GA1 A&amp;B, GA2 A&amp;B)</FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="3">The following day (04.09.14), we picked 4 colonies from each plate, for subsequent colony PCRs and cultures. The tips which picked the colonies were agitated into 10uL water and then dropped into 5mL LB chloramphenicol. Cells were cultured for 6 hours for subsequent glycerol stocks preparation and minipreps. The cells in water were heat shocked for 10 minutes at 95°C and used for colony PCR. The minipreps were used for restriction analysis.</FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="3"><STRONG>COLONY PCR</STRONG></FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="3">Primers used for colony PCR:</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="3">GA1: zip01 and rLucC_rev</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="3">GA2: zip01 and fLucC_rev</FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="3">Reaction Set Up: Taq Polymerase protocol</FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="3">Colony PCR results:</FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"> </P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="3">[GEL]</FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"> </P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="3">Colonies GA1 A3 and GA2 A1, A2, B2, B3, B4 were discarded. As the colony PCR wasn't very clear, we did a restriction analysis the following day (05.09.14)</FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="3"><STRONG>Restriction Analysis 1 (05.09.14)</STRONG></FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="3">Each sample was double digested with NheI and SpeI.</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="3">Sample preparation: 500ng of template DNA, 5uL of Cutsmart buffer, 0.5 uL of each restriction enzyme and water to 50uL. Samples were put at 37°C during one hour and then stored on ice to be load on agarose gel.</FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="3">For GA1, we should obtain 2 fragments, of 3273 and 1356 base pairs. For GA2, we should obtain two fragments of 3273 and 2142 base pairs.</FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"> </P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="3">[GEL]</FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="3">Conclusion:</FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="3">The insert for GA2 seemed to be smaller than expected. At this point, we also decided to discard sample B3.</FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"> </P><P lang="en-US"> </P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="3"><STRONG>Restriction Analysis 2</STRONG></FONT></FONT></P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="3">We tried another restriction analysis. Each sample was used for a simple and a double digestion. We compared the obtained fragments to the supercoiled plasmids.</FONT></FONT></P><P lang="en-US"> </P><P lang="en-US"> </P><P lang="en-US"> </P><P lang="en-US"><FONT face="Calibri, sans-serif"><FONT size="3">Reaction Set up</FONT></FONT></P><P lang="en-US"> </P><P> </P><TABLE class="table table-striped"><TBODY><TR valign="top"><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="3">Sample</FONT></FONT></P></TD><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="3">Name</FONT></FONT></P></TD><TD width="8%"><P><FONT face="Calibri, sans-serif"><FONT size="3">NheI</FONT></FONT></P></TD><TD width="9%"><P><FONT face="Calibri, sans-serif"><FONT size="3">Hind III</FONT></FONT></P></TD><TD width="8%"><P><FONT face="Calibri, sans-serif"><FONT size="3">XbaI</FONT></FONT></P></TD><TD width="18%"><P><FONT face="Calibri, sans-serif"><FONT size="3">Cutsmart Buffer</FONT></FONT></P></TD><TD width="19%"><P><FONT face="Calibri, sans-serif"><FONT size="3">Template (500ng)</FONT></FONT></P></TD><TD width="9%"><P><FONT face="Calibri, sans-serif"><FONT size="3">Water</FONT></FONT></P></TD><TD width="8%"><P><FONT face="Calibri, sans-serif"><FONT size="3">Total</FONT></FONT></P></TD></TR><TR valign="top"><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="3">1</FONT></FONT></P></TD><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="3">GA1 A1</FONT></FONT></P></TD><TD width="8%"><P><FONT face="Calibri, sans-serif"><FONT size="3">0.5</FONT></FONT></P></TD><TD bgcolor="#cccccc" width="9%"><P> </P></TD><TD bgcolor="#cccccc" width="8%"><P> </P></TD><TD width="18%"><P><FONT face="Calibri, sans-serif"><FONT size="3">5</FONT></FONT></P></TD><TD width="19%"><P><FONT face="Calibri, sans-serif"><FONT size="3">2.81</FONT></FONT></P></TD><TD width="9%"><P><FONT face="Calibri, sans-serif"><FONT size="3">41.69</FONT></FONT></P></TD><TD width="8%"><P><FONT face="Calibri, sans-serif"><FONT size="3">50</FONT></FONT></P></TD></TR><TR valign="top"><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="3">2</FONT></FONT></P></TD><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="3">GA1 A2</FONT></FONT></P></TD><TD width="8%"><P><FONT face="Calibri, sans-serif"><FONT size="3">0.5</FONT></FONT></P></TD><TD bgcolor="#cccccc" width="9%"><P> </P></TD><TD bgcolor="#cccccc" width="8%"><P> </P></TD><TD width="18%"><P><FONT face="Calibri, sans-serif"><FONT size="3">5</FONT></FONT></P></TD><TD width="19%"><P><FONT face="Calibri, sans-serif"><FONT size="3">2.00</FONT></FONT></P></TD><TD width="9%"><P><FONT face="Calibri, sans-serif"><FONT size="3">42.5</FONT></FONT></P></TD><TD width="8%"><P><FONT face="Calibri, sans-serif"><FONT size="3">50</FONT></FONT></P></TD></TR><TR valign="top"><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="3">3</FONT></FONT></P></TD><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="3">GA1 A4</FONT></FONT></P></TD><TD width="8%"><P><FONT face="Calibri, sans-serif"><FONT size="3">0.5</FONT></FONT></P></TD><TD bgcolor="#cccccc" width="9%"><P> </P></TD><TD bgcolor="#cccccc" width="8%"><P> </P></TD><TD width="18%"><P><FONT face="Calibri, sans-serif"><FONT size="3">5</FONT></FONT></P></TD><TD width="19%"><P><FONT face="Calibri, sans-serif"><FONT size="3">2.71</FONT></FONT></P></TD><TD width="9%"><P><FONT face="Calibri, sans-serif"><FONT size="3">41.79</FONT></FONT></P></TD><TD width="8%"><P><FONT face="Calibri, sans-serif"><FONT size="3">50</FONT></FONT></P></TD></TR><TR valign="top"><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="3">4</FONT></FONT></P></TD><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="3">GA1 B1</FONT></FONT></P></TD><TD width="8%"><P><FONT face="Calibri, sans-serif"><FONT size="3">0.5</FONT></FONT></P></TD><TD bgcolor="#cccccc" width="9%"><P> </P></TD><TD bgcolor="#cccccc" width="8%"><P> </P></TD><TD width="18%"><P><FONT face="Calibri, sans-serif"><FONT size="3">5</FONT></FONT></P></TD><TD width="19%"><P><FONT face="Calibri, sans-serif"><FONT size="3">4.42</FONT></FONT></P></TD><TD width="9%"><P><FONT face="Calibri, sans-serif"><FONT size="3">40.08</FONT></FONT></P></TD><TD width="8%"><P><FONT face="Calibri, sans-serif"><FONT size="3">50</FONT></FONT></P></TD></TR><TR valign="top"><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="3">5</FONT></FONT></P></TD><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="3">GA1 B2</FONT></FONT></P></TD><TD width="8%"><P><FONT face="Calibri, sans-serif"><FONT size="3">0.5</FONT></FONT></P></TD><TD bgcolor="#cccccc" width="9%"><P> </P></TD><TD bgcolor="#cccccc" width="8%"><P> </P></TD><TD width="18%"><P><FONT face="Calibri, sans-serif"><FONT size="3">5</FONT></FONT></P></TD><TD width="19%"><P><FONT face="Calibri, sans-serif"><FONT size="3">2.50</FONT></FONT></P></TD><TD width="9%"><P><FONT face="Calibri, sans-serif"><FONT size="3">42</FONT></FONT></P></TD><TD width="8%"><P><FONT face="Calibri, sans-serif"><FONT size="3">50</FONT></FONT></P></TD></TR><TR valign="top"><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="3">6</FONT></FONT></P></TD><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="3">GA1 B4</FONT></FONT></P></TD><TD width="8%"><P><FONT face="Calibri, sans-serif"><FONT size="3">0.5</FONT></FONT></P></TD><TD bgcolor="#cccccc" width="9%"><P> </P></TD><TD bgcolor="#cccccc" width="8%"><P> </P></TD><TD width="18%"><P><FONT face="Calibri, sans-serif"><FONT size="3">5</FONT></FONT></P></TD><TD width="19%"><P><FONT face="Calibri, sans-serif"><FONT size="3">3.50</FONT></FONT></P></TD><TD width="9%"><P><FONT face="Calibri, sans-serif"><FONT size="3">41</FONT></FONT></P></TD><TD width="8%"><P><FONT face="Calibri, sans-serif"><FONT size="3">50</FONT></FONT></P></TD></TR><TR valign="top"><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="3">7</FONT></FONT></P></TD><TD width="11%"><P><FONT face="Calibri, sans-serif"><FONT size="3">GA2 A3</FONT></FONT></P>&lt;</TD><DIV class="whitebg box"><H3 id="date-09-16-2014">Reparation and improvement of the Biobrick BBa_K339007</H3><SPAN>09-16-2014</SPAN><DIV class="notebook-content"><P> </P><P> </P><H1>Experiment 1: PCR amplification</H1><P>Our sequencing results of the BioBrick BBa_K3396007 showed us that the CpxR promoter is not infroant of the RFP coding sequence. In order to improve that of the iGEM library we repair the BBa_K3396007 meaning to put the CpxR promoter in front of the RFP, which is not the case jet as we realized after sequencing. We also want to test its sensitivity to different kinds of stress.</P><H2>1.1       Material and methods</H2><P>BBa_K3396007</P><P>Primers :                  AD_01_00_fw</P><P>                               AD_01_00_rv</P><P>Phiusion PCR kit</P><P> </P></DIV></DIV></TR><TR><TD width="126"><P><STRONG> </STRONG></P></TD><TD width="142"><P><STRONG>1</STRONG></P></TD><TD width="121"><P><STRONG>2</STRONG></P></TD></TR><TR><TD width="126"><P><STRONG>BBa_</STRONG><STRONG>K3396007</STRONG></P></TD><TD width="142"><P>0.1 µl</P></TD><TD width="121"><P>0.1 µl</P></TD></TR><TR><TD width="126"><P><STRONG>AD_01_00_fw</STRONG></P></TD><TD width="142"><P>2.5 µl</P></TD><TD width="121"><P>2.5 µl</P></TD></TR><TR><TD width="126"><P><STRONG>AD_01_00_rv</STRONG></P></TD><TD width="142"><P>2.5 µl</P></TD><TD width="121"><P>2.5 µl</P></TD></TR><TR><TD width="126"><P><STRONG>Mastermix</STRONG></P></TD><TD width="142"><P>11.5 µl</P></TD><TD width="121"><P>11.5 µl</P></TD></TR><TR><TD width="126"><P><STRONG>H<SUB>2</SUB>O</STRONG></P></TD><TD width="142"><P>33.4 µl</P></TD><TD width="121"><P>33.4 µl</P></TD></TR><TR><TD width="126"><P><STRONG>Total volume</STRONG></P></TD><TD width="142"><P>50 µl</P></TD><TD width="121"><P>50 µl</P></TD></TR></TBODY></TABLE><P> </P><P>PCR program :</P><P>          Tem          Time      repeat from step         repetitions</P><P>       1:         98 °C               30 s</P><P>       2:         98 °C               10 s</P><P>       3:         69 °C               30 s</P><P>       4:         72 °C               4 min               2                  29</P><P>       5:         72 °C               10 min</P><P> </P><P> </P><H2>1.2       Results</H2><P>Expected (thoretical) Result :</P><P> </P><P>Agarose gels (expected vs real):</P><P>      </P><P> </P><P>Nano Drop results:</P><TABLE class="table table-striped"><TBODY><TR><TD width="94"><P><STRONG>Sample NR</STRONG></P></TD><TD width="97"><P><STRONG>Product Name</STRONG></P></TD><TD width="97"><P><STRONG>Concentration</STRONG></P></TD><TD width="95"><P><STRONG>A260/A280</STRONG></P></TD><TD width="95"><P><STRONG>A260/A230</STRONG></P></TD></TR><TR><TD width="94"><P>1</P></TD><TD width="97"><P>AD_01_00</P></TD><TD width="97"><P>45.8 ng/µl</P></TD><TD width="95"><P>1.88</P></TD><TD width="95"><P>2.24</P></TD></TR><TR><TD width="94"><P>2</P></TD><TD width="97"><P>AD_01_00</P></TD><TD width="97"><P>74.3 ng/µl</P></TD><TD width="95"><P>1.87</P></TD><TD width="95"><P>2.25</P></TD></TR></TBODY></TABLE><P> </P><H3>1.2.1     Interpretation</H3><P>The fragments have the expected size. For further experiments Sample nr 2 will be used.</P><H1>Experiment 2: PCR-product digestion with DPN1</H1><P>The objective of this step is to digest the plasmids that served as a template, so they can not be transformed in later experiments.</P><H2>2.1       Material and methods</H2><UL><LI>DpnI (NEB)</LI><LI>Neb CutSmart buffer</LI><LI>AD_01_00 (sample 2)</LI><LI>PCR-purification Kit (Qiagen)</LI></UL><TABLE class="table table-striped"><TBODY><TR><TD width="161"><P><STRONG>Tube nr</STRONG></P></TD><TD width="270"><P><STRONG>1</STRONG></P></TD></TR><TR><TD width="161"><P><STRONG>AD_01_00</STRONG></P></TD><TD width="270"><P>8 µl</P></TD></TR><TR><TD width="161"><P><STRONG>Dpn1</STRONG></P></TD><TD width="270"><P>1 µl</P></TD></TR><TR><TD width="161"><P><STRONG>Cutsmart buffer</STRONG></P></TD><TD width="270"><P>1 µl</P></TD></TR><TR><TD width="161"><P><STRONG>Total Volume</STRONG></P></TD><TD width="270"><P>10 µl</P></TD></TR></TBODY></TABLE><P>Incubate at 37 °C for 30 min</P><P>Head-inactivation by incubating at 80 °C for 20 min</P><P>PCR-Purification (Qiagen)</P><H2>2.2       Results</H2><TABLE class="table table-striped"><TBODY><TR><TD width="120"><P><STRONG>Sample NR</STRONG></P></TD><TD width="120"><P><STRONG>Concentration</STRONG></P></TD><TD width="120"><P><STRONG>A260/A280</STRONG></P></TD><TD width="120"><P><STRONG>A260/A230</STRONG></P></TD></TR><TR><TD width="120"><P>AD_01_00</P></TD><TD width="120"><P>13.4 ng/µl</P></TD><TD width="120"><P>1.75</P></TD><TD width="120"><P>2.11</P></TD></TR></TBODY></TABLE><H3>2.2.1     Interpretation</H3><P>The concentration of any remaning plasmid in the sample would be to low to see on a gel. The concentration is very lowbut suffisient to run a PCR on it.</P><P> </P><H1>Experiment 3: PCR Amplification and Extention</H1><P>The objective is to add the CpxR promoter to the linear sequence by PCR. The sequence is in the primers.</P><H2>3.1       Material and methods</H2><UL><LI>Template: AD_01_00 (sample 2)</LI><LI>Primers :
</LI><LI>AD_01_01_fw</LI><LI>AD_01_01_rv</LI></UL><LI>Phiusion PCR Kit</LI><P> </P><TABLE class="table table-striped"><TBODY><TR><TD width="126"><P><STRONG> </STRONG></P></TD><TD width="142"><P><STRONG>1</STRONG></P></TD><TD width="121"><P><STRONG>2</STRONG></P></TD></TR><TR><TD width="126"><P><STRONG>AD_01_00</STRONG></P></TD><TD width="142"><P>0.1 µl</P></TD><TD width="121"><P>0.1 µl</P></TD></TR><TR><TD width="126"><P><STRONG>AD_01_01_fw</STRONG></P></TD><TD width="142"><P>2.5 µl</P></TD><TD width="121"><P>2.5 µl</P></TD></TR><TR><TD width="126"><P><STRONG>AD_01_01_rv</STRONG></P></TD><TD width="142"><P>2.5 µl</P></TD><TD width="121"><P>2.5 µl</P></TD></TR><TR><TD width="126"><P><STRONG>Mastermix</STRONG></P></TD><TD width="142"><P>11.5 µl</P></TD><TD width="121"><P>11.5 µl</P></TD></TR><TR><TD width="126"><P><STRONG>H<SUB>2</SUB>O</STRONG></P></TD><TD width="142"><P>33.4 µl</P></TD><TD width="121"><P>33.4 µl</P></TD></TR><TR><TD width="126"><P><STRONG>Total volume</STRONG></P></TD><TD width="142"><P>50 µl</P></TD><TD width="121"><P>50 µl</P></TD></TR></TBODY></TABLE><P> </P><P>PCR program :</P><P>       1:         98 °C               30 s</P><P>       2:         98 °C               10 s</P><P>       3:         69 °C               30 s</P><P>       4:         72 °C               4 min               2          29</P><P>       5:         72 °C               10 min</P><P> </P><H2>3.2       Results</H2><P>Expected result :</P><P> Gels (Theoretical vs real):</P><P>    </P><P> </P><P>Nano Drop results:</P><TABLE class="table table-striped"><TBODY><TR><TD width="94"><P><STRONG>Sample NR</STRONG></P></TD><TD width="97"><P><STRONG>Product Name</STRONG></P></TD><TD width="97"><P><STRONG>Concentration</STRONG></P></TD><TD width="95"><P><STRONG>A260/A280</STRONG></P></TD><TD width="95"><P><STRONG>A260/A230</STRONG></P></TD></TR><TR><TD width="94"><P>1</P></TD><TD width="97"><P>AD_01_01</P></TD><TD width="97"><P>43.8 ng/µl</P></TD><TD width="95"><P>1.88</P></TD><TD width="95"><P>2.24</P></TD></TR><TR><TD width="94"><P>2</P></TD><TD width="97"><P>AD_01_01</P></TD><TD width="97"><P>77.3 ng/µl</P></TD><TD width="95"><P>1.87</P></TD><TD width="95"><P>2.25</P></TD></TR></TBODY></TABLE><P> </P><H3>3.2.1     Interpretation</H3><P>Sample 2 will be used for the sitedirected mutagenesis</P><H1>4        Experiment 3: site directed mutagenesis</H1><P>The purpose of this experiment is to close the linear plasmid with a enzymatic reaction.</P><P> </P><H2>4.1       Material and methods</H2><P>Materials:</P><UL><LI>Q5® Site-Directed Mutagenesis Kit (NEB)</LI><LI>AD_01_01 sample 2</LI></UL><P> </P><P>Methode:</P><TABLE class="table table-striped"><TBODY><TR><TD width="246"><P><STRONG> Sample name </STRONG></P></TD><TD width="246"><P><STRONG> AD_01</STRONG></P></TD></TR><TR><TD width="246"><P><STRONG>AD_01_01</STRONG></P></TD><TD width="246"><P>1 µl</P></TD></TR><TR><TD width="246"><P><STRONG>2X KLD Reaction Buffer</STRONG></P></TD><TD width="246"><P>5 µl</P></TD></TR><TR><TD width="246"><P><STRONG>10X KLD Enzyme Mix</STRONG></P></TD><TD width="246"><P>1 µl</P></TD></TR><TR><TD width="246"><P><STRONG>Nuclease free H<SUB>2</SUB>O</STRONG></P></TD><TD width="246"><P>3 µl</P></TD></TR><TR><TD width="246"><P><STRONG>Totol</STRONG></P></TD><TD width="246"><P><STRONG>10 µl</STRONG></P></TD></TR></TBODY></TABLE><P> Incubated for 5 min at RT.</P><P> </P><H2>4.2       Results</H2><P>Expected Results:</P><P> </P><P>Sequencing results of the CpxR Promoter in front of the RFP show that the matagenesis worked and that there ane no muations in the sequence of interest:</P><P> </P><H1>1        Experiment Plate reader with the repaired BBa_K339007</H1><P>The aim of this experiment is to verifie if the biobrick workes the way it claimes to work</P><H2>1.1       Material and Methods</H2><P>96-Well plate</P><P>Cells with the BBa_K339007</P><P>DH5<STRONG>α</STRONG></P><P>PBS (1X)</P><P>KCl 80 mM</P><P>KCl 160 mM</P><P>10% Ethanol</P><P>20% Ethanol</P><P> </P><P>After centrifuging the cells they where resuspended in PBS and then in each well of the plate there was put 50 µl of either cell type. The wells where then filled up with either kind of liquid (50 µl):</P><P><STRONG> </STRONG><STRONG> </STRONG></P><TABLE><TBODY><TR><TD width="121"><P><STRONG>Cells</STRONG></P></TD><TD width="119"><P><STRONG>Name</STRONG></P></TD><TD width="119"><P><STRONG>Fill up (50 µl)</STRONG></P></TD><TD width="119"><P><STRONG>Stress</STRONG></P></TD></TR><TR><TD width="121"><P>BBa_K339007 transformed</P></TD><TD width="119"><P>RFP -</P></TD><TD width="119"><P>PBS</P></TD><TD width="119"><P>No stress</P></TD></TR><TR><TD width="121"><P>BBa_K339007 transformed</P></TD><TD width="119"><P>RFP + 40 mM KCl</P></TD><TD width="119"><P>80 mM KCl</P></TD><TD width="119"><P>40 mM KCl</P></TD></TR><TR><TD width="121"><P>BBa_K339007 transformed</P></TD><TD width="119"><P>RFP + 80 mM KCl</P></TD><TD width="119"><P>160 mM KCl</P></TD><TD width="119"><P>80 mM KCl</P></TD></TR><TR><TD width="121"><P>BBa_K339007 transformed</P></TD><TD width="119"><P>RFP + 5% Ethanol</P></TD><TD width="119"><P>10 % Ethanol</P></TD><TD width="119"><P>5 % Ethanol</P></TD></TR><TR><TD width="121"><P>BBa_K339007 transformed</P></TD><TD width="119"><P>RFP + 10% Ethanol</P></TD><TD width="119"><P>20 % Ethanol</P></TD><TD width="119"><P>10 % Ethanol</P></TD></TR><TR><TD width="121"><P>DH5<STRONG>α</STRONG></P></TD><TD width="119"><P>Neg -</P></TD><TD width="119"><P>PBS</P></TD><TD width="119"><P>No stress</P></TD></TR><TR><TD width="121"><P>DH5<STRONG>α</STRONG></P></TD><TD width="119"><P>Neg + 40 mM KCl</P></TD><TD width="119"><P>80 mM KCl</P></TD><TD width="119"><P>40 mM KCl</P></TD></TR><TR><TD width="121"><P>DH5<STRONG>α</STRONG></P></TD><TD width="119"><P>Neg + 80 mM KCl</P></TD><TD width="119"><P>160 mM KCl</P></TD><TD width="119"><P>80 mM KCl</P></TD></TR><TR><TD width="121"><P>DH5<STRONG>α</STRONG></P></TD><TD width="119"><P>Neg + 5% Ethanol</P></TD><TD width="119"><P>10 % Ethanol</P></TD><TD width="119"><P>5 % Ethanol</P></TD></TR><TR><TD width="121"><P>DH5<STRONG>α</STRONG></P></TD><TD width="119"><P>Neg + 10% Ethanol</P></TD><TD width="119"><P>20 % Ethanol</P></TD><TD width="119"><P>10 % Ethanol</P></TD></TR></TBODY></TABLE><P><STRONG> </STRONG></P><P>The plate was then put in the plate reader where for 3h every 5 min the OD and the fluorescence was measured (exitation: 555 nm ; emission : 632)</P><P> </P><H2>1.2       Results</H2><P> </P><P>Neg stands for negative control (cells lacking the RFP coding sequence).</P><P> </P><H2>1.3       Interpretation</H2><P>We successfully improved the BBa_K339007 by putting the missing sequence in front of the RFP. With the further documentation we discovered that the promoter is not specific for dimerised CpxR.</P><P> </P><H1>2        Experiment. PCR Amplifie the Backbone for the Gibson</H1><P>To obtain the necessary backbone for the assembly.</P><P> </P><H2>2.1       Material and methods</H2><P> </P><P>Template :         AD_01</P><P>Primers :                  AD_01_00_rv</P><P>                               NH_04_00_fw</P><P>Phusion PCR kit</P><P> </P><TABLE width="388"><TBODY><TR><TD width="126"><P><STRONG> </STRONG></P></TD><TD width="142"><P><STRONG>1</STRONG></P></TD><TD width="121"><P><STRONG>2</STRONG></P></TD></TR><TR><TD width="126"><P><STRONG>AD_01_00</STRONG></P></TD><TD width="142"><P>0.1 µl</P></TD><TD width="121"><P>0.1 µl</P></TD></TR><TR><TD width="126"><P><STRONG>NH_04_00_fw</STRONG></P></TD><TD width="142"><P>2.5 µl</P></TD><TD width="121"><P>2.5 µl</P></TD></TR><TR><TD width="126"><P><STRONG>AD_01_00_rv</STRONG></P></TD><TD width="142"><P>2.5 µl</P></TD><TD width="121"><P>2.5 µl</P></TD></TR><TR><TD width="126"><P><STRONG>Mastermix</STRONG></P></TD><TD width="142"><P>11.5 µl</P></TD><TD width="121"><P>11.5 µl</P></TD></TR><TR><TD width="126"><P><STRONG>H<SUB>2</SUB>O</STRONG></P></TD><TD width="142"><P>33.4 µl</P></TD><TD width="121"><P>33.4 µl</P></TD></TR><TR><TD width="126"><P><STRONG>Total volume</STRONG></P></TD><TD width="142"><P>50 µl</P></TD><TD width="121"><P>50 µl</P></TD></TR></TBODY></TABLE><P> </P><P>PCR program :</P><P>       1:         98 °C               30 s</P><P>       2:         98 °C               10 s</P><P>       3:         69 °C               30 s</P><P>       4:         72 °C               2 min              2          29</P><P>       5:         72 °C               10 min</P><P> </P><H1>3        Experiment 6 : Gibson assembly</H1><P>To obtain a plasmid with RFP wiht the native promoter that is found in front of the CpxA and the CpxP</P><P> </P><H2>3.1       Materials and Methodes</H2><P>Fragments :             NH_04_00 (sample 1)</P><P>                               NH_04_01</P><P> </P><TABLE><TBODY><TR><TD width="95"><P><STRONG> </STRONG></P></TD><TD width="95"><P><STRONG>Moles</STRONG></P></TD><TD width="95"><P><STRONG>Mass </STRONG></P></TD><TD width="95"><P><STRONG>NH_04_A</STRONG></P></TD><TD width="95"><P><STRONG>NH_04_B</STRONG></P></TD></TR><TR><TD width="95"><P><STRONG>NH_04_00</STRONG></P></TD><TD width="95"><P>0.03 pmol</P></TD><TD width="95"><P>54.0 ng</P></TD><TD width="95"><P>0.72 µl</P></TD><TD width="95"><P>0.72 µl</P></TD></TR><TR><TD width="95"><P><STRONG>NH_04_01</STRONG></P></TD><TD width="95"><P>0.05 pmol</P></TD><TD width="95"><P>66.7 ng</P></TD><TD width="95"><P>3.3 µl</P></TD><TD width="95"><P>---</P></TD></TR><TR><TD width="95"><P><STRONG>NH_04_02</STRONG></P></TD><TD width="95"><P>0.05 pmol</P></TD><TD width="95"><P>66.7 ng</P></TD><TD width="95"><P>---</P></TD><TD width="95"><P>3.3 µl</P></TD></TR><TR><TD width="95"><P><STRONG>H<SUB>2</SUB>O</STRONG></P></TD><TD width="95"><P> </P></TD><TD width="95"><P> </P></TD><TD width="95"><P>5.98 µl</P></TD><TD width="95"><P>5.98 µl</P></TD></TR><TR><TD width="95"><P><STRONG>Gibson Master Mix</STRONG></P></TD><TD width="95"><P> </P></TD><TD width="95"><P> </P></TD><TD width="95"><P>10 µl</P></TD><TD width="95"><P>10µl</P></TD></TR><TR><TD width="95"><P><STRONG>Total</STRONG></P></TD><TD width="95"><P><STRONG> </STRONG></P></TD><TD width="95"><P><STRONG> </STRONG></P></TD><TD width="95"><P><STRONG>20 µl</STRONG></P></TD><TD width="95"><P><STRONG>20 µl</STRONG></P></TD></TR></TBODY></TABLE><P> </P><P>Incubated samples at 50°C for 15 min according to the NEB protocol</P><P> </P><H2>3.2       Results</H2><P>The resulting plasmids are the BBa_K1486049 (NH_04_00 + NH_04_01) and the BBa_K1486050 (NH_04_00 + NH_04_01)</P><P>Sequencing reveald that there where no mutations.</P><P> </P><P> </P><P> </P><P> </P><P> </P><H1>Characterisation of BBa_K1486049 and BBa_K1486050</H1><P>To charcterise theyre behavior in response to stress those two biobricks and the BBa_K1486048 were put in PBS (negative control) and in PBS + 80 mM KCl (stress). The samples were loaded on a 96-well plate reader. The light emission (ex: 555 nm em: 632 nm) and the OD was measured during several houers.</P><H2>Results</H2><P>As we can see, there is no stress sensitivity. The improvement we made is that our promoter is no &quot;leaky&quot;.</P><P> </P><P> </P><H1>Primer librery</H1><TABLE><TBODY><TR><TD width="112"><P><STRONG>Primer name</STRONG></P></TD><TD width="367"><P><STRONG>Sequence (5’ -&gt; 3’)</STRONG></P></TD></TR><TR><TD width="112"><P>AD_01_00_fw</P></TD><TD width="367"><P>AAAGAGGAGAAATACTAGATGGCTTCCTCC</P></TD></TR><TR><TD width="112"><P>AD_01_00_rv</P></TD><TD width="367"><P>CTCTAGAAGCGGCCGCGA</P></TD></TR><TR><TD width="112"><P>AD_01_01_fw</P></TD><TD width="367"><P>atgacgtaatttctgcctctactagagAAAGAGGAGAAATA</P><P>CTAGATGGCTTCCTCC</P></TD></TR><TR><TD width="112"><P>AD_01_01_rv</P></TD><TD width="367"><P>cagacgtcgctaatccatgactttacgttgttttacCTCTAGAAGCGGCCGCGA</P></TD></TR><TR><TD width="112"><P>NH_04_00</P></TD><TD width="367"><P>ATGGCTTCCTCCgaagacgttatc</P></TD></TR></TBODY></TABLE><P> </P><H1>Ordered sequences</H1><TABLE><TBODY><TR><TD width="112"><P><STRONG>Name </STRONG></P></TD><TD width="367"><P><STRONG>Sequence (5’ -&gt; 3’)</STRONG></P></TD></TR><TR><TD width="112"><P>NH_04_01</P></TD><TD width="367"><P>atgatttctggaatTCGCGGCCGCTTCTAGAGTTGCTCCCAAAATCTTTC</P><P>TGTCGCGATTCAACGATAGAGAGTTTACGATTCAGGCTGCAAACA</P><P>TGCGTCAGGGGGTGTAAAACAACGTAAAGTCATGGATTAGCGACG</P><P>TCTGATGACGTAATTTCTGCCTCGGAGGTATTTAAACAATGGCTTCC</P><P>TCCgaagacgttatcaaagagttca</P></TD></TR><TR><TD width="112"><P>NH_04_02</P></TD><TD width="367"><P>atgatttctggaatTCGCGGCCGCTTCTAGAGTGTTTAAATACCTCCGAGG</P><P>CAGAAATTACGTCATCAGACGTCGCTAATCCATGACTTTACGTTGTT</P><P>TTACACCCCCTGACGCATGTTTGCAGCCTGAATCGTAAACTCTCTAT</P><P>CGTTGAATCGCGACAGAAAGATTTTGGGAGCAAATGGCTTCCTCCga</P><P>agacgttatcaaagagt</P></TD></TR></TBODY></TABLE><P> </P><P> </P><P> </P><P> </P><H3> </H3></DIV></DIV></DIV><DIV class="col col-md-3"><NAV id="affix-nav" class="sidebar hidden-sm hidden-xs"><UL class="nav sidenav box sidenav-hidden" data-spy="affix" data-offset-top="200" data-offset-bottom="600"><LI class="active"><A href="#date-07-07-2014">July</A></LI><LI class="active"><A href="#date-07-07-2014">07-07-2014</A></LI><LI><A href="#date-07-13-2014">07-13-2014</A></LI><LI><A href="#date-07-20-2014">07-20-2014</A></LI><LI><A href="#date-07-21-2014">07-21-2014</A></LI><LI><A href="#date-07-27-2014">07-27-2014</A></LI><LI><A href="#date-07-28-2014">07-28-2014</A></LI></UL><LI><A href="#date-08-03-2014">August</A></LI><LI><A href="#date-08-03-2014">08-03-2014</A></LI><LI><A href="#date-08-10-2014">08-10-2014</A></LI><LI><A href="#date-08-12-2014">08-12-2014</A></LI><LI><A href="#date-08-17-2014">08-17-2014</A></LI><LI><A href="#date-08-24-2014">08-24-2014</A></LI><LI><A href="#date-09-16-2014">September</A></LI><LI><A href="#date-09-16-2014">09-16-2014</A></LI></NAV></DIV></DIV></DIV><DIV id="contactSection"><DIV><H1 class="cntr">Sponsors</H1></DIV><DIV class="footerSection container"><DIV class="copyright"><P>Copyright © iGEM EPFL 2014. All Rights Reserved</P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:EPF_Lausanne/Notebook/Bacteria">http://2014.igem.org/Team:EPF_Lausanne/Notebook/Bacteria</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:EPF_Lausanne/Notebook/Bacteria" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:EPF_Lausanne/Notebook/Bacteria" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:EPF_Lausanne/Notebook/Bacteria&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:EPF_Lausanne/Notebook/Bacteria&amp;oldid=330941" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>