<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_EPF_Lausanne_Applications"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:EPF_Lausanne/Applications">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:EPF_Lausanne/Applications&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:EPF_Lausanne/Applications&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:EPF_Lausanne/Applications&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:EPF_Lausanne/Applications" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:EPF Lausanne/Applications</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><DIV class="nav-collapse"><UL class="nav"><LI><A href="https://2014.igem.org/Team:EPF_Lausanne">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle active" data-toggle="dropdown">Project </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Overview">Overview</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Envelope_stress_responsive_bacteria">Stress Responsive Bacteria</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Yeast">Osmo Responsive Yeast</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Microfluidics">Microfluidics</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Hardware">Hardware</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Applications" class="active">Applications</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Achievements </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Data">Data</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Judging">Judging</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Parts">Parts</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Policy &amp; Practice </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/HumanPractice">Human Practice</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Safety">Bio Safety</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/PolicyPractice">Metafluidics</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Notebook </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Notebook/Bacteria">Bacteria</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Notebook/Yeast">Yeast</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Notebook/Microfluidics">Microfluidics</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Protocol">Protocols</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Notebook">Timeline</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Team">Meet us!</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Acknowledgments">Acknowledgments</A></LI></DIV></UL></DIV></DIV><DIV class="container"><DIV class="box" id="boxbread"><OL class="breadcrumb breadcrumb-arrow"><LI><A href="https://2014.igem.org/Team:EPF_Lausanne"> Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Project </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Overview">Overview</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Envelope_stress_responsive_bacteria">Stress Responsive Bacteria</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Yeast">Osmo Responsive Yeast</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Microfluidics">Microfluidics</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Hardware">Hardware</A></LI><LI class="active"><SPAN> Applications</SPAN></LI></OL></DIV><DIV class="row"><DIV class="col col-md-9"><DIV class="whitebg box"><H1 class="cntr"><B> Applications</B></H1><H1 id="basicscience">Basic Sciences Related</H1><H2 id="protein">Protein complementation &amp; biosensors</H2><P>Biosensors have been extensively characterized in previous iGEM projects. They consist of cells engineered to respond to a given stimulus through an intracellular signalization cascade, which often results in the transcription of a reporter gene. In most cases, the corresponding reporter proteins are fluorescent or luminescent and their synthesis leads to a measurable output several hours after the presence of the stimulus<SUP><A href="#ref1">1</A></SUP>.

</P><P>Our team aims to alter this way of thinking and make biosensor signal measurement fast! The key to this relies in the use of protein complementation techinques. Protein complementation techniques allow the fusion of complementary fragments of a split reporter protein to proteins of interest that interact together at a certain point. When the interaction happens, the fused reporter fragments are physically brought together, which reconstitutes the activity of the reporter protein. By choosing two proteins which interaction is caused by a certain stimulus, one engineers a biosensor that responds much faster than traditional transcriptional systems – in the range of minutes rather than hours! Indeed, the reporter protein does not need to be synthesized but is already present in the cell and can activate as soon as its full structure is reconstituted.</P><P>In research, protein complementation studies are mostly used to validate protein interaction in the context of signal cascades and other pathways<SUP><A href="#ref1">2</A></SUP>. We think that their use as reporter systems in biosensors can have a broad range of applications in which the reporter signal needs to be measured quickly. One can think for example of water safety biosensors, in which one wants to know if water is contaminated or not before drinking it.</P><P>The great advantage of such a system is its modularity as it can be adapted to potentially any stimulus-responsive pathway in which two proteins are to interact at a certain point. It is also possible to choose between several split reporter proteins. The ones existing so far are fluorescent proteins – such as GFP, YFP or IFP – and luminescent proteins – such as Renilla or Firefly luciferases. But one could also imagine engineering split chromoproteins  to have a device that changes colour straight away in response to a certain stimuli.</P><H2 id="microfluidics">Microfluidics as an interface for the study of biosensors</H2><P>
Microfluidics can be a powerful tool to implement biosensor studies at the laboratory scale as well as for their development as commercial devices.</P><P>At the laboratory scale, microfluidics allows one to perform tests on several different modified organisms in parallel. One can then select the cells that have the best response to a given stimuli and therefore develop a robust biosensor. The <A target="_blank" href="https://2014.igem.org/Team:EPF_Lausanne/Microfluidics/Designing#filter">new chip</A> that we designed also allows one to replace the medium during the experiment. By successively flowing medium containing or not the stimulus of interest, one can then analyse the on and off-response of the biosensor.</P><P>Microfluidics is also very interesting as one thinks of a final device containing biosensors. As several different odified organisms can be loaded in the chip, it is possible to detect various stimuli at the same time. If we elaborate on the water safety problem, one can imagine a device that allows the user to analyse for several contaminants with only a small sample of water. On top of that, having biosensors contained in an enclosed environment is a very interesting feature as far as <A target="_blank" href="https://2014.igem.org/Team:EPF_Lausanne/Safety">biosafety</A> is concerned.
</P><H2 id="transcription">Study of the transcription time</H2><P>
Our project also introduces a new way of studying the relationship between gene transcription and the corresponding activating signal. The idea would be to combine the fusion of split complementary fragments to dimerizing transcription factors, and the introduction of reporter constructs under promoters sensitive to the same transcription factor. One could study the relationship between these two signals and notably measure the time elapsed between binding of the transcription factor to the promoter and effective transcription of the reporter gene. Especially in yeast, induction of genes is much less understood than in bacterial cells and much more difficult. Our system would be very useful in controlling the intracellular environment of yeast cells, which would help the advance of synthetic biology in non-bacterial cells.</P><H2 id="hogpathway">Enlightening the HOG pathway, but not only!</H2><P>The yeast part of our project is one of the first to study the HOG pathway directly with split complementation. Our PBS2 and HOG1 tags are thought to allow extensive characterization of the pathway in a very direct manner. The MAPKK-MAPK interaction could be quantified in terms of reaction time and sensibility to different stresses and the results would probably apply to many other pathways. Using our system, the pathway activation can indeed be triggered and verified very easily using the split luciferase strains, while localization of the interacting proteins can be observed with the GFP tag strains and single cell imaging.</P><P>Furthermore, the PBS2-HOG1 pathway is analogous to another pathway in mammalian cells<SUP><A href="#ref1">4</A></SUP>, which involves the MEK5 and ERK5 proteins. Fluorescent mice are a thing of the past, we would now have touch-sensitive luminescent mice! While that might be a bit extreme as a potential application, expanding the project to mammalian cells using homologous recombination in mouse cells is not as far-fetched as it sounds. It would give an indication on how mammalian cells sense different stresses, since we could test them separately in an in vitro environment. This would be very useful since it is not yet clear how each pathway is regulated depending on different conditions and our system would provide a basis for mass screening.</P><H1 id="appliedsciences">Applied Sciences Related</H1><H2 id="screening"> Fast, efficient and accurate antibiotic screening system</H2><P>

The 2014 EPF Lausanne iGEM team engineered the CpxA-R pathway to develop its BioPad. The CpxA-R pathway responds to periplasmic stress via the presence of misfolded/aggregated proteins in the periplasm. Our team hypothesised that the presence of antibiotics would lead to a certain degree of protein misfolding/aggregation and thus would activate the signal. Since our device emits fluorescence upon periplasmic stress, our system could be used to quantify the strength of an antibiotic in a CpxA-R dependent manner. Combined to a microfluidic chip, this system could provide the scientific community with a cheap, fast, efficient, and accurate antibiotic screening system. This could result in easily quantifiable high-throughput screenings for antibiotic candidates.</P><H2 id="antibiotic">Antibiotic Complement</H2><P>

Bacterial envelopes are often remodeled when encountering hosts. These changes lead to the synthesis of complex envelope structures that are important virulence factors. Improper assembly of these structures can harm the bacterial envelope and lead to Extracytosolic Stress. Bacteria counter the potential envelope stresses by downregulating these virulence factors. The CpxA-R pathway, used by the 2014 EPF Lausanne, turns out to be envolved in this process. In fact when turned on, the CpxA-R pathway activates a bacterial survival response which among other things, down regulates the biogenesis of complex surface virulence factors such as pili/fimbiae and type III and type IV secretion systems. A paper supporting this hypothesis has recently been published on the 6th of October. 

</P><P>
Given this information, our device could lead to the discovery of an &quot;antibiotic complement&quot; enabling the removal of virulence factors from pathogenic bacteria to improve the efficiency of current antibiotics. Our vision of this novel form of antibiotic would be as a supplement to be taken with current antibiotics to improve the efficiency of treatment. Take a look at the paper <A target="_blank" href="http://www.jbc.org/content/early/2014/10/06/jbc.M114.565762.short"> here </A>.</P><H2 id="tumor">Tumor progression evaluation</H2><P>

A possible application of the organisms developped through this project would be related to cancer. In modern research, in vivo tumor progression in experimental animals is fairly difficult to evaluate: most scientists rely on the size of a tumor to get an idea of how developed it is. Our idea would be to integrate our engineered organisms within the tumor's cellular matrix (Matrigel) to allow researchers to be able to assess the progression of tumors by how much fluorescence the tumor emits. This would allow scientists to reduce unnecessary animal sacrifices in tumor research.</P><H2 id="tumorsensing">Tumor sensing</H2><P>Further expanding on the mammalian application for the yeast part of our project, we could potentially create stress-sensing mice. As mentioned above, detection of a tumor is not the simplest of tasks. We would apply our system to whole mice, meaning that all their cells would be stress-sensors. At the injection of coelenterazine, the substrate for the Renilla Luciferase, the most stressed cells would luminesce meaning that we would easily detect the most stressed regions of the animal in question without hurting the animal or affecting its metabolism.</P><H3 id="references"> References </H3><P><OL><LI><A target="_blank" href="http://www.nature.com/nrmicro/journal/v8/n7/pdf/nrmicro2392.pdf">Van der Meer, J. R., &amp; Belkin, S. (2010). Where microbiology meets microengineering: design and applications of reporter bacteria. Nature Reviews Microbiology, 8(7), 511-522.</A></LI><LI><A target="_blank" href="http://www.sciencedirect.com/science/article/pii/S0014579309001665">Morell, M., Ventura, S., &amp; Avilés, F. X. (2009). Protein complementation assays: approaches for the in vivo analysis of protein interactions. FEBS letters, 583(11), 1684-1691.</A></LI><LI><A target="_blank" href="http://www.readcube.com/articles/10.1038/nmeth.2934">Michnick, S., Tchekanda, E., &amp; Sivanesan, D. (2014, April 20). An infrared reporter to detect spatiotemporal dynamics of protein-protein interactions. <I>Nature Methods</I>, 6-6</A></LI><LI><A target="_blank" href="http://www.nature.com/cr/journal/v12/n1/pdf/7290105a.pdf">Zhang, W., &amp; Liu, H. T. (2002). MAPK signal pathways in the regulation of cell proliferation in mammalian cells. Cell research, 12(1), 9-18.</A></LI></OL></P></DIV></DIV><DIV class="col col-md-3"><NAV id="affix-nav" class="sidebar hidden-sm hidden-xs"><UL class="nav sidenav box" data-spy="affix" data-offset-top="200" data-offset-bottom="600"><LI class="active"><A href="#basicscience">Basic Sciences Related</A></LI><LI><A href="#protein">Protein complementation &amp; biosensors</A></LI><LI><A href="#microfluidics">Microfluidics as an interface</A></LI><LI><A href="#transcription">Study of the transcription time</A></LI><LI><A href="#hogpathway">Enlightening the HOG pathway</A></LI></UL><LI class="active"><A href="#appliedsciences">Applied Sciences Related</A></LI><LI><A href="#screening">Antibiotic screening system</A></LI><LI><A href="#antibiotic">Antibiotic Complement</A></LI><LI><A href="#tumor">Tumor progression evaluation</A></LI><LI><A href="#tumorsensing">Tumor sensing</A></LI><LI class="active"><A href="#references">References</A></LI></NAV></DIV></DIV></DIV><DIV id="contactSection"><DIV><H1 class="cntr">Sponsors</H1></DIV><DIV class="footerSection container"><DIV class="copyright"><P>Copyright © iGEM EPFL 2014. All Rights Reserved</P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:EPF_Lausanne/Applications">http://2014.igem.org/Team:EPF_Lausanne/Applications</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:EPF_Lausanne/Applications" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:EPF_Lausanne/Applications" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:EPF_Lausanne/Applications&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:EPF_Lausanne/Applications&amp;oldid=396106" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>