"
Page
Discussion
View source
History
teams
Log in
 
Team:EPF Lausanne/Notebook/Bacteria
From 2014.igem.org
Toggle navigation
Home
Project
Overview
Stress Responsive Bacteria
Osmo Responsive Yeast
Microfluidics
Hardware
Applications
Achievements
Results
Data
Judging
Parts
Policy & Practices
Human Practices
Bio Safety
Metafluidics
Notebook
Bacteria
Yeast
Microfluidics
Protocols
Team
Timeline
Meet us!
Attributions
Acknowledgments
Home
Notebook
Yeast
Microfluidics
Protocols
Bacteria
Split IFP & GFP  - Week 1
07-07-2014
Split IFP & GFP  - Week 1
1        CPXR extraction
Extraction of CPXR from genome of E. Coli strain K-12 MG1655
 
1.1       Material and methods
NEB Phusion HF PCR Protocol (see protocol), with corresponding primers (cf primers datasheet) for amplification. 
1.2       Results
1.2.1     Data
 
1.2.2     Interpretation
CPXR expected size : 699 bp
Amplification of CPXR worked as expected.
2        PCR for amplification of iGEM plasmid pSB1C3 and for addition of overlapps to CPXR
Amplification of pSB1C1 backbone from Cambridge biobrick K325219. This backbone contains the RBS, an arabinose-inducible pBAD promoter, iGEM prefix (containing ECORI and xbaI restriction sites) and suffix (contaning SpeI and PstI restriction sites), a chloramphenicol resistance gene, and an origin of replicaiton. Additon of overlapp to CPXR in order to make a Gisbon assembly of the backbone and the CPXR
2.1       Material and methods
NEB Phusion PCR and suitable primers.
2.2       Results
2.2.1     Data
 
 
2.2.2     Interpretation
We obtained the expected amplicon, and are now able to make a Gibson assembly to fuse the backbone amplicon and the CPXR.
Split IFP & GFP - Week 2
07-13-2014
 
1      Gibson assembly and colony PCR of pCPXR
Fusion of pSB1C3 backbone and CPXR
 
1.1       Material and methods
Gibson assembly kit (see protocol) for the Gibson assembly, and Taq polymerase for colony PCR (see protocol)
 
1.2       Results
 
1.2.1     Data
Colonies were obtained and analyzed by colony PCR, using VF2 and VR primers from iGEM
 
 
1.2.2     Interpretation
The amplicons have the expected size, meaning the Gibson assmbly worked and that we very likely obtained pCPXR (plasmid with pSB1C3 backbone and CPXR). Sequencing will reveal if the plasmid contain any mutation.
 
2        Sequencing of pCPXR
Confirmation of pCPXR sequence with Microsynth sequencing services.
2.1       Material and methods
We sent the plasmid with a concentration of 80 ng/μl and with premixed primers VR and VF2
2.2       Results
 
2.2.1     Data
We obtained the right sequence, excepted one deletion of a cytosine in the ara promoter at the position -41. This deletion appeared to be originally in the backbone pSB1C1 that we amplified from Cambridge biobrick K325219, without consequence on the transcription of the coding sequence.
 
3     Amplification of IFP1, IFP2 and pCPXR with suitable overlaps to fuse the split at the C or N terminal of CPXR
PCR amplification of fragments to fuse IFP1 and IFP2 at the C or N terminal of CPXR in pCPXR plasmid
3.1       Material and methods
NEB Phusion kit (see protocol) and suitable primers (see list). Template come from Michnick lab IFP PCA.
3.2       Results
3.2.1     Data
3.2.2     Interpretation
IFP1 C didn't work but the other ones have the expected size. We are going to try again IFP1 C in a gradient PCR
 
4        PCR gradient to obtain IFP1 C fragment
Amplification of IFP1 with overlap in order to insert it in pCPXR at the C terminal of CPXR
4.1       Material and methods
Gradient temperature was performed as the PCR didn’t work the first time. NEB Phusion kit, same procedure as last time
4.2       Results
4.2.1     Data
4.2.2     Interpretation
We finally obtain the amplicon, for each temperature.
 
CheY CheZ - Week 2
07-13-2014
Experiment 1: Transformation
Transform competent cells with plasmids recovered from Michnick’s and Waldor’s labs
 
1.1 Material and methods
14/07/14
 
Preparation of SOC: 3 x (20µl of glucose 20mM in 1ml SOB)
 
Transformation Protocol from iGEM HQ with 1 uL DNA in each Ependorf
 
Ependorf number
Content
Comments
1, 2
pYNZC (34.8 ng/uL)
Waldor, CamR
3, 4
prLucNrLucC (33 ng/uL)
Waldor, CamR
5, 6
prLuc (30 ng/uL)
Waldor, CamR
7, 8
pAG25-L-IFP [1]
Michnick, AmpR
9, 10
pAG32-L-IFP [2]
Michnick, AmpR
11, 12
10pg/ul RFP Control (pSB1A3 w/ BBa_J04450)
AmpR
 
Incubation times: 2h shaking at 37°C / 180 rpm and 2h resting on ice.
Plates set for incubation overnight at 37°C
 
15/07/14
 
Inoculated cultures in LB Chloramphenicol or Ampicillin (3ml)
Tubes left in the fridge all day
Incubation overnight at 37°C / 180 rpm
 
16/07/14
 
OD measurement
1.2 Results
1.2.1 Data
OD: 1. pYZNC A: 2.046
2. pYZNC B: 2.033
3. prLucNrLucC A: failed (overcrowded plate)
4. prLucNrLucC B: 2.072
5. prLuc A: 2.028
6. prLuc B: 2.055
7. pIFP 1 A: 1.950
8. pIFP 1 B: 1.953
9. pIFP 2 A: 1.954
10. pIFP 2 B :1.948
1.2.2 Interpretation
Tube 3 discarded, others used for miniprep and glycerol stock preparation
Experiment 2: Bioluminescence assay
Make a bioluminescence assay with the previously prepared cultures
2.1Material and methods
17.07.14:
Overnight cultures (3ml LB + chloramphenicol) inoculated with cells from a frozen glycerol stock from previously transformed competent cells (pYNZC A & B, prLucNrLucC B, prLuc A & B) were incubated at 37°C with shaking at 180 rpm). OD was measured the following day.
18.07.14:
The cultures were diluted 1:100 in 5mL fresh LB-Chloramphenicol containing 670ul of 100mM L-arabinose and grown for 3h on a rotary shaker to an OD 0.8-0.9. The cultures (500ul of each) were washed twice with phosphate buffered saline (PBS; pH 7), adjusted to an OD 0.4-0.5 (final volume of culture should be 1ml) and aliquoted into a white 96-well plates in triplicates.
Aliquots of native coelenterazine (clz) dissolved in ethanol (10mM) were stored at 80°C. Solution of glucose (30mM in bi-distilled water) were prepared and used at a concentration of 1mM. Prior to the assay, an aliquot of clz was thawed, diluted in PBS to a final concentration of 250uM, and incubated in the dark at room temperature (RT) for 1h. Clz was added to a final concentration of 7.5uM to the PBS-washed cells, which were subsequently incubated in the dark for 30 minutes at RT. The total luminescence of each well was then measured every two minute for a total of 10 min with an integration time of 1 sec using (WHICH MACHINE??). Immediatly thereafter, 10 ul of glucose solution was manually injected into each well and a second luminescence read was initiated as before.
 
2.2Results
2.2.1Data
 
 
2.2.2Interpretation
The measurements were taken too late. The substrate (clz) already started to be depleted. The positive control gave a very strong signal at the beginning, while the negative control gave a small or no signal. The construct itself gave a signal higher than the negative control but much lower than the negative control, meaning that the construct works but isn't optimal. The effect of the addition of glucose after 10 minutes can't be determined precisely. As there was no more clz in the wells after 6-7 minutes, the shut down of the luminescence observed in the construct wasn't necessarily due to glucose addition, as exprected. We cannot conclude on this experiment. We should start again and take measurements faster.
 
Competent cells
07-20-2014
07.07.14-13.07.14
 
Competent Cells
 
Protocol : Thermocompetent cells protocol
First try – Not much care, all resulting tubes were thrown away
Second try – Lesser volume, more care to cold, poured liquid nitrogen right into box for freezing.
 
Results : Tests seem to indicate that our method for transformation was simply not adapted to our competent cell protocol. We do not have the appropriate sterile cold room to produce high efficiency competent cells. Therefore, for biobricks and gibson assemblies, we need to use commercial competent cells. For transformations with plasmid constructs where the concentration is in the scale of 1 ng/ul or more our competent cells are usable.
 
CheY CheZ - Week 3
07-20-2014
 
1 Experiment 1: Bioluminescence assay
Make a bioluminescence assay with the cultures transformed with plasmids from Waldor's lab
1.1 Material and methods
22.07.14: Overnight cultures (3ml LB + chloramphenicol) inoculated with cells from a frozen glycerol stock from previously transformed competent cells (pYNZC, prLucNrLucC, prLuc) were incubated at 37°C with shaking at 180 rpm). OD was measured the following day.
23.07.14: The cultures were diluted 1:100 in 5mL fresh LB-Chloramphenicol containing 670ul of 100mM L-arabinose and grown for 3h on a rotary shaker to an OD 0.6-0.7. The cultures (1ml of each) were washed twice with phosphate buffered saline (PBS; pH 7), (final volume of culture should be 1ml) and aliquoted into a white 96-well plates in triplicates.
Aliquots of native coelenterazine (clz) dissolved in ethanol (10mM) were stored at 80°C. Prior to the assay, an aliquot of clz was thawed, diluted in PBS to a final concentration of 250uM, and incubated in the dark at room temperature (RT) for 1h. Clz was added to the PBS-washed cells, at different concentrations: 7.5uM and 75uM for pYNZC and rLucNrLucC, 7.5uM, 15uM, 75uM and 150uM for rLuc, for a total volume of 300ul in each well (see table below). Cells were subsequently incubated in the dark for 5 minutes at RT. The total luminescence of each well was then measured every three minutes for a total of 30 min with an integration time of 1 sec.
 
PYNZC
7.5uM cz
PYNZC
7.5uM cz
PYNZC
7.5uM cz
PYNZC
75uM cz
PYNZC
75uM cz
PYNZC
75uM cz
 
 
 
 
 
 
PrLNrLC
7.5uM cz
PrLNrLC
7.5uM cz
PrLNrLC
7.5uM cz
PrLNrLC
75uM cz
PrLNrLC
75uM cz
PrLNrLC
75uM cz
 
 
 
 
 
 
PrLuc
7.5uM cz
PrLuc
7.5uM cz
PrLuc
7.5uM cz
PrLuc
15uM cz
PrLuc
15uM cz
PrLuc
15uM cz
PrLuc
75uM cz
PrLuc
75uM cz
PrLuc
75uM cz
PrLuc
150uM cz
PrLuc
150uM cz
PrLuc
150uM cz
PBS
7.5uM cz
PBS
7.5uM cz
PBS
7.5uM cz
PBS
75uM cz
PBS
75uM cz
PBS
75uM cz
 
 
 
 
 
 
 
1.2Results
1.2.1Data
 
1.2.2Interpretation
After addition of coelenterazine to the cells in the plate, wells were full and the cultures may have spread between the wells. Our results aren't as expected, as the signal in the negative control is higher than in the pYNZC construct.
The measurements should be taken in an opaque 96-well plate, from the bottom of the plate. The wells shouldn't be filled completely to avoid spreading.
The concentration of bacteria should be taken in account to make the gradient of concentrations.
 
 
2 Experiment 2: PCR extraction of fragments of interest
 
Extraction of sequences of interest from plasmids to proceed to Gibson assembly of the final construct: CheYrLucNCheZrLucC.
 
2.1 Material and methods
PCR kit (NEB), see protocol from kit
Minipreps from following plasmids: CheY (Bba_K569017), CheZ (Bba_K629003), rLucN (YNZC, Waldor), rLucC (YNZC, Waldor)
 
Primers (TABLE)
 
Preparation of the tubes (in duplicates) for 50uL reaction:
[uL]
MasterMix*
For_primer
Rev_primer
Template DNA
Nuclease free Water
CheY
11.5
2.5
2.5
11.7
21.8
CheZ
11.5
2.5
2.5
3.4
30.1
RlucN
11.5
2.5
2.5
4.1
29.4
RLucC
11.5
2.5
2.5
4.1
29.4
 
*MasterMix: for one tube: 9uL HF buffer, 0.9uL dNTP, 0.5uL polymerase
 
Control tubes:
[uL]
MasterMix*
For_primer
Rev_primer
Template DNA
Nuclease free Water
CheY
11
2.5
2.5
11.7
22.3
CheZ
11
2.5
2.5
3.4
30.6
RlucN
11
2.5
2.5
4.1
29.9
RLucC
11
2.5
2.5
4.1
29.9
*Control MasterMix: for one tube: 9uL HF buffer, 0.9uL dNTP
 
PCR program according to Phusion Polymerase protocol
 
2.2 Material and methods
 
1.2.1 Data
 
Split IFP & GFP - Week 3
07-21-2014
 
1  Gibson assembly and colony PCR for IFP2-pCPXR (at the C and N terminal)
 
Assembly of IFP2 at the C ter or N ter of CPXR in pCPXR by Gibson assembly. Colony PCR is useful to check if competent cells were transformed by our expected Gibson Assembly product and not by the empty plasmid pCPXR.
We did the GA with PCR product of pCPXR opened with corresponding primers and amplified insert, but we didn’t get rid of the pCPXR DNA template so competent cells may have been transformed with empty vectors thus acquiring chloramphenicol resistance. This is why we obtained so many negative colonies. DPN1 digestion of our PCR product would have removed the plasmids template.
1.1       Material and methods
We used VR ans VF2 sequencing primers.
pick a colony and pouf pouf in 10ul tubes of water. Directly put the tip in 3ml of LB with chloramphenicol in culture tubes and put at 37°.
Boil the 10ul tubes of water containing the picked bacteria at 95° for 10 minutes. It will be your « DNA template ».
Prepare the PCR mix for Taq polymerase, with 1ul of DNA template (NEB Taq protocol)
 
 
 
1.2       Results
1.2.1     Data
 
 
    
 
We also loaded PCR product of pCPXR "opened" as negative control, in order to compare the size of our GA product and pCPXR and see if it contains the IFP2. Colonies that are thought to contain the insert IFP2 are:
IFP2 C terminal: 1, 4, 5, 6, 7, 9, 10, 11, 12
IFP2 N terminal: 1, 2, 3, 4, 6, 9, 12
1.2.2     Interpretation
By observing the difference of size between the negative control (PCR of pCPXR that doesn’t contain the insert) and the colony PCR fragment, we see that we obtained some colonies containing empty vectors and some that contains very likely IFP2 insert. Further anlyisis and sequencing will confirm it.
2        PCR amplification of IFP2 in the plasmid thought to contain IFP2
Amplification of IFP2 from IFP2-pCPXR plasmid, to confirm that our GA product contains the insert IFP2.
 
2.1       Material and methods
We used primers previously used to amplify and to add an overlap to IFP2 Cter and Nter respectively. As the « tail » of the primers were now annealing the plasmid, one could think that changing the Annealing temp was necessary but you can keep the same that you used the first time. Unfortunately some of the PCR tubes opend in the PCR machine so we couldn’t analyzed all of the plasmid.
2.2       Results
2.2.1     Data
 
 
2.2.2     Interpretation
All of the plasmids analysed contain the insert IFP2 and the size is correct (600 bp). Our GA worked, at least for the colonies that were analyzed on this gel. We are now ready for sequencing
3        Sequencing IFP2-pCPXR plasmid
To check if our GA products, pCPXR fused to IFP2 at its Cter or Nter don’t contain any mutation. We sent sample 9 for IFP2 Nter and sample 10 for C terminal.
3.1       Material and methods
Microsynth Barcode and a plasmid concentartion of 80 ng/ul
3.2       Results
3.2.1     Data
No mutation for IFP2 at the C terminal. One mutation in the RBS of IFP2 at the Nterminal, position -18 a A instead of a T. It shouldn't matter. 
 
4       
Gibson assembly and colony PCR for IFP1-pCPXR (at the C and N terminal)
Assembly of IFP1 at the C ter or N ter of CPXR in pCPXR by Gibson assembly. Colony PCR to check if competent cells were transformed by our expected Gibson Assembly product and not by the empty plasmid pCPXR.
4.1       Material and methods
(See above for protocol)
WARNING : We should always do a negative control for the colony PCR with Bacteria transformed with the empty vectors, not a PCR of an empty plasmid. Otherwise you cannot make any conclusion if the colony PCR fails or if the Gibson Assembly fails. You can compare the colony PCR quality of an expected product (pCPXR) and colony PCR of your new Gibson Assembly.
 
4.2       Results
4.2.1     Data
 
 
 
 
In blue: IFP1 at the C terminal. In red: IFP2 at the N terminal
 
4.2.2     Interpretation
Colonies that are thought to contain IFP1:
IFP1 at the C terminal: all
IFP1 at the N terminal: 3, 4, 6, 7, 8, 9, 10
  
5        PCR amplification of IFP1 in IFP1-pCPXR plasmid (after mini-prep)
 
Colony PCR was not really conculsive so we decided to min-prep all the colonies and analyse the obtained plasmids.
 
5.1       Material and methods
IFP1 amplification of the obtained plasmid was done with primers previously used to add to IFP1 the overlapp (IFP1_Cter and IFP1_Nter). We used Taq polymerase.
5.2       Results
5.2.1     Data
IFP1 amplification from colonies supposed to have the plasmid pCPXR with IFP1 at its Nterminal:
 
 
IFP1 amplification from colonies supposed to have the plasmid pCPXR with IFP1 at its Cterminal.
 
 
Expected size: 484 bp which correspond to our results.
 
5.2.2     Interpretation
Amplicon expected size is 463 bp, which correspond to our results. Colonie 5 (N terminal) apparently doesn’t have IFP1, the other ones do contain the expected amplicon.  “+” means that we amplified IFP1 from the plasmid known to contain this sequence. We also obtained a band at 850 bp for all the colonies that have the IFP1 fragment. Interestingly we also obtained this band in the positive control.
We very likely contain pIFP1-CPXR and pCPXR-IFP1 and further sequencing will confirm it.
Split IFP & GFP - Week 4
07-27-2014
 
1        Restriction Analysis of IFP1 Cter and Nter
 
To check if the plasmid contains IFP1 in our Gibson Assembly product, by digesting the plasmid with NheI and SpeI and see if we obtain the expected sizes.
  
1.1       Material and methods
500 ng of DNA (plasmid)
0.5 ul of SpeI and 0.5 ul NheI (should be added at the end)
5 ul of cut smart buffer
Add water to 50 ul
 
Incubate for one hour at 37°
Load on gel
 
1.2       Results
1.2.1     Data
 
Plasmid that have been digested in 2 fragments of size 1169 bp and 2773 bp contains a fragment, which is likely IFP1 (expected size). First gel reveals that all the plasmid supposed to contain IFP1 at the Cterminal of CPXR likely have IFP1. The second gel reveals that all colonies exept the 1st one likely contains IFP1 at the Nterminal. Further sequencing will confirm it.
2        IFP1 Cter and Nter sequencing
To check if the inserts IFP1 Cter and Nter, IFP2 Cter and Nter are present in the Gibson Assembly products (pCPXR fused by the insert below)
2.1       Material and methods
We will use the primer pBad For provided by mycrosynth (standard primer list) which will allow the sequencing from the end of the arabinose promoter. We will use also the VR primer to sequence Reverse from the suffix
 
IFP1 Cter : 10 (90 ng/ul)
IFP1 Nter : 9 (94 ng/ul)
2.2       Results
2.2.1     Data
 9   IFP1 Nter : No deletion 
10   IFP1 Cter : one deletion in the linker, doesn’t change the reading frame of IFP1
 
 
Expected sequence                                      Our sequence
GGG TCC TCC                                     GGA ---- TCC
Glycine Ser Ser                                   Gylcine----Serine
 
 
2.2.2     Interpretation
The deletion in the linker doesn't shift the reading frame. One Serine is missing, meaning the linker will certainly not be as flexible as it should be between the split IFP and CPXR. We decided to keep working on this construct, because it shouldn't alterate too much the construct. 
 
 
Split IFP & GFP - Week 5
07-27-2014
1        PCR of pCPXR-IFP1 and pCPXR-IFP2 to put both fragment in the same vector
 
To do a new plasmid containing our two fragments of interest, CPXR-IFP1 and CPXR-IFP2 into each configuration (IFP1 at the C or N terminal of CPXR and IFP2 at the C or Nterminal of CPXR) to avoid co-transformation during further experiment.
1.1       Material and methods
Phusion PCR with a temperature gradient (as our two first trial failed completely). Gradient temperature was achieved from 60° to 70°, the theorical temperature being around 70°. pCPXR-IFP2 will be entirely amplified and IFP1-CPXR fragment will be amplified by PCR with suitable overlapping ends for further Gibson assembly reaction.
1.2       Results
1.2.1     Data
 
 
Every template has been amplified 7 times, each with a different annealing temperature, respectively from the first amplicons : 70°, 69.5°, 68.4°, 66.4°, 64°, 62°, 60°
1.2.2     Interpretation
Not all the annealing temperature worked but we obtained one amplicon of each IFP1-CPXR, CPXR-IFP1, pCPXR-IFP2 opened, pIFP2-CPXR opened.
2        Gibson assembly to make 4 different constructs containing both IFP1-CPXR and IFP2-CPXR in different configuration and colony PCR
Fusion of the insert IFP1-CPXR (IFP1 at the C or N terminal of CPXR) in the plasmid containing IFP2-CPXR (IFP2 at the C or N terminal of CPXR), to have our four constructs:
 
GA1: IFP1 at the Cterminal of CPXR and IFP2 at the Cterminal of CPXR
GA2: IFP1 at the Cterminal of CPXR and IFP2 at the Nterminal of CPXR
GA3: IFP1 at the Nterminal of CPXR and IFP2 at the Cterminal of CPXR
GA4: IFP1 at the Nterminal of CPXR and IFP2 at the Nterminal of CPXR
 
2.1       Material and methods
NEB Gibson assembly kit and NEB Taq polymerase for the colony PCR with primers VR and VF2
2.2       Results
We didn’t obtain any colonies for GA1 and GA3, and only a few for GA2 and GA4. We picked 13 colonies of GA4 and 10 colonies for GA.
2.2.1     Data
Colony PCR of GA4 with VF2 and VR primers. 
 
 
 
Colony PCR of GA2 with VF2 and VR primers. Expectations are the same.
 
 
 
2.2.2     Interpretation
Expectation is about 4050 for plasmid containing both IFP1-CPXR and IFP2-CPXR. Expectation is about 3000 for a plasmid containing only one fragment.
We can’t conclude anything with this ladder, and the difference between our wanted product and one that doesn’t contain both fragment of interest is of 1000 bases so we can’t reaaly see it on the gel. Mini-prep has been done and restriciton analysis will allow us to conclude if the colonies contained the expected product.
 
3        Restriction enzyme analysis of GA2 and GA4
Digestion of our constructs GA2 (containing IFP1 at the Cter of CPXR and IFP2 at the Nter) and GA4 (containing IFP1 at the Nter and IFP2 at the Nter) with enzymes SpeI and NHeI to see if they contain the expected fragments.
3.1       Material and methods
NEB protocol for restriction enzyme digestion was applied and enzyme SpeI and NheI were used.
3.2       Results
3.2.1     Data
Restriction analysis of GA4
 
Restriction analysis of GA2
3.2.2     Interpretation
Expected fragment with both inserts: 3273 bp + 2539 bp
With only CPXR-IFP1: 1200 bp + 3273 bp
With only CPXR-IFP2: 1400 bp + 3273 bp
GA4: plasmids 2, 4, 6, 7, 10 correspond to our expectation
GA2: plasmids 5, 8 correspond to our expectation 
 
 
CheY CheZ - Week 4
07-27-2014
 
1 Experiment 1: PCR for fLucN/C extraction
Purpous: extract N and C temini of firefly Luciferase from EPIC_firefly biobrick (iGEM)
1.1Material and methods
Material:
Phusion PCR kit (NEB)
EPIC_firefly Miniprep (Bba_K325108)
Method
PCR tubes preparation:
 
Fragment
Mastermix (ul)
Fp (ul)
Rp(ul)
DNA (ul)
Water (ul)
Total (ul)
PCR program
1
fLucN_A
11.5
2.5
2.5
4.1
29.4
50
2
2
fLucN_B
11.5
2.5
2.5
4.1
29.4
50
1
3
fLucC_A
11.5
2.5
2.5
4.1
29.4
50
2
4
fLucC_B
11.5
2.5
2.5
4.1
29.4
50
1
5
FlucN control
11*
2.5
2.5
4.1
29.9
50
2
6
FlucN control
11*
2.5
2.5
4.1
29.9
50
1
 * For the controls, there was no phusion polymerase in the MasterMix
Primers: [TABLE]
PCR programs according to Phusion polymerase protocol
 
1.2Results
 
1.2.1Data
 
1.2.2Interpretation
The expected amplicon size for fLucN is 1248 bp. As the bands are between 1000 and 1500 bases, the amplicon seems to be correct. The expected amplicon size for fLucC is 477 bp. As the band is a bit lower than 500 bp, the amplicon seems to be correct. The negative controls without polymerase are also OK.
 
 
2Experiment 2: PCR for Gibson overlaps
Purpose of the experiment: add overlapping sequences to prepare Gibson assemblies
2.1Material and methods
Material:
Phusion PCR kit (NEB)
EPIC_firefly Miniprep (Bba_K325108)
 
Method
PCR tubes preparation:
 
Fragment
Mastermix (ul)
Fp (ul)
Rp(ul)
DNA (ul)
Water (ul)
Total (ul)
PCR program
1
CheY_A
11.5
2.5
2.5
12.7
20.8
50
1
2
CheY_B
11.5
2.5
2.5
NO DNA
33.5
50
1
3
CheZ_A
11.5
2.5
2.5
5.5
28
50
1
4
CheZ_B
11.5
2.5
2.5
5.5
28
50
1
5
CheY ctrl
11 (no phusion)
2.5
2.5
12.7
21.3
50
1
6
CheZ ctrl
11 (no phusion)
2.5
2.5
5.5
28.5
50
1
7
rLucN_A
11.5
2.5
2.5
9.1
24.4
50
3
8
rLucN_B
11.5
2.5
2.5
9.1
24.4
50
3
9
rLucC_A
11.5
2.5
2.5
4.6
28.9
50
1
10
rLucC_B
11.5
2.5
2.5
4.6
33.5
50
1
11
rlucN ctrl
11 (no phusion)
2.5
2.5
9.1
24.9
50
1
12
rlucC ctrl
11 (no phusion)
2.5
2.5
4.6
29.4
50
1
13
rlucC neg_A
11.5
2.5
2.5
4.6
28.9
50
1
14
rlucC neg_B
11.5
2.5
2.5
4.6
28.9
50
1
15
rlucN neg_A
11.5
2.5
2.5
9.1
24.4
50
1
16
rLucN_neg_B
11.5
2.5
2.5
NO DNA
33.5
50
1
17
rlucC neg ctrl
11 (no phusion)
2.5
2.5
4.6
29.9
50
1
18
rlucN neg ctrl
11 (no phusion)
2.5
2.5
9.1
29.4
50
1
19
flucN_A
11.5
2.5
2.5
4.3
29.2
50
2
20
flucN_B
11.5
2.5
2.5
4.3
29.2
50
2
21
fLucC_A
11.5
2.5
2.5
4.6
28.9
50
1
22
fLucC_B
11.5
2.5
2.5
4.6
28.9
50
1
23
fLucN_neg_A
11.5
2.5
2.5
4.3
29.2
50
2
24
fLucN_neg_B
11.5
2.5
2.5
4.3
29.2
50
2
25
fLucC_neg_A
11.5
2.5
2.5
4.6
28.9
50
1
26
fLucC_neg_B
11.5
2.5
2.5
4.6
28.9
50
1
27
plasmid_A
11.5
2.5
2.5
4.1
29.4
50
4
28
plasmid_B
11.5
2.5
2.5
4.1
29.4
50
4
Primers[TABLE]
For the backbone: primers opener 1 and opener 2 were used.
PCR Programs accorgind to Phusion PCR protocol
2.2Results
2.2.1Data
 
2.2.2Interpretation
The expected amplicon size for CheY (lane 1) is 486 bp. As the band is near 500 bases, the amplicon seems good. However, we detect a weak band at approximately 1000 bases that shouldn't appear. This could be an sequence that we obtained after the first PCR for CheY extraction (week 3) and that we didn't see on the gel from last week. However, this shouldn't be a problem. As the band is weak, it has not been amplified this time and it will not react in our future Gibson assembly. The negative control without polymerase (lane 5) shows the previous PCR product that has not been amplified, which is OK.
The expected amplicon size for CheZ (lanes 3&4) is 703 bp. We see that the band is appoximately at 700 bases which confirms that the amplicon is OK. The negative control without polymerase (lane 6) shows the previous PCR product that has not been amplified, which is OK.
The expected amplicon size for rLucN (lanes 7&8) is 388 bp. As the band is detected between 300 and 500 bases, the amplicon seems to be OK.The negative control without polymerase (lane 11) shows the previous PCR product that has not been amplified, which is OK.
The expected amplicon size for rLucC (lanes 9&10) is 669 bp. We see a band a little bit lower than 700 bases, so the amplicon seems right. The negative control without polymerase (lane 12) shows the previous PCR product that has not been amplified, which is OK.
The expected amplicon size for rLucC (future negative control, lanes 13&14) is 662 bp. The band is a bit lower than 700 bases, which is as expected. The negative control without polymerase (lane 17) shows the previous PCR product that has not been amplified, which is OK.
The expected amplicon size for rLucN (future negative control, lane 15) is 420 bp. As the band is between 300 and 500 bases, which is right. The negative control without polymerase (lane 18) shows the previous PCR product that has not been amplified, which is OK.
The expected amplicon size for fLucN (lanes 19&20) is 1306 bp. The band appears a little bit lower than 1500 bases, which is OK. The expected amplicon size for fLucC (lanes 21&22) is 538 bases. We can see a band at approximately 500 bases, which is as expected.
The expected amplicon size for fLucN control (lanes 25&26) is 1306 bp and the band appears a little bit lower than 1500 bases. The expected amplicon size gor fLucC control (lanes 27&28) is 531 bp, and the band appears at approximately 500 bases. An error of pipetting happened for lane 25: no PCR product was loaded.
The expected amplicon size for the backbone was 3311 bp. The band appears a bit higher than 2500 bases, which is as expected.
 
Our products all seem to be ready for the following Gibson assemblies:
CheY-rLucN-CheZ-rLuccC into pSB1C3 (5 fragments)
rLucN-rLucC into pSB1C3 (negative control) (3 fragments)
CheY-fLucN-CheZ-fLucC into pSB1C3 (5 fragments)
fLucN-fLucC into pSB1C3 (negative control) (3 fragments)
 
 
3 Experiment 3: Gibson assembly
Purpous: to obtain the following final constructs:
CheY-fLucN-CheZ-fLucC into pSB1C3 (5 fragments) → fYNZC
CheY-rLucN-CheZ-rLuccC into pSB1C3 (5 fragments) → rYNZC
fLucN-fLucC into pSB1C3 (negative control) (3 fragments) → fLucNfLucC
rLucN-rLucC into pSB1C3 (negative control) (3 fragments) → rLucNrLucC
3.1Material and methods
We followed the protocol “Gibson Assembly Master Mix” from BioLabs, for 4-6 fragments. (Total amount of fragments should be 0.2-1pmols)
The tubes were prepared according to the following table:
 
uL
1
2
3
4
5 (neg control)
Fragment 1
0.44 (CheY)
0.44 (CheY)
0.89 (fLucNneg)
0.96 (rLucNneg)
 
Fragment 2
0.81 (fLucN)
0.39 (rLucN)
0.45 (fLucCneg)
0.68 (rLucCneg)
 
Fragment 3
0.58 (CheZ)
0.58 (CheZ)
 
 
 
Fragment 4
0.37 (fLucC)
0.48 (rLucC)
 
 
 
pSB1C3 (opened)
1.27
1.27
1.27
1.27
1.27
MasterMix
10
10
10
10
10
Deionized Water
6.53
6.84
7.39
7.09
18.73
The tubes were then incubated in a thermocycler according to the protocol.
Competent cells were transformed with the products, plated (2 plates for each construct) and incubated overnight.
3.2Results
3.2.1Data
3.2.2Interpretation
All the plates had a lot of colonies, except for the negative control, which had a lot less, as expected, as it was transformed with linear fragments. The remaining colonies can be due to some vectors that have been used for the PCR amplification and that stayed circular.
 
4 Experiment 4: Gibson inoculation
Purpose of the experiment: Pick up transformed single colonies in order to obtain the plasmids of our final constructs.
4.1Material and methods
Procedure:
Pick one colony with a tip (best = small colonies)
Agitate the tip in 1uL nuclease free water, then heat-shock the tubes for 10 min at 95° (→ colony PCR, see experiment 3)
Drop the tip in 3mL LB chloramphenicol and incubate overnight at 37°C, 180rpm (→ Minipreps + Glycerol stocks, see exp 4)
Plate 1A → tubes 1-5 Plate 1B → tubes 6-10
Plate 2A → tubes 11-15 Plate 2B → tubes 16-20
Plate 3A → tubes 21-25 Plate 3B → tubes 26-30
Plate 4A → tubes 31-35 Plate 4B → tubes 36-40
Plate Neg → tubes C1-4
 
4.2Results
Results for the colony PCR will be discussed in the third part.
 
 
5 Experiment 5: Colony PCR
Purpose of the experiment: We want to verify that the transformed cell contain the construct of interest.
5.1Material and methods
Protocol for 25uL reaction (with TAQ Polymerase)
2.5uL 10x ThermoPol buffer
0.5uL 10mM dNTPs
0.125uL Taq DNA Polymerase
1uL template DNA (from heat-shocked tubes)
0.5uL forward primer
0.5uL reverse primer
19.875uL nuclease free water
 
PCR program:
Initial denaturation: 30 sec at 95°C
30 cycles: 10-30 sec at 95°C
15-60 sec at 45-68°C (annealing temperature (A.T.))
1 min/kb at 68°C
Final extension: 5 min at 68°C
Pause at 4°C
 
Program used:
CHETAQON (A.T. 54°C, ext. time 3 min) → tubes 1 to 10 + C1, primers CheY for / fLucC rev
CHETAQTW (A.T. 51°C, ext. time 2 min) → tubes 11 to 20 + C2, primers CheY For / rLucC rev
CHETAQTH (A.T. 52°C, ext. time 2 min) → tubes 21 to 30 + C3, primers fLucN for / fLucC rev
CHETAQFO (A.T. 51°C, ext time 1 min) → tubes 31 to 40 + C4, primers rLucN for / fLucC rev
Rem: control tubes were prepared the same way (1uL from heat shocked tubes + 0.5 from each primers)
The PCR products were analyzed by gel electrophoresis (1.2% agarose gels. 2uL PCR reaction + 8 uL water + 2uL 6X loading dye. The ladder that was used was 1kb ladder)
 
 
5.2Results
5.2.1Data
 
 
3.2.2Interpretation
For the construct with firefly luciferase, (fYNZC)none of the colonies had the right insert* with the expected size of 2887 bp (firefly). For the construct with renilla, lane 11 revealed a good colony with the right insert of the expected size of 2101 bp (renilla).
For the split luciferases, tubes 21, 24, 25, 26, 27, 28, 29 and 30 (split fLuc) have the good insert (1750 bp), as well as 31, 32, 33, 34, 35, 36, 38, 39 and 40 (split rLuc) (964 bp).
We will send samples 26 and 33 for sequencing. We will pick other colonies in order to obtain a colony with the construct.
6 Experiment 6: Minipreps of negative controls
Purpose of the experiment: Make minipreps of the negative controls (split alone)
6.1Material and methods
Cf. Miniprep kit from Qiagen
Done on 5mL pelleted cultures from tubes 11, 21, 24, 26, 27, 28, 29, 31, 32, 33 34, 35, 36, 38, 39, 40 (in big centrifuge)
DNA concentration was determined using the Nanodrop Spectrophotometer.
6.2Results
6.2.1Data
Sample
Concentration [ng/uL]
260/280
260/230
11
320.3
1.94
2.12
21
390.1
1.95
2.14
24
482.4
1.94
2.23
26
494.1
1.94
2.27
27
795.4
1.95
2.3
28
578.1
1.94
2.23
29
381.9
1.92
2.16
31
543.8
1.93
2.13
32
401.6
1.94
2.27
33
442.3
1.94
2.3
34
429.2
1.94
2.31
35
436.1
1.92
2.17
36
753.2
1.92
2.24
38
230
1.84
1.96
39
525.7
1.94
2.19
40
271
1.92
1.89
 
6.2.2Interpretation
Concentration were very good.
Cpxr Overexpression
07-28-2014
1 Experiment 1: Gibson Assembly (GA) and PCR Colonies
Assembly of GFP at the C ter or N ter of the Bba_1496001 cpxr under arabinose promoter biobrick by Gibson assembly.
The purpose of this colony PCR is to check if the insert is present and so determine if the cloning worked.
1.1 Material and methods
We did the GA with Bba_1496001 cpxr under arabinose promoter biobrick opened with corresponding primers and amplified insert. Our plasmid have a chloramphenicol resistance.
We used the Gibson assembly kit (see protocol).
 
PCR colony, Taq polymerase 25µl reaction (see protocol) with the  primers VR and RF2 ( TM= 49°C)
 
Thermocycling: 
Initial denaturation
95°C
30 seconds
30 cycles
95°C
15-30 seconds
 
45-68°C
15-60 seconds
 
68°C
1 min/kb
Final Extension
68°C
5 min
stop
4°C
 
 
1.2Results
1.2.1Data
Gel of the PCR of the plasmid CpxR CGFP with VR and VF2 primers (gels 1 and 2)
 
 
Upper Gel :
lane 1 : ladder
lane 2 : plasmid of CpxR CGFP f rom the colonie 1
lane 3 :plasmid of CpxR CGFP f rom the colonie 2
lane 4 :plasmid of CpxR CGFP f rom the colonie 3
lane 5 :plasmid of CpxR CGFP f rom the colonie 4
lane 6 :plasmid of CpxR CGFP f rom the colonie 5
lane 7 :plasmid containing only CpxR
 
Down Gel :
lane 1 :ladder
lane 2 :plasmid of CpxR CGFP f rom the colonie 6
lane 3 :plasmid of CpxR CGFP f rom the colonie 7
lane 4 :plasmid of CpxR CGFP f rom the colonie 8
lane 5 :plasmid of CpxR CGFP f rom the colonie 9
lane 6 :plasmid of CpxR CGFP f rom the colonie 10
lane 7: plasmid containing only CpxR (negatif control)
 
 
 
 
 
Gel of the PCR of the plasmid CpxR NGFP with VR and VF2 primers ( gels 3 and 4)
 
 
Upper Gel :
lane 1 : ladder
lane 2 : plasmid of CpxR NGFP f rom the colonie 1
lane 3 :plasmid of CpxR NGFP f rom the colonie 2
lane 4 :plasmid of CpxR NGFP f rom the colonie 3
lane 5 :plasmid of CpxR NGFP f rom the colonie 4
lane 6 :plasmid of CpxR NGFP f rom the colonie 5
lane 7 :plasmid containing only CpxR
 
Down Gel :
lane 1 :ladder
lane 2 :plasmid of CpxR NGFP f rom the colonie 6
lane 3 :plasmid of CpxR NGFP f rom the colonie 7
lane 4 :plasmid of CpxR NGFP f rom the colonie 8
lane 5 :plasmid of CpxR NGFP f rom the colonie 9
lane 6 :plasmid of CpxR NGFP f rom the colonie 10
lane 7: plasmid containing only CpxR (negatif control)
 
 
 
Remarks:
Gel 1, ladder 2 log
Gel 2, 3, 4 1 kb ladder
 
 
1.2.2Interpretation
It seems to work for all the insert of the GFP in Cterminal
but only for one (the plasmid from the colonie 10) for the insert of the GFP in Nterminal
 
2 Experiment 2: Minprep the plamids containing the insert 
 
2.1 Material and methods
Minipreps protocol (see protocol) 
 
2.2Results
2.2.1Data
 
Nanodrops of the miniprep
 
1.2.2Interpretation
 
Our Minipreps have a good concentration, we can now send them to sequencing.
 
2 Experiment 3: Sequencing of CGFP and NGFP plamids
 
3.1 Material and methods
 
We sent to sequencing the sample: gfpC_3 ans gfpN_10.
 
Primers used :
 
Nterm :
pBad for 
cpxR for gen
cpxR rv gen
 
Cterm :
pBad for 
CGFP control fw
VR
 
plasmid concentration: 80 ng/µl ()
primers concentration : 2µM (diluer 10X les plasmids et prendre 0,6 µl)
total volume : 15µl (complete with  nuclease free water)
 
3.2Results
3.2.1Data
The sequencing confimed the GA worked, the insert have the correct sequence without any mutation.
 
3.2.2Interpretation
 
We wanted to confirm the expression of the GFP with a wet bench experiment :
 
incubation overnight with 10^3 µM of Arabinose (30 µL in 3mL of LB)
Conclusion: GFP is express.
 
CheY CheZ - Week 5
08-03-2014
 
1 Experiment 1: Colony PCR (cont.)
Purpous: Retry a colony PCR to obtain the constructs of interest.
1.1Material and methods
We should pick colonies from the most efficient plate.
20 colonies were picked from plate 1B → tubes 1-20
10 colonies were picked from plate 2B → tubes 21-30
 
 
Same protocol as the week before for Colony PCR
CHETAQON → tubes 1 to 20 + C1 (primers = CheY for/fLucC rev)
→ tubes +1 to +4 (primers = openers 1 & 2)
CHETAQTW → tubes 21 to 30 + C3 (primers = CheY for/rLucC rev)
→ tubes +5, +6 (primers = openers 1 & 2)
 
1.2% Agarose gels were used for analysing the samples. The ladder used was 1kb plus ladder →
1.2Results
1.2.1Data
 
1.2.2Interpretation
For the construct with Renilla (gel on the right), it seems that lane 26 is a sample containing the expected insert (2101 bp). Miniprep will be done on this tube.
We have to pick other colonies for the construct with Firefly luciferase, as none of the picked colonies seemed to contain the right insert.
2 Experiment 2: Sequencing
Purpose of the experiment: To verify that the inserts don't have any mutations.
2.1Material and methods
Sequencing was done on tubes 26 for fLucNfLucC and 33 for rLucNrLucC
 
FlucNFlucC:
Primer pBAD for → 736 bp
SEQUENCING_Fluc_Middle → 619 bp
FlucC_pSB1C3_Forward_PCR_SN → 456 bp
 
RlucNRlucC:
primer pBAD for → 458 bp
rLucC_pBAD33_forward_PCR_SN → 583 bp
 
Sequencing on tube 26 for rYNZC
Primer pBAD For → 562 bp
rLucN_pBAD33_forward_PCR_SN → 327 bp
Ec_CheZ_pSB1C3_forward_PCR_SN → 658 bp
rLucC_pBAD33_forward_PCR_SN → 583 bp
2.2Results
2.2.1Data
[SEQUENCING RESULTS]
2.2.2Interpretation
 
 
3Experiment 3: Colony PCR (cont.)
Purpose of the experiment: Find a colony that has the right insert for Firefly construct
3.1Material and methods
PCR on firefly construct
20 colonies were picked from plate with firefly construct → tubes 1 to 20
Same protocol as the week before for Colony PCR
primers: CheY_for with FlucN_rev
CHETAQA , annealing temperature 52°C
3.2Results
 
3.2.1Data
 
3.2.2Interpretation
The expected size of 2887 seems to appear on lane 17: this sample should contain the right insert with firefly luciferase construct. The other ones are not valid.
Split IFP & GFP - Week 6
08-03-2014
1        Gibson assembly and Colony PCR of GA3 (IFP1 at the N terminal of CPXR and IFP2 at the C terminal of CPXR)
Last time we tried we obtained only colonies containing either only IFP1-CPXR either IFP2-CPXR.
1.1       Material and methods
Gibson assembly kit and Taq polymerase (see protocols)
1.2       Results
1.2.1     Data
1.2.2     Interpretation
The ladder doesn’t allow us to see precisely the size of the plasmid. Moreover, the colony PCR didn’t work very well as we obtained a lot of unspecific bands. Mini-prep will be necessary to analyse our Gibson assembly product.
 
2        Restriction analysis of GA3
Digestion of our Gibson assembly product GA3 (IPF1 at the N terminal of CPXR and IFP2 at the C terminal of CPXR,) to see which plasmids contain the expected insert.
2.1       Material and methods
Restriction digestion with NheI and SpeI (see protocols)
2.2       Results
2.2.1     Data
2.2.2     Interpretation
Expectation with both fragments: 3273bp + 2535 bp: Colonies 9 and 10
Expectation with only IFP1-CPXR: 3273 bp + 1110 bp: Colonies 4 and 8
Expectation with only IFP2-CPXR: 3273 bp + 1394 bp: Colonies 1, 2, 3, 5, 6, 7
 
We obtained two plasmids that seem to contain IFP1-CPXR (N terminal) and IFP2-CPXR (C terminal)
3        GA2, GA3 and GA4 sequencing
Restriction analyses of these 3 plasmids are thought to contain the expected fragments, but sequencing is necessary to see if they contain any mutations
3.1       Material and methods
Microsynth barcodes
3.2       Results
3.2.1     Data
GA2: no mutation
GA3: no mutation
GA4: no mutation (one deletion in the promoter, not important)
3.2.2     Interpretation 
We obtained our expected products for GA2, GA3, GA4
 
 
CheY CheZ - Week 6
08-10-2014
 
1Experiment 1: Verification PCR
Purpous: Verification in the constructs fYNZC and rYNZC to verify that the right inserts were present
1.1Material and methods
Done on minipreps of tubes 17 (fYNZC) and 26 (rYNZC)
1.2Results
 
1.2.1Data
 
1.2.2Interpretation
The expected fragments were obtained for rYNZC construct. As the first gel wasn't clear for fYNZC, we did an other one, which gave fragments that are all explained and expected.
We see different bands. The top one in all lanes is the whole plasmid. For lane 7, fLucN_rev primer also binds in fLucC → 2000bp instead of 1245. Lane 8: fLucC_for primer also binds to the end of fLucN → 1252 bp instead of the expected 474. Lane 4: we only obtain the “opened” primers in addition to the initial primer.
 
2Experiment 2: Restriction analysis
Purpose of the experiment: Restriction enzyme digestion of rYNZC and fYNZC in order to confirm the presence of our constructs in each plasmid, using enzymes NheI and SpeI
2.1Material and methods
Restriction digest:
uL
FYNZC – sample 17
RYNZC – sample 26
Template DNA (500 ng)
1.47
1.59
NheI
0.5
0.5
SpeI
0.5
0.5
Smart Cut Buffer
5
5
Water
42.53
42.41
Total reaction volume
50
50
 
Samples were heated for 60 minutes at 37°C before being run on a gel.
2.2Results
 
2.2.1Data
 
 
2.2.2Interpretation
The expected sizes of fragments are observed. 
Conclusion
The plasmids can be sent for sequencing.
 
 
3 Experiment 3: Bioluminescence assay 3
Purpous: We want to check if our constructs work.
3.1Material and methods
Overnight cultures were prepared from glycerol stocks of fYNZC (tube 17), rYNZC (tube 26), fLucNfLucC (tube 26-), rLucNrLucC (tube 31-), fLuc (EPIC Firefly) and rLuc (rLuc from Waldor's lab) into 5mL LB-Chloramphenicol.
50mL of overnight cultures were transferred into 5mL fresh chloramphenicol containing 1mM arabinose. These sample were incubated for 4 hours at 37°C, 180rpm shaking, to an OD ~1.
Cultures were centrifuged, medium was discarded, cells were washed once with PBS.
The substrate (coelenterazine (clz) for renilla Luciferase and d-luciferin (d-luc) for firefly Luciferase) were prepared in aliquots of 100uM and were added at 4 different concentrations (7.5uM, 15uM, 22,5uM, 30uM).
A white opaque 96-Well plate was loaded as following. Samples were loaded in triplicate, in a total volume of 200uL contaning 140uL of culture, ajusting the OD to 0.7:
 
uL
Clz
D-luc
Culture (OD 1)
PBS
A/C/E 1-3
15
 
140
45
A/C/E 4-6
30
 
140
30
A/C/E 7-9
 
15
140
45
A/C/E 10-12
 
30
140
30
B/D/F 1-3
45
 
140
15
B/D/F 4-6
60
 
140
0
B/D/F 7-9
 
45
140
15
B/D/F 10-12
 
60
140
0
Substrates for luciferins were added at the end.
After ~10 minutes RT incubation, the plate was read by a plate reader (1CPS) every 4 minutes.
 
Lanes A & B: constructs
Lanes C & E: negative controls (split luciferases alone)
Lanes D & F: positive controls (luciferases)
 
3.2Results
3.2.1Data
Figure 1: Graph of the Luminescence of the constructs with Renilla Luciferase and of the corresponding negative controls (each with 4 different substrate concentration) in function of the time. RYNZC stands for the construct EcCheY_rLucN_EcCheZ_rLucC, and RlucNrLucC stands for the split luciferase alone. “clz” is for coelenterazine.
 
Figure 2: Graph of the Luminescence of the constructs with Firefly Luciferase and of the corresponding negative controls (each with 4 different substrate concentration) in function of the time. FYNZC stands for the construct EcCheY_fLucN_EcCheZ_fLucC, and flucNfLucC stands for the split luciferase alone. “luc” is for luciferin
 
Figure 3: Graph of the luminescence of the positive and negative controls for Renilla Luciferase (each with 4 different substrate concentration) in function of the time.
 
Figure 4: Graph of the luminescence of the positive and negative controls for Firefly Luciferase (each with 4 different substrate concentration) in function of the time.
 
Figure 5: Graph of the Luminescence of the constructs with Renilla Luciferase (rYNZC) and Firefly Luciferase (fYNZC) in function of the different concentrations of substrate (coelenterazine for rLuc and luciferin for fLuc).
 
Figure 6: Graph of the Luminescence of the positive and negative controls with Renilla Luciferase and Firefly Luciferase in function of the different concentrations of substrate (coelenterazine for rLuc and luciferin for fLuc).
 
3.2.2Interpretation
As the signal detected in the plates containing our constructs was lower than the negative control, we conclude that our construct didn't work.
The problem might come from the fact that as the dimerization of CheY and CheZ should only happen in the absence of chemoattractants, it has not worked because of the presence of an attractant. This could have happened, as arabinose was used to induce the gene, and as the cells were washed only once with PBS. The next experiment should be done after more wash steps, to get rid of the arabinose.
However, the positive and negative controls of our experiment gave good results. The gap between the positive and negative controls is quite big, and it seems that the substrate concentration affects the activity of both luciferases.
We see that firefly luciferase seems to be better than renilla, because the gap between + and – controls is bigger and because it seems to be more sensitive to a substrate concentration gradient. We should consider these results for the choice of the Luciferase we will use for the rest of the project.
 
 
4 Experiment 4: Bioluminescence assay 4
Purpous: To test our constructs by measuring their relative emitted bioluminescence.
4.1Material and methods
Overnight cultures were prepared from glycerol stocks of unstransformed DH5 alpha, fYNZC (tube 17), rYNZC (tube 26), fLucNfLucC (tube 26-), rLucNrLucC (tube 31-), fLuc (EPIC Firefly) and rLuc (rLuc from Waldor's lab) into 5mL LB-Chloramphenicol.
50mL of overnight cultures were transferred into 5mL fresh chloramphenicol containing 1mM arabinose. These sample were incubated for 4 hours at 37°C, 180rpm shaking, to an OD ~1.
Cultures were centrifuged, medium was discarded, cells were washed three times with PBS.
The substrate (coelenterazine (clz) for renilla Luciferase and d-luciferin (d-luc) for firefly Luciferase) were prepared in aliquots of 100uM and added at 30 uM (d-luc) and 22.5uM (clz).
 
A white opaque 96-Well plate was loaded as following. Samples were loaded in triplicate, in a total volume of 180uL contaning 120uL of culture, ajusting the OD to 0.7:
 
4.2Results
 
4.2.1Data
 
 
Figure 1: (A) Graph of bioluminescence of different cultures with 22.5uM d-luciferin. F-PBS = PBS, DH5-alpha = untransformed competent cells, split fLuc = split Firefly Luciferase, fLuc = Firefly Luciferase, fYNZC = construct with CheY-fLucN-CheZ-fLucC. (B) Graph of bioluminescence of different cultures with 30mM coelenterazine. R-PBS = PBS, DH5-alpha = untransformed competent cells, split rLuc = split Renilla Luciferase, rLuc = Renilla Luciferase, fYNZC = construct with CheY-fLucN-CheZ-fLucC.
 
 
Figure 2: (A) Graph of bioluminescence of different cultures, with 22.5uM d-luciferin, without positive control. Same legend as 1A. (B) Graph of bioluminescence of different cultures, with 30uM coelenterazine, without positive control. Same legend as 1B.
 
Figure 3: Comparison of bioluminescence of the two constructs, with Firefly Luciferase (with 22.5uM d-luciferin) and Renilla Luciferase (with 30uM coelenterazine).
 
Figure 4: Comparison of bioluminescence of the positive and negative controls with Firefly Luciferase and Renilla Luciferase
 
4.2.2Interpretation
The obtained values are as expected for the controls, compared to the blanks with PBS and unstransformed competent cells. The values of our constructs being a bit lower than the negative controls, these results are not as good as expected. However, there is a good hope that our constructs can work, as their signals are higher than the blanks.
Conclusion
We have to try again with Waldor construct (pYNZC, with proteins from Vibrio Cholerae). We will do more steps of wash with PBS.
Split IFP & GFP - Week 7
08-10-2014
1        Gibson assembly and Colony PCR of GA1
To fuse CPXR-IFP1 and CPXR.IFP2 and analyse the products by colony PCR
1.1       Material and methods
Gibson assembly kit and Taq polymerase (see protocol)
1.2       Results
1.2.1     Data
1.2.2     Interpretation
None of our colony contains our expected product. Colonies 1, 3, 5, 6, 8 only contain IFP2-CPXR, and the other ones only contain IFP1-CPXR, which are our PCR templates. We should have done our PCR with a lower amount of DNA template. It results that our Gibson assembly product contains a lot of DNA template, decreasing the probability to have bacteria transformed by our expected product. We will do the PCR again with a lower amount of DNA template. (Gibson assembly kit recommends a DNA template of 0.1-0.5 ng and we used 10 ng)
2       Gibson assembly and of GA1: 2nd try
We tried again the Gibson assembly once we did the PCR again with a lower amount of DNA template.
2.1       Material and methods
Gibson assembly kit and Taq polymerase (see protocol)
2.2       Results
2.2.1     Data
2.2.2     Interpretation
It seems that all our colonies contain the expected fragement. Restriction analysis will confirm it
3        Restriction analysis of GA1
To see if our GA1 product contains the expected fragment
3.1       Material and methods
NheI and SpeI restriction enzymes. (See protocol)
3.2       Results
3.2.1     Data
3.2.2     Interpretation
 
 We finally obtain our GA1 product (IFP1 and IFP2 at the C terminal of CPXR) in all the colonies (except the 11th). Sequencing will confirm it.
4        GA1 sequencing
To see if our GA1 product contains any mutation
4.1       Material and methods
Microsynth barcodes
4.2       Results
4.2.1     Data
No mutation occured.
Arabinose promoter characterization
08-12-2014
Experiment 1: Arabinose promoter characterization
1.1       Material and methods
Cells with sfGFP CpxR & CpxR sfGFP constructs were cultured in different concentrations of arabinose to check the functionality of the promoter. The concentration of arabinose per sample is indicated in Table 1.
 Sample
uM Ara
uL Ara
ml LB-chl
tube 96-w
       plate
A
0
0
5
A
1
B
5
0.025
5
B
2
C
10
0.05
5
C
3
D
50
0.25
4.9998
D
4
E
100
0.5
4.9995
E
5
F
500
2.5
4.9975
F
6
G
1000
5
4.995
G
7
H
5000
25
4.975
H
8
I
10 000
50
4.95
I
9
J
50 000
250
4.75
J
10
K
100 000
500
4.5
K
11
L
500 000
2500
2.5
L
12
+
0
0
5
+
 
-
5000
25
4.975
-
 
Table 1 - This table shows the concentration of Arabinose in each sample, as well as the quantity of medium and of Arabinose needed to reach it.
The negative control was the medium without cells, and the positive control was a sample of cells expressing sfGFP contitutively.
 
Procedure
Cells from glycerol stocks were grown in 3 mL LB selective medium (chloramphenicol) during 3h (37 ºC & shaking).
50 uL of cells were resuspended in a total volume of 5 mL of LB-chloramphenicol and Arabinose and incubated 5h at 37 ºC with shaking until an OD of ~1 was reached.
The samples were then washed twice with PBS to a final OD of 0.4 - 0.5.
 
Table 2 shows the quantity of PBS added to each sample (depending on the OD), as well as the OD after both PBS washes. This information is used to normalize the plate reader results.
tube 96-w
        plate
OD N ter
PBS N
OD PBS
OD C ter
PBS C
OD PBS
tube 96-w
       plate
A
1
1.136
10.5
0.281
1.224
11
0.369
A
1
B
2
1.071
10
0.402
1.061
10
0.384
B
2
C
3
1.014
10
0.360
1.082
10
0.332
C
3
D
4
1.098
10
0.342
1.068
10
0.393
D
4
E
5
1.016
10
0.346
1.07
10
0.338
E
5
F
6
1.038
10
0.349
1.06
10
0.351
F
6
G
7
0.912
9.5
0.336
1.06
10
0.314
G
7
H
8
0.686
7
0.316
1.028
10
0.307
H
8
I
9
0.645
7
0.314
0.999
10
0.305
I
9
J
10
0.52
6
0.305
0.918
9.5
0.281
J
10
K
11
0.449
5
0.281
0.822
9
0.285
K
11
L
12
0.315
4
0.296
0.505
6
0.248
L
12
+
 
1.117
10
 0.283
-
-
-
+
 
-
 
0.005
10.5
 
-
-
-
-
 
 Table 2 - OD measurments of the samples can be seen, as well as the OD after the PBS washes.
Results
1.4.1     Data
https://static.igem.org/mediawiki/2014/0/06/Sfgfpcpxr_gradient.jpg
Figure 1 - Photo of the pelleted samples between PBS wash steps 1 and 2.
As can be seen in the image, there is a gradient of fluorescence intensity.
The results of the plate reader are depicted in Figure 2.
https://static.igem.org/mediawiki/2014/f/f5/Twotermscpxrgfp.jpg
Figure 2 - Graphs of the intensity curves of the samples. Cterm corresponds to CpxR sfGFP (sfGFP attached to CpxR's C temrinus) and Nterm corresponds to sfGFP CpxR.
1.4.2     Interpretation
As can be seen in the graph, the the results of the sfGFP CpxR sample was much better (closer to the expected results) than the one on the CpxR sfGFP, although in general, the curves grow.
It could be seen that adding too much arabinose decreases viability, so we considered that a concentration of 5 mM was the optimum arabinose concentration.
CheY CheZ - Week 7
08-17-2014
 
1 Experiment 1: Bioluminescence assay 5
Purpous: We want to test our constructs by measuring the emitted bioluminescence with a plate reader
1.1 Material and methods
Overnight cultures were prepared from glycerol stocks of unstransformed DH5 alpha, fYNZC (tube 17), rYNZC (tube 26), pYNZC (received from Waldor's lab) fLucNfLucC (tube 26-), rLucNrLucC (tube 31-), fLuc (EPIC Firefly) and rLuc (rLuc from Waldor's lab) into 5mL LB-Chloramphenicol.
50mL of overnight cultures were transferred into 5mL fresh LB (+ chloramphenicol, except for untransformed DH5 alpha) containing 1mM arabinose. These sample were incubated for 3 hours at 37°C, 180rpm shaking, to an OD ~1.
Cultures were centrifuged, medium was discarded, cells were washed five times with PBS.
The substrate (coelenterazine (clz) for renilla Luciferase and d-luciferin (d-luc) for firefly Luciferase) were prepared in aliquots of 100uM and added at 30 uM (d-luc) and 22.5uM (clz).
 
A white opaque 96-Well plate was loaded as following. Samples were loaded in triplicate, in a total volume of 180uL contaning 120uL of culture, ajusting the OD to 0.7:
Measures were taken for 1h every 2 minutes with the plate reader Tekan Infinite.
 
 
1.2 Results
As shown in the graphs below (fig.1A and 1B), we couldn't really observe a high signal for our complementation assay. However, the signal being higher than the blanks, it is an encouraging sign that the splits luciferase can be used for other experiments of this kind. A possible explanation for these results is that arabinose being a chemoattractant, we might need to do more wash steps with PBS to get rid of the arabinose before taking the measurements. Moreover, CheY and CheZ being endogenously expressed in bacteria, the edogenous proteins could interfere with our fusion proteins and weaken our signal. This complementation assay should be tested with CheY/CheZ knock out strains, as it was done in Waldor Laboratory.
We also could determine which of the luciferases would best suit our following experiments. As shown in fig. 2, for the same concentration of substrate, we see that firefly luciferase has a more stable and higher signal. Moreover, the difference between the background noise (negative control, non fused split luciferase) and the full luciferase is bigger for Firefly luciferase, which is also preferable.
 
$
1.2.2 Interpretation
 
Leucine zippers - week 8
08-24-2014
 
 
1. Experiment 1: amplification of fragments of interest
Purpous: We want to amplify the fragments of interest from different vectors in order to make the following constructs:
Fig1A: Vector of zipper-rLucN + zipper-rLucC Fig1B: Vector of zipper-fLucN + zipper-fLucC
 
The sequences will be amplified with overlapping fragments in order to further do a fusion PCR (exp. 2).
 
1.1 Material and methods
 
Material:
Phusion PCR kit
The following primers: zip01, zip03, zip06, zip04, zip07, zip08, zip09, zip11, zip12, zip14, zip15, zip17, zip18 (see Sequences/Primers on Google drive)
Vectors: zip_IFP1m rlucNrLucC (from Waldor's lab), EPIC Firefly
PCR purification kit (Qiagen)
Nanodrop spectrophotometer
 
Procedure:
 
Reaction set up:
 
PCR
Fragment
Template
Primer for
PRimer rev
Amp. size (bp)
A. temp (°C)
Programme
1
Zip + fusion OL1 for Ren
zip_IFP1
zip01
zip03
200
71
1
2
Zip + fusion OL2 for Ren 
zip_IFP1
zip01
zip06
200
71
1
3
Zip + fusion OL1 for Fir
zip_IFP1
zip01
zip04
200
71
1
4
Zip + fusion OL2 for Fir
zip_IFP1
zip01
zip07
200
71
1
5
RLucN + fusion OL1
rlucNrlucC (Waldor)
zip08
zip09
1:
340
72
1
6
RLucC + fusion OL2
rlucNrlucC (Waldor)
zip11
zip12
621
71
1
7
FlucN + fusion OL1
EPIC Firefly
zip14
zip15
1263
69
2
8
FlucC + OL2
EPIC Firefly
zip17
zip18
498
69
3
 
 
Each sample was done for 50uL of reaction. See protocol for phusion PCR for detailed quantities.
 
 
Thermocycling programs:
 
 
Initial denaturation
98°C
30 seconds
30 cycles
98°C
10 seconds
 
71°C
30 seconds
 
72°C
30 seconds
Final Extension
72°C
10 min
stop
4°C
 
 
2:
 
Initial denaturation
98°C
30 seconds
30 cycles
98°C
10 seconds
 
69°C
30 seconds
 
72°C
1 minute
Final Extension
72°C
10 min
stop
4°C
 
 
3:
 
Initial denaturation
98°C
30 seconds
30 cycles
98°C
10 seconds
 
69°C
30 seconds
 
72°C
30 seconds
Final Extension
72°C
10 min
stop
4°C
 
 
 
After PCR, the products were purified according to the Qiagen PCR purification kit. Samples were analyzed by agarose gel electrophoresis
 
1.2 Results
1.2.1 Data:
Fig2A: Gel of the PCR for the amplification of the overlap fragments for fusion PCR
 
Caption: (lanes)
1: Quick-load DNA marker
2: Zip + fusion OL1 for Ren
3: Zip + fusion OL2 for Ren
4: Zip + fusion OL1 for Fir
5: Zip + fusion OL2 for Fir
6: RLucN + fusion OL1
7: RLucC + fusion OL2
8: FlucN + fusion OL1
9: FlucC + fusion OL
 
 
Fig2B: Gel of the PCR for the amplification of rLucN + fusion OL1 (second trial, with all 4 samples of split renilla luciferase we received from Waldor's lab)
 
 
Caption: (Lanes)
2: Quick-load DNA marker
3: RLucN + fusion OL1 (1)
4: RLucN + fusion OL1 (2)
5: RLucN + fusion OL1 (3)
6: RLucN + fusion OL1 (4)
 
Fig3A: Nanodrops of purified PCR products
 
Fig3B: Nanodrops of purified PCR products (rLucN + fusion OL1 only, second trial)
 
 
1.2.2 Interpretation:
There is nothing in the lane 5 in fig 2A, the PCR for RlucN+fusion OL1 didn't work. We tried it again with all the samples of split rLuc we had (which gave the gel in fig 2B) and obtained the right fragment in all 4 samples.
All the other obtained fragments have the expected size, so we can continue our experiments to obtain the 2 vectors we want.
 
 
 
 
2. Experiment 2: fusion PCR
Purpous: We want to perform a fusion PCR to obtain the following products: zipper + rLucN, zipper + rLucC, zipper + fLucN and zipper + fLucC
 
2.1 Material and methods
 
Material:
Phusion PCR kit
Primers: zip02, zip10, zip05, zip13, zip16, zip19 (see Sequences/Primers on Google drive)
PCR products from previous experiment
PCR purification kit (Qiagen)
Nanodrop spectrophotometer
 
Procedure:
 
Each sample was done for 50uL total volume reaction. See protocol for phusion PCR for detailed quantities to add. The first PCR (part A) is done without primers, for 15 cycles. Primers are add before part B
 
Reaction set up:
 
 
 
1
2
3
4
5
6
7
8
Water
Tube 1
0.32µl
---
---
---
1.86µl
---
---
---
32.68µl
Tube 2
---
0.2µl
---
---
---
1.72µl
---
---
32.77µl
Tube 3
---
---
0.21µl
---
---
---
0.61µl
---
31.32µl
Tube 4
---
---
---
0.28µl
---
---
---
0.45µl
31.58µl
 
Master Mix: (for 5 samples): 50µl 5X Buffer, 5µl dNTPs, 2.5 µl polymerase
 
Thermocycling programs:
 
4A
Initial denaturation
98°C
30 seconds
15 cycles
98°C
10 seconds
 
60°C
30 seconds
 
72°C
45 seconds
Final Extension
72°C
10 min
stop
4°C
 
 
5A
Initial denaturation
98°C
30 seconds
15 cycles
98°C
10 secon
 
60°C
30 seconds
 
72°C
9 seconds
Final Extension
72°C
10 min
stop
4°C
 
 
4B
Initial denaturation
98°C
30 seconds
15 cycles
98°C
10 seconds
 
72°C
30 seconds
 
72°C
45 seconds
Final Extension
72°C
10 min
stop
4°C
 
 
5B
Initial denaturation
98°C
30 seconds
15 cycles
98°C
10 seconds
 
72°C
30 seconds
 
72°C
90 seconds
Final Extension
72°C
10 min
stop
4°C
 
 
After the PCR, the products were purified. The concentration of DNA was determined with the nanodrop spectrophotometer. The fragments were verified by agarose gel electrophoresis
Changes in PCR program for sample 4 (as it didn't work as expected the first time)
 
6A
Initial denaturation
98°C
30 seconds
15 cycles
98°C
10 seconds
 
59°C
30 seconds
 
72°C
45 seconds
Final Extension
72°C
10 min
stop
4°C
 
 
6B
Initial denaturation
98°C
30 seconds
15 cycles
98°C
10 seconds
 
73°C
30 seconds
 
72°C
45 seconds
Final Extension
72°C
10 min
stop
4°C
 
2.2 Results
2.2.1 Data:
Fig 4A: Gel of the fusion PCR
 
Caption
Lane 2: ladder quick-load DNA marker
Lane 3:Zip-rLucN
Lane 4:Zip-rLucC
Lane 5: Zip-fLucN
Lane 6: Zip-fLucC
 
 
 
 
 Fig 4B: Gel of the fusion PCR for sample 4 only (second trial)
 
Caption
Lane 3: ladder quick-load DNA marker
Lane 4: Zip-fLucC
 
 
 
 
Fig5A: Nanodrops of purified PCR products
 
Fig5B: Nanodrop of purified PCR product (sample 4 only)
 
2.2.2 Interpretation:
The first gel shows that the first three fusion PCR have been successful, as we obtain the expected sizes. We retried a PCR (changing the program) for sample 4, which was then successful too. The fragments are now ready for a Gibson assembly
 
 
3. Experiment 3: Gibson assembly
 
Purpous: We want to perform a Gibson assembly with the previous prepared fragments, in order to obtain the vectors from fig 1A and 1B
 
2.1 Material and methods
 
Material:
PCR products from previous experiment
Vector (opened) BB3 (69.6 [ng/uL])
Competent cells
Gibson assembly master mix
Taq polymerase kit
 
Procedure:
 
GA1: zip-rLucN + zip-rLucC
GA2: zip fLucN + zip-fLucC
 
We put 50 ng of vector (BB3, 69.6ng/uL) and 150ng of insert.
Reaction set up :
[uL]
Vector (BB3)
Zip-rLucN
Zip-rLucC
Zip-fLucN
Zip-fLucC
MasterMix
Water
Total
GA1
0.72
1.16
0.97
 
 
10
7.15
20
GA2
0.72
 
 
1.05
1.45
10
6.78
20
 
We transformed the competent cells according to the protocol from Neb
Cells were plated onto 2 plates per Gibson Assembly reaction (GA1 A&B, GA2 A&B)
 
The following day (04.09.14), we picked 4 colonies from each plate, for subsequent colony PCRs and cultures. The tips which picked the colonies were agitated into 10uL water and then dropped into 5mL LB chloramphenicol. Cells were cultured for 6 hours for subsequent glycerol stocks preparation and minipreps. The cells in water were heat shocked for 10 minutes at 95°C and used for colony PCR. The minipreps were used for restriction analysis.
 
 
COLONY PCR
 
Primers used for colony PCR:
GA1: zip01 and rLucC_rev
GA2: zip01 and fLucC_rev
 
Reaction Set Up: Taq Polymerase protocol
 
Colony PCR results:
 
 
 
[GEL]
 
 
 
Colonies GA1 A3 and GA2 A1, A2, B2, B3, B4 were discarded. As the colony PCR wasn't very clear, we did a restriction analysis the following day (05.09.14)
 
 
Restriction Analysis 1 (05.09.14)
Each sample was double digested with NheI and SpeI.
Sample preparation: 500ng of template DNA, 5uL of Cutsmart buffer, 0.5 uL of each restriction enzyme and water to 50uL. Samples were put at 37°C during one hour and then stored on ice to be load on agarose gel.
 
For GA1, we should obtain 2 fragments, of 3273 and 1356 base pairs. For GA2, we should obtain two fragments of 3273 and 2142 base pairs.
 
 
 
[GEL]
 
Conclusion:
The insert for GA2 seemed to be smaller than expected. At this point, we also decided to discard sample B3.
 
 
 
 
Restriction Analysis 2
We tried another restriction analysis. Each sample was used for a simple and a double digestion. We compared the obtained fragments to the supercoiled plasmids.
 
 
 
Reaction Set up
 
 
Sample
Name
NheI
Hind III
XbaI
Cutsmart Buffer
Template (500ng)
Water
Total
1
GA1 A1
0.5
 
 
5
2.81
41.69
50
2
GA1 A2
0.5
 
 
5
2.00
42.5
50
3
GA1 A4
0.5
 
 
5
2.71
41.79
50
4
GA1 B1
0.5
 
 
5
4.42
40.08
50
5
GA1 B2
0.5
 
 
5
2.50
42
50
6
GA1 B4
0.5
 
 
5
3.50
41
50
7
GA2 A3
<
Reparation and improvement of the Biobrick BBa_K339007
09-16-2014
 
 
Experiment 1: PCR amplification
Our sequencing results of the BioBrick BBa_K3396007 showed us that the CpxR promoter is not infroant of the RFP coding sequence. In order to improve that of the iGEM library we repair the BBa_K3396007 meaning to put the CpxR promoter in front of the RFP, which is not the case jet as we realized after sequencing. We also want to test its sensitivity to different kinds of stress.
1.1       Material and methods
BBa_K3396007
Primers :                  AD_01_00_fw
                               AD_01_00_rv
Phiusion PCR kit
 
 
1
2
BBa_ K3396007
0.1 µl
0.1 µl
AD_01_00_fw
2.5 µl
2.5 µl
AD_01_00_rv
2.5 µl
2.5 µl
Mastermix
11.5 µl
11.5 µl
H2O
33.4 µl
33.4 µl
Total volume
50 µl
50 µl
 
PCR program :
                   Tem                 Time          repeat from step               repetitions
       1:         98 °C               30 s
       2:         98 °C               10 s
       3:         69 °C               30 s
       4:         72 °C               4 min               2                                 29
       5:         72 °C               10 min
 
 
1.2       Results
Expected (thoretical) Result :
 
Agarose gels (expected vs real):
           
 
Nano Drop results:
Sample NR
Product Name
Concentration
A260/A280
A260/A230
1
AD_01_00
45.8 ng/µl
1.88
2.24
2
AD_01_00
74.3 ng/µl
1.87
2.25
 
1.2.1     Interpretation
The fragments have the expected size. For further experiments Sample nr 2 will be used.
Experiment 2: PCR-product digestion with DPN1
The objective of this step is to digest the plasmids that served as a template, so they can not be transformed in later experiments.
2.1       Material and methods
DpnI (NEB)
Neb CutSmart buffer
AD_01_00 (sample 2)
PCR-purification Kit (Qiagen)
Tube nr
1
AD_01_00
8 µl
Dpn1
1 µl
Cutsmart buffer
1 µl
Total Volume
10 µl
Incubate at 37 °C for 30 min
Head-inactivation by incubating at 80 °C for 20 min
PCR-Purification (Qiagen)
2.2       Results
Sample NR
Concentration
A260/A280
A260/A230
AD_01_00
13.4 ng/µl
1.75
2.11
2.2.1     Interpretation
The concentration of any remaning plasmid in the sample would be to low to see on a gel. The concentration is very lowbut suffisient to run a PCR on it.
 
Experiment 3: PCR Amplification and Extention
The objective is to add the CpxR promoter to the linear sequence by PCR. The sequence is in the primers.
3.1       Material and methods
Template: AD_01_00 (sample 2)
Primers :
AD_01_01_fw
AD_01_01_rv
Phiusion PCR Kit
 
 
1
2
AD_01_00
0.1 µl
0.1 µl
AD_01_01_fw
2.5 µl
2.5 µl
AD_01_01_rv
2.5 µl
2.5 µl
Mastermix
11.5 µl
11.5 µl
H2O
33.4 µl
33.4 µl
Total volume
50 µl
50 µl
 
PCR program :
       1:         98 °C               30 s
       2:         98 °C               10 s
       3:         69 °C               30 s
       4:         72 °C               4 min               2          29
       5:         72 °C               10 min
 
3.2       Results
Expected result :
 Gels (Theoretical vs real):
      
 
Nano Drop results:
Sample NR
Product Name
Concentration
A260/A280
A260/A230
1
AD_01_01
43.8 ng/µl
1.88
2.24
2
AD_01_01
77.3 ng/µl
1.87
2.25
 
3.2.1     Interpretation
Sample 2 will be used for the sitedirected mutagenesis
4        Experiment 3: site directed mutagenesis
The purpose of this experiment is to close the linear plasmid with a enzymatic reaction.
 
4.1       Material and methods
Materials:
Q5® Site-Directed Mutagenesis Kit (NEB)
AD_01_01 sample 2
 
Methode:
 Sample name 
 AD_01
AD_01_01
1 µl
2X KLD Reaction Buffer
5 µl
10X KLD Enzyme Mix
1 µl
Nuclease free H2O
3 µl
Totol
10 µl
 Incubated for 5 min at RT.
 
4.2       Results
Expected Results:
 
Sequencing results of the CpxR Promoter in front of the RFP show that the matagenesis worked and that there ane no muations in the sequence of interest:
 
1        Experiment Plate reader with the repaired BBa_K339007
The aim of this experiment is to verifie if the biobrick workes the way it claimes to work
1.1       Material and Methods
96-Well plate
Cells with the BBa_K339007
DH5 α
PBS (1X)
KCl 80 mM
KCl 160 mM
10% Ethanol
20% Ethanol
 
After centrifuging the cells they where resuspended in PBS and then in each well of the plate there was put 50 µl of either cell type. The wells where then filled up with either kind of liquid (50 µl):
  
Cells
Name
Fill up (50 µl)
Stress
BBa_K339007 transformed
RFP -
PBS
No stress
BBa_K339007 transformed
RFP + 40 mM KCl
80 mM KCl
40 mM KCl
BBa_K339007 transformed
RFP + 80 mM KCl
160 mM KCl
80 mM KCl
BBa_K339007 transformed
RFP + 5% Ethanol
10 % Ethanol
5 % Ethanol
BBa_K339007 transformed
RFP + 10% Ethanol
20 % Ethanol
10 % Ethanol
DH5 α
Neg -
PBS
No stress
DH5 α
Neg + 40 mM KCl
80 mM KCl
40 mM KCl
DH5 α
Neg + 80 mM KCl
160 mM KCl
80 mM KCl
DH5 α
Neg + 5% Ethanol
10 % Ethanol
5 % Ethanol
DH5 α
Neg + 10% Ethanol
20 % Ethanol
10 % Ethanol
 
The plate was then put in the plate reader where for 3h every 5 min the OD and the fluorescence was measured (exitation: 555 nm ; emission : 632)
 
1.2       Results
 
Neg stands for negative control (cells lacking the RFP coding sequence).
 
1.3       Interpretation
We successfully improved the BBa_K339007 by putting the missing sequence in front of the RFP. With the further documentation we discovered that the promoter is not specific for dimerised CpxR.
 
2        Experiment. PCR Amplifie the Backbone for the Gibson
To obtain the necessary backbone for the assembly.
 
2.1       Material and methods
 
Template :                AD_01
Primers :                  AD_01_00_rv
                               NH_04_00_fw
Phusion PCR kit
 
 
1
2
AD_01_00
0.1 µl
0.1 µl
NH_04_00_fw
2.5 µl
2.5 µl
AD_01_00_rv
2.5 µl
2.5 µl
Mastermix
11.5 µl
11.5 µl
H2O
33.4 µl
33.4 µl
Total volume
50 µl
50 µl
 
PCR program :
       1:         98 °C               30 s
       2:         98 °C               10 s
       3:         69 °C               30 s
       4:         72 °C               2 min               2          29
       5:         72 °C               10 min
 
3        Experiment 6 : Gibson assembly
To obtain a plasmid with RFP wiht the native promoter that is found in front of the CpxA and the CpxP
 
3.1       Materials and Methodes
Fragments :             NH_04_00 (sample 1)
                               NH_04_01
 
 
Moles
Mass
NH_04_A
NH_04_B
NH_04_00
0.03 pmol
54.0 ng
0.72 µl
0.72 µl
NH_04_01
0.05 pmol
66.7 ng
3.3 µl
---
NH_04_02
0.05 pmol
66.7 ng
---
3.3 µl
H2O
 
 
5.98 µl
5.98 µl
Gibson Master Mix
 
 
10 µl
10µl
Total
 
 
20 µl
20 µl
 
Incubated samples at 50°C for 15 min according to the NEB protocol
 
3.2       Results
The resulting plasmids are the BBa_K1486049 (NH_04_00 + NH_04_01) and the BBa_K1486050 (NH_04_00 + NH_04_01)
Sequencing reveald that there where no mutations.
 
 
 
 
 
Characterisation of BBa_K1486049 and BBa_K1486050
To charcterise theyre behavior in response to stress those two biobricks and the BBa_K1486048 were put in PBS (negative control) and in PBS + 80 mM KCl (stress). The samples were loaded on a 96-well plate reader. The light emission (ex: 555 nm em: 632 nm) and the OD was measured during several houers.
Results
As we can see, there is no stress sensitivity. The improvement we made is that our promoter is no "leaky".
 
 
Primer librery
Primer name
Sequence (5’ -> 3’)
AD_01_00_fw
AAAGAGGAGAAATACTAGATGGCTTCCTCC
AD_01_00_rv
CTCTAGAAGCGGCCGCGA
AD_01_01_fw
atgacgtaatttctgcctctactagagAAAGAGGAGAAATA
CTAGATGGCTTCCTCC
AD_01_01_rv
cagacgtcgctaatccatgactttacgttgttttacCTCTAGAAGCGGCCGCGA
NH_04_00
ATGGCTTCCTCCgaagacgttatc
 
Ordered sequences
Name
Sequence (5’ -> 3’)
NH_04_01
atgatttctggaatTCGCGGCCGCTTCTAGAGTTGCTCCCAAAATCTTTC
TGTCGCGATTCAACGATAGAGAGTTTACGATTCAGGCTGCAAACA
TGCGTCAGGGGGTGTAAAACAACGTAAAGTCATGGATTAGCGACG
TCTGATGACGTAATTTCTGCCTCGGAGGTATTTAAACAATGGCTTCC
TCCgaagacgttatcaaagagttca
NH_04_02
atgatttctggaatTCGCGGCCGCTTCTAGAGTGTTTAAATACCTCCGAGG
CAGAAATTACGTCATCAGACGTCGCTAATCCATGACTTTACGTTGTT
TTACACCCCCTGACGCATGTTTGCAGCCTGAATCGTAAACTCTCTAT
CGTTGAATCGCGACAGAAAGATTTTGGGAGCAAATGGCTTCCTCCga
agacgttatcaaagagt
 
 
 
 
 
July
07-07-2014
07-13-2014
07-20-2014
07-21-2014
07-27-2014
07-28-2014
August
08-03-2014
08-10-2014
08-12-2014
08-17-2014
08-24-2014
September
09-16-2014
Sponsors
Copyright © iGEM EPFL 2014. All Rights Reserved
Retrieved from " http://2014.igem.org/Team:EPF_Lausanne/Notebook/Bacteria "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
