<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_EPF_Lausanne_Notebook_Yeast"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:EPF_Lausanne/Notebook/Yeast">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:EPF_Lausanne/Notebook/Yeast&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:EPF_Lausanne/Notebook/Yeast&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:EPF_Lausanne/Notebook/Yeast&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:EPF_Lausanne/Notebook/Yeast" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:EPF Lausanne/Notebook/Yeast</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><DIV class="nav-collapse"><UL class="nav"><LI><A href="https://2014.igem.org/Team:EPF_Lausanne">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Project </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Overview">Overview</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Envelope_stress_responsive_bacteria">Stress Responsive Bacteria</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Yeast">Osmo Responsive Yeast</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Microfluidics">Microfluidics</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Hardware">Hardware</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Applications">Applications</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Achievements </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Data">Data</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Judging">Judging</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Parts">Parts</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Policy &amp; Practices </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/HumanPractice">Human Practices</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Safety">Bio Safety</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/PolicyPractice">Metafluidics</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle active" data-toggle="dropdown">Notebook </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Notebook/Bacteria">Bacteria</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Notebook/Yeast" class="active">Yeast</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Notebook/Microfluidics">Microfluidics</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Protocol">Protocols</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Notebook">Timeline</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Team">Meet us!</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Acknowledgments">Acknowledgments</A></LI></DIV></UL></DIV></DIV><DIV class="container"><DIV class="box" id="boxbread"><OL class="breadcrumb breadcrumb-arrow"><LI><A href="https://2014.igem.org/Team:EPF_Lausanne"> Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Notebook </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Notebook/Bacteria">Bacteria</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Notebook/Microfluidics">Microfluidics</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Protocol">Protocols</A></LI><LI class="active"><SPAN> Yeast</SPAN></LI></OL></DIV><DIV class="row"><DIV class="col col-md-9"><DIV class="whitebg box"><H3 id="date-08-01-2014">Final constructs and primers design </H3><SPAN>08-01-2014</SPAN><DIV class="notebook-content"><H3 style="text-align: right;"><SPAN style="text-decoration: underline;"><STRONG>15.07.14-01.08.14</STRONG></SPAN></H3><P style="text-align: right;"> </P><H2 style="text-align: center;"><SPAN style="text-decoration: underline;"><STRONG>Final constructs and primers design</STRONG></SPAN></H2><P> </P><P>After several weeks of brainstorming, doubts and research, we finally have all our primers for the PCRs required to assemble our different constructs. </P><P>Our design is summarized in the table below. You can also download our primers sequences <A href="https://2014.igem.org/File:Primer_sequences_yeast_bioPad.xls">here</A>.</P><P> </P><P> </P><P> </P><P style="text-align: center;"> </P><P style="text-align: left;"> </P></DIV></DIV><DIV class="whitebg box"><H3 id="date-08-07-2014">Extraction of Renilla Luciferase and Split Renilla Luciferase from Waldor's plasmids</H3><SPAN>08-07-2014</SPAN><DIV class="notebook-content"><P style="text-align: right;"><STRONG>07.08.14</STRONG></P><H1 style="text-align: left;"> </H1><H1 style="text-align: left;"><STRONG>PCR of - RLuc + addition of hog1 flanking sequence H1</STRONG><STRONG>- RLuc + addition of pbs2 flanking sequence P1</STRONG><STRONG> - split RLucN + pbs2 flanking sequence P2</STRONG><STRONG> - split RLucC + hog1 flanking sequence H2</STRONG></H1><P> </P><P> </P><H1 class="western" lang="en-US">1 Experiment 1: First try</H1><P> </P><H2 class="western" lang="en-US">1.1Material and methods</H2><P>Master Mix</P><P>5x phusion GC buffer 85 μL</P><P>10mM dNTPs 8,5 μL</P><P>Phusion polymerase 4,25 μL</P><P> </P><P>Reaction:</P><TABLE class="table table-striped"><TBODY><TR><TD width="128"><P> </P></TD><TD width="128"><P>H1</P></TD><TD width="128"><P>H2</P></TD><TD width="128"><P>P1</P></TD><TD width="128"><P>P2</P></TD></TR><TR><TD width="128"><P>Master Mix</P></TD><TD width="128"><P>11,5</P></TD><TD width="128"><P>11,5</P></TD><TD width="128"><P>11,5</P></TD><TD width="128"><P>11,5</P></TD></TR><TR><TD width="128"><P>Forward primer 10mM</P></TD><TD width="128"><P>2.5</P></TD><TD width="128"><P>2,5</P></TD><TD width="128"><P>2,5</P></TD><TD width="128"><P>2,5</P></TD></TR><TR><TD width="128"><P>Reverse primer</P><P>10mM</P></TD><TD width="128"><P>2,5</P></TD><TD width="128"><P>2,5</P></TD><TD width="128"><P>2,5</P></TD><TD width="128"><P>2,5</P></TD></TR><TR><TD width="128"><P>Template pRLuc ou pRLucNRLucC</P></TD><TD width="128"><P>0.21</P></TD><TD width="128"><P>0,2</P></TD><TD width="128"><P>0,21</P></TD><TD width="128"><P>0,2</P></TD></TR><TR><TD width="128"><P>Nuclease Free Water</P></TD><TD width="128"><P>33.32</P></TD><TD width="128"><P>33,33</P></TD><TD width="128"><P>33,33</P></TD><TD width="128"><P>33,33</P></TD></TR></TBODY></TABLE><P> </P><P>PCR programs: H1, H2 HLuc1, P1, P2 PLuc1</P><P> </P><H2 class="western" lang="en-US">1.2Results</H2><H3 class="western" lang="en-US">1.2.1Data</H3><P> </P><P> </P><P> </P><P> </P><H3 class="western" lang="en-US">1.2.2Interpretation</H3><P>It didn't work so we tried it a second time.</P><P> </P><P> </P><H1 class="western" lang="en-US">2 Experiment 2: Second try</H1><P> </P><H2 class="western" lang="en-US">2.1Material and methods</H2><P>Master Mix</P><P>5x phusion GC buffer 85 μL</P><P>10mM dNTPs 8,5 μL</P><P>Phusion polymerase 4,25 μL</P><P> </P><P><SPAN style="text-decoration: underline;">Remark</SPAN>: We added more template to the reaction: 3 μL (60ng/μL solution).</P><P> </P><TABLE class="table table-striped"><TBODY><TR><TD width="128"><P> </P></TD><TD width="128"><P>H1</P></TD><TD width="128"><P>H2</P></TD><TD width="128"><P>P1</P></TD><TD width="128"><P>P2</P></TD></TR><TR><TD width="128"><P>Master Mix</P></TD><TD width="128"><P>11,5</P></TD><TD width="128"><P>11,5</P></TD><TD width="128"><P>11,5</P></TD><TD width="128"><P>11,5</P></TD></TR><TR><TD width="128"><P>Forward primer 10mM</P></TD><TD width="128"><P>2.5</P></TD><TD width="128"><P>2,5</P></TD><TD width="128"><P>2,5</P></TD><TD width="128"><P>2,5</P></TD></TR><TR><TD width="128"><P>Reverse primer</P><P>10mM</P></TD><TD width="128"><P>2,5</P></TD><TD width="128"><P>2,5</P></TD><TD width="128"><P>2,5</P></TD><TD width="128"><P>2,5</P></TD></TR><TR><TD width="128"><P>Template pRLuc ou pRLucNRLucC</P></TD><TD width="128"><P>3</P></TD><TD width="128"><P>3</P></TD><TD width="128"><P>3</P></TD><TD width="128"><P>3</P></TD></TR><TR><TD width="128"><P>Nuclease Free Water</P></TD><TD width="128"><P>30,75</P></TD><TD width="128"><P>30,75</P></TD><TD width="128"><P>30,75</P></TD><TD width="128"><P>30,75</P></TD></TR></TBODY></TABLE><P> </P><H2 class="western" lang="en-US">1.2Results</H2><H3 class="western" lang="en-US">1.2.1Data</H3><P> </P><H3 class="western" lang="en-US">2.2.2Interpretation</H3><P>Only one PCR worked: the C terminal part of the split renilla luciferase was successfully amplified.</P><P>Expected amplicons size: 695 bp</P><P>Nanodrop after purification: 89,2 ng/μL</P><P>Note: three tubes exploded</P><P> </P><P>What we have: RLuc + hog1 flanking sequences</P><P><A href="https://static.igem.org/mediawiki/2014/2/25/Hog.JPG">https://static.igem.org/mediawiki/2014/2/25/Hog.JPG</A></P><P> </P><P> </P><P> </P><P> </P><P> </P><P> </P><P> </P></DIV></DIV><DIV class="whitebg box"><H3 id="date-08-08-2014">Gradient PCR of problematic sequences</H3><SPAN>08-08-2014</SPAN><DIV class="notebook-content"><P style="text-align: right;"><STRONG>08.08.14</STRONG></P><P style="text-align: center;"><STRONG> </STRONG></P><P style="text-align: center;"> </P><H1 class="western" lang="en-US"><STRONG>Gradient PCR of - RLuc + addition of hog1 flanking sequence H1</STRONG><STRONG>   - RLuc + addition of pbs2 flanking sequence P1</STRONG><STRONG>  - split RLucN + pbs2 flanking sequence P2</STRONG></H1><P><STRONG>  </STRONG></P><H2 class="western" lang="en-US">1.Material and methods</H2><P> </P><P> </P><P>Since we had some trouble with our PCRs and our primers are quite long we decided to try a gradient PCR.</P><P>Master Mix</P><P>GC buffer 250μL</P><P>dNTPs 25μL</P><P>Phusion 12,5μL</P><P> </P><P>Reaction</P><TABLE class="table table-striped"><TBODY><TR><TD width="160"><P> </P></TD><TD width="160"><P>H1</P></TD><TD width="160"><P>P1</P></TD><TD width="160"><P>P2</P></TD></TR><TR><TD width="160"><P>Master Mix</P></TD><TD width="160"><P>11,5</P></TD><TD width="160"><P>11,5</P></TD><TD width="160"><P>11,5</P></TD></TR><TR><TD width="160"><P>Forward primers 10mM</P></TD><TD width="160"><P>2,5</P></TD><TD width="160"><P>2,5</P></TD><TD width="160"><P>2,5</P></TD></TR><TR><TD width="160"><P>Reverse primers 10mM</P></TD><TD width="160"><P>2,5</P></TD><TD width="160"><P>2,5</P></TD><TD width="160"><P>2,5</P></TD></TR><TR><TD width="160"><P>Template pRLuc</P></TD><TD width="160"><P>0,6</P></TD><TD width="160"><P>0,6</P></TD><TD width="160"><P>0,6</P></TD></TR><TR><TD width="160"><P>Water</P></TD><TD width="160"><P>33</P></TD><TD width="160"><P>33</P></TD><TD width="160"><P>33</P></TD></TR></TBODY></TABLE><P> </P><P>Temperature gradient:</P><P>H1 64à72°C</P><P>P1, P2 55à63°C</P><P> </P><P>We also increase the extension time: 30secondesà1min30seconds!</P><P>We used the Invitrogen 1kb Plus DNA Ladder. In each well of a 1% agarose gel, we loaded 8.8ul Nuclease free Water, 2ul of PCR products or Ladder and 1.2ul 10X Loading Blue Dye. The electrophoresis ran 40 minutes at 120V.</P><P> </P><P> </P><H2 class="western" lang="en-US">2.Results</H2><P>H1</P><P>We purified two PCR samples for each experiment and measured their concentration and purity with the Nanodrop.</P><P> </P><P> </P><P> </P><P> </P><P> </P><P> </P><P> </P></DIV></DIV><DIV class="whitebg box"><H3 id="date-08-12-2014">Extraction of yeast resistance genes from psfGFPkan and psfGFPUra plasmids</H3><SPAN>08-12-2014</SPAN><DIV class="notebook-content"><P style="text-align: right;"><STRONG> 12.08.14</STRONG></P><P><STRONG> </STRONG></P><P style="text-align: center;"> </P><H1 class="western" lang="en-US"><STRONG>1. Experiment 1: </STRONG></H1><H1 class="western" lang="en-US"><STRONG>PCR of </STRONG><STRONG>- sfGFP+hog1 flanking sequence H4 </STRONG><STRONG>- split sfGFPC+hog1 flanking sequence H5 </STRONG><STRONG>- URA3+ rLuc overlap H3 </STRONG><STRONG>- sfGFP+pbs2 flanking sequence P3 </STRONG><STRONG>- split sfGFPN+pbs2 flanking sequence P4 </STRONG><STRONG>- Kan+ rLuc overlap P5 </STRONG><STRONG>- Kan+ nLuc overlap P6 </STRONG><STRONG>- Kan+ sfGFPN overlap P7 </STRONG></H1><H1 class="western" lang="en-US"><STRONG>(First try)</STRONG></H1><P> </P><P><STRONG>  </STRONG></P><P> </P><H2 class="western" lang="en-US">1.1.Material and methods</H2><P> </P><TABLE class="table table-striped"><TBODY><TR><TD width="321"><P>Template psfGFPURA3 or psfGFPKan</P></TD><TD width="321"><P>0.6</P></TD></TR><TR><TD width="321"><P>Forward primer 10mM</P></TD><TD width="321"><P>2,5</P></TD></TR><TR><TD width="321"><P>Reverse primer 10mM</P></TD><TD width="321"><P>2,5</P></TD></TR><TR><TD width="321"><P>Master Mix (with GC buffer)</P></TD><TD width="321"><P>11,5</P></TD></TR><TR><TD width="321"><P>Water</P></TD><TD width="321"><P>33</P></TD></TR></TBODY></TABLE><P> </P><H2 class="western" lang="en-US">1.2Results</H2><P>Results visualized by gel electrophoresis (1% agar, 120V, 40min)</P><P>Constructs we should have now:</P><H1 class="western" lang="en-US"> </H1><H1 class="western" lang="en-US"> </H1><H1 class="western" lang="en-US"><STRONG>2. Experiment 2:</STRONG></H1><H1 class="western" lang="en-US"><STRONG>PCR of </STRONG><STRONG>- sfGFP+hog1 flanking sequence H4 </STRONG><STRONG>- split sfGFPC+hog1 flanking sequence H5 </STRONG><STRONG>- URA3+ rLuc overlap H3 </STRONG><STRONG>- sfGFP+pbs2 flanking sequence P3 </STRONG><STRONG>- split sfGFPN+pbs2 flanking sequence P4 </STRONG><STRONG>- Kan+ rLuc overlap P5 </STRONG><STRONG>- Kan+ nLuc overlap P6 </STRONG><STRONG>- Kan+ sfGFPN overlap P7</STRONG></H1><H1 class="western" lang="en-US"><STRONG>(Second try)</STRONG></H1><P>H3, H4, H5 second try: Gradient PCR</P><P>Temperature gradient 58°Cà66°C</P><P> </P><P>25μL reaction (instead of 50μL)</P><P> </P><P>For all reactions:</P><TABLE class="table table-striped"><TBODY><TR><TD width="321"><P>Template psfGFPURA3</P></TD><TD width="321"><P>0,6</P></TD></TR><TR><TD width="321"><P>Forward primer 10mM</P></TD><TD width="321"><P>1,25</P></TD></TR><TR><TD width="321"><P>Reverse primer 10mM</P></TD><TD width="321"><P>1,25</P></TD></TR><TR><TD width="321"><P>Master Mix (with GC buffer)</P></TD><TD width="321"><P>5,75</P></TD></TR><TR><TD width="321"><P>DMSO</P></TD><TD width="321"><P>0,75</P></TD></TR><TR><TD width="321"><P>Water</P></TD><TD width="321"><P>15,4</P></TD></TR></TBODY></TABLE><P> </P><H2 class="western" lang="en-US">2.2Results</H2><P> </P><P> </P><P> </P><P> </P></DIV></DIV><DIV class="whitebg box"><H3 id="date-08-14-2014">Overlap Extension PCRs: test with only one construct</H3><SPAN>08-14-2014</SPAN><DIV class="notebook-content"><P style="text-align: right;"><STRONG>14.08.14</STRONG></P><P><STRONG> </STRONG></P><H1 style="text-align: left;"><STRONG>Overlap extension PCR of</STRONG><STRONG>- rLuc-Hog1+Ura3 H6 </STRONG><STRONG>PCR of</STRONG><STRONG>split sfGFPC+hog1 flanking sequence H5</STRONG></H1><H1><STRONG> </STRONG></H1><H3><STRONG>Overlap extension PCR</STRONG></H3><P>25μL reaction</P><P>Template: 1μL (60ng) H1</P><P>                 1,7μL (100ng) H3</P><P> </P><P>15 cycles with 62°C of annealing temperature</P><P>+15 cycles with 62°C of annealing temperature after addition of primers H7 and H1_rv</P><P> </P><H3><STRONG>PCR H5</STRONG></H3><P>25μL reaction with good primers this time !</P><P> </P><H3><STRONG>Results</STRONG></H3><P> </P><P>Our first overlap PCR worked!!! :)</P><P> </P></DIV></DIV><DIV class="whitebg box"><H3 id="date-08-15-2014">Overlap Extension PCRs: Final constructs assembly</H3><SPAN>08-15-2014</SPAN><DIV class="notebook-content"><P style="text-align: right;"><STRONG>15.08.14</STRONG></P><P><STRONG> </STRONG></P><P style="text-align: center;"> </P><H1 class="western" lang="en-US"><STRONG>Overlap extension PCR of</STRONG><STRONG>- Spit LucC-Hog1+Ura3 H7 </STRONG><STRONG>-split GFP-N+hog1+Kan P8 </STRONG><STRONG>-rLuc-pbs2+Kan P9 </STRONG><STRONG>-split rLucN-pbs2+Kan P10 </STRONG></H1><H1 class="western" lang="en-US"><STRONG>Extension PCR of </STRONG><STRONG>Split GFP-C H55 (repeat previous experiment)</STRONG></H1><P> </P><P><STRONG> </STRONG></P><H2 class="western" lang="en-US">1.Material and methods</H2><P> </P><P>30μL reactions</P><P>Equimolar template ratios: we used 100 ng of the longest template, and estimated the necessary amount of the shortest template with its size.</P><TABLE class="table table-striped"><TBODY><TR><TD width="106"><P>In μL</P></TD><TD width="106"><P>Master Mix</P></TD><TD width="106"><P>Template 1</P></TD><TD width="143"><P>Template 2 (100ng)</P></TD><TD width="70"><P>Water</P></TD><TD width="148"><P>Primers*</P></TD></TR><TR><TD width="106"><P>P8: P4+P7</P></TD><TD width="106"><P>5.75</P></TD><TD width="106"><P>P4: 0.83</P></TD><TD width="143"><P>P7: 2</P></TD><TD width="70"><P>16.4</P></TD><TD width="148"><P>2.5 each (p12/p2)</P></TD></TR><TR><TD width="106"><P>P9: P1+P5</P></TD><TD width="106"><P>5.75</P></TD><TD width="106"><P>P1: 0.53</P></TD><TD width="143"><P>P5: 1</P></TD><TD width="70"><P>17.7</P></TD><TD width="148"><P>2.5 each (p12/p2)</P></TD></TR><TR><TD width="106"><P>P10: P2+P6</P></TD><TD width="106"><P>5.75</P></TD><TD width="106"><P>P2: 0.4</P></TD><TD width="143"><P>P6: 2</P></TD><TD width="70"><P>16.9</P></TD><TD width="148"><P>2.5 each (p12/p2)</P></TD></TR><TR><TD width="106"><P>H7: H3+H2</P></TD><TD width="106"><P>5.75</P></TD><TD width="106"><P>H1: 1</P></TD><TD width="143"><P>H3: 1.7</P></TD><TD width="70"><P>15.4</P></TD><TD width="148"><P>2.5 each (h7/h1.Rv)</P></TD></TR></TBODY></TABLE><P> </P><P>*Add primers after the first program (after fusion of the templates).</P><P> </P><P><STRONG>Thermocycler programs :</STRONG></P><P>P8,9,10 : 15 cycles at 60°C for overlap, add primers, then 15 cycles at 72°C annealing. Always 2min30 of extension time at 72°C.</P><P> H7: 15 cycles at 62°C for overlap, add primers, then 15 cycles at 62°C annealing. Always 2min30 of extension time at 72°C.</P><P> </P><H2 class="western" lang="en-US">2.Results</H2><P> </P><P> </P></DIV></DIV><DIV class="whitebg box"><H3 id="date-08-18-2014">Overlap Extension PCRs: Final constructs assembly 2</H3><SPAN>08-18-2014</SPAN><DIV class="notebook-content"><P style="text-align: right;"><STRONG>18.08.14</STRONG></P><P><STRONG> </STRONG></P><P style="text-align: center;"> </P><P style="text-align: center;"> </P><H1 class="western" lang="en-US">1.Experiment: <STRONG>Overlap extension PCR of</STRONG><STRONG>-split GFP-N+hog1+Kan P8 </STRONG><STRONG>-rLuc-pbs2+Kan P9 </STRONG><STRONG>-split rLucN-pbs2+Kan P10</STRONG></H1><P><STRONG> </STRONG></P><P> </P><H2 class="western" lang="en-US">1.1Material and methods</H2><P> </P><P>25μL reactions</P><P>Equimolar template ratios: we used 200 ng of the longest template, and estimated the necessary amount of the shortest template with its size.</P><P> </P><TABLE class="table table-striped"><TBODY><TR><TD width="96"><P>In μL</P></TD><TD width="106"><P>Master Mix (GC buffer)</P></TD><TD width="106"><P>Template 1</P></TD><TD width="143"><P>Template 2 (100ng)</P></TD><TD width="70"><P>Water</P></TD><TD width="148"><P>Primers*</P></TD></TR><TR><TD width="96"><P>P8: P4+P7</P></TD><TD width="106"><P>5.75</P></TD><TD width="106"><P>P4: 1.6</P></TD><TD width="143"><P>P7: 4</P></TD><TD width="70"><P>8.75</P></TD><TD width="148"><P>2.5 each (p12/p2)</P></TD></TR><TR><TD width="96"><P>P9: P1+P5</P></TD><TD width="106"><P>5.75</P></TD><TD width="106"><P>P1: 1.1</P></TD><TD width="143"><P>P5: 2</P></TD><TD width="70"><P>11.2</P></TD><TD width="148"><P>2.5 each (p12/p2)</P></TD></TR><TR><TD width="96"><P>P10: P2+P6</P></TD><TD width="106"><P>5.75</P></TD><TD width="106"><P>P2: 0.8</P></TD><TD width="143"><P>P6: 4</P></TD><TD width="70"><P>9.5</P></TD><TD width="148"><P>2.5 each (p12/p2)</P></TD></TR><TR><TD colspan="6" width="668"><P>Duplicates were made</P></TD></TR></TBODY></TABLE><P> </P><P>*Add primers after the first program (after fusion of the templates).</P><P> </P><P><STRONG>Thermocycler programs : </STRONG>P8,9,10 : 15 cycles at 63°C for overlap, add primers, then 15 cycles at 70°C annealing. Always 2min30 of extension time at 72°C.</P><P> </P><H2 class="western" lang="en-US">1.2Results:</H2><P> </P><P> </P><P>The fusion of all the fragments seems to have proceeded (thanks to 63°C annealing temperature, the primers of the previous PCR that were not washed away could not interfere), but the 70°C temperature chosen for the amplification doesn’t work for P8 and P10. Next time: DMSO duplicates and annealing of 72°C.</P><P>It isn’t impossible that 200ng of the largest template is way too high for a 25ul reaction: next time, only 50ng. </P><P> </P><P style="text-align: center;"> </P><H1 class="western" lang="en-US">2.Experiment: <STRONG>Overlap extension PCR of  </STRONG><STRONG>-split GFP-N+hog1+Kan P8  </STRONG><STRONG>-split rLucN-pbs2+Kan P10</STRONG></H1><P><STRONG> </STRONG></P><P> </P><H2 class="western" lang="en-US">2.1Material and methods</H2><P> </P><P>25μL reactions</P><P>Equimolar template ratios: we used 50 ng of the longest template, and estimated the necessary amount of the shortest template with its size.</P><TABLE class="table table-striped"><TBODY><TR><TD width="96"><P>In μL</P></TD><TD width="106"><P>Master Mix (GC buffer)</P></TD><TD width="106"><P>Template 1</P></TD><TD width="143"><P>Template 2 (100ng)</P></TD><TD width="70"><P>Water</P></TD><TD width="148"><P>Primers*</P></TD></TR><TR><TD width="96"><P>P8: P4+P7</P></TD><TD width="106"><P>5.75</P></TD><TD width="106"><P>P4: 2.76</P></TD><TD width="143"><P>P7: 2</P></TD><TD width="70"><P>9.49</P></TD><TD width="148"><P>2.5 each (p12/p2)</P></TD></TR><TR><TD width="96"><P>P10: P2+P6</P></TD><TD width="106"><P>5.75</P></TD><TD width="106"><P>P2: 0.4</P></TD><TD width="143"><P>P6: 2</P></TD><TD width="70"><P>11.1</P></TD><TD width="148"><P>2.5 each (p12/p2)</P></TD></TR><TR><TD colspan="6" width="668"><P>Duplicates with DMSO were made -&gt; Add 0.75ul DMSO and 0.75ul of water</P></TD></TR></TBODY></TABLE><P> </P><P>*Add primers after the first program (after fusion of the templates).</P><P> </P><P><STRONG>Thermocycler programs :</STRONG>P8,10 : 15 cycles at 63°C for overlap, add primers, then 15 cycles at 72°C annealing. Always 2min30 of extension time at 72°C.</P><P> </P><H2 class="western" lang="en-US">1.2Results:</H2><P> </P></DIV></DIV><DIV class="whitebg box"><H3 id="date-08-27-2014">First transformation of yeast cells </H3><SPAN>08-27-2014</SPAN><DIV class="notebook-content"><P> </P><P align="RIGHT"> </P><P align="RIGHT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3"><STRONG>27.08.14 </STRONG></FONT></FONT></FONT></P><H1 class="western" lang="en-US">First yeast transformation</H1><H1 style="text-align: left;" align="CENTER"> </H1><H2 class="western" lang="en-US">1. Material and methods</H2><P> </P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3">Yeast transformtation and homologous recombination procedure from clontech yeast Handbook (see <EM>Protocols</EM>)</FONT></FONT></FONT></P><P align="LEFT"> </P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3">We used a 300mL culture with an OD of 0.588.</FONT></FONT></FONT></P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3">We plated all the transformation reaction tubes on selective agar plate. </FONT></FONT></FONT></P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3">Expected transformants</FONT></FONT></FONT></P><P align="LEFT"> </P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3">DNA template</FONT></FONT></FONT></P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3">- hogLucUra H10 </FONT></FONT></FONT></P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3">- pbsLucKan P13 </FONT></FONT></FONT></P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3">- hogSplitLucUra H11 </FONT></FONT></FONT></P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3">- psbSplitLucKan P14 </FONT></FONT></FONT></P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3">- hogGFPUra H4 </FONT></FONT></FONT></P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3">- pbsGFPKan P11 </FONT></FONT></FONT></P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3">- hogSplitGFPUra H9 </FONT></FONT></FONT></P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3">- pbsSplitGFPKan P12</FONT></FONT></FONT></P><P align="LEFT"> </P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3">Note: for each transformation we use between 1 and 2 microgramms of DNA template (we put the 30microliters of purified PCR products so the exact amount depends on the concentration 50-90ng/μL)</FONT></FONT></FONT></P><P align="LEFT"> </P><P align="LEFT"> </P><H2 class="western" lang="en-US">2.Results</H2><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3">We only obtained plates with pbsGFPKan transformed yeast cells. </FONT></FONT></FONT></P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3">Troubleshooting </FONT></FONT></FONT></P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3">- not enough DNA template --&gt;<FONT face="Wingdings, Wingdings, sans-serif"> </FONT>we’ll try again with more than 1ng of DNA </FONT></FONT></FONT></P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3">- poor quality and purity of our templates : we had some unexpected bands when we loaded our PCRs products and sometimes even smears </FONT></FONT></FONT></P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3">- mistakes during the transformation ? It was the first time we tried this protocol </FONT></FONT></FONT></P><P align="RIGHT"> </P><P> </P></DIV></DIV><DIV class="whitebg box"><H3 id="date-09-04-2014"> Digestion+ligation of final constructs into pSB1C3 (Second try )</H3><SPAN>09-04-2014</SPAN><DIV class="notebook-content"><P> </P><P align="RIGHT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3"><STRONG>04.09.14 </STRONG></FONT></FONT></FONT></P><P align="CENTER"> </P><H1 class="western" lang="en-US">1.<STRONG>Restriction enzyme digestion of linearized pSB1C3 and our constructs (second Try)</STRONG></H1><P align="LEFT"> </P><H2 class="western" lang="en-US">1.1 Material and methods</H2><UL><LI><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3"><FONT size="2">iGEM Restriction Digest Protocol <A href="http://parts.igem.org/Help:Protocols/Restriction_Digest">http://parts.igem.org/Help:Protocols/Restriction_Digest</A></FONT></FONT></FONT></FONT></LI></UL><P align="LEFT"> </P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="2">We prepare the following reaction according to this protocol:</FONT></FONT></FONT></P><P align="LEFT"> </P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3"><FONT size="2"><STRONG>Note: digested template amount X2 compare to first try</STRONG></FONT></FONT></FONT></FONT></P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3"><FONT size="2">Note: We let the reaction proceed overnight instead of only 30 minutes (one of our teaching assistant advised us to do so). </FONT></FONT></FONT></FONT></P><P align="LEFT"> </P><P align="LEFT"> </P><H1 class="western" lang="en-US"><STRONG>2.Ligation of our constructs into linearized pSB1C3</STRONG></H1><P align="LEFT"> </P><H2 class="western" lang="en-US">2.1 Material and methods</H2><UL><LI><FONT color="#000000"> <FONT face="Calibri, Calibri, sans-serif"><FONT size="3"><FONT size="2">iGEM Ligation Protocol <A href="http://parts.igem.org/Help:Protocols/Ligation">http://parts.igem.org/Help:Protocols/Ligation</A></FONT></FONT></FONT></FONT></LI></UL><P align="LEFT"> </P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3">Reaction Mix</FONT></FONT></FONT></P><P align="LEFT"> </P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3"><FONT size="2"><STRONG>Note: digested template 3uL (no equimolar amounts between digested plasmid and digested construct) </STRONG></FONT></FONT></FONT></FONT></P><P align="LEFT"> </P><P align="RIGHT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3"><STRONG>05.09.14 </STRONG></FONT></FONT></FONT></P><P align="CENTER"> </P><H1 class="western" lang="en-US"><STRONG>3.Transformation of DH5α with our ligation products</STRONG></H1><P align="LEFT"> </P><H2 class="western" lang="en-US">3.1 Material and methods</H2><P> </P><UL><LI><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3"><FONT size="2">iGEMTransformation Protocol <A href="http://parts.igem.org/Help:Protocols/Transformation">http://parts.igem.org/Help:Protocols/Transformation</A></FONT></FONT></FONT></FONT></LI></UL><P align="LEFT"> </P><H2 class="western" lang="en-US"> </H2><H2 class="western" lang="en-US">Results</H2><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3">We plated the cells and we did obtain many colonies but it was only the background. </FONT></FONT></FONT></P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3">We should have made a PCR purification before the ligation it would have reduce the background and we could have isolated the transformants. </FONT></FONT></FONT></P><P align="LEFT"> </P><P> </P><P> </P></DIV></DIV><DIV class="whitebg box"><H3 id="date-09-08-2014">Second transformation of yeast cells with our 8 final constructs</H3><SPAN>09-08-2014</SPAN><DIV class="notebook-content"><P> </P><P align="RIGHT"> </P><P align="RIGHT"> </P><P align="RIGHT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3"><STRONG>08.09.14-12.09.14 </STRONG></FONT></FONT></FONT></P><H1 class="western" lang="en-US">Yeasts Transformation and homologous recombination (second Try)</H1><P> </P><P> </P><P align="LEFT"> </P><H2 class="western" lang="en-US">1.Material and methods</H2><P align="LEFT"> </P><P align="LEFT"><FONT size="3" face="Calibri, Calibri, sans-serif">Yeast tranformation and homologous recombination procedure from clontech Yeast Handbook (see <EM>protocols</EM>)</FONT></P><P align="LEFT"> </P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3">We follow the same protocol as the first time but different DNA templates: we improved the PCR conditions and now have our fragments in higher concentrations and they seem quite pure.</FONT></FONT></FONT></P><H3 id="yeast transformation_protocol"> </H3><P align="LEFT"> </P><H2 class="western" lang="en-US">2.Results</H2><P> </P><H3 class="western" lang="en-US">2.1Data</H3><P> </P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3">We obtained colonies on all plates corresponding to pbs2 +… constructs! </FONT></FONT></FONT></P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3">We inoculated them in selective medium and they grew again.</FONT></FONT></FONT></P><P align="LEFT"> </P><P align="LEFT"> </P><H3 class="western" lang="en-US">2.2Interpretation</H3><P> </P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3"> We still need to sequence the genomic DNA to be sure that they integrated the right sequences and that no mutation are present but we observed the PBS2GFP cells under the microscope and they do express the PBS2 gen fused to GFP as we can see in previous photo (figure1).</FONT></FONT></FONT></P><P align="LEFT"> </P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3">Strains we might have:</FONT></FONT></FONT></P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3">- Pbs2 GFP Kan </FONT></FONT></FONT></P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3">- Pbs2 Luc Kan </FONT></FONT></FONT></P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3">- Pbs2 splitGFP Kan </FONT></FONT></FONT></P><P align="LEFT"><FONT color="#000000"><FONT face="Calibri, Calibri, sans-serif"><FONT size="3">- Pbs2 splitLuc kan </FONT></FONT></FONT></P><P align="LEFT"> </P></DIV></DIV><DIV class="whitebg box"><H3 id="date-09-25-2014">Cotransformation of N and C terminal parts of the split GFP and Renilla Luciferase</H3><SPAN>09-25-2014</SPAN><DIV class="notebook-content"><P> </P><P style="text-align: right;"><STRONG>25.09.14</STRONG></P><P style="text-align: center;"> </P><H1 class="western" lang="en-US"><STRONG>Cotransformation  </STRONG><STRONG>- split GFP  </STRONG><STRONG>-split Renilla Luciferase</STRONG></H1><H1 class="western" lang="en-US"> </H1><H2 class="western" lang="en-US">1.Aim</H2><P>Since we’re starting to run out of time, we’ve decided to try a cotransformation in parallel of the sequential transformations we did previously, one the two methods should give us the expected strains!</P><P> </P><H2 class="western" lang="en-US">2.Material and methods</H2><P> </P><UL><LI>Yeast transformation and homolgous recombination procedure from clontech Yeast Handbook (see <EM>Protocols</EM>)</LI></UL><P> </P><P>DNA Templates:</P><P>For the split GFP strain= Pbs2-split_GFP_N-Kan construct + hog1-split_GFP_C-Ura construct</P><P>For the split RLuc strain= Pbs2-split_RLuc_N-Kan construct + hog1-split_RLuc_C-Ura construct</P><P>Between 1 and 2μg (depending on the purified PCR products concentration, typically 60-100ng/μL) are used to ensure a good efficiency.</P><P> </P><P> </P><H2 class="western" lang="en-US">3.Results</H2><P> </P><P>We plated the transformants on SD Ura- Kan+ plates (our 2 selection markers), here are the results!</P><P>We can see several colonies on each of these plates whereas no colony grew on the negative control plate! The transformation seems to have worked well , we now  need to confirm it by testing the splits with plate reader experiments.</P><P> </P><P style="text-align: center;"> </P></DIV></DIV><DIV class="whitebg box"><H3 id="date-10-07-2014">Split GFP stress test</H3><SPAN>10-07-2014</SPAN><DIV class="notebook-content"><H1 class="western" lang="en-US">1 Experiment 1:  split GFP stress test</H1><H1 class="western" lang="en-US"> </H1><H2 class="western" lang="en-US">1.1 Material and methods</H2><P> </P><P>Procedure Generale Plate reader:</P><UL><LI>Culture overnight cells (not necessarily in selective medium) that you will need in the plate reader. Including the positive and negative The OD should be between 0.7 and 1.</LI><LI>Centrifuge 5 mL of our cells cultures, throw the medium and resuspend the in approximatly 800μL of PBS</LI><LI>Fill the wells with 150 μ If you want to put coelenterazine, add 135μL cell culture resuspended in PBS and 15 μL of the coelenterazine dilluated our expected concentration. If you don’t want to add coelenterazine, just add 150μL of cell culture resuspended in PBS.This you want to stress the cells, add 120μL of cell culture resuspended in PBS, 15μL of coelenterazine and 15μL of stress.</LI></UL><P> </P><P> </P><P> </P><P> </P><TABLE class="table table-striped"><TBODY><TR><TD> </TD><TD style="text-align: center;"><P>1-3:</P><P>No cells</P></TD><TD style="text-align: center;"><P>4-6:</P><P>Non transformed</P></TD><TD style="text-align: center;"><P>7-9:</P><P>PBS2</P><P>-GFP</P></TD><TD style="text-align: center;"><P>10-12:</P><P>split GFP</P></TD></TR><TR><TD>A: No stress</TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>B: Glucose 4%</TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>C: NaCl 0.07M</TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR></TBODY></TABLE><P><STRONG>Plate Layout ( 135μl of cells in PBS, 15μl of stress tested)</STRONG></P><P><STRONG> </STRONG></P><P>Non-transformed cells were grown in YPD, PBS2-GFP cells were grown in YPD supplemented with kanamycin, and split GFP strains were grown in SD-Ura dropout supplemented with kanamycin.</P><P>Non-transformed cells and PBS2-GFP cells were autofluorescent, which turned out to be a problem in th medium and a mutation in the ADE genes in the initial strain. Fluorescence normalized by OD600 seemed to drop with time contrarily to split GFP tests.</P><P> </P><H2 class="western" lang="en-US">1.2Results</H2><H3 class="western" lang="en-US">1.2.1Data</H3><P style="text-align: center;"><STRONG>figure 1:  Fluorescence of the split GFP strain with different stresses in fonction of the time.</STRONG></P><P style="text-align: center;"><STRONG>figure 2:  Average fluorescence of the split GFP strain stress by different stresses.</STRONG></P><P> </P><P> </P><H3 class="western" lang="en-US">1.2.2Interpretation</H3><P>First graph (figure1) shows the fluorescence of the split GFP strain with different stresses, Glucose 2% and NaCl 0.07M. The second graph (figure2) gives the average of the three wells. Fluorescence seems to increase but a Fisher test comparing 3 samples to 3 samples indicate that the difference isn't significant at 95%</P><P> </P></DIV></DIV><DIV class="whitebg box"><H3 id="date-10-10-2014">split Luciferase under stress Test</H3><SPAN>10-10-2014</SPAN><DIV class="notebook-content"><H1 class="western" lang="en-US">1.Experiment 1:  split Luciferase under stress test _ Plate Reader 1</H1><H1 class="western" lang="en-US"> </H1><H2 class="western" lang="en-US">1.1 Material and methods</H2><P>Procedure Generale Plate reader:</P><UL><LI>Culture overnight cells (not necessarily in selective medium) that you will need in the plate reader. Including the positive and negative The OD should be between 0.7 and 1.</LI><LI>Centrifuge 5 mL of our cells cultures, throw the medium and resuspend the in approximatly 800μL of PBS</LI><LI>Fill the wells with 150 μ If you want to put coelenterazine, add 135μL cell culture resuspended in PBS and 15 μL of the coelenterazine dilluated our expected concentration. If you don’t want to add coelenterazine, just add 150μL of cell culture resuspended in PBS.This you want to stress the cells, add 120μL of cell culture resuspended in PBS, 15μL of coelenterazine and 15μL of stress.</LI></UL><P> </P><P> </P><P>Cells were all grown in YPD.</P><P> </P><TABLE class="table table-striped"><TBODY><TR><TD style="text-align: center;"> </TD><TD style="text-align: center;"><P>1-3:</P><P>NoCells</P></TD><TD style="text-align: center;"><P>4-6:</P><P>Non-tranformed</P></TD><TD style="text-align: center;"><P>7-9:</P><P>PBS2-GFP</P></TD><TD style="text-align: center;"><P>10-12:</P><P>split GFP</P></TD></TR><TR><TD>A: No stress</TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>B: Glucose 8%</TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>C: Glucose 6%</TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>D: Glucose 4%</TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>E: NaCl 0.28M</TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>F: NaCl 0.21M</TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>G: NaCl 0.1M</TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>H: NaCl 07M</TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR></TBODY></TABLE><P><STRONG> Plate Layout (120μl of cells, 30μl of stress)</STRONG></P><P> </P><H2 class="western" lang="en-US">1.2Results</H2><P> </P><P>All cells were autofluorescent due to mutation in ADE gene of initial strain and medium.</P><P>There were no significant measurements with the different stresses, 3 different concentrations of glucose and 4 different concentrations of NaCl. However, this is not surprising since the signal from our split GFP tag would be covered by the noise.</P><P> </P><H1 class="western" lang="en-US">2.Experiment 2:  split Luciferase under stress test _ Plate reader 2</H1><H1 class="western" lang="en-US"> </H1><H2 class="western" lang="en-US">2.1 Material and methods</H2><P>Procedure Generale Plate reader:</P><UL><LI>Culture overnight cells (not necessarily in selective medium) that you will need in the plate reader. Including the positive and negative The OD should be between 0.7 and 1.</LI><LI>Centrifuge 5 mL of our cells cultures, throw the medium and resuspend the in approximatly 800μL of PBS</LI><LI>Fill the wells with 150 μ If you want to put coelenterazine, add 135μL cell culture resuspended in PBS and 15 μL of the coelenterazine dilluated our expected concentration. If you don’t want to add coelenterazine, just add 150μL of cell culture resuspended in PBS.This you want to stress the cells, add 120μL of cell culture resuspended in PBS, 15μL of coelenterazine and 15μL of stress.</LI></UL><P> </P><P>Cells were all grown in SD medium with all amino acids</P><P> </P><TABLE class="table table-striped"><TBODY><TR><TD style="text-align: center;"> </TD><TD style="text-align: center;"><P>1-3:</P><P>NoCells</P></TD><TD style="text-align: center;"><P>4-6:</P><P>Non-tranformed</P></TD><TD style="text-align: center;"><P>7-9:</P><P>PBS2-GFP</P></TD><TD style="text-align: center;"><P>10-12:</P><P>split GFP</P></TD></TR><TR><TD>A: No stress</TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>B: Glucose 8%</TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>C: Glucose 6%</TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>D: Glucose 4%</TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>E: NaCl 0.28M</TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>F: NaCl 0.21M</TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>G: NaCl 0.1M</TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>H: NaCl 07M</TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR></TBODY></TABLE><P> </P><H4 class="western" lang="en-US"><STRONG>Plate Layout (120μl of cells, 30μl of stress)</STRONG></H4><H2 class="western" lang="en-US"> </H2><H2 class="western" lang="en-US">2.2Results</H2><P>Two plate reader experiments were conducted but the cells still contained a small amount of residual auto fluorescence. However, the PBS2-GFP strain seemed to be more fluorescent than the non-transformed cells. The average in different conditions was always higher but significant in a Fisher test of three and three samples in 5 out of 8 conditions.</P><P> </P><H1 class="western" lang="en-US">3.Experiment 3:  split Luciferase under stress test _  plate Reder 3</H1><H1 class="western" lang="en-US"> </H1><H2 class="western" lang="en-US">3.1 Material and methods</H2><P>PLATE READER PROTOCOL</P><P> </P><P>Cells were all grown in SD medium with all amino acids</P><P> </P><TABLE class="table table-striped"><TBODY><TR><TD> </TD><TD><P>1-3:</P><P>No cells</P></TD><TD><P>4-6:</P><P>Non-transformed</P></TD><TD><P>7-9:</P><P>PBS2-GFP</P></TD><TD><P>10-12:</P><P>split GFP</P></TD></TR><TR><TD>A: No stress</TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>B: Glucoise 4%</TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>C: NaCl 0.07M</TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR></TBODY></TABLE><P><STRONG>Plate Layout ( 135μl of cells in PBS, 15μl of stress tested)<FONT face="Times New Roman, serif"> </FONT></STRONG></P><H2 class="western" lang="en-US">3.2Results</H2><H3 class="western" lang="en-US">3.2.1Data</H3><P><STRONG>Figure 1: Luminescence in function of time for cells in presence of coelenterazine 1 μM</STRONG></P><P>Signal-to-noise ratio : 4</P><P> </P><P><STRONG>Figure 2: Luminescence in function of time for cells in presence of coelenterazine 5 μM</STRONG></P><P>Signal-to-noise ratio : 10</P><P><STRONG>Figure 3: Variantion of coelenterazine concentration</STRONG></P><P> </P><H3 class="western" lang="en-US">3.2.2Interpretation</H3><P> </P><P>Hog-rLuc strain died within the plate. Pbs2-Luc strain showed positive results which correlate with documented values.</P><P>Indeed we observed that cells in presence of  coelenterazine have a much higher concentration than cell without coelenterazine. And  the higher the concentration in coelenterazine is the higher the fluorescent signal is.</P></DIV></DIV></DIV><DIV class="col col-md-3"><NAV id="affix-nav" class="sidebar hidden-sm hidden-xs"><UL class="nav sidenav box sidenav-hidden" data-spy="affix" data-offset-top="200" data-offset-bottom="600"><LI class="active"><A href="#date-08-01-2014">August</A></LI><LI><A href="#date-08-01-2014">08-01-2014</A></LI><LI><A href="#date-08-07-2014">08-07-2014</A></LI><LI><A href="#date-08-08-2014">08-08-2014</A></LI><LI><A href="#date-08-12-2014">08-12-2014</A></LI><LI><A href="#date-08-14-2014">08-14-2014</A></LI><LI><A href="#date-08-15-2014">08-15-2014</A></LI><LI><A href="#date-08-18-2014">08-18-2014</A></LI><LI><A href="#date-08-27-2014">08-27-2014</A></LI></UL><LI><A href="#date-09-04-2014">September</A></LI><LI><A href="#date-09-04-2014">09-04-2014</A></LI><LI><A href="#date-09-08-2014">09-08-2014</A></LI><LI><A href="#date-09-25-2014">09-25-2014</A></LI><LI><A href="#date-10-07-2014">October</A></LI><LI><A href="#date-10-07-2014">10-07-2014</A></LI><LI><A href="#date-10-10-2014">10-10-2014</A></LI></NAV></DIV></DIV></DIV><DIV id="contactSection"><DIV><H1 class="cntr">Sponsors</H1></DIV><DIV class="footerSection container"><DIV class="copyright"><P>Copyright © iGEM EPFL 2014. All Rights Reserved</P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:EPF_Lausanne/Notebook/Yeast">http://2014.igem.org/Team:EPF_Lausanne/Notebook/Yeast</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:EPF_Lausanne/Notebook/Yeast" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:EPF_Lausanne/Notebook/Yeast" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:EPF_Lausanne/Notebook/Yeast&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:EPF_Lausanne/Notebook/Yeast&amp;oldid=402521" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>