<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_EPF_Lausanne_Applications_ctp"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:EPF_Lausanne/Applications.ctp">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:EPF_Lausanne/Applications.ctp&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:EPF_Lausanne/Applications.ctp&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:EPF_Lausanne/Applications.ctp&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:EPF_Lausanne/Applications.ctp" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:EPF Lausanne/Applications.ctp</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><DIV class="nav-collapse"><UL class="nav"><LI><A href="https://2014.igem.org/Team:EPF_Lausanne">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle active" data-toggle="dropdown">Project </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Overview">Overview</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Envelope_stress_responsive_bacteria">Stress Responsive Bacteria</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Yeast">Osmo Responsive Yeast</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Microfluidics">Microfluidics</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Hardware">Hardware</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Applications" class="active">Applications</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Achievements </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Data">Data</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Judging">Judging</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Parts">Parts</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Policy &amp; Practice </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/HumanPractice">Human Practice</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Safety">Bio Safety</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/PolicyPractice">Metafluidics</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Notebook </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Notebook/Bacteria">Bacteria</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Notebook/Yeast">Yeast</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Notebook/Microfluidics">Microfluidics</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Protocol">Protocols</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Notebook">Timeline</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Team">Meet us!</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Acknowledgments">Acknowledgments</A></LI></DIV></UL></DIV></DIV><DIV class="container"><DIV class="box" id="boxbread"><OL class="breadcrumb breadcrumb-arrow"><LI><A href="https://2014.igem.org/Team:EPF_Lausanne"> Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Project </A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Overview">Overview</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Envelope_stress_responsive_bacteria">Stress Responsive Bacteria</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Yeast">Osmo Responsive Yeast</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Microfluidics">Microfluidics</A></LI><LI><A href="https://2014.igem.org/Team:EPF_Lausanne/Hardware">Hardware</A></LI><LI class="active"><SPAN> Applications</SPAN></LI></OL></DIV><DIV class="row"><DIV class="col col-md-9"><DIV class="whitebg box"><H1 class="cntr"><B> Applications</B></H1><H1 id="basicscience"><U>Basic Sciences Related</U></H1><H2>Protein Complementation techniques &amp; Biosensors</H2><P>Think quick ! That's the message that the EPF Lausanne iGEM team wants to convey. The BioPad project is centered around the use of Protein Complementation Techniques to enable fast in vivo spatiotemporal analysis of biological signals by bacterial biosensors.</P><P>But seriously, what does that mean ? Protein complementation is a technique consisting of the association of reporter protein fragments to components of a same macromolecular complex. Upon reconstitution of the macromolecular structure (active state), the unfolded fused reporter fragments are physically brought together to allow their proper folding. This allows the reconstitution of their chemical properties. 

In research, protein complementation studies are mostly used to validate protein interaction in the context of signal cascades and other pathways. In this context, the most frequently used split reporters are related to fluorescence (GFP, YFP, RFP), bioluminescence (firefly, renilla luciferases), and cAMP production (Adenylyl cyclase). </P><P>The EPF Lausanne iGEM team distinguishes itself from this train of thought, as our team implemented a novel split fluorescent reporter to assess the spatiotemporal dynamics of bacterial biosensors - a novel way of thinking about biosensors &amp; protein complementation to both iGEM and the scientific community. The fluorescent protein used by our iGEM team is the split IFP1.4. The split IFP1.4  (engineered Infrared Fluorescent Protein) is a split fluorescent protein developed early in 2014 by the Michnick Lab<SUP><A href="#ref1">1</A></SUP>. The split IFP1.4 is the first of its kind as it is both fluorescent and reversible (most fluorescent proteins are irreversible). The reversibility is possible as its chromophore - biliverdin - is an organic molecule to which the protein binds. Moreover, the IFP1.4 has advantage of having very low background noise as fluorescence in the far-red spectrum is limited. 
</P><P>
The bacterial biosensor allowing the demonstration of our idea was a stress responsive two component regulatory system: the CpxA-R pathway. Our team successfully showed that spatiotemporal dynamics of the biosensor was possible upon fusion of split IFP1.4 fragment to the relay protein of the pathway, CpxR.
</P><H2 id="microfluidics">Microfluidics as an interface for the in vivo study of Biosensors</H2><P>PUT TEXT HERE</P><H2 id="hogpathway">Enlightening the HOG pathway, but not only</H2><P>

The yeast part of the project will be one of the first to study the HOG pathway directly with split complementation. Our PBS2 and HOG1 tags will allow extensive studying of the Hog pathway in a very direct manner. Furthermore, the MAPKK-MAPK interaction could be quantified in terms of reaction time and sensibility to different stresses and would apply to many other pathways. Using our system, the pathway's activation can be triggered and verified with ease using the split Luciferase strains upon which localization of the proteins can be observed with the GFP tag strains and single cell imaging. Furthermore, the PBS2-HOG1 pathway has a analogous pathway in mammalian cells, with the MEK5 and ERK5 proteins. Fluorescent mice are a thing of the past, we would have touch-sensitive luminescent mice! While that might be a bit extreme as a potential application, expanding the project to mammalian cells using homologous recombination in mouse cells is not as farfetched as it sounds. It would give an indication on how mammalian cells sense different stress, since we could test them separately in an in vitro environment. This would be very useful since it is not yet clear how each pathway is regulated depending on different conditions and our system would provide a basis for mass screening.</P><H2 id="relationship">Relationship between genes and their corresponding activating signals</H2><P>Our project also introduces a new of studying the relationship between genes and their corresponding activating signals. By combining the fusing of split complementary fragments to dimerizing transcription factors, and the introduction of reporter constructs with promoters sensitive to the same transcription factor, one could study the relationship between these two signals. Such an experiment would lead to valuable data about the interconnection between post-transcriptional and transcriptional effects in vivo. Especially in yeast, induction of genes is much less understood than in bacterial cells and much more difficult. Our system would be very useful in controlling the intracellular environment of yeast cells, which would help the advance of synthetic biology in non-bacterial cells.</P><H1><U>Applied Sciences Related</U></H1><H2 id="screening"> Cheap, fast, efficient, and accurate antibiotic screening system</H2><P>The 2014 EPF Lausanne iGEM team engineered the CpxA-R pathway to develop its BioPad. The CpxA-R pathway responds to periplasmic stress via the presence of misfolded/aggregated proteins in the periplasm. Our team hypothesised that the presence of antibiotics would lead to a certain degree of protein misfolding/aggregation and thus would activate the signal. Since our device emits fluorescence upon periplasmic stress, our system could be used to quantify the strength of an antibiotic in a CpxA-R dependent manner. Combined to a microfluidic chip, this system could provide the scientific community with a cheap, fast, efficient, and accurate antibiotic screening system. This could result in easily quantifiable high-throughput screenings for antibiotic candidates.</P><H2 id="antibiotic">Antibiotic Complement</H2><P>Bacterial envelopes are often remodeled when encountering hosts. These changes lead to the synthesis of complex envelope structures that are important virulence factors. Improper assembly of these structures can harm the bacterial envelope and lead to Extracytosolic Stress. Bacteria counter the potential envelope stresses by downregulating these virulence factors. The CpxA-R pathway, used by the 2014 EPF Lausanne, turns out to be envolved in this process. In fact when turned on, the CpxA-R pathway activates a bacterial survival response which among other things, down regulates the biogenesis of complex surface virulence factors such as pili/fimbiae and type III and type IV secretion systems <SUP>2, 3, 4</SUP>.

Given this information, our device could lead to the discovery of an &quot;antibiotic complement&quot; enabling the removal of virulence factors from pathogenic bacteria to improve the efficiency of current antibiotics. Our vision of this novel form of antibiotic would be as a supplement to be taken with current antibiotics to improve the efficiency of treatment.</P><H2 id="tumor">Tumor progression evaluation</H2><P>A possible application of the organisms developped through this project would be related to cancer. In modern research, in vivo tumor progression in experimental animals is fairly difficult to evaluate: most scientists rely on the size of a tumor to get an idea of how developed it is. Our idea would be to integrate our engineered organisms within the tumor's cellular matrix (Matrigel) to allow researchers to be able to assess the progression of tumors by how much fluorescence the tumor emits. This would allow scientists to reduce unnecessary animal sacrifices in tumor research.</P><H2 id="tumorsensing">Tumor sensing</H2><P>Further expanding on the mammalian application for the yeast part of our project, we could potentially create stress-sensing mice. As mentioned above, detection of a tumor is not the simplest of tasks. We would apply our system to whole mice, meaning that all their cells would be stress-sensors. At the injection of coelenterazine, the substrate for the Renilla Luciferase, the most stressed cells would luminesce meaning that we would easily detect the most stressed regions of the animal in question without hurting the animal or affecting its metabolism.</P><H4 id="references"> References </H4><P>1: Michnick, S., Tchekanda, E., &amp; Sivanesan, D. (2014, April 20). An infrared reporter to detect spatiotemporal dynamics of protein-protein interactions. <I>Nature Methods</I>, 6-6.
</P></DIV></DIV><DIV class="col col-md-3"><NAV id="affix-nav" class="sidebar hidden-sm hidden-xs"><UL class="nav sidenav box" data-spy="affix" data-offset-top="200" data-offset-bottom="600"><LI class="active"><A href="#basicscience">Basic Sciences Related</A></LI><LI><A href="#microfluidics">Microfluidics as an interface</A></LI><LI><A href="#hogpathway">Enlightening the HOG pathway</A></LI><LI><A href="#thehogpathway">The HOG Pathway</A></LI><LI><A href="#relationship">Relationship between genes and their corresponding activating signals</A></LI><LI><A href="#screening">Antibiotic screening system</A></LI><LI><A href="#antibiotic">Antibiotic Complement</A></LI><LI><A href="#tumor">Tumor progression evaluation</A></LI><LI><A href="#tumorsensing">Tumor sensing</A></LI><LI><A href="#references">References</A></LI></UL></NAV></DIV></DIV></DIV><DIV id="contactSection"><DIV><H1 class="cntr">Sponsors</H1></DIV><DIV class="footerSection container"><DIV class="copyright"><P>Copyright Â© iGEM EPFL 2014. All Rights Reserved</P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:EPF_Lausanne/Applications.ctp">http://2014.igem.org/Team:EPF_Lausanne/Applications.ctp</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:EPF_Lausanne/Applications.ctp" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:EPF_Lausanne/Applications.ctp" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:EPF_Lausanne/Applications.ctp&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:EPF_Lausanne/Applications.ctp&amp;oldid=277182" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>