<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Hamburg_Biochemistry skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Hamburg/Biochemistry</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar"><DIV class="navbar_box"><DIV class="dropdown"><A class="menu_link_lone" href="https://2017.igem.org/Team:Hamburg#Section_Anchor" style=" color:#f8f8f8; text-decoration:none;">Home</A></DIV><DIV class="dropdown"><BUTTON class="menu_link">Team</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Hamburg/Team">Members</A><A href="https://2017.igem.org/Team:Hamburg/Collaborations">Collaborations</A><A href="https://2017.igem.org/Team:Hamburg/Attributions">Attributions</A></DIV></DIV><DIV class="dropdown"><BUTTON class="menu_link">Project</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Hamburg/Description">Description</A><A href="https://2017.igem.org/Team:Hamburg/Biochemistry">Biochemistry</A><A href="https://2017.igem.org/Team:Hamburg/Chemistry">Chemistry</A><A href="https://2017.igem.org/Team:Hamburg/Nanoscience">Nanoscience</A><A href="https://2017.igem.org/Team:Hamburg/Applied_Design">Applied Design</A><A href="https://2017.igem.org/Team:Hamburg/Parts">Parts</A></DIV></DIV><DIV class="dropdown"><BUTTON class="menu_link">Notebook</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Hamburg/Lab_Book">Lab Book</A><A href="https://2017.igem.org/Team:Hamburg/Protocols">Protocols</A></DIV></DIV><DIV class="dropdown"><BUTTON class="menu_link">Results</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Hamburg/Lab_Results">Lab Results</A><A href="https://2017.igem.org/Team:Hamburg/Demonstrate">Demonstrate</A><A href="https://2017.igem.org/Team:Hamburg/Improve">Improve a previous part or project</A><A href="https://2017.igem.org/Team:Hamburg/InterLab">InterLab Study</A></DIV></DIV><DIV class="dropdown"><BUTTON class="menu_link">Human Practices</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Hamburg/HP/Silver">Human Practices</A><A href="https://2017.igem.org/Team:Hamburg/HP/Gold_Integrated">Integrated HP</A><A href="https://2017.igem.org/Team:Hamburg/Business_Plan">Entrepreneurship</A></DIV></DIV><DIV class="dropdown"><BUTTON class="menu_link">Safety</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Hamburg/Safe_Design">Safe Design</A></DIV></DIV><DIV class="dropdown"><BUTTON class="menu_link">Fun</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Hamburg/The_stupid_Mistakes">The Pinky and the Brain</A><A href="https://2017.igem.org/Team:Hamburg/The_funniest_Things">The funniest Things</A><A href="https://2017.igem.org/Team:Hamburg/Records">Records</A><A href="https://2017.igem.org/Team:Hamburg/Other_languages">Other languages</A></DIV></DIV></DIV></DIV><DIV class="section_std"><DIV class="main_block"><H1 class="h1" style="margin-top:100px;">
    Biochemistry
    </H1></DIV></DIV><DIV class="section_std"><DIV class="main_block"><H2 class="h2" style="margin-bottom:70px;">General Idea </H2><H2><CENTER><DIV class="text_block text"><P>
To tackle the upcoming problem posed by multiresistant bacteria like <I>P. aeruginosa and K. pneumoniae</I>, we wanted to invent a therapeutic method to inhibit the conserved iron metabolism of multiresistant germs. Therefore we used a „Trojan Horse“approach, which means we used Pyochelin, a gallium-loaded, endogenous siderophore, as a vector delivery system for the drug. Through a chemical exchange reaction, we wanted to remove the iron from the siderophores and add gallium ions instead to form a similar complex, that is then taken up by the targeted bacteria. The high specificity of siderophores will prevent that the natural host flora is being hit by the cytotoxic effects of gallium.
</P><P> 
To create an efficient method, we combined all disciplines that our team members brought to the table.  Our Life Sciences and Biology Division manipulated bacteria to create two types of siderophores from the salicylate pathway, pyochelin and yersiniabactin, which are prevalently synthesized by pathogens. In the next step we performed toxicity measurements to analyze the effectivity of the described method. The Nanoscience Division worked on a microfluidic “lung-on-a-chip” device, that can carry out long term toxicity tests with human lung epithelial and alveolar epithelial cells. Our Chemistry Division synthesized both siderophores with a pure chemical synthesis and helped to analyze and characterize our products.  
</P><P>
On this page you will foundations on which our biochemistry experiments are footed, as well as a short description of the coducted experiments and the design process. 
</P></DIV></CENTER></H2><H2 class="h2">
    Foundation
    </H2><H3 class="h3"> Infections and building of resistences </H3><DIV class="text_block text"><P>
Every day the human body has to repel a multitude of attacks from its environ-ment. Microorganisms like bacteria or fungi are able to exploit the resources of the human body, which delivers nutrients and an optimal environment for growth and reproduction. Due to the infections the body can suffer from the loss of nu-trients, damage from exploited cells which are not able to fulfil their function and besides, metabolism products from the microorganisms which harm the system. [1,2]
</P><P>
To prevent the system to be exploited by other organisms the human body has evolved a lot of defending mechanisms against these intruders over millions of years of evolution. These defending mechanisms include different sizes from the 2 m2 mechanical barrier of the skin and the small immune cells like natural killer cells and T cells which comprise a size of only a few micrometres. All of these parts work together as a whole and prevent microorganisms to get into the body or are able to target the organisms when they have overcome the barrier of the body. But despite the complex system of fighting microbiotic attacks some organ-isms are able to overcome all defending mechanisms and are able to infect. Due to this mankind has suffered from diseases since the beginning of history and was also eager to fight these infections with the help of exterior sources. A major breakthrough was made by Alexander Flemming, who discovered the antibiotic effect of the fungi <I>Penicillium notatum</I> on the infectious bacteria family staphylo-coccusa. The molecule responsible for this effect was one of the first discovered antibiotics which are able to target microorganism and was called after the pro-ducing strain: Penicillin. Penicillin is able to inhibit the cell wall synthesis, which leads to instable cells and therefore to their destruction. Due to the fact, that human cells do not have a cell wall, this cells are not targeted. [1, 2, 3]
</P><P>
After the discovery of Penicillin, a lot more antibiotics were discovered. Besides the inhibition of the cell wall synthesis a lot of molecules with other targets has been found. The most important targets are besides the cell wall synthesis (e.g. Penicillin), the protein biosynthesis (e.g. Tetracyclines), DNA and RNA production (e.g. Ciprofloxacin) and bacterial metabolism (e.g. Sulfomethoxacole). Due to this a lot of sicknesses from which mankind has suffered for many years can be cured. For example, Streptomycin is used to cure the plague caused by <I>Yersinia pestis</I>, if discovered in an early stage in most cases. To compare, in the Middle Ages one third of the European population has died during outbreaks of the plaque. [2, 3, 4]
</P><P>
But as the body has developed over a long period of evolution, microorganisms were also able to develop resistance against some antibiotics. This resistance is achieved by impaired influx, efflux, target mutation, target modification, over-production of target mimic, factor-associated protection, drug modification or drug degradation. [2]
</P><P>
One of the most investigated antibiotic and its resistance are the Tetracyclines. Tetracyclines bind to the A-site in the bacterial ribosome. For example, the Tet1 binding site is at the helix 34 and party helix 31 of the ribosomal A site. The tRNA tries to decode the mRNA but steric clashes between the tRNA and Tet1 compete with this event. Therefore, the mRNA cannot bind and cannot be translated into proteins, which are essential for the survival of the microorganism. Tetracycline resistance results for example from mutations in the helix 34 and helix 31 in the 16S rRNA. Therefore, Tetracycline cannot bind, meanwhile the mRNA is still able to and therefore the protein biosynthesis is restored. [2]
</P><P>
This kind of resistance impede the elimination of these microorganisms. Still, if one strain shows one resistance there is a possible chance to use another antibi-otic to eliminate the infection. But due to overuse and wrong usage of antibiotics more and more microorganisms show an increase of resistances. These organ-isms are called multi resistant. One example for this multi resistant organism is <I>Pseudomonas aeruginosa</I>. This bacterium is a hospital pathogen which shows multiple resistance against a lot of antibiotics. In Germany this pathogen is responsible for 10 % of all hospital infections. [2, 5] 
</P><P>
The possibility to use other antibiotics is reduced by every new resistance. There are already infections, which cannot be treated with the common antibiotics. There are still backup stocks from antibiotics, which are only used, when no common antibiotics show an effect. But there are already germs which show a resistance against those. [1, 2, 5]
</P><P>
To keep the upper hand against multiresistant germs it is necessary to invent new ways to fight these germs. This is the aim on which this project is based on.  
</P></DIV><H3 class="h3"> 
    Siderophores
    </H3><DIV class="text_block text"><P>    
Siderophores are a class of small peptides originating in bacteria. In the next couple of years siderophores will reenter the spotlight of medical research again after a short period in the 90’s where they were a molecule of interest to find a solution for multiresistant bacteria.
</P><P>
However, till today most siderophores are still poorly researched and more and more basic functions of these very versatile peptides become revealed as more and more work groups around the world jump onto the siderophore train.
The main aspect why we and most other groups are interested in siderophores is their key ability: Metal-ion-complexation.
</P><P>
Bacteria mostly utilize siderophores to scavenge iron3+-ions from their surroundings, an important trace element direly needed for proliferation. Many studies show that iron is not the only 3+ metal-ion that siderophores are able to form chelate complexes with, aluminum, vanadium, copper and gallium are known to form complexes with siderophores as well.
</P><P>
For our experiments gallium3+ is the ion of interest studies suggesting the cytotoxic abilities of gallium and the biochemical similarities, with the key difference that gallium inhibits enzymes irreversibly, while iron activates them.
Other known properties of siderophores are: antimicrobial effects against bacteria that utilize other iron acquisition methods (or other siderophore types), the inactivation of specific toxins, biofilm formation, and of course to outcompete other cells in iron deprived environments.
</P></DIV><H3 class="h3">
    Yersiniabactin
    </H3><DIV class="text_block text"><P>    
Now one might think, with Pseudomonas as the bacterial target, what role does Yersiniabactin, a siderophore once originating in Yersinia pestis, which later became a common siderophore in many types of enterobacteria, play in the project design. Well there are two good reasons.
</P><P>
First Pseudomonas mostly causes pneumonia in hospital infections due to its resilience and ability to proliferate in sparse environments, which he has in common with some enterobacteria. Enterobacteria like Klebsiella pneumonia, and Escherichia coli often cause lung infections leading to pneumonia when they carry the genetic sequences for Yersiniabactin synthesis, and tend to be multiresistant as well.
Yersiniabactin is an important factor in pneumonia caused by enterobacteria since it has a high efficiency at acquiring iron, and possesses the ability to cleanse ions from human storage proteins like transferrin and lactoferrin. Another important factor for the prevalence of Yersiniabactin in lung infection is the fact, that yersiniabactin, as many siderophores, is bound by Lipocalin 2 in human serum, which significantly reduces the efficiency.
</P><P>
Second, and of utmost importance, was the fact that we had problems acquiring the genes from pseudomonas since a synthesis is extremely difficult due to the high GC-content. During our research we stumbled upon an image showing several pathways of siderophore synthesis, including similarities and the involved enzymes for the non-ribosomal synthesis. A short inquiry made sure that our initial thought was correct. Yersiniabactin is synthesized by two different enzymes, HMWP1 and HMWP2 synthases, of which HMWP2 became the synthase of interest for all our following research. Hence, HMWP2 synthesizes a compound that is known as Nor-Pyochelin. Therefore, we figured that with different induction methods we could create bacterial strains that could be induced to either produce yersiniabactin (utilizing the full casette), or Nor-Pyochelin with HMWP1 turned off. Essentially the gene cassette for Yersiniabactin allows us to synthesize a siderophore from a completely different organism but its own as well. Which leads to the elegant design of our project, not only are we able to target multiple different bacteria specifically by introducing toxic ions with their specific siderophores, but we are able to produce two different siderophores targeting pneumonia mediating multiresistant bacteria strains. 
</P></DIV><H3 class="h3">
    Pyochelin
    </H3><DIV class="text_block text"><P> 

At the beginning of our project we decided that we wanted to do something about multiresistant bacteria that cause lung infections. Therefore, pyochelin, one of two native siderophores of Pseudomonas aeruginosa, became one of our prime targets for bio- and chemical synthesis, as P. aeruginosa is known for its manifold antibiotic resistances and oppurtunistic lung infections. Therefore P. aeruginosa poses a big problem in hospitals as one of the main reasons for opportunistic infections in weakened or immunosuppressed patients. Being extremely resilient and undemanding, pseudomonads are known to survive in hostile environments, like the tubes of breathing devices and even disinfectants, it is hard to prevent pseudomonade infections. Since Pyochelin is relatively small and easy to synthesize, and pseudomonas is one of the most resilient bacteria known, we chose pyochelin as our main target for our trojan horse strategy.
</P><P>
However, fighting pseudomonas with a gallium complex with its native siderophore wasn’t enough. Additionally, during the biobricks design for the pyochelin synthesis we found out that pseudomonas gene sequences consist of a very high GC-content which proved to be very difficult for synthesis and other molecular manipulation. Therefore, we had to design our project towards a similar but different approach.  The new target became Nor-Pyochelin.
</P></DIV><H3 class="h3"> Nor-Pyochelin </H3><DIV class="text_block text"><P>    
Nor-Pyochelin is a chemical derivate of pyochelin, which lacks a single methyl group on its second cyclized cysteine residue. Based on the work of Abdallah et. al., which included a detailed analysis of pyochelin and a couple of similar compounds. We figured that Nor-Pyochelin would outcompete native pyochelin as a potential antibiotic carrier molecule since Abdallah et. Al. measured a 30% higher receptor affinity and iron transport capability for Nor-Pyochelin. An effect that we wanted to replicate in our antibiotic efficiency study.
      </P><H2 class="h2">Experimental Design</H2><H3 class="h3">Biobrick Design</H3><H3 class="h3">Siderophore Biosynthesis</H3><P>For the synthesis of pyochelin and yersiniabactin we designed two different Biobricks with a total legnth of about 20 kbp.
</P><H3 class="h3">Siderophore Transportsystems</H3><H3 class="h3">Extraction and Mutation</H3><DIV class="text_block text"><P>
As our target organism <I>P. aeruginosa</I> tend to create GC-heavy sequences [1], a synthesis of the needed sequences for our target structure pyochelin was a challenge.
</P><P> 
We first thought about accessing the important genes for pyochelin synthesis via PCR-extraction directly from <I>P. aeruginosa</I>. But having to remove several iGEM restriction sites prior to the creation of viable biobricks would have made this method very complicated. We would have needed a follow-up mutagenesis with about 56 different primers and sequencing to verify the outcome. As the gene loci pchE+F+G is about 11 kb long the PCR itself would also be a difficult tool. 
</P></DIV><H3 class="h3">Amplifikation</H3><DIV class="text_block text"><P>
Since extraction and mutation seemed unlikely to work, we decided to go the standard route and order all our genes via IDT. Sadly our contingent only included 20 Kbp, while our genes of interest totalled at 30 Kbp. After the arrival of all our parts we needed to amplify all genes with our standard primers to increase the DNA amount for further experiments. We utilized 4 different polymerases: Taq/DreamTaq, Pfu, Pfu Turbo, phusion and Q5. We tried different annealing temperatures, touchdown PCR, gradient PCR, and several GC enhancing substances like DMSO.
</P><H3 class="h3">Assembly</H3>
Innitially we planned to utilize variants of the 3A-Assembly to ligate our plasmids. Addionally we had to utilize Golden-Gate-Assembly.

<H3 class="h3">Toxicology</H3>
For toxicological experiments we planned standard MIC-tests, for bacteria and MTT-tests, for human celllines.  
<H2 class="h2">Sources</H2><UL><LI>[1] Janeway C.A., Travers P. (2009): “Immunologie 5. Auflage”, Spektrum Akademischer Verlag</LI><LI>[2] Fischbach M.A., Walsh C.T., (2009): “ Antibiotics for Emerging Patho-gens” Science Vol. 325, Issue 5944, pp. 1089-1093</LI><LI>[3] Fille M., Hausdorfer J., Dierich M.P. (2016) “β-Laktam-Antibiotika I: Penicilline.“ In: Suerbaum S., Burchard GD., Kaufmann S., Schulz T. “Me-dizinische Mikrobiologie und Infektiologie“ Springer Verlag</LI><LI>[4] Yang R. (2017): “Plague: Recognition, Treatment and Prevention” Journal of Microbiology doi:10.1128/JCM.01519-17</LI><LI>[5] http://pseudomonasaeruginosa.info/</LI></UL></DIV></DIV></DIV><DIV class="footer"><SPONSORED_BY class="h6">Sponsored by
  </SPONSORED_BY></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>