<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Hamburg_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Hamburg/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar"><DIV class="navbar_box"><DIV class="dropdown"><A class="menu_link_lone" href="https://2017.igem.org/Team:Hamburg#Section_Anchor" style=" color:#f8f8f8; text-decoration:none;">Home</A></DIV><DIV class="dropdown"><BUTTON class="menu_link">Team</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Hamburg/Team">Members</A><A href="https://2017.igem.org/Team:Hamburg/Collaborations">Collaborations</A><A href="https://2017.igem.org/Team:Hamburg/Attributions">Attributions</A></DIV></DIV><DIV class="dropdown"><BUTTON class="menu_link">Project</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Hamburg/Description">Description</A><A href="https://2017.igem.org/Team:Hamburg/Biochemistry">Biochemistry</A><A href="https://2017.igem.org/Team:Hamburg/Chemistry">Chemistry</A><A href="https://2017.igem.org/Team:Hamburg/Nanoscience">Nanoscience</A><A href="https://2017.igem.org/Team:Hamburg/Applied_Design">Applied Design</A><A href="https://2017.igem.org/Team:Hamburg/Parts">Parts</A></DIV></DIV><DIV class="dropdown"><BUTTON class="menu_link">Notebook</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Hamburg/Lab_Book">Lab Book</A><A href="https://2017.igem.org/Team:Hamburg/Protocols">Protocols</A></DIV></DIV><DIV class="dropdown"><BUTTON class="menu_link">Results</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Hamburg/Lab_Results">Lab Results</A><A href="https://2017.igem.org/Team:Hamburg/Demonstrate">Demonstrate</A><A href="https://2017.igem.org/Team:Hamburg/Improve">Improve a previous part or project</A><A href="https://2017.igem.org/Team:Hamburg/InterLab">InterLab Study</A></DIV></DIV><DIV class="dropdown"><BUTTON class="menu_link">Human Practices</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Hamburg/HP/Silver">Human Practices</A><A href="https://2017.igem.org/Team:Hamburg/HP/Gold_Integrated">Integrated HP</A><A href="https://2017.igem.org/Team:Hamburg/Business_Plan">Entrepreneurship</A></DIV></DIV><DIV class="dropdown"><BUTTON class="menu_link">Safety</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Hamburg/Safe_Design">Safe Design</A></DIV></DIV><DIV class="dropdown"><BUTTON class="menu_link">Fun</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Hamburg/The_stupid_Mistakes">The Pinky and the Brain</A><A href="https://2017.igem.org/Team:Hamburg/The_funniest_Things">The funniest Things</A><A href="https://2017.igem.org/Team:Hamburg/Records">Records</A><A href="https://2017.igem.org/Team:Hamburg/Other_languages">Other languages</A></DIV></DIV></DIV></DIV><DIV class="section_std"><DIV class="main_block"><H1 class="h1" style="margin-top:100px;">
    Resistant Germs: And how to fight them!
    </H1><H3 class="h3">
    The world on the brink to a new age of infections
    </H3><DIV class="text_block text"><P>
      In the last hundreds of years mankind was ravaged by various mysterious threats. Infectious diseases, back then unknown, struck fear into the hearts of tens of millions of people worldwide while leaving more millions dead. One of the most prominent examples has been long forgotten in the western hemisphere, the pneumonic plague, an airborne infection caused by a bacterial strain called <I>Yersinia pestis</I>. Once one of the most prolific killers among men the plague got pushed back massively by new hygiene standards and antibiotic therapies.<SUP>[1]</SUP></P><P>
      A more recent example still known to elderly westerners is tuberculosis, an even more threatening disease  present in most countries around the world, extremely virulent, lethal and hard to cure. Multiresistant <I>Mycobacterium tuberculosis</I> might soon reclaim the top spot among life threatening conditions.  In some regions multiresistant tuberculosis already has a share of 20-50 % among diagnosed tuberculosis infections, adding up to 480 thousand cases in total.<SUP>[2]</SUP></P></DIV><H3 class="h3">
    Current antibiotic usage breeding multiresitant bacteria
    </H3><DIV class="text_block text"><P>
      Now one might think: well, tuberculosis and plague are only two strains among a vast amount of different bacterial species and mostly present in the eastern and southern hemisphere, I could care less.
      </P><P>
      For those we have a dire warning, sure it is true the most proficient killers among diseases are still somewhat out of reach for the populations in Europe and the US but as we all know the world is growing closer together and the agglomeration proceeds undamped.
      </P><P>
      But we face another subtler danger. Worldwide the usage of antibiotics becomes less and less responsible: from patients who don’t take their medication properly to cattle farmer feeding vast amounts of antibiotics to their livestock and to contaminations due to unsafe production plants, the overall contamination with antibiotics of our planet proceeds quickly. And that becomes more and more dangerous for a very simple reason: evolution.
      </P></DIV><H3 class="h3">
    Antibiotic development stagnant
    </H3><DIV class="text_block text"><P>
        Resistances alone don’t pose a problem, shortly after the development of our first commercial antibiotics widespread resistances could be observed. As history shows we were always able to develop novel antibiotics or develop modifications for known antibiotics to cope with the evolution of bacteria. For the last thirty years antibiotic development has been stagnant, not a single novel antibiotic was developed, many promising compounds were discovered and tested but all of them failed leading to an ongoing antibiotic discovery void. <SUP>[3]</SUP></P><P>
        That trend puts modern methods of antibiotic development into question. Bioinformatics supported compound screenings could generate huge amounts of potential agents and targets of which most did look promising in “in vitro” studies and initial testing’s. However, all of them failed in toxicological or “in vivo” experiments. Posing the question if understanding mechanisms and calculating potential binding abilities is the way to go. Some researchers already turned their backs to modern methods and returned to the testing of natural compounds originating in widely uncharted organisms like tropical plants and sea organisms. Another approach is utilizing bacterial abilities and compounds to either trick bacteria to deliberately take up toxins/antibiotics or to utilize the abilities of predatory bacteria like Myxobacteria possess to exterminate their natural prey.  Myxobacteria are a bacterial species of which our understanding is still very poor. What we know is that they hunt in packs and exterminate other bacterial species with a vast arsenal of antibiotic compounds which they express based on their prey.<SUP>[4]</SUP></P></DIV><H3 class="h3">
				Harmless bacteria evolve into relentless threats
			</H3><DIV class="text_block text"><P>
					The presence of antibiotics in almost every aspect of life leads to the adaptation of common bacteria to our most efficient weapon in the war against infectious diseases. Common bacteria becoming more and more resistant poses a twofold threat:
				</P><OL><LI>Bacteria are capable of horizontal gene transfer, a mechanism that speeds up evolution among a population by transferring useful genes, like antibiotic resistances, from bacteria to bacteria.</LI><LI>Common bacteria, like Enterobacteria and Pseudomonades, can cause facultative infections. And any infection can potentially lead to a lethal outcome through sepsis and biofilm formation.</LI></OL><P>
					Thus, Hospital acquired infections are on the rise and become less and less treatable.
				</P></DIV><H3 class="h3">
				Standard surgeries might become life threatening again
			</H3><DIV class="text_block text"><P>
					The most common bacterial threats in hospitals are Enterobacteria like <I>E. coli</I> and <I>K. pneumonia</I>, <I>Pseudomonas aeruginosa</I> and methicillin resistant <I>Staphylococcus aureus</I>. Those strains developed many different resistance mechanisms and the ability to develop potent biofilms which leads to chronic infections that are almost impossible to cure.<SUP>[5]</SUP></P><P>
					Therefore, the Royal Medical Society recently shed some light on current prophylaxes which become less and less effective due to the rising inefficiency of antibiotics. A global trend that leads to the point where even the most standard surgeries, like cesarean section, become potentially lethal again.
				</P></DIV><H3 class="h3">
				The Lung, a prime target for opportunistic infections
			</H3><DIV class="text_block text"><P>
					The lung is a common target for various infections and diseases, which are commonly overlooked or considered harmless. That has multiple reasons: As everybody knows,the flu and catching a cold aren’t major threats and belong to the most common diseases that affect humans. The other point is, that the lung itself lacks nervous tissue and therefore infections, tumors and other alien compounds are easily detected by the patients themselves, even though it is often too late because that implies that neighboring tissue is affected as well. 
				</P><P>
					Many facultative causative agents like E. coli, K. pneumoniae and P. aeruginosa reach the lung by bacteria commonly living in the gut microbiome. Pseudomonas is one of the most common causes for hospital acquired pneumonia since it can contaminate breathing devices, survive in disinfectants and other unlikely places. All those strains can form biofilms which lead to chronic infections that potentially have a lethal outcome and are extremely hard to cure. <SUP>[6], [7]</SUP></P></DIV><H3 class="h3">A Trojan Horse, high jacking bacterial virulence strategies</H3><DIV class="text_block text"><P>
Since developing novel antibiotic compounds is a very difficult task that requires vast amounts of capital, we decided to tackle multiresistant bacteria in a more elegant way by utilizing endogenous peptides for a new angle of attack,  utilizing the conserved iron pathway that most bacteria share, that plays a huge role in virulence as iron acquisition is a crucial factor for the development of stable cultures in a host organism and thus infect it. Due to coevolution of hosts and bacteria, immune systems have adapted deprivation of harmful bacteria of any iron sources to effectively contain infections.
</P><P>
Therefore, harmful bacteria developed multiple pathways to acquire iron in the human body. Two of those are solely parasitic, capturing human proteins like transferrin or hemophores to internalize them and harvest the iron3+-ions to finally degrade the proteins. 
</P><P>
The third mechanism originates from bacteria. Many virulent species carry genes for siderophore synthesis. Siderophores are relatively small peptides with a high affinity for iron3+-ions that scavenge iron from the bacterial surroundings. This mechanism is crucial for bacterial proliferation under infectious conditions, since the human immune system includes many mechanisms to deprive harmful bacteria of the needed trace element. Some siderophores include mechanisms or a sufficient affinity to extract iron3+-ions from human proteins. The utmost important factor for our project, however, is the ability of siderophores to form chelate complexes with other, sometimes cytotoxic ions.
</P></DIV><H3 class="h3">A new antibiotic approach: Gallium loaded Siderophores</H3><DIV class="text_block text"><P>
At the beginning of our iGEM year we split up into 4 groups to brainstorme for  something to research. After our presentations we decided to research novel antibiotic compounds: Gallium-siderophore complexes. However, the first ideas to develop a method in this direction turned out to be either impossible or non iGEM compatible.  
</P><P>
But we could find potential targets in carbapenem resistant Enterobacteria and <I>Pseudomonas aeruginosa</I>,which in turn narrowed the selection of potential siderophores to research. Hence, <I>Pseudomonas aeruginosa</I> is the most prevalent multiresistant strain in our hometown Hamburg, and sometimes untreatable, it became our prime target.  
</P></DIV><H3 class="h3">Gallium mediated cytotoxicity </H3><H3><DIV class="text_block text"><P>
For <I>Pseudomonas aeruginosa</I> many experiments have been done with gallium in various forms. Recent findings show that gallium affects pseudomonades and familiar strains in multiple ways. Notably, gallium ions disrupt biofilm formation which is an essential part of chronic infections and increases resistances against normal antibiotic compounds and immune responses from the host.
</P><P>
Most of the toxic effects of gallium are mediated by its similarity to iron, especially Ga3+ and Fe3+ are very similar. Speaking of ions, one of the most important factors why gallium inhibits bacteria originates in its inability to form Ga2+ ions which in turn means that gallium is useless in a redox cycle. Therefore Ga3+-Ions can disrupt the electron transport chain. Another important factor is that iron ions are an important co-factor for many metallo-proteins. However, gallium-ions can effectively replace it chemically, which is linked to a loss of function. <SUP>[8], [9]</SUP></P></DIV></H3><H3 class="h3">Pneumonia as our prime target</H3><H3><DIV class="text_block text"><P><I>P. aeruginosa</I>, <I>K. pneumonia</I> and <I>E. coli</I>, three potential multiresistant strains that have one thing in common: they can cause pneumonia. Therefore, we included a testing system into our experiments to monitor the effect of our potential treatment as closely as we can in relation to the lung. Most of us think that animal testing is a necessary evil which should be abandoned if possible. Depending on the research done in animals the results are potentially unadaptable. Thanks to the Harvard Wyss institute and other inspired research groups there might be the chance to find a better solution.
</P><P>
Microfluidic chips that are utilized to simulate organs, organ systems or even whole organisms are on the horizon. Inspired by the neat design of Wyss institutes lung-on-a-chip, we could produce a similar device to test our method in an environment that is at least like a human lung. Including human cell lines, including alveolar and endothelial cells and the mimicry of the breathing movement. Depending on the results to come we might have been able to simulate an infection in a chip and cured it with our newly derived method.  
</P></DIV></H3><H3 class="h3">Salicylate Siderophore Cluster, Pch, Ybt Mbt</H3><DIV class="text_block text"><P>
We utilized two different siderophores and multiple derivatives during our biosynthesis and the toxicological experiments. They all have a single thing in common, the biosynthesis starts with salicylate and mostly proceeds with cyclized cysteine. Therefore they are chemically similar. We coined the selection of salicylate based siderophores: salicylate cluster. In this defined space we could find another siderophore, mycobactin, which belongs to <I>Mycobacterium tuberculosis</I>, the biggest player among multiresistant bacteria. <SUP>[10]</SUP></P><P> 
Tuberculosis (TBC) claims 1.4 Mio lives every year, including about 500 000 people dying to multiresistant TBC, the therapy becomes less and less effective and involves multiple therapeutics and the treatment often takes months, months were the affected essentially must be under quarantine.
</P></DIV><H3 class="h3"> A design concept easily applied to new strains </H3><DIV class="text_block text"><P>
Another goal during our design process was to derive a method to test any siderophores-gallium compound in conjunction with their corresponding bacteria. Therefore, we developed standard testing strains to simulate the uptake by including all the transport systems but no synthesis pathways to measure specific uptake rates. Additionally, we introduced lung-on-a-chip devices to the University of Hamburg, providing novel test methods for lung environments to prolong the usage of animal models and potentially generate hints if human cells are damaged by the tested compounds and to find out if anything related to the novel therapeutics passes into the bloodstream.
</P><P>
By combination of both methods scientists should be able to easily replicate our findings and to effectively apply different siderophore-gallium therapies for lung infections.
</P></DIV><H3 class="h3">How does our interdisciplinarity affect our design? </H3><DIV class="text_block text"><P>
This year’s iGEM team Hamburg was very diverse including students from many unexpected faculties. We had a psychologist monitoring our work, helping with our communicational issues and structures, as well as counseling people who faced troublesome issues in the project or personal. We also had a graphic department, which had the task to create a corporate design and an overall marketing concept.  Of course, we had a diverse field of natural scientists, too  who covered the actual synthetic biology part.
</P><P>
Scientific, we had three half-autonomous groups which each had their specific tasks and goals to fulfill, but also cooperated on anything. Our chemists had the assignment to synthesize our target siderophores ahead of time to test our hypothesis if the compounds work and how they function. Therefore, we had an organic synthesis of each pyochelin, nor-pyochelin and desferriothicin, we monitored the complexation of gallium with each siderophore, a measurement that hasn’t been done yet, and utilized the chemically synthesized siderophore complexes to test the toxicity while also figuring out MIC-values.
</P><P>
The synthetic biology department had to develop and produce the corresponding biobricks and the biological testing system as well as preparing bacterial and human cell cultures. As the core section of our project, most lab and design hours were spent in the development here. For the core project 4 composite parts consisting of 12 basic parts were designed to express 2 different types of siderophores. While our other composite parts were designed to express transport systems to create <I>E. coli testing</I> strains for <I>Klebsiella pneumonia</I> and <I>Pseudomonas aeruginosa</I> siderophore-gallium complexes.
</P><P>
For our nanotechnology department we reserved the last and most fancy part: Toxicological experiments to mimic a real therapy in a lung-on-a-chip microfluidic device. Therefore, a microfluidic device had to be designed and casted that allows a coculture of alveolar and endothelial cells that we can infect with pseudomonades. Luckily, we could rely on former results by the Harvard Wyss institute, who developed a sophisticated device that suits our needs. Eventually we could create a slightly less complicated microfluidic chip to mimic the testing device invented by Harvard. Since the coculturing and measurement proved to be very difficult we could not measure results before the Wiki-freeze.
</P></DIV><H3 class="h3">What’s next? Mycobactin derivatives targeting multiresistant tuberculosis.</H3><DIV class="text_block text"><P>
As described earlier, we derived a specific set of methods that allows the development of any siderophore-gallium therapies, specifically those that are based on salicylate. Tuberculosis the biggest player among the multiresistant bacteria, could be the next target. Replicating the utilized effects to test mycobactin derivatives complexed with gallium ions might prove to be  useful against the rise of MR-TBC.
</P><H3 class="h3">Sources:</H3><UL><LI>[1]Black-Death @ www.britannica.com. (n.d.). Retrieved from https://www.britannica.com/event/Black-Death </LI><LI>[2]1ee7d3875a39ca8442d2b48590ea40f08ea2beda @ www.who.int. (n.d.). Retrieved from http://www.who.int/tb/en/ </LI><LI>[3]Report, G., &amp; Yewale, V. N. (2014). Antimicrobial resistance. Bulletin of the World Health Organization, 61(3), 171–172. https://doi.org/10.1007/s13312-014-0374-3</LI><LI>[4]Guest lecture by Prof. Dr. Rolf Müller</LI><LI>[5]Brisabois, A., Cazin, I., Breuil, J., &amp; Collatz, E. (1997). Surveillance of antibiotic resistance in Salmonella. Eurosurveillance, 2(3), 19–20. https://doi.org/10.1098/rstb.2014.0080</LI><LI>[6]Martínez‐Solano, L., Macia, M. D., Fajardo, A., Oliver, A., &amp; Martinez, J. L. (2008). Chronic Pseudomonas aeruginosa Infection in Chronic Obstructive Pulmonary Disease. Clinical Infectious Diseases, 47(12), 1526–1533. https://doi.org/10.1086/593186</LI><LI>[7]klebsiella @ www.cdc.gov. (n.d.). Retrieved from https://www.cdc.gov/hai/organisms/klebsiella/klebsiella.html</LI><LI>[8]Ross-Gillespie, A., Weigert, M., Brown, S. P., &amp; Kümmerli, R. (2014). Gallium-mediated siderophore quenching as an evolutionarily robust antibacterial treatment. Evolution, Medicine, and Public Health, 2014(1), 18–29. https://doi.org/10.1093/emph/eou003</LI><LI>[9]Ahmed, M., Brode, E., Brown, T., Eltoweissy, S., Gross, S., Markowitz, S., … Woodard, B. (n.d.). Effects of Gallium-Desferrioxamine Compounds on Bacteria.</LI><LI>[10]Show_Pathway @ Www.Genome.Jp. (n.d.). Retrieved from http://www.genome.jp/kegg-bin/show_pathway?sce01100</LI></UL></DIV></DIV></DIV><DIV class="footer"><SPONSORED_BY class="h6">Sponsored by
  </SPONSORED_BY></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>