<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Hamburg_Improve skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Hamburg/Improve</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar"><DIV class="navbar_box"><DIV class="dropdown"><A class="menu_link_lone" href="https://2017.igem.org/Team:Hamburg#Section_Anchor" style=" color:#f8f8f8; text-decoration:none;">Home</A></DIV><DIV class="dropdown"><BUTTON class="menu_link">Team</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Hamburg/Team">Members</A><A href="https://2017.igem.org/Team:Hamburg/Collaborations">Collaborations</A><A href="https://2017.igem.org/Team:Hamburg/Attributions">Attributions</A></DIV></DIV><DIV class="dropdown"><BUTTON class="menu_link">Project</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Hamburg/Description">Description</A><A href="https://2017.igem.org/Team:Hamburg/Biochemistry">Biochemistry</A><A href="https://2017.igem.org/Team:Hamburg/Chemistry">Chemistry</A><A href="https://2017.igem.org/Team:Hamburg/Nanoscience">Nanoscience</A><A href="https://2017.igem.org/Team:Hamburg/Applied_Design">Applied Design</A><A href="https://2017.igem.org/Team:Hamburg/Parts">Parts</A></DIV></DIV><DIV class="dropdown"><BUTTON class="menu_link">Notebook</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Hamburg/Lab_Book">Lab Book</A><A href="https://2017.igem.org/Team:Hamburg/Protocols">Protocols</A></DIV></DIV><DIV class="dropdown"><BUTTON class="menu_link">Results</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Hamburg/Lab_Results">Lab Results</A><A href="https://2017.igem.org/Team:Hamburg/Demonstrate">Demonstrate</A><A href="https://2017.igem.org/Team:Hamburg/Improve">Improve a previous part or project</A><A href="https://2017.igem.org/Team:Hamburg/InterLab">InterLab Study</A></DIV></DIV><DIV class="dropdown"><BUTTON class="menu_link">Human Practices</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Hamburg/HP/Silver">Human Practices</A><A href="https://2017.igem.org/Team:Hamburg/HP/Gold_Integrated">Integrated HP</A><A href="https://2017.igem.org/Team:Hamburg/Business_Plan">Entrepreneurship</A></DIV></DIV><DIV class="dropdown"><BUTTON class="menu_link">Safety</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Hamburg/Safe_Design">Safe Design</A></DIV></DIV><DIV class="dropdown"><BUTTON class="menu_link">Fun</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Hamburg/The_stupid_Mistakes">The Pinky and the Brain</A><A href="https://2017.igem.org/Team:Hamburg/The_funniest_Things">The funniest Things</A><A href="https://2017.igem.org/Team:Hamburg/Records">Records</A><A href="https://2017.igem.org/Team:Hamburg/Other_languages">Other languages</A></DIV></DIV></DIV></DIV><DIV class="section_std"><DIV class="main_block"><H1 class="h1" style="margin-top:100px;">
    Improve a previous Part or Project
    </H1></DIV></DIV><DIV class="section_std"><DIV class="main_block"><H2 class="h2">
BioBrick <I>BBa_C0082</I> of <I>Antiquity</I> 2004
</H2><DIV class="text_block text"><P>
Last year, the iGEM Team Hamburg (Finding Chlamydori) worked on and modifying the receptor tar-envZ. This receptor is registered in the iGEM data bank as a fusion protein of the tar-domain and the intracellular EnvZ-domain, although the receptor has an aspartate-binding domain consisting of 484 amino acids. 255 amino acids belong to the N-terminal Tar-domain and 229 amino acids to the C-terminal EnvZ-domain. The assembled construct is 1516 base pairs long and registered as <I>BBa_C0082</I>. from the iGEM Team Antiquity. 
</P><P>
The function of both domains has already been thoroughly investigated Utsumi et al. [1] und Mise et al. [2], thereby only the periplasmic aspartate-binding domain (amino acids 26 to 193) has been expressed and studied.  In the original paper by Mise et al. [2], the Tar-domain was amplified out of the genome of Escherichia coli DH5a with primer pair 5‘ – ATG GCT AGC GAT GAC GAC GAC AAG GGC AGC CTG TTT TTT TCT TC – 3‘ (forward) and 5‘ – CTC GAA TTC TCA TTA TTG CCA CTG GGC AAA TC – 3‘ (reverse), and cloned in the expression vector pET-28a(+). In addition to the N-terminal start methionine, the vector has a thrombine cleavage site upstream of the C-terminal his-tag. According to the paper, tar domain was purified using the His-tag. His-tag renders the protein aggregation-prone, thus it has been removed afterwards through the thrombine cleavage site. The non-tagged protein was used for a crystallography.
</P><P>
The team last year did not investigate this part further. Instead, the Tar-EnvZ-receptor was cloned in an OmpR/GFP reporter construct and the GFP concentration measured after a mDAP induction of cells that were transformed with the whole construct. The results were published <A href="https://2016.igem.org/Team:Hamburg/Demonstrate" style="color: grey">(shown here) </A>, although the yields might have been higher, if the aggregation propensity has been known before.
</P><P>
Our goal this year was to investigate whether the his-tagged tar-protein aggregate completely or only to a certain degree and how this can be prevented. We used the expression vector pET-28a(+) with N-terminal methionine and C-terminal His-tag for both the Tar-domain from Mise et al. and the biobrick <I>BBa_C0082</I>. Instead of only using the sequence 26 to 193 amino acids, we used the entire 255 amino acid sequence of the tar-receptor of the Tar-EnvZ-receptor. We used the following primers:
</P><UL><LI>Forward primer: 5'-ATT GCA GCT AGC ATG ATT AAC CGT ATC CGC GTA GTC ACG CTG TTG G-3'</LI><LI>Reverse primer: 5'-ATT GCA GGA TCC TGA AAC GCT CTG CGC CAG GTC GC-3'</LI></UL><P>
The product is 765 base pairs long (255 amino acids) and has a molecular weight of 31.2 kDa. With this primer pair and a DNA sample of the biobrick <I>BBa_C0082</I> provided by the iGEM Headquarters we amplified the domain using the flowing PCR reaction:
</P><CENTER><TABLE class="table"><TBODY><TR><TD>
PCR reaction:
</TD></TR><TR><TD>
10x Pfu Polymerase Buffer
</TD><TD>
5.0 µL
</TD></TR><TR><TD>
dNTP's (10 mM each)
</TD><TD>
1.0 µL
</TD></TR><TR><TD>
Forward Primer
</TD><TD>
1.25 µL
</TD></TR><TR><TD>
Reverse Primer
</TD><TD>
1.25 µL
</TD></TR><TR><TD><I>BBa_C0082</I> (25 ng/µL)
</TD><TD>
1.0 µL
</TD></TR><TR><TD><I>Pfu</I> Polymerase (2.5 U/µL)
</TD><TD>
1.0 µL
</TD></TR><TR><TD>
Distilled Water
</TD><TD>
39.5 µL
</TD></TR><TR><TH>
total volume
</TH><TH>
50.0 µL
</TH></TR></TBODY></TABLE><TABLE class="table"><TBODY><TR><TD>
PCR program:
</TD></TR><TR><TD>
1.
</TD><TD>
Initial Denaturation
</TD><TD>
98 °C
</TD><TD>
2 min
</TD></TR><TR><TD>
2. A)
</TD><TD>
Denaturation
</TD><TD>
98 °C
</TD><TD>
30 sec
</TD></TR><TR><TD>
2. B)
</TD><TD>
Annealing
</TD><TD>
65 °C
</TD><TD>
30 sec
</TD></TR><TR><TD>
2. C)
</TD><TD>
Elongation
</TD><TD>
72 °C
</TD><TD>
1 min
</TD><TD>
GO TO 2. A) -&gt; repeat 25x
</TD></TR><TR><TD>
3.
</TD><TD>
Final Extension
</TD><TD>
72 °C
</TD><TD>
10 min
</TD></TR></TBODY></TABLE></CENTER><P>
The PCR product was purified using the GeneJET PCR Purification Kit by Thermo Fischer Scientific and eluded in 20 µL water. The purified product was cut with the NEB enzymes NheI und BamHI:
</P><CENTER><TABLE class="table"><TBODY><TR><TD>
BamHI-HF
</TD><TD>
1.0 µL
</TD></TR><TR><TD>
NheI-HF
</TD><TD>
1.0 µL
</TD></TR><TR><TD>
CutSmart Buffer
</TD><TD>
5.0 µL
</TD></TR><TR><TD>
PCR Product
</TD><TD>
20.0 µL
</TD></TR><TR><TD>
Distilled Water
</TD><TD>
23.0 µL
</TD></TR><TR><TD><B>
Total volume
</B></TD><TD><B>
50.0 µL
</B></TD></TR></TBODY></TABLE></CENTER><P>
The pET-28a(+) vector was restricted with both enzymes, but instead of adding the entire 20 µL PCR product only 10 µL vector were used and 10 µL more water. Both samples were incubated at 37 °C for one hour and purified with the Thermo Fischer Scientific PCR Purification Kit. Thereafter, the vector was dephosphorylated:
</P><CENTER><TABLE class="table"><TBODY><TR><TD>
Restricted and purified Plasmid
</TD><TD>
10.0 µL
</TD></TR><TR><TD>
10X reaction buffer for AP used in reaction
</TD><TD>
2.0 µL
</TD></TR><TR><TD>
FastAP Thermosensitive Alkaline Phosphatase
</TD><TD>
1.0 µL (1 U)
</TD></TR><TR><TD>
Distilled Water
</TD><TD>
7.0 µL
</TD></TR><TR><TD><B>
Total volume
</B></TD><TD><B>
50.0 µL
</B></TD></TR></TBODY></TABLE></CENTER><P>
The dephosphorylation was incubated at 37 °C for 30 minutes and heated to 75 °C for 5 minutes to stop the reaction.
</P><P>
Afterwards vector and insert were ligated with the Thermo Fischer Scientific T4 ligase.
</P><CENTER><TABLE class="table"><TBODY><TR><TD>
BamHI-HF
</TD><TD>
1.0 µL
</TD></TR><TR><TD>
NheI-HF
</TD><TD>
1.0 µL
</TD></TR><TR><TD>
CutSmart Buffer
</TD><TD>
5.0 µL
</TD></TR><TR><TD>
PCR Product
</TD><TD>
20.0 µL
</TD></TR><TR><TD>
Distilled Water
</TD><TD>
23.0 µL
</TD></TR><TR><TD><B>
Total volume
</B></TD><TD><B>
50.0 µL
</B></TD></TR></TBODY></TABLE></CENTER><P>
The pET-28a(+) vector was restricted with both enzymes, but instead of adding the entire 20 µL PCR product only 10 µL vector were used and 10 µL more water. Both samples were incubated at 37 °C for one hour and purified with the Thermo Fischer Scientific PCR Purification Kit. Thereafter, the vector was dephosphorylated:
</P><CENTER><TABLE class="table"><TBODY><TR><TD>
Linear vector DNA
</TD><TD>
100 ng
</TD></TR><TR><TD>
Insert DNA
</TD><TD>
1:5 molar ration over vector
</TD></TR><TR><TD>
10x T4 DNA Ligase buffer
</TD><TD>
2.0 µL
</TD></TR><TR><TD>
T4 DNA Ligase
</TD><TD>
1.0 µL (1 U)
</TD></TR><TR><TD>
Distilled Water
</TD><TD>
up to 20.0 µL
</TD></TR><TR><TD><B>
Total volume
</B></TD><TD><B>
20.0 µL
</B></TD></TR></TBODY></TABLE></CENTER><P>
The ligation mixture was transformed into <I>E.coli</I> DH5α cells (following the protocol from Zang Gong). Four colonies picked and the plasmid was isolated with the GeneJET Plasmid Miniprep Kit by Thermo Fischer Scientific and verified by sequencing. A colony-PCR was not performed, since the tar-receptor can be found in the native E. coli genome and therefore every colony would be positive.
</P><P>
A plasmid isolated from one colony of <I>E.coli</I> DH5α cells (verified by sequencing) was transformed and expressed in <I>E.coli</I> BL21(DE3). Therefore, 50 µL kanamycin [50 mg/mL] and 500 µL of the pET-28a(+) / Tar pre-culture were added to 50 mL fresh LB-media in a 300 mL flask and incubated at 37 °C at 220 rpm. The OD at 595 nm was measured every half hour and a 1 mL sample taken at an OD [595 nm] = 0.3. The culture was divided into 20 mL cultures and transferred in 100 mL flasks. One of the flasks was induced with 20 µL 400 mM IPTG and further incubated for 2h. The OD [595 nm] was measured every 30 min and a 1 mL sample for analysis was withdrawn every hour. The samples were centrifuged at 4 °C and 12,000 g for 10 minutes, the media was removed and cell pellets stored at -20 °C.
</P><CENTER><TABLE class="table"><CAPTION><DIV class="text_block text">
Table 1: Growth curve of the expression test of pET-28a(+): tar in <I>E.coli</I> BL21(DE3) at 37 °C, 220 rpm and OD at 595 nm.
</DIV></CAPTION><TBODY><TR><TH>
time (in min)
</TH><TH>
control
</TH><TH>
IPTG induced
</TH></TR><TR><TD>
0
</TD><TD>
0.038
</TD><TD>
0.038
</TD></TR><TR><TD>
30
</TD><TD>
0.061
</TD><TD>
0.061
</TD></TR><TR><TD>
60
</TD><TD>
0.030
</TD><TD>
0.030
</TD></TR><TR><TD>
90
</TD><TD>
0.058
</TD><TD>
0.058
</TD></TR><TR><TD>
120
</TD><TD>
0.151
</TD><TD>
0.151
</TD></TR><TR><TD>
145
</TD><TD>
0.294
</TD><TD>
0.294
</TD></TR><TR><TD>
175
</TD><TD>
0.632
</TD><TD>
0.559
</TD></TR><TR><TD>
205
</TD><TD>
1.242
</TD><TD>
0.570
</TD></TR><TR><TD>
235
</TD><TD>
2.044
</TD><TD>
0.658
</TD></TR><TR><TD>
265
</TD><TD>
2.490
</TD><TD>
0.606
</TD></TR></TBODY></TABLE></CENTER><P>
The expression was analysed on SDS polyacrylamide electrophoresis (SDS-PAGE). 
</P><CENTER><TABLE class="table"><TBODY><TR><TH>
Stacking gel (5 %)
</TH></TR><TR><TD>
30 % Polyacrylamide
</TD><TD>
0.85 mL
</TD></TR><TR><TD>
H<SUB>2</SUB>O
</TD><TD>
3.40 mL
</TD></TR><TR><TD>
1 M Tris-HCl (pH 6.8)
</TD><TD>
0.625 mL
</TD></TR><TR><TD>
10 % APS
</TD><TD>
50 µL
</TD></TR><TR><TD>
10 % SDS
</TD><TD>
50 µL
</TD></TR><TR><TD>
TEMED
</TD><TD>
5 µL
</TD></TR><TR><TD><B>
Total volume
</B></TD><TD><B>
5 mL
</B></TD></TR></TBODY></TABLE><TABLE class="table"><TBODY><TR><TH>
Separating gel (12 %)
</TH></TR><TR><TD>
30 % Polyacrylamide
</TD><TD>
4 mL
</TD></TR><TR><TD>
H<SUB>2</SUB>O
</TD><TD>
3.3 mL
</TD></TR><TR><TD>
1.5 M Tris-HCl (pH 8.8)
</TD><TD>
2.5 mL
</TD></TR><TR><TD>
10 % APS
</TD><TD>
100 µL
</TD></TR><TR><TD>
10 % SDS
</TD><TD>
100 µL
</TD></TR><TR><TD>
TEMED
</TD><TD>
5 µL
</TD></TR><TR><TD><B>
Total volume
</B></TD><TD><B>
10 mL
</B></TD></TR></TBODY></TABLE></CENTER><P>
The SDS-PAGE gels were prepared and stored at 4 °C in a wet cloth after polymerisation. 
</P><P>
Cell pellets were mixed with 2x SDS - loading buffer containing dye and ß-mercaptoethanol (for equal dilution of the samples the following was considered: OD[595 nm] = 0,1 was mixed with 100 µL of 2x SDS-loading buffer). The samples heated to 95 °C for 5 minutes and cooled down on ice.
</P><P>
Two parallel 12 % SDS-PAGE gels were loaded with samples. One gel was used for Coomassie staining and one for immunostaining. The gels were run at 10 mA for the stacking gel and 60 mA for the separating gel. The Coomassie stain was put in a Coomassie-Neuhaus/methanol solution and destained with water. The gel for the immunoblot was blotted on a PVDF membrane, blocked in 5 % milk powder in TBST buffer, and subsequently washed in TBST-buffer. Thereafter, the membrane was incubated with the first antibody, a mouse anti-His-antibody (dilutions 1:2,000), washed in TBST-buffer and incubated with the second goat-anti-mouse-HRP-antibody in TBST (dilution 1:10,000) containing 1 % milk powder. The detection was performed with ECL – solution (solution I: 100 µL luminole, 44 µL cumarine acid, 1 M Tris/HCl pH 8.5 with 8,85 mL water and solution II: 6 µL 30 % H<SUB>2</SUB>O<SUB>2</SUB>, 1 mL 1 M Tris/HCl pH 8.5 with 9 mL water).
</P><P>
In the Coomassie-gel the band of the tar-protein was not detectable, whereas it was clearly visible on the immunoblot and as expected only in the induced samples. Thus, we proved the expression of the protein with non-cleaved His-tag. The expression was repeated in larger expression volume (2.5 L) to investigate whether the protein can be purified without cleaving the His-tag, which is suggested to trigger partitioning of the protein into aggregates. The cells were cultured to an OD [595 nm] of 0,8, induced for 1h and collected by centrifugation. In the following, the samples were stored overnight at 4°C and resolved in NPI-10 buffer next morning. One sample was subjected to lysis by lysozyme and the other was lysed by mechanical treatment with Retsch-mill. His-tagged protein was purified with nickel-NTA-column and fractions from the washing and elution step were collected.
</P><P>
The first SDS-PAGE gels that were made could not be used for an evaluation since there are no separate lanes. For a better result, the first fraction of the washing step of the Redschmill-sample and the first three fractions of the elution were loaded on another SDS-PAA gel.
</P><P>
Clearly, the purification using His-tag was successful which is evidenced by the most intensive band at 30 kDa (lane 2-5 and 6-8, Fig. 4). Notably, there is no band in the wash fraction. The intensity of the band decreases with the increased elution volume (e.g. from lane 2 to 5 and form lane 6 to 8, Fig. 4) suggesting that the protein was quickly eluted with the first volume of the elution buffer. The successful purification is a clear proof that it is possible to purify this protein using the His-tag which does not seem to cause any aggregation. The latter might be due to the fact that we express short and might be a problem at long induction cycles. 
</P><P><B>References:</B>
[1] Utsumi, R., Brissette, R. E., Rampersand, A., Forst, S. A., Oosawa, K., Inonye, M. (1989). Activation of bacterial porin gene expression by a chimeric signal transducer in response to aspartate
</P><P>
[2] Mise, T., Matsunami, H., Samatey, F. A., Maruyama, I. N. (2014). Crystallization and preliminary X-ray diffraction analysis of the periplasmic domain of the Escherichia coli aspartate receptor Tar and its complex with aspartate
</P></DIV></DIV></DIV></DIV><DIV class="footer"><SPONSORED_BY class="h6">Sponsored by
  </SPONSORED_BY></DIV></DIV></DIV></DIV></DIV></BODY></HTML>