<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Hamburg_Collaborations skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Hamburg/Collaborations</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar"><DIV class="navbar_box"><DIV class="dropdown"><A class="menu_link_lone" href="https://2017.igem.org/Team:Hamburg#Section_Anchor" style=" color:#f8f8f8; text-decoration:none;">Home</A></DIV><DIV class="dropdown"><BUTTON class="menu_link">Team</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Hamburg/Team">Members</A><A href="https://2017.igem.org/Team:Hamburg/Collaborations">Collaborations</A><A href="https://2017.igem.org/Team:Hamburg/Attributions">Attributions</A></DIV></DIV><DIV class="dropdown"><BUTTON class="menu_link">Project</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Hamburg/Description">Description</A><A href="https://2017.igem.org/Team:Hamburg/Biochemistry">Biochemistry</A><A href="https://2017.igem.org/Team:Hamburg/Chemistry">Chemistry</A><A href="https://2017.igem.org/Team:Hamburg/Nanoscience">Nanoscience</A><A href="https://2017.igem.org/Team:Hamburg/Applied_Design">Applied Design</A><A href="https://2017.igem.org/Team:Hamburg/Parts">Parts</A></DIV></DIV><DIV class="dropdown"><BUTTON class="menu_link">Notebook</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Hamburg/Lab_Book">Lab Book</A><A href="https://2017.igem.org/Team:Hamburg/Protocols">Protocols</A></DIV></DIV><DIV class="dropdown"><BUTTON class="menu_link">Results</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Hamburg/Lab_Results">Lab Results</A><A href="https://2017.igem.org/Team:Hamburg/Demonstrate">Demonstrate</A><A href="https://2017.igem.org/Team:Hamburg/Improve">Improve a previous part or project</A><A href="https://2017.igem.org/Team:Hamburg/InterLab">InterLab Study</A></DIV></DIV><DIV class="dropdown"><BUTTON class="menu_link">Human Practices</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Hamburg/HP/Silver">Human Practices</A><A href="https://2017.igem.org/Team:Hamburg/HP/Gold_Integrated">Integrated HP</A><A href="https://2017.igem.org/Team:Hamburg/Business_Plan">Entrepreneurship</A></DIV></DIV><DIV class="dropdown"><BUTTON class="menu_link">Safety</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Hamburg/Safe_Design">Safe Design</A></DIV></DIV><DIV class="dropdown"><BUTTON class="menu_link">Fun</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Hamburg/The_stupid_Mistakes">The Pinky and the Brain</A><A href="https://2017.igem.org/Team:Hamburg/The_funniest_Things">The funniest Things</A><A href="https://2017.igem.org/Team:Hamburg/Records">Records</A><A href="https://2017.igem.org/Team:Hamburg/Other_languages">Other languages</A></DIV></DIV></DIV></DIV><DIV class="section_std"><DIV class="main_block"><H1 class="h1" style="margin-top:100px;">Collaborations
    </H1></DIV></DIV><DIV class="section_std"><DIV class="main_block"><H2 class="h2" style="margin-bottom:70px;"><A href="https://2017.igem.org/Team:Evry_Paris-Saclay/Collaborations" style=" color:#f8f8f8; text-decoration:none;">Evry Paris-Saclay : Golden Gate Assembly for HMWP2</A></H2><DIV class="text_block text"><TEXT class="text">
                 We had problems from the very beginning with restricting/ligating and transforming one of our main genes. Namely HMWP2, this gene encodes for a fairly big synthetase that catalyzes the first part of yersiniabactin, which is identical to N-Pch, therefore we wanted to utilize the catalytic functions of HMWP2 to synthesize our main target.
         <P class="text">
                 However, since it is a fairly big protein with 6.5 kb encoding for it, we could not order it as a single DNA part from IDT, instead we had to break it up in 3 parts, inexperienced as we were we thought well, let’s split it up in three parts, as the maximum size we could order was 3 kb. This approach implied, that we would need to include restriction sites for single cutting enzymes, which we did, XmaI and MaubI where the enzymes of choice and we tried to ligate and transform all three parts into psB1C3.
         </P><P class="text">
                 As it turned out it is quite hard to achieve a ligation that complicated, still we tried our approach for several weeks, fueled by some false positive results we thought well we might make it. In the end though we gave up and admitted that we would need some help to finish our most important BioBrick.
         </P><P class="text">
                 So, we reached out to our fellow iGEM teams first we had a discussion with Heidelberg, which proposed to use the Golden Gate method as well as Evry did later then. The collaboration with Evry sprung from a personal conversation between our team leader Shanti and Evry’s head of human practices Yanis, who established the contact with their head of science Jérémy.
         </P><P class="text">
                 Jérémy showed us how to design a golden gate assembly, while also providing us an entire <A href="https://2017.igem.org/Team:Hamburg/GGProtocoll" style=" color:#f8f8f8; text-decoration:none;"><U>protocol</U></A> to ensure our ligation step, with the planned assembly a transformation is almost guaranteed to work, however the correct order of the fragments is not therefore a relatively extensive screening process is needed to find the colonies that carry the designed fragment.
                 Additionally, we had ongoing support by Jérémy to fix upcoming issues, which we had a lot of.
         </P><P class="text">
                 Alina, who was in charge of our security level 2 lab, contacted Jérémy for further troubleshooting, hence we had starting issues with the protocol. Continuous communication between our team and Jérémy helped us to fix our issues. We needed to tweak the PCR reactions due to some chemicals, enzymes being unavailable to us, being inexperienced with the method we needed council to make sure our changes would not alter the outcome.
         </P><P class="text">
                 Eventually Alina was able to troubleshoot the Golden-Gate-Assembly with Jérémy’s help so that we were finally able to amplify the whole construct with 6107 bp. Which we were able to verify by colony PCR and sequencing.
         </P><P class="text">
                 Since we were unable to find a proper solution to our problems with HMWP2 ourselves, many thanks go to Evry for their dedicated support.
         </P></TEXT></DIV><H2 class="h2">TU Delft</H2><H2><DIV class="text_block text"><TEXT class="text">
                 A collaboration with iGEM Delft felt natural to us for multiple reasons, first in the current lineup from Delft is a former student, Isabell, from our university, which prompted us to contact team Delft early. While we were visiting Delft for the European meetup we met the  team multiple times. During our stay  in Delft we had multiple occasions where we exchanged ideas and spoke about each other's projects.
         <P class="text">
                 Second our projects are basically designed to work together, Delft worked on a crispr-cas based detection method for multiresistant bacteria, and we developed a new antibiotic compound to target some of those. Sadly, this would involve both teams completing their projects before we start a collaboration. Which might have worked, in reality we both found out that synthetic biology often takes a lot of time and the verification of the own experiments has to be prioritized.
         </P><P class="text">
                 Nonetheless we made plans, even though they never came to fruition. The planned cooperation involved Delft identifying enterobacteria that are carbapenem resistant, identified strains would then be tried with our novel antibiotic compounds and tested for siderophore receptors. All in all we would have been able to identify and eliminate multiresistant bacterial threats.
                 Many thanks to iGEM Delft who were great partners in developing important ideas to change our world for the better, and of course for the organisation of the European Meetup!
                 </P></TEXT></DIV></H2><H2 class="h2" style="margin-bottom:70px;">Delft &amp; Aalto Helsinki</H2><DIV class="text_block text"><P>
                 Hence our first collaboration, with iGEM Delft would have started way to late in the current competition. We developed further ideas as how to combine projects most of which did not work out, this specific example however is something that we want to share because it would have been amazing and did fail due to one reason alone: The lack of time.
          </P><P class="text">
                 So how did it start? Delft informed us, that they were working on tardigrade proteins in a side project, which can help folding and unfolding proteins during and after states of cryptobiosis. Which essentially means damage through repeated freezing and thawing, clumping and the like are prevented by the proteins.
         </P><P class="text">
                 Aalto Helsinki, who worked on antimicrobial peptides as well informed us earlier, that they didn’t have access to security level 2 labs to test their peptide on real causative agents, so we offered them to test their peptides on some clinical strains that we could work with in our security level 2 lab. Aalto Helsinki told us that they would have problems to send us their peptides due to activity loss and too little time.
         </P><P class="text">
                 Which led to our conclusion to bring both teams together which was the start of a wild ride. After proposing our idea to Delft, they told us that one of their team members would travel to Helsinki in the next couple of weeks in October, which would allow the transportation of the tardigrade proteins to Helsinki. So that Aalto Helsinki could freeze their peptides send them to us so that we could test them on strains of <I>Klebsiella pneumoniae</I> and <I>Pseudomonas aeruginosa</I>.
         </P><P class="text">
In the end it did not work out, as mentioned above due to a lack of time, but we are still positive that the attempt was totally worth it and want to encourage future iGEM teams to think big, even if it might not work out in the end.
</P></DIV><H2 class="h2"> Heidelberg </H2><H2><DIV class="text_block text"><P>
Since the beginning of our project we experienced difficulties performing gene amplifications. After
tedious trial and error using different gradient-PCRs, experimenting with GC-enhancer solutions and
DMSO concentrations we figured out a way to amplify eleven out of our twelve genes and gene
fragments. The only one that did not function despite our efforts was the AraC family transcriptional
regulator.
Luckily the iGEM Team Heidelberg offered a PCR First Aid – Service this year which we gratefully
accepted. After a skype call in which we described our problem and attempts at solving it in detail
they gave us valuable tips. Their ideas were a raise of the annealing temperature to 72 °C, adding 0.5
M betaine, try adding DMSO in 3 %, 5 % or even 10 % concentration or a combination of both DMSO
and betaine.
We gave a try to all these suggestions, unfortunately without having a hit. We expected a lane at
about 1100 bp.
</P><P>
The PCR was tested with a Q5-High- Fidelity polymerase and a Phusion polymerase but neither led to
a positive result.
</P><P>
We discussed the result with the iGEM Team Heidelberg who did not want to give up and gave us
additional tips to try. Their next suggestion was to design new primes, as ours had a very high GC
content in the middle part and were therefore prone to the formation of secondary structures. Also,
the new primer pairs should be designed to anneal at 72 °C. They offered to design to take over the
design for us.
These were the original primers:
<UL><LI>Fw: CATTATGAATTCGCGGCCGCTTCTAGA</LI><LI>Rv: GATAATCTGCAGCGGCCGCTACTAGTA</LI></UL>
These are the newly designed primers by the iGEM Team Heidelberg:
<UL><LI>Fw: CATTATGAATTCGCGGCCGCTTCTAGATGACCATCACCATC</LI><LI>Rv: GATAATCTGCAGCGGCCGCTACTAGTATTAG</LI></UL></P><P>
Unfortunately, we did not have enough time at this stage of the project to order and test new
primers. Therefore, we will only add the primers to the gene to facilitate the work for the next group
who wants to work on the gene.
</P><P>
Even though the gene amplification was not successful in the end, we are very grateful for the help of
the iGEM Team Heidelberg, especially Pauline for her advice and not giving up on our problem. Thank
you very much!
</P></DIV></H2><H2 class="h2"> Harvard </H2><H2><DIV class="text_block text"><P>
The iGEM team Harvard offered the opportunity to participate in the design process of a set of questions for human practices partner in politics, economics, and science. The questions were targeted to set standards for future surveys so that the results become comparable. And that future teams can easily navigate through interviews with highly qualified personnel originating from the respective fields.
</P><P>
Many teams came together working on the questionnaire, however we want to mention one team especially: Aalto Helsinki, adding a ton of good questions while also fueling the discussion.
</P></DIV><DIV class="footer"><SPONSORED_BY class="h6">Sponsored by
  </SPONSORED_BY></DIV></H2></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>