<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Brown-Stanford_PowerCell_NutrientSecretion"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Brown-Stanford/PowerCell/NutrientSecretion">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Brown-Stanford/PowerCell/NutrientSecretion&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Brown-Stanford/PowerCell/NutrientSecretion&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Brown-Stanford/PowerCell/NutrientSecretion&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Brown-Stanford/PowerCell/NutrientSecretion" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Brown-Stanford/PowerCell/NutrientSecretion</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV style="margin-left: auto; margin-right: auto; margin-top:-15px; z-index:-10; text-align: center;"><A href="/Team:Brown-Stanford" style="text-decoration: none !important;"><SPAN style="font-size: 40pt !important; line-height: 40pt !important; color: white !important; font-weight: bold !important;">Brown-StanfordiGEM</SPAN></A></DIV><DIV id="header" style="height:100px;padding: 0px 0px 0px 0px; margin-bottom:0px"><UL id="sdt_menu" class="sdt_menu"><LI><A href="/Team:Brown-Stanford/REGObricks/Introduction"><SPAN class="sdt_wrap"><SPAN class="sdt_link">REGObricks</SPAN></SPAN></A><DIV class="sdt_box"><A href="/Team:Brown-Stanford/REGObricks/Introduction">Intro</A><D>---------------------</D><A href="/Team:Brown-Stanford/REGObricks/ISRU">ISRU</A><D>---------------------</D><A href="/Team:Brown-Stanford/REGObricks/Biocementation">Biocement</A><D>---------------------</D><A href="/Team:Brown-Stanford/REGObricks/Characterization"><EM>S. pasteurii</EM></A><D>---------------------</D><A href="/Team:Brown-Stanford/REGObricks/Balloon">BalloonFlights</A><D>---------------------</D><A href="/Team:Brown-Stanford/REGObricks/Transforming">Transforming</A><D>---------------------</D><A href="/Team:Brown-Stanford/REGObricks/Biobrick">Biobrick<SUP>2</SUP></A></DIV></LI><LI><A href="/Team:Brown-Stanford/PowerCell/Introduction"><SPAN class="sdt_wrap"><SPAN class="sdt_link">PowerCell</SPAN></SPAN></A><DIV class="sdt_box"><A href="/Team:Brown-Stanford/PowerCell/Introduction">Intro</A><D>---------------------</D><A href="/Team:Brown-Stanford/PowerCell/Cyanobacteria">Cyanobacteria</A><D>---------------------</D><A href="/Team:Brown-Stanford/PowerCell/Background">Mars</A><D>---------------------</D><A href="/Team:Brown-Stanford/PowerCell/NutrientSecretion">Nutrients</A></DIV></LI><LI><A href="/Team:Brown-Stanford/FRETSensor/Introduction"><SPAN class="sdt_wrap"><SPAN class="sdt_link">FRETcetera</SPAN></SPAN></A><DIV class="sdt_box"><A href="/Team:Brown-Stanford/FRETSensor/Introduction">Intro</A><D>---------------------</D><A href="/Team:Brown-Stanford/FRETSensor/Biosensing">FRET etc.</A><D>---------------------</D><A href="/Team:Brown-Stanford/FRETSensor/Construct">Construct</A></DIV></LI><LI><A href="/Team:Brown-Stanford/Team"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Lab</SPAN></SPAN></A><DIV class="sdt_box" style="height:60px"><A href="/Team:Brown-Stanford/Team">Team</A><D>---------------------</D><A href="/Team:Brown-Stanford/Lab/Parts">Parts</A><D>---------------------</D><A href="/Team:Brown-Stanford/Lab/Notebook">Notebook</A><D>---------------------</D><A href="/Team:Brown-Stanford/Lab/Protocols">Protocols</A><D>---------------------</D><A href="/Team:Brown-Stanford/Lab/Safety">Safety</A></DIV></LI><LI><A href="/Team:Brown-Stanford/Collaboration"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Partners</SPAN></SPAN></A><DIV class="sdt_box"><A href="/Team:Brown-Stanford/Collaboration">Collab</A><D>---------------------</D><A href="/Team:Brown-Stanford/Sponsors">Sponsors</A></DIV></LI><LI><A href="/Team:Brown-Stanford/Outreach"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Outreach</SPAN></SPAN></A><DIV class="sdt_box"><A href="/Team:Brown-Stanford/SB5">SB5.0</A><D>---------------------</D><A href="/Team:Brown-Stanford/NASA">NASA</A><D>---------------------</D><A href="/Team:Brown-Stanford/Lunar">Lunar Science</A><D>---------------------</D><A href="/Team:Brown-Stanford/MakerFaire">Maker Faire</A><D>---------------------</D><A href="/Team:Brown-Stanford/BBC">BBC</A><D>---------------------</D><A href="http://openwetware.org/wiki/IGEM_Outreach" target="_blank">CBricks</A><D>---------------------</D><A href="JavaScript:ethics('/Team:Brown-Stanford/SynEthics/Intro');">Ethics</A><D>---------------------</D><A href="/Team:Brown-Stanford/RegionalJamboree">RegionalJamboree</A></DIV></LI></UL></DIV><DIV id="subHeader"><UL id="subHeaderList"><LI><A href="/Team:Brown-Stanford/PowerCell/Introduction">Introduction</A></LI><LI><A href="/Team:Brown-Stanford/PowerCell/Cyanobacteria">Cyanobacteria</A></LI><LI><A href="/Team:Brown-Stanford/PowerCell/Background">Photosynthesis on Mars</A></LI><LI id="active"><A href="#" id="current">Nutrient Secretion and Utilization</A></LI></UL></DIV><DIV class="earth" style="text-align: left; padding: 5px;"><A href="/" style="font-size: 20px; line-height: 20px; color: white; font-weight: bold; text-decoration: none;">Back toiGEM HQ</A></DIV><DIV class="pageContent"><H2><SPAN class="mw-headline" id="Nutrient_Secretion_and_Utilization"><B>Nutrient Secretion and Utilization</B></SPAN></H2><P>PowerCell is the generator that will power all of the BioTools in a Martian settlement. PowerCell will secrete nutrients for use by other biological tools a source of energy and raw materials to construct useful products.
</P><P>Like any other tools, biological tools need energy to run and raw materials to work with. The two major nutrient sources needed for biological tools are carbon sources (sugars) and nitrogenous nutrients (ammonia). In a Martian settlement, PowerCell will harness the energy of the sun to harvest these nutrients directly from the  atmosphere and provide them to the settlement's biological tools.
</P><P>This summer, we tackled only the half involving sugar secretion. This was partially to due to the time constraints of a summer project, and partially because there is evidence that ammonia secretion occurs in cyanobacteria by means of passive diffusion without the need for genetic tinkering (<A href="http://www.sciencedirect.com/science/article/pii/0378109791906924" class="external text" rel="nofollow"><I>Ammonia translocation in cyanobacteria</I></A>). More research needs to be done to determine if this level of diffusion is sufficient to sustain other biological tools. In any case, we designed our system on a <A href="https://2011.igem.org/Team:Brown-Stanford/PowerCell/Cyanobacteria" class="external text" rel="nofollow">platform</A> (<I>Anabaena 7120</I>) with the capability to fix atmospheric nitrogen, so that this avenue is open for exploration in the future.
</P><H3><SPAN class="mw-headline" id="Sucrose"> Sucrose </SPAN></H3><P>The sugar we chose to secrete is sucrose. The major inspiration for our project was the paper <A href="http://aem.asm.org/cgi/content/abstract/76/11/3462" class="external text" rel="nofollow"><I>Engineering Cyanobacteria To Synthesize and Export Hydrophilic Products</I></A>. This work was done in Dr. Pamela Silver's lab, and describes a glucose/fructose secretion device, achieved in the single-celled bacterium <I>S. elongatus</I>.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Sugar secretion <I>Engineering Cyanobacteria To Synthesize and Export Hydrophilic Products</I>. <I>Niederholtmeyer et al.</I></DIV></DIV></DIV><P>In a nutshell, <I>S. elongatus</I> were forced under salt stress to produce sucrose, which was broken down into glucose and fructose by the glf enzyme, then transported out of the cell by the invA transporter.
</P><P>At the Fifth International Conference on Synthetic Biology, <A href="https://2011.igem.org/Team:Brown-Stanford/SB5" class="external text" rel="nofollow">SB5.0</A>, we spoke with a member of Dr. Silver's lab, Danny Ducat. He advised us that one of the problems with the glucose/fructose secretion system is its low yield. He suspected that because glucose and fructose are directly metabolizable by the cell, much of these sugars are consumed by the cell before they ever have a chance to be secreted. For this reason, it may be better to directly secrete sucrose, which is not metabolizable by the cell, and worry about breaking it down later. This is what we did, although the choice does have some ramifications.
</P><P>Because <I>E. coli</I> is the a primary host species for genetic modification, it is likely that many biological tools for space exploration will be designed in <I>E. coli</I>. The most common laboratory strains of <I>E. coli</I> , TOP10 and K12, cannot directly metabolize sucrose. For this reason, they cannot make use of PowerCell's output directly. In order to utilize this output, sucrose will need to be broken down into glucose and fructose , or biological tools that intend to utilize PowerCell should be hosted by a strain that can directly metabolize sucrose. <I>E. coli W</I> is a safe and easily transformable laboratory strain with this ability, and, as outlined in <A href="http://www.biomedcentral.com/1471-2164/12/9#IDAJ1FOQ" class="external text" rel="nofollow"><I>The genome sequence of E. coli W (ATCC 9637): comparative genome analysis and an improved genome-scale reconstruction of E. coli</I></A>, may confer some other advantages to biological tools to be used in space, including including fast growth, stress tolerance, growth to high cell densities. Furthermore, there is some indication that sucrose as a feedstock can confer other benefits, such as oxidative, heat, and acid stress (<A href="http://www.springerlink.com/content/b80870618103w67l/" class="external text" rel="nofollow"><I>Development of sucrose-utilizing Escherichia coli K-12 strain by cloning Î²-fructofuranosidases and its application for l-threonine production</I></A>).
</P><H3><SPAN class="mw-headline" id="PowerCell_Contruct_Design"> PowerCell Contruct Design </SPAN></H3><P>We wanted to isolate sucrose secretion to just vegetative cells, as heterocysts do not participate in production of sucrose through photosynthesis. A similar cell-type-specific gene expression was was achieved in <A href="http://www.sciencedirect.com/science/article/pii/S0167701204001745" class="external text" rel="nofollow"><I>Construction and use of GFP reporter vectors for analysis of cell-type-specific gene expression in Nostoc punctiforme</I></A>. Arguetta et al. confined expression of a fluorescent marker to only vegetative cells in a similar filamentous diazotrophic cyanobacterium, <I>Nostoc Punctiforme</I>. They placed the GFP gene behind a promoter present in the Photosystem I of <I>Nostoc punctiforme</I>, psaC. This way, GFP expression was only turned on in photosynthesizing cells, and not in heterocysts. They did not know the exact sequence of the psaC promoter, so they just took the 400 base pairs upstream of the psaC gene as a unit containing the promoter. We have done similarly to isolate the psaC promoter from <I>Anabaena 7120</I>. Below is an image showing cell-type-specific GFP expression achieved in the paper. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:674px;"><DIV class="thumbcaption">Cell-type-specific GFP expression in <I>Nostoc  punctiforme</I>. Arguetta et al.</DIV></DIV></DIV></DIV><P>
We placed a sucrose symporter gene, cscB, behind our <I>Anabaena 7120</I> psaC promoter in order to  confine sucrose secretion to only vegetative cells. Registry standard RBSes and terminators were used for ease of assembly and compatibility. Below is a diagram of our sucrose secretion device. A discussion of the inner workings of the cscB gene can be found in <A href="http://www.sciencedirect.com/science/article/pii/S0006291X85714490" class="external text" rel="nofollow"><I>Active Transport by the CscB Permease in Escherichia coli K-12</I></A>.
</P><P>
We have also created several other constructs containing GFP for debugging purposes. We used a modified version of GFP, GFPmut3B because normal florescence markers are easily lost among background chlorophyll pigments in cyanobacteria (<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2860132/" class="external text" rel="nofollow"><I>Design and characterization of molecular tools for a Synthetic Biology approach towards developing cyanobacterial biotechnology</I></A>). 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Final PowerCell Construct: <A href="http://partsregistry.org/Part:BBa_K656012" class="external free" rel="nofollow">http://partsregistry.org/Part:BBa_K656012</A></DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Tranformation_into_Anabaena_7120"><B>Tranformation into Anabaena 7120</B></SPAN></H2><P>Cyanobacterial transformation can be difficult.
</P><P>There are cyanobacteria which will accept DNA without complaint.  Many model cyanobacterium species, such as <I>Synechococcus elongatus</I> or <I>Synechocystis</I> PCC6803, for example, will simply take up naked DNA in solution and express it. 
</P><P><I>Anabaena 7120</I>, on the other hand, must take its DNA through a rather circuitous path; the DNA construct must first be placed in a cargo plasmid and transformed into <I>E. coli</I> by traditional means.  Transfer to <I>Anabaena</I> takes place by conjugation, facilitated by a second <I>E. coli</I> strain carrying a plasmid encoding the machinery for bacterial conjugation. 
</P><P>Another problem is the propensity of <I>Anabaena</I> to slice and dice foreign DNA with isoschizomers of the restriction enzymes AvaI, AvaII and AvaIII.  This has been addressed with methyltransferases targeting the same sequences; that means a third <I>E. coli</I> strain carrying these methyltransferases (a helper plasmid) participates in the conjugation.  At the end of all this, a certain number of cyanobacterial cells take up the DNA, and are selected for with neomycin on minimal media.  As soon as the unsuccessful exconjugates and the bacterial parental strains die off, transformant colonies can be picked.  One significant hindrance associated with this process is the slow cyanobacterial doubling time; the transformants can take upwards of a week to grow.
</P><P>Our first trials were performed with the helper plasmid pDS4101, the conjugative plasmid pRL443 and the cargo plasmid pRL25.  pDS4101 does not possess the methyltransferases which can epigenetically protect the incoming DNA from restriction, and so can only be used to transfer plasmids lacking restriction sites, such as pRL25, as a proof of concept.  
</P><P>Towards the end of the summer we were able to obtain the helper plasmid pRL623 containing the three methyltransferases which together offer complete protection from restriction inside <I>Anabaena</I>.  There was an added obstacle to this new route, however: pRL623 can only be carried against certain <I>E. coli</I> genetic backgrounds lacking methyl-restricting enzymes, in order to prevent restriction of the host genome.  This meant that transfer of the helper plasmid into the cargo plasmid strain before conjugation into <I>Anabaena</I> results in the suicide of that cargo strain, and no transformation.  The solution to this was transformation by more traditional means of the cargo plasmid into the strain already carrying the helper plasmid; the cargo was then be delivered into <I>Anabaena</I> without transfer of the dangerous helper plasmid, and the helper plasmid had its opportunity to protectively methylate the cargo prior to transformation.
</P><P>We are sincerely grateful to Dr. Jeff Elhai (Virginia Commonwealth University), Dr. Peter Wolk (Michigan State University), James Golden (University of California, San Diego) for their crucial help and guidance with cyanobacterial transformation.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Triparental mating: Our desired construct (from the <B>donor strain</B>) and a helper plasmid are inserted into a <B>helper strain</B>. The <B>helper strain</B> and <B>conjugative strain</B> are spotted with the <B>recipient</B> Anabaena for the three-parent mating</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="PowerCell_Transformation"><B>PowerCell Transformation</B></SPAN></H2><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Anabaena culture after triparental mating (bright field light micrograph)</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Same Anabaena culture showing GFP expression in successful transformant</DIV></DIV></DIV></DIV><P>We transformed the final PowerCell construct into Anabeana using the methods described above. Expression of GFP is clearly visible, indicating the transformation was successful. Some non-transformants can be seen in the background.
</P><P>We selected for our transformants using neomycin, and let the cultures grow.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Left: BG11(N-) w/o neomycin selection + 50Âµl of the original conjugation mixture. Middle: BG11(N-) with neomycin + 50Âµl of the original conjugation mixture. Right: BG11(N-) with neomycin + 50Âµl of WT Anabaena.</DIV></DIV></DIV></DIV><P>The initial selection looked promising. The left-most culture tube contains BG11(N-) w/o neomycin selection + 50Âµl of the original conjugation mixture. Both transformed and untransformed Anabaena should be able to grow in this mixture. As expected, this culture tube has the highest cell density. The middle culture tube contains BG11(N-) with neomycin + 50Âµl of the original conjugation mixture. Only Anabaena containing our plasmid should be able to grow in this mixture. As desired, growth is still visible, but is less dense. The right-most tube contains BG11(N-) with neomycin + 50Âµl of WT Anabaena. Nothing should be able to grow here. As expected, everything in this tube is dead.
</P><CENTER><DIV style="width: 35%;"><P>Transformed Anabaena showing cell-type specific GFP fluorescence under the control of the pSac promoter. GFP expression is on in vegetative cells, off in heterocysts. Some background fluorescence from chlorophyll is visible.
</P></DIV><DIV style="width: 35%;"><P>Transformed Anabaena with heterocysts stained by alcian blue. Heterocyst spacing corresponds to the &quot;off&quot; cells from the GFP image, implying that the sucrose secretion construct in not active in heterocysts, as desired.
</P></DIV></CENTER><P>
After about 2 weeks of growth, we can confirm the correct regulation of our sugar secretion construct by our pSac promoter by observing the presence of the GFP reporter. GFP expression is on in vegetative cells, and off in heterocysts, which implies that the sugar secretion construct is only active in vegetative cells, as we desire. Work is currently underway to assay the sucrose concentrations that are generated by our construct. We hope that these concentrations will be high enough to support growth of E. coli W; our experiments suggest the minimum concentration is around 5 mM (see section below). If the concentrations are high enough, we intend to grow E. coli W hosting several arbitrary active BioBrick plasmids to show that PowerCell can power general biological tools. Updates will be posted here as they become available! 
</P><H2><SPAN class="mw-headline" id="Utilizing_sucrose_from_PowerCell"><B>Utilizing sucrose from PowerCell</B></SPAN></H2><H3><SPAN class="mw-headline" id="E._coli_W"><B>E. coli W</B></SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Sugarcane is an extremely low-cost source of bulk sucrose used in production</DIV></DIV></DIV><P>An essential aspect of the PowerCell project lies in microorganisms being able to survive off the nutrients being secreted by our genetically engineered <I>Anabaena</I>. Our carbon source, sucrose, is a disaccharide that is not necessarily able to be metabolized by all species.
</P><P>Here our project intersects with current trends of bioproduction on Earth; sucrose is a cheap and easily obtained sugar, so there is interest in engineerable strains of microorganisms able to survive on it as a sole carbon source. 
</P><P>We came across research suggesting that <I>E. coli</I> W is one such promising strain. It is one of the few strains of non-pathogenic <I>E. coli</I> able to metabolize sucrose, has good tolerance for environmental stresses, and grows rapidly. (Archer 2011) There have been papers describing the process of adapting <I>E. coli</I> W as a bioreactor organism, and methods of growing them to high density and productivity using fed-batch cultures. (Lee 1993, Lee 1997). 
</P><P>In fact, the strain has already been used to produce a number of useful products. These include D(-)-lactate, a precursor to the formation of certain biodegradable plastic polymers, and L-alanine, a food additive and nutritional supplement. (Shukla 2004, Zhang 2007) With these examples, it is not difficult to imagine the production role of <I>E. coli</I> W in our Martian colony.
</P><H3><SPAN class="mw-headline" id="E._coli_W_Sucrose_Metabolism_Experiment"><B>E. coli W Sucrose Metabolism Experiment</B></SPAN></H3><P>In the application of our project, PowerCell would grow alongside and support a productive microorganism strain such as E. coli W. We were thus interested in seeing whether <I>E. coli</I> W could grow in the conditions generated by PowerCell. This meant culturing on minimal BG-11 media with added sucrose to simulate secretion by <I>Anabaena</I>.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Growth of our <I>E. coli</I> W varies by amount of sucrose (there was no 200ul plate for 80mM sucrose)</DIV></DIV></DIV><P>Based on literature (Niederholtmeyer, et al,2010) and correspondence with Mr. Ducat, we predicted that our <I>Anabaena</I> culture would produce no more than 8mM of sucrose. It would be difficult to support substantial E. coli growth on such low levels without a means of concentrating sugar via in the bioreactor design. 
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">E. coli W colonies on BG-11 agar + 60 mM sucrose</DIV></DIV></DIV><P>Initially we attempted to compare growth on BG-11 + sucrose with liquid cultures. However, multiple attempts yielded no appreciable change in O.D. after several days of incubation. We suspect that this was because <I>E. coli</I> growth was too small to detect via spectrophotometer. 
</P><P>Next, we moved to agar plates as a more sensitive means of detecting cell growth. <I>E. coli</I> W with Amp resistance were grown, washed and resuspended in PBS to eliminate residual LB media, and streaked them on BG-11 Amp+ plates with varying concentrations of sugar (5mM, 10mM, 30mM, 60mM, and 80mM).
</P><P>The cells were incubated at 37C and observed regularly before colonies appeared on the fifth day. Visible colonies were observed on plates with sucrose concentrations as low as 5mM. These data suggest that, with the sugar secreted by PowerCell expressing cscB and no additional metabolic engineering, it is possible to support another organism.
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P style="line-height:90%; text-indent: -15px !important;"><SUP><SPAN style="font-size:11px;"><A href="#1" name="1num">1</A></SPAN></SUP><SPAN style="font-size:11px">Sammy Boussiba, Jane Gibson, Ammonia translocation in cyanobacteria, FEMS Microbiology Letters, Volume 88, Issue 1, July 1991, Pages 1-14, ISSN 0378-1097, 10.1016/0378-1097(91)90692-4
Niederholtmeyer, Henrike, Wolfstadter, Bernd T., Savage, David F., Silver, Pamela A., Way, Jeffrey C. Engineering Cyanobacteria To Synthesize and Export Hydrophilic Products. Appl. Environ. Microbiol. 2010 76: 3462-3466</SPAN></P><P style="line-height:90%; text-indent: -15px !important;"><SUP><SPAN style="font-size:11px;"><A href="#2" name="2num">2</A></SPAN></SUP><SPAN style="font-size:11px">Kim, Jihyun F , Lars K Nielsen, Colin T Archer, Sang Yup Lee, Claudia E Vickers, Jin Hwan Park, and Haeyoung Jeong. &quot;The genome sequence of E. coli W (ATCC 9637): comparative genome analysis and an improved genome-scale reconstruction of E. coli .&quot;BioMed Central 12 (2011).</SPAN></P><P style="line-height:90%; text-indent: -15px !important;"><SUP><SPAN style="font-size:11px;"><A href="#3" name="3num">3</A></SPAN></SUP><SPAN style="font-size:11px">Lee, Jeong Wook, Sol Choi, Jin Hwan Park, Claudia E. Vickers, Lars K. Nielsen, and Sang Yup Lee. &quot;Development of Sucrose-utilizing Escherichia Coli K-12 Strain by Cloning Î²-fructofuranosidases and Its Application for L-threonine Production.&quot;Applied Microbiology and BiotechnologyÂ 88.4 (2010): 905-13. Print.</SPAN></P><P style="line-height:90%; text-indent: -15px !important;"><SUP><SPAN style="font-size:11px;"><A href="#4" name="4num">4</A></SPAN></SUP><SPAN style="font-size:11px">Claudia Argueta, Kamile Yuksek, Michael Summers, Construction and use of GFP reporter vectors for analysis of cell-type-specific gene expression in Nostoc punctiforme, Journal of Microbiological Methods, Volume 59, Issue 2, November 2004, Pages 181-188, ISSN 0167-7012, 10.1016/j.mimet.2004.06.009.</SPAN></P><P style="line-height:90%; text-indent: -15px !important;"><SUP><SPAN style="font-size:11px;"><A href="#5" name="5num">5</A></SPAN></SUP><SPAN style="font-size:11px">M. Sahintoth, S. Frillingos, J.W. Lengeler, H.R. Kaback, Active Transport by the CscB Permease in Escherichia coli K-12, Biochemical and Biophysical Research Communications, Volume 208, Issue 3, 28 March 1995, Pages 1116-1123, ISSN 0006-291X, 10.1006/bbrc.1995.1449.</SPAN></P><P style="line-height:90%; text-indent: -15px !important;"><SUP><SPAN style="font-size:11px;"><A href="#6" name="6num">6</A></SPAN></SUP><SPAN style="font-size:11px">Huang, H. H., D. Camsund, P. Lindblad, and T. Heidorn. &quot;Design and Characterization of Molecular Tools for a Synthetic Biology Approach towards Developing Cyanobacterial Biotechnology.&quot;Â Nucleic Acids ResearchÂ 38.8 (2010): 2577-593. Print.</SPAN></P><P style="line-height:90%; text-indent: -15px !important;"><SUP><SPAN style="font-size:11px;"><A href="#7" name="7num">7</A></SPAN></SUP><SPAN style="font-size:11px">Lee, SY and Ho Nam Chang. High cell density cultivation of Escherichia coli W using sucrose as a carbon source. Biotechnology Letters 15:9 (1993) 971--974.</SPAN></P><P style="line-height:90%; text-indent: -15px !important;"><SUP><SPAN style="font-size:11px;"><A href="#8" name="8num">8</A></SPAN></SUP><SPAN style="font-size:11px">Lee JS, Lee SY and Sunwon Park. Fed-batch culture of Escherichia coli W by exponential feeding of sucrose as a carbon source. Biotechnology Techniques 11:1 (1997) 59-62.</SPAN></P><P style="line-height:90%; text-indent: -15px !important;"><SUP><SPAN style="font-size:11px;"><A href="#9" name="9num">9</A></SPAN></SUP><SPAN style="font-size:11px">Shukla, VB et al. Production of D(-)-lactate from sucrose and molasses. Biotechnology Letters 26 (2004) 689-693.</SPAN></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Brown-Stanford/PowerCell/NutrientSecretion">http://2011.igem.org/Team:Brown-Stanford/PowerCell/NutrientSecretion</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Brown-Stanford/PowerCell/NutrientSecretion" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Brown-Stanford/PowerCell/NutrientSecretion" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Brown-Stanford/PowerCell/NutrientSecretion&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Brown-Stanford/PowerCell/NutrientSecretion&amp;oldid=262984" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>