<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Brown-Stanford_Lab_Notebook_Week8"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Brown-Stanford/Lab/Notebook/Week8">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Brown-Stanford/Lab/Notebook/Week8&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Brown-Stanford/Lab/Notebook/Week8&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Brown-Stanford/Lab/Notebook/Week8&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Brown-Stanford/Lab/Notebook/Week8" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Brown-Stanford/Lab/Notebook/Week8</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV style="margin-left: auto; margin-right: auto; margin-top:-15px; z-index:-10; text-align: center;"><A href="/Team:Brown-Stanford" style="text-decoration: none !important;"><SPAN style="font-size: 40pt !important; line-height: 40pt !important; color: white !important; font-weight: bold !important;">Brown-StanfordiGEM</SPAN></A></DIV><DIV id="header" style="height:100px;padding: 0px 0px 0px 0px; margin-bottom:0px"><UL id="sdt_menu" class="sdt_menu"><LI><A href="/Team:Brown-Stanford/REGObricks/Introduction"><SPAN class="sdt_wrap"><SPAN class="sdt_link">REGObricks</SPAN></SPAN></A><DIV class="sdt_box"><A href="/Team:Brown-Stanford/REGObricks/Introduction">Intro</A><D>---------------------</D><A href="/Team:Brown-Stanford/REGObricks/ISRU">ISRU</A><D>---------------------</D><A href="/Team:Brown-Stanford/REGObricks/Biocementation">Biocement</A><D>---------------------</D><A href="/Team:Brown-Stanford/REGObricks/Characterization"><EM>S. pasteurii</EM></A><D>---------------------</D><A href="/Team:Brown-Stanford/REGObricks/Balloon">BalloonFlights</A><D>---------------------</D><A href="/Team:Brown-Stanford/REGObricks/Transforming">Transforming</A><D>---------------------</D><A href="/Team:Brown-Stanford/REGObricks/Biobrick">Biobrick<SUP>2</SUP></A></DIV></LI><LI><A href="/Team:Brown-Stanford/PowerCell/Introduction"><SPAN class="sdt_wrap"><SPAN class="sdt_link">PowerCell</SPAN></SPAN></A><DIV class="sdt_box"><A href="/Team:Brown-Stanford/PowerCell/Introduction">Intro</A><D>---------------------</D><A href="/Team:Brown-Stanford/PowerCell/Cyanobacteria">Cyanobacteria</A><D>---------------------</D><A href="/Team:Brown-Stanford/PowerCell/Background">Mars</A><D>---------------------</D><A href="/Team:Brown-Stanford/PowerCell/NutrientSecretion">Nutrients</A></DIV></LI><LI><A href="/Team:Brown-Stanford/FRETSensor/Introduction"><SPAN class="sdt_wrap"><SPAN class="sdt_link">FRETcetera</SPAN></SPAN></A><DIV class="sdt_box"><A href="/Team:Brown-Stanford/FRETSensor/Introduction">Intro</A><D>---------------------</D><A href="/Team:Brown-Stanford/FRETSensor/Biosensing">FRET etc.</A><D>---------------------</D><A href="/Team:Brown-Stanford/FRETSensor/Construct">Construct</A></DIV></LI><LI><A href="/Team:Brown-Stanford/Team"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Lab</SPAN></SPAN></A><DIV class="sdt_box" style="height:60px"><A href="/Team:Brown-Stanford/Team">Team</A><D>---------------------</D><A href="/Team:Brown-Stanford/Lab/Parts">Parts</A><D>---------------------</D><A href="/Team:Brown-Stanford/Lab/Notebook">Notebook</A><D>---------------------</D><A href="/Team:Brown-Stanford/Lab/Protocols">Protocols</A><D>---------------------</D><A href="/Team:Brown-Stanford/Lab/Safety">Safety</A></DIV></LI><LI><A href="/Team:Brown-Stanford/Collaboration"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Partners</SPAN></SPAN></A><DIV class="sdt_box"><A href="/Team:Brown-Stanford/Collaboration">Collab</A><D>---------------------</D><A href="/Team:Brown-Stanford/Sponsors">Sponsors</A></DIV></LI><LI><A href="/Team:Brown-Stanford/Outreach"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Outreach</SPAN></SPAN></A><DIV class="sdt_box"><A href="/Team:Brown-Stanford/SB5">SB5.0</A><D>---------------------</D><A href="/Team:Brown-Stanford/NASA">NASA</A><D>---------------------</D><A href="/Team:Brown-Stanford/Lunar">Lunar Science</A><D>---------------------</D><A href="/Team:Brown-Stanford/MakerFaire">Maker Faire</A><D>---------------------</D><A href="/Team:Brown-Stanford/BBC">BBC</A><D>---------------------</D><A href="http://openwetware.org/wiki/IGEM_Outreach" target="_blank">CBricks</A><D>---------------------</D><A href="JavaScript:ethics('/Team:Brown-Stanford/SynEthics/Intro');">Ethics</A><D>---------------------</D><A href="/Team:Brown-Stanford/RegionalJamboree">RegionalJamboree</A></DIV></LI></UL></DIV><DIV id="subHeader"><UL id="subHeaderList"><LI><A href="/Team:Brown-Stanford/Team">Team</A></LI><LI><A href="/Team:Brown-Stanford/Lab/Parts">Parts</A></LI><LI id="active"><A href="#" id="current">Notebook</A></LI><LI><A href="/Team:Brown-Stanford/Lab/Protocols">Protocols</A></LI><LI><A href="/Team:Brown-Stanford/Lab/Safety">Safety</A></LI></UL></DIV><DIV class="earth" style="text-align: left; padding: 5px;"><A href="/" style="font-size: 20px; line-height: 20px; color: white; font-weight: bold; text-decoration: none;">Back toiGEM HQ</A></DIV><DIV class="pageContent"><DIV id="notebook"><H2><SPAN class="mw-headline" id="August_1.2C_2011"><B> August 1, 2011</B></SPAN></H2><H2><SPAN class="mw-headline" id="PowerCell">PowerCell</SPAN></H2><UL><LI>Picked colony of pDS4101 x pRL25 to liquid culture.  
</LI><LI>Gibson of Ana med and Nos med (the cscB ttsp constructs!  Maybe saw faint bands after loading 5ul of Gibson product
</LI><LI>Transformed 7.5ul of Gibson product into TOP10; plated on LB chlor plates (100ul on one plate, 200ul on another) at 7pm
</LI></UL><TABLE><TBODY><TR><TD>AnaM-RBSm</TD><TD>cscB-cscB</TD><TD>TTSP-pSB1C3 (oops--Taq products!)</TD></TR><TR><TD>1ul</TD><TD>0.2ul</TD><TD>1.2ul</TD><TD>1.6ul</TD></TR><TR><TD>NosM-RBSm</TD><TD>cscB-cscB</TD><TD>TTSP-pSB1C3 (oops--Taq products!)</TD></TR><TR><TD>1.4ul</TD><TD>0.2ul</TD><TD>1.2ul</TD><TD>1.6ul</TD></TR></TBODY></TABLE><UL><LI>used assembly master mix and protocol described in “Gibson protocol-openwetware” file
</LI><LI>15ul of master mix with above part amounts incubated at 50?C for 1 hour.
</LI><LI>Ran gel of backbone amplification, saw nothing
</LI><LI>Plasmid was at 5x concentration, 125ng/ul!!!  add more water and try again.
</LI><LI>Innoculate liquid Nostoc cultures (2x 20ml cultures)
</LI><LI>Made LB+agar+chloramphenicol plates
</LI><LI>Redo PCR of plasmid backbonds pSB1A2, pSB1C3:
</LI><LI>gradient from 40-50C
</LI></UL><LI>imaged (CAUTION: the key for lane/samples may be reversed; ie pSB1A2 and pSB1C3 may be flipped; the description in the lab notebook is 75% certain)
</LI><P>gel extracted what is likely pSB1C3 (heavy band between 2000 and 2500bp ladder); results: 25ng/ul
</P><H2><SPAN class="mw-headline" id="August_1.2C_2011_2"><B> August 1, 2011</B></SPAN></H2><H3><SPAN class="mw-headline" id="REGOBricks">REGOBricks</SPAN></H3><UL><LI>First Transformation of S. pasteurii: 
</LI><LI>Materials: 
</LI><LI>Cells (frozen cryostock of 1 mL: suspended in 18% glycerol and 1X Spizzen’s)
</LI><LI>SMMP (+BSA)
</LI><LI>SMMP
</LI><LI>Lysozyme
</LI><LI>screw-top test
</LI><LI>centrifuge
</LI><LI>40% PEG
</LI><LI>2xSSM
</LI><LI>5 ug of transforming DNA, for each 1 mL aliquot of cells
</LI><LI>vy-R5 regeneration plate
</LI><LI>vy-R5 regeneration plate + Kan 
</LI></UL><UL><LI>Procedure: 
</LI></UL><OL><LI> Cell Preparation: 
</LI><LI>Day 1: Streak strain to be protoplasted and incubate at 25 C. 
</LI><LI>Day 2: 
</LI><LI>Different Day prep: 
</LI><LI>inoculate 200 mL of MM2 (subtilis), Bang (pasteurii) in a 2 L flask. Incubate at 37, until OD reading of 0.2. Harvest at 4 degrees, 8000g (rcf) for 10 minutes. Wash cells with sterile distilled water. Resuspend cells in 6 mL of 1x Spizizen with 18% glycerol and freeze in 1 mL aliquots. Store at -80 C at least overnight. 
</LI></OL><LI>Same Day prep: 
</LI><LI>inoculate 30 mL of MM2 (subtilis) or Bang (S. past) with cells from an overnight culture in a 300 mL. Incubate at 37 until an OD of 0.2. Harvest cells and resuspend in 4.5 mL of SSMP + BSA. 
</LI><LI>Day 3: 
</LI><LI>LYSOZYME
</LI><LI>Thaw a 1 mL aliquot, added to 3.5 mL of SSMP (+BSA) in a 125 mL flask
</LI><LI>Add 0.5 mL of 5 mg/ml lysozyme dissolved in SMMP + BSA (final concentration 0.5 mg/ml)
</LI><LI>(Wild type strains may require 10 ug/ml of lysozme)
</LI><LI>(if cells are in lysozyme &gt;1 h, regeneration of cells is difficult)
</LI><LI>Incubate cells at 37 C and 75 rpm: remove a sample of 10 ul every 15 minutes, examine microscopically for extent of protoplastation
</LI><LI>TRANSFORMATION
</LI><LI>prepare protoplast suspension:
</LI><LI>transfer suspension to sterile 13 x 100 mm screw top test tube
</LI><LI>harvest cells by centrifuge (2580 g for 15 minutes) [clean out lysozymes]
</LI><LI>Decant, wash cells GENTLY with 5 mL of SSMP [wash out cells]
</LI><LI>Centrifuge at 2580 for 10 minutes
</LI><LI>Resuspend protoplast GENTLY with 1 mL SSMP
</LI><LI>DNA
</LI><LI>Add 5 ug of transforming DNA + equal volume 2xSSM to screwtop tube
</LI><LI>Add 0.5 mL of protoplast suspension
</LI><LI>IMMEDIATELY add 1.5 mL of 40% PEG (have another 1000 ul pipette ready in the hood, set to 750 ul, pipette twice)
</LI><LI>Invert solution for a minute
</LI><LI>add 5 ml of SSMP + BSA
</LI><LI>Mix gently and centrifuge*** for 5 minutes. Speed: 
</LI><LI>40% PEG very viscous, makes pelleting by centrifuge difficult
</LI><OL><LI>Tried 5 mins, 2580 - FAIL
</LI><LI>Tried 5 mins, 3000 - FAIL
</LI><LI>Tried 5 mins, 4000 - FAIL
</LI></OL><LI>Tried 15 mins, 2580 - FAIL
</LI><LI>Tried 5 mins, 4500 - FAIL
</LI><LI>Tried 10 minutes, 4800- FAIL
</LI><LI>Resuspend pellet in 1 mL of SMMP + BSA
</LI><LI>incubate at 30 C for 90 minutes to allow for expression
</LI><LI>Recentrifuge and suspend in 1 mL SMMP + BSA
</LI><UL><LI>Plating/Dilution 
</LI><LI>Set up 5 test tubes- 
</LI></UL><UL><LI>Plate onto Regeneration plates 
</LI><LI>Plate onto transformation plates
</LI></UL><UL><LI>Possible selection of variation
</LI></UL><OL><LI>Concentration of lysozyme (but it can be standardized by equal concentration of protoplasts as seen under the microscrope: According to Ryan, protoplast cells transform from rod shaped to spherical shape as they undergo protoplastation)
</LI><LI>centrifuge speed- should redo at 2580 rcf, maybe sample protoplast under microscope to see extent of cell survival
</LI></OL><UL><LI>ELECTROPORATION 
</LI><LI>Dilute overnight culture 10 fold in neutral complex media (maybe just use BANG?)
</LI><LI>Add glycine to a final concentration of 1% and grow for 1h a 37?C (Glycine can be varied)
</LI><LI>Cool cells on ice for 10min
</LI><LI>Centrifuge @ 4?C 5000xg for 5 min to collect cells
</LI><LI>Wash 4x with electroporation media (wash-spin-wash-spin…)
</LI><LI>Resuspend in electroporation medium
</LI><LI>Can stop at this step and freeze @ -80?C
</LI><LI>For electroporation, mix 60ul of cells with 2.5ul DNA and transfer to ice cold electro-cuvette
</LI><LI>Incubate on ice water for 10 min
</LI><LI>Expose cuvette to single pulse using Gene-PulserII set at 25uF and 200 ohms
</LI><LI>Immediately after discharge, add 1ml of recovery medium to cells and shake at 37?C for 3h
</LI><LI>Plate on BANG and BANG Kan plates
</LI></UL><UL><LI>Making Bricks: 
</LI></UL><UL><LI>Process: 
</LI><LI>Grow up S. pasteurii
</LI><LI>Spin them down, 
</LI><LI>Resuspend them in three 
</LI><LI>Insert into column
</LI><LI>Insert CaCl2 
</LI><LI>Wait- 7 days
</LI></UL><UL><LI>Variations: 
</LI><LI>Vary cells (S. past 11859 vs Dosier)
</LI><LI>Vary concentrations of CaCl2, urea (1:1 M, 2:1 M; 2:2 M; 1:3) 
</LI><LI>Vary resuspension
</LI><LI>Vary concentration of cells
</LI><LI>How much cell we are putting in (volume)
</LI></UL><H2><SPAN class="mw-headline" id="August_2.2C_2011"><B> August 2, 2011</B></SPAN></H2><H3><SPAN class="mw-headline" id="PowerCell_2">PowerCell</SPAN></H3><UL><LI>Check on plated Gibson products: no transformants :(
</LI><LI>Prepped all of our Gibson bits for sequencing
</LI><LI>cryostocked pDS4101 x pRL25
</LI><LI>prepare transformation of just pRL25, pDS4101, pRL443, Anabaena with serial dilutions.
</LI><LI>made plates (BG11+kan, BG11+LB, BG11) and E. coli liquid cultures (afternoon)
</LI><LI>liquid cultures of plasmid strains
</LI><LI>Gibson attempt #2, Evan. 
</LI></UL><TABLE><TBODY><TR><TD>AnaM-RBSm</TD><TD>cscB-cscB</TD><TD>TTSP-pSB1C3 from Registry</TD></TR><TR><TD>1ul</TD><TD>0.2ul</TD><TD>1.2ul</TD><TD>1.75ul</TD></TR><TR><TD>AnaM-RBSm</TD><TD>cscB-cscB</TD><TD>TTSP-pSB1C3 from PCR (from 8/1)</TD></TR><TR><TD>1ul</TD><TD>0.2ul</TD><TD>1.2ul</TD><TD>1.75ul</TD></TR><TR><TD>NosM-RBSm</TD><TD>cscB-cscB</TD><TD>TTSP-pSB1C3 from Registry</TD></TR><TR><TD>1.4ul</TD><TD>0.2ul</TD><TD>1.2ul</TD><TD>1.75ul</TD></TR><TR><TD>NosM-RBSm</TD><TD>cscB-cscB</TD><TD>TTSP-pSB1C3 from PCR (from 8/1)</TD></TR><TR><TD>1.4ul</TD><TD>0.2ul</TD><TD>1.2ul</TD><TD>1.75ul</TD></TR></TBODY></TABLE><UL><LI>Incubation begun @ 11:20, take out @ 12:20.
</LI><LI>ran 5ul of Gibson products on gel; saw a collage of various band lengths (Lane 1: Ana Med w/ registry backbone | Lane 2: Ana M w/ PCR backbone | Lane 3: Nos M w/ reg backbone | Lane 4: Nos M w/ PCR backbone)
</LI><LI>Transformed 7.5ul of Gibson product into TOP10s, plated on chlor plates
</LI><LI>We are running out of Phusion-made cscB|TTSP! Maybe 8 uL left.
</LI></UL><H2><SPAN class="mw-headline" id="August_3.2C_2011"><B> August 3, 2011</B></SPAN></H2><H3><SPAN class="mw-headline" id="PowerCell_3">PowerCell</SPAN></H3><UL><LI>Transformation
</LI><LI>transformation of just pRL25, pDS4101, pRL443, Anabaena with serial dilutions.
</LI><LI>balls!  Kanamycin won't select for cyanobacteria, so neomycin is necessary.  ordered overnighted, proceeding in the meantime.  Neomycin plates are only necessary for selection of exconjugates 1-2 days into the process, so hopefully it’ll be here in time.
</LI></UL><LI>Due to time constraints and demand for the hood, the plate mating protocol was followed, rather than spot mating.  The nitrocellulose was wet.
</LI><LI>autoclaved nitrocellulose slips for transformation filters
</LI><LI>Found 1 colony from our Gibson Attempt (#2) Aug3 transformants, on the Anabaena-PCR-backbone plate! Innoculated liquid cultures in preparation for mini prep.
</LI><LI>left at room temperature until morning of 8/4, oops
</LI><P>Trying the transformation again with greater cells to DNA ratio. All 4 backbone/species combinations.
</P><UL><LI>PCR: cscB|TTsp, cscB|GFP linker, GFP ttsp, NosS
</LI><LI>left to run overnight
</LI><LI>Began to prepare materials for cyanos on balloon flight
</LI></UL><H2><SPAN class="mw-headline" id="FRETSensor">FRETSensor</SPAN></H2><UL><LI>New UV directed evolution testing:
</LI><LI>Grow puc18 cells in 25 mL LB
</LI><LI>Zap at 1 min, 5 min, 10 min in:
</LI><LI>1 mL LB exposed (a)
</LI><LI>1 mL PBS exposed (b)
</LI><LI>plated on LB amp plates
</LI></UL><LI>Plate (a) and (b)
</LI><LI>Zap amp plates 10 min, inoculate with top10, to see if it degrades
</LI><LI>Made competent E. coli W with CT2, transformed with RFP
</LI><H2><SPAN class="mw-headline" id="August_4.2C_2011"><B> August 4, 2011</B></SPAN></H2><H3><SPAN class="mw-headline" id="PowerCell_4">PowerCell</SPAN></H3><UL><LI>Transformation of just pRL25xpDS4101, pRL25, pDS4101, pRL443, Anabaena and Nostoc with Elhai’s plate transformation protocol.  Incubating in 30?.
</LI><LI>Found weird mystery red colony on one of the Aug2 gibson attempt plates - took microscope pictures, they look like E. coli. Could they be cross-contamination from Lei and Jovian’s RFP strains? Those shouldn’t grow on our chlor plates...
</LI><LI>Going to pick colonies for liquid culture, mini-prep anyway, possible sequencing anyway.
</LI><LI>Also pick 4 more colonies from non-suspicious colony and grow up for liquid culture/miniprep, because we forgot to put the liquid cultures from yesterday in the incubator, they may not grow.... 
</LI><LI>cultures from 8/3 looking more promising
</LI><LI>Ran gels on PCR from 8/3 PM
</LI><LI>cscB|TTsp and cscB|GFP fail
</LI><LI>lanes cut and saved from GFP|TTsp and Nosstr
</LI><LI>Continue to prepare cyanos for balloon flight.
</LI><LI>redesign primers for bio brick assembly approach as alternative to gibson (Primers ordered from Elim! Melting temps between 55-57C)
</LI></UL><H2><SPAN class="mw-headline" id="FRETSensor_2">FRETSensor</SPAN></H2><UL><LI>Discovered that ampicillin does not degrade after 10 min of UV!
</LI><LI>LB media shields cells
</LI><LI>Plating directly onto LB amp plates seems to work best/easiest. No E. coli survived even  1 min.
</LI><LI>Plating from PBS, no e coli survived 1 min of UV
</LI><LI>Made liquid cultures of E. coli W w/ RFP for plasmid prep and replating
</LI><LI>GeneArt order #2 came in (where #1??) with the C. cellulolyticum Doc-YFP and C. cellulolyticum YFP-Doc. Unfortunately, they do not work with Chad’s rosettazyme, but we will try transforming and plasmid prepping today.
</LI><LI>RFP successfully transformed into E. coli W, plated on to BG11 sucrose plates with ampicillin resistance (5 mM, 10mM, 30mM, 60mM, 80mM, in increments of 50uL, 100uL, and 200uL)
</LI><LI>Spread some of the gel stab of the cohesin-cohesin construct on an LB amp plate
</LI></UL><H2><SPAN class="mw-headline" id="August_5.2C_2011"><B> August 5, 2011</B></SPAN></H2><H3><SPAN class="mw-headline" id="PowerCell_5">PowerCell</SPAN></H3><UL><LI>Made BG11 + 1% agar + 50ug/ml neomycin plates
</LI><LI>Transferred conjugation filter from BG11+5%LB+1% agar plates to above neo plates.
</LI><LI>began construct design for synthesis.  uploaded to construct design folder.
</LI><LI>Miniprep Gibson colonies
</LI><LI>worked on balloon experiment prep 
</LI><LI>Chucked the first and second cyanobacteria transformation attempts.
</LI><LI>no cyanobacterial growth
</LI><LI>appearance of conjugative parents, numerous e. coli colonies
</LI><LI>Prepped Cyanos for balloon flight.
</LI><LI>Took hemocytometer/OD readings of Nos, Ana so we know how many cells we flew
</LI><LI>Anabeana: 1.4e6 cells/mL = .265 OD -&gt; added 10 ul cells to nitrocellulose
</LI><LI>Nostoc: 1.1e7 cells / mL (can’t OD, too clumpy) -&gt; added 10 ul cells to nitrocellulose
</LI></UL><H2><SPAN class="mw-headline" id="August_5.2C_2011_2"><B> August 5, 2011</B></SPAN></H2><H3><SPAN class="mw-headline" id="REGOBricks_2">REGOBricks</SPAN></H3><UL><LI>To Do List for friday 8/5/11
</LI></UL><OL><LI>Look at ecoli transformation in 37C (no transformation - repeat)
</LI><LI>If there are colonies, inoculate LB amp liquid culture to harvest puBC3 
</LI><LI>Look at s. pasteurii in 30C to see if recovery KAN plates actually have KAN (Did not grow, so plates are still okay? Will make new plates)
</LI><LI>growth = no KAN (either way probably make some new plates)
</LI></OL><LI>Make new recovery plates, w and w/o KAN
</LI><LI>Find two tubes in 37 C, in a white tub on the lower level with some culture tubes there are two blue 1.5mL tubes labeled 1 and 2 (I know helpful right?), these are recovering from the 
</LI><P>protoplast/electroporation protocol (its like these cells got their skin ripped off and then were electrocuted ha) They need to be plated on recovery plates both antibiotic and non
</P><OL><LI>Make new recovery plates, w and w/o KAN
</LI><LI>re-do protoplast transformation with good plates?
</LI><LI>Be awesome.
</LI></OL><H2><SPAN class="mw-headline" id="FRETSensor_3">FRETSensor</SPAN></H2><UL><LI>GeneArt order #1 came in!!
</LI><LI>E. coli W has not grown on sucrose plates
</LI><LI>Gel stab overgrown
</LI></UL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Brown-Stanford/Lab/Notebook/Week8">http://2011.igem.org/Team:Brown-Stanford/Lab/Notebook/Week8</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Brown-Stanford/Lab/Notebook/Week8" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Brown-Stanford/Lab/Notebook/Week8" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Brown-Stanford/Lab/Notebook/Week8&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Brown-Stanford/Lab/Notebook/Week8&amp;oldid=192753" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>