<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_CU-Boulder_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:CU-Boulder/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:CU-Boulder/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:CU-Boulder/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:CU-Boulder/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:CU-Boulder/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:CU-Boulder/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="menu-igem"><UL id="menu" class="sf-menu"><LI><A style="border-left:0px;" href="https://2012.igem.org/Team:CU-Boulder">Home</A></LI><LI><A href="https://2012.igem.org/Team:CU-Boulder/Team">Team</A></LI><LI><A href="https://2012.igem.org/Team:CU-Boulder/Team#theplayers">Students</A></LI><LI><A href="https://2012.igem.org/Team:CU-Boulder/Team#theadvisors">Advisors</A></LI><LI><A href="https://2012.igem.org/Team:CU-Boulder/Team#thepics">Photos</A></LI><LI><A href="https://2012.igem.org/Team:CU-Boulder/Team#contactus">Contact</A></LI></UL><LI><A href="https://2012.igem.org/Team:CU-Boulder/Project">Project</A></LI><LI><A href="https://2012.igem.org/Team:CU-Boulder/Project/Brainstorming">Brainstorming</A></LI><LI><A href="#">Characterization of <I>Vibrio fischeri</I></A></LI><LI><A href="https://2012.igem.org/Team:CU-Boulder/Project#bacterialnightlight">Project 1: Background</A></LI><LI><A href="https://2012.igem.org/Team:CU-Boulder/Project#experiments1">Project 1: Experiments</A></LI><LI><A href="https://2012.igem.org/Team:CU-Boulder/Project#results1">Project 1: Results</A></LI><LI><A href="#">Biofilm Prevention</A></LI><LI><A href="https://2012.igem.org/Team:CU-Boulder/Project#antibiofilm">Project 2: Background</A></LI><LI><A href="https://2012.igem.org/Team:CU-Boulder/Project#experiments2">Project 2: Experiments</A></LI><LI><A href="https://2012.igem.org/Team:CU-Boulder/Project#results2">Project 2: Results</A></LI><LI><A href="https://2012.igem.org/Team:CU-Boulder/Project#applications">Applications</A></LI><LI><A href="https://2012.igem.org/Team:CU-Boulder/Project#medalconsideration">Medal Consideration</A></LI><LI><A href="https://2012.igem.org/Team:CU-Boulder/Parts">Parts</A></LI><LI><A href="https://2012.igem.org/Team:CU-Boulder/Parts">Parts Used</A></LI><LI><A href="https://2012.igem.org/Team:CU-Boulder/Parts">Parts Submited</A></LI><LI><A href="https://2012.igem.org/Team:CU-Boulder/Modeling">Model</A></LI><LI><A href="#">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:CU-Boulder/Notebook"> Calendar </A></LI><UL><LI><A href="https://2012.igem.org/Team:CU-Boulder/Notebook/Protocols">Protocols</A></LI></UL><LI><A href="https://2012.igem.org/Team:CU-Boulder/Safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:CU-Boulder/Attributions">Attributions</A></LI></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Project_1:_Model_System_for_studying_quorum_activation_of_pathogenicity"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Project 1: Model System for studying quorum activation of pathogenicity</SPAN></A></LI><LI class="toclevel-2"><A href="#Project_1:_Experiments"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Project 1: Experiments</SPAN></A></LI><LI class="toclevel-2"><A href="#Project_1:_Results"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Project 1: Results</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#Project_2:_Biofilm_Prevention"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Project 2: Biofilm Prevention</SPAN></A></LI><LI class="toclevel-2"><A href="#Project_2:_Experiments"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Project 2: Experiments</SPAN></A></LI><LI class="toclevel-2"><A href="#Project_2:_Results"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Project 2: Results</SPAN></A></LI><LI class="toclevel-1"><A href="#Applications"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Applications</SPAN></A></LI><LI class="toclevel-1"><A href="#Medal_Consideration"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Medal Consideration</SPAN></A></LI><LI class="toclevel-2"><A href="#Bronze"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Bronze</SPAN></A></LI><LI class="toclevel-2"><A href="#Silver"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Silver</SPAN></A></LI><LI class="toclevel-2"><A href="#Gold"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Gold</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Project_1:_Model_System_for_studying_quorum_activation_of_pathogenicity"><P style="text-align: center;">Project 1: Model System for studying quorum activation of pathogenicity</P></SPAN></H1><P>
Quorum sensing is a system of bacterial communication that coordinates gene expression based on population density. Bacteria secrete a signaling molecule, such as N-Acyl Homoserine Lactones (AHL), that binds to specific extracellular receptors on neighboring bacteria to activate transcription factors.  As the bacterial population increases, the signaling molecule reaches a certain threshold that initiates a positive-feedback loop and causes all members of the population to activate certain genes at roughly the same time.  Many pathogenic bacteria use quorum sensing to activate their virulent factors, as it is not energetically favorable for a single bacterium to release its virulent factors until it is in a large enough population to infect a host.  In our project, the CU-Boulder iGEM team aims to disrupt quorum sensing in order to reduce pathogenicity of otherwise virulent bacteria. </P><P>It has been well documented in research that bacteria use quorum sensing signals to activate pathogenic factors. The gram-negative bacteria <I>P. aeruginosa</I> synthesizes AHLs as a bacterial density indicator, and upon reaching a specific threshold, pathogenic factors are activated (Iglewski et al., 2000). <I>P. aeruginosa</I> is a common opportunistic infection that occurs most frequently in immune-compromised patients (Iglewski et al., 2000).  This quorum sensing mechanism for activation of pathogenic traits has also been demonstrated in <I>Salmonella, E. coli, and Y. pestis</I> (Choi et al., 2007). </P><P>Additionally, quorum sensing factors play a role in activating bacterial responses such as biofilm production and food rot.  Biofilm production can contribute to pathogenicity of bacteria by the formation of safe and nutrient-rich havens for bacteria populations to grow.   Biofilms are made up of a hydrated matrix of polypeptides, polysaccharides, and nucleic acids that serve as a protective barrier and microenvironment for microbes (Nijland et al., 2010).  In order to study how to inhibit these pathogenic effects of quorum sensing, we utilized a system that would respond to quorum signals and was safe to work with, as an alternative to pathogenic bacteria. Therefore, we characterized the model quorum sensing system of <I>Aliivibrio fischeri</I> (VF), which luminesces in response to AHLs.</P><P>VF is a gram-negative bacteria that is a biological model for both quorum sensing and bioluminescence research. This rod-shaped bacteria participates in symbiotic relationships with various marine organisms, most notably the bobtail squid. The squid hosts the VF in an internal light organ and utilizes the light emitted by the bacteria to hide its shadow in shallow waters, therefore avoiding potential predators.  We decided to use VF WH1 strain as our model organism because it has a LuxI/LuxR quorum sensing system that it uses to detect the concentration of AHLs and it responds with luminescence at a visible wavelength when the threshold level of AHLs is reached. </P><P>When LuxR detects AHLs, it activates the rest of the Lux operon. The Lux operon (below) consists of five essential genes: LuxA, LuxB, LuxC, LuxD, and LuxE.
</P><P>LuxC, LuxD, and Lux E form the fatty acid reductase enzyme complex that synthesizes fatty acid aldehydes with acyl-CoA as the starting substrate. The three gene products form a complex conglomerate (below) to increase kinetic efficiency. 
</P><P>LuxA and LuxB form a dimer that reduces O<SUB>2</SUB> while oxidizing the aldehyde and FMNH<SUB>2</SUB>. This oxidation produces the blue-green light (~490nm). Another representation of the overall reaction is shown below:</P><P>
Additionally, it has been shown that the LuxG gene product increases brightness by functioning as a flavin reductase (Nijvipakul et al., 2008).  This enzyme is not essential for the luminous phenotype but we determined that it would be a useful addition to the registry because it recycles the FMN substrate, therefore generating more FMNH<SUB>2</SUB> ready to be oxidized with the fatty aldehyde and increasing light output.  </P><P>
We utilized VF because the luminosity is easy to measure with a high degree of sensitivity and low background noise. The organism is luminous at a wavelength that can be perceived by the human eye, and we can therefore use the organism as a model system for reporting when pathogenic factors would have been initiated in other bacteria.  Using a plate reader, we characterized at what OD the VF starts to glow, and characterized how the density of the organism affected the luminosity.
</P><H3><SPAN class="mw-headline" id="Project_1:_Experiments"><P style="text-align: center;">Project 1: Experiments</P></SPAN></H3><P>
1.  <B>Vibrio fischeri as a model system</B>: Characterize VF as a model system for quorum sensing and luminescence via plate reader experiments. 
2.  <B>Assemble the LuxABCDE operon + LuxG</B>: The Cambridge 2010 &quot;E. Glowi&quot; team submitted the Lux operon to the registry, BBa_K325905, but several attempts at transforming <I>E. coli</I> with this part failed either due to its large size (6478bp) or due to an inaccurate registry submission. Several other teams have tried to use their part since 2010 but none have been successful. For this reason, our team is BioBricking each Lux gene individually and submitting them to the registry. 
&lt;a name=&quot;results1&quot;&gt;&lt;/a&gt;&lt;/html&gt;</P><H3><SPAN class="mw-headline" id="Project_1:_Results"><P style="text-align: center;">Project 1: Results</P></SPAN></H3><P>The figure above shows the plate reader data characterizing <I>Aliivibrio fisheri's</I> luminosity at a specific OD. The luminosity starts to increase at OD .23, and then after the A. fisheri becomes stationary, the luminescence sharply decreases. This experiment was read on a plate reader every 20min with 5 trials. A. fisheri was grown up for 4 hours in a room temperature shaker. Then pelleted and washed three times with LB + NaCl. Diluted to an OD of .05 and placed in the plate reader.</P><P>Additionally, LuxC,D,E, and G were isolated from WH1 VF and standardized to biobrick format.  LuxA and B were isolated from VF WH1 and contained internal EcoRI/PstI cut sites.  We used site directed mutagenesis to modify these sites in order to obtain biobrick format, but had a difficult time getting the mutagenesis to work.  

</P><H1><SPAN class="mw-headline" id="Project_2:_Biofilm_Prevention"><P style="text-align: center;">Project 2: Biofilm Prevention</P></SPAN></H1><P><B>Quorum Sensing in LuxR containing Bacteria</B></P><P>This year the CU iGEM team has harnessed the quorum sensing factors endogenous in the <I>Salmonella enterica serovar typhimurium</I> LT2 strain to detect AHLs produced by other bacteria. Neither <I>Salmonella</I> nor <I>E. coli</I> have been shown to produce detectable levels of AHLs, so they were model organisms for the detection of other AHL producing bacteria such as VF, or pathogenic bacteria such as <I>Yersinia pestis</I>. AHLs are small molecules synthesized by most gram negative bacteria, and are able to freely diffuse throughout the membrane. When the concentration of AHL producing bacteria in a specific area increases, the total concentration of AHLs diffusing into the cytoplasm also increases. Once a threshold level of AHLs are in the cytosol, transcription factors like the VF LuxR or <I>Salmonella enterica</I> SdiA activate gene synthesis for quorum factors to be produced. In pathogenic bacteria, these signals have been shown to activate transcription of pathogenic factors.
</P><P><B>Detection and silencing of AHL producing bacteria</B></P><P>The SdiA transcription factor has been shown to be more sensitive to lower concentrations and a greater diversity of AHLs than its LuxR homolog. We took advantage of this extra-sensitivity to create a detection system for AHL producing bacteria, while additionally attaching the detection system to the secretion of potent AHLases such as the Aiia enzyme, and a reporter protein RFP. Using this construct we tested whether we could disrupt the quorum sensing signal AHLs in order to keep them from reaching the threshold concentration to activate the less sensitive LuxR receptor. The VF species and other pathogenic bacteria use the LuxR receptor, therefore inhibition of AHLs would keep VF from producing light. Keeping the VF from producing light is an analogous model to keeping pathogenic bacteria from being stimulated by AHLs to release their pathogenesis factors. 
</P><P>Inhibiting quorum sensing in bacteria has been shown to not only inhibit the transcription of pathogenesis factors, but also to keep biofilms from forming. In the history of iGEM, teams have used secreted enzymes to digest biofilms that have already been made, but our use of quorum sensing inhibitors have gone a step further, and are being tested in their ability to keep bacteria from making biofilms in the first place.
</P><P>We also identified and obtained a novel biofilm degradation enzyme, NucB.  NucB is a deoxyribonuclease synthesized by <I>Bacillus subtilis</I>, a sporulating gram-positive bacteria.  When <I>B. subtilis</I> goes through sporulation, it divides asymmetrically and generates two compartments of unequal size and different developmental fates.  The smaller compartment eventually develops into a mature spore, while the larger compartment serves to protect and nurture the developing spore (Eichenberger et al., 2003).  NucB is part of a large operon that responds to regulator molecules activated in the large compartment during sporulation.  Its task is to degrade nucleic acids present in surrounding biofilm in order for the spore to escape and travel to a new location.  DNA in biofilms is responsible for increased viscoelastic and adhesive properties.  Since our goal is to prevent biofilm formation, we wanted to express NucB as an additional method of prevention.  
</P><H3><SPAN class="mw-headline" id="Project_2:_Experiments"><P style="text-align: center;">Project 2: Experiments</P></SPAN></H3><P>1.	 <B>Test whether the SdiA pSrgE cassette is more sensitive to extracellular AHLs than the LuxR LuxpR cassette</B>:
This will be done by placing an RFP behind the cassette, and using the plate reader to determine the fluorescence at a given concentration of RFP.
</P><P>2.	<B>Use the SdiA pSrgE cassetteâs sensitivity to inhibit the LuxR LuxpR system</B>: 
Attach an AHLase and YFP to the more sensitive SdiA/pSrgE cassette. Co-culture the AHLase/YFP construct with a LuxR/LuxpR-RFP bacteria.  The culture should turn fluoresce yellow, and not turn red.
</P><P>3.	<B>Test the SdiA pSrgE cassetteâs inhibition effects with VF acting as a pathogen analogue</B>:
Co-culture the SdiA/pSrgE/YFP/Aiia cells with V. fisheri and use the plate reader to show that co-cultures will keep V. fisheri from emitting light at given ODs. 
</P><P>4.      <B>Isolate and submit NucB</B>:  Isolate NucB and standardize it to biobrick format.
</P><H3><SPAN class="mw-headline" id="Project_2:_Results"><P style="text-align: center;">Project 2: Results</P></SPAN></H3><H1><SPAN class="mw-headline" id="Applications"><P style="text-align: center;">Applications</P></SPAN></H1><P style="text-align: center;">Soil biofilm colonized around plant root system.</P><P>Biofilms are all around us, from our bathrooms to our gums and teeth. By utilizing our construct - which intercepts the bacterial signal preventing the quorum threshold from being met - the majority of biofilms would be prevented from forming. This could be utilized in a variety of different applications such as applying our construct to areas that may not receive regular cleaning. For instance applying our construct throughout a shower curtain would reduce biofilm activity and inhibit bacterial flourishing in the shower. Theoretically, stationary bodies of water prone to generating intense bacterial mats on their surface could be preemptively stopped by adding our construct.  Biofilms also form in the plants we eat, usually resulting in disease of the plant and spoilage of food. Our construct could be a marker for consumers applied to encased produce (such as bananas).  If our reporter mechanism began to glow intensely, it would be a good indicator that the food was beginning to turn bad. 
While our project has relevance and application today, if it was continued to be worked on, there is much more that could be accomplished.  Many plants have defense mechanisms against biofilm, secreting proteins that inhibit the formation of biofilm similar to our project.  If in the future we could express our construct in plants, we could produce healthier plants less prone to disease and with a longer shelf life, while at the same time limiting the amount of pesticide needed to keep them from spoiling. The plants would effectively be defending themselves from pathogenic bacterial invasion and farmers would be selling a more healthy product. 
Another future application for our project is developing the AHL recognition complex.  Our complex recognizes a broad range of AHLs because we wanted our construct to react to a broad range of bacteria.  If we could âtuneâ our receptors to recognize species-specific AHLs, we could determine bacterial environmental composition.   If we could take a broad range of reporting mechanisms all with different color outputs and have them activate when the species-specific AHL is present, then we would know exactly which bacterial species is present.  By having a cocktail of these constructs with specific reporters linked to specific AHL receptors, we could efficiently visualize a microbial environment at low cost.  This could be a useful tool diagnostically where today physicians learn so much about the body from bacterial fecal population; a bacterial population so crucial to humans that fecal transplants are undergone to re-culture humans with poor bacterial environments.  By adding this cocktail to fecal matter, it would be possible to image and observe the type of fluorescence in the feces in order to make a diagnosis. This would be especially useful to physicians in the third world who lack modern medical equipment. This goes much further than humans as well; farmers could also cheaply survey bacterial composition of their soil with this construct and assess the fertility of the land. AHLs could prove to become very important in years to come and have a real effect on our interaction with the microbial world. 
</P><H1><SPAN class="mw-headline" id="Medal_Consideration"><P style="text-align: center;">Medal Consideration</P></SPAN></H1><H3><SPAN class="mw-headline" id="Bronze"><P style="text-align: left;">Bronze</P></SPAN></H3><UL><P style="text-align: left;"> Team Registration:<I> April 11, 2012</I></P><LI>___ Complete Judging form</LI><P style="text-align: left;"> Team Wiki</P><LI> ___ Present Poster and a talk at the iGEM Jamboree:<I> North American West Regional Jamboree at Stanford University on Oct.16, 2012</I></LI><LI>___ At least one new submitted BioBrick part or a new application of an existing part with quantitative documentation</LI></UL><H3><SPAN class="mw-headline" id="Silver"><P style="text-align: left;">Silver</P></SPAN></H3><UL><LI> ___ Demonstrate that a newly submitted BioBrick part works as expected</LI><LI> ___ Characterize the operation of at least one new BioBrick Part and enter this information in the Registry</LI></UL><H3><SPAN class="mw-headline" id="Gold"><P style="text-align: left;">Gold</P></SPAN></H3><P style="text-align: left;"><B>Complete one or more of the following</B></P><UL><LI> ___ Improve the function of an existing BioBrick part and enter it's information in the registry</LI><LI> ___ Help another iGEM team by, for example, characterizing a part, debugging a construct, or modeling or simulating their system</LI><LI> ___ Outline and detail a new approach to an issue of Human Practice in synthetic biology as it relates to your project, such as safety, security, ethics, or ownership, sharing, and innovation</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:CU-Boulder/Project">http://2012.igem.org/Team:CU-Boulder/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:CU-Boulder/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:CU-Boulder/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:CU-Boulder/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:CU-Boulder/Project&amp;oldid=266935" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>