<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_MIT_project skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:MIT/project</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navdiv-wrapper"><DIV class="navdiv"><UL><LI><A href="https://2017.igem.org/Team:MIT/Team">ABOUT US</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/Team">Team</A><A href="https://2017.igem.org/Team:MIT/Attributions">Attributions</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/AS">BACKGROUND</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/AS">Alternative Splicing</A><A href="https://2017.igem.org/Team:MIT/CRISPR">CRISPR</A><A href="https://2017.igem.org/Team:MIT/RBPs">RNA Binding Proteins</A><A href="https://2017.igem.org/Team:MIT/REST">REST</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/project">PROJECT</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/GuidesandASOs">Guides and ASOs</A><A href="https://2017.igem.org/Team:MIT/mk-FF4">mKate-FF4</A><A href="https://2017.igem.org/Team:MIT/2mk-HBG">2 Exon mKate HBG</A><A href="https://2017.igem.org/Team:MIT/3mk-HBG">3 Exon mKate HBG</A><A href="https://2017.igem.org/Team:MIT/3mk-DF">3 Exon Dual Fluorescence</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/notebook">LAB WORK</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/parts">Parts/Improved Parts</A><A href="https://2017.igem.org/Team:MIT/protocols">Protocols</A><A href="https://2017.igem.org/Team:MIT/notebook">Notebook</A><A href="https://2017.igem.org/Team:MIT/Future-Work">Future Work</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/Model">MODELLING</A></LI><LI><A href="#">HUMAN PRACTICES</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/HP/Gold_Integrated">Integrated HP</A><A href="https://2017.igem.org/Team:MIT/Engagement">Public Engagement</A><A href="https://2017.igem.org/Team:MIT/Collaborations">Collaborations</A><A href="https://2017.igem.org/Team:MIT/InterLab">InterLab</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/award">AWARDS</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/HP/Silver">HP Silver</A><A href="https://2017.igem.org/Team:MIT/HP/Gold_Integrated">HP Gold</A><A href="https://2017.igem.org/Team:MIT/Engagement">Education &amp; Public Engagement</A><A href="https://2017.igem.org/Team:MIT/Model">Model</A><A href="https://2017.igem.org/Team:MIT/Part_Collection">Parts Collection</A></DIV></LI></UL></DIV></DIV><H1 style="color:#ff0000; text-align: center; font-size: 40px; line-height: 40px;">Experimental Design</H1><H2 style="color:#ff0000;">Reporters </H2><P>In order to test the functionality of our system, we needed constructs to demonstrate the level of control we had over splicing, and whether the dCas13a or Ms2 protein was doing what was expected. After some discussion, we decided that fluorescence would be the best indicator for the level of splicing. A flow cytometer can measure and quantify the level of fluorescence that's given off by a cell, and those levels can be compared among cells. We used 4 reporters to characterize the level of splicing.</P><DIV class="column half_size"><CENTER><A href="https://2017.igem.org/Team:MIT/mk-FF4"><H2 style="color:#ffffff; background-color:#ff0000; -moz-border-radius: 15px; -webkit-border-radius: 15px; padding:15px; width: 300px">mKate FF4 Reporter</H2></A></CENTER><CENTER><A href="https://2017.igem.org/Team:MIT/2mk-HBG"><H2 style="color:#ffffff; background-color:#ff0000; -moz-border-radius: 15px; -webkit-border-radius: 15px; padding:15px; width: 300px">2 Exon mKate HBG Reporter</H2></A></CENTER></DIV><DIV class="column half_size"><CENTER><A href="https://2017.igem.org/Team:MIT/3mk-HBG"><H2 style="color:#ffffff; background-color:#ff0000; -moz-border-radius: 15px; -webkit-border-radius: 15px; padding:15px; width: 300px">3 Exon mKate HBG Reporter</H2></A></CENTER><CENTER><A href="https://2017.igem.org/Team:MIT/3mk-DF"><H2 style="color:#ffffff; background-color:#ff0000; -moz-border-radius: 15px; -webkit-border-radius: 15px; padding:15px; width: 300px">3 Exon Dual Fluorescence Reporter</H2></A></CENTER></DIV><H2 style="color:#ff0000;">Choosing an Intron</H2><P>To create the “best” intron for our reporters, we needed to determine what qualities made it more likely for a sequence of nucleotides to be spliced out. In other words, we needed to define what made an intron, an intron. 

</P><P>There are many motifs characteristic to introns outlined by various sources, so we compiled the most prominent sequences mentioned. One such motif includes the <B>G triplet (GGG)</B>. This sequence pattern typically exists in clusters adjacent to the 5’ splice site of the mRNA strand, and have been correlated with splicing efficiency through another mechanism that also involves the 5’ splice site[1]. For example, in the human alpha-Globin Intron 2, there are quite a few GGG sequences that exist about 70 bases downstream of the 5’ splice site [1].

</P><P>Another intronic motif corresponds to the binding site of a <B>heterogeneous ribonucleoprotein (hnRNP)</B> that is essential to the splicing of mRNA. These proteins are primarily responsible for suppressing RNA splicing by binding to particular exons and preventing spliceosomes from attaching and splicing correctly. In a particular case, moving the binding sequence from an exon to an intron has been found to stimulate rather than inhibit splicing [2].

</P><P>Using this information, we decided to apply these sequences to a pre-existing intron, with the intent of seeing if adding known intron motifs would modify the efficiency of splicing. Our desired pre-existing intron needed to 1) be constitutively spliced and 2) contain some of the traits characteristic of many introns. The former reason is to ensure that we eliminate the potential that this intron has a tendency to be included in the final transcript. The latter reason is so we could potentially test whether removal of the motifs outlined above would have an impact on splicing. After some consideration, we settled upon the second intron of the human beta globin gene. Literature showed that intron 2 was constitutively spliced out and that intron 2 of human beta-globin enhanced the expression of chimeric genes.  [3] Our intron carried a mutation at location 654, mimicking a mutation in the literature we found conducting experiments on increasing splicing efficiency. </P><H2 style="color:#ff0000;">Experiments</H2><P> Although each experiment followed more precise optimization and procedures, they followed a similar streamline:</P><P>Each cell received a protein, guide sequence, and reporter construct. Following transfection, some cells received a DOX induction curve to introduce a time delay so that a sufficient amount of protein would be made before the reporters were. Lastly, we used flow cytometry to measure the fluorescence of each cell, and therefore characterize the splicing efficiency.</P><P>Look under the tab for specific experiments and results for a given reporter construct.</P><P><B>DOX Induction</B> works when the target gene is downstream of a TRE promoter. The DOX bind to rtTa3, which then is an activator for the TRE promoter. The gene will not be made in the absence of DOX. As levels of DOX rise, more active rtTa3 is available to activate the TRE promoter, leading to a greater output of your target gene</P><H3>References</H3><P>[1] McCullough, Andrew and Susan Berget. &quot;An Intronic Splicing Enhancer Binds U1 snRNPs To Enhance Splicing and Select 5′ Splice Sites&quot;</P><P>[2] Rebeca Martinez-Contreras, et al.&quot;Intronic Binding Sites for hnRNP A/B and hnRNP F/H Proteins Stimulate Pre-mRNA Splicing&quot;</P><P>[3] Pereverzev AP, et al. &quot;[Intron 2 of human beta-globin in 3'-untranslated region enhances expression of chimeric genes].&quot;</P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>