<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_MIT_3mk-HBG skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:MIT/3mk-HBG</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navdiv-wrapper"><DIV class="navdiv"><UL><LI><A href="https://2017.igem.org/Team:MIT/Team">ABOUT US</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/Team">Team</A><A href="https://2017.igem.org/Team:MIT/Attributions">Attributions</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/AS">BACKGROUND</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/AS">Alternative Splicing</A><A href="https://2017.igem.org/Team:MIT/CRISPR">CRISPR</A><A href="https://2017.igem.org/Team:MIT/RBPs">RNA Binding Proteins</A><A href="https://2017.igem.org/Team:MIT/REST">REST</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/project">PROJECT</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/GuidesandASOs">Guides and ASOs</A><A href="https://2017.igem.org/Team:MIT/mk-FF4">mKate-FF4</A><A href="https://2017.igem.org/Team:MIT/2mk-HBG">2 Exon mKate HBG</A><A href="https://2017.igem.org/Team:MIT/3mk-HBG">3 Exon mKate HBG</A><A href="https://2017.igem.org/Team:MIT/3mk-DF">3 Exon Dual Fluorescence</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/notebook">LAB WORK</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/parts">Parts/Improved Parts</A><A href="https://2017.igem.org/Team:MIT/protocols">Protocols</A><A href="https://2017.igem.org/Team:MIT/notebook">Notebook</A><A href="https://2017.igem.org/Team:MIT/Future-Work">Future Work</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/Model">MODELLING</A></LI><LI><A href="#">HUMAN PRACTICES</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/HP/Gold_Integrated">Integrated HP</A><A href="https://2017.igem.org/Team:MIT/Engagement">Public Engagement</A><A href="https://2017.igem.org/Team:MIT/Collaborations">Collaborations</A><A href="https://2017.igem.org/Team:MIT/InterLab">InterLab</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/award">AWARDS</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/HP/Silver">HP Silver</A><A href="https://2017.igem.org/Team:MIT/HP/Gold_Integrated">HP Gold</A><A href="https://2017.igem.org/Team:MIT/Engagement">Education &amp; Public Engagement</A><A href="https://2017.igem.org/Team:MIT/Model">Model</A><A href="https://2017.igem.org/Team:MIT/Part_Collection">Parts Collection</A></DIV></LI></UL></DIV></DIV><H1 style="color:#ff0000; text-align: center; font-size: 40px; line-height: 40px;">Experiments with 3 Exon mKate HBG Reporter</H1><P>After designing our 2-exon mKate HBG reporter, which only showed mKate fluorescent knockdown, we needed a positive reporter. We designed two 3-exon reporters, one of which is what we called the <B>3 Exon mKate HBG Reporter.</B></P><H3> 3-exon mKate Fluorescent Reporter </H3><P> The reporter was designed to contain mKate exon one, HBG Intron two, the REST-4 included exon from our <A href="https://2017.igem.org/Team:MIT/REST"> disease model,</A> HBG Intron one, and mKate exon two.</P><P>The mKate exons and the HBG Intron two were taken from the 2-exon mKate reporter. The REST-4 exon was chosen because it contains a stop codon, and because we wanted our reporter to mirror our disease model. The HBG Intron one was <A href="https://2017.igem.org/Team:MIT/project"> chosen </A> based on the literature.  A g-block was designed with the REST exon, HBG intron 2 and complementary sticky ends to be added to the 2-exon mKate HBG reporter between HBG Intron 2 and mKate exon two. </P><DIV class="column half_size"><P>If our guide system is not present in the cell, mKate exon one, the REST exon, and mKate exon two will be spliced together. During translation, the stop codon in the REST exon will cause the ribosome to fall off, resulting in an incomplete mKate and therefore a knockdown of red fluorescence. </P></DIV><DIV class="column half_size"><P>When our guide system is introduced to the cells, it will cover the 5’ splice site on HBG Intron 2, causing the spliceosome to skip over the REST exon. The cell will then be able to translate mKate as normal, and we expect to see a rise in red fluorescence. This reporter therefore achieves our goal of being a positive reporter. </P></DIV><H2>Results</H2><H3> Verifying the construct </H3><H4> mKate Titration </H4><P>Before being able to test our guides against our newly designed reporter, we needed to ensure it was exhibiting proper behavior. To that end, we ran an titration on the 3-exon mKate-HBG construct. We expected that in absence of our system, there would be no mKate fluorescence, as the stop codon in the REST exon will be included. If our system is present, we expect that levels of red output will rise with increase in reporter construct. These expected results can be seen below. The different colored lines corrispond to different <A href="https://2017.igem.org/Team:MIT/HowtoreadaCytoflowgraph"> transfection bins</A></P><P> In our experiment, we varied the amounts of 3-exon mKate-HBG from 10 to 500 ng. We also transfected Guide3 with dCas13a, and ASO2+ with Ms2 at optimized amounts from earlier experiments. (For a detailed explanation of how to plan a mammalian transfection <A href="https://static.igem.org/mediawiki/2017/6/63/Planning_Mamallian_Transfections_%C2%B7_Benchling.pdf"> click here</A>) </P><CENTER><H4>mKate Titration for 3-exon mKate-HBG</H4></CENTER><P><I>3-exon mKate-HBG reporter amounts (10ng-500ng) vs. the amount of red fluorescence (AU). The color of the line indicate the <A href="https://2017.igem.org/Team:MIT/HowtoreadaCytoflowgraph"> transfection bins</A> of each result.</I></P><P>We expected that the reporter would produce no fluorescence, but our results here show normal output levels for mKate across the reporter concentrations. Additionally, as shown below adding dCas13a and Ms2 have no effect on the red output.</P><CENTER><H4>3-exon mKate-HBG titration with ASO2+                                      3-exon mKate-HBG titration with Guide3 </H4></CENTER><CENTER><I>DOX Concentrations (5-500uM) vs. the amount of red fluorescence (AU) for ASO1 (left) and plain mKate(right). The color of the line indicate the <A href="https://2017.igem.org/Team:MIT/HowtoreadaCytoflowgraph"> transfection bins</A> of each result.</I></CENTER><STRONG>Unfortunately, we were unable to test any of our constructs against this reporter, because the reporter was producing red in the control condition.</STRONG> '

<P>Our next step is to debug our reporter. We suspect the problem may be with the intron we chose, or the way we built that intron, because the mKate-ff4 was outputting fine.</P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>