<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_MIT_alternativesplicing skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:MIT/alternativesplicing</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>Alternative splicing is a process by which exons or portions of exons or noncoding regions within a pre-mRNA transcript are differentially joined or skipped, resulting in multiple protein isoforms being encoded by a single gene.
With such a process, a single gene can result in many types of transcripts and proteins. In mammals this process is nearly ubiquitous; around 95% of human genes are alternatively spliced, lending a great deal of diversity to the human proteome []; More than 100 000 mRNA transcripts and proteins coded from less than 25 000 genes. 
However, misregulation of this process can disrupt the balance of endogenous isoforms, any many diseases have been associated with an error in the splicing machinery, primarily due to the lack of the proper levels of a certain functional protein, or the favored expression of a dysfunctional protein.
</P><P>(insert images from inkscape here)
</P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>