<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_MIT_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:MIT/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navdiv-wrapper"><DIV class="navdiv"><UL><LI><A href="https://2017.igem.org/Team:MIT/Team">ABOUT US</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/Team">Team</A><A href="https://2017.igem.org/Team:MIT/Attributions">Attributions</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/AS">BACKGROUND</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/AS">Alternative Splicing</A><A href="https://2017.igem.org/Team:MIT/CRISPR">CRISPR</A><A href="https://2017.igem.org/Team:MIT/RBPs">RNA Binding Proteins</A><A href="https://2017.igem.org/Team:MIT/REST">REST</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/project">PROJECT</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/GuidesandASOs">Guides and ASOs</A><A href="https://2017.igem.org/Team:MIT/mk-FF4">mKate-FF4</A><A href="https://2017.igem.org/Team:MIT/2mk-HBG">2 Exon mKate HBG</A><A href="https://2017.igem.org/Team:MIT/3mk-HBG">3 Exon mKate HBG</A><A href="https://2017.igem.org/Team:MIT/3mk-DF">3 Exon Dual Fluorescence</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/notebook">LAB WORK</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/parts">Parts/Improved Parts</A><A href="https://2017.igem.org/Team:MIT/protocols">Protocols</A><A href="https://2017.igem.org/Team:MIT/notebook">Notebook</A><A href="https://2017.igem.org/Team:MIT/Future-Work">Future Work</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/Model">MODELLING</A></LI><LI><A href="#">HUMAN PRACTICES</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/HP/Gold_Integrated">Integrated HP</A><A href="https://2017.igem.org/Team:MIT/Engagement">Public Engagement</A><A href="https://2017.igem.org/Team:MIT/Collaborations">Collaborations</A><A href="https://2017.igem.org/Team:MIT/InterLab">InterLab</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/award">AWARDS</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/HP/Silver">HP Silver</A><A href="https://2017.igem.org/Team:MIT/HP/Gold_Integrated">HP Gold</A><A href="https://2017.igem.org/Team:MIT/Engagement">Education &amp; Public Engagement</A><A href="https://2017.igem.org/Team:MIT/Model">Model</A><A href="https://2017.igem.org/Team:MIT/Part_Collection">Parts Collection</A></DIV></LI></UL></DIV></DIV><H1 style="color:#ff0000; text-align: center; font-size: 40px; line-height: 40px;">SpliceMIT – Splice Modelling Intronic Technology</H1><P> SpliceMIT is a tool to generate antisense oligonucleotides (ASO) for a given DNA/RNA sequence, and then analyze and output the most effective ASOs. This tool could be used for a variety of scenarios. For our test samples, SpliceMIT was used to generate and analyze the gRNA sequences (complementary to the region on the intron) in the CRISPR – dCas13a system to control alternative splicing by targeting mRNA sequences. </P><P> We used this program to generate some of our gRNA sequences.</P><CENTER><H4>This page briefly discusses the important concepts associated with our model. For more information, see our <A href="https://static.igem.org/mediawiki/2017/1/11/SpliceMIT_-_Documentation1.pdf">SpliceMIT Documentation</A></H4></CENTER><CENTER><H4> Check out the program on<A href="https://github.com/MITiGEM2017/SpliceMIT"> Github!</A></H4></CENTER><H2 style="color: #ff0000;">Overview</H2><P>SpliceMIT is a tool to generate antisense oligonucleotides (ASO) for a given DNA/RNA sequence, and then analyze and output the most effective ASOs. This tool could be used for a variety of scenarios. For our test samples, SpliceMIT was used to generate and analyze the gRNA sequences (complementary to the region on the intron) in the CRISPR – dCas13a system to control alternative splicing by targeting mRNA sequences.</P><H2 style="color: #ff0000;">Mechanism</H2><P>To compute and obtain the most effective antisense oligonucleotides, we considered four factors:</P><UL><LI>1. GC content of the ASO</LI><LI>2. Off-target binding</LI><LI>3. RNA-binding proteins competition on the ASO binding site</LI><LI>4. Secondary structure of the ASO</LI></UL><P>The general process is:</P><UL><LI>1. Add additional features if needed (e.g. Add pre-gRNA secondary structure analysis part if the goal is to find gRNAs instead of ASOs)</LI><LI>2. Ask user inputs, including DNA/RNA sequence, preferred ASO length, etc.</LI><LI>3. Run the calculations that require online platform first.</LI><LI>4. Store all data locally.</LI><LI>5. Analyze all local data.</LI><LI>6. Output highest ranked ASOs.</LI></UL><P>Anywhere in this process, the user could choose his or her preferred scoring or rank mechanism, or ideally all of them and compare to each other. (For our sample test trials, we have done data analysis comparing all three rank/scoring mechanisms. You could use the same file to analyze the results based on your own test samples.)</P><H3>GC content:</H3><P>Measure the amount of Guanine and Cytosine bases on the ASO, then calculate the proportion of GC bases. Based on previous research results, the effective GC content is between 35% and 70%[1] [2] [3]. Then the GC content value would be either calculated into an arbitrary score or used to apply cutoffs.</P><H3>Off Target Binding:</H3><P>Search for potential off-target binding with human cDNA.</P><H3>RNA-binding proteins competition:</H3><P>RNA-binding protein (RBP) competes with other types of proteins to bind at a region. The level of competition could affect the binding effectiveness of a certain ASO.</P><H3>ASO Secondary Structure:</H3><P> Here we calculate the probability of obtaining a secondary structure for each generated ASO. SpliceMIT calculates the probabilities of a base binding to another on Nupack web interface.</P><H2 style="color: #ff0000;">Ranking Methods</H2><H3>Arbitrary Score</H3><P>An Arbitrary score is calculated respectively in each factor, and then the sum of those arbitrary score is used to rank the effectiveness and efficiency of the ASOs. This score is is based on the following factors:</P><UL><LI>GC content of the ASO</LI><LI>Off-target binding</LI><LI> RNA-binding proteins competition on the ASO binding site</LI><LI>Secondary structure of the ASO</LI></UL><H3>Cutoff</H3><P>The cut-off method preserves the basic scoring methods in the four sub-models.However, the weight and multiplication actors are removed. Instead, the model tests 10 sets of cut-off thresholds on 16 (currently completed) intron sequences.</P><H3>Rank-Product</H3><P>The Rank-Product method also preserves the basic scoring methods in the four sub-models except the removal of the weight and multiplication factors. The method derives from a biologically motivated test for the detection of
differentially expressed genes in replicated microarray experiments23. It is a simple non-parametric statistical method based on ranks of fold changes.</P><H3>Comparison of the Three Rank Methods</H3><P>The three rank methods were also analyzed. For the cut-off method, the analysis used the 10th set of thresholds as shown in Table 2. For the arbitrary score and the rank product methods, only the top 20 ranked ASOs remained in the list. Then the program found the identical ASO sequences between two lists of outputs.</P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>