<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_MIT_GuidesandASOs skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:MIT/GuidesandASOs</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navdiv-wrapper"><DIV class="navdiv"><UL><LI><A href="https://2017.igem.org/Team:MIT/Team">ABOUT US</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/Team">Team</A><A href="https://2017.igem.org/Team:MIT/Attributions">Attributions</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/AS">BACKGROUND</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/AS">Alternative Splicing</A><A href="https://2017.igem.org/Team:MIT/CRISPR">CRISPR</A><A href="https://2017.igem.org/Team:MIT/RBPs">RNA Binding Proteins</A><A href="https://2017.igem.org/Team:MIT/REST">REST</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/project">PROJECT</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/GuidesandASOs">Guides and ASOs</A><A href="https://2017.igem.org/Team:MIT/mk-FF4">mKate-FF4</A><A href="https://2017.igem.org/Team:MIT/2mk-HBG">2 Exon mKate HBG</A><A href="https://2017.igem.org/Team:MIT/3mk-HBG">3 Exon mKate HBG</A><A href="https://2017.igem.org/Team:MIT/3mk-DF">3 Exon Dual Fluorescence</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/notebook">LAB WORK</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/parts">Parts/Improved Parts</A><A href="https://2017.igem.org/Team:MIT/protocols">Protocols</A><A href="https://2017.igem.org/Team:MIT/notebook">Notebook</A><A href="https://2017.igem.org/Team:MIT/Future-Work">Future Work</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/Model">MODELLING</A></LI><LI><A href="#">HUMAN PRACTICES</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/HP/Gold_Integrated">Integrated HP</A><A href="https://2017.igem.org/Team:MIT/Engagement">Public Engagement</A><A href="https://2017.igem.org/Team:MIT/Collaborations">Collaborations</A><A href="https://2017.igem.org/Team:MIT/InterLab">InterLab</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/award">AWARDS</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/HP/Silver">HP Silver</A><A href="https://2017.igem.org/Team:MIT/HP/Gold_Integrated">HP Gold</A><A href="https://2017.igem.org/Team:MIT/Engagement">Education &amp; Public Engagement</A><A href="https://2017.igem.org/Team:MIT/Model">Model</A><A href="https://2017.igem.org/Team:MIT/Part_Collection">Parts Collection</A></DIV></LI></UL></DIV></DIV><H1 style="color:#ff0000; text-align: center; font-size: 40px; line-height: 40px;">Guides and ASOs</H1><P>Both the anti-sense oligonucleotide/MS2 and Cas13a iterations of our experiments necessitate the production of short RNAs with sequences specific to the target site. The explorative nature of our experiments necessitated the use of a large number of distinct RNAs. By nature, RNA is designed to be temporary, and therefore readily degraded by cells. This poses a challenge in the field of RNA technology where the life span of the RNA isn't trivial. To increase our chances of success we also wished to produce these RNAs in our cells continuously, instead of transfecting them, out of concern for rapid degradation rates. But before we launch into how we built the ASO plasmids, what are ASOs...?</P><H2 style="color:#ffffff; background-color:#ff0000; -moz-border-radius: 15px; -webkit-border-radius: 15px; padding:15px; text-align: center"> What are ASOs?</H2><P> Anti-sense oligonucleotides (ASOs) are sequences of 15-20 nucleotides. They are considered &quot;antisense&quot; because they are complementary to an mRNA strand, which is considered to be &quot;sense&quot;. Because of its targeting and binding capability, ASOs have been used for gene therapies and to study gene function [1]. We used ASOs as the base of multiple splice site targeting systems: ASOs on their own, ASOs attached to an Ms2 hairpin loop, and as the basis of our RNA guide sequences for the Cas13 protein.</P><H2 style="color:#ffffff; background-color:#ff0000; -moz-border-radius: 15px; -webkit-border-radius: 15px; padding:15px; text-align: center"> What are Guides?</H2><P> We used the term &quot;guides&quot; to refer to the ASOs specific to the Cas13a protein. These oligonucleotides must be flanked by sequences known as &quot;direct repeats&quot; (DR) so the RNA can be recognized by the CRISPR protein. Although all &quot;guides&quot; are technically ASOs by the fact that we used them to help bring proteins to a reporter, we decided to call the ASOs specific to Cas13a &quot;guides&quot; to make the distinction easier for us.</P><H2 style="color:#ffffff; background-color:#ff0000; -moz-border-radius: 15px; -webkit-border-radius: 15px; padding:15px; text-align: center"> How Did We Choose Our ASOs and Guides?</H2><P> When the spliceosome is removing an intron, it first locates the 5' splice site to mark the beginning of the intron. It then cuts the 5' site and attaches the loose end to the branch point. The spliceosome then locates the end of the intron and cuts at the 3' splice site, releasing the intron. We hypothesized that the best place to target our ASOs was on the 3' splice site; The spliceosome would not be able to recognize the end of the intron, and would include the next exon and intron as one large intron, leading to the exon being skipped. </P><P> Another good place to target would be the branchpoint. When the spliceosome is unable to reach the branchpoint on the intron, it will look for the next branchpoint. The loop formed would now include the exon in between the two introns, leading to exon skipping</P><P> In order to identify which site would be the most effective to target, we created ASOs for the span between the 3' splice site, going 15 nucleotides at a time</P><H2 style="color:#ffffff; background-color:#ff0000; -moz-border-radius: 15px; -webkit-border-radius: 15px; padding:15px; text-align: center"> RNA Secondary Structure</H2><P>One thing we had to keep in mind while designing our ASOs and Guides was the issue of RNA Secondary Structure. Because RNA is a single stranded nucleotide, it is able to bind with itself. In some situations, such as the Ms2 hairpin loop, this is the preferable outcome. However, we wanted our ASOs to be available to bind with the target sequence, which they would not be if they were bound to themselves. We used <A href="http://unafold.rna.albany.edu/?q=mfold">Mfold </A> and <A href="http://www.nupack.org/">NUPACK</A>, two RNA secondary structure generating programs, to test our proposed ASOs.</P><CENTER><I> An example output of NUPACK, each of the colors of the dots represent the probability that they will end up in that configuration. This is our desired structure for our plain ASOs.</I></CENTER><H2 style="color:#ffffff; background-color:#ff0000; -moz-border-radius: 15px; -webkit-border-radius: 15px; padding:15px; text-align: center"> ASO with Ms2 Hairpin Loop</H2><P>Ms2 is an RBP that bind to a hairpin loop structures. In order to stabilize our ASOs, we attached them to a hairpin loop structure, which recruits Ms2 to bind with it. Ms2, as a protein, is more stable than the free floating nucleotides.</P><P> In addition, once the Ms2-ASO construct reaches the splice-site, it will provide an extra barrier to reaching the splice site for the spliceosome.</P><P> It is especially important to check the secondary structure of this ASO construct. We need to maintain the hairpin loop while also leaving the ASO open to bind to the target strand. We once again used NUPACK and Mfold to ensure proper folding of the RNA. If the ASO interfered with the folding, then we added in extra base pairs to stabilize the structure. Below is an example construction of the ASO-Ms2 structure.</P><CENTER><I>The red circle is circling the maintained Ms2 Hairpin loop structure. The green circle is around the maintain ASO structure.</I></CENTER><H2 style="color:#ffffff; background-color:#ff0000; -moz-border-radius: 15px; -webkit-border-radius: 15px; padding:15px; text-align: center"> Building a Backbone</H2><P>In humans the <B>U6 promoter</B> is used to recruit RNA Polymerase III for the production of non-coding RNAs [1]. We obtained a plasmid carrying the U6 promoter and two BbsI sites for Golden Gate cloning form Samira Kiani of the Weiss Lab at MIT. </P><P>From this plasmid we produced a series of variants to optimize the cloning processes for our constructs. We first wished to have an indication of successful integration of the sequences specific to our targeting RNAs downstream of the promoter. To achieve this we amplified a LacZ cassette out of the pDONR vector used in the majority of our Golden Gate cloning reactions with added BbsI sites flanking the cassette to allow for integration into the U6 plasmid. We called this resulting plasmid backbone <B>LacZ Full</B>. Within the BbsI sites on either side of the plasmid, we also added BsaI sites such that the LacZ cassette could be knocked out by our targeting sequences in a second Golden Gate reaction.</P><P> For the anti-sense oligonucleotides specific to Cas13a, we added additional &quot;direct repeat&quot; (DR) sequences in between the BbsI and BsaI restriction sites of the backbone. These sequences contains the code for the RNA loop where the CRISPR protein binds, and including it in the backbone allowed us to reduce the length of the target specific portion we cloned in.</P><P>Following a Golden Gate reaction using the BbsI enzyme, the DR and LacZ cassette will lie downstream of the U6 promoter.
 We called the resulting plasmid backbone <B>LacZ DR</B>.</P><P>Following another Golden Gate reaction using the BsaI enzyme, the inserted guide would replace the LacZ gene because it is flanked by BsaI sites.</P><P>In this way we had a preliminary screen for success in that colonies in which the plasmid had successfully integrated the targeting sequence would be white in the presence of X-gal, while those with intact LacZ cassettes would be blue. After sequencing the plasmid, we could precisely determine whether the plasmid was simply the backbone or included the targeting sequence.</P><H3>References </H3><P> [1] Dias, Nathalie and C. A. Stein. &quot;Antisense Oligonucleotides: Basic Concepts and Mechanisms.&quot; Molecular Cancer Therapeutics. March, 2002.</P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>