Team:MIT/AS
ABOUT US
Team Attributions
BACKGROUND
Alternative Splicing CRISPR RNA Binding Proteins REST
PROJECT
Guides and ASOs mKate-FF4 2 Exon mKate HBG 3 Exon mKate HBG 3 Exon Dual Fluorescence
LAB WORK
Parts/Improved Parts Protocols Notebook Future Work
MODELLING
HUMAN PRACTICES
Integrated HP Public Engagement Collaborations InterLab
AWARDS
HP Silver HP Gold Education & Public Engagement Model Parts Collection
Alternative Splicing
What is RNA Splicing?
In eukaryotic organisms, genes contain noncoding sequences called introns that are interspersed between coding sequences called exons. This means that the DNA sequences that codes for the amino acid sequence of proteins are actually not continuous, but rather split into segments. When a gene is initially transcribed, the primary RNA transcript contains both the exons and introns of the gene. However, it is known that mRNA molecules that enter the cytoplasm are significantly smaller that the primary transcripts. In fact, in humans, the average length of primary RNA transcripts are 27 kb, but the average size protein requires only 1.2 kb of coding sequence. So what is happening?[1]
In 1977, it was observed that the intron regions of primary RNA transcripts are actually removed, or spliced out, and the exons are joined together, resulting in a mRNA molecule that has a continuous coding region.[2] Soon after, it was determined that RNA-protein complexes called small nuclear ribonucleoproteins (or snRNPs) combine with various other proteins to form a larger complex called a spliceosome. The spliceosome recognizes specific sites/ conserved motifs along the intron to facilitate the bringing together of the two ends of the intron. Subsequently, the RNA molecules in the snRNPs catalyze the cutting of the ends and the ligating of the exons. Finally, the spliceosome disassemblies and leaves the mRNA.
What is Alternative RNA Splicing?
Unlike traditional RNA splicing where just the introns of a pre-mRNA sequence are removed, alternative RNA splicing results in some exons either being included or excluded from the final mRNA sequence. This differential splicing occurs depending on what segments are treated as exons during the RNA processing. Thus, one gene has the potential to be spliced into several different mRNA sequences, resulting in several different proteins variants, or protein isoforms, being produced. It is hypothesized that alternative splicing may, in part, explain why humans have a similar number of genes to much less complex organisms, such as nematodes.[3]
On average, a human gene contains of 8.8 exons and 7.8 introns, [5] and it has been projected that over 90% of all human genes undergo alternative splicing.[6] Yet at this time, it is unclear how many of these expected mRNA isoforms are translated into protein. High-throughput studies dedicated to analysing the human proteome, such as large-scale mass spectrometry experiments, have only identified a small portion of the expect alternative mRNA isoforms from RNA-seq analysis. Rather, it appears that human genes have one predominant protein isoform, and that tissue specificity has more to do with gene expression than alternative splicing. [6]
Nevertheless, alternative splicing is still an important eukaryotic post-transcriptional mechanism. Errors in splicing of genes contribute to the development of numerous disease such as myotonic dystrophy, spinal muscular atrophy, retinitis pigmentosa, and various forms of cancer. [7]
Controlling Alternative Splicing
Due to the fact that aberrant splicing patterns lead to so many diseases, gaining control over alternative splicing has been a goal in the growing field of RNA Technology.
RNA-targeting technologies often make use of the fact that RNA is a single stranded nucleic acid. Once the sequence of an RNA strand is known, its complementary strand is also known, and can be used to target and bind to the RNA. These short RNA targeting and binding sequences are called Antisense Oligonucleotides (ASOs).
When attempting to gain control of alternative splicing, researchers have designed ASOs to target different splice sites along the introns and exons. If the ASO is bound to the sequence that the spliceosome needs to recognize the end of an intron, the spliceosome will pass that site and continue searching for the next site. When the spliceosome reaches the next cut site, it will include everything in between the covered site and the current site as part of the intron and will splice it out.
These splice-switching oligonucleotides are currently being tested for their therapeutic applications. The problem that researchers are facing is that short RNA sequences are, by nature, quickly degraded in the cell. Researchers have been looking for ways to make the ASOs more stable, while not also making them more toxic. These design considerations have slowed the progress of the field substantially. That's where our project comes in. We are coupling the RNA-targeting power of ASOs with the stability of a CRISPR Protein or an RNA Binding Protein.
References
[1] Reece, Jane B., and Neil A. Campbell. Campbell biology / Jane B. Reece. Benjamin Cummings, 2011.
[2] Berk, Arnold J. “Discovery of RNA Splicing and Genes in Pieces.” Proceedings of the National Academy of Sciences of the United States of America 113.4 (2016): 801–805. PMC. Web. 1 Nov. 2017.
[3] Sharp, Phillip A. “The discovery of split genes and RNA splicing.” Trends in Biochemical Sciences, vol. 30, no. 6, 2005, pp. 279–281., doi:10.1016/j.tibs.2005.04.002.
[4] Sakharkar, M K, et al. “Distributions of exons and introns in the human genome.” In silico biology., U.S. National Library of Medicine, www.ncbi.nlm.nih.gov/pubmed/15217358.
[5] Wang, Eric T., et al. “Alternative isoform regulation in human tissue transcriptomes.” Nature News, Nature Publishing Group, 2 Nov. 2008, www.nature.com/nature/journal/v456/n7221/abs/nature07509.html.
[6] Tress, Michael et al. “Alternative Splicing May Not Be the Key to Proteome Complexity.” Cell, www.cell.com/trends/biochemical-sciences/fulltext/S0968-0004(16)30118-9.
[7] Garcia-Blanco, Mariano A. “Alternative Splicing: Therapeutic Target and Tool.” Alternative Splicing and Disease Progress in Molecular and Subcellular Biology, 2006, pp. 47–64., doi:10.1007/978-3-540-34449-0_3.
