<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_MIT_2mk-HBG skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:MIT/2mk-HBG</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navdiv-wrapper"><DIV class="navdiv"><UL><LI><A href="https://2017.igem.org/Team:MIT/Team">ABOUT US</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/Team">Team</A><A href="https://2017.igem.org/Team:MIT/Attributions">Attributions</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/AS">BACKGROUND</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/AS">Alternative Splicing</A><A href="https://2017.igem.org/Team:MIT/CRISPR">CRISPR</A><A href="https://2017.igem.org/Team:MIT/RBPs">RNA Binding Proteins</A><A href="https://2017.igem.org/Team:MIT/REST">REST</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/project">PROJECT</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/GuidesandASOs">Guides and ASOs</A><A href="https://2017.igem.org/Team:MIT/mk-FF4">mKate-FF4</A><A href="https://2017.igem.org/Team:MIT/2mk-HBG">2 Exon mKate HBG</A><A href="https://2017.igem.org/Team:MIT/3mk-HBG">3 Exon mKate HBG</A><A href="https://2017.igem.org/Team:MIT/3mk-DF">3 Exon Dual Fluorescence</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/notebook">LAB WORK</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/parts">Parts/Improved Parts</A><A href="https://2017.igem.org/Team:MIT/protocols">Protocols</A><A href="https://2017.igem.org/Team:MIT/notebook">Notebook</A><A href="https://2017.igem.org/Team:MIT/Future-Work">Future Work</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/Model">MODELLING</A></LI><LI><A href="#">HUMAN PRACTICES</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/HP/Gold_Integrated">Integrated HP</A><A href="https://2017.igem.org/Team:MIT/Engagement">Public Engagement</A><A href="https://2017.igem.org/Team:MIT/Collaborations">Collaborations</A><A href="https://2017.igem.org/Team:MIT/InterLab">InterLab</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/award">AWARDS</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/HP/Silver">HP Silver</A><A href="https://2017.igem.org/Team:MIT/HP/Gold_Integrated">HP Gold</A><A href="https://2017.igem.org/Team:MIT/Engagement">Education &amp; Public Engagement</A><A href="https://2017.igem.org/Team:MIT/Model">Model</A><A href="https://2017.igem.org/Team:MIT/Part_Collection">Parts Collection</A></DIV></LI></UL></DIV></DIV><H1 style="color:#ff0000; text-align: center; font-size: 40px; line-height: 40px;">Experiments with 2 Exon mKate HBG Reporter</H1><P>One of our main goals for our project was to design a reporter with a swappable intron. This would allow our system to be modular, as we could quickly test guides that were targeting any intron. We used the mKate-ff4 as inspiration in designing another split mkate, 2-exon reporter.  For the purpose of this project, we used human beta globin (HBG) intron 2. We used this split mKate 2-exon construct as a reporter construct that we used to determine if our dCas13 or Ms2 systems were successful in splicing out the HBG Intron 2. We used HBG Intron 2 for reasons outlined in our <A href="https://2017.igem.org/Team:MIT/project">intron design </A> discussion. Upstream of these sequences is a either the constitutive hEF1a promoter or the DOX controlled TRE-tight promoter. </P><P>The figure above represents our construct. The sequence in between the exons represents HBG intron 2. The 3' splice site will be targeted by the guide-dCas13a complex (or guide-Ms2 complex). In the absence of our system, the intron will be spliced out and result in mRNA transcript with both exons included, and will lead to the creation of a red fluorescent protein. In the presence of our system, the RNA binding protein should block the splice site, and the second exon will be spliced out along with the intron. The fluorescent protein wouldn't be produced, and we should see less red fluorescence. </P><DIV class="column half_size"><P><I>Absence of our system leads to fluorescence</I></P></DIV><DIV class="column half_size"><P><I>Presence of our system leads to knock down in fluorescence</I></P></DIV><H2>Results</H2><H3> Verifying the construct </H3><H4> mKate Titration </H4><P>Before being able to test our guides against our newly designed reporter, we needed to ensure it was exhibiting proper behavior. To that end, we ran an titration on the mKate-HBG construct. We expected that as we increased the amount of reporter in the system, the amount of mKate would rise, as the reporter was designed to output normally in the absence of our system. These expected results can be seen below. The different colored lines corrispond to different <A href="https://2017.igem.org/Team:MIT/HowtoreadaCytoflowgraph"> transfection bins</A></P><P> In our experiment, we varied the amounts of mKate-HBG from 10 to 500 ng. (For a detailed explanation of how to plan a mammalian transfection <A href="https://static.igem.org/mediawiki/2017/6/63/Planning_Mamallian_Transfections_%C2%B7_Benchling.pdf"> click here</A>) </P><CENTER><H4>mKate Titration for mKate-HBG</H4></CENTER><P><I>mKate-HBG reporter amounts (10ng-500ng) vs. the amount of red fluorescence (AU). The color of the line indicate the <A href="https://2017.igem.org/Team:MIT/HowtoreadaCytoflowgraph"> transfection bins</A> of each result.</I></P><P>The reporter is not producing red fluorescence above background noise. </P><STRONG>Unfortunately, we were unable to test any of our constructs against this reporter, because the reporter was not producing red.</STRONG> '

<P>Our next step is to debug our reporter. We suspect the problem may be with the intron we chose, or the way we built that intron, because the mKate-ff4 was outputting fine.</P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>