<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_MIT_mk-FF4 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:MIT/mk-FF4</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navdiv-wrapper"><DIV class="navdiv"><UL><LI><A href="https://2017.igem.org/Team:MIT/Team">ABOUT US</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/Team">Team</A><A href="https://2017.igem.org/Team:MIT/Attributions">Attributions</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/AS">BACKGROUND</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/AS">Alternative Splicing</A><A href="https://2017.igem.org/Team:MIT/CRISPR">CRISPR</A><A href="https://2017.igem.org/Team:MIT/RBPs">RNA Binding Proteins</A><A href="https://2017.igem.org/Team:MIT/REST">REST</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/project">PROJECT</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/GuidesandASOs">Guides and ASOs</A><A href="https://2017.igem.org/Team:MIT/mk-FF4">mKate-FF4</A><A href="https://2017.igem.org/Team:MIT/2mk-HBG">2 Exon mKate HBG</A><A href="https://2017.igem.org/Team:MIT/3mk-HBG">3 Exon mKate HBG</A><A href="https://2017.igem.org/Team:MIT/3mk-DF">3 Exon Dual Fluorescence</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/notebook">LAB WORK</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/parts">Parts/Improved Parts</A><A href="https://2017.igem.org/Team:MIT/protocols">Protocols</A><A href="https://2017.igem.org/Team:MIT/notebook">Notebook</A><A href="https://2017.igem.org/Team:MIT/Future-Work">Future Work</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/Model">MODELLING</A></LI><LI><A href="#">HUMAN PRACTICES</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/HP/Gold_Integrated">Integrated HP</A><A href="https://2017.igem.org/Team:MIT/Engagement">Public Engagement</A><A href="https://2017.igem.org/Team:MIT/Collaborations">Collaborations</A><A href="https://2017.igem.org/Team:MIT/InterLab">InterLab</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/award">AWARDS</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/HP/Silver">HP Silver</A><A href="https://2017.igem.org/Team:MIT/HP/Gold_Integrated">HP Gold</A><A href="https://2017.igem.org/Team:MIT/Engagement">Education &amp; Public Engagement</A><A href="https://2017.igem.org/Team:MIT/Model">Model</A><A href="https://2017.igem.org/Team:MIT/Part_Collection">Parts Collection</A></DIV></LI></UL></DIV></DIV><H1 style="color:#ff0000; text-align: center; font-size: 40px; line-height: 40px;">Experiments with mKate FF4 Reporter</H1><P>The first reporter we tested is what we call the mKate FF4 reporter.</P><P>The mKate FF4 reporter features the red fluorescent mKate gene split into two exons, and has an intron that contains 3 microRNA sites for FF4 in between these two exons. We used this split mKate 2-exon construct as a reporter to determine if our dCas13 or Ms2 systems were successful in hindering the splicing of the FF4 intron. Upstream of these sequences is either the constitutive mammalian hEF1a promoter or the DOX controlled TRE-tight promoter. </P><P>Lucky for us, the entry vector version of this reporter were available to us via the Weiss Lab. After inserting the hEF1a and TRE-tight promoters, the reporter could be used in transfections.</P><P>In order to streamline our design process, we used this reporter to test our guides and ASOs. While we were designing reporters that provided us with more information about our system, we used this reporter to provide an initial proof of concept for our system. We also used this reporter to determine what concentrations of plasmids would provide the best results for our system. </P><P>For this reporer, in the absence of dCas13a or Ms2, the intron should be spliced out as normal, leading to a complete mKate mRNA transcript and flouresence would be seen.</P><P> In the presence of dCas13a or Ms2, which targets the 3' splice site of the intron, the second exon would be spliced out along with the intron, and the final mRNA transcript would not include the second exon. Thus, with out system, we expect a knock down of flourescence.</P><DIV class="column half_size"><P><I>Absence of our system leads to fluorescence</I></P></DIV><DIV class="column half_size"><P><I>Presence of our system leads to knock down in fluorescence</I></P></DIV><H1 style="color:#ff0000; text-align: center; font-size: 30px; line-height: 40px;">Results</H1><H2 style="color:#ffffff; background-color:#ff0000; -moz-border-radius: 15px; -webkit-border-radius: 15px; padding:15px; text-align: center"> ASO and MS2</H2><EM>Before testing the tiling of our ASO guides, we needed to determine the ideal amount of mKate, Ms2, and ASO required to see the effects of our system.</EM><H3> mKate Titration </H3><P> As discussed <A href="https://2017.igem.org/Team:MIT/GuidesandASOs"> earlier</A>, ASOs are unstable in the cell. Therefore, we needed to ensure the proper ASO to reporter ratio. We suspected that our system would be able to affect the splicing of mKate up to a threshold, after which the mKate would saturate the system. These expected results can be seen below. The different colored lines corrispond to different <A href="https://2017.igem.org/Team:MIT/HowtoreadaCytoflowgraph"> transfection bins</A></P><P> In our experiment, we varied the amounts of mKate-ff4 from 10 to 500 ng using <A href="https://2017.igem.org/Team:MIT/Optimizations">ASO3</A> and ASO2+. The ASO2+ was co-transfected with Ms2, so that the Ms2 could bind to the hairpin loop attached to the ASO+. (For a detailed explanation of how to plan a mammalian transfection <A href="https://static.igem.org/mediawiki/2017/6/63/Planning_Mamallian_Transfections_%C2%B7_Benchling.pdf"> click here</A>) </P><CENTER><H4>mKate Titration for ASO2+</H4></CENTER><P><I>mKate-ff4 reporter amounts (10ng-500ng) vs. the amount of red fluorescence (AU) for ASO2+. The color of the line indicate the <A href="https://2017.igem.org/Team:MIT/HowtoreadaCytoflowgraph"> transfection bins</A> of each result.</I></P><P>For the mkate titration for ASO2+, we observe a disturbance at 100 ng of reporter. This indicates that our system is affecting the red output at this concentration. While expected to see a consistent output of red below our saturation threshold, the results still hold for the 100 ng case.</P><P><STRONG>Based on these results, we decided to transfect with 100ng of mkate-ff4 moving forward.</STRONG></P><CENTER><P><I>Click <A href="https://2017.igem.org/Team:MIT/Optimizations">here</A> to see the rest of our optimization experiments.</I></P></CENTER><H3> Tre:Ms2 DOX Induction </H3><P>Once we had determined a proper amounts of mKate-ff4 and ASO plasmid, we wanted to check how much of an effect the Ms2 was having on ASOs with hairpin loops (ASO+). We expected that as the amount of Ms2 increased in the system, more Ms2 would be available to bind the hairpin loops, and lead to increased blocking of the spliceosome. After a certain amount of Ms2 was made, the system would become saturated with Ms2 and the knockdown would level-off. The expected results can be seen below. The different colored lines corrispond to different <A href="https://2017.igem.org/Team:MIT/HowtoreadaCytoflowgraph"> transfection bins</A></P><P> In our experiment, we put Ms2 downstream of Tre, which is a DOX inducible promoter. The more DOX we added to the system, the higher Ms2 production we would see off the Tre promoter. We transfected a constant 300 ng of Tre:Ms2, and varied the amount of DOX from 5 to 500 uM. We ran DOX inductions for ASO0+, ASO2+ and ASO4+. (For a detailed explanation of how to plan a mammalian transfection <A href="https://static.igem.org/mediawiki/2017/6/63/Planning_Mamallian_Transfections_%C2%B7_Benchling.pdf"> click here</A>) </P><CENTER><H4>Tre:Ms2 DOX Induction for ASO0+</H4></CENTER><P><I>DOX Concentrations (5-500uM) vs. the amount of red fluorescence (AU) for ASO0+. The color of the line indicate the <A href="https://2017.igem.org/Team:MIT/HowtoreadaCytoflowgraph"> transfection bins</A> of each result.</I></P><P>Results for ASO0+ showed that as levels of DOX, and therefore Ms2 levels, rose in the system, there was a decrease in red fluorescence.</P><CENTER><H4>Tre:Ms2 DOX Induction for ASO2+</H4></CENTER><P><I>DOX Concentrations (5-500uM) vs. the amount of red fluorescence (AU) for ASO2+. The color of the line indicate the <A href="https://2017.igem.org/Team:MIT/HowtoreadaCytoflowgraph"> transfection bins</A> of each result.</I></P><P>Results for ASO2+, unexpectedly, show a slight increase in red across DOX concentrations.</P><CENTER><H4>Tre:Ms2 DOX Induction for ASO4+</H4></CENTER><P><I>DOX Concentrations (5-500uM) vs. the amount of red fluorescence (AU) for ASO2+. The color of the line indicate the <A href="https://2017.igem.org/Team:MIT/HowtoreadaCytoflowgraph"> transfection bins</A> of each result.</I></P><P>Results for ASO4+ show a slight decrease in red across DOX concentrations.</P><P>The results from ASO0+ show a log decrease in DOX concentrations, indicating that our system is functioning as expected. Based on these results we concluded that maximizing the amount of Ms2 in our system was preferable. Therefore, we decided to transfect with 300 ng of Ms2 moving forward, as this was the maximum amount of DNA per plasmid we transfected.
</P><P><STRONG>Based on these results, we decided to transfect with 300 ng of Ms2 moving forward.</STRONG></P><EM>Now that we had optimized the different amounts for mKate-ff4, ASOs and Ms2, we were ready to test the effectiveness of our tiled guides</EM><H3> ASO Tiling </H3><P>Our ASOs are tiled from the 3' splice site to the polypyrimidine stretch. In order to determine which site is most effective to target, we transfected all our ASOs under the same conditions, and compared their output to a non-targeting ASO. We expect that the non-targeting ASO will have no effect on the red output. We compare how much mKate each ASO knocks down to this standard. We expect there to be a variation between the ASOs.These expected results are shown below.</P><P> In our experiment, we tested four plain ASOs, ASO0 through ASO3, and five ASOs with hairpin loops, ASO0+ through ASO4+. ASO+s were co-transfected with Ms2. (For a detailed explanation of how to plan a mammalian transfection <A href="https://static.igem.org/mediawiki/2017/6/63/Planning_Mamallian_Transfections_%C2%B7_Benchling.pdf"> click here</A>) </P><CENTER><H4>Plain ASO Tiling</H4></CENTER><CENTER><I> ASO Type (ASO0-3, Junk ASO) vs. the amount of red fluorescence (AU).</I>&lt;</CENTER><P>ASO 0, 1, and 3 had a knockdown in red from the control, with ASO3 showing the greatest decrease at about a log. ASO 2 had similar levels of red as the control. </P><CENTER><H4> ASO+ Tiling </H4></CENTER><CENTER><I> ASO Type (ASO0+-4+, Junk ASO) vs. the amount of red fluorescence (AU).</I></CENTER><P>The plain ASOs behaved as expected. Unexpectedly, the ASO+s had an increase in red from the control. Our hypothesis was that too much mKate was being produced before the ASOs and Ms2 had enough time to be translated. We did further experiments to add a time delay to the production of mKate.
</P><H3> Tre:mKate DOX Induction </H3><P>ASOs and Ms2 need to be produced by the cell before they are able to affect the splicing of mKate-ff4. In order to add a time delay between when they were being produced and when mKate was being produced, we put mKate downstream of a DOX inducible promoter. DOX was added to cells 24 hours after the initial transfection, allowing the cells to undergo a doubling before producing mKate. We want to compare the results of the ASO-affected output with a normal Tre:mKate-ff4 DOX induction. We expect that the ASOs will cause a disruption in what would otherwise be a normal mKate induction curve. These expected results are compared side by side below. The different colored lines corrispond to different <A href="https://2017.igem.org/Team:MIT/HowtoreadaCytoflowgraph"> transfection bins</A></P><P> In our experiment, we used ASO1 and ASO1+, and a well with just mKate for control. DOX was added to the system 24 hours after transfection. Each cell was transfected with optimized amounts of plasmid from previous experiments (For a detailed explanation of how to plan a mammalian transfection <A href="https://static.igem.org/mediawiki/2017/6/63/Planning_Mamallian_Transfections_%C2%B7_Benchling.pdf"> click here</A>) </P><CENTER><H4>ASO1 Tre:mKate-ff4 DOX Induction                                      Plain Tre:mKate-ff4 DOX Induction </H4></CENTER><CENTER><I>DOX Concentrations (5-500uM) vs. the amount of red fluorescence (AU) for ASO1 (left) and plain mKate(right). The color of the line indicate the <A href="https://2017.igem.org/Team:MIT/HowtoreadaCytoflowgraph"> transfection bins</A> of each result.</I></CENTER><P>Unexpectedly, the ASO condition has a higher red output than the plain mKate condition.</P><CENTER><H4>ASO1+ Tre:mKate-ff4 DOX Induction                                      Plain Tre:mKate-ff4 DOX Induction </H4></CENTER><CENTER><I>DOX Concentrations (5-500uM) vs. the amount of red fluorescence (AU) for ASO1 (left) and plain mKate(right). The color of the line indicate the <A href="https://2017.igem.org/Team:MIT/HowtoreadaCytoflowgraph"> transfection bins</A> of each result.</I></CENTER><P>Unexpectedly, the ASO condition has a higher red output than the plain mKate condition.</P><P>Unfortunately, there was no significant difference between the control titration and the ASO or ASO+ inducton. All the ASOs will have to be tested in this way before determining whether or not this system is effective. 
</P><H2 style="color:#ffffff; background-color:#ff0000; -moz-border-radius: 15px; -webkit-border-radius: 15px; padding:15px; text-align: center"> Guides and Cas13a</H2><EM> Before testing the tiling of our Cas13a guides, we needed to determine the ideal amount of mKate and guide required to see an effect on our sytem. </EM><H4> mKate Titration </H4><P> As discussed <A href="https://2017.igem.org/Team:MIT/GuidesandASOs"> earlier</A>, guides are unstable in the cell. Therefore, we needed to ensure the proper ASO to reporter ratio. We suspected that our system would be able to affect the splicing of mKate up to a threshold, after which the mKate would saturate the system. These expected results can be seen below. The different colored lines corrispond to different <A href="https://2017.igem.org/Team:MIT/HowtoreadaCytoflowgraph"> transfection bins</A></P><P> In our experiment, we varied the amounts of mKate-ff4 from 10 to 500 ng using Guide2. The guide was co-transfected with dCas13a, which uses the guide to locate the target intron and block the spliceosome from recognizing the splice site. (For a detailed explanation of how to plan a mammalian transfection <A href="https://static.igem.org/mediawiki/2017/6/63/Planning_Mamallian_Transfections_%C2%B7_Benchling.pdf"> click here</A>) </P><CENTER><H4>mKate Titration for Guide2</H4></CENTER><P><I>mKate-ff4 reporter amounts (10ng-500ng) vs. the amount of red fluorescence (AU) for Guide2. The color of the line indicate the <A href="https://2017.igem.org/Team:MIT/HowtoreadaCytoflowgraph"> transfection bins</A> of each result.</I></P><P>We observe a standard mkate curve for all reporter concentration above 100 ng. At 100 ng there was a disturbance in the normal curve, indicating that our system might be acting on the system.</P><STRONG>Based on these results, we decided to transfect with 100ng of mkate-ff4 moving forward.</STRONG><CENTER><P><I>Click <A href="https://2017.igem.org/Team:MIT/Optimizations">here</A> to see the rest of our optimization experiments.</I></P></CENTER><EM>Now that we had optimized the different amounts for mKate-ff4, and guides, we were ready to test the effectiveness of our tiled guides</EM><H3>Guide Tiling </H3><P>Our guides are tiled from the 3' splice site to the polypyrimidine stretch. In order to determine which site is most effective to target, we transfected all our guides under the same conditions, and compared their output to a non-targeting guide. We expect that the non-targeting guide will have no effect on the red output. We compare how much mKate each guide knocks down to this standard. We expect there to be a variation between the guides.These expected results are shown below.</P><P> In our experiment, we tested five plain guides, Guide1 through Guide5.The guides were co-transfected with dCas13a, which uses the guide to locate the target intron and block the spliceosome from recognizing the splice site. (For a detailed explanation of how to plan a mammalian transfection <A href="https://static.igem.org/mediawiki/2017/6/63/Planning_Mamallian_Transfections_%C2%B7_Benchling.pdf"> click here</A>) </P><CENTER><H4>Guide Tiling</H4></CENTER><CENTER><I> Guide Type (Guide1-5, Junk ASO) vs. the amount of red fluorescence (AU).</I>&lt;</CENTER> In the presence of dCas13a, there was minimal knockdown. The greatest knockdown occurred in the presence of Guide2. Our hypothesis about the reason we were not seeing the knockdown we expected was that too much mKate was being produced before the guides and dCas13a had enough time to be translated. We did further experiments to add a time delay to the production of mKate.

<H3> Tre:mKate DOX Induction </H3><P>Guides and Cas13a need to be produced by the cell before they are able to affect the splicing of mKate-ff4. In order to add a time delay between when they were being produced and when mKate was being produced, we put mKate downstream of a DOX inducible promoter. DOX was added to cells 24 hours after the initial transfection, allowing the cells to undergo a doubling before producing mKate. We want to compare the results of the Cas-affected output with a normal Tre:mKate-ff4 DOX induction. We expect that the Cas will cause a disruption in what would otherwise be a normal mKate induction curve. These expected results are compared side by side below. The different colored lines corrispond to different <A href="https://2017.igem.org/Team:MIT/HowtoreadaCytoflowgraph"> transfection bins</A></P><P> In our experiment, we used Guide3, and a well with just mKate for control. DOX was added to the system 24 hours after transfection. Each cell was transfected with optimized amounts of plasmid from previous experiments (For a detailed explanation of how to plan a mammalian transfection <A href="https://static.igem.org/mediawiki/2017/6/63/Planning_Mamallian_Transfections_%C2%B7_Benchling.pdf"> click here</A>) </P><CENTER><H4>Guide 3 Tre:mKate-ff4 DOX Induction                                      Plain Tre:mKate-ff4 DOX Induction </H4></CENTER><CENTER><I>DOX Concentrations (5-500uM) vs. the amount of red fluorescence (AU) for Guide3 (left) and plain mKate(right). The color of the line indicate the <A href="https://2017.igem.org/Team:MIT/HowtoreadaCytoflowgraph"> transfection bins</A> of each result.</I></CENTER><P>Unfortunately, there was no significant difference between the control titration and the Guide3 or Junk guide induction. However, even the control in this experiment was not showing clear results, so the experiment should be rerun before drawing conclusions about the system. All the guides will have to be tested in this way before determining whether or not this system is effective. </P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>