<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_MIT_parts skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:MIT/parts</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navdiv-wrapper"><DIV class="navdiv"><UL><LI><A href="https://2017.igem.org/Team:MIT/Team">ABOUT US</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/Team">Team</A><A href="https://2017.igem.org/Team:MIT/Attributions">Attributions</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/AS">BACKGROUND</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/AS">Alternative Splicing</A><A href="https://2017.igem.org/Team:MIT/CRISPR">CRISPR</A><A href="https://2017.igem.org/Team:MIT/RBPs">RNA Binding Proteins</A><A href="https://2017.igem.org/Team:MIT/REST">REST</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/project">PROJECT</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/GuidesandASOs">Guides and ASOs</A><A href="https://2017.igem.org/Team:MIT/mk-FF4">mKate-FF4</A><A href="https://2017.igem.org/Team:MIT/2mk-HBG">2 Exon mKate HBG</A><A href="https://2017.igem.org/Team:MIT/3mk-HBG">3 Exon mKate HBG</A><A href="https://2017.igem.org/Team:MIT/3mk-DF">3 Exon Dual Fluorescence</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/notebook">LAB WORK</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/parts">Parts/Improved Parts</A><A href="https://2017.igem.org/Team:MIT/protocols">Protocols</A><A href="https://2017.igem.org/Team:MIT/notebook">Notebook</A><A href="https://2017.igem.org/Team:MIT/Future-Work">Future Work</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/Model">MODELLING</A></LI><LI><A href="#">HUMAN PRACTICES</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/HP/Gold_Integrated">Integrated HP</A><A href="https://2017.igem.org/Team:MIT/Engagement">Public Engagement</A><A href="https://2017.igem.org/Team:MIT/Collaborations">Collaborations</A><A href="https://2017.igem.org/Team:MIT/InterLab">InterLab</A></DIV></LI><LI><A href="https://2017.igem.org/Team:MIT/award">AWARDS</A><DIV class="submenu"><A href="https://2017.igem.org/Team:MIT/HP/Silver">HP Silver</A><A href="https://2017.igem.org/Team:MIT/HP/Gold_Integrated">HP Gold</A><A href="https://2017.igem.org/Team:MIT/Engagement">Education &amp; Public Engagement</A><A href="https://2017.igem.org/Team:MIT/Model">Model</A><A href="https://2017.igem.org/Team:MIT/Part_Collection">Parts Collection</A></DIV></LI></UL></DIV></DIV><H1 style="color:#ff0000; text-align: center; font-size: 40px; line-height: 40px;">Parts</H1><P>The following charts document our submitted parts.
</P><P>Entry vectors pertain to genes inserted into a <A href="http://parts.igem.org/Part:pSB1K16">Mammo Block backbone</A>. Expression vectors pertain to constructs that contain a promoter and the gene of interest that would be expressed in our mammalian cells.
</P><P>Guide sequences are necessary for the dCas13a and Ms2 proteins to find the mRNA and bind. Part of our design and experiment was to find the optimal binding site to control splicing, and using this information, perhaps create the &quot;best&quot; intron for splicing. In order to fulfill these goals we made a series of guides that &quot;tile&quot; along a sequence on the targeted mRNA. These guide sequences are categorized into &quot;families&quot; based on the sequences/reporters they target. </P><H2 style="color:#ffffff; background-color:#ff0000; -moz-border-radius: 15px; -webkit-border-radius: 15px; padding:15px; text-align: center"> Improved Part: pDEST mCherry</H2><P>This is an updated version of a part developed by the previous MIT team (BBa_K2100015). The original part is a destination vector similar to the ccdB destination vector (pDEST ccdB) but instead of having the ccdB domain that leads to cell apoptosis, there is an mCherry domain that would normally make colonies containing the plasmid red. After a successful LR reaction this mCherry domain would be substituted with the gene (s) of interest from the entry vector and when picking colonies, you would select for the white colonies instead of red (red means the destination vector in those colonies remained unaltered hence unsuccessful LR). However, what ends up happening is that there are lots of red colonies with few white colonies in between which makes picking difficult.</P><P>Our team added Sce1 restriction sites flanking the mCherry domain on the pDest, but not flanking the R restriction sites. We also added the Sce1 yeast restriction enzymes during the LR process when golden gating the entry with the destination vector. The final product was a destination vector that uses the color red to denote unsuccessful <A href="https://static.igem.org/mediawiki/2017/6/65/LR_Reaction.pdf">LR reaction</A>, while white denotes disruption of the mCherry gene and a successful LR reaction is likely. The distinction between white and red, however, is much clearer than when the original part was used.</P><P> The pictures above demonstrate the improvement of the part. The picture of the left shows colonies containing the previous version of pDEST mCherry. The level of red coloring varies among the colonies, but white colonies are often lost in a sea of red. These factors makes it difficult to determine whether a colony has the desired plasmid, and also difficult choosing colonies.</P><P>The picture on the right shows colonies containing the updated version of pDEST mCherry, which, by comparison, allows for easier colony selection. </P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>