Team:MIT/Attributions
ABOUT US
Team Attributions
BACKGROUND
Alternative Splicing CRISPR RNA Binding Proteins REST
PROJECT
Guides and ASOs mKate-FF4 2 Exon mKate HBG 3 Exon mKate HBG 3 Exon Dual Fluorescence
LAB WORK
Parts/Improved Parts Protocols Notebook Future Work
MODELLING
HUMAN PRACTICES
Integrated HP Public Engagement Collaborations InterLab
AWARDS
HP Silver HP Gold Education & Public Engagement Model Parts Collection
Attributions
Each of our team members played an integral role in helping to develop a strong iGEM project, which included designing constructs, developing a model, running experiments in the lab, and coordinating outreach. We were also supported by our host lab instructors and advisors, and we would like to acknowledge all those who helped make iGEM possible and fun this year!
Team training and Project start
Our iGEM team came together in January of 2017. We began to research interesting topics and brainstorm ideas and continued to refine our focus until we selected our project during the spring semester. We started learning protocols and techniques in the Weiss lab during the spring semester. Our advisors and other members of the 2016 MIT iGEM, as well as one of our instructors, Brian Teague, led these training sessions. We wanted to be able to start the summer with some experience using these techniques. We solidified our project design in the early weeks of the summer and started consistently working in the lab shortly thereafter.
Design & Experimentation
Members of our team designed the primers and most other constructs that we used. Our guide sequences and primers were synthesized by IDT. We ordered the L. shahii construct from Addgene, and we received the U6 promoter, MS2, and fluorescent proteins from the Weiss Lab. Members of our team also performed the lab work that was required for our project, such as PCR, cloning reactions, minipreps, midipreps, and transfections. Our advisors or instructor, Brian Teague, would help us troubleshoot if reactions were not going as planned. Sequencing reactions were sent to Genewiz or Quintara Biosciences.
Model
Members of our team reached out to MIT Burge Lab and Harvard Schier Lab to discuss and learn from their model, ChopChop. We eventually developed our own model, SpliceMIT, to predict gRNA sequences and antisense oligonucleotides. MIT Burge Lab and our supervisor, Brian Teague, both provided feedbacks to the rank methods and scoring algorithms. The lab crews also provided results with the predicted sequences by the model.
Human Practices
Two of our team members spearheaded the outreach efforts of our team by corresponding with our contacts for each event and planning the content as well. The rest of the team provided feedback on lesson plans, poster designs, and whatever else was presented. All team members went to the MIT Museum to engage with patrons for our workshop, and even though only a few people were involved in teaching our visitors from St. Paul's, everyone was able to engage with the students.
The Weiss Lab - MIT Center for Synthetic Biology
Brian Teague
Mentored us throughout the process of developing our own project and coached us in the lab skills we needed to implement it.
Ron Weiss
Provided us with resources from the lab and feedback on the direction of the project.
The Zhang Lab - McGovern Institute for Brain Research at MIT
Dr. Feng Zhang
Provided us with insight and feedback for the inital development of our project
BE Communication Lab
MIT's Biological Engineering Communication Lab features trained graduate student and postdoc Communication Fellows who offer individual coaching, targeted workshops, and a range of initiatives to support their peers as they learn key transferable communication skills that will help them achieve their career goals. They were extremely helpful by giving us feedback on many aspects of our project including presentations, written documents, and graphics.
MIT Museum
A special thanks to Brindha Muniappan and Jennifer Novotney for helping us coordinate our outreach event at the museum.
