Team:MIT/Future-Work
ABOUT US
Team Attributions
BACKGROUND
Alternative Splicing CRISPR RNA Binding Proteins REST
PROJECT
Guides and ASOs mKate-FF4 2 Exon mKate HBG 3 Exon mKate HBG 3 Exon Dual Fluorescence
LAB WORK
Parts/Improved Parts Protocols Notebook Future Work
MODELLING
HUMAN PRACTICES
Integrated HP Public Engagement Collaborations InterLab
AWARDS
HP Silver HP Gold Education & Public Engagement Model Parts Collection
Future Work
There are few aspects to our project that we were unable to explore due to time constraints; however, in the coming months there is potential for further exploration.
"Intron Doping": Making Introns more Intronic
Is it possible to create the "best" intron that has no potential of ever being included in a mature mRNA transcript?  To create the “best” intron, we needed to determine what qualities made it more likely for a sequence of nucleotides to be spliced out. In other words, we needed to define what made an intron, an intron.
There are many motifs characteristic to introns outlined by various sources, so we compiled the most prominent sequences mentioned. One such motif includes the G triplet (GGG). This sequence pattern typically exists in clusters adjacent to the 5’ splice site of the mRNA strand, and have been correlated with splicing efficiency through another mechanism that also involves the 5’ splice site[1]. For example, in the human alpha-Globin Intron 2, there are quite a few GGG sequences that exist about 70 bases downstream of the 5’ splice site [1].
Another intronic motif corresponds to the binding site of a heterogeneous ribonucleoprotein (hnRNP) that is essential to the splicing of mRNA. These proteins are primarily responsible for suppressing RNA splicing by binding to particular exons and preventing spliceosomes from attaching and splicing correctly. In a particular case, moving the binding sequence from an exon to an intron has been found to stimulate rather than inhibit splicing [2].
Using this information, we decided to apply these sequences to a pre-existing intron, with the intent of seeing if adding known intron motifs would modify the efficiency of splicing. Our desired pre-existing intron needed to 1) be constitutively spliced and 2) contain some of the traits characteristic of many introns. The former reason is to ensure that we eliminate the potential that this intron has a tendency to be included in the final transcript. The latter reason is so we could potentially test whether removal of the motifs outlined above would have an impact on splicing. After some consideration, we settled upon the second intron of the human beta globin gene. Literature showed that intron 2 was constitutively spliced out and that intron 2 of human beta-globin enhanced the expression of chimeric genes.  [3] Our intron carried a mutation at location 654, mimicking a mutation in the literature we found conducting experiments on increasing splicing efficiency.
We designed 10 iterations of introns, then decided to prioritize designs so that we settle on 5:
The original HBG intron 2 with just the point mutation at the 654th base pair
Intron with “gggATggg” sequence from base pair positions 14-21, and “gggTTATggg” from position 31-30.
Intron with the binding sequence for hnRNP A1 inserted “tagagtcctagagt” 26 base pairs from the 5’ splice site and 88 base pairs from the 3’ splice site (these sequences and placements were used in a previous paper)
Intron with the binding sequence for hnRNP A1(“tagggt”) added right before the first of two potential branch points, near base pair 830.
Intron with the binding sequence for hnRNP A1 (“tagggt”) before the second of two potential branch points, near base pair 830( without overlapping the first branch point)
Unfortunately, due to time constraints we were unable to build and test theses introns in our 2-Exon and 3-Exon reporters using the second HBG intron; however, the effects of these intronic motifs could be explored in future experiments.
