<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_University_of_Ottawa_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:University_of_Ottawa/Project">Page               </A></LI><LI><A href="/Talk:Team:University_of_Ottawa/Project">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:University_of_Ottawa/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:University_of_Ottawa/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:University_of_Ottawa/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:University of Ottawa/Project</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><P><TITLE>Untitled Document</TITLE><UL id="MenuBar1" class="MenuBarHorizontal"><LI><A class="MenuBarItemSubmenu" href="https://2008.igem.org/Team:University_of_Ottawa">Home</A></LI><LI><A href="https://2008.igem.org/Team:University_of_Ottawa#Welcome_to_the_uOttawa_Team_Wiki.21">Welcome</A></LI><LI><A href="https://2008.igem.org/Team:University_of_Ottawa#Announcements">Announcements</A></LI></UL><LI><A class="MenuBarItemSubmenu" href="https://2008.igem.org/Team:University_of_Ottawa/Team">The Team</A></LI><LI><A href="https://2008.igem.org/Team:University_of_Ottawa/Team#Who_we_are">Who We Are</A></LI><LI><A href="https://2008.igem.org/Team:University_of_Ottawa/Team#Advisors">Advisors</A></LI><LI><A href="https://2008.igem.org/Team:University_of_Ottawa/Team#Undergrads">Undergrads</A></LI><LI><A href="https://2008.igem.org/Team:University_of_Ottawa/Team#What_we_did">What We've Done</A></LI><LI><A href="https://2008.igem.org/Team:University_of_Ottawa/Team#Where_we.27re_from">Where We're From</A></LI><LI><A href="https://2008.igem.org/Team:University_of_Ottawa/Team#Contact_Us">Contact Us</A></LI><LI><A class="MenuBarItemSubmenu" href="https://2008.igem.org/Team:University_of_Ottawa/Project">The Project</A></LI><LI><A href="https://2008.igem.org/Team:University_of_Ottawa/Project#Project_Overview">Overview</A></LI><LI><A href="https://2008.igem.org/Team:University_of_Ottawa/Project#The_Template">Template</A></LI><LI><A href="https://2008.igem.org/Team:University_of_Ottawa/Project#The_Design">Design</A></LI><LI><A href="https://2008.igem.org/Team:University_of_Ottawa/Project#Applications">Application</A></LI><LI><A href="https://2008.igem.org/Team:University_of_Ottawa/Project#References">References</A></LI><LI><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K149001">BioBricks</A></LI><LI><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling">Modeling</A></LI><LI><A class="MenuBarItemSubmenu" href="https://2008.igem.org/Team:University_of_Ottawa/Wet_Lab">Wet Lab</A></LI><LI><A href="https://2008.igem.org/Team:University_of_Ottawa/Lab_Protocols">Lab Protocols</A></LI><LI><A href="https://2008.igem.org/Team:University_of_Ottawa/WetWare">WetWare</A></LI><LI><A href="https://2008.igem.org/Team:University_of_Ottawa/Notebook">Notebook</A></LI><LI><A class="MenuBarItemSubmenu" href="https://2008.igem.org/Team:University_of_Ottawa/Sponsors">Sponsors</A></LI><LI><A href="https://2008.igem.org/Team:University_of_Ottawa/Sponsors#Acedemic_Sponsors">Academic</A></LI><LI><A href="https://2008.igem.org/Team:University_of_Ottawa/Sponsors#Research_Sponsors">Research</A></LI><LI><A href="https://2008.igem.org/Team:University_of_Ottawa/Sponsors#Corporate_Sponsors">Corporate</A></LI></P><P> </P><P> </P><H1 align="center">The Pulsilator</H1><H2 align="center">A synthetic pulse generator in yeast for sustainable expression of recombinant proteins<P>Large-scale production of recombinant proteins typically involves growing genetically modified microorganisms in bioreactors in which metabolic stress and selective pressure tend to diminish culture productivity over time. To alleviate the detrimental effects of continual high-rate synthesis, we have designed a yeast strain capable of producing bursts of gene expression in a controlled and inducible manner. These &quot;pulses&quot; are generated by the action of an inducer molecule that triggers the synthesis of a protein of interest and simultaneously induces a repressor protein to terminate expression as well as an enzyme to degrade the inducer signal, thereby returning the system to its initial state. By co-culturing populations of inducer-synthesizing cells and pulse-generating receiver cells, we hope to achieve self-sustaining oscillatory gene expression dynamics that could render long term culture-based recombinant protein synthesis more sustainable. This would open the door for the production of considerably toxic proteins for numerous applications including anti-cancer therapeutics and antiseptics.
</P></H2><H2><SPAN class="mw-headline" id="Project_Overview"><B>Project Overview</B></SPAN></H2><H3><SPAN class="mw-headline" id="The_Problem"><B>The Problem</B></SPAN></H3><P>When large quantities of protein are required for industrial, pharmaceutical and research purposes, these are often produced by expression in a genetically modified organism. Once these transgenic organisms are built, they are grown to sufficient titer before extracting the protein of interest. In the case of microorganisms, this is performed in large bioreactors, where cells are grown to high densities to maximize productivity. When the protein of interest is excessively large, or strongly overexpressed, a heavy metabolic burden is imposed on the host, causing cell growth to slow and resulting in loss of productivity [ <A href="/Team:University_of_Ottawa/Project#References" title="Team:University of Ottawa/Project">1</A> ]. Additionally, selective pressure against highly productive cells tends to exacerbate this problem over time. Soon the bioreactor will become inefficient, and will have to be “restarted”, a long and costly process. The goal of this year’s uOttawa iGEM team consists of generating a yeast strain capable of long term protein expression.
</P><H3><SPAN class="mw-headline" id="Yeast"><B>Yeast</B></SPAN></H3><P>Heterologous (foreign) proteins are often expressed in simple microorganisms such as bacteria (e.g. <I>E. coli</I>) or yeast. The two most used yeast species are <I>Saccharomyces cerevisiae</I> and the methylotrophic yeast <I>Pichia pastoris</I>, which are well characterized genetically. Yeast provides several advantages for large-scale production, especially for the production of eukaryotic proteins, including fast growth rate, high yield, post-translational processing similar to mammalian cells (e.g. glycosylation), and efficient secretion systems. In our project, we employ the yeast <I>S. cerevisiae</I> (baker's/brewer's yeast) as the chassis for our expression system.
</P><H3><SPAN class="mw-headline" id="Pulsate_Gene_Expression"><B>Pulsate Gene Expression</B></SPAN></H3><P>Gene expression in cells is a dynamic process in which transient responses are often observed.  This can be conceptualized as a “pulse” of protein concentration.  Achieving such a pulse involves a complex network of activities, which makes it difficult to elucidate the mechanisms in naturally occurring systems.  Nevertheless, simple synthetic networks can be created to model this behaviour.  We are working on a feed-forward regulatory motif to generate a pulsate expression of a protein of interest in response to an external stimulus. By expressing protein in short pulses rather than overexpressing continuously, the population may be able to recover from metabolic stress and remain competent for much longer periods of time.
</P><H3><SPAN class="mw-headline" id="Cell-to-Cell_Communication"><B>Cell-to-Cell Communication</B></SPAN></H3><P>An important aspect of naturally occurring biological systems is the cell-to-cell communication that triggers the biomolecular responses in multicellular settings.  To build the feed-forward network, our pulse generator system uses signalling elements from <I>Arabidopsis thaliana</I> that have been implemented in yeast [ <A href="/Team:University_of_Ottawa/Project#References" title="Team:University of Ottawa/Project">2</A> ].  One population of cells send the signal molecule cytokinin isopentenyladenine (IP), while another population receives the signal via the cytokinin receptor AtCRE1, which induces the pulsate expression of the protein of interest.
</P><H3><SPAN class="mw-headline" id="Oscillatory_Dynamics"><B>Oscillatory Dynamics</B></SPAN></H3><P>After the pulsate protein expression event, the gene of interest in a repressed state.  It is desirable to regenerate the system so that it is sensitive to the provided stimulus again – i.e., so that the protein of interest is not repressed.  This is achieved by expressing an enzyme that can degrade the inducer molecule.    With the signal degraded, the protein of interest and the repressor protein are no longer produced, and the system returns to its initial state. The two populations of sender and receiver cells can be grown together. The inducing IP signal is continually supplied to the receiver cells, which would in turn go through cycles of IP stimulation and degradation. Thus, oscillatory dynamics of pulsatile protein expression (pulsilations) can be accomplished.
</P><H2><SPAN class="mw-headline" id="The_Template"><B>The Template</B></SPAN></H2><P>We discovered a useful template system developed by Ming-Tang Chen and Ron Weiss of Princeton University [ <A href="/Team:University_of_Ottawa/Modeling#References" title="Team:University of Ottawa/Modeling">2</A> ]. They developed a yeast strain containing signaling elements from the flowering plant, <I>Arabidopsis thaliana</I>. These yeast express a surface cell receptor AtCRE1 that binds a cytokinin (a type of plant hormone) called isopentenyladenine (IP). This signaling element triggers an endogenous signal transduction pathway , leading to the activation of the nuclear aspartate response regulator SKN7, a transcription factor. By employing a synthetic promoter constructed from a modified yeast MEL1 promoter and containing one or two tandem synthetic SKN7 response elements (SSRE), target genes can be put under inducible control of IP. There are several advantages to using such an inducer molecule. Being completely foreign to the host organism reduces the concern of affecting alternate pathways or of endogenous activation of the pathway. IP is relatively cheap compared to other inducer compounds and can be synthesized on site using the recombinant enzyme AtIPT4 from <I>A. Thaliana</I>. Our ultimate plan is to incorporate this enzyme into a strain of sender cells as part of a dual population system. In addition, given that the selection of biobricks for yeast is currently limited when compared to the spectrum of parts developed for <I>E. coli</I>, we thought that this would be a novel yeast-based system to expand upon.
</P><H3><SPAN class="mw-headline" id="The_original_CRE1_system"> The original CRE1 system </SPAN></H3><P>To construct the original yeast strain, several genetic manipulations had to be performed to ensure proper function [  <A href="/Team:University_of_Ottawa/Project#References" title="Team:University of Ottawa/Project">2</A> ]. It is known that upon binding inducer, CRE1 phosphorylates the histidine phosphor-transfer protein YPD1. In yeast, YPD1 is normally affected by the cell surface osmosensor histidine kinase SLN1. Under normal conditions, SLN1 is constitutively phosphorylated and its kinase activity maintains YPD1 in its phosphorylated state. YPD1 in turn regulates two parallel pathways by transferring its phosphate to two proteins: SSK1 and SKN7. SSK1 phosphorylation is key to suppressing the yeast’s HOG1 pathway – a stress reponse pathway to high osmolarity, whose activity is lethal under normal conditions. YPD1 also shuttles into the nucleus and activates the SKN7 transcription factor by phosphorylation. In the design, SKN7 was preferred over SSK1 because the SLN1-YPD1-SSK1 pathway is known to affect many high-osmolarity response genes. In order to make SKN7 inducible to IP stimulation (i.e. not constitutively phosphorylated), SLN1 had to be deleted; however, SLN1 deletion is lethal due to activation of the HOG1 pathway. In order to keep the HOG1 pathway silenced in the absence of SLN1, the cells were transformed with a plasmid to overexpress PTP2, an endogenous HOG1 phosphatase, thus rescuing the lethal mutant phenotype. 
</P><P>Using this chassis strain (TM182α) [  <A href="/Team:University_of_Ottawa/Project#References" title="Team:University of Ottawa/Project">2,3</A>], the yeast could now be engineered to express the <I>Arabidopsis</I> CRE1 cytokinin receptor and respond to IP. The Weiss group used the CRE1 strain to express IP-inducible GFP in response to sender cells synthesizing IP, as well as a quorum sensing system where the sender and receiver circuits were incorporated into the same cell to achieve cell density dependent expression.
</P><H2><SPAN class="mw-headline" id="The_Design"><B>The Design</B></SPAN></H2><P>Using a CRE1-expressing yeast strain, our goal was to develop a gene regulatory network that would trigger a transient pulse of gene expression upon induction. Many transcription networks in nature have recently been found to contain certain recurring “network motifs” that occur more often than in randomized networks [  <A href="/Team:University_of_Ottawa/Project#References" title="Team:University of Ottawa/Project">4,5</A> ]. One type of motif that has been described having pulsing ability is the feedforward loop. Specifically, it is the Incoherent Type 1 Feedforward Loop (FFL), and has the following topology:
</P><DIV align="center"><P><B>Figure I - </B>Incoherent Type 1 Feedforward regulatory motif
</P></DIV><P>Here A, B represent transcriptional regulators of a gene C. If A is inducible, it can turn on genes B and C upon induction. If B-mediated repression of C is strong when induced by A, the result is a pulse generator. The incoherent type I FFL is the most common of all natural feedforward loops in both <I>Escherichia coli</I> and <I>Saccharomyces cerevisiae</I> as determined by screening transcription network databases [  <A href="/Team:University_of_Ottawa/Project#References" title="Team:University of Ottawa/Project">5</A> ], suggesting that the structure may possess an evolutionary robustness.
</P><P>In our system, the induced transcriptional activator (A) is SKN7 and C is our protein of interest. We include TetR as our transcriptional repressor (B). When activated, SKN7 will induce both the protein of interest C (e.g. GFP) and TetR. TetR will accumulate and repress transcription of C, resulting in a pulse of C expression. A similar incoherent T1 feedforward loop was implemented in <I>E. coli</I> by Basu et al. (2004) [ <A href="/Team:University_of_Ottawa/Project#References" title="Team:University of Ottawa/Project">6</A> ]. 
</P><P>However, we wanted to improve on the design by adding an auto-reset mechanism to allow the cell to return to its initial state and become competent for subsequent pulse inductions. Therefore, we added another component (D), a gene expressing the enzyme cytokinin dehydrogenase (aka cytokinin oxidase, CKX) which is responsible for regulating endogenous cytokinin levels in <I>Arabidopsis thaliana</I> and related plants [ <A href="/Team:University_of_Ottawa/Project#References" title="Team:University of Ottawa/Project">7,8</A> ]. It catalyzes the degradation of cytokinins such as isopentenyladenine and zeatin.
</P><P>
In our design, this gene is also activated by SKN7 and degrades the inducer molecule (isopentenyladenine), thus allowing the levels of A, B and C to return to pre-induction steady state levels. In terms of network structure, we are incorporating a delayed negative feedback loop into our feedforward motif:
</P><DIV align="center"><P><B>Figure II - </B>An additional element is added to the feed forward regulatory motif, where element D inhibits the initial stimulus A
</P></DIV><P>
To construct our genetic network, we used the pSSRE promoter construct provided to us. There are two configurations of this promoter, one containing a single SSRE element and one containing two tandem SSREs. The tandem SSRE configuration was found to have a higher signal, but at the expense of leaky basal expression [  <A href="/Team:University_of_Ottawa/Project#References" title="Team:University of Ottawa/Project">2</A> ]. The gene of interest used for these experiments is the reporter gene coding for yeast enhanced green fluorescent protein (GFP). Our design involves three gene cassettes (one for each of GFP, TetR and CKX) regulated by distinct SSRE promoters. The GFP promoter is modified to contain tet operator elements, allowing repression by TetR.
</P><P>It is obvious that pulse dynamics depend highly on the kinetics of GFP expression as well as TetR repressor expression and repression of GFP. Therefore, we planned to construct a library of constructs, containing different combinations of SSRE elements and tetO elements, and then to screen the strains produced from these for optimal pulse dynamics.
</P><P>We also plan, if necessary, to modify gene expression kinetics of the pSSRE promoters by inducing rational TATA box mutations, based on the study of Blake et al. (2006) [ <A href="/Team:University_of_Ottawa/Project#References" title="Team:University of Ottawa/Project">9</A> ]. Here, it was found that TATA box mutations could affect expression output, without greatly affecting Hill parameters for expression. TATA box mutations were also found to decrease expression noise.
</P><P>With a functioning pulse generator strain of yeast, we plan to design a separate strain of sender cells that express the enzyme IPT4 to synthesize the inducer molecule, IP. It is hoped that by culturing the sender cells and receiver cells together, the receivers will express the gene of interest in successive pulses or oscillations. Such a configuration would allow the culture to regulate recombinant protein expression on its own. It was later also realized that the delayed negative feedback loop could be sufficient for producing oscillatory gene expression, so while working on the various TetR constructs, we decided to also build a strain not containing the TetR cassette at all.
</P><P>
Other levels of non-genetic fine tuning are also available, including control of TetR activity by the use of tetracycline family compounds, and the modification of CKX activity with certain redox reagents. Additionally, the IPT4 gene is placed under the control of a yeast GAL1 promoter, which can be modulated by the addition of glucose or galactose to the media, allowing us to control the rate of IP synthesis. Unfortunately, however, in the original TM182α strain, PTP2 (the overexpressed phosphatase that keeps the Hog1 pathway turned off) is also under the control of a GAL1 promoter. Thus, modifying IP synthesis rate in the sender cells would also affect the viability of the receiver cells. To remedy this problem, we attempted to replace the PTP2 promoter with something different. One of our candidates was the PRP22 promoter available in our lab, which natively regulates a gene involved in RNA splicing. Over the course of our experiments, we determined that the PRP22 promoter is upregulated in response to cell stress caused by DNA damage induced by the compound methyl methanesulfonate (MMS). Although basal expression levels were too low to function effectively in our system, we were able to characterize an interesting new <A href="http://partsregistry.org/Part:BBa_K149001:Design" class="external text" rel="nofollow">biobrick</A> for yeast.
</P><H2><SPAN class="mw-headline" id="Application"><B>Application</B></SPAN></H2><P>With our pulse generator protein expression system, the proteins are only produced in short pulses, reducing the burden on the cells, and preventing toxic accumulation and giving time for the cells to recuperate from stress. These engineered cells can thus be resilient in a bioreactor setting, making them ideal for large-scale protein production applications.
</P><P>The pulse generator can also serve as a useful signaling component to incorporate into more complex biological devices.
</P><H2><SPAN class="mw-headline" id="References"><B>References</B></SPAN></H2><P>[1] Pinasch J., de Mas C., Lopez-Santin J. (2008). Induction strategies in fed-batch cultures for recombinant protein production in <I>Escherichia coli</I>: application to rhamnulose 1-phosphate aldolase. <I>Biochem. Eng. J.</I> 41: 181-187.
</P><P>[2] Weiss R., and  Chen M.-T. (2005). Artificial cell-cell communication in yeast <I>Saccharomyces cerevisiae</I> using signaling elements from <I>Arabidopsis thaliana</I>. <I>Nature Biotech.</I> 23(12): 1551-1555.
</P><P>[3] Maeda T., Wurgler-Murphy S. M., Saito H (1994) A two-component system that regulates an osmosensing MAP kinase cascade in yeast. <I>Nature</I> 369: 242-245
</P><P>[4] Alon U. (2007). Network motifs: theory and experimental approaches. <I>Nature Reviews Genetics</I> 8: 450-461.
</P><P>[5] Alon U., and  Mangan S. (2003). Structure and function of the feed-foward loop network motif. PNAS 100(21): 11980-11985.
</P><P>[6] Basu S., R. Mehreja, et al. (2004). Spatiotemporal control of gene expression with pulse-generating networks. <I>PNAS</I> 101(17): 6355-636
</P><P>[7] Schmulling T, Werner T, Riefler M, Krupkova E, Bartrina y Manns I (2003) Structure and function of cytokinin oxidase/dehydrogenase genes of maize, rice, <I>Arabidopsis</I> and other species. <I>J Plant Res</I> 116:241-252 
</P><P>[8] Frebortova J, Galuszka P, Werner T, Schmulling T, Frebort I (2007) Functional expression and purification of cytokinin dehydrogenase from <I>Arabidopsis thaliana</I> (AtCKX2) in <I>Saccharomyces cerevisiae</I>. <I>Biologia Plantarium</I> 51 (4):673-682 
</P><P>[9] Blake, W., G. Balazsi, et al. (2006). &quot;Phenotypic consequences of promoter-mediated transcriptional noise.&quot; Molecular Cell 24: 853-865.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:University_of_Ottawa/Project">http://2008.igem.org/Team:University_of_Ottawa/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:University_of_Ottawa/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:University_of_Ottawa/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:University_of_Ottawa/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:University_of_Ottawa/Project&amp;oldid=99311" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>