Team:Danci-K8/Design
Danci-K8
MENU ▤
HOME
TEAM
Team
★  Collaborations
PROJECT
★  Description
★ Design
Experiments
★ Proof of Concept
★ Demonstrate
Results
Notebook
PARTS
Parts
★ Basic Parts
★ Composite Parts
★ Part Collection
SAFETY
ATTRIBUTIONS
HUMAN PRACTICES
Human Practices
★ Silver
★ Gold
★ Integrated Practices
★ Engagement
AWARDS
★ Entrepreneurship
★ Hardware
★ Software
★  Measurement
★ Model
Follow us:
Thanks to:
The aim of our project is to create a prototype panel that can characterize the composition in the “sense of taste” receptors for each patient. In the future we can be able to conduct a genetic modification as needed or requested.
For each subject, we performed two operations:
First Operation: Includes a unique characterization of the composition of taste receptor alleles for the five different tastes. This part will be carried out for each subject.  Characterization will be carried out using Taqman technology with the FLUIDIGM system.
Second Operation: Identification of the selected allele for a specific taste receptor encoded and sliced in vitro using the Cas9 RNA-guided enzyme. This change will allow us to control and navigate the ability of the sense of taste as needed.
The basic assumption of the project is based on the fact that there is a direct link between particular taste affection and the number of receptors for that taste. The ability to control the polymorphism of particular taste receptors, while leaving the others intact, may be useful to human society in many ways. For example, as a specific diet aid by reducing the ability to feel the a specific taste, reducing alcohol consumption by changing sensitivity to alcohol, reducing the risk of high blood pressure by eliminating the need for salt taste, changing the sensitivity for bitterness is what will allow swallowing a pill for the purpose of providing medical care and more ..
Design 1: Preliminary stage in the development of a panel which identifies unique alleles of taste receptors
We engineered 16 selected SNPs in favor of this reaction. Each of the SNP’s that were created enables identification of a certain taste out of the five known senses of taste, and each of the tastes will change the variability in the number of alleles that make it possible to identify the taste. The SNP’s as ordered:
Number
Taste
RS
1
Sweet
rs35874116
2
Sweet
rs307355
3
PTC
rs713598
4
PTC
rs1726866
5
PTC
rs1024639
6
Consuming Alcohol
rs846672
7
Coffee Bitterness
RS765007
8
Coffee Bitterness
RS2234001
9
Coffee Bitterness
RS2234012
10
Coffee Bitterness
RS2227264
11
Grapefruit juice bitterness
rs10772420
12
Cilantro Preference
rs72921001
13
Taste of NaC1 & Sour
rs17492553
14
Taste of NaC1 & Sour
rs34160967
15
Umami& Monosodium gluten
rs307377
16
Umami& Monosodium gluten
rs76755863
Using SNP (Single Nucleotide Polymorphism) segments – the Taqman reaction makes it possible to identify the presence of a specific allele tested for a specific taste. Whenever sample DNA is tested for the presence of a specific allele for one of the 16 SNP's we receive a fluorescence signal due to the release of the PROBE if the tested SNP version is present. 70 DNA samples were tested using this method (samples were collected from 70 students and teachers from the school). Each sample was tested to check for the presence of alleles to identify different tastes in accordance to the sequence of the SNP ordered above.
For negative monitoring this reaction a sequence without an SNP was also tested for the taste sense which was engineered especially for this purpose.
For proof of concept
We made sure to create a relatively varied testing system with the number alleles for each of the five tastes that were being checked.
The people we checked filled out a questionnaire while doing the phenotype test, (characterization of the ability to recognize and feel tastes in practice). The questionnaire was analyzed statistically from the test results
Whenever we found a match between the SNP and the sequence tested, the DNA polymerase continued the process of reproduction; the probe was freed and formed a fluorescent signal. We compared the results to the genotype test for all tastes, and for each taste with a number of SNP’s.
We engineered the SNP to be used for negative monitoring.
Design 2- Specific CRISPR/Cas9-mediated activation of the gene of interest
This part was done to check the programming in vitro to the experimental system. The success of this step will enable us to move to the next stage for the development of the final product.
The experimental system consisted of four parts:
Five synthetic gRNA molecules. These molecules were specially designed so that four of them were able to identify SNPs in four flavors respectively, and the fifth molecule will be used as a negative control (does not identify a specific SNP). gRNA- (guide RNA) consists of a unique two dimensional structure (crRNA and tracrRNA) which binds Cas9 and guides it to a dsDNA sequence complementary to the 5' end of the molecule (a 20NT “guide” sequence at the 5’ end of the crRNA). The target of each of these guides is one of the 16 SNP’s that we have synthesized and cloned into synthetic DNA vector (pUC plasmid).
Cas9 endonuclease - the Cas9 endonuclease was ordered as an active enzyme.  Cas9 is the nuclease that complexes with and is guided by the crRNA and tracrRNA (annealed to form a guide RNA) to cleave specific DNA sequences. Binding specificity is based on the guide RNA and a three nucleotide (NGG) sequence called the protospacer adjacent motif (PAM) sequence. Guides were specifically designed for SNP’s whose polymorphism created/destroyed a PAM site, so that only one allele would be disrupted by the Crispr-Cas9 system.
We designed two Synthetic DNA constructs. The two constructs were cloned separately into two Puc57 plasmids. The first construct contains all 16 of the SNPs that allow modifying tastes, the second construct will be used as a monitor for the chosen DNA because it contains SNP’s that don’t allow taste recognition, and will be used as a control for the biological DNA that were collected from the 70 participants who were tested.
Vector – puc57 plasmid – this vector was cloned twice, one clone with 16 SNP’s that were created synthetically in the first part. The second cloning included a negative control segment that does not encode the alleles connected to the taste sense.
For proof of concept
The “Flavoff” system is derived from a number of steps that depend on each other. The occurrence in the last stage, meaning, cutting the suitable SNP and obtaining the expected modifications, prove the venture feasibility of "Flavoff ". Various combinations of gRNA would be expressed and DNA run on a gel to check for expected band sizes. A comprehensive results analysis will further substantiate the feasibility estimate.
Running negative control A – using the fifth gRNA molecule which is not supposed to identify the encoded SNP for the specific different taste receptors. There is no expected cut by Cas9 in this monitor.
Running negative control B – using the vector including a negative construct which does not encode any significant SNP. There is no expected cut by Cas9 in this control.
Synthesized Components
Name
Description
Source
Cas9
"Classical" Cas9 endonuclease for knock-out of target genes
Enzyme ordered from Synthego
gRNA 1:  3 TAS2R19 rs10772420
Grapefruit juice bitterness
crRNA and tracrRNA ordered from Synthego
gRNA 2:  8 TAS2R3 RS765007
Coffee bitterness
crRNA and tracrRNA ordered from Synthego
gRNA 3:  14. TAS1R1 rs34160967
Taste of NaCl & Sour
crRNA and tracrRNA ordered from Synthego
gRNA 4:  15. TAS1R3 rs307377
Umami & Monosodium gluten
crRNA and tracrRNA ordered from Synthego
gRNA 5:  Negative control (non targeting)
Grapefruit juice bitterness
crRNA and tracrRNA ordered from Synthego
Clone construct (16 segments) into vector 1
