<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Uppsala_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Uppsala/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav pull-right" style="align:center !important;"><LI><A href="https://2017.igem.org/Team:Uppsala">Home</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" role="button">Team
      	</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown">Project
      	</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/CrocinPathway">Crocin Pathway</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Zea-Strain">Zea-Strain</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Model">Modeling</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Parts">Parts</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown">Human Practices
      	</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/HP/Silver">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/HP/Gold_Integrated">Ethics-Gold HP</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Engagement">Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Entrepreneurship">Market Analysis</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown">Results
      	</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Medals">Medals</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Portfolio">Portfolio</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown">Extended Work
      	</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Hardware">Chipgineering2.0</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Improve">Improve</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Contribution">Contributions</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Safety">Safety</A></LI></DIV></DIV><DIV style="height:auto !important; width: 99vw !important; !important;padding: 5px 5px 5px 5px;" class="jumbotron"><DIV class="container-fluid"><DIV class="textbox"> The saffron biosynthetic pathway is an extension of the β-carotene pathway, with zeaxanthin being a key intermediate <A href="#ref1">(1)</A>. The pathway from farnesyl pyrophospate (FPP) to zeaxanthin (figure 1) has been BioBricked before, but we wanted to extend the β-carotene pathway to continue from zeaxanthin to crocin.
<FIGURE class="figure"><FIGCAPTION class="figure-caption figtext" style="padding-bottom: 2%; text-align:center;"> Figure 1. The pathway from farnesyl pyrophospate to zeaxanthin.
</FIGCAPTION></FIGURE><B>Integrating the FPP to Zeaxanthin pathway in the Chromosome</B><DIV>Zeaxanthin has previously been expressed in <I>E. coli</I> by iGEM teams using a plasmid. We decided to <A href="https://2017.igem.org/Team:Uppsala/Zea-Strain">integrate this pathway into the chromosome</A> using the <A href="https://2017.igem.org/Team:Uppsala/Experiments">lambda red recombineering method</A>. This would give our project several advantages such as releasing all the plasmid origins and cassettes which would make the insertion of the crocin pathway genes or any other genes of a pathway that originates from zeaxanthin easier. It would make the strain more stable because no constant selective pressure is needed and makes it possible to introduce larger constructs and longer pathways. This also means that there is no need for the use of antibiotics which makes the purification process easier, especially if the product is later used for nutritional purposes. And of course since the first step of our zeaxanthin pathway – farnesyl pyrophosphate – is endogenous to <I>E. coli</I> we would be able to express the whole pathway from farnesyl pyrophosphate to crocin with no need for costly intermediates. </DIV><DIV style="padding-bottom:3%"> If we use <I>cat-sacB</I> selection/counterselection in lambda red recombineering (which we did) we get a scarless method that does not leave behind resistance markers. Lambda red is based on homologous recombination which is mediated by bacteriophage lambda proteins and usually requires only about 35 base pairs of homology on both sides of the inserted gene to work. </DIV><DIV style="padding-bottom:3%"> The pathway that leads from FPP to zeaxanthin includes five genes: <I>crtE</I>, <I>crtB</I>, <I>crtI</I>, <I>crtY</I> and <I>crtZ</I>. All these genes have been previously BioBricked into an operon for <I>E. coli</I>. Due to the time limitations of the project we decided to synthesize two constructs each containing two of the genes – <I>crtEB</I> and <I>crtZY</I>. For successful integration with Lambda red recombination the inserts should not exceed the size of 3000 bp and that is why we could not synthesize all of the genes as a single construct. Each construct also contained promoter, ribosome-binding sites and the necessary homologies for the future integration into the strains. The <I>crtI</I> gene was amplified from a <A href="http://parts.igem.org/Part:BBa_K323125">BioBrick</A> (iGEM Slovenia 2010), without the zinc finger domain, with the primers that contained the homologous sites.</DIV><DIV style="padding-bottom:3%"> The <I>E. coli</I> strains that were used for recombination contained a pSIM5-tet temperature sensitive plasmid with the lambda red system and tetracycline resistance. When temperature rises above 37 °C the lambda red enzymes are expressed. They also had <I>cat-sacB</I> selection cassette (based on Uppsalas own BioBrick <A href="http://parts.igem.org/Part:BBa_K864150">BBa_K864150</A>) carrying chloramphenicol resistance gene and <I>Bacillus subtilis</I> levansucrase <I>sacB</I> gene that is lethal for gram-negative bacteria when expressed in presence of sucrose. </DIV><DIV style="padding-bottom:3%"> Three different starting strains were used during the experiments. We also used three different promoters to avoid homology among these regions. The strain genotypes were:</DIV><DIV style="padding-bottom:2%"> 1. Eco <I>∆gsp::cat-sacB /pSIM5-tet</I></DIV><DIV style="padding-bottom:2%"> 2. Eco <I>∆bglGFB::cat-sacB /pSIM6</I></DIV><DIV style="padding-bottom:3%"> 3. Eco <I>∆IS150::CP25-cat-sacB /pSIM5-tet</I></DIV><DIV style="padding-bottom:3%"> Lambda red recombineering was performed in three steps. For more detailed information, please, see the protocol for <A href="https://2017.igem.org/Team:Uppsala/Experiments#red">Lambda red</A> and the <A href="https://2017.igem.org/Team:Uppsala/Notebook">Lab Notebook</A>.
</DIV><DIV class="miniheader"><B>Step 1</B></DIV><DIV style="padding-bottom:3%"> Phytoene dehydrogenase gene <I>crtI</I> was amplified with PCR from zeaxanthin BioBrick and the homologies were added with the primers. Lambda red was performed to replace <I>cat-sacB-T0</I> selection cassette with <I>crtI</I>, and the cells were plated on sucrose agar plates to select for transformants that had successfully integrated the <I>crtI</I> gene.</DIV><DIV style="padding-bottom:3%"> Eco <I>∆IS150::CP25-cat-sacB-T0 /pSIM5-tet</I> + <I>crtI</I> (PCR product) →  Lambda Red (<I>sucR</I>) →  Eco <I>∆IS150::CP25-crtI /pSIM5-tet</I></DIV><DIV style="padding-bottom:3%"> A construct with beta-carotene hydroxylase gene <I>crtZ</I> and lycopene cyclase gene <I>crtY</I> was inserted with lambda red into another <I>E. coli</I> strain. </DIV><DIV style="padding-bottom:3%">Eco <I>∆gsp::cat-sacB /pSIM5-tet</I> + <I>crtZY</I> (PCR product) → Lambda Red (<I>sucR</I>) →  Eco <I>∆gsp::pFAB70-crtZY /pSIM5-tet</I></DIV><DIV class="miniheader"><B>Step 2</B></DIV><DIV style="padding-bottom:3%"> The trimethoprim resistance gene <I>dhfr</I> (dihydrofolate reductase) was inserted with lambda red into the strain that now contained <I>crtZY</I> genes. The genes we placed very close to each other to enable the following transduction with P1 bacteriophage.</DIV><DIV style="padding-bottom:3%"> Eco <I>∆gsp::pFAB70-crtZY /pSIM5-tet</I> + <I>dhfr</I> (PCR product) →  Lambda Red (<I>tmpR</I>) →  Eco <I>∆gsp::pFAB70-crtZY gspC::dhfr</I></DIV><DIV style="padding-bottom:3%"> Transduction with P1 bacteriophage was done to transfer <I>cat-sacB-T0</I> selection cassette from a donor strain into the strain containing <I>crtI</I> gene.</DIV><DIV style="padding-bottom:3%"> Eco <I>∆IS150::CP25-crtI /pSIM5-tet</I> + <I>∆bglGFB::cat-sacB</I> (P1 lysate) →  Transduction (<I>camR</I>) →  Eco <I>∆IS150::CP25-crtI ∆bgl::cat-sacB /pSIM5-tet</I></DIV><DIV class="miniheader"><B>Step 3</B></DIV><DIV style="padding-bottom:3%"> Lambda red was performed to replace this newly inserted <I>cat-sacB-T0</I> cassette with a construct containing geranylgeranyl diphosphate synthase gene <I>crtE</I> and phytoene synthase gene <I>crtB</I>. The clones with the correct sequence changed color to light red. Our lycopene producing strain was constructed!</DIV><DIV style="padding-bottom:3%"> Eco <I>∆IS150::CP25-crtI ∆bgl::cat-sacB /pSIM5-tet</I> + <I>pFAB46-crtEB</I> (PCR product) → Lambda Red (<I>sucR</I>) →  Eco <I>∆IS150::CP25-crtI ∆bgl::pFAB46-crtEB</I> →  Lycopene!</DIV><DIV style="padding-bottom:3%"> Another transduction with P1 bacteriophage was done with the strain containing <I>crtZY</I> as a donor strain and the strain containing <I>crtEBI</I> as a recipient strain. The clones with the correct sequence turned bright yellow.<DIV style="padding-bottom:3%"> Eco <I>∆IS150::CP25-crtI ∆bgl::pFAB46-crtEB</I> + <I>∆gsp::pFAB70-crtZY gspC::dhfr</I> (P1 lysate) →  Transduction (<I>tmpR</I>) →  Eco <I>IS150::CP25-crtI ∆bgl::pFAB46-crtEB ∆gsp::pFAB70-crtZY gspC::dhfr</I> →  Zeaxanthin</DIV><B>Plan to Extend the pathway from Zeaxanthin to Crocin</B>
To extend the pathway for the conversion of zeaxanthin to <A href="https://2017.igem.org/Team:Uppsala/CrocinPathway">crocin</A>, we needed to add the three step pathway, catalyzed by three different enzyme classes:<OL><LI>Carotenoid cleavage dioxygenases (CCD) are responsible for the symmetric cleavage of zeaxanthin at the 7,8/7′,8′ positions to form crocetin dialdehyde from zeaxanthin <A href="#ref2">(2)</A>.</LI><LI>Aldehyde dehydrogenases (ALDH) converts the 20 carbon cleavage product, crocetin dialdehyde to crocetin <A href="#ref3">(3)</A>.</LI><LI>UDP-glucuronosyltransferase (UGT) catalyzes the glucuronidation reaction of forming crocin from crocetin <A href="#ref3">(3)</A>.</LI></OL>
Next we researched for the most appropriate enzymes under the above enzyme classes to execute the successful conversion of all the intermediates to crocin. We found <I>Crocus Ancyrensis</I> carotenoid cleavage dioxygenase 2 (CaCCD2)<A href="#ref4">(4)</A>, <I>Crocus Sativus</I> aldehyde dehydrogenase 2946 (CsADH2946)<A href="#ref3">(3)</A> and <I>Crocus Sativus</I> UDP-glucuronosyltransferase 2 (UGTCs2) <A href="#ref5">(5)</A> to be the most promising for our project.
The genes of the three individual enzymes were identified and synthesized as a gBlocks by IDT. Prior to synthesis, the genes were also codon optimized for <I>E. coli</I>, to get the desired overexpression of our enzymes. An N-terminal his-tag was added to aid downstream purification process. This was done after drawing conclusions from the <A href="https://2017.igem.org/Team:Uppsala/Model">modeling results</A> that predicted the N-terminal to be outside of the enzymes and far from the active sites after folding. The promoter used was an inducible promoter <A href="http://parts.igem.org/Part:BBa_J04500">BBa_J04500</A> from the iGEM kit. The enzymes were created into BioBricks using plasmid pSB1C3-J04500 with IPTG-inducible expression. The plasmid was linearised with <A href="https://2017.igem.org/Team:Uppsala/Experiments">Phusion PCR</A> and the constructs were inserted into the iGEM plasmid using <A href="https://2017.igem.org/Team:Uppsala/Experiments">Gibson assembly</A>. We <A href="https://2017.igem.org/Team:Uppsala/Experiments">transformed</A> the plasmid into <I>E. coli</I> (TOP10 competent cells), screened the colonies using <A href="https://2017.igem.org/Team:Uppsala/Experiments">colony PCR</A> and run <A href="https://2017.igem.org/Team:Uppsala/Experiments">gel electrophoresis</A> to validate that the insert had been successfully assembled into the plasmid. We also created <A href="https://2017.igem.org/Team:Uppsala/Parts">Biobricks</A> with a part for RFP expression.
</DIV><DIV class="container-fluid"><DIV class="row"><DIV class="col-md-12" style="background-color:#f8d7cf; margin-top: -1.5vh;"><DIV class="heading" style="color:#4d4d4d; height:8vh">References</DIV><DIV class="textbox">
1. Ángela, L. G., and Ahrazem, R. O. 2010. Understanding Carotenoid Metabolism in Saffron Stigmas : Unravelling Aroma and Colour Formation. Functional Plant Science and Biotechnology 4, 56–63.

2. Frusciante S, Diretto G, Bruno M, Ferrante P, Pietrella M, Prado-Cabrero A, et al. Novel carotenoid cleavage dioxygenase catalyzes the first dedicated step in saffron crocin biosynthesis. Proceedings of the National Academy of Sciences. 2014 Aug 19;111(33):12246–51.

3. Gómez-Gómez L, Parra-Vega V, Rivas-Sendra A, Seguí-Simarro JM, Molina RV, Pallotti C, et al. Unraveling Massive Crocins Transport and Accumulation through Proteome and Microscopy Tools during the Development of Saffron Stigma. Int J Mol Sci [Internet]. 2017 Jan 1 [cited 2017 Oct 29];18(1). Available from: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5297711/

4. Ahrazem O, Rubio-Moraga A, Berman J, Capell T, Christou P, Zhu C, et al. The carotenoid cleavage dioxygenase CCD2 catalysing the synthesis of crocetin in spring crocuses and saffron is a plastidial enzyme. New Phytol. 2016 Jan 1;209(2):650–63.

5. Moraga AR, Nohales PF, Pérez JAF, Gómez-Gómez L. Glucosylation of the saffron apocarotenoid crocetin by a glucosyltransferase isolated from Crocus sativus stigmas. Planta. 2004 Oct;219(6):955–66.</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>