<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Uppsala_Contribution skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Uppsala/Contribution</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav pull-right" style="align:center !important;"><LI><A href="https://2017.igem.org/Team:Uppsala">Home</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" role="button">Team
      	</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown">Project
      	</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/CrocinPathway">Crocin Pathway</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Zea-Strain">Zea-Strain</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Model">Modeling</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Parts">Parts</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown">Human Practices
      	</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/HP/Silver">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/HP/Gold_Integrated">Ethics-Gold HP</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Engagement">Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Entrepreneurship">Market Analysis</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown">Results
      	</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Medals">Medals</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Portfolio">Portfolio</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown">Extended Work
      	</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Hardware">Chipgineering2.0</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Improve">Improve</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Contribution">Contributions</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Safety">Safety</A></LI></DIV></DIV><P><TITLE>Zeaxanthin</TITLE></P><DIV class="container-fluid"><DIV class="row"><DIV class="mainheader"> Zexanthin Biobrick Testing </DIV></DIV><DIV class="row text"><DIV class="col-xs-10"><DIV style="padding-bottom:3%; padding-top:3%"> In our project we chose to concentrate on the pathway that leads from farnesyl pyrophosphate (FPP) to crocin. The whole pathway consists of eight genes that code for eight enzymes which might make integrating all of the genes into a plasmid and keeping the plasmid in the bacteria more difficult. Dividing the pathway and integrating the part that leads from FPP to zeaxanthin into the chromosome would both give us a stable zeaxanthin-producing E.coli strain and make performing the remaining steps easier. </DIV><DIV class="miniheader"> Testing the Slovenia Zeaxanthine strain </DIV><DIV style="padding-bottom:3%"> In this experiment we tested a strain of E.Coli created by Slovenia iGEM team in 2010 that produces Zeaxanthin.  The strain contains a biobrick <A href="http://parts.igem.org/Part:BBa_K323122">BBa_K323122</A>. This biobrick includes five genes that convert farnesyl pyrophospate to zeaxanthin. The biobrick is under a lac operon and therefore expression of the enzymes can be induced by IPTG. </DIV><DIV style="padding-bottom:3%"> The zeaxanthin producing strain is important starting point of our project. However, there was a previous concern about the production not being stable and optimal. Therefore, we decided to test production of protein and plasmid DNA under different conditions. For the testing we chose three different temperatures (30 °C, 37 °C and 42 °C) and four different media (LB, TB, M9 and SOB). As LB is usually the standard media for experiments. We decided to use LB with the basic E. coli strain of DH5α to give a baseline for our measurements.  We have also tested two super-rich media (TB and SOB) in attempt to produce higher yield of bacteria in shorter time. To test the possible growth in minimal media we chose M9. </DIV><DIV class="miniheader"> How we did it </DIV><DIV style="padding-bottom:3%"> Zeaxanthin has previously been expressed in E. coli by iGEM teams using a plasmid. We have transformed the plasmid into DH5alpha cells and tested by colony PCR (See figure 1). The positive colonies were sent to sequencing which came out positive. Then we used the positive colonies to for Growth assay testing.
<FIGURE class="figure"><FIGCAPTION class="figure-caption figtext" style="padding-bottom: 2%; text-align:center;"> figure 1.Colony PCR after transformation of the zeaxanthin strain. Marker used is 1 Kb.</FIGCAPTION></FIGURE><DIV class="miniheader"> Growth Assay </DIV><DIV style="padding-bottom:3%">
      The transform strain was inoculated into overnight culture of 5 ml in LB, TB, M9 and SOB with chloramphenicol. DH5-alpha strain was grown overnight in LB. All the cultures were grown at 37 °C on a shaker. Then they were inoculated into fresh media to a total volume of 100 ml and OD600 of 0.05. M9 culture did not results in any growth on two occasion and was discarded as the strain probably is not capable of growing on minimal media. Cultures were placed on a shaker in three temperatures (30°C, 37°C and 40°C) and induced with IPTG to a final concentration of 1mM. After inoculating the OD600 was measured every 30 minutes for 350 minutes. </DIV><DIV class="miniheader"> Plasmid Preparation </DIV><DIV style="padding-bottom:3%"> To determine which media and temperature is optimal for the best DNA yield cultures in LB, SOB and TB in 37 °C and 42 °C. This experiment was not done for 30 °C as the growth there was generally very slow. The plasmids were prepared according to the protocol for PureLink Quick Plasmid Miniprep kit from Invitrogen.  </DIV><DIV class="miniheader"> Results </DIV><DIV style="padding-bottom:3%"> It is apparent from the results that the most optimal way of growing the zeaxanthin strain is in 37 C and in TB media (See figures 2-5). This result is probably not surprising as TB is a super-rich media and 37 C in the usual optimal temperature for growth. However, we can see that the plasmid is taking its toe on the cells with the growth being a lot faster for the wild-type strain in LB in all occasion even outgrowing super-rich media. Another surprising observation is that SOB media measured worse than any other of the rich media despite being also richer than LB. The main difference between the media is use of tryptone in SOB instead of peptone. SOB also uses more salts including KCl, MgCl<SUB>2</SUB> and MgSO<SUB>4</SUB> in addition to NaCl which is also present in LB. Both the different amino acid source and different salt concentration might cause difference in growth in case of adding plant genes into bacterium and to say with resolution which one it might be we would need to conduct more experiments. However, we did not observe any growth in minimal media (M9) which does not use either source of amino acids (instead it uses glucose) and use various salts to provide all basic building blocks for the cells. In this media the strain does not grow altogether. 

<FIGCAPTION class="figure-caption figtext" style="padding-bottom: 2%; text-align:center;"> figure 2.Measurments of OD600 in time for 30 degrees in TB (blue), LB (orange), LB with DH5a(grey) and SOB (yellow). Red triangle suggest inducing with IPTG.</FIGCAPTION><FIGCAPTION class="figure-caption figtext" style="padding-bottom: 2%; text-align:center;"> figure 3.Measurments of OD600 in time for 37 degrees in TB (blue), LB (orange), LB with DH5a(grey) and SOB (yellow). Red triangle suggest inducing with IPTG.</FIGCAPTION><FIGCAPTION class="figure-caption figtext" style="padding-bottom: 2%; text-align:center;"> figure 4. Measurments of OD600 in time for 42 degrees in TB (blue), LB (orange), LB with DH5a(grey) and SOB (yellow). Red triangle suggest inducing with IPTG.</FIGCAPTION></DIV><DIV style="padding-bottom:3%">We also observed little change in growth rate after addition of IPTG. Usually induction of protein synthesis slows the cells growth as it requires resources otherwise used in the cell division. We believe this is most likely caused by slow growth in general as we sequenced the used strain and confirmed the sequence of the insert and it was previously showed to be working by <A href="https://2013.igem.org/Team:Uppsala/zeaxanthin">iGEM Uppsala 2013</A>. The slow growth probably also causes the little division into the four stages of bacterial growth together with the fact we inoculated into the same media with attempt to reduce lag phase. 


<FIGCAPTION class="figure-caption figtext" style="padding-bottom: 2%; text-align:center;"> figure 5. : Growth of the zeaxanthin strain in TB in different temperatures. Red square marks induction with IPTG.</FIGCAPTION></DIV><DIV style="padding-bottom:3%"> In the case of preparation of DNA for transformation or biobrick preparation we tested which media and temperature yields in most prepared plasmid DNA. TB at 42 °C yielded into most DNA per cell compared to other media (See figure 6). However, SOB worked the second best in both temperatures when the OD was adjusted for comparison. This suggests that despite slower growth rate SOB is able to hold more copies of the plasmid than both cultures of LB and the 37 °C fraction of TB. In general LB also provided the worst yield of plasmid DNA. Therefore, we would suggest using TB in 42 °C for DNA preparation of this strain or SOB in 37 °C. 

<FIGCAPTION class="figure-caption figtext" style="padding-bottom: 2%; text-align:center;"> figure 6. The ng of DNA per cell extracted after overnight culture in various media and temperatures</FIGCAPTION></DIV><DIV class="miniheader"> References </DIV><DIV style="padding-bottom:3%">DNA Submission - parts.igem.org. Available at: http://parts.igem.org/DNA_Submission 
</DIV><DIV style="padding-bottom:3%">iGEM Slovenia, 2010. Part:BBa K323122 - parts.igem.org. Available at: http://parts.igem.org/Part:BBa_K323122</DIV><DIV style="padding-bottom:3%">iGEM Uppsala, 2013. Zeaxanthin. Available at: 2013.igem.org/Team:Uppsala/zeaxanthin </DIV>
Li, X.-R. et al., 2015. Metabolic engineering of Escherichia coli to produce zeaxanthin. Journal of Industrial Microbiology &amp; Biotechnology, 42(4), pp.627–636. Available at: http://www.ncbi.nlm.nih.gov/pubmed/25533633      <DIV style="padding-bottom:3%">Madigan, M.T. et al., 2014. Brock biology of microorganisms 14th ed., Pearson.</DIV><DIV style="padding-bottom:3%"> Ruther, A. et al., 1997. Production of zeaxanthin in Escherichia coli transformed with different carotenogenic plasmids. Applied microbiology and biotechnology, 48(2), pp.162–7. Available at: http://www.ncbi.nlm.nih.gov/pubmed/9299773 </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>