Team:Uppsala/CrocinPathway
Home
Team
Team
Collaborations
Attributions
Project
Description
Design
Crocin Pathway
Zea-Strain
Modeling
Parts
Human Practices
Human Practices
Ethics-Gold HP
Engagement
Market Analysis
Results
Results
Demonstrate
Medals
Portfolio
Extended Work
Chipgineering2.0
InterLab
Improve
Contributions
Safety
<!DOCTYPE html>
Crocin Pathway
CROCIN PATHWAY
We wanted to extend the saffron pathway from FFP to zeaxanthin to crocin, but the pathway was poorly characterized. We identified and created sequence verified BioBricks out of three enzymes that can perform the three step conversion from zeaxanthin to crocin: CaCCD2, CsADH2946 and UGTCs2. We have also characterized these enzymes with experiments and simulations. Above all, we are the first to purify and confirm activity of CsADH2946 as well as measuring the kinetic parameters of the enzyme (KM = 20.7842 µM ± 3.5264). In addition, we performed steered molecular dynamics (pulling) with CsADH2946 and the substrate crocetin dialdehyde, which showed that CsADH2946 has a high affinity towards the substrate. Our experimental data and modeling results show that CsADH2946 is a very good enzyme for this crocin pathway reaction.
Step 1: Zeaxanthin → Crocetin dialdehyde
The gene for CaCCD2 (Crocus Ancyrensis carotenoid cleavage dioxygenase 2) wasn’t discovered until 2014 (1) with further studies from 2015 (2), and has not been used by any iGEM team previously. Other iGEM teams have used another gene (ZCD) to make crocetin dialdehyde but the enzyme was malfunctioning for this reaction. We instead identified and used CaCCD2: a perfect candidate for our pathway. We successfully made a sequence verified BioBrick of CaCCD2 with his-tag ( BBa_K2423005 ). The BioBrick was also combined with the other steps in the pathway and inserted into the zeaxanthin producing E. coli strain for a complete pathway from FPP to crocin. See the result here !
Modeling of CaCCD2
Since the enzyme is poorly characterized, we created a homology model and performed stability simulations to verify that our model was reasonable. The resulting structure of the homology modeling and its corresponding RMSD plot from simulations can be seen in figure 1. The RMSD plot indicated that the model conforms to a stable structure. Read more about the homology modeling and dynamics modeling in the Modeling section .
Figure 1. Homology model of CaCCD2 and RMSD plot for the same model. The simulation was run for 100 ns and displayed a stable homology model.
Step 2: Crocetin dialdehyde → Crocetin
We are the first to express and characterize CsADH2946 (Crocus Sativus aldehyde dehydrogenase 2946)! This aldehyde dehydrogenase gene from Crocus Sativus has previously only been identified as a candidate gene through proteome analysis, and has thus never been isolated or characterized before (3). We successfully made a sequence verified BioBrick of CsADH2946 with his-tag ( BBa_K2423007 ). The BioBrick was also combined with the other steps in the pathway and inserted into the zeaxanthin producing E. coli strain for a complete pathway from FPP to crocin. See the result here ! In summary, our experimental data and modeling results show that CsADH2946 is a very good enzyme for this reaction.
Purification of CsADH2946
CsADH2946 was transformed and expressed in E. coli strain BL21(DE3*) and purified using IMAC on an ÄKTA protein purification system. We used a gradient of imidazole concentration from 20–500 mM, in order to get our enzyme as separated as possible from other proteins that ended up in the fractions. The peak highlighted by the arrow in the chromatogram (figure 2) indicates protein that elutes at high imidazole concentration, i.e. our desired his-tagged CsADH2946. The purification was followed by SDS-PAGE to analyse the fractions, control purity and verify the protein product. In figure 3a the band at around 60 kDa in the crude pellet indicates an overexpression of a protein in that size range. In the SDS gel of fractions 16–26 collected between 115–145 mL elution volume (figure 3b) there is a strong band at 60 kDa corresponding to the molecular weight of CsADH2946, indicating that our protein was successfully overexpressed and well-separated.
Figure 2. Chromatogram from IMAC-purification of CsADH2946.
Figure 3. a) SDS-PAGE gel of from IMAC purification. 1) Crude pellet. 2) Pellet after lysis. 3) Supernatant after lysis. 4) Flow through. 5) Wash with buffer A. 6) PageRuler protein prestained ladder. b) SDS-PAGE gel from IMAC purification. Fractions 16-26 were collected between 115 and 145 ml elution volume. A band at about 60 kDa is clearly visible.
Activity measurements of purified CsADH2946
To verify the activity of our purified enzyme CsADH2946 to convert crocetin dialdehyde to crocetin, an activity measurement assay was performed on a plate reader measuring absorbance of the substrate and product of the reaction. For the experiment we used a 96-well plate in which we included wells with enzyme from pooled fractions + substrate, as well as positive and negative controls, see table 1 for the specifics.
Table 1. Content of wells used for activity measurement of CsADH2946.
As can be seen in figure 4, the absorbance of the product crocetin increases over time in well 2 containing enzyme and the substrate crocetin dialdehyde. After 9 hours of reaction, the blue curve corresponding to the enzyme + substrate mixture has increased its absorbance in the exact range of the product. The negative and positive control curves look similar to time point zero, apart from some precipitation of product and substrate indicated by the decreased curves — definite evidence that we succeeded to produce a functional CsADH2946 enzyme. Using this data, we could estimate KM = 20.7842 µM ± 3.5264.
In addition, in figure 5 we can see that well 2 containing enzyme and crocetin dialdehyde has changed color compared to the negative control, to become more yellow like the product crocetin in well 8. This also shows that CsADH2946 was produced and that it converts crocetin dialdehyde into crocetin.
Figure 4. Activity measurement curve. The dotted lines describe wells at the starting time and the fully drawn lines describe the absorbance after 9 hours. The blue lines indicate wells containing protein and crocetin dialdehyde, red lines describe positive control with only crocetin dialdehyde and black lines correspond to the negative control only containing the desired product crocetin.
Figure 5. 96-well plate for activity measurements post 24 hours reaction and plate reading. The plate includes pooled enzyme fractions 10–15 + substrate crocetin dialdehyde (well 1), pooled enzyme fractions 16–23 + crocetin dialdehyde (well 2), flow through + crocetin dialdehyde (well 3), negative control with only crocetin dialdehyde (well 4), pooled enzyme fractions 10-15 + product crocetin (well 5) pooled enzyme fractions 16–23 + crocetin (well 6), flow through + crocetin (well 7) and positive control with only crocetin (well 8).
Modeling of CsADH2946
Since the enzyme is poorly characterized, we created a homology model and performed stability simulations to verify that our model was reasonable. The homology modeling revealed that CsADH2946 is in fact tetrameric, which helped us in the purification and characterization process. We performed a pulling simulation between the enzyme and its substrate in order to estimate binding energy and calculate a theoretical Kd (=4.9321 µM). The resulting structure of the homology modeling and a plot of the pulling simulation can be seen in figure 6. Using the results from the activity measurement, the earlier unknown Michaelis-Menten kinetic parameters of the reaction could also be estimated using a Bayesian inference algorithm. With this method we got KM (=20.7842 µM). Read more about the homology modeling, dynamics modeling and the kinetic parameter estimation in the Modeling section .
Figure 6. Homology model of CsADH2946 and a plot demonstrating the pulling of the substrate crocetin dialdehyde from the active site of CsADH2946.
Step 3: Crocetin → Crocin
We successfully made a sequence verified BioBrick of UGTCs2 (Crocus Sativus UDP-glucuronosyltransferase 2) with his-tag ( BBa_K2423008 ). The BioBrick was also combined with the other steps in the pathway and inserted into the zeaxanthin producing E. coli strain for a complete pathway from FPP to crocin. See the result here !
Modeling of UTGCs2
Since the enzyme is poorly characterized, we created a homology model and performed stability simulations to verify that our model was reasonable. The resulting structure of the homology modeling and its corresponding RMSD plot from simulations can be seen in figure 7. The RMSD plot indicated that the model conforms to a stable structure. Read more about the homology modelling and dynamics modelling in the Modeling section .
Figure 7. Homology model of UGTCs2 and RMSD plot for the same model. The simulation was run for 100 ns and displayed a stable homology model.
References
(1) Frusciante S, Diretto G, Bruno M, Ferrante P, Pietrella M, Prado-Cabrero A, et al. Novel carotenoid cleavage dioxygenase catalyzes the first dedicated step in saffron crocin biosynthesis. Proceedings of the National Academy of Sciences. 2014 Aug 19;111(33):12246–51.
(2) Ahrazem O, Rubio-Moraga A, Berman J, Capell T, Christou P, Zhu C, et al. The carotenoid cleavage dioxygenase CCD2 catalysing the synthesis of crocetin in spring crocuses and saffron is a plastidial enzyme. New Phytol. 2016 Jan 1;209(2):650–63.
(3) Gómez-Gómez L, Parra-Vega V, Rivas-Sendra A, Seguí-Simarro JM, Molina RV, Pallotti C, et al. Unraveling Massive Crocins Transport and Accumulation through Proteome and Microscopy Tools during the Development of Saffron Stigma. Int J Mol Sci [Internet]. 2017 Jan 1 [cited 2017 Oct 29];18(1). Available from: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5297711/
