<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Uppsala_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Uppsala/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav pull-right" style="align:center !important;"><LI><A href="https://2017.igem.org/Team:Uppsala">Home</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" role="button">Team
      	</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown">Project
      	</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/CrocinPathway">Crocin Pathway</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Zea-Strain">Zea-Strain</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Model">Modeling</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Parts">Parts</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown">Human Practices
      	</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/HP/Silver">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/HP/Gold_Integrated">Ethics-Gold HP</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Engagement">Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Entrepreneurship">Market Analysis</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown">Results
      	</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Medals">Medals</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Portfolio">Portfolio</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown">Extended Work
      	</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Hardware">Chipgineering2.0</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Improve">Improve</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Contribution">Contributions</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Safety">Safety</A></LI></DIV></DIV><DIV class="woof"><DIV class="container bookie"><DIV id="dairy" class="carousel slide"><DIV class="carousel-inner"><DIV class="item active"><DIV class="container-fluid"><DIV class="row entry"><DIV class="col-sm-4 txt"><H3>Week: 07/06–16/06</H3><P>
– A safety demonstration was done by the Lab technicians and Project managers which enabled us to know the fire escapes, first aid and emergency contacts. Lab space was assigned.
– Buffers and stock solutions were prepared. LB media and LB-agar plates with different antibiotics (Kanamycin, Chloramphenicol and Tetracycline) were prepared.
– <I>E. coli DH5α</I> competent cells were prepared. Ordered constructs for the three genes (CaCCD2, CsADH2946 and UGTCs2) were digested with available enzyme (E and S) and ligated to Linearized plasmid backbones from the iGEM Kit.


</P></DIV><DIV class="col-sm-4 txt"><P>- Prepared competent cells were tested with the iGEM Competent Cells Test kit. No growth was observed. Possible causes: Faulty competent cells, improper digestion or transformation protocol.
- Made DH5α competent cells with another new protocol to be tested and used later.</P></DIV></DIV></DIV></DIV><DIV class="item"><DIV class="container-fluid"><DIV class="row entry"><DIV class="col-sm-4 txt"><H3>Week: 19/06–22/06</H3><P>
– Testing of the competent cells made last week was done with the competent cells test kit. There was growth suggesting that competent cells were working good.
– To test if the restriction digestion, ligation and transformation worked well, they were repeated again with the newly prepared and tested competent cells above. No growth was observed even when high concentrations were plated, suggesting that the fault was in the Restriction digestion, ligation steps</P></DIV><DIV class="col-sm-4 txt"><H3>Week: 26/06–30/06</H3><P>
– On expert suggestion, it was understood that the enzymes need few extra bases on the ends for efficient activity. Thus we designed and ordered primers that would add overhangs to our constructs. Other reason could be that the promoters were too strong and the genes were toxic to the cell. So a set of primers were also designed to remove promoter from between the prefix and RBS.
– To circumvent the high toxicity of the gene, we decided to clone the devices in low copy plasmids. For that low copy plasmids were identified from the master strain database of glycerol stock at the lab, grown overnight in agar plates, grown overnight in LB and plasmids were purified. 


- Digestion was tested with X and S to optimize the protocol for digestion</P></DIV></DIV></DIV></DIV><DIV class="item"><DIV class="container-fluid"><DIV class="row entry"><DIV class="col-sm-4 txt"><H3>Week: 03/07–7/07</H3><P>
– The primers designed last week were tested and PCR Parameters optimized. PCR amplicons were electrophoresed and positive sections were eluted from the gel and their concentrations measured. 
– Low copy plasmid backbones purified last week were digested and linearized. The PCR amplified constructs were digested and ligated to Low copy plasmid backbone. Transformation was done
– High copy plasmid backbones from the glycerol stock were purified. </P></DIV><DIV class="col-sm-4 txt"><H3>Lambda red integration:</H3><P>
– Planning/research for project.
– Made competent cells of IS150 and gsp strain.</P></DIV></DIV></DIV></DIV><DIV class="item"><DIV class="container-fluid"><DIV class="row entry"><DIV class="col-sm-4"><H3>Week: 10/07–14/07</H3><P>
– Low copy plasmids transformed last week were re-streaked. Colony PCR followed by gel electrophoresis was done on them.
– Positive colonies, screened from colony PCR were re-streaked and glycerol stock was made for them. 
– Electroporation-competent DH5α cells were prepared.
– The PCR amplified constructs were restricted digested and ligated to high copy plasmid backbones cells. These plasmids were then transformed by heat-shock.
– More low copy plasmid for each antibiotics with Red fluorescent protein  RFP were purified from the master stock.
– UGTCs2 gene was cloned in pSB1C3 plasmid backbone since using tetracyclin was being troublesome.
– Electro-competent cells were tested with cloned genes. No growth was observed. Possible cause: cells died during electroporation. Desalt the ligated DNA.
– Colony PCR of the positive colonies (white colonies) obtained from heat shock transformation of cloned genes in high copy plasmids. No bands visible.

</P></DIV><DIV class="col-sm-4"><H3>Lambda red integration:</H3><P>
– Tested transduction with bgl strain as the donor and MG1655 as receiver (<I>cat-sacB</I> will be transduced). 1 colony was found on the plate with chloramphenicol. This one was restreaked on chloramphenicol which then had more growth. 
– Tried to insert <I>crtI</I> gene in IS150 strain and <I>dhfr</I> gene in gsp strain using lambda red. The IS150 strain grew on sucrose (indicating that the <I>sacB</I> sequence had been replaced) but tetracycline had not been added so the strain could have lost the lambda red plasmid. The gsp strain had strange growth pattern.</P></DIV></DIV></DIV></DIV><DIV class="item"><DIV class="container-fluid"><DIV class="row entry"><DIV class="col-sm-4"><H3>Week: 17/07–21/07</H3><P>
– Fresh start of restriction digestion and ligation of PCR amplified constructs into high copy plasmids. Used both heat shock (with commercial competent cells) and electroporation.
– Positive (white) colonies were restreaked and colony PCR was done. No bands were observed. 
– Troubleshooting of colony PCR. Optimised PCR parameters and new buffer components.
– SDS-PAGE parameters were optimised and a test run was done with BSA dilutions. No bands were observed in the first run due to improper buffer. Troubleshoot and Run again!</P></DIV><DIV class="col-sm-4"><H3>Lambda red integration:</H3><P>
– PCR of inserts <I>crtI</I>, <I>dhfr</I>, <I>crtZY</I> and <I>crtEB</I>. Gel elecrophoresis for verification (see figure below).
– PCR of inserts <I>crtZY</I> and <I>crtEB</I>. Gel electrophoresis for verification but there was nothing on gel purification of inserts <I>crtI</I> and <I>dhfr</I>.
– Gel electrophoresis of pooled PCR products of insert <I>crtEB</I> and <I>crtZY</I> from 17/7 (see figure below). Electroporation of purified <I>crtI</I> gene in IS150 strain and purified <I>dhfr</I> gene in gsp strain. Put overnight on shaking water bath.
– The overnight cultures of IS150 strain was streaked on tetracycline and sucrose plates and gsp strain was streaked on tetracycline and trimetoprim plates. Gel electrophoresis of <I>crtEB</I> insert (see figure below).</P></DIV></DIV></DIV></DIV><DIV class="item"><DIV class="container-fluid"><DIV class="row entry"><DIV class="col-sm-4"><H3>Week: 24/07–28/07</H3><P>
– Positive colonies carrying construct in tetracycline backbone from last week were re-streaked to check for viability. 
– Buffers for immobilised metal affinity chromatography (IMAC) were prepared and degassed. IMAC column was packed.
– Positive cells (white colonies) from containing genes in high copy plasmids were grown overnight and sonicated. The lysates were purified by IMAC.
– More LB agar plates with antibiotics were prepared.</P><H3>Lambda red integration:</H3><P>
– PCR of insert crtEB. Restreak clones of IS150 strain from 21/7 on sucrose and tetracycline plates and gsp strain from 21/7 on and tetracycline and trimetoprim plates. Liquid cultures with same respective antibiotics for the strains were made.
– Gel electrophoresis of <I>crtEB</I> PCR product from 24/7. Restreak colonies of IS150 strain from 24/7 plates on sucrose plates then chloramphenicol plates. Prepare cultures for transduction of IS150 by growing overnight in P1 LB. Purify <I>crtZY</I> and <I>crtEB</I> and use nanodrop on all purified inserts to check DNA concentration. Ordered new primers for <I>crtEB</I>.</P></DIV><DIV class="col-sm-4"><P>
– gsp strain did not grow on plates but did in liquid culture. 5 of 8 colonies of IS150 strain from 25/7 grew on sucrose but not chloramphenicol. New plates were made (tet + chlor, tet + tmp, tet + suc, chlor). Transduction of bgl strain as the donor and IS150 as receiver. Take some gsp liquid culture and put in some tet + sucrose to make sure it's our strain that's growing. Make liquid cultures with gsp (<I>dhfr</I>) in tet + tmp with the different tet to check if antibiotics work.
– No growth on transducer IS150 from 26/7. Start overnight culture of bgl strain to make lysate (donor strain). Colony PCR of the colonies from 26/7 that grew on sucrose but not on chlor and gel electrophoresis to check inserts. Make tet + chlor and tet + tmp plates. Start overnight cultures of IS150 strains with inserts that were confirmed by colony PCR.
– Colony PCR of gsp strain and gel electrophoresis</P></DIV></DIV></DIV></DIV><DIV class="item"><DIV class="container-fluid"><DIV class="row entry"><DIV class="col-sm-4"><H3>Week: 31/07–4/08</H3><P>
– To confirm the presence of genes in the colonies, Screening was performed by restriction digestion and electrophoresis.
– Troubleshooting of colony PCR was done with commercial Taq polymerase master mix. 
– Biobrick from 2010 Slovenia Team containing 5 enzymes <I>crtEBIYZ</I> was transformed to <I>E. coli</I> by heat shock.
– Fresh batches of 3 selected positive colonies (one for each enzyme) were grown overnight in TB and lysed for sonication. Lysates were purified by IMAC.
- Eluted fractions were subjected to SDS-PAGE.
- The three colonies with recombinant plasmids were grown and plasmid prep was done. Restriction digestion and gel electrophoresis was done for screening.
- Fresh, presumably positive colonies (white colonies) were selected from master plates and re-streaked. Colony PCR was done for them followed by electrophoresis
- Gel chromatography and SDS was done for IMAC purified elutes. 
- A second round of screening by digestion was done with modifications as the plasmids and the construct were of similar size therefore needed better resolution. 
- Making all the constructs in pSB1C3 for biobrick submission.</P></DIV><DIV class="col-sm-4"><H3>Lambda red integration:</H3><P>
– Start overnight culture of bgl with chlor for lysate.
– Make lysate for transduction with bgl strain as donor. Start overnight culture of gsp strain (to make competent the next day) and one of the colonies from 27/7 of the IS150 strain that had <I>crtI</I> insert (to have as receiver for transduction next day).
– Finish lysate with bgl strain as donor. Make competent cells of gsp strain. Transduction of bgl strain as the donor and IS150 (prepared 1/8) as receiver and streak on tet+chlor plates.
– No colonies on transduced plates from 2/8. Electroporation of dhfr insert in gsp strain.</P></DIV></DIV></DIV></DIV><DIV class="item"><DIV class="container-fluid"><DIV class="row entry"><DIV class="col-sm-4"><H3>Week: 7/08–11/08</H3><H3>Lambda red integration:</H3><P>
– PCR of <I>crtEB</I> insert. Buffer/media preparation. Redo transduction from 2/8 and electroporation from 3/8. Start overnight culture of bgl with chlor for lysate and IS150 strain that had <I>crtI</I> insert (to have as receiver for transduction next day). 
– Gel electrophoresis of <I>crtEB</I> PCR product from 8/8 (see figure 8). Make lysate for transduction with bgl strain as donor (in 30 °C and 37 °C). Transduction of bgl strain (lysate made 2/8) as the donor and IS150 (prepared 8/8). Start overnight culture of the IS150 (to have as receiver for transduction next day).
– No colonies on transduced plates from 9/8. Make plates and SOB. Finish lysate (30 °C and 37 °C) from 9/8. Purify <I>crtEB</I> from PCR 9/8. Electroporation of <I>dhfr</I> insert in gsp strain and <I>crtZY</I> in gsp strain. Transduction of bgl strain (lysate made same day) as the donor and IS150 (prepared 9/8).
– No colonies on transduced plates from 10/8. Streak electroporated gsp strains from 10/8 on tet+tmp+chlor plates for strain with <I>dhfr</I> and tet+suc plates for <I>crtZY</I>). </P></DIV><DIV class="col-sm-4"><H3>Week: 14/8–18/8</H3><H3>Lambda red integration:</H3><P>
– Electroporation of <I>crtEB</I> insert in IS150 strain and <I>crtZY</I> in gsp strain. Start overnight culture of gsp with <I>crtZY</I> + <I>dhfr</I> and IS150 with <I>crtI</I> + <I>crtEB</I>.
– Make lysate for transduction with gsp strain as donor. Transduction of bgl strain (lysate made same day) as the donor and IS150 (prepared 14/8).
– No colonies on transduced plates from 15/8.</P></DIV></DIV></DIV></DIV><DIV class="item"><DIV class="container-fluid"><DIV class="row entry"><DIV class="col-sm-4"><H3>Week: 21/8–27/8</H3><H3>Lambda red integration:</H3><P>
– P1 transduction of bgl::<I>cat-sacB</I> into the IS150::<I>CP25-crtI</I> strains + wt control + cam control
Incubate 30 minutes in 37 °C
Plate <I>crtI</I> recipients on cam+tet, incubate in 30 °C
One control on only cam, 30 °C
Plate wt control on cam, incubate in 30 °C
Run screening PCR of IS150::<I>crtI</I> and gsp::<I>crtYZ</I> strains
– Gel analyze <I>crtI</I> and <I>crtYZ</I> screening – all clones correct size!
Purify PCR products <I>crtI</I> and <I>crtYZ</I>
Check transduction plates – the transduction worked well, more than 100 colonies per plate.</P></DIV><DIV class="col-sm-4"><P>
– Run screening PCR of IS150::<I>crtI</I> and gsp::<I>crtYZ</I> strains
Gel analyze PCR products<I>crtI</I> screening (4 wells) – weak bands<I>crtYZ</I> screening (3 wells) – good bands
Start overnight cultures <I>crtI</I> + bgl::<I>csb</I>
– Purify PCR products
Send for sequencing<I>crtI</I> – use IS150_seq_2, crtI_scr_1 and crtI_scr2 (IS150_scr_R)<I>crtYZ</I> – use crtZY_scr_1,crtZY_scr_2 and gsp_scr_R (gsp_ins_R)
Freeze <I>crtI</I> + bgl::<I>cat-sacB</I> strains + streak them on plate and leave in RT over the weekend</P></DIV></DIV></DIV></DIV><DIV class="item"><DIV class="container-fluid"><DIV class="row entry"><DIV class="col-sm-4"><H3>Week: 28/8–3/9</H3><P>
– Ordered more UGTCs2 from IDT
– linearised iGEM plasmid pSB1C3-J04500 using Phusion PCR and transformed CsADH2946 construct into the plasmid using Gibson assembly.
– Transformed into commercial competent TOP10 <I>E. coli</I> cells. </P><H3>Lambda red integration:</H3><P>
– Sequencing results showed that the clones are correct.
Streak up from -80 °C
Eco gsp::<I>crtZY</I> /pSIM5-tet clone 1</P></DIV><DIV class="col-sm-4"><P>
– Start overnight cultures of <I>crtI</I> and <I>crtZY</I> strains (Salt free LB + tet, 30 °C)
Eco gsp::crtZY /pSIM5-tet clone 1
Eco IS150::<I>crtI</I> bgl::<I>cat-sacB</I> /pSIM5-tet 
– Lambda Red
Eco IS150::<I>crtI</I> bgl::<I>cat-sacB</I> /pSIM5-tet
+ <I>crtEB</I> (PCR product)
Eco gsp::<I>crtZY</I> /pSIM5-tet
+ gspC::<I>dhfr</I> (PCR product)
– Plate cells Lambda Red
Eco IS150::<I>crtI</I> bgl::<I>cat-sacB</I> /pSIM5-tet + <I>crtEB</I>
Plate on sucrose + tet
Eco gsp::<I>crtZY</I> /pSIM5-tet + gspC::<I>dhfr</I>
Plate on tmp
- Restreak transductants lambda Red – leave in RT over the weekend
Eco IS150::<I>crtI</I> bgl::<I>cat-sacB</I> /pSIM5-tet + <I>crtEB</I>
Plate on sucrose + tet
Eco gsp::<I>crtZY</I> /pSIM5-tet + gspC::<I>dhfr</I>
Plate on tmp</P></DIV></DIV></DIV></DIV><DIV class="item"><DIV class="container-fluid"><DIV class="row entry"><DIV class="col-sm-4"><H3>Week: 4/9–10/9</H3><P>
– Colony PCR and ran gel electrophoresis for validation of insertion. Sent for sequencing.
– Transformed BL21(DE3)* with sequence verified CsADH2946 plasmid. Large scale expression and cell harvest.</P></DIV><DIV class="col-sm-4"><H3>Lambda red integration:</H3><P>
– Start overnight cultures of Lambda Red transformants
Eco IS150::<I>crtI</I> bgl::<I>crtEB</I> /pSIM5-tet
Grow in LB + tet in 30 °C (1.5 ml, will also use for transduction)
Eco gsp::<I>crtZY</I> gspC::<I>dhfr</I>
Grow in LB + tmp in 37 °C (1.5 ml, will also use for P1 lysate)
– Make P1 lysate (put in fridge after lysis)
Eco gsp::<I>crtZY</I> gspC::<I>dhfr</I>
Freeze overnight cultures of lambda Red transformants
Eco IS150::<I>crtI</I> bgl::<I>crtEB</I> /pSIM5-tet
Eco gsp::<I>crtZY</I> gspC::<I>dhfr</I></P></DIV></DIV></DIV></DIV><DIV class="item"><DIV class="container-fluid"><DIV class="row entry"><DIV class="col-sm-4"><H3>Week: 11/9–17/9</H3><P>
- Purification of CsADH2946 on IMAC</P><H3>Lambda red integration:</H3><P>
– Screening of <I>crtEB</I> clones, run screening PCR, gel analysis</P></DIV><DIV class="col-sm-4"><H3>Week: 18/9–24/9</H3><P>
– SDS-PAGE of CsADH2946 IMAC fractions.
– Changed buffer and concentrated pooled enzyme fractions.
– Activity measurements of CsADH2946.
– Created BioBricks of UGTCs2 by inserting it into the linearised iGEM plasmid pSB1C3-J04500 using Gibson Assembly.</P><H3>Lambda red integration:</H3><P>
– Screening of <I>crtEB</I> clones
Wrong band size! Redo lambda red.</P></DIV></DIV></DIV></DIV><DIV class="item"><DIV class="container-fluid"><DIV class="row entry"><DIV class="col-sm-4"><H3>Week: 25/9–1/10</H3><P>
– Colony PCR of UGTCs2, gel electrophoresis and send for sequencing.
– Transformation into BL21(DE3)* for expression
– Ordered more CaCCD2 construct from IDT.
– Sat overnight culture with UGTCs2 and performed expression testing with different concentrations of IPTG.
– Harvested cells and performed IMAC purification on ÄKTA. Ran fractions on SDS-gel. No significant bands could be seen.</P></DIV><DIV class="col-sm-4"><H3>Week: 2/10–8/10</H3><P>
– Inserted construct of CaCCD2 into plasmid with Gibson Assembly.
– Colony PCR of CaCCD2, gel electrophoresis and send for sequencing. 
– Expression testing of UGTCs2 and CaCCD2 with different concentrations of IPTG. SDS-PAGE on the crude lysates.</P><H3>Lambda red integration:</H3><P>
– Start cultures for Lambda Red
Eco IS150::<I>crtI</I> bgl::<I>cat-sacB</I> /pSIM5-tet
– Lambda red
Eco IS150::<I>crtI</I> bgl::<I>cat-sacB</I> /pSIM5-tet
+ <I>crtEB</I>
– Plate lambda red, incubate in RT over weekend
Eco IS150::<I>crtI</I> bgl::<I>cat-sacB</I> /pSIM5-tet<I>crtEB</I> plate on sucrose agar (4 plates)</P></DIV></DIV></DIV></DIV><DIV class="item"><DIV class="container-fluid"><DIV class="row entry"><DIV class="col-sm-4"><H3>Week: 9/10–15/10</H3><P>
– Set up pulling simulations for CsADH2946
– Combining CaCCD2 and CsADH2946 into one BioBrick using 3A assembly.
– Sent for sequencing.</P></DIV><DIV class="col-sm-4"><H3>Lambda red integration:</H3><P>
– Restreak <I>crtEB</I> clones – There are a few colonies that look reddish
16 clones on 4 LA agar plates
– Restreak crtEB transformants – red colonies.
They grow really bad, and look quite sick.
Transduce <I>crtYZ</I> into <I>crtEB</I> strain with P1
Clone 1–1 and 2–1–1.
– Restreak 6 clones of the <I>crtYZ</I> + <I>crtEB</I> transductants on tmp agar
Start overnight cultures of restreaked <I>crtEB</I> clones in 30 °C
4 clones started
– Freeze overnight cultures of restreaked <I>crtEB</I> clones in 30 °C
4 clones frozen
Restreak crtZY transductants on tmp agar
Clone 1–1 and 2–1–1
Restreak <I>crtZY</I> transductants a second time on LB agar
Clone 1–1 and 2–1–1
– Move plates to fridge or RT
Crt strains from 30 °C</P></DIV></DIV></DIV></DIV><DIV class="item"><DIV class="container-fluid"><DIV class="row entry"><DIV class="col-sm-4"><H3>Week: 16/10–22/10</H3><P>
– Prep of UGTCs2, CsADH2946 and CaCCD2 for BioBrick submission.
– Assembled J04500-CaCCD2 and J04500-CsADH2946 on pSB1K3 plasmid using 3A assembly into zeaxanthin producing <I>E. coli</I> strain.
– Transformed zeaxanthin strain with the assembled plasmid and another batch with plasmid containing only CaCCD2.
– Extracted zeaxanthin from the zeaxanthin producing strain</P><H3>Lambda red integration:</H3><P>
– Take photos of <I>crtEBIZY</I> strains
Start overnight cultures<I>crtEBIZY</I> strains (LB, 30 °C)
– Freeze overnight cultures<I>crtEBIZY</I> strains (LB, 30 °C)</P></DIV><DIV class="col-sm-4"><H3>Week: 23/10–29/10</H3><P>
– Large scale expression of zeaxanthin strain containing CaCCD2 + CsADH2946.
– Assembled J04500-CaCCD2-J04500-CsADH2946 and J04500-UGTCs2 on pSB1A3 using 3A assembly into our zeaxanthin producing <I>E. coli</I> strain.
– TLC comparing cultures of different zeaxanthin strain transformations (Zeaxanthin strain only, zeaxanthin strain with CaCCD2, zeaxanthin strain with CaCCD2 + CsADH2946 and zeaxanthin strain with CaCCD2 + CsADH2946 + UGTCs2).</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>