Team:Uppsala/Design
Home
Team
Team
Collaborations
Attributions
Project
Description
Design
Crocin Pathway
Zea-Strain
Modeling
Parts
Human Practices
Human Practices
Ethics-Gold HP
Engagement
Market Analysis
Results
Results
Demonstrate
Medals
Portfolio
Extended Work
Chipgineering2.0
InterLab
Improve
Contributions
Safety
The saffron biosynthetic pathway is an extension of the β-carotene pathway, with zeaxanthin being a key intermediate (1) . The pathway from farnesyl pyrophospate (FPP) to zeaxanthin (figure 1) has been BioBricked before, but we wanted to extend the β-carotene pathway to continue from zeaxanthin to crocin.
Figure 1. The pathway from farnesyl pyrophospate to zeaxanthin.
Integrating the FPP to Zeaxanthin pathway in the Chromosome
Zeaxanthin has previously been expressed in E. coli by iGEM teams using a plasmid. We decided to integrate this pathway into the chromosome using the lambda red recombineering method . This would give our project several advantages such as releasing all the plasmid origins and cassettes which would make the insertion of the crocin pathway genes or any other genes of a pathway that originates from zeaxanthin easier. It would make the strain more stable because no constant selective pressure is needed and makes it possible to introduce larger constructs and longer pathways. This also means that there is no need for the use of antibiotics which makes the purification process easier, especially if the product is later used for nutritional purposes. And of course since the first step of our zeaxanthin pathway – farnesyl pyrophosphate – is endogenous to E. coli we would be able to express the whole pathway from farnesyl pyrophosphate to crocin with no need for costly intermediates.
If we use cat-sacB selection/counterselection in lambda red recombineering (which we did) we get a scarless method that does not leave behind resistance markers. Lambda red is based on homologous recombination which is mediated by bacteriophage lambda proteins and usually requires only about 35 base pairs of homology on both sides of the inserted gene to work.
The pathway that leads from FPP to zeaxanthin includes five genes: crtE, crtB, crtI, crtY and crtZ. All these genes have been previously BioBricked into an operon for E. coli. Due to the time limitations of the project we decided to synthesize two constructs each containing two of the genes – crtEB and crtZY. For successful integration with Lambda red recombination the inserts should not exceed the size of 3000 bp and that is why we could not synthesize all of the genes as a single construct. Each construct also contained promoter, ribosome-binding sites and the necessary homologies for the future integration into the strains. The crtI gene was amplified from a BioBrick (iGEM Slovenia 2010), without the zinc finger domain, with the primers that contained the homologous sites.
The E. coli strains that were used for recombination contained a pSIM5-tet temperature sensitive plasmid with the lambda red system and tetracycline resistance. When temperature rises above 37 °C the lambda red enzymes are expressed. They also had cat-sacB selection cassette (based on Uppsalas own BioBrick BBa_K864150 ) carrying chloramphenicol resistance gene and Bacillus subtilis levansucrase sacB gene that is lethal for gram-negative bacteria when expressed in presence of sucrose.
Three different starting strains were used during the experiments. We also used three different promoters to avoid homology among these regions. The strain genotypes were:
1. Eco ∆gsp::cat-sacB /pSIM5-tet
2. Eco ∆bglGFB::cat-sacB /pSIM6
3. Eco ∆IS150::CP25-cat-sacB /pSIM5-tet
Lambda red recombineering was performed in three steps. For more detailed information, please, see the protocol for Lambda red and the Lab Notebook .
Step 1
Phytoene dehydrogenase gene crtI was amplified with PCR from zeaxanthin BioBrick and the homologies were added with the primers. Lambda red was performed to replace cat-sacB-T0 selection cassette with crtI, and the cells were plated on sucrose agar plates to select for transformants that had successfully integrated the crtI gene.
Eco ∆IS150::CP25-cat-sacB-T0 /pSIM5-tet + crtI (PCR product) →  Lambda Red (sucR) →  Eco ∆IS150::CP25-crtI /pSIM5-tet
A construct with beta-carotene hydroxylase gene crtZ and lycopene cyclase gene crtY was inserted with lambda red into another E. coli strain.
Eco ∆gsp::cat-sacB /pSIM5-tet + crtZY (PCR product) → Lambda Red (sucR) →  Eco ∆gsp::pFAB70-crtZY /pSIM5-tet
Step 2
The trimethoprim resistance gene dhfr (dihydrofolate reductase) was inserted with lambda red into the strain that now contained crtZY genes. The genes we placed very close to each other to enable the following transduction with P1 bacteriophage.
Eco ∆gsp::pFAB70-crtZY /pSIM5-tet + dhfr (PCR product) →  Lambda Red (tmpR) →  Eco ∆gsp::pFAB70-crtZY gspC::dhfr
Transduction with P1 bacteriophage was done to transfer cat-sacB-T0 selection cassette from a donor strain into the strain containing crtI gene.
Eco ∆IS150::CP25-crtI /pSIM5-tet + ∆bglGFB::cat-sacB (P1 lysate) →  Transduction (camR) →  Eco ∆IS150::CP25-crtI ∆bgl::cat-sacB /pSIM5-tet
Step 3
Lambda red was performed to replace this newly inserted cat-sacB-T0 cassette with a construct containing geranylgeranyl diphosphate synthase gene crtE and phytoene synthase gene crtB. The clones with the correct sequence changed color to light red. Our lycopene producing strain was constructed!
Eco ∆IS150::CP25-crtI ∆bgl::cat-sacB /pSIM5-tet + pFAB46-crtEB (PCR product) → Lambda Red (sucR) →  Eco ∆IS150::CP25-crtI ∆bgl::pFAB46-crtEB →  Lycopene!
Another transduction with P1 bacteriophage was done with the strain containing crtZY as a donor strain and the strain containing crtEBI as a recipient strain. The clones with the correct sequence turned bright yellow.
Eco ∆IS150::CP25-crtI ∆bgl::pFAB46-crtEB + ∆gsp::pFAB70-crtZY gspC::dhfr (P1 lysate) →  Transduction (tmpR) →  Eco IS150::CP25-crtI ∆bgl::pFAB46-crtEB ∆gsp::pFAB70-crtZY gspC::dhfr →  Zeaxanthin
Plan to Extend the pathway from Zeaxanthin to Crocin
To extend the pathway for the conversion of zeaxanthin to crocin , we needed to add the three step pathway, catalyzed by three different enzyme classes:
Carotenoid cleavage dioxygenases (CCD) are responsible for the symmetric cleavage of zeaxanthin at the 7,8/7′,8′ positions to form crocetin dialdehyde from zeaxanthin (2) .
Aldehyde dehydrogenases (ALDH) converts the 20 carbon cleavage product, crocetin dialdehyde to crocetin (3) .
UDP-glucuronosyltransferase (UGT) catalyzes the glucuronidation reaction of forming crocin from crocetin (3) .
Next we researched for the most appropriate enzymes under the above enzyme classes to execute the successful conversion of all the intermediates to crocin. We found Crocus Ancyrensis carotenoid cleavage dioxygenase 2 (CaCCD2) (4) , Crocus Sativus aldehyde dehydrogenase 2946 (CsADH2946) (3) and Crocus Sativus UDP-glucuronosyltransferase 2 (UGTCs2) (5) to be the most promising for our project.
The genes of the three individual enzymes were identified and synthesized as a gBlocks by IDT. Prior to synthesis, the genes were also codon optimized for E. coli, to get the desired overexpression of our enzymes. An N-terminal his-tag was added to aid downstream purification process. This was done after drawing conclusions from the modeling results that predicted the N-terminal to be outside of the enzymes and far from the active sites after folding. The promoter used was an inducible promoter BBa_J04500 from the iGEM kit. The enzymes were created into BioBricks using plasmid pSB1C3-J04500 with IPTG-inducible expression. The plasmid was linearised with Phusion PCR and the constructs were inserted into the iGEM plasmid using Gibson assembly . We transformed the plasmid into E. coli (TOP10 competent cells), screened the colonies using colony PCR and run gel electrophoresis to validate that the insert had been successfully assembled into the plasmid. We also created Biobricks with a part for RFP expression.
References
1. Ángela, L. G., and Ahrazem, R. O. 2010. Understanding Carotenoid Metabolism in Saffron Stigmas : Unravelling Aroma and Colour Formation. Functional Plant Science and Biotechnology 4, 56–63.
2. Frusciante S, Diretto G, Bruno M, Ferrante P, Pietrella M, Prado-Cabrero A, et al. Novel carotenoid cleavage dioxygenase catalyzes the first dedicated step in saffron crocin biosynthesis. Proceedings of the National Academy of Sciences. 2014 Aug 19;111(33):12246–51.
3. Gómez-Gómez L, Parra-Vega V, Rivas-Sendra A, Seguí-Simarro JM, Molina RV, Pallotti C, et al. Unraveling Massive Crocins Transport and Accumulation through Proteome and Microscopy Tools during the Development of Saffron Stigma. Int J Mol Sci [Internet]. 2017 Jan 1 [cited 2017 Oct 29];18(1). Available from: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5297711/
4. Ahrazem O, Rubio-Moraga A, Berman J, Capell T, Christou P, Zhu C, et al. The carotenoid cleavage dioxygenase CCD2 catalysing the synthesis of crocetin in spring crocuses and saffron is a plastidial enzyme. New Phytol. 2016 Jan 1;209(2):650–63.
5. Moraga AR, Nohales PF, Pérez JAF, Gómez-Gómez L. Glucosylation of the saffron apocarotenoid crocetin by a glucosyltransferase isolated from Crocus sativus stigmas. Planta. 2004 Oct;219(6):955–66.
