Team:Uppsala/Medals
Home
Team
Team
Collaborations
Attributions
Project
Description
Design
Crocin Pathway
Zea-Strain
Modeling
Parts
Human Practices
Human Practices
Ethics-Gold HP
Engagement
Market Analysis
Results
Results
Demonstrate
Medals
Portfolio
Extended Work
Chipgineering2.0
InterLab
Improve
Contributions
Safety
Bronze
Register and attend
Our team has been part of the iGEM competition and had an amazing summer developing our project. 13 members of our team are registered to attend the Giant Jamboree in Boston and could not be more excited about it.
Deliverables
We have completed and submitted all 8 required deliverables on time.
Attribution
We have many people and institutions we are thankful for this year. Special thank you belongs, of course, to our university that provided us with finances and space to execute the project. The many others who deserve a great thanks can be found at our Attributions page.
Characterization/Contribution
This year our team decided to both participate in Interlab Measument Study and to characterize an existing biobrick from iGEM Slovenia 2010 which has been an inspiration to our project. You can read more about the characterization in the Contribution page of our project.
Silver
Validated part
One base of our project was to extend the zeaxanthin pathway by introduction of three enzymes into E. Coli. We validated all three enzyme BioBricks by sequencing ( BBa_K2423005 and BBa_K2423008 ). The BioBrick with CsADH (BBa_K2423007) was, is in addition to sequencing verified by SDS-PAGE and activity assay. You can read more in our Results page .
Collaboration
Our team takes pride in collaborating with other teams and getting insight from the iGEM community. This year our most notable collaboration was a panel discussion we organized with iGEM Stockholm where more than 100 people participated. But we also worked with iGEM Lund to help them with their first year, organized webinars, made postcard and more! You can read about our collaborations on our Collaborations page.
Human Practices
This year we decided to focus most of our Human Practice on ethics in science and having fun as a team. We started off by doing a market analysis to see the potential market value. The Ethics package we constructed can be found on our Ethics page, here it can be seen how we put our package to practice and where we looked at different ethical aspects of our project idea and decided accordingly. You can read about everything on our Human Practice page.
Gold
Integrated Human Practices
As mentioned before, we developed something our team calls The Ethics Package. In this we first went to experts to discuss ethical issues of our project. Then we talked to the iGEM community in our webinars and lastly we reached out to general public in our panel discussion and lectures. Out of all these parts and our own research we compiled Ethical guidelines for iGEM teams with a list of questions that every iGEM team should answer to ethically asses their project. You can read more about our Ethics Package in our Ethics page.
Improve a previous Part or Project
This year we were working with several parts from previous iGEM projects. Most notably with Zeaxanthine strain from iGEM Slovenia 2010 (BBa_K323122) . We decided that the production was too unstable in such a large plasmid and adding more steps to it would be too risky. Therefore, we used lambda red recombeenering to add the parts into the chromosome. You can read more about this part of the project at the Zeaxanthin page.
Another improvement to a previous BioBrick is BBa_K2423008 which is an improved version of BBa_K1033112 developed by iGEM Uppsala 2013. We decided to optimize codons of the BioBrick, add His-tag linker to ensure proper folding and easier purification and to add two stop codons to make transcription smoother and more effective. You can read more in our Results page . Specific details on our improvements can also be found on our You can read more in our Improve page .
Model your project
The enzymes we identified for the crocin pathway were poorly characterized without determined structures. In addition, we were only able to find enzymatic parameters concerning one of the enzymes, UGTCs2. Via homology modeling we could draw the conclusion that UGTCs2 and CaCCD2 are monomers while CsADH2946 is a tetramer. We could also ensure that the N-terminal was facing outward from the protein and could safely put a His-tag there for purification. Furthermore, the extensive 100 ns molecular dynamics  (MD) simulation of our models verified that the models were stable in solvent and that we could further work with the models to estimate the binding affinity between enzyme and substrate. Steered molecular dynamics simulations showed that CsADH2946 is specific to its substrate crocetin dialdehyde and that it has an affinity in the ÂµM range to the substrate. The estimation of KM from the experimental activity measurements also showed that CsADH2946 has a high affinity towards the substrate crocetin dialdehyde, in agreement with Steered Molecular Dynamics results. All modeling results show that CsADH2946 is a very good enzyme for usage in the crocin pathway and thus in our project. Read all about how modeling contributed to our project in the Modeling page
Demonstrate your work
We successfully integrated the first five steps of the crocin pathway from FPP to zeaxanthin into the E. coli chromosome. The result is a E. coli strain expressing zeaxanthin. We have created sequence verified BioBricks of our enzymes in the extended crocin pathway: CaCCD2, CsADH2946 and UGTCs2. We have also characterized these enzymes with experiments and simulations. We are the first to purify and confirm activity of CsADH2946 as well as measuring the kinetic parameters of the enzyme. We created and combined the zeaxanthin producing strain with a plasmid containing the extended crocin pathway which gave us an E. coli strain including the entire production pathway from FPP to crocin. Read all about this at the Demonstrate page, Zeaxanthin strain page, Crocin Pathway page and the Results page.
