<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Uppsala_Zea-Strain skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Uppsala/Zea-Strain</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav pull-right" style="align:center !important;"><LI><A href="https://2017.igem.org/Team:Uppsala">Home</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" role="button">Team
      	</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown">Project
      	</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/CrocinPathway">Crocin Pathway</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Zea-Strain">Zea-Strain</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Model">Modeling</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Parts">Parts</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown">Human Practices
      	</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/HP/Silver">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/HP/Gold_Integrated">Ethics-Gold HP</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Engagement">Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Entrepreneurship">Market Analysis</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown">Results
      	</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Medals">Medals</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Portfolio">Portfolio</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown">Extended Work
      	</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Hardware">Chipgineering2.0</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Improve">Improve</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Contribution">Contributions</A></LI><LI><A href="https://2017.igem.org/Team:Uppsala/Safety">Safety</A></LI></DIV></DIV><P><TITLE>Zeaxanthin</TITLE></P><DIV class="container-fluid"><DIV class="row"><DIV class="mainheader"> ZEAXANTHIN STRAIN </DIV></DIV><DIV class="row text"><DIV class="col-xs-10"><DIV style="padding-bottom:3%;"> In our project we chose to concentrate on the pathway that leads from farnesyl pyrophosphate (FPP) to crocin (figure 1). The whole pathway consists of eight genes that code for eight enzymes which might make integrating all of the genes into a plasmid and keeping the plasmid in the bacteria more difficult. Dividing the pathway and integrating the part that leads from FPP to zeaxanthin into the chromosome would both give us a stable zeaxanthin-producing <I>E. coli</I> strain and make performing the remaining steps easier.
</DIV><FIGURE class="figure"><FIGCAPTION class="figure-caption figtext" style="padding-bottom: 2%; text-align:center;"> Figure 1. The pathway from farnesyl pyrophospate to zeaxanthin.
</FIGCAPTION></FIGURE><DIV class="miniheader"> Resulting Zeaxanthin Producing <I>E. coli</I> Strain </DIV><DIV style="padding-bottom:3%">We created a zeaxanthin producing <I>E. coli</I> strain using lambda red recombineering, with the whole pathway from FPP to zeaxanthin integrated into the chromosome (figure 2), which identified by the yellow pigment. All of the steps were confirmed with PCR, gel electrophoresis and sequencing. We were able to extract and purify the expensive yellow zeaxanthin compound from our strain. After creating the zeaxanthin strain, we combined it with the plasmid containing the extended crocin pathway which gave us an <I>E. coli</I> strain including the entire production pathway from FPP to crocin. This will hopefully give other iGEM teams more freedom to work with and build on carotenoid pathways and make zeaxanthin more affordable to use in experiments.</DIV><FIGURE class="figure"><FIGCAPTION class="figure-caption figtext" style="padding-bottom: 2%; text-align:center;"> Figure 2. Top: Wild-type <I>E. coli</I>. Bottom: Zeaxanthin producing <I>E. coli</I> strain with 5 genes inserted into the chromosome.</FIGCAPTION></FIGURE><DIV style="padding-bottom:3%"> Zeaxanthin has previously been expressed in <I>E. coli</I> by iGEM teams using a plasmid. We decided to integrate this pathway into the chromosome using the <A href="https://2017.igem.org/Team:Uppsala/Experiments">Lambda red recombineering method</A>. This would give our project several advantages such as releasing all the plasmid origins and cassettes which would make the insertion of the crocin pathway genes or any other genes of a pathway that originates from zeaxanthin easier. It would make the strain more stable because no constant selective pressure is needed and makes it possible to introduce larger constructs and longer pathways. This also means that there is no need for the use of antibiotics which makes the purification process easier, especially if the product is later used for nutritional purposes. And of course since the first step of our zeaxanthin pathway – farnesyl pyrophosphate – is endogenous to <I>E. coli</I> we would be able to express the whole pathway from farnesyl pyrophosphate to crocin with no need for costly intermediates. You can read about the design and details of the zeaxanthin strain production <A href="https://2017.igem.org/Team:Uppsala/Design">here</A>.</DIV><DIV style="padding-bottom:3%"><B>We got our zeaxanthin producing strain with the whole pathway from FPP to zeaxanthin integrated into the chromosome!</B> All of the steps were confirmed with PCR, gel electrophoresis and sequencing. We were able to extract and purify the expensive yellow zeaxanthin compound from our strain (figure 3).</DIV><FIGURE class="figure" style="padding-left:20%"><FIGCAPTION class="figure-caption figtext" style="padding-bottom: 3%; padding-left:0%"> Figure 3. Left: Large scale expression of zeaxanthin from the zeaxanthin producing <I>E. coli</I> strain. Right: Extracted and purified zeaxanthin.</FIGCAPTION></FIGURE><DIV style="padding-bottom:3%"> Besides observing the extracted zeaxanthin by eye we performed an absorbance measurement in the UV-Vis spectra (figure 4). Here we compared the absorbance spectra after <A href="https://2017.igem.org/Team:Uppsala/Experiments">two-phase extraction</A> from the zeaxanthin strain, from wildtype <I>E. coli</I> and a zeaxanthin standard. The zeaxanthin strain had two peaks at 460 and 482 nm which were not present in wildtype <I>E. coli</I>. These peaks were also present in the standard, therefore we can conclude that our produced strain produces zeaxanthin. For the measurements the extracted compounds were dissolved in toluene.</DIV><FIGURE class="figure"><FIGCAPTION class="figure-caption figtext" style="padding-bottom: 2%; text-align:center;"> Figure 4. Absorbance spectra for the extraction of zeaxanthin. MG1665 constitutes the negative control (the same extraction protocol on wildtype <I>E. coli</I>).</FIGCAPTION><DIV class="miniheader">Combining the Zeaxanthin Producing Strain and the Crocin Pathway Enzymes</DIV><DIV style="padding-bottom:3%">
After creating the zeaxanthin producing strain, we combined it with the plasmid containing the extended crocin pathway which gave us an <I>E. coli</I> strain including the entire production pathway from FPP to crocin. The <A href="https://2017.igem.org/Team:Uppsala/CrocinPathway">three enzyme BioBricks</A> BBa_K2423005, BBa_K2423007 and BBa_K2423008 in the zeaxanthin-crocin pathway were assembled to one plasmid (pSB1A3) using <A href="https://2017.igem.org/Team:Uppsala/Experiments">3A assembly</A> and was inserted into the zeaxanthin producing <I>E.coli</I> strain using <A href="https://2017.igem.org/Team:Uppsala/Experiments">electroporation</A>. The resulting plate can be seen in figure 5. </DIV><FIGURE class="figure"><FIGCAPTION class="figure-caption figtext" style="padding-bottom: 2%; text-align:center;"> Figure 5. Ampicilin plate with Zeaxanthin expressing <I>E. coli</I> strain transformed with pSB1A3 plasmid containing all three crocin pathway enzymes CaCCD2, CsADH2946 and UGTCs2.</FIGCAPTION></FIGURE><DIV style="padding-bottom:3%">
The color of the colonies changes slightly at each addition of another enzyme construct (another step in the crocin pathway). This is an indication that something is indeed happening with the bacterial pigment production when we introduce our pathway steps (figure 6) into the zeaxanthin strain. In the future it would be good to integrate the whole pathway from farnesyl pyrophosphate (FPP) to crocin into the chromosome for a stable crocin producing strain that does not require antibiotic selection which would make it easier to use as for example food coloring. </DIV><FIGURE class="figure"><FIGCAPTION class="figure-caption figtext" style="padding-bottom: 2%; text-align:center;"> Figure 6. Left: Chloramphenicol plate with Zeaxanthin expressing <I>E. coli</I> strain transformed with pSB1C3 plasmid containing CaCCD2. Right: Kanamycin plate with Zeaxanthin expressing <I>E. coli</I> strain transformed with pSB1K3 plasmid containing CaCCD2+CsADH2946 </FIGCAPTION></FIGURE><DIV style="padding-bottom:3%"> We wanted to analyze the compounds found after extraction of the zeaxanthin strains with and without plasmids from the crocin pathway. <A href="https://2017.igem.org/Team:Uppsala/Experiments">Thin layer chromatography (TLC)</A> was used to do this on a zeaxanthin strain, a zeaxanthin strain containing CaCCD2 (<A href="https://2017.igem.org/Team:Uppsala/Parts">BBa_K2423005</A>), a zeaxanthin strain containing a plasmid with both CaCCD2 and CsADH2946, standards for crocetin and crocetin dialdehyde and a wild-type <I>E. coli</I> strain. As you can see from the resulting TLC plate in figure 7, many different pigments (represented by the bands) are present in the extract from the zeaxanthin strain. No pigment bands are present in the extract from the negative control strain (wild type <I>E. coli</I>), showing that the pigments in the zeaxanthin strain are precursors to zeaxanthin and zeaxanthin itself. This is consistent with the absorbance measurement where zeaxanthin was seen to be present in the extract (Figure 4.). Multiple pigments are also formed by the zeaxanthin + CaCCD2 and zeaxanthin + CaCCD2 + CsADH2946 strains (Figure 7.). Due to the mix of pathway pigments found in the samples, and limitations in resolution in the TLC, identification of specific pigment cannot be made from this data. In the future it would be very interesting to uniquelly identify the identity of the different pigments formed by our strains using a more informative method such as High-performance liquid chromatography-mass spectrometry (HPLC-MS).
</DIV></FIGURE><FIGCAPTION class="figure-caption figtext" style="padding-bottom: 2%; text-align:center;"> Figure 7. Thin layer chromatography for zeaxanthin producing strains with and without BioBricks in the crocin pathway, standards for crocetin, crocetin dialdehyde and negative control strain (MG1665). Samples from left to right: zeaxanthin producing strain, zeaxanthin producing strain with CaCCD2 (<A href="https://2017.igem.org/Team:Uppsala/Parts">BBa_K2423005</A>) plasmid, zeaxanthin producing strain with a combined CaCCD2 and CsADH2946 plasmid, crocetin standard, crocetin dialdehyde standard and MG1665 (wild type <I>E. coli</I>).</FIGCAPTION></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>