Team:Uppsala/Zea-Strain
Home
Team
Team
Collaborations
Attributions
Project
Description
Design
Crocin Pathway
Zea-Strain
Modeling
Parts
Human Practices
Human Practices
Ethics-Gold HP
Engagement
Market Analysis
Results
Results
Demonstrate
Medals
Portfolio
Extended Work
Chipgineering2.0
InterLab
Improve
Contributions
Safety
Zeaxanthin
ZEAXANTHIN STRAIN
In our project we chose to concentrate on the pathway that leads from farnesyl pyrophosphate (FPP) to crocin (figure 1). The whole pathway consists of eight genes that code for eight enzymes which might make integrating all of the genes into a plasmid and keeping the plasmid in the bacteria more difficult. Dividing the pathway and integrating the part that leads from FPP to zeaxanthin into the chromosome would both give us a stable zeaxanthin-producing E. coli strain and make performing the remaining steps easier.
Figure 1. The pathway from farnesyl pyrophospate to zeaxanthin.
Resulting Zeaxanthin Producing E. coli Strain
We created a zeaxanthin producing E. coli strain using lambda red recombineering, with the whole pathway from FPP to zeaxanthin integrated into the chromosome (figure 2), which identified by the yellow pigment. All of the steps were confirmed with PCR, gel electrophoresis and sequencing. We were able to extract and purify the expensive yellow zeaxanthin compound from our strain. After creating the zeaxanthin strain, we combined it with the plasmid containing the extended crocin pathway which gave us an E. coli strain including the entire production pathway from FPP to crocin. This will hopefully give other iGEM teams more freedom to work with and build on carotenoid pathways and make zeaxanthin more affordable to use in experiments.
Figure 2. Top: Wild-type E. coli. Bottom: Zeaxanthin producing E. coli strain with 5 genes inserted into the chromosome.
Zeaxanthin has previously been expressed in E. coli by iGEM teams using a plasmid. We decided to integrate this pathway into the chromosome using the Lambda red recombineering method . This would give our project several advantages such as releasing all the plasmid origins and cassettes which would make the insertion of the crocin pathway genes or any other genes of a pathway that originates from zeaxanthin easier. It would make the strain more stable because no constant selective pressure is needed and makes it possible to introduce larger constructs and longer pathways. This also means that there is no need for the use of antibiotics which makes the purification process easier, especially if the product is later used for nutritional purposes. And of course since the first step of our zeaxanthin pathway – farnesyl pyrophosphate – is endogenous to E. coli we would be able to express the whole pathway from farnesyl pyrophosphate to crocin with no need for costly intermediates. You can read about the design and details of the zeaxanthin strain production here .
We got our zeaxanthin producing strain with the whole pathway from FPP to zeaxanthin integrated into the chromosome! All of the steps were confirmed with PCR, gel electrophoresis and sequencing. We were able to extract and purify the expensive yellow zeaxanthin compound from our strain (figure 3).
Figure 3. Left: Large scale expression of zeaxanthin from the zeaxanthin producing E. coli strain. Right: Extracted and purified zeaxanthin.
Besides observing the extracted zeaxanthin by eye we performed an absorbance measurement in the UV-Vis spectra (figure 4). Here we compared the absorbance spectra after two-phase extraction from the zeaxanthin strain, from wildtype E. coli and a zeaxanthin standard. The zeaxanthin strain had two peaks at 460 and 482 nm which were not present in wildtype E. coli. These peaks were also present in the standard, therefore we can conclude that our produced strain produces zeaxanthin. For the measurements the extracted compounds were dissolved in toluene.
Figure 4. Absorbance spectra for the extraction of zeaxanthin. MG1665 constitutes the negative control (the same extraction protocol on wildtype E. coli).
Combining the Zeaxanthin Producing Strain and the Crocin Pathway Enzymes
After creating the zeaxanthin producing strain, we combined it with the plasmid containing the extended crocin pathway which gave us an E. coli strain including the entire production pathway from FPP to crocin. The three enzyme BioBricks BBa_K2423005, BBa_K2423007 and BBa_K2423008 in the zeaxanthin-crocin pathway were assembled to one plasmid (pSB1A3) using 3A assembly and was inserted into the zeaxanthin producing E.coli strain using electroporation . The resulting plate can be seen in figure 5.
Figure 5. Ampicilin plate with Zeaxanthin expressing E. coli strain transformed with pSB1A3 plasmid containing all three crocin pathway enzymes CaCCD2, CsADH2946 and UGTCs2.
The color of the colonies changes slightly at each addition of another enzyme construct (another step in the crocin pathway). This is an indication that something is indeed happening with the bacterial pigment production when we introduce our pathway steps (figure 6) into the zeaxanthin strain. In the future it would be good to integrate the whole pathway from farnesyl pyrophosphate (FPP) to crocin into the chromosome for a stable crocin producing strain that does not require antibiotic selection which would make it easier to use as for example food coloring.
Figure 6. Left: Chloramphenicol plate with Zeaxanthin expressing E. coli strain transformed with pSB1C3 plasmid containing CaCCD2. Right: Kanamycin plate with Zeaxanthin expressing E. coli strain transformed with pSB1K3 plasmid containing CaCCD2+CsADH2946
We wanted to analyze the compounds found after extraction of the zeaxanthin strains with and without plasmids from the crocin pathway. Thin layer chromatography (TLC) was used to do this on a zeaxanthin strain, a zeaxanthin strain containing CaCCD2 ( BBa_K2423005 ), a zeaxanthin strain containing a plasmid with both CaCCD2 and CsADH2946, standards for crocetin and crocetin dialdehyde and a wild-type E. coli strain. As you can see from the resulting TLC plate in figure 7, many different pigments (represented by the bands) are present in the extract from the zeaxanthin strain. No pigment bands are present in the extract from the negative control strain (wild type E. coli), showing that the pigments in the zeaxanthin strain are precursors to zeaxanthin and zeaxanthin itself. This is consistent with the absorbance measurement where zeaxanthin was seen to be present in the extract (Figure 4.). Multiple pigments are also formed by the zeaxanthin + CaCCD2 and zeaxanthin + CaCCD2 + CsADH2946 strains (Figure 7.). Due to the mix of pathway pigments found in the samples, and limitations in resolution in the TLC, identification of specific pigment cannot be made from this data. In the future it would be very interesting to uniquelly identify the identity of the different pigments formed by our strains using a more informative method such as High-performance liquid chromatography-mass spectrometry (HPLC-MS).
Figure 7. Thin layer chromatography for zeaxanthin producing strains with and without BioBricks in the crocin pathway, standards for crocetin, crocetin dialdehyde and negative control strain (MG1665). Samples from left to right: zeaxanthin producing strain, zeaxanthin producing strain with CaCCD2 ( BBa_K2423005 ) plasmid, zeaxanthin producing strain with a combined CaCCD2 and CsADH2946 plasmid, crocetin standard, crocetin dialdehyde standard and MG1665 (wild type E. coli).
