<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_DTU-Denmark_Project_MAGE skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:DTU-Denmark/Project/MAGE</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><NAV class="navbar navbar-default navbar-fixed-top" role="navigation"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#nav-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="nav-collapse"><UL class="nav navbar-nav"><LI><A href="/Team:DTU-Denmark">Home
             </A></LI><LI><A href="" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Team and Attributions
                   </A></LI><LI><A href="/Team:DTU-Denmark/Team">Team
                   </A></LI><LI><A href="/Team:DTU-Denmark/Attributions">Attributions
                   </A></LI></UL><LI class="active"><A href="" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Project
                   </A></LI><LI><A href="/Team:DTU-Denmark/Project/Overview">Overview
                   </A></LI><LI><A href="/Team:DTU-Denmark/Project/Background">Background
                   </A></LI><LI class="active"><A href="/Team:DTU-Denmark/Project/MAGE">MAGE subtilis
                   </A></LI><LI><A href="/Team:DTU-Denmark/Project/Tyrocidine">Tyrocidine
                   </A></LI><LI><A href="/Team:DTU-Denmark/Project/Chip">Lab-on-a-disc
                   </A></LI><LI><A href="/Team:DTU-Denmark/Project/Inteins">Inteins
                   </A></LI><LI><A href="/Team:DTU-Denmark/Project/Detection">Detection of NRP
                   </A></LI><LI><A href="/Team:DTU-Denmark/Practices">Human Practices
                   </A></LI><LI><A href="" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Parts Collection
                   </A></LI><LI><A href="/Team:DTU-Denmark/Parts">Parts
                   </A></LI><LI><A href="/Team:DTU-Denmark/Description">Characterisation of xylR
                   </A></LI><LI><A href="/Team:DTU-Denmark/Journal">Journal
                   </A></LI><LI><A href="/Team:DTU-Denmark/Software">Software
                   </A></LI><LI><A href="" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Achievements
                   </A></LI><LI><A href="/Team:DTU-Denmark/Achievements">Key Achievements
                   </A></LI><LI><A href="/Team:DTU-Denmark/Collaborations">Collaborations
                   </A></LI><LI><A href="/Team:DTU-Denmark/Judging_Form">Judging Form
                   </A></LI><LI class="hidden-xs"><A href="http://facebook.com/dtubiobuilders" data-toggle="tooltip" data-placement="bottom" title="Follow us on Facebook"><SPAN class="visible-xs"> Follow us on Facebook</SPAN></A></LI><LI class="visible-xs"><A href="http://facebook.com/dtubiobuilders"> Follow us on Facebook</A></LI><LI class="visible-xs"><A href="https://twitter.com/IGEM_DTU"> Follow us on Twitter</A></LI></DIV></DIV></NAV><NAV class="navbar navbar-default extrabar hidden-xs hidden" data-spy="affix"><DIV class="container-fluid" id="scrollspy"><UL class="nav navbar-nav"><LI><A class="page-scroll" href="#Abstract">Abstract</A></LI><LI><A class="page-scroll" href="#MAGEcompetentBsubtilisstrains">MAGE competent B. subtilis strains</A></LI><LI><A class="page-scroll" href="#ProofofconceptofMAGEinBsubtilis">Proof of concept of MAGE in B. subtilis</A></LI><LI><A class="page-scroll" href="#OptimizationofMAGEinBsubtilis">Optimization of MAGE in B. subtilis</A></LI><LI><A class="page-scroll" href="#Surfactin">Surfactin</A></LI><LI><A class="page-scroll" href="#DilutionEquation">Dilution Equation</A></LI><LI><A class="page-scroll" href="#References">References</A></LI></UL></DIV></NAV><DIV class="jumbotron hero" style="background-image: url(https://static.igem.org/mediawiki/2015/0/0b/DTU-Denmark_hero.jpg)"><DIV class="container-fluid"><DIV class="row hero-text"><DIV class="col-md-12"><H1>MAGE subtilis</H1><UL class="list-inline hidden-xs hidden-sm"><LI><A id="Abstract-submenu" class="btn btn-default btn-transparent btn-lg page-scroll" href="#Abstract">Abstract</A></LI><LI><A id="MAGEcompetentBsubtilisstrains-submenu" class="btn btn-default btn-transparent btn-lg page-scroll" href="#MAGEcompetentBsubtilisstrains">MAGE competent B. subtilis strains</A></LI><LI><A id="ProofofconceptofMAGEinBsubtilis-submenu" class="btn btn-default btn-transparent btn-lg page-scroll" href="#ProofofconceptofMAGEinBsubtilis">Proof of concept of MAGE in B. subtilis</A></LI><LI><A id="OptimizationofMAGEinBsubtilis-submenu" class="btn btn-default btn-transparent btn-lg page-scroll" href="#OptimizationofMAGEinBsubtilis">Optimization of MAGE in B. subtilis</A></LI><LI><A id="Surfactin-submenu" class="btn btn-default btn-transparent btn-lg page-scroll" href="#Surfactin">Surfactin</A></LI><LI><A id="DilutionEquation-submenu" class="btn btn-default btn-transparent btn-lg page-scroll" href="#DilutionEquation">Dilution Equation</A></LI><LI><A id="References-submenu" class="btn btn-default btn-transparent btn-lg page-scroll" href="#References">References</A></LI></UL><UL class="list-inline visible-xs visible-sm"><LI><A id="Abstract-submenu" class="page-scroll" href="#Abstract">Abstract</A></LI><LI><A id="MAGEcompetentBsubtilisstrains-submenu" class="page-scroll" href="#MAGEcompetentBsubtilisstrains">MAGE competent B. subtilis strains</A></LI><LI><A id="ProofofconceptofMAGEinBsubtilis-submenu" class="page-scroll" href="#ProofofconceptofMAGEinBsubtilis">Proof of concept of MAGE in B. subtilis</A></LI><LI><A id="OptimizationofMAGEinBsubtilis-submenu" class="page-scroll" href="#OptimizationofMAGEinBsubtilis">Optimization of MAGE in B. subtilis</A></LI><LI><A id="Surfactin-submenu" class="page-scroll" href="#Surfactin">Surfactin</A></LI><LI><A id="DilutionEquation-submenu" class="page-scroll" href="#DilutionEquation">Dilution Equation</A></LI><LI><A id="References-submenu" class="page-scroll" href="#References">References</A></LI></UL></DIV></DIV><DIV class="row"><DIV class="col-md-12">
            Scroll down for more</DIV></DIV></DIV></DIV><DIV id="Abstract"><DIV class="container"><DIV class="row col-md-12"><H1>
        Abstract
      </H1><P style="text-align: justify;">We showed indications that we made a  MAGE competent strains of <EM>B. subtilis 168</EM>, in which we were able to introduce mutations using oligos. For a proof of concept three different approaches were tried, but only the last method turned out to be useful. In this approach, we took advantage of a point mutation in the ribosome of <EM>B. subtilis 168 </EM>which provides the strain streptomycin resistant. We were not able generate a sufficient amount of data to significantly proof our results. Experiments were carried out to optimize MAGE in <EM>B. subtilis 168, </EM>but these result were inconclusive<EM>. </EM>In spite of the unclear results, we decided to continue to see if we would be able to change the specificity of a NRPS module. We did get vague data suggesting that we were able to change the product of the native <EM>B. subtilis 168 </EM>nonribosomal peptide (NRP) - surfactin. Due to time constrains the strain was not sequence this mutant, so the successful substitution in surfactin is not confirmed.</P></DIV></DIV></DIV><DIV id="MAGEcompetentBsubtilisstrains"><DIV class="container"><DIV class="row col-md-12"><H1>
        MAGE competent B. subtilis strains
      </H1><H2>Overview</H2><P style="text-align: justify;">Compared to <EM>E. coli</EM>, the <EM>B. subtilis </EM>has more native NRPS’, therefore it has more NRP precursors natively available. Proof and optimization of MAGE in <EM>B. subtilis </EM>would be valuable knowledge for developing novel NRPS’. To proof the concept of MAGE in <EM>B. subtilis</EM> we designed oligos that could introduce a point mutation in the s12 subunit of the ribosome, inducing streptomycin resistance [1]. In order for MAGE (Multiplex Automated Genome Engineering) to work at a high efficiency, a strain with inserted recombinase and inhibited or knocked out mismatch repair gene has to be used [2]. In our project, two different recombinases were used: a recombination protein Beta from the <EM>E. coli</EM> phage Lambda, which was codon optimized for <EM>B. subtilis 168</EM>, and GP35, a recombinase from the B.subtilis phage SPP1 [3]. The mismatch repair proteins known as MutS and MutL were knocked out by transforming pSB1C3_recombinase plasmid into the <EM>B. subtilis W168</EM>. Since the MutL protein is dependent on the binding of MutS, the knockout of the mutS disables the function of the MutL protein [4].</P><P>Four <EM>Bacillus subtilis</EM> strains which expressed a recombinase were created by genetically engineering the wild type strain 168:</P><UL><LI><EM>∆amyE::beta-neoR</EM></LI><LI><EM>∆amyE::GP35-neoR</EM></LI><LI><EM>∆mutS::beta-neoR</EM></LI><LI><EM>∆mutS::GP35-neoR</EM></LI></UL><P>The growth of all the mutants was compared to the wild type to test for growth bias. The growth of <EM>∆mutS::GP35-neoR,</EM><EM>∆amyE::beta-neoR </EM>and <EM>∆amyE::GP35-neoR </EM>was shown to be faster growing than the wild type strain.</P><P> </P><H2>Achievements</H2><UL><LI>We made the following four <EM>B. subtilis </EM>strains

	</LI><LI><EM>∆amyE::beta-neoR</EM></LI><LI><EM>∆amyE::GP35-neoR</EM></LI><LI><EM>∆mutS::beta-neoR</EM></LI><LI><EM>∆mutS::GP35-neoR</EM></LI></UL><P> </P><H2>Methods</H2><P style="text-align: justify;">All the strain were made by homologous recombineering. For this purpose four different plasmids were assembled. Two plasmids contained homologous regions to up- and downstream of mutS and two plasmids containing homologous regions to the <EM>amyE </EM>locus. Thus, the plasmids are able to do a double-crossover into the genome of <EM>B. subtilis 168</EM> deleting the CDS of <EM>mutS </EM>or<EM>amyE </EM>from the genome. These regions were enclosing a neomycin resistance cassette carrying its own promoter, RBS and terminator (the exact position of these are unknown to us). Besides the neomycin resistance cassette, the mutS homologous regions were enclosing an expression cassette for a recombinase. Two different recombinases, GP35 and beta, were used resulting in two plasmids. The content of the expression cassette is shown in the following table.</P><P> </P><TABLE border="0" cellpadding="0" cellspacing="0" style="width:482px;" width="602"><TBODY><TR><TD><P>Feature</P></TD><TD><P>Name</P></TD><TD><P>Obtained form</P></TD></TR><TR><TD><P>Promoter</P></TD><TD><P>BBa_k823002</P></TD><TD><P><A href="http://parts.igem.org/Part:BBa_K823002">iGEM part registry </A></P></TD></TR><TR><TD><P>RBS</P></TD><TD> </TD><TD><P>Optimized for the recombinase CDS using the RBS calculator provided by https://salislab.net/software/</P></TD></TR><TR><TD><P>CDS</P></TD><TD><P>Beta or GP35</P></TD><TD><P>See below</P></TD></TR><TR><TD><P>Terminator</P></TD><TD><P>BBa_B0014</P></TD><TD><P><A href="http://parts.igem.org/Part:BBa_B0014">iGEM part registry</A></P></TD></TR></TBODY></TABLE><P>Tabel 1. The general structure of the recombinase expression cassette.</P><P> </P><DIV aria-multiselectable="true" class="panel-group" id="accordion" role="tablist"><DIV class="panel panel-default"><DIV class="panel-heading" id="headingOne" role="tab"><H4 class="panel-title"><A aria-controls="collapseOne" aria-expanded="false" class="collapsed" data-parent="#accordion" data-toggle="collapse" href="#collapseOne" role="button">Obtaining the recombinase CDS'</A></H4></DIV><DIV aria-labelledby="headingOne" class="panel-collapse collapse" id="collapseOne" role="tabpanel"><DIV class="panel-body"><DIV class="panel-body"><H3>Beta Protein</H3><P style="text-align: justify;">The sequence was obtained from GenBank (Id: KT232076.1), since this sequence is from an <EM>E. coli </EM>phage the sequence was codon optimized for <EM>B. subtilis 168 </EM>avoiding the restriction sites suggested by the iGEM REF10 standards. A TAA stop codon was added at the end  of the CDS.</P><P> </P><H3><STRONG>GP35</STRONG></H3><P style="text-align: justify;">As in Sun2015 this sequence was obtained from the genome of the Bacillus phage SPP1 (NCBI: NC_004166.2)[3]. The CDS for “gene 35” is from basepair 32175 to basepair 33038 in the genome of SPP1. The only alteration done to this CDS was that the native TAG stop codon was changed to two TAA stop codons because of iGEM preferences.</P></DIV></DIV></DIV></DIV><P style="text-align: justify;">FOr each of the two recombinases a DNA sequence containing promoter, RBS, CDS and terminator flacked by homologous regions to <EM>amyE</EM> integration plasmids pDG268neo for <EM>B. subtilis</EM> was designed. Each of these two sequences was splitted to two sequences with 20bp of overlapping sequence and ordered as four gblocks from IDT.</P><P> </P><H3>Cloning</H3><P style="text-align: justify;">The two correlated gblocks were cloned into pDG268neo using gibson assembly and transformed into <EM>E. coli</EM> - resulting in pDG268neo_beta and pDG268_GP35. The insert of the correct sequence were verified by sequencing.</P><P> </P><P style="text-align: justify;"><SPAN style="font-size:14px;">Figure 1. Representation of the inserted plasmid pDG268neo_recombinase. The plasmid exists with each of the recombinase proteins CDSs (Beta and GP35). They also have different RBSs since they are optimized for the CDS. The neoR cassette contains a promoter and RBS and a terminator, but sequences and positions of these features are not known.</SPAN></P><P> </P><P style="text-align: justify;">These two plasmids were transformed into <EM>B. subtilis 168 </EM>using natural competence and transformants was verified using colony PCR. Resulting in the following two strains:</P><UL><LI><EM>∆amyE::beta-neoR</EM></LI><LI><EM>∆amyE::GP35-neoR</EM></LI></UL><P style="text-align: justify;">For the construction of the remaining two strains, a DNA fragment containing the <EM>neoR</EM> cassette and the recombinase expression cassette was amplified by PCR from pDG268neo_beta and pDG268neo_GP35. This was carried out by primers with homologous tails for a mutS upstream fragment in one side and for a mutS downstream fragment in the other side. Appropriate mutS up- and downstream fragments were amplified from the <EM>B. subtilis</EM> genome, using a cPCR approach. This PCR was carried out by primers that contained tails homologous to the biobrick suffix and the biobrick prefix. The biobrick backbone was amplified from BBa_J04450, with primers that amplifies from the biobrick prefix to the biobrick suffix. Thus, the four fragments: <EM>recombinase-neoR, mutS upstream, mutS downstream </EM>and the linearized pSB1C3 could be assembled using gibson assembly into two different plasmids: pSB1C3_beta-neoR and pSB1C3_GP35-neoR, see Figure 2.</P><P><SPAN style="font-size:14px;">Figure 2. Representation of the inserted plasmids pSB1C3_recombinase after the inital insertion of the plasmid pDG268neo_recombinase. The plasmid exists with each of the recombinase proteins CDSs (Beta and GP35), They also have different RBSs since they are optimized for the CDS and the neoR cassette contains a promoter and RBS and a terminator, but sequences and positions of these features are not known.</SPAN></P><P> </P><P>The two plasmids were linearized by XhoI cutting out the <EM>cmR</EM> from the pSB1C3 backbone. Using natural competence the two linearized plasmids were transformed into <EM>B. subtilis 168. </EM>Inserts were verified by cPCR. Resulting in the last two strains:</P><UL><LI><EM>∆mutS::beta-neoR</EM></LI><LI><EM>∆mutS::GP35-neoR</EM></LI></UL><P> </P><P><STRONG><SPAN style="font-size:20px;">Growth Experiment</SPAN></STRONG></P><P>The protocol was followed for creation of MAGE competent Bacillus. The growth of the wild type B. subtilis 168 and all mutants was measured using OD<SUB>600</SUB> measurements. </P><P> </P><H2>Results and Conclusion</H2><H3>Construction of MAGE competent strains</H3><P>For results on the construction of the four MAGE competent strains take a look in our <A href="https://static.igem.org/mediawiki/2015/7/75/DTU-Denmark_straingeneration.pdf">lab-notebook</A>.</P><P> </P><P>Figure 3. Generation time of the different mutants measure in minutes.</P><P>The different mutants were compared to the wild type. From figure 3 it is clear that the generation time of the mutS::GP35 and the amyE mutants showed to be faster growing than the wild type. The mutS::beta was shown to be slower growing than the wild type, this mutant is the one that is best at recombineering. It is possible that the problem is the high constitutive expression of the beta protein in the cell that interferes with the growth of the cell. This could be solved by using an inducible promoter.</P><P> </P></DIV></DIV></DIV></DIV><DIV id="ProofofconceptofMAGEinBsubtilis"><DIV class="container"><DIV class="row col-md-12"><H1>
        Proof of concept of MAGE in B. subtilis
      </H1><H3>Overview</H3><P style="text-align: justify;">To establish proof-the-concept of MAGE in <EM>B. subtilis 168</EM> several methods were tested, but only one method turned out to be useable. The methods used for proofing MAGE in <EM>B. subtilis</EM> were as following:</P><OL><LI>Knockout of <EM>upp</EM></LI><LI>Knockout of <EM>amyE</EM></LI><LI>Introducing streptomycin resistance</LI></OL><P style="text-align: justify;">The ideal method was established to be the introduction of streptomycin resistance, allowing indication of successful oligo induced changes in the genome of <EM>B. subtilis</EM><EM>.</EM></P><P> </P><H3>Achievements</H3><UL><LI>Indications suggest a successful introduction of MAGE in <EM>B. subtilis.</EM></LI><LI>Indications identifies the beta-protein to be more efficient, than the GP35.</LI></UL><H3>Methods</H3><P style="text-align: justify;">The <A href="http://modest.biosustain.dtu.dk/" target="_blank">MODEST</A> program was used to design oligos for the experiments[5]. The oligo was designed to introduce a point mutation in the 12S subunit in the ribosome of <EM>B. subtilis 168, </EM>introducing streptomycin resistance [1]. Thus, it allows the selection of mutants growing on streptomycin. The four engineered strains and the wild type of <EM>B. subtilis 168</EM> were prepared electroporation competent and electroporated with the oligo. The recovering bacteria was diluted and spread onto LB plates and incubated overnight. Single colonies were screened for streptomycin resistance by streaking them onto both LB and LB + streptomycin and following replicaplated.</P><P> </P><DIV aria-multiselectable="true" class="panel-group" id="accordion22" role="tablist"><DIV class="panel panel-default"><DIV class="panel-heading" id="headingTwentytwo" role="tab"><H4 class="panel-title"><A aria-controls="collapseTwentytwo" aria-expanded="false" class="collapsed" data-parent="#accordion" data-toggle="collapse" href="#collapseTwentytwo" role="button"><EM>upp</EM> and <EM>amyE</EM> screening</A></H4></DIV><DIV aria-labelledby="headingTwentytwo" class="panel-collapse collapse" id="collapseTwentytwo" role="tabpanel"><DIV class="panel-body"><DIV class="panel-body"><P>To test MAGE compliance, an attempt was made to knockout the amyE and upp gens in <EM>B. subtilis</EM> by using oligoes to incorporate three stop codons in the coding sequences. Both <EM>upp</EM> and <EM>amyE</EM> selection showed to be unable to yield conclusive results.</P><H4><STRONG>Theory</STRONG></H4><P style="text-align: justify;"><STRONG><EM>amyE</EM></STRONG> is a gene in <EM>B. subtilis</EM> coding for a amylase protein that can degrade starch. Starch can be colored by an iodine in an ethanol solution and amylase activity is seen by a formed clearing zone due to the degraded starch.</P><P style="text-align: justify;">Knocking out the <STRONG><EM>upp</EM></STRONG> gene is a common method for counter selection and the knockout prevents the cell to retrieve pyrimidine’s from the media. <EM>upp<SUP>+</SUP> c</EM>ells growing on minimal media with the pyrimidine analog 5-FU and without any pyrimidine’s, would accumulate toxins derived form 5-FU causing the cells to die. The screening allows the cells that have an inactivated <EM>upp </EM>gene to persist as the toxin is not accumulated [6]. </P><H4><STRONG>Method</STRONG></H4><P style="text-align: justify;">Knockout was attempted using the oligoes shown in Table 1. The oligoes were designed using the program <A href="http://modest.biosustain.dtu.dk/">MODEST.</A> The oligo was designed with three point mutations resulting in stop codons in the sequence of the genes.</P><TABLE border="0" cellpadding="0" cellspacing="1"><THEAD><TR><TH style="width:37px;"><P align="center"><STRONG>Name</STRONG></P></TH><TH style="width:427px;"><P align="center"><STRONG>Oligos</STRONG></P></TH><TH><P align="center"><STRONG>Length</STRONG></P></TH><TH><P align="center"><STRONG>point mutation</STRONG></P><P align="center"><STRONG>position</STRONG></P></TH></TR></THEAD><TBODY><TR><TD style="width:37px;"><P>mage_amyE-1</P></TD><TD style="width:427px;"><P>AAGTAACGGTTGCCAATTTGATACGATGTCGGCTGATACAGtCAtTACtAGTTCGACATGCTTTTATCTCCTTGATTCCCTTCCTTTACT</P></TD><TD><P>90</P></TD><TD><P>45-53</P></TD></TR><TR><TD style="width:37px;"><P>mage_upp-1</P></TD><TD style="width:427px;"><P>GGGTAATTTCAAATGCCATGAGTGTAGCCACTTCATCTACTtACTaTCaAAAATCCTTCGTACCTGTATTTTCATTCCGTATATATGTCA</P></TD><TD><P>90</P></TD><TD><P>44-52</P></TD></TR></TBODY></TABLE><P><SPAN style="font-size:14px;">Table 2. The oligoes used for knockout attempt of <EM>amyE</EM> and <EM>upp</EM>.</SPAN></P><P style="text-align: justify;"><SPAN style="font-size:14px;"><EM>B. subtilis strain 168 </EM>with <EM>GP35</EM> or <EM>lambda beta</EM> inserted in <EM>amyE</EM> or <EM>mutS</EM> knockouts were made electrocompetent using protocol found <A href="http://dtuwiki2-drewt.rhcloud.comhttps://static.igem.org/mediawiki/2015/2/2e/DTU-Denmark_Electro_competent_protocol.pdf">here</A>. <EM>amyE</EM> and <EM>upp </EM>were electroporated with the oligoes shown in Table 1. The cells were electroporated at 2.0kV using 0.2cm cuvettes. Prior to the screening the electroporated cells were grown over night on 5y neomycin + LB agar plates, and following restreaked to respectively screening media, e.i.  5y neomycin + 1% starch + LB for or minimal media with 25uM 5-FU using the <A href="http://dtuwiki2-drewt.rhcloud.comhttps://static.igem.org/mediawiki/2015/5/52/DTU-Denmark_Minimal_media_protocol.pdf">minimal media protocol</A>. </SPAN></P><P> </P><H4><STRONG>Results </STRONG></H4><P style="text-align: justify;">The <EM>amyE</EM> screening proved to be insuficient to produce clear results and therefore this selection was abandoned. The 5-FU plates were incubated for three days before colonies were visible. To confirm that strain could grow on 5-FU, the colonies were restreacked onto new 5-FU plates. The bacteria could not grow after four days of incubation.</P><P>The methodshowed to be inadequate to prove the MAGE method in <EM>Bacillus</EM>.</P></DIV></DIV></DIV></DIV><P> </P><H3>Results      </H3><H4>MAGE proof of concept</H4><P style="text-align: justify;">To screen for the introduced streptomycin resistance single colonies were colony picked onto both LB and LB + streptomycin. The CFUs on the different plates are listed in the Table 2 below and plates shown in Figure 1. Unfortunately, the data for the two <EM>amyE</EM> strains got lost.</P><P style="text-align: justify;"> </P><TABLE border="0" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="text-align: center;"> </TD><TD><P style="text-align: center;"><STRONG>CFUs on LB </STRONG></P></TD><TD><P style="text-align: center;"><STRONG>CFUs on 500y streptomycin </STRONG></P></TD><TD><P style="text-align: center;"><STRONG>Frequency </STRONG></P></TD></TR><TR><TD><P style="text-align: center;"><STRONG><EM>∆mutS::beta-neoR</EM></STRONG></P></TD><TD><P style="text-align: center;">52</P></TD><TD><P style="text-align: center;">7</P></TD><TD><P style="text-align: center;">0,13</P></TD></TR><TR><TD><P style="text-align: center;"><STRONG><EM>∆mutS::GP35-neoR</EM></STRONG></P></TD><TD><P style="text-align: center;">100</P></TD><TD><P style="text-align: center;">1</P></TD><TD><P style="text-align: center;">0,01</P></TD></TR><TR><TD><P style="text-align: center;"><STRONG>WT</STRONG></P></TD><TD><P style="text-align: center;">100</P></TD><TD><P style="text-align: center;">0</P></TD><TD><P style="text-align: center;">0</P></TD></TR></TBODY></TABLE><P>Table 3. Listing of CFUs in both the selective and non-selective streptomycin resistance plates and their calculated frequency.</P><P> </P><P style="text-align: justify;"><SPAN style="font-size:14px;">Figure 4. The pictures show the colonies that was able to grow on 500y streptomycin after electroporation of the oligo. To the left is <EM>∆mutS::beta-neoR</EM> and to the right is <EM>∆mutS::GP35-neoR</EM>. It is a typo that the plates say “spec” and not “strep” on the plates.</SPAN></P><P> </P><P>The screened mutant was verified by “stamp replicating” the plates with the mutants.</P><P>Figure 5.  Stamp replications of the streptomycin resistant mutant. To the left is ∆mutS::beta-neoR and to the right is ∆mutS::GP35-neoR. It is a typo that the plates say “spec” and not “strep”.</P><P> </P><H2>Discussion</H2><P style="text-align: justify;"><SPAN style="font-size:14px;">It was established that streptomycin resistance as screening method did not provide sufficient data. The main issue was to quantify the actual mutants and separate them from spontaneous mutants. Time restrains limited the option of sequencing any of the mutants to verify the insertions. Experiments could be replicated to support the data. If the experimental data is accurate, the findings suggest the beta protein to be more efficient than the GP35, contradicts Sun et al. 2015, but we hypothesis that this could be due to the shorter oligos used compared to theirs[3]. Other reasons could be that GP35 is dependent on other gene products from the SPP1 phage such as GP34 or GP33, but this has not been confirmed.</SPAN></P></DIV></DIV></DIV></DIV><DIV id="OptimizationofMAGEinBsubtilis"><DIV class="container"><DIV class="row col-md-12"><H1>
        Optimization of MAGE in B. subtilis
      </H1><P><SPAN style="font-size:20px;"><STRONG>Overview </STRONG></SPAN></P><P>Different experiments were made to optimize the MAGE procedure. Three different experiments were conducted, to test the right amount and length of the oligos. additionally the mismatch frequency was attempted to be quantified. In our experiments the optimal length was shown to be 80nt which correlates with the expected 90nt. Optimal amount of oligo was showed to be 5uM, which also fits with the expectations. Interestingly the number of mismatches with the highest transformation rate was 5 mismatches, this is unexpected. Using streptomycin oligoes seemed to have a high systematic error.</P><P> </P><P><SPAN style="font-size:20px;"><STRONG>Background </STRONG></SPAN></P><P>We ran different optimization experiments to test if recombineering in B. subtilis could be optimized in the same way as for E. coli. For E. coli the optimal amount of oligo is 5µM [2].  with a legnth of 90nt [7]. The mismatch frequency of E. coli could be fitted by a binomial distribution.</P><P> </P><P><SPAN style="font-size:20px;"><STRONG>Experimental design</STRONG></SPAN></P><P>Using the dilution equation and the functional MAGE method, different experiments were run to optimize the efficiency of MAGE in <EM>Bacillus</EM>.</P><P>The three analyzed factors includes:
amount of oligo used, the length of the oligos, and the number of base pair mismatches inserted into the oligo.</P><P> </P><P><SPAN style="font-size:20px;"><STRONG>Achievements </STRONG></SPAN></P><UL><LI>Characterized the insertion frequency of mismatches in the genome of B. subtilis.</LI><LI>Characterized the insertion frequency of oligoes with different length in the genome of B. subtilis.</LI><LI>The an estimate of the optimal amount of oligo was found.</LI></UL><P> </P><P><SPAN style="font-size:20px;"><STRONG>Methods</STRONG></SPAN></P><P>All three experiments followed the &quot;MAGE in Bacillus subtilis 168&quot; <A href="https://static.igem.org/mediawiki/2015/0/0e/DTU-Denmark_MAGE_in_baccilus.pdf" target="_blank">protocol</A>. The oligo we used is shown below introducing streptomycin resistance with one mismatch.</P><TABLE border="1" cellpadding="1" cellspacing="1" style="width: 500px;"><TBODY><TR><TD><STRONG>oligo name</STRONG></TD><TD><STRONG>sequence </STRONG></TD><TD><STRONG>length</STRONG></TD></TR><TR><TD>B_Sub_Mods0007.1mutationrpsL </TD><TD><P dir="ltr" id="docs-internal-guid-104a6dbc-e363-bef5-da9b-8e98ab13ead0" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><SPAN style="font-size: 14.6667px; font-family: Arial; background-color: transparent; font-weight: 400; font-style: normal; font-variant: normal; text-decoration: none; vertical-align: baseline;">GAAGTGCTGAGTTCGGTTTgttCGGTGTCATTGTACCAACACGAGTACATACCCCGCGTTTTTGTGGAGAAGATACGTTAGTGTGCTCTT</SPAN></P></TD><TD> 90</TD></TR></TBODY></TABLE><P> </P><P>The amount  of oligo was varied between 0.05 - 6.25uM.</P><P><SPAN style="font-size:14.666666666666666px;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;">In the mismatch frequency experiment, six oligos with 1-6 mismatches individually was created to be inserted. </SPAN></P><TABLE border="1" cellpadding="1" cellspacing="1" style="width: 500px;"><TBODY><TR><TD>name</TD><TD>Sequence</TD><TD>Length</TD></TR><TR><TD><SPAN data-sheets-userformat="[null,null,8320,null,null,null,null,null,null,null,2,null,null,null,null,null,11]" data-sheets-value="[null,2,&quot;rpsL 1mm&quot;]" style="font-size:110%;font-family:arial,sans,sans-serif;">rpsL 1mm</SPAN></TD><TD><SPAN data-sheets-userformat="[null,null,8705,[null,0],null,null,null,null,null,null,null,null,0,null,null,null,10]" data-sheets-value="[null,2,&quot;G*A*AGTGCTGAGTTCGGTTTTCTCGGTGTCATTGTACCAACACGAGTACATACCCCGCGTTTTTGTGGAGAAGATACGTTAGTGTGCTCTT&quot;]" style="font-size:100%;font-family:arial,sans,sans-serif;">G*A*AGTGCTGAGTTCGGTTTTCTCGGTGTCATTGTACCAACACGAGTACATACCCCGCGTTTTTGTGGAGAAGATACGTTAGTGTGCTCTT</SPAN></TD><TD>90</TD></TR><TR><TD><SPAN data-sheets-userformat="[null,null,8320,null,null,null,null,null,null,null,2,null,null,null,null,null,11]" data-sheets-value="[null,2,&quot;rpsL 1mm&quot;]" style="font-size:110%;font-family:arial,sans,sans-serif;">rpsL 2mm</SPAN></TD><TD><SPAN data-sheets-userformat="[null,null,8705,[null,0],null,null,null,null,null,null,null,null,0,null,null,null,10]" data-sheets-value="[null,2,&quot;G*A*AGTGCTGAGTTCGGTTTCCTCGGTGTCATTGTACCAACACGAGTACATACCCCGCGTTTTTGTGGAGAAGATACGTTAGTGTGCTCTT&quot;]" style="font-size:100%;font-family:arial,sans,sans-serif;">G*A*AGTGCTGAGTTCGGTTTCCTCGGTGTCATTGTACCAACACGAGTACATACCCCGCGTTTTTGTGGAGAAGATACGTTAGTGTGCTCTT</SPAN></TD><TD>90</TD></TR><TR><TD><SPAN data-sheets-userformat="[null,null,8320,null,null,null,null,null,null,null,2,null,null,null,null,null,11]" data-sheets-value="[null,2,&quot;rpsL 1mm&quot;]" style="font-size:110%;font-family:arial,sans,sans-serif;">rpsL 3mm</SPAN></TD><TD><SPAN data-sheets-userformat="[null,null,8705,[null,0],null,null,null,null,null,null,null,null,0,null,null,null,10]" data-sheets-value="[null,2,&quot;C*G*AAGTGCTGAGTTCGGTTTCCGCGGTGTCATTGTACCAACACGAGTACATACCCCGCGTTTTTGTGGAGAAGATACGTTAGTGTGCTCT&quot;]" style="font-size:100%;font-family:arial,sans,sans-serif;">C*G*AAGTGCTGAGTTCGGTTTCCGCGGTGTCATTGTACCAACACGAGTACATACCCCGCGTTTTTGTGGAGAAGATACGTTAGTGTGCTCT</SPAN></TD><TD>90</TD></TR><TR><TD><SPAN data-sheets-userformat="[null,null,8320,null,null,null,null,null,null,null,2,null,null,null,null,null,11]" data-sheets-value="[null,2,&quot;rpsL 1mm&quot;]" style="font-size:110%;font-family:arial,sans,sans-serif;">rpsL 4mm</SPAN></TD><TD><SPAN data-sheets-userformat="[null,null,8705,[null,0],null,null,null,null,null,null,null,null,0,null,null,null,10]" data-sheets-value="[null,2,&quot;C*A*AACGAACACGAGCATATTTACGAAGTGCTGAGTTCGGTTTCCGTGGTGTCATTGTACCAACACGAGTACATACCCCGCGTTTTTGTGG&quot;]" style="font-size:100%;font-family:arial,sans,sans-serif;">C*A*AACGAACACGAGCATATTTACGAAGTGCTGAGTTCGGTTTCCGTGGTGTCATTGTACCAACACGAGTACATACCCCGCGTTTTTGTGG</SPAN></TD><TD>90</TD></TR><TR><TD><SPAN data-sheets-userformat="[null,null,8320,null,null,null,null,null,null,null,2,null,null,null,null,null,11]" data-sheets-value="[null,2,&quot;rpsL 1mm&quot;]" style="font-size:110%;font-family:arial,sans,sans-serif;">rpsL 5mm</SPAN></TD><TD><SPAN data-sheets-userformat="[null,null,8705,[null,0],null,null,null,null,null,null,null,null,0,null,null,null,10]" data-sheets-value="[null,2,&quot;A*C*GAGCATATTTACGAAGTGCTGAGTTCGGCTTCCGTGGTGTCATTGTACCAACACGAGTACATACCCCGCGTTTTTGTGGAGAAGATAC&quot;]" style="font-size:100%;font-family:arial,sans,sans-serif;">A*C*GAGCATATTTACGAAGTGCTGAGTTCGGCTTCCGTGGTGTCATTGTACCAACACGAGTACATACCCCGCGTTTTTGTGGAGAAGATAC</SPAN></TD><TD>90</TD></TR><TR><TD><SPAN data-sheets-userformat="[null,null,8320,null,null,null,null,null,null,null,2,null,null,null,null,null,11]" data-sheets-value="[null,2,&quot;rpsL 1mm&quot;]" style="font-size:110%;font-family:arial,sans,sans-serif;">rpsL 6mm</SPAN></TD><TD><SPAN data-sheets-userformat="[null,null,8705,[null,0],null,null,null,null,null,null,null,null,0,null,null,null,10]" data-sheets-value="[null,2,&quot;A*G*TCAAACGAACACGAGCATATTTACGAAGTGCTGAGTTTGGCTTCCGTGGTGTCATTGTACCAACACGAGTACATACCCCGCGTTTTTG&quot;]" style="font-size:100%;font-family:arial,sans,sans-serif;">A*G*TCAAACGAACACGAGCATATTTACGAAGTGCTGAGTTTGGCTTCCGTGGTGTCATTGTACCAACACGAGTACATACCCCGCGTTTTTG</SPAN></TD><TD>90</TD></TR></TBODY></TABLE><P> </P><P>For the varying of length ssDNA from 50-100nt was used with one mismatch.</P><P><SPAN style="font-size:14.666666666666666px;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;">In all experiment single colonies were taken from appropriate dilutions and colony picked onto 500ɣ strep. plates. Number colonies that was growing on colony picked strep plates was counted. The transformation efficiency was calculated as the ratio between re-streaked colonies growing on LB and on 500ɣ strep.</SPAN></P><P> </P><P><SPAN style="font-size:20px;"><STRONG>Results</STRONG></SPAN></P><P>The transformation frequency varies from 18-50% in the oligo amount experiment, this seems to be a too high number when comparing to earlier results, and the standard transformation frequency for MAGE in <EM>E. coli</EM>. The data seems to suggest that the optimal oligo amount for transformation is 5 uM. This correlates with the optimal amount for E coli [7]. The Figure 4 below shows the transformation frequency for the oligo amount experiment. The oligo length data shown in Figure 5 seems to suggest that the optimal oligo length is 80nt this correlates well with the the optimal length for E coli being 90nt.</P><P><SPAN style="font-size:14px;">Figure 6. the efficacy of oligo amount versus transformation rate</SPAN> </P><P> </P><P><SPAN style="font-size:14px;">Figure 7. Shows the transformation frequency of oligo length.</SPAN></P><P> </P><P dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><SPAN style="font-size:14px;">Figure 8 Shows transformation frequency of with extra mismatches from 1-6.</SPAN></P><P dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"> </P><P>The Figure 8 above shows the transformation frequency for mismatch insertion. This results does not correlate with the assumption that the lower the amount of mismatches the higher the transformation rate. Here the opposite is shown. This can be do to the high uncertainty in using streptomycin resistance. because of this the frequency pr. mismatch was not calculated.</P><P> </P><P dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><SPAN style="font-size:14.666666666666666px;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;">The three figures shown above varies greatly. It is possible that the cells were not optimally electro competent. These experiment need to be redone to define if the experimental setup is incorrect or if some other variation is in play.</SPAN></P><P> </P><P>Selection of the colonies was difficult because of possible background from spontaneous mutants that became visible after 2 days and the risk of adding too much cell mass onto the plate when colony picking, this could be mistaken as a growing colony.</P><P> </P><P>The results will need to be validated preferably using another screening method, since false positives seems to be an issue when using streptomycin as selection marker. It could be suggested to use a GFP with a inserted stop codon in the genome. The MAGE protocol could then be optimized for making knockins, this could be screened by using flow cytometry.</P><P> </P><P><SPAN style="font-size:20px;"><STRONG>Conclusion</STRONG></SPAN></P><P>The experiments indicate that 5bp mismatches with a length of 80nt and using 5uM of oligo would optimize the MAGE method in Bacillus. The data is of pure quality and nothing certain can be concluded from this experiment. </P><P> </P><P><SPAN style="font-size:26px;"><STRONG>Multiple Bacillus MAGE cycles</STRONG></SPAN></P><P> </P><P><SPAN style="font-size:20px;"><STRONG>Background</STRONG></SPAN></P><P>A great strength of the  MAGE method is that it can be iterated until the amount of cells with the wanted change is high [7]. We wanted to test if this also is the case in B. subtilis. Our experiments suggest that the this might be the case.</P><P><SPAN style="font-size:20px;"><STRONG>Experimental design</STRONG></SPAN></P><P>We hypothesized that if the MAGE protocol was repeated multiple times, the amount of transformants would rise. This was tested by running four cycles of the MAGE protocol. The progress could be followed by plating a dilution of the sample on streptomycin plates after every round and calculating the start value of the culture from the OD<SUB>600</SUB> measurements. It was necessary to colony pick onto streptomycin plates, which gave usable results.</P><P><SPAN style="font-size:20px;"><STRONG>Achievements</STRONG></SPAN></P><UL><LI>Identified that colony picking is a insufficient method to quantifying MAGE frequency.</LI><LI>Showed that the knockout of mutS has a significant effect onthe transformation frequency</LI><LI>developing a procedure for repeating MAGE in <EM>Bacillus</EM></LI></UL><P><SPAN style="font-size:20px;"><STRONG>Materials </STRONG></SPAN></P><P>The protocol specially made for this procedure was followed. This protocol takes approximately 6 hours for every cycle. Four cycles was run.</P><P> </P><P><SPAN style="font-size:20px;"><STRONG>Results</STRONG></SPAN></P><P>We had problems with finding the optimal dilutions. This cause a lack of data for some of the samples, new cells were re-plated from the glycerol stock of the differed MAGE cycles, but the same problem was encounted again.  There was a high variation in the amounts of transformants on the plates. We recommend to use the OD calculator for making an approximation of the correct dilution factor.</P><P> </P><P><SPAN style="font-size:14px;">Figure 9. The frequency for the insertion of the streptomycin phenotype can be seen for the samples.</SPAN></P><P>It seems that <EM>mutS::beta</EM> is better than the wild type, <EM>mutS::GP35</EM> and <EM>amyE::GP35</EM>. The experiment indicates that many MAGE cycles gives a higher yield than a single cycle, but the data is inconsistent when looking at Figure 4. The experiment will need to be run with a better method of testing if the insertion has been incorporated into the genome.</P></DIV></DIV></DIV><DIV id="Surfactin"><DIV class="container"><DIV class="row col-md-12"><H1>
        Surfactin
      </H1><P><SPAN style="font-size:20px;"><STRONG>Overview </STRONG></SPAN></P><P>To test if the MAGE method could be used to change the affinity of the A domain in a NRPS. The A domain in the 5th module of the synthase producing surfactin was changed and results were verified using using MALDI-TOF. Results indicate that the desired change was incorporated. The production efficiency of the bacteria was strongly reduced.</P><P> </P><P><SPAN style="font-size:20px;"><STRONG>Achievements</STRONG></SPAN></P><UL><LI>showed that the MAGE method can be used to change the specificity of a A domain in the surfactine synthase</LI></UL><P> </P><P><SPAN style="font-size:20px;"><STRONG>Experimental design</STRONG></SPAN></P><P>Surfactin is a surfactant cyclic lipopeptide produced by <EM>Bacillus subtilis</EM> important for sporulation in <EM>B. subtilis</EM>[8] and is used as an antibiotic[9]. The cyclic peptide of surfactin is produced by a nonribosomal peptide synthase (NRPS). AntiSMASH prediction of adenylation domain specificity corresponds to surfactant. The NRPS modules are divided out on three contigs (ctg1_354-5) with 3, 3, and 1 module, respectively as shown in figure 1</P><P> </P><P><SPAN style="font-size:14px;">Figure 10. Picture of surfactine synthase [10] </SPAN></P><P><SPAN style="font-size:14px;">Figure 11. Picture of surfactin</SPAN></P><P>The fifth module of surfactin synthetase is responsible for incorporation of aspartic acid. Using the Stachelhaus code the fewest changes on nucleotide level that would lead to a change in amino acid is Asp-&gt;Asn. Two different oligos with either a change or no change in wobble position of the Stachelhaus code and with different length were designed (Table 1), yielding different number of mismatches in the oligo.</P><P><SPAN style="font-size:14px;"><STRONG>Table 4 </STRONG>List of oligos used to modify surfactin NRPS.</SPAN></P><TABLE border="1" cellpadding="1" cellspacing="1" style="width: 500px;"><TBODY><TR><TD> oligo name</TD><TD>sequence</TD><TD>LengthMutation(Stacelhaus)</TD><TD><P>length</P></TD></TR><TR><TD><P><SPAN style="font-size:12px;"><SPAN data-sheets-userformat="[null,null,10755,[null,0],[null,2,16119285],null,null,null,null,null,null,null,0,null,[null,2,3355443],null,11]" data-sheets-value="[null,2,&quot;Oligo_surf_Asp-&gt;Asn_1_l&quot;]" style="font-family: arial,sans,sans-serif; color: rgb(51, 51, 51);">Oligo_surf_</SPAN></SPAN></P><P><SPAN style="font-size:12px;"><SPAN data-sheets-userformat="[null,null,10755,[null,0],[null,2,16119285],null,null,null,null,null,null,null,0,null,[null,2,3355443],null,11]" data-sheets-value="[null,2,&quot;Oligo_surf_Asp-&gt;Asn_1_l&quot;]" style="font-family: arial,sans,sans-serif; color: rgb(51, 51, 51);">Asp-&gt;Asn_1_l</SPAN></SPAN></P></TD><TD><P><SPAN style="font-size:14px;"><SPAN data-sheets-userformat="[null,null,10755,[null,0],[null,2,16777215],null,null,null,null,null,null,null,0,null,[null,2,3355443],null,11]" data-sheets-value="[null,2,&quot;C*A*TACAGATCAACCCGCCCGGCGATGGCGCCGACCGTTGCTTCTGTCGGGCCGTACTCATTGATAAATTCGGTATGTCCATACATCTTAC&quot;]" style="font-family: arial,sans,sans-serif; color: rgb(51, 51, 51);">C*A*TACAGATCAACCCGCCCGGCGATGGCGCCGACCGTTGCTTCTGTCGGGCCGTACTCATTGA</SPAN></SPAN></P><P><SPAN style="font-size:14px;"><SPAN data-sheets-userformat="[null,null,10755,[null,0],[null,2,16777215],null,null,null,null,null,null,null,0,null,[null,2,3355443],null,11]" data-sheets-value="[null,2,&quot;C*A*TACAGATCAACCCGCCCGGCGATGGCGCCGACCGTTGCTTCTGTCGGGCCGTACTCATTGATAAATTCGGTATGTCCATACATCTTAC&quot;]" style="font-family: arial,sans,sans-serif; color: rgb(51, 51, 51);">TAAA</SPAN><SPAN data-sheets-userformat="[null,null,10755,[null,0],[null,2,16777215],null,null,null,null,null,null,null,0,null,[null,2,3355443],null,11]" data-sheets-value="[null,2,&quot;C*A*TACAGATCAACCCGCCCGGCGATGGCGCCGACCGTTGCTTCTGTCGGGCCGTACTCATTGATAAATTCGGTATGTCCATACATCTTAC&quot;]" style="font-family: arial,sans,sans-serif; color: rgb(51, 51, 51);">TTCGGTATGTCCATACATCTTAC</SPAN></SPAN></P></TD><TD>H322E, I330S</TD><TD>200</TD></TR><TR><TD><SPAN data-sheets-userformat="[null,null,513,[null,0],null,null,null,null,null,null,null,null,0]" data-sheets-value="[null,2,&quot;oligo_surf asp-&gt;Asn_2_I&quot;]" style="font-size:13px;font-family:arial,sans,sans-serif;">oligo_surf asp-&gt;Asn_2_I</SPAN></TD><TD><P><SPAN style="font-size:14px;"><SPAN data-sheets-userformat="[null,null,10755,[null,0],[null,2,16777215],null,null,null,null,null,null,null,0,null,[null,2,3355443],null,11]" data-sheets-value="[null,2,&quot;T*T*CGCAAATGCATCCGGCTCATACAGATCAACCCGCCCGGCGATGGCGCCGACCGTTGCTTCTGTCGGGCCGTACTCATTGATAAATTCGGTATGTCCATACATCTTACGGAAGGCGATAACATCAGTCGGGATGATTTTTTCTCCTCCCAAGAGGATCAAGCGCAAGGATTCAAAGTTCGCATCTTTTGCAAAACTGGC&quot;]" style="font-family: arial,sans,sans-serif; color: rgb(51, 51, 51);">T*T*CGCAAATGCATCCGGCTCATACAGATCAACCCGCCCGGCGATGGCGCCGACCGTTGCTTCT</SPAN></SPAN></P><P><SPAN style="font-size:14px;"><SPAN data-sheets-userformat="[null,null,10755,[null,0],[null,2,16777215],null,null,null,null,null,null,null,0,null,[null,2,3355443],null,11]" data-sheets-value="[null,2,&quot;T*T*CGCAAATGCATCCGGCTCATACAGATCAACCCGCCCGGCGATGGCGCCGACCGTTGCTTCTGTCGGGCCGTACTCATTGATAAATTCGGTATGTCCATACATCTTACGGAAGGCGATAACATCAGTCGGGATGATTTTTTCTCCTCCCAAGAGGATCAAGCGCAAGGATTCAAAGTTCGCATCTTTTGCAAAACTGGC&quot;]" style="font-family: arial,sans,sans-serif; color: rgb(51, 51, 51);">GTCGGGCCGTACTCATTGATAAATTCGGTATGTCCATACATCTTACGGAAGGCGATAACATCAGTCGG</SPAN></SPAN></P><P><SPAN style="font-size:14px;"><SPAN data-sheets-userformat="[null,null,10755,[null,0],[null,2,16777215],null,null,null,null,null,null,null,0,null,[null,2,3355443],null,11]" data-sheets-value="[null,2,&quot;T*T*CGCAAATGCATCCGGCTCATACAGATCAACCCGCCCGGCGATGGCGCCGACCGTTGCTTCTGTCGGGCCGTACTCATTGATAAATTCGGTATGTCCATACATCTTACGGAAGGCGATAACATCAGTCGGGATGATTTTTTCTCCTCCCAAGAGGATCAAGCGCAAGGATTCAAAGTTCGCATCTTTTGCAAAACTGGC&quot;]" style="font-family: arial,sans,sans-serif; color: rgb(51, 51, 51);">GATGATTTTTTCTCCTCCCAAGAGGATCAAGCGCAAGGATTCAAAGTTCGCATCTTTTGCAAAACTGGC</SPAN></SPAN></P></TD><TD>V299L, H322E, I330S </TD><TD>200</TD></TR></TBODY></TABLE><P> </P><H2><SPAN style="font-size:20px;">Methods</SPAN></H2><P>Electrocompetent <EM>B. subtilisΔmutS::beta-neo<SUP>R</SUP></EM>or <EM>ΔmutS::GP35-neo<SUP>R</SUP></EM>  mutation was used.  Three oligoes were used for this experiment. The two showed in table 1 was used separately, and the streptomycin resisters oligo called B_sub_Mods0007.1mutationrpsL was used to select for the desired change. 100uL of cells was mixed with 5uL of the surfactine changing oligo, and 0.5uL of the streptomycin resistance oligo was used in accordance with the protocol for <A href="https://static.igem.org/mediawiki/2015/2/2e/DTU-Denmark_Electro_competent_protocol.pdf" target="_blank">electroporation.</A></P><P> </P><P><SPAN style="font-size:20px;"><STRONG>Results </STRONG></SPAN></P><P>See surfactine part in Detection of NRP</P></DIV></DIV></DIV><DIV id="DilutionEquation"><DIV class="container"><DIV class="row col-md-12"><H1>
        Dilution Equation
      </H1><P dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><SPAN style="font-size:20px;"><SPAN style="font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; font-style: normal; font-variant: normal; text-decoration: none; vertical-align: baseline;">Overview</SPAN></SPAN></P><P dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><SPAN style="font-size:14.666666666666666px;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;">The</SPAN><A href="https://2008.igem.org/Team:Imperial_College" style="text-decoration:none;"><SPAN style="font-size:14.666666666666666px;font-family:Arial;color:#1155cc;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:underline;vertical-align:baseline;">Imperial iGEM 2008</SPAN></A><SPAN style="font-size:14.666666666666666px;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;"> team has made an equation for calculating CFU from OD<SUB>600</SUB>. We tried to validate their equation by using our own data. Unfortunately the variation in our results was too high to validate their equation. Based on their results we made a calculator that could compute the optimal dilution for plating to get a countable number of colonies.</SPAN></P><P> </P><P dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><SPAN style="font-size:20px;"><SPAN style="font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; font-style: normal; font-variant: normal; text-decoration: none; vertical-align: baseline;">Method</SPAN></SPAN></P><P dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><SPAN style="font-size:14.666666666666666px;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;">All the LB plate counts that we have done in our project was gathered and analyzed for this experiment. the data that was used can be seen here.</SPAN></P><P dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"> </P><P dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><SPAN style="font-size:20px;"><SPAN style="font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; font-style: normal; font-variant: normal; text-decoration: none; vertical-align: baseline;">Results</SPAN></SPAN></P><P dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><SPAN style="font-size:14.666666666666666px;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;">The Imperial College team  modulated following equation.</SPAN></P><P dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><SPAN style="font-size:14.666666666666666px;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;">Imperial equation: Y= 2*10</SPAN><SPAN style="font-size:8.799999999999999px;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:super;">8</SPAN><SPAN style="font-size:14.666666666666666px;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;"> *X</SPAN></P><P dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><SPAN style="font-size:14.666666666666666px;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;">Y = CFU/mL</SPAN></P><P dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><SPAN style="font-size:14px;"><SPAN style="font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 400; font-style: normal; font-variant: normal; text-decoration: none; vertical-align: baseline;">Figure 12. Our attempt to validate the Imperial College 2008 teams OD too CFU measurements. It is clear that the R</SPAN><SPAN style="font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 400; font-style: normal; font-variant: normal; text-decoration: none; vertical-align: super;">2</SPAN><SPAN style="font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 400; font-style: normal; font-variant: normal; text-decoration: none; vertical-align: baseline;"> value is not close optimal.</SPAN></SPAN></P><P dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"> </P><P dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><SPAN style="font-size:14.666666666666666px;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;">As can be seen from the figure 8 shown above our data could not validate the equation completely. Though our data trends towards Imperials 2008s equation</SPAN><SPAN style="font-size:14.666666666666666px;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;">.</SPAN></P><P dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"> </P><P dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><SPAN style="font-size:14.666666666666666px;font-family:Arial;color:#000000;background-color:transparent;font-weight:700;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;">Dilution predictor</SPAN></P><P dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><SPAN style="font-size:14.666666666666666px;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;">An equation for predicting a dilutions that will result in a countable number of CFUs was made from the Imperial College equation. The equation assume that 100µl is plated on a LB plate. The optimal amount of colonies is set to 150CFU on each plate.</SPAN></P><P dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"> </P><P dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><SPAN style="font-size:14.666666666666666px;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;">Y</SPAN><SPAN style="font-size:8.799999999999999px;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:sub;">CFU</SPAN><SPAN style="font-size:14.666666666666666px;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;">=2*X</SPAN><SPAN style="font-size:8.799999999999999px;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:sub;">OD</SPAN><SPAN style="font-size:14.666666666666666px;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;">*10</SPAN><SPAN style="font-size:8.799999999999999px;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:super;">8</SPAN></P><P dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><SPAN style="font-size:14.666666666666666px;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;">Y</SPAN><SPAN style="font-size:8.799999999999999px;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:sub;">optimal</SPAN><SPAN style="font-size:14.666666666666666px;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;">=150CFU/plate. This number can be varied to fit the user's preference.</SPAN></P><P dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><SPAN style="font-size:14.666666666666666px;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;">d is the optimal dilution factor for getting 150CFU/plate.  E.i. optimal dilution will be 10</SPAN><SPAN style="font-size:8.799999999999999px;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:super;">d</SPAN><SPAN style="font-size:14.666666666666666px;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;">.</SPAN></P><P><SUB><SPAN class="math-tex">\(Y_{optimal} = {{Y_{CFU} } \over {10^d* 10}}\)</SPAN></SUB></P><P><SPAN class="math-tex">\(Y_{optimal} = {{2*X_{OD}*10^8} \over {10^fd* 10}}\)</SPAN></P><P><SPAN class="math-tex">\(10^d= {{2*X_{OD}*10^8} \over {10*Y_{optimal}}}\)</SPAN></P><P><SPAN class="math-tex">\(d= log_{10}( {{2*X_{OD}*10^8} \over {10*Y_{optimal}}} ) , Y_{optimal}=150\)</SPAN></P><P><SPAN class="math-tex">\(d= log_{10}( {{2*X_{OD}*10^8} \over {10*150}} ) \)</SPAN></P><P><SPAN class="math-tex">\(d= log_{10}( {{1.33*X_{OD}*10^5} } ) \)</SPAN></P><P dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><SPAN style="font-size:14.666666666666666px;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;">The formula has not been thoroughly test and the correlation between OD and CFU is low in for our data. Generally the formula overestimates dilution. Therefore we suggest that both 10</SPAN><SPAN style="font-size:8.799999999999999px;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:super;">d</SPAN><SPAN style="font-size:14.666666666666666px;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;"> and </SPAN><SPAN style="font-size:14.666666666666666px;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;">10</SPAN><SPAN style="font-size:8.799999999999999px;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:super;">d-1</SPAN><SPAN style="font-size:14.666666666666666px;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;"> are plated. A future solution to the problem could be to introduce a calibration constant to the right hand side of the equation. The constant can be fitted by rerunning the experiment with more samples.</SPAN></P></DIV></DIV></DIV><DIV id="References"><DIV class="container"><DIV class="row col-md-12"><H1>
        References
      </H1><OL><LI>Acquisition of Certain Streptomycin-Resistant (str) Mutations Enhances Antibiotic Production in Bacteria, YOSHIKO HOSOYA,1 SUSUMU OKAMOTO,1 HIDEYUKI MURAMATSU,2 AND KOZO OCHI, National Food Research Institute,1 and Exploratory Research Laboratories, Fujisawa Pharmaceutical Co.,2 Tsukuba, Ibaraki, Japan</LI><LI>Carr, P. A., Wang, H. H., Sterling, B., Isaacs, F. J., Lajoie, M. J., Xu, G., … Jacobson, J. M. (2012). Enhanced multiplex genome engineering through co-operative oligonucleotide co-selection. Nucleic Acids Research, 40(17), e132–e132. <A href="http://dx.doi.org/10.1093/nar/gks455" target="_blank">doi:10.1093/nar/gks455</A></LI><LI>Sun, Z., Deng, A., Hu, T., Wu, J., Sun, Q., Bai, H., … Wen, T. (2015). A high-efficiency recombineering system with PCR-based ssDNA in Bacillus subtilis mediated by the native phage recombinase GP35. Applied Microbiology and Biotechnology, 99(12), 5151–5162. <A href="http://dx.doi.org/10.1007/s00253-015-6485-5" target="_blank">doi:10.1007/s00253-015-6485-5</A></LI><LI>Ginetti, F., Perego, M., Albertini, A. M., &amp; Galizzi, A. (1996). Bacillus subtilis mutS mutL operon: identification, nucleotide sequence and mutagenesis. Microbiology, 142(8), 2021–2029. <A href="http://dx.doi.org/10.1099/13500872-142-8-2021" target="_blank">doi:10.1099/13500872-142-8-2021</A></LI><LI>Bonde, M. T., Klausen, M. S., Anderson, M. V., Wallin, A. I. N., Wang, H. H., &amp; Sommer, M. O. A. (2014). MODEST: a web-based design tool for oligonucleotide-mediated genome engineering and recombineering. Nucleic Acids Research, 42(W1), W408–W415. <A href="http://dx.doi.org/10.1093/nar/gku428" target="_blank">doi:10.1093/nar/gku428</A></LI><LI>Dong, H., &amp; Zhang, D. (2014). Current development in genetic engineering strategies of Bacillus species. Microbial Cell Factories, 13(1), 63. <A href="http://dx.doi.org/10.1186/1475-2859-13-63" target="_blank">doi:10.1186/1475-2859-13-63</A></LI><LI>Wang, H. H., Isaacs, F. J., Carr, P. A., Sun, Z. Z., Xu, G., Forest, C. R., &amp; Church, G. M. (2009). Programming cells by multiplex genome engineering and accelerated evolution. Nature, 460(7257), 894–898. <A href="http://dx.doi.org/10.1038/nature08187" target="_blank">doi:10.1038/nature08187</A></LI><LI>Nakano MM, Magnuson R, Myers A, Curry J, Grossman AD, Zuber P. srfA is an operon required for surfactin production, competence development, and efficient sporulation in Bacillus subtilis. J Bacteriol. 1991 Mar;173(5):1770-8</LI><LI>Drug Development Research July - August 2000, Volume 50, Issue 3-4 Pages 203–583, Issue edited by: David Gurwitz</LI><LI>Koglin, A., Löhr, F., Bernhard, F., Rogov, V. V., Frueh, D. P., Strieter, E. R., … Dötsch, V. (2008). Structural basis for the selectivity of the external thioesterase of the surfactin synthetase. Nature, 454(7206), 907–911. <A href="http://dx.doi.org/10.1038/nature07161" target="_blank">doi:10.1038/nature07161</A></LI></OL></DIV></DIV></DIV><FOOTER class="container"><DIV class="row"><DIV class="col-md-4"><ADDRESS><STRONG>Technical University of Denmark</STRONG>
            Department of Systems Biology 
            Søltofts Plads 221 
            2800 Kgs. Lyngby Denmark<ABBR title="Phone">P:</ABBR> +45 45 25 25 25<ABBR title="Mail">M:</ABBR><A href="mailto:dtu-igem-2015@googlegroups.com">dtu-igem-2015@googlegroups.com</A></ADDRESS></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>