<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_DTU-Denmark_Project_POC_MAGE skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:DTU-Denmark/Project/POC MAGE</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><NAV class="navbar navbar-default navbar-fixed-top" role="navigation"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#nav-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="nav-collapse"><UL class="nav navbar-nav"><LI><A href="/Team:DTU-Denmark">Home
                                           </A></LI><LI><A href="" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Team and Attributions
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Team">Team
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Attributions">Attributions
                                           </A></LI></UL><LI class="active"><A href="" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Project
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Project/Overview">Overview
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Project/Background">Background
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Project/MAGE">MAGE Subtilis
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Project/Surfactin">Surfactin
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Project/Tyrocidine">Tyrocidine
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Project/Chip">Lab-on-a-disc
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Project/Intein">Inteins
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Project/Detection">Detection of NRP
                                           </A></LI><LI class="active"><A href="/Team:DTU-Denmark/Project/POC_MAGE">Proof of concept of MAGE in B. subtilis
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Practices">Human Practices
                                           </A></LI><LI><A href="" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Parts Collection
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Parts">Parts
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Description">Characterization of xylR
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Journal">Journal
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Software">Software
                                           </A></LI><LI><A href="" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Achievements
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Achievements">Key Achievements
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Collaborations">Collaborations
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Judging%20Form">Judging Form
                                           </A></LI><LI class="hidden-xs"><A href="http://facebook.com/dtubiobuilders" data-toggle="tooltip" data-placement="bottom" title="Follow us on Facebook"><SPAN class="visible-xs"> Follow us on Facebook</SPAN></A></LI><LI class="visible-xs"><A href="http://facebook.com/dtubiobuilders"> Follow us on Facebook</A></LI><LI class="visible-xs"><A href="https://twitter.com/IGEM_DTU"> Follow us on Twitter</A></LI></DIV></DIV></NAV><NAV class="navbar navbar-default extrabar hidden-xs hidden" data-spy="affix"><DIV class="container-fluid" id="scrollspy"><UL class="nav navbar-nav"><LI><A class="page-scroll" href="#Overview">Overview</A></LI><LI><A class="page-scroll" href="#Achievements">Achievements</A></LI><LI><A class="page-scroll" href="#MAGE-competent-strains">MAGE competent strains</A></LI><LI><A class="page-scroll" href="#Methods-for-proof-of-concept">Methods for proof of concept</A></LI><LI><A class="page-scroll" href="#Discussion">Discussion</A></LI><LI><A class="page-scroll" href="#References">References</A></LI></UL></DIV></NAV><DIV class="jumbotron hero bg-hero"><DIV class="container-fluid"><DIV class="row hero-text"><DIV class="col-md-12"><H1>Proof of concept of MAGE in B. subtilis</H1><UL class="list-inline hidden-xs hidden-sm"><LI><A id="Overview-submenu" class="btn btn-default btn-transparent btn-lg page-scroll" href="#Overview">Overview</A></LI><LI><A id="Achievements-submenu" class="btn btn-default btn-transparent btn-lg page-scroll" href="#Achievements">Achievements</A></LI><LI><A id="MAGE-competent-strains-submenu" class="btn btn-default btn-transparent btn-lg page-scroll" href="#MAGE-competent-strains">MAGE competent strains</A></LI><LI><A id="Methods-for-proof-of-concept-submenu" class="btn btn-default btn-transparent btn-lg page-scroll" href="#Methods-for-proof-of-concept">Methods for proof of concept</A></LI><LI><A id="Discussion-submenu" class="btn btn-default btn-transparent btn-lg page-scroll" href="#Discussion">Discussion</A></LI><LI><A id="References-submenu" class="btn btn-default btn-transparent btn-lg page-scroll" href="#References">References</A></LI></UL><UL class="list-inline visible-xs visible-sm"><LI><A id="Overview-submenu" class="page-scroll" href="#Overview">Overview</A></LI><LI><A id="Achievements-submenu" class="page-scroll" href="#Achievements">Achievements</A></LI><LI><A id="MAGE-competent-strains-submenu" class="page-scroll" href="#MAGE-competent-strains">MAGE competent strains</A></LI><LI><A id="Methods-for-proof-of-concept-submenu" class="page-scroll" href="#Methods-for-proof-of-concept">Methods for proof of concept</A></LI><LI><A id="Discussion-submenu" class="page-scroll" href="#Discussion">Discussion</A></LI><LI><A id="References-submenu" class="page-scroll" href="#References">References</A></LI></UL></DIV></DIV><DIV class="row"><DIV class="col-md-12">
            Scroll down for more</DIV></DIV></DIV></DIV><DIV id="Overview"><DIV class="container"><DIV class="row col-md-12"><H1>
        Overview
      </H1><P>In order for MAGE (Multiplex Automated Genome Engineering) to work at a high efficiency, a strain with inserted recombinase and inhibited or knocked out mismatch repair gene has to be used [1]. In our project, two different recombinases were used: a recombination protein Beta from the <EM>E. coli</EM> phage Lambda, which was codon optimized for B. subtilis 168, and GP35, a recombinase from the B.subtilis phage SPP1 [2]. The mismatch repair proteins known as MutS and MutL were knocked out by transforming pSB1C3_recombinase plasmid into the B. subtilis W168. Since the MutL protein is dependent on the binding of MutS, the knockout of the mutS disables the function of the MutL protein [3].
Four <EM>Bacillus subtilis</EM> strains which expressed a recombinase were created by genetically engineering the wild type strain 168:</P><UL><LI><EM>∆amyE::beta-neoR</EM></LI><LI><EM>∆amyE::GP35-neoR</EM></LI><LI><EM>∆mutS::beta-neoR</EM></LI><LI><EM>∆mutS::GP35-neoR</EM></LI></UL><P>For proof of concept we decided to make a single protein substitution in the ribosomal S12 protein in B. subtilis results in resistance to the antibiotic streptomycin. The required change is a lysine to arginine substitution at position 56 of the protein. The S12 subunit is coded by the rpsL gene [4]. An oligo that could integrate this change was design using MODEST [5]. These oligos were successfully integrated into the two strains: ​<EM>∆mutS::beta-neo<SUP>R</SUP></EM> and <EM>∆mutS::GP35-neo<SUP>R</SUP>.</EM></P></DIV></DIV></DIV><DIV id="Achievements"><DIV class="container"><DIV class="row col-md-12"><H1>
        Achievements
      </H1><UL><LI>Made <EM>Bacillus subtilis 168</EM> WT MAGE compatible</LI><LI>Showed that the MODEST program can make oligoes that can be inserted in <EM>Bacillus. subtilis 168.</EM></LI><LI>Introducing Multiplex Automated Genome Engineering (MAGE) in <EM>Bacillus  subtilis 168</EM>.</LI></UL></DIV></DIV></DIV><DIV id="MAGE-competent-strains"><DIV class="container"><DIV class="row col-md-12"><H1>
        MAGE competent strains
      </H1><P>All the strain were made by homologous recombineering. Plasmids containing cassettes that were able to do a double-crossover homologous recombineering into the genome of B. subtilis 168 (referred to as knockout (KO) plasmid). These were used to, simultaneously, delete the desired gene (amyE or mutS) and inserting the expression cassette for one of the recombinases: beta or GP35.</P><P><SPAN style="font-size:14px;">Figure 1. Shows the general concepts of the two plasmids pDG268neo_recombinase and pSB1C3_recombinase. Both exist in two versions, one with each of the recombinase proteins CDSs (Beta and GP35). They also have different RBSs since they are optimized for the CDS. Upstream of neoR is a promoter and RBS and downstream of neoR is a terminator, but sequences and positions of these features are not known.</SPAN></P><P> </P><P>Four different plasmids were assembled to make the four MAGE ready strains:
pDG268neo_Beta-neoR
pDG268neo_GP35-neoR
pSB1C3_Beta-neoR
pSB1C3_GP35-neoR</P><P> </P><DIV class="panel panel-default"><DIV class="panel-heading" id="headingTen" role="tab"><H4 class="panel-title"><A aria-controls="collapseTen" aria-expanded="false" class="collapsed" data-parent="#accordion" data-toggle="collapse" href="#collapseTen" role="button">Insert of recombinase in <EM>amyE</EM></A></H4></DIV><DIV aria-labelledby="headingTen" class="panel-collapse collapse" id="collapseTen" role="tabpanel"><DIV class="panel-body"><DIV class="panel-body"><P>For pDG268neo_recobinase a DNA sequence containing following features</P><P>●      Promoter: PliaG from BBa_K823002 was used.</P><P>●      RBSs were optimized for the specific CDS using the salis lab RBS calculator (<A href="https://www.denovodna.com/software/">https://www.denovodna.com/software/</A>)</P><P>●      CDS for recombination protein beta or GP35</P><P>●      Terminator: we use rho-independent Part:BBa_B0014</P>
Download gblocks <A href="/wiki/images/6/66/DTU-Denmark_Fused_gblocks.xls">here</A><P><I>Sequence was ordered from IDT as two gblocks (for each recombinase) with overlapping regions, thus they can be assembled with Gibson assembly.</I></P><P> </P><P>pDG268neo was linearized in a PCR with primers, so that the native <I>lacZ </I>was omitted. This linearized plasmid was purified and used in a Gibson assembly reaction with two cognate gblocks.</P><P><SPAN style="font-size:14px;">Figure 2. Gibson assembly of the pDG268neo_GP35, the gblocks was fused to the “gp35 Gblocks fused” in the same reaction. The Gibson assembly of pDG268neo_beta was similar.</SPAN></P><P> </P><P>This resulted in two different plasmids pDG268neo_beta and pDG268neo_gp35.</P><P> </P></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" id="headingEleven" role="tab"><H4 class="panel-title"><A aria-controls="collapseEleven" aria-expanded="false" class="collapsed" data-parent="#accordion" data-toggle="collapse" href="#collapseEleven" role="button"><EM>mutS</EM> deletion</A></H4></DIV><DIV aria-labelledby="headingTen" class="panel-collapse collapse" id="collapseEleven" role="tabpanel"><DIV class="panel-body"><P>The two mutS KO plasmids was made of the following DNA fragments:</P><P>1.     About 500 bp up- and downstream from <I>mutS</I> was amplified, using primers with tails.</P><P>2.     Recombinase and neoR expression cassettes was amplified from the pDG268neo_recombinase</P><P>3.    Linearized pSB1C3 was used as template.</P><P> </P><P>These fragments was assembled into two different plasmids pSB1C3_<EM>mutS::beta-neo<SUP>R</SUP></EM> and pSB1C3_<EM>mutS::gp35-neo<SUP>R</SUP></EM> using Gibson assembly.</P><P> </P><P><SPAN style="font-size:14px;">Figure 3. Gibson assembly of the pSB1C3_beta. The feature called “mutL” corresponding mutS downstream. The Gibson assembly of pSB1C3_GP35 was similar.</SPAN></P><P>All plasmids were verified by restriction enzyme digestion.</P><P>We prepared naturally competent <I>B. subtilis 168</I> to be transformed. The plasmids were linearized with restriction enzymes. Then transformed into naturally competent <I>B. subtilis 168</I>, transformants were selected on 5ɣ neo. Transformants were verified with colony PCRs.</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="Methods-for-proof-of-concept"><DIV class="container"><DIV class="row col-md-12"><H1>
        Methods for proof of concept
      </H1><P>Electroporation competent cells were prepared in following the protocol: Electroporation competent <EM>Bacillus subtilis 168. </EM>Electropration was carried out according to the MAGE in <EM>Bacillus subtilis 168</EM> protocol. The colonies from the LB plates were colonies picked onto an LB and LB + 500y streptomycin (strep) plates. Plates were incubated at 37 degC for 48 hours and CFUs were counted on both the LB and the strep plates.Results</P><DIV aria-multiselectable="true" class="panel-group" id="accordion" role="tablist"><DIV class="panel panel-default"><DIV class="panel-heading" id="headingOne" role="tab"><H4 class="panel-title"><A aria-controls="collapseOne" aria-expanded="false" class="collapsed" data-parent="#accordion" data-toggle="collapse" href="#collapseOne" role="button">Results</A></H4></DIV><DIV aria-labelledby="headingOne" class="panel-collapse collapse" id="collapseOne" role="tabpanel"><DIV class="panel-body"><DIV class="panel-body"><TABLE border="1" cellpadding="1" cellspacing="1" class="table table" style="width:500px;"><THEAD><TR><TH scope="col"> </TH><TH scope="col" style="text-align: center;">Total number of CFUs</TH><TH scope="col" style="text-align: center;">Number of transformant</TH><TH scope="col" style="text-align: center;">Transformation frequency</TH></TR></THEAD><TBODY><TR><TD>Δ<I>mutS::beta-neo</I><I><SUP>R</SUP></I></TD><TD style="text-align: center;">52</TD><TD style="text-align: center;">7</TD><TD style="text-align: center;">0.13</TD></TR><TR><TD>Δ<I>mutS::GP35-neo</I><I><SUP>R</SUP></I></TD><TD style="text-align: center;">100</TD><TD style="text-align: center;">1</TD><TD style="text-align: center;">0.01</TD></TR></TBODY></TABLE><P><SPAN style="font-size:14px;">Table 1. shows the data for the MAGE proof of concept.</SPAN></P><P> </P><P>Second replication of the plates:</P><P><SPAN style="font-size:14px;">Figure 4. Pictures shows the first and second replication of the transformants. From left to right of ΔmutS::GP35-neo<SUP>R</SUP> <SPAN style="font-size:14px;"> replicate1</SPAN>, <EM>ΔmutS::GP35-neo</EM><SUP><EM>R</EM> </SUP><SPAN style="font-size:14px;">replicate2 </SPAN>, Δ<I>mutS::beta-neo</I><I><SUP>R </SUP></I>replica1, and <SPAN style="font-size:14px;">Δ<I>mutS::beta-neo</I><I><SUP>R </SUP></I>replica2.</SPAN></SPAN></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="Discussion"><DIV class="container"><DIV class="row col-md-12"><H1>
        Discussion
      </H1><P>The results of this proof of concept indicates that MAGE works in two of the MAGE compatible strains Δ<EM>mutS::beta-neo</EM><EM><SUP>R</SUP></EM> and Δ<EM>mutS::GP35-neo</EM><EM><SUP>R</SUP></EM><EM>. </EM>More thorough experiments need to be done. The results suggest that the beta protein is a more effective recombinase for the type of oligo used in this experiment.</P></DIV></DIV></DIV><DIV id="References"><DIV class="container"><DIV class="row col-md-12"><H1>
        References
      </H1><OL><LI>Carr, P. A., Wang, H. H., Sterling, B., Isaacs, F. J., Lajoie, M. J., Xu, G., … Jacobson, J. M. (2012). Enhanced multiplex genome engineering through co-operative oligonucleotide co-selection. Nucleic Acids Research, 40(17). doi:10.1093/nar/gks455</LI><LI>Sun, Z., Deng, A., Hu, T., Wu, J., Sun, Q., Bai, H., … Wen, T. (2015). A high-efficiency recombineering system with PCR-based ssDNA in Bacillus subtilis mediated by the native phage recombinase GP35. Applied Microbiology and Biotechnology, 99(12), 5151–5162. <A href="http://dx.doi.org/10.1007/s00253-015-6485-5" target="_blank">doi:10.1007/s00253-015-6485-5</A></LI><LI>Ginetti, F., Perego, M., Albertini, A. M., &amp; Galizzi, A. (1996). Bacillus subtilis mutS mutL operon: Identification, nucleotide sequence and mutagenesis. Microbiology, 142(8), 2021–2029. doi:10.1099/13500872-142-8-2021</LI><LI>Barnard, A. M. L., Simpson, N. J. L., Lilley, K. S., &amp;amp;amp; Salmond, G. P. C. (2010). Mutations in rpsL that confer streptomycin resistance show pleiotropic effects on virulence and the production of a carbapenem antibiotic in Erwinia carotovora. Microbiology, 156(4), 1030–1039. &amp;amp;lt;a href=&amp;amp;#34;http://dx.doi.org/10.1099/mic.0.034595-0&amp;amp;#34; target=&amp;amp;#34;_blank&amp;amp;#34;&amp;amp;gt;doi:10.1099/mic.0.034595-0&amp;amp;lt;/a&amp;amp;gt;</LI><LI>Bonde, M. T., Klausen, M. S., Anderson, M. V., Wallin, A. I. N., Wang, H. H., &amp; Sommer, M. O. A. (2014). MODEST: a web-based design tool for oligonucleotide-mediated genome engineering and recombineering. Nucleic Acids Research, 42(W1), W408–W415. <A href="http://dx.doi.org/10.1093/nar/gku428" target="_blank">doi:10.1093/nar/gku428</A></LI></OL></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-md-4"><ADDRESS><STRONG>Technical University of Denmark</STRONG>
            Department of Systems Biology 
            Søltofts Plads 221 
            2800 Kgs. Lyngby
            Denmark<ABBR title="Phone">P:</ABBR> +45 45 25 25 25<ABBR title="Mail">M:</ABBR><A href="mailto:dtu-igem-2015@googlegroups.com">dtu-igem-2015@googlegroups.com</A></ADDRESS></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>