<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_DTU-Denmark_Project_Intein skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:DTU-Denmark/Project/Intein</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><NAV class="navbar navbar-default navbar-fixed-top" role="navigation"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#nav-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="nav-collapse"><UL class="nav navbar-nav"><LI><A href="/Team:DTU-Denmark">Home
                                           </A></LI><LI><A href="" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Team and Attributions
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Team">Team
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Attributions">Attributions
                                           </A></LI></UL><LI class="active"><A href="" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Project
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Project/Overview">Overview
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Project/Background">Background
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Project/MAGE">MAGE Subtilis
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Project/Surfactin">Surfactin
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Project/Tyrocidine">Tyrocidine
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Project/Chip">Lab-on-a-disc
                                           </A></LI><LI class="active"><A href="/Team:DTU-Denmark/Project/Intein">Inteins
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Project/Detection">Detection of NRP
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Project/POC_MAGE">Proof of concept of MAGE in B. subtilis
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Practices">Human Practices
                                           </A></LI><LI><A href="" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Parts Collection
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Parts">Parts
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Description">Characterization of xylR
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Journal">Journal
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Software">Software
                                           </A></LI><LI><A href="" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Achievements
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Achievements">Key Achievements
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Collaborations">Collaborations
                                           </A></LI><LI><A href="/Team:DTU-Denmark/Judging%20Form">Judging Form
                                           </A></LI><LI class="hidden-xs"><A href="http://facebook.com/dtubiobuilders" data-toggle="tooltip" data-placement="bottom" title="Follow us on Facebook"><SPAN class="visible-xs"> Follow us on Facebook</SPAN></A></LI><LI class="visible-xs"><A href="http://facebook.com/dtubiobuilders"> Follow us on Facebook</A></LI><LI class="visible-xs"><A href="https://twitter.com/IGEM_DTU"> Follow us on Twitter</A></LI></DIV></DIV></NAV><NAV class="navbar navbar-default extrabar hidden-xs hidden" data-spy="affix"><DIV class="container-fluid" id="scrollspy"><UL class="nav navbar-nav"><LI><A class="page-scroll" href="#Purpose">Purpose</A></LI><LI><A class="page-scroll" href="#Achievements">Achievements</A></LI><LI><A class="page-scroll" href="#Background">Background</A></LI><LI><A class="page-scroll" href="#Experimental-Design-and-Methods">Experimental Design and Methods</A></LI><LI><A class="page-scroll" href="#Results">Results</A></LI><LI><A class="page-scroll" href="#Discussion">Discussion</A></LI><LI><A class="page-scroll" href="#References">References</A></LI></UL></DIV></NAV><DIV class="jumbotron hero bg-hero"><DIV class="container-fluid"><DIV class="row hero-text"><DIV class="col-md-12"><H1>Inteins</H1><UL class="list-inline hidden-xs hidden-sm"><LI><A id="Purpose-submenu" class="btn btn-default btn-transparent btn-lg page-scroll" href="#Purpose">Purpose</A></LI><LI><A id="Achievements-submenu" class="btn btn-default btn-transparent btn-lg page-scroll" href="#Achievements">Achievements</A></LI><LI><A id="Background-submenu" class="btn btn-default btn-transparent btn-lg page-scroll" href="#Background">Background</A></LI><LI><A id="Experimental-Design-and-Methods-submenu" class="btn btn-default btn-transparent btn-lg page-scroll" href="#Experimental-Design-and-Methods">Experimental Design and Methods</A></LI><LI><A id="Results-submenu" class="btn btn-default btn-transparent btn-lg page-scroll" href="#Results">Results</A></LI><LI><A id="Discussion-submenu" class="btn btn-default btn-transparent btn-lg page-scroll" href="#Discussion">Discussion</A></LI><LI><A id="References-submenu" class="btn btn-default btn-transparent btn-lg page-scroll" href="#References">References</A></LI></UL><UL class="list-inline visible-xs visible-sm"><LI><A id="Purpose-submenu" class="page-scroll" href="#Purpose">Purpose</A></LI><LI><A id="Achievements-submenu" class="page-scroll" href="#Achievements">Achievements</A></LI><LI><A id="Background-submenu" class="page-scroll" href="#Background">Background</A></LI><LI><A id="Experimental-Design-and-Methods-submenu" class="page-scroll" href="#Experimental-Design-and-Methods">Experimental Design and Methods</A></LI><LI><A id="Results-submenu" class="page-scroll" href="#Results">Results</A></LI><LI><A id="Discussion-submenu" class="page-scroll" href="#Discussion">Discussion</A></LI><LI><A id="References-submenu" class="page-scroll" href="#References">References</A></LI></UL></DIV></DIV><DIV class="row"><DIV class="col-md-12">
            Scroll down for more</DIV></DIV></DIV></DIV><DIV id="Purpose"><DIV class="container"><DIV class="row col-md-12"><H1>
        Purpose
      </H1><P>We did not succeed in expression of tyrocidine from <I>Brevibacillus parabrevis </I>in <I>Bacillus subtilis </I>[<A href="https://2015.igem.org/Team:DTU-Denmark/Project/Tyrocidine">here</A>]. This inspired us to think of alternative approaches for production of nonribosomal peptide drugs than through NRPS engineering. We came up with a peculiar idea of generating short cyclized peptides with similar length to tyrocidine by using self-splicing proteins. Inteins are such short self-splicing proteins that have no function in the proteins they are a part of, besides catalyzing their own excision after translation. The splicing makes a peptide bond between the two adjacent amino acids next to the inteins (Figure 1).  If the N-intein and C-intein are placed at each end of the same polypeptide; cyclisation would give a cyclized protein - or in our case a short cyclic peptide cyclized from head-to-tail.</P></DIV></DIV></DIV><DIV id="Achievements"><DIV class="container"><DIV class="row col-md-12"><H1>
        Achievements
      </H1><P style="text-align: justify;"> </P><UL><LI style="text-align: justify;">Designed constructs for expression of two short cyclic peptides using BioBrick <A href="http://parts.igem.org/Part:BBa_K1362000" target="_blank">BBa_K1362000</A> and two short oligo primers.</LI><LI style="text-align: justify;">Generated the cyclisation construct</LI><LI style="text-align: justify;">Analyzed constructs for cyclic peptides, but no products could be detected.</LI></UL></DIV></DIV></DIV><DIV id="Background"><DIV class="container"><DIV class="row col-md-12"><H1>
        Background
      </H1><TABLE align="right" border="0" cellpadding="1" cellspacing="1" style="margin-left: 10px; font-size: 8pt; width: 300px;"><TBODY><TR><TD style="text-align: justify;"><STRONG>Figure 1</STRONG> N-intein and C-intein is spliced out by forming a leftover extein scar of the size of the N-extein and C-extein. The size and amino acid sequenceof the scar depends</TD></TR></TBODY></TABLE><P style="text-align: justify;">Nonribosomal peptide synthetases (NRPS) are not exclusive in their ability to produce short cyclic peptides. CyBase contains 818 entries of ribosomal head-to-tail cyclic peptides from 105 different species [1]. The peptides vary from 8 to 80 amino acids in length and in number of internal cysteine bridges that stabilizes the cyclic peptide [2]. Ribosomal cyclic peptides are potential drug candidates and their synthesis may be exploited to produce peptide drugs with increased stability or bioavailability. In one mechanism of cyclization (Figure 1), self-splicing proteins (inteins) bind and catalyze their own excision, while forming a peptide bond between the N-extein and C-extein. Numerous of inteins have been characterized, including split intein Npu DnaE from <I>Nostoc punctiforme</I> PCC73102 (Npu) (Figure 1) [3][4]. When the Npu DnaE intein catalyzes its own excisions, it leaves a six amino acid scar behind. The scar is called the extein. Npu DnaE has a native scar of “AEY/CFN”. This scar can in large parts be changed to other amino acids to better fit a potential synthetic peptide target [4].</P><P style="text-align: justify;"> </P></DIV></DIV></DIV><DIV id="Experimental-Design-and-Methods"><DIV class="container"><DIV class="row col-md-12"><H1>
        Experimental Design and Methods
      </H1><P style="text-align: justify;">The BioBrick <A href="http://parts.igem.org/Part:BBa_K1362000" target="_blank">BBa_K1362000</A> was added by Heidelberg to the iGEM registry in 2014. It contains Npu DnaE inteins surrounding a RFP selection marker, which can be cut out with restriction enzyme <I>BsaI</I>. Digestion of BBa_K1362000 with <I>BsaI</I><I> </I>generates to different overhangs, thus making it possible to insert the coding sequence (CDS) for your protein of interest. The BioBrick was used to cyclize different proteins. To our knowledge, it was not used to cyclize short peptides.</P><P style="text-align: justify;">We decided to try and cyclise a synthetic 10 amino acid peptide which has similar amino acid sequence to tyrocidine. In addition, we searched the cyclic peptide database for bioactive cyclic peptide, which contained part or the entire or part of the extein scar in its native sequence. We selected a hit for a cyclotides with 33 amino acids. The sequences for both native peptides along with the sequence chosen for cyclization experiments are shown in Table 1.</P><TABLE border="0" cellpadding="0" cellspacing="1"><TBODY><TR><TD><SPAN style="font-size:12px;"><STRONG>Table 1</STRONG> Peptides selected for this study. Npu DnaE requires a cystine at position 1 in the extein[4]. For tyrocidine the native extein sequence was kept. </SPAN></TD></TR><TR><TH scope="row"> </TH><TH scope="col">Cycloviolacin Y1 (33 aa)[3]</TH><TH scope="col">Tyrocidine (10 aa)</TH></TR><TBODY><TR><TH scope="row" style="text-align: left;">Native</TH><TD>GGTIFDCGETCFLGTCYTPGCSCGNYGFCYGTN</TD><TD>F<SUB>D</SUB>PF<SUB>D</SUB>NQYVOrnL</TD></TR><TR><TH scope="row" style="text-align: left;">Synthetic</TH><TD>GGTIFDC<U>GETCFL</U>GTCYTPGCSCGNYGFCYGTN</TD><TD><U>EYCFN</U>QYVK<U>A</U></TD></TR><TR><TH scope="row" style="text-align: left;">Extein (N/C)</TH><TD>GET/CFL</TD><TD>AEY/CFN</TD></TR></TBODY></TBODY></TABLE><P style="text-align: justify;">The peptide insert was created by designing two complementary oligos that can anneal to each other, except at their 5’ ends where they are compatible with the BsaI-generated overhangs (Table 2):</P><TABLE border="0" cellpadding="0" cellspacing="1"><TBODY><TR><TD><SPAN style="font-size:12px;"><STRONG>Table 2</STRONG> Oligos ordered for cycloviolacin Y1 and tyrocidine-like (see Table 1). Oligos were annealed by heating 10µM of the paired oligos to 94° C in a thermocycler. After 2 minutes the thermocycler was turned off to allow the oligos to anneal while cooling down.</SPAN></TD></TR><TR><TH scope="row" style="text-align: left;">Cycloviolacin Y1</TH><TD><P><SPAN style="font-size:14px;"><SPAN style="font-family:courier new,courier,monospace;">5'-CAACTGCTTCCTTGGCACATGCTACACACCTGGCTGCTCTTGCGGCAACTACGGCTTCTGCTACGGCACAAACGGCGGCACAATCTTCGATTGCGGCGAAACA-3'</SPAN><SPAN style="font-family:courier new,courier,monospace;">   3'-ACGAAGGAACCGTGTACGATGTGTGGACCGACGAGAACGCCGTTGATGCCGAAGACGATGCCGTGTTTGCCGCCGTGTTAGAAGCTAACGCCGCTTTGTACGA-3'</SPAN></SPAN></P></TD></TR><TR><TH scope="row" style="text-align: left;">Tyrocidine-like</TH><TD><SPAN style="font-size:14px;"><SPAN style="font-family:courier new,courier,monospace;">5'-CAACTGCTTCAACCAATACGTTAAAGCTGAATAC-3'   3'-ACGAAGTTGGTTATGCAATTTCGACTTATGACGA-5'</SPAN></SPAN></TD></TR></TBODY></TABLE><DIV aria-multiselectable="true" class="panel-group" id="accordion" role="tablist"><DIV class="panel panel-default"><DIV class="panel-heading" id="headingOne" role="tab"><H4 class="panel-title"><A aria-controls="collapseOne" aria-expanded="false" class="collapsed" data-parent="#accordion" data-toggle="collapse" href="#collapseOne" role="button">Generation of constructs for expression of cyclic peptides</A></H4></DIV><DIV aria-labelledby="headingOne" class="panel-collapse collapse" id="collapseOne" role="tabpanel"><DIV class="panel-body"><A href="http://parts.igem.org/Part:BBa_K1362000">BBa_K1362000</A> was digested for 8 hours with restriction enzyme <I>BsaI</I>, which was then heat-inactivated at 65ºC for 20 minutes. Oligos were annealed in PCR thermocycler by heating 10µM oligos up to 94ºC for 2 minutes before turning of the machines. The sample was allowed to return to room temperature before it was ligated by mixing 1µL 10µM annealed oligos and 50ng/µL of digested construct with T4 DNA ligase (#M0202). Reaction was setup according to the manufacture's protocol. 1µL ligation mix was transformed into chemical competent NEB <I>DH5α E. coli</I> cells (#C2987) according to the manufacture's protocol. Transformants were selected overnight on LB agar plates containing 6µg/mL chloramphenicol.

<P>Transformants with the cycloviolacin Y1 or tyrocidine split-intein casette were selected and grown overnight in 3mL LB media containing 6µg/mL chloramphenicol. Plasmids were purified using Qiaprep spin columns (#27104). The expression casette was combined with promoter <A href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_J23101">BBa_J23101</A> from Anderson's promoter library. BBa_J23101 was digested with <I>EcoRI </I>and <I>SpeI </I>and the split-intein expression cassette was digested with <I>EcoRI</I> and <I>XbaI </I>for 15 minutes and subsequently heat-inactivated at 65ºC for 5 minutes. The digested expression cassette and BBa_J23101 was mixed with T4 DNA ligase in 1:3 and 1µL ligation product was transformed into chemical competent NEB <I>DH5α E. coli</I> cells, as previously described. Transformants were selected overnight on LB agar plates containing 6µg/mL chloramphenicol.</P></DIV></DIV></DIV></DIV><DIV aria-multiselectable="true" class="panel-group" id="accordion2" role="tablist"><DIV class="panel panel-default"><DIV class="panel-heading" id="headingTwo" role="tab"><H4 class="panel-title"><A aria-controls="collapseTwo" aria-expanded="false" class="collapsed" data-parent="#accordion2" data-toggle="collapse" href="#collapseTwo" role="button">Detection of cyclic peptides by UHPLC-DAD-QTOFMS and SDS-PAGE</A></H4></DIV><DIV aria-labelledby="headingTwo" class="panel-collapse collapse" id="collapseTwo" role="tabpanel"><DIV class="panel-body">The presence of cyclic peptides was determined by UHPLC-DAD-QTOFMS. Peptides were extracted using methanol extraction. A detailed protocol for detection can be found <A href="https://2015.igem.org/Team:DTU-Denmark/Project/Detection">here</A>.

<P>Expression of inteins was analyzed by SDS-PAGE using Novex NuPAGE. Samples were boiled with 1x SDS in presence of a reducing agent. The protein gel was stained using Coomassie Blue and samples were run according to the manufacture’s protocol.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="Results"><DIV class="container"><DIV class="row col-md-12"><H1>
        Results
      </H1><P style="text-align: justify;">Transformants expressing the short peptide flanked by Npu DnaE intein (<A href="http://parts.igem.org/Part:BBa_K1362000" target="_blank"><U>BBa_K1362000</U></A>) and under control of promoter <A href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_J23101" target="_blank"><U>BBa_J23101</U> </A>were selected and re-streaked in order to grow enough biomass for detection of peptides by chromatography. Due to time constraints, the sequence of the transformants were not confirmed. Instead, six samples for each peptide (total of 12) were analyzed by UHPLC-DAD-QTOFMS. It was not possible to detect the cyclized protein from methanol extraction of secondary metabolites. In order to confirm whether this was due to improper splicing of intein, cell lysates were analyzed by SDS-PAGE. The size of the unspliced protein iwas 19.4 kDA (cycloviolacin Y1) and 17.2 kDa (tyrocidine-like). It was not possible to detect bands at this molecular weight on the gel (data not shown). The size of the cyclic peptide was below the lower detection limit for SDS-PAGE.</P></DIV></DIV></DIV><DIV id="Discussion"><DIV class="container"><DIV class="row col-md-12"><H1>
        Discussion
      </H1><P>It was not possible to produce short cyclic peptides using the Npu DnaE. No unspliced variant were detected by SDS-PAGE and no cyclized peptides were detected by LC/MS. This may be explained by (1) a sequence mistake in the construct or incorrect integration of the promoter and/or (2) that the peptide was not proper cyclized. As 12 colonies were sequenced, it seems unlikely that a mutation in the oligo should be present in all transformants. The promoter may not have been inserted correctly, but the results are contradicting. Lack of cyclisation may be due to either its small size (tyrocidine) or the fact that the extein sequence was mutated from its native for expression of cycloviolacin Y1. It may also not be possible to distinguish the unspliced intein-peptide due to background in <EM>E. coli</EM>. Cyclisation of cylotides (cycloviolacin Y1 is a cyclotide) by Npu DnaE has been achieved in <EM>E. coli</EM>. [4]. It has however to our knowledge not been possible to use Npu DnaE to successfully cyclize a 10 amino acid peptide. It may be possible to achieve cyclisation of these peptides by exploring the diversity of inteins used by different organisms [5]. Heidelberg tested an <A href="https://2014.igem.org/Team:Heidelberg/Parts#Intein%20Library" target="_blank"><U>intein library</U></A>, which may be a start point for such intein screening in the future.</P></DIV></DIV></DIV><DIV id="References"><DIV class="container"><DIV class="row col-md-12"><H1>
        References
      </H1><OL><LI>Lipmann, F., Roskoski, R., Gevers, W., &amp; Kleinkauf, H. (1970). Tyrocidine biosynthesis by three complementary fractions from Bacillus brevis (ATCC 8185). Biochemistry, 9(25), 4839–4845. <A href="http://dx.doi.org/10.1021/bi00827a002" target="_blank">doi:10.1021/bi00827a002</A></LI><LI>Mootz, H.D., &amp; Marahiel, M.A. (1997). The tyrocidine biosynthesis operon of Bacillus brevis: Complete nucleotide sequence and biochemical characterization of functional internal adenylation domains. Journal of Bacteriology, 179(21):6843-50</LI><LI>Thorstholm L, Craik DJ. Discovery and applications of naturally occurring cyclic peptides. Drug Discov Today Technol. 2012;9:e13–e21. doi: 10.1016/j.ddtec.2011.07.005.</LI><LI>Jagadish, K., Borra, R., Lacey, V., Majumder, S., Shekhtman, A., Wang, L., Camarero, J.A. Expression of fluorescent cyclotides using protein trans-splicing for easy monitoring of cyclotide-protein interactions. Angew Chem Int Ed Engl. 2013 March 11; 52(11): 3126–3131.</LI><LI>Shah, N. H., &amp; Muir, T. W. (2014). Inteins: nature’s gift to protein chemists. Chem. Sci., 5(2), 446–461. <A href="http://dx.doi.org/10.1039/c3sc52951g" target="_blank">doi:10.1039/c3sc52951g</A></LI></OL></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-md-4"><ADDRESS><STRONG>Technical University of Denmark</STRONG>
            Department of Systems Biology 
            Søltofts Plads 221 
            2800 Kgs. Lyngby
            Denmark<ABBR title="Phone">P:</ABBR> +45 45 25 25 25<ABBR title="Mail">M:</ABBR><A href="mailto:dtu-igem-2015@googlegroups.com">dtu-igem-2015@googlegroups.com</A></ADDRESS></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>