"
Page
Discussion
View source
History
teams
Log in
 
Team:KU Leuven/Journal/MeS/qPCR
From 2013.igem.org
iGem
Secret garden
Congratulations! You've found our secret garden! Follow the instructions below and win a great prize at the World jamboree!
A video shows that two of our team members are having great fun at our favourite company. Do you know the name of the second member that appears in the video?
For one of our models we had to do very extensive computations. To prevent our own computers from overheating and to keep the temperature in our iGEM room at a normal level, we used a supercomputer. Which centre maintains this supercomputer? (Dutch abbreviation)
We organised a symposium with a debate, some seminars and 2 iGEM project presentations. An iGEM team came all the way from the Netherlands to present their project. What is the name of their city?
Now put all of these in this URL:https://2013.igem.org/Team:KU_Leuven/(firstname)(abbreviation)(city), (loose the brackets and put everything in lowercase) and follow the very last instruction to get your special jamboree prize!
Menu
Data Page
Project
Aphid Background
Honeydew system
Sticker system
E. coligy
Parts
Modelling
Cellular level
Colony level
Ecosystem level
Team
Students & Advisors
Attributions
Collaborations
Sponsors
Official Team Profile
Human Practices
Top Down
Bottom Up: Framework
Bottom Up: Public Outreach
Notebook
Journal
Protocols
Safety
iGEM
 
Modelling
Modelling our different systems
Ecology
Plant and insect experiments
E-β-farnesene
Work about our EBF-biobrick
Feed forward loop
Implementation of oscillator in vivo
Methyl salicylate
Work about the MeS-biobrick
qPCR
You are here!
qRT-PCR
To find some more data for his model, Sander did qPCR experiments. To find out more about why we did these experiments go here . These experiments were, as Sander said: "A story about love, hate, confusion and disappointments". If you want to find out why, read his journal below. Sander would like to thank the people at the Centre of Microbial and Plant Genetics , where the experiments were perfomed. A special thank you goes to Tine Verhoeven, Elke Van Assche and David De Coster for their supervision and help.
22/07 – 26/07
Week 4: Let's get this started!
The first preparations for performing the qPCR were done. We contacted the CMPG, a laboratory from the faculty of Bioscience Engineering (FBSE), that is willing the help us and provide us the space for our qRT-PCR experiment.
The first four steps of the protocol ‘qRT_PCR_Protocol’ were followed with success:
We now have a fresh plate with an E. coli strain harbouring the methyl salicylate MIT brick (2006; BBa_J45700)
We have the isolated plasmid DNA from this very same brick with a concentration of 155 ng/µl.
We successfully designed primers for pchA/pchB and for bsmt1 using Primer Express Software (Applied Biosystems, Foster CA).
After we got this, we started with the preparation of three samples according to the ‘qRT_PCR_Protocol’ (step 5 – 9 and step 12).
29/07 – 02/08
Week 5: DNA contamination?
We continued with the isolation of mRNA explained in step 13 of the ‘qRT_PCR_Protocol’. This protocol was followed without any problems. We then converted a part of the isolated mRNA into cDNA (step 14 of the ‘qRT_PCR_Protocol’).
Before you start with the actual qPCR, it’s a good idea to check whether all the DNA in your RNA-isolation is fully degraded and removed to prevent any false positive results. Therefore we ran a regular PCR, which amplifies a plasmid DNA region, to check if there is still any DNA contamination. Because all of the DNA should be degraded, we should not be able to see any bands after a PCR, unless for our positive control. We put this PCR on gel and got the following result:
Because also our negative control gives a very clear band, we concluded that we have some contamination somewhere in one of our PCR products. So we had to retry this PCR with other products and it took a while until we finally got a  negative control which was actually negative:
Unfortunately it is clear that we still have DNA-contamination in our RNA-sample, even after a DNase treatment.
19/08 – 23/08
Week 8: LET’S DO THIS! (again)
Because of the DNA-contamination we decided to start this experiment all over again. This time with closer assistance from a more experienced RNA-expert.
We prepared the samples again using the same protocol (‘qRT_PCR_Protocol’) as we did the first time but now included a few extra steps where we flash freeze the samples in liquid nitrogen (step 1-12). Then we also did the RNA-isolation (step 13), and again checked for any DNA-contamination.
We first checked whether we had any genomic DNA-contamination by doing a PCR on a genomic region of our E. coli (csrA gene). The results were very good:
Only our positive control showed a decent band, which means that we have successfully removed the genomic DNA out of our RNA-sample. (The small bands are primer-dimers.)
So with good expectation we started a new PCR to check if our plasmid DNA is completely removed with the following result:
Unfortunately it is pretty clear that the plasmid DNA is still present in the sample. Even with a good DNase treatment, which clearly works on the genomic DNA but it does not manage to break down the plasmid DNA.
So we did some troubleshooting and mailed to the company whose DNase treatment we’ve used. Apart from that we’ve also contacted dr. Basak Ozturk, who had some previous experience with qPCR on a plasmid region. Soon we discovered that doing a qPCR on plasmid DNA, especially a high copy number plasmid, is a very delicate case and almost impossible to remove the DNA contamination using regular DNase treatments.
26/08 – 30/08
Week 9: Overheating our brains
The troubleshooting goes on this week. We thought about a lot of different ways to solve our problem with the contaminating DNA, without damaging our precious and fragile RNA. But all of them were either too dangerous for our RNA, or impossible to do in the timeframe we had.
The only one were we might have a chance for success is doing a restriction digest, with an enzyme that cuts the fragments that will be amplified in the qPCR itself. This way the contaminating DNA will not be amplified while the cDNA will be correctly amplified and we could take unbiased conclusions. So we went looking for a good restriction enzyme that does the job perfectly for the 2 genes.
We came to the restriction enzyme MvaI. This should cut our plasmid on the right places as shown on the image below:
The MvaI restriction enzyme should cut inside both of the qPCR amplified regions (the regions between IGEM00019&IGEM00020 and IGEM0021&IGEM0022), so that only linear amplification is possible there and that interference should be eliminated.
After all this thinking we ordered the restriction enzyme and just sat back and relaxed (read: Do lots of other work).
09/09 – 13/09
Week 11: Attack of the scissors
First we checked if our RNA concentration was still high enough and intact by doing a nanodrop. The 260/280 values (2.00;1.93;1.97) were still convincing enough to conclude that our RNA is intact and we still have a good concentration to work with.
So we decided to go on with the digest. We took 30 µl of our sample and digested it with MvaI. Again, we ran some PCR’s and gels to check whether we finally got rid of our contaminating DNA.
It might not be clear on the images but all of our samples still show bands which means that our digestion-trick did not work.
The qPCR adventure ends here and we learned one very important lesson: Want to do a qPCR? Then clone it into your genome or use a low copy number plasmid!
Retrieved from " http://2013.igem.org/Team:KU_Leuven/Journal/MeS/qPCR "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
