<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_KU_Leuven_Project_Glucosemodel_MeS_Modelling-FBA"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:KU_Leuven/Project/Glucosemodel/MeS/Modelling-FBA">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:KU_Leuven/Project/Glucosemodel/MeS/Modelling-FBA&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:KU_Leuven/Project/Glucosemodel/MeS/Modelling-FBA&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:KU_Leuven/Project/Glucosemodel/MeS/Modelling-FBA&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:KU_Leuven/Project/Glucosemodel/MeS/Modelling-FBA" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:KU Leuven/Project/Glucosemodel/MeS/Modelling-FBA</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>iGem</TITLE></P><DIV class="container"><DIV id="topbar"><DIV id="secret"><DIV class="row-fluid"><DIV class="span12"><DIV class="row-fluid"><DIV class="span9"><H3>Secret garden</H3><P style="color:white">Congratulations! You've found our secret garden! Follow the instructions below and win a great prize at the World jamboree!</P><UL style="color:white"><LI>A video shows that two of our team members are having great fun at our favourite company. Do you know the name of the <B>second member</B> that appears in the video?</LI><LI>For one of our models we had to do very extensive computations. To prevent our own computers from overheating and to keep the temperature in our iGEM room at a normal level, we used a supercomputer. Which <B>centre</B> maintains this supercomputer? (Dutch abbreviation)</LI><LI>We organised a symposium with a debate, some seminars and 2 iGEM project presentations. An iGEM team came all the way from the Netherlands to present their project. What is the name of their <B>city</B>? </LI></UL><P style="color:white">Now put all of these in this URL:https://2013.igem.org/Team:KU_Leuven/(firstname)(abbreviation)(city), (loose the brackets and put everything in lowercase) and follow the very last instruction to get your special jamboree prize!</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="container"><DIV class="container"><DIV class="row-fluid"><DIV id="menu"><A class="toggleMenu" href="#">Menu</A><UL class="nav"><LI class="test"><A href="https://2013.igem.org/Team:KU_Leuven/Project/DataPage">Data Page</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Project">Project</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Project/Aphid_Background">Aphid Background</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Project/HoneydewSystem">Honeydew system</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Project/StickerSystem">Sticker system</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Project/E.coligy"><I>E. coligy</I></A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Parts">Parts</A></LI></UL><LI><A href="https://2013.igem.org/Team:KU_Leuven/Project/modelling">Modelling</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Project/Modelling/Cellular_Level">Cellular level</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Project/Modelling/Colony_Level">Colony level</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Project/Modelling/Ecosystem_Level">Ecosystem level</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Team">Team</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Team/Students">Students &amp; Advisors</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Team/Attributions">Attributions</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Team/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Sponsors">Sponsors</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013">Official Team Profile</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Human_Practices">Human Practices</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Human_Practices/Top_Down">Top Down</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Human_Practices/Bottom_Up_Philosopher">Bottom Up: Framework</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Human_Practices/Bottom_Up_Public">Bottom Up: Public Outreach</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Journal">Journal</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Protocols">Protocols</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Safety">Safety</A></LI><LI><A href="https://igem.org/Main_Page">iGEM</A></LI><DIV class="clear-both"> </DIV></DIV></DIV></DIV></DIV></DIV><DIV id="container"><DIV class="container"><DIV id="header" class="row-fluid"><DIV class="span12"><H3 class="bg-green">Modelling on cellular level</H3></DIV></DIV><DIV class="row-fluid"><DIV class="span6 icon white"><DIV class="row-fluid"><DIV class="span9 icon-text"><H3>Flux Balance Analysis</H3><P>You are here!</P></DIV></DIV></DIV><DIV class="span6 icon white"><DIV class="row-fluid"><DIV class="span9 icon-text"><H3>Kinetic Parameters</H3><P>BanAphids MeS production?</P></DIV></DIV></DIV></DIV><DIV id="header" class="row-fluid"><DIV class="span12"><H3 class="bg-oscillator">Flux Balance Analysis on Methyl Salicylate</H3></DIV></DIV><DIV class="row-fluid"><DIV class="span12 white"><P align="justify">
We needed to check whether the introduction of our MeS brick and the production of the components influences the overall BanAphid metabolism and/or growth rate. This modelling results will be checked against wetlab data, namely the growth curves we obtained while characterising our MeS biobricks.

We also composed a Kinetic Parameter Model to estimate the average production rate of MeS. Approach and results can be found <A href="https://2013.igem.org/Team:KU_Leuven/Project/Glucosemodel/MeS/Modelling">here</A>.</P></DIV></DIV><DIV id="header" class="row-fluid"><DIV class="span12"><H3 class="bg-oscillator">Matlab: COBRA Toolbox</H3><H3><DIV class="row-fluid"><DIV class="span12 white"><P align="justify"> A FBA calculates possibilities for the flow of metabolites through a metabolic network while maximising a set objective, in our case the growth rate of an organism or the production of a biotechnologically important metabolite.
We ran the FBA for methyl salicylate using the <A href="http://opencobra.sourceforge.net/openCOBRA/Welcome.html" target="_blank">COBRA Toolbox</A> for MATLAB.<B>COBRA</B> stands for <B>Constraint-Based Reconstruction and Analysis</B> (COBRA) approach. It provides a biochemically and genetically consistent framework for the generation of hypotheses and the testing of functions of microbial cells. The <B>Flux Balance Analysis (FBA)</B> is probably the most used analysis within COBRA. <B>COBRA has been successfully applied to study the possible phenotypes that arise from a genome </B>(Covert, Schilling <I>et al.</I> 2001; Orth <I>et al.</I> 2010). COBRA consists of two fundamental steps.
First, a GENRE (=GEnome-scale Network REconstruction) is formed, composed of the mathematical representation of all known metabolic reactions.
Second, the appropriate constraints are applied to form the corresponding GEMS (GEnome-scale Model <I>in Silico</I>). 

</P><P align="justify">Two fundamental types of constraints exist: <B>balances and bounds</B> (Price, Reed <I>et al.</I> 2004). Balance constraints are associated with conserved quantities such as energy, mass etc. Bounds limit numerical ranges of individual variables and parameters such as concentrations, fluxes or kinetic constants. <B>At steady state, there is no accumulation or depletion of metabolites in a metabolic network, so the production rate of each metabolite in the network must equal its rate of consumption.</B> This balance of fluxes can be represented mathematically as S . v = 0, where v is a vector of fluxes through the metabolic network and S is the stoichiometric matrix containing the stoichiometry of all reactions in the network.

Both bound and balance constraints limit the allowed functional states of reconstructed networks. Constraints can be very diverse in a biological system : physico-chemical constraints (reaction rates, enzyme turnover rates, diffusion rates etc.) , topo-biological (e.g. organisation of DNA in <I>Escherichia coli</I> by spatio-temporal patterns (Huang, Zhang <I>et al.</I> 2003)), environmental (nutrient availability, pH, temperature, osmolarity and the availability of electron acceptors and regulatory constraints).In mathematical terms, the constraints define a system of linear equations which will be solved by linear programming in FBA. This will result in a range of allowable network states, described by a solution space which, in biology, represents the phenotypic potential of an organism. All allowable network states are contained in this solution space. (Covert and Palsson 2003; Price, Papin <I>et al.</I> 2003)
Thus, we can predict the growth rate potential of our BanAphids, defined by the constraints we impose, e.g. the growth medium, temperature, co-factor/precursor presence etc.</P></DIV></DIV></H3><H3 class="bg-oscillator">Results: what Did COBRA Do For us?</H3></DIV></DIV><DIV class="row-fluid"><DIV class="span12 white"><P align="justify">
We used an <I>E. coli</I> model from 2007 (iAF1260 by Feist, AM. <I>et al.</I>) in all the following COBRA toolbox analyses.<B>As a first step we tried to predict the growth under default conditions for this model</B>. This gave us the following results (after setting the biomass as objective function): for <I>E. coli</I> under default conditions a growth rate of 0.74 hr<SUP>-1</SUP> is predicted. When performing the same calculations, but for LB medium conditions, a growth rate of 5.34 hr<SUP>-1</SUP> is predicted. This shows that <I>E. coli</I> benefit from the LB medium conditions.
  </P><P align="justify"><B>In a next step we wanted to add the reactions which are necessary for our model, but lacking in the iAF1260 model.</B>
We added 'pchA', 'chor[c] -&gt; ichor[c]' for the isochorismate synthesis reaction, 'pchB', 'ichor[c] -&gt; sali[c] + pyr[c]' for the salicylate synthesis reaction and 'BSMT1', 'sali[c] -&gt; methylsalicylate' for the methyl salicylate synthesis reaction. We also added the exchange reaction for methylsalicylate ('Ex_methylsalicylate').
When we performed the growth calculation analysis for this modified model with the biomass set as objective function, we also observed a growth rate of 0.737 hr<I>-1</I></P><P align="justify"><B>Since our bacteria will be grown on LB medium, we changed the default medium settings towards those for LB medium.</B> This means that we changed the relevant exchange reactions for the metabolites present in LB medium as seen in <I>Tawornsamretkit et al.</I>. The lower reaction bounds of the relevant reactions were set as following:
 model = changeRxnBounds (model,'EX_glc(e)',0,'l')
model = changeRxnBounds (model,'EX_phe_L(e)',-0.1,'l')
model = changeRxnBounds (model,'EX_cys_L(e)',0,'l')
model = changeRxnBounds (model,'EX_ile_L(e)',-0.089,'l')
model = changeRxnBounds (model,'EX_ins(e)',-0.1,'l')
model = changeRxnBounds (model,'EX_hxan(e)',-0.1,'l')
model = changeRxnBounds (model,'EX_h2o(e)',-1000,'l')
model = changeRxnBounds (model,'EX_o2(e)',-1000,'l')
model = changeRxnBounds (model,'EX_co2(e)',-1000,'l')
model = changeRxnBounds (model,'EX_nh4(e)',-1000,'l')
model = changeRxnBounds (model,'EX_so4(e)',-1000,'l')
model = changeRxnBounds (model,'EX_ca2(e)',-1000,'l')
model = changeRxnBounds (model,'EX_h(e)',-1000,'l')
model = changeRxnBounds (model,'EX_k(e)',-1000,'l')
model = changeRxnBounds (model,'EX_mg2(e)',-1000,'l')
model = changeRxnBounds (model,'EX_na1(e)',-1000,'l')
model = changeRxnBounds (model,'EX_fe3(e)',-1000,'l')
model = changeRxnBounds (model,'EX_nac(e)',-1000,'l')
model = changeRxnBounds (model,'EX_thym(e)',-1000,'l')
model = changeRxnBounds (model,'EX_ade(e)',-1000,'l')
model = changeRxnBounds (model,'EX_thr_L(e)',-0.288,'l')
model = changeRxnBounds (model,'EX_val_L(e)',-0.071 ,'l')
model = changeRxnBounds (model,'EX_pro_L(e)',-0.1,'l')
model = changeRxnBounds (model,'EX_his_L(e)',-1.642,'l')
model = changeRxnBounds (model,'EX_leu_L(e)',-0.1,'l')
model = changeRxnBounds (model,'EX_ura(e)',-1000,'l')
model = changeRxnBounds (model,'EX_tyr_L(e)',-0.035 ,'l')
model = changeRxnBounds (model,'EX_trp_L(e)',0,'l')
model = changeRxnBounds (model,'EX_met_L(e)',-0.1,'l')
model = changeRxnBounds (model,'EX_ser_L(e)',-1.722,'l')
model = changeRxnBounds (model,'EX_arg_L(e)',-1.17,'l')
model = changeRxnBounds (model,'EX_asp_L(e)',-0.041,'l')
model = changeRxnBounds (model,'EX_lys_L(e)',-0.1,'l')
model = changeRxnBounds (model,'EX_lys_L(e)',-0.1,'l')
model = changeRxnBounds (model,'EX_ala_L(e)',-0.369,'l')
model = changeRxnBounds (model,'EX_zn2(e)',-1000,'l')
model = changeRxnBounds (model,'EX_cd2(e)',-1000,'l')
model = changeRxnBounds (model,'EX_glyc(e)',-0.014,'l')
model = changeRxnBounds (model,'EX_gln_L(e)',-0.445,'l')
model = changeRxnBounds (model,'EX_glu_L(e)',0,'l')
model = changeRxnBounds (model,'EX_leu_L(e)',-0.235,'l')
model = changeRxnBounds (model,'EX_met_D(e)',-0.084,'l')
model = changeRxnBounds (model,'EX_met_L(e)',-0.084,'l')
model = changeRxnBounds (model,'EX_tre(e)',-0.6,'l')</P><P align="justify">When we performed the growth calculation with the biomass as objective function the flux to chorismate(a precursor of methyl salicylate) was 0.274 mol/hr in <B>non-LB medium</B> conditions and the growth of <I>E. coli</I> 0.737 hr<SUP>-1</SUP>. When we do the same but for the <B>LB medium conditions</B> we observe the flux to chorismate as 0.80 mol/hr and a growth rate of 5.34 hr<SUP>-1</SUP>. This suggests that LB-medium is beneficial for <I>E. coli</I> growth and improves the flux towards chorismate.  Our aroG BioBrick <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1060000">(Part:BBa_K1060000)</A> approach, aims however for a higher flux towards chorismate. This BioBrick contains mutations that can prevent the repression by Phenylalanine, that would occur otherwise and is in favour of chorismate production at the same time. </P><P align="justify">We were also interested to see how the maximal production of MeS is related to the maximal growth of <I>E. coli</I> under minimal conditions and in LB medium conditions. Therefore we set the objective function to MeS and set the lower bound for the biomass at different percentages of the maximal growth rate predicted with the biomass as objective function.</P><P align="justify">As can be seen in the graph, there is a linear correlation between maximal flux towards MeS and maximal <I>E. coli</I> growth. This linear correlation is qualitatively similar both under minimal growth conditions as in LB medium conditions and shows that we are dealing with a trade off between bacterial growth and MeS production.
If a higher flux to MeS is preferred over a lower <I>E. coli</I> growth mass a value at the left side of the graph should be considered, whereas a value to the right would give a higher <I>E. coli</I> production rate and a lower flux towards MeS. This trade-off is reminiscent of our wet-lab growth curve results for the MeS brick.</P><P align="justify">This figure shows how higher concentrations of added salicylate (0.1 mM) result in a longer lag phase. Higher levels of salicylate may lead to higher levels of S-adenosylmethionine (SAM) consumption, a co-substrate of the methyltransferase reaction producing methylsalicylate from salicylate. These higher consumption levels can in turn be associated with higher homocysteine levels, a side-product that stays behind when the methylgroup has been transferred to salicylate. Increased homocysteine levels are toxic for <I>E. coli</I> strains (Tuite <I>et al.</I> which may explain the ceiling we observe when adding salicylate, hoping for higher MeS fluxes.</P></DIV></DIV><DIV id="header" class="row-fluid"><DIV class="span12"><H3 class="bg-oscillator">Conclusion</H3></DIV></DIV><DIV class="row-fluid"><DIV class="span12 white"><P align="justify">FBA analysis predicted LB medium conditions to be beneficial for both <I>E. coli</I> growth and the flux towards chorismate, an important precursor for methylsalicylate. Moreover, a linear correlation between maximal predicted MeS flux and maximal growth rate exists, showing a <B>trade off</B> between the two. This linear correlation is present not only for LB medium conditions but also for minimal medium conditions and is qualitatively similar, showing a steeper correlation for LB medium conditions. <B>These trade-offs are not only present <I>in silico</I> but we also found them <I>in vivo</I></B> in the choice between methyl salicylate production and growth rate. Finally, also our GC-MS analysis fits with this modelling, when salicylate was added to the medium, we not only observed a reduced growth rate but also a distinct MeS production peak.
These results not only show correlation between wetlab and modelling data but also suggest that cellular level modelling can result in colony wide effects (as observed in the delayed growth).  
 </P></DIV></DIV><DIV id="header" class="row-fluid"><DIV class="span12"><H3 class="bg-oscillator">References</H3></DIV></DIV><DIV class="row-fluid"><DIV class="span12 white"><P align="justify">
Covert, M. W. and B. O. Palsson (2003). &quot;Constraints-based models: regulation of gene expression reduces the steady-state solution space.&quot; J Theor Biol 221(3): 309-325.
Covert, M. W., C. H. Schilling, I. Famili, J. S. Edwards, Goryanin, II, E. Selkov and B. O. Palsson (2001). &quot;Metabolic modeling of microbial strains in silico.&quot; Trends Biochem Sci 26(3): 179-186. 
Feist, AM <I>et al.</I> (2007) “A genome-scale metabolic reconstruction for Escherichia coli K-12 MG1655 that accounts for 1260 ORFs and thermodynamic information.” Mol. Syst. Biol. 3 121. 
Huang, J., Q. Zhang and T. Schlick (2003). &quot;Effect of DNA superhelicity and bound proteins on mechanistic aspects of the Hin-mediated and Fis-enhanced inversion.&quot; Biophys J 85(2): 804-817.
Iyarest Tawornsamretkit, Rattana Thanasomboon, Jittrawan Thaiprasit, Dujduan Waraho, Supapon Cheevadhanarak, Asawin Meechai, Analysis of Metabolic Network of Synthetic Escherichia coli Producing Linalool Using Constraint-based Modeling, Procedia Computer Science, Volume 11, 2012, Pages 24-35, ISSN 1877-0509Mahadevan, R. and C. H. Schilling (2003). &quot;The effects of alternate optimal solutions in constraint-based genome-scale metabolic models.&quot; Metab Eng 5(4): 264-276.
Orth, J. D., I. Thiele, and B.O. Palsson (2010). &quot;What is Flux Balance Analysis?&quot; Nature Biotechnology (28): 245-248. 
Papin, J. A., N. D. Price, J. S. Edwards and B. B. Palsson (2002). &quot;The genome-scale metabolic extreme pathway structure in Haemophilus influenzae shows significant network redundancy.&quot; J Theor Biol 215(1): 67-82.
Price, N. D., J. A. Papin, C. H. Schilling and B. O. Palsson (2003). &quot;Genome-scale microbial in silico models: the constraints-based approach.&quot; Trends Biotechnol 21(4): 162-169.
Price, N. D., J. L. Reed and B. O. Palsson (2004). &quot;Genome-scale models of microbial cells: evaluating the consequences of constraints.&quot; Nat Rev Microbiol 2(11): 886-897.
Schilling, C. H., M. W. Covert, I. Famili, G. M. Church, J. S. Edwards and B. O. Palsson (2002). &quot;Genome-scale metabolic model of Helicobacter pylori 26695.&quot; J Bacteriol 184(16): 4582-4593.
Tuite, N. L.,Fraser, K. R.,O'Byrne, C. P.(2005).&quot;Homocysteine toxicity in Escherichia coli is caused by a perturbation of branched-chain amino acid biosynthesis&quot; Journal of bacteriology 187(13):4362-4371.</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:KU_Leuven/Project/Glucosemodel/MeS/Modelling-FBA">http://2013.igem.org/Team:KU_Leuven/Project/Glucosemodel/MeS/Modelling-FBA</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:KU_Leuven/Project/Glucosemodel/MeS/Modelling-FBA" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:KU_Leuven/Project/Glucosemodel/MeS/Modelling-FBA" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:KU_Leuven/Project/Glucosemodel/MeS/Modelling-FBA&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:KU_Leuven/Project/Glucosemodel/MeS/Modelling-FBA&amp;oldid=360444" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>