"
Page
Discussion
View source
History
teams
Log in
 
Team:KU Leuven/Project/Glucosemodel/qPCR
From 2013.igem.org
(Redirected from Team:KU Leuven/Project/qPCR )
iGem
Secret garden
Congratulations! You've found our secret garden! Follow the instructions below and win a great prize at the World jamboree!
A video shows that two of our team members are having great fun at our favourite company. Do you know the name of the second member that appears in the video?
For one of our models we had to do very extensive computations. To prevent our own computers from overheating and to keep the temperature in our iGEM room at a normal level, we used a supercomputer. Which centre maintains this supercomputer? (Dutch abbreviation)
We organised a symposium with a debate, some seminars and 2 iGEM project presentations. An iGEM team came all the way from the Netherlands to present their project. What is the name of their city?
Now put all of these in this URL:https://2013.igem.org/Team:KU_Leuven/(firstname)(abbreviation)(city), (loose the brackets and put everything in lowercase) and follow the very last instruction to get your special jamboree prize!
Menu
Data Page
Project
Aphid Background
Honeydew system
Sticker system
E. coligy
Parts
Modelling
Cellular level
Colony level
Ecosystem level
Team
Students & Advisors
Attributions
Collaborations
Sponsors
Official Team Profile
Human Practices
Top Down
Bottom Up: Framework
Bottom Up: Public Outreach
Notebook
Journal
Protocols
Safety
iGEM
 
The Honeydew System
Designing the Honeydew System
Want to know more about our constructs?
E-β-Farnesene construction
BanAphids produce EBF!
Methyl Salicylate construction
BanAphids produce MeS!
Methyl Salicylate - qPCR
You are here!
qPCR
Why qPCR?
We performed this qPCR for two main reasons:
With a qPCR we can check if our genes of interest are properly transcribed. This is a good characterisation of the methyl salicylate brick (BBa_K1060003) .
We would like to know the amount of transcripts as an input for our methylsalicylate model . Because the mRNA production step is responsible for a large part of uncertainty in our model we would like to bypass it. In order to attain our goal without needing the transcription rates we tried to determine the in vivo mRNA concentrations using qPCR.
The notebook on the qPCR experiments can be found in the journal .
Sample Preparation
We first started with the sample preparation, as described in the qPCR protocol . For this experiment we used our regular E. coli strains (DH5α), harbouring our methyl salicylate brick and used 3 biological repeats.
After preparing three samples we started with the isolation of RNA. In a next step we removed DNA contamination by doing an extra DNase (Turbo DNase, Ambion) step as described in our protocol.
We then used the Nanodrop to get to know the concentration of our RNA and got the following results:
ng/µl
260/280
260/230
Sample A
100,3
2,19
1,42
Sample B
56,3
2,17
0,78
Sample C
92
2,17
1,28
These results are satisfactory. Only the 260/230 values are low because of the high salt level of the DNase buffer. After purification these values will rise.
Genomic DNA contamination
After the DNase step we needed to test whether there still is any DNA contamination in our RNA prep. We definitely have to remove all DNA contamination because the qPCR reaction will not differentiate between the original copy (gene on the plasmid) and the cDNA (Reverse Transcribed from the mRNA). If the original copy is still present in the sample, this will lead to overestimation of the quantity of mRNA molecules, and thus give false results.
To test whether there still is any DNA contamination we used PCR. We first tested if there is still any genomic DNA present in our sample by choosing a primer set on a conserved region. In our case we chose the csrA gene. As a positive control we used genomic DNA from a MG1655 strain. After gelelectrophoresis we got the following results.
We can conclude that we removed the genomic DNA successful since there are no visible bands for A, B and C (except for the primer dimer) (See Figure 1).
Figure 1 | DNA gel electrophoresis after a PCR that amplifies a genomic region (csrA).
Plasmid DNA contamination
Since our genes of interest are not in the genome of our bacteria but on a plasmid, we have to check if all the plasmid DNA is completely removed. So we did the same PCR but with a primer set (iGEM0001 and iGEM0020) that amplifies a region on our plasmid.
We clearly still have plasmid DNA contamination even after a double DNase treatment. Which surprised us, because doing a double DNase treatment should have removed all of the contaminating DNA.
Figure 2 | DNA gel electrophoresis after a PCR that amplifies a region on our plasmid.
Troubleshooting
Thanks to some brainwork we got to a hypothesis that could explain this problem. Because we used a high copy number plasmid (ORI: pMB1), the proportion of plasmid DNA is far higher than the proportion of genomic DNA. The manufacturer advised to use a more rigorous treatment, but could not assure us that all traces of plasmid DNA will be removed because we are using a high copy number plasmid. So we had to find another way around of this problem.
After advice from dr. Basak Ozturk we decided to go for a restriction digest on the RNA samples. We figured that we’d best cut it with an enzyme that cuts the fragments that will be amplified in the qPCR itself. On this way the contaminating DNA will not be amplified while the cDNA will be correctly amplified and we could take unbiased conclusions.So we went looking for a good restriction enzyme that does the job perfectly for our two genes of interest.
New hopes crushed
We came to the restriction enzyme MvaI. This should cut our plasmid on the right places as shown on figure 3.
The MvaI restriction enzyme should cut inside both of the qPCR amplified regions (the regions between the “blue boxes”), so that only linear amplification is possible there and that interference should be eliminated.
Unfortunately, after running some PCR’s and DNA electrophoresis gels to check whether we finally got rid of our contaminating DNA, we still had a disappointing result (see figure 4).
It might not be clear on the images but all our samples still show bands which means that our digestion-trick did not work.
This means that we are unable to remove the contaminating plasmid DNA. As previously mentioned we think that is due to the fact that the proportion of plasmid DNA is far higher than the proportion of genomic DNA, as we used a high copy number plasmid. This higher proportion of plasmid DNA is very hard to remove using DNase treatment. If we really want to go on with the qPCR, the only option is to clone our genes of interest in a low copy number vector or into the genome of our host strain. Since we were already quite busy with cloning, we decided to end the qPCR adventure here.
Figure 4 | Drawing that shows the MvaI restriction sites. The blue boxes show the primers we designed for the qPCR.
Figure 5 | DNA gel electrophoresis after two different PCR reactions where each amplifies a different region on our plasmid.
Conclusion
We decided to publish this story (even though it was unsuccesful) to warn the other teams for the difficulties that come with the removal of contaminating plasmid DNA in a RNA sample preparation.
Retrieved from " http://2013.igem.org/Team:KU_Leuven/Project/Glucosemodel/qPCR "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
