<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_KU_Leuven_Project_Oscillator_Design"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:KU_Leuven/Project/Oscillator/Design">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:KU_Leuven/Project/Oscillator/Design&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:KU_Leuven/Project/Oscillator/Design&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:KU_Leuven/Project/Oscillator/Design&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:KU_Leuven/Project/Oscillator/Design" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:KU Leuven/Project/Oscillator/Design</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>iGem</TITLE></P><DIV class="container"><DIV id="topbar"><DIV id="secret"><DIV class="row-fluid"><DIV class="span12"><DIV class="row-fluid"><DIV class="span9"><H3>Secret garden</H3><P style="color:white">Congratulations! You've found our secret garden! Follow the instructions below and win a great prize at the World jamboree!</P><UL style="color:white"><LI>A video shows that two of our team members are having great fun at our favourite company. Do you know the name of the <B>second member</B> that appears in the video?</LI><LI>For one of our models we had to do very extensive computations. To prevent our own computers from overheating and to keep the temperature in our iGEM room at a normal level, we used a supercomputer. Which <B>centre</B> maintains this supercomputer? (Dutch abbreviation)</LI><LI>We organised a symposium with a debate, some seminars and 2 iGEM project presentations. An iGEM team came all the way from the Netherlands to present their project. What is the name of their <B>city</B>? </LI></UL><P style="color:white">Now put all of these in this URL:https://2013.igem.org/Team:KU_Leuven/(firstname)(abbreviation)(city), (loose the brackets and put everything in lowercase) and follow the very last instruction to get your special jamboree prize!</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="container"><DIV class="container"><DIV class="row-fluid"><DIV id="menu"><A class="toggleMenu" href="#">Menu</A><UL class="nav"><LI class="test"><A href="https://2013.igem.org/Team:KU_Leuven/Project/DataPage">Data Page</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Project">Project</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Project/Aphid_Background">Aphid Background</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Project/HoneydewSystem">Honeydew system</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Project/StickerSystem">Sticker system</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Project/E.coligy"><I>E. coligy</I></A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Parts">Parts</A></LI></UL><LI><A href="https://2013.igem.org/Team:KU_Leuven/Project/modelling">Modelling</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Project/Modelling/Cellular_Level">Cellular level</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Project/Modelling/Colony_Level">Colony level</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Project/Modelling/Ecosystem_Level">Ecosystem level</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Team">Team</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Team/Students">Students &amp; Advisors</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Team/Attributions">Attributions</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Team/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Sponsors">Sponsors</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013">Official Team Profile</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Human_Practices">Human Practices</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Human_Practices/Top_Down">Top Down</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Human_Practices/Bottom_Up_Philosopher">Bottom Up: Framework</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Human_Practices/Bottom_Up_Public">Bottom Up: Public Outreach</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Journal">Journal</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Protocols">Protocols</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Safety">Safety</A></LI><LI><A href="https://igem.org/Main_Page">iGEM</A></LI><DIV class="clear-both"> </DIV></DIV></DIV></DIV></DIV></DIV><DIV id="container"><DIV class="container"><DIV id="header" class="row-fluid"><DIV class="span12"><H3 class="bg-green">Modelling on Colony Level</H3></DIV></DIV><DIV class="row-fluid"><DIV class="span6 icon white"><DIV class="row-fluid"><DIV class="span9 icon-text"><H3>Design</H3><P>You are here!</P></DIV></DIV></DIV><DIV class="span6 icon white"><DIV class="row-fluid"><DIV class="span9 icon-text"><H3>Modelling</H3><P>Mathematical view on our oscillator</P></DIV></DIV></DIV></DIV><DIV id="header" class="row-fluid"><DIV class="span12"><H3 class="bg-green">Oscillator Design</H3></DIV></DIV><DIV class="row-fluid"><DIV class="span12 white"><P align="justify">
As an alternative to the honeydew system which requires our BanAphids to be released in the environment, we have been working on an oscillator model. This way, <B>we can keep the BanAphids contained, and still prevent habituation of the aphids against E-β-farnesene.</B> On this page, we talk about <A href="#benefits">the benefits</A> of a synchronised oscillation, and our own <A href="#Design">design</A>. Finally, we have thought of a way to cope with <A href="#noise">noise</A> in the oscillator.</P></DIV></DIV><DIV id="header" class="row-fluid"><DIV class="span12"><H3 class="bg-green">An Oscillator as an Alternative Solution</H3></DIV></DIV><DIV class="row-fluid"><DIV class="span12 white"><H3>Colony-wide oscillations</H3><DIV class="row-fluid"><DIV class="span12"><DIV class="row-fluid"><DIV class="span8"><P align="justify">
Before we delve into the details of our oscillator we will first discuss the intricacies of oscillating one cell versus oscillating a colony, which consists of millions of cells. <B>The repressilator of Elowitz and Leibler</B> (2000) is a nice example of how to create oscillations in one cell. Their work has been a cornerstone for synthetic biology since the repressilator was among the first synthetic networks successfully introduced into bacteria. However, they did not succeed in a <B>colony-wide synchronization</B>. This means that even though each bacterium oscillates on its own, they do not oscillate together. For every bacterium in the colony, there will be one oscillating in opposite phase. The average of their signals would be a flat line in the case of perfect sinusoidal oscillations (Figure 1a). Clearly this underlines the <B>need for synchronization on the colony-level.</B> Providing this happens, the oscillations of the different cells would amplify each other (Figure 1b), resulting in an oscillating colony.</P><P align="justify">The importance of synchronization can also be seen in nature, where single organisms organize themselves into larger groups (Ioannou et al., 2007). A well-known example is the behaviour of relatively small fish that group into schools. This brings them potential benefits, one of which is exemplified in Figure 2.</P></DIV><DIV class="span4 greytext"><P align="justify">Figure 1a| The summed input of two sinusoidal functions in anti-phase result in a flat line. </P><P align="justify">Figure 1b| The summed input of two sinusoidal functions in phase result in an overall sinusoidal function, with a higher amplitude.</P></DIV></DIV></DIV><DIV class="span12"><DIV class="row-fluid"><DIV class="span3 greytext"><P align="justify">Figure 2| Figure : This shows how small fish can confuse their predators when they work together in a synchronized fashion, something they cannot achieve on their own.</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row-fluid"><DIV class="span12 white"><H3>Transcription factor network</H3><P align="justify">
Obviously, the first and foremost requirement for our oscillator is <B>synchronization on the colony level</B>. The iGEM team from Wageningen tried to attain exactly this in 2011, based on the model proposed by Danino et al. (2010). This model is thoroughly described by the <B><A href="https://2011.igem.org/Team:Wageningen_UR" target="_blank">Wageningen 2011 iGEM team</A></B>. As they mention on their wiki, it heavily depends on parameter values and conditions. We refer to Box 1 for an explanation of parameters and conditions.
Prior to determining the parameters that will define our oscillating network, we have to establish the nature of the network. Following the repressilator of Elowitz and Leibler (2000) and the synchronized oscillator of Danino et al. (2010) we also choose <B>a transcription factor (TF) network</B>. Our TF network will consist of two modules. The first controls the production of E-β-farnesene, the second consists of oscillating transcription factors. We will design our TF network such that the oscillating module controls the crucial step in the E-β-farnesene production module, resulting in the oscillating production of E-β-farnesene.</P></DIV></DIV><DIV class="row-fluid"><DIV class="span12 bg-grey1"><H3>Box 1: Parameters and conditions</H3><P align="justify">
Within a mathematical equation a parameter has a constant value (e.g. in<I> y = a*x + b</I> , <I>a</I> and <I>b</I>  are parameters). In a biological network parameters could be maximal protein production rates, degradation rates or promoter transcription factor binding energies. This implies that these remain constant within unchanging conditions. If conditions change (rising temperatures, changing pH), these parameter values will change towards new, constant values.
Note: The conditions fully define the parameters. In turn, the parameters are the sole elements within the network to reflect the conditions. Consequently, changes in conditions can always be interpreted in terms of changes in parameters.</P></DIV></DIV><DIV id="header" class="row-fluid"><DIV class="span12"><H3 class="bg-green">Our design</H3></DIV></DIV><DIV class="row-fluid"><DIV class="span12 white"><DIV class="row-fluid"><DIV class="span4 greytext"><P align="justify">Figure 3 | Logical circuit displaying our oscillating model, combined with its link to the production system</P></DIV><DIV class="span8"><P align="justify">
The most important feature of our model remains <B>the sustained colony-wide oscillations</B> under a broad range of parameters. The production of E-β-farnesene should not be constitutive, but the height of the production peaks and the time in between different peaks can vary as long as they remain within reasonable ranges. Therefore we do not pose stringent requirements for the amplitude and frequency of the E-β-farnesene production peaks. To build up the actual model, we started from the seminal work of Uri Alon (2006), a lot of coffee and nightly endeavours. The result of this slog is shown in Figure 3. <B>Our model consists of six transcription factors (A, B, C, D, X and Y)</B>. The arrows represent the transcriptional activation of the target (e.g. an arrow from A to X means that transcription factor A induces the production of transcription factor X). Similarly, the inhibitory sign represents the transcriptional repression of the target. The AND and OR gates are logical gates. For an AND gate to function both of the inputs must be ‘positive’ (e.g. for TF A to be transcribed, TF D must be present <U>and</U> TF C must be absent). For an OR gate to function one positive input suffices (e.g. TF C is transcribed when TF A <U>or</U> TF X is present <U>or</U> when both are present). One of those transcription factors must eventually be linked to the production of E-β-farnesene. Based on some of our modelling, we selected TF A to be linked to the production of E-β-farnesene. The main reason being that TF A and TF B show a <B>sharp production peak</B>, the goal of an oscillating production system. This sharp production peak of TF A can also be seen in Figure 5c and 5d.</P></DIV></DIV></DIV></DIV><DIV class="row-fluid"><DIV class="span12 white"><H3>Oscillations of our system</H3><P align="justify">We will now walk you through the network and how it reaches oscillations. A high level of TF A will induce the production of both TF C and TF X. After a little while, the <B>level of TF C will be sufficient to initiate the repression of the top AND gate</B> (and the stimulation of the bottom AND gate). Once the top AND gate is significantly repressed, the <B>level of TF A starts to decline</B>. When TF A drops underneath its threshold, the level of TF X starts to decline too. Be aware that at this point the level of TF C does not decline yet. Only when the level of TF X also dropped underneath its own threshold will the level of TF C start to wane. This prolonged expression of TF C <B>results in an extended repression of the production of TF A</B>. This ascertains the level of A drops sufficiently. On the other hand, an extended expression of TF C also results in an extended activation of the production of TF B, which reaches a high level. The same explanation as before can now be applied to TF B, TF D and TF Y. This eventually results in a decrease in the level of TF B and a high level of TF A, bringing the system <B>back to its starting point</B>. Every time this cycle is completed, the levels of each of the transcription factors dramatically change and eventually return to their original level, or in other words, oscillate. This network design inherently results in such a behaviour, meaning that the dependence on the parameter values is very low. This aspect of our model has been <B>supported with in silico results</B>, which can be found on the <A href="https://2013.igem.org/Team:KU_Leuven/Project/Oscillator/Modelling">modelling page</A> regarding the subject.</P></DIV></DIV><DIV class="row-fluid"><DIV class="span12 white"><H3>Synchronisation</H3><P align="justify">We achieved <B>low parameter dependence</B> through an intelligent network design. The other requirement is that the <B>oscillation of the colony is synchronised</B>. The oscillations of different cells are linked because two of the components of our oscillatory network can freely diffuse out of the cells. Diffusion is something that occurs very rapidly at the scale relevant for bacteria. It distributes the molecules almost evenly throughout the colony, at least as long as most partake in the production of these colony-wide molecules. This results in colony-wide oscillations of those molecules. Consequently, this is a very effective means of synchronising a population. Using two colony-wide molecules that (indirectly) repress each other’s production allows the oscillation of a broader spectrum of colony densities. When there are more cells per volume the perceived production rate of the colony-wide molecules also increases. The reciprocal (indirect) repression of the two colony-wide molecules helps to cancel this effect and equalises the oscillations for a larger amount of cell densities.
By studying the possibilities for colony-wide transcription factors, we found <B>quorum-sensing molecules</B> as good candidates, since the ones we encountered serve as transcription factors (combined with intracellular receptors) and can passively diffuse out of the cells (Fugua, Winans and Greenberg, 1994). Quorum-sensing molecules are produced by enzymes, which are thoroughly discussed on the <A href="https://2013.igem.org/Team:KU_Leuven/Project/Oscillator/wetlab">oscillator wetlab page</A> and the extra enzyme step alters the equation as is discussed on the <A href="https://2013.igem.org/Team:KU_Leuven/Project/Oscillator/Modelling">modelling page</A>. Danino et al. (2010) also opted for a quorum-sensing molecule for their synchronised oscillator, but they used only one. Our model would use 2 (displayed as A and B), to allow a broader range of colony densities.</P></DIV></DIV><DIV class="row-fluid"><DIV class="span12 white"><H3>Filling in the dots</H3><P align="justify">For biologists the most interesting part is most likely <B>giving a name to the transcription factors</B>. The primary requirement would be that their biological interactions are according to the network displayed in Figure 3. For example in our wetlab attempts we used the transcription factor araC (biobrick C0080) to be TF X, which had a promoter (R0078) inducible by a quorum-sensing molecule. For more details and practical implications we refer you to the <A href="https://2013.igem.org/Team:KU_Leuven/Project/Oscillator/wetlab">oscillator wetlab page</A>.</P></DIV></DIV><DIV id="header" class="row-fluid"><DIV class="span12"><H3 class="bg-green">Coping with noise</H3></DIV></DIV><DIV class="row-fluid"><DIV class="span12 white"><P align="justify">A principal problem that goes hand in hand with colony synchronisation, is the <B>constant presence of noise</B>. This noise is partitioned into two sources: <B>intrinsic noise</B>, which originates within the regarded subsystem and <B>extrinsic noise</B>, which originates outside of it. We will describe this specifically for gene expression since we built a transcription factor circuit with oscillating behaviour. The <B>intrinsic noise</B> is due to the stochastic effects involved in the reaction events leading to transcription and translation. This leads to fluctuations in timing and order of those events, even in cells that are in an equal state (equal nutritional status, place in the cell cycle, …). The <B>extrinsic noise</B> is for example due to differences in the number of RNA polymerase and ribosome molecules. Because they are gene products themselves, they are subject to their own intrinsic noise. Other factors that lead to extrinsic noise include the stage in the cell cycle, the quantity of the protein, the mRNA degradation machinery and the cell environment (Swain, Elowitz and Siggia, 2002). This substantial presence of noise has to be taken into account when developing a genetic circuit, either by suppressing it, or by building a network that is robust to it (Elowitz et al., 2002).
The presence of noise <B>affects our system in two ways</B>. First it means the different cells in a colony are in a different state and exhibit a variance in their parameters.  Second it causes fractions of the colony to become unsynchronised. Both of these effects have been tested on the <A href="https://2013.igem.org/Team:KU_Leuven/Project/Oscillator/Modelling">modelling page</A> and the results show our designed network resists this very well. In the rest of this text we will discuss how the use of diffusible molecules and the shape of our network achieve the different requirements posed to an oscillator.</P></DIV></DIV><DIV class="row-fluid"><DIV class="span12 white"><H3>Coherent type 1 feed forward loop</H3><DIV class="row-fluid"><DIV class="span8"><P align="justify">As mentioned above, there is a <B>prolonged production</B> of TF C (D) after the disappearance of TF A (B). However, when there is TF A (B) present, the production of repressor TF C (D) starts immediately. This means there’s an asymmetric time delay for the production of the repressing molecules. The network motif that exhibits this behaviour is called a <B>‘coherent type 1 feed forward loop’</B> (C1-FFL) with an OR logical gate (Mangan and Alon, 2003), one of which is highlighted in Figure 4. This extended period in which one of the colony-wide molecules is repressed and the other produced, helps to reduce the effect of cell-to-cell variability in the different parameters, which is a necessity in order to have a synchronized oscillation. Even if not all cells produce a sufficient amount of repressing molecules, the prolonged repression by the others still results in an effective reduction in the amount of that colony-wide molecule, and analogously also a rise in the concentration of the other. The prolonged production of TF C, and similarly TF D, ascertains that <B>the switching event they are responsible for is fully executed</B>.</P></DIV><DIV class="span4 greytext"><P align="justify">Figure 4 | Logical circuit displaying our oscillator, with one C1 FFL highlighted in green</P></DIV></DIV></DIV></DIV><DIV class="row-fluid"><DIV class="span12 white"><H3>A small desynchronized fraction of the cells cannot distort the oscillations</H3><P align="justify">At the small space-scale that is relevant to bacteria, <B>diffusion rapidly spreads out molecules throughout the colony</B>. If for instance, the unsynchronized subpopulation produces the other colony-wide molecule, that concentration does not reach any significant level because of the rapid diffusion at that scale. This can be explained by using an approximation of the diffusion coefficient of a typical quorum sensing molecule; D(N-(3-Oxododecanoyl)-L-homoserine lactone) equals 4.9 * 10<SUP>2</SUP> µm<SUP>2</SUP>s<SUP>-1</SUP> (Stewart, 2003). The equation  (Einstein, 1905) gives the average displacement of molecules because of diffusion in 3 dimensions after time t. In this example the average displacement after 1 second would be 54 µm, which is about equal to 50 times the size of one cell. This supports our claim that <B>the pheromone production of one cell is rapidly dispersed throughout its environment</B> and that only larger fractions of the colony can control the level of colony-wide molecules. </P></DIV></DIV><DIV class="row-fluid"><DIV class="span12 white"><H3>The colony-wide concentration of some transcription factors benefits rapid resynchronization</H3><P align="justify">The distance between cells in moderate density cultures is approximately 16 µm (= 2*10<SUP>8</SUP> cells/ml, Park et al., 2003) or one third of the diffusion length after one second. This implies that <B>all cells experience an approximately equal concentration of diffusible molecules</B>.  If, by random occurrence, a certain amount of cells have an excess amount of non-diffusible molecules, as for instance one of the repressors, they will produce the colony-wide molecules at a different rate. The next switch in production, however, occurs for all cells approximately at the same time and serves as a <B>resynchronization event</B>. This can be explained because it is the colony-wide molecule that is being produced by the majority that first reaches the threshold needed to switch to the production of the other colony-wide molecule in all of the cells. Since the cells that were out of sync had not yet stopped producing that other molecule, they are resynchronized with the majority. This has to be seen in relation with the extended production of TF C, and TF D, responsible for the switching events.</P></DIV></DIV><DIV class="row-fluid"><DIV class="span12 white"><DIV class="row-fluid"><DIV class="span6 greytext"><P align="justify">Figure 5 | Graphical reproduction of the rapid resynchronization | a, The spatial-temporal fluctuations of a colony-wide molecule in a population in which 40% initially behaves differently. b, The concentration of one of the non-diffusible molecules in a population in which 40% initially behaves differently. c, The evolution over time of the production rate of several molecules in a cell belonging to the majority of the population. d, The evolution over time of the production rate of several molecules in a cell belonging to the minority of the population.</P></DIV><DIV class="span6"><H3>Example of rapid resynchronization</H3><P align="justify">Figure 5 shows an extreme example of this principle. These images have been created by solving a partial differential equation (PDE) with matlab (provided by MathWorks). The initial conditions in this PDE are that <B>40% of the cells start with a 4-fold excess of TF C and a quarter of TF D</B>, with respect to the other 60% of the cells. Figures 5a and 5b show the spatial-temporal oscillations of a colony-wide molecule, here TF A, (Figure 5a) and a non-permeable molecule, here TF C (Figure 5b), respectively. Even though the population behaves differently in the beginning, <B>they show a rapid resynchronization</B>. Figure 5 c and d display the production rates of the colony-wide molecules and of the molecules that each repress one and induce the other colony-wide molecule in the two different cell subtypes. Figure 5 c displays this for a cell belonging to the majority (60%) and Figure 5 d for a minority (40%) cell.
The model shows only <B>small differences between the production rates of the non-diffusible molecules</B> (red and yellow) while protein production switches occur simultaneously. This is because those are controlled by the concentration of colony-wide molecules, which are approximately uniformly distributed. In reality this will be a stronger effect, since we chose a much smaller diffusion coefficient (2 * 10<SUP>1</SUP> µm<SUP>2</SUP>s<SUP>-1</SUP> instead of 4.9 * 10<SUP>2</SUP> µm<SUP>2</SUP>s<SUP>-1</SUP>) in order to display the principles. The production of the colony-wide molecules (blue and cyan) show a big difference during the first oscillations, but those differences rapidly disappear; there is no significant difference anymore after the second oscillation and resynchronization is a fact. We also measured how the colony would react if cells would go “out of sync”: how quickly would the model reinstate colony-wide synchronization? We refer to the <A href="https://2013.igem.org/Team:KU_Leuven/Project/Oscillator/Modelling"> modelling page</A> for details on these quantitative results and the algorithms used. </P></DIV></DIV></DIV></DIV><DIV id="header" class="row-fluid"><DIV class="span12"><H3 class="bg-green">Further elaboration</H3></DIV></DIV><DIV class="row-fluid"><DIV class="span12 white"><P align="justify">A very short and preliminary study of the characteristics of our oscillator design can be found on the <A href="https://2013.igem.org/Team:KU_Leuven/Project/Oscillator/Modelling">oscillator modelling page</A>. We also started <B>the <I>in vivo</I> counterpart of our model</B>. Bringing the entire model into a living cell poses some problems, the most important one being the time required for the many cloning steps. We do foresee that in the future this will become easier, when the field of synthetic biology gains importance and the possibility of synthetizing large gene constructs becomes cheaper. We chose to focus on a crucial part of the oscillator in the wetlab, since it serves as a useful network motif by itself and it can help other teams in further developing this oscillator. The transcriptional network motif we are making into a biobrick is a <B>coherent type 1 feed forward loop</B> which uses a quorum sensing molecule as the first transcription factor. The oscillator envelops this network motif twice, one of which is highlighted in green in Figure 4. The reason we have chosen this natural occurring network motif is because of its central role in our proposed oscillator model, as well as in many other natural systems (Mangan and Alon, 2003). For the implementation, we refer you to the <A href="https://2013.igem.org/Team:KU_Leuven/Project/Oscillator/wetlab">oscillator wetlab page</A>.</P></DIV></DIV><DIV id="header" class="row-fluid"><DIV class="span12"><H3 class="bg-green">References</H3></DIV></DIV><DIV class="row-fluid"><DIV class="span12 white"><P align="justify">Alon, U. (2006). An introduction to systems biology: design principles of biological circuits. CRC Press, 301.
Danino, T. et al. (2010). A synchronized quorum of genetic clocks. Nature, 463:326-330.
Einstein, A. (1905). On the movement of small particles suspended in stationary liquids required by the molecular-kinetic theory of heat. Annalen der Physik 17:549-560.
Elowitz, M. B., and Leibler, S. (2000). A synthetic oscillatory network of transcriptional regulators. Nature, 403(6767):335-338.
Elowitz, M. B. et al. (2002). Stochastic gene expression in a single cell. Science, 297(5584):1183-1186.
Fuqua, W. C., Winans, S. C., and Greenberg, E. P. (1994). Quorum sensing in Bacteria: the luxR-luxI family of cell density-responsive transcriptional regulators. J. Bacteriol, 176:269-275.
Ioannou, C. C., Tosh, C. R., Neville, L., and Krause, J. (2007). The confusion effect-from networks to reduced predation risk. Behavioral Ecology, 19(1):126-130.
Kunert, G., Reinhold, C. and Gershenzon, J. (2010). Constitutive emission of the aphid alarm pheromone, (E)-β-farnesene, from plants does not serve as a direct defense against aphids. BMC Ecology, 10:23.
Mangan, S., and Alon, U. (2003). Structure and function of the feed-forward loop network motif. PNAS, 100:11980-11985.
Park, S. et al. (2003). Motion to form a quorum. Science, 301:188.
Stewart, P. S. (2003). Diffusion in biofilms. Journal of Bacteriology, 185:1485-1491.
Swain, P. S., Elowitz, M. B., and Siggia, E. D. (2002). Intrinsic and extrinsic contributions to stochasticity in gene expression. PNAS, 99:12795-12800.</P></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:KU_Leuven/Project/Oscillator/Design">http://2013.igem.org/Team:KU_Leuven/Project/Oscillator/Design</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:KU_Leuven/Project/Oscillator/Design" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:KU_Leuven/Project/Oscillator/Design" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:KU_Leuven/Project/Oscillator/Design&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:KU_Leuven/Project/Oscillator/Design&amp;oldid=360473" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>