<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_KU_Leuven_Project_Glucosemodel_MeS"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:KU_Leuven/Project/Glucosemodel/MeS">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:KU_Leuven/Project/Glucosemodel/MeS&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:KU_Leuven/Project/Glucosemodel/MeS&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:KU_Leuven/Project/Glucosemodel/MeS&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:KU_Leuven/Project/Glucosemodel/MeS" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:KU Leuven/Project/Glucosemodel/MeS</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>iGem</TITLE></P><DIV class="container"><DIV id="topbar"><DIV id="secret"><DIV class="row-fluid"><DIV class="span12"><DIV class="row-fluid"><DIV class="span9"><H3>Secret garden</H3><P style="color:white">Congratulations! You've found our secret garden! Follow the instructions below and win a great prize at the World jamboree!</P><UL style="color:white"><LI>A video shows that two of our team members are having great fun at our favourite company. Do you know the name of the <B>second member</B> that appears in the video?</LI><LI>For one of our models we had to do very extensive computations. To prevent our own computers from overheating and to keep the temperature in our iGEM room at a normal level, we used a supercomputer. Which <B>centre</B> maintains this supercomputer? (Dutch abbreviation)</LI><LI>We organised a symposium with a debate, some seminars and 2 iGEM project presentations. An iGEM team came all the way from the Netherlands to present their project. What is the name of their <B>city</B>? </LI></UL><P style="color:white">Now put all of these in this URL:https://2013.igem.org/Team:KU_Leuven/(firstname)(abbreviation)(city), (loose the brackets and put everything in lowercase) and follow the very last instruction to get your special jamboree prize!</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="container"><DIV class="container"><DIV class="row-fluid"><DIV id="menu"><A class="toggleMenu" href="#">Menu</A><UL class="nav"><LI class="test"><A href="https://2013.igem.org/Team:KU_Leuven/Project/DataPage">Data Page</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Project">Project</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Project/Aphid_Background">Aphid Background</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Project/HoneydewSystem">Honeydew system</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Project/StickerSystem">Sticker system</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Project/E.coligy"><I>E. coligy</I></A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Parts">Parts</A></LI></UL><LI><A href="https://2013.igem.org/Team:KU_Leuven/Project/modelling">Modelling</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Project/Modelling/Cellular_Level">Cellular level</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Project/Modelling/Colony_Level">Colony level</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Project/Modelling/Ecosystem_Level">Ecosystem level</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Team">Team</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Team/Students">Students &amp; Advisors</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Team/Attributions">Attributions</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Team/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Sponsors">Sponsors</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013">Official Team Profile</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Human_Practices">Human Practices</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Human_Practices/Top_Down">Top Down</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Human_Practices/Bottom_Up_Philosopher">Bottom Up: Framework</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Human_Practices/Bottom_Up_Public">Bottom Up: Public Outreach</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Journal">Journal</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Protocols">Protocols</A></LI><LI><A href="https://2013.igem.org/Team:KU_Leuven/Safety">Safety</A></LI><LI><A href="https://igem.org/Main_Page">iGEM</A></LI><DIV class="clear-both"> </DIV></DIV></DIV></DIV></DIV></DIV><DIV id="container"><DIV class="container"><DIV id="header" class="row-fluid"><DIV class="span12"><H3 class="bg-green">The Honeydew System</H3></DIV></DIV><DIV class="row-fluid"><DIV class="span6 icon white"><DIV class="row-fluid"><DIV class="span9 icon-text"><H3>Designing the Honeydew System</H3><P>Want to know more about our constructs?</P></DIV></DIV></DIV><DIV class="span6 icon white"><DIV class="row-fluid"><DIV class="span9 icon-text"><H3>E-β-Farnesene construction</H3><P>BanAphids produce EBF!</P></DIV></DIV></DIV></DIV><DIV class="row-fluid"><DIV class="span6 icon white"><DIV class="row-fluid"><DIV class="span9 icon-text"><H3>Methyl Salicylate construction</H3><P>You are here!</P></DIV></DIV></DIV><DIV class="span6 icon white"><DIV class="row-fluid"><DIV class="span9 icon-text"><H3>Methyl Salicylate - qPCR</H3><P>Wetlab data for the MeS model</P></DIV></DIV></DIV></DIV><DIV id="header" class="row-fluid"><DIV class="span12"><H3 class="bg-green">Methyl Salicylate</H3></DIV></DIV><DIV class="row-fluid"><DIV class="span12 white"><DIV class="row-fluid"><DIV class="span9"><P align="justify">Methyl salicylate (MeS) or wintergreen oil is a plant hormone. It is converted from <B>chorismate</B> with <B>salicylate</B> as an intermediate. <B>In this page we will discuss the <A href="#pathway">pathway</A> that leads up to methyl salicylate and the <A href="#Our bricks">biobricks</A> that we made.</B> We will end with characterisation of these biobricks using a <A href="#Smell Test">smell</A> test, an <A href="#SDS PAGE">SDS-PAGE</A> and <A href="#Headspace GC"> GC analyses</A>.</P></DIV><DIV class="span3 greytext"><P align="justify">Figure 1 | Methyl Salicylate</P></DIV></DIV></DIV></DIV><DIV class="row-fluid"><DIV class="span4 icon white"><DIV class="row-fluid"><DIV class="span7 icon-text"><H3>MeS brick background</H3><P>Genes, pathway and improvements</P></DIV></DIV></DIV><DIV class="span4 icon white"><DIV class="row-fluid"><DIV class="span7 icon-text"><H3>Brick characterisation</H3><P>Aphid experiments, GC-MS and more</P></DIV></DIV></DIV><DIV class="span4 icon white"><DIV class="row-fluid"><DIV class="span7 icon-text"><H3>Our bricks</H3><P>Bricks and wetlab overview</P></DIV></DIV></DIV></DIV><DIV id="header" class="row-fluid"><DIV class="span12"><H3 class="bg-green">The Pathway To Methyl Salicylate</H3></DIV></DIV><DIV class="row-fluid"><DIV class="span12 white"><DIV class="row-fluid"><DIV class="span3 greytext"><P align="justify">Figure 2 | The Shikimate Pathway</P></DIV><DIV class="span9"><P align="justify">The conversion of Ery4P and PEP to DAHP (3-deoxy-D-arabinoheptulosonate 7-phosphate) is catalysed by DAHP synthase, which exists in three isoforms, coded by the genes <I>aroF, aroG</I> and <I>aroH</I>. These isoforms are selectively inhibited by Tyr, Phe and Trp, respectively. In <I>E. coli</I>, about 80% of the enzyme activity by contributed by the DAHPS of <I>aroG</I>(Ikeda <I>et al.</I>, 2006).
When there is <B>enough phenylalanine</B>, the <B>production of DAHP, and hence chorismate, will stop by inhibition of the <I>aro</I> genes</B>. 
The conversion from chorismate to MeS starts with a conversion to isochorismate, catalysed by isochorismate synthase, encoded by <B><I>pchA</I></B>. Isochorismate is directly converted to salicylate by isochorismate pyruvate/lyase, encoded by <B><I>pchB</I></B>. This conversion happens immediately, since <I>pchA</I> and <I>pchB</I> form the <B><I>pchBA</I></B> operon which is always transcribed to one mRNA. Conversion of salicylate to MeS is catalysed by S-adenosylmethionine-dependent methyltransferase, encoded by <B><I>bsmt1</I></B> (Gaille <I>et al.</I>, 2003). 
</P></DIV></DIV></DIV></DIV><DIV class="row-fluid"><DIV class="span12 white"><H3>A Biobrick to start from</H3><DIV class="row-fluid"><DIV class="span6"><P align="justify">The <B>iGEM team of MIT 2006 already constructed a Biobrick </B>(<A href="http://parts.igem.org/Part:BBa_J45700" target="_blank">BBa_J45700</A>) encoding <I>pchBA</I> and <I>bsmt1</I>, for the production of MeS. However, the MIT team only observed MeS production when salicylic acid was added to the medium. <B>Our experiments with this Biobrick confirmed this lack of MeS production </B>(see the smell test below). There had to be something wrong with the conversion of chorismate to salicylate. One possibility is that there is something wrong with the enzymes produced by the <I>pchBA</I> genes. Another one is that there is a lack of chorismate present in the cell. We have attempted to increase the productivity of the MIT 2006 brick by using stronger ribosome binding sites. Also, <B>we replaced the <I>lac</I> promoter by a TetR-repressible promoter, so the lacI produced in our system doesn't interfere with the transcription of <I>pchBA</I>.</B>.
</P></DIV><DIV class="span6 greytext"><P align="justify">Figure 3 | The original methyl salicylate biobrick <A href="http://parts.igem.org/Part:BBa_J45700" target="_blank">BBa_J45700</A></P></DIV></DIV></DIV></DIV><DIV class="row-fluid"><DIV class="span12 white"><H3>The Chorismate Problem</H3><DIV class="row-fluid"><DIV class="span12"><P align="justify">Since chorismate is the precursor for three amino acids in <I>E. coli</I>, we believe that there is not much of the chorismate left to be converted to MeS.
To overcome this problem, we looked at a study by Sun <I>et al.</I> (2011), in which a synthetic pathway was introduced for the production of mandelic acid from chorismate. They achieved this by deleting different genes encoding enzymes that catalyse competing pathways, as well as by <B>introducing a feedback-insensitive DAHP synthase mutant to increase the carbon flow down the shikimate pathway</B>. This last method gave us inspiration to overcome our own problem.
If Tyr, Phe or Trp are present in the medium, the conversion of chorismate to these amino acids will be inhibited allosterically, as wel as the production of DAHP (Figure 2). <B>Our plan is to mutate the <I>aroG</I> gene in a manner that the enzymatic function still remains, but that it is insensitive to allosteric inhibition by phenylalanine. We only mutate the <I>aroG</I> gene since this isoform is responsible for 80% of DAHP synthase activity </B> (Hu <I>et al.</I>, 2003).</P></DIV><DIV class="span12 greytext"><P align="center">Figure 4 | The different pointmutations of <I>aroG</I> and their effect</P></DIV></DIV></DIV></DIV><DIV class="row-fluid"><DIV class="span12 white"><DIV class="row-fluid"><DIV class="span12"><H3>Necessary Mutations</H3><P align="justify">It is proven that a Pro150Leu point mutation is Phe-insensitive. This mutation is used as a positive control in a study by Hu <I>et al.</I> (2003) in which they compare the effects of different mutations in the <I>aroG</I> gene on the specific enzymatic activity. The results showed that a Leu175Asp mutation also lead to a Phe-insensitive enzyme. Leu175 is located at the bottom of the possible inhibitor binding pocket, and it is believed to be a critical residue.
For reasons still unknown, L175D mutation showed an increased specific enzymatic activity compared to the wild type. In more recent studies, L175D is mostly used to obtain a Phe-insensitive DAHPS (Lin <I>et al.</I>, 2012). That is why we will try to <B>introduce a plasmid containing a L175D mutated <I>aroG</I> gene</B>.
It is also proven that transcription of the normal DAHP synthase gene will be inhibited when phenylalanine is present in the medium, so only the mutated form will be produced (Adhja &amp; Gottesman, 1984). The unmutated version (P150 and L175) of DAHP synthase is shown in the upper two figures, with the inhibitor phenylalanine in the binding pocket on the right. Below it is shown that the <B>two mutations, P150L and L175D, clearly occupy the binding pocket and thereby sterically hinder the binding of phenylalanine</B>.
</P></DIV><DIV class="span12" greytext=""><DIV class="span6"><P align="center">The unmutated DAHP synthase shows a clear Phe binding pocket between residues 150 and 175.</P></DIV><DIV class="span6"><P align="center">Phe binds in the binding pocket between proline and leucine.</P></DIV></DIV><DIV class="span12"><DIV class="span6"><P align="center">Leucine and asparagine sidechains fill up the Phe binding pocket.</P></DIV><DIV class="span6"><P align="center">The binding of Phe is sterically hindered by Leu150 and Asn175.</P></DIV></DIV></DIV></DIV></DIV><DIV id="header" class="row-fluid"><DIV class="span12"><H3 class="bg-green">Characterisation</H3></DIV></DIV><DIV class="row-fluid"><DIV class="span12 white"><DIV class="row-fluid"><DIV class="span12"><P align="justify">We characterised the MeS BioBrick by means of <A href="#Smell Test">a smell test</A> as well as characterising this brick with <A href="#Headspace GC">Headspace GC</A>,<A href="#GC-MS analysis of MeS in ethyl acetate extracts"> GC-MS analysis of MeS in ethyl acetate extracts</A>  and <A href="#SDS PAGE">SDS PAGE analysis</A>. </P></DIV></DIV></DIV></DIV><DIV id="header" class="row-fluid"><DIV class="span12"><H3 class="bg-farnesene">Smell test</H3></DIV></DIV><DIV class="row-fluid"><DIV class="span12 white"><P align="justify">To functionally test our Methyl salicylate production brick (<A href="http://parts.igem.org/Part:BBa_K1060003" target="_blank">BBa_K1060003</A>), we prepared 68 microcentrifuge tubes with <I>E.coli</I> expressing the MeS construct to conduct a blind smell test. This experiment consisted of 4 sets, different in incubation time and temperature. The first set was grown at room temperature for 24 hours, the second was incubated at 37ºC for 24 hours, the third was grown at 37ºC for 8 hours and the last set was grown at room temperature for 8 hours. In each set, we inoculated our brick <A href="http://parts.igem.org/Part:BBa_K1060003" target="_blank">BBa_K1060003</A> with IPTG, IPTG + salicylate (concentration range from 0.01mM to 1mM), and IPTG + chorismate (concentration range from 0.01mM to 1mM), the same for MIT 2006 brick <A href="http://parts.igem.org/Part:BBa_K1060003" target="_blank">BBa_J45700</A>. In addition, the same combinations of additives were introduced in parallel for our negative controls (non-MeS producing <I>E. Coli</I>). 
We asked 11 people to participate in this blind test and calculated the percentage of people who smelled MeS. We can conclude that <B>the MeS/wintergreen smell was stronger after 24 hours incubation time compared to 8 hours. Growing the bacteria at 37 degrees also produced a more pronounced smell than those at room temperature. What’s more, in the microcentrifuge tubes where 0.1mM or 1mM salicylate was added, the smell became more obvious both after 8 and 24hrs of growth. The positive effect of adding chorismate became noticable after 24 hours</B>. The reason why chorismate requires more time to show an effect may be that chorismate is the very first precursor of the methyl salicylate production pathway, whereas the salicylate is an intermediate closer to the end product. As such, the impact of salicylate on the end product will become noticeable sooner. Most importantly, our brick worked in terms of producing methyl salicylate. 
The results are shown below:</P><DIV class="row-fluid"><DIV class="span6"><P align="justify">Figure 5 | Incubate at room temperature for 24 hours</P><P align="justify">Figure 7 | Incubate at 37 degrees for 8 hours</P></DIV><DIV class="span6"><P align="justify">Figure 6 | Incubate at 37 degrees for 24 hours</P><P align="justify">Figure 8 | Incubate at room temperature for 8 hours</P></DIV><DIV class="row-fluid"><DIV class="span12 greytext"><P align="justify">Figure 9 | Brief summary of smell test data</P></DIV></DIV></DIV></DIV></DIV><DIV id="header" class="row-fluid"><DIV class="span12"><H3 class="bg-farnesene">Headspace GC</H3></DIV></DIV><DIV class="row-fluid"><DIV class="span12 white"><DIV class="row-fluid"><DIV class="span12"><P align="justify"> Headspace gas chromatography (HS-GC) was used for the measurement of MeS production by our <A href="http://parts.igem.org/Part:BBa_K1060003" target="_blank"><B>BBa_K1060003</B></A> brick in a BL21 strain. Samples of 5 ml supernatant (centrifuged and filtersterilized) were collected in 15-ml precooled glass tubes, which were immediately closed and cooled on ice. Samples were then analyzed with a calibrated Autosystem XL gas chromatograph with a headspace sampler (HS40; Perkin-Elmer, Wellesley, Mass.) and equipped with a CP-Wax 52 CB column (length, 50 m; internal diameter, 0.32 mm; layer thickness, 1.2 μm; Chrompack; Varian, Palo Alto, Calif.). Samples were heated for 16 min at 72°C in the headspace autosampler. The injection block and flame ionization detector (FID) temperatures were kept constant at 180 and 250°C, respectively; helium was used as the carrier gas. The oven temperature was 75°C held for 6 min and then increased to 110°C at 25°C min−1 and held at 100°C for 3.5 min. Results were analyzed with Perkin-Elmer Turbochrom Navigator software.

MeS (Sigma-Aldrich, M 6752) was used as a standard and was detected at 12.832 minutes post injection (A). Cultures of <I>E. coli</I> BL21 and BL21 harbouring BBa_K1060003 with or without 0.1 mM salicylic acid (SA) supplemented to the medium were also analyzed. <B>No MeS could be detected with the FID in the samples when no SA was added (B), while MeS could be detected when SA was supplemented (C)</B>. 
The results are shown below:</P></DIV><DIV class="span12 greytext"><DIV class="row-fluid"><P> Samples were analyzed with a calibrated Autosystem XL gas chromatograph with a headspace sampler (HS40; Perkin-Elmer, Wellesley, Mass.) and equipped with a CP-Wax 52 CB column. Headspace GC was performed on pure MeS (Sigma-Aldrich) (A), on culture supernatant from a <I>E. coli</I> BL21 strain harbouring BBa_K1060003 without (B) or with 0.1 mM salicylic acid (C). MeS could be detected at 12.832 minutes in (A) and (C) but not when no salicylic acid was added.</P></DIV></DIV></DIV></DIV></DIV><DIV id="header" class="row-fluid"><DIV class="span12"><H3 class="bg-farnesene"> GC-MS analysis of MeS in ethyl acetate extracts </H3></DIV></DIV><DIV class="row-fluid"><DIV class="span12 white"><P align="justify"> To back-up our headspace GC data, we also analyzed our our brick <A href="http://parts.igem.org/Part:BBa_K1060003" target="_blank"><B>BBa_K1060003</B></A> with GC-MS with the help of Dr. Jan Baeten (Centre for Surface Chemistry and Catalysis; Prof. Dirk De Vos). Supernatant of bacterial cultures were extacted 3 times with ethyl acetate. GC-MS analyses were carried out using a 7890A Agilent gas chromatograph coupled to a 5977A mass spectrometric detector. The GC was equiped with a HP-5MS capillary column (30m x 0.25 mm x 0.25 µm). 1 µl of each sample was injected using splitless injection at a temperature of 250 °C. The initial oven temperature of 40 °C was held for 1 min., ramped at 6 °C / min. to 124 °C, then ramped at 20 °C / min. to 320 °C and finally kept at this temperature for 5 min. The transfer line and ion source were held at 320 °C and 230 °C, respectively. Mass spectra were taken between masses m/z 30-300 with an ionization potential of 70 eV. Retention indices of standards were determined by co-injection of a C7-C30 n-alkane mixture (Supelco) and were compared with published retention indices.

MeS (Sigma-Aldrich, M 6752) was used as a standard. MeS was detected as a pure peak with a retention index of RI = 1198 (lit. 1191). Ethyl acetate extractions of LB cultures (either controle BL21, BBa_K1060003 or BBa_K1060003 with 0.1 mM salicylic acid (SA) supplemented in the medium) were measured in SCAN-modus with a splitless injectie of 1 µl. No MeS was detected in the controle sample (BL21) but in both other samples MeS was present. For the sample without addition of SA, just a small amount of MeS could be observed (see figure 1).</P><DIV class="row-fluid"><DIV class="span12 greytext"><P> Total ion count (TIC) and extracted ion (EIC m/z 120+152) chromatograms of hexane extract of an <I>E. coli</I> BL21 culture harbouring brick BBa_K1060003 without (A) and with 0.1 mM salicylic acid (B). The mass spectrometer was turned off between 14.5-16 min. to avoid the measurement of extremely abundant peak (which was identified as indole). MeS could be detected at 13.6 min. Inset: measured spectrum at 13.6 min, confirmed as MeS (C8H8O3).</P></DIV></DIV></DIV></DIV><DIV id="header" class="row-fluid"><DIV class="span12"><H3 class="bg-farnesene">SDS PAGE analysis</H3></DIV></DIV><DIV class="row-fluid"><DIV class="span12 white"><DIV class="row-fluid"><DIV class="span6 greytext"><P align="justify">
Lanes a-d were induced at OD600nm 0.5, Lanes e-h were induced at OD600nm 1.0 with the indicated concentration of IPTG.
Cells were grown further for 1 hrs at 25°C. An identical experiment was performed with cells grown for 6hrs after induction. Finally, the brick was also tested with cells grown at 37°C.
</P></DIV><DIV class="span6"><P align="justify">
Similar to our set-up for the EBF synthase protein expression, we tested the MIT biobrick (<A href="http://parts.igem.org/Part:BBa_J45700" target="_blank">BBa_J45700</A>) and our novel MeS construct (<A href="http://parts.igem.org/Part:BBa_K1060003" target="_blank">BBa_K1060003</A>) via protein expression. We transformed our constructs in an <I>E.coli</I> expression strain, grew them at various temperatures (room temperature and 37 degrees celcius) and induced expression with increasing amounts of IPTG. We also added salicylate or chorismate to the growth medium in an attempt to increase MeS production.
The figure shows results obtained with the BBa_J45700 brick. Our biobrick (BBa_K1060003) showed similar results (data not shown). Increasing the amount of IPTG did not influence the protein expression profile (compare lanes a-d or lanes e-h) but we do see some bands in the lanes a-d which we cannot see in lanes e-f (eg a band just above 55 kDa). Nonetheless, this observation can be verified with lower amounts of the protein extracts. Nonetheless, our smell test would suggest that the MeS brick does work. Hence, we need a more sensitive approach to identify the protein production of the MeS brick. Possible approaches would be via classic western blot experiment or a GC-MS set-up.
</P></DIV></DIV></DIV></DIV><DIV id="header" class="row-fluid"><DIV class="span12"><H3 class="bg-green">Our Bricks</H3></DIV></DIV><DIV class="row-fluid"><DIV class="span12 white"><DIV class="row-fluid"><DIV class="span8"><P align="justify"><A href="http://parts.igem.org/Part:BBa_K1060000" target="_blank">BBa_K1060000</A> coding biobrick of DAHP synthase, encoded by the <I>aroG</I> gene from <I>E. coli</I> in the pSB1C3 backbone. The insert is 1053 bp long.
</P><P align="justify" class="greytext"> Figure 9 | <I>aroG</I> coding sequence (1053 bp) in pSB1C3 backbone (2070 bp). Digestion confirmation, cut with <I>EcoRI</I> and <I>PstI</I> restriction sites. Sequence confirmed. </P></DIV></DIV></DIV></DIV><DIV class="row-fluid"><DIV class="span12 white"><DIV class="row-fluid"><DIV class="span8"><P align="justify"><A href="http://parts.igem.org/Part:BBa_K1060003" target="_blank">BBa_K1060003</A> generator biobrick. It is a twin of the <A href="http://parts.igem.org/Part:BBa_J47500" target="_blank">BBa_J47500</A> biobrick made by the 2006 MIT team, put into a pSB1C3 backbone. The coding sequence is 3255 bp long.
</P><P align="justify" class="greytext">Figure 10 | Methyl salicylate producing construct (3255 bp) in pSB1C3 backbone (2070 bp). Digestion confirmation, cut with <I>EcoRI</I> and <I>PstI</I> restriction sites. Sequence confirmed. </P></DIV></DIV></DIV></DIV><DIV class="row-fluid"><DIV class="span12 white"><P align="justify"><A href="http://parts.igem.org/Part:BBa_K1060004" target="_blank">BBa_K1060004</A> intermediate biobrick consisting of <I>bsmt1</I> followed by a double terminator. It is in the pSB1C3 backbone and the insert is 1177 bp long. Sequence confirmed.
<A href="http://parts.igem.org/Part:BBa_K1060005" target="_blank">BBa_K1060005</A> intermediate biobrick consisting of <I>pchBA</I> followed by a double terminator. It is in the pSB1C3 backbone and the insert is 1839 bp long. Sequence confirmed.</P></DIV></DIV><DIV id="header" class="row-fluid"><DIV class="span12"><H3 class="bg-green">References</H3></DIV></DIV><DIV class="row-fluid"><DIV class="span12 white"><P align="justify">

Dosselaere, F., Vanderleyden, J., A metabolic node in action: chorismate-utilizing enzymes in microorganisms, Crit Rev Microbiol 2001;27(2):75-131.
Gaille, C., Reimmann, C., Haas D., Isochorismate synthase (PchA), the first and rate-limiting enzyme in salicylate biosynthesis of <I>Pseudomonas aeruginosa</I>, The Journal of Biological Chemistry 2003, 278(19):16893-16898.Hu, C., Jiang, P., Xu, J., Wu, Y., Huang, W., Mutation analysis of the feedback inhibition site of phenylalanine-sensitive 3-deoxy-D-arabino-heptulosonate 7-phosphate synthase of <I>Escherichia coli</I>, J. Basic Microbiol. 43 (2003) 5, 399-406.
Ikeda, M., Towards bacterial strains overproducing L-tryptophan and other aromatics by metabolic engineering, Appl Microbiol Biotechnol 2006, 69:615-626.
Lin, S., Meng, ., Jiang, J., Pang, D., Jones, G., OuYang, H. Ren, L., Site-directed mutagenesis and over expression of aroG gene of <I>Escherichia coli</I> K-12, International Journal of Biological Macromolecules 51 (2012) 915-919.
Sprenger, G., From scratch to value: engineering <I>Escherichia coli</I> wild type cells to the production of L-phenylalanine and other fine chemicals derived from chorismate, Appl Microbiol Biotechnol 2007, 75:739-749.
Sun, Z., Ning, Y., Liu, L., Liu, Y., Sun, B., Jiang, W., Yang, C., Yang, S., Metabolic engineering of the L-phenylalanine pathway in <I>Escherichia coli</I> for the production of S- or R-mandelic acid, Microbial Cell Factories 2011, 10:71.</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:KU_Leuven/Project/Glucosemodel/MeS">http://2013.igem.org/Team:KU_Leuven/Project/Glucosemodel/MeS</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:KU_Leuven/Project/Glucosemodel/MeS" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:KU_Leuven/Project/Glucosemodel/MeS" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:KU_Leuven/Project/Glucosemodel/MeS&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:KU_Leuven/Project/Glucosemodel/MeS&amp;oldid=360053" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>