"
Page
Discussion
View source
History
teams
Log in
 
Team:KU Leuven/Journal/FFL/wetlab
From 2013.igem.org
iGem
Secret garden
Congratulations! You've found our secret garden! Follow the instructions below and win a great prize at the World jamboree!
A video shows that two of our team members are having great fun at our favourite company. Do you know the name of the second member that appears in the video?
For one of our models we had to do very extensive computations. To prevent our own computers from overheating and to keep the temperature in our iGEM room at a normal level, we used a supercomputer. Which centre maintains this supercomputer? (Dutch abbreviation)
We organised a symposium with a debate, some seminars and 2 iGEM project presentations. An iGEM team came all the way from the Netherlands to present their project. What is the name of their city?
Now put all of these in this URL:https://2013.igem.org/Team:KU_Leuven/(firstname)(abbreviation)(city), (loose the brackets and put everything in lowercase) and follow the very last instruction to get your special jamboree prize!
Menu
Data Page
Project
Aphid Background
Honeydew system
Sticker system
E. coligy
Parts
Modelling
Cellular level
Colony level
Ecosystem level
Team
Students & Advisors
Attributions
Collaborations
Sponsors
Official Team Profile
Human Practices
Top Down
Bottom Up: Framework
Bottom Up: Public Outreach
Notebook
Journal
Protocols
Safety
iGEM
 
Modelling
Modelling our different systems
Ecology
Plant and insect experiments
E-β-farnesene
Work about our EBF-biobrick
Feed forward loop
You are here!
Methyl salicylate
Work about our MeS-biobrick
qPCR
Finding wetlab data for the models
Feed-Forward Loop Journal
This is the journal of the Feed-Forward Loop team, a part of the oscillator model. For more information about their project we refer you to their page .
24/07 – 26/07
Week 4: Getting started
This week we started working on the proof of concept of the feed forward loop in the lab.  First of all we selected the necessary bricks and took the plasmids out of the iGEM distribution kit plates. We transformed them into chemically competent inoue cells (which were made before and stored at -80°C) according to the ‘transformation kit protocol’.
Following bricks were transformed: C0076, K823016, C0077, R0080, P0340, R0078, K082003, C0080 and R0011.
All of them were plated out in the presence of  chloramphenicol, except for R0011 which is resistant to ampicilline. We made 2 plates for every brick, one with 100 µl and one with 1000 µl LB. The bacteria were grown overnight at 37°C.
The next day we checked our plates and all of them had colonies except for the plates of R0011. We picked one colony of every plate and put the bacteria into tubes with 4 ml of LB medium and 4 µL chloramfenicol to incubate overnight in a shaking incubator at 37°C. Then we decided to transform the failed brick (R0011) and an extra brick (C0060) by electroporation because the efficiency of this method should be higher. We plated them out and let them grow overnight at 37°C.
The plates with brick C0060 had nicely grown colonies and we put them at 4°C for usage next week. The plates with brick R0011 didn't show colonies. Afterwards we noticed that we used the wrong antibiotic resistance. We took the tubes with the successful bricks out of the shaking incubator and we used 400 µl of the LB with bacteria together with 400 µl of glycerol to make a stock in the -80°C.
The remaining 3.6 ml was used for a plasmid extraction using the home-made protocol. After the extraction we used the NanoDrop to see if we had a good amount of DNA and no interference of proteins or salts. The values are shown below:
ng/µl
260/280
260/230
C0076
A
94.6
1.34
0.55
 
B
10.9
1.37
0.41
K823016
A
12.5
1.29
0.43
 
B
8.7
1.34
0.38
C0077
A
4.1
1.91
0.48
 
B
2.9
1.70
0.46
R0080
A
18.1
2.02
0.71
 
B
4.9
1.23
0.32
P0340
A
73.3
1.83
0.99
 
B
155.8
1.92
1.52
R0078
 
111.6
1.73
1.07
K082003
A
127.3
1.89
1.55
 
B
197.1
2.11
1.29
C0080
A
200.6
1.81
1.24
 
B
7.2
1.92
1.04
29/07 – 02/08
Week 5: Our very first Feed Forward Loop-BioBrick!
This week, we decide to do our first digestions. We digested the GFP brick (K082003) with SpeI and EcoRI and the brick with the double terminator (B0015) with XbaI and EcoRI. We put the digested GFP brick on gel in three lanes. After running the gel, we cut the gel in two: one part containing the ladder and one lane with GFP, the other part containing two lanes with GFP. We only stained the part of the gel containing the ladder, this to avoid mutations in the insert of the two other lanes. These lanes were used for gel extraction (and ligation afterwards), the lane next to the ladder was used to determine where the insert was located on the gel. After visualisation we marked where our digest was and cut out the part at the same height in the other gel. Afterwards the second gel was stained to check if the bands were cut out correctly. Then we used a kit for gel extraction of the GFP fragment.
digested GFP brick (vector+insert)
After we dephosphorylated the digested vector with terminator and did a pcr purification with a kit, we ligated the GFP insert in this vector according to the ‘Digestion and ligation protocol’. We transformed the product by electroporation. The day after no colonies had appeared, which was expected because the electroporation machine indicated that there was no DNA in the sample (which is of course not possible). So we did the transformation again, but this time with chemically competent inoue cells according to the ‘transformation kit protocol’. Colonies appeared and we inoculated 2 of them in 4 ml LB with chloramphenicol.
We also inoculated some bricks that were stored at -80°C for usage in the next few weeks, namely B0015, B0034 and K608002. We did plasmid extraction of these and of the cells with GFP + terminator next day. We used the NanoDrop to measure the concentrations shown below. We digested 10 µl of the construct with GFP and terminator with EcoRI and PstI to confirm if the right insert is present. This was indeed the case, as you can see on the gel.
 
ng/µl
260/280
260/230
K082003+B0015
A
141.1
2.05
2.18
 
B
82.6
1.63
1.65
B0015
A
93.1
2.12
1.75
 
B
51.6
1.72
1.49
 
C
106.6
2.08
1.78
B0034
A
317.2
1.92
2.01
 
B
300.7
1.94
2.12
K608002
A
110.9
1.87
1.32
 
B
145.9
1.89
1.51
Because we didn’t get high plasmid concentrations for all of the bricks we transformed in week 4, we inoculated a colony from the plates containing K823016, C0060, R0080, C0076 and C0077 in 4ml LB, in the presence of chloramphenicol. The bacteria were grown overnight in the shaking incubator at 37°C. The day after we extracted the plasmids, but this time we used a kit instead of the home-made protocol. The values of the NanoDrop are shown below. Both the yield and purity were better than before.
ng/µl
260/280
260/230
K823016
405.4
1.93
2.29
C0060
110.2
1.45
1.64
R0080
71.3
2.28
2.16
C0076
199.2
1.30
0.97
C0077
200.3
1.94
2.17
This week we also digested the bricks C0067, C0077, C0080, K823016, P0340 and C0060 with SpeI en EcoRI. We put the digestions on a gel and all bricks showed the correct bands (one for the insert and one for the vector). We excised the bands and purified them with a kit.
Order: ladder - K823016 - P0340 – C0080 – C0077 – C0060 – C0076
We cut the plasmid containing brick I719005 with SpeI and PstI, dephosphorylated the vector and did a pcr purification. We used the NanoDrop and measured a concentration of 34,2 ng/µl.
05/08 – 09/08
Week 6: Ligations, ligations and some more ligations!
We started this week by doing the following digestions:
B0015 with EcoRI and XbaI
K608002 with SpeI and PstI
B0034 with SpeI and PstI
Afterwards we treated the digestion mixtures with phosphatase and used a kit for pcr purification. The NanoDrop gave us these values afterwards:
 
ng/µl
260/280
260/230
B0015
A
73
1.7
0.82
 
B
35.6
1.96
1.09
 
C
56
1.83
1.09
 
D
43.6
1.91
1.05
 
E
70.9
1.74
0.84
K608002
 
37.6
2.04
1.5
B0034
 
38
2.01
2.02
Our new brick with GFP + terminator (K082003+B0015) was cut with XbaI and PstI and put on a gel. The insert was extracted and purified with a gel extraction kit.
Order: ladder - Digested GFP + terminator genes
Afterwards we did following ligations:
C0076 (insert) + B0015 (vector)
C0077  (insert) + B0015 (vector)
C0080 (insert) + B0015 (vector)
K823016 (insert) + B0015 (vector)
(K082003+B0015; insert) + K608002 (vector)
(K082003+B0015; insert) + B0034 (vector) Ampicilline resistant
P0340 (insert) + I719005 (vector)  Ampicilline resistant
C0060 (insert) + B0015 (vector)
We transformed inoue cells with the ligation products, according to the ‘transformation kit protocol’. Two of the plates contained ampicilline, the other ones chloramfenicol. We incubated them overnight at 37°C. The next day all of the plates contained colonies, except for the one containing C0060+B0015. Therefore we used the same ligation product again to transform some new inoue cells. This time the transformation worked. We inoculated one colony of each plate in 4 ml LB in the presence of the correct antibiotics. Unfortunately only in 3 of the tubes bacteria had grown. We inoculated a colony from the failed plates again. The tubes with growth were used for plasmid extraction, the result is displayed below:
ng/µl
260/280
260/230
K082003+B0015+K608002
203.2
1.94
2.48
C0077+B0015
136.4
1.97
2.55
P0340+I719005
206.9
1.94
2.46
We used 10 µl of each to do a digestion with EcoRI and PstI and put them on a gel to check if the correct inserts are present, which was not the case.
The next day all tubes contained grown cells, so this time the inoculation didn’t fail. We did a plasmid extraction and NanoDrop:
 
 
ng/µl
260/280
260/230
Enzymes
Correct insert?
1
K823016 + B0015
 
89.2
1.69
1.25
XbaI +PstI
 
2
(K082003+B0015) + B0034
 
38.8
3.04
0.62
XbaI +PstI
Ok
3
C0076 +B0015
 
47
2.26
1.25
XbaI +PstI
Ok
4
P0340 + I719005
A
37.6
3.28
0.74
EcoRI + PstI
Ok
5
 
B
30.4
3.28
1.30
EcoRI + PstI
Ok
6
C0077 + B0015
A
125.3
2.27
1.86
XbaI +PstI
 
7
 
B
24.6
5.92
1.2
XbaI +PstI
 
8
C0060 + B0015
 
31.3
3.42
0.99
XbaI +PstI
 
9
(K082003+B0015)+ K608002
A
44.9
2.32
1.82
XbaI +PstI
 
10
 
B
31.2
4.34
0.87
XbaI +PstI
 
11
C0080 + B0015
 
54
2.27
0.56
XbaI +PstI
Afterwards we digested the vectors with the enzymes indicated in the table. We put them all on a gel but only four had the correct bands (see last column). These plasmids were sent for sequencing.
Order: ladder - 4 - 5 – 9- 10 – 6 – 7 - 8 - 11 - 3 - 1 - 2
We took some cells containing our new brick (GFP + terminator) out of the -80°C and put them in 4 ml LB with chloramphenicol. After plasmid extraction (using the kit), we became the values  shown below. We sent this plasmids for sequencing.
 
ng/µl
260/280
260/230
K082003+B0015
A
101.3
1.98
2.67
 
B
105.3
1.96
2.42
We did the same thing for the 4 ligations that succeeded. After plasmid extraction we got the following concentrations:
 
ng/µl
260/280
260/230
(K082003+B0015) + B0034
 
25.8
2.05
2.12
P0340 + I719005
A
59.2
1.95
1.68
 
B
72.8
1.94
1.99
C0076 + B0015
 
117.7
1.87
1.35
We digested the vectors containing R0078, R0080 and B0034 with SpeI and PstI. The vector with R0078 was also cut once with EcoRI and PstI. We dephosphorylated the vectors and did a clean-up with the pcr purification kit.
We extracted the four correct bands out of the gel and purified them. Afterwards we did the following ligations:
R0087 (vector; cut with EcoRI and PstI) + (P0340 + I719005; A; insert)
R0087 (vector; cut with EcoRI and PstI) + (P0340 + I719005; B; insert)
B0034 (vector)+ (C0076 +B0015; insert)
R0087 (vector; cut with SpeI and PstI)+ (K082003+B0015 + B0034; insert)
R0080 (vector) + (K082003+B0015 + B0034; insert)
We digested the vector with K398326 with SpeI and PstI and EBF 4 with EcoRI and PstI. Both will be used in the napkin model. The first one was dephosphorylated and pcr purified with a kit. The following ligations were done:
K398326 (vector) + P0340 (insert)
R0087 (vector; cut with EcoRI and PstI) + EBF 4
We transformed inoue cells with those seven ligation products and we also transformed some cells again with the remaining ligation product of K823016 + B0015; C0077 + B0015; (K082003+B0015)+ K608002; C0080 + B0015 (these are the ones that didn’t show good bands on gel). So in total we did eleven transformations. All plates showed some growth and we inoculated four colonies from each plate in LB in the presence of the correct antibiotics.
12/08 – 16/08
Week 7: Everything fails
We started with a plasmid extraction of R0087+EBF4 and K398326 + P0340. Therefore we used the home-made protocol. These are the results of the NanoDrop:
 
ng/µl
260/280
260/230
K398326 + P0340
33
6.6
1.88
0.66
 
34
14.6
1.91
1.49
 
35
3.2
2.49
1.37
 
36
9.4
1.81
0.75
R0087+EBF4
37
1.8
48.92
0.4
 
38
32.1
1.79
1.35
 
39
1.8
-21.30
3.02
 
40
0.5
-0.46
-0.47
As you can see we only got low DNA concentrations. We digested all the plasmids and put the digestion product on gel to see if the correct insert was present. This was not the case.
top: ladder - 33 - 34 - 35 - 35, bottom: ladder - 37 - 38 - 39 - 40
Because of the low plasmid colonies we inoculated some bacteria of the same colonies (which were stored at -80°C) again in LB and decided to use to kit this time to do the plasmid extraction. The cells with the nine other constructs were centrifuged and stored at -20°C for later usage.
The results of the plasmid extraction with the kit are shown below:
 
ng/µl
260/280
260/230
K398326 + P0340
33
46.8
1.37
1.51
 
34
36.9
1.83
1.30
 
35
45.9
1.80
1.40
 
36
37.4
2.32
1.42
R0087+EBF4
37
38.5
2.33
1.60
 
38
46.1
2.13
1.51
 
39
47.8
2.15
1.35
 
40
45.5
2.12
1.39
Plasmids were again digested and put on gel, again the results were not good.
We decided to use the home-made protocol for plasmid extraction for 3 of the 4 colonies of the other nine constructs. For the remaining colonies we used a kit. These are the results of the home-made protocol:
 
ng/µl
260/280
260/230
K823016 + B0015
1
141.3
1.56
0.64
 
2
36.4
1.82
2.32
 
3
110.4
1.53
1.22
 
 
 
 
 
R0087+ (K082003+B0015 + B0034)
5
5.7
1.69
1.15
 
6
5.9
0.71
0.49
 
7
11.4
1.3
0.53
 
 
 
 
 
R0080+ (K082003+B0015 + B0034)
9
3.3
1.47
1.21
 
10
37.5
1.77
1.93
 
11
99.5
1.54
0.83
 
 
 
 
 
R0087 + (P0340 + I719005; A)
13
5.5
1.38
1.04
 
14
25.8
1.25
0.7
 
15
256.7
1.5
0.59
 
 
 
 
 
(K082003+B0015)+ K608002
17
127.6
1.53
0.66
 
18
87.9
1.6
0.8
 
19
8.1
1.7
1.4
 
 
 
 
 
C0077 + B0015
21
12.7
1.61
1.82
 
22
35.6
1.62
1.05
 
23
8
1.76
1.34
 
 
 
 
 
B0034 +(C0076 +B0015)
25
97.6
1.59
0.78
 
26
45.9
1.51
0.61
 
27
179.4
1.5
0.75
 
 
 
 
 
C0080 + B0015
29
39.3
1.53
0.82
 
30
58.7
1.47
0.47
 
31
64.5
1.56
0.6
 
 
 
 
 
R0087 + (P0340 + I719005; B)
41
28
1.49
0.66
 
42
26.3
1.48
0.66
 
43
58
1.46
0.59
These are the results of the kit:
 
ng/µl
260/280
260/230
K823016 + B0015
4
43.8
1.09
1.1
R0087+ (K082003+B0015 + B0034)
8
35.2
1.12
1.14
R0080+ (K082003+B0015 + B0034)
12
120.4
1.35
0.78
R0087 + (P0340 + I719005; A)
16
62.8
1.11
1.08
(K082003+B0015)+ K608002
20
53.3
1.15
1.00
C0077 + B0015
24
36.2
1.58
1.07
B0034 +(C0076 +B0015)
28
58
1.11
1.09
C0080 + B0015
32
9.70
-3.36
1.57
R0087 + (P0340 + I719005; B)
44
12.6
13.42
1.94
We digested all the plasmids and put them on gel. None of them had the correct inserts.
ladder -1-2-3-5-6-7
ladder-4-8-9-10-11-12-13-14-15-16-20-24-28-32-44
ladder-17-18-19-21-22-23-25-26-27-29-30-31-41-42-43
Since all ligations failed, we decided to throw away the petri dishes and do some of the ligations again. We cut brick C0077 and C0080 out of the vector, put them on a gel, cut out the correct bands and did a gel purification. We cut the vector with B0015 (terminator) open, dephosphorylated the ends and did a pcr purification with a kit. Afterwards we did the ligation of both bricks in the vector. This time we used a longer time for ligation than the first time.
19/08 – 23/08
Week 8: Trying new strategies
This week we transformed inoue cells with the two ligation products that were made at the end of previous week. We also digested K823016 and C0060 with EcoRI and SpeI, put them on gel and did a gel extraction of the insert. After gel purification we ligated K823016+B0015 and C0060+B0015 and did a chemical transformation of inoue cells. All our transformations have succeeded, there was growth on all plates. Two colonies from each plate were inoculated in LB with antibiotics. These are the results:
ng/µl
260/280
260/230
C0060+B0015
126.9
1.85
1.77
 
77.2
1.74
0.99
C0080+B0015
27.8
1.37
1.23
 
22.4
2.19
1.01
K823016+B0015
38.6
1.48
0.78
 
33.7
1.79
1.24
C0077+B0015
45.7
1.95
0.99
 
139.4
1.86
1.71
Plasmids were digested and put on gel. Only one of the eight colonies had the correct insert, although you can see three bands on gel instead of two. One of +- 2000 bp (the vector), one around 800 bp (the insert) and one of 3000 bp. We think this is undigested plasmid. The correct band belongs to the ligation C0077 + B0015. The vector is sent for sequencing.
Since we only tested two colonies of each plate, we decided to inoculate ten other colonies in LB for the ligation of C0060+B0015. After plasmid extraction with the home-made protocol, the NanoDrop gave us these values:
ng/µl
260/280
260/230
A
97.7
1.73
1.3
B
/
/
/
C
130.9
1.67
0.83
D
107.7
1.73
0.98
E
98.1
1.59
0.83
F
203.8
1.8
1.36
E
98.1
1.59
0.83
F
203.8
1.8
1.36
G
93.1
1.61
0.78
H
186.1
1.6
0.71
I
59.7
1.59
0.78
J
130.3
1.65
0.88
The ligation of K398326 + P0340 was also repeated and inoue cells were again transformed. These plates did show some growing bacteria after incubation at 37°C. We inoculated ten colonies in LB medium. After plasmid extraction with the home-made protocol, the NanoDrop gave us these values:
ng/µl
260/280
260/230
A
111.6
1.58
0.75
B
98.4
1.66
0.81
C
153.8
1.69
0.98
D
103.1
1.62
0.73
E
51.3
1.68
0.82
F
28.6
1.57
0.98
G
26
1.74
1.18
H
87.2
1.64
1.11
I
2.2
1.67
0.71
J
59.1
1.74
1.18
All samples were digested with XbaI and PstI and put on gel. No correct bands were seen. All bands were around 1500 bp but we expected bands of around 800 bp. It’s possible that two inserts have ligated into the same vector.
We decided to try another strategy: colony PCR. This was performed for 8 colonies of C0080+B0015 with the primers igem001 and igem002 according to the ‘colony pcr protocol’. KAPA robust mastermix was used. When the PCR was finished, the mixtures were put on gel. We only got bands of 500 bp instead of 800 bp. This difference could be due to the dependency of the migration rate on the amount of DNA loaded. But it’s also possible that that correct insert is not present, because some other teams also got bands at 500 bp when they did a colony PCR.
A colony PCR was also done for 8 colonies of C0060+B0015. This time we also did a positive control (our new GFP+terminator brick) and a negative control (no DNA added). The negative control showed the same bands as all the colonies, so probably something was contaminated.
26/08 – 30/08
Week 9: Troubleshooting
Because much of the previous ligations failed, we decided to do a troubleshooting this week. We performed the ligation of K398326 + P0340 again, but with a shorter ligation time and with different ratio’s of vector and insert. The ligation time was chosen to be 15 minutes and the inactivation time 5 minutes at 70°C. The different insert/vector ratios were chosen to be 14/3 - 1/1 - 2/2 - 3/3 - 5/5 - 1/5. Afterwards the ligation product was put on gel to check if the correct bands were present (one band of 2100 bp and one of 800 bp).  Each lane of the gel shows a nice ladder as expected: a band containing unligated vector, a band with vector+insert, a band with vector + 2 inserts, ... After we looked at the gel, we decided that a ratio of 5/5 is the best to do a ligation. In this lane the correct band (vector + insert) was the clearest and there were not much wrong bands containing more inserts. Ten minutes of ligation time should be enough, since no insert was visible on gel anymore and all the insert was ligated.
ladder - 14/3 - 1/1 - 2/2 - 3/3 - 5/5 - 1/5
Retrieved from " http://2013.igem.org/Team:KU_Leuven/Journal/FFL/wetlab "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
