"
Page
Discussion
View source
History
teams
Log in
 
Team:KU Leuven/Journal/EBF/wetlab
From 2013.igem.org
iGem
Secret garden
Congratulations! You've found our secret garden! Follow the instructions below and win a great prize at the World jamboree!
A video shows that two of our team members are having great fun at our favourite company. Do you know the name of the second member that appears in the video?
For one of our models we had to do very extensive computations. To prevent our own computers from overheating and to keep the temperature in our iGEM room at a normal level, we used a supercomputer. Which centre maintains this supercomputer? (Dutch abbreviation)
We organised a symposium with a debate, some seminars and 2 iGEM project presentations. An iGEM team came all the way from the Netherlands to present their project. What is the name of their city?
Now put all of these in this URL:https://2013.igem.org/Team:KU_Leuven/(firstname)(abbreviation)(city), (loose the brackets and put everything in lowercase) and follow the very last instruction to get your special jamboree prize!
Menu
Data Page
Project
Aphid Background
Honeydew system
Sticker system
E. coligy
Parts
Modelling
Cellular level
Colony level
Ecosystem level
Team
Students & Advisors
Attributions
Collaborations
Sponsors
Official Team Profile
Human Practices
Top Down
Bottom Up: Framework
Bottom Up: Public Outreach
Notebook
Journal
Protocols
Safety
iGEM
 
Modelling
Modelling our different systems
Ecology
Plant and insect experiments
E-β-farnesene
You are here!
Feed forward loop
Implementation of the oscillator in vivo
Methyl salicylate
Work about our MeS-biobrick
qPCR
Finding wetlab data for the models
E-β-Farnesene Journal
This is the journal of the EBF team, a part of the glucose model . For more information about their project we refer you to their page .
26/06 – 30/06
Week 0: Starting off easy
Some of us have already finished their exams so we started working in the lab, preparing everything for the main work that wil start July 1st. We poured the agar plates with and without the antibiotics and made the FSB buffer that will be needed to make competent cells.
01/07 – 07/07
Week 1: Selecting the bricks
Finally, all of us are finished with the exams, so we can really start working in the lab, We started to make chemically competent Top10 and DH5alpha cells. To test our competent cells we tried the iGEM transformation efficiency kit, but surprisingly no colonies appeared in any of them! Now we have to figure out what went wrong. Possible reasons are:
Bad competent cells
We didn't use enough recovery time for the cells.
The cells need more time to grow.
To examine what went wrong we did the experiment over again, but this time we used our own pUC 19-vector and let the cells recover for 2 hours instead of one. Now we do have cell growth, so we counted the cells and calculated the transformation efficiency. This was still quite low though...
We prepared the GTE-buffer for future use and prepared agar medium. Later, we found out that the medium in the autoclave spilled out everywhere in the autoclave, possibly due to the pressure. Lukas was the lucky guy who got to clean out the autoclave. The parts from iGEM have arrived! We selected 16 parts to work with further to make our EBF-producing-devices.
RBS
Promoter
Composite
Coding/generator
intermediate
terminator
BBa_B0032
BBa_K398326Promoter o f the CaiF protein
BBa_K314100 High constitutive expression cassette
BBa_J45199 Banana odor enzyme (ATF1) generator
BBa_K608006
BBa_K864600
BBa_B0034
BBa_K808000 araC-Pbad - Arabinose inducible regulatory promoter/repressor unit
BBa_K314103 Lac induced expression cassette
BBa_J45014 alcohol acetyltransferase I; converts isoamyl alcohol to isoamyl acetate (banana odor)
BBa_K608002
BBa_K823017
 
BBa_R0040 TetR repressible promoter
 
 
 
 
 
BBa_I13453 Pbad promoter
 
 
 
 
 
BBa_I719005 T7 Promoter
 
 
 
These parts were transformed to our chemically competent cells. During the heat shock the heat automatically turned off after one round, wich we didn't realize at first. So for bricks 1I, 7E, 5E, 20O, the heat shock temperature is a bit lower (around 37°C). Hence during the plating we used 50µl of transformed competent cells instead of 20µl for these samples. The cells were left to grow over the weekend in medium containing chloramphenicol.
08/07 – 14/07
Week 2: Gettin' the gene
Unpatient as we were, we first checked the plates of the transformed competent cells and found out that only 4 petri dishes have a small amount of colonies (7E2 top10, 5E3 top10, 1G1top10, 5E3 DH5alpha). Interestingly, the ones where we used 50µl of cells to plate instead of 20µl, were most successful. This means that the lack of colonies may be caused by a too low amount of cells put on the petri dishes. This in turn also suggests that the transforming efficiency is quite low. We repeated the transformation with 50µl or even more cells for the unsuccessfully transformed biobricks. Later we also performed the plasmid extraction on all the successfully transformed biobricks.
We wanted to do a colony PCR on Streptomyces coelicor, so first we prepared the primer stocks for that. Streptomyces have a gene coding for EBF synthase. This enzyme produces EBF out of farnesyl diphosphate.
We extracted the genome from Streptomyces using 4 different methods (as Streptomyces is gram negative, it is a bit difficult to extract the genome):
Microwave Streptomyces for 4 mins
Streptomyces in water with 0.2%SDS, 4 min microwave.
Streptomyces in water and 1% SDS, 4 min microwave.
Streptomyces in TE buffer, 0.2% SDS, 4 min microwave.
We ran a colony PCR for all these Streptomyces and a negative control. We cleaned up the PCR product and ran the gel of the DNA we got from the colony PCR. The gel was stored in TAE buffer at 4° C, so that we could visualize the gel next Monday.
We also tried to make competent TOP10 and DH5alpha cells using the Inoue method instead of our regular chemical competent cells.
15/07 – 21/07
Week 3: Multitasking
Amplify Backbone
As the amount of backbone DNA we received from iGEM is very small, we decided to amplify the backbones ourselves. For this we chose 3D1, a small biobrick coding for a double terminator. We inoculated the 3D1 transformed strain into 3 ml LB medium, then isolated the plasmid and cut the backbone out of the plasmid. To make sure that we have enough backbone we did this in quadruple. Before we did the digestion, we ran a nanodrop for these 4 samples to check the DNA concentration.
conc (ng/µl)
260/280
260/230
1
102.7
1.84
1.28
2
186.4
1.87
1.29
3
248.3
1.88
1.26
4
72.3
1.80
1.08
The digestion was done with EcoRI+PstI.
The band of the backbone does not fit our expected band very well so we decided not to use these backbones.
Device
We visualized the gel we ran last Friday, containing the EBF-producing genes that were obtained from the colony PCR. For pretreatment 1 and 2 we can see a clear band of DNA. For these we than ran a high fidelity PCR. These PCR products were made into stocks and diluted 15 times (1µl PCR products + 14µl milliQ). Then we purified the PCR products and did a nanodrop for the PCR products we got:
Gene
concentration(ng/µl)
260/280
260/230
EBF 1
23.3
1.89
0.43
EBF 2
37.0
1.61
0.68
To make our very first biobrick we performed digestion and ligation of the EBF1 gene and the chloramphenicol backbone that we received from iGEM.
Biobricks
We also performed plasmid isolation for the successfully electroporated biobricks (1I2, 1G1, 5E3, 2M5, 1H5, 7E2).
We ran nanodrop for the biobrick plasmids we wanted to digest.
parts
conc. (ng/µl)
260/280
260/230
17D1
18.1
1.65
0.97
3O1
299.3
1.99
1.71
1C1
107.3
1.61
0.73
1I2
32.6
1.74
1.34
2M5
75.5
1.70
0.94
5,00E+001
0.3
(failed)
Then we digested these parts with EcoRI+SpeI, except for 17D1 which was digested with XbaI+PstI. For all of them we used 1µg of DNA except for 17D1 (434.4ng due to the limited amount we have). We ran a gel with 20 µL of the product. The rest of the digested DNA we stored at -20°C after denaturing the enzyme by putting it at 80 °C for 20min.
After visualizing the gel with EtBr, we could only observe the bands of the backbone (and some other bands that we didn’t know where they come from), the band of the parts we want could not be seen. The possible reason could be that these parts are too small (they are promoters, RBS, etc, with a size rangeing from 12 to 72bp), hence the EtBr cannot bind them in a sufficient amount and therefore we could not see them. This is why we will need another strategy to tackle the problem of assembly. We transformed Inoue competent cells with 4F3, 5D3, 23L5 (an alternative for double terminator), 5E1 and EBF. All these transformations were successful! We inoculated these cells with 4ml LB with correct antibiotics, incubated them at 37°C overnight and transferred them to 4°C.
Device 2
First we digested the Ampicillin, Tetracyclin and Kanamycin backbones from iGEM with EcoRI+PstI restriction enzymes and purified the DNA with the PCR purification kit. We digested EBF1 with Xba1+PstI restriction sites, purified it with the PCR purification kit, then ligated the digested 3O1, EBF1 and Kan backbone to make our second device. We left the incubation of the ligation at room temperature over the weekend.
22/07 – 28/07
Week 4: Problems and problemsolving
We extracted the plasmid from the cultivated cells containing 5E1 4F3 23L5 and D1 (device 1) with the use of the plasmid purification kit, and ran the nanodrop for the isolated plasmids:
Sample
Concentration (ng/µl)
A260/280
A260/280
5E1,a
170.2
1.89
2.16
5E1,b
97.7
1.89
1.92
D1,1
128.1
1.90
2.12
D1,2
133.2
1.87
2.06
4F3,a
130.4
1.91
2.08
4F3,b
187.2
1.89
2.15
23L5,a
191.1
1.89
2.08
23L5,b
210.8
1.88
2.15
5D3,a
95.5
1.89
2.08
5D3,b
95.6
1.89
2.00
In the afternoon, we digested 5E1a  with EcoRI + SpeI and D1,1 and D1,2 with EcoRI + PstI. We ran a gel for D1,1 and D1,2 to confirm we have the EBF synthase gene. Then we ligated 5E1 + EBF (=D1)+ pSB1K3 (the standard iGEM backbone).  We incubated it at room temperature overnight.
We ran a gel for these, but the results were not exactly cheering. We only got 1 band from the digested EBF device D1, the possible reason could be that the backbones ligated to eachother to form a bigger plasmid, instead of inserting the EBF into the plasmid. Since we only took 2 colonies, these results are possible.
We did a colony PCR for 10 randomly selected D1 colonies and ran the gel to confirm they have D1 plasmid. The results are weird however, we only got a band with a short length of DNA. A possible reason could be that the anealing temp for the primer was a little low so that the primer anealed unspecifically, which lead to the short secquence products.
Therefore we randomly selected 17 more colonies, plus 10 colonies from yesterday, and performed another round of colony PCR. The nanodrop results for these colony PCR are really high, meaning that the PCR worked. We later ran the gel, but the results were again not pleasant. It's very possible that the ligation (or even digestion) was not good.
We transformed D2 and D3 (D2 is promoter 3O1 in front of EBF, D3 is promoter 5E1 in front of EBF) into inoue cells. This wass unsuccessful so we tried again with electroporation. We transformed D2 and D3 by electroporation and plated out the transformed cells on Kan plates. The tranformation was again not successful, so we tried again with a higher DNA concentration.
The EBF-plasmid finally arrived. This plasmid is the (E)-beta-farnesene synthase from Artemisia Annua in the pET28 backbone. The plasmid was on filter paper, so we used an elution buffer to retrieve it. We transformed the EBF plasmid to competent cells from the mutagenesis kit (with this kit we want to mutate a specific site in the EBFaa pET28 plasmid)) using electroporation and plated them out on Kan plates. This transformation was successful.
We started our new strategy for digestion and ligation: First we ran a nanodrop for the bricks
brick
conc (ng/µl)
260/280
260/230
 
3O1?
54.3
1.87
0.94
 
3O1
331
2.02
1.71
chosen
5E1,a
185.9
1.92
2.23
chosen
5E1,b
95.3
1.93
2.07
 
4F3,a
116.6
1.93
2.19
 
4F3,b
184.9
1.91
2.19
chosen
5D3,a
95.8
1.95
2.14
chosen
5D3,b
95.1
1.95
2.15
Then we did digestion for the chosen bricks, this time we digested 2µg of DNA instead of 500ng. the digestion sites:
promoters
3O1
SpeI+PstI
 
5E1
S+PstI
terminators
4F3
EcoRI+XhoI
 
5D3
EcoRI+X
We incubated these @ 37°C for 25-30mins and inactivated @ 80°C for 20mins. Then we did a PCR purification for the digested bricks as well as the chloramphenicol backbones. Finally, we ran a nanodrop. The results, hower, are really weird, indicating there is barely any DNA. This means that the reason that the transformations of D2 and D3 all failed this week is because we dont have any backbones to ligate!
The problems happens at the purification step!
29/07 – 04/08
Week 5: More colony PCR
Monday morning we did robust colony PCR for the 27 colonies, the gel results were still the same as the colony PCR we did last week.
We also performed a site specific mutation for EBF plasmid of Artemisia Annua using the mutagenesis kit. With this we transformed the cells and plated them out on Kanamycin plates, this apparently failed.
We tried the mutagenesis kit again, this time using a hotstart robust PCR. This also failed.
As all the mutations failed so far we changed and will start using megawhop to mutate the EBFaa.
After the megawhop PCR we performed PCR product purification and a nanodrop:
ng/ul
280/260
280/230
Megaprimer
911.9
1.90
2.27
We also electroporated EBFaa into Rosetta strain and plated them out. We did this to check if the problem with transforming could be because there is a problem with the strain of E. coli that we otherwise use. This transformation was succesful.
We removed all the backbones, 3O1, 5E1 and 4F3, 5D3, with additional dephosphorylation step after inactivation. We purified these digestion products. These are the nanodrop results:
ng/µl
260/280
260/230
5D3
36.7
1.98
2.33
301
4.1
2.61
0.14
5,00E+001
41.7
1.86
1.46
A
13.5
2.13
0.94
C
14.7
2.04
1.37
T
16.5
2.10
0.59
K
16.9
2.13
1.05
We ligated EBF into chloramphenicol and behind 5E1 (this time we used a 3X higher amount of EBF than backbone to ensure a better/correct ligation), thus making D1 & D5. We then electroporated them into electrocompetent cells and plated them out. These transformations are successful. Then we did a colony PCR for 30 randomly selected colonies from D1 and D5 (15 each). We ran a gel after.
The gel only has bands around 500bp, the possible reason may be that the primers can aneal to genomic DNA, so we decided to inoculate the transformed cells, do a miniprep, then digest and run the gel to see if there are any correct bands.
We extracted the plasmid of 3O1 by miniprep, purified it, digested it with S+P sites and dephosphorylated it. These are the nanodrop results:
ng/ul
280/260
280/230
3O1 (a)
119.3
1.85
1.96
3O1 (b)
116.6
1.83
1.96
3O1 (c)
117
1.87
2.18
We miniprepped overnight inoculated LB tubes with 5 colonies of D1 and 5 of D5.
Nanodrop results after MiniPrep:
ng/ul
280/260
280/230
D1
 
 
 
1
81.3
1.94
2.08
2
77.9
1.90
1.90
3
84.3
1.89
1.93
4
84.3
1.89
1.93
5
105.9
1.87
1.74
D5
 
 
 
1
109.8
1.87
1.99
2
64.7
1.78
1.40
3
85.3
1.88
1.73
4
98.6
1.87
1.75
5
120.5
1.86
1.64
Ran gel with D1 1-5, D5 1-5, digested inserts and insert stock (EBF1 and EBF2). The vector seems OK, but we have no bands were the insert should be in all lanes. So we decided to make more insert and did a high fidelity PCR on EBF stock (EBF1 and EBF2).
We also did PCR for 6 other colonies with primer VF2 (the primer to amplify the insert) and reverse primer of EBF synthase gene.
05/08 – 11/08
Week 6: Inoculations
We did PCR purification for the PCR we ran for EBF1 and EBF2, nanodrop them:
ng/µl
260/280
260/230
EBF1
116.9
1.91
1.79
EBF2
238.3
1.91
1.89
Then we ran the gel for unpurified and purified products, the results are not good. So we decided to do PCR for the original gene (EBF3 and EBF4) and ran the gel. This finally corresponds with the right bands. Afterwards we loaded all the PCR products to a new gel to purify the target gene, and cut the correct bands out. After gel extraction these were the nanodrop results:
products
ng/µl
260/280
260/230
EBF3
99.1
1.95
2.04
EBF4
104.2
2.00
2.15
To confirm that we would still have EBF3 and EBF4 after extraction and purification, we did another gel electrophoresis. The results were positive.
We digested 2µg of EBF3 with XhoI+PstI restriction enzymes and 1µg of EBF3 with EcoRI+PstI restriction enzymes, purified it, nanodrop:
ng/µl
260/280
260/230
X+P
34.6
2.01
2.27
E+P
15.6
2.44
2.01
We ran a gel of both and had positive results.
We used a crystal violet (1%) gel for the megaprimers to get rid of primers and other inpurities. We used crystal violet in the gel to avoid DNA-damage due to UV. Then we cut the correct band out.  After gel extraction these were the nanodrop results:
products
ng/µl
260/280
260/230
megaprimer
119.2
1.92
1.88
To confirm that we would still have the megaprimer we did another gel electrophoresis. We noticed 3 bands for megaprimer sample in the gel, thus we cut the correct band of megaprimer (1200bp), and extract the megaprimer from the gel and nanodrop it:
260/280
260/230
5.6ng/µl
2.83
1.07
We did the PCR with megaprimer for the mutagenesis of EBF from Artemisia Annua with Phire polymerase. We ran a gel for the 4 PCR products, but with negative results. We did the PCR over again, this time with the following set-up :
Tube1: EBFaa (the original EBF)
Tube2: EBFaae (originated from the transformed EBF)
Tube3: control, without template
Tube4: EBFaae but do not put into PCR machine
We ran the crystal violet gel for megaprimer, and did a gel extraction:
EBFaa: 3.4ng/µl
EBFaae: 0.6ng/µl
these results are really bad.
We repeated the second PCR with the megaprimer we made last friday with Phire polymerase.
We phosphorylated primer 13 and 14 with kinase, then performed blunt end PCR for EBFaa with Phire polymerase.
We started the ligation of EBF into the Chloramphenicol backbone (D6), as well as inserting EBF after 3O1 and 5E1 promotors (D7 & D8). For the ligation of D6, the EBF amount was twice the amount of the backbone; whereas for 3O1 & 5E1, the EBF amount is 3 times higher than the vector. Then we transformed the Inoue cells with the D6, D7 and D8 plasmids. We plated these out and incubated at 37°C overnight.
Our transformations were successful, so we did a colony PCR (21 colonies) to confirm the correct insert. The gel results were still around 500bp however, so we decided to inoculate the 21 colonies, do the plasmid extraction tomorrow, confirm the correct insert by digesting the plasmid tomorrow.
In the morning we did plasmid extraction for the 21 inoculations and ran the nanodrop:
ng/µl
260/280
260/230
D6 1
248
1.99
1.67
D6 2
83
2.09
1.82
D6 3
144
1.82
1.46
D7 4
123.8
1.99
1.96
D7 5
140.6
1.87
1.48
D7 6
152.3
2.01
1.55
D8 7
83.2
1.86
1.20
D8 8
98.7
2.04
1.88
D8 9
137.8
2.01
1.80
D8 10
307.9
2.06
1.97
D8 11
70.9
1.90
1.34
D8 12
197.3
1.96
1.84
D8 13
249.4
1.96
1.79
D8 14
91.1
2.05
1.70
D8 15
123
1.96
1.56
D8 16
142.4
1.96
1.39
D8 17
159.4
1.88
1.53
D8 18
240
1.85
1.38
D8 19
165.3
1.9
1.28
D8 20
226
1.90
1.65
D8 21
138
2.17
2.14
We digested the 21 samples' plasmids with E+P sites and ran the gel, we got band in between 1500 and 2000 bp, close to 2000 though.
12/08 – 17/08
Week 7: Keep on trying to make those devices...
We did a PCR for the EBF synthetase from Artemisia annua as well as the PCR for the use of type IIS restriction enzyme. Then we retried the ligation of EBF with Chloramphenicol backbone, 3O1 and 5E1, this time we ligate them over night instead of 20mins.
The confirmation gel for EBF synthase was negative, thus we tried PCR with Phire polymerase, the weird thing is there’s no band either with this polymerase.
Meanwhile we tested the electrocompetent cells we made before with different concentrations of pUC19 vector, plate out, and the results indicate a high efficiency, which means that the elctrocompetent cells we made were very good. With the megaprimer PCR products we made last week, we did DpnI treatment, then electroporated them after the purification step, sadly, there’s no growth on the plate.
With the growth of the colonies of the ligation products, we performed colony PCR (Robust) for them, the gel finally indicated that we have successfully ligated EBF into the chloramphenicol backbone.(lane 2, 3 and 4 were positive results)
In order to confirm whether or not we still have EBFaa after the plasmid extraction, we digested both EBFaa (from the paper we received), EBFaae (from plasmid extraction). But the gel results were not satisfying.
Moreover, we did Phire PCR for EBFaa, ran the gel, that indicated that we have EBF synthase after PCR.
After the digestion of EBF with X+P, we did purification, ran nanodrop:
ng/µL
260/280
260/230
EBFSaa1
123,8
1,89
2,16
EBFSaa2
14,2
2,1
0,64
EBF3 X+P
15,6
2,11
1,64
EBF4 X+P
19,5
2,07
2,03
We also digested EBFaa with E+P (here we forgot that there’s a E site in the EBFaa gene, even though we got successful ligation of it in the C backbone later, but the work was in vain), ligated it into Chloramphenicol backbone, in parallel, the ligation of digested EBF3 with 3O1 and 5E1 were done, the ligation was at 16 °C, overweekend.
The EBF+C (colony 2, 3, 4) were also inoculated at 25degrees overweekend.
19/08 – 23/08
Week 8: Nanodrop the base.
We started our week with the plasmid of EBF+C 2,3,4 and EBFaa. nanodrop:
ng/µL
260/280
260/230
EBF+C 2,1
25,4
1,8
1,07
EBF+C 2,2
26,8
1,77
0,85
EBF+C 3,1
25,6
1,72
0,95
EBF+C 3,2
32,6
1,78
1,04
EBF+C 4,1
32,5
1,82
1,11
EBF+C 4,2
30,3
1,78
0,88
EBFaa3
44,7
1,85
1,43
EBFaa4
61,4
1,82
1,81
then digest 500ng of EBF+C2,3,4 with E+P sites ran the gel, the results were all positive!
But the gel result of typeII restriction enzyme PCR of EBFaa indicated a failure.
Then we transformed the over-weekend ligation products, plate out. The transformation was successful, but the gel of Robust colony PCR for the randomly selected 12 colonies showed a negative results. But still, we inoculated 24 random samples, did plasmid extraction, nanodrop:
ng/µl
260/280
260/230
L1 1
58.1
1.84
1.15
L1 2
127.7
1.89
1.50
L1 3
80.4
1.75
0.93
L1 4
28.2
2.03
1.79
L1 5
83.2
1.80
1.03
L1 6
71.4
1.80
1.14
L1 7
37.8
1.84
1.39
L1 8
61.5
1.80
1.12
L2 1
26.6
1.86
0.81
L2 2
10.1
2.74
3.55
L2 3
35.6
1.79
0.80
L2 4
67.0
1.74
0.81
L2 5
63.1
1.76
0.80
L2 6
55.1
1.75
0.81
L2 7
69.0
1.68
0.67
L2 8
69.1
1.66
0.68
L3 1
60.1
1.69
0.70
L3 2
72.0
1.72
0.69
L3 3
67.1
1.69
0.66
L3 4
65.6
1.74
0.80
L3 5
62.9
1.73
0.71
L3 6
59.5
1.81
0.93
L3 7
66.3
1.76
0.75
L3 8
48.6
1.80
0.77
Then digested them with E+P sites, ran the gel, the results were good (see gel), we got several positive results, so we made the good ones into stock.
We also prepared samples of EBF+C for sequencing.
Then we retried type IIS-tailed PCR with EBFaa plasmid, the gel for the type IIS-tailed PCR products, the results were weird, we got a band at 5kb instead of 7kb which was the band we wanted.
We digested EBFSaa with X+P, EBF4 with E+S, 5E1+EBF with E+S, 4F3 with X+P, then dephosphorylated the digested 4F3, then did PCR purification for all the digested products and nanodropped them:
ng/µl
260/280
260/230
5E1+EBF
26.2
1.68
0.68
4F3
20.1
1.83
0.65
EBFSaa
35.5
1.71
0.68
EBF4
30.2
1.82
0.80
Then ran the gel to confirm, the results were positive.
We also tried high fidelity PCR for the type IIS restriction enzyme. (instead of using Phire)
Then we did ligation: EBF4(E+S) + 4F3(E+X) and EBFaa(X+P) + 4F3 (X+P).
26/08 – 01/09
Week 9: Mutagenesis
We tried type IIS PCR again with different combinations of primers, aim at finding out if it's something wrong with the primers.
Then transformed the ligation products from last Friday, plate out. The transformation was successful. And the colony PCR of it has no positive results. But we still inoculated 20 samples, after the miniprep,  we digest with E+P for 1 to 10 samples, X+P for 11 to 20 samples, ran the gel, got several positive results
The good colonies are:
for EBF4+4F3: colony 7,8,9
for EBFaa+C: colony 1, 12, 15, 16, 17, 19
then we made them stock, and inoculate them, the nanodrop for the miniprep of the good samples are:
ng/µl
260/280
260/230
7
47.4
1.75
1.02
8
67.0
1.90
1.38
9
56.7
1.88
1.20
11
56.5
1.87
1.08
12
38.1
1.97
1.92
15
82.6
1.94
1.93
16
80.8
1.94
2.02
17
48.3
1.95
1.55
19
120.7
1.88
1.66
From the successful colonies, we performed Robust PCR for EBFaa+C (#19colony) with template concentration range of 0.01, 0.1, 1 and 120.7 ng/µl to see which concentration works the best, and it worked equally well for all of them according to the gel.
We also digested EBF4+4F3 (#8 colony) with X+P, ran the gel to seperate the insert with the backbone. This insert will be used to ligate behind the promoters. However, we also digested EBF+4F3 (#7,8,9 colonies) with X+P, ran the gel, results showed that we dont have EBF in front the double terminator yet. In the end of this week, we did a huge PCR with 46 samples with different combinations of primers in order to see where went wrong in the previous mutagenesis PCR, the template is EBFaa in the new C backbone. Then we purified the mutagenesis PCR products, DpnI treatment, cut with BpiI restriction enzyme to cut the tail created by the primer on the linear plasmid, after ligated the linear plasmid to itself we transformed them.
02/09 – 08/09
Week 10: Some good results: finally!
Monday embraced us with some successful grow colonies from the transformation we did last week, so we did colony PCR for randomly selected 16 colonies, ran the gel, the gel gave positive results for all the 16 colonies! so we inoculated all the 16 colonies!
Then we extracted the plasmid of inoculations, ran nanodrop:
ng/µl
260/280
260/230
mEBFaa1
14.1
1.81
0.92
mEBFaa2
62.3
1.79
1.30
mEBFaa3
75.0
1.76
1.08
mEBFaa4
26.8
1.77
0.95
mEBFaa5
31.8
1.58
0.62
mEBFaa6
158.4
1.69
0.76
mEBFaa7
31.9
1.64
0.75
mEBFaa8
7.5
1.97
1.61
mEBFaa9
11.0
2.15
2.14
mEBFaa10
15.2
1.97
2.05
mEBFaa11
26.7
2.00
2.29
mEBFaa12
38.6
2.00
2.29
mEBFaa13
29.7
1.97
2.03
mEBFaa14
3.2
2.33
1.19
mEBFaa15
14.6
1.94
1.27
mEBFaa16
6.0
2.08
2.04
With the extracted plasmid, we digested them with E+P, ran the gel, the results were good! (for mEBFaa1,2,3,4,5,6,7,9,11,16, especially from 1 to 7).
.
Therefore, we prepared the samples for sequence. Thus we used almost all the plasmid; we inoculated and extracted them again:
ng/µl
260/280
260/230
mEBFaa1
125.3
1.92
1.74
mEBFaa2
137.4
1.90
1.98
mEBFaa3
144.6
1.91
2.15
mEBFaa4
114.7
1.92
1.96
mEBFaa5
137.5
1.90
2.09
mEBFaa6
153.7
1.90
2.21
mEBFaa7
137.5
1.92
2.13
From this point, we could start to build up the construct. We digested 2µg of mEBFaa6 with E+S, did gel extraction of the insert:
17.4ng/µl
260/280:1.93
260/230:1.00
Then ligate it in front of 4F3 overnight at 16°C. In parallel, we digested mEBFaa6 with X+P sites, then gel extraction:
28.5ng/µl
260/280:1.89
260/230:0.95
The following step is the transformation of the ligation products, there were some growth of colonies on the plate of the mEBFaa6+4F3 as well as on the plate of mEBFaa6+5E1, so we did colony PCR, however the gel indicated not good results. But we still inoculate 8 samples anyway to do miniprep then digest.
09/09 – 15/09
Week 11: Building up the construct
This week, we started with plasmid extraction of the culture we inoculated last weekend, run nanodrop:
ng/µl
260/280
260/230
4F3 1
118.2
1.93
2.18
4F3 2
147.9
1.93
2.21
5E1 1
135.9
1.94
2.21
5E1 2
140.1
1.92
2.23
mEBFaa6+4F3 1
107.8
1.92
2.23
mEBFaa6+4F3 2
90.4
1.93
2.24
mEBFaa6+4F3 3
98.3
1.92
2.26
mEBFaa6+4F3 4
196.0
1.92
2.30
mEBFaa6+4F3 5
90.8
1.94
2.23
mEBFaa6+4F3 6
115.9
1.93
2.27
mEBFaa6+4F3 7
146.0
1.93
2.29
mEBFaa6+4F3 8
177.5
1.92
2.28
mEBFaa6+5E1 1
79.8
1.97
2.28
mEBFaa6+5E1 2
98.7
1.95
2.29
mEBFaa6+5E1 3
91.2
1.96
2.32
mEBFaa6+5E1 4
78.0
1.95
2.25
mEBFaa6+5E1 5
80.8
1.91
2.25
mEBFaa6+5E1 6
76.1
1.97
2.32
mEBFaa6+5E1 7
62.1
1.96
2.21
mEBFaa6+5E1 8
81.3
1.96
2.29
This followed up by the digestion of  4F3 with E+X sites and  5E1 with S+P sites, mEBFaa6+4F3 1 to 8 with X+P sites, mEBFaa6+5E1 1 to 8 with E+S sites. However the gel results indicated that the ligation failed.
But, we did gel extraction of the digested 4F3 and 5E1 in order to obtain more digested backbones with terminator and promoter, with which, we could ligated 5E1 (S+P) with mEBFaa6(X+P) and 4F3 (E+X) with mEBFaa6(E+S), ran nanodrop:
gel extracted 5E1: 24.6ng/ul
260/280: 1.93
260/230: 0.86
gel extracted 4F3: 26.5ng/ul
260/280: 1.90
260/230: 0.98
n order to increase the right percentage, we first dephosphorylated the digested 5E1 and 4F3, then ligate mEBFaa into them, together with the ligation products we did electroporation, and plate out. There were only a few colonies on the plates, we did colony PCR for them, and the gel indicated that we have mEBFaa6 in front of 4F3. Thus we inoculated that colony.
Lane3 is EBFaa in front of 4F3.
Meanwhile, we digested the ligation products we did last week (the ones we almost forgot), the gel indicated that we have EBF + 4F3 and EBF+5E1.
We inoculated them as well. The next step was the plasmid extraction of them, nanodrop them:
ng/µl
260/280
260/230
16J3+1C1 1
58.1
1.91
2.05
16J3+1C1 5
82.7
1.89
2.21
16J3+1C1 6
90.2
1.91
2.18
EBF+5E1 1
115.6
1.90
2.22
EBF+5E1 2
130.0
1.90
2.23
EBF+5E1 3
96.8
1.91
2.17
EBF+5E1 4
95.3
1.92
2.23
EBF+5E1 5
118.8
1.91
2.17
EBF+4F3 1
79.3
1.93
2.18
EBF+4F3 3
96.0
1.91
2.19
EBF+4F3 4
78.8
1.91
2.19
EBF+4F3 5
98.5
1.92
2.19
mEBFaa+4F3 1
168.3
1.92
2.31
mEBFaa+4F3 2
175.6
1.90
2.33
We digested EBF+5E1 3and 4 with E+S, EBF+4F3 4&5 with X+P, mEBFaa+4F3 with X+P, ran the gel of it, the weird thing is there’s no positive results for EBF+4F3 and EBF+5E1, due to the tight time schedule, we decided not to continue with EBF.
In the case of mEBFaa+4F3, we found 3 more colonies in the plate, so we inoculated them and did plasmid extraction:
ng/µl
260/280
260/230
mEBFaa+4F3 3
129.7
1.96
2.75
mEBFaa+4F3 4
121.7
2.01
2.89
mEBFaa+4F3 5
273.3
1.95
2.59
After digesting them with X+P, ran the gel, results showed mEBFaa+4F3 2 & 5 are positive.
Then we did gel extraction:
ng/µl
260/280
260/230
mEBFaa+4F3 2
8.0
1.98
1.02
mEBFaa+4F3 5
6.3
1.72
0.20
So we ligated them after 5E1 at 16°C overnight. And transformed them.
16/09 – 22/09
Week 12: Continue gBlocks
There's no colony growing on the plates we transformed last week.
So we retried the ligation. We first digest 2µg of mEBFaa+4F3 2&5 with X+P as well as 5E1 with S+P, then did gel extraction of them:
ng/µl
260/280
260/230
mEBFaa+4F3 2
9.0
1.62
0.54
mEBFaa+4F3 5
10.5
1.85
0.59
5,00E+001
21.7
1.88
0.83
After the ligation, we purified the ligation products, electrotransformed and plated out.
There were some colonies growing, so we did colony PCR, however the gel results were not good, but we still inoculated 20 samples, among which were some slow growing colonies, which were not appeared when we did the colony PCR, but appeared in the afternoon when we inoculated them.
In this week, the gBlocks assembly kit finally arrived, so we started it.
We assembled the parts together, then did Phire PCR to amplify the assembled parts with different template concentration (original conc, 1ng/µl, 0.1ng/µl). The gel conformation showed that we got the correct bands for gEBFaa and gTetR, not gMS though.
So we digested them with E+P for 40mins, followed up by PCR purification:
ng/µl
260/280
260/230
gEBFaa 1
41.1
1.92
1.23
gEBFaa 2
23.5
1.95
1.17
gEBFaa 3
14.6
2.12
0.93
gTetR 1
39.5
1.86
1.20
gTetR 2
28.8
1.97
0.37
gTetR 3
16.4
1.92
0.62
gMS 1
34.6
1.86
0.72
gMS 2
28.8
1.89
0.25
gMS 3
15.4
1.97
0.97
4F3 (E+P)
17.2
1.89
0.51
With which, we ligated them into the backbones with different vector: insert ratios (1:1, 1:3, 1:0) for time range from 20mins to 2 hours to overnight. We transformed the ligation products of 20mins. At the same night, during the waiting time of recovery, we had a movie night!
For the ligation of mEBFaa6 + 4F3 behind the 5E1, we got some colonies, so we minipreped 24 samples, then digested them with EcoRI and BamHI (cut in the middle of EBFaa), the gel showed that 1 sample is positive!
And this one is a slow growing colony we picked yesterday. Make stock of it.
In the case of the gBlock story, there were too much colonies growing on the plates, even the negative control, indicated that the digestion of 4F3 is not good, 40mins is too short. But we inoculated 6 colonies. The digestion after miniprep of these colonies were negative as we expected.
We also transformed the overnight ligated gMS (because it was ligated to the pure backbone even though the gel results were incorrect after assembly). There were a few colonies of gMS, but the digestion results showed a negative outcome. Mean while, we sent the 5E1+mEBFaa6+4F3 for sequence. Good news, the sequence of mEBFaa6+4F3 is correct!
To continue the gBlock story, we digested 4F3 with E+P for 2hours, PCR purification:
ng/µl
260/280
260/230
4F3 1
16.7
1.96
2.05
4F3 2
17.2
1.84
2.32
After the purification of the ligation products, we transformed them along with the full construct and the mEBFaa6+4F3 into Rossetta strain. Sadly, there were still way too much colonies on the plates including the negative control.
There were still too many colonies including the negative control. Thus we decided to first make the negative control reliable, which is pure enough backbone that will not lead to false positive. So we digested 4F3 with E and P separately for 2 hours, after which, we tool a small portion of the digestion to ran the gel to confirm the linerization of the plasmids, then digest them with the second restriction enzyme, after the purification, went to the ligation of only the vectors to see if they still give false positive. Transformed this ligation products.
Meanwhile, we digested 19K4 (because the insert is big enough, so we can do gel extraction to get the pure backbone) with E+P for 2 hours. Then did gel extraction to obtain the backbone, ligate gBlock products into the backbone overnight.
23/09 – 29/09
Week 13: The final weeks
We did Phire PCR again with primer 33 and 36 for gMS, the gel still showed band at around 1500bp, so not good result.
We then digested the pure pSB1C3 backbones we got from iGEM, then ligate the gEBF and gTetR into it. The ligation products were transformed. No colonies.
Since we kind of ran out of the pure backbone, so we did Phire PCR to amplify the pure Chloramphenecol backbone. We retried the ligation and transformed them.
Meanwhile, we prepared the epps containing medium of variations of salicylate and Mg2+ for smell test. And the new bricks cells were inoculated to make the growth curve.
We PCR purified the gEBFaa, gTetR and pSB1C3, nanodrop them:
ng/µl
260/280
260/230
gEBFaa
12.8
6.48
1.09
gTetR
13.9
6.65
0.93
pSB1C3
261.7
1.87
2.14
Followed by the digestion with E+P for 2 hours. Then purified them, nanodrop again:
gEBFaa: 1.2 ng/µl
gTetR:  1.9 ng/µl
pSB1C3: 26.3 ng/µl
Ligated the inserts into the backbones, transform them.
There were colonies on the plates, so we did colony PCR for 24 samples, seemed the gel results indicated that the TetR brick was correct. We inoculated 30 samples to do miniprep tomorrow.
In parallel, we digested mEBFaa6+4F3 plasmid with E+P+NcoI, ran the gel, gel extraction to get the mEBFaa6+4F3 and get rid of the backbone, then dephosphorylated the cut opened strong promoter 3O1 and medilum promoter 5E1 vectors, ligated them with mEBFaa6+4F3 with different ratios, transformed them.
We first did the Phire PCR to amplify the gEBFaa and gTetR.
After the miniprep of the 30 samples, we digested them, and the gel indicated that we got one good gEBFaa colony!
Lane 5 is the correct one
Then we inoculated this colony.
We also transformed the ligation products of gBlocks and homemade device.
After the miniprep, we digested gEBFaa5 with E+BamHI, the rest digested with E+P.
Purified the Phire PCR products, digested them with E+P as well, also dephosphorelated the pSB1C3 backbone. Then the ligated the digested gEBFaa and gTetR into the backbone.
And there were colonies on the plate of the transformation, so we did colony PCR for randomly selected 30 colonies. The gel indicated we have successfully ligated the promoters in front of the mEBFaa6+4F3.
lane 2, 3, 5, 6, 8 were good.
We inoculated them with some other random colonies over night.
After miniprep of the inoculation samples, we did all different combinations of restriction enzymes to digest the plasmids we thought were the correct ones, and which confirmed that we have the final construct of gEBFaa.
The digestion gel confirmed we have the full construct of 5E1+LacI+mEBFaa+4F3, there's just an extra EcoRI site very close to PstI site, which will not influence the gene. Moreover, another digestion indicated that we have the promoter+mEBFaa6+4F3 (with both strong and medium promoters, name the final construts with HD12 and HD13).
We then transformed the gEBFaa5 and HD12 and HD13 into expression strain (JM110 and BL21) cells.
14/09 – 04/10
Week 14: Characterization
We inoculated the transformed expression strains (BE21) for the use of making growth curve and protein extraction.  The growth curve experiment was performed under 37°C, the samples order was:
gEBFaa5
HD12
HD13
gEBFaa5 + 5mM Mg2+
HD12 + 5mM Mg2+
HD13 + 5mM Mg2+
control: BL21 strain
Meanwhile, the protein expression was performed in different conditions as well:
37°C:
gEBFaa5 + 5mM Mg2+
HD12 + 5mM Mg2+
HD13 + 5mM Mg2+
25°C
gEBFaa5 + 5mM Mg2+
HD12 + 5mM Mg2+
HD13 + 5mM Mg2+
With the extracted proteins, we ran the SDS-PAGE.
Moreover, we tested our HD12 and gEBFaa5 brick with aphids, we put an EBF-producing bacterium plate on the left, a blank on the right, aphids are placed on the leaf in the middle. The first results seem positive, as we can see them moving from left to right.
We also tried another aphid experiment with the gEBFaa5 construct. This time we connected two empty leaves which were on EBF-producing bacterium plate and control plate with the leaf in the middle, therefore the aphids can move onto the other leaves. There’s no significant difference of the amount of aphids on the two leaves. This may due to the in this construct, there was lac operator after the promoter, the lacI in the bacteria may interfere the production of EBF.
In addition, we also tried the aphids without leaf. We put 30 aphids in the middle, and on the left side sat a leaf with 10micro liter of EBF-producing bacteria and on the right side a normal leaf. After 2 hours we counted the number of aphids on the leaf, there were 4 aphids on the leaf where the EBF concentration should be higher and 6 aphids on the control leaf, the rest of aphids just walked randomly in the big petri dish.
14/10 – 27/10
Week 15 & 16: More quantitative characterization
Right after the regional jamboree, we immediately started more quantitative characterizations for our EBF and MeS producing bricks. In the case of EBF synthase producing brick, we redid the in vivo test with aphids. It was conducted with 3 biological repeats for both control (BL21 wild type) and the brick (BBa_K1060009). We selected sweet pepper leaves with similar size infested with aphids, put them in big glass container with sufficient natural ventilation. The EBF producing bacteria and the control strain were only introduced one hour later when the aphids were settled down on the leaves. Subsequently, we counted the amount of aphids that stayed on top of the leaves as well as the number of aphids that was moving every half an hour. We observed a trend of more aphids scattering away from the leave in the EBF compared to the control setup. A statistical difference was observed in the percentage of aphids moving in the EBF compared to the controle group. We retried the SDS-PAGE for EBF brick, this time we compared control strain, DH5alpha + BBa_K1060009 and BL21 + BBa_K1060003. As a result, we observed an extra band in the pattern of the EBF producing strains, in combination with our growth curve, we think that the reason of slow growth in the transformed DH5alpha strain may due to the activity of EBF synthase. For more details please visit our project page. Moreover, we performed gas chromatography - mass spectroscopy (GC-MS) for both BBa_K1060009 and BBa_K1060003. In the situation of EBF, we could not detect the presence of it, this may due to either the extraction step is not completely right, or the amount of EBF produced was too low to be detected, or EBF was evaporated during the shaking cultivation. However, in the case of MeS, we detected it if we add Sodium Salicylate as supplement.
Retrieved from " http://2013.igem.org/Team:KU_Leuven/Journal/EBF/wetlab "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
