"
Page
Discussion
View source
History
teams
Log in
 
Team:StanfordBrownSpelman/Material Waterproofing
From 2014.igem.org
Stanford–Brown–Spelman iGEM 2014 — Amberless Hell Cell
Material Waterproofing
Images ● Methods ● Results ● Alternate Approach ● References ● BioBricks
Primary approach: Paper wasp protein
While cellulose-based biomaterials have promising applications in aeronautics and various other fields due to their lightweight, biodegradable nature, they risk structural failure if they absorb too much water. This poses a problem for anyone who wishes to fly our UAV on a rainy day. Fortunately, nature has provided us with a potential solution. Paper wasps of the genus Polistes are well known for their ability to collect cellulose from the plants around them, mix it with their saliva, and use the resulting cement to construct nests with paper-like properties.
Polistes dominula, also known as the European paper wasp. The nest paper is a durable mixture of saliva and cellulose pulp.
The most significant property of the wasp-produced paper is that it is hydrophobic and therefore waterproof. Research has shown that the nest paper is composed primarily of cellulose, but coated with a protein-rich oral secretion [1]. Until now, scientific knowledge of this protein coating was limited mainly to total amino acid composition of all the proteins in the paper. To gain more insight into the specific proteins that may exist in wasp nest paper, we collected Polistes dominula, an invasive European species of paper wasp, and sequenced the proteins found in their nests using peptide mass fingerprinting. We believe there may be a single protein in wasp saliva that is chiefly responsible for the hydrophobic nature of their nests.
Our ultimate goal is to identify the gene that codes for this wasp waterproofing protein and transform into Saccharomyces cerevisiae so that we can produce an inherently biomimetic solution to shield lightweight bacterial cellulose (BC) or bacterial cellulose acetate (BCOAc) films from water in the environment. This project is particularly exciting because of its potential for discovery; never before have the proteins in wasp saliva been identified or applied as functional biomaterials.
Photo Reel
A polyacrylamide gel containing proteins from the paper wasp nests we collected. We excised the dominant bands for protein analysis.
Frozen wasp paper sample collected during the summer from an active nest.
A paper wasp on the nest we cultivated on the roof of our lab at the NASA Ames Research Center. Our personal wasp nest provided us with the freshest possible samples.
Protein samples extracted from three paper wasp nests collected with Dave Kavanaugh, entomologist from the California Academy of Sciences.
Jotthe Kannappan grinds a frozen wasp in the process of extracting RNA.
A macroscopic photo of one of our paper wasps used for species identification.
Paper wasp actively working on building its nest.
Approach & Methods
Our approach to identifying the Polistes dominula waterproofing protein relied on the acquisition of nest samples and of individual wasps. Our plan, detailed in the graphic below, was to extract protein from the nest samples and use analytical techniques such as peptide mass fingerprinting to gather information on the proteins present, and then to use this information to identify candidate Polistes dominula genes for cloning and testing in model organisms.
Figure 1. Schematic for wasp nest protein identification via peptide mass fingerprinting.
We extracted total protein from the nest samples using a plant protein extraction kit . We ran the proteins on two polyacrylamide gels, one with a ten minute 70ºC heat denaturation step and the other without. We then excised all dominant individual bands and sent them to Dr. Gary Wessel’s lab at Brown University for peptide mass fingerprinting.
Figure 2. A small sample of a Polistes dominula nest waiting to be ground with a mortar and pestle for protein extraction.
Peptide mass fingerprinting is an analytical protein identification technique in which the protein of interest is cleaved into small fragments via site-specific proteolytic enzymes such as trypsin. The molecular masses of these fragments can be measured accurately through mass spectrometry. Once these masses are known, they can be compared with computer predictions based on a reference genome or transcriptome to see which of the reference’s proteins are most abundant.
Figure 3. A short time lapse video documenting the running of a wasp protein gel.
Our initial plan was to extract RNA from female nest-building wasps so we could purify their messenger RNA, generate a complementary DNA library, and get the library sequenced for use as a reference transcriptome. Fortunately for us, the Polistes dominula genome was published shortly after we began our project [2], saving us the trouble and extreme expense of sequencing wasp RNA to create a reference transcriptome ourselves. The genome was used as a reference for peptide mass fingerprinting, we saved our RNA extracts for eventual RT-PCR amplification, and the project moved onwards. We truly live in an exciting time for genetic engineering!
Once we obtained the mass fingerprinting data, we were faced with a long list of uncharacterized peptide fragments with hits in the Polistes dominula genome. We used the NCBI Position-Specific Iterated Basic Local Alignment Search Tool, or PSI-BLAST, to search online databases for proteins with similar sequences and domains. This gave us hints towards the possible functions of these uncharacterized proteins.
We then narrowed our list down to six proteins with favorable PSI-BLAST hits and obtained the DNA sequences for these genes from the genome. Three of these genes were codon-optimized and synthesized for Escherichia coli expression in the pF1A T7 Flexi® vector, while the other three were amplified from wasp RNA via RT-PCR for S. cerevisiae expression in the pYES2.1/V5-His-TOPO® vector.
Material Waterproofing Project Notebook: Complete documentation of our methods, protocols, and scientific processes can be found in the linked document.
Results
We collected two wasp nests in the field: one new nest that we saw wasps actively working on, and an older, abandoned nest that Dr. Kavanaugh estimated to be about one year old. After extracting total protein from the nests, we ran the protein samples on two polyacrylamide gels – one with a ten minute 70ºC heat denaturation step and the other without – and obtained the images below.
Figure 4. Wasp nest protein extracts run on a polyacrylamide gel after a ten minute 70ºC denaturation step.
Figure 5. Wasp nest protein extracts run on a polyacrylamide gel without any denaturation step.
The presence of a few dominant bands in the protein samples indicated that there may be a single protein chiefly responsible for the hydrophobicity of wasp nest paper. The relative faintness of bands from the older nest also suggests that the protein may degrade over time, which supports our qualitative observations that the older nest was somewhat less waterproof.
Once we obtained peptide mass fingerprinting data on the bands we excised from the protein gels, we were faced with a list of thirty uncharacterized proteins whose fragments had hits in the Polistes dominula genome. Interestingly, the proteins present in each gel slice did not always have the sizes we expected in comparison to the protein ladder. Why this was the case remains to be seen, but we postulate that proteins of different affinities traveled through the gel at different speeds, may have complexed and traveled more slowly, or have been so abundant that traces were present in multiple bands.
We used the NCBI Position-Specific Iterated Basic Local Alignment Search Tool, or PSI-BLAST, to search online databases for proteins with similar sequences and domains. This gave us hints towards the possible functions of these uncharacterized proteins.
Nearly all of the PSI-BLAST results came back with strong functional predictions for each of our proteins, the majority of which we deemed irrelevant to waterproofing ability. For example, many proteins appeared to be variants of ubiquitin, a small regulatory protein found in virtually all eukaryotic tissues. Others were similar to vitellogenin, an egg yolk precursor protein used to nurture larvae in related insect species such as Vespula vulgaris (also known as the common yellow-jacket) and Formica exsecta (narrow-headed ant). While it was interesting to see the diversity of proteins present in the wasp nests, we had to narrow our scope to test candidate genes in the lab.
Click here to download a .zip file containing the results of our peptide mass fingerprinting, as well as a document listing the thirty proteins present, their first PSI-BLAST hits, predicted sizes, and amino acid sequences.
Out of this list of thirty proteins present in the protein gel slices, we narrowed our list down to six candidate proteins with favorable PSI-BLAST hits. The genes that coded for these candidate proteins are what we ultimately ended up working on in the lab. These six proteins are detailed in the table below.
Figure 6. Wasp candidate genes.
We chose the first two proteins, PdomMRNAr1.2-03231.1 and PdomMRNAr1.2-08705.1 (nicknamed C1 and C2), because the PSI-BLAST predicted they would have chitin-binding domains based on similar proteins in related species. These proteins intrigued us because chitin is a close chemical relative of cellulose, implying that the proteins might also bind to cellulose, and there appeared to be a relatively high proportion of hydrophobic amino acids in the peptide sequences. Furthermore, fungal mycelium is rich in chitin, meaning that if these proteins exhibit chitin-binding activity and hydrophobicity, they might also be used to waterproof chitin.
We chose the next two proteins, PdomMRNAr1.2-10508.1 and PdomMRNAr1.2-04156.1 (nicknamed G1 and G2), because the PSI-BLAST found similarity between these proteins and glucose dehydrogenases characterized in related species. This sparked our interest because it suggested that these proteins could have glucose binding domains, and cellulose is a glucose-based polysaccharide. The predicted sizes of these two proteins also best matched the sizes of the bands we excised from the protein gels.
We chose the last two proteins, PdomMRNAr1.2-02758.1 and PdomMRNAr1.2-10259.1 (nicknamed U1 and U2), because the PSI-BLAST found no significant hits for similar proteins in its entire database. This indicates these two proteins are completely uncharacterized and may be unique to paper wasps, having evolved recently as an adaptation to impart hydrophobicity to their nests.
We obtained the DNA sequences for these genes from the genome. Three of the genes – C1, C2, and U1 – were codon-optimized, synthesized, and ligated into the pF1A T7 Flexi® vector for E. coli expression. U1 was synthesized with a polyhistidine-tag for eventual purification, while C1 and C2 were planned to be purified with a chitin-coated magnetic bead purification kit.The other three genes – G1, G2, and U2 – were amplified from wasp RNA via reverse transcription PCR (RT-PCR) and ligated into the pYES2.1/V5-His-TOPO® vector (which contains a polyhistidine tag) for S. cerevisiae expression and purification.
Figure 7. Results of an RT-PCR performed on Polistes dominula total RNA extract. Primers were designed for G1, G2, U1, and U2. U1 was not successfully amplified, but fortunately was short enough to synthesize.
As seen in the gel image above, G1, G2, and U2 were successfully isolated from wasp total RNA via RT-PCR. Curiously, after transforming E. coli with these three genes to prepare enough plasmid to transform into yeast, we noticed after a series of sequencing orders that the genes amplified from our Californian wasps varied slightly from the Polistes dominula whose genome was sequenced. These variants were often but not always silent point mutations. We also occasionally experienced mutations in all six of our genes when cloning in E. coli.
Click here to download a .zip file containing the DNA and amino acid sequences for all six of our candidate genes, including original genomic predictions, codon-optimized versions, California variant sequences, and bacterial cloning mutations.
Figure 8. Saccharomyces cerevisiae transformants plated on uracil-deficient selective medium. Clockwise from top left: G2, U2, G1, RFP control. The cells were given glucose as a carbon source to repress pYES2.1/V5-His-TOPO® expression and help stimulate growth.
As seen above, G1, G2, and U2 have successfully been transformed into yeast and we are in the process of culturing the yeast in uracil-deficient selective medium with galactose as a carbon source to induce pYES2.1/V5-His-TOPO® expression. Stay tuned for updates on this subset of the project!
At the time of writing, U1 was received relatively late from the synthesis company, and so has not yet been transformed successfully into T7 E. coli for expression and purification. However, C1 and C2 have been transformed and expressed in T7 E. coli. To attempt to purify the proteins from cell lysate, we used the New England Biolabs Chitin Magnetic Beads protein purification kit, which isolates proteins that bind to chitin-coated magnetic beads. After eluting from the magnetic beads, we ran the purified C1 and C2 extracts on a polyacrylamide gel.
Figure 9. C1 and C2 after chitin magnetic bead purification. C2 was successfully purified with chitin magnetic beads, suggesting that it has functional chitin-binding domains as predicted by PSI-BLAST.
This gel image shows that although C1 is absent, C2 was successfully purified with the chitin magnetic beads, suggesting that it exhibits chitin-binding activity. Pending cellulose-binding and waterproofing assays, C2 is a candidate for waterproofing both fungal mycelium and bacterial cellulose. Stay tuned for updates on this subset of the project!
We successfully BioBricked C1, C2, and U1. Check out our BioBricks page for links to the parts in the Parts Registry, where we’ll have the most up-to-date characterization data!
Alternate approach: Wax ester biosynthesis
In addition to the wasp protein waterproofing project, we searched literature for other organisms that would be able to produce a highly hydrophobic substance to waterproof the UAV. We stumbled across Marinobacter hydrocarbonoclasticus, a marine bacterium that synthesizes isoprenoid wax esters for use as storage compounds [3].
Figure 10. A phytol-based wax ester produced by Marinobacter hydrocarbonoclasticus.
Wax esters, such as the example shown above, consist of two very long carbon chains joined by an ester group. These chains impart great hydrophobicity to the entire molecule, indicating that a wax ester coating on a surface could be used to repel water. We decided to repurpose the wax ester synthesis genes in Marinobacter to create a pathway through which Escherichia coli could synthesize the wax esters and use them to waterproof our UAV.
The two enzymes in Marinobacter that are involved in the production of isoprenoid wax esters were wax synthase 1 (WS1) and wax synthase 2 (WS2). In the presence of a fatty alcohol and fatty acyl-CoA (carboxylic acid derivative), either of these two proteins can catalyze the synthesis of isoprenoid wax esters.
The DNA sequences for both WS1 and WS2 were obtained from the NCBI database [4][5]. They were then codon optimized for Escherichia coli using a codon optimizer from the Integrated DNA Technologies (IDT) website and then synthesized. After the genes were received from IDT, a polymerase chain reaction (PCR) was done to amplify the proteins and to ensure that they were present at around 1.4kb. The proteins were then ligated to a promoter and RBS and also into a chloramphenicol vector, then transformed into E. coli. After completing a colony PCR on the transformants, colonies 5, 7, and 11 on the WS2 plate appeared to contain the WS2 gene. Liquid cultures were inoculated with colonies 5, 7, and 11 to create more cells with the gene. After the cultures were grown in the incubator, they were miniprepped in order to extract their plasmid DNA. The DNA was then sent for sequencing with a 15ul reaction volume.
After sequencing the data came back with miniprepped genes that aligned with the original synthesized gene for roughly half the length of the sequence, but with a puzzling truncation that was present in all three plasmid samples:
Figure 11. WS2 sequencing alignment in Geneious 7.
Unfortunately, the wax esters project had to be discontinued due to an error in the synthesis of the WS2 gene. In order to insert the gene into the vector, there are restriction sites that are cleaved by their respective restriction enzymes. When we codon-optimized WS2 for E. coli expression, we accidentally created a PstI cut site at base 720 in the synthesized gene. Because there was a cut site in the middle of the gene, only a fragment of the WS2 gene could ever make its way into the chloramphenicol vector, thereby disabling the protein the gene would have coded for. We also discovered that the fatty acyl-CoA reagent we intended to use for our enzymatic synthesis reaction was exorbitantly expensive (over $60 per milligram).  For this reason, and because it would have been too costly to resynthesize the WS2 gene, the project was discontinued.
References
1. Espelie, Karl E.; Himmelsbach, David S. (1990): Characterization of pedicel, paper, and larval silk from nest of Polistes annularis. J. Chem. Ecol. 16(12): 3467-3477.
2. Toth, et al. (2014): Polistes dominula genome. NCBI . January 10.
3. Holtzapple, E et al. (2007) Biosynthesis of Isoprenoid Wax Ester in Marinobacter hydrocarbonoclasticus DSM 8798: Identification and Characterization of Isoprenoid Coenzyme A Synthetase and Wax Ester Synthases. J. Bacteriology 189: 3804-3812. PMID: 17351040 .
4. Holtzapple, E et al. (2007) WS1 sequence in Marinobacter hydrocarbonoclasticus. NCBI . June 14.
5. Holtzapple, E et al. (2007) WS2 sequence in Marinobacter hydrocarbonoclasticus. NCBI . June 14.
   
   
sbsigem@googlegroups.com
View our Complete Project List
Meet our team!
Built atop Foundation. Content & Development © Stanford–Brown–Spelman iGEM 2014.
Retrieved from " http://2014.igem.org/Team:StanfordBrownSpelman/Material_Waterproofing "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
