"
Page
Discussion
View source
History
teams
Log in
 
Team:StanfordBrownSpelman/Amberless Hell Cell
From 2014.igem.org
Stanford–Brown–Spelman iGEM 2014 — Amberless Hell Cell
Amberless Hell Cell
Introduction ● Methods ● Results ● Conclusions
References ● BioBricks
For an application of synthetic biology where live, genetically-modified cells will come in direct contact with the environment, such as biological sensors on a UAV, two concerns must be addressed. First, the cells need to be resistant to widely-varying conditions that may be present in the environment. Second, in order to address ethical concerns about releasing genetically-modified organisms, it is desirable to reduce horizontal gene transfer from the engineered cells into cells naturally present in the environment. In order to solve both of these issues, and therefore to create an ideal chassis for synthetic biology in environmental applications, we will combine two research goals:
1. The "Hell Cell" project by the 2012 Stanford-Brown iGEM team isolated genes from extremophile bacterial species and inserted them into Escherichia coli, in order to create bacteria that are resistant to extremes in pH, temperature, and moisture. We sought to further characterize, improve, and search for new resistance genes that would help our chassis survive in earth and space applications.
2. The Church Lab at Harvard Medical School in 2013 created a strain of E. coli (C321.ΔA) in which all 321 instances of the UAG ("Amber") stop codon in the E. coli genome had been replaced with the UAA stop codon [1]. Release factor 1, which terminates translation at UAG, was also removed. With this system, the Church group incorporated artificial amino acids with a tRNA that recognizes UAG as its codon.
Figure 1. Our approach to the Amberless Hell Cell. By recoding the UAG stop codon to translate into an amino acid, only cells that have a tRNA with the anticodon AUC will produce the complete protein. In our experiment, we used a tRNA that charges with leucine to translate the UAG codon.
We developed a novel approach for preventing horizontal transfer of engineered genes into the environment by inserting a UAG-leucine tRNA, and using UAG for leucine in all of the inserted, engineered genes. The engineered genes will not have any effect in naturally-occurring bacteria in the environment, which lack the ability to translate UAG into leucine. We call this strategy Codon Security. Our project involves synthesizing UAG-leucine coded versions of the Hell Cell genes and inserting them into the amberless E. coli strain, along with a UAG-leucine (supP) tRNA [2]. This will create a strain of bacteria that is both resilient and safe for environmental applications.
Approach & Methods
We were interested in two avenues of research. The first was test our hypothesis that the amberless chassis would enable us to create an orthogonal protein expression system that would not function properly in other bacteria. By replacing 2-4 leucines with TAG stops in a gene, we make it such that an organism that does not express the supP tRNA, which translates UAG into leucine, produces a truncated product. We have named this novel system Codon Security. The second goal of the project was to apply Codon Security to the Hell Cell genes from our 2012 team in order to limit the horizontal transfer of resistance genes when using synthetic biology in the environment. Applying the Leucine->Stop modifications to the Hell Cell genes and transforming them into amberless cells, we could make the world's first Amberless Hell Cell.
Figure 2. We transformed DH5-alpha and amberless cells with test plasmids containing GFP or aeBlue reporter genes with stop codons and the supP tRNA in order to establish a proof-of-concept for Codon Security.
In order to test our hypothesis for Codon Security, we designed two test plasmids with GFP and aeBlue reporter genes. The BioBrick parts pages for these constructs are BBa_K1499252 and BBa_K1499253 respectively. The GFP generator construct was synthesized by taking the BBa_E0040 sequence, then modifying two leucine codons into TAG stops. Similarly, the aeBlue  construct was modified from iGEM Team Uppsala 2012's chromoprotein BBa_K864401 with three TAG stops coding for leucine. We chose to have the supP tRNA, which translates UAG to leucine, downstream of the terminator for the reporter genes. The supP tRNA sequence was found in Thorbjarnardóttir et al. [2]. However, we decided to include 100bp downstream and upstream of the tRNA coding region in the original organism to include any native promoters and assembly sequences to ensure normal tRNA expression in E. coli. Thus we created a BioBrick of the supP tRNA that included 100bp upstream and downstream of the tRNA gene and validated that it indeed worked in amberless cells: BBa_K1499251 .
We thought of several design strategies for implementing Codon Security. Among them were placing the supP tRNA on a separate plasmid, incorporating supP into the amberless genomic DNA, and placing it under an inducible promoter. We moved forward with a testing construct with the reporter gene + stop codons and tRNA in the same plasmid. The reason was that we wanted to perform the hardest test on system first and then employ more creative solutions only if it failed this robustness test. With the gene and tRNA on the same construct, it is easier for non-amberless cells to express the protein. However, we hypothesized that the supP tRNA would be toxic enough to non-amberless cells, because it would read through UAG stops, that mutations to the tRNA would be selected for. We tested this hypothesis by transforming DH5-alpha and amberless cells with the test plasmids and measuring protein output.
Figure 3. Workflow for Amberless Hell Cell side of the project. We isolated 5 radiation resistance genes from D. radiodurans genomic DNA using PCR. We moved forward with 2 candidates, uvsE and MntH.  We first tested the radiation resistance they conferred in DH5-alpha. Then, we performed mutagenesis PCR at 2-3 leucine codons to create nonsense mutations, tested non-functionality in DH5-alpha, and recovered radiation resistance in amberless by adding the supP tRNA construct.
For the Amberless Hell Cell, we are taking resistance genes found in extremophiles in nature and mutating leucine codons into stop codons before putting the genes into the Amberless chassis. In this way, we produce cells that can withstand stresses, like dessication, pH, and radiation, but cannot transfer those capabilities to other organisms. We focused on radiation resistance genes this summer, drawing both from bricks produced by the 2012 Stanford-Brown iGEM Team and new bricks produced by this year's team.
Click here to go to our project journal, which details our design and engineering process and included descriptions of the protocols we developed and used.
Results Part 1 - Codon Security
We first tested the Codon Security hypothesis with the GFP test plasmid. We cloned the synthesized GFP+tRNA construct into pSB1C3. We initially tried to transform it into DH5-alpha cells, but effeciency was very low. Sequencing of the growing colonies showed interesting deletions or mutations in the tRNA portion of the construct, giving some evidence supporting our hypothesis that the tRNA was able to be expressed but toxic to the cells. We then BioBricked the GFP and tRNA portions separately. The GFP with 2 stop codons but no tRNA (GFP-2S) was stably transformed into DH5-alpha and amberless cells, and as expected, neither cell-type had fluorescent colonies. The tRNA biobrick could not be cloned in DH5-alpha cells; all colonies came back with mutations in the tRNA, further supporting the idea that non-amberless cells cannot tolerate the supP tRNA. We could only biobrick the tRNA in the Amberless chassis.
We were then able to generate amberless clones that had the full sequence-verified GFP-2S+tRNA construct in pSB1C3. These were significantly fluorescent, compared to the amberless with just the GFP-2S that had no measurable fluorescence. This demonstrated that the supP tRNA was expressed and translated UAG codons into leucine to produce the complete protein product. We then took a fluorescent amberless GFP-2S+tRNA clone, miniprepped the construct, and attempted a heat-shock transformation under the exact same conditions for competent DH5-alpha and amberless cells. Although there were more successful transformants in the amberless plate, both plates had visibly fluorescing cells. In order to accurately compare GFP expression, we grew one fluorescent clone from DH5-alpha and amberless for flow cytometry analysis.
Figure 4. Flow cytometry histogram showing that Amberless cells with the GFP-2S+tRNA construct strongly express GFP, while DH5-alpha cells have a mixed population of low-expressing and non-expressing cells.
Figures 4 and 5 demonstrate that GFP expression is 40X greater in amberless E. coli than in wild type E. coli given they both have the supP tRNA . The wild type cells with the GFP-2S serve as a negative control. In wild type cells that contain the GFP-2S and supP tRNA, fluorescence is observed in a small population of cells, and it is much less intense than in the amberless cells. Normally, the supP tRNA is toxic to wild type E. coli, but when these cells expressing some GFP were sequenced, we found mutations in the tRNA. This indicates that with some mutations, the supP tRNA can be tolerated in wild type cells. There mixed population of low-GFP expressing and non-expressing DH5-alpha cells suggest that the mutations render less efficient tRNAs that impair the translation of the complete protein product. This indicates that our Codon Security strategy is effective in significantly reducing protein expression in wild type cells.
Figure 5. Mean fluorescence from flow cytometry data in Figure 4. The data show a high mean fluorescence in amberless cells and a much lower mean fluorescence in DH5-alpha cells containing the same construct. .
Our results with the chromogenic aeBlue protein test construct further corroborate the  effectiveness of Codon Security in preventing gene expression in non-amberless E. coli. We created a version of Team Uppasala's 2012 aeBlue protein with 3 UAG stop codons substituted for leucine (aeBlue-3S) and added the supP tRNA to create a test construct called aeBlue-3S+tRNA. This was sequence verified and BioBricked ( BBa_K1499253 ) in the Amberless cells. Then, we transformed the cleaned-up BioBrick plasmid into Amberless and DH5-alpha cells using the exact same heat shocking and plating protocol (Fig. 6).
Figure 6. Plates after transforming (at Day 0) DH5-alpha and Amberless cells with aeBlue-3S+tRNA. At Day 1, after overnight incubation at 37C, there are no DH5-alpha colonies, but there are over 200 Amberless colonies. At Day 5, 4 days of room temperature incubation, Amberless colonies express the aeBlue protein and colonies appear in DH5-alpha plate, although possibly because of degradation of Chlor antibiotic. Side-by-side comparison of cell pellets shows that only Amberless strongly express the blue protein.
.
As seen in Figure 6, we observed that the sequence-verified aeBlue-3S+tRNA construct could only be transformed into and expressed in Amberless cells. The kinetics of blue protein accumulation in the Amberless cells were relatively slow compared to GFP, with first visible signs at Day 2. We plan to determine the kinetics more thoroughly in an future experiment. Additionally, higher temperatures of incubation were less conducive to protein production; room temperature to 30C was the optimal temperature. The cells were removed from the each plate at Day 5 and lysed for Western blotting.
Figure 7. The aeBlue-3S construct has an N-terminal FLAG-tag that can be detected by Western blot using an anti-FLAG antibody. Lane A has twice the cell lysate concentration of lane B. The band at approximately 30kDa is near the expected molecular weight of the complete aeBlue product (28kDa), meaning that Amberless cells, but not DH5-alpha cells, express the complete protein.
Figure 7 shows that the Amberless cells only produce the full aeBlue-3S protein. Any truncated products would appear at 7, 13, and 20 kDa depending on the stop codon. DH5-alpha do not produce any protein, or a possible alternative is that truncated products degrade quickly in the cell and do not appear in the gel. These data strongly indicate that the supP tRNA enables protein expression through UAG stop codons only in Amberless cells and serve as a proof of concept for our Codon Security strategy.
Results Part 2 - Amberless Hell Cell
In parallel with demonstrating the Codon Security concept, we worked on applying it to radiation resistance genes to create the Amberless Hell Cell chassis. The goal of this part of the project was to confer resistances to Amberless Cells so that they could survive in harsh environments encountered in UAV or space applications. At the same time, we wanted to prove that applying Codon Security, we could ensure that using these "super-strains" of Amberless in the environment would not pose a risk of horizontal gene transfer into other bacteria.
We first isolated 5 radiation resistance genes by PCR from D. radiodurans genomic DNA: uvsE [3], uracil glycosylase 1 , uracil glycosylase 2 [4], MntH , and DpsMP1 . Of these, the last two were previous Stanford-Brown 2012 biobricks that we re-isolated for sequence fidelity. The uvsE and MntH were chosen for initial testing.
Figure 8. Comparison of radiation resistance genes by measuring the fraction of colonies present after exposure to different amounts of UVC radiation. SOD was a resistance gene with unknown radiation resistance qualities. MntH Mut3 was a version of MntH with a UAG stop codon for leucine. Only uvsE-transformed cells showed an indication of radiation resistance.
After determining that uvsE was a new gene that could confer increased radiation resistance to DH5-alpha cells, we repeated the experiment with three replicates of a DH5-alpha negative control and DH5-alpha transformed with uvsE.
Figure 9. UvsE confers significant radiation resistance to DH5-alpha cells after exposure to UVC. Data were determined to be significant to p<.05 and p<.01 using a two-tailed Student's t-test.
We determined that the D. radiodurans UV damage endonuclease uvsE gene can confer extra radiation resistance to E. coli. After confirming its efficacy, we used mutagenesis PCR to introduce stop codons at two different sites on the uvsE gene. Sequencing showed that our mutagenesis protocol for TTG(leucine)->TAG(stop) produces 70-95% of plasmids with the desired mutation. We are currently working on assembling this uvsE-stop codon gene with the supP tRNA to show that radiation resistance will only occur in Amberless cells when Codon Security is employed.
Conclusions
● We showed that Codon Security prevents DH5-alpha from expressing a construct containing stop codons. Despite having the supP tRNA in the construct, the cells do not survive transformation or fail to produce the protein of interest with mutated supP due to the toxicity of having a UAG-reading tRNA.
● We have created an orthogonal synthetic biology system such that genes only properly translate in the Amberless chassis.
● We isolated a radiation resistance gene from D. radiodurans, uvsE, and have for the first time shown its radiation resistance effects in E. coli.
References
1. Lajoie, MJ et al. (2013) Genomically Recoded Organisms Impart New Biological Functions. Science 342: 357-60. PMID: 24136966 .
2. Thorbjarnardóttir, S et al. (1985) Leucine tRNA family of Escherichia coli: nucleotide sequence of the supP(Am) suppressor gene. J. Bacteriol. 161: 219–22. PMID: 2981802 .
3. Earl, AM et al. (2002) Genetic evidence that the uvsE gene product of Deinococcus radiodurans R1 is a UV damage endonuclease. J. Bacteriol. 184(4):1003-9. PMID: 11807060 .
4. Sandigursky, M et al. (2004) Multiple uracil-DNA glycosylase activities in Deinococcus radiodurans. DNA Repair (Amst) 3(2):163-9. PMID: 14706350 .
Additional Information
● Read about how our submitted Amberless Hell Cell idea was used as a government regulatory case study on synthetic biology. We then began a conversation with Dr. Mark Segal at the EPA about the regulation and safety of the use of engineered bacteria in the environment.
● Submitted biobricks :  We submitted 9 biobricks for this sub-project. Six of these bricks include parts that can enable other teams to use the Amberless chassis as a system for more responsible synthetic biology.
   
   
sbsigem@googlegroups.com
View our Complete Project List
Meet our team!
Built atop Foundation. Content & Development © Stanford–Brown–Spelman iGEM 2014.
Retrieved from " http://2014.igem.org/Team:StanfordBrownSpelman/Amberless_Hell_Cell "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
