"
Page
Discussion
View source
History
teams
Log in
 
Team:Dundee/Project/pqs
From 2014.igem.org
Dundee 2014
Toggle navigation
Dundee 2014
Site Information
Policy & Practice
Bulletin Board
Policy & Practice
Cystic Fibrosis Stories
Media and Talks
Fundraising
Project
School
Project
Cystic Fibrosis
Polymicrobial Infection in the CF lung
The Lung Ranger
PQS Sensor
PAI-1 Sensor
BDSF Sensor
DSF Sensor
Modelling
Introduction
Modelling: PQS
Modelling: BDSF
Modelling: DSF
NetLogo
Implementation
L.A.S.S.O.
Methodology
Hardware
Software
Out in the real world
Quantified Impact
Future
Log Book
Library
Safety & Protocols
Notebook
Family Album
iGEM Community
Saloon
Team
Attributions
Collaboration
Medal Fulfillment
Parts
Contact the team
Team Page
Facebook
Twitter
The Pseudomonas Quinolone Signal (PQS) Sensing System
Initial Planning and Cloning Strategy
Building the PQS Sensor
Characterisation
Initial Planning and Cloning Strategy
Pseudomonas quinolone signal (2-heptyl-3-hydroxy-4-quinolone) is a quorum-sensing molecule produced by Pseudomonas aeruginosa, which regulates the expression of genes involved in biofilm development and virulence1. Expression of these traits is mediated through the LysR-type transcriptional regulator, PqsR . Bioinformatic analysis of the PqsR sequence predicted that it is a soluble protein. Fractionation experiments of P. aeruginosa cells has shown that PqsR is primarily associated with the inner membrane2. It is not clear however, whether this is through protein-lipid or protein-protein interactions with membrane lipids or with a yet unidentified integral inner membrane protein, respectively. In the presence of PQS, PqsR interacts with the promoter region of the pqsA operon, allowing transcription of the downstream genes3. We have engineered E. coli to express this signal transduction system for the detection of PQS (as shown in Fig 1), with a promoter-less mCherry coding gene fused to the pqsA promoter to give a fluorescent output.
Building the PQS Sensor
Chromosomal DNA from Pseudomonas aeruginosa PA01 strain was kindly gifted to us by Drs Robert Ryan and Shi-Qi An from the Division of Molecular Microbiology in the College of Life sciences at the University of Dundee. This was used as a template for the amplification of the pqsA promoter gene.
The pqsA promoter region was cloned into the pSB1C3 plasmid (BioBrick BBa_K1315001 ), and was then subcloned into pBluescript. Promoterless mCherry was amplified (template used: BBa_K562012 ), and was cloned into pBluescript downstream of the pqsA promoter. The PpqsA-mCherry construct was then subcloned into pUniprom. A schematic of our completed construct can be seen in Fig 2.
The pqsR gene (present in BBa_K1315001 ) which was designed and cloned by Lin Ang Chieh from iGEM13-NTU-Taida was used as a template for the amplification of pqsR with an  influenza virus haemagglutinin (HA) tag coding sequence, which can be detected by commercially available antibodies. The HA-tag was fused to the C-terminus of the protein to allow visualisation by immunohistochemistry.
The plasmid was verified by sequencing.
The completed construct was transformed into E. coli strain MC1061 as a chassis for our biosensor.
Characterisation
Initially, western blots were undertaken to test for the production of PqsR-HA. We transformed our plasmid encoding PqsR-HA into E. coli strain MC1061 and blotted for recombinantely expressed protein. An overnight culture of the cells was lysed and proteins separated by SDS-PAGE in a 12% acrylamide gel. Anti-HA antibodies linked to horseradish peroxidase were used for detection of PqsR. Fig 3A shows the expression of PqsR within our system.
As shown in Fig 3, successful production of PqsR-HA (expected mass 38 kDa) was observed. With the PqsR regulator being produced we could begin to test for a response to PQS. To test how the system would respond to PQS, cells containing the construct were cultured in LB medium and spiked with synthetic PQS (in 100% DMSO) at finals concentrations of 50 and 500 μM. A western blot with anti-mCherry antibodies was performed on the treated cells alongside a control (DMSO). Two further controls - MC1061 cells harbouring the empty pUniprom vector (mCherry negative control) and BBa_K562011 (mCherry positive control) were also included.
As shown in Fig 4, incubation of cells with synthetic PQS, did not induce expression of mCherry. It is not clear why the biosensor did not respond. One possibility we considered is that PqsR is not correctly localized to the membrane in E. coli. We therefore fractionated E. coli cells containing PqsR-HA into soluble and membrane fractions. However, as shown in Fig 5, all of the detectable PqsR-HA was found in the membranes as it is in the native organism.
We discussed the results of the PQS biosensor extensively with the modelling team . They suggested increasing the number of promoters (by at least 10-fold). This should allow for sufficient production of mCherry, which can then be detected. We are in the process of building this new expression system.
The following parts were deposited as BioBricks:
Part
Description
Registry
pqsA
Promoter
BBa_K1315001
References
1Reen, F.J. et al. (2011) FEMS Microbiol Ecol 77, 413-428.
2Cao, H. et al. (2001) Proc Natl Acad Sci USA 98, 14613–14618.
3Dekimpe, V. et al. (2009) Microbiology 155, 712-723.
Previous Lesson: The Lung Ranger
Back to School
Next Lesson: PAI-1 Sensor
×
Welcome to Dundee's 2014 Wiki (Site Information)
Abstract
Navigation Instructions
Sponsors
Cystic Fibrosis is a genetic disease that results in the accumulation of thick mucus in the epithelial linings of the entire body, particularly the respiratory tract. Over the course of a patient’s life, the mucus-lined lung epithelium becomes repeatedly infected with pathogenic bacteria that stimulate an immune response; leading to inflammation, tissue damage, and ultimately, respiratory failure.
The microflora of the Cystic Fibrosis lung changes over time. In childhood, the major coloniser is Staphylococcus aureus, but as the patient matures other bacterial pathogens infect. The later-dominating pathogens, Pseudomonas aeruginosa and Burkholderia cepacia, are very difficult to eradicate and are associated with chronic decline in lung function. Burkholderia is so infectious that patients have to be isolated from one another, and can be denied lung transplants due to the persistence of this bacterium.
It is important, therefore, for medics to monitor and identify the levels of bacteria within the respiratory tract of a Cystic Fibrosis patient. Currently, patients must provide sputum samples and identification of bacteria takes between 72 hours and 2 weeks, by which point the bacteria can be in an antibiotic resistant state.
The Dundee 2014 iGEM project is focused on designing and testing a device that will rapidly and non-invasively identify the bacteria colonising a Cystic Fibrosis patient. Three biosensors will be developed that recognise external signal molecules produced by key bacteria, all of which are known to be in sputum samples of Cystic Fibrosis patients. A quinolone signal (PQS) is produced by Pseudomonas aeruginosa, a Diffusible Signal Factor (DSF) is produced by Stenotrophomonas maltophilia, an organism that is also associated with Cystic Fibrosis lung infection in adults, and BDSF is a related, but chemically distinct molecule that is produced by Burkholderia cepacia.
By engineering signal recognition and signal transduction proteins from these biological systems the Dundee iGEM team want to produce a portable electronic device that will identify infection outside of the clinic, so that patients do not have to travel great distances, and can be used to help health professionals make informed and rapid decisions on antibiotic treatments.
Read this again by clicking on Site Information in the navigation bar!
Bulletin Board (Policy & Practice)
Come look at the poster explaining how we involved the local and national Cystic Fibrosis community in our project, how we listened to people on the front line of the disease and how they affected our project.
School (Project)
Come in and learn about all the parts that make The Lung Ranger. The lab work that we carried out over the             summer is explained here in detail, along with the modelling and implementation.
Library (Log Book)
Read about the experience the team has had over the summer. A week by week description of our work, how we carried out that work and some photos of our time together.
Saloon (iGEM community)
Come on in, take a seat and learn how we have interacted with the larger iGEM community. Who the team are, who helped them and how we helped others.
College of Life Sciences
With more than 1100 staff and research students and external funding of around £60 million per year, the College of Life Sciences is one of the largest and most productive research institutes in Europe. This reputation is genuinely global and is reflected in the fact that researchers in our laboratories represent no fewer than 62 different nationalities
Division of Mathematics A small friendly Division which offers a lively and modern programme of study at both undergraduate and postgraduate level.We have research groups in Mathematical Biology, Numerical Analysis and Scientific Computing, Magnetohydrodynamics and Applied Analysis.
School of Computing With research areas in Human Centred Computing, Intelligent Systems, Space Technologies and Theory of Computation, the department has an emphasis on development. Furthermore, this department teaches popular undergraduate programs in Applied Computing and Computing Science.
We are a global charitable foundation dedicated to achieving extraordinary improvements in human and animal health. We support the brightest minds in biomedical research and the medical humanities. Our breadth of support includes public engagement, education and the application of research to improve health. We are independent of both political and commercial interests.
BBSRC has a unique and central place in supporting the UK’s world-leading position in bioscience. We are an investor in research and training, with the aim of furthering scientific knowledge, to promote economic growth, wealth and job creation and to improve quality of life in the UK and beyond.
The Biochemical Society exists for the advancement of the molecular and cellular biosciences, both as an academic discipline and to promote its impact on areas of science including biotechnology, agriculture, and medicine.
The Society for General Microbiology is a membership organisation for scientists who work in all areas of microbiology. The Society promotes the understanding of microbiology to a diverse range of stakeholders, including policy-makers, students, teachers, journalists and the wider public, through a comprehensive framework of communication activities and resources.
Established in 2007, Scottish Universities Life Sciences Alliance (SULSA) is a research pooling partnership between the Universities of Aberdeen, Dundee, Edinburgh, Glasgow, St Andrews and Strathclyde that is supported by the Scottish Funding Council.
EASTBIO - the BBSRC East of Scotland Bioscience Doctoral Training Partnership - is a partnership between the Universities of Aberdeen, Dundee, Edinburgh and St Andrews; the Scottish Universities Life Sciences Alliance; and the Scottish Universities Physics Alliance. The DTP scheme will provide students with excellent cross-disciplinary research training in line with the current BBSRC strategy for research support and training.
Retrieved from " http://2014.igem.org/Team:Dundee/Project/pqs "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
