"
Page
Discussion
View source
History
teams
Log in
 
Team:Dundee/Project/dsf
From 2014.igem.org
Dundee 2014
Toggle navigation
Dundee 2014
Site Information
Policy & Practice
Bulletin Board
Policy & Practice
Cystic Fibrosis Stories
Media and Talks
Fundraising
Project
School
Project
Cystic Fibrosis
Polymicrobial Infection in the CF lung
The Lung Ranger
PQS Sensor
PAI-1 Sensor
BDSF Sensor
DSF Sensor
Modelling
Introduction
Modelling: PQS
Modelling: BDSF
Modelling: DSF
NetLogo
Implementation
L.A.S.S.O.
Methodology
Hardware
Software
Out in the real world
Quantified Impact
Future
Log Book
Library
Safety & Protocols
Notebook
Family Album
iGEM Community
Saloon
Team
Attributions
Collaboration
Medal Fulfillment
Parts
Contact the team
Team Page
Facebook
Twitter
The Diffusible Signalling Factor (DSF) Sensing System
Initial Planning and Cloning Strategy
Building the DSF Sensor
Characterisation
Initial Planning and Cloning Strategy
Diffusible Signalling Factor (DSF) is a long chain cis-unsaturated fatty acid produced by bacteria of the Xanthomonas genus and the CF lung pathogen Stenotrophomonas maltophilia1,2.  To build our synthetic DSF detector, we chose to reconstitute the well-characterised DSF signal perception system from Xanthomonas campestris in our E. coli chassis. As can be seen in Fig 1, the RpfC/RpfG system, which perceives and responds to DSF, is more complex than the average two component system. DSF is detected at the periplasmic side of the inner membrane by the RpfC sensor protein, which autophosphorylates and then transfers its phospho group to the  cytoplasmic response regulator RpfG . However, instead of acting as a direct transcriptional regulator, phosphorylated RpfG acts as a phosphodiesterase cleaving the second-messenger molecule cyclic di-guanosine monophosphate (c-di-GMP) into two GMP molecules3,4. This reduction in cellular c-di-GMP relieves inhibition of the transcriptional activator Clp , which is then free to bind to the manA promoter and allow transcription of the downstream genes5. In our synthetic system (as shown in Fig 1) we fused the manA promoter to promoter-less gfp to give a fluorescent output.
Building the DSF Sensor
Chromosomal DNA from Xanthomonas campestris pathovar campestris 8004 was kindly gifted to us by Drs Robert Ryan and Shi-Qi An from the Division of Molecular Microbiology in the College of Life Sciences at the University of Dundee. This was used as template for the amplification of the rpfC, rpfG and clp genes, and the manA promoter region.
The manA promoter region was cloned into the pSB1C3 plasmid (to give BioBrick BBa_K1315006 ), and was then subcloned into pBluescript. Promoterless gfp was amplified (using BBa_K562012 as a template), and was cloned into pBluescript downstream of the manA promoter. The manA promoter-gfp construct was then subcloned into pUniprom. A schematic of our completed construct can be seen below (Fig 2).
To adhere to the iGEM rules and regulations, it was necessary to remove three specified illegal PstI restriction sites and one EcoRI site present in rpfC. We also removed one illegal EcoRI site from rpfG and one illegal PstI site from clp. Each of the modified genes were individually cloned into the pSB1C3 plasmid (to give BioBricks BBa_K1315002 , BBa_K1315003 , BBa_K1315005 , respectively. The genes were then sequentially subcloned into the pUniprom vector that already harboured PmanA-gfp. To facilitate immunochemistry we chose to supply each of RpfC, Clp and RpfG with an influenza virus hemagglutinin (HA) tag which can be detected with commercial antibodies. This tag was added to the C-terminus of each protein.
The plasmid was verified by sequencing.
The completed construct was transformed into E.coli strain MC1061 as a chassis for our biosensor.
Characterisation
A series of western blots were performed to test for the sequential production of each of the HA-tagged proteins. We transformed our plasmid series (encoding RpfC-HA, RpfC-HA + Clp-HA and RpfC-HA +Clp-HA + RpfG-HA) into E. coli strain MC1061 and blotted for recombinant protein production.
As shown in Fig 3, successful production of RpfC-HA (expected mass 81kDa), RpfG-HA (expected mass of 43kDa) and Clp-HA (expected mass 27kDa) was observed. We then went on to test our DSF detector by incubating E. coli producing the three HA-tagged proteins with increasing concentrations of pure DSF (obtained from Sigma) and monitoring for production of GFP from the manA promoter by Western blotting. As shown in Fig 4A, we were able to specifically detect GFP production in E. coli cells harbouring our DSF-detection plasmid. However we noted that GFP was being produced in the absence of exogenous DSF addition. We reasoned that this could be due to either: (i) the endogenous activation of the manA by an E. coli transcriptional regulator or (ii) the presence of DSF or a DSF-like compound in the cells and/or growth medium. To distinguish between these two possibilities we introduced pUniprom carrying the PmanA-gfp fusion alone (i.e. in the absence of the rpfC, rpfG and clp genes) into E. coli and blotted for GFP production. As shown in Fig 4B, GFP was produced by E. coli in the absence of the X. campestris DSF sensing proteins. We therefore conclude that the manA promoter is recognised by an E. coli transcription factor.
For the DSF sensing system to work we need to eliminate the background activation of PmanA-gfp. Since the X. campestris Clp protein is very similar to the E. coli cyclic AMP receptor protein (CRP) transcription factor (46% identity, 71% similarity over the whole length of the protein), we next tested whether the production of GFP was dependent upon CRP. To this end we used a cep deletion strain from the E. coli Keio mutant collection6 and compared GFP production in this strain background with the cognate wild type strain (Fig 5). However GFP was still produced (although possibly at a lower level) than in the cep+ background.
We are continuing to investigate this issue by screening other E. coli mutants carrying deletions in genes coding for transcription factors which show similarity to Clp, for example the anaerobic regulator, FNR. We anticipate that it may be necessary to make combinatorial mutations to completely eliminate exogenous GFP production.
The following parts were deposited as BioBricks:
Part
Description
Registry
rpfC
DSF Sensor Kinase
BBa_K1315002
rpfG
Two-component system regulator activated by sensor kinase
BBa_K1315003
clp
Clp Transcriptional activator inhibited by c-di-GMP
BBa_K1315005
manA
manA gene promoter activated by Clp
BBa_K1315006
References
1Slater, H. et al. (2000) Mol Microbiol 38, 986-1003.
2Ryan, R.P. et al. (2009) Nat Rev Microbiol 7, 514-525.
3Ryan, R.P. et al. (2006) Proc Natl Acad Sci USA 103, 6712-6717.
4Ryan, R.P. et al. (2010) Proc Natl Acad Sci USA 107, 5989-5994.
5Lu, X.-H. et al. (2012) PLOS ONE 7, e52646.
6Baba, T. et al. (2006) Mol Syst Bio 2, 2006.0008.
Previous Lesson: BDSF Sensor
Back to School
Next Lesson: Modelling: Introduction
×
Welcome to Dundee's 2014 Wiki (Site Information)
Abstract
Navigation Instructions
Sponsors
Cystic Fibrosis is a genetic disease that results in the accumulation of thick mucus in the epithelial linings of the entire body, particularly the respiratory tract. Over the course of a patient’s life, the mucus-lined lung epithelium becomes repeatedly infected with pathogenic bacteria that stimulate an immune response; leading to inflammation, tissue damage, and ultimately, respiratory failure.
The microflora of the Cystic Fibrosis lung changes over time. In childhood, the major coloniser is Staphylococcus aureus, but as the patient matures other bacterial pathogens infect. The later-dominating pathogens, Pseudomonas aeruginosa and Burkholderia cepacia, are very difficult to eradicate and are associated with chronic decline in lung function. Burkholderia is so infectious that patients have to be isolated from one another, and can be denied lung transplants due to the persistence of this bacterium.
It is important, therefore, for medics to monitor and identify the levels of bacteria within the respiratory tract of a Cystic Fibrosis patient. Currently, patients must provide sputum samples and identification of bacteria takes between 72 hours and 2 weeks, by which point the bacteria can be in an antibiotic resistant state.
The Dundee 2014 iGEM project is focused on designing and testing a device that will rapidly and non-invasively identify the bacteria colonising a Cystic Fibrosis patient. Three biosensors will be developed that recognise external signal molecules produced by key bacteria, all of which are known to be in sputum samples of Cystic Fibrosis patients. A quinolone signal (PQS) is produced by Pseudomonas aeruginosa, a Diffusible Signal Factor (DSF) is produced by Stenotrophomonas maltophilia, an organism that is also associated with Cystic Fibrosis lung infection in adults, and BDSF is a related, but chemically distinct molecule that is produced by Burkholderia cepacia.
By engineering signal recognition and signal transduction proteins from these biological systems the Dundee iGEM team want to produce a portable electronic device that will identify infection outside of the clinic, so that patients do not have to travel great distances, and can be used to help health professionals make informed and rapid decisions on antibiotic treatments.
Read this again by clicking on Site Information in the navigation bar!
Bulletin Board (Policy & Practice)
Come look at the poster explaining how we involved the local and national Cystic Fibrosis community in our project, how we listened to people on the front line of the disease and how they affected our project.
School (Project)
Come in and learn about all the parts that make The Lung Ranger. The lab work that we carried out over the             summer is explained here in detail, along with the modelling and implementation.
Library (Log Book)
Read about the experience the team has had over the summer. A week by week description of our work, how we carried out that work and some photos of our time together.
Saloon (iGEM community)
Come on in, take a seat and learn how we have interacted with the larger iGEM community. Who the team are, who helped them and how we helped others.
College of Life Sciences
With more than 1100 staff and research students and external funding of around £60 million per year, the College of Life Sciences is one of the largest and most productive research institutes in Europe. This reputation is genuinely global and is reflected in the fact that researchers in our laboratories represent no fewer than 62 different nationalities
Division of Mathematics A small friendly Division which offers a lively and modern programme of study at both undergraduate and postgraduate level.We have research groups in Mathematical Biology, Numerical Analysis and Scientific Computing, Magnetohydrodynamics and Applied Analysis.
School of Computing With research areas in Human Centred Computing, Intelligent Systems, Space Technologies and Theory of Computation, the department has an emphasis on development. Furthermore, this department teaches popular undergraduate programs in Applied Computing and Computing Science.
We are a global charitable foundation dedicated to achieving extraordinary improvements in human and animal health. We support the brightest minds in biomedical research and the medical humanities. Our breadth of support includes public engagement, education and the application of research to improve health. We are independent of both political and commercial interests.
BBSRC has a unique and central place in supporting the UK’s world-leading position in bioscience. We are an investor in research and training, with the aim of furthering scientific knowledge, to promote economic growth, wealth and job creation and to improve quality of life in the UK and beyond.
The Biochemical Society exists for the advancement of the molecular and cellular biosciences, both as an academic discipline and to promote its impact on areas of science including biotechnology, agriculture, and medicine.
The Society for General Microbiology is a membership organisation for scientists who work in all areas of microbiology. The Society promotes the understanding of microbiology to a diverse range of stakeholders, including policy-makers, students, teachers, journalists and the wider public, through a comprehensive framework of communication activities and resources.
Established in 2007, Scottish Universities Life Sciences Alliance (SULSA) is a research pooling partnership between the Universities of Aberdeen, Dundee, Edinburgh, Glasgow, St Andrews and Strathclyde that is supported by the Scottish Funding Council.
EASTBIO - the BBSRC East of Scotland Bioscience Doctoral Training Partnership - is a partnership between the Universities of Aberdeen, Dundee, Edinburgh and St Andrews; the Scottish Universities Life Sciences Alliance; and the Scottish Universities Physics Alliance. The DTP scheme will provide students with excellent cross-disciplinary research training in line with the current BBSRC strategy for research support and training.
Retrieved from " http://2014.igem.org/Team:Dundee/Project/dsf "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
