<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Dundee_Safety_normal"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Dundee/Safety/normal">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Dundee/Safety/normal&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Dundee/Safety/normal&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Dundee/Safety/normal&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Dundee/Safety/normal" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Dundee/Safety/normal</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Dundee 2014</TITLE></P><DIV class="navbar navbar-custom navbar-fixed-top"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON class="navbar-toggle" data-toggle="collapse" data-target="#navBarContent"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A href="/Team:Dundee" class="navbar-brand" style="padding-left:70px;">Dundee 2014</A></DIV><DIV class="collapse navbar-collapse" id="navBarContent"><UL class="nav navbar-nav"><LI><A href="#" id="info">Site Information</A></LI><LI class="dropdown"><A href="'#" class="dropdown-toggle" data-toggle="dropdown">Policy &amp; Practice</A></LI><LI class="nav-header "><I><U>Bulletin Board</U></I></LI><LI><A href="/Team:Dundee/policypractice/experts">Policy &amp; Practice</A></LI><LI><A href="/Team:Dundee/policypractice/clinic">Cystic Fibrosis Stories</A></LI><LI><A href="/Team:Dundee/policypractice/mt">Media and Talks</A></LI><LI><A href="/Team:Dundee/policypractice/fundraising">Fundraising</A></LI></UL><LI class="dropdown"><A href="'#" class="dropdown-toggle" data-toggle="dropdown">Project</A></LI><LI class="nav-header "><I><U>School</U></I></LI><LI class="nav-header ">Project</LI><LI><A href="/Team:Dundee/Project/CF">Cystic Fibrosis</A></LI><LI><A href="/Team:Dundee/Project/lungRanger">Polymicrobial Infection in the CF lung</A></LI><LI><A href="/Team:Dundee/Project/Aims">The Lung Ranger</A></LI><LI><A href="/Team:Dundee/Project/pqs">PQS Sensor</A></LI><LI><A href="/Team:Dundee/Project/PAI-1">PAI-1 Sensor</A></LI><LI><A href="/Team:Dundee/Project/bdsf">BDSF Sensor</A></LI><LI><A href="/Team:Dundee/Project/dsf">DSF Sensor</A></LI><LI class="nav-header">Modelling</LI><LI><A href="/Team:Dundee/Modeling/introduction">Introduction</A></LI><LI><A href="/Team:Dundee/Modeling/pqs">Modelling: PQS</A></LI><LI><A href="/Team:Dundee/Modeling/bdsf">Modelling: BDSF</A></LI><LI><A href="/Team:Dundee/Modeling/dsf">Modelling: DSF</A></LI><LI><A href="/Team:Dundee/Modeling/netlogo">NetLogo</A></LI><LI class="nav-header">Implementation</LI><LI><A href="/Team:Dundee/Implementation/introduction">L.A.S.S.O.</A></LI><LI><A href="/Team:Dundee/Implementation/methodology">Methodology</A></LI><LI><A href="/Team:Dundee/Implementation/lasso">Hardware</A></LI><LI><A href="/Team:Dundee/Implementation/lassoInterface">Software</A></LI><LI><A href="/Team:Dundee/Implementation/realWorld">Out in the real world</A></LI><LI><A href="/Team:Dundee/Implementation/financial">Quantified Impact</A></LI><LI><A href="/Team:Dundee/Implementation/future">Future</A></LI><LI class="dropdown"><A href="'#" class="dropdown-toggle" data-toggle="dropdown">Log Book</A></LI><LI class="nav-header "><I><U>Library</U></I></LI><LI><A href="/Team:Dundee/Safety">Safety &amp; Protocols</A></LI><LI><A href="/Team:Dundee/Notebook">Notebook</A></LI><LI><A href="/Team:Dundee/famAlbum">Family Album</A></LI><LI class="dropdown"><A href="'#" class="dropdown-toggle" data-toggle="dropdown">iGEM Community</A></LI><LI class="nav-header "><I><U>Saloon</U></I></LI><LI><A href="/Team:Dundee/Team">Team</A></LI><LI><A href="/Team:Dundee/Attributions">Attributions</A></LI><LI><A href="/Team:Dundee/collaborations">Collaboration</A></LI><LI><A href="/Team:Dundee/medals">Medal Fulfillment</A></LI><LI><A href="https://2014.igem.org/Team:Dundee/Parts">Parts</A></LI><LI class="dropdown"><A href="'#" class="dropdown-toggle" data-toggle="dropdown">Contact the team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=Dundee">Team Page</A></LI><LI><A href="https://www.facebook.com/igemdundee2014">Facebook</A></LI><LI><A href="https://twitter.com/DundeeiGEMTeam">Twitter</A></LI></DIV></DIV></DIV><DIV class="container"><DIV class="jumbotron"><H1>Safety &amp; Protocols</H1><P class="lead">Better Safe Than Sorry!</P></DIV><DIV class="row"><DIV class="col-col-sm-3 col-xs-3 col-md-3" id="nav"><UL class="nav list-group affix"><LI class="list-group-item"><A href="#0" class="">Safety</A></LI><LI class="list-group-item hide"><A href="#s-1" class="">Distribution and Storage of Genetically Modified Microorganisms</A></LI><LI class="list-group-item hide"><A href="#s-2" class="">Rehydration of Freeze-Dried Cultures</A></LI><LI class="list-group-item hide"><A href="#s-3" class=""> Addition of Sputum to Rehydrated Cultures</A></LI><LI class="list-group-item hide"><A href="#s-4" class=""> Processing of Samples</A></LI><LI class="list-group-item hide"><A href="#s-5" class="">     Removal of Used Culture Plates </A></LI></UL><LI class="list-group-item"><A href="#1" class="">Safety Form</A></LI><LI class="list-group-item"><A href="#2" class="">Protocols</A></LI><LI class="list-group-item hide"><A href="#p-1" class="">Competent Cells</A></LI><LI class="list-group-item hide"><A href="#p-2" class="">Polymerase Chain Reaction</A></LI><LI class="list-group-item hide"><A href="#p-3" class="">Agarose Gel Electrophoresis</A></LI><LI class="list-group-item hide"><A href="#p-4" class="">Gel Extraction</A></LI><LI class="list-group-item hide"><A href="#p-5" class="">Restriction Digests</A></LI><LI class="list-group-item hide"><A href="#p-6" class="">Strataclean</A></LI><LI class="list-group-item hide"><A href="#p-6" class="">Ligations</A></LI><LI class="list-group-item hide"><A href="#p-7" class="">Heat Shock Transformation of competent cells</A></LI><LI class="list-group-item hide"><A href="#p-8" class="">Colony PCR</A></LI><LI class="list-group-item hide"><A href="#p-9" class="">Plasmid Miniprep</A></LI><LI class="list-group-item hide"><A href="#p-10" class="">Site-directed mutagenesis (Quick-Change)</A></LI><LI class="list-group-item hide"><A href="#p-11" class="">Glycerol Stock</A></LI><LI class="list-group-item hide"><A href="#p-12" class="">Testing the biosensors</A></LI><LI class="list-group-item hide"><A href="#p-13" class="">Cell Fractionation</A></LI><LI class="list-group-item hide"><A href="#p-14" class="">Western Blot</A></LI></DIV><DIV class="col-sm-9 col-xs-9"><DIV class="row"><DIV class="col-12"><H2 id="0">Safety</H2></DIV></DIV><DIV class="row"><DIV class="col-12"><P>
The Lung Ranger project has resulted in the construction of a device (the L.A.S.S.O.) intended to be used initially by healthcare workers (and potentially in the longer term by cystic fibrosis patients themselves). Therefore, the team felt that it was essential to implement a short list of well-considered safety protocols to ensure that:



1. Users do not come into direct contact with microorganisms, genetically modified or otherwise
2. Genetically modified microorganisms do not come into contact with wild-type microorganisms which could result in horizontal gene-transfer
3. Genetically modified organisms are disposed of safely

This has resulted in the design and prototyping of a set of components and practices to be utilised in the first instance by trained medical staff in the context of home-visits to cystic fibrosis patients. 

</P></DIV></DIV><DIV class="row"><DIV class="col-12"><H3 id="s-1">Distribution and Storage of Genetically Modified Microorganisms</H3><P>

Sealed LB-Agar plates containing freeze-dried cultures of our modified <I>E.coli</I> would be distributed to cystic fibrosis clinics where they would be stored in the dark at 4 <SUP>o</SUP>C.
 
</P></DIV></DIV><DIV class="row"><DIV class="col-12"><H3 id="s-2">Rehydration of Freeze-Dried Cultures</H3><P>
On the day prior to a home-visit, water would be added to cultures with a sterile syringe via a one-way, filter-sterilizing valve and the plates incubated at 37 <SUP>o</SUP>C overnight. This would be carried out by a member of the clinical staff at the clinic and following training by a member of The Lung Ranger team. 
</P></DIV></DIV><DIV class="row"><DIV class="col-12"><H3 id="s-3">Addition of Sputum to Rehydrated Cultures</H3><P>
On the day of testing, at the patient's home sputum samples would be collected in a sterile syringe that would then be fitted to a plate via a threaded valve (Fig 2). After ensuring that the syringe has been secured to the valve, the plunger would be depressed to add the sputum to the culture. It may be necessary, due to the variability in sputum viscosity, to aspirate Sputolysin<SUP>®</SUP> - a reducing agent- along with the sputum sample to sufficiently thin the sample for ease of use. the filter sterilising valve will prevent wild-type microorganisms from contacting the genetically modified bacteria inside the plate and thus removing the possibility of any uptake of transgenes by other organisms. 
</P></DIV></DIV><DIV class="row"><DIV class="col-12"><H3 id="s-4">Processing of Samples</H3></DIV></DIV><DIV class="row"><DIV class="col-12"><P>
The plate containing a sputum sample would be agitated by hand to ensure adequate coverage of the medium by the sputum before being placed in the L.A.S.S.O. (Fig 4). After light detection, the plate would be transferred to a waste compartment by depressing the plunger-arm (Fig 5). The plunger-arm would be reset and the process can be repeated until the waste compartment is filled.
</P></DIV></DIV><DIV class="row"><DIV class="col-12"><H3 id="s-5">Removal of Used Culture Plates</H3></DIV></DIV><DIV class="row"><DIV class="col-12"><P>
The L.A.S.S.O. can be separated into two sections (Fig 6), allowing the waste compartment when full, to be either emptied or replaced with an empty compartment. In either case, the spent samples would be placed into a sealable bag and would be removed for autoclaving by the health worker ensuring the safe disposal of the genetically modified organisms. 

</P></DIV></DIV><DIV class="row"><DIV class="col-12"><H2 id="1">Safety Form</H2></DIV></DIV>

This is our original Safety Form submitted to iGEM on 1st September 2014. 
<A href="https://static.igem.org/mediawiki/2014/f/fa/DUN_Safety_Form_Transcript.pdf">https://static.igem.org/mediawiki/2014/f/fa/DUN_Safety_Form_Transcript.pdf</A><DIV class="row"><DIV class="col-12"><H2 id="2">Protocols</H2><H3 id="p-1">Competent Cells</H3><B>Day 1</B>
1. Streak out frozen glycerol stock of bacterial cells onto an LB plate (no antibiotics). Work sterile. Grow cells overnight at 37 <SUP>o</SUP>C. 
<B>Day 2</B>
2. Select a single colony of <I>E. coli</I> from fresh LB plate and inoculate a 5 ml starter culture of LB (with no antibiotics). Grow at 37 <SUP>o</SUP>C in shaker overnight.
<B>
Day 3
</B>
3. Inoculate 50 ml fresh growth medium with 500 μl starter culture and grow at 37 <SUP>o</SUP>C for 2 hours.

4. When OD<SUB>600</SUB> reaches 0.35-0.4, immediately put the cells on ice, chill the culture for 20-30 minutes, swirling occasionally to ensure even cooling. Chill centrifuge bottles on ice. 

5. Harvest the cells by centrifugation at 3000g (around 4000rpm in the Beckman Ja-10 rotor) for 15 minutes at 4 <SUP>o</SUP>C.

6. Decant the supernatant and gently resuspend each pellet in about 5 ml of ice-cold transformation buffer<SUP>1</SUP>. 

7. Aliquot in 200 μl batches into sterile 1.5 ml microcentrifuge tubes and snap freeze with liquid nitrogen.
   Store frozen cells at -80 <SUP>o</SUP>C.
</DIV></DIV><DIV class="row"><DIV class="col-12"><H3 id="p-2">Polymerase Chain Reaction (PCR)</H3>
This protocol can be used to amplify the target genes required for the biosensors from genomic DNA and add appropriate restriction sites required for ligation into the vector pUniprom. Primers were supplied by Sigma-Aldrich, the DNA polymerase used was Herculase II Fusion DNA Polymerase which was supplied by Aligent Technologies.
<U>Set up Standard PCR reaction (final volume = 50 μl):</U><TABLE class="table"><THEAD><TR><TH>Component</TH><TH>Volume (μl) </TH></TR></THEAD><TBODY><TR><TD>5x Herculase buffer</TD><TD>10</TD></TR><TR><TD>100 μM forward primer </TD><TD>0.5</TD></TR><TR><TD>100 μM reverse primer </TD><TD>0.5</TD></TR><TR><TD>10 mM dNTPs </TD><TD>0.5</TD></TR><TR><TD>DMSO </TD><TD>2.5</TD></TR><TR><TD>H<SUB>2</SUB>0 </TD><TD>34.5</TD></TR><TR><TD>gDNA </TD><TD>1</TD></TR><TR><TD>Herculase II </TD><TD>0.5</TD></TR></TBODY></TABLE><U>Thermocycler protocol</U><TABLE class="table"><THEAD><TR><TH>Cycle(s)</TH><TH>Time/Temp </TH><TH>Process </TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>3 mins at 95 <SUP>o</SUP>C</TD><TD>Denature DNA</TD></TR><TR><TD>30 </TD><TD>1 min at 95 <SUP>o</SUP>C</TD><TD>DNA denatures into single strands</TD></TR><TR><TD>0.5 min at 75 <SUP>o</SUP>C</TD><TD>Primers bind to DNA template</TD></TR><TR><TD>1 min/kb at 68 <SUP>o</SUP>C</TD><TD>Primers extended 5’-3’ (1 min/kb)-elongation phase</TD></TR><TR><TD> 1 </TD><TD>5 min at 72 <SUP>o</SUP>C</TD><TD>Final elongation</TD></TR><TR><TD>10 <SUP>o</SUP>C</TD><TD>Final hold</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="row"><DIV class="col-12"><H3 id="p-3">Agarose Gel Electrophoresis</H3><P> 
DNA products were run on a 1% agarose gel to separate charged nucleic acids by size for analysis/purification. 

GelRed was used to stain nucleic acids and gels were visualised on a BioRad GelDoc. 

Standard 1% agarose gel

1. Measure 1g of agarose.
2. Pour agarose powder to 100 ml of 1xTAE<SUP>1</SUP> (tris-acetate EDTA buffer) in a microwavable flask.
3. Microwave until agarose has dissolved (1-3 mins) and there is a rolling boil - Caution HOT! 
4. Allow solution to cool down.
5. Add Gel Red (6 µl per 100 ml 1% agarose solution).
6. Pour cooled agarose solution into a gel tray and place required combs in place (prevent any air bubbles).
7. Allow poured gel to sit on bench top for 20-30 minutes to solidify.
8. Once gel has solidified, remove the comb and place gel into the electrophoresis unit.
9. Fill the gel tank with 1xTAE.
10. Carefully load DNA weight ladder into the first lane of the gel. 
11. Carefully load samples into the additional wells of the gel (be careful not to disturb the wells).
12. Run the gel at 100 V for 30 mins until the dye line is approximately 75-80% of the way down the gel. 
13. Turn off the power, disconnect the electrodes from the power source and then carefully remove the gel from the gel box.
14. UV light used to visualize the DNA fragments. 
</P></DIV></DIV><DIV class="row"><DIV class="col-12"><H3 id="p-4">Gel Extraction</H3><P>
QIAquick gel Extraction Kit Protocol designed to extract and purify DNA of 70 bp to 10 kb from standard or low-melt agarose gels in TAE buffer. 

1. Excise the DNA fragment from the agarose gel with a clean, sharp scalpel.
2. Weigh the gel slice in a colorless tube. Add 3 volumes of Buffer QG to 1 volume of gel (100 mg=100 μl).
3. Incubate at 50 <SUP>o</SUP>C for 10 min (or until the gel slice has completely dissolved). To help dissolve gel, mix by vortexing the tube every 2-3 min during the incubation. 
4. After the gel slice has dissolved completely, check that the colour of the mixture is yellow.
5. Add 1 gel volume of isopropanol to the sample and mix.
6. if the DNA will be subsequently used for sequencing, in vitro transcription, or microinjection, add 500 μl Buffer QG to the QIAquick column and centrifuge for 1 min. Discard flow-through and place the QIAquick column back into the same tube. 
7. To wash, add 750 μl Buffer PE to QIAquick column and centrifuge for 1 min. Discard flow-through and place the QIAquick column back into the same tube. Centrifuge the QIAquick column in the provided 2 ml collection tube for 1 min to remove residual wash buffer. 
8. Place QIAquick into a clean 1.5 ml microcentrifuge tube.
9. To elute DNA, add 50 μl of water to the centre of the QIAquick membrane and centrifuge the column for 1 min. For increased DNA concentration, add 30 µl water to the centre of the QIAquick membrane, let stand for 1min, and then centrifuge for 1 min. 
10. If the purified DNA is to be analysed on a gel, add 1 volume of loading dye to 5 volumes of purified DNA.
</P></DIV></DIV><DIV class="row"><DIV class="col-12"><H3 id="p-5">Restriction Digests</H3><P>
Restriction Digests were performed to prepare DNA for ligation. Digests were carried out using enzymes supplied by Roche and New England Biolabs. Digests were performed at 37 <SUP>o</SUP>C for 3 hours. 

An aliguot of alkaline phosphatase was added to restriction digests of vectors at 2 hrs and a second aliquot at 2.5 hrs Reactions were stopped after three hours. Digested PCR products were separated from restriction enzymes and buffers using Strataclean resin (Aligent Technologies). Digested vectors were purified on a 1% agarose gel and then extracted. 

60 μl reactions following PCR:

1. 50 μl PCR product
2. 1.5 μl restriction enzyme 1
3. 1.5 μl restriction enzyme 2
4. 6 μl cognate digestion Buffer for enzyme(s)
5. 1 μl H<SUB>2</SUB>0
Heatblock for 3 hours at 37 <SUP>o</SUP>C
</P></DIV></DIV><DIV class="row"><DIV class="col-12"><H3>Strataclean</H3><P>
√RD sample= volume of Strataclean
√60= 7.745 μl

1. Ensure Strataclean is vortexed before use. 
2. Add 7.75 μl to each restriction digest. 
3. Vortex for 15 sec and then centrifuge for 1 min at 13,000rpm in a microfuge. 
4. Pipette supernatant into a clean microcentrifuge tube.</P></DIV></DIV><DIV class="row"><DIV class="col-12"><H3 id="p-6">Ligations</H3><P>
For ligation of genes into vectors, the digested gene of interest and the appropriate vector were incubated with T4 DNA ligase at room temperature for 3 hrs.
<U>Total reaction volume= 10 μl</U><TABLE class="table"><THEAD><TR><TH>Cycles</TH><TH>Vector Only </TH><TH>2:1</TH><TH>3:1 </TH></TR></THEAD><TBODY><TR><TD>10x T4 ligase buffer</TD><TD>1 μl </TD><TD>1 μl </TD><TD>1 μl</TD></TR><TR><TD>Insert</TD><TD>-</TD><TD>2 μl</TD><TD>2μl</TD></TR><TR><TD>Vector</TD><TD>1 μl</TD><TD>1 μl</TD><TD>1 μl </TD></TR><TR><TD> T4 ligase</TD><TD>1 μl</TD><TD>1 μl</TD><TD>1 μl </TD></TR><TR><TD> H<SUB>2</SUB>o</TD><TD>7 μl </TD><TD>5 μl </TD><TD>4 μl </TD></TR></TBODY></TABLE><B>N.b. T4 ligase added last to the reaction.</B></P></DIV></DIV><DIV class="row"><DIV class="col-12"><H3 id="p-7">Heat Shock Transformation of competent cells</H3><P>
Plasmids or ligation products containing the desired genes were transformed into chemocompetent <I>E. coli</I> cells. 

1. Thaw competent cells on ice. Usually 10-20 minutes.
2. Pipette 100 μl of cells into a pre-chilled eppendorf tube. 
3. Add 10 μl of ligation reaction or 2 μl of mini prepped plasmid.
4. Mix by gently flicking the tube.
5. Chill on ice for 30 minutes.
6. Heat shock at 42 <SUP>o</SUP>C for 90 seconds. 
7. Return to ice for 2 minutes.
8. Add 1 ml of LB medium to cells and recover by shaking at 37 <SUP>o</SUP>C (ampicillin selection requires less recovery time than chloramphenicol selection).
9. Centrifuge for 3 minutes. 
10. Remove supernatant and resuspend in 100 μl of LB.
11. Plate out the cells on selective LB agar plates containing appropriate antibiotics and spread using sterile glass spreader.
12. Incubate overnight at 37 <SUP>o</SUP>C.
</P></DIV></DIV><DIV class="row"><DIV class="col-12"><H3 id="p-8">Colony PCR</H3><P>
PCR was used to identify the desired clones on the plate of transformed cells. PCR was performed on the samples using <I>Taq</I> polymerase supplied by Aligent Technologies. 
<B> DNA preparation</B>
1. Pick a single colony and twizzle into 30 μl of nuclease-free H<SUB>2</SUB>O.
2. Boil for 10 minutes at 100 <SUP>o</SUP>C to lyse cells.
3. Centrifuge for 1 minute at 13000rpm in a microfuge.
4. 5 μl of DNA was then used as part of the reaction mixture outlined below.

<U>Colony PCR reaction</U></P><P>
Prepare the following PCR reaction on ice (N.B. adding the enzyme last):
</P><TABLE class="table"><THEAD><TR><TH>Component</TH><TH>Volume (μl) </TH></TR></THEAD><TBODY><TR><TD>10x PCR reaction Mg (Roche)</TD><TD>2</TD></TR><TR><TD>100 μM forward primer </TD><TD>0.2</TD></TR><TR><TD>100 μM reverse primer </TD><TD>0.2 </TD></TR><TR><TD>10 mM dNTPs </TD><TD>0.2</TD></TR><TR><TD>DMSO </TD><TD>2</TD></TR><TR><TD> Template DNA </TD><TD>5</TD></TR><TR><TD>H<SUB>2</SUB>0 </TD><TD>11.2</TD></TR><TR><TD><I>Taq</I> Polymerase  </TD><TD>0.5</TD></TR></TBODY></TABLE><U>Thermocycler Protocol</U><TABLE class="table"><THEAD><TR><TH>Cycle(s)</TH><TH>Time/Temp </TH><TH>Process </TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>3 mins at 95 <SUP>o</SUP>C</TD><TD>Denature DNA</TD></TR><TR><TD>30 </TD><TD>1 min at 95 <SUP>o</SUP>C</TD><TD>DNA denatures into single strands</TD></TR><TR><TD>0.5 min at 75 <SUP>o</SUP>C</TD><TD>Primers bind to DNA template</TD></TR><TR><TD>1 min/kb at 68 <SUP>o</SUP>C</TD><TD>Primers extended 5’-3’ (1 min/kb)-elongation phase</TD></TR><TR><TD> 1 </TD><TD>5 min at 72 <SUP>o</SUP>C</TD><TD>Final elongation</TD></TR><TR><TD>10 <SUP>o</SUP>C</TD><TD>Final hold</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="row"><DIV class="col-12"><H3 id="p-9">Plasmid Miniprep</H3><P>
Plasmids were isolated using a QIAprep Spin Miniprep Kit Protocol.

A 5 ml overnight culture of cells was centrifuged at 4000rpm for 10 min. The resulting pellet was lysed in an alkaline lysis buffer and centrifuged. Supernatant is passed through a silica membrane to isolate plasmid DNA.

1. Resuspend pelleted bacterial cells in 250 μl Buffer P1 and transfer to a microcentrifuge tube. <B>N.b.</B> Ensure that RNase A has been added to Buffer P1. 
2. Add 250 μl Buffer P2 and gently invert the tube 4-6 times to mix.
Mix gently by inverting the tube. Do not vortex, as this can result in shearing of genomic DNA.<B>N.b.</B> Do not allow the lysis reaction to proceed for more than 5 min. 
3. Add 350 μl Buffer N3 and invert the tube immediately but gently 4-6 times. 
4. Centrifuge for 10 min at 13,000 rpm in a microfuge.
5. Apply the supernatant from step 4 to the QIAprep Spin column by decanting or pipetting.
6. Centrifuge for 30-60 seconds. Discard the flow-through.
7. (Optional): wash the QIAPrep Spin Column by adding 0.5ml Buffer PB and centrifuging for 30-60 seconds. Discard the flow-through. 
8. Wash QIAPrep Spin column by adding 0.7 ml of Buffer PE and centrifuging for 30-60s. 
9. Discard the flow-through, and centrifuge for an additional 1 min to remove residual wash buffer.
10. Place the QIAprep column in a clean 1.5 ml microcentrifuge tube. To elute DNA, add 50 μl water to the centre of each QIAprep Spin column, let stand for 1 min, and centrifuge for 1 min.
</P></DIV></DIV><DIV class="row"><DIV class="col-12"><H3 id="p-10">Site directed mutagenesis (QuikChange™)</H3><P>
Site directed mutagenesis was used to remove internal restriction sites from genes. The method relies on PCR using primers containing a single nucleotide mutation, which maintains the amino acid sequence and has specificity 15 bp up- and downstream of the target nucleotide. Herculase II polymerase was used in PCR reactions to amplify the target plasmid containing the mutation. PCR products were digested with the restriction enzyme <I>Dpn</I>I to remove any methylated DNA leaving only the PCR product.
<U>PCR reaction</U><TABLE class="table"><THEAD><TR><TH>Component</TH><TH>0.5x DNA template mixture (μl) </TH><TH>3x DNA template mixture (μl) </TH></TR></THEAD><TBODY><TR><TD>5x Herculase buffer</TD><TD>10</TD><TD>10</TD></TR><TR><TD>100 μM forward primer </TD><TD>1</TD><TD>1</TD></TR><TR><TD>100 μM reverse primer </TD><TD>1 </TD><TD>1</TD></TR><TR><TD>10 mM dNTP </TD><TD>1</TD><TD>1</TD></TR><TR><TD>H<SUB>2</SUB>0 </TD><TD>36.5</TD><TD>34</TD></TR><TR><TD>DNA template</TD><TD>0.5</TD><TD>3</TD></TR><TR><TD>Herculase II </TD><TD>1</TD><TD>10</TD></TR></TBODY></TABLE></P><U>Thermocycler protocol</U><TABLE class="table"><THEAD><TR><TH>Cycle (s)</TH><TH>Time/Temp </TH><TH>Process </TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>3 mins at 95 <SUP>o</SUP>C</TD><TD>Denature DNA</TD></TR><TR><TD>30</TD><TD>1 min at 95 <SUP>o</SUP>C</TD><TD>DNA denatures into single strands</TD></TR><TR><TD>0.5 min at 75 <SUP>o</SUP>C</TD><TD>Primers bind to DNA template</TD></TR><TR><TD>1 min/kb at 68 <SUP>o</SUP>C</TD><TD>Primers extended 5’-3’ (1 min/kb)-elongation phase</TD></TR><TR><TD> 1 </TD><TD>5 min at 72 <SUP>o</SUP>C</TD><TD>Final elongation</TD></TR><TR><TD>10 <SUP>o</SUP>C</TD><TD>Final Hold</TD></TR></TBODY></TABLE>
Run a 5 µl aliquot of each PCR product on a 1 % agarose gel to check for amplified target gene. Digest PCR products with restriction enzyme <I>Dpn</I>I for 3 hours at 37 <SUP>o</SUP>C to remove any methylated DNA leaving only the PCR product. 5 μl or 10 μl of PCR product then transformed into competent <I>E. coli</I> cells and plated onto appropriate LB selective media.
</DIV></DIV><DIV class="row"><DIV class="col-12"><H3 id="p-11">Glycerol Stock</H3><P>
1. Pick single colonies from agar plate.
2. Inoculate 5 ml LB broth overnight.
3. Add 1 ml of overnight culture to 1 ml of 60 % glycerol in a cryotube.
4. Freeze at -80 <SUP>o</SUP>C.
</P></DIV></DIV><DIV class="row"><DIV class="col-12"><H3 id="p-12">Testing the biosensors</H3><P>
In order to test our complete biosensors the following protocol was followed:

1. Prepare 5 ml O/N cultures in LB medium containing the completed constructs supplemented with appropriate antibiotics.
2. Following 16 hr of growth at 37 <SUP>o</SUP>C, subculture each sample into fresh growth medium (1:100.)
3. Spike cultures with saturating concentrations of synthetic signalling molecules. 
4. Grow cultures for a further 2 hours to allow sufficient time for GFP /mCherry production.
5. Take 1 ml aliquots and proceed with western blot protocol.</P></DIV></DIV><DIV class="row"><DIV class="col-12"><H3 id="p-13">Cell Fractionation</H3><P>
In order to test our complete biosensors the following protocol was followed:

1. Inoculate 5 ml of LB growth medium supplemented with appropriate antibiotic. 
2. Allow 16 hr growth at 37 <SUP>o</SUP>C and then subculture into 100 ml of fresh growth medium (1:100 dilution). 
3. Grow cells for a further 8 hours and pellet by centrifugation.
4. Wash pellet once in 50 mM Tris-HCl, pH 7.5. 
5. Repellet the cells and then resuspend in 2 ml of the same buffer.
6. Lyse by sanitation. 
7. Centrifuge sonicated samples to remove unbroken cells.
8. Ultracentrifuge the supernatant to pellet the cell membranes.
9. Remove the supernatant (corresponds to the soluble fraction of the cell).
10. Resuspend membrane pellet in 1 ml of 50 mM Tris-HCl, pH 7.5.
11. Separate 10 μl of the samples by SDS PAGE (12 % acrylamide), transfer to PVDF and probe with specific antibody (detailed western blot protocol below).
</P></DIV></DIV><DIV class="row"><DIV class="col-12"><H3 id="p-13">Western blot</H3><P>
Western blotting was used to detect the expression of proteins in the <I>E. coli</I> chassis. 1 ml of an overnight culture of cells were pelleted by centrifugation. 
The pellets were resuspended in Laemmli buffer and boiled for 2 mins to lyse the cells. Samples were then loaded on freshly prepared 4-12 % polyacrylamide gels. Gels were run in SDS buffer at 180 V for 45-60 mins to separate proteins by size.
Proteins were transferred onto PVDF membrane by semi-dry transfer at 175 mA for 20 min for western blot analysis, following the transfer membranes were blocked in 5 % milk/TBS overnight. The target proteins contained a hemagglutinin tag for detection by monoclonal Anti-HA−Peroxidase antibodies supplied by Sigma-Aldrich. Membranes were washed in Tris-buffered saline containing 0.1 % Tween 20 and incubated with the antibody at 1:10000 concentration in TBST for 1 hour at room temperature.
Photo-detection of the cross-reacting bands used ECL reagent (luminol + HRP) and exposure on photographic film.
</P><DIV class="row"><DIV class="col-12"><DIV id="ref"><H3>References</H3><P><SUP>1</SUP>Sambrook, J., and Russell, D. W. (2001) Molecular cloning : a laboratory manual, 3rd ed., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row" id="btnRow"><DIV class="btn-group btn-group-justified"><DIV class="btn-group"><A href="https://2014.igem.org/Team:Dundee/inside/library" class="btn btn-default toClass">Back to library</A></DIV><DIV class="btn-group"><A href="https://2014.igem.org/Team:Dundee/Notebook/normal" class="btn btn-default toLesson">Next book: Notebook</A></DIV></DIV></DIV></DIV><DIV class="modal static" id="welcomeNote" role="dialog" data-backdrop="static"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-hidden="true" id="closeNote">×</BUTTON><H4 class="modal-title" id="myModalLabel">Welcome to Dundee's 2014 Wiki (<B>Site Information</B>)</H4></DIV><DIV class="modal-body"><UL class="nav nav-tabs"><LI class="active"><A href="#ab" data-toggle="tab">Abstract</A></LI><LI><A href="#instr" data-toggle="tab">Navigation Instructions</A></LI><LI><A href="#spon" data-toggle="tab">Sponsors</A></LI></UL><DIV class="tab-content"><DIV class="tab-pane active" id="ab">  
   Cystic Fibrosis is a genetic disease that results in the accumulation of thick mucus in the epithelial linings of the entire body, particularly the respiratory tract. Over the course of a patient’s life, the mucus-lined lung epithelium becomes repeatedly infected with pathogenic bacteria that stimulate an immune response; leading to inflammation, tissue damage, and ultimately, respiratory failure.  

The microflora of the Cystic Fibrosis lung changes over time. In childhood, the major coloniser is <I>Staphylococcus aureus</I>, but as the patient matures other bacterial pathogens infect. The later-dominating pathogens, <I>Pseudomonas aeruginosa</I> and <I>Burkholderia cepacia</I>, are very difficult to eradicate and are associated with chronic decline in lung function. <I>Burkholderia</I> is so infectious that patients have to be isolated from one another, and can be denied lung transplants due to the persistence of this bacterium. 

It is important, therefore, for medics to monitor and identify the levels of bacteria within the respiratory tract of a Cystic Fibrosis patient. Currently, patients must provide sputum samples and identification of bacteria takes between 72 hours and 2 weeks, by which point the bacteria can be in an antibiotic resistant state. 

The Dundee 2014 iGEM project is focused on designing and testing a device that will rapidly and non-invasively identify the bacteria colonising a Cystic Fibrosis patient. Three biosensors will be developed that recognise external signal molecules produced by key bacteria, all of which are known to be in sputum samples of Cystic Fibrosis patients. A quinolone signal (PQS) is produced by <I>Pseudomonas aeruginosa</I>, a Diffusible Signal Factor (DSF) is produced by <I>Stenotrophomonas maltophilia</I>, an organism that is also associated with Cystic Fibrosis lung infection in adults, and BDSF is a related, but chemically distinct molecule that is produced by <I>Burkholderia cepacia</I>. 

By engineering signal recognition and signal transduction proteins from these biological systems the Dundee iGEM team want to produce a portable electronic device that will identify infection outside of the clinic, so that patients do not have to travel great distances, and can be used to help health professionals make informed and rapid decisions on antibiotic treatments. 

Read this again by clicking on <B>Site Information</B> in the navigation bar!      
   </DIV><DIV class="tab-pane" id="instr"><DIV class="row"><DIV class="col-sm-9 col-md-9"><H3>Bulletin Board (Policy &amp; Practice)</H3><P>
     Come look at the poster explaining how we involved the local and national Cystic Fibrosis community in our project, how we listened to people on the front line of the disease and how they affected our project.
    </P></DIV></DIV><DIV class="row"><DIV class="col-sm-9 col-md-9"><H3>School (Project)</H3><P>
       Come in and learn about all the parts that make The Lung Ranger. The lab work that we carried out over the    
        summer is explained here in detail, along with the modelling and implementation. 
    </P></DIV></DIV><DIV class="row"><DIV class="col-sm-9 col-md-9"><H3>Library (Log Book)</H3><P>
        Read about the experience the team has had over the summer. A week by week description of our work, how we carried out that work and some photos of our time together.
    </P></DIV></DIV><DIV class="row"><DIV class="col-sm-9 col-md-9"><H3>Saloon (iGEM community)</H3><P>
Come on in, take a seat and learn how we have interacted with the larger iGem community. Who the team are, who helped them and how we helped others.
    </P></DIV></DIV></DIV><DIV class="tab-pane" id="spon"><DIV class="row"><DIV class="col-xs-4"><P><B>College of Life Sciences</B>
With more than 1100 staff and research students and external funding of around £60 million per year, the College of Life Sciences is one of the largest and most productive research institutes in Europe. This reputation is genuinely global and is reflected in the fact that researchers in our laboratories represent no fewer than 62 different nationalities
</P></DIV><DIV class="col-xs-4"><P><B>Division of Mathematics</B>
A small friendly Division which offers a lively and modern programme of study at both undergraduate and postgraduate level.We have research groups in Mathematical Biology, Numerical Analysis and Scientific Computing, Magnetohydrodynamics and Applied Analysis.
</P></DIV><DIV class="col-xs-4"><P><B>School of Computing</B>
With research areas in Human Centred Computing, Intelligent Systems, Space Technologies and Theory of Computation, the department has an emphasis on development. Furthermore, this department teaches popular undergraduate programs in Applied Computing and Computing Science.
</P></DIV></DIV><DIV class="row"><DIV class="col-xs-6"><P>
We are a global charitable foundation dedicated to achieving extraordinary improvements in human and animal health. We support the brightest minds in biomedical research and the medical humanities. Our breadth of support includes public engagement, education and the application of research to improve health. We are independent of both political and commercial interests.
</P></DIV></DIV><DIV class="row"><DIV class="col-xs-6">
BBSRC has a unique and central place in supporting the UK’s world-leading position in bioscience. We are an investor in research and training, with the aim of furthering scientific knowledge, to promote economic growth, wealth and job creation and to improve quality of life in the UK and beyond.
</DIV></DIV><DIV class="row"><DIV class="col-xs-6">
The Biochemical Society exists for the advancement of the molecular and cellular biosciences, both as an academic discipline and to promote its impact on areas of science including biotechnology, agriculture, and medicine.
</DIV></DIV><DIV class="row"><DIV class="col-xs-6">
The Society for General Microbiology is a membership organisation for scientists who work in all areas of microbiology. The Society promotes the understanding of microbiology to a diverse range of stakeholders, including policy-makers, students, teachers, journalists and the wider public, through a comprehensive framework of communication activities and resources.
</DIV></DIV><DIV class="row"><DIV class="col-xs-6">
Established in 2007, Scottish Universities Life Sciences Alliance (SULSA) is a research pooling partnership between the Universities of Aberdeen, Dundee, Edinburgh, Glasgow, St Andrews and Strathclyde that is supported by the Scottish Funding Council.
</DIV></DIV><DIV class="row"><DIV class="col-xs-6">
EASTBIO - the BBSRC East of Scotland Bioscience Doctoral Training Partnership - is a partnership between the Universities of Aberdeen, Dundee, Edinburgh and St Andrews; the Scottish Universities Life Sciences Alliance; and the Scottish Universities Physics Alliance. The DTP scheme will provide students with excellent cross-disciplinary research training in line with the current BBSRC strategy for research support and training.
</DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Dundee/Safety/normal">http://2014.igem.org/Team:Dundee/Safety/normal</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Dundee/Safety/normal" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Dundee/Safety/normal" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Dundee/Safety/normal&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Dundee/Safety/normal&amp;oldid=363179" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>