<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Dundee_Notebook_normal"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Dundee/Notebook/normal">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Dundee/Notebook/normal&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Dundee/Notebook/normal&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Dundee/Notebook/normal&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Dundee/Notebook/normal" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Dundee/Notebook/normal</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Dundee 2014</TITLE></P><DIV class="navbar navbar-custom navbar-fixed-top"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON class="navbar-toggle" data-toggle="collapse" data-target="#navBarContent"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A href="/Team:Dundee" class="navbar-brand" style="padding-left:70px;">Dundee 2014</A></DIV><DIV class="collapse navbar-collapse" id="navBarContent"><UL class="nav navbar-nav"><LI><A href="#" id="info">Site Information</A></LI><LI class="dropdown"><A href="'#" class="dropdown-toggle" data-toggle="dropdown">Policy &amp; Practice</A></LI><LI class="nav-header "><I><U>Bulletin Board</U></I></LI><LI><A href="/Team:Dundee/policypractice/experts">Policy &amp; Practice</A></LI><LI><A href="/Team:Dundee/policypractice/clinic">Cystic Fibrosis Stories</A></LI><LI><A href="/Team:Dundee/policypractice/mt">Media and Talks</A></LI><LI><A href="/Team:Dundee/policypractice/fundraising">Fundraising</A></LI></UL><LI class="dropdown"><A href="'#" class="dropdown-toggle" data-toggle="dropdown">Project</A></LI><LI class="nav-header "><I><U>School</U></I></LI><LI class="nav-header ">Project</LI><LI><A href="/Team:Dundee/Project/CF">Cystic Fibrosis</A></LI><LI><A href="/Team:Dundee/Project/lungRanger">Polymicrobial Infection in the CF lung</A></LI><LI><A href="/Team:Dundee/Project/Aims">The Lung Ranger</A></LI><LI><A href="/Team:Dundee/Project/pqs">PQS Sensor</A></LI><LI><A href="/Team:Dundee/Project/PAI-1">PAI-1 Sensor</A></LI><LI><A href="/Team:Dundee/Project/bdsf">BDSF Sensor</A></LI><LI><A href="/Team:Dundee/Project/dsf">DSF Sensor</A></LI><LI class="nav-header">Modelling</LI><LI><A href="/Team:Dundee/Modeling/introduction">Introduction</A></LI><LI><A href="/Team:Dundee/Modeling/pqs">Modelling: PQS</A></LI><LI><A href="/Team:Dundee/Modeling/bdsf">Modelling: BDSF</A></LI><LI><A href="/Team:Dundee/Modeling/dsf">Modelling: DSF</A></LI><LI><A href="/Team:Dundee/Modeling/netlogo">NetLogo</A></LI><LI class="nav-header">Implementation</LI><LI><A href="/Team:Dundee/Implementation/introduction">L.A.S.S.O.</A></LI><LI><A href="/Team:Dundee/Implementation/methodology">Methodology</A></LI><LI><A href="/Team:Dundee/Implementation/lasso">Hardware</A></LI><LI><A href="/Team:Dundee/Implementation/lassoInterface">Software</A></LI><LI><A href="/Team:Dundee/Implementation/realWorld">Out in the real world</A></LI><LI><A href="/Team:Dundee/Implementation/financial">Quantified Impact</A></LI><LI><A href="/Team:Dundee/Implementation/future">Future</A></LI><LI class="dropdown"><A href="'#" class="dropdown-toggle" data-toggle="dropdown">Log Book</A></LI><LI class="nav-header "><I><U>Library</U></I></LI><LI><A href="/Team:Dundee/Safety">Safety &amp; Protocols</A></LI><LI><A href="/Team:Dundee/Notebook">Notebook</A></LI><LI><A href="/Team:Dundee/famAlbum">Family Album</A></LI><LI class="dropdown"><A href="'#" class="dropdown-toggle" data-toggle="dropdown">iGEM Community</A></LI><LI class="nav-header "><I><U>Saloon</U></I></LI><LI><A href="/Team:Dundee/Team">Team</A></LI><LI><A href="/Team:Dundee/Attributions">Attributions</A></LI><LI><A href="/Team:Dundee/collaborations">Collaboration</A></LI><LI><A href="/Team:Dundee/medals">Medal Fulfillment</A></LI><LI><A href="https://2014.igem.org/Team:Dundee/Parts">Parts</A></LI><LI class="dropdown"><A href="'#" class="dropdown-toggle" data-toggle="dropdown">Contact the team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=Dundee">Team Page</A></LI><LI><A href="https://www.facebook.com/igemdundee2014">Facebook</A></LI><LI><A href="https://twitter.com/DundeeiGEMTeam">Twitter</A></LI></DIV></DIV></DIV><DIV class="container"><DIV class="jumbotron"><H1>Notebook</H1><P class="lead">When it happened</P></DIV><DIV class="row"><H3>Week 0</H3><DIV class="col-sm-9 col-xs-9 col-md-9"><P>
A few members of the team, fresh from their exams, began the initial literature reviews necessary to identify the genes and promoters that were required in our biological sensors that would detect and respond to signaling molecules released by <I>Stenotrophomonas maltophilia</I> (DSF), <I>Burkholderia cenocepacia</I> (BDSF) and <I>Pseudomonas aeruginosa</I> (PQS). Finally, <I>rpfC</I>, <I>rpfG</I>, <I>clp</I> and P<I><SUB>manA</SUB></I> were to be cloned to detect DSF; <I>BCAM0227</I>, <I>BCAM0228</I> and P<I><SUB>cbld</SUB></I> for BDSF; and <I>pqsR</I> and P<I><SUB>pqsA</SUB></I> for PQS. 


Primers were designed (which included restriction sites required for cloning and also HA-tags to detect for protein expression) for the PQS sensing system and ordered from Sigma Aldrich.

All sequences were analyzed with Webcutter 2.0 to detect restriction sites that conflict with the BioBrick assembly standard RFC [10]. It was found <I>clp</I> contained an illegal <I>Pst</I>I site; <I>BCAM0228</I> and <I>rpfg</I> contained an illegal <I>Eco</I>RI site. Primers were then designed for use with QuikChange™ Site-Directed Mutagenesis Kit to remove illegal sites. It was decided at this stage that for the DSF system, homologues of all sequences would be taken from <I>Xanthomonas campestris</I> pv. <I>campestris</I> as they have identical functions but contain fewer illegal restriction sites.
However, <I>BCAM0227</I> was proving problematic. It was so large (3000+bp), and had so many illegal restriction sites, it would have taken the full ten weeks to QuikChange™  them out! We decided to send it for synthetic sequencing by Dundee Cell Products in two separate parts. We hoped this would allow us to successfully ligate the two pieces together into one without mistakes.</P></DIV></DIV><DIV class="row"><H3>Week 1</H3><DIV class="col-sm-9 col-xs-9 col-md-9"><P>
The team are all together! Work begins…
We began organizing our raw materials, setting up overnight cultures of our vectors (pSB1C3 for BioBricks, and our supervisor’s own pUniprom for experiments) and obtaining gDNA from <I>P. aeruginosa</I>, <I>X. campestris</I> and <I>B. cenocepacia</I> from Drs Robert Ryan and Shi-Qi An from the Division of Molecular Microbiology in the College of Life sciences at the University of Dundee
By the end of the week we had cloned <I>pqsR</I> and P<I><SUB>pqsA</SUB></I> into pSB1C3, phew!
As for the dry-lab, the first week consisted mostly of reading papers and harassing Gillian, our mathematical biologist, for definitions of all the bio-jargon. In fact, most of our early discussions about modelling our systems consisted of furrowed brows from both sides, and questions such as “a differential what?” and “Uhhh, transcriptional whatnow?” 
</P></DIV></DIV><DIV class="row"><H3>Week 2</H3><DIV class="col-sm-9 col-xs-9 col-md-9"><P>
In order to test our biological sensors we decided to clone them into pUniprom, a high copy-number plasmid which has a large and varied multiple cloning site downstream of a strong, constitutive promoter: P<I><SUB>tat</SUB></I>.  In order to arrange our sequences in such a way that there would be no leakage of our reporter, we had to employ a bit of cloning trickery by inserting our sensing and responding fragments in opposite orientations. This involved cloning them into separate fragments and then flipping them round into pUniprom. And so to the bench we went!  
Now that the plan for the plasmid was in place, we began to enter all the information onto Netlogo to model how the system would behave in the presence of different concentrations of exogenous PQS. 
</P></DIV></DIV><DIV class="row"><H3>Week 3</H3><DIV class="" col-sm-9="" col-xs-9="" col-md-9=""><P>
While members of the team continued to work on building the PQS sensor, we also got started on planning and constructing the DSF sensor. From this point onwards we decided to use GFP rather than mCherry, as we already had antibodies against it for blotting later. Our desired DSF sensor has four sensory components! This posed a bit of a problem for cloning, as we thought we would need to use more restriction enzymes than we had access to. The solution? Order primers that include a 5’ <I>Bgl</I>II site and 3’ <I>Bam</I>HI-<I>Xba</I>I sites. This way, fragments could be cloned in sequentially. Each new insert could be cut with <I>Bgl</I>II-<I>Xba</I>I, and the ever-growing vector cut with <I>Bam</I>HI-<I>Xba</I>I. This is because the <I>Bgl</I>II and <I>Bam</I>HI overhangs are complementary, but upon ligation leave a scar that is not recognized by either enzyme (see below). The 3’ <I>Bam</I>HI-<I>Xba</I>I site allows this process to be repeated indefinitely. Woo, idempotency! A lot of cloning followed...
</P></DIV></DIV><DIV class="row"><H3>Week 4</H3><DIV class="col-sm-9 col-xs-9 col-md-9"><P>
With lots of cloning, comes lots of sequencing… which was all kindly carried out by the sequencing service downstairs from our lab. To help us along the way Professor Philip Cohen kindly granted us 100 free sequencing runs (which we quickly exceeded!). After every cloning step for building our PQS and DSF sensor, we sent a sample to be sequenced – the results were a source of merriment and heartbreak in equal measure…it turned out that our cloned PQS sensor wasn’t quite right so we spent the week trying to rectify this. 
We also began the lengthy task of removing all the illegal restriction sites from the fragments bound for BioBricking; and so began Dimitrios’ epic journey into the world of BioBricks!
It turned out, (after a bit more sifting through papers), that the Netlogo model for PQS was not quite right. While we figured that out, we began to calculate the ordinary differential equations for the same system.
</P></DIV></DIV><DIV class="row"><H3>Week 5</H3><DIV class="" col-sm-9="" col-xs-9="" col-md-9=""><P>
We successfully biobricked P<I><SUB>pqsA</SUB></I>!
Thankfully we had succesfully ligated P<SUB><I>pqsA</I></SUB> with <I>pqsR-HA</I> into pUniprom. 

After western blotting the PQS system, we obtained results which showed that PqsR was being expressed. We then obtained synthetic PQS from Sigma Aldrich to test this sensor for the production of GFP. 
Meanwhile...the dry team were finding the the DSF model slightly harder to work with than PQS (sorry guys!), as it contained a lot of components; our first model started with about 12 equations! We were able to reduce these down and simplify the model, which enabled us to input them into MATLAB and produce some graphs. We had a few problems determining exactly what our GFP production was reliant on, but after a lot of work on our blackboard, we came to a conclusion! Again, our deterministic model allowed us to change our initial conditions allowing us to see what would happen.
A lot of stochastic modelling happened during this week; PQS wild type and PQS <I>E.coli</I> being our priority. Adding in an external signal proved a little difficult, as we did not get the expected results! There was a lot of trial and error in trying to match up parameters from MatLab and Maple, but by the end of the week we thought we were there! Netlogo was also coming along nicely, with PQS wild type and PQS <I>E. coli</I> finished! BDSF <I>E. coli</I> was almost finished too, with only DSF to tackle after that!
</P></DIV></DIV><DIV class="row"><H3>Week 6</H3><DIV class="col-sm-9 col-xs-9 col-md-9"><P>

We were all excited to test our first sensor, an inducible GFP production assay was set up, by spiking overnight cultures of transformed <I>E. coli</I> MC1061 with a range of concentrations of synthetic PQS. Unfortunately, we obtained no mCherry expression....along with a lot of perplexed biologists!
We successfully removed the first illegal site from <I>rpfC</I>. Meanwhile <I>clp</I> BioBricking was underway, and work had begun on the BDSF system.
We were halfway through the 10 weeks and we were feeling productive! The dry lab were certainly ahead of the game as far as BDSF was concerned; after working through the mammoth DSF system, the closely related two-component BCAM0227/BCAM0228 system was a breeze. In fact, with such a head-start it was decided that there would be a presentation given by the maths boffins of the team entitled: “Modelling for the Perplexed Biologists and Dave”. This was an interactive lesson on how to apply equations to biological systems which helped the lab-rats get to grips with how the figures being generated by the dry team could inform the final design of the biological systems. By the end of the week, <I>rpfG</I>, <I>rpfC</I> and <I>clp</I> were cloned into pUniprom!
Around this time, the “Device Committee” was formed; a small group of wet and dry members who would begin the design of a physical unit to house and interpret the output from our engineered bacteria.  This began with background reading on creating a GFP-fluorescence reading device. Order initial parts to build a light sensing device based on LED light source so we could modulate the light levels. We considered that we would have to use MatLab to carry out calculations. Our visits to the clinic took a more direct focus, and during the interviews we were conducting each week, we started asking patients for their thoughts on the design of the physical detector.
</P></DIV></DIV><DIV class="row"><H3>Week 7</H3><DIV class="col-sm-9 col-xs-9 col-md-9"><P>
Sequencing results for <I>rpfG</I> QuikChange™  showed that it was actually <I>rpfC</I>…oops! Steps were made to rectify this quickly!
All promoters had now been successfully cloned and sequenced and so the job now was to clone them along with their fluorescent reporters into pUniprom. 
Our two sequences of <I>BCAM0227</I> were successfully returned from the sequencing company, Dundee Cell Products, for us to work with. The two separate sequences were digested, and work commenced to try and ligate them together into pbluescript...easier said than done!
Initial experiments with a photodiode, Arduino and a green LED for the physical device, commenced. First designs showed us that we would need an excitation source to induce fluorescence, this would necessitate the inclusion of filters and amplifiers…extremely complicated! Discussions were had with the entire team about eventually using firefly luciferase as a reporter in the place of fluorescent proteins. </P></DIV></DIV><DIV class="row"><H3>Week 8</H3><DIV class="col-sm-9 col-xs-9 col-md-9"><P>
After lots of modelling of the PQS system, and trying to troubleshoot why our system was not expressing mCherry, the decision was made to design another two biological sensors for <I>P. aeruginosa</I>, which would detect another quorums sensing molecule <I>Pseudomonas</I> autoinducer 1 (PAI-1). Lots of work for the wet team!
An overnight culture of <I>E. coli</I> MC1061 transformed with the DSF plasmid was pelleted and prepared for western blotting for detection of our proteins - alas, RpfC-HA is not being expressed! Back to the cloning board…
BioBricking of <I>rpfG</I> was undertaken upon receiving good news from sequencing that our illegal <I>Eco</I>RI restriction site had been removed.</P></DIV></DIV><DIV class="row"><H3>Week 9</H3><DIV class="col-sm-9 col-xs-9 col-md-9"><P>
The tricksome <I>BCAM0227</I> was successfully ligated together into one gargantuan sequence, and swiftly cloned into pUniprom (already containing the P<SUB><I>cbld</I></SUB>-<I>gfp</I> fusion).
BCAM0228 QuikChange of the illegal <I>Eco</I>RI site was complete and sent for sequencing to confirm this was actually the case.
The Device Committee met up with Dr. Stephen Reynolds from the University’s physics department, who helped us to optimize our light-detecting circuit. We constructed a crude (but effective!) Faraday cage from aluminium foil and a cardboard box to eliminate electrical interference from the buzzing computers in the dry lab…
We managed to clone <I>rpfC</I> into pUniprom, along with the rest of the DSF system; time for some tests!</P></DIV></DIV><DIV class="row"><H3>Week 10</H3><DIV class="col-sm-9 col-xs-9 col-md-9"><P>
Sequencing results came back for the QuikChanged <I>BCAM0227</I> and confirmed that the <I>Eco</I>RI site had been removed and so we handed it over to Dimitrios to clone it into pSB1C3.We obtained a culture of <I>Vibrio fischeri</I> from Dr Sarah Coulthurst’s lab, and began streaking plates to test out our physical device for detection of the bioluminescence from <I>V. fischeri</I> luciferase – it worked! At this point our device was essentially half of a water bottle covered in insulating tape, an Arduino and a voltage meter...However, we had come up with a product name: the Light Amplifying Signal Sensing Object, because every good cowboy needs a L.A.S.S.O., right?! It was time for a more robust design; our thoughts moved to 3D printing, and iGEM alumnus and polymath Nasir Ahmad showed us how to draw up plans using Google Sketch. 

</P></DIV></DIV><DIV class="row"><H3>Week 11</H3><DIV class="col-sm-9 col-xs-9 col-md-9"><P>
At this stage, we had all of our systems (barring PAI-1) cloned into pUniprom and ready for blotting. So with a little, (actually, rather a lot), of help from &quot;The Blotfather&quot; Prof. Frank Sargent, we set to the task with all the elation of a team that can see the end of cloning in sight. 
Meanwhile, some of the wet team set to the noble task of cloning our replacement <I>Pseudomonas</I> system. Luckily, all the components we needed were included in the 2014 iGEM distribution pack so we stabbed, rehydrated and got to work. We started by using the P<I><SUB>tet</SUB></I> BioBrick as a vector and cloned in <I>lasR</I> downstream.
The Device Committee met with Dr. Mike MacDonald from the Institute for Medical Science and Technology (IMSAT) to discuss plans for building a physical device. They learned from him the importance of following international standards of operation (ISOs). From this point on we endeavoured to follow ISOs 13485 (medical devices) &amp; 9001(general quality management) so that, if the device were to be taken on by a company, our design and parts would be easily replicable.</P></DIV></DIV><DIV class="row"><H3>Week 12</H3><DIV class="col-sm-9 col-xs-9 col-md-9"><P>

After western blotting the BDSF system, we obtained results which showed that both BCAM0227 and BCAM0228 were both being expressed. We then set up for an inducible GFP production assay, by spiking overnight cultures of transformed <I>E. coli</I> MC1061 with a range of concentrations of synthetic BDSF. Unfortunately, GFP appeared to be expressed at all concentrations, including 0 µM! This experiment clearly needed to be repeated...
The cloning of the two PAI-1 sensors was continued, by ligating two different promoters downstream of LasR; P<I><SUB>lasb</SUB></I> and P<I>luxR</I>.
The Device Committee were having a great time designing the L.A.S.S.O.; a cavity here, a plunger there, maybe some springs... taking into account the ISO requirements as specified by Dr MacDonald.</P></DIV></DIV><DIV class="row"><H3>Week 13</H3><DIV class="col-sm-9 col-xs-9 col-md-9"><P>
Dimitrios has been busy and managed to biobrick P<SUB>cblD</SUB>.
The growing PAI-1 system was complemented with the addition of the nanoluc BioBrick.
The dry team set about modelling the potential savings that could be made from the implementation of The Lung Ranger/L.A.S.S.O. They began by investigating the cost for the NHS of keeping patients in hospital for treatment, which could be given domestically if the need was identified earlier. Also under consideration was the amount of days taken off work by the patients when ill or visiting the clinic – a concern raised by some patients we met at the clinic.</P></DIV></DIV><DIV class="row"><H3>Week 14</H3><DIV class="col-sm-9 col-xs-9 col-md-9"><P>
The biobrick man strikes yet again.....P<I><SUB>manA</SUB></I> was successfully cloned into pSB1C3. 
More L.A.S.S.O. testing with <I>V. fischeri</I> – this time with plates containing different amounts of bacteria, obtained by serial dilution of an overnight culture. This resulted in our first attempt to map the sensitivity of our device vs. c.f.u. of bacteria.
The BDSF GFP blot was repeated with similar results, and the decision was made to begin removing genes from the system until we identified which one was causing the problem.
Western blots of the DSF system showed successful expression of all three proteins; RpfC, RpfG and Clp. 
We then set up for an inducible GFP production assay, by spiking overnight cultures of transformed <I>E. coli</I> MC1061 with a range of concentrations of synthetic DSF. 
Similar to our BDSF system, GFP appeared to be expressed at all concentrations, including 0 µM! We need to come up with a plan, quick!!
Sequencing of the PAI-1 system showed that the nanoluc sequence was missing, hmm….. </P></DIV></DIV><DIV class="row"><H3>Week 15</H3><DIV class="col-sm-9 col-xs-9 col-md-9"><P>
Repeated ligations of PAI-1 system with nanoluc luciferase.
The final sequencing results from the BioBricked <I>rpfc</I> was celebrated with party hats...no more BioBricks for Dimitrios, woohooo!!

We received our 3D printed L.A.S.S.O., it looked great but didn’t fit together as planned… out with the Stanley knife! No problem. 
The GFP assay of the DSF system showed similar results to that of the BDSF system; surely not?! The experiment was repeated and the same results obtained. Oh, dear!
The final L.A.S.S.O. design was almost complete, with only some means of safe disposal for our engineered bacteria to be factored in. We settled on a design that would involve sealed petri-dishes containing freeze-dried <I>E. coli</I> with a one-way valve on the lid, through which a sample of sputum and water would be passed. The plate would then be inserted into the L.A.S.S.O. and all sources of external light eliminated by sliding flaps. The photodiode would pick up luminescence from the luciferase, expressed in response to the bacterial signaling molecule of interest, and send a voltage to the Arduino. This would pass this signal via a µUSB cable to a computer, where an application would interpret the signal into an email containing information about the bacterial species present, which would be sent to a clinician. The plate would then be transferred mechanically into an isolated waste compartment which can be removed and emptied into a biological waste container when full. 
</P></DIV></DIV><DIV class="row"><H3>Week 16</H3><DIV class="col-sm-9 col-xs-9 col-md-9"><P>
Why are our sensors constantly producing GFP??
Instead of painstakingly removing each gene, we decided (on the advice of the University’s top microbiologist Prof. Tracy Palmer) to blot the DSF and BDSF systems for GFP when only the promoter/reporter fusions were present. This was a breakthrough as the P<I><SUB>manA</SUB></I>-<I>gfp</I> fusion still showed GFP expression while the P<I><SUB>cblD</SUB></I>-<I>gfp</I> fusion did not. 

This lead us to perform a BLAST search of the Clp sequence from the DSF system against the <I>E. coli</I> genome to look for similar transcription factors in the wild type. CRP from <I>E. coli</I> showed significant similarity so we obtained a CRP-deletion mutant (BW25113 ∆CRP) and transformed it with our promoter/reporter fusion in preparation for blotting.<I>BCAM0228</I>-P<SUB><I>cblD</I></SUB>-GFP was set up in <I>E. coli</I> MC1061 for blotting, as there may be potential crosstalk causing GFP to be produced in the absence of our sensor kinase, BCAM0227.</P></DIV></DIV><DIV class="row"><H3>Week 17</H3><DIV class="col-sm-9 col-xs-9 col-md-9"><P>The two PAI-1 sensors looked good from sequencing and so was transformed into <I>E. coli</I> MC1061 in preparation for blotting for GFP after induction with synthetic PAI-1 obtained from Sigma Aldrich.
Western blot of <I>BCAM0228</I>-P<SUB><I>cblD</I></SUB>-<I>gfp</I> showed GFP expression. Contrasted with the negative results from the P<SUB>cblD</SUB>-<I>gfp</I> blot, this suggested that something native to <I>E. coli</I> was activating BCAM0228. 
A Western blot of P<SUB><I>manA</I></SUB>-<I>gfp</I> in the CRP deletion strain of <I>E. coli</I> still showed expression of GFP. 
</P></DIV></DIV><DIV class="row"><H3>Week 18</H3><DIV class="col-sm-9 col-xs-9 col-md-9"><P>
The two PAI-1 sensors containing different promoters both worked to different extents. The sensor containing P<SUB><I>lasB</I></SUB>-<I>gfp</I> showed slow induction of GFP over time, however the sensor containing P<SUB><I>luxR</I></SUB>-<I>gfp</I> was initially on but showed increasing GFP production over time. 
What a relief. After much head-scratching and brow-furrowing over the other two systems it was great to get a positive result. We have learned a lot about the potential cross-talk between signal transduction pathways when they are expressed in different species. We believe that research into untangling these interactions will significantly speed-up the process of applying synthetic biology solutions to real-world problems. 

Time to pack up the lab and hand over our equipment to the new PhD students for training, it’s been a rollercoaster of a project that every one of us has benefited from in ways we will likely still be discovering years from now.

See you in Boston!

</P></DIV></DIV></DIV><DIV class="row" id="btnRow"><DIV class="btn-group btn-group-justified"><DIV class="btn-group"><A href="https://2014.igem.org/Team:Dundee/Safety/normal" class="btn btn-default toLesson">Previous Book: Safety &amp; Protocols</A></DIV><DIV class="btn-group"><A href="https://2014.igem.org/Team:Dundee/inside/library" class="btn btn-default toClass">Back to Library</A></DIV><DIV class="btn-group"><A href="https://2014.igem.org/Team:Dundee/famAlbum" class="btn btn-default toLesson">Next Book: Family Album</A></DIV></DIV></DIV></DIV><DIV class="modal static" id="welcomeNote" role="dialog" data-backdrop="static"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-hidden="true" id="closeNote">×</BUTTON><H4 class="modal-title" id="myModalLabel">Welcome to Dundee's 2014 Wiki (<B>Site Information</B>)</H4></DIV><DIV class="modal-body"><UL class="nav nav-tabs"><LI class="active"><A href="#ab" data-toggle="tab">Abstract</A></LI><LI><A href="#instr" data-toggle="tab">Navigation Instructions</A></LI><LI><A href="#spon" data-toggle="tab">Sponsors</A></LI></UL><DIV class="tab-content"><DIV class="tab-pane active" id="ab">  
   Cystic Fibrosis is a genetic disease that results in the accumulation of thick mucus in the epithelial linings of the entire body, particularly the respiratory tract. Over the course of a patient’s life, the mucus-lined lung epithelium becomes repeatedly infected with pathogenic bacteria that stimulate an immune response; leading to inflammation, tissue damage, and ultimately, respiratory failure.  

The microflora of the Cystic Fibrosis lung changes over time. In childhood, the major coloniser is <I>Staphylococcus aureus</I>, but as the patient matures other bacterial pathogens infect. The later-dominating pathogens, <I>Pseudomonas aeruginosa</I> and <I>Burkholderia cepacia</I>, are very difficult to eradicate and are associated with chronic decline in lung function. <I>Burkholderia</I> is so infectious that patients have to be isolated from one another, and can be denied lung transplants due to the persistence of this bacterium. 

It is important, therefore, for medics to monitor and identify the levels of bacteria within the respiratory tract of a Cystic Fibrosis patient. Currently, patients must provide sputum samples and identification of bacteria takes between 72 hours and 2 weeks, by which point the bacteria can be in an antibiotic resistant state. 

The Dundee 2014 iGEM project is focused on designing and testing a device that will rapidly and non-invasively identify the bacteria colonising a Cystic Fibrosis patient. Three biosensors will be developed that recognise external signal molecules produced by key bacteria, all of which are known to be in sputum samples of Cystic Fibrosis patients. A quinolone signal (PQS) is produced by <I>Pseudomonas aeruginosa</I>, a Diffusible Signal Factor (DSF) is produced by <I>Stenotrophomonas maltophilia</I>, an organism that is also associated with Cystic Fibrosis lung infection in adults, and BDSF is a related, but chemically distinct molecule that is produced by <I>Burkholderia cepacia</I>. 

By engineering signal recognition and signal transduction proteins from these biological systems the Dundee iGEM team want to produce a portable electronic device that will identify infection outside of the clinic, so that patients do not have to travel great distances, and can be used to help health professionals make informed and rapid decisions on antibiotic treatments. 

Read this again by clicking on <B>Site Information</B> in the navigation bar!      
   </DIV><DIV class="tab-pane" id="instr"><DIV class="row"><DIV class="col-sm-9 col-md-9"><H3>Bulletin Board (Policy &amp; Practice)</H3><P>
     Come look at the poster explaining how we involved the local and national Cystic Fibrosis community in our project, how we listened to people on the front line of the disease and how they affected our project.
    </P></DIV></DIV><DIV class="row"><DIV class="col-sm-9 col-md-9"><H3>School (Project)</H3><P>
       Come in and learn about all the parts that make The Lung Ranger. The lab work that we carried out over the    
        summer is explained here in detail, along with the modelling and implementation. 
    </P></DIV></DIV><DIV class="row"><DIV class="col-sm-9 col-md-9"><H3>Library (Log Book)</H3><P>
        Read about the experience the team has had over the summer. A week by week description of our work, how we carried out that work and some photos of our time together.
    </P></DIV></DIV><DIV class="row"><DIV class="col-sm-9 col-md-9"><H3>Saloon (iGEM community)</H3><P>
Come on in, take a seat and learn how we have interacted with the larger iGem community. Who the team are, who helped them and how we helped others.
    </P></DIV></DIV></DIV><DIV class="tab-pane" id="spon"><DIV class="row"><DIV class="col-xs-4"><P><B>College of Life Sciences</B>
With more than 1100 staff and research students and external funding of around £60 million per year, the College of Life Sciences is one of the largest and most productive research institutes in Europe. This reputation is genuinely global and is reflected in the fact that researchers in our laboratories represent no fewer than 62 different nationalities
</P></DIV><DIV class="col-xs-4"><P><B>Division of Mathematics</B>
A small friendly Division which offers a lively and modern programme of study at both undergraduate and postgraduate level.We have research groups in Mathematical Biology, Numerical Analysis and Scientific Computing, Magnetohydrodynamics and Applied Analysis.
</P></DIV><DIV class="col-xs-4"><P><B>School of Computing</B>
With research areas in Human Centred Computing, Intelligent Systems, Space Technologies and Theory of Computation, the department has an emphasis on development. Furthermore, this department teaches popular undergraduate programs in Applied Computing and Computing Science.
</P></DIV></DIV><DIV class="row"><DIV class="col-xs-6"><P>
We are a global charitable foundation dedicated to achieving extraordinary improvements in human and animal health. We support the brightest minds in biomedical research and the medical humanities. Our breadth of support includes public engagement, education and the application of research to improve health. We are independent of both political and commercial interests.
</P></DIV></DIV><DIV class="row"><DIV class="col-xs-6">
BBSRC has a unique and central place in supporting the UK’s world-leading position in bioscience. We are an investor in research and training, with the aim of furthering scientific knowledge, to promote economic growth, wealth and job creation and to improve quality of life in the UK and beyond.
</DIV></DIV><DIV class="row"><DIV class="col-xs-6">
The Biochemical Society exists for the advancement of the molecular and cellular biosciences, both as an academic discipline and to promote its impact on areas of science including biotechnology, agriculture, and medicine.
</DIV></DIV><DIV class="row"><DIV class="col-xs-6">
The Society for General Microbiology is a membership organisation for scientists who work in all areas of microbiology. The Society promotes the understanding of microbiology to a diverse range of stakeholders, including policy-makers, students, teachers, journalists and the wider public, through a comprehensive framework of communication activities and resources.
</DIV></DIV><DIV class="row"><DIV class="col-xs-6">
Established in 2007, Scottish Universities Life Sciences Alliance (SULSA) is a research pooling partnership between the Universities of Aberdeen, Dundee, Edinburgh, Glasgow, St Andrews and Strathclyde that is supported by the Scottish Funding Council.
</DIV></DIV><DIV class="row"><DIV class="col-xs-6">
EASTBIO - the BBSRC East of Scotland Bioscience Doctoral Training Partnership - is a partnership between the Universities of Aberdeen, Dundee, Edinburgh and St Andrews; the Scottish Universities Life Sciences Alliance; and the Scottish Universities Physics Alliance. The DTP scheme will provide students with excellent cross-disciplinary research training in line with the current BBSRC strategy for research support and training.
</DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Dundee/Notebook/normal">http://2014.igem.org/Team:Dundee/Notebook/normal</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Dundee/Notebook/normal" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Dundee/Notebook/normal" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Dundee/Notebook/normal&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Dundee/Notebook/normal&amp;oldid=362889" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>