"
Page
Discussion
View source
History
teams
Log in
 
Team:Dundee/Project/bdsf
From 2014.igem.org
Dundee 2014
Toggle navigation
Dundee 2014
Site Information
Policy & Practice
Bulletin Board
Policy & Practice
Cystic Fibrosis Stories
Media and Talks
Fundraising
Project
School
Project
Cystic Fibrosis
Polymicrobial Infection in the CF lung
The Lung Ranger
PQS Sensor
PAI-1 Sensor
BDSF Sensor
DSF Sensor
Modelling
Introduction
Modelling: PQS
Modelling: BDSF
Modelling: DSF
NetLogo
Implementation
L.A.S.S.O.
Methodology
Hardware
Software
Out in the real world
Quantified Impact
Future
Log Book
Library
Safety & Protocols
Notebook
Family Album
iGEM Community
Saloon
Team
Attributions
Collaboration
Medal Fulfillment
Parts
Contact the team
Team Page
Facebook
Twitter
The Burkholderia Diffusible Signalling Factor (BDSF) Sensing System
Initial Planning and Cloning Strategy
Building the BDSF Sensor
Characterisation
Initial Planning and Cloning Strategy
Cis-2 fatty acids are used by many bacterial species as signalling molecules to facilitate inter- and intra-species communication and as a method of regulation of gene expression. Burkholderia Diffusible Signalling Factor (BDSF) is a cis-2-dodecenoic acid that is produced exclusively by the pathogenic bacteria of the Burkholderia cepacia complex where it regulates expression of genes involved in virulence1,2. It is structurally similar to, but distinct from, DSF (cis-11-methyl-2-dodecenoic acid) produced by Stenotrophomonas maltophilia.
In B. cenocepacia BDSF activates gene expression through a two component phosphorelay. BCAM0227 is a transmembrane histidine kinase which phosphorylates the response regulator BCAM0228 in the presence of exogenous BDSF. BCAM0228 then binds and activates transcription of cblD , a gene involved in Burkholderia virulence2.
We engineered E. coli to express this signal transduction system for the detection of BDSF (as shown in Fig 1), with a promoter-less gfp gene downstream of the cblD promoter. With this, our sensor will detect any BDSF in its environment via the BCAM0227 receptor and activate GFP production.
Building the BDSF Sensor
Chromosomal DNA from Burkholderia cenocepacia J2315 was kindly gifted to us by Drs Robert Ryan and Shi-Qi An from the Division of Molecular Microbiology in the College of Life Sciences at the University of Dundee. This was used as template for the amplification of BCAM0228 and the cblD promoter region.
The cblD promoter region was cloned into the pSB1C3 plasmid (to give BioBrick BBa_K1315008 ), and was then subcloned into pBluescript. Promoterless gfp was amplified (using BBa_K562012 as a template), and was cloned into pBluescript downstream of the cblD promoter. The manA promoter-gfp construct was then subcloned into pUniprom. A schematic of our completed construct can be seen below (Fig 2).
A modified version of the BCAM0227 gene which was compatible with BioBrick specifications and standards was synthesised by a third party (Dundee Cell Products). This was subsequently subcloned into the pUniprom vector harbouring PcblD-gfp.
To adhere to the iGEM rules and regulations, it was necessary to remove an illegal EcoRI restriction site present in BCAM0228. The modified gene was cloned into the pSB1C3 plasmid (to give BioBrick BBa_K1315007 ). BCAM0228 was then subcloned into the pUniprom vector that already harboured PmanA-gfp and BCAM0227. To facilitate immunochemistry, we chose to supply BCAM0227 and BCAM0228 with an influenza virus hemagglutinin (HA) tag which can be detected with commercial antibodies. This tag was added to the C-terminus of each protein.
The plasmid was verified by sequencing. The completed construct was transformed into MC1061 E. coli as a chassis for our biosensor.
Characterisation
Initially, Western blots were undertaken to test for the sequential production of BCAM0227-HA and BCAM0228-HA.  An overnight culture of the cells were lysed and proteins separated by SDS-PAGE in a 12% acrylamide gel. Anti-HA antibodies linked to horseradish peroxidase were used for detection of BCAM0227 and BCAM0228. Fig 3 shows that both of the proteins are being expressed in the system.
With all of the components of the system being produced, we could begin to test for a response to BDSF. To test how the system would respond to BDSF, cells containing the construct were cultured in LB medium and spiked with synthetic BDSF in DMSO at concentrations of 50 µM, which corresponds to the levels found in the sputum of lungs colonised by Burkholderia3 and 500 µM. A western blot with anti-GFP antibodies was performed on the treated cells alongside an un-spiked, BDSF-negative control, and MC1061 cells harbouring the empty pUniprom vector. The results are shown in Fig 4.
These results indicate that GFP production is activated regardless of the presence of BDSF. To test whether our E. coli chassis was responsible for activating GFP production, either via phosphorylation of BCAM0228 or by directly activating the cblD promoter, we made a new construct harbouring PcblD-gfp and BCAM0228-HA, but lacking BCAM0227. Fig 5 shows that GFP was only produced in the presence of BCAM0228-HA. We therefore concluded that GFP output is likely caused by crosstalk by two component regulatory systems within E. coli promoting the phosphorylated state of the BCAM0228 response regulator.
We are continuing to investigate this issue by screening E. coli mutants carrying deletions in genes coding for sensor kinases, taking advantage of the E. coli Keio mutant collection3. Alternatively it may be that BCAM0228 is being phosphorylated by a small molecular weight phosphate donor such acetylphosphate. Acetylphosphate production can be eliminated in a pta-ackA double deletion mutant. Further work is required before the BDSF sensor is fully functional. The following parts were deposited as BioBricks:
Part
Description
Registry
BCAM0228
BDSF receptor/histidine kinase
BBa_K1315007
cblD
BCAM0228 inducible promoter
BBa_K1315008
References
1Deng, Y.Y. et al. (2010) Appl Environ Microbiol 76, 4675-4683.
2McCarthy, Y. et al. (2010) Mol Microbiol 77, 1220-1236.
3Twomey, K.B. et al. (2012) ISME J 6, 939-950.
Previous Lesson: PAI-1 Sensor
Back to School
Next Lesson: DSF Sensor
×
Welcome to Dundee's 2014 Wiki (Site Information)
Abstract
Navigation Instructions
Sponsors
Cystic Fibrosis is a genetic disease that results in the accumulation of thick mucus in the epithelial linings of the entire body, particularly the respiratory tract. Over the course of a patient’s life, the mucus-lined lung epithelium becomes repeatedly infected with pathogenic bacteria that stimulate an immune response; leading to inflammation, tissue damage, and ultimately, respiratory failure.
The microflora of the Cystic Fibrosis lung changes over time. In childhood, the major coloniser is Staphylococcus aureus, but as the patient matures other bacterial pathogens infect. The later-dominating pathogens, Pseudomonas aeruginosa and Burkholderia cepacia, are very difficult to eradicate and are associated with chronic decline in lung function. Burkholderia is so infectious that patients have to be isolated from one another, and can be denied lung transplants due to the persistence of this bacterium.
It is important, therefore, for medics to monitor and identify the levels of bacteria within the respiratory tract of a Cystic Fibrosis patient. Currently, patients must provide sputum samples and identification of bacteria takes between 72 hours and 2 weeks, by which point the bacteria can be in an antibiotic resistant state.
The Dundee 2014 iGEM project is focused on designing and testing a device that will rapidly and non-invasively identify the bacteria colonising a Cystic Fibrosis patient. Three biosensors will be developed that recognise external signal molecules produced by key bacteria, all of which are known to be in sputum samples of Cystic Fibrosis patients. A quinolone signal (PQS) is produced by Pseudomonas aeruginosa, a Diffusible Signal Factor (DSF) is produced by Stenotrophomonas maltophilia, an organism that is also associated with Cystic Fibrosis lung infection in adults, and BDSF is a related, but chemically distinct molecule that is produced by Burkholderia cepacia.
By engineering signal recognition and signal transduction proteins from these biological systems the Dundee iGEM team want to produce a portable electronic device that will identify infection outside of the clinic, so that patients do not have to travel great distances, and can be used to help health professionals make informed and rapid decisions on antibiotic treatments.
Read this again by clicking on Site Information in the navigation bar!
Bulletin Board (Policy & Practice)
Come look at the poster explaining how we involved the local and national Cystic Fibrosis community in our project, how we listened to people on the front line of the disease and how they affected our project.
School (Project)
Come in and learn about all the parts that make The Lung Ranger. The lab work that we carried out over the             summer is explained here in detail, along with the modelling and implementation.
Library (Log Book)
Read about the experience the team has had over the summer. A week by week description of our work, how we carried out that work and some photos of our time together.
Saloon (iGEM community)
Come on in, take a seat and learn how we have interacted with the larger iGEM community. Who the team are, who helped them and how we helped others.
College of Life Sciences
With more than 1100 staff and research students and external funding of around £60 million per year, the College of Life Sciences is one of the largest and most productive research institutes in Europe. This reputation is genuinely global and is reflected in the fact that researchers in our laboratories represent no fewer than 62 different nationalities
Division of Mathematics A small friendly Division which offers a lively and modern programme of study at both undergraduate and postgraduate level.We have research groups in Mathematical Biology, Numerical Analysis and Scientific Computing, Magnetohydrodynamics and Applied Analysis.
School of Computing With research areas in Human Centred Computing, Intelligent Systems, Space Technologies and Theory of Computation, the department has an emphasis on development. Furthermore, this department teaches popular undergraduate programs in Applied Computing and Computing Science.
We are a global charitable foundation dedicated to achieving extraordinary improvements in human and animal health. We support the brightest minds in biomedical research and the medical humanities. Our breadth of support includes public engagement, education and the application of research to improve health. We are independent of both political and commercial interests.
BBSRC has a unique and central place in supporting the UK’s world-leading position in bioscience. We are an investor in research and training, with the aim of furthering scientific knowledge, to promote economic growth, wealth and job creation and to improve quality of life in the UK and beyond.
The Biochemical Society exists for the advancement of the molecular and cellular biosciences, both as an academic discipline and to promote its impact on areas of science including biotechnology, agriculture, and medicine.
The Society for General Microbiology is a membership organisation for scientists who work in all areas of microbiology. The Society promotes the understanding of microbiology to a diverse range of stakeholders, including policy-makers, students, teachers, journalists and the wider public, through a comprehensive framework of communication activities and resources.
Established in 2007, Scottish Universities Life Sciences Alliance (SULSA) is a research pooling partnership between the Universities of Aberdeen, Dundee, Edinburgh, Glasgow, St Andrews and Strathclyde that is supported by the Scottish Funding Council.
EASTBIO - the BBSRC East of Scotland Bioscience Doctoral Training Partnership - is a partnership between the Universities of Aberdeen, Dundee, Edinburgh and St Andrews; the Scottish Universities Life Sciences Alliance; and the Scottish Universities Physics Alliance. The DTP scheme will provide students with excellent cross-disciplinary research training in line with the current BBSRC strategy for research support and training.
Retrieved from " http://2014.igem.org/Team:Dundee/Project/bdsf "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
