<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Dundee_Project_dsf"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Dundee/Project/dsf">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Dundee/Project/dsf&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Dundee/Project/dsf&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Dundee/Project/dsf&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Dundee/Project/dsf" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Dundee/Project/dsf</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Dundee 2014</TITLE></P><DIV class="navbar navbar-custom navbar-fixed-top"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON class="navbar-toggle" data-toggle="collapse" data-target="#navBarContent"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A href="/Team:Dundee" class="navbar-brand" style="padding-left:70px;">Dundee 2014</A></DIV><DIV class="collapse navbar-collapse" id="navBarContent"><UL class="nav navbar-nav"><LI><A href="#" id="info">Site Information</A></LI><LI class="dropdown"><A href="'#" class="dropdown-toggle" data-toggle="dropdown">Policy &amp; Practice</A></LI><LI class="nav-header "><I><U>Bulletin Board</U></I></LI><LI><A href="/Team:Dundee/policypractice/experts">Policy &amp; Practice</A></LI><LI><A href="/Team:Dundee/policypractice/clinic">Cystic Fibrosis Stories</A></LI><LI><A href="/Team:Dundee/policypractice/mt">Media and Talks</A></LI><LI><A href="/Team:Dundee/policypractice/fundraising">Fundraising</A></LI></UL><LI class="dropdown"><A href="'#" class="dropdown-toggle" data-toggle="dropdown">Project</A></LI><LI class="nav-header "><I><U>School</U></I></LI><LI class="nav-header ">Project</LI><LI><A href="/Team:Dundee/Project/CF">Cystic Fibrosis</A></LI><LI><A href="/Team:Dundee/Project/lungRanger">Polymicrobial Infection in the CF lung</A></LI><LI><A href="/Team:Dundee/Project/Aims">The Lung Ranger</A></LI><LI><A href="/Team:Dundee/Project/pqs">PQS Sensor</A></LI><LI><A href="/Team:Dundee/Project/PAI-1">PAI-1 Sensor</A></LI><LI><A href="/Team:Dundee/Project/bdsf">BDSF Sensor</A></LI><LI><A href="/Team:Dundee/Project/dsf">DSF Sensor</A></LI><LI class="nav-header">Modelling</LI><LI><A href="/Team:Dundee/Modeling/introduction">Introduction</A></LI><LI><A href="/Team:Dundee/Modeling/pqs">Modelling: PQS</A></LI><LI><A href="/Team:Dundee/Modeling/bdsf">Modelling: BDSF</A></LI><LI><A href="/Team:Dundee/Modeling/dsf">Modelling: DSF</A></LI><LI><A href="/Team:Dundee/Modeling/netlogo">NetLogo</A></LI><LI class="nav-header">Implementation</LI><LI><A href="/Team:Dundee/Implementation/introduction">L.A.S.S.O.</A></LI><LI><A href="/Team:Dundee/Implementation/methodology">Methodology</A></LI><LI><A href="/Team:Dundee/Implementation/lasso">Hardware</A></LI><LI><A href="/Team:Dundee/Implementation/lassoInterface">Software</A></LI><LI><A href="/Team:Dundee/Implementation/realWorld">Out in the real world</A></LI><LI><A href="/Team:Dundee/Implementation/financial">Quantified Impact</A></LI><LI><A href="/Team:Dundee/Implementation/future">Future</A></LI><LI class="dropdown"><A href="'#" class="dropdown-toggle" data-toggle="dropdown">Log Book</A></LI><LI class="nav-header "><I><U>Library</U></I></LI><LI><A href="/Team:Dundee/Safety">Safety &amp; Protocols</A></LI><LI><A href="/Team:Dundee/Notebook">Notebook</A></LI><LI><A href="/Team:Dundee/famAlbum">Family Album</A></LI><LI class="dropdown"><A href="'#" class="dropdown-toggle" data-toggle="dropdown">iGEM Community</A></LI><LI class="nav-header "><I><U>Saloon</U></I></LI><LI><A href="/Team:Dundee/Team">Team</A></LI><LI><A href="/Team:Dundee/Attributions">Attributions</A></LI><LI><A href="/Team:Dundee/collaborations">Collaboration</A></LI><LI><A href="/Team:Dundee/medals">Medal Fulfillment</A></LI><LI><A href="https://2014.igem.org/Team:Dundee/Parts">Parts</A></LI><LI class="dropdown"><A href="'#" class="dropdown-toggle" data-toggle="dropdown">Contact the team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=Dundee">Team Page</A></LI><LI><A href="https://www.facebook.com/igemdundee2014">Facebook</A></LI><LI><A href="https://twitter.com/DundeeiGEMTeam">Twitter</A></LI></DIV></DIV></DIV><DIV class="container"><DIV class="jumbotron"><H1><FONT size="14">The Diffusible Signalling Factor (DSF) Sensing System </FONT></H1></DIV><DIV class="row"><DIV class="col-col-sm-3 col-xs-3 col-md-3" id="nav"><UL class="nav list-group affix"><LI class="list-group-item"><A href="#0" class="">Initial Planning and Cloning Strategy</A></LI><LI class="list-group-item"><A href="#1" class="">Building the DSF Sensor</A></LI><LI class="list-group-item"><A href="#2" class="">Characterisation</A></LI></UL></DIV><DIV class="col-sm-9 col-xs-9"><H2 id="0">Initial Planning and Cloning Strategy</H2><DIV class="row"><DIV class="col-xs-12"><P>
Diffusible Signalling Factor (DSF) is a long chain cis-unsaturated fatty acid produced by bacteria of the <I>Xanthomonas</I> genus and the CF lung pathogen <I>Stenotrophomonas maltophilia</I><SUP>1,2</SUP>.  To build our synthetic DSF detector, we chose to reconstitute the well-characterised DSF signal perception system from <I>Xanthomonas campestris</I> in our <I>E. coli</I> chassis. As can be seen in Fig 1, the RpfC/RpfG system, which perceives and responds to DSF, is more complex than the average two component system. DSF is detected at the periplasmic side of the inner membrane by the <A href="http://parts.igem.org/Part:BBa_K1315002">RpfC </A>sensor protein, which autophosphorylates and then transfers its phospho group to the  cytoplasmic response regulator <A href="http://parts.igem.org/Part:BBa_K1315003">RpfG</A>. However, instead of acting as a direct transcriptional regulator, phosphorylated RpfG acts as a phosphodiesterase cleaving the second-messenger molecule cyclic di-guanosine monophosphate (c-di-GMP) into two GMP molecules<SUP>3,4</SUP>. This reduction in cellular c-di-GMP relieves inhibition of the transcriptional activator <A href="http://parts.igem.org/Part:BBa_K1315005">Clp</A>, which is then free to bind to the <A href="http://parts.igem.org/Part:BBa_K1315006"><I>manA</I></A> promoter and allow transcription of the downstream genes<SUP>5</SUP>. In our synthetic system (as shown in Fig 1) we fused the <I>manA</I> promoter to promoter-less <A href="http://parts.igem.org/Part:BBa_K562012">gfp</A> to give a fluorescent output.
</P></DIV></DIV><H2 id="1">Building the DSF Sensor</H2><DIV class="row"><DIV class="col-xs-12"><P>
Chromosomal DNA from <I>Xanthomonas campestris</I> pathovar <I>campestris</I> 8004 was kindly gifted to us by Drs Robert Ryan and Shi-Qi An from the Division of Molecular Microbiology in the College of Life Sciences at the University of Dundee. This was used as template for the amplification of the <I>rpfC</I>, <I>rpfG</I> and <I>clp</I> genes, and the <I>manA</I> promoter region.

The <I>manA</I> promoter region was cloned into the pSB1C3 plasmid (to give BioBrick <A href="http://parts.igem.org/Part:BBa_K1315006">BBa_K1315006</A>), and was then subcloned into pBluescript. Promoterless <I>gfp</I> was amplified (using <A href="http://parts.igem.org/Part:BBa_K562012">BBa_K562012</A> as a template), and was cloned into pBluescript downstream of the <I>manA</I> promoter. The <I>manA</I> promoter-<I>gfp</I> construct was then subcloned into pUniprom. A schematic of our completed construct can be seen below (Fig 2). 

To adhere to the iGEM rules and regulations, it was necessary to remove three specified illegal <I>Pst</I>I restriction sites and one <I>Eco</I>RI site present in <I>rpfC</I>. We also removed one illegal <I>Eco</I>RI site from <I>rpfG</I> and one illegal <I>PstI</I> site from <I>clp</I>. Each of the modified genes were individually cloned into the pSB1C3 plasmid (to give BioBricks <A href="http://parts.igem.org/Part:BBa_K1315002">BBa_K1315002</A>, <A href="http://parts.igem.org/Part:BBa_K1315003">BBa_K1315003</A>, <A href="http://parts.igem.org/Part:BBa_K1315005">BBa_K1315005</A>, respectively. The genes were then sequentially subcloned into the pUniprom vector that already harboured P<SUB><I>manA</I></SUB>-<I>gfp</I>. To facilitate immunochemistry we chose to supply each of RpfC, Clp and RpfG with an influenza virus hemagglutinin (HA) tag which can be detected with commercial antibodies. This tag was added to the C-terminus of each protein.

</P></DIV></DIV><DIV class="row"><DIV class="col-xs-12"><P>
The plasmid was verified by sequencing.

The completed construct was transformed into <I>E.coli</I> strain MC1061 as a chassis for our biosensor.  
</P></DIV></DIV><H2 id="2">Characterisation</H2><DIV class="row"><DIV class="col-xs-12"><P>
A series of western blots were performed to test for the sequential production of each of the HA-tagged proteins. We transformed our plasmid series (encoding RpfC-HA, RpfC-HA + Clp-HA and RpfC-HA +Clp-HA + RpfG-HA) into <I>E. coli</I> strain MC1061 and blotted for recombinant protein production. 
</P></DIV></DIV><DIV class="row"><DIV class="col-xs-12"><P>
As shown in Fig 3, successful production of RpfC-HA (expected mass 81kDa), RpfG-HA (expected mass of 43kDa) and Clp-HA (expected mass 27kDa) was observed. We then went on to test our DSF detector by incubating <I>E. coli</I> producing the three HA-tagged proteins with increasing concentrations of pure DSF (obtained from Sigma) and monitoring for production of GFP from the <I>manA</I> promoter by Western blotting. As shown in Fig 4A, we were able to specifically detect GFP production in <I>E. coli</I> cells harbouring our DSF-detection plasmid. However we noted that GFP was being produced in the absence of exogenous DSF addition. We reasoned that this could be due to either: (i) the endogenous activation of the <I>manA</I> by an <I>E. coli</I> transcriptional regulator or (ii) the presence of DSF or a DSF-like compound in the cells and/or growth medium. To distinguish between these two possibilities we introduced pUniprom carrying the P<I><SUB>manA</SUB></I>-<I>gfp</I> fusion alone (i.e. in the absence of the <I>rpfC</I>, <I>rpfG</I> and <I>clp</I> genes) into <I>E. coli</I> and blotted for GFP production. As shown in Fig 4B, GFP was produced by <I>E. coli</I> in the absence of the <I>X. campestris</I> DSF sensing proteins. We therefore conclude that the <I>manA</I> promoter is recognised by an <I>E. coli</I> transcription factor.
</P></DIV></DIV><DIV class="row"><DIV class="col-xs-12"><P>
For the DSF sensing system to work we need to eliminate the background activation of P<I><SUB>manA</SUB></I>-<I>gfp</I>. Since the <I>X. campestris</I> Clp protein is very similar to the <I>E. coli</I> cyclic AMP receptor protein (CRP) transcription factor (46% identity, 71% similarity over the whole length of the protein), we next tested whether the production of GFP was dependent upon CRP. To this end we used a <I>cep</I> deletion strain from the <I>E. coli</I> Keio mutant collection<SUP>6</SUP> and compared GFP production in this strain background with the cognate wild type strain (Fig 5). However GFP was still produced (although possibly at a lower level) than in the <I>cep<SUP>+</SUP></I> background.
</P></DIV></DIV><DIV class="row"><DIV class="col-xs-12"><P>
We are continuing to investigate this issue by screening other <I>E. coli</I> mutants carrying deletions in genes coding for transcription factors which show similarity to Clp, for example the anaerobic regulator, FNR. We anticipate that it may be necessary to make combinatorial mutations to completely eliminate exogenous GFP production.

The following parts were deposited as BioBricks:
<TABLE class="table"><THEAD><TR><TH>Part</TH><TH>Description</TH><TH>Registry</TH></TR></THEAD><TBODY><TR><TD><I>rpfC</I></TD><TD>DSF Sensor Kinase</TD><TD><A href="http://parts.igem.org/Part:BBa_K1315002">BBa_K1315002</A></TD></TR><TR><TD><I>rpfG</I></TD><TD>Two-component system regulator activated by sensor kinase</TD><TD><A href="http://parts.igem.org/Part:BBa_K1315003">BBa_K1315003</A></TD></TR><TR><TD><I>clp</I></TD><TD>Clp Transcriptional activator inhibited by c-di-GMP</TD><TD><A href="http://parts.igem.org/Part:BBa_K1315005">BBa_K1315005</A></TD></TR><TR><TD><I>manA</I></TD><TD><I>manA</I> gene promoter activated by Clp</TD><TD><A href="http://parts.igem.org/Part:BBa_K1315006">BBa_K1315006</A></TD></TR></TBODY></TABLE></P></DIV></DIV><DIV class="row"><DIV class="col-12"><DIV id="ref"><H3>References</H3><P><SUP>1</SUP>Slater, H. et al. (2000) Mol Microbiol 38, 986-1003.  <SUP>2</SUP>Ryan, R.P. et al. (2009) Nat Rev Microbiol 7, 514-525. <SUP>3</SUP>Ryan, R.P. et al. (2006) Proc Natl Acad Sci USA 103, 6712-6717. <SUP>4</SUP>Ryan, R.P. et al. (2010) Proc Natl Acad Sci USA 107, 5989-5994.<SUP>5</SUP>Lu, X.-H. et al. (2012) PLOS ONE 7, e52646.   <SUP>6</SUP>Baba, T. et al. (2006) Mol Syst Bio 2, 2006.0008. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="btn-group btn-group-justified"><DIV class="btn-group"><A href="bdsf" class="btn btn-default toLesson">Previous Lesson: BDSF Sensor</A></DIV><DIV class="btn-group"><A href="https://2014.igem.org/Team:Dundee/inside/school" class="btn btn-default toClass">Back to School</A></DIV><DIV class="btn-group"><A href="https://2014.igem.org/Team:Dundee/Modeling/introduction" class="btn btn-default toLesson">Next Lesson: Modelling: Introduction</A></DIV></DIV></DIV><DIV class="modal static" id="welcomeNote" role="dialog" data-backdrop="static"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-hidden="true" id="closeNote">×</BUTTON><H4 class="modal-title" id="myModalLabel">Welcome to Dundee's 2014 Wiki (<B>Site Information</B>)</H4></DIV><DIV class="modal-body"><UL class="nav nav-tabs"><LI class="active"><A href="#ab" data-toggle="tab">Abstract</A></LI><LI><A href="#instr" data-toggle="tab">Navigation Instructions</A></LI><LI><A href="#spon" data-toggle="tab">Sponsors</A></LI></UL><DIV class="tab-content"><DIV class="tab-pane active" id="ab">  
   Cystic Fibrosis is a genetic disease that results in the accumulation of thick mucus in the epithelial linings of the entire body, particularly the respiratory tract. Over the course of a patient’s life, the mucus-lined lung epithelium becomes repeatedly infected with pathogenic bacteria that stimulate an immune response; leading to inflammation, tissue damage, and ultimately, respiratory failure.  

The microflora of the Cystic Fibrosis lung changes over time. In childhood, the major coloniser is <I>Staphylococcus aureus</I>, but as the patient matures other bacterial pathogens infect. The later-dominating pathogens, <I>Pseudomonas aeruginosa</I> and <I>Burkholderia cepacia</I>, are very difficult to eradicate and are associated with chronic decline in lung function. <I>Burkholderia</I> is so infectious that patients have to be isolated from one another, and can be denied lung transplants due to the persistence of this bacterium. 

It is important, therefore, for medics to monitor and identify the levels of bacteria within the respiratory tract of a Cystic Fibrosis patient. Currently, patients must provide sputum samples and identification of bacteria takes between 72 hours and 2 weeks, by which point the bacteria can be in an antibiotic resistant state. 

The Dundee 2014 iGEM project is focused on designing and testing a device that will rapidly and non-invasively identify the bacteria colonising a Cystic Fibrosis patient. Three biosensors will be developed that recognise external signal molecules produced by key bacteria, all of which are known to be in sputum samples of Cystic Fibrosis patients. A quinolone signal (PQS) is produced by <I>Pseudomonas aeruginosa</I>, a Diffusible Signal Factor (DSF) is produced by <I>Stenotrophomonas maltophilia</I>, an organism that is also associated with Cystic Fibrosis lung infection in adults, and BDSF is a related, but chemically distinct molecule that is produced by <I>Burkholderia cepacia</I>. 

By engineering signal recognition and signal transduction proteins from these biological systems the Dundee iGEM team want to produce a portable electronic device that will identify infection outside of the clinic, so that patients do not have to travel great distances, and can be used to help health professionals make informed and rapid decisions on antibiotic treatments. 

Read this again by clicking on <B>Site Information</B> in the navigation bar!      
   </DIV><DIV class="tab-pane" id="instr"><DIV class="row"><DIV class="col-sm-9 col-md-9"><H3>Bulletin Board (Policy &amp; Practice)</H3><P>
     Come look at the poster explaining how we involved the local and national Cystic Fibrosis community in our project, how we listened to people on the front line of the disease and how they affected our project.
    </P></DIV></DIV><DIV class="row"><DIV class="col-sm-9 col-md-9"><H3>School (Project)</H3><P>
       Come in and learn about all the parts that make The Lung Ranger. The lab work that we carried out over the    
        summer is explained here in detail, along with the modelling and implementation. 
    </P></DIV></DIV><DIV class="row"><DIV class="col-sm-9 col-md-9"><H3>Library (Log Book)</H3><P>
        Read about the experience the team has had over the summer. A week by week description of our work, how we carried out that work and some photos of our time together.
    </P></DIV></DIV><DIV class="row"><DIV class="col-sm-9 col-md-9"><H3>Saloon (iGEM community)</H3><P>
Come on in, take a seat and learn how we have interacted with the larger iGEM community. Who the team are, who helped them and how we helped others.
    </P></DIV></DIV></DIV><DIV class="tab-pane" id="spon"><DIV class="row"><DIV class="col-xs-4"><P><B>College of Life Sciences</B>
With more than 1100 staff and research students and external funding of around £60 million per year, the College of Life Sciences is one of the largest and most productive research institutes in Europe. This reputation is genuinely global and is reflected in the fact that researchers in our laboratories represent no fewer than 62 different nationalities
</P></DIV><DIV class="col-xs-4"><P><B>Division of Mathematics</B>
A small friendly Division which offers a lively and modern programme of study at both undergraduate and postgraduate level.We have research groups in Mathematical Biology, Numerical Analysis and Scientific Computing, Magnetohydrodynamics and Applied Analysis.
</P></DIV><DIV class="col-xs-4"><P><B>School of Computing</B>
With research areas in Human Centred Computing, Intelligent Systems, Space Technologies and Theory of Computation, the department has an emphasis on development. Furthermore, this department teaches popular undergraduate programs in Applied Computing and Computing Science.
</P></DIV></DIV><DIV class="row"><DIV class="col-xs-6"><P>
We are a global charitable foundation dedicated to achieving extraordinary improvements in human and animal health. We support the brightest minds in biomedical research and the medical humanities. Our breadth of support includes public engagement, education and the application of research to improve health. We are independent of both political and commercial interests.
</P></DIV></DIV><DIV class="row"><DIV class="col-xs-6">
BBSRC has a unique and central place in supporting the UK’s world-leading position in bioscience. We are an investor in research and training, with the aim of furthering scientific knowledge, to promote economic growth, wealth and job creation and to improve quality of life in the UK and beyond.
</DIV></DIV><DIV class="row"><DIV class="col-xs-6">
The Biochemical Society exists for the advancement of the molecular and cellular biosciences, both as an academic discipline and to promote its impact on areas of science including biotechnology, agriculture, and medicine.
</DIV></DIV><DIV class="row"><DIV class="col-xs-6">
The Society for General Microbiology is a membership organisation for scientists who work in all areas of microbiology. The Society promotes the understanding of microbiology to a diverse range of stakeholders, including policy-makers, students, teachers, journalists and the wider public, through a comprehensive framework of communication activities and resources.
</DIV></DIV><DIV class="row"><DIV class="col-xs-6">
Established in 2007, Scottish Universities Life Sciences Alliance (SULSA) is a research pooling partnership between the Universities of Aberdeen, Dundee, Edinburgh, Glasgow, St Andrews and Strathclyde that is supported by the Scottish Funding Council.
</DIV></DIV><DIV class="row"><DIV class="col-xs-6">
EASTBIO - the BBSRC East of Scotland Bioscience Doctoral Training Partnership - is a partnership between the Universities of Aberdeen, Dundee, Edinburgh and St Andrews; the Scottish Universities Life Sciences Alliance; and the Scottish Universities Physics Alliance. The DTP scheme will provide students with excellent cross-disciplinary research training in line with the current BBSRC strategy for research support and training.
</DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Dundee/Project/dsf">http://2014.igem.org/Team:Dundee/Project/dsf</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Dundee/Project/dsf" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Dundee/Project/dsf" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Dundee/Project/dsf&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Dundee/Project/dsf&amp;oldid=361446" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>