<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Dundee_Project_pqs"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Dundee/Project/pqs">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Dundee/Project/pqs&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Dundee/Project/pqs&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Dundee/Project/pqs&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Dundee/Project/pqs" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Dundee/Project/pqs</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Dundee 2014</TITLE></P><DIV class="navbar navbar-custom navbar-fixed-top"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON class="navbar-toggle" data-toggle="collapse" data-target="#navBarContent"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A href="/Team:Dundee" class="navbar-brand" style="padding-left:70px;">Dundee 2014</A></DIV><DIV class="collapse navbar-collapse" id="navBarContent"><UL class="nav navbar-nav"><LI><A href="#" id="info">Site Information</A></LI><LI class="dropdown"><A href="'#" class="dropdown-toggle" data-toggle="dropdown">Policy &amp; Practice</A></LI><LI class="nav-header "><I><U>Bulletin Board</U></I></LI><LI><A href="/Team:Dundee/policypractice/experts">Policy &amp; Practice</A></LI><LI><A href="/Team:Dundee/policypractice/clinic">Cystic Fibrosis Stories</A></LI><LI><A href="/Team:Dundee/policypractice/mt">Media and Talks</A></LI><LI><A href="/Team:Dundee/policypractice/fundraising">Fundraising</A></LI></UL><LI class="dropdown"><A href="'#" class="dropdown-toggle" data-toggle="dropdown">Project</A></LI><LI class="nav-header "><I><U>School</U></I></LI><LI class="nav-header ">Project</LI><LI><A href="/Team:Dundee/Project/CF">Cystic Fibrosis</A></LI><LI><A href="/Team:Dundee/Project/lungRanger">Polymicrobial Infection in the CF lung</A></LI><LI><A href="/Team:Dundee/Project/Aims">The Lung Ranger</A></LI><LI><A href="/Team:Dundee/Project/pqs">PQS Sensor</A></LI><LI><A href="/Team:Dundee/Project/PAI-1">PAI-1 Sensor</A></LI><LI><A href="/Team:Dundee/Project/bdsf">BDSF Sensor</A></LI><LI><A href="/Team:Dundee/Project/dsf">DSF Sensor</A></LI><LI class="nav-header">Modelling</LI><LI><A href="/Team:Dundee/Modeling/introduction">Introduction</A></LI><LI><A href="/Team:Dundee/Modeling/pqs">Modelling: PQS</A></LI><LI><A href="/Team:Dundee/Modeling/bdsf">Modelling: BDSF</A></LI><LI><A href="/Team:Dundee/Modeling/dsf">Modelling: DSF</A></LI><LI><A href="/Team:Dundee/Modeling/netlogo">NetLogo</A></LI><LI class="nav-header">Implementation</LI><LI><A href="/Team:Dundee/Implementation/introduction">L.A.S.S.O.</A></LI><LI><A href="/Team:Dundee/Implementation/methodology">Methodology</A></LI><LI><A href="/Team:Dundee/Implementation/lasso">Hardware</A></LI><LI><A href="/Team:Dundee/Implementation/lassoInterface">Software</A></LI><LI><A href="/Team:Dundee/Implementation/realWorld">Out in the real world</A></LI><LI><A href="/Team:Dundee/Implementation/financial">Quantified Impact</A></LI><LI><A href="/Team:Dundee/Implementation/future">Future</A></LI><LI class="dropdown"><A href="'#" class="dropdown-toggle" data-toggle="dropdown">Log Book</A></LI><LI class="nav-header "><I><U>Library</U></I></LI><LI><A href="/Team:Dundee/Safety">Safety &amp; Protocols</A></LI><LI><A href="/Team:Dundee/Notebook">Notebook</A></LI><LI><A href="/Team:Dundee/famAlbum">Family Album</A></LI><LI class="dropdown"><A href="'#" class="dropdown-toggle" data-toggle="dropdown">iGEM Community</A></LI><LI class="nav-header "><I><U>Saloon</U></I></LI><LI><A href="/Team:Dundee/Team">Team</A></LI><LI><A href="/Team:Dundee/Attributions">Attributions</A></LI><LI><A href="/Team:Dundee/collaborations">Collaboration</A></LI><LI><A href="/Team:Dundee/medals">Medal Fulfillment</A></LI><LI><A href="https://2014.igem.org/Team:Dundee/Parts">Parts</A></LI><LI class="dropdown"><A href="'#" class="dropdown-toggle" data-toggle="dropdown">Contact the team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=Dundee">Team Page</A></LI><LI><A href="https://www.facebook.com/igemdundee2014">Facebook</A></LI><LI><A href="https://twitter.com/DundeeiGEMTeam">Twitter</A></LI></DIV></DIV></DIV><DIV class="container"><DIV class="jumbotron"><H1><FONT size="14">The <I>Pseudomonas</I> Quinolone Signal (PQS) Sensing System</FONT></H1></DIV><DIV class="row"><DIV class="col-col-sm-3 col-xs-3 col-md-3" id="nav"><UL class="nav list-group affix"><LI class="list-group-item"><A href="#0" class="">Initial Planning and Cloning Strategy</A></LI><LI class="list-group-item"><A href="#1" class="">Building the PQS Sensor</A></LI><LI class="list-group-item"><A href="#2" class="">Characterisation</A></LI></UL></DIV><DIV class="col-sm-9 col-xs-9"><DIV class="row"><DIV class="col-xs-12"><H2 id="0">Initial Planning and Cloning Strategy</H2><P><I>Pseudomonas</I> quinolone signal (2-heptyl-3-hydroxy-4-quinolone) is a quorum-sensing molecule produced by <I>Pseudomonas aeruginosa</I>, which regulates the expression of genes involved in biofilm development and virulence<SUP>1</SUP>. Expression of these traits is mediated through the LysR-type transcriptional regulator, <A href="http://parts.igem.org/Part:BBa_K1157001">PqsR</A>. Bioinformatic analysis of the PqsR sequence predicted that it is a soluble protein. Fractionation experiments of <I>P. aeruginosa</I> cells has shown that PqsR is primarily associated with the inner membrane<SUP>2</SUP>. It is not clear however, whether this is through protein-lipid or protein-protein interactions with membrane lipids or with a yet unidentified integral inner membrane protein, respectively. In the presence of PQS, PqsR interacts with the promoter region of the <A href="http://parts.igem.org/Part:BBa_K1315001"><I>pqsA</I></A> operon, allowing transcription of the downstream genes<SUP>3</SUP>. We have engineered<I> E. coli</I> to express this signal transduction system for the detection of PQS (as shown in Fig 1), with a promoter-less <A href="http://parts.igem.org/Part:BBa_K562011"> mCherry</A> coding gene fused to the <I>pqsA</I> promoter to give a fluorescent output.


</P></DIV></DIV><H2 id="1">Building the PQS Sensor</H2><DIV class="row"><DIV class="col-xs-12"><P>
Chromosomal DNA from <I>Pseudomonas aeruginosa</I> PA01 strain was kindly gifted to us by Drs Robert Ryan and Shi-Qi An from the Division of Molecular Microbiology in the College of Life sciences at the University of Dundee. This was used as a template for the amplification of the <I>pqsA</I> promoter gene. 


The <I>pqsA</I> promoter region was cloned into the pSB1C3 plasmid (BioBrick <A href="http://parts.igem.org/Part:BBa_K1315001">BBa_K1315001</A>), and was then subcloned into pBluescript. Promoterless mCherry was amplified (template used: <A href="http://parts.igem.org/Part:BBa_K562011"> BBa_K562012</A>), and was cloned into pBluescript downstream of the <I>pqsA</I> promoter. The P<I><SUB>pqsA</SUB></I>-mCherry construct was then subcloned into pUniprom. A schematic of our completed construct can be seen in Fig 2. 

The <I>pqsR</I> gene (present in <A href="http://parts.igem.org/Part:BBa_K1157001">BBa_K1315001</A>) which was designed and cloned by Lin Ang Chieh from iGEM13-NTU-Taida was used as a template for the amplification of <I>pqsR</I> with an 
influenza virus haemagglutinin (HA) tag coding sequence, which can be detected by commercially available antibodies. The HA-tag was fused to the C-terminus of the protein to allow visualisation by immunohistochemistry. 
</P></DIV></DIV><DIV class="row"><DIV class="col-xs-12">
The plasmid was verified by sequencing. 

The completed construct was transformed into <I>E. coli </I>strain MC1061 as a chassis for our biosensor.
</DIV></DIV><H2 id="2">Characterisation</H2><DIV class="row"><DIV class="col-xs-12"><P>
Initially, western blots were undertaken to test for the production of PqsR-HA. We transformed our plasmid encoding PqsR-HA into <I>E. coli </I>strain MC1061 and blotted for recombinantely expressed protein. An overnight culture of the cells was lysed and proteins separated by SDS-PAGE in a 12% acrylamide gel. Anti-HA antibodies linked to horseradish peroxidase were used for detection of PqsR. Fig 3A shows the expression of PqsR within our system. 
</P></DIV></DIV><DIV class="row"><DIV class="col-xs-12"><P>

As shown in Fig 3, successful production of PqsR-HA (expected mass 38 kDa) was observed. With the PqsR regulator being produced we could begin to test for a response to PQS. To test how the system would respond to PQS, cells containing the construct were cultured in LB medium and spiked with synthetic PQS (in 100% DMSO) at finals concentrations of 50 and 500 μM. A western blot with anti-mCherry antibodies was performed on the treated cells alongside a control (DMSO). Two further controls - MC1061 cells harbouring the empty pUniprom vector (mCherry negative control) and <A href="http://parts.igem.org/Part:BBa_K562011"> BBa_K562011</A> (mCherry positive control) were also included.
</P></DIV></DIV><DIV class="row"><DIV class="col-xs-12"><P>
As shown in Fig 4, incubation of cells with synthetic PQS, did not induce expression of mCherry. It is not clear why the biosensor did not respond. One possibility we considered is that PqsR is not correctly localized to the membrane in <I>E. coli</I>. We therefore fractionated <I>E. coli</I> cells containing PqsR-HA into soluble and membrane fractions. However, as shown in Fig 5, all of the detectable PqsR-HA was found in the membranes as it is in the native organism.
</P></DIV></DIV><DIV class="row"><DIV class="col-xs-12"><P>
We discussed the results of the PQS biosensor extensively with the
<A href="https://2014.igem.org/Team:Dundee/Modeling/pqs">modelling team</A>. They suggested increasing the number of promoters (by at least 10-fold). This should allow for sufficient production of mCherry, which can then be detected. We are in the process of building this new expression system. 
</P>
The following parts were deposited as BioBricks:
<TABLE class="table"><THEAD><TR><TH>Part</TH><TH>Description</TH><TH>Registry</TH></TR></THEAD><TBODY><TR><TD><I>pqsA</I></TD><TD>Promoter</TD><TD><A href="http://parts.igem.org/Part:BBa_K1315001">BBa_K1315001</A></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="row"><DIV class="col-12"><DIV id="ref"><H3>References</H3><P><SUP>1</SUP>Reen, F.J. et al. (2011) FEMS Microbiol Ecol 77, 413-428. <SUP>2</SUP>Cao, H. et al. (2001) Proc Natl Acad Sci USA 98, 14613–14618.<SUP>3</SUP>Dekimpe, V. et al. (2009) Microbiology 155, 712-723.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row" id="btnRow"><DIV class="btn-group btn-group-justified"><DIV class="btn-group"><A href="Aims" class="btn btn-default toLesson">Previous Lesson: The Lung Ranger</A></DIV><DIV class="btn-group"><A href="https://2014.igem.org/Team:Dundee/inside/school" class="btn btn-default toClass">Back to School</A></DIV><DIV class="btn-group"><A href="PAI-1" class="btn btn-default toLesson">Next Lesson: PAI-1 Sensor</A></DIV></DIV></DIV></DIV><DIV class="modal static" id="welcomeNote" role="dialog" data-backdrop="static"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-hidden="true" id="closeNote">×</BUTTON><H4 class="modal-title" id="myModalLabel">Welcome to Dundee's 2014 Wiki (<B>Site Information</B>)</H4></DIV><DIV class="modal-body"><UL class="nav nav-tabs"><LI class="active"><A href="#ab" data-toggle="tab">Abstract</A></LI><LI><A href="#instr" data-toggle="tab">Navigation Instructions</A></LI><LI><A href="#spon" data-toggle="tab">Sponsors</A></LI></UL><DIV class="tab-content"><DIV class="tab-pane active" id="ab">  
   Cystic Fibrosis is a genetic disease that results in the accumulation of thick mucus in the epithelial linings of the entire body, particularly the respiratory tract. Over the course of a patient’s life, the mucus-lined lung epithelium becomes repeatedly infected with pathogenic bacteria that stimulate an immune response; leading to inflammation, tissue damage, and ultimately, respiratory failure.  

The microflora of the Cystic Fibrosis lung changes over time. In childhood, the major coloniser is <I>Staphylococcus aureus</I>, but as the patient matures other bacterial pathogens infect. The later-dominating pathogens, <I>Pseudomonas aeruginosa</I> and <I>Burkholderia cepacia</I>, are very difficult to eradicate and are associated with chronic decline in lung function. <I>Burkholderia</I> is so infectious that patients have to be isolated from one another, and can be denied lung transplants due to the persistence of this bacterium. 

It is important, therefore, for medics to monitor and identify the levels of bacteria within the respiratory tract of a Cystic Fibrosis patient. Currently, patients must provide sputum samples and identification of bacteria takes between 72 hours and 2 weeks, by which point the bacteria can be in an antibiotic resistant state. 

The Dundee 2014 iGEM project is focused on designing and testing a device that will rapidly and non-invasively identify the bacteria colonising a Cystic Fibrosis patient. Three biosensors will be developed that recognise external signal molecules produced by key bacteria, all of which are known to be in sputum samples of Cystic Fibrosis patients. A quinolone signal (PQS) is produced by <I>Pseudomonas aeruginosa</I>, a Diffusible Signal Factor (DSF) is produced by <I>Stenotrophomonas maltophilia</I>, an organism that is also associated with Cystic Fibrosis lung infection in adults, and BDSF is a related, but chemically distinct molecule that is produced by <I>Burkholderia cepacia</I>. 

By engineering signal recognition and signal transduction proteins from these biological systems the Dundee iGEM team want to produce a portable electronic device that will identify infection outside of the clinic, so that patients do not have to travel great distances, and can be used to help health professionals make informed and rapid decisions on antibiotic treatments. 

Read this again by clicking on <B>Site Information</B> in the navigation bar!      
   </DIV><DIV class="tab-pane" id="instr"><DIV class="row"><DIV class="col-sm-9 col-md-9"><H3>Bulletin Board (Policy &amp; Practice)</H3><P>
     Come look at the poster explaining how we involved the local and national Cystic Fibrosis community in our project, how we listened to people on the front line of the disease and how they affected our project.
    </P></DIV></DIV><DIV class="row"><DIV class="col-sm-9 col-md-9"><H3>School (Project)</H3><P>
       Come in and learn about all the parts that make The Lung Ranger. The lab work that we carried out over the    
        summer is explained here in detail, along with the modelling and implementation. 
    </P></DIV></DIV><DIV class="row"><DIV class="col-sm-9 col-md-9"><H3>Library (Log Book)</H3><P>
        Read about the experience the team has had over the summer. A week by week description of our work, how we carried out that work and some photos of our time together.
    </P></DIV></DIV><DIV class="row"><DIV class="col-sm-9 col-md-9"><H3>Saloon (iGEM community)</H3><P>
Come on in, take a seat and learn how we have interacted with the larger iGEM community. Who the team are, who helped them and how we helped others.
    </P></DIV></DIV></DIV><DIV class="tab-pane" id="spon"><DIV class="row"><DIV class="col-xs-4"><P><B>College of Life Sciences</B>
With more than 1100 staff and research students and external funding of around £60 million per year, the College of Life Sciences is one of the largest and most productive research institutes in Europe. This reputation is genuinely global and is reflected in the fact that researchers in our laboratories represent no fewer than 62 different nationalities
</P></DIV><DIV class="col-xs-4"><P><B>Division of Mathematics</B>
A small friendly Division which offers a lively and modern programme of study at both undergraduate and postgraduate level.We have research groups in Mathematical Biology, Numerical Analysis and Scientific Computing, Magnetohydrodynamics and Applied Analysis.
</P></DIV><DIV class="col-xs-4"><P><B>School of Computing</B>
With research areas in Human Centred Computing, Intelligent Systems, Space Technologies and Theory of Computation, the department has an emphasis on development. Furthermore, this department teaches popular undergraduate programs in Applied Computing and Computing Science.
</P></DIV></DIV><DIV class="row"><DIV class="col-xs-6"><P>
We are a global charitable foundation dedicated to achieving extraordinary improvements in human and animal health. We support the brightest minds in biomedical research and the medical humanities. Our breadth of support includes public engagement, education and the application of research to improve health. We are independent of both political and commercial interests.
</P></DIV></DIV><DIV class="row"><DIV class="col-xs-6">
BBSRC has a unique and central place in supporting the UK’s world-leading position in bioscience. We are an investor in research and training, with the aim of furthering scientific knowledge, to promote economic growth, wealth and job creation and to improve quality of life in the UK and beyond.
</DIV></DIV><DIV class="row"><DIV class="col-xs-6">
The Biochemical Society exists for the advancement of the molecular and cellular biosciences, both as an academic discipline and to promote its impact on areas of science including biotechnology, agriculture, and medicine.
</DIV></DIV><DIV class="row"><DIV class="col-xs-6">
The Society for General Microbiology is a membership organisation for scientists who work in all areas of microbiology. The Society promotes the understanding of microbiology to a diverse range of stakeholders, including policy-makers, students, teachers, journalists and the wider public, through a comprehensive framework of communication activities and resources.
</DIV></DIV><DIV class="row"><DIV class="col-xs-6">
Established in 2007, Scottish Universities Life Sciences Alliance (SULSA) is a research pooling partnership between the Universities of Aberdeen, Dundee, Edinburgh, Glasgow, St Andrews and Strathclyde that is supported by the Scottish Funding Council.
</DIV></DIV><DIV class="row"><DIV class="col-xs-6">
EASTBIO - the BBSRC East of Scotland Bioscience Doctoral Training Partnership - is a partnership between the Universities of Aberdeen, Dundee, Edinburgh and St Andrews; the Scottish Universities Life Sciences Alliance; and the Scottish Universities Physics Alliance. The DTP scheme will provide students with excellent cross-disciplinary research training in line with the current BBSRC strategy for research support and training.
</DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Dundee/Project/pqs">http://2014.igem.org/Team:Dundee/Project/pqs</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Dundee/Project/pqs" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Dundee/Project/pqs" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Dundee/Project/pqs&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Dundee/Project/pqs&amp;oldid=361292" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>