<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Exeter_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Exeter/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><NAV id="navbar-top" class="navbar navbar-expand justify-content-center fixed-top"><UL class="nav justify-content-center"><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="" id="projectDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Project</A><DIV class="dropdown-menu" aria-labelledby="projectDropdown"><A class="dropdown-item" href="https://2017.igem.org/Team:Exeter/Description">Description</A><A class="dropdown-item" href="https://2017.igem.org/Team:Exeter/HP/Fieldtrips">Field Trip</A><A class="dropdown-item" href="https://2017.igem.org/Team:Exeter/Design">Design</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="" id="labDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Lab</A><DIV class="dropdown-menu" aria-labelledby="labDropdown"><A class="dropdown-item" href="https://2017.igem.org/Team:Exeter/Lab_Introduction">Introduction</A><A class="dropdown-item" href="https://2017.igem.org/Team:Exeter/Experiments">Materials &amp; Methods</A><A class="dropdown-item" href="https://2017.igem.org/Team:Exeter/Results">Results</A><A class="dropdown-item" href="https://2017.igem.org/Team:Exeter/Lab_Conclusion">Conclusions</A><A class="dropdown-item" href="https://2017.igem.org/Team:Exeter/Collaborations">Collaborations</A><A class="dropdown-item" href="https://2017.igem.org/Team:Exeter/InterLab">Interlab</A><A class="dropdown-item" href="https://2017.igem.org/Team:Exeter/Notebook">Notebook</A><A class="dropdown-item" href="https://2017.igem.org/Team:Exeter/Safety">Safety</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="" id="partsDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Parts</A><DIV class="dropdown-menu" aria-labelledby="partsDropdown"><A class="dropdown-item" href="https://2017.igem.org/Team:Exeter/Basic_Part">Basic Parts</A><A class="dropdown-item" href="https://2017.igem.org/Team:Exeter/Composite_Part">Composite Parts</A></DIV></LI><LI class="nav-item"><A class="nav-link" href="https://2017.igem.org/Team:Exeter/Model">Model</A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="" id="filterDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Applied Design</A><DIV class="dropdown-menu" aria-labelledby="filterDropdown"><A class="dropdown-item" href="https://2017.igem.org/Team:Exeter/Applied_Design">Introduction</A><A class="dropdown-item" href="https://2017.igem.org/Team:Exeter/Current_Methods">Current Methods</A><A class="dropdown-item" href="https://2017.igem.org/Team:Exeter/Hydrocyclone">Hydrocyclone</A><A class="dropdown-item" href="https://2017.igem.org/Team:Exeter/MBR">Metal Binding Reactor</A><A class="dropdown-item" href="https://2017.igem.org/Team:Exeter/Bio-security">Biosecurity</A><A class="dropdown-item" href="https://2017.igem.org/Team:Exeter/Filter_Stakeholders">Stakeholders' Influence</A><A class="dropdown-item" href="https://2017.igem.org/Team:Exeter/Future">Future of our Filter</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="" id="hpDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Human Practices</A><DIV class="dropdown-menu" aria-labelledby="hpDropdown"><A class="dropdown-item" href="https://2017.igem.org/Team:Exeter/HP/Intro">Introduction</A><A class="dropdown-item" href="https://2017.igem.org/Team:Exeter/HP/Background">Background</A><A class="dropdown-item" href="https://2017.igem.org/Team:Exeter/HP/Silver">Silver</A><A class="dropdown-item" href="https://2017.igem.org/Team:Exeter/HP/Gold_Integrated">Integrated-Gold</A><A class="dropdown-item" href="https://2017.igem.org/Team:Exeter/Engagement">Education &amp; Public Engagement</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="" id="teamDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Team</A><DIV class="dropdown-menu" aria-labelledby="teamDropdown"><A class="dropdown-item" href="https://2017.igem.org/Team:Exeter/Team">Our Members</A><A class="dropdown-item" href="https://2017.igem.org/Team:Exeter/Attributions">Attributions</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="" id="medalsDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Medals &amp; Awards</A><DIV class="dropdown-menu" aria-labelledby="medalsDropdown"><A class="dropdown-item" href="https://2017.igem.org/Team:Exeter/Medals">Medals</A><A class="dropdown-item" href="https://2017.igem.org/Team:Exeter/Awards">Awards</A></DIV></LI></UL></NAV></P><DIV class="container-fluid"><DIV class="row"><DIV class="col-2"><NAV id="navbar-page" class="navbar"><NAV class="nav nav-pills flex-column"><A class="nav-link" href="#h1">Chassis  determination</A><A class="nav-link" href="#h2">Modified Pili  Expression</A><A class="nav-link" href="#h3">Applied Design </A><A class="nav-link" href="#pageHeader">back to top</A></NAV></NAV></DIV><DIV class="col-10" id="pageContent"><DIV><H1 id="pageHeader">Results</H1><H2 id="h1">Chassis determination </H2><P> As we wanted to express modified pili in <I>E. coli</I> we ideally needed a chassis that did not natively produce pili. The genome of <I>E. coli</I> MG1655 is annotated as containing the <I> fim operon </I> (Accession NC_000913) and was therefore proposed to act as a positive control in all experiments. For modified pili production two strains were initially proposed: <I>E. coli</I> Top10 (Invitrogen) and <I>E. coli </I> DH5α (NEB).  Top10  is a derivative of DH10B, the genome of which (Accession CP000948) is lacking an annotated <I>fim operon</I> whereas there is no published genome for DH5α. Therefore it was important to verify whether these two strains were lacking the pili producing <I>fim operon</I>.</P><P>Using a <A href="https://2017.igem.org/Team:Exeter/Experiments">genomic extraction and PCR </A> we attempted to isolate and amplify three known genes of the <I>fim operon </I> in MG1655 and DH5α. We ran the products of these two preparations down an agarose gel. The results of the <A href="https://2017.igem.org/Team:Exeter/Experiments">electrophoresis</A> procedure showed the same bands in the MG1655 as in the DH5α, representative of three genes in the operon. This demonstrates DH5α's possession of the operon and therefore the unsuitability of DH5α as the chassis for our plasmid inserts. </P><FIGURE class="border border-dark rounded"><FIGCAPTION style="text-align:justify"><B>Figure 1:</B> This gel and the gel in Figure 2 show the genetic analysis of three strains of <I>E. coli</I> : MG1655, DH5α and Top10. In both cases, three known genes of the <I>fim operon </I> have been amplified. The wells 0-8 on this gel contain the following: hyperladder 1, DH5α FimD, MG1655 FimD, Negative control (water) FimD, DH5α FimH, MG1655 FimH, Negative FimH, DH5α FimA, MG1655 FimA, Negative FimA. </FIGCAPTION></FIGURE><FIGURE class="border border-dark rounded"><FIGCAPTION style="text-align:justify"><B>Figure 2:</B> The wells in this gel contain the following, 0-9: hyperladder 1, Top10 FimA, Top10 FimD, Top10 FimH, MG1655 FimA, Negative control(water) FimD, MG1655 FimH, Negative FimA, MG1655 FimD and negative control FimH. Evidently, MG1655 and DH5α contain the three genes and therefore produce pili natively. Meanwhile Top10 lacks all three and so does not produce pili.</FIGCAPTION></FIGURE><P> The other strain identified in the literature as potentially lacking the <I> fim operon </I> was Top10. A similar procedure was used as above, involving a genomic extraction and an enzymatic amplification of the genes of interest. Once more, the MG1655 strain was used as a positive control. The PCR products were run down an agarose gel and the results showed that Top10 does not contain the three genes of interest and so does not have the operon or natively produce pili. </P><P>Overall, these results confirmed that MG1655 would be ideal as a positive pili producing control and, that Top10 would be a suitable chassis for our modified pili whereas DH5α would not. In addition two further chassis strains were chosen, the <I>E. coli</I> K-12 FimB knockout strain (JW4275-1 from the Keio collection Baba et al) that the Harvard 2015 iGEM team demonstrated did not produce pili and the <I>E. coli</I> K12 FimH knockout strain (JW4283-3 from the Keio collection Baba et al) which is deficient in the gene that we specifically wanted to modify. Finally, <I>E. coli </I> BL21(DE3) was used to allow initial protein expression to be investigated using the T7 promoter.</P><H2 id="h2">Modified Pili Expression</H2><H3>Part construction </H3><P>The modular cloning strategy (described in the materials and methods) was used an attempt to build plasmids containing either: the fim operon under two different promoters; or six modified versions of fimH (and wild type fimH) under for different promoters. The results of our cloning attempts are given in Table 1. </P><FIGURE><TABLE style="width:100%; margin:0 auto; border-collapse: collapse;  border: 2px solid black" border="1"><TBODY><TR><TH>Coding sequence</TH><TH>P_T7</TH><TH>P_Rha</TH><TH>P_Ara</TH><TH>P_T5</TH><TH>P_J23100</TH></TR><TR><TD>FimH</TD><TD>Yes</TD><TD>N/A</TD></TR><TR><TD>FimH_1_His</TD><TD>Yes</TD><TD>N/A</TD></TR><TR><TD>FimH_1_SynMT</TD><TD>Yes</TD><TD>N/A</TD></TR><TR><TD>FimH_1_MouseMT</TD><TD>Yes</TD><TD>N/A</TD></TR><TR><TD>FimH_1_Plastocyanin</TD><TD>N/A</TD></TR><TR><TD>FimH_1_sfGFP</TD><TD>Yes</TD><TD>N/A</TD></TR><TR><TD>FimH_225_sfGFP</TD><TD>Yes</TD><TD>Yes</TD><TD>N/A</TD></TR><TR><TD>FimH_258_sfGFP</TD><TD>Yes</TD><TD>N/A</TD></TR><TR><TD>Fim operon</TD><TD>N/A</TD><TD>N/A</TD><TD>Yes</TD><TD>N/A</TD><TD>Yes</TD></TR><CAPTION><B>Table 1</B>: Combinations of Promoters and FimH fusion proteins for which cloning was attempted</CAPTION></TBODY></TABLE></FIGURE><H3> Part testing </H3><H4>FimH_sfGFP expression</H4><P> The literature has shown that the terminal pili protein FimH (Le Trong et al 2010) can be modified by inserting heterologous sequences at position 225 and 258 (Pallesen et al 1995, Shembri et al 1999). This part produces a FimH adhesin protein fused with sfGFP (Pedelaq et al 2005) at its 225th amino acid residue, after signal peptide cleavage. Expression is under the control of an IPTG-inducible, T7 promoter (BBa_K1614000), with BBa_B0034 RBS and BBa_B0015 terminator. The part, when induced, produces a fluorescent FimH protein that should initiate pilus biosynthesis when co-transformed with a plasmid containing the fim operon.The T7 promoter should give very strong expression and sfGFP should both give a visual indication of successful expression and folding. As a large protein, sfGFP would push the chaperone-usher pathway to its steric limits.</P><P>We have expressed this construct in BL21(DE3). Fluorescence was measured using a plate reader (Tecan) and an Amnis ImageStream ISX. Protein expression was determined via Western Blot and TEM with Immunogold labelling.</P><FIGURE class="border border-dark rounded"><FIG caption=""><B>Figure 3:</B> Data from the plate reader showing fluorescence intensity for WT-BL21(DE3) and T7_FimH_225_sfGFP, average from n=3 and error bars show standard deviation of the mean.These results show that fluorescence from cultures containing T7_FimH_225_sfGFP is higher than WT-BL21(DE3). This shows that FimH_225_sfGFP is expressed and does give rise to fluorescence. According to standard curves made in the lab on the same equipment and settings, we produced a 130 ng/ml concentration of sfGFP in the sample.  </FIG></FIGURE><FIGURE class="border border-dark rounded"><FIGCAPTION style="text-align:justify"><B>Figure 4:</B> These data from an image stream show the fluorescence profile for wild-type (left) BL21(DE3) and BL21(DE3) with T7_FimH_225sfGFP (right).The wild type demonstrates no significant fluorescence, while same cell with the single plasmid insert shows a strong fluorescent signal in the highlighted portion of cells.</FIGCAPTION></FIGURE><FIGURE class="border border-dark rounded"><FIGCAPTION style="text-align:justify"><B>Figure 5:</B> Data from Amnis ImageStream TMX show the fluorescence profile for individual cells of BL21(DE3) with T7_FimH_225_sfGFP. Due to the limitations arising from the magnification, the image stream can not be used to locate where the fluorescent protein resides.</FIGCAPTION></FIGURE><P> These results show that a number of cells in the overall culture produced strong fluorescence. This fluorescence suggests successful folding of the sfGFP which can be taken as evidence by proxy of FimH folding. The result also suggests that sfGFP is able to move through the pore formed during pilus biosynthesis. Moving on from this result, we were confident that the fusion of metal binding domains to the FimH protein would not negatively affect pilus formation as our chosen domains, and other potentials that may make up a modular toolkit, were smaller than sfGFP . </P><FIGURE class="border border-dark rounded"><FIGURECAPTION><B>Figure 6:</B> Western blot of BL21 WT and BL21 with our T7-FimH_225_GFP construct, 1-5: sfGFP, BL21 WT soluble fraction, BL21 WT pellet, BL21 with T7-FimH_225_GFP solluble fraction, BL21 with T7-FimH_225_GFP pellet.
</FIGURECAPTION></FIGURE><P>The Western-Blot indicated that FimH-225_sfGFP was mainly in the cytoplasm but to determine whether any proportion of expressed FimH-225sfGFP was exported from the cell and forming pili, TEM with immunogold labelling was attempted. </P><FIGURE class="rounded mx-auto d-block w-100 border border-dark rounded"><FIGCAPTION><B>Figure 7:</B> Pictured here are two wild type BL21(DE3) cells which should not contain the sfGFP antigen in their FimH proteins. A few gold particles are visible in these micrographs, though they can likely be attributed to non-specific binding.   </FIGCAPTION></FIGURE><FIGURE class="rounded mx-auto d-block w-100 border border-dark rounded"><FIGCAPTION><B>Figure 8: </B>In contrast, when BL21(DE3) in transformed with T7_FimH_225_sfGFP, it appears to specifically bind the gold particles as shown. The images suggest successful expression and export of these modified FimH proteins and specific binding. On the image on the right, the gold particles appear to align with the pili on the cell surface. </FIGCAPTION></FIGURE><P>We were unable to detect fluorescence in cells that had the FimH fusion protein under control of the rhamnose inducible promoter despite experimenting with the induction protocol over a range of growth temperatures and rhamnose concentrations . Given the relatively low proportion of cells in the overall culture of the T7_FimH_225_sfGFP that showed significant fluorescence, it is perhaps unsurprising that a weaker and less well characterised promoter(than T7) should not give strong results. Our consequent decision was to attempt to test the rhamnose inducible promoter by other means. We used a fusion of 6xhistine tag and FimH as an alternative reporter mechanism with the aim of establishing whether modified FimH constructs could be expressed with P_Rha. </P><H4>FimH_1_His expression</H4><P>We therefore looked to express FimH with a HisTag at the first amino acid position under control of P_Rha.</P><P> Harvard iGEM 2015 also introduced modifications at position 1 of the mature FimH protein without characterising the resultant part. This part produces a FimH protein with a 6xHistidine tag inserted at the first amino acid position, that is the residue that remains at the N-terminus after the signal peptide has been cleaved during the membrane export process. This position is intended to improve the steric properties of the protein so as to ease the cell surface membrane export and to prevent interference with any native protein domains in the FimH which are involved in pilus biogenesis. The coding sequence is under the control of a rhamnose-inducible promoter (BBa_K902065), with the BBa_B0034 RBS and BBa_B0015 terminator. The part, when induced, produces a metal binding FimH protein that should involve itself in pilus biosynthesis when co-transformed with a plasmid containing the fim operon
As well as being utilised as a metal binding protein, this part also acts as a reporter. It gives clear and unambiguous evidence of protein expression. Once transformed into a number of E. coli strains (BL21(DE3), Top10, ΔFimB, and ΔFimH), the fusion protein can be expressed by inducing the culture at 0.6 OD with 2% rhamnose. The production of the 6xHistidine tag can be probed by the use of an SDS-PAGE and a Western blot. </P><P>After 24 h growth, cultures containing the P_Rha_FimH_1His and wild-type strains were harvested. Cells were disrupted using BugBuster protein extraction reagent (Merck) and samples of the soluble and insoluble fractions were prepared for SDS-PAGE. Western Blots were probed with an anti-6xHis primary antibody raised in Mouse and anti-Mouse alkaline phosphatase-conjugated secondary antibody and visualised with SigmaFast BCIP/NBT.</P><P>The results of this <A href="https://2017.igem.org/Team:Exeter/Experiments">SDS-PAGE</A> and subsequent Western blot provided bands at the corresponding molecular weights for the FimH_1His protein which also had binding affinities for an anti-His antibody. The result of a band that corresponds to a molecular weight marginally lighter than the entire intact FimH_1His evidentiated expression of the protein, and suggested that the signal peptide had been cleaved upon the proteins delivery to the cell surface membrane. This makes a case for successful export of the protein and therefore successful pilus biogenesis. </P><FIGURE class="rounded mx-auto d-block w-100 border border-dark rounded"><FIGCAPTION><B>Figure 9</B>: The wells of the polyacrylamide gel contained the following, 1-10:marker,ΔFimH_1His pellet, ΔFimB_1His pellet, Top10_1His pellet, ΔFimH_1His pellet, ΔFimH_1His supernatant, Top10_1His supernatant, BL21(DE3)_1His supernatant, ΔFimB_1His supernatant and marker. The band of importance is the one at 28-30kDa which represents the FimH protein, potentially having been exported and had its signal peptide cleaved.  </FIGCAPTION></FIGURE><FIGURE class="rounded mx-auto d-block w-100 border border-dark rounded"><FIGCAPTION><B>Figure 10:</B> The wells is this gel are filled, 0-9, as follows: marker, BL21(DE3)WT pellet,Top10 WT pellet, ΔFimB WT pellet, ΔFimH WT pellet, BL21(DE3) WT supernatant, Top10 WT supernatant, ΔFimB WT supernatant, ΔFimH WT supernatant and BL21(DE3)_1His pellet.</FIGCAPTION></FIGURE><FIGURE class="rounded mx-auto d-block w-100 border border-dark rounded"><FIGCAPTION><B>Figure 11:</B> The columns of the Western blot correspond to the following samples,0-9:marker,ΔFimB_1His pellet, Top10_1His pellet,BL21(DE3)_1His pellet,BL21(DE3)_1His pellet,BL21(DE3)_1His supernatant, ΔFimH_1His supernatant, ΔFimB_1His supernatant, Top10_1His supernatant and marker. The bands present here demonstrate the affinity of protein with the same molecular weight as FimH with anti-His antibodies. This would imply that FimH_1His is being expressed. </FIGCAPTION></FIGURE><P>The images of the SDS-PAGE gels demonstrate that the cells were successfully disrupted as protein bands were seen in both soluble and insoluble fractions. However, due to the fact that the samples were crude protein extracts, the gel images do not definitively show whether FimH-1His is expressed. The Western Blot image shows bands at the corresponding molecular weights for the FimH_1His protein with (32 kDa) and without (30 kDa) the signal peptide. In the ΔFimH culture, a band is seen in the soluble fraction corresponding to the molecular weight of the un-cleaved signal peptide-FimH_1His protein. This indicates that the protein is expressed but remains in the cytoplasm of the cell. In the BL21(DE3), Top10 and ΔFimB cultures, bands are seen in the insoluble fraction corresponding to the molecular weight of the cleaved FimH_1His protein. The lack of signal peptide demonstrates that the protein has been exported from the cytoplasm and the fact that the bands only appear in the insoluble fraction suggests that FimH-1His is somehow associated with the cell membrane. It is unclear as to why export should not occur in the ΔFimH strain.</P><H4>Fim Operon expression</H4><FIGURE class="rounded mx-auto d-block w-100 border border-dark rounded"><FIGCAPTION style="clear:left;"><B>Figure 12:</B> On the left is an electron micrograph of an <I>E. coli </I> MG1655 cell. Pili are clearly visible on the surface of the cell. On the right is an image of Top10, which displays no pili.  </FIGCAPTION></FIGURE><FIGURE class="rounded mx-auto d-block w-100 border border-dark rounded"><FIGCAPTION><B>Figure 13:</B> Top10 with the <I>fim operon</I> under the control of promoter P_J23100 (left) and P_Ara(right), showed no visible signs of pili expression with insertion of the operon alone. </FIGCAPTION></FIGURE><FIGURE class="rounded mx-auto d-block w-100 border border-dark rounded"><FIGCAPTION><B>Figure 14: </B>ΔFimB should not, theoretically, produce pili. The regulatory gene FimB has been knocked out, and so the operon has effectively been switched off. These electron micrographs show wild type ΔFimB with strong, peritrichous flagellar expression, but no visible signs of pili. The image on the right shows little evidence of pili or flagella connected to the cell surface, which suggests that the negative staining technique can be damaging to these structures and could cause detachment and fragmentation.  </FIGCAPTION></FIGURE><FIGURE class="rounded mx-auto d-block w-100 border border-dark rounded"><FIGCAPTION><B>Figure 15:</B> We transformed a plasmid containing the <I>fim operon</I> under control of promoters P_J23100(left)
 and P_Ara(right) into ΔFimB. Both exhibit flagella on their cell surface, but the right electron micrograph shows a suggestion of pili.
  </FIGCAPTION></FIGURE><P>The electron micrograph images presented above suggest that under control of the arabinose inducible promoter, the fim operon is expressed and pili (albeit incomplete as they are lacking FimH) are formed on the cell surface. Further work is needed to fully characterise this, particularly in combination with out modified FimH constructs. We have successfully shown a suggestion of pili using the arabinose inducible promoter and no pili production using a constitutive promoter. </P><H2 id="h3">Applied Design</H2><H3> Hydrocyclone </H3>

To see how effective our hydrocyclone was at separating sediment from a contaminated solution, follow this link to our hydrocyclone page, <A href="https://2017.igem.org/Team:Exeter/Hydrocyclone"> Hydrocyclone Results</A>.

<H3> Metal Binding Reactor </H3><P> To see our use of experimental design software in testing the Metal Binding Reactor, follow this link to our MBR page, <A href="https://2017.igem.org/Team:Exeter/MBR">Metal Binding Reactor</A>.

</P><H3> Biosecurity </H3>

To see that results we obtained for how the percentage of bacteria changed with exposure time to UV light, follow this link to our applied design page, <A href="#https://2017.igem.org/Team:Exeter/Bio-security">UV results</A>.



      </DIV></DIV></DIV></DIV><DIV id="footer" class="container"><SPAN><A href="https://2017.igem.org/Team:Exeter/Attributions#h7">We would like to thank all our sponsors.</A></SPAN></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>