<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Exeter_Notebook_Text skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Exeter/Notebook/Text</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>10/07/2017
</P><P><UL><LI>Extracted interlab devices from kit and made up spread plates. </LI></UL></P><P>11/07/2017
</P><P><UL><LI>Streak plates of DH5α and MG1655.</LI><LI>Overnight cultures for interlab devices- 5ml + chloramphenicol + one colony.</LI></UL></P><P>12/07/2017
</P><P><UL><LI>Overnight cultures for wild type (DH5α, MG1655) - 5ml LB + one colony.</LI><LI>Glycerol stocks (DH5α, MG1655).</LI><LI>Miniprep of interlab devices (and pBAD).</LI><LI>Qubit of devices (and pBAD).</LI></UL></P><P>13/07/2017
</P><P><UL><LI>Glycerol stocks of wild types DH5α and MG1655.</LI><LI>Made 5x 200ml of LB (autoclaved)</LI><LI>Genomic DNA prep of DH5α and MG1655.</LI><LI>Qubit for wild types.</LI></UL></P><P>14/07/2017
</P><P><UL><LI>Introduction to TECAN plate reader.</LI><LI>Competent cell preparation for DH5α (see protocol).</LI><LI>Measurements for interlab.</LI></UL></P><P>17/07/2017
</P><P><UL><LI>Interlab device transformations (expt. 3)</LI><LI>Restriction digest for insertion of fim sections into 
plasmids (Expts. 1+2)</LI><LI>Stored fim plasmids.</LI></UL></P><P>18/07/2017
</P><P><UL><LI>Liquid overnights for interlab (error expt.6)</LI><LI>PCR and electrophoresis (Taq polymerase) of fim genes (D,H and A)
 - confirms genes in DH5α (bad news) (expt.4)</LI></UL></P><P>19/07/2017
</P><P><UL><LI>Transformed plasmids from expt.2 into cells (expt.5)</LI><LI>Liquid overnights for interlab.(expt.9)</LI><LI>Electrophoresis of fim operon from expt.4 (expt.7)</LI><LI>PCR (Phusion) of whole operon. (expt.8)</LI></UL></P><P>20/07/2017
</P><P><UL><LI>Interlab measurements.(expt.10)</LI><LI>Miniprep and Qubit for fim constituents. (expt.11)</LI><LI>Competent cell prep (expt.12)</LI><LI>Genomic DNA extraction of Top10 strain (expt. 13)</LI><LI>Electrophoresis for PCR from expt.8 (expt.14)</LI></UL></P><P>21/07/2017
</P><P><UL><LI>Prepared DNA from miniprep for sequencing (to confirm that transformation
 worked)</LI><LI>Competent cell prep (expt.12)</LI><LI>Genomic DNA extraction of Top10 strain (expt. 13)</LI><LI>Electrophoresis for PCR from expt.8 (expt.14)</LI></UL></P><P>24/07/2017
</P><P><UL><LI>Interlab data analysis</LI><LI>Overnights for Top10</LI></UL></P><P>25/07/2017
</P><P><UL><LI>Genomic extraction from Top10 (x2)</LI><LI>DNA extraction from gel band (MG1655 operon)</LI><LI>PCR of Top10 and MG1655 fim genes.</LI></UL></P><P>26/07/2017
</P><P><UL><LI>Electrophoresis of Top10 and MG1655</LI><LI>One pot cloning of fimAIC, D, and FG with promoter.</LI><LI>Transformed fim containing plasmids into DH5ɑ</LI><LI>Streaked MG1655 and Top10 for Bioimaging. </LI></UL></P><P>27/07/2017
</P><P><UL><LI>Overnights for MG1655 and Top10 for imaging.</LI><LI>Transformed MoClo products into DH5ɑ.</LI></UL></P><P>28/07/2017
</P><P><UL><LI>Negative stain and transmission electron microscopy of MG1655 and 
Top10 samples. </LI></UL></P><P>31/07/2017
</P><P><UL><LI>Overnights of MG1655, Top10 and MoClo products (x8) from 
	restriction digest of fimH constructs and ligation into plasmids.</LI></UL></P><P>01/08/2017
</P><P><UL><LI>Transformed fim H constructs (x10) into DH5α</LI><LI>One pot: prepared for sequencing, made glycerol stocks, miniprep and qbit.</LI><LI>DH5α overnights, made competent cells of top10.</LI></UL></P><P>02/08/2017
</P><P><UL><LI>Restriction digest of operon MoClo products (for gel)</LI><LI>Prep. for fim operon sequencing</LI></UL></P><P>03/08/2017
</P><P><UL><LI>MoClo of fimH constructs.</LI></UL></P><P>
04/08/2017
</P><P><UL><LI>Sequencing results confirm successful plasmid construction.</LI><LI>Transformation of MoClo products into DH5α.</LI></UL></P><P>
07/08/2017
</P><P><UL><LI>Transformation of fimH MoClos and preparation of fim operon for sequencing.</LI></UL></P><P>
08/08/2017
</P><P><UL><LI>Top 10 competent cells made</LI></UL></P><P>10/08/2017
</P><P><UL><LI>Miniprep and Qubit of P_Rha+fimH+psB1A3 constructs.</LI><LI>Overnights cultures of P_Ara+fimop(pX3’6’0’0), 
P_J23100+fimop(pX3’6’0’0), P_J23100+fimop(PSB1A3)</LI></UL></P><P>11/08/2017
</P><P><UL><LI>Prep of P_Rha+fimH miniprep DNA for sequencing.</LI><LI>Qubit of overnight cultures: (P_J23100 x 2 + P_Ara) + fimH.</LI></UL></P><P>14/08/2017
</P><P><UL><LI>Restriction digest of Fim operon in: pX3’6’0’0 with P_J23100 promoter, 
pX3’6’0’0 with P_Ara, pSB1A3 with P_J23100.</LI><LI>The products of the restriction digest were ran on the electrophoresis
 and only the pSB1A3 product was successful. Only two pSB1A3 plasmids were 
sent for sequencing (5+6).</LI><LI>The P_Rha+fimH+psB1A3 constructs were also sent for sequencing (13 constructs).</LI></UL></P><P>15/08/2017
</P><P><UL><LI>Poured CAM plates, Restriction digest of P_Ara_fimoperon_pSB1A3, P_J23100_fimoperon_
pSB1A3 and the pX3’6’0’0 plasmid, followed by a gel electrophoresis and gel extraction 
of the DNA (the wrong gel bands were extracted, so Chloe did it again for us).</LI><LI>Transformation of the FimH constructs in the pSB1C3 and pSB1A3 into DH5α.</LI><LI>Made a mannose serial dilution from 4% to 0.0625% and ran in the HPLC.</LI></UL></P><P>16/08/2017
</P><P><UL><LI>Gel extraction of the bands (after that we found out the bands we extracted were
 the wrong ones).</LI><LI>Overnights of FimH constructs in the pSB1C3 and pSB1A3 into DH5α.</LI><LI>Grew MG1655 on silica sand and NP-Bacto-Pellets.</LI></UL></P><P>17/08/2017
</P><P><UL><LI>Mini prep and Qubit from overnights of FimH constructs and transformation of 
the ligation from the gel extraction on CAM plates that Chloe did.</LI><LI>Prepared silica sand and NP-Bacto-Pellets samples for SEM imaging.</LI></UL></P><P>21/08/2017
</P><P><UL><LI>Miniprep, glycerol stocks and qubit of P_J23100 and P_Ara fim_op.</LI><LI>Diagnostic digest and electrophoresis of above - shows successful construction 
and transformation of plasmids. </LI></UL></P><P>22/08/2017
</P><P><UL><LI>Double transformations of two plasmids into Top10.</LI><LI>Transformation of fimH constructs into pEX1A3.</LI><LI>Transformation for UV experiments.</LI></UL></P><P>23/08/2017
</P><P><UL><LI>Liquid overnights for previous transformations.</LI><LI> Grow overnight cultures of <I>E.coli</I> Top10 and <I>E.coli</I> Top10, transformed with chloramphenicol and ampicillin resistance genes for UV experiment. </LI><LI> Imaged silica sand and NP-Bacto-Pellets samples using SEM.</LI></UL></P><P>24/08/2017
</P><P><UL><LI>First attempt at induction of fimH constructs.</LI><LI>OD of overnight cultures set to one. Sample were irradiated, spread on plates and then incubated.</LI></UL></P><P>25/08/2017
</P><P><UL><LI> Colonies on plates were counted, unexpected results. </LI></UL></P><P>28/08/2017
</P><P><UL><LI>Overnights of transformed Top10, WT Top10 and WT MG1655.</LI></UL></P><P>29/08/2017
</P><P><UL><LI>Revised induction protocol.</LI><LI>Measured construct fluorescence in TECAN.</LI><LI>Grew MG1655 on silica sand and NP-Bacto-Pellets again following unexpected results previously.</LI></UL></P><P>30/08/2017
</P><P><UL><LI>First attempt at agglutination protocol</LI><LI>Prepared and imaged silica sand and NP-Bacto-Pellets samples for SEM.</LI><LI>Investigated the flow rates that were achieved when polypropylene scaffold torus structures were inside the metal binding reactor.</LI></UL></P><P>31/08/2017
</P><P><UL><LI>Induction protocol</LI></UL></P><P>01/09/2017
</P><P><UL><LI>Samples run through the TECAN</LI><LI>Grew MG1655 on polypropylene scaffold torus structures, using a range of concentrations of a chlorhexidine gluconate surfactant.</LI></UL></P><P>04/09/2017
</P><P><UL><LI>Overnight cultures of constructs</LI><LI>Examined the tori to observe which concentration of surfactant produced a biofilm on the surface.</LI><LI>Repeated the growing of MG1655 on polypropylene scaffold torus structures, using a range of concentrations of a chlorhexidine gluconate surfactant.</LI></UL></P><P>05/09/2017
</P><P><UL><LI>Induction protocol</LI><LI>Agglutination protocol</LI><LI>Successful transformation of mouse metallothionein </LI><LI>Overnight cultures of <I>E.coli</I> Top10 for UV </LI><LI>Examined the tori to observe which concentration of surfactant produced a biofilm on the surface.</LI></UL></P><P>06/09/2017
</P><P><UL><LI>Samples run through FACS and fluorescent cells sorted</LI><LI>Sorted cells plated on agar in 7x8 grid </LI><LI>1 in 1000 dilution of overnight culture </LI><LI>Cultures irradiated and plates spread </LI></UL></P><P>07/09/2017
</P><P><UL><LI>Induction protocol</LI><LI> Colonies on plates counted, OD of culture too great.</LI></UL></P><P>08/09/2017
</P><P><UL><LI>Dialysis trial run</LI></UL></P><P>11/09/2017
</P><P><UL><LI>Induction protocol</LI><LI>Repeated the growing of MG1655 on polypropylene scaffold torus structures, using a range of concentrations of a chlorhexidine gluconate surfactant.</LI></UL></P><P>12/09/2017
</P><P><UL><LI>Miniprep of P_Rha_GFP constructs for DoE</LI><LI>Ran the fluidising media reactor with tori at a range of concentrations of a chlorhexidine gluconate surfactant.</LI><LI>Imaged samples of the tori using SEM before and after being run through the metal binding reactor.</LI><LI>Ran HPLC for the mannose concentration of samples taken from the use of the metal binding reactor with the tori.</LI></UL></P><P>13/09/2017
</P><P><UL><LI>Induction of three FimH constructs at 0.1 and 0.2% rhamnose concentrations in BL21 and DH5α</LI><LI>Overnight cultures of <I>E. coli</I> Top10.</LI></UL></P><P>14/09/2017
</P><P><UL><LI>TECAN reading of previous day's induced cultures.</LI><LI>Induction of three FimH constructs at 1 and 2% rhamnose concentrations.</LI><LI>Culture irradiated using UV trans-illuminator</LI></UL></P><P>
15/09/2017
</P><P><UL><LI>TECAN reading of the previous day's induced cultures.</LI><LI>Colonies counted, positive results.</LI></UL></P><P>
19/09/2017
</P><P><UL><LI>DoE: the first block</LI><LI>Restriction digest and ligation of P_Rha_FimH_1sfGFP and with pSB1A3.</LI></UL></P><P>
20/09/2017
</P><P><UL><LI>TECAN reading of previous day's cultures for DoE</LI><LI>Restriction digest, electrophoresis, gel extraction and ligation of P_Rha_FimH_1sfGFP and pSB1A3.</LI><LI> Overnight culture of <I>E. coli</I> Top10 </LI><LI>Grew MG1655 on polypropylene scaffold torus structures, for runs 1-3 for design of experiment.</LI></UL></P><P>
21/09/2017
</P><P><UL><LI>TECAN reading of previous day's cultures for DoE</LI><LI>OD of overnight culture set to 1 and 1 in 1,000,000 applied</LI><LI>Cultures irradiated</LI><LI>Runs 1-3 for the design of experiment in the metal binding reactor.</LI><LI>Grew MG1655 on polypropylene scaffold torus structures, for runs 4-5 for design of experiment.</LI></UL></P><P>22/09/2017
</P><P><UL><LI>Digestion and ligation of FimH_1sfGFP with pSB1A3 </LI><LI>Colonies counted, results are as expected.</LI><LI>Runs 4-5 for the design of experiment in the metal binding reactor.</LI></UL></P><P>25/09/2017
</P><P><UL><LI>Transformed FimH_1sfGFP_pSB1A3 into <I>E.coli</I></LI></UL></P><P>26/09/2017
</P><P><UL><LI>Grew MG1655 on polypropylene scaffold torus structures, for runs 6-8 for design of experiment.</LI></UL></P><P>27/09/2017
</P><P><UL><LI>Runs 6-8 for the design of experiment in the metal binding reactor.</LI><LI>Grew MG1655 on polypropylene scaffold torus structures, for runs 9, 10 and 1 for design of experiment.</LI></UL></P><P>28/09/2017
</P><P><UL><LI>TECAN reading for DoE</LI><LI>Inoculated and induced cultures for DoE </LI><LI>Diagnostic digestion of FimH_1sfGFP_pSB1A3 plasmid DNA </LI><LI>Runs 9, 10 and 1 for the design of experiment in the metal binding reactor.</LI></UL></P><P>29/09/2017
</P><P><UL><LI>Transformation for Newcastle collaboration</LI></UL></P><P>02/10/2017
</P><P><UL><LI> Transformation of T7_FimH_225sfGFP into BL21 </LI></UL></P><P>04/10/2017
</P><P><UL><LI> Induction protocol </LI></UL></P><P>09/10/2017
</P><P><UL><LI>Ran Newcastle samples through FACS, analysed the data and sent it off. </LI></UL></P><P>11/10/2017
</P><P><UL><LI>Prepared samples for subsequent SDS page analysis of fim operon in top10 </LI></UL></P><P>12/10/2017
</P><P><UL><LI>Induction protocol </LI></UL></P><P>13/10/2017
</P><P><UL><LI> Restriction digest of all parts in pSB1C3 and diagnostic electrophoresis gel showed successful construction </LI><LI> Preparation of all parts for sequencing </LI></UL></P><P>16/10/2017
</P><P><UL><LI>Qubit of all parts in pSB1C3 </LI></UL></P><P>17/10/2017
</P><P><UL><LI>SDS-PAGE and Western blot of FimH_1His constructs and wild types in four <I>E.coli</I> strains </LI></UL></P><P>18/10/2017
</P><P><UL><LI>Western blot and SDS-PAGE results showed successful expression and binding of FimH_1His pili with no bands for the wild types. </LI></UL></P><P>20/10/2017
</P><P><UL><LI>Preparation of DNA parts for submission </LI></UL></P><P>23/10/2017
</P><P><UL><LI>Transformation of FimB KO with fim operon (under both promoters) with FimH, FimH_1His, FimH_1SynMT, FimH_1MouseMT</LI></UL></P><P>24/10/2017
</P><P><UL><LI>DNA parts sent off</LI></UL></P><P>25/10/2017
</P><P><UL><LI>Inoculate 50ml cultures with FimB KO with either P_Ara-Fim_Operon or PJ23100-Fim_Operon and FimH, FimH_1_His, FimH_1_SynMT, FimH_1_MouseMT</LI><LI>TECAN readings for T7-FimH_225_GFP in BL21(DE3) after induction as well as BL21(DE3) wild type (similar treatment)</LI></UL></P><P>26/10/2017
</P><P><UL><LI>Dialysis: FimB KO of PJ23100 and PRha_FimH / PRha_FimH_1_His / PRha_FimH_1_SynMT / PRha_FimH_1_MouseMT by adding either CuCl<SUB>2</SUB> and/or NiSO<SUB>4</SUB></LI></UL></P><P>31/10/2017
</P><P><UL><LI>SDS page and Western blot of T7-FimH_225_GFP in BL21(DE3) after induction as well as BL21(DE3) wild type (similar treatment) </LI></UL></P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>