Team:UST Beijing/Modeling
USTB-Beijing | Welcome
Toggle navigation
Background
Project
Cyclase
Glucosidase
Modeling
HomePage
Parts
Team
Awards
HP
Collaborations
Safety
Attributions
InterLab
Modeling
Functions are determined by structures. It is helpful for us to grasp the protein structure, and we could know the function of proteins better. So, we use supercomputer to predict the protein structure, since it is much less expensive, faster, and able to a large scale generate protein structures than classical methods.
Protein Structure Models
Due to the fact that the real structure of Lanosterol Synthase is known, so we began from protein data bank in Europe. We firstly obtained Canonical number: P48449(pdb|1w6k|), and then the amino acid sequence of Lanosterol were found in Uniprot, shown as following:
>pdb|1w6k|A
MTEGTCLRRRGGPYKTEPATDLGRWRLNCERGRQTWTYLQDERAGREQTGLEAYALGLDTKNYFKDLPKAHTAFEGALNGMTFY VGLQAEDGHWTGDYGGPLFLLPGLLITCHVARIPLPAGYREEIVRYLRSVQLPDGGWGLHIEDKSTVFGTALNYVSLRILGVGP DDPDLVRARNILHKKGGAVAIPSWGKFWLAVLNVYSWEGLNTLFPEMWLFPDWAPAHPSTLWCHCRVDGPASTAFQEHVSRIPD YLWMGLDGMKMQGTNGSQIWDTAFAIQALLEAGGHHRPEFSSCLQKAHEFLRLSQVPDNPPDYQKYYRQMRKGGFSFSTLDCGW IVSDCTAEALKAVLLLQEKCPHVTEHIPRERLCDAVAVLLNMRNPDGGFATYETKRGGHLLELLNPSEVFGDIMIDYTYVECTS AVMQALKYFHKRFPEHRAAEIRETLTQGLEFCRRQQRADGSWEGSWGVCFTYGTWFGLEAFACMGQTYRDGTACAEVSRACDFL LSRQMADGGWGEDFESCEERRYVQSAQSQIHNTCWAMMGLMAVRHPDIEAQERGVRCLLEKQLPNGDWPQENIAGVFNKSCAIS YTSYRNIFPIWALGRFSQLYPERALAGHP
And the starting about 90 amino acids are membrane bound sequences. We made a chimera sequence using the Lanosterol Synthase homologue, Dammarenediol Synthase. It’s Pangu sequence. Then, we use the online servers, Swiss-Model, to generate high quality predictions of 3D structure of protein molecules from amino sequences. shown as following:
>pdb|Pangu|A
MTEGTCLRRRGGPYKTEPATDLGRWRLNCERGRQTWTYLQDERAGREQTGLEAYALGLDTKNYFKDLPKAHTAFEGALNGMTFYV GLQAEDGHYDAVTTAVKKALRLNRAIQAHDGHWPAENAGSLLYTPPLIIALYISGTIDTILTKQHKKELIRFVYNHQNEDGGWGS YIEGHSTMIGSVLSYVMLRLLGEGLAESDDGNGAVERGRKWILDHGGAAGIPSWGKTYLAVLGVYEWEGCNPLPPEFWLFPSSFP FHPAKMWIYCRCTYMPMSYLYGKRYHGPITDLVLSLRQEIYNIPYEQIKWNQQRHNCCKEDLYYPHTLVQDLVWDGLHYFSEPFL KRWPFNKLRKRGLKRVVELMRYGATETRFITTGNGEKALQIMSWWAEDPNGDEFKHHLARIPDFLWIAEDGMTVQSFGSQLWDCI LATQAIIATNMVEEYGDSLKKAHFFIKESQIKENPRGDFLKMCRQFTKGAWTFSDQDHGCVVSDCTAEALKCLLLLSQMPQDIVG EKPEVERLYEAVNVLLYLQSRVSGGFAVWEPPVPKPYLEMLNPSEIFADIVVEREHIECTASVIKGLMAFKCLHPGHRQKEIEDS VAKAIRYLERNQMPDGSWYGFWGICFLYGTFFTLSGFASAGRTYDNSEAVRKGVKFFLSTQNEEGGWGESLESCPSEKFTPLKGN RTNLVQTSWAMLGLMFGGQAERDPTPLHRAAKLLINAQMDNGDFPQQEITGVYCKNSMLHYAEYRNIFPLWALGEYRKRVW
Swiss Model is an automated system for modeling the 3D structure of a protein from its amino acid sequence using homology modeling techniques. We use OSC as the template to predict 3D model of Pangu. The model is named Pangu-Swiss Model.
After docking simulation, we made analysis of the results of the docking and evaluate model quality on Pangu. It shows that ligands and substrates bind strongly.
Logistic Function And Enzymatic Activity Simulation
Integration of Logistic functions to simulate enzymatic measurement during bacterium fermentation. In our beta-glucosidase expression assay, we incubated the growing bacterium E.coli BL21(DE3) harboring pSB1C3/BBa_K2072000 plasmid with PNPG, a colorless substrate for beta-glucosidase hydrolyzing into yellow PNP. We measured OD620 nm to reflect cell number and OD405 nm-1.5*OD620 nm to reflect PNP concentration. During the fermentation, both cell number and PNP are increasing. In order to simulate these factors, we integrate the logistic functions (1):
So, we get the formula,
We used Excel to graph functions (1) and (3) resulting following curves:
Reference
(1) Marco Biasini; Stefan Bienert; Andrew Waterhouse; Konstantin Arnold; Gabriel Studer; Tobias Schmidt; Florian Kiefer; Tiziano Gallo Cassarino; Martino Bertoni; Lorenza Bordoli; Torsten Schwede. (2014). SWISS-MODEL: modelling protein tertiary and quaternary structure using evolutionary information. Nucleic Acids Research (1 July 2014) 42 (W1): W252-W258; doi: 10.1093/nar/gku340.
(2) Arnold, K., Bordoli, L., Kopp, J. and Schwede, T. (2006) The SWISS-MODEL workspace: a web-based environment for protein structure homology modelling. Bioinformatics, 22, 195-201. Benkert, P., Biasini, M. and Schwede, T. (2011) Toward the estimation of the absolute quality of individual protein structure models. Bioinformatics, 27, 343-350.
Copyright © 2017 UST_Beijing iGEM. All rights reserved.
