<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UST_Beijing_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UST Beijing/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>USTB-Beijing | Welcome</TITLE></P><DIV class="header_box"><NAV class="navbar navbar-inverse" role="navigation"><DIV class="navbar-header"><BUTTON type="button" id="nav-toggle" class="navbar-toggle" data-toggle="collapse" data-target="#main-nav"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV id="main-nav" class="collapse navbar-collapse navStyle"><UL class="nav navbar-nav" id="mainNav"><LI><A href="https://2017.igem.org/Team:UST_Beijing/Background"><DIV class="scroll-link">Background</DIV></A></LI><LI><A id="cs2"><DIV class="scroll-link">Project</DIV></A></LI><LI><A href="https://2017.igem.org/Team:UST_Beijing/Cyclase">Cyclase</A></LI><LI><A href="https://2017.igem.org/Team:UST_Beijing/Experiments">Glucosidase</A></LI></UL><LI><A id="cs3" href="https://2017.igem.org/Team:UST_Beijing/Modeling"><DIV class="scroll-link">Modeling</DIV></A></LI><LI><A href="https://2017.igem.org/Team:UST_Beijing"><DIV class="scroll-link">HomePage</DIV></A></LI><LI><A href="https://2017.igem.org/Team:UST_Beijing/Basic_Part"><DIV class="scroll-link">Parts</DIV></A></LI><LI><A href="https://2017.igem.org/Team:UST_Beijing/Team"><DIV class="scroll-link">Team</DIV></A></LI><LI><A id="cs5"><DIV class="scroll-link">Awards</DIV></A></LI><LI><A href="https://2017.igem.org/Team:UST_Beijing/HP/Silver">HP</A></LI><LI><A href="https://2017.igem.org/Team:UST_Beijing/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:UST_Beijing/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:UST_Beijing/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:UST_Beijing/InterLab">InterLab</A></LI></DIV></NAV></DIV><SECTION style="background-image:url(https://static.igem.org/mediawiki/2017/f/f8/Ustbbjbj.png);background-position:center center;background-repeat:no-repeat;-moz-background-size: cover;background-size: cover;background-attachment:fixed;"><DIV class="container" align="center"><DIV class="panel-body"><H2 style="font-family:Arial;font-size: 55px;">Glucosidase</H2><P class="text-justify" style="width:800px;text-indent:1em;text-align: left; font-size:18px;line-height:28px;font-family:Arial">
In this year's project, we have mainly done two things：Firstly, we have tested the plasmid that we designed in 2016 iGEM program (<A href="http://parts.igem.org/Part:BBa_K2072000" style="color:blue">BBa_K2072000</A>), and we have found something interesting about ARA promoter; Secondly, we have also found that we could use E.coli BL21(DE3) strain alone to hydrolyze PNPG, therefore possibly ginsenosides. This method is safer than E.coli with plasmid we designed, and we think it fits our theme―food and nutrition.
    	  </P><H2 style="font-family:Arial;font-size: 45px;">Result</H2><H2 style="font-family:Arial;font-size: 45px;">Disscussion</H2><P class="text-justify" style="width:700px;text-indent:1em;text-align: left; font-size:18px;line-height:28px;font-family:Arial">
 In this year’s project, we did two things:
	</P><P class="text-justify" style="width:700px;text-indent:1em;text-align: left; font-size:18px;line-height:28px;font-family:Arial">
 Firstly, we tested the ara operon. Since ara operon has a response to Arabia sugar, we added lactose, arabinose, PNPG, and diluted bacteria culture into each well of 96-well plates. After a period of time of culturing at 37C, we measured the A620 and A450 using a micro-plate reader (A450 reflects the concentration of PNP and cell density, A620 only reflects cell density). To calculate the PNP concentration, we used this formula:
   </P><P class="text-justify" style="width:700px;text-indent:1em; text-align: left;font-size:18px;line-height:28px;font-family:Arial">
  For example, we have got the following result:
   </P><P class="text-justify" style="width:700px;text-indent:1em;text-align: left; font-size:18px;line-height:28px;font-family:Arial">
  The picture shows that, From the form, we can know that if the concentration of glucose is above 0.5%, the response of gene expression to the other things’ effecting will be completely suppressed. Since E. coli also have arabinose, we consider that with the specific changes in gene expression interfered by the plasmid we transferred arabinose in the cell is disassociated and to provide extra arabinose, so the response to our promoter has abnormal occurrence.
	</P><P class="text-justify" style="width:700px;text-indent:1em;text-align: left; font-size:18px;line-height:28px;font-family:Arial">
  Secondly, in order to use this plasmid to hydrolyze pseudo-ginseng which is rich in nutrition but difficult to be used by our human being, we also tested monosodium glutamate, jijing(a kind of Chinese spic which is mainly composed of monosodium glutamate and 5-Ribonucleotide), pseudo-ginseng and so on.
	</P><P class="text-justify" style="width:700px;text-indent:1em;text-align: left; font-size:18px;line-height:28px;font-family:Arial">
  By a lot of experimental conditions and statistics method, we found that with monosodium glutamate or extract of malt (a kind of Chinese spic, which is mainly composed of monosodium glutamate and 5-Ribonucleotide), enzyme production rate is really high, which means substrates will be hydrolyzed no more than 6 hours, but in other conditions, it will take more than 12 hours to get the same result.
	</P><P class="text-justify" style="width:700px;text-indent:1em;text-align: left; font-size:18px;line-height:28px;font-family:Arial">
  Last but not least, before we began our project, we’ve tested whether E.coli itself can hydrolyze Glycosidic bond, and the result has showed that it can not hydrolyse Glycosidic bond. Unfortunately, after we finished our project, we found in most cases E.coli itself can hydrolyze Glycosidic bond. So the result of our project contains notable error.
    </P><P class="text-justify" style="width:700px;text-indent:1em;text-align: left; font-size:18px;line-height:28px;font-family:Arial">
  But the good news is that in the actual production, we can use E.coli itself to hydrolyze pseudo-ginseng, and it is safer than E.coli with plasmid we designed. And we think it fits our theme―food and nutation.
    </P><DIV class="container" align="center"><H5>Copyright © 2017 UST_Beijing iGEM. All rights reserved.</H5></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></BODY></HTML>