Team:UST Beijing/Basic Part
USTB-Beijing | Welcome
Toggle navigation
Background
Project
Cyclase
Glucosidase
Modeling
HomePage
Parts
Team
Awards
HP
Collaborations
Safety
Attributions
InterLab
Basic Part
We designed a new coding BioBrick part( BBa_K2519000 ) and submitted it to iGEM registry for medal qualification. In addition, we used the plasmid BBa_K2072000 for exploration.
Plasmid Design
We used Swiss Model to make the three-Dimension model of ginseng cyclase. Then we used the Chimera to match amino acid sequences and constructions of 1w6k(the human cyclase) and ginseng cyclase. We found the first 90 amino acids on the two sequences were extremely different, which indicates obvious species specificity. In order to allow the squalene cyclase can function in the human body later, we replaced the 100 amino acid residues on N-terminal of the ginseng cyclase with 90 amino acid residues on N-terminal of 1w6k. And we found there were 5 amino acid residues site existing steric to the new peptide, so we switched them to the amino acid residues of same site on 1w6k.
We used the GeneArt(Thermal Fisher) software to do reverse translation to get the nucleotide sequence of Pangu cyclase and the website continue optimization codes. Finally we added additional stop codons to gene sequence for environmental safety in the rare event of recombination. Because the optimal gene fragments that the Gene Synthesis Company can synthesize is 1600bp, so we cut target sequence of 2224bp into two fragments of same size.
Plasmid Construction( BBa_K2519000 )
At first, we used the Gibson Assembly Master Mix to assemble the target fragments and pSB1c3 backbone.
Sequence1（25ng/μL）
4μL
Sequence2（25ng/μL）
4μL
pSB1C3 backbone（25ng/μL）
2μL
Enzyme
10μL
Total
20μL
Incubate at 50℃ for 1 hour. However we got many results of false positive. There have been appeared white and pink bacterial colonies on our medium plate.
And the agarose gel electrophoresis figure only showed the electrophoresis strips about 2000bp and 3000bp, which are backbone and plasmid containing red fluorescent gene. And we have tried many times, but we could’t find any colony containing our designed plasmid.
Finally we used PCR to amplificate the DNA fragments and isolated the DNA sequence of 2224bp. The target segment and the plasmid backbone of pSB1C3 were digested by the PstI ang EcoRI enzyme respectively. Then we used T4 DNA ligase to connect the backbone and target fragment.
Target fragment（40ng/μL）
4μL
Backbone（17ng/μL）
3μL
10Xbuffer
2μL
T4 DNA ligasee
0.4μL
ddH2O
11μL
Total
20.4μL
Incubate at 16℃ for 2 hours.
The electrophoresis figure2 showed our designed plasmid.
Not every team has the Dpn1 enzyme to cut up original template DNA, so we want to make a suggestion for IGEM foundation, the community may use Dpn1 enzyme to cut up the RFP plasmid before the delivery of backbone.
Plasimd Sequencing
Comparing the sequencing result with the sequence of pangu cyclase, we found that there were two mutation sites on the sequence.
According to the figure1, AAG mutated to GAG, so the 30th amino acid on peptide mutated from Glutamate to Lysine. Structural analysis showed that this amino acid is not interfering with other parts of the protein.
As showed on figure2, ATC mutated to ATA, but the amino acid kept as Isoleucine.
The first site may produce influence for the expression of squalene cyclase, which we have not known. We will verificate the function of the part later.
Copyright © 2017 UST_Beijing iGEM. All rights reserved.
