<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Glasgow_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Glasgow/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Glasgow/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Glasgow/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Glasgow/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Glasgow/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Glasgow/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="menu"><DIV id="menuback"><UL id="menulist"><LI><A href="#" class="listhead" id="project">Project</A></LI><LI><A href="#" class="listhead" id="team">Team</A></LI><LI><A href="#" class="listhead" id="modelling">Modelling</A></LI><LI><A href="#" class="listhead" id="hp">Human Practices</A></LI><LI><A href="#" class="listhead" id="safety">Project Safety</A></LI><LI><A href="#" class="listhead" id="ref">Acknowledgements</A></LI><DIV id="headpic"><DIV id="headover"><UL class="project"><LI><A href="https://2011.igem.org/Team:Glasgow/Project" class="listmain">Project Summary</A></LI><LI><A href="https://2011.igem.org/Team:Glasgow/Lab_Book" class="listmain">Lab Books</A></LI><LI><A href="https://2011.igem.org/Team:Glasgow/Results" class="listmain">Results</A></LI><LI><A href="https://2011.igem.org/Team:Glasgow/Parts" class="listmain">Parts</A></LI></UL><UL class="team"><UL class="team"><LI><A href="https://2011.igem.org/Team:Glasgow/Team" class="listmain">Meet the Team</A></LI><LI><A href="https://2011.igem.org/Team:Glasgow/iGEM_Diary" class="listmain">Team Vlog</A></LI><LI><A href="https://2011.igem.org/Team:Glasgow/Gallery" class="listmain">Photo Gallery</A></LI></UL></UL><UL class="modelling"><UL class="modelling"><UL class="modelling"><LI><A href="https://2011.igem.org/Team:Glasgow/Modeling" class="listmain">Modelling</A></LI><LI><A href="https://2011.igem.org/Team:Glasgow/ModelingTutorial" class="listmain">Tutorial</A></LI></UL></UL></UL><UL class="hp"><UL class="hp"><UL class="hp"><UL class="hp"><LI><A href="https://2011.igem.org/Team:Glasgow/Communicating_Science" class="listmain">Human Practices</A></LI></UL></UL></UL></UL><UL class="safety"><UL class="safety"><UL class="safety"><UL class="safety"><UL class="safety"><LI><A href="https://2011.igem.org/Team:Glasgow/Safety" class="listmain">Safety Overview</A></LI><LI><A href="https://2011.igem.org/Team:Glasgow/Safetybiobricks" class="listmain">Biobricks</A></LI></UL></UL></UL></UL></UL><UL class="ref"><UL class="ref"><UL class="ref"><UL class="ref"><UL class="ref"><UL class="ref"><LI><A href="https://2011.igem.org/Team:Glasgow/Acknowledgements" class="listmain">Acknowledgements</A></LI><LI><A href="https://2011.igem.org/Team:Glasgow/Attributions" class="listmain">Attributions</A></LI></UL></UL></UL></UL></UL></UL></DIV></DIV><H1>DISColi: 3D manufacturing platform for modular product synthesis</H1><P>Systems evolved to respond to light exist throughout nature; from unicellular photosynthetic organisms, to the phototropism response in plants.  Light is an abundantly available resource, offering precise control over a system at little cost or effort, even in remote locations. </P><P>These are just some of the reasons why we have decided to exploit light. DISColi is a novel bio-photolithographic system for engineering biofilms, and the utilization of light allows the precise control needed to create functional 2D and 3D structures and devices. Such technology already has applications in fields as microfluidics, nanotechnology and tissue engineering.</P><P>Our work with light responsive constructs has allowed us to take full advantage of the modular nature of synthetic biology.  The main aims of our project can be separated into three light-controlled components: the designed sculpting of biofilms; 3D printing for encapsulation of cells; and the controlled modular synthesis of a variety of products.  We expect our technology to have applications for material synthesis and compound manufacture in remote locations, for example outer space. 
</P><H2>System Design</H2><P>The system we have designed for iGEM 2011 takes advantage of the modular nature of synthetic biology by combining relatively few parts to create a complex and highly regulated system of gene expression. This diagram shows how the biobricks we intend to create can be organised to allow for the bio-photo-lithography process.</P><DIV align="center"><DIV id="centre"><P><B><FONT size="5">Please take time to click our <FONT color="red">interactive diagram!</FONT></FONT></B></P><P><B>Use the switch and the different colours of light to choose which device is active! Mousing over the highlighted devices will display the biobrick constructs we have designed to perform the device functions. Clicking the Pattern Formation, Product Selection or the Structure Fixation tab will highlight the devices used for the selected stage. DEPENDING ON CONNECTION SPEED THE DIAGRAM MAY TAKE SOME TIME TO LOAD.</B></P></DIV></DIV><H4>1)	The sculpting of Biofilms</H4><P>The complex 3D structures formed by microorganisms provide an ideal model for sculpting.  However, biofilms are also medically significant and can cause problems for human health (Høiby, Ciofu, and Bjarnsholt, 2010). Our first priority therefore was researcher safety.  We also had to deal with the problem of a lack of shuttle vector between <I>E.coli</I> and common biofilm forming lab strains, such as species from the genus <I>Pseudomonas</I>.  For this reason, we have worked with a novel biofilm forming synthetic biology chassis, <I>E.coli</I> Nissle 1917.  We intend to make this available to the Registry to allow future teams who wish to have a safer alternative that is still compatible with existing parts on the registry, to work with biofilms.</P><P>In order to sculpt shapes into the biofilm, it is necessary to disperse some of the microorganisms.  We have developed three novel biobricks designed to cause dispersal of the biofilm.  Under control of the light responsive promoters, we can use light to cause targeted dispersal of the biofilm. </P><P>Phosphodiesterase, an enzyme isolated from <I>Pseudomonas aeruginosa</I> PA01, breaks down the signalling molecule cyclic diguanylate (c-di-GMP).  This molecule is extremely important in biofilm formation and motility, and so the targeted overexpression of phosphodiesterase is designed to interfere with biofilm formation (Barraud et al., 2009)  </P><P>We have also created two biobricks of novel surfactant proteins.   These are Latherin and Ranaspumin, which are surfactant proteins with antimicrobial properties.  Both these proteins are studied for their antibiofilm activity (Kennedy, 2011.)  </P><P>To further enhance biofilm dispersal, we have also used two biobricks from the registry – Colicin E2 <A href="http://partsregistry.org/Part:BBa_K131000"> (Part BBa_ K131000)</A> and T4 Endolysin <A href="http://partsregistry.org/Part:BBa_K112806"> (Part BBa_K112806)</A>.  All five of these proteins have been modelled to predict their diffusion through a biofilm.  This allowed us to understand how large an area would be affected by triggering dispersal using our light responsive constructs. This meant we could predict how fine a resolution we could get when sculpting with light.</P><P>DIScoli centres on the use of three light responsive promoters – the BLUF domain <A href="http://partsregistry.org/Part:BBa_K238013"> (Part BBa_K238013)</A> which is activated by blue light, OmpC<A href="http://partsregistry.org/Part:BBa_R0082"> (Part BBa_R0082)</A> which is activated by green light and OmpF <A href="http://partsregistry.org/Part:BBa_R0084"> (Part BBa_R0084)</A> which is inhibited by red light.  We have collaborated with the University of Edinburgh team, who shared with us the improved versions of OmpC and OmpF.  </P><P>To characterise the light promoters, we developed two novel reporters, LOV2 and iLOV.  These reporters have numerous advantages over GFP derived fluorescent proteins, such as their small size and, in the case of iLOV, the ability to quickly recover from photobleaching.  Another key advantage of the reporters LOV2 and iLOV is their ability to function under anoxic conditions.  This unique ability was particularly desirable to our team due to our work with biofilms.  We are using biofilms to engineer functional 2D and 3D structures using our light responsive constructs.  Bio-photolithography has many applications, including the creation of structures for tissue engineering and manufacturing nanodevices.  </P><H4>
2.)	3D printing for structure fixation/encapsulation</H4><P>The second component of our final product is structure fixation/encapsulation. For this we have chosen colanic acid, which is an exopolysaccharide that is naturally produced by E.coli  in order to protect from acidic environments. Colanic acid was previously worked on by the <A href="https://2009.igem.org/Team:Imperial_College_London">2009 Imperial College London</A> team, and its production can be overexpressed using biobricks of the transcription factors RcsA and RcsB. </P><H4>
3.)	The controlled modular synthesis of a variety of products</H4><P>A key component of DISColi is flexibility in manufacturing.  Taking advantage of the modular nature of synthetic biology, our engineered bacteria will be able to produce a variety of useful compounds simply by inserting the genes for the desired compound into the system. </P><P>This would mean the ability to use different wavelengths of light to select your desired product from a single precursor molecule.  As a proof of principle of manufacturing in response to light, we have been working with the carotenoid pathway.  In 2009, Cambridge submitted a number of biobricks involved in this pathway.  We have improved this by synthesizing the fourth gene, crtY, in the pathway and testing it under a pBAD promoter.   </P><P>
We believe it is vital to include a constant interaction with the public throughout our project. We aim to accomplish this through the use of regular Vlogs and personal Blogs. Uniquely, we have also created a daily time-lapse, with the hope that others can gain an insight into life in the lab.  Check out the rest of our wiki to find out what else we’ve been up to!</P><H2>References</H2><OL>1. Toh et al., 2009. <A href="http://pubs.acs.org/doi/abs/10.1021/la9019537"> &quot;Direct Biophotolithographic Method for Generating Substrates with Multiple Overlapping Biomolecular Patterns and Gradients&quot;</A> Langmuir, 25 (16), pp. 8894-8898</OL><OL>2. Voros J., 2007 <A href="http://www.lbb.ethz.ch/Publications/Presentations/Biolithography.pdf">
&quot;Biolithography: DNA-assisted Manufacturing of Nanodevices for Optical and Electronic Biosensing&quot;</A> Web source, date accessed:20/09/2011</OL><OL>3. Cao et al., 2010. <A href="http://www.ncbi.nlm.nih.gov/pubmed/20408974">&quot;A blue light-inducible phosphodiesterase activity in the cyanobacterium <I>Synechococcus elongates</I>&quot; </A>Photochemistry and photobiology, 86 (3), pp. 606-611</OL><OL>4. Christie et al., 1999. <A href="http://www.pnas.org/content/96/15/8779.short">&quot;LOV (light, oxygen, or voltage) domains of the blue-light photoreceptor phototropin (nph1): Binding sites for the chromophore flavin mononucleotide&quot;</A> PNAS, 96 (15), pp. 8779-8783</OL><OL>5. Chapman et al., 2008. <A href="http://www.pnas.org/content/105/50/20038.short">&quot;The photoreversible fluorescent protein iLOV outperforms GFP as a reporter of plant virus infection&quot; </A>PNAS, 105(50), pp. 20038-20043</OL><OL>6. Cooper A. and Kennedy M., 2010. <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2954283/">&quot;Biofoams and natural protein surfactants&quot;</A> Biophysical chemistry, 151 (3), pp. 96-104 </OL><OL>7. Kennedy M., 2011. <A href="http://www.biochemsoctrans.org/bst/039/bst0391017.htm"> &quot;Latherin and other biocompatible surfactant proteins&quot; </A>Biochemical Society Transactions, 39, pp. 1017-1022</OL><OL>8. Westendorf et al., 2005. <A href="http://www.ncbi.nlm.nih.gov/pubmed/15708311"> &quot;Intestinal immunity of <I>Escherichia coli</I> NISSLE 1917: a safe carrier for therapeutic molecules.&quot;</A> FEMS immunology and medical microbiology, 43 (3), pp. 373-384</OL><OL>9. Wang et al., 2000. <A href="http://www.ncbi.nlm.nih.gov/pubmed/11018145"> &quot;HOLINS: The Protein Clocks of Bacteriophage Infections&quot;</A> Annual review of microbiology, 54, pp. 799-825</OL><OL>10.Høiby, N., Ciofu, O., Bjarnsholt, T. 2010. <A href="http://www.futuremedicine.com/doi/abs/10.2217/fmb.10.125?journalCode=fmb"> &quot;Aeruginosa Biofilms in Cystic Fibrosis&quot;</A> Future of Microbiology 5(11) pp. 1663-1674</OL><OL>11.Barraud et al., 2009 <A href="http://jb.asm.org/cgi/content/abstract/191/23/7333"> &quot;Nitric Oxide Signaling in Pseudomonas aeruginosa Biofilms Mediates Phosphodiesterase Activity, Decreased Cyclic Di-GMP Levels, and Enhanced Dispersal.&quot;</A> Journal of Bacteriology 191, pp. 7333-7342</OL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Glasgow/Project">http://2011.igem.org/Team:Glasgow/Project</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Glasgow/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Glasgow/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Glasgow/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Glasgow/Project&amp;oldid=169600" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>