"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Glasgow/LOVresults
From 2011.igem.org
Project
Team
Modelling
Human Practices
Project Safety
Acknowledgements
Project Summary
Lab Books
Results
Parts
Meet the Team
Team Vlog
Photo Gallery
Modelling
Tutorial
Human Practices
Safety Overview
Biobricks
Acknowledgements
Attributions
iLOV Results
Back to Results
Aims
iLOV ( K660003 ) is an optimised version of the LOV2 fluorescent domain ( K660000 ). It shows better fluorescence and photostability. We aim to format this with biobrick ends, no illegal restriction sites, and ligate it into the submission vector in order to make it into a fluorescent reporter. The activity of iLOV can be monitored by it's fluorescence.
Methods
Due to the number of illegal restriction sites within iLOV, we had the construct synthesised and codon optimised for expression in E.coli.
Evolutionary, organisms have a bias towards certain codons. Due to the degenerate nature of the genetic code, there is more than one codon present for most amino acids. E.coli an organism with well studied codon preferences, and for this reason, we had the iLOV constuct synthesised in such a way that it was codon optimised.(Sharp et al., 1988)
Once the construct had been synthesised we ligated it into the submission vector and submitted it to the registry.
Results
The iLOV domain was successfully synthesised and submitted into the submission vector.
iLOV shows fluorescence under an excitation wavelength of 476nm, with an emission of 510-550nm.
Figure 1: Gel electrophoriesis of iLOV Biobrick (labeled b) and iLOV construct (labeled c). The samples are alternated uncut and EcorI-PstI double digest. Each iLOV variant was cloned onto the pSB1C3 submission vector before digestion. each second sample from the ladder shows two clear bands: the submission vector at 2kb and the iLOV insert at around 400kb. iLOV construct size: 404bp. iLOV biobrick size: 336 bp.
LOV2 Results
We have converted LOV2 into biobrick format by undertaking site-directed mutagenesis, adding biobrick ends and ligating it into a submission vector. ( K660000 )
Aims
LOV2 is a fluorescent domain which we have aimed to engineer as a reporter. Under excitation with the correct wavelength of light (476nm), fluorescence can be measured at an emission spectra of 510-550nm.
Figure 2: Graph showing absorption specra of LOV2 wild type (solid line) and a LOV 2 double mutant. Picture courtesy of Dr John Christie.
We obtained the LOV2 domain contained within a PUC vector in Top 10 cells. To ensure that we indeed have the domain, we were required to do a restriction digest. We also transformed DH5alpha cells with the LOV2 domain and made glycerol stocks.
The main aim for us in order to obtain the LOV2 domain was to get it into a biobrick format without any illegal restriction sites. This involved us designing specific PCR primers which contain the biobrick ends, and using these to PCR up our LOV2. After this we were required design primers for, and to do site-directed mutagenesis to get rid of the illegal pst1 site contained within the LOV2 sequence.
In order to test that the LOV2 domain works, we aimed to test the fluorescence emisson.
Once LOV2 has been adapted to suit the biobrick format and has been tested for activity, we were required to ligate it into the submission vector and submit it to the registry.
Methods
Checked we have LOV2 by restriction digest. This confirmed we have LOV2.
The LOV2 domain does not contain biobrick ends in nature, therefore we specially designed the forward and reverse primers shown in Table 1. The LOV2 domain was then PCRed up using these primers to ensure the finished PCR product had the correct biobrick ends. This PCR product was the transformed into Top 10 so fluorescence could be tested.
One round of site-directed mutagenesis was performed on LOV2,  using the forward and reverse SDM primers shown in Table 1. This got rid of the illegal pst1 site.
Table 1
Name of the primer
Sequence
Melting Temperature (oC)
LOV2 Forward
5'-GTGTGTGAATTCGCGGCCGCTTCTAGAGTCGCTGAAGGATCCAAGG-3'
73
LOV2 Reverse
5'-GTGTGTCTGCAGCGGCCGCTACTAGTATTATTAAACGTGGTCGGAACC-3'
72
LOV2 SDM Forward
5'-CGCAAAGGCGGTCTTCAGTACTTCATTGGTG-3'
64
LOV2 SDM Reverse
5'-CACCAATGAAGTACTGAAGACCGCCTTTGCG-3'
65
LOV2 Fluorescence was tested by growth on IPTG and illumination by both the transilluminator and Blak Ray lamp.
LOV2 was ligated into the submission vector. This product was then digested and run on a gel to ensure that it was successful. .
The LOV2 which has been successfully ligated into the submission vector was submitted to the registry.
Results
Table 2: Table showing the changed made to LOV2 by us during site directed mutagenesis to get rid of the illegal pst1 site.
Figure 3: Image showing LOV2 PCR product run on gel
Figure 4: Image showing our LOV2 containing DH5alpha plated on IPTG and Ampicillan plate.
Figure 5: Image showing uncut (uc) LOV2 site directed mutagenesis product compared to cut (c) LOV2 SDM product contained within submission vector. The product was cut with ECOR1 and PST1.
Figure 6: Image showing e.coli with an empty vector (left) compared to those containing the LOV2 sequence when viewed with UV light. Image from : Christie, J.M. et al (2007) Steric interactions stabilize the signaling state of LOV2 domain of phototropin 1. Biochemistry, 46 pp. 9310-9319
References
Cho, H.-Y., Tseng, T.S., Kaiserli, E., Sullivan, J., Christie, J.M. and Briggs, W.R. 2007 "Physiological roles of the light, oxygen, or voltage domains of phototropin 1 and phototropin 2 in Arabidopsis1." Plant Physiology, 143 (1). pp. 517-529. ISSN 0032-0889 Christie, J.M. et al 2007 "Steric interactions stabilize the signaling state of LOV2 domain of phototropin 1. " Biochemistry, 46 pp. 9310-9319 Sharp et al, 1988. "Codon usage patterns in Escheria coli, Bacillus subtilis, Saccharomyces cerevisiae, Schizosaccharomyces pombe, Drosophila melanogaster and Homo sapiens; a review of the considerable within-species diversity." Nucl. Acids Res.16, pp. 8207-8211
Retrieved from " http://2011.igem.org/Team:Glasgow/LOVresults "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
