<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Glasgow_LOV2"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Glasgow/LOV2">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Glasgow/LOV2&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Glasgow/LOV2&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Glasgow/LOV2&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Glasgow/LOV2" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Glasgow/LOV2</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="menu"><DIV id="menuback"><UL id="menulist"><LI><A href="#" class="listhead" id="project">Project</A></LI><LI><A href="#" class="listhead" id="team">Team</A></LI><LI><A href="#" class="listhead" id="modelling">Modelling</A></LI><LI><A href="#" class="listhead" id="hp">Human Practices</A></LI><LI><A href="#" class="listhead" id="safety">Project Safety</A></LI><LI><A href="#" class="listhead" id="ref">Acknowledgements</A></LI><DIV id="headpic"><DIV id="headover"><UL class="project"><LI><A href="https://2011.igem.org/Team:Glasgow/Project" class="listmain">Project Summary</A></LI><LI><A href="https://2011.igem.org/Team:Glasgow/Lab_Book" class="listmain">Lab Books</A></LI><LI><A href="https://2011.igem.org/Team:Glasgow/Results" class="listmain">Results</A></LI><LI><A href="https://2011.igem.org/Team:Glasgow/Parts" class="listmain">Parts</A></LI></UL><UL class="team"><UL class="team"><LI><A href="https://2011.igem.org/Team:Glasgow/Team" class="listmain">Meet the Team</A></LI><LI><A href="https://2011.igem.org/Team:Glasgow/iGEM_Diary" class="listmain">Team Vlog</A></LI><LI><A href="https://2011.igem.org/Team:Glasgow/Gallery" class="listmain">Photo Gallery</A></LI></UL></UL><UL class="modelling"><UL class="modelling"><UL class="modelling"><LI><A href="https://2011.igem.org/Team:Glasgow/Modeling" class="listmain">Modelling</A></LI><LI><A href="https://2011.igem.org/Team:Glasgow/ModelingTutorial" class="listmain">Tutorial</A></LI></UL></UL></UL><UL class="hp"><UL class="hp"><UL class="hp"><UL class="hp"><LI><A href="https://2011.igem.org/Team:Glasgow/Communicating_Science" class="listmain">Human Practices</A></LI></UL></UL></UL></UL><UL class="safety"><UL class="safety"><UL class="safety"><UL class="safety"><UL class="safety"><LI><A href="https://2011.igem.org/Team:Glasgow/Safety" class="listmain">Safety Overview</A></LI><LI><A href="https://2011.igem.org/Team:Glasgow/Safetybiobricks" class="listmain">Biobricks</A></LI></UL></UL></UL></UL></UL><UL class="ref"><UL class="ref"><UL class="ref"><UL class="ref"><UL class="ref"><UL class="ref"><LI><A href="https://2011.igem.org/Team:Glasgow/Acknowledgements" class="listmain">Acknowledgements</A></LI><LI><A href="https://2011.igem.org/Team:Glasgow/Attributions" class="listmain">Attributions</A></LI></UL></UL></UL></UL></UL></UL></DIV></DIV>


In order to visualise structures and cells under a microscope,  a tag or fluorescent marker is often used. Traditionally this is GFP (green fluorescent protein), however GFP is large, and fails to work in anoxic conditions. As we are working with biofilms, parts of which are anoxic,  we have decided to design two new fluorescent reporters (iLOV and LOV2) due to their small size and ability to work in such conditions.

<H6><A href="https://2011.igem.org/Team:Glasgow/Parts">Parts</A></H6><H6><A href="https://2011.igem.org/Team:Glasgow/Results">Results</A></H6><H1>LOV 2 Domain</H1><TABLE><TBODY><TR><TD><P>Fluorescent proteins have transformed our ability to visualise, quantify and monitor expression of proteins and other molecules within cells. One commonly used fluorescent proteins is GFP (Green Fluorescent Protein)</P><P>

Phototropins are structurally made up of two regions: an N-terminal photo-sensory domain and a C-terminal output serine/threonine kinase domain (Figure 1A). </P><P>
The photo-sensory region consists of two LOV (Light, Oxygen and Voltage) domains known as LOV1 and LOV2, which are each ~110 amino acids long. 
These domains are a subgroup of the larger receptor group known as the PAS family because they are associated with co-factor binding. Each of the two LOV domains acts by binding a flavin mono-nucleotide (FMN) in order to form a covalent adduct with a conserved cysteine residue (Figure 1B). Aside from their presence in plants, the LOV domains are also present in fungi and bacteria. </P><P>


Research was carried out to investigate the roles played by both LOV domains in phototropism, and it was found that LOV2 of both phot1 and phot2 plays a significant role, whereas LOV1 only plays a role in phot1 (Cho et al 2007).</P><P>

LOV domain activity can be monitored by measuring fluorescence. The FMN is responsible for the green fluorescence by changing from binding in a non-covalent state in the dark to binding covalently upon irradiation with light around 476nm. Fluorescence occurs with an emission spectrum peak of 510-550nm. As the covalent bond is broken when the molecule is returned to the dark, the reaction can be regarded as a reversible photo-cycle. </P></TD><TD><B>Figure 1:</B> (A) Diagram showing the structure of the blue light receptive phototropin. (B) Diagram showing the structure of the LOV2 domain with bound FMN co-factor. (Image by Dr John Christie, University of Glasgow)<DIV><B>Figure 2:</B> Image showing the fluorescence of LOV2 under UV light.(Image by Dr John Christie, University of Glasgow)</DIV></TD></TR></TBODY></TABLE><H2>iLOV</H2> 

Over time, LOV variants have been isolated, some of which show better fluorescence and photo-stability than classic LOV2. One of these has been named iLOV, and in studies whereby it has been used in comparison to GFP to track plant infections, it has outperformed GFP (Figure 2). GFP is around 225 amino acids  long, whereas iLOV is much smaller at around 100 amino acids. For this reason it better suited as a reporter for the movement of things such as viruses (Figure 3). Unlike GFP which photo-bleaches irreversibly, iLOV undergoes spontaneous recovery from photo-bleaching under high intensity exposure to UV (Figure 4). This photo-bleaching reversibility is due to the changing state between the fluorescent and non-fluorescent form of the bound FMN chromopore. Another advantage of LOV in comparison to other fluorescent proteins is it's ability to work in anoxic conditions, making it ideally suited to work within biofilms.

<P>
Fluorescent imaging of iLOV can be done by using an excitation wavelength of 476nm, with fluorescent emission between 510 and 550 nm. (Chapman et al 2008)

</P><B>Figure 3: Digram showing movement of TMV (tobacco mosaic virus).</B> On the left shows TMV with iLOV, and the centre and right show TMV with GFP. We can see that TMViLOV shows systemic infection, whereas TMVGFP shows poor, or no infection.Image taken from: Chapman, S. et al (2008) The photo-reversible fluorescent protein iLOV outperforms GFP as a reporter of plant virus infection. PNAS, 105 (50) pp. 20038 - 20043

<P><B>Figure 4: Image demonstrating photo-bleaching reversibility of iLOV.</B> Left – iLOV pre-bleach, Centre – iLOV post-bleach, and Right – iLOV post-recovery. Image taken from Chapman et al (2008) The photo-reversible fluorescent protein iLOV outperforms GFP as a reporter of plant virus infection. PNAS, 105 (50) pp. 20038 - 20043

 iLOV can be regarded as a useful fluorescent protein as it confers many advantages compared to GFP.  Table 1: Table showing the advantaged of iLOV compared to GFP. </P><H2>Biobrick Information</H2><H3>LOV2 (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K660000">K660000</A>)</H3>
LOV2 was amplified from a plasmid containing <I>lov2</I> which we acquired from Dr Andrew Roe at the University of Glasgow. The LOV2 gene contains one illegal restriction site which we removed site directed mutagenesis (SDM) using the Stratagene QuikChange kit. The primers used to remove this site: forward CGCAAAGGCGGTCTTCAGTACTTCATTGGTG  and reverse CACCAATGAAGTACTGAAGACCGCCTTTGCG.
<H3>iLOV (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K660004">K660004</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K660003">K660003</A>)</H3>Taking the amino acid sequence for iLOV we codon optimised the nucleotide sequence for <I>E. Coli</I> and remove three illegal restriction sites in the sequence. The final sequence was synthesised by Geneart. Strong RBS (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_B0030">BBa_B0030</A>) and a terminator (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_B1006">BBa_B1006</A>) were also added to the iLOV sequence as a handy construct for characterising promoters. This construct was also synthesised by Geneart.<H2>References</H2>
Chapman, S. et al (2008) <A href="http://www.pnas.org/content/105/50/20038.full"> &quot;The photo-reversible fluorescent protein iLOV outperforms GFP as a reporter of plant virus infection&quot; </A> PNAS, 105 (50) pp. 20038 - 20043

<P>Cho, H.-Y., Tseng, T.S., Kaiserli, E., Sullivan, J., Christie, J.M. and Briggs, W.R. 2007 <A href="http://eprints.gla.ac.uk/54876/"> &quot;Physiological roles of the light, oxygen, or voltage domains of phototropin 1 and phototropin 2 in Arabidopsis1.&quot;</A> Plant Physiology, 143 (1). pp. 517-529. ISSN 0032-0889


Christie, J.M. et al 2007 <A href="http://eprints.gla.ac.uk/10520/"> &quot;Steric interactions stabilize the signaling state of LOV2 domain of phototropin 1. &quot;</A>Biochemistry, 46 pp. 9310-9319


 For results, click <A href="https://2011.igem.org/Team:Glasgow/LOVresults">here.</A></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Glasgow/LOV2">http://2011.igem.org/Team:Glasgow/LOV2</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Glasgow/LOV2" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Glasgow/LOV2" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Glasgow/LOV2&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Glasgow/LOV2&amp;oldid=169381" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>