"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Glasgow/PDE
From 2011.igem.org
Project
Team
Modelling
Human Practices
Project Safety
Acknowledgements
Project Summary
Lab Books
Results
Parts
Meet the Team
Team Vlog
Photo Gallery
Modelling
Tutorial
Human Practices
Safety Overview
Biobricks
Acknowledgements
Attributions
C-Di-GMP Phosphodiesterase
Back to Parts
Phosphodiesterases are a family of enzymes naturally present in microorganisms, which break phosphodiester bonds. The specific enzyme we are using breaks the phosphodiester bond in the second messenger nucleotide, cyclic digaunylate (cyclic-di-GMP). Cyclic-di-GMP is important in many bacterial processes, including biofilm formation, motility and virulance. The phosphodiesterase regulates signal transduction by controlling levels of the signalling molecule in cells. High cellular levels of c-di-GMP are required to trigger behaviour that is crucial to biofilm formation. For example in E. Coli c-di-GMP binds to the protein YcgR which is a member of the PilZ(+) family of effector proteins which in turn triggers decreased flagella activity. If this signal is stopped the bacteria will increase flagella activity causing the biofilm to disperse. This approach of quenching the c-di-GMP signal has been shown to disperse biofilm by (Ma et al). This enzyme was amplified from the genome of Pseudomonas aeruginosa strain PA01. The enzyme contains a domain (vieA) which specifically targets the cyclic-di-GMP molecule and an EAL domain that that performs the catalytic function. We are using the enzyme under conditional promoters to control the levels of cyclic-di-GMP within the cell. We expect that the targeted expression of phosphodesiterase could be used to interfere with biofilm formation or to trigger dispersal.
Figure 1: The structure and function of C-Di-GMP. C-Di-GMP is created by the enzyme deguanylate cyclase and is broken down by phosphodiesterase. High levels of c-di-GMP activate functions that are necessary for biofilm growth. Image from "Principles of c-di-GMP signalling in bacteria" by Regine Hengge.
Biobrick Information
c-di-GMP Phosphodiesterase ( K660300 )
This biobrick was amplified from the genome of Pseudomonas Aeruginosa strain PAO1 using the following primers:
Forward GCACGATGGGCTCGACCTCG,
reverse CGCGGGAACCGGAAATCCGTAG.
This amplifies a region that is 1261 bases long. The start codon is unusual in that it is GTG instead of the normal ATG although it still encodes for a methionine. Site directed mutagenesis is required to remove two illegal restriction sites.
The primers for site 1:
forward CAGCCAGGCCGACCTGCAACGCCTGCGCG,
reverse CGCGCAGGCGTTGCAGGTCGGCCTGGCTG.
The primers for site 2:
forwards CGCCCTGCTTTGCAGCCGTGCCAAGG,
reverse CCTTGGCACGGCTGCAAAGCAGGGCG.
The success of the SDM will be determined by sequencing.
Due to the number of processes in which cyclic-di-GMP is used by prokarytes, we hope that future iGEM teams will find diverse uses for this biobrick.
Figure 2: Crystal structure of a phosphodiesterase bound to c-di-GMP. The c-di-GMP (sticks) molecule fits in a groove in the phosphodiesterase (cartoon).
References
Hengge, Regina "Principles of c-di-GMP signalling in bacteria", Nature Reviews Microbiology, 2009, Vol 7, pages 263-273. Ma, Qun et al "Engineering a novel c-di-GMP-binding protein for biofilm dispersal", Environmental Microbiology, 2011, Vol 13, pages 631-642. Tamayo et al., 2005. The EAL domain Protein VieA is a cyclic diguanylate phosphodiesterase. The Journal of Biological Chemistry, 280(39), pp. 33324-33330
Retrieved from " http://2011.igem.org/Team:Glasgow/PDE "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
