<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_USTC-China_project_cimager"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:USTC-China/project/cimager">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:USTC-China/project/cimager&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:USTC-China/project/cimager&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:USTC-China/project/cimager&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:USTC-China/project/cimager" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:USTC-China/project/cimager</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><H1><A href="https://2014.igem.org/Team:USTC-China">USTC-iGEM</A></H1><LI class="toggle-topbar menu-icon"><A href="#"><SPAN>Menu</SPAN></A></LI><SECTION class="top-bar-section"><UL class="right"><LI id="li-1"><A href="https://2014.igem.org/Team:USTC-China/project"><SPAN>P</SPAN>roject</A></LI><LI id="li-2"><A href="https://2014.igem.org/Team:USTC-China/modeling"><SPAN>M</SPAN>odeling</A></LI><LI id="li-3"><A href="https://2014.igem.org/Team:USTC-China/notebook"><SPAN>N</SPAN>otebook</A></LI><LI id="li-4"><A href="https://2014.igem.org/Team:USTC-China/hp"><SPAN>H</SPAN>uman Practice</A></LI><LI id="li-5"><A href="https://2014.igem.org/Team:USTC-China/team"><SPAN>T</SPAN>eam</A></LI><LI id="li-6"><A href="https://2014.igem.org/Team:USTC-China/safety"><SPAN>S</SPAN>afety</A></LI></UL></SECTION><DIV id="main" class="row"><DIV class="large-3 columns" id="side-navbar"><DIV class="side-nav"><DL class="accordion" data-accordion=""><DD class="accordion-navigation"><A href="#panel">Project</A><DIV id="panel" class="content active"><UL class="side-nav"><LI><A href="https://2014.igem.org/Team:USTC-China/project/background">Background </A></LI><LI><A href="https://2014.igem.org/Team:USTC-China/project/ecoliphotography">E.coli Photography</A></LI><LI><A href="https://2014.igem.org/Team:USTC-China/project/cimager" class="active-a">C. imager</A></LI><LI><A href="https://2014.igem.org/Team:USTC-China/project/rna-rec">RNA Logic Gates &amp; Recombinase</A></LI><LI><A href="https://2014.igem.org/Team:USTC-China/project/killswitch">Kill Switch</A></LI><LI><A href="https://2014.igem.org/Team:USTC-China/project/results">Results</A></LI><LI><A href="https://2014.igem.org/Team:USTC-China/project/parts">Parts</A></LI></UL></DIV></DD></DL></DIV></DIV><DIV class="large-9 columns" id="content-page"><DIV class="title"><H1>C. imager</H1><DIV data-magellan-expedition="fixed"><DL class="sub-nav"><DD data-magellan-arrival="overviewoncaulobactercrescentus"><A href="#overviewoncaulobactercrescentus">Overview</A></DD><DD data-magellan-arrival="lifecycleandnutritionalprovide"><A href="#lifecycleandnutritionalprovide">Life Cycle and Nutritional Provide</A></DD><DD data-magellan-arrival="dgradgrb"><A href="#dgradgrb">DgrA DgrB</A></DD><DD data-magellan-arrival="hf"><A href="#hf">HfiA HfsJ</A></DD><DD data-magellan-arrival="circuitbasedonlight"><A href="#circuitbasedonlight">Circuit Based on Light</A></DD><DD data-magellan-arrival="conjugation"><A href="#conjugation">Conjugation</A></DD></DL></DIV></DIV><DIV class="text"><H4 data-magellan-destination="overviewoncaulobactercrescentus">Overview on Caulobacter Crescentus</H4><P>Caulobacter crescentus is a kind of bacteria with many necessity for our project: <I>C.crescentus</I> has a special life cycle, including swarmed period when <I>C.crescentus</I> possesses flagellum in one pole and stalked period when they cut up flagellum and develop an extremely strong polar adhesive envelope structure known as the holdfast which is composed of protein and polysaccharide. According to the research, when a holdfast clings to a surface, the force on it reaches 68 N/mm2 high, which is regarded as the strongest biological glue known to exist in nature and is three times stronger as commercial 'super glue'. With the knowledge we have already known, if we are able to regulate the biosynthesis of flagellum and holdfast simultaneously with flourescent proteins controlled by light input, <I>C.crescentus</I> will stop swimming, develop holdfast to stay firmly and evenly on the plate while expressing flourescent proteins corresponding to light color simulation.</P><P><FIGURE style="text-align:center"><FIGCAPTION>
Fig.1 Measurement of the size of holdfast of <EM> C.crescentus</EM> by AFM.(PNAS 2006,103-15)
</FIGCAPTION></FIGURE></P><P>In order to realize the blueprint of our design, we have these works to do as following: </P><P><OL><LI>Figure out the exact proteins regulating the biosynthesis of flagellum and holdfast or keeping bacteria staying in a relatively narrow place and then construct a circuit which contains controlling proteins triggered or inhibited by light signal.</LI><LI>Discover some methods to transfer accomplished light sensing-imaging systems working in <I>E. coli</I> into C.crescentus. </LI><LI>Confirm the exact nutrition conditions for an even <I>C.crescentus</I> biofilm genesis with modeling work and lab tests.</LI></OL></P><P>Here, we are attempting to introduce the whole view of this amazing bacteria for you to better understande our project and its specialty. </P><H4 data-magellan-destination="lifecycleandnutritionalprovide">Life Cycle and Nutritional Provide</H4><P>The cyclic developmental program of <EM>Caulobacter crescentus</EM> starts with a stalked cell with a polar adhesive holdfast at the tip of the stalk. The stalked cell firstly turns into S phase when it is time for DNA replication. At that time, the bacteria grow up into pre-divisional cells. Later the cell enters the G2 phase when the cell becomes incompetent for DNA replication and it keep growing until finally compartmentalizing into two cells, either of who contains stalk and flagellum in one pole. For the swarmer cell which possesses flagellum, the rotation of flagellum is activated and two pili are generated as well. The swarmer one enters <I>G1</I> phase when its chromosome is different from the stalked one and it cannot reproduce itself. However, in <I>G1</I> phase, the holdfast is formed predominantly in the swarmer cell stage and these swarmer cells reenter S phase like a cell differentiation process exactly simultaneously when rotating flagellum disappears and holdfast is synthesized. As the research says, the development of stalk and holdfast help <EM>C.crescentus</EM> to live in some tough surroundings and enhance its ability to absorb phosphate passively when nutrition provide is limited and swimming using flagellum waste the energy that is original manuscript for necessary metabolism. Scientists found that <I>Pst</I> family correlate the length of stalk. As the hypothesis contains, if it is correct, stalk elongation may function to elevate single cell away from surface thus bacteria receive much high-level nutrient flux and if <EM>C.crescentus</EM> colonize with other organisms, stalk ensures bacteria to have greater access to nutrients comparing to other nearby surface species to be superior in competition and survival. Consequently, C.crescentus is able to survive without flagellum and even much more stronger when containing holdfast.</P><P><FIGURE style="text-align:center"><FIGCAPTION>
Fig.2 Life cycle of <EM>C.crescentus</EM> (Microbiol. Mol. Biol. Rev 2010,74(1):13)
</FIGCAPTION></FIGURE></P><P>In accordance with recent research, many kinds of proteins are assistedly, antagonistically or stimulatively working for a distinct process during the life cycle of <EM>C.crescentus</EM> such as RodA and MreB are required for stalk synthesis and prevetion of ectopic pole formation, PodJ is an organelle development protein that differently localized and required for polar targeting on PleC development regulator, sigma-factor is also required for polar morphogensis and normal celldivision, BapE DNA endonuclease induces an apoptotic-like response to DNA damage which is useful for future kill switch design and SpmX localizaion relates to stalk position not only working in <EM>C.crescentus</EM>, but in <EM>A.xcentricus</EM> and <EM>A.biprosthecum</EM> etc. </P><P><FIGURE style="text-align:center"><FIGCAPTION>
Fig.3 Stalk elongation may function to elevate single cells away from surfaces.(Communicative &amp; Integrative Biology, 2013 6:4) 
</FIGCAPTION></FIGURE></P><P>In the whole life cycle of <EM>C.crescentus</EM>, we mostly focus on the process of degradation of flagellum and biosynthesis of holdfast. Thus, we will discuss the pathway of the two parts dividedly and design a reasonable circuit containing protein involved the pathway controlled by light signals. </P><H4 data-magellan-destination="dgradgrb">DgrA/DgrB</H4><P>The concentration of <I>c-di-GMP</I> is varying, and for our experiment we will promote its expression to inhibit rotation of flagella. It will bind to <I>DgrA</I> and <I>DgrB</I>. The resulting complex will have two routes to go: one is the [<I>c-di-GMP&amp;amp</I>;<I>DgrA</I>] complex which will inhibit <I>FliL</I>, a protein bounds to the membrane playing a key role in rotation of flagella. The other route is that the <I>DgrB</I> will directly affect the rotation of the flagella.</P><H4 data-magellan-destination="hf">HfiA/HfsJ</H4><P><I>Ref: A Cell Cycle and Nutritional Checkpoint Controlling Bacterial Surface Adhesion</I></P><P>In this paper, author describes a novel regulatory mechanism by which the <I>C.c.</I> bacterium integrates cell cycle and nutritional signals to control development of an adhesive envelop structure known as holdfast.</P><P>They discovered a novel inhibitor of holdfast development. <I>HfiA</I>, that is regulated downstream of <I>lovK-lovR</I>. They also discovered a bio-synthesis related gene named <I>HfsJ</I>. The suppressing mutations in HfsJ attenuate the <I>HfsJ-HfiA</I> interaction.</P><P><FIGURE style="text-align:center"><FIGCAPTION>
Fig.4 Biosynthesis pathway of holdfast.
</FIGCAPTION></FIGURE></P><P>These results support a model in which <I>HfiA</I> inhibits holdfast development via direct interaction with an enzyme required for holdfast bio-synthesis.</P><P>In conclusion, the author claimed that they demonstrated that the predicted glycosyltransferase, <I>HfsJ</I>, is a required component of the holdfast development machinery and that residues at the C-terminus of <I>HsfJ</I> mediate a direct interaction with <I>HfiA</I>, leading to a post-translational inhibition of <I>HfsJ</I>.</P><H4 data-magellan-destination="circuitbasedonlight">Circuit Based on Light</H4><P>Integrating photographic system with motion control and holdfast biosynthesis control by light, we got the whole view of the concept of <EM> C. imager </EM>, the circuits are the following:</P><P>In blue light system, darkness inhibits expression BFP and DgrA ,DgrB circuit, while blue light stimulates BFP expression and express DgrA and DgrB to let bacteria slow down. Blue light inhibits HfiA expression simultaneously, which leads to biosynthesis of holdfast to make bacteria adhere to surface more quickly.</P><P>In red light system, darkness inhibits expression mRFP and DgrA ,DgrB circuit as well, and red light guides BFP expression and produces DgrA and DgrB to cease flagellum rotation. Red light inhibits HfiA expression simultaneously, which leads to biosynthesis of holdfast to make bacteria adhere to surface more quickly.</P><H4 data-magellan-destination="conjugation">Conjugation</H4><P>Due to <EM>C. crescentus</EM> 's special traits, the achievement in this bacteria would extremely maxmize effectness of photography.</P><P>Then how to guide plasmids into <EM>C.crescentus</EM> from <EM>E.coli</EM> which is used to amplify vectors? The first method that hits our mind is chemical conversion. But through the literature, the main ways to transfer plasmids into C.crescentus are conjugation, electrotransformation and transduction. We test the first two-methods and only conjugation works at last. Bacterial conjugation is the transfer of genetic material (plasmid) between bacterial cells by direct cell-to-cell contact or by a bridge-like connection between two cells. Conjugation, regarded as a key bridge bewteen <EM>E.coli</EM> and <EM>C. crescentus</EM>, will improve the stability and resolution of pictures. </P><P>Protocol of conjugation is the following:</P><OL><LI>Culture <I>C.crescentus</I> and <I>E. coli</I> S17-1 overnight with the liquid medium.When the OD of <EM>C. crescentus</EM> reaches 2 and S17-1’s reaches 0.5,we can stop cultivating.</LI><LI>Mix S17-1 and <EM>C. crescentus</EM>  in a volume ratio of 2:1.Centrifuge it with 4000r/min and suspend the thallus again with 200μl LB liquid medium.</LI><LI>Spread out an 0.45μm nitrocellulose filter on the LB solid medium without antibiotics.</LI><LI>Spread the mixed bacteria liquid in step 2 onto the filter and incubate it at 30℃ for 3-7h.</LI><LI>After incubation,elute the thallus from the filter with 200-1000μl LB liquid medium.</LI><LI>Spread it on the PYE solid medium with antibiotics and incubate it at 30℃.Then we can get the <EM>C. crescentus</EM> with the target vector.</LI></OL></DIV></DIV></DIV><DIV class="footer-ustc"><DIV class="row"><P>Lots of thanks to <A href="https://2014.igem.org/Team:USTC-China/acknowledgement">our sponsors</A>, who help us to achieve what we have today</P><P>© 2014 USTC-iGEM Design. Distributed under <A href="http://creativecommons.org/licenses/by-sa/3.0/">BY-SA 3.0</A></P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:USTC-China/project/cimager">http://2014.igem.org/Team:USTC-China/project/cimager</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:USTC-China/project/cimager" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:USTC-China/project/cimager" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:USTC-China/project/cimager&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:USTC-China/project/cimager&amp;oldid=376112" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>